<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Aachen_Wetlab_Notebook skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Aachen/Wetlab/Notebook</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="navbarwrapper"><NAV id="content" class="navbar navbar-toggleable-lg navbar-light bg-faded" style="background-color:white"><NAV class="collapse navbar-collapse" id="navbarNavDropdown"><UL class="navbar-nav ml-auto"><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" id="navbarDropdownMenuLink" href="#" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
					PROJECT
				</A><NAV class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink"><A class="dropdown-item" href="https://2018.igem.org/Team:Aachen/Description">	Description</A><A class="dropdown-item" href="https://2018.igem.org/Team:Aachen/Project/Motivation">Motivation</A><A class="dropdown-item" href="https://2018.igem.org/Team:Aachen/Project/Results">Results</A></NAV></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" href="#" id="navbarDropdownMenuLink" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
					LAB
				</A><NAV class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink"><A class="dropdown-item" href="https://2018.igem.org/Team:Aachen/Wetlab/Overview">Overview</A><A class="dropdown-item" href="https://2018.igem.org/Team:Aachen/Wetlab/Workflow">Workflow</A><A class="dropdown-item" href="https://2018.igem.org/Team:Aachen/InterLab">InterLab Study</A><A class="dropdown-item" href="https://2018.igem.org/Team:Aachen/Wetlab/Parts">Part</A><A class="dropdown-item" href="https://2018.igem.org/Team:Aachen/Wetlab/Notebook">Notebook</A><A class="dropdown-item" href="https://2018.igem.org/Team:Aachen/Wetlab/Protocols">Protocols/Methods</A><A class="dropdown-item" href="https://2018.igem.org/Team:Aachen/Safety">Safety</A></NAV></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" id="navbarDropdownMenuLink" href="#" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
					HARDWARE
				</A><NAV class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink"><A class="dropdown-item" href="https://2018.igem.org/Team:Aachen/Hardware/Overview">Overview</A><A class="dropdown-item" href="https://2018.igem.org/Team:Aachen/Hardware">Spectrometer</A><A class="dropdown-item" href="https://2018.igem.org/Team:Aachen/Hardware/Wafer">Wafer</A></NAV></LI><LI class="nav-item "><A class="nav-link" href="https://2018.igem.org/Team:Aachen/Model">
					MODEL
				</A></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" href="#" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
					HP
				</A><NAV class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink"><A class="dropdown-item" href="https://2018.igem.org/Team:Aachen/Human_Practices">Human Practices Gold/Silver</A><A class="dropdown-item" href="https://2018.igem.org/Team:Aachen/Collaborations">Collaborations</A></NAV></LI><LI class="nav-item dropdown" style="padding-right0.5rem"><A class="nav-link dropdown-toggle" href="#" id="navbarDropdownMenuLink" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
					PEOPLE
				</A><NAV class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink"><A class="dropdown-item" href="https://2018.igem.org/Team:Aachen/Team">Team</A><A class="dropdown-item" href="https://2018.igem.org/Team:Aachen/supporter">Sponsors</A><A class="dropdown-item" href="https://2018.igem.org/Team:Aachen/Attributions">Attributions</A></NAV></LI><LI class="nav-item dropdown" style="padding-right:0rem"><A class="nav-link dropdown-toggle" href="#" id="navbarDropdownMenuLink" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
					AWARDS
				</A><NAV class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink"><A class="dropdown-item" href="https://2018.igem.org/Team:Aachen/Model">MODEL</A><A class="dropdown-item" href="https://2018.igem.org/Team:Aachen/Hardware">HARDWARE</A><A class="dropdown-item" href="https://2018.igem.org/Team:Aachen/Human_Practices">HUMAN PRACTICES</A><A class="dropdown-item" href="https://2018.igem.org/Team:Aachen/Measurement">MEASUREMENT</A><A class="dropdown-item" href="https://2018.igem.org/Team:Aachen/Entrepreneurship">Entrepreneurship</A><A class="dropdown-item" href="https://2018.igem.org/Team:Aachen/Public_Engagement">Public Engagement</A></NAV></LI></UL></NAV></NAV></DIV><DIV class="container"><DIV class="row justify-content-center"><DIV class="col-sm-1"><DIV style="background-color: #62676D!important;  border-radius: 5px;"><P style="color: white!important;">Weeks:</P><P><A class="weeksnotebook" href="#week1">Week 01</A><A class="weeksnotebook" href="#week2">Week 02</A><A class="weeksnotebook" href="#week3">Week 03</A><A class="weeksnotebook" href="#week4">Week 04</A><A class="weeksnotebook" href="#week5">Week 05</A><A class="weeksnotebook" href="#week6">Week 06</A><A class="weeksnotebook" href="#week7">Week 07</A><A class="weeksnotebook" href="#week8">Week 08</A><A class="weeksnotebook" href="#week9">Week 09</A><A class="weeksnotebook" href="#week10">Week 10</A><A class="weeksnotebook" href="#week11">Week 11</A><A class="weeksnotebook" href="#week12">Week 12</A><A class="weeksnotebook" href="#week13">Week 13</A><A class="weeksnotebook" href="#week14">Week 14</A><A class="weeksnotebook" href="#week15">Week 15</A><A class="weeksnotebook" href="#week16">Week 16</A><A class="weeksnotebook" href="#week17">Week 17</A><A class="weeksnotebook" href="#week18">Week 18</A><A class="weeksnotebook" href="#week19">Week 19</A><A class="weeksnotebook" href="#week20">Week 20</A></P></DIV></DIV><DIV class="col-sm-10"><H1>NOTEBOOK</H1><H2>
				Documentation of our work </H2><P>
				The Notebook provides comprehensibility to understand how and what we did. You can click on the bar on the left, to get directly to the week you want to see.
				</P><H3 id="week1">Week 1</H3><H4>05/28</H4><UL><LI><U><B>Creation of YPD-Medium:</B></U></LI><P> 
							1000 ml YPD-Medium were created after the protocol <A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol24">Liquid YPD Medium</A> in order to enable production of S. cerevisiae overnight cultures
							wich are needed for creation of competent cells and genome isolation. 
						</P><LI><U><B>S. cerevisiae overnight culture</B></U></LI><P>
							In order to produce competent cells for our project S. cerevisiae BY4742 cells were used for two overnight in YPD-Medium.two different colonies of S.cerevisiae 
							were used for two different overnight culturein 5 ml YPD medium each. Both inoculated tubes were let grown overnight at 30 Â°C.
						</P><LI><U><B>Creation of Yeast dropout medium without leucine</B></U></LI><P>
							500 ml Yeast dropout medium without leucine were produced after protocol <A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol27">yeast dropout medium</A>. In this case it was necessary to produce medium without leucin 
							because the fist KO it has to be done has an auxotrophy marker for leucin.
						</P></UL><H4>05/29</H4><UL><LI><U><B>IDT Primer Resuspension (PAA1 KO)</B></U></LI><P> 
							The first primers for the KO of the yeast gene PAA1 were resuspended in TE-Puffer (Volume = 10 times their nanomoles) to reduce the possibility 
							that DNAses could damage the storaged DNA.
						</P><LI><U><B>Creation of Competent Cells (BY4742)</B></U></LI><P>
							The genetic modification of WT BY4247 S. cerevisiae cells can only be achieved if they has been maden competent. In order achieve this
							the zymo research kit for creation of competent cells was used(<A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol20">Frozen-EZ Yeast Transformation II Kit</A>).
							Overnight cultures has been already prepared on the previous day.
						</P><LI><U><B>Creation of 1,2 % Agarose Solution</B></U></LI><P>
							500 ml agarose solution for gel electrophoresis were produced (Protopcol: <A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol10">Gel Electrophoresis solution</A>).
						</P><LI><U><B>PCR for KO Fragments (PAA1 KO)</B></U></LI><P>
							After the primer dilution, it was possible to start a PCR to amplify the needed fragments for the ztransformation. In this case, three were
							the amplified fragments: upside homologous arm, downside homologous arm and Leu5 auxotrophie marker. These are needed for the creation of 
							an integration cassette which can be used for genome integration. 
						</P></UL><H4>05/30</H4><UL><LI><U><B>Gel Electrophoresis for PAA1 fragments production PCR (05/29):</B></U></LI><P> 
							the gele electrophoresis showed only for the upside homologous arm positive results. It was necessary to repeat the PCR for the other fragments.
						</P></UL><UL><LI><U><B>Purification of ampflified Upside Homologous arm</B></U></LI><P>
							For the purifcation of the upside homologous arm for the PAA1 KO cassette a PCR clean-up kit was used (<A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol2">PCR clean up kit</A>.
							After the purification the concentration was measured with a nanodrop machine. 
							</P>
							Results: DNA: 37,6 [ng/ul]      A260/280: 1,8 [-]
							
							New PCR was needed to achieve higher DNA concentration of the fragment for the transformation.
						</UL><UL><LI><U><B>PCR for Upside Homologous Arm, Leu5 Auxotropie Marker and Downside Homologous Arm</B></U></LI><P>
							A new PCR was needed because the previous one did not worked or showed results wich are not good enough for a transformation in BY4742. A gradient PCR
		                    was started for the fragments which did not showed any results (downside homologous arm and Leu5 auxotropie marker). For the upside homologous arm
							the same annealing temperature was chosen as for the previous PCR (05/29) and as template was used the newly purified upside homologous arm fragment 
							(05/30). 
						</P></UL><UL><LI><U><B>Gel Electrophoresis for upside homologous arm (05/30) and PCR clean-up</B></U></LI><P>
							Gel electrophoresis showed good results for the upside homologous arm fragment. Consequently, fragment was purified with a PCR clean-up kit.
							
							Results: DNA: 50,5 ng/ul       A260/280: 1,98 [-]
						</P></UL><UL><LI><U><B>Gel Electrophoresis for Leu5 Auxotrophie Marker and Downside Homologous Arm (05/30) and PCR clean-up</B></U></LI><P>
							Gel electrophoresis showed bad results for every single fragments. Consequently it was necessary to try a new PCR with fresh isolated genome 
							and a different gradient for the annealing temperature.
						</P></UL><UL><LI><U><B>Test of competent BY4742 (05/30) with Plasmid p0255 on plates without leucin</B></U></LI><P>
							Before usege it is necessary to test the created competed cells produced with <A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol20">Frozen-EZ Yeast Transformation II Kit</A>. To test them,
							a trasmformation with the plasmid p0255 was performed (<A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol20">Frozen-EZ Yeast Transformation II Kit</A> was also used for the transformation).
							Cells were let grown overnight at 30 Â°C on plates without leucin.
						</P></UL><H4>06/01</H4><UL><LI><U><B>PCR for Leu5 Auxotropie Marker and Downside Homologous Arm:</B></U></LI><P> 
							A new PCR was needed because the previous one did not worked or showed results wich are not good enough for a transformation in BY4742. A gradient PCR
		                    was started for the fragments which did not showed any results (downside homologous arm and Leu5 auxotropie marker). For the upside homologous arm
							the same annealing temperature was chosen as for the previous PCR (05/30) and as template was used the newly purified upside homologous arm fragment 
							(05/30). 
						</P></UL><UL><LI><U><B>Gel Electrophoresis for Leu5 Auxotrophie Marker and Downside Homologous Arm (06/01)</B></U></LI><P>
							Gel electrophoresis showed good results for both fragments. Consequently, fragment was purified with a PCR clean-up kit.
							</P>
							Results: DNA: 57,1 ng/ul       A260/280: 1,83 [-]
							
							Results: DNA: 95,1 ng/ul       A260/280: 1,87[-]
						</UL><UL><LI><U><B>Creation of LiOAC with 10% SDS</B></U></LI><P>
							LiOAC with 10% SDS is an important compound for cell transformation and because it was no more avaiable it was necessary to create it again .
						</P></UL><UL><LI><U><B>Gel Electrophoresis for upside homologous arm (05/30) and PCR clean-up</B></U></LI><P>
							Gel electrophoresis showed good results for the upside homologous arm fragment. Consequently, fragment was purified with a PCR clean-up kit.
							</P>
							Results: DNA: 50,5 ng/ul       A260/280: 1,98 [-]
						</UL><UL><LI><U><B>Gel Electrophoresis for Leu5 Auxotrophie Marker and Downside Homologous Arm (05/30) </B></U></LI><P>
							Gel electrophoresis showed bad results for every single fragments. Consequently it was necessary to try a new PCR with fresh isolated genome 
							and a different gradient for the annealing temperature.
						</P></UL><UL><LI><U><B>Isolation of S. cerevisiae BY4742 genome:</B></U></LI><P> 
							Genome was isolated to be used as template for the gradient PCR for downside homologous arm and Leu5 auxotrophie marker. Isolation was performed 
							after <A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol19"> &quot;Quick and Dirty&quot; protocol</A>. Four genomes from four different BY4742 colonies were isolated. Final concentratioj was determined 
							with a nanodrop.
							</P>
							Result (Genome 1): DNA: 449,4  A260/280: 1,81
							
							Result (Genome 2): DNA: 510,3  A260/280: 1,83
							
							Result (Genome 3): DNA: 877,5  A260/280: 1,92
							
							Result (Genome 4): DNA: 861,3  A260/280: 1,45
							</UL><H3 id="week2">Week 2</H3><H4>06/04</H4><UL><LI><U><B>S. cerevisiae overnight culture</B></U></LI><P>
							In order to produce competent cells for our project S. cerevisiae BY4742 cells were used for two overnight in YPD-Medium.two different colonies of S.cerevisiae 
							were used for two different overnight culturein 5 ml YPD medium each. Both inoculated tubes were let grown overnight at 30 Â°C.
						</P><LI><U><B>Gel Electrophoresis for Leu5 Auxotrophie Marker and Downside Homologous Arm (06/01)</B></U></LI><P>
							Gel electrophoresis showed good results for both fragments. Consequently, fragment was purified with a PCR clean-up kit.
							</P>
							Results (downside homologouse arm): DNA: 57,1 ng/ul       A260/280: 1,83 [-]
							
							Results (Leu5 auxotrophie marker): DNA: 95,1 ng/ul       A260/280: 1,87[-]
						<LI><U><B>Transformation of BY4742 cells with PAA1 KO Cassette</B></U></LI><P>
							Transformation was performed after <A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol18">chemical transformation of S. cerevisiae protocol</A>. Amplified fragments upside homologous arm,
							downside homologous arm and Leu5 auxotrophie marker were used for the transformation. As a positive controll plasmid p0255 was transformed in competent
							cells. As a negative control WT S. cerevisiae BY4742 was used. All different transformation approches were plated on plates without leucin and incubated
							at 30 Â°C.
						</P></UL><H4>06/05</H4><UL><LI><U><B>PCR for Upside Homologous arm and Downside Homologous Arm:</B></U></LI><P> 
							A new PCR was needed because the concentration of the PAA1 KO fragments was nearly enough for a single transformation in BY4742. A gradient PCR
		                    was started for the fragments which did not showed any results (downside homologous arm and Leu5 auxotropie marker). For the upside homologous arm
							the same annealing temperature was chosen as for the previous PCR (05/30) and as template was used the newly purified upside homologous arm fragment 
							(05/30). 
						</P><LI><U><B>Gel Electrophoresis for  Upside Homologous arm and Downside Homologous Arm(06/02)</B></U></LI><P>
							Gel electrophoresis showed good results for both fragments. Consequently, fragment was purified with a PCR clean-up kit.
							</P>
							Results: DNA (upside homologous arm): 12,4 ng/ul       A260/280: 2,29 [-]
							
							Results: DNA (downside homologous arm): 22,7 ng/ul       A260/280: 2,26 [-]
							
							Due to low DNA concentration and high number of proteins and impurities, it was necessary to dispose of the purified fragments.
						<LI><U><B>Transformation of BY4742 cells with PAA1 KO Cassette</B></U></LI><P>
							Transformation was performed after <A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol20">Frozen-EZ Yeast Transformation II Kit</A>. Amplified fragments upside homologous arm,
							downside homologous arm and Leu5 auxotrophie marker were used for the transformation. As a positive controll plasmid p0255 was transformed in competent
							cells. As a negative control WT S. cerevisiae BY4742 was used. All different transformation approches were plated on plates without leucin and incubated
							at 30 Â°C.
						</P><LI><U><B>Creation of 2 M Betaine Stock Solution</B></U></LI><P>
							Thus, it has been observed that the performed PCR do not lead to high product concentration, it was decided to add betaine to the PCR mastermix 
							(10% of the total mastermix volume has been shown to have a positive effect on the DNA production). Consequently, 30 ml of 2 M betaine solution 
							were produced. Betaine was afterward stored at 4 Â°C.
						</P></UL><H4>06/06</H4><UL><LI><U><B> Production PCR for Upside Homologous arm and Downside Homologous Arm:</B></U></LI><P> 
							A new PCR was needed because the concentration of the PAA1 KO fragments was not enough for any single transformation in BY4742. A gradient PCR
		                    was started for the fragments which did not showed any results (downside homologous arm and Leu5 auxotropie marker). For the upside homologous arm
							the same annealing temperature was chosen as for the previous PCR (05/30). As template were used the newly purified upside homologous arm fragment 
							with low concentrartion(05/30) and BY4742 isolated genome (genome 1 06/01) for downside homologous fragment. 
						</P></UL><UL><LI><U><B>Gel Electrophoresis for  Upside Homologous arm and Downside Homologous Arm</B></U></LI><P>
							Gel electrophoresis showed good results for both fragments. Consequently, fragment was purified with a PCR clean-up kit.
							</P>
							Results(upside homologous arm): DNA: 4,1 ng/ul       A260/280: 1,78 [-]
							
							Results (downside homologous arm): DNA: 93,8 ng/ul       A260/280: 1,53 [-]
							
							Due to low DNA concentration and high number of proteins and impurities, it was necessary to dispose of the upside homologous arm.
						</UL><UL><LI><U><B>Transformation of dH5-alpha E. coli cells</B></U></LI><P>
							Every used auxotrophie marker is contained in plasmid which we recieved from our laboratory, however it was necessary to produce some backups 
							in E. coli dh5-alpha, because the recieved quantity of plasmid was barley enough for two PCRs. Consequently, a the recieved plasmid p0257 was 
							transformed in E. coli. Transformed cell were incubated at 37 Â°C overnight.
						</P></UL><H4>06/07</H4><UL><LI><U><B> Overnight cultures with ampicillin LB-Medium for transformed E. coli dh5-alpha cells (06/06):</B></U></LI><P> 
							The transformation of the E. coli cells dh5-alpha was successful. Two colonies were used for an overnight culture in 5 ml LB-medium with ampicillin 
							each. Overnight cultures will be used for plasmid isolation.
						</P></UL><UL><LI><U><B>Gradient PCR for the Downside Homologous Arm and gel electrophoresis</B></U></LI><P>
							A new PCR was needed because new DNA of the downside momologous arm has to be used for a transformation. A gradient PCR
							was started. In spite of using betaine and fresh isolated genome, it was not possible to achieve downside homologous arm production. 
						</P></UL><H4>06/08</H4><UL><LI><U><B>Gradient PCR for the Downside Homologous Arm and gel electrophoresis</B></U></LI><P>
								A new PCR was needed because new DNA of the downside momologous arm has to be used for a transformation. A gradient PCR
								was started. With this PCR three different PCR protocols should be tested for the same PAA1 KO fragment (downside homologous arm).
							</P></UL><UL><LI><U><B> Gel electrophoresis for the gradient PCR to test different PCR protocols:</B></U></LI><P> 
								Only, did one protocols work. The protocol which worked is the one with usage of ten percent betaine in the PCR mastermix and twice as much Enzyme 
								as forseen in the original PCR protocol. Now, a unique protocol could be establish for further PCRs.
								Due to the small used sample volume for the PCR, it was not possible to purified the amplified fragment.
				</P></UL><H4>06/07</H4><UL><LI><U><B> Overnight cultures with ampicillin LB-Medium for transformed E. coli dh5-alpha cells (06/06):</B></U></LI><P> 
							The transformation of the E. coli cells dh5-alpha was successful. Two colonies were used for an overnight culture in 5 ml LB-medium with ampicillin 
							each. Overnight cultures will be used for plasmid isolation.(<A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol5">Overnight of E. coli Cells</A>)
						</P></UL><UL><LI><U><B>Gradient PCR for the Downside Homologous Arm and gel electrophoresis</B></U></LI></UL><UL>
							A new PCR was needed because new DNA of the downside momologous arm has to be used for a transformation. A gradient PCR
							was started. In spite of using betaine and fresh isolated genome, it was not possible to achieve downside homologous arm production.
				
						</UL><H3 id="week3">Week 3</H3><H4>06/11</H4><UL><LI><U><B>Gradient PCR for the Downside Homologous Arm and gel electrophoresis</B></U></LI><P>
								two PCRs were carried out with the new establish protocol. Both, were gradient PCR. However, the new protocol was used, both PCR failed.
								Probably, failure happend because we had to use a very small amout of  BY4742 genome which was difficult to pipette.
							</P></UL><UL><LI><U><B>Plasmid isolation of p0257 shuttle vector from E. coli dh5-alpha</B></U></LI><P>
										Produced overnight E. coli dh5-alpha overnight cultures with p0257 (06/10) were used to extract the plasmid p0257. Extraction 
										was performed with a purification kit (protocol <A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol1"> plasmid clean up</A>).
										Result: DNA:    ng/ul     A260/280:   [-]
									</P></UL><H4>06/12</H4><UL><LI><U><B>Creation of 100 ng/ml stock solutions for the isolated genome</B></U></LI><P>
								Isolated genpome was diluited with TE puffer in order to achieve a concentration of 100 ng/ul, wich can be used as a stock 
								solution for PCR mastermixes. 5 ml of stock solution were prepared. (Genome 1 produced on the 06/01 was used for the diluition)
						</P></UL><UL><LI><U><B> Gradient PCR for downside homologous arm with different template concentrations:</B></U></LI><P> 
								As it was not possible to produce ampflify any downside homologous arm anymore, it was decided to test different genome concentration
								which has always been used as template. It was supposed that genome could reduce the efficiency of the PCR, so it was decided to test different concentrations
								of it. Tested were three different concentration: 425 ng/ul, 100 ng/ul and 30 ng/ul.
							</P></UL><UL><LI><U><B>Gel electrophoresi for PCR to test different template concentrations</B></U></LI><P>
								Best results could be observed for a template concentration of 100 ng /ul in the PCR mastermix.
						</P></UL><UL><LI><U><B> touchdown PCR for downside homologous arm:</B></U></LI><P> 
								As it was not possible to produce ampflify any downside homologous arm anymore, it was decided to try a touchdown PCR whose annealing temperature 
								is changed every cycle, just because it was supposed that also the annealing temperature could be a problem.
							</P></UL><UL><LI><U><B>Gel electrophoresi for the touchdown PCR:</B></U></LI><P>
								Gel showed no bands at all. The PCR did not worked. We will try again a production PCR with 1000 ng/ul DNA as a template and a different 
								gradient for the annealing temperature.
						</P></UL><H4>06/13</H4><UL><LI><U><B>Creation of YPD-Medium plates and YPD liquid Medium</B></U></LI><P>
								On this day two plates with modified E. coli cellls from Helsinki has came. They contain DNA modifications which enable the usage of RZR1
								transcription and its detection (luciferase was integrated as a reporter). In order to utilise them it was necessary to make backups and 
								overnight cultures. Consequently, it was necessary to produce fresh solid and liquid YPD-Medium. 500 ml of each were produced.(Protocols: <A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol24">YPD-Liquid-Medium</A> and <A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol25">YPD-Solid-Medium</A>) In addition, these modified cells contain a plasmid with auxotrophie marker which
								we are not to need. As a result, it is necessary to make the cell lose the plasmid. In cÃ³nclusion, it was decided to let them grew a few days
								in full YPD medium.
/						</P></UL><UL><LI><U><B> Plating of modified BMA64-a1 S. cerevisiae cells from Helsinki </B></U></LI><P> 
								Recieved S. cerevisiae cells from Helsiki were plated on YPD-medium plates.
							</P></UL><UL><LI><U><B>IDT Primer Resuspension (PAA1 KO) for Helsinki cells</B></U></LI><P>
								New primers were recieved for the PAA1 KO because the old ones did not show any goos result and consequently they were rtesuspended in TE buffer to achieve a final concentration in the tubes of 100 ng DNA per ml.
								the used resuspension volume was ten times the value of the nanomoles primers contained in the tube
						</P></UL><H4>06/14</H4><UL><LI><U><B>gradient PCR for downside Homologous Arm with Leu5 Auxotrophoe marker as a positive control</B></U></LI><P>
								New gradient PCR was started to test the new primers we orderderd from IDT for the amplification of downside homologous arm. As a positive control for
								the mastermix, we amplificated within the same PCR the Leu5 Auxotrophoe marker fragment which has already shown good results in many different
								PCRs. 
						</P></UL><UL><LI><U><B> Gel electrophoresis for the gradient PCR for downside homologous arm </B></U></LI><P> 
								The gel electrophoresis only showed good results for the positive control (Leu5 Auxotrophoe marker fragment). It probably depends on the gradient or on the 
								used template concentration.
							</P></UL><UL><LI><U><B>Preparaion of yeast dropout medium withouth Leucine and Tryptophan </B></U></LI><P>
								Due to the new recieved cells and the necessity to work in the future with different genes and consequently markers, it was needed to produce new media suitable
								for different selectable markers.500 ml yeast dropout medium withouth Leucine and 250 ml yeast dropout medium withouth Tryptophan were produced. (Protocol: 
								<A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol27">yeast dropout medium - Sigma Alderich </A>)
						</P></UL><H4>06/15</H4><UL><LI><U><B>Production of ampicillin stocks (1000x in EtOH)</B></U></LI><P>
								5 ml Ampicillin were produced and stored at -80 Â°C. 
						</P></UL><UL><LI><U><B> Production of LB-Medium plates with ampicillin </B></U></LI><P> 
								500 ml LB-Medium with Agar and ampicillin were produced. 24 plates were poured and stored in asterile plastic bag at 4 Â°C.  (Protocol: 
								<A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol22"> LB-Medium </A>)
							</P></UL><H3 id="week4">Week 4</H3><H4>06/18</H4><UL><LI><U><B>Production of salmon Sperm for Yeast transformation</B></U></LI><P>
								Salmon sperm is normally used as a DNA carrier in chemical transformation of yeast cells. 10 ml salmon sperm were produced
								(<A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol15">Salmon Sperm</A>), divided in 5 different cryo culture tubes (2 ml each) and stored at - 20 Â°C.
						</P></UL><UL><LI><U><B> Results of the PAA1 KO transformation (06/04)</B></U></LI><P> 
								It has been observed some growth on the YPD-medium without leucin which has been used for the transformation of BY42742 S. cerevisiae cells
								with the PAA1 KO cassette. This is a positive result, however it genome had to be isolated to check the results of the integration in the cell.
								Genome of three different positive colonies was isolated. (Protocol: <A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol19"> Quick and Dirty </A>)
							</P></UL><H4>06/19</H4><UL><LI><U><B>Verification PCR for the transformation of the PAA1 KO cassete in S. cerevisiae </B></U></LI><P>
								PCR was started with a wide annealing temperature gradient to test the transformation of the PAA1 KO cassette in the BY4742 cells genome(06/18).
						</P></UL><UL><LI><U><B> Results of the verification PCR (06/10)</B></U></LI><P> 
								Good results has been obtained for one of the analysed colonies. It was possible to see that a band of the expected size has been produced in the PCR 
								at four different annealing temperatures. It confirms that the integration has occoured at the right place in the genome and the integration cassette 
								has been completely integrated without deletion or major modification. Oveernight culture in 5 ml YPD-medium without leucine of the positive clone has been done.
							</P></UL><H4>06/20</H4><UL><LI><U><B>production PCR for downside homologous arm</B></U></LI><P>
								PCR was started in order to amplify the downside homologous arm fragment. It was not a gradient PCR because it has already been shown once 
								that it was possible at a certain annealing temnperature to successfully amplify the chosen fragment.(06/06). This time, fresh polymerase and 
								new primers were used. 
						</P></UL><UL><LI><U><B> Results of the production PCR </B></U></LI><P> 
								Good results has been obtained for the chosen fragments. Consequently, the fragment was purified with a PCR clean-up kit (protocol 
								<A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol2">PCR clean-up</A>).</P>
								Result: DNA: 83,3 ng/ul  A260/280: under 2,0
								</UL><UL><LI><U><B> Test of the Helsinki cells (06/13) </B></U></LI><P> 
								It was needed to eliminate a plasmid contained in the BMA64-a1 modified cells from Helsinki before using them for further experiments and
								transformations. A colony has been growing for three day in full mediu to achieve plasmid loss. To test it, it was firstly prepared a diluition
								series (from 10^0 to 10^-6) to obtain single colony to test. Plates were incubated at 30 Â°C.
							</P></UL><H4>06/21</H4><UL><LI><U><B> Creation of competent BY4742 + PAA1 KO cassette S. cerevisiae cells</B></U></LI><P> 
								Due to the good results of the Transformation (06/19), BY4742 cells with the PAA1 KO cassetted integrated in their genome has been growth 
								overnight (between the 06/20 and the 06/21)in order to be maden competent on the following day. (protocol: <A href="">chemical competent Yeast cells</A>)
								1 ml of the produced overnight culture has been used in this experiment. 10 aliqouts of 80 ul each has been produced in this expirement and stored at -80 Â°C.
							</P></UL><H4>06/22</H4><UL><LI><U><B>test Transformation of competent BY4742 + KO PAA1 cells with KanMX plasmid:</B></U></LI><P>
								Competent BY4742 cells with PAA1 KO cassetted were tested with a shuttle vector containing a kanMX selectable marker. Cells were transformed after
								<A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol18"> transformation of chemical competent yeast cells </A>. Finally, they were incubated at 30 Â°C.
						</P></UL><UL><LI><U><B> Growth on selection medium of modified BMA64-a1 cells </B></U></LI><P> 
								BMA64-a1 has grown on the full medium plate (06/20). Different colonies has been used to test plasmid loss. To test it, YPD medium withouth
								tryptophane has been used. Six different colonies from plate 10^-6 has been picked and used in this experiment. inocolated cells has been grown overnight
								at a temperature of 30 Â°C.
							</P></UL><H3 id="week5">Week 5</H3><H4>06/25</H4><UL><LI><U><B>production of YPD Medium</B></U></LI><P>
								1 l of YPD medium has been produced. (Protocol: <A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol24">YPD Medium</A>)
						</P></UL><H4>06/27</H4><UL><LI><U><B>Results of the overnight cultures on selected medium (06/22):</B></U></LI><P>
							It was expected no growth on medium without tryptophane for the modified BMA64-a1 cells withouth plasmid. It was observed that only colony #6 could not
							grow in liquid medium withouth tryptophane. This colony could consequently be used for further experiments.
							
						</P></UL><UL><LI><U><B>Genome Isolation of BMA64-a1 cells withouth plasmid </B></U></LI><P> 
								An overnight culture has been already prepared on the previous day. The genome was isolated twice from a liquid overnight culture. (Protocol: <A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol19"> 
								Quick and Dirty </A>). 
								</P>
								Result genome 1: DNA: 1000 ng/ul    A260/280: 2,35[-]
								
								Result genome 2: DNA:888,3 ng/ul    A260/280:2,6 [-]
								
								isolated genome could not be used for further experimentations or PCRs because its impurity level was too high. Samples were eliminated.
							</UL><UL><LI><U><B>Transformation of E. coli dh5-alpha with KanMX plasmid</B></U></LI><P> 
								KanMX plasmid was transformed in E. coli dh5-alpha because backups for the chosen plasmid were needed.(Protocol: <A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol13"> E. coli
								transformation</A>)
							</P></UL><UL><LI><U><B>Overnight culture of BMA64-a1 cells without plasmid</B></U></LI><P> 
								An overnight culture has been prepared for the BMA64-a1 cells without plasmid in full YPD medium. 5 ml were produced and incubated overnight.
							</P></UL><H4>06/28</H4><UL><LI><U><B>Creation of competent BMA64-a1 cells without plasmid:</B></U></LI><P>
							Competent BMA64-a1 cells without plasmid has been produced using the overnight culture  produced on the previous day. (protocol: <A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol20">Competent
							yeast cells</A>) 100 ul aliquots of competent cells has been produced.
						</P></UL><UL><LI><U><B>Genome Isolation of BMA64-a1 cells withouth plasmid </B></U></LI><P> 
								An overnight culture has been already prepared on the previous day. The genome was isolated from it. (Protocol: <A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol19"> 
								Quick and Dirty </A>). 
								</P>
								Result: DNA: 370 ng/ul    A260/280: under 2,0[-]
								
								isolated genome could be used for further experimentation because enough DNA was isolated and its purity was ok.
							</UL><H3 id="week6">Week 6</H3><H4>07/02</H4><UL><LI><U><B>Creation of 1,2 % agarose solution:</B></U></LI><P>
							500 ml agarose solution for gel electrophoresis were produced.  (protocol: <A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol10">Gel Electrophoresis Solution</A>) 
						</P></UL><UL><LI><U><B>IDT Primer dilution </B></U></LI><P> 
								New primers for all our integration cassettes and selectable markers has arrived. They were resuspended in TE buffer in order to achieve
								a final concentration of 100 ng/ul in each tube. every primer was diluted with as much TE buffer as 10 times their nanomoles. Primers were 
								stored at -20 Â°C.
							</P></UL><UL><LI><U><B>PCR for isolation of His Cassette from <A href="https://2017.igem.org/Team:Aachen/Demonstrate">NHX1 cells</A></B></U></LI><P>
							The previous iGEM team of our city also worked with yeast and used the same genetic toools as we are. Consequently, we decided to isolate the genome 
							of their modified yeast cells to extract via PCR their used auxotrophie markers. In this case, we isolated the NHX1 genome (DNA: 96,4 ng/ul). (protocol: <A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol19">Genome Isolation</A>) Afterwards, a production PCR was startet to isolate from the NHX1 genome the desired His
							marker.
						</P></UL><UL><LI><U><B>Gel electrophoresis for His cassette </B></U></LI><P> 
								Gel electrophoresis did not show good results. No band could be observed on the Gel. 
							</P></UL><H4>07/03</H4><UL><LI><U><B>Overnight for transformed E. coli dh5-alpha with KanMX plasmid(06/27)</B></U></LI><P>
							Transformed E. coli dh5-alpha cells with KanMX plasmid has grown on LB plates with kanamycin. An overnight for a positive transformed colony has
							has been prepared. Liquid LB medium with Kanamycin was used for the overnight to avoid plasmid loss.
						</P></UL><H4>07/04</H4><UL><LI><U><B>Gradient PCR for His cassette</B></U></LI><P>
							Gradient PCR was used to find the right annealing temperature and achieve the highest amplification of the His cassette. As a template the isolated genome
							of NHX1 cells was used. 
						</P></UL><UL><LI><U><B>Gel electrophoresis for the gradient PCR for the His cassette</B></U></LI><P>
							PCR worked. We were able to see bands of the right size at many different annealing temperature. Fragments were stored at -20 Â°C. 
						</P></UL><UL><LI><U><B>Transformation of competent BMA64-a1 cells with PAA1 KO cassette</B></U></LI><P>
							PAA1 KO cassette was transformed in BMA64-a1 cells. We do not know if our designed fragments are able to be integrated in the genome of 
							BMA64-a1 cells due to some diversities in the DNA sequence compared with the genome of BY4742 cells.(protocol: <A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol18">Transformation of Chemical competent
							Yeast cells</A>). Transformed cells were plated on plates with medium withouth leucine and incubated at 30 Â°C.
						</P></UL><H4>07/05</H4><UL><LI><U><B>Production PCR for PAA1 KO fragments</B></U></LI><P>
							Due to the need of new fragments, it has been necessary to produce new fragments for the PAA1 KO integration cassette. 
						</P></UL><UL><LI><U><B>Gel electrophoresis and PCR clean up of the PAA1 KO fragments</B></U></LI><P>
							Gel electrophoresis showed good results. Bands were to see for both fragments at the expected length. Consequently, both fragments were purified with a PCR
							clean-up kit. (protocol: <A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol2">PCR clean-up</A>).
							</P>
							Result for downside homologous arm: DNA: 4,1 ng/ul
							
							Result for upside homologous arm: DNA: 93,6 ng/ul  A260/280: 1,83 [-]
							
							The downside fragment has to be amplified again du to the low obtained concentration. Upside homologous arm could be stored at -20 Â°C because
							the DNA concentration was high.
						</UL><H4>07/06</H4><UL><LI><U><B>gradient PCR for fragments from NHX1-cells for a new cassette </B></U></LI><P>
							New PCR was needed to isolate from the mutants of the previous iGEM team part of thei integration cassette for NHX1.
						</P></UL><UL><LI><U><B>Gel electrophoresis for the His cassette and PCR clean-up</B></U></LI><P>
							PCR worked. Many bands of the same lenght could be observed for different annealing temperatures. All amplified fragments 
							for the His cassette showed good results. Afterwards fragments were purified with a PCR clean-up. (Protocol: <A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol2">PCR clean-up</A>)
							</P>
							Result upside fragment: DNA: 27,8 ng/ul   A260/280: 1,98 [-]
							
							Result downside fragment: DNA: 30,8 ng/ul  A260/280: 1,58 [-]
							
							Both purified fragments were stored at -20Â°C for further experiment.
						</UL><UL><LI><U><B>Isolation of the genome of BMA64-a1 cells with PAA1 KO cassette </B></U></LI><P>
							Genome was isolted from colony on the transformation plate. (protocol: <A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol19">Quick and Dirty</A>)
							</P>
							Result: DNA: 3,1 ng/ul
							</UL><UL><LI><U><B>Verification PCR for the isolated genomes of a BMA64-a1 cells with PAA1 KO cassette</B></U></LI><P>
							To test the success of the PAA1 KO cassette in the BMA64-a1 cells, a PCR with verification primers and the previously isolated 
							genome of BMA64-a1 cells with PAA1 KO cassette was started. 
						
						</P></UL><UL><LI><U><B>Results of the verification verification PCR</B></U></LI><P>
							Transformation did not worked. It was only possible to see bands which did not have the expected size for the PAA1 integration cassette. More
							genomes are needed to find a positive colony which can be used in further experimental steps of our project.
						
						</P></UL><H3 id="week7">Week 7</H3><H4>07/09</H4><UL><LI><U><B>Isolation of the genome of a second BMA64-a1 cell transformed with PAA1 KO cassette </B></U></LI><P>
							Genome was isolted from a colony on the transformation plate. (protocol: <A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol19">Quick and Dirty</A>)
							</P>
							Result: DNA: 8,8 ng/ul  A260/280: 1,35 [-]
							</UL><UL><LI><U><B>second Verification PCR for the isolated genomes of a BMA64-a1 cells with PAA1 KO cassette </B></U></LI><P>
							To test the success of the PAA1 KO cassette in the BMA64-a1 cells, a PCR with verification primers and the previously isolated 
							genome of BMA64-a1 cells with PAA1 KO cassette was started. 
						
						</P></UL><UL><LI><U><B>Results of the verification second verification PCR</B></U></LI><P>
							Transformation did not worked. It was only possible to see bands which did not have the expected size for the PAA1 integration cassette. More
							genomes are needed to find a positive colony which can be used in further experimental steps of our project.
						
						</P></UL><UL><LI><U><B> Creation of YPD-medium+G418 (solid):</B></U></LI><P>
							Because our positive control for every transformation is a plasmid with a G418 resistance, new YPD plates with G418 antibiotic are needed. 
							500 ml of LB medium plus G418 were produced (<A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol21">LB Medium with G418</A>). About 22 Plates could be produced.
						</P></UL><H4>07/10</H4><UL><LI><U><B>Isolation of the genome of a four further BMA64-a1 cell transformed with PAA1 KO cassette </B></U></LI><P>
							Genome was isolted from four different colonies on the transformation plate. (protocol: <A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol2">Quick and Dirty</A>)
							</P>
							Result #1: DNA: 3,5 ng/ul  A260/280: 1,48 [-]
							
							Result #2: DNA: 2,5 ng/ul  A260/280: 1,47 [-]
							
							Result #3: DNA: 45,8 ng/ul  A260/280: 1,86 [-]
							
							Result #4: DNA: 67,5 ng/ul  A260/280: 1,86 [-]
							
							Colony #1 and #2 were discarded due to their low DNA concentration. Colony #3 and #4 were used as templates for a verification PCR 
							for the integration of the PAA1 KO cassette in BMA64-a1 cells.
						</UL><UL><LI><U><B>Verification PCR for both isolated genomes from BMA64-a1 cells with PAA1 KO cassette </B></U></LI><P>
							To test the success of the PAA1 KO cassette in the BMA64-a1 cells, a PCR with verification primers and the previously isolated 
							genomes of BMA64-a1 cells with PAA1 KO cassette was started. 
						
						</P></UL><H4>07/11</H4><UL><LI><U><B>Transformation of BMA64-a1 cells with PAA1 cassette</B></U></LI><P>
							We tested that the produced competent BMA64-a1 cellls worked and had enough fragments of the PAA1 cassette left to try a transformation in BMA64-a1 cells
							(Protocol: <A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol13"> Transformation of Chemical Competent Yeast Cells</A>). As a positive control competent BMA64-a1 transformed with the kanMX 
							were used. As a negative controlo WT BMA64-a1 cells were used. Positive control was plated on YPD plates with G418 antibiotic, while cells transformed
							with the PAA1 KO cassette and WT negative control were plated on yeast dropout medium without leucine. Cells were incubated at 30 Â°C. 
						</P></UL><UL><LI><U><B>Production PCR for downside Homologous Arm and Downside homologousn Arm of the PAA1 KO cassette</B></U></LI><P>
							New fragments for the PAA1 KO cassette were needed, consequently a new production PCR was started. In this case, isolated genome of BMA64-a1 cells 
							was used as a template. No specific primers has been created for this PCR, instead the same primers were used which has been used for previous cassette isolation 
							in BY4742 cells (genome similarity).
						</P></UL><UL><LI><U><B>Results of the production PCR</B></U></LI><P>
							Nothing could be seen on the Gel. The PCR did not work as expected. It was not clear if the primer were the problem or the chosen PCR conditions.
						</P></UL><H4>07/13</H4><UL><LI><U><B>Isolation of the genome of a four BMA64-a1 cell transformed with PAA1 KO cassette </B></U></LI><P>
							Genome was isolted from four new colonies on the transformation plate. (protocol: <A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol19">Quick and Dirty</A>)
							</P>
							Result #1: DNA: 1,5 ng/ul  A260/280: 1,71 [-]
							
							Result #2: DNA: 10,5 ng/ul  A260/280: 1,96 [-]
							
							Result #3: DNA: 10,8 ng/ul  A260/280: 1,86 [-]
							
							Result #4: DNA: 2,5 ng/ul  A260/280: 1,82 [-]
							</UL><UL><LI><U><B>Verification PCR for isolated genomes from BMA64-a1 cells with PAA1 KO cassette </B></U></LI><P>
							To test the success of the PAA1 KO cassette in the BMA64-a1 cells, a PCR with verification primers and the previously isolated 
							genomes of BMA64-a1 cells with PAA1 KO cassette as a template was started. This was a gradient PCR to control the success of the transformation. 
						
						</P></UL><UL><LI><U><B>Results of the verification PCR</B></U></LI><P>
							For Genome #1 and #4 it was not possible to observe any band, while for genome #3 and #2 bands with the wrong length were obtained.
						</P></UL><UL><LI><U><B>Production PCR for downside Homologous Arm and Downside homologousn Arm of the PAA1 KO cassette</B></U></LI><P>
							New fragments for the PAA1 KO cassette were needed, consequently a new production PCR was started. In this case, isolated genome of BMA64-a1 cells 
							was used as a template. No specific primers has been created for this PCR, instead the same primers were used which has been used for previous cassette isolation 
							in BY4742 cells (genome similarity). A new Type of polymerase with high efficiency has been used in this PCR. 
						</P></UL><UL><LI><U><B>Results of the production PCR</B></U></LI><P>
							Nothing could be seen on the Gel. The PCR did not work as expected. Probably, it is not possible to isolate the desired fragments from the BMA64-a1 genome 
							with the primers designed for homologous arm extraction from BY4742 cells. New primers need to be designed.
						</P></UL><UL><LI><U><B>Creation of competent BMA64-a1 cells</B></U></LI><P>
							There were no more competent BMA64-a1 cells in our cabinet, as a results it was needed to create new ones (<A href=""> Competent Yeast cells after Zymo Research kit
							</A>). 9 tubes with 100 ul competent cells each were produced and stored at -20 Â°C.
						</P></UL><H3 id="week8">Week 8</H3><H4>07/17</H4><UL><LI><U><B>Creation of LB Medium with Chloramphenicol</B></U></LI><P>
							800 ml of LB with Chloramphenicol for biobricks production has been created. (Protocol: <A href="">LB medium with Chloramphenicol</A>)
						</P></UL><UL><LI><U><B>Test Transformation of competent BMA64-a1 cells (07/13) </B></U></LI><P>
							To test the success of the competence of the new produced BMA64-a1 competent cells, a transformation with the plasmid KanMX was done. (Protocol:
							<A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol18"> Transformation of Chemical Competent Yeast Cells</A>). Transformed cellls were plated on YPD plates with G418 antibiotic and 
							incubated at 30 Â°C.
						</P></UL><H4>07/19</H4><UL><LI><U><B>Production PCR for downside Homologous Arm and Downside homologousn Arm of the PAA1 KO cassette</B></U></LI><P>
							New fragments for the PAA1 KO cassette were needed, consequently a new production PCR was started. In this case, isolated genome from BMA64-a1 cells and
							isolated genome from WT BY4742 cell were used as templates. No specific primers has been created for this PCR, instead the same primers were used
							which has been used for previous cassette isolation in BY4742 cells. The usage of tw different templates should show us if the primers are the 
							problem for the isolation of the two different homologous arms or the PCR working condition are not ok.
						</P></UL><UL><LI><U><B>Results of the production PCR</B></U></LI><P>
							Nothing could be seen on the Gel. However, we could not use the obtained results for any conclusion because there was something wrong with the 
							agarose gel.
						</P></UL><UL><LI><U><B>Second Production PCR for downside Homologous Arm and Downside homologousn Arm of the PAA1 KO cassette</B></U></LI><P>
							Due to the not utilisable results of the previous PCR, it was necesary to repeat the production PCR for both homologous fragments of the PAA1 KO cassetted. 
							In this case, isolated genome from BMA64-a1 cells and isolated genome from WT BY4742 cell were used as templates. No specific primers has been created for this PCR, instead the same primers were used
							which has been used for previous cassette isolation in BY4742 cells. The usage of tw different templates should show us if the primers are the 
							problem for the isolation of the two different homologous arms or the PCR working condition are not ok.
						</P></UL><UL><LI><U><B>Results of the production PCR</B></U></LI><P>
							Because no bands could not be seen, It was supposed that the problem must be the PCR mastermix and not the primers. Primers should at least
							work for the WT BY4742 genome, however also there nothing could be observed.
						</P></UL><H3 id="week9">Week 9</H3><H4>07/23</H4><UL><LI><U><B>Isolation of ten different genomes of BMA 64-a1 cells with PAA KO (07/04):</B></U></LI><P>
							We isolated ten different genomes of BMA 64-a1 cells with PAA KO. The genome was isolated because we
							wanted to verificate the succsess of the PAA K.O. in this strain. We used the genome isolation protocol &quot;<A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol19">Quick and Dirty</A>. In addition, 
							we created a masterplate as back-up (YPD medium without leucin). This plate was incubated at 30 Â°C. Genome concentration was not reported because
							it was not planed to work with this isolated genomes again, but instead only to test the transformation of the PAA1 KO cassette in BMA64-a1 cells
							(07/04).
							
						</P></UL><UL><LI><U><B>Verification PCR for PAA KO in BMA 64-a1 cells:(colonies 1-4)</B></U></LI><P>
							To test the success of the PAA1 KO cassette in the BMA64-a1 cells, a PCR with verification primers and the previously isolated 
							genomes of BMA64-a1 cells with PAA1 KO cassette as a template was started (only four of the previous isolated genomes were used).
 							This was a gradient PCR to control the success of the transformation.
						</P></UL><UL><LI><U><B>Results of the verification PCR</B></U></LI><P>
							For the first four colonies it was not possible to observed the expected band for a corect transformation of the BMA64-a1 cassette. Further colonies 
							will be tested.
						</P></UL><UL><LI><U><B>Second Verification PCR for PAA KO in BMA 64-a1 cells (colonies 5-8):</B></U></LI><P>
							To test the success of the PAA1 KO cassette in the BMA64-a1 cells, a PCR with verification primers and the previously isolated 
							genomes of BMA64-a1 cells with PAA1 KO cassette as a template was started (only four of the previous isolated genomes were used).
 							This was a gradient PCR to control the success of the transformation.
						</P></UL><UL><LI><U><B>Results of the second verification PCR</B></U></LI><P>
							For the second four colonies it was not possible to observed the expected band for a corect transformation of the BMA64-a1 cassette. Further colonies 
							will be tested.
						</P></UL><H4>07/24</H4><UL><LI><U><B>Production of Agarose solution:</B></U></LI><P>
							500 ml of agarose solution had to be produced due to shortage of them. (Protocol: <A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol10">Agarose Solution</A>)
							
						</P></UL><UL><LI><U><B>Isolation of ten different genomes of BMA 64-a1 cells with PAA KO (07/04):</B></U></LI><P>
							We isolated eight different genomes of BMA 64-a1 cells with PAA KO. The genome was isolated because we
							wanted to verificate the succsess of the PAA K.O. in this strain. We used the genome isolation protocol &quot;<A href="">Quick and Dirty</A>. In addition, 
							we created a masterplate as back-up (YPD medium without leucin). This plate was incubated at 30 Â°C. Genome concentration was not reported because
							it was not planed to work with this isolated genomes again, but instead only to test the transformation of the PAA1 KO cassette in BMA64-a1 cells
							(07/04).
							
						</P></UL><UL><LI><U><B>Verification PCR for PAA KO in BMA 64-a1 cells:(colonies 1-8)</B></U></LI><P>
							To test the success of the PAA1 KO cassette in the BMA64-a1 cells, a PCR with verification primers and the previously isolated 
							genomes of BMA64-a1 cells with PAA1 KO cassette as a template was started.
 							This was a gradient PCR to control the success of the transformation.
						</P></UL><UL><LI><U><B>Results of the verification PCR</B></U></LI><P>
							Positive results could be observed for colony 1, 2, 7 and 8. For these four different colonies, we could show that the PAA1 KO cassette had been
							integrated correctly in the Yeast genome. An overnight culture in yeast dropuot medium without leucine of these four was prepared and incubated 
							at 30 Â°C.
						</P></UL><UL><LI><U><B>production PCR for fragments from the NHX1 cassette (mutants of the previous iGEM Team)</B></U></LI><P>
							Gradient PCR wa started to isolated two fragments for the next genome integration in BMA64-a1 and BY4742 cells with PAA1 KO cassette. As a template the isolated
							genome of NHX1 mutants of the iGEM team Aachen 2017 has  been used. 
						</P></UL><UL><LI><U><B>Results of the NHX1 production PCR</B></U></LI><P>
							It was not possible to amplificate at the chosen temperature the upside fragment of the cassette, however we it was possible to see
							some bands of the expected size for the downside fragment of the cassette at different annealing temperatures.
						</P></UL><UL><LI><U><B>PCR clean-up of the downside fragment of from the NHX1 cassette</B></U></LI><P>
							Three sample of the producion PCR for the upside NHX1 fragment of the NHX1 cassette were chosen and purified. (<A href="">PCR clean-up</A>)
							</P>
							Result sample 1: DNA: 60,6 ng/ul   A260/280: 1,56 [-]
							
							Result sample 2: DNA: 147,5 ng/ul   A260/280: 1,50 [-]
							
							Result sample 3: DNA: 67,2 ng/ul   A260/280: 1,58 [-]
							
							Fragments were stored at -20 Â°C.
						</UL><UL><LI><U><B>Gradient PCR for fragments up and down from p0255 plasmid and GAL1 from NHX1 mutants</B></U></LI><P>
							Gradient PCR wa started to find the right annealing temperature for four fragments needed for a transformation in BMA64-a1 cells and BY4742
							cells with PAA1 KO cassette. As template the Plasmid P0255 and the isolated genome of NHX1 cells were used. 
						</P></UL><UL><LI><U><B>Results of the Gradient PCR for fragments up and down from p0255 plasmid and GAL1 from NHX1 mutants:</B></U></LI><P>
							PCR worked. On the gel it ws possible to find for every fragment many different bands of the right size at many different annealing temperatures.
							These results can now be used for a new production PCR to isolate enough fragments to perform a transformation in yeast the already modified 
							BMA64-a1 and BY4742 cells.
						</P></UL><H4>07/25</H4><UL><LI><U><B>Production PCR for fragments up and down from p0255 plasmid and GAL1 from NHX1 mutants:</B></U></LI><P>
							Production PCR was started to amplify enough of the desired fragments to perform a transormation. As template p0255 Plasmid and isolated NHX1 genome were
							utilise. 
						</P></UL><UL><LI><U><B>Results of the production PCR for fragments up and down from p0255 plasmid and GAL1 from NHX1 mutants:</B></U></LI><P>
							PCR worked only for two (upside fragment and GAL1 fragment) out of the four fragments which has to be amplified. The gel showed for these two clear bands,
							however for the others it was not possible to see anything or just bands of the wrong size.
							
						</P></UL><UL><LI><U><B>PCR clean-up for upside NHX1 and GAL1 fragments</B></U></LI><P>
							Three sample from the PCR were used for the purification of the GAL1 fragments, while just one PCR sample could be used for the upside fragment.
							(protocol: <A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol2">PCR clean-up</A>)
							</P>
							Result sample 1 GAL1: DNA: 55.1 ng/ul   A260/280: 1,57 [-]
							
							Result sample 2 GAL1: DNA: 55.3 ng/ul   A260/280: 1,54 [-]
							
							Result sample 3 GAL1: DNA: 48.2 ng/ul   A260/280: 1,61 [-]
							
							Results upside NHX1 fragment: 17.9 ng/ul   A260/280: 1,73 [-]
							
							Purified fragments were stored at -20 Â°C.
						</UL><UL><LI><U><B>Production PCR for fragments down from p0255 plasmid and kanMX and down fragments:</B></U></LI><P>
							Production PCR was started to amplify enough of the desired fragments to perform a transormation. As template p0255 Plasmid for the downside fragment.
							As a template for the kanMX fragment purified plasmid pUC19 was used. As a template for the downside NHX1 fragment, isolated genome from NHX1 
							cells were used.
						</P></UL><UL><LI><U><B>Results of the production PCR for fragments up and down from p0255 plasmid and GAL1 from NHX1 mutants:</B></U></LI><P>
							PCR worked only for two (downside fragment from p0255 plasmid and kanMX fragment from pUC19 plasmid) out of the three fragments which has to be amplified.
							The gel showed for these two fragments clear bands, however for the other it was not possible to see anything.
							
						</P></UL><UL><LI><U><B>PCR clean-up for downside fragment from p0255 plasmid and kanMX fragment from pUC19 plasmid</B></U></LI><P>
							Two samples from the PCR were used for the purification of the downside fragment from the p0255 plasmid, while just one PCR sample could be used for the kanMX fragment.
							(protocol: <A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol2">PCR clean-up</A>)
							</P>
							Result sample 1 GAL1: DNA: 25.4 ng/ul   A260/280: 1.53 [-]
							
							Result sample 2 GAL1: DNA: 57.9 ng/ul   A260/280: 1.50 [-]
							
							Results upside NHX1 fragment: 61.1 ng/ul   A260/280: 1.58 [-]
							
							Purified fragments were stored at -20 Â°C.
						</UL><H4>07/26</H4><UL><LI><U><B>Production PCR for fragments up and down from p0255 plasmid and GAL1 from NHX1 mutants:</B></U></LI><P>
							Production PCR was started to amplify enough of the desired fragments to perform a transormation. As template p0255 Plasmid and isolated NHX1 genome were
							utilise. 
						</P></UL><UL><LI><U><B>Results of the production PCR for fragments up and down from p0255 plasmid and GAL1 from NHX1 mutants:</B></U></LI><P>
							PCR worked only for two (upside fragment and downside fragment) out of the four fragments which has to be amplified. The gel showed for these two clear bands,
							however for the others it was not possible to see anything.
							
						</P></UL><UL><LI><U><B>Production PCR for kanMX fragment and gradient PCR for upside and downside fragments from NHX1 mutants:</B></U></LI><P>
							Production PCR was started to amplify enough of the desired fragments to perform a transormation. As a template pUC19 plasmid was used.
							For the gradient PCR, isolated genome from NHX1 cells were used as a template.
						</P></UL><UL><LI><U><B>Results of the production PCR for for kanMX fragment and gradient PCR for upside and downside fragments from NHX1 mutants:</B></U></LI><P>
							The PCR worked for every fragments. Only, was the band for the KanMX fragment not clear enough. Consequently, only the upside and downside fragment were
							purified.
						</P></UL><H4>07/27</H4><UL><LI><U><B>PCR clean-up for upside and downside fragment from NHX1 mutants:</B></U></LI><P>
							Purification was performed after the <A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol2">PCR clean-up</A> protocol.
							</P>
							Result downside fragment: DNA: 30,3 ng/ul A260/280: 1,53 [-]
							
							Result upside fragment: DNA: 95,8 ng/ul A260/280: 1,55 [-]
							
							Purified fragments were stored at -20 Â°C.
						</UL><UL><LI><U><B>Production PCR for kanMX , upside and downside fragments:</B></U></LI><P>
							The Production PCR was started to amplify enough of the desired fragments to perform a transormation. As  template pUC19 plasmid and isolated
							genome from NHX1 cells were used as a template.
							
						</P></UL><UL><LI><U><B>Results of the production PCR for kanMX, upside and downside fragments::</B></U></LI><P>
							The PCR  workesfor kanMX and upside fragments, but not for the downside fragment. KanMX and upside fragments were afterwards isolated.
						</P></UL><UL><LI><U><B>PCR clean-up for upside and kanMX fragments:</B></U></LI><P>
							Purification was performed after the <A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol2">PCR clean-up</A> protocol.
							</P>
							Result kanMX fragment: DNA: 33 ng/ul A260/280: 1,85 [-]
							
							Result upside fragment: DNA: 54,3 ng/ul A260/280: 1,59 [-]
							
							Purified fragments were stored at -20 Â°C.
						</UL><UL><LI><U><B>Production PCR for GAL1 fragment from NHX1 mutants:</B></U></LI><P>
							The Production PCR was started to amplify enough of the desired fragments to perform a transormation. As a template isolated
							genome from NHX1 cells was used.
							
						</P></UL><UL><LI><U><B>Results of the production PCR for the GAL1 fragment:</B></U></LI><P>
							The PCR worked. clear bands could be observed at the right size. Next step is the fragment purification.
						</P></UL><UL><LI><U><B>PCR clean-up for the GAL1 fragment:</B></U></LI><P>
							Purification was performed after the <A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol2">PCR clean-up</A> protocol.
							</P>
							Result GAL1 fragment: DNA: 54,3 ng/ul A260/280: 1,59 [-]
							
							Purified fragments were stored at -20 Â°C.
						</UL><H3 id="week10">Week 10</H3><H4>07/30</H4><UL><LI><U><B>Genome isolation of BMA cells with PAA1 KO with Promega-kit:</B></U></LI><P>
							positive colonies 7 and 8 from the masterplate (07/24) has been picked and used for the genome isolation (protocol: <A href="">Genome Isolation after Kit</A>).
						</P></UL><UL><LI><U><B>Preparation of overnight cultures (colony 7 and 8) from BMA-cells with PAA1 KO</B></U></LI><P>
							two different overnight cultures were prepared with the chosen colonies 7 and 8. 5 ml yeast dropout medium without leucine were used. Overnight
							cultures were incubated at 30 Â°C. (protocol: <A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol6">Overnight for S. cerevisiae Cells</A>)
							
						</P></UL><H4>07/31</H4><UL><LI><U><B>Gradient PCR for downside NHX1 fragment from NHX1 mutants with Promega Mastermix:</B></U></LI><P>
							In order to produce the downside fragment from the NHX1 mutants, a PCR was needed. This time, we used the mastermix from Promega becuse previous attempts
							to amplify this fragment have failed. As a template NHX1 isolated genome was used.
						</P></UL><UL><LI><U><B>Results of the gradient PCR for the downside fragment:</B></U></LI><P>
							Bands appered for all temperatures at the right length. Afterwards, the fragment has been purified.
							
						</P></UL><UL><LI><U><B>Purification of the Downside fragment:</B></U></LI><P>
							Purification was performed after the <A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol2">PCR clean-up</A> protocol. For purification, two different PCR samples were used.
							</P>
							Result downside fragment #1: DNA: 268,1 ng/ul A260/280: 1,83 [-]
							
							Result downside fragment #2: DNA: 308,8 ng/ul A260/280: 1,86 [-]
							
							Purified fragments were stored at -20 Â°C.
						</UL><UL><LI><U><B>Overnight culture for ordered primers with genes for melatonin detection:</B></U></LI><P>
							The plasmids we ordered for our project with the gene we want to transform in yeast cells have arrived (hER, MTNR1A, RORA and beta-Arrestin).
							For further experimentations with them it was necessary to create an overnight culture in LB medium with ampicillin for each one of them.(Plasmids
							were contained in E. coli dh5-alpha cells). The next step will be the creation of back ups and plasmid isolation.
						</P></UL><H3 id="week11">Week 11</H3><H4>08/07</H4><UL><LI><U><B>Production PCR for Gal1 fragment and kanMX fragment with Promega-kit:</B></U></LI><P>
							In order to produce the GAL1 fragment from the NHX1 mutants and kanMX from the pUC19 plasmid, a PCR was needed. This time, we used the mastermix from Promega becuse previous attempts
							to amplify this fragment have not been quite successful. As a template NHX1 isolated genome was used.
						</P></UL><UL><LI><U><B>Results of the gradient PCR for the downside fragment:</B></U></LI><P>
							Bands at the expected size could be seen for both fragments. The PCR worked. PCR clean-up will be done to purify isolated primers.
							
						</P></UL><UL><LI><U><B>Purification of the Gal1 fragment and kanMX fragment:</B></U></LI><P>
							Purification was performed after the <A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol2">PCR clean-up</A> protocol. 
							</P>
							Result Gal1 fragment: DNA: 43,8 ng/ul A260/280: 1,89 [-]
							
							Result kanMX fragment: DNA: 158,4 ng/ul A260/280: 1,9 [-]
							
							Purified fragments were stored at -20 Â°C.
						</UL><UL><LI><U><B>Plasmid isolation of hER, MTNR1A, RORA and beta-Arrestin plasmids:</B></U></LI><P>
							First of all, plasmids were isolated from 
							previous overnight cultures (08/06) (protocol: <A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol1">Plasmid clean-up</A>). Only, did the purification for the RORA plasmid need to be repeated with 
							a new overnight culture because DNA concentration was too low. Other plasmids were stored at -20 Â°C.
						</P></UL><UL><LI><U><B>Gradient-PCR for beta-arrestin and hER-fragment:</B></U></LI><P>
							With new primers, we wanted to extract the beta-arrestin and hER-fragment gene from plasmids. To achieve this, it was necessar to start a gradient PCR
							to find out the most suitable annealing temperature for our genes. As a template the plasmid hER and beta-arrestin were used.
							
						</P></UL><UL><LI><U><B>Results of the gradient PCR for beta-arrestin and hER-fragment:</B></U></LI><P>
							Bands at the expected size could be seen for both genes. The PCR worked. PCR clean-up will be carry out to purify isolated primers.
						</P></UL><H4>08/08</H4><UL><LI><U><B>Production PCR for beta-arrestin, MT1 and RZR2 fragment:</B></U></LI><P>
							In order to extract the genes from the plasmids, a PCR was started. As templates MT1, RZR2 and beta-arrestin plasmids have been used.
						</P></UL><UL><LI><U><B>Results of the production PCR for beta-arrestin, MT1 and RZR2 fragment:</B></U></LI><P>
							PCR worked for both fragment.
							
						</P></UL><UL><LI><U><B>Purification of the RZR2 and MT1 fragment:</B></U></LI><P>
							Purification was performed after the <A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol2">PCR clean-up</A> protocol. 
							</P>
							Result RZR2 fragment: DNA: 221,1 ng/ul A260/280: 1,91 [-]
							
							Result MT1 fragment: DNA: 324 ng/ul A260/280: 1,89 [-]
							
							Purified fragments were stored at -20 Â°C.
						</UL><UL><LI><U><B>Gradient PCR for RZR1 fragment and hER-fragment:</B></U></LI><P>
							In order to extract the genes from the plasmids, a PCR was started. As templates RZR1 and hER plasmids have been used.
						</P></UL><UL><LI><U><B>Results of the Gradient PCR for RZR1 fragment and hER-fragment:</B></U></LI><P>
							PCR worked for both fragments. The next step is their purification.
							
						</P></UL><UL><LI><U><B>Production PCR for Gal1 fragment from NHX1 mutants:</B></U></LI><P>
							For further transformations, it was necessary to amplify the GAL1 fragment from NHX1 mutants. Due to that, a PCR was started with isolated genome from NHX1 mutants.
						</P></UL><UL><LI><U><B>Results of the production PCR for Gal1 fragment from NHX1 mutants:</B></U></LI><P>
							PCR worked. The next step is the fragment purification.
							
						</P></UL><UL><LI><U><B>Purification of the GAL1 fragment:</B></U></LI><P>
							Purification was performed after the <A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol2">PCR clean-up</A> protocol. 
							</P>
							Result GAL1 fragment: DNA: 76.5 ng/ul A260/280: 1,96 [-]
							
							
							
							Purified fragments were stored at -20 Â°C.
						</UL><H4>08/09</H4><UL><LI><U><B>Plasmid isolation of RZR2 plasmid:</B></U></LI><P>
							RZR2 was isolated from an E. coli overnight culture (08/08) which has been grown overnight in LB medium with ampicillin.(protocol: 
							<A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol1">Plasmid clean-up</A>).
						</P></UL><UL><LI><U><B>Production PCR for RZR1 and RZR2 fragments:</B></U></LI><P>
							To achieve better concentrations of RZR1 and RZR2, a new PCR has been carried out. As templates were used RZR1 plasmid and RZR2 plasmid. 
							
						</P></UL><UL><LI><U><B>Result of the production PCR for RZR1 and RZR2:</B></U></LI><P>
							Both fragments could be replicated without problem. PCR worked. 
						</P></UL><UL><LI><U><B>Purification of the RZR2 and RZR1 fragments:</B></U></LI><P>
							Purification was performed after the <A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol2">PCR clean-up</A> protocol. 
							</P>
							Result RZR1 fragment: DNA: 112 ng/ul A260/280: 1,89 [-]
							
							Result RZR2 fragment: DNA: 352,5 ng/ul A260/280: 2,05 [-]
							
							Purified fragments were stored at -20 Â°C.
						</UL><UL><LI><U><B>Creation of Agarose solution:</B></U></LI><P>
							500 ml of agarose solution have been produced. (protocol: <A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol10">Agarose Solution</A></P></UL><UL><LI><U><B>Transformation of competent BMA64-a1 cells with PAA1 KO cassette with RZR1 + hER:</B></U></LI><P>
							We tested that the produced competent BMA64-a1 cellls with the PAA KO cassette were competent consequently it was decided to try
							a transformation with the gene RZR1 and the recognition site hER (Protocol:<A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol18"> Transformation of chemical competent Yeast cells</A>).
							As a positive control competent BMA64-a1 with PAA1 KO cassette transformed with the kanMX were used. As a negative controlo WT BMA64-a1 
							cells with the PAA1 KO cassette were used. Positive control was plated on YPD plates with G418 antibiotic, while cells transformed with the 
							RZR1+hRE cassette and the negative control were plated on yeast dropout medium without Histidin. Afterwards, cells were incubated
							at 30 Â°C. 
						</P></UL><H4>08/10</H4><UL><LI><U><B>Genome isolation  from transformed BMA64-a1 cells with PAA1 KO cassette with RZR1 + hER:</B></U></LI><P>
							The genome of three different colony from the transformation plate (08/09) was isolated for verification of the transformation with RZR1 + hER.
							(protocol: <A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol19">Quick and Dirty</A>)
						</P></UL><UL><LI><U><B>Verification PCR for transformed BMA64-a1 cells with PAA1 KO cassette with RZR1 + hER:</B></U></LI><P>
							PCR was ran to test the success of the transformation of our BMA64-a1 cells with PAA1 KO cassette and RZR1 + hER. As a template 
							the isolated genome of BMA64-a1 cells with PAA1 KO cassette and RZR1 + hER was used.
							
						</P></UL><UL><LI><U><B>Result of the verification PCR for BMA64-a1 cells with PAA1 KO cassette with RZR1 + hER:</B></U></LI><P>
							For all three genomes, it was not possible to see any bands. Consequently, we can say that the analysed colonies have not been correctly transformed. 
						</P></UL><UL><LI><U><B>Production of histidine drop out plates:</B></U></LI><P>
							500 ml yeast dropout medium without histidine have been produced. (protocol: <A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol27">Yeast dropout medium without histidine</A>) 
						</P></UL><H4>08/11</H4><UL><LI><U><B>Genome isolation  from transformed BMA64-a1 cells with PAA1 KO cassette with RZR1 + hER:</B></U></LI><P>
							The genome single colony from the transformation plate (08/09) was isolated for verification of the transformation with RZR1 + hER.
							(protocol: <A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol19">Quick and Dirty</A>)
						</P></UL><UL><LI><U><B>Verification PCR for transformed BMA64-a1 cells with PAA1 KO cassette with RZR1 + hER:</B></U></LI><P>
							PCR was ran to test the success of the transformation of our BMA64-a1 cells with PAA1 KO cassette and RZR1 + hER. As a template 
							the isolated genome of BMA64-a1 cells with PAA1 KO cassette and RZR1 + hER was used.
							
						</P></UL><UL><LI><U><B>Result of the verification PCR for BMA64-a1 cells with PAA1 KO cassette with RZR1 + hER:</B></U></LI><P>
							For the analysed genome, it was not possible to see any bands. Consequently, we can say that the analysed colony has not been correctly transformed. 
						</P></UL><UL><LI><U><B>Creation of backups in E. coli dh5-alpha for the IDT plasmids:</B></U></LI><P>
							Competent E. coli dh5-alpha were utilise to create backups of the obtained IDT plasmids. Plasmids were transformed in competent E. coli cells
							(protocol: <A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol13">transformation of chemical competent E. coli cells</A>) and plated on LB medium plates with ampicillin as an antibiotic for
							selection. Plates were incubated at 37 Â°C.
						</P></UL><H3 id="week12">Week 12</H3><H4>08/15</H4><UL><LI><U><B>Creation of Yeast dropout medium without leucine, uracyle and histidine:</B></U></LI><P>
							800 ml Yeast dropout medium without leucine, uracyle and histidine has been created. (protocol: <A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol27">Yeast Dropout Medium</A>) Such a medium was
							needed because our final mutants will have three different selectable markers from the three diffent modifcations we plan to carry out.
						</P></UL><UL><LI><U><B>Overnight culture for BY4742 cells with PAA1 KO cassette:</B></U></LI><P>
							Due to shortage of competent BY4247 cells with PAA1 KO cassette, it was necessary to prepare an overnight culture (5 ml) with yeast dropout medium 
							without leucine to create competent cells. Cells were incubated at 28 Â°C.
						</P></UL><UL><LI><U><B>Overnight culture for RZR2 plasmid isolation:</B></U></LI><P>
							5 ml of LB medium with ampicillin were used to create an overnight culture from the E. coli cells which contains the RZR2 plasmid.
							They were incubated at 37 Â°C.
						</P></UL><H4>08/16</H4><UL><LI><U><B>Creation of Competent BY4742cells with KO cassete: </B></U></LI><P>
							10 tubes 100 ul each of competent cells were produced. (protocol: <A href="">Kit for CompÃ¼etent Yeast Cells</A>) Tubes were afterwards stored at 
							-80 Â°C.
						</P></UL><H4>08/17</H4><UL><LI><U><B>transformation of competent BMA64-a1 cells with PAA1 KO cassette wiith RZR1+hER: </B></U></LI><P>
							For the last step of the creation of yeast cells able to detect melatonin, it was needed to transform the already competent BMA64-a1 cells 
							with PAA1 KO cassette with the transcription factor RZR1 and its corresponding recognition sequence hER. As a control for the success of the transformtaion 
							kanMX plasmid was used. (protocol: <A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol20">Transformation after Kit Protocol</A>) For selection, plates without histidine were used, while for the 
							positive control plates with G418 antibiotic were used. Plates were incubated at 30 Â°C after transformation.
						</P></UL><UL><LI><U><B>Gradient PCR for kanMX fragment and small fragment for beta-arrestine (3EA) transformation:</B></U></LI><P>
							Gradient PCR was started to produce new kanMX fragments which we were short of and find out a suitable annealing temperature for the 3EA fragment.
							 As templates, kanMX plasmid and 3EA plasmid were used.
						</P></UL><UL><LI><U><B>Gradient PCR for GAL1 fragment and TCYC fragment from NHX1 mutants:</B></U></LI><P>
							Gradient PCR was started to produce new GAL1 fragments and TCYC fregments from NHX1 mutants. As a template, isolated genome from NHX1 cells was
							used.
						</P></UL><H3 id="week13">Week 13</H3><H4>08/20</H4><UL><LI><U><B>Results of the previous gradient PCRs (08/17): </B></U></LI><P>
							Gel electrophoresis only showed good results for kanMX, GAL1 and 3EA fragments. We were not able to see any band for the terminator TCYC.
							afterwards, Amplified fragments have been purified. (Concentration were not noted)
						</P></UL><UL><LI><U><B>Preparation of agarose solution:</B></U></LI><P>
							500 ml of agarose solution has been produced. (protocol: <A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol10">Agarose Solution</A>)
						</P></UL><H4>08/21</H4><UL><LI><U><B>Gradient PCR for TCYC fragment from NHX1 mutants:</B></U></LI><P>
							Gradient PCR was started to produce new TCYC fragments from NHX1 mutants. As a template, isolated genome from NHX1 cells was
							used.
						</P></UL><UL><LI><U><B>Result of the gradient PCR:</B></U></LI><P>
							No bands could be seen on the gel. PCR did not work. We will try a PCR with different annealing temperatures and template concentration.
						</P></UL><H4>08/22</H4><UL><LI><U><B>Gradient PCR for TCYC fragment from NHX1 mutants and kanMX fragment:</B></U></LI><P>
							Gradient PCR was started to produce new TCYC fragments from NHX1 mutants and kanMX fragments. As templates, isolated genome from NHX1 cells 
							and isolated kanMX plasmid were used. Due to some recent problems we were
							facing in amplifing our fragments, we decided to use the PCR mastermix from Promega. 
						</P></UL><UL><LI><U><B>Result of the gradient PCR for TCYC fragment from NHX1 mutants and kanMX fragment:</B></U></LI><P>
							It was possible to observe bands of the expected length for both fragments at different annealing temperatures. Consequently, it was possible to
							confirm the success of the gradient PCR.  
						</P></UL><UL><LI><U><B>Purification of TCYC fragments from NHX1 mutants and kanMX fragments:</B></U></LI><P>
							Both fragments were purified and stored at -20 Â°C (protocol: <A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol2">PCR clean-up</A>).
							</P>
							Result for TCYC fragment: DNA: 255,2 ng/ul A260/280: 1,86 [-]
							
							Result for kanMX fragment: DNA: 22,9 ng/ul A260/280: 2,11 [-]
							</UL><H4>08/23</H4><UL><LI><U><B>Gradient PCR for TCYC fragment from NHX1 mutants:</B></U></LI><P>
							Gradient PCR was started to produce new TCYC fragments. As a template, isolated fragments from the previous PCR were used (08/22).							Due to some recent problems we were
							Due to problems we were facing in amplifing our fragments, we decided to use the PCR mastermix from Promega. 
						</P></UL><UL><LI><U><B>Result of the gradient PCR for TCYC fragment from NHX1 mutants and kanMX fragment:</B></U></LI><P>
							It was possible to observe bands of the expected length for the TCYC fragment at different annealing temperatures. Consequently, it was possible to
							confirm the success of the gradient PCR.  
						</P></UL><UL><LI><U><B>Purification of TCYC fragments from NHX1 mutants and kanMX fragments:</B></U></LI><P>
							Amplified fragments were purified and stored at -20 Â°C (protocol: <A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol2">PCR clean-up</A>).
							</P>
							Result for TCYC fragment: DNA: 30,1 ng/ul A260/280: 1,7 [-]
							</UL><UL><LI><U><B>Gradient PCR for beta-arrestin, Gal-x4 and EA3 fragments:</B></U></LI><P>
							Gradient PCR was started to produce new  beta-arrestin, Gal-x4 and EA3 fragments. 
							As templates, isolated NHX1 genome for Gal-x4, beta-arrestin plasmid and EA3 plasmid were used.We decided to use the PCR mastermix from Promega for
							this PCR. 
						</P></UL><UL><LI><U><B>Result of the Gradient PCR for beta-arrestin, Gal-x4 and EA3 fragments:</B></U></LI><P>
							It was possible to observe bands of the expected length for every amplified fragments. however, due to the bad quality of the gel and the presence of 
							unexpected bands, fragments could not be purified and utilised for further experiments.
						</P></UL><H4>08/24</H4><UL><LI><U><B>Gradient PCR for 3EA and beta-arrestin:</B></U></LI><P>
							Gradient PCR was started to produce new 3EA and beta-arrestin fragments. As templates, the 3EA plasmid and the beta-arrestin plasmid were used. 
							In this case, new PCR for the amplification of beta-arrestin were used. As well, Promega mastermix was utilise for this PCR.
						</P></UL><UL><LI><U><B>Result of the gradient PCR for 3EA and beta-arrestin:</B></U></LI><P>
							It was possible to observe bands of the expected length for tboth fragments at different annealing temperatures. Consequently, it was possible to
							confirm the success of the gradient PCR.  
						</P></UL><UL><LI><U><B>Purification of 3EA and beta-arrestin fragments:</B></U></LI><P>
							Amplified fragments were purified and stored at -20 Â°C (protocol: <A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol2">PCR clean-up</A>).
							</P>
							Result for 3EA fragment: DNA: 53,9 ng/ul A260/280: 1,9 [-]
							
							Result for beta-arrestin fragment: DNA: 467,2 ng/ul A260/280: 1,9 [-]
							</UL><UL><LI><U><B> Production LB solid medium:</B></U></LI><P>
							500 ml of LB solid medium were produced. (protocol:<A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol22">LB solid medium</A>)
						</P></UL><UL><LI><U><B>Gradient PCR for big fragment for beta-arrestin approach (2EA):</B></U></LI><P>
							Gradient PCR was started to produce 2EA fragments. As a templates, the 2EA plasmid was used. Promega mastermix was utilise for this PCR.
						</P></UL><UL><LI><U><B>Result of the gradient PCR for 2EA fragment:</B></U></LI><P>
							It was possible to observe bands of the expected length for the 2EA fragments at different annealing temperatures. The next step is the fragment purification.
							
						</P></UL><H3 id="week14">Week 14</H3><H4>08/27</H4><UL><LI><U><B>Purification of 3EA and beta-arrestin fragments:</B></U></LI><P>
							Amplified fragments were purified and stored at -20 Â°C (protocol: <A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol2">PCR clean-up</A>).
							</P>
							Result for 2EA fragment: DNA: 155,5 ng/ul A260/280: 1,84 [-]
							</UL><UL><LI><U><B>Gradient PCR for Gal-x4 fragment:</B></U></LI><P>
							Gradient PCR was started to produce Gal-x4 fragments. As a template, the isolated genome from NHX1 mutants was used. Promega mastermix was utilise for this PCR.
						</P></UL><UL><LI><U><B>Result of the gradient PCR for gal-x4 fragment:</B></U></LI><P>
							It was possible to observe bands of the expected length for the gal-x4 fragments at different annealing temperatures. The next step is the fragment purification.
							
						</P></UL><UL><LI><U><B>Test transformation of competent BY4742 cells  with PAA1 KO cassette:</B></U></LI><P>
							Competence of competent BY4742 cells with PAA1 KO cassette was tested with a transformation with the plasmid kanMX.(protocol: <A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol20">Transformation after
							kit protocol</A>) Afterwards, cells were plated on YPD plates with G418 antibiotic and incubated at 30 Â°C.
							
						</P></UL><UL><LI><U><B>Plasmid isolation of p0255 plasmid:</B></U></LI><P>
							Plasmid p0255 was isolated from an E. coli overnight culture with kanamycin as an antibiotic. (<A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol20">Plasmid Purification Kit</A>)
							</P>
							Result: DNA: 441,9 ng/ul A260/280: 1.88 [-]
							
							Tube was stored at -20 Â°C.
							
						</UL><H4>08/28</H4><UL><LI><U><B>PCR clean-up for gal-x4 fragment(08/27):</B></U></LI><P>
							Fragments from the gradient PCR for the production of gal-x4 fragments (08/27) were purified. (<A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol1">Plasmid Purification Kit</A>)
							</P>
							Result: DNA: 22.4 ng/ul A260/280: 1.75 [-]
							
							Tube was stored at -20 Â°C.
							
						</UL><H4>08/29</H4><UL><LI><U><B>Creation of a joined cassette for MT1, RZR1 and RZR2 genes:</B></U></LI><P>
							Due to some difficulties in transforming competent yeast cells with our fragments after the usual protocol, we tryed a new approach.
							We used for the creation of these two different cassettes the NEB hifi DNA asembly kit.
						</P></UL><UL><LI><U><B>Amplification of joined cassettes for  MT1, RZR1 and RZR2:</B></U></LI><P>
							PCR was needed to amplify the new produced cassette with the NEB hifi assembly kit. As templates, the three joined cassettes were used.
							Promega mastermix was used for this PCR.
							
						</P></UL><UL><LI><U><B>Result of the amplification PCR for  MT1, RZR1 and RZR2:</B></U></LI><P>
							Gel electrophoresis showed good results for RZR2 and MT1 . No bands could be seen for RZR1 cassette. Amplified cassettes for
							RZR2 and MT1 can now be purified.
							
						</P></UL><UL><LI><U><B>PCR clean-up for RZR2, MT1 and beta-arrestin:</B></U></LI><P>
							Fragments from the amplification PCR of joined cassettes for  MT1 and RZR1 were purified. (<A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol1">Plasmid Purification Kit</A>)
							</P>
							Result for MT1: DNA: 306.3 ng/ul A260/280: under 2 [-]
							
							Result for RZR1: DNA: 443.6 ng/ul A260/280: under 2 [-]
							
							Tubes were stored at -20 Â°C.
							
						</UL><UL><LI><U><B>transformation of competent BMA64-a1 cells with PAA1 KO casette with RZR1 and transformation of competent BY4742 cells with PAA1 KO cassette with MT1:</B></U></LI><P>
							For the last step of the creation of yeast cells able to detect melatonin, it was needed to transform the already competent BMA64-a1 cells with 
							PAA1 KO cassette with the transcription factor RZR1 as well as competent BY4742 cells with PAA1 KO cassette with the melatonine receptor MT1.
							(protocol: <A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol20">Transformation after Kit Protocol</A>) isolated kanMX plasmid was used for the positive control transformation.
							For selection, plates without histidine were used, 
							while for the positive control plates with G418 antibiotic were used. Plates were incubated at 30 ÃÂ°C after transformation. 							
						</P></UL><H4>08/30</H4><UL><LI><U><B>Amplification of the joined cassette for  RZR1:</B></U></LI><P>
							PCR was needed to amplify the new produced RZR1 cassette with the NEB hifi assembly kit. As a template, the  joined cassette for RZR1 was used.
							Promega mastermix was used for this PCR.
						</P></UL><UL><LI><U><B>Result of the amplification PCR for  MT1, RZR1 and RZR2:</B></U></LI><P>
							Gel electrophoresis showed good results for RZR1 . Amplified cassette for
							RZR1 can now be purified.
							
						</P></UL><UL><LI><U><B>PCR clean-up for RZR1 cassette:</B></U></LI><P>
							Cassette from the amplification PCR was purified. (<A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol1">Plasmid Purification Kit</A>)
							</P>
							Result for MT1: DNA: 135.7 ng/ul A260/280: under 1.83 [-]
							
							Tube were stored at -20 Â°C.
							
						</UL><H4>08/31</H4><UL><LI><U><B>Amplification of the joined cassette for beta-Galactosidase (with 2EA and 3EA):</B></U></LI><P>
							PCR was needed to amplify the new MT1 cassette produced with the NEB hifi assembly kit. As a template, the  joined cassette for MT1 was used.
							Promega mastermix was used for this PCR.
						</P></UL><UL><LI><U><B>Result of the amplification PCR for  MT1, RZR1 and RZR2:</B></U></LI><P>
							Gel electrophoresis was so bad that the fragments had to be discarded.
							
						</P></UL><UL><LI><U><B>Transformation of competent BY4742 cells with PAA1 KO cassette with EA2 and EA3 fragments:</B></U></LI><P>
							To create yeast cells able to detect melatonin, it was needed to transform the already competent
							BY4742-a1 cells with PAA1 KO cassette with the melatonin receptor MT1.
							(protocol: <A href="">Transformation after Kit Protocol</A>) isolated kanMX plasmid was used for the positive
							control transformation. For selection, plates without histidine were used, while for the positive control plates with G418 
							antibiotic were used. Plates were incubated at 30 Â°C after transformation. 
						</P></UL><H3 id="week15">Week 15</H3><H4>09/03</H4><UL><LI><U><B>Genome isolation of positive BY4742 colonies with PAA1 KO cassette and RZR2 gene:</B></U></LI><P>
							On the selection plates for the transformation of BY4742 cells with PAA1 KO cassette with RZR2 gene, it was possible to see some
							cell growth. Consequently, the genome was isolated after <A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol20">quick and dirty</A> protocol. Genomes of ten different colonies were isolated.
						</P></UL><UL><LI><U><B>Verification PCR for BY4742 colonies with PAA1 KO cassette and RZR2 gene:</B></U></LI><P>
							PCR was started to test the correct integration of the cassette for the RZR2 gene in the genome. As templates, the isolated genomes from the 
							positive colonies of BY4742 cells with PAA1 KO cassette and RZR2 gene were used.
							
						</P></UL><UL><LI><U><B>Results of the verification PCR for BY4742 colonies with PAA1 KO cassette and RZR2 gene :</B></U></LI><P>
							Bands were only seen  for colony 2 and 5. It is enough to confirm the success of the transformation at leat for two colonies.
						</P></UL><H4>09/04</H4><UL><LI><U><B>Transformation of E. coli chemical competent cells with pSB1C3 (iGEM-plasmid) and with pH6HTN His6HaloTag (Promega) (protocol: <A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol12"> TSS-protocol</A>):</B></U></LI><P>
							For the realisation of our biobrick, it was necessary to transform pSB1C3 and pH6HTN His6HaloTag in E. coli in order to have a backup and the
							necessary quantity for the creation of biobricks.
						</P></UL><H4>09/05</H4><UL><LI><U><B>Overnight culture of E. coli with pH6HTN (promega) plasmid (protocol: <A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol5"> E. coli overnight culture</A>):</B></U></LI><P>
							The transformation of dh5-alpha E. coli with pH6HTN plasmid (09/04) showed positive results. Consequently, an overnight culture was prepare
							to isolate the plasmid from the E. coli cells. 5 ml overnight culture were prepared with LB liquid medium with ampicillin as an antibiotic.
						</P></UL><UL><LI><U><B>Overnight culture of BY4742 cells with PAA1 KO cassette and RZR2 gene(protocol: <A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol6"> S. cerevisiae overnight culture</A>):</B></U></LI><P>
					        An overnight culture was prepare. 5 ml overnight culture were prepared with yeast dropout medium without leucine and histidine. 
							(protocol: <A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol27">yeast dropout medium</A>)
						</P></UL><UL><LI><U><B>Overnight of E. coli with pH6HTN (promega) plasmid (protocol: <A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol5"> E. coli overnight culture</A>):</B></U></LI><P>
							The transformation of dh5-alpha E. coli with pH6HTN plasmid (09/04) showed positive results. Consequently, an overnight culture was prepare
							to isolate the plasmid from the E. coli cells. 5 ml overnight culture were prepared with LB liquid medium (protocol: <A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol21"> LB medium With Antibiotic</A>)
							with ampicillin as an antibiotic.
						</P></UL><UL><LI><U><B>Transformation of E. coli chemical competent cells with pSB1C3 (protocol: <A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol13"> TSS-protocol</A>):</B></U></LI><P>
							For the realisation of our biobrick, it was necessary to transform pSB1C3 in E. coli in order to have a backup and the
							necessary quantity for the creation of biobricks. Due to the failure of the previous transformation (09/04), it was necessary
							to repeat the experiment.
						</P></UL><H4>09/07</H4><UL><LI><U><B>Overnight culture of E. coli with pH6HTN (promega) plasmid (protocol: <A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol5"> E. coli overnight culture</A>):</B></U></LI><P>
							The transformation of dh5-alpha E. coli with pH6HTN plasmid (09/04) showed positive results. Consequently, an overnight culture was prepare
							to isolate the plasmid from the E. coli cells. 5 ml overnight culture were prepared with LB liquid medium with ampicillin as an antibiotic.
						</P></UL><UL><LI><U><B>PCR for RZR2 for amplification as an insert for promega vector:</B></U></LI><P>
					       PCR was started to produce RZR2 fragments which were needed for the creation of a plasmid which should enables our modified cells to
						   detect melatonin in saliva. (cells: BY4742 with PAA1 KO cassette) As a template, RZR2 plasmid was used.
						</P></UL><UL><LI><U><B>Gel electrophoresis for PCR for RZR2 for amplification as an insert for promega vector:</B></U></LI><P>
							PCR showed good results, however gel extraction was necessary becase too many bands of different lenghts could be observed on the gene.
						</P></UL><UL><LI><U><B>Genome isolation of positive transformed BY4742 cells with PAA1 KO cassette and RZR2 gene:</B></U></LI><P>
							Genome of two different colonies was isolated (colony 2 and 5 09/03) and stored at 4 Â°C. (protocol: <A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol19">Quick and Dirty</A>).
							Afterwards, DNA concentration was measured.
							
							Result colony 2: DNA: 429 ng/ul   A260/280: 2.14 [-]
							
							Result colony 5: DNA: 374.6 ng/ul   A260/280: 2.13 [-]
							</P></UL><H3 id="week16">Week 16</H3><H4>09/10</H4><UL><LI><U><B>Plasmid Isolation of pBS1C3 plasmid:</B></U></LI><P>
							previously prepared overnight culture (09/07) was used for the isolation of pBS1C3 plasmid. (protocol: <A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol2">Plasmid clean-up</A>)
							Two separeted isolation with 2,5 ml each of the overnight culture were done. 
							
							Result #1: DNA: 219.7 ng/ul   A260/280: 1.86 [-]
							
							Result #2: DNA: 215.5 ng/ul   A260/280: 1.88 [-]
							</P></UL><UL><LI><U><B>Gel extraction and clean-up of RZR2 (09/07):</B></U></LI><P>
					       two bands of the right length obtained at different annealing temperatures has been chosen for gel extraction.
						   Gel extraction was performed after <A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol3">gel extraction protocol</A>. Afterwards, DNA was purified and stored at -20 Â°C.
						   
							Result #1: DNA: 17.2 ng/ul   A260/280: 2.1 [-]
						   
							Result #2: DNA: 113.1 ng/ul   A260/280: 1.93 [-]
						   </P></UL><UL><LI><U><B>Creation of YPD Medium with Galactose:</B></U></LI><P>
							PCR showed good results, however gel extraction was necessary becase too many bands of different lenghts could be observed on the gene.
						</P></UL><UL><LI><U><B>Genome isolation of positive transformed BY4742 cells with PAA1 KO cassette and RZR2 gene:</B></U></LI><P>
							750 ml of YPD medium with galactose instead of glucose were created to induce gene expression in the cell.
							(protocol: <A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol28">YPD Medium with Galactose</A>)
						</P></UL><UL><LI><U><B>Overnight Culture for transformed BMA64-a1 cells with PAA1 KO cassette and MT1 receptor (09/03):</B></U></LI><P>
							positive colony 2 and 5 (09/03) were inoculated in 5 ml yeast dropout medium each and icubated at 30 Â°C. (protocol: <A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol6">S. cerevisiae overnight</A>)
						</P></UL><H4>09/11</H4><UL><LI><U><B>Plasmid Isolation of pBS1C3 plasmid:</B></U></LI><P>
							previously prepared overnight culture (09/07) was used for the isolation of pBS1C3 plasmid. (protocol: <A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol1">Plasmid clean-up</A>)
							Two separeted isolation with 2,5 ml each of the overnight culture were done. 
							
							Result #1: DNA: 219.7 ng/ul   A260/280: 1.86 [-]
							
							Result #2: DNA: 215.5 ng/ul   A260/280: 1.88 [-]
							</P></UL><UL><LI><U><B>Gel extraction and clean-up of RZR2 (09/07):</B></U></LI><P>
					       two bands of the right length obtained at different annealing temperatures has been chosen for gel extraction.
						   Gel extraction was performed after <A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol3">gel extraction protocol</A>. Afterwards, DNA was purified and stored at -20 Â°C.
						   
							Result #1: DNA: 17.2 ng/ul   A260/280: 2.1 [-]
						   
							Result #2: DNA: 113.1 ng/ul   A260/280: 1.93 [-]
						   </P></UL><UL><LI><U><B>Creation of YPD Medium with Galactose:</B></U></LI><P>
							PCR showed good results, however gel extraction was necessary becase too many bands of different lenghts could be observed on the gene.
						</P></UL><UL><LI><U><B>Overnight Culture for transformed BMA64-a1 cells with PAA1 KO cassette and MT1 receptor (09/03):</B></U></LI><P>
							positive colony 2 and 5 (09/03) were inoculated in 5 ml yeast dropout medium each and icubated at 30 Â°C. (protocol: <A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol6">S. cerevisiae overnight</A>)
						</P></UL><H4>09/12</H4><UL><LI><U><B>Production of competent BMA64-a1 cells with PAA1 KO cassette and MT1 gene:</B></U></LI><P>
							Cells from the overnight (09/10) were used to be maden competent. (protocol: <A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol20">Competent Yeast Cells after Kit</A>)
							 10 tubes were produced (100 ul each) and stored at -80 Â°C.
						</P></UL><H4>09/13</H4><UL><LI><U><B>Yeast dropout medium without histidine (solid):</B></U></LI><P>
							500 ml yeast dropout medium without histidine were produced. (<A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol27">Yeast Dropout Medium Without Histidine (solid)</A>) About 20 plates could be 
							obtained.
						</P></UL><UL><LI><U><B>Plasmid Isolation of p0258 from dh5-alpha E. coli cells:</B></U></LI><P>
					       Previously prepared overnight culture (09/11) was used for the isolation of p0258 plasmid. (protocol: <A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol1">Plasmid clean-up</A>)
						   
							Result #1: DNA: 23.1 ng/ul   A260/280: 1.75 [-]
						   </P></UL><UL><LI><U><B>PCR for X4-down fragment from p0258 for BioBricks:</B></U></LI><P>
							PCR with new designed primers was started to amplify the X4-down fragment. This fragment is needed for the creation of biobricks. As a templae,
							the plasmid p0258 was used.
						</P></UL><H4>09/14</H4><UL><LI><U><B>Restriction and Ligation of Gal-RZR2 fragment and pSB1C3 plasmid for biobricks:</B></U></LI><P><OL><LI>restirction of plasmid:</LI></OL>
							Mix was incubated at 37Â°C for 4h.
						</P><P><OL><LI>restriction of insert:</LI></OL>
							Mix was incubated at 37Â°C for 2 h. Clean-up of the fragments done afterwards with 2x10 uL TE-Buffer for RZR2 and 25 uL TE-Buffer for pSB1C3.
							Concentration measured using IMPLEN NanoPhotometer:
							
							Result pSB1C3: 57.9 ng/uL
							
							Result RZR1: DNA: 31.45 ng/uL
							</P><P><OL><LI>Ligation:</LI></OL>
							Ligation was done twice: 1x with the same ammount of vector and insert (1:1) and 1x with three times the ammount of insert (1:3).
							Incubation overnight at 16Â°C
						</P></UL><UL><LI><U><B>Transformation of E. coli dh5-alpha cells with pSB1C3 ligated with Gal-RZR2 fragment:</B></U></LI><P>
							Competent E. coli dh5-alpha cells were transformed with the ligated Gal-RZR2 fragment (09/13). (protocol: <A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol13">Transformation of chemical competent
							E. coli cells</A>). Transformed cells were plated on LB plates with chloramphenicol.
						</P></UL><H4>09/15</H4><UL><LI><U><B>Results of the Transformation of E. coli dh5-alpha cells with pSB1C3 ligated with Gal-RZR2 fragment (09/14):</B></U></LI><P>
							Transformation was not successful because no colonies could be seen on the transformation plate.
						</P></UL><UL><LI><U><B>Production PCR for RZR2 fragment:</B></U></LI><P>
							Due to the bad results of the ligation, it was necessary to amplify new RZR2 fragments for biobrick creation. As a template, the plasmid for RZR2
							was used.
						</P></UL><UL><LI><U><B>Results of the production PCR for RZR2 fragment:</B></U></LI><P>
							It was possible to observe strong bands at the right length of 1500 bp for our PCR approach. The PCR worked. Next step will be the 
							fragment purification.
						</P></UL><UL><LI><U><B>PCR clean-up for RZR2 fragment:</B></U></LI><P>
							Amplified RZR2 amplified fragments has been purified with a <A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol2">PCR clean-up kit</A>. Afterwards, DNA concentration was measured.
							
							Result: DNA: 273.6 ng/ul  A260/280: 1.9 [-]
							</P></UL><UL><LI><U><B>Second Restriction and Ligation of Gal-RZR2 fragment and pSB1C3 plasmid for biobricks:</B></U></LI><P><OL><LI>restirction of plasmid:</LI></OL>
							Mix was incubated at 37Â°C for 4h.
						</P><P><OL><LI>restriction of insert:</LI></OL>
							Mix was incubated at 37Â°C for 2 h. Clean-up of the fragments done afterwards with 2x10 uL TE-Buffer for RZR2 and 25 uL TE-Buffer for pSB1C3.
							Concentration measured using IMPLEN NanoPhotometer:
							
							Result pSB1C3: 57.9 ng/uL
							
							Result RZR1: DNA: 31.45 ng/uL
							</P><P><OL><LI>Ligation:</LI></OL>
							Ligation was done twice: 1x with the same ammount of vector and insert (1:1) and 1x with three times the ammount of insert (1:3).
							Incubation overnight at 16Â°C
						</P></UL><H3 id="week17">Week 17</H3><H4>09/18</H4><UL><LI><U><B>Transformation of E. coli dh5-alpha cells with pSB1C3 ligated with Gal-RZR2 fragment:</B></U></LI><P>
							Competent E. coli dh5-alpha cells were transformed with the ligated Gal-RZR2 fragment (09/17). (protocol: <A href="">Transformation of chemical competent
							E. coli cells</A>). Transformed cells were plated on LB plates with chloramphenicol.
						</P></UL><UL><LI><U><B>Third Restriction and Ligation of Gal-RZR2 fragment and pSB1C3 plasmid for biobricks:</B></U></LI><P><OL><LI>restirction of plasmid:</LI></OL>
							Mix was incubated at 37Â°C for 4h.
						</P><P><OL><LI>restriction of insert:</LI></OL>
							Mix was incubated at 37Â°C for 2 h. Clean-up of the fragments done afterwards with 2x10 uL TE-Buffer for RZR2 and 25 uL TE-Buffer for pSB1C3.
							Concentration measured using IMPLEN NanoPhotometer:
							
							Result pSB1C3: 57.9 ng/uL
							
							Result RZR1: DNA: 31.45 ng/uL
							</P><P><OL><LI>Ligation:</LI></OL>
							Ligation was done twice: 1x with the same ammount of vector and insert (1:1) and 1x with three times the ammount of insert (1:3).
							Incubation overnight at 16Â°C
						</P></UL><UL><LI><U><B>Overnight of BMA64-a1 cells with MT1 and PAA1 KO cassette:</B></U></LI><P>
							5 ml yeast dropout medium without histidine were used to create an overnight culture of BMA64-a1 cells with MT1 and PAA1 KO cassette.
							(protocol<A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol6">Overnight culture S. cerevisiae</A>)
						</P></UL><H4>09/19</H4><UL><LI><U><B>Overnight culture of E.coli dh5-alpha+pSB1C3+Gal-RZR2 for BioBricks:</B></U></LI><P>
							Transformation with pSB1C3+Gal-RZR2 plasmid in E.coli dh5-alpha worked. Consequently, two overnight cultures of two positive colonies were prepared.
							(protocol: <A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol5">E. coli Overnight Culture</A>) Tubes were incubated overnight at 30 Â°C.
						</P></UL><UL><LI><U><B>Overnight culture of E.coli dh5-alpha+pSB1C3+Gal-RZR2 for BioBricks:</B></U></LI><P>
							Transformation with pSB1C3+Gal-RZR2 plasmid in E.coli dh5-alpha worked. Consequently, two overnight cultures of two positive colonies were prepared.
							(protocol: <A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol5">E. coli Overnight Culture</A>) Tubes were incubated overnight at 30 Â°C.
						</P></UL><UL><LI><U><B> third transformation of E. coli dh5-alpha cells with pSB1C3 ligated with Gal-RZR2 fragment:</B></U></LI><P>
							Competent E. coli dh5-alpha cells were transformed with the ligated Gal-RZR2 fragment (09/18). (protocol: <A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol13">Transformation of chemical competent
							E. coli cells</A>). Transformed cells were plated on LB plates with chloramphenicol.
						</P></UL><UL><LI><U><B>Production PCR for beta-Arrestin:</B></U></LI><P>
							Beta-arrestine fragments were needed to complete the complete the creation of our second yeast strain able to detect melatonin in saliva.
							Consequently, a production PCR with beta-arrestine plasmid as a template has been started. 
						</P></UL><UL><LI><U><B>Result of the Production PCR for beta-Arrestin:</B></U></LI><P>
							 Clear and good band at 2250bp could be seen. The next step will be purification of the fragment by gel extraction.
						</P></UL><UL><LI><U><B>Gel extraction for beta-Arrestin:</B></U></LI><P>
							 Gel extraction was performed for the amplified fragment. (protocol: <A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol3">Gel Extraction</A>) Afterwards, DNA concentration was measured.
							 
							 Result: DNA: 23.3  ng/ul  A260/280: 2.5 [-]
						</P></UL><H4>09/20</H4><UL><LI><U><B>Overnight culture of E.coli dh5-alpha+pSB1C3+Gal-RZR2 for BioBricks:</B></U></LI><P>
							Transformation with pSB1C3+Gal-RZR2 plasmid in E.coli dh5-alpha worked. Consequently, two overnight cultures of two positive colonies were prepared.
							(protocol: <A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol5">E. coli Overnight Culture</A>) Tubes were incubated overnight at 37 Â°C.
						</P></UL><UL><LI><U><B> third transformation of E. coli dh5-alpha cells with pSB1C3 ligated with Gal-RZR2 fragment:</B></U></LI><P>
							Competent E. coli dh5-alpha cells were transformed with the ligated Gal-RZR2 fragment (09/18). (protocol: <A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol13">Transformation of chemical competent
							E. coli cells</A>). Transformed cells were plated on LB plates with chloramphenicol and incubated at 30 Â°C.
						</P></UL><H3 id="week18">Week 18</H3><H4>09/24</H4><UL><LI><U><B>genome-isolation for BMA64-a1 cells with PAA1 KO cassette, MT1 gene and beta arrestine:</B></U></LI><P>
							We were able to obsereve the growth of some yeast colonies on the transformation plate (09/20). As a result, we decided to isolate the genom eof
							ten colonies to check the correct genome integration of the MT1 cassette. (protocol: <A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol19"> Quick and Dirty</A> )
						</P></UL><UL><LI><U><B>Verification PCR for BMA64-a1 cells with PAA1 KO cassette, MT1 gene and beta arrestine:</B></U></LI><P>
							Verification PCR was performed to check the success of the transformation (09/20). As a template, fresh isolated genome of ten transformed colonies was used.
							
						</P></UL><UL><LI><U><B>Gel-electrophoresis for verification of MT1 with beta-arrestin:</B></U></LI><P>
							Gel electrophoresis did not show good results. Only, could bands be observed which had the wrong length. Check PCR did  not worked.
						</P></UL><H4>09/25</H4><UL><LI><U><B>Plasmid isolation for BioBricks from E.coli (09/20):</B></U></LI><P>
							We were able to obsereve the growth of some E. coli colonies with the plasmid pSB1C3+Gal-RZR2 on the transformation plate (09/20). As a 
							result, we decided to isolate the plasmids of
							ten colonies to check the pSB1C3+Gal-RZR2 plasmid. (protocol: <A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol1"> Plasmid clean-up</A> )
						</P></UL><UL><LI><U><B>Verification PCR for BioBricks from E.coli:</B></U></LI><P>
							Verification PCR was performed to check the success of the transformation (09/20) for the biobricks. As a template, fresh isolated plasmids
							of ten transformed E. coli dh5-alpha colonies were used.
							
						</P></UL><UL><LI><U><B>Gel-electrophoresis for verification of pSB1C3+Gal-RZR2 plasmid:</B></U></LI><P>
							Gel electrophoresis showed good results. It was possible to detect bands of the right length for eight colonies out of ten. 
						</P></UL><UL><LI><U><B>Overnight Culture for the positive transformed e. coli colonies with pSB1C3+Gal-RZR2 Plasmid:</B></U></LI><P>
							Transformation with pSB1C3+Gal-RZR2 plasmid in E.coli dh5-alpha worked. Consequently, two overnight cultures of two
							positive colonies were prepared. (protocol:<A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol5"> E. coli Overnight Culture</A>) Tubes were incubated overnight at 37 Â°C. 
						</P></UL><H3 id="week19">Week 19</H3><H4>10/01</H4><UL><LI><U><B>genome-isolation for MT1+beta-arrestin cells:</B></U></LI><P>
								We decided to isolate the genome to check the correct genome integration of MT1 and beta-arrestin. (protocol: <A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol19"> Quick and Dirty</A> )
							</P></UL><UL><LI><U><B>Verification PCR for  MT1 gene and beta-arrestin:</B></U></LI><P>
									Verification PCR was performed to check the success of the transformation. As a template, fresh isolated genome was used.
									We decided to use Q5X2-Mastermix for PCR.
							</P></UL><UL><LI><U><B>Gel-electrophoresis for verification of MT1 with beta-arrestin:</B></U></LI><P>
								Gel electrophoresis showed good results. Bands could be observed at about 5000bp, which is the right length.
							</P></UL><UL><LI><U><B> Production of LB liquid medium:</B></U></LI><P>
								800 ml of LB liquid medium were produced for cultivation of <I>E.coli</I>. (protocol:<A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol21">LB liquid medium</A>)
							</P></UL><H4>10/02</H4><UL><LI><U><B>Purification of RZRbeta:</B></U></LI><P>
								RZRbeta was prepared for purification. (protocol:<A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol35">Purification of proteins1</A>)
								Ni-NTA affinity chromatography was done. (protocol:<A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol34">Ni-NTA affinity chromatography</A>)
							</P></UL><UL><LI><U><B>Preparation of SDS PAGE gels:</B></U></LI><P>
							 Two gels were prepared for protein separation. (Protopcol: <A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol33">Preparation of SDS page gels</A>)
						</P></UL><H4>10/05</H4><UL><LI><U><B>Plasmid isolation for biobricks from <I>E. coli</I>:</B></U></LI><P>
								We were able to obsereve the growth of some <I>E. coli</I> colonies with the plasmid pSB1C3+Gal-RZR2 on the transformation plate (09/20). As a 
								result, we decided to isolate the plasmids of
								ten colonies to check the pSB1C3+Gal-RZR2 plasmid. (protocol: <A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol1"> Plasmid clean-up</A> )
							</P></UL><UL><LI><U><B>Preparation of YPG-medium:</B></U></LI><P>
								500ml of liquid medium were prepared. (protocol: <A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol32"> YPG-medium</A> )
							</P></UL><UL><LI><U><B>Purification of RZR:</B></U></LI><P>
						RZR was prepared for purification. (protocol:<A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol35">Purification of proteins1</A>)
								Ni-NTA affinity chromatography was done. (protocol:<A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol34">Ni-NTA affinity chromatography</A>)
						</P></UL><UL><LI><U><B>Preparation of SDS page gels:</B></U></LI><P>
							 Two gels were prepared for protein separation. (Protopcol: <A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol33">Preparation of SDS page gels</A>)
						</P></UL><H3 id="week20">Week 20</H3><H4>10/06</H4><UL><LI><U><B>Overnights of YPG-cells:</B></U></LI><P>
							Overnights were prepared for measurement of luminiscence
						</P></UL><UL><LI><U><B>Overnights of <I>E. coli</I> with RZRalpha in pPR-IBA1:</B></U></LI><P>
						 Overnights were prepared (Protopcol: <A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol5">Overnight culture E.coli</A>).
						</P></UL><H4>10/07</H4><UL><LI><U><B>IPTG-induction of RZRalpha in pPR-IBA1:</B></U></LI><P>
							IPTG-induction was done to induce expression of RZRalpha-protein in <I>E.coli</I>.
						</P></UL><UL><LI><U><B>Ligation check of RZRbeta in pPR-IBA1:</B></U></LI><P>
							Restriction was done, to check if the Ligation was successful and RZRbeta has been integrated into pPR-IBA1.
						</P></UL><H3 id="week20">Week 20</H3><H4>10/08</H4><UL><LI><U><B>Restriction of RZRalpha from pPR-IBA1:</B></U></LI><P>
							 To check, if the Ligation of RZRalpha into the plasmid pPR-IBA1 has been successful, restriction was done.
						</P></UL><UL><LI><U><B>PCR for RZRbeta for biobricks:</B></U></LI><P>
							PCR was succes, so we were able to use the RZRbeta fragment for insertion into our biobricks.
						</P></UL><UL><LI><U><B>Digestion of RZRbeta for biobricks:</B></U></LI><P><OL><LI>restriction of insert:</LI></OL>
							Mix was incubated at 37Â°C for 2 h. Clean-up of the fragments done afterwards with 2x10 ul TE-Buffer for RZR2 and 25 uL TE-Buffer for pSB1C3.
						</P></UL><UL><LI><U><B>Digestion of pSB1C3 plasmid:</B></U></LI><P><OL><LI>restirction of plasmid:</LI></OL>
							Mix was incubated at 37Â°C for 4h.
						</P></UL><H4>10/09</H4><UL><LI><U><B>Purification of RZRbeta from pPR-IBA1:</B></U></LI><P>
							 RZRbeta was prepared for purification. (protocol:<A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol35">Purification of proteins1</A>)
								Ni-NTA affinity chromatography was done. (protocol:<A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol34">Ni-NTA affinity chromatography</A>)
						</P></UL><UL><LI><U><B>Purification of RZRalpha:</B></U></LI><P>
							 RZRalpha was prepared for purification. (protocol:<A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol35">Purification of proteins1</A>)
								Ni-NTA affinity chromatography was done. (protocol:<A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol34">Ni-NTA affinity chromatography</A>)
						</P></UL><H4>10/10</H4><UL><LI><U><B>Verification of beta-arrestine colonies:</B></U></LI><P>
							 Verification PCR was successful. Bands with the right length were visible for all temperatures.
						</P></UL><H4>10/11</H4><UL><LI><U><B>PCR for RZRalpha:</B></U></LI><P>
							 PCR was done to amplify the fragment for RZRalpha.
						</P></UL><H4>10/12</H4><UL><LI><U><B>Plasmid isolation and sequencing for biobricks:</B></U></LI><P>
							We were able to obsereve the growth of some <I>E. coli</I> colonies with the plasmid pSB1C3+RZRbeta on the transformation plate. As a 
							result, we decided to isolate the plasmids of
							some colonies to check the pSB1C3+RZR plasmid. (protocol: <A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol1"> Plasmid clean-up</A> ) 
						</P></UL><UL><LI><U><B>Purification of RZRbeta:</B></U></LI><P>
							 RZRbeta was prepared for purification. (protocol:<A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol35">Purification of proteins1</A>)
								Ni-NTA affinity chromatography was done. (protocol:<A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol34">Ni-NTA affinity chromatography</A>)
						</P></UL><UL><LI><U><B>Verification PCR for RZRbeta:</B></U></LI><P>
							 PCR was started to test the correct integration of the cassette for the RZRbeta gene in the genome.
						</P></UL><H4>10/13</H4><UL><LI><U><B>Preparation of agarose-gel:</B></U></LI><P>
							 500 ml agarose solution for gel electrophoresis were produced (Protopcol: <A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol10">Gel Electrophoresis solution</A>).
						</P></UL><UL><LI><U><B>Overnights of RZRalpha and RZRbeta in YPG-medium:</B></U></LI><P>
							 Overnight cultures of <I>S. cerevisiae</I> with RZR were prepared in YPG-medium (Protopcol: <A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol6">Overnights of S. cerevisiae</A>)
						</P></UL><UL><LI><U><B>Preparation of Histidine and Leucine for Yeast-dropout medium:</B></U></LI><P>
							Histidine and Leucine solutions (5mg/ml) were prepared in order to mix them with the Yeast-dropout medium without His, Leu and Ura (08/15).
						</P></UL><UL><LI><U><B>Overnights of beta-arrestin in YPG-medium:</B></U></LI><P>
							 Overnight cultures of <I>S. cerevisiae</I> with beta-arrestin were prepared in YPG-medium (Protopcol: <A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol6">Overnights of S. cerevisiae</A>)
						</P></UL><UL><LI><U><B>Dilution of overnight cultures:</B></U></LI><P>
							 Optical density was measured and overnight cultures were diluted, in order to have cells in mid-log phase.
						</P></UL><H4>10/15</H4><UL><LI><U><B>Purification of RZRalpha and RZRbeta:</B></U></LI><P>
							 RZRalpha and RZRbeta was prepared for purification. (Protocol:<A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol35">Purification of proteins</A>)
								Ni-NTA affinity chromatography was done. (protocol:<A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol34">Ni-NTA affinity chromatography</A>)
						</P></UL><H4>10/16</H4><UL><LI><U><B>Preparation of SDS page gels:</B></U></LI><P>
							 Two gels were prepared for protein separation. (Protocol: <A href="https://2018.igem.org/Team:Aachen/Wetlab/Protocol33">Preparation of SDS page gels</A>)
						</P></UL><LI><U><B>First measurament with the MT1-GPCR and Ã-Arrestin approach:</B></U></LI><UL><LI>We prepared overnight cultures with galactose as principal carbon source to activate the production of the MT1 GPCR and Ã-Arrestin. </LI><LI>Under presence of melatonin, Ã-Arrestin will bind with the MT1 Receptor and the inactive subunits from the beta-galactosidase will be active. </LI><LI> With the addition of X-Gal with melatonin should appear a blue coloring. </LI></UL><U>Troubleshooting with the assay</U><UL><LI>Possible lack of Cyclin G-associated kinase (GAK) in yeasts hinder the bound between beta-arrestin and MT1.</LI><LI>A low membran permeability for X-Gal impede the reaction with Ã-Galactosidase</LI><LI><U>See the perspectives in <A href="https://2018.igem.org/Team:Aachen/Project/Results">Results</A> to know how we will improve this procediment</U></LI><DIV><FOOTER class="footer-layout"><DIV class="container-fluid"><DIV class="row"><DIV class="col-sm-2"><P style="margin-top: 18px; letter-spacing:1.5px;"><B>CONTACT US:</B><A href="https://2018.igem.org/Team:Aachen/impressum">impressum</A></P></DIV></DIV></DIV></FOOTER></DIV></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>