Team:Aachen/Project/Results
PROJECT
Description Motivation Results
LAB
Overview Workflow InterLab Study Part Notebook Protocols/Methods Safety
HARDWARE
Overview Spectrometer Wafer
MODEL
HP
Human Practices Gold/Silver Collaborations
PEOPLE
Team Sponsors Attributions
AWARDS
MODEL HARDWARE HUMAN PRACTICES MEASUREMENT Entrepreneurship Public Engagement
RESULTS
During the whole process of iGEM we obtained different achievements allowing us to keep on with the different approaches in our project.
Wet lab
The first successful result was the knock-out of the responsible gene for the melatonin production in S. cerevisiae. Through one verification-PCR using the isolated genome from grown colonies in drop-out medium without leucine, we checked the presence of the enquired DNA sequence. We also did a DNA sequencing to reassure about this step.
Figure 1: Electrophoresis results with the successful bands      by 3500 bp.
For the detection approaches we used the NHX1-cassette with the X4 homologue regions. This time we assembled 6 DNA fragments coding for the RZRß and RZR⍺ receptors with overhangs and we transformed the yeast strain ( BMA64-1A ) with the desired receptors for our proof of concept. We used once again an auxotrophy marker (histidine) and we proved the success of this transformation with a verification PCR and sequencing.
Figure 2: Electrophoresis results with the successful bands      by approx. 5000 bp checking the successful transformation of RZRß.
Figure 3: Electrophoresis results with the successful bands      by approx. 5000 bp checking the successful transformation of RZR⍺.
To find the most sensitive method we engineered another detection approach testing the interaction between the ß-arrestin protein with the MT1 GPCR. This time we used the antibiotic G418 as a marker and the homologue regions x2 from the plasmid p0257.
Figure 4: Electrophoresis results with the successful bands      by approx. 5000 bp checking the successful transformation of MT1-GPCR.
Figure 5: Electrophoresis results with the successful bands      by approx. 5000 bp checking the successful transformation of ß-Arrestin.
To understand how the cell-based biosensor will work, we did a model one of  the biological approach.
The connector of the biological part with the hardware part is the purification from the nuclear receptors RZRß and RZRɑ . We cloned the genes in two different vectors, one with his-tag and one with the HaloTag . After the sonication we checked any kind of over-expression in E. coli cells and we started the purification using the Ni-NTA through columns.
Figure 6: Verification of the over-expression from the desired nuclear receptors after the sonication of the cells.
Figure 7: RZRß purification using the Promega HaloTag purification system .
Figure 8: RZRɑ purification using the polyhistidine affinity tag.
Perspectives
The challenge now is to proceed with the measurement of the interaction between luciferase and D-Luciferin in our yeast cells. Therefore we created an optimized protocol for a one-step yeast luminescence measurement.
The measurement with the MT1-ß-Arrestin approach was not successful cause of the problematic permeability of the yeast cell membran and therefore the interaction of the ß-galactosidase and X-gal. We created an improved protocol that could facilitate the measurement.
Hardware
During our hardware project we built the first ever spectrometer in iGEM history. It has a spectral resolution of 4 nm. The accessibility of our results is very important to us. Therefore we are publishing the manual for our device on our wiki page. At this moment costs of a commercial spectrometer is around 4,000$.
Moreover we developed the concept for a cell free measurement method using LSPR. Our sensor would cost at this point around 350$ for the spectrometer and 5$ for the sensor chip (only in mass production). This concept can be used for all kinds of molecules which work with gold nano particles. Commercial SPR devices like GE Healthcare Biacore T200 cost upwards of 50,000$.
For further information on our hardware sensor click here .
CONTACT US: impressum
