"
Page
Discussion
View source
History
teams
Log in
 
Team:KAIST-Korea/Notebook/Diary/August
From 2010.igem.org
 
August 1-8
  During the period of waiting(for the gene synthesis), we divided the iGEM works into three parts.
Modeling
For the modeling part, we found hundreds of other antibody sequences to compare. We then ran a program to find a structure of each antibody. After, we investigated the similarity (numbers and figures) by using structural alignment program. We are now searching for the mathmetical data that can help us to estimate our system's reaction rate and speed.
BioBrick Design
In the meantime, bio brick team finished the wiki design. they put contents on bio brick submittion page. We have 6 bio bricks so far, and all has been uproaded on our wiki page.
Introduction video of the Presentation
The presentation team made a short video. This clip will be rolled before our main presentation for hook. It contains our motivation and our efforts.
August 9
Do list
structural alignment
We almost completed structural alignments. We will post them on the wiki once they are finished.
modeling
We need theoretical data on affinity of our elements through the pathway for modeling. So, we need to dig in.
main page update
We decided to put contents on the menu to relieve confusion.
finishing introduction video
We are going to take a picture of us for the video
adding 'tools' page
We are making new page, named 'tools'. We will record programs or tools that we used so far.
August 11
do list
modifying main menu
taking picture for introduction video
main page content
We already finished the design we only need contents
affinity data search
We only need to find FGFR1-STAT1 Kd data
putting contents on discoverY wiki page
August 18
We sent an e-mail that contains how we want our gene synthesis to be done. Today, we got the reply from Bioneer, our sponsor. We had a discussion on how this synthesis can be done, and some questions reagarding this project were raised. So, we gathered our ideas and made a conclusion.  Answers and discussions
Cloning on S.pombe and cloning in pSB1C3 for submission are necessary.
pSB1C3 plasmid information: http://partsregistry.org/Part:pSB1C3:Design
Part that we will put our fusion FGFR in is S.pombe > http://old.genedb.org/genedb/Search?name=wsc1&organism=pombe&desc=yes&wildcard=yes&searchId=Search
They suggested that we use a linker to make a Fusion FGFR gene. And we agreed on this idea.
We decided to use middle expression vector instead of high expression vector for STAT1 cloning. Because high expression of foreign protein can make bad influence to the cell
There were misunderstanding on GAS promoter. It must be activated only by the GFP (We sent our pathway to them for better understanding).
Our primer design :
<Back of Fusion FGFR> - Forward:23bp, Tm 59. 5' - GTTCTGGAAGCCCTGGAAGAGAG - 3' - Reverse:23bp, Tm 60 3' - TACCGCCTGAGTTTGCGGCGACT - 5'
<Front of Fusion FGFR> - Forward : 21bp, Tm 58 5' - ATGGTCTTTTTAAATTCCTCT - 3' - Reverse : 23bp, Tm 59 3' - AGGAGTCGGTTTTGTTGTCGGGG – 5'
<STAT1> - Forward : 24bp,  Tm 59  5' - ATGTCTCAGTGGTACGAACTTCAG - 3' - Reverse : 24bp, Tm 58 3' - AACTAAAGACACAGACTTCACATT - 5'
PS. we are going to meet tomorrow for gene extracting.
August 19
We went to KIB for gene extraction. Today, as a preparation, we made an LB broth. And we set our experiment table.
prepare LB broth powder
add 500mL of DW on LB broth powder using mass cylinder
stir with magnetic bar
do autoclave
cool down LB culture medium
store it in cylinder (100mL * 2)
add ampicillin(1000x) to one of cylinder : 100uL / 100mL
put FGFR into the cylinder that contains E.coli in ampicillin, and add STAT in other cylinder
store these two cylinders at 37℃ and 200 rpm condition
Tomorrow we will extract DNA from today's products. And we decided to make a individual experimental note and keep record of our individual works.
August 20
we went to KIB for cell lysis & DNA extraction.
put 45mL of STAT1 solution into a cornical tube, put 45mL of FGFR solution into the other cornical tube.
add 6mL of SolutionI, then do vortexing.
add 6mL of SolutionII, then shake and roll.
add 6mL of SolutionIII, then shake and roll.
put each STAT1, FGFR solution in first column (10 min).
put 4mL of equilibrium buffer(QBT) into the second buffer.
put solution that passed by first column into second column.
add 20mL of washing buffer to the second column.
add 5mL of elution buffer to the second column.
add 5mL of isopropane in second column (5 min).
push the piston for DNA condensation
add TE buffer for elution
check DNA concentration using machine
After DNA extraction, we went to CMS builing for gel electrophoresis.
prepare 1.2% of agarose gel, 0.5x TAE buffer
mix 1.5uL of STAT1, FGFR sample  and 1uL of loading buffer
loading STAT1, FGFR, DNA marker (100V, (-)→(+))
We liked STAT1, FGFR DNA sample, and we will give these samples to bioneer next week.
August 26
We went to Bioneer for communication.
Communicated topic
Primer
We requested primer synthesis. But our list of requested primer was hard for them to recognize. Therefore they asked us to change and organize our request form.
TA cloning
They asked us if we can do TA cloning in our lab. And we answered that we have no system that can manage TA cloning. Today, they offered us that they can do TA cloning for us. TA cloning is one additional way to store genes in safe way.
Vector
We tell them that SAL1 is the only one that can be used in STAT1 cloning. But it is safe to have two enzyme sites. Therefore we need to check other enzyme sites such as BamH1, Sal1.
pT218UHA12 vector
It is a way to change selection marker. Selection marker can be used first in this vector. Then after culture, we need to culture it in 5-FOA medium. Then selection marker can be deleted. If we use this method, we can use one selection marker twice, and this allow us to pick more vector in large choice.
Retrieved from " http://2010.igem.org/Team:KAIST-Korea/Notebook/Diary/August "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
