<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_KAIST-Korea_Notebook_Tools"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:KAIST-Korea/Notebook/Tools">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:KAIST-Korea/Notebook/Tools&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:KAIST-Korea/Notebook/Tools&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:KAIST-Korea/Notebook/Tools&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:KAIST-Korea/Notebook/Tools" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:KAIST-Korea/Notebook/Tools</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><DIV id="dashed_line"> </DIV><DIV id="wrapper"><TABLE width="100%" border="0" cellpadding="20px"><TBODY><TR><TD valign="top" width="75%"><P><SPAN style="font-size:30px"><FONT face="Maiandra GD"><B>Data Sources </B></FONT></SPAN></P><P>During our project, we needed many data. We used data of protein-protein interactions for signal transduction pathway deciding, protein sequence and domain description for modified FGFR design, gene sequence for modified FGFR sequence synthesis, and structures or sequences of antibodies and FGF binding domain of FGFR for structural alignment. These data were acquired from Uniprot, PID, NCBI and RCSB PDB.
</P><H3><SPAN class="mw-headline" id="Uniprot">Uniprot</SPAN></H3><P>Uniprot provides sequences of many proteins and domain description of well-researched proteins. Key advantage of using Uniprot is domain description: to design engineered protein for detecting Mycobacterium without knowledge of function of each part of protein. Tuberculosis antigen MPT51 is impossible one. Uniprot provided the information of location of FGF binding domain of FGFR, which is replaced by our single-chain antibody 16A1. Uniprot also provided sequences of some antibodies to make single chain antibody sequences.
</P><UL><LI> Site URL: <A href="http://www.uniprot.org/" class="external free" rel="nofollow">http://www.uniprot.org/</A></LI></UL><H3><SPAN class="mw-headline" id="PID">PID</SPAN></H3><P>PID (Pathway Interaction Database) provides the interaction networks between protein and protein or protein and DNA through certain signal transduction pathway. This Protein-protein interaction (PPI) and protein-DNA interaction (PDI) data helped us to port Human signal transduction pathway which is activated by FGF to fission yeast. Without knowledge of PPI and PDI through FGF signal pathway, we must undergo many trials and errors of adding and removing of proteins and promoters to form working signal transduction pathway. With the data from PID, we decided to port the FGF-&gt;FGFR1-&gt;STAT1-&gt;GAS pathway from human to fission yeast.
</P><UL><LI> Site URL: <A href="http://pid.nci.nih.gov/" class="external free" rel="nofollow">http://pid.nci.nih.gov/</A></LI></UL><H3><SPAN class="mw-headline" id="NCBI">NCBI</SPAN></H3><P>NCBI provides biologists with many data. We used protein sequences and DNA sequences from NCBI nucleotide database and protein database. DNA sequence is very important to us because gene sequence of original protein is required to synthesize novel engineered protein. Even we don't synthesize gene, we should know the sequence of gene because BioBrick requires not only nucleotide material, but also its sequence information. NCBI also provided us with sequences of some antibodies to make single chain antibody sequence, and many Journals through PubMed service. 
</P><UL><LI> Site URL: <A href="http://ncbi.nlm.nih.gov/" class="external free" rel="nofollow">http://ncbi.nlm.nih.gov/</A></LI></UL><H3><SPAN class="mw-headline" id="RCSB_PDB">RCSB PDB</SPAN></H3><P>RCSB PDB (Protein Data Bank) provides the data of structure of protein or other biomolecules. Key feature of data from PDB is structure. NCBI or Uniprot provide the sequence of proteins, but it does not show us their 3D-structures. With the structural similarity between FGF binding domain of FGFR and single chain antibody 16A1, we can ensure that the replacement of FGF binding domain with 16A1 to detect MPT51 is appropriate.
</P><UL><LI> Site URL: <A href="http://www.pdb.org/" class="external free" rel="nofollow">http://www.pdb.org/</A></LI></UL><P><SPAN style="font-size:30px"><FONT face="Maiandra GD"><B>Tools </B></FONT></SPAN></P><P>During our project, we processed much bio-information. It is not easy to process that massive information manually, so we used many bioinformatics tools for our projects. We marked restriction sites to select proper restriction enzyme, searched the nucleotide which codes query peptide sequence to find the coding region of certain genes, virtual-translated given nucleotide sequence to check if our sequence codes expected protein, and predicted and aligned structure of single chain antibodies with FGF binding domain of FGFR to check single chain antibodies are structurally similar to confirm that replacement of FGF binding domain with single chain antibodies is appropriate. We used BioEdit to mark restriction site, BLAST to find the coding region of certain genes and to compare similar proteins, Transeq for virtual-translation, Modeller for structural prediction of single chain antibody, and Matt for structural alignment between single chain antibodies and FGF binding domain of FGFR.
</P><H3><SPAN class="mw-headline" id="BioEdit">BioEdit</SPAN></H3><P>BioEdit is the program for displaying biological sequences. It displays different amino acids or nucleotides in different colors to allow its users to check the change between sequences. It has many simple but useful functions. We used BioEdit for marking the restriction sites on given sequence to select proper restriction enzyme, which does not restrict coding region of gene. Other functions of BioEdit like phylogeny making or front-end of ClustalW are not used in our project, but they are also useful.
</P><UL><LI> Supporting Platform: Windows
</LI><LI> License: Freeware
</LI><LI> Download: <A href="http://www.mbio.ncsu.edu/BioEdit/bioedit.html" class="external free" rel="nofollow">http://www.mbio.ncsu.edu/BioEdit/bioedit.html</A></LI></UL><H3><SPAN class="mw-headline" id="BLAST">BLAST</SPAN></H3><P>BLAST is the alignment search tools for protein or nucleotide sequences. There are five modes of BLAST; blastn (nucleotide to nucleotide), blastp (Protein to Protein), blastx (nucleotide to protein), tblastn (protein to nucleotide), and tblastx (translated nucleotide to translated nucleotide). We used tblastn to find the location of coding region of given protein and blastn to find the differences between transcription variants of same genes.
</P><UL><LI> Web service: <A href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" class="external free" rel="nofollow">http://blast.ncbi.nlm.nih.gov/Blast.cgi</A></LI><LI> Supporting Platform: Solaris, Linux, Windows, and Mac OS X
</LI><LI> License: Public Domain
</LI><LI> Download: <A href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" class="external free" rel="nofollow">http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download</A></LI><LI> For PowerPC G4/G5 processors and Mac OS X, optimized version of standard BLAST is also available  <A href="http://developer.apple.com/opensource/tools/blast.html" class="external text" rel="nofollow">here</A></LI></UL><H3><SPAN class="mw-headline" id="Transeq">Transeq</SPAN></H3><P>It is not difficult to make protein sequence from nucleotide sequence. With codon table, we can make protein sequence manually without any special ability. To translate long nucleotide sequence, however, is not easy. Therefore, we used Transeq to virtually translate given nucleotides sequence. Transeq also do virtual-translation with shifted reading frame or non-transitional translation code tables like mitochondrial translation code table.
</P><UL><LI> Web service: <A href="http://www.ebi.ac.uk/emboss/transeq/" class="external free" rel="nofollow">http://www.ebi.ac.uk/emboss/transeq/</A></LI><LI> Supporting Platform: Basically Linux. some porting projects are available.
</LI><LI> License: GNU General Public License as EMBOSS package
</LI><LI> Download: <A href="http://emboss.sourceforge.net/download/" class="external free" rel="nofollow">http://emboss.sourceforge.net/download/</A></LI><LI> For windows, <A href="http://emboss.sourceforge.net/download/cygwin.html" class="external text" rel="nofollow">CygWin notes</A> may be helpful
</LI><LI> For Macintosh, <A href="http://emboss.sourceforge.net/download/macosx.html" class="external text" rel="nofollow">Mac OS X notes</A> may be helpful
</LI></UL><H3><SPAN class="mw-headline" id="Modeller">Modeller</SPAN></H3><P>Modeller is the program to predict the structure of protein with given peptide sequence based on the homology model. When given query sequence, Modeller searches for the similar sequence from database sequence whose structure is known. It assumes that similar sequences have similar structures and predicts the structure of query protein as the combination of known structure with similar sequence. This method is very useful for prediction of single chain antibodies because the structures of many original antibodies are known.
</P><UL><LI> Supporting Platform: Unix, Linux, Windows, and Mac
</LI><LI> License: Free fore non-profit academic institutions
</LI><LI> Download: <A href="http://www.salilab.org/modeller/download_installation.html" class="external free" rel="nofollow">http://www.salilab.org/modeller/download_installation.html</A></LI></UL><H3><SPAN class="mw-headline" id="Matt">Matt</SPAN></H3><P>It is also possible to compare the structures between FGF binding domain of FGFR and single chain antibodies by using Modeller manually, but it is not quantitative and estimated by the rule of thumb. So, the result is not useful for further analysis. We used Matt to compare the structures of FGF binding domain of FGFR and single chain antibodies. Matt uses algorithms to maximize shared structure with small translation and rotations. Matt provides quantitative results to enable estimation of similarity and aligned structures of proteins to visualize the alignment.
</P><UL><LI> Supporting Platform: Unix, Linux, and Windows
</LI><LI> License: GNU public license
</LI><LI> Commercial Matt license is available through the MIT and Tufts offices of Technology Transfer for a non-GPL software package.
</LI></UL><LI> Download: <A href="http://groups.csail.mit.edu/cb/matt/" class="external free" rel="nofollow">http://groups.csail.mit.edu/cb/matt/</A></LI><H3><SPAN class="mw-headline" id="PyMol">PyMol</SPAN></H3><P>Structure of complex of protein and other biomolecules are often saved as format of “*.PDB”. Visualization and analysis of that structure and sequence are also important to design novel engineered proteins. PyMol is used for such purposes. We used PyMol for two processes. First of all, it is used to confirm iG-like regions of FGFR are really FGF binding domain. We downloaded the FGF binding domain of FGFR from RCSB PDB and checked the sequence binds to the FGF is really marked as iG-like regions. (Interleukin receptors have iG-like regions, but they do not bind to their signal molecules.) Other process using PyMol is to visualize the structural alignment result made by Matt. PyMol saves the image of biomolecules in png format.
</P><UL><LI> Supporting Platform: Unix, Linux, Windows and Mac OS X
</LI><LI> License: Free for Older builds, registering is required for recent version
</LI><LI> Download: <A href="http://pymol.org/rel/099/" class="external text" rel="nofollow">free older builds</A> and <A href="http://pymol.org/rel/" class="external text" rel="nofollow">registering page</A></LI></UL><H3><SPAN class="mw-headline" id="Other_Media_Tools">Other Media Tools</SPAN></H3><UL><LI> Wiki decoration with images : Adobe Photoshop and MS paint
</LI><LI> Wiki decoration with flash movie : Adobe Flash
</LI><LI> Movie production for presentation : Vegas Movie player
</LI></UL></TD></TR></TBODY></TABLE><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:KAIST-Korea/Notebook/Tools">http://2010.igem.org/Team:KAIST-Korea/Notebook/Tools</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:KAIST-Korea/Notebook/Tools" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:KAIST-Korea/Notebook/Tools" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:KAIST-Korea/Notebook/Tools&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:KAIST-Korea/Notebook/Tools&amp;oldid=49660" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></BODY></HTML>