<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_KAIST-Korea_Notebook_Memo_Etc"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:KAIST-Korea/Notebook/Memo/Etc">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:KAIST-Korea/Notebook/Memo/Etc&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:KAIST-Korea/Notebook/Memo/Etc&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:KAIST-Korea/Notebook/Memo/Etc&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:KAIST-Korea/Notebook/Memo/Etc" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:KAIST-Korea/Notebook/Memo/Etc</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><DIV id="dashed_line"> </DIV><DIV id="wrapper"><TABLE width="100%" border="0" cellpadding="20px"><TBODY><TR><TD><H2><SPAN class="mw-headline" id="Etc._.C2.A0.C2.A0.C2.A0.C2.A0.C2.A0We_handle_information_and_idea_at_ETC"><SPAN style="font-size:20px"><B> Etc. </B></SPAN>     <SPAN style="font-size:15px">We handle information and idea at ETC </SPAN></SPAN></H2><P><SPAN style="font-size:15px"><B> MAB </B></SPAN>
Until now, mab is composed of mouse's antigen injection, splenocyte seperation, fusion between splenocyte and myeloma cell, and antibody making cell selection.
Myeloma cell, which is myelogenous leukemia, doesn't have antibody produce ability, because it is immobilized, when it fuses B-cell in splenocyte, it can take a culture and make strain to produce antibody.
But, because of cancer cell's unique lability, if it take a long period culture, it can be cloning again.
Most of all, fusion of after fusion selection spends lots of cost and time.
Because B-cell activation needs T-cell, it is hard to take mouse's antigen injection process in vitro.
So to save time, elemination of fusion is needed.
To use transfomation protocol, if cell growth division gene insert it, it is possible.
<SPAN style="font-size:15px"><B> Leech anticoagulent </B></SPAN>
Long since leech was used to pluck out the blood from people because they though blood reducing relieve the hypertension.
Even now, it is used to relieve swelling of injury.
Leech also have a effect which continues bleeding because it secrets anti-coagulent enzyem. 
In mechanical heart valve implementation or thrombosis, to prevent blood clot of vessel people use anti coagulent.
But this obstructs body defense mechanism. If we use leech's anti-coagulent enzyme, we can make medicine.
<SPAN style="font-size:15px"><B> Hemostasis </B></SPAN>
Undesirable clotting is one of disorder of hemostasis.
Thrombus is a clot that develops and persists in an unbroken blood vessel.
Embolus is kind of thrombus that breaks away from the vessel wall and floats freely in the bloodstream.
These can obstruct tiny blood vessel like arteriol or venule.
How about making a bacteria which eliminate them?
<SPAN style="font-size:15px"><B> Bacteria Electricity </B></SPAN>
Magnetotactic bacteria has a magnet in its body. This bacteria has a magnetosome which makes magnet.
If magnetosome has a regular period and it can make or destroy own magnet, to use change of magnetic field, it can make electricity power.
Actually, this bacteria makes voltage like half of AA battery during 48 hours.
To use this, we can also make electricity.
<SPAN style="font-size:15px"><B> ATP Electricity </B></SPAN>
Contemporary society's energy source is electricity. 
On the other hand, an organism's energy source is ATP.
Such as bio-mass, organism used energy sources researches how chemical energy makes electric energy.
Especially, Making electricity using bio-mass uses method of rolling turbun. But this method are inefficient.
So, from ATP, we can make charge gradient, to use this we can make electric energy. In this case, we don't use living cells.
Just we need ATPase.
<SPAN style="font-size:15px"><B> Stanford team </B></SPAN>
Stanford team's large protein secretion system is hard to use with liposome.
Stanford team combined interleukin6 gene to</P><DL><DD>HylA signal sequence</DD><DD>HylR, HylC, IS1 : Front flanking sequence</DD><DD>HylB, HylD : Rear flanking sequence</DD><DD>tolC : channel protein</DD></DL><P>HylA is secretion inducing sequence, and the others are induced by HylA and make channel to pump proteins to outside of the cell.
It is different from liposome production.
So we need to check another way for liposome based secretion.
<SPAN style="font-size:15px"><B> Bacterial Secretory Vesicle </B></SPAN>
The appended file says that Bacterial Secretory Vesicle is hard to use.
<SPAN style="font-size:15px"><B> Cancer diagnosis marker </B></SPAN>
I couldn’t find thesis about cancer diagnosis by bacterial method in NCBI.
It can be both good and bad news.
Attached file is a paper of Prof. Park published in PLoS ONE.
It says that antibody detects surface marker of cancer tissue.
It seems like detecting it by 4 kinds of color reaction.
Maybe we could find another marker from another paper by prof.Lee Do Hyun, for example.
The general concept is to transfer the system to bacteria.
<SPAN style="font-size:15px"><B> BBa_K126002 </B></SPAN></P><P>The result of blastx of 'BBa_K126002' is
'vascular endothelial growth factor receptor 2[Mus musculus](NCBI Reference Sequence: NP_034742.2)'.
And the result of blast of refseq rna is 
'Mus musculus kinase insert domain protein receptor (Kdr), mRNA (NCBI Reference Sequence: NM_010612.2)'.
We now know that the kinase generally come from VEGFR of mouse, so we are searching that the mechanism how VEGFR accerelates growth. 
Our purpose is that we intercept the signal of VEGFR's growth accerelating mechanism to induce GFP revelation.
<SPAN style="font-size:15px"><B> G-protein Signaling </B></SPAN></P><P>There is a signaling by using G-protein, but Tyrosing kinase.
You could find more information with this address.</P><DL><DD><A href="https://2008.igem.org/Team:Illinois/Antibody_GPCR_Fusion" class="external free" rel="nofollow">https://2008.igem.org/Team:Illinois/Antibody_GPCR_Fusion</A></DD></DL><P>We found the tyrosine receptor's identity using 'BLASTX' and 'refseq-rna DB'.
<SPAN style="font-size:15px"><B> Y can D pathway </B></SPAN></P></TD></TR><TR><TD>
1. 'Y. canD' is spread on the sample.</TD></TR><TR><TD>
2. 'Y. canD’s antibody combines with the antigen on the <I>Mycobacterium tuberculosis</I>.</TD></TR><TR><TD>
3. The tyrosine kinase JAK1, LMO4 and gp130 dimer bind to the cytokine receptor integral to membrane.</TD></TR><TR><TD>
4. JAK autophosphorylation activates transcription factors called STAT3.</TD></TR><TR><TD>
5. The activated STAT3 dissociate from the receptor.</TD></TR><TR><TD>
6. The activated STAT3 form dimers.</TD></TR><TR><TD>
7. The STAT3 dimer go into nucleus and regulates GFP gene that attached to APRE elements of promoter.</TD></TR><TR><TD>
8. 'Y.canD' expresse GFP.</TD></TR></TBODY></TABLE><TABLE width="100%" border="0" cellpadding="20px"><TBODY><TR><TD><P><SPAN style="font-size:15px"><B> Why Yeast? </B></SPAN>  Why we use ‘Yeast’ instead of ‘E. coli’?
E. coli control is easier than Yeast control. E. coli has faster cell cycle than Yeast. Moreover, there are many E. coli templates. But, although E. coli has many profits, we choose Yeast. Why? The following table is E. coli properties.[2]
</P><P>  According to the table, we can find easily some reasons that E. coli cannot work human pathway well.  First, the periplasm’s thickness of E. coli is the very small as 10nm. In our project, we have to use human proteins. Almost human proteins are much bigger than E. coli’s proteins. (They are over 10nm*10nm*10nm.) Furthermore, in our project, STAT3 has 60nm height. Therefore, this protein cannot be in periplasm, and our pathway cannot work completely.[3]   Second, the average E. coli protein has only 360 residues, 5nm diameter, and 40kD MW. As we mentioned, our project’s proteins have over 1000residues, 10nm diameter and 50kD MW. Thus, the possibility that E. coli have our pathway is very low.   However, Yeast has big volume size from 3~4 µm to 40 µm. [4] And Yeast is a eukaryotic, so it is more similar to human cell than E. coli. There is no physical problem of using human protein in yeast. And has more possibility to do our pathway. Therefore, we use Yeast instead of E. coli.
<SPAN style="font-size:15px"><B> Biochemical Receptors </B></SPAN></P><P>  The upper table represents classification of all biological receptors. First of all, since our genetically engineered machine has to detect on mycobacteria directly, we should select cell surface receptors. Next, we should consider using fusion-antibody-receptors. We used gene combination simply (add antibody to biological receptor without original receipt part). If the receptor has conformational change property when it accepts input signal, we guess that our fusion antibody receptor cannot operate well because antibodies characteristics (antigen size, antibody size, etc.) are different. They don’t have special receipt part in a whole receptor, of course, so whole receptor is important receipt part; we cannot create fusion-antibody-receptor with this protein anyway. Therefore, we choose ‘Dimerization’ receptor. Most of these receptors have Immunoglobulin-like receptor part which can be alternated by antibodies easily. Also, since biosensors should activate gene expression, we selected gene-regulation related receptors.   Finally we choose cytokine receptors and receptor tyrosine kinases. Among them, interleukin-6 alpha receptor was selected.
<SPAN style="font-size:15px"><B> Fusion Antibody Receptors </B></SPAN>  <B>The Fusion Antibody-Receptor</B> is a protein which receptor genetically combines with a specific antibody. The following figures show what the fusion antibody-receptor is for details.
</P><P>  The fusion antibody-receptor is designed by simple gene modification. We altered the <I>Mycobacterium tuberculosis</I> antigen detectable antibody single chain instead of the cytokine receptor’s antigen acceptable part (Immunoglobulin-like parts). Light chain part of the antibody – Linker – Heavy chain part of antibody is enough. (There are some evidences to success making fusion antibody-receptor with this method in some experiments. [4]) Then in order to express the receptor on cell surface membrane of Yeast, we change the original signal peptide to Omp(Outer membrane protein) signal peptide of Yeast. Surely the receptor should satisfy our standard for biosensor. 
</P><P>  <B>Schizosaccharomyces pombe Signal peptide</B> from Cell wall integrity and stress response component 1 </P><P>  Protein sequence:   MVFLNSSPFKGRLLFFVYLLIISTRLVAA </P><P>  mRNA sequece:  atggtgtttctgaacagcagcccgtttaaaggccgcctgctgttttttgtgtatctgctgattattagcacccgcctggtggcggcg <SPAN style="font-size:15px"><B> Signal Pathway : JAK-STAT Pathway </B></SPAN>  There are many signal pathways from plasma membrane receptor to the gene regulation. Figure B is an example of these pathways, MAP kinase pathway linked with RAS pathway. In this pathway,
</P><OL><LI> With existence of signal molecule, receptor kinases dimerize themselves and phosphorylate each other.
</LI><LI> This phosphorylated receptor dimer is cognized by Grb2 and recruit SOS protein, the Guanine nucleotide Exchange Factor(GEP)
</LI><LI> Recruited SOS protein substitute as GDP of Ras protein with GTP.
</LI><LI> Ras protein with GTP changes its conformation and activate its own kinase activity(Ras pathway so far)
</LI><LI> Ras kinase activate the MAP kinase kinase kinase, MAP KKK(also called as Raf protein) with phosphorylation
</LI><LI> MAP KKK activates the MAP kinase kinase, MAP KK(also called as Mek) with phosphorylation
</LI><LI> MAP KK activates the MAP kinase, MAP K(also called as Erk) with phosphorylation
</LI><LI> Activated MAP Kinase phosphorylates many cytosol proteins.
</LI><LI> For example, if Jun TF is phosphorylated by MAP K, phosphorylated Jun go inside of nucleus and activate the transcription to bind to DNA. (MAP pathway so far)
</LI></OL><P>  But to port this signal pathway into the Yeast is not easy because this pathway is related with too many new proteins, while JAK-STAT pathway is easy to port because it doesn’t have many participating proteins. The signal transduction step of JAK-STAT pathway as follows.
</P>   <OL><LI> Cytokine binds to the cytokine receptor with JAK kinase.
</LI><LI> Kinases form dimer and are phosphorylated by JAK.
</LI><LI> These phosphorylated tyrosine residue is cognized by STAT protein.
</LI><LI> STAT which cognize phosphorylated tyrosine phosphorylate other STAT protein.
</LI><LI> Two phosphorylated STAT protein form dimer and go inside to the nucleus and activate near gene to bind to the APRE element.
</LI></OL><P>  Because JAK-STAT pathway have only 5 steps while RAS-MAP pathway require 9 steps, we decide to port JAK-STAT pathway to the Yeast.
</P>   <P>   Actually, there are many JAK-STAT pathways in human cell. JAK and STAT are important components of many cytokine receptor systems. According to combination of the receptor, JAK and STAT, there exits many pathways which acts different works in eukaryotic cell. Among these pathways, we choose IL-6α, JAK1, STAT3 pathway. In this pathway, gp130 dimer and LMO4 are combined with JAK1, and assists a whole response. Why we selected the IL-6α specially? That is, IL-6α is a unique receptor which has a Ig-like region at outer part as different with other cytokine receptors. The introduction of IL-6α, gp130, and LMO4 are below. 
</P><P><SPAN style="font-size:15px"><B> Gene Activation for Output </B></SPAN>
STAT1 and STAT3 proteins with the formation of STAT3/3 and STAT1/3 dimers are confirmed using probe, namely the APRE probe. <B>APRE probe</B>(<I>A</I>cute-<I>p</I>hase <I>r</I>esponse <I>e</I>lement probe) is the acute phase responsive element of the α2M gene promoter and known to bind STAT1 and STAT3 proteins. Using this pathway, we altered the<B>green fluorescent protein (GFP) gene</B> instead of cytokine inducible genes. So, we could show GFP expression through the previous whole pathway.<B>[Fig. 5]</B></P><P>  <B>FGF signaling pathway</B>  We use FGF signaling pathway(FGF-2) which acts critical role in Craniosynostosis in suture closure. Whole pathway of FGF signaling change osteoblast cell activity eventually. In that one, We focused on osteopontin expression.
</P><SPAN style="font-size:15px"><CENTER><B>Fig 8. FGF signaling Pathway. Red line pathway is selected in our project.</B></CENTER></SPAN><OL><LI> FGF-2s(Fibroblast Growth Factor-2) binds to FGFR(Fibroblast Growth Factor Receptor)
</LI><LI> There are Syndecan-2 which help signal pathway of FGF in membrane. It make FGFR to activate to pass the signal.
</LI><LI> There are FOS and JUN proteins in cytoplasm.(In addtion, FOS and JUN mRNA synthesize more when FGF-2 signal binds.) Especially, c-FOS and c-JUN attach each other and form AP-1(Activate Protein 1).
</LI><LI> Changed FGFR and FGF-2 composition activate AP-1.
</LI><LI> Activated AP-1 can bound to DNA and osteopontin gene is expressd.
</LI></OL><P>  Like this,FGF signaling pathways have only 5 steps while RAS-MAP pathway require 9 steps, we decide to port <B>FGF signaling pathway</B> to the Yeast.   The introduction of <B>Syndecan-2</B> and <B>AP-1</B> are below. 
</P><SPAN style="font-size:15px"><CENTER><B>Fig 9. PDZ-like domain of syndecan-2 </B></CENTER></SPAN><SPAN style="font-size:15px"><CENTER><B>Fig 10.  AP-1 (c-FOS + c-JUN) bound to DNA  </B></CENTER></SPAN><H2><SPAN class="mw-headline" id="referance"> referance </SPAN></H2><DL><DD>Fig 5. RCSB Protein Database “Crystal structure of the hexameric human IL-6/IL-6 alpha receptor/gp130 complex”
</DD><DD><A href="http://www.rcsb.org/pdb/explore/explore.do?structureId=1P9M" class="external free" rel="nofollow">http://www.rcsb.org/pdb/explore/explore.do?structureId=1P9M</A></DD></DL><DL><DD>Fig 7. Invitrogen “JAK STAT”
</DD><DD><A href="http://www.invitrogen.com/site/us/en/home/Products-and-Services/Applications/Cell-and-Tissue-Analysis/Signaling-Pathways/Jak-STAT.html" class="external free" rel="nofollow">http://www.invitrogen.com/site/us/en/home/Products-and-Services/Applications/Cell-and-Tissue-Analysis/Signaling-Pathways/Jak-STAT.html</A></DD></DL><DL><DD>Fig 8. Akihiko Yoshimura, Hitomi Nishinakamura, Yumiko Matsumura and Toshikatsu Hanada, Negative regulation of cytokine signaling and immune responses by SOCS proteins, Arthritis Research &amp; Therapy(2005)
</DD><DD>Fig 9. RCSB Protein Database “THIRD N-TERMINAL DOMAIN OF GP130”
</DD><DD><A href="http://www.rcsb.org/pdb/explore/explore.do?structureId=1BJ8" class="external free" rel="nofollow">http://www.rcsb.org/pdb/explore/explore.do?structureId=1BJ8</A></DD></DL><DL><DD>Fig 10. RCSB Protein Database “Complex of LMO4 LIM domains 1 and 2 with the ldb1 LID domain”
</DD><DD><A href="http://www.rcsb.org/pdb/explore/explore.do?structureId=1RUT" class="external free" rel="nofollow">http://www.rcsb.org/pdb/explore/explore.do?structureId=1RUT</A></DD></DL><DL><DD>Fig 11. Roberto, Proteína fluorescente verde – história e perspectivas, Química de Produtos Naturais (2009)
</DD><DD>Fig 9. PDZ-like domain of syndecan-2 
</DD><DD>RCSB Protein Database, &quot;Solution structure of the atypical PDZ-like domain of synbindin&quot;
</DD></DL><DL><DD>Fig 10. AP-1 (c-FOS + c-JUN) bound to DNA  
</DD><DD>RCSB Protein Database, &quot;STRUCTURE OF THE DNA BINDING DOMAINS OF NFAT, FOS AND JUN BOUND TO DNA &quot;
</DD></DL><DL><DD>[2] Garrett, R.H., and Grisham, C.M. Biochemistry, 2nd Edition (2002), pg. 32&lt;/br&gt;</DD><DD>[3] RCSB PDB(protein database), <A href="http://www.rcsb.org/" class="external free" rel="nofollow">http://www.rcsb.org/</A></DD><DD>[4] Walker K, Skelton H, Smith K. (2002). accessdate=2009-11-28 &quot;Cutaneous lesions showing giant yeast forms of Blastomyces dermatitidis&quot;. Journal of Cutaneous Pathology 29 (10): 616–18. doi:10.1034/j.1600-0560.2002.291009.x
</DD><DD>[5] Dong-Uk Kim et.al, &quot;Analysis of a genome-wide set of gene deletions in the fission yeast Schizosaccharomyces pombe&quot;, Nat.Biotechnol(2010)
</DD><DD>[6] Uniprot Database “Interleukin-6 receptor subunit alpha”; <A href="http://www.uniprot.org/uniprot/P08887" class="external free" rel="nofollow">http://www.uniprot.org/uniprot/P08887</A></DD><DD>[7] Lichtman, Andrew H.; Abbas, Abul K. Cellular and molecular immunology (5th ed.). (2003). Philadelphia Saunders. <A href="/Special:BookSources/0721600085" class="internal mw-magiclink-isbn">ISBN 0-7216-0008-5</A>.
</DD><DD>[8] SR Roffler1,6, H-E Wang2,6, H-M Yu2, W-D Chang3,4, C-M Cheng3,4, Y-L Lu5, B-M Chen1 and T-L Cheng3,4. A membrane antibody receptor for noninvasive imaging of gene expression. Gene Therapy. Nature (2005)
</DD><DD>[9] Martine I. Darville, Ye-Shih Ho and Decio L. Eizirik, NF-B Is Required for Cytokine-Induced Manganese Superoxide Dismutase Expression in Insulin-Producing Cells, The Endocrine Society(2000)
</DD></DL></TD></TR></TBODY></TABLE><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:KAIST-Korea/Notebook/Memo/Etc">http://2010.igem.org/Team:KAIST-Korea/Notebook/Memo/Etc</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:KAIST-Korea/Notebook/Memo/Etc" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:KAIST-Korea/Notebook/Memo/Etc" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:KAIST-Korea/Notebook/Memo/Etc&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:KAIST-Korea/Notebook/Memo/Etc&amp;oldid=105784" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></BODY></HTML>