<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_KAIST-Korea_Project_Introduction"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:KAIST-Korea/Project/Introduction">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:KAIST-Korea/Project/Introduction&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:KAIST-Korea/Project/Introduction&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:KAIST-Korea/Project/Introduction&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:KAIST-Korea/Project/Introduction" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:KAIST-Korea/Project/Introduction</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><DIV id="dashed_line"> </DIV><DIV id="wrapper"><TABLE width="100%" border="0" cellpadding="20px"><TBODY><TR><TD valign="top" width="75%"><P><SPAN style="font-size:30px"><FONT face="Maiandra GD"><B>Motivation </B></FONT></SPAN></P><P>  We are suffering from various kinds of diseases. Three most prevalent ones would be malaria, HIV and tuberculosis. Malaria is transmitted from human to human by mosquitoes. Each year, malaria causes nearly one million deaths, and the majority of casualties occur in Africa. Furthermore, disease is high-risk in children, pregnant women, travelers, refugees, displaced persons, and laborers entering endemic areas. In the case of HIV, there are 33 million infected people in the world and 95 percent of them are living in developing countries. Last but not least, there is tuberculosis, which is caused by mycobacterium tuberculosis, affecting lungs most. Until now, more than 2 billion people are infected with tuberculosis bacilli and 90 percent of them live in developing countries. As we can see, all of these diseases are especially problematic in developing country.
</P><SPAN style="font-size:15px"><CENTER><B>Fig 1. Estimated TB incidence rates, 2008 </B></CENTER></SPAN><P>  In developing country, economic is unstable, and development in medical technology is not a priority. Unfortunately, basic medical examination like blood tests, biopsy and others need certain foundation of medical technology. In addition, some areas in developing countries do not offer electricity or access to high-tech equipments. Therefore, we need new and affordable diagnosis system especially for the diseases mentioned above. For our system, we choose TB as a target. In recent decades, diagnosis and treatment of TB have been developed. Diagnostic methods include Tuberculin Skin Test, X-ray Test for its activation and examination of the sputum. In addition, people take a blood test that involves ESR, white blood cell, CRP status checking. If necessary, CT scan and endoscopy can be performed. Most of these tests, however, require high technology and proper machines. Therefore, in developing countries, these tests are not really options. As an alternative, we introduce DiscoverY. 
</P><SPAN style="font-size:15px"><CENTER><B>Fig 2. People suffering from malaria and TB              Fig 3. Researching disease</B></CENTER></SPAN><P>  We suggest a system using antigen-antibody reaction. Actually, antibody cannot release perceivable signals by antigen-antibody reaction alone, so we thought of fusion antibody-receptor as the solution. Fusion antibody-receptor is a receptor which is fused with other functional protein, antibody. In this way, we can get result by gene expression which would generate GFP expression after our system is triggered by target antigens. As the result, we may develop universal sensor by changing only target antigen’s antibody in fusion antibody-receptor. It will provide relatively simple and direct way of diagnosis.
</P><SPAN style="font-size:15px"><CENTER><B>Fig 4. Antibody</B></CENTER></SPAN><P><SPAN style="font-size:30px"><FONT face="Maiandra GD"><B>Whole Pathway </B></FONT></SPAN></P></TD></TR><TR><TD><P><SPAN style="font-size:15px"><B> DiscoverY pathway </B></SPAN></P></TD></TR><TR><TD>
1. 'DiscoverY' is spread on the sample.</TD></TR><TR><TD>
2. 'DiscoverY’s antibody combines with the antigen on the <I>Mycobacterium tuberculosis</I>.</TD></TR><TR><TD>
3. The tyrosine kinase receptor FGFR binds to the cytokine receptor integral to membrane.</TD></TR><TR><TD>
4. FGFR autophosphorylation activates transcription factors called STAT1.</TD></TR><TR><TD>
5. The activated STAT1 dissociates from the receptor.</TD></TR><TR><TD>
6. The activated STAT1 forms dimers.</TD></TR><TR><TD>
7. The STAT1 dimer goes into nucleus and regulates GFP gene that is attached to GAS elements of promoter.</TD></TR><TR><TD>
8. 'DiscoverY' expresses GFP.</TD></TR></TBODY></TABLE><P><SPAN style="font-size:30px"><FONT face="Maiandra GD"><B>Antibodies for Detecting<I>Mycobacterium tuberculosis</I></B></FONT></SPAN></P><P>  The major problem of Tuberculosis includes the prevalence of multi-drug-resistant strains and co-infection with human immunodeficiency virus. We found a report claiming that <B>MPT51</B> is over-expressed on the tuberculosis cell membrane, so we can use MPT51 as a core antigen to detect tuberculosis. Also, other normal human cell in our body does not have MPT51 protein on a cell. In conclusion, existence of MPT51 on a cell tells if subject has tuberculosis or not. Therefore, we will make fusion antibody that can cognize MPT51 as an antigen, and put it into yeast. 
</P><SPAN style="font-size:15px"><CENTER><B>Fig. 5 The crystal structure of Mycobacterium tuberculosis MPT51 </B></CENTER></SPAN><P><SPAN style="font-size:30px"><FONT face="Maiandra GD"><B>Making Biosensor – Yeast Detector  </B></FONT></SPAN></P><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Biosensor"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Biosensor</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Organism_Decision_:_Why_Yeast.3F"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Organism Decision : Why Yeast?</SPAN></A></LI><LI class="toclevel-1 tocsection-3"><A href="#Organism_Decision_:_Minimal_genome_fission_yeast.2C_Schizosaccharomyces_pombe"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Organism Decision : Minimal genome fission yeast, Schizosaccharomyces pombe</SPAN></A></LI><LI class="toclevel-1 tocsection-4"><A href="#Biochemical_Receptors_Selection"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Biochemical Receptors Selection</SPAN></A></LI><LI class="toclevel-1 tocsection-5"><A href="#Designing_Fusion_Antibody_Receptor"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Designing Fusion Antibody Receptor</SPAN></A></LI><LI class="toclevel-1 tocsection-6"><A href="#Signaling_Pathway_Research"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">Signaling Pathway Research</SPAN></A></LI><LI class="toclevel-1 tocsection-7"><A href="#Signaling_Pathway_Research_:_Fibroblast_Growth_Factor_Signaling_Pathway"><SPAN class="tocnumber">7</SPAN><SPAN class="toctext">Signaling Pathway Research : Fibroblast Growth Factor Signaling Pathway</SPAN></A></LI><LI class="toclevel-1 tocsection-8"><A href="#Design_Gene_Activation_For_Output"><SPAN class="tocnumber">8</SPAN><SPAN class="toctext">Design Gene Activation For Output</SPAN></A></LI></UL></TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Biosensor"> Biosensor </SPAN></H3><P>  A <B>biosensor</B> is a device for the detection of an analyst that combines a biological component with a physicochemical detector component.[1] In order to make biosensor with genetically engineered organism, we should consider the following basic sensing process. 
</P><CENTER><U>Receive input -&gt; Process -&gt; Extract output</U></CENTER><P>  The ‘Receive input’ part can be alternated by represented receptor, the ‘Process’ part can be signaling pathway which changes input to output, and the ‘Extract output’ part can be something easily detectable by human. Nowadays, pigments or fluorescence proteins are usually used for output, and, in order to express these substances, we make genetically modified organism. Thus, all engineered biosensors with organisms should be modeled on (consist of)  <B>1) biochemical receptors</B> which can activate gene expression, <B>2) biological pathway</B>, and <B>3) gene activation</B> for output.
</P><H3><SPAN class="mw-headline" id="Organism_Decision_:_Why_Yeast.3F"> Organism Decision : Why Yeast? </SPAN></H3><P>  Why we use ‘Yeast’ instead of ‘E. coli’? E. coli control is easier than Yeast control. E. coli has faster cell cycle than Yeast. Moreover, there are many E. coli templates. But, although E. coli has many profits, we choose Yeast. Why? The following table is E. coli properties.[2]
</P><SPAN style="font-size:15px"><CENTER><B>Fig 6. E.coli statistics</B></CENTER></SPAN><P>  According to the table, we can easily find reasons why E. coli cannot support human pathway well.  First, the periplasm thickness of E. coli is as small as 10nm. In our project, we use human proteins. Almost all human proteins are much bigger than E. coli’s proteins. (They are over 10nm*10nm*10nm.) Furthermore, in our project, FGFR has 20nm height. Therefore, this protein cannot fit in periplasm, and our pathway cannot work completely.[3]   Second, the average E. coli protein has only 360 residues, 5nm diameter, and 40kD MW. As we mentioned, our project’s proteins have over 1000residues, 10nm diameter and 50kD MW. Thus, the possibility of E. coli having our pathway is very low.   However, yeast has big volume size from 3~4 µm to 40 µm, [4] and yeast is a eukaryotic, so it is more similar to human cell than E. coli. There is no physical problem of using human protein in yeast, and possibility of success is higher. Therefore, we use yeast instead of E. coli.
</P><H3><SPAN class="mw-headline" id="Organism_Decision_:_Minimal_genome_fission_yeast.2C_Schizosaccharomyces_pombe"> Organism Decision : Minimal genome fission yeast, <I>Schizosaccharomyces pombe</I></SPAN></H3><P>   Porting of cell signal transduction pathway is not easy for wild type organism. Most important reason is unwanted protein degradation. For wild type organism, unfamiliar protein can be harmful because it may be fragmented or misfolded proteins which disturb cellular pathway or viral proteins which may infect organisms. So wild type organism degrade unfamiliar proteins whether it is really harmful or not. And even degradation of at least one element of signal transduction pathway can disconnect whole pathway. So for successful porting of cell signal transduction pathway, inhibition of protease is important. Another problem is signal confusion. Basically, cellular signal transduction pathway is based on the protein-protein interaction(PPI). So unwanted PPI with elements of ported signal transduction pathway can confuse the signal. Of course if origin species of ported pathway is far enough from target species, we can exclude predictable unwanted PPI to avoid homologous proteins. But there are many unpredictable PPIs which can confuse the pathway. To avoid this problems, deletion of unnecessary proteins is required. And fission with minimal genome is made[5]. This yeast have only 1033 necessary genes of 5776 genes from wild type. So it may be useful to port cell signal transduction pathway from other organism(Human) without unwanted protein degradation or signal confusion.
</P><SPAN style="font-size:15px"><CENTER><B>Fig 7. Schizosaccharomyces pombe homologus recombinant</B></CENTER></SPAN><P>  Schizosaccharomyces pombe (S. pombe) Deletion Mutant Library Sets were created by Dr. Kwang-Lae Hoe at KRIBB (Korea Research Institute of Biotechnology and Bioscience, Korea) through collaboration with Sir Paul Nurse at CRC (Cancer Research Center, UK, currently at the Rockefeller University in USA) and Bioneer based on the genome sequences of the S. pombe provided by the Wellcome Trust Sanger Institute. This genome-wide S. pombe deletion collection covers more than 98% of genome consisting of 4,914 genes. As each deletion strain has its distinct tags, functional analysis can be performed both in a gene-specific manner and a genome-wide scale. Two different sets of mutant collections are available: h+ haploids and h+/ h+ diploids.
</P><H3><SPAN class="mw-headline" id="Biochemical_Receptors_Selection"> Biochemical Receptors Selection </SPAN></H3><P>  The upper table represents how we chose our biological receptor. First of all, since our genetically engineered machine has to detect the mycobacterium directly, we should select cell surface receptors. Next, we should consider using fusion antibody-receptors. We simply used gene combination (add antibody to biological receptor instead of original receipt part). If the receptor has property of changing its conformation upon acceptance of input signal, we believe that our fusion antibody receptor cannot operate well because antibodies characteristics (antigen size, antibody size, etc.) are different. They don’t have special receipt part in a whole receptor, of course, so whole receptor is important receipt part; so, we cannot create fusion-antibody-receptor with this protein. Therefore, we choose ‘Dimerization’ receptor. Most of these receptors have immunoglobulin-like receipt part, which can be alternated by antibodies easily. Also, since biosensors should activate gene expression, it has to be one of gene-regulation related receptors. </P><P>   Finally we choose “cytokine receptors and receptor tyrosine kinases”. Among them <B>Fibroblast Growth Factor Receptor (FGFR)</B> in receptor tyrosine kinases was selected. 
</P><SPAN style="font-size:15px"><CENTER><B>Fig 8. Crystal structure of a ternary FGF-FGFR-heparin complex</B></CENTER></SPAN><H3><SPAN class="mw-headline" id="Designing_Fusion_Antibody_Receptor"> Designing Fusion Antibody Receptor </SPAN></H3><P>  Fusion antibody-receptor is a protein whose receptor genetically combines with a specific antibody. The following figures show fusion antibody-receptor in details. 
</P><P>  The fusion antibody-receptor is designed by simple gene modification. We altered the mycobacterium tuberculosis antigen detecting antibody single chain instead of the FGFR receptor’s original antigen accepting part (immunoglobulin-like parts): Light chain part of the antibody – Linker – Heavy chain part of antibody is enough. (There are some cases of successfully making fusion antibody-receptor with this method. [4]) Then, in order to express the receptor on cell surface membrane of Schizosaccharomyces pombe, we change the original signal peptide to outer membrane signal peptide of Schizosaccharomyces pombe. Surely, the receptor should satisfy our standard for biosensor. 
</P><TABLE><TBODY><TR><TD style="background:#ffffff; border:2px solid #ffffff;" width="800"><P><B><I>Schizosaccharomyces pombe</I> Signal peptide for cell surface</B> from Cell wall integrity and stress response component 1 </P><P>Protein sequence: 
MVFLNSSPFKGRLLFFVYLLIISTRLVAA </P><P>mRNA sequece:
atggtgtttctgaacagcagcccgtttaaaggccgcctgctgttttttgtgtatctgctgattattagcacccgcctggtggcggcg </P></TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Signaling_Pathway_Research"> Signaling Pathway Research </SPAN></H3><P>  There are many signal pathways: from plasma membrane receptor to the gene regulation. Here is a famous example of these pathways, MAP kinase pathway linked with RAS pathway. In this pathway,
</P><OL><LI> With existence of signal molecule, receptor kinases dimerize themselves and phosphorylate each other.
</LI><LI> This phosphorylated receptor dimer is cognized by Grb2 and recruits SOS protein, the Guanine nucleotide Exchange Factor (GEP).
</LI><LI> Recruited SOS protein substitutes as GDP of Ras protein with GTP.
</LI><LI> Ras protein with GTP changes its conformation and activates its own kinase activity (Ras pathway so far).
</LI><LI> Ras kinase activates the MAP kinase kinase kinase, MAP KKK(also called as Raf protein) with phosphorylation.
</LI><LI> MAP KKK activates the MAP kinase kinase, MAP KK(also called as Mek) with phosphorylation.
</LI><LI> MAP KK activates the MAP kinase, MAP K(also called as Erk) with phosphorylation.
</LI><LI> Activated MAP Kinase phosphorylates many cytosol proteins.
</LI><LI> For instance, if MAP K phosphorylates Jun TF, phosphorylated Jun goes inside of nucleus and activates the transcription to bind to DNA (MAP pathway so far).
</LI></OL><P>  Porting this signal pathway into the Yeast is not easy because this pathway involves many new proteins, so we had to find simpler pathway. The followings are our history of the change in pathways. You can see the details 'Memo-etc page' in our wiki.
</P><P>  (1) Interleukin-1 pathways (8 proteins are required)  (2) Interleukin-6 JAK-STAT pathways (6 proteins are required)  (3) Fc Epsilon pathways (5 proteins are required)  (4) FGFR AP-1 pathways (4 proteins are required)  (5) FGFR STAT1 pathways (3 proteins are required)</P><P>  Finally, we selected FGF signaling pathway, which is the simplest. The signal transduction step of FGF signaling pathway is as followed.
</P><H3><SPAN class="mw-headline" id="Signaling_Pathway_Research_:_Fibroblast_Growth_Factor_Signaling_Pathway"> Signaling Pathway Research : Fibroblast Growth Factor Signaling Pathway </SPAN></H3><SPAN style="font-size:15px"><CENTER><B>Fig.9 FGF signaling pathway</B></CENTER></SPAN><P>  The figures show simple pathway of FGF (Fibroblast Growth Factor) and FGFR (Fibroblast Growth Factor).  FGF is a growth factor related with wound healing and embryonic development.
</P><TABLE><TBODY><TR><TD align="center" valign="center" width="500"><CENTER><SPAN style="font-size:15px"><B>Fig.10 Unneccesary Syndecan-2 Function</B></SPAN></CENTER></TD></TR></TBODY></TABLE><OL><LI> FGF binds to FGFR (Syndecan-2 helps in this process).
</LI><LI> STAT1 sticks to phosphorylated FGFR and then it gets phosphoryl group from FGFR (Phosphorylation).
</LI><LI> Two phosphorylated STAT1s attach to each other and form a dimer.
</LI><LI> The dimer goes inside a nucleus and binds to GAS promoter.
</LI><LI> The promoted gene expresses.
</LI></OL><P>  We applied this pathway to our project. Actually, syndecan is a cell surface proteoglycan. Its core protein spans the plasma membrane and contains three heparan sulfate chains and two chondroitin sulfate chains (not shown). The proteoglycans can modulate the activity of fibroblast growth factor (FGF). Free FGF binds poorly to the FGF receptor. Binding of FGF to the heparan sulfate chains like those on syndecan allows FGF to bind efficiently its receptor. Binding of FGF to heparan sulfate in the ECM protects FGF and forms a reservoir of the growth factor. In addition we use antibody to detect tuberculosis antigen and GFP to confirm whether antigen is bound. Therefore, we don’t need Syndecan-2 because there is no FGF particle in this project. 
  In conclusion, we need three genes (Fusion FGFR, STAT1, GFP with GAS promoter) in our project.
</P><H3><SPAN class="mw-headline" id="Design_Gene_Activation_For_Output"> Design Gene Activation For Output </SPAN></H3><SPAN style="font-size:15px"><CENTER><B>Fig 11. Structural bases of unphosphorylated STAT1 association and receptor binding</B></CENTER></SPAN><P>  STAT1 proteins with the formation of STAT1/1 dimers are confirmed using probe, namely the GAS probe. <B>GAS probe</B>(<I>G</I>amma <I>A</I>ctivation <I>S</I>ite probe) is an acute phase responsive element of the IL gene promoter and known to bind to STAT1 dimer proteins. Using this pathway, we placed  <B>green fluorescent protein (GFP) gene</B> instead of cytokine inducible genes. We can show GFP expression at the previous pathway.
</P><SPAN style="font-size:15px"><CENTER><B>Fig 12. Proteína fluorescente verde – história e perspectivas</B></CENTER></SPAN><P><SPAN style="font-size:30px"><FONT face="Maiandra GD"><B>For Success! Evidences of our project’s feasibility  </B></FONT></SPAN></P><SPAN style="font-size:15px"><CENTER><B>Fig 13. Success KAIST!</B></CENTER></SPAN><P>  To successfully complete our project, we need <B>4 conditions</B>.
</P><OL><LI>	FGFR should be expressed completely in yeast.
</LI><LI>	FGFR and STAT1 should be located at appointed position in yeast.
</LI><LI>	FGFR-STAT1 pathway should be operated as we expected to act normally as a transcription factor. 
</LI><LI>	Antibody fusion-receptor should work properly to phosphorylate sub-pathway molecules, STAT1s.
</LI></OL><P>
We have 4 papers that suggest our project will work. 
  <B>1.</B> [Reconstitution of Fibroblast Growth Factor Receptor Interactions in the Yeast Two Hybrid System] By  Ronit A loni-Grinstein, Andrew Seddon, and Avner Yayon
  &gt;&gt; This paper used yeast two hybrid system to show that FGFR interaction is possible in yeast. Through this paper, we knew that FGFR can be expressed completely in yeast without any functional disability and located in outer cellular membrane.
  <B>2.</B> [Jak2-Stat5 Interactions Analyzed in Yeast (Received for publication, January 30, 1998, and in revised form, March 2, 1998)] By Fariba Barahmand-Pour, Andreas Meinke, Bernd Groner, and Thomas Decker
  &gt;&gt; This paper proves that JAK-STAT pathway is operated completely in yeast. The JAK in yeast phosphorylates STAT5 without any trouble. FGFR belongs to tyrosine kinase receptor family, which is also JAK’s family, and STAT1 belongs to same protein family as STAT5 does. This rational suggests feasibility of our signal delivery system. Consequently, we were convinced that FGFR and STAT1 will be expressed completely to work as a signal pathway, and locate their appointed position properly. 
  <B>3.</B> [Fibroblast Growth Factor Receptor-Induced Phosphorylation of STAT1 at the Golgi Apparatus  Without Translocation to the Nucleus] By  LUCI´A CITORES, LING BAI,2 VIGDIS SØRENSEN, AND SJUR OLSNES2
  &gt;&gt; This paper shows that FGFR phosphorylates STAT1 at the golgi apparatus in human cell. The expressed FGFR at the golgi apparatus phosphorylates STAT1, and two phosphorylated STAT1 become dimerized to activate GAS promoter. The result of this study assures our FGFR-STAT1 pathway.
  <B>4.</B> [Novel Recognition Motif on Fibroblast Growth Factor Receptor Mediates Direct Association and Activation of SNT Adapter Proteins](Received for publication, April 13,  1998) by Hong Xu, Kyung W. Lee, and Mitchell Goldfarb
  &gt;&gt; In this paper, the researchers identified fusion FGFR, which is combined with antigen called SNT, was phosphorylated itself automatically. Through this result, we confirmed that our fusion FGFR would be phosphorylated properly when combined with the targeted antigen. 
  According to these papers and searched results, <B>we are convinced that our project’s pathway will work completely as we expected.</B></P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:KAIST-Korea/Project/Introduction">http://2010.igem.org/Team:KAIST-Korea/Project/Introduction</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:KAIST-Korea/Project/Introduction" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:KAIST-Korea/Project/Introduction" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:KAIST-Korea/Project/Introduction&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:KAIST-Korea/Project/Introduction&amp;oldid=51736" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></BODY></HTML>