<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_KAIST-Korea_Project_StructureAlignment"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:KAIST-Korea/Project/StructureAlignment">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:KAIST-Korea/Project/StructureAlignment&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:KAIST-Korea/Project/StructureAlignment&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:KAIST-Korea/Project/StructureAlignment&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:KAIST-Korea/Project/StructureAlignment" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:KAIST-Korea/Project/StructureAlignment</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><DIV id="dashed_line"> </DIV><DIV id="wrapper"><TABLE width="100%" border="0" cellpadding="20px"><TBODY><TR><TD valign="top" width="75%"><P><SPAN style="font-size:30px"><FONT face="Maiandra GD"><B> Structure Alignment </B></FONT></SPAN></P><H3><SPAN class="mw-headline" id="Single_chain_antibody_structural_alignment"> Single chain antibody structural alignment </SPAN></H3><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Single chain antibody structural alignment protocol</DIV></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Protocol"> Protocol </SPAN></H3><P>There are four steps to compare structure of single chain antibody and FGF binding domain of FGFR. The first step is to take variable region sequences of antibodies. The next one is to combine these variable region sequences with a linker sequence to make single chain antibody sequence. Then, we predict the structure of single chain antibody with a structure prediction program like Modeller. Lastly, we structurally align these structures of antibodies with structure of FGF binding domain of FGFR (PDB ID: 1EVT).
</P><H4><SPAN class="mw-headline" id="Data_source"> Data source </SPAN></H4><P>Single chain antibody is the combination of variable regions of known antibodies with linker sequence, which can bind to the antigens. We need to know the VL and VH sequences to make single chain antibody. The sources of these antibody sequences include NCBI, Uniprot and RCSB PDB. NCBI and Uniprot provide the single chain sequence of variable regions (VL and VH) and antigen binding fragments (Fab). RCSB provide the structure of antigen binding fragment complexes that bind to their antigens. We, however, only need the sequence of variable region. So, we get the last 120~150 residues and assume them as the variable region. And, data from RCSB contain not only sequence of antibody but also antigens. Therefore, we filter them based on label of files to get heavy chains and light chains of antibody.
</P><H4><SPAN class="mw-headline" id="Single_chain_antibody_synthesis"> Single chain antibody synthesis </SPAN></H4><P>We combine the antibody variable region sequences in order of VH-linker-VL to make single chain antibody sequence. The sequence of the linker is GGGGSGGGGS.
</P><H4><SPAN class="mw-headline" id="Structure_Prediction"> Structure Prediction </SPAN></H4><P>We used the program called Modeller to predict the structure of single chain antibody from its sequence. Modeller predicts 3D structure of protein with structure of known similar proteins based on homology model. Input file is the sequence of single chain antibody in fasta format and output file is the structure of single chain antibody in pdb format.
</P><H4><SPAN class="mw-headline" id="Structure_Alignment"> Structure Alignment </SPAN></H4><P>In this step, we check the structural similarity between single chain antibodies and FGF binding domain to align the structure of single chain antibody with that of FGF binding domain of FGFR. The structure of FGF binding domain of FGFR is provided by RCSB PDB (PDB ID: 1EVT). We used Matt structural alignment program to do this job. Matt performs the structural alignment, which minimizes the distance between α-carbon chains of two proteins based on the common structure (α helix). Input file is the structure of single chain antibody in pdb format and output file is a text file that contains the number of amino acids, which are composed of shared structure (Core residue), average distance between alpha carbon chains of two proteins (Core RMSD), the score of similarity, which is calculated by Matt (Raw score), and the probability that this similarity is just a product of random (p-value) and pdb files, which contain the alignment result of single chain antibody with FGF binding domain of FGFR.
</P><H3><SPAN class="mw-headline" id="Result_:_Table"> Result : Table</SPAN></H3><UL><LI><B>Name</B> -- Except 16A1(Name of antibody itself), all name of single chain antibodies are come from PDB ID of its sources.
</LI><LI> A row filled with blue is for our single chain antibody 16A1.
</LI><LI> Rows filled with green are for single chain antibodies for comparison.
</LI><LI> Rows filled with brown are for random proteins for comparison
</LI></UL></TD></TR><TR align="left" bgcolor="#CCCCCC"><TH> Name
</TH><TH>Core Residues</TH><TH>Core RMSD</TH><TH>Raw Score</TH><TH>P-value
</TH></TR><TR bgcolor="#C3C4D8"><TD> 16A1
</TD><TD> 86
</TD><TD> 2.139
</TD><TD> 89.893
</TD><TD> 0.00443
</TD></TR><TR bgcolor="palegreen"><TD> 2VXT
</TD><TD> 95
</TD><TD> 2.676
</TD><TD> 84.306
</TD><TD> 0.000222
</TD></TR><TR bgcolor="palegreen"><TD> 2VXU
</TD><TD> 98
</TD><TD> 2.563
</TD><TD> 92.118
</TD><TD> 0.0000823
</TD></TR><TR bgcolor="palegreen"><TD> 2VXV
</TD><TD> 77
</TD><TD> 2.64
</TD><TD> 52.269
</TD><TD> 0.0644
</TD></TR><TR bgcolor="palegreen"><TD> 2ZKH
</TD><TD> 96
</TD><TD> 2.855
</TD><TD> 93.411
</TD><TD> 0.001
</TD></TR><TR bgcolor="palegreen"><TD> 3AAZ
</TD><TD> 84
</TD><TD> 4.197
</TD><TD> 68.078
</TD><TD> 0.0642
</TD></TR><TR bgcolor="palegreen"><TD> 3D69
</TD><TD> 137
</TD><TD> 12.224
</TD><TD> 98.134
</TD><TD> 0.00541
</TD></TR><TR bgcolor="palegreen"><TD> 3EO9
</TD><TD> 80
</TD><TD> 2.282
</TD><TD> 84.665
</TD><TD> 0.00295
</TD></TR><TR bgcolor="palegreen"><TD> 3EOA
</TD><TD> 116
</TD><TD> 10.402
</TD><TD> 98.544
</TD><TD> 0.0547
</TD></TR><TR bgcolor="palegreen"><TD> 3EOB
</TD><TD> 68
</TD><TD> 4.685
</TD><TD> 56.357
</TD><TD> 0.3753
</TD></TR><TR bgcolor="palegreen"><TD> 3EYV
</TD><TD> 85
</TD><TD> 4.704
</TD><TD> 66.669
</TD><TD> 0.1653
</TD></TR><TR bgcolor="palegreen"><TD> 3FMG
</TD><TD> 81
</TD><TD> 2.629
</TD><TD> 73.298
</TD><TD> 0.0129
</TD></TR><TR bgcolor="palegreen"><TD> 3FOG
</TD><TD> 90
</TD><TD> 7.243
</TD><TD> 74.031
</TD><TD> 0.0785
</TD></TR><TR bgcolor="palegreen"><TD> 3G6D
</TD><TD> 72
</TD><TD> 4.403
</TD><TD> 69.774
</TD><TD> 0.1281
</TD></TR><TR bgcolor="palegreen"><TD> 3GBM
</TD><TD> 119
</TD><TD> 10.331
</TD><TD> 115.929
</TD><TD> 0.0314
</TD></TR><TR bgcolor="palegreen"><TD> 3GBN
</TD><TD> 77
</TD><TD> 2.788
</TD><TD> 57.204
</TD><TD> 0.0592
</TD></TR><TR bgcolor="palegreen"><TD> 3GHB
</TD><TD> 101
</TD><TD> 12.861
</TD><TD> 81.189
</TD><TD> 0.3984
</TD></TR><TR bgcolor="palegreen"><TD> 3GHE
</TD><TD> 96
</TD><TD> 3.302
</TD><TD> 109.208
</TD><TD> 0.00117
</TD></TR><TR bgcolor="palegreen"><TD> 3GI8
</TD><TD> 127
</TD><TD> 6.837
</TD><TD> 113.467
</TD><TD> 0.00038
</TD></TR><TR bgcolor="palegreen"><TD> 3GI9
</TD><TD> 52
</TD><TD> 3.534
</TD><TD> 43.56
</TD><TD> 0.166
</TD></TR><TR bgcolor="palegreen"><TD> 3GIZ
</TD><TD> 89
</TD><TD> 8.217
</TD><TD> 68.702
</TD><TD> 0.3172
</TD></TR><TR bgcolor="palegreen"><TD> 3GK8
</TD><TD> 91
</TD><TD> 6.84
</TD><TD> 84.048
</TD><TD> 0.00811
</TD></TR><TR bgcolor="palegreen"><TD> 3GKW
</TD><TD> 74
</TD><TD> 3.205
</TD><TD> 47.096
</TD><TD> 0.1215
</TD></TR><TR bgcolor="palegreen"><TD> 3GNM
</TD><TD> 135
</TD><TD> 4.739
</TD><TD> 103.812
</TD><TD> 0.00000297
</TD></TR><TR bgcolor="palegreen"><TD> 3GO1
</TD><TD> 103
</TD><TD> 8.842
</TD><TD> 77.631
</TD><TD> 0.2587
</TD></TR><TR bgcolor="palegreen"><TD> 3GRW
</TD><TD> 107
</TD><TD> 9.819
</TD><TD> 108.405
</TD><TD> 0.0449
</TD></TR><TR bgcolor="palegreen"><TD> 3H42
</TD><TD> 69
</TD><TD> 4.389
</TD><TD> 59.165
</TD><TD> 0.3421
</TD></TR><TR bgcolor="palegreen"><TD> 3HC0
</TD><TD> 67
</TD><TD> 3.065
</TD><TD> 40.982
</TD><TD> 0.1028
</TD></TR><TR bgcolor="palegreen"><TD> 3HC3
</TD><TD> 67
</TD><TD> 3.173
</TD><TD> 49.109
</TD><TD> 0.0944
</TD></TR><TR bgcolor="palegreen"><TD> 3HC4
</TD><TD> 36
</TD><TD> 1.998
</TD><TD> 42.538
</TD><TD> 0.206
</TD></TR><TR bgcolor="palegreen"><TD> 3HI5
</TD><TD> 77
</TD><TD> 3.222
</TD><TD> 58.096
</TD><TD> 0.1832
</TD></TR><TR bgcolor="palegreen"><TD> 3HI6
</TD><TD> 81
</TD><TD> 3.646
</TD><TD>63.526
</TD><TD> 0.0394
</TD></TR><TR bgcolor="palegreen"><TD> 3HMW
</TD><TD> 83
</TD><TD> 4.159
</TD><TD> 62.598
</TD><TD> 0.1976
</TD></TR><TR bgcolor="palegreen"><TD> 3HMX
</TD><TD> 70
</TD><TD> 5.215
</TD><TD> 62.814
</TD><TD> 0.2228
</TD></TR><TR bgcolor="palegreen"><TD> 3HNT
</TD><TD> 84
</TD><TD> 2.62
</TD><TD> 74.666
</TD><TD> 0.00118
</TD></TR><TR bgcolor="palegreen"><TD> 3HNV
</TD><TD> 84
</TD><TD> 2.998
</TD><TD> 77.17
</TD><TD> 0.00374
</TD></TR><TR bgcolor="palegreen"><TD> 3HR5
</TD><TD> 78
</TD><TD> 4.158
</TD><TD> 58.026
</TD><TD> 0.2978
</TD></TR><TR bgcolor="palegreen"><TD> 3I50
</TD><TD> 65
</TD><TD> 2.206
</TD><TD> 52.252
</TD><TD> 0.1156
</TD></TR><TR bgcolor="palegreen"><TD> 3I9G
</TD><TD> 84
</TD><TD> 5.026
</TD><TD> 64.155
</TD><TD> 0.1506
</TD></TR><TR bgcolor="palegreen"><TD> 3IU3
</TD><TD> 75
</TD><TD> 3.192
</TD><TD> 56.006
</TD><TD> 0.2552
</TD></TR><TR bgcolor="palegreen"><TD> 3IXT
</TD><TD> 78
</TD><TD> 2.951
</TD><TD> 68.627
</TD><TD> 0.0378
</TD></TR><TR bgcolor="palegreen"><TD> 3KDM
</TD><TD> 87
</TD><TD> 4.273
</TD><TD> 65.722
</TD><TD> 0.1683
</TD></TR><TR bgcolor="palegreen"><TD> 3KS0
</TD><TD> 81
</TD><TD> 3.039
</TD><TD> 55.562
</TD><TD> 0.0192
</TD></TR><TR bgcolor="palegreen"><TD> 3KYK
</TD><TD> 88
</TD><TD> 4.728
</TD><TD> 68.681
</TD><TD> 0.1016
</TD></TR><TR bgcolor="palegreen"><TD> 3KYM
</TD><TD> 85
</TD><TD> 6.52
</TD><TD> 66.102
</TD><TD> 0.5135
</TD></TR><TR bgcolor="palegreen"><TD> 3L1O
</TD><TD> 93
</TD><TD> 2.545
</TD><TD> 85.909
</TD><TD> 0.0014
</TD></TR><TR bgcolor="palegreen"><TD> 3L5W
</TD><TD> 89
</TD><TD> 4.925
</TD><TD> 72.178
</TD><TD> 0.0637
</TD></TR><TR bgcolor="palegreen"><TD> 3L5X
</TD><TD> 88
</TD><TD> 4.796
</TD><TD> 65.262
</TD><TD> 0.1111
</TD></TR><TR bgcolor="palegreen"><TD> 3L5Y
</TD><TD> 64
</TD><TD> 4.689
</TD><TD> 65.227
</TD><TD> 0.2007
</TD></TR><TR bgcolor="palegreen"><TD> 3L95
</TD><TD> 73
</TD><TD> 5.105
</TD><TD> 70.312
</TD><TD> 0.0515
</TD></TR><TR bgcolor="palegreen"><TD> 3LMJ
</TD><TD> 76
</TD><TD> 2.699
</TD><TD> 56.627
</TD><TD> 0.0237
</TD></TR><TR bgcolor="palegreen"><TD> 3LQA
</TD><TD> 75
</TD><TD> 8.371
</TD><TD> 58.374
</TD><TD> 0.8315
</TD></TR><TR bgcolor="palegreen"><TD> 3LS4
</TD><TD> 79
</TD><TD> 2.458
</TD><TD> 72.68
</TD><TD> 0.00694
</TD></TR><TR bgcolor="palegreen"><TD> 3LS5
</TD><TD> 81
</TD><TD> 2.443
</TD><TD> 73.043
</TD><TD> 0.0014
</TD></TR><TR bgcolor="palegreen"><TD> 3LZF
</TD><TD> 96
</TD><TD> 2.468
</TD><TD> 93.292
</TD><TD> 0.00155
</TD></TR><TR bgcolor="palegreen"><TD> 3MLR
</TD><TD> 71
</TD><TD> 2.574
</TD><TD> 64.979
</TD><TD> 0.0165
</TD></TR><TR bgcolor="palegreen"><TD> 3MLS
</TD><TD> 75
</TD><TD> 2.534
</TD><TD> 57.386
</TD><TD> 0.0202
</TD></TR><TR bgcolor="palegreen"><TD> 3MLU
</TD><TD> 127
</TD><TD> 10.025
</TD><TD> 73.766
</TD><TD> 0.0305
</TD></TR><TR bgcolor="palegreen"><TD> 3MLV
</TD><TD> 75
</TD><TD> 2.454
</TD><TD> 62.321
</TD><TD> 0.0218
</TD></TR><TR bgcolor="palegreen"><TD> 3MLW
</TD><TD> 83
</TD><TD> 8.94
</TD><TD> 59.029
</TD><TD> 0.4846
</TD></TR><TR bgcolor="palegreen"><TD> 3MLX
</TD><TD> 118
</TD><TD> 11.485
</TD><TD> 105.391
</TD><TD> 0.0961
</TD></TR><TR bgcolor="palegreen"><TD> 3MLY
</TD><TD> 84
</TD><TD> 4.704
</TD><TD> 63.962
</TD><TD> 0.1023
</TD></TR><TR bgcolor="palegreen"><TD> 3MLZ
</TD><TD> 82
</TD><TD> 3.995
</TD><TD> 64.563
</TD><TD> 0.123
</TD></TR><TR bgcolor="palegreen"><TD> 3MUG
</TD><TD> 79
</TD><TD> 2.827
</TD><TD> 68.72
</TD><TD> 0.0223
</TD></TR><TR bgcolor="palegreen"><TD> 3MXV
</TD><TD> 125
</TD><TD> 10.857
</TD><TD> 121.911
</TD><TD> 0.01
</TD></TR><TR bgcolor="palegreen"><TD> 3MXW
</TD><TD> 102
</TD><TD> 11.262
</TD><TD> 78.348
</TD><TD> 0.4213
</TD></TR><TR bgcolor="darkkhaki"><TD> 1DSE
</TD><TD> 33
</TD><TD> 4.821
</TD><TD> 25.400
</TD><TD> 0.8574
</TD></TR><TR bgcolor="darkkhaki"><TD> 2KR2
</TD><TD> 75
</TD><TD> 4.13
</TD><TD> 43.618
</TD><TD> 0.2895
</TD></TR><TR bgcolor="darkkhaki"><TD> 2WVN
</TD><TD> 19
</TD><TD> 2.445
</TD><TD> 11.423
</TD><TD> 0.7067
</TD></TR><TR bgcolor="darkkhaki"><TD> 2WVP
</TD><TD> 57
</TD><TD> 6.861
</TD><TD> 52.39
</TD><TD> 0.6536
</TD></TR><TR bgcolor="darkkhaki"><TD> 2X55
</TD><TD> 37
</TD><TD> 2.307
</TD><TD> 45.547
</TD><TD> 0.2647
</TD></TR><TR bgcolor="darkkhaki"><TD> 2X56
</TD><TD> 37
</TD><TD> 2.269
</TD><TD> 44.949
</TD><TD> 0.2645
</TD></TR><TR bgcolor="darkkhaki"><TD> 2X5E
</TD><TD> 45
</TD><TD> 3.922
</TD><TD> 22.009
</TD><TD> 0.636
</TD></TR><TR bgcolor="darkkhaki"><TD> 2X6A
</TD><TD> 69
</TD><TD> 2.518
</TD><TD> 67.2
</TD><TD> 0.0277
</TD></TR><TR bgcolor="darkkhaki"><TD> 2X6B
</TD><TD> 72
</TD><TD> 2.694
</TD><TD> 66.076
</TD><TD> 0.0433
</TD></TR><TR bgcolor="darkkhaki"><TD> 2X6C
</TD><TD> 69
</TD><TD> 2.521
</TD><TD> 68.046
</TD><TD> 0.0277
</TD></TR><TR bgcolor="darkkhaki"><TD> 2X6D
</TD><TD> 31
</TD><TD> 2.19
</TD><TD> 27.183
</TD><TD> 0.3866
</TD></TR><TR bgcolor="darkkhaki"><TD> 2X7M
</TD><TD> 39
</TD><TD> 5.697
</TD><TD> 22.742
</TD><TD> 0.9606
</TD></TR><TR bgcolor="darkkhaki"><TD> 2X8D
</TD><TD> 31
</TD><TD> 3.335
</TD><TD> 28.656
</TD><TD> 0.6961
</TD></TR><TR bgcolor="darkkhaki"><TD> 2X9Z
</TD><TD> 79
</TD><TD> 3.027
</TD><TD> 70.217
</TD><TD> 0.0131
</TD></TR><TR bgcolor="darkkhaki"><TD> 2XB7
</TD><TD> 44
</TD><TD> 5.444
</TD><TD> 34.426
</TD><TD> 0.746
</TD></TR><TR bgcolor="darkkhaki"><TD> 2XCH
</TD><TD> 32
</TD><TD> 2.435
</TD><TD> 26.269
</TD><TD> 0.427
</TD></TR><TR bgcolor="darkkhaki"><TD> 2XCK
</TD><TD> 34
</TD><TD> 3.479
</TD><TD> 33.038
</TD><TD> 0.6521
</TD></TR><TR bgcolor="darkkhaki"><TD> 2XD6
</TD><TD> 30
</TD><TD> 2.056
</TD><TD> 29.429
</TD><TD> 0.3361
</TD></TR><TR bgcolor="darkkhaki"><TD> 2XEY
</TD><TD> 29
</TD><TD> 2.732
</TD><TD> 28.271
</TD><TD> 0.6627
</TD></TR><TR bgcolor="darkkhaki"><TD> 2XEZ
</TD><TD> 28
</TD><TD> 2.938
</TD><TD> 29.547
</TD><TD> 0.6961
</TD></TR><TR bgcolor="darkkhaki"><TD> 2XF0
</TD><TD> 27
</TD><TD> 1.682
</TD><TD> 24.751
</TD><TD> 0.4138
</TD></TR><TR bgcolor="darkkhaki"><TD> 2XF4
</TD><TD> 43
</TD><TD> 3.488
</TD><TD> 42.04
</TD><TD> 0.5361
</TD></TR><TR bgcolor="darkkhaki"><TD> 2XGW
</TD><TD> 23
</TD><TD> 3.213
</TD><TD> 13.37
</TD><TD> 0.8533
</TD></TR><TR bgcolor="darkkhaki"><TD> 2XIL
</TD><TD> 37
</TD><TD> 5.304
</TD><TD> 24.721
</TD><TD> 0.8651
</TD></TR><TR bgcolor="darkkhaki"><TD> 2XJ5
</TD><TD> 37
</TD><TD> 5.302
</TD><TD> 24.937
</TD><TD> 0.865
</TD></TR><TR bgcolor="darkkhaki"><TD> 2XJ8
</TD><TD> 41
</TD><TD> 5.224
</TD><TD> 24.707
</TD><TD> 0.9166
</TD></TR><TR bgcolor="darkkhaki"><TD> 2XJG
</TD><TD> 54
</TD><TD> 7.366
</TD><TD> 41.923
</TD><TD> 0.899
</TD></TR><TR bgcolor="darkkhaki"><TD> 2XJX
</TD><TD> 47
</TD><TD> 7.351
</TD><TD> 39.362
</TD><TD> 0.9192
</TD></TR><TR bgcolor="darkkhaki"><TD> 2XL7
</TD><TD> 35
</TD><TD> 2.871
</TD><TD> 38.17
</TD><TD> 0.5818
</TD></TR><TR bgcolor="darkkhaki"><TD> 3A5J
</TD><TD> 44
</TD><TD> 3.577
</TD><TD> 38.745
</TD><TD> 0.5856
</TD></TR><TR bgcolor="darkkhaki"><TD> 3A5K
</TD><TD> 44
</TD><TD> 3.107
</TD><TD> 43.763
</TD><TD> 0.4649
</TD></TR><TR bgcolor="darkkhaki"><TD> 3A7N
</TD><TD> 30
</TD><TD> 2.989
</TD><TD> 16.076
</TD><TD> 0.6424
</TD></TR><TR bgcolor="darkkhaki"><TD> 3HJL
</TD><TD> 43
</TD><TD> 3.221
</TD><TD> 16.018
</TD><TD> 0.7916
</TD></TR><TR bgcolor="darkkhaki"><TD> 3HN4
</TD><TD> 32
</TD><TD> 2.071
</TD><TD> 30.695
</TD><TD> 0.3804
</TD></TR><TR bgcolor="darkkhaki"><TD> 3HRA
</TD><TD> 16
</TD><TD> 1.764
</TD><TD> 12.557
</TD><TD> 0.6734
</TD></TR><TR bgcolor="darkkhaki"><TD> 3I0C
</TD><TD> 30
</TD><TD> 2.273
</TD><TD> 22.59
</TD><TD> 0.4501
</TD></TR><TR bgcolor="darkkhaki"><TD> 3I0D
</TD><TD> 29
</TD><TD> 2.318
</TD><TD> 22.849
</TD><TD> 0.4659
</TD></TR><TR bgcolor="darkkhaki"><TD> 3I0E
</TD><TD> 31
</TD><TD> 2.238
</TD><TD> 27.618
</TD><TD> 0.4123
</TD></TR><TR bgcolor="darkkhaki"><TD> 3I0H
</TD><TD> 27
</TD><TD> 2.07
</TD><TD> 25.607
</TD><TD> 0.4119
</TD></TR><TR bgcolor="darkkhaki"><TD> 3I0I
</TD><TD> 30
</TD><TD> 2.294
</TD><TD> 23.368
</TD><TD> 0.4412
</TD></TR><TR bgcolor="darkkhaki"><TD> 3IG0
</TD><TD> 21
</TD><TD> 1.55
</TD><TD> 23.451
</TD><TD> 0.4946</TD></TR><TR bgcolor="darkkhaki"><TD> 3IJZ
</TD><TD> 32
</TD><TD> 2.372
</TD><TD> 24.986
</TD><TD> 0.4372
</TD></TR><TR bgcolor="darkkhaki"><TD> 3IK0
</TD><TD> 32
</TD><TD> 2.375
</TD><TD> 25.483
</TD><TD> 0.4369
</TD></TR><TR bgcolor="darkkhaki"><TD> 3IK1
</TD><TD> 32
</TD><TD> 2.366
</TD><TD> 24.831
</TD><TD> 0.4407
</TD></TR><TR bgcolor="darkkhaki"><TD> 3K4Z
</TD><TD> 74
</TD><TD> 3.632
</TD><TD> 54.636
</TD><TD> 0.0337
</TD></TR><TR bgcolor="darkkhaki"><TD> 3KWA
</TD><TD> 42
</TD><TD> 2.624
</TD><TD> 34.064
</TD><TD> 0.2795
</TD></TR><TR bgcolor="darkkhaki"><TD> 3M0J
</TD><TD> 33
</TD><TD> 3.338
</TD><TD> 22.992
</TD><TD> 0.7442
</TD></TR><TR bgcolor="darkkhaki"><TD> 3M0K
</TD><TD> 33
</TD><TD> 3.347
</TD><TD> 22.876
</TD><TD> 0.7458
</TD></TR><TR bgcolor="darkkhaki"><TD> 3M2W
</TD><TD> 38
</TD><TD> 5.231
</TD><TD> 32.085
</TD><TD> 0.795
</TD></TR><TR bgcolor="darkkhaki"><TD> 3M7G
</TD><TD> 21
</TD><TD> 1.537
</TD><TD> 15.84
</TD><TD> 0.5617
</TD></TR><TR bgcolor="darkkhaki"><TD> 3MBL
</TD><TD> 40
</TD><TD> 3.433
</TD><TD> 34.5
</TD><TD> 0.3949
</TD></TR><TR bgcolor="darkkhaki"><TD> 3MBR
</TD><TD> 36
</TD><TD> 4.071
</TD><TD> 34.958
</TD><TD> 0.7351
</TD></TR><TR bgcolor="darkkhaki"><TD> 3MNR
</TD><TD> 37
</TD><TD> 3.391
</TD><TD> 28.647
</TD><TD> 0.6731
</TD></TR><TR bgcolor="darkkhaki"><TD> 3MRW
</TD><TD> 28
</TD><TD> 1.956
</TD><TD> 30.63
</TD><TD> 0.3969
</TD></TR><TR bgcolor="darkkhaki"><TD> 3MU7
</TD><TD> 29
</TD><TD> 2.567
</TD><TD> 21.007
</TD><TD> 0.4913
</TD></TR><TR bgcolor="darkkhaki"><TD> 3MW3
</TD><TD> 47
</TD><TD> 2.665
</TD><TD> 38.629
</TD><TD> 0.3011
</TD></TR><TR bgcolor="darkkhaki"><TD> 3MXT
</TD><TD> 62
</TD><TD> 2.828
</TD><TD> 34.447
</TD><TD> 0.2004
</TD></TR><TR bgcolor="darkkhaki"><TD> 3MY3
</TD><TD> 28
</TD><TD> 2.995
</TD><TD> 10.878
</TD><TD> 0.5999
</TD></TR><TR bgcolor="darkkhaki"><TD> 3N0R
</TD><TD> 33
</TD><TD> 3.11
</TD><TD> 18.853
</TD><TD> 0.6775
</TD></TR><TR bgcolor="darkkhaki"><TD> 3N11
</TD><TD> 19
</TD><TD> 2.307
</TD><TD> 17.508</TD><TD> 0.6986
</TD></TR><TR bgcolor="darkkhaki"><TD> 3N4T
</TD><TD> 30
</TD><TD> 2.657
</TD><TD> 26.947
</TD><TD> 0.4295
</TD></TR><TR bgcolor="darkkhaki"><TD> 3NE8
</TD><TD> 41
</TD><TD> 4.674
</TD><TD> 25.792
</TD><TD> 0.8496
</TD></TR><TR bgcolor="darkkhaki"><TD> 3NGW
</TD><TD> 44
</TD><TD> 4.214
</TD><TD> 23.603
</TD><TD> 0.7468
</TD></TR><TR bgcolor="darkkhaki"><TD> 3NIZ
</TD><TD> 41
</TD><TD> 3.035
</TD><TD> 32.651
</TD><TD> 0.3067
</TD></TR><TR bgcolor="darkkhaki"><TD> 3NQK
</TD><TD> 98
</TD><TD> 4.418
</TD><TD> 64.835
</TD><TD> 0.0125
</TD></TR><TR bgcolor="darkkhaki"><TD> 3NWO
</TD><TD> 35
</TD><TD> 2.127
</TD><TD> 30.301
</TD><TD> 0.3048
</TD></TR><TR bgcolor="darkkhaki"><TD> 3NXH
</TD><TD> 38
</TD><TD> 2.224
</TD><TD> 38.17
</TD><TD> 0.3394
</TD></TR><TR bgcolor="darkkhaki"><TD> 3O3T
</TD><TD> 38
</TD><TD> 2.854
</TD><TD> 23.219
</TD><TD> 0.4752
</TD></TR><TR bgcolor="darkkhaki"><TD> 3O5X
</TD><TD> 65
</TD><TD> 10.8
</TD><TD> 44.035
</TD><TD> 0.9368
</TD></TR><TR bgcolor="darkkhaki"><TD> 3O6C
</TD><TD> 41
</TD><TD> 3.624
</TD><TD> 22.934
</TD><TD> 0.6685
</TD></TR><TR bgcolor="darkkhaki"><TD> 3O6D
</TD><TD> 41
</TD><TD> 3.602
</TD><TD> 23.293
</TD><TD> 0.6638
</TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Result_:_Figures"> Result : Figures</SPAN></H3><P>Using arranged pdb files and protein 3D structure drawing program - PyMOL, again, we predict structural similarity between the structure of FGF binding domain and that of single chain antibodies associated with the table. In order to show the structures clearly, we control the shape setting 'cartoon' and 'chain'.  <FONT color="#00c5c5"><B> Cyan Color </B></FONT>represents <B>the structure of FGF binding domain</B> and <FONT color="#25b925"><B> Green Color </B></FONT> is for <B>the comparative antibodies</B>. In the result, we can easily see all antibodies' structures are very similar, completely or symmetrically.
</P><H3><SPAN class="mw-headline" id="Result_Analysis">Result Analysis</SPAN></H3><P>The table above shows the result of alignments of 65 single chain antibodies for control group and 1 single chain antibody for our real experiment (16A1). Higher core residue and Raw score, lower core RMSD, and p-value mean more similarity. 16A1 antibody shows 26th highest in core residue, 2nd lowest in core RMSD, 13th highest Raw score and 13th lowest p-value. These result means that our 16A1 antibody has higher similarity than average antibodies have.
</P>
In general, Core RMSD value of randomly selected small protein (colored brown) is lower than that of single chain antibody (colored green). It may seem confusing, but lower Core RMSD is caused by RMSD calculation of Matt. Matt calculates RMSD for not whole proteins, but only shared structures. Therefore, we should check if protein pairs with lower Core RMSD have enough number of Core residues.<DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:KAIST-Korea/Project/StructureAlignment">http://2010.igem.org/Team:KAIST-Korea/Project/StructureAlignment</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:KAIST-Korea/Project/StructureAlignment" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:KAIST-Korea/Project/StructureAlignment" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:KAIST-Korea/Project/StructureAlignment&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:KAIST-Korea/Project/StructureAlignment&amp;oldid=48319" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></BODY></HTML>