"
Page
Discussion
View source
History
teams
Log in
 
Team:KAIST-Korea/Project/Methods
From 2010.igem.org
 
Cloning Plan
Aim
Build DiscoverY yeast: transform genes to S.pombe and turn it into the yeast we want.
BioBricks to submit: insert genes to vectors provided by iGEM (pSB1C3).
Protocol
Design primer (1 day) – Team
Make primer (2~3 d) – Bioneer
Gene: PCR / Vector: E.coli transformation, raise, prep -> Both: Purification (1 d) – Team
Both: Restriction enzyme treatment; 2 cut -> Purification (1 d) – Team
TA cloning (1 d) – Team
Insert RE treated gene to RE treated vector -> Ligation -> E.coli transformation, raise, prep -> Loading (2 d) – Team
Sequencing (3 d) – Bioneer
Yeast transformation (1 d) – Bioneer
Yeast growth (1 week) – Bioneer
Confirm result (2 d) - Team
cf) TA cloning
For safer storage and confirmation purpose
Insert RE treated gene to TA cloning vector -> Ligation -> E.coli transformation -> Prep -> Loading (2 d) – team
PCR
  To manipulate gene products obtained from gene-bank, our team used PCR/Real-Time PCR methods. Commercially obtained genes contain few more base pairs added at front and back site. As for FGPR, the receptor protein of human has human-specific signal peptide at its starting point, and this region had to be replaced by S.pombe specific signal peptide to express the protein at membrane region. To do this, PCR primers containing signal peptide region of S.pombe were synthesized by method describeds below. Then, FGPR gene was amplified by PCR to get gene product whose signal region is replaced. Also, additional sequence at rear site of gene was removed. STAT gene went through similar process; removing additional region.
STAT
  We need two kinds of brick to do PCR. One is STAT and the other is V_STAT. STAT would be cut by EcoRI and inserted into pBS1C3 for the submission. V_STAT would be cut by NdeI and BamHI sites and inserted into pREP41 vector for the expression
Fusion Antibody Receptor
  We need six bricks to do PCR. They are B_FGFR, F_FGFR, SIG, IG, SIG + IG, and Ab. B_FGFR is a orignal back part of FGFR to operate our system, and F_FGFR is a handdled front part of FGFR assembled our biobrick. SIG is a signal peptide, and IG is IG-like-1, Ab is IG-like-2,and 3 we use. We will use B_FGFR and F_FGFR for the expression and not only these but also the others for the submission.
Homologous Recombination
  We need five bricks to do PCR. They are Fusion Antibody Receptor, 5' UTR-Receptor, 5' UTR-Receptor + Selection marker, 5' UTR-Recptor +  Selection Marker + 3' UTR, and Homologous Recombination. Fusion antibody receptor needs to be linked together with UTR and selection marker in order to be inserted into genome by homologous recombination. "Homologous Recombination" is inserted into genome by homologous recombination.
GAS promoter + GFP
  We need three bricks to do PCR. They are Proximal Promoter, Core Promoter+GFP, and PstI+Proximal Promoter+Core Promoter+GFP+NdeI. PstI and NdeI are restiriction enzymes, and Proximal + Core promoter is GAS promoter to express GFP
Oligo Synthesis
  Oligo-synthesis was supported by Bioneer.
  We requested synthesis of fusion antibody gene that is mainly used in our project.
  Process of oligo-synthesis by Bioneer is summarized below.
  Codon optimization Oligo Design S/W
High-efficient optimiztion software made Bioinformatics technology of KAIST.
  Mega-base Oligo Synthesizer
Mass oligo synthesizer which can synthesize 25 million base pair of oligonucleotide.
  In addition, through the following processes, Bioneer completed our oligo-synthesis perfectly.
  Sequencing Service
High-accuracy sequencing service
  AccuRapid Cell-Free Protein Expression Kit
Cell free protein expression system
  Gene Synthesis Kit
The world’s first one-stop gene synthesis kit
Homologous Recombination
  Homologous recombination is a type of genetic recombination in which nucleotide sequences are exchanged between two similar or identical molecules of DNA. It is most widely used by cells to accurately repair harmful breaks that occur on both strands of DNA, known as double-strand breaks. Homologous recombination also produces new combinations of DNA sequences during meiosis, the process performed by many eukaryotes like animals and plants: production of sperm and egg cells. These new combinations of DNA promote genetic variations in offspring, which in turn enable populations to evolve. Homologous recombination is also used in horizontal gene transfer to exchange genetic material between different strains and species of bacteria and viruses.
  We use this technique to generate the yeast S. pombe that we want. Diploid deletion mutants in the S. pombe genome were systematically constructed with targeted mutagenesis at each target ORF. The chromosomal location of the ORFs and their DNA sequence information were obtained from the public fission yeast database at the Welcome Trust Sanger Institute. The deletion cassette module contains a selection marker (Fig. 1), tag sequences (molecular barcodes), and the sequences for homologous recombination (Fig 2). The deletion cassette modules were constructed by PCR with well-designed primers. The cassettes were transformed into the S. pombe, SP286 diploid host strain (h+/ h+, ade6-M210/ade6-M216 ura4-D18/ura4-D18 leu1-32/leu1-32). The result was screened; deletion of the target ORF was confirmed by G418 antibiotic selection (Fig 3). If chromosomal integration occurs properly via homologous recombination at the target ORF in a colony, that colony would obtain antibiotic resistance from the KanMX4 selection marker gene.
TA cloning
  TA cloning is one of the subcloning techniques. This technique starts from product of PCR that uses taq DNA polymerase. Taq DNA polymerase tends to add one adenine overhang to the 3` end of PCR product. This result occurs because there is lack of ability to proofread from 3` to 5`. Therefore after PCR process, we can get amplified DNA sequences that has single adenine overhang on 3` end.
In a mean time, it needs target vector for TA cloning. Target vector can be prepared by cutting target vector with blunt-end restriction enzyme. After this process, it is tailed with ddTTP using terminal transferase. Then target vector gets single thymine on each 3` ends. Finally, adenine on PCR product and thymine on target vector can bind together. This way, we can ligate PCR product and vector together without restriction enzyme.
Biggest advantage of TA cloning is that we don’t need restriction enzyme for ligation process. This fact allows us to save time and money from making primer that has restriction enzyme site. In addition, this process is much simpler and faster than traditional subcloing techniques. Finally, TA cloning is more stabilized way of storing gene than storing it on its own. But TA cloning can’t regulate ligating direction and this can generate problem during DNA expression stage.
References
Bioneer PCR service:
http://www.bioneer.co.kr/product/product_sub.jsp?pclassCode=1
Bioneer Oligo sythetis service
Put reference list here
Retrieved from " http://2010.igem.org/Team:KAIST-Korea/Project/Methods "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
