<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_KAIST-Korea_Project_Methods"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:KAIST-Korea/Project/Methods">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:KAIST-Korea/Project/Methods&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:KAIST-Korea/Project/Methods&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:KAIST-Korea/Project/Methods&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:KAIST-Korea/Project/Methods" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:KAIST-Korea/Project/Methods</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><DIV id="dashed_line"> </DIV><DIV id="wrapper"><TABLE width="100%" border="0" cellpadding="20px"><TBODY><TR><TD valign="top" width="75%"><H2><SPAN class="mw-headline" id="Cloning_Plan"><B> Cloning Plan</B></SPAN></H2><P><B>Aim</B></P><OL><LI>	Build DiscoverY yeast: transform genes to S.pombe and turn it into the yeast we want.
</LI><LI>	BioBricks to submit: insert genes to vectors provided by iGEM (pSB1C3).
</LI></OL><P><B>Protocol</B></P><OL><LI>	Design primer (1 day) – Team
</LI><LI>	Make primer (2~3 d) – Bioneer
</LI><LI>	Gene: PCR / Vector: E.coli transformation, raise, prep -&gt; Both: Purification (1 d) – Team 
</LI><LI>	Both: Restriction enzyme treatment; 2 cut -&gt; Purification (1 d) – Team
</LI><LI>	TA cloning (1 d) – Team
</LI><LI>	Insert RE treated gene to RE treated vector -&gt; Ligation -&gt; E.coli transformation, raise, prep -&gt; Loading (2 d) – Team
</LI><LI>	Sequencing (3 d) – Bioneer 
</LI><LI>	Yeast transformation (1 d) – Bioneer
</LI><LI>	Yeast growth (1 week) – Bioneer
</LI><LI>	Confirm result (2 d) - Team
</LI></OL><P>
cf) TA cloning

For safer storage and confirmation purpose

Insert RE treated gene to TA cloning vector -&gt; Ligation -&gt; E.coli transformation -&gt; Prep -&gt; Loading (2 d) – team
</P><H2><SPAN class="mw-headline" id="PCR"><B> PCR </B></SPAN></H2><P>  To manipulate gene products obtained from gene-bank, our team used PCR/Real-Time PCR methods. Commercially obtained genes contain few more base pairs added at front and back site. As for FGPR, the receptor protein of human has human-specific signal peptide at its starting point, and this region had to be replaced by S.pombe specific signal peptide to express the protein at membrane region. To do this, PCR primers containing signal peptide region of S.pombe were synthesized by method describeds below. Then, FGPR gene was amplified by PCR to get gene product whose signal region is replaced. Also, additional sequence at rear site of gene was removed. STAT gene went through similar process; removing additional region.<B>STAT</B>  We need two kinds of brick to do PCR. One is STAT and the other is V_STAT. STAT would be cut by EcoRI and inserted into pBS1C3 for the submission. V_STAT would be cut by NdeI and BamHI sites and inserted into pREP41 vector for the expression<B>Fusion Antibody Receptor</B></P><P>  We need six bricks to do PCR. They are B_FGFR, F_FGFR, SIG, IG, SIG + IG, and Ab. B_FGFR is a orignal back part of FGFR to operate our system, and F_FGFR is a handdled front part of FGFR assembled our biobrick. SIG is a signal peptide, and IG is IG-like-1, Ab is IG-like-2,and 3 we use. We will use B_FGFR and F_FGFR for the expression and not only these but also the others for the submission.<B>Homologous Recombination</B></P><P>  We need five bricks to do PCR. They are Fusion Antibody Receptor, 5' UTR-Receptor, 5' UTR-Receptor + Selection marker, 5' UTR-Recptor +  Selection Marker + 3' UTR, and Homologous Recombination. Fusion antibody receptor needs to be linked together with UTR and selection marker in order to be inserted into genome by homologous recombination. &quot;Homologous Recombination&quot; is inserted into genome by homologous recombination.</P><P><B>GAS promoter + GFP</B></P><P>  We need three bricks to do PCR. They are Proximal Promoter, Core Promoter+GFP, and PstI+Proximal Promoter+Core Promoter+GFP+NdeI. PstI and NdeI are restiriction enzymes, and Proximal + Core promoter is GAS promoter to express GFP
</P><H2><SPAN class="mw-headline" id="Oligo_Synthesis"><B> Oligo Synthesis </B></SPAN></H2><P>  Oligo-synthesis was supported by Bioneer.  We requested synthesis of <B>fusion antibody gene</B> that is mainly used in our project.  Process of oligo-synthesis by Bioneer is summarized below.</P><P><B>  Codon optimization Oligo Design S/W</B></P><DL><DD><UL><LI> High-efficient optimiztion software made Bioinformatics technology of KAIST.</LI></UL></DD></DL><P><B>  Mega-base Oligo Synthesizer</B></P><DL><DD><UL><LI> Mass oligo synthesizer which can synthesize 25 million base pair of oligonucleotide.</LI></UL></DD></DL><P>  In addition, through the following processes, Bioneer completed our oligo-synthesis perfectly.<B>  Sequencing Service</B></P><DL><DD><UL><LI> High-accuracy sequencing service</LI></UL></DD></DL><P><B>  AccuRapid Cell-Free Protein Expression Kit</B></P><DL><DD><UL><LI> Cell free protein expression system</LI></UL></DD></DL><P><B>  Gene Synthesis Kit</B></P><DL><DD><UL><LI> The world’s first one-stop gene synthesis kit</LI></UL></DD></DL><H2><SPAN class="mw-headline" id="Homologous_Recombination"><B> Homologous Recombination</B></SPAN></H2><P>  Homologous recombination is a type of genetic recombination in which nucleotide sequences are exchanged between two similar or identical molecules of DNA. It is most widely used by cells to accurately repair harmful breaks that occur on both strands of DNA, known as double-strand breaks. Homologous recombination also produces new combinations of DNA sequences during meiosis, the process performed by many eukaryotes like animals and plants: production of sperm and egg cells. These new combinations of DNA promote genetic variations in offspring, which in turn enable populations to evolve. Homologous recombination is also used in horizontal gene transfer to exchange genetic material between different strains and species of bacteria and viruses.
</P><P>  We use this technique to generate the yeast S. pombe that we want. Diploid deletion mutants in the S. pombe genome were systematically constructed with targeted mutagenesis at each target ORF. The chromosomal location of the ORFs and their DNA sequence information were obtained from the public fission yeast database at the Welcome Trust Sanger Institute. The deletion cassette module contains a selection marker (Fig. 1), tag sequences (molecular barcodes), and the sequences for homologous recombination (Fig 2). The deletion cassette modules were constructed by PCR with well-designed primers. The cassettes were transformed into the S. pombe, SP286 diploid host strain (h+/ h+, ade6-M210/ade6-M216 ura4-D18/ura4-D18 leu1-32/leu1-32). The result was screened; deletion of the target ORF was confirmed by G418 antibiotic selection (Fig 3). If chromosomal integration occurs properly via homologous recombination at the target ORF in a colony, that colony would obtain antibiotic resistance from the KanMX4 selection marker gene.
</P><H2><SPAN class="mw-headline" id="TA_cloning"><B>TA cloning</B></SPAN></H2><P>  TA cloning is one of the subcloning techniques. This technique starts from product of PCR that uses taq DNA polymerase. Taq DNA polymerase tends to add one adenine overhang to the 3` end of PCR product. This result occurs because there is lack of ability to proofread from 3` to 5`. Therefore after PCR process, we can get amplified DNA sequences that has single adenine overhang on 3` end. 
In a mean time, it needs target vector for TA cloning. Target vector can be prepared by cutting target vector with blunt-end restriction enzyme. After this process, it is tailed with ddTTP using terminal transferase. Then target vector gets single thymine on each 3` ends. Finally, adenine on PCR product and thymine on target vector can bind together. This way, we can ligate PCR product and vector together without restriction enzyme.
Biggest advantage of TA cloning is that we don’t need restriction enzyme for ligation process. This fact allows us to save time and money from making primer that has restriction enzyme site. In addition, this process is much simpler and faster than traditional subcloing techniques. Finally, TA cloning is more stabilized way of storing gene than storing it on its own. But TA cloning can’t regulate ligating direction and this can generate problem during DNA expression stage.</P><H2><SPAN class="mw-headline" id="References"><B>References</B></SPAN></H2><P>Bioneer PCR service: <A href="http://www.bioneer.co.kr/product/product_sub.jsp?pclassCode=1" class="external free" rel="nofollow">http://www.bioneer.co.kr/product/product_sub.jsp?pclassCode=1</A>
Bioneer Oligo sythetis service 
Put reference list here </P></TD></TR></TBODY></TABLE><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:KAIST-Korea/Project/Methods">http://2010.igem.org/Team:KAIST-Korea/Project/Methods</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:KAIST-Korea/Project/Methods" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:KAIST-Korea/Project/Methods" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:KAIST-Korea/Project/Methods&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:KAIST-Korea/Project/Methods&amp;oldid=98838" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></BODY></HTML>