<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_HKU-HKBU_protocols"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:HKU-HKBU/protocols">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:HKU-HKBU/protocols&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:HKU-HKBU/protocols&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:HKU-HKBU/protocols&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:HKU-HKBU/protocols" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:HKU-HKBU/protocols</H1><DIV id="bodyContent"><H3 id="siteSub">From 2009.igem.org</H3><DIV id="wrapper"><DIV id="leftnav"><DIV class="glossymenu" id="glossymenu1"><A class="menuitem" href="https://2009.igem.org/Team:HKU-HKBU">Home</A></DIV><DIV class="glossymenu" id="glossymenu2"><A class="menuitem" href="https://2009.igem.org/Team:HKU-HKBU/Project">Project</A><A class="menuitem" href="https://2009.igem.org/Team:HKU-HKBU/Parts">Parts</A><A class="menuitem submenuheader" href="https://2009.igem.org/Team:HKU-HKBU/Motor_Overview">Device 1:Micro-Motor</A><DIV class="submenu"><UL><LI><A href="https://2009.igem.org/Team:HKU-HKBU/Motor_Overview">Overview</A></LI><LI><A href="https://2009.igem.org/Team:HKU-HKBU/Motor_Design">Design</A></LI><LI><A href="https://2009.igem.org/Team:HKU-HKBU/Motor_Preliminary_Trials">Preliminary Trials</A></LI><LI><A href="https://2009.igem.org/Team:HKU-HKBU/Motor_Methodology">Methodology</A></LI></UL></DIV><A class="menuitem submenuheader" href="https://2009.igem.org/Team:HKU-HKBU/Polar_Expression_Design">Device 2:Direction Controller</A><DIV class="submenu"><UL><LI><A href="https://2009.igem.org/Team:HKU-HKBU/Polar_Expression_Design">Design</A></LI><LI><A href="https://2009.igem.org/Team:HKU-HKBU/Polar_Expression_Results">Results</A></LI><LI><A href="https://2009.igem.org/Team:HKU-HKBU/Polar_Expression_Methodology">Methodology</A></LI></UL></DIV><A class="menuitem submenuheader" href="https://2009.igem.org/Team:HKU-HKBU/Speed_Control_Design">Device 3:Speed Controller</A><DIV class="submenu"><UL><LI><A href="https://2009.igem.org/Team:HKU-HKBU/Speed_Control_Design">Design</A></LI><LI><A href="https://2009.igem.org/Team:HKU-HKBU/Speed_Control_Results">Results</A></LI><LI><A href="https://2009.igem.org/Team:HKU-HKBU/Speed_Control_Methodology">Methodology</A></LI></UL></DIV><A class="menuitem" href="https://2009.igem.org/Team:HKU-HKBU/Assembly">Assembly</A><A class="menuitem" href="https://2009.igem.org/Team:HKU-HKBU/Modeling">Modeling</A><A class="menuitem" href="https://2009.igem.org/Team:HKU-HKBU/Application">Applications</A></DIV><DIV class="glossymenu" id="glossymenu3"><A class="menuitem submenuheader" href="https://2009.igem.org/Team:HKU-HKBU/Human_Practice">Human Practices Advance</A><DIV class="submenu"><UL><LI><A href="https://2009.igem.org/Team:HKU-HKBU/Human_Practice">Overview</A></LI><LI><A href="https://2009.igem.org/Team:HKU-HKBU/Human_Practice_Questionnaire">Questionnaire</A></LI><LI><A href="https://2009.igem.org/Team:HKU-HKBU/Human_Practice_Website">Website</A></LI><LI><A href="https://2009.igem.org/Team:HKU-HKBU/Human_Practice_Interview">Interview</A></LI><LI><A href="https://2009.igem.org/Team:HKU-HKBU/Human_Practice_Lecture">Introductory Lecture</A></LI></UL></DIV></DIV></DIV><DIV id="mainbox"><DIV id="tabs"><UL><LI><A href="https://2009.igem.org/Team:HKU-HKBU/Brainstorming"><SPAN>Brainstorming</SPAN></A></LI><LI><A href="https://2009.igem.org/Team:HKU-HKBU/Team"><SPAN>Team</SPAN></A></LI><LI><A href="https://2009.igem.org/Team:HKU-HKBU/Gallery"><SPAN>Gallery</SPAN></A></LI><LI><A href="https://2009.igem.org/Team:HKU-HKBU/Calendar"><SPAN>Calendar</SPAN></A></LI><LI><A href="https://2009.igem.org/Team:HKU-HKBU/Protocols"><SPAN>Protocols</SPAN></A></LI><LI><A href="https://2009.igem.org/Team:HKU-HKBU/Biosafety"><SPAN>Biosafety</SPAN></A></LI><LI><A href="https://2009.igem.org/Team:HKU-HKBU/Acknowledgments"><SPAN>Acknowledgments</SPAN></A></LI><LI><A href="https://2009.igem.org/Team:HKU-HKBU/Contact"><SPAN>Contact</SPAN></A></LI></UL></DIV><DIV id="wikicontent"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Protocols_.28in_alphabetical_order.29"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Protocols (in alphabetical order)</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Bacteria_Lysis"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Bacteria Lysis</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#BCA_Quantification_Analysis"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">BCA Quantification Analysis</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Competent_Cell_Preparation_for_Electro_Transformation"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Competent Cell Preparation for Electro Transformation</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Electro_Transformation"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">Electro Transformation</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Ligation"><SPAN class="tocnumber">1.5</SPAN><SPAN class="toctext">Ligation</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#Membrane_Biotinylation"><SPAN class="tocnumber">1.6</SPAN><SPAN class="toctext">Membrane Biotinylation</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#Photolithography"><SPAN class="tocnumber">1.7</SPAN><SPAN class="toctext">Photolithography</SPAN></A></LI><LI class="toclevel-2 tocsection-9"><A href="#Pre-culture"><SPAN class="tocnumber">1.8</SPAN><SPAN class="toctext">Pre-culture</SPAN></A></LI><LI class="toclevel-2 tocsection-10"><A href="#Recombineering"><SPAN class="tocnumber">1.9</SPAN><SPAN class="toctext">Recombineering</SPAN></A></LI><LI class="toclevel-2 tocsection-11"><A href="#SDS-PAGE_and_Western_Blotting"><SPAN class="tocnumber">1.10</SPAN><SPAN class="toctext">SDS-PAGE and Western Blotting</SPAN></A></LI><LI class="toclevel-2 tocsection-12"><A href="#Reference"><SPAN class="tocnumber">1.11</SPAN><SPAN class="toctext">Reference</SPAN></A></LI></UL></TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="Protocols_.28in_alphabetical_order.29">Protocols (in alphabetical order)</SPAN></H1><H2><SPAN class="mw-headline" id="Bacteria_Lysis">Bacteria Lysis</SPAN></H2><OL><LI> Harvest the culture by centrifuge (13krpm*1.5min) followed by washing with 1ml PBS twice. Later steps should be done on the ice.
</LI><LI> Re-suspend the pellet with PBS and protease inhibitor cocktail. 
</LI><LI> Sonication for (5.5seconds+1second pulse)*10 minutes and protein solutions are obtained.
</LI></OL><H2><SPAN class="mw-headline" id="BCA_Quantification_Analysis">BCA Quantification Analysis</SPAN></H2><OL><LI> Prepare a working solution of BCA reagent just prior to use by adding BCA Reagent A and BCA Reagent B with a ratio of 50:1. Mix the two solutions until a clear green solution forms. Prepare the BCA working reagent fresh daily. 100uL is required for each sample. 
</LI><LI> For each protein determination, add 2~5uL protein sample and 100uL BCA mixture into each well. Usually each sample will be loaded into 3 wells to reduce errors.
</LI><LI> Put the 96-well plate in the warm room for about 30 minutes.
</LI><LI> Read the wells in a suitable plate reader (e.g. Molecular Devices) at the wavelength of 562 nm.
</LI></OL><H2><SPAN class="mw-headline" id="Competent_Cell_Preparation_for_Electro_Transformation">Competent Cell Preparation for Electro Transformation</SPAN></H2><UL><LI> Materials
</LI><LI> Media: LB (Both liquid media and media containing agar. Add certain antibiotic if it is necessary.) 
</LI><LI> Buffers and Solutions: 10% Glycerol. 
</LI><LI> Special Equipments : EP tubes (1.5mL), micropipette tips, centrifugation bottles (polypropylene tubes, 50mL), graduated flask (250mL*2, 5mL*1), plates and test tubes. 
</LI><LI> Steps
</LI><LI> Sterilization 
</LI><LI> Including all materials in part2. 
</LI><LI> Caution (Remind): use some special marks to distinguish the sterilized materials from the unsterilized ones. 
</LI><LI> Preparation after sterilization 
</LI><LI> Chill the 10%Glycerin to 4 centigrade degree 
</LI><LI> Decant LB containing agar into the plates.(If antibiotics are necessary, be sure that they are added when the media temperature is below 60 centigrade degree.) 
</LI><LI> Streak the prepared strains onto the agar plates. Then incubate it at 37℃ for 10-16 hours. 
</LI><LI> Overnight pre-culture
</LI><LI> Take 0.5mL overnight culture to 50mL LB bottle. 
</LI><LI> 37℃ shaking 100~120 min to O.D. 600=0.45~0.6 
</LI><LI> On ice for 30min 
</LI><LI> 4000rpm 7min at 4℃ 
</LI><LI> Add origin volume 10% glycerol, suspend softly. 
</LI><LI> Repeat the step 7,8 
</LI><LI> Add origin volume 1/10 10% glycerol, suspend softly. 
</LI><LI> 4000rpm7 min at 4℃
</LI><LI> Add origin volume 1/100 10% glycerol, suspend softly, store at -80℃.
</LI></UL><H2><SPAN class="mw-headline" id="Electro_Transformation">Electro Transformation</SPAN></H2><OL><LI> Hold competent cells (from -80℃ refrigerator) on ice. 
</LI><LI> Gently mix ligation product (1-5 μL) with cells. 
</LI><LI> Transfer the cell/DNA mix into an electroporation cuvette.Note: the gene pulser should already be set properly
</LI><LI> time constant = 4.5 - 5.0 ms
</LI><LI> resistance = 200 W
</LI><LI> capacitance = 25 mFD for 0.1 cm gap cuvettes, set the volts to 1.8 kV
</LI><LI> Pulse the cells once; the voltage display blinks, and the gene pulser beeps
</LI><LI> Quickly transfer 37 ℃ SOC to cuvette, mix by gently pipetting up and down, and transfer SOC/cells back to culture tube.
</LI><LI> Bath in 37℃ for 30~60 min. 
</LI><LI> Separate cells on petri-dishes, and cultivate them in 37℃ for 12-16 hours.
</LI></OL><H2><SPAN class="mw-headline" id="Ligation">Ligation</SPAN></H2><P>England Biolabs T4 DNA ligases are used here. 
</P><OL><LI> Choose reaction volume: 5-10 μL 
</LI><LI> Mix proper proportion (usually 3~5:1) of DNA fragment and vector. 
</LI><LI> Add 10 × ligase buffers. 
</LI><LI> Add 0.5 μL ligase per 10 μL final volume. 
</LI><LI> Bath in 16℃ water for 12 hour, 
</LI><LI> Begin transformation. 
</LI></OL><H2><SPAN class="mw-headline" id="Membrane_Biotinylation">Membrane Biotinylation</SPAN></H2><OL><LI> Activate the membrane with methanol for 10-30seconds
</LI><LI> Balance the membrane in PBS for 5 minutes
</LI><LI> Place the membrane on a piece of filter paper.
</LI><LI> Drop the protein-biotin complex onto the membrane.
</LI><LI> Air-dry for 5 minutes.
</LI><LI> Soak in methanol for 1minutes.
</LI><LI> Place the membrane on a piece of filter paper.
</LI><LI> Air dry for 15 minutes.
</LI></OL><H2><SPAN class="mw-headline" id="Photolithography">Photolithography</SPAN></H2><P>Photolithography
The motors are fabricated by using a general surface manufacturing process for Si and SiO2. The motor bodies are made by reactive ion etching of thermal SiO2 layer on a Si substrate with a mask. Then the Si under the motors is anisotropically etched by wet etching, so that the motors are tethered to the Si base by two thin bridges designed to break upon sonication. 
</P><P>Photolithography is an optical means for transferring patterns onto a substrate. It is essentially the same process that is used in lithographic printing. Patterns are first transferred to an imagable photoresist layer. Photoresist is a liquid film that can be spread out onto a substrate, exposed with a desired pattern, and developed into a selectively placed layer for subsequent processing. 
</P><P>Overview of the photolithography process:
1、Surface Preparation
(1) Typical contaminants that must be removed prior to photoresist coating:
• dust from scribing or cleaving (minimized by laser scribing)
• atmospheric dust (minimized by good clean room practice)
• abrasive particles (from lapping or CMP)
• lint from wipers (minimized by using lint-free wipers)
• photoresist residue from previous photolithography (minimized by performing oxygen plasma ashing)
• bacteria (minimized by good DI water system)
• films from other sources:
– solvent residue
– H2O residue
– photoresist or developer residue
– oil
– silicone
(2) Wafer priming: Adhesion promoters are used to assist resist coating. 
a. Resist adhesion factors:
• moisture content on surface
• wetting characteristics of resist
• type of primer
• delay in exposure and prebake
• resist chemistry
• surface smoothness
• stress from coating process
• surface contamination
b. Wafer primers
Used for silicon:
– primers form bonds with surface and produce a polar (electrostatic) surface
– most are based upon siloxane linkages (Si-O-Si)
• 1,1,1,3,3,3-hexamethyldisilazane (HMDS), (CH3)3SiNHSi(CH3)3
• trichlorophenylsilane (TCPS), C6H5SiCl3
• bistrimethylsilylacetamide (BSA), (CH3)3SiNCH3COSi(CH3)3
• Used for gallium arsenide:
– GaAs already has a polar surface
• monazoline C
• trichlorobenzene
• xylene
</P><P>2、Coating (Spin Casting)
Wafer is held on a spinner chuck by vacuum and resist is coated to uniform thickness by spin coating. Resist thickness is set by primarily resist viscosity and secondarily spinner rotational speed. Most resist thicknesses are 1-2 mm for commercial Si processes. 
</P><P>Fig 1 Photoresist Spin Coating
</P><P> 
Fig 2 Stages of Resist Coating 
</P><P>3、Pre-Bake (Soft Bake)
Pre-bake is used to evaporate the coating solvent and to densify the resist after spin coating. Commercially, microwave heating or IR lamps are also used in production lines. Hot plating the resist is usually faster, more controllable, and does not trap solvent like convection oven baking.
</P><P>Fig 3 Soft Bake
</P><P>4、Alignment
</P><P>Fig 4 Overview of Align/Expose/Develop Steps 
</P><P>5、Exposure
</P><P>Fig 5 Exposure (&quot;printing&quot;) systems
</P><P>For simple contact, proximity, and projection systems, the mask is the same size and scale as the printed wafer pattern. I.e. the reproduction ratio is 1:1.
</P><P>Projection systems give the ability to change the reproduction ratio. Going to 10:1 reduction allows larger size patterns on the mask, which is more robust to mask defects.
Mask size can get unwieldy for large wafers.
</P><P>Most wafers contain an array of the same pattern, so only one cell of the array is needed on the mask. This system is called Direct Step on Wafer (DSW). These machines are also called “Steppers”. 
</P><P>6、Development
(1) Types:
– photographic emulsion on soda lime glass (cheapest)
– Fe2O3 on soda lime glass
– Cr on soda lime glass
– Cr on quartz glass (most expensive, needed for deep UV litho)
(2) Dimensions:
– 4” x 4” x 0.060” for 3-inch wafers
– 5” x 5” x 0.060” for 4-inch wafers
(3) Polarity:
– “light-field” = mostly clear, drawn feature = opaque
– “dark-field” = mostly opaque, drawn feature = clear 
</P><P>Normally, this process requires at least two alignment mark sets on opposite sides of wafer or stepped region.
</P><P>7、Post-Bake (Hard Bake)
Post bake is used to stabilize and harden the developed photoresist prior to processing steps that the resist will mask. The main parameter of this process is the plastic flow or glass transition temperature. It can remove any remaining traces of the coating solvent or developer. Also, it eliminates the solvent burst effects in vacuum processing. Post-bake introduces some stress into the photoresist and some shrinkage of the photoresist may occur. Longer or hotter post-bake makes resist removal much more difficult. 
</P><P>8、Processing Using the Photoresist as a Masking Film
Photoresist will undergo plastic flow with sufficient time and/or temperature.
</P><P>9、Stripping
This process is to remove the photoresist and any of its residues. Simple solvents are generally sufficient for non-postbaked photoresists. There are two kinds of photoresist, i.e. positive photoresist and negative photoresist. Besides, plasma etching with O2 (ashing) is also effective for removing organic polymer debris.
</P><P>10、Post Processing Cleaning (Ashing)
</P><P>Light Sources: 
1.  Ultraviolet light from gas-discharge lamps
2.	&quot;deep ultraviolet&quot;, produced by excimer lasers
3.	Immersion lithography with numerical apertures
4.	High-index immersion lithography (193nm)
</P><P>Basics of Photolithography for Processing:
1. Additive → deposition
2. Subtractive → etching
3. Modifying → doping, annealing, or curing
</P><P>Fig 6 Microfabrication Processes
</P><P>Two primary techniques for patterning additive and subtractive processes:
1. Etch-back:
</P><PRE> • photoresist is applied overtop of the layer to be patterned
 • unwanted material is etched away
</PRE><P>2. Lift-off:
</P><PRE> • patterned layer is deposited over top of the photoresist
 • unwanted material is lifted off when resist is removed

</PRE><P>Fig 7 Etch-back
</P><P>Fig 8 Lift-off
</P><H2><SPAN class="mw-headline" id="Pre-culture">Pre-culture</SPAN></H2><OL><LI> One single colony is picked up from the agar plate and transferred to a tube.
</LI><LI> Add 3~5mL LB broth to the tube and the specific resistance.
</LI><LI> Culture overnight at 32℃ or 37℃.
</LI></OL><H2><SPAN class="mw-headline" id="Recombineering">Recombineering</SPAN></H2><OL><LI>Overnight cultures: 5mL medium (containing antibiotic where applicable) from single colonies grown at 32°C for 18 h.
</LI><LI>Get 500uL from overnight cultures expanded into 50 ml of L medium in a 250 ml Erlenmeyer flask, and incubated at 32°C for 2 h (until OD600 of ca. 0.4–0.6). 
</LI><LI>Flasks were transferred to a shaking water bath at 42°C and incubated for 14–15 min, before cooling to 0°C as rapidly as possible in iced water. 
</LI><LI>After 15–20 min, cells were harvested by centrifugation at 0°C (4000 g, 9 min). 
</LI><LI>Cell pellets were carefully washed three times with sterilized ice-cold water (2 × 50 ml, then 1 × 1.5 ml) then re-suspended in 100–200 μl of ice-cold water. 
</LI><LI>Competent cells (50 μl) were transformed with 50–200 ng of (gel purified) linear dsDNA targeting cassette using a BioRad electroporator (1.8 kV, 25 mF, 200 W). 
</LI><LI>The LB medium (1 ml) was added to the transformed cell mixture, which was incubated at 32°C, for 2 h. Cells were collected by centrifugation, ca. 900 μl of supernatant media was discarded, and then the re-suspended cells were plated onto LB agar containing the appropriate antibiotic to select for resistant colonies. Reference to Watt et al [1].
</LI></OL><H2><SPAN class="mw-headline" id="SDS-PAGE_and_Western_Blotting">SDS-PAGE and Western Blotting</SPAN></H2><OL><LI> Buffer preparation 
</LI><LI> Rususpension buffer: 1ml stock + 0.1ml protease inhibitor (10x)
</LI><LI> Loading buffer: 1ml stock + 0.2 ml DTT + Phenol Blue 
</LI><LI> Sample treatment 
</LI><LI> centrifuge the culture into pellet (13k rpm 10min)
</LI><LI> Resuspend the pellet with 5xVolume(pellet) Resuspension buffer
</LI><LI> Add 5xVolume(pellet) loading buffer 
</LI><LI> Boil the resuspend for 10 minutes at 100’C
</LI><LI> Centrifuge for 1 min at 13krpm
</LI><LI> SDS-PAGE 
</LI><LI> 15% separation gel
</LI><LI> 5% stacking gel
</LI><LI> Load the sample and run under the constant voltage of 100V.
</LI><LI> Visualize your proteins using Coomassie Brilliant Blue, Silver stain, or any of the other protein stains or blot the gel for western blotting.
</LI><LI> Western blotting 
</LI><LI> Transfer from gel to membrane
</LI><LI> Assemble &quot;sandwich&quot; Transblot. </LI><LI> Prewet the sponges, filter papers (slightly bigger than gel) in 1x Blotting buffer. 
</LI><LI> Transfer for 1 hr at 1 amp at 4°C on a stir plate. Bigger proteins might take longer to transfer. For the Mini-Transblot, it's 100 V for 1 hr with the cold pack and prechilled buffer. When finished, immerse membrane in Blocking buffer and block overnight.
</LI><LI> Hybridization with antibodies 
</LI><LI> Incubate with primary antibody diluted in Blocking buffer for 60 min at room temp. 
</LI><LI> Wash 3 x 10 min with 0.05% Tween 20 in PBS. 
</LI><LI> Incubate with secondary antibody diluted in PBS for 45 min at room temp. 
</LI><LI> Wash 3 x 10 min with 0.05% Tween 20 in PBS. 
</LI><LI> Detect with SUPER SIGNAL WEST PICO Kit (1ml luminol solution + 1ml stable peroxide solution
</LI></OL><UL><LI> Result analysis 
</LI><LI> The size of the tagged protein can be determined by the marker (protein ladder)
</LI><LI> The amount of the protein can be estimated by the brightness of the band, or accurately analyzed by the software.
</LI></UL><LI> Trouble shooting 
</LI><LI> Unspecific binding : 1st antibody-membrane and irrelevant protein to 1st antibody
</LI><LI> Film over-exposure or lack of exposure
</LI><H2><SPAN class="mw-headline" id="Reference">Reference</SPAN></H2><OL><LI> Watt RM, Wang J, Leong M, Kung HF, Cheah KS, Liu D, Danchin A, Huang JD. Visualizing the proteome of Escherichia coli: an efficient and versatile method for labeling chromosomal coding DNA sequences (CDSs) with fluorescent protein genes. Nucleic Acids Res. 2007, 35(6):e37. 
</LI></OL></DIV><DIV id="sponsorlogosbox"><DIV id="sponsortag">Sponsors</DIV></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2009.igem.org/Team:HKU-HKBU/protocols">http://2009.igem.org/Team:HKU-HKBU/protocols</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:HKU-HKBU/protocols" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:HKU-HKBU/protocols" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:HKU-HKBU/protocols&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:HKU-HKBU/protocols&amp;oldid=128293" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2009.igem.org:Privacy_policy" title="2009.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2009.igem.org:General_disclaimer" title="2009.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>