"
Page
Discussion
View source
History
teams
Log in
 
Team:OUC-China/RNA guardian/Design
From 2013.igem.org
Overview
The navigation
OUC-China IGEM 2013
Home
Project
Overview
Achievement & judge criteria
Intracellular compartment
Review
Design
Microfluidics
Results
RNA guardian
Introduction
Design
Result
Reference
Part description
Future work
Modeling
Magnetic Analysis
Model in RNA guardian
Human Practice
Team
Members
Instructors
Advisor
Lab
Acknowledgement
Safety
Notebook
Lab note
Protocol
Introduction
Design
Results
Reference
We have added RBS to an mRNA 5’ Untranslated Region (UTR), single or poly. And we have designed a start codon and several stop codons in different reading frames. From a chemical perspective, the ribosome that is binding and dissociating will finally achieve a balance. We regard ribosome as a competitive inhibitor, inhibiting RNase E from being activated by binding on the 5’-end to some extents, as a result preventing mRNA degradation. In E.coli, many exonuclease such as polynucleotide phosphorylase (PNPase), RNase II, RNase R, and oligoribonuclease are also involved in the degradation progress. So, in the 3’ untranslated region, we designed a similar device.
(A)Genetic map of part K1059003
(B)Genetic map of part K1059004
Fig.1 Genetic map of part K1059003 and K1059004. (Here we named it “guardian”).
(A)We make part K1059003 a standard Biobrick by adding standard prefix and suffix. In that part, there’s a promoter(J23101) to start the transcript ,a RBS(B0035) to binding extra ribosome, a start codon to promote the binding efficiency of ribosome and several stop codon to prevent ribosome from going down. All of them are designed at 5’-UTR of GFP mRNA.
(B)At the 3’-UTR, we design part K1059004. In the part, there’re more than forty nucleotides between UAA and the extra RBS. The start and stop codon perform the same function as aprt K1059003.
Fig.2 A brief introduction to our expectation of RNA guardian.
From a chemical perspective, the ribosome is binding and dissociating from a mRNA and finally achieving a balance. At 5’-end UTR, We regard ribosome as a competitive inhibitor, inhibiting RNase E from being activated by binding on the 5’-end; at 3’-UTR, the ribosome performed the same function to another endonuclease.
We have constructed these following circuits. And to detect if the part works well, we use GFP as a reporter to reflect the translation efficiency. For experimental convenience, we choose GFP with a tag which can accelerate the degradation of GFP, So the fluorescence can reach a plateau quickly for easier measuring. To test the mRNA level, we extract the total RNA and proposed RT-PCR (reverse transcription-polymerase chain reaction).
Fig.3 Circuits constructed by OUC –iGEM.
In order to find which combination works well, we constructed these circuits and measured it by fluorescence and RT-PCR. All of the guardians are located in the untranslated region, 3’-end or 5’-end. Both of them consist of a RBS than can bind an extra ribosome, a start codon and several stop codons in different reading frames.
There may be a question troubling you. Will it lower the translational efficiency if the extra Ribosome interacts with the real ribosome? We have considered it. Between the two or several RBS, there are over forty nucleotides and that is considered enough to prevent two ribosomes from interfering with each other. Because of the several stop codons in different reading frames, the extra ribosome cannot go down the mRNA.
About
Thanks to:
     Bootstrap     Flat-ui     Unslider
Designed and built by @PengYong and @ZhenYuchen.
Code licensed under Apache License v2.0
Contact US
E-mail: oucigem@163.com
Official website
Find us on Renren
Find us on Weibo
Find us on Google Map
Retrieved from " http://2013.igem.org/Team:OUC-China/RNA_guardian/Design "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
