"
Page
Discussion
View source
History
teams
Log in
 
Team:OUC-China/Part I
From 2013.igem.org
Model in RNA guardian
Home
Project
Overview
Achievement & judge criteria
Intercellular compartment
Review
Design
Microfluidics
Results
RNA guardian
Instruction
Design
Result
Reference
Part description
Future work
Modeling
Magnetic Analysis
Model in RNA guardian
Human Practice
Team
Members
Instructors
Advisor
Lab
Acknowledgement
Safety
Notebook
Lab note
Protocol
Overview
Preliminary Prediction
ODE Simulation
Error Analysis
Using RBS Calculator
Aim: Use RBS Calculator from Salis Lab to find the difference between different Experimental Groups, and rule out some unnecessary consideration, and summarize assumption for the next step.
Step:
1. Use the RBS Calculator to get the following data.
2. Analysis.
Each group consists of a conding sequence(CDS),one or two guardian sequence (Ribosome Binding Site here,the RBS before CDs we called it RBS0,the RBS else which inhibits mRNA's decay we called it RBS1)
We use a free energy model for ribosome assembly according to the article[1] to estimate the difference between each groups . The thermodynamic model calculates the difference in Gibbs free energy before and after the 30S complex assembles onto an mRNA transcript denoted by Gtot. Given an mRNA subsequence five free energy terms are calculated and summed together:
ΔG(tot)= ΔG(final)- ΔG(initial)= (ΔGmRNA-rRNA+ΔGstart+ΔGspacing-ΔGstandby)- ΔGmRNA
ΔGmRNA:  The initial state is the unbound 30S complex and mRNA subsequence. The mRNA subsequence is folded to its minimum free energy secondary structure with a corresponding free energy ΔGmRNA
ΔGstandby: The ΔGstandby  is the energy released when the standby site is folded (DGstandby < 0), we define the standby site as the four nucleotides upstream of the 16S rRNA-binding site.
ΔGspacing: The ΔGspacing is an energetic penalty for a nonoptimal distance between the 16S rRNA-binding site and the start codon  (ΔGspacing)..
ΔGstart:The ΔGstart is the energy released when the tRNAfMet 's anticodon hybridizes to the start codon (ΔGstart < 0).
ΔGmRNA-rRNA: The ΔGmRNA-rRNA  is the energy released when the last nine nucleotides of the 16S rRNA cofolds and hybridizes with the mRNA subsequence at the 16S rRNA-binding site. The ΔGmRNA-rRNA  calculation includes both intermolecular nucleotide base pairings between the 16S rRNA and mRNA and intramolecular nucleotide base pairings within the mRNA itself (ΔGmRNA-rRNA < 0).
The translation initiation rate r is related to the ΔGtot  according to where β is the Boltzmann factor for the system. And the start position means the RBS0's position from the 5'end.Next, we run the tool putting our sequences in and get the result following :
From the chart above, we can see that, ΔGstart , ΔGspacing are the same,because we use the same RBS in each group. Also, the ΔGmRNA-rRNA in control group and  Experimental Group 4 are the same, comparing another 3 groups we can conclude that the difference is  if  there is RBS1 in front of RBS0,if not, the calculated ΔGmRNA-rRNA is lower, which contribute to the translation initiation rate. Each group's ΔGmRNA is not the same that's because their sequence's length and the content of A,U,G,C  are not the same based on the free energy model. We can't get any useful information from the calculated result more, but at least we know that the RBS 's location and number in each group could not make huge change in the expression of GFP in our experiment, for the translation initiation rate in each group is similar. We get some prediction below:
1.The data in Control Group and Experimental Group 4 may be higher relatively than othen groups, according to the row data we get in the next step, in fact ,it is true ,we  will show you in Part III.
2.From the result we get, there is not obvious difference in each group, but if there exists  obvious difference between each group from the row data ,we will find if there are other factors such as the RNase E inhibiting the RNA's degration, or the exonucleases' effect causing this difference. We repeat a couple of experiments to rule out the errors bringing in by the difference between each group's growth of bacteria .
3.We assum each group's transcriptional level is a constant, and  we make a preliminary prediction without considering other factors just based on a thermodynamic model about  the protein expression below:
Experimental Group 2 > Experimental Group 1 >Experimental Group 3 > Control group > Experimental Group 4 .
But that is not true, we will try to amend it considering other influence factors in the next model.
Reference:
1.Automated Design of Synthetic Ribosome Binding Sites to Precisely Control Protein Expression. Nat Biotechnol. 2009 October; 27(10): 946�C950.
Retrieved from " http://2013.igem.org/Team:OUC-China/Part_I "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
