<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Virginia_Design skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Virginia/Design</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container-fluid"><UL class="nav navbar-nav navbar-right"><LI class="active"><A href="https://2017.igem.org/Team:Virginia">Home</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">Team
        </A></LI><LI><A href="/Team:Virginia/Team">Team</A></LI><LI><A href="/Team:Virginia/Collaborations">Collaborations</A></LI><LI><A href="/Team:Virginia/Attributions">Attributions</A></LI></UL><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">Project
        </A></LI><LI><A href="/Team:Virginia/Description">Description</A></LI><LI><A href="/Team:Virginia/Design">Design</A></LI><LI><A href="/Team:Virginia/Model">Model</A></LI><LI><A href="/Team:Virginia/Part_Collection">Parts</A></LI><LI><A href="/Team:Virginia/Improve">Improve</A></LI><LI><A href="/Team:Virginia/Results">Results</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">Documentation
        </A></LI><LI><A href="/Team:Virginia/Notebook">Notebook</A></LI><LI><A href="/Team:Virginia/InterLab">InterLab</A></LI><LI><A href="/Team:Virginia/Safety">Safety</A></LI><LI><A href="/Team:Virginia/Experiments">Protocols</A></LI><LI><A href="/Team:Virginia/MedalCriteria">Medal Criteria</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">Human Practices
        </A></LI><LI><A href="/Team:Virginia/Engagement">Public Engagement</A></LI><LI><A href="/Team:Virginia/HP/SilverGold">Integrated Human Practices</A></LI><LI><A href="https://igem.org/2017_Judging_Form?team=Virginia">Judging</A></LI></DIV><DIV class="igem_2017_content_wrapper"><CENTER><H1>Design</H1></CENTER><DIV class="column full_size"><DIV style="padding-left: 100px; padding-right: 100px;line-height: 25px;"><H2 style="font-size:25px; LINE-HEIGHT:35px;"><CENTER>The purpose of our project is to modify the tertiary sewage treatment process in a way that eliminates the inefficiencies created by the dynamic between aerobic nitrifying bacteria and heterotrophic denitrifying bacteria. </CENTER></H2><H4>Examining the Current Nitrification-Denitrification Processes
<DIV class="column half_size"><DIV style="padding-left: 100px; 50px;line-height: 25px;"><P>Sewage Treatment systems employ processes designed around bacteria that occupy specific niches to treat human wastewater. The Biological Nutrient Removal stage, a part of the tertiary sewage wastewater treatment, in wastewater treatment uses bacteria capable of nitrification and denitrification to remove dissolved nitrogen from secondary effluent sewage. The process of denitrification has been found in both aerobic and anaerobic organisms, and major steps of the denitrification pathway have been elucidated in multiple strains (Potter; Brons; Costa; Gavira; Hasegawa; Moreno-Vivián) . Denitrification is the progressive anaerobic reduction of nitrates to nitrites to nitric oxide, nitrous oxide, and eventually nitrogen gas (Peng et al 2006). Nitrification involves the oxidation of ammonia into hydroxylamine and hydroxylamine into nitrites (Arp et al 2002). Our primary goal was to bring the capabilities of nitrification and denitrification into a single cell.</P></DIV></DIV><DIV class="column half_size"><DIV style="padding-right: 10px; line-height: 25px;"><P><B>Figure 1</B>: <I>Illustration of Nitrification and Denitrification: The formulaic depiction of how ammonia is oxidized to nitrites and how nitrites are reduced into nitrogen gas. Figure from Peng et al (2006).</I></P></DIV></DIV></H4><H4>Chassis Selection</H4><P>Our search for organisms that possess the ability to nitrify and denitrify aerobically came up with three species of interest: <I>Paracoccus denitrificans</I>, <I>Pseudomonas aeruginosa</I>, and <I>Nitrosomonas europaea</I>. Among these organisms are the genes for comprehensive denitrification from ammonia and nitrates to nitrogen gas. <I>Pseudomonas aeruginosa</I> is a well characterized facultative anaerobe that can reduce nitrates and nitrites during anaerobic respiration. There are BioBricks made using genes from <I>P. aeruginosa</I> and it has been used in multiple iGEM projects  (SCU_China; DTU-Denmark; BYU_Provo; NYMU-Taipei). The main drawback to <I>P. aeruginosa</I> is that it cannot reduce nitrates and nitrites under aerobic conditions. Some strains are also known human pathogens making it a potential Risk Group 2 organism (Palmer; U.S. Dept. of Energy). Paracoccus denitrificans is a denitrifying chemolithoautotroph (Harms; U.S. Dept. of Energy). It can survive and multiply in both aerobic and anaerobic conditions and is shown to be capable of partial denitrification in aerobic conditions (Baumann; Takaya). Most common molecular biological techniques work on <I>Pc. denitrificans</I> and most broad-range vectors and promoters used in <I>E. coli</I> also work in <I>Pc. denitrificans</I> and it has even been shown to possess minor nitrification capabilities (Crossman et al 1997). <I>Nitrosomonas europaea</I> is an obligate chemolithoautotroph with a strictly aerobic metabolism. It feeds off the oxidation of ammonia to nitrites in a process called nitrification  (U.S. Dept. of Energy). It is often found in aerated sewage where there is a high concentration of dissolved nitrogen in the form of ammonia. The genes responsible for nitrification in <I>N. europaea</I> are mostly characterized and available as a BioBrick in the iGEM Parts Registry (DTU-Denmark). <I>N. europaea</I> is an efficient nitrifier, but it cannot survive under anoxic conditions. This presents a challenge for sewage treatment plants when heterotrophic denitrifiers consume oxygen, necessitating continued mixing and aeration of the sewage during tertiary treatment to maintain the <I>N. europaea</I> component.    <B>We have chosen <I>Pc. denitrificans</I> as a prospective chassis because it is a well characterized organism and is amenable to genetic modification. Most importantly, it is an efficient denitrifying bacteria. </B>Sewage treatment plants currently use a multi-strain, multi-step system that employs a mixture of bacteria called sludge, which is known to contain <I>N. europaea</I>, <I>Pc. denitrificans</I>, and <I>P. aeruginosa</I>, to remove dissolved nitrates, nitrites, and ammonia. In current systems, the presence of multiple strains in a single reaction adds the risk that one strain outcompetes others and the reactor breaking down or becoming less efficient (Castillo; Heufelder). The sludge must be moved between two separate chambers in the Modified Bardenpho Process (Heufelder 2017), one being anoxic and the other oxygenated. Alternatively, in Sequential Batch Processing a single chamber is intermittently aerated to sustain both the aerobic and anaerobic microbes present in sludge through fluctuations in the dissolved oxygen content. </P></DIV></DIV><DIV class="column full_size"><DIV style="padding-left: 100px; padding-right: 100px;line-height: 25px;"><H4>Device Implementation</H4></DIV><DIV class="column half_size"><DIV style="padding-left: 100px; padding-right: 20px;line-height: 25px;"><P>Early research suggested that <I>Pc. denitrificans</I> was effective in some sewage treatment designs as a co-culture with <I>N. europaea</I> (Kokufuta; Uemoto). In this design, small amounts of ethanol must be added daily to act as an electron donor so that nitrification reaction can be sustained. Instead, we propose a single-organism sewage treatment system achieved by cloning the entire amo operon from <I>N. europaea</I> to express the nitrification circuit in <I>Pc. denitrificans</I> when under aerobic conditions (Fig. 2, 3). Our initial intent was to tie expression of nitrification genes to dissolved oxygen levels in the sewage. By design, our device would need to increase the expression of nitrification genes in response to increasing oxygen concentrations. 



Having a single organism that is capable of both denitrification and nitrification will simplify the maintenance of a bioreactor and increase the efficiency of the wastewater treatment process. <I>Pc. denitrificans</I> is not restricted by the oxygen supply, so it is a superior competitor for space and resources under hypoxic or anoxic conditions when compared with <I>N. europaea</I>, which can only inhabit certain zones where the dissolved oxygen content is high enough. In the previous system, nitrates had to leave <I>N. europaea</I> and be absorbed by <I>Pc. denitrificans</I> before being converted into nitrogen gas (Kokufuta; Uemoto). By confining the nitrites to a single cellular compartment (Fig. 2, 3), they can build up and encounter denitrification enzymes sooner, making the process faster. The oxidative process of nitrification produces reduced electron acceptors sometimes in the form of quinone that can then be used as donors in the reductive denitrification steps. This will negate the need for regular infusions of electron donors.     </P></DIV></DIV><DIV class="column half_size"><DIV style="padding-left: 20px; padding-right: 100px;line-height: 25px;"><P><B>Figure 2: </B><I>Device Implementation During the Aerobic Cycle: Atmospheric oxygen is aerated into the sequential batch reactor as sewage is pumped in from secondary processing. Under aerobic conditions, the chassis will nitrify ammonia.  Denitrification of nitrates and nitrites at partial capacity, producing more nitrous oxide than nitrogen. During the aerobic cycle, the sludge is also mixed.</I></P><P><B>Figure 3:</B><I> Device Implementation During the Anaerobic Cycle: Mixing of the sludge is ceased along with aeration and secondary sewage influx to minimize oxygen influx. <I>Pc. denitrificans</I> consumes the remaining oxygen. Under anaerobic conditions it performs comprehensive denitrification to nitrogen gas. The effluent tertiary sewage is then released.</I></P></DIV></DIV><DIV class="column full_size"><DIV style="padding-left: 100px; padding-right: 100px;line-height: 25px;"><H4>Early Characterization</H4><P>
Early in the project timeline, we attempted to characterize the growth properties of <I>Pc. denitrificans</I> by creating a standard growth curve in both LB broth and recommended ATCC broth for <I>Pc. denitrificans</I>. The goal is to construct a growth curve of the relatively uncharacterized chassis for use in future protocols that require <I>Pc. denitrificans</I> to be at a certain stage in its growth. The use of both LB and paracoccal media is for two main goals: first, LB is needed so that <I>Pc. denitrificans</I> and <I>E. coli</I> be directly compared. Second, the ATCC media is needed so that <I>Pc. denitrificans</I> growth can be characterized under ideal conditions. After these are established, work towards characterizing part activity within the chassis will begin. Growth curves will also be constructed for <I>N. europaea</I> so that comparisons can be made between the final device and the original nitrifying strain it borrows from. 

</P><H4>Paracoccus Transformation</H4><P>Our search for relevant methods for Paracoccus transformation yielded few results. Most references of the transformation of <I>Pc. denitrificans</I> describe conjugation with a donor <I>E. coli</I> strain (Spence <I>et al</I>, 1981). Due to time constraints and limits in technical expertise, we had to find an alternative method. This method came in the form of an electroporation method developed by Holo et al for general use and more specifically in lactococcal strains. Work by Barak et al picked up on this research and reported successful transformation of<I>Pc. denitrificans</I> using a modified version of the original protocol. We successfully attempted to replicate the results reported by Barak et al and Holo et al to transform <I>Pc. denitrificans</I> with our biobricks of interest.</P><H4>Comparative Expression</H4><P>Once we have successfully transformed <I>Pc. denitrificans</I>, we transformed <I>E. Coli</I> in with reporter constructs designed to help characterize the behavior of certain promoter elements in <I>Pc. denitrificans</I>. Our initial plan was to tie a standard GFP coding sequence to a termination sequence and ribosome binding site. Next, we worked on creating composite biobricks by using restriction digests on the prefix of this construct. Digested promoter sequences for VhB, Mn-SOD, and a T7 constitutive promoter will all be tied to the same RBS, CDS, and termination sequence, thus direct comparisons between promoter activity can be made. We measured promoter activity as a function of fluorescence emitted by transformed cells. The measurements will be made using a fluorometer or comparable equipment. Samples will then be compared to control untransformed bacteria. We used a qPCR detection system to measure fluorescence of transformed cells using the protocol and guidelines set by Utermark and Karlovsky (2006). </P><H4>Circuit Design</H4><P>The finished device will carry the coding sequences for amoABC, HAO, Cc554, and Ccm552 from <I>N. europaea</I> (Fig. 4). All ammonia oxidation genes from <I>N. europaea</I> are iGEM BioBricks and make up the composite part BBa_K1067005. The amoABC gene cluster encodes the proteins for ammonia monooxygenase (AMO), a heterotrimeric enzyme that catalyzes the oxidation of ammonia into a hydroxylamine intermediate (DTU-Denmark; Arp et al 2002). HAO encodes hydroxylamine oxidoreductase (HAO), which oxidizes hydroxylamine to nitrite. The genes for cytochromes A and X (as annotated in BBa_K1067005) encode for Cc554 and Ccm552, two c-type cytochromes. Cc554 accepts an electron from HAO and donates it to Ccm552, which then donates the electron to quinone or other more terminal electron acceptors. Together, these genes catalyze the conversion of ammonia into nitrites. The denitrification circuit will be placed behind a strong RBS and an Mn-SOD oxygen-inducible promoter (BBa_K258005).    </P><P><B>Figure 4:</B> Ammonia Oxidation Circuit: Our circuit for ammonia oxidation is mostly derived from BBa_K1067005. RBS sequences and an oxygen-sensitive VHb promoter have been added upstream of the coding sequences (CDS). An additional promoter has been added at the midpoint to minimize the risk of premature RNA polymerase drop-off. </P><H4>Device Testing</H4><P>We quantified the activity of our device both in <I>E. coli</I> and in <I>Pc. denitrificans</I> so that to justify the decision to use a paracoccal strain instead of a standard chassis organism. We would also quantify the responses of untransformed <I>E. coli</I>, <I>Pc. denitrificans</I>, and <I>N. europaea</I> after being treated with an “ammonia spike”. Our assay would be conducted in a erlenmeyer flask on top of a hotplate set to 30 C and a magnetic stirrer. We tracked the levels of nitrates, ammonia, dissolved oxygen, and the OD 600 of each culture over time after introduction of excess ammonia. Ideally, data from the growth curve assays will inform the project and allow us to grow cultures to mid-exponential phase for maximum responsiveness to excess ammonia. The responses of each strain to an ammonia strike were then standardized against the corresponding reported OD 600 so that the ability of a unit of biomass to oxidize ammonia and reduce nitrates can be calculated and compared to the results of other strains.</P></DIV></DIV><P style="font-size:10pt">
References:
Barak, Y., Yarmus, M., Shapira, R. and van Rijn, J. (2002), Nitrite reduction in Paracoccus sp. is affected by a novel plasmid pYR1. FEMS Microbiology Letters, 208: 111–116. 

Holo, Helge, and Ingolf F. Nes. “High-Frequency Transformation, by Electroporation, of Lactococcus Lactis Subsp. cremoris Grown with Glycine in Osmotically Stabilized Media.” Applied and Environmental Microbiology 55.12 (1989): 3119–3123. Print.
 
J. Utermark and P. Karlovsky. “Quantification of green fluorescent protein fluorescence using real-time PCR thermal cycler.” Goettingen University, Goettingen, Germany. BioTechniques, Vol. 41, No. 2, August 2006, pp. 150–154

Atkinson, Mariette R., Timothy A. Blauwkamp, Vladamir Bondarenko, Vasily Studitsky, and Alexander J. Ninfa. “Activation of the glnA, glnK, and Nac Promoters as Escherichia Coli Undergoes the Transition from Nitrogen Excess Growth to Nitrogen Starvation.” Journal of Bacteriology 184, no. 19 (October 2002): 5358–63. 

Anderson, Phil. Ohio USA. Water Environment Association. Activated Sludge Design, Startup, Operation, Monitoring and Troubleshooting. 1 Sept., 2010. PDF.

Arp, Daniel J., Luis A. Sayavedra-Soto, and Norman G. Hommes. “Molecular Biology and Biochemistry of Ammonia Oxidation by Nitrosomonas Europaea.” Archives of Microbiology 178, no. 4 (October 1, 2002): 250–55. 

Baker, Simon C., Stuart J. Ferguson, Bernd Ludwig, M. Dudley Page, Oliver-Matthias H. Richter, and Rob J. M. van Spanning. “Molecular Genetics of the Genus Paracoccus: Metabolically Versatile Bacteria with Bioenergetic Flexibility.” Microbiology and Molecular Biology Reviews 62, no. 4 (December 1998): 1046–78.

Baumann, B, M Snozzi, A J Zehnder, and J R Van Der Meer. “Dynamics of Denitrification Activity of Paracoccus Denitrificans in Continuous Culture during Aerobic-Anaerobic Changes.” Journal of Bacteriology 178, no. 15 (August 1996): 4367–74.

(BCDHE) Barnstable County Department of Health and Environment. “Basics of Wastewater Treatment.” www.barnstablecountyhealth.org, Web. Accessed April 24, 2017. 

Boer, A. P. de, W. N. Reijnders, J. G. Kuenen, A. H. Stouthamer, and R. J. van Spanning. “Isolation, Sequencing and Mutational Analysis of a Gene Cluster Involved in Nitrite Reduction in Paracoccus Denitrificans.” Antonie Van Leeuwenhoek 66, no. 1–3 (1994): 111–27. 

Brindle, Keith, and Tom Stephenson. “The Application of Membrane Biological Reactors for the Treatment of Wastewaters.” Biotechnology and Bioengineering 49, no. 6 (March 20, 1996): 601–10. 

Brons, Hans J., and Alexander J. B. Zehnder. “Aerobic Nitrate and Nitrite Reduction in Continuous Cultures of Escherichia Coli E4.” Archives of Microbiology 153, no. 6 (May 1, 1990): 531–36. 

Burgin, Amy, and Stephen Hamilton. “Have We Overemphasized the Role of Denitrification in Aquatic Ecosystems? A Review of Nitrate Removal Pathways.” Front Ecol Environ, 2007, 5(2): 89–96.  

BYU_Provo. “Reclaiming Wastewater Reclamation”. Brigham Young University. iGEM Team.  (2014). 

Carlson, C A, and J L Ingraham. “Comparison of Denitrification by Pseudomonas Stutzeri, Pseudomonas Aeruginosa, and Paracoccus Denitrificans.” Applied and Environmental Microbiology 45, no. 4 (April 1983): 1247–53.

Castillo, Tim. Interview by Phone. Rivanna Water and Sewer Authority. April 24, 2017.  

Costa, Patrícía N. da, Miguel Teixeira, and Lígia M. Saraiva. “Regulation of the Flavorubredoxin Nitric Oxide Reductase Gene in Escherichia Coli: Nitrate Repression, Nitrite Induction, and Possible Post-Transcription Control.” FEMS Microbiology Letters 218, no. 2 (January 28, 2003): 385–93. 

Crossman, L. C., J. W. Moir, J. J. Enticknap, D. J. Richardson, and S. Spiro. “Heterologous Expression of Heterotrophic Nitrification Genes.” Microbiology (Reading, England) 143 ( Pt 12) (December 1997): 3775–83. 
Drews, Anja, Herman Evenblij, and Sandra Rosenberger. “Potential and Drawbacks of Microbiology–membrane Interaction in Membrane Bioreactors.” Environmental Progress 24, no. 4 (December 1, 2005): 426–33.   

DTU-Denmark. “Requiem for a Stream”. Technical University of Denmark. iGEM Team. (2013). 

Eddleman, Harold. “Optimum Temperature for Growth of Bacteria.” Edu. Univ. of Indiana. Accessed April 25, 2017. http://www.disknet.com/indiana_biolab/b062.htm.

(EPA) United States Environmental Protection Agency &quot;Addressing Nutrient Pollution in the Chesapeake Bay.&quot; United States Environmental Protection Agency, 24 Jan. 2017. https://www.epa.gov/nutrient-policy-data/addressing-nutrient-pollution-chesapeake-bay.

(EPM) Emerson Process Management. “Dissolved Oxygen Measurement in Wastewater Treatment”. Application Data Sheet. Rosemount Analytical Inc. Jan. 2009. Web. Accessed 26 Apr. 2017. http://www2.emersonprocess.com/siteadmincenter/PM%20Rosemount%20Analytical%20Documents/Liq_ADS_4950-01.pdf. 

(FAO) Food and Agricultural Organization of the United Nations. &quot;Wastewater Treatment.&quot; 3. Wastewater Treatment. National Resources Management and Environmental Department, Web. 29 Apr. 2017. http://www.fao.org/docrep/t0551e/t0551e05.htm. 

(FOSR) Friends of the Shenandoah. &quot;About FOSR.&quot;, 2013. Web. 29 Apr. 2017. 

Gallant, Micha. &quot;Menu.&quot; Wheatley River Improvement Group. Wheatley River Improvement Group, 2010. Web. 16 Apr. 2017.

Gavira, Mónica, M. Dolores Roldán, Francisco Castillo, and Conrado Moreno-Vivián. “Regulation of Nap Gene Expression and Periplasmic Nitrate Reductase Activity in the Phototrophic Bacterium Rhodobacter Sphaeroides DSM158.” Journal of Bacteriology 184, no. 6 (March 2002): 1693–1702. 

Glymph, Toni. “Bacteria- Protozoa of Wastewater”. Iowa Water Safety Department: Iowa Rural Water. Web. PDF. https://iowaruralwater.org/tools_tips/toni_glymp/Bacteria-Protozoa.pdf.  

Goulding, Keith, Steve Jarvis, and Andy Whitmore. “Optimizing Nutrient Management for Farm Systems.” Philosophical Transactions of the Royal Society: Biological Sciences 363, no. 1491 (February 12, 2008): 667–80. 

Harms, N., W. N. M. Reijnders, S. Koning, and R. J. M. van Spanning. “Two-Component System That Regulates Methanol and Formaldehyde Oxidation in Paracoccus Denitrificans.” Journal of Bacteriology 183, no. 2 (January 2001): 664–70. 

Hasegawa, N., H. Arai, and Y. Igarashi. “Two c-Type Cytochromes, NirM and NirC, Encoded in the Nir Gene Cluster of Pseudomonas Aeruginosa Act as Electron Donors for Nitrite Reductase.” Biochemical and Biophysical Research Communications 288, no. 5 (November 16, 2001): 1223–30. 

Heufelder, George. Interview by Phone. Barnstable County Dept. of Health and Env. April 24, 2017.

Heyde, Eva Laura von der, Benjamin Klein, Lars Abram, and Armin Hallmann. “The Inducible nitA Promoter Provides a Powerful Molecular Switch for Transgene Expression in Volvox Carteri.” BMC Biotechnology 15, no. 1 (February 18, 2015).  

Jones KM. “Increased production of the exopolysaccharide succinoglycan enhances Sinorhizobium meliloti 1021 symbiosis with the host plant Medicago truncatula.” J Bacteriol. 2012 Aug;194(16):4322-31. 

Knockaert, Carolien. &quot;Marine Biodiversity Wiki.&quot; Possible Consequences of Eutrophication - Marine Biodiversity Wiki. N.p., 10 Oct. 2014. Web. 12 Apr. 2017.

Kokufuta, Etsuo, Masato Shimohashi, and Isei Nakamura. “Simultaneously Occurring Nitrification and Denitrification under Oxygen Gradient by Polyelectrolyte Complex-Coimmobilized Nitrosomonas Europaea and Paracoccus Denitrificans Cells.” Biotechnology and Bioengineering 31, no. 4 (March 1, 1988): 382–84. 

Leung, KP et al. “iGEM Report: Characterization of Azotobacter vinelandii and Kits for its Synthetic Biology Applications”. PLOS Collections. iGEM 2015. 

Lyon-INSA-ENS. “Cobalt Buster”. Institut National des Sciences Appliquées, École Normale Supérieure de Lyon. Lyon, France. iGEM Team. (2011).  

Mattson, Neil, Roland Leatherwood, and Cari Peters. “Nitrogen: All Forms Are Not Equal.” GMPro Magazine, June 2009. 

Middlebrooks, J. Crites, RW. &quot;Dissolved Oxygen Solubility in Freshwater.&quot; Civil and Environmental Engineering CRC Press (2009): 529-37. 

(MN DNR) Minnesota Dept. of Natural Resources.&quot;Minnesota Lake Access.&quot; Lake Ecology -- Biological Differences. Empact Metro Project, 2013. Web. 12 Apr. 2017.

Moreno-Vivián, Conrado, Purificación Cabello, Manuel Martínez-Luque, Rafael Blasco, and Francisco Castillo. “Prokaryotic Nitrate Reduction: Molecular Properties and Functional Distinction among Bacterial Nitrate Reductases.” Journal of Bacteriology 181, no. 21 (November 1999): 6573–84.  

Moss, Brian. “Water Pollution by Agriculture.” Philosophical Transactions of the Royal Society: Biological Sciences 363, no. 1491 (February 12, 2008): 659–66.  

N Print. &quot;University of Virginia N Footprint.&quot; University of Virginia N Footprint | Nitrogen Footprint. 2011. Web. 29 Apr. 2017. http://www.n-print.org/UVaNFootprint.

NYMU-Taipei. “Conquer Venus”. National Yang Ming University. iGEM Team. (2012).   

Pace, Michael, PhD. Interview. UVa Dept. of Env. Sci. Clarke Hall. April 14, 2017.  

Palmer, Gregory C., and Marvin Whiteley. “Metabolism and Pathogenicity of Pseudomonas Aeruginosa Infections in the Lungs of Individuals with Cystic Fibrosis.” Microbiology Spectrum 3, no. 4 (August 2015). 

Peng, Yongzhen, and Guibing Zhu. “Biological Nitrogen Removal with Nitrification and Denitrification via Nitrite Pathway.” Applied Microbiology and Biotechnology 73, no. 1 (November 1, 2006): 15–26.

(PNA) Potassium Nitrate Association. “Nitrate (NO3-) versus Ammonium (NH4+)”. www.kno3.org, Web. Accessed April 11, 2017. 

Potter, L C, P Millington, L Griffiths, G H Thomas, and J A Cole. “Competition between Escherichia Coli Strains Expressing Either a Periplasmic or a Membrane-Bound Nitrate Reductase: Does Nap Confer a Selective Advantage during Nitrate-Limited Growth?” Biochemical Journal 344, no. Pt 1 (November 15, 1999): 77–84. 

Poutiainen, H., S. Laitinen, S. Pradhan, M. Pessi, and H. Heinonen-Tanski. &quot;Nitrogen Reduction in Wastewater Treatment Using Different Anox-circulation Flow Rates and Ethanol as a Carbon Source.&quot; Environmental Technology. U.S. National Library of Medicine, May 2010. 

Pretty, Jules N., Christopher F. Mason, David B. Nedwell, Rachel E. Hine, Simon Leaf, and Rachael Dils. “Environmental Costs of Freshwater Eutrophication in England and Wales.” Environmental Science &amp; Technology 37, no. 2 (January 1, 2003): 201–8. 

Robertson, Lesley A., and J. Gijs Kuenen. “Aerobic Denitrification: A Controversy Revived.” Archives of Microbiology 139, no. 4 (November 1, 1984): 351–54. 


Robertson, L. A., R. Cornelisse, P. De Vos, R. Hadioetomo, and J. G. Kuenen. “Aerobic Denitrification in Various Heterotrophic Nitrifiers.” Antonie van Leeuwenhoek 56, no. 4 (November 1, 1989): 289–99. 

SCU_China. “E. Pangu”. SiChuan University. Chengdu, China. iGEM Team. (2015). 

Spence, D. W. and Barr, G. C. (1981), A method for transformation of Paracoccus denitrificans. FEMS Microbiology Letters, 12: 159–161. 
 
Saunders, N. F., E. N. Houben, S. Koefoed, S. de Weert, W. N. Reijnders, H. V. Westerhoff, A. P. De Boer, and R. J. Van Spanning. “Transcription Regulation of the Nir Gene Cluster Encoding Nitrite Reductase of Paracoccus Denitrificans Involves NNR and NirI, a Novel Type of Membrane Protein.” Molecular Microbiology 34, no. 1 (October 1999): 24–36. 

Saunders, Neil F. W., Jorrit J. Hornberg, Willem N. M. Reijnders, Hans V. Westerhoff, Simon de Vries, and Rob J. M. van Spanning. “The NosX and NirX Proteins of Paracoccus Denitrificans Are Functional Homologues: Their Role in Maturation of Nitrous Oxide Reductase.” Journal of Bacteriology 182, no. 18 (September 15, 2000): 5211–17. 

Suzuki, Isamu, Usha Dular, and S. C. Kwok. “Ammonia or Ammonium Ion as Substrate for Oxidation by Nitrosomonas Europaea Cells and Extracts.” Journal of Bacteriology 120, no. 1 (October 1, 1974): 556–58.

Takaya, Naoki, Maria Antonina B. Catalan-Sakairi, Yasushi Sakaguchi, Isao Kato, Zhemin Zhou, and Hirofumi Shoun. “Aerobic Denitrifying Bacteria That Produce Low Levels of Nitrous Oxide.” Applied and Environmental Microbiology 69, no. 6 (June 2003): 3152–57. 

Tanaka, Kazuhiro, Minoru Tada, Takashi Kimata, Shouji Harada, Yuhko Fujii, Tamotu Mizuguchi, Naomichi Mori, and Hiroyoshi Emori. “Development of New Nitrogen Removal System Using Nitrifying Bacteria Immobilized in Synthetic Resin Pellets.” Water Science and Technology 23, no. 4–6 (February 1, 1991): 681–90. 

Thomsen, Jens K., Torben Geest, and Raymond P. Cox. “Mass Spectrometric Studies of the Effect of pH on the Accumulation of Intermediates in Denitrification by Paracoccus Denitrificans.” Applied and Environmental Microbiology 60, no. 2 (February 1994): 536–41.

Uemoto, H., and H. Saiki. “Nitrogen Removal by Tubular Gel Containing Nitrosomonas Europaea and Paracoccus Denitrificans.” Applied and Environmental Microbiology 62, no. 11 (November 1996): 4224–28
U.S. Dept. of Energy. “Nitrosomonas Europaea.” Gov. JGI Genome Portal. Accessed April 24, 2017. http://genome.jgi.doe.gov/niteu/niteu.home.html.

(UNEP) United Nations Environmental Program &quot;Economic Impacts of Eutrophication.&quot; Planning and Management of Lakes and Reservoirs: An Integrated Approach to Eutrophication. Osaka/Shiga, Japan. 2000.

U.S. Dept. of Energy. “Paracoccus Denitrificans.” Gov. JGI Genome Portal. Accessed April 24, 2017. http://genome.jgi.doe.gov/parde/parde.home.html. 

U.S. Dept. of Energy. “Pseudomonas Aeruginosa.” Gov. JGI Genome Portal. Accessed April 24, 2017. http://genome.jgi.doe.gov/psesy/psesy.home.html.

Weaver, Grant. &quot;A New Approach to Nitrogen Removal from Wastewater.&quot; Biotechnology and Bioengineering 101.6 (2008) Nitrogen Removal from Wastewater. The Water Planet Company. Web. 

Westminster College- SIM “Determining the Amount of Nitrate in Water.” Web. April 24, 2017. http://www.westminster.edu/about/community/sim/documents/SDeterminingtheAmountofNitrateinWater.pdf. 

Withers, Paul J. A., Colin Neal, Helen P. Jarvie, and Donnacha G. Doody. “Agriculture and Eutrophication: Where Do We Go from Here?” Sustainability 6, no. 9 (September 2, 2014): 5853–75. 

Wyant, Dan. Activated Sludge Process Control. State of Michigan Department of Environmental Quality. PDF. Web. 21 Apr. 2017.  https://www.michigan.gov/documents/deq/wrd-ot-activated-sludge-manual_460007_7.pdf.

Yang, Xiao-e, Xiang Wu, Hu-lin Hao, and Zhen-li He. &quot;Mechanisms and Assessment of Water Eutrophication.&quot; Journal of Zhejiang University. Science. B. Zhejiang University Press, Mar. 2008. Web. 13 Apr. 2017.
</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>