Team:Virginia/Model
Home
Team
Team
Collaborations
Attributions
Project
Description
Design
Model
Parts
Improve
Results
Documentation
Notebook
InterLab
Safety
Protocols
Medal Criteria
Human Practices
Public Engagement
Integrated Human Practices
Judging
Modeling
Metabolic Flux Modeling
Flux Balance Analysis
Recent advances in computational biology led to emergence of databases with whole-genome metabolic networks, which for simplicity we will call models. They describe the flow of metabolites within a particular organism. Applying constraint-based methods (see this figure ) to a particular model allows one to make quantitative predictions about cell phenotype while eliminating many complex parameters. Of particular interest to our project is flux balance analysis (FBA)[1], which allows us to predict optimal steady-state biomass yield, which is tightly correlated with cell growth rate and is the most likely observed phenotype[1,2]. One major advantage of FBA is that it does not require knowledge of enzymatic parameters. This figure[3] elucidates the mathematics of FBA.
One of the primary questions for the project is the following. Will the synthetic P. denitrificans strain containing the device grow better than a wild type strain in the presence of ammonia? It is important to know the answer to this question because, once implemented in a real setting, the synthetic strain will have to compete against multiple other bacteria. To that end, we performed comparative analysis of the two Paracoccus strains using FBA on the corresponding whole-genome metabolic model. The analysis pipeline involved a slew of open-source computational tools, which we will describe below.
First, we reconstructed a metabolic model of Paracoccus denitrificans strain DSM 413 on a complete medium using ModelSEED[4]. A complete medium is such that any nutrient, including ammonia, is available for uptake. Thus, the set of reactions included in the model is the largest of all possible sets. Although the largest, this set is incomplete because certain essential reactions that result in cell growth may be missing. In the next step, the model is gapfilled with the reactions necessary for measurable cell growth.
To quantify the difference between wild-type and synthetic strains, we must include several new reactions, metabolites and genes (e.g. oxygenation of ammonia by the AMO enzyme complex) into the model generated by ModelSEED. Such functionality is not available in ModelSEED. To implement this, we turned to COBRApy: Constraint-Based Reconstruction and Analysis[5] package written in Python. However, COBRApy does not natively work with ModelSEED models. To overcome this compatibility issue, we used Mackinac package[6] to convert the ModelSEED model into COBRApy-compatible format. Using COBRApy, we then added the new reactions into the model and performed FBA to compare the biomass yields, and hence the growth rates, of the two Paracoccus strains. The script is available here.
First, the FBA was run on the gapfilled unmodified (wild-type) model, which initially contained 1550 reactions and 1556 metabolites. The optimal biomass yield was found to be \( 224.3248 (\text{g dry weight}\cdot\text{h})^{-1} \). Next, we added the nitrification reactions, whose list is given below. \(\ce{Q}\) and \(\ce{QH_2}\) represent ubiquinone and ubiquinol, respectively. \( \text{UqO} \) is the ubiquinone oxidoreductase enzyme which catalyzes the last reaction, and is already present in the proteome of P. denitrificans.  \[ \ce{NH_3 + QH_2 + O_2 ->[\text{AMO}] H_2O + Q + NH_2OH} \]  \[ \ce{NH_3 + NAD + H_2O ->[\text{AMO}] 2H^+ + NADH + NH_2OH} \]  \[ \ce{NH_2OH + O_2 ->[\text{HAO}] NO_2^- + H^+ + H_2O} \]  \[ \ce{NH_2OH + 2Q + H_2O ->[\text{HAO}] NO_2^- + 2QH_2} \]  \[ \ce{QH_2 ->[\text{UqO}] 2H^+ + Q} \]  With the new model containing 1555 reactions and 1559 metabolites (with metabolites hydroxylamine, Q and QH2 added), the optimal biomass yield of the modified model was found to be \( \boxed{228.6980~(\text{g dry weight}\cdot\text{h})^{-1}} \). This is approximately 2% greater than the wild-type yield, indicating that the synthetic strain grows approximately 2% faster than the wild-type, according to our prediction.
Exchange Fluxes and Flux Variability Analysis
To gain a deeper understanding of how inclusion of new genes affects the metabolism of the cell, it is important to consider the difference in exchange fluxes between the wild type and synthetic strain. Exchange fluxes determine the intake and expulsion of certain extracellular metabolites, such as glucose, H+, or certain dipeptides. The set of exchange reactions also determines the composition of the growth medium. The set of fluxes that gives rise to the optimal biomass, however, is not unique. This includes exchange fluxes. The system of stoichiometric equations for FBA is usually underdetermined, which means that there are actually infinitely many solutions to the problem, all of which are optimal up to certain degree of significance. In other words, to investigate the significance of all exchange fluxes with respect to the optimum, we need to move away from analyzing specific numbers determined by FBA and instead consider ranges, or variations of fluxes that allow for the optimal solution.
To obtain the allowed range of fluxes, we employ a technique called Flux Variability Analysis (FVA)[7]. The main parameter of FVA is the so called fraction of optimum, denoted by \(f\), which specifies the allowed deviation from the optimal biomass. In essence, during FVA we perturb each specified exchange flux in both directions under the constraint that the solution must within the fraction of optimum, thereby obtaining the allowed lower and upper bounds on the flux. If the fraction equals 1 (i.e. no deviation from the optimal biomass yield is allowed, in contrast to 0.95, which corresponds to 95% of the optimum), then any fluxes within the obtained range will reproduce the optimal solution. COBRApy package contains a module of functions which performs FVA on an existing model (see script for details).
The data obtained during FVA with \(f=0.999\) (i.e. a really tight variability constraint) confirms that the addition of nitrification circuit does, in fact, enhance the metabolism of the entire cell. However, the data are insufficient to determine how exactly the additional reactions and metabolites allow for such improvement.
Nitrite Exchange Knock-out and New Project Directions
To visualize metabolic pathways in our model, we used escher. The two figures below are based on the data from COBRApy simulations. The numbers are precisely the fluxes of corresponding reactions.
Fig. 1: An escher map of the synthetic strain metabolism without nitrite transport block (optimal yield 228.6980 [gdw.h]-1)
Fig. 2: An escher map of the synthetic strain metabolism with nitrite transport knocked out (optimal yield 227.8975 [gdw.h]-1)
The first figure represents the nitrification-denitrification metabolic pathway of the strain with the device. As one can see in this figure, all the nitrite produced is expelled from the cell through nitrite transport. This most likely happens due to cytotoxicity of nitrite[8]. However, with nitrite transport genes knocked out as is illustrated in the second figure, we see that all nitrite flows directly into the denitrification pathway. This offers a potential idea for a side project: how much will denitrification efficiency be improved after knocking out nitrite exchange in Paracoccus denitrificans? This idea is worthwhile to explore in particular because knocking out nitrite exchange does not severely decrease the biomass yield (see figure captions).
Results, Summary and Discussion
Analysis of the data obtained from FBA revealed that the biomass yield of the P. denitrificans strain containing our device is greater than the wild-type yield by nearly 2%, which means that our bacterium will use the inserted nitrification pathway in order to enhance its metabolism. As such, our device potentially confers fitness advantage. We predict that our synthetic strain will out-compete the native wild-type strain if both strains are growing in the same environment, including sewage sludge. If our device is used in wastewater treatment facilities, we predict that there will be no need to artificially sustain the new culture.
As was stated earlier, the expected cell phenotype is the one with the largest biomass yield. This is only true under the assumption that the cell lives in an ideal or near-ideal environment with no over-population. Several studies have shown that under different growth environments or culture densities, cells can exhibit non-optimal yield metabolism[9,10], which means that pathways different from those obtained through FBA will be employed by the cell. We do not know whether wastewater causes a similar shift in metabolism of Paracoccus denitrificans. However, seeing as it is able to thrive in such environment gives enough reason to believe that the growth rate will not lose correlation with the biomass. Furthermore, it should be noted that this model is oblivious to protein structure and enzyme kinetics, and as such is unable to capture the full scale of effects caused by inserting the nitrification construct into the model.
(De)nitrification Kinetics
Description
In this section, we present a kinetic model of bioreactors which are able to perform both nitrification and denitrification. In particular, sequential batch reactors and bioreactors containing our synthetic strain of P. denitrificans are described by the model. Literature searching led to the discovery of a denitrification model called Activated Sludge Model with Indirect Coupling of Electrons (ASM-ICE), proposed by Pan et al. in 2013. [11] Mathematica notebook is available here.
In essence, the model is given by a system of coupled nonlinear time-dependent differential equations that describes the following intermediate concentrations: \(S_{NO_3}\) - nitrate, \(S_{NO_2}\) - nitrite, \(S_{NO}\) - nitric oxide, \(S_{N_2O}\) - nitrous oxide, \(S_{N_2}\) - nitrogen gas. The rate of change of each of these variables is coupled to \(X\) - the heterotrophic biomass, \(S_S\) - carbon source concentration, and one of \(S_{Mox}\) or \(S_{Mred}\) - concentrations of oxidized (reduced) electron carriers. The relationships are summarized by a stoichiometric matrix below.
Fig. 3: The stoichiometric matrix of denitrification kinetics model given in Pan et al. (2014). We use the bottom matrix corresponding to ASM-ICE model.
It is more instructive to interpret the table by looking at columns. Thus, for instance, we have \(\frac{dS_{N_2O}}{dt}=\frac{1}{2}\cdot R4-1\cdot R5\), where \(R4, R5\) are the defined on the right. The rates \(R1\) through \(R5\) depend on a large number of kinetic parameters which vary in different scenarios. We have chosen the set of parameters from Case 1 described in the paper as it is the closest to the experiment our wet lab team performed: "Case 1 (Kucera et al., 1983): The branching of the electron flow to individual terminal acceptors NO3, NO2 and N2O was investigated in a pure denitrifying culture P. denitrificans (N.C.1.B. 8944). The culture was cultivated anaerobically to early stationary phase. A closed reactor with magnetic stirrer was used to carry out batch tests, during which N2 was provided into the reactor to ensure oxygen free environment[...]"
Fig. 4: Table of constants for ASM-ICE model. The first numeric column corresponds to Case 1. Adapted from Pan et al. (2014)
Nitrification Amendment
Unfortunately, the above model does not describe the nitrification process. To that end, our main goal is to update the model in a proper way so that it incorporates both nitrification and denitrification, which is the full ammonia removal process. To accomplish this, we need to include two new dependent variables (i.e. concentrations): \(S_{NH_3}\) - ammonia and \(S_{NH_2OH}\) - hydroxylamine. Furthermore, we would need to incorporate several new reaction rates that couple the concentrations. These rates would be of the same functional form as \(R1\) since each nitrification step consumes oxidized acceptors and returns their reduced counterparts:
\[ RN\_ = r_{\_,max}\cdot X \cdot \frac{S_{\_}}{K^N_{\_} + S_{\_}}\cdot\frac{S_{Mox}}{K_{Mox,\_} + S_{Mox}}\]
Evidently, each such reaction would introduce 3 new kinetic parameters that would need to be estimated. With at least 2 new reactions (conversion of ammonia and hydroxylamine), the number of new parameters would be at least 6. Wolfram Mathematica has the capacity to obtain the best fit parameter estimates of a system of parametric nonlinear differential equations without knowing the closed form of the solutions. However, for more complicated models such as ours, the fitting becomes progressively slower as the dimension of the parameter space increases. This issue is resolved due to specifics of nitrification biochemistry: it was found that the concentration of hydroxylamine during ammonia oxidation is negligible[12]. As such, we can assume that it gets instantly converted by the Hydroxylamine Oxidoreductase (HAO), thereby allowing us to write a single combined reaction for conversion of ammonia as long as we properly account for electron transfer.
Sometimes life happens and you get scooped. This happened to us less than a week away from the end of our research. It turns out that Hydroxylamine Oxidoreductase (HAO) converts hydroxylamine to nitric oxide (NO), as opposed to nitrite (NO2-) as we had originally thought[13]. Nitrite is a non-enzymatic product of a chemical reaction of NO with oxygen O2. With this new knowledge, we had to change our course of action and define the combined reaction from the previous section to be \(\ce{NH_3 ->[\text{AMO+HAO}] NO}. \) Now we can finally write down the new equations to be added to the model:
\[ RN1 = r_{NH_3,max}\cdot X \cdot \frac{S_{NH_3}}{K^N_{NH_3} + S_{NH_3}}\cdot\frac{S_{Mox}}{K_{Mox,1} + S_{Mox}}\] \[ \frac{dS_{NH_3}}{dt} = -RN1 \] \[ \frac{dS_{NO}}{dt} = R3 - R4 + RN1 \]
Incorporating nitrification also changes electron carrier concentrations:
\[ \frac{dS_{Mox}}{dt} = -(1-Y_H)R1 + R2 + 0.5R3 + 0.5R4 + R5 - RN1 = -\frac{dS_{Mred}}{dt} \]
We are not yet able to simulate the model, for we do not know the values of the maximal reaction rate \(r_{NH_3,max}\) and the affinity constants \( K^N_{NH_3}, K_{Mox,1}. \)
Parameter Fitting and Sensitivity
Using the data we obtained during the bioreactor experiment (culture: N. europaea, ammonia spike at t=0), we ran the nonlinear model fit algorithm in Mathematica to obtain the best fit parameters. Their values are given below:
\[ r_{NH_3,\text{max}} = 0.7681335 \frac{\text{mmol }NH_3}{\text{mmol biomass}\cdot\text{h}},\ K^N_{NH_3} = 0.010363 \frac{\text{mmol }NH_3}{L},\ K_{Mox,1} = 0.0341977 \frac{\text{mmol}}{\text{mmol biomass}} \]
Nonlinear fit performs best when provided with sufficiently good initial guess values. To prevent the algorithm from producing extremely unlikely results, we imposed the following constraints, obtained through trial and error:
\[ 0 < r_{NH_3,\text{max}} < 1, 0 < K^N_{NH_3} < 1, 0 < K_{Mox,1} < 0.1 \]
Unfortunately, the use of constrained optimization made it impossible to directly obtain the errors on these values. To amend that, we manually performed parameter sensitivity analysis to gain a better understanding of the errors. The results are summarized in the figure below.
Fig. 5: Parameter sensitivity analysis. Blue points correspond to experimental data from an artificial bioreactor experiment performed by our team. Time is measured in hours.
Model Validation
To test the accuracy of the model, we have performed several simulations. In the first simulation, we set \(S_{NH_3}(0) = 34~\text{mmol/L}\) and the concentrations of all denitrification intermediates to zero. Furthermore, assume we are in excess of carbon source and have a large enough heterotrophic biomass. In this case, we expect that denitrification reactions do not produce enough oxidized electron carriers to support ammonia oxidation. The results of the simulation are in accord with these expectations.
Fig. 6: Simulation 1 - absence of oxidized electron carriers halts ammonia oxidation. Units of concentration: (mmol NH3)/L, time is in hours. Note the short time scale.
In the second simulation, we reproduced Case 1 scenario from the Pan et al. paper, now using the upgraded model, and compared the nitrogen concentrations of the original and expanded models. We expect the nitric oxide (NO) produced from ammonia oxidation to undergo conversion to nitrogen gas, the final product of ammonia removal. This is indeed the case, as is illustrated by the figure below.
Fig. 7: Simulation 2 - final concentrations of nitrogen with (blue) and without (yellow) ammonia oxidation. Units of concentration: (mmol N2)/L, time is in hours
Applications
Let us apply our model to solve a simple influent timing optimization problem. Suppose between windows of t0=1.5 hours we have an influx of wastewater with a constant concentration of ammonia equal to 10 mmol NH3/L and nitrite concentration of 2 mmol NO3/L.
Fig. 8: Simulation 3 - concentrations of NH3 (blue), NO3 (red) and N2 (black) are shown, in their respective units. Time is in hours
Upon closer look at the first cycle, we see that nitrogen gas production stalls due to exhaustion of ammonia.
Fig. 9: Simulation 3 - zoomed in version of Fig. 7
Mathematica can obtain the exact moment in time at which \(S_{NH_3}\) crosses a certain small threshold if an appropriate WhenEvent[] trigger is set up. In our case, exhaustion occurs at \(t\approx 0.83~\text{h}.\) This tells us that to maximize the productivity of the bioreactor, we should receive an influent at or slightly lower than 0.83 hours. We can simulate this scenario just as easily.
Fig. 10: Simulation 3.1 - optimized scenario (cut-off after 6 cycles)
And finally, let us put nitrogen production curves side by side.
Fig. 11: Comparison of nitrogen concentrations of the initial and optimized scenarios
The average rate of nitrogen production is clearly higher in the optimized model, even with a cutoff imposed. Naturally, this example could be extended in many different ways to predict more realistic situations (e.g. randomly distributed influent concentrations), but in the absence of readily available wastewater data measurements, we have decided to stop it here.
Results and Discussion
Let us discuss some strengths and weaknesses of this model. On the pro side, the nitrification extension of the original model was proven to be faithful -- i.e., the new model behaves as it is supposed to, retaining the full range of benefits of the original one. Secondly, by proxy of being implemented in Mathematica, our model offers a great deal of flexibility in choosing which scenarios can be simulated. Time-dependent and variable-dependent triggers in differential equations are extremely powerful tools in that respect. Only minor tweaks are required to simulate different initial conditions. If one suspects that some kinetic parameters differ from those that we have found earlier, using our code one can fit the model to experimental data to find more accurate estimates. Finally, our model predicts the accumulation of denitrification intermediates, which makes it possible to address the concern of excess production of nitrogen species that are harmful to the environment[11].
On the other hand, there are several major drawbacks. First, this model is incomplete: non-enzymatic reaction \(\ce{NO + 1/2 O_2 -> NO_2^-}\) and the action of nitrite oxidoreductase (Nor) given by \(\ce{NO_2^- ->[\text{Nor}] NO_3^-}\) are not part of this model. Although it is possible to expand the model further and incorporate these reactions, the lack of sufficient data and a growing number of technical difficulties prevented us from doing so. Secondly, this model does not account for oxygen O2 levels in the environment. Incorporating it into the model, given the complexity of its interactions with nearly all of the species present, is beyond the scope of a summer project, and most likely beyond the numeric capacity of Mathematica's algorithms.
The current age is, without a doubt, an age of numerics. In mathematical biology, closed-form formulas amenable to rigorous mathematical analysis are a rarity. Let us also not forget the rise of all the different -omics with their seemingly obscure meta-data that, upon heavy analysis, lead to a completely new understanding of the structure and function of living organisms. It is imperative that future generations of computational biologists embrace that many problems do not have a nice solution, and the only way to study them is to stand on the shoulders of numerical giants (whose names, for some reason, always start with letter M). We can only hope that one day, someone somewhere will stumble upon our code and repurpose it to their own project.
Resources
Generic template of the Metabolic Modeling script for use by other teams
Script used in the Metabolic Modeling section
Mathematica script with nitrogen removal kinetics
References
[1] Oberhardt M., Chavali A., Papin J. (2009) Flux Balance Analysis: Interrogating Genome-Scale Metabolic Networks. In: Maly I. (eds) Systems Biology. Methods in Molecular Biology (Methods and Protocols), vol 500. Humana Press
[2] Feist, Adam M., and Bernhard O. Palsson. “The Biomass Objective Function.” Current opinion in microbiology 13.3 (2010): 344–349. PMC. Web. 28 July 2017.
[3] Cuevas, Daniel A. et al. “From DNA to FBA: How to Build Your Own Genome-Scale Metabolic Model.” Frontiers in Microbiology 7 (2016): 907. PMC. Web. 27 July 2017.
[4] Henry, C.S., DeJongh, M., Best, A.B., Frybarger, P.M., Linsay, B., and R.L. Stevens. High-throughput Generation and Optimization of Genome-scale Metabolic Models. Nature Biotechnology, (2010).
[5] Ebrahim, Ali et al. “COBRApy: COnstraints-Based Reconstruction and Analysis for Python.” BMC Systems Biology 7 (2013): 74. PMC. Web. 17 Aug. 2017.
[6] Mundy, Michael, Helena Mendes-Soares, and Nicholas Chia. "Mackinac: a bridge between ModelSEED and COBRApy to generate and analyze genome-scale metabolic models." Bioinformatics (2017): btx185.
[7] Gudmundsson, Steinn, and Ines Thiele. “Computationally Efficient Flux Variability Analysis.” BMC Bioinformatics 11 (2010): 489. PMC. Web. 1 Nov. 2017.
[8] Hartop, K.R. et al. “The Metabolic Impact of Extracellular Nitrite on Aerobic Metabolism of Paracoccus Denitrificans.” Water Research 113 (2017): 207–214. PMC. Web. 1 Nov. 2017.
[9] Adadi, Roi et al. “Prediction of Microbial Growth Rate versus Biomass Yield by a Metabolic Network with Kinetic Parameters.” Ed. Nathan D. Price. PLoS Computational Biology 8.7 (2012): e1002575. PMC. Web. 31 July 2017.
[10] Molenaar, Douwe et al. “Shifts in Growth Strategies Reflect Tradeoffs in Cellular Economics.” Molecular Systems Biology 5 (2009): 323. PMC. Web. 31 July 2017.
[11] Pan et al. “Evaluating two concepts for the modelling of intermediates accumulation during biological denitrification in wastewater treatment.” Water Research, 25 Dec. 2014.
[12] Keener, Arp "Kinetic Studies of Ammonia Monooxygenase Inhibition in Nitrosomonas europaea by Hydrocarbons and Halogenated Hydrocarbons in an Optimized Whole-Cell Assay." Applied and Environmental Microbiology, Aug. 1993
[13] Caranto, JD., Lancaster, KM. "Nitric oxide is an obligate bacterial nitrification intermediate produced by hydroxylamine oxidoreductase." Web. 17 July 2017.
