<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Warsaw_Project"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Warsaw/Project">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Warsaw/Project&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Warsaw/Project&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Warsaw/Project&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Warsaw/Project" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Warsaw/Project</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><P><TITLE>iGEM Team Warsaw</TITLE></P><UL class="main-menu"><LI><A href="https://2014.igem.org/Team:Warsaw">Project</A></LI><LI><A href="https://2014.igem.org/Team:Warsaw/Project#background">Background</A></LI><LI><A href="https://2014.igem.org/Team:Warsaw/Project#proof_of_concept">Proof of concept</A></LI><LI><A href="https://2014.igem.org/Team:Warsaw/Project#detailed_explanation">Detailed explanation</A></LI><LI><A href="https://2014.igem.org/Team:Warsaw/Project#modelling">Modelling</A></LI><LI><A href="https://2014.igem.org/Team:Warsaw/Project#weeestudy">WEEE study</A></LI><LI><A href="https://2014.igem.org/Team:Warsaw/Project#safety">Safety</A></LI><LI><A href="https://2014.igem.org/Team:Warsaw/Project#possibilities_of_development">Possibilities of development</A></LI></UL><LI><A href="/Team:Warsaw/Achievements">Achievements</A></LI><LI><A href="https://2014.igem.org/Team:Warsaw/Achievements#parts">Parts</A></LI><LI><A href="https://2014.igem.org/Team:Warsaw/Achievements#results">Results</A></LI><LI><A href="https://2014.igem.org/Team:Warsaw/Achievements#cooperation">Cooperation</A></LI><LI><A href="https://2014.igem.org/Team:Warsaw/Achievements#medal_criteria">Medal Criteria</A></LI><LI><A href="/Team:Warsaw/HP">HP</A></LI><LI><A href="https://2014.igem.org/Team:Warsaw/HP#overview">Overview</A></LI><LI><A href="https://2014.igem.org/Team:Warsaw/HP#lanthan_hospital">Lanthan Hospital</A></LI><LI><A href="https://2014.igem.org/Team:Warsaw/HP#science_festival">Science Festival</A></LI><LI><A href="https://2014.igem.org/Team:Warsaw/HP#media">Media</A></LI><LI><A href="/Team:Warsaw/EXTRAS">EXTRAS</A></LI><LI><A href="https://2014.igem.org/Team:Warsaw/EXTRAS#bioprocess">Bioprocess</A></LI><LI><A href="https://2014.igem.org/Team:Warsaw/EXTRAS#discussion">Discussion</A></LI><LI><A href="https://2014.igem.org/Team:Warsaw/EXTRAS#alternative_methods">Alternative methods</A></LI><LI><A href="https://2014.igem.org/Team:Warsaw/Notebook">Notebook</A></LI><LI><A href="https://2014.igem.org/Team:Warsaw/Protocoles">Protocoles</A></LI><LI><A href="/Team:Warsaw/Team">Team</A></LI><LI><A href="https://2014.igem.org/Team:Warsaw/Team#members">Members</A></LI><LI><A href="https://2014.igem.org/Team:Warsaw/Team#advisors">Advisors</A></LI><LI><A href="https://2014.igem.org/Team:Warsaw/Team#sponsors">Sponsors</A></LI><LI><A href="https://2014.igem.org/Team:Warsaw/Team#acknowledgements">Acknowledgements</A></LI><DIV class="main-content"><H1>The Project</H1><A name="parts"><H2>Background</H2></A><P align="justify">
Lanthanides are a series of fifteen chemical elements with atomic numbers 57 through 71, from lanthanum to lutetium. All but one of the lanthanides are f-block elements, corresponding to the filling of the 4f electron shell (lutetium is a d-block element). The presence of f orbitals is responsible for their unique properties such as strong paramagnetism.</P>

They are required in a variety of modern technologies, such as electronics, aviation (e.g. jet engines) and superconductors
In spite of their name 'rare earths metals' their deposits on Earth are not scarce but very dispersed. Hence, there are only a few places (e.g. China) where their concentration is high enough to be exploited on a commercial scale. Due to that situation the lanthanides market is not getting any better, especially taking into account that demand on these metals is still on increase.


1. Worldwide deposits of rare earths elements - source: [1]<CENTER><CENTER>
2. Total deposits of rare earth metals - source: [1]
Other problems with lanthanides avaibility are their difficult extraction from ore. Most lanthanides appear together and very similar physical properties make extraction and purification difficult. Despite high costs and technical difficulties, lanthanides are usually purified by ion-exchange chromatography.
[1] Chapter 6, Kołodyńska D., Hubicki Z., <I>Investigation of Sorption and Separation of Lanthanides on the Ion Exchangers of Various Types</I>, <I>Ion exchange technologies</I>, edited by Ayben Kilislioğlu, Published: November 7, 2012 under CC BY 3.0 license<A href="https://2014.igem.org/Team:Warsaw/Project"><H6>Up↑</H6></A><A name="proof_of_concept"><H2>Proof of concept</H2></A><P align="justify">
In 2013, group of prof. He from the University of Chicago published paper in <I>Journal of the American Chemical Society</I> (<I>J. Am. Chem. Soc.</I> 2013 Feb 13;135(6):2037-9) in which they described their devised lanthanide detecting system.

To accomplish this, they engineered a two-component system from <I>Salmonella enterica</I>, hence creating the first bacteria capable of detecting lanthanides.
These findings inspired us to create our bioremediating system.
A general scheme of PmrA-PmrB system.
</P><A href="https://2014.igem.org/Team:Warsaw/Project"><H6>Up↑</H6></A><A name="detailed_explanation"><H2>Detailed explanation</H2></A><P align="justify">
Initially, our project was intended to have two different parts. First being a lanthanide detecting system in BioBrick standard, much like the one constructed by group of prof. He
and the second being lanthanide binding/recovery system, which would bind lanthanides much more effectively than the detecting system.
Both of these systems were based on PmrA-PmrB two-component system, native to <I>Salmonella enterica</I>. This system consists of two proteins, PmrA and PmrB. PmrB is a transmembrane kinase with iron (III) binding motif on its extracellular loop.
When iron (III) is bound to this tag, PmrB gains kinase activity and phosphorylates PmrA. PmrA is a transcriptional factor and, upon activation, binds to pmr<SUP>C</SUP> promoter and induces expression of CheZ, a chemotaxis protein.
So much for native systems.</P><H4>Design</H4><P align="justify"><B>Detecting system</B>
Our detecting system is planned as follows:
Iron binding tag would be replaced with lanthanide binding tag (of which a various collection can be find in literature) and a reporter protein would be inserted downstream of pmr<SUP>C</SUP>. Thus, in the presence of lanthanides, fluorescence of GFP should be observed.<B>Binding system</B>
Binding system has a more complicated design. PmrA-PmrB is not changed significantly, the only modification was introduction of LBT (lanthanide binding tag) instead of iron binding motif. The difference is downstream the pmrC promoter. First of all, we need to introduce some sort of binding agent, presumably a small protein.
We decided  to use ubiquitin or an artificial structurised peptide and combine it with a LBT to create synthetic protein capable of binding lanthanide ions. Since lanthanide cations are not transported to the bacterium cell the binding agent need to be secreted outside the cytoplasm. Hence, we planned to add a signal peptide to the N or C terminus of the protein. Such modification could allow the protein to be located in the bacterial periplasmic space.
Another possible problem is connected with pmr<SUP>C</SUP>, which is a very weak promoter (even if induced by PmrA). So, even in the presence of lanthanides, expression of a binding agent could be inefficient. To overcome that, we planned to use some activating sequences to boost the expression from upon the pmr<SUP>C</SUP>. Our first idea was to put two subsequent inverters (based on different proteins, eg. tetR and lacI), which should alleviate the problem. Expression of binding agent is expected to be high in the presence of lanthanides and low in their absence.</P><TABLE border="1"><TBODY><TR><TD colspan="3">Binding agent expression</TD></TR><TR><TD>Lanthanide presence</TD><TD>pmr<SUP>C</SUP></TD><TD>pmr<SUP>C</SUP>-inverter1</TD><TD>pmr<SUP>C</SUP>-inverter1-inverter2</TD></TR><TR><TD>none</TD><TD>zero (very low)</TD><TD>high</TD><TD>low</TD></TR><TR><TD>present</TD><TD>low</TD><TD>low</TD><TD>high</TD></TR></TBODY></TABLE><P align="justify">
This may seem like an excessive mean, but we could not have invented anything subtler.</P><H4>Project goals</H4><OL><LI>Construction of a lanthanide sensor in the BioBrick standard</LI><LI>Cloning of PmrA/PmrB parts into pSB1C3 in the BioBrick standard</LI><LI>Construction of a lanthanide sensoring system with
other LBT described in the literature </LI><LI>Construction of a lanthanide binding system</LI></OL><A href="https://2014.igem.org/Team:Warsaw/Project"><H6>Up↑</H6></A><A name="modelling"><H2>Modelling</H2></A><P> Two-component systems (TCSs) are the most prevalent mechanism of transmembrane signal transduction. They control gene expression thus make bacteria respond to environmental changes and drive pathogen-host interactions. A typical TCS consists of a membrane-bound histidine kinase and
a partner response regulator protein. The pmrA/pmrB system, which our team used in the project,  also belongs to this class. pmrB is a histidine kinase and pmrA is a response regulator which strongly enhances expression upon binding to Pmr<SUP>C</SUP>. In order to understand better the mechanism of the system and to prevent any problems before starting the experiments in the wetlab we decided to create a simple model of this signaling pathway. Some other TCSs were successfully modeled before, but not the pmrA/pmrB.
</P><B>The model</B><P> When designing our model we assumed the following pathway:
<OL><LI> lanthanide ion binds to the pmrB protein which leads to its autophosphorylation, </LI><LI> phosphorylated pmrB transfers the phosphate group onto pmrA </LI><LI> phosphorylated pmrA binds to pmrC and initiate expression of the reporter GFP protein </LI><LI> dephosphorylated pmrB induces pmrA dephoshporylation </LI>
Additionally for model to work properly feedback loop in which phoshporylated pmrA induces pmrA expression is needed.
</OL></P><P>
The model diagram looks as follows:

</P><P> We concluded that quantities of observed species change according to these equations:


where:
<UL><LI><I>mRNApmrB</I>  is concentration of pmrB mRNA, the same goes for <I>mRNApmrA</I> and <I>mRNARP</I>, </LI><LI><I>L</I> is lanthanide concentration, </LI><LI><I>RP</I> is reporter protein concentration, </LI><LI><I>pmrB.bound</I> is <I>pmrB</I> with lanthanide ion bound, </LI><LI><I>prmB.bound.ph</I> is phosphorylated <I>pmrB</I> with lanthanide ion bound, </LI><LI><I>pmrA.ph</I> is phosphorylated pmrA, </LI><LI><I>ABComplex</I> is complex of <I>pmrA</I> and <I>pmrB.bound.ph</I> during <I>pmrA</I> phosphorylation, </LI><LI><I>AComplex</I>, <I>RPComplex</I> are <I>pmrA.ph</I> inductors bound to respective promoters, </LI><LI><I>ABRevComplex</I> is complex of <I>pmrA.ph</I> and <I>pmrB</I> during <I>pmrA</I> dephosphorylation </LI></UL></P><B> The parameters </B><P>
Initial parameters were found in literature as we did not make independent component measures.
</P><B> Simulation and results </B><P>
Deterministic simulations were performed using TinkerCell software. There are few bugs in it, but it allows for fast model building and makes changes to the model quite easy. 
Simulation showed that signal greatly enhances GFP expression, the its growth is exponential and correlate positively with increased concentration of lanthanide ions.
</P><P>
GFP level when there is no lanthanide ions:
</P><P>
GFP levels with 100 um of ions:
</P><B> References </B><P><I> Kierzek AM, Zhou L, Wanner BL. Stochastic kinetic model of two compo-
nent system signalling reveals all-or-none, graded and mixed mode stochastic switching responses. Mol Biosyst. 2010;6(3):531-42
</I></P><A href="https://2014.igem.org/Team:Warsaw/Project"><H6>Up↑</H6></A><A name="weeestudy"><H2>WEEE study</H2></A><P align="justify">
WEEE stands for ‘Waste Electrical and Electronic Equipment’ such as computers, mobile phones, TV-sets and fridges. 
Modern electronic products contain up to 60 elements, many of them are very valuable. The most complex of it is usually presented in printed wiring boards. Metals represent on average 23% of weight of the phone, in majority copper. Single mobile phone can contain up to 9 g Cu, 250 mg Ag, 24mg Au and 0,5 mg Tb. 
<I>Material content mobile phone [Umicore 2008]</I> 
It seems to be not too much, but we have to remember how much WEEE average European citizen products.</P><P align="justify">
By 2014 number of active cell phones will reach 7.3 billion. It gives 375 000 tons of Tb. All of it can be stored on landfill or recycled. Many of e-waste is transported to China, Ghana or Pakistan where fly dumpling is cheaper and the law is not as strict as European or American one. 
The developing countries become toxic yard for e-waste. Heavy metals and toxins leak trough landfills into waterways, poisoing local people.
</P><A href="https://2014.igem.org/Team:Warsaw/Project"><H6>Up↑</H6></A><A name="safety"><H2>Safety</H2></A><H3>1.	Introduction</H3><P align="justify">
We at Team Warsaw understand the need for good safety training and biosafe conduct in the lab.
In the following sections, we will show you how we went about making sure we didn't put anyone at an unnecessary risk either at our faculty or in the outside world.
</P><H3>2.	(Bio)safe conduct</H3><P align="justify">
Before summer, our work in the lab began with a safety training provided by our instructors. We were trained 
in accordance with the biosafety guidelines of our institution, focusing on lab-practical aspects of biosafety,
i.e. where to work with bacteria, to always do it in the same place, to account for where the bacteria-containing material is being put, to always disinfect the immediate vicinity of your workbench once the work is finished, etc. We were also taught to properly store biological material, such as bacterial broths imbued with colonies, waste agar plates, or pipette tips and plastic tubes, and handle them in a manner suitable for preventing the spread of bacteria. Whenever some biological material-containing glass, as flasks or tubes, was broken into pieces, the adjacent area was mopped dry and disinfected with ethanol,
and broken glass was stored in a separate container for glass, but only after possible remains of liquids have been removed by mopping and the pieces were disinfected by spraying with 70% ethanol.
</P><P align="justify">
Usually, we worked with DNA constructs so it was a matter of keeping everything else out the workplace (i.e. every type of contamination). Therefore, we had a set place for bacterial work on a bench, which was cleaned after each use, always worked with bacteria under conditions of closed windows and burner turned on, always worked in non-reusable gloves, which were disinfected with 70% ethanol at the start of work, proceeded to disinfect them regularly, stored contaminated plastic and liquids in separate containers suitable for autoclaving under standard  conditions (which is taken care of by our Institute) and removed contaminated materials from our lab on a weekly basis. Whenever working with bacteria, we also refrained from touching objects outside the workbench (to prevent the possible spread of bacteria) and disinfected the working place using 70% ethanol.Of course, since our DNA constructs often carried antibiotic resistance gene, we took particular care to make sure all of our liquids remained in their respective tubes and cleaned the leakage places with 70% ethanol whenever these happened. In terms of work with hazardous substances, the only one we encountered was the ethidium bromide (used when working with agarose gels): to avoid any skin contact we always used gloves and we worked on a separate bench, devoted to ethidium bromide. Gels after visualization were stored in a separate container to be destroyed.
</P><H3>3.	Biosafety level</H3><P align="justify">
In this year's project, we used the K-12 <I>E. coli</I> bacteria as chassis for our proteins: which are PmrA, PmrB and GFP. All the proteins we worked on, as well as with <I>Escherichia</I> chassis are harmless (non-pathogenic)
to humans, which allowed for classification of our work at biosafety level 1 according to
<A href="http://www.who.int/csr/resources/publications/biosafety/Biosafety7.pdf?ua=1">WHO</A>.
We therefore worked with our bacteria and constructs on ordinary open top benches, using Bunsen burners for sterility whenever aliquoting media, imbuing overnight cultures or sowing onto agar plates.
We used <I>Salmonella enterica</I> subsp. <I>enterica</I> ser. Typhimurium, which is technically a biosafety level 2 organism, as source of parts isolated by PCR. However, we worked with a non-pathogenic, attenuated
<A href="http://www.ncbi.nlm.nih.gov/pubmed/15063560">χ3987</A><I>Salmonella enterica</I> subsp. <I>enterica</I> ser. Typhimurium carrying an <I>asd</I> (aspartate dehydrogenase) gene from  <I>E.coli</I> on the p3342 plasmid, as derived from the <I>Salmonella Typhimurium</I><A href="http://www.ncbi.nlm.nih.gov/pubmed/21622747">UK-1 strain</A>. This strain is non-virulent (Δcrp, Δcya), hence provides no risk to personal or community health. Even though the genomic DNA isolation was a one-time operation, we still performed all manual activities under a suitable biological safety cabinet.</P><P align="justify">
As regarding the parts isolated, i.e. proteins coded by the BasR (PmrA) and BasS (PmrB) genes, they are said to partake in the <I>Salmonella</I> virulence. Nonetheless, these two proteins merely regulate expression of the genes, whose products (usually LPS-modifying enzymes) take part in virulence processes and so, &quot;our&quot; proteins are not involved in these processes themselves. We also used a PmrC/GFP construct-carrying plasmid (which we were glad to have received from
<A href="http://he-group.uchicago.edu/">Prof. Chuan He's group</A> at the University of Chicago), whereby GFP was expressed from a PmrA-induced PmrC promoter. The PmrC gene codes for the phosphoethanolaminetransferase enzyme, which is required for Salmonella resistance to polymyxin. However, the plasmid we used was not carrying PmrC gene, but only the mentioned promoter which [the promoter] does not constitute a biosafety risk and neither does the GFP protein.
</P><H3>4.	Safety forms</H3><P align="justify">
We submitted our <A href="https://igem.org/Safety/About_Our_Lab?team_id=1459">About Our Lab</A>  form as well as the
<A href="https://igem.org/Safety/Safety_Form?team_id=1459">Safety Form</A>, which can be found by clicking
the hyperlinks on their respective names. We did not, however, need to fill out any Check-Ins, as neither of our parts nor the chassis fell under the required categories. The White List of parts
and organisms and their Check-In necessity status can be found on the website of the
<A href="https://2014.igem.org/Safety/White_List"> Safety Hub</A>.
</P><H3>5.	Environmental concerns</H3><P align="justify">
The question about potential environmental concerns of our project was central to our attempts. However, due to the nature of the proteins expressed, our chassis bacteria have not acquired any characteristics that would enable them to compromise human immune system/other systems
or evade detection and destruction by the former or facilitate spread between people/animals, which makes them harmless from both a personal and public health point of view.
At the same time, neither the proteins encoded themselves, nor the functionality of the lanthanide detecting/binding system as a whole,  imbue the bacteria with characteristics that would convey
an evolutionary advantage against other organisms in the environment, both microorganisms and plants or animals, or act as toxins against the aforementioned, making the bacteria modified with the PmrA/PmrB system environmentally biosafe with no risk or them dominating any ecological niche. Our modified bacteria have, however, survival capabilities comparable to the wildtype ones.
There must be the point stressed, however, that since the transformed bacteria carry
a chloramphenicol resistance-encoding plasmid, the actual biosafety of the detection/binding system (i.e. prevention of HGT of the antibiotic resistance between the modified and wildtype bacteria) and so - the potential impact on the environment - depends greatly on the design of the bioreactor and the technological process, to minimize, or best prevent, the influx and efflux
of non-transformed bacteria, and microorganisms in general, into the reactor.
To sum up, our bacteria are not toxic towards either humans, plants, animals or other microorganisms, making them both biosafe both environmentally and health-wise. However, they can survive in the environment just as well as the wildtype bacteria, therefore the potential technological process of lanthanide detection and recycling must be optimized (esp. in the terms of preventing GMO bacteria efflux from the bioreactor) to prevent HGT and efflux of the acceptor bacteria back into the environment.
</P><A href="https://2014.igem.org/Team:Warsaw/Project"><H6>Up↑</H6></A><A name="possibilities_of_development"><H2>Possibilities of development</H2></A><P align="justify">
We envisage two opportunities allowing our project to be improved. First, to test more LBTs described in literature (or even design new ones) and second to create more effective binding systems.
Furthermore, we considered utilising some kind of sulphur bacterias instead of <I>E. coli</I>. Their sulphur-based metabolism and ability to survive in low pH (in which metal leaching is more efficient) makes them excellent candidates for industrial application of our project.
Another thing which is worth investigating: our system should not be present in bacterias as plasmids. It could be interesting to integrate it with bacteria genome, so it would be more stable within bacteria. We briefly investigated applying pMAT plasmid (known for it's remarkable stability in bacterial cells) in our project to fix the problem of 'deevolution' and eradication of our construct plasmids from bacterias. We also planned to construct a BioBrick compatible version of pMAT, but had to scratch the idea because of lack of time.</P><A href="https://2014.igem.org/Team:Warsaw/Project"><H6>Up↑</H6></A><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Warsaw/Project">http://2014.igem.org/Team:Warsaw/Project</A>&quot;</DIV></CENTER></CENTER></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Warsaw/Project" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Warsaw/Project" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Warsaw/Project&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Warsaw/Project&amp;oldid=389011" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></BODY></HTML>