<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Berlin_diagnostX_Experiments skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Berlin diagnostX/Experiments</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container-fluid ohnepad"><NAV class="navbar navbar-expand navbar-light bg-white fixed-top text-uppercase mt-2 pt-2"><UL class="navbar-nav ml-auto mt-2 mt-lg-0"><LI class="nav-item dropdown invisborder"><A class="nav-link dropdown-toggle" href="#" id="navbarDropdownMenuLink" data-toggle="dropdown">Team </A><DIV class="dropdown-menu"><A class="dropdown-item" href="https://2017.igem.org/Team:Berlin_diagnostX/Team">Team</A><A class="dropdown-item" href="https://2017.igem.org/Team:Berlin_diagnostX/Attributions">Attributions</A><A class="dropdown-item" href="https://2017.igem.org/Team:Berlin_diagnostX/we_special">What makes us special?</A></DIV></LI><LI class="nav-item dropdown invisborder"><A class="nav-link dropdown-toggle" href="#" id="navbarDropdownMenuLink" data-toggle="dropdown">Project </A><DIV class="dropdown-menu"><A class="dropdown-item" href="https://2017.igem.org/Team:Berlin_diagnostX/Description">Description</A><A class="dropdown-item" href="https://2017.igem.org/Team:Berlin_diagnostX/Design">Design</A><A class="dropdown-item" href="https://2017.igem.org/Team:Berlin_diagnostX/Experiments">Experiments</A><A class="dropdown-item" href="https://2017.igem.org/Team:Berlin_diagnostX/Results">Results</A><A class="dropdown-item" href="https://2017.igem.org/Team:Berlin_diagnostX/Demonstrate">Demonstration</A><A class="dropdown-item" href="https://2017.igem.org/Team:Berlin_diagnostX/InterLab">InterLab Study</A><A class="dropdown-item" href="https://2017.igem.org/Team:Berlin_diagnostX/Basic_Parts">Basic Parts</A><A class="dropdown-item" href="https://2017.igem.org/Team:Berlin_diagnostX/Improve">Improved Part</A></DIV></LI><LI class="navbar-item invisborder"><A class="nav-link" href="https://2017.igem.org/Team:Berlin_diagnostX/Collaborations">Collaborations</A></LI><LI class="nav-item dropdown invisborder"><A class="nav-link dropdown-toggle" href="#" id="navbarDropdownMenuLink" data-toggle="dropdown">Human Practices </A><DIV class="dropdown-menu"><A class="dropdown-item" href="https://2017.igem.org/Team:Berlin_diagnostX/HP/Silver">Human Practices</A><A class="dropdown-item" href="https://2017.igem.org/Team:Berlin_diagnostX/HP/Gold_Integrated">Integrated Practices</A><A class="dropdown-item" href="https://2017.igem.org/Team:Berlin_diagnostX/Engagement">Education and Public Engagement </A><A class="dropdown-item" href="https://2017.igem.org/Team:Berlin_diagnostX/Human_Practices">HP Summary </A><A class="dropdown-item" href="https://2017.igem.org/Team:Berlin_diagnostX/Safety">Safety </A></DIV></LI><LI class="navbar-item invisborder"><A class="nav-link" href="https://2017.igem.org/Team:Berlin_diagnostX/Notebook">Notebook</A></LI><LI class="nav-item dropdown invisborder"><A class="nav-link dropdown-toggle" href="#" id="navbarDropdownMenuLink" data-toggle="dropdown">Entrepreneurship </A><DIV class="dropdown-menu"><A class="dropdown-item" href="https://2017.igem.org/Team:Berlin_diagnostX/Entrepreneurship">Social Entrepreneurship</A><A class="dropdown-item" href="https://2017.igem.org/Team:Berlin_diagnostX/Applied_Design">Applied Design</A></DIV></LI><LI class="nav-item dropdown invisborder" id="judges_drop"><A class="nav-link dropdown-toggle mr-5" id="navbarDropdownMenuLink" href="#" data-toggle="dropdown">For Judges </A><DIV class="dropdown-menu"><P class="dropdown-item igem_blue"><STRONG>Medals</STRONG></P><A class="dropdown-item" href="https://2017.igem.org/Team:Berlin_diagnostX/Attributions">Attributions</A><A class="dropdown-item" href="https://2017.igem.org/Team:Berlin_diagnostX/InterLab">InterLab Study</A><A class="dropdown-item" href="https://2017.igem.org/Team:Berlin_diagnostX/Basic_Parts">Basic Parts</A><A class="dropdown-item" href="https://2017.igem.org/Team:Berlin_diagnostX/Collaborations">Collaborations</A><A class="dropdown-item" href="https://2017.igem.org/Team:Berlin_diagnostX/HP/Silver">HP Silver </A><A class="dropdown-item" href="https://2017.igem.org/Team:Berlin_diagnostX/HP/Gold_Integrated">HP Gold Integrated</A><A class="dropdown-item" href="https://2017.igem.org/Team:Berlin_diagnostX/Improve">Improved Part</A><A class="dropdown-item" href="https://2017.igem.org/Team:Berlin_diagnostX/Demonstrate">Demonstration</A><P class="dropdown-item igem_blue"><STRONG>Prizes</STRONG></P><A class="dropdown-item" href="https://2017.igem.org/Team:Berlin_diagnostX/HP/Gold_Integrated">Integrated HP</A><A class="dropdown-item" href="https://2017.igem.org/Team:Berlin_diagnostX/Engagement">Engagement </A><A class="dropdown-item" href="https://2017.igem.org/Team:Berlin_diagnostX/Entrepreneurship">Entrepreneurship</A><A class="dropdown-item" href="https://2017.igem.org/Team:Berlin_diagnostX/Applied_Design">Applied Design</A></DIV></LI></UL></NAV></DIV><DIV class="abstand_navbar conti_head"><H1 class="text-center head_us head-light">Experiments</H1></DIV><DIV class="container abstand_navbar" id="inhalt"><DIV class="row" id="button_row"><DIV class="col-4 text-center"><BUTTON type="button" style="width: 80%;cursor:pointer" class="btn btn-primary mx-auto igem_but btn-lg" id="01">Project Management</BUTTON></DIV><DIV class="col-4 text-center"><BUTTON type="button" style="width: 80%;cursor:pointer" class="btn btn-primary mx-auto igem_but btn-lg" id="02">Experiments – Overview</BUTTON></DIV><DIV class="col-4 text-center"><BUTTON type="button" style="width: 80%;cursor:pointer" class="btn btn-primary mx-auto igem_but btn-lg" id="03">Protocols</BUTTON></DIV></DIV><DIV id="projectman" style="display: none;"><DIV class="row mt-5"><DIV class="col-12"><H2 class="text-center igem_blue">Project Management</H2><P class="text-justify mb-3">Our team consists of more than 30 members who are involved into very different parts of the project. Of course, the larger part of the team is involved into the lab work at some point, but other tasks have to be covered: financial resources need to be acquired, collaborations have to be set up, digital solutions demand programming and networking, and last but not least our immense focus on science communication and public relations required some manpower (or womanpower) behind it. </P><H5 class="d-inline pl-3 my-3"> How can such a large team work?</H5><P class="mt-5 text-justify"> As a first and most important basic rule, we took the dictum “divide and conquer” to heart: no one can do everything, and even more, no one likes to do everything. Every member of our team thus associated to one or several subteams he or she personally felt most enticing. These subteams included the following affiliations: </P><OL><LI>Science Team, further divided into </LI><LI>In Vitro RNA sensor design (Team 0)</LI><LI>Sensor synthesis (Team 1)</LI><LI>Sensor screening/Cell free expression (Team 2)</LI></OL><LI>Fundraising Team</LI><LI>Entrepreneurship Team</LI><LI>Public Relations &amp; Collaborations Team</LI><P class="text-justify mb-3">We embraced a strategy of low hierarchies: Henrik Sadlowski was named project leader, and each subteam also had a team leader. However, each team member had high personal responsibility and the authority to independently make decisions in his or her field of expertise. </P><P class="text-justify">In order to ensure the individual team members in their subteams working as interlocking parts towards one goal, another key feature was required: <STRONG>communication</STRONG>. Throughout the months, we set up several interlocking strategies in order to keep information flowing between and within subteams. One part of the communication took place online – Trello, a free web-based project management application, helps to divide tasks and set deadlines. Furthermore, the Science team used Benchling, a life science data management and collaboration platform, as means of a digital lab notebook. The digital communication in place, we also set up strategies for actual face-to-face communication. 
In the first phase of the project, concerned primarily with planning and raising funds, fortnightly team leader meetings would allow for the debates needed to propel an idea towards its realization, while the day-to-day work took place in the subteams. Later, when the project was at full speed and everyone was working on very specific tasks, we replaced the team leader meetings by a “diagnost-x co-working” – every Tuesday and Thursday night, we would work at the same time in the laboratories, offices and even in the kitchen of Prof. Schuelkes facilities. These co-workings ensured many questions could be solved at the same time, not requiring everyone to follow each and every problem.
You can find out about the detailed work of the Science Team <A class="igem_blue" href="https://2017.igem.org/Team:Berlin_diagnostX/Design">here</A>.
The Entrepreneur Team shows its work <A class="igem_blue" href="https://2017.igem.org/Team:Berlin_diagnostX/Entrepreneurship">here</A>, and the results of the Collaborations and Public Relations Teams can be found <A class="igem_blue" href="https://2017.igem.org/Team:Berlin_diagnostX/Collaborations">here</A> and <A class="igem_blue" href="https://2017.igem.org/Team:Berlin_diagnostX/Human_Practices">here</A>. In order not to omit a part of work without which this project could not have existed, we lay open our funding strategy below.</P></DIV></DIV><DIV class="row mt-5"><DIV class="col-12"><H3 class="text-center igem_blue">Fundraising: How we financed one year of diagnost-x</H3><P class="text-justify">The basic principles of our project can be divided into three aspects:</P><UL><LI>Humanitarian Aid</LI><LI>Establishment of new technology</LI><LI>Encouraging young scientists</LI></UL><P class="text-justify mt-3">All of these aspects are of interest for different technological as well as charitable institutions. 
We approached these institutions by presenting on the one hand the global challenge of T.solium infection and on the other hand the opportunities that go along with the toehold switch technology. In this respect, we aimed to show potential sponsors and promoters how this technology could revolutionize field diagnostics – and to present the advantages of a team consisting of young, motivated scientists who are not yet primed to think inside their silos. 
By this approach, we were able to convince different institutions to sponsor different aspects of our project. Whereas corporates of the biotechnological sector supported us with equipment, finance and consulting corporates were convinced of our team structure, and would support our participation in the iGEM competition. Institutional funding would often be given for international collaborations such as our project with India, or our participation in the Cystinet conference. </P></DIV></DIV></DIV><DIV id="experi" style="display: none;"><DIV id="inhaltsverzeichnis"><DIV class="row"><DIV class="col-12"><H3 class="text-center igem_blue mt-4"><STRONG>Experiments: Overview</STRONG></H3></DIV></DIV><DIV class="row"><DIV class="col-6"><UL><LI><A class="igem_blue" href="#transkriptom">Transcriptome analysis of T. solium</A></LI><LI>RNA Isolation</LI><LI>RNA sequencing</LI><LI><A class="igem_blue" href="#bioinformatics">Bioinformatics</A></LI><LI><A class="igem_blue" href="#switch_synthese">Switch Synthesis</A></LI><LI>Extension PCR</LI><LI>Nested PCR</LI><LI>Colony PCR</LI><LI>Gel Electrophoresis</LI><LI>DNA Clean-Up</LI></UL></DIV><DIV class="col-6"><UL><LI><A class="igem_blue" href="#switch_testing">Toehold Switch Sensor Test – Testing Pipeline</A></LI><LI>High troughput screening in liquid medium</LI><LI>Reliable cell free expression on cellulose membranes</LI><LI><A class="igem_blue" href="#screening_opti">Optimizing the Screening Pipeline</A></LI><LI><A class="igem_blue" href="#andere_experimente">Other Experiments</A></LI><LI>Preparation of nucleic acids (DNA preparation, Restriction digest/ DNA ligation, in vitro transcription, RNA purification)</LI><LI>Preparation of sensors for iGEM Submission (Mutagenesis &amp; Cloning)</LI><P class="mt-4"><STRONG>Please find detailed protocols under the &quot;Protocol&quot; Section</STRONG></P></UL></DIV></DIV></DIV><DIV id="transkriptom"><DIV class="row"><DIV class="col-12"><H3 class="text-center igem_blue mt-4">Transcriptome Analysis of <EM>T. solium</EM></H3></DIV></DIV><DIV class="row mt-4"><DIV class="col-6"><H5 class="igem_blue text-center pb-1">RNA Isolation</H5><P class="text-justify">Toehold switch sensors are based on synthetic biology. Essentially, they are RNA molecules which code for a reporter protein. They consist of a specific toehold sequence, a ribosome-binding site (which is important for the production of proteins) and a sequence for the reporter protein. The reporter protein can only be produced if the sensor has bonded with its specific target RNA sequence [4]. When producing the sensors, we can select both the toehold sequence and the reporter protein with complete flexibility to match our needs. In this way we can fashion our Wormspotter so that it only sends a desired signal when it binds to RNA molecules specific to T. solium. We are planning to use T. solium-specific RNA sequences for the toehold sequence and beta-Galactosidase as a reporter protein.</P></DIV><DIV class="col-6"><H5 class="igem_blue text-center pb-1">RNA Sequencing</H5><P class="text-justify">Toehold switch sensors are based on synthetic biology. Essentially, they are RNA molecules which code for a reporter protein. They consist of a specific toehold sequence, a ribosome-binding site (which is important for the production of proteins) and a sequence for the reporter protein. The reporter protein can only be produced if the sensor has bonded with its specific target RNA sequence [4]. When producing the sensors, we can select both the toehold sequence and the reporter protein with complete flexibility to match our needs. In this way we can fashion our Wormspotter so that it only sends a desired signal when it binds to RNA molecules specific to T. solium. We are planning to use T. solium-specific RNA sequences for the toehold sequence and beta-Galactosidase as a reporter protein.</P></DIV></DIV></DIV><DIV id="bioinformatics"><DIV class="row"><DIV class="col-12"><H3 class="text-center igem_blue mt-4">Bioinformatics for Switch Design</H3></DIV></DIV><DIV class="row mt-4"><DIV class="col-12"><DIV class="text-center"><BUTTON style="cursor:pointer" type="button" class="btn btn-primary igem_but btn-lg mx-auto"><A href="https://2017.igem.org/Team:Berlin_diagnostX/Design#team0">More Information: Visit Our Design Page</A></BUTTON></DIV></DIV></DIV></DIV><DIV id="switch_synthese"><DIV class="row"><DIV class="col-12"><H3 class="text-center igem_blue mt-4">Switch Synthesis</H3></DIV></DIV><DIV class="row mt-4"><DIV class="col-6"><H5 class="igem_blue text-center pb-1">Extension PCR</H5><P class="text-justify"><STRONG>Goal:</STRONG>Assembling our toehold sensors: combining the hairpin-region with a LacZ reporter element</P><P class="text-justify"><STRONG>Description</STRONG>Each sensor molecule should consist of the core sensor (recognition structure, characteristic hairpin, ribosome binding site and linker domain), a LacZ reporter element and a T7 promoter region.</P><OL><LI>In a first step, the core sensor was amplified from a circular DNA template.</LI><LI>n two PCR-based steps, the recognition and hairpin region was attached to a LacZ reporter element, using primers specifically designed for each individual sensor.</LI><LI>In a third PCR step, the sensors were purified and a t7 promoter region was added.</LI></OL></DIV></DIV><DIV class="row mt-4"><DIV class="col-6"><H5 class="igem_blue text-center pb-1">Nested PCR</H5><P class="text-justify"><STRONG>Goal:</STRONG>Combining two PCR steps into one PCR cycle (increased time-efficiency)</P><P class="text-justify"><STRONG>Description</STRONG>As mentioned above, the assembly of the RNA sensors via extension PCR usually requires two PCR steps in order to amplify the core sensor and furnish it with a LacZ reporter element. The approach of nested PCR can reduce this to one PCR cycle, reducing time and the amount of non-specific PCR products. The approach of nested PCR consists of two processes:</P><OL><LI>In a first run of PCR, the DNA is amplified with a first set of primers. As alternative/similar primer binding sites cannot be fully excluded, a fraction of PCR product will be non-specific.</LI><LI>In a second step, the product from the first reaction undergoes a second PCR with a second set of primers. The likelihood of any unspecific product containing binding sites for this second primer set is low, reducing overall contamination.</LI><LI>In a third PCR step, the sensors were purified and a t7 promoter region was added.</LI></OL></DIV><DIV class="col-6"><H5 class="igem_blue text-center pb-1">Colony PCR</H5><P class="text-justify"><STRONG>Goal:</STRONG>Determine which transformed colony contains the plasmid with the correctly integrated insert.</P><P class="text-justify"><STRONG>Description</STRONG>A colony-PCR is a variant of PCR used to directly amplify specific regions of vector DNA from bacteria without having to extract and clean the DNA beforehand. Thus, instead of adding a clean template DNA strand to the PCR reaction mix, whole bacteria from the colonies in question are used. This requires the following steps</P><OL><LI>Primers are chosen in a way that ensures a PCR product containing both vector- and insert-sequences. This allows for monitoring for inverted inserts by checking the size of the PCR product.</LI><LI>Colonies are picked with a pipet tip and transferred into purified water, where they get osmotically lysed – this step ensures that the bacterial cell wall is broken and genetic material is accessible</LI><LI>Then, PCR master mix is added to the bacteria lysate</LI><LI>Finally, PCR is performed in a thermocycler</LI></OL></DIV></DIV><DIV class="row mt-4"><DIV class="col-6"><H5 class="igem_blue text-center pb-1">Gel electrophoresis</H5><P class="text-justify"><STRONG>Goal:</STRONG>Evaluating the result of a PCR </P><P class="text-justify"><STRONG>Description</STRONG>In order to characterize the product of a PCR, gel electrophoresis can be used to determine the length of the resulting DNA fragment. This is achieved by the following steps:</P><OL><LI>An agarose gel is prepared.</LI><LI>The PCR-Product is mixed with a dye and inserted into “pockets” of the gel, along with a “ladder” containing DNA fragments of known size</LI><LI>By applying an electrical field, the negatively charged DNA is moved through the gel matrix, separating the DNA fragments by length.</LI><LI>The DNA bands are then compared to the ladder and their size can be calculated. By comparing the observed length to the expected length of the fragment,  the success of the PCR can be evaluated</LI></OL></DIV><DIV class="col-6"><H5 class="igem_blue text-center pb-1">DNA Clean Up </H5><P class="text-justify"><STRONG>Goal:</STRONG>Removing excess nucleotides, salts and additives. 
        </P><P class="text-justify"><STRONG>Description</STRONG></P></DIV></DIV></DIV><DIV id="switch_testing"><DIV class="row"><DIV class="col-12"><H3 class="text-center igem_blue mt-4">Toehold Switch Sensor Test – Pipeline</H3></DIV></DIV><DIV class="row mt-4"><DIV class="col-6"><H5 class="igem_blue text-center pb-1">High throughput screening in liquid medium</H5><P class="text-justify"><STRONG>Goal:</STRONG>High-throughput screening for sensors that react specifically to target RNA</P><P class="text-justify"><STRONG>Description</STRONG>In silico design and PCR allows for assembly of a large number of potential sensor candidates. In order to evaluate which of these sensors reacts to the target RNA in a sensitive and specific manner, the following steps are required:</P><OL><LI>A master mix of cell free expression system, reporter molecule and certain amount of target RNA or control RNA is prepared and transferred to a special low-volume 384-well plate.</LI><LI>Adding sensor DNA to each well</LI><LI>Measure absorption at 405nm (yellow – starting color) and 560nm (violet – end color) in a GloMax Discover ® plate reader (Promega) for 2-4 hours at 37° C in order to follow the kinetic of the reaction from yellow to violet.</LI><LI>Results files are evaluated using a python script. By automating the process of data evaluation, a empirically sound conclusion can be reached in less than 15 minutes.</LI></OL></DIV></DIV><DIV class="row mt-4"><DIV class="col-6"><H5 class="igem_blue text-center pb-1">Reliable cell free expression on cellulose membranes</H5><P class="text-justify"><STRONG>Goal:</STRONG>Membranes are known to be efficient microfluidic environments, but due to the charge of nucleic acids and the negative charge of cellulose, optimization is required to make the system storable and the color change easily recognizable.</P><P class="text-justify"><STRONG>Description</STRONG>In silico design and PCR allows for assembly of a large number of potential sensor candidates. In order to evaluate which of these sensors reacts to the target RNA in a sensitive and specific manner, the following steps are required:</P><UL><LI><STRONG>Blocking the charge of the cellulose membrane.</STRONG> Classic approaches of membrane blocking like incubation with 5% BSA or milk powder failed due to negative interaction with the cell free expression system. Hence we used an amphiphilic component, creating a liquid barrier shielding the sensor system from negative charges and successfully creating a stable color reaction on the membrane.</LI><LI><STRONG>Altering the composition of our sensor system components, to maximize color reaction.</STRONG> We discovered that about one third of color intensity is lost on membranes in comparison to fluid environments. Reactions on membranes thus require to double the amount of sensor molecule added to the reaction</LI><LI><STRONG>Stabilizing the cell-free system for storage by lyophilisation</STRONG> We succeeded in freeze-drying the cell-free expression system onto cellulose membrane and were furthermore able to trigger an active color change after resuspension.</LI></UL></DIV></DIV></DIV><DIV id="screening_opti"><DIV class="row"><DIV class="col-12"><H3 class="text-center igem_blue mt-4">Optimizing the Screening Pipeline</H3></DIV></DIV><DIV class="row mt-4"><DIV class="col-6"><P class="text-justify"><STRONG>Goal:</STRONG>Establishing the PureExpress® cell free protein expression kits (PureExpress®, NEB) as our main tool for RNA detection and assess its capabilities.</P><P class="text-justify"><STRONG>Description</STRONG>The PureExpress®-System is composition of completely recombinant molecules to effectively transcribe any form of DNA into mRNA and then translate this into a protein. To use this system for field diagnostics, it has to meet the following requirements:</P><UL><LI>Heat stability up to a temperature of 50°C</LI><LI>High efficiency even with low volumes of reagents and/or low concentrations</LI><LI>Stability towards changes in electrolyte concentration/ addition of foreign molecules</LI><LI>Long-term storability.</LI></UL></DIV><DIV class="col-6"><P class="text-justify">We verified this by evaluating the kinetics of sensor reactions in different experimental conditions, including different temperatures, different reactant concentrations, changes in buffer as well as re-measurements after storage.We could show that the system meets all these requirements. Especially noteworthy, we were able to create significant color change in a volume of 2,5 uL which is only 10% of the volume recommended by the manufacturer. Furthermore, we could show that the system is still highly active in concentrations of 40-50%. This makes it possible to use the system with patient samples of only estimated concentration without risking reduced functionality. Last but not least, the system is not vulnerable to possibly interfering molecules, as for example the chlor-phenol-red used  as a reporter dye. 
         We concluded that the PureExpress® - System is most suitable for our essay; also due to the fact, that it composed of recombinant parts instead of cell lysate and so has very few variability</P></DIV></DIV><DIV id="andere_experimente"><DIV class="row"><DIV class="col-12"><H3 class="text-center igem_blue mt-4">Other Experiments</H3></DIV></DIV><DIV class="row"><DIV class="col-12"><H4 class="text-center mb-2">Preparation of nucleic acids<P class="text-justify">For our experiments, a large set of different DNAs and RNAs - primers, templates, targets and controls – are required in varied concetrations. Thus, a set of standardized but easily adaptable protocols to extract and purify nucleic acids was established.</P></H4><H5 class="text-center igem_blue mb-1">DNA preparation out of E.coli bacteria (Mini-prep, Midi-prep)</H5><P class="text-justify"><STRONG>Goal:</STRONG>Frequently used DNA (and RNA coding DNA) is stored as plasmid in apathogenic and nuclease-down-regulating strands of E.coli bacteria and extracted when needed.</P><P class="text-justify"><STRONG>Description</STRONG>: Depending of the required quantity, we use different techniques to extract the DNA from the bacteria:</P><UL><LI><STRONG>Miniprep: </STRONG>Lysis of up to 5mL of bacterial culture and isolation of plasmids via column purification and repeated centrifugation. <EM>Suitable for desired concentrations between 50-100ng/uL and volumes of up 30uL; mostly used for DNA that is purposed for in vitro transcription</EM>.</LI><LI><STRONG>Midiprep: </STRONG>Lysis of up to 250 mL of bacterial culture and isolation of Plasmids via vacuum assisted column purification and high speed centrifugation. <EM>High throughput method for concentration requirements of 200-400ng/uL and volumes of up to 500-1000uL.</EM></LI></UL></DIV></DIV><DIV class="row mt-4"><DIV class="col-6"><H5 class="igem_blue text-center pb-1">DNA linearization</H5><P class="text-justify"><STRONG>Goal:</STRONG> Both for cell-free expression and in vitro transcription, plasmids need to be transformed to a linearized form.</P><P class="text-justify"><STRONG>Description</STRONG> To this end, we use high fidelity restriction enzymes (e.g. EcoRI-HF). The choice of restriction enzyme determines the number of cuts in the plasmid: This can be either one cut to simply linearize the entire plasmid, or two cuts to cut out the plasmid backbone without any insert. The success of the digest can be verified via gel electrophoresis.</P></DIV><DIV class="col-6"><H5 class="igem_blue text-center pb-1">In vitro transcription</H5><P class="text-justify"><STRONG>Goal:</STRONG>Obtain specific concentrations of purified target RNA need to be added to the sensor test reaction.</P><P class="text-justify"><STRONG>Description</STRONG>The DNA is obtained by transcribing linearized DNA in vitro with a SP6 RNA-Polymerase (RiboMax®, Promega).  This kit requires adding linearized template DNA, RNA polymerase, nucleotides and water into a microcentrifuge tube and incubating it at 37 °C. The protocol allows for transcribing RNA in the range of double microgram from smallest concentrations of DNA template in less than one hour. </P></DIV></DIV><DIV class="row"><DIV class="col-12"><H5 class="igem_blue text-center pb-1">RNA purification</H5><P class="text-justify">We require high RNA concentrations to minimize the volume we have to add to the cell free expression system – too high dilutions significantly decrease its efficiency.  To ensure RNA yields with maximal concentration and purity of salts/contaminants, we don’t use column purification systems but rather rely on phenol-chloroform extraction. This method efficiently separates organic and inorganic phases to get rid of any unwanted salt contamination. Moreover the final yield is much higher than with column purification, which mostly reduces final concentration considerably due to column occlusion by contaminants. </P></DIV></DIV></DIV><DIV class="row mt-5"><DIV class="col-12"><H4 class="text-left mb-2 igem_blue">Cloning into pSB1C<DIV class="row"><DIV class="col-6"><P class="text-justify">All of our experiments were PCR-Based. For this reason, we had to clone our sensors into pSB1C for iGEM-Submission only and were selecting two strategies to reach this goal: </P><UL><LI>Cloning of our PCR-Product into pGEM®-T Easy vector and subsequent insertion of the insert to pSB1C via Not1-Restriciton sites.</LI><LI>Amplifying our PCR-Product using tailed primers introducing the restriction sites SpeI and XbaI. Direct insertion into pSB1C</LI></UL><P class="text-justify"><STRONG>Methods: </STRONG>For TA-Cloning 5,5 uL of PCR-Product, generated by Phusion PCR according to our SOPs, were incubated with 0,5 uL Units GoTaq® Flexi DNA Polymerase (5U/uL) from Promega, 2ul dATP (2mM), and 2uL GoTaq® Flexi Buffer (5X) for 30 minutes at 70°C. Ligation was performed over night at 4°C using T4-Ligase (M180A Promega) at a molar vector:insert ratio of 1:1 and transformed to ultracompetent DH5a-Cells (NEB). Positive clones were identified by colony PCR using a primer annealing to the T7-promoter as forward-primer and two reverse primers annealing to lacZ. These primers worked well for sequencing the switches already and thus we decided to use them again. FW_T7: GCGAATTAATACGACTCACTATAGGG  Rev_Seq1 (310BP-Product): TGAATGGCGAATGGCGCTTTG 
        Rev_Seq2 (445 BP-Product): CATCTACACCAACGTGACCTATC  5mL bacterial cultures were grown from 4 positive clones per sensor. DNA was isolated the next day using NucleoSpin MiniPrep System (Merchery-Nagel) according to the manufacturer’s instructions. DNA was digested using NotI HF (1U/ug of DNA in CutSmart Buffer) for one hour and heat-inactivated for 20 minutes at 65°C. 2,5 ug of digested DNA was loaded on a gel, bands were excised and cleaned up using ZymoClean™ Gel DNA recovery kit according to the manufacturer’s instructions. Recovery of DNA was low (12-30ng/uL). Never the less we tried to clone the Gel-Purified, dephosphorylated pSB1C-Backbone, which was a kind gift from iGEM Potsdam. Ligation was performed using Quick Ligase (NEB) with a Backbone:Insert Ration of 1:1 and 1:3. Subsequent transformation into ultracompetent DH5a-Cells and growth on Agar plates containing Chloramphenicol did not result in any colonies.        
    </P></DIV><DIV class="col-6"><P class="text-justify">For direct cloning into pSB1C using tailed primers, we designed primers introducing XbaI to the 5’ end and SpeI to the 3’ end of the PCR Product. For forward primers two variations were designed to speed experiments up: 

FW1: XbaI.T7 GCCGCTTCTAGAtaatacgactcactatagg
FW2: XbaI.T7.6NT GCCGCTTCTAGAGCGAATTAATACGACTCAC
Rev1: SpeI.LacZ.stop  GCCGCTACTAGTTTATTTTTGACACCAGACC


Inserts were amplified using Phusion DNA Polymerase (Thermo Fisher) and cleaned up using ZymoClean™ PCR CleanUp-Columns (Zymo Research). Purified PCR products were digested using SpeI and XbaI for 1 hour (all restriction enzymes and Cutsmart Buffer were purchased by NEB).Following heat inactivation at 80°C for 20 minutes the PCR product was purified as described above. The pSB1C-Backbone was cut using the restriction enzymes SpeI and XbaI. To decrease the chance for re-ligation of the vector, we dephosphorylated the vector backbone using calf intestinal alkaline phosphatase (CIP) according to the manufacturer’s instructions (NEB). The digested and phosphorylated backbone plasmid was run on a 1% agarose gel and the aprox. 2.2kb band was cut and purified using the Gel Extraction ZymoClean Kit (Zymo Research). Subsequently, we ligated the purified digested PCR products with the phosphorylated pSB1C backbone using Quick Ligase for 15 minutes at room temperature and transformed the ligated product into ultracompetent DH5a cells. Positive colonies were identified using Colony PCR as mentioned above but with a different forward primer, that binds to the iGEM backbone (VF2 primer ATTACCGCCTTTGAGTGAGC). Bacterial cultures were grown over night and DNA was purified using Quick DNA universal Kit (ZymoResearch) according to the manufacturer’s instructions.</P></DIV></DIV></H4><H3 class="text-center igem_blue mt-4"><STRONG>Protocols</STRONG></H3></DIV></DIV><DIV class="row"><DIV class="col-6"><UL><LI><A class="igem_blue" href="https://static.igem.org/mediawiki/2017/4/47/T--Berlin_diagnostX--Protocol_RNA_Iso_Seq.pdf">RNA Isolation and Sequencing</A></LI><LI><A class="igem_blue" href="https://static.igem.org/mediawiki/2017/7/76/T--Berlin_diagnostX--12_Extension_PCR%2C_Gel_electrophoresis_%26_DNA_Clean_UP.pdf">Extension PCR, Electrophoresis and DNA Clean-Up</A></LI><LI><A class="igem_blue" href="https://static.igem.org/mediawiki/2017/4/49/T--Berlin_diagnostX--NESTED.pdf">Nested PCR</A></LI><LI><A class="igem_blue" href="https://static.igem.org/mediawiki/2017/a/a2/T--Berlin_diagnostX--Protocol_Colony_PCR.pdf">Colony PCR</A></LI><LI><A class="igem_blue" href="https://static.igem.org/mediawiki/2017/6/6a/T--Berlin_diagnostX--Sensor_Screening.pdf">Toehold Switch Sensor – High troughput screening</A></LI><LI><A class="igem_blue" href="https://static.igem.org/mediawiki/2017/3/37/T--Berlin_diagnostX--Protocol_Mutagenesis.pdf">Sensor Mutagenesis and Cloning</A></LI></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>