<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Berlin_diagnostX_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Berlin diagnostX/Results</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container-fluid ohnepad"><NAV class="navbar navbar-expand navbar-light bg-white fixed-top text-uppercase mt-2 pt-2"><UL class="navbar-nav ml-auto mt-2 mt-lg-0"><LI class="nav-item dropdown invisborder"><A class="nav-link dropdown-toggle" href="#" id="navbarDropdownMenuLink" data-toggle="dropdown">Team </A><DIV class="dropdown-menu"><A class="dropdown-item" href="https://2017.igem.org/Team:Berlin_diagnostX/Team">Team</A><A class="dropdown-item" href="https://2017.igem.org/Team:Berlin_diagnostX/Attributions">Attributions</A><A class="dropdown-item" href="https://2017.igem.org/Team:Berlin_diagnostX/we_special">What makes us special?</A></DIV></LI><LI class="nav-item dropdown invisborder"><A class="nav-link dropdown-toggle" href="#" id="navbarDropdownMenuLink" data-toggle="dropdown">Project </A><DIV class="dropdown-menu"><A class="dropdown-item" href="https://2017.igem.org/Team:Berlin_diagnostX/Description">Description</A><A class="dropdown-item" href="https://2017.igem.org/Team:Berlin_diagnostX/Design">Design</A><A class="dropdown-item" href="https://2017.igem.org/Team:Berlin_diagnostX/Experiments">Experiments</A><A class="dropdown-item" href="https://2017.igem.org/Team:Berlin_diagnostX/Results">Results</A><A class="dropdown-item" href="https://2017.igem.org/Team:Berlin_diagnostX/Demonstrate">Demonstration</A><A class="dropdown-item" href="https://2017.igem.org/Team:Berlin_diagnostX/InterLab">InterLab Study</A><A class="dropdown-item" href="https://2017.igem.org/Team:Berlin_diagnostX/Basic_Parts">Basic Parts</A><A class="dropdown-item" href="https://2017.igem.org/Team:Berlin_diagnostX/Improve">Improved Part</A></DIV></LI><LI class="navbar-item invisborder"><A class="nav-link" href="https://2017.igem.org/Team:Berlin_diagnostX/Collaborations">Collaborations</A></LI><LI class="nav-item dropdown invisborder"><A class="nav-link dropdown-toggle" href="#" id="navbarDropdownMenuLink" data-toggle="dropdown">Human Practices </A><DIV class="dropdown-menu"><A class="dropdown-item" href="https://2017.igem.org/Team:Berlin_diagnostX/HP/Silver">Human Practices</A><A class="dropdown-item" href="https://2017.igem.org/Team:Berlin_diagnostX/HP/Gold_Integrated">Integrated Practices</A><A class="dropdown-item" href="https://2017.igem.org/Team:Berlin_diagnostX/Engagement">Education and Public Engagement </A><A class="dropdown-item" href="https://2017.igem.org/Team:Berlin_diagnostX/Human_Practices">HP Summary </A><A class="dropdown-item" href="https://2017.igem.org/Team:Berlin_diagnostX/Safety">Safety </A></DIV></LI><LI class="navbar-item invisborder"><A class="nav-link" href="https://2017.igem.org/Team:Berlin_diagnostX/Notebook">Notebook</A></LI><LI class="nav-item dropdown invisborder"><A class="nav-link dropdown-toggle" href="#" id="navbarDropdownMenuLink" data-toggle="dropdown">Entrepreneurship </A><DIV class="dropdown-menu"><A class="dropdown-item" href="https://2017.igem.org/Team:Berlin_diagnostX/Entrepreneurship">Social Entrepreneurship</A><A class="dropdown-item" href="https://2017.igem.org/Team:Berlin_diagnostX/Applied_Design">Applied Design</A></DIV></LI><LI class="nav-item dropdown invisborder" id="judges_drop"><A class="nav-link dropdown-toggle mr-5" id="navbarDropdownMenuLink" href="#" data-toggle="dropdown">For Judges </A><DIV class="dropdown-menu"><P class="dropdown-item igem_blue"><STRONG>Medals</STRONG></P><A class="dropdown-item" href="https://2017.igem.org/Team:Berlin_diagnostX/Attributions">Attributions</A><A class="dropdown-item" href="https://2017.igem.org/Team:Berlin_diagnostX/InterLab">InterLab Study</A><A class="dropdown-item" href="https://2017.igem.org/Team:Berlin_diagnostX/Basic_Parts">Basic Parts</A><A class="dropdown-item" href="https://2017.igem.org/Team:Berlin_diagnostX/Collaborations">Collaborations</A><A class="dropdown-item" href="https://2017.igem.org/Team:Berlin_diagnostX/HP/Silver">HP Silver </A><A class="dropdown-item" href="https://2017.igem.org/Team:Berlin_diagnostX/HP/Gold_Integrated">HP Gold Integrated</A><A class="dropdown-item" href="https://2017.igem.org/Team:Berlin_diagnostX/Improve">Improved Part</A><A class="dropdown-item" href="https://2017.igem.org/Team:Berlin_diagnostX/Demonstrate">Demonstration</A><P class="dropdown-item igem_blue"><STRONG>Prizes</STRONG></P><A class="dropdown-item" href="https://2017.igem.org/Team:Berlin_diagnostX/HP/Gold_Integrated">Integrated HP</A><A class="dropdown-item" href="https://2017.igem.org/Team:Berlin_diagnostX/Engagement">Engagement </A><A class="dropdown-item" href="https://2017.igem.org/Team:Berlin_diagnostX/Entrepreneurship">Entrepreneurship</A><A class="dropdown-item" href="https://2017.igem.org/Team:Berlin_diagnostX/Applied_Design">Applied Design</A></DIV></LI></UL></NAV></DIV><DIV class="abstand_navbar conti_head"><H1 class="text-center head_us head-light">Results</H1></DIV><DIV class="row"><DIV class="col-8"><H4 class="text-justify">Throughout the iGEM season 2017 we developed Standard Operating Procedures and an in silico sensor design algorithm that is described in the design section. These process innovations led to the subsequent optimization of a well performing sensor for Taenia solium:</H4></DIV></DIV><DIV class="row"><DIV class="col-8"><H4 class="text-justify">In the first series of screenings we reproduced published data of sensor 27B from the 2016 Pardee et al paper describing a toehold switch for Zika Virus:</H4></DIV></DIV><DIV class="row"><DIV class="col-8"><H4 class="text-justify">We tested a first series of sensors that were designed by predicting the secondary structure of possible sensors and then selecting sensors with a low normalized ensemble defect. A low normalized ensemble defect indicates that a potential sensor spontaneously forms a secondary structure that is similar to the secondary structure of a toehold switch. This design principle led to sensors that did not show unspecific reaction, but that neither reacted upon adding trigger RNA. Trigger RNA was the only published RNA-Sequence of <I>T. solium</I>: <B>TSO_31</B></H4></DIV></DIV><DIV class="row"><DIV class="col-8"><H4 class="text-justify">In the next series of sensors tested, we replaced the selection based on the normalized ensemble defect with the score calculation mentioned in the design section. This led to a significant improvement of the sensor opening. At the same time, sensors showed a quite high degree of unspecific reactions.</H4></DIV></DIV><DIV class="row"><DIV class="col-12"><H3 class="text-center igem_blue mb-3">Transcriptome</H3></DIV></DIV><DIV class="row"><DIV class="col-6"><P class="text-justify">We obtained raw RNAseq data of Taenia Solium from GEO database (GSM2227058) and mapped it against a published genome of T. solium (PRJNA170813). In this way we identified 215 potential targets for sensor development. The interactive map on the left shows these targets. The darker the dot, the higher the expression level. Upon clicking on a dot, absolute expression level, Coding Sequence, information on splicing and GeneDB identifier is displayed. You can download the underlying data in an excel file <A href="https://static.igem.org/mediawiki/2017/3/3a/T--Berlin_diagnostX--RNAseq2.xlsx">here</A>.

In our analysis we ranked the 215 potential RNA targets by their expression level. The target with the highest expression level was chosen for further sensor development: TsM_000297600. </P></DIV></DIV><DIV class="row"><DIV class="col-8"><H4 class="text-justify">Upon identifying a new, unique and highly expressed RNA molecule in the transcriptome of Taenia Solium, we created a new series of sensors aiming at this new target without changing the design algorithm. This approach led to two promising sensor candidates that were further analyzed:</H4></DIV></DIV></DIV><DIV class="row"><DIV class="col-12"><H3 class="text-center igem_blue mb-3">RNAseq</H3></DIV></DIV><DIV class="row"><DIV class="col-6"><P class="text-justify">

By visiting India as part of our Human Practices activities and by building up a strong collaboration with iGEM IIT Delhi and hospitals as well as NGOs in Varanasi, we were able to obtain RNA-samples from lysed T. solium eggs. We were able to isolate RNA from these samples and sent it for sequencing to the sequencing core facility of the Max-Planck-Institute for Molecular Genetics in Berlin. We received two sets of RNAseq data from our samples. One containing 48 million fragments (96 miollion total reads) and the other one containing 44 million fragments (88 million total reads). Since the two sets of data could be treated as technical replicates, reads were combined and aligned to the T. solium reference genome. In total, 70.98% of the reads were aligned unique mappings, 13.57% multiple mappings and only 15.42% were unmapped reads.



The figure on the right shows a large amount of alignments to genes that may be associated to mitochondrial RNA which may be highly expressed in T. solium eggs and could be an interesting target for future sensor design. Until now, analysis of our RNA sequencing data still is in progress. To this date we can say that we are to our knowledge the first group to describe RNA sequencing data from T. solium eggs. This is particularly important, since RNA expression may differ in T. solium eggs compared to samples from the whole organism. To this date we were able to find out, that many targets we identified in our first analysis of the whole worm transcriptome, are present in eggs, but in much lower expression levels. This underlines again the importance of this sample and its potential to contribute to the discovery of new RNAs that may serve as targets for tapeworm diagnostics.


</P></DIV></DIV><DIV class="row"><DIV class="col-12"><H3 class="text-center igem_blue mb-3">Nested PCR</H3></DIV></DIV><DIV class="row"><DIV class="col-6"><P class="text-justify">New RNA sequencing data and new transcriptome analysis leads to constantly changing targets. Increasing the scale of our test pipeline, while reducing the time and costs spent during the generation of a single sensor, is essential to a successful future of our project. Thus, we focused on establishing a reliable Nested PCR method by which the switch production can be performed within a single reaction.



Using the single-tube nested PCR method, we add both extension primers and the template simultaneously to the PCR mastermix. First, the primer P1 will add the individual first half of the toehold loop and the conserved loop region onto the reporter gene sequence. The second extension primer P2 anneals to the conserved loop region that was added through the P1 primer. The P2 primer extends the PCR product by the second half of the individual toehold switch design flanked by at T7 promoter at the 5’ end. This creates a full dsDNA template containing 5’ a T7 promoter for RNA synthesis followed by the toehold switch and its loop structure that is ending into the reporter gene 3’ end.



In our study, we performed first trials of the nested PCR method on two toehold switch designs. The size of the amplified switches of the first PCR was evaluated on an agarose gel (Fig. 1). All the PCR products were of the expected length (aprox. 3.2 kb). Having this auspicious result in mind, we performed a second attempt with 7 other switch designs. This time 5 out of 7 nested attempts showed the desired outcome (data not shown). 
</P></DIV><DIV class="col-6"><P class="text-justify"><B>Fig. 1:</B> Agarose gel (1%) showing Nested PCR products of switch TSO#898 in lane 1 and TSO#16 in lane 3. Nested PCR was performed on a PCR product of lacZ containing the constant linker of a diagnostX toehold switch. Lane 2 and 4 contain Nested template with P2 primer only. This control aims at excluding unspecific binding of P2 to the template which would lead to false positive results.
</P></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>