Team:Berlin diagnostX/Design
Team
Team Attributions What makes us special?
Project
Description Design Experiments Results Demonstration
InterLab Study Basic Parts Improved Part
Collaborations
Human Practices
Human Practices Integrated Practices Education and Public Engagement HP Summary Safety
Notebook
Entrepreneurship
Social Entrepreneurship Applied Design
For Judges
Medals
Attributions InterLab Study Basic Parts Collaborations HP Silver HP Gold Integrated Improved Part Demonstration
Prizes
Integrated HP Engagement Entrepreneurship Applied Design
Design
The vision of diagnost-x
Developing a low cost, easy-to-handle diagnostic test for infection with the tapeworm Taenia solium
Basics: RNA toehold switch
Toehold switches are RNA molecules that can regulate the downstream translation (in our case of a reporter protein) depending on the presence or absence of specific trigger RNA.
They consist of:
a specific single-stranded toehold sequence
a ribosome-binding site (RBS)
a coding sequence for the reporter protein.
The reporter protein can only be produced if the toehold domain (recognition site) of the sensor has bound to its specific trigger RNA sequence. In absence of trigger RNA, the toehold switch forms a hairpin structure that prevents the ribosome from binding to the RBS, and thereby the translation (production of the reporter protein) can’t be initiated.
However, an RNA sensor alone is not sufficient for a screening test. Naked RNA cannot produce a protein nor induce a colour change – we thus relied on cell-free expression systems, a synthetic alternative to cell culture providing both easy handling as well as a controlled micro-environment for synthetic biology.
Basics: Cell free expression system
A cell free expression system is a system allowing for the production of protein using biological machinery without the use of living cells. The in vitro protein synthesis environment is highly controllable and more stable in the face of changes in temperature, buffers or similar interfering factors.
In order to establish our diagnostic test we thus needed:
An RNA sensor designed to target the tapeworm
A cell free expression system able to produce a color change
T. solium specific trigger RNA
(for field use: a carrier system to make the cell-free expression system storable)
We thus divided the work over three subteams:
In-silico RNA sensor design (Team 0)
Sensor synthesis (Team 1)
Sensor Screening/Cell free expression (Team 2)
Preliminary Data (Bioinformatics)
We started the project knowing that there is only one potential target RNA published from T. solium. For this reason one of our main goals was to shed light into the transcriptome of T. solium with a special focus on T. solium eggs, which are excreted with stool and easily accessible for diagnosis.   We reached this goal by obtaining raw RNAseq data of the whole worm from GEO database (GSM2227058) and mapped it against a published genome of T. solium (PRJNA170813). We identified 215 potential targets for sensor development by mapping the transcripts against tapeworms that our test should not detect (like T.asiatica and T.saginata)
>> Click on the picture to discover our targets on an interactive map! Despite this first success we continued our efforts describe the transcriptome of eggs from T. solium for the first time, since RNA expression in eggs may differ from the whole organism. We reached this goal by building transcontinental research collaborations to India.
>> Learn how we became to our knowledge the first group to describe the transcriptome of T. solium eggs
Team 0: In-silico RNA sensor design
1. Analysing existing toehold sensors to extract basic switch design
Using recent publications on toehold switches [1, 2], a text editor (Sublime Text Editor) and the “Analysis” function of the NUPACK software, we identified the most promising basic structural units:
a 36-nt long recognition site of which the first 25 nucleotides are unpaired (toehold domain)
a 11-nt long bottom stem complementary to the toehold domain
a 5-nt long top stem complementary to the toehold domain
12-nt long hairpin loop containing the RBS (ribosomal binding site) and connecting the bottom & top stem
3-nt long bulge containing the start-codon for the translation of the reporter protein (located between the top and bottom stem)
22-nt long linking sequence (consisting of a variable & an invariable domain) attaching the reporter protein sequence (LacZ) to the bottom stem thus providing an open reading frame in combination with the 11-nt long bottom stem.
2. Optimizing the switch design
We used the “Multitube-Design” function of the NUPACK software and optimized the secondary structure towards the desired “toehold structure”. In this optimization, we included not only the sequence of the core sensor/toehold switch but also the first 50 nucleotides of the reporter protein mRNA sequence.
3. Automating the switch design
To compute an optimal toehold-sensor for each 36-nt long target RNA sequence, we wrote a Python-Script. In addition to the NUPACK “Multitube-Design” function, the script incorporated a function that checks for in-frame stop codons in order to avoid premature translation arrest.
4. Selecting promising switches for screening
The python script generates almost N possible sensors for a target sequence of length N, a number far too large be entirely synthesized and screened.  Thus, a selection step prior to in vitro testing of the sensors was introduced: We selected candidates by their normalized ensemble defect (NED). The NED is a parameter used by the NUPACK software and describes the difference between the most probable secondary structure of each sensor candidate and the desired secondary structure (classic toehold structure).
5. Optimizing switch selection
We improved selection step by adding further criteria, adapted from the original publication [1]: we considered not only the normalized ensemble defect (NED) but also the “single-strandedness” of both the toehold region and the target RNA. “Single-strandedness” describes the probabilities of finding unpaired nucleotides in a sequence of a given length. We established a scoring system rating each sensor candidate in terms of
The local single-strandedness of the target mRNA at the binding site.
The local single-strandedness of the toehold of the sensor
The normalized ensemble defect (NED) of the sensor
The score was calculated as follows:
Score = 5 * (1) + 4 * (2) + 3 * (3)
The sensors with the lowest score were then select as the most promising sensors for synthesis and screening.
Team 1: Sensor Synthesis
Initial approach: Extension PCR
In a first step, the core sensor is amplified from a circular DNA template.
In two PCR-based steps, the recognition and hairpin region was attached to a LacZ reporter element, using primers specifically designed for each individual sensor. (P1 & P2)
In a third PCR step, the sensors were purified and a t7 promoter region was added. (P3)
Optimization of the PCR protocol: Nested PCR
The assembly of the RNA sensors via extension PCR usually requires two PCR steps in order to amplify the core sensor and furnish it with a LacZ reporter element.
The approach of nested PCR can reduce this to one PCR cycle, reducing time and the amount of non-specific PCR products. It involves two sets of primers, used in two successive runs of PCR.
Evaluation of results
In order to evaluate the success of DNA plasmid production or PCRs, colony PCR, restriction digests and gel electrophoresis methods were standardized and could be used to judge the fidelity of sensor synthesis.
Team 2: Sensor Screening/Cell free expression system
The aim of this team was to establish a protocol for cell free expression (CFE) systems and then use this system to screen sensitivity and specificity of sensor candidates.
1. Establishing the CFE system:
Using the published Zika Virus Sensor [1], we established a protocol to screen toehold sensors with a cell free expression system. Before adding the sensor to the expression system, it is mixed with chlorphenol-red – a yellow dye that in the presence of the enzyme beta-galactosidase changes its color to violet. During contact with a specific trigger RNA, the toehold sensor RNA unfolds and translates the sequence for beta-galactosidase into protein, leading to a color change from yellow to violet. Thus, we can analyze the efficiency of our sensors by the amount of color change in a defined time.
2. Screening of Tso_31 Sensors :
We started our quest for a sensor specific for Taenia solium with sensors targeting the published T. solium antigen Tso_31 [3]. During these screenings, we realized that experimental conditions were not yet ideal.
3. Optimizing CFE system using Zika sensor:
We went back to an established toehold sensor (the aforementioned) Zika Virus sensor in order to improve experimental conditions. We tested several concentrations, different buffers as well as different temperatures and concluded that the ideal sensor concentration lies between 0.625 nM and 6.24 nM. In all further screenings, we used 3 nM – a concentration adequate for a novel, non-optimized sensor candidate.
4. Screening Tsm Sensors:
We now had all parameters in place for a high-throughput screening of novel sensors. We screened more than 150 sensor candidates targeting TSM_000297600 and found two sensors with promising properties.
5. Efforts to immobilize the CFE on membranes
In order to use our testing device in the field, the CFE system needs to be transportable, storable and highly stable. We aimed to achieve this by freeze-drying the CFE system onto a cellulose membrane. The membrane-CFE complex would then be storable and could easily be resuspended with liquid whenever needed. By several optimization steps, we obtained a protocol which reduces non-specific interactions between CFE and membrane. We succeeded in freeze-drying and reactivating the complex.
Summary
In order to develop a diagnostic tool for the tapeworm infection Taeniasis, we developed a pipeline that allowed for designing, synthesizing and testing any given sensor within 2 days.
Step 1: In silico design
Design of a toehold switch sensor specific to T. solium RNA
Generating primers necessary for switch synthesis
Step 2: DNA-Switch Synthesis
By using extension PCR or nested PCR, the toehold switches were synthesized
Reporter gene (Beta-galactosidase) and T7 promoter were added to the sensor
Step 3: Sensor Screening
Transcription of DNA sensor into RNA takes place in CFE system
Specific binding of RNA sensor to trigger RNA leads to translation of reporter gene in CFE system
By measuring absorption at 560 nm, color change was measured and enzymatic reaction quantified
Splitting the work into three teams enabled us to work continuously – trouble shooting in one team did not stall work in the other teams. Alongside the work on the core piece – a CFE system containing a sensitive and specific sensor – we tried to keep the ultimate goal in mind: a test usable in the field.
We thus investigated fixing the system onto cellulose membranes, making it storable, durable and easily transportable. Collaborations with partners in Berlin, Europe and India enabled us to further adapt our system to challenges of real-life application.
References
[1] Pardee, K. et al. Rapid, Low-Cost Detection of Zika Virus Using Programmable Biomolecular Components. Cell 165, 1255–1266 (2016).
[2] Green, A. A., Silver, P. A., Collins, J. J. & Yin, P. Toehold Switches: De-Novo-Designed Regulators of Gene Expression. Cell 159, 925–939 (2014)
[3] Gomez, S. et al. Genome analysis of Excretory/Secretory proteins in Taenia solium reveals their Abundance of Antigenic Regions (AAR). Sci. Rep. 5, 9683 (2015)
