<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_ZJU-China_project_htm"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:ZJU-China/project.htm">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:ZJU-China/project.htm&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:ZJU-China/project.htm&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:ZJU-China/project.htm&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:ZJU-China/project.htm" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:ZJU-China/project.htm</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><P><TITLE>PROJECT</TITLE></P><DIV class="wrapper"><DIV class="container"><DIV class="content"><DIV id="columnRight"><DIV id="main"><H2 class="acc_trigger">01 <STRONG>ABSTRACT</STRONG></H2><DIV class="acc_container" style="display: none; "><P align="justify">In cells, engineered multi-enzyme pathways are common and are often physically and spatially organized, thus leading to the high output efficiency. But engineered synthetic pathways utilizing non-homologous enzymes often suffer from low efficiency of production caused by relative lack of spatial organization. <B class="orange">RNA scaffold is designed to co-localize enzymes through interactions between binding domains on the scaffold and target peptides fused to each enzyme in engineered biological pathways <I>in vivo</I></B>. The scaffold allows <B class="orange">efficient channeling of substrates to products</B> over several enzymatic steps by <B class="orange">limiting the diffusion of intermediates</B> thus providing a bright future for solving the problem.</P><P align="justify"> </P><P align="justify">ZJU-China aims to design and realize tunable RNA scaffolds to accelerate biological pathways and control them on and off. In order to achieve the object, we added an aptamer structure on RNA scaffold as a switch to regulate biological pathways by micromolecular ligands. Then we can <B class="orange">control the all-or-none binding relationship</B> between the enzymes and the scaffold by the absence and the presence of a special ligand. </P><P align="justify"> </P><P align="justify">We demonstrated RNA scaffold do make the split GFPs get closer and fluoresce. As was expected, the riboscaffold with a theophylline aptamer can be regulated by theophylline in the range of 0-0.5mM IPTG. A scaffold library was also desired. By changing the sequence of MS2 aptamer binding site, we made the fluorescent decreased. The mutations with different arm length decrease the fluorescent intensity of split GPF by extending the distance between two split GFP parts FA and FB. It provides a series of half-on scaffolds. </P><P align="justify"> </P><P align="justify">Besides, we aimed to find some application for RNA scaffold to make the production of the multi-enzyme pathways more efficient. We have been working on the pathway of the production of IAA from tryptophan and the result will be gained soon later. </P><P align="justify"> </P></DIV><H2 class="acc_trigger">02 <STRONG>BACKGROUND</STRONG></H2><DIV class="acc_container" style="display: none; "><P align="justify">In cells, engineered multi-enzyme pathways are common and are often physically and spatially organized, thus leading to the high output efficiency. But engineered synthetic pathways utilizing non-homologous enzymes often suffer from low efficienty of production caused by relative lack of spatial organization. Thus important issue lies in the method to <B class="orange">increase the efficiency of the multi-enzyme pathways</B>. </P><P align="justify"> </P><P align="justify">Protein scaffolds can be designed to make enzymes closed through interactions between binding domains on the scaffold and target peptides fused to each enzyme. However, protein scaffold is usually large, has limit binding sites, and is hard to be engineered in architecture. DNA can be designed to self-assemble in vitro into many and varied nanostructures. However, DNA scaffold is hard to be controlled and might cause some potential problems <I>in vivo</I>. By contrast, RNA scaffold shows great advantages. For instance, <B class="orange">RNA is more flexible</B>, whose structures are varied, thus leading to their ease to splice. RNA scaffold is able <B class="orange">to be controlled</B> and has a satisfactory regulating efficiency. RNA scaffold <B class="orange">works fast</B>, because it doesn’t need translation like protein scaffold. Camille J. Delebecque and his colleagues have designed and assembled RNA structures and used them to speed up the reaction of hydrogen production. And that is what our project based on.</P><P align="justify"> </P><P align="justify"> </P><P class="fig" align="justify"><B>Fig.1</B>  The function of binding enzymes together of RNA scaffold illustrated by comic. The yellow girl is called “Syn”, the blue boy “Bio”. They represent non-homologous enzymes utilized in engineered synthetic pathways. Usually, they are far away from each other in E.coli, due to lack of spatial organization. But when RNA scaffold designed comes into E.coli, enzymes can be co-localized through interaction between binding domains on scaffold and target peptides fused each enzymes. That is to say, Syn and Bio can live together!</P><P align="justify"> </P></DIV><H2 class="acc_trigger">03 <STRONG>S0: BASIC RNA SCAFFOLD</STRONG></H2><DIV class="acc_container" style="display: none; "><DIV style="height:800px;overflow:scroll;"><H2>Backround</H2><P align="justify">Camille J. Delebecque and his colleagues have designed and assembled RNA structures and used them as scaffolds for the spatial organization of bacterial metabolism (Fig.1). Scaffold D0 consists of PP7 and MS2 aptamer domains that bind PP7 and MS2 fusion proteins. As told above, our project is based on the existing scaffold D0. In order to make sure that we can do further work on it, we planned to repeat the work about scaffold D0. </P><P align="justify"> </P><H2>Design</H2><P align="justify"> </P><DIV class="floatC"><P> </P><P class="fig" align="justify"><B>Fig.1</B>  How RNA scaffold works. FA and FB represent two halves of EGFP. FA and MS2 are connected with a linker of 30bp. FB and PP7 did the same. The purple scaffold is scaffold D0. MS2 and PP7 can specifically bind to two stem-loops on scaffold, thus FA and FB get closer and fluoresce under excitation of 480nm.</P><P> </P></DIV><H2>Materials and Methods</H2><P> </P><H3>1. Plasmids and Strains</H3><P align="justify">pCJDFA and pCJDFB respectively comprising the gene of half split EGFP (fragment A and fragment B) and MS2 or PP7 protein were constructed by overlap extension PCR. (See the Overlap PCR protocal) Genes MS2, PP7 and pCJDD0 are provided by Dr. Camille J. Delebecque. pEGFP is provided by Prof. Jianzhong Shao. </P><P> </P><P align="justify">Information of pCJDFA, pCJDFB and pCJDD0 are as follows:</P><H5>1). pCJDFA: FA-MS2 cloned into T7 duet expression vectors pACYCDuet-1  Spr</H5><P> </P><H5>2) pCJDFB (FB-PP7 cloned into T7 duet expression vector pCOLADuet-1)  Kanr</H5><P> </P><H5>3) pCJDD0 (Scaffold D0 cloned into T7 duet expression vector PETDuet)  Ampr</H5><P> </P><H5>4) BL21-star(DE3)</H5><P align="justify">BL21-star(DE3) cells were used to co-express plasmids. The most important feature of BL21-star(DE3) is that it carries a mutated rne gene (rne131)  which encodes a truncated RNase E enzyme that lacks the ability to degrade mRNA, resulting in an increase in mRNA stability.</P><P> </P><H3>2. Transformation and induction</H3><P align="justify">Three groups of transformation were conducted. The first is BL21-star(DE3) transformed only with pCJDD0, the second with pCJDFA+pCJDFB, and the third with pCJDFA+pCJDFB+pCJDD0. </P><P> </P><P align="justify">Pick the single colony to cultivate in 3mL liquid LB with relative resistances. And when OD reached 0.4, induce with 0.2mM IPTG for 2h at 25 degree.</P><P> </P><P align="justify">Wash the bacteria twice with equivalent PBS. Then test the Fluorescence intensity (FI) and OD with Biotek Synergy Hybrid Reader.
</P><P> </P><P align="justify">Data was shown in Fig.3.  The fluorescence of different expression systems are pictured by Olympus fluoview fv1000 confocal laser scanning microscope (Fig.2)</P><P align="justify">They were transformed with the pCJDD0 (plasmid with scaffold D0) into BL21-star-(DE3). </P><P> </P><H2>Results</H2><P> </P><P>Contrasted to the fluorescence intensity (FI) of the E.coli which only express FA-MS2 and FB-PP7 fusion proteins, the fluorescence intensity of the E.coli with scaffold D0 was obviously increased. Thus, it was possible for us to carry out our development and reformation of RNA scaffold.</P><P> </P><P class="fig"><B>Fig.2</B> FI of Split GFPs without or with RNA scaffold. A.  BL21*(DE3) transformed with pCJDFA and pCJDFB.  B. BL21*(DE3) transformed with pCJDFA, pCJDFB and pCJDD0. The contrast of FI obviously shown that RNA scaffold D0 could bind split GFPs together, so that split GFPs could fluoresce. (Pictures were obtained with Olympus fluoview fv1000 confocal laser scanning microscope, using a 60X objective.)</P><P> </P><P class="fig"><B>Fig.3</B>   FI/OD of different transformation groups.  There exist significant differences among three groups. And as expected, split GFPs with scaffold D0 together can fluoresce stronger than those without scaffold. </P><H3>Reference:</H3><P class="ref">1. Thodey, K. &amp; Smolke, C.D. Bringing It Together with RNA. Science 333, 412-413 (2011).
2. Delebecque, C.J., Lindner, A.B., Silver, P.A. &amp; Aldaye, F.A. Organization of Intracellular Reactions with Rationally Designed RNA Assemblies. Science 333, 470-474 (2011).</P><P> </P></DIV></DIV><H2 class="acc_trigger">04 <STRONG>S1: ALLOSCAFFOLD</STRONG></H2><DIV class="acc_container" style="display: none; "><DIV class="projectNav"><TABLE class="tm"><TBODY><TR><TD class="tm" width="450px"><DIV class="projectNavFloat"><A target="brainFrame" href="https://2012.igem.org/Team:ZJU-China/project_s1_1.htm" style="text-decoration:none">1. Summary</A><A target="brainFrame" href="https://2012.igem.org/Team:ZJU-China/project_s1_2.htm" style="text-decoration:none">2. Design</A><A target="brainFrame" href="https://2012.igem.org/Team:ZJU-China/project_s1_4.htm" style="text-decoration:none">3. Preparation: Characterize previous parts</A></DIV></TD><TD class="tm" width="250px"><DIV class="projectNavFloat"><A target="brainFrame" href="https://2012.igem.org/Team:ZJU-China/project_s1_3.htm" style="text-decoration:none">4. Characterization</A><A target="brainFrame" href="https://2012.igem.org/Team:ZJU-China/project_s1_5.htm" style="text-decoration:none">5. Results</A></DIV></TD></TR></TBODY></TABLE></DIV></DIV><H2 class="acc_trigger">05 <STRONG>S2: SCAFFOLD LIBRARY</STRONG></H2><DIV class="acc_container" style="display: none; "><DIV style="height:800px;overflow:scroll;"><P>Several mutations of RNA scaffold D0 have been designed and made. They show quite different characterizes and functions. With the experiment, more RNA scaffold mutations are characterized. Concept Library of RNA Scaffold is suggested.</P><P> </P><P>What is the Library of RNA Scaffold for? Evolution! The variable of RNA structures accommodates a wide application prospect. Though the point mutation reduced uncertainty of selection and the blindness, trying to find a suitable construction is vast project. Various experimental methods, selection and modeling should be used in this part. By analyzing existing mutations, derivation can be made to construct and find an enhanced RNA scaffold. We called this process evolution. </P><P> </P><P>The Library may contain changes of self, self-assemble, RNA-RNA interaction, RNA-protein interaction. Some examples are shown below.</P><P> </P><H2>Categories of mutants</H2><H3>1. Mutating arm length</H3><P>changing the arm length of RNA scaffold D0. As the mechanism of D0 is reducing the distance of two key enzyme of the pathway, in other words, the output and reaction efficiency is depend on the local concentration. The two aptamer binding site in our project is on two hairpin arms witch are designed in the same length. The change of the arm length provides feasibility of distance-efficiency research. We used split GFP experiments. We made some mutations with different arm length, the result of D0M4 and D0M 5 split GFP experiment shows the light decreasing lend by split GFP FA-FB distance. The difference (PD0M4=0.079, PD0M5=0.025) suggests that the mutating arm length scaffold doesn’t provide an on/off switch but a definability one. It characterized the D0 in another way.</P><DIV class="floatC"><P class="fig"><B>fig 1a.</B> D0 is the original scaffold. D0 a-d were mutated to the scaffold with different aptamer arm length. </P></DIV><DIV class="floatC"><P class="fig"><B>fig 1b.</B> The result of arm length mutating. Both D0M4 and D0M5 scaffold half-on GEP.</P></DIV><P> </P><H3>2. Mutating aptamer binding site</H3><P>Mutating the PP7 and MS2 binding sites prevented protein scaffolding. Preventing protein scaffolding lead to the key enzyme dissociation and the decrease of enzyme local concentration. By chancing the sequence of MS2 aptamer binding site, the fluorescent light decreased. D0M3 in our project is the molecular with mutated aptamer binding site. Split GFP experiment shows that there is a significant difference between D0 an D0M3(P≦0.05, fig2.c). Camille J. Delebecque has done the same work for the H2 biosynthesis pathway.</P><P class="fig"><B>Fig2a.</B> MS2 and PP7 bind to the scaffold and make GFP work. <B>Fig2b.</B> By mutating aptamer binding site, scaffolding is stop. </P><P> </P><DIV class="floatC"><P class="fig"><B>Fig2c.</B> significant difference between D0 an D0M3</P></DIV><P> </P><H3>3. Assemblage</H3><P>adding extra sequence for self-, RNA-, protein-assemblage. The added sequence may be a riboswitch, RNA or protein binding site, self-assemble structure. Regulation molecular search is also wanted synchronously. </P><P> </P><H2>Applications and outlook</H2><P> </P><H3>1. sRNA regulation</H3><P>Simple an direct RNA-RNA interaction change the object RNA  scaffold structure. As a Foundation regulation, it substantially enhances the possibilities of forthcoming experiment. </P><P class="fig"><B>Fig3.</B> The designed scaffold has a interaction to regulatory sRNA. Same mechanism, regulatory molecule can be changed to mRNA a. Turn off the scaffold by the competitive binding with aptamer binding site (green) b. The RNA scaffold has a secondary structural switch controls accessibility of sRNA-binding sites(blue) witch can change the arm length. Output regulated by arm length change. c. both methods were used. d. bind and release the object molecular.)</P><P> </P><H3>2. Protein expression (mRNA) regulation</H3><P>RNA scaffold as a free molecular in cell can specific bind mRNA and protein. Binding molecular changes the structure of scaffold to release or combine something. So that oncogene and virogene can be found and controlled by the drug from RNA scaffold. The problem of cancer therapeutic drug side effecting may solved by it. </P><P> </P><H3>3. Self quenching（Self regulation）</H3><P>Adding self binding site, a balance of “on” and “off” scaffolds is built. The relationship between the binding site size, CG bases, binding form and the rate binding molecular is urgently modeled. Forming dimerization and trimerization, the concentration of working scaffold could be regulated.</P><P> </P><H3>4. Polo-scaffold</H3><P>Scaffold with intermolecular binding component. These scaffolds bind each other or bind through mediate molecular. And this binding mode has been proved both in vitro and vivo. The aggregation of molecular also makes artificial organelle achievable. </P><P class="fig"><B>Fig4a.(Upper)</B> Dimerization and trimerization. Protein binding site is sealed off by the scaffolds themselves. Too much scaffold molecular lend to the self regulation.<B>Fig4b.(Lower left)</B> Dimerization and trimerization. Protein binding site is sealed off by the scaffolds themselves. Too much scaffold molecular lend to the self regulation.<B>Fig4c.(Lower right)</B> Polo-scaffold be made by head-tail binding.</P><P> </P><P>Several RNA scaffold mutations are constructed and characterize, but they are the tip of the iceberg. There is still plenty to do in this part. The charms of library are the selection and combination. It introduces a new concept of biobrick combination mode.</P></DIV></DIV><H2 class="acc_trigger">06 <STRONG>S3: BIOSYNTHESIS OF IAA</STRONG></H2><DIV class="acc_container" style="display: none; "><DIV style="height:800px;overflow:scroll;"><H2>Background</H2><P>Indole 3-acetic acid (IAA) is a plant growth hormone that serves as a potent and important auxin to many plants. Although auxin is a key factor for plant growth, it can be a metabolic burden to plants at high concentrations and prohibit plant growth, which contrasts its initial goal. </P><P>Imperial College London 2011 iGME team claimed that molecules of IAA are chemically labile in aqueous solution, so they produce IAA in soil to promote rather than stunt plant root growth. They also regulated output of IAA through change However, root sensitivity to IAA differs from variety to variety. There is no guarantee that a changeless IAA production will fit in all situation. Our designed RNA alloscaffold makes it possible to change the output of IAA in a range to adapt the need of plant root growth.</P><P>There are two enzymes responsible for IAA biosynthesis in <I>E.coli</I>, IaaM and IaaH. IaaM catalyzes tryptophan to Indole-3-acetamide while IaaH catalyzes it to Indole-3-acetic acid.</P><H2>Experiment Design</H2><P>With the help of riboscaffold, we can regulate IAA biosynthesis in the following two aspects, increasing yields and regulating reaction speed.</P><P>IaaM and IaaH were fused to a dimer of MS2 and a single copy of PP7 protein respectively (in plasmids pZCM and pZCH).</P><DIV class="floatC"><P class="fig"><B>Fig 1.</B> pZCM (the upper one) and pZCH (the lower one)</P></DIV><H3>1. Increasing yields</H3><P>Basic scaffold D0 has been proved to be effective in bringing two proteins closer by split-GFP assay. We further proved that D0 can be used in accelerating reaction speed.</P><P>Basic scaffold D0 (in plasmids pCJDD0) and pZCM, pZCH were transform into <I>E.coli</I> strain BL21*(DE3) for co-expression.</P><DIV class="floatC"><P class="fig"><B>Fig 2.</B> Two enzymes related to the reaction are fused to basic scaffold D0 to get spatial organized</P></DIV><H3>2. Regulating reaction speed</H3><P>Alloscaffolds (clover 2 and clover 3) have been proved to be effective in regulating distance between two proteins by split-GFP assay. We further proved that they can be used in regulating reaction speed.</P><P>The interaction inhibits the binding function of MS2 aptamer in the absence of theophylline. IaaM-2X-MS2 cannot bind on RNA scaffold thus the speed of reaction is normal.</P><DIV class="floatC"><P class="fig"><B>Fig 3.</B> Illustration of alloscaffolds in biosynthesis pathway.</P></DIV><P>When theophylline is added, the fold of the loop is changed and thus the interaction will disappear, leading to the binding of MS2 aptamer and corresponding protein. IaaM and IaaH will get closer to accelerate reaction speed.</P><P>When theophylline is added, the fold of the loop is changed and thus the interaction will disappear, leading to the binding of MS2 aptamer and corresponding protein. IaaM and IaaH will get closer to accelerate reaction speed.</P><P>Alloscaffolds (in plasmids pZCCOV2 and pZCCOV3) and pZCM, pZCH were transformed into E.coli strain BL21*(DE3) for coexpression.</P><H2>Results</H2><H3>1. Standard curve</H3><P>We plan to determined the concentration of IAA with salkowski assay. The standard curve has been made with IAA in LB.</P><DIV class="floatC"><P class="fig"><B>Fig 4.</B> Standard curve for testing IAA concentration.</P></DIV><P>Our experiment is on going, click <A class="parts" href="http://bis.zju.edu.cn/igem2012/project-biosyn-result.htm" target="_blank">here</A> for latest results.</P></DIV></DIV><H2 class="acc_trigger">07 <STRONG>S4: POLYSCAFFOLD</STRONG></H2><DIV class="acc_container" style="display: none; "><DIV class="projectNav"><TABLE class="tm"><TBODY><TR><TD class="tm" width="300px"><DIV class="projectNavFloat"><A target="s4Frame" href="https://2012.igem.org/Team:ZJU-China/project_s4_1.htm" style="text-decoration:none">1. Summary</A><A target="s4Frame" href="https://2012.igem.org/Team:ZJU-China/project_s4_2.htm" style="text-decoration:none">2. Design</A></DIV></TD><TD class="tm" width="350px"><DIV class="projectNavFloat"><A target="s4Frame" href="http://bis.zju.edu.cn/igem2012/project-s4-3.htm" style="text-decoration:none">3. Results</A><A target="s4Frame" href="https://2012.igem.org/Team:ZJU-China/project_s4_4.htm" style="text-decoration:none">4. Future Work</A></DIV></TD></TR></TBODY></TABLE></DIV></DIV><H2 class="acc_trigger">08 <STRONG>PARTS</STRONG></H2><DIV class="acc_container" style="display: none; "><DIV style="height:800px;overflow:scroll;"><H2>Summary</H2><P>This is a summary of the parts that we have submitted to the <A href="http://partsregistry.org/Main_Page">Registry of Standard Biological Parts</A>. These parts include: </P><P><B>ncRNA scaffold generator: </B><A class="parts" href="http://partsregistry.org/Part:BBa_K738000">BBa_K738000</A>, <A class="parts" href="http://partsregistry.org/Part:BBa_K738002">BBa_K738002</A></P><P><B>protein coding domains: </B><A class="parts" href="http://partsregistry.org/Part:BBa_K738004">BBa_K738004</A> , <A class="parts" href="http://partsregistry.org/Part:BBa_K738005">BBa_K738005</A> , <A class="parts" href="http://partsregistry.org/Part:BBa_K738006">BBa_K738006</A> , <A class="parts" href="http://partsregistry.org/Part:BBa_K738007">BBa_K738007</A></P><P>These parts have all been well characterized. Please visit the Registry of Standard Biological Parts for more information.</P><H2>List</H2><TABLE><TBODY><TR><TD>?</TD><TD>?</TD><TD>Name</TD><TD>Type</TD><TD>Description</TD><TD>Designer</TD><TD>Length</TD></TR><TR><TD> </TD><TD>W</TD><TD><A class="parts" href="http://partsregistry.org/Part:BBa_K738000">BBa_K738000</A></TD><TD>Generator</TD><TD>RNA Scaffold generator</TD><TD>Huachun Liu</TD><TD>171</TD></TR><TR><TD>W</TD><TD><A class="parts" href="http://partsregistry.org/Part:BBa_K738002">BBa_K738002</A></TD><TD>Generator</TD><TD>Theophyline riboswitch regulated RNA Scoffold(clover version 2)</TD><TD>Huachun Liu</TD><TD>209</TD></TR><TR><TD> </TD><TD>W</TD><TD><A class="parts" href="http://partsregistry.org/Part:BBa_K738004">BBa_K738004</A></TD><TD>Generator</TD><TD>FA-2X-MS2;Split GFP N-terminal domain fused with MS2 protein</TD><TD>Huachun Liu</TD><TD>1284</TD></TR><TR><TD> </TD><TD>W</TD><TD><A class="parts" href="http://partsregistry.org/Part:BBa_K738005">BBa_K738005</A></TD><TD>Coding</TD><TD>FB-2X-PP7;Split GFP C-terminal domain fused with PP7 protein</TD><TD>Huachun Liu</TD><TD>654</TD></TR><TR><TD> </TD><TD> </TD><TD><A class="parts" href="http://partsregistry.org/Part:BBa_K738006">BBa_K738006</A></TD><TD>Coding</TD><TD>FA: Split GFP N-terminal domain</TD><TD>Huachun Liu</TD><TD>480</TD></TR><TR><TD> </TD><TD> </TD><TD><A class="parts" href="http://partsregistry.org/Part:BBa_K738007">BBa_K738007</A></TD><TD>Coding</TD><TD>FB, Split GFP C-terminal domain</TD><TD>Huachun Liu</TD><TD>255</TD></TR></TBODY></TABLE><H2>Future work</H2><H3>Theophylline responded RNA riboscaffold</H3><P>We have designed two RNA riboscaffold responded to theophylline. Unfortunately, we only managed to submit one of them (BBa_K738002) to the Registry of Biological Parts in time (that means before September 26). </P><P>We have started the work of constructing a second theophylline responded RNA riboscaffold clover vision 3(but not finished). Clover vision 3 is different with BBa_K738002 in 3D structure, which may lead to results that are far away from that of BBa_K738002.</P><P>Please visit <A href="http://partsregistry.org/Part:BBa_K738002">here</A> for more information.</P><H3>Library</H3><P>We plan to develop a RNA scaffold library that offers more tunable responses. We’ve got several members in this library by now and our ultimate goal is acquiring a series of members which span a large acceleration rate range from about 10% to 90%. Thus, researchers may be able to choose a member in the library to acquire the targeted acceleration rate easily.</P><H3>Pathway of producing IAA</H3><P>Accelerating production of IAA with RNA scaffold has been proved to be efficient. Two enzymes, IaaH (BBa_K515000) and IaaM (BBa_K515001), are related to the process. We’ve fused IaaH and IaaM with MS2 and PP7 respectively to get IaaM-2X-MS2 and IaaH-2X-PP7, which are able to bind on RNA scaffold. We plan to make and submit this two protein as parts later. Thus, we’d like to regulate the biosynthesis process efficiency with RNA riboscaffold. We plan to submit BBa_K738014 and BBa_738015 later.</P></DIV></DIV><H2 class="acc_trigger">09 <STRONG>PERSPECTIVES</STRONG></H2><DIV class="acc_container" style="display: none; "><DIV style="height:800px;overflow:scroll;"><H2>1. Angel Riboscaffold</H2><P>This is an extension application of our designed clover series of riboscaffold in drug diliver therapy. Some diseases, such as Cancer, will release some small molecular or change microenvironments beside it thus produce detectable signals. Different from using a biosensor to detect these signals, we utilize our scaffold’s aptamer, accompanying with the production of medicine target the disease. If we change Theophylline aptamer into nidus(disease) molecular aptamer, when riboscaffold bind nidus molecular and change conformation, MS2 aptamer &amp; PP7 aptamer are going to set closer. Enzymes which combining MS2 aptamer &amp; PP7 aptamer and producing drugs are ready to catalyze thus bring out targeting agents. It turns out to be a one-stop agency, once detect the focus of diease, will generate corresponding drug targeting the diease. </P><DIV class="floatC"><P class="fig"><B>Fig1.</B> Riboscaffold which can detect and treat diseases. </P></DIV><H2>2. Shining Riboscaffold</H2><DIV><DIV class="floatR" width="200px"><P class="fig"><B>Fi2.</B> Aptamers that can shine upon binding. </P></DIV><DIV width="500px"><P>Paige[1] has reported some RNA aptamers that can bind fluorophores, which are small compounds, and in this way resemble the chemical bonds in GFP, then give out fluorescence. </P><P>If we use these aptamers in replace of the theophylline aptamer on our riboscaffold, we can make the riboscaffold shining upon the binding of signal compounds mentioned above. This is a cool method to visualize the states, dynamics and localization of riboscaffold in the living cell. </P></DIV></DIV><H2>3. LEGO Riboscaffold</H2><P>Riboscaffold has unbelievable ability to extend itself through base pairing with each other, just like LEGO bricks! The assembly of LEGO riboscaffolds can load more enzymes and to a large degree accelerate the reaction or artificially construct a longer pathway with high efficiency. For example, artificial TCA cycle abd artificial EMP are promising results. The following pictures show our wide imagination of the possible structure of LEGO riboscaffolds. </P><P>But how to obtain these LEGO riboscaffolds? Wachtveitlb[2] has reported a fantastic method to detect RNA-RNA interaction by introducing fluorophores like 1-ethynylpyrene into the 2-position of RNA adenosine. When two single-stranded RNAs with this fluorophore base pair with each other, the fluorescence spectrum changes and thus suggesting their interaction. So it is hopeful to find the desired riboscaffolds as LEGO bricks by selecting from the library! </P><P class="fig"><B>Fig3.1.(Upper Left)</B> LEGO bricks.<B>Fig3.2.(Upper right)</B> Long scaffold that has multiple binding sites.<B>Fig3.3.(Lower Left)</B> A possible device built by LEGO riboscaffold.<B>Fig3.4.(Lower Right)</B> Sheets and tubes constructed by LEGO riboscaffolds <I>in vivo</I>. [6]</P><H2>4. Activator scaffold</H2><P>In eukaryote, there are naturally produced long non-coding RNAs that attract more and more attention these days and display intriguing potential to act as scaffolds [3]. And our riboscaffold can mimic them and bring their functions to prokaryote. One of the functions is combining related transcription factors and bring them to promoter, as a result enhance the expression of target gene. That is because ncRNA can binds both DNA and Proteins, and can travel freely between nucleus and cytoplasm, which displays great advantage as a bridge. </P><P>Aptamers can be selected in vitro against nearly any target of choice. There are RNA aptamers that can specifically bind some transcriptional regulator. For example, Hunsicker [4] has selected one RNA aptamer that can bind TetR, which usually binds on operator sequence and repress gene expression. So once aptamers mentioned above are designed into a riboscaffold, it can initiate the expression of target genes with higher efficiency. </P><DIV class="floatC"><P class="fig"><B>Fig3.</B> Riboscaffold that can bring transcription factors to promoters. </P></DIV><H2>5. Medicine &amp; Health</H2><P>To date, many groups have successfully identified aptamers with a variety of functions, including inhibitory and decoy-like aptamers, regulatable aptamers, multivalent/agonistic aptamers, and aptamers that act as delivery vehicles [5]. </P><P>By designing these different aptamers into our RNA scaffold, we can endow our scaffold various potential applications in therapeutics and/or diagnostics. </P><P>For instance, designing inhibitory aptamers that targets VEGF into our RNA scaffold can be used to treat the wet age-related macular degeneration and that has been approved by the FDA in December 2004. </P><P>Designing Decoy-like aptamers that can mimic the target sequce of the proteins into our RNA scaffold can be used as decoys to inhibit binding of transcriptional factors such as HIV-tat, NF-κB, and E2F to their cognate sequences on DNA and thus prevent transcription of target genes and may result in powerful therapeutics for treating many human pathologies. </P><P>Designing aptamers behavior as delivery tools into our RNA scaffold can be used to deliver not only some siRNAs to target cells but also toxins, radioisotopes, and chemotherapeutic agents encapsulated in nanoparticles. </P><H2>References:  </H2><P class="ref"> [1] Jeremy S. Paige, Karen Y. Wu, Samie R. Jaffrey, RNA Mimics of Green Fluorescent Protein science, 2011 vol 333, 642-646. 
[2] Josef Wachtveitlb, Joachim W. Engels, ect. RNA as scaffold for pyrene excited complexes, Bioorganic &amp; Medicinal Chemistry 16 (2008) 19-26. 
[3] Mitchell Guttman, John L. Rinn. Modular regulatory principles of large non-coding RNAs. Nature. 2012 Feb 15;482(7385):339-46. 
[4] Anke Hunsicker, Markus Steber, ect. An RNA Aptamer that Induces Transcription, Chemistry &amp; Biology, 2009,Volume 16, Issue 2, 173–180. 
[5] Kristina W. Thiel and Paloma H. Giangrande, Therapeutic Applications of DNA and RNA Aptamers. Oligonucleotides, 2009, Volume 19, Number 3, 209-222. 
[6] Thodey, K. &amp; Smolke, C.D. Bringing It Together with RNA. Science 333, 412-413 (2011). </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:ZJU-China/project.htm">http://2012.igem.org/Team:ZJU-China/project.htm</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:ZJU-China/project.htm" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:ZJU-China/project.htm" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:ZJU-China/project.htm&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:ZJU-China/project.htm&amp;oldid=297093" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>