<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Tuebingen_FruitDesign skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Tuebingen/FruitDesign</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="grailWrapper"><NAV><UL><LI id="sectionTeam" class="iGemSabber"><A onclick="return true;">Team</A></LI><LI><A href="https://2016.igem.org/Team:Tuebingen">Home</A></LI><LI><A href="https://2016.igem.org/Team:Tuebingen/Team">The Team</A></LI></UL><LI id="sectionFruitforce" class="whiteSabber"><A onclick="return true;">Fruitforce</A></LI><LI><A href="https://2016.igem.org/Team:Tuebingen/FruitSummary">Introduction</A>
                            #</LI><LI><A href="https://2016.igem.org/Team:Tuebingen/L_johnsonii">L. johnsonii</A></LI><LI><A href="https://2016.igem.org/Team:Tuebingen/FruitDesign">Project design</A></LI><LI><A href="https://2016.igem.org/Team:Tuebingen/Model">Model</A></LI><LI><A href="https://2016.igem.org/Team:Tuebingen/Results">Results</A></LI><LI id="sectionSide" class="blueSabber"><A onclick="return true;">Side Projects</A></LI><LI><A href="https://2016.igem.org/Team:Tuebingen/CloneWars">Clone Wars</A></LI><LI><A href="https://2016.igem.org/Team:Tuebingen/DeathStar">Death Star</A></LI><LI><A href="https://2016.igem.org/Team:Tuebingen/Measurement">Interlab Study</A></LI><LI id="sectionLab" class="orangeSabber"><A onclick="return true;">Lab</A></LI><LI><A href="https://2016.igem.org/Team:Tuebingen/Parts">Parts</A></LI><LI><A href="https://2016.igem.org/Team:Tuebingen/Notebook">Notebook</A></LI><LI><A href="https://2016.igem.org/Team:Tuebingen/Safety">Safety</A></LI><LI><A href="https://2016.igem.org/Team:Tuebingen/Methods">Methods</A></LI><LI id="sectionHR" class="redSabber"><A onclick="return true;">HP</A></LI><LI><A href="https://2016.igem.org/Team:Tuebingen/HP/Silver">Summary</A></LI><LI><A href="https://2016.igem.org/Team:Tuebingen/HP/Gold">Internship</A></LI><LI><A href="https://2016.igem.org/Team:Tuebingen/HP/Marburg1">Symposium Marburg</A></LI><LI><A href="https://2016.igem.org/Team:Tuebingen/HP/Marburg2">iGEM Marburg</A></LI><LI><A href="https://2016.igem.org/Team:Tuebingen/HP/Paris">EE16 Paris</A></LI><LI><A href="https://2016.igem.org/Team:Tuebingen/HP/Constance">Science Slam</A></LI><LI><A href="https://2016.igem.org/Team:Tuebingen/HP/SciencePub">Science Pub Tue</A></LI><LI><A href="https://2016.igem.org/Team:Tuebingen/HP/Symposium">Symposium Tue</A></LI><LI id="sectionAttributions" class="purpleSabber"><A onclick="return true;">Attributions</A></LI><LI><A href="https://2016.igem.org/Team:Tuebingen/Collaborations">Collaborations</A></LI><LI><A href="https://2016.igem.org/Team:Tuebingen/Sponsoring">Sponsoring</A></LI><LI><A href="https://2016.igem.org/Team:Tuebingen/Attributions">Acknowledgements</A></LI><LI><A href="https://2016.igem.org/Team:Tuebingen/References">References</A></LI></NAV><DIV class="contentArea"><DIV class="contentBothCols"><H1 id="Project Design">Project Design</H1><P>
            Metabolic pathway engineering was used to modify <EM>Lactobacillus johnsonii</EM> for increased fructose metabolism. Those modifications aim to increase fructose uptake, while reducing glucose consumption, because both carbohydrates display the main energy sources of <EM>L. johnsonii</EM> ATCC33200. To search for targets in both metabolic pathways the  <A href="https://www.patricbrc.org/portal/portal/patric/Genome?cType=genome&amp;cId=525330.3">Patric database</A> was used. <EM>L. johnsonii’s</EM> genome was documented and analyzed in course of the Human Microbiome Project in 2009. Glucose and fructose metabolism in wild type<EM> L. johnsonii</EM> (FIGURE 1) differs from our engineered bacterium in following points:

 </P><FIGURE style="background-color:rgba(223,220,228,0);"><FIGCAPTION>
                Schematic illustration of sugar metabolism in <EM>L. johnsonii</EM>.
            </FIGCAPTION></FIGURE><P>To downregulate the glucose metabolism, a gene knockout via CRISPR/Cas9 system was planned. Carbohydrates are taken up into the cell via transporters of the ABC-superfamily, getting phosphorylated in this process, but for glucose no specific transporter was annotated. Therefore, the two main enzymes for glucose-6-phosphate metabolism were targeted: 
Glucose-6-phosphate-isomerase is part of glycolysis, converting glucose-6-phosphate to fructose-6-phosphate and reverse, as well as glucose-6-phosphate-dehydrogenase which converts glucose-6-phosphate to 6-phospho-D-glucono-1,5-lactone, being the starting point for the pentose phosphate pathway. These two knockouts should result in an accumulation of glucose-6-phosphate which in turn causes decreased glucose uptake by product inhibition of glucose ingesting transporters. On the other hand, fructose metabolism should be enhanced by upregulation of endogenous enzymes: Fructose enters the cell as monosaccharide or, in conjunction with glucose, as sucrose. Both carbohydrates are taken up by substrate specific PTS transporters and get phosphorylated in this process. Therefore, both transporters were identified as possible targets for pathway enhancement. </P><P>
Besides, the phosphofructokinase was chosen for elevated fructose turnover: This enzyme catalyzes the reaction of fructose-6-phosphate to fructose-1,6-bisphosphat preventing fructose-6-phosphate accumulation after increased uptake. Enhancing this enzyme’s activity should result in higher ATP production via glycolysis using mainly fructose and sucrose as substrates.
Enhancement of fructose and sucrose metabolism should be achieved by introducing the three endogenous proteins via the lactobacillus shuttle vector pNZ124Tue. The hypothetic results of the metabolic engineering and its effects are summed up in FIGURE 2.
After we picked out possible knock-out and overexpression targets by educated guessing a systems biology approach was developed to simulate the proliferative behaviour of our modified lactobacillus. To get more information about our modeling approach please click <A href="https://2016.igem.org/Team:Tuebingen/Model">here</A>.
 </P><FIGURE style="background-color:rgba(223,220,228,0);"><FIGCAPTION>
                Theoretical scheme of modified sugar metabolism of <EM>L. johnsonii</EM>.
            </FIGCAPTION></FIGURE><P>
In human, hereditary fructose intolerance is caused by accumulation of fructose-1-phosphate. To simulate this cellular problem in a model organism, a ketohexokinase was codon optimized for expression in Saccharomyces cerevisiae. This experiment is based on a paper by Donaldson et al. from 1993. Yeast cannot metabolize fructose-1-phosphate causing it to be toxic for the cells and inhibiting its growth. For this reason, no yeast enzyme produces fructose-1-phosphate. So the idea was to introduce a ketohexokinase into yeast leading to the production and accumulation of fructose-1-phosphate in fructose containing environment. Hence, if there is any fructose consuming organism the modified yeast cells should grow better. The ketohexokinase designed for this project is based on the ketohexokinase of <EM>Rattus norvegicus</EM>, originally used for the experiments in Donaldson’s publication. 
</P></DIV><DIV class="contentRow"><H1>Knock out via CRISPR/Cas9</H1><P>Our primary goal being an increase in the utilization of fructose in the Lactobacillus’ metabolism, we
planned to disable key enzymes in the metabolic pathway of glucose, thus forcing the bacterium to
rely on other carbohydrates as a source of energy.
To that end, we aimed to use CRISPR (clustered regularly interspaced short palindromic repeats), being the most flexible and affordable tool for genomic
modification, as a means to achieve a knockout. Using a Cas9 endogenous to another strain of L.
Johnsonii, our plan was to replace our target genes with a reporter construct of our own design.</P><P>By taking advantage of the polycistronic nature of transcription in prokaryotes, it is possible to
construct a template for homologous recombination that is only transcribed once successfully copied
into the genome. Choosing homology arms that only include a RBS and the coding sequences of
adjacent genes, we planned to introduce a split intein into the genome. The two halves of this intein,
when coexpressed, rejoin a split GFP-protein. This strategy should enable a quick and inexpensive
selection of clones bearing the desired modifications.</P><P>Mainly due to time constraints, the modification of the Lactobacillus genome was not realized.
In the course of the project, several Cas9-fusion proteins were designed. Due to a lack of opportunity
to test these constructs in Lactobacillus Johnsonii, they will be tested in different model organisms.
</P></DIV><DIV class="contentRow"><H1>Quantitative Fructose test</H1><P>To prove the effectiveness of our transformed lactobacilli, we had to come up with a quantitative fructose assay. Many coulometric assays with the possibility of quantitative measurement are available for sugars, many of which having the problem of distinguishing between different kinds of sugars, especially between Glucose and Fructose due to chemical similarity. After screening multiple tests we decided to use a test published by the WHO for the proof of fertility in human semen using an Indole reaction. The test provides a rather sensitive method to measure the Fructose concentration even in presence of Glucose. 
Following the indole reaction, the quantitative measurement of the test was done photometrically at 470nm with a standard spectrophotometer.
</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>