<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Tuebingen_Model skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Tuebingen/Model</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="grailWrapper"><NAV><UL><LI id="sectionTeam" class="iGemSabber"><A onclick="return true;">Team</A></LI><LI><A href="https://2016.igem.org/Team:Tuebingen">Home</A></LI><LI><A href="https://2016.igem.org/Team:Tuebingen/Team">The Team</A></LI></UL><LI id="sectionFruitforce" class="whiteSabber"><A onclick="return true;">Fruitforce</A></LI><LI><A href="https://2016.igem.org/Team:Tuebingen/FruitSummary">Introduction</A>
                            #</LI><LI><A href="https://2016.igem.org/Team:Tuebingen/L_johnsonii">L. johnsonii</A></LI><LI><A href="https://2016.igem.org/Team:Tuebingen/FruitDesign">Project design</A></LI><LI><A href="https://2016.igem.org/Team:Tuebingen/Model">Model</A></LI><LI><A href="https://2016.igem.org/Team:Tuebingen/Results">Results</A></LI><LI id="sectionSide" class="blueSabber"><A onclick="return true;">Side Projects</A></LI><LI><A href="https://2016.igem.org/Team:Tuebingen/CloneWars">Clone Wars</A></LI><LI><A href="https://2016.igem.org/Team:Tuebingen/DeathStar">Death Star</A></LI><LI><A href="https://2016.igem.org/Team:Tuebingen/Measurement">Interlab Study</A></LI><LI id="sectionLab" class="orangeSabber"><A onclick="return true;">Lab</A></LI><LI><A href="https://2016.igem.org/Team:Tuebingen/Parts">Parts</A></LI><LI><A href="https://2016.igem.org/Team:Tuebingen/Notebook">Notebook</A></LI><LI><A href="https://2016.igem.org/Team:Tuebingen/Safety">Safety</A></LI><LI><A href="https://2016.igem.org/Team:Tuebingen/Methods">Methods</A></LI><LI id="sectionHR" class="redSabber"><A onclick="return true;">HP</A></LI><LI><A href="https://2016.igem.org/Team:Tuebingen/HP/Silver">Summary</A></LI><LI><A href="https://2016.igem.org/Team:Tuebingen/HP/Gold">Internship</A></LI><LI><A href="https://2016.igem.org/Team:Tuebingen/HP/Marburg1">Symposium Marburg</A></LI><LI><A href="https://2016.igem.org/Team:Tuebingen/HP/Marburg2">iGEM Marburg</A></LI><LI><A href="https://2016.igem.org/Team:Tuebingen/HP/Paris">EE16 Paris</A></LI><LI><A href="https://2016.igem.org/Team:Tuebingen/HP/Constance">Science Slam</A></LI><LI><A href="https://2016.igem.org/Team:Tuebingen/HP/SciencePub">Science Pub Tue</A></LI><LI><A href="https://2016.igem.org/Team:Tuebingen/HP/Symposium">Symposium Tue</A></LI><LI id="sectionAttributions" class="purpleSabber"><A onclick="return true;">Attributions</A></LI><LI><A href="https://2016.igem.org/Team:Tuebingen/Collaborations">Collaborations</A></LI><LI><A href="https://2016.igem.org/Team:Tuebingen/Sponsoring">Sponsoring</A></LI><LI><A href="https://2016.igem.org/Team:Tuebingen/Attributions">Acknowledgements</A></LI><LI><A href="https://2016.igem.org/Team:Tuebingen/References">References</A></LI></NAV><DIV class="contentArea"><DIV class="contentBothCols"><H1 id="summary">Summary</H1><P>Our iGEM project intends to reduce the ability of lactobacilli to metabolize glucose. A possible approach is to knock-out genes along glucose-consuming pathways. However, the absence of the knocked-out genes must not jeopardize the bacteria's ability to live and proliferate. Such sets of genes could be identified either by educated guessing or by systems biological analysis.</P><P>Systems biological studies are capable of inferring the response of cells—or more generally: systems—under consideration of all genes and their network of interactions (Hine &amp; Martin, 2015). In our project, we are interested in the rates of metabolism for <EM>fructose</EM> and <EM>glucose</EM>. Because we investigate gene knock-outs, we do not consider the time-course of metabolite concentrations (in contrast to last year's project). Instead, we analyse the flow of metabolites through the metabolic pathways of a system (Orth et al., 2010). Such a flow is computed with a flux balance analysis (FBA).</P><P>FBA assigns to each possible reaction a quantity, called flux value, that specifies with which rate the reactants are converted into the corresponding product in correct stoichiometric proportions. However, these fluxes are subjected to the following constraints:</P><UL><OL style="list-style-type: decimal"><LI>Mass conversion must be respected. Metabolites may only enter/leave a system over explicitly defined in-/outbound fluxes.</LI><LI>A flux has a maximal and minimal rate</LI><LI>Metabolites have a lower and upper bound amount</LI></OL></UL><P>These constraints are mathematically formulated as a set of inequalities. Furthermore, FBA optimizes specific fluxes, which are called objective functions. A possible objective it the maximization of the growth function. Concrete flux values—that fulfill these inequalities and optimize the objective—are easily computed by an algorithmic approach called <EM>Linear Programming</EM>.</P><P>In order to construct these inequalities, knowledge about the pathways is needed, which comes in the form of a systems biological model. Comprehensive databases for models are the <A href="http://www.ebi.ac.uk/biomodels-main/">BioModels</A> (Li et al., 2010) database hosted and maintained by EBI and the San Diego <A href="http://bigg.ucsd.edu">BiGG</A> project (King et al., 2016).</P><P>Until now, no manually curated model can be found in neither of these resources for the organism of interest: <EM>Lactobacillus johnsonii</EM> (ATCC 33200). Consequently, suitable knock-out targets were based on a systemic analysis of genome-scale metabolic network models of related species by investigating the effect of virtual gene deletions on the biomass production.</P><P>Our lab team selected the knock-out candidates <EM>glucose 6 phosphate isomerase</EM> and <EM>glucose 6-phosphate dehydrogenase</EM>. The herein described <EM>in-silico</EM> analysis was able to confirm this choice. In addition, we found <EM>phosphoglucomutase</EM> to be a possible knock-out candidate. Within the limited time frame of this project and since only a model of a related species is available, the results found cannot be considered definite. In an optimal case, subsequent studies would be required that include the bottom-up reconstruction of the exact strain under study. Despite these restrictions, the analysis at hand yielded already promising results and has saved numerous try and error runs, hence significant amounts of labor time and material.</P><H1 id="first-steps">First steps</H1><H2 id="inference-of-a-model">Inference of a model</H2><P>A Flux Balance Analysis (FBA) requires a detailed model, a so-called bottom-up reconstruction of the organism of interest, to be available. Unfortunately, no curated or at least auto-generated model is currently available for the organism of interest, <EM>Lactobacillus johnsonii</EM> (ATCC 33200). Since a large collection of models had been created as part of the path1models project (Büchel et al., 2013), we initially attempted to adopt an automatically created model from <A href="http://www.ebi.ac.uk/biomodels-main/">BioModels</A> datbase entries of relates species (see table 1).</P><TABLE><CAPTION>Caption: List of auto-generated BioModels entries which were used for
    the inference of a model for <EM>Lactobacillus johnsonii</EM> (ATCC
    33200).</CAPTION><THEAD><TR><TH>Model ID</TH><TH>Organism</TH></TR></THEAD><TBODY><TR class="odd"><TD><TT>BMID000000140353</TT></TD><TD><EM>Lactobacillus casei</EM> BL23</TD></TR><TR class="even"><TD><TT>BMID000000140390</TT></TD><TD><EM>Lactobacillus kefiranofaciens</EM> ZW3</TD></TR><TR class="odd"><TD><TT>BMID000000140702</TT></TD><TD><EM>Lactobacillus crispatus</EM> ST1</TD></TR><TR class="even"><TD><TT>BMID000000140776</TT></TD><TD><EM>Lactobacillus acidophilus</EM> 30SC</TD></TR><TR class="odd"><TD><TT>BMID000000140939</TT></TD><TD><EM>Lactobacillus johnsonii </EM>CNCM I-12250 / La1 / NCC 533</TD></TR><TR class="even"><TD><TT>BMID000000141107</TT></TD><TD><EM>Lactobacillus johnsonii</EM> FI9785</TD></TR><TR class="odd"><TD><TT>BMID000000141192</TT></TD><TD><EM>Lactobacillus casei</EM> Zhang</TD></TR><TR class="even"><TD><TT>BMID000000141757</TT></TD><TD><EM>Lactobacillus salivarius</EM> UCC118</TD></TR><TR class="odd"><TD><TT>BMID000000141915</TT></TD><TD><EM>Lactobacillus plantarum</EM> JDM1</TD></TR><TR class="even"><TD><TT>BMID000000142054</TT></TD><TD><EM>Lactobacillus casei</EM> LC2W</TD></TR><TR class="odd"><TD><TT>BMID000000142095</TT></TD><TD><EM>Lactobacillus helveticus</EM> R0052</TD></TR><TR class="even"><TD><TT>BMID000000142180</TT></TD><TD><EM>Lactobacillus casei</EM> W56</TD></TR><TR class="odd"><TD><TT>BMID000000142342</TT></TD><TD><EM>Lactobacillus helveticus</EM> H10</TD></TR><TR class="even"><TD><TT>BMID000000142548</TT></TD><TD><EM>Lactobacillus plantarum</EM> ZJ316</TD></TR><TR class="odd"><TD><TT>BMID000000142550</TT></TD><TD><EM>Lactobacillus rhamnosus</EM> ATCC 8530</TD></TR><TR class="even"><TD><TT>BMID000000142634</TT></TD><TD><EM>Lactobacillus johnsonii</EM> DPC 6026</TD></TR><TR class="odd"><TD><TT>BMID000000142799</TT></TD><TD><EM>Lactobacillus buchneri</EM> CD034</TD></TR><TR class="even"><TD><TT>BMID000000142807</TT></TD><TD><EM>Lactobacillus brevis</EM> KB290</TD></TR></TBODY></TABLE><P>Our inference approach was to manipulate the model for the <EM>Lactobacillus johnsonii</EM> NCC 533. Based on the METACYC identifiers, we added those reactions for which <SPAN class="math inline">\(70%\)</SPAN> of the other lactobacilli models contained an orthologue correspondents. Afterwards, we used the feature lists of the PATRIC (Wattam et al., 2014) database to identify those reactions that are unique for each strain, and made according changes, although, we still retained those reactions with <SPAN class="math inline">\(70%\)</SPAN> prevalence.</P><H2 id="analysis-for-the-inferred-model">Analysis for the inferred model</H2><P>For fulfilling an external function, metabolites can leave a system through outbound fluxes. For instance, ATP can leave a system through a growth function and thus driving proliferation. Often, the objective function is an outbound flux. Our project attempts to create an organism incapable of processing <EM>glucose-6-phosphate</EM> (G6P) while still being viable. Consequently, the corresponding analysis should have two objectives: maximize growth and minimize G6P metabolism. In general, models contain a growth function. However, we had to create an objective function for the G6P metabolism.</P><P>We implemented such an objective by considering G6P production and usage separately. For each scenario, we introduced a pseudo-metabolism whose concentration was linked to fluxes that involve G6P. The pseudo-metabolisms serve as counters. A technical detail is that each reversible reaction had to be substituted by two irreversible ones. Doing so prevents a counter from decreasing, and thus underestimating G6P metabolism, if an associated flux is reversed. For the pseudo-metabolites, two outbound fluxes were added as new objective functions.</P><P>Given these objectives, we conducted an FBA and tested knock-outs combinations of up to three reactions with the COBRApy package (Ebrahim, Lerman, Palsson, &amp; Hyduke, 2013) in an iPython notebook (Perez &amp; Granger, 2007) under usage of IBM's <A href="https://www-01.ibm.com/software/commerce/optimization/cplex-optimizer/" style="padding-right: 0;">cplex</A> optimizer. For computation time reasons, the knock-out candidates were limited to those <SPAN class="math inline">\(36%\)</SPAN> reactions that were more active in the FBA which minimizes G6P usage compared to a maximization.</P><P>Unfortunately, the results were in the noisy range of numerical zero (<SPAN class="math inline">\(10^{-11}\)</SPAN>), and therefore inconclusive.</P><H1 id="a-well-annotated-non-curated-model">A well annotated, non-curated model</H1><P>After the inference of a model for <EM>Lactobacillus johnsonii</EM> (ATCC 33200) failed, we used more comprehensive models of more distant related organisms. In the following we concentrate on a model for <EM>Lactobacillus plantarum</EM> WCFS1 from the BioModels database with the ID <TT><A href="http://www.ebi.ac.uk/biomodels-main/MODEL1507180045">MODEL1507180045</A></TT>. The model was manually generated by Teusink et al., 2006. However, the database considers it a non-curated model, because the semantics has not been fully checked yet.</P><P>Similar to the last analysis, pseudo-metabolites were used for creating an objective functions for the G6P metabolism. This time flux variability analysis (FVA) was used instead of FBA. FBA only finds a single solution (rate values for all fluxes) that optimize the objective functions under consideration of the specified constraints of the model. In contrast, FVA determines the range of rates a flux can have without jeopardizing the objective functions or constraints. Furthermore, we allowed the ranges of the FVA to deviate from the optimal solution by <SPAN class="math inline">\(10%\)</SPAN> in order to account for some biological variability.</P><P>Using FVA we identified <SPAN class="math inline">\(40\)</SPAN> knock-out candidates, for which combinations of up to three reactions were tested. These candidates were chosen by the following criteria:</P><UL><OL style="list-style-type: decimal"><LI>It is possible for the reaction to be both active and inactive in a living cell.</LI><LI>It is possible for the reaction to be inactive if no G6P is metabolised.</LI><LI>On average, the reaction is less active for minimal G6P metabolism.</LI></OL></UL><P>These criteria were tested by conducting two FVAs. One that maximizes growth and a second that minimizes G6P metabolism. The mathematical formulation for the given criteria are then:</P><UL style="list-style-image: url('https://static.igem.org/mediawiki/2016/5/5e/Tuebingen_Bull.png');"><LI>The minimum of the range is <SPAN class="math inline">\(\le 0\)</SPAN> in both FVAs.</LI><LI>The maximum of the range is <SPAN class="math inline">\(\ge 0\)</SPAN> in both FVAs.</LI><LI>The means of the ranges from both FVAs differ more than <SPAN class="math inline">\(200\)</SPAN>.</LI></UL></DIV><DIV class="contentRow"><DIV class="contentCell"><P>The analysis yields that a knockout of the <EM><SPAN class="math inline">\(\beta\)</SPAN>-phosphoglucomutase</EM> and the <EM>G6P-isomerase</EM> are able to prevent <SPAN class="math inline">\(\beta\)</SPAN>-G6P metabolism. We summarized the results for up to three knock-out reactions by counting how often a reaction participated in KO combination, which shut down <EM><SPAN class="math inline">\(\beta\)</SPAN>-G6P</EM> metabolism. The ranked list of the top 5 knock-out reactions read:</P><UL><OL style="list-style-type: decimal"><LI>beta-phosphoglucomutase</LI><LI>G6P-isomerase</LI><LI>G6P-dehydrogenase</LI><LI>alpha-phosphoglucomutase</LI><LI>maltose exchange</LI></OL></UL></DIV><DIV class="contentCell contentSmallCell"><FIGURE><FIGCAPTION>
                Overview over the reactions identified as knock-out candidates and
                complementary glycolysis reactions.
            </FIGCAPTION></FIGURE></DIV></DIV><DIV class="contentBothRows"><P>The reaction top 14 is <EM>adenylate-kinase</EM> and top 15 is <EM>asparagine-synthetase</EM>, which are biologically unlikely knock-out candidates. At no point we were able to bar the system from metabolising <EM><SPAN class="math inline">\(\alpha\)</SPAN>-G6P</EM>.</P><P>After disregarding unlikely candidates, the following pattern emerged in a visualization with the Escher tool, see figure 1 (King et al., 2015).</P><P>For an illustration of the normal metabolism flow see figure 2, and how a successful <EM><SPAN class="math inline">\(\beta\)</SPAN>-G6P</EM> knock-out affects that flow see figure 3.</P></DIV><DIV class="contentRow"><DIV class="contentCell"><FIGURE><FIGCAPTION>
                Sample flow for a growth maximizing objective. Thicker reaction arrows
                indicate higher flux rates.
            </FIGCAPTION></FIGURE></DIV><DIV class="contentCell"><FIGURE><FIGCAPTION>
                Fluxes are redistributed as a consequence of the knock-out (orange),
                leading to a decrease of Maltose phosporylase activity compared to the fluxes in figure 2.
            </FIGCAPTION></FIGURE></DIV></DIV><DIV class="contentBothCols"><H1 id="discussion-and-conclusion">Discussion and Conclusion</H1><P>Our analysis involved flux balance analysis (FBA) of an inferred model for <EM>Lactobacillus johnsonii</EM> (ATCC 33200), and a flux variability analysis (FVA) of <EM>Lactobacillus plantarum</EM> WCFS1. The inferred model yielded only noisy, numerically zero, fluxes, which did not allow for conclusions. In contrast, the manually generated model of another <EM>Lacobacillus</EM> allowed sound knock-out testing. The top candidates were the <EM>phosphoglucomutase</EM>, the <EM>G6P-isomerase</EM>, and the <EM>G6P-dehydogenase</EM>. The latter two reactions were in our experiments.</P><P>After assessing the results with the biochemists of our group, we conclude that the FVA was biologically reasonable. This is even more so, because the knock-out candidates used in the experiments were verified by the FVA. As a consequence, we suggested to test a knock-out of the <EM>phosphoglucomutase</EM>. These promising initial results should be experimentally validated in follow-up studies. The simulation allowed the wet-lab group to reduce serendipities, and saved numerous try and error runs.</P><P>In our simulations, we were unable to prevent <SPAN class="math inline">\(\alpha\)</SPAN>-G6P metabolism from taking place, because the metabolism is connected to more fluxes than the <SPAN class="math inline">\(\beta\)</SPAN> isomer (nine compared to just two). Therefore, a small reflection upon the projects intend is needed: We intend to synthesise a model organism for fructose intolerance. Such an organism is unable to acquire energy (ATP) from G6P, which we want to achieve by reducing G6P metabolism while increasing F6P metabolism. A more precise question would have been to test if the atoms of G6P could be integrated in F6P of the glycolysis pathway. The analysis for that question would involve a system with detailed description of cell compartments and a more sophisticated objective function. However, the complexity of such an analysis could easily be extended to a significantly larger study. Therefore, we cannot give a definite answer within the scope of an iGem project! But still, our simpler analysis suggest that our project is plausible to be achieved.</P><P>(References are listed <A href="https://2016.igem.org/Team:Tuebingen/References">here</A>)</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>