<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Tuebingen_References skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Tuebingen/References</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="grailWrapper"><NAV><UL><LI id="sectionTeam" class="iGemSabber"><A onclick="return true;">Team</A></LI><LI><A href="https://2016.igem.org/Team:Tuebingen">Home</A></LI><LI><A href="https://2016.igem.org/Team:Tuebingen/Team">The Team</A></LI></UL><LI id="sectionFruitforce" class="whiteSabber"><A onclick="return true;">Fruitforce</A></LI><LI><A href="https://2016.igem.org/Team:Tuebingen/FruitSummary">Introduction</A>
                            #</LI><LI><A href="https://2016.igem.org/Team:Tuebingen/L_johnsonii">L. johnsonii</A></LI><LI><A href="https://2016.igem.org/Team:Tuebingen/FruitDesign">Project design</A></LI><LI><A href="https://2016.igem.org/Team:Tuebingen/Model">Model</A></LI><LI><A href="https://2016.igem.org/Team:Tuebingen/Results">Results</A></LI><LI id="sectionSide" class="blueSabber"><A onclick="return true;">Side Projects</A></LI><LI><A href="https://2016.igem.org/Team:Tuebingen/CloneWars">Clone Wars</A></LI><LI><A href="https://2016.igem.org/Team:Tuebingen/DeathStar">Death Star</A></LI><LI><A href="https://2016.igem.org/Team:Tuebingen/Measurement">Interlab Study</A></LI><LI id="sectionLab" class="orangeSabber"><A onclick="return true;">Lab</A></LI><LI><A href="https://2016.igem.org/Team:Tuebingen/Parts">Parts</A></LI><LI><A href="https://2016.igem.org/Team:Tuebingen/Notebook">Notebook</A></LI><LI><A href="https://2016.igem.org/Team:Tuebingen/Safety">Safety</A></LI><LI><A href="https://2016.igem.org/Team:Tuebingen/Methods">Methods</A></LI><LI id="sectionHR" class="redSabber"><A onclick="return true;">HP</A></LI><LI><A href="https://2016.igem.org/Team:Tuebingen/HP/Silver">Summary</A></LI><LI><A href="https://2016.igem.org/Team:Tuebingen/HP/Gold">Internship</A></LI><LI><A href="https://2016.igem.org/Team:Tuebingen/HP/Marburg1">Symposium Marburg</A></LI><LI><A href="https://2016.igem.org/Team:Tuebingen/HP/Marburg2">iGEM Marburg</A></LI><LI><A href="https://2016.igem.org/Team:Tuebingen/HP/Paris">EE16 Paris</A></LI><LI><A href="https://2016.igem.org/Team:Tuebingen/HP/Constance">Science Slam</A></LI><LI><A href="https://2016.igem.org/Team:Tuebingen/HP/SciencePub">Science Pub Tue</A></LI><LI><A href="https://2016.igem.org/Team:Tuebingen/HP/Symposium">Symposium Tue</A></LI><LI id="sectionAttributions" class="purpleSabber"><A onclick="return true;">Attributions</A></LI><LI><A href="https://2016.igem.org/Team:Tuebingen/Collaborations">Collaborations</A></LI><LI><A href="https://2016.igem.org/Team:Tuebingen/Sponsoring">Sponsoring</A></LI><LI><A href="https://2016.igem.org/Team:Tuebingen/Attributions">Acknowledgements</A></LI><LI><A href="https://2016.igem.org/Team:Tuebingen/References">References</A></LI></NAV><DIV class="contentArea"><DIV class="contentBothCols"><H2 id="bibliography">Bibliography</H2><P><STRONG>(Ali et al., 1998)</STRONG>
Ali, M., Rellos, P., &amp; Cox, T. M. (1998). Hereditary fructose intolerance. Journal of medical genetics, 35(5), 353-365.
ISO 690	
</P><P><STRONG>(Büchel et al., 2013)</STRONG>
Büchel, F., Rodriguez, N., Swainston, N., Wrzodek, C., Czauderna, T., Keller, R., … Le Novère, N. (2013). Path2Models: large-scale generation of computational models from biochemical pathway maps. BMC Systems Biology, 7, 116. https://doi.org/10.1186/1752-0509-7-116
</P><P><STRONG>(Donaldson et al., 1993)</STRONG>
Donaldson, I., Doyle, C., Matas, N., Expression of rat liver hexokinase in yeast results in fructose intolerance. BMC Systems Biology, 7, 116.
</P><P><STRONG>(Ebrahim, Lerman, Palsson, &amp; Hyduke, 2013)</STRONG>
Ebrahim, A., Lerman, J. A., Palsson, B. O., &amp; Hyduke, D. R. (2013). COBRApy: COnstraints-Based Reconstruction and Analysis for Python. BMC Systems Biology, 7, 74. https://doi.org/10.1186/1752-0509-7-74
</P><P><STRONG>(Falsen et al., 1999)</STRONG> 
Falsen, E., Pascual C., Sjoden B., Ohlen M., and Collins M.D. “Phenotypic and phylogenetic characterization of a novel Lactobacillus species from human sources: description of Lactobacillus iners sp. nov.” International Journal of Systemic Bacteriology. 1999. Volume 49. p. 217-221.</P><P><STRONG>(Hamilton-Miller et al, 2003)</STRONG>
Hamilton-Miller, J.M. “The role of probiotics in the treatment and prevention of Helicobacter pylori infection.” International Journal of Antimicrobial Agents. 2003. Volume 22. p. 360-366.
</P><P><STRONG>(Hine &amp; Martin, 2015)</STRONG>
Hine, R., &amp; Martin, E. (2015). A Dictionary of Biology (7th ed.). OUP Oxford.
</P><P><STRONG>(King et al., 2015)</STRONG>
King, Z. A., Dräger, A., Ebrahim, A., Sonnenschein, N., Lewis, N. E., &amp; Palsson, B. O. (2015). Escher: A Web Application for Building, Sharing, and Embedding Data-Rich Visualizations of Biological Pathways. PLOS Comput Biol, 11(8), e1004321. https://doi.org/10.1371/journal.pcbi.1004321
</P><P><STRONG>(King et al., 2016)</STRONG>
King, Z. A., Lu, J., Dräger, A., Miller, P., Federowicz, S., Lerman, J. A., … Lewis, N. E. (2016). BiGG Models: A platform for integrating, standardizing and sharing genome-scale models. Nucleic Acids Research, 44(D1), D515–D522. https://doi.org/10.1093/nar/gkv1049
</P><P><STRONG>(Klaenhammer et al., 2007)</STRONG>
Klaenhammer, T.R., Azcarate-Peril, M.A., Altermann, E., and Barrangou, R. “Influence of the Dairy Environment on Gene Expression and Substrate Utilization in Lactic Acid Bacteria.” The Journal of Nutrition. 2007. Volume 137. p. 748S-750S.
</P><P><STRONG>(La Ragione et al. 2004)</STRONG>
La Ragione, R.M., Narbad, A., Gasson M.J., Woodward M.J. “In vivo characterization of Lactobacillus johnsonii FI9785 for use as a defined competitive agent against bacterial pathogens in poultry.” Letters in Applied Microbiology. 2004. Volume 38. p. 197-205.
</P><P><STRONG>(Lee et al., 2010) </STRONG>
Lee, H. J., Kim, E., &amp; Kim, J. S. (2010). Targeted chromosomal deletions in human cells using zinc finger nucleases. Genome research, 20(1), 81-89.
</P><P><STRONG>(Li et al., 2010)</STRONG>
Li, C., Donizelli, M., Rodriguez, N., Dharuri, H., Endler, L., Chelliah, V., … Laibe, C. (2010). BioModels Database: An enhanced, curated and annotated resource for published quantitative kinetic models. BMC Systems Biology, 4, 92. https://doi.org/10.1186/1752-0509-4-92
</P><P><STRONG>(Liu et al., 1997) </STRONG>
Liu, Q., Segal, D. J., Ghiara, J. B., &amp; Barbas, C. F. (1997). Design of polydactyl zinc-finger proteins for unique addressing within complex genomes. Proceedings of the National Academy of Sciences, 94(11), 5525-5530.
</P><P><STRONG>(Lombardo et al., 2007) </STRONG>
Lombardo, A., Genovese, P., Beausejour, C. M., Colleoni, S., Lee, Y. L., Kim, K. A., ... &amp; Holmes, M. C. (2007). Gene editing in human stem cells using zinc finger nucleases and integrase-defective lentiviral vector delivery. Nature biotechnology, 25(11), 1298-1306.
</P><P><STRONG>(Mutschler et al., 2011) </STRONG>
Mutschler, H., Gebhardt, M., Shoeman, R. L., &amp; Meinhart, A. (2011). A novel mechanism of programmed cell death in bacteria by toxin–antitoxin systems corrupts peptidoglycan synthesis. PLoS Biol, 9(3), e1001033.
</P><P><STRONG>(O'Sullivan et al., 1993) </STRONG> 
O'Sullivan, D. J., &amp; Klaenhammer, T. R. (1993). High-and low-copy-number Lactococcus shuttle cloning vectors with features for clone screening. Gene, 137(2), 227-231.
</P><P><STRONG>(Orth et al., 2010)</STRONG>
Orth, Jeffrey D., Ines Thiele, and Bernhard Ø. Palsson. “What Is Flux Balance Analysis?” Nature Biotechnology 28, no. 3 (March 2010): 245–48. doi:10.1038/nbt.1614. Orth, J. D., Thiele, I., &amp; Palsson, B. Ø. (2010). What is flux balance analysis? Nature Biotechnology, 28(3), 245–248. https://doi.org/10.1038/nbt.1614
</P><P><STRONG>(Perez &amp; Granger, 2007)</STRONG>
Perez, F., &amp; Granger, B. E. (2007). IPython: A System for Interactive Scientific Computing. Computing in Science Engineering, 9(3), 21–29. https://doi.org/10.1109/MCSE.2007.53
</P><P><STRONG>(Pantoflickova et al., 2003)</STRONG>
Pantoflickova, D., Corthesy-Theulaz, I., Dorta, G., Stolte, M., Isler, P., Rochat, F., Enslen, M., Blum, A.L. “Favourable effect of regular intake of fermented milk containing Lactobacillus johnsonii on Helicobacter pylori associated gastritis.” Alimentary Pharmacology &amp; Therapeutics. 2003. Volume 18. p. 805-813.
</P><P><STRONG>(Pridmore et al., 2004)</STRONG>
Pridmore, R.D., Berger, B., Desiere, F., Vilanova, D., Barretto, C., Pittet, A.C., Zwahlen, M.C., Rouvet, M., Altermann, E., Barrangou, R., Mollet, B., Mercenier, A., Klaenhammer, T., Arigoni, F., and Schell, M.A. “The genome sequence of the probiotic intestinal bacterium Lactobacillus johnsonii NCC 533.” Proceedings of the National Academy of Sciences of the United States of America. 2004. Volume 101. p. 2512-2517.
</P><P><STRONG>(Skerra, 1994) </STRONG>
Skerra, A. (1994). Use of the tetracycline promoter for the tightly regulated production of a murine antibody fragment in Escherichia coli. Gene, 151(1), 131-135.
</P><P><STRONG>(Teusink et al., 2006)</STRONG>
Teusink, B., Wiersma, A., Molenaar, D., Francke, C., Vos, W. M. de, Siezen, R. J., &amp; Smid, E. J. (2006). Analysis of Growth of Lactobacillus plantarum WCFS1 on a Complex Medium Using a Genome-scale Metabolic Model. Journal of Biological Chemistry, 281(52), 40041–40048. https://doi.org/10.1074/jbc.M606263200
</P><P><STRONG>(Wattam et al., 2014)</STRONG>
Wattam, A. R., Abraham, D., Dalay, O., Disz, T. L., Driscoll, T., Gabbard, J. L., … Sobral, B. W. (2014). PATRIC, the bacterial bioinformatics database and analysis resource. Nucleic Acids Research, 42(Database issue), D581–D591. https://doi.org/10.1093/nar/gkt1099
</P><P><STRONG>(Yasawy et al., 2009)</STRONG>
Yasawy, M. I., Folsch, U. R., Schmidt, W. E., &amp; Schwend, M. (2009). Adult hereditary fructose intolerance. World J Gastroenterol, 15(19), 2412-2413.</P><P><STRONG>(Young, 2002) </STRONG>
Young, R. Y. (2002). Bacteriophage holins: deadly diversity. Journal of molecular microbiology and biotechnology, 4(1), 21-36.
</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>