Team:Tuebingen/Model
Team
Home
The Team
Fruitforce
Introduction #
L. johnsonii
Project design
Model
Results
Side Projects
Clone Wars
Death Star
Interlab Study
Lab
Parts
Notebook
Safety
Methods
HP
Summary
Internship
Symposium Marburg
iGEM Marburg
EE16 Paris
Science Slam
Science Pub Tue
Symposium Tue
Attributions
Collaborations
Sponsoring
Acknowledgements
References
Summary
Our iGEM project intends to reduce the ability of lactobacilli to metabolize glucose. A possible approach is to knock-out genes along glucose-consuming pathways. However, the absence of the knocked-out genes must not jeopardize the bacteria's ability to live and proliferate. Such sets of genes could be identified either by educated guessing or by systems biological analysis.
Systems biological studies are capable of inferring the response of cells—or more generally: systems—under consideration of all genes and their network of interactions (Hine & Martin, 2015). In our project, we are interested in the rates of metabolism for fructose and glucose. Because we investigate gene knock-outs, we do not consider the time-course of metabolite concentrations (in contrast to last year's project). Instead, we analyse the flow of metabolites through the metabolic pathways of a system (Orth et al., 2010). Such a flow is computed with a flux balance analysis (FBA).
FBA assigns to each possible reaction a quantity, called flux value, that specifies with which rate the reactants are converted into the corresponding product in correct stoichiometric proportions. However, these fluxes are subjected to the following constraints:
Mass conversion must be respected. Metabolites may only enter/leave a system over explicitly defined in-/outbound fluxes.
A flux has a maximal and minimal rate
Metabolites have a lower and upper bound amount
These constraints are mathematically formulated as a set of inequalities. Furthermore, FBA optimizes specific fluxes, which are called objective functions. A possible objective it the maximization of the growth function. Concrete flux values—that fulfill these inequalities and optimize the objective—are easily computed by an algorithmic approach called Linear Programming.
In order to construct these inequalities, knowledge about the pathways is needed, which comes in the form of a systems biological model. Comprehensive databases for models are the BioModels (Li et al., 2010) database hosted and maintained by EBI and the San Diego BiGG project (King et al., 2016).
Until now, no manually curated model can be found in neither of these resources for the organism of interest: Lactobacillus johnsonii (ATCC 33200). Consequently, suitable knock-out targets were based on a systemic analysis of genome-scale metabolic network models of related species by investigating the effect of virtual gene deletions on the biomass production.
Our lab team selected the knock-out candidates glucose 6 phosphate isomerase and glucose 6-phosphate dehydrogenase. The herein described in-silico analysis was able to confirm this choice. In addition, we found phosphoglucomutase to be a possible knock-out candidate. Within the limited time frame of this project and since only a model of a related species is available, the results found cannot be considered definite. In an optimal case, subsequent studies would be required that include the bottom-up reconstruction of the exact strain under study. Despite these restrictions, the analysis at hand yielded already promising results and has saved numerous try and error runs, hence significant amounts of labor time and material.
First steps
Inference of a model
A Flux Balance Analysis (FBA) requires a detailed model, a so-called bottom-up reconstruction of the organism of interest, to be available. Unfortunately, no curated or at least auto-generated model is currently available for the organism of interest, Lactobacillus johnsonii (ATCC 33200). Since a large collection of models had been created as part of the path1models project (Büchel et al., 2013), we initially attempted to adopt an automatically created model from BioModels datbase entries of relates species (see table 1).
Caption: List of auto-generated BioModels entries which were used for     the inference of a model for Lactobacillus johnsonii (ATCC     33200).
Model ID
Organism
BMID000000140353
Lactobacillus casei BL23
BMID000000140390
Lactobacillus kefiranofaciens ZW3
BMID000000140702
Lactobacillus crispatus ST1
BMID000000140776
Lactobacillus acidophilus 30SC
BMID000000140939
Lactobacillus johnsonii CNCM I-12250 / La1 / NCC 533
BMID000000141107
Lactobacillus johnsonii FI9785
BMID000000141192
Lactobacillus casei Zhang
BMID000000141757
Lactobacillus salivarius UCC118
BMID000000141915
Lactobacillus plantarum JDM1
BMID000000142054
Lactobacillus casei LC2W
BMID000000142095
Lactobacillus helveticus R0052
BMID000000142180
Lactobacillus casei W56
BMID000000142342
Lactobacillus helveticus H10
BMID000000142548
Lactobacillus plantarum ZJ316
BMID000000142550
Lactobacillus rhamnosus ATCC 8530
BMID000000142634
Lactobacillus johnsonii DPC 6026
BMID000000142799
Lactobacillus buchneri CD034
BMID000000142807
Lactobacillus brevis KB290
Our inference approach was to manipulate the model for the Lactobacillus johnsonii NCC 533. Based on the METACYC identifiers, we added those reactions for which \(70%\) of the other lactobacilli models contained an orthologue correspondents. Afterwards, we used the feature lists of the PATRIC (Wattam et al., 2014) database to identify those reactions that are unique for each strain, and made according changes, although, we still retained those reactions with \(70%\) prevalence.
Analysis for the inferred model
For fulfilling an external function, metabolites can leave a system through outbound fluxes. For instance, ATP can leave a system through a growth function and thus driving proliferation. Often, the objective function is an outbound flux. Our project attempts to create an organism incapable of processing glucose-6-phosphate (G6P) while still being viable. Consequently, the corresponding analysis should have two objectives: maximize growth and minimize G6P metabolism. In general, models contain a growth function. However, we had to create an objective function for the G6P metabolism.
We implemented such an objective by considering G6P production and usage separately. For each scenario, we introduced a pseudo-metabolism whose concentration was linked to fluxes that involve G6P. The pseudo-metabolisms serve as counters. A technical detail is that each reversible reaction had to be substituted by two irreversible ones. Doing so prevents a counter from decreasing, and thus underestimating G6P metabolism, if an associated flux is reversed. For the pseudo-metabolites, two outbound fluxes were added as new objective functions.
Given these objectives, we conducted an FBA and tested knock-outs combinations of up to three reactions with the COBRApy package (Ebrahim, Lerman, Palsson, & Hyduke, 2013) in an iPython notebook (Perez & Granger, 2007) under usage of IBM's cplex optimizer. For computation time reasons, the knock-out candidates were limited to those \(36%\) reactions that were more active in the FBA which minimizes G6P usage compared to a maximization.
Unfortunately, the results were in the noisy range of numerical zero (\(10^{-11}\)), and therefore inconclusive.
A well annotated, non-curated model
After the inference of a model for Lactobacillus johnsonii (ATCC 33200) failed, we used more comprehensive models of more distant related organisms. In the following we concentrate on a model for Lactobacillus plantarum WCFS1 from the BioModels database with the ID MODEL1507180045 . The model was manually generated by Teusink et al., 2006. However, the database considers it a non-curated model, because the semantics has not been fully checked yet.
Similar to the last analysis, pseudo-metabolites were used for creating an objective functions for the G6P metabolism. This time flux variability analysis (FVA) was used instead of FBA. FBA only finds a single solution (rate values for all fluxes) that optimize the objective functions under consideration of the specified constraints of the model. In contrast, FVA determines the range of rates a flux can have without jeopardizing the objective functions or constraints. Furthermore, we allowed the ranges of the FVA to deviate from the optimal solution by \(10%\) in order to account for some biological variability.
Using FVA we identified \(40\) knock-out candidates, for which combinations of up to three reactions were tested. These candidates were chosen by the following criteria:
It is possible for the reaction to be both active and inactive in a living cell.
It is possible for the reaction to be inactive if no G6P is metabolised.
On average, the reaction is less active for minimal G6P metabolism.
These criteria were tested by conducting two FVAs. One that maximizes growth and a second that minimizes G6P metabolism. The mathematical formulation for the given criteria are then:
The minimum of the range is \(\le 0\) in both FVAs.
The maximum of the range is \(\ge 0\) in both FVAs.
The means of the ranges from both FVAs differ more than \(200\).
The analysis yields that a knockout of the \(\beta\)-phosphoglucomutase and the G6P-isomerase are able to prevent \(\beta\)-G6P metabolism. We summarized the results for up to three knock-out reactions by counting how often a reaction participated in KO combination, which shut down \(\beta\)-G6P metabolism. The ranked list of the top 5 knock-out reactions read:
beta-phosphoglucomutase
G6P-isomerase
G6P-dehydrogenase
alpha-phosphoglucomutase
maltose exchange
Overview over the reactions identified as knock-out candidates and                 complementary glycolysis reactions.
The reaction top 14 is adenylate-kinase and top 15 is asparagine-synthetase, which are biologically unlikely knock-out candidates. At no point we were able to bar the system from metabolising \(\alpha\)-G6P.
After disregarding unlikely candidates, the following pattern emerged in a visualization with the Escher tool, see figure 1 (King et al., 2015).
For an illustration of the normal metabolism flow see figure 2, and how a successful \(\beta\)-G6P knock-out affects that flow see figure 3.
Sample flow for a growth maximizing objective. Thicker reaction arrows                 indicate higher flux rates.
Fluxes are redistributed as a consequence of the knock-out (orange),                 leading to a decrease of Maltose phosporylase activity compared to the fluxes in figure 2.
Discussion and Conclusion
Our analysis involved flux balance analysis (FBA) of an inferred model for Lactobacillus johnsonii (ATCC 33200), and a flux variability analysis (FVA) of Lactobacillus plantarum WCFS1. The inferred model yielded only noisy, numerically zero, fluxes, which did not allow for conclusions. In contrast, the manually generated model of another Lacobacillus allowed sound knock-out testing. The top candidates were the phosphoglucomutase, the G6P-isomerase, and the G6P-dehydogenase. The latter two reactions were in our experiments.
After assessing the results with the biochemists of our group, we conclude that the FVA was biologically reasonable. This is even more so, because the knock-out candidates used in the experiments were verified by the FVA. As a consequence, we suggested to test a knock-out of the phosphoglucomutase. These promising initial results should be experimentally validated in follow-up studies. The simulation allowed the wet-lab group to reduce serendipities, and saved numerous try and error runs.
In our simulations, we were unable to prevent \(\alpha\)-G6P metabolism from taking place, because the metabolism is connected to more fluxes than the \(\beta\) isomer (nine compared to just two). Therefore, a small reflection upon the projects intend is needed: We intend to synthesise a model organism for fructose intolerance. Such an organism is unable to acquire energy (ATP) from G6P, which we want to achieve by reducing G6P metabolism while increasing F6P metabolism. A more precise question would have been to test if the atoms of G6P could be integrated in F6P of the glycolysis pathway. The analysis for that question would involve a system with detailed description of cell compartments and a more sophisticated objective function. However, the complexity of such an analysis could easily be extended to a significantly larger study. Therefore, we cannot give a definite answer within the scope of an iGem project! But still, our simpler analysis suggest that our project is plausible to be achieved.
(References are listed here )
