"
Page
Discussion
View source
History
teams
Log in
 
Team:Genspace/Notebook/August
From 2014.igem.org
Home
Team
Official Team Profile
Project
Open Lab Blueprint OpenTrons Robot IP-Free Genes BioGlyphics
Human Practices
Parts
Notebook
Protocols July August September October
Safety
Attributions
August Notebook
8.9.14: Transformation repeat
Background
We transformed the following ligation products:
Tube A = OFP
Tube B = YFP
Tube C = RFP
Tube D = CFP (Only had 4 vials of competent cells so did not transform CFP)
Tube BB = Backbone
Tube pUC19 = pUC19 Positive Control
1. Added the following to each tube:
Tubes A-D (20 µl total)
14 µl PCR product
3 µl 1:10 bb
2 µl buffer
1 µl ligase
Tube BB control (20 µl total)
14 µl H2O
3 µl 1:10 bb
2 µl buffer
1 µl ligase
2. Waited 45 mins
3. Pipetted 5 µl of each tube into new tubes labeled A-D and BB.
4. Pipetted 1 µl of pUC19 to pUC19 labeled tube.
5. Add 50 µl of DH5-Alpha to tubes A-D.
6. Add 1 µl of control DNA to pUC19
Followed NEB 5-alpha Competent E.coli (high efficiency) Transformation Protocol
(SEE ATTACHED AND BELOW)
For C2987H: Thaw a tube of NEB 5-alpha Competent E. coli cells on ice for 10 minutes.
For C2987I: Thaw a tube of NEB 5-alpha Competent E. coli cells on ice until the last ice crystals disappear. Mix gently and carefully pipette 50 µl of cells into a transformation tube on ice.
Add 1-5 µl containing 1 pg-100 ng of plasmid DNA to the cell mixture. Carefully flick the tube 4-5 times to mix cells and DNA. Do not vortex. (WE ADDED 5 µl)
Place the mixture on ice for 30 minutes. Do not mix.
Heat shock at exactly 42°C for exactly 30 seconds. Do not mix.
Place on ice for 5 minutes. Do not mix.
Pipette 950 µl of room temperature SOC into the mixture.
Place at 37°C for 60 minutes. Shake vigorously (250 rpm) or rotate. We used rotator inside incubator.
Warm selection plates to 37°C.
Mix the cells thoroughly by flicking the tube and inverting, then WE DID NOT perform several 10-fold serial dilutions in SOC.
Spread 50-100 µl of each dilution onto a selection plate and incubate overnight at 37°C. Alternatively, incubate at 30°C for 24-36 hours or 25°C for 48 hours.
We plated 200 µl from each tube into each plate. Julie will update tomorrow with results.
8.10.14 Results of ligation part 1:
Hi gang,
Backbone
Insert
Drug Res
Purpose
Results
circular pUC19
-
amp
test if cells work
100s of colonies
digested pSB1C3
-
chlor
test nonspecific BB alone ligation
no colonies
digested pSB1C3
digested OFP
chlor
to generate plasmid
no colonies
digested pSB1C3
digested YPF
chlor
to generate plasmid
no colonies
digested pSB1C3
digested RFP
chlor
to generate plasmid
no colonies
digested pSB1C3
digested CFP
chlor
to generate plasmid
no colonies
Why didn't anything grow? Two possibilities come to mind:
1. Chloramphenicol levels are too high. I will double check my math from making them previously. I will also stick the plates back in the 37C incubator - sometimes if the dose is just a bit too high, the cells need an extra day to grow into colonies
2. Transformation efficiency was very low - we didn't plate enough cells and should have plated the entire culture to get just a few colonies.
8.11.14: Results of ligation part 2
Yesterday, Marty helped me out by following this protocol after I made fresh LB+Chlor plates:
Pull out tubes from fridge (they are in the "class samples" fridge). They are on te top shelf in a styrofoam container.
Spin all tubes 3 min 4000rpm.
Remove 500 ul sup by aspirating and discarding in proper waste.
Resuspend remaining sup and plate on an lb+chlor plate (on the bench top). Same as before, making sure to use te clean beads, not the waste beads!!
Incubate in 37c overnight.
Today, he went to check on the plates but there is still no sign of growth on any LB+Chlor plates.
Regarding Chlor concentration: I previously treated the 34 mg/ml stock as a 294x stock. This makes the final concentration 115 ug/ml, within the standard range of 34 ug/ml - 174 ug/ml for this drug. According to the IGEM website, however, they recommend a 34 ug/ml final concentration, which is how I treated the plates I made yesterday. I thought this less stringent concentration would allow more growth, but apparently no dice so far.
Random thoughts on why this hasn't worked:
DNA concentration? Both the insert and the backbone were both at high enough concentrations for 3 ul to show up on a gel (indicating lack of DNA is not an issue). (I've misplaced this gel image so I can't attach it at this moment).
Transformation competency? We know the cells can be transformed and grow on LB+Amp. We don't know that the cells can be transformed and grow on LB+Chlor.
Feel free to add to the discussion if you can think of reasons we may not be getting transformations!
8.9.14: PCR of GFP
Background
We need more GFP because round of ligation did not work, and we have digested GFP.
We can't find the GFP plasmid to PCR amplify it, so we are amplifying the previously PCRed Digested and Undigested pieces of GFP (instead of the plasmid) hoping that one of them will yield product.
Two PCR tubes are in the PCR machine. Tube 1D is the digested GFP (left side of machine). Tube 2 is the undigested GFP (right side of machine).
8.12.14: Colony PCRs to check colonies that came from 8.10 ligations
Background
In his email on Monday, Eric said he saw a few colonies on the ligation transformation plates (the ones Marty plated on 8.10.14):
Name
Construct
# Colonies
BB alone
BB alone
BB + A
OFP
1
BB + B
YFP
2
BB + C
RFP
4
BB + D
CFP
2
Even though there are more colonies on BB alone than BB+insert, I didn't want to throw away our potential constructs. So we will screen these while simultaneously setting up ligations for the next round of transformations. We will screen in two ways: (1) Colony PCR, which will give us results tonight IF IT WORKS, but I've never done colony PCR before and we don't have any positive control primers (meaning that if we see nothing on the gel, we won't know *why* we see nothing on the gel) and (2) traditional mini preps, from cultures we will inoculate tonight into 3 ml LB+Chlor.
From the internet, I gathered several protocols and made an amalgam of them, hoping that this will work to generate template DNA plasmid:
Add 20 ul DEPC-treated water to a PCR tube (one per colony to be tested).
Using a sterile pipette tip, take *just a smidge* of the edge of the colony. This is the potential biggest source of error, as too many cells or too old of cells will not work. These colonies are old, but if we use the edge of the colony, we can scrape just a bit of the fastest-growing cells (found on the edge) to inoculate into the water.
Incubate at 95C for 10 min in the thermocycler.
Spin down cell debris on tabletop PCR tube centrifuge (~20 sec). Use 2 ul of supernatant as template DNA in PCR reaction below.
Next, we set up a cocktail for our PCR reactions:
Component
ul/rxn
x 16 rxns
DEPC-treated H2O
20
320
Primer 1: VF2
1.5
24
Primer 2: VR
1.4
24
From this cocktail, we added 23 ul to PCR tubes (containing desiccated PCR  reagents) and 2 ul DNA template, for a total of 25 ul. We ran this in the thermocycler with the following program:
95C/4m - [(95C/30s - 55C/30s - 74C/1m)x30 cycles] - 74C/4 min - 4C/infinity
We added DNA dye directly to the PCR reactions and ran the entire reaction out on a 0.8% agarose gel.
Results
The gel is beautiful! The colony PCR worked!! We can use this screen to decide which cultures need to be mini prepped tomorrow.
Here's the bad news: the cultures and PCR reaction numbers are not the same. I can't decipher one from another. I have an *idea* of which is which, but no certainty. This should be avoided in the future with better communication among all team members (I'm not pointing fingers; I just hate that now we have to mini prep all the inoculated cultures. Why for do the short way (colony PCR) if we have to do the long way (mini prep) for all the samples?)
Conclusions
Lanes 1, 3, and 5-15 look very similar in size to one another. These are likely due to backbone ligation with itself in some fashion.
Lanes 2 and 4 look like they have a small insert. This may or may not be the size of one of our fluorescent protein genes. We expect the inserts to be ~700-800 bp *larger* than backbone alone, but since these are consistent in size for the same construct (I think), it's possible they are positive hits.
Lane 16 is the most promising hit. This band shows a shift of about 700 bp above the rest of the bands. This is most likely the A construct, which is OFP. Hopefully mini preps tomorrow confirm this!
8.13.14: Miniprep and digest to check constructs
Background
The cultures inoculated yesterday are going to be mini prepped by Jonathan. He's also agreed to digest and run a gel of them, to indicate whether the constructs are correct or not.
A. Minipreps were done according to LifeTechnology's protocol. Plasmid products were eluted in 50 ul TE and were used in the digest reactions.
B. Digests were done by first making the following cocktail:
Component
ul/rxn
x 22 rxns
DEPC-treated water
13
286
Cutsmart buffer
2
44
EcoRI
1
22
PstI
1
22
 
From the cocktail, 17 ul was added to 3 ul of mini prep plasmid. This was incubated 15 min at 37C.
C. Running the gel. DNA loading dye was added to the digest and the entire digest was run on a 0.8% agarose gel.
Results
More beautiful gels! With more promising results!
The digests were designed to release the fluorescent protein genes so there should be a ~700-800 bp band in addition to a larger band (representing the backbone).
From A (OFP): Both A1 and A2 are promising and should be sent for sequencing!
From B (YFP): No promising results. If I recall correctly, B is the PCR product that was cut by either EcoRI or PstI (I've lost that gel image!). So it makes sense that the mini preps wouldn't' contain this insert.
From C (RFP): No promising results.
From D (CFP): Lots of promising plasmids! D3, D4, and D6-D9 all have bands of the expected size! I don't know that we need to sequence all of these, but a few should be selected at random and sent for sequencing.
Conclusions
Both OFP and CFP look to have been cloned into the backbone! Next steps: sequence and submit!
We should also start designing a promoter/RBS to add to these plasmids so we can check them for functionality! I'm thinking we could maybe 1. anneal primers that include these sequences and have overlaps with the target plasmid and 2. cut the plasmids at the SpeI site and 3. use Gibson assembly to generate the plasmids!
8.19.14 Colony PCRs and Transformation
RFP prep:
20 µl H2O
heated 95 C  for 10 mins
spin to remove debris
PCR: 30 cycles both RFP and GFP
with compositions:
2 µl template
21 µl H2O
1 µl general forward GFP Primer
1 µl specific reversed RFP Primer
run on 1% agarose gel:
Results:
good amount of PCR products for both RFP and GFP
RFP #1
RFP #2
GFP
 
While waiting for result
Transformation: GFP on competent E.coli
Incubate 1 ul plasmid with 50 ul competent E. coli for 30 min on ice
heatshock 42 C for 45 sec
Incubate on ice 5 min
Can't fine SOC so we use LB
rotate for 60 mins
plated on LB+Amp and incubated at 37C
Results
Bands look great! The colony PCR protocol is a real winner!
Conclusions
Next step: clone into pSB1C3 background
8.21.14: Digest, PCR purification, and ligation of RFP/GFP inserts
Background
We want to take Tuesday's PCR products and put them into the pSB1C3 backbone.
Digest PCRs:
10 ul PCR product
2 ul cutsmart buffer (10x stock)
1 ul PstI
1 ul EcoRI
20 ul total --> incubate for 30 min at 37C
PCR purify digest product (only 14 ul - save 6 ul):
Follow kit protocol. Elute in elution buffer.
Worried that the washed columns won't bind DNA, we are going to use some of the set-aside (unpurified) digest product for a backup ligation. We'll run a gel of our purification, but we are going to set up a ligation beforehand, so we won't have even rough estimates of DNA concentrations.
Set up ligations:
Component
Using purified digest product
Using unpurified digest product
BB alone
dH2O
x
11
14
Insert (RFP or GFP)
14
3
x
1:10 BB
3
3
3
T4 buffer (10
1
1
1
The above were incubated 30 min at RT then stored at -20C.
Results
We ran a 1% gel of the digest before and after purification. We had a decent yield, maybe 40% of our initial digest product in the purified lanes.
Conclusions
Next step: Transform the ligated plasmids into E. coli!
8.25.14 Colony PCR to check E. coli transformants
Background
We have colonies growing from the transformations and want to check if they have the correct plasmid insert. We'll run a colony PCR on these before deciding which colonies to grow and make mini preps from.
Ligation
No. Colonies
Notes
Sample Name
BB + GFP (unpur)
0
BB + GFP (pur)
2
1-2
BB + RFP (pur)
7
3-10
BB + RFP (unpur)
7
11-17
BB alone
15
why so many???
18
A piece of each of the colonies included in the samples (so not all the BB alone) was inoculated into 20 ul H2O. This was incubated at 95C for 10 min, after which cell debris was removed by a quick centrifugation. 2 ul of the supernatant was used in each of the PCRs.
PCR set up started by making a cocktail:
Component
ul/rxn
x 18
DEPC-treated water
20
360
Primer 1 = VF2
1.5
27
Primer 2 = VR
1.5
27
Template
2
-
Total
25 (CT = 23)
414
From the CT, 23 ul was aliquoted into each of 18 PCR tubes. 2 ul of the template was added. PCRs were run in the machine with the following protocol:
94C/2min - [94C/30sec  - 55C/30sec - 72C/1min]x25cycles
Dye was added and the PCR run on a 0.8% agarose gel.
Results
Success!!! From the chart above:
Ligation
No. Colonies
Sample Name
Successful PCRs
BB + GFP (pur)
2
1-2
1,2
BB + RFP (pur)
7
3-10
3,8,9
BB + RFP (unpur
7
11-17
11,12,16
Conclusions
We have multiple constructs for both RFP and GFP that may have the correct sequence!
NEXT STEPS:
1. Inoculate the correct colonies into LB+Chlor
2. Miniprep cultures
3. Digest to verify insert size is correct
4. Send for sequence analysis!
8.21.14: Digestion of GFP and RFP PCR products
Background
Digestion of RFP and GFP amplicons using EcoRI and PstI Digest parameters: total volume 20ul H2O: 6ul Buffer(10x): 2ul DNA: 10ul EcoRI: 1ul PstI: 1 ul Made a 1% gel Agarose: 0.5 g TBE: 50ml etBr: 5ul
8.21.14: Ligation of purified and unpurified GFP, RFP, and backbone
8.24.14: Transformation of ligation products into E. coli
Background
Followed NEB recommended protocol to transform 5 ul of ligation products into E. coli. We changed the protocol by adding only 40 ul of competent cells (rather than 50 ul as recommended). We also changed the protocol because we wanted to plate all the cells onto the LB+Chlor plates. To concentrate cells, we centrifuged 3 min at 4000 rpm, removed 900 ul, resuspended the cell pellet in the remaining 100 ul, and spread this on an LB+Chlor plate using glass beads.
STEPS
Thawed a tube of NEB 5-alpha Competent E. coli cells on ice until the last ice crystals disappeared. Mixed gently and pipetted 50uL of cells into a transformation tube on ice.
Added 5uL containing 1pg - 100ng of plasmid DNA to the cell mixture. Flicked the tube several times to mix the cells and DNA.
Placed the mixture on ice for 30 minutes.
Heat shock the mixture for exactly 30 seconds at 42 degrees Celisus.
Placed back on ice for 5 minutes.
Pipetted 950uL of room temperature SOC with mostly LB into the mixture.
Place mixture at 37 degrees Celisus for an hour. Shaked vigorously.
Retrieved plates that were warmed at 37 degrees Celisus.
Flicked the tube to mix the cells thoroughly. Didn't do any serial dilutions.
Spin the cells for 3 minutes at 4,000rpm in the centerfudge. Removed 900mL and then placed the last 100mL onto the plates, using mixing beads, after mixing the cells with the liquid by pipetting up and down. Then we incubated at 30 degrees Celisus for 24hrs.
Results
There was colonies on the plates!
Conclusions
The next step is to check to see if the transformation was done correctly and if E. coli took up the RFP and GFP plasmid.
8.29.14: Miniprep of purified and unpurified GFP and RFP colonies
Miniprep on 8.29.14 of colonies 1-gfp-p, 2-gfp-p, 3-rfp-p, 8-rfp-p, 9-rfp-p, 11-rfp-un, 12-rfp-un, and 16-rfp-un
Retrieved from " http://2014.igem.org/Team:Genspace/Notebook/August "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
