"
Page
Discussion
View source
History
teams
Log in
 
Team:Genspace/Notebook/Protocol
From 2014.igem.org
Home
Team
Official Team Profile
Project
Open Lab Blueprint OpenTrons Robot IP-Free Genes BioGlyphics
Human Practices
Parts
Notebook
Protocols July August September October
Safety
Attributions
Protocols
Protocol: Use PCR to Amplify a Gene
Materials
PCR bead in tube (bead contains polymerase and nucleotides)
19 µL of deionized water (we used 16 µL by mistake and it still worked)
1 µL of template plasmid
2.5 µL of 10x forward primer solution
2.5 µL of 10x reverse primer solution
Prepare Primer DNA
Forward and reverse primer DNA are shipped in separate storage tubes. The first step is to make a 100X solution of each in the storage tubes by adding deionized water. Multiply the number of nanomoles by ten and add that many microliters of purified water.
Make 10x working solutions of the primer DNAs
In separate small Eppendorf tubes, mix 10 µL of the primers with 90 µL of deionized water. Close top and vortex each tube. Make sure to label the tubes. Set aside.
PCR Reaction
The objective in this step is to PCR out the gene of interest from a plasmid and add biobrick prefix and suffix so it can be cloned into pSB1C3.
Step 1: Add deionized water to PCR tube.  Flick tube to dissolve PCR bead. (DO NOT VORTEX – amount is too small.)
Step 2: Add template plasmid, forward primer, and reverse primer.  Flick tube to mix (DO NOT VORTEX)
Step 3: Place in PCR machine and run through cycle (takes about 90 minutes)
Step 4: Remove & label tube
If all went well, you should have a tube of lots of the template DNA with blunt ends.
*BREAKPOINT* (A breakpoint is a place where you can stop working. Please make sure all of your test tubes are labeled and placed in the freezer in the iGEM box.)
Protocol: PCR Product Purification
In this next step we need to remove the polymerase, template DNA and any primers from the test tube. A later step in this process will be to convert the blunt ends of the GFP DNA into sticky ends. If the polymerase is not removed then it will be impossible to create the sticky ends since as the sticky ends are created, the polymerase will convert them back to blunt ends. The purification process is done with a kit. In our lab we used the PureLink Quick Gel Extraction and PCR Purification Combo made by Invitrogen (Catalog number: K2200-01)
Step 1: Combine. Add 4 volumes of Binding Buffer (B2) with isopropanol to 1 volume of PCR sample (in our case, 88 µL of B2). Mix well.
Step 2: Load. Pipet the sample into a PureLink Clean-up Spin Column in a Wash Tube. Centrifuge the column at >10,000 x g for 1 minute. Discard the flow through.
Step 3: Wash. Re-insert the column into the Wash Tuben and add 650 µL Wash Buffer (W1) with ethanol. Centrifuge the column at >10,000 x g for 1 minute.
Step 4: Remove ethanol. Discard the flow-through and place the column in the same Wash Tube.  Centrifuge the column at maximum speed for 2-3 minutes.
Step 5: Elute. Place the column into a clean 1.7 mL Elution Tube. Add 50 µL Elution Buffer (E1) to the column. Incubate the column at room temperature for 1 minute.  Centrifuge the column at maximum for 1 minute.
The elution tube contains the purified PCR product. Make sure to label the tube.
*BREAKPOINT*
Protocol: Run a Gel to Check PCR Product
Now that we have a purified PCR product we would like to confirm that we actually have some DNA that appears to be correct. To do this, we will run a gel. The fluorescent protein genes we worked with are approximately 700 base pairs in length so we will run 100 bp ladder to compare the PCR product to.
The first step is to make a gel of the right concentration. Gels concentrations run 0.7% - 2%.
For 500 bp sequences, concentration of > 1% is good. For 783 bp, a 1% gel should work.
Make the gel:
Materials:
0.5 g of agarose
50 mL of TAE
5 µL of Ethidium Bromide (for small gel)
Small Electrophoresis box
Step 1: Measure .5 g of agarose. Use folding paper (fold in quarters first). Also be sure to take scale after putting on folding paper.
Step 2: In 250 mL Erlenmeyer Flask, mix agarose and 50 mL of TAE. Swirl flask to dissolve most of the agarose. Liquid will be foggy. Cover flask with Saran Wrap and microwave for 2 minutes at high. Liquid will be clear.
Step 3: Add 5 µL of Ethidium Bromide. Swirl.
Step 4: Set up electrophoresis box to make gel. Make sure tray is oriented so that liquid gel will not leak out. Insert well comb. Pour in enough gel mix to fill tray. Let sit for about 15 minutes (will turn foggy when ready) to allow gel to set.
Run the gel:
Step 5: Add a splash of TAE to lubricate gel. Wiggle out well comb being careful not to break the agar. Gently take out tray and rotate 90 degrees. Make sure that wells are on NEGATIVE side (black lead) or sample will run the wrong direction (DNA is negative and will be drawn to the positive side).
Step 6: Fill gel box with TAE 1% solution. It should just go over the top of the gel.
Step 7: Get DNA sample ready to run gel. Place a small piece of parafilm over a test tube tray and make a small indentation (area will be used to mix DNA and loading dye). Mix 2 µL (we used 9 in error) of DNA sample with 1 µL of loading dye in parafilm indentation. Using pipette, place full mixture into one of the gel wells. In a neighboring well, place in 3 µL of 100 base pair ladder.
Step 8: Run the gel by plugging leads into the power supply. If power is on, you will see small bubbles coming off of wires. Let the gel run until the dye has gone about 1/3 the way across the gel.
Step 9: To see the actual DNA, take the gel tray and view it under ultraviolet light. If everything worked you will see one band in the DNA column and multiple bands in the ladder column. See where the DNA band lines up against the ladder to estimate the number of base pairs and to see if they correspond to the expected amount.
Take a picture of the gel
Procedure: Digest a PCR product
Materials
9.5 µL of deonized water
6 µL of purified PCR product
1 µL of EcoRI (restriction enzyme)
1.5 µL if PstI (restriction enzyme)
2 µL of buffer (must match restriction enzymes used – check with lab director)
IMPORTANT: Add materials in this order:
Water
Buffer
DNA
Restriction Enzymes
Flick to mix.
Heat in PCR machine at 37 degrees C for 1 hour, 20min at 80C to inactivate enzymes
Ice mixture
*BREAKPOINT*
Procedure: Restriction digest of pUC19 plasmid
Materials
pUC19 plasmid prep
10x buffer designed to work with enzyme
enzyme
dH2O
thermal cycler or heat bath
gel box
agarose
1x TAE
ethidium bromide
Procedure
for a 20ul reaction:
for one reaction tube:
reagent
ul
dH2O
12
buffer (10x)
2
enzyme
1
DNA
5
 
Total Volume
20
In a 25ul PCR reaction tube, add 12 ul of dH2O, 5ul of 10x buffer (buffer must be compatible with enzyme being used), add 5 ul of DNA product, and lastly add 1 ul of enzyme.
* if making a number of tubes, create a cocktail with enough reagents to aliquot into all tubes used. combine dH2O, buffer, and enzyme into a 2ml tube. aliquot 15ul of cocktail into 25ul PCR reaction tubes. Add DNA directly to PCR tubes.
cover tubes and place in thermal cycler and run restriction digest protocol
after the run is completed, follow procedure 3: running a gel in order to visualize the results of your restriction digest
Procedure: How to calculate molar ratios
Background info:
The plasmid is about 2000 base pairs (bp).
Our PCRed GFP (gene of interest) is about 750 bp.
Assume that 1 uL of this PCR purified GFP = 50 ng (info from Ellen).
Calculations:
We have 6 uL of our gene of interest in a solution totaling 20 uL.
So we have:
\(\displaystyle \frac{6\mu L }{1}PCR \; purified \; GFP * \frac{50 ng}{1 \mu L} * \frac{1}{20\mu L} = \frac{300 ng}{20 \mu L} PCR \; purified \: GFP = 15\frac{ng}{\mu L}PCR \; purified \; GFP\)
The iGEM kit provides a linearized plasmid backbone in a solution with a concentration of 25 ng/uL.
In our digestion step, we used 4 uL of the linearized plasmid backbone and 4 uL of the enzyme master mix.
So, we have:
\(\displaystyle \frac{4\mu L }{1}plasmid * \frac{25 ng}{1 \mu L} * \frac{1}{(4+4)\mu L} = \frac{100 ng}{8 \mu L} plasmid= 12.5\frac{ng}{\mu L}plasmid\)
We need to have a GFP:plasmid ratio of at least 3:1 to make sure that we have enough pieces of the gene of interest to successfully connect to the plasmid backbone.
Now, we'll use the NEBioCalculator  to get the number of moles for each (see the pictures).
So, we have:
\(\displaystyle \frac{32 . 36 \frac{fmol}{\mu L} \; GFP}{9 . 632 \frac{fmol}{\mu L} \; plasmid} = 3 . 36 \; GFP : 1 \; plasmid \gt 3 \; GFP : 1 \; plasmid\)
Therefore, we can use a ratio of  1 uL of the PCR purified GFP solution to 1 uL of the digested plasmid solution.
Procedure: Transformation of Top 10 Cells
Materials
50ul vial of top 10 cells
DNA of desired plasmid
ice block
pipette
water bath (set to 42oC)
Vial of SoC
Shaker
plate with LB + limiting factor (for this it is Chlore)
Incubation chamber set to 37oC
Procedure
thaw on ice one 50ul vial of one shot top 10 cells for each reaction you will be performing
add 2ul of plasmid DNA to the vial. mix by tapping gently. do not pipette up and down
incubate on ice for 30 minutes
heat shock the tubes for 30 seconds at 42oC
incubate cells on ice immediately after heat shocking for 1-5 minutes
off ice, add 250ul of Soc to the reaction tube. shake for 1 hr at 37oC
spread 10-50 ul of the reaction to pre-set LB+ chlor plates
incubate overnight at 37oC
Retrieved from " http://2014.igem.org/Team:Genspace/Notebook/Protocol "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
