<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Tsinghua_Modeling"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:Tsinghua/Modeling">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Tsinghua/Modeling&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Tsinghua/Modeling&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Tsinghua/Modeling&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Tsinghua/Modeling" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Tsinghua/Modeling</H1><DIV id="bodyContent"><H3 id="siteSub">From 2009.igem.org</H3><P><CENTER><TABLE class="menu"><TBODY><TR><TD class="cell"><A class="menu" href="https://2009.igem.org/Team:Tsinghua">Home</A></TD><TD class="cell"><A class="menu" href="https://2009.igem.org/Team:Tsinghua/Team">Team</A></TD><TD class="cell"><A class="menu" href="https://2009.igem.org/Team:Tsinghua/Project">Project</A></TD><TD class="cell"><A class="menu" href="https://2009.igem.org/Team:Tsinghua/Modeling">Modeling</A></TD><TD class="cell"><A class="menu" href="http://partsregistry.org/cgi/partsdb/pgroup.cgi?pgroup=iGEM2009&amp;group=Tsinghua">Parts</A></TD><TD class="cell"><A class="menu" href="https://2009.igem.org/Team:Tsinghua/ELSI">ELSI</A></TD></TR></TBODY></TABLE></CENTER></P><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Introduction"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Introduction</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#SynGenome_Based_GenSniper_Production_Modeling"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">SynGenome Based GenSniper Production Modeling</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Bottom-Up_Strategy"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Bottom-Up Strategy</SPAN></A></LI><LI class="toclevel-3 tocsection-4"><A href="#Diagram_of_dynamic_process"><SPAN class="tocnumber">2.1.1</SPAN><SPAN class="toctext">Diagram of dynamic process</SPAN></A></LI><LI class="toclevel-3 tocsection-5"><A href="#Results"><SPAN class="tocnumber">2.1.2</SPAN><SPAN class="toctext">Results</SPAN></A></LI><LI class="toclevel-3 tocsection-6"><A href="#Paratmeters"><SPAN class="tocnumber">2.1.3</SPAN><SPAN class="toctext">Paratmeters</SPAN></A></LI></UL><LI class="toclevel-2 tocsection-7"><A href="#Top-Down_Strategy"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">Top-Down Strategy</SPAN></A></LI><LI class="toclevel-3 tocsection-8"><A href="#Reaction_kinetics"><SPAN class="tocnumber">2.2.1</SPAN><SPAN class="toctext">Reaction kinetics</SPAN></A></LI><LI class="toclevel-3 tocsection-9"><A href="#Parameters"><SPAN class="tocnumber">2.2.2</SPAN><SPAN class="toctext">Parameters</SPAN></A></LI><LI class="toclevel-3 tocsection-10"><A href="#Results_2"><SPAN class="tocnumber">2.2.3</SPAN><SPAN class="toctext">Results</SPAN></A></LI><LI class="toclevel-1 tocsection-11"><A href="#Plasmid_GenSniper_Production_Modeling"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Plasmid GenSniper Production Modeling</SPAN></A></LI><LI class="toclevel-2 tocsection-12"><A href="#Dimmer_formation_and_dissociation_between_.CE.BB_rep-fiber"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">Dimmer formation and dissociation between λ rep-fiber</SPAN></A></LI><LI class="toclevel-2 tocsection-13"><A href="#DNA_protein_complex.E2.80.99s_binding_and_dissociation_is_a_dynamical_process"><SPAN class="tocnumber">3.2</SPAN><SPAN class="toctext">DNA protein complex’s binding and dissociation is a dynamical process</SPAN></A></LI><LI class="toclevel-2 tocsection-14"><A href="#Replication_of_plasmid"><SPAN class="tocnumber">3.3</SPAN><SPAN class="toctext">Replication of plasmid</SPAN></A></LI><LI class="toclevel-2 tocsection-15"><A href="#Plasmid-protein_complex_affect_plasmid_replication"><SPAN class="tocnumber">3.4</SPAN><SPAN class="toctext">Plasmid-protein complex affect plasmid replication</SPAN></A></LI><LI class="toclevel-2 tocsection-16"><A href="#Numerical_simulation"><SPAN class="tocnumber">3.5</SPAN><SPAN class="toctext">Numerical simulation</SPAN></A></LI><LI class="toclevel-2 tocsection-17"><A href="#Parameter_dependency"><SPAN class="tocnumber">3.6</SPAN><SPAN class="toctext">Parameter dependency</SPAN></A></LI><LI class="toclevel-2 tocsection-18"><A href="#Conclusions"><SPAN class="tocnumber">3.7</SPAN><SPAN class="toctext">Conclusions</SPAN></A></LI><LI class="toclevel-1 tocsection-19"><A href="#Molecular_Docking_for_Targeted_Biobrick"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Molecular Docking for Targeted Biobrick</SPAN></A></LI><LI class="toclevel-1 tocsection-20"><A href="#Outline_of_SynGenome_Based_GenSniper_Intercellular_Trafficking_Modeling"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Outline of SynGenome Based GenSniper Intercellular Trafficking Modeling</SPAN></A></LI><LI class="toclevel-1 tocsection-21"><A href="#References"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">References</SPAN></A></LI></TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="Introduction">Introduction</SPAN></H1><P>Our wetlab design is subdivided into three interrelated modules (<B>in parallel</B>) according to the BioBrick principle. In contrast, our drylab design is <B>in line</B>. Three subdividions are proposed, with one of them just outlined due to lack of experimental evidence.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:554px;"><DIV class="thumbcaption">Modeling Scheme</DIV></DIV></DIV></DIV><H1><SPAN class="mw-headline" id="SynGenome_Based_GenSniper_Production_Modeling">SynGenome Based GenSniper Production Modeling</SPAN></H1><P>In this part, we will use mathematical methods(by solve a set of ODEs with proper parameters) to predict the amount of GenSnipers we can get in a given time(the zero point for time is set as the time when IPTG is added in). This model can inform us whether our system is efficient in industry production.
</P><H2><SPAN class="mw-headline" id="Bottom-Up_Strategy">Bottom-Up Strategy</SPAN></H2><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:802px;"><DIV class="thumbcaption">Bottom-Up approach</DIV></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Diagram_of_dynamic_process">Diagram of dynamic process</SPAN></H3><P>This figure is generated by Sim biology in matlab as a diagram of production modeling in phage vectors production
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption">Production modeling in vector production</DIV></DIV></DIV></DIV><P>In a simple model, the assembly of proteins are considered negligible compared with the time of transcription and translation of these proteins precursors. The amount of product is the smaller one of the amount of package proteins of phage and the amount of targeted DNA ( to be packaged by phage) per bacterial cell.
</P><H3><SPAN class="mw-headline" id="Results">Results</SPAN></H3><P>Beginning with 50 plasmid per cell, the protein amount comes into equilibrium between synthesis and degradation about 10 hours later. The steady state amount of package proteins can reach the amount of about 10 in the orders of 7, given the parameters as showed in the figure, which means it is enough for packaging target DNAs.
</P><P>Since there are 420 major protein D and 420 major protein E on the surface of bacteriophage lambda viroin, the number of the theoretically packaged GenSniper viroin is 5.0 in the orders of 4. The wildtype bacteriophage lambda requires a lytic cycle of 40-45 min with a production rate of 100 viroins per cell. In contrast, the production model suggests that the yield of our synthesized gene therapy vector overweighs the wildtype lambda viroin production by 2 orders, while the lytic cycle seems to be longer and less violent. This point can be reflected in our wet lab work, where the actual IPTG induction time was about 10h.
</P><P>Moreover, the accepted high efficiency gene therapy vector production system requires a titer of 1×10^10-1×10^12 genome/ml. Assuming that OD600≈2 means about 10^8 cells per ml in the liquid culture, and that very viroin should contain one genome, our Bottom-Up project can give rise to a titer of 5.0×10^12 Therapeutic DNA/ml, which meets the criteria for industrialization.
</P><H3><SPAN class="mw-headline" id="Paratmeters">Paratmeters</SPAN></H3><P>Protein tranaslation rate:       r_Translation	0.0833/second
</P><P>T7promoter transcription rate:   T7_trsc	 5.0/second
</P><P>protein degradation rate:        d_protein	1.93E-4/second
</P><P>mRNA degradation rate:           d_mRNA	        0.00578/second
</P><H2><SPAN class="mw-headline" id="Top-Down_Strategy">Top-Down Strategy</SPAN></H2><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:802px;"><DIV class="thumbcaption">Top-Down approach</DIV></DIV></DIV></DIV><P>Q protein is a native λ phage protein which regulates λ phage's late gene transcription. In our system, Q protein is transcripted under regulation of T7 promomter then regulates the transcription of λ phage package protein through its anti-termination property to R'promoter. In this modeling section, we'll predict how much package proteins we will get using Pr'-Q regulatioin system.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Top-Down model outline</DIV></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Reaction_kinetics">Reaction kinetics</SPAN></H3><P>the production of Q protein
</P><P>the production of target package proteins when R' promoter is fully activated 
</P><P>Q protein's influence on R' promoter activity
</P><P>As indicated in Ref(13), the Hill coefficient fitted by experiment is 6.4-3.3~6.4+3.3, which means that Q protein is very efficient in antitermination. As we don't know exactly the other parameters in this function, so we can not mimic Q protein's regulation effect for R' promoter dynamically, instead we suppose when 90% percent of Q protein have been reached, the R' promoter is fully activated
</P><H3><SPAN class="mw-headline" id="Parameters">Parameters</SPAN></H3><P>T7 promoter transcription rate <B>k</B><SUB>t7</SUB> = 5 molecule/second  
</P><P>R' promoter transcription rate <B>k</B><SUB>pr'</SUB>= 20 molecule/second (estimated according to the fact that it is faster than T7)   
</P><P>mRNA degradation rate <B>d</B><SUB>m</SUB> = 0.05 molecule/second
</P><P>protein translation rate <B>k</B><SUB>q</SUB>  = 0.1 molecule/second
</P><P>protein degradation rate <B>d</B><SUB>Q</SUB>  = 0.003 molecule/second 
</P><H3><SPAN class="mw-headline" id="Results_2">Results</SPAN></H3><P>This is the Q protein level changing with time. When time= 1000s , 90% Q protein versus maximum amount haven been produced. At this time point, we assume that R' promoter is fully activated. 
</P><P>This figure is the Q protein level versus time, notice the zero point of time axis is set to the time point that R' promoter is fully activated.
</P><P>Defining the time of adding IPTG as t=0, the simulated Q concentration increases with the structral proteins. When t≈2000s, the desired proteins reach a maximum and their growth rate decrease, when the lytic apparatus is designated to open. The entire lytic cycle lasts 35 min, which is much shorter than the Bottom-Up strategy and in the same order with the wildtype bacteriophage lambda production.
</P><P>This modeling result matches our project design, where Top-Down approach uses the same transcription strategy (promoter R') as the wildtype lambda genome and Bottom-Up approach uses a engineered one (promoter T7). 
</P><P>At the steady state, the concentration of the structral proteins reach at 1.3×10^4. Assuming that there are 420 D or E major structral protein on the surface of one GenSniper viroin, the packaged viroin number is 3.0×10^1 per cell, which is near to the wildtype bacteriophage lambda packaging. When the density of the cells is 10 in the orders of 9, the titer of synthetic GenSniper will be 3.0×10^10 Therapeutic DNA/ml, which also meets the standard for industrialization.
</P><P>Coupled with our wetlab observation, we outlined a production procedure regarding both Bottome-Up and Top-Down GenSniper Genome.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:802px;"><DIV class="thumbcaption">Production Scheme of GenSniper</DIV></DIV></DIV></DIV><H1><SPAN class="mw-headline" id="Plasmid_GenSniper_Production_Modeling">Plasmid GenSniper Production Modeling</SPAN></H1><P>One of our subproject is the use nakes plasmid bound with a specified protein as the vector of gene therapy DNA. The normal replication process of plasmid will be disrupted due to the binding of specified protein(in our model,lambda repressor). If the protein has a high affinity with the plasmid we will have more fraction of plasmid protein complex however with less plasmid substrates. Considering production of the plasmid protein complex, we want to maxize the amounts of plasmid protein complex, so a balance between the amounts of plasmid and protein is needed. In this model, we want to find out the optimized parameters for this plasmid vector production process. 
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption">Plasmid GenSniper project design</DIV></DIV></DIV></DIV><H2><SPAN class="mw-headline" id="Dimmer_formation_and_dissociation_between_.CE.BB_rep-fiber"> Dimmer formation and dissociation between λ rep-fiber </SPAN></H2><P>parameters:
</P><P><B>k</B><SUB>don</SUB> =0.05/(M*sec) [1,6]
</P><P><B>k</B><SUB>doff</SUB> =0.5/sec
</P><H2><SPAN class="mw-headline" id="DNA_protein_complex.E2.80.99s_binding_and_dissociation_is_a_dynamical_process">DNA protein complex’s binding and dissociation is a dynamical process </SPAN></H2><P>
parameters:
</P><P><B>k</B><SUB>on</SUB> =0.021/(mole*sec) binding rate of dimers to DNA
</P><P><B>k</B><SUB>off1</SUB> =0.04/sec  unbinding rate of cI dimer from Or1
</P><P><B>k</B><SUB>off1</SUB> =1.026/sec unbinding rate of cI dimer from Or2
</P><P><B>k</B><SUB>off1</SUB> =5.197/sec unbinding rate of cI dimer from Or3
</P><P>Notes:we get these values from [2]. Cao.<I>etc</I> estimate these values 
from experimentally measured protein diffusion coefficient in <I>e.coli</I> and
Gibbs free energy. We give all the three unbinding rate of cI dimer from three kinds of binding sites to find out the best site we want to use in the experiment.
</P><H2><SPAN class="mw-headline" id="Replication_of_plasmid">Replication of plasmid</SPAN></H2><P>plasmid replication is a complex process, here we use decay rate replication as a simplified model to mimic negative regulation. Three general classes of regulatory mechanisms have been studied in depth, namely those that involve directly repeated sequences (iterons), those that use only antisense RNAs and those that use a mechanism involving an antisense RNA in combination with a protein. For more detailed info about <I>Control of plasmid replication and copy-number</I>: <A href="http://www.sci.sdsu.edu/~smaloy/MicrobialGenetics/topics/plasmids/plasmid-repln.html" class="external autonumber" rel="nofollow">[1]</A> . An detailed mathematical model is described in (12)
</P><P>Suppose in the steady sate, the total number of plasmid per cell is a, and we will get
</P><P>We set a = 20 as usually the case in experiments, and λ = 0.001 (so that the time to get to steady state is 1000) and r = 15000 so that the initial amount of plasmid per cell is 5.
</P><P>Here's the figure of the number of plasmid per cell versus time without protein binding module.
</P><P>
The transcription of λ rep-fiber, notice that λ rep-fiber is coded in the plasmid, so the copy number of plasmid will affect the amount of λ rep-fiber.
</P><P>parameters: 
</P><P><B>υ</B><SUB>m</SUB>  = 0.033/s 
</P><P><B>k</B><SUB>dm</SUB> = 0.006/s  
</P><P>Notes: 
The rate limiting step in ranscription including transcription initiation and elongation. The rate of transcript initiation reaction is 9.46*10^5 for cI (8,9). Average elongaration rate is 30 nt/sec. λ repressor is about 720bp, so <B>υ</B><SUB>m</SUB>  = 24/720=0.033/s . The half lifetime of mRNA in <I>e.coli</I> is 2 min(7), k<SUB>dm</SUB>=ln2/2*60=0.006/s   
</P><P>The translation of λ repressor
</P><P>
parameters:
</P><P><B>υ</B><SUB>p</SUB>  = 0.0288/sec
</P><P><B>k</B><SUB>d_cI_w</SUB> = 0.0002/s   (widetype)
</P><P><B>k</B><SUB>d_cI_tag</SUB> = 0.003/s (with degradation tag) 
</P><P>Notes: An average ten copies of proteins are assumed to be produced per transcript. Given the half lifetime of mRNA of cI to be 4 min,  <B>υ</B><SUB>p</SUB>  = 10/(4*60/ln2)=0.0288/sec
The half-life of  widetype λ repressor is 60 mins (10), however, with degradation tag, it could be as short as 4 min (11).
</P><H2><SPAN class="mw-headline" id="Plasmid-protein_complex_affect_plasmid_replication">Plasmid-protein complex affect plasmid replication</SPAN></H2><P>If a replication is already initiated but when the replication forks meet a lamda protein on the way, this replication will be aborted. So only when replication forks pass the protein binding point without lamda protein there, the replication processes are effective. This is a simplified model on how plasmid-protein complex affect plasmid replication, we will try to find out some more detailed mechanisms in the future.
The probability of a plasmid to become a protein-DNA complex is:
</P><P>Because of the DNA-protein complex, the effective replication rate of plasmid is 
</P><H2><SPAN class="mw-headline" id="Numerical_simulation"> Numerical simulation </SPAN></H2><P>According to the previous analysis, the parameters which we can manipulate in the experiment is protein binding affinity(by choosing different binding sites) and protein degradation rate(by adding degradation tag). Here is the simulation results with a parameter of k and kdp
</P><H2><SPAN class="mw-headline" id="Parameter_dependency"> Parameter dependency </SPAN></H2><P>final product plasmid_protein complex we got by using different protein_binding sites and protein degradation rates:
</P><P>We can see that higher binding affinity favors the final production of plasmid vectors(plasmid-protein complex). And the influence of protein degration rate is a little ambiguous: fast degradation may either favor or adverse to the production of plasmid vectors (compare when k= 0.021/1.626 and 0.021/5.197)according to the protein binding affinity. After all, both of the influence of protein binding affinity and protein degradation rate is minor and the binding of protein doesn't change the nature replication process much.
</P><H2><SPAN class="mw-headline" id="Conclusions"> Conclusions </SPAN></H2><P>According to the analysis in the modeling section, we find that different parameters such as protein binding affinity and protein degradation rate won't affect the amount of the target protein significantly in the range of experimental values. However, it does give us the clue that protein with higher affinity and moderate degradation rate is suitable for large scale industry production.
</P><H1><SPAN class="mw-headline" id="Molecular_Docking_for_Targeted_Biobrick">Molecular Docking for Targeted Biobrick</SPAN></H1><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption">Integrin structure</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:702px;"><DIV class="thumbcaption">Fiber structure</DIV></DIV></DIV></DIV><P>We used ZDOCK software to conduct the molecular docking day experiment. In the generated docking graph, the ligand (we used the crystalized fiber protein to substitute the targeted biobrick) and the receptor (integrin) are both in a stretched confirmation in which the spacial clash is relatively small. Moreover, in the fiber confirmation, the positions for both RGD domain and cell-specific peptide are exposed to the surface to faciliate the specific interaction with the cell specific receptor and integrin.
</P><H1><SPAN class="mw-headline" id="Outline_of_SynGenome_Based_GenSniper_Intercellular_Trafficking_Modeling">Outline of SynGenome Based GenSniper Intercellular Trafficking Modeling</SPAN></H1><P>Since the intercellular process of GenSniper is quite complicated, we just outline a model for this process according to the established modeling on adenovirus.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:616px;"><DIV class="thumbcaption">Intercellular Trafficking Mechanism of GenSniper</DIV></DIV></DIV></DIV><P>According to the adenovirus trafficking model, GenSniper enters the cell from the top. In the nuclear region (r , RN), it can only diffuse. In the cytoplasmic region (RN , r , RC), GenSniper intermittently switches between diffusion and motor-assisted transport. As an approximation, we can assume uniform concentration of endosomal and cytosolic GenSnipers along the vertical direction.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption">Schematic representation of a model cell for GenSniper trafficking</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption">Transition map of GenSniper trafficking</DIV></DIV></DIV></DIV><P>As for this model, it is of note that the nuclear localization signal has not been incorporated into GenSniper. Thus, the trafficking process is hard to predict. Since naked plasmid can also work even without such signals and protein capsid, the design of GenSniper without such signals seems to be OK. In addition, as for a newly proposed approach, this design is recommended for a well-defined wetlab system and a simplified modeling. As for practical use, the NLS is highly recommended to be added by phage display on E of lambda head.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:702px;"><DIV class="thumbcaption">GenSniper Intracellular trafficking model can be ensured by display NLS sequence on E</DIV></DIV></DIV></DIV><H1><SPAN class="mw-headline" id="References">References</SPAN></H1><P>[1]	Burz, D.S., et al., <I>Self-assembly of bacteriophage lambda cI repressor: effects of single-site mutations on the monomer-dimer equilibrium.</I> Biochemistry, 1994. <B>33</B>(28): p. 8399-405.
</P><P>[2]	Cao, Y., H.M. Lu, and J. Liang, <I>Stochastic probability landscape model for switching efficiency, robustness, and differential threshold for induction of genetic circuit in phage lambda.</I> Conf Proc IEEE Eng Med Biol Soc, 2008. <B>2008</B>: p. 611-4.
</P><P>[3]	Elowitz, M.B. and S. Leibler, <I>A synthetic oscillatory network of transcriptional regulators.</I> Nature, 2000. <B>403</B>(6767): p. 335-8.
</P><P>[4]	Koblan, K.S. and G.K. Ackers, <I>Cooperative protein-DNA interactions: effects of KCl on lambda cI binding to OR.</I> Biochemistry, 1991. <B>30</B>(31): p. 7822-7.
</P><P>[5]	Merabet, E. and G.K. Ackers, <I>Calorimetric analysis of lambda cI repressor binding to DNA operator sites.</I> Biochemistry, 1995. <B>34</B>(27): p. 8554-63.
</P><P>[6]	Shea, M.A. and G.K. Ackers, <I>The OR control system of bacteriophage lambda. A physical-chemical model for gene regulation.</I> J Mol Biol, 1985. <B>181</B>(2): p. 211-30.
</P><P>[7]     Kushner, S. R. in Escherichia Coli and Salmonella: Cellular and Molecular Biology (ed. Neidhardt, F. C.) (ASM, Washington DC, 1996).
</P><P>[8]    Li M, McClure W, <I>Susskind M. Changing the mechanism of transcriptional activation by phage lambda repressor.</I> Proc Natl Acad Sci U S A 1997;<B>94</B>(8):3691–3696. [PubMed: 9108039]
</P><P>[9]   Hawley D, McClure W. <I>In vitro comparison of initiation properties of bacteriophage lambda wildtype pr and x3 mutant promoters.</I> Proc Natl Acad Sci U S A 1980;<B>77</B>(11):6381–6385. [PubMed:
6450417]
</P><P>[10]  Keiler, K. C., Waller, P. R. &amp; Sauer, R. T. <I>Role of a peptide tagging system in degradation of proteins synthesized from damaged messenger RNA.</I> Science 271, 990±993 (1996).
</P><P>[11]  Andersen, J. B. et al.<I> New unstable variants of green ¯uorescent protein for studies of transient gene expression in bacteria.</I> Appl. Environ. Microbiol. 64, 2240±2246 (1998).
</P><P>[12]  Brendel, V. and A.S. Perelson, <I>Quantitative model of ColE1 plasmid copy number control.</I> J Mol Biol, 1993. 229(4): p. 860-72.
</P><P>[13]  Kobiler, O., et al., <I>Quantitative kinetic analysis of the bacteriophage lambda genetic network.</I> Proc Natl Acad Sci U S A, 2005. 102(12): p. 4470-5.
</P><P>[14] Anh-Tuan Dinh, Theo Theofanous, and Samir Mitragotri. A Model for Intracellular Trafﬁcking of Adenoviral Vectors. Biophysical Journal, 2005. 89: 1574–1588.
</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2009.igem.org/Team:Tsinghua/Modeling">http://2009.igem.org/Team:Tsinghua/Modeling</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Tsinghua/Modeling" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Tsinghua/Modeling" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Tsinghua/Modeling&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Tsinghua/Modeling&amp;oldid=160329" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2009.igem.org:Privacy_policy" title="2009.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2009.igem.org:General_disclaimer" title="2009.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>