"
Page
Discussion
View source
History
teams
Log in
 
Team:Tsinghua/Result1
From 2009.igem.org
Home
Background
Brainstorming
Design
Experiment
Results
Conclusion
Protocol
Contents
1 Synthesis of the Genome of GenSniper
1.1 Bottom-Up Approach
1.2 Top-Down Approach
2 GenSniper Production Testing
Synthesis of the Genome of GenSniper
As indicated in our project description, the Bottom-Up and Top-Down approaches are paralell and the accomplishment of one of them is sufficient for further characterization.
Bottom-Up Approach
Map of GenSniper genome synthesized by Bottom-Up approach
Showed above are the maps of the GenSniper using bottom-up approach. The genes of Nu1, A, W and B are incorporated into pET-28a, while C, D, E, FI, FII and lysis genes (S, R, Rz) are synthesized into pACYCDuet-1. In other word, the GenSniper will have two linkage groups.
PCR amplification of lmbda stuctural genes. A. Nu1+A、W+B、C；B. D+E+FI+FII
We have successfully cloned the Nu1-A, W-B, C and D-E-FI-FII gene segments from the bacteriophage lambda genome.
Identification of pET-28a+Nu1+A+W+B. A.Plasmid Gel Electrophoresis; B.PCR identification, we amplified Nu1+A and W+B respectively
[[Image:j23.png|400px|center|j23|thumb|Identification of pACYCDuet-1+C+D+E+FI+FII. A.Plasmid Gel Electrophoresis; B.PCR identification, we amplified C+D+E+FI+FII, three out of four are positive clones, non-specific bands are due to low annealing temperature]
Positive clones of the two linkage groups of the GenSniper genome have been achieved (termed pACYCDuet-1+C+D+E+FI+FII and pET-28a+Nu1+A+W+B). After PCR of the purified recombinant plasmids, we can get the desired bands on the gels.
PCR amplification of S+R+Rz
Identification of pACYCDuet-1+S+R+RZ+C+D+E+FI+FII. A.Plasmid Gel Electrophoresis; B.Enzyme Digestion Identification
After constructing the GenSniper genome with the structral proteins we need, we further incorporated the lysis genes (S+R+Rz) into pACYCDuet-1+C+D+E+FI+FII. Enzyme digestion identification confirmed that the cloning of pACYCDuet-1+S+R+Rz+C+D+E+FI+FII is correct.
result of co-transformation. A.co-transformation of pET-28a+Nu1+A+W+B and pACYCDuet-1+C+D+E+FI+FII; B.co-transformation of pET-28a+Nu1+A+W+B and pACYCDuet-1+S+R+RZ+C+D+E+FI+FII
After constructing all the desired clones, we had to testing their compatibility. A shows the plate after cotransformation of pET-28a+Nu1+A+W+B and pACYCDuet-1+C+D+E+FI+FII. B shows plate after cotransformation of pET-28a+Nu1+A+W+B and pACYCDuet-1+S+R+Rz+C+D+E+FI+FII. The result clearly indicates that they can work together, which provides the basis for the functioning of the proteins encoded by the GenSniper genome via Bottom-Up synthesis.
In summary, we have completed all the objectives in this section. The synthesized GenSniper genome will be functionally identified in the following experiments.
Top-Down Approach
Map of GenSniper genome synthesized by Top-Down approach
The GenSniper genome synthesized by Top-Down approach only needs one plasmid. However, large DNA segment will be required for ligation, which will add difficulties to our experiment. The above picture is the map of the GenSniper genome with only one linkage group.
Synthesis of the genes in the Top-Down synthesized GenSniper genome. A.Enzyme digested PR'-C1 segment; B. Plasmid gel electrophoresis after incorporation of PR'-C1 segment; C. PCR amplification of C2-FII segment
The large segment from Promoter R' to part of C in bacteriophage lambda (termed PR' to C1) is generated after circulization (the lambda genome we used are linear DNA of 48502 bp) and restriction enzyme digestion. The other segment needed (termed C2-FII) is generated via PCR.
After ligated PR'-C1 into pET-28a, we can see apparent loss of mobility on the agarose gel. However, PCR identification of part of segments on the PR'-C1 region failed to reflect any evidence for PR'-C1 incorporation.
Synthesis of Q gene of GenSniper Genome. A.First round PCR, the theoretical length of the product is 658 bp; B. Second round PCR, the theoretical length of the product is 673 bp
Q gene on the lambda genome was also cloned for the induction of the expression of structral genes for GenSniper viroin package. We added a double direction terminator down stream of Q for the termination of transcription for both Q as well as lysis and structral genes.
In all, our Bottom-Up approach seems to have proceed faster than the Top-Down approach. Thus, we firstly used the GenSniper genome synthesized by Bottom-Up approach for further identification. Nevertheless, we reasoned that Top-Down approach do have the advantage for a larger chance of wildtype-like functioning.
GenSniper Production Testing
Effect of lysis gene expression to the density of bacteria. A number of flasks of bacteria had been culture, and two of them with the most close OD600 curves were selected for expression characterization. 2mmol/L IPTG was used for induction
In order to test the functional expression of GenSniper proteins, we firstly measured the changes in OD600 after IPTG induction for characterization lysis gene function. After transforming the BL21 DE3 E.coli strain with pACYCDuet-1+lysis+C+D+E+FI+FII, we cultured the bacteria in flasks for 8h before 2mmol/L induction.
The result shows that the OD600 of the bacteria containing pACYCDuet-1 reduced significantly, while that of the control group was still raising. This implied that the lysis genes might be functionally correct. Subsequently, we transformed the entire GenSniper genome into BL21 DE3 strain with the similar induction procedure. Considering the possible delayed growth after double plasmid transformation, we added IPTG after 10h culture. The bacteria containing the GenSniper genome still showed reduced OD600, while the OD600 of the control group raised over time.
Characterization of lysis gene expression in the entire GenSniper genome. Before IPTG induction, we equally divided the culture media into two new flasks with the IPTG+ group added 40μl IPTG to 20 ml media (2mmol/L IPTG)
Effect of IPTG on bacteria growth
To rule out the possibility that IPTG may have affected the growth of bacteria, we further use the GenSniper genome without lysis genes to cotranformed the cells as control group. The results show that IPTG, as described in molecular cloning protocols, will inhibit bacterial growth, but this effect cannot account for the OD600 reduction in the GenSniper genome containing bacteria after IPTG induction.
Thus, we went on to the mini-production of the GenSniper by 20ml flask. Before identification by electronic microscopy, we measured the protein concentration of the purified viroins, which indicates there are proteins in the purified viroin solutions (weed proteins will be ruled out according to the protocol, however this result is not sufficient to prove the presence of the GenSniper viroin).
Protein concentration measurement of GenSniper viroin solution. Whether or not the protein purification process contains sonication seems to be same
The electronic spectroscopy picture confirmed that the desired viroins, namely the bacteriophage lambda viroin without a tail, were generated. We also tested a control group where the empty vectors pET-28a and pACYCDuet-1 were introduced into BL21 DE3 strain.
When we further co-tranformed the GenSniper genome and Therapeutic DNA for GenSniper viroin production with the packaged DNA, the purified solution also contains the desired protein complex. In addition, PCR identification of cos site indicates positive result. These experiments will be introduced in detail in the results of Module II.
Electronic spectroscopy picture of the GenSniper viroin without Therapeutic DNA package and Targeted Biobrick modification
Electronic spectroscopy picture of the GenSniper viroin with Therapeutic DNA package and without Targeted Biobrick modification
Electronic spectroscopy picture of the control group
In conclusion, we have built and tested the GenSniper genome function and generated the desired protein complex. Frankly speaking, we have to admit that the identification of the function of these protein complex should be accomplished after modification with Targeted Biobrick. We think this work will be hard considering the time limit of iGEM09, so we will mainly suggest the significance of our project via the three modules, which also accords with the modulization principle in synthetic biology.
Retrieved from " http://2009.igem.org/Team:Tsinghua/Result1 "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
