<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_BNU-China_Chemotaxis_html"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:BNU-China/Chemotaxis.html">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:BNU-China/Chemotaxis.html&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:BNU-China/Chemotaxis.html&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:BNU-China/Chemotaxis.html&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:BNU-China/Chemotaxis.html" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:BNU-China/Chemotaxis.html</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><DIV id="thumbs"><A href="https://static.igem.org/mediawiki/2014/c/c9/Bnu_bg4.jpg">1</A><A href="https://static.igem.org/mediawiki/2014/2/2f/Bnu_bg5.jpg">2</A></DIV><DIV id="wrapper"><DIV id="menu" class="menu opacity"><UL><LI><A href="https://2014.igem.org/Team:BNU-China/home.html">Home</A></LI><LI><A href="#">Project</A></LI><LI><A href="https://2014.igem.org/Team:BNU-China/overview.html">Overview</A></LI><LI><A href="https://2014.igem.org/Team:BNU-China/ModA&amp;INPN.html">ModA+INPN</A></LI><LI><A href="https://2014.igem.org/Team:BNU-China/Chemotaxis.html">Delivery System</A></LI><LI><A href="https://2014.igem.org/Team:BNU-China/modeling.html">Modeling</A></LI><LI class="maps"><A href="https://2014.igem.org/Team:BNU-China/maps.html">       Maps</A></LI></UL><LI><A href="#">Achievements</A></LI><LI><A href="https://2014.igem.org/Team:BNU-China/parts.html">Submitted Parts</A></LI><LI><A href="https://2014.igem.org/Team:BNU-China/judging.html">Judging Criteria</A></LI><LI><A href="https://2014.igem.org/Team:BNU-China/future.html">Bright Future</A></LI><LI><A href="#">Safety</A></LI><LI><A href="https://2014.igem.org/Team:BNU-China/safety-overview">Overview</A></LI><LI><A href="https://2014.igem.org/Team:BNU-China/labenv">Lab &amp; Env safety</A></LI><LI><A href="https://2014.igem.org/Team:BNU-China/kill">Kill Switch</A></LI><LI><A href="#">Policy &amp; Practice</A></LI><LI><A href="https://2014.igem.org/Team:BNU-China/pp_overview.html">Overview</A></LI><LI><A href="https://2014.igem.org/Team:BNU-China/pubad.html">Public Promotion</A></LI><LI><A href="https://2014.igem.org/Team:BNU-China/3e.html" title="Environment, Efficiency and Economic">Three &quot;E&quot;s</A></LI><LI><A href="https://2014.igem.org/Team:BNU-China/3c.html" title="Collaboration, Conference and Communication">Three &quot;C&quot;s</A></LI><LI><A href="https://2014.igem.org/Team:BNU-China/edu.html">Edu-justice</A></LI><LI><A href="https://2014.igem.org/Team:BNU-China/chinese-biopolicy.html">Bio-policy</A></LI><LI><A href="#">Our Team</A></LI><LI><A href="https://2014.igem.org/Team:BNU-China/profile.html">Team Profile</A></LI><LI><A href="https://2014.igem.org/Team:BNU-China/attributions.html">Attributions</A></LI><LI><A href="https://2014.igem.org/Team:BNU-China/time_axis.html">Our Notebook</A></LI><LI><A href="https://2014.igem.org/Team:BNU-China/collaboration.html">Collaboration</A></LI><LI><A href="https://2014.igem.org/Team:BNU-China/acknowledge.html">Acknowledge</A></LI></DIV><DIV id="container" class="opacity"><H1 align="center">Delivery System</H1><H2>Overview</H2><P>The plants root exudates contain TCA intermediates that can attract bacteria having the ability of chemotaxis.  Click <A href="#rootexudates">here</A> to learn more about root exudates. <I>E.coli</I> has five kinds of chemoreceptors, which interact with factors of the flagella that leads to <A href="#chemo">chemotaxis</A>. But <I>E.coli</I> doesn’t have specific chemotaxis towards some TCA intermediates while <I>Pseudomonas putida</I> has some McpS, like McfQ and McfR. We made a part <A href="#top">BBa_K1405004</A> containing the sequence of McfR, which detects succinate, malate and fumarate. Then we detected its chemotaxis towards malate and succinate, you can download our <A href="https://static.igem.org/mediawiki/2014/b/b7/Bnu_Protocol_on_Delivery_System.pdf" target="_blank">protocol</A> which is based on the work <A href="https://2011.igem.org/Team:Imperial_College_London/Achievements" target="_blank">2011_Imperial_College_London</A> had done before.click <A href="#results">here</A> to see the results. At last, we designed a <A href="https://2014.igem.org/Team:BNU-China/modeling.html">model</A> to mimic the movement pattern and predict the efficiency of the Prometh<I>E.coli Prometheus</I>.</P><A href="#top"><P class="fig" style="margin-left:790px">Back to Top</P></A><H2 id="rootexudates">Root exudates</H2><P>Plants secrete both high- and low-molecular weight compounds from their roots, and these root exudates play an important role not only as nutrients for soil microbes but also as signal molecules in plant–microbe interactions <A href="#reff">(Sugiyama &amp; Yazaki)</A>. In the wild, legume plants establish symbiotic interactions with rhizobia and arbuscular mycorrhizal fungi to obtain several nutrients such as nitrogen and phosphate. The root exudates of legume plants contain organic acids, including some TCA intermediates, such as malate, succinate, citrate and fumarate. Especially, in response to phosphorus starvation, malonie, succinic, fumaric, malic, citric, and t-aconitic acids were detected in the root exudates<A href="#reff">(Ohwaki &amp; Hirata, 1992)</A>. Here, we design the <I>E.coli Prometheus</I> that responses to malate and succinate and will swim to the roots to improve the interaction between the roots and rhizobium. </P><A href="#top"><P class="fig" style="margin-left:790px">Back to Top</P></A><H2 id="chemo">Chemoreceptor and Chemotaxis</H2><P>In isotropic chemical environments, <I>E. coli</I> swims in a random - walking pattern produced by alternating episodes of counter-clockwise (CCW) and clockwise (CW) flagellar rotation (Fig. 1, left panel).  While an attractant or repellent appear somewhere in the environment, the cells suppress tumbling and their locomotor responses specific runs that take the cells in favorable directions (toward attractants and away from repellents), resulting in net movement towards preferred environments.  (Fig. 1, right panel). </P><P class="fig">Fig. 1  Random and biased walks. Left: A random walk in isotropic environments. When the cell's motors rotate CCW, the flagellar filaments push the cell forward. When one or more of the flagellar motors reverses to CW rotation, that filament undergoes a shape change (owing to the torque reversal) that disrupts the bundle. Until all motors once again turn in the CCW direction, the filaments act independently to push and pull the cell in a chaotic tumbling motion. Tumbling episodes  enable the cell to try new, randomly-determined swimming directions. Right: A biased walk in a chemoeffector gradient. As cells swim up a concentration gradient of attractant, they spend more time smooth swimming than tumbling; this modulation of swimming behavior is manifested as a chemotactic response.</P><P class="fig">The pictures are from Parkinson Lab - University of Utah ( The website address [<A href="http://chemotaxis.biology.utah.edu/Parkinson_Lab/projects/ecolichemotaxis/ecolichemotaxis.html">http://chemotaxis.biology.utah.edu/Parkinson_Lab/projects/ecolichemotaxis/ecolichemotaxis.html</A>])</P><P><I>E. coli</I> has five chemoreceptors, four are methyl-accepting chemotaxis proteins (McpS) which have periplasmic ligand binding sites and conserved cytoplasmic signaling domains (Fig. 2).  The McpS initiate a motor control response and a feedback circuit that updates the methylation record to achieve sensory adaptation and cessation of the motor response. A fifth MCP-like protein, Aer, mediates methylation-independent aeroTCAtic responses by monitoring redox changes in the electron transport chain. The five MCP-family receptors in <I>E. coli</I> utilize a common set of cytoplasmic signaling proteins to control flagellar rotation and sensory adaptation (Fig. 2).  CheW and CheA generate receptor signals; CheY and CheZ modulate rotation of the flagellar motor and thus change cell behavior and movement; CheR and CheB regulate MCP methylation state to make feedback to CheW and CheA. </P><P class="fig">Fig. 2 Signaling components and circuit logic.  <I>E.coli</I> receptors employ a common set of cytoplasmic signaling proteins: CheW and CheA interact with receptor molecules to form stable ternary complexes that generate stimulus signals; CheY transmits those signals to the flagellar motors, CheZ controls their lifetime; CheR (methyltransferase) and CheB (methylesterase) regulate MCP methylation state.  Abbreviations: OM (outer membrane); PG (peptidoglycan layer of the cell wall); CM (cytoplasmic membrane).</P><P class="fig">The pictures are from Parkinson Lab - University of Utah ( The website address <A href="http://chemotaxis.biology.utah.edu/Parkinson_Lab/projects/ecolichemotaxis/ecolichemotaxis.html">[http://chemotaxis.biology.utah.edu/Parkinson_Lab/projects/ecolichemotaxis/ecolichemotaxis.html])</A></P><P>But the original chemoreceptors of <I>E.coli</I> do not respond to the root exudates, while the McpS of the <I>Pseudomonas putida</I> , McfQ and McfR do<A href="#reff">(Parales, et al.)</A>. McfQ responds to citrate and fumarate, and McfR detects succinate, malate and fumarate. To compare with the work what 2011_Imperial_College_London had done, we chose McfR which also responds to malate. The mechanisms of prokaryote chemoreceptors are similar, so we designed a plasmid to express chemoreceptor. McfR in the <I>E.coli</I> makes the bacteria have the ability of chemotaxis towards TCA intermediates. As our experimental condition is limited, we just did experiments on chemotaxis towards succinate and malate. What’s more, in an attractant gradient, the chemoreceptors monitor chemoeffector concentration changes as they move around. And they use that information to modulate the probability of the next tumbling event. And we searched the pattern of their chemotaxis towards a concentration gradient of attractant using capillary assay. The results are demonstrated below.</P><A href="#top"><P class="fig" style="margin-left:790px">Back to Top</P></A><H2 id="results">Results</H2><P>We did both agar assay and capillary assay to detect the response of <I>E.coli</I> to different attractants and different concentrations of each attractant. Because the agar assay (Fig.3) is difficult to replicate and collect the data from, we just show the results of capillary assay (Fig.4) . We made a negative control using washing buffer and five concentration gradients (100mM/10mM/1mM/0.01mM/0.0001mM) of attractants. These <I>E.coli</I>s were divided into three groups based on the plasmid they have been transformed into. The plasmids are biobricks, BBa_K608003 and <A href="http://parts.igem.org/Part:BBa_K515102" target="_blank">BBa_K515102</A> (they are from 5A and 8F wells in plate1), and the McfR plasmid was designed by us. <A href="http://parts.igem.org/Part:BBa_K608003" target="_blank">BBa_K608003</A> (5A) only has a strong promoter and medium RBS, so it doesn’t have specific chemotaxis towards TCA intermediates. BBa_K515102 (8F) is a biobrick from 2011_Imperial_College_London, which responds to L(-)malic acid (HO2CCH2CH(OH)CO2H).</P><DIV style="float:left; margin-left: 20px; margin-top: 40px; width: 40%; padding: 30px 20px 30px 30px;"><P class="fig">Fig.3 Agar assay: The three plates shows the chemotaxis results of 8F towards three kinds of attractants, malate, citrate and succinate from left to right. The filter paper of attractants is put on the left, the PBS is put on the right, bacteria is in the middle. Here, 8F swims fastest to malate. </P></DIV><P class="fig">Fig.4  <I>E. coli</I>’s ability of chemotaxis towards different concentrations of succinate or malate.The cells were diluted 20000 times. 5A is a control which doesn’t have chemotaxis towards malate or succinate.<STRONG>Malate</STRONG>: McfR showed the strongest respond. The tendency of curve of 8F was biphasic with maximums at attractant concentration of about 10<SUP>-2</SUP> M and 10<SUP>-5</SUP> M, while the tendency of curve of McfR was biphasic with maximums at attractant concentration of about10<SUP>-2</SUP> M and 10 <SUP>-7</SUP> M. Both of them reached minimum at attractant concentration of about 10 <SUP>-1</SUP> M, the number of cells decreased slowly with attractant concentration decreasing for 8F and McfR. There was a significant difference among test and control (p &lt; 0.05), and quantity of cells of 8F and McfR was much more than that of 5A. So both of 8F and McfR have chemotaxis towards malate. And McfR shows stronger response. <STRONG>Succinate</STRONG>: The tendencies of curves of 8F and McfR towards succinate are same. As the attractant concentration increased, the number of cells arose and reached the maximum at attractant concentration of about 10<SUP>-2</SUP> M and fell sharply with the minimum at attractant concentration of 10<SUP>-1</SUP> M. The quantities of cells of 8F and McfR were almost equivalent and did not have significant difference. But there were significant differences among 8F &amp; 5A, and McfR &amp; 5A: the number of cells of 5A are far less than 8F or McfR (p &lt; 0.05), which demonstrated that 8F and McfR have chemotaxis towards succinate and the capacity of chemotaxis of 8F and McfR towards succinate are almost equal. When compared with malate, 8F shows stronger response, while McfR shows weaker response.
</P><A href="#top"><P class="fig" style="margin-left:790px">Back to Top</P></A><H2>Analysis</H2><P>The tendency of the chemotaxis towards chemical gradient may be influenced by two factors: pH and phosphorylation state of McpS. The motor of flagella is energized by a H<SUP>＋</SUP> (or Na<SUP>＋</SUP> in some species) gradient across the cytoplasmic membrane<A href="#reff">(Schmitt, 2003)</A>, but the mechanism is not clear. Lisa M. Maurer and Elizabeth Yohannes proved that low pH (at pH 5.0) induced the expression of some flagellar and chemotaxis genes and proton export, while coinducing oxidative stress and heat shock regulons to maintain internal pH homeostasis and to prepare the cell to survive future exposure to more extreme pH conditions (below pH 5 or above pH 9 which is higher or lower than the cytoplasmic pH 7.6)<A href="#reff">(Maurer, Yohannes, Bondurant, Radmacher, &amp; Slonczewski, 2005)</A>. The pH of attractants showed below (Fig.5) is at the range of pH 1.0 – 6.4. The highest concentration of 0.1M is pH 1.0 of malate and 2.0 of succinate, which leads to weakest chemotaxis, for oxidative stress induces its protective responses more than chemotaixs genes expression; while the curves reach maximum at concentration of 0.01M because of the opposite reason. </P><P class="fig">Fig.5 The pH of the concentration gradients of malate and succinate. The pH of  malate is lower than succinate at high concentration (10<SUP>-1</SUP> and 10<SUP>-2</SUP> M), and is equal with succinate at lower concentration. Both of them remain pH 6.4 at low concentration(10<SUP>-5</SUP>, 10<SUP>-7</SUP> and 0 M).</P><P>On the other hand, the <I>E. coli</I> chemotaxis along gradients is influenced by reversible MCPs methylation and related proteins phosphorylation (Fig. 6 A). Such adaptation is achieved in <I>E. coli</I> by modulating the methylation state of the MCPs using two proteins, a constitutively active methyltransferase CheR and a methylesterase CheB (Fig.6 B), the activity of which is stimulated after phosphorylation by CheA-P. Increased methylation of the MCPs dampens the response to ligand binding, whereas decreased methylation restores their high binding affinities for their attractants(Butler, S. M., &amp; Camilli, A., et al.). </P><P class="fig">Fig. 6  The mechanism of <I>E. coli</I> chemotaxis along gradients<STRONG>A</STRONG>: Chemotaxis pathway. Reactions and components that augment CW rotation are depicted in green; those that augment CCW rotation are depicted in red. The flagellar motors of <I>E. coli</I> spin CCW by default of Che-Y.
MCPs in response to chemoeffector stimuli transfer the signals and make CheA autophosphorylated via the adapter protein CheW. Then phospho-CheA molecules serve as donors that transfer phosphoryl groups to specific aspartate residues in CheY and CheB. Phospho-CheY enhances CW rotation of the flagellar motors FliM; phospho-CheB has high MCP methylesterase activity that helps to change the CW rotation to CCW rotation, and they will be inactive because of loss of phosphoryl group through spontaneous self-hydrolysis. CheZ further enhances the dephosphorylation rate of phospho-CheY to ensure rapid locomotor responses to changes in the supply of signaling phosphoryl groups to CheY.  
 <STRONG>B</STRONG>: At low attractant concentrations, or when a repellant is bound, most of the MCPs are in the CW signaling state. Under these conditions Che A is activated, and Che B removes most of the methyl groups. In this state the McpS have high affinities for the attractants and are able to detect very low concentrations of these effectors. If the attractant concentration increases, the MCPs tend to adopt the CCW signaling conformation, Che A and Che B thereby lose their activities, and methyl group removal becomes progressively less efficient.With the attractant concentrations increase, the McpS are progressively converted into the fully methylated state with a low affinity for the attractants.
</P><P class="fig">The picture A is from Parkinson Lab - University of Utah ( The website address [http://chemotaxis.biology.utah.edu/Parkinson_Lab/projects/ecolichemotaxis/ecolichemotaxis.html]). The picture B is from Dr. John Illingworth's Lab - University of Leeds ( The website address <A href="http://www.bmb.leeds.ac.uk/illingworth/motors/flagella.htm">[http://www.bmb.leeds.ac.uk/illingworth/motors/flagella.htm])</A></P><P>Compared with the work of 2011_Imperial_College_London had done, we find that both of their biobrick BBa_K515102 and our biobrick BBa_K1405004 (originally found in P. aeruginosa and P. putida separately) have chemotaxis towards two kinds of TCA intermediates, malate and succinate (Fig.7 A). Estela Pineda-Molina, et al. reported high-resolution structures of the ligand- binding region of the cluster II McpS chemotaxis receptor (McpS-LBR) of <I>Pseudomonas putida</I> KT2440 in complex with malate and succinate (Fig.7 B&amp; C) . The root mean square deviation (rmsd) between the malate and succinate cocrystal structure is only 0.2 Å indicating that differences in the nature of the ligand do not significantly alter the protein structure (Table 1). In a sequence alignment of McpS-LBR homologs containing McpS-LBR of Pseudomonas aeruginosa, the amino acids involved in malate and succinate binding are conserved. R60 is almost fully conserved, whereas the remaining residues are conserved to between 30–80% (Estela Pineda-Molina, et al.) So both of biobrick BBa_K515102 and our biobrick BBa_K1405004 have chemotaxis towards malate, and other McpS with the conserved sequence may response to the same ligand. And if we slice the conserved sequence of different chemoreceptors responding to other TCA intermediates, we may gain a super <I>E. coli</I> Prometheus who can respond to more universal root exudates.</P><P class="fig" style="width:20%; margin-left: 270px;">Table 1. Properties of the model</P><P class="fig">Fig.7 (A) The structure of malate and succinate, and the thermodynamic parameters for the 2 compounds that were found to bind. (B and C) View of the interaction of malate (B) and succinate (C) with the McpS-LBR. Malate binds to a cavity formed by α1 and the C-terminal segment of α6, and its coordination is achieved through interactions with R60, R63, R254, and T258 from one monomer and Q65 from the other monomer. Residues R60, R63, and Q65 form part of helix α1, whereas R254 and T258 are part of α6. Succinate binds to McpS-LBR in an almost identical manner as malate. Unlike malate, succinate has no hydroxyl group at C2. Therefore, the two hydrogen bonds that are established between the malate hydroxyl group, and the protein cannot be formed in the succinate structure. The remaining interactions of succinate with R60, R63, R254, and Q65 are analogous to the malate structure where they establish six hydrogen bonds.</P><A href="#top"><P class="fig" style="margin-left:790px">Back to Top</P></A><H2 id="reff">Reference</H2><P>Maurer, L. M., Yohannes, E., Bondurant, S. S., Radmacher, M., &amp; Slonczewski, J. L. (2005). pH regulates genes for flagellar motility, catabolism, and oxidative stress in <I>Escherichia coli</I> K-12. Journal of Bacteriology, 187(1), 304-319.</P><P>Ohwaki, Yoshinari, &amp; Hirata, Hiroshi (1992). Differences in carboxylic acid exudation among P-starved leguminous crops in relation to carboxylic acid contents in plant tissues and phospholipid level in roots. Soil Science and Plant Nutrition, 38(2), 235-243.</P><P>Parales, Rebecca E, Luu, Rita A, Chen, Grischa Y, Liu, Xianxian, Wu, Victoria, Lin, Pamela, et al. <I>Pseudomonas putida</I> F1 has multiple chemoreceptors with overlapping specificity for organic acids. Microbiology, 159(Pt 6), 1086-1096.</P><P>Schmitt, R. (2003). Helix rotation model of the flagellar rotary motor. Biophysical Journal, 85(2), 843-852.</P><P>Sugiyama, Akifumi, &amp; Yazaki, Kazufumi. Root exudates of legume plants and their involvement in interactions with soil microbes Secretions and exudates in biological systems (pp. 27-48): Springer.</P><P>Pineda-Molina, E., Reyes-Darias, J. A., Lacal, J., Ramos, J. L., García-Ruiz, J. M., Gavira, J. A., &amp; Krell, T. (2012). Evidence for chemoreceptors with bimodular ligand-binding regions harboring two signal-binding sites. Proceedings of the National Academy of Sciences, 109(46), 18926-18931.</P><P>Butler, S. M., &amp; Camilli, A. (2005). Going against the grain: chemotaxis and infection in Vibrio cholerae. Nature Reviews Microbiology, 3(8), 611-620.</P><P>*You can download the protocol of this part here: <A href="https://static.igem.org/mediawiki/2014/b/b7/Bnu_Protocol_on_Delivery_System.pdf?action=raw" target="_blank">Protocol on Delivery System.pdf</A></P><A class="button blue" style="margin-left:20px; float: left;" href="https://2014.igem.org/Team:BNU-China/ModA&amp;INPN.html">Prev Page</A><A class="button green" style="margin-left:680px" href="https://2014.igem.org/Team:BNU-China/modeling.html">Next Page</A></DIV><DIV id="copyright" class="opacity"><P>The Story of <I>E.coli Prometheus</I></P><P style="font:smaller">BNU-China</P></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:BNU-China/Chemotaxis.html">http://2014.igem.org/Team:BNU-China/Chemotaxis.html</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:BNU-China/Chemotaxis.html" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:BNU-China/Chemotaxis.html" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:BNU-China/Chemotaxis.html&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:BNU-China/Chemotaxis.html&amp;oldid=389221" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>