"
Page
Discussion
View source
History
teams
Log in
 
Team:BNU-China/ModA&amp;INPN.html
From 2014.igem.org
1 2
Home
Project
Overview
ModA+INPN
Delivery System
Modeling
   
    Maps
Achievements
Submitted Parts
Judging Criteria
Bright Future
Safety
Overview
Lab & Env safety
Kill Switch
Policy & Practice
Overview
Public Promotion
Three "E"s
Three "C"s
Edu-justice
Bio-policy
Our Team
Team Profile
Attributions
Our Notebook
Collaboration
Acknowledge
ModA
Background
Nitrogen fixation is a process in which nitrogen in the atmosphere is converted into ammonium (NH4+).No matter natural and synthetic, the process is essential for all forms of life because nitrogen is required to biosynthesize basic building blocks of plants, animals and other life forms, like nucleotides and amino acids.
All biological nitrogen fixation is done by way of nitrogenase, a metal-enzymes which contain iron, molybdenum, or vanadium. The enzyme is composed of the heterotetrameric MoFe protein that is transiently associated with the homodimeric Fe protein. Therefore, Molybdenum is an indispensable chemical element to nitrogen fixation and agriculture.
According to the Liebig's law of the minimum, which states that growth is controlled not by the total amount of resources available, but by the scarcest resource, molybdenum as a trace mineral, is identified as limiting the growth of peanuts in large field of China. So molybdenum powder is commonly used as a fertilizer in agriculture. And our project aims to establish an E.coli strain that is able to capture Mo in the medium, directional deliver itself to root system and kill itself eventually.
ModA
The genes encoding a molybdate transport system (mod) have been isolated and sequenced from Escherichia coli[2–5] Four genes modABCD, organized in an operon, share strong structural similarity and sequence homology with genes encoding binding -protein dependent solute(or ABC-type) transport systems [2–5]. The first gene, modA, encodes a periplasmic binding protein. ModB and ModC have sequence homology with integral membrane and cytoplasmic ATP hydrolysis components of ABC transport systems, respectively. ModD does not show sequence similarity to any gene product in current databases and its role in transport is unclear. Two more genes, modEF, are also involved in molybdate transport. The modE gene encodes a regulatory protein that effects molybdenum-dependent repression at the modA promoter [6–8]. The exact role of modF is not yet clear [6].
The mod operon encodes a very high affinity(Km=15 nM) molybdate transport system. The low K values agree well with previous results obtained with the enteric diazotroph K. pneumoniae ([1]; Km=20 nM, J. Imperial, unpublished results). and adequately explain the high molybdenum scavenging capacity observed with most bacteria[11].
ModA, the periplasmic binding component of the mod system. had a K for molybdate of 3–5 mM. This is one of the highest substrate Kd values reported for any periplasmic binding protein [9,10] and contrasts with the low Kd values for molybdate uptake The Structure of modA[14]
How to bind molybdenum[15]
Stereo view of bound tungstate in the ModA binding site. The W atom is shown as a green sphere, all other elements are shown in ball-and-stick representation, and the residues are labeled. The blue mesh represents a 2Fo-Fc omit map contoured at a level of 2σ and shown only around the 5 atoms of the tungstate ion. Note that the tungstate and molybdate binding sites are indistinguishable at the current resolution, which reflects the similar ionic radii of the two ions[2,3]
Design
We cloned the ModA( sequence ) into the pET21a, and translated into BL21 to expressed and purified modA, detected molybdate binding to the ModA protein based on the migration of the pure protein in native polyacrylamide gels.( protocol )
Result
Figure 1 Expression and purification of ModA. Expression of ModA and its purification by His-chelating chromatography. Lane 1, molecular weight standards (kDa); lane 2, total bacterial proteins before IPTG induction; lane 3, total bacterial proteins after 0.5 mM IPTG induction; lane 4, total bacterial lysate after 0.5 mM IPTG induction; lane 5,the sediment after 16,000 rpm 30 min; lane 6 the flow through of column loading; lane 7,the flow of buffer B .lane 8 The purification of ModA by size-elution chromatography.
Figure 1 Expression and purification of ModA. Expression of ModA and its purification by His-chelating chromatography. Lane 1, molecular weight standards (kDa); lane 2, total bacterial proteins before IPTG induction; lane 3, total bacterial proteins after 0.5 mM IPTG induction; lane 4, total bacterial lysate after 0.5 mM IPTG induction; lane 5,the sediment after 16,000 rpm 30 min; lane 6 the flow through of column loading; lane 7,the flow of buffer B .lane 8 The purification of ModA by size-elution chromatography.
Figure 2 Native-PAGE to provide the molybdate lane 1, the modA(acetic buffer PH 5.0); lane 2, the modA incubate with 20mM molybdate.
Discussion
Why were there 2 bands obviously in figure 1？
The two bands were in the lane 3-lane 8, so when the modA was inducted, maybe the modA has been cut, and for the pET-21a sequence, we knew that the his-tag was in the C-terminal, and the His-modA can chelate to the Ni column, so we guessed what the modA was cut was its C-terminal or pET-21a’s T7-tag(for western), what we work next maybe clone a new modA that there were few animo acid lost to see whether there were 2 bands, and we cut or use clone to cut the T7-tag to verify.
Why were there so many bands obviously in figure 2?
We actually did not know why there were few band above to lane 1, maybe the modA were polymers in the 20mM molybdate, the band below may be the modA binding molybdate that made its charge changed and got faster than others, and because we use excessive protein, not all proteins bind to the molybdate. For the binding molybdate, we refer to this [16]fig3
Refferences
[1]J. Imperial, R.A. Ugalde, V.K. Shah, V.K. Brill, J. Bacteriol. 158 _1984. 187–194.
[2] P.E. Bishop, D.M. Jarlenski, D.R. Hetherington, Proc. Natl.Acad. Sci. U.S.A. 77 _1980. 7342–7346.
[3] K. Schneider, A. Mueller, K.-U. Johannes, E. Diemann, J. Kottmann, Anal. Biochem. 193 1991. 292–298.
[4] M. Hadi, Characterization of mod locus in Escherichia coli: sequence of the genes, regulation of expression, and possible functions of the gene products, PhD Thesis, University of California-Berkeley, 1995.
[5] S. Johann, S.M. Hinton, J. Bacteriol. 169 _1987. 1911–1916.
[6] J.A. Maupin-Furlow, J.K. Rosentel, J.H. Lee, U. Deppenmeier, R.P. Gunsalus, K.T. Shanmugam, J. Bacteriol. 177_1995. 4851–4856.
[7] H.M. Walkenhorst, S.K. Hemschemeier, R. Eichenlaub, Mi-crobiol. Res. 150 _1995. 347–361.
[8] A.M. Grunden, R.M. Ray, J.K. Rosentel, F.G. Healy, K.T. Shanmugam, J. Bacteriol. 178 _1996. 735–744.
[9] P.M. McNicholas, S.A. Rech, R.P. Gunsalus, Mol. Microbiol. 23 _1997. 515–524.
[10] C.E. Furlong, in: F.C. Neidhardt _Ed.., Escherichia coli and Salmonella typhimurium: Cellular and Molecular Biology, ASM Press, Washington, DC, 1987, pp. 768–796.
[11] R. Tam, M.H.J. Saier, Microbiol. Rev. 57 _1993. 320–346.
[12] W. Boos, J.M. Lucht, in: F.C. Neidhardt, R. Curtiss, _Eds.., Escherichia coli and Salmonella typhimurium: Cellular and Molecular Biology, 2nd edn., ASM Press, Washington, DC, 1996, pp. 1175–1209.
[13]Juan Imperial ), Margono Hadi, Nancy K. Amy Molybdate binding by ModA, the periplasmic component of the Escherichia coli mod molybdate transport system Biochimica et Biophysica Acta 1370 _1998. 337–346
[14] Hu Y, Rech S, Gunsalus R P, et al. Crystal structure of the molybdate binding protein ModA[J]. Nature Structural & Molecular Biology, 1997, 4(9): 703-707.
[15] Kaspar Hollenstein, Dominik C. Frei, and Kaspar P. Locher Structure of an ABC transporter-binding protein Complex Kaspar nature05626
[16] Properties of the periplasmic ModA molybdate-binding protein of Escherichia coli Rech, S. ; Wolin, C. ; Gunsalus, R.P. Journal of Biological Chemistry, 2 February 1996, Vol.271(5), pp.2557-2562
Back to Top
INPN
WHY INP
Ice-nucleation protein (INP), an outer membrane protein from Pseudomonas syringae, is responsible for promoting nucleation of ice at relatively high temperatures (above -50C). The proteins are localised at the outer membrane surface and can cause frost damage to many plants. It is composed of three domains structurally distinguished as the N-terminal domain (191 amino acids, 15% of the protein), which is the portion most responsible for targeting to the cell surface, the C-terminal domain (49 amino acids, 4% of the protein), and the central domain, composed of repeats comprising an 8-, 16-, and 48-residue periodicity that acts as a template for ice crystal formation.
WHY INPNC
INP has been successfully used to display several proteins, such as levansucrase, carboxymethylcellulase (CMCase), Hepatitis B surface antigen (HbsAg), human immunodeficiency virus type 1 (HIV-1) gp120 and so on[1]. This was achieved using either full-length sequences ortruncated portions containing only N- and C-domains (INP- NC) or INP-NC with five additional internal repeating units to display foreign protein on the surface of cell. containing only N- and C-domains means no repeat sequent, producing no ice-nucleation activity. This indicates that the central repeating domains are not required for export to the cell surface, and are therefore, ideal spacer units to control the distance between the passenger protein and the cell surface. Importantly, INP can be expressed at the cell surface of E. coli at a very high level, without affecting cell viability: comparable to the endogenous expression of the OmpA porin[2].
Before us, there was some teams that used INPNC as display system successfully, which is encoded by inaZ. INPNC is used to display passenger protein, such as EYFP, silica binding protein and so on. After researching NCBI, we found that there are several types of nucleotides encoding INP, including inaZ, inaQ, inaK, inaV, inaX. Because inaZ, inaV ,inaX have less support by the published literature comparing to inaQ and inaK, we hope that we can use inaQ or inaK as target gene of recombinant plasmid to alleviate burden of E.coli.
WHY INPN
Then we found that using INP derivatives containing only N-domain (INPN) as display system successfully is possible. The passenger proteins are also various, such as Japanese Encephalitis Virus Pathogenicity, phosphate-binding protein and so on. There is also literature which alleges that since full-length INP is quite large (1,200–1,500 amino acid residues), functional truncated INP molecules may serve as better anchoring motifs to carry large heterologous proteins.
So we start thinking that we also could use INP derivatives containing only N-domain (INPN) as display system. As we say early, inaZ has less support. So, we use inaK and inaQ as ingredient. According to the results of literature survey, inaK-N has about 658bp and inaQ-N has about 525bp. We hoped truncated INP molecules serve as better anchoring motifs to carry heterologous proteins, our ModA.
Reference
[1] Mei Li Wu, Chun Yung Tsai& Tsai Hsia Chen, 2005, Cell surface display of Chi92 on Escherichia coliusing ice nucleation protein for improved catalytic and antifungal activity
[2] Edwin van Bloois, Remko T. Winter, Harald Kolmar and Marco W. Fraaije, 2010, Decorating microbes: surface display of proteins on Escherichia coli.
[3] jinalin Dou, Janet Daly, Zhiming Yuan, Tao Jing, and Tom Solomon,2009,Bacterial cell surface display: a method for studying Japanese Encephalitis virus pathogenicity
[4] Qianqian Li, Ziniu Yu, Xiaohu Shao, Jin He & Lin Li, 2009,Improved phosphate biosorption by bacterial surface display of phosphate-binding protein utilizing ice nucleation protein
Back to Top
INPN + ModA
As we talked about INPN as display system, INPN is used to carry ModA. Briefly, we want fusion protein that INPN is integration with ModA, as result. Everybody knows, two kinds of proteins are liked with each other may lead to change of high levels of protein structure. And other problems together make constructing fusing protein difficult.
We had two methods to fuse two protein:
1. Fusing PCR
After we tried Gibson Assembly unsuccessfully, we started work on fusing PCR. To avoid change the senior structure of proteins, we decide add a linker between two protein, which is about 6bp. We designed our primer, did our experiments.But because of spending much time on Gibson Assembly, we did not have the certain results yet. And we still work on so far.
2. Gibson Assembly
We did the Gibson Assembly first. Gibson Assembly is a high-efficiency DNA end-linking technique developed by Daniel Gibson at the JCVI in 2009. The technique was invented and perfected as part of the genome assembly efforts at JCVI. The method uses three enzymes to join two or more sequences of DNA when they have overlapping end sequences at their joining point (~40bp). These overlapping regions can be easily added to the ends of any length of DNA by using PCR with primers which have added adapter sequences. Thus PCR followed by Gibson Assembly allows you to join any two blunt ended pieces of DNA. And it can be used to fuse heterogeneous proteins. Yet we failed.
After we tried so many times, we found the results had very high false positive rate. When we looked for reasons why we can not make it , we found something in wiki of Team: Washington.   https://2011.igem.org/Team:Washington/Magnetosomes/GibsonVectors The pSB standard BioBrick vectors available in iGEM are not suit for efficient multiple-insert cloning beyond three fragments. The primary problem with a standard pSB is the self-ligation of the two NotI sequences located on both the BioBrick prefix and suffix. These cohesive ends prevent fragments from being incorporated efficiently and correctly.
Back to Top
*You can download the protocol of this part here: Protocol_ModA.pdf
Prev Page Next Page
The Story of E.coli Prometheus
BNU-China
Retrieved from " http://2014.igem.org/Team:BNU-China/ModA%26INPN.html "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
