<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_BNU-China_modeling_html"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:BNU-China/modeling.html">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:BNU-China/modeling.html&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:BNU-China/modeling.html&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:BNU-China/modeling.html&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:BNU-China/modeling.html" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:BNU-China/modeling.html</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><DIV id="thumbs"><A href="https://static.igem.org/mediawiki/2014/c/c9/Bnu_bg4.jpg">1</A><A href="https://static.igem.org/mediawiki/2014/2/2f/Bnu_bg5.jpg">2</A></DIV><DIV id="wrapper"><DIV id="menu" class="menu opacity"><UL><LI><A href="https://2014.igem.org/Team:BNU-China/home.html">Home</A></LI><LI><A href="#">Project</A></LI><LI><A href="https://2014.igem.org/Team:BNU-China/overview.html">Overview</A></LI><LI><A href="https://2014.igem.org/Team:BNU-China/ModA&amp;INPN.html">ModA+INPN</A></LI><LI><A href="https://2014.igem.org/Team:BNU-China/Chemotaxis.html">Delivery System</A></LI><LI><A href="https://2014.igem.org/Team:BNU-China/modeling.html">Modeling</A></LI><LI class="maps"><A href="https://2014.igem.org/Team:BNU-China/maps.html">       Maps</A></LI></UL><LI><A href="#">Achievements</A></LI><LI><A href="https://2014.igem.org/Team:BNU-China/parts.html">Submitted Parts</A></LI><LI><A href="https://2014.igem.org/Team:BNU-China/judging.html">Judging Criteria</A></LI><LI><A href="https://2014.igem.org/Team:BNU-China/future.html">Bright Future</A></LI><LI><A href="#">Safety</A></LI><LI><A href="https://2014.igem.org/Team:BNU-China/safety-overview">Overview</A></LI><LI><A href="https://2014.igem.org/Team:BNU-China/labenv">Lab &amp; Env safety</A></LI><LI><A href="https://2014.igem.org/Team:BNU-China/kill">Kill Switch</A></LI><LI><A href="#">Policy &amp; Practice</A></LI><LI><A href="https://2014.igem.org/Team:BNU-China/pp_overview.html">Overview</A></LI><LI><A href="https://2014.igem.org/Team:BNU-China/pubad.html">Public Promotion</A></LI><LI><A href="https://2014.igem.org/Team:BNU-China/3e.html" title="Environment, Efficiency and Economic">Three &quot;E&quot;s</A></LI><LI><A href="https://2014.igem.org/Team:BNU-China/3c.html" title="Collaboration, Conference and Communication">Three &quot;C&quot;s</A></LI><LI><A href="https://2014.igem.org/Team:BNU-China/edu.html">Edu-justice</A></LI><LI><A href="https://2014.igem.org/Team:BNU-China/chinese-biopolicy.html">Bio-policy</A></LI><LI><A href="#">Our Team</A></LI><LI><A href="https://2014.igem.org/Team:BNU-China/profile.html">Team Profile</A></LI><LI><A href="https://2014.igem.org/Team:BNU-China/attributions.html">Attributions</A></LI><LI><A href="https://2014.igem.org/Team:BNU-China/time_axis.html">Our Notebook</A></LI><LI><A href="https://2014.igem.org/Team:BNU-China/collaboration.html">Collaboration</A></LI><LI><A href="https://2014.igem.org/Team:BNU-China/acknowledge.html">Acknowledge</A></LI></DIV><DIV id="container" class="opacity"><H1 align="center">Modeling</H1><DIV class="one-third" style="background-color:antiquewhite; padding: 10px 30px 10px 30px"><H3>Achievements</H3><OL><LI>Construct a dynamic model to describe the chemotaxis quantitatively
</LI><LI>Build a 3-D model of peanut root and its excudate
</LI><LI>Simulate (in c++) chemotaxis dynamically and quantitively
</LI><LI>Fit experiment results with theoretical curve
</LI><LI>Estimate the effectiveness of “Prometheus” from the results of program and do forecast
</LI></OL></DIV><H2>Abstract</H2><P>Analyzing dynamics of <I>E.coli</I>’s chemotaxis and estimating effectiveness of the &quot;Prometheus&quot; require high accuracy, and thus hard to realize with experiments. Here, we construct a virtual peanut root-<I>E.coli</I> system (by c++ and OpenGL), storing the 3-D environmental information by k-d tree. We quantify the process in which “Promethei”, our engineered bacteria, carrying Mo, move towards peanut roots. We at last estimate around 20-40% of bacteria could work in reality, and that the best “memory” time be around 40 min (<A href="#res">See here</A>). All codes (in c++) of the project are available*.</P><P class="fig">*The size of the code is too big, please contact Li Si-yao (201211211043@mail.bnu.edu.cn) to get it.</P><P class="fig" style="margin-left:30px; width:86%">Fig. 1 The Motion Graph of the screenshots while running the Simulation Program. The purple points are <I>E.coli</I> tending towards the root surface.</P><A href="#top"><P class="fig" style="margin-left:790px">Back to Top</P></A><H2 id="eco"><I>E.coli</I> model</H2><H3>Mechanism</H3><P>We divide <I>Escherichia coli</I>’s behavior into two parts: “running” straight lines, and “tumbling” for reorientation[1]. When attractant concentration uniformed, <I>E.coli</I> can be described as doing “random walk”[4], and the running time fits a normal distribution. With spatial concentration gradients of chemical attractance,when bacteria sense a higher attractant concentration than before,the mean of the statistical distribution of running time increases. Otherwise, the time keeps the same as that in the uniform environment, no matter how high or low the concentration is[4].</P><H4>Running</H4><P>The running of <I>E.coli</I> can be descibed as a straight line. The distribution of running duration time fits a normal curve, with mean <STRONG>τ<SUB>run0</SUB></STRONG> and standard division <STRONG>σ<SUB>run</SUB></STRONG>.</P><P>Brown and Berg[4] suggested that the mean <STRONG>τ</STRONG> has a functional relationship, shown as follows, with the current attractant (in their case, glutamate) concentration and the change of concentration over time. (See formula (1),(2)).

</P><P><STRONG>K<SUB>D</SUB></STRONG> is the dissociation constant of the complex formed by glutamate and its receptor. <STRONG>Pb</STRONG> is the fraction of receptors (protein) bound with the attractant.</P><P>In their model, the mean velocity during a run, however, is independent from <STRONG>C</STRONG> and <STRONG>dC/dt</STRONG> [5,6].</P><P>Brown and Berg found the curve <STRONG>K<SUB>D</SUB>C</STRONG>/(<STRONG>K<SUB>D</SUB></STRONG> + <STRONG>C</STRONG>)<SUP>2</SUP> fits the chemotaxis sensitivity assays well (Fig. 2).</P><H4>Obtain K<SUB>D</SUB></H4><P>Brown and Berg found the curve <STRONG>K<SUB>D</SUB>C</STRONG>/(<STRONG>K<SUB>D</SUB></STRONG> + <STRONG>C</STRONG>)<SUP>2</SUP> fits the chemotaxis sensitivity assays well (Fig. 2).</P><P class="fig" style="width:20%; margin-left:280px">Fig. 2</P><P>We cannot determine the precise dissociation constants <STRONG>K<SUB>D</SUB></STRONG> by experiment. However, the trend revealed in out experiments (Fig. 3) fits with the model well. With data from McfR-Succinate experiment, we set the <STRONG>K<SUB>D</SUB></STRONG> value of succinate-receptor complex as 3.5 mM by Least Square method.</P><P class="fig" style="margin-left:100px; width:180px; margin-top:260px">Fig. 3 the result from experiment.</P><P class="fig" style="width:305px; margin-left:500px; margin-top:-40px">Fig. 4 Fitting the experiment by <STRONG>a</STRONG>( <STRONG>K<SUB>D</SUB>C</STRONG> / (<STRONG>K<SUB>D</SUB></STRONG> + <STRONG>C</STRONG>)<SUP>2</SUP>) + <STRONG>d</STRONG>, where <STRONG>a</STRONG> = 389, <STRONG>d</STRONG> = 260 and <STRONG>K<SUB>D</SUB></STRONG> is 3.5mM. This work is made by c++ and OpenGL. The code to get <STRONG>K<SUB>D</SUB></STRONG> and <STRONG>a</STRONG> and to make sketch is available. Please contact us.</P><P>Given a concrete constant number of α in function (1), we would be able to calculate the value of τrun. Series of experiments[4] convinced us to* set α as 660 sec.</P><P class="fig">*We have no time to repeat the experiments with succinate.</P><P>In conclusion, modeling running is based on following statement:</P><UL><LI>The velocity during a run is constant (always equal to the mean);
</LI><LI><I>E.coli</I> move straightly during a run, and neither die nor divide in the process. The effect of quorum sensing and nutritional factors are eliminated;
</LI><LI>The <I>E.coli</I>’s running time t<SUB>run</SUB> is a random number from N(&lt;τ<SUB>run</SUB>&gt;,σ<SUB>run</SUB>);
</LI><LI>The mean duration time &lt;τ<SUB>run</SUB>&gt; can be derived from formula (1) and (2), given the value of C and dC/dt;
</LI><LI>The specific duration_time of a single bacterium changes in the same pace with the changes of the statistical distribution’s mean (relying on C and dC/dt). (Fig. 5)
</LI></UL><P class="fig" style="margin-left:400px;width:4%;">Fig. 5</P><P>For a single bacterium, its surroundings decides the statistical distribution of its own running time. In other words, the discrepancy doesn’t change:</P><P>where t<SUB>run</SUB> is the specified starting and stopping time for a single bacterium. τ<SUB>run</SUB>2 is the current mean duration time and τ<SUB>run</SUB>1 is the mean duration time at last second; both are derived from the function (1) and (2). </P><H4>Tumbling</H4><P><I>E. coli</I> reorient between two runs.In modeling this process, we assume:</P><UL><LI><I>E. coli</I> do not change positions as they are tumbling;
</LI><LI>the direction after reorientation is randomly chosen.
</LI></UL><P align="center" class="fig" style="width:320px; margin-left:250px">Fig. 6 Schematic drawing of a tumble, from [2], page 45</P><P>The angle dθ is not strictly random [1,5,6] (its distribution shown in Fig. 7). However, the distribution is not formularily clear and would require too much effort to build a random number fitting this distribution, thus we roughly regard it as a random process.</P><P class="fig" style="margin-left: 250px; width:350px;">Fig. 7 the distribution of change in direction ( A figure from [1] ).</P><P class="fig">*The statement is made to calculate the process more easily. We suppose it is the run duration time, not the difference of velocity, is the main variant of the whole chemotaxis system.</P><H3>Coefficients</H3><P>velocity = 100μm</P><P>run duration time mean in uniform environment = 1.3</P><P>run duration standard division = 3</P><P> mean tumble duration time = 0.14 sec</P><P>alpha (See equation (1)) = 660 sec</P><P><STRONG>K<SUB>D</SUB></STRONG> = 3500 μmol/L</P><A href="#top"><P class="fig" style="margin-left:790px">Back to Top</P></A><H2 id="mod">Modeling Root</H2><H3>Constructing Virtual Root from Laser Scanning data</H3><P>We use laser scanner to get the root surface topological information. The scanner is Konica Minolta Vivid910, with lens of middle focal length, offered by the College of Information Science and Technology of Beijing Normal University. We obtained 142735 vertices and 253404 faces from it (Fig. 8).</P><P class="fig">Fig. 8 The virtual peanut from the scanning, scanned by Konica Minolta Vivid910, with lens of middle focal length, offered by the College of Information Science and Technology of Beijing Normal University.</P><H3>Building concentration gradient</H3><P>The concentration gradient of the root’s exudate can be regarded as stable in the soil. Exudate concentration decreases as the distance from the root surface increases. (Fig. 9.) </P><P class="fig">Fig. 9 Sketch of the expected process of chemotaxis. Root exudate’s concentration gradient are shown in different colors. Green points are imaginary <I>E.coli</I>running up the gradient towards the root.</P><P>The concentration value of points is calculated from [8]. Despite limited study directly discussing the spatial distribution patterns of peanut’s root exudates, we can still describe, as [8] proposed for maize and wheat, the spatial distribution of succinate as Y = A * X<SUP>-B</SUP> (Fig.11), where Y represents the concentration of succinate at the distance X from the peanut root surface, when X &gt;= 1mm. </P><P class="fig" style="float:left; width:50%; margin-top:300px;">Fig. 10 Concentration gradient of maize and wheat proposed in [8]. Y-axis is the percentage of C14 activity, reflecting the concentration. </P><P>As the relationship is applicable only when x &gt;= 1 mm. We estimate the percentage decrease linearly from 0 to 1 mm, and the Y value is 70% at x = 1 mm. Thus, the function of the concentration C on the distance to the root surface x is</P><P>C = 100 - 0.3 * x   if   x &lt; 100   and</P><P>C = 70 * x^(-1.2)   if   x &gt; 100.         (3)</P><H3>Using k-d tree to store points</H3><P>K-dimension tree[7] is a smart and widely-used data structure to store information of points in k-dimension space. It is very fast, typically requiring O(logN) time, to find the nearest point with a given position. Here we implement k-d tree to construct all 10,000 points.</P><P>We derive the concentration at the bacterium’s vincinity from the formula (3) described above.</P><A href="#top"><P class="fig" style="margin-left:790px">Back to Top</P></A><H2 id="pro">Programming</H2><P>The program simulates movement per second, herein attractant is succinate. The pseudo-code reads as follows:</P><P><STRONG>move</STRONG> (a bunch of <I>E.coli</I> , the concentration gradient)</P><P>each <I>E.coli</I> has <STRONG>[state, timer, position, direction, duration_time]</STRONG></P><P class="fig">
*The points of concentration gradient are stored in a 3-d tree It will expounded clearly in the part below.
** The mean value is calculate by the formula(1), (2) and (4).
*** This step is by calculating duration_time (this moment) - timer, and if the result &gt; 0, then it should go on this behavior.
</P><A href="#top"><P class="fig" style="margin-left:790px">Back to Top</P></A><H2 id="res">Results and analysis</H2><H3>Assessment of “Prometheus”</H3><P>In each run estimating the effectiveness of the whole system, the program simulates behaviors of 1000 bacteria over 1 hour (3600 sec). Fig. 11 encompasses 4 different orders of initial succinate concentration on root surface. Compared with the null control (0 M), the engineered <I>E. coli</I> can be successfully recruited around root. Under the concentration of 10<SUP>-2</SUP> M, the number of bacteria within 1mm around root surface doubled that of null control.</P><P class="fig" style="width:95%;">Fig. 11 The react of <I>E. coli</I> to different concentration of succinate. The numbers of bacteria within 1mm of root surface (y-axis) represents the bacteria successfully attracted and utilized overtime. We assumed the total number of bacteria being 1000. Under the model, the peak performance of “Promethei” are expected between 10<SUP>-2</SUP> and 10<SUP>-3</SUP>. Our simulation fits the experiments and the theoretical curve K<SUB>D</SUB>C / (K<SUB>D</SUB> + C)<SUP>2</SUP> well. </P><P>The results amazingly fit the 96-well plate experiment and the theoretical curve (Fig. 12). The running of the program has the similar mechanism with the experiments done in a laboratory. That can prove our codes are correct from the other aspect.</P><P class="fig" style="width:2%;margin-left:400px ">(a) </P><P class="fig" style="width:2%; margin-left:120px; float:left;">(b)</P><P class="fig" style="width:2%;float:left; margin-left:450px">(c) </P><P class="fig">Fig.12 The contract of the program result in computer (a), the experiment result (b) ( <A href="https://2014.igem.org/Team:BNU-China/Chemotaxis.html" target="_blank">get to know more click here</A>) and the theoretical curve of K<SUB>D</SUB>C / (K<SUB>D</SUB>+C)<SUP>2</SUP> (c).</P><P>Considering the overall amount of succinate secreted via root by a plant could reach 13 mg [9], we estimate the concentration on root surface be between 0.1-10 mM. We conservatively reckon that around 20-40% bacteria could eventually “transport” cargos to the root (cut a half for unconsidered conditions in soil). Thus about 60-80% would lead to indirect use in the process, raising the issue of potential Mo pollution.</P><H3>Estimation of “Memory” Time</H3><P>We further estimate the best time interval to set the “memory” time, the time most appropriate for <I>E. coli</I> to suicide. Such time is determined as immediately after most of bacteria has fulfilled their task. With results in Fig. 2, the “memory” time is estimated around 40 minutes. </P><A href="#top"><P class="fig" style="margin-left:790px">Back to Top</P></A><H2 id="rev">Review and forecast</H2><P>We think our model applies to the situation where bacteria swim freely without casualty. However, for the model has many statements, we could expect nuances from the real complex world. The differences in condition are listed in Table 1.</P><P class="fig" style="width:95%">Table 1 The difference between conditions in the real world and those assumed in our model. Shown in red are factors that could significantly alter the eventual result. Viscid layers around root could potentially boost the system’s performance. </P><P>We can probably improve the performance by triggering suicide with a threshold concentration, instead of time. But since we lack corresponding element for such function, and with certain biosafety concerns (to prevent gene pollution), we stick to time trigger.</P><P>
Still, the modeling, albeit simple, provides us with an otherwise infeasible evaluation of the whole project. The Prometheus has certain potential to work as we wished.</P><A href="#top"><P class="fig" style="margin-left:790px">Back to Top</P></A><H2 id="ref">Reference</H2><P>[1]Chemotaxis in <I>Escherichia coli</I> analyed by Three-dimensional Tracking, Nature 239:500-504, by Howard C. Berg &amp; Douglas A. Brown</P><P>[2]<I>E. coli</I> in Motion, Springer, by Howard C. Berg</P><P>[3]The Gradient-Sensing Mechanism in Bacterial Chemotaxis, Proc. Natl. Acad. Sci. USA 69:2509-2512, by Macnab, R. M., D. E. Koshland</P><P>[4]Temporal Stimulation of Chemotaxis in <I>Escherichia coli</I>, Proc. Natl. Acad. Sci. USA 71:1388-1392, by Brown, D. A., and H. C. Berg</P><P>[5]Directional persistence of chemotactic bacteria in a traveling concentration wave, PNAS, by J. Saragosti, V. Calvez, N. Bournaves, B. Perthame, A. Buguin, and P. Silberzan</P><P>[6]Modeling <I>E.coli</I> Tumbles by Rotational Diffusion Implications for Chemotaxis, PLOS one, by Jonathan Saragosti, Pascal SilBerzran, Axel Buguin</P><P>[7]Algorithms 4th edition, Pearson, Robert Sedgwick and Kane Wayne
</P><P>[8]Spatial distribution of root exudates of five plant species as assessed by labeling, J. Plant Nutr. Soil Sci. 2006, 169,360-362 , by Daniela Sauer, Yakov Kuzyakov, and Karl Stahr</P><P>[9] Mineral Nutrition of Higher Plants. China Agricultural University Press. 2008.</P><A class="button blue" style="margin-left:20px; float: left;" href="https://2014.igem.org/Team:BNU-China/Chemotaxis.html">Prev Page</A><A class="button green" style="margin-left:680px" href="https://2014.igem.org/Team:BNU-China/parts.html">Next Page</A></DIV><DIV id="copyright" class="opacity"><P>The Story of <I>E.coli Prometheus</I></P><P style="font:smaller">BNU-China</P></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:BNU-China/modeling.html">http://2014.igem.org/Team:BNU-China/modeling.html</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:BNU-China/modeling.html" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:BNU-China/modeling.html" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:BNU-China/modeling.html&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:BNU-China/modeling.html&amp;oldid=395567" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>