<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_BostonU_Notebook skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:BostonU/Notebook</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><UL id="nav"><LI><A href="https://2015.igem.org/Team:BostonU">Home</A></LI><LI><A href="">Research</A></LI><LI><A href="https://2015.igem.org/Team:BostonU/Temporal_Control">Controlling Protein Activity</A></LI><LI><A href="https://2015.igem.org/Team:BostonU/Modeling">Experimental Design</A></LI><LI><A href="https://2015.igem.org/Team:BostonU/App_1/Motivation">Controlling Integrases and RDFs</A></LI><LI><A href="https://2015.igem.org/Team:BostonU/App_2/Motivation">Controlling saCas9</A></LI><LI><A href="https://2015.igem.org/Team:BostonU/Design">Functional Prototype</A></LI><LI><A href="https://2015.igem.org/Team:BostonU/Notebook/Protocols/Bacterial">Notebook</A></LI></UL><LI><A href="">iGEM Community</A></LI><LI><A href="https://2015.igem.org/Team:BostonU/Mammalian_synbio/Significance">Mammalian Synthetic Biology</A></LI><LI><A href="https://2015.igem.org/Team:BostonU/Interlab">Interlab Study</A></LI><LI><A href="https://2015.igem.org/Team:BostonU/Parts">Part Submissions</A></LI><LI><A href="">Outreach Initiatives</A></LI><LI><A href="https://2015.igem.org/Team:BostonU/Education/Building_with_Biology">Education</A></LI><LI><A href="https://2015.igem.org/Team:BostonU/Collaborations">Collaborations</A></LI><LI><A href="https://2015.igem.org/Team:BostonU/Industry">Industry</A></LI><LI><A href="">Acknowledgements</A></LI><LI><A href="https://2015.igem.org/Team:BostonU/Team">Our Team</A></LI><LI><A href="https://2015.igem.org/Team:BostonU/Attributions">Attributions</A></LI><LI><A href="https://2015.igem.org/Team:BostonU/Medals">Judging</A></LI><TITLE>Our Project</TITLE><TABLE align="center" style="margin: 0px auto;"><TBODY><TR><TD><A href="https://2015.igem.org/Team:BostonU/Notebook/Protocols/Bacterial" class="button">Bacterial Protocols</A></TD><TD><A href="https://2015.igem.org/Team:BostonU/Notebook/Protocols/Mammalian" class="button">Mammalian Protocols</A></TD><TD><A href="https://2015.igem.org/Team:BostonU/Notebook" class="button">Day by Day</A></TD></TR></TBODY></TABLE><DIV class="contentdiv"><DIV class="mainpage"><H3>Notebook</H3><H4>May</H4><H5>Week One - May 28th to May 29th</H5><DL><DD> May 28th 2015</DD></DL><OL><LI>Made broth and agar</LI><LI>Plated the agar</LI><LI>Aliquot 50mL LB media tubes</LI><LI>Innoculate Ben’s mammalian backbones with dimerization domains and Ben’s phiC-31, TP901-1, ofr7 and gp3 plasmids</LI></OL><DL><DD> May 29th 2015</DD></DL><OL><LI>Cell stock creation from frozen plasmids</LI><LI>Miniprepped the plasmids we grew up and cell stocked for iGEM cell stocks</LI><LI>Using the nanodrop fluorospectrometer</LI></OL><H5>Week Two - June 1st to June 5th</H5><DL><DD> June 1st 2015</DD></DL><OL><LI>Digested Ben's backbones</LI><LI>PCR set-up</LI><LI>Made an agarose gel to separate the products of the PCR</LI><LI>Made more pS-5 and pS-18 cell stocks (Our pS-5 and pS-18 cell stocks gave us a very low yield when their concentrations were tested with a nanodropper).</LI></OL><DL><DD> June 2nd 2015</DD></DL><OL><LI>Mini-prepped the pS-5 and pS-18 cell stocks that were made yesterday. </LI><LI>Measured the concentrations of DNA from the purified digestion results and purified gel results. </LI><LI>Digested pS-5 because it is a backbone. pS-18 is not a backbone that is why it was not re-digested. </LI><LI>Ran the PCR for pS-14 and 10 different primers for a second time, the first time we did this was yesterday but the test yielded strange results in that our bands were thick, clouded, and our ladder was relatively useless because it did not run for a long enough period of time. </LI><LI>Ran the gel for the results from today’s PCR in order to see if we can get better results than yesterday and we did.</LI><LI>Gel purified</LI><LI>Digesting Our Two Integrases with Enzymes in Order to Ligate (the pS-14 phi 31 integrases into pS-1:12 backbones)</LI><LI> PCR Purification of Our Restriction Results to Get the Integrase Inserts</LI><LI>Ligation</LI></OL><DL><DD> June 3rd 2015</DD></DL><OL><LI>Checked out the colonies and Ben found that our pS-9 and pS-11 had very low yields when checked against their respective negative control colonies</LI><LI>Picked colonies</LI><LI>Emailed former BU students who developed a light stimulation device</LI></OL><DL><DD> June 4th 2015</DD></DL><OL><LI>Took colonies out of incubator, decanted media</LI><LI>Miniprepped from deep wells</LI><LI>Test cut plasmids</LI><LI>Ran gel electrophoresis</LI></OL><H5>Week Three - June 8th to June 12th</H5><UL><LI>June 8th 2015
</LI><LI>Repicked colonies for bad test cuts
</LI><LI>200, 201, 204, 205, and 208</LI><LI>Created new deep well plates with LB + Carb</LI><LI>Picked three new colonies from each of these plates</LI><LI>Incubated reference plate and deep well plate overnight</LI><LI>Saving a Gel (There was an incorrect number of wells in a gel we made so: )
</LI><LI>We tore up the gel and put it into the iGEM gel erlemyer flask</LI><LI>Microwaved the 100mL of gel for one minute until it was melted</LI><LI>We re-added the ethidium bromide</LI><LI>We re-poured the gel</LI><LI>June 9th 2015
</LI><LI>Redid test cut of bad test cuts from yesterday
</LI><LI>Miniprep colonies</LI><LI>Digest plasmid and incubate for 30 minutes</LI><LI>Run gel to compare current digested bands to how long they should be (found by looking at initial sequence)</LI><LI>Notice that maybe inserts 4, 5 and 8 are not properly ligated, contaminated or in general not properly prepared so plan to redo PCR of those insert fragments and start from there</LI><LI>Re-did the gel again, got the same results</LI><LI>Tried to PCR prep 4, 5 and 8 but the settings were incorrect so we threw away the samples and resolved to start again tomorrow.</LI><LI>June 10th 2015
</LI><LI>Jeff re-made the PCR solutions</LI><LI>We ran the PCR with the correct iGEM settings</LI><LI>Ran a gel with the PCR results</LI><LI>We decided to re-do the PCR but use half the amount of primmers in pREC 205 and 208 because there may be a primer dimer band showing up; we also decided to double the extension time because pREC 204 looked like it wasn't getting enough time to extend</LI><LI>Ran a gel with the new PCR products; we'll call this gel #2</LI><LI>Cut out the fragments we identified as the proper inserts</LI><LI>Purified them</LI><LI>Digested the fragments</LI><LI>Purified the digested fragments</LI><LI>June 11th 2015
</LI><LI>Our previous ligation did not work, it had grown no colonies when we got into lab this morning and we did not make a negative plate. </LI><LI>Today, we redid the test cuts for pRECs 195, 196, 197, 198, 199, 203 and 207</LI><LI>Today we are also going to send DNA for sequencing, based on gel results, we have decided to send pREC 193 (2), 194 (1,3), 202 (1,2), 206 (2,3)</LI><LI>Digested the backbones. Instructions from Ben’s ABA and rapamyacin backbones in order to get the inserts we want to send for sequencing </LI><LI>Abha and Jeff redid the ligation for pREC 200, 201, 204, 205, 208.</LI><LI>Abha and Jeff did the transformation by smearing 50 microliters of this mixture: 50 microliters Top 10 Cells, 50 microliters KCM, 10 microliters ligation; placed in incubator at noon</LI><LI>After digesting Ben’s Inserts, we are going to run a gel and purify the bands that correlate to the insert</LI><LI>The results of my test cuts turned out pretty much the same as the June 5th gel so we are not going to send any additional pRECs for sequencing. </LI><LI>Ben’s inserts digested really well and we now we are going to ligase the purified bands (inserts) into the backbones</LI><LI>June 12th 2015
</LI><LI>Today we checked Abha and Jeff transformations. We found no growth on the plates except for the negatives, which had quite a bit. This means that most likely we did not digest the backbones properly or we did not perform the ligation correctly. </LI><LI>Shaheer’s plates with Ben’s inserts and the iGEM team’s backbones did yeild positive results. Additionally, Shaheer’s negative controls did not grow any colonies, so we can be confident that the colonies obtained are transformed colonies. </LI><LI>Shaheer harvested the colonies from his transformed plates, as described by the Deep Well Miniprep procedure.</LI><LI>We got back the results of the pRECs Ben sent for sequencing yesterday and have yet to cross examine them with their expected sequences. All of the pRECs chosen, with the exception of pREC 206 (3), yeilded at least one “Good” sequence.</LI><LI>Ben analyzed the results and hypothesized that our pS-9”12 backbones were digested incorrectly. </LI><LI>We are going to test cut the backbones to verify their sequences by digesting each backbone with the opposite enzyme with which we are going to ligate it.</LI><LI>Because of all of the vectors looked exactly as expected, we suspect the switch up Ben identified from the sequencing occurred at some later point. </LI><LI>Jeff and Kate digested the backbones</LI><LI>Kate did the digestion with two mastermixes, 240 microliters water, 40 microliters buffer, 12 microliters enzyme 1 and 12 microliters enzyme 2. Then she loaded 38 microliters mastermix into the corresponding cryotubes, then added 12 microliters DNA to the corresponding cryotubes.</LI><LI>June 13 2015</LI><OL><LI>Centrifuged deep wells at 3500 rpm for 6 min</LI><LI>Removed supernatant</LI><LI>Sttored in -20 degrees celsius fridge</LI></OL></UL><H5>Week Four - June 15th to June 18th</H5><UL><LI>June 15th 2015
</LI><LI>Abha and Jeff miniprepped their ligated plasmids from the deep wells</LI><LI>Then they test cut them; the results did not turn as desired</LI><LI>June 16th 2015
</LI><LI>The colonies looked good so we picked three from each pREC colony for pRECs 193-208. </LI><LI>Abha filled a deep well with 2 mL LB brother + carb for growing up cell colonies</LI><LI>June 17th 2015
</LI><LI>We centrifuged the deep wells at maximum speed for 6 minutes.</LI><LI>We miniprepped Ben’s inserts and iGEM backbones from the deep wells.</LI><LI>Abha and John started the PCR of our new prRECs.</LI><LI>We test cutted the miniprepped plasmids and found that pRECs 193-200 turned out really well but the pREC 201-208 did not show up. We hypothesized that there is something wrong with the enzymes EcoRI or Kpn1 or buffer. </LI><LI>We test cutted pRECs 201-208 again, and found the same results, which were that a backbone appeared in the gel, but no insert was digested out. </LI><LI>Abha and Jeff ran a gel for the prRECs that they PCRed and found that their results were not promising. </LI><LI>June 18th 2015
</LI><LI>Test cutting Divya’s PO-004 plasmids to see if our EcoRI or Kpn1 or Cutsmart buffer is bad. </LI><LI>Ben said this morning that our sequencing results from pS-9 and pS-11 are very promising!</LI><LI>Our test cut did not work. This means that we do not know whether it is our enzymes or our buffer that is faulty. </LI><LI>PCR of prREC1-36 (1-18 for TP901 and 19-36 for PhiC31)</LI><LI>Gel purified prREC 1-36 (many did not show the proper band size)</LI><LI>Made new carb+agar plates</LI></UL><H5>Week Five - June 22nd to June 26th</H5><UL><LI>June 22nd 2015
</LI><LI>Test cutting pS-10 and pS-12 pRECs 201-208 with Mlul and AccIII because we may have a bad enzyme in the EcoRI/Kpnl digests of pREC 201-208</LI><LI>Our gel returned the expected results from our test cut. This means that our issue probably lies in the enzymes or buffer and not in the plasmids themselves. </LI><LI>Nanodropped purified prRECS 1-36 and many had low yield</LI><LI>Redid PCR on 17 prRECS (most bands aligned the expected size!)</LI><LI>Repurified redone PCRs</LI><LI>Sent 23 tubes for sequencing
</LI><LI>pREC 193-196 using BHW100 reverse primer (this is redoing the reverse CIBN sequencing from 06/17/15 because the reverse primer was too far down to show anything in the actual PhiC31 region)</LI><LI>pREC 197 (3 colonies) using BPO0038 forward primer (for some reason these were not sent in the 06/17/15 sequencing; reverse was not sent because we do not have the proper CRY2 reverse primer)</LI><LI>pREC 201-204 using BPO0038 forward primer and BHW98 reverse primer</LI><LI>pREC 205-209 using BHW99 forward primer and BHW100 reverse primer</LI><LI>Ordering from Quintera:
</LI><LI>label plasmid files with pREC # and primer used </LI><LI>open recent orders</LI><LI>download then extract the zipfile; copy extracted files into dropbox</LI><LI>open .seq file in Ape: compare to pREC sequence map in the dropbox</LI><LI>Tools -&gt; align two sequences (align the quintara sequence with the pREC map)</LI><LI>Mismatches towards the end are okay because there is a lot of noise towards the end of sequencing (you can see the progression in the .api quintara files)</LI><LI>Reading and sequence verifying the alignment data: wrong primer would give you a lot of red in the results where there were mismatches; good results would show matching in the backbone and part of the phiC31 domain (verifies correct plasmid backbone and insert ligated)</LI><LI>June 23rd 2015
</LI><LI>Split into Team A and B, Team B is in charge of Ben’s primers and the Cas9 splits. </LI><LI>We read the results of the sequencing from Monday about pRECs 197-208 and did not get good results - Divya hypothesizes that this is because there is something wrong with our Primers or on Quintara’s end. She said she was having trouble with her sequencing as well and that is why she thinks it is a sequencing problem and not a plasmid/ligation problem.  </LI><LI>June 24th 2015
</LI><LI>We are going to re-send pRECs 197-208 for sequencing</LI></UL><H5>Week Six - June 29th to July 3rd</H5><UL><LI>June 29th 2015
</LI><LI>The prRECs 37-54 came in. We added the micromolar amount of autoclaved deionized water specified on the IDT prREC order sheets. </LI><LI>Then we labeled microcentrifuges for each prREC and made a working stock by adding 90 microliters deionized autoclaved water to the microcentrifuge tubes so that we could use only one tip. Then we added 10 micromolars of each respective prREC to each microcentrifuge tube so that we have a 10 micromolar working stock now.</LI><LI>June 30th 2015
</LI><LI>Shaheer re-digested the backbones pS 1 to pS 4, and digested backbones from pS 1:4 stock. He gel purified the samples and these are what we will use in our ligation later today. </LI><LI>PCR of prRECs 37-54</LI><LI>Attempt #1: Jeff and John ligated the pRECs</LI><LI>July 1st 2015
</LI><LI>Attempt #2: Jeff re-transformed the ligations from yesterday</LI><LI>July 2nd 2015
</LI><LI>Attempt #3: Shaheer re-ligated the pRECS that did not work from Attempt #1 and #2</LI><LI>July 3rd 2015&lt;</LI><LI>Attempt #4: Jeff and Shaheer re-ligated the pRECs that did not work from Attempt #1, #2, and #3 </LI><LI>July 4th 2015
</LI><LI>Jeff picked the colonies from ligation attempt #4 and set them up for inoculation</LI><LI>July 5th 2015
</LI><LI>Shaheer mini-prepped the deep wells from ligation attempt #4.</LI></UL><H5>Week Seven - July 6th to July 10th</H5><UL><LI>July 6th 2015
</LI><LI>Need to discuss gooing to Wellsley on Sat for collaboration</LI><LI>E-mail the team for booking a room for the Japanese team</LI><LI>E-mail Mo about the MIT parts we have been looking for</LI><LI>Ran a PCR for prREC 37-54 </LI><LI>Made a medium gel for the PCR products</LI><LI>July 6th 2015
</LI><LI>Set-up PCR for prRECs 37-54 with same PCR machine settings as 6/30/15. Extension time of 25 sec for the forward reactions and 45 seconds for the reverse. 
</LI><LI>Fwd primers include: 37,39; 37,41; 37,43; 37,45; 37,47; 37,49; 37,51; 37,53; </LI><LI>Reverse primers include: 38,40; 38,42; 38,44; 38,46; 38,48; 38,50; 38,52; 38,54</LI><LI>Shaheer and Jeff made 50 more agar plates</LI><LI>Divya discovered that there was a miscommunication between myself, Jeff and John about what step of the process we were on. I left them with a gel last week and the “To Do Checklist for Team B” but the digestion and PCR clean-up of the inserts failed to occur. </LI><LI>We do not have enough insert left to digest and re-ligate and so I am PCRing again</LI><LI>Made a gel for the PCR results</LI><LI>Ran a gel</LI><LI>The following inserts were digested properly: 37,39; 37,41; 37,43; 37,45; 37,51; 37,53; 38,44; 38,46; 38,48; 38,52; 38,54;</LI><LI>Jeff cut out the proper bands</LI><LI>Jeff digested the inserts that were digested properly</LI><LI>Shaheer did a PCR clean-up on the product of the digestion </LI><LI>Created more T53 saCas9 cell stock by growing some up from the -80 C with Thomas</LI><LI>Shaheer made more pS 1:4 cell stock by growing them up from the -80 C</LI><LI>July 7th 2015
</LI><LI>Miniprepped the pS 1:4 inserts and the saCas9 T53 from incubator </LI><LI>Made more Top 10 cell stocks</LI><LI>Re-digested PCR products from Attempt #1</LI><LI>Cleaned up the fridge</LI><LI>Alloquated buffer</LI><LI>Made more MX1</LI><LI>Digested pS 1:4 backbones that I miniprepped and also from Friday because we are not sure whether or not Shaheer had digested the ones he made on Friday</LI><LI>Did PCR clean up on PCR products from Attempt #1 -Kate (now these are in my working box)</LI><LI>PCRed all of the prRECs that did not work from yesterday</LI><LI>Made a gel to run the PCR products</LI><LI>Did a PCR clean up on the backbones</LI><LI>Ran PCR products on gel</LI><LI>Digested PCR products</LI><LI>Did a PCR clean up on the digested products and now these are in my working box </LI><LI>Made the spreadsheet for ligating</LI><LI>Test transformation of the new Top 10 cells</LI><LI>July 8th 2015
</LI><LI>Kate ligated, transformed and plated 32 constructs; used the original attempt #1 inserts; used the backbones and inserts in Kate’s working box</LI><LI>Jeff autoclaved tip boxes</LI><LI>Shaheer is test and transforming with the competency kit from iGEM</LI><LI>July 9th 2015
</LI><LI>Kate picked colonies and made a reference plate from pRECs 209-224 and pREC 243-258</LI><LI>Shaheer’s test transformations failed again so he is trying to do them for a 3rd time today</LI><LI>Kate ligated, transformed and plated 7 constructs; was not able to ligate an 8th because John said prREC 12,2 is not working during PCR and they tried to do it more than 3 different times. </LI><LI>July 10th 2015
</LI><LI>Jeff Shaheer and Kate mini-prepped the four deep wells from the colonies she picked yesterday. </LI><LI>Kate moved her reference plates to the cold room.</LI><LI>Kate picked colonies and made a reference plate from the 7 constructs yesterday. </LI><LI>Shaheer made more competent Top10 cells. </LI><LI>Jeff made more LB+Carb</LI><LI>Jeff made more WN</LI><LI>Shaheer and Kate digested and test cut the miniprepped DNA.</LI></UL><H5>Week Eight - July 11th to July 18th</H5><UL><LI>July 11th 2015
</LI><LI>Nick and John spun down, vacuumed and placed into the -20C freezer the deep wells from yesterday. </LI><LI>Nick and John moved the reference plate to the cold room. </LI><LI>July 12th 2015
</LI><LI>Kate re-test cut the saCas9 pRECs. </LI><LI>Kate tested the Top10 cells for competency with the iGEM test kit by following the iGEM Competency Test Kit Protocol. </LI><LI>July 13th 2015
</LI><LI>Kate re-test cut the even numbered saCas9 pRECs.</LI><LI>Kate re-test cut the even numbered TP901 test cuts </LI><LI>Kate re-did the iGEM competency test kit with much lower concentrations because the results of yesterday’s test were that there were too many cells to count. </LI><LI>July 14th 2015
</LI><LI>Kate sent the TP901 pRECs for sequencing. </LI><LI>Kate ligated, transformed and plated 12 constructs.</LI><LI>Used pS 5, 6, 9 and 10 from June 12 2015; used prREC 13, 14, 17, 11, and 5 from the same date; used prREC 6 and 18 from July 9 2015.</LI><LI>July 15th 2015
</LI><LI>Picked colonies</LI><LI>Threw away our 2 working aliquots of water </LI><LI>Got 5 Carb plates form Ben/Matje </LI><LI>Made new water aliquots</LI><LI>Make new 1x KCM aliquot</LI><LI>Making controls and ligations
</LI><LI>only top 10 cells new</LI><LI>top 10 cells new and old kcm</LI><LI>only old top 10 cells</LI><LI>only old KCM</LI><LI>Picked pREC 53(1), 65(2), 77(1) from 1st attempt reference plates. </LI><LI>Read sequencing results previous to step 11 </LI><LI>Found sequencing primers for saCas9</LI><LI>Designed sequencing primers for saCas9</LI><LI>Sequencing Results notes
</LI><LI>the pREC 65 (2 and 3) are pREC 77</LI><LI>sequencing for 53(1) worked well</LI><LI>going to re-grow up 53(1) 65(2) and 77(1) and then re-sequence</LI><LI>July 16th 2015
</LI><LI>We away the products in the deep wells from yesterday because we determined that they were contaminated with yeast. </LI><LI>Results from controls:
</LI><LI>the new top 10 cells are contaminated with yeast</LI><LI>the old top 10 cells and the KCM are fine</LI><LI>Threw away the reference plate I made yesterday.</LI><LI>Threw away the plates from yesterday</LI><LI>Re-transforming all 14 products from yesterday into old top 10 cells</LI><LI>Miniprepping 53(1), 65(2), 77(1)</LI><LI>Cell stocking 53(1), 65(2), and 77(1)</LI><LI>Retransformed yesterday’s ligation mixes</LI><LI>Plated</LI><LI>Sent pRECs for sequencing. </LI><LI>Test cut phiC31 precs</LI><LI>Made 60 well large gel</LI><LI>Retransformed Jeff’s ligation mixtures</LI><LI>Re-test cutted the PREC 201-208 test-cuts because they did not turn out well</LI><LI>The pRECs that turned out well pREC 193(1), 194(1,2), 195(1), 196(1), 197(1,2), 198(1,3), 199(2,3), 200(1,2) I nanodropped and sent for sequencing.</LI><LI>We did a PCR clean-up on the digested inserts 12, 26, and 27</LI><LI>Ligated constructs 77, 78, 81, and 82 according to the table on the dropbox</LI><LI>July 17th 2015
</LI><LI>Re-test cut 201-208 constructs</LI><LI>Picked my old TP901 constructs</LI><LI>pREC 89 did not grow again</LI><LI>Made a 28 well medium gel</LI><LI>Analysed test cuts on Long Form pREC database</LI><LI>Nanodropped successful pRECs 201-208</LI><LI>Picked colonies</LI><LI>Aliquotted and diluted the new sequencing primers that came in</LI><LI>Diluted and sent all the pRECs that needed to be for sequencing</LI><LI>July 18th 2015
</LI><LI>Spun down all 3 deep wells</LI><LI>Re-labeled pREC 65 to be pREC 77 in the -80 and -20 freezers based on the results of the sequencing </LI><LI>Miniprepped my 3 deep wells of DNA</LI><LI>Test cutted the 3 deep wells worth of DNA</LI></UL><H5>Week Nine - July 20th to July 25th</H5><UL><LI>July 20th 2015
</LI><LI>I picked cells to grow up from the 6/16/15 reference plates 193(1), 194(1), 195(1), 196(3), 197(1), 198(1), 199(3), 200(1), 201(2) in order to cell stock and miniprep them for tomorrow</LI><LI>Grew up more pS 5</LI><LI>Ligated</LI><LI>Cleaned out some old plates from the cold room because it was getting cluttered </LI><LI>Sent pRECs 54, 57, 58, 66, 69, 70, 78, 81, and 82 for sequencing </LI><LI>July 21st 2015
</LI><LI>pREC 65, 90, and 12 did not grow anything</LI><LI>sent pREC 215, 216, and 220 for sequencing</LI><LI>John made more LB + Carb</LI><LI>John made more LB + Carb plates</LI><LI>Picked pRECs</LI><LI>Grew up pRECs 202:210 and 212</LI><LI>Cell stocked pRECs 193:201</LI><LI>Miniprepped pRECs 193:201, pS 5 and Divya’s four plasmids for the Upward Bound Students </LI><LI>Digested Divya’s plasmids for the upward bound students</LI><LI>Analyzed sequencing from yesterday for pRECs 54, 57, 58, 66, 69, 70, 78, 81 and 82</LI><LI>July 22nd 2015
</LI><LI>Digested pS 5 and gel purified</LI><LI>Threw away all of the product because I made a mistake on the digestion</LI><LI>Spun down deep wells from yesterday</LI><LI>MIniprepped pRECs 94, 11, 23, 24, 35, 47 and 48</LI><LI>Made a final decision on the saCas9 pRECs that were having ligation issues: start over from growing up pS 1 and PCRing inserts</LI><LI>July 23rd 2015
</LI><LI>Test cut</LI><LI>While they were incubating, I made a gel</LI><LI>Send pRECs for sequencingGrew up pRECs 57(3), 66(1), 69(1), 70(2), 78(1), 81(2), 215(2), 216(1), 220(3)</LI><LI>Cell stocked pRECs 202:210 and 212</LI><LI>Miniprepped pRECs 202:210 and 212</LI><LI>July 24th 2015
</LI><LI>Digest pRECs</LI><LI>Cell stocked pRECs that I grew up from yesterday</LI><LI>Miniprepped the same pRECs I cell stocked</LI><LI>Cut out the band that corresponded to insert 10</LI><LI>Gel purified the DNA in insert 10 band</LI><LI>Sent pRECs for more sequencing 6, 18, 24 and 47</LI><LI>Nanodropped the digested insert</LI><LI>July 25th 2015
</LI><LI>Grew up pRECs 5, 6, 11, 17, 18, 23, 24, 35, 47, 48</LI><LI>Ligated </LI><LI>Gel purified pS 8, 10 and 12</LI></UL><H5>Week Ten - July 27th to August 1st</H5><UL><LI>July 28th 2015
</LI><LI>Jeff picked my colonies on Monday (after 2 days of incubation so some of the colonies were overgrown)</LI><LI>Miniprepped</LI><LI>Analyzed sequencing</LI><LI>July 29th 2015
</LI><LI>Sent pRECs for seuqencing </LI><LI>Grew up pRECs</LI><LI>July 30th 2015
</LI><LI>Miniprepped pRECs</LI><LI>Cell stocked pRECs </LI><LI>Read sequencing</LI><LI>Grew up pRECs 8, 12, 20, 29, 30, 31, 32, 36, 41, 42, 43, 44, 65, 102, 114, 126, 173</LI><LI>Send these pRECs for more sequencing 8, 12, 20, 29, 31, 41, 42, 65, 114, 126</LI><LI>Grow up pS 5, 6, and 7</LI><LI>prRECs have been failing: 6, 13, 14</LI><LI>Abha is making more prREC 5 right now</LI><LI>Abha and Nick are sending pRECs for sequencing with inserts 6, 13, and 14 today, if they’re successful I’ll digest out the inserts from these</LI><LI>Grew up more pS 5 and 6 to digest and PCR purify. </LI><LI>Grew up more pS 7 to digest and PCR purfiy </LI><LI>Ligated </LI><LI>July 31 2015
</LI><LI>Picked colonies for all 16 pRECs</LI><LI>Connot use the pRECs or pS’s I grew up yesterday because I grew them up in LB instead of LB + Carb on accident</LI><LI>Prepared cryotubes for tomorrow</LI><LI>Growing up pRECs from yesterday and also pS 5:7</LI><LI>Made more LB + Carb</LI><LI>Got tubes ready for Monday’s miniprepping</LI><LI>Nanodropped </LI><LI>August 1 2015
</LI><LI>Miniprepped some pRECs </LI><LI>Spin down and froze the rest in -20 freezer</LI></UL><H5>Week Eleven - Auguest 2nd to August 8th</H5><UL><LI>August 3 2015
</LI><LI>Miniprepped the spun down pRECs from Saturday</LI><LI>Jeff grew up pRECs for me</LI><LI>Abha and John’s pRECs 86 and 88 have insert 14 in them and were sequencing verified this morning</LI><LI>Nanodropped backbones that I grew up and miniprepped</LI><LI>Made 60 well gel </LI><LI>Test cut pRECs</LI><LI>Digested backbones pS 5:7</LI><LI>Did a PCR clean up on the backbone digests</LI><LI>Nanodropped the PCR clean-up products </LI><LI>Nanodropped successfully test cut pRECs to send for sequecning tomorrow</LI><LI>August 4 2015
</LI><LI>Set up PCR</LI><LI>Sent 19 pRECs for sequecing</LI><LI>Miniprepped pRECs</LI><LI>Test cut pRECs</LI><LI>Ran the PCR products on a gel and found that well #1 and well #3 didn’t PCR anything. Divya suggested modifying the annealing temperature and changing the time</LI><LI>Gel purified inserts from pREC 86 and 88 but I think I digested them with the wrong thing because the backbones were much longer than they were supposed to be and corresponded to digesting the pRECs with the opposite enzymes that I was supposed to digest them</LI><LI>Sent pREC 67 for sequencing</LI><LI>August 5 2015
</LI><LI>Digested pS 1, 2 pREC 86 and 88 </LI><LI>Grew up 12 pRECs</LI><LI>Sent 5 pRECs for more sequencing</LI><LI>Did a PCR clean-up on the digests </LI><LI>Ligated 29 pRECs and 5 controls</LI><LI>August 6 2015
</LI><LI>Picked colonies</LI><LI>pRECs 101, 158, and 161 didn’t grow</LI><LI>pS 1 and pS 7 had a lot of background </LI><LI>Made reference plates</LI><LI>Grew up pRECs</LI><LI>Ligated pRECs</LI><LI>August 7 2015
</LI><LI>Miniprepped August 5th constructs</LI><LI>Shaheer picked colonies and made reference plates for yesterday’s constructs</LI><LI>Test cut pRECs</LI><LI>Sent pRECs for sequencing</LI><LI>August 8 2015
</LI><LI>Miniprepped pRECS from August 6th 2015</LI><LI>Test cut pRECs </LI><LI>Made a gel </LI><LI>Sent pRECs for sequencing </LI></UL><H5>Week Twelve - August 9th to August 15th</H5><UL><LI>August 12 2015
</LI><LI>Jeff re-ordered prREC 13 and 35</LI><LI>Grew up sequence verified pRECs</LI><LI>Grew up pRECs to send for more sequencing</LI><LI>Resent pRECs for more sequencing</LI><LI>Grew up pRECs for inserts to religate</LI><LI>Figured out the pREC 86, 88 and prREC 14 mix up </LI><LI>Digested test cuts</LI><LI>Cell stocked and miniprepped pRECs 7, 19, 97, 98, 109, 125, 146, 138</LI><LI>Miniprepped pRECs</LI><LI>Abha made a digest mastermix for me</LI><LI>Digested 56, 114, 181, 126 and 173 for 1 hour</LI><LI>Shaheer made a gel to run them on</LI><LI>Miniprepped pREC 182, 174, 120, 157, 150, 122 and 90</LI><LI>Abha nanodropped all of my pRECs for me</LI><LI>Analyzed sequencing</LI><LI>Found that some pRECs need to be sent for more sequencing</LI><LI>Ran digests</LI><LI>Sent all of them for sequencing</LI><LI>Gel purified inserts from pRECs 114, 126, 173 and 181</LI><LI>Autoclaved tips and deep wells</LI><LI>Organized reference plates in the cold room </LI><LI>August 13 2015
</LI><LI>Made a gel to run pS 5 and 7 on </LI><LI>Ran it for 30 min at 400A and 130 mV</LI><LI>Cut out 4 peices </LI><LI>Ligated pRECs</LI><LI>Made a “Conclusions Drawn from the Last 4 days” document to help clarify things while I’m gone</LI><LI>All pRECs with insert 13 are on hold until we get the primer same with prREC 35</LI><LI>Grew up pRECs that are sequence verified 170, 92, 67, 116, 152, 176, 104, 115, 127, 175</LI><LI>All pRECs with insert 13 are on hold until we get the primer same with prREC 35</LI><LI>Grew up pRECs that are sequence verified 170, 92, 67, 116, 152, 176, 104, 115, 127, 175</LI></UL></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>