Team:BostonU/App 2
Home
Research
Controlling Protein Activity
Experimental Design
Controlling Integrases and RDFs
Controlling saCas9
Functional Prototype
Notebook
iGEM Community
Mammalian Synthetic Biology
Interlab Study
Part Submissions
Outreach Initiatives
Education
Collaborations
Industry
Acknowledgements
Our Team
Attributions
Judging
Motivation
Design
Results
The CRISPR/Cas9 method of genome editing is based on natural system used by bacteria to prevent infection. CRISPRs (Clustered regularly interspaced short palindromic repeats) are arranged in an array of identical repeat sequences separated by spacer sequences, and they help confer genetic memory and provide immunity against foreign genetic elements. The CRISPR/Cas9 system utilizes the Cas9 protein, an RNA-guided endonuclease. The Cas9 protein involves the use of a sgRNA, a guide RNA molecule. sgRNA is a short strand of RNA that identifies a complementary sequence of DNA in the entire genome. If the DNA contains a complementary sequence and a PAM (protospacer adjacent motif) downstream, Cas9 will bind to the target sequence and produce a double strand break. Now one of two things happens. The DSB is either repaired through homologous directed repair or non-homologous end joining. Using a donor template, HDR can lead to precise gene modification and NHEJ can lead to indels. Gaining temporal control of Cas9 allows you to increase the efficiency of gene targeting and producing desired effects at a gene of interest. By producing indels, you can efficiently knock out the genes. One important use of this is being able to distinguish between driver and passenger mutations in cancer. It also simplifies the study of oncogenes and tumor-suppressing genes. This system is by no means perfect, as was demonstrated by the Chinese team that recently attempted to genetically edit human embryos. They succeeded in editing only a little more than a quarter of the embryos, and ignited a massive ethical debate. This is why it is important to gain temporal control over Cas9 â€“ eliminating basal activity and creating an inducible response are important in improving the accuracy and efficiency of Cas9. Splitting Cas9 can also allow for the restriction of genome editing to intersections of cell populations. The most popular version of Cas9 is streptococcus pyrogenes cas9. This protein was already split by the Zhang lab, with methods similar to ours. His study revealed that split-cas9 fragments can be used to induce indels without high levels of mutations at off-target sites. So we decided to do the same thing but with saCas9, stapholococcus aureus cas9. We plan to identify good split sites for saCas9 and split the protein and induce activity afterwards. We wanted to use saCas9 because it is over 1kbp smaller than spCas9, allowing you to additional regulatory elements to the vector and making it easier to virally package. We plan to test our split saCas9 using a traffic light reporter. Remember that the DSB produced by Cas9 is repaired by either NHEJ or HDR, and in each case, the TLR will light up in a different color. Originally, neither color is expressed. If the DSB is repaired by NHEJ, the GFP shown here becomes gibberish due to a frame shift, and mCherry will now be in frame, and will be expressed. If the DSB is repaired by HDR, the GFP will be repaired using the GFP template donor, and GFP will be expressed. An important note here is that we care more about the saCas9 actually producing its desired activity and making the DSB, so either color being expressed will prove the success of our induced saCas9. The TLR allows us to not only verify the success, but also characterize the activity of our saCas9. This experiment can lead way to the same applications as split spCas9, but we want to examine the effectiveness of the saCas9 which is significantly smaller than spCas9. What is significant about splitting Cas9 is that you can not only edit genomes in test tubes, but ultimately allow for inducible in-vivo genome editing.
