<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_BostonU_App_2_Design skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:BostonU/App 2/Design</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><UL id="nav"><LI><A href="https://2015.igem.org/Team:BostonU">Home</A></LI><LI><A href="">Research</A></LI><LI><A href="https://2015.igem.org/Team:BostonU/Temporal_Control">Controlling Protein Activity</A></LI><LI><A href="https://2015.igem.org/Team:BostonU/Modeling">Experimental Design</A></LI><LI><A href="https://2015.igem.org/Team:BostonU/App_1/Motivation">Controlling Integrases and RDFs</A></LI><LI><A href="https://2015.igem.org/Team:BostonU/App_2/Motivation">Controlling saCas9</A></LI><LI><A href="https://2015.igem.org/Team:BostonU/Design">Functional Prototype</A></LI><LI><A href="https://2015.igem.org/Team:BostonU/Notebook/Protocols/Bacterial">Notebook</A></LI></UL><LI><A href="">iGEM Community</A></LI><LI><A href="https://2015.igem.org/Team:BostonU/Mammalian_synbio/Significance">Mammalian Synthetic Biology</A></LI><LI><A href="https://2015.igem.org/Team:BostonU/Interlab">Interlab Study</A></LI><LI><A href="https://2015.igem.org/Team:BostonU/Parts">Part Submissions</A></LI><LI><A href="">Outreach Initiatives</A></LI><LI><A href="https://2015.igem.org/Team:BostonU/Education/Building_with_Biology">Education</A></LI><LI><A href="https://2015.igem.org/Team:BostonU/Collaborations">Collaborations</A></LI><LI><A href="https://2015.igem.org/Team:BostonU/Industry">Industry</A></LI><LI><A href="">Acknowledgements</A></LI><LI><A href="https://2015.igem.org/Team:BostonU/Team">Our Team</A></LI><LI><A href="https://2015.igem.org/Team:BostonU/Attributions">Attributions</A></LI><LI><A href="https://2015.igem.org/Team:BostonU/Medals">Judging</A></LI><TABLE align="center" style="margin: 0px auto;"><TBODY><TR><TD><A href="https://2015.igem.org/Team:BostonU/App_2/Motivation" class="button">Motivation</A></TD><TD><A href="https://2015.igem.org/Team:BostonU/App_2/Design" class="button">Design</A></TD><TD><A href="https://2015.igem.org/Team:BostonU/App_2/Results" class="button">Results</A></TD><TABLE><H3>Design</H3><P style="font-family: " trebuchet="" ms="">We applied our split protein pipeline to determine promising split locations for saCas9. We chose 16 split sites, and cloned these into our mammalian expression plasmid backbones that included fusion to our sets of dimerizable domains. Here, we focused on creating FKBP-FRB conditionally dimerizable saCas9 variants, as these domains are small enough to still fit into the AAV limit. We independently tested the functionality of these conditionally dimerizable proteins using a fluorescent reporter plasmid.</P><P>We tested our split saCas9 using a traffic light reporter developed by the Scharenburg lab<SUP>1</SUP>. It was originally designed for the assessment of activity of a different endonuclease, so we designed a sgRNA that would recognize a complementary sequence in the traffic light reporter corresponding to the saCas9 PAM criteria, and would recruit the saCas9 protein to produce a DSB. This DSB could be repaired either through non-homologous end joining (NHEJ) or homology directed repair (HDR) pathways<SUP>2</SUP>.</P><P>For testing the functionality of our conditionally dimerizable saCas9, we used the traffic light reporter containing fluorescent proteins - EGFP and mCherry. Initially, with an inactive saCas9, neither EGFP nor mCherry would be expressed, as there is a premature stop codon within the EGFP sequence. Presence of the inducer would lead to activity of saCas9, such that it would create a DSB.</P><P>If the DSB was repaired by NHEJ, a two base-pair frameshift would occur; the EGFP would be rendered out of frame and would be expressed as a “gibberish” sequence, while the mCherry would be in frame and would be expressed. If the DNA was repaired by HDR, the DSB could be repaired by copying off a correct sequence using a separate EGFP donor template; the EGFP would thus be expressed, while the mCherry would be out of frame and not expressed. This way, not only we could assay for a functional, conditionally dimerizable saCas9, but also to assay its ability to carry out NHEJ and HDR.</P><H4 style="font-size:16px; text-align:center;">Citations</H4><OL style="font-size:12px;"><LI>Scharenberg, Andrew M. et al., “Tracking genome engineering outcome at individual DNA breakpoints”, Nature Methods, 2011.</LI><LI style="padding-bottom:60px;">Cox, David Benjamin Turitz, Platt, Randall Jeffrey, Zhang, Feng, “Therapeutic Genome Engineering: prospects and challenges”, Nature Medicine, 2015.</LI></OL></TABLE></TR></TBODY></TABLE></DIV></DIV></DIV></DIV></BODY></HTML>