<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_BostonU_App_1_Motivation skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:BostonU/App 1/Motivation</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><UL id="nav"><LI><A href="https://2015.igem.org/Team:BostonU">Home</A></LI><LI><A href="">Research</A></LI><LI><A href="https://2015.igem.org/Team:BostonU/Temporal_Control">Controlling Protein Activity</A></LI><LI><A href="https://2015.igem.org/Team:BostonU/Modeling">Experimental Design</A></LI><LI><A href="https://2015.igem.org/Team:BostonU/App_1/Motivation">Controlling Integrases and RDFs</A></LI><LI><A href="https://2015.igem.org/Team:BostonU/App_2/Motivation">Controlling saCas9</A></LI><LI><A href="https://2015.igem.org/Team:BostonU/Design">Functional Prototype</A></LI><LI><A href="https://2015.igem.org/Team:BostonU/Notebook/Protocols/Bacterial">Notebook</A></LI></UL><LI><A href="">iGEM Community</A></LI><LI><A href="https://2015.igem.org/Team:BostonU/Mammalian_synbio/Significance">Mammalian Synthetic Biology</A></LI><LI><A href="https://2015.igem.org/Team:BostonU/Interlab">Interlab Study</A></LI><LI><A href="https://2015.igem.org/Team:BostonU/Parts">Part Submissions</A></LI><LI><A href="">Outreach Initiatives</A></LI><LI><A href="https://2015.igem.org/Team:BostonU/Education/Building_with_Biology">Education</A></LI><LI><A href="https://2015.igem.org/Team:BostonU/Collaborations">Collaborations</A></LI><LI><A href="https://2015.igem.org/Team:BostonU/Industry">Industry</A></LI><LI><A href="">Acknowledgements</A></LI><LI><A href="https://2015.igem.org/Team:BostonU/Team">Our Team</A></LI><LI><A href="https://2015.igem.org/Team:BostonU/Attributions">Attributions</A></LI><LI><A href="https://2015.igem.org/Team:BostonU/Medals">Judging</A></LI><TABLE align="center" style="margin: 0px auto;"><TBODY><TR><TD><A href="https://2015.igem.org/Team:BostonU/App_1/Motivation" class="button">Motivation</A></TD><TD><A href="https://2015.igem.org/Team:BostonU/App_1/Design" class="button">Design</A></TD><TD><A href="https://2015.igem.org/Team:BostonU/App_1/Results" class="button">Results</A></TD></TR></TBODY></TABLE><H3>Motivation</H3><P>Cells naturally integrate environmental stimuli and signals, process changes in their surroundings, and implement responses that enable them to successfully survive and thrive. These natural genetic-regulatory mechanisms can be understood as “genetic circuits”. Cellular behavior and phenotype is intricately linked with protein expression. Cells tune their production levels of various proteins in order to carry out virtually all fundamental processes. Genetic circuits enable cells to regulate protein expression in a logical way.
</P><P>Synthetic biologists are often interested in creating simple yet sophisticated genetic circuits that can enable control of desired cellular behaviors<SUP>1</SUP>. These synthetic circuits can be powerful tools used to accomplish a variety of applications, including “smart” diagnosis and treatment of particular diseases. One interesting example is using cells as conditional therapeutic delivery agents – an engineered cell could detect a wide range of environmental markers associated with tumors and could actuate a conditional response to secrete cyto-toxins in the environment. Furthermore, engineering genetic circuits can enable conditional expression of proteins in specific tissues within mouse models<SUP>2</SUP>.</P><P>There are various ways to engineer genetic circuits that regulate gene expression in cells. One classic method to do this utilizes transcriptional-based regulation, using promoters that express transcription factors which in turn can regulate expression of downstream genes. Promoters that can conditionally express genes, based on the presence or absence of specific transcription factors, are known as inducible promoters. By layering promoter and transcription factor elements, higher order logic circuits can be created<SUP>3</SUP>.</P><CENTER><CENTER><P>Transcriptional-based regulation can result in several undesired issues for genetic circuitry. One primary issue is that nesting logic, such that transcription and translation of circuit inputs, can introduce time delays and slow responses into circuits. Because the system may require a very precise level of input concentrations, it may be difficult to optimally tune a complex system. </P><P>Another fundamental method to regulate gene expression to engineer is the use of recombinase proteins. Recombinase proteins are a class of proteins that can naturally recognize specific DNA sequences called recombination sites and catalyze site-specific genetic manipulation of the DNA sequences between these sites.</P><CENTER><CENTER><P>Synthetic biologists have observed that by flanking a DNA sequence of interest with specific recombination sites in specific orientations, expression of recombinase proteins can manipulate the DNA sequence of interest. Based on the specific recombinase and recombination site sequence, three unique reactions can be catalyzed: inversion, deletion, and cassette exchange. </P><CENTER><NBP><NBP><NBP><NBP><NBP><NBP><CENTER><P>While all of these reactions are directional, they are generally irreversible. However, a class of recombinase proteins called “Large Serine Integrases” (Integrases), which naturally catalyze the inversion reaction, have been identified to be able to catalyze the reverse reaction in the presence of another small protein called a Recombination Directionality Factor (RDF). This is ultimately important because the internal DNA sequence is not lost – it is merely inverted. The ability to switch back and forth between states is a fundamental unit that enables scaling up to dynamic, higher-order genetic circuits.</P><CENTER><CENTER><P>In the literature, there are multiple characterized integrases and RDFs that are each associated with specific recombination sites. Because the sites are unique, these systems are orthogonal to each other, and have powerful implications for creating even more complex logic.</P><P>While the integrase and RDF proteins naturally catalyze this site-specific recombination, there needs to be temporal control of the activity of these proteins in order to regain directional control of the switch. If both are permanently expressed, there is no way to precisely regulate the orientation of the internal DNA sequence.</P><P style="padding-bottom:60px;">This summer, our team hoped to gain tight temporal control over the activity of integrases and RDFs by creating conditionally dimerizable variants. With this method, these proteins could be very useful tools in developing robust genetic circuits.</P><H4 style="font-size:16px; text-align:center;">Citations</H4><OL style="font-size:12px;"><LI>Brophy, Jennifer A. N., Voigt, Christopher A., &quot;Principles of Genetic Circuit Design&quot;, Nature Methods, 2014.</LI><LI>Lewandowski, Mark, &quot;Conditional control of gene expression in the mouse&quot;, Nature Reviews: Genetics, 2001.</LI><LI>Ye, Haifeng, Fusseneggar, Martin, &quot;Synthetic therapeutic gene circuits in mammalian cells&quot;, ScienceDitect, 2014.</LI></OL></CENTER></CENTER></CENTER></NBP></NBP></NBP></NBP></NBP></NBP></CENTER></CENTER></CENTER></CENTER></CENTER></DIV></DIV></DIV></DIV></BODY></HTML>