<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_BostonU_Temporal_Control_Current_Methods skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:BostonU/Temporal Control/Current Methods</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><UL id="nav"><LI><A href="https://2015.igem.org/Team:BostonU">Home</A></LI><LI><A href="">Research</A></LI><LI><A href="https://2015.igem.org/Team:BostonU/Temporal_Control">Controlling Protein Activity</A></LI><LI><A href="https://2015.igem.org/Team:BostonU/Modeling">Experimental Design</A></LI><LI><A href="https://2015.igem.org/Team:BostonU/App_1/Motivation">Controlling Integrases and RDFs</A></LI><LI><A href="https://2015.igem.org/Team:BostonU/App_2/Motivation">Controlling saCas9</A></LI><LI><A href="https://2015.igem.org/Team:BostonU/Design">Functional Prototype</A></LI><LI><A href="https://2015.igem.org/Team:BostonU/Notebook/Protocols/Bacterial">Notebook</A></LI></UL><LI><A href="">iGEM Community</A></LI><LI><A href="https://2015.igem.org/Team:BostonU/Mammalian_synbio/Significance">Mammalian Synthetic Biology</A></LI><LI><A href="https://2015.igem.org/Team:BostonU/Interlab">Interlab Study</A></LI><LI><A href="https://2015.igem.org/Team:BostonU/Parts">Part Submissions</A></LI><LI><A href="">Outreach Initiatives</A></LI><LI><A href="https://2015.igem.org/Team:BostonU/Education/Building_with_Biology">Education</A></LI><LI><A href="https://2015.igem.org/Team:BostonU/Collaborations">Collaborations</A></LI><LI><A href="https://2015.igem.org/Team:BostonU/Industry">Industry</A></LI><LI><A href="">Acknowledgements</A></LI><LI><A href="https://2015.igem.org/Team:BostonU/Team">Our Team</A></LI><LI><A href="https://2015.igem.org/Team:BostonU/Attributions">Attributions</A></LI><LI><A href="https://2015.igem.org/Team:BostonU/Medals">Judging</A></LI><TABLE align="center" style="margin: 0px auto; bgcolor:#E2E2FF;"><TBODY><TR><TD><A href="https://2015.igem.org/Team:BostonU/Temporal_Control" class="button">Overview</A></TD><TD><A href="https://2015.igem.org/Team:BostonU/Temporal_Control/Current_Methods" class="button">Current Methods</A></TD><TD><A href="https://2015.igem.org/Team:BostonU/Temporal_Control/Workflow" class="button">Our Workflow</A></TD></TR></TBODY></TABLE><H3> Current Methods</H3><P>Cells naturally regulate protein expression at various stages of protein expression. Generally, protein activity control in synthetic systems arises as two particular time points: pre-transcription and post-translation. Pre-transcriptional control involves controlling protein activity through protein expression. If a protein is not even transcribed, it is not expressed and thus not active; in contrast, if it is expressed, it has full natural activity. Post-translational control involves controlling activity of a protein after it has been translated - this often uses chemical or genetic modifications that control activity after expression. </P><CENTER><CENTER><P>Pre-translational control can involve controlling protein expression through an inducible promoter, which typically involves a transcription factor interacting with a small molecule to either repress or activate downstream gene expression. For the protein to be expressed, it has to be transcribed and translated. Post-translational control methods can technically yield a faster response to induction by bypassing the process of transcribing and translating the sequence encoding the protein. </P><P>For proteins that manipulate the genome, basal activity can be a big concern. If there was any “accidental” expression a few of these proteins (for example through leaky expression under an inducible promoter), the genetic state can easily be switched since the expressed protein is naturally active. However, using post-translational control, the protein can be expressed in an inactive form and only activated in presence of the inducer. This method also has practical limitations, but optimization may potentially lead to lower basal activity.</P><P>While there are advantages and disadvantages to both of these methods of protein activity control, <U>they are not mutually exclusive.</U> Combining these systems can lead to more intricate genetic manipulation and genome engineering applications. This summer, our team decided to study post-translational control of genetically-modifying proteins and how these might be useful tools for different synthetic biology applications.</P></CENTER></CENTER></DIV></DIV></DIV></DIV></BODY></HTML>