<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_BostonU_App_2_Motivation skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:BostonU/App 2/Motivation</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><UL id="nav"><LI><A href="https://2015.igem.org/Team:BostonU">Home</A></LI><LI><A href="">Research</A></LI><LI><A href="https://2015.igem.org/Team:BostonU/Temporal_Control">Controlling Protein Activity</A></LI><LI><A href="https://2015.igem.org/Team:BostonU/Modeling">Experimental Design</A></LI><LI><A href="https://2015.igem.org/Team:BostonU/App_1/Motivation">Controlling Integrases and RDFs</A></LI><LI><A href="https://2015.igem.org/Team:BostonU/App_2/Motivation">Controlling saCas9</A></LI><LI><A href="https://2015.igem.org/Team:BostonU/Design">Functional Prototype</A></LI><LI><A href="https://2015.igem.org/Team:BostonU/Notebook/Protocols/Bacterial">Notebook</A></LI></UL><LI><A href="">iGEM Community</A></LI><LI><A href="https://2015.igem.org/Team:BostonU/Mammalian_synbio/Significance">Mammalian Synthetic Biology</A></LI><LI><A href="https://2015.igem.org/Team:BostonU/Interlab">Interlab Study</A></LI><LI><A href="https://2015.igem.org/Team:BostonU/Parts">Part Submissions</A></LI><LI><A href="">Outreach Initiatives</A></LI><LI><A href="https://2015.igem.org/Team:BostonU/Education/Building_with_Biology">Education</A></LI><LI><A href="https://2015.igem.org/Team:BostonU/Collaborations">Collaborations</A></LI><LI><A href="https://2015.igem.org/Team:BostonU/Industry">Industry</A></LI><LI><A href="">Acknowledgements</A></LI><LI><A href="https://2015.igem.org/Team:BostonU/Team">Our Team</A></LI><LI><A href="https://2015.igem.org/Team:BostonU/Attributions">Attributions</A></LI><LI><A href="https://2015.igem.org/Team:BostonU/Medals">Judging</A></LI><TABLE align="center" style="margin: 0px auto;"><TBODY><TR><TD><A href="https://2015.igem.org/Team:BostonU/App_2/Motivation" class="button">Motivation</A></TD><TD><A href="https://2015.igem.org/Team:BostonU/App_2/Design" class="button">Design</A></TD><TD><A href="https://2015.igem.org/Team:BostonU/App_2/Results" class="button">Results</A></TD></TR></TBODY></TABLE><H3>Motivation</H3><P>One major long-standing goal of synthetic biology is to be able to successfully engineer and modify biological systems at the most fundamental level. Recent literature has demonstrated many novel technologies that can enable precise genome editing - giving researchers a way to insert, delete, and replace sequences of DNA in the genomes of various organisms as they desire. Genome editing offers many significant downstream applications, and there is considerable excitement about what these emergent technologies can accomplish for the medical field. For example, an individual with a particular genetic disease caused by defective gene could theoretically be cured by replacing their defective allele with a normal allele, instead of having to go through an invasive surgical procedure or taking medications with potentially harmful side effects.</P><P>One of the most easily programmable genome editing tools is the CRISPR/Cas9 system. The CRISPR/Cas9 method of genome editing is based on a natural system used by bacteria to as a mechanism of adaptive immunity to viral infection. CRISPRs (clustered regularly interspaced short palindromic repeats) are arranged in an array of identical repeat sequences separated by spacer sequences, and they help confer genetic memory and provide immunity against foreign genetic elements by cutting up foreign DNA and placing short sequences between the repeats. These short sequences can be expressed in the cell as RNA and can recognize presence of the same viral DNA in a subsequent infection through complementary base-pair binding. The CRISPR system incorporates a protein called Cas9, an RNA-guided endonuclease that will catalyze degradation of such recognized complexes by creating double stranded breaks (DSBs).</P><P>Researchers have engineered the CRISPR/Cas9 system to be able to target sequences of choice in the genomes of many organisms. By expressing single guide RNAs (sgRNA) that can recognize specific genomic sequences through complementary base-pair binding, Cas9 can recognize these sequences and precisely create DSBs, which can enable insertion, deletion, or mutation through natural DNA repair pathways. Mutated variants of Cas9 have also been developed, enabling single stranded breaks (SSBs) and even pure DNA binding and genetic regulation via catalytically-dead Cas9 (dCas9)<SUP>1</SUP>.</P><CENTER><CENTER><P>Gaining temporal control of the Cas9 protein offers several important downstream applications in gene therapy. One particular application that could benefit from a temporally-controlled Cas9 protein is to precisely induce mutations and genetic changes at very specific time points within an animal model; scientists can use this platform to efficiently determine the roles that certain genes play in progression of complex genetic diseases, such as cancer and Huntington’s disease. Precisely inducing timed mutations in an animal’s genome can simplify the study of oncogenes and tumor suppressors. Additionally, temporal control of the Cas9 protein can enable restriction of genome editing activity to certain intersections of cell populations, which is an important tool when studying different cell types and tissue-specific promoters<SUP>2</SUP>.</P><CENTER><CENTER><P>The most widely studied variant of Cas9 is streptococcus pyogenes Cas9 (spCas9). This protein has been used for almost all aspects of CRISPR/Cas9 genome editing technologies, and has even been demonstrated to have been split using conditionally dimerizable systems for temporal control. One limitation with the spCas9 protein is its large size: spCas9 is over 4.3kB long, which is a major limitation for packaging into certain viral vectors, such as the adeno-associated virus (AAV). AAV is a particularly valuable viral packaging delivery method compared to other virus options due to its ability to infect both dividing and nondividing cells and its lack of apparent pathogenicity<SUP>3</SUP>. The AAV vector has a limited cargo size of about 4.7kB; thus, it is difficult to introduce a conditionally dimerizable spCas9 protein into AAV for in vivo genome editing studies. </P><CENTER><CENTER><P>In contrast, a recently identified variant known as staphylococcus aureus Cas9 (saCas9) has been identified to have good endonuclease activity<SUP>4</SUP>. The major advantage of saCas9 is that it is around 3kB long, which means it is roughly 1kB shorter than spCas9. This presents an important advantage, in that it could be packaged more easily into the AAV vector and can even incorporate conditional dimerization for temporal control.</P><P style="padding-bottom:60px;">This summer, our team hoped to gain tight temporal control over the activity of saCas9 by creating conditionally dimerizable variants. With this method, this protein could be a very useful tool in enabling in vivo genome editing applications, particularly for studying disease progression in animal models.</P><H4 style="font-size:16px; text-align:center;">Citations</H4><OL style="font-size:12px;"><LI>Sander, Jeffry D., Joung, J. Keith, “CRISPR-Cas systems for editing, regulating, and targeting genomes”, Nature Biotechnology, 2013.</LI><LI>Zetsche, Bernd, Volz, Sara E., Zhang, Feng, “A split-Cas9 architecture for inducible genome editing and transcription modulation”, Nature Biotechnology, 2015.</LI><LI>Daya, Shyam, Berns, Kenneth I., “Gene Therapy using Adeno-Associated Virus Vectors”, Clinical Microbiology Reviews, 2008.</LI><LI style="padding-bottom:60px;">Zhang, Feng et al., “In vivo genome editing using Staphylococcus aureus Cas9”, Nature, 2015.</LI></OL></CENTER></CENTER></CENTER></CENTER></CENTER></CENTER></DIV></DIV></DIV></DIV></BODY></HTML>