<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_BostonU_Attributions_Citations skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:BostonU/Attributions/Citations</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><UL id="nav"><LI><A href="https://2015.igem.org/Team:BostonU">Home</A></LI><LI><A href="">Research</A></LI><LI><A href="https://2015.igem.org/Team:BostonU/Temporal_Control">Controlling Protein Activity</A></LI><LI><A href="https://2015.igem.org/Team:BostonU/Modeling">Experimental Design</A></LI><LI><A href="https://2015.igem.org/Team:BostonU/App_1/Motivation">Controlling Integrases and RDFs</A></LI><LI><A href="https://2015.igem.org/Team:BostonU/App_2/Motivation">Controlling saCas9</A></LI><LI><A href="https://2015.igem.org/Team:BostonU/Design">Functional Prototype</A></LI><LI><A href="https://2015.igem.org/Team:BostonU/Notebook/Protocols/Bacterial">Notebook</A></LI></UL><LI><A href="">iGEM Community</A></LI><LI><A href="https://2015.igem.org/Team:BostonU/Mammalian_synbio/Significance">Mammalian Synthetic Biology</A></LI><LI><A href="https://2015.igem.org/Team:BostonU/Interlab">Interlab Study</A></LI><LI><A href="https://2015.igem.org/Team:BostonU/Parts">Part Submissions</A></LI><LI><A href="">Outreach Initiatives</A></LI><LI><A href="https://2015.igem.org/Team:BostonU/Education/Building_with_Biology">Education</A></LI><LI><A href="https://2015.igem.org/Team:BostonU/Collaborations">Collaborations</A></LI><LI><A href="https://2015.igem.org/Team:BostonU/Industry">Industry</A></LI><LI><A href="">Acknowledgements</A></LI><LI><A href="https://2015.igem.org/Team:BostonU/Team">Our Team</A></LI><LI><A href="https://2015.igem.org/Team:BostonU/Attributions">Attributions</A></LI><LI><A href="https://2015.igem.org/Team:BostonU/Medals">Judging</A></LI><TABLE class="secondary" style="margin: 0px auto;"><TBODY><TR><TD><A href="https://2015.igem.org/Team:BostonU/Attributions" class="button">Attributions</A></TD><TD><A href="https://2015.igem.org/Team:BostonU/Attributions/Our_Advisors" class="button">Our Advisors</A></TD><TD><A href="https://2015.igem.org/Team:BostonU/Attributions/Citations" class="button">References</A></TD><TD><A href="https://2015.igem.org/Team:BostonU/Attributions/Sponsors" class="button">Sponsors</A></TD></TR></TBODY></TABLE><H3>Citations</H3><P>Ran FA, Cong L, Yan WX, Scott DA, Gootenberg JS, Kriz AJ, et al. In vivo genome editing using Staphylococcus aureus Cas9. Nature. 2015;520:186–191. doi: 10.1038/Nature14299.</P><P>Alex Reis Ph.D, Breton Hornblower Ph.D, Brett Robb Ph.D, George Tzertzinis Ph.D. CRISPR/Cas9 and Targeted Genome Editing: A New Era in Molecular Biology. NED expressions Issue I, 2014.</P><P>Margaret C.M. Smith, Paul C.M. Fogg, Sean Colloms, Susan Rosser, and Marshall Stark. New Applications for Phage Integrases. Review, 2014.</P><P>Protein Splitting Tool , Courtesy of Billy Law and Wilson Wong</P><P>Zhang, Feng et. al. Crystal Structure of Staphylococcus Aureus Cas9 in Complex with SgRNA and Target DNA (TTGAAT PAM). (2015).</P><P>Certo, Michael, et. al.Tracking genome engineering outcome at individual DNA breakpoints Nature Methods 2011</P><P>Brophy, Jennifer A. N., Voigt, Christopher A., &quot;Principles of Genetic Circuit Design&quot;, Nature Methods, 2014.</P><P>Ye, Haifeng, Fusseneggar, Martin, &quot;Synthetic therapeutic gene circuits in mammalian cells&quot;, ScienceDitect, 2014.</P><P>Sander, Jeffry D., Joung, J. Keith, “CRISPR-Cas systems for editing, regulating, and targeting genomes”, Nature Biotechnology, 2013.</P><P>Zetsche, Bernd, Volz, Sara E., Zhang, Feng, “A split-Cas9 architecture for inducible genome editing and transcription modulation”, Nature Biotechnology, 2015.</P><P>Zhang, Feng et al., “In vivo genome editing using Staphylococcus aureus Cas9”, Nature, 2015.</P><P>Liang, Fu-Sen, Ho, Wen Qi, Crabtree, Gerald R., “Engineering the ABA Stress Pathway for Regulation of Induced Proximity”, Sci Signal, 2011.</P><P>Lewandowski, Mark, &quot;Conditional control of gene expression in the mouse&quot;, Nature Reviews: Genetics, 2001.</P><P>Janin, J., “Surface and inside volumes in globular proteins”, Nature, 1979.</P><P>Nureki, Osamu et al., “Crystal structure of staphylococcus aureus Cas9”, Cell, 2015.</P><P>Ho, Steffan N. et al., “Dimeric ligands define a role for transcriptional activation domains in reinitiation”, Nature 1996.</P><P>Liang, Fu-Sen, Ho, Wen Qi, Crabtree, Gerald R., “Engineering the ABA Plant Stress Pathway for Regulation of Induced Proximity”, Science Signaling, 2011.</P><P>Kennedy Matthew J. et al., “Rapid blue-light–mediated induction of protein interactions in living cells”, Clinical Nature Methods, 2010.</P></DIV></DIV></DIV></DIV></BODY></HTML>