"
Page
Discussion
View source
History
teams
Log in
 
Team:USTC-China/Project/application
From 2011.igem.org
Home
Project
Overview
module
results
application
Dry lab
modeling
Riboswitch
Parts
Team
USTC China 2011
members
collaboration
attributions and contributions
acknowledgments
Wet Lab
protocol
notebook
Human Practice
Overview
safety
Activities
China Meetup
Group
Contents
1 Artificial Bacterial Barrier
1.1 Background
1.2 Purpose
1.3 Principle
1.4 Theoretical Design
1.5 Simulated Experiment Design and Results (protocol)
1.6 Conclusion and Discussion
1.7 Future Work
1.8 References
Artificial Bacterial Barrier
 1.Background
 2.Purpose
 3.Principle
 4.Theoretical Design
 5.Expermental Design and Results
 6.Conclusion and Discussion
Background
    From mid May to June 2011, enteropathogenic E.coli brings a big panic to many different European countries. Why such an ordinary bacteria as E.coli can lead to a clinical catastrophe and kill a lot of people? The answer may be complicate, but the most important reason is that we can not find out the pathogens rapidly and can not apply the effective treatments.To solve this problem, we try to use the normal E.coli, which keep the symbiotic relationship with us humanbeings, to be a safety weapon to defend ourselves against pathogens.
Figure1.Enteropathogenic Escherichia coli(left) brings us a big panic(right,Frank Bellew, New York Daily Graphic, September 29, 1873)
Purpose
    Using engineered intestinal bacteria as a safety weapon to destroy the pathogens which invade the intestinal mucosal system.
Principle
    As shown in the Flash below, When pathogens invade the intestinal epithelium ,they will release some kind of chemical signal. After the engineered intestinal bacteria capture this signal, such signal can activate the expression of the function gene ,then it will drive the engineered intestinal bacteria to move toward the infection site from the original colony. Then the moving engineered bacteria will express some lethal proteins at the infection site to kill pathogens by self-destruction, and the original colony will remain.
Theoretical Design
1.AI(Auto Inducer）as a Chemical Signal
    By using the device from the article Engineering microbes to sense and eradicate Pseudomonas aeruginosa, a human pathogen (Figure2.), we can modify our original design to achieve the function which we have described before.(As shown in Figure3.)
Figure2.This image comes from  Nazanin Saeidi.etc (2011).
Figure3.Theoretical design 1
    In this design, the rhlPR is a RHLR/RHL Inducible Promoter, and we can assume the AI comes from pathogen is RHL. So the reprogrammed bacteria with green fluorescence will move toward the the infection site with a high concentration of RHL, while the reprogrammed bacteria with red fluorescence will stay at the original site waiting to creat the second wave of the reprogrammed bacteria with green fluorescence , then the reprogrammed bacteria with green fluorescence will decompose and release the pyosin to destroy the pathogens.
2.Other small molecules as chemical signals
    By using the device from the article Engineering microbes to sense and eradicate Pseudomonas aeruginosa, a human pathogen (Figure 2.), we can modify our original design to achieve the function which we have described before(Figure 4.).
Figure4.Theoretical design 2
    In this design, we assume the small molecule is theophylline. So the reprogrammed bacteria with green fluorescence will move toward the infection site with a high concentration of theophylline and the expression of Lysis becomes more and more, while the reprogrammed bacteria with red fluorescence will stay at the original site and wait to creat the second wave of the reprogrammed bacteria with green fluorescence, they have been protected from being destroyed by the pyosin from the reprogrammed bacteria with green fluorescence. The reprogrammed bacteria with green fluorescence will destroy themselves and release the Pyosin, which is expressed at the beginning of the process, to kill the pathogens.
    Besides, the Lysis protein can be replaced by ccdB protein, and the Pyosin also can be substituted by other proteins of destruction, and the theophylline also can be replaced by other small molecules or catalyzed from other molecules like caffein or amino acid.
Simulated Experiment Design and Results (protocol)
    To put the original Toggle Switch-Aptamer-cheZ in use, we choose the second design above(As shown in Figure 4.) to be the simulated experiment construction(Figure 5.).
Figure5.Simulated experiment construction
    In this design, we use biobrick BBa_K117000 as the Lysis protein, and use biobrick BBa_K117001 to replace the pyosin. In which, the biobrick BBa_K117001 is ColE7-ImmE7 protein complex, and the lysis protein can remove Immunity protein from the complex with ColE7, then ColE7 regains its toxicity and ability to kill the EHEC bacteria or other bacteria nearby. Lysis protein itself can induce the lysis of host cell, exposing the interior cell contents and hence allows free diffusion of ColE7 proteins towards pathogenic bacteria strain.
    Under the consideration of bio-safety, we just use the ordinary E.coli strain with the chloramphenicol and tetracycline resistance as the target bacteia. After the experiment (details are shown in the Protocol), we have some reliable results shown below.
Figure6.The growing state of the target bacteria. Left(after 3h), Right(after 10h)
    Figure6 shows the growing state of the target bacteria as the control of the experiment, in which the red circle represent the range of motion of the target, and from left to right the concentration of the theophylline increases.
Figure7.The growing state of the engineered bacteria(colony1,from tube2) and the target bacteria(colony2). Left(after 3h), Right(after 10h)
    Figure7 shows the growing state of the engineered bacteria(from tube2) and the target bacteria, in which the yellow circle represent the range of motion of engineered bacteria and the red circle represent the range of motion of the target bacteria, and from left to right the concentration of the theophylline increases.
Figure8.The fluorescent Photo(4X Objective)of two colonies shown in Figure9, the engineered bacteria(Left,from tube2) and the target bacteria(Right).
    Figure8 shows two colonies, the colony of the engineered bacteria which keeps the normal form, and the colony of the target bacteria which is deformed and has a hole as a plaque in the middle of it.
                    
Figure9.The growing state of the engineered bacteria(colony1,from tube3) and the target bacteria(colony2)(after 10h)
    Figure9 shows the growing state of the engineered bacteria(from tube3) and the target bacteria, in which the yellow circle represent the range of motion of engineered bacteria and the red circle represent the range of motion of the target bacteria, and from left to right the concentration of the theophylline increases.
Figure10.The fluorescent Photo(4X Objective) of two colonies shown in Figure11, the engineered bacteria(Left,from tube2) and the target bacteria(Right).
    Figure10 shows two colonies, the colony of the engineered bacteria  and the colony of the target bacteria,both of them are deformed and have holes as plaques in the middle of the colony.
Figure11.The SDS-PAGE result of bacteriocin colE7(Blue Triangle:the gradient of thepphylline)
    Figure11 shows bacteriocin colE7 released by the engineered bacteria increase along the theophylline gradient.
Conclusion and Discussion
    Comparing with the control, the reprogrammed bacteria actually can destroy the target bacteria as the design requires and basically decrease the range of motion of the target bacteria.
    By measuring the range of motion of the control and the target bacteria dealed with the reprogrammed bacteria from different tubes(tube1, tube2, tube3, tube4), we get histogram below.
Figure12. capability of eliminating the target bacteria grows with the number of bacteria with red fluorescen decreases.
    Figure12 reveals a fact that the properties of the reprogrammed bacteria mainly depend on the random fluctuation of the expression of the Toggle Switch Device.This fact also explain the differences between the form of the colony1 from tube2(Figure8 left) and the colony1 from tube3(Figure10 Left).
Figure13.The fluctuation of expression of the Toggle Switch device, the number of bacteria with red fluorescent decreases from tube1 to tube4.
    
    Therefore to increase the controllability of this device, we can introduce the  quorum sensing into our design to make the reprogrammed bacteria destroy the target more effectively.
Future Work
    The clinical potential of the application of the synthetic biology is very huge. Such potential includes the development of the therapies for the treatment of infectious diseases and cancer, as well as approaches in vaccine development, microbiome engineering, cell therapy, and regenerative medicine.
    With such background, our Toggle Switch-Aptamer-cheZ Device show us a new way to fight against different diseases.Using our analysis result about Riboswitch , we can standardize the riboswitches or aptamers as Plug-ins ,then our device is just like a kind of "weapon" platform, which uses different standard equipments, such as bacteriocin(just like our design), RNAi(shown in Figure14), or some other metabolites, to cure the diseases and keep us healthy.
Figure14.This image comes from Warren C. Ruder.etc(2011)
References
1.Timothy S. Gardner,  Charles R. Cantor, James J. Collins (2000) Construction of a  Genetic toggle switch in Escherichia coli. Nature, 403,339-342
2.Shana Topp, Justin P. Gallivan (2006) Guiding Bacteria with Small Molecules   and RNA. J.Am.Chem.Soc, 129,6870-6811
3.Chunbo Lou, Xili Liu, Ming Ni.etc (2010) Synthesizing a novel genetic sequential logic circuit: a push-on push-off switch. Molecular Systems Biology, 6:350
4.Joy Sinha, Samuel J Reyes, Justin P Gallivan (2010) Reprogramming bacteria to seek and destroy an herbicide. Nature Chemical Biology, 6,464-470
5.Steven L. Porter, George H. Wadhams, Judith P. Armitage (2011) Signal processing in complex chemotaxis pathways. Nature Reviews Microbiology, 9,153-165
6.Warren C. Ruder, Ting Lu, James J. Collins (2011) Synthetic Biology Moving into the Clinic. Science,333,1248-1252
7.Nazanin Saeidi, Choon Kit Wong, Tat-Ming Lo .etc (2011) Engineering microbes to sense and eradicate Pseudomonas aeruginosa, a human pathogen. Molecular Systems Biology,7:251
Retrieved from " http://2011.igem.org/Team:USTC-China/Project/application "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
