<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_USTC-China_Project_application"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:USTC-China/Project/application">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:USTC-China/Project/application&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:USTC-China/Project/application&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:USTC-China/Project/application&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:USTC-China/Project/application" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:USTC-China/Project/application</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><DIV id="main_container"><DIV class="header"><DIV align="center"><UL id="nav"><LI><A href="https://2011.igem.org/Team:USTC-China">Home</A></LI><LI><A href="#">Project</A></LI><LI><A href="https://2011.igem.org/Team:USTC-China/Project">Overview</A></LI><LI><A href="https://2011.igem.org/Team:USTC-China/Project/module">module</A></LI><LI><A href="https://2011.igem.org/Team:USTC-China/Project/results"> results</A></LI><LI><A href="https://2011.igem.org/Team:USTC-China/Project/application">application</A></LI></UL><LI><A href="#">Dry lab</A></LI><LI><A href="https://2011.igem.org/Team:USTC-China/Drylab/modeling">modeling</A></LI><LI><A href="https://2011.igem.org/Team:USTC-China/Drylab/riboswitch">Riboswitch</A></LI><LI><A href="https://2011.igem.org/Team:USTC-China/Parts">Parts</A></LI><LI><A href="#">Team</A></LI><LI><A href="https://2011.igem.org/Team:USTC-China/Team">USTC China 2011</A></LI><LI><A href="https://2011.igem.org/Team:USTC-China/Team/members">members</A></LI><LI><A href="https://2011.igem.org/Team:USTC-China/Team/collaboration"> collaboration </A></LI><LI><A href="https://2011.igem.org/Team:USTC-China/Team/attributions">attributions and contributions</A></LI><LI><A href="https://2011.igem.org/Team:USTC-China/Team/acknowledgment">acknowledgments</A></LI><LI><A href="#">Wet Lab</A></LI><LI><A href="https://2011.igem.org/Team:USTC-China/Wet Lab/protocol">protocol</A></LI><LI><A href="https://2011.igem.org/Team:USTC-China/Wet Lab/notebook">notebook</A></LI><LI class="nimei"><A href="#">Human Practice</A></LI><LI><A href="https://2011.igem.org/Team:USTC-China/Human practice">Overview</A></LI><LI><A href="https://2011.igem.org/Team:USTC-China/Human practice/safety">safety</A></LI><LI><A href="https://2011.igem.org/Team:USTC-China/Human practice/Activities">Activities</A></LI><LI><A href="https://2011.igem.org/Team:USTC-China/Human practice/China Meetup">China Meetup</A></LI><LI><A href="https://2011.igem.org/Team:USTC-China/Human practice/Group">Group</A></LI></DIV></DIV><FONT size="3" style="font-family: Arial,Helvetica ,sans-serif,'Trebuchet MS';"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1"><A href="#Artificial_Bacterial_Barrier"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Artificial Bacterial Barrier</SPAN></A></LI><LI class="toclevel-2 tocsection-1"><A href="#Background"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Background</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Purpose"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Purpose</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Principle"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Principle</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Theoretical_Design"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">Theoretical Design</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Simulated_Experiment_Design_and_Results_.28protocol.29"><SPAN class="tocnumber">1.5</SPAN><SPAN class="toctext">Simulated Experiment Design and Results (protocol)</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Conclusion_and_Discussion"><SPAN class="tocnumber">1.6</SPAN><SPAN class="toctext">Conclusion and Discussion</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#Future_Work"><SPAN class="tocnumber">1.7</SPAN><SPAN class="toctext">Future Work</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#References"><SPAN class="tocnumber">1.8</SPAN><SPAN class="toctext">References</SPAN></A></LI></UL></TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="Artificial_Bacterial_Barrier">Artificial Bacterial Barrier</SPAN></H1><P><A href="#Background"> <U>1.Background</U></A><A href="#Purpose"> <U>2.Purpose</U></A><A href="#Principle"> <U>3.Principle</U></A><A href="#Theoretical Design"> <U>4.Theoretical Design</U></A><A href="#Expermental Design and Results"> <U>5.Expermental Design and Results</U></A><A href="#Conclusion and Discussion"> <U>6.Conclusion and Discussion</U></A></P><H2><SPAN class="mw-headline" id="Background">Background</SPAN></H2><P>    From mid May to June 2011, enteropathogenic E.coli brings a big panic to many different European countries. Why such an ordinary bacteria as E.coli can lead to a clinical catastrophe and kill a lot of people? The answer may be complicate, but the most important reason is that we can not find out the pathogens rapidly and can not apply the effective treatments.To solve this problem, we try to use the normal E.coli, which keep the symbiotic relationship with us humanbeings, to be a safety weapon to defend ourselves against pathogens.</P><DIV class="center"><DIV class="thumb tnone" style="width:352px;"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption">Figure1.Enteropathogenic <I>Escherichia coli</I>(left) brings us a big panic(right,Frank Bellew, New York Daily Graphic, September 29, 1873)</DIV></DIV></DIV></DIV><H2><SPAN class="mw-headline" id="Purpose">Purpose</SPAN></H2><P>    Using engineered intestinal bacteria as a safety weapon to destroy the pathogens which invade the intestinal mucosal system.</P><H2><SPAN class="mw-headline" id="Principle">Principle</SPAN></H2><P>    As shown in the Flash below, When pathogens invade the intestinal epithelium ,they will release some kind of chemical signal. After the engineered intestinal bacteria capture this signal, such signal can activate the expression of the function gene ,then it will drive the engineered intestinal bacteria to move toward the infection site from the original colony. Then the moving engineered bacteria will express some lethal proteins at the infection site to kill pathogens by self-destruction, and the original colony will remain.</P><H2><SPAN class="mw-headline" id="Theoretical_Design">Theoretical Design</SPAN></H2><P><B>1.AI(Auto Inducer）as a Chemical Signal</B></P><P>    By using the device from the article <I>Engineering microbes to sense and eradicate Pseudomonas aeruginosa, a human pathogen</I> (Figure2.), we can modify our original design to achieve the function which we have described before.(As shown in Figure3.)</P><DIV class="center"><DIV class="thumb tnone" style="width:382px;"><DIV class="thumbinner" style="width:382px;"><DIV class="thumbcaption">Figure2.This image comes from  Nazanin Saeidi.etc (2011).</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone" style="width:562px;"><DIV class="thumbinner" style="width:562px;"><DIV class="thumbcaption">Figure3.Theoretical design 1</DIV></DIV></DIV></DIV><P>    In this design, the rhlPR is a RHLR/RHL Inducible Promoter, and we can assume the AI comes from pathogen is RHL. So the reprogrammed bacteria with green fluorescence will move toward the the infection site with a high concentration of RHL, while the reprogrammed bacteria with red fluorescence will stay at the original site waiting to creat the second wave of the reprogrammed bacteria with green fluorescence , then the reprogrammed bacteria with green fluorescence will decompose and release the pyosin to destroy the pathogens.</P><P><B>2.Other small molecules as chemical signals</B></P><P>    By using the device from the article <I>Engineering microbes to sense and eradicate Pseudomonas aeruginosa, a human pathogen</I> (Figure 2.), we can modify our original design to achieve the function which we have described before(Figure 4.).</P><DIV class="center"><DIV class="thumb tnone" style="width:562px;"><DIV class="thumbinner" style="width:562px;"><DIV class="thumbcaption">Figure4.Theoretical design 2</DIV></DIV></DIV></DIV><P>    In this design, we assume the small molecule is theophylline. So the reprogrammed bacteria with green fluorescence will move toward the infection site with a high concentration of theophylline and the expression of Lysis becomes more and more, while the reprogrammed bacteria with red fluorescence will stay at the original site and wait to creat the second wave of the reprogrammed bacteria with green fluorescence, they have been protected from being destroyed by the pyosin from the reprogrammed bacteria with green fluorescence. The reprogrammed bacteria with green fluorescence will destroy themselves and release the Pyosin, which is expressed at the beginning of the process, to kill the pathogens.</P><P>    Besides, the Lysis protein can be replaced by ccdB protein, and the Pyosin also can be substituted by other proteins of destruction, and the theophylline also can be replaced by other small molecules or catalyzed from other molecules like caffein or amino acid.</P><H2><SPAN class="mw-headline" id="Simulated_Experiment_Design_and_Results_.28protocol.29">Simulated Experiment Design and Results <A href="https://2011.igem.org/Team:USTC-China/Wet_Lab/protocol#Artificial"><FONT size="4"><U>(protocol)</U></FONT></A></SPAN></H2><P>    To put the original Toggle Switch-Aptamer-cheZ in use, we choose the second design above(As shown in Figure 4.) to be the simulated experiment construction(Figure 5.).</P><DIV class="center"><DIV class="thumb tnone" style="width:562px;"><DIV class="thumbinner" style="width:562px;"><DIV class="thumbcaption">Figure5.Simulated experiment construction</DIV></DIV></DIV></DIV><P>    In this design, we use biobrick <A href="http://partsregistry.org/Part:BBa_K117000" class="external text" rel="nofollow">BBa_K117000</A> as the Lysis protein, and use biobrick <A href="http://partsregistry.org/Part:BBa_K117001" class="external text" rel="nofollow">BBa_K117001</A> to replace the pyosin. In which, the biobrick <A href="http://partsregistry.org/Part:BBa_K117001" class="external text" rel="nofollow">BBa_K117001</A> is ColE7-ImmE7 protein complex, and the lysis protein can remove Immunity protein from the complex with ColE7, then ColE7 regains its toxicity and ability to kill the EHEC bacteria or other bacteria nearby. Lysis protein itself can induce the lysis of host cell, exposing the interior cell contents and hence allows free diffusion of ColE7 proteins towards pathogenic bacteria strain.</P><P>    Under the consideration of bio-safety, we just use the ordinary E.coli strain with the chloramphenicol and tetracycline resistance as the target bacteia. After the experiment (details are shown in the Protocol), we have some reliable results shown below.</P><DIV class="center"><DIV class="thumb tnone" style="width:652px;"><DIV class="thumbinner" style="width:652px;"><DIV class="thumbcaption">Figure6.The growing state of the target bacteria. Left(after 3h), Right(after 10h)</DIV></DIV></DIV></DIV><P>    Figure6 shows the growing state of the target bacteria as the control of the experiment, in which the red circle represent the range of motion of the target, and from left to right the concentration of the theophylline increases.</P><DIV class="center"><DIV class="thumb tnone" style="width:652px;"><DIV class="thumbinner" style="width:652px;"><DIV class="thumbcaption">Figure7.The growing state of the engineered bacteria(colony1,from tube2) and the target bacteria(colony2). Left(after 3h), Right(after 10h)</DIV></DIV></DIV></DIV><P>    Figure7 shows the growing state of the engineered bacteria(from tube2) and the target bacteria, in which the yellow circle represent the range of motion of engineered bacteria and the red circle represent the range of motion of the target bacteria, and from left to right the concentration of the theophylline increases.</P><DIV class="center"><DIV class="thumb tnone" style="width:652px;"><DIV class="thumbinner" style="width:652px;"><DIV class="thumbcaption">Figure8.The fluorescent Photo(4X Objective)of two colonies shown in Figure9, the engineered bacteria(Left,from tube2) and the target bacteria(Right).</DIV></DIV></DIV></DIV><P>    Figure8 shows two colonies, the colony of the engineered bacteria which keeps the normal form, and the colony of the target bacteria which is deformed and has a hole as a plaque in the middle of it.</P><P>                    </P><DIV class="center"><DIV class="thumb tnone" style="width:352px;"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption">Figure9.The growing state of the engineered bacteria(colony1,from tube3) and the target bacteria(colony2)(after 10h)</DIV></DIV></DIV></DIV><P>    Figure9 shows the growing state of the engineered bacteria(from tube3) and the target bacteria, in which the yellow circle represent the range of motion of engineered bacteria and the red circle represent the range of motion of the target bacteria, and from left to right the concentration of the theophylline increases.</P><DIV class="center"><DIV class="thumb tnone" style="width:652px;"><DIV class="thumbinner" style="width:652px;"><DIV class="thumbcaption">Figure10.The fluorescent Photo(4X Objective) of two colonies shown in Figure11, the engineered bacteria(Left,from tube2) and the target bacteria(Right).</DIV></DIV></DIV></DIV><P>    Figure10 shows two colonies, the colony of the engineered bacteria  and the colony of the target bacteria,both of them are deformed and have holes as plaques in the middle of the colony.</P><DIV class="center"><DIV class="thumb tnone" style="width:502px;"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Figure11.The SDS-PAGE result of bacteriocin colE7(Blue Triangle:the gradient of thepphylline)</DIV></DIV></DIV></DIV><P>    Figure11 shows bacteriocin colE7 released by the engineered bacteria increase along the theophylline gradient.</P><H2><SPAN class="mw-headline" id="Conclusion_and_Discussion">Conclusion and Discussion</SPAN></H2><P>    <B>Comparing with the control, the reprogrammed bacteria actually can destroy the target bacteria as the design requires and basically decrease the range of motion of the target bacteria.</B></P><P>    By measuring the range of motion of the control and the target bacteria dealed with the reprogrammed bacteria from different tubes(tube1, tube2, tube3, tube4), we get histogram below.</P><DIV class="center"><DIV class="thumb tnone" style="width:502px;"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Figure12. capability of eliminating the target bacteria grows with the number of bacteria with red fluorescen decreases.</DIV></DIV></DIV></DIV><P>    Figure12 reveals a fact that the properties of the reprogrammed bacteria mainly depend on the random fluctuation of the expression of the Toggle Switch Device.This fact also explain the differences between the form of the colony1 from tube2(Figure8 left) and the colony1 from tube3(Figure10 Left).</P><DIV class="center"><DIV class="thumb tnone" style="width:502px;"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Figure13.The fluctuation of expression of the Toggle Switch device, the number of bacteria with red fluorescent decreases from tube1 to tube4.</DIV></DIV></DIV></DIV><P>    </P><P>    Therefore to increase the controllability of this device, we can introduce the  quorum sensing into our design to make the reprogrammed bacteria destroy the target more effectively.</P><H2><SPAN class="mw-headline" id="Future_Work">Future Work</SPAN></H2><P>    The clinical potential of the application of the synthetic biology is very huge. Such potential includes the development of the therapies for the treatment of infectious diseases and cancer, as well as approaches in vaccine development, microbiome engineering, cell therapy, and regenerative medicine.</P><P>    With such background, our Toggle Switch-Aptamer-cheZ Device show us a new way to fight against different diseases.Using our analysis result about <A href="https://2011.igem.org/Team:USTC-China/Drylab/riboswitch" class="external text" rel="nofollow">Riboswitch</A>, we can standardize the riboswitches or aptamers as Plug-ins ,then our device is just like a kind of &quot;weapon&quot; platform, which uses different standard equipments, such as bacteriocin(just like our design), RNAi(shown in Figure14), or some other metabolites, to cure the diseases and keep us healthy.</P><DIV class="center"><DIV class="thumb tnone" style="width:470px;"><DIV class="thumbinner" style="width:470px;"><DIV class="thumbcaption">Figure14.This image comes from Warren C. Ruder.etc(2011)</DIV></DIV></DIV></DIV><H2><SPAN class="mw-headline" id="References">References</SPAN></H2><P>1.Timothy S. Gardner,  Charles R. Cantor, James J. Collins (2000) Construction of a 
Genetic toggle switch in Escherichia coli. Nature, 403,339-342</P><P>2.Shana Topp, Justin P. Gallivan (2006) Guiding Bacteria with Small Molecules   and RNA. J.Am.Chem.Soc, 129,6870-6811</P><P>3.Chunbo Lou, Xili Liu, Ming Ni.etc (2010) Synthesizing a novel genetic sequential logic circuit: a push-on push-off switch. Molecular Systems Biology, 6:350</P><P>4.Joy Sinha, Samuel J Reyes, Justin P Gallivan (2010) Reprogramming bacteria to seek and destroy an herbicide. Nature Chemical Biology, 6,464-470</P><P>5.Steven L. Porter, George H. Wadhams, Judith P. Armitage (2011) Signal processing in complex chemotaxis pathways. Nature Reviews Microbiology, 9,153-165</P><P>6.Warren C. Ruder, Ting Lu, James J. Collins (2011) Synthetic Biology Moving into the Clinic. Science,333,1248-1252</P><P>7.Nazanin Saeidi, Choon Kit Wong, Tat-Ming Lo .etc (2011) Engineering microbes to sense and eradicate Pseudomonas aeruginosa, a human pathogen. Molecular Systems Biology,7:251</P></FONT><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:USTC-China/Project/application">http://2011.igem.org/Team:USTC-China/Project/application</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:USTC-China/Project/application" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:USTC-China/Project/application" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:USTC-China/Project/application&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:USTC-China/Project/application&amp;oldid=258361" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></BODY></HTML>