<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_INSA-Lyon_Notebook"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:INSA-Lyon/Notebook">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:INSA-Lyon/Notebook&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:INSA-Lyon/Notebook&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:INSA-Lyon/Notebook&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:INSA-Lyon/Notebook" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:INSA-Lyon/Notebook</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><P><TITLE>Curly'on - IGEM 2014 INSA-LYON</TITLE></P><DIV id="menu"><UL id="menuul"><LI class="menuulli"><A class="menuullia" href="https://2014.igem.org/Team:INSA-Lyon/Project" id="link-project"><SPAN><P>PROJECT</P></SPAN></A></LI><LI class="iconmenu">PROJECT</LI><LI><A class="hi-icon" href="https://2014.igem.org/Team:INSA-Lyon/Project">PROJECT SUMMARY</A></LI><LI><A class="hi-icon" href="https://2014.igem.org/Team:INSA-Lyon/Achievements">ACHIEVEMENTS</A></LI></UL><LI class="menuulli"><A class="menuullia" href="https://2014.igem.org/Team:INSA-Lyon/Biology" id="link-wetlab"><SPAN><P>WETLAB</P></SPAN></A></LI><LI class="iconmenu">WETLAB</LI><LI><A class="hi-icon" href="https://2014.igem.org/Team:INSA-Lyon/Biology">WETLAB SUMMARY</A></LI><LI><A class="hi-icon" href="https://2014.igem.org/Team:INSA-Lyon/Results">RESULTS</A></LI><LI><A class="hi-icon" href="https://2014.igem.org/Team:INSA-Lyon/Notebook">NOTEBOOK</A></LI><LI><A class="hi-icon" href="https://2014.igem.org/Team:INSA-Lyon/Protocol">PROTOCOLS</A></LI><LI><A class="hi-icon" href="https://2014.igem.org/Team:INSA-Lyon/DataPage">DATA PAGE</A></LI><LI class="menuulli"><A class="menuullia" href="https://2014.igem.org/Team:INSA-Lyon/Modeling" id="link-modeling"><SPAN><P>MODELING</P></SPAN></A></LI><LI class="iconmenu">MODELING</LI><LI><A class="hi-icon" href="https://2014.igem.org/Team:INSA-Lyon/Modeling">MODELING SUMMARY</A></LI><LI><A class="hi-icon" href="https://2014.igem.org/Team:INSA-Lyon/Molecular">MOLECULAR MODELISATION</A></LI><LI><A class="hi-icon" href="https://2014.igem.org/Team:INSA-Lyon/CurliSynthesis">CURLI SYNTHESIS</A></LI><LI class="menuulli"><A class="menuullia" href="https://2014.igem.org/Team:INSA-Lyon/HumanPractice" id="link-human"><SPAN>  HUMAN PRACTICE</SPAN></A></LI><LI class="iconmenu">HUMANPRACTICE</LI><LI><A class="hi-icon" href="https://2014.igem.org/Team:INSA-Lyon/HumanPractice">HUMAN PRACTICE SUMMARY</A></LI><LI><A class="hi-icon" href="https://2014.igem.org/Team:INSA-Lyon/Perception">SYNBIO PERCEPTION</A></LI><LI><A class="hi-icon" href="https://2014.igem.org/Team:INSA-Lyon/Purpose">PROJECT'S INTEREST</A></LI><LI class="menuulli"><A class="menuullia" href="https://2014.igem.org/Team:INSA-Lyon/Team" id="link-team"><SPAN><P>TEAM</P></SPAN></A></LI><LI class="iconmenu">TEAM</LI><LI><A class="hi-icon" href="https://2014.igem.org/Team:INSA-Lyon/Team">MEMBERS &amp; ATTRIBUTIONS</A></LI><LI><A class="hi-icon" href="https://2014.igem.org/Team:INSA-Lyon/Sponsors">SPONSORS</A></LI><LI><A class="hi-icon" href="https://2014.igem.org/Team:INSA-Lyon/Partnership">PARTNERSHIPS</A></LI><LI><A class="hi-icon" href="https://2014.igem.org/Team:INSA-Lyon/Havefun">HAVE FUN WITH IGEM</A></LI><LI><A class="hi-icon" href="https://2014.igem.org/Team:INSA-Lyon/Gallery">GALLERY</A></LI></DIV><DIV id="soustitre" style="background-image: url('https://static.igem.org/mediawiki/2014/d/de/Fond_wetlab.jpg');"><H4>NOTEBOOK</H4><DIV id="icones"><UL><LI class="iconmulti"><A href="https://2014.igem.org/Team:INSA-Lyon/Biology" class="hu-icon">WETLAB SUMMARY</A></LI><LI class="icon"><A href="https://2014.igem.org/Team:INSA-Lyon/Results" class="hu-icon">RESULTS</A></LI><LI class="icon"><A href="https://2014.igem.org/Team:INSA-Lyon/Notebook" class="hu-icon">NOTEBOOK</A></LI><LI class="icon"><A href="https://2014.igem.org/Team:INSA-Lyon/Protocol" class="hu-icon">PROTOCOLS</A></LI><LI class="icon"><A href="https://2014.igem.org/Team:INSA-Lyon/DataPage" class="hu-icon">DATA PAGE</A></LI></UL></DIV></DIV><DIV id="presentation"><P>
Here you can find a summary of all the experiments we made throughout the project.
</P><P> To download a pdf with plasmids' description, resistance and names click <A href="https://static.igem.org/mediawiki/2014/9/98/Plasmids.pdf" target="_blank">here</A></P><P> To download a pdf with the strains' description, resistance and names click <A href="https://static.igem.org/mediawiki/2014/1/14/Strains.pdf" target="_blank">here</A></P><UL style="list-style-type: none !important;"><LI><A href="#contenu1" onclick="$('#contenu1').slideToggle('slow')"><H1>February-May</H1></A></LI><UL id="contenu1" style="list-style-type: none !important;display:none;"><LI><P>Literature searches were performed about Peptide Display, Curli biogenesis and metal trapping.</P></LI></UL><LI><A href="#contenu2" onclick="$('#contenu2').slideToggle('slow')"><H1>June</H1></A></LI><UL id="contenu2" style="list-style-type: none !important;display:none;"><LI><H6>11/06 </H6><P> Reception of the four DNA constructions: CsgA Wild type, CsgA PolyHis1, CsgA PolyHis2, CsgA Modular</P><H6>24/06 </H6><P> NM522 and DH5α cells (in liquid LB medium) were cultured overnight under agitation at 37°C for later transformation. </P><H6>25/06 </H6><P>Transformation of NM522 and DH5α strains with each one of the plasmids: pHC01-09  </P><P>Selection of transformed bacteria on antibiotic plates depending on the plasmid.  </P><H6>26/06 </H6><P>Transformation analysis: all the transformations were successful except the ones with plasmids pHC03, pHC06, pHC07 and pHC09. Transformation of these four plasmids was repeated with a new protocol. </P><P> Clones of transformed strains containing pHC01, 02, 04, 05 and 08 were cultured in 5 mL LB medium containing 50 µL of the specific antibiotic at 37°C and appropriate plates were streaked with isolated colonies likely to contain transformed bacteria. </P><H6>27/06 </H6><P>Transformation analysis: all the transformations were successful and plates containing pHC06 plasmids showed pink bacteria, as the rfp in the plasmid is expressed in presence of a strong promoter (like J23100). </P><P>A protocol for nickel titration was searched and nickel was stored for later experiments. </P><H6>30/06 </H6><P> Extraction of the different plasmids with mini prep ABC protocol.</P></LI></UL><LI><A href="#contenu3" onclick="$('#contenu3').slideToggle('slow')"><H1>July</H1></A></LI><UL id="contenu3" style="list-style-type: none !important;display:none;"><LI><H6>1/07 </H6><P>Transformation of the NM522 strain with the four constructions from Genecust : pHC10-13</P><H6>2/07 </H6><P> Digestion of all plasmids with restriction enzymes (except pHC05 which was incorrect, with one gene too many)</P><P> Electrophoresis gel test revealed no difference between the wells. Hypothesis: problems with enzymes or solutions A and C in mini prep ABC protocol. 
</P><P>A calibration curve of nickel was made using different concentrations of metal and DMG.</P><H6>3/07 </H6><P>Strains containing the 9 plasmids were cultured in 5mL of LB medium + 50 µL specific antibiotic at 30°C.  </P><P>Midi prep kit tested with pHC06. The kit was functional.  </P><P>Gel test of pHC06 and nanodrop were both used to determine pHC06 concentration. </P><H6>4/07 </H6><P> Storage of some strains such as ompR324 or csgA- that might be interesting for the project were put in storage. </P><P> New liquid cultures of the plasmids were incubated at 37°C this time. </P><H6>5/07 </H6><P> Mini prep ABC solution tests: pHC06 was extracted with different combinations of solutions A and C and then digested. Solution C seemed to be the problem in previous ABC miniprep protocol.</P><P> Transformation of NM522 with pHC05.</P><H6>8/07 </H6><P> Midi Prep of pHC01, pHC02, pHC04, pHC07, and pHC08.</P><P> Digestion of plasmids with appropriate restriction enzymes </P><P> Gel test results: </P><P> - pHC01 : no digestion</P><P> -	pHC02 and 04 : digestion </P><P> -	pHC07 : the simple digestion wasn’t enough to conclude</P><P> -	pHC08 : 2 digestions with contradictory results</P><H6>10/07 </H6><P> Digestion of pHC06 and pHC07 with N+P. Gel test showed digestion worked for pHC06 but not for pHC07.</P><P> Midi prep of Genecust constructions.</P><P> Determination of plasmids concentrations by nanodrop and by gel analysis. </P><H6>11/07 </H6><P> Digestions of psB1C3, pHC06, pHC07 and the four DNA constructions with appropriate enzymes for ligation.</P><P>Electrophoresis gel test: the gel was too thin and DNA wasn’t enough concentrated so conclusions about digestions couldn’t be clearly drawn. </P><P> Ligations:</P><P> -	3A ligation of pHC06 with psB1C3 and each of the four Genecust constructions (csgA WT, His1, His2, mod)</P><P> -	Standard ligation of pHC07 with each of the four DNA constructions (csgA WT, His1, His2, mod)</P><P> Transformation of DH5α strain with the assembled plasmids and selection on a Cm plate.</P><H6>14/07 </H6><P>Extraction with mini prep ABC protocol of 120 clones in total of the transformed strains.  </P><H6>15/07 </H6><P> Simple digestion of all clones with EcoRI and verification by gel electrophoresis revealed no successful cloning.</P><H6>16/07 </H6><P> Extraction with mini prep ABC of 4 new clones of pHC15 and pHC18 but and two clones seemed promising after verification with gel electrophoresis.</P><P><U>Dry lab: </U>A new strategy for cloning was decided: gel purification of the inserts would be carried out after digestion, and assay test of phosphatase to eventually use it before ligation. </P><P> Digestions of pHC04 (from midi prep), pHC07 and 3 DNA constructions (Wild Type, His1X and His2X) </P><P> pHC07 was used to test phosphatase action : a control with 1,5 µL DNA and 8,5 µL water was made to see if some digestions were incomplete.  Phosphatase was then inactivated 20 minutes at 80°C.</P><P> Verification with gel electrophoresis: </P><P> -	Genecust constructions : successful digestion</P><P> -	pHC04 : no digestion</P><P> -	pHC07 : no digestion </P><P> Hypothesis to understand digestion failure: maybe one NEB 2 buffer was out of date.</P><H6>17/07 </H6><P> Extraction with mini prep kit of these 2 clones: gels suggested that potentially good clone of pHC20 didn’t contain the plasmid and that the clone of pHC15 contained two vectors and the promoter but no Genecust construction.</P><H6>25/07 </H6><P> Verification of buffers number 2 used for digestion: gel electrophoresis showed that problems of DNA digestions weren’t linked with buffer 2 as digestion worked with all buffers.</P><P> Ligation: 3A assembly of psB1C3, pHC06 and CsgA Wild Type (plasmid pHC14) from former digestions. </P><P> Verification by gel electrophoresis of:	</P><P> -	pHC14 (ligase + insert before and after ligation) : failure</P><P> -	pHC07 (with and without digestion) : digestion of pHC07 seemed to have failed</P><P> -	pHC09 (with and without digestion) : digestion seemed complete</P><P> Transformations in DH5 α and selection on Cm plates of: </P><P> -	pHC14 (ligase + insert; ligase control and control) : hundreds of clones on “ligase + insert” plate against 10 more times on “ligase without insert” -&gt; maybe due to an overproduction of CsgA (J23100 is a strong promoter) that kill bacteria ?</P><P>-	pHC07 (with and without digestion) -&gt; tests to check if the digestion was complete or not : confirmation of gel electrophoresis </P><P> -	pHC09 (with and without digestion) -&gt; tests to check if the digestion was complete or not : 7 clones after digestion against about 200 without digestion</P><P> Gel purification of pHC 10, 12 and 13: high amounts of DNA were obtained (even if there were some loss too) as the gel was overloaded.</P><P> Digestion of pHC11 (CsgA Modular) with XbaI and SpeI for future gel purification.</P><P> Digestion of pHC03 with BamHI and SpeI to test again the action of phosphatase enzyme.</P><P> Liquid preculture of pHC04 was prepared for next extraction with midi prep kit.</P><H6>19/07 </H6><P>200mL liquid culture of pHC04 (dilution 1/1000 of preculture)  </P><H6>20/07 </H6><P>Extraction of pHC04 with midi prep. </P><P>Digestion of pHC04 with E + P and verification by gel electrophoresis : one out of four replicates was digested.  </P><P>Transformations analysis:  </P><P> -	pHC07 : confirmation of digestion failure</P><P> -	pHC09 : 5 clones (against about 100 on control plates)</P><P> Preculture of 15 clones from pHC14 ligations was prepared for next mini prep.</P><H6>21/07 </H6><P>DNA extraction of pHC14 clones with mini prep ABC protocol. </P><P> Digestion of the 15 clones with N (for those assembled in pHC06) and E (for the rest) and verification by gel electrophoresis : no clone seemed to contain the plasmid.</P><P> Gel purification of pHC04 and pHC11 (concentrations with nanodrop): </P><P> -	pHC04 : 2,3 ng/µL</P><P> -	pHC11 : 22,7 ng/µL</P><P> Digestion of pHC04 with E + P and verification by gel electrophoresis: successful digestion.</P><P> Ligations of pHC09 and pHC12, transformation and selection on plates with Tet antibiotic.</P><P> Digestion of pHC05 with E + S.</P><H6>22/07  </H6><P> Storage of some Keio strains (E Coli 12 with one muted gene like CsgA- or CsgB-).</P><P> Transformation results of pHC09 and pHC12: abou.t 50 white clones on control plate and 2 pink clones on plate with ligase.</P><P> Verification by gel electrophoresis of pHC25 ligation and pHC05 digestion: ligation worked (a strip associated with a size of 3000 bp was seen) and digestion of pHC05 was complete for three different Eppendorf.</P><P> Test of phosphatase enzyme: Digestion of pHC03 with E followed by ligations (with and without phosphatase) </P><P> New digestion of pHC04 with E + P.</P><P> Transformation of DH5α with pHC25.</P><H6>23/07  </H6><P>DNA extraction with ABC mini prep of 24 c transformed clones with pHC25 ligations: few amounts of DNA were extracted because cultures were incubated for 7 hours and only 1,5mL of them were used for mini prep. </P><P>Digestion with E of the 24 clones and verification by gel electrophoresis: 2 clones seemed to contain pHC25. </P><P> Ligations: Standard assembly of pHC05 and pHC10 followed by transformations selected on plate.</P><H6>24/07  </H6><P> Ligations: Standard assembly of pHC05 with pHC10-11-13.</P><H6>25/07 </H6><P> DNA extraction with mini prep kit of the 2 clones that seemed to contain pHC25 and digestions with P and E+P followed by verification by gel electrophoresis: clones were found good.</P><P>Digestion of pHC04 with E+P and pHC05 with E+S for next 3A ligations. </P><P>DNA extraction with ABC mini prep of 2 clones of pHC27, digestion with E and verification by gel electrophoresis: one of the two clones seemed promising. New digestions with P and E+P were then made and proved that the clone was the good one. </P><P> Transformations of DH5α with pHC23, 24 and 26. </P><H6>27/07 </H6><P> 8 clones of each one of the plasmid pHC23, 24 and 26 were cultured.</P><H6>28/07 </H6><P>DNA extraction with ABC mini prep of pHC23, 24 and 26 clones. </P><P>Digestion of pHC23, 24 and 26 with E and verification by gel electrophoresis. </P><P> Digestion of pHC08 with N + P and culture of 200 mL of strains containing this plasmid for next digestion.</P><P> DNA extraction with mini prep kit of pHC27 from verified clone.</P><H6>29/07</H6><P>Midi prep of pHC08. </P><P> DNA extraction with mini prep kit for 3A assemblies (pHC23, 24 and 26) and standard ones (pHC27), followed by digestions with E + P and verification by gel electrophoresis that showed successful digestions.</P><P> Transformations of pHC23, 24, 25 and 26 in appropriate strains for characterization: </P><P>-	s226 (CsgA-) which is fluorescent </P><P> -	s204 (CsgA WT) , S227 (CsgA-)</P><P> Preparation of Congo Red plates for quantitative test of adherence with Curli.</P><P> Qualitative test of bacteria adherence with crystal violet method: strains containing wild type csgA (s225, s204 and s216) showed high percent of adherence compared with CsgA- strains (s226, s227 and s216).</P><P> Different aliquots of nickel were prepared to test pH influence for next dosages with DMG.</P><P> Preculture of 5 clones of pHC28, 29, 30 for next mini prep.</P><H6>30/07  </H6><P>Transformations of pHC23, 24, 25 and 26 in appropriate strains for characterization: s211 (CsgA wild type) and -	s216 (CsgA-)</P><P>Digestion of pHC08 with N and N+P and verification by gel electrophoresis: pHC08 was thus verified. </P><P>Culture of strains s227 and s204 on LB plates and M63 liquid medium. </P><P> DNA extraction with mini prep kit for standard ligations (pHC28, 29 and 30), digestion with E and verification by gel electrophoresis: 1 clone containing pHC29 and three containing pHC30 were found.</P><P>New qualitative test of bacteria adherence with crystal violet method: similar results as the previous day were observed. </P><P> Characterization of strains s225, s226 and s227:</P><P> - Determination of delay </P><P> - Determination of generation time</P><P> - Determination of OD=f(fluorescence)</P><H6>31/07 </H6><P> Digestion of three clones transformed with pHC30 assemblies with E+P and verification by gel electrophoresis: the three clones contained the plasmid.</P><P> Ligations of pHC35, 36, 37, 38 and of pHC31, 32, 33, 34.</P><P> Transformations of s216 and s204 with pHC23-26.</P><P> DNA extraction of pHC30 with mini prep kit and of pHC28 with ABC miniprep.</P><P> Storage of strains s204, S227 and s260-267.</P><P> Adherence test in a 24 wells plate, strains s227 and s224 complemented with pHC23, 24, 25, 26 and not transformed were cultured at 30ºC.</P></LI></UL><LI><A href="#contenu4" onclick="$('#contenu4').slideToggle('slow')"><H1>August</H1></A></LI><UL id="contenu4" style="list-style-type: none !important;display:none;"><LI><H6>1/08  </H6><P> Digestion of clones that seemed to contain pHC30 with E+P and of pHC28 with E and verification by gel electrophoresis proved clones contained pHC30 and digestions with E+P were made for 2 clones likely to contain pHC28.</P><P>6 clones containing plasmids pHC35, 36, 37, 38 were cultured.  </P><P> Strains s211 and s216 with pHC23, 24, 25, 26 were put into storage.</P><P> Adherence test of strains s227 and s224 complemented with pHC23, 24, 25, 26 and not transformed.</P><P> Bacteria transformed with plasmids pHC35, 36, 37, 38 covered all the plate, so no isolated colonies were observed. New plates were streaked to obtain isolated colonies.</P><P> Liquid cultures of s227 were incubated at 30ºC on Petri dishes containing M63 mannitol and Tc. Some of them contained plastic materials in order to see if the biofilm also adheres to the materials.</P><P> Liquid cultures of s227 + pHC26 (His2) were incubated at 30ºC set on Petri dishes containing M63 mannitol and Cm.</P><H6>3/08 </H6><P> Precultures of clones obtained after ligation of pHC31-38 were prepared (37ºC, 300 rpm)</P><H6>4/08 </H6><P> Minipreps of clones transformed with pHC31-38 assemblies were done and digested with EcoRI. Clones containing pHC35, pHC37 and pHC38 were obtained but none were found containing pHC31-34 and pHC37.</P><P> Cultures of strains 204, 227, 211, 207 and 260-275 on Petri dish (LB, Congo red+ LB/4 and M63+mannitol and congo red) Cm for transformed strains and Tc for the parental strains were started.</P><P> Adherence test of strains 204, 227, 211, 207 and 260-275 with crystal violet.</P><P> Calibration set of Nickel and DMG.</P><P> Precultures of strains 225, 226, Keio csgB-, 211, 216, 207 and 227 in LB milieu.</P><P> Plastic materials that had been cultured with bacteria over the weekend were “baked” at 100ºC for 30min and then coloured with Cristal violet. Results showed bacteria had developed and adhered on the plastic surface.</P><H6>5/08 </H6><P>Precultures of 6 new clones of pHC36 were prepared, then miniprepped. </P><P> Digestions with PstI and EcoRI of clones containing potentially f pHC35, 36 and 38.</P><P> Ligation and transformation of pHC28 in DH5α.</P><P> Microscopy tests of strains 227 and 204 marked with ThioflavineS were started but fluorescence was only observed in ethanol but not in water.</P><P> Transformations of:</P><P> -	strains 225 and 226 with pHC35, 36 and 39 (AmpR plasmids)</P><P> -	strain keio csgB- with pHC29 and 30 (for purification)</P><P> -	strains 204, 227 with pHC27, 29 and 30 (CmR plasmids)</P><P> 5mL precultures of strains 204, 227 and 260-267 (for Congo red tests) and 216, 211 and 268-275 (for adherence tests) were started.</P><P> Strains s227, 267 were cultured with NiSO4.</P><H6>6/08 </H6><P>Digestion of minipreps of pHC37 with EcoRI. Gel test revealed clones that seemed to contain pHC37. This was confirmed afterwards by double digestion with EcoRI and PstI and gel. </P><P> Precultures of clones containing pHC28.</P><P> Quantification of curli in strains s204, 227, 260, 261, 262, 263, 264, 265, 266, 267 with congo red procedure.</P><P> 36 wells plates were inoculated for adherence test.</P><P> Microscopy tests of strains 211 and 216 marked with ThioflavineS on 96 wells plate.</P><P> Strains 227, 267 were assayed for biofilm nickel absorption on Petri dish cultures using the calibration set. Both biofilms absorbed Nickel but no difference between csgA WT and csgA His2 was observed.</P><H6>7/08 </H6><P>Minipreps of pHC28 and digestion with EcoRI. Test gel revealed clones that potentially contained pHC28. This was then confirmed by double digestion with EcoRI and PstI gel check. </P><P> Transformation of strains s225, s227 with pHC37 and strains s204, s227 with pHC28.</P><P> Reception of antibodies from Life Technologies and UV lamp.</P><P> Adherence tests of s216 and 216 and derivatives.</P><P> Fluorescence assays of strains 211 and 216with thioflavine S. Procedure failed and a new protocol was developed. </P><P> Slides were inoculated with strains 211, 216, 225 (30ºC in M63+ mannitol) for thioflavineS test and 225 for epifluorescence.</P><P> Strains 227, 267 were assayed for biofilm nickel absorption on liquid cultures using the calibration set. Both biofilms absorbed Nickel but still no difference between csgA WT and csgA His2 was observed.</P><P> W3110 transformed strains were streaked in Cm red congo plates to check for presence of plasmids. Bacteria grew, so plasmids were present but red congo didn’t stain the colonies which means there was no expression of curli with promoter lac. From now on, we’ll focus on strains transformed with promoter curli plasmids.</P><H6>8/08 </H6><P>Fluorescence and epifluorescence assays of strains 204 and 227 with thioflavine S. Thioflavine S tests didn’t seem to work on microplate reader, maybe because the excitation wavelength is not adapted for thioflavine S fluorescence. </P><P> 50mL of strains s227, s263 and s267 were cultured for comparative Nickel test.</P><P> Induction assay of strains 226 and 225 on LB, LB + IPTG, M63 and M63 + IPTG to check induction of plac in order to see if there is curli formation. Result: Curli wasn’t formed so we abandoned the idea of using plac.
</P><H6>11/08 </H6><P>Strains 204, 227 and 260,261, 262, 263, 264, 265, 266, 267 were streaked on Tet and Cm plates respectively to obtain isolated colonies. </P><P> Digestion of pHC01 with EcoRI and PstI.</P><P> UV lamp was tested with diluted samples at different concentrations of:</P><P> - strain 227 cultured at 30°C 200 rpm and 37°C 300rpm</P><P> - strain 204 cultured at 30°C 200 rpm</P><P> The samples were exposed to UV light during 1, 5 and 15 min and then were grown in plates at 37°C overnight. UV light killed all bacteria for exposure times over 5 min, only 2 colonies grew after 1 min exposure.</P><P> Baclight kit test to see if it was functional.</P><P> Visual comparative test of nickel between strains s227, s263 and s26. A decolouration was observed between the control (s227) and strains s263 and s267 but no difference in decolouration was observed between s263 (csgA WT) and s267 (csgA His2). This might be due to an acidification of the media that could lead to modification of His charge. </P><P> Strains 204, 227, 260,261, 262, 263, 264, 265, 266 and 267 were streaked on 2 red congo plates and incubated each one at two temperatures (30ºC, and 37ºC) to check for difference of expression of curli promoter at different temperatures. </P><P> =&gt; Results were not as expected: there was no difference of expression between the two temperatures and some strains seemed to have lost the plasmid, because they weren’t stained with congo red. It was decided that a new transformation of bacteria would be done the day after.</P><H6>12/08 </H6><P>Gel test revealed pHC01 (pcurli AmpR) was entirely digested. </P><P> Standard ligation of pHC01 with pHC10, 11, 12, 13 (AmpR plasmids for fluorescent strains s211 and s216) and transformation in DH5α. Gel test to check ligation for fluorescent strains. </P><P> Double transformations of strain 217 (csgB-) with plasmids pKK-csgD (overexpressing csgD) or pKK-OmpR234 and plasmid pHC25 to check which one of the pKK plasmids is better for csgA His1 purification. Only a double transformation clone containing pKK-csgD and pHC25 (His2) was obtained, for the others, no clones were observed in the double antibiotic plate. </P><P> The remaining culture of the double transformed strains of 217 were streaked on Amp plates for clones containing at least one of the pKK plasmids (pKK-csgD and pKK-OmpR234).</P><P> Transformation:</P><P> -	Strain 204 with pHC23, pHC25, pHC26</P><P> -	Strain DH5α with psB1C3+gfp (for stock)</P><P> Strains 204, 227 and 260,261, 262, 263, 264, 265, 266, 267 were streaked this time on 2 red Cm congo plates and incubated 30ºC, and 37ºC to check again for difference of expression of curli promoter at different temperatures. The same results were observed.</P><P> Number of dead and alive cells in a liquid culture of strain 227 after UV light exposure during 1 and 5 minutes was determined with Baclight (with epifluorescence). UV light killed bacteria effectively but some bacteria seemed still alive. Cultures were conserved after exposure to check for alive bacteria the day after.</P><P><U>DRY LAB:</U> Design of primers for characterization of pcurli promoter.</P><H6>13/08 </H6><P> Transformation of strain 227 with pHC23, pHC25, pHC26.</P><P> 3 colonies from each one of the transformed 204 strains was streaked on congo red plates to look for the clone that forms the more curli and the same 3 clones were inoculated in Eppendorfs containing LB.</P><P> 5mL LB medium tubes were inoculated with clones of pHC42, 43, 44, 45 for miniprep.</P><P> Isolation DNA from UV light exposed bacteria by sonication and gel test to check for DNA degradation.  A smear between 50 and 2000 bp indicating DNA degradation was observed in all the UV light exposed DNA but also in the unexposed control, maybe due to a strong sonication.  Fragments were too big and could still contain some genes that could be released in the environment and taken up from bacteria, so we needed to reduce the DNA fragments by exposing the bacteria to UV light during a longer time.</P><P> Development of red congo tests: 10 µL drops of different concentrations of strains 204, 227 and 260-264 were spotted on a congo red plate in order to see up till which concentration the colonies were stained. </P><P> pH check of bacterial colonies. pH=6 was observed, which might cause a change in histidine motifs, and could explain the results that showed no better chelation of nickel.</P><P> Inoculation of a 96 wells plate with:</P><P> -	Strains 225 and 226 (for fluorescence tests)</P><P> -	Strains 204 and 227 (for baclight test) </P><P> -	« Old » and « new » transformed strains 260 and 263 cultured with thioflavine S </P><P> Results :</P><P> -	Fluorescence tests were successful and the backlight test was able to colour all the cells even in thick biofilms.</P><P> -	Thioflavine S test successful in microscopy, the media didn’t absorb much (low background fluorescence) and seemed to be specifically attached to curli. </P><P> Strains 204 and 227 were cultured in liquid M63 with congo red at 2 different temperatures (30ºC and 37ºC) at low agitation. As expected, strains 204 and 227 only produced curli at 30ºC.</P><H6>14/08 </H6><P> Minipreps of pHC42, 43, 44, 45 and digestion with EcoRI. Test gel of digested plasmids revealed clones that seemed to contain the plasmid in all cases.</P><P> Miniprep Kits of pKK-csgD and pKK-OmpR234.</P><P> Precultures of :</P><P>-	2 clones of strain 306 (DH5α containing pKK) </P><P>-	2 clones of strain DH5α containing psB1C3+gfp (for stock) </P><P> -	1 clone of strain 305 (217 containing pKK-ompR234 and pHC25) </P><P> -	1 clone of strain 304 (217 containing pKK-csgD and pHC25) </P><P> 3 colonies from each one of the transformed 227 strains were streaked in congo red plates to look for the clone that forms the more curli and the same 3 clones were inoculated in Eppendorfs containing LB.</P><P> Strains 217 (csgB-), 225 and s226 (csgA-) were cultured for future complementation test.</P><H6>15/08 </H6><P> Digestion of pHC40 (pKK-csgG), 41 (pKK-OmpR234) with EcoRI and pHC42-45 with both EcoRI and PstI and test gel revealed some clones contained the right plasmids.</P><P> Nickel test on liquid bacterial cultures of strains 204, 264 and 267 after basifying with TRIS buffer to keep the pH of the solution around 8. Results obtained were not as expected. It seemed TRIS buffer competed with DMG and interfered with the complexation reaction with DMG. Another method had to be used to control the pH of the solution next time.</P><P> CsgA purification: strain305, and strain 217 cgB- transformed with pHC25 only were cultured in order to produce csgA His and WT in the external media. </P><P> Transformed bacteria containing pKK and psB1A2-gfp was streaked again on Amp plates and incubated at 37ºC over the weekend. Only satellite clones were obtained.</P><H6>18/08 </H6><P> Nickel test on liquid bacterial cultures of strains 204, 227, 264, 265, 266 after basifying the bacterial culture with NaOH to keep the pH of the solution around 8. All biofilms absorbed Nickel but this time a difference between csgA WT and csgA His2 was observed. His 2 chelated more nickel than the WT.</P><P>3 of the congo red stained clones were isolated from the transformed strains 264, 226, 26. Each one of the 3 clones were inoculated with 5mL M63 tubes and also streaked on congo red plates to see which one was the best and prepare cultures for adherence and liquid congo red tests. </P><P> 5mL M63 tubes were inoculated with strains 204 and 227 for adherence and liquid congo red tests.</P><P> The old transformed strains from the storage were streaked on Congo red to see if the plasmids were still present in the stored strains.</P><P> Strains transformed with pKK and psB1A2-gfp were streaked on Amp plates and inoculated on liquid cultures of LB containing Amp or not to check for AmpR problems. Also, DH5α was streaked on Amp and LB plates to check for contaminations. The experiment revealed a problem in Amp plates: strains cultured in liquid Amp cultures didn’t grow but they grew on the Amp plates. Also, DH5α grew on all plates (Amp and LB). New plates were prepared.</P><P> Digestion of pHC40 (pKK-csgG) and pHC41 (pKK-OmpR234) with EcoRI and PstI and test gel revealed clones containing the right plasmids. </P><P> Supernatant of strains 305 and 217 (cgB-) transformed with pHC25 was purified with a NiNTA column to obtain the csgA in the media.</P><P> Complementation test for csgA purification: Another fraction of the strain 305 supernatant was inoculated in the strain 266 culture (csgA-) containing thioflavine S. Curli formation was assayed to see which transformed 217 strain was better for production of csgA for purification (qualitatively).</P><P> =&gt; The complementation test didn’t show a difference in thioflavine S colouration, so no curli formation was revealed.</P><H6>19/08 </H6><P> The transformed bacteria containing pKK and psB1A2 was streaked on new Amp plates.</P><P> SDS-PAGE gel was used to check purified csgA His2. There were no bands on the gel after revelation, the concentration of protein csgA seemed to be too little.</P><P> Promoter pcurli was extracted successfully by PCR using the plasmid p127 and the genomic DNA of strains MC4100, 1105, 818 and 1129 as templates.</P><P> Two 24 wells plates were inoculated with each of the 3 clones selected from the transformed strains 264, 266, 267 and 204, 227 for adherence tests.</P><P> Transformation of strains 204 and 227 with pHC04. </P><P> Nickel test on liquid bacterial cultures of strains 204, 227, 264, 266 and 267 after basifying with NaOH buffer to keep the pH of the solution around 8. Also, the influence of incubation time in the Nickel protocol was tested to improve it. The tests didn’t work.</P><H6>20/08 </H6><P> Nickel test on liquid bacterial cultures of strains 204, 227, 264, 266 and 267 without basifying with NaOH to keep this time the pH of the solution around 6. A calibration curve was obtained, however the test must be improved to determine the best conditions for the absorbance maximum.</P><P> Isolation of 3 of the red Congo stained clones from the transformed strains 204, 227, 264, 266 and 267. Each one of the 3 clones was inoculated in M63 and also streaked on congo red plates to see which one is the best and prepare cultures for adherence and liquid congo red tests.</P><P> Minipreps of pKK (empty) and psB1A2 to characterize further the Curli promoter (70 bp).</P><P> Culture of strains s204 and s227 in 5 mL of M63 medium for future tests to kill bacteria.</P><P> Culture of strains s204, s227, s264 and s267 in 50 mL of M63 medium at 37°C and 250 rpm.</P><P> Characterization of Curli promoter: </P><P> Extraction of the 70bp long Curli promoter (PC70) from the registry (BBa_K342000). Extraction of the 750bp long Curli promoter, (PC750) from the E. coli genome by PCR and following the biobrick standards (RFC10). </P><P> Extraction of the GFP from the plasmid collection at the INSA-Lyon 2011 (pIG57) where it is coupled to an upstream RBS (RBS-GFP in pSB1K3).</P><P> Digestion of the RBS-GFP by EcoRI and XbaI and the promoters by EcoRI and SpeI.</P><H6>21/08 </H6><P> Sowed more bacteria in 5 mL M63 medium for future tests to kill bacteria (because no Curly was seen)</P><P> Nickel test with same bacteria concentrations but different nickel concentrations.</P><P> Kinetic Curli expression obtained by determining Curli concentrations with Congo red test at different times and temperatures for 4 strains : 204, 227, 264 and 267. Experiments revealed surprising results : the Curli promoter was expressed at 37°C despite the thermal regulation known at this temperature. It implied that it could be possible for cells to grow and to produce Curli at the same time.  Some hypothesis can be drawn from the experiments : His2 strain absorbs better Congo Red than Wild Type strain. Curli promoter at 37°C or </P><P> Digestion of pKK (empty) with BamH1 and psB1A2 with EcoRI and PstI from previous minipreps. Electrophoresis gel test revealed pKK wasn’t enough concentrated but psB1A2 gave good results.</P><P> Test to improve Congo Red experiments : Test with different concentrations of Congo Red and change of medium (NaCl instead of M63). The tests didn’t show remarkable improvements however 10 µL will be preferred to 1 µL for next experiments.</P><P> Characterization of Curli promoter:  : Ligation of the RBS-GFP with PC70 or PC750 for 2 hours at 37°C.  It appeared RBS-GFP was the wrong sample and the right pIG57 (pSB1K3) was transformed into DH5α.</P><P> Complementation test for csgA purification: Supernatant of the strains 217 transformed with His1 CsgD and CsgD- was inoculated in the strain 266 culture (csgA-) containing thioflavine S. Curli formation will be checked to see which transformed 217 strain is better for production of csgA for purification (qualitatively). Again, the complementation test didn’t show a difference in thioflavine S colouration, so no curli formation was revealed.</P><H6>22/08 </H6><P> Nickel test with two new protocols.</P><P> UV Test with backlight kit to kill bacteria: wells containing M63 cultures of strains 204 and 227 were put under UV light for different lengths of time (1, 3, 7 and 11 minutes). Well contents were gradually transferred into Eppendorf  and diluted (100, 300, 900 and 2700 times). Five LB plates (without antibiotic) corresponding to UV exposure times (+ one plate for control) were then spotted with s204 and s227 five concentrations in order to be able to count survival bacteria after incubation at 37°C.</P><P> Curli promoter characterization :  The transformations were successful. pSB1K3 was red, control negative was indeed negative and pIG57 were all white. </P><P> 2 colonies of pIG57 were picked and put into 2x 3mL of LB for 3h culture at 37°C.</P><H6>24/08 </H6><P>Cultured 5ml DH5α tubes at 37°C and 230 rpm </P><P> Cultured 5 ml M63 tubes with Thioflavin S and strains s204, s227,s264, s266 and s267 at 30°C and 200 rpm.</P><P> Cultured 5 ml M63 tubes with strains s204 and s227 at 28°C and 200 rpm</P><P> Cultured 50 ml M63 erlens with strains s204 and s217 217 transformed with His1 CsgD and CsgD- at 28°C and 200 rpm</P><P> Storage of plates from UV test at 37°C.</P><H6>25/08 </H6><P> Cultured strains s264 and s266.</P><P> Transformation of strains s204 and s227 with pHC09 (empty plasmid with Curli promoter) for nickel titration.</P><P> Nickel test : redaction of a protocol with ICP.</P><P> pCurli characterization : cloning of pHC09, PIG50 and PCR product (70 bp pCurli promoter)</P><P> Precultures of DH5α, pHC23, 24, 25, 26 and pHC42, 43, 44, 45 </P><P> New tests with UV light to kill bacteria : 5 LB plates were spotted after exposure of 1,3,5 and 7 min to UV and genomic DNA was extracted from 50µL of each culture by heating 5min at 100°C and centrifuging 5 min at 14000 rpm.</P><P> Electrophoresis gel revealed no bands, more DNA had to be dropped off on gel.</P>
 Characterization of Curli promoter : Miniprep of pIG57 for future digestions and ligations.<H6>26/08</H6><P>Control of plasmid phenotypes of strains s308 to s315 </P><P> Miniprep kit of s308 to s315 and s264, s266 and s267</P><P> Transformation of s227 with pHC09 (because of contamination)</P><P> Extraction of genomic DNA of s204 and s227 cultures after UV exposure with a new protocol. Electrophoresis gel revealed small size DNA fragments for bacteria exposed to UV light which wasn’t the case for the control (which showed a band at more than 6000bp).  </P><P> Done a PCR on cultures exposed to UV light with Taq Polymerase and primers designed to amplify 750 bp Curli promoter, run electrophoresis gel after dropping off on gel 12µL of the 25µL PCR supernatant : PCR didn’t work as expected, maybe because the Taq Polymerase was one year expired.</P><P> Characterization of Curli promoter : digested pIG57 (E + X), ligated to PC750 (E + S) or PC70 (E + S) and then transformed in Kanamycin plates.</P><P> Culture of s225, s226+His1, s226+His2, s228, s229 for epifluorescence observation and immunocytochemistry. Culture of the same strains in 96 wells plates for Immunocytochemistry and for ThioflavineS staining.</P><P>Ni-DMG test at normal pH</P><H6>27/08 </H6><P> Storage of strains s309 to s324.</P><P>Preparation of Congo Red plates with s204, s227, s264, s266 and s267 for next adherence tests. </P><P> Miniprep Kit of s267 again.</P><P> New digestion of pHC43,44 and 45.</P><P> PCR on cultures exposed to UV light with Q5 polymerase and primers designed to amplify 750 bp Curli promoter. Electrophoresis gel of PCR supernatant revealed  bands were a lot thinner after 3 min of UV exposure, proving that genomic DNA was indeed degraded by UV light.</P><P> Baclight test of strain s204 and s227 cultures exposed to UV light and preparation on a sterile microscope slide for epifluorescence.</P><P> Characterization of Curli promoter : Culture of transformated colonies that potentially contain PC750-RBS-GFP-pSB1K3 or PC70-RBS-GFP-pSB1K3 into 3mL of M63 in order to anticipate the miniprep for the day after.</P><P> Thioflavin S dosage optimisation with the s204, s227, s264, s266, s267 </P><P> Immuno staining: antibody incubation with anti-Histag rabbit antibody and secondary antibody incubation with Alexa Fluor 568 goat anti-rabbit antibody. Observation at the epifluorescence microscope.</P><P> Test for the thioflavine S dosage with the fluorimeter.</P><P> Mesure of the fluorescence and OD of the 96 wells plate with ThS with the Tecan with or without biofilm resuspension.</P><P> Culture of s225 and s226His2.</P><P> Culture of each strain for icp-ms nickel test pH</P><H6>28/08 </H6><P>Epifluorescence microscopy was used to observe back light slides : s204 and s227 presented  about same quantities of red (dead) and green (alive) bacteria after different UV exposure times.  </P><P> Kinetic Curli expression was followed by determining Curli concentrations with Congo red test every hour for strains s204, s227, s264 and s267.</P><P> Precultures of s227 in M63 medium with Cm at 28°C, 220 rpm.</P><P> Extraction with mini prep Kit, digestion and storage of s267.</P><P> Digestion of s204 and s227 transformed with pHC09.</P><P> Extraction of CsgA with formic acid to do Western Blot later.</P><P> Characterization of Curli promoter : Miniprep of PC750-RBS-GFP-pSB1K3 and PC70-RBS-GFP-pSB1K3</P><P> Kinetics of s204, s227, WT and His2 with and without Thioflavin S. Mesure with the Tecan after centrifugation and resuspension in water/ethanol.</P><P> Culture of s204, s227, His1, His2, WT and CsgB- pkkCsgD for MET observation.</P><P>Culture of s225, s226+His2 in 96 wells plate. </P><P> ICP-MS nickel test at 3 concentrations</P><H6>29/08 </H6><P>New UV light tests to kill bacteria : spotted 5 LB plates after exposition to UV (1,3,5 and 10 min) and extracted genomic DNA from concentrated cultures.  Electrophoresis gel revealed no bands were visible, more DNA had to be dropped off on gel. </P><P> Kinetic Curli expression was followed by determining Curli concentrations with Congo red test every hour for strains s204, s227, s264 and s267. Protocol was modified. </P><P>Continuation of the kinetic mesurement with ThS.</P><P> Immuno staining with the primary antibody (without blocking) overweekend (anti His tag and anti Curli).</P><P> Characterization of Curli promoter : Digestion (EcoRI + PstI) of the following miniprep samples. Letters have a double digestion, numbers have a single digestion and T are the controls for non-digestion:</P></LI></UL><LI><A href="#contenu5" onclick="$('#contenu5').slideToggle('slow')"><H1>September</H1></A></LI><UL id="contenu5" style="list-style-type: none !important;display:none;"><LI><P><U>Dry lab</U>: Worked on a paper where a model for CsgB in vitro polymerisation was proposed : &quot; A Kinetic Study of Amyloid Formation : Fibril Growth and Length Distributions&quot; by John S. Schreck and Jian-Min Yuan.</P><P> Figured this model could be used for the curli growth, since CsgA et CsgB are really close in structure. Started thinking of how to code it.</P><H6>1/09  </H6><P>Characterization of Curli promoter :  </P><P>-	Isolation of 16 PC750bp-RBS-GFP on a Kanamycin plate. </P><P> -	Isolation of 16 PC70bp-RBS-GFP on a Kanamycin plate.</P><P> Both for conservation purposes.</P><P> End of immune staining and observation with the epifluorescence microscope.</P><P> Samples for MET preparation.</P><H6>2/09 </H6><P> Results from UV test : after a one night incubation at 37°C, a few bacteria grew even on the LB plate corresponding to a 10 min UV light exposition. A new experience must be done with 5, 10, 15 and 20 min exposition times.</P><P> 5 Cm LB plates from liquid cultures of strains s204, s227, s264 and s267 were streaked.</P><P> Kinetic Curli expression was followed by determining Curli concentrations with Congo red test every hour for strains s204, s227, s264 and s267. New modifications in the protocol.</P><P> Characterization of Curli promoter : Precultures for 16 PC750bp-RBS-GFP and 3 for PC70bp-RBS-GFP (as a control to indeed check the relevance and identity of the construction).</P><P> Observation of s227, His2, His1, WT, CsgB-pKKCsgD with the TEM with a negative colouration.</P><P> Blocking and primary antibody anti curli staining.</P><H6>3/09</H6><P><U> Dry Lab: </U></P><P>Began coding the model with GNU Octave, which is an open language very similar to the Matlab language.  </P><P> Coded the resolution method instead of using a pre-encoded solver. We hence tried to use the explicit Euler method as well as the Runge-Kutta 4 method.
However, we couldn't get any result out of it, and we suspected that our lacking skills in that language were partly responsible, so we translated the code from Octave to C, which was a bit faster to run the calculations and easier for us to us
</P><P> Managed to get interesting results by the end of the week, but still far from the actual shape of a curli growth.</P><P> New tests with UV light to kill bacteria : </P><P> -	5 Cm LB plates were spotted with s227 liquid culture (diluted 100, 104, 105 and 106 times) after exposure to UV (5,10,15 and 20 min). 
</P><P>-	Extraction of genomic DNA from concentrated cultures </P><P> -	Extraction of genomic DNA from concentrated cultures </P><P> -	Electrophoresis gel revealed fragments of small size but no band at more than 600 bp could be observed. </P><P> -	Back Light protocol with 170µL s227 liquid cultures and prepared microscope slides for epifluorescence observations</P><P> Characterization of Curli promoter : Miniprep and digestion (XbaI + SpeI) of 16 PC750bp-RBS-GFP and 3 for PC70bp-RBS-GFP, verification with electrophoresis gel. Results on gel revealed that PC70bp was OK for glycerol and plasmid conservation. The 16 PC750-RBS-GFP were all wrong since no band on 1500bp (Promoter + GFP) was visible. The reason for this was probably that that PC750 was a PCR product and was possibly not sufficiently digested, not in enough quantity or the primer constructions were too short on the restriction site extremities (EcoRI and PstI).</P><P>Cultured  225, 226 , 309, 311 , 312, 313, 315,316 in 5mL M63 + mannitol tubes at 30°C 230 rpm</P><P> Cultured DH5α  in 5 mL LB tubes at 37°C 250 rpm.</P><P>Cultured  strains 326 and DH5α  in 5mL M63 + mannitol tubes at 30°C 230 rpm</P><H6>4/09 </H6><P> PCR on cultures exposed to UV light with Q5 polymerase and primers designed to amplify 750 bp Curli promoter, electrophoresis gel revealed bands were a lot thinner after 10 min of UV exposition, proving that genomic DNA was indeed degraded by UV light.</P><P> Immunolabeling with 2 antibodies and observation with the epifluorescence microscope.</P><P> Red Congo test on DH5α and 326 in batch.</P><P>ICP-MS nickel results</P><H6>5/09 </H6><P>Characterization of Curli promoter : digestion of PCR production (P750) with EcoRI and SpeI and of pIG57 (EX-RBS-GFP-SP-pSB1K3) with EcoRI and XbaI. </P><P> Immunolabeling in batch on DH5α and 326 in order to validate the negative control (DH5α).</P><P> Cultured strains 225,226,309,311,312,313,315,316 in a 96 wells plate.</P><H6>8/09 </H6><P><U>Dry Lab: </U> Kept working on the model, having spotted some flaws in the code, but still without any good result.</P><P>Confocal microscope : Z-stack acquisition of225,226,309,311,312,313,315,316. Z projection. </P><P> Cultured 225, 226, 309, 311, 312, 313, 315, 316 in 5ml tubes at 30°C 230 rpm.</P><P> New protocols to prepare microscope slides for Back Light : </P><P> -	Washing liquid cultures with NaCl 0,85% after coloration </P><P> -	Drying microscope slides at 60°C after preparation</P><P> =&gt; The observation was easier as the bacteria didn’t move on the slides, leading to better Back Light pictures with microscope.</P><H6>9/09 </H6><P><U> Dry lab:</U> Part of the footage explicating synthetic biology and our project was shot.</P><P> Cultured strains 225, 226 , 309, 311 , 312, 313, 315,316 in three 96 wells plates for the confocal analyses and the immunolabeling. Some wells were filled with M63M supplemented with 5µM or 10 µM of Nickel.</P><P> New Back Light test five days after coloration with the new protocol and from 100 times diluted cultures.</P><P> =&gt; less bacteria were observed but the coloration interpretation were similar to the previous ones.</P><P> Red Congo test were made on strains 204, 227, 264, 266 and 267 but DO measures weren’t exploitable</P><H6>10/09 </H6><P> Tests to kill bacteria with temperature were done on M63 cultures of strain 326 that were put separately at 60° and 70°C for different lengths of time (15, 30, and 45 minutes). Tests of bacteria survival and extraction of strain 326 genomic DNA after heating were done.</P><P> Cultured strains 325, 326, 313, 315, 316 in 5mL tubes at 28°C 200 rpm during 10h for MET observation.</P><P> Immunolabeling tests.</P><P> Antibiotic test of P70-GFP 11-12-13 at 30°C</P><P><U> Dry Lab:</U> After another day without results, an e-mail was sent to the authors of the publication to ask them why we couldn't find the same graphs as they did. Turns out that a way more sophisticated solver as well as much more powerful calculating systems (CUDA and GPU computing) than what we had at our disposal were necessary in order to solve the system. </P><H6>11/09 </H6><P><U> Dry lab:</U> We tried to look into those more developped methods and found out they were out of my reach considering the little time we had left and the means we had at our disposal. So we cannot do any more for the model and we will use what we have and assume it is accurate, while of course keeping in mind that it is not.</P><P> Results of test to kill bacteria with temperature: </P><P>Tests of bacteria survival revealed that after a one night incubation at 37°C, a few bacteria grew on the LB plates corresponding to a 15, 30 and 45 min 60°C exposure whereas no one could be seen on the plates corresponding to cultures heated at 70°C . </P><P> =&gt; 70°C is the minimal temperature to kill bacteria and to prevent them from growing. </P><P> -	Electrophoresis gel with DNA extracted from 70°C heated cultures : the gel didn’t reveal so big differences between bacteria exposed to temperature and control.</P><P> -	PCR was done on the cultures exposed to 70°C with Taq Polymerase and primers designed to amplify 750 bp Curli promoter, run. Electrophoresis gel revealed similar bands between bacteria exposed to temperature and control.</P><P> =&gt; Temperature doesn't degrade genomic DNA.</P><P> Immunolabelling tests.</P><H6>12/09 </H6><P><U> Dry Lab:</U></P><P> Observation of 225,226,309,311,312,313,315,316 with a confocal microscope (with of without Nickel in the culture medium) : Z-stack acquisition in order to mesure the biofilm thickness.</P><P> Observation of 325,326,WT, His1, His2 with MET.</P><P> Cultured strain 266 in 70mL of M63+mannitol media at 28°C and 190 rpm.</P><H6>15/09 </H6><P> UV exposure of plates containing strain 266 liquid bacteria cultures.
Strain seemed to not flocculate, so a new transformation of strain 204 with plasmids pHC23, 25 and 26 was realized and streaked in Red Congo plates.
</P><H6>17/09 </H6><P> Transformation negative control was positive, strain 204 was cultured in presence of Cm and revealed it was resistant, so a new preculture of s204 was made with the strain that had been stored. </P><P>Transformation of the newly unstored strain 204 with plasmids pHC23, 25 and 26 was realized and streaked in Red Congo plates. This time, the negative control was negative and the transformation was successful. </P><P> Cultures of stored strains 264, 266, 267 and 325 were set in 5 mL M63 + mannitol media and incubated at 29°C and 140 rpm.</P><H6>19/09 </H6><P>Cultured 3 clones and a clone mix for each one of the newly transformed strains 224, 266, 267 in 5 mL M63+mannitol media tubes and also cultured a clone of strain 325 at 29ºC, 130 rpm for flocculation test. </P><P>Streaked 3 clones and a clone mix for each one of the newly transformed strains 224, 266, 267 plus control strain 325 on a red congo plate to determine the best clone for each strain.</P><H6>22/09 </H6><P> Stained bottom of preculture tubes with crystal violet.</P><P> Cultured the most adherent clones of each one of the newly transformed strains 224, 266, 267 and 325 in 150 mL of M63+mannitol media at 29ºC, 130 rpm.</P><H6>25/09 </H6><P> Nickel liquid test on newly transformed strains 224, 266, 267 and also on 325 with two different Nickel concentrations : 100 and 1000 μM.</P><P> MET observation of the newly transformed strains 224, 266, 267 and also on 325.</P><P> Red congo tests on newly transformed strains 224, 266, 267 and also on 325.</P><P> SDS-PAGE and formic acid treatment of newly transformed strains 224, 266, 267 cultures.
Stained bottom of Erlenmeyers were the strains were cultured with crystal violet.
</P><H6>30/09  </H6><P> ICP-MS nickel test of mineralized samples of transformed strains 224, 266, 267 and also on 325.</P><H5>October </H5><H6>2/10 </H6><P> Transformation of the newly unstored strain 204 with plasmids pHC23, 25, 26 (again) and pHC04.</P><P> Transformation of strains 206 (MG1655 ompR331::Tn10 csgA-uidA::kan), 1171 (MG1655 ompR331::Tn10) and 1623 (MG1655 ompR331::Tn10 with chromosomal insertion of PA1/04/03-gfpmut3*) with pHC04.</P><P> =&gt; All transformation were successful except for 1171 and 206 where negative control was positive. Maybe due to a problem in the strain stored.</P></LI></UL><LI><A href="#contenu6" onclick="$('#contenu6').slideToggle('slow')"><H1>October</H1></A></LI><UL id="contenu6" style="list-style-type: none !important;display:none;"><LI><H6>2/10 </H6><P>
Pre-cultures of the negative control, positive control, PHC46 (P70-GFP) and PHC47(P750-GFP).
</P><H6>3/10 </H6><P>
We distributed the blank, growth control, positive control for GFP expression, PHC46 (P70-GFP) and PHC47 (P750-GFP) in a 96 well plate and explored the GFP expression at 37 degrees.
</P><H6>3/10 </H6><P>
We gathered the results at 37°C and interpreted the results of GFP expression from the P70 or P750 long promoter upstream.
</P><H6>4/10 </H6><P> Selected 5 clones of each one of the newly transformed strains 224, 266, 267. Each clone was cultured in two Petri dishes containig M63+mannitol media.</P><H6>5/10 </H6><P> Stained half of the Petri dishes the 5 clones of each one of the newly transformed strains 224, 266, 267 with crystal violet.</P><H6>6/10 </H6><P> Changed media of all the remaining Petri dishes containing the 5 clones of each one of the newly transformed strains 224, 266, 267.</P><H6>7/10 </H6><P> Petri dish nickel test on the Petri dishes containing the best clones for each one of the newly transformed strains 224, 266, 267.</P><P>
Pre-cultures of the negative control, positive control, PHC46 (P70-GFP) and PHC47(P750-GFP).
</P><H6>8/10  </H6><P>
Cultured the best clones for each one of the newly transformed strains 224, 266, 267, 325 and 334 in a 24 wells plate.
</P><P>
We distributed the blank, growth control, positive control for GFP expression, PHC46 (P70-GFP) and PHC47 (P750-GFP) in a 96 well plate and explored the GFP expression at 30 degrees.
</P><H6>9/10</H6><P> Red congo and adherence tests on the newly transformed strains 224, 266, 267, 325 and 334 cultured in the 24 wells plate.</P><P>
We interpreted and compared the results of GFP expression from the P70 or P750 long promoter upstream.
</P></LI></UL></UL></DIV><DIV id="footer"><DIV id="contact"><DIV id="footer2"><P>INSA ADN Concept Club
	INSA de Lyon - Bâtiment Louis Pasteur
        11 avenue Jean Capelle
	69621 - VILLEURBANNE CEDEX</P><P>Mail : ia2c.asso@gmail.com</P></DIV></DIV><DIV id="copyright"><P>©2014 <A href="https://igem.org/Main_Page">IGEM.ORG</A> INSA-LYON - DESIGNED BY <A href="http://potopov.fr">ALICE BLOT</A></P></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:INSA-Lyon/Notebook">http://2014.igem.org/Team:INSA-Lyon/Notebook</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:INSA-Lyon/Notebook" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:INSA-Lyon/Notebook" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:INSA-Lyon/Notebook&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:INSA-Lyon/Notebook&amp;oldid=378389" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>