<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_INSA-Lyon_Results"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:INSA-Lyon/Results">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:INSA-Lyon/Results&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:INSA-Lyon/Results&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:INSA-Lyon/Results&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:INSA-Lyon/Results" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:INSA-Lyon/Results</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><P><TITLE>Curly'on - IGEM 2014 INSA-LYON</TITLE></P><DIV id="menu"><UL id="menuul"><LI class="menuulli"><A class="menuullia" href="https://2014.igem.org/Team:INSA-Lyon/Project" id="link-project"><SPAN><P>PROJECT</P></SPAN></A></LI><LI class="iconmenu">PROJECT</LI><LI><A class="hi-icon" href="https://2014.igem.org/Team:INSA-Lyon/Project">PROJECT SUMMARY</A></LI><LI><A class="hi-icon" href="https://2014.igem.org/Team:INSA-Lyon/Achievements">ACHIEVEMENTS</A></LI></UL><LI class="menuulli"><A class="menuullia" href="https://2014.igem.org/Team:INSA-Lyon/Biology" id="link-wetlab"><SPAN><P>WETLAB</P></SPAN></A></LI><LI class="iconmenu">WETLAB</LI><LI><A class="hi-icon" href="https://2014.igem.org/Team:INSA-Lyon/Biology">WETLAB SUMMARY</A></LI><LI><A class="hi-icon" href="https://2014.igem.org/Team:INSA-Lyon/Results">RESULTS</A></LI><LI><A class="hi-icon" href="https://2014.igem.org/Team:INSA-Lyon/Notebook">NOTEBOOK</A></LI><LI><A class="hi-icon" href="https://2014.igem.org/Team:INSA-Lyon/Protocol">PROTOCOLS</A></LI><LI><A class="hi-icon" href="https://2014.igem.org/Team:INSA-Lyon/DataPage">DATA PAGE</A></LI><LI class="menuulli"><A class="menuullia" href="https://2014.igem.org/Team:INSA-Lyon/Modeling" id="link-modeling"><SPAN><P>MODELING</P></SPAN></A></LI><LI class="iconmenu">MODELING</LI><LI><A class="hi-icon" href="https://2014.igem.org/Team:INSA-Lyon/Modeling">MODELING SUMMARY</A></LI><LI><A class="hi-icon" href="https://2014.igem.org/Team:INSA-Lyon/Molecular">MOLECULAR MODELISATION</A></LI><LI><A class="hi-icon" href="https://2014.igem.org/Team:INSA-Lyon/CurliSynthesis">CURLI SYNTHESIS</A></LI><LI class="menuulli"><A class="menuullia" href="https://2014.igem.org/Team:INSA-Lyon/HumanPractice" id="link-human"><SPAN>  HUMAN PRACTICE</SPAN></A></LI><LI class="iconmenu">HUMANPRACTICE</LI><LI><A class="hi-icon" href="https://2014.igem.org/Team:INSA-Lyon/HumanPractice">HUMAN PRACTICE SUMMARY</A></LI><LI><A class="hi-icon" href="https://2014.igem.org/Team:INSA-Lyon/Perception">SYNBIO PERCEPTION</A></LI><LI><A class="hi-icon" href="https://2014.igem.org/Team:INSA-Lyon/Purpose">PROJECT'S INTEREST</A></LI><LI class="menuulli"><A class="menuullia" href="https://2014.igem.org/Team:INSA-Lyon/Team" id="link-team"><SPAN><P>TEAM</P></SPAN></A></LI><LI class="iconmenu">TEAM</LI><LI><A class="hi-icon" href="https://2014.igem.org/Team:INSA-Lyon/Team">MEMBERS &amp; ATTRIBUTIONS</A></LI><LI><A class="hi-icon" href="https://2014.igem.org/Team:INSA-Lyon/Sponsors">SPONSORS</A></LI><LI><A class="hi-icon" href="https://2014.igem.org/Team:INSA-Lyon/Partnership">PARTNERSHIPS</A></LI><LI><A class="hi-icon" href="https://2014.igem.org/Team:INSA-Lyon/Havefun">HAVE FUN WITH IGEM</A></LI><LI><A class="hi-icon" href="https://2014.igem.org/Team:INSA-Lyon/Gallery">GALLERY</A></LI></DIV><DIV id="soustitre" style="background-image: url('https://static.igem.org/mediawiki/2014/d/de/Fond_wetlab.jpg');"><H4>RESULTS</H4><DIV id="icones"><UL><LI class="iconmulti"><A href="https://2014.igem.org/Team:INSA-Lyon/Biology" class="hu-icon">WETLAB SUMMARY</A></LI><LI class="icon"><A href="https://2014.igem.org/Team:INSA-Lyon/Results" class="hu-icon">RESULTS</A></LI><LI class="icon"><A href="https://2014.igem.org/Team:INSA-Lyon/Notebook" class="hu-icon">NOTEBOOK</A></LI><LI class="icon"><A href="https://2014.igem.org/Team:INSA-Lyon/Protocol" class="hu-icon">PROTOCOLS</A></LI><LI class="icon"><A href="https://2014.igem.org/Team:INSA-Lyon/DataPage" class="hu-icon">DATA PAGE</A></LI></UL></DIV></DIV><DIV id="presentation"><UL style="list-style-type: none !important;"><LI><A href="#contenu1" onclick="$('#contenu1').slideToggle('slow')"><H1>Curli characterization</H1></A></LI><UL id="contenu1" style="list-style-type: none !important;display:none;"><LI><DIV><P align="justify">Five complementary tests were performed to evaluate the ability of the modified cells to assemble functional curli:
<B>1)</B> determination of the percentage of adherent cells to polystyrene in 24 wells-plates,<B>
 2)</B> crystal violet staining of biofilm formed on polystyrene in 24 wells-plates, <B>
3)</B> ability to bind the congo red,<B>
 4)</B> biofilm maximum thickness measurement and biovolumes quantification of GFP-tagged biofilm observed with a confocal microscopy and <B> 5)</B> curli structure observation using Transmission Electron Microscopy (MET).</P><H5>Adhesion test and curli production</H5><B>Figure 1 : Engineered bacteria Percentage of adhesion</B><P align="justify"><I>csgA-</I>knockout <I>E. coli</I> strain was transformed with BBa_CsgA-WT (<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1404006">BBa_K1404006</A>); BBa_CsgA-His1 (<A href="http://parts.igem.org/Part:BBa_K1404007">BBa_K1404007</A>); BBa_CsgA-His2 (<A href="http://parts.igem.org/Part:BBa_K1404008">BBa_K1404008</A>). The corresponding positive and negative controls are, respectively, Wild-type <I>E.coli</I> curli producing strain transformed with the empty vector and <I>csgA-</I>-knockout <I>E. coli</I> strain transformed with the empty vector. 
Strains with our parts, the positive and negative controls were cultured in a 24-well microplate in M63 Mannitol during 24H at 30°C. The supernatant was removed and the OD600 measured, then the bacteria forming the biofilm were resuspended and the OD600 was measured in order to estimate the number of cells (<A href="https://static.igem.org/mediawiki/2014/8/80/Adhesion_test_protocole.pdf" target="_blank">See protocol for details</A>). The percentage of adhesion was calculated as follows: 
(OD600 of  the biofilm)/ (OD600 of  the supernatant + OD600 of the biofilm) 
Different uppercase letters displayed on the graph  indicate significant differences  between strains (Tukey’s test, p &lt; 0.05) 
These results show that <B>the percentage of adhesion is similar between the strains containing the three parts and the positive control, thus tagged CsgA were still functional</B>. CsgA with one or two tags expressed by the P70 promoter were sufficient to form thick biofilms. </P><B>Figure 2 : Engineered bacteria Biofilm formation</B><P align=" justify ">The cells were cultured as described in figure 1. </P><DIV align="justify"><P>The supernatant was removed and the remaining biofilm was fixed to the microplate by heat treatment at 80°C during 1H. The crystal violet solution was added in each well in order to stain the cells and the wells were washed with water to remove crystal violet in excess (<A href="https://static.igem.org/mediawiki/2014/e/ef/Crystal_Violet_protocole.pdf" target="_blank">See protocol for details</A>).
Crystal violet staining shows that <B>the strain containing the three parts could form a biofilm like the positive control. Thus tagged CsgA were still functional</B>. CsgA with one or two tags expressed by the P70 promoter were sufficient to form thick biofilms.</P></DIV><B>Figure 3 : Engineered bacteria curli production</B><P align="justify">Strains are the same as in figure 1. 
Strains with our parts, the positive and negative control were cultured in M63 Mannitol at 30°C and 180rpm. After centrifugation, the supernatant was removed and the cell pellet was resuspended in the Congo Red solution, in order to specifically stain the curli. The samples were centrifuged again and the pellets were observed (<A href="https://static.igem.org/mediawiki/2014/3/39/CongoRed.pdf" target="_blank">See protocol for more details</A>). 
Congo Red staining shows that <B>the CsgA with one or two tags expressed by the P70 promoter allows to form curli fibers</B> which are able to bind Congo Red.</P></DIV><H5>Confocal Laser Scanning Microscopy Analyses</H5><DIV align="justify"><P>For the Confocal Laser Scanning Microscopy biofilm acquisitions, all the strains were cultivated in 96-well microplate in M63 Mannitol during 16H at 30°C (<A href="https://static.igem.org/mediawiki/2014/7/7e/Culture_confocal_analyse.pdf" target="_blank">See Protocol for details</A>). See results in <B>Figure 4</B>.</P><P><B>Figure 4: Engineered bacteria biofilm characterization and quantification using Confocal Laser Scanning Microscopy</B></P><P>All the strains used are constitutively fluorescent to allow detection with confocal laser microscopy (ZEISS LSM510 META, 40X/1.3OILDIC, laser Argon 4 lines 30 W 458 nm, 477 nm, 488 nm, 514 nm, <A href="https://static.igem.org/mediawiki/2014/7/7e/Culture_confocal_analyse.pdf" target="_blank">See Protocol</A>). Positive control/CsgA+ (Wild-type <I>E. coli</I> curli producing strain); Negative control/CsgA- (<I>csgA</I>-knockout <I>E. coli</I> strain); BBa_CsgA (<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1404006">BBa_K1404006</A>); BBa_CsgAHis1 (<A href="http://parts.igem.org/Part:BBa_K1404007">BBa_K1404007</A>); BBa_CsgAHis2 (<A href="http://parts.igem.org/Part:BBa_K1404008">BBa_K1404008</A>). <B>A)</B> Biofilm sections obtained by Z-stack acquisitions. <B>B)</B> Biofilm 3D reconstruction using IMARIS® from acquisitions in A). <B>C) </B>Bio-volume quantification and maximum of thickness measurement using COMSTAT2 (ImageJ). The strain marked with a star is significantly different from all others (Tukey’s test, p&lt;0.05).</P><P>As no strain carrying our parts showed a significant difference with the positive control, our part’s insertion doesn’t modify the biofilm formation properties. <B>The His-Tag and His2-Tag engineered CsgA doesn’t disturb the curli formation.</B></P><H6> Transmission Electron Microscopy
</H6><P>For the Transmission Electron Microscopy, all the strains were cultured in conditions that allow curli formation: 48h (for a 50 mL culture),  28⁰C temperature and low agitation. The ammonium molybdate was used as a negative colorant (Microscope: MET PHILIPS CM120). </P><P><B>Figure 5: Engineered bacteria curli structure observation using Transmission Electron Microscopy</B></P><P>The bacteria cultures we used are a <I>csgA</I>-knockout strain as negative control, and the three engineered csgA constructions, the WT, the His1-Tag and His2-Tag. </P><P>The images show that there is <B>no significant difference between our positive control and our constructions</B>.  Thus, we can conclude that <B>our parts insertions don’t affect the structure of the amyloid fibers and the configuration of the curli formation</B>. </P><H6> General Conclusion
</H6><P><B>The expression of CsgA derivatives</B> expressed by the p70 <I>csg</I> promoter carried by the psb1c3 plasmid leads to <B>functional CsgA</B>, and allows <I>E. coli</I> to <B>stick and form biofilm</B>. Moreover, our results show that the <B>addition of one or two His-Tag on the C-term of CsgA doesn’t disturb the normal properties of curli</B> (sturdiness, adhesion, structure and folding of CsgA).</P></DIV></LI></UL><LI><A href="#contenu2" onclick="$('#contenu2').slideToggle('slow')"><H1>Nickel chelation</H1></A></LI><UL id="contenu2" style="list-style-type: none !important;display:none;"><LI><DIV align="justify"><P> Nickel(II) chelation was evaluated in a CsgA- MG1655 background (in order to have only our modified or unmodified curlis at the surface of the strain) for each one of the constructions (<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1404006">BBa_CsgA-WT (BBa_K1404006)</A>, <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1404007">BBa_CsgA-His1 (BBa_K1404007)</A>,or <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1404008">BBa_CsgA-His2 (BBa_K1404008)</A>). Dimethylglyoxime (DMG) was used as a complexing reagent, which forms a pink-colored complex (peak absorption at 554nm) in the presence of Ni(II). </P><DIV align="center"><B>Figure 1 : DMG-Ni complex formation with or without chelating bacteria</B></DIV><P>Firstly, a<B> calibration curve </B> of the formation Nickel and DMG complexes was established. </P><P>Then, liquid cultured strains were assayed for biofilm nickel absorption using the calibration curve, after measuring the OD of the complex formed for each strain at 554nm. (<A href="https://static.igem.org/mediawiki/2014/0/01/Ni_chelation_DMG_n.pdf" target="_blank">See Protocol for details</A>)
Although quantification is possible, this technique lacks precision and is more suited for <B>qualitative</B> studies. However, it is a cheaper alternative to ICP-MS. </P><DIV align="center"><B>Figure 2 : Calibration set picture</B><B>Figure 3 : Calibration set : DMG-Ni complex OD554 (optical density)=f([Ni(II)]</B></DIV><P>Nickel-DMG complex colorimetry measurement follows a <B>linear regression</B> for concentrations  from 20uM to 100uM. Visually, we can see gradient from transparency (at [20uM]) to pink (at [100uM]). This visual method allows us to compare the Ni chelation between our strains. The paler the color is, the more Ni has been chelated. The culture supernatant of CsgA- bacteria from strain with the part <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1404008">BBBa_CsgA-His2</A> is less colored than the others, which shows that this part allows to capture more nickel.

These results show that <B>the part BBa_CsgA-His2 confers increased chelation</B> to strain CsgA-. It is shown that it chelates more than part <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1404006">BBa_CsgA-WT</A> and part <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1404007">BBa_CsgA-His1</A>. </P><P>A second method has been used, more <B>quantitative</B> and more precise (but more expensive) : <B> ICP-MS (Inductively coupled plasma mass spectrometry)</B>. (<A href="https://static.igem.org/mediawiki/2014/f/f7/Ni_chelation.pdf" target="_blank">See Protocol for more details</A>) The metal content of the bacterial pellets were assayed. The quantity of chelated nickel for each strain was compared to the quantity of curlis formed by each strain.</P><DIV align="center"><B>Figure 4 : ICP-MS results : Ni(II) chelated for each construction linked to the quantity of curlis</B></DIV><P> Different lowercase letters displayed on the graph  indicate significant differences  between strains (Tukey’s test, p &lt; 0.05). Error bars represent standard deviations.</P><P>Taken together, these results show that the CsgA- Strain with part <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1404008">BBa_CsgA-His2</A> chelates twice as much as strain CsgA- with part <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1404007">BBa_CsgA-His1</A>. That means that <B>only two His-tags on C-term can improve the natural nickel chelation capacities of CsgA </B>. CsgA with a single His-tag did not perform better than a wild-type CsgA. It can be explained by the conformation of the His1-tag which could be folded on the side of CsgA, as presented in the <A href="https://2014.igem.org/Team:INSA-Lyon/Molecular#nichelation" target="_blank">modelization section</A>. Potentially, further increase of the amount of His-tags could improve the nickel accumulation capacities of CsgA. </P></DIV></LI></UL><LI><A href="#contenu3" onclick="$('#contenu3').slideToggle('slow')"><H1>Survival after UV and high temperature exposure</H1></A></LI><UL id="contenu3" style="list-style-type: none !important;display:none;"><LI><DIV align="justify"><P>
To address biosafety issues linked with GMOs, we worked on destroying our bacteria after letting them grow in a biofilm. As the captured metal is extracellular and Curli proteins are very resistant to environmental changes, alive bacteria are not needed for our biofilter. Our goal was to obtain a biomaterial made out of modified Curli able to chelate nickel. 
</P><P>
In order to find the best way to degrade bacteria and DNA, the <A href="https://static.igem.org/mediawiki/2014/1/1e/UV_temperature.pdf" target="_blank">following protocol</A>  was used to test the influence of UV light and temperature separately : </P></DIV></LI><LI> Wells containing M63 cultures of strain 227 were put under UV light or exposed to heat treatments (at 60 or 70°C) for different lengths of time. Well contents were then gradually transferred into Eppendorfs and diluted (100, 300, 900 and 2700 fold).</LI><LI> LB plates (without antibiotic) corresponding to UV/temperature exposure times (+ one plate for control) were then spotted with different concentrations of strain 227 in order to be able to count bacterial survival after incubation at 37°C.</LI><LI> Genomic DNA was extracted from strain 227 concentrated culture. From the solution obtained, Curli promoter(750 bp) was amplified by PCR with Q5 polymerase and designed primers. </LI><LI>Epifluorescence observations were made after Back Light coloration of 227 strain liquid cultures.</LI></UL><H6> UV light influence </H6><DIV align="center"><I>LB plates</I></DIV><DIV align="center"><TABLE><TBODY><TR><TD><DIV align="center"><FIGCAPTION>Control plate</FIGCAPTION></DIV></TD><TD><DIV align="center"><FIGCAPTION>5 min UV</FIGCAPTION></DIV></TD><TD><DIV align="center"><FIGCAPTION>10 min UV</FIGCAPTION></DIV></TD></TR></TBODY></TABLE></DIV><DIV align="center"><TABLE><TBODY><TR><TD><DIV align="center"><FIGCAPTION>15 min UV</FIGCAPTION></DIV></TD><TD><DIV align="center"><FIGCAPTION>20 min UV</FIGCAPTION></DIV></TD></TR></TBODY></TABLE><B>Figure 1: Monitoring of bacteria kill after UV exposure</B></DIV>
No bacteria grew on LB plates after 15 minutes UV light exposure.⇒<B>Bacterial growth can be stopped this way. </B><DIV align="center"><I> DNA extraction</I></DIV><DIV align="center"><TABLE><TBODY><TR><TD><DIV align="center"><FIGCAPTION><B>Figure 2: PCR gel after UV exposure  </B></FIGCAPTION></DIV></TD></TR></TBODY></TABLE></DIV> Bacterial DNA seemed to be degraded after 10 min UV light exposure.⇒<B>In consequence, UV light can be used to destroy DNA.</B><DIV align="center"><I>Backlight</I></DIV><DIV align="center"><TABLE><TBODY><TR><TD><DIV align="center"><FIGCAPTION>Control plate</FIGCAPTION></DIV></TD><TD><DIV align="center"><FIGCAPTION>15 min UV</FIGCAPTION></DIV></TD><TD><DIV align="center"><FIGCAPTION>20 min UV</FIGCAPTION></DIV></TD></TR></TBODY></TABLE><B>Figure 3: Observations of bacteria exposed to UV and after backlight coloration : 
alive bacteria appear in green and dead ones in red.</B></DIV> Still some green-colored bacteria could be seen after 20 min UV exposure. ⇒<B>UV light isn’t enough to kill bacteria.</B><H6><B>Temperature influence</B></H6><DIV align="center"><I>LB plates</I></DIV><DIV align="center"><TABLE><TBODY><TR><TD><DIV align="center"><FIGCAPTION>Control plate</FIGCAPTION></DIV></TD><TD><DIV align="center"><FIGCAPTION>15 min 60°C</FIGCAPTION></DIV></TD></TR><TR><TD><DIV align="center"><FIGCAPTION>30 min 60°C</FIGCAPTION></DIV></TD><TD><DIV align="center"><FIGCAPTION>45 min 60°C</FIGCAPTION></DIV></TD></TR></TBODY></TABLE><B>Figure 4: Monitoring of bacteria heated at 60°C</B></DIV> 
Bacteria grew on LB plates even 45 min after being heated at 60°C. ⇒<B>60°C isn't enough high to kill bacteria.</B><P>So we tried experiments with a temperature of 70°C.</P><DIV align="center"><I>LB plates</I></DIV><DIV align="center"><TABLE><TBODY><TR><TD><DIV align="center"><FIGCAPTION>Control plate</FIGCAPTION></DIV></TD><TD><DIV align="center"><FIGCAPTION>15 min 70°C</FIGCAPTION></DIV></TD></TR><TR><TD><DIV align="center"><FIGCAPTION>30 min 70°C</FIGCAPTION></DIV></TD><TD><DIV align="center"><FIGCAPTION>45 min 70°C</FIGCAPTION></DIV></TD></TR></TBODY></TABLE><B>Figure 5: Monitoring of bacteria heated at 70°C</B></DIV>
No more bacteria grew on LB plates after 15min at 70°C⇒<B>Bacterial growth can be stopped as well as with UV light.</B><DIV align="center"><I>DNA extraction</I></DIV><DIV align="center"><TABLE><TBODY><TR><TD><DIV align="center"><FIGCAPTION><B>Figure 6 : PCR gel after DNA extraction from bacterial culture exposed to heat treatment with 70°C</B></FIGCAPTION></DIV></TD></TR></TBODY></TABLE></DIV>
No DNA degradation at all.⇒<B>In consequence, unlike UV light,  temperature treatment doesn't destroy DNA.</B><DIV align="center"><I>Backlight</I></DIV><DIV align="center"><TABLE><TBODY><TR><TD><DIV align="center"><FIGCAPTION>Control plate</FIGCAPTION></DIV></TD><TD><DIV align="center"><FIGCAPTION>15 min at 70°C</FIGCAPTION></DIV></TD></TR><TR><TD><DIV align="center"><FIGCAPTION>30 min at 70°C</FIGCAPTION></DIV></TD><TD><DIV align="center"><FIGCAPTION>45 min at 70°C</FIGCAPTION></DIV></TD></TR></TBODY></TABLE><B>Figure 7: Observations of bacteria heated at 70°C and after backlight coloration</B></DIV> 
No difference of coloration was observed between the control and the samples heated at 70°C : indeed a lot of green-colored bacteria remained after 45 min of heating.⇒<B>Temperature isn’t enough to kill bacteria just like UV light.</B><P> To solve this last problem, bacteria were put in contact with 70% ethanol. The Backlight coloration gives the following picture. </P><DIV align="center"><TABLE><TBODY><TR><TD><DIV align="center"><B>Figure 8 : Backlight after DNA extraction of bacterial culture exposed to ethanol</B></DIV></TD></TR></TBODY></TABLE></DIV><P>These numerous experiments lead us to develop a protocol in three steps, illustrated by the drawing below :</P><DIV align="center"><TABLE><TBODY><TR><TD><DIV align="center"><FIGCAPTION><B>Figure 9 : Global strategy to kill bacteria</B></FIGCAPTION></DIV></TD></TR></TBODY></TABLE></DIV></UL><LI><A href="#contenu4" onclick="$('#contenu4').slideToggle('slow')"><H1>Promoter optimization and characterization</H1></A></LI><UL id="contenu4" style="list-style-type: none !important;display:none;"><LI><P align="justify">
As a follow-up to the exploration of curli production and nickel chelation, we want to know the kinetics behind the 70 base-pair long promoter sequence that we used during the whole summer.
In fact, it has the interesting property of being activated at 37°C instead of the 30°C of the natural 750 base-pair <I>csgA</I> promoter from where it is originally isolated. However, we explored these two promoters' kinetics at 30 and 37°C by inserting a GFP downstream.



</P><DIV align="center"><FIGCAPTION><B>Figure 1: GFP expression plasmid for promoter characterization. The promoter sequence can either be the 70 (P70) or the 750 base-pair promoter sequence.</B></FIGCAPTION></DIV><P>On a pKK backbone, two essential parts have been assembled: a promoter and a reporter gene. The reporter gene in this case is always the same: GFP. However, the promoter is different for each construction.
On the one hand, P70 is the 70 base-pair long promoter sequence and when combined to the reporter GFP, the construction is called p70:GFP.
On the other hand, P750 is the 750 base-pair long promoter sequence coding for the inter-genic regulation region of curli production and combined to the reporter GFP, the construction is called P750:GFP. These constructs allow to compare the expression of both promoters.

</P><DIV align="center"><FIGCAPTION><B>Figure 2: Promoter expression as a function of the Optical Density. At early growth stage, P70 has a higher expression rate at 37°C relatively to the P750.</B></FIGCAPTION></DIV><P><B>Early growth stage promoter kinetics</B></P><P align="justify">During the early growth stages at 37°C, we can observe that the P70 (orange) has a higher expression level compared to the P750 (red).
However at 30°C, both P70 (light blue) and P750 (dark blue) have low expression levels.
We conclude that P70 has the ability to prematurely activate downstream expression at 37°C.
</P><DIV align="center"><FIGCAPTION><B>Figure 3: Promoter expression as a function of the Optical Density. At mid/late growth stage, P750 has a delayed but higher expression rate at 30°C relatively to the P70.</B></FIGCAPTION></DIV><P><B>Mid/late growth stage promoter kinetics</B></P><P align="justify">During the mid/late growth stages at 37°C, we can observe that both P70 (orange) and P750 (red) have a decreased promoter expression and are moving towards lower rates.
However at 30°C, P70 (light blue) stabilizes within the range of low expression levels and P750 (dark blue) reaches the highest promoter expression rates.
We conclude that P750 has a delayed, albeit extremely high-leveled, promoter expression at 30°C.
</P></LI></UL></DIV></DIV><DIV id="footer"><DIV id="contact"><DIV id="footer2"><P>INSA ADN Concept Club
	INSA de Lyon - Bâtiment Louis Pasteur
        11 avenue Jean Capelle
	69621 - VILLEURBANNE CEDEX</P><P>Mail : ia2c.asso@gmail.com</P></DIV></DIV><DIV id="copyright"><P>©2014 <A href="https://igem.org/Main_Page">IGEM.ORG</A> INSA-LYON - DESIGNED BY <A href="http://potopov.fr">ALICE BLOT</A></P></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:INSA-Lyon/Results">http://2014.igem.org/Team:INSA-Lyon/Results</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:INSA-Lyon/Results" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:INSA-Lyon/Results" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:INSA-Lyon/Results&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:INSA-Lyon/Results&amp;oldid=394837" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>