Team:Aix-Marseille/Results
Chew fight
Toggle navigation
Home
Project
Overview
Project design
Parts
Team
Members
Supervisors
Collaborations
Attributions
Notebook
Calendar
Results
Protocols
Practices
Interlab Study
Calendar
Results
Protocols
Sequencing data
Modeling
Modeling
Software
Achievement
Results
1-What is cytochrome C?
The cytochrome complex, or cyt c is a small hemeprotein found in association with the inner membrane of the mitochondrion. It is interesting because unlike other cytochromes, cytochrome c is highly soluble (100g/L). It is capable of undergoing oxidation and reduction. It transfers electrons in the electron transport chain (between complex III and IV), where it is indispensable.
More than its biological functions, cytochrome c can catalyse several reactions such as hydroxylation and aromatic oxidation, and shows peroxidase activity by oxidation of various electron donors such as 2,2-azino-bis(3-ethylbenzthiazoline-6-sulphonic acid) (ABTS).
2-How does the cytochrome C work in our project?
In our project, we hypothesized that the cytochrome c is important to degrade the chewing-gum. It contains a heme with a central iron, which is first light excited and then oxidised by the laccase. Then, a series of oxidation/reduction reactions will lead to the degradation of the main compounds of the chewing-gum: cis-1,4-polyisoprene, trans-1,4-polyisoprene and poly-styrene-butadiene.
Why do we choose the laccase/cytochrome c couple?
3 conditions make the laccase able to oxidise the cytochrome:
1-The correct distance between redox centres of the laccase and the cytochrome
2-The right orientation of both proteins
3-The consistent redox potential difference (ΔE) between donor and acceptor
3-What are our constructions
We decided to use 3 different cytochromes c. First, we chose the cytochrome c of Escherichia coli because we worked with E.coli strains. Then, we chose the cytochrome c of Shewanella oneidensis because it can use other final electron acceptors than O2 that suggests a very efficient cytochrome c. Finally we worked with the cytochrome c of Synechocystis sp. PCC 6803. Indeed, it is a photosynthetic bacterium capable of cellular respiration suggesting an efficient electron transfer. It is an interesting point in our strategy.
In each case, two types of construction have been done: a cytochrome c alone and a cytochrome c linked to a laccase. In both cases, we anticipated a problem to overcome. To be functional, a cytochrome c must integrate a heme into its structure. As we overexpressed the cytochrome C, we should overexpress simultaneously hemes. In this purpose, we chose to clone the heme A gene under the control of the T7 promotor (Bba_.......). This construction is carried by an independent plasmid.
Results
•Genetic construction works well. T7 promotor has been put in the 5’extremity of the heme A encoding gene. Three tests showed that the right construction was obtained: Colony PCR with the expected length DNA amplification (1), E/P digestion with an expected length DNA (2) and sequencing results (3).
•Expression of heme A protein did not work. Under aerobic condition in BL21 strain, when induced by IPTG, the heme A protein is not produced.  Cell extracts of a non-induced and an induced culture were analysed by a SDS-PAGE colored by Coomassie-blue staining. It should be more rigorous to confirm this result doing a western-blot analyse with an anti-hemeA antibody. But we couldn’t do it because of a lack of time and lack of the antibody.
To improve purification process, a 10-histidine tail (Bba_K525998) has been added in C-term extremity of the cytochrome c. During purification process, we used a nickel column. The 10-histidine-tag will be fixed to the column with a high affinity, and so the cytochrome c will be purified. A T7 (IPTG inducible) promotor (Bba_K525998) was put in the 5’extremity of the cytochrome C encoding gene in addition to the deletion of the codon stop.
We decided to make a translational fusion between the cytochrome c and the laccase to statistically increase interaction between both proteins. The problem is that the laccase is transported by the TAT system and can’t use the SEC pathway, whereas the cytochrome c is transported by the SEC system. To be export together, laccase and cytochrome C have to take the TAT pathway. So we have to clone the sequence signal TAT instead of the sequence signal SEC. Then, we modified the secretion pathway of the cytochrome c from SEC to TAT system.
In this case, it has been shown previously that the cytochrome c could be transported by the TAT system but, in order to be correctly matured it must contain an extended CXXCH motif. This motif will be recognized by a yeast cytochrome c heme-lyase (CCHL), which will mature the cytochrome c. That is why we did constructions described below.
•Cytochrome c + laccase
Unfortunately, we did not manage to get the whole construction.
Purification of Cytochrom C
We produced the cytochrome C from Shewanella oneidensis using two constructions named 01-1202 (1) and 01-1202 (2).
After sonication, we analyzed cell extracts by SDS-PAGE gel 15% (Gel1) to determine the localisation of the cytochrome C: in the supernatant or in the cell pellet.
Note that we produced the cytochrome C under two different conditions: in an aerobic (Aero) or in an anaerobic (Anaero) condition.
As we can see, the cytochrome c is located in the cell pellet, more precisely in the cell membranes. This result suggests us that the cytochrome C is associated to or inserted in the membrane. We solubilized the cytochrome C with detergent (triton) overnight 3 times.  We did mistakes such as using the wrong detergent, the wrong SDS-PAGE gel or a not effective bad sonication that did not break cells…) Finally, we managed to analyze our western-blot. After solubilization, the cytochrome C is in low quantity in the cytoplasm and in the membrane extracts. We obtained more quantity in the solubilized membranes.
Unfortunately, we did not have anything after purification and we missed time to try again.
But we know that we can produce it. Maybe if we had more time, we could purify it and then test it on polymers.
Why did it not work ?
Maybe it’s because we did not produce Heme A with our Cytochrom C.
Problem with the folding
Production of Laccase E.coli
To obtain our Laccase E.coli, we used the BioBrick Bba_K863006 and we removed its stop codon. Then we added to this BioBrick a promoter and a His-Tag.
Thanks to digestion, ligation and transformation, we managed to get our BioBrick named “01-35-02” with a size of about 1700 pb:
Figure A: Schematic representation of “01-35-02”
Figure B: Digestion from “01-35-02” miniprep to check the size of the insert  The band corresponds to the expected size for the insert, which is about 1700 pb.
Then we inserted our BioBrick into E.coli strain (BL21) to express it. We induced it by addition of IPTG into the cell culture. We made a Western-Blot using a primary antibody against the His-tag and an anti-mouse secondary antibody conjugate with HRP (horseradish peroxidase). The expected size of the protein “01-35-02” is 53 kDa.
Figure C: Western Blot of the expression of “01-35-02” into E.coli The expected size of the protein is about 53 kDa.
Production of laccase T.thermophilus from iGEM parts
To get our Laccase T.thermophilus, we used the BioBrick Bba_K863011 and we removed its stop codon.
Then we tried to add to this BioBrick a promoter and a His-Tag.
Unfortunately we managed to add only the promoter.
Figure D: Schematic representation of “01-36”
Figure E: Digestion from “01-36” miniprep to check the size of the insert  The band corresponds to the expected size for the insert, which is about 1500 pb.”
Production of laccase T.thermophilus from IDT
We ordered from IDT a laccase optimized for an expression into E.coli.
From this optimised laccase, we added a promoter and a His-Tag.
This new BioBrick is named “01-30-02” with an expected size of about 1500 pb
Figure F : Schematic representation of “01-30-02”
Figure G: Digestion from “01-30-02” miniprep to check the size of the insert  The band corresponds to the expected size for the insert, which is about 1500 pb.
Purification of Laccase E.coli
01-3502 Laccase E.coli
We produced the E.coli Laccase. We compared non-induced and induced culture on a SDS-PAGE gel (15%) by Coomassie blue staining and in parallel by Western-Blot (anti-His).
We saw that the IPTG induction worked ! So we tried to purify it on Ni-NTA resin. Then we did another SDS-PAGE (15%) with flow through and eluted parts.
We successfully purified it !  But when we did our enzymatic tests on it, we had no activity. We think that maybe we should produce under anaerobic conditions. But we missed time to try it…
01-3002 Laccase TT optimized
  The construction was validated, we tried to purify it under anaerobic condition but we did not succeed…
Enzymatic activity
All tests were performed using laccases and cytochromes C obtained by ISM2 (Institut des Sciences Moléculaires  de Marseille)and LISM (Laboratoire d’Ingénierie des Systèmes Macromoléculaires)).
Question 1: Can the laccase oxidize the cytochrome C?
To answer this question, we used absorbance properties of the cytochrome C (FIG.1). Indeed, the reduced cytochrome C (curve in red) absorbs at 550 nm whereas the oxidized cytochrome C (curve in green) doesn’t absorb. By spectrophotometry, we analyzed the change in oxidation state.  When the cytochrome C is alone, we don’t observe oxidation (FIG.2, blue curve). When we add the laccase, the cytochrome C oxidizes as we can see a decreased absorbance at 550 nm (FIG.2, red curve). We can see the effect of the laccase on the cytochrome C.
Figure 1 : Absorbance properties of cytochrome C
Figure 2: Oxydation of cytochrom C by laccase
Conclusion 1: The laccase can oxidize the cytochrome C.
Question 2: Can the cytochrome C and the laccase oxidize chewing-gum polymer?
The aim of our project is to show that the chewing gum can be degraded by the laccase and the cytochrome C in presence of the light.
Using the X compound, the light and the laccase, the styrene that is a close chewing gum polymer can be oxidized (FIG.3 & FIG.4). The X compound is a rare metal so we don’t want to use it.
Hypothesis:
Could the cytochrome C substitute the X compound to oxidize the styrene?
To test our hypothesis, we used an oxygraph. This engine allows us to measure dioxygen concentration consumed during the reoxidation of the X compound or the cytochrome C by the laccase.
The laccase coupled with the X compound can oxidize the styrene (FIG.4). Replacing the X compound by the cytochrome C, we don’t observe a decreased dioxygen concentration and then the polymer degradation (FIG.4).
Figure 3: Putative schema of reaction with the laccase and the cytochrom C
Conclusion 2: The laccase coupled with the cytochrome C can’t oxidize the styrene.
Figure 4: Oxydation of cytochrom C by laccase
Question 3: Do our enzymes have the same enzymatic activity?
We tried to clone several laccases from T.thermophilus, E.coli, B. subtilis and Laccase 15 (from a uncultured bacterium) and several cytochromes C from S. oneidensis and Synechosystis sp.
We cloned with success two laccases (T. thermophilus and E.coli) and one cytochrome C (S.oneidensis). We tried to produce these proteins into the E.coli BL21 strain.
We purified the E.coli laccase and performed enzymatic tests to determine if it has an activity. To do this, we used the ABTS (2, 2'-azino-bis (3-ethylbenzothiazoline-6-sulphonic acid)), which is oxidized by laccases. This compound is colored in green when it is oxidized.
Unfortunately, we don’t observe activity with purified E.coli laccase. We know that copper binds laccases. So we incubate our laccase with copper and with no success.
Figure 5: Mersure of ABTS oxydation by the laccase from E.coli we purified
Conclusion 3: With or without copper, the purified E.coli laccase can’t oxidize the styrene.
Discussion
We can make some hypothesis concerning these results.
1) We produced the E.coli laccase in aerobic condition. However, the E.coli laccase could have a bad folding in these conditions and then needs to be produced in anaerobic condition.
2) Usually, a laccase shows a blue color given by copper presence. Our purified E.coli laccase is not blue. Maybe the copper is absent. Ni-NTA column is known to remove metal from protein. We can hypothesis that the use of the Ni-NTA is not appropriate.
In perspective, we would like to perform the production of our E.coli laccase in anaerobic condition. Using another purification column such as an anion-exchange column or size exclusion, we hope to conserve copper center and enzymatic activity.
MORE INFORMATION ON FACEBOOK !
SPONSORS
Useful Links
Home
iGEM 2015
Contact Us
Aix Marseille Université
Marseille
France
Chew figth project, for the iGEM competition. See you soon in Boston !
