Team:Aix-Marseille/Notebook
Chew fight
Toggle navigation
Home
Project
Overview
Project design
Parts
Team
Members
Supervisors
Collaborations
Attributions
Notebook
Calendar
Results
Protocols
Practices
Interlab Study
Calendar
Results
Protocols
Sequencing data
Modeling
Modeling
Software
Achievement
CALENDAR
March 16
First meeting with the whole team. Presentation of each member.
The team is composed of Simon Arias, Laure Linet, Marion Aruanno, Camille Houy, Axel Levier, Sebastien Nin, Yoann Chabert, Myriam Choukour, Yassine Cherrak, Daniel Calendini James Strurgis (LISM director) is the team leader.
Gaël Chambonnier is an Instructor. He is a PhD candidate. His thesis is about the transition between the acute and chronic Pseudomonas aeruginosa infections.
First talk about what is iGEM, what are biobricks and what could be our project for this year 2015.
March 20
Talk about the project, the field of the project
March 24
Team registration for iGEM competition.
March 30
Welcome to Wilson, a new member of the Aix-Marseille University team. He will be the biocomputer scientist of the team and will work on modeling and software.
April 2
Welcome to Laureen, a new Instructor of the AMU team. She is a PhD candidate. Her thesis is on the assembly of the bacterial type VI secretion system.
April 12
Presentation of all subject ideas. Election of the best, called “Chew Fight”. We decided to work on the chew degradation using enzymes produced by E.Coli.
April 19
Presentation of our project to the instructors and advisors: they liked it and encouraged us to continue on this way.
April 20
Applying for funding: we sent a grant file to the French embassy in USA.
April 22
Registration in the iGEM website.
April 23
A present from the scientific culture unit of the AMU: a new backpack for each team member: we are ready to come to Boston !
May 5
Creation of the logo of the AMU team :
May 11
Welcome to Yoann Chabert, a new member of the team!
May 18
Presentation of our project to the Institute of Microbiology of the Mediterranean (IMM) for the summer (CNRS - Aix-Marseille Joseph Aiguier)
May 22
Welcome to Romain Clément, a new instructor. He is a PhD candidate and his thesis is on CO2 concentration mechanism study in diatoms (Pseudonana)
June 1
A new supervisor named Valerie Prima joined our team. The team presented our projet “Chew fight” to the laboratory, they liked it and will support us during the summer.
June 10
Our project was published in the local newspaper “Grand Luminy”
June 12
Welcome to Ella De Gaullejaque who comes from Paris and joined the team! The game Marseille VS Paris will begin !
June 15
Meeting with the whole team (members, advisors, instructors) to finalize the timeline of the experiments. We created a money pot on Leetchi.com to collect money online
June 16
The linker between the laccase and the cytochrome has been designed by Wilson.
The sequences needed were ordered (IDT).
June 17
Writing of a newsletter about our project and the synthetic biology. Successful application for being sponsored by the french embassy in USA.
June 18
First day at the laboratory, meeting of everyone. Cleaning and room arrangement.
June 19
Familiarization and training sessions with technics, supervised by the instructors. We learned basics, as running a PCR, and where we can find everything needed to work in the lab.
June 22
Training sessions for PCR
June 23
Training sessions for bacteria transformation
Registration for measurement
June 24
Training session for Gel purification, PCR clean up, plasmid purification (miniprep)
First purchases of the lab material.
June 25
Resuspension of biobricks K863011, K863006, K525998, K844000, E0240, K823005, K823012, I20260 and transformation into E.coli (DH5-alpha).
Negative results (no colony on the plates).
June 26
New try for Resuspension of biobricks K863011, K863006, K525998, K844000, E0240, K823005, K823012, I20260 and transformation into E.coli (DH5-alpha).
Positive results (several colonies on the plate).
June 29
First meeting with the director of innovations and communications from the cleaning company named ONET.
We talked about our idea and our Chew fight project in order to be sponsored.
June 30
Plasmid purification (miniprep) on K863011, K863006, K525998, K844000, E0240, K823005, K823012, I20260 - digestion J23101 et J23115(E/S) et E0240 (X/P) + plasmide (?) I20260 (E/P Dpn1)
July 6
Reception of pipettes borrowed from Polytech school. All the pipettes were calibrated
July 7
Reception of the microcentrifuge, gel electrophoresis, western blot and SDS-page materials, borrowed from Polytech school.
July 8
ONET company presented our project to the Executive office.
July 9
Positive answer from ONET: it sponsored our project with 7000 euros !
July 10
Resuspension of the biobrick “12” (cyt C Shewanella) received from IDT, ligation into pSB1C3, transformation into DH5-alpha.
July 15
Colony PCR on BL21 strain to amplify the 3 parts of ccm operon. Oligos were too concentrated and inhibited the reaction.
Colony PCR and electrophoresis to check the size of the biobrick “12” (cytochrome c Shewanella). Positive result.
Resuspension of biobricks “05”, “08”, “30”, “11” received from IDT. E/P digestion, ligation into pSB1C3, transformation into E.coli (TG1). Negative result (no colony on the plate).
July 16
Second try to resuspend the biobricks “05”, “08”, “30”, “11” received from IDT and resuspension of the biobrick “13” received from IDT. E/P digestion, ligation into pSB1C3, transformation into E.Coli (TG1). Positive result (several colonies on the plate).
Second try for Colony PCR on DH5-alpha strain to amplify the 3 parts of ccm operon with diluted oligos at 1/10. Positive result except for part 1: there are some contaminants due to nonspecific hybridization.
Reception of oligos from SIGMA to amplify the laccase of E.coli and T.thermophilus.
July 17
Third try for colony PCR on DH5-alpha to amplify part 1 of ccm operon. The Tm changed from 55 to 60°C. Positive result.
Colony PCR and electrophoresis to check the size of the biobricks “05”, “08”, “30”, “11” and “13”. Positive results except for “08” et “11”.
Starters of “05”, “13”, “30”.
Plasmid purification (miniprep) of the biobrick “12”, E/P digestion to check the size of the insert. Positive result.
High fidelity DNA amplification of biobricks “09” and “10” by PCR with the Q5 polymerase in order to remove the codon-STOP. PCR clean up. Negative result (nonspecific amplification).
Reception of the biobrick “15” from IDT.
July 20
Colony PCR and electrophoresis to check the size of the biobricks “08” and “11”. Positive result for “08” but not for “11”. Starter of “08”.
New DH5-alpha competent cells done.
July 21
Second try for high fidelity DNA amplification of biobricks “09” and “10” by PCR with the Q5 polymerase in order to remove the codon-STOP (the new synthesized biobricks are called respectively “35” and “36”). PCR clean up. Negative result (non-specific amplification). An electrophoresis was done on the total PCR product and was extracted and purified by PCR clean up.
Miniprep of “08”. E/P digestion to check the size of the insert. Negative result (unexpected size).
Resuspension of biobrick “15” received from IDT. E/P digestion, ligation into pSB1C3, transformation into E.coli (DH5-alpha). Negative result (no colony on the plate).
New TG1 competent cells done.
Transformation efficiency test with the transformation efficiency kit of iGEM on the new DH5-alpha competent cells. Positive result (the DH5-alpha cells are competent enough to be transformed with our constructions).
July 22
Colony PCR and electrophoresis to check the size of the biobricks “05” and “13”. Positive result. Starters of « 05 » and « 13 ».
SLIC reaction to ligate the 3 parts of ccm operon into the pSB1C3 plasmid. Transformation into E.coli (TG1). Negative result (no colony on the plate).
SLIC reaction to ligate the new biobricks “35” and “36” into pSB1C3. Transformation into E.coli (TG1). Negative result (no colony on the plate).
Transformation efficiency test with the transformation efficiency kit of iGEM on the new TG1 competent cells. Positive result (the TG1 cells are competent enough to be transformed with our constructions)
July 23
Miniprep of “05” and “13” and E/P digestion to check the size of the insert.
New try to transform “08” into pSB1C3 into DH5-alpha.
Resuspension of biobricks “09” and “10” from the registry, transformation into TG1.
Construction of “30-02”, “12-02” : E/X digestion of “30” and “12”, S/P digestion of “02”, ligation into pSB1A3, transformation into TG1. Negative result (no colony on the plate).
New try for the SLIC reaction (ccm operon), “35” and “36”. Negative result (no colony on the plate).
July 24
New try with different volumes for resuspension of biobricks “15” and “08” received from IDT. E/P digestion, ligation into pSB1C3, transformation into E.coli (TG1). Positive result (several colonies on the plate).
Colony PCR and electrophoresis to check the size of the biobricks”08”, “09”, “10”,”12-02”, “30-02”, “32” and “34”. Positive result for “09” and “10” (expected size). Starters of « 09 » and « 10 ».
Negative result for “32”, “34”, “12-02”, “30-02” (No colony on the plate) and for “08” (unexpected size).<
New try to construct “30-02”, “12-02”: E/X digestion of “30” and “12”, S/P digestion of “02”, ligation into pSB1A3, transformation into TG1. Positive result (several colonies on the plate).
July 27
Colony PCR and electrophoresis of “08”, “15”, “12-02” and “30-02” to check the size of the insert. Positive results for “15”, “12-02” and “13-02” (expected size) but negative result for “08” (several sizes observed). Starters of “08”, “15”, “12-02” and “30-02”.
Plasmid purification (miniprep) of “05”, “09” and “10”. Problem encountered: genomic DNA contamination.
E/P digestion and electrophoresis to check the size of the insert of “05”, “09”, and “10”.
July 28
Miniprep of “08”, “15”, 12-02”, “30-02”. E/P digestion and electrophoresis to check the size of the insert of “08”, “15”, 12-02”, “30-02”. Positive results for “15”, “12-02”, “30-02” (expected size) and negative result for “08” (unexpected size).
Comment: constructs “12-02” and “30-02” have to be compared with the single “12” and “30” to check if “02” was added.
July 29
Colony PCR and electrophoresis to check the size of the insert of “08”. Negative result
Electrophoresis of “12”, “12-02”, “30” and “30-02”. No result (no DNA observed on the agarose gel)
Construction of “09-02”, “10-02” and “13-02”: E/S digestion of “09”, “10” and “13”. X/P digestion of “02”. Ligation into pSB1A3, transformation into TG1. Positive results for “09-02” and “13-02” (1 colony on the plate). Negative result for “10-02” (no colony on the plate).
Resuspension of biobrick “11” and new try for the “05” and the “08” received from IDT. E/P digestion and ligation into pSB1C3. Transformation into TG1. Positive result for “11” (several colonies on the plate) but not for “05” and “08” (no colony on the plate)
July 30
Colony PCR and electrophoresis to check the size of the insert of « 11 », “09-02” and “13-02”. Uncertain results (problems: DNA is not observed on the agarose gel). “13-02” seems good: starter of “13-02”.
New try with new oligos for the colony PCR and electrophoresis of « 08 ». No result (no DNA observed on the agarose gel).
New try with new volumes for resuspension of the biobrick “05” received from IDT. E/P digestion and ligation into pSB1C3. Transformation into TG1. Negative result (no colony on the plate).
New try to construct “09-02”, “10-02” and “13-02”. E/S digestion of “09”, “10” and “13”. X/P digestion of “02”. Ligation overnight into pSB1A3, transformation into TG1 (07/31). Positive result for “13-02” (several colonies on the plate. Negative result for “09-02” and “10-02” (no colony on the plate).
E/P digestion to check the size of the insert of “30”, “30-02”, “12”, “12-02”. Results seems good for “12” and “12-02”: send for sequencing. Negative results for “30”, “30-02” (unexpected sizes).
July 31
Miniprep of « 13-02 ». E/P digestion and electrophoresis to check the size of the insert. Positive result (expected size).
Colony PCR and electrophoresis to check the size of the insert of “13-02”. Negative result (unexpected size).
Transformation of “09-02”, “10-02” and “13-02” into TG1.
pSB1C3 backbone was running out. New backbone done: E/P digestion on measurement biobricks, electrophoresis and purification of the digested plasmid by gel extraction and PCR clean up.
Problem encountered: no DNA observed on the agarose gel.
August 3
Colony PCR and electrophoresis to check the size of the insert of “09-02”, “10-02” and “13-02”. Positive results (expected sizes). Starters of “09-02”, “10-02” and “13-02.
New try for making pSB1C3 backbone: E/P digestion of measurement biobricks, electrophoresis and purification of the digested plasmid by gel extraction and PCR clean up.
Resuspension of the biobrick “07” (cotA) received from IDT. E/P digestion and ligation into pSB1A3, transformation into TG1
August 4
Colony PCR and electrophoresis to check the size of the insert on “07” => we got a positive clone, starter of this clone launched
Plasmid purification of “13-02” “09-02” “10-02”
E/P digestion of “05” and “11” and ligation “05” + pSB1K3 and “11” + pSB1K3
Transformation of ligation products into E.coli (TG1)
PCR on “07” and “08” => seems to be unclean with the Q5 high fidelity DNA amplification
X/P digestion of “15”
Overnight ligation “12”+”02”+pSB1A3 and “01”+”15”+pSB1T3
“30-02” and “08” got bad results in sequencing
August 5
Transformation of ”01-15” and “12-02” into E.coli (TG1)
Plasmid purification of “09-02” “07” “13-02” “09-02” “10-02” => low concentration of “09-02” but we got good result for “13-02”
Transformation of “12-02” and “01-15” into TG1
PCR clean up “07” “08” “24” ==> low plasmid concentration of “07” and “08” to do again.
August 6
Colony PCR of “05” “11” “12-02” “01-15” => good but only for “12-02”
PCR clean up of “08” and “07” → PCR products contaminated → PCR clean up
Transformation of “10-02” and “09-02”→ Wrong size starter relaunched
Ligation “09-02” and “10-02” into  pSB1A3 and “01-15” into pSB1T3
August 7
Colony PCR of “05” and “01-15” --> no results on this PCR
Transformation of “09-02” “10-02” “01-15” into E.coli (TG1)
Ligation “05” + pSB1K3 and “11” + pSB1K3
Plasmid purification of “12-02” --> Bad results with electrophoresis
For “05” “07” “08” “11” → PCR with Q5 high fidelity polymerase and “PCR clean up”
Preparation of BL21 competent cells
August 10
Colony PCR of “01-15” → No results => Ligation with the digestion product “01-15” relaunched
For “05” “07” “08” “11” : Digestion E/S + Ligation pSB1C3 + Transformation into TG1
For “30” : Digestion E/P + Ligation into pSB1K3
For “01-15” : Ligation into pSB1T3
PCR clean up of “28”
August 11
Colony PCR of “09-02” “10-02” “05” “07” “08” “11”
Preparation of TG1 competent cells
We used the wrong “02” Biobrick we have to do everything again with the good one →  Colony PCR of the “new” “02” →  size = ok
Transformation of “30” “01-01” “02” into TG1
August 12
Preparation of the backbone pSB1A3
Ligation of “05” “08” “07” “11” into pSB1A3
Colony PCR of “30” “21-17” “23-17” “01-15”
For “35” et “36” : Digestion E/S + ligation into pSB1A3 + transformation into TG1
For “37” “38” : Digestion E/P + Ligation into pSB1K3 + transformation into “supercompetent” cells
Plasmid purification of “02” --> Low concentration of plasmid, starter relaunched
For “02” : Digestion X/P
For “13” : Digestion E/S
Ligation of “35-02” “36-02” “12-02” “13-02”  into pSB1A3
PCR  on “09” et “10” to remove the stop codon
Transformation of “05” “07” “08” “11” into  “supercompetent” cells
August 13
Colony PCR on “05” “07” “08” “21-17” “11” “35” “36” “37” “38”
Transformation of “12-02” “13-02” “35-02” “36-02” “36-02” into TG1
PCR clean up of “35” “36”
Digestion E/S of “12” “35” “36” “16” “13”
For “12-02” “13-02” “35-02” “36-02” : Ligation into pSB1A3 + transformation in TG1
For “01-15” : Ligation into pSB1T3 + transformation into TG1
For “16-17” : Ligation into pSB1C3 + transformation into TG1
Colony PCR on “38” “05” “07” “36” ”08” “11”
For “39” “30” : Digestion E/P + Ligation into pSB1K3
August 14
Colony PCR on “12-02” “13-02” “39” “36-02” “23-17” “35-02”
Plasmid purification of “36” “38” “07” “05” “35” “11” “37” --> Verification ok for “36” “38” “35” “37”
August 15
Colony PCR on “05” “07” “08” “11” “16-17” “21-17” “05” “07” “08” “11” “16-17” “21-17” --> Bad results for all
Plasmid purification of  “38” “05” “11” “08” “39” “12-02” “13-02” “35-02” “23-17” “35” “01-15” “16-17” “21-17” --> Problem with the ladder, we cannot read the gel
August 17
For “38” “39” “08” “05” : Digestion E/S
For “12” “13” “39” “35” “35-02” “13-02” “12-02” : Digestion X/P
For “38-12” “38-13” “39-02” “38-39”  “08-02” “36” “05-02”: Ligation into pSB1A3
For “01-3502” “01-1302” “01-1202” “01-35” : Ligation into pSB1T3
For “21-17” “22-17” : Ligation into pSB1C3
==> Transformation for all ligation products into TG1
Colony PCR of “07” “05” “08” “11” with “pool” technic
August 18
Colony PCR of “11” “7” => starters launched on positive clones
Plasmid verification of “05” “07” “08” “11” and “16-17”
“07” and “08” have to be relaunched
Colony PCR on transformants from August 17 => Starters on positive clones launched and the constructions “01-3502” and “38-39” have to be rebuilt
Preparation of the backbone pSB1A3
Ligation of “01-1302” “01-1202” “01-35” into pSB1T3
August 19
Starters launched of “05” and “11”
Preparation of the backbone pSB1A3
Sequencing results “12-02” “13-02” “35-02” “01-15” are ok
“08” is bad and “05” couldn’t be sequenced
Plasmid purification of “30” “38-13” “39-02” “08-02” “05-02” “38-12” “11” “07” “16-17”
“38-13” “39-02” “16-17” “38-12” send for sequencing
“30” “08-02” “05-02” “11” “07” are bad we cannot use its
August 20
Colony PCR of “05-02” “01-1302”
Plasmid purification of “05” and “11” and of 2 starters to make some backbones (pSB1A3 and pSB1T3)
The digestion to prepare pSB1T3 is bad, starter relaunched
Ligation of “38-39” “11-02” into pSB1A3
“38-11” into pSB1C3
“01-1202” “05-02” “01-36” “01-1302” “01-3502” “01-35” “21-17” “22-17” into pSB1K3
Transformation of previous ligation products into DH5-Alpha for “21-17” and “22-17”, into TG1 for “11-02” “38-11” “05-02” and “38-39” and into BL21 for “01-1202” “01-36” “01-1302” “01-3502” and “01-35”
“38-13” “39-02” “11” and “38-12” sent for sequencing
Sequencing result of “05” => bad
Transformation of “18-17” “19-17” “20-17” and “23-17” into DH5-Alpha
Colony PCR of “07”
August 21
Digestion E/S fof “16” “38” “11” “22”, X/P for “17” “39” “02” “11” “35”
Ligation of “16-17” “22-17” + pSB1C3 “38-39” “11-02” “38-11” + pSB1A3
“01-35” “05” “07” “08” “30” + pSB1K3
Transformation of “30” “08” “07” “05” into C2987 (“supercompetent” cells)
Transformation of “38-39” “11-02” “38-11” and “01-35” into TG1
Plasmid purification of “01-1202” “01-3502” “05” “01-1302” “07” and of pSBT3 to get some backbone.
Colony PCR of “08” “01-3502” “01-1202” “01-1302” “01-36”
We got many bad results today so bad luck, but we got some good results also.
August 22
Colony PCR of “38-39” “38-11” “11-02” “01-35”
Starters launched of positive clones
Transformation of overnight ligation produtcs (“16-17” and “22-17” from August 21)
Plasmid purification to make some pSB1A3 and pSB1T3
First protein expression test, we maybe succeed to express our first laccase
August 23
Colony PCR of “16-17” and “22-17” launched overnight => green colonies!!
Starters launched of positive results
Plasmid purification of “22”-17” “16-17” and ‘01-35” => all results are good!!!!
Digestion E/S for “38” and X/P for “39”, ligation overnight into pSB1A3
August 24
Transformation of “30” “08” “07” “11” “05” into TG1
Plasmid purification of “16-17” “01-1302” “01-1202” “22-17” “01-3502”
Colony PCR of “05” “07” “08” “11” => no positive results!
Q5 PCR of “05” “07” “08” “11” “30” => positive results
“05” and “01-35” sent for sequencing
August 25
Transformation of “05” “07” “08” “11” “30” into TG1
Plasmid purification of “01-36” “19-17” “18-17” “23-17” “20-17” => good results
“01-1202” “01-1302” “01-3502” sent for sequencing
Sequencing results of “01-35” “38-12” “38-13” “39-02” => good results “05” “11” => bad results
Starters launched of “39” “01-1302” “01-1202” “01-3502” “01-15” “13-02” “12-02” “38”
August 26
Protein expression => the cytochrome c and the laccase are expressed.
August 27
Isolation and starter launched of “32”
Plasmid purification of “39” “01-1302” “01-1202” “01-3502” “01-15” “13-02” “12-02” “38” “16-17”=> no results!
Ligation of “40” “41” “42” “43” “44” “45” “46” “47” “48” “22-17” + pSB1C3
“30” + pSB1K3
Transformation of “30” “40” “45” “48” into TG1
Transformations didn’t work! => “40” “45” “48” relaunched
Sequencing results “01-1302” => without promoter
“01-1202” and “01-3502” are good
August 31
Colony PCR of “40” “45” “48”
Starters launched of “18-17” “19-17” “20-17” “21-17” “22-17” “23-17” “32” “33” “01-24” each in triplicate
Stock of backbone pSB1C3 done
Preparation of SDS-PAGE, of solutions and of the column to make the protein purification
Digestion E/P of “22-17” “43” “44” “47” and digestion X/P of “1302” “3902”
Ligation “43” “44” “47” into pSB1C3
“013002” into pSB1K3
“01-3902” “01-1302” into pSB1C3
September 1
Colony PCR of “40” “45” “48” all clones are positive, starters launched
Protein purification of “01-3502” “01-1202”
SDS-PAGE of “01-3502” “01-1202”
01-3502 purified and pure!!!! We got our first protein which is a laccase from E.coli
Starters launched of “38” “13-02” “01-36” “01-15” “01-3502” “01-1202” “37” “35-02” “12-02” “35” “36” “01-35” “39” “40” “45” “48”
September 2
Plasmid purification of “01-36” “40” “39” “48” “45” “37” “36” “01-1202” “01-36” “01-15” “35-02” “35” “38” “13-02” “12-02” “01-3502” “01-35”
Protein expression of 01-3502 and 01-1202
Colony PCR of “01-15” beause we have doubt concerning colonies => all colonies were positive
Ligation of “43” “44” “47” “03002” “01-1302” “013902” into pSB1C3
Transformation of “43” “44” “47” “013002” into C2987 and “01-1302” “01-3902” “3813-02” “3812-02” into TG1
September 3
Verification digestion on “40” “38” “01-15” “48” “45” “01-36” “01-1202” “01-36” “37”
Plasmid purification of “01-1202”
Colony PCR of “3812-02” “3813-02” “01-3902” “01-1302” starters launched on positive results
No colony for “43” “44” “47” “013002”
Preparation of the backbone pSB1C3
Transformation of “013002” “41” “44” “47” into C2987
September 4
Plasmid purification of “45” “01-36” “381302” “38” “01-1202” “3812-02” “3813-02”
Colony PCR of “47” “01-3002” “43” “44” => almost everything is positive!!! Great day!
September 7
Plasmid purification (miniprep) of “44” “47” “43” “38” “01-1302” “01-1202” “01-3002” “45”
“01-1302” “01-36” “3813-02” seems good so they are send for sequencing.
Construction : Digestion E/S of “38” “01-36” “01-35” “01” and Digestion X/P of “44” “47” “43” “48” “44” “45” “1302” “3813-02” “39-02” “40” “3812-02”
Starter launched on “01-1202” “01-3002” for production and on “0136” for miniprep
September 8
Ligation in pSB1C3 of “0136-02” “01-381302” “01-381202” “01-3902” “01-1302” “38-3902” “0136-40” “0136-45” “0136-48” “0135-43” “0135-44” “0135-47”
Transformation of “01-3002” in BL21, of pSB4K5 and pSB3T5 in TG1 and transformation of ligation product in TG1
Purification of “01-1202” (1) and (2):
Induction by IPTG
Storing the cells at -80°C
September 9
Colony PCR of “013002” “013002” => all results are positive
Production of “01-1202” : Sonication of cells, centrifugation 10 min at 4500 rpm, ultracentrifugation 45 min at 4500 rpm, solubilizing membrane in Triton 1% during 2h and incubation overnight.
September 10
Preparation of SDS Page 15% and a western blot with products of yesterday => results are not good so we have to resart the production
September 14
Culture of “01-1202” and “01-3002” in anaerobic condition, cells were centrifuged 10 min at 4500 rpm and pellet were stored at -20°C
A western blot was done → Production of Cytochrome, monomer in the membrane and dimer in the cytoplasm
The next step is to purify “01-1202” and “01-3002” and start a culture on 01-3502 in anaerobic condition
MORE INFORMATION ON FACEBOOK !
SPONSORS
Useful Links
Home
iGEM 2015
Contact Us
Aix Marseille Université
Marseille
France
Chew figth project, for the iGEM competition. See you soon in Boston !
