<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Aix-Marseille_Modeling skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Aix-Marseille/Modeling</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Chew fight</TITLE></P><DIV class="container"><DIV class="row"><NAV class="navbar navbar-default " role="navigation" data-spy="affix" data-offset-top="600"><DIV class="navbar-inner"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse  " id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav  " id="nav"><LI><A href="/Team:Aix-Marseille">Home</A></LI><LI class="dropdown " id="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Project</A></LI><LI><A href="/Team:Aix-Marseille/Project/Description">Overview</A></LI><LI><A href="/Team:Aix-Marseille/Design">Project design</A></LI><LI><A href="/Team:Aix-Marseille/Parts">Parts</A></LI></UL><LI class="dropdown " id="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Team</A></LI><LI><A href="/Team:Aix-Marseille/Team">Members</A></LI><LI><A href="/Team:Aix-Marseille/Supervisors">Supervisors</A></LI><LI><A href="/Team:Aix-Marseille/Collaborations">Collaborations</A></LI><LI><A href="/Team:Aix-Marseille/Attributions">Attributions</A></LI><LI class="dropdown " id="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Notebook</A></LI><LI><A href="/Team:Aix-Marseille/Notebook">Calendar</A></LI><LI><A href="/Team:Aix-Marseille/Results">Results</A></LI><LI><A href="/Team:Aix-Marseille/Protocols">Protocols</A></LI><LI><A href="/Team:Aix-Marseille/Practices">Practices</A></LI><LI class="dropdown " id="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Interlab Study</A></LI><LI><A href="/Team:Aix-Marseille/Notebook2">Calendar</A></LI><LI><A href="/Team:Aix-Marseille/Measurement">Results</A></LI><LI><A href="/Team:Aix-Marseille/Protocols2">Protocols</A></LI><LI><A href="/Team:Aix-Marseille/Sequencing/data">Sequencing data</A></LI><LI class="dropdown " id="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Modeling</A></LI><LI><A href="/Team:Aix-Marseille/Modeling">Modeling</A></LI><LI><A href="/Team:Aix-Marseille/Software">Software</A></LI><LI class="dropdown " id="dropdown"><A href="/Team:Aix-Marseille/Achievement" class="dropdown-toggle" data-toggle="dropdown">Achievement</A></LI></DIV></DIV></NAV></DIV></DIV><SECTION style="padding:30px 0px 50px;" class="arrow_box" id="team"><DIV class="container"><DIV class="row"><DIV class="col-md-6 left"><H2 class="title wow bounce in up"><SPAN style="color:#8E3B8C"><SPAN style="font-family:Armalite Rifle">Our idea</SPAN></SPAN></H2><DIV align="justify"><SPAN style="font-family:Courier New">We are delighted to present our models and hope that they will seem clear and explicit.
During our project, we have expressed several enzymes and built several systems. Therefore, we needed enzymatic models and computer simulation to assess their effectiveness, their stability, and improved them.
We initially chose 3 laccases (<I>B.subtilis</I>, <I>E.coli</I>, <I>T.thermophilus</I>) and 3 cytochromes C (<I>Shewanella oneidensis</I>, <I>Synechocystis sp</I>, <I>Paracoccus denitrificans</I>).
The purpose of our simulations was to determine a custom linker for each of our constructions. 
The challenge was to increase the chances of contact between the two enzymes to allow electron transfer. So we had to simulate a system in which our enzymes are pre-positioned. It also involved the selection of a structured linker (alpha helix) to maintain proper conformation.
Another advantage of a linker is a practical one. In fact, it is easier to manipulate one molecule than two. Then, there is no need to observe a proportion ratio.</SPAN></DIV></DIV></DIV></DIV></SECTION><SECTION style="padding:5px 0px 25px;background-color: #FBF1FB;"><DIV style="background:#FBF1FB;"><DIV class="container"><DIV class="row"><H2 class="title wow bounce in up"><SPAN style="color:#000000"><SPAN style="font-family:Armalite Rifle"><DIV align="center">Method</DIV></SPAN></SPAN></H2></DIV><DIV align="justify"><SPAN style="color:#000000"><SPAN style="font-family:courier new">To calculate and view our linkers, we used several bioinformatic tools and structural biology methods. 
We offer a brief summary of these:
<LI><B>Swiss-PdbViewer :</B>
Swiss-PdbViewer (aka DeepView) is an application that provides a user friendly interface allowing to analyze several proteins at the same time. The proteins can be superimposed in order to deduce structural alignments and compare their active sites or any other relevant parts. Amino acid mutations, H-bonds, angles and distances between atoms are easy to obtain thanks to the intuitive graphic and menu interface.</LI><LI><B>Gromacs :</B>
GROMACS is a versatile package to perform molecular dynamics, i.e. simulate the Newtonian equations of motion for systems with hundreds to millions of particles. It is primarily designed for biochemical molecules like proteins, lipids and nucleic acids that have a lot of complicated bonded interactions, but since GROMACS is extremely fast at calculating the nonbonded interactions (that usually dominate simulations) many groups are also using it for research on non-biological systems, e.g. polymers.</LI><LI><B>VMD :</B>
VMD is a molecular visualization program for displaying, animating, and analyzing large biomolecular systems using 3-D graphics and built-in scripting. VMD supports computers running MacOS X, Unix, or Windows, is distributed free of charge, and includes source code. 

All these tools need a pdb file as input. The first step is to download the file pdb corresponding to each of the proteins on RCSB PDB (http://www.rcsb.org/pdb/home/home.do).
Then, you have to pre-position the two proteins using Swiss-pdb Viewer. The electron exchange surface must be up to 15 Angstrom from each other. 
It is continued by a minimization step to eliminate steric strain. This calcul considers electrostatic forces, Van der Wall interactions etc…
A conformation is generated from these constraints. It is sufficient to take the coordinates of the N-term residue (A) and C-term (B) and to calculate the distance (AB) between the two to generate a linker suitable.

AB = √( (xA-xB)^2+ 〖(yA-yB)^2+(zA-zB)〗^2 )

A trun-helix contains 3.6 residues and measures 5.4 Angstrom (1.5 Angstrom per residue).</LI></SPAN></SPAN></DIV></DIV></DIV></SECTION><SECTION style="padding:30px 0px 50px;" class="arrow_box" id="team"><DIV class="container"><DIV class="row"><DIV class="col-md-6 left"><H2 class="title wow bounce in up"><SPAN style="color:#8E3B8C"><SPAN style="font-family:Armalite Rifle">Our modelisation</SPAN></SPAN></H2><SPAN style="font-family:courier new"><U><B>Laccase <I>Thermus thermophilus</I> + Cytochrome C <I>Paracoccus denitrificans</I></B></U></SPAN><P><SPAN style="font-family:courier new">The minimum distance between heme and the electron exchange surface of the laccase is: 29.84 Angstroms. </SPAN></P><P class="space 30"><SPAN style="font-family:courier new">It exceeds the limit distance of 15 Angstroms. This construction seems not very optimized.</SPAN></P><P class="space 30"><SPAN style="font-family:courier new">Order of the construction: 
	CtermCyt – LINKER – NtermLacc </SPAN></P><P class="space 30"><SPAN style="font-family:courier new">Distance calculated between the C-term and N-term:  23.44 Angstroms.</SPAN></P><P class="space30"><SPAN style="font-family:courier new">Linker sequence deducted : 
	<B>GAEAAAKEAAAKEAAAKEAAAKGG</B></SPAN></P></DIV><DIV class="col-md-6 left"><SPAN style="font-family:courier new"><U><B>Laccase <I>E.Coli</I> + Cytochrome C <I>Paracoccus denitrificans</I></B></U></SPAN><P><SPAN style="font-family:courier new">The minimum distance between heme and the electron exchange surface of the laccase is: 15.2 Angstroms. 
 </SPAN></P><P class="space 30"><SPAN style="font-family:courier new">This construct is useful. It can be expected that the structural dynamics allow rappochement less than 15 Angstroms, making a possible oxidation of the cytochrome. 
</SPAN></P><P class="space 30"><SPAN style="font-family:courier new">Order of the construction: 
	CtermCyt – LINKER – NtermLacc
</SPAN></P><P class="space 30"><SPAN style="font-family:courier new">Distance calculated between the C-term and N-term is: 35.37 Angstroms.
</SPAN></P><P class="space30"><SPAN style="font-family:courier new">Linker sequence deducted : 
	<B>GAEAAAKEAAAKEAAAKEAAAKEAAAKGG
</B></SPAN></P></DIV><DIV class="col-md-6 left"><SPAN style="font-family:courier new"><U><B>Laccase <I>Bacillus subtilis </I>CotA + Cytochrome C <I>Paracoccus denitrificans</I></B></U></SPAN><P><SPAN style="font-family:courier new">The minimum distance between heme and the electron exchange surface of the laccase is: 13 Angstroms.

 </SPAN></P><P class="space 30"><SPAN style="font-family:courier new">It is one of our best constructions. The distance seems ideal for electron transfer. We just have to hope that its expression goes well on the bench.

</SPAN></P><P class="space 30"><SPAN style="font-family:courier new">Order of the construction: 
	CtermCyt – LINKER – NtermLacc

</SPAN></P><P class="space 30"><SPAN style="font-family:courier new">Distance between the C-term and N-term is: 16.5 Angstroms.

</SPAN></P><P class="space30"><SPAN style="font-family:courier new">Linker sequence deducted : 
	<B>GAEAAAKEAAAKG</B></SPAN></P></DIV><DIV class="col-md-6 left"><SPAN style="font-family:courier new"><U><B>Laccase <I>Bacillus subtilis </I>CotA + Cytochrome C <I>Shewanella oneidensis </I></B></U></SPAN><P><SPAN style="font-family:courier new">The minimum distance between heme and the electron exchange surface of the laccase is: 17 Angstroms.  
 </SPAN></P><P class="space 30"><SPAN style="font-family:courier new">We can hope to gain a few angstroms thanks to the dynamics of the structure.

</SPAN></P><P class="space 30"><SPAN style="font-family:courier new">Order of the construction: 
	CtermLacc – LINKER – NtermCyt

</SPAN></P><P class="space 30"><SPAN style="font-family:courier new">Distance between the C-term and N-term is: 44.5 Angstroms
</SPAN></P><P class="space30"><SPAN style="font-family:courier new">Linker sequence deducted : 
	<B>GGAEAAAKEAAAKAEAAAKEAAAKAEAAAKEAAAKGG
</B></SPAN></P></DIV><DIV class="col-md-6 left"><SPAN style="font-family:courier new"><U><B>Laccase <I>E.Coli</I> + Cytochrome C <I>Shewanella oneidensis </I></B></U></SPAN><P><SPAN style="font-family:courier new">The minimum distance between heme and the electron exchange surface of the laccase is: 19.5 Angstroms. 
 </SPAN></P><P class="space 30"><SPAN style="font-family:courier new">We can hope to gain a few angstroms thanks to the dynamics of the structure.

</SPAN></P><P class="space 30"><SPAN style="font-family:courier new">Order of the construction: 
	CtermCyt – LINKER – NtermLacc

</SPAN></P><P class="space 30"><SPAN style="font-family:courier new">Distance between the C-term and N-term is:55.48 Angstroms.
</SPAN></P><P class="space30"><SPAN style="font-family:courier new">Linker sequence deducted : 
	<B>GAEAAAKEAAAKEAAAKEAAAKEAAAKEAAAKEAAAKG
</B></SPAN></P></DIV><DIV class="col-md-6 left"><SPAN style="font-family:courier new"><U><B>Laccase <I>Thermus thermophilus  </I> + Cytochrome C <I>Shewanella oneidensis </I></B></U></SPAN><P><SPAN style="font-family:courier new">The minimum distance between heme and the electron exchange surface of the laccase is: 19.4 Angstroms.  
 </SPAN></P><P class="space 30"><SPAN style="font-family:courier new">Same analyse, we can hope to gain a few angstroms thanks to the dynamics of the structure.

</SPAN></P><P class="space 30"><SPAN style="font-family:courier new">Order of the construction: 
	CtermLacc – LINKER – NtermCyt
</SPAN></P><P class="space 30"><SPAN style="font-family:courier new">Distance between the C-term and N-term is: 38.65 Angstroms.
</SPAN></P><P class="space30"><SPAN style="font-family:courier new">Linker sequence deducted : 
	<B>GAEAAAKEAAAKEAAAKEAAAKEAAAKG
</B></SPAN></P></DIV><DIV class="col-md-6 left"><SPAN style="font-family:courier new"><U><B>Laccase <I>Bacillus subtilis </I> CotA + Cytochrome C <I>Synechocystis sp</I></B></U></SPAN><P><SPAN style="font-family:courier new">The minimum distance between heme and the electron exchange surface of the laccase is: 15.29 Angstroms. 

 </SPAN></P><P class="space 30"><SPAN style="font-family:courier new">Order of the construction: 
	CtermCyt – LINKER – NtermLacc

</SPAN></P><P class="space 30"><SPAN style="font-family:courier new">Distance between the C-term and N-term is:21.6 Angstroms.

</SPAN></P><P class="space30"><SPAN style="font-family:courier new">Linker sequence deducted : 
	<B>GAEAAAKEAAAKG
</B></SPAN></P></DIV><DIV class="col-md-6 left"><SPAN style="font-family:courier new"><U><B>Laccase <I>E.Coli</I> + Cytochrome C <I>Synechocystis sp
</I></B></U></SPAN><P><SPAN style="font-family:courier new">The minimum distance between heme and the electron exchange surface of the laccase is: 38.53 Angstroms.

 </SPAN></P><P class="space 30"><SPAN style="font-family:courier new">It is the worst construction. The heme/copper laccase is way too far to allow oxidation.

</SPAN></P><P class="space 30"><SPAN style="font-family:courier new">Order of the construction: 
	CtermCyt – LINKER – NtermLacc

</SPAN></P><P class="space 30"><SPAN style="font-family:courier new">Distance calculated between the C-term and N-term is: 40.44 Angstroms. 

</SPAN></P><P class="space30"><SPAN style="font-family:courier new">Linker sequence deducted : 
	<B>GGAEAAAKEAAAKEAAAKEAAAKEAAAKEAAAKGG

</B></SPAN></P></DIV><DIV class="col-md-6 left"><SPAN style="font-family:courier new"><U><B>Laccase <I>Thermus thermophilus </I> + Cytochrome C <I>Synechocystis sp</I></B></U></SPAN><P><SPAN style="font-family:courier new">The minimum distance between heme and the electron exchange surface of the laccase is: 33.23 Angstroms. 

 </SPAN></P><P class="space 30"><SPAN style="font-family:courier new">It is one of the worst construction. The heme/copper laccase is way too far to allow oxidation.

</SPAN></P><P class="space 30"><SPAN style="font-family:courier new">Order of the construction: 
	CtermCyt – LINKER – NtermLacc

</SPAN></P><P class="space 30"><SPAN style="font-family:courier new">Distance calculated between the C-term and N-term is: 40.6 Angstroms.

</SPAN></P><P class="space30"><SPAN style="font-family:courier new">Linker sequence deducted : 
	<B>GAEAAAKEAAAKEAAAKEAAAKEAAAKG

</B></SPAN></P></DIV></DIV></DIV></SECTION></DIV><SECTION style="padding:50px 0px;" class="arrow_box" id="ethics"><DIV class="container"><DIV class="row"><DIV class="col-md-12 text-center"><H1 class="blue">MORE INFORMATION ON FACEBOOK !</H1></DIV></DIV></DIV></SECTION><SECTION style="padding:50px 0px;" class="arrow_box-2"><DIV class="container"><DIV class="row"><DIV class="col-md-12 text-center"><H2 class="gray">SPONSORS</H2></DIV></DIV></DIV></SECTION><FOOTER><DIV class="container"><DIV class="row"><DIV class="col-md-4"><H3>Useful Links</H3><UL class="f-menu space40"><LI><A href="/Team:Aix-Marseille">Home</A></LI><LI><A href="https://2015.igem.org/Main_Page">iGEM 2015</A></LI></UL></DIV><DIV class="col-md-4"><H3>Contact Us</H3><UL class="f-menu space40"><LI>Aix Marseille Université</LI><LI>Marseille</LI><LI>France</LI></UL></DIV><DIV class="col-md-4"><DIV class="f-text"><P class="fsize18 space10">Chew figth project, for the iGEM competition. See you soon in Boston !</P></DIV></DIV></DIV></DIV></FOOTER></DIV></DIV></DIV></BODY></HTML>