Team:Aix-Marseille/Modeling
Chew fight
Toggle navigation
Home
Project
Overview
Project design
Parts
Team
Members
Supervisors
Collaborations
Attributions
Notebook
Calendar
Results
Protocols
Practices
Interlab Study
Calendar
Results
Protocols
Sequencing data
Modeling
Modeling
Software
Achievement
Our idea
We are delighted to present our models and hope that they will seem clear and explicit. During our project, we have expressed several enzymes and built several systems. Therefore, we needed enzymatic models and computer simulation to assess their effectiveness, their stability, and improved them. We initially chose 3 laccases (B.subtilis, E.coli, T.thermophilus) and 3 cytochromes C (Shewanella oneidensis, Synechocystis sp, Paracoccus denitrificans). The purpose of our simulations was to determine a custom linker for each of our constructions.  The challenge was to increase the chances of contact between the two enzymes to allow electron transfer. So we had to simulate a system in which our enzymes are pre-positioned. It also involved the selection of a structured linker (alpha helix) to maintain proper conformation. Another advantage of a linker is a practical one. In fact, it is easier to manipulate one molecule than two. Then, there is no need to observe a proportion ratio.
Method
To calculate and view our linkers, we used several bioinformatic tools and structural biology methods.  We offer a brief summary of these:
Swiss-PdbViewer : Swiss-PdbViewer (aka DeepView) is an application that provides a user friendly interface allowing to analyze several proteins at the same time. The proteins can be superimposed in order to deduce structural alignments and compare their active sites or any other relevant parts. Amino acid mutations, H-bonds, angles and distances between atoms are easy to obtain thanks to the intuitive graphic and menu interface.
Gromacs : GROMACS is a versatile package to perform molecular dynamics, i.e. simulate the Newtonian equations of motion for systems with hundreds to millions of particles. It is primarily designed for biochemical molecules like proteins, lipids and nucleic acids that have a lot of complicated bonded interactions, but since GROMACS is extremely fast at calculating the nonbonded interactions (that usually dominate simulations) many groups are also using it for research on non-biological systems, e.g. polymers.
VMD : VMD is a molecular visualization program for displaying, animating, and analyzing large biomolecular systems using 3-D graphics and built-in scripting. VMD supports computers running MacOS X, Unix, or Windows, is distributed free of charge, and includes source code.   All these tools need a pdb file as input. The first step is to download the file pdb corresponding to each of the proteins on RCSB PDB (http://www.rcsb.org/pdb/home/home.do). Then, you have to pre-position the two proteins using Swiss-pdb Viewer. The electron exchange surface must be up to 15 Angstrom from each other.  It is continued by a minimization step to eliminate steric strain. This calcul considers electrostatic forces, Van der Wall interactions etc… A conformation is generated from these constraints. It is sufficient to take the coordinates of the N-term residue (A) and C-term (B) and to calculate the distance (AB) between the two to generate a linker suitable.  AB = √( (xA-xB)^2+ 〖(yA-yB)^2+(zA-zB)〗^2 )  A trun-helix contains 3.6 residues and measures 5.4 Angstrom (1.5 Angstrom per residue).
Our modelisation
Laccase Thermus thermophilus + Cytochrome C Paracoccus denitrificans
The minimum distance between heme and the electron exchange surface of the laccase is: 29.84 Angstroms.
It exceeds the limit distance of 15 Angstroms. This construction seems not very optimized.
Order of the construction:   CtermCyt – LINKER – NtermLacc
Distance calculated between the C-term and N-term:  23.44 Angstroms.
Linker sequence deducted : GAEAAAKEAAAKEAAAKEAAAKGG
Laccase E.Coli + Cytochrome C Paracoccus denitrificans
The minimum distance between heme and the electron exchange surface of the laccase is: 15.2 Angstroms.
This construct is useful. It can be expected that the structural dynamics allow rappochement less than 15 Angstroms, making a possible oxidation of the cytochrome.
Order of the construction:   CtermCyt – LINKER – NtermLacc
Distance calculated between the C-term and N-term is: 35.37 Angstroms.
Linker sequence deducted : GAEAAAKEAAAKEAAAKEAAAKEAAAKGG
Laccase Bacillus subtilis CotA + Cytochrome C Paracoccus denitrificans
The minimum distance between heme and the electron exchange surface of the laccase is: 13 Angstroms.
It is one of our best constructions. The distance seems ideal for electron transfer. We just have to hope that its expression goes well on the bench.
Order of the construction:   CtermCyt – LINKER – NtermLacc
Distance between the C-term and N-term is: 16.5 Angstroms.
Linker sequence deducted : GAEAAAKEAAAKG
Laccase Bacillus subtilis CotA + Cytochrome C Shewanella oneidensis
The minimum distance between heme and the electron exchange surface of the laccase is: 17 Angstroms.
We can hope to gain a few angstroms thanks to the dynamics of the structure.
Order of the construction:   CtermLacc – LINKER – NtermCyt
Distance between the C-term and N-term is: 44.5 Angstroms
Linker sequence deducted : GGAEAAAKEAAAKAEAAAKEAAAKAEAAAKEAAAKGG
Laccase E.Coli + Cytochrome C Shewanella oneidensis
The minimum distance between heme and the electron exchange surface of the laccase is: 19.5 Angstroms.
We can hope to gain a few angstroms thanks to the dynamics of the structure.
Order of the construction:   CtermCyt – LINKER – NtermLacc
Distance between the C-term and N-term is:55.48 Angstroms.
Linker sequence deducted : GAEAAAKEAAAKEAAAKEAAAKEAAAKEAAAKEAAAKG
Laccase Thermus thermophilus + Cytochrome C Shewanella oneidensis
The minimum distance between heme and the electron exchange surface of the laccase is: 19.4 Angstroms.
Same analyse, we can hope to gain a few angstroms thanks to the dynamics of the structure.
Order of the construction:   CtermLacc – LINKER – NtermCyt
Distance between the C-term and N-term is: 38.65 Angstroms.
Linker sequence deducted : GAEAAAKEAAAKEAAAKEAAAKEAAAKG
Laccase Bacillus subtilis CotA + Cytochrome C Synechocystis sp
The minimum distance between heme and the electron exchange surface of the laccase is: 15.29 Angstroms.
Order of the construction:   CtermCyt – LINKER – NtermLacc
Distance between the C-term and N-term is:21.6 Angstroms.
Linker sequence deducted : GAEAAAKEAAAKG
Laccase E.Coli + Cytochrome C Synechocystis sp
The minimum distance between heme and the electron exchange surface of the laccase is: 38.53 Angstroms.
It is the worst construction. The heme/copper laccase is way too far to allow oxidation.
Order of the construction:   CtermCyt – LINKER – NtermLacc
Distance calculated between the C-term and N-term is: 40.44 Angstroms.
Linker sequence deducted : GGAEAAAKEAAAKEAAAKEAAAKEAAAKEAAAKGG
Laccase Thermus thermophilus + Cytochrome C Synechocystis sp
The minimum distance between heme and the electron exchange surface of the laccase is: 33.23 Angstroms.
It is one of the worst construction. The heme/copper laccase is way too far to allow oxidation.
Order of the construction:   CtermCyt – LINKER – NtermLacc
Distance calculated between the C-term and N-term is: 40.6 Angstroms.
Linker sequence deducted : GAEAAAKEAAAKEAAAKEAAAKEAAAKG
MORE INFORMATION ON FACEBOOK !
SPONSORS
Useful Links
Home
iGEM 2015
Contact Us
Aix Marseille Université
Marseille
France
Chew figth project, for the iGEM competition. See you soon in Boston !
