<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Tokyo_Tech_Projects_PHAs_index_htm"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Tokyo_Tech/Projects/PHAs/index.htm">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Tokyo_Tech/Projects/PHAs/index.htm&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Tokyo_Tech/Projects/PHAs/index.htm&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Tokyo_Tech/Projects/PHAs/index.htm&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Tokyo_Tech/Projects/PHAs/index.htm" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Tokyo Tech/Projects/PHAs/index.htm</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><P><TITLE>bar</TITLE></P><DIV id="navigation"><UL id="TopMenu"><LI id="menu_Home"><A href="https://2012.igem.org/Team:Tokyo_Tech">Home</A></LI><LI id="menu_Project">
				Project
				</LI><LI><A href="https://2012.igem.org/Team:Tokyo_Tech/Project">Cell-cell communication    </A></LI><LI><A href="https://2012.igem.org/Team:Tokyo_Tech/Projects/PHAs/index.htm">P(3HB) Production
 </A></LI></UL><LI id="menu_Extra">
				Experiments
				</LI><LI><A href="https://2012.igem.org/Team:Tokyo Tech/Experiment/C6">3OC6HSL-dependent 3OC12HSL production
</A></LI><LI><A href="https://2012.igem.org/Team:Tokyo Tech/Experiment/C12">3OC12HSL-dependent 3OC6HSL production
</A></LI><LI><A href="https://2012.igem.org/Team:Tokyo Tech/Experiment/positivefeedback1">Positive feedback assay
~Co-culture assay~
</A></LI><LI><A href="https://2012.igem.org/Team:Tokyo Tech/positivefeedback2">Positive feedback assay~Time-dependent change assay~</A></LI><LI><A href="https://2012.igem.org/Team:Tokyo Tech/Experiment/banddetect">Lux/Tet hybrid promoter assay
</A></LI><LI><A href="https://2012.igem.org/Team:Tokyo Tech/Experiment/PHB">P(3HB) Production</A></LI><LI><A href="https://2012.igem.org/Team:Tokyo Tech/Experiment/PHB2">Making purified P(3HB) sheets
</A></LI><LI id="menu_Home"><A href="https://2012.igem.org/Team:Tokyo_Tech/Modeling">Modeling</A></LI><LI id="menu_data_page"><A href="https://2012.igem.org/Team:Tokyo Tech/Parts">Parts Submitted to the Registry</A></LI><LI id="menu_Human Practice"><A href="https://2012.igem.org/Team:Tokyo_Tech/HumanPractice.htm">Human Practice</A></LI><LI id="menu_Extra">
				More
				</LI><LI><A href="https://2012.igem.org/Team:Tokyo Tech/Safety">Safety</A></LI><LI><A href="https://2012.igem.org/Team:Tokyo Tech/Attributions">Attribution and contribution</A></LI><LI><A href="https://2012.igem.org/Team:Tokyo Tech/Notebook">NoteBook</A></LI><LI><A href="https://2012.igem.org/Team:Tokyo Tech/Team">Team</A></LI><LI><A href="https://igem.org/Team.cgi?year=2012&amp;team_name=Tokyo_Tech">Official Team Profile</A></LI><LI><A href="https://2012.igem.org/Team:Tokyo Tech/Comments.htm">Comments</A></LI><LI><A href="https://2012.igem.org/Team:Tokyo Tech/Sponsers.htm">Sponsors</A></LI></DIV><DIV class="whitebox"><DIV id="tokyotech" style="font:bold ;left ; font-size: 50px; color: #1E90FF; padding: 10px;">
P(3HB) Production </DIV></DIV><DIV class="whitebox"><DIV id="tokyotech" style="font:Arial ;left ; font-size: 15px; color: #000000; padding: 30px;"><DIV class="thumb tright"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">Fig2-2-1-1, Rose silhouette on the LB agar plate containing Nile red.</DIV></DIV></DIV><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Achivement"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Achivement</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#What_is_PHAs.3F"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">What is PHAs?</SPAN></A></LI><LI class="toclevel-1 tocsection-3"><A href="#Construction_of_phaC1-A-B1_in_Biobrick_format"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Construction of phaC1-A-B1 in Biobrick format</SPAN></A></LI><LI class="toclevel-1 tocsection-4"><A href="#P.283HB.29_production_by_E.coli_.26_Confirmation_of_P.283HB.29"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">P(3HB) production by E.coli &amp; Confirmation of P(3HB)</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#4-1_Confirmation_of_P.283HB.29_synthesized_on_colonies"><SPAN class="tocnumber">4.1</SPAN><SPAN class="toctext">4-1 Confirmation of P(3HB) synthesized on colonies</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#4-2_Confirmation_of_P.283HB.29_accumulated_in_cells"><SPAN class="tocnumber">4.2</SPAN><SPAN class="toctext">4-2 Confirmation of P(3HB) accumulated in cells</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#4-3_Confirmation_of_P.283HB.29_by_GC.2FMS"><SPAN class="tocnumber">4.3</SPAN><SPAN class="toctext">4-3 Confirmation of P(3HB) by GC/MS</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#4-4_Optimization_of_the_best_culture_condition_to_synthesize_P.283HB.29"><SPAN class="tocnumber">4.4</SPAN><SPAN class="toctext">4-4 Optimization of the best culture condition to synthesize P(3HB)</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-9"><A href="#Application"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Application</SPAN></A></LI><LI class="toclevel-1 tocsection-10"><A href="#Reference"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">Reference</SPAN></A></LI></TD></TR></TBODY></TABLE><DIV id="tokyotech" style="font:bold ;left ; font-size: 30px; color: #0000FF; padding: 2px;">
1.</DIV><H1><SPAN class="mw-headline" id="Achivement">Achivement</SPAN></H1><P>We made a new biobrick part and succeeded in synthesizing Polyhydroxyalkanoates(PHAs). This is the first Biobrick part to synthesize P(3HB), a kind of PHAs. We identified the products as P(3HB) by GC/ MS, and optimised best culture condition to synthesize P(3HB).
</P><P>In our project, we also drew rose silhouette to reproduce the balcony scene of “Romeo and Juliet” by the synthesis of  P(3HB).
</P><DIV id="tokyotech" style="font:bold ;left ; font-size: 30px; color: #0000FF; padding: 2px;">
2.</DIV><H1><SPAN class="mw-headline" id="What_is_PHAs.3F">What is PHAs?</SPAN></H1><P>Polyhydroxyalkanoates(PHAs) are biological polyester synthesized by a wide range of bacteria, and can be produced by fermentation from renewable carbon sources such as sugars and vegetable oils. These polyesters are biodegradable thermoplastics and elastomers, which exhibit interesting material properties. PHAs are also a kind of bio plastics, which can be biodegraded a lot faster than fossil-fuel plastics in the environment. Poly-3-hydroxybutyrate, P(3HB) is the most common type of PHAs. P(3HB) is synthesized by the enzymes coded in the gene of PHA synthesis (<I>pha C1-A-B1</I>) from <I>Ralstonia eutropha</I> H16.
</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Fig2-2-2-1, Gene of PHA synthesis (<I>pha C1-A-B1</I>) from <I>Ralstonia eutropha</I> H16.</DIV></DIV></DIV><P>
Poly-3-hydroxybutyrate, P(3HB) is synthesized by three enzymes.
</P><P>The A gene encodes for the 393 amino acids protein, 3-ketothiolase (PhaA)
</P><P>The B gene encodes for the 246 amino acids protein, acetoacetyl-CoA  reductase (PhaB)
</P><P>The C gene encodes for the 589 amino acids protein, PHA Synthase (PhaC)
</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:152px;"><DIV class="thumbcaption">Fig2-2-2-2, synthesis mechanism of P(3HB)</DIV></DIV></DIV><P>
The pathway and regulation of Poly[(R)-3-hydroxybutyrate], P(3HB), synthesis in <I>Ralstonia eutropha</I> H16 is shown in Fig2-2-2-2. Pyruvic acid is metabolized from glucose by glycolysis, and pyruvate dehydrogenase complex (PDC) transforms pyruvic acid into acetyl-CoA. At first, two molecules of acetyl-CoA are ligated to one molecule acetoacetyl-CoA by the action of 3-ketothiolase (coded in PhaA). Acetoacetyl-CoA is transformed into (R)-3-hydroxybutyl-CoA by NADPH dependent acetoacetyl-CoA reductase (coded in PhaB). P(3HB) is then synthesized by the polymerization of (R)-3-hydroxybutyryl-CoA by the action of PHA synthase (PhaC).(<A href="#Reference">[1][2]</A>
)
</P><DIV id="tokyotech" style="font:bold ;left ; font-size: 30px; color: #0000FF; padding: 2px;">
3.</DIV><H1><SPAN class="mw-headline" id="Construction_of_phaC1-A-B1_in_Biobrick_format">Construction of <I>phaC1-A-B1</I> in Biobrick format</SPAN></H1><P>In this study, we constructed a part containing <I>phaC1-A-B1</I> in Biobrick format(<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K934001" class="external text" rel="nofollow">BBa_K934001</A>).[<A href="https://2012.igem.org/Team:Tokyo_Tech/Experiment/PHB#Construction_of_pha-C1-A-B1_in_Biobrick_format" class="external text" rel="nofollow">Construction of <I>PHA</I>-C1-A-B1 in Biobrick format</A>]
This is the first Biobrick part which worked as expected though some teams had tried to synthesize PHAs in the past iGEM.[<A href="https://2012.igem.org/Team:Tokyo_Tech/Projects/PHAs/detail/index.htm#Production_trial_of_PHAs_by_past_teams" class="external text" rel="nofollow">Production trial of <I>PHA</I>s by past teams</A>]
</P><DIV id="tokyotech" style="font:bold ;left ; font-size: 30px; color: #0000FF; padding: 2px;">
4.</DIV><H1><SPAN class="mw-headline" id="P.283HB.29_production_by_E.coli_.26_Confirmation_of_P.283HB.29">P(3HB) production by <I>E.coli</I>&amp; Confirmation of P(3HB)</SPAN></H1><P>To synthesize P(3HB) by <I>E.coli</I>, we transformed <I>E.coli</I> JM109 with the constructed <I>pha C1-A-B1</I> part on pSB1C3 (<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K934001" class="external text" rel="nofollow">BBa_K934001</A>). <I>E.coli</I> JM109 is used to synthesize P(3HB), because it tends to have a high density accumulation of P(3HB)(<A href="#Reference">[5]</A>
). As a negative control, we transformed <I>E.coli</I> JM109 with PlasI-gfp on pSB1C3.
</P><H2><SPAN class="mw-headline" id="4-1_Confirmation_of_P.283HB.29_synthesized_on_colonies">4-1 Confirmation of P(3HB) synthesized on colonies</SPAN></H2><P>We observed the accumulation of P(3HB) in the <I>E.coli</I> colonies on Nile red positive medium under UV. Nile red has been widely used to stain colonies and distinguish between PHA-accumulating and non-accumulating colonies. Nile red in the agar medium doesn’t affect the growth of the cells, and the accumulation of PHAs in the colonies can be directly monitored(<A href="#Reference">[3][4][5]</A>
). We cultured the transformant on LB agar medium plates with Nile red. After several days, colonies storing P(3HB) were stained orange by Nile red when observed under UV. This result indicates that transformant synthesized and stored P(3HB).
Fig2-2-4-1-1 is the photographs of <I>E.coli</I> colonies on Nile red positive medium taken under UV. The orange colonies in Fig2-2-4-1-1A show that the accumulated P(3HB) in cells was stained by Nile red. This result indicates that part <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K934001" class="external text" rel="nofollow">BBa_K934001</A> synthesized P(3HB). Fig2-2-4-1-1B is the photograph of negative control cells. In this figure we observed that there were no remarkable colored colonies. Fig2-2-4-1-2 shows the difference between cells storing P(3HB) and those not storing P(3HB) on one plate. The cells in blue rectangle area are the cells with P(3HB) synthesis gene and the cells in green rectangle area are the cells with PlasI-gfp gene as a negative control. Using the cells storing P(3HB), we drew a rose silhouette on the LB agar plate containing Nile red (Fig2-2-4-1-3).[<A href="https://2012.igem.org/Team:Tokyo_Tech/Experiment/PHB#A_.P.283HB.29_production_on_colonies_and_preparation_before_confirmation_with_Nile_red_under_UV" class="external text" rel="nofollow">Protocol</A>]
</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Fig2-2-4-1-1   Fig2-2-4-1-1A: <I>E.coli</I> JM109 colonies with <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K934001" class="external text" rel="nofollow">BBa_K934001</A> gene, PHB accumulation  Fig2-2-4-1-1B: <I>E.coli</I> JM109 colonies with PlasI-gfp gene, no P(3HB) accumulation</DIV></DIV></DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Fig2-2-4-1-2, Difference between cells storing P(3HB) and cells not storing P(3HB). Blue rectangle: with <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K934001" class="external text" rel="nofollow">BBa_K934001</A> gene, PHB accumulation. Green rectangle: with PlasI-gfp gene, no PHB accumulation</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:152px;"><DIV class="thumbcaption">Fig2-2-4-1-3, Rose silhouette on the LB agar plate containing Nile red.</DIV></DIV></DIV><H2><SPAN class="mw-headline" id="4-2_Confirmation_of_P.283HB.29_accumulated_in_cells">4-2 Confirmation of P(3HB) accumulated in cells</SPAN></H2><P>To confirm the accumulation condition of P(3HB) in <I>E.coli</I> with a microscope, we stained the P(3HB) with Nile blue A reagent. Nile blue A is also used to detect the existence of P(3HB) and has no toxicity to the cells(<A href="#Reference">[5]</A>). Before the observation, we stained the dried cells with Nile blue A solution. We then took photographs of the sample under fluorescence microscope. 
Fig2-2-4-2-1 is the photograph of dried <I>E.coli</I> (with <I>pha C1-A-B1</I> gene) cells dyed with Nile blue A solution taken by fluorescence microscope. The fluorescent areas in Fig2-2-4-2-1A are the accumulated P(3HB) in the cells. This result also indicates that part <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K934001" class="external text" rel="nofollow">BBa_K934001</A> synthesized P(3HB). In the photograph of negative control (Fig2-2-4-2-1B), no remarkable fluorescent area was observed.[<A href="https://2012.igem.org/Team:Tokyo_Tech/Experiment/PHB#B.P.283HB.29_production_in_cells_and_preparation_before_the_confirmation_with_Nile_blue_A" class="external text" rel="nofollow">Protocol</A>]
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:802px;"><DIV class="thumbcaption">Fig2-2-4-2-1A, <I>E.coli</I> JM109 dried cells with P(3HB) accumulation stained by Nile blue A  Fig2-2-4-2-1B, <I>E.coli</I> JM109 dried cells without P(3HB) accumulation stained by Nile blue A</DIV></DIV></DIV></DIV><H2><SPAN class="mw-headline" id="4-3_Confirmation_of_P.283HB.29_by_GC.2FMS">4-3 Confirmation of P(3HB) by GC/MS</SPAN></H2><P>We successfully identified the products by <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K934001" class="external text" rel="nofollow">BBa_K934001</A> as 3HB, monomer of P(3HB), by Gas Chromatography/ Mass Spectrometry (GC/ MS). To confirm the products using GC/ MS, the products are methylated because 3HB is difficult to measure. Fig. 2-2-4-3-1 shows the GC/ MS result of the products by <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K934001" class="external text" rel="nofollow">BBa_K934001</A>. The peaks of sample are same to those of standard control of methylated 3HB. This shows that <I>E.coli</I> synthesized P(3HB) correctly.
[<A href="https://2012.igem.org/Team:Tokyo_Tech/Experiment/PHB#C._Preparation_for_GC.2FMS" class="external text" rel="nofollow">Protocol</A>]
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:802px;"><DIV class="thumbcaption">Fig2-2-4-3-1, Result of GC/MS</DIV></DIV></DIV></DIV><H2><SPAN class="mw-headline" id="4-4_Optimization_of_the_best_culture_condition_to_synthesize_P.283HB.29">4-4 Optimization of the best culture condition to synthesize P(3HB)</SPAN></H2><P>To figure out best culture condition, we tried culturing <I>E.coli</I> JM109 in 10 different conditions for 48h. Each condition is shown in Fig.2-2-4-4-1. Composition of LB and TB medium is shown in Fig. 2-2-4-4-2.
</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Fig2-2-4-4-1, different conditions</DIV></DIV></DIV><P>
Pantothenic acid (PA), also called vitamin B5 is required to synthesize coenzyme A (CoA). If the glycolytic pathway has become a rate-limiting step, P(3HB) synthesis would be more efficiently by adding PA.
</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:382px;"><DIV class="thumbcaption">Fig2-2-4-4-2, Composition of LB &amp; TB</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">Fig2-2-4-4-3, Structure of Pantothenic acid</DIV></DIV></DIV><P>
The culture result is shown in Fig. 2-2-4-4-4.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:802px;"><DIV class="thumbcaption">Fig2-2-4-5-4, Culture results of ten conditions</DIV></DIV></DIV></DIV><UL><LI>“Dried cells (g/L)” is the amount of the cells in the medium after culturing.
</LI></UL><UL><LI>“Polymer content rate (%)” is the rate of the polymer in the dried cells.
</LI></UL><UL><LI>“Polymer concentration (g/L)” is the amount of the polymer in the medium after culturing. This value is calculated by multiplying “Dried cells” and “Polymer content rate”.
</LI></UL><P>
The results showed that TB medium was much better than LB medium to synthesize P(3HB). In both LB and TB, in the 37°C culturing containing glucose and PA-Ca, <I>E.coli</I> synthesized the polymer in maximum content rate. However, the growth of <I>E.coli</I> in 37°C was worse than that in 30°C, therefore final polymer concentration in 37°C and 30°C didn’t make a significant difference. Even if there was no glucose, <I>E.coli</I> synthesized polymer (condition 9 &amp; 10). We think that TB medium had glycerol and a lot of yeast extra, and then<I>E.coli</I> might have used them as carbon sources. In addition, the comparison of condition 4 &amp; 5 indicates PA-Ca was not used as carbon sources. LB medium didn’t contain many carbon sources, so <I>E.coli</I> synthesized little polymer. In this case, adding PA-Ca didn’t have big effect. On the other hand TB medium contains enough carbon sources, so we think that the rate-limiting step was the glycolytic pathway. In this case, polymer production would be increased by adding PA-Ca. (the comparison of condition 7 &amp; 8 and 9&amp;10)
[<A href="https://2012.igem.org/Team:Tokyo_Tech/Experiment/PHB#D._Optimization_of_the_best_culture_condition_to_synthesize_P.283HB.29." class="external text" rel="nofollow">Protocol</A>]
</P><DIV id="tokyotech" style="font:bold ;left ; font-size: 30px; color: #0000FF; padding: 2px;">
5.</DIV><H1><SPAN class="mw-headline" id="Application">Application</SPAN></H1><DIV class="thumb tright"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">Fig2-2-5-1, PHA synthesis gene expression spatially manipulated</DIV></DIV></DIV><P>The achievement of our project “P(3HB) Production” is that we registered available P(3HB) synthetic gene in Biobrick parts. We can control the expression of the P(3HB) synthetic gene spatially by using combination of Biobrick parts. What we want to claim as an example of the spatial manipulation of gene expression is water-repellent. A stronger water-repellent requires hydrophobicity as well as the increase in real surface area that can be achieved as ruggedness of P(3HB) adsorbed on particular surface. If we can control the expression of the P(3HB) synthetic gene spatially by using genetic parts which are registered in Biobrick parts, the application of a super water-repellent sheet will become available.
</P><P>
We made P(3HB) sheets. Contact angle is an indicator to represent the strength of the water-repellent. The angle shows the physical properties, especially surface tension.When contact angle of sheets is larger than 90°, from Young equation, the sheets would have  stronger water-repellent by increasing real surface area. From literature data, contact angle of P(3HB) sheets is about 100°.[<A href="https://2012.igem.org/Team:Tokyo_Tech/Experiment/PHB2#Protocol" class="external text" rel="nofollow">Protocol</A>]
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption">Fig2-2-5-2, P(3HB) sheet</DIV></DIV></DIV></DIV><DIV id="tokyotech" style="font:bold ;left ; font-size: 30px; color: #0000FF; padding: 2px;">
6.</DIV><H1><SPAN class="mw-headline" id="Reference">Reference</SPAN></H1><P>[1] Jumiarti Agus, Altered expression of polyhydroxyalkanoate synthase gene and its effect on poly[(R)-3-hydroxybutyrate] synthesis in recombinant Escherichia coli, Polymer Degradation and Stability(2006) 91:1645-1650
</P><P>[2] Joanne Stubbe and Jiamin Tian, Polyhydroxyalkanoate (PHA) homeostasis: the role of the PHA synthase, 2003, Nat. Prod. Rep.,20, 445–457.
</P><P>[3] Stanley D. Fowler and Phillip Greenspan, Application of Nile red, a fluorescent hydrophobic probe, for the detection of neutral lipid deposits in tissue sections, Histochemistry &amp; Cytochemistry(1985), vol 33.No 8, 833-836
</P><P>[4] Pinzon NM, Nile red detection of bacterial hydrocarbons and ketones in a high-throughput format, mBio (2011),vol 2. issue 4.e-00109-11
</P><P>[5] Patricia Spiekermann, A sensitive, viable-colony staining method using Nile red for direct screening of bacteria that accumulate polyhydroxyalkanoic acids and other lipid storage compounds, Arch Microbiol (1999), 171:73–80
</P><P>[6] Vladimir K. Vanag, Cross-diffusion and pattern formation in reaction–diffusion systems, Physical Chemistry Chemical Physics(2009), vol 11.897-912
</P>
[7] Pohlmann A, et al, Genome sequence of the bioplastic-producing &quot;Knallgas&quot; bacterium Ralstonia eutropha H16, Nat Biotechnol(2006),  24:1257-62</DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:Tokyo_Tech/Projects/PHAs/index.htm">http://2012.igem.org/Team:Tokyo_Tech/Projects/PHAs/index.htm</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Tokyo_Tech/Projects/PHAs/index.htm" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Tokyo_Tech/Projects/PHAs/index.htm" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Tokyo_Tech/Projects/PHAs/index.htm&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Tokyo_Tech/Projects/PHAs/index.htm&amp;oldid=298429" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>