<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_MIT_MaterialsAndMethods"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:MIT/MaterialsAndMethods">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:MIT/MaterialsAndMethods&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:MIT/MaterialsAndMethods&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:MIT/MaterialsAndMethods&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:MIT/MaterialsAndMethods" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:MIT/MaterialsAndMethods</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><DIV id="project_title" style="text-align: center; margin-top: -0.8em; float:left; z-index:1; position:relative;
margin-left:3em;"><A href="https://2013.igem.org/Team:MIT"><H1 style="border: none; line-height: 1.2em; margin: 0; padding: 0; padding-top: 0.7em;">

Exosome Mediated Mammalian Cell-Cell Communication
</H1></A></DIV></DIV><UL class="nav"><LI id="nav_motivation"><A href="https://2013.igem.org/Team:MIT/Motivation">BACKGROUND</A></LI><LI id="nav_results"><A href="https://2013.igem.org/Team:MIT/Project">RESULTS</A></LI><LI><A href="https://2013.igem.org/Team:MIT/Project">Project Overview</A></LI><LI><A href="https://2013.igem.org/Team:MIT/miRNA">miRNA Signal</A></LI><LI><A href="https://2013.igem.org/Team:MIT/Protein">Protein Signals</A></LI><LI><A href="https://2013.igem.org/Team:MIT/Venus">Novel DNA Sensor: Cas9 Split Venus Fusion</A></LI><LI><A href="https://2013.igem.org/Team:MIT/ResultsBiobricks">BioBricks</A></LI><LI><A href="https://2013.igem.org/Team:MIT/Attributions">Attributions</A></LI></UL><LI id="nav_methods"><A href="https://2013.igem.org/Team:MIT/MaterialsAndMethods">METHODS</A></LI><LI id="nav_notebook"><A href="https://2013.igem.org/Team:MIT/Notebook">NOTEBOOK</A></LI><LI><A onclick="nav_notebook('construction')">Construction &amp;<EM>In Vitro</EM></A></LI><LI><A onclick="nav_notebook('summertc')">Summer Tissue Culture</A></LI><LI><A onclick="nav_notebook('falltc')">Fall Tissue Culture</A></LI><LI id="nav_human_practices"><A href="https://2013.igem.org/Team:MIT/HumanPractices">HUMAN PRACTICES</A></LI><LI id="nav_safety"><A href="https://2013.igem.org/Team:MIT/Safety">SAFETY</A></LI><LI id="nav_team"><A href="https://2013.igem.org/Team:MIT/Team">TEAM</A></LI><LI id="nav_partners"><A href="https://2013.igem.org/Team:MIT/Partners">SPONSORS</A></LI><LI id="nav_contact"><A href="https://2013.igem.org/Team:MIT/Contact">CONTACT US</A></LI><DIV id="col_nav"><DIV id="accordion"><H3><A href="#">Bacterial</A></H3><DIV class="col_list"><UL><LI id="b1"> Annealing Oligos</LI><LI id="b2"> Antarctic Phosphatase Treatment of Backbone</LI><LI id="b3"> Blue-White screening Xgal-IPTG plates</LI><LI id="b4"> BP Reaction</LI><LI id="b5"> Cell Stock </LI><LI id="b6"> Digestion Reaction</LI><LI id="b7"> Gel Extraction - QIAGEN</LI><LI id="b8"> Gel Extraction – Zymo kit</LI><LI id="b9"> Gel Preparation </LI><LI id="b10"> Gel Running </LI><LI id="b11"> Golden Gate Reaction </LI><LI id="b12"> Inoculation - Midiprep </LI><LI id="b13"> Inoculation - Miniprep </LI><LI id="b14"> LR Reaction </LI><LI id="b15"> Miniprep QIAGEN </LI><LI id="b16"> Midiprep QIAGEN </LI><LI id="b17"> Nanodrop </LI><LI id="b18"> PCR - Using PFX </LI><LI id="b19"> Pouring LB Agar Plates </LI><LI id="b20"> Restriction Mapping </LI><LI id="b21"> Sequencing </LI><LI id="b22"> Transformation </LI></UL></DIV><H3><A href="#">Mammalian</A></H3><DIV class="col_list"><UL><LI id="m1"> DNA Transfection of HEK293 cells</LI><LI id="m2"> Co-Transfection of DNA and RNA in HEK293 cells </LI><LI id="m13"> DNA Transfection of Jurkat T cells using the Neon® system</LI><LI id="m14"> siRNA Annealing</LI><LI id="m4"> Gelatin Pretreatment </LI><LI id="m5"> Passaging HEK293 Cells </LI><LI id="m6"> Passaging Jurkat T Cells </LI><LI id="m7"> DAPI Nuclear Staining</LI><LI id="m11"> Rh-PE Staining</LI><LI id="m8"> Exosome Isolation</LI><LI id="m12"> Poly-L-Lysine Coating</LI><LI id="m9"> Preparing Samples for Flow Cytometry </LI><LI id="m10"> Cryopreservation </LI></UL></DIV><H3><A href="#">Instrumentation</A></H3><DIV class="col_list"><UL><LI id="i1"> Fluorescent Microscopy </LI><LI id="i2"> Confocal Microscopy </LI><LI id="i3"> Flow Cytometry </LI></UL></DIV></DIV></DIV><DIV id="col_left"><DIV class="section" id="headerbio"><H1> Methods and Protocols Overview</H1>
In order to assess the functionality of exosome-mediated cell-cell communication <EM>in vivo</EM>, as well as to design and implement our various modules for our proposed genetic circuits, our team employed a variety of protocols and methods in our experimentation.
The experiments that we conducted to produce our results included the generation of nucleic-acid circuitry components in <EM>E. coli</EM> bacteria, the <EM>in vitro</EM> construction and testing of components, the <EM>in vivo</EM> testing of various circuit components in HEK293 and Jurkat mammalian cells, and the use of laboratory instrumentation for data generation and analysis.
</DIV><DIV class="section" id="b1bio"><H1> Annealing Oligos </H1><OL><LI> Create 100uM stock solutions of each oligo. </LI><LI> Create a 20uL solution of the following: 2 uL of 100uM stock solution of top oligo, 2 uL of 100uM stock solution of complementary bottom oligo, 2 uL T4 DNA ligase buffer, 1 uL T4 poly nuclease kinase (T4PNK), 13 uL water. </LI><LI> Put in thermocycler and run the following program: 37°C for 30 mins, 95°C for 5 min, Ramp to 25°C by 0.1% (0.1°C per second) and keep at 25°C for 5 min, End at 4°C.</LI><LI>Allow to cool down to room temperature, oligos should be annealed. </LI></OL></DIV><DIV class="section" id="b2bio"><H1>Antarctic Phosphatase Treatment of Backbone </H1><OL><LI> Use 5 units of Antarctic phosphatase per 1µg of DNA (over digesting by factor of X). </LI><LI> Calculate volumes: DNA µg = amount of DNA in backbone digest = X, Enzyme volume = X/5 uL (minimum of 1 uL), Buffer is dilution factor x dilution of the total volume.</LI><LI> Fill in this order: Backbone Digestion, Antarctic Phophatase Buffer, Antarctic Phosphatase.</LI><LI>Incubate for 15 minutes at the specified temperature for the enzyme (37C).</LI><LI>Heat Inactivate for 5 minutes at 65C.</LI><LI>Keep the buffer on ice and the antarctic phosphatase in the benchtop coolers when on the bench.</LI></OL></DIV><DIV class="section" id="b3bio"><H1> Blue-White screening Xgal-IPTG plates</H1><OL><LI> Add: 1ul of 1000X Kan per ml, 2ul of 500X Xgal per ml, 2ul of 500X IPTG per ml.</LI><LI>Pour 25 mL of prepared LB agar into each Petri dish.</LI></OL></DIV><DIV class="section" id="b4bio"><H1> BP Reaction </H1><OL><LI> Thaw on ice the BP Clonase II enzyme mix for about 2 minutes. Vortex the BP Clonase II enzyme mix briefly twice (2 seconds each time). </LI><LI> Mix 0.5 uL BP clonase with 1 uL appropriate pDONOR (typically ~150 ng/uL) and 1 uL attB-flanked PCR fragment (typically ~50 ng/uL). </LI><LI> Return BP Clonase II enzyme mix to -20°C or -80°C storage. </LI><LI> Incubate at room temperature for 1-1.5 hours.</LI><LI> Ad 0.5 uL Proteinase K. Vortex briefly. </LI><LI> Incubate samples at 37°C for 20 minutes. </LI><LI> Transform 1 uL into 10G competent cells. </LI><LI> Outgrow w/ SOC 1hr. </LI><LI> Plate 20 uL and 100 uL of each on LB plates containing 100 ug/mL kanamycin and grow at 30C overnight.</LI></OL></DIV><DIV class="section" id="b5bio"><H1> Cell Stock </H1><OL><LI> Add 500 uL of overnight LB liquid culture (sequence verified) to a new cryo-vial. </LI><LI> Add 500 uL of sterilized 50% glycerol in water to vial. </LI><LI> Label tube, store at -80C.  </LI></OL></DIV><DIV class="section" id="b6bio"><H1> Digestion Reaction </H1><OL><LI> Use X units of enzymes per 1ug of DNA (over digesting by factor of X) Usually we use over digest factor of 10 unless otherwise specified in the enzyme tech sheet. If over digestion results in star activity use 3X </LI><LI> Calculate the volume required for each: DNA µg = DNA volume * concentration, Enzyme volume = # units * Enzyme unit/µl = X [µl], Buffer is dilution factor x dilution of the total volume.</LI><LI> Fill in this order: DNA, Water, Buffer, Enzyme. </LI><LI> Incubate for 3 hours at the specified temperature for the enzyme and deactivate at the appropriate temperature for 20 mins.</LI></OL></DIV><DIV class="section" id="b7bio"><H1> Gel Extraction – QIAGEN </H1><OL><LI> Prepare agarose gel (1%), use thick-welled comb. </LI><LI> Run desired sample on gel. </LI><LI> Excise desired band from gel using a razor blade. Place in 2mL Eppendorf tube. </LI><LI> Weigh slice. </LI><LI> Add 3 volumes of Buffer QG to 1 volume of gel (100 mg ~ 100 uL) </LI><LI> Incubate at 50°C for 10 min or until gel is dissolved; vortex every 2-3 min. </LI><LI> Confirm that the color of the mixture is yellow (if not, add 10 uL of 3M sodium acetate, pH 5.0) </LI><LI> Add 1 gel volume of isopropanol. </LI><LI> Apply 750 uL of mixture to QIAquick column and centrifuge for 1 min (max speed, ~13000 rpm, RT). Discard flow-through and place column back in tube. Repeat until all of the mixture has been applied to the column. </LI><LI> Add 500 uL Buffer QG to column, centrifuge for 1 min, discard flow-through. </LI><LI> Add 750 uL Buffer PE to column, centrifuge for 1 min, discard flow-through. </LI><LI> Dry spin by centrifuging for 1 min. </LI><LI> Place column in a fresh 1.5 mL Eppendorf tube. </LI><LI> Add 50 uL Buffer EB to the center of the membrane to elute DNA, centrifuge for 1 min. </LI><LI> Measure the concentration of DNA using Nanodrop (see Nanodrop protocol). </LI></OL></DIV><DIV class="section" id="b8bio"><H1> Gel Extraction – using Zymo kit </H1><OL><LI> Place the extraction gel on the blue light table. </LI><LI> Cut out the appropriate bands. Place into 2mL microtube(s). Try to cut out as small a piece as possible while still getting all the DNA. </LI><LI> Weigh gel slice (tare with empty microtube). Add 3 volumes of ADB buffer per mg of gel (so a 100mg gel gets 300 uL of ADB buffer). </LI><LI> Incubate at 55C for 10 minutes. Make sure that the gel is completely dissolved.</LI><LI> Add dissolved gel solution to Zymo column in collection tube. Max volume is 800 uL at a time. </LI><LI> Spin 14000 rpm for 30 sec. </LI><LI> Discard liquid in collection tube.</LI><LI> Repeat step 5-7 if had more than 800 uL dissolve gel.</LI><LI> Add 200 uL DNA wash buffer. </LI><LI> Spin 14000 rpm 30 seconds. </LI><LI> Discard liquid in collection tube.</LI><LI> Add 200 uL DNA wash buffer.</LI><LI> Spin 14000 rpm 1 min.</LI><LI> Discard liquid in collection tube. </LI><LI> Spin 14000 rpm 1 min one more time (dry spin). </LI><LI> Discard collection tube (but not the column).</LI><LI> Place column in a clean labeled microtube.</LI><LI> Add 10 uL (min 6 uL for higher DNA concentration) of sterile DDH2O to top of column. Water should be pipetted directly onto center of filter.</LI><LI> Incubate at RT 1 min (or longer).</LI><LI> Spin 1 min at 14000 rpm. Discard the column.</LI><LI> Measure the concentration on the nanodrop. (You may recover the 1uL from the nanodrop if needed.)</LI></OL></DIV><DIV class="section" id="b9bio"><H1>Gel Preparation</H1><OL><LI> To make a 1% agarose gel: dissolve UltraPure agarose to a final concentration of 1% in 1X TAE buffer in a glass bottle. For a small gel, measure ~ .5 g agarose, add 1X TAE buffer until 50 mL total volume. For larger gel, scale up as necessary. </LI><LI> Heat the solution in the microwave with frequent stirring to dissolve the agarose homogeneously. ~ 1 minute/200 mL solution. </LI><LI> Add 10 uL SYBRSafe (1:10000) per 100 mL of the solution and mix well. </LI><LI> Pour the solution into a clean small gel tray with comb added for wells. </LI><LI> Wait ~ 15-20 mins for gel to solidify. </LI><LI> If stored, place in Ziploc bag and store at 4°C. </LI><DIV class="section" id="b10bio"><H1>Gel Running</H1><OL><LI> While casting gel, add two sets of lanes; use one set to load an analytical gel. </LI><LI> Add 2ul gel loading buffer (Orange G 6X; it helps DNA sink into the bottom of the well) to DNA. </LI><LI> Make sure there is enough 1xTAE in the plate holder. </LI><LI> Load 5.0ul of appropriate hyperladder to one of the lanes. </LI><LI> Load appropriate amount of DNA - As much as possible! Usually 15-18ul - (mixed with the buffer) in each well.</LI><LI> Set the timer and voltage to 100V and 25 min. </LI><DIV class="section" id="b11bio"><H1> Golden Gate Reaction </H1><OL><LI> Add the following to a PCR tube (0.6 mL) at room temperature and mix: 1 uL (~ 100 ng/uL) donor vector, 2-4 uL DNA (equimolar ratio), 2 uL T4 Ligase Buffer, 1 uL T4 Ligase, 1 uL BsaI, To 20 uL sH2O </LI><LI> Place tube in thermocycler, run standard golden gate protocol: 37C 5min, 50X: , 37 2.5min, 4C 0.5min, 16C 5.5min, 37C 10 min, 80C 20 min, 4C hold.</LI><LI> Transform 1 uL of each GG reaction into competent cells (see transformation protocol). </LI><LI> Plate 20 uL and 100 uL of each on LB plates containing 100 ug/mL kanamycin with IPTG and X-gal. 
</LI><LI> Incubate at 30°C for at least 16 hrs, select correct colonies through blue/white screening (white = correct, blue = incorrect).</LI></OL></DIV><DIV class="section" id="b12bio"><H1> Inoculation – Midiprep </H1><OL><LI> Add 50 mL of fresh, sterile LB broth to a clean 250 mL Erlenmeyer flask. </LI><LI> Add 50 uL appropriate antibiotic, mix well. </LI><LI> Pick a colony from plate using sterile inoculation loop or pipette tip, and gently swirl in the solution. </LI><LI> Cap flask, place securely in shaking incubator, grow at 37C overnight (~ 16 hours). </LI></OL></DIV><DIV class="section" id="b13bio"><H1> Inoculation – Miniprep </H1><OL><LI> Add 5 mL of fresh, sterile LB broth to a clean 15 mL tube. </LI><LI> Add 5 uL appropriate antibiotic, mix well. </LI><LI> Pick a colony from plate using sterile inoculation loop or pipette tip, and gently swirl in the solution. </LI><LI> Cap tube, place in shaking incubator, grow at 37°C overnight (~ 16 hours). </LI></OL></DIV><DIV class="section" id="b14bio"><H1> LR Reaction </H1><OL><LI> Add the following components to a 1.5 mL tube at room temperature and mix:1uL of 5 fmol of pENTR_L4_Promoter_R1, 1uL of 5 fmol pENTR_L1_Gene_L2, 1uL of 10 fmol pDEST_R4_R2. </LI><LI> Thaw on ice the LR Clonase II enzyme mix for about 2 minutes. Vortex the LR Clonase II enzyme mix briefly twice (2 seconds each time). </LI><LI> To each sample (Step 1, above), add 2 uL of LR Clonase II enzyme mix to the reaction and mix well by vortexing briefly twice. Microcentrifuge briefly. </LI><LI> Return LR Clonase II enzyme mix to -20°C or -80°C storage. </LI><LI> Incubate reactions at 25°C for 1 hour. </LI><LI> Transform 1 uL of each LR reaction into competent cells (see transformation protocol). Plate 20 uL and 100 uL of each on LB plates containing 100 ug/mL ampicillin or carbenicillin. </LI><LI> Plate 20 uL and 100 uL of each on LB plates containing 100 ug/mL ampicillin or carbenicillin.</LI></OL></DIV><DIV class="section" id="b15bio"><H1> Miniprep – QIAGEN </H1><OL><LI> Transfer 1-5mL of overnight culture of plasmid cells into 2ml microcentrifuge collection tubes (1 per try) provided in the kit. Pellet for 1 min. Decant all the liquid and add 1 ml of the culture into the corresponding tube. Make sure not to mix up the tries.</LI><LI> Resuspend pelleted cells in 250 uL Buffer P1 and transfer to microcentrifuge tube. </LI><LI> Add 250uL Buffer P2 and mix thoroughly by inverting tube 4-6 times. Do NOT vortex. Mixture turns blue. </LI><LI> Add 350uL of Buffer N3 and mix IMMEDIATELY by inverting tube 4-6 times. Do NOT vortex. Mixture is no longer blue. </LI><LI> Centrifuge 10min at 13,000 rpm in table-top centrifuge. </LI><LI> Apply the supernatant to a QIAprep spin column by decanting. Do NOT get any of the sticky precipitate.</LI><LI> Centrifuge for 30 - 60s at 13000rpm. Discard flowthrough.</LI><LI> Wash the QIAprep column by adding 0.5 mL Buffer PB. </LI><LI> Centrifuge for 30 - 60s at 13000rpm. Discard flowthrough. </LI><LI> Wash the QIAprep column by adding 0.75 mL Buffer PE.</LI><LI> Centrifuge for 30 - 60s at 13000rpm. Discard flowthrough. </LI><LI> Place the QIAprep column in a clean 1.5mL microcentrifuge tube. To elute DNA, add 50uL Buffer EB to center of each column. Be careful NOT to pierce column. </LI><LI> Let stand for 1 minute. </LI><LI> Centrifuge for 60s at 13000rpm. </LI><LI> Remove column and discard, tube now contains DNA. </LI><LI> Measure the concentration of DNA using Nanodrop (see Nanodrop protocol). </LI><DIV class="section" id="b15bio"><H1> Midiprep – QIAGEN </H1><OL><LI> Transfer the overnight culture of plasmid cells into a 50 mL tube.</LI><LI> Centrifuge for 15 min at 6000g and 4°C. Pour out supernatant.</LI><LI> Add 4 mL Buffer P1 and resuspend cells. </LI><LI> Add 4 mL Buffer P2 and mix by inverting the tube 4-6 times. The cell suspension should turn uniformly blue. Incubate 5 min at RT. </LI><LI> Add 4mL Buffer P3 and mix immediately by inverting tube 4-6 times. </LI><LI> Centrifuge for 10 min at 6000g. If supernatant still cloudy, centrifuge an additional 5 min.</LI><LI> Apply the supernatant to a QIAprep syringe column by decanting. Do NOT get any of the sticky precipitate.</LI><LI> Plunge the filter into an empty tube until the supernatant is all in the tube. </LI><LI> Add 4 mL of S3 and mix by inverting the tube. </LI><LI> Attach funnels to a clean 1.5 microcentrifuge tube. While applying a vacuum, pour the supernatant into the funnel.</LI><LI> While still applying the vacuum, add .7mL of ETS. </LI><LI> Centrifuge for 60s at 13000rpm.</LI><LI> Let stand for 1 minute. </LI><LI> Centrifuge for 60s at 13000rpm. </LI><LI> Remove column and discard. Add .7mL of EB to the tube which now contains DNA. </LI><LI> Measure the concentration of DNA using Nanodrop (see Nanodrop protocol). </LI><DIV class="section" id="b17bio"><H1> Nanodrop </H1><OL><LI> Open program, select Nucleic Acid, make sure sample type is set to DNA-50. </LI><LI> Make sure Nanodrop pedestal is wiped clean using a Kim wipe. </LI><LI> Blank the spec using 1 uL Buffer EB or sH2O. </LI><LI> Add 1 uL of sample to pedestal, measure concentration. </LI></OL></DIV><DIV class="section" id="b18bio"><H1> PCR – Using PFX </H1><OL><LI> Thaw PFX supermix, primers, template DNA on ice. </LI><LI> To a new PCR tube (0.6 mL tube), add: 22.5 uL PFX supermix, 1 uL 5 uM Forward primer, 1 uL 5 uM Reverse, primer ,0.5 uL (~ 100 ng/uL) Template DNA </LI><LI> Mix solution well. </LI><LI> Place tube in PCR thermocycler. Set thermocycler program: Initial Denaturation: 5 min @ 95°C ; Loop (35 cycles), Denaturation: 30s @ 95°C , Annealing: 30s @ *(see below), Elongation: ** (see below) @ 68°C; Final Elongation: 15 min @ 68°C. Store: 16°C. </LI><LI> *: Calculate Tm based on annealing region: (4(#G or C)+2(#A or T))C </LI><LI> **: Calculate Extension Time based on length of product: 30s/500bp </LI></OL></DIV><DIV class="section" id="b19bio"><H1> Pouring LB Agar Plates </H1><OL><LI> Weigh 17.5 g of LB Agar, add into a sterilized Pyrex container. </LI><LI> Fill with sterile water up to 500 mL mark. </LI><LI> Cap loosely and tape with autoclave tape. </LI><LI> Autoclave for 40 mins. </LI><LI> Let sit at room temperature so solution cools sufficiently – should not hurt to touch bottle. </LI><LI> Add 500 uL of appropriate antibiotic(s) (1000X), gently swirl to mix. </LI><LI> Transfer 20 mL of solution into a new plate using a pipette. Use flame to get rid of any bubbles. </LI><LI> Cap plate and let sit at room temperature until the agar cools and solidifies. </LI><LI> Label plate with date and antibiotic(s) added. </LI></OL></DIV><DIV class="section" id="b20bio"><H1> Restriction Mapping </H1><OL><LI> Use Geneious, Vector NTI, or related program to select an appropriate restriction enzyme for mapping. A good enzyme should cut both the wildtype backbone vector and the correct vector with insert to yield band patterns that can be distinguished reasonably on a gel. Follow appropriate protocol based on enzyme properties (see NEB website) </LI><LI> Add the following to a PCR tube (0.6 uL) and mix: 10 uL (~ 100 ng/uL) DNA, 2 uL (10X) appropriate NEB Buffer, 0.5 uL Enzyme, To 20 uL sH2O. </LI><LI> Incubate at 37°C for 1-2 hours or appropriate enzyme digestion conditions.</LI><LI> Run sample on agarose gel to visualize. </LI></OL></DIV><DIV class="section" id="b21bio"><H1> Sequencing </H1><OL><LI> Fill out Genewiz sequencing form. </LI><LI> To a fresh tube add the following and mix: 10 uL Template DNA 5 uL 5 uM appropriate Sequencing primer. </LI><LI> Cap tube, place in Ziploc bag and attach to bag to sequencing form. Submit to Genewiz. </LI><LI> When results are obtained, align sequence with predicted sequence in Geneious program. </LI></OL></DIV><DIV class="section" id="b22bio"><H1> Transformation </H1><OL><LI> Make sure that the incubator (37°C) and a heat block (42°C) are on. Make sure SOC medium is clear and uncontaminated. </LI><LI> Allow DNA to thaw on ice. </LI><LI> Thaw competent (10G) cells on ice for 5 mins. </LI><LI> Add 1 uL DNA to cells, mix by gently tapping tube. </LI><LI> Incubate cells on ice for 30 min. </LI><LI> Heat shock cells at 42°C;C in heat block for 30 sec. </LI><LI> Immediately place cells back on ice for 2 min. </LI><LI> Add 900 uL SOC medium to cells, place at RT. </LI><LI> Place tube in shaking incubator at 37°C;C for 1 hour. </LI><LI> Plate 100 uL cells onto a new plate with appropriate antibiotic. </LI><LI> Place plate upside down overnight and let incubate at 37°C;C (~ 16 hrs). Should see colonies if successful. </LI></OL></DIV><DIV class="section" id="m1bio"><H1> DNA Transfection of HEK293 - Lipofectamine 2000</H1><OL><LI>Label enough eppendorf tubes for the number of conditions/wells you are transfecting plus one for a master mix of transfection reagent and unsupplemented DMEM. If possible, try to pool conditions/wells, i.e, if doing a small molecule induction ladder.</LI><LI> Aliquot 50 uL of unsupplemented DMEM to every tube. If you are pooling n conditions/wells, aliquot 50*n. For your mastermix, aliquot 50* total number of conditions/wells of unsupplemented DMEM. </LI><LI> Dilute plasmid DNA in tubes containing 50 (or 50*n) unsupplemented DMEM. Be sure to mix well. For our transfections, we transfect at most 500 ng of plasmid DNA. Be sure that DNA is of high purity and concentrated enough that it can be pipetted in small volumes (from .5 uL- 1 uL). Anything larger will dilute the final concentration. </LI><LI> Dilute transfection reagent into unsupplemented DMEM to create the mastermix. We use 1.65 uL Lipofectamine per 500 ng of plasmid DNA per well. So, if transfecting n conditions/wells, dilute 1.65 * n transfection reagent. </LI><LI> Add 51.65 uL of master mix per tube and pipette gently to mix.  If pooling, mix 51.65 * # of conditions pooled. Let complexes form for 20 minutes in the hood. </LI><LI> While complexes are forming, cells must be seeded into a 24 well plate at 100,000 cells in 500 uL. <B> Note. </B> If transfecting in <B> glass </B> bottom well plates, you must pretreat with gelatin (see menu).</LI><LI> Trypsinize and count confluent cells. Centrifuge at 1180 rpm for 4 minutes and resuspend in fresh media. Dilute to the 100,000 cells/ 500 uL. Add 500 uL of cells to each well. </LI><LI> When complexes are done forming, add dropwise to each well. Swirl the entire plate to mix, and then place into an incubator. Small molecule induction can be done 24 hrs later, and gene expression can be assayed 48 hrs later. </LI> *Adapted from Invitrogen, <I> Lipofectaime® 2000 Reagent Product Manual.</I><DIV class="section" id="m2bio"><H1>Co-Transfection of DNA and RNA in HEK293 cells</H1> *Adapted from Invitrogen, <I> Transfecting Stealth™ or siRNA into Mammalian Cells using Lipofectamine™ RNAiMAX. </I></DIV><DIV class="section" id="m13bio"><H1>DNA Transfection of Jurkat T cells using the Neon® system</H1><I>Adapted from the Invotrogen <A href="http://tools.lifetechnologies.com/content/sfs/manuals/neon_device_man.pdf">Neon® system manual</A></I><B>Prepare Neon system and program protocol</B><OL><LI> Turn on Neon system.</LI><LI> Fill Neon Tubes with 5 mL Buffer E1 (1 tube per 10 reactions).</LI><LI> Make sure there enough Neon tip (1 tip is good for 2 reactions).</LI><LI> Set pulse voltage, pulse width, and number of pulses.</LI><LI> Save protocol.</LI></OL><B>Prepare transfection plate</B><OL><LI> Add 500 μL P/S free medium to wells of 24 well plate.</LI><LI> Incubate in 37C cell culture incubator</LI></OL><B>Prepare DNA mix</B><OL><LI> Thaw DNA</LI><LI> Dilute DNA in Resuspension buffer R to desired concentration (100ng/uL).</LI></OL><B>Prepare cells</B><OL><LI> Ensure 70 - 90% confluent on day of transfection</LI><LI> Warm up P/S free medium, PBS</LI><LI> Pipette up and down to break clump.</LI><LI> Take an aliquot to count. Transfer the rest to conical tubes. Centrifuge 1000 - 1200 rpm RT 3 mins.</LI><LI> Resuspend in PBS to 1x106 cells/mL. Aliquot the calculated amount to tubes (100 μL per well of 24 well plate).</LI><LI> Centrifuge 1000 - 1200 rpm RT 3 mins to wash.
</LI></OL><B>Electroporation</B><OL><LI> Take transfection plate from incubator.</LI><LI> Resuspend pellets in DNA mix to 107 cells/mL density (Use 10 μL per well). Gently pipette up and down to
    obtain single cell suspension. AVOID BUBBLES!</LI><LI> Insert Neon Tube containing Electrolytic Buffer E into Neon Pipette Station until click.</LI><LI> Select your Neon protocol.</LI><LI> Take Neon pipette. Press until second stop to take up tip.</LI><LI> Take Neon pipette with tip. Press to first stop until gold electrode emerge from polypropylene shealth. Take up</LI><LI> DNA/cell mix. AVOID BUBBLES!</LI><LI> Insert Neon pipette tip into NeonTube and dock Neon pipette onto the pipette station until click.</LI><LI> Click Start. Keep an eye on Neon Tube while electroporation is taking place. There’ll be two beeps. First one indicates the START of the electroporation. Second one indicates completion.</LI><LI> Move Neon tip to transfection plate containing warm P/S free medium. Gently pipette up and down to mix cell/DNA with medium then disperse completely. REMEMBER TO CHANGE TIP EVERY 2 WELLS. REMEMBER TO CHANGE BUFFER E EVERY 10 WELLS.</LI><LI>Change P/S free medium after 12-24 hr. </LI></OL></DIV><DIV class="section" id="m14bio"><H1>siRNA Annealing</H1><OL><LI>Resuspend each strand to 200 uM in Annealing/Duplexing buffer (we used PBS)</LI><LI>Mix the two strands (They will now be 100uM after annealing)</LI><LI>Place tubes with siRNA in a water bath in a heat block. Heat at 94C for ~1 min</LI><LI>Turn off the heat block, allow the bath to cool over several hours to RT.</LI><LI>Make a working dilution (10uM) and store at -20C</LI></OL></DIV><DIV class="section" id="m4bio"><H1>Gelatin Pretreatment</H1>
This protocol is for use in 24 well, glass bottom, plate format. It can be scaled up or down depending on what size dish you are using.
<OL><LI>Find the 0.1X bottle of gelatin. It is already at the proper concentration. It is usually kept above the TC room centrifuge. Otherwise, possibly check the Cold Room if the TC room was recently cleaned.</LI><LI> Pipette gelatin into each well to be used of the plate such that the bottom of the well is fully coated with gelatin. For a 24 well plate, add 0.5 to 1.0 mL gelatin per well.For other plates, scale appropriately --&gt;  glass 35 mm dishes would be at least 1.5 mL.</LI><LI> Swirl plate (figure eight or shake forward/side) gently to evenly spread the gelatin. (Make sure the bottom of each well is covered!) </LI><LI>Let gelatin set for 20 minutes. In the meantime, proceed normally with transfection protocol. </LI><LI> Just prior to seeding the cells, gently aspirate the gelatin from each well. (Make sure you remove all the liquid but don't scrape the bottom of the plate -- tilt the plate to make it easier)</LI><LI> Trypsinize the cells and put DEME to neutralize.</LI><LI> Centrifuge down the cells at 5000 rcf for one minute</LI><LI> Immediately proceed with seeding the cells and continue the rest of the transfection protocol.</LI><LI> Add 500ul PBS to re-suspend cells</LI></OL></DIV><DIV class="section" id="m5bio"><H1> Passaging HEK293 Cells</H1>
Note. This protocol is for culturing cells in 100mm^2 round dishes. For other culture volumes, the amounts of reagents required will be different. 

Before moving cells from the incubator into the hood,
<OL><LI> Wipe down hood with 70% EtOH. </LI><LI> Pre-warm reagents in a 37 °C water bath. For a full 500 mL bottle of media this can take up to 20 mins to fully warm. </LI><LI> Anything moving into the hood, especially from the water bath, must be washed off with 70% EtOH. </LI></OL>
Once your hood and reagents are ready,
<OL><LI> Move cells from incubator to hood. </LI><LI> Aspirate off media using a sterilized (autoclaved), glass pastuer pipette, or a disposable aspirating pipette. </LI><LI> Optional: Wash cells gently using 2.5 mL PBS, then aspirate. </LI><LI> Add 2.5 mL of trypsin to cells. Incubate at most 1 min at 37°C. Work quickly, as over trypsinization will damage the cells.</LI><LI> Add 9.5 mL of supplemented media to inactivate trypsin. Pipette to get cells into a single cell suspension. </LI><LI> Pipette cells into a conical tube and take note of the final volume (which should be 12mL). This will dictate how much is transferred to a new cell culture dish. If splitting 1:6, then you need 2 mL of cell solution. </LI><LI> After transferring cells to new culture dishes, add supplemented media to a final volume of 12-13 mL. </LI><LI> Label dishes with Name, Date, Cell type, splitting ratio, and passage number. </LI><LI> Place cells into incubator. </LI><LI> Destroy unused cells by aspirating into the vacuum trap, or by adding bleach. </LI><LI> Wash out of hood with 70% EtOH, close the sash, and turn on UV Light. </LI></OL></DIV><DIV class="section" id="m6bio"><H1> Passaging Jurkat T Cells</H1>
Note. This protocol is for culturing cells in 100mm^2 round dishes. For other culture volumes, the amounts of reagents required will be different. 

Before moving cells from the incubator into the hood,
<OL><LI> Wipe down hood with 70% EtOH. </LI><LI> Pre-warm reagents in a 37 °C water bath. For a full 500 mL bottle of media this can take up to 20 mins to fully warm. </LI><LI> Anything moving into the hood, especially from the water bath, must be washed off with 70% EtOH. </LI></OL>
Once your hood and reagents are ready,
<OL><LI> Move cells from incubator to hood. </LI><LI> Pipette cells in media from dish into a 15mL tube. </LI><LI> Centrifuge at 1500rpm for 3 min. </LI><LI> Aspirate media and resuspend pellet in 10 mL of fresh complete media.</LI><LI> Add the appropriate amount of cell suspension to a fresh dish. If splitting 1:5, then you need 2 mL of cell solution. </LI><LI> After transferring cells to new culture dishes, add supplemented media to a final volume of 10 mL. </LI><LI> Label dishes with Name, Date, Cell type, splitting ratio, and passage number. </LI><LI> Place cells into incubator. </LI><LI> Destroy unused cells by aspirating into the vacuum trap, or by adding bleach. </LI><LI> Wash out of hood with 70% EtOH, close the sash, and turn on UV Light. </LI></OL></DIV><DIV class="section" id="m7bio"><H1> DAPI nuclear staining</H1><OL><LI> Dilute DAPI with PBS</LI><LI> Add the DAPI-PBS solution into the well with final concentration of 1ul/ml</LI><LI> Incubate cells for 15 minutes at 37 degrees</LI><LI> Pellet cells at 500g for 5 min and aspirate the supernatant</LI><LI> Add 400ul PBS, resuspend cells</LI><LI> Pellet and resuspend in PBS two more times. </LI><LI> Pellet and resuspend cells in colorless media for microscopy, or in PBS for FACS.</LI></OL></DIV><DIV class="section" id="m8bio"><H1> Exosome Isolation</H1><EM>Adapted from <A href="http://www.lifetechnologies.com/order/catalog/product/4478359">Invitrogen</A></EM><H3>Prepare Media</H3><OL><LI> Harvest cell culture media.</LI><LI> Centrifuge at 2000g for 30min</LI><LI> Transfer supernatant (without cells) to new tube.</LI></OL><H3>Prepare Media</H3><OL><LI>Add Total Exosome Isolation (TEI) reagent at a ratio of 2:1 media:reagent</LI><UL><LI>For 10mL of cell-free culture media add 5mL TEI reagent</LI></UL><LI>Mix well by vortexing until solution is homogeneous</LI><LI>Incubate the mixture at 4C overnight</LI><LI>Centrifuge at 10,000g for 1 hour at 4C</LI><LI>Aspirate and discard the supernatant (Pelleted exosomes are usually not visible)</LI><LI>Resuspend in 100uL PBS</LI></OL></DIV><DIV class="section" id="m9bio"><H1> Preparing Samples for Flow Cytometry </H1>
Note, this protocol is for use with 24 well plates. Trypsin volumes will need to scale according to the plate format you use.
<OL><LI> Take well plate from the incubator. </LI><LI> Aspirate off the supernatant in each well. If cells have not adhered, pipette the supernatant into a conical tube. </LI><LI> Add 500 uL of trypsin to each well. </LI><LI> Incubate at 37°C for at most a minute. </LI><LI> Add 1 mL of supplemented media to stop the trypsinization. </LI><LI> Transfer cells to the appropriate conical tube. </LI><LI> Centrifuge for 1180 rpm for 4-5 minutes. </LI><LI> Apsirate supernatant carefully, not disturbing the cell pellet (it will be rather small). </LI><LI> Resuspend  in 1 mL of PBS, aiming for a single cell suspension. Any clumps of cells could clog the flow cytometer. </LI><LI> Transfer from the conical tube to the 5 mL polystyrene tube for FACS. Any other tube (such as polypropylene) will not work. </LI></OL></DIV><DIV class="section" id="m10bio"><H1> Cryopreservation and Cell Thawing </H1><H2><B> Cryopreservation</B></H2>
This protocol requires the use of Dimethyl Sulfoxide (DMSO), and isopropyl alcohol (IPA). Before using these reagents, you should read a materials safety data sheet (MSDS) for proper handling.
Also, for cryopreservation, you ideally want to preserve cells that are at a low passage number (P4 or less). 

<OL><LI> Label as many cryovials as you need (if aiming to freeze down 10*10<SUP>6</SUP> cells at 1*10<SUP>6</SUP> cells/mL, you need 10 cryovials) with the the passage number of the cells, concentration, initials, date, and cell type. Be sure to also label the caps with something to be able to pick out cryovials easily. </LI><LI> Prepare freezing medium by mixing unsupplemented media (DMEM) (50% v/v),  FBS (40% v/v) , and  DMSO (10% v.v)  DMSO is hygroscopic, so you must work quickly when transferring from stocks. Be sure to seal the bottle tightly. </LI><LI> Count cells using a hemocytometer (see menu) or an automated cell counter. Dilute cells to a concentration of 1.0*10<SUP>6</SUP>- 1.5*10<SUP>6</SUP> cells/mL. </LI><LI> Centrifuge cells at 1180 rpm for 4-5 minutes. Aspirate supernatant and resuspend in freezing media, to a single cell suspension. </LI><LI> While cells are in the centrifuge, be sure to check the IPA in the Mr. Frosty freezing chamber. It must be changed every 5th use. Be sure to dispose of IPA properly. </LI><LI> Aliquot 1 mL cells to cryovials. </LI><LI> Place croyvials in the Mr. Frosty chamber. Freeze at -80°C from 6-24 hrs. Then transfer to either liquid nitrogen tanks, or a -140°C freezer. </LI><LI> A few days later, thaw a vial to check cell viability. </LI><H2><B> Thawing Mammalian Cells </B></H2>
Mamallian cells must be thawed quickly for optimal viability. Since you are removing cells from the -140°C, you need to move quickly and consider extra protection such as a face shield (cryovials can explode). 
<OL><LI> Have 12 mL of media pre-aliquotted and warmed. In addition, warm the stock bottle of media. </LI><LI> Move media into hood. </LI><LI> Working quickly, retrieve cells from -140°C freezer. Slightly untwist cap, to reduce the chance of the cryovial exploding. </LI><LI> Carefully dip the bottom of the cryvial into the 37°C waterbath. DO NOT SUBMERGE or you will contaminate your cells. </LI><LI> When there is a small ice-chunk, bring cells to hood (cells will continue to thaw in transport). </LI><LI> Pipette cells into pre-warmed aliquot of media. Centrifige at 1180 rpm for 5 minutes. </LI><LI> Aspirate supernatant carefully, to not disturb the cell pellet. Resuspend in 12 mL fresh, warmed media. </LI><LI> Place into culture dish, and observe for confluency over the next few days, changing media after about two days. In our experience, 1-1.2*10<SUP>6</SUP> cells reaches confluency in 2-3 days.</LI></OL><DIV class="section" id="m11bio"><H1> Rh-PE Staining</H1>
Note. This protocol is for Rh-PE that has been suspended in ethanol.
<OL><LI> Chill the cells down to 4 Celsius</LI><LI> Add 5uM final concentration of Rh-PE into each well</LI><LI> Continue incubating the cells at 4C for an hour.</LI><LI> Take out the cells and wash them three times with PBS</LI><LI> Resuspend the cells in colorless media</LI></OL></DIV><DIV class="section" id="m12bio"><H1> Poly-L-Lysine Coating For Microscopy</H1>
This technique is useful for preparing suspension cells (such as Jurkat T cells) for microscopy because it reduces the amount the cells move as they float in the media.

Prepare stock solution by dissolving 100mg poly-L-lysine (Sigma) in 100 ml water. Filter sterilize the solution by passing through a 0.22mm filter. Store in 5 ml aliquots at -20C.
Working solution is 50 mg/ml(1 in 20 dilution of stock)
<OL><LI> Sterilize the surface in 95% ethanol &amp; dry before coating.</LI><LI> Place the coverslips in a single layer in a sterile petri dish containing working solution &amp; incubate for 1 hr at 37C.</LI><LI> Remove coverslips using sterile forceps &amp; allow the surface to air dry.</LI></OL>
Note: During drying allow the coverslips to stand on a slant. Otherwise they stick to the surface on which they are laid.
</DIV><DIV class="section" id="i1bio"><H1>Fluorescent Microscopy</H1><P>
All fluorescent microscopy observations were carried out using a Zeiss Axiovert 200M epifluorescence microscope equipped with a 1344 x 1024 pixel cooled ORCA-ER CCD camera (Hamamatsu Corporation) and a Zeiss 10X objective. False coloring was performed by capturing images with the appropriate colors and assembling them into larger mosaics using custom software (GFP filters: 470/40 excitation &amp; 525/50 emission, DsRedExpress: 565/30&amp; 620/60). Cells were live-cell imaged in 24-well Tissue Culture treated plastic plates (Corning Corporation) in DMEM media with appropriate supplements as detailed in cell culture protocols.
</P></DIV><DIV class="section" id="i2bio"><H1>Confocal Microscopy</H1><P>
All confocal microscopy observations were carried out using a Leica TCS SP5 within a 37C humidity controlled chamber supplemented with 5% CO<SUB>2</SUB> and a Leica 10x objective with 3.0 zoom enabled. PMT wavelength ranges were set using default spectra for GFP, YFP, Alexa, ROX, mKate, BFP spectrums in Leica AFS Software with laser excitation as follows:
GFP - 488nm, 30%; YFP - 488nm, 30%, Alexa - 488nm, 30%; ROX - 630nm, 40%, mKate - 565nm, 30%, BFP - 405nm, 12%.Cells were live-cell imaged in 24-well glass bottom plates (MatTEK Corporation) treated with 1% gelatin in DMEM media with appropriate supplements as detailed in cell culture protocols.
</P></DIV><DIV class="section" id="i3bio"><H1>Flow Cytometry</H1><P>
All flow cytometry observations were carried out using a LSRFortessa  flow analyzers (BD Biosciences) with the following sets of setting. Alexa and GFP/YFP were measured using a 488 nm Laser, a 505 nm Longpass filter and a 530/30 emission filter in the linear range of PMT values adjusted such that negative cells fell at 10<SUP>2</SUP>. BFP was measured with a 405 nm Laser, a 460 nm Longpass filter and a 480/40 emission filter. mKate and ROX were measured using a 561 nm laser and a 582/15 emission filter. For each sample,  cells were trypsinized according to details in cell culture protocols and approximately 1 x 10<SUP>4</SUP> to ~  1 x 10<SUP>5</SUP> cell events  were  collected. In parallel, Rainbow Calibration Particles (BD Biosciences) were measured in order to equalize the data between different instruments and settings.
</P></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:MIT/MaterialsAndMethods">http://2013.igem.org/Team:MIT/MaterialsAndMethods</A>&quot;</DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:MIT/MaterialsAndMethods" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:MIT/MaterialsAndMethods" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:MIT/MaterialsAndMethods&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:MIT/MaterialsAndMethods&amp;oldid=360295" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></OL></DIV></OL></DIV></OL></DIV></OL></DIV></OL></DIV></OL></DIV></DIV></DIV></DIV></BODY></HTML>