"
Page
Discussion
View source
History
teams
Log in
Â 
Team:MIT/Attributions
From 2013.igem.org
Exosome Mediated Mammalian Cell-Cell Communication
BACKGROUND
RESULTS
Project Overview
miRNA Signal
Protein Signals
Novel DNA Sensor: Cas9 Split Venus Fusion
BioBricks
Attributions
METHODS
NOTEBOOK
Construction & In Vitro
Summer Tissue Culture
Fall Tissue Culture
HUMAN PRACTICES
SAFETY
TEAM
SPONSORS
CONTACT US
Overview
Project Overview
miRNA Signal
Overview
siRNA Characterization
Exosome Isolation and Co-Culturing
Cell-Cell Co-Culturing
Protein Signals
Overview
GFP
rtTA3
Cre
L7Ae
Cas9-VP16
Novel DNA Sensor: Cas9 Split Venus Fusion
Overview
Leucine Zipper Fusion
DNA Sensing
Our BioBricks
Favorites
All BioBricks
Attributions
Attributions
Attributions
All primer design, PCR, cloning reactions, gels, minipreps, midipreps, nanodropping, transfections, splitting, competent cell making, etc were performed by members of MIT iGEM 2013. All Zeiss and Evos microscopy was done by MIT iGEM 2013 students. All sample preparation for BCA assays, sequencing, confocal, and cytometry were done by MIT iGEM 2013. Sequencing reactions were dispatched to Genewiz and alignments were done by students. Primers were ordered from IDT and resuspended and used by students. Confocal and Cytometry samples were prepped by students, run by instructors, and analyzed by students. All DNA designs were done by students, presented to instructors for feedback (and subsequent revisions usually), and iterated until we were confident it would work.  To identify exactly where instructors assisted:
Deepak Mishra and Brian Teague, two of our advisors, taught us how to clone, general experimental techniques, and conducted training sessions on fluorescence microscopy on the Zeiss scope. After 3 hours of supervised training with one of them, several of us students were given access to the scheduling interface and could book the scope during normal hours to operate it ourselves. Similarly, we were given access to a simple tissue culture scope, the Evos system, for unsupervised use after training.
For cytometry, due to new rules enacted at MIT, undergraduates were not allowed to physically run samples on the instrument. Thus, team members would plan experiments, execute, and prepare samples suspended in PBS-1X solution before putting them on ice and giving them to an instructor to run for them. The staff member (NOT iGEM member) would sit with the student and run the samples and then provide a copy of *.fcs files to the team that we then wrote custom MATLAB code in conjunction with FlowJo to analyze.
For confocal microscopy, the lab does not allow undergraduates or any grad/post-doc operator that has not completed the 10 hour Leica Training offered once a year to operate the instrument. Thus, team members would plan experiments, execute, and prepare the wells in media before bringing them to the microscope for an instructor to load onto the scope. Each team member that prepared samples went through mandatory lectures on confocal microscopy (3 hours total) but were not allowed to operate the instrument. The instructors provided *.lif and *.tif files to us which we then used ImageJ to open, analyze, and export for publication on the wiki.
Nathan Kipniss, a member of the 2012 MIT iGEM team, assisted with the initial tissue culture training. All experiments on the wiki were executed by team members and no instructor ever split our cells, supplemented our media, or gave us any assistance outside of ordering reagants for us due to our inability to navigate MIT PO numbers.
Samira Kiani, whose work inspired the Cas9-CRISPR parts of our project, also advised our team and was a valuable resource as a trained medical doctor.
We would also like to thank the 2012 MIT iGEM team for allowing us to adapt their wiki design for our use with permission. Without all of these contributions, our work this summer would not have been possible.
Retrieved from " http://2013.igem.org/Team:MIT/Attributions "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
