<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_AQA_Unesp_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:AQA Unesp/Results</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P>﻿
</P><H1 class="logo">iGEM AQA_Unesp</H1><NAV class="menu"><UL><LI><A href="#">PROJECT ▾</A></LI><LI><A href="https://2017.igem.org/Team:AQA_Unesp/Description">Description</A></LI><LI><A href="https://2017.igem.org/Team:AQA_Unesp/Applied_Design">Design</A></LI><LI><A href="https://2017.igem.org/Team:AQA_Unesp/Model">Model</A></LI><LI><A href="https://2017.igem.org/Team:AQA_Unesp/Results">Results</A></LI><LI><A href="https://2017.igem.org/Team:AQA_Unesp/Demonstrate">Demonstrate</A></LI><LI><A href="https://2017.igem.org/Team:AQA_Unesp/Parts">Parts</A></LI></UL><LI><A href="#">HUMAN PRACTICES ▾</A></LI><LI><A href="https://2017.igem.org/Team:AQA_Unesp/HP/Silver">Silver HP</A></LI><LI><A href="https://2017.igem.org/Team:AQA_Unesp/HP/Gold_Integrated">Integrated HP</A></LI><LI><A href="https://2017.igem.org/Team:AQA_Unesp/Engagement">Public Engagement</A></LI><LI><A href="https://2017.igem.org/Team:AQA_Unesp/Safety">Safety</A></LI><LI><A href="#">TEAM ▾</A></LI><LI><A href="https://2017.igem.org/Team:AQA_Unesp/Team">Team</A></LI><LI><A href="https://2017.igem.org/Team:AQA_Unesp/Attributions">Attributions</A></LI><LI><A href="https://2017.igem.org/Team:AQA_Unesp/Collaborations">Collaborations</A></LI><LI><A href="https://2017.igem.org/Team:AQA_Unesp/InterLab">INTERLAB</A></LI><LI><A href="https://2017.igem.org/Team:AQA_Unesp/Notebook">NOTEBOOK</A></LI><LI><A href="https://2017.igem.org/Team:AQA_Unesp/Entrepreneurship">ENTREPRENEURSHIP</A></LI></NAV><P style="font-size:5em; color: #4b5ea4; text-align: center; font-family: 'Quicksand', sans-serif; font-weight: lighter; padding:0; margin:0">results</P><DIV class="sublinks"><H2><A href="#editing">editing</A></H2><H2><A href="#characterization">characterization</A></H2></DIV><DIV class="conteudo"><DIV style="padding:3%">

We decided to classify our experiments and results in two categories: editing and characterization. The editing part is destined for altering the chassis’ genes, in order to establish our genetic circuits and the characterization is directed for determining the behavior of the projects important parts such as the promoters, relationship between gene reporter production and bacterial growth, sRNA and the chassis stability into the intestine environment.
</DIV></DIV><H2 class="title-conteudo" id="editing">editing</H2><DIV class="conteudo"><DIV style="padding:3%">

We have successfully cloned the 174 and 175 sequences in <I>E. coli</I> TOP 10 and <I>Lactococcus lactis cremoris</I> (figure 1).

<P class="figure-caption" style="text-align:center">Figure 1. Electrophoresis gel for the cloned plasmids: pSEUDO_GFP (Ctrl), pSEUDO_174 (174) and pSEUDO_175.</P><H2><I>Lactococcus lactis</I></H2>
We also had success at 177, 179 and 180 sequences in <I>E. coli TOP 10</I> and <I>Lactococcus lactis cremoris</I> as shown in figure 2 and 3. 

<P class="figure-caption" style="text-align:center">Figure 2. Electrophoresis gel for the cloned plasmids: pSEUDO_177 and pSEUDO_180.</P><P class="figure-caption" style="text-align:center">Figure 3. Gene sequence alignment executed in Serial Cloner, comparing obtained sequence by capillary electrophoresis and planned nucleotide sequence.</P>

In order to detect insulin production, we made an western blot (targeting His6-tag) by selecting the supernatant of a 50 mL penetratin_SCI-57_His6 (177) producer Lactococcus lactis culture. Despite of the “noisy” band, we concluded that this strain is capable of producing and secreting insulin and the use of dry milk might have interfered at the band detection [1]. In the future, more western blotting procedure could elucidate this hypothesis.

<P class="figure-caption" style="text-align:center">Figure 4. Western Blot membrane for the cloned plasmids: pSEUDO_GFP (Ctrl) and pSEUDO_177 (177).</P><H2><I>Bacillus subtilis</I></H2><I>Bacillus subtilis</I> K07 transformation was also achieved as shown in figure 5.

<P class="figure-caption" style="text-align:center">Figure 5. Western Blot membrane for the cloned plasmids: pSEUDO_GFP (Ctrl) and pSEUDO_177 (177).</P>

To detect our peptide production and secretion, we’ve made a SDS-Page of a 50 mL 182 producer <I>Bacillus subtilis</I> K07 culture supernatant, right after its clarification with a 5 kDa cut-off membrane filter. We can see, through the figure 6, that only pLike-rep transformed strain presented a few insulin production.

<P class="figure-caption" style="text-align:center">Figure 6. SDS-page gel for cloned plasmids: Bs K07 (control), pBs1C (Bs SCI-int) and pLike-rep (Bs SCI-rep).</P></DIV></DIV><H2 class="title-conteudo" id="characterization">characterization</H2><DIV class="conteudo"><DIV style="padding:3%"><H2>Growth and GFP production</H2>
We have characterized bacterial growth and gene reporter production influence of two promoters: srfA (for <I>Bacillus subtilis</I>) and usp45 (for <I>Lactococcus lactis</I>).  

<P class="figure-caption" style="text-align:center">Figure 7. Growth curve and fluorescence from <I>Bacillus subtilis</I> K07 cloned with Prsfa_GFP genes.</P><P class="figure-caption" style="text-align:center">Figure 8. Growth curve and fluorescence from <I>Lactococcus lactis cremoris</I> cloned with Pusp45_GFP genes.</P>

Through the presented graphics on figures 7 and 8, we could assume that both srfa and usp45 promoters do not interfere in the bacterial growth and GFP production follows the same cellular growth behavior, presenting the most product accumulation in the stationary phase. 

<H2>sRNA</H2>
After cloning the 175 (Pgal_sRNA_termrnnb (Ec)) sRNA sequence, we tested its behavior and efficiency when associated at different carbon sources, such as glucose, fructose, mannitol (a sugar alcohol) and galactose.

<P class="figure-caption" style="text-align:center">Figure 9. Relative fluorescence (Fluorescence/OD) of sRNA (175) producers and non-producers (Ctrl) in different carbon sources. </P>

After analysing the graphic in figure 9, we can admit that glucose and fructose presence “turned ON” the gene reporter expression, showing similar relative fluorescence values to the control. Meanwhile, when exposed to other carbon sources, such as mannitol and galactose, the gene reporter expression was “turned OFF”, presenting then lower relative fluorescence values, when compared with the control. This result shows that the sRNA design was successful, but further characterization should be executed for better understanding.


References
[1] MAHMOOD, Tahrin; YANG, Ping-Chang. Western blot: technique, theory, and troubleshooting. North American journal of medical sciences, v. 4, n. 9, p. 429, 2012.

</DIV></DIV><TITLE>Team: AQA_Unesp</TITLE><DIV id="footer"><DIV id="sponsors" class="btm_box"><H2 class="sponsors">SPONSORS</H2></DIV><DIV id="contact" class="btm_box"><P><B>WHERE TO FIND US</B>
			São Paulo State University (UNESP)
			School of Pharmaceutical Sciences
			Araraquara, São Paulo, Brazil</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>