<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_AQA_Unesp_Model skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:AQA Unesp/Model</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P>﻿
</P><H1 class="logo">iGEM AQA_Unesp</H1><NAV class="menu"><UL><LI><A href="#">PROJECT ▾</A></LI><LI><A href="https://2017.igem.org/Team:AQA_Unesp/Description">Description</A></LI><LI><A href="https://2017.igem.org/Team:AQA_Unesp/Applied_Design">Design</A></LI><LI><A href="https://2017.igem.org/Team:AQA_Unesp/Model">Model</A></LI><LI><A href="https://2017.igem.org/Team:AQA_Unesp/Results">Results</A></LI><LI><A href="https://2017.igem.org/Team:AQA_Unesp/Demonstrate">Demonstrate</A></LI><LI><A href="https://2017.igem.org/Team:AQA_Unesp/Parts">Parts</A></LI></UL><LI><A href="#">HUMAN PRACTICES ▾</A></LI><LI><A href="https://2017.igem.org/Team:AQA_Unesp/HP/Silver">Silver HP</A></LI><LI><A href="https://2017.igem.org/Team:AQA_Unesp/HP/Gold_Integrated">Integrated HP</A></LI><LI><A href="https://2017.igem.org/Team:AQA_Unesp/Engagement">Public Engagement</A></LI><LI><A href="https://2017.igem.org/Team:AQA_Unesp/Safety">Safety</A></LI><LI><A href="#">TEAM ▾</A></LI><LI><A href="https://2017.igem.org/Team:AQA_Unesp/Team">Team</A></LI><LI><A href="https://2017.igem.org/Team:AQA_Unesp/Attributions">Attributions</A></LI><LI><A href="https://2017.igem.org/Team:AQA_Unesp/Collaborations">Collaborations</A></LI><LI><A href="https://2017.igem.org/Team:AQA_Unesp/InterLab">INTERLAB</A></LI><LI><A href="https://2017.igem.org/Team:AQA_Unesp/Notebook">NOTEBOOK</A></LI><LI><A href="https://2017.igem.org/Team:AQA_Unesp/Entrepreneurship">ENTREPRENEURSHIP</A></LI></NAV><P style="font-size:5em; color: #4b5ea4; text-align: center; font-family: 'Quicksand', sans-serif; font-weight: lighter; padding:0; margin:0">model</P><DIV class="sublinks"><H2><A href="#srna">sRNA hybridization</A></H2><H2><A href="#insulin">insulin</A></H2></DIV><H2 class="title-conteudo" id="srna">sRNA hybridization</H2><DIV class="conteudo"><DIV style="padding:3%">

In order to predict the behavior of our sRNA-based control system, we used the softwares RiboMaker and NUPACK. First, from RiboMaker we selected the two best sequences and free-energy data about the molecule and its complex with the target-mRNA. These data are shown on figures 1 and 2. The sequences generated by RiboMaker includes the complementary sequence and a generic terminator.
<P class="figure-caption" style="text-align:center">Figure 2. Sequence and data for (a) sRNA1 and (b) sRNA2. Obtained with RiboMaker.</P>
Low values for the free-energy parameters means that the sRNA will have a better hybridization with its target. From this first analysis, we could saw that the sRNA2 shows better values for the free-energy parameters than the sRNA1, leading us to assume that the sRNA 2 would be a better choice. To confirm this hypothesis, we used the software NUPACK to predict the hybridization between the sRNA and mRNA molecules, this software gave us a prediction of the secondary structure of the complex sRNA-mRNA and the probability of hybridization between the base pairs. These predictions are shown on figure 3.
<P class="figure-caption" style="text-align:center">Figure 3. Hybridization probability and the secondary structure formed by (a) sRNA1-mRNA target and (b) sRNA2-mRNA target.</P>
By this analysis, we were able to choose sRNA2 as the best to work with. This sRNA was then used in our <A href="https://2017.igem.org/Team:AQA_Unesp/Design#control">final construction.</A> Figure 4 shows the NUPACK analysis for the sRNA with the B. subtilis rrnb terminator and the target mRNA <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2270006">(BBa_K2270006).</A> This model allowed us to rationally design a control system and predict its behavior.
<P class="figure-caption" style="text-align:center">Figure 4. Hybridization probability and the secondary structure formed by the final sRNA-mRNA target.</P>
You can access the RiboMaker tool <A href="https://absynth.issb.genopole.fr/Bioinformatics/tools/Ribomaker">here</A> and NUPACK <A href="http://www.nupack.org/">here.</A><H3 style="color:#4b5ea4; text-align:center; margin-top: 5px"><A href="#top">▲ top</A></H3></DIV></DIV><H2 class="title-conteudo" id="insulin">insulin structure and interactions</H2><DIV class="conteudo"><DIV style="padding:3%">
The protein modeling in our project had the objective to visualize and simulate interactions of the insulin produced by our experiment (SCI-57) with other molecules. We also thought it would be very interesting to compare the structure of the SCI-57 insulin produced with the traditional one (E.coli expressed) usually applied for diabetes treatment. 
In this way, to begin our modeling trajectory we decided to visualize the 3D protein structure of the traditional insulin, Figure 1, the model was obtained using the data from RCSB [1] and Chimera software. 

<P class="figure-caption" style="text-align:center">Figure 1. Tridimensional structure of human insulin - E.coli expressed, chain A in blue and chain B in pink.</P>

The human insulin has two chains A and B, formed by alpha-helix and β-sheets, A contains 21 residues and B a total of 30. According to researches the interaction between insulin and its receptor happens by the terminal fraction of the alpha chain and by residues of the B chain. However, the complete mechanism of interaction is still a target of study [2]. In a demonstrative way, the next figure shows the human insulin and the site-1 of interaction forming the binding complex, Figure 2. 

<P class="figure-caption" style="text-align:center">Figure 2. Human insulin (E.coli expressed) and site-1 receptor forming the complex interaction. Insulin is circled in red, receptor domain L1-CR (green), alpha-CT peptide receptor (yellow).</P>

The complex is formed mainly by the receptor domains of insulin L1-CR and by the receptor peptide alpha-CT. These transmembrane receptors also have molecular functions of ATP binding and biological functions in the phosphorylation of proteins and in the signing route of Protein Tyrosine Kinase. Even though there is a lot of information about the insulin receptor, the process of understanding how insulin change its conformation to bind with the receptor it is still an enigma.
<H3>Prediction of tridimensional structure using the insulin sequences used in our project</H3>
In this second part of our analysis, we tried to predict the structure of insulin produced by the modified strains of Lactococcus lactis used by our team. For this prediction, we used the nucleotide sequences prepared for the genetic modification of these microorganisms. The nucleotide sequences were used to obtain the produced amino sequence and posteriorly a simulated structure model using Swiss-Model.
Swiss-Model (available in http://swissmodel.expasy.org) works as a server for automated comparative modeling of three-dimensional (3D) protein structures. In the mode we used, the  ‘first approach mode’  an amino acid sequence of a protein is submitted to build a structural model, being the template selection, alignment and model building done automated by the server [3].
Let’s start the modeling!
First, we have analyzed the first nucleotide sequence, the <B>USP45-signal::penetratin::SCI57::His6</B> used to transform the <I>Lactococcus lactis</I> for the production of insulin.
The model built by Swiss-Model had almost 100% of similarity with an insulin template found by the program. We can visualize in Figure 3 that this insulin structure is formed by just one chain. 

<P class="figure-caption" style="text-align:center">Figure 3. Simulated tridimensional structure of produced insulin using the sequence USP45-signal:penetratin:SCI57:His6, image using Chimera software.</P>

The second sequence analyzed was the USP45-signal:penetratin:SCI57 also used to transform <I>L. lactis.</I>
This time the model built had 94,74% of similarity with an insulin template found by Swiss-Model, formed by a single chain, Figure 4.

<P class="figure-caption" style="text-align:center">Figura 4. Simulated tridimensional structure of produced insulin using the sequence USP45-signal:penetratin:SCI57, image using Chimera software.</P>

In the next step, we have compared these two structures found by the different sequences for transformation of <I>L. lactis</I>, Figure 5.

<P class="figure-caption" style="text-align:center">Figura 5. Comparison between the structures of insulin obtained by the sequences USP45-signal:penetratin:SCI57:His6 (left), USP45-signal:penetratin:SCI57 (middle), match align of the two structures (right).</P>

In the same way, it is also interesting to compare the structures of the insulin formed by our sequences and the traditional one, expressed in E.coli, Figure 6. Let’s check the results!

<P class="figure-caption" style="text-align:center">Figure 6. Structural comparison (1.c) between insulin produced by the sequence USP45-signal:penetratin:SCI57:His6 (1.a) and traditional insulin (1.b) and structure comparison (2.c) between insulin produced by the sequence USP45-signal:penetratin:SCI57 (2.a) and traditional insulin (2.b).</P>

The RMSD tool showed a higher similarity between the structures of traditional insulin and the one produced using the sequence USP45-signal:penetratin:SCI57, being the value of RMSD equal to 1.561 angstroms against 2.327 angstroms for the USP45-signal:penetratin:SCI57:His6.<H3>Simulating interactions between insulin and other drugs</H3>
Having in mind that the insulin produced in the Insubiota project would be a different type of insulin, we believed it was important to check the interactions between this insulin and other drugs which could also be consumed by the diabetics. For example, we have simulated the interaction between insulin and Metformin, Figure 7, marked under the trade name of Glucophage, a medication taken by mouth and used for the treatment of diabetes particularly for overweighed people.

<P class="figure-caption" style="text-align:center">Figure 7. Chemical structure of Metformin (1), interaction between Metformin and: Insulin produced by the sequence USP45-signal:penetratin:SCI57:His6 (a), Insulin produced by the sequence USP45-signal:penetratin:SCI57 (c) and traditional insulin expressed in E.coli (b)</P>

The tests were made using the Autodock Vina tool from the Chimera Software, which is normally used for the calculation of interactions between protein and its ligands. This tool is able to give us the energy score of the molecular interactions in kcal/mol. Demonstrating our results, the docking of highest score between insulin USP45-signal:penetratin:SCI57:His6 and Metformin was 3,4 kcal/mol, two H-bonds were found in this interaction. A single H-bond also exists in the interaction of traditional insulin and Glucophage, however a stronger bond due to its smaller length, in this case the score was 3,3kcal/mol. Insulin produced by USP45-signal:penetratin:SCI57 did not show any H-bond interactions with the drug molecule. 
The results are interesting because we could predict that probably the interaction between insulin and the metformin are weak and would not interfere in the function of each of them in the metabolism.
Proceeding to the next drug, it was probable that patients with diabetes someday in their lives would need to make use of medicaments for stomach problems for example. In this way, why not trying to see how the insulin molecules interact with Omeprazole, a drug is extensively used for the treatment of gastroesophageal diseases, Figure 8.

<P class="figure-caption" style="text-align:center">Figure 8. Chemical structure of Omeprazole (1), interaction between Omeprazole and: Insulin produced by the sequence USP45-signal:penetratin:SCI57:His6 (c), Insulin produced by the sequence USP45-signal:penetratin:SCI57 (b) and traditional insulin expressed in E.coli (a))</P>

We observed that the interaction between the traditional insulin and Omeprazole forming one H-bond with 1.960 angstroms of length and a docking score of 3.3 kcal/mol. However the stronger docking happened in the test with the insulin USP45-signal:penetratin:SCI57 and the drug, the score was 5.8 kcal/mol . Even though there are some interactions between this pair of insulin structures and the omeprazole, they seem residual and not significant in a way that would modified or change the viability of the insulin produced by Insuobiota. The constructed insulin from USP45-signal:penetratin:SCI57:His6 did not show relevant interaction with the medicament. 

References
[1] TIMOFEEV, V. I. et al. X-ray investigation of gene-engineered human insulin crystallized from a solution containing polysialic acid. Acta Crystallographica Section F: Structural Biology and Crystallization Communications, v. 66, n. 3, p. 259-263, 2010.
[2] MENTING, John G. et al. How insulin engages its primary binding site on the insulin receptor. Nature, v. 493, n. 7431, p. 241-245, 2013.
[3] SCHWEDE, Torsten et al. SWISS-MODEL: an automated protein homology-modeling server. Nucleic acids research, v. 31, n. 13, p. 3381-3385, 2003.,4.

<H3 style="color:#4b5ea4; text-align:center; margin-top: 5px"><A href="#top">▲ top</A></H3></DIV></DIV><TITLE>Team: AQA_Unesp</TITLE><DIV id="footer"><DIV id="sponsors" class="btm_box"><H2 class="sponsors">SPONSORS</H2></DIV><DIV id="contact" class="btm_box"><P><B>WHERE TO FIND US</B>
			São Paulo State University (UNESP)
			School of Pharmaceutical Sciences
			Araraquara, São Paulo, Brazil</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>