Team:Yale/notebook
<!DOCTYPE html>
Yale iGem 2015: Notebook
NonModel
Project
Overview
Methods
Results
Modeling
Parts
Notebook
Weekly
PDF Summary
Collaborations
Human Practices
SSRI
LGBTQ Survey
Team
People
Attributions
Standard Pages
Standard Pages
Gold Standards
Silver Standards
Bronze Standards
See Our Lab Notebook
Lab Notebook: Weekly Summaries
Week -1
Dan has a Ball
Week 1
Getting Started
Week 2
Practice
Week 3
Plasmid Problems
Week 4
Primed for Primers
Week 5
Growing and Amplifying
Week 6
Troubleshoots
Week 7
I see the Light
Week 8
FLPping Out
Week 9
Wrapping Up
Week 10
∆mutS Mutants!
Week 10+
Lionel has a Ball
Lessons from Week -1
Dan began his research for the summer two weeks before the other researchers arrived. This was the first week any iGEM researcher was in the lab, so it was focused on preparing the lab and assessing the viability of two of our possible projects. The project which was started this week focused on porting Multiplex Automated Genome Engineering (MAGE) from E. Coli, in which the technique is proven to work, to Cyanobacteria and Sinorhizobium bacteria.
In order to port MAGE from E. Coli to other organisms, we had to first find inducible promoters that worked in Cyanobacteria and Sinorhizobia, and then beta-homolog proteins (reference to MAGE section of our wiki) for both organisms. At this point in the project, we began with finding cyanobacterial inducible promoters. We found promoters in Cyanobacteria which should, in theory, be inducible, and one constitutive promoter. The promoters were a nitrate-inducible promoter nirA, an iron-inducible promoter isiAB, a green-light-inducible promoter pCpG2, and a constitutive promoter, psaA. We designed and ordered primers to isolate all of these promoters from their respective backbones, whether they were Synechococcus sp. 7002 or Synechocystis sp. 6803. We also focused our efforts on growing up copies of a backbone plasmid, BBa_K125000, which was to contain our primers.
Go to the Lab Notebook
Entry for week -1 1 2 3 4 5 6 7 8 9 10 10+
x
Lessons from Week 1
This week served as an orientation and planning period. The graduate mentors showed us around our lab space, and we practiced some lab techniques (such as proper aseptic technique and a transformation protocol in E. coli) with the Isaacs lab graduate students. The plan is to begin working with our rhizobia strains (Sinorhizobium meliloti 1021 and Rhizobium tropici CIAT) next week; we ordered some cyanobacterium Synechococcus sp. PCC 7002 from the Pasteur collection, but it is backordered and may take some weeks to arrive.
We laid out a series of experiments which would eventually allow us to express a nonspecific recombinase (beta-homolog) to incorporate foreign DNA into our organism's genome; this process is at the heart of the MAGE technique. Much of this week was spent identifying recombinases which could potentially function in our organisms through literature and BLAST searches. We also spent a significant amount of time identifying inducible promoters (either native to our organisms or within the BioBrick Registry) which we could use to express our recombinase.
The first step in our plan is to determine optimal growth conditions, transformation protocols, and selection screens for our strains. Once this is complete, we can test the effectiveness of our identified promoters by placing them upstream of a fluorescent reporter; we will use the yellow-fluorescent citrine protein since its emission wavelength does not overlap with the autofluorescence wavelengths of our cyanobacteria. After identifying the most effective promoter (lowest leakiness and highest expression level when induced), we can replace the citrine gene in our construct with our synthesized beta-homolog genes. Testing the effectiveness of our recombinases will be a bit of a challenge; we need an assay to be implemented in each organism that will allow us to quantify mutagenesis efficiency.
While all of this is going on, we will also need to knock out the mutS gene in each of our organisms. mutS is involved in identifying nucleotide mismatches during DNA replication. Since MAGE is founded upon such mismatches, it is in our best interest to allow them to go unnoticed by the cell's DNA proofreading systems. Silencing mutS should allow us to do this.
Go to the Lab Notebook
Entry for week -1 1 2 3 4 5 6 7 8 9 10 10+
x
Lessons from Week 2
For Rhizobium, Holly, Lionel, Jessica, and Erin tried to determine the optimal growth conditions.  Following the example of literature, this team inoculated LB broth as well as different antibiotic mixtures of Tryptic Soy Broth and LB with the two strains of Rhizobium we had acquired from the Handelsman lab (Rhizobium tropici CIAT 899 and Rhizobium etli CE3). Rhizobium etli CE3 showed no growth, and, therefore, was dropped as a target for research.  Rhizobium tropici CIAT 899 showed growth in all three mixtures of media; however, it showed the fastest growth in 50% Tryptic Soy Broth/ LB.
For Cyanobacteria, Dan inoculated both Synechocystis sp. 6803 and UTEX 2973 in BG-11.  Along with this, Danny read literature and determined the best proteins to use for increasing the efficiency of MAGE in cyanobacteria.  Colin searched for protocols that could be used for transformation, growth, and various other important processes.
For the lab overall, we attempted to electroporate various plasmids into E. Coli to improve our technique.  However, these efforts were met with limited success.
Go to the Lab Notebook
Entry for week -1 1 2 3 4 5 6 7 8 9 10 10+
x
Lessons from Week 3
This week, we focused on obtaining and using new plasmids, pKT230  and pZE21G, because our prior plasmid, BBa_K125000 was causing various problems, from being improperly annotated online to stubbornly refusing to be isolated using a maxi prep. However, very little actual lab work was done this week. In regards to non-lab work, we researched and described a way to knock out the mutS gene in both Cyanobacteria and Rhizobia and explored the possibility of also porting a CRISPR/Cas9 system into Cyanobacteria or Rhizobia.
For lab work: we successfully miniprepped pKT230 and pZE21G out of E. Coli once the OD was 0.611. The Jacobs-Wagner lab at Yale also graciously gave us S. Meliloti cultures to use: #1021, #WM249, and #MB501. S. meliloti WM249 and MB501 are derivatives of Rm1021 with transposon insertions permitting the efficient electroporation of E. coli plasmid DNA. WM249 contains a Tn5-233 element encoding gentamicin resistance, whereas MB501 (obtained from M. Barnett, Stanford University) contains the same Tn5-233 swapped for trimethoprim resistance. We were also able to isolate the isiAB promoter from Cyanobacteria Synechococcus PCC 6803 using the PCR procedure outlined in the procedural notes. However, the ends of the promoter were custom-made to be gibson-assembly-ed into the BBa_K125000 plasmid, so if we end up switching plasmids we’d need to re-isolate the promoter.
Let us explain CRISPR. CRISPR (clustered regularly interspaced short palindromic repeats) is a recently discovered mechanism from certain bacterial adaptive immunity systems  that uses guide RNA and the Cas9 proteins to create gene knockouts and gene replacements. Guide RNA locates a specific homologous sequence in the DNA following a PAM sequence. The Cas9 protein then causes a double stranded break in the DNA (see the graphic below put together by the Wuhan University iGEM team in China). Currently, we have found in literature that Synechococcus does not have a fully functional CRISPR-Cas system. This means that imported systems will work more effectively due to lack of cross talk. We also found that cyanobacteria are capable of non-homologous end joining (NHEJ). This means that cyanobacteria are prime targets for the porting of CRISPR- Cas systems. In order to test the ability of the imported CRISPR-Cas systems, we will target the UreC gene, a gene which confers sensitivity to solutions of urea and nickel sulfate. Already, we have designed the plasmids for the guide RNA and the Cas9 protein. Cas9 will be placed on a low copy number plasmid (Bba_125000) with an inducible promoter such as isiAB or nirA. gRNA will be placed on high copy number plasmid (KT230) with a constitutive promoter such as psaA. If necessary, the cas9 protein can be placed next to a tetR promoter, a promoter found in E. Coli but not natively in cyanobacteria. In addition, recently it has been found that CRISPR works in Arabidopsis and several strains of rhizobium. However, no CRISPR has been found in the strains of Rhizobia we intend to use. It has also been shown that Rhizobia exhibits NHEJ. Whether or not pursuing CRISPR would be useful is an idea that needs to be discussed.
In order to port MAGE into Cyanobacteria and Sinorhizobia, the mutS gene must be knocked out. Why? MutS functions in the DNA mismatch repair pathway, which is a highly conserved system from prokaryotes to higher eukaryotes. The mutS protein recognizes a DNA mismatch, but since DNA mismatch is absolutely needed for the MAGE mechanism, a mutS knockout (KO) is essential for high mutagenesis efficiency. We decided that the best way to do this would be with a Flp-FRT Recombination. Here is the strategy for mutS Knockout in Synechococcus sp. PCC 7002:
1) Amplify and assemble KO construct: 1 kb Homology Arm (UpF) → FRT → kanR → FRT → 1 kb Homology Arm (DownF)
2) Transform KO construct into PCC 7002
3) Recombination event occurs between KO construct and cell genome
4) Screen for recombinants by culturing on kanamycin/PCR verify mutants
5) Transform plasmid containing Flp gene into cell; induce Flp expression
6) Flp cuts out kanR cassette, leaving an FRT site scar (Note: Flp-containing plasmid also contains another selectable marker gene, such as spcR)
7) Remove Flp-containing plasmid by heat shock
Go to the Lab Notebook
Entry for week -1 1 2 3 4 5 6 7 8 9 10 10+
x
Lessons from Week 4
This week, we made progress in amplifying cyanobacteria and rhizobium promoters. We performed multiple antibiotic resistance assays and transformation experiments in both types of bacteria as part of our attempt to create a framework for propagating and selecting nonmodel organisms.
All of the primers for our cyanobacteria promoters have arrived. Dan worked on amplifying them so that they can be assembled with citrine into one of the vectors we have and tested for strength. Unfortunately, due to a power outage that halted the PCR run, no amplifications were successful.
One of the bigger issues we handled this week was the cyanobacteria growth media. We ordered freshwater BG-11 from Sigma-Aldrich, but we needed marine BG-11 to grow PCC 7002. We researched and found the reagents necessary to prepare MN marine medium from a seawater base in our lab.
Our rhizobium transformation experiments gave us mixed results. Using electroporation, R. tropici was able to take up the KT230 plasmid, but S. meliloti 356, 370, and 371 demonstrated kan and spec resistance and we were not able to determine whether the transformation was successful for those strains. We also experimented with conjugation as an alternative transformation method. We found a protocol for conjugating E. coli to UTEX 2973 and began to incubate E. coli containing pKT230.2 When UTEX 2973 reaches OD750 0.5, we will proceed with conjugation.
We performed several rhizobium antibiotic resistance assays to determine whether our strains of rhizobium have natural resistance to certain antibiotics. The results suggested that none of our strains have kanamycin resistance, but R. tropici and S. meliloti 371 have spectinomycin resistance. We also performed a rifampicin assay to determine whether we could eventually use MAGE to induce resistance to rifampicin in rhizobium. The assay worked with 2x1010 cells, but even 20 uL/mL rifampicin was below the limit of detection (see Fig. 1). Our goal moving forward is to redo this assay at lower concentrations.
We also met with Professor Dellaporta, one of our PIs, about the possibility of using ligation-independent cloning (LIC) as an alternative to Gibson Assembly. With LIC, the BsaI T4 DNA polymerase generates long complimentary sticky ends between the vector and insert and eliminates the need for ligase. The advantage is that one LIC cloning vector can be used repeatedly to build many promoter and beta homolog constructs without PCR amplifying the large vector backbone each time. However, we would need to order new primers and spend a week constructing the LIC vector.
Go to the Lab Notebook
Entry for week -1 1 2 3 4 5 6 7 8 9 10 10+
x
Lessons from Week 5
This week was centered on characterizing Rhizobia and cyanobacteria, PCR amplifying promoters and citrine, and transforming our plasmids of interest into cyanobacteria and E. coli.
For Rhizobia (Sinorhizobium meliloti 1021 strains 356, 370, 371 and Rhizobium tropici CIAT 899), we began antibiotic (rifampicin) and growth assays. The purpose of the rifampicin assays was to determine the antibiotic concentrations our Rhizobial strains were naturally resistant to; we would then be able to use an effective amount for strain selection (MAGE, transformations, etc). The growth assays were conducted to figure out the mid-log concentrations and doubling times of the Rhizobia strains. With this information, we could optimize our Rhizobia electroporation protocols—incubation periods and recovery times. Unfortunately, we encountered technical errors with our growth assays so these must be repeated next week.
For cyanobacteria, we worked on adjusting the pH of our growing media to standardize growth conditions (pH = 8.5). We were able to successfully grow Synechococcus PCC 7002 in ATCC 1047 media; it took around five days to reach a semi-confluent growing stage. We also continued to look into the efficiency of cyanobacterial growth under CO2 conditions, which according to literature, should be optimized in 3% CO2. According to our growth assays, cyanobacteria seems to be the most sensitive to OD 730, so this is the OD we will use for our future assays.
We were able to PCR amplify three rhizobium-specific inducible promoters: melA, bacA, and nodF. We also have the Anderson constitutive promoters from the iGEM registry, which we will use for both cyanobacteria and Rhizobia. For both organisms, we started amplifying our control genes of interest (GFP for Rhizobia, citrine for cyanobacteria) with overhangs to the promoters.
Much of our transformations this week were in preparation for Exonuclease and Ligation Independent Cloning (ELIC) or for amplification of broad host range plasmids into E. coli. We are considering using ELIC as a backup or more efficient alternative to Gibson assembly. For ELIC, we chose to work with the plasmid pZE21G as a control experiment; this plasmid along with the chromoprotein amilCP should be able to be successfully assembled. Additionally, we have been experimenting with natural transformation in cyanobacteria and are still waiting for the transformation cultures to grow up more before forming conclusions. Regarding electroporation in E. coli, we are still troubleshooting transformations with our two main plasmids pKT230 and k125000.
Go to the Lab Notebook
Entry for week -1 1 2 3 4 5 6 7 8 9 10 10+
x
Lessons from Week 6
This week we focused on troubleshooting our most pressing problems: growth for cyanobacteria and transformation for cyanobacteria and rhizobia. In addition, we continued to work on amplifying and creating constructs for FLP and CRISPR, methods we will try to use to knockout mutS. For the assembly of our promoters and genes into our plasmid backbone, we decided to test ELIC and LIC, cheaper and faster alternatives to Gibson Assembly. We abandoned the plasmid BBa_K125000 because it was too unreliable, having unclear sequencing and annotation data and yielding unsuccessful electroporation results. We continue to focus on pKT230 and pETcoco-2 as possible backbones for our beta homologs.
For cyanobacteria, the amount of time to form colonies on solid media is an obstacle to obtaining results for our experiments, thus we looked into getting a CO2 incubator to promote growth and researched liquid culture selection methods. For the transformation, we attempted natural transformation of pKT230 and pETcoco-2 into PCC 7002, and we hope to have results next week.
For rhizobia, we focused on troubleshooting our unclear electroporation results from last week. We redid our antibiotic resistance assays (kan, spec, strep, carb, rif) for all strains to confirm innate antibiotic resistances.
We set up a growth assay to obtain a growth curve with which we can calculate doubling times and mid-log growth for our Rhizobia strains. In our first growth assay, R. tropici CIAT 899 showed growth but the S. meliloti strains did not. We set up another antibiotic assay and we are awaiting results.
For transformation, we decided to try chemical transformation (CaCl2 method) and conjugation in addition to electroporation. To address the issues with contamination, we submitted 16S sequencing of master and working stocks of all our strains and of unexpected growth on selective plates and verified that the stocks we were using were in fact the strains we believed them to be.
Go to the Lab Notebook
Entry for week -1 1 2 3 4 5 6 7 8 9 10 10+
x
Lessons from Week 7
This week consisted of more troubleshooting for everyone on the team. Experiments for determining growth, transformation, and selection parameters for PCC 7002, S. meliloti and R. tropici CIAT are well under way, but most of us ran into problems with the design and/or the results of these experiments.
Erin obtained some unexpected results on her second exonuclease and ligation independent cloning (ELIC) experiment. The vector-only group, which is not supposed to confer resistance to antibiotic when transformed into E. coli, returned some growth. This may be due to faulty antibiotic, though it seems unlikely since the water-only control (row 4 in Fig. 1) returned the expected result of zero colonies.
Colin continued characterizing the growth and selection parameters for PCC7002 in the shaking incubator with ambient air (no supplemental CO2). He designed a growth assay which he ran on a 96-well plate to characterize the growth of the bacteria in the two candidate media (ATCC 1047 and A+) as well as to determine the minimum inhibitory concentrations (MICs) of six antibiotics of interest for PCC7002. Danny researched transformation methods for PCC7002 and designed experiments to test which method would me most efficient. He identified conjugation and natural competency as two possible methods for getting DNA into the cells, and began experimenting with these procedures. Erin focused less on PCC7002 issues this week to conduct her ELIC experiments for high-throughput DNA assembly (Fig. 1). This would be a fast alternative to Gibson assembly for putting together the large number of promoter-citrine and promoter-recombinase constructs we have designed. Members of the team focusing on PCC7002 studies (Colin, Danny, and Erin) took up much of Dan’s work since he left last week. This consists mainly of isolating several of the promoters he identified and assembling promoter-citrine constructs.
Colin and Danny also went to a hydroponics store to purchase a bright (20,000 lux) grow lamp to place on top of the PCC7002 shaking incubator.
The Rhizobium team also conducted selection assays to determine which antibiotics are most effective against Rhizobium tropici CIAT and Sinorhizobium meliloti 1021. Holly obtained inconclusive results in her selection assays (the assays report different resistances for what is supposedly the same organism), which may be an indication that the frozen stocks from which she was working are contaminated. Holly will conduct a colony PCR of the samples’ 16S region and send the products for sequencing. Since every prokaryotic species has a unique 16S sequence, this experiment should provide a definitive answer as to whether or not the frozen stocks are contaminated. Holly and Jessica continued working out transformation protocols for the strains, focusing mainly on conjugation. They conducted multiple conjugation experiments, along with an electroporation, and will analyze the results next week. They also conducted a centrifugation experiment in which they centrifuged CIAT 899 and Sm 356 at different centrifugation speeds; this was to determine which centrifugation speed to do their electroporations at (Fig. 2). Lionel spent much of his time in the Dellaporta lab working out LIC procedures. LIC is another high-throughput DNA assembly method which would provide an alternative to Gibson assembly.
Go to the Lab Notebook
Entry for week -1 1 2 3 4 5 6 7 8 9 10 10+
x
Lessons from Week 8
This week, we continued to make progress in the cyanobacteria and rhizobia growth assays, FLP constructs, cloning, transforming, and antibiotic assays.
We started with transforming pCP20, the plasmid containing the FLP recombinase, into two E. coli strains. While initially the FLP construct PCRs for cyanobacteria and rhizobia did not yield excellent results, we have successfully amplified the linear FLP construct for cyanobacteria. The construct for rhizobia has not yet been amplified and circularized successfully. We have furthermore been able to BsaI digest pKT230 and ligate the LIC cassette and the T7 terminator into it. The purpose of LIC is to simplify the cloning of promoter-citrine constructs into our backbone.
For cyanobacteria, we decided to use A+ media for culturing because the ATCC media is heavily reliant on seawater, which is variable. From our antibiotic assays, we determined that all antibiotics tested were effective at high concentrations (4x the standard E. coli concentrations). Figure 1 illustrates the inhibitory effects of different concentrations of kanamycin on PCC 7002 growth. Kanamycin's power for selection is promising, since it is the antibiotic most often cited in PCC 7002 transformation protocols and our FLP construct contains a kanamycin resistance cassette. So now we are confident that we will be able to select for kanR-positive transformants using kanamycin. We have also experimented with the live/dead sorting protocol (adapted from the LifeTech kit L-34856) to see if we could obtain single genotypes using liquid-only culturing methods because of the inconvenient length of time it takes to obtain single colonies through plating. Our preliminary results show that many of the UTEX cells form long chains that are difficult to sort so we will need to work on optimizing the protocol.
For the conjugations in rhizobia, we sequenced the colonies that grew on the conjugant plates and determined that the colonies were E. coli contaminants. We redid the conjugation, and the results were inconclusive because the negative controls were still able to grow on the antibiotic resistant plates, even when the antibiotic concentration was increased. To continue off of previous work, we redid the antibiotic assays, both in liquid culture and on solid agar plates for verification. We re-streaked strains obtained from the Jacobs-Wagner lab in case our stock cultures were contaminated. To troubleshoot our contamination issues, we experimented with growing the Rhizobia in LB instead of TSB, which has free phosphates that could cause the antibiotics to be less effective (Fig. 2).
Go to the Lab Notebook
Entry for week -1 1 2 3 4 5 6 7 8 9 10 10+
x
Lessons from Week 9
This week was spent as a wrapping up of the bulk of summer research for many researchers.  In cyanobacteria, Danny and Colin made important advances in determining optimal growth conditions, setting up a CRISPR cassette, and transforming linear DNA segments into cyanobacteria.  More specifically, Colin ran a natural transformation of his FLP cassette into 7002.  Danny finished amplifying the components of his CRISPR cassette and was able to Gibson assemble them together.  Danny also ran tests to determine the effect that carbon dioxide has on cyanobacterial growth, and his assay showed that proper carbon dioxide levels drastically increased growth. In rhizobium, Lionel made important progress setting up the LIC cassette in [pKT230] and analyzing electroporation results into the CIAT strain.  Lionel discovered, based off of restriction enzyme digests, that there might have been some issues with plasmid stability.  Holly and Jessica worked on making rhizobium cells electrocompetent as well as performing various electroporations into our rhizobium strains.  Erin, finished with ELIC, helped the team by researching and contacting other iGEM teams that were working in non model organisms.
Colin also ran a growth assay to test which media was best for growing 7002, and, after some troubleshooting, found that medium A+ was the most useful media for our purposes. He tested the growth of PCC 7002 in A+ medium, A+ medium without nitrates, and ATCC 1047 medium. This was compared to the growth of UTEX 2973 (another fast-growing freshwater cyanobacterium). Colin also tested growth in media supplemented with 10 mM glycerol. Some growth protocols for PCC 7002 suggest that growth in glycerol-supplemented media leads to modest improvements in growth rates without shunting it away from photosynthetic pathways. Colin found that growth of PCC 7002 was indeed faster in glycerol-supplemented media (see top two graphs), and confirmed that this growth condition does not shunt away from photosyntheses—for A+ medium, the ratio of chlorophyll to cell density (measured as OD600:OD730 as a proxy) remains constant throughout growth (bottom graph).
Go to the Lab Notebook
Entry for week -1 1 2 3 4 5 6 7 8 9 10 10+
x
Lessons from Week 10
All of the undergraduate wet lab researchers were gone this week except for Colin. Colin spent much of the week wrapping up experiments and confirming the presence of mutS-free genomes in transformed cell populations from last week.
The natural transformation which Colin ran last week yielded partially successful results. Neither of the linearized plasmids which he tried to transform (pKT230 and pETcoco-2) seem to be able to propagate in the cell. This could be due to multiple factors: The plasmids, which are both around 11kb, may be too large for the cell to absorb naturally. Alternatively, there may exist an incompatibility between the plasmid's origin of replication and PCC 7002's DNA replication machinery, or the promoter upstream of the antibiotic resistance genes (kanR for pKT230 and ampR for pETcoco-2) may not be active in PCC 7002. Finally, the AvaI restriction endonucleases present in the PCC 7002 cell may be cutting up the plasmid before it has a chance to replicate. We can't be sure which of these factors are preventing successful transformation without further experimentation.
While the transformation of plasmids into PCC 7002 was not successful, Colin seems to have success in transforming his linear FLP construct into the cell. Furthermore, the construct seems to have recombined successfully with the genome, resulting in kanamycin-resistant cell populations (Fig. 1).
Since each PCC 7002 cell has 5 to 6 chromosome copies, no cell may be homozygous for the mutS knockout. This was confirmed by PCR amplification of the mutS region of transformed cell populations and control populations (Fig. 2, below. The middle gel, in green, indicates non-homozygosity). Growing cells in high concentrations of kanamycin (up to 200 µg/ml) should encourage cells to increase the ratio of mutant genomes to wild-type genomes. The next challenge is to PCR-screen single colonies of cells grown on plates to isolate a homozygous genotype.
Ariel and Natalie also worked to assemble promoter-citrine constructs and amplify beta-homolog biobricks for submission to the parts registry. This effort will continue in the following weeks and when all of the undergraduate researchers return to campus.
Go to the Lab Notebook
Entry for week -1 1 2 3 4 5 6 7 8 9 10 10+
x
Lessons from Week 10+
Between Aug 10 and Aug 28, Lionel found time to continue working on the project. He made good progress on multiple fronts. He confirmed the orientation of the LIC-cassette in the pKT230-LIC vector. He then confirmed that both pKT230-LIC as well as pPZP200b-LIC could be successfully and consistently electroporated into the electrocompetent R. tropici and S. meliloti respectively that we prepared in Weeks 8 and 9. (R. tropici is already resistant to the spec marker on pPZP200b-LIC, while S. meliloti is already resistant to the kan marker on pKT230-Lic.) Lionel was also able to successfully insert the following promoter citrine constructs into both vectors by ligation-independent cloning: melA-cit, bacA-cit, AS-cit, AM-cit, AW-cit, tac-cit.
He then transformed the vectors by electroporation into E. coli. In pKT230-LIC, fluorescence was observed for AS-cit, AM-cit, AW-cit, and tac-cit, with some leaky expression observed for melA-cit and bacA-cit. Fluorescence for the same inserts was markedly lower in pPZP200b-LIC, possibly because of the lack of a downstream terminator. The plasmids were miniprepped from E. coli and transformed by electroporation into rhizobium. None of the pKT230-LIC transformations worked, and RE digest followed by gel-electrophoresis showed that we had once again run into problems with the instability of pKT230: even when the plasmid had the right size going into the LIC cloning, the miniprepped plasmids were markedly smaller. The pPZP200b-LIC transformations all worked, but we had low levels of fluorescence in line with the E. coli readings, with fluorescence observed for tac, AS and AM.
The lab had a meeting with our faculty advisors on August 31st to strategize for the upcoming few weeks. We have made significant progress towards our goal, even as we had hoped to be further along. Now, since school was starting, we need to prioritize experiments to get the most out of the project before the Jamboree in late September. With the help of our advisors, we identified goals which we feel are safe to achieve others which were reach targets. We did this for PCC 7002 and S. meliloti (we decided not to work with pKT230-LIC given the instability issues)
...
Safe (PCC 7002):
Isolate and confirm a homozygous ∆mutS mutant; characterize mutagenecy of ∆mutS mutants versus wild-type
Build promoter-citrine constructs
Build MAGE test constructs
...
Safe (S. meliloti):
Insert promoter-citrine-terminator constructs into pPZP200b and characterize in S. meliloti
Construct promoter-recombinase inserts and transform into S. meliloti (we need an assay designed for this, which might make this goal risky if it takes a long time to design the assay)
...
Risky (PCC 7002):
Successful plasmid transformation (since it takes 2-3 weeks for transformed PCC 7002 colonies to grow on plates, any experiment that involves transformation is risky at this point)
Successful promoter-citrine characterization
FLP out kanR cassette using pCP20 plasmid
...
Risky (S. meliloti):
CRISPR in S. meliloti
Knock out bacA as a MAGE test (bacA is a possible endogenous target for characterizing MAGE mutagenecy)
Promoter-citrine-terminator constructs have been successfully made and transformed into S. meliloti and fluorescence measurements are under way.
Promoter-recombinase constructs have been successful made and transformed into S. meliloti and MAGE testing is under way.
Go to the Lab Notebook
Entry for week -1 1 2 3 4 5 6 7 8 9 10 10+
x
Yale iGEM 2015
Main Campus:
Yale Department of Molecular, Cellular & Developmental Biology
Attn: Farren Isaacs/iGEM
219 Prospect St
PO Box 208103
New Haven, CT 06520
Tel. +1(203) 432-3783
E-mail: igem.yale@gmail.com
© Yale iGEM 2015
