<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Yale_methods skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Yale/methods</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P>&lt;!DOCTYPE html&gt;<TITLE>Yale iGem 2015: Project Methods</TITLE><NAV class="nav-contain"><UL class="nav"><LI><A href="https://2015.igem.org/Team:Yale" class="logoMark"><SPAN>Non</SPAN><SPAN>Model</SPAN></A></LI><LI class="submenu"><A href="https://2015.igem.org/Team:Yale/project">Project</A></LI><LI><A href="https://2015.igem.org/Team:Yale/project#overview" alt="Overview">Overview</A></LI><LI><A href="https://2015.igem.org/Team:Yale/methods" alt="Methods">Methods</A></LI><LI><A href="https://2015.igem.org/Team:Yale/results" alt="Results">Results</A></LI><LI><A href="https://2015.igem.org/Team:Yale/modeling" alt="Modeling">Modeling</A></LI><LI><A href="https://2015.igem.org/Team:Yale/parts" alt="Parts">Parts</A></LI></UL><LI class="submenu"><A href="notebook">Notebook</A></LI><LI><A href="https://2015.igem.org/Team:Yale/notebook" alt="Weekly">Weekly</A></LI><LI><A href="https://static.igem.org/mediawiki/2015/f/fb/Yale_iGEM_Project_Summary_2015.pdf" alt="PDF Summary">PDF Summary</A></LI><LI class="submenu"><A href="collaborations">Collaborations</A></LI><LI class="submenu"><A href="practices">Human Practices</A></LI><LI><A href="https://2015.igem.org/Team:Yale/practices#video" alt="Documentary">Documentary</A></LI><LI><A href="https://2015.igem.org/Team:Yale/practices#ssri" alt="SSRI">SSRI</A></LI><LI><A href="https://2015.igem.org/Team:Yale/practices#lgbtq" alt="LGBTQ Survey">LGBTQ Survey</A></LI><LI class="submenu"><A href="team">Team</A></LI><LI><A href="https://2015.igem.org/Team:Yale/team" alt="People">People</A></LI><LI><A href="https://2015.igem.org/Team:Yale/Attributions" alt="Acknowledgements">Attributions</A></LI><LI class="submenu"><A href="standards">Standards</A></LI><LI><A href="https://2015.igem.org/Team:Yale/standards#gold" alt="Gold">Gold</A></LI><LI><A href="https://2015.igem.org/Team:Yale/standards#silver" alt="Silver">Silver</A></LI><LI><A href="https://2015.igem.org/Team:Yale/standards#bronze" alt="Bronze">Bronze</A></LI></NAV></P><H2 class="page__head">Developing a Framework for the Genetic Manipulation of Non-Model and Environmentally Significant Microbes</H2><DIV class="row overview__blocks"><DIV class="small-12 switch-border"><H2><A href="project#overview" alt="Overview">Overview</A></H2><H2><A href="methods" alt="Methods">Methods</A></H2><H2><A href="results" alt="Results">Results</A></H2><H2><A href="modeling" alt="Modeling">Modeling</A></H2><H2><A href="parts" alt="Parts">Parts</A></H2></DIV></DIV><SECTION class="content__section"><H2 id="overview">Project Strategy- Overview</H2><H3>Click on a component to learn more about our method.</H3></SECTION><DIV id="clickmap" class="contain__overview"><DIV class="left__block"><H2>Technique-Independent Procedures</H2></DIV><DIV class="diverge-arrows"><SPAN class="bigblock">»</SPAN><SPAN class="bigblock">»</SPAN></DIV><DIV class="right__block"><DIV class="upper" style="padding: 0 0 15px 0; background-color:#a5a5a5"><H2>MAGE Implementation Pathway</H2></DIV><DIV class="lower" style="background-color:#a5a5a5"><H2>CRISPR-Cas9 Implementation Pathway</H2></DIV></DIV></DIV><SECTION class="content__section__alt"><H2 id="growth" class="overview__head">Growth</H2><DIV class="row over__blocks"><DIV class="small-6 columns cyano_block"><H2>Cyanobacterium</H2><P> One major component to characterizing nonmodel organisms is determining optimal growth conditions. For cyanobacteria, a key hurdle was building our own shaking incubator with a bubbling CO2 source. We also determined which media to use for growing our strains, both for liquid and solid cultures. Once we were able to successfully culture our strains, we then conducted growth assays to calculate doubling times. </P><DIV class="launch__modal"><A href="https://static.igem.org/mediawiki/2015/7/7b/Growth_Conditions_PC7002.pdf" target="_blank" class="custom__button">Protocol PDF</A></DIV></DIV><DIV class="small-6 columns sino_block"><H2 class="sino__head">Sinorhizobium</H2><P> Similar to cyanobacteria, we experimented with different growing media--we used both Tryptic Soy Broth and LB media, finally settling on using LB in an effort to reduce contamination problems. Our other rhizobium growing conditions were taken from literature sources; we did not have any issues with obtaining the correct equipment, but we did need to conduct various growth assays to determine doubling times and mid-log optical density.  </P><DIV class="launch__modal"><A href="https://static.igem.org/mediawiki/2015/8/83/Growth_Conditions_Rhizobium.pdf" target="_blank" class="custom__button">Protocol PDF</A></DIV></DIV></DIV></SECTION><SECTION class="content__section__alt overview__section"><H2 id="transform" class="overview__head">Transformation</H2><DIV class="row over__blocks"><DIV class="small-6 columns cyano_block"><H2>Cyanobacterium</H2><P> The protocol that follows was designed to screen for recombination events, so it is useful for gene knockouts (i.e. FLP-FRT knockout and recombination). The linear construct to be transformed must have 600 to 1000 bp homology arms both up- and downstream of the DNA fragment of interest in order to successfully recombine with the PCC7002 genome. Yale iGEM tested 1000 bp fragments, but other literature protocols report successful recombinations with homology arms as short as 600 bp (Ruffing 2014). The homology arms should be amplified by colony PCR and assembled into the full linear construct by Gibson assembly. </P><DIV class="launch__modal"><A href="https://static.igem.org/mediawiki/2015/5/5c/PCC7002_Transformation_Recombination_Protocol.pdf" target="_blank" class="custom__button">Protocol PDF</A></DIV></DIV><DIV class="small-6 columns sino_block"><H2 class="sino__head">Sinorhizobium</H2><P>In sinorhizobium, the two main transformation protocols we experimented with were electroporation and conjugation. Conjugation was difficult in rhizobium because many of our strains had natural resistances to antibiotics--we had to conduct a series of antibiotic assays to figure out which selective conjugation plates to make. Our electroporation protocol was modeled after the standard E. coli one; we explored different centrifugation speeds and incubation periods.</P><DIV class="launch__modal"><A href="https://static.igem.org/mediawiki/2015/5/5c/Rhizobium_Electroporation_Protocol.pdf" target="_blank" class="custom__button">Electroporation Protocol PDF</A></DIV><DIV class="launch__modal"><A href="https://static.igem.org/mediawiki/2015/7/78/Rhizobium_Conjugation_Protocol.pdf" target="_blank" class="custom__button">Conjugation Protocol PDF</A></DIV></DIV></DIV></SECTION><SECTION class="content__section__alt overview__section"><H2 id="select" class="overview__head">Selection</H2><DIV class="row over__blocks"><DIV class="small-6 columns cyano_block"><H2>Cyanobacterium</H2><P> This protocol was designed for determine the minimum inhibitory concentrations (MICs) of various antibiotics of interest for the cyanobacterium Synechococcus sp. PCC7002, but can be adapted for determining the MICs of antibiotics for any non-model microorganism. The experiment is designed to run in a 96-well plate placed in a shaking incubator. The antibiotics and concentrations tested can vary depending on the MICs cited in the literature; this experiment can be run to determine the effectiveness of an antibiotic in new liquid growth media, or to verify literature values. We recommend running an analogous parallel assay with E. coli DH5ɑ (in LB media at the pH of the media for the non-model organism) to determine if pH or media composition plays a role in the effectiveness of the antibiotic. The antibiotic concentrations chosen were relative to standard E. coli MICs. Testing six antibiotics on one plate allows for the testing of two media controls. </P><DIV class="launch__modal"><A href="https://static.igem.org/mediawiki/2015/3/31/Yale_PCC7002_Selection_Protocol.pdf" target="_blank" class="custom__button">Protocol PDF</A></DIV></DIV><DIV class="small-6 columns sino_block"><H2 class="sino__head">Sinorhizobium</H2><P>This protocol was designed for determine the minimum inhibitory concentrations (MICs) of various antibiotics of interest for the Rhizobium sp. R. tropeci and S. meliloti, but can be adapted for determining the MICs of antibiotics for any non-model microorganism. The experiment is designed to run in a 96-well plate placed in a shaking incubator. The antibiotics and concentrations tested can vary depending on the MICs cited in the literature; this experiment can be run to determine the effectiveness of an antibiotic in new liquid growth media, or to verify literature values. We recommend running an analogous parallel assay with E. coli DH5ɑ (in LB media at the pH of the media for the non-model organism) to determine if pH or media composition plays a role in the effectiveness of the antibiotic. The antibiotic concentrations chosen were relative to standard E. coli MICs. Testing six antibiotics on one plate allows for the testing of two media controls.</P><DIV class="launch__modal"><A href="https://static.igem.org/mediawiki/2015/f/fa/Selection_for_rhizo.pdf" target="_blank" class="custom__button">Protocol PDF</A></DIV></DIV></DIV></SECTION><SECTION class="content__section"><H2 id="mage1">Identifying Potential Parts</H2><H3>Identifying Promoters</H3><P>Promoters were identified through extensive literature searches on the organisms in question. Once promoters were identified, they were sought for in the iGEM registry and if not found, PCR amplified from the source organism.</P><H3>Identifying Recombinases</H3><P> MAGE accomplishes targeted mutagenesis through the annealing and incorporation of mutagenic DNA oligonucleotides into an actively replicating DNA strand. This annealing is mediated by a single-stranded binding protein. In E. coli, the most efficient annealing protein is Beta recombinase from the phage Lambda.</P><P>For affecting MAGE recombineering in Cyanobacteria and Rhizobacteria, we identified a panel of Beta protein homologs from across various phage and prophage genomes, as well as a panel of other representative classes of single-stranded DNA binding recombinases. This was accomplished by extensive NCBI Blast searches for protein structures similar to that of Lambda Phage beta-recombinase.</P><P>This panel of recombinases will be cloned into vectors containing either constitutive or inducible promoters (validated through fluorescent assays) for each of the test strains. The resulting matrix of promoter – recombinase variants will them be tested from MAGE efficiencies. <SPAN class="return"><A href="#top"> Back to Top</A></SPAN></P></SECTION><SECTION class="content__section"><H2 id="mage2">Gene Knockouts for Mage</H2><H3>FLP-FRT Recombination</H3><P> FLP-FRT recombination was used to achieve a marker-free knockout of the mismatch repair gene mutS, as well as to enable future genomic manipulations in cyanobacteria PCC7002. In brief, this two-step procedure first integrates a selectable marker, flanked by two FRT sites, into a targeted genomic locus, then employs the FLP recombinase to “flip out” the selectable marker, leaving behind a single FRT site as a scar.
</P><P>To knock out mutS, we designed overlap PCR primers to amplify the kanamycin resistance gene with flanking FRT sites, as well as 1000bp homology arms complementary to the sequence flanking mutS. This cassette was circularized via ligation, and subsequently transformed naturally into PCC7002 by adding the DNA into the liquid growth media. Native cyanobacterial recombination machinery was employed to mediate the integration. After recovery, the cell culture was passaged into media containing kanamycin. Surviving cells were assayed via PCR for a mutS::kanR replacement.</P><P>Knockout clones will then be transformed with a plasmid-borne <A href="http://cgsc.biology.yale.edu/Site.php?ID=64621">FLP recombinase</A>, which will excise the kanamycin resistance cassette. This excision will be verified via PCR.<SPAN class="return"><A href="#top"> Back to Top</A></SPAN></P></SECTION><SECTION class="content__section"><H2 id="crispr5">Gene KnockOuts With CRISPR</H2><P> As a complementary approach for cyanobacterial genetic engineering, we investigated the potential use of CRISPR-cas9. Strain PCC7002 is capable of both nonhomologous end joining and homologous recombination, suggesting that this approach could be used for both gene knockouts and knock-ins, respectively</P><P>In order to test this capability, we pursued a plasmid-based expression system, wherein cas9 would be under the control of the nitrate-inducible promoter nirA, while the sgRNA would be constitutively expressed under the control of the cpcAB promoter. Placing cas9 under an inducible promoter would allow us to titer activation to mitigate any possible toxicity and reduce the accumulation of off-target effects.</P><P>To test the efficiency of this system, we will target disruption of the ureC gene, encoding the alpha subunit of urea permease, which confers resistance to urea. Upon CRISPR induction, we will measure the proportion of surviving clones resistant to urea. We may swap certain expression parameters to optimize this efficiency value if needed. <SPAN class="return"><A href="#top"> Back to Top</A></SPAN></P></SECTION><SECTION class="content__section"><H2 id="mage3">DNA Assemblies</H2><DIV class="row content__dark"><DIV class="small-4 columns readable"><H4>Gibson Assembly</H4><P>Gibson Assembly cloning enables the assembly of arbitrary DNA fragments using overlapping homologous ends. This isothermal reaction employs a DNA exonuclease to create single-stranded DNA overhangs in each fragment; a DNA polymerase to fill in gaps upon annealing of complementary fragments; and a DNA ligase to repair remaining nicks. For our purposes we utilized the<A href="https://www.neb.com/products/e5510-gibson-assembly-cloning-kit"> NEB Gibson Assembly Cloning Kit.</A></P></DIV><DIV class="small-4 columns readable"><H4>LIC</H4><P> Ligation-independent cloning makes constructs without use of ligase. LIC-compatible vectors contain the LIC cassette, and these vectors are linearized with 14bp single-strand overhangs when cut with BsaI and treated with T4-polymerase and one appropriate dNTP. These overhangs are complementary to overhangs generated in inserts amplified with the appropriate primers and subsequently T4-polymerase-treated. </P><DIV class="launch__modal"><A href="https://static.igem.org/mediawiki/2015/4/40/LIC_Cloning_Protocol.pdf" target="_blank" class="custom__button">Protocol PDF</A></DIV></DIV><DIV class="small-4 columns readable"><H4>ELIC</H4><P>The protocol that follows describes a new type of molecular cloning. Linear double-stranded DNA with homology are mixed in water, undergo homologous recombination, and are readily transformed into E. coli. </P><DIV class="launch__modal"><A href="https://static.igem.org/mediawiki/2015/5/50/ELIC_Protocol.pdf" target="_blank" class="custom__button">Protocol PDF</A></DIV></DIV></DIV></SECTION><SECTION class="content__section"><H2 id="mage4">Testing DNA Constructs</H2><DIV class="row content__dark"><DIV class="small-6 columns readable"><H4>Testing Promoters</H4><P>The first step in our plan is to test the effectiveness of our identified promoters by placing them upstream of a fluorescent reporter; we will use the yellow-fluorescent citrine protein since its emission wavelength (530) does not overlap with the auto-fluorescence wavelengths of our cyanobacteria. After identifying the most effective promoter (lowest leakiness and highest expression level when induced), we can replace the citrine gene in our construct with our synthesized beta-homolog genes.</P></DIV><DIV class="small-6 columns readable"><H4>Testing Beta-Homologs</H4><P>In order to test beta-homolog activity in our non-model organisms we located genes that when modified can confer a novel phenotype. In cyanobacteria we can target the ureC gene which when knocked out confers resistance to urea and nickel sulfate. <A href="http://www.ncbi.nlm.nih.gov/pubmed/9647800">(Sakamoto et al. 1998)</A>. In the case of rhizobium we can target the bacA gene which when knocked out confers greater bleomycin resistance <A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1447001/">(Ferguson et al. 2006)</A>. By introducing oligonucleotide of various sizes (70mer, 90mer, and 120mer) we can knock out the gene and plate the bacteria in selective media and non-selective media and observe the MAGE efficiency of the knock out <A href="http://www.nature.com/nature/journal/v460/n7257/full/nature08187.html">(Wang et al. 2009)</A>. Useful beta-homologs will demonstrate increased MAGE efficiency.</P></DIV></DIV></SECTION><SECTION class="content__section"><H2 id="mage7">Optimizing for Highest Mutagenesis Efficiency</H2><P>We also designed a method for testing the mutagenicity of an inducible recombinase system (MAGE) at multiple resolutions while simultaneously knocking out mutS. A linear construct consisting of an FRT-flanked kanamycin cassette, an ampicillin resistance gene (ampR) with a premature stop codon, and an mCit gene with a premature stop codon (all flanked by 1000 bp homology arms) can be incorporated at the genomic mutS locus. Oligonucleotides targeting restored function of the premature stop codon of either the ampR cassette or the mCit gene can be transformed into mutants, along with recombinases under inducible promoters. Targeting either broken gene will yield a different dynamic range for assaying the mutation frequency per MAGE cycle—the ampR gene can be targeted to determine mutation frequencies on the order of 10-7 to 10-5, while the mCit gene can be targeted to determine mutation frequencies from 10-4 to 10-1. <SPAN class="return"><A href="#top"> Back to Top</A></SPAN></P></SECTION><SECTION class="content__section"><H2>Safety - Protocol from 2014</H2><H3>Researchers</H3><DIV class="readable"><P> Our laboratory is certified for Biosafety Level 1 (BS-1) work, and we have access to a Biosafety Level 2 lab for BSL2 work. Our work fell within the BSL-1 domain, as indicated by the Center of Disease Control guidelines. All biological waste was stored in autoclave bags and was autoclaved prior to disposal. Sharps and broken glassware were disposed of according to institutional guidelines. Hazardous liquid waste was clearly labeled, and stored in secondary containment for disposal by the institution. Thus, although there is potential for harm to researchers, it is minimized through following procedures approved and used by many laboratories at Yale. It is also minimized by training and common sense. E. coli strains used were common laboratory strains and not pathogenic.</P><UL><LI>http://ehs.yale.edu/training/biosafetypartI</LI><LI>http://ehs.yale.edu/training/biosafetypartII</LI><LI>http://learn.caim.yale.edu/chemsafe/login.html</LI><LI>http://ehs.yale.edu/training/hazardous-chemical-waste-management</LI></UL><P>All materials were used in accordance with local, national, and Yale Biosafety requirements. Standard lab practices were followed, including secondary containment of chemicals, proper storage of volatiles and flammables, and separation of acids and bases. Nitrile gloves were worn at all times within the lab. A pipet was kept exclusively for ethidium bromide use. Fume hoods were used when handling volatile compounds, concentrated acids and bases, and other reagents. Inhalation and skin contact was avoided. Chemical agents were properly disposed of in designated biohazard waste bins. When UV light was used to visualize gels or GFP, special care was taken to avoid skin or eye exposure. Absolutely no food was allowed in the lab.</P><P>Our project was overseen by the Yale Biological Safety Committee and the Office of Environmental Health and Safety (OEHS). Our project has been approved as consistent with Yale's safety regulations. No changes to our project were required since proper protocols were followed. Training was completed as described above.p </P><H3>Public Safety</H3><P>We do not anticipate any threat to public safety. Organisms worked with are all non-pathogenic. They are likely unable to survive outside of the lab environment, because they will be unable to compete with other organisms in nature. Biomaterials were autoclaved after use. We did not use gloves to touch doors outside of the laboratory to avoid others coming into contact with our chemical and biological agents. </P><P>Hands were washed before and after leaving the laboratory. </P><P>Our project will ideally be scaled up for medical and industrial use. Possible issues are allergies to the product and if used in immense qualities, DOPA toxicity to the environment. However, proteins and amino acids will be degraded and recycled by the environment, so we do not believe the product will be extremely harmful. </P><P>If we continued future work on this project we would assess the risks and hazards associated with it. Once understood, we would attempt to alter the design to minimize these risks. The E. Coli chassis for the product we are designing is not likely to be a great risk on its own.</P><H3>Environmental Safety</H3><P>There are no additional risks posed by our projects compared to other general BSL1 lab concerns. Our bacteria are not pathogenic and are unable to survive outside of the lab environment, because they are unable to effectively compete with other organisms in nature. They do not cause adverse reactions in immunocompetent humans. They do not cause infection. Toxic materials were all disposed of according to Yale waste standards to prevent adverse environmental impact. </P><P>There are no safety issues raised by the BioBrick parts submitted to the Registry this year. None of our constructs create toxic gene products for humans or animals.</P></DIV><DIV id="growthOne" data-reveal="" aria-labelledby="iGEM Modal" aria-hidden="true" role="dialog" class="reveal-modal greenModal"><H2 class="modal__title">Cyanothingy Growth Facts</H2><P>These are the tales of Steven Spinach, one brave molecule. </P><P> Lorem ipsum dolor sit amet, consectetur adipisicing elit. Necessitatibus enim non iste molestias, facere repellendus assumenda, ipsum molestiae praesentium magni fugiat sit? Reprehenderit accusantium ratione quia cumque recusandae, tempora temporibus?</P><P>Biofilm formation on surfaces is an issue in the medical field, naval industry, and other areas. We developed an anti-fouling peptide with two modular components: a mussel adhesion protein (MAP) anchor and LL-37, an antimicrobial peptide. MAPs can selectively attach to metal and organic surfaces via L-3,5-dihydroxyphenylalanine (L-DOPA), a nonstandard amino acid that was incorporated using a genetically recoded organism (GRO). Because this peptide is toxic to the GRO in which it is produced, we designed a better controlled inducible system that limits basal expression. This was achieved through a novel T7 riboregulation system that controls expression at both the transcriptional and translational levels.</P><P>Biofilm formation on surfaces is an issue in the medical field, naval industry, and other areas. We developed an anti-fouling peptide with two modular components: a mussel adhesion protein (MAP) anchor and LL-37, an antimicrobial peptide. MAPs can selectively attach to metal and organic surfaces via L-3,5-dihydroxyphenylalanine (L-DOPA), a nonstandard amino acid that was incorporated using a genetically recoded organism (GRO). Because this peptide is toxic to the GRO in which it is produced, we designed a better controlled inducible system that limits basal expression. This was achieved through a novel T7 riboregulation system that controls expression at both the transcriptional and translational levels.</P><P>Biofilm formation on surfaces is an issue in the medical field, naval industry, and other areas. We developed an anti-fouling peptide with two modular components: a mussel adhesion protein (MAP) anchor and LL-37, an antimicrobial peptide. MAPs can selectively attach to metal and organic surfaces via L-3,5-dihydroxyphenylalanine (L-DOPA), a nonstandard amino acid that was incorporated using a genetically recoded organism (GRO). Because this peptide is toxic to the GRO in which it is produced, we designed a better controlled inducible system that limits basal expression. This was achieved through a novel T7 riboregulation system that controls expression at both the transcriptional and translational levels.</P><A href="#">Download a PDF of our Protocol!</A><A aria-label="Close" class="close-reveal-modal">x</A></DIV><DIV id="growthTwo" data-reveal="" aria-labelledby="iGEM Modal" aria-hidden="true" role="dialog" class="reveal-modal redModal"><H2 class="modal__title">Sinorhizobium Growth Facts</H2><P>Many a young child went swimming today and learned about the sea</P><P>Biofilm formation on surfaces is an issue in the medical field, naval industry, and other areas. We developed an anti-fouling peptide with two modular components: a mussel adhesion protein (MAP) anchor and LL-37, an antimicrobial peptide. MAPs can selectively attach to metal and organic surfaces via L-3,5-dihydroxyphenylalanine (L-DOPA), a nonstandard amino acid that was incorporated using a genetically recoded organism (GRO). Because this peptide is toxic to the GRO in which it is produced, we designed a better controlled inducible system that limits basal expression. This was achieved through a novel T7 riboregulation system that controls expression at both the transcriptional and translational levels.</P><P>Biofilm formation on surfaces is an issue in the medical field, naval industry, and other areas. We developed an anti-fouling peptide with two modular components: a mussel adhesion protein (MAP) anchor and LL-37, an antimicrobial peptide. MAPs can selectively attach to metal and organic surfaces via L-3,5-dihydroxyphenylalanine (L-DOPA), a nonstandard amino acid that was incorporated using a genetically recoded organism (GRO). Because this peptide is toxic to the GRO in which it is produced, we designed a better controlled inducible system that limits basal expression. This was achieved through a novel T7 riboregulation system that controls expression at both the transcriptional and translational levels.</P><P>Biofilm formation on surfaces is an issue in the medical field, naval industry, and other areas. We developed an anti-fouling peptide with two modular components: a mussel adhesion protein (MAP) anchor and LL-37, an antimicrobial peptide. MAPs can selectively attach to metal and organic surfaces via L-3,5-dihydroxyphenylalanine (L-DOPA), a nonstandard amino acid that was incorporated using a genetically recoded organism (GRO). Because this peptide is toxic to the GRO in which it is produced, we designed a better controlled inducible system that limits basal expression. This was achieved through a novel T7 riboregulation system that controls expression at both the transcriptional and translational levels.</P><A aria-label="Close" class="close-reveal-modal">x</A></DIV><DIV id="transformOne" data-reveal="" aria-labelledby="iGEM Modal" aria-hidden="true" role="dialog" class="reveal-modal greenModal"><H2 class="modal__title">Cyan Transforms</H2><P>These are the tales of Steven Spinach, one brave molecule.</P><P> Lorem ipsum dolor sit amet, consectetur adipisicing elit. Necessitatibus enim non iste molestias, facere repellendus assumenda, ipsum molestiae praesentium magni fugiat sit? Reprehenderit accusantium ratione quia cumque recusandae, tempora temporibus?</P><P>Biofilm formation on surfaces is an issue in the medical field, naval industry, and other areas. We developed an anti-fouling peptide with two modular components: a mussel adhesion protein (MAP) anchor and LL-37, an antimicrobial peptide. MAPs can selectively attach to metal and organic surfaces via L-3,5-dihydroxyphenylalanine (L-DOPA), a nonstandard amino acid that was incorporated using a genetically recoded organism (GRO). Because this peptide is toxic to the GRO in which it is produced, we designed a better controlled inducible system that limits basal expression. This was achieved through a novel T7 riboregulation system that controls expression at both the transcriptional and translational levels.</P><P>Biofilm formation on surfaces is an issue in the medical field, naval industry, and other areas. We developed an anti-fouling peptide with two modular components: a mussel adhesion protein (MAP) anchor and LL-37, an antimicrobial peptide. MAPs can selectively attach to metal and organic surfaces via L-3,5-dihydroxyphenylalanine (L-DOPA), a nonstandard amino acid that was incorporated using a genetically recoded organism (GRO). Because this peptide is toxic to the GRO in which it is produced, we designed a better controlled inducible system that limits basal expression. This was achieved through a novel T7 riboregulation system that controls expression at both the transcriptional and translational levels.</P><P>Biofilm formation on surfaces is an issue in the medical field, naval industry, and other areas. We developed an anti-fouling peptide with two modular components: a mussel adhesion protein (MAP) anchor and LL-37, an antimicrobial peptide. MAPs can selectively attach to metal and organic surfaces via L-3,5-dihydroxyphenylalanine (L-DOPA), a nonstandard amino acid that was incorporated using a genetically recoded organism (GRO). Because this peptide is toxic to the GRO in which it is produced, we designed a better controlled inducible system that limits basal expression. This was achieved through a novel T7 riboregulation system that controls expression at both the transcriptional and translational levels.</P><A href="#">Download a PDF of our Protocol!</A><A aria-label="Close" class="close-reveal-modal">x</A></DIV><DIV id="transformTwo" data-reveal="" aria-labelledby="iGEM Modal" aria-hidden="true" role="dialog" class="reveal-modal redModal"><H2 class="modal__title">Sinorhizobium Transforms</H2><P>Many a young child went swimming today and learned about the sea</P><P>Biofilm formation on surfaces is an issue in the medical field, naval industry, and other areas. We developed an anti-fouling peptide with two modular components: a mussel adhesion protein (MAP) anchor and LL-37, an antimicrobial peptide. MAPs can selectively attach to metal and organic surfaces via L-3,5-dihydroxyphenylalanine (L-DOPA), a nonstandard amino acid that was incorporated using a genetically recoded organism (GRO). Because this peptide is toxic to the GRO in which it is produced, we designed a better controlled inducible system that limits basal expression. This was achieved through a novel T7 riboregulation system that controls expression at both the transcriptional and translational levels.</P><P>Biofilm formation on surfaces is an issue in the medical field, naval industry, and other areas. We developed an anti-fouling peptide with two modular components: a mussel adhesion protein (MAP) anchor and LL-37, an antimicrobial peptide. MAPs can selectively attach to metal and organic surfaces via L-3,5-dihydroxyphenylalanine (L-DOPA), a nonstandard amino acid that was incorporated using a genetically recoded organism (GRO). Because this peptide is toxic to the GRO in which it is produced, we designed a better controlled inducible system that limits basal expression. This was achieved through a novel T7 riboregulation system that controls expression at both the transcriptional and translational levels.</P><P>Biofilm formation on surfaces is an issue in the medical field, naval industry, and other areas. We developed an anti-fouling peptide with two modular components: a mussel adhesion protein (MAP) anchor and LL-37, an antimicrobial peptide. MAPs can selectively attach to metal and organic surfaces via L-3,5-dihydroxyphenylalanine (L-DOPA), a nonstandard amino acid that was incorporated using a genetically recoded organism (GRO). Because this peptide is toxic to the GRO in which it is produced, we designed a better controlled inducible system that limits basal expression. This was achieved through a novel T7 riboregulation system that controls expression at both the transcriptional and translational levels.</P><A aria-label="Close" class="close-reveal-modal">x</A></DIV><DIV id="selectOne" data-reveal="" aria-labelledby="iGEM Modal" aria-hidden="true" role="dialog" class="reveal-modal greenModal"><H2 class="modal__title">Cyan Selects</H2><P>Biofilm formation on surfaces is an issue in the medical field, naval industry, and other areas. We developed an anti-fouling peptide with two modular components: a mussel adhesion protein (MAP) anchor and LL-37, an antimicrobial peptide. MAPs can selectively attach to metal and organic surfaces via L-3,5-dihydroxyphenylalanine (L-DOPA), a nonstandard amino acid that was incorporated using a genetically recoded organism (GRO). Because this peptide is toxic to the GRO in which it is produced, we designed a better controlled inducible system that limits basal expression. This was achieved through a novel T7 riboregulation system that controls expression at both the transcriptional and translational levels.</P><A href="#">Download a PDF of our Protocol!</A><A aria-label="Close" class="close-reveal-modal">x</A></DIV><DIV id="selectTwo" data-reveal="" aria-labelledby="iGEM Modal" aria-hidden="true" role="dialog" class="reveal-modal redModal"><H2 class="modal__title">Sinorhizobium Selects</H2><P>Biofilm formation on surfaces is an issue in the medical field, naval industry, and other areas. We developed an anti-fouling peptide with two modular components: a mussel adhesion protein (MAP) anchor and LL-37, an antimicrobial peptide. MAPs can selectively attach to metal and organic surfaces via L-3,5-dihydroxyphenylalanine (L-DOPA), a nonstandard amino acid that was incorporated using a genetically recoded organism (GRO). Because this peptide is toxic to the GRO in which it is produced, we designed a better controlled inducible system that limits basal expression. This was achieved through a novel T7 riboregulation system that controls expression at both the transcriptional and translational levels.</P><A href="#">Download a PDF of our Protocol!</A><A aria-label="Close" class="close-reveal-modal">x</A></DIV></SECTION><DIV class="igem__footer"><DIV class="foot__contain"><DIV class="row foot__row"><DIV class="small-7 columns foot__text"><H2>Yale iGem 2015</H2><P>Main Campus:</P><P>Yale Department of Molecular, Cellular &amp; Developmental Biology</P><P>Attn: Farren Isaacs/iGEM</P><P>219 Prospect St</P><P>PO Box 208103</P><P>New Haven, CT 06520</P><P>Tel. +1(203) 432-3783</P><P>E-mail: igem.yale@gmail.com</P></DIV></DIV><DIV class="row"><P class="text-center">© Yale iGEM 2015</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>