<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Yale_Project skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Yale/Project</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Yale iGem 2015: Project Overview</TITLE><NAV class="nav-contain"><UL class="nav"><LI><A href="index.html" class="logoMark"><SPAN>Non</SPAN><SPAN>Model</SPAN></A></LI><LI class="submenu"><A href="https://2015.igem.org/Team:Yale/project">Project</A></LI><LI><A href="https://2015.igem.org/Team:Yale/project#overview" alt="Overview">Overview</A></LI><LI><A href="https://2015.igem.org/Team:Yale/methods" alt="Methods">Methods</A></LI><LI><A href="https://2015.igem.org/Team:Yale/results" alt="Results">Results</A></LI><LI><A href="https://2015.igem.org/Team:Yale/modeling" alt="Modeling">Modeling</A></LI><LI><A href="https://2015.igem.org/Team:Yale/parts" alt="Parts">Parts</A></LI></UL><LI><A href="https://2015.igem.org/Team:Yale/notebook" alt="Notebook">Notebook</A></LI><LI class="submenu"><A href="collaborations">Collaborations</A></LI><LI><A href="https://2015.igem.org/Team:Yale/collaborations#guidebook" alt="Handbook">Handbook</A></LI><LI><A href="https://2015.igem.org/Team:Yale/collaborations#protocat" alt="Protocat">Protocat</A></LI><LI class="submenu"><A href="practices">Human Practices</A></LI><LI><A href="https://2015.igem.org/Team:Yale/practices#video" alt="Documentary">Documentary</A></LI><LI><A href="https://2015.igem.org/Team:Yale/practices#ssri" alt="SSRI">SSRI</A></LI><LI><A href="https://2015.igem.org/Team:Yale/practices#lgbtq" alt="LGBTQ Survey">LGBTQ Survey</A></LI><LI class="submenu"><A href="team">Team</A></LI><LI><A href="https://2015.igem.org/Team:Yale/team#people" alt="People">People</A></LI><LI><A href="https://2015.igem.org/Team:Yale/team#acknowledgements" alt="Acknowledgements">Acknowledgements</A></LI><LI class="submenu"><A href="standards">Standards</A></LI><LI><A href="https://2015.igem.org/Team:Yale/standards#gold" alt="Gold">Gold</A></LI><LI><A href="https://2015.igem.org/Team:Yale/standards#silver" alt="Silver">Silver</A></LI><LI><A href="https://2015.igem.org/Team:Yale/standards#bronze" alt="Bronze">Bronze</A></LI></NAV></P><H2 class="page__head">Developing a Framework for the Genetic Manipulation of Non-Model and Environmentally Significant Microbes</H2><DIV class="row overview__blocks"><DIV class="small-12 switch-border"><H2><A href="project#overview" alt="Overview">Overview</A></H2><H2><A href="methods" alt="Methods">Methods</A></H2><H2><A href="results" alt="Results">Results</A></H2><H2><A href="modeling" alt="Modeling">Modeling</A></H2><H2><A href="parts" alt="Parts">Parts</A></H2></DIV></DIV><SECTION class="content__section"><H2 id="overview">Project Overview</H2><H3>Let's make today's most important microbes easier to engineer</H3><DIV class="page__button"><A href="#summary" class="custom__button">Overview</A><A href="#rationale" class="custom__button">Rationale</A><A href="#mage" class="custom__button">Mage</A><A href="#crispr" class="custom__button">CRISPR</A><A href="#to1" class="custom__button">Target Organism 1</A><A href="#to2" class="custom__button">Target Organism 2</A><A href="#outcomes" class="custom__button">Potential Outcomes</A><A href="#references" class="custom__button">References</A></DIV></SECTION><SECTION class="content__section"><H2 id="summary">Overview</H2><P>Our iGEM research project involves developing a framework for the implementation of genetic manipulation techniques'specifically, multiplex automated genome engineering (MAGE) and CRISPR-Cas9 systems into non-model, environmentally significant microbes. MAGE was developed as a rapid, high efficiency tool for increasing the genetic diversity of a cell population at targeted loci within the genome, and has so far been ported into the model organism <EM>Escherichia coli</EM> and a few other members of the family Enterobacteriaceae (unpublished data). CRISPR (clustered regularly interspaced short palindromic repeats)/Cas systems, based on prokaryotic adaptive immunity mechanisms, have emerged as a powerful cleavage-based genome editing technique (Cong et al. 2013).</P><P>The organisms we used to develop our framework are <EM>Rhizobium tropici</EM> CIAT and <EM>Synechococcus</EM> sp. PCC 7002. <EM>Rhizobium tropici </EM>CIAT 899 (hereafter RT-CIAT) is a nitrogen-fixing bacterium capable of forming root nodules with certain legume-producing <EM>Leucaena</EM> and <EM>Phaseolus</EM> trees (Martinez-Romero et al). <EM>Synechococcus</EM> sp. PCC 7002 (hereafter PCC7002) is a fast-growing marine cyanobacterium capable of photosynthesis and free fatty acid (FFA) production (Ruffing 2014).</P><P>We envision numerous potential applications for MAGE, CRISPR-Cas9, and other genetic manipulation techniques in these organisms; for example, the nitrogen fixation mechanisms in <EM>Rhizobium tropici</EM> CIAT could be modified to enable plant growth in otherwise hostile environments, and the FFA biosynthesis pathway of <EM>Synechococcus</EM> sp. PCC 7002 could be optimized for the production of molecules that serve as precursors to lipid biofuels.<SPAN class="return"><A href="#overview"> Back</A></SPAN></P></SECTION><SECTION class="content__section__alt"><H2 id="rationale">Rationale for Project</H2><H3>Rationale for Project</H3><P>Numerous genetic manipulation techniques have been developed for prokaryotes in the past decade, including zinc-finger nucleases, TALENs, SCALEs, CRISPR/Cas systems, and MAGE (Gaj et al. 2013, Lynch et al. 2007, Cong et al. 2013, Wang et al. 2009). However, these techniques have so far only been effective in a small handful of organisms beyond the model prokaryote <EM>Eschrichia coli</EM>. Despite their environmental significance and potential as producers of industrially-relevant small molecules, PCC7002, RT-CIAT, and many other non-model microbes lag far behind <EM>E. coli</EM> in terms of genome engineering technologies (Ramey et al. 2015). We reason that the development of a framework for the implementation of the most exciting genetic manipulation technologies into non-model organisms is necessary in order to maximize the positive potential which synthetic biology may have on industry and the environment.<SPAN class="return"><A href="#overview"> Back</A></SPAN></P></SECTION><SECTION class="content__section"><H2 id="mage">Multiplex Automated Genome Engineering (MAGE) in <EM>Escherichia coli</EM></H2><H3>Mutate. Grow. Screen. Repeat.</H3><P>Multiplex automated genome engineering (MAGE) is an iterative, oligonucleotide-mediated genetic manipulation developed by the George Church lab at Harvard Medical School in 2009 (Wang et al. 2009). The technique relies on the introduction of degenerate (randomized) ssDNA oligos into a bacterial cell. The oligos are designed with 30-45 bp homology arms that are complimentary to the target sequence in the cell's genome up- and downstream of the degenerate sequence. Phage homologous recombination proteins <EM>Gamma, Exo, </EM>and<EM> Beta</EM> (Lambda-Red cassette) are induced, and allow the oligo to anneal to the lagging target strand as the cell's chromosome separates into single strands for replication (Gallagher et al. 2014). Desirable genotypes are then selected based upon a phenotypic change in cell populations. MAGE can be used to optimize biosynthetic pathways which are then selected for based on a desirable phenotype.</P><P>MAGE can be used to create insertions and mismatches of up to 12 bp per cycle, and deletions of up to 1 kb per cycle (Gallagher et al. 2014). Each cycle, which involves introducing degenerate oligos into cells via electroporation, inducing the Lambda-Red cassette, and growing transformed cell populations, takes approximately 2.5 hours; thus, a highly diverse population of cells resulting from multiple MAGE cycles can be created in a matter of days. Since mutations created by MAGE rely upon DNA mismatches in the chromosome to go unnoticed, <EM>mutS</EM>-deficient <EM>E. coli </EM>strains are used when executing MAGE cycles. <EM>mutS</EM> is a highly-conserved gene in the DNA mismatch repair pathway whose associated protein is responsible for identifying and marking single-base mismatches after replication (Culligan et al. 2000). ∆<EM>mutS</EM> cell populations demonstrated increased mutagenesis efficiency per MAGE cycle; thus, creating cell populations deficient in the gene is an important aspect of porting MAGE technology into non-model organisms.<SPAN class="return"><A href="#overview"> Back</A></SPAN></P></SECTION><SECTION class="content__section__alt"><H2 id="crispr">CRISPR-Cas9 Systems</H2><H3>The search-and-modify tool of genome engineering</H3><P>In the past several years, CRISPR-Cas systems have emerged as an extremely powerful gene perturbation and genome editing technique. Based upon the RNA-guided restriction endonuclease Cas9 from the microbial adaptive immunity system CRISPR (clustered regularly interspaced short palindromic repeats), these systems allow virtually any sequence within a cell's genome to be targeted (Hsu et al. 2014). The Cas9 protein relies on a 20-bp guide RNA (gRNA) sequence to base-pair directly with a DNA target. Once this occurs, Cas9 creates a double-stranded break (DSB) upstream of a sequence referred to as a protospacer-adjacent motif (PAM). The PAM sequence is unique for every organism which possesses a CRISPR system.</P><P>In a native CRISPR system, the gRNA is typically a sequence derived from a viral genome which is used to cleave foreign DNA if the cell becomes re-infected with the same virus. The CRISPR-Cas9 system's power as a genetic editing tool arises when the gRNA is used to target endogenous DNA. The Cas9 protein from the bacterium <EM>Streptococcus pyogenes</EM> is typically used, since its PAM sequence of NGG (where N = any nucleotide) allows the protein to create DSBs at a large number of loci (for instance, the sequence NGG occurs every 8 bp on average within the human genome) (Cong et al. 2013, Hsu et al. 2014).</P><P>Cas9-mediated DNA manipulation procedures hold promise in basic research, therapeutics, metabolic pathway engineering, and beyond. The Cas9 protein, guided by specific gRNA, can be coupled with a fluorescent reporter protein to visualize genomic structure in real-time. Cas9 under a light- or small molecule-inducible promoter, along with synthesized libraries of gRNA, can be transfected into organisms and induced to facilitate multiplex, genome-wide editing (Mali et al. 2013). The DSBs created by Cas9 can be repaired by homologous recombination-based (HR) methods or more error-prone non-homologous end joining (NHEJ), allowing for precise gene editing if HR is favored or nonspecific indel mutations if NHEJ is favored (Hsu et al. 2013).<SPAN class="return"><A href="#overview"> Back</A></SPAN></P></SECTION><SECTION class="content__section"><H2 id="to1">Target Organism: <EM>Synechococcus</EM> sp. PCC 7002 </H2><H3>A fast-growing cyanobacterium for lipid biofuel production</H3><P><EM>Synechococcus sp</EM>. PCC 7002 is a marine cyanobacterium capable of rapid growth in a wide variety of environmental conditions (Song et al. 2015). The bacterium was first isolated from the waters off Magueyes Island, in southwestern Puerto Rico, in 1962 (Ludwig and Bryant 2012). The doubling time of PCC7002 in optimized, CO<SUB>2</SUB>-enriched conditions is under 3 hours, making the organism an ideal model for photosynthetic prokaryotes. PCC7002 is also a prime candidate for genetic modification: The organism's genome is fully sequenced (NCBI Taxonomy ID: 32049), its metabolic pathways are well-characterized (Hamilton and Reed 2012), and a system for protein overexpression has been developed for PCC7002 (Xu et al. 2011). Since it is native to marine environments and grows readily in seawater, the use of PCC7002 as a large-scale producer of small molecules would not place a burden on increasingly limited freshwater sources (Ruffing 2014). PCC7002 is naturally competent and readily undergoes homologous recombination with linear DNA fragments (Widger et al. 1998).</P><P>Despite its favorable characteristics for basic research and environmental applications, PCC7002s capabilities as a chassis for genetic modification have not been fully realized (Ramey et al. 2015). The organism possesses no native CRISPR systems, making it an ideal candidate to receive a foreign CRISPR/Cas system (such as that of <EM>S. pyogenes</EM>) for genomic editing (Cong et al. 2013). Its tendency towards natural recombination suggests that a high-efficiency MAGE protocol could be developed for PCC7002 with relative ease. We sought to address the gap in genetic manipulation technologies available for PCC7002 by testing and implementing our framework on the organism.<SPAN class="return"><A href="#overview"> Back</A></SPAN></P></SECTION><SECTION class="content__section__alt"><H2 id="to2">Target Organism: <EM>Rhizobium tropici </EM>  CIAT 899</H2><H3>A broadly symbiotic, nitrogen-fixing bacterium</H3><P><EM>Rhizobium tropici</EM> CIAT 899 is a promiscuous α-proteobacterium first isolated from a common-bean nodule in Colombia (Martinez-Romero et al. 1991). As a Rhizobium species, CIAT 899 can fix atmospheric nitrogen into a form that can be absorbed by plants, thus reintroduce nitrogen as an essential nutrient into the soil (New Mexico State University 2005). Rhizobium must nodulate plant roots in order to fix nitrogen, and although some Rhizobium species can only nodulate a limited number of legume species, CIAT 899 can establish nitrogen-fixing symbioses with a broad range of legume hosts (Hungria et al. 2000). CIAT 899 is more environmentally adaptable when compared to other Rhizobium species due to its tolerance of stressful conditions such as acidic soils and high temperatures (Graham et al. 1994). It has also shown resistance to various antibiotics, pesticides and fungicides (Bernal et al. 2004), all of which are properties that have led to CIAT 899's commercial use in inoculating the common bean <EM>Phaseolus vulgaris</EM> in South America and Africa (Ormeno-Orrillo et al. 2012). Because nitrogen-fixing legumes reduce reliance on synthetic fertilizers that are costly and ecologically damaging, scientists have recognized the role of Rhizobia in agricultural sustainability and ecological preservation (Balkan 2007).</P><P><EM>R. tropici</EM> CIAT 899 is an ideal candidate for MAGE because it has a reasonably fast doubling time of &lt; 6 hr (Morón et al. 2005) and its genome is fully sequenced (NCBI Taxonomy ID: 698761). CIAT 899 can produce a wide variety of Nod factor structures, which influence the range of symbiotic hosts (Morón et al. 2005). Although it is already used commercially to reintroduce soil nitrogen and has been very successful in increasing crop yields in some experimental plots, nitrogen-fixing legumes on average do not fix enough nitrogen to appreciably increase crop yield (Gilbert, 2012). There is potential for MAGE to further diversify the range of hosts that CIAT 899 require to fix nitrogen and increase the output of nitrogen from nitrogen-fixing symbioses.<SPAN class="return"><A href="#overview"> Back</A></SPAN></P></SECTION><SECTION class="content__section"><H2 id="outcomes">Potential Outcomes</H2><H3>Genetic Manipulation Technologies in PCC7002 and RT-CIAT</H3><P>We chose PCC7002 and RT-CIAT as organisms for developing our framework due to their high potential for making a positive impact in carbon-neutral and ecological industries. PCC7002 could be engineered to produce and tolerate higher concentrations of free fatty acids (FFAs), which could be harvested and developed into lipid biofuels. MAGE is an ideal technology for modifying PCC7002's FFA biosynthesis pathway due to its ability to produce large numbers of genetic variants at targeted loci. The nitrogen-fixation mechanisms of RT-CIAT could be optimized, reducing the need for artificial, nitrate-based fertilizers in plant crops.<SPAN class="return"><A href="#overview"> Back</A></SPAN></P></SECTION><SECTION class="content__section__alt"><H2 id="references"><A href="#" data-reveal-id="referencesList">References List</A></H2></SECTION><DIV id="referencesList" data-reveal="" aria-labelledby="iGEM Modal" aria-hidden="true" role="dialog" class="reveal-modal grayModal"><H2 class="modal__title">References</H2><DIV class="references"><P>Balkin, J.M. &quot;How New Genetic Technologies Will Transform Roe v. Wade&quot;, 56 Emory L. Rev. 843 (2007).</P><P>Cong L, Ann Ran F et al. &quot;Multiplex Genome Engineering Using CRISPR/Cas Systems.&quot; <I>Science</I><B>2013</B> 339 819-823.</P><P>Culligan, K., Meyer-Gauen, G., Lyons-Weiler, J. &amp; Hays, J. Evolutionary origin, diversification and specialization of eukaryotic MutS homolog mismatch repair proteins. <I>Nucleic Acids Res.</I><B>28,</B> 463-471 (2000).</P><P>Gaj, T., Gersbach, C. &amp; III, C. ZFN, TALEN, and CRISPR/Cas-based methods for genome engineering. <I>Trends in Biotechnology</I><B>31,</B> (2013).</P><P>Gallagher RR, Li Z, Lewis AO, and Issacs FJ. &quot;Rapid editing and evolution of bacterial genomes using libraries of synthetic DNA.&quot; <I>Nature Protocols</I><B>2014</B> 9(10) 2301-2316.</P><P>Gilbert, Natasha. &quot;African Agriculture: Dirt Poor.&quot; <I>Nature</I> 483.7391 (2012): 525-27. Web.</P><P>Graham PH, Draeger KJ, Ferrey ML, Conroy MJ, Hammer BE, Martinez E, Aarons SR, Quinto C. Acid pH tolerance in strains of Rhizobium and Bradyrhizobium, and initial studies on the basis for acid tolerance of Rhizobium tropici UMR1899. Can J Microbiol/Rev Can Microbiol.1994;40:198-207. doi: 10.1139/m94-033</P><P>Hamilton JJ and Reed JA. &quot;Identification of Functional Differences in Metabolic Networks Using Comparative Genomics and Constraint-Based Models.&quot; <I>PLOS One</I><B>2012</B> 7(4) 1-19.</P><P>Hsu, P., Lander, E. &amp; Zhang, F. Development and Applications of CRISPR-Cas9 for Genome Engineering. <I>Cell</I><B>157,</B> (2014).</P><P>Hungria M, Andrade DD, Chueire LMD, Probanza A, Guttierrez-Manero FJ, Megias M. Isolation and characterization of new efficient and competitive bean (Phaseolus vulgaris L.) rhizobia from Brazil. Soil Biol Biochem.2000;32:1515-1528. doi: 10.1016/S0038-0717(00)00063-8.</P><P>Ludwig M and Bryant DA.  &quot;<I>Synechococcus</I> sp. Strain PCC 7002 Transcriptome: Acclimation to Temperature, Salinity, Oxidative Stress, and Mixotrophic Growth Conditions.&quot; <I>Front. in Microbiology</I><B>2012</B> 3(324) 1-14.</P><P>Lynch, M. D., Warnecke, T. &amp; Gill, R. T. SCALEs: multiscale analysis of library enrichment. <I>Nat. Methods</I><B>4,</B> 87-93 (2007).</P><P>Mali P, Yang L et al. &quot;RNA-Guided Human Genome Engineering via Cas9.&quot; <I>Science</I><B>2013</B> 339, 823-826.</P><P>Martinez-Romero E, Segovia L et al. &quot;Rhizobium tropici, a Novel Species Nodulating <I>Phaseolus vulgaris</I>, L. Beans and <I>Leucaena</I> sp. Trees.&quot; <I>Int. J. of System. Biotech.</I><B>1991</B> 41(3), 417-426.</P><P>Martínez-Romero E, Segovia L, Mercante FM, Franco AA, Graham P, Pardo MA. Rhizobium tropici, a novel species nodulating Phaseolus vulgaris L. beans and Leucaena sp. trees. Int J Syst Evol Microbiol. 1991;41:417-426.</P><P>Moron B, Soria-Diaz ME, Ault J, Verroios G, Noreen S, Rodriguez-Navarro DN, Gil-Serrano A, Thomas-Oates J, Megias M, Sousa C. Low pH changes the profile of nodulation factors produced by Rhizobium tropici CIAT899. Chem Biol. 2005;12:1029-1040. doi: 10.1016/j.chembiol.2005.06.014.</P><P>&quot;Nitrogen Fixation by Legumes.&quot; <I>Nitrogen Fixation by Legumes</I>. New Mexico State University, n.d. Web. 02 July 2015.</P><P>Ormeño-Orrillo, Ernesto et al. &quot;Genomic Basis of Broad Host Range and Environmental Adaptability of <I>Rhizobium Tropici</I> CIAT 899 and <I>Rhizobium</I> Sp. PRF 81 Which Are Used in Inoculants for Common Bean (<I>Phaseolus Vulgaris</I>L.).&quot; <I>BMC Genomics</I> 13 (2012): 735. <I>PMC</I>. Web. 28 Aug. 2015.</P><P>Ramey, C., Barón-Sola, Á., Aucoin, H. &amp; Boyle, N. Genome Engineering in Cyanobacteria: Where We Are and Where We Need To Go. <I>ACS Synth. Biol.</I>150605142013007 (2015). doi:10.1021/acssynbio.5b00043</P><P>Ruffing, A. &quot;Improved Free Fatty Acid Production in Cyanobacteria with Synechococcus sp. PCC 7002 as Host.&quot; <I>Front. Bioeng. Biotechnol.</I><B>2014</B> 2.</P><P>Song HS, McClure RS, Beliaev AS et al. &quot;Integrated <I>in silico</I> Analyses of Regulatory and Metabolic Networks of <I>Synechococcus sp.</I> PCC 7002 Reveal Relationships Between Gene Centrality and Essentiality.&quot;  <I>Life</I><B>2015</B> 5(2) 1127-1140.</P><P>Wang, H. <I>et al.</I> Programming cells by multiplex genome engineering and accelerated evolution. <I>Nature</I><B>460,</B> 894-898 (2009).</P><P>Widger WI, Chen X, and Samartzidou H. &quot;<I>Synechococcus</I> PCC 7002.&quot; In <I>Bacterial Genomes: Physical Structure and Analysis</I>. De Bruijn FJ, Lupski JR, GM Weinstock, Eds. Springer US: <B>1998</B>, 763-770.</P><P>Xu Y, Alvey RM, Byrne PO, Graham JE, Shen G, and Bryant DA. &quot;Expression of genes in cyanobacteria: adaptation of endogenous plasmids as platforms for high-level gene expression in <I>Synechococcus sp.</I> PCC 7002.&quot; <I>Methods Mol. Biol.</I><B>2011</B> 684, 273-293.</P></DIV><A aria-label="Close" class="close-reveal-modal">x</A></DIV><DIV class="igem__footer"><DIV class="foot__contain"><DIV class="row foot__row"><DIV class="small-7 columns foot__text"><H2>Yale iGem 2015</H2><P>Main Campus:</P><P>Yale Department of Molecular, Cellular &amp; Developmental Biology</P><P>Attn: Farren Isaacs/iGEM</P><P>219 Prospect St</P><P>PO Box 208103</P><P>New Haven, CT 06520</P><P>Tel. +1(203) 432-3783</P><P>E-mail: igem.yale@gmail.com</P></DIV></DIV><DIV class="row"><P class="text-center">© Yale iGEM 2015</P></DIV></DIV></DIV>&lt;/html&gt;</DIV></DIV></DIV></DIV></BODY></HTML>