<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Yale_results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Yale/results</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P>&lt;!DOCTYPE html&gt;<TITLE>Yale iGem 2015: Results</TITLE><NAV class="nav-contain"><UL class="nav"><LI><A href="https://2015.igem.org/Team:Yale" class="logoMark"><SPAN>Non</SPAN><SPAN>Model</SPAN></A></LI><LI class="submenu"><A href="https://2015.igem.org/Team:Yale/project">Project</A></LI><LI><A href="https://2015.igem.org/Team:Yale/project#overview" alt="Overview">Overview</A></LI><LI><A href="https://2015.igem.org/Team:Yale/methods" alt="Methods">Methods</A></LI><LI><A href="https://2015.igem.org/Team:Yale/results" alt="Results">Results</A></LI><LI><A href="https://2015.igem.org/Team:Yale/modeling" alt="Modeling">Modeling</A></LI><LI><A href="https://2015.igem.org/Team:Yale/parts" alt="Parts">Parts</A></LI></UL><LI class="submenu"><A href="notebook">Notebook</A></LI><LI><A href="https://2015.igem.org/Team:Yale/notebook" alt="Weekly">Weekly</A></LI><LI><A href="https://static.igem.org/mediawiki/2015/f/fb/Yale_iGEM_Project_Summary_2015.pdf" alt="PDF Summary">PDF Summary</A></LI><LI class="submenu"><A href="collaborations">Collaborations</A></LI><LI class="submenu"><A href="practices">Human Practices</A></LI><LI><A href="https://2015.igem.org/Team:Yale/practices#video" alt="Documentary">Documentary</A></LI><LI><A href="https://2015.igem.org/Team:Yale/practices#ssri" alt="SSRI">SSRI</A></LI><LI><A href="https://2015.igem.org/Team:Yale/practices#lgbtq" alt="LGBTQ Survey">LGBTQ Survey</A></LI><LI class="submenu"><A href="team">Team</A></LI><LI><A href="https://2015.igem.org/Team:Yale/team" alt="People">People</A></LI><LI><A href="https://2015.igem.org/Team:Yale/Attributions" alt="Acknowledgements">Attributions</A></LI><LI class="submenu"><A href="standards">Standards</A></LI><LI><A href="https://2015.igem.org/Team:Yale/standards#gold" alt="Gold">Gold</A></LI><LI><A href="https://2015.igem.org/Team:Yale/standards#silver" alt="Silver">Silver</A></LI><LI><A href="https://2015.igem.org/Team:Yale/standards#bronze" alt="Bronze">Bronze</A></LI></NAV></P><H2 class="page__head">Developing a Framework for the Genetic Manipulation of Non-Model and Environmentally Significant Microbes</H2><DIV class="row overview__blocks"><DIV class="small-12 switch-border"><H2><A href="project#overview" alt="Overview">Overview</A></H2><H2><A href="methods" alt="Methods">Methods</A></H2><H2><A href="results" alt="Results">Results</A></H2><H2><A href="modeling" alt="Modeling">Modeling</A></H2><H2><A href="parts" alt="Parts">Parts</A></H2></DIV></DIV><SECTION class="content__section"><H2 id="overview">Project Results: Overview</H2><H3>Click on a component to learn more about our method.</H3></SECTION><DIV id="clickmap" class="contain__overview"><DIV class="left__block"><H2>Technique-Independent Procedures</H2></DIV><DIV class="diverge-arrows"><SPAN class="bigblock">»</SPAN><SPAN class="bigblock">»</SPAN></DIV><DIV class="right__block"><DIV class="upper" style="padding: 0 0 15px 0; background-color:#a5a5a5"><H2>MAGE Implementation Pathway</H2></DIV><DIV class="lower" style="background-color:#a5a5a5"><H2>CRISPR-Cas9 Implementation Pathway</H2></DIV></DIV></DIV><SECTION class="content__section__alt"><H2 id="growth" class="overview__head">Growth</H2><DIV class="row over__blocks"><DIV class="small-6 columns cyano_block"><H2>Cyanobacterium</H2><P> The growth rate of PCC7002 was found to be highly sensitive to growth conditions, particularly light saturation and CO2 content of air. PCC7002 samples grown in CO2-supplemented conditions grew significantly faster (as measured by OD 730 nm) than samples grown in atmospheric conditions, and samples supplemented with 10mM glycerol grew to a much higher saturation than non-supplemented samples in the same period of time (glycerol supplement = top graph; no glycerol = middle graph). We also measured the ratio of chlorophyll content (OD 650 nm) to cell saturation (OD 730 nm) for cell populations grown in glycerol-supplemented versus nonsupplemented conditions. The glycerol was also found to not shunt cells away from photosynthetic pathways, as populations grown in glycerol had the same chlorophyll content per cell as nonsupplemented populations (bottom graph, PCC7002 grown in A+ medium is shown in green).
    </P></DIV><DIV class="small-6 columns sino_block"><H2 class="sino__head">Sinorhizobium</H2><P>Rhizobium tropici CIAT 899 and our Sinorhizobium meliloti 1021 strains were incubated at 30C. We conducted growth assays on each strain we worked with in order to determine mid-log growth phases and doubling times; this was important in optimizing our transformation efficiency.</P><DIV class="launch__modal"><A href="https://static.igem.org/mediawiki/2015/8/87/Characterization_of_Rhizobium_Growth.pdf" data-reveal-id="growthTwo" class="custom__button">Further Reading</A></DIV></DIV></DIV></SECTION><SECTION class="content__section__alt overview__section"><H2 id="transform" class="overview__head">Transformation</H2><DIV class="row over__blocks"><DIV class="small-6 columns cyano_block"><H2>Cyanobacterium</H2><P> PCC7002 cell populations transformed with linear kanR genes flanked by 1000 bp homology arms up- and downstream of the gene demonstrated resistance to 200 µh/ml of kanamycin after a 24 hour incubation period and recovery in A+ medium. The six leftmost tubes in the picture below contain experimental samples which were recovered and inoculated in kanamycin-containing A+ medium; the four rightmost samples are all wild-type controls in kanamycin medium.
    </P></DIV><DIV class="small-6 columns sino_block"><H2 class="sino__head">Sinorhizobium</H2><P>We pursued two types of transformation into rhizobia: conjugation and electroporation. While our conjugation efficiencies of the desired pKT230 plasmid into E. coli were very high (~90%), our conjugation was inconclusive in rhizobium because many of our strains had several innate antibiotic resistances and thus there was growth on the negative controls. However, once we optimized our electroporation protocols by adjusting centrifugation speeds, antibiotic concentrations, and electroporation voltages and pulses, we were able to successfully transform our plasmids of interest into rhizobia.</P><DIV class="launch__modal"><A href="#" data-reveal-id="transformTwo" class="custom__button">Further Reading</A></DIV></DIV></DIV></SECTION><SECTION class="content__section__alt overview__section"><H2 id="select" class="overview__head">Selection</H2><DIV class="row over__blocks"><DIV class="small-6 columns cyano_block"><H2>Cyanobacterium</H2><P> PCC7002 was found to be susceptible to all antibiotics tested around standard E. coli concentrations, though light-sensitive antibiotics (tetracycline and rifampicin) were less effective due to degradation in light-saturated growth chambers. Click on the links below to see the growth curves for PCC7002 samples grown in different concentrations of tested antibiotics.
      </P><P class="text-center"><A href="https://static.igem.org/mediawiki/2015/9/9c/CARB.jpeg" class="file__link">Carbenicillin</A></P><P class="text-center"><A href="https://static.igem.org/mediawiki/2015/c/c2/KAN.jpeg" class="file__link">Kanamycin</A></P><P class="text-center"><A href="https://static.igem.org/mediawiki/2015/5/50/RIF.jpeg" class="file__link">Rifampicin</A></P><P class="text-center"><A href="https://static.igem.org/mediawiki/2015/9/9f/SPEC.jpeg" class="file__link">Spectinomycin</A></P><P class="text-center"><A href="https://static.igem.org/mediawiki/2015/2/2a/STREP.jpeg" class="file__link">Streptomycin</A></P><P class="text-center"><A href="https://static.igem.org/mediawiki/2015/7/7d/TET.jpeg" class="file__link">Tetracycline</A></P><DIV class="launch__modal"><A href="#" data-reveal-id="selectOne" class="custom__button">Further Reading</A></DIV></DIV><DIV class="small-6 columns sino_block"><H2 class="sino__head">Sinorhizobium</H2><P>Our main selection for transformation was antibiotic resistance. Before conducting electroporations, it was important to establish the innate resistances in our rhizobial strains by doing antibiotic resistance assays, both on plates and in liquid cultures. We observed that resistances changed depending on the growing media (Tryptic Soy Broth vs. LB) and conditions (liquid vs. plate). Our selection for the incorporation of LIC plasmids was sucrose.</P><DIV class="launch__modal"><A href="https://static.igem.org/mediawiki/2015/d/d6/Antibiotic_Resistance_Assays.pdf" data-reveal-id="selectTwo" class="custom__button">Further Reading</A></DIV></DIV></DIV></SECTION><SECTION class="content__section"><H2 id="mage1">mutS Gene Knockout</H2><P> We successfully obtained <I>mutS</I> knockout genotypes of PCC7002 by transforming linear fragments for homologous recombination. Presumably due to the polyploidy of PCC7002 cells, however, we did not obtain homozygous <I>∆mutS</I> mutants; doing so will require several more rounds of growth and plating for single colonies. <SPAN class="return"><A href="#top"> Back to Top</A></SPAN></P></SECTION><SECTION class="content__section"><H2 id="mage2">DNA Assemblies</H2><DIV class="row content__dark"><DIV class="small-4 columns readable"><H4>Gibson Assembly</H4><P>We successfully PCR amplified promoters to express our recombinases in rhizobia as well as assembled promoter-citrine and promoter-recombinase constructs. Our band sizes were verified through gel electrophoresis.</P><DIV class="launch__modal"><A href="https://static.igem.org/mediawiki/2015/7/7c/Amplification_of_Promoters_to_Drive_Recombinase_Expression.pdf" data-reveal-id="growthTwo" class="custom__button">Further Reading</A></DIV></DIV></DIV></SECTION><SECTION class="content__section"><H2 id="mage3">Testing DNA Constructs</H2><DIV class="row content__dark"><DIV class="small-6 columns readable"><H4>Testing Promoters</H4><P>We conducted a fluorescence test in order to determine which promoters were inducible in rhizobium and what level of expression was obtainable. Before testing the promoter-citrine constructs in rhizobium, we first tested them in E. coli. In E. coli, we were able to obtain fairly high fluorescence readings for the lac and tac promoters as well as the constitutive Anderson ones. MelA and bacA are promoters specific to rhizobia, so we did not expect those to work well in E. coli. When we tested the promoters in rhizobia, we found that out of the inducible promoters, tac had the greatest expression and out of the constitutive promoters, Anderson Medium resulted in the most fluorescence. The incorporation of the promoter-citrine constructs in our organisms of interest was confirmed by sequencing.</P><DIV class="launch__modal"><A href="https://static.igem.org/mediawiki/2015/2/22/Expression_of_Promoter-Cit_Constructs_in_Rhizo.pdf" data-reveal-id="growthTwo" class="custom__button">Further Reading</A></DIV></DIV><DIV class="small-6 columns readable"><H4>Testing Beta-Homologs</H4><P>Using search tools that compare nucleotide sequences to organisms and viruses in NCBI and UniProtKB databases, nine lambda beta and SPP1 35 homologues were identified that exist in Rhizobium species. Further reading: https://static.igem.org/mediawiki/2015/f/f8/Search_for_Beta_Homologue_Recombinases.pdf</P></DIV></DIV></SECTION><SECTION class="content__section"><DIV id="growthOne" data-reveal="" aria-labelledby="iGEM Modal" aria-hidden="true" role="dialog" class="reveal-modal greenModal"><H2 class="modal__title">Cyanothingy Growth Results</H2><P>These are the tales of Steven Spinach, one brave molecule. </P><P> Lorem ipsum dolor sit amet, consectetur adipisicing elit. Necessitatibus enim non iste molestias, facere repellendus assumenda, ipsum molestiae praesentium magni fugiat sit? Reprehenderit accusantium ratione quia cumque recusandae, tempora temporibus?</P><P>Biofilm formation on surfaces is an issue in the medical field, naval industry, and other areas. We developed an anti-fouling peptide with two modular components: a mussel adhesion protein (MAP) anchor and LL-37, an antimicrobial peptide. MAPs can selectively attach to metal and organic surfaces via L-3,5-dihydroxyphenylalanine (L-DOPA), a nonstandard amino acid that was incorporated using a genetically recoded organism (GRO). Because this peptide is toxic to the GRO in which it is produced, we designed a better controlled inducible system that limits basal expression. This was achieved through a novel T7 riboregulation system that controls expression at both the transcriptional and translational levels.</P><P>Biofilm formation on surfaces is an issue in the medical field, naval industry, and other areas. We developed an anti-fouling peptide with two modular components: a mussel adhesion protein (MAP) anchor and LL-37, an antimicrobial peptide. MAPs can selectively attach to metal and organic surfaces via L-3,5-dihydroxyphenylalanine (L-DOPA), a nonstandard amino acid that was incorporated using a genetically recoded organism (GRO). Because this peptide is toxic to the GRO in which it is produced, we designed a better controlled inducible system that limits basal expression. This was achieved through a novel T7 riboregulation system that controls expression at both the transcriptional and translational levels.</P><P>Biofilm formation on surfaces is an issue in the medical field, naval industry, and other areas. We developed an anti-fouling peptide with two modular components: a mussel adhesion protein (MAP) anchor and LL-37, an antimicrobial peptide. MAPs can selectively attach to metal and organic surfaces via L-3,5-dihydroxyphenylalanine (L-DOPA), a nonstandard amino acid that was incorporated using a genetically recoded organism (GRO). Because this peptide is toxic to the GRO in which it is produced, we designed a better controlled inducible system that limits basal expression. This was achieved through a novel T7 riboregulation system that controls expression at both the transcriptional and translational levels.</P><A href="#">Download a PDF of our Protocol!</A><A aria-label="Close" class="close-reveal-modal">x</A></DIV><DIV id="growthTwo" data-reveal="" aria-labelledby="iGEM Modal" aria-hidden="true" role="dialog" class="reveal-modal redModal"><H2 class="modal__title">Sinorhizobium Growth Results</H2><P>Many a young child went swimming today and learned about the sea</P><P>Biofilm formation on surfaces is an issue in the medical field, naval industry, and other areas. We developed an anti-fouling peptide with two modular components: a mussel adhesion protein (MAP) anchor and LL-37, an antimicrobial peptide. MAPs can selectively attach to metal and organic surfaces via L-3,5-dihydroxyphenylalanine (L-DOPA), a nonstandard amino acid that was incorporated using a genetically recoded organism (GRO). Because this peptide is toxic to the GRO in which it is produced, we designed a better controlled inducible system that limits basal expression. This was achieved through a novel T7 riboregulation system that controls expression at both the transcriptional and translational levels.</P><P>Biofilm formation on surfaces is an issue in the medical field, naval industry, and other areas. We developed an anti-fouling peptide with two modular components: a mussel adhesion protein (MAP) anchor and LL-37, an antimicrobial peptide. MAPs can selectively attach to metal and organic surfaces via L-3,5-dihydroxyphenylalanine (L-DOPA), a nonstandard amino acid that was incorporated using a genetically recoded organism (GRO). Because this peptide is toxic to the GRO in which it is produced, we designed a better controlled inducible system that limits basal expression. This was achieved through a novel T7 riboregulation system that controls expression at both the transcriptional and translational levels.</P><P>Biofilm formation on surfaces is an issue in the medical field, naval industry, and other areas. We developed an anti-fouling peptide with two modular components: a mussel adhesion protein (MAP) anchor and LL-37, an antimicrobial peptide. MAPs can selectively attach to metal and organic surfaces via L-3,5-dihydroxyphenylalanine (L-DOPA), a nonstandard amino acid that was incorporated using a genetically recoded organism (GRO). Because this peptide is toxic to the GRO in which it is produced, we designed a better controlled inducible system that limits basal expression. This was achieved through a novel T7 riboregulation system that controls expression at both the transcriptional and translational levels.</P><A aria-label="Close" class="close-reveal-modal">x</A></DIV><DIV id="growthLessons" data-reveal="" aria-labelledby="iGEM Modal" aria-hidden="true" role="dialog" class="reveal-modal grayModal"><H2 class="modal__title">Lessons Learned About the Framework</H2><P>Many a young child went swimming today and learned about the sea</P><P>Biofilm formation on surfaces is an issue in the medical field, naval industry, and other areas. We developed an anti-fouling peptide with two modular components: a mussel adhesion protein (MAP) anchor and LL-37, an antimicrobial peptide. MAPs can selectively attach to metal and organic surfaces via L-3,5-dihydroxyphenylalanine (L-DOPA), a nonstandard amino acid that was incorporated using a genetically recoded organism (GRO). Because this peptide is toxic to the GRO in which it is produced, we designed a better controlled inducible system that limits basal expression. This was achieved through a novel T7 riboregulation system that controls expression at both the transcriptional and translational levels.</P><P>Biofilm formation on surfaces is an issue in the medical field, naval industry, and other areas. We developed an anti-fouling peptide with two modular components: a mussel adhesion protein (MAP) anchor and LL-37, an antimicrobial peptide. MAPs can selectively attach to metal and organic surfaces via L-3,5-dihydroxyphenylalanine (L-DOPA), a nonstandard amino acid that was incorporated using a genetically recoded organism (GRO). Because this peptide is toxic to the GRO in which it is produced, we designed a better controlled inducible system that limits basal expression. This was achieved through a novel T7 riboregulation system that controls expression at both the transcriptional and translational levels.</P><P>Biofilm formation on surfaces is an issue in the medical field, naval industry, and other areas. We developed an anti-fouling peptide with two modular components: a mussel adhesion protein (MAP) anchor and LL-37, an antimicrobial peptide. MAPs can selectively attach to metal and organic surfaces via L-3,5-dihydroxyphenylalanine (L-DOPA), a nonstandard amino acid that was incorporated using a genetically recoded organism (GRO). Because this peptide is toxic to the GRO in which it is produced, we designed a better controlled inducible system that limits basal expression. This was achieved through a novel T7 riboregulation system that controls expression at both the transcriptional and translational levels.</P><A aria-label="Close" class="close-reveal-modal">x</A></DIV><DIV id="transformOne" data-reveal="" aria-labelledby="iGEM Modal" aria-hidden="true" role="dialog" class="reveal-modal greenModal"><H2 class="modal__title">Cyan Transforms Results</H2><P>These are the tales of Steven Spinach, one brave molecule.</P><P> Lorem ipsum dolor sit amet, consectetur adipisicing elit. Necessitatibus enim non iste molestias, facere repellendus assumenda, ipsum molestiae praesentium magni fugiat sit? Reprehenderit accusantium ratione quia cumque recusandae, tempora temporibus?</P><P>Biofilm formation on surfaces is an issue in the medical field, naval industry, and other areas. We developed an anti-fouling peptide with two modular components: a mussel adhesion protein (MAP) anchor and LL-37, an antimicrobial peptide. MAPs can selectively attach to metal and organic surfaces via L-3,5-dihydroxyphenylalanine (L-DOPA), a nonstandard amino acid that was incorporated using a genetically recoded organism (GRO). Because this peptide is toxic to the GRO in which it is produced, we designed a better controlled inducible system that limits basal expression. This was achieved through a novel T7 riboregulation system that controls expression at both the transcriptional and translational levels.</P><P>Biofilm formation on surfaces is an issue in the medical field, naval industry, and other areas. We developed an anti-fouling peptide with two modular components: a mussel adhesion protein (MAP) anchor and LL-37, an antimicrobial peptide. MAPs can selectively attach to metal and organic surfaces via L-3,5-dihydroxyphenylalanine (L-DOPA), a nonstandard amino acid that was incorporated using a genetically recoded organism (GRO). Because this peptide is toxic to the GRO in which it is produced, we designed a better controlled inducible system that limits basal expression. This was achieved through a novel T7 riboregulation system that controls expression at both the transcriptional and translational levels.</P><P>Biofilm formation on surfaces is an issue in the medical field, naval industry, and other areas. We developed an anti-fouling peptide with two modular components: a mussel adhesion protein (MAP) anchor and LL-37, an antimicrobial peptide. MAPs can selectively attach to metal and organic surfaces via L-3,5-dihydroxyphenylalanine (L-DOPA), a nonstandard amino acid that was incorporated using a genetically recoded organism (GRO). Because this peptide is toxic to the GRO in which it is produced, we designed a better controlled inducible system that limits basal expression. This was achieved through a novel T7 riboregulation system that controls expression at both the transcriptional and translational levels.</P><A href="#">Download a PDF of our Protocol!</A><A aria-label="Close" class="close-reveal-modal">x</A></DIV><DIV id="transformTwo" data-reveal="" aria-labelledby="iGEM Modal" aria-hidden="true" role="dialog" class="reveal-modal redModal"><H2 class="modal__title">Sinorhizobium Transforms Results</H2><P>Many a young child went swimming today and learned about the sea</P><P>Biofilm formation on surfaces is an issue in the medical field, naval industry, and other areas. We developed an anti-fouling peptide with two modular components: a mussel adhesion protein (MAP) anchor and LL-37, an antimicrobial peptide. MAPs can selectively attach to metal and organic surfaces via L-3,5-dihydroxyphenylalanine (L-DOPA), a nonstandard amino acid that was incorporated using a genetically recoded organism (GRO). Because this peptide is toxic to the GRO in which it is produced, we designed a better controlled inducible system that limits basal expression. This was achieved through a novel T7 riboregulation system that controls expression at both the transcriptional and translational levels.</P><P>Biofilm formation on surfaces is an issue in the medical field, naval industry, and other areas. We developed an anti-fouling peptide with two modular components: a mussel adhesion protein (MAP) anchor and LL-37, an antimicrobial peptide. MAPs can selectively attach to metal and organic surfaces via L-3,5-dihydroxyphenylalanine (L-DOPA), a nonstandard amino acid that was incorporated using a genetically recoded organism (GRO). Because this peptide is toxic to the GRO in which it is produced, we designed a better controlled inducible system that limits basal expression. This was achieved through a novel T7 riboregulation system that controls expression at both the transcriptional and translational levels.</P><P>Biofilm formation on surfaces is an issue in the medical field, naval industry, and other areas. We developed an anti-fouling peptide with two modular components: a mussel adhesion protein (MAP) anchor and LL-37, an antimicrobial peptide. MAPs can selectively attach to metal and organic surfaces via L-3,5-dihydroxyphenylalanine (L-DOPA), a nonstandard amino acid that was incorporated using a genetically recoded organism (GRO). Because this peptide is toxic to the GRO in which it is produced, we designed a better controlled inducible system that limits basal expression. This was achieved through a novel T7 riboregulation system that controls expression at both the transcriptional and translational levels.</P><A aria-label="Close" class="close-reveal-modal">x</A></DIV><DIV id="selectOne" data-reveal="" aria-labelledby="iGEM Modal" aria-hidden="true" role="dialog" class="reveal-modal greenModal"><H2 class="modal__title">Cyan Selects</H2><P>Biofilm formation on surfaces is an issue in the medical field, naval industry, and other areas. We developed an anti-fouling peptide with two modular components: a mussel adhesion protein (MAP) anchor and LL-37, an antimicrobial peptide. MAPs can selectively attach to metal and organic surfaces via L-3,5-dihydroxyphenylalanine (L-DOPA), a nonstandard amino acid that was incorporated using a genetically recoded organism (GRO). Because this peptide is toxic to the GRO in which it is produced, we designed a better controlled inducible system that limits basal expression. This was achieved through a novel T7 riboregulation system that controls expression at both the transcriptional and translational levels.</P><A href="#">Download a PDF of our Protocol!</A><A aria-label="Close" class="close-reveal-modal">x</A></DIV><DIV id="selectTwo" data-reveal="" aria-labelledby="iGEM Modal" aria-hidden="true" role="dialog" class="reveal-modal redModal"><H2 class="modal__title">Sinorhizobium Selects</H2><P>Biofilm formation on surfaces is an issue in the medical field, naval industry, and other areas. We developed an anti-fouling peptide with two modular components: a mussel adhesion protein (MAP) anchor and LL-37, an antimicrobial peptide. MAPs can selectively attach to metal and organic surfaces via L-3,5-dihydroxyphenylalanine (L-DOPA), a nonstandard amino acid that was incorporated using a genetically recoded organism (GRO). Because this peptide is toxic to the GRO in which it is produced, we designed a better controlled inducible system that limits basal expression. This was achieved through a novel T7 riboregulation system that controls expression at both the transcriptional and translational levels.</P><A href="#">Download a PDF of our Protocol!</A><A aria-label="Close" class="close-reveal-modal">x</A></DIV></SECTION><DIV class="igem__footer"><DIV class="foot__contain"><DIV class="row foot__row"><DIV class="small-7 columns foot__text"><H2>Yale iGem 2015</H2><P>Main Campus:</P><P>Yale Department of Molecular, Cellular &amp; Developmental Biology</P><P>Attn: Farren Isaacs/iGEM</P><P>219 Prospect St</P><P>PO Box 208103</P><P>New Haven, CT 06520</P><P>Tel. +1(203) 432-3783</P><P>E-mail: igem.yale@gmail.com</P></DIV></DIV><DIV class="row"><P class="text-center">© Yale iGEM 2015</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>