Team:Yale/project
<!DOCTYPE html>
Yale iGem 2015: Project Overview
NonModel
Project
Overview
Methods
Results
Modeling
Parts
Notebook
Weekly
PDF Summary
Collaborations
Human Practices
Documentary
SSRI
LGBTQ Survey
Team
People
Acknowledgements
Standards
Gold
Silver
Bronze
Developing a Framework for the Genetic Manipulation of Non-Model and Environmentally Significant Microbes
Overview
Methods
Results
Modeling
Parts
Project Overview
Let's make today's most important microbes easier to engineer
Overview Rationale Mage CRISPR
Synechococcus sp. PCC 7002 R. tropici CIAT 899 Potential Outcomes Improving a BioBrick
References
Overview
Our iGEM research project involves developing a framework for the implementation of genetic manipulation techniques'specifically, multiplex automated genome engineering (MAGE) and CRISPR-Cas9 systems into non-model, environmentally significant microbes. MAGE was developed as a rapid, high efficiency tool for increasing the genetic diversity of a cell population at targeted loci within the genome, and has so far been ported into the model organism Escherichia coli and a few other members of the family Enterobacteriaceae (unpublished data). CRISPR (clustered regularly interspaced short palindromic repeats)/Cas systems, based on prokaryotic adaptive immunity mechanisms, have emerged as a powerful cleavage-based genome editing technique (Cong et al. 2013).
The organisms we used to develop our framework are Rhizobium tropici CIAT and Synechococcus sp. PCC 7002. Rhizobium tropici CIAT 899 (hereafter RT-CIAT) is a nitrogen-fixing bacterium capable of forming root nodules with certain legume-producing Leucaena and Phaseolus trees (Martinez-Romero et al). Synechococcus sp. PCC 7002 (hereafter PCC7002) is a fast-growing marine cyanobacterium capable of photosynthesis and free fatty acid (FFA) production (Ruffing 2014).
We envision numerous potential applications for MAGE, CRISPR-Cas9, and other genetic manipulation techniques in these organisms; for example, the nitrogen fixation mechanisms in Rhizobium tropici CIAT could be modified to enable plant growth in otherwise hostile environments, and the FFA biosynthesis pathway of Synechococcus sp. PCC 7002 could be optimized for the production of molecules that serve as precursors to lipid biofuels. Back
Rationale for Project
Numerous genetic manipulation techniques have been developed for prokaryotes in the past decade, including zinc-finger nucleases, TALENs, SCALEs, CRISPR/Cas systems, and MAGE (Gaj et al. 2013, Lynch et al. 2007, Cong et al. 2013, Wang et al. 2009). However, these techniques have so far only been effective in a small handful of organisms beyond the model prokaryote Eschrichia coli. Despite their environmental significance and potential as producers of industrially-relevant small molecules, PCC7002, RT-CIAT, and many other non-model microbes lag far behind E. coli in terms of genome engineering technologies (Ramey et al. 2015). We reason that the development of a framework for the implementation of the most exciting genetic manipulation technologies into non-model organisms is necessary in order to maximize the positive potential which synthetic biology may have on industry and the environment. Back
Multiplex Automated Genome Engineering (MAGE) in Escherichia coli
Mutate. Grow. Screen. Repeat.
Multiplex automated genome engineering (MAGE) is an iterative, oligonucleotide-mediated genetic manipulation developed by the George Church lab at Harvard Medical School in 2009 (Wang et al. 2009). The technique relies on the introduction of degenerate (randomized) ssDNA oligos into a bacterial cell. The oligos are designed with 30-45 bp homology arms that are complimentary to the target sequence in the cell's genome up- and downstream of the degenerate sequence. Phage homologous recombination proteins Gamma, Exo, and Beta (Lambda-Red cassette) are induced, and allow the oligo to anneal to the lagging target strand as the cell's chromosome separates into single strands for replication (Gallagher et al. 2014). Desirable genotypes are then selected based upon a phenotypic change in cell populations. MAGE can be used to optimize biosynthetic pathways which are then selected for based on a desirable phenotype.
MAGE can be used to create insertions and mismatches of up to 12 bp per cycle, and deletions of up to 1 kb per cycle (Gallagher et al. 2014). Each cycle, which involves introducing degenerate oligos into cells via electroporation, inducing the Lambda-Red cassette, and growing transformed cell populations, takes approximately 2.5 hours; thus, a highly diverse population of cells resulting from multiple MAGE cycles can be created in a matter of days. Since mutations created by MAGE rely upon DNA mismatches in the chromosome to go unnoticed, mutS-deficient E. coli strains are used when executing MAGE cycles. mutS is a highly-conserved gene in the DNA mismatch repair pathway whose associated protein is responsible for identifying and marking single-base mismatches after replication (Culligan et al. 2000). ∆mutS cell populations demonstrated increased mutagenesis efficiency per MAGE cycle; thus, creating cell populations deficient in the gene is an important aspect of porting MAGE technology into non-model organisms. Back
CRISPR-Cas9 Systems
The search-and-modify tool of genome engineering
In the past several years, CRISPR-Cas systems have emerged as an extremely powerful gene perturbation and genome editing technique. Based upon the RNA-guided restriction endonuclease Cas9 from the microbial adaptive immunity system CRISPR (clustered regularly interspaced short palindromic repeats), these systems allow virtually any sequence within a cell's genome to be targeted (Hsu et al. 2014). The Cas9 protein relies on a 20-bp guide RNA (gRNA) sequence to base-pair directly with a DNA target. Once this occurs, Cas9 creates a double-stranded break (DSB) upstream of a sequence referred to as a protospacer-adjacent motif (PAM). The PAM sequence is unique for every organism which possesses a CRISPR system.
In a native CRISPR system, the gRNA is typically a sequence derived from a viral genome which is used to cleave foreign DNA if the cell becomes re-infected with the same virus. The CRISPR-Cas9 system's power as a genetic editing tool arises when the gRNA is used to target endogenous DNA. The Cas9 protein from the bacterium Streptococcus pyogenes is typically used, since its PAM sequence of NGG (where N = any nucleotide) allows the protein to create DSBs at a large number of loci (for instance, the sequence NGG occurs every 8 bp on average within the human genome) (Cong et al. 2013, Hsu et al. 2014).
Cas9-mediated DNA manipulation procedures hold promise in basic research, therapeutics, metabolic pathway engineering, and beyond. The Cas9 protein, guided by specific gRNA, can be coupled with a fluorescent reporter protein to visualize genomic structure in real-time. Cas9 under a light- or small molecule-inducible promoter, along with synthesized libraries of gRNA, can be transfected into organisms and induced to facilitate multiplex, genome-wide editing (Mali et al. 2013). The DSBs created by Cas9 can be repaired by homologous recombination-based (HR) methods or more error-prone non-homologous end joining (NHEJ), allowing for precise gene editing if HR is favored or nonspecific indel mutations if NHEJ is favored (Hsu et al. 2013). Back
Target Organism: Synechococcus sp. PCC 7002
A fast-growing cyanobacterium for lipid biofuel production
Synechococcus sp. PCC 7002 is a marine cyanobacterium capable of rapid growth in a wide variety of environmental conditions (Song et al. 2015). The bacterium was first isolated from the waters off Magueyes Island, in southwestern Puerto Rico, in 1962 (Ludwig and Bryant 2012). The doubling time of PCC7002 in optimized, CO2-enriched conditions is under 3 hours, making the organism an ideal model for photosynthetic prokaryotes. PCC7002 is also a prime candidate for genetic modification: The organism's genome is fully sequenced (NCBI Taxonomy ID: 32049), its metabolic pathways are well-characterized (Hamilton and Reed 2012), and a system for protein overexpression has been developed for PCC7002 (Xu et al. 2011). Since it is native to marine environments and grows readily in seawater, the use of PCC7002 as a large-scale producer of small molecules would not place a burden on increasingly limited freshwater sources (Ruffing 2014). PCC7002 is naturally competent and readily undergoes homologous recombination with linear DNA fragments (Widger et al. 1998).
Despite its favorable characteristics for basic research and environmental applications, PCC7002s capabilities as a chassis for genetic modification have not been fully realized (Ramey et al. 2015). The organism possesses no native CRISPR systems, making it an ideal candidate to receive a foreign CRISPR/Cas system (such as that of S. pyogenes) for genomic editing (Cong et al. 2013). Its tendency towards natural recombination suggests that a high-efficiency MAGE protocol could be developed for PCC7002 with relative ease. We sought to address the gap in genetic manipulation technologies available for PCC7002 by testing and implementing our framework on the organism. Back
Target Organism: Rhizobium tropici CIAT 899
A broadly symbiotic, nitrogen-fixing bacterium
Rhizobium tropici CIAT 899 is a promiscuous α-proteobacterium first isolated from a common-bean nodule in Colombia (Martinez-Romero et al. 1991). As a Rhizobium species, CIAT 899 can fix atmospheric nitrogen into a form that can be absorbed by plants, thus reintroduce nitrogen as an essential nutrient into the soil (New Mexico State University 2005). Rhizobium must nodulate plant roots in order to fix nitrogen, and although some Rhizobium species can only nodulate a limited number of legume species, CIAT 899 can establish nitrogen-fixing symbioses with a broad range of legume hosts (Hungria et al. 2000). CIAT 899 is more environmentally adaptable when compared to other Rhizobium species due to its tolerance of stressful conditions such as acidic soils and high temperatures (Graham et al. 1994). It has also shown resistance to various antibiotics, pesticides and fungicides (Bernal et al. 2004), all of which are properties that have led to CIAT 899's commercial use in inoculating the common bean Phaseolus vulgaris in South America and Africa (Ormeno-Orrillo et al. 2012). Because nitrogen-fixing legumes reduce reliance on synthetic fertilizers that are costly and ecologically damaging, scientists have recognized the role of Rhizobia in agricultural sustainability and ecological preservation (Balkan 2007).
R. tropici CIAT 899 is an ideal candidate for MAGE because it has a reasonably fast doubling time of < 6 hr (Morón et al. 2005) and its genome is fully sequenced (NCBI Taxonomy ID: 698761). CIAT 899 can produce a wide variety of Nod factor structures, which influence the range of symbiotic hosts (Morón et al. 2005). Although it is already used commercially to reintroduce soil nitrogen and has been very successful in increasing crop yields in some experimental plots, nitrogen-fixing legumes on average do not fix enough nitrogen to appreciably increase crop yield (Gilbert, 2012). There is potential for MAGE to further diversify the range of hosts that CIAT 899 require to fix nitrogen and increase the output of nitrogen from nitrogen-fixing symbioses. Back
Potential Outcomes
Genetic Manipulation Technologies in PCC7002 and RT-CIAT
We chose PCC7002 and RT-CIAT as organisms for developing our framework due to their high potential for making a positive impact in carbon-neutral and ecological industries. PCC7002 could be engineered to produce and tolerate higher concentrations of free fatty acids (FFAs), which could be harvested and developed into lipid biofuels. MAGE is an ideal technology for modifying PCC7002's FFA biosynthesis pathway due to its ability to produce large numbers of genetic variants at targeted loci. The nitrogen-fixation mechanisms of RT-CIAT could be optimized, reducing the need for artificial, nitrate-based fertilizers in plant crops. Back
Improving the Function of a Previously Existing BioBrick
We have improved the characterization of three previously existing BioBricks, namely three of the Anderson Promoters: BBa_J23100 (Anderson Strong), BBa_J23111 (Anderson Medium), BBa_J23114 (Anderson Weak).
We assembled these three promoters to citrine and the T7 terminator, and then ligated the constructs into two different vectors that could be transformed into R. tropici and S. meliloti. We measured expression using these promoters in E. coli, finding that bacA, lac, tac, and all three anderson promoters all drive expression to different extents. We then successfully transformed plasmids bearing promoter-citrine constructs into S. meliloti, finding successful expression for tac, Anderson Strong, and Anderson Medium.
We further assembled the Anderson Medium promoter to 5 different recombinases and confirmed that they were successfully transformed into S. meliloti. MAGE experiments are underway.
Our constructs have been confirmed by sequencing in pKT230, a broad-host range plasmid, and pPZP200, a plasmid that can be transformed into rhizobium and agrobacterium. The anderson weak-citrine and S-RSM4 single stranded binding protein constructs have been sequenced in the biobrick constructs, while sequencing for the other constructs is being done.
We also determined that BBa_K125000 does not have the advertised sequence, even as it does contain the spectinomycin marker and could also independently replicate in E. coli, as determined by our electroporation experiments.
Back
References List
References
Balkin, J.M. "How New Genetic Technologies Will Transform Roe v. Wade", 56 Emory L. Rev. 843 (2007).
Cong L, Ann Ran F et al. "Multiplex Genome Engineering Using CRISPR/Cas Systems." Science2013 339 819-823.
Culligan, K., Meyer-Gauen, G., Lyons-Weiler, J. & Hays, J. Evolutionary origin, diversification and specialization of eukaryotic MutS homolog mismatch repair proteins. Nucleic Acids Res.28, 463-471 (2000).
Gaj, T., Gersbach, C. & III, C. ZFN, TALEN, and CRISPR/Cas-based methods for genome engineering. Trends in Biotechnology31, (2013).
Gallagher RR, Li Z, Lewis AO, and Issacs FJ. "Rapid editing and evolution of bacterial genomes using libraries of synthetic DNA." Nature Protocols2014 9(10) 2301-2316.
Gilbert, Natasha. "African Agriculture: Dirt Poor." Nature 483.7391 (2012): 525-27. Web.
Graham PH, Draeger KJ, Ferrey ML, Conroy MJ, Hammer BE, Martinez E, Aarons SR, Quinto C. Acid pH tolerance in strains of Rhizobium and Bradyrhizobium, and initial studies on the basis for acid tolerance of Rhizobium tropici UMR1899. Can J Microbiol/Rev Can Microbiol.1994;40:198-207. doi: 10.1139/m94-033
Hamilton JJ and Reed JA. "Identification of Functional Differences in Metabolic Networks Using Comparative Genomics and Constraint-Based Models." PLOS One2012 7(4) 1-19.
Hsu, P., Lander, E. & Zhang, F. Development and Applications of CRISPR-Cas9 for Genome Engineering. Cell157, (2014).
Hungria M, Andrade DD, Chueire LMD, Probanza A, Guttierrez-Manero FJ, Megias M. Isolation and characterization of new efficient and competitive bean (Phaseolus vulgaris L.) rhizobia from Brazil. Soil Biol Biochem.2000;32:1515-1528. doi: 10.1016/S0038-0717(00)00063-8.
Ludwig M and Bryant DA.  "Synechococcus sp. Strain PCC 7002 Transcriptome: Acclimation to Temperature, Salinity, Oxidative Stress, and Mixotrophic Growth Conditions." Front. in Microbiology2012 3(324) 1-14.
Lynch, M. D., Warnecke, T. & Gill, R. T. SCALEs: multiscale analysis of library enrichment. Nat. Methods4, 87-93 (2007).
Mali P, Yang L et al. "RNA-Guided Human Genome Engineering via Cas9." Science2013 339, 823-826.
Martinez-Romero E, Segovia L et al. "Rhizobium tropici, a Novel Species Nodulating Phaseolus vulgaris, L. Beans and Leucaena sp. Trees." Int. J. of System. Biotech.1991 41(3), 417-426.
Martínez-Romero E, Segovia L, Mercante FM, Franco AA, Graham P, Pardo MA. Rhizobium tropici, a novel species nodulating Phaseolus vulgaris L. beans and Leucaena sp. trees. Int J Syst Evol Microbiol. 1991;41:417-426.
Moron B, Soria-Diaz ME, Ault J, Verroios G, Noreen S, Rodriguez-Navarro DN, Gil-Serrano A, Thomas-Oates J, Megias M, Sousa C. Low pH changes the profile of nodulation factors produced by Rhizobium tropici CIAT899. Chem Biol. 2005;12:1029-1040. doi: 10.1016/j.chembiol.2005.06.014.
"Nitrogen Fixation by Legumes." Nitrogen Fixation by Legumes. New Mexico State University, n.d. Web. 02 July 2015.
Ormeño-Orrillo, Ernesto et al. "Genomic Basis of Broad Host Range and Environmental Adaptability of Rhizobium Tropici CIAT 899 and Rhizobium Sp. PRF 81 Which Are Used in Inoculants for Common Bean (Phaseolus VulgarisL.)." BMC Genomics 13 (2012): 735. PMC. Web. 28 Aug. 2015.
Ramey, C., Barón-Sola, Á., Aucoin, H. & Boyle, N. Genome Engineering in Cyanobacteria: Where We Are and Where We Need To Go. ACS Synth. Biol.150605142013007 (2015). doi:10.1021/acssynbio.5b00043
Ruffing, A. "Improved Free Fatty Acid Production in Cyanobacteria with Synechococcus sp. PCC 7002 as Host." Front. Bioeng. Biotechnol.2014 2.
Song HS, McClure RS, Beliaev AS et al. "Integrated in silico Analyses of Regulatory and Metabolic Networks of Synechococcus sp. PCC 7002 Reveal Relationships Between Gene Centrality and Essentiality." Life2015 5(2) 1127-1140.
Wang, H. et al. Programming cells by multiplex genome engineering and accelerated evolution. Nature460, 894-898 (2009).
Widger WI, Chen X, and Samartzidou H. "Synechococcus PCC 7002." In Bacterial Genomes: Physical Structure and Analysis. De Bruijn FJ, Lupski JR, GM Weinstock, Eds. Springer US: 1998, 763-770.
Xu Y, Alvey RM, Byrne PO, Graham JE, Shen G, and Bryant DA. "Expression of genes in cyanobacteria: adaptation of endogenous plasmids as platforms for high-level gene expression in Synechococcus sp. PCC 7002." Methods Mol. Biol.2011 684, 273-293.
x
Yale iGEM 2015
Main Campus:
Yale Department of Molecular, Cellular & Developmental Biology
Attn: Farren Isaacs/iGEM
219 Prospect St
PO Box 208103
New Haven, CT 06520
Tel. +1(203) 432-3783
E-mail: igem.yale@gmail.com
© Yale iGEM 2015
