<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Lambert_GA_Project skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Lambert GA/Project</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="headerLinks"><DIV id="header"><A href="https://2015.igem.org/Team:Lambert_GA">Home</A><A href="https://2015.igem.org/Team:Lambert_GA/Project" style="color:#476353">Project</A><A href="https://2015.igem.org/Team:Lambert_GA/Parts">Parts</A><A href="https://2015.igem.org/Team:Lambert_GA/Team">Team</A><A href="https://2015.igem.org/Team:Lambert_GA/Attributions">Attributions</A><A href="https://2015.igem.org/Team:Lambert_GA/Safety">Safety</A><A href="https://2015.igem.org/Team:Lambert_GA/Practices">Human Practices </A><A href="https://2015.igem.org/Team:Lambert_GA/Collaborations">Collaborations and Other</A></DIV></DIV><H1>Project</H1><DIV id="text"><H3 style="text-align:center; font-size:25px"> OVERVIEW </H3><P1>The Lambert High School iGEM team from Suwanee, Georgia is a group of dedicated and self described nerds who love the challenges of doing science. We set out to further our work from our previous year’s project, as we never had the chance to express our protein, chitin deacetylase. <P>During the year 2013 we isolated the chitin deacetylase gene, CDA2 from Sacchromyces cerevesai. Studies have shown that this gene could be expressed in E.coli successfully. Martinou, A., D. Koutsioulis, and V. Bouriotis. 2003)  CDA breaks down chitin, a naturally occurring polysaccharide into chitosan. Chitosan is widely used in a multitude of industries; including agriculture, aquaculture, health, biofilm production and water treatment. Current methods of Chitosan production use strong acids and bases to deacetylate the chitin, these may be cost effective at the moment, but the waste is incredibly harsh on the environment and is not sustainable. Our idea was to engineer cells to accomplish this task. The cells could be grown and used in a bioreactor, producing acetic acid as a waste product, which also has useful industrial applications. Current industries that produce chitosan are often located near coastal areas due to the natural abundance of shrimp shells and other exoskeletons. Unfortunately the wastewater form production can be polluting to the fragile ecosystems.</P><P>Chitosan has antifungal and antibacterial properties. We needed to engineer a system where we could turn on and off CDA production in such a way as to not kill our host cells. To this end we designed several constructs that would direct expression to the periplasm using a pelB tag combined with a Tet repressible promoter. We added GFP to the construct so that we might be able to visualize the proper expression in the periplasm. We also wanted to engineer a kill switch by incorporating 2013 Carnegie-Mellon's Killer Red   BBa_K1184000.  This would enable an environmental safety switch in our cells.</P><P>After research and discussion with our mentoring team and Dr. Styczynski at the Georgia Institute of Technology, we chose a threefold approach to the design. Sometimes large engineered proteins have difficulty folding properly in the periplasm so we needed to address the issue using different placement of promoters.</P><P>The first design we call the bicistronic construct. We engineered the pelB tag and CDA under TetR promotion and the GFP under a lac inducible promoter. Having only the pelB tag and CDA in the periplasm would hopefully eliminate the difficulty with a 2000bp gene folding properly in the space. The GFP as a reporter would just let us know our transformations were successful and we could choose colonies for colony PCR using the color selection. The introduction of two independent genes however could have been a drain on the cell leading to early cell death.</P><P>The second design of the bidirectional is a way to use just a repressible promoter of TetR, but have the transcription of the two genes occur independently in opposite directions. There are several commercial plasmids that use bidirectional promoters to accomplish multi gene expression. This construct proved very problematic as we were ordering the gBlocks from IDT. Because the back to back promoter sequences were likely to form hairpin structures we had to order this in two parts using a multiple cloning site as a linker between the two TetR promoters. </P><P>The last construct of our project was a fusion protein consisting pf the pelB leader sequence, CDA and GFP all under the promotion of one TetR promoter. The benefit we aimed for with this sequence would be the ability to visualize the GFP in the periplasm of the cells. The large size though of the protein could lead to folding difficulties and then a complete loss of expression of either genes; CDA or GFP.</P><P>Our project, if successful, could lead to two significant advances, the first being a more environmentally friendly way to produce chitosan. Using engineered E.coli cells, we named Chitinite, to create to useful products of chitosan and acetic acid from waste shrimp shells would help save coastal ecosystems. A second significant contribution to science is our Biobricked bidirectional promoter system. This can help scientists with dual gene expression.</P><P>Another highlight of our efforts was a Discovery Dialogue held in partnership with the Atlanta Science Festival. We brought together on one stage, in an open forum; a legislator, ethicist, immunologist, FBI agent and scientist. The public was invited to ask questions and the concluding polls showed both an increase in the positive perception of synthetic biology, but also a request to hold another session this coming year.</P><H3 style="text-align:center; font-size:25px">MATERIALS &amp; METHODS</H3><H3>Resuspension</H3><OL><LI style="font-size:16px">Centrifuge the tube for 3-5 sec in small centrifuge to ensure the material is in the bottom of the tube
</LI><LI style="font-size:16px">Add TE to reach a final concentration of 10 ng\ul
</LI><LI style="font-size:16px">Vortex briefly
</LI><LI style="font-size:16px">Put tube (use butterfly float) into water bath at 50° C for 20 minutes
</LI><LI style="font-size:16px">Briefly vortex and centrifuge
</LI></OL><H3>Double Digestion for DNA constructs and Plasmids (pSB1C3 and pSB1T3)</H3><OL><LI>Shake DNA gently to reach equal DNA concentration throughout the liquid.
</LI><LI>Add 33 uL of water to each PCR tube of DNA constructs; 38 uL of water to each PCR tube of plasmids
</LI><LI>Add 5 uL of 10x NEB buffer to each of the tubes
</LI><LI>Add DNA to each tube
</LI><LI>10 uL Fusion
</LI><LI>10 uL Bicistronic 
</LI><LI>10 uL Bidirectional 1
</LI><LI>10 uL Bidirectional 2
</LI><LI>5 uL pSB1C3
</LI><LI>5 uL pSB1T3
</LI></OL><LI>Add enzymes to each tube
</LI><LI>1 uL of both EcoRI and PstI to Fusion tube
</LI><LI>1 uL of both EcoRI and PstI to Bicistronic tube
</LI><LI>1 uL of both EcoRI and Bam HI to Bidirectional 1 tube
</LI><LI>1 uL of both PstI and Bam HI to Bidirectional 2 tube
</LI><LI>1 uL of both EcoRI and PstI to pSB1C3 tube
</LI><LI>1 uL of both EcoRI and PstI to pSB1T3 tube
</LI><LI>Incubate at 37°C for 5-15 min
</LI><LI>Stop reaction by heating up at 65°C for 20 minutes.
<H3>Ligation </H3></LI><LI>Ligate these DNA constructs with plasmid PSB1C3-2070
<TABLE id="tables"><TBODY><TR><TH>Construct</TH><TH>Vector(pSB1C3)</TH><TH>DNA</TH><TH>Buffer</TH><TH>Ligase</TH></TR><TR><TD>Bicistronic</TD><TD>4μL</TD><TD>15μL</TD><TD> 19μL</TD><TD>1.9μL</TD></TR><TR class="alt"><TD>Bidirectional 1</TD><TD>4μL</TD><TD>4μL</TD><TD> 11.5μL</TD><TD>1.15μL</TD></TR><TR><TD>Bidirectional 2</TD><TD>4μL</TD><TD>4μL</TD><TD> 11.5μL</TD><TD>1.15μL</TD></TR><TR class="alt"><TD>Fusion</TD><TD>4μL</TD><TD>13.5μL</TD><TD> 17,5μL</TD><TD>1.75μL</TD></TR></TBODY></TABLE></LI><LI> Note: Bidirectional 1 and bidirectional 2 are 

both placed into the same tube so the 

vector, buffer, and ligase are only placed 

once into that tube. The bidirectional 1 

and 2 have separate amounts that are 

placed into the tubes. (applies to both 

pSB1C3 and pSB1T3) </LI><LI>Ligate these DNA constructs with plasmid PSB1C3-2070
<TABLE id="tables"><TBODY><TR><TH>Construct</TH><TH>Vector(pSB1C3)</TH><TH>DNA</TH><TH>Buffer</TH><TH>Ligase</TH></TR><TR><TD>Bicistronic</TD><TD>4μL</TD><TD>12μL</TD><TD> 16μL</TD><TD>1.6μL</TD></TR><TR class="alt"><TD>Bidirectional 1</TD><TD>4μL</TD><TD>3.5μL</TD><TD> 10.5μL</TD><TD>1.05μL</TD></TR><TR><TD>Bidirectional 2</TD><TD>4μL</TD><TD>11.5μL</TD><TD> 15.5μL</TD><TD>1.55μL</TD></TR><TR class="alt"><TD>Fusion</TD><TD>4μL</TD><TD>11.5μL</TD><TD> 15,5μL</TD><TD>1.55μL</TD></TR></TBODY></TABLE><H3> Nanodrop</H3></LI><LI>Use 1 uL of EB buffer to use as blanks. Measure the blank twice; second time to confirm.
</LI><LI>Measure the concentration of the DNA for each of the DNA tubes by placing 1 uL  into the nanodrop.
<H3>Transform with competent E. coli Strains (NEB 10-beta competent or NEB 5-alpha competent)</H3></LI><LI>Thaw a tube of E. coli cells on ice until the last ice crystals disappear. Mix gently and carefully pipette 50 uL of cells into a transformation tube on ice.
</LI><LI>Add 1-5 uL containing 1pg-100ng of plasmid DNA to the cell mixture. Carefully flick the tube 4-5 times to mix cells and DNA. DO NOT VORTEX!
</LI><LI>Place the mixture on ice for 30 minutes. Do not mix.
</LI><LI>Heat shock at exactly 42°C for exactly 30 seconds. Do not mix.
</LI><LI>Place on ice for 5 minutes. Do not mix.
</LI><LI>Pippette 950 uL of room temperature SOC into the mixture.
</LI><LI>Place at 37°C for 60 minutes. Shake vigorously (250 rpm) or rotate.
</LI><LI>Warm selection plates to 37°C
</LI><LI>Mix the cells thoroughly by flicking the tube and inverting, then perform several 10-fold serial dilutions in SOC.
</LI><LI>Spread 50-100 uL of each dilution onto a selection plate and incubate overnight at 37°C.
<H3>Create Liquid Cultures </H3></LI><LI>Fill clean culture tube with 7 mL of LB
</LI><LI>Use clean inoculating lube to take a sample from colonies. (you want a few cells)
</LI><LI>Place inoculating lube into the culture tube and spin the lube, making sure to release some of the cells into the liquid
</LI><LI>Cover tubes with caps lightly/lightly parafilm 
</LI><LI>Set in a shaking water bath for 37°C
</LI><LI>Grow for 18-24 hours
<H3> Mini-Prep </H3></LI><LI>Pellet 1-5 mL bacterial overnight culture by centrifugation at &gt;8000 rpm (6800 x g) for 3 min at room temperature (15°C-25°C)
</LI><LI>Resuspend pelleted bacterial cells in 250 uL Buffer P1 and transfer to a microcentrifuge tube.
</LI><LI>Add 250 uL Buffer P2 and mix thoroughly by inverting the tube 4-6 times until the solution becomes clear. Do not allow the lysis reaction to proceed for </LI><LI>more than 5 min. If using LyseBlue reagent, the solution will turn blue.
</LI><LI>Add 350 uL Buffer N3 and mix immediately and thoroughly by inverting the tube 4-6 times. If using LyseBlue reagent, the solution will turn colorless. Vortex 30 seconds.
</LI><LI>Centrifuge for 10 minutes at 13,000 rpm (~17,900 x g) in a table-top microcentrifuge. 
</LI><LI>Apply the supernatant from step 5 to the QIAprep spin column by decanting or pipetting. Centrifuge for 60 seconds and discard the flow through.
</LI><LI>Wash the QIAprep spin column by adding 750 uL Buffer PE. Centrifuge for 30-60 seconds and discard the flow-through. Transfer the QIAprep spin column to the collection tube.
</LI><LI>Centrifuge for 1 minute to remove residual wash buffer.
</LI><LI>Place the QIAprep column in a clean 1.5 mL microcentrifuge tube. To elute DNA, add 30 uL Buffer EB to the center of the QIAprep spin column, let it stand for 5 minutes, and centrifuge for 1 min.
</LI><LI>Nanodrop to find out concentration of DNA.
<H3> Double Digestion </H3></LI><LI>Shake DNA gently to reach equal DNA concentration throughout the liquid.
</LI><LI>Add 36 uL of water to each PCR tube of DNA constructs
</LI><LI>Add 5 uL of 10x NEB buffer to each of the tubes
</LI><LI>Add DNA to each tube
</LI><LI>7 uL Fusion w/ pSB1C3
</LI><LI>7 uL Bicistronic w/ pSB1C3
</LI><LI>7 uL Bidirectional w/ pSB1C3
</LI><LI>7 uL Fusion w/ pSB1T3
</LI><LI>7 uL Bicistronic w/ pSB1T3
</LI><LI>7 uL Bidirectional w/ pSB1T3
</LI><LI> Add enzymes to each tube </LI><OL type="a"><LI>1 uL of both EcoRI and PstI to Fusion w/ pSB1C3 tube
</LI><LI>1 uL of both EcoRI and PstI to Bicistronic w/ pSB1C3 tube
</LI><LI>1 uL of both EcoRI and PstI to Bidirectional w/ pSB1C3 tube
</LI><LI>1 uL of both EcoRI and PstI to Fusion w/ pSB1T3 tube
</LI><LI>1 uL of both EcoRI and PstI to Bicitronic w/ pSB1T3 tube
</LI><LI>1 uL of both EcoRI and PstI to Bidirectional w/ pSB1T3 tube
</LI></OL><LI>Incubate at 37°C for 5-15 min
</LI><LI>Stop reaction by heating up at 65°C for 20 minutes.
<H3> Gel Electrophoresis </H3></LI><LI> Loading buffer was added to each DNA sample.
</LI><LI> Gel box filled with TAE buffer until it covered the gel (1% agarose gel).
</LI><LI> Ladder was loaded into the wells of the gel.
</LI><LI>Samples are loaded into the wells of the gel.
</LI><LI> Gel is run at 90V until dye line is approximately 75-80% of the way down the gel. (Run for about 1-1.5 hours)
</LI><LI>Gel is placed in Fast Blast stain or other appropriate DNA stain
</LI><LI>Use any device that has UV light and visualize your DNA fragments and confirm base pairs of DNA fragments. 
 <H3 style="text-align:center; font-size:25px">RESULTS</H3><P>Mature CDA2 gene that was inserted into the pSB1C3 plasmid was transformed into NEB 10 beta E. coli cells. They were placed onto chloramphenicol plates to grow but it yielded no results.
</P><P>Tested transformation efficiency of NEB 10 beta E. coli cells with iGEM transformation efficiency kits. RFP was inserted into pSB1C3 and transformed into NEB 10 beta E. coli cells and grown on chloramphenicol plates. RFP was expressed in the colonies. RFP was subsequently used to test our foldscope.
</P><P>Ligation of our constructs into pSB1C3 and pSB1T3 plasmids was carried out and they were transformed with NEB 10 beta E. coli and then placed on chloramphenicol and tetracycline plates respectively to grow. The chloramphenicol transformations were successful and the plates grew colonies. However, the tetracycline transformations did not yield growth. Transformations were carried out again for the pSB1T3 plasmids but again they yielded no results. The colonies on the chloramphenicol plates were taken and grown on liquid cultures, mini prepped and then ran on the gel to find the base pair length of the DNA. Due to the fact that the pSB1C3 plasmid had a similar base pair length with our constructs, the gel was inconclusive. (pSB1C3 has a base pair length of 2070 bp, Fusion construct is 1828 bp, Bidirectional construct is 2118 bp, and Bicistronic construct is 1929 bp and so the base pair length of the constructs could have been covered up by the plasmid base pair length.)


</P><H4 style="text-align:center"> Successful Transformations</H4><TABLE><TBODY><TR><TH> puc19 positive control </TH><TH> bicistronic contract of pelB CDA with GFP in psb1c3 </TH><TH> polycistronic construct pelB CDA with GFP in psb1c3 </TH></TR><TR><TH> fusion construct of pelB CDA with GFPin psb1c3</TH><TH> negative control- no plasmid</TH><TH> Transformation Efficiency</TH></TR></TBODY></TABLE><H4 style="text-align:center"> Results of Miniprep and Diagnostic Gels </H4><TABLE><TBODY><TR><TH> Miniprep concentration of CDA Fusion </TH><TH> Digested Miniprep showing 2000bp band </TH><TH> Digested Miniprep showing 2000bp band </TH></TR></TBODY></TABLE><H3 style="text-align:center; font-size:25px">MODELS</H3><H3> Determination of Chitosan in Liquid Samples using a UV-VIS Spectrophotometer</H3><H4>1.0      Purpose</H4><P>This Standard Operating Procedure (SOP) describes a method for the determination of chitosan in liquid samples with a Ultravilot-Visible (UV-VIS) spectrophotometer. </P><H4>2.0      Principle </H4><P>Chitosan reacts with Reactive Red 4A dye to form a colloid which absorb light at 575 nm.  The absorbance is relatively linear with the chitosan concentration in the range of 5 to 20 ppm.  

</P><H4>3.0       Health and Safety</H4><P>3.1     Wear goggles, a lab coat, and gloves at all times during the experiment. 
</P><P>3.2     Wear a dust mask when weighing out the solid dye powder to avoid inhalation.

</P><H4>4.0 Instrument Parameters and Labware</H4><H6><B>UV-VIS Spectrophotometer:</B></H6></LI><LI>Slit width:			1 nm 
</LI><LI>Path length:			1 cm
</LI><LI>	Measuring wavelength:		575 nm 
</LI><LI>	Cuvettes, glass or quartz (glass is for visible range only)
<P>	Weighing paper
</P><P>	Distilled water or deionized water
</P><P>	Class A glass flasks
</P><P>	Class A glass pipets
</P><P>	Transfer pipette
</P><P>	Eppendorf pipette and tips, 10-1,000 microliter range
</P><P>	10-15 ml centrifuge tube or glass tube 
</P><P>	Watch/timer


</P><H4>5.0 Reagent and Standard </H4></LI><LI>Chitosan: Low molecular weight, Aldrich or equivalent	
</LI><LI>Acetic acid 10%, Fisher Scientific or equivalent
</LI><LI>Glycine-HCl buffer, pH 2.5: Fisher Scientific or equivalent  (Keep refrigerated)
</LI><LI>Reactive Red 4 A dye: MP Biomedical or equivalent	
<H4>6.0 Reagent and Calibration Standard Preparation </H4></LI><LI><B>1000 ppm Chitosan stock solution preparation</B></LI><LI>Wash a 1-liter class A flask thoroughly.  Rinse well with deionized or distilled water.

</LI><LI>	Add 200 ml 10% acetic acid and approximately 500 ml water to the flask.

</LI><LI>Weigh 1 gram of chitosan using weighing paper and record the exact weight.

</LI><LI>Transfer the sample completely into the flask containing water and acetic acid

</LI><LI>	Completely fill the flask to the mark with deionized/distilled water. 

</LI><LI>Add one clean stir bar to the flask.

</LI><LI>Cover the mouth of the flask with parafilm or plug it with a stopper.

</LI><LI>	After stirring for four hours, Invert the flask and stir for another 30 minutes.

</LI><LI>	If there is any remaining precipitate at this point, filter the solution.

</LI><LI>The chitosan stock solution 1000 ppm is complete and ready for use.
</LI><LI><B>Chitosan Calibration Solution Preparation Using Volumetric Flasks</B></LI><OL><LI><B>5 ppm Chitosan Standard</B><P>Pipet 5 ml 1000 ppm stock solution into a 1-L flask.  Fill the flask to the mark with deionized/distilled water.  Cover the mouth of the flask with a stopper or parafilm.  Invert the flask to mix well. 

</P></LI><LI><B>7.5 ppm Chitosan Solution</B><P>Pipet 15 ml 1000 ppm stock solution into a 2-L flask.  Fill the flask to the mark with deionized/distilled water.  Cover the mouth of the flask with a stopper or parafilm.  Invert the flask to mix well.

</P></LI><LI><B>10 ppm Chitosan Solution</B><P>Pipet 10 ml 1000 ppm stock solution into a 1-L flask.  Fill the flask to the mark with deionized/distilled water.  Cover the mouth of the flask with a stopper or parafilm.  Invert the flask to mix well.

</P></LI><LI><B>12.5 ppm Chitosan Solution</B><P>Pipet 25 ml 1000 ppm stock solution into a 2-L flask.  Fill the flask to the mark with deionized/distilled water.  Cover the mouth of the flask with a stopper or parafilm.  Invert the flask to mix well.

</P></LI><LI><B>15 ppm Chitosan Solution</B><P>Pipet 15 ml 1000 ppm stock solution into a 1-L flask.  Fill the flask to the mark with deionized/distilled water.  Cover the mouth of the flask with a stopper or parafilm.  Invert the flask to mix well.

</P></LI><LI><B>20 ppm Chitosan Solution</B><P>Pipet 10 ml 1000 ppm stock solution into a 500 ml flask.  Fill the flask to the mark with deionized/distilled water.  Cover the mouth of the flask with a stopper or parafilm.  Invert the flask to mix well.

</P></LI><LI><B>25ppm Chitosan Solution</B><P>Pipet 25 ml 1000 ppm stock solution into a 1-L flask.  Fill the flask to the mark with deionized/distilled water.  Cover the mouth of the flask with a stopper or parafilm.  Invert the flask to mix well.

</P></LI><LI><B>30 ppm Chitosan Solution</B><P>Pipet 15 ml 1000 ppm stock solution into a 500 ml flask.  Fill the flask to the mark with deionized/distilled water.  Cover the mouth of the flask with a stopper or parafilm.  Invert the flask to mix well.
</P></LI></OL><LI><B>Chitosan Calibration Standard Solution Preparation using an Eppendorf Pipette</B><P>If class A flasks are not available, Eppendorf pipettes can be used to prepare standards with a total of 4 ml for each concentration.

</P><P>In each 15 ml centrifuge tube, pipet the following various amounts of 1000 ppm chitosan stock solution and distilled/deionized water: 

<TABLE id="tables"><TBODY><TR><TH>Concentration of Chitosan</TH><TH>1000 ppm Stock Solution</TH><TH>Water (distilled/deionized)</TH></TR><TR><TD>5.0 ppm</TD><TD>20μL</TD><TD>3980μL</TD></TR><TR class="alt"><TD>7.5 ppm</TD><TD>30μL</TD><TD>3970μL</TD></TR><TR><TD>10.0 ppm</TD><TD>20μL</TD><TD>3980μL</TD></TR><TR class="alt"><TD>12.5 ppm</TD><TD>50μL</TD><TD>3950μL</TD></TR><TR><TD>15.0 ppm</TD><TD>60μL</TD><TD>3940μL</TD></TR><TR class="alt"><TD>20.0 ppm</TD><TD>80μL</TD><TD>3920μL</TD></TR><TR><TD>25.0 ppm</TD><TD>100μL</TD><TD>3900μL</TD></TR><TR class="alt"><TD>30.0 ppm</TD><TD>120μL</TD><TD>3880μL</TD></TR></TBODY></TABLE></P></LI><LI><B>Preparation of Reactive Dye Solution</B></LI><LI>Weigh 0.075g of Reactive Red 4A dye powder using a weighing paper. 

</LI><LI>Transfer completely to a 500 ml class A flask.

</LI><LI>Completely fill flask to the mark with water
 
</LI><LI>	Add a stir bar to the flask. Cap the mouth of the flask with parafilm or a stopper and mix well.  The solution is now ready for use.

<H4> 7.0 Calibration Procedures </H4></LI><LI>Transfer 4 ml of each standard into a separate 15 ml centrifuge tube

</LI><LI>Add 1-ml of pH 2.5 glycine buffer to each tube

</LI><LI>Add 1ml of 75 ppm dye to each tube

</LI><LI>Cap the tubes and shake for 20 minutes.

</LI><LI>Prepare a blank sample using 4 ml of distilled/deionized water, 1 ml of dye and 1 ml of buffer.  Shake for 20 minutes.

</LI><LI>Set the wavelength of spectrophotometer to 575 nm and warm up the spectrometer for at least 15 minutes. 

</LI><LI>Fill a cuvette with the blank and wipe away any liquid on the outside surface of the cuvette. Do not touch the side of a cuvette in the light path. 

</LI><LI>Put the blank in the spectrometer and zero it out. 

</LI><LI>Fill a new cuvette with one of the standard solutions. Record the absorbance and repeat for all standards.

</LI><LI>Use Excel to plot the absorbances and create a calibration
<H4>8.0  Determination of Chitosan in a Liquid Sample </H4></LI><LI>Transfer 4 ml of the sample to a 15 ml centrifuge tube. Add 1 ml of 75 ppm red dye and 1 ml of glycine HCl buffer.  Mix well, then shake the solution for 20 minutes.

</LI><LI>Zero out the spectrometer with a blank.  Read and record the sample absorbance.

</LI><LI>If the sample absorbance is too high, dilute the sample until its absorbance fall within the linear range of the calibration curve.

</LI><LI>	Read the concentration of the diluted sample from the calibration chart.

</LI><LI>If necessary, multiply the concentration by the dilution factor to obtain the chitosan concentration in the original sample. 
<H4> 9.0 Quality Control </H4></LI><LI>Run a 15 ppm chitosan standard for every ten samples, and at the end of a sample sequence. 
</LI><LI>10.2 	Run a duplicate for every ten samples and at the end of a sample sequence. 
<H4>10.0 Documentation </H4><P>Record all data timely in a logbook or a notebook.

</P><H3 style="text-align:center; font-size:25px">CHARACTERIZATION OF EXISTING PARTS</H3><H3> BBa_K1184000 </H3><P> We characterized this part made by iGEM13_Carnegie_Mellon. This part is known as KillerRed,a red oligomeric fluorescent protein that is engineered to be phototoxic. It also acts as a kill-switch when expressed and irradiated with light, so it is a viable use to kill our cells if needed. </P><H3> BBa_E2050 </H3><P> We characterized this part made by Antiquity. This part is known as mOrange, which is a mRFP derivative. It is used as a yeast surface display system, which will be used in the Fold Scope aspect our project to view cells.

</P><H3 style="text-align:center; font-size:25px">NOTEBOOK</H3><H3> March</H3><P> Atlanta Science Festival </P><P> Received foldscope from Stanford Prakash Lab</P><H3> April </H3><P> Organized Chick Fil A Biscuit sale fundraiser </P><P> Worked with Next Generation Focus to provide science education enrichment </P><H3> May </H3><P> Trained new members in basic lab procedures </P><H3> June </H3><P> Researched yellow stripe rust/ CRISPR cas9 project idea </P><P> Researched PelB tags for relocating chitin deacetylase to the periplasm</P><P> Emailed FBI agents for outreach project</P><P> Researched expression vectors </P><H3> July </H3><P> Narrowed down focus of research to the PelB tags</P><P> Designed constructs using 3a assembly method </P><P> Designed constructs using gene synthesis method </P><P> Decided to use gene synthesis constructs due to resource restraints </P><P> Met with Georgia Tech iGEM team; received assistance for construct design </P><H3> August </H3><P> Completed constructs </P><P> Added a fusion protein </P><P> Practiced transformation procedures </P><P> Practiced digest procedures; fixed errors in procedure </P><P> Fixed illegal restriction sites in plasmids </P><P> Researched bidirectional and bicistronic constructs </P><P> Designed and received constructs from IDT </P><P> Resuspended sequences from IDT </P><P> Received filters from Rosco </P><P> Designed team shirts and jackets </P><H3> September </H3><P> Completed wiki </P><P> Completed banner </P><P> Submitted parts to parts registry</P><P> Characterized two existing parts Killer Red and mOrange </P><P> Designed handouts and flyers for Jamboree </P><P> Designed &quot;Lab Hacks&quot; outreach project for high school labs</P><P> Completed presentation for Jamboree </P><P> Completed poster for Jamboree </P></LI></P1></DIV><DIV id="sponsors-bottom"><P>
      This wiki is designed and constructed by Lambert iGEM.<A href="https://twitter.com/lamberths_iGEM">Twitter</A> | <A href="mailto:lambertigemteam@gmail.com" target="_top">Email</A> | <A href="https://drive.google.com/open?id=0B7VsrNZQ3hq7RXIxY2pzbXI1VjA"> Video </A></P></DIV></DIV></DIV></DIV></DIV></BODY></HTML>