<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_TU_Darmstadt_Results_Modeling_Open_Software"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:TU_Darmstadt/Results/Modeling/Open_Software">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:TU_Darmstadt/Results/Modeling/Open_Software&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:TU_Darmstadt/Results/Modeling/Open_Software&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:TU_Darmstadt/Results/Modeling/Open_Software&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:TU_Darmstadt/Results/Modeling/Open_Software" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:TU Darmstadt/Results/Modeling/Open Software</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><P><TITLE>Home</TITLE></P><DIV id="allWrap"><DIV id="headerBG"><DIV id="headerWrap" class="container_24"><DIV id="topBar" class="grid_18"><UL class="top"><LI class="active first"><A href="https://2014.igem.org/Team:TU_Darmstadt">Home</A></LI><LI><A href="https://2014.igem.org/Team:TU_Darmstadt/Project">Project</A></LI><LI><A href="https://2014.igem.org/Team:TU_Darmstadt/Results">Results</A></LI><LI><A href="https://2014.igem.org/Team:TU_Darmstadt/PolicyandPractices">Policy &amp; Practices</A></LI><LI><A href="https://2014.igem.org/Team:TU_Darmstadt/Achievements">Achievements</A></LI><LI><A href="https://2014.igem.org/Team:TU_Darmstadt/Notebook">Notebook</A></LI><LI><A href="https://2014.igem.org/Team:TU_Darmstadt/Team">Team</A></LI><LI><A href="https://2014.igem.org/Team:TU_Darmstadt/Gallery">Gallery</A></LI><LI><A href="https://2014.igem.org/Team:TU_Darmstadt/Sitemap">Sitemap</A></LI><LI class="last"><A href="https://2014.igem.org/Team:TU_Darmstadt/Contact">Contact</A></LI></UL></DIV></DIV></DIV><DIV id="topWrap" class="container_24"><UL class="slides grid_18"><LI class="slide-container"><DIV class="nav"><LABEL for="img-6" class="prev">‹</LABEL><LABEL for="img-2" class="next">›</LABEL></DIV></LI><LI class="slide-container"><DIV class="nav"><LABEL for="img-1" class="prev">‹</LABEL><LABEL for="img-3" class="next">›</LABEL></DIV></LI><LI class="slide-container"><DIV class="nav"><LABEL for="img-2" class="prev">‹</LABEL><LABEL for="img-4" class="next">›</LABEL></DIV></LI><LI class="slide-container"><DIV class="nav"><LABEL for="img-3" class="prev">‹</LABEL><LABEL for="img-5" class="next">›</LABEL></DIV></LI><LI class="slide-container"><DIV class="nav"><LABEL for="img-4" class="prev">‹</LABEL><LABEL for="img-6" class="next">›</LABEL></DIV></LI><LI class="slide-container"><DIV class="nav"><LABEL for="img-5" class="prev">‹</LABEL><LABEL for="img-1" class="next">›</LABEL></DIV></LI></UL><DIV id="promoWrap" class="grid_6"><DIV class="grid_6 promo1"><A href="https://2014.igem.org/Team:TU_Darmstadt/Achievements/Parts" target="_top"><H1>Our Biobricks</H1><P>Look at our bricks!</P></A></DIV><DIV class="grid_6 promo2"><A href="https://2014.igem.org/Team:TU_Darmstadt/Results/Modeling/ANS_Engineering" target="_top"><H1>eANS</H1><P>Check out the best protein engineering model!</P></A></DIV><DIV class="grid_6 promo3"><A href="https://2014.igem.org/Team:TU_Darmstadt/Team" target="_top"><H1>The People</H1><P>See the people behind the project!</P></A></DIV></DIV></DIV><DIV id="contentWrap" class="container_24"><DIV id="footerBG"><DIV id="footerInner"><DIV id="footerWrap" class="container_24"><DIV class="grid_4"><UL class="menu"><LI class="first"><A href="https://2014.igem.org/Team:TU_Darmstadt">iGEM TUD</A></LI><LI><A href="https://2014.igem.org/Team:TU_Darmstadt/Sitemap">Sitemap</A></LI><LI><A href="https://2014.igem.org/Team:TU_Darmstadt/Team">Team</A></LI><LI><A href="https://2014.igem.org/Team:TU_Darmstadt/Contact">Kontakt</A></LI><LI class="last"><A href="https://2014.igem.org/Team:TU_Darmstadt/Team/Sponsors">Sponsoren</A></LI></UL></DIV></DIV></DIV></DIV><DIV id="contentWrap" class="container_24"><DIV id="leftNavi" class="grid_5"><NAV><UL class="menu"><LI class="first"><A href="https://2014.igem.org/Team:TU_Darmstadt">Home</A></LI><LI><A href="https://2014.igem.org/Team:TU_Darmstadt/Project">Project</A></LI><LI class="active"><A href="https://2014.igem.org/Team:TU_Darmstadt/Results">Results</A></LI><LI class="first"><A href="https://2014.igem.org/Team:TU_Darmstadt/Results/Pathway">Pathway</A></LI><LI><A href="https://2014.igem.org/Team:TU_Darmstadt/Results/Scaffold">Scaffold</A></LI><LI class="active"><A href="https://2014.igem.org/Team:TU_Darmstadt/Results/Modeling">Modeling</A></LI><LI class="first"><A href="https://2014.igem.org/Team:TU_Darmstadt/Results/Modeling/Theory">Theory</A></LI><LI><A href="https://2014.igem.org/Team:TU_Darmstadt/Results/Modeling/ANS_Engineering">ANS Engineering</A></LI><LI class="active"><A href="https://2014.igem.org/Team:TU_Darmstadt/Results/Modeling/Open Software">Open Software</A></LI><LI class="last"><A href="https://2014.igem.org/Team:TU_Darmstadt/Results/Modeling/Rule based Modelling">Rule based Modeling</A></LI></UL><LI><A href="https://2014.igem.org/Team:TU_Darmstadt/Results/Safety">Safety</A></LI><LI class="last"><A href="https://2014.igem.org/Team:TU_Darmstadt/Results/Open Hardware">Open Hardware</A></LI><LI><A href="https://2014.igem.org/Team:TU_Darmstadt/Results/PolicyandPractices">Policy &amp; Practices</A></LI><LI><A href="https://2014.igem.org/Team:TU_Darmstadt/Achievements">Achievements</A></LI><LI><A href="https://2014.igem.org/Team:TU_Darmstadt/Notebook">Notebook</A></LI><LI><A href="https://2014.igem.org/Team:TU_Darmstadt/Team">Team</A></LI><LI class="last"><A href="https://2014.igem.org/Team:TU_Darmstadt/Gallery">Gallery</A></LI></NAV></DIV><DIV id="wikicontent" class="grid_19"><DIV id="c381" class="csc-default"><DIV class="csc-header csc-header-n1"><H1 class="csc-firstHeader">Open Software</H1></DIV></DIV><DIV id="c382" class="csc-default"><ARTICLE><HEADER><HEADER><ARTICLE><P>According to our Open Hardware approach, we would like to contribute an automated version of a general sequence and structure file analysis. Furthermore it could be used as a corporate design data visualization tool during iGEM Projects.</P></ARTICLE></HEADER></HEADER><P>Our preferred programming language is <I>R</I>, due to its user friendly interface. Changing code is easy and intuitive even for beginners. We implemented the following automated functions, which are free to use or to modify.</P></ARTICLE></DIV><DIV id="c383" class="csc-default"><DIV class="csc-textpic-text"><TABLE summary="" style="margin: 0px auto;" class="contenttable"><THEAD><TR><TH scope="col"><HEADER><H2>Sequence Analysis</H2></HEADER></TH><TH><H2>    </H2></TH><TH><H2><SPAN style="text-align: start;">Structure Analysis</SPAN></H2></TH><TH><H2>    </H2></TH><TH scope="col"><H2>Plotting</H2></TH></TR></THEAD><TBODY><TR><TD><P> </P><UL><LI>Clustal Omega</LI></UL><UL><LI>Consensus  Seqeuence</LI></UL><UL><LI>Conservative Side Detection</LI></UL><UL><LI>Shannon Entropy</LI></UL><UL><LI>Mutual Information</LI></UL><P> </P></TD><TD><P> </P><UL><LI> Normal mode Analysis</LI></UL><UL><LI>Comparison of models (NMA)</LI></UL><UL><LI>Trajectory Analysis</LI></UL><UL><LI>RMSD</LI></UL><UL><LI>RMSF</LI></UL><UL><LI>Binding estimation</LI></UL><UL><LI>Torison/Dihedral analysis</LI></UL><UL><LI>Distance matrix calculation</LI></UL><P> </P></TD><TD><P> </P><UL><LI>ggplot</LI></UL><UL><LI>HeatMap</LI></UL><UL><LI>Volcano Plot</LI></UL><UL><LI>Wireframe</LI></UL><UL><LI>'Fancy' 3D Scatter Plot</LI></UL><P> </P><P> </P></TD></TR></TBODY></TABLE></DIV></DIV><DIV id="c384" class="csc-default"><DIV class="csc-header csc-header-n4"><H1>Sequence Analysis</H1></DIV><DIV class="csc-textpic-text"><P>Bioinformatics relies essentially on sequences and their corresponding alignment. Bad sequence alignment will worse results received from calculation of Shannon Entropy and Mutual Information. After using the Basic Local Alignment Search Tool (<I>BLAST</I>) you will have to align all your sequences by using a distribution of Clustal Omega for instance. When using any Linux system you can use this function after installing the needed software package.<SUP>1</SUP> '<I>MSA_File</I>' will enable pre-aligning of your sequences by using the <I>tcltk</I> interface. After finished calculation you should rework output for an optimal solution.  The next function '<I>Analyse_Start</I>' is an automated version of sequence analysis. Per default it will calculate Shannon Entropy, two sets of mutual information (<I>SUMI</I> &amp;<I>ORMI</I> of the <I>BioPhysConnectoR</I> package) and a mutual information based contact map. General information like consensus sequence and potential conservative sites will also be computed and plotted automatically. Modifying your scope by using  other default options like MI-Treshold for contact map objects and choosing nullmod counter for different calculation of mutual information can be chosen at start. Another option of your choice would be the change of used amino acid alphabet instead of the common set. Therefore you could gain knowledge about the distributed amino acids on a specific position and relevant chemical properties.</P></DIV></DIV><DIV id="c385" class="csc-default"><DIV class="csc-header csc-header-n5"><H1>Structure Analysis</H1></DIV><DIV class="csc-textpic-text"><P>Not only distribution of amino acid at a specific position inside an alignment is important but also knowledge about the three-dimensional structure and their implication on the function is crucial. As written in our theory section, we used a normal mode analysis based on the <I>bio3d </I>package developed by <I>The Grant Lab</I>. Using<I> 'igem_NMA' </I>we can validate motion of protein by using different force fields described in the corresponding R documentation. These will be automatically compared and relative residual cross correlation matrix will be plotted indicating a positive or negative correlation. Atomic fluctuations and deformation energy will also be quantified and saved as a pdb-file. Using the provided trajectory analysis calculation will enable calculation of RMSD and RMSF. Another interesting option would be computation of distance calculation between two different chains, ligand or chain and the absolute distance between all atoms inside a pdb as a distance matrix. General structural information like Torsion/Dihedral analysis can also be plotted easily.</P></DIV></DIV><DIV id="c386" class="csc-default"><DIV class="csc-header csc-header-n6"><H1>Plotting</H1></DIV><DIV class="csc-textpic-text"><FOOTER><ASIDE><P class="align-center"><I>Different data need different plots.</I></P><P>Therefore we are willing to provide standardized plotting functions in R.<SUP>2</SUP>  Although though of an corporate design, these can be modified by the user easily by adding new layers onto an existing graph. Although you do minor changes like another main title or using other fonts the output will be the same.Most data visualization will be in a two dimensional space but can be achieved - in R - with different input classes like 'data.frame','vector' and 'matrix', although latter must be converted into class 'data.frame' before plot initialization.<SUP>3 '</SUP><I>Save2D_Vec</I>' and '<I>Save2D_DF'</I> need different input information as written in each function name. Both displayed dot plots connected via respective coloured line. Auxilliary it will create a corresponding bar or density plot, due to its input information. Width of bar plot can be calculated during runtime or taken from command line. Other plots describe a three dimensional space as shown in HeatMap and Volcano Plot. The last two function create plots best used for short fragments, due to automatic highlightning of data as text inside plot. Setting ticks manually inside all plots is preferred, while using short sequences because of possible overplotting. </P></ASIDE></FOOTER></DIV></DIV><DIV id="c389" class="csc-default"><DIV><OL><LI><A href="http://www.clustal.org/omega/" target="_blank">http://www.clustal.org/omega/</A></LI><LI><A href="http://www.r-project.org/" target="_blank">http://www.r-project.org/</A></LI><LI><A href="http://docs.ggplot2.org/current/" target="_blank">http://docs.ggplot2.org/current/</A></LI></OL></DIV></DIV><DIV id="c387" class="csc-default csc-space-before-10"><DIV class="csc-header csc-header-n8"><H1>Downloadable Content</H1></DIV><UL class="csc-uploads csc-uploads-0"><LI class="li-odd li-first csc-uploads-element csc-uploads-element-r"><SPAN class="csc-uploads-fileName"><A href="https://static.igem.org/mediawiki/2014/a/a4/TUD14_OpenSoftware.zip" target="_blank">Open Software.zip</A></SPAN></LI></UL></DIV><DIV id="c388" class="csc-default"><DIV class="csc-header csc-header-n9"><H1>Reference</H1></DIV><FOOTER><FOOTER><FOOTER><FOOTER><OL><LI>Hoffgaard, F., Weil, P., &amp; Hamacher, K. (2010). BioPhysConnectoR: Connecting sequence information and biophysical models. BMC Bioinformatics, 11, 199. doi:10.1186/1471-2105-11-199. </LI><LI>Sievers, F., Wilm, A., Dineen, D., Gibson, T. J., Karplus, K., Li, W., Lopez, R., et al. (2011). Fast, scalable generation of high-quality protein multiple sequence alignments using Clustal Omega. Molecular Systems Biology, 7(539), 539. Nature Publishing Group. doi:10.1038/msb.2011.75 </LI><LI>Shanon, C. E. (1948). A Mathematical Theory of Communication. The Bell System Technical Journal, 27, 379-423. </LI><LI>Bio3D: An R package for the comparative analysis of protein structures. Grant, Rodrigues, ElSawy, McCammon, Caves, (2006) Bioinformatics 22, 2695-2696.org/bio3d/index.php</LI><LI>Timischl, Werner, Biostatistik, Eine Einführung für Biologen und Mediziner, Springer, 3. Auflage 2012</LI><LI><A href="http://bio.math-inf.uni-greifswald.de/viscose/html/alphabets.html" target="_blank">bio.math-inf.uni-greifswald.de/viscose/html/alphabets.html</A></LI></OL></FOOTER></FOOTER></FOOTER></FOOTER></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:TU_Darmstadt/Results/Modeling/Open_Software">http://2014.igem.org/Team:TU_Darmstadt/Results/Modeling/Open_Software</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:TU_Darmstadt/Results/Modeling/Open_Software" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:TU_Darmstadt/Results/Modeling/Open_Software" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:TU_Darmstadt/Results/Modeling/Open_Software&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:TU_Darmstadt/Results/Modeling/Open_Software&amp;oldid=372891" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></DIV></BODY></HTML>