"
Page
Discussion
View source
History
teams
Log in
 
Team:TU Darmstadt/Results/Pathway
From 2014.igem.org
Home
Home
Project
Results
Policy & Practices
Achievements
Notebook
Team
Gallery
Sitemap
Contact
‹
›
‹
›
‹
›
‹
›
‹
›
‹
›
Our Biobricks
Look at our bricks!
eANS
Check out the best protein engineering model!
The People
See the people behind the project!
iGEM TUD
Sitemap
Team
Kontakt
Sponsoren
Home
Project
Results
Pathway
Scaffold
Modeling
Safety
Open Hardware
Policy & Practices
Achievements
Notebook
Team
Gallery
Overview
We constructed a biosynthetic pathway consisting of seven enzymes (TAL, 4CL, CHS, CHI, F3H, DFR, ANS), which were split into two separate operons. In addition we designed a biosensor, which allowed us to measure the concentration of the key intermediate naringenin.
Now, we are proud to present the following results:
1. Our naringenin operon ( K1497016 ) works well.
2. Our naringenin biosensor ( K1497019 - K1497022 ) works in several constructions.
3. We engineered an improved anthocyanidin synthase eANS (K1497002) by using rational design methods.
4. Our pelargonidin operon ( K1497015 ) works well and we could extract the desired pelargonidin.
5. We constructed a functional Grätzel cell (DSC) with our produced and extracted pelargonidin.
Naringenin operon (K1497016)
This part is a composite of four genes each with the strong RBS ( BBa_B0034 ).
 
-  4-Coumaryl ligase - 4CL (BBa_K1033001)
-  Tyrosine ammonia lyase - TAL (BBa_K1033000)
-  Chalcone isomerase - CHI (BBa_K1497000)
-  Chalcone synthase - CHS (BBa_K1497001)
 
Together, these genes define the naringenin biosynthesis operon without a promotor. In addition of a promotor part the device is able to produce S-naringenin. This device is working in E. coli K and B strains.
Figure 1: Genetic map of the naringenin operon (K1479016)
Naringenin is produced stereoselectively from tyrosine. Malonyl-CoA is consumed during the biosynthesis process.
Figure 2: Pathway of the naringenin operon. Starting with tyrosine and ending with naringenin.
 
We created the naringenin biosynthesis operon under the control of the T7 promoter BBa_I712074 and the strong constitutive promoter BBa_J23100 , respectively . We measured the naringenin production after 16 h of incubation with the naringenin biosensor BBa_K1497020 .
The cell pellets from E. coli BL21(DE3) - pSB1C3-fdeR-gfp with and without T7-naringenin operon ( BBa_K1497017 ) are shown in figure 3. Only the cell transformed with BBa_K1497017 exhibited GFP fluorescence.
We were also able to measure the GFP fluorescence quantitatively and to calculate the production yield of both operons with the help of a calibration curve  for the naringenin sensor (Figure 4). We calculated a concentration of 1.9 µM ( BBa_K1497016 ) and 3 µM ( BBa_K1497017) naringenin repectively.
Figure 3: Cell pellets with and without T7-Naringenin operon from E. coli BL21(DE3)-pSB1C3-fdeR-gfp. By using ultraviolet light the pellet containing the naringenin operon shows a GFP fluorescence.
Figure 4:  Fluorescence of cells with and without the T7-naringenin operon BBa_K1497017 from E. coli BL21(DE3)-pSB1C3-fdeR-gfp and J23100-naringenin operon (BBa_K1497016) from E. coli Top10-pSB1C3-fdeR-gfp, respectively. E. coli BL21(DE3)-pSB1C3-fdeR-gfp without T7-naringenin operon showed no detectable fluorescence. Only in the cells with the functional operon is the GFP fluorescence measurable. The estimated yields are 3 µM for BBa_K1497017 and 1,9 µM for BBa_K1497016.
fdeR - Naringenin sensor (K1497019 - K1497022)
FdeR is a homo dimeric protein from Herbaspirillum seropedicae. In the presence of naringenin (or naringenin chalchone), FdeR activates the specific promoter region upstream of the fdeR region and induces a strong gene expression. In Herbaspirillum seropedicae the FdeR activates the Fde-Operon (Fde: Flavanone degradation) and enable growth with naringenin and the naringenin chalcone.
So in combination with GFP or another fluorescense protein this part can be used as an in vivo naringenin sensor.
Figure 5: Flow chart of the FdeR activated gfp expression. The constitutive expression of fdeR the FdeR proteins form homodimers. In the presence of naringenin, naringenin molecules bind to the FdeR homodimer and operate a conformational change of the homodimeric FdeR structure. This conformational change activates FdeR, which is now enabled to bind to the uncharacterized promotor domain. Binding to the promotor domain induces expression of genes downstream of the fdeR promoter region.
You can use the reporters for measuring naringenin concentrations in your samples.
Depending on which fluorophor you want to detect, you can use one of three biosensors:
A: with CFP response use BBa_K1497022
B: with mKate response use BBa_K1497021
C: with no reporter BBa_K1497019
D: with GFP response use BBa_K1497020
Figure 6: E. coli Top10 with different Naringenin biosensors. Left: On agar plate without naringenin no colour is visible. Middle: On agar plate with 100 µM naringenin colour is visible, except of negative sample BBa_K1497019 without fluorophor. Right: On agar plate with 100 µM Naringenin under UV light. The fluorescence of GFP, CFP and mKate is visible.
You can create your own naringenin sensor or your own naringenin dependent gene expression device as well. For these reasons use the Biobrick  K1497019  and clone your parts of interest (without RBS!) behind the device.
In addition, we proved the expression of GFP by running a SDS-PAGE (12,5% polyacrylamide gel).
Figure 7: SDS-PAGE of FdeR-GFP-pSB1C3 in BL21 using different naringenin concentrations. We used "Promega Broad Range Protein Marker" and a 12,5% acrylamide gel. GFP has a mass of 26.9 kDa.
The Biobrick  BBa_K1497019  produces in E. coli B and K strains the FdeR Protein. We measured the fluorescense of GFP and mKate after the incubation with different concentrations of naringenin. The results are shown in Figure 8 & 9.
Figure 8: Characterization of BBa_K1497020. GFP fluorescence depends on the concentration of naringenin. We measured the GFP fluorescence after 16 h incubation with different concentrations of naringenin. By setting higher concentrations of naringenin, we gained higher fluorescence of GFP as well.
Figure 9: Characterization of BBa_K1497021. mKate (BBa_K1055000) fluorescence depends on the concentration of naringenin. We measured the mKate (BBa_K1055000) fluorescence after 16 h incubation with different concentrations of Naringenin. By setting higher concentrations of naringenin, we gained higher fluorescence of mKate as well.
eANS (K1497002)
The anthocyanidin synthase from Fragaria x ananassa (ANS, EC 1.14.11.19) catalyzes many reactions in the anthocyanidin pathway. We used its functionality by catalyzing the conversion of the leucoanthocyanidin (2R,3S,4S)-cis-lucopelargonidin to the anthocyanidin pelargonidin. It also catalyzes the conversion of the leucoanthocyanidin to flavonol (kampferol). In order to avoiding this side reaction and enhance metabolic flux, we designed a protein scaffold ( BBa_K1497031 ).
Earlier studies hypothesized that ANS may be involved in metabolic channeling in their native organisms. By modelling the mechanical movements of ANS, we discovered a strong flexibility at the C-terminus. Subsequently we modelled ANS' structure and its movements. Thereby we detected a tail at the C-terminus of the enzyme fluctuating and covering the active site (Fig. 15). In oder to improve the enzyme we decided to remove the tail and to construct our pelargonidin operon with this engineered ANS (eANS). The laboratory results confirmed the previous modeling results. The engineered ANS exhibited better yields than the original one when used in an operon producing pelargonidin (BBa_K1497015).
Figure 10: Workflow of the eANS rational design approach
In order to optimize the metabolic channeling of ANS, we chose a rational protein engineering approach. 
The first step of our multi scale and rational engineering project was the creation of
a sophisticated 3D model with YASARA structure. This model was then used for a structural refinement with the SCWRL alghorithm and was energy minimized with YASARA nova force field.
Afterwards, we started a true mechanical engineering approach to determine the movements within the protein. Therefore, a Gaussian Network Model (GNM) (Fig. 11) and an Anisotropic Network Model (Fig. 12) were implemented.
Those are simple models which simulate the mechanical behavior of the protein. Moreover, Linear Response Theory (LRT) (Fig. 14) was used to simulate the substrate binding inside the pocket and thus trigger an induced fit mechanism. 
Afterwards we collect our data, defined rational mutations and finally constructed eANS. With this eANS version another MD simulation was started and the sequence of the protein was given to the wetlab for in vitro construction and in vivo characterization.
Figure 11: GNM computation of ANS shows a great peak at the C terminus. It leads to the assumption that the C terminal region of the ANS is highly flexible.
Figure 12: ANM of wild-type ANS shows its flexibility
Figure 13: We observed a strong deformation of the enzyme in the LRT model. This result reveals that the C-terminal region of the ANS is highly flexible during the process of substrate binding (induced fit).
Figure 14: LRT model simulates the substrate binding in the active site of ANS by applying a force vector to the active site and binding region.
Figure 15: Elastic Network Model (ENM) shows the fluctuating tail covering the active site of ANS
Pelargonidin operon (K1497015)
We constructed a pelargonidin producing operon under the control of a T7 promoter (K1497015). The operon consists of 3 genes (flavonon-3beta-hydroxylase, dihydroflavonol 4-reductase, anthocyanindin synthase) each with strong RBS (Fig.16) This operon catalyses the reaction from naringenin to pelargonidin (Fig. 17).
The F3H gene from Petroselinum crispum and the DFR gene from Dianthus gratianopolitanus were kindly provided by Dr. Stefan Martens (Research and Innovation Centre, Fondazione Edmund Mach, Italy). The ANS from Fragaria x ananassa was E.coli codon optimized and synthezised by Eurofins MWG Operon .
Futhermore, We engineered an improved anthocyanidin synthase ( eANS ) by using rational design methods.
Figure 16: Genetic map of pelargonidin producing operon (K1497015). R: RBS; F3H: flavonon-3beta-hydroxylase; DFR: dihydroflavonol 4-reductase; ANS: anthocyanindin synthase.
Figure 17: Pathway of the pelargonidin operon. Starting with naringenin and ending with pelargonidin.
Production
To analyze the pelargonidin production operon ( K1497015 ), we transformed it into E.coli BL21 (DE3). An overnight LB culture was used to inoculate an expression-culture. The expression of pelargonidin was performed according to Yan et al., (2007). After the induction with 1mM Isopropyl-beta-D-thiogalactopyranosid (IPTG) E.coli BL21 (DE3) cells were transferred into M9 media and fermented for 48h at 30°C in the presence of 0.1mM naringenin.
Figure 18: E.coli BL21 (DE3) pellet containing the pelargonidin producing operon after the fermentation. According to Yan et al. (2007) a pelargonidin producing E.coli should be red after a pelargenidin production. The operon with the engineered anthocyanindin synthase produces more pelargonidin.
Extraction
After the production of pelargonidin producing operon with eANS ( K1497015 ) in the presence of 0.5mM narigenin, we performed an extraction of pelargonidin with methanol /dichloromethane from the pellet. We verified the pH-dependency of pelargonidin color (Fig. 19).
Figure 19: Extracted pelargonidin from E.coli BL21 (DE3) under day light. The color of pelargonidin depends on pH value and solvent. This indicates the present of pelargonidin. Left: Methanol extraction; right: Dichlormethane extraction.
Application
Construction of Grätzel cells
We used the extracted Pelargonidin (produced from E.coli BL21 (DE3) containing the pelargonidin producing operon, K1497015) to build a Grätzel cell . This was done according to Michael Grätzel & Brian O'Reagan . We were able to produce a voltage of 380 mV and a current 0.5 mA under daylight conditions. 
Figure 20: Schematic representation of a Grätzel cell
Figure 21: Constructed Grätzel cell with extracted pelargonidin. We were able to produce a voltage of 380 mV and 0.5 mA under day light conditions.
Retrieved from " http://2014.igem.org/Team:TU_Darmstadt/Results/Pathway "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
