<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_TU_Darmstadt_Results_Pathway"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:TU_Darmstadt/Results/Pathway">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:TU_Darmstadt/Results/Pathway&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:TU_Darmstadt/Results/Pathway&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:TU_Darmstadt/Results/Pathway&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:TU_Darmstadt/Results/Pathway" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:TU Darmstadt/Results/Pathway</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><P><TITLE>Home</TITLE></P><DIV id="allWrap"><DIV id="headerBG"><DIV id="headerWrap" class="container_24"><DIV id="topBar" class="grid_18"><UL class="top"><LI class="active first"><A href="https://2014.igem.org/Team:TU_Darmstadt">Home</A></LI><LI><A href="https://2014.igem.org/Team:TU_Darmstadt/Project">Project</A></LI><LI><A href="https://2014.igem.org/Team:TU_Darmstadt/Results">Results</A></LI><LI><A href="https://2014.igem.org/Team:TU_Darmstadt/PolicyandPractices">Policy &amp; Practices</A></LI><LI><A href="https://2014.igem.org/Team:TU_Darmstadt/Achievements">Achievements</A></LI><LI><A href="https://2014.igem.org/Team:TU_Darmstadt/Notebook">Notebook</A></LI><LI><A href="https://2014.igem.org/Team:TU_Darmstadt/Team">Team</A></LI><LI><A href="https://2014.igem.org/Team:TU_Darmstadt/Gallery">Gallery</A></LI><LI><A href="https://2014.igem.org/Team:TU_Darmstadt/Sitemap">Sitemap</A></LI><LI class="last"><A href="https://2014.igem.org/Team:TU_Darmstadt/Contact">Contact</A></LI></UL></DIV></DIV></DIV><DIV id="topWrap" class="container_24"><UL class="slides grid_18"><LI class="slide-container"><DIV class="nav"><LABEL for="img-6" class="prev">‹</LABEL><LABEL for="img-2" class="next">›</LABEL></DIV></LI><LI class="slide-container"><DIV class="nav"><LABEL for="img-1" class="prev">‹</LABEL><LABEL for="img-3" class="next">›</LABEL></DIV></LI><LI class="slide-container"><DIV class="nav"><LABEL for="img-2" class="prev">‹</LABEL><LABEL for="img-4" class="next">›</LABEL></DIV></LI><LI class="slide-container"><DIV class="nav"><LABEL for="img-3" class="prev">‹</LABEL><LABEL for="img-5" class="next">›</LABEL></DIV></LI><LI class="slide-container"><DIV class="nav"><LABEL for="img-4" class="prev">‹</LABEL><LABEL for="img-6" class="next">›</LABEL></DIV></LI><LI class="slide-container"><DIV class="nav"><LABEL for="img-5" class="prev">‹</LABEL><LABEL for="img-1" class="next">›</LABEL></DIV></LI></UL><DIV id="promoWrap" class="grid_6"><DIV class="grid_6 promo1"><A href="https://2014.igem.org/Team:TU_Darmstadt/Achievements/Parts" target="_top"><H1>Our Biobricks</H1><P>Look at our bricks!</P></A></DIV><DIV class="grid_6 promo2"><A href="https://2014.igem.org/Team:TU_Darmstadt/Results/Modeling/ANS_Engineering" target="_top"><H1>eANS</H1><P>Check out the best protein engineering model!</P></A></DIV><DIV class="grid_6 promo3"><A href="https://2014.igem.org/Team:TU_Darmstadt/Team" target="_top"><H1>The People</H1><P>See the people behind the project!</P></A></DIV></DIV></DIV><DIV id="contentWrap" class="container_24"><DIV id="footerBG"><DIV id="footerInner"><DIV id="footerWrap" class="container_24"><DIV class="grid_4"><UL class="menu"><LI class="first"><A href="https://2014.igem.org/Team:TU_Darmstadt">iGEM TUD</A></LI><LI><A href="https://2014.igem.org/Team:TU_Darmstadt/Sitemap">Sitemap</A></LI><LI><A href="https://2014.igem.org/Team:TU_Darmstadt/Team">Team</A></LI><LI><A href="https://2014.igem.org/Team:TU_Darmstadt/Contact">Kontakt</A></LI><LI class="last"><A href="https://2014.igem.org/Team:TU_Darmstadt/Team/Sponsors">Sponsoren</A></LI></UL></DIV></DIV></DIV></DIV><DIV id="contentWrap" class="container_24"><DIV id="leftNavi" class="grid_5"><NAV><UL class="menu"><LI class="first"><A href="https://2014.igem.org/Team:TU_Darmstadt">Home</A></LI><LI><A href="https://2014.igem.org/Team:TU_Darmstadt/Project">Project</A></LI><LI class="active"><A href="https://2014.igem.org/Team:TU_Darmstadt/Results">Results</A></LI><LI class="active"><A href="https://2014.igem.org/Team:TU_Darmstadt/Results/Pathway">Pathway</A></LI><LI><A href="https://2014.igem.org/Team:TU_Darmstadt/Results/Scaffold">Scaffold</A></LI><LI><A href="https://2014.igem.org/Team:TU_Darmstadt/Results/Modeling">Modeling</A></LI><LI><A href="https://2014.igem.org/Team:TU_Darmstadt/Results/Safety">Safety</A></LI><LI class="last"><A href="https://2014.igem.org/Team:TU_Darmstadt/Results/Open_Hardware">Open Hardware</A></LI></UL><LI><A href="https://2014.igem.org/Team:TU_Darmstadt/PolicyandPractices">Policy &amp; Practices</A></LI><LI><A href="https://2014.igem.org/Team:TU_Darmstadt/Achievements">Achievements</A></LI><LI><A href="https://2014.igem.org/Team:TU_Darmstadt/Notebook">Notebook</A></LI><LI><A href="https://2014.igem.org/Team:TU_Darmstadt/Team">Team</A></LI><LI class="last"><A href="https://2014.igem.org/Team:TU_Darmstadt/Gallery">Gallery</A></LI></NAV></DIV><DIV id="wikicontent" class="grid_19"><H1>Overview</H1><P>We constructed a biosynthetic pathway consisting of seven enzymes (TAL, 4CL, CHS, CHI, F3H, DFR, ANS), which were split into two separate operons. In addition we designed a biosensor, which allowed us to measure the concentration of the key intermediate naringenin.
</P><P>Now, we are proud to present the following results:
</P><P>1. Our <B>naringenin operon</B> (<A href="http://parts.igem.org/Part:BBa_K1497016" title="Link to the iGEM registry" target="_blank" class="external-link-new-window">K1497016</A>) works well.
</P><P>2. Our <B>naringenin biosensor</B> (<A href="http://parts.igem.org/Part:BBa_K1497019" title="Link to the iGEM registry" target="_blank" class="external-link-new-window">K1497019</A> - <A href="http://parts.igem.org/Part:BBa_K1497022" title="Link to the iGEM registry" target="_blank" class="external-link-new-window">K1497022</A>) works in several constructions.
</P><P>3. We engineered an <B>improved anthocyanidin synthase</B><A href="http://parts.igem.org/Part:BBa_K1497002" title="Link to the iGEM registry" target="_blank" class="external-link-new-window">eANS (K1497002)</A> by using rational design methods.
</P><P>4. Our <B>pelargonidin operon</B> (<A href="http://parts.igem.org/Part:BBa_K1497015" title="Link to the iGEM registry" target="_blank" class="external-link-new-window">K1497015</A>) works well and we could extract the desired pelargonidin.
</P><P>5. We constructed a functional <B><A href="https://2014.igem.org/Team:TU_Darmstadt/Project/Graetzel_Cell" title="Internal Link" target="_blank" class="external-link-new-window">Grätzel cell (DSC)</A></B> with our produced and extracted pelargonidin.</P><H1>Naringenin operon (K1497016)</H1><P>This part is a composite of four genes each with the strong RBS (<A href="http://parts.igem.org/Part:BBa_B0034" title="Link to the iGEM registry" target="_blank" class="Link to the iGEM registry">BBa_B0034</A>).
</P><P> </P><P>- <A href="http://parts.igem.org/Part:BBa_K1033001" title="Link to the iGEM registry" target="_blank">4-Coumaryl ligase - 4CL (BBa_K1033001)</A></P><P>- <A href="http://parts.igem.org/Part:BBa_K1033000" title="Link to the iGEM registry" target="_blank">Tyrosine ammonia lyase - TAL (BBa_K1033000)</A></P><P>- <A href="http://parts.igem.org/Part:BBa_K1497000" title="Link to the iGEM registry" target="_blank" class="external-link-new-window">Chalcone isomerase - CHI (BBa_K1497000)</A></P><P>- <A href="http://parts.igem.org/Part:BBa_K1497001" title="Link to the iGEM registry" target="_blank" class="external-link-new-window">Chalcone synthase - CHS (BBa_K1497001)</A></P><P> </P><P>Together, these genes define the naringenin biosynthesis operon without a promotor. In addition of a promotor part the device is able to produce S-naringenin. This device is working in <I>E. coli</I> K and B strains.</P><DIV class="contentcenter"><P>
Figure 1: Genetic map of the naringenin operon (K1479016)
</P></DIV><P>Naringenin is produced stereoselectively from tyrosine. Malonyl-CoA is consumed during the biosynthesis process.</P><DIV class="contentcenter"><P>
Figure 2: Pathway of the naringenin operon. Starting with tyrosine and ending with naringenin.
</P></DIV><P> </P><P>We created the naringenin biosynthesis operon under the control of the T7 promoter <A href="http://parts.igem.org/Part:BBa_I712074" title="Link to the iGEM registry" target="_blank">BBa_I712074</A> and the strong constitutive promoter <A href="http://parts.igem.org/Part:BBa_J23100" title="Link to the iGEM registry" target="_blank">BBa_J23100</A>, respectively . We measured the naringenin production after 16 h of incubation with the naringenin biosensor <A href="http://parts.igem.org/Part:BBa_K1497020" title="Link to the iGEM registry" target="_blank">BBa_K1497020</A>.
</P><P>The cell pellets from <I>E. coli</I> BL21(DE3) - pSB1C3-fdeR-gfp with and without T7-naringenin operon (<A href="http://parts.igem.org/Part:BBa_K1497017" title="Link to the iGEM registry" target="_blank">BBa_K1497017</A>) are shown in figure 3. Only the cell transformed with <A href="http://parts.igem.org/Part:BBa_K1497017" title="Link to the registry" target="_blank">BBa_K1497017</A> exhibited GFP fluorescence. 
</P><P>We were also able to measure the GFP fluorescence quantitatively and to calculate the production yield of both operons with the help of a calibration curve  for the naringenin sensor (Figure 4). We calculated a concentration of 1.9 µM (<A href="http://parts.igem.org/Part:BBa_K1497016" title="Link to the iGEM registry" target="_blank">BBa_K1497016</A>) and 3 µM (<A href="http://parts.igem.org/Part:BBa_K1497017" title="Link to the iGEM registry" target="_blank">BBa_K1497017)</A> naringenin repectively. </P><DIV class="contentcenter"><P>
Figure 3: Cell pellets with and without T7-Naringenin operon from <I>E. coli</I> BL21(DE3)-pSB1C3-fdeR-gfp. By using ultraviolet light the pellet containing the naringenin operon shows a GFP fluorescence.
</P></DIV><DIV class="contentcenter"><P>
Figure 4:  Fluorescence of cells with and without the T7-naringenin operon BBa_K1497017 from <I>E. coli</I> BL21(DE3)-pSB1C3-fdeR-gfp and J23100-naringenin operon (BBa_K1497016) from <I>E. coli</I> Top10-pSB1C3-fdeR-gfp, respectively. <I>E. coli</I> BL21(DE3)-pSB1C3-fdeR-gfp without T7-naringenin operon showed no detectable fluorescence. Only in the cells with the functional operon is the GFP fluorescence measurable. The estimated yields are 3 µM for BBa_K1497017 and 1,9 µM for BBa_K1497016. 
</P></DIV><H1>fdeR - Naringenin sensor (K1497019 - K1497022)</H1><P><B>FdeR</B> is a homo dimeric protein from <I>Herbaspirillum seropedicae</I>. In the presence of naringenin (or naringenin chalchone), FdeR activates the specific promoter region upstream of the fdeR region and induces a strong gene expression. In <I>Herbaspirillum seropedicae</I> the FdeR activates the Fde-Operon (Fde: Flavanone degradation) and enable growth with naringenin and the naringenin chalcone.
</P><P>So in combination with GFP or another fluorescense protein this part can be used as an <I>in vivo</I> naringenin sensor.</P><DIV class="contentcenter"><P>
Figure 5: Flow chart of the FdeR activated gfp expression. The constitutive expression of fdeR the FdeR proteins form homodimers. In the presence of naringenin, naringenin molecules bind to the FdeR homodimer and operate a conformational change of the homodimeric FdeR structure. This conformational change activates FdeR, which is now enabled to bind to the uncharacterized promotor domain. Binding to the promotor domain induces expression of genes downstream of the fdeR promoter region.
</P></DIV><P>You can use the reporters for measuring naringenin concentrations in your samples.</P><P>Depending on which fluorophor you want to detect, you can use one of three biosensors:</P><TABLE border="0" cellpadding="0" cellspacing="0" class="contenttable"><TBODY><TR><TD><UL><LI><A href="http://parts.igem.org/Part:BBa_K1497022" title="Link to the iGEM registry" target="_blank" class="external-link-new-window">A: with CFP response use BBa_K1497022</A></LI></UL><UL><LI><A href="http://parts.igem.org/Part:BBa_K1497021" title="Link to the iGEM registry" target="_blank" class="external-link-new-window">B: with mKate response use BBa_K1497021</A></LI></UL><UL><LI><A href="http://parts.igem.org/Part:BBa_K1497019" title="Link to the iGEM registry" target="_blank" class="external-link-new-window">C: with no reporter BBa_K1497019</A></LI></UL><UL><LI><A href="http://parts.igem.org/Part:BBa_K1497020" title="Link to the iGEM registry" target="_blank" class="external-link-new-window">D: with GFP response use BBa_K1497020</A></LI></UL></TD></TR></TBODY></TABLE><DIV class="contentcenter"><P>
Figure 6: <I>E. coli</I> Top10 with different Naringenin biosensors. Left: On agar plate without naringenin no colour is visible. Middle: On agar plate with 100 µM naringenin colour is visible, except of negative sample BBa_K1497019 without fluorophor. Right: On agar plate with 100 µM Naringenin under UV light. The fluorescence of GFP, CFP and mKate is visible.
</P></DIV><P>You can create your own naringenin sensor or your own naringenin dependent gene expression device as well. For these reasons use the Biobrick <A href="http://parts.igem.org/Part:BBa_K1497019" title="Link to the iGEM registry" target="_blank">K1497019</A> and clone your parts of interest (without RBS!) behind the device.
</P><P>In addition, we proved the expression of GFP by running a SDS-PAGE (12,5% polyacrylamide gel).</P><DIV class="contentcenter"><P>
Figure 7: SDS-PAGE of FdeR-GFP-pSB1C3 in BL21 using different naringenin concentrations. We used &quot;Promega Broad Range Protein Marker&quot; and a 12,5% acrylamide gel. GFP has a mass of 26.9 kDa.
</P></DIV><P><SPAN style="color: rgb(40, 40, 40); font-family: arial, sans-serif; font-size: 13px; line-height: 19.5px; ">The Biobrick </SPAN><SPAN style="text-decoration: none; color: rgb(0, 43, 184); font-family: arial, sans-serif; font-size: 13px; line-height: 19.5px; background-image: none; background-attachment: initial; background-size: initial; background-origin: initial; background-clip: initial; background-position: initial; background-repeat: initial; "><A href="http://parts.igem.org/Part:BBa_K1497019" target="_blank">BBa_K1497019</A></SPAN><SPAN style="color: rgb(40, 40, 40); font-family: arial, sans-serif; font-size: 13px; line-height: 19.5px; "> produces in </SPAN><I style="color: rgb(40, 40, 40); font-family: arial, sans-serif; font-size: 13px; line-height: 19.5px; ">E. coli</I><SPAN style="color: rgb(40, 40, 40); font-family: arial, sans-serif; font-size: 13px; line-height: 19.5px; "><SPAN style="color: rgb(40, 40, 40); font-family: arial, sans-serif; font-size: 13px; line-height: 19.5px; "> B and K strains the FdeR Protein. We measured the fluorescense of GFP and mKate after the incubation with different concentrations of naringenin. The results are shown in Figure 8 &amp; 9.</SPAN></SPAN></P><DIV class="contentcenter"><P>
Figure 8: Characterization of BBa_K1497020. GFP fluorescence depends on the concentration of naringenin. We measured the GFP fluorescence after 16 h incubation with different concentrations of naringenin. By setting higher concentrations of naringenin, we gained higher fluorescence of GFP as well.
</P></DIV><DIV class="contentcenter"><P>
Figure 9: Characterization of BBa_K1497021. mKate (BBa_K1055000) fluorescence depends on the concentration of naringenin. We measured the mKate (BBa_K1055000) fluorescence after 16 h incubation with different concentrations of Naringenin. By setting higher concentrations of naringenin, we gained higher fluorescence of mKate as well.
</P></DIV><H1>eANS (K1497002)</H1><P>The anthocyanidin synthase from <I>Fragaria x ananassa</I> (ANS, EC 1.14.11.19) catalyzes many reactions in the anthocyanidin pathway. We used its functionality by catalyzing the conversion of the leucoanthocyanidin (2R,3S,4S)-cis-lucopelargonidin to the anthocyanidin pelargonidin. It also catalyzes the conversion of the leucoanthocyanidin to flavonol (kampferol). In order to avoiding this side reaction and enhance metabolic flux, we designed a protein scaffold (<A href="http://parts.igem.org/Part:BBa_K1497031" title="Opens external link in new window" target="_blank" class="external-link-new-window">BBa_K1497031</A>).
</P><P>Earlier studies hypothesized that ANS may be involved in metabolic channeling in their native organisms. By modelling the mechanical movements of ANS, we discovered a strong flexibility at the C-terminus. Subsequently we modelled ANS' structure and its movements. Thereby we detected a tail at the C-terminus of the enzyme fluctuating and covering the active site (Fig. 15). In oder to improve the enzyme we decided to remove the tail and to construct our pelargonidin operon with this engineered ANS (eANS). The laboratory results confirmed the previous modeling results. The engineered ANS exhibited better yields than the original one when used in an operon producing pelargonidin (BBa_K1497015).</P><DIV class="contentcenter"><P>
Figure 10: Workflow of the eANS rational design approach
</P></DIV><P>In order to optimize the metabolic channeling of ANS, we chose a rational protein engineering approach. The first step of our multi scale and rational engineering project was the creation ofa sophisticated 3D model with <A href="http://www.yasara.org/" title="Link to yasara.org" target="_blank" class="external-link-new-window">YASARA</A> structure. This model was then used for a structural refinement with the <A href="http://www1.jcsg.org/scripts/prod/scwrl/serve.cgi" title="Link to jcsg.org" target="_blank" class="external-link-new-window">SCWRL</A> alghorithm and was energy minimized with YASARA nova force field.Afterwards, we started a true mechanical engineering approach to determine the movements within the protein. Therefore, a Gaussian Network Model (GNM) (Fig. 11) and an Anisotropic Network Model (Fig. 12) were implemented.Those are simple models which simulate the mechanical behavior of the protein. Moreover, Linear Response Theory (LRT) (Fig. 14) was used to simulate the substrate binding inside the pocket and thus trigger an induced fit mechanism. Afterwards we collect our data, defined rational mutations and finally constructed eANS. With this eANS version another MD simulation was started and the sequence of the protein was given to the wetlab for <I>in vitro</I> construction and <I>in vivo</I> characterization.</P><DIV class="contentcenter"><P>
Figure 11: GNM computation of ANS shows a great peak at the C terminus. It leads to the assumption that the C terminal region of the ANS is highly flexible.
</P></DIV><DIV class="contentcenter"><P>
Figure 12: ANM of wild-type ANS shows its flexibility
</P></DIV><DIV class="contentcenter"><P>
Figure 13: We observed a strong deformation of the enzyme in the LRT model. This result reveals that the C-terminal region of the ANS is highly flexible during the process of substrate binding (induced fit).
</P></DIV><DIV class="contentcenter"><P>
Figure 14: LRT model simulates the substrate binding in the active site of ANS by applying a force vector to the active site and binding region.
</P></DIV><DIV class="contentcenter"><P>
Figure 15: Elastic Network Model (ENM) shows the fluctuating tail covering the active site of ANS
</P></DIV><H1>Pelargonidin operon (K1497015)</H1><P style="margin-bottom: 0.35cm; line-height: 115%"><SPAN lang="en-US">We constructed a pelargonidin producing operon under the control of a T7 promoter</SPAN><SPAN lang="en-US"> (</SPAN><SPAN lang="en-US">K1497015)</SPAN><SPAN lang="en-US">. </SPAN><SPAN lang="en-US">The operon consists of 3 genes (flavonon-3beta-hydroxylase, dihydroflavonol 4-reductase, anthocyanindin synthase) each with strong RBS (Fig.16)</SPAN><SPAN lang="en-US"> This </SPAN><SPAN lang="en-US">operon catalyses the reaction from naringenin to </SPAN><SPAN lang="en-US"><SPAN style="background-image: initial; background-attachment: initial; background-size: initial; background-origin: initial; background-clip: initial; background-position: initial; background-repeat: initial; ">pelargonidin (Fig. 17). </SPAN></SPAN></P><P style="margin-bottom: 0.35cm; line-height: 115%"><SPAN lang="en-US"><SPAN style="background-image: initial; background-attachment: initial; background-size: initial; background-origin: initial; background-clip: initial; background-position: initial; background-repeat: initial; ">The F3H gene from </SPAN></SPAN><SPAN style="display: inline-block; border: none; padding: 0cm"><SPAN lang="en-US"><I><SPAN style="background-image: initial; background-attachment: initial; background-size: initial; background-origin: initial; background-clip: initial; background-position: initial; background-repeat: initial; ">Petroselinum crispum </SPAN></I></SPAN><SPAN lang="en-US"><SPAN style="background-image: initial; background-attachment: initial; background-size: initial; background-origin: initial; background-clip: initial; background-position: initial; background-repeat: initial; ">and the DFR gene from </SPAN></SPAN></SPAN><SPAN lang="en-US"><SPAN style="background-image: initial; background-attachment: initial; background-size: initial; background-origin: initial; background-clip: initial; background-position: initial; background-repeat: initial; ">Dianthus gratianopolitanus</SPAN></SPAN><SPAN lang="en-US"><SPAN style="background-image: initial; background-attachment: initial; background-size: initial; background-origin: initial; background-clip: initial; background-position: initial; background-repeat: initial; "> were kindly provided by </SPAN></SPAN><SPAN lang="en-US"><SPAN style="background-image: initial; background-attachment: initial; background-size: initial; background-origin: initial; background-clip: initial; background-position: initial; background-repeat: initial; ">Dr</SPAN></SPAN><SPAN lang="en-US"><SPAN style="background-image: initial; background-attachment: initial; background-size: initial; background-origin: initial; background-clip: initial; background-position: initial; background-repeat: initial; ">. Stefan Martens (Research and Innovation Centre, Fondazione Edmund Mach, Italy). The ANS from </SPAN></SPAN><SPAN style="display: inline-block; border: none; padding: 0cm"><SPAN lang="en-US"><I><SPAN style="background-image: initial; background-attachment: initial; background-size: initial; background-origin: initial; background-clip: initial; background-position: initial; background-repeat: initial; ">Fragaria x ananassa </SPAN></I></SPAN><SPAN lang="en-US"><SPAN style="background-image: initial; background-attachment: initial; background-size: initial; background-origin: initial; background-clip: initial; background-position: initial; background-repeat: initial; ">was </SPAN></SPAN><SPAN lang="en-US"><I><SPAN style="background-image: initial; background-attachment: initial; background-size: initial; background-origin: initial; background-clip: initial; background-position: initial; background-repeat: initial; ">E.coli</SPAN></I></SPAN><SPAN lang="en-US"><SPAN style="background-image: initial; background-attachment: initial; background-size: initial; background-origin: initial; background-clip: initial; background-position: initial; background-repeat: initial; "> codon optimized and synthezised by <A href="http://www.eurofinsgenomics.eu/" title="Link" to="" eurofinsgenomics.eu="">Eurofins MWG Operon</A>. </SPAN></SPAN></SPAN></P><P lang="en-US" style="margin-bottom: 0.35cm; line-height: 115%"><SPAN style="display: inline-block; border: none; padding: 0cm"><SPAN lang="en-US"><SPAN style="background-image: initial; background-attachment: initial; background-size: initial; background-origin: initial; background-clip: initial; background-position: initial; background-repeat: initial; ">Futhermore, </SPAN></SPAN></SPAN>We engineered an improved anthocyanidin synthase (<A href="http://parts.igem.org/Part:BBa_K1497002" title="Opens external link in new window" target="_blank" class="external-link-new-window">eANS</A>) by using rational design methods.</P><DIV class="contentcenter"><P>
Figure 16: Genetic map of pelargonidin producing operon (K1497015). R: RBS; F3H: flavonon-3beta-hydroxylase; DFR: dihydroflavonol 4-reductase; ANS: anthocyanindin synthase.
</P></DIV><DIV class="contentcenter"><P>
Figure 17: Pathway of the pelargonidin operon. Starting with naringenin and ending with pelargonidin.
</P></DIV><P><B>Production</B></P><P>To analyze the pelargonidin production operon (<A href="http://parts.igem.org/Part:BBa_K1497015" title="Link to the iGEM registry" target="_blank" class="external-link-new-window">K1497015</A>), we transformed it into <I>E.coli</I> BL21 (DE3). An overnight LB culture was used to inoculate an expression-culture. The expression of pelargonidin was performed according to Yan <I>et al.</I>, (2007). After the induction with 1mM Isopropyl-beta-D-thiogalactopyranosid (IPTG) <I>E.coli</I> BL21 (DE3) cells were transferred into M9 media and fermented for 48h at 30°C in the presence of 0.1mM naringenin.</P><DIV class="contentcenter"><P>
Figure 18: <I>E.coli</I> BL21 (DE3) pellet containing the pelargonidin producing operon after the fermentation. According to Yan <I>et al.</I> (2007) a pelargonidin producing <I>E.coli</I> should be red after a pelargenidin production. The operon with the engineered anthocyanindin synthase produces more pelargonidin.
</P></DIV><P><B>Extraction</B></P><P>After the production of pelargonidin producing operon with eANS (<A href="http://parts.igem.org/Part:BBa_K1497015" title="Link to the iGEM registry" target="_blank" class="external-link-new-window">K1497015</A>) in the presence of 0.5mM narigenin, we performed an extraction of pelargonidin with methanol /dichloromethane from the pellet. We verified the pH-dependency of pelargonidin color (Fig. 19).</P><DIV class="contentcenter"><P>
Figure 19: Extracted pelargonidin from <I>E.coli</I> BL21 (DE3) under day light. The color of pelargonidin depends on pH value and solvent. This indicates the present of pelargonidin. Left: Methanol extraction; right: Dichlormethane extraction.
</P></DIV><H1>Application</H1><P><B>Construction of Grätzel cells</B></P><P>We used the extracted Pelargonidin (produced from <I>E.coli</I> BL21 (DE3) containing the pelargonidin producing operon, K1497015) to build a <A href="https://2014.igem.org/Team:TU_Darmstadt/Project/Graetzel_Cell" title="Opens internal link in current window" class="internal-link">Grätzel cell</A>. This was done according to <A href="http://www.nature.com/nature/journal/v353/n6346/abs/353737a0.html" title="Link to nature.com" target="_blank" class="external-link-new-window">Michael Grätzel &amp; Brian O'Reagan</A>. We were able to produce a voltage of <B>380 mV</B> and a current <B>0.5 mA</B> under daylight conditions. </P><DIV class="contentcenter"><P>
Figure 20: Schematic representation of a Grätzel cell
</P></DIV><DIV class="contentcenter"><P>
Figure 21: Constructed Grätzel cell with extracted pelargonidin. We were able to produce a voltage of 380 mV and 0.5 mA under day light conditions.
</P></DIV></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:TU_Darmstadt/Results/Pathway">http://2014.igem.org/Team:TU_Darmstadt/Results/Pathway</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:TU_Darmstadt/Results/Pathway" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:TU_Darmstadt/Results/Pathway" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:TU_Darmstadt/Results/Pathway&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:TU_Darmstadt/Results/Pathway&amp;oldid=391980" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></BODY></HTML>