"
Page
Discussion
View source
History
teams
Log in
 
Team:TU Darmstadt/Project/Safety
From 2014.igem.org
Home
Home
Project
Results
Policy & Practices
Achievements
Notebook
Team
Gallery
Sitemap
Contact
‹
›
‹
›
‹
›
‹
›
‹
›
‹
›
Our Biobricks
Look at our bricks!
eANS
Check out the best protein engineering model!
The People
See the people behind the project!
iGEM TUD
Sitemap
Team
Kontakt
Sponsoren
Home
Project
Overview
Anthocyanins
Grätzel cell / DSC
Pathway
Scaffold
Safety
Results
Policy & Practices
Achievements
Notebook
Team
Gallery
Introduction
It is important to address the concerns and possible consequences of intentional or accidental release into the environment when working with genetically engineered systems. These issues were already under discussion in the early years of Genetic Engineering and the Asilomar Conference in 1975 came to the conclusion that containment should always be made an important consideration in the design of the experiment. Furthermore should the effectiveness of the containment approach match the estimated risk as closely as possible [1]. These principles still hold up until today and their importance even increased as the technology in recent years has shifted from the use of transgenic  organisms to designed synthetically constructs.
The goal of our iGEM safety project 2014 is to produce anthocyans in E.coli under laboratory or industrial conditions. Therefore our bacteria are not intended to be released into the environment and our considerations about biosafety are focused on preventing proliferation in case of an accidental release. To do so we decided to follow an active strategy for containment by designing a suicide construct that should prevent the survival of our cells if they get released into the environment but still does not limit the bacteria growth under controlled laboratory conditions. The main item of our suicide construct is the hokD gene of E. coli which transcribes for a small polypeptide that results in cell death by elimination of vital cell wall functions if over expressed. The hokD gene is located in the E. coli chromosome as part of the relB operon. It’s gene product shows 40% homology to the polypeptide encoded by the hok (host killing) gene and induces the same characteristic cell deaths when over expressed [2]. It has been shown that the hokD gene can be used to design efficient suicide functions to contain bacteria growth. The efficiency of the containment system hereby is limited only by the mutation rate of the designed construct and the reduced growth rate resulting from a low basal level of hokD-expression [3, 4]. To avoid unwanted cell death due to basal expression we want to design a suicide backbone in wich hokD is under control of a T3 promotor (phi4.3) but the expression of T3-polymerase itself is controlled by an AraC-repressed pBad-promotor (BBa_K808000). In the presence of glucose the AraC repressor tightly inhibits the expression of T3-Polymerase and expression of hokD is inhibited so that the bacteria can grow with normal growth rate under controlled conditions [5]. In case of a release into the environment and upon depletion of glucose T3-polymerase is produced and the basal expression level of the T3 promotor results in expression of hokD and inhibits the proliferation of the bacteria.
Fig. 1: Schematics of our envisioned killing switch.
To evaluate the basal and induced expression levels of our suicide backbone we will first engineer a test backbone (pSB1C3-YAT) in which hokD is replaced by eYFP (BBa_E0030) (Fig. 2). By measuring the fluorescence level under different glucose and arabinose concentrations we are able to determine whether our construct is regulated enough to prevent unwanted cell death and the efficiency of containment after induction.
Fig. 2: Map of the planned test-plasmid (pSB1C3-YAT).
Please visit also our Safety Form for more information about our safety procedures and precautions during the labwork.
References
1.    Berg, P., et al., Summary statement of the Asilomar conference on recombinant DNA molecules. Proc Natl Acad Sci U S A, 1975. 72(6): p. 1981-4.
2.    Gerdes, K., et al., Mechanism of postsegregational killing by the hok gene product of the parB system of plasmid R1 and its homology with the relF gene product of the E. coli relB operon. Embo j, 1986. 5(8): p. 2023-9.
3.    Knudsen, S.M. and O.H. Karlstrom, Development of efficient suicide mechanisms for biological containment of bacteria. Appl Environ Microbiol, 1991. 57(1): p. 85-92.
4.    Knudsen, S., et al., Development and testing of improved suicide functions for biological containment of bacteria. Appl Environ Microbiol, 1995. 61(3): p. 985-91.
5.    Saida, F., et al., Expression of highly toxic genes in E. coli: special strategies and genetic tools. Curr Protein Pept Sci, 2006. 7(1): p. 47-56.
Retrieved from " http://2014.igem.org/Team:TU_Darmstadt/Project/Safety "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
