<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_NTU-LIHPAO-Taiwan_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:NTU-LIHPAO-Taiwan/Results</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>NTU-LIHPAO-Taiwan</TITLE></P><DIV id="Wrapper"><DIV id="Wrapper_TopPic"><DIV class="Slidemenu"><UL><LI><DIV class="width_small"><A href="https://2015.igem.org/Team:NTU-LIHPAO-Taiwan">Home</A></DIV></LI><LI><DIV class="width_small" span="" style="cursor:default"><A>Team</A></DIV></LI><LI><A href="https://2015.igem.org/Team:NTU-LIHPAO-Taiwan/Team">Team</A></LI><LI><A href="https://2015.igem.org/Team:NTU-LIHPAO-Taiwan/Attributions">Attributions</A></LI></UL><LI><DIV class="width_small" span="" style="cursor:default"><DIV id="Position_Now"><A>Project</A></DIV></DIV></LI><LI><A href="https://2015.igem.org/Team:NTU-LIHPAO-Taiwan/Description">Description</A></LI><LI><A href="https://2015.igem.org/Team:NTU-LIHPAO-Taiwan/Design">Design</A></LI><LI><A href="https://2015.igem.org/Team:NTU-LIHPAO-Taiwan/Results">Results</A></LI><LI><A href="https://2015.igem.org/Team:NTU-LIHPAO-Taiwan/Modeling">Modeling</A></LI><LI><A href="https://2015.igem.org/Team:NTU-LIHPAO-Taiwan/Experiments">Protocols</A></LI><LI><DIV class="width_small" span="" style="cursor:default"><A>Parts</A></DIV></LI><LI><A href="https://2015.igem.org/Team:NTU-LIHPAO-Taiwan/Parts">Team Parts</A></LI><LI><A href="https://2015.igem.org/Team:NTU-LIHPAO-Taiwan/Basic_Part">Basic Parts</A></LI><LI><A href="https://2015.igem.org/Team:NTU-LIHPAO-Taiwan/Composite_Part">Composite Parts</A></LI><LI><DIV class="width_small"><A href="https://2015.igem.org/Team:NTU-LIHPAO-Taiwan/Notebook">Notebook</A></DIV></LI><LI><DIV class="width_small"><A href="https://2015.igem.org/Team:NTU-LIHPAO-Taiwan/Safety">Safety</A></DIV></LI><LI><DIV class="width_small" span="" style="cursor:default"><A>Society</A></DIV></LI><LI><A href="https://2015.igem.org/Team:NTU-LIHPAO-Taiwan/Practices">Human Practices</A></LI><LI><A href="https://2015.igem.org/Team:NTU-LIHPAO-Taiwan/Collaborations">Collaborations</A></LI><LI><A href="https://2015.igem.org/Team:NTU-LIHPAO-Taiwan/Entrepreneurship">Entrepreneurship</A></LI></DIV></DIV><DIV id="Content_Container"><DIV id="Content"><UL class="main-Content"><LI><SPAN class="title">CPP-PYY</SPAN></LI><LI><A href="#First1">Fusion protein and test part</A></LI></UL><LI><SPAN class="title">Nisin Selection</SPAN></LI><LI><A href="#Second1">Nisin Immunity Test</A></LI><LI><SPAN class="title">Suicide</SPAN></LI><LI><A href="#Third1">Suicide Kill Part</A></LI><LI><A href="#Third2">Suicide Sence Part</A></LI></DIV></DIV><DIV id="Articles"><DIV class="ContentBox"><DIV class="ContentHolder"><DIV class="Text1">CPP-PYY</DIV><DIV class="Text2" id="First1">Fusion protein and test part</DIV><DIV class="Text3"><DIV class="Text_TitleUnderline">
1.  Plasmid construction
						</DIV></DIV><DIV class="Text3"><B>Signal peptide-PYY-His and His-PYY</B></DIV><DIV class="Text3">
We used EcoRI and PstI to digest our inserts and vectors to produce stick ends, and combined them to make recombinant plasmids. After construction of the plasmids, we used EcoRI and PstI and electrophoresis to check the validity.
						</DIV><DIV class="Container_Article_Picture11"><DIV class="Article_Picture11"><DIV class="Article_PictureText11"><DIV class="Text_Picture">[Fig.1-1] Electrophoresis result <A href="https://2015.igem.org/File:NTU_CPP-PYY1.jpg">[Full Size]</A></DIV></DIV></DIV></DIV><DIV class="Text3">
After digested with EcoRI and PstI, the plasmids were smaller than that were digested with EcoRI. This result show the construction is successful.
						</DIV><DIV class="Text3"><B>TAT-PYY</B></DIV><DIV class="Container_Article_Picture12"><DIV class="Article_Picture12"><DIV class="Article_PictureText12"><DIV class="Text_Picture">[Fig.1-2] Electrophoresis result <A href="https://2015.igem.org/File:NTU_CPP-PYY2.jpg">[Full Size]</A></DIV></DIV></DIV></DIV><DIV class="Text3">
After digested with EcoRI and PstI, the plasmids were smaller than that digested with EcoRI, and there were bands below, which were our inserts (TAT-PYY). The result show the combination of TAT-PYY DNA sequence and pSB1C3 is completed.
						</DIV><DIV class="Text3"><B>Y2R</B></DIV><DIV class="Container_Article_Picture13"><DIV class="Article_Picture13"><DIV class="Article_PictureText13"><DIV class="Text_Picture">[Fig.1-3] Electrophoresis result <A href="https://2015.igem.org/File:NTU_CPP-PYY3.jpg">[Full Size]</A></DIV></DIV></DIV></DIV><DIV class="Text3">
After digested with EcoRI and PstI, there were two kinds of bands which were our vectors (upper bands, pcDNA 3.0) and inserts (lower bands, Y2R). The result mean the construction is successful.
						</DIV><DIV class="Text3"><DIV class="Text_TitleUnderline">
2.  Protein expression
						</DIV></DIV><DIV class="Text3"><B>Signal peptide-PYY-His and negative control</B></DIV><DIV class="Container_Article_Picture14"><DIV class="Article_Picture14"><DIV class="Article_PictureText14"><DIV class="Text_Picture">[Fig.1-4] Western blot result <A href="https://2015.igem.org/File:NTU_CPP-PYY4.jpg">[Full Size]</A></DIV></DIV></DIV></DIV><DIV class="Text3">
We transformed plasmid into BL21 competent cell, extracted total protein, and used western blot to check whether the target protein was overexpressed.

						</DIV><DIV class="Text3">
We used antibody which would bind His-tag, so the band should appear when target protein perform.
						</DIV><DIV class="Text3">
But there are similar bands in both SP and negative control, which mean there are non-specific binding between antibody and other protein, and our target protein didn’t overexpress.
						</DIV><DIV class="Text1">Nisin Selection</DIV><DIV class="Text2" id="Second1">Nisin Immunity Test</DIV><DIV class="Text3"><DIV class="Text_TitleUnderline">
1.  Nisin sensitivity of <I>E. coli</I> transformed with the plasmid expressing <I>nisI</I> in different concentration of nisin
						</DIV></DIV><DIV class="Text3">
The nisin sensitivity of <I>E. coli</I> was tested using agar diffusion method. Overnight cell culture carrying <I>nisI</I> gene was intensely streaked on LB agar plates. Discs containing different concentration of nisin were also put on the plates to examine the sensitivity. The plates were incubated at 37℃ overnight and the diameter of the inhibition zones were measured.
						</DIV><DIV class="Container_Article_Picture21"><DIV class="Article_Picture21"><DIV class="Article_PictureText21"><DIV class="Text_Picture">[Fig.2-1] Nisin immunity test <A href="https://2015.igem.org/File:NTU-Nisin1.png">[Full Size]</A></DIV></DIV></DIV></DIV><DIV class="Container_Article_Picture22"><DIV class="Article_Picture22"><DIV class="Article_PictureText22"><DIV class="Text_Picture">[Fig.2-2] Nisin immunity test <A href="https://2015.igem.org/File:NTU-Nisin2.jpg">[Full Size]</A></DIV></DIV></DIV></DIV><DIV class="Text3">
The result shows that in the low concentration of the nisin, <I>E. coli</I> has the immunity towards nisin. From the pictures shown above, there are no evident zones of inhibition in 0, 20, 50 IU. Therefore, we can assure that <I>E. coli</I> can expel nisin out of the cell body and survive well.
						</DIV><DIV class="Text3"><DIV class="Text_TitleUnderline">
2.  Nisin sensitivity of <I>E. coli</I> which cell wall is weakened by EDTA and transformed with the plasmid expressing <I>nisI</I> in different concentrations of nisin
						</DIV></DIV><DIV class="Text3">
Overnight <I>E. coli</I> cell culture was refreshed to O.D.<SUB>600</SUB> ~ 0.1 and then the LB broth was replaced by cell buffer containing 40 mM of EDTA. 100 µL of cell culture was mixed with the same volume of nisin solution of various concentrations and incubated at 37℃ for 1 hour. After that, nisin solution was removed by centrifuge. The cell culture that went through serial dilution to 10<SUP>-4</SUP> was dispersed on LB agar plates and incubated at 37℃ overnight to calculate the living amount of bacteria.
						</DIV><DIV class="Container_Article_Picture23"><DIV class="Article_Picture23"><DIV class="Article_PictureText23"><DIV class="Text_Picture">[Fig.2-3] Nisin immunity test treated with EDTA <A href="https://2015.igem.org/File:NTU-Nisin3.jpg">[Full Size]</A></DIV></DIV></DIV></DIV><DIV class="Text3">
When treated with EDTA, a chelate that can weaken the outer membrane of <I>E. coli</I>, nisin was more likely to enter the cell body. Furthermore, we can easily deduce the factor, like cell wall thickness that influences the ability of nisin resistancy. From the result above, we can see that the higher concentration of nisin solution, the less the bacterial colonies, which indicates that the gene <I>nisI</I> help <I>E. coli</I> to get the ability to resist nisin.
						</DIV><DIV class="Text1">Suicide</DIV><DIV class="Text2" id="Third1">Suicide Kill Part</DIV><DIV class="Text3"><DIV class="Text_TitleUnderline">
1.  Plasmid construct
						</DIV></DIV><DIV class="Text3">
2015/08/14
						</DIV><DIV class="Text3">
[1] Gel electrophoresis
						</DIV><DIV class="Container_Article_Picture31"><DIV class="Article_Picture31"><DIV class="Article_PictureText31"><DIV class="Text_Picture">[Fig.3-1] pC1-RBS-NucA-Ter-pSB1C3(nucA) <A href="https://2015.igem.org/File:NTU_Suiside1.jpg">[Full Size]</A></DIV></DIV></DIV></DIV><DIV class="Text3">
[2] NucA(B) SEQUENCING
						</DIV><DIV class="Container_Article_Picture32"><DIV class="Article_Picture32"><DIV class="Article_PictureText32"><DIV class="Text_Picture">[Fig.3-2] NucA(B) SEQUENCING <A href="https://2015.igem.org/File:NTU_Suiside2.jpg">[Full Size]</A></DIV></DIV></DIV></DIV><DIV class="Text3">
There is no consistence between our gene and sequencing.
						</DIV><DIV class="Text3">
2015/08/19
						</DIV><DIV class="Text3">
[1] Gel electrophoresis
						</DIV><DIV class="Container_Article_Picture33"><DIV class="Article_Picture33"><DIV class="Article_PictureText33"><DIV class="Text_Picture">[Fig.3-3] pC1-RBS-NucA-Ter-pSB1C3(nucA) <A href="https://2015.igem.org/File:NTU_Suiside3.jpg">[Full Size]</A></DIV></DIV></DIV></DIV><DIV class="Text3">
[2] NucA(C) SEQUENCING
						</DIV><DIV class="Container_Article_Picture34"><DIV class="Article_Picture34"><DIV class="Article_PictureText34"><DIV class="Text_Picture">[Fig.3-4] NucA(C) SEQUENCING <A href="https://2015.igem.org/File:NTU_Suiside4.jpg">[Full Size]</A></DIV></DIV></DIV></DIV><DIV class="Text3">
There is no consistence between our gene and sequencing.
						</DIV><DIV class="Text3">
2015/08/24
						</DIV><DIV class="Text3">
[1] Gel electrophoresis
						</DIV><DIV class="Container_Article_Picture35"><DIV class="Article_Picture35"><DIV class="Article_PictureText35"><DIV class="Text_Picture">[Fig.3-5] pC1-RBS-NucA-Ter-pSB1C3(nucA) <A href="https://2015.igem.org/File:NTU_Suiside5.jpg">[Full Size]</A></DIV></DIV></DIV></DIV><DIV class="Text3">
Result
						</DIV><DIV class="Text3">
From the test 1 and 2, we proof that nuclease A have its function and it can cut the genomic DNA and plasmid DNA, because we found that the sequencing result have no consistence with our gene sequence. From the test 3, we can assure that nuclease A can really cut the DNA because we acquire a lot of weird band in the gel electrophoresis. Maybe the band we get is the genomic DNA which is cut by nuclease A.
						</DIV><DIV class="Text3"><DIV class="Text_TitleUnderline">
2.  <I>E. coli</I> growth curve
						</DIV></DIV><DIV class="Text3">
Furthermore, we compare the wild type <I>E. coli</I> and the <I>E. coli</I> transformed with <I>nucA</I> , that cause the cell death.
						</DIV><DIV class="Container_Article_Picture36"><DIV class="Article_Picture36"><DIV class="Article_PictureText36"><DIV class="Text_Picture">[Fig.3-6] <I>E. coli</I> growth curve
 <A href="https://2015.igem.org/File:NTU_Suiside6.jpg">[Full Size]</A></DIV></DIV></DIV></DIV><DIV class="Container_Article_Picture37"><DIV class="Article_Picture37"><DIV class="Article_PictureText37"><DIV class="Text_Picture">[Fig.3-7] <I>nucA</I> growth curve
 <A href="https://2015.igem.org/File:NTU_Suiside7.jpg">[Full Size]</A></DIV></DIV></DIV></DIV><DIV class="Text3">
From the growth curve, we find a evident difference between the empty plasmid and the suicide plasmid. We can proof the function of the <I>nucA</I>.
						</DIV><DIV class="Text2" id="Third2">Suicide Sence Part</DIV><DIV class="Text3"><DIV class="Text_TitleUnderline">
1.  For <I>L. casei</I></DIV></DIV><DIV class="Text3">
[1] Gel electrophoresis
						</DIV><DIV class="Container_Article_Picture38"><DIV class="Article_Picture38"><DIV class="Article_PictureText38"><DIV class="Text_Picture">[Fig.3-8] plac-RBS-CI-Ter-pSB1C3(Plac) <A href="https://2015.igem.org/File:NTU_Suiside8.jpg">[Full Size]</A></DIV></DIV></DIV></DIV><DIV class="Text3">
[2] Transformation
						</DIV><DIV class="Container_Article_Picture39"><DIV class="Article_Picture39"><DIV class="Article_PictureText39"><DIV class="Text_Picture">[Fig.3-9] Transformation <A href="https://2015.igem.org/File:NTU_Suiside9.jpg">[Full Size]</A></DIV></DIV></DIV></DIV><DIV class="Text3">
From the gel electrophoresis, we make sure the plasmid we construct in the <I>E. coli</I> is correct. However, our transformation in the <I>L. casei</I> always failed in all kinds of electroporation voltage and time.
						</DIV><DIV class="Text3"><DIV class="Text_TitleUnderline">
2.  For <I>E. coli</I></DIV></DIV><DIV class="Text3">
[1] Gel electrophoresis
						</DIV><DIV class="Container_Article_Picture310"><DIV class="Article_Picture310"><DIV class="Article_PictureText310"><DIV class="Text_Picture">[Fig.3-10] plac-RBS-CI-Ter-pSB1C3(PlacE) <A href="https://2015.igem.org/File:NTU_Suiside10.jpg">[Full Size]</A></DIV></DIV></DIV></DIV><DIV class="Text3">
It should have a 3K band in A1 B1 C1, but we found that there are 2k band. Maybe our ligation is not successful. Also we can not find 3K band in A1 B1.
						</DIV><DIV class="Text3">
[2] Gel electrophoresis
						</DIV><DIV class="Container_Article_Picture311"><DIV class="Article_Picture311"><DIV class="Article_PictureText311"><DIV class="Text_Picture">[Fig.3-11] plac-RBS-CI-Ter-pSB1C3(PlacE) <A href="https://2015.igem.org/File:NTU_Suiside11.jpg">[Full Size]</A></DIV></DIV></DIV></DIV><DIV class="Text3">
We can not find any band in A, B, C so we presume that its ligation failed.
In sample D, E, F we found that there are no 2K and 1K band. Therefore, we think ligation failed.
						</DIV><DIV class="Text_Sponsor">
							Maintained by the iGEM team NTU-LIHPAO-Taiwan    ©2015 NTU-LIHPAO-Taiwan
						</DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>