Team:TUDelft/Measurement
ADOPE
Project
Description
Design
Results
Demonstration
Wetlab
Experiments (Protocols)
Notebook
InterLab
Safety
Measurement
Parts
Parts Overview
Basic Parts
Composite Parts
Part Collection
Sequences
Modeling
Model
Software Tool
Improvement
Human Practices
Integrated Human Practices
Education and Engagement
Entrepreneurship
Applied Design
EurAsian Meetup
Team
Team Members
Collaborations
Contributions
Team Building
Acknowledgements
Attributions
Sponsors
Judging form
Medal Requirements
Awards
Overview
The final step in our detection method consists on measuring and analyzing DNA molecules that have been selectively tagged for sequencing. We have succeeded in developing such a method based on the use of our own designed fusion protein dxCas9-Tn5 ( Picelli, et al, 2014., Hu, et al, 2018 ). This page is dedicated to describe how we designed this novel approach, how this was tested and what applications this method can be extended to.
Targeted DNA enrichment during library preparation
We focused on the Oxford Nanopore Technology sequencing platform as the core technology to measure and obtain DNA sequences from samples. In this sequencing platform, DNA molecules diffuse through a protein pore embedded on a silica membrane. Changes in electrical signal due to this DNA diffusion are measured. The specific changes in voltage are sequence dependent, and software tools from ONT can convert this signal into DNA sequences. On one sequencing device (Flow-Cell), more than 2000 active pores can generate data at the same time. The amount of data produced from this type of sequencing is truly gigantic, and efficient data analysis of these measurements is crucial to filter and compare data ( Jain, Oslen, Paten & Akeson, 2016 ).
To control this molecule diffusion through the pore and facilitate the determination of specific sequences, the DNA molecules to be sequenced are attached to a so called motor-protein. This protein regulates the rate of diffusion through the nanopore and is indispensable for this type of sequencing. During library preparation with ONT an enzyme called transposase is used to randomly fragment DNA sequences and add DNA adapters. These DNA adapters are ligated to another DNA molecule attached to the motor protein in a subsequent step ( Jain, Oslen, Paten & Akeson, 2016 ). We aimed to change the fragmentation step of this reaction with a newly designed fusion protein that will make this adapter integration sequence specific. Thus, during library preparation, specific DNA sequences will be favourably adapted and enriched for sequencing, as shown in figure 1.
Figure 1. Differences between random library preparation with transposase from ONT library preparation (left) and targeted library preparation with our fusion protein (right). The principle behind targeted library preparation is to enrich certain DNA targets before the sequencing run.
Enrichment for specific DNA sequences allows more detailed studies and processing of higher number of samples with lower data generation. To make multiplexing easier, we developed a software tool that generates specific DNA adapter sequences needed for ONT sequencing. Each sequence generated can be directly linked to the sample identification. To analyse and use this tool, please visit our improvement page .
Proof of principle
Targeted integration of DNA adapters during library preparation and sequencing with ONT MinION
We constructed, expressed and purified the fusion protein dxCas9-Tn5 . We demonstrated that this protein can integrate DNA adapters on a specific target (using a sgRNA for dxCas9) on human Erythropoyetin coding sequence (EPO cds) as substrate. To see these detailed results, please visit our Demonstration page ). After this result was obtained, we performed the integration reaction with DNA adapters that were compatible with ONT sequencing. These adapters were integrated on the specific EPO cds DNA target. The sample used for the reaction with our fusion protein contained two molecules: EPO cds (the target of fusion) and a different molecule (not targeted by the sgRNA) to assess any off-target adapter integration.
After this integration reaction, the sample was used for library preparation for ONT MinION sequencing. This protocol was started at the second step, as the targeted library preparation with our fusion protein replaced  the first reaction. After sequencing, data was aligned to both DNA molecules as reference. This experiment proved that target EPO cds was enriched for sequencing in comparison with negative control DNA that was not targeted (89 unique aligned reads against 0 unique aligned reads respectively). This result shows sample enrichment towards EPO cds plasmid.
For further proof of targeted integration, figure 2 shows the specific alignment of one of the unique reads from this experiment to reference EPO cds DNA used.
Figure 2. (A) Alignment of DNA sequence read with reference EPO cds in plasmid DNA. (B) Zoom into the sequence alignment to determine sgRNA target. Fusion protein depicted with arrows representing binding of dxCas9 position (red) and the addition of sequencing adapters (blue).
For more information and discussion on these experiments, please visit our sequencing results .
Further improvements and prospective applications
We obtained a proof of principle that indicates how library preparation is enriched towards a specific DNA sequence. However, the method will still need to be optimized before it can be used for detection applications. One of the major improvements to be made is the optimization of the protocol for integration of our fusion protein. Buffer composition must be altered to favour reaction kinetics of both dxCas9 binding to target and Tn5 integration. Another improvement of the method should include the design of optimal adapters compatible with ONT library preparation. This could be done by interacting with sequencing experts from ONT because the specific adapter sequences are not known due to corporate secrets. For our prospective collaboration with Oxford Nanopore technologies, please visit our Entrepreneurship page .
This method was designed and developed while searching for a solution in detection methods for gene doping, but the application scope is much broader than that. By simply interchanging the sgRNA with one of your own interest the target can be changed unlimitedly. The ability to enrich samples for specific DNA molecules prior to sequencing can be used in research areas that require highly detailed information of DNA mutations. Such studies could include single point mutations to large structural variations or copy number alterations, detection of viral infections, fetal DNA screening and food safety maintenance ( Gabrieli, Sharim, Michaeli & Ebanstein, 2017 ).
References
Gabrieli, T., Sharim, H., Michaeli, Y., & Ebenstein, Y. (2017). Cas9-Assisted Targeting of CHromosome segments (CATCH) for targeted nanopore sequencing and optical genome mapping. bioRxiv. https://doi.org/10.1101/110163
Jain, M., Oslen, HE.,  Paten, B., Akeson, M. ( 2016) The Oxford Nanopore MinION: delivery of nanopore sequencing to the genomics community. Genome Biol, 17(1):239.
Hu J.H., Miller S. M., Geurts M. H., Tang W., Chen L., Sun N., Zeina C. M., Gao X., Rees H. A., Lin Z., Liu D. R. (2018). Evolved Cas9 variants with broad PAM compatibility and high DNA specificity. Nature; 556(7699): 57–63. doi: 10.1038/nature26155
Picelli, S., Björklund, Å. K., Reinius, B., Sagasser, S., Winberg, G., & Sandberg, R. (2014). Tn5 transposase and tagmentation procedures for massively-scaled sequencing projects. Genome research, gr-177881.
We thank our sponsors:
