<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_TUDelft_Project_Demonstration skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:TUDelft/Project/Demonstration</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container-fluid"><DIV class="row"><DIV class="col-lg-10 col-md-10 col-sm-12 col-xs-12 nowbg" style="margin-left:30px;"><H1 class="jnnbl">Introduction</H1><DIV class="clearfix"><FIGURE class="lefti1"><FIGCAPTION class="jnnbl"><B>Figure 1.</B> Fusion of Tn5 transposase with dxCas9. During purification, the Tn5 part of the protein forms a dimer with another Tn5 structure from another fusion protein, thus forming the fusion dimer structure represented here in proportion.</FIGCAPTION></FIGURE><P>The core of our detection method is our novel targeted Next Generation Sequencing platform. This technology reduces the enormous amount of data generated by <A class="jnnbl" href="https://2018.igem.org/Team:TUDelft/Applied_Design#wholegenome" target="_blank">whole genome</A> sequencing and reduces the time and resources invested in analyzing multiple samples, for the detection of gene doping. We accomplished our goal by designing an innovative dxCas9-Tn5 fusion protein shown in <B class="jnnbl">figure 1</B>. The dxCas9 part is responsible for guiding the fusion to the target site via complementary matching between the sgRNA and target site. Whereas the Tn5 part is responsible for performing the integration of the adapter. This fusion thus has the capability of performing target specific integration of adapters required for sequencing. We could therefore replace the random enzymatic protein, Tn5, used for fragmentation during sequencing library preparation, transforming the Oxford Nanopore Technologies MinION platform to targeted Next Generation Sequencing. </P></DIV><H1 class="jnnbl">Fusion protein <I>In Vitro</I> functionality</H1><P> We constructed, expressed and purified the fusion protein, and demonstrated its intended activity: <B>targeted integration of specific DNA sequences</B>. For this, we analyzed the integration of small DNA sequences (DNA adapters) directed with a single guide RNA (sgRNA) to a specific site on a substrate gene, generating two DNA fragments of known length based on the position that the sgRNA targeted. </P><FIGCAPTION class="jnnbl"><B>Figure 2. (A)</B> Experimental design to prove targeted integration of our fusion protein. The fusion was directed by sgRNA at position 179 bp in <I>EPO cds</I>, adding two adapters of 50 bp (one at each side). The product of the integration are two separate DNA molecules of different lengths that can be seen by PCR amplification and gel electrophoresis, <B>(B).</B> Tris-Borate-EDTA buffer native 5% Polyacrylamide Gel Electrophoresis (PAGE) stained by EtBr and imaged by GelDoc system. Lane 1: DNA ladder (100-1000bp), lane 2: amplified product with fw EPO lane 3: amplified product with rv EPO, lane 4: amplified product with fw EPO without sgRNA, lane 5: amplified product with rv EPO without sgRNA. </FIGCAPTION><P>The fusion was loaded with sgRNAs designed to target the substrate DNA,  erythropoietin (EPO) coding DNA sequence (cds), at position 179 bp (from a total length of 623 bp). If the integration of DNA adapters was site-specific, two main products should be obtained: ~250 bp and ~470 bp (one from the forward and one from the reverse strand), schematically shown in <B class="jnnbl">figure 2</B>. This integration was demonstrated and presented in <B class="jnnbl">figure 2.B</B>, in which lanes 2 and 3 represent each one of the fragments described. These lanes can be compared to the negative control in lanes 4 and 5. These lanes represent integration in the absence of sgRNA, the component required from targeted integration.</P><H1 class="jnnbl">Targeted Next generation sequencing</H1><P>Not only did we demonstrate that our fusion protein is capable of targeted integration, but we used our protein in its designed application: targeted sequencing with Oxford Nanopore Technologies (ONT) platform. We used our fusion protein during library preparation to integrate sequencing adapters at the target site in the <I>EPO cds</I>, using the same sgRNA from the assay described above. The library reaction mixture had equimolar concentrations of two molecules: </P><UL class="uljnnbl"><LI>Target DNA: EPO linear plasmid DNA containing a complimentary sgRNA target site.</LI><LI>Background DNA:Tn5 cds DNA containing no sgRNA compatible target sites.</LI></UL><P>After <A class="jnnbl" href="https://2018.igem.org/Team:TUDelft/Experiments#sequencingminion-scroll" target="_blank">library preparation</A>, we also added equimolar concentrations of randomly tagged reference DNA, as an internal control to verify the sequencing run. We sequenced the DNA molecules with an ONT MinION device. Sequencing results showed multiple reads of the reference DNA (internal control), 89 unique aligned reads of to the target DNA, EPO linear plasmid, and 0 unique aligned reads to Tn5 cds DNA, background DNA. Demonstrating a successful sequencing run and the target specificity of our fusion protein. Close analysis of one of the alignment reads demonstrates the precise site where the sequencing adapters were added. We could thus identify the exact distance from the binding of dxCas9 to the integration site of the adapters as shown in <B class="jnnbl">figure 3</B>.
 </P><FIGCAPTION class="jnnbl"><B>Figure 3. (A)</B> Alignment of the DNA sequence read with reference <I>EPO cds</I> in plasmid DNA. <B>(B)</B> Zoom-in into the sequence alignment to determine the sgRNA target. Fusion protein depicted with arrows representing binding of dxCas9 position (red) and the addition of sequencing adapters (blue).</FIGCAPTION><H1 class="jnnbl">Conclusion</H1><P>We proved that our designed fusion protein could direct <B>DNA integration to a specific sequence</B> by the use of a sgRNA. Furthermore, these results demonstrate that this novel protein can be used to <B>perform targeted next generation sequencing</B>. We were able to identify 89 sequencing events that align to our target DNA (EPO linear plasmid DNA). We were even able to identify the exact position of adapter integration by our fusion protein based on one single DNA read out of over 200 000 reads (including reads from the internal control). This demonstrates that targeted sequencing with library preparation by our fusion protein significantly simplifies data analysis and could be used to detect gene doping DNA. </P><CENTER><FIGURE><P class="jnnbl" style="text-align:center;font-size:16px !important;"><B>Figure 4.</B> Library preparation and ONT sequencing in the lab.</P></FIGURE></CENTER></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>