<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_TUDelft_Wetlab_Notebook skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:TUDelft/Wetlab/Notebook</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container"><DIV class="row"><DIV class="col-lg-8 col-md-8 col-sm-12 col-xs-12"><P>Please select the the model of which you would like to view the notebooks.</P><LABEL class="containeri">Sample Preparation</LABEL><LABEL class="containeri">Pre-screen</LABEL><LABEL class="containeri">Strain Construction</LABEL><LABEL class="containeri"><I>In Vitro</I></LABEL><LABEL class="containeri"><I>In Vivo</I></LABEL><LABEL class="containeri">Purification</LABEL><LABEL class="containeri">Sequencing</LABEL></DIV></DIV></DIV><DIV class="container nowbg" id="SamplePrep"><DIV class="row nowbg"><DIV class="col-xs-12 col-sm-3 col-lg-3"><NAV id="myScrollspy" class="hidden-xs hidden-sm"><UL class="nav nav-pills nav-stacked" data-spy="affix"><LI class="dropdown"><A class="dropdown-toggle adpbl" data-toggle="dropdown" href="#">September</A></LI><LI><A href="#SeptemberSamplePrep1" class="adpbl">Week: 03/09/2018 - 09/09/2018</A></LI><LI><A href="#SeptemberSamplePrep2" class="adpbl">Week: 10/09/2018 - 16/09/2018</A></LI><LI><A href="#SeptemberSamplePrep3" class="adpbl">Week: 17/09/2018 - 23/09/2018</A></LI><LI><A href="#SeptemberSamplePrep4" class="adpbl">Week: 24/09/2018 - 30/09/2018</A></LI></UL><LI class="drowdown"><A class="dropdown-toggle adpbl" data-toggle="dropdown" href="#">October</A></LI><LI><A href="#OctoberSamplePrep1" class="adpbl">Week: 01/10/2018 - 07/10/2018</A></LI><LI><A href="#OctoberSamplePrep2" class="adpbl">Week: 08/10/2018 - 14/10/2018</A></LI></NAV></DIV><DIV class="col-lg-8 col-md-8 col-sm-12 col-xs-12 nowbg" style="margin-right:20px;"><H3 class="adpbl">September</H3><FORM><LABEL for="WeekSPS1" class="subdropdown cadpbl">Week 1 03/09/2018 - 09/09/2018</LABEL><UL class="submenu"><DIV class="subcontent"><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#WednesdaySPS1">Wednesday, 05/09/2018</BUTTON><DIV id="WednesdaySPS1" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>05/09/2018</DD><DT>Operator</DT><DD>Lisa Büller</DD></DL><H4>Extraction of 3 serum samples</H4><P>cfDNA was extracted from 3 serum samples without any spiking of artifical gene doping DNA. For the extraction the QIAamp DNA Blood Mini Kit <A class="adpbl" href="https://2018.igem.org/Team:TUDelft/Experiments#serumextraction" target="_blank">protocol</A> was used</P><H4>Qubit dsDNA High Sensitivity DNA Quantification</H4><P>The 3 extracted samples were quantified using the Qubit dsDNA High Sensitivity DNA Assay Kit.</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Sample extraction</TH><TH class="tableheaderadpbl">cfDNA Concentration (ng/μL)</TH></TR><TR class="tableunevenadpbl"><TD>Serum 1</TD><TD>0.514</TD></TR><TR class="tableunevenadpbl"><TD>Serum 2</TD><TD>0.502</TD></TR><TR class="tableunevenadpbl"><TD>Serum 3</TD><TD>0.530</TD></TR></TBODY></TABLE></CENTER></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#ThursdaySPS1">Thursday, 06/09/2018</BUTTON><DIV id="ThursdaySPS1" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>06/09/2018</DD><DT>Operator</DT><DD>Lisa Büller</DD></DL><H4>Albumin PCR extractions 05/09</H4><P>GoTaq PCR was done to verify the presence of the albumin gene after extraction of the 3 samples of 05/09/2018. Using the primers creating the small albumin fragment of 153 bp. 
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL)</TH></TR><TR class="tableunevenadpbl"><TD>Samples extr. 1-3 05/09  </TD><TD>3</TD></TR><TR class="tableevenadpbl"><TD>066</TD><TD>0.6</TD></TR><TR class="tableunevenadpbl"><TD>067</TD><TD>0.6</TD></TR><TR class="tableevenadpbl"><TD>Expected amplicon size (bp)</TD><TD>153</TD></TR></TBODY></TABLE></CENTER><P>5 tubes were prepared for the PCR, containing the serum extractions of 05/09/2018 as template DNA, a positive control of pure serum and negative control with MiliQ as template. The thermocycler programme was installed to have an extension time of 1 minute and an annealing temperature of 62 °C, 30 cycles. After the programme was finished, samples were kept at 4°C.</P><P>The gel showed no results, no bands.</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV></DIV></UL></FORM><FORM><LABEL for="WeekSPS2" class="subdropdown cadpbl">Week 2 10/09/2018 - 16/09/2018</LABEL><UL class="submenu"><DIV class="subcontent"><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#MondaySPS2">Monday, 10/09/2018</BUTTON><DIV id="MondaySPS2" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>10/09/2018</DD><DT>Operator</DT><DD>Lisa Büller</DD></DL><H4>Extraction of 6 serum samples </H4><P>cfDNA was extracted from 6 serum samples, of which 3 were spiked with high amounts of Cy5-tagged artificial gene doping EPO DNA as shown in the table. For the extraction the QIAamp DNA Blood Mini Kit <A class="adpbl" href="https://2018.igem.org/Team:TUDelft/Experiments#serumextraction" target="_blank">protocol</A> was used</P><CENTER>Setup of the samples</CENTER><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Sample</TH><TH class="tableheaderadpbl">Spiked gene doping EPO (ng/ml serum)</TH></TR><TR class="tableunevenadpbl"><TD>Serum 1</TD><TD>0</TD></TR><TR class="tableunevenadpbl"><TD>Serum 2</TD><TD>0</TD></TR><TR class="tableunevenadpbl"><TD>Serum 3</TD><TD>0</TD></TR><TR class="tableunevenadpbl"><TD>Serum 4</TD><TD>60</TD></TR><TR class="tableunevenadpbl"><TD>Serum 5</TD><TD>60</TD></TR><TR class="tableunevenadpbl"><TD>Serum 6</TD><TD>60</TD></TR></TBODY></TABLE></CENTER><H4>Qubit dsDNA High Sensitivity DNA Quantification</H4><P>The 6 extracted samples were quantified using the Qubit dsDNA High Sensitivity DNA Assay Kit.</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Sample</TH><TH class="tableheaderadpbl">cfDNA Concentration (ng/μL)</TH></TR><TR class="tableunevenadpbl"><TD>Serum 1</TD><TD>0.378</TD></TR><TR class="tableunevenadpbl"><TD>Serum 2</TD><TD>0.380</TD></TR><TR class="tableunevenadpbl"><TD>Serum 3</TD><TD>0.378</TD></TR><TR class="tableunevenadpbl"><TD>Serum 4 - spiked</TD><TD>0.704</TD></TR><TR class="tableunevenadpbl"><TD>Serum 5 - spiked</TD><TD>0.606</TD></TR><TR class="tableunevenadpbl"><TD>Serum 6 - spiked</TD><TD>0.926</TD></TR></TBODY></TABLE></CENTER><P> 10ul of each spiked extraction was loaded on a gel and imaged with the Typhoon. Next to that also 5, 10 and 15 ng of Cy5-labeled EPO where loaded on the gel as reference. Showing that circa 5 ng of Cy5-labeled EPO was present in the 10ul of the extractions. The total extraction volume was 60ul. So circa 5*6=30 ng of Cy5-labeled EPO was present in the final extraction volumes. Showing a roughly 50% extraction efficency, since 60 ng was spiked per sample. 
    Whereas the difference in cfDNA measurement by using the Qubit would suggest an extraction efficiency of circa 37%. But the three spiked samples were too different from eachother in values, probably due to using a too high concentration of sample for spiking.3</P><H4>Albumin PCR extractions 10/09</H4><P>GoTaq PCR was done to verify the presence of the albumin gene after extraction of the 6 samples. Using the primers creating the small albumin fragment of 153 bp. 
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL)</TH></TR><TR class="tableunevenadpbl"><TD>Samples extr. 1-6 10/09  </TD><TD>3</TD></TR><TR class="tableevenadpbl"><TD>066</TD><TD>0.6</TD></TR><TR class="tableunevenadpbl"><TD>067</TD><TD>0.6</TD></TR><TR class="tableevenadpbl"><TD>Expected amplicon size (bp)</TD><TD>153</TD></TR></TBODY></TABLE></CENTER><P>6 tubes were prepared for the PCR, containing the serum extractions of 10/09/2018 as template DNA. The thermocycler programme was installed to have an extension time of 1 minute and an annealing temperature of 62 °C, 30 cycles. After the programme was finished, samples were kept at 4°C.</P><P>The gel showed no results, no bands.</P><H4>EPO PCR extractions 10/09</H4><P>GoTaq PCR was done to verify the presence of the EPO gene after extraction of the 3 spiked samples. Using the primers creating the small albumin fragment of 140 bp. 
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL)</TH></TR><TR class="tableunevenadpbl"><TD>Samples extr. 4-6 10/09  </TD><TD>3</TD></TR><TR class="tableevenadpbl"><TD>074</TD><TD>0.6</TD></TR><TR class="tableunevenadpbl"><TD>075</TD><TD>0.6</TD></TR><TR class="tableevenadpbl"><TD>Expected amplicon size (bp)</TD><TD>140</TD></TR></TBODY></TABLE></CENTER><P>4 tubes were prepared for the PCR, containing the serum extractions 4-6 of 10/09/2018 as template DNA. As a negative control serum sample 1 was used. The thermocycler programme was installed to have an extension time of 1 minute and an annealing temperature of 62 °C, 30 cycles. After the programme was finished, samples were kept at 4°C.</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#ThursdaySPS2">Thursday, 13/09/2018</BUTTON><DIV id="ThursdaySPS2" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>13/09/2018</DD><DT>Operator</DT><DD>Lisa Büller</DD></DL><H4>Albumin PCR extractions 10/09 - 2<SUP>nd</SUP> round</H4><P>GoTaq PCR was done to verify the presence of the albumin gene after extraction of the 6 samples. Using the primers creating the small albumin fragment of 153 bp. As a template the PCR product of the previous round of this PCR on the extracted samples were used.  
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL)</TH></TR><TR class="tableunevenadpbl"><TD>Samples extr. PCR 1-6 10/09  </TD><TD>3</TD></TR><TR class="tableevenadpbl"><TD>066</TD><TD>0.6</TD></TR><TR class="tableunevenadpbl"><TD>067</TD><TD>0.6</TD></TR><TR class="tableevenadpbl"><TD>Expected amplicon size (bp)</TD><TD>153</TD></TR></TBODY></TABLE></CENTER><P>6 tubes were prepared for the PCR, containing the serum extractions of 10/09/2018 as template DNA. The thermocycler programme was installed to have an extension time of 1 minute and an annealing temperature of 62 °C, 30 cycles. After the programme was finished, samples were kept at 4°C.</P><P>The gel showed really faint bands at the expected hight, but also a lot of off target effects and thus different sized bands. </P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV></DIV></UL></FORM><FORM><LABEL for="WeekSPS3" class="subdropdown cadpbl">Week 3 17/09/2018 - 23/09/2018</LABEL><UL class="submenu"><DIV class="subcontent"><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#ThursdaySPS3">Thursday, 20/09/2018</BUTTON><DIV id="ThursdaySPS3" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>20/09/2018</DD><DT>Operator</DT><DD>Lisa Büller</DD></DL><H4>Extraction of 18 serum samples </H4><P>cfDNA was extracted from 18 serum samples, of which 15 were spiked piked with artificial gene doping EPO in values coming from the kinetics model as shown in the table. For the extraction the QIAamp DNA Blood Mini Kit <A class="adpbl" href="https://2018.igem.org/Team:TUDelft/Experiments#serumextraction" target="_blank">protocol</A> was used</P><CENTER>Setup of the samples</CENTER><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Sample</TH><TH class="tableheaderadpbl">Spiked gene doping EPO (ng/ml serum)</TH></TR><TR class="tableunevenadpbl"><TD>Serum 1</TD><TD>0</TD></TR><TR class="tableunevenadpbl"><TD>Serum 2</TD><TD>0</TD></TR><TR class="tableunevenadpbl"><TD>Serum 3</TD><TD>0</TD></TR><TR class="tableunevenadpbl"><TD>Serum 4</TD><TD>0.287 linear EPO</TD></TR><TR class="tableunevenadpbl"><TD>Serum 5</TD><TD>0.287 linear EPO</TD></TR><TR class="tableunevenadpbl"><TD>Serum 6</TD><TD>0.287 linear EPO</TD></TR><TR class="tableunevenadpbl"><TD>Serum 7</TD><TD>0.0319 linear EPO</TD></TR><TR class="tableunevenadpbl"><TD>Serum 8</TD><TD>0.0319 linear EPO</TD></TR><TR class="tableunevenadpbl"><TD>Serum 9</TD><TD>0.0319 linear EPO</TD></TR><TR class="tableunevenadpbl"><TD>Serum 10</TD><TD>0.00127 linear EPO</TD></TR><TR class="tableunevenadpbl"><TD>Serum 11</TD><TD>0.00127 linear EPO</TD></TR><TR class="tableunevenadpbl"><TD>Serum 12</TD><TD>0.00127 linear EPO</TD></TR><TR class="tableunevenadpbl"><TD>Serum 13</TD><TD>0.134 BBa_K2643004</TD></TR><TR class="tableunevenadpbl"><TD>Serum 14</TD><TD>0.134 BBa_K2643004</TD></TR><TR class="tableunevenadpbl"><TD>Serum 15</TD><TD>0.134 BBa_K2643004</TD></TR><TR class="tableunevenadpbl"><TD>Serum 16</TD><TD>0.0053 BBa_K2643004</TD></TR><TR class="tableunevenadpbl"><TD>Serum 17</TD><TD>0.0053 BBa_K2643004</TD></TR><TR class="tableunevenadpbl"><TD>Serum 18</TD><TD>0.0053 BBa_K2643004</TD></TR></TBODY></TABLE></CENTER><H4>Qubit dsDNA High Sensitivity DNA Quantification</H4><P>The 18 extracted samples were quantified using the Qubit dsDNA High Sensitivity DNA Assay Kit.</P><CENTER>Setup of the samples</CENTER><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Sample</TH><TH class="tableheaderadpbl">cfDNA Concentration (ng/μL)</TH></TR><TR class="tableunevenadpbl"><TD>Serum 1</TD><TD>0.343</TD></TR><TR class="tableunevenadpbl"><TD>Serum 2</TD><TD>0.436</TD></TR><TR class="tableunevenadpbl"><TD>Serum 3</TD><TD>0.356</TD></TR><TR class="tableunevenadpbl"><TD>Serum 4</TD><TD>0.362</TD></TR><TR class="tableunevenadpbl"><TD>Serum 5</TD><TD>0.335</TD></TR><TR class="tableunevenadpbl"><TD>Serum 6</TD><TD>0.392</TD></TR><TR class="tableunevenadpbl"><TD>Serum 7</TD><TD>0.363</TD></TR><TR class="tableunevenadpbl"><TD>Serum 8</TD><TD>0.394</TD></TR><TR class="tableunevenadpbl"><TD>Serum 9</TD><TD>1.18</TD></TR><TR class="tableunevenadpbl"><TD>Serum 10</TD><TD>0.394</TD></TR><TR class="tableunevenadpbl"><TD>Serum 11</TD><TD>0.378</TD></TR><TR class="tableunevenadpbl"><TD>Serum 12</TD><TD>0.432</TD></TR><TR class="tableunevenadpbl"><TD>Serum 13</TD><TD>0.332</TD></TR><TR class="tableunevenadpbl"><TD>Serum 14</TD><TD>0.329</TD></TR><TR class="tableunevenadpbl"><TD>Serum 15</TD><TD>0.366</TD></TR><TR class="tableunevenadpbl"><TD>Serum 16</TD><TD>0.399</TD></TR><TR class="tableunevenadpbl"><TD>Serum 17</TD><TD>0.322</TD></TR><TR class="tableunevenadpbl"><TD>Serum 18</TD><TD>0.322</TD></TR></TBODY></TABLE></CENTER><H4>EPO PCR extractions </H4><P>GoTaq PCR was done to verify the presence of the spiked EPO gene after extraction of the 18 samples of 20/09/2018. 
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL)</TH></TR><TR class="tableunevenadpbl"><TD>Samples extr. 4-18 20/09  </TD><TD>3</TD></TR><TR class="tableevenadpbl"><TD>074</TD><TD>0.6</TD></TR><TR class="tableunevenadpbl"><TD>075</TD><TD>0.6</TD></TR><TR class="tableevenadpbl"><TD>Expected amplicon size (bp)</TD><TD>140</TD></TR></TBODY></TABLE></CENTER><P>19 tubes were prepared for the EPO PCR, containing the serum extractions of 20/09/2018 as template DNA, two positive controls (linear and BB EPO used for spiking), a negative control with MiliQ as template and a non spiked serum extraction as negative control. The thermocycler programme was installed to have an extension time of 40 seconds and an annealing temperature of 65 °C, 30 cycles. After the programme was finished, samples were kept at 4°C.</P><P>The PCR gave positive bands for all the spiked samples. But also for the negative control, the unspiked serum. After redoing of this PCR for the non spiked and spiked samples using the two EPO primer combinations 072+073 and 074+075 it became clear that there was a contamination in the non spiked serum samples.</P><P>To avoid further contaminations the PCR reagents were tested with PCR on contaminations by using different reagents and also the reagents of the extraction kit were tested. There was no contamination in these reagents. 
    </P><H4>Albumin PCR extractions 20/09 - 2 rounds</H4><P>The exact same Albumin PCR steps were followed for the extractions of 20/09 as for the extractions of 10/09 that has positive bands for albumin after two rounds of PCR with the same primers. But, doing this with the samples of 20/09 did not give a positive result. No bands were visible. </P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV></DIV></UL></FORM><H3 class="adpbl">October</H3><FORM><LABEL for="WeekSPO1" class="subdropdown cadpbl">Week 1 01/10/2018 - 07/10/2018</LABEL><UL class="submenu"><DIV class="subcontent"><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#FridaySPO1">Friday, 05/10/2018</BUTTON><DIV id="FridaySPO1" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>05/10/2018</DD><DT>Operator</DT><DD>Lisa Büller</DD></DL><H4>Albumin PCR Serum - 1<SUP>st</SUP> round</H4><P>GoTaq PCR was done to verify the presence of the albumin gene before extraction of 6 serum samples spiked like the extraction samples of 10/10/2018. First PCR of the nested PCR setup. The serum was spun down at 13.000 xg 4°C fro 30 minutes before it got used as a template. Different volumes of serum were used as template to see the effects of the proteins from the serum on the PCR reaction. 
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Sample</TH><TH class="tableheaderadpbl">Template for GoTaq PCR</TH></TR><TR class="tableunevenadpbl"><TD>1 - positive control</TD><TD>3ul extraction 03/10</TD></TR><TR class="tableunevenadpbl"><TD>2 serum</TD><TD>3ul</TD></TR><TR class="tableunevenadpbl"><TD>3 serum</TD><TD>3ul</TD></TR><TR class="tableunevenadpbl"><TD>4 serum</TD><TD>5ul</TD></TR><TR class="tableunevenadpbl"><TD>5 serum</TD><TD>5ul</TD></TR><TR class="tableunevenadpbl"><TD>6 - negative control</TD><TD>3ul MiliQ</TD></TR></TBODY></TABLE></CENTER><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL)</TH></TR><TR class="tableunevenadpbl"><TD>Serum samples </TD><TD>3-5</TD></TR><TR class="tableevenadpbl"><TD>091</TD><TD>0.6</TD></TR><TR class="tableunevenadpbl"><TD>092</TD><TD>0.6</TD></TR><TR class="tableevenadpbl"><TD>Expected amplicon size (bp)</TD><TD>229</TD></TR></TBODY></TABLE></CENTER><P>6 tubes were prepared for the first round of PCR, containing the serum as template DNA, a positive control extraction from 03/10/2018 and negative control with MiliQ as template. The thermocycler programme was installed to have an extension time of 1 minute and an annealing temperature of 62 °C, 30 cycles. After the programme was finished, samples were kept at 4°C.</P><H4>Albumin PCR Serum- 2<SUP>nd</SUP> round</H4><P>GoTaq PCR was done to verify the presence of the albumin gene from the serum samples. Second PCR of the nested PCR setup. 
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (ul)</TH></TR><TR class="tableunevenadpbl"><TD>Albumin PCR first round serum </TD><TD>3</TD></TR><TR class="tableevenadpbl"><TD>066</TD><TD>0.6</TD></TR><TR class="tableunevenadpbl"><TD>067</TD><TD>0.6</TD></TR><TR class="tableevenadpbl"><TD>Expected amplicon size (bp)</TD><TD>153</TD></TR></TBODY></TABLE></CENTER><P>6 tubes were prepared for the second round of PCR, containing the first round of albumin PCR on the serum as template DNA, a positive control from 03/10/2018 and negative control with MiliQ as template. The same volumes of the new templates were used for the second round as for the first round. The thermocycler programme was installed to have an extension time of 1 minute and an annealing temperature of 62 °C, 30 cycles. After the programme was finished, samples were kept at 4°C.</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV></DIV></UL></FORM><FORM><LABEL for="WeekSPO2" class="subdropdown cadpbl">Week 2 08/10/2018 - 14/10/2018</LABEL><UL class="submenu"><DIV class="subcontent"><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#WednesdaySPO2">Wednesday, 10/10/2018</BUTTON><DIV id="WednesdaySPO2" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>10/10/2018</DD><DT>Operator</DT><DD>Lisa Büller</DD></DL><H4>Extraction of 6 serum samples - 1<SUP>st</SUP> round</H4><P>cfDNA was extracted from 6 serum samples, of which some were spiked with artificial gene doping EPO DNA as shown in the table. For the extraction the QIAamp DNA Blood Mini Kit <A class="adpbl" href="https://2018.igem.org/Team:TUDelft/Experiments#serumextraction" target="_blank">protocol</A> was used</P><CENTER>Setup of the samples</CENTER><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Sample</TH><TH class="tableheaderadpbl">Spiked gene doping EPO (ng/ml serum)</TH></TR><TR class="tableunevenadpbl"><TD>Serum 1</TD><TD>0</TD></TR><TR class="tableunevenadpbl"><TD>Serum 2</TD><TD>0</TD></TR><TR class="tableunevenadpbl"><TD>Serum 3</TD><TD>0.287 linear gene doping EPO</TD></TR><TR class="tableunevenadpbl"><TD>Serum 4</TD><TD>0.287 linear gene doping EPO</TD></TR><TR class="tableunevenadpbl"><TD>Serum 5</TD><TD>1.203 BBa_K2643004</TD></TR><TR class="tableunevenadpbl"><TD>Serum 6</TD><TD>1.203 BBa_K2643004</TD></TR></TBODY></TABLE></CENTER><H4>Qubit dsDNA High Sensitivity DNA Quantification</H4><P>The 6 extracted samples were quantified using the Qubit dsDNA High Sensitivity DNA Assay Kit.</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Sample</TH><TH class="tableheaderadpbl">cfDNA Concentration (ng/μL)</TH></TR><TR class="tableunevenadpbl"><TD>Serum 1</TD><TD>0.398</TD></TR><TR class="tableunevenadpbl"><TD>Serum 2</TD><TD>0.426</TD></TR><TR class="tableunevenadpbl"><TD>Serum 3</TD><TD>0.386</TD></TR><TR class="tableunevenadpbl"><TD>Serum 4</TD><TD>0.384</TD></TR><TR class="tableunevenadpbl"><TD>Serum 5</TD><TD>0.386</TD></TR><TR class="tableunevenadpbl"><TD>Serum 6</TD><TD>0.386</TD></TR></TBODY></TABLE></CENTER></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#SaturdaySPO2">Saturday, 13/10/2018</BUTTON><DIV id="SaturdaySPO2" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>13/10/2018</DD><DT>Operator</DT><DD>Lisa Büller</DD></DL><H4>Albumin PCR extractions - 1<SUP>st</SUP> round</H4><P>GoTaq PCR was done to verify the presence of the albumin gene after extraction of the 6 samples of 10/10/2018. First PCR of the nested PCR setup. 
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL)</TH></TR><TR class="tableunevenadpbl"><TD>Samples extr. 1-6 10/10  </TD><TD>3</TD></TR><TR class="tableevenadpbl"><TD>091</TD><TD>0.6</TD></TR><TR class="tableunevenadpbl"><TD>092</TD><TD>0.6</TD></TR><TR class="tableevenadpbl"><TD>Expected amplicon size (bp)</TD><TD>229</TD></TR></TBODY></TABLE></CENTER><P>8 tubes were prepared for the first round of PCR, containing the serum extractions of 10/10/2018 as template DNA, a positive control from 03/10/2018 and negative control with MiliQ as template. The thermocycler programme was installed to have an extension time of 1 minute and an annealing temperature of 62 °C, 30 cycles. After the programme was finished, samples were kept at 4°C.</P></DIV><H4>Albumin PCR extractions - 2<SUP>nd</SUP> round</H4><P>GoTaq PCR was done to verify the presence of the albumin gene after extraction of the 6 samples of 10/10/2018. Second PCR of the nested PCR setup. 
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL)</TH></TR><TR class="tableunevenadpbl"><TD>Albumin PCR first round 13/10  </TD><TD>3</TD></TR><TR class="tableevenadpbl"><TD>066</TD><TD>0.6</TD></TR><TR class="tableunevenadpbl"><TD>067</TD><TD>0.6</TD></TR><TR class="tableevenadpbl"><TD>Expected amplicon size (bp)</TD><TD>153</TD></TR></TBODY></TABLE></CENTER><P>8 tubes were prepared for the second round of PCR, containing the first round of albumin PCR on the serum extractions of 10/10/2018 as template DNA, a positive control from 03/10/2018 and negative control with MiliQ as template. The thermocycler programme was installed to have an extension time of 1 minute and an annealing temperature of 62 °C, 30 cycles. After the programme was finished, samples were kept at 4°C.</P><H4>EPO PCR extractions </H4><P>GoTaq PCR was done to verify the presence of the spiked EPO gene after extraction of the 6 samples of 10/10/2018. Two primer combinations were used.
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL)</TH></TR><TR class="tableunevenadpbl"><TD>Samples extr. 1-6 10/10  </TD><TD>3</TD></TR><TR class="tableevenadpbl"><TD>072</TD><TD>0.6</TD></TR><TR class="tableunevenadpbl"><TD>073</TD><TD>0.6</TD></TR><TR class="tableevenadpbl"><TD>Expected amplicon size (bp)</TD><TD>297</TD></TR></TBODY></TABLE></CENTER><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL)</TH></TR><TR class="tableunevenadpbl"><TD>Samples extr. 1-6 10/10  </TD><TD>3</TD></TR><TR class="tableevenadpbl"><TD>074</TD><TD>0.6</TD></TR><TR class="tableunevenadpbl"><TD>075</TD><TD>0.6</TD></TR><TR class="tableevenadpbl"><TD>Expected amplicon size (bp)</TD><TD>140</TD></TR></TBODY></TABLE></CENTER><P>18 tubes were prepared for the first round of PCR, containing the serum extractions of 10/10/2018 as template DNA, two positive controls (linear and BB EPO used for spiking) and negative control with MiliQ as template. The thermocycler programme was installed to have an extension time of 40 seconds and an annealing temperature of 65 °C, 20 cycles. After the programme was finished, samples were kept at 4°C.</P><H4>Albumin PCR Serum - 1<SUP>st</SUP> round</H4><P>GoTaq PCR was done to verify the presence of the albumin gene before extraction of 6 serum samples spiked like the extraction samples of 10/10/2018. First PCR of the nested PCR setup. The serum was spun down at 13.000 xg 4°C fro 30 minutes before it got used as a template.
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Sample</TH><TH class="tableheaderadpbl">Spiked gene doping EPO (ng/ml serum)</TH></TR><TR class="tableunevenadpbl"><TD>Serum 1</TD><TD>0</TD></TR><TR class="tableunevenadpbl"><TD>Serum 2</TD><TD>0</TD></TR><TR class="tableunevenadpbl"><TD>Serum 3</TD><TD>0.287 linear gene doping EPO</TD></TR><TR class="tableunevenadpbl"><TD>Serum 4</TD><TD>0.287 linear gene doping EPO</TD></TR><TR class="tableunevenadpbl"><TD>Serum 5</TD><TD>1.203 BBa_K2643004</TD></TR><TR class="tableunevenadpbl"><TD>Serum 6</TD><TD>1.203 BBa_K2643004</TD></TR></TBODY></TABLE></CENTER><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL)</TH></TR><TR class="tableunevenadpbl"><TD>Serum samples </TD><TD>3</TD></TR><TR class="tableevenadpbl"><TD>091</TD><TD>0.6</TD></TR><TR class="tableunevenadpbl"><TD>092</TD><TD>0.6</TD></TR><TR class="tableevenadpbl"><TD>Expected amplicon size (bp)</TD><TD>229</TD></TR></TBODY></TABLE></CENTER><P>8 tubes were prepared for the first round of PCR, containing the serum as template DNA, a positive control from 03/10/2018 and negative control with MiliQ as template. The thermocycler programme was installed to have an extension time of 1 minute and an annealing temperature of 62 °C, 30 cycles. After the programme was finished, samples were kept at 4°C.</P><H4>Albumin PCR Serum- 2<SUP>nd</SUP> round</H4><P>GoTaq PCR was done to verify the presence of the albumin gene from the 6 serum samples. Second PCR of the nested PCR setup. 
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL)</TH></TR><TR class="tableunevenadpbl"><TD>Albumin PCR first round serum </TD><TD>3</TD></TR><TR class="tableevenadpbl"><TD>066</TD><TD>0.6</TD></TR><TR class="tableunevenadpbl"><TD>067</TD><TD>0.6</TD></TR><TR class="tableevenadpbl"><TD>Expected amplicon size (bp)</TD><TD>153</TD></TR></TBODY></TABLE></CENTER><P>8 tubes were prepared for the second round of PCR, containing the first round of albumin PCR on the serum as template DNA, a positive control from 03/10/2018 and negative control with MiliQ as template. The thermocycler programme was installed to have an extension time of 1 minute and an annealing temperature of 62 °C, 30 cycles. After the programme was finished, samples were kept at 4°C.</P><H4>EPO PCR Serum - 1<SUP>st</SUP> round</H4><P>GoTaq PCR was done to try to verify the presence of the spiked EPO gene before extraction of 6 serum samples spiked like the extraction samples of 10/10/2018. First PCR of a nested PCR setup. The serum was spun down at 13.000 xg 4°C for 30 minutes  before it got used as a template.
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL)</TH></TR><TR class="tableunevenadpbl"><TD>Serum samples </TD><TD>3</TD></TR><TR class="tableevenadpbl"><TD>072</TD><TD>0.6</TD></TR><TR class="tableunevenadpbl"><TD>073</TD><TD>0.6</TD></TR><TR class="tableevenadpbl"><TD>Expected amplicon size (bp)</TD><TD>297</TD></TR></TBODY></TABLE></CENTER><P>9 tubes were prepared for the first round of PCR, containing the serum as template DNA, two positive controls (linear and BB EPO used for spiking) and negative control with MiliQ as template. The thermocycler programme was installed to have an extension time of 40 seconds and an annealing temperature of 65 °C, 30 cycles. After the programme was finished, samples were kept at 4°C.</P><H4>EPO PCR Serum- 2<SUP>nd</SUP> round</H4><P>GoTaq PCR was done to verify the presence of the EPO gene from the 6 serum samples. Second PCR of the nested PCR setup. 
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL)</TH></TR><TR class="tableunevenadpbl"><TD>Serum PCR first round serum </TD><TD>3</TD></TR><TR class="tableevenadpbl"><TD>074</TD><TD>0.6</TD></TR><TR class="tableunevenadpbl"><TD>075</TD><TD>0.6</TD></TR><TR class="tableevenadpbl"><TD>Expected amplicon size (bp)</TD><TD>140</TD></TR></TBODY></TABLE></CENTER><P>9 tubes were prepared for the second round of PCR, containing the first round of EPO PCR on the serum as template DNA, two positive controls (linear and BB EPO used for spiking) and negative control with MiliQ as template. The thermocycler programme was installed to have an extension time of 40 seconds and an annealing temperature of 65 °C, 30 cycles. After the programme was finished, samples were kept at 4°C.</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV></UL></FORM></DIV></DIV></DIV><DIV class="container nowbg" id="PreScreen"><DIV class="row nowbg"><DIV class="col-xs-12 col-sm-3 col-lg-3"><NAV id="myScrollspy" class="hidden-xs hidden-sm"><UL class="nav nav-pills nav-stacked" data-spy="affix"><LI class="dropdown"><A class="dropdown-toggle adpbl" data-toggle="dropdown" href="#">July</A></LI><LI><A href="#JulyPreScreen3" class="adpbl">Week: 16/07/2018 - 22/07/2018</A></LI></UL><LI class="dropdown"><A class="dropdown-toggle adpbl" data-toggle="dropdown" href="#">August</A></LI><LI><A href="#AugustPreScreen5" class="adpbl">Week: 27/08/2018 - 02/09/2018</A></LI><LI class="dropdown"><A class="dropdown-toggle adpbl" data-toggle="dropdown" href="#">September</A></LI><LI><A href="#SeptemberPreScreen1" class="adpbl">Week: 03/09/2018 - 09/09/2018</A></LI><LI><A href="#SeptemberPreScreen2" class="adpbl">Week: 10/09/2018 - 16/09/2018</A></LI><LI><A href="#SeptemberPreScreen3" class="adpbl">Week: 17/09/2018 - 23/09/2018</A></LI><LI><A href="#SeptemberPreScreen4" class="adpbl">Week: 24/09/2018 - 30/09/2018</A></LI><LI class="drowdown"><A class="dropdown-toggle adpbl" data-toggle="dropdown" href="#">October</A></LI><LI><A href="#OctoberPreScreen1" class="adpbl">Week: 01/10/2018 - 07/10/2018</A></LI><LI><A href="#OctoberPreScreen2" class="adpbl">Week: 08/10/2018 - 14/10/2018</A></LI></NAV></DIV><DIV class="col-lg-8 col-md-8 col-sm-12 col-xs-12 nowbg" style="margin-right:20px;"><H3 class="adpbl">July</H3><FORM><LABEL for="WeekPSJ3" class="subdropdown cadpbl">Week 3 16/07/2018 - 22/07/2018</LABEL><UL class="submenu"><DIV class="subcontent"><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#ThursdayPSJ3">Thursday, 19/07/2018</BUTTON><DIV id="ThursdayPSJ3" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>19/07/2018</DD><DT>Operator</DT><DD>Jard Mattens</DD></DL><H4 class="adpbl">d-AuNPs Generation</H4><P>In order to generate dextrin-capped gold nanoparticles, first a gold chloride (HAuCl<SUB>4</SUB>.H<SUB>2</SUB>O) stock solution of 50 mL was prepared by dissolving 393.8 mg of HAuCl<SUB>4</SUB>.H<SUB>2O</SUB> in 50 mL of distilled sterile water. After preparation, this solution was stored at 4 °C covered in aluminium foil.
Secondly, a 10% sterile sodium carbonate (Na<SUB>2</SUB>CO<SUB>3</SUB>) solution was prepared by dissolving 4.985 g Na<SUB>2</SUB>CO<SUB>3</SUB> in 50 mL MilliQ water, after which the solution was filter sterilized.
Then, 200 mL distilled sterile water was be adjusted to pH 9 by carefully adding 1 M sodium hydroxide (NaOH) drops of 20 uL at a time, while keeping the solution sterile under a flame.
Subsequently, 25 mL of dextrin solution was prepared by dissolving 625 mg of dextrin powder in 25 mL of sterile MilliQ water to obtain a concentration of 25 g/L. This solution was then added to a sterile 250 mL flask.

Then, 5 mL of the gold chloride stock solution was added to the 250 mL flask and the pH was adjusted to 9 with the sterile 10% sodium carbonate solution, while working sterile.
Lastly, the reaction volume was completed to 50 mL by adding 20 mL distilled water (pH = 9).
For the actual generation of the d-AuNPs the flask was then incubated at 50 °C, covered in aluminium foil with continuous shaking for 3 to 8 hours. After 3 to 8 hours shaking, a reddish colour was observed on the solution that is characteristic for the correct size nanoparticle formation. The solution was observed every 30 minutes to check for red color formation.
Transmission Electron Microscope analysis was performed to check the size and size distribution of the d-AuNPs. the suspension of d-AuNPs in distilled water (ratio 1:4) and sonicate it for 5 min. Then, after plasma treatment this solution was be transferred to a copper grid for imaging. The correct size of the d-AuNPs be in the order of 10 nm.
         </P><P>We observed d-AuNPs in a relatively low concentrations. Also, the sizes varied unequally around 100 nm, which is an indication of aggregation as the expected sizes are around 10 nm. Therefore, a new batch had to be made for the actual analysis.
         </P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV></DIV></UL></FORM><H3 class="adpbl">August</H3><FORM><LABEL for="WeekPSA5" class="subdropdown cadpbl">Week 5 27/08/2018 - 02/09/2018</LABEL><UL class="submenu"><DIV class="subcontent"><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#TuesdayPSA5">Tuesday, 28/08/2018</BUTTON><DIV id="TuesdayPSA5" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>28/08/2018</DD><DT>Operator</DT><DD>Jard Mattens</DD></DL><H4 class="adpbl">d-AuNPs Generation</H4><P>In order to generate dextrin-capped gold nanoparticles, first a gold chloride (HAuCl<SUB>4</SUB>.H<SUB>2</SUB>O) stock solution of 50 mL was prepared by dissolving 393.8 mg of HAuCl<SUB>4</SUB>.H<SUB>2O</SUB> in 50 mL of distilled sterile water. After preparation, this solution was stored at 4 °C covered in aluminium foil.
Secondly, a 10% sterile sodium carbonate (Na<SUB>2</SUB>CO<SUB>3</SUB>) solution was prepared by dissolving 4.985 g Na<SUB>2</SUB>CO<SUB>3</SUB> in 50 mL MilliQ water, after which the solution was filter sterilized.
Then, 200 mL distilled sterile water was be adjusted to pH 9 by carefully adding 1 M sodium hydroxide (NaOH) drops of 20 uL at a time, while keeping the solution sterile under a flame.
Subsequently, 25 mL of dextrin solution was prepared by dissolving 625 mg of dextrin powder in 25 mL of sterile MilliQ water to obtain a concentration of 25 g/L. This solution was then added to a sterile 250 mL flask.

Then, 5 mL of the gold chloride stock solution was added to the 250 mL flask and the pH was adjusted to 9 with the sterile 10% sodium carbonate solution, while working sterile.
Lastly, the reaction volume was completed to 50 mL by adding 20 mL distilled water (pH = 9).
For the actual generation of the d-AuNPs the flask was then incubated at 50 °C, covered in aluminium foil with continuous shaking for 3 to 8 hours. After 3 to 8 hours shaking, a reddish colour was observed on the solution that is characteristic for the correct size nanoparticle formation. The solution was observed every 30 minutes to check for red color formation.
Transmission Electron Microscope analysis was performed to check the size and size distribution of the d-AuNPs. the suspension of d-AuNPs in distilled water (ratio 1:4) and sonicate it for 5 min. Then, after plasma treatment this solution was be transferred to a copper grid for imaging. The correct size of the d-AuNPs be in the order of 10 nm.
         </P><P>We observed d-AuNPs in relatively low concentrations. However, the sizes were as the expected within a range of 7 nm to 11 nm, which is sufficient for the d-AuNP analysis. In the future however, the protocol should be optimized to generate more uniform batches.
    
         </P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#WednesdayPSA5">Wednesday, 29/08/2018</BUTTON><DIV id="WednesdayPSA5" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>29/08/2018</DD><DT>Operator</DT><DD>Jard Mattens</DD></DL><H4 class="adpbl">d-AuNPs Testing: d-AuNPs stability upon salt</H4><P>
The dAuNPs are tested for their stability under different salt concentrations as an initial functionality test. 20 µL of the d-AuNPs solution, 20 µL MilliQ and 20 µL NaCl solutions (range from 0 mM to 1000 mM) were added in one of the 96 wells. The plate was incubated at 21 ºC during 10 minutes.</P><P>The aggregation of d-AuNPs, as observed by a colour change from intensely red to purple occured at a final NaCl concentration of 1M*⅓ = 333 mM, but increased when adding more salt. 
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV></DIV></UL></FORM><H3 class="adpbl">September</H3><FORM><LABEL for="WeekPSS1" class="subdropdown cadpbl">Week 1 03/09/2018 - 09/09/2018</LABEL><UL class="submenu"><DIV class="subcontent"><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#WednesdayPSS1">Wednesday, 05/09/2018</BUTTON><DIV id="WednesdayPSS1" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>05/09/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">High Fidelity PCR</H4><P> 
To amplify the EPO (582 bp) coding sequence, high fidelity PCR was done using the EPO Gblock ordered from IDT as template DNA and 038 and 039 as forward and reverse primers. Two 50µL PCR mixtures were prepared, according to the <A href="https://2018.igem.org/Team:TUDelft/Experiments#highfidelityPCR-scroll" target="_blank" class="adpbl">High Fidelity PCR protocol</A>. 
Primer combination(s) and expected fragment(s)
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Fw primer</TH><TH class="tableheaderadpbl">Rv primer</TH><TH class="tableheaderadpbl">Expected fragment</TH><TH class="tableheaderadpbl">Expected size (bp) </TH></TR><TR class="tableunevenadpbl"><TD>038</TD><TD>039</TD><TD>EPO</TD><TD>582</TD></TR></TBODY></TABLE></CENTER><P>The thermocycler programme was installed to have an extension time of 0 seconds and an annealing temperature of 65 °C. After the programme was finished, samples were kept at 4 °C.
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV></DIV></UL></FORM><FORM><LABEL for="WeekPSS2" class="subdropdown cadpbl">Week 2 10/09/2018 - 16/09/2018</LABEL><UL class="submenu"><DIV class="subcontent"><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#MondayPSS2">Monday, 10/09/2018</BUTTON><DIV id="MondayPSS2" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>10/09/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">DNA Gel Electrophoresis</H4><P>The obtained PCR products with EPO coding sequence were run on a 0.8% agarose gel in TAE buffer. 4 μL was pipetted into each lane. The gel was imaged using a gel documentation system with UV-light.
</P><P>
The EPO fragment that should be amplified by the PCR is 582 bp in size. As can be seen on the gel, EPO was successfully amplified.
</P><DL><DT>Date</DT><DD>10/09/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">PCR Clean-Up</H4><P>50 20 μL of each PCR reaction mixture was subjected to PCR Clean-Up, according to the <A href="https://2018.igem.org/Team:TUDelft/Experiments#PCRcleanup-scroll" class="adpbl" target="_blank">protocol</A>. Elution was done in 30 µL Milli-Q.</P><DL><DT>Date</DT><DD>10/09/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">Nanodrop DNA Quantification</H4><P>The following isolates were quantified according to the <A href="https://2018.igem.org/Team:TUDelft/Experiments#NanodropDNA-scroll" class="adpbl" target="_blank">Nanodrop DNA Quantification protocol</A>. The concentration and purity of the DNA (260/280 and 260/230 ratios) were measured. </P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Isolate</TH><TH class="tableheaderadpbl">Concentration (ng/uL)</TH><TH class="tableheaderadpbl">260/280 ratio</TH><TH class="tableheaderadpbl">260/230 ratio</TH></TR><TR class="tableunevenadpbl"><TD>EPO</TD><TD>56.0</TD><TD>1.90</TD><TD>1.58</TD></TR></TBODY></TABLE></CENTER></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#FridayPSS2">Friday, 14/09/2018</BUTTON><DIV id="FridayPSS2" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>14/09/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">d-AuNPs Testing: d-AuNPs stability upon salt</H4><P>
The dAuNPs are tested for their stability under different salt concentrations. 20 µL of the d-AuNPs solution, 20 µL MilliQ and 20 µL NaCl solutions (range from 0 mM to 2000 mM) were added in one of the 96 wells on the 92 wells plate. The plate was incubated at 21 ºC during 10 minutes. The visible absorption spectrum from 450 nm to 650 nm of the solutions was measure and the ratio 620/520 nm was calculated for analysis. The experiment was performed in triplicate. The plate view, relevant absorption spectra and ratio 620/520 nm is shown.</P><P>The aggregation of d-AuNPs occured at a final NaCl concentration of 1M*⅓ = 333 mM, but increased when adding more salt. At a final concentration of 2M*⅓=666 mM NaCl, the aggregates were the most intensively changed of color. One mistake was made by pipetting the complete d-AuNPs tube into the wells without mixing the tube constantly. The d-AuNPs were sedimentating to the bottom of the tube and this was pipetted into wells E1 and E2. These results should be left out of the data analysis.

From the data, the spectrum of the mixtures with 0 mM, 1000 mM and 2000 mM NaCl were plotted. Due to pipetting errors and mixing errors, the standard deviations at lower salt concentrations were high. Additionally, the from all the samples with different salt concentrations, the ratio 620/520 was calculated and plotted over the salt concentration.
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV></DIV></UL></FORM><FORM><LABEL for="WeekPSS3" class="subdropdown cadpbl">Week 3 17/09/2018 - 23/09/2018</LABEL><UL class="submenu"><DIV class="subcontent"><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#MondayPSS3">Monday, 17/09/2018</BUTTON><DIV id="MondayPSS3" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>17/09/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">d-AuNPs Testing: d-AuNPs stability upon salt</H4><P>
The dAuNPs are tested for their stability under different salt concentrations. 20 µL of the d-AuNPs solution, 20 µL MilliQ and 20 µL NaCl solutions (range from 0 mM to 2000 mM) were added in one of the 96 wells on the 92 wells plate. The plate was incubated at 21 ºC during 10 minutes. The visible absorption spectrum from 450 nm to 650 nm of the solutions was measure and the ratio 620/520 nm was calculated for analysis. The experiment was performed in triplicate. The plate view, relevant absorption spectra and ratio 620/520 nm is shown.</P><P>The aggregation of d-AuNPs occured at a final NaCl concentration of 1M*⅓ = 333 mM, but increased when adding more salt. At a final concentration of 2M*⅓=666 mM NaCl, the aggregates were the most intensively changed of color. 
From the data, the spectrum of the mixtures with 0 mM, 1000 mM and 2000 mM NaCl were plotted. In comparison with the previous measurements, the error bars are lower. Additionally, the from all the samples with different salt concentrations, the ratio 620/520nm was calculated and plotted over the salt concentration.
</P><DL><DT>Date</DT><DD>17/09/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">d-AuNPs Testing: d-AuNPs stability upon salt with ssDNAp</H4><P>
The dAuNPs are tested for their stability under different salt concentrations and the influence of ssDNAp is determined. 20 µL of the d-AuNPs solution, 16 µL MilliQ, <A href="https://2018.igem.org/Team:TUDelft/Primers#ssDNAp" target="_blank" class="adpbl">4 µL ssDNAp 1</A>  and 20 µL NaCl solutions (range from 0 mM to 2000 mM) were added in one of the 96 wells on the 96 wells plate. The plate was incubated at 21 ºC during 10 minutes. The visible absorption spectrum from 450 nm to 650 nm of the solutions was measure and the ratio 620/520 nm was calculated for analysis. The experiment was performed in triplicate. The plate view, relevant absorption spectra and ratio 620/520 nm is shown.</P><P>The aggregation of d-AuNPs occured at a final NaCl concentration of 1500 mM * ⅓ = 500 mM, but increased when adding more salt. At a final concentration of 2M*⅓=666 mM NaCl, the aggregates were the most intensively changed of color.
From the data, the spectrum of the mixtures with 0 mM, 1000 mM and 2000 mM NaCl were plotted. Additionally, the from all the samples with different salt concentrations, the ratio 620/520 nm was calculated and plotted over the salt concentration.
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#WednesdayPSS3">Wednesday, 19/09/2018</BUTTON><DIV id="WednesdayPSS3" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>19/09/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">d-AuNPs Testing: d-AuNPs stability upon salt with(out) ssDNAp</H4><P>
The dAuNPs are tested for their stability under different salt concentrations and the influence of ssDNAp is determined. 20 µL of the d-AuNPs solution, 16 µL MilliQ, <A href="https://2018.igem.org/Team:TUDelft/Primers#ssDNAp" target="_blank" class="adpbl">4 µL ssDNAp 1</A> and 20 µL NaCl solutions (range from 0 mM to 2000 mM) were added in one of the 96 wells on the 92 wells plate. As negative controls, 0 L ssDNAp 1 was added. The plate was incubated at 21 ºC during 10 minutes. The visible absorption spectrum from 450 nm to 650 nm of the solutions was measure and the ratio 620/520 nm was calculated for analysis. The experiment was performed in triplicate. The plate view, relevant absorption spectra and ratio 620/520 nm is shown.</P><P>
The d-AuNPs aggregated when salt was added from a final concentration of 1M*⅓=333 mM to 2M*⅓=666 mM NaCl. When ssDNAp (1 uM) were added, the concentrations at which the d-AuNPs aggregated increased. However, at 2 M NaCl, the d-AuNPs still aggregated although the aggregation was less intense compared to the salt aggregation without ssDNAp’s.  
From the data, the spectrum of the mixtures with 0 mM, 1000 mM and 2000 mM NaCl were plotted from both samples: with and without ssDNAp. Additionally, the from all the samples either with or without ssDNAp addition, the ratio 620/520 was calculated and plotted over the salt concentration.      
</P><DL><DT>Date</DT><DD>19/09/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">d-AuNPs Generation</H4><P>In order to generate dextrin-capped gold nanoparticles, first a gold chloride (HAuCl<SUB>4</SUB>.H<SUB>2</SUB>O) stock solution of 50 mL was prepared by dissolving 393.8 mg of HAuCl<SUB>4</SUB>.H<SUB>2O</SUB> in 50 mL of distilled sterile water. After preparation, this solution was stored at 4 °C covered in aluminium foil.
Secondly, a 10% sterile sodium carbonate (Na<SUB>2</SUB>CO<SUB>3</SUB>) solution was prepared by dissolving 4.985 g Na<SUB>2</SUB>CO<SUB>3</SUB> in 50 mL MilliQ water, after which the solution was filter sterilized.
Then, 200 mL distilled sterile water was be adjusted to pH 9 by carefully adding 1 M sodium hydroxide (NaOH) drops of 20 uL at a time, while keeping the solution sterile under a flame.
Subsequently, 25 mL of dextrin solution was prepared by dissolving 625 mg of dextrin powder in 25 mL of sterile MilliQ water to obtain a concentration of 25 g/L. This solution was then added to a sterile 250 mL flask.

Then, 5 mL of the gold chloride stock solution was added to the 250 mL flask and the pH was adjusted to 9 with the sterile 10% sodium carbonate solution, while working sterile.
Lastly, the reaction volume was completed to 50 mL by adding 20 mL distilled water (pH = 9).
For the actual generation of the d-AuNPs the flask was then incubated at 50 °C, covered in aluminium foil with continuous shaking for 3 to 8 hours. After 3 to 8 hours shaking, a reddish colour was observed on the solution that is characteristic for the correct size nanoparticle formation. The solution was observed every 30 minutes to check for red color formation.
Transmission Electron Microscope analysis was performed to check the size and size distribution of the d-AuNPs. the suspension of d-AuNPs in distilled water (ratio 1:4) and sonicate it for 5 min. Then, after plasma treatment this solution was be transferred to a copper grid for imaging. The correct size of the d-AuNPs be in the order of 10 nm.
          </P><P>We observed d-AuNPs in a relatively low concentration. The size distribution is ranging from 8 nm to 13 nm, which is sufficient for d-AuNPs analysis for prescreening.
          </P></DIV><DL><DT>Date</DT><DD>19/09/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">d-AuNPs Testing: d-AuNPs stability upon salt</H4><P>
The newly generated d-AuNPs are tested for their stability under different salt concentrations. 20 µL of the d-AuNPs solution, 20 µL MilliQ and 20 µL NaCl solutions (range from 0 mM to 2000 mM) were added in one of the 96 wells on the 92 wells plate. The plate was incubated at 21 ºC during 10 minutes. The plate view is shown.</P><P>The aggregation of d-AuNPs occured at a final NaCl concentration of 100 mM. In comparison with the old d-AuNPs batch, they aggregate at a lower salt concentration indicating that they are less stable. This observation can be explained by the time that the gold nanoparticles were in solution. During the generation, dextrin solution was added. The incubation time was approximately 8 hours and in this time the gold nanoparticles were capped by dextrin. The dextin makes the gold nanoparticles more stable. However, when the gold nanoparticles stay in the solution for a longer time, the remaining dextrin can adhere to the gold nanoparticles, making them more stable.
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#ThursdayPSS3">Thursday, 20/09/2018</BUTTON><DIV id="ThursdayPSS3" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>20/09/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">d-AuNPs Testing: d-AuNPs stability upon salt</H4><P>
The newly generated dAuNPs are tested for their stability under different salt concentrations. 20 µL of the d-AuNPs solution, 20 µL MilliQ and 20 µL NaCl solutions (range from 0 mM to 1000 mM) were added in one of the 96 wells on the 92 wells plate. The plate was incubated at 21 ºC during 10 minutes. The visible absorption spectrum from 450 nm to 650 nm of the solutions was measure and the ratio 620/520 nm was calculated for analysis. The experiment was performed in triplicate. The plate view, relevant absorption spectra and ratio 620/520 nm is shown.</P><P>With the new batch of d-AuNPs, the salt concentration variation was performed again. The range of salt concentration is adapted from 0 to 1000 mM NaCl added concentration. From the plate view it can be seen that the d-AuNPs are aggregating at a lower salt concentration than the previous (older) batch on 14/09/2018, 17/09/2019 and 19/09/2018.
</P></DIV><DL><DT>Date</DT><DD>20/09/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">d-AuNPs Testing: d-AuNPs stability upon salt with(out) ssDNAp</H4><P>
The dAuNPs are tested for their stability under different salt concentrations and the influence of ssDNAp is determined. 20 µL of the d-AuNPs solution, 16 µL MilliQ, <A href="https://2018.igem.org/Team:TUDelft/Primers#ssDNAp" target="_blank" class="adpbl">4 µL ssDNAp 1</A> and 20 µL NaCl solutions (range from 0 mM to 1000 mM) were added in one of the 96 wells on the 96 wells plate. As negative controls, 0 L ssDNAp 1 was added. The plate was incubated at 21 ºC during 10 minutes. The visible absorption spectrum from 450 nm to 650 nm of the solutions was measure and the ratio 620/520 nm was calculated for analysis. The experiment was performed in triplicate. The plate view, relevant absorption spectra and ratio 620/520 nm is shown.</P><P>
With the new batch of d-AuNPs, the salt concentration variation was performed again. The range of salt concentration is adapted from 0 to 1000 mM NaCl added concentration. Additionally, ssDNAp cause a stabilizing effect as the color change occurs at a higher salt concentration compared to the solutions without ssDNAp.   
</P><DL><DT>Date</DT><DD>20/09/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">d-AuNPs Testing: d-AuNPs stability upon salt with(out) ssDNAp</H4><P>
The dAuNPs are tested for their stability under different salt concentrations and the influence of ssDNAp is determined. 20 µL of the d-AuNPs solution and 20 µL NaCl solutions (250, 500 and 1000 mM respectively) were added in the 96 wells plate. To test the influence of the ssDNAp concentration on d-AuNPs stabilization, different amounts of <A href="https://2018.igem.org/Team:TUDelft/Primers#ssDNAp" target="_blank" class="adpbl">0 to 10 µL ssDNAp 1</A> was added in subsequent wells. The plate was incubated at 21 ºC during 10 minutes. The visible absorption spectrum from 450 nm to 650 nm of the solutions was measure and the ratio 620/520 nm was calculated for analysis. The experiment was performed in triplicate. The plate view, relevant absorption spectra and ratio 620/520 nm is shown.</P><P>With a higher ssDNAp concentration, the d-AuNPs are being stabilized. The color change to purple, which indicates destabilization, occurs at a higher salt concentration. </P><DL><DT>Date</DT><DD>20/09/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">High Fidelity PCR</H4><P> 
To amplify the EPO (582 bp) coding sequence, high fidelity PCR was done using the EPO Gblock ordered from IDT as template DNA and 038 and 039 as forward and reverse primers. Two 50µL PCR mixtures were prepared, according to the <A href="https://2018.igem.org/Team:TUDelft/Experiments#highfidelityPCR-scroll" target="_blank" class="adpbl">High Fidelity PCR protocol</A>. 
Primer combination(s) and expected fragment(s)
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Fw primer</TH><TH class="tableheaderadpbl">Rv primer</TH><TH class="tableheaderadpbl">Expected fragment</TH><TH class="tableheaderadpbl">Expected size (bp) </TH></TR><TR class="tableunevenadpbl"><TD>038</TD><TD>039</TD><TD>EPO</TD><TD>582</TD></TR></TBODY></TABLE></CENTER><P>The thermocycler programme was installed to have an extension time of 0 seconds and an annealing temperature of 65 °C. After the programme was finished, samples were kept at 4 °C.
</P><DL><DT>Date</DT><DD>20/09/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">PCR Clean-Up</H4><P>50 20 μL of each PCR reaction mixture was subjected to PCR Clean-Up, according to the <A href="https://2018.igem.org/Team:TUDelft/Experiments#PCRcleanup-scroll" class="adpbl" target="_blank">protocol</A>. Elution was done in 30 µL Milli-Q.</P><DL><DT>Date</DT><DD>20/09/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">Nanodrop DNA Quantification</H4><P>The following isolates were quantified according to the <A href="https://2018.igem.org/Team:TUDelft/Experiments#NanodropDNA-scroll" class="adpbl" target="_blank">Nanodrop DNA Quantification protocol</A>. The concentration and purity of the DNA (260/280 and 260/230 ratios) were measured. </P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Isolate</TH><TH class="tableheaderadpbl">Concentration (ng/uL)</TH><TH class="tableheaderadpbl">260/280 ratio</TH><TH class="tableheaderadpbl">260/230 ratio</TH></TR><TR class="tableunevenadpbl"><TD>EPO (1)</TD><TD>144.2</TD><TD>1.89</TD><TD>1.94</TD></TR><TR class="tableevenadpbl"><TD>EPO (2)</TD><TD>150.9</TD><TD>1.88</TD><TD>2.06</TD></TR><TR class="tableunevenadpbl"><TD>EPO (3)</TD><TD>128.2</TD><TD>1.88</TD><TD>1.99</TD></TR></TBODY></TABLE></CENTER></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#FridayPSS3">Friday, 21/09/2018</BUTTON><DIV id="FridayPSS3" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>21/09/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">d-AuNPs Testing: ssDNAp interaction with EPO target DNA</H4><P>The prescreening with d-AuNPs is tested with EPO DNA as target. Prior, the ssDNAp is heated up to let the ssDNAp anneal to EPO target DNA and create the secondary structure. The ssDNAp, EPO target DNA and hybridization buffer were incubated:</P><UL class="uladpbl"><LI>5 minutes at 95 ºC (denature dsDNA target)</LI><LI>1 minute at 57 ºC (anneal ssDNAp to dsDNA target)</LI><LI>10 minutes at 21 ºC</LI></UL><P>20 µL d-AuNPs, 10 µL target dsDNA, 6 µL hybridization buffer, 4 µL <A href="https://2018.igem.org/Team:TUDelft/Primers#ssDNAp" target="_blank" class="adpbl">ssDNAp 1</A> and 20 µL of a varying salt concentration (range of 0 mM to 1000 mM) were added to the 96 wells plate. The plate was incubated at 21 ºC during 10 minutes. The visible absorption spectrum from 450 nm to 650 nm of the solutions was measure and the ratio 620/520 nm was calculated for analysis. The experiment was performed in triplicate. The plate view, relevant absorption spectra and ratio 620/520 nm is shown.</P><P>The aggregation of d-AuNPs due to increasing salt concentration starts at 500 mM of NaCl. When a ssDNAp is added, the aggregation is extended to a NaCl concentration of 600 mM. However, presence of dsDNA target with a ssDNAp that can specifically bind to the target dsDNA does not influence the aggregation behavior drastically compared to the aggregation when dsDNA target is absent. Presence of only dsDNA target does decrease the stabilization of the d-AuNPs as they already aggregate at a NaCl added of 250 mM.</P><DL><DT>Date</DT><DD>21/09/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">d-AuNPs Testing: ssDNAp interaction with EPO target DNA</H4><P>The prescreening with d-AuNPs is tested with EPO DNA as target. The ssDNAp concentration is decreased to avoid to much stabilization of d-AuNPs by the ssDNAp. Prior, the ssDNAp is heated up to let the ssDNAp anneal to EPO target DNA and create the secondary structure. The ssDNAp, EPO target DNA and hybridization buffer were incubated:</P><UL class="uladpbl"><LI>5 minutes at 95 ºC (denature dsDNA target)</LI><LI>1 minute at 57 ºC (anneal ssDNAp to dsDNA target)</LI><LI>10 minutes at 21 ºC</LI></UL><P>20 µL d-AuNPs, 10 µL target dsDNA, 6 µL hybridization buffer, 4 µL <A href="https://2018.igem.org/Team:TUDelft/Primers#ssDNAp" target="_blank" class="adpbl">ssDNAp 1</A> (1nM) and 20 µL of a varying salt concentration (range of 0 mM to 1000 mM) were added to the 96 wells plate. Again, the primer 051 was used as off-target ssDNAp. The plate was incubated at 21 ºC during 10 minutes. The visible absorption spectrum from 450 nm to 650 nm of the solutions was measure and the ratio 620/520 nm was calculated for analysis. The experiment was performed in triplicate. The plate view, relevant absorption spectra and ratio 620/520 nm is shown.</P><DL><DT>Date</DT><DD>21/09/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">d-AuNPs Testing: ssDNAp interaction with EPO target DNA</H4><P>The prescreening with d-AuNPs is tested with EPO DNA as target. The ssDNAp concentration is decreased to avoid to much stabilization of d-AuNPs by the ssDNAp. Prior, the ssDNAp is heated up to let the ssDNAp anneal to EPO target DNA and create the secondary structure. The ssDNAp, EPO target DNA and hybridization buffer were incubated:</P><UL class="uladpbl"><LI>5 minutes at 95 ºC (denature dsDNA target)</LI><LI>1 minute at 57 ºC (anneal ssDNAp to dsDNA target)</LI><LI>10 minutes at 21 ºC</LI></UL><P>20 µL d-AuNPs, 10 µL target dsDNA, 6 µL hybridization buffer, 4 µL <A href="https://2018.igem.org/Team:TUDelft/Primers#ssDNAp" target="_blank" class="adpbl">ssDNAp 1</A> (100nM) and 20 µL of a varying salt concentration (range of 0 mM to 1000 mM) were added to the 96 wells plate. Again, the primer 051 was used as off-target ssDNAp. The plate was incubated at 21 ºC during 10 minutes. The visible absorption spectrum from 450 nm to 650 nm of the solutions was measure and the ratio 620/520 nm was calculated for analysis. The experiment was performed in triplicate. The plate view, relevant absorption spectra and ratio 620/520 nm is shown.</P><DL><DT>Date</DT><DD>21/09/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">d-AuNPs Testing: ssDNAp interaction with EPO target DNA</H4><P>The prescreening with d-AuNPs is tested with EPO DNA as target. The ssDNAp concentration is increased to check the stabilization effect on d-AuNPs by the ssDNAp. Prior, the ssDNAp is heated up to let the ssDNAp anneal to EPO target DNA and create the secondary structure. The ssDNAp, EPO target DNA and hybridization buffer were incubated:</P><UL class="uladpbl"><LI>5 minutes at 95 ºC (denature dsDNA target)</LI><LI>1 minute at 57 ºC (anneal ssDNAp to dsDNA target)</LI><LI>10 minutes at 21 ºC</LI></UL><P>20 µL d-AuNPs, 10 µL target dsDNA, 6 µL hybridization buffer, 4 µL <A href="https://2018.igem.org/Team:TUDelft/Primers#ssDNAp" target="_blank" class="adpbl">ssDNAp 1</A> (100nM) and 20 µL of a varying salt concentration (range of 0 mM to 1000 mM) were added to the 96 wells plate. Again, the primer 051 was used as off-target ssDNAp. The plate was incubated at 21 ºC during 10 minutes. The visible absorption spectrum from 450 nm to 650 nm of the solutions was measure and the ratio 620/520 nm was calculated for analysis. The experiment was performed in triplicate. The plate view, relevant absorption spectra and ratio 620/520 nm is shown.</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV></DIV></UL></FORM><H3 class="adpbl">October</H3><FORM><LABEL for="WeekPSO1" class="subdropdown cadpbl">Week 1 01/10/2018 - 07/10/2018</LABEL><UL class="submenu"><DIV class="subcontent"><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#TuesdayPSO1">Tuesday, 02/10/2018</BUTTON><DIV id="TuesdayPSO1" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>02/10/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">d-AuNPs Testing: ssDNAp interaction with EPO target DNA</H4><P>The prescreening with d-AuNPs is tested with EPO DNA as target. To test the sensitivity of the prescreening, the EPO target DNA is lowered to 100 pM. Prior, the ssDNAp is heated up to let the ssDNAp anneal to EPO target DNA and create the secondary structure. The ssDNAp, EPO target DNA and hybridization buffer were incubated:</P><UL class="uladpbl"><LI>5 minutes at 95 ºC (denature dsDNA target)</LI><LI>1 minute at 57 ºC (anneal ssDNAp to dsDNA target)</LI><LI>10 minutes at 21 ºC</LI></UL><P>20 µL d-AuNPs, 10 µL target dsDNA (1 nM and 100 pM), 6 µL hybridization buffer, 4 µL <A href="https://2018.igem.org/Team:TUDelft/Primers#ssDNAp" target="_blank" class="adpbl">ssDNAp 1</A> (1 µM) and 20 µL of a varying salt concentration (range of 0 mM to 1000 mM) were added to the 96 wells plate. Again, the primer 051 was used as off-target ssDNAp. The plate was incubated at 21 ºC during 10 minutes. The visible absorption spectrum from 450 nm to 650 nm of the solutions was measure and the ratio 620/520 nm was calculated for analysis. The experiment was performed in triplicate. The plate view, relevant absorption spectra and ratio 620/520 nm is shown.</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV></DIV></UL></FORM><FORM><LABEL for="WeekPSO2" class="subdropdown cadpbl">Week 2 08/10/2018 - 14/10/2018</LABEL><UL class="submenu"><DIV class="subcontent"><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#MondayPSO2">Monday, 08/10/2018</BUTTON><DIV id="MondayPSO2" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>08/10/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">d-AuNPs Testing: ssDNAp interaction with EPO target DNA</H4><P>The prescreening with d-AuNPs is tested with EPO DNA as target. To test the versatility of the prescreening, multiple variants of the ssDNAp were designed to target the EPO DNA. The ssDNAp 2 targets the EPO coding sequence. Prior, the ssDNAp is heated up to let the ssDNAp anneal to EPO target DNA and create the secondary structure. The <A href="https://2018.igem.org/Team:TUDelft/Primers#ssDNAp" target="_blank" class="adpbl">ssDNAp 2</A> was used in this experiment to target the EPO DNA. The ssDNAp 2, EPO target DNA and hybridization buffer were incubated:</P><UL class="uladpbl"><LI>5 minutes at 95 ºC (denature dsDNA target)</LI><LI>1 minute at 57 ºC (anneal ssDNAp to dsDNA target)</LI><LI>10 minutes at 21 ºC</LI></UL><P>20 µL d-AuNPs, 10 µL target dsDNA (1 µM), 6 µL hybridization buffer, 4 µL <A href="https://2018.igem.org/Team:TUDelft/Primers#ssDNAp" target="_blank" class="adpbl">ssDNAp 2</A> (1 µM) and 20 µL of a varying salt concentration (range of 0 mM to 1000 mM) were added to the 96 wells plate. Again, the primer 051 was used as off-target ssDNAp. The plate was incubated at 21 ºC during 10 minutes. The visible absorption spectrum from 450 nm to 650 nm of the solutions was measure and the ratio 620/520 nm was calculated for analysis. The experiment was performed in triplicate. The plate view, relevant absorption spectra and ratio 620/520 nm is shown.</P><DL><DT>Date</DT><DD>08/10/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">d-AuNPs Testing: ssDNAp interaction with EPO target DNA</H4><P>The prescreening with d-AuNPs is tested with EPO DNA as target. To test the versatility of the prescreening, multiple variants of the ssDNAp were designed to target the EPO DNA. The ssDNAp 3 targets the EPO coding sequence. Prior, the ssDNAp is heated up to let the ssDNAp anneal to EPO target DNA and create the secondary structure. The <A href="https://2018.igem.org/Team:TUDelft/Primers#ssDNAp" target="_blank" class="adpbl">ssDNAp 3</A> was used in this experiment to target the EPO DNA. The ssDNAp 3, EPO target DNA and hybridization buffer were incubated:</P><UL class="uladpbl"><LI>5 minutes at 95 ºC (denature dsDNA target)</LI><LI>1 minute at 57 ºC (anneal ssDNAp to dsDNA target)</LI><LI>10 minutes at 21 ºC</LI></UL><P>20 µL d-AuNPs, 10 µL target dsDNA (1 µM), 6 µL hybridization buffer, 4 µL <A href="https://2018.igem.org/Team:TUDelft/Primers#ssDNAp" target="_blank" class="adpbl">ssDNAp 3</A> (1 µM) and 20 µL of a varying salt concentration (range of 0 mM to 1000 mM) were added to the 96 wells plate. Again, the primer 051 was used as off-target ssDNAp. The plate was incubated at 21 ºC during 10 minutes. The visible absorption spectrum from 450 nm to 650 nm of the solutions was measure and the ratio 620/520 nm was calculated for analysis. The experiment was performed in triplicate. The plate view, relevant absorption spectra and ratio 620/520 nm is shown.</P><DL><DT>Date</DT><DD>08/10/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">d-AuNPs Testing: ssDNAp interaction with EPO target DNA</H4><P>The prescreening with d-AuNPs is tested with EPO DNA as target. To test the versatility of the prescreening, multiple variants of the ssDNAp were designed to target the EPO DNA. The ssDNAp 4 targets the EPO coding sequence. Prior, the ssDNAp is heated up to let the ssDNAp anneal to EPO target DNA and create the secondary structure. The <A href="https://2018.igem.org/Team:TUDelft/Primers#ssDNAp" target="_blank" class="adpbl">ssDNAp 4</A> was used in this experiment to target the EPO DNA. The ssDNAp 4, EPO target DNA and hybridization buffer were incubated:</P><UL class="uladpbl"><LI>5 minutes at 95 ºC (denature dsDNA target)</LI><LI>1 minute at 57 ºC (anneal ssDNAp to dsDNA target)</LI><LI>10 minutes at 21 ºC</LI></UL><P>20 µL d-AuNPs, 10 µL target dsDNA (1 µM), 6 µL hybridization buffer, 4 µL <A href="https://2018.igem.org/Team:TUDelft/Primers#ssDNAp" target="_blank" class="adpbl">ssDNAp 4</A> (1 µM) and 20 µL of a varying salt concentration (range of 0 mM to 1000 mM) were added to the 96 wells plate. Again, the primer 051 was used as off-target ssDNAp. The plate was incubated at 21 ºC during 10 minutes. The visible absorption spectrum from 450 nm to 650 nm of the solutions was measure and the ratio 620/520 nm was calculated for analysis. The experiment was performed in triplicate. The plate view, relevant absorption spectra and ratio 620/520 nm is shown.</P><DL><DT>Date</DT><DD>08/10/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">d-AuNPs Testing: ssDNAp interaction with EPO target DNA</H4><P>The prescreening with d-AuNPs is tested with EPO DNA as target. To test the versatility of the prescreening, multiple variants of the ssDNAp were designed to target the EPO DNA. The ssDNAp 5 targets the EPO coding sequence. Prior, the ssDNAp is heated up to let the ssDNAp anneal to EPO target DNA and create the secondary structure. The <A href="https://2018.igem.org/Team:TUDelft/Primers#ssDNAp" target="_blank" class="adpbl">ssDNAp 5</A> was used in this experiment to target the EPO DNA. The ssDNAp 5, EPO target DNA and hybridization buffer were incubated:</P><UL class="uladpbl"><LI>5 minutes at 95 ºC (denature dsDNA target)</LI><LI>1 minute at 57 ºC (anneal ssDNAp to dsDNA target)</LI><LI>10 minutes at 21 ºC</LI></UL><P>20 µL d-AuNPs, 10 µL target dsDNA (1 µM), 6 µL hybridization buffer, 4 µL <A href="https://2018.igem.org/Team:TUDelft/Primers#ssDNAp" target="_blank" class="adpbl">ssDNAp 5</A> (1 µM) and 20 µL of a varying salt concentration (range of 0 mM to 1000 mM) were added to the 96 wells plate. Again, the primer 051 was used as off-target ssDNAp. The plate was incubated at 21 ºC during 10 minutes. The visible absorption spectrum from 450 nm to 650 nm of the solutions was measure and the ratio 620/520 nm was calculated for analysis. The experiment was performed in triplicate. The plate view, relevant absorption spectra and ratio 620/520 nm is shown.</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#TuesdayPSO2">Tuesday, 09/10/2018</BUTTON><DIV id="TuesdayPSO2" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>09/10/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">d-AuNPs Testing: ssDNAp interaction with EPO target DNA</H4><P>The prescreening with d-AuNPs is tested with EPO DNA as target. To test the effect of background DNA on the ability to screen for target DNA, linear DNA was added. The <A href="https://2018.igem.org/Team:TUDelft/Primers#ssDNAp" target="_blank" class="adpbl">ssDNAp 1</A> was used in this experiment to target the EPO DNA while background DNA is present. The ssDNAp 1, EPO target DNA, background DNA and hybridization buffer were incubated:</P><UL class="uladpbl"><LI>5 minutes at 95 ºC (denature dsDNA target)</LI><LI>1 minute at 57 ºC (anneal ssDNAp to dsDNA target)</LI><LI>10 minutes at 21 ºC</LI></UL><P>20 µL d-AuNPs, 10 µL target dsDNA (1 µM), 6 µL hybridization buffer, 4 µL <A href="https://2018.igem.org/Team:TUDelft/Primers#ssDNAp" target="_blank" class="adpbl">ssDNAp 1</A> (1 µM) and 20 µL of a varying salt concentration (range of 0 mM to 1000 mM) were added to the 96 wells plate. Again, the primer 051 was used as off-target ssDNAp. The plate was incubated at 21 ºC during 10 minutes. The visible absorption spectrum from 450 nm to 650 nm of the solutions was measure and the ratio 620/520 nm was calculated for analysis. The experiment was performed in triplicate. The plate view, relevant absorption spectra and ratio 620/520 nm is shown.</P><DL><DT>Date</DT><DD>09/10/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">d-AuNPs Testing: ssDNAp interaction with EPO target DNA</H4><P>The prescreening with d-AuNPs is tested with EPO DNA as target. To test the effect of background DNA on the ability to screen for target DNA, linear DNA was added. The <A href="https://2018.igem.org/Team:TUDelft/Primers#ssDNAp" target="_blank" class="adpbl">ssDNAp 1</A> was used in this experiment to target the EPO DNA while background DNA is present. The ssDNAp 1, EPO target DNA, background DNA and hybridization buffer were incubated:</P><UL class="uladpbl"><LI>5 minutes at 95 ºC (denature dsDNA target)</LI><LI>1 minute at 57 ºC (anneal ssDNAp to dsDNA target)</LI><LI>10 minutes at 21 ºC</LI></UL><P>20 µL d-AuNPs, 10 µL target dsDNA (1 µM), 6 µL hybridization buffer, 4 µL <A href="https://2018.igem.org/Team:TUDelft/Primers#ssDNAp" target="_blank" class="adpbl">ssDNAp 2</A> (1 µM) and 20 µL of a varying salt concentration (range of 0 mM to 1000 mM) were added to the 96 wells plate. Again, the primer 051 was used as off-target ssDNAp. The plate was incubated at 21 ºC during 10 minutes. The visible absorption spectrum from 450 nm to 650 nm of the solutions was measure and the ratio 620/520 nm was calculated for analysis. The experiment was performed in triplicate. The plate view, relevant absorption spectra and ratio 620/520 nm is shown.
In order to check the versatility of the prescreening, the effect of background DNA present in the sample, the d-AuNPs test is also performed with <A href="https://2018.igem.org/Team:TUDelft/Primers#ssDNAp" target="_blank" class="adpbl">ssDNAp 3, 4 and 5</A>.</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV></DIV></UL></FORM></DIV></DIV></DIV><DIV class="container nowbg" id="StrainConstruction"><DIV class="row nowbg"><DIV class="col-xs-12 col-sm-3 col-lg-3"><NAV id="myScrollspy" class="hidden-xs hidden-sm"><UL class="nav nav-pills nav-stacked" data-spy="affix"><LI class="dropdown"><A class="dropdown-toggle adpbl" data-toggle="dropdown" href="#">May</A></LI><LI><A href="#MayStrainCon" class="adpbl">Week: 28/05/2018 - 03/06/2018</A></LI></UL><LI class="dropdown"><A class="dropdown-toggle adpbl" data-toggle="dropdown" href="#">June</A></LI><LI><A href="#JuneStrainCon1" class="adpbl">Week: 04/06/2018 - 10/06/2018</A></LI><LI><A href="#JuneStrainCon2" class="adpbl">Week: 11/06/2018 - 17/06/2018</A></LI><LI><A href="#JuneStrainCon3" class="adpbl">Week: 18/06/2018 - 24/06/2018</A></LI><LI><A href="#JuneStrainCon4" class="adpbl">Week: 25/06/2018 - 01/07/2018</A></LI><LI class="dropdown"><A class="dropdown-toggle adpbl" data-toggle="dropdown" href="#">July</A></LI><LI><A href="#JulyStrainCon1" class="adpbl">Week: 02/07/2018 - 08/07/2018</A></LI><LI><A href="#JulyStrainCon2" class="adpbl">Week: 09/07/2018 - 15/07/2018</A></LI><LI><A href="#JulyStrainCon3" class="adpbl">Week: 16/07/2018 - 22/07/2018</A></LI><LI class="dropdown"><A class="dropdown-toggle adpbl" data-toggle="dropdown" href="#">August</A></LI><LI><A href="#AugustStrainCon1" class="adpbl">Week: 30/07/2018 - 05/08/2018</A></LI><LI><A href="#AugustStrainCon2" class="adpbl">Week: 06/08/2018 - 12/08/2018</A></LI><LI><A href="#AugustStrainCon3" class="adpbl">Week: 13/08/2018 - 19/08/2018</A></LI><LI><A href="#AugustStrainCon4" class="adpbl">Week: 20/08/2018 - 26/08/2018</A></LI><LI><A href="#AugustStrainCon5" class="adpbl">Week: 27/08/2018 - 02/09/2018</A></LI><LI class="dropdown"><A class="dropdown-toggle adpbl" data-toggle="dropdown" href="#">September</A></LI><LI><A href="#SeptemberStrainCon1" class="adpbl">Week: 03/09/2018 - 09/09/2018</A></LI><LI><A href="#SeptemberStrainCon2" class="adpbl">Week: 10/09/2018 - 16/09/2018</A></LI><LI><A href="#SeptemberStrainCon3" class="adpbl">Week: 17/09/2018 - 23/09/2018</A></LI><LI><A href="#SeptemberStrainCon4" class="adpbl">Week: 24/09/2018 - 30/09/2018</A></LI><LI class="drowdown"><A class="dropdown-toggle adpbl" data-toggle="dropdown" href="#">October</A></LI><LI><A href="#OctoberStrainCon1" class="adpbl">Week: 01/10/2018 - 07/10/2018</A></LI><LI><A href="#OctoberStrainCon2" class="adpbl">Week: 08/10/2018 - 14/10/2018</A></LI></NAV></DIV><DIV class="col-lg-8 col-md-8 col-sm-12 col-xs-12 nowbg" style="margin-right:20px;"><H3 class="adpbl">May</H3><FORM><LABEL for="WeekSCM1" class="subdropdown cadpbl">Week 5 28/05/2018 - 03/06/2018</LABEL><UL class="submenu"><DIV class="subcontent"><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#ThursdaySCM">Thursday, 30/05/2018</BUTTON><DIV id="ThursdaySCM" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>30/5/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>Plasmid Isolation</H4><P> 
    An overnight starter culture of <I>E. coli</I> DH5α harboring plasmid pTXB1-Tn5 was subjected to plasmid isolation following the Plasmid Isolation Protocol. Final elution was done with 30 μL of pre-warmed MilliQ.
</P><DL><DT>Date</DT><DD>30/5/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>Nanodrop DNA Quantification</H4><P> 
    The following isolates were quantified according to the Nanodrop DNA Quantification protocol. The concentration and purity of the DNA (260/280 and 260/230 ratios) were measured. 
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Isolate</TH><TH class="tableheaderadpbl">Concentration (ng/uL)</TH><TH class="tableheaderadpbl">260/280 ratio</TH><TH class="tableheaderadpbl">260/230 ratio</TH></TR><TR class="tableunevenadpbl"><TD>pTXB1-Tn5</TD><TD>36.3</TD><TD>1.92</TD><TD>2.06</TD></TR><TR class="tableevenadpbl"><TD>pTXB1-Tn5</TD><TD>40.4</TD><TD>1.88</TD><TD>2.17</TD></TR><TR class="tableunevenadpbl"><TD>pTXB1-Tn5</TD><TD>47</TD><TD>1.82</TD><TD>1.75</TD></TR></TBODY></TABLE></CENTER></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#FridaySCM">Friday, 01/06/2018</BUTTON><DIV id="FridaySCM" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>1/6/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>Primer working stock preparation</H4><P> The following primers were resuspended in sterile Milli-Q water, in order to make 100µM stock solutions.  
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Primer name</TH><TH class="tableheaderadpbl">Amount (ng)</TH><TH class="tableheaderadpbl">Volume (uL)</TH></TR><TR class="tableunevenadpbl"><TD>001</TD><TD>290</TD><TD>277</TD></TR><TR class="tableevenadpbl"><TD>002</TD><TD>380</TD><TD>304</TD></TR><TR class="tableunevenadpbl"><TD>003</TD><TD>280</TD><TD>377</TD></TR><TR class="tableevenadpbl"><TD>004</TD><TD>280</TD><TD>344</TD></TR><TR class="tableunevenadpbl"><TD>005</TD><TD>4nmol</TD><TD>40*</TD></TR><TR class="tableevenadpbl"><TD>006</TD><TD>260</TD><TD>255</TD></TR><TR class="tableunevenadpbl"><TD>007</TD><TD>290</TD><TD>284</TD></TR><TR class="tableevenadpbl"><TD>008</TD><TD>4nmol</TD><TD>40*</TD></TR><TR class="tableunevenadpbl"><TD>009</TD><TD>250</TD><TD>290</TD></TR><TR class="tableevenadpbl"><TD>010</TD><TD>270</TD><TD>320</TD></TR><TR class="tableunevenadpbl"><TD>011</TD><TD>220</TD><TD>308</TD></TR><TR class="tableevenadpbl"><TD>012</TD><TD>200</TD><TD>301</TD></TR><TR class="tableunevenadpbl"><TD>013</TD><TD>220</TD><TD>357</TD></TR><TR class="tableevenadpbl"><TD>014</TD><TD>180</TD><TD>301</TD></TR><TR class="tableunevenadpbl"><TD>015</TD><TD>200</TD><TD>334</TD></TR><TR class="tableevenadpbl"><TD>016</TD><TD>200</TD><TD>330</TD></TR><TR class="tableunevenadpbl"><TD>017</TD><TD>190</TD><TD>350</TD></TR><TR class="tableevenadpbl"><TD>018</TD><TD>170</TD><TD>348</TD></TR><TR class="tableunevenadpbl"><TD>019</TD><TD>290</TD><TD>345</TD></TR><TR class="tableevenadpbl"><TD>020</TD><TD>220</TD><TD>365</TD></TR><TR class="tableunevenadpbl"><TD>021</TD><TD>240</TD><TD>343</TD></TR><TR class="tableevenadpbl"><TD>022</TD><TD>230</TD><TD>334</TD></TR><TR class="tableunevenadpbl"><TD>023</TD><TD>200</TD><TD>319</TD></TR><TR class="tableevenadpbl"><TD>024</TD><TD>230</TD><TD>336</TD></TR><TR class="tableunevenadpbl"><TD>025</TD><TD>220</TD><TD>339</TD></TR><TR class="tableevenadpbl"><TD>026</TD><TD>160</TD><TD>355</TD></TR><TR class="tableunevenadpbl"><TD>027</TD><TD>150</TD><TD>301</TD></TR><TR class="tableevenadpbl"><TD>028</TD><TD>170</TD><TD>343</TD></TR></TBODY></TABLE></CENTER><P> The primers in these 100 µM stocks were heated to 60 °C for 20 minutes and centrifuged  at maximum speed (~17,000 x g) for 2 minutes. 10µM primer working stock solutions were prepared by making 10x dilution aliquots with sterile Milli-Q, each in a total volume of 100µl. </P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV></DIV></UL></FORM><H3 class="adpbl">June</H3><FORM><LABEL for="WeekSC1" class="subdropdown cadpbl">Week 1 04/06/2018 - 10/06/2018</LABEL><UL class="submenu"><DIV class="subcontent"><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#MondaySC1">Monday, 04/06/2018</BUTTON><DIV id="MondaySC1" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>04/06/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">High Fidelity PCR</H4><P> 
To amplify Linker-Tn5 (1508 bp), Tn5 (1452 bp) and Tn5-Linker (1505 bp) coding sequence, high fidelity PCR was done using pTXB1-Tn5 as template DNA and FP005, FP006, FP007 and FP008 as forward and reverse primers. Two 50µL PCR mixtures were prepared, according to the <A href="https://2018.igem.org/Team:TUDelft/Experiments#highfidelityPCR-scroll" target="_blank" class="adpbl">High Fidelity PCR protocol</A>. 
Primer combination(s) and expected fragment(s)
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Sample</TH><TH class="tableheaderadpbl">Fw primer</TH><TH class="tableheaderadpbl">Rv primer</TH><TH class="tableheaderadpbl">Expected fragment</TH><TH class="tableheaderadpbl">Expected size (bp) </TH></TR><TR class="tableunevenadpbl"><TD>1</TD><TD>005</TD><TD>006</TD><TD>Linker-Tn5</TD><TD>1508</TD></TR><TR class="tableevenadpbl"><TD>2</TD><TD>007</TD><TD>006</TD><TD>Tn5</TD><TD>1452</TD></TR><TR class="tableunevenadpbl"><TD>3</TD><TD>007</TD><TD>008</TD><TD>Tn5-Linker</TD><TD>1505</TD></TR></TBODY></TABLE></CENTER><P>The thermocycler programme was installed to have an extension time of 1 min 30 seconds and an annealing temperature of 55 °C for one of the tubes and 60 °C for the second tubes. After the programme was finished, samples were kept at 4 °C.
</P><DL><DT>Date</DT><DD>04-06-2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">DNA Gel Electrophoresis</H4><P>The obtained PCR products Linker-Tn5, Tn5, Tn5-Linker were run on a 0.8% agarose gel in TAE buffer. 4 μL was pipetted into each lane. The gel was imaged using a gel documentation system with UV-light.
</P><P>
The fragments that should be amplified by the PCR are respectively 1508, 1452 and 1505 bp in size. As can be seen on the gel, only the Tn5 amplification was successfully amplified. The Linker-Tn5 and Tn5-Linker fragments were not successfully amplified.          
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#TuesdaySC1">Tuesday, 05/06/2018</BUTTON><DIV id="TuesdaySC1" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>05/06/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">PCR clean-up</H4><P>
    Two times 50 μL of PCR product Tn5 was subjected to PCR Clean-Up, according to the <A href="https://2018.igem.org/Team:TUDelft/Experiments#PCRcleanup-scroll" class="adpbl" target="_blank">DNA clean up protocol</A>. Elution was done in 30 µL prewarmed Milli-Q.
</P><DL><DT>Date</DT><DD>05-06-2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">Nanodrop DNA Quantification</H4><P> 
          The following isolates were quantified according to the <A href="https://2018.igem.org/Team:TUDelft/Experiments#NanodropDNA-scroll" class="adpbl" target="_blank">Nanodrop DNA Quantification protocol</A>. The concentration and purity of the DNA (260/280 and 260/230 ratios) were measured. 
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Isolate</TH><TH class="tableheaderadpbl">Concentration (ng/uL)</TH><TH class="tableheaderadpbl">260/280 ratio</TH><TH class="tableheaderadpbl">260/230 ratio</TH></TR><TR class="tableunevenadpbl"><TD>Tn5 (annealed at 55 °C)</TD><TD>59.6</TD><TD>1.88</TD><TD>1.65</TD></TR><TR class="tableevenadpbl"><TD>Tn5 (annealed at 60 °C)</TD><TD>47.4</TD><TD>1.82</TD><TD>0.59</TD></TR></TBODY></TABLE></CENTER><DL><DT>Date</DT><DD>05/06/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">Liquid Starter Culture Preparation</H4><P><I>E.coli</I> strain harbouring the plasmid pTXB1-Tn5 was used for inoculation of 10 mL liquid starter culture (Luria Broth, complemented with ampicillin) following the <A href="https://2018.igem.org/Team:TUDelft/Experiments#liquidstarterculture-scroll" class="adpbl" target="_blank">Liquid Starter Culture Protocol</A>.
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#WednesdaySC1">Wednesday, 06/06/2018</BUTTON><DIV id="WednesdaySC1" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>06/06/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">Restriction</H4><P>
    Restriction cloning was done for assembling Tn5 into pACYC-Duet. The following table shows an overview of what fragment(s) were cut with which restriction enzyme(s).
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL)</TH></TR><TR class="tableunevenadpbl"><TD>10x CutSmart buffer</TD><TD>2.5</TD></TR><TR class="tableevenadpbl"><TD>Fragment (pACYC-Duet)</TD><TD>10 (1000 ng)</TD></TR><TR class="tableunevenadpbl"><TD>Enzyme (EcoRI)</TD><TD>1</TD></TR><TR class="tableevenadpbl"><TD>Enzyme (KpnI)</TD><TD>1</TD></TR><TR class="tableunevenadpbl"><TD>MilliQ</TD><TD>10.5</TD></TR></TBODY></TABLE></CENTER><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL)</TH></TR><TR class="tableunevenadpbl"><TD>10x CutSmart buffer</TD><TD>2.5</TD></TR><TR class="tableevenadpbl"><TD>Fragment (Tn5)</TD><TD>20.5 (1000 ng)</TD></TR><TR class="tableunevenadpbl"><TD>Enzyme (EcoRI)</TD><TD>1</TD></TR><TR class="tableevenadpbl"><TD>Enzyme (KpnI)</TD><TD>1</TD></TR><TR class="tableunevenadpbl"><TD>MilliQ</TD><TD>0</TD></TR></TBODY></TABLE></CENTER><P>The samples were incubated for 2 hours at 37 °C. 
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#FridaySC1">Friday, 08/06/2018</BUTTON><DIV id="FridaySC1" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>08/06/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">PCR Clean up</H4><P>
          25 μL of restriction mixture with Tn5 or pACYCDuet-1 was subjected to DNA Clean-Up, according to the <A href="https://2018.igem.org/Team:TUDelft/Experiments#PCRcleanup-scroll" class="adpbl" target="_blank">protocol</A>. Elution was done in 30 µL prewarmed Milli-Q.
</P><DL><DT>Date</DT><DD>08/06/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">Nanodrop DNA Quantification</H4><P> 
          The following isolates were quantified according to the <A href="https://2018.igem.org/Team:TUDelft/Experiments#NanodropDNA-scroll" class="adpbl" target="_blank">Nanodrop DNA Quantification protocol</A>. The concentration and purity of the DNA (260/280 and 260/230 ratios) were measured. 
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Isolate</TH><TH class="tableheaderadpbl">Concentration (ng/uL)</TH><TH class="tableheaderadpbl">260/280 ratio</TH><TH class="tableheaderadpbl">260/230 ratio</TH></TR><TR class="tableunevenadpbl"><TD>Tn5 55 °C (EcoRI, KpnI restricted)</TD><TD>8.8</TD><TD>1.97</TD><TD>0.79</TD></TR><TR class="tableevenadpbl"><TD>Tn5 60 °C (EcoRI, KpnI restricted)</TD><TD>6.8</TD><TD>1.68</TD><TD>0.52</TD></TR><TR class="tableunevenadpbl"><TD>Duet-1 (EcoRI, KpnI restricted)</TD><TD>5.6</TD><TD>2.05</TD><TD>0.40</TD></TR></TBODY></TABLE></CENTER><DL><DT>Date</DT><DD>08/06/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">Ligation</H4><P>           
In order to assemble cut fragments Tn5 55 °C/60 °C and pACYCDuet-1, ligation was performed. This table gives an overview of which fragments were cut with which enzymes.
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">DNA</TH><TH class="tableheaderadpbl">Size (bp)</TH><TH class="tableheaderadpbl">Cut with enzymes</TH></TR><TR class="tableunevenadpbl"><TD>Vector pACYCDuet-1</TD><TD>3768</TD><TD>EcoRI, KnpI</TD></TR><TR class="tableevenadpbl"><TD>Fragment Tn5</TD><TD>1442</TD><TD>EcoRI, KpnI</TD></TR></TBODY></TABLE></CENTER><P>A 20 μL ligation reaction mixture was prepared with the following composition:
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL) </TH></TR><TR class="tableunevenadpbl"><TD>Ligase Buffer (10x) </TD><TD>2</TD></TR><TR class="tableevenadpbl"><TD>T4 DNA Ligase</TD><TD>1</TD></TR><TR class="tableunevenadpbl"><TD>DNA vector (~50 ng) </TD><TD>9</TD></TR><TR class="tableunevenadpbl"><TD>DNA fragment (~60 ng) </TD><TD>8</TD></TR><TR class="tableunevenadpbl"><TD>MilliQ</TD><TD>0</TD></TR></TBODY></TABLE></CENTER><P>The samples were incubated for 72 hours (over the weekend) at 4°C for ligation.</P><DL><DT>Date</DT><DD>08/06/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">High Fidelity PCR</H4><P>To amplify Linker-Tn5 (1508 bp), Tn5 (1452 bp) and Tn5-Linker (1505 bp) coding sequence, high fidelity PCR was done using pTXB1-Tn5 as template DNA and FP005, FP006, FP007 and FP008 as forward and reverse primers. As the PCR reaction was not successful before, different conditions were tested to optimize the PCR. Multiple 50µL PCR mixtures were prepared, according to the <A href="https://2018.igem.org/Team:TUDelft/Experiments#highfidelityPCR-scroll" target="_blank" class="adpbl">High Fidelity PCR protocol.</A> 
Primer combination(s) and expected fragment(s)
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Sample</TH><TH class="tableheaderadpbl">Fw primer</TH><TH class="tableheaderadpbl">Rv primer</TH><TH class="tableheaderadpbl">Expected fragment</TH><TH class="tableheaderadpbl">Expected size (bp) </TH></TR><TR class="tableunevenadpbl"><TD>1</TD><TD>005</TD><TD>006</TD><TD>Linker-Tn5</TD><TD>1508</TD></TR><TR class="tableevenadpbl"><TD>2</TD><TD>007</TD><TD>006</TD><TD>Tn5</TD><TD>1452</TD></TR><TR class="tableunevenadpbl"><TD>3</TD><TD>007</TD><TD>008</TD><TD>Tn5-Linker</TD><TD>1505</TD></TR></TBODY></TABLE></CENTER><P>
          The sets of conditions tested were:</P><UL class="uladpbl"><LI>Touch Down PCR with HF buffer</LI><LI>Touch Down PCR with GC buffer</LI><LI>2 step PCR with HF buffer</LI><LI>2 step PCR with GC buffer</LI></UL><P>
Touch Down PCR program
</P><CENTER><TABLE><TBODY><TR><TH class="tableheaderadpbl">Step</TH><TH class="tableheaderadpbl">Time (s)</TH><TH class="tableheaderadpbl">Temperature (°C)</TH></TR><TR class="tableunevenadpbl"><TD>Initial denaturation</TD><TD>30</TD><TD>98</TD></TR><TR class="tableevenadpbl"><TD>Denaturation</TD><TD>10</TD><TD>98</TD></TR><TR class="tableunevenadpbl"><TD>Annealing</TD><TD>15</TD><TD>60 down to 50</TD></TR><TR class="tableevenadpbl"><TD>Extension</TD><TD>1 min 30 sec</TD><TD>72</TD></TR><TR class="tableunevenadpbl"><TD>Final extension</TD><TD>300</TD><TD>72</TD></TR><TR class="tableevenadpbl"><TD>Hold</TD><TD>∞</TD><TD>4</TD></TR></TBODY></TABLE></CENTER><P>
2 Step PCR program </P><CENTER><TABLE><TBODY><TR><TH class="tableheaderadpbl">Step</TH><TH class="tableheaderadpbl">Time (s)</TH><TH class="tableheaderadpbl">Temperature (°C)</TH></TR><TR class="tableunevenadpbl"><TD>Initial denaturation</TD><TD>30</TD><TD>98</TD></TR><TR class="tableevenadpbl"><TD>Denaturation</TD><TD>10</TD><TD>98</TD></TR><TR class="tableunevenadpbl"><TD>Annealing</TD><TD>15</TD><TD>72</TD></TR><TR class="tableevenadpbl"><TD>Extension</TD><TD>1 min 30 sec</TD><TD>72</TD></TR><TR class="tableunevenadpbl"><TD>Final extension</TD><TD>300</TD><TD>72</TD></TR><TR class="tableevenadpbl"><TD>Hold</TD><TD>∞</TD><TD>4</TD></TR></TBODY></TABLE></CENTER><P>
After the programme was finished, samples were kept at 4 °C.
</P><DL><DT>Date</DT><DD>08/06/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">DNA Gel Electrophoresis</H4><P>The obtained PCR products Linker-Tn5, Tn5, Tn5-Linker were run on a 0.8% agarose gel in TAE buffer. 4 μL DNA was pipetted into each lane. The gel was imaged using a gel documentation system with UV-light.</P><P>The fragments that should be amplified by the PCR are respectively 1508, 1452 and 1505 bp in size. As can be seen on the gel, only the Linker-Tn5 amplification was successfully amplified in the 2 Step PCR program with GC buffer. The Linker-Tn5, Tn5 and Tn5-Linker fragments were not successfully amplified when a touch down PCR program was performed. </P><DL><DT>Date</DT><DD>08/06/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">High Fidelity PCR</H4><P> 
To amplify Linker-Tn5 (1508 bp), Tn5 (1452 bp) and Tn5-Linker (1505 bp) coding sequence, high fidelity PCR was done using the amplified Tn5 (in previous PCR reactions) as template DNA and FP005, FP006, FP007 and FP008 as forward and reverse primers. Two times eight 50µL PCR mixtures were prepared, according to the <A href="https://2018.igem.org/Team:TUDelft/Experiments#highfidelityPCR-scroll" target="_blank" class="adpbl">High Fidelity PCR protocol</A>. 
Primer combination(s) and expected fragment(s)
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Sample</TH><TH class="tableheaderadpbl">Fw primer</TH><TH class="tableheaderadpbl">Rv primer</TH><TH class="tableheaderadpbl">Expected fragment</TH><TH class="tableheaderadpbl">Expected size (bp) </TH></TR><TR class="tableunevenadpbl"><TD>1</TD><TD>005</TD><TD>006</TD><TD>Linker-Tn5</TD><TD>1508</TD></TR><TR class="tableunevenadpbl"><TD>2</TD><TD>007</TD><TD>008</TD><TD>Tn5-Linker</TD><TD>1505</TD></TR></TBODY></TABLE></CENTER><P>To test the effect of DMSO on the PCR performance, one extra tube per amplicon was made that was supplemented with 1.5 μL DMSO (3% final) and 34 μL MilliQ.
Additionally, instead of the amplified Tn5 PCR product as template, the original pTXB1-Tn5 plasmid was used as template. 
Finally, a negative control sample was taken into account that contained no primers and 37.5 μL MilliQ instead.
The thermocycler programme was installed to have an extension time of 1 min 30 seconds and an annealing temperature of 60 °C. In order to check the difference between two PCR machines (older one and new one), the tubes were divided over the two thermocyclers. After the programme was finished, samples were kept at 4 °C.
</P><DL><DT>Date</DT><DD>08/06/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">DNA Gel Electrophoresis</H4><P>The obtained PCR products Linker-Tn5 and Tn5-Linker were run on a 0.8% agarose gel in TAE buffer. 4 μL DNA was pipetted into each lane. The gel was imaged using a gel documentation system with UV-light.</P><P>As can be seen, the PCR amplification with primers 005 and 006 resulting in Linker-Tn5 were successful. The primers 007 and 008 did not result in any PCR amplicons. Also, the HF buffer supplemented with 3% DMSO resulted in the required amplicon. Remarkably, the old PCR machine did result in thicker and brighter bands DNA. This might be due to the differences in speed when changing temperature.
 </P><DL><DT>Date</DT><DD>08/06/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">Nanodrop DNA Quantification</H4><P> 
          The following isolates were quantified according to the <A href="https://2018.igem.org/Team:TUDelft/Experiments#NanodropDNA-scroll" class="adpbl" target="_blank">Nanodrop DNA Quantification protocol</A>. The concentration and purity of the DNA (260/280 and 260/230 ratios) were measured. 
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Isolate</TH><TH class="tableheaderadpbl">Concentration (ng/uL)</TH><TH class="tableheaderadpbl">260/280 ratio</TH><TH class="tableheaderadpbl">260/230 ratio</TH></TR><TR class="tableunevenadpbl"><TD>Linker-Tn5, 7 tubes (new machine) </TD><TD>117.9</TD><TD>1.82</TD><TD>1.35</TD></TR><TR class="tableevenadpbl"><TD>Linker-Tn5, 1 tube (old machine)</TD><TD>67.7</TD><TD>1.82</TD><TD>1.17</TD></TR></TBODY></TABLE></CENTER><DL><DT>Date</DT><DD>08-06-2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">High Fidelity PCR</H4><P> 
To amplify Linker-Tn5 (1508 bp), Tn5 (1452 bp) and Tn5-Linker (1505 bp) coding sequence, high fidelity PCR was done using pTXB1-Tn5 as template DNA and FP005, FP006, FP007 and FP008 as forward and reverse primers. Two times three 50µL PCR mixtures were prepared, according to the <A href="https://2018.igem.org/Team:TUDelft/Experiments#highfidelityPCR-scroll" target="_blank" class="adpbl">High Fidelity PCR protocol</A>. 
Primer combination(s) and expected fragment(s)
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Sample</TH><TH class="tableheaderadpbl">Fw primer</TH><TH class="tableheaderadpbl">Rv primer</TH><TH class="tableheaderadpbl">Expected fragment</TH><TH class="tableheaderadpbl">Expected size (bp) </TH></TR><TR class="tableunevenadpbl"><TD>1</TD><TD>005</TD><TD>006</TD><TD>Linker-Tn5</TD><TD>1508</TD></TR><TR class="tableevenadpbl"><TD>2</TD><TD>007</TD><TD>006</TD><TD>Tn5</TD><TD>1452</TD></TR><TR class="tableunevenadpbl"><TD>3</TD><TD>007</TD><TD>008</TD><TD>Tn5-Linker</TD><TD>1505</TD></TR></TBODY></TABLE></CENTER><P>The thermocycler programme was installed to have an extension time of 2 min 30 seconds (1 minute longer than PCR reactions before) and an annealing temperature of 60 °C. Again, in order to check the difference between two PCR machines (older one and new one), the tubes were divided over the two thermocyclers. After the programme was finished, samples were kept at 4 °C.
</P><DL><DT>Date</DT><DD>08-06-2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">DNA Gel Electrophoresis</H4><P>All the PCR amplicons were run on a 0.8% agarose gel in TAE buffer. 4 μL DNA was pipetted into each lane. The gel was imaged using a gel documentation system with UV-light.</P><P>
As can be seen, the PCR amplification with primers 005, 006, 007 and 008 resulting in Linker-Tn5, Tn5, Tn5-Linker (respective sizes: 1508, 1452 and 1505 bp) were successful. In contradiction with the previous results, the new PCR machine did result in thicker and brighter bands DNA and is therefore considered more efficient for this PCR reaction. From this experiment it can be concluded that the standard PCR conditions work sufficient for this purpose.
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV></DIV></UL></FORM><FORM><LABEL for="WeekSC2" class="subdropdown cadpbl">Week 2 11/06/2018 - 17/06/2018</LABEL><UL class="submenu"><DIV class="subcontent"><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#MondaySC2">Monday, 11/06/2018</BUTTON><DIV id="MondaySC2" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>11/06/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">Transformation of chemically competent cells</H4><P>
The following transformations were carried out according to the protocol for transformation of chemically competent cells:</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Strain</TH><TH class="tableheaderadpbl">Plasmid and size (bp)</TH><TH class="tableheaderadpbl">Cloning method</TH><TH class="tableheaderadpbl">Volume added (μl)</TH><TH class="tableheaderadpbl">Medium on which was plated</TH></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I> DH5α</TD><TD>pACYC1Duet-1_Tn5 (5210 bp)</TD><TD>Restriction/Ligation</TD><TD>20</TD><TD>LB Cam</TD></TR><TR class="tableevenadpbl"><TD><I>E.coli</I> DH5α</TD><TD>pACYCDuet-1_Tn5(5210 bp)</TD><TD>Restriction/Ligation</TD><TD>20</TD><TD>LB Cam</TD></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I> DH5α</TD><TD>pTXB1-Tn5</TD><TD>-</TD><TD>2</TD><TD>LB Amp</TD></TR><TR class="tableevenadpbl"><TD><I>E.coli</I> DH5α</TD><TD>pACYCDuet-1 (pos.control)</TD><TD>-</TD><TD>1</TD><TD>LB Cam</TD></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I> DH5α</TD><TD>none (neg. control)</TD><TD>Restriction/Ligation</TD><TD>0</TD><TD>LB Cam</TD></TR></TBODY></TABLE></CENTER><P>
200 μl LB medium was added as recovery medium.</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#TuesdaySC2">Tuesday, 12/06/2018</BUTTON><DIV id="TuesdaySC2" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>12/06/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">Colony PCR</H4><P>Colony PCR was done to verify propagation of pACYC1Duet-1_Tn5 after transformation from 11-06-2018. This plasmid contains the Tn5 insert, which can be checked with primers 029 and 030. As negative control, the colonies containing pACYCDuet-1 were checked according to the same protocol.</P><CENTER><TABLE><TBODY><TR class="tableunevenadpbl"><TD>Template DNA</TD><TD>pACYCDuet-1_Tn5</TD></TR><TR class="tableevenadpbl"><TD>Fw primer</TD><TD>029</TD></TR><TR class="tableunevenadpbl"><TD>Rv primer</TD><TD>030</TD></TR><TR class="tableevenadpbl"><TD>expected amplicon size (bp)</TD><TD>1686 bp</TD></TR></TBODY></TABLE></CENTER><P>
Six tubes were prepared for colony PCR, containing pACYCDuet-1_Tn5 as template DNA. Additionally, one tube with pACYCDuet-1 was taken along as positive control. The thermocycler programme was installed to have an extension time of 2 min and an annealing temperature of 60 °C. After the programme was finished, samples were kept at 4°C.</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#WednesdaySC2">Wednesday, 13/06/2018</BUTTON><DIV id="WednesdaySC2" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>13/06/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">DNA Gel Electrophoresis</H4><P>The PCR amplicons of the colony PCR performed on 12-06-2018 were run on a 0.8% agarose gel in TAE buffer. 4μL was pipetted into each lane. The gel was imaged using a gel documentation system with UV-light.</P><P>Unfortunately, the PCR reaction did not result in the required PCR amplicons. However, the positive control colony (pACYCDuet-1) did also not results in any PCR amplicon. Therefore, it is likely that the PCR reaction itself was not successful and needs to be repeated.</P><DL><DT>Date</DT><DD>13-06-2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">PCR Clean-Up</H4><P>The PCR amplicons made on 08-06-2018 were subjected to PCR Clean-Up, according to the <A href="https://2018.igem.org/Team:TUDelft/Experiments#PCRcleanup-scroll" class="adpbl" target="_blank">protocol</A>. Elution was done in 30 µL Milli-Q.</P><DL><DT>Date</DT><DD>13-06-2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">Nanodrop DNA Quantification</H4><P>The PCR amplicons were quantified according to the <A href="https://2018.igem.org/Team:TUDelft/Experiments#NanodropDNA-scroll" class="adpbl" target="_blank">Nanodrop DNA Quantification protocol</A>. The concentration and purity of the DNA (260/280 and 260/230 ratios) were measured. </P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Isolate</TH><TH class="tableheaderadpbl">Concentration (ng/uL)</TH><TH class="tableheaderadpbl">260/280 ratio</TH><TH class="tableheaderadpbl">260/230 ratio</TH></TR><TR class="tableunevenadpbl"><TD>Linker-Tn5</TD><TD>494.1</TD><TD>1.88</TD><TD>1.95</TD></TR><TR class="tableevenadpbl"><TD>Tn5</TD><TD>463.1</TD><TD>1.86</TD><TD>1.90</TD></TR><TR class="tableunevenadpbl"><TD>Tn5-Linker</TD><TD>398.3</TD><TD>1.88</TD><TD>1.87</TD></TR></TBODY></TABLE></CENTER></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#ThursdaySC2">Thursday, 14/06/2018</BUTTON><DIV id="ThursdaySC2" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>14/06/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">Liquid Starter Culture Preparation</H4><P>The six colonies that have grown on LB Cam after transformation on 11-06-2018 and probably harbour our desired plasmid pACYCDuet-1_Tn5 were used for inoculation of 50 mL liquid starter culture (Luria Broth, complemented with Cam) following the <A href="https://2018.igem.org/Team:TUDelft/Experiments#liquidstarterculture-scroll" class="adpbl" target="_blank">Liquid Starter Culture Protocol</A>. Additionally, as positive control one colony of pACYCDuet-1 that has also grown was inoculated in the same way.</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#FridaySC2">Friday, 15/06/2018</BUTTON><DIV id="FridaySC2" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>15/06/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">Restriction</H4><P>Restriction cloning was done for assembling Tn5, Tn5-Linker and Linker-Tn5 separately into pACYCDuet-1. The following table shows an overview of what fragment(s) was/were cut with what restriction enzyme(s).</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL)</TH><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL)</TH><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL)</TH></TR><TR class="tableunevenadpbl"><TD>10x CutSmart buffer</TD><TD>5</TD><TD>10x CutSmart buffer</TD><TD>5</TD><TD>10x CutSmart buffer</TD><TD>5</TD></TR><TR class="tableevenadpbl"><TD>Fragment (Tn5)</TD><TD>4.5 (1000 ng)</TD><TD>Fragment (Linker-Tn5)</TD><TD>4 (1000 ng)</TD><TD>Fragment (Tn5-Linker)</TD><TD>5 (1000 ng)</TD></TR><TR class="tableunevenadpbl"><TD>Enzyme (EcoRI)</TD><TD>1</TD><TD>Enzyme (FseI)</TD><TD>1</TD><TD>Enzyme (EcoRI)</TD><TD>1</TD></TR><TR class="tableevenadpbl"><TD>Enzyme (KpnI)</TD><TD>1</TD><TD>Enzyme (KpnI)</TD><TD>1</TD><TD>Enzyme (NotI)</TD><TD>1</TD></TR><TR class="tableunevenadpbl"><TD>MilliQ</TD><TD>38.5</TD><TD>MilliQ</TD><TD>39</TD><TD>MilliQ</TD><TD>38</TD></TR></TBODY></TABLE></CENTER><P>The samples were incubated for 1 hour at 37 °C, and followed by 20 minute at 65 °C heat inactivation of the restriction enzyme(s).</P><DL><DT>Date</DT><DD>15/06/2018</DD><DT>Operator</DT><DD>Timothy Páez</DD></DL><H4>Restriction </H4><P>Restriction cloning was done for assembling Tn5-lin and lin-Tn5 into pACYCduet-1. The following table shows an overview of what fragments were cut with what restriction enzymes.</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL)</TH></TR><TR class="tableunevenadpbl"><TD>10x CutSmart buffer    </TD><TD>2</TD></TR><TR class="tableunevenadpbl"><TD>Fragment (Tn5-lin)</TD><TD>9</TD></TR><TR class="tableunevenadpbl"><TD>Enzyme (ECORI HF)</TD><TD>1</TD></TR><TR class="tableunevenadpbl"><TD>Enzyme (NotI HF)</TD><TD>1</TD></TR><TR class="tableunevenadpbl"><TD>MilliQ</TD><TD> 7</TD></TR></TBODY></TABLE></CENTER><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL)</TH></TR><TR class="tableunevenadpbl"><TD>10x CutSmart buffer    </TD><TD>2</TD></TR><TR class="tableunevenadpbl"><TD>Vector (pACYC)</TD><TD>16</TD></TR><TR class="tableunevenadpbl"><TD>Enzyme (EcoRI HF)</TD><TD>1</TD></TR><TR class="tableunevenadpbl"><TD>Enzyme (NotI HF)</TD><TD>1</TD></TR><TR class="tableunevenadpbl"><TD>MilliQ</TD><TD> 0</TD></TR></TBODY></TABLE></CENTER><P>The samples were incubated for 1 hour at 37 °C. 

</P><DL><DT>Date</DT><DD>15/06/2018</DD><DT>Operator</DT><DD>Timothy Páez</DD></DL><H4>PCR clean-up</H4><P>
    20 μL of restriction product were subjected to PCR Clean-Up, according to the protocol. Elution was done in 20 µL Milli-Q. 
    Products were quantified with Nanodrop, yielding concentration of:
    - Tn5-lin: 30.4 ng/µL 
    - pACYC: 8.5 ng/µL </P><DL><DT>Date</DT><DD>15/06/2018</DD><DT>Operator</DT><DD>Timothy Páez</DD></DL><H4>Ligation</H4><P>In order to assemble cut fragments Tn5-lin into pACYCduet-1, ligation was performed. This table gives an overview of which fragments were cut with which enzymes.

</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">DNA</TH><TH class="tableheaderadpbl">Size (bp) </TH><TH class="tableheaderadpbl">Cut with enzymes</TH></TR><TR class="tableunevenadpbl"><TD>pACYCDuet</TD><TD>1.5 kbp</TD><TD>EcoRI and NotI</TD></TR><TR class="tableevenadpbl"><TD>Tn5-lin</TD><TD>1.5 kbp</TD><TD>EcoRI and NotI</TD></TR></TBODY></TABLE></CENTER><P>The ligation reaction mixture was prepared with the following composition</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL) </TH></TR><TR class="tableunevenadpbl"><TD>Ligase Buffer (10x) </TD><TD>2 µL</TD></TR><TR class="tableevenadpbl"><TD>T4 DNA Ligase</TD><TD>1 µL</TD></TR><TR class="tableunevenadpbl"><TD>pACYC (~100 ng) </TD><TD>13.3 µL</TD></TR><TR class="tableunevenadpbl"><TD>Tn5 (300 ng) </TD><TD>3.7 µL</TD></TR><TR class="tableunevenadpbl"><TD>MilliQ</TD><TD>0 µL</TD></TR></TBODY></TABLE></CENTER><P>The samples were incubated overnight at 4 ºC for ligation. </P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#SaturdaySC2">Saturday, 16/06/2018</BUTTON><DIV id="SaturdaySC2" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>16/06/2018</DD><DT>Operator</DT><DD>Timothy Páez</DD></DL><H4> Transformation of chemically competent cells </H4><P>The following transformations were carried out according to the protocol for transformation of chemically competent cells:
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Strain</TH><TH class="tableheaderadpbl">Plasmid &amp; size (bp) </TH><TH class="tableheaderadpbl">Cloning method</TH><TH class="tableheaderadpbl">Volume added (uL)</TH><TH class="tableheaderadpbl">Medium on which was plated</TH></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I>DH5α</TD><TD>pACYC Tn5 (3000 bp)</TD><TD>Restriction Ligation</TD><TD>10 µL</TD><TD>LB with Chloroamphenicol</TD></TR></TBODY></TABLE></CENTER><P>400 µL LB medium was added as recovery medium.
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#SundaySC2">Sunday, 17/06/2018</BUTTON><DIV id="SundaySC2" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>17/06/2018</DD><DT>Operator</DT><DD>Timothy Páez</DD></DL><H4> Transformation results from 16/06/2018 </H4><P>No colonies grew on LB plates with chloroamphenicol. Repeat protocol with higher DNA concentrations starting from restriction.
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV></DIV></UL></FORM><FORM><LABEL for="WeekSC3" class="subdropdown cadpbl">Week 3 18/06/2018 - 24/06/2018</LABEL><UL class="submenu"><DIV class="subcontent"><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#MondaySC3">Monday, 18/06/2018</BUTTON><DIV id="MondaySC3" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>18-06-2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">Plasmid Isolation</H4><P>
Overnight starter cultures of Tn5 (two times) and pACYCDuet-1 (two times) were subjected to plasmid isolation following the <A href="https://2018.igem.org/Team:TUDelft/Experiments#plasmidisolation-scroll" class="adpbl" target="_blank">Plasmid Isolation Protocol</A>. Final elution was done with 30 μL of pre-warmed MilliQ.
</P><DL><DT>Date</DT><DD>18-06-2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">Nanodrop DNA Quantification</H4><P>The following isolates were quantified according to the <A href="https://2018.igem.org/Team:TUDelft/Experiments#NanodropDNA-scroll" class="adpbl" target="_blank">Nanodrop DNA Quantification protocol</A>. The concentration and purity of the DNA (260/280 and 260/230 ratios) were measured. </P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Isolate</TH><TH class="tableheaderadpbl">Concentration (ng/uL)</TH><TH class="tableheaderadpbl">260/280 ratio</TH><TH class="tableheaderadpbl">260/230 ratio</TH></TR><TR class="tableunevenadpbl"><TD>Tn5 (1)</TD><TD>119.1</TD><TD>2.02</TD><TD>2.08</TD></TR><TR class="tableevenadpbl"><TD>Tn5 (2)</TD><TD>127.3</TD><TD>2.00</TD><TD>2.32</TD></TR><TR class="tableunevenadpbl"><TD>pACYCDuet-1 (1)</TD><TD>47.4</TD><TD>1.96</TD><TD>2.03</TD></TR><TR class="tableevenadpbl"><TD>pACYCDuet-1 (2)</TD><TD>238.9</TD><TD>2.12</TD><TD>2.44</TD></TR></TBODY></TABLE></CENTER><DL><DT>Date</DT><DD>18-06-2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">Restriction</H4><P>Restriction cloning was done for assembling Tn5, Tn5-Linker and Linker-Tn5 separately into pACYCDuet-1. The following table shows an overview of what fragment(s) was/were cut with what restriction enzyme(s).</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL)</TH><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL)</TH><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL)</TH><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL)</TH></TR><TR class="tableunevenadpbl"><TD>10x CutSmart buffer</TD><TD>5</TD><TD>10x CutSmart buffer</TD><TD>5</TD><TD>10x CutSmart buffer</TD><TD>5</TD><TD>10x CutSmart buffer</TD><TD>5</TD></TR><TR class="tableevenadpbl"><TD>Fragment (Tn5)</TD><TD>4.5 (2000 ng)</TD><TD>Fragment (Linker-Tn5)</TD><TD>4 (2000 ng)</TD><TD>Fragment (Tn5-Linker)</TD><TD>5 (2000 ng)</TD><TD>Fragment (pACYCDuet-1)</TD><TD>5 (2000 ng)</TD></TR><TR class="tableunevenadpbl"><TD>Enzyme (EcoRI)</TD><TD>1</TD><TD>Enzyme (FseI)</TD><TD>1</TD><TD>Enzyme (EcoRI)</TD><TD>1</TD><TD>Enzyme (EcoRI)</TD><TD>1</TD></TR><TR class="tableevenadpbl"><TD>Enzyme (KpnI)</TD><TD>1</TD><TD>Enzyme (KpnI)</TD><TD>1</TD><TD>Enzyme (NotI)</TD><TD>1</TD><TD>Enzyme (NotI)</TD><TD>1</TD></TR><TR class="tableunevenadpbl"><TD>MilliQ</TD><TD>38.5</TD><TD>MilliQ</TD><TD>39</TD><TD>MilliQ</TD><TD>38</TD><TD>MilliQ</TD><TD>38</TD></TR></TBODY></TABLE></CENTER><P>The samples were incubated for 1 hour at 37 °C, and followed by 20 minute at 65 °C heat inactivation of the restriction enzyme(s).</P><DL><DT>Date</DT><DD>18/06/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">PCR Clean-Up</H4><P>The restriction reactions Tn5, Tn5-Linker, Linker-Tn5 and pACYCDuet-1 were subjected to PCR Clean-Up, according to the <A href="https://2018.igem.org/Team:TUDelft/Experiments#PCRcleanup-scroll" class="adpbl" target="_blank">protocol</A>. Elution was done in 30 µL Milli-Q.</P><DL><DT>Date</DT><DD>18-06-2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">Transformation of chemically competent cells</H4><P>
The following transformations were carried out according to the protocol for transformation of chemically competent cells:</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Strain</TH><TH class="tableheaderadpbl">Plasmid and size (bp)</TH><TH class="tableheaderadpbl">Cloning method</TH><TH class="tableheaderadpbl">Volume added (μl)</TH><TH class="tableheaderadpbl">Medium on which was plated</TH></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I> DH5α</TD><TD>pACYCDuet-1_Tn5 (5210 bp)</TD><TD>Restriction/Ligation</TD><TD>5</TD><TD>LB Cam</TD></TR><TR class="tableevenadpbl"><TD><I>E.coli</I> DH5α</TD><TD>pACYCDuet-1_Linker-Tn5 (5477 bp)</TD><TD>Restriction/Ligation</TD><TD>5</TD><TD>LB Cam</TD></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I> DH5α</TD><TD>pACYCDuet-1 restricted (neg. control)</TD><TD>Restriction</TD><TD>5</TD><TD>LB Cam</TD></TR><TR class="tableevenadpbl"><TD><I>E.coli</I> DH5α</TD><TD>pACYCDuet-1 (pos. control)</TD><TD>-</TD><TD>0.5</TD><TD>LB Cam</TD></TR></TBODY></TABLE></CENTER><P>
200 μl LB medium was added as recovery medium.</P><DL><DT>Date</DT><DD>08-06-2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">Liquid Starter Culture Preparation</H4><P>The colonies that have grown on the plates of the transformation performed on 11-06-2018 were used for inoculation of 5 mL liquid starter culture (Luria Broth, complemented with chloramphenicol) following the <A href="https://2018.igem.org/Team:TUDelft/Experiments#liquidstarterculture-scroll" class="adpbl" target="_blank">Liquid Starter Culture Protocol</A>. This was done to be able to test the colonies for correct insert.</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#TuesdaySC3">Tuesday, 19/06/2018</BUTTON><DIV id="TuesdaySC3" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>19/06/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">High Fidelity PCR</H4><P>To amplify Linker-Tn5 (1508 bp), Tn5 (1452 bp) and Tn5-Linker (1505 bp) coding sequence, high fidelity PCR was done using pTXB1-Tn5 as template DNA and FP005, FP006, FP007 and FP008 as forward and reverse primers. Two times three 50µL PCR mixtures were prepared, according to the <A href="https://2018.igem.org/Team:TUDelft/Experiments#highfidelityPCR-scroll" target="_blank" class="adpbl">High Fidelity PCR protocol</A>. 
Primer combination(s) and expected fragment(s)
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Sample</TH><TH class="tableheaderadpbl">Fw primer</TH><TH class="tableheaderadpbl">Rv primer</TH><TH class="tableheaderadpbl">Expected fragment</TH><TH class="tableheaderadpbl">Expected size (bp) </TH></TR><TR class="tableunevenadpbl"><TD>1</TD><TD>005</TD><TD>006</TD><TD>Linker-Tn5</TD><TD>1508</TD></TR><TR class="tableevenadpbl"><TD>2</TD><TD>007</TD><TD>006</TD><TD>Tn5</TD><TD>1452</TD></TR><TR class="tableunevenadpbl"><TD>3</TD><TD>007</TD><TD>008</TD><TD>Tn5-Linker</TD><TD>1505</TD></TR></TBODY></TABLE></CENTER><P>The thermocycler programme was installed to have an extension time of 2 min 30 seconds (1 minute longer than PCR reactions before) and an annealing temperature of 60 °C. Again, in order to check the difference between two PCR machines (older one and new one), the tubes were divided over the two thermocyclers. After the programme was finished, samples were kept at 4 °C.
</P><DL><DT>Date</DT><DD>19/06/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">Plasmid Isolation</H4><P>Overnight starter cultures of the colonies from the transformation performed on 11-06-2018 were subjected to plasmid isolation following the <A href="https://2018.igem.org/Team:TUDelft/Experiments#plasmidisolation-scroll" class="adpbl" target="_blank">Plasmid Isolation Protocol</A>. Final elution was done with 20 μL of pre-warmed MilliQ.</P><DL><DT>Date</DT><DD>19/06/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">Nanodrop DNA Quantification</H4><P>The following isolates were quantified according to the <A href="https://2018.igem.org/Team:TUDelft/Experiments#NanodropDNA-scroll" class="adpbl" target="_blank">Nanodrop DNA Quantification protocol</A>. The concentration and purity of the DNA (260/280 and 260/230 ratios) were measured. </P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Isolate</TH><TH class="tableheaderadpbl">Concentration (ng/uL)</TH><TH class="tableheaderadpbl">260/280 ratio</TH><TH class="tableheaderadpbl">260/230 ratio</TH></TR><TR class="tableunevenadpbl"><TD>Colony 1: pACYCDuet-1_Tn5</TD><TD>19.7</TD><TD>1.75</TD><TD>0.86</TD></TR><TR class="tableevenadpbl"><TD>Colony 2: pACYCDuet-1_Tn5</TD><TD>38.6</TD><TD>1.66</TD><TD>0.79</TD></TR><TR class="tableunevenadpbl"><TD>Colony 3: pACYCDuet-1_Tn5</TD><TD>93.4</TD><TD>1.56</TD><TD>0.62</TD></TR><TR class="tableevenadpbl"><TD>Colony 4: pACYCDuet-1_Tn5</TD><TD>34.6</TD><TD>1.75</TD><TD>0.98</TD></TR><TR class="tableunevenadpbl"><TD>Colony 5: pACYCDuet-1_Tn5</TD><TD>57.4</TD><TD>1.62</TD><TD>0.74</TD></TR><TR class="tableevenadpbl"><TD>Colony 6: pACYCDuet-1_Tn5</TD><TD>39.8</TD><TD>1.73</TD><TD>0.99</TD></TR><TR class="tableunevenadpbl"><TD>Colony 7: pACYCDuet-1_Tn5</TD><TD>32.5</TD><TD>1.77</TD><TD>1.10</TD></TR><TR class="tableevenadpbl"><TD>Colony pACYCDuet-1</TD><TD>54.3</TD><TD>1.73</TD><TD>1.05</TD></TR></TBODY></TABLE></CENTER><DL><DT>Date</DT><DD>19/06/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">Colony PCR</H4><P>Colony PCR was done to verify propagation of pACYCDuet-1_Tn5 after transformation from 11-06-2018. This plasmid contains Tn5, which can be checked with primers 006 and 007. Additionally, the primers binding at the T7 promoter and termination were used to check the Tn5 insert.</P><CENTER><TABLE><TBODY><TR class="tableunevenadpbl"><TD>Template DNA</TD><TD>pACYCDuet-1_Tn5</TD><TD>pACYCDuet-1_Tn5</TD></TR><TR class="tableevenadpbl"><TD>Fw primer</TD><TD>007</TD><TD>029</TD></TR><TR class="tableunevenadpbl"><TD>Rv primer</TD><TD>006</TD><TD>030</TD></TR><TR class="tableevenadpbl"><TD>expected amplicon size (bp)</TD><TD>1452 bp</TD><TD>1576 bp</TD></TR></TBODY></TABLE></CENTER><P>Eight tubes were prepared for colony PCR, containing pACYCDuet-1-Tn5 as template DNA. Additionally, MgCl<SUB>2</SUB> was added to increase the efficiency of the PCR reaction. The thermocycler programme was installed to have an extension time of 2 minutes and an annealing temperature of 60 °C. After the programme was finished, samples were kept at 4°C.</P><DL><DT>Date</DT><DD>19/06/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">DNA Gel Electrophoresis</H4><P>The colony PCR’s of the transformation of 11-06-2018 were run on a 0.8% agarose gel in TAE buffer. 5 μL was pipetted into each lane. The gel was imaged using a gel documentation system with UV-light.</P><P>Colonies 1,2,3,4,6 and 7 show the presence of Tn5 and T7p-Tn5-T7t, indicating that these colonies were positive for Tn5 integration.</P><DL><DT>Date</DT><DD>19/06/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">High Fidelity PCR</H4><P>To amplify Linker-Tn5 (1508 bp), Tn5 (1452 bp) and Tn5-Linker (1505 bp) coding sequence, high fidelity PCR was done using pTXB1-Tn5 as template DNA and FP005, FP006, FP007 and FP008 as forward and reverse primers. Multiple 50µL PCR mixtures were prepared, according to the <A href="https://2018.igem.org/Team:TUDelft/Experiments#highfidelityPCR-scroll" target="_blank" class="adpbl">High Fidelity PCR protocol.</A> 
Primer combination(s) and expected fragment(s)
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Sample</TH><TH class="tableheaderadpbl">Fw primer</TH><TH class="tableheaderadpbl">Rv primer</TH><TH class="tableheaderadpbl">Expected fragment</TH><TH class="tableheaderadpbl">Expected size (bp) </TH></TR><TR class="tableunevenadpbl"><TD>1</TD><TD>005</TD><TD>006</TD><TD>Linker-Tn5</TD><TD>1508</TD></TR><TR class="tableevenadpbl"><TD>2</TD><TD>007</TD><TD>006</TD><TD>Tn5</TD><TD>1452</TD></TR><TR class="tableunevenadpbl"><TD>3</TD><TD>007</TD><TD>008</TD><TD>Tn5-Linker</TD><TD>1505</TD></TR></TBODY></TABLE></CENTER><P>The thermocycler programme was installed to have an extension time of 2 min 30 seconds (1 minute longer than PCR reactions before) and an annealing temperature of 60 °C. After the programme was finished, samples were kept at 4 °C.
</P><DL><DT>Date</DT><DD>19/06/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">Plasmid Isolation</H4><P>
Overnight starter cultures of the strain with pACYCDuet-1 were subjected to plasmid isolation following the <A href="https://2018.igem.org/Team:TUDelft/Experiments#plasmidisolation-scroll" class="adpbl" target="_blank">Plasmid Isolation Protocol</A>. Final elution was done with 30 μL of pre-warmed MilliQ.
</P><DL><DT>Date</DT><DD>19/06/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">Nanodrop DNA Quantification</H4><P>The following isolates were quantified according to the Nanodrop DNA Quantification protocol. The concentration and purity of the DNA (260/280 and 260/230 ratios) were measured. </P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Isolate</TH><TH class="tableheaderadpbl">Concentration (ng/uL)</TH><TH class="tableheaderadpbl">260/280 ratio</TH><TH class="tableheaderadpbl">260/230 ratio</TH></TR><TR class="tableunevenadpbl"><TD>pACYCDuet-1 (1)</TD><TD>15.5</TD><TD>1.85</TD><TD>1.36</TD></TR><TR class="tableevenadpbl"><TD>pACYCDuet-1 (2)</TD><TD>14.0</TD><TD>1.60</TD><TD>0.86</TD></TR></TBODY></TABLE></CENTER><DL><DT>Date</DT><DD>19-06-2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">Liquid Starter Culture Preparation</H4><P>Colonies 1,2,3,4,6 and 7 harbouring the pACYCDuet-1_Tn5 plasmid and strains containing the plasmids pTXB1-Tn5 and pACYCDuet-1 were used for inoculation of 50 mL liquid starter culture (Luria Broth, complemented with chloramphenicol) following the <A href="https://2018.igem.org/Team:TUDelft/Experiments#liquidstarterculture-scroll" class="adpbl" target="_blank">Liquid Starter Culture Protocol</A>.</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#WednesdaySC3">Wednesday, 20/06/2018</BUTTON><DIV id="WednesdaySC3" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>20/06/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">Plasmid Isolation</H4><P>Overnight starter cultures of pACYCDuet-1 and pTXB1-Tn5 were subjected to plasmid isolation following the <A href="https://2018.igem.org/Team:TUDelft/Experiments#plasmidisolation-scroll" class="adpbl" target="_blank">Plasmid Isolation Protocol</A>. Final elution was done with 30 μL of pre-warmed MilliQ.</P><DL><DT>Date</DT><DD>20/06/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">Nanodrop DNA Quantification</H4><P>The following isolates were quantified according to the <A href="https://2018.igem.org/Team:TUDelft/Experiments#NanodropDNA-scroll" class="adpbl" target="_blank">Nanodrop DNA Quantification protocol</A>. The concentration and purity of the DNA (260/280 and 260/230 ratios) were measured. </P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Isolate</TH><TH class="tableheaderadpbl">Concentration (ng/uL)</TH><TH class="tableheaderadpbl">260/280 ratio</TH><TH class="tableheaderadpbl">260/230 ratio</TH></TR><TR class="tableunevenadpbl"><TD>pACYCDuet-1 (1)</TD><TD>120.9</TD><TD>1.67</TD><TD>0.85</TD></TR><TR class="tableevenadpbl"><TD>pACYCDuet-1 (2)</TD><TD>108.1</TD><TD>1.70</TD><TD>1.03</TD></TR><TR class="tableunevenadpbl"><TD>pACYCDuet-1 (3)</TD><TD>56.8</TD><TD>1.86</TD><TD>1.52</TD></TR><TR class="tableevenadpbl"><TD>pACYCDuet-1 (4)</TD><TD>113.8</TD><TD>1.68</TD><TD>0.91</TD></TR><TR class="tableunevenadpbl"><TD>pTXB1-Tn5 (1) </TD><TD>90.6</TD><TD>1.81</TD><TD>1.45</TD></TR><TR class="tableevenadpbl"><TD>pTXB1-Tn5 (2)</TD><TD>96.6</TD><TD>1.82</TD><TD>1.65</TD></TR><TR class="tableunevenadpbl"><TD>pTXB1-Tn5 (3)</TD><TD>106.4</TD><TD>1.79</TD><TD>1.36</TD></TR><TR class="tableevenadpbl"><TD>pTXB1-Tn5 (4)</TD><TD>121.7</TD><TD>1.73</TD><TD>1.18</TD></TR></TBODY></TABLE></CENTER><DL><DT>Date</DT><DD>20/06/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">PCR Clean-Up</H4><P>Two sets of 50 μL of PCR product of Tn5, Linker-Tn5 and Tn5-linker performed on and 19-06-2018 was subjected to PCR Clean-Up, according to the <A href="https://2018.igem.org/Team:TUDelft/Experiments#PCRcleanup-scroll" class="adpbl" target="_blank">protocol</A>. Elution was done in 30 µL prewarmed Milli-Q.</P><DL><DT>Date</DT><DD>20/06/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">Nanodrop DNA Quantification</H4><P>The following isolates were quantified according to the <A href="https://2018.igem.org/Team:TUDelft/Experiments#NanodropDNA-scroll" class="adpbl" target="_blank">Nanodrop DNA Quantification protocol</A>. The concentration and purity of the DNA (260/280 and 260/230 ratios) were measured. </P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Isolate</TH><TH class="tableheaderadpbl">Concentration (ng/uL)</TH><TH class="tableheaderadpbl">260/280 ratio</TH><TH class="tableheaderadpbl">260/230 ratio</TH></TR><TR class="tableunevenadpbl"><TD>Tn5-Linker (1)</TD><TD>190.3</TD><TD>1.90</TD><TD>1.38</TD></TR><TR class="tableevenadpbl"><TD>Tn5-linker (2)</TD><TD>206.0</TD><TD>1.90</TD><TD>1.98</TD></TR><TR class="tableunevenadpbl"><TD>Tn5 (1)</TD><TD>258.4</TD><TD>1.88</TD><TD>2.04</TD></TR><TR class="tableevenadpbl"><TD>Tn5 (2)</TD><TD>260.7</TD><TD>1.88</TD><TD>2.06</TD></TR><TR class="tableunevenadpbl"><TD>Linker-Tn5 (1) </TD><TD>199.7</TD><TD>1.88</TD><TD>1.86</TD></TR><TR class="tableevenadpbl"><TD>Linker-Tn5 (2)</TD><TD>234.6</TD><TD>1.89</TD><TD>1.99</TD></TR></TBODY></TABLE></CENTER><DL><DT>Date</DT><DD>20/06/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">Restriction</H4><P>Restriction cloning was done for assembling Tn5, Tn5-Linker and Linker-Tn5 separately into pACYCDuet-1. The following table shows an overview of what fragment(s) was/were cut with what restriction enzyme(s).</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL)</TH><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL)</TH><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL)</TH></TR><TR class="tableunevenadpbl"><TD>10x CutSmart buffer</TD><TD>2</TD><TD>10x CutSmart buffer</TD><TD>2</TD><TD>10x CutSmart buffer</TD><TD>2</TD></TR><TR class="tableevenadpbl"><TD>Fragment (Tn5-Linker)</TD><TD>10 (2000 ng)</TD><TD>Fragment (Tn5)</TD><TD>8 (2000 ng)</TD><TD>Fragment (Linker-Tn5)</TD><TD>9 (2000 ng)</TD></TR><TR class="tableunevenadpbl"><TD>Enzyme (FseI)</TD><TD>1</TD><TD>Enzyme (EcoRI)</TD><TD>1</TD><TD>Enzyme (EcoRI)</TD><TD>1</TD></TR><TR class="tableevenadpbl"><TD>Enzyme (KpnI)</TD><TD>1</TD><TD>Enzyme (KpnI)</TD><TD>1</TD><TD>Enzyme (NotI)</TD><TD>1</TD></TR><TR class="tableunevenadpbl"><TD>MilliQ</TD><TD>6</TD><TD>MilliQ</TD><TD>8</TD><TD>MilliQ</TD><TD>7</TD></TR></TBODY></TABLE></CENTER><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL)</TH><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL)</TH><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL)</TH></TR><TR class="tableunevenadpbl"><TD>10x CutSmart buffer</TD><TD>2</TD><TD>10x CutSmart buffer</TD><TD>2</TD><TD>10x CutSmart buffer</TD><TD>2</TD></TR><TR class="tableevenadpbl"><TD>Fragment (pACYCDuet-1)</TD><TD>16 (2000 ng)</TD><TD>Fragment (pACYCDuet-1)</TD><TD>16 (2000 ng)</TD><TD>Fragment (pACYCDuet-1)</TD><TD>16 (2000 ng)</TD></TR><TR class="tableunevenadpbl"><TD>Enzyme (FseI)</TD><TD>1</TD><TD>Enzyme (EcoRI)</TD><TD>1</TD><TD>Enzyme (EcoRI)</TD><TD>1</TD></TR><TR class="tableevenadpbl"><TD>Enzyme (KpnI)</TD><TD>1</TD><TD>Enzyme (KpnI)</TD><TD>1</TD><TD>Enzyme (NotI)</TD><TD>1</TD></TR><TR class="tableunevenadpbl"><TD>MilliQ</TD><TD>0</TD><TD>MilliQ</TD><TD>0</TD><TD>MilliQ</TD><TD>0</TD></TR></TBODY></TABLE></CENTER><P>The samples were incubated for 1 hour at 37 °C, and followed by 20 minute heat inactivation (65 °C) of the restriction enzyme(s).</P><DL><DT>Date</DT><DD>20/06/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">PCR Clean-Up</H4><P>20 μL of each restriction reaction mixture was subjected to PCR Clean-Up, according to the <A href="https://2018.igem.org/Team:TUDelft/Experiments#PCRcleanup-scroll" class="adpbl" target="_blank">protocol</A>. Elution was done in 20 µL Milli-Q.</P><DL><DT>Date</DT><DD>20/06/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">Nanodrop DNA Quantification</H4><P>The following isolates were quantified according to the <A href="https://2018.igem.org/Team:TUDelft/Experiments#NanodropDNA-scroll" class="adpbl" target="_blank">Nanodrop DNA Quantification protocol</A>. The concentration and purity of the DNA (260/280 and 260/230 ratios) were measured. </P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Isolate</TH><TH class="tableheaderadpbl">Concentration (ng/uL)</TH><TH class="tableheaderadpbl">260/280 ratio</TH><TH class="tableheaderadpbl">260/230 ratio</TH></TR><TR class="tableunevenadpbl"><TD>pACYCDuet-1 cut with FseI and KpnI</TD><TD>18.7</TD><TD>1.71</TD><TD>1.02</TD></TR><TR class="tableevenadpbl"><TD>pACYCDuet-1 cut with EcoRI and KpnI</TD><TD>41.2</TD><TD>1.80</TD><TD>1.44</TD></TR><TR class="tableunevenadpbl"><TD>pACYCDuet-1 cut with EcoRI and NotI</TD><TD>22.6</TD><TD>1.77</TD><TD>0.56</TD></TR><TR class="tableevenadpbl"><TD>Tn5-Linker cut with FseI and KpnI</TD><TD>56.7</TD><TD>1.85</TD><TD>1.63</TD></TR><TR class="tableunevenadpbl"><TD>Tn5 cut with EcoRI and KpnI</TD><TD>59.2</TD><TD>1.84</TD><TD>1.20</TD></TR><TR class="tableevenadpbl"><TD>Linker-Tn5 cut with EcoRI and NotI</TD><TD>70.0</TD><TD>1.87</TD><TD>1.73</TD></TR></TBODY></TABLE></CENTER><DL><DT>Date</DT><DD>20/06/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">Ligation</H4><P>In order to assemble cut fragments pACYCDuet-1 and Tn-Linker, Tn5 and Linker-Tn5 respectively, ligation was performed. This table gives an overview of which fragments were cut with which enzymes.</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">DNA</TH><TH class="tableheaderadpbl">Size (bp) </TH><TH class="tableheaderadpbl">Cut with enzymes</TH></TR><TR class="tableunevenadpbl"><TD>Vector pACYCDuet-1</TD><TD>3970</TD><TD>EcoRI and NotI</TD></TR><TR class="tableevenadpbl"><TD>Fragment Tn5-Linker</TD><TD>1490</TD><TD>EcoRI and NotI</TD></TR></TBODY></TABLE></CENTER><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">DNA</TH><TH class="tableheaderadpbl">Size (bp) </TH><TH class="tableheaderadpbl">Cut with enzymes</TH></TR><TR class="tableunevenadpbl"><TD>Vector pACYCDuet-1</TD><TD>3768</TD><TD>KpnI and EcoRI</TD></TR><TR class="tableevenadpbl"><TD>Fragment Tn5</TD><TD>1442</TD><TD>KpnI and EcoRI</TD></TR></TBODY></TABLE></CENTER><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">DNA</TH><TH class="tableheaderadpbl">Size (bp) </TH><TH class="tableheaderadpbl">Cut with enzymes</TH></TR><TR class="tableunevenadpbl"><TD>Vector pACYCDuet-1</TD><TD>3984</TD><TD>FseI and KpnI</TD></TR><TR class="tableevenadpbl"><TD>Fragment Linker-Tn5</TD><TD>1493</TD><TD>FseI and KpnI</TD></TR></TBODY></TABLE></CENTER><P>A 20 μL ligation reaction mixture was prepared with the following composition:</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (μL)</TH><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (μL)</TH><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (μL)</TH></TR><TR class="tableunevenadpbl"><TD>Ligase Buffer (10X)</TD><TD>2</TD><TD>Ligase Buffer (10X)</TD><TD>2</TD><TD>Ligase Buffer (10X)</TD><TD>2</TD></TR><TR class="tableevenadpbl"><TD>T4 DNA Ligase </TD><TD>1</TD><TD>T4 DNA Ligase </TD><TD>1</TD><TD>T4 DNA Ligase </TD><TD>1</TD></TR><TR class="tableunevenadpbl"><TD>DNA vector (~200 ng)</TD><TD>10</TD><TD>DNA vector (~200 ng)</TD><TD>6</TD><TD>DNA vector (~200 ng)</TD><TD>11</TD></TR><TR class="tableevenadpbl"><TD>DNA Tn5-Lin (~80 ng)</TD><TD>1.5</TD><TD>DNA Tn5 (~80 ng)</TD><TD>2</TD><TD>DNA Lin-Tn5 (~80 ng)</TD><TD>2</TD></TR><TR class="tableunevenadpbl"><TD>MilliQ</TD><TD>6.5</TD><TD>MilliQ</TD><TD>9</TD><TD>MilliQ</TD><TD>4</TD></TR></TBODY></TABLE></CENTER><P>The samples were incubated for 4 hours at 20°C for ligation.</P><DL><DT>Date</DT><DD>20/06/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">Transformation of chemically competent cells</H4><P>The following transformations were carried out according to the protocol for transformation of chemically competent cells:</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Strain</TH><TH class="tableheaderadpbl">Plasmid and size (bp)</TH><TH class="tableheaderadpbl">Cloning method</TH><TH class="tableheaderadpbl">Volume added (μl)</TH><TH class="tableheaderadpbl">Medium on which was plated</TH></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I> DH5α</TD><TD>pACYCDuet-1_Tn5-Lin (5460 bp)</TD><TD>Restriction/Ligation</TD><TD>4</TD><TD>LB-Cam</TD></TR><TR class="tableevenadpbl"><TD><I>E.coli</I> DH5α</TD><TD>pACYCDuet-1_Tn5 (5210 bp)</TD><TD>Restriction/Ligation</TD><TD>4</TD><TD>LB Cam</TD></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I> DH5α</TD><TD>pACYCDuet-1_Linker-Tn5 (5477 bp)</TD><TD>Restriction/Ligation</TD><TD>4</TD><TD>LB Cam</TD></TR><TR class="tableevenadpbl"><TD><I>E.coli</I> DH5α</TD><TD>pACYCDuet-1 restricted (neg. control)</TD><TD>Restriction</TD><TD>1</TD><TD>LB Cam</TD></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I> DH5α</TD><TD>pACYCDuet-1 (pos. control)</TD><TD>-</TD><TD>1</TD><TD>LB Cam</TD></TR></TBODY></TABLE></CENTER><P>
200 μl LB medium was added as recovery medium.</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#ThursdaySC3">Thursday, 21/06/2018</BUTTON><DIV id="ThursdaySC3" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>21-06-2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">Colony PCR</H4><P>Colony PCR was done to verify propagation of pACYC1Duet-1_Lin-Tn5 after transformation from 20-06-2018. This plasmid contains the Lin-Tn5 insert, which can be checked with primers 029 and 030. </P><CENTER><TABLE><TBODY><TR class="tableunevenadpbl"><TD>Template DNA</TD><TD>pACYCDuet-1_Lin-Tn5</TD></TR><TR class="tableevenadpbl"><TD>Fw primer</TD><TD>029</TD></TR><TR class="tableunevenadpbl"><TD>Rv primer</TD><TD>030</TD></TR><TR class="tableevenadpbl"><TD>expected amplicon size (bp)</TD><TD>1722 bp</TD></TR></TBODY></TABLE></CENTER><P>23 tubes were prepared for colony PCR, containing pACYCDuet-1_Lin-Tn5 as template DNA. The thermocycler programme was installed to have an extension time of 2 minutes and an annealing temperature of 60 °C. After the programme was finished, samples were kept at 4°C.
</P><DL><DT>Date</DT><DD>21-06-2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">Liquid Starter Culture Preparation</H4><P>Strain E.coli DH5a harbouring the pACYCDuet-1 plasmid was used for inoculation of 50 mL liquid starter culture (Luria Broth, complemented with chloramphenicol) following the <A href="https://2018.igem.org/Team:TUDelft/Experiments#liquidstarterculture-scroll" class="adpbl" target="_blank">Liquid Starter Culture protocol </A></P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#FridaySC3">Friday, 22/06/2018</BUTTON><DIV id="FridaySC3" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>22-06-2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">Colony PCR</H4><P>Colony PCR was done to verify propagation of pACYC1Duet-1_Lin-Tn5 after transformation from 20-06-2018. This plasmid contains the Lin-Tn5 insert, which can be checked with primers 029 and 030. </P><CENTER><TABLE><TBODY><TR class="tableunevenadpbl"><TD>Template DNA</TD><TD>pACYCDuet-1_Lin-Tn5</TD></TR><TR class="tableevenadpbl"><TD>Fw primer</TD><TD>029</TD></TR><TR class="tableunevenadpbl"><TD>Rv primer</TD><TD>030</TD></TR><TR class="tableevenadpbl"><TD>expected amplicon size (bp)</TD><TD>1722 bp</TD></TR></TBODY></TABLE></CENTER><P>23 tubes were prepared for colony PCR, containing pACYCDuet-1_Lin-Tn5 as template DNA. The thermocycler programme was installed to have an extension time of 2 minutes and an annealing temperature of 60 °C. After the programme was finished, samples were kept at 4°C.
</P><DL><DT>Date</DT><DD>21-06-2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">DNA Gel Electrophoresis</H4><P>The colony PCR performed on 21-06-2018 checking for pACYCDuet-1_Linker-Tn5 was run on a 0.8% agarose gel in TAE buffer. 4 μL was pipetted into each lane. The gel was imaged using a gel documentation system with UV-light.</P><P>Unfortunately, no band was observed that corresponds to the Linker-Tn5 fragment in pACYCDuet-1. The bands are at the same height as the positive control pACYCDuet-1, meaning that no insertion has taken place</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV></DIV></UL></FORM><FORM><LABEL for="WeekSC4" class="subdropdown cadpbl">Week 4 25/06/2018 - 01/07/2018</LABEL><UL class="submenu"><DIV class="subcontent"><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#WednesdaySC4">Wednesday, 27/06/2018</BUTTON><DIV id="WednesdaySC4" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>27/06/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">Restriction</H4><P>Restriction cloning was done for assembling Tn5, Tn5-Linker and Linker-Tn5 separately into pACYCDuet-1. The following table shows an overview of what fragment(s) were cut with what restriction enzymes.</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL)</TH><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL)</TH><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL)</TH></TR><TR class="tableunevenadpbl"><TD>10x CutSmart buffer</TD><TD>2</TD><TD>10x CutSmart buffer</TD><TD>2</TD><TD>10x CutSmart buffer</TD><TD>2</TD></TR><TR class="tableevenadpbl"><TD>Fragment (Tn5-Linker)</TD><TD>10 (2000 ng)</TD><TD>Fragment (Tn5)</TD><TD>8 (2000 ng)</TD><TD>Fragment (Linker-Tn5)</TD><TD>9 (2000 ng)</TD></TR><TR class="tableunevenadpbl"><TD>Enzyme (FseI)</TD><TD>1</TD><TD>Enzyme (EcoRI)</TD><TD>1</TD><TD>Enzyme (EcoRI)</TD><TD>1</TD></TR><TR class="tableevenadpbl"><TD>Enzyme (KpnI)</TD><TD>1</TD><TD>Enzyme (KpnI)</TD><TD>1</TD><TD>Enzyme (NotI)</TD><TD>1</TD></TR><TR class="tableunevenadpbl"><TD>MilliQ</TD><TD>6</TD><TD>MilliQ</TD><TD>8</TD><TD>MilliQ</TD><TD>7</TD></TR></TBODY></TABLE></CENTER><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL)</TH><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL)</TH><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL)</TH></TR><TR class="tableunevenadpbl"><TD>10x CutSmart buffer</TD><TD>2</TD><TD>10x CutSmart buffer</TD><TD>2</TD><TD>10x CutSmart buffer</TD><TD>2</TD></TR><TR class="tableevenadpbl"><TD>Fragment (pACYCDuet-1)</TD><TD>16 (2000 ng)</TD><TD>Fragment (pACYCDuet-1)</TD><TD>16 (2000 ng)</TD><TD>Fragment (pACYCDuet-1)</TD><TD>16 (2000 ng)</TD></TR><TR class="tableunevenadpbl"><TD>Enzyme (FseI)</TD><TD>1</TD><TD>Enzyme (EcoRI)</TD><TD>1</TD><TD>Enzyme (EcoRI)</TD><TD>1</TD></TR><TR class="tableevenadpbl"><TD>Enzyme (KpnI)</TD><TD>1</TD><TD>Enzyme (KpnI)</TD><TD>1</TD><TD>Enzyme (NotI)</TD><TD>1</TD></TR><TR class="tableunevenadpbl"><TD>MilliQ</TD><TD>0</TD><TD>MilliQ</TD><TD>0</TD><TD>MilliQ</TD><TD>0</TD></TR></TBODY></TABLE></CENTER><P>The samples were incubated for 1 hour at 37 °C, and followed by 20 minute heat inactivation (65 °C) of the restriction enzyme(s).</P><DL><DT>Date</DT><DD>20-06-2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">PCR Clean-Up</H4><P>20 μL of each restriction reaction mixture was subjected to PCR Clean-Up, according to the <A href="https://2018.igem.org/Team:TUDelft/Experiments#PCRcleanup-scroll" class="adpbl" target="_blank">protocol</A>. Elution was done in 30 µL Milli-Q.</P><DL><DT>Date</DT><DD>27/06/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">Nanodrop DNA Quantification</H4><P>The following isolates were quantified according to the <A href="https://2018.igem.org/Team:TUDelft/Experiments#NanodropDNA-scroll" class="adpbl" target="_blank">Nanodrop DNA Quantification protocol</A>. The concentration and purity of the DNA (260/280 and 260/230 ratios) were measured. </P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Isolate</TH><TH class="tableheaderadpbl">Concentration (ng/uL)</TH><TH class="tableheaderadpbl">260/280 ratio</TH><TH class="tableheaderadpbl">260/230 ratio</TH></TR><TR class="tableunevenadpbl"><TD>pACYCDuet-1 cut with FseI and KpnI</TD><TD>24.9</TD><TD>1.89</TD><TD>0.66</TD></TR><TR class="tableevenadpbl"><TD>pACYCDuet-1 cut with EcoRI and NotI</TD><TD>20.8</TD><TD>1.81</TD><TD>1.08</TD></TR><TR class="tableunevenadpbl"><TD>Tn5-Linker cut with NotI and EcoRI</TD><TD>56.0</TD><TD>1.85</TD><TD>1.61</TD></TR><TR class="tableevenadpbl"><TD>Linker-Tn5 cut with FseI and KpnI</TD><TD>53.4</TD><TD>1.88</TD><TD>1.57</TD></TR></TBODY></TABLE></CENTER><DL><DT>Date</DT><DD>27/06/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">Ligation</H4><P>In order to assemble cut fragments pACYCDuet-1 and Tn-Linker and Linker-Tn5 respectively, ligation was performed. This table gives an overview of which fragments were cut with which enzymes.</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">DNA</TH><TH class="tableheaderadpbl">Size (bp) </TH><TH class="tableheaderadpbl">Cut with enzymes</TH></TR><TR class="tableunevenadpbl"><TD>Vector pACYCDuet-1</TD><TD>3970</TD><TD>EcoRI and NotI</TD></TR><TR class="tableevenadpbl"><TD>Fragment Tn5-Linker</TD><TD>1490</TD><TD>EcoRI and NotI</TD></TR></TBODY></TABLE></CENTER><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">DNA</TH><TH class="tableheaderadpbl">Size (bp) </TH><TH class="tableheaderadpbl">Cut with enzymes</TH></TR><TR class="tableunevenadpbl"><TD>Vector pACYCDuet-1</TD><TD>3984</TD><TD>FseI and KpnI</TD></TR><TR class="tableevenadpbl"><TD>Fragment Linker-Tn5</TD><TD>1493</TD><TD>FseI and KpnI</TD></TR></TBODY></TABLE></CENTER><P>A 20 μL ligation reaction mixture was prepared with the following composition:</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (μL)</TH><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (μL)</TH></TR><TR class="tableunevenadpbl"><TD>Ligase Buffer (10X)</TD><TD>2</TD><TD>Ligase Buffer (10X)</TD><TD>2</TD></TR><TR class="tableevenadpbl"><TD>T4 DNA Ligase </TD><TD>1</TD><TD>T4 DNA Ligase </TD><TD>1</TD></TR><TR class="tableunevenadpbl"><TD>DNA vector (~200 ng)</TD><TD>14.5</TD><TD>DNA vector (~200 ng)</TD><TD>14.5</TD></TR><TR class="tableevenadpbl"><TD>DNA Tn5-Lin (~80 ng)</TD><TD>2.5</TD><TD>DNA Lin-Tn5 (~80 ng)</TD><TD>2.5</TD></TR><TR class="tableunevenadpbl"><TD>MilliQ</TD><TD>0</TD><TD>MilliQ</TD><TD>0</TD></TR></TBODY></TABLE></CENTER><P>The samples were incubated overnight at 20°C for ligation.</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#ThursdaySC4">Thursday, 28/06/2018</BUTTON><DIV id="ThursdaySC4" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>28/06/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">Transformation of chemically competent cells</H4><P>The following transformations were carried out according to the protocol for transformation of chemically competent cells:</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Strain</TH><TH class="tableheaderadpbl">Plasmid and size (bp)</TH><TH class="tableheaderadpbl">Cloning method</TH><TH class="tableheaderadpbl">Volume added (μl)</TH><TH class="tableheaderadpbl">Medium on which was plated</TH></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I> DH5α</TD><TD>pACYCDuet-1_Tn5-Lin (5460 bp)</TD><TD>Restriction/Ligation</TD><TD>1.5 (~20 ng)</TD><TD>LB-Cam</TD></TR><TR class="tableevenadpbl"><TD><I>E.coli</I> DH5α</TD><TD>pACYCDuet-1_Tn5-Lin (5460 bp)</TD><TD>Restriction/Ligation</TD><TD>4 (~50 ng)</TD><TD>LB Cam</TD></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I> DH5α</TD><TD>pACYCDuet-1_Linker-Tn5 (5477 bp)</TD><TD>Restriction/Ligation</TD><TD>1.5 (~20 ng)</TD><TD>LB Cam</TD></TR><TR class="tableevenadpbl"><TD><I>E.coli</I> DH5α</TD><TD>pACYCDuet-1_Linker-Tn5 (5477 bp)</TD><TD>Restriction</TD><TD>4 (~50 ng)</TD><TD>LB Cam</TD></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I> DH5α</TD><TD>pACYCDuet-1 (pos. control)</TD><TD>-</TD><TD>0.1 (~10 ng)</TD><TD>LB Cam</TD></TR><TR class="tableevenadpbl"><TD><I>E.coli</I> DH5α</TD><TD>pACYCDuet-1 restricted (neg. control)</TD><TD>Restriction</TD><TD>2.5</TD><TD>LB Cam</TD></TR></TBODY></TABLE></CENTER><P>
200 μl LB medium was added as recovery medium.</P><DL><DT>Date</DT><DD>28/06/2018</DD><DT>Operator</DT><DD>Timothy Páez</DD></DL><H4>Restriction </H4><P>Restriction cloning was done for assembling Tn5-lin and lin-Tn5 into pACYCduet-1. The following table shows an overview of what fragments were cut with what restriction enzymes.
    First Reaction:
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL)</TH></TR><TR class="tableunevenadpbl"><TD>10x CutSmart buffer    </TD><TD>2</TD></TR><TR class="tableunevenadpbl"><TD>Fragment (Tn5-lin)</TD><TD>9</TD></TR><TR class="tableunevenadpbl"><TD>Enzyme (ECORI HF)</TD><TD>1</TD></TR><TR class="tableunevenadpbl"><TD>Enzyme (NotI HF)</TD><TD>1</TD></TR><TR class="tableunevenadpbl"><TD>MilliQ</TD><TD> 7</TD></TR></TBODY></TABLE></CENTER><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL)</TH></TR><TR class="tableunevenadpbl"><TD>10x CutSmart buffer    </TD><TD>2</TD></TR><TR class="tableunevenadpbl"><TD>Vector (pACYC)</TD><TD>16</TD></TR><TR class="tableunevenadpbl"><TD>Enzyme (EcoRI HF)</TD><TD>1</TD></TR><TR class="tableunevenadpbl"><TD>Enzyme (NotI HF)</TD><TD>1</TD></TR><TR class="tableunevenadpbl"><TD>MilliQ</TD><TD> 0</TD></TR></TBODY></TABLE></CENTER><P>Second reaction:</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL)</TH></TR><TR class="tableunevenadpbl"><TD>10x CutSmart buffer    </TD><TD>2</TD></TR><TR class="tableunevenadpbl"><TD>Fragment (lin-Tn5)</TD><TD>10</TD></TR><TR class="tableunevenadpbl"><TD>Enzyme (FseI HF)</TD><TD>1</TD></TR><TR class="tableunevenadpbl"><TD>Enzyme (KpnI HF)</TD><TD>1</TD></TR><TR class="tableunevenadpbl"><TD>MilliQ</TD><TD> 6</TD></TR></TBODY></TABLE></CENTER><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL)</TH></TR><TR class="tableunevenadpbl"><TD>10x CutSmart buffer    </TD><TD>2</TD></TR><TR class="tableunevenadpbl"><TD>Vector (pACYC)</TD><TD>16</TD></TR><TR class="tableunevenadpbl"><TD>Enzyme (FseI HF)</TD><TD>1</TD></TR><TR class="tableunevenadpbl"><TD>Enzyme (KpnI HF)</TD><TD>1</TD></TR><TR class="tableunevenadpbl"><TD>MilliQ</TD><TD> 0</TD></TR></TBODY></TABLE></CENTER><P>The samples were incubated for 1 hour at 37 °C. 

</P><DL><DT>Date</DT><DD>28/06/2018</DD><DT>Operator</DT><DD>Timothy Páez</DD></DL><H4>PCR clean-up</H4><P>
    20 μL of restriction product were subjected to PCR Clean-Up, according to the protocol. Elution was done in 20 µL Milli-Q. 
    Products were quantified with Nanodrop, yielding concentration of:
    - Tn5-lin: 32.3 ng/µL 
    - pACYC: 12.2 ng/µL 
    - lin-Tn5: 33.6 ng/µL 
    - pACYC: 11.2 ng/µL </P><DL><DT>Date</DT><DD>28/06/2018</DD><DT>Operator</DT><DD>Timothy Páez</DD></DL><H4>Ligation</H4><P>In order to assemble cut fragments Tn5-lin into pACYCduet-1 and lin-Tn5 into pACYCduet-1, ligation was performed. This table gives an overview of which fragments were cut with which enzymes.

</P><P>First reaction:</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">DNA</TH><TH class="tableheaderadpbl">Size (bp) </TH><TH class="tableheaderadpbl">Cut with enzymes</TH></TR><TR class="tableunevenadpbl"><TD>pACYCDuet</TD><TD>1.5 kbp</TD><TD>EcoRI and NotI</TD></TR><TR class="tableevenadpbl"><TD>Tn5-lin</TD><TD>1.5 kbp</TD><TD>EcoRI and NotI</TD></TR></TBODY></TABLE></CENTER><P>The ligation reaction mixture was prepared with the following composition</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL) </TH></TR><TR class="tableunevenadpbl"><TD>Ligase Buffer (10x) </TD><TD>2 µL</TD></TR><TR class="tableevenadpbl"><TD>T4 DNA Ligase</TD><TD>1 µL</TD></TR><TR class="tableunevenadpbl"><TD>pACYC (~100 ng) </TD><TD>7.35 µL</TD></TR><TR class="tableunevenadpbl"><TD>Tn5 (300 ng) </TD><TD>3.39 µL</TD></TR><TR class="tableunevenadpbl"><TD>MilliQ</TD><TD>6.26 µL</TD></TR></TBODY></TABLE></CENTER><P>Second reaction:</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">DNA</TH><TH class="tableheaderadpbl">Size (bp) </TH><TH class="tableheaderadpbl">Cut with enzymes</TH></TR><TR class="tableunevenadpbl"><TD>pACYCDuet</TD><TD>1.5 kbp</TD><TD>FseI and KpnI</TD></TR><TR class="tableevenadpbl"><TD>lin-Tn5</TD><TD>1.5 kbp</TD><TD>FseI and KpnI</TD></TR></TBODY></TABLE></CENTER><P>The ligation reaction mixture was prepared with the following composition</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL) </TH></TR><TR class="tableunevenadpbl"><TD>Ligase Buffer (10x) </TD><TD>2 µL</TD></TR><TR class="tableevenadpbl"><TD>T4 DNA Ligase</TD><TD>1 µL</TD></TR><TR class="tableunevenadpbl"><TD>pACYC (~100 ng) </TD><TD>8.77 µL</TD></TR><TR class="tableunevenadpbl"><TD>lin-Tn5 (300 ng) </TD><TD>3.4 µL</TD></TR><TR class="tableunevenadpbl"><TD>MilliQ</TD><TD>4.8 µL</TD></TR></TBODY></TABLE></CENTER><P>The samples were incubated overnight at 4 ºC for ligation. </P><DL><DT>Date</DT><DD>28/06/2018</DD><DT>Operator</DT><DD>Timothy Páez</DD></DL><H4>Liquid Starter Culture Preparation</H4><P><I>Escherichia coli</I> DH5 alpha harbouring pACYC1-duet plasmid was used for inoculation of 10 mL liquid starter culture (Luria Broth, complemented with chloroamphenicol) following the Liquid Starter Culture Protocol.
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#FridaySC4">Friday, 29/06/2018</BUTTON><DIV id="FridaySC4" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>29/06/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">Colony PCR</H4><P>Colony PCR was done to verify propagation of pACYC1Duet-1_Lin-Tn5 after transformation from 28-06-2018. This plasmid contains the Lin-Tn5 or Tn5-Lin insert, which can be checked with primers 029 and 030. There are two ways for colony PCR. The first one is picking a colony and directly resuspending the cells in PCR mixture. The second one is resuspending the colony in MilliQ, heating the suspension, centrifugation the broken cell pieces and taking the supernantant for PCR. In orde to test the efficiency of both methods, I tested both simultaneously. 22 colony PCR were performed by picking the colony and direct resuspension in the PCR mixture, while 20 colonies were picked and heated prior to pipetting the DNA into the PCR mixture.</P><CENTER><TABLE><TBODY><TR class="tableunevenadpbl"><TD>Template DNA</TD><TD>pACYCDuet-1_Lin-Tn5</TD></TR><TR class="tableevenadpbl"><TD>Fw primer</TD><TD>029</TD></TR><TR class="tableunevenadpbl"><TD>Rv primer</TD><TD>030</TD></TR><TR class="tableevenadpbl"><TD>expected amplicon size (bp)</TD><TD>1722 bp</TD></TR></TBODY></TABLE></CENTER><CENTER><TABLE><TBODY><TR class="tableunevenadpbl"><TD>Template DNA</TD><TD>pACYCDuet-1_Tn5-Lin</TD></TR><TR class="tableevenadpbl"><TD>Fw primer</TD><TD>029</TD></TR><TR class="tableunevenadpbl"><TD>Rv primer</TD><TD>030</TD></TR><TR class="tableevenadpbl"><TD>expected amplicon size (bp)</TD><TD>1935 bp</TD></TR></TBODY></TABLE></CENTER><P>The thermocycler programme was installed to have an extension time of 2 minutes and an annealing temperature of 60 °C. After the programme was finished, samples were kept at 4°C.
</P><DL><DT>Date</DT><DD>29/06/2018</DD><DT>Operator</DT><DD>Timothy Páez</DD></DL><H4> Transformation of chemically competent cells </H4><P>The following transformations were carried out according to the protocol for transformation of chemically competent cells:
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Strain</TH><TH class="tableheaderadpbl">Plasmid &amp; size (bp) </TH><TH class="tableheaderadpbl">Cloning method</TH><TH class="tableheaderadpbl">Volume added (uL)</TH><TH class="tableheaderadpbl">Medium on which was plated</TH></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I>DH5α</TD><TD>pACYC lin-Tn5 (3000 bp)</TD><TD>Restriction Ligation</TD><TD>10 µL</TD><TD>LB with Chloroamphenicol</TD></TR></TBODY></TABLE></CENTER><P>400 µL LB medium was added as recovery medium.
</P><DL><DT>Date</DT><DD>29/06/2018</DD><DT>Operator</DT><DD>Timothy Páez</DD></DL><H4>Liquid Starter Culture Preparation</H4><P><I>Escherichia coli</I> DH5 alpha harbouring plasmid with dxCas9  from Addgene was used for inoculation of 10 mL liquid starter culture (Luria Broth, complemented with ampicillin) following the Liquid Starter Culture Protocol.
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#FridaySC4">Saturday, 30/06/2018</BUTTON><DIV id="FridaySC4" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>30/06/2018</DD><DT>Operator</DT><DD>Timothy Páez</DD></DL><H4>Colony PCR</H4><P>Colony PCR was done to verify propagation of <I>Escherichia coli</I> DH5 alpha cells after transformation with pACYC containing lin-Tn5 (reaction 1) or Tn5-lin (reaction 2) as insert. This insertion can be checked with primers 029 and 030 (on T7 promoter and terminator).
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH></TR><TR class="tableunevenadpbl"><TD>Template DNA</TD><TD>Colony solution</TD></TR><TR class="tableevenadpbl"><TD>Forward primer</TD><TD>029</TD></TR><TR class="tableunevenadpbl"><TD>Reverse primer</TD><TD>030</TD></TR><TR class="tableunevenadpbl"><TD>Expected amplicon size (bp)</TD><TD>1500 bp</TD></TR></TBODY></TABLE></CENTER><P> The number of colonies screened were:
    - Tn5-lin: Colonies 1-6 
    - lin-Tn6: Colonies 8-10
    - Control self ligation: Colony 11
The thermocycler programme was installed to have an extension time of 1 min 30 sec and an annealing temperature of 55 °C. After the programme was finished, samples were kept at 4°C.</P><DL><DT>Date</DT><DD>30/06/2018</DD><DT>Operator</DT><DD>Timothy Páez</DD></DL><H4>DNA Gel Electrophoresis</H4><P>Colony PCR products were run on a 0.8 % agarose gel in TAE buffer. 
    - 5 μL of PCR product was pipetted into each lane (no loading buffer required as PCR buffer contains loading buffer).
    - 4 µL of 100 bp DNA ladder 
    - 80 V for 30 minutes 
    - The gel was imaged using a gel documentation system with UV-light.</P><FIGURE><FIGCAPTION class="adpbl">Figure 1. Gele electrophoresis from colony PCR. Lanes are (L) ladder 100 bp, (1) Colony 1,
    (2) Colony 2, (3) colony 3, (4) colony 4, (5) colony 5, (6) colony 6, (7) colony 7, (8) colony 8, (9) colony 9, (10) colony 10, (11) colony 11 </FIGCAPTION></FIGURE><DL><DT>Date</DT><DD>30/06/2018</DD><DT>Operator</DT><DD>Timothy Páez</DD></DL><H4>Liquid Starter Culture Preparation</H4><P><I>Escherichia coli</I> DH5 alpha cells that possibly harbour positive pACYC1-duet plasmids with lin-Tn5 were used for inoculation of 10 mL liquid starter culture (Luria Broth, complemented with chloroamphenicol) following the Liquid Starter Culture Protocol.
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV></DIV></UL></FORM><H3 class="adpbl">July</H3><FORM><LABEL for="WeekSCJ1" class="subdropdown cadpbl">Week 1 02/07/2018 - 08/07/2018</LABEL><UL class="submenu"><DIV class="subcontent"><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#MondaySCJ1">Monday, 02/07/2018</BUTTON><DIV id="MondaySCJ1" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>2/7/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>Sequence Verification </H4><H5>Option 1: Sequence from plasmid</H5><P> For verification of pTXB1-Tn5, primer FP030 was used for sequencing. The sequencing starts from the T7 terminator, in the reverse orientation respective to the Tn5 CDS. The 10µL sequencing sample was prepared in a 1.5 mL tube as follows:  
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL)</TH><TH class="tableheaderadpbl">Final concentration</TH></TR><TR class="tableunevenadpbl"><TD>Plasmid DNA (500ng)</TD><TD>5.3</TD><TD>50 ng/uL</TD></TR><TR class="tableevenadpbl"><TD>10µM primer 030 </TD><TD>2.5</TD><TD>25 pmol</TD></TR><TR class="tableunevenadpbl"><TD>MilliQ</TD><TD>2.1</TD><TD>-</TD></TR></TBODY></TABLE></CENTER><P>Within 7 days, the sequence was retrievable online and alignment with the <I>in silico</I> designed molecule showed 100% match of the 1050 bp downstream of the sequencing primer binding site. </P><DL><DT>Date</DT><DD>02/07/2018</DD><DT>Operator</DT><DD>Timothy Páez</DD></DL><H4>Sequence Verification </H4><H5>Sequence from plasmid</H5><P> For verification of Tn5-lin insertion in pACYC, primers 029 and 030 was used for sequencing. The sequencing starts from the -20 bp position of fragment Tn5-lin, flanking the promoter of T7 and terminator from T7 of the vector. Sequencing sample was prepared in 1.5 mL tubes for each primer as indicated my Macrogen for specific concentration of plasmid DNA.
    </P><P> Within 2 days, the sequence was retrievable online and alignment with the in silico designed molecule showed 100% match of the insert lin-Tn5, by aligning both primer sequences obtained. </P><DL><DT>Date</DT><DD>02/07/2018</DD><DT>Operator</DT><DD>Timothy Páez</DD></DL><H4>High Fidelity PCR</H4><P> To amplify dxCas9 (4 kbp), high fidelity PCR was done using Addgene plasmid harbouring dxCas9 as template DNA and 001 and 002 as forward and reverse primers respectively. Primer sequences can be found at: <A href="https://2018.igem.org/Team:TUDelft/Primers" target="_blank"> Primer sequences</A>. 
    50 µL PCR mixtures were prepared, according to the High Fidelity PCR protocol.
    
    
    </P><P>The thermocycler programme was installed to have an extension time of 60 seconds and an annealing temperature of 60 °C. After the programme was finished, samples were kept at 4 °C.</P><DL><DT>Date</DT><DD>02/07/2018</DD><DT>Operator</DT><DD>Timothy Páez</DD></DL><H4>DNA Gel Electrophoresis</H4><P>dxCas9 PCR product was run on a 0.8 % agarose gel in TAE buffer. 5μL was pipetted into each lane with 1 µL of loading dye. The gel was imaged using a gel documentation system with UV-light. No amplification was observed. It is recomended to do PCR amplification again with longer extention time.</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#TuesdaySCJ1">Tuesday, 03/07/2018</BUTTON><DIV id="TuesdaySCJ1" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>3/7/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>High Fidelity PCR</H4><P> To amplify dxCas9 (4172bp), high fidelity PCR was done using dxCas(3.7)-VPR as template DNA and FP001 and FP002 as forward and reverse primers respectively. 6 25µL PCR mixtures were prepared, according to the High Fidelity PCR protocol.

The thermocycler programme was installed to have an extension time of 2 minutes and an annealing temperature of 65 °C. After the programme was finished, samples were kept at 4°C.

</P><DL><DT>Date</DT><DD>3/7/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>DNA Gel Electrophoresis</H4><P>dxCas9 PCR reaction products were run on a 0.8% agarose gel in TAE buffer. 5μL was pipetted into each lane. The gel was imaged using a gel documentation system with UV-light.
    This gel shows that only samples 1, 2, 4 and 5 were successful attempts.</P><DL><DT>Date</DT><DD>02/07/2018</DD><DT>Operator</DT><DD>Timothy Páez</DD></DL><H4>High Fidelity PCR</H4><P> To amplify dxCas9 (4 kbp), high fidelity PCR was done using Addgene plasmid harbouring dxCas9 as template DNA and 001 and 002 as forward and reverse primers respectively. Primer sequences can be found at: <A href="https://2018.igem.org/Team:TUDelft/Primers" target="_blank"> Primer sequences</A>. 
    50 µL PCR mixtures were prepared, according to the High Fidelity PCR protocol.
    
    
    </P><P>The thermocycler programme was installed to have an extension time of 2 minutes and an annealing temperature of 60 °C and 65 ºC. After the programme was finished, samples were kept at 4 °C.</P><DL><DT>Date</DT><DD>03/07/2018</DD><DT>Operator</DT><DD>Timothy Páez</DD></DL><H4>DNA Gel Electrophoresis</H4><P>PCR products of dxCas9 PCR were run on a 0.8 % agarose gel in TAE buffer. 
    - 5 μL of PCR product was pipetted into each lane with 1 µL of loading buffer.
    - 4 µL of 500 bp DNA ladder 
    - 80 V for 30 minutes 
    - The gel was imaged using a gel documentation system with UV-light.</P><FIGURE><FIGCAPTION class="adpbl">Figure 1. Gel electrophoresis from dxCas9 PCR. Lanes are (1-2) Amplification at 60 ºC anhelaing temperature, (3) negative control, (4-5) amplification at 65 ºC anhealing temperature and (6) negative control. </FIGCAPTION></FIGURE></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#WednesdaySCJ1">Wednesday, 04/07/2018</BUTTON><DIV id="WednesdaySCJ1" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>04/07/2018</DD><DT>Operator</DT><DD>Timothy Páez</DD></DL><H4>Restriction </H4><P>Restriction cloning was done for assembling Cas9 into pACYCduet-1 and pACYC-duet lin-Tn5. The following table shows an overview of what fragments were cut with what restriction enzymes.
    First Reaction (dxCas9 into pACYC):
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL)</TH></TR><TR class="tableunevenadpbl"><TD>10x CutSmart buffer    </TD><TD>2</TD></TR><TR class="tableunevenadpbl"><TD>Fragment (dxCas9)</TD><TD>15</TD></TR><TR class="tableunevenadpbl"><TD>Enzyme (NotI HF)</TD><TD>0.5</TD></TR><TR class="tableunevenadpbl"><TD>Enzyme (KpnI HF)</TD><TD>2.5</TD></TR><TR class="tableunevenadpbl"><TD>MilliQ</TD><TD> 0</TD></TR></TBODY></TABLE></CENTER><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL)</TH></TR><TR class="tableunevenadpbl"><TD>10x CutSmart buffer    </TD><TD>2</TD></TR><TR class="tableunevenadpbl"><TD>Vector (pACYC)</TD><TD>15</TD></TR><TR class="tableunevenadpbl"><TD>Enzyme (NotI HF)</TD><TD>0.5</TD></TR><TR class="tableunevenadpbl"><TD>Enzyme (KpnI HF)</TD><TD>2.5</TD></TR><TR class="tableunevenadpbl"><TD>MilliQ</TD><TD> 0</TD></TR></TBODY></TABLE></CENTER><P>Second reaction (dxCas9 into pACYC lin-Tn5):</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL)</TH></TR><TR class="tableunevenadpbl"><TD>10x CutSmart buffer    </TD><TD>2</TD></TR><TR class="tableunevenadpbl"><TD>Fragment (dxCas9)</TD><TD>15</TD></TR><TR class="tableunevenadpbl"><TD>Enzyme (NotI HF)</TD><TD>0.5</TD></TR><TR class="tableunevenadpbl"><TD>Enzyme (FseI HF)</TD><TD>2.5</TD></TR><TR class="tableunevenadpbl"><TD>MilliQ</TD><TD> 0</TD></TR></TBODY></TABLE></CENTER><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL)</TH></TR><TR class="tableunevenadpbl"><TD>10x CutSmart buffer    </TD><TD>2</TD></TR><TR class="tableunevenadpbl"><TD>Vector (pACYC lin-Tn5)</TD><TD>15</TD></TR><TR class="tableunevenadpbl"><TD>Enzyme (NotI HF)</TD><TD>0.5</TD></TR><TR class="tableunevenadpbl"><TD>Enzyme (FseI HF)</TD><TD>2.5</TD></TR><TR class="tableunevenadpbl"><TD>MilliQ</TD><TD> 0</TD></TR></TBODY></TABLE></CENTER><P>The samples were incubated for 1 hour at 37 °C. 

</P><DL><DT>Date</DT><DD>04/07/2018</DD><DT>Operator</DT><DD>Timothy Páez</DD></DL><H4>PCR clean-up</H4><P>
    20 μL of restriction product were subjected to PCR Clean-Up, according to the protocol. Elution was done in 20 µL Milli-Q. 
    Products were quantified with Nanodrop, yielding concentration of:
    - pACYC: 12.7 ng/µL 
    - dxCas9: 40 ng/µL 
    - pACYC Tn5-lin: 9 ng/µL 
    - dxCas9: 31 ng/µL </P></DIV><DL><DT>Date</DT><DD>04/07/2018</DD><DT>Operator</DT><DD>Timothy Páez</DD></DL><H4>Ligation</H4><P>In order to assemble cut fragments Tn5-lin into pACYCduet-1 and lin-Tn5 into pACYCduet-1, ligation was performed. This table gives an overview of which fragments were cut with which enzymes.

</P><P>First reaction:</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">DNA</TH><TH class="tableheaderadpbl">Size (bp) </TH><TH class="tableheaderadpbl">Cut with enzymes</TH></TR><TR class="tableunevenadpbl"><TD>pACYCDuet</TD><TD>1.5 kbp</TD><TD>NotI and KpnI</TD></TR><TR class="tableevenadpbl"><TD>dxCas9</TD><TD>4 kbp</TD><TD>NotI and KpnI</TD></TR></TBODY></TABLE></CENTER><P>The ligation reaction mixture was prepared with the following composition</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL) </TH></TR><TR class="tableunevenadpbl"><TD>Ligase Buffer (10x) </TD><TD>2 µL</TD></TR><TR class="tableevenadpbl"><TD>T4 DNA Ligase</TD><TD>1 µL</TD></TR><TR class="tableunevenadpbl"><TD>pACYC (~100 ng) </TD><TD>5.13 µL</TD></TR><TR class="tableunevenadpbl"><TD>dxCas9 (300 ng) </TD><TD>11.87 µL</TD></TR><TR class="tableunevenadpbl"><TD>MilliQ</TD><TD>0 µL</TD></TR></TBODY></TABLE></CENTER><P>Second reaction:</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">DNA</TH><TH class="tableheaderadpbl">Size (bp) </TH><TH class="tableheaderadpbl">Cut with enzymes</TH></TR><TR class="tableunevenadpbl"><TD>pACYCDuet lin Tn5</TD><TD>3 kbp</TD><TD>NotI and FseI</TD></TR><TR class="tableevenadpbl"><TD>dxCas9</TD><TD>4 kbp</TD><TD>NotI and FseI</TD></TR></TBODY></TABLE></CENTER><P>The ligation reaction mixture was prepared with the following composition</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL) </TH></TR><TR class="tableunevenadpbl"><TD>Ligase Buffer (10x) </TD><TD>2 µL</TD></TR><TR class="tableevenadpbl"><TD>T4 DNA Ligase</TD><TD>1 µL</TD></TR><TR class="tableunevenadpbl"><TD>pACYC linTn5 (~100 ng) </TD><TD>8.55 µL</TD></TR><TR class="tableunevenadpbl"><TD>dxCas9 (300 ng) </TD><TD>8.45 µL</TD></TR><TR class="tableunevenadpbl"><TD>MilliQ</TD><TD>0 µL</TD></TR></TBODY></TABLE></CENTER><P>The samples were incubated overnight at 4 ºC for ligation. </P></DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#ThursdaySCJ1">Thursday, 05/07/2018</BUTTON><DIV id="ThursdaySCJ1" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>5/7/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>High Fidelity PCR</H4><P>To amplify Tn5-linker (1505bp), high fidelity PCR was done using pTXB1-Tn5 as template DNA and FP007 and FP008 as forward and reverse primers respectively. 3 100µL PCR mixtures were prepared, according to the High Fidelity PCR protocol.

The thermocycler programme was installed to have an extension time of 1 minute and an annealing temperature of 60 °C. After the programme was finished, samples were kept at 4°C.


</P><DL><DT>Date</DT><DD>5/7/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>DNA Gel Electrophoresis</H4><P>Tn5-Linker PCR reaction products were run on a 0.8% agarose gel in TAE buffer. 5μL was pipetted into each lane. The gel was imaged using a gel documentation system with UV-light. The gel was not saved properly, but the results showed correct amplification of a ~1.5kb amplicon in sample 1 and 3.</P><DL><DT>Date</DT><DD>5/7/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>PCR clean-up</H4><P>
200 μL of PCR product was subjected to PCR Clean-Up, according to the protocol. Elution was done in 60 µL Milli-Q.
</P><DL><DT>Date</DT><DD>5/7/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>Nanodrop DNA Quantification</H4><P> 
    The following isolates were quantified according to the Nanodrop DNA Quantification protocol. The concentration and purity of the DNA (260/280 and 260/230 ratios) were measured. 
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Isolate</TH><TH class="tableheaderadpbl">Concentration (ng/uL)</TH><TH class="tableheaderadpbl">260/280 ratio</TH><TH class="tableheaderadpbl">260/230 ratio</TH></TR><TR class="tableunevenadpbl"><TD>Tn5-Linker</TD><TD>50</TD><TD>1.63</TD><TD>2.26</TD></TR></TBODY></TABLE></CENTER><DL><DT>Date</DT><DD>5/7/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>Restriction </H4><P>Restriction cloning was done for assembling Tn5-Linker and dxCas9 into pACYC-Duet &amp; for assembling dxCas9 into pACYC-Duet. The following table shows an overview of what fragments were cut with what restriction enzymes.
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (µL)</TH><TH class="tableheaderadpbl">Volume (µL)</TH><TH class="tableheaderadpbl">Volume (µL)</TH><TH class="tableheaderadpbl">Volume (µL)</TH><TH class="tableheaderadpbl">Volume (µL)</TH></TR><TR class="tableunevenadpbl"><TD>10x CutSmart buffer    </TD><TD>2</TD><TD>2</TD><TD>2</TD><TD>2</TD><TD>2</TD></TR><TR class="tableevenadpbl"><TD>Fragment</TD><TD>10 (~500ng) (Tn5-Linker)</TD><TD>15 (~1.5µg)(dxCas9)</TD><TD>15 (~1.5µg)(dxCas9)</TD><TD>15.25 (~2µg) (pACYC-DUET)</TD><TD>15.25 (~2µg) (pACYC-DUET)</TD></TR><TR class="tableunevenadpbl"><TD>Enzyme </TD><TD>0.25 (EcoRI)</TD><TD>2.5 (FseI)</TD><TD>0.5 (NotI)</TD><TD>2.5 (FseI)</TD><TD>0.5 (NotI)</TD></TR><TR class="tableevenadpbl"><TD>Enzyme </TD><TD>0.5 (NotI)</TD><TD>0.5 (NotI)</TD><TD>0.5 (KpnI)</TD><TD>0.25 (EcoRI)</TD><TD>0.5 (KpnI)</TD></TR><TR class="tableunevenadpbl"><TD>MilliQ</TD><TD> 7.25</TD><TD> 0</TD><TD> 2</TD><TD> 0</TD><TD> 2</TD></TR></TBODY></TABLE></CENTER><P>The samples were incubated for 1 hour(s) at 37 °C, and were not heat inactivated. 

</P><DL><DT>Date</DT><DD>5/7/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>PCR clean-up</H4><P>
    All restriction reaction mixtures were subjected to PCR Clean-Up, according to the protocol. Elution was done in 40 µL Milli-Q.
</P><DL><DT>Date</DT><DD>5/7/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>Nanodrop DNA Quantification</H4><P> 
    The following isolates were quantified according to the Nanodrop DNA Quantification protocol. The concentration and purity of the DNA (260/280 and 260/230 ratios) were measured. 
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Isolate</TH><TH class="tableheaderadpbl">Concentration (ng/uL)</TH><TH class="tableheaderadpbl">260/280 ratio</TH><TH class="tableheaderadpbl">260/230 ratio</TH></TR><TR class="tableunevenadpbl"><TD>Tn5-Linker (cut with EcoRI &amp; NotI)</TD><TD>6.5</TD><TD>1.74</TD><TD>2.26</TD></TR><TR class="tableevenadpbl"><TD>dxCas9 (cut with NotI &amp; FseI)</TD><TD>71</TD><TD>1.88</TD><TD>2.13</TD></TR><TR class="tableunevenadpbl"><TD>pACYC-Duet (cut with EcoRI &amp; FseI)</TD><TD>24</TD><TD>1.76</TD><TD>2.19</TD></TR><TR class="tableevenadpbl"><TD>dxCas9 (cut with NotI &amp; KpnI)</TD><TD>84</TD><TD>1.77</TD><TD>2.22</TD></TR><TR class="tableunevenadpbl"><TD>pACYC-Duet (Cut with Noti &amp; KpnI)</TD><TD>26</TD><TD>1.69</TD><TD>2.16</TD></TR></TBODY></TABLE></CENTER><DL><DT>Date</DT><DD>5/7/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>Ligation</H4><P>In order to assemble cut Tn5-Linker and dxCas9 into pACYC-Duet, ligation was performed. This table gives an overview of which fragments were cut with which enzymes.

</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">DNA</TH><TH class="tableheaderadpbl">Size (bp) </TH><TH class="tableheaderadpbl">Cut with enzymes</TH></TR><TR class="tableunevenadpbl"><TD>Vector (pACYC-DUET)</TD><TD>3792</TD><TD>EcoRI &amp; FseI</TD></TR><TR class="tableevenadpbl"><TD>Fragment 1 (Tn5-Linker)</TD><TD>1490</TD><TD>EcoRI &amp; NotI</TD></TR><TR class="tableunevenadpbl"><TD>Fragment 2 (dxCas9)</TD><TD>4172</TD><TD>FseI &amp; NotI</TD></TR></TBODY></TABLE></CENTER><P>A 20µL ligation reaction mixture was prepared with the following composition*</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (µL) </TH></TR><TR class="tableunevenadpbl"><TD>Ligase Buffer (10x) </TD><TD>2</TD></TR><TR class="tableevenadpbl"><TD>T4 DNA Ligase</TD><TD>1</TD></TR><TR class="tableunevenadpbl"><TD>DNA vector (~100 ng) </TD><TD>8</TD></TR><TR class="tableevenadpbl"><TD>DNA fragment 1 (26 ng) </TD><TD>4</TD></TR><TR class="tableunevenadpbl"><TD>DNA fragment 2 (150 ng) </TD><TD>2.4</TD></TR><TR class="tableevenadpbl"><TD>MilliQ</TD><TD>2.6</TD></TR></TBODY></TABLE></CENTER><P>In order to assemble cut dxCas9 into pACYC-Duet, another ligation was performed. This table gives an overview of which fragments were cut with which enzymes.

</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">DNA</TH><TH class="tableheaderadpbl">Size (bp) </TH><TH class="tableheaderadpbl">Cut with enzymes</TH></TR><TR class="tableunevenadpbl"><TD>Vector (pACYC-DUET)</TD><TD>3806</TD><TD>KpnI &amp; NotI</TD></TR><TR class="tableevenadpbl"><TD>Fragment 1 (dxCas9)</TD><TD>4161</TD><TD>KpnI &amp; NotI</TD></TR></TBODY></TABLE></CENTER><P>A 20µL ligation reaction mixture was prepared with the following composition*</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (µL) </TH></TR><TR class="tableunevenadpbl"><TD>Ligase Buffer (10x) </TD><TD>2</TD></TR><TR class="tableevenadpbl"><TD>T4 DNA Ligase</TD><TD>1</TD></TR><TR class="tableunevenadpbl"><TD>DNA vector (~100 ng) </TD><TD>2.2</TD></TR><TR class="tableevenadpbl"><TD>DNA fragment 1 (26 ng) </TD><TD>2</TD></TR><TR class="tableunevenadpbl"><TD>MilliQ</TD><TD>12.8</TD></TR></TBODY></TABLE></CENTER><P><I>*For both ligations, a negative control was included separately (in which the insert fragment was substituted with equivalent volume of milli-Q).</I>
All samples were incubated overnight on the bench (room temperature) for ligation.</P><DL><DT>Date</DT><DD>05/07/2018</DD><DT>Operator</DT><DD>Timothy Páez</DD></DL><H4> Transformation of chemically competent cells </H4><P>Using the ligate reactions from 04/07/2018, the following transformations were carried out according to the protocol for transformation of chemically competent cells:
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Strain</TH><TH class="tableheaderadpbl">Plasmid &amp; size (bp) </TH><TH class="tableheaderadpbl">Cloning method</TH><TH class="tableheaderadpbl">Volume added (uL)</TH><TH class="tableheaderadpbl">Medium on which was plated</TH></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I>DH5αlpha</TD><TD>pACYC dxCas9-lin-Tn5 (7500 bp)pACYC dxCas9 (5500 bp)</TD><TD>Restriction Ligation</TD><TD>10 µL</TD><TD>LB with Chloroamphenicol</TD></TR></TBODY></TABLE></CENTER><P>400 µL LB medium was added as recovery medium.
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#FridaySCJ1">Friday, 06/07/2018</BUTTON><DIV id="FridaySCJ1" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>6/7/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4> Transformation of chemically competent cells </H4><P>The following transformations were carried out according to the protocol for transformation of chemically competent cells:
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Strain</TH><TH class="tableheaderadpbl">Plasmid &amp; size (bp) </TH><TH class="tableheaderadpbl">Cloning method</TH><TH class="tableheaderadpbl">Volume added (uL)</TH><TH class="tableheaderadpbl">Medium on which was plated</TH></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I>DH5α</TD><TD>pACYC-Tn5Linker-dxCas9 (9419bp)</TD><TD>Restriction-ligation</TD><TD>2.78 (~50ng)</TD><TD>LB+Cam</TD></TR><TR class="tableevenadpbl"><TD><I>E.coli</I>DH5α</TD><TD>pACYC-dxCas9 (7967bp)</TD><TD>Restriction-ligation</TD><TD>4.6 (~50ng)</TD><TD>LB+Cam</TD></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I>DH5α</TD><TD>3 negative controls (2x ligate without insert, 1x plain DH5α)</TD><TD>Restriction-ligation</TD><TD>20 </TD><TD>LB+Cam</TD></TR><TR class="tableevenadpbl"><TD><I>E.coli</I>DH5α</TD><TD>pACYC-DUET (pos. ctrl)</TD><TD>-</TD><TD>3</TD><TD>LB+Cam</TD></TR></TBODY></TABLE></CENTER><P>500 μl LB medium was added as recovery medium. 1x and 10x dilutions were plated for each transformation. 
</P><DL><DT>Date</DT><DD>06/07/2018</DD><DT>Operator</DT><DD>Timothy Páez</DD></DL><H4>Colony PCR</H4><P>Colony PCR was done to verify propagation of <I>Escherichia coli</I> DH5 alpha cells after transformation with pACYC containing dxCas9 (reaction 1) or pACYC containing dxCas9-lin-Tn5 (reaction 2) as insert. This insertion can be checked with primers 029 and 030 (on T7 promoter and terminator). Primer sequences can be found at: <A href="https://2018.igem.org/Team:TUDelft/Primers" target="_blank"> Primer sequences</A>.
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH></TR><TR class="tableunevenadpbl"><TD>Template DNA</TD><TD>Colony solution</TD></TR><TR class="tableevenadpbl"><TD>Forward primer</TD><TD>029</TD></TR><TR class="tableunevenadpbl"><TD>Reverse primer</TD><TD>030</TD></TR><TR class="tableunevenadpbl"><TD>Expected amplicon size (bp)</TD><TD>(R1) 4000 bp (R2) 6000 bp</TD></TR></TBODY></TABLE></CENTER><P> The number of colonies screened were:
    - Cas9-lin-Tn5: Colonies 1-17 
    - Cas9: Colonies 18-45
    - Control lin-Tn5: Colonies 46-68
    - Control pACYC: Colonies 69-70
The thermocycler programme was installed to have an extension time of 6 min and an annealing temperature of 55 °C. After the programme was finished, samples were kept at 4°C.</P><DL><DT>Date</DT><DD>06/07/2018</DD><DT>Operator</DT><DD>Timothy Páez</DD></DL><H4>DNA Gel Electrophoresis</H4><P>Colony PCR products were run on a 0.8 % agarose gel in TAE buffer. 
    - 5 μL of PCR product was pipetted into each lane (no loading buffer required as PCR buffer contains loading buffer).
    - 4 µL of 500 bp DNA ladder 
    - 80 V for 30 minutes 
    - The gel was imaged using a gel documentation system with UV-light.</P><FIGURE><FIGCAPTION class="adpbl">Figure 1. Gel electrophoresis from colony PCR. Lanes are (L) ladder 500 bp, (1-30) Colonies 1-39. </FIGCAPTION></FIGURE><FIGURE><FIGCAPTION class="adpbl">Figure 2. Gel electrophoresis from colony PCR. Lanes are (L) ladder 500 bp, (40-45) Colonies 40-45. </FIGCAPTION></FIGURE><FIGURE><FIGCAPTION class="adpbl">Figure 3. Gel electrophoresis from colony PCR. Lanes are (L) ladder 500 bp, (46-62) Colonies 46-62. </FIGCAPTION></FIGURE></DIV><DL><DT>Date</DT><DD>06/07/2018</DD><DT>Operator</DT><DD>Timothy Páez</DD></DL><H4>Liquid Starter Culture Preparation</H4><P><I>Escherichia coli</I> DH5 alpha cells that possibly harbour positive pACYC1-duet plasmids with dxCas9 were used for inoculation of 10 mL liquid starter culture (Luria Broth, complemented with chloroamphenicol) following the Liquid Starter Culture Protocol.
</P></DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></UL></FORM><FORM><LABEL for="WeekSCJ2" class="subdropdown cadpbl">Week 2 09/07/2018 - 15/07/2018</LABEL><UL class="submenu"><DIV class="subcontent"><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#MondaySCJ2">Monday, 09/07/2018</BUTTON><DIV id="MondaySCJ2" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>09/07/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">Restriction</H4><P>In our laboratory, we experienced some trouble with restriction ligation cloning. Therefore, I aimed for checking the efficiency of all restriction enzymes used in the cloning process. I used the pACYCDuet-1 plasmid and restricted this plasmid with the restriction enzyme XbaI and one of the restriction enzymes EcoRI, KpnI, NotI or FseI. To compare, I restricted the pACYCDuet-1 plasmid only with XbaI and as negative control I used no restriction enzyme to cut pACYCDuet-1.
</P><P>The following table shows an overview of what fragment(s) were cut with what restriction enzyme(s).
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL)</TH><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL)</TH><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL)</TH></TR><TR class="tableunevenadpbl"><TD>10x CutSmart buffer</TD><TD>2</TD><TD>10x CutSmart buffer</TD><TD>2</TD><TD>10x CutSmart buffer</TD><TD>2</TD></TR><TR class="tableunevenadpbl"><TD>pACYCDuet-1</TD><TD>10 (~1000 ng)</TD><TD>pACYCDuet-1</TD><TD>10 (~1000 ng)</TD><TD>pACYCDuet-1</TD><TD>10 (~1000 ng)</TD></TR><TR class="tableunevenadpbl"><TD>Enzyme (XbaI)</TD><TD>0.5</TD><TD>Enzyme (XbaI)</TD><TD>0.5</TD><TD>Enzyme (none)</TD><TD>0</TD></TR><TR class="tableunevenadpbl"><TD>Enzyme (EcoRI, KpnI, FseI or NotI)</TD><TD>0.5</TD><TD>Enzyme (none)</TD><TD>0</TD><TD>Enzyme (none)</TD><TD>0</TD></TR><TR class="tableunevenadpbl"><TD>MilliQ</TD><TD>7</TD><TD>MilliQ</TD><TD>7.5</TD><TD>MilliQ</TD><TD>8</TD></TR></TBODY></TABLE></CENTER><P>The samples were incubated for 4 hour(s) at 37 °C, and followed by 20 minute heat inactivation of the restriction enzyme(s). 
</P><DL><DT>Date</DT><DD>09/07/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">DNA Gel Electrophoresis</H4><P>The restriction efficiency experiment performed on 09/07/2018 checking for activity of NotI, EcoRI, FseI and KnpI was run on a 0.8% agarose gel in TAE buffer. 4 μL was pipetted into each lane. The gel was imaged using a gel documentation system with UV-light.</P><P>From the DNA gel electrophoresis, it can be concluded that the restriction enzymes work efficiently for restriction ligation cloning purposes.</P><DL><DT>Date</DT><DD>9/7/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4> Transformation Results </H4><P> After carrying out the transformation protocol on 6/7/2018, the plates were left for overnight incubation. After ~14-18hr incubation, the plates were stored at 4°C until 9/7/2018 for observing the results </P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Strain</TH><TH class="tableheaderadpbl">DNA </TH><TH class="tableheaderadpbl">Medium</TH><TH class="tableheaderadpbl">Result</TH></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I> DH5α </TD><TD>pACYC-Tn5Linker-dxCas9 (9419bp)</TD><TD>LB+Cam</TD><TD>20 colonies (1x diluted)
        4 colonies (10x diluted)</TD></TR><TR class="tableevenadpbl"><TD><I>E.coli</I> DH5α </TD><TD>pACYC-dxCas9 (7967bp)</TD><TD>LB+Cam</TD><TD>3 colonies (1x diluted)
          No colonies (10x diluted)</TD></TR><TR class="tableevenadpbl"><TD><I>E.coli</I> DH5α </TD><TD>None (all neg. controls)</TD><TD>LB+Cam</TD><TD>no colonies</TD></TR><TR class="tableevenadpbl"><TD><I>E.coli</I> DH5α </TD><TD>pACYC-Duet (pos. ctrl.)</TD><TD>LB+Cam</TD><TD>&gt;500 colonies</TD></TR></TBODY></TABLE></CENTER><DL><DT>Date</DT><DD>9/7/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>Colony PCR</H4><P>Colony PCR was done to verify propagation of pACYC-Tn5Linker-dxCas9 (9419bp) and pACYC-dxCas9 (7967bp) after transformation on 6/7/2018. These plasmids contain the T7 expression system, which allow primers FP029 and FP030 to anneal.
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl"><B>Template DNA</B></TH><TH class="tableheaderadpbl">pACYC-Tn5Linker-dxCas9</TH><TH class="tableheaderadpbl">pACYC-dxCas9</TH></TR><TR class="tableunevenadpbl"><TD><B>FW primer</B></TD><TD>029</TD><TD>029</TD></TR><TR class="tableevenadpbl"><TD><B>RV primer</B></TD><TD>030</TD><TD>030</TD></TR><TR class="tableunevenadpbl"><TD><B>Expected amplicon size (bp)</B></TD><TD>5894</TD><TD>4442</TD></TR></TBODY></TABLE></CENTER><P>27 tubes were prepared for colony PCR, containing <U>picked colony</U> as template DNA. pACYC was used as positive control template. The thermocycler programme was installed to have an extension time of 6 minutes and an annealing temperature of 60 °C. After the programme was finished, samples were kept at 4°C.</P><DL><DT>Date</DT><DD>9/7/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>DNA Gel Electrophoresis</H4><P>All colony PCR samples were run on a 0.8% agarose gel in TAE buffer. 5μL was pipetted into each lane. The gel was imaged using a gel documentation system with UV-light. None of the reactions, not even the positive control showed presence of amplicon.</P><DL><DT>Date</DT><DD>09/07/2018</DD><DT>Operator</DT><DD>Timothy Páez</DD></DL><H4>Sequence Verification </H4><H5>Sequence from plasmid</H5><P> For verification of Tn5-lin insertion in pACYC, primers 003, 012, 029 and 030 were used for sequencing. The sequencing starts from the -20 bp position of fragment Tn5-lin, flanking the promoter of T7 and terminator from T7 of the vector. Sequencing sample was prepared in 1.5 mL tubes for each primer as indicated my Macrogen for specific concentration of plasmid DNA.
    
    Primer sequences can be found at: <A href="https://2018.igem.org/Team:TUDelft/Primers" target="_blank"> Primer sequences</A>. </P><P> Within 2 days, the sequence was retrievable online and alignment with the in silico designed molecule showed 100% match of the insert lin-Tn5, by aligning both primer sequences obtained. </P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#TuesdaySCJ2">Tuesday, 10/07/2018</BUTTON><DIV id="TuesdaySCJ2" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>10/07/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">Restriction</H4><P>Restriction cloning was done for assembling Tn5-Linker and dxCas9 separately into pACYCDuet-1 or pACYCDuet-1_Lin-Tn5, respectively. The following table shows an overview of what fragment(s) were cut with what restriction enzymes.</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL)</TH><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL)</TH></TR><TR class="tableunevenadpbl"><TD>10x CutSmart buffer</TD><TD>5</TD><TD>10x CutSmart buffer</TD><TD>5</TD></TR><TR class="tableevenadpbl"><TD>pACYCDuet-1</TD><TD>41 (2000 ng)</TD><TD>Fragment (Tn5-Lin)</TD><TD>41 (2000 ng)</TD></TR><TR class="tableunevenadpbl"><TD>Enzyme (NotI)</TD><TD>2</TD><TD>Enzyme (NotI)</TD><TD>2</TD></TR><TR class="tableevenadpbl"><TD>Enzyme (EcoRI)</TD><TD>2</TD><TD>Enzyme (EcoRI)</TD><TD>2</TD></TR><TR class="tableunevenadpbl"><TD>MilliQ</TD><TD>0</TD><TD>MilliQ</TD><TD>0</TD></TR></TBODY></TABLE></CENTER><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL)</TH><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL)</TH></TR><TR class="tableunevenadpbl"><TD>10x CutSmart buffer</TD><TD>5</TD><TD>10x CutSmart buffer</TD><TD>5</TD></TR><TR class="tableevenadpbl"><TD>pACYCDuet-1_Lin-Tn5</TD><TD>41 (2000 ng)</TD><TD>Fragment (dxCas9)</TD><TD>41 (2000 ng)</TD></TR><TR class="tableunevenadpbl"><TD>Enzyme (NotI)</TD><TD>2</TD><TD>Enzyme (NotI)</TD><TD>2</TD></TR><TR class="tableevenadpbl"><TD>Enzyme (FseI)</TD><TD>2</TD><TD>Enzyme (FseI)</TD><TD>2</TD></TR><TR class="tableunevenadpbl"><TD>MilliQ</TD><TD>0</TD><TD>MilliQ</TD><TD>0</TD></TR></TBODY></TABLE></CENTER><P>The samples were incubated overnight at 37 °C and followed by 20 minute heat inactivation (65 °C) of the restriction enzymes.</P><DL><DT>Date</DT><DD>10/07/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">PCR Clean-Up</H4><P>20 μL of each restriction reaction mixture was subjected to PCR Clean-Up, according to the <A href="https://2018.igem.org/Team:TUDelft/Experiments#PCRcleanup-scroll" class="adpbl" target="_blank">protocol</A>. Elution was done in 30 µL Milli-Q.</P><DL><DT>Date</DT><DD>10/07/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">Nanodrop DNA Quantification</H4><P>The following isolates were quantified according to the <A href="https://2018.igem.org/Team:TUDelft/Experiments#NanodropDNA-scroll" class="adpbl" target="_blank">Nanodrop DNA Quantification protocol</A>. The concentration and purity of the DNA (260/280 and 260/230 ratios) were measured.</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Isolate</TH><TH class="tableheaderadpbl">Concentration (ng/uL)</TH><TH class="tableheaderadpbl">260/280 ratio</TH><TH class="tableheaderadpbl">260/230 ratio</TH></TR><TR class="tableunevenadpbl"><TD>Tn5-Lin cut with EcoRI and NotI</TD><TD>36.3</TD><TD>1.80</TD><TD>0.09</TD></TR><TR class="tableevenadpbl"><TD>pACYCDuet-1 cut with EcoRI and NotI</TD><TD>53.6</TD><TD>1.88</TD><TD>0.17</TD></TR><TR class="tableunevenadpbl"><TD>dxCas9 cut with NotI and FseI</TD><TD>59.5</TD><TD>1.87</TD><TD>0.49</TD></TR><TR class="tableevenadpbl"><TD>pACYCDuet-1_Lin-Tn5 cut with FseI and NotI</TD><TD>50.0</TD><TD>2.00</TD><TD>0.47</TD></TR></TBODY></TABLE></CENTER><DL><DT>Date</DT><DD>10/07/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">Ligation</H4><P>In order to assemble cut fragments pACYCDuet-1 with Tn-Linker and pACYCDuet-1_Lin-Tn5 with dxCas9 ligation was performed. This table gives an overview of which fragments were cut with which enzymes.</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">DNA</TH><TH class="tableheaderadpbl">Size (bp) </TH><TH class="tableheaderadpbl">Cut with enzymes</TH></TR><TR class="tableunevenadpbl"><TD>Vector pACYCDuet-1</TD><TD>3970</TD><TD>EcoRI and NotI</TD></TR><TR class="tableevenadpbl"><TD>Fragment Tn5-Linker</TD><TD>1490</TD><TD>EcoRI and NotI</TD></TR></TBODY></TABLE></CENTER><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">DNA</TH><TH class="tableheaderadpbl">Size (bp) </TH><TH class="tableheaderadpbl">Cut with enzymes</TH></TR><TR class="tableunevenadpbl"><TD>Vector pACYCDuet-1_Lin-Tn5</TD><TD>5299</TD><TD>FseI and NotI</TD></TR><TR class="tableevenadpbl"><TD>Fragment dxCas9</TD><TD>4150</TD><TD>FseI and NotI</TD></TR></TBODY></TABLE></CENTER><P>A 20 μL ligation reaction mixture was prepared with the following composition:</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (μL)</TH><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (μL)</TH></TR><TR class="tableunevenadpbl"><TD>Ligase Buffer (10X)</TD><TD>2</TD><TD>Ligase Buffer (10X)</TD><TD>2</TD></TR><TR class="tableevenadpbl"><TD>T4 DNA Ligase </TD><TD>1</TD><TD>T4 DNA Ligase </TD><TD>1</TD></TR><TR class="tableunevenadpbl"><TD>DNA vector (~200 ng)</TD><TD>5</TD><TD>DNA vector (~200 ng)</TD><TD>5</TD></TR><TR class="tableevenadpbl"><TD>DNA Tn5-Lin (~100 ng)</TD><TD>5</TD><TD>DNA dxCas9 (~600 ng)</TD><TD>12</TD></TR><TR class="tableunevenadpbl"><TD>MilliQ</TD><TD>7</TD><TD>MilliQ</TD><TD>0</TD></TR></TBODY></TABLE></CENTER><P>The samples were incubated for 8 hours at 4°C for ligation.</P><DL><DT>Date</DT><DD>20-06-2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">Transformation of chemically competent cells</H4><P>The following transformations were carried out according to the protocol for transformation of chemically competent cells:</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Strain</TH><TH class="tableheaderadpbl">Plasmid and size (bp)</TH><TH class="tableheaderadpbl">Cloning method</TH><TH class="tableheaderadpbl">Volume added (μl)</TH><TH class="tableheaderadpbl">Medium on which was plated</TH></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I> DH5α</TD><TD>pACYCDuet-1_Tn5-Lin (5460 bp)</TD><TD>Restriction/Ligation</TD><TD>4</TD><TD>LB-Cam</TD></TR><TR class="tableevenadpbl"><TD><I>E.coli</I> DH5α</TD><TD>pACYCDuet-1_dxCas9-Linker-Tn5 (9449 bp)</TD><TD>Restriction/Ligation</TD><TD>4</TD><TD>LB Cam</TD></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I> DH5α</TD><TD>pACYCDuet-1 restricted (neg. control)</TD><TD>Restriction</TD><TD>1</TD><TD>LB Cam</TD></TR><TR class="tableevenadpbl"><TD><I>E.coli</I> DH5α</TD><TD>pACYCDuet-1 (pos. control)</TD><TD>-</TD><TD>1</TD><TD>LB Cam</TD></TR></TBODY></TABLE></CENTER><P>
200 μl LB medium was added as recovery medium.</P><DL><DT>Date</DT><DD>10/7/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>Colony PCR</H4><P>Colony PCR was done to verify propagation of pACYC-Tn5Linker-dxCas9 (9419bp) and pACYC-dxCas9 (7967bp) after transformation on 6/7/2018. These plasmids contain the T7 expression system, which allow primers FP029 and FP030 to anneal.
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl"><B>Template DNA</B></TH><TH class="tableheaderadpbl">pACYC-Tn5Linker-dxCas9</TH><TH class="tableheaderadpbl">pACYC-dxCas9</TH></TR><TR class="tableunevenadpbl"><TD><B>FW primer</B></TD><TD>029</TD><TD>029</TD></TR><TR class="tableevenadpbl"><TD><B>RV primer</B></TD><TD>030</TD><TD>030</TD></TR><TR class="tableunevenadpbl"><TD><B>Expected amplicon size (bp)</B></TD><TD>5894</TD><TD>4442</TD></TR></TBODY></TABLE></CENTER><P>27 tubes were prepared for colony PCR, containing boiled colony as template DNA (colonies were picked from the restreaked plate on 9/7/2018). Vector pACYC was used as positive control template. The thermocycler programme was installed to have an extension time of 6 minutes and an annealing temperature of 60 °C. After the programme was finished, samples were kept at 4°C.</P><DL><DT>Date</DT><DD>10/7/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>DNA Gel Electrophoresis</H4><P>All colony PCR samples were run on a 0.8% agarose gel in TAE buffer. 5μL was pipetted into each lane. The gel was imaged using a gel documentation system with UV-light.
    Colonies 11, 13, 20, 21, 22, 23, 24 and 25 show vague smears around the expected height (Tn5LCas = 5894bp)</P><DL><DT>Date</DT><DD>10/7/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>Liquid Starter Culture Preparation</H4><P>
Colonies 11, 13, 20, 21, 22, 23, 24 and 25 (all potentially pACYC-Tn5Linker-dxCas9) were used for inoculation of 5mL liquid starter culture (LB+Cam) each, following the Liquid Starter Culture Protocol.
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#WednesdaySCJ2">Wednesday, 11/07/2018</BUTTON><DIV id="WednesdaySCJ2" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>11/07/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">Restriction</H4><P>In our laboratory, we experienced some trouble with restriction ligation cloning. Therefore, will use a different restriction buffer: the universal buffer of JenaBioscience. Also, I will try to ligate the Tn-Linker and Linker-Tn5 separately to the dxCas9 prior to cloning into pACYCDuet-1. The following table shows an overview of what fragment(s) were cut with what restriction enzyme(s).
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL)</TH><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL)</TH></TR><TR class="tableunevenadpbl"><TD>10x Universal buffer</TD><TD>2</TD><TD>10x Universal buffer</TD><TD>2</TD></TR><TR class="tableevenadpbl"><TD>dxCas9</TD><TD>10 (~2000 ng)</TD><TD>Tn5-Linker</TD><TD>10 (~2000 ng)</TD></TR><TR class="tableunevenadpbl"><TD>Enzyme (NotI)</TD><TD>0.5</TD><TD>Enzyme (NotI)</TD><TD>0.5</TD></TR><TR class="tableevenadpbl"><TD>MilliQ</TD><TD>7.5</TD><TD>MilliQ</TD><TD>7.5</TD></TR></TBODY></TABLE></CENTER><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL)</TH><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL)</TH></TR><TR class="tableunevenadpbl"><TD>10x Universal buffer</TD><TD>2</TD><TD>10x Universal buffer</TD><TD>2</TD></TR><TR class="tableevenadpbl"><TD>dxCas9</TD><TD>10 (~2000 ng)</TD><TD>Linker-Tn5</TD><TD>10 (~2000 ng)</TD></TR><TR class="tableunevenadpbl"><TD>Enzyme (FseI)</TD><TD>0.5</TD><TD>Enzyme (FseI)</TD><TD>0.5</TD></TR><TR class="tableevenadpbl"><TD>MilliQ</TD><TD>7.5</TD><TD>MilliQ</TD><TD>7.5</TD></TR></TBODY></TABLE></CENTER><P>The samples were incubated for 4 hour(s) at 37 °C, and followed by 20 minute heat inactivation of the restriction enzyme(s). 
</P><DL><DT>Date</DT><DD>11/07/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">Colony PCR</H4><P>Colony PCR was done to verify propagation of pACYC1Duet-1_Tn5-Lin after transformation from 10/07/2018. This plasmid contains the Tn5-Lin insert, which can be checked with primers 029 and 030. Also, the pACYC1Duet-1_dxCas9-Lin-Tn5 transformation resulted in colonies, which were also checked. However, on the negative control transformation plate with restricted pACYCDuet-1, colonies were observed.</P><CENTER><TABLE><TBODY><TR class="tableunevenadpbl"><TD>Template DNA</TD><TD>pACYCDuet-1_Tn5-Lin</TD></TR><TR class="tableevenadpbl"><TD>Fw primer</TD><TD>029</TD></TR><TR class="tableunevenadpbl"><TD>Rv primer</TD><TD>030</TD></TR><TR class="tableevenadpbl"><TD>expected amplicon size (bp)</TD><TD>1935 bp</TD></TR></TBODY></TABLE></CENTER><CENTER><TABLE><TBODY><TR class="tableunevenadpbl"><TD>Template DNA</TD><TD>pACYCDuet-1_dxCas9-Lin-Tn5</TD></TR><TR class="tableevenadpbl"><TD>Fw primer</TD><TD>029</TD></TR><TR class="tableunevenadpbl"><TD>Rv primer</TD><TD>030</TD></TR><TR class="tableevenadpbl"><TD>expected amplicon size (bp)</TD><TD>5924 bp</TD></TR></TBODY></TABLE></CENTER><P>15 tubes were prepared for colony PCR, containing pACYCDuet-1_dxCas9-LinTn5 as template DNA. Additionally, 14 tubes were prepared for colony PR containing pACYCDuet-1_Tn5-Lin as template DNA. For positive control, 4 tubes for colonies with pACYCDuet-1 were used. The thermocycler programme was installed to have an extension time of 6 and 4 minutes respectively and an annealing temperature of 60 °C. After the programme was finished, samples were kept at 4°C.
</P><DL><DT>Date</DT><DD>11/7/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>Plasmid Isolation</H4><P> 
    Overnight starter cultures of colonies 11, 13, 19, 20, 21, 22, 23, and 24 (all potentially pACYC-Tn5Linker-dxCas9)  were subjected to plasmid isolation following the Plasmid Isolation Protocol. Final elution was done with 30 μL of pre-warmed MilliQ.
</P><DL><DT>Date</DT><DD>11/7/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>Nanodrop DNA Quantification</H4><P> 
    Plasmid isolates were quantified according to the Nanodrop DNA Quantification protocol. The concentrations were measured and verified to be higher than 10ng/µl. The purity of the DNA (260/280 and 260/230 ratios) was also checked to be approximately 1.8~1.9.
</P><DL><DT>Date</DT><DD>11/7/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>Colony PCR</H4><P>Diagnostic PCR on isolated plasmids from inoculated colonies was done as attempt to verify propagation of pACYC-Tn5Linker-dxCas9 (9419bp) after transformation on 6/7/2018. This plasmids contains the T7 expression system, which allow primers FP029 and FP030 to anneal.
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl"><B>Template DNA</B></TH><TH class="tableheaderadpbl">pACYC-Tn5LinkerdxCas9</TH></TR><TR class="tableunevenadpbl"><TD><B>Fw primer</B></TD><TD>029</TD></TR><TR class="tableevenadpbl"><TD><B>Rv primer</B></TD><TD>030</TD></TR><TR class="tableunevenadpbl"><TD><B>Expected amplicon size (bp)</B></TD><TD>5894</TD></TR></TBODY></TABLE></CENTER><P>8 tubes were prepared for colony PCR, containing isolated plasmid as template DNA (isolated from colonies inoculated on 10/7/2018). Vector pACYC was used as positive control template. The thermocycler programme was installed to have an extension time of 6.5 minutes and an annealing temperature of 60 °C. After the programme was finished, samples were kept at 4°C.</P><DL><DT>Date</DT><DD>11/7/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>DNA Gel Electrophoresis</H4><P>Colony PCR samples were run on a 0.8% agarose gel in TAE buffer. 5μL was pipetted into each lane. The gel was imaged using a gel documentation system with UV-light. 
    This gel showed all of the plasmid isolate templates result in double bands, which is corresponding to the pACYC-DUET control (lane 9). Compared to the Tn5 insert (lane 10), this indicates all of the plasmid isolates are pACYC-DUET and contain no insert.</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#ThursdaySCJ2">Thursday, 12/07/2018</BUTTON><DIV id="ThursdaySCJ2" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>12/07/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">Colony PCR</H4><P>Colony PCR was done to verify propagation of pACYC1Duet-1_Lin-Tn5 after transformation from 11/07/2018. This plasmid contains the Lin-Tn5 insert, which can be checked with primers 029 and 030. </P><CENTER><TABLE><TBODY><TR class="tableunevenadpbl"><TD>Template DNA</TD><TD>pACYCDuet-1_Lin-Tn5</TD></TR><TR class="tableevenadpbl"><TD>Fw primer</TD><TD>029</TD></TR><TR class="tableunevenadpbl"><TD>Rv primer</TD><TD>030</TD></TR><TR class="tableevenadpbl"><TD>expected amplicon size (bp)</TD><TD>1722 bp</TD></TR></TBODY></TABLE></CENTER><P>23 tubes were prepared for colony PCR, containing pACYCDuet-1_Lin-Tn5 as template DNA. The thermocycler programme was installed to have an extension time of 2 minutes and an annealing temperature of 60 °C. After the programme was finished, samples were kept at 4°C.
</P><DL><DT>Date</DT><DD>12/06/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">DNA Gel Electrophoresis</H4><P>The colony PCR performed on 11-07-2018 checking for pACYCDuet-1_Tn5-Linker and pACYCDuet-1_dxCas9-Lin-Tn5 was run on a 0.8% agarose gel in TAE buffer. 4 μL was pipetted into each lane. The gel was imaged using a gel documentation system with UV-light.</P><P>Unfortunately, no band was observed that corresponds to the dxCas9 fragment in pACYCDuet-1_Linker-Tn5. Also, no band was observed that indicates integration of Tn5-Lin into pACYCDuet. The bands are at the same height as the positive control pACYCDuet-1, meaning that no insertion has taken place.</P><DL><DT>Date</DT><DD>12/7/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>Restriction </H4><P>Restriction cloning was done for assembling Tn5-Linker and dxCas9 into pACYC-Duet &amp; for assembling dxCas9 and Linker-Tn5 into pACYC-Duet. The following table shows an overview of what fragments were cut with what restriction enzymes.</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (µL)</TH><TH class="tableheaderadpbl">Volume (µL)</TH><TH class="tableheaderadpbl">Volume (µL)</TH><TH class="tableheaderadpbl">Volume (µL)</TH><TH class="tableheaderadpbl">Volume (µL)</TH></TR><TR class="tableunevenadpbl"><TD>10x CutSmart buffer    </TD><TD>5</TD><TD>5</TD><TD>5</TD><TD>5</TD><TD>5</TD></TR><TR class="tableevenadpbl"><TD>Fragment</TD><TD>5 (~1µg) (Tn5-Linker)</TD><TD>14 (~1µg)(dxCas9)</TD><TD>25 (~1µg)(pACYC-DUET)</TD><TD>5 (~2µg) (Linker-Tn5)</TD><TD>25 (~1µg) (pACYC-DUET)</TD></TR><TR class="tableunevenadpbl"><TD>Enzyme </TD><TD>0.5 (EcoRI)</TD><TD>1 (FseI)</TD><TD>1 (FseI)</TD><TD>1 (FseI)</TD><TD>0.5 (NotI)</TD></TR><TR class="tableevenadpbl"><TD>Enzyme </TD><TD>0.5 (NotI)</TD><TD>0.5 (NotI)</TD><TD>0.25 (EcoRI)</TD><TD>0.5 (KpnI)</TD><TD>0.5 (KpnI)</TD></TR><TR class="tableunevenadpbl"><TD>MilliQ</TD><TD> 39</TD><TD> 29.5</TD><TD> 18.5</TD><TD> 38.5</TD><TD> 19</TD></TR></TBODY></TABLE></CENTER><P>The samples were incubated for 4 hour(s) at 37 °C, and were not heat inactivated. 

</P><DL><DT>Date</DT><DD>12/7/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>PCR clean-up</H4><P>
   Restriction products from 12/7/2018 were subjected to PCR Clean-Up, according to the protocol. Elution was done in 30 µL Milli-Q.
</P><DL><DT>Date</DT><DD>12/7/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>Nanodrop DNA Quantification</H4><P> 
    The following isolates were quantified according to the Nanodrop DNA Quantification protocol. The concentration and purity of the DNA (260/280 and 260/230 ratios) were measured. 
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Isolate</TH><TH class="tableheaderadpbl">Cut with</TH><TH class="tableheaderadpbl">Concentration (ng/uL)</TH><TH class="tableheaderadpbl">260/280 ratio</TH><TH class="tableheaderadpbl">260/230 ratio</TH></TR><TR class="tableunevenadpbl"><TD>Tn5-Linker</TD><TD>EcoRI&amp;NotI</TD><TD>48</TD><TD>1.9</TD><TD>1.1</TD></TR><TR class="tableevenadpbl"><TD>Linker-Tn5</TD><TD>FseI&amp;KpnI</TD><TD>30</TD><TD>2</TD><TD>0.9</TD></TR><TR class="tableunevenadpbl"><TD> dxCas9</TD><TD> FseI &amp; NotI</TD><TD>8.2 </TD><TD>2.2 </TD><TD> 0.3</TD></TR><TR class="tableunevenadpbl"><TD>pACYC-DUET</TD><TD>FseI&amp;EcoRI</TD><TD>18</TD><TD>1.8</TD><TD>0.6</TD></TR><TR class="tableunevenadpbl"><TD>pACYC-DUET</TD><TD>FotI&amp;KpnI</TD><TD>15</TD><TD>1.8</TD><TD>0.6</TD></TR></TBODY></TABLE></CENTER><DL><DT>Date</DT><DD>12/7/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>Ligation</H4><P>In order to assemble cut Tn5-Linker and cut dxCas9 into cut pACYC-Duet, ligation was performed. This table gives an overview of which fragments were cut with which enzymes. the approach taken is a single-pot ligation, in which both inserts (dxCas9 and Tn5)

</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">DNA</TH><TH class="tableheaderadpbl">Size (bp) </TH><TH class="tableheaderadpbl">Cut with enzymes</TH></TR><TR class="tableunevenadpbl"><TD>Vector (pACYC-DUET)</TD><TD>3792</TD><TD>EcoRI &amp; FseI</TD></TR><TR class="tableevenadpbl"><TD>Fragment 1 (Tn5-Linker)</TD><TD>1490</TD><TD>EcoRI &amp; NotI</TD></TR><TR class="tableunevenadpbl"><TD>Fragment 2 (dxCas9)</TD><TD>4172</TD><TD>NotI &amp; FseI</TD></TR></TBODY></TABLE></CENTER><P>A 20 µl ligation reaction mixture was prepared with the following composition*</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL) </TH></TR><TR class="tableunevenadpbl"><TD>Ligase Buffer (10x) </TD><TD>3</TD></TR><TR class="tableevenadpbl"><TD>T4 DNA Ligase</TD><TD>1.5</TD></TR><TR class="tableunevenadpbl"><TD>DNA vector (~50 ng) </TD><TD>2.5</TD></TR><TR class="tableunevenadpbl"><TD>DNA fragment 1 (~50 ng) </TD><TD>1</TD></TR><TR class="tableunevenadpbl"><TD>DNA fragment 2 (~60 ng) </TD><TD>20</TD></TR><TR class="tableunevenadpbl"><TD>MilliQ</TD><TD>2</TD></TR></TBODY></TABLE></CENTER><P>In order to assemble cut dxCas9 and cut Linker-Tn5 into cut pACYC-Duet, another ligation was performed. This table gives an overview of which fragments were cut with which enzymes.</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">DNA</TH><TH class="tableheaderadpbl">Size (bp) </TH><TH class="tableheaderadpbl">Cut with enzymes</TH></TR><TR class="tableunevenadpbl"><TD>Vector (pACYC-DUET)</TD><TD>3806</TD><TD>NotI &amp; KpnI</TD></TR><TR class="tableevenadpbl"><TD>Fragment 1 (dxCas9)</TD><TD>4161</TD><TD>FseI &amp; NotI</TD></TR><TR class="tableunevenadpbl"><TD>Fragment 2 (Linker-Tn5)</TD><TD>1508</TD><TD>FseI &amp; KpnI</TD></TR></TBODY></TABLE></CENTER><P>A 20 µl ligation reaction mixture was prepared with the following composition*</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL) </TH></TR><TR class="tableunevenadpbl"><TD>Ligase Buffer (10x) </TD><TD>3</TD></TR><TR class="tableevenadpbl"><TD>T4 DNA Ligase</TD><TD>1.5</TD></TR><TR class="tableunevenadpbl"><TD>DNA vector (~50 ng) </TD><TD>2.5</TD></TR><TR class="tableunevenadpbl"><TD>DNA fragment 1 (~50 ng) </TD><TD>20</TD></TR><TR class="tableunevenadpbl"><TD>DNA fragment 2 (~50 ng) </TD><TD>1.8</TD></TR><TR class="tableunevenadpbl"><TD>MilliQ</TD><TD>1.2</TD></TR></TBODY></TABLE></CENTER><P><I>*For both ligations, a negative control was included separately (in which ONE of the insert fragments was substituted with equivalent volume of milli-Q).</I>
     All samples were incubated overnight on the bench (room temperature) for ligation.</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#FridaySCJ2">Friday, 13/07/2018</BUTTON><DIV id="FridaySCJ2" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>13/07/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">PCR Clean-Up</H4><P>20 μL of each restriction reaction mixture done on 11/07/2018 was subjected to PCR Clean-Up, according to the <A href="https://2018.igem.org/Team:TUDelft/Experiments#PCRcleanup-scroll" class="adpbl" target="_blank">protocol</A>. Elution was done in 30 µL Milli-Q.</P><DL><DT>Date</DT><DD>13/07/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">Nanodrop DNA Quantification</H4><P>The following isolates were quantified according to the <A href="https://2018.igem.org/Team:TUDelft/Experiments#NanodropDNA-scroll" class="adpbl" target="_blank">Nanodrop DNA Quantification protocol</A>. The concentration and purity of the DNA (260/280 and 260/230 ratios) were measured. </P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Isolate</TH><TH class="tableheaderadpbl">Concentration (ng/uL)</TH><TH class="tableheaderadpbl">260/280 ratio</TH><TH class="tableheaderadpbl">260/230 ratio</TH></TR><TR class="tableunevenadpbl"><TD>Tn5-Lin cut with NotI</TD><TD>58.1</TD><TD>1.92</TD><TD>0.70</TD></TR><TR class="tableevenadpbl"><TD>dxCas9 cut with NotI</TD><TD>72.3</TD><TD>1.84</TD><TD>1.68</TD></TR><TR class="tableunevenadpbl"><TD>Linker-Tn5 cut with FseI</TD><TD>59.5</TD><TD>1.88</TD><TD>1.34</TD></TR><TR class="tableevenadpbl"><TD>dxCas9 cut with FseI</TD><TD>48.7</TD><TD>1.84</TD><TD>0.76</TD></TR></TBODY></TABLE></CENTER><DL><DT>Date</DT><DD>13/07/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">Ligation</H4><P>In order to assemble cut fragments dxCas9 with Tn-Linker and dxCas9 with Linker-Tn5 respectively, ligation was performed. This table gives an overview of which fragments were cut with which enzymes.</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">DNA</TH><TH class="tableheaderadpbl">Size (bp) </TH><TH class="tableheaderadpbl">Cut with enzymes</TH></TR><TR class="tableunevenadpbl"><TD>dxCas9</TD><TD>4172</TD><TD>NotI</TD></TR><TR class="tableevenadpbl"><TD>Fragment Tn5-Linker</TD><TD>1495</TD><TD>NotI</TD></TR></TBODY></TABLE></CENTER><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">DNA</TH><TH class="tableheaderadpbl">Size (bp) </TH><TH class="tableheaderadpbl">Cut with enzymes</TH></TR><TR class="tableunevenadpbl"><TD>dxCas9</TD><TD>4157</TD><TD>FseI</TD></TR><TR class="tableevenadpbl"><TD>Linker-Tn5</TD><TD>1508</TD><TD>FseI</TD></TR></TBODY></TABLE></CENTER><P>A 20 μL ligation reaction mixture was prepared with the following composition:</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (μL)</TH><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (μL)</TH></TR><TR class="tableunevenadpbl"><TD>Ligase Buffer (10X)</TD><TD>2</TD><TD>Ligase Buffer (10X)</TD><TD>2</TD></TR><TR class="tableevenadpbl"><TD>T4 DNA Ligase </TD><TD>1</TD><TD>T4 DNA Ligase </TD><TD>1</TD></TR><TR class="tableunevenadpbl"><TD>DNA dxCas9 (~200 ng)</TD><TD>4</TD><TD>DNA dxCas9 (~200 ng)</TD><TD>4</TD></TR><TR class="tableevenadpbl"><TD>DNA Tn5-Lin (~100 ng)</TD><TD>2</TD><TD>DNA Lin-Tn5 (~100 ng)</TD><TD>2</TD></TR><TR class="tableunevenadpbl"><TD>MilliQ</TD><TD>11</TD><TD>MilliQ</TD><TD>11</TD></TR></TBODY></TABLE></CENTER><P>The samples were incubated for 2.5 hours at 4°C for ligation.</P><DL><DT>Date</DT><DD>19-06-2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">High Fidelity PCR</H4><P>To amplify the dxCas9-Linker-Tn5 (5655 bp) Tn5-Linker-dxCas9 (5660 bp) coding sequence from the ligation product, high fidelity PCR was done. The template DNA is the ligation mixture and the primers that are used are the combinations 001 &amp; 006 and 002 &amp; 007 respectively. Two 50µL PCR mixtures were prepared, according to the <A href="https://2018.igem.org/Team:TUDelft/Experiments#highfidelityPCR-scroll" target="_blank" class="adpbl">High Fidelity PCR protocol</A>. 
Primer combination(s) and expected fragment(s)
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Sample</TH><TH class="tableheaderadpbl">Fw primer</TH><TH class="tableheaderadpbl">Rv primer</TH><TH class="tableheaderadpbl">Expected fragment</TH><TH class="tableheaderadpbl">Expected size (bp) </TH></TR><TR class="tableunevenadpbl"><TD>1</TD><TD>001</TD><TD>006</TD><TD>dxCas9-Linker-Tn5</TD><TD>5655</TD></TR><TR class="tableevenadpbl"><TD>2</TD><TD>002</TD><TD>007</TD><TD>Tn5-Linker-dxCas9</TD><TD>5660</TD></TR></TBODY></TABLE></CENTER><P>The thermocycler programme was installed to have an extension time of 6 min 30 seconds and an annealing temperature of 60 °C. After the programme was finished, samples were kept at 4 °C.
</P><DL><DT>Date</DT><DD>13/7/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>DNA Gel Electrophoresis</H4><P>The restriction products from 11/7/2018 were run on a 0.8% agarose gel in TAE buffer. 5μL was pipetted into each lane. The gel was imaged using a gel documentation system with UV-light. 
    This gel shows that one of the dxCas9 samples (lane 3) was not purified. </P><DL><DT>Date</DT><DD>13/7/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>Diagnostic PCR</H4><P>Diagnostic PCR on ligation samples from 12/7/2018 were done to check for presence of correct constructs. Primer sets were chosen based on unique binding opportunities either to pACYC-Duet, dxCas9 and/or Tn5. The expected bands were estimated based on  <I>in silico</I> design.
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Template DNA</TH><TH colspan="4" class="tableheaderadpbl">pACYC-Tn5Linker-dxCas9</TH><TH colspan="4" class="tableheaderadpbl">pACYC-dxCas9-LinkerTn5</TH></TR><TR class="tableunevenadpbl"><TD>Fw primer</TD><TD>029</TD><TD>007</TD><TD>001</TD><TD>029</TD><TD>029</TD><TD>001</TD><TD>007</TD><TD>029</TD></TR><TR class="tableevenadpbl"><TD>Rv primer</TD><TD>030</TD><TD>002</TD><TD>030</TD><TD>006</TD><TD>030</TD><TD>006</TD><TD>030</TD><TD>002</TD></TR><TR class="tableunevenadpbl"><TD>Expected amplicon size (bp)</TD><TD>5907</TD><TD>5660</TD><TD>4295</TD><TD>1576</TD><TD>5924</TD><TD>5655</TD><TD>1561</TD><TD>4332</TD></TR></TBODY></TABLE></CENTER><P>The thermocycler programme was installed to have an extension time of 6.5 minutes and an annealing temperature of 60 °C. After the programme was finished, samples were kept at 4°C</P><DL><DT>Date</DT><DD>13/7/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4> Transformation of chemically competent cells </H4><P>The following transformations were carried out according to the protocol for transformation of chemically competent cells:
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Strain</TH><TH class="tableheaderadpbl">Plasmid &amp; size (bp) </TH><TH class="tableheaderadpbl">Cloning method</TH><TH class="tableheaderadpbl">Volume added (µL)</TH><TH class="tableheaderadpbl">Medium</TH></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I>DH5α</TD><TD>pACYC-Tn5Linker-dxCas9 (~10kb)</TD><TD>Restriction-ligation</TD><TD>20µL</TD><TD>LB+Cam</TD></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I>DH5α</TD><TD>pACYC-dxCas9-LinkerTn5 (~10kb)</TD><TD>Restriction-ligation</TD><TD>20µL</TD><TD>LB+Cam</TD></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I>DH5α</TD><TD>pACYC-dxCas9-LinkerTn5 (~10kb)</TD><TD>Restriction-ligation</TD><TD>-</TD><TD>LB+Cam</TD></TR></TBODY></TABLE></CENTER><P>500 μl LB medium was added as recovery medium. 1x and 10x dilutions were plated for each transformation. 
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV></DIV></UL></FORM><FORM><LABEL for="WeekSCJ3" class="subdropdown cadpbl">Week 3 16/07/2018 - 22/07/2018</LABEL><UL class="submenu"><DIV class="subcontent"><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#MondaySCJ3">Monday, 16/07/2018</BUTTON><DIV id="MondaySCJ3" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>16/07/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4>DNA Gel Electrophoresis</H4><P>The ligation performed on 13/07/2018 for construction of Tn5-Linker-dxCas9 and dxCas9-Lin-Tn5 was run on a 0.8% agarose gel in TAE buffer. 4 μL was pipetted into each lane. The gel was imaged using a gel documentation system with UV-light.</P><P>The ligation reaction resulted in the expected sizes of the dxCas9-Linker-Tn5 (5655 bp) Tn5-Linker-dxCas9 (5660 bp) coding sequence.</P><DL><DT>Date</DT><DD>16/07/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4>High Fidelity PCR</H4><P> To amplify dxCas9-Linker-Tn5 (5655 bp) Tn5-Linker-dxCas9 (5660 bp), high fidelity PCR was done using the ligation reactions as template DNA ausing the forward and reverse primers sets 001 &amp; 007 and 002 &amp; 006 respectively. 2 times 50µL PCR mixtures were prepared, according to the <A href="https://2018.igem.org/Team:TUDelft/Experiments#highfidelityPCR-scroll" target="_blank" class="adpbl">High Fidelity PCR protocol</A>.
 
Primer combination(s) and expected fragment(s)
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Sample</TH><TH class="tableheaderadpbl">Fw primer</TH><TH class="tableheaderadpbl">Rv primer</TH><TH class="tableheaderadpbl">expected fragment</TH><TH class="tableheaderadpbl">expected size (bp) </TH></TR><TR class="tableunevenadpbl"><TD>1</TD><TD>001</TD><TD>006</TD><TD>dxCas9-Linker-Tn5</TD><TD>5655</TD></TR><TR class="tableevenadpbl"><TD>2</TD><TD>002</TD><TD>007</TD><TD>Tn5-Linker-dxCas9</TD><TD>5660</TD></TR></TBODY></TABLE></CENTER><P>The thermocycler programme was installed to have an extension time of 60 minutes and an annealing temperature of 60 °C. After the programme was finished, samples were kept at 4 °C.</P><DL><DT>Date</DT><DD>16/07/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4>DNA Gel Electrophoresis</H4><P>To amplify the ligation products Tn5-Linker-dxCas9 and dxCas9-Lin-Tn5 a High Fidelity PCR was performed and the result was run on a 0.8% agarose gel in TAE buffer. 4 μL was pipetted into each lane. The gel was imaged using a gel documentation system with UV-light.</P><P>The PCR reaction on the ligation products resulted in the expected sizes of the dxCas9-Linker-Tn5 (5655 bp) Tn5-Linker-dxCas9 (5660 bp) coding sequence in large amounts of DNA. However, there is also one other band of DNA present, indicating an off-target amplicon generated during the PCR.</P><DL><DT>Date</DT><DD>16-07-2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">DNA Gel Excision and Purification</H4><P>Due to presence of off-target amplicons, the PCR mixture is again loaded on a gel. Subsequently, the required fragments containing Tn5-Linker-dxCas9 and dxCas9-Lin-Tn5 were excised. The agarose slice containing the desired DNA were purified by a DNA Clean-Up, according to the <A href="https://2018.igem.org/Team:TUDelft/Experiments#PCRcleanup-scroll" class="adpbl" target="_blank">protocol</A>. Elution was done in 30 µL prewarmed Milli-Q.</P><DL><DT>Date</DT><DD>16/07/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">Nanodrop DNA Quantification</H4><P> 
          The following isolates were quantified according to the <A href="https://2018.igem.org/Team:TUDelft/Experiments#NanodropDNA-scroll" class="adpbl" target="_blank">Nanodrop DNA Quantification protocol</A>. The concentration and purity of the DNA (260/280 and 260/230 ratios) were measured. 
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Isolate</TH><TH class="tableheaderadpbl">Concentration (ng/uL)</TH><TH class="tableheaderadpbl">260/280 ratio</TH><TH class="tableheaderadpbl">260/230 ratio</TH></TR><TR class="tableunevenadpbl"><TD>Tn5-Linker-dxCas9</TD><TD>37.0</TD><TD>1.32</TD><TD>0.77</TD></TR><TR class="tableevenadpbl"><TD>dxCas9-Lin-Tn5</TD><TD>35.9</TD><TD>1.87</TD><TD>0.52</TD></TR></TBODY></TABLE></CENTER><DL><DT>Date</DT><DD>16/07/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4>High Fidelity PCR</H4><P> To amplify dxCas9, Linker-Tn5 (1508 bp) and Tn5-Linkr (1505 bp), high fidelity PCR was done. The forward and reverse primers sets that were used are 001 &amp; 002 and 005 &amp; 006 and 007 &amp; 008 respectively. Two times 3 times 50µL PCR mixtures were prepared, according to the <A href="https://2018.igem.org/Team:TUDelft/Experiments#highfidelityPCR-scroll" target="_blank" class="adpbl">High Fidelity PCR protocol</A>. However, one of the sets of 3 times 50µL PCR mixtures uses HF buffer and Phusion Polymerase, while the other set of 3 times 50µL PCR mixtures uses Q5 buffer and Q5 Polymerase.
 
Primer combination(s) and expected fragment(s)
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Sample</TH><TH class="tableheaderadpbl">Fw primer</TH><TH class="tableheaderadpbl">Rv primer</TH><TH class="tableheaderadpbl">Expected fragment</TH><TH class="tableheaderadpbl">Expected size (bp) </TH></TR><TR class="tableunevenadpbl"><TD>1</TD><TD>001</TD><TD>002</TD><TD>dxCas9</TD><TD>4172</TD></TR><TR class="tableevenadpbl"><TD>2</TD><TD>005</TD><TD>006</TD><TD>Linker-Tn5</TD><TD>1508</TD></TR><TR class="tableunevenadpbl"><TD>3</TD><TD>007</TD><TD>008</TD><TD>Tn5-Linker</TD><TD>1505</TD></TR></TBODY></TABLE></CENTER><P>The thermocycler programme was installed to have an extension time of 6 minutes and 30 seconds and an annealing temperature of 60 °C. After the programme was finished, samples were kept at 4 °C.</P><DL><DT>Date</DT><DD>16/07/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4>DNA Gel Electrophoresis</H4><P>The amplicons generated during the High Fidelity PCR were run on a 0.8% agarose gel in TAE buffer. 4 μL was pipetted into each lane. The gel was imaged using a gel documentation system with UV-light.</P><P>The PCR reaction o resulted in the expected sizes of the dxCas9 (4172 bp) Tn5-Linker (1505 bp) and Linker-Tn5 (1508 bp) coding sequence in large amounts of DNA. </P><DL><DT>Date</DT><DD>16/07/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">Liquid Starter Culture Preparation</H4><P><I>E.coli</I> strains harbouring the plasmid pACYCDuet-1 and pACYCDuet-1_Lin-Tn5 were used for inoculation of 10 mL liquid starter culture (Luria Broth, complemented with ampicillin) following the <A href="https://2018.igem.org/Team:TUDelft/Experiments#liquidstarterculture-scroll" class="adpbl" target="_blank">Liquid Starter Culture Protocol</A>.
</P><DL><DT>Date</DT><DD>16/7/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>DNA Gel Electrophoresis</H4><P>Diagnostic PCR reaction products from 13/7/2018 were run on a 0.8% agarose gel in TAE buffer. 5μL was pipetted into each lane. The gel was imaged using a gel documentation system with UV-light. 
This gel shows that partially, the constructs have been ligated (e.g. lane 8 &amp; 9), but the complete plasmid does not seem to be constructed (lanes 2 &amp; 6).</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#TuesdaySCJ3">Tuesday, 17/07/2018</BUTTON><DIV id="TuesdaySCJ3" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>17/07/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">Plasmid Isolation</H4><P>
Overnight starter cultures of pACYCDuet-1_Lin-Tn5 and pACYCDuet-1 were subjected to plasmid isolation following the <A href="https://2018.igem.org/Team:TUDelft/Experiments#plasmidisolation-scroll" class="adpbl" target="_blank">Plasmid Isolation Protocol</A>. Final elution was done with 30 μL of pre-warmed MilliQ.
</P><DL><DT>Date</DT><DD>17/07/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">PCR Clean-Up</H4><P>50 μL of each PCR reaction  done on 16/07/2018 was subjected to PCR Clean-Up, according to the <A href="https://2018.igem.org/Team:TUDelft/Experiments#PCRcleanup-scroll" class="adpbl" target="_blank">protocol</A>. Elution was done in 30 µL Milli-Q.</P><DL><DT>Date</DT><DD>17-07-2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">Nanodrop DNA Quantification</H4><P>The following isolates were quantified according to the <A href="https://2018.igem.org/Team:TUDelft/Experiments#NanodropDNA-scroll" class="adpbl" target="_blank">Nanodrop DNA Quantification protocol</A>. The concentration and purity of the DNA (260/280 and 260/230 ratios) were measured. </P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Isolate</TH><TH class="tableheaderadpbl">Concentration (ng/uL)</TH><TH class="tableheaderadpbl">260/280 ratio</TH><TH class="tableheaderadpbl">260/230 ratio</TH></TR><TR class="tableunevenadpbl"><TD>pACYCDuet-1</TD><TD>7.1</TD><TD>2.19</TD><TD>1.23</TD></TR><TR class="tableevenadpbl"><TD>pACYCDuet-1</TD><TD>7.9</TD><TD>1.81</TD><TD>1.19</TD></TR><TR class="tableunevenadpbl"><TD>pACYCDuet-1</TD><TD>10.7</TD><TD>1.52</TD><TD>0.77</TD></TR><TR class="tableevenadpbl"><TD>pACYCDuet-1_Lin-Tn5</TD><TD>15.5</TD><TD>1.69</TD><TD>0.89</TD></TR><TR class="tableunevenadpbl"><TD>pACYCDuet-1_Lin-Tn5</TD><TD>10.1</TD><TD>1.59</TD><TD>0.99</TD></TR><TR class="tableevenadpbl"><TD>pACYCDuet-1_Lin-Tn5</TD><TD>10.2</TD><TD>1.64</TD><TD>0.97</TD></TR><TR class="tableunevenadpbl"><TD>Q5: Lin-Tn5</TD><TD>180.9</TD><TD>1.80</TD><TD>1.52</TD></TR><TR class="tableevenadpbl"><TD>Q5: Tn5-Lin</TD><TD>148.3</TD><TD>1.80</TD><TD>1.92</TD></TR><TR class="tableunevenadpbl"><TD>Q5: dxCas9</TD><TD>259.6</TD><TD>1.84</TD><TD>1.82</TD></TR><TR class="tableevenadpbl"><TD>Phusion: Lin-Tn5</TD><TD>183.8</TD><TD>1.85</TD><TD>1.86</TD></TR><TR class="tableunevenadpbl"><TD>Phusion: Tn5-Lin</TD><TD>164.1</TD><TD>1.85</TD><TD>1.86</TD></TR><TR class="tableevenadpbl"><TD>Phusion: dxCas9</TD><TD>233.9</TD><TD>1.83</TD><TD>1.77</TD></TR></TBODY></TABLE></CENTER><DL><DT>Date</DT><DD>17/7/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>High Fidelity PCR</H4><P> The following high fidelity PCR reactions were carried out:
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Amplicon</TH><TH class="tableheaderadpbl">Size (bp)</TH><TH class="tableheaderadpbl">Fw primer</TH><TH class="tableheaderadpbl">Rv primer</TH><TH class="tableheaderadpbl">template</TH><TH class="tableheaderadpbl">Elongation time (sec)</TH><TH class="tableheaderadpbl">Annealing temperature (°C)</TH></TR><TR class="tableunevenadpbl"><TD>LacZ-gRNA</TD><TD>314</TD><TD>IV003</TD><TD>IV004</TD><TD>Gblock</TD><TD>60</TD><TD>60</TD></TR><TR class="tableevenadpbl"><TD>KanR CDS</TD><TD>862</TD><TD>IV005</TD><TD>IV006</TD><TD>pSB1K3</TD><TD>60</TD><TD>60</TD></TR><TR class="tableunevenadpbl"><TD>dxCas9</TD><TD>~4200</TD><TD>001</TD><TD>002</TD><TD>dxCas9(3.7)-VPR</TD><TD>240</TD><TD>60</TD></TR><TR class="tableevenadpbl"><TD>Tn5-Linker</TD><TD>~1500</TD><TD>007</TD><TD>008</TD><TD>pACYC-Tn5Linker</TD><TD>240</TD><TD>60</TD></TR><TR class="tableunevenadpbl"><TD>Linker-Tn5</TD><TD>~1500</TD><TD>005</TD><TD>006</TD><TD>pACYC-LinkerTn5</TD><TD>240</TD><TD>60</TD></TR><TR class="tableevenadpbl"><TD>dxCas9-LinkerTn5</TD><TD>~6000</TD><TD>001</TD><TD>006</TD><TD>dxCas9 &amp; Tn5</TD><TD>240</TD><TD>60</TD></TR><TR class="tableunevenadpbl"><TD>Tn5Linker-dxCas9</TD><TD>~6000</TD><TD>007</TD><TD>002</TD><TD>dxCas9 &amp; Tn5</TD><TD>240</TD><TD>60</TD></TR></TBODY></TABLE></CENTER><P>PCR mixtures were prepared, according to the High Fidelity PCR protocol. After the programme was finished, samples were kept at 4°C.</P><DL><DT>Date</DT><DD>17/7/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>PCR clean-up</H4><P>
PCR products were subjected to PCR Clean-Up, according to the protocol. Elution was done in 50 µL Milli-Q.
</P><DL><DT>Date</DT><DD>17/7/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>Nanodrop DNA Quantification</H4><P> 
    The following isolates were quantified according to the Nanodrop DNA Quantification protocol. The concentration and purity of the DNA (260/280 and 260/230 ratios) were measured. 
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Isolate</TH><TH class="tableheaderadpbl">260/280 ratio</TH><TH class="tableheaderadpbl">260/230 ratio</TH><TH class="tableheaderadpbl">Concentration (ng/uL)</TH></TR><TR class="tableunevenadpbl"><TD>LacZ-gRNA</TD><TD>85</TD></TR><TR class="tableevenadpbl"><TD>KanR cassette</TD><TD>209</TD></TR><TR class="tableunevenadpbl"><TD>dxCas9</TD><TD>128</TD></TR><TR class="tableevenadpbl"><TD>Tn5Linker</TD><TD>162</TD></TR><TR class="tableunevenadpbl"><TD>LinkerTn5</TD><TD>1.87</TD><TD>2.02</TD><TD>181</TD></TR><TR class="tableevenadpbl"><TD>dxCas9Linker-Tn5</TD><TD>1.82</TD><TD>1.88</TD><TD>74.8</TD></TR><TR class="tableunevenadpbl"><TD>Tn5Linker-dxCas9</TD><TD>1.81</TD><TD>1.91</TD><TD>78.7</TD></TR><TR class="tableevenadpbl"><TD>dxCas9Tn5+upstr EcoRI</TD><TD>1.79</TD><TD>1.81</TD><TD>56.8</TD></TR><TR class="tableunevenadpbl"><TD>Tn5dxCas9 + upstr EcoRI</TD><TD>1.8</TD><TD>1.99</TD><TD>89.4</TD></TR><TR class="tableevenadpbl"><TD>dxCas9Tn5 + downstr EcoRI</TD><TD>1.85</TD><TD>1.93</TD><TD>109</TD></TR><TR class="tableunevenadpbl"><TD>Tn5dxCas9 + downstr EcoRI</TD><TD>1.82</TD><TD>1.99</TD><TD>101.4</TD></TR><TR class="tableevenadpbl"><TD>dxCas9</TD><TD>1.85</TD><TD>2.11</TD><TD>138</TD></TR></TBODY></TABLE></CENTER><DL><DT>Date</DT><DD>17/7/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>Restriction </H4><P>Restriction cloning was done for assembling Tn5-Linker and dxCas9 into pACYC-Duet &amp; for assembling dxCas9 into pACYC-Duet. The following table shows an overview of what fragments were cut with what restriction enzymes.
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL)</TH><TH class="tableheaderadpbl">Volume (uL)</TH><TH class="tableheaderadpbl">Volume (uL)</TH><TH class="tableheaderadpbl">Volume (uL)</TH><TH class="tableheaderadpbl">Volume (uL)</TH></TR><TR class="tableunevenadpbl"><TD>10x CutSmart buffer    </TD><TD>5</TD><TD>5</TD><TD>5</TD><TD>5</TD><TD>5</TD></TR><TR class="tableunevenadpbl"><TD>Fragment</TD><TD>43 (pACYC-LinkerTn5)</TD><TD>10 (dxCas9)</TD><TD>43 (pACYC-LinkerTn5)</TD><TD>15 (Tn5-Linker)</TD><TD>10 (dxCas9)</TD></TR><TR class="tableunevenadpbl"><TD>Enzyme</TD><TD>1 (NotI)</TD><TD>1 (FseI)</TD><TD>1 (EcoRI)</TD><TD>1 (EcoRI)</TD><TD>1 (NotI)</TD></TR><TR class="tableunevenadpbl"><TD>Enzyme</TD><TD>1 (FseI)</TD><TD>1 (NotI)</TD><TD>1 (KpnI)</TD><TD>1 (NotI)</TD><TD>1 (KpnI)</TD></TR><TR class="tableunevenadpbl"><TD>MilliQ</TD><TD> 0</TD><TD> 0</TD><TD> 0</TD><TD> 28</TD><TD> 33</TD></TR></TBODY></TABLE></CENTER><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL)</TH><TH class="tableheaderadpbl">Volume (uL)</TH><TH class="tableheaderadpbl">Volume (uL)</TH><TH class="tableheaderadpbl">Volume (uL)</TH><TH class="tableheaderadpbl">Volume (uL)</TH></TR><TR class="tableunevenadpbl"><TD>10x CutSmart buffer    </TD><TD>5</TD><TD>5</TD><TD>5</TD><TD>5</TD><TD>5</TD></TR><TR class="tableunevenadpbl"><TD>Fragment</TD><TD>43 (pACYC-DUET)</TD><TD>25 (Tn5Linker-dxCas9)</TD><TD>15 (Tn5-Linker)</TD><TD>43 (pACYC-DUET)</TD><TD>10 (dxCas9-LinkerTn5)</TD></TR><TR class="tableunevenadpbl"><TD>Enzyme</TD><TD>1 (NotI)</TD><TD>1 (FseI)</TD><TD>1 (EcoRI)</TD><TD>1 (EcoRI)</TD><TD>1 (NotI)</TD></TR><TR class="tableunevenadpbl"><TD>Enzyme</TD><TD>1 (FseI)</TD><TD>1 (NotI)</TD><TD>1 (KpnI)</TD><TD>1 (NotI)</TD><TD>1 (KpnI)</TD></TR><TR class="tableunevenadpbl"><TD>MilliQ</TD><TD> 0</TD><TD> 0</TD><TD> 0</TD><TD> 28</TD><TD> 33</TD></TR></TBODY></TABLE></CENTER><P>The samples were incubated for 4 hour(s) at 37 °C, and were not heat inactivated.

</P><DL><DT>Date</DT><DD>17/7/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>PCR clean-up</H4><P>
   Restriction products were subjected to PCR Clean-Up, according to the protocol. Elution was done in 50 µL Milli-Q.
</P><DL><DT>Date</DT><DD>17/7/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>Nanodrop DNA Quantification</H4><P> 
    The following isolates were quantified according to the Nanodrop DNA Quantification protocol. The concentration and purity of the DNA (260/280 and 260/230 ratios) were measured. 
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Fragment</TH><TH class="tableheaderadpbl">Cut with</TH><TH class="tableheaderadpbl">260/280 ratio</TH><TH class="tableheaderadpbl">260/230 ratio</TH><TH class="tableheaderadpbl">Concentration (ng/uL)</TH></TR><TR class="tableunevenadpbl"><TD>pACYC-LinkerTn5</TD><TD>NotI &amp; FseI</TD><TD>1.75</TD><TD>0.69</TD><TD>8</TD></TR><TR class="tableevenadpbl"><TD>dxCas9</TD><TD>NotI &amp; FseI</TD><TD>1.85</TD><TD>1.51</TD><TD>2.37</TD></TR><TR class="tableunevenadpbl"><TD>pACYC</TD><TD>EcoRI &amp; KpnI</TD><TD>1.88</TD><TD>1.05</TD><TD>10.6</TD></TR><TR class="tableevenadpbl"><TD>Tn5Linker</TD><TD>NotI &amp; EcoRI</TD><TD>1.82</TD><TD>1.33</TD><TD>26.1</TD></TR><TR class="tableunevenadpbl"><TD>dxCas9</TD><TD>NotI &amp; KpnI</TD><TD>1.86</TD><TD>1.51</TD><TD>29.6</TD></TR><TR class="tableevenadpbl"><TD>pACYC-DUET</TD><TD>NotI &amp; KpnI</TD><TD>1.81</TD><TD>1.03</TD><TD>13.8</TD></TR><TR class="tableunevenadpbl"><TD>dxCas9</TD><TD>NotI &amp; FseI</TD><TD>1.84</TD><TD>1.19</TD><TD>20.8</TD></TR><TR class="tableevenadpbl"><TD>Linker-Tn5</TD><TD>FseI &amp; KpnI</TD><TD>1.84</TD><TD>1.54</TD><TD>42</TD></TR><TR class="tableunevenadpbl"><TD>pACYC-DUET</TD><TD>NotI &amp; KnpI</TD><TD>1.79</TD><TD>1.06</TD><TD>13.1</TD></TR><TR class="tableevenadpbl"><TD>dxCas9-LinkerTn5</TD><TD>NotI &amp; KnpI</TD><TD>1.79</TD><TD>0.54</TD><TD>5.5</TD></TR><TR class="tableunevenadpbl"><TD>pACYC-DUET</TD><TD>EcoRI &amp; KpnI</TD><TD>1.67</TD><TD>0.97</TD><TD>13</TD></TR><TR class="tableevenadpbl"><TD>Tn5Linker-dxCas9</TD><TD>EcoRI &amp; KpnI</TD><TD>2</TD><TD>0.78</TD><TD>6.1</TD></TR></TBODY></TABLE></CENTER><DL><DT>Date</DT><DD>17/7/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>Ligation</H4><P>In order to assemble cut fragments, ligation was performed. This table gives an overview of which fragments were cut with which enzymes.

</P><P>A 50µl ligation reaction mixture was prepared with the following composition</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">L1* Volume (uL) </TH><TH class="tableheaderadpbl">L2* Volume (uL) </TH><TH class="tableheaderadpbl">L3* Volume (uL) </TH><TH class="tableheaderadpbl">L4* Volume (uL) </TH><TH class="tableheaderadpbl">L5* Volume (uL) </TH></TR><TR class="tableunevenadpbl"><TD>Ligase Buffer (10x) </TD><TD>5</TD><TD>5</TD><TD>5</TD><TD>5</TD><TD>5</TD></TR><TR class="tableevenadpbl"><TD>T4 DNA Ligase</TD><TD>2.5</TD><TD>2.5</TD><TD>2.5</TD><TD>2.5</TD><TD>2.5</TD></TR><TR class="tableunevenadpbl"><TD>DNA vector </TD><TD>24 (pACYC-LinkerTn5, ~200ng)</TD><TD>17 (pACYC, ~200ng)</TD><TD>12.5 (pACYC, ~150ng)</TD><TD>4 (pACYC, ~50ng)</TD><TD>4.3 (pACYC, ~100ng)</TD></TR><TR class="tableunevenadpbl"><TD>DNA insert 1 </TD><TD>18 (dxCas9, ~200ng)</TD><TD>17.5 (Tn5Linker, ~200ng)</TD><TD>25 (dxCas9, ~500ng)</TD><TD>39(dxCas9LinkerTn5, ~200ng)</TD><TD>17.5 (Tn5LinkerdxCas9, ~200ng)</TD></TR><TR class="tableevenadpbl"><TD>DNA nsert 2 (optional) </TD><TD>18 (dxCas9, ~200ng)</TD><TD>5 (LinkerTn5, ~200ng)</TD></TR><TR class="tableunevenadpbl"><TD>MilliQ</TD><TD>0.5</TD><TD>0</TD><TD>0</TD><TD>0</TD><TD>0.2</TD></TR></TBODY></TABLE></CENTER><P><I>*L1, L2, L3, L4 and L5 are the names of the ligation reactions. L2 and L3 are single-pot ligation reactions, which aim to ligate two inserts into 1 vector simultaneously. </I>
    The samples were incubated for 6 hours at 37°C for ligation. Samples were not heat inactivated.</P><DL><DT>Date</DT><DD>17/7/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>Solid Medium Preparation</H4><P>
Solid Luria Broth (LB) medium was prepared, complemented with chloramphenicol (Cam)
</P><DL><DT>Date</DT><DD>17/7/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4> Transformation of chemically competent cells </H4><P>The following transformations were carried out according to the protocol for transformation of chemically competent cells:
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Strain</TH><TH class="tableheaderadpbl">Plasmid &amp; size (bp) </TH><TH class="tableheaderadpbl">Cloning method</TH><TH class="tableheaderadpbl">Volume added (uL)</TH><TH class="tableheaderadpbl">Medium</TH></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I>DH5α</TD><TD>L1 (Ligate)</TD><TD>Restriction&amp;Ligation</TD><TD>20</TD><TD>LB+Cam</TD></TR><TR class="tableevenadpbl"><TD><I>E.coli</I>DH5α</TD><TD>L2 (Ligate)</TD><TD>Restriction&amp;Ligation</TD><TD>20</TD><TD>LB+Cam</TD></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I>DH5α</TD><TD>L3 (Ligate)</TD><TD>Restriction&amp;Ligation</TD><TD>20</TD><TD>LB+Cam</TD></TR><TR class="tableevenadpbl"><TD><I>E.coli</I>DH5α</TD><TD>L4 (Ligate)</TD><TD>Restriction&amp;Ligation</TD><TD>20</TD><TD>LB+Cam</TD></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I>DH5α</TD><TD>L5 (Ligate)</TD><TD>Restriction&amp;Ligation</TD><TD>20</TD><TD>LB+Cam</TD></TR></TBODY></TABLE></CENTER><P>200 uL LB medium was added as recovery medium.
</P><DL><DT>Date</DT><DD>17/07/2018</DD><DT>Operator</DT><DD>Timothy Páez</DD></DL><H4>Restriction </H4><P>Restriction cloning was done for assembling Cas9 into pACYCduet lin-Tn5 or lin-Tn5 into pACYCduet dxCas9. Both approaches will render the same insert. The following table shows an overview of what fragments were cut with what restriction enzymes.
    First Reaction (lin-Tn5 into pACYC dxCas9):
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL)</TH></TR><TR class="tableunevenadpbl"><TD>10x CutSmart buffer    </TD><TD>2</TD></TR><TR class="tableunevenadpbl"><TD>Fragment (lin-Tn5)</TD><TD>16</TD></TR><TR class="tableunevenadpbl"><TD>Enzyme (Fse HF)</TD><TD>1</TD></TR><TR class="tableunevenadpbl"><TD>Enzyme (KpnI HF)</TD><TD>1</TD></TR><TR class="tableunevenadpbl"><TD>MilliQ</TD><TD> 0</TD></TR></TBODY></TABLE></CENTER><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL)</TH></TR><TR class="tableunevenadpbl"><TD>10x CutSmart buffer    </TD><TD>2</TD></TR><TR class="tableunevenadpbl"><TD>Vector (pACYC dxCas9)</TD><TD>16</TD></TR><TR class="tableunevenadpbl"><TD>Enzyme (Fse HF)</TD><TD>1</TD></TR><TR class="tableunevenadpbl"><TD>Enzyme (KpnI HF)</TD><TD>1</TD></TR><TR class="tableunevenadpbl"><TD>MilliQ</TD><TD> 0</TD></TR></TBODY></TABLE></CENTER><P>Second reaction (dxCas9 into pACYC lin-Tn5):</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL)</TH></TR><TR class="tableunevenadpbl"><TD>10x CutSmart buffer    </TD><TD>2</TD></TR><TR class="tableunevenadpbl"><TD>Fragment (dxCas9)</TD><TD>16</TD></TR><TR class="tableunevenadpbl"><TD>Enzyme (NotI HF)</TD><TD>1</TD></TR><TR class="tableunevenadpbl"><TD>Enzyme (FseI HF)</TD><TD>1</TD></TR><TR class="tableunevenadpbl"><TD>MilliQ</TD><TD> 0</TD></TR></TBODY></TABLE></CENTER><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL)</TH></TR><TR class="tableunevenadpbl"><TD>10x CutSmart buffer    </TD><TD>2</TD></TR><TR class="tableunevenadpbl"><TD>Vector (pACYC lin-Tn5)</TD><TD>16</TD></TR><TR class="tableunevenadpbl"><TD>Enzyme (NotI HF)</TD><TD>1</TD></TR><TR class="tableunevenadpbl"><TD>Enzyme (FseI HF)</TD><TD>1</TD></TR><TR class="tableunevenadpbl"><TD>MilliQ</TD><TD> 0</TD></TR></TBODY></TABLE></CENTER><P>The samples were incubated for 4 hours at 37 °C. 

</P><DL><DT>Date</DT><DD>17/07/2018</DD><DT>Operator</DT><DD>Timothy Páez</DD></DL><H4>PCR clean-up</H4><P>
    20 μL of restriction product were subjected to PCR Clean-Up, according to the protocol. Elution was done in 20 µL Milli-Q. 
    Products were quantified with Nanodrop, yielding concentration of:
    - pACYC dxCas9: 42.7 ng/µL 
    - lin-Tn5: 34 ng/µL 
    - pACYC lin-Tn5: 31 ng/µL 
    - dxCas9: 46 ng/µL </P><DL><DT>Date</DT><DD>17/07/2018</DD><DT>Operator</DT><DD>Timothy Páez</DD></DL><H4>Ligation</H4><P>In order to assemble cut fragments lin-Tn5 into pACYC dxCas9 (Reaction 1) and dxCas9 into pACYC lin-Tn5, ligation was performed. This table gives an overview of which fragments were cut with which enzymes.

</P><P>First reaction:</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">DNA</TH><TH class="tableheaderadpbl">Size (bp) </TH><TH class="tableheaderadpbl">Cut with enzymes</TH></TR><TR class="tableunevenadpbl"><TD>pACYCDuet dxCas9</TD><TD>5.5 kbp</TD><TD>Fse and KpnI</TD></TR><TR class="tableevenadpbl"><TD>lin-Tn5</TD><TD>1.5 kbp</TD><TD>Fse and KpnI</TD></TR></TBODY></TABLE></CENTER><P> Two different molar ratios between insert and plasmid were tried to attempt higher integration efficiency. Ratios of 1:3  and 1:10 (vector:plasmid) were incubated with T4 ligase (1 µL) and ligase buffer (2 µL). Total volume of reaction was 20 µL.
    </P><P>Second reaction:</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">DNA</TH><TH class="tableheaderadpbl">Size (bp) </TH><TH class="tableheaderadpbl">Cut with enzymes</TH></TR><TR class="tableunevenadpbl"><TD>pACYCDuet lin Tn5</TD><TD>3 kbp</TD><TD>NotI and FseI</TD></TR><TR class="tableevenadpbl"><TD>dxCas9</TD><TD>4 kbp</TD><TD>NotI and FseI</TD></TR></TBODY></TABLE></CENTER><P> Two different molar ratios between insert and plasmid were tried to attempt higher integration efficiency. Ratios of 1:3  and 1:10 (vector:plasmid) were incubated with T4 ligase (1 µL) and ligase buffer (2 µL). Total volume of reaction was 20 µL.
    </P><P>The samples were incubated overnight at 4 ºC for ligation. </P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#WednesdaySCJ3">Wednesday, 18/07/2018</BUTTON><DIV id="WednesdaySCJ3" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>18/7/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4> Transformation Results </H4><P> After carrying out the transformation protocol on 17/7/2018, the plates were left for overnight incubation. After ~14-18hr incubation, the following results were observed: </P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Strain</TH><TH class="tableheaderadpbl">DNA </TH><TH class="tableheaderadpbl">Medium</TH><TH class="tableheaderadpbl">Result</TH></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I> DH5α  </TD><TD>L1  </TD><TD>LB+Cam  </TD><TD>2 colonies</TD></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I> DH5α  </TD><TD>L2  </TD><TD>LB+Cam  </TD><TD>30 colonies</TD></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I> DH5α  </TD><TD>L3  </TD><TD>LB+Cam  </TD><TD>2 colonies</TD></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I> DH5α  </TD><TD>L4  </TD><TD>LB+Cam  </TD><TD>4 colonies</TD></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I> DH5α  </TD><TD>L5  </TD><TD>LB+Cam  </TD><TD>19 colonies</TD></TR></TBODY></TABLE></CENTER><DL><DT>Date</DT><DD>18/7/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>Colony PCR</H4><P>Colony PCR was done to verify propagation of ligate constructs after transformation (17/7/2018). This plasmid contains T7 expression system, which can be checked with primers 029 and 030.
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">L1</TH><TH class="tableheaderadpbl">L2</TH><TH class="tableheaderadpbl">L3</TH><TH class="tableheaderadpbl">L4</TH><TH class="tableheaderadpbl">L5</TH></TR><TR class="tableunevenadpbl"><TD>Fw primer</TD><TD>029</TD><TD>029</TD><TD>029</TD><TD>029</TD><TD>029</TD></TR><TR class="tableevenadpbl"><TD>Rv primer</TD><TD>030</TD><TD>030</TD><TD>030</TD><TD>030</TD><TD>030</TD></TR><TR class="tableunevenadpbl"><TD>Expected amplicon size (kbp)</TD><TD>~5.7</TD><TD>~5.7</TD><TD>~5.7</TD><TD>~5.7</TD><TD>~5.7</TD></TR></TBODY></TABLE></CENTER><P>48 tubes were prepared for colony PCR, containing boiled colony as template DNA. The thermocycler programme was installed to have an extension time of 6.5 minutes and an annealing temperature of 60 °C. After the programme was finished, samples were kept at 4°C.</P><DL><DT>Date</DT><DD>18/7/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>DNA Gel Electrophoresis</H4><P>Colony PCR samples were run on a 0.8% agarose gel in TAE buffer. 5μL was pipetted into each lane. The gel was imaged using a gel documentation system with UV-light. </P><P>    It is apparent that no amplicons of interest are found, other than template/uncut/self-closing plasmids (lane 3 &amp; 25). Judging from the double bands around 200 and 500, most of the clones are identified to contain plain pACYC-DUET, which contains double T7 promoter, resulting in double amplicons around that height.</P><DL><DT>Date</DT><DD>18/07/2018</DD><DT>Operator</DT><DD>Timothy Páez</DD></DL><H4> Transformation of chemically competent cells </H4><P>Using the ligate reactions from 17/07/2018, the following transformations were carried out according to the protocol for transformation of chemically competent cells:
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Strain</TH><TH class="tableheaderadpbl">Plasmid &amp; size (bp) </TH><TH class="tableheaderadpbl">Cloning method</TH><TH class="tableheaderadpbl">Volume added (uL)</TH><TH class="tableheaderadpbl">Medium on which was plated</TH></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I>DH5αlpha</TD><TD>pACYC dxCas9-lin-Tn5 (7500 bp)</TD><TD>Restriction Ligation</TD><TD>10 µL</TD><TD>LB with Chloroamphenicol</TD></TR></TBODY></TABLE></CENTER><P>400 µL LB medium was added as recovery medium.
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#ThursdaySCJ3">Thursday, 19/07/2018</BUTTON><DIV id="ThursdaySCJ3" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>19/07/2018</DD><DT>Operator</DT><DD>Timothy Páez</DD></DL><H4>Colony PCR</H4><P>Colony PCR was done to verify propagation of <I>Escherichia coli</I> DH5 alpha cells after transformation with pACYC dxCas9 containing lin-Tn5 (Reaction 1) or pACYC linTn5 containing dxCas9 (Reaction 2) as insert. This insertion can be checked with primers 029 and 030 (on T7 promoter and terminator), but were also tested with primers 001 with 002 for dxCas9 amplification and 005 with 006 for Tn5 amplification. Primer sequences can be found at: <A href="https://2018.igem.org/Team:TUDelft/Primers" target="_blank"> Primer sequences</A>.
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH></TR><TR class="tableunevenadpbl"><TD>Template DNA</TD><TD>Colony solution</TD></TR><TR class="tableevenadpbl"><TD>Forward primer</TD><TD>029, 001, 005</TD></TR><TR class="tableunevenadpbl"><TD>Reverse primer</TD><TD>030, 002, 006</TD></TR><TR class="tableunevenadpbl"><TD>Expected amplicon size (bp)</TD><TD>6 kbp, 4 kbp, 1.5 kbp</TD></TR></TBODY></TABLE></CENTER><P> The number of colonies screened were:
    - Tn5 control : Colony 1 
    - Cas9-lin-Tn5 (Reaction 1): Colonies 2-10 
    - Cas9-lin-Tn5 (Reaction 2): Colonies 11-14
The thermocycler programme was installed to have an extension time of 6 min and an annealing temperature of 55 °C. After the programme was finished, samples were kept at 4°C.</P><DL><DT>Date</DT><DD>19/07/2018</DD><DT>Operator</DT><DD>Timothy Páez</DD></DL><H4>DNA Gel Electrophoresis</H4><P>Colony PCR products were run on a 0.8 % agarose gel in TAE buffer. 
    - 5 μL of PCR product was pipetted into each lane (no loading buffer required as PCR buffer contains loading buffer).
    - 4 µL of 500 bp DNA ladder 
    - 80 V for 30 minutes 
    - The gel was imaged using a gel documentation system with UV-light.</P><FIGURE><FIGCAPTION class="adpbl">Figure 1. Gel electrophoresis from colony PCR for dxCas9-lin-Tn5 amplification. Expected size of integration is 6000 bp. Lanes are (L) ladder 500 bp, (1-14) Colonies 1-14. </FIGCAPTION></FIGURE><FIGURE><FIGCAPTION class="adpbl">Figure 2. Gel electrophoresis from colony PCR for dxCas9 amplification. Expected size of amplification is 4000 bp. Lanes are (L) ladder 500 bp, (1-14) Colonies 1-14. </FIGCAPTION></FIGURE><FIGURE><FIGCAPTION class="adpbl">Figure 3. Gel electrophoresis from colony PCR for Tn5 amplification. Expected size of integration is 1500 bp. Lanes are (L) ladder 500 bp, (1-14) Colonies 1-14. </FIGCAPTION></FIGURE></DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV></UL></FORM><H3 class="adpbl">August</H3><FORM><LABEL for="WeekSCA1" class="subdropdown cadpbl">Week 1 30/07/2018 - 05/08/2018</LABEL><UL class="submenu"><DIV class="subcontent"><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#FridaySCA1">Friday, 03/07/2018</BUTTON><DIV id="FridaySCA1" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>03/08/2018</DD><DT>Operator</DT><DD>Timothy Páez</DD></DL><H4>Liquid Starter Culture Preparation</H4><P>
Strains <I>Escherichia coli DH5 alpha</I> harbouring plasmid with possible fusion protein dxCas9-lin-Tn5 were used for inoculation of 10 mL liquid starter culture (Luria Broth, complemented with chloroamphenicol) following the Liquid Starter Culture Protocol.
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#SaturdaySCA1">Saturday, 04/07/2018</BUTTON><DIV id="SaturdaySCA1" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>04/08/2018</DD><DT>Operator</DT><DD>Timothy Páez</DD></DL><H4>Plasmid Isolation</H4><P> 
    Overnight starter cultures of <I>Escherichia coli DH5 alpha</I> harbouring plasmid with possible fusion protein dxCas9-lin-Tn5 were subjected to plasmid isolation following the Plasmid Isolation Protocol. Final elution was done with 30 μL of pre-warmed MilliQ.
</P><DL><DT>Date</DT><DD>04/08/2018</DD><DT>Operator</DT><DD>Timothy Páez</DD></DL><H4>High Fidelity PCR</H4><P> To amplify dxCas9-lin-Tn5 (6 kbp), high fidelity PCR was done using colonies' plasmid harbouring the fusion protein as template DNA and 029 and 030 as forward and reverse primers respectively. Primer sequences can be found at: <A href="https://2018.igem.org/Team:TUDelft/Primers" target="_blank"> Primer sequences</A>. 
    50 µL PCR mixtures were prepared, according to the High Fidelity PCR protocol.
    
    
    </P><P>The thermocycler programme was installed to have an extension time of 6 minutes and an annealing temperature of 55 °C. After the programme was finished, samples were kept at 4 °C.</P><DL><DT>Date</DT><DD>03/07/2018</DD><DT>Operator</DT><DD>Timothy Páez</DD></DL><H4>DNA Gel Electrophoresis</H4><P>PCR products of dxCas9 PCR were run on a 0.8 % agarose gel in TAE buffer. 
    - 5 μL of PCR product was pipetted into each lane with 1 µL of loading buffer.
    - 4 µL of 500 bp DNA ladder 
    - 80 V for 30 minutes 
    - The gel was imaged using a gel documentation system with UV-light.</P><FIGURE><FIGCAPTION class="adpbl">Figure 1. Gel electrophoresis from fusion protein dxCas9-lin-Tn5. Lanes are (L) Ladder 500 bp, (1) Blank, (2-3) Positive integration. </FIGCAPTION></FIGURE><DL><DT>Date</DT><DD>04/08/2018</DD><DT>Operator</DT><DD>Timothy Páez</DD></DL><H4>Sequence Verification </H4><H5>Sequence from plasmid</H5><P> For verification of dxCas9-Tn5-lin insertion in pACYC, primers 003, 012, 029 and 030 were used for sequencing. Sequencing sample was prepared in 1.5 mL tubes for each primer as indicated my Macrogen for specific concentration of plasmid DNA.
    
    Primer sequences can be found at: <A href="https://2018.igem.org/Team:TUDelft/Primers" target="_blank"> Primer sequences</A>. </P><P> Within 2 days, the sequence was retrievable online and alignment with the in silico designed molecule showed 100% match of the insert lin-Tn5, by aligning both primer sequences obtained. </P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV></DIV></UL></FORM><FORM><LABEL for="WeekSCA2" class="subdropdown cadpbl">Week 2 06/08/2018 - 12/08/2018</LABEL><UL class="submenu"><DIV class="subcontent"><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#WednesdaySCA2">Wednesday, 08/08/2018</BUTTON><DIV id="WednesdaySCA2" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>08/08/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">Liquid Starter Culture Preparation</H4><P><I>E.coli</I> strain harbouring the plasmid pACYCDuet-1_Tn5-Lin was used for inoculation of 10 mL liquid starter culture (Luria Broth, complemented with chloramphenicol) following the <A href="https://2018.igem.org/Team:TUDelft/Experiments#liquidstarterculture-scroll" class="adpbl" target="_blank">Liquid Starter Culture Protocol</A>.
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#ThursdaySCA2">Thursday, 09/08/2018</BUTTON><DIV id="ThursdaySCA2" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>09/08/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">Plasmid Isolation</H4><P>
Overnight starter culture of pACYCDuet-1_Tn5-Lin were subjected to plasmid isolation following the <A href="https://2018.igem.org/Team:TUDelft/Experiments#plasmidisolation-scroll" class="adpbl" target="_blank">Plasmid Isolation Protocol</A>. Final elution was done with 30 μL of pre-warmed MilliQ.
</P><DL><DT>Date</DT><DD>09/08/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">Nanodrop DNA Quantification</H4><P>The following isolates were quantified according to the <A href="https://2018.igem.org/Team:TUDelft/Experiments#NanodropDNA-scroll" class="adpbl" target="_blank">Nanodrop DNA Quantification protocol</A>. The concentration and purity of the DNA (260/280 and 260/230 ratios) were measured. </P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Isolate</TH><TH class="tableheaderadpbl">Concentration (ng/uL)</TH><TH class="tableheaderadpbl">260/280 ratio</TH><TH class="tableheaderadpbl">260/230 ratio</TH></TR><TR class="tableunevenadpbl"><TD>pACYCDuet-1_Tn5-Lin</TD><TD>66.5</TD><TD>1.88</TD><TD>1.83</TD></TR><TR class="tableevenadpbl"><TD>pACYCDuet-1_Tn5-Lin</TD><TD>70.1</TD><TD>1.83</TD><TD>1.77</TD></TR><TR class="tableunevenadpbl"><TD>pACYCDuet-1_Tn5-Lin</TD><TD>90.2</TD><TD>1.72</TD><TD>1.06</TD></TR><TR class="tableevenadpbl"><TD>pACYCDuet-1_Tn5-Lin</TD><TD>108.8</TD><TD>1.73</TD><TD>1.09</TD></TR></TBODY></TABLE></CENTER><DL><DT>Date</DT><DD>09/08/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">Restriction</H4><P>The following table shows an overview of what fragment(s) were cut with what restriction enzyme(s) to obtain fragments Tn5-Lin-dxCas9 and pACYCDuet-1 for ligation.
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL)</TH><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL)</TH></TR><TR class="tableunevenadpbl"><TD>10x CutSmart</TD><TD>2</TD><TD>10x CutSmart</TD><TD>2</TD></TR><TR class="tableevenadpbl"><TD>dxCas9</TD><TD>8.7 (~2000 ng)</TD><TD>pACYCDuet-1_Tn5-Linker</TD><TD>17.2 (~2000 ng)</TD></TR><TR class="tableunevenadpbl"><TD>Enzyme (NotI)</TD><TD>0.4</TD><TD>Enzyme (NotI)</TD><TD>0.4</TD></TR><TR class="tableevenadpbl"><TD>Enzyme (KpnI)</TD><TD>0.4</TD><TD>Enzyme (KpnI)</TD><TD>0.4</TD></TR><TR class="tableunevenadpbl"><TD>MilliQ</TD><TD>8.5</TD><TD>MilliQ</TD><TD>0</TD></TR></TBODY></TABLE></CENTER><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL)</TH><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL)</TH></TR><TR class="tableunevenadpbl"><TD>10x CutSmart</TD><TD>2</TD><TD>10x CutSmart</TD><TD>2</TD></TR><TR class="tableevenadpbl"><TD>Tn5-Lin</TD><TD>12.5 (~2000 ng)</TD><TD>pACYCDuet-1_dxCas9</TD><TD>17.2 (~2000 ng)</TD></TR><TR class="tableunevenadpbl"><TD>Enzyme (NotI)</TD><TD>0.4</TD><TD>Enzyme (NotI)</TD><TD>0.4</TD></TR><TR class="tableunevenadpbl"><TD>Enzyme (EcoRI)</TD><TD>0.4</TD><TD>Enzyme (EcoRI)</TD><TD>0.4</TD></TR><TR class="tableevenadpbl"><TD>MilliQ</TD><TD>4.7</TD><TD>MilliQ</TD><TD>0</TD></TR></TBODY></TABLE></CENTER><P>The samples were incubated for 4 hour(s) at 37 °C, and followed by 20 minute heat inactivation of the restriction enzyme(s). 
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#FridaySCA2">Friday, 10/08/2018</BUTTON><DIV id="FridaySCA2" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>10/08/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">PCR Clean-Up</H4><P>20 μL of each restriction reaction mixture was subjected to PCR Clean-Up, according to the <A href="https://2018.igem.org/Team:TUDelft/Experiments#PCRcleanup-scroll" class="adpbl" target="_blank">protocol</A>. Elution was done in 30 µL Milli-Q.</P><DL><DT>Date</DT><DD>10/08/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">Nanodrop DNA Quantification</H4><P>The following isolates were quantified according to the <A href="https://2018.igem.org/Team:TUDelft/Experiments#NanodropDNA-scroll" class="adpbl" target="_blank">Nanodrop DNA Quantification protocol</A>. The concentration and purity of the DNA (260/280 and 260/230 ratios) were measured. </P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Isolate</TH><TH class="tableheaderadpbl">Concentration (ng/uL)</TH><TH class="tableheaderadpbl">260/280 ratio</TH><TH class="tableheaderadpbl">260/230 ratio</TH></TR><TR class="tableunevenadpbl"><TD>Tn5-Lin cut with NotI and EcoRI</TD><TD>88.1</TD><TD>1.83</TD><TD>1.69</TD></TR><TR class="tableevenadpbl"><TD>pACYCDuet-1_dxCas9 cut with NotI and EcoRI</TD><TD>11.8</TD><TD>1.44</TD><TD>0.69</TD></TR><TR class="tableunevenadpbl"><TD>dxCas9 cut with NotI and EcoRI</TD><TD>34.5</TD><TD>1.76</TD><TD>1.31</TD></TR><TR class="tableevenadpbl"><TD>pACYC_Tn5-Linker cut with NotI and EcoRI</TD><TD>41.9</TD><TD>1.73</TD><TD>1.36</TD></TR></TBODY></TABLE></CENTER><DL><DT>Date</DT><DD>10/08/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">Ligation</H4><P>In order to assemble cut fragments pACYCDuet-1_Tn5-Linker with dxCas9 Tn-Linker with pACYCDuet-1_dxCas9 respectively, ligation was performed. to optimize the protocol, I used both vector to insert ratio 1:3 and 1:10. This table gives an overview of which fragments were cut with which enzymes.</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">DNA</TH><TH class="tableheaderadpbl">Size (bp) </TH><TH class="tableheaderadpbl">Cut with enzymes</TH></TR><TR class="tableunevenadpbl"><TD>Vector pACYCDuet-1_dxCas9</TD><TD>7929</TD><TD>EcoRI and NotI</TD></TR><TR class="tableevenadpbl"><TD>Fragment Tn5-Linker</TD><TD>1490</TD><TD>EcoRI and NotI</TD></TR></TBODY></TABLE></CENTER><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">DNA</TH><TH class="tableheaderadpbl">Size (bp) </TH><TH class="tableheaderadpbl">Cut with enzymes</TH></TR><TR class="tableunevenadpbl"><TD>Vector pACYCDuet-1_Tn5-Lin</TD><TD>5258</TD><TD>NotI and KpnI</TD></TR><TR class="tableevenadpbl"><TD>Fragment dxCas9</TD><TD>4161</TD><TD>NotI and KpnI</TD></TR></TBODY></TABLE></CENTER><P>A 20 μL ligation reaction mixture was prepared with the following composition:</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (μL)</TH><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (μL)</TH><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (μL)</TH></TR><TR class="tableunevenadpbl"><TD>Ligase Buffer (10X)</TD><TD>2</TD><TD>Ligase Buffer (10X)</TD><TD>2</TD><TD>Ligase Buffer (10X)</TD><TD>2</TD></TR><TR class="tableevenadpbl"><TD>T4 DNA Ligase </TD><TD>1</TD><TD>T4 DNA Ligase </TD><TD>1</TD><TD>T4 DNA Ligase </TD><TD>1</TD></TR><TR class="tableunevenadpbl"><TD>DNA vector pACYCDuet-1_dxCas9 (~100 ng)</TD><TD>8.5</TD><TD>DNA vector pACYCDuet-1_dxCas9 (~100 ng)</TD><TD>8.5</TD><TD>DNA vector pACYCDuet-1_dxCas9 (~100 ng)</TD><TD>8.5</TD></TR><TR class="tableevenadpbl"><TD>none</TD><TD>DNA Lin-Tn5 (~60 ng)</TD><TD>0.7</TD><TD>DNA Lin-Tn5 (~190 ng)</TD><TD>2.2</TD></TR><TR class="tableunevenadpbl"><TD>MilliQ</TD><TD>12</TD><TD>MilliQ</TD><TD>11.3</TD><TD>MilliQ</TD><TD>9.8</TD></TR></TBODY></TABLE></CENTER><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (μL)</TH><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (μL)</TH><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (μL)</TH></TR><TR class="tableunevenadpbl"><TD>Ligase Buffer (10X)</TD><TD>2</TD><TD>Ligase Buffer (10X)</TD><TD>2</TD><TD>Ligase Buffer (10X)</TD><TD>2</TD></TR><TR class="tableevenadpbl"><TD>T4 DNA Ligase </TD><TD>1</TD><TD>T4 DNA Ligase </TD><TD>1</TD><TD>T4 DNA Ligase </TD><TD>1</TD></TR><TR class="tableunevenadpbl"><TD>DNA vector pACYCDuet-1_Tn5-Linker (~100 ng)</TD><TD>2.5</TD><TD>DNA vector pACYCDuet-1_Tn5-Linker (~100 ng)</TD><TD>2.5</TD><TD>DNA vector pACYCDuet-1_Tn5-Linker (~100 ng)</TD><TD>2.5</TD></TR><TR class="tableevenadpbl"><TD>none</TD><TD>DNA dxCas9 (~220 ng)</TD><TD>5.4</TD><TD>DNA dxCas9 (~730 ng)</TD><TD>14.5</TD></TR><TR class="tableunevenadpbl"><TD>MilliQ</TD><TD>14.5</TD><TD>MilliQ</TD><TD>7.1</TD><TD>MilliQ</TD><TD>0</TD></TR></TBODY></TABLE></CENTER><P>The samples were incubated for 8 hours at 20°C for ligation.</P><DL><DT>Date</DT><DD>10/08/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">Transformation of chemically competent cells</H4><P>The following transformations were carried out according to the protocol for transformation of chemically competent cells:</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Strain</TH><TH class="tableheaderadpbl">Plasmid and size (bp)</TH><TH class="tableheaderadpbl">Cloning method</TH><TH class="tableheaderadpbl">Volume added (μl)</TH><TH class="tableheaderadpbl">Medium on which was plated</TH></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I> DH5α</TD><TD>pACYCDuet-1_dxCas9 restricted (neg control)</TD><TD>Restriction/Ligation</TD><TD>8</TD><TD>LB-Cam</TD></TR><TR class="tableevenadpbl"><TD><I>E.coli</I> DH5α</TD><TD>pACYCDuet-1_dxCas9-Linker-Tn5 ratio 1:3 (9449 bp)</TD><TD>Restriction/Ligation</TD><TD>5.5</TD><TD>LB Cam</TD></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I> DH5α</TD><TD>pACYCDuet-1_dxCas9-Linker-Tn5 ratio 1:10 (9449 bp)</TD><TD>Restriction/Ligation</TD><TD>3</TD><TD>LB Cam</TD></TR><TR class="tableevenadpbl"><TD><I>E.coli</I> DH5α</TD><TD>pACYCDuet-1_Tn5-Lin restricted (neg. control)</TD><TD>Restriction</TD><TD>8</TD><TD>LB Cam</TD></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I> DH5α</TD><TD>pACYCDuet-1_dxCas9-Linker-Tn5 ratio 1:3 (9449 bp)</TD><TD>Restriction/Ligation</TD><TD>3</TD><TD>LB Cam</TD></TR><TR class="tableevenadpbl"><TD><I>E.coli</I> DH5α</TD><TD>pACYCDuet-1_dxCas9-Linker-Tn5 ratio 1:10 (9449 bp)</TD><TD>Restriction/Ligation</TD><TD>1</TD><TD>LB Cam</TD></TR></TBODY></TABLE></CENTER><P>
For the transformation 40 ng of total ligate was used. 200 μl LB medium was added as recovery medium.</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#SaturdaySCA2">Saturday, 11/08/2018</BUTTON><DIV id="SaturdaySCA2" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>11/8/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>Plasmid Isolation</H4><P> 
    Overnight DH5alpha starter cultures of dxCas9(3.7)-VPR and pACYC-dxCas9-LinkerTn5 were subjected to plasmid isolation following the Plasmid Isolation Protocol. Final elution was done with 40 μl of pre-warmed MilliQ (2 times 20µl).
</P><DL><DT>Date</DT><DD>11/8/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>Nanodrop DNA Quantification</H4><P> 
    The following isolates were quantified according to the Nanodrop DNA Quantification protocol. The concentration and purity of the DNA (260/280 and 260/230 ratios) were measured. 
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Isolate</TH><TH class="tableheaderadpbl">Concentration (ng/uL)</TH><TH class="tableheaderadpbl">260/280 ratio</TH><TH class="tableheaderadpbl">260/230 ratio</TH></TR><TR class="tableunevenadpbl"><TD>dxCas9(3.7)-VPR</TD><TD>23</TD><TD>1.82</TD><TD>1.83</TD></TR><TR class="tableevenadpbl"><TD>dxCas9(3.7)-VPR</TD><TD>28</TD><TD>1.82</TD><TD>1.9</TD></TR><TR class="tableunevenadpbl"><TD>dxCas9(3.7)-VPR</TD><TD>24</TD><TD>1.83</TD><TD>1.44</TD></TR><TR class="tableevenadpbl"><TD>dxCas9(3.7)-VPR</TD><TD>26</TD><TD>1.81</TD><TD>1.97</TD></TR><TR class="tableunevenadpbl"><TD>dxCas9(3.7)-VPR</TD><TD>13</TD><TD>1.59</TD><TD>0.97</TD></TR><TR class="tableevenadpbl"><TD>dxCas9(3.7)-VPR</TD><TD>23</TD><TD>1.88</TD><TD>1.99</TD></TR><TR class="tableunevenadpbl"><TD>dxCas9(3.7)-VPR</TD><TD>26</TD><TD>1.83</TD><TD>1.59</TD></TR><TR class="tableevenadpbl"><TD>pACYC-dxCas9-LinkerTn5</TD><TD>30</TD><TD>1.83</TD><TD>1.85</TD></TR><TR class="tableunevenadpbl"><TD>pACYC-dxCas9-LinkerTn5</TD><TD>31</TD><TD>1.81</TD><TD>1.88</TD></TR><TR class="tableevenadpbl"><TD>pACYC-dxCas9-LinkerTn5</TD><TD>31</TD><TD>1.78</TD><TD>1.63</TD></TR><TR class="tableunevenadpbl"><TD>pACYC-dxCas9-LinkerTn5</TD><TD>31</TD><TD>1.72</TD><TD>1.69</TD></TR><TR class="tableevenadpbl"><TD>pACYC-dxCas9-LinkerTn5</TD><TD>29</TD><TD>1.85</TD><TD>1.79</TD></TR><TR class="tableunevenadpbl"><TD>pACYC-dxCas9-LinkerTn5</TD><TD>27</TD><TD>1.74</TD><TD>1.71</TD></TR><TR class="tableevenadpbl"><TD>pACYC-dxCas9-LinkerTn5</TD><TD>28</TD><TD>1.81</TD><TD>1.74</TD></TR></TBODY></TABLE></CENTER></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV></DIV></UL></FORM><FORM><LABEL for="WeekSCA3" class="subdropdown cadpbl">Week 3 13/08/2018 - 19/08/2018</LABEL><UL class="submenu"><DIV class="subcontent"><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#SundaySCA3">Sunday, 19/08/2018</BUTTON><DIV id="SundaySCA3" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>19/08/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">High Fidelity PCR</H4><P>To amplify the pSB1C3 BioBrick backbone, Tn5 and Linker coding sequences and T7p-Tn5-T7t expression cassette, high fidelity PCR was done. The template DNA is the BioBrick plasmid pSB1C3 (primerset 051 &amp; 052 &amp; 053) and pACYCDuet-1_Tn5 (054 &amp; 055, 058 &amp; 059 and 060 &amp; 061). Eight 50µL PCR mixtures were prepared, according to the <A href="https://2018.igem.org/Team:TUDelft/Experiments#highfidelityPCR-scroll" target="_blank" class="adpbl">High Fidelity PCR protocol</A>. 
Primer combination(s) and expected fragment(s)
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Sample</TH><TH class="tableheaderadpbl">Fw primer</TH><TH class="tableheaderadpbl">Rv primer</TH><TH class="tableheaderadpbl">Expected fragment</TH><TH class="tableheaderadpbl">Expected size (bp) </TH></TR><TR class="tableunevenadpbl"><TD>1</TD><TD>051</TD><TD>052</TD><TD>pSB1C3 backbone</TD><TD>2068</TD></TR><TR class="tableevenadpbl"><TD>2</TD><TD>051</TD><TD>053</TD><TD>pSB1C3 backbone with extra nucleotide</TD><TD>2070</TD></TR><TR class="tableunevenadpbl"><TD>3</TD><TD>054</TD><TD>055</TD><TD>Tn5</TD><TD>1511</TD></TR><TR class="tableevenadpbl"><TD>4</TD><TD>058</TD><TD>059</TD><TD>Linker</TD><TD>143</TD></TR><TR class="tableevenadpbl"><TD>5</TD><TD>060</TD><TD>061</TD><TD>T7p-Tn5-T7t</TD><TD>1756</TD></TR></TBODY></TABLE></CENTER><P>The thermocycler programme was installed to have an extension time of 1 min 30 seconds for pSB1C3 backbones, Tn5 and T7p-Tn5-T7t and 23 seconds for Linker and both an annealing temperature of 60 °C. After the programme was finished, samples were kept at 4 °C.
</P><DL><DT>Date</DT><DD>19/08/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4>DNA Gel Electrophoresis</H4><P>The PCR amplicons for construction of the BioBricks Tn5, T7p-Tn5-T7t and Linker into pSB13 were run on a 0.8% agarose gel in TAE buffer. 4 μL was pipetted into each lane. The gel was imaged using a gel documentation system with UV-light.</P><P>The gel image shows that the pSB1C3 BioBrick backbone is succesfully amplified. However, when the extra nucleotide is incorporated for non coding sequence BioBrick construction, this resulted not in the desired amplicon. The Tn5 was correctly amplified as well as the T7p-Tn5-T7t expression cassette. The Linker sequence is not correctly amplified as there is no band seen at 143 bp.</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV></DIV></UL></FORM><FORM><LABEL for="WeekSCA4" class="subdropdown cadpbl">Week 4 20/08/2018 - 26/08/2018</LABEL><UL class="submenu"><DIV class="subcontent"><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#TuesdaySCA4">Tuesday, 21/08/2018</BUTTON><DIV id="TuesdaySCA4" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>21/08/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">DpnI digestion</H4><P>To eliminate any residual template DNA, a DpnI digestion was performed with the following reaction mixture:
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL)</TH></TR><TR class="tableunevenadpbl"><TD>10X CutSmart buffer</TD><TD>5</TD></TR><TR class="tableevenadpbl"><TD>PCR product</TD><TD>44</TD></TR><TR class="tableunevenadpbl"><TD>Enzyme (DpnI)</TD><TD>1</TD></TR><TR class="tableunevenadpbl"><TD>MilliQ</TD><TD>0</TD></TR></TBODY></TABLE></CENTER><P>The reaction was incubated for 30 minutes at 37 °C. 
Heat inactivation was performed for 20 minutes at 80 °C.</P></DIV><DL><DT>Date</DT><DD>21/08/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">PCR Clean-Up</H4><P>50 μL of each DpnI digestion reaction mixture was subjected to PCR Clean-Up, according to the <A href="https://2018.igem.org/Team:TUDelft/Experiments#PCRcleanup-scroll" class="adpbl" target="_blank">protocol</A>. Elution was done in 30 µL Milli-Q.</P><DL><DT>Date</DT><DD>21/09/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4>DNA Gel Electrophoresis</H4><P>The PCR amplicons for construction of the BioBricks Tn5, T7p-Tn5-T7t and Linker into pSB13 were run on a 0.8% agarose gel in TAE buffer. Upon electroseparation based on size, the fragments that are undesired were separated from the desired amplicons. The desired fragments were excised and proceded for PCR Clean Up. The complete reaction mixture (50 μL) was pipetted into each lane. The gel was not imaged due to damage of UV-light.</P><DL><DT>Date</DT><DD>21/08/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">High Fidelity PCR</H4><P>To amplify the pSB1C3 BioBrick backboneand Linker coding sequence, high fidelity PCR was done. The template DNA are the BioBrick plasmid pSB1C3 (primerset 051 &amp; 053) and pACYCDuet-1_Linker-Tn5 (058 &amp; 059). Four 50µL PCR mixtures were prepared, according to the <A href="https://2018.igem.org/Team:TUDelft/Experiments#highfidelityPCR-scroll" target="_blank" class="adpbl">High Fidelity PCR protocol</A>. 
Primer combination(s) and expected fragment(s)
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Sample</TH><TH class="tableheaderadpbl">Fw primer</TH><TH class="tableheaderadpbl">Rv primer</TH><TH class="tableheaderadpbl">Expected fragment</TH><TH class="tableheaderadpbl">Expected size (bp) </TH></TR><TR class="tableunevenadpbl"><TD>1</TD><TD>051</TD><TD>053</TD><TD>pSB1C3 backbone with extra nucleotide</TD><TD>2070</TD></TR><TR class="tableevenadpbl"><TD>4</TD><TD>058</TD><TD>059</TD><TD>Linker</TD><TD>143</TD></TR></TBODY></TABLE></CENTER><P>The thermocycler programme was installed to have an extension time of 1 min 30 seconds for pSB1C3 backbones, Tn5 and T7p-Tn5-T7t and 23 seconds for Linker and both an annealing temperature of 60 °C. After the programme was finished, samples were kept at 4 °C.
</P><DL><DT>Date</DT><DD>22/09/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">DNA Gel Electrophoresis</H4><P>The PCR amplicons for construction of the BioBrick Linker into pSB13 were run on a 0.8% agarose gel in TAE buffer. 4 μL was pipetted into each lane. The gel was imaged using a gel documentation system with UV-light.</P><P>The gel image shows that the pSB1C3 BioBrick backbone is succesfully amplified. However, the Linker sequence is not correctly amplified as there is no band seen at 143 bp.</P><DL><DT>Date</DT><DD>21/08/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">Liquid Starter Culture Preparation</H4><P><I>E.coli</I> strains harbouring the plasmid pACYCDuet-1_dxCas9-Lin-Tn5 and pACYCDuet-1_Tn5 respectively were used for inoculation of 10 mL liquid starter culture (Luria Broth, complemented with chloramphenicol) following the <A href="https://2018.igem.org/Team:TUDelft/Experiments#liquidstarterculture-scroll" class="adpbl" target="_blank">Liquid Starter Culture Protocol</A>.
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#WednesdaySCA4">Wednesday, 22/08/2018</BUTTON><DIV id="WednesdaySCA4" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>22/08/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">Plasmid Isolation</H4><P>
Overnight starter cultures of pACYCDuet-1_Lin-Tn5 and pACYCDuet-1 were subjected to plasmid isolation following the <A href="https://2018.igem.org/Team:TUDelft/Experiments#plasmidisolation-scroll" class="adpbl" target="_blank">Plasmid Isolation Protocol</A>. Final elution was done with 30 μL of pre-warmed MilliQ.
</P><DL><DT>Date</DT><DD>22/08/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">PCR Clean-Up</H4><P>50 μL of each PCR reaction  done on 16-07-2018 was subjected to PCR Clean-Up, according to the <A href="https://2018.igem.org/Team:TUDelft/Experiments#PCRcleanup-scroll" class="adpbl" target="_blank">protocol</A>. Elution was done in 30 µL Milli-Q.</P><DL><DT>Date</DT><DD>22/08/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">Nanodrop DNA Quantification</H4><P>The following isolates were quantified according to the <A href="https://2018.igem.org/Team:TUDelft/Experiments#NanodropDNA-scroll" class="adpbl" target="_blank">Nanodrop DNA Quantification protocol</A>. The concentration and purity of the DNA (260/280 and 260/230 ratios) were measured. </P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Isolate</TH><TH class="tableheaderadpbl">Concentration (ng/uL)</TH><TH class="tableheaderadpbl">260/280 ratio</TH><TH class="tableheaderadpbl">260/230 ratio</TH></TR><TR class="tableunevenadpbl"><TD>T7p-Tn5-T7t</TD><TD>94.7</TD><TD>1.84</TD><TD>1.78</TD></TR><TR class="tableevenadpbl"><TD>T7p-Tn5-T7t</TD><TD>105.7</TD><TD>1.83</TD><TD>1.93</TD></TR><TR class="tableunevenadpbl"><TD>Tn5</TD><TD>157.1</TD><TD>1.66</TD><TD>1.26</TD></TR><TR class="tableevenadpbl"><TD>pSB1C3</TD><TD>58.8</TD><TD>1.87</TD><TD>1.71</TD></TR><TR class="tableunevenadpbl"><TD>pSB1C3</TD><TD>67.8</TD><TD>1.86</TD><TD>1.88</TD></TR><TR class="tableevenadpbl"><TD>pSB1C3</TD><TD>64.8</TD><TD>1.84</TD><TD>1.77</TD></TR></TBODY></TABLE></CENTER><DL><DT>Date</DT><DD>22/08/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">Gibson Assembly</H4><P>According to the displayed reaction mixture composition, Gibson Assembly was done to assemble pSB1C3 with Tn5 to obtain pSB1C3_Tn5 (3499 bp) and pSB1C3 with T7p-Tn5-T7t to obtain pSB1C3_T7p-Tn5-T7t (3755 bp).
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL)</TH><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL)</TH></TR><TR class="tableunevenadpbl"><TD>Gibson Assembly Mastermix 2X</TD><TD>10</TD><TD>Gibson Assembly Mastermix 2X</TD><TD>10</TD></TR><TR class="tableevenadpbl"><TD>Vector pSB1C3</TD><TD>2 (100 ng)</TD><TD>Vector pSB1C3</TD><TD>2 (100 ng)</TD></TR><TR class="tableunevenadpbl"><TD>Fragment T7p-Tn5-T7t</TD><TD>3 (300 ng)</TD><TD>Fragment Tn5</TD><TD>2 (225 ng)</TD></TR><TR class="tableunevenadpbl"><TD>MilliQ</TD><TD>5</TD><TD>MilliQ</TD><TD>4</TD></TR></TBODY></TABLE></CENTER><P>The samples were incubated at 50°C for 1 hour, after which they were kept on ice for subsequent transformation.</P></DIV><DL><DT>Date</DT><DD>22/08/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">Transformation of chemically competent cells</H4><P>The following transformations were carried out according to the protocol for transformation of chemically competent cells:</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Strain</TH><TH class="tableheaderadpbl">Plasmid and size (bp)</TH><TH class="tableheaderadpbl">Cloning method</TH><TH class="tableheaderadpbl">Volume added (μl)</TH><TH class="tableheaderadpbl">Medium on which was plated</TH></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I> DH5α</TD><TD>pSB1C3_Tn5 (3499 bp)</TD><TD>Gibson Assembly</TD><TD>2.5</TD><TD>LB-Cam</TD></TR><TR class="tableevenadpbl"><TD><I>E.coli</I> DH5α</TD><TD>pSB1C3_T7p-Tn5-T7t (3755 bp)</TD><TD>Gibson Assembly</TD><TD>2.5</TD><TD>LB Cam</TD></TR></TBODY></TABLE></CENTER><P>
200 μl LB medium was added as recovery medium.</P><DL><DT>Date</DT><DD>22/08/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">Nanodrop DNA Quantification</H4><P>The following isolates were quantified according to the <A href="https://2018.igem.org/Team:TUDelft/Experiments#NanodropDNA-scroll" class="adpbl" target="_blank">Nanodrop DNA Quantification protocol</A>. The concentration and purity of the DNA (260/280 and 260/230 ratios) were measured. </P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Isolate</TH><TH class="tableheaderadpbl">Concentration (ng/uL)</TH><TH class="tableheaderadpbl">260/280 ratio</TH><TH class="tableheaderadpbl">260/230 ratio</TH></TR><TR class="tableunevenadpbl"><TD>pACYCDuet-1_dxCas9-Lin-Tn5</TD><TD>68.9</TD><TD>1.69</TD><TD>1.03</TD></TR><TR class="tableevenadpbl"><TD>pACYCDuet-1_dxCas9-Lin-Tn5</TD><TD>74.2</TD><TD>1.68</TD><TD>1.06</TD></TR><TR class="tableunevenadpbl"><TD>pACYCDuet-1_dxCas9-Lin-Tn5</TD><TD>92.8</TD><TD>1.68</TD><TD>1.07</TD></TR><TR class="tableevenadpbl"><TD>pACYCDuet-1_dxCas9-Lin-Tn5</TD><TD>77.8</TD><TD>1.66</TD><TD>0.92</TD></TR><TR class="tableunevenadpbl"><TD>pACYCDuet-1_Tn5</TD><TD>57.0</TD><TD>1.72</TD><TD>1.37</TD></TR><TR class="tableevenadpbl"><TD>pACYCDuet-1_Tn5</TD><TD>52.4</TD><TD>1.75</TD><TD>1.42</TD></TR><TR class="tableunevenadpbl"><TD>pACYCDuet-1_Tn5</TD><TD>53.4</TD><TD>1.77</TD><TD>1.58</TD></TR><TR class="tableevenadpbl"><TD>pACYCDuet-1_Tn5</TD><TD>70.9</TD><TD>1.76</TD><TD>1.742</TD></TR></TBODY></TABLE></CENTER><DL><DT>Date</DT><DD>22/08/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">High Fidelity PCR</H4><P>The dxCas9 coding sequence contains forbidden restriction sites to be RFC compatible. Therefore, Gblocks were ordered at IDT. The dxCas9 is to large to order as one Gblock. Therefore I ordered two Gblocks, also incorporating Linker-Tn5 coding sequences (Gblock 1 and 2). To be able to construct also a different variant of the fusion protein, two Gblocks were ordered in the order Tn5-Linker-dxCas9 (Gblock 3 and 4).

To amplify the Gblocks 1 and 2 as well as the pACYCDuet-1 backbone, high fidelity PCR was done. The template DNA are the Gblocks 1 (primerset 045 &amp; 046) and 2 (primerset 047 &amp; 048) and pACYCDuet-1 (primerset 049 &amp; 050). Six 50µL PCR mixtures were prepared, according to the <A href="https://2018.igem.org/Team:TUDelft/Experiments#highfidelityPCR-scroll" target="_blank" class="adpbl">High Fidelity PCR protocol</A>. 
Primer combination(s) and expected fragment(s)
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Sample</TH><TH class="tableheaderadpbl">Fw primer</TH><TH class="tableheaderadpbl">Rv primer</TH><TH class="tableheaderadpbl">Expected fragment</TH><TH class="tableheaderadpbl">Expected size (bp) </TH></TR><TR class="tableunevenadpbl"><TD>1</TD><TD>045</TD><TD>046</TD><TD>Gblock 1</TD><TD>2892</TD></TR><TR class="tableevenadpbl"><TD>4</TD><TD>047</TD><TD>048</TD><TD>Gblock 2</TD><TD>2890</TD></TR><TR class="tableunevenadpbl"><TD>1</TD><TD>049</TD><TD>050</TD><TD>pACYCDuet-1</TD><TD>3778</TD></TR></TBODY></TABLE></CENTER><P>The thermocycler programme was installed to have an extension time of 2 min 30 seconds and an annealing temperature of 60 °C. After the programme was finished, samples were kept at 4 °C.
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#ThursdaySCA4">Thursday, 23/08/2018</BUTTON><DIV id="ThursdaySCA4" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>23/08/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">DNA Gel Electrophoresis</H4><P>The PCR amplicons of Gblock 1 and 2 and pACYCDuet-1 were run on a 0.8% agarose gel in TAE buffer. 4 μL was pipetted into each lane. The gel was imaged using a gel documentation system with UV-light.</P><P>The gel image shows that the pACYCDuet-1 backbone for Gblock1 and Gblock2 cloning is succesfully amplified. However, the Gblocks 1 and 2 are not correctly amplified as there is a smeer visible.</P><DL><DT>Date</DT><DD>23/08/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">High Fidelity PCR</H4><P>To amplify the Gblocks 1 and 2 as well as the pACYCDuet-1 backbone, high fidelity PCR was done. The template DNA are the Gblocks 1 (primerset 045 &amp; 046) and 2 (primerset 047 &amp; 048) and pACYCDuet-1 (primerset 049 &amp; 050). Six 50µL PCR mixtures were prepared, according to the <A href="https://2018.igem.org/Team:TUDelft/Experiments#highfidelityPCR-scroll" target="_blank" class="adpbl">High Fidelity PCR protocol</A>. 
Primer combination(s) and expected fragment(s)
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Sample</TH><TH class="tableheaderadpbl">Fw primer</TH><TH class="tableheaderadpbl">Rv primer</TH><TH class="tableheaderadpbl">Expected fragment</TH><TH class="tableheaderadpbl">Expected size (bp) </TH></TR><TR class="tableunevenadpbl"><TD>1</TD><TD>045</TD><TD>046</TD><TD>Gblock 1</TD><TD>2892</TD></TR><TR class="tableevenadpbl"><TD>4</TD><TD>047</TD><TD>048</TD><TD>Gblock 2</TD><TD>2890</TD></TR><TR class="tableunevenadpbl"><TD>1</TD><TD>049</TD><TD>050</TD><TD>pACYCDuet-1</TD><TD>3778</TD></TR></TBODY></TABLE></CENTER><P>The thermocycler programme was installed to have an extension time of 1 min 30 seconds and an annealing temperature of 60 °C. After the programme was finished, samples were kept at 4 °C.
</P><DL><DT>Date</DT><DD>23/08/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">DNA Gel Electrophoresis</H4><P>The PCR amplicons of Gblock 1 and 2 and pACYCDuet-1 were run on a 0.8% agarose gel in TAE buffer. 4 μL was pipetted into each lane. The gel was imaged using a gel documentation system with UV-light.</P><P>The gel image shows that the pACYCDuet-1 backbone for Gblock1 and Gblock2 cloning is succesfully amplified. However, the Gblocks 1 and 2 are not correctly amplified as there is a smeer visible.</P><DL><DT>Date</DT><DD>23/08/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">Transformation of chemically competent cells</H4><P>The following transformations were carried out according to the protocol for transformation of chemically competent cells:</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Strain</TH><TH class="tableheaderadpbl">Plasmid and size (bp)</TH><TH class="tableheaderadpbl">Cloning method</TH><TH class="tableheaderadpbl">Volume added (μl)</TH><TH class="tableheaderadpbl">Medium on which was plated</TH></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I> DH5α</TD><TD>pSB1C3_Tn5 (3499 bp)</TD><TD>Gibson Assembly</TD><TD>2.5</TD><TD>LB-Cam</TD></TR><TR class="tableevenadpbl"><TD><I>E.coli</I> DH5α</TD><TD>pSB1C3_T7p-Tn5-T7t (3755 bp)</TD><TD>Gibson Assembly</TD><TD>2.5</TD><TD>LB Cam</TD></TR></TBODY></TABLE></CENTER><P>
200 μl LB medium was added as recovery medium.</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV></DIV></UL></FORM><FORM><LABEL for="WeekSCA5" class="subdropdown cadpbl">Week 5 27/08/2018 - 02/09/2018</LABEL><UL class="submenu"><DIV class="subcontent"><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#MondaySCA5">Monday, 27/08/2018</BUTTON><DIV id="MondaySCA5" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>27/08/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">Gel Purification</H4><P>The amplicon pACYCDuet-1 backbone and Gblock 2 have a size of 3778 bp and 2890 bp respectively, but the PCR samples were contaminated with by-products. The products were loaded on a gel prior to <A href="https://2018.igem.org/Team:TUDelft/Experiments#DNApurificationfromgel-scroll" target="_blank">DNA purification from agarose gel conform protocol</A>.
Elution was done in 50 µL Milli-Q.
</P><DL><DT>Date</DT><DD>27/08/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">Nanodrop DNA Quantification</H4><P>The following isolates were quantified according to the <A href="https://2018.igem.org/Team:TUDelft/Experiments#NanodropDNA-scroll" class="adpbl" target="_blank">Nanodrop DNA Quantification protocol</A>. The concentration and purity of the DNA (260/280 and 260/230 ratios) were measured. </P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Isolate</TH><TH class="tableheaderadpbl">Concentration (ng/uL)</TH><TH class="tableheaderadpbl">260/280 ratio</TH><TH class="tableheaderadpbl">260/230 ratio</TH></TR><TR class="tableunevenadpbl"><TD>pACYCDuet-1 backbone (1)</TD><TD>29.7</TD><TD>1.86</TD><TD>0.44</TD></TR><TR class="tableevenadpbl"><TD>pACYCDuet-1 backbone (2)</TD><TD>25.4</TD><TD>1.88</TD><TD>1.46</TD></TR><TR class="tableunevenadpbl"><TD>pACYCDuet-1 backbone (3)</TD><TD>24.4</TD><TD>1.86</TD><TD>1.46</TD></TR><TR class="tableevenadpbl"><TD>Gblock 2</TD><TD>10.5</TD><TD>1.73</TD><TD>1.14</TD></TR></TBODY></TABLE></CENTER><DL><DT>Date</DT><DD>27/08/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">Gibson Assembly</H4><P>According to the displayed reaction mixture composition, Gibson Assembly was done to assemble pSB1C3 with Tn5 to obtain pSB1C3_Tn5 (3499 bp) and pSB1C3 with T7p-Tn5-T7t to obtain pSB1C3_T7p-Tn5-T7t (3755 bp).
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL)</TH><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL)</TH><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL)</TH></TR><TR class="tableunevenadpbl"><TD>Gibson Assembly Mastermix 2X</TD><TD>10</TD><TD>Gibson Assembly Mastermix 2X</TD><TD>10</TD><TD>Gibson Assembly Mastermix 2X</TD><TD>10</TD></TR><TR class="tableevenadpbl"><TD>Vector pACYCDuet-1</TD><TD>1.6 (40 ng)</TD><TD>Vector pSB1C3</TD><TD>2 (100 ng)</TD><TD>Vector pSB1C3</TD><TD>2 (100 ng)</TD></TR><TR class="tableunevenadpbl"><TD>Fragment Gblock 1</TD><TD>4.3 (43 ng)</TD><TD>Fragment T7p-Tn5-T7t</TD><TD>4 (300 ng)</TD><TD>Fragment Tn5</TD><TD>2 (225 ng)</TD></TR><TR class="tableevenadpbl"><TD>Fragment Gblock 2</TD><TD>4.3 (43 ng)</TD></TR><TR class="tableunevenadpbl"><TD>MilliQ</TD><TD>0</TD><TD>MilliQ</TD><TD>4</TD><TD>MilliQ</TD><TD>6</TD></TR></TBODY></TABLE></CENTER><P>The samples were incubated at 50°C for 1 hour, after which they were kept on ice for subsequent transformation.</P></DIV><DL><DT>Date</DT><DD>27/08/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">Gel Purification</H4><P>The PCR reaction performed on 24/08/2018 were loaded on a gel. Since the expectation of contamination based on previous results, I decided to not use UV imaging due to the potential of creating mutations in the DNA. The amplicon pACYCDuet-1 backbone, Glock 1 and  2 have a size of 3778 bp, 2892 bp and 2890 bp respectively, but indeed the PCR samples were contaminated with by-products. The products were loaded on a gel prior to <A href="https://2018.igem.org/Team:TUDelft/Experiments#DNApurificationfromgel-scroll" target="_blank">DNA purification from agarose gel conform protocol</A>.
Elution was done in 50 µL Milli-Q.
</P><DL><DT>Date</DT><DD>27/08/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">Nanodrop DNA Quantification</H4><P>The following isolates were quantified according to the <A href="https://2018.igem.org/Team:TUDelft/Experiments#NanodropDNA-scroll" class="adpbl" target="_blank">Nanodrop DNA Quantification protocol</A>. The concentration and purity of the DNA (260/280 and 260/230 ratios) were measured. </P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Isolate</TH><TH class="tableheaderadpbl">Concentration (ng/uL)</TH><TH class="tableheaderadpbl">260/280 ratio</TH><TH class="tableheaderadpbl">260/230 ratio</TH></TR><TR class="tableunevenadpbl"><TD>pACYCDuet-1 backbone (1)</TD><TD>43.7</TD><TD>1.89</TD><TD>0.16</TD></TR><TR class="tableevenadpbl"><TD>pACYCDuet-1 backbone (2)</TD><TD>48.3</TD><TD>1.88</TD><TD>0.46</TD></TR><TR class="tableunevenadpbl"><TD>Gblock 1 (1)</TD><TD>34.4</TD><TD>1.87</TD><TD>1.16</TD></TR><TR class="tableevenadpbl"><TD>Gblock 1 (2)</TD><TD>24.8</TD><TD>1.91</TD><TD>1.08</TD></TR><TR class="tableunevenadpbl"><TD>Gblock 2 (1)</TD><TD>23.3</TD><TD>1.81</TD><TD>0.65</TD></TR><TR class="tableevenadpbl"><TD>Gblock 2 (2)</TD><TD>23.2</TD><TD>1.85</TD><TD>0.92</TD></TR></TBODY></TABLE></CENTER></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#WednesdaySCA5">Wednesday, 29/08/2018</BUTTON><DIV id="WednesdaySCA5" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>29/08/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">High Fidelity PCR</H4><P>The dxCas9 coding sequence contains forbidden restriction sites to be RFC compatible. Therefore, Gblocks were ordered at IDT. The dxCas9 is to large to order as one Gblock. Therefore I ordered two Gblocks, also incorporating Linker-Tn5 coding sequences (Gblock 1 and 2). To be able to construct also a different variant of the fusion protein, two Gblocks were ordered in the order Tn5-Linker-dxCas9 (Gblock 3 and 4).

To amplify the Gblocks 3 and 4 as well as the pACYCDuet-1 backbone, high fidelity PCR was done. The template DNA are the Gblocks 1 (primerset 045 &amp; 046) and 2 (primerset 047 &amp; 048) and pACYCDuet-1 (primerset 049 &amp; 050). Additionally, we use the templates Gblocks 3 (primerset 079 &amp; 080) and 4 (primerset 081 &amp; 082) and pACYCDuet-1 (primerset 083 &amp; 084). Multiple 50µL PCR mixtures were prepared, according to the <A href="https://2018.igem.org/Team:TUDelft/Experiments#highfidelityPCR-scroll" target="_blank" class="adpbl">High Fidelity PCR protocol</A>. 
Primer combination(s) and expected fragment(s)
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Sample</TH><TH class="tableheaderadpbl">Fw primer</TH><TH class="tableheaderadpbl">Rv primer</TH><TH class="tableheaderadpbl">Expected fragment</TH><TH class="tableheaderadpbl">Expected size (bp) </TH></TR><TR class="tableunevenadpbl"><TD>1</TD><TD>045</TD><TD>046</TD><TD>Gblock 1</TD><TD>2892</TD></TR><TR class="tableevenadpbl"><TD>2</TD><TD>047</TD><TD>048</TD><TD>Gblock 2</TD><TD>2890</TD></TR><TR class="tableunevenadpbl"><TD>3</TD><TD>049</TD><TD>050</TD><TD>pACYCDuet-1</TD><TD>3778</TD></TR><TR class="tableunevenadpbl"><TD>4</TD><TD>079</TD><TD>080</TD><TD>Gblock 3</TD></TR><TR class="tableevenadpbl"><TD>5</TD><TD>081</TD><TD>082</TD><TD>Gblock 4</TD></TR><TR class="tableunevenadpbl"><TD>6</TD><TD>083</TD><TD>084</TD><TD>pACYCDuet-1</TD><TD>3652</TD></TR></TBODY></TABLE></CENTER><P>The thermocycler programme was installed to have an extension time of 4 minutes and an annealing temperature in a gradient from  52 °C 66 °C. After the programme was finished, samples were kept at 4 °C.
</P><DL><DT>Date</DT><DD>29/08/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">High Fidelity PCR</H4><P>To amplify the pSB1C3 BioBrick backbone, Tn5 and Linker coding sequences and T7p-Tn5-T7t expression cassette, high fidelity PCR was done. The template DNA is the BioBrick plasmid pSB1C3 (primerset 051 &amp; 052 &amp; 053) and pACYCDuet-1_Tn5 (054 &amp; 055, 058 &amp; 059 and 060 &amp; 061). 54 50µL PCR mixtures were prepared, according to the <A href="https://2018.igem.org/Team:TUDelft/Experiments#highfidelityPCR-scroll" target="_blank" class="adpbl">High Fidelity PCR protocol</A>. 
Primer combination(s) and expected fragment(s)
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Sample</TH><TH class="tableheaderadpbl">Fw primer</TH><TH class="tableheaderadpbl">Rv primer</TH><TH class="tableheaderadpbl">Expected fragment</TH><TH class="tableheaderadpbl">Expected size (bp) </TH></TR><TR class="tableunevenadpbl"><TD>1</TD><TD>051</TD><TD>052</TD><TD>pSB1C3 backbone</TD><TD>2068</TD></TR><TR class="tableevenadpbl"><TD>2</TD><TD>051</TD><TD>053</TD><TD>pSB1C3 backbone with extra nucleotide</TD><TD>2070</TD></TR><TR class="tableunevenadpbl"><TD>3</TD><TD>054</TD><TD>055</TD><TD>Tn5</TD><TD>1511</TD></TR><TR class="tableevenadpbl"><TD>4</TD><TD>058</TD><TD>059</TD><TD>Linker</TD><TD>143</TD></TR><TR class="tableevenadpbl"><TD>5</TD><TD>060</TD><TD>061</TD><TD>T7p-Tn5-T7t</TD><TD>1756</TD></TR></TBODY></TABLE></CENTER><P>The thermocycler programme was installed to have an extension time of 2 min 30 seconds for pSB1C3 backbones, Tn5 and T7p-Tn5-T7t and 30 seconds for Linker and both an annealing temperature of 60 °C. After the programme was finished, samples were kept at 4 °C.
</P><DL><DT>Date</DT><DD>29/08/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">High Fidelity PCR</H4><P>
To amplify the Gblocks 1, 2, 3 and 4 a high fidelity PCR was done, but using a 2 step PCR program. The template DNA are the Gblocks 1 (primerset 045 &amp; 046) and 2 (primerset 047 &amp; 048) and pACYCDuet-1 (primerset 049 &amp; 050). Additionally, we use the templates Gblocks 3 (primerset 079 &amp; 080) and 4 (primerset 081 &amp; 082) and pACYCDuet-1 (primerset 083 &amp; 084). Multiple 50µL PCR mixtures were prepared, according to the <A href="https://2018.igem.org/Team:TUDelft/Experiments#highfidelityPCR-scroll" target="_blank" class="adpbl">High Fidelity PCR protocol</A>. 
Primer combination(s) and expected fragment(s)
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Sample</TH><TH class="tableheaderadpbl">Fw primer</TH><TH class="tableheaderadpbl">Rv primer</TH><TH class="tableheaderadpbl">Expected fragment</TH><TH class="tableheaderadpbl">Expected size (bp) </TH></TR><TR class="tableunevenadpbl"><TD>1</TD><TD>045</TD><TD>046</TD><TD>Gblock 1</TD><TD>2892</TD></TR><TR class="tableevenadpbl"><TD>2</TD><TD>047</TD><TD>048</TD><TD>Gblock 2</TD><TD>2890</TD></TR><TR class="tableunevenadpbl"><TD>4</TD><TD>079</TD><TD>080</TD><TD>Gblock 3</TD></TR><TR class="tableevenadpbl"><TD>5</TD><TD>081</TD><TD>082</TD><TD>Gblock 4</TD></TR></TBODY></TABLE></CENTER><P>The thermocycler programme was installed to have an extension time of 4 minutes and an annealing temperature of 72 °C (2 step PCR). After the programme was finished, samples were kept at 4 °C.
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#ThursdaySCA5">Thursday, 30/08/2018</BUTTON><DIV id="ThursdaySCA5" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>30/08/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4>DNA Gel Electrophoresis</H4><P>The PCR amplicons for construction of the fusion protein with Gblock 1 and 2 and Gblock 3 and 4 as well as the amplicons for BioBricks Tn5, T7p-Tn5-T7t and Linker into pSB13 were run on multiple 0.8% agarose gel in TAE buffer to check for amplification. 4 μL was pipetted into each lane. The gel was imaged using a gel documentation system with UV-light.</P><DL><DT>Date</DT><DD>30/08/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">Gel Purification</H4><P>The PCR amplicons were contaminated with by-products. The products were loaded on a gel prior to <A href="https://2018.igem.org/Team:TUDelft/Experiments#DNApurificationfromgel-scroll" target="_blank">DNA purification from agarose gel conform protocol</A>.
Elution was done in 50 µL Milli-Q.
</P><DL><DT>Date</DT><DD>09/08/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">Restriction</H4><P>The following table shows an overview of what fragment(s) were cut with what restriction enzyme(s) to obtain fragments Tn5-Lin-dxCas9 and pACYCDuet-1 for ligation.
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL)</TH><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL)</TH><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL)</TH></TR><TR class="tableunevenadpbl"><TD>10x CutSmart</TD><TD>5</TD><TD>10x CutSmart</TD><TD>2</TD><TD>10x CutSmart</TD><TD>2</TD></TR><TR class="tableevenadpbl"><TD>pSB1C3</TD><TD>43 (~2000 ng)</TD><TD>Tn5</TD><TD>17 (~2000 ng)</TD><TD>T7p-Tn5-T7t</TD><TD>17 (~2000 ng)</TD></TR><TR class="tableunevenadpbl"><TD>Enzyme (EcoRI)</TD><TD>1</TD><TD>Enzyme (EcoRI)</TD><TD>0.5</TD><TD>Enzyme (EcoRI)</TD><TD>0.5</TD></TR><TR class="tableevenadpbl"><TD>Enzyme (PstI)</TD><TD>1</TD><TD>Enzyme (PstI)</TD><TD>0.5</TD><TD>Enzyme (PstI)</TD><TD>0.5</TD></TR><TR class="tableunevenadpbl"><TD>MilliQ</TD><TD>0</TD><TD>MilliQ</TD><TD>0</TD><TD>MilliQ</TD><TD>0</TD></TR></TBODY></TABLE></CENTER><P>The samples were incubated for 4 hours at 37 °C, and followed by 20 minute heat inactivation of the restriction enzyme(s). 
</P><DL><DT>Date</DT><DD>30/08/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">PCR Clean-Up</H4><P>50 or 20 μL of each restriction reaction mixture was subjected to PCR Clean-Up, according to the <A href="https://2018.igem.org/Team:TUDelft/Experiments#PCRcleanup-scroll" class="adpbl" target="_blank">protocol</A>. Elution was done in 30 µL Milli-Q.</P><DL><DT>Date</DT><DD>30/08/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">Nanodrop DNA Quantification</H4><P>The following isolates were quantified according to the <A href="https://2018.igem.org/Team:TUDelft/Experiments#NanodropDNA-scroll" class="adpbl" target="_blank">Nanodrop DNA Quantification protocol</A>. The concentration and purity of the DNA (260/280 and 260/230 ratios) were measured. </P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Isolate</TH><TH class="tableheaderadpbl">Concentration (ng/uL)</TH><TH class="tableheaderadpbl">260/280 ratio</TH><TH class="tableheaderadpbl">260/230 ratio</TH></TR><TR class="tableunevenadpbl"><TD>pSB1C3 cut with EcoRI and PstI</TD><TD>49.0</TD><TD>1.82</TD><TD>1.55</TD></TR><TR class="tableevenadpbl"><TD>Tn5 cut with EcoRI and PstI</TD><TD>39.4</TD><TD>1.86</TD><TD>1.79</TD></TR><TR class="tableunevenadpbl"><TD>T7p-Tn5-T7t cut with EcoRI and PstI</TD><TD>34.9</TD><TD>1.79</TD><TD>1.30</TD></TR></TBODY></TABLE></CENTER><DL><DT>Date</DT><DD>30/08/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">Ligation</H4><P>In order to assemble cut fragments pSb1C with Tn5 or T7p-Tn5-T7t ligation was performed. This table gives an overview of which fragments were cut with which enzymes.</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">DNA</TH><TH class="tableheaderadpbl">Size (bp) </TH><TH class="tableheaderadpbl">Cut with enzymes</TH></TR><TR class="tableunevenadpbl"><TD>pSB1C3</TD><TD>2068</TD><TD>EcoRI and PstI</TD></TR><TR class="tableevenadpbl"><TD>Fragment Tn5</TD><TD>1511</TD><TD>EcoRI and PstI</TD></TR><TR class="tableunevenadpbl"><TD>Fragment T7p-Tn5-T7t</TD><TD>1756</TD><TD>NotI and PstI</TD></TR></TBODY></TABLE></CENTER><P>A 20 μL ligation reaction mixture was prepared with the following composition:</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (μL)</TH><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (μL)</TH></TR><TR class="tableunevenadpbl"><TD>Ligase Buffer (10X)</TD><TD>2</TD><TD>Ligase Buffer (10X)</TD><TD>2</TD></TR><TR class="tableevenadpbl"><TD>T4 DNA Ligase </TD><TD>1</TD><TD>T4 DNA Ligase </TD><TD>1</TD></TR><TR class="tableunevenadpbl"><TD>DNA vector pSB1C3 (~200 ng)</TD><TD>4</TD><TD>DNA vector pSB1C3 (~200 ng)</TD><TD>13</TD></TR><TR class="tableevenadpbl"><TD>DNA Tn5 (~450 ng)</TD><TD>0.7</TD><TD>DNA T7p-Tn5-T7t (~600 ng)</TD><TD>13</TD></TR><TR class="tableunevenadpbl"><TD>MilliQ</TD><TD>0</TD><TD>MilliQ</TD><TD>0</TD></TR></TBODY></TABLE></CENTER><P>The samples were incubated overnight at 4 °C for ligation.</P><DL><DT>Date</DT><DD>30/08/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">High Fidelity PCR</H4><P>To amplify the Gblocks 1 and 2 and Gblock 3 and 4 as one fragment, high fidelity overlap PCR was done. The template DNA are the Gblocks 1 and 2 (primerset 045 &amp; 048) and Gblocks 3 and 4 (primerset 079 &amp; 082). two 50µL PCR mixtures were prepared, according to the <A href="https://2018.igem.org/Team:TUDelft/Experiments#highfidelityPCR-scroll" target="_blank" class="adpbl">High Fidelity PCR protocol</A>. 
Primer combination(s) and expected fragment(s)
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Sample</TH><TH class="tableheaderadpbl">Fw primer</TH><TH class="tableheaderadpbl">Rv primer</TH><TH class="tableheaderadpbl">Expected fragment</TH><TH class="tableheaderadpbl">Expected size (bp) </TH></TR><TR class="tableunevenadpbl"><TD>1</TD><TD>045</TD><TD>048</TD><TD>Gblock 1&amp;2</TD><TD>5741</TD></TR><TR class="tableevenadpbl"><TD>2</TD><TD>079</TD><TD>082</TD><TD>Gblock 3&amp;4</TD><TD>5897</TD></TR></TBODY></TABLE></CENTER><P>The thermocycler programme was installed to have an extension time of 6 minutes 30 seconds and an annealing temperature of 60 °C. After the programme was finished, samples were kept at 4 °C.
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#FridaySCA5">Friday, 31/08/2018</BUTTON><DIV id="FridaySCA5" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>31/08/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">Transformation of chemically competent cells</H4><P>The following transformations were carried out according to the protocol for transformation of chemically competent cells:</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Strain</TH><TH class="tableheaderadpbl">Plasmid and size (bp)</TH><TH class="tableheaderadpbl">Cloning method</TH><TH class="tableheaderadpbl">Volume added (μl)</TH><TH class="tableheaderadpbl">Medium on which was plated</TH></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I> DH5α</TD><TD>pSB1C3_Tn5 (3499 bp)</TD><TD>Restriction/Ligation</TD><TD>5</TD><TD>LB-Cam</TD></TR><TR class="tableevenadpbl"><TD><I>E.coli</I> DH5α</TD><TD>pSB1C3_T7p-Tn5-T7t (3755 bp)</TD><TD>Restriction/Ligation</TD><TD>5</TD><TD>LB Cam</TD></TR></TBODY></TABLE></CENTER><P>
200 μl LB medium was added as recovery medium.</P><DL><DT>Date</DT><DD>30/08/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">Nanodrop DNA Quantification</H4><P>The excised gel fragments that were cleaned using PCR Clean up were quantified according to the <A href="https://2018.igem.org/Team:TUDelft/Experiments#NanodropDNA-scroll" class="adpbl" target="_blank">Nanodrop DNA Quantification protocol</A>. The concentration and purity of the DNA (260/280 and 260/230 ratios) were measured. </P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Isolate</TH><TH class="tableheaderadpbl">Concentration (ng/uL)</TH><TH class="tableheaderadpbl">260/280 ratio</TH><TH class="tableheaderadpbl">260/230 ratio</TH></TR><TR class="tableunevenadpbl"><TD>pSB1C3</TD><TD>740.3</TD><TD>1.84</TD><TD>2.08</TD></TR><TR class="tableevenadpbl"><TD>pACYCDuet-1 backbone Gblock 1,2</TD><TD>150.9</TD><TD>1.92</TD><TD>1.93</TD></TR><TR class="tableunevenadpbl"><TD>pACYCDuet-1 backbone Gblock 3,4</TD><TD>173.0</TD><TD>1.91</TD><TD>1.98</TD></TR><TR class="tableevenadpbl"><TD>Gblock 2</TD><TD>21.4</TD><TD>1.88</TD><TD>1.00</TD></TR><TR class="tableunevenadpbl"><TD>Gblock 3</TD><TD>113.6</TD><TD>1.90</TD><TD>1.83</TD></TR><TR class="tableevenadpbl"><TD>Gblock 4</TD><TD>11.4</TD><TD>1.91</TD><TD>0.53</TD></TR></TBODY></TABLE></CENTER><DL><DT>Date</DT><DD>31/08/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4>DNA Gel Electrophoresis</H4><P>The overlap PCR amplicons for construction of the fusion protein with Gblock 1 and 2 and Gblock 3 and 4 during overlap PCR were run on multiple 0.8% agarose gel in TAE buffer to check for amplification. Also, the ligation reaction mixture was loaded on the gel, to check if the desired plasmids were already assembled and ligated. 4 μL was pipetted into each lane. The gel was imaged using a gel documentation system with UV-light.</P><DL><DT>Date</DT><DD>31/08/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4>Plasmid drying</H4><P>In order to sent our pACYCDuet-1_dxCas9, pACYCDuet-1_Tn5 and pACYCDuet-1_dxCas9-Lin-Tn5 to Marburg for our <A href="https://2018.igem.org/Team:TUDelft/Collaborations" class="adpbl" target="_blank">collaboration</A>, I dried the plasmids in a 96 wells plate.
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV></DIV></UL></FORM><H3 class="adpbl">September</H3><FORM><LABEL for="WeekSCS1" class="subdropdown cadpbl">Week 1 03/09/2018 - 09/09/2018</LABEL><UL class="submenu"><DIV class="subcontent"><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#MondaySCS1">Monday, 03/09/2018</BUTTON><DIV id="MondaySCS1" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>03/09/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">Restriction</H4><P>The following table shows an overview of what fragment(s) were cut with what restriction enzyme(s) to obtain fragments Tn5-Lin-dxCas9 and pACYCDuet-1 for ligation.
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL)</TH><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL)</TH><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL)</TH></TR><TR class="tableunevenadpbl"><TD>10x CutSmart</TD><TD>5</TD><TD>10x CutSmart</TD><TD>2</TD><TD>10x CutSmart</TD><TD>2</TD></TR><TR class="tableevenadpbl"><TD>pSB1C3</TD><TD>6 (~3000 ng)</TD><TD>Tn5</TD><TD>4 (~3000 ng)</TD><TD>T7p-Tn5-T7t</TD><TD>4 (~3000 ng)</TD></TR><TR class="tableunevenadpbl"><TD>Enzyme (EcoRI)</TD><TD>1</TD><TD>Enzyme (EcoRI)</TD><TD>0.5</TD><TD>Enzyme (EcoRI)</TD><TD>0.5</TD></TR><TR class="tableevenadpbl"><TD>Enzyme (PstI)</TD><TD>1</TD><TD>Enzyme (PstI)</TD><TD>0.5</TD><TD>Enzyme (PstI)</TD><TD>0.5</TD></TR><TR class="tableunevenadpbl"><TD>MilliQ</TD><TD>10</TD><TD>MilliQ</TD><TD>12</TD><TD>MilliQ</TD><TD>12</TD></TR></TBODY></TABLE></CENTER><P>The samples were incubated for 4 hours at 37 °C, and followed by 20 minute heat inactivation of the restriction enzyme(s). 
</P><DL><DT>Date</DT><DD>30/08/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">PCR Clean-Up</H4><P>20 μL of each restriction reaction mixture was subjected to PCR Clean-Up, according to the <A href="https://2018.igem.org/Team:TUDelft/Experiments#PCRcleanup-scroll" class="adpbl" target="_blank">protocol</A>. Elution was done in 30 µL Milli-Q.</P><DL><DT>Date</DT><DD>30/08/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">Nanodrop DNA Quantification</H4><P>The following isolates were quantified according to the <A href="https://2018.igem.org/Team:TUDelft/Experiments#NanodropDNA-scroll" class="adpbl" target="_blank">Nanodrop DNA Quantification protocol</A>. The concentration and purity of the DNA (260/280 and 260/230 ratios) were measured. </P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Isolate</TH><TH class="tableheaderadpbl">Concentration (ng/uL)</TH><TH class="tableheaderadpbl">260/280 ratio</TH><TH class="tableheaderadpbl">260/230 ratio</TH></TR><TR class="tableunevenadpbl"><TD>pSB1C3 cut with EcoRI and PstI</TD><TD>56.3</TD><TD>1.85</TD><TD>0.43</TD></TR><TR class="tableevenadpbl"><TD>Tn5 cut with EcoRI and PstI</TD><TD>45.1</TD><TD>1.76</TD><TD>0.62</TD></TR><TR class="tableunevenadpbl"><TD>T7p-Tn5-T7t cut with EcoRI and PstI</TD><TD>157.8</TD><TD>1.76</TD><TD>0.71</TD></TR></TBODY></TABLE></CENTER><DL><DT>Date</DT><DD>30/08/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">Ligation</H4><P>In order to assemble cut fragments pSb1C with Tn5 or T7p-Tn5-T7t ligation was performed. This table gives an overview of which fragments were cut with which enzymes.</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">DNA</TH><TH class="tableheaderadpbl">Size (bp) </TH><TH class="tableheaderadpbl">Cut with enzymes</TH></TR><TR class="tableunevenadpbl"><TD>pSB1C3</TD><TD>2068</TD><TD>EcoRI and PstI</TD></TR><TR class="tableevenadpbl"><TD>Fragment Tn5</TD><TD>1511</TD><TD>EcoRI and PstI</TD></TR><TR class="tableunevenadpbl"><TD>Fragment T7p-Tn5-T7t</TD><TD>1756</TD><TD>NotI and PstI</TD></TR></TBODY></TABLE></CENTER><P>A 20 μL ligation reaction mixture was prepared with the following composition:</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (μL)</TH><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (μL)</TH></TR><TR class="tableunevenadpbl"><TD>Ligase Buffer (10X)</TD><TD>2</TD><TD>Ligase Buffer (10X)</TD><TD>2</TD></TR><TR class="tableevenadpbl"><TD>T4 DNA Ligase </TD><TD>1</TD><TD>T4 DNA Ligase </TD><TD>1</TD></TR><TR class="tableunevenadpbl"><TD>DNA vector pSB1C3 (~200 ng)</TD><TD>5</TD><TD>DNA vector pSB1C3 (~200 ng)</TD><TD>5</TD></TR><TR class="tableevenadpbl"><TD>DNA Tn5 (~450 ng)</TD><TD>12</TD><TD>DNA T7p-Tn5-T7t (~600 ng)</TD><TD>12</TD></TR><TR class="tableunevenadpbl"><TD>MilliQ</TD><TD>0</TD><TD>MilliQ</TD><TD>0</TD></TR></TBODY></TABLE></CENTER><P>The samples were incubated overnight at 4 °C for ligation.</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#TuesdaySCS1">Tuesday, 04/09/2018</BUTTON><DIV id="TuesdaySCS1" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>04-09-2018</DD><DT>Operator</DT><DD>Susan Bouwmeester</DD></DL><H4>Restriction </H4><P>Restriction cloning was done for assembling both Tn5 (157.1ng/uL) and T7-Tn5-T7 (94.7ng/uL) into with previous with PCR linearized pSB1C3 backbone (64.8ng/uL). The following table shows an overview of what fragments were cut with what restriction enzymes.
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (Tn5) (uL)</TH><TH class="tableheaderadpbl">Volume (T7p.Tn5.T7t) (uL)</TH><TH class="tableheaderadpbl">Volume (pSB1C3) (uL)</TH></TR><TR class="tableunevenadpbl"><TD>10x CutSmart buffer    </TD><TD>2</TD><TD>2</TD><TD>2</TD></TR><TR class="tableunevenadpbl"><TD>Fragment</TD><TD>1.6</TD><TD>2.6</TD><TD>3.8</TD></TR><TR class="tableunevenadpbl"><TD>Enzyme (EcoRI)</TD><TD>0.5</TD><TD>0.5</TD><TD>0.5</TD></TR><TR class="tableunevenadpbl"><TD>Enzyme (PstI)</TD><TD>0.5</TD><TD>0.5</TD><TD>0.5</TD></TR><TR class="tableunevenadpbl"><TD>MilliQ</TD><TD> 14.4</TD><TD> 15.4</TD><TD> 13.2</TD></TR></TBODY></TABLE></CENTER><P>The samples were incubated for 30 minutes at 37 °C, and followed by 20 minutes heat inactivation of the restriction enzymes at 80°C.
</P><DL><DT>Date</DT><DD>04/09/2018</DD><DT>Operator</DT><DD>Susan Bouwmeester</DD></DL><H4>Ligation</H4><P>In order to assemble cut fragments both Tn5 and T7p.Tn5.T7t to the pSB1C3 backbone, ligation was performed. This table gives an overview of which fragments were cut with which enzymes.
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">DNA</TH><TH class="tableheaderadpbl">Size (bp) </TH><TH class="tableheaderadpbl">Cut with enzymes</TH></TR><TR class="tableunevenadpbl"><TD>Vector pSB1C3</TD><TD>2068</TD><TD>EcoRI and PstI</TD></TR><TR class="tableevenadpbl"><TD>Fragment 1 Tn5</TD><TD>1503</TD><TD>EcoRI and PstI</TD></TR><TR class="tableunevenadpbl"><TD>Fragment 2 T7p.Tn5.T7t</TD><TD>1756</TD><TD>EcoRI and PstI</TD></TR></TBODY></TABLE></CENTER><P>A 10uL ligation reaction mixture was prepared with the following composition</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume Tn5 (uL) </TH><TH class="tableheaderadpbl">Volume T7p.Tn5.T7t (uL) </TH></TR><TR class="tableunevenadpbl"><TD>Ligase Buffer (10x) </TD><TD>1</TD><TD>1</TD></TR><TR class="tableevenadpbl"><TD>T4 DNA Ligase</TD><TD>0.5</TD><TD>0.5</TD></TR><TR class="tableunevenadpbl"><TD>DNA vector </TD><TD>2</TD><TD>2</TD></TR><TR class="tableunevenadpbl"><TD>DNA fragment </TD><TD>2</TD><TD>2</TD></TR><TR class="tableunevenadpbl"><TD>MilliQ</TD><TD>4.5</TD><TD>4.5</TD></TR></TBODY></TABLE></CENTER><P>The samples were incubated for 4 hours at 16°C for ligation and followed by 20 minutes heat inactivation of the ligase at 80°C.</P><DL><DT>Date</DT><DD>04/09/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">High Fidelity PCR</H4><P>To amplify the 3 and 4 as one fragment, high fidelity overlap PCR was done. The template DNA are Gblocks 3 and 4 (primerset 079 &amp; 082). Eight 50µL PCR mixtures were prepared, according to the <A href="https://2018.igem.org/Team:TUDelft/Experiments#highfidelityPCR-scroll" target="_blank" class="adpbl">High Fidelity PCR protocol</A>. 
Primer combination(s) and expected fragment(s)
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Sample</TH><TH class="tableheaderadpbl">Fw primer</TH><TH class="tableheaderadpbl">Rv primer</TH><TH class="tableheaderadpbl">Expected fragment</TH><TH class="tableheaderadpbl">Expected size (bp) </TH></TR><TR class="tableunevenadpbl"><TD>1</TD><TD>079</TD><TD>082</TD><TD>Gblock 3&amp;4</TD><TD>5897</TD></TR></TBODY></TABLE></CENTER><P>The thermocycler programme was installed to have an extension time of 6 minutes 30 seconds and an annealing temperature of 60 °C. After the programme was finished, samples were kept at 4 °C.
</P><DL><DT>Date</DT><DD>04/09/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">Transformation of chemically competent cells</H4><P>The following transformations were carried out according to the protocol for transformation of chemically competent cells:</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Strain</TH><TH class="tableheaderadpbl">Plasmid and size (bp)</TH><TH class="tableheaderadpbl">Cloning method</TH><TH class="tableheaderadpbl">Volume added (μl)</TH><TH class="tableheaderadpbl">Medium on which was plated</TH></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I> DH5α</TD><TD>pSB1C3_Tn5 (3499 bp)</TD><TD>Restriction/Ligation</TD><TD>2.5</TD><TD>LB-Cam</TD></TR><TR class="tableevenadpbl"><TD><I>E.coli</I> DH5α</TD><TD>pSB1C3_T7p-Tn5-T7t (3755 bp)</TD><TD>Restriction/Ligation</TD><TD>2.5</TD><TD>LB Cam</TD></TR></TBODY></TABLE></CENTER><P>
200 μl LB medium was added as recovery medium.</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#WednesdaySCS1">Wednesday, 05/09/2018</BUTTON><DIV id="WednesdaySCS1" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>05/09/2018</DD><DT>Operator</DT><DD>Susan Bouwmeester</DD></DL><H4> Transformation of chemically competent cells </H4><P>The following transformations were carried out according to the protocol for transformation of chemically competent cells:
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Strain</TH><TH class="tableheaderadpbl">Plasmid &amp; size (bp) </TH><TH class="tableheaderadpbl">Cloning method</TH><TH class="tableheaderadpbl">Volume added (uL)</TH><TH class="tableheaderadpbl">Medium on which was plated</TH></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I>DH5α</TD><TD>[pSB1C3-Tn5] (3499bp)</TD><TD>Restriction/ligation cloning</TD><TD>2uL</TD><TD>LBcam</TD></TR></TBODY></TABLE></CENTER><P>950 uL LB medium was added as recovery medium.
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#ThursdaySCS1">Thursday, 06/09/2018</BUTTON><DIV id="ThursdaySCS1" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>06/09/2018</DD><DT>Operator</DT><DD>Susan Bouwmeester</DD></DL><H4> DNA Purification of linker from Agarose Gel </H4><P>The amplicon of interest, the linker, has a size of 69bp, but the PCR samples were contaminated with by-products. The products were loaded on a 2% agarose gel prior to DNA purification from agarose gel conform protocol.
Elution was done in 30 µL Milli-Q.
</P><H4>DNA Purification of pSB1C3, with extra nucleotide, from Agarose Gel  </H4><P>The amplicon of interest, pSB1C3 with extra nucleotide, has a size of around 2000bp, but the PCR samples were contaminated with by-products. The products were loaded on a 0.8% agarose gel prior to DNA purification from agarose gel conform protocol.
    Elution was done in 30 µL Milli-Q.</P><DL><DT>Date</DT><DD>06/09/2018</DD><DT>Operator</DT><DD>Susan Bouwmeester</DD></DL><H4>Gibson Assembly</H4><P>According to the displayed reaction mixture composition, Gibson Assembly was done to assemble both Tn5 and T7p.Tn5.T7t and pSB1C3 to obtain pSB1C3-Tn5 (3499bp) and pSB1C3-T7p.Tn5.T7t (3605bp).
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL) pSB1C3-Tn5</TH><TH class="tableheaderadpbl">Volume (uL) pSB1C3-T7p.Tn5.T7t</TH></TR><TR class="tableunevenadpbl"><TD>Gibson Assembly Mastermix 2X</TD><TD>10</TD><TD>10</TD></TR><TR class="tableevenadpbl"><TD>Vector</TD><TD>1.54uL (100 ng)</TD><TD>1.54uL (100 ng)</TD></TR><TR class="tableunevenadpbl"><TD>Fragment</TD><TD>1.34 uL (210.6 ng)</TD><TD>2.3 uL (2107.9 ng)</TD></TR><TR class="tableunevenadpbl"><TD>MilliQ</TD><TD>7.12</TD><TD>6.2</TD></TR></TBODY></TABLE></CENTER><P>The samples were incubated at 50°C for 60 minutes, after which they were kept on ice for subsequent transformation.</P><DL><DT>Date</DT><DD>06/09/2018</DD><DT>Operator</DT><DD>Susan Bouwmeester</DD></DL><H4> Transformation of chemically competent cells </H4><P>The following transformations were carried out according to the protocol for transformation of chemically competent cells:
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Strain</TH><TH class="tableheaderadpbl">Plasmid &amp; size (bp) </TH><TH class="tableheaderadpbl">Cloning method</TH><TH class="tableheaderadpbl">Volume added (uL)</TH><TH class="tableheaderadpbl">Medium on which was plated</TH></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I>DH5α</TD><TD>[pSB1C3-Tn5] (3499bp)</TD><TD>Gibson Assembly</TD><TD>2</TD><TD>LB Cam</TD></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I>DH5α</TD><TD>[pSB1C3-T7p.Tn5.T7t] (3605bp)</TD><TD>Gibson Assembly</TD><TD>2</TD><TD>LB Cam</TD></TR></TBODY></TABLE></CENTER><P>950 uL LB medium was added as recovery medium.
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#FridaySCS1">Friday, 07/09/2018</BUTTON><DIV id="FridaySCS1" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><H4>Nanodrop DNA Quantification</H4><P> 
    The following isolates were quantified according to the Nanodrop DNA Quantification protocol. The concentration and purity of the DNA (260/280 and 260/230 ratios) were measured. 
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Isolate</TH><TH class="tableheaderadpbl">Concentration (ng/uL)</TH><TH class="tableheaderadpbl">260/280 ratio</TH><TH class="tableheaderadpbl">260/230 ratio</TH></TR><TR class="tableunevenadpbl"><TD>pSB1C3extranuc1</TD><TD>366.7</TD><TD>1.88</TD><TD>2.04</TD></TR><TR class="tableevenadpbl"><TD>pSB1C3extranuc2</TD><TD>415.5</TD><TD>1.88</TD><TD>2.17</TD></TR><TR class="tableunevenadpbl"><TD> linker1</TD><TD>41.7</TD><TD>1.83</TD><TD>1.43</TD></TR><TR class="tableunevenadpbl"><TD>linker2</TD><TD>39.3</TD><TD>1.83</TD><TD>1.44</TD></TR><TR class="tableevenadpbl"><TD>Fusionduet1</TD><TD>120.3</TD><TD>1.82</TD><TD>1.61</TD></TR><TR class="tableunevenadpbl"><TD> Fusionduet2</TD><TD>49.7</TD><TD>1.68</TD><TD>0.81</TD></TR><TR class="tableunevenadpbl"><TD>Fusionbb</TD><TD>294.9</TD><TD>1.85</TD><TD>2.10</TD></TR></TBODY></TABLE></CENTER><DL><DT>Date</DT><DD>9/7/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>High Fidelity PCR</H4><P> To amplify multiple parts required for strain construction, high fidelity PCR was done using the following primer and template combinations. The sample preparation was done according to the High Fidelity PCR protocol.
    
Primer combination(s) and expected fragment(s):

</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Template*</TH><TH class="tableheaderadpbl">Fw &amp; Rv primer</TH><TH class="tableheaderadpbl">Amplicon name &amp; size in bp </TH><TH class="tableheaderadpbl">Elongation time (min) </TH></TR><TR class="tableunevenadpbl"><TD>Gblock1 &amp; Gblock2</TD><TD>054 &amp; 057</TD><TD>Gblock1-Gblock2 (~6kb)</TD><TD>6.5</TD></TR><TR class="tableevenadpbl"><TD>Gblock1 &amp; Gblock2</TD><TD>048 &amp; 088</TD><TD>Gblock1-Gblock2 (~6kb)</TD><TD>6.5</TD></TR><TR class="tableunevenadpbl"><TD>Gblock3-Gblock4</TD><TD>079 &amp; 082</TD><TD>Gblock3-Gblock4 (~6kb)</TD><TD>6.5</TD></TR><TR class="tableevenadpbl"><TD>Gblock3 &amp; Gblock4</TD><TD>056 &amp; 055</TD><TD>Gblock3-Gblock4~6kb</TD><TD>6.5</TD></TR><TR class="tableunevenadpbl"><TD>Gblock1</TD><TD>088 &amp; 089</TD><TD>Gblock 1 (~3kb)</TD><TD>4</TD></TR><TR class="tableevenadpbl"><TD>pACYC</TD><TD>079 &amp; 082</TD><TD>pACYC backbone (~4kb)</TD><TD>4</TD></TR><TR class="tableunevenadpbl"><TD>pACYC</TD><TD>083 &amp; 084</TD><TD>pACYC backbone (~4kb)</TD><TD>4</TD></TR></TBODY></TABLE></CENTER>     
 *when the template consists of two molecules (like Gblock1 &amp; Gblock2), this is indicated with an &amp; sign, and means overlap PCR. A hyphen (-) indicates that the two molecules are already connected.   
<P>The thermocycler programme was installed to have an an annealing temperature of 60 °C. After the programme was finished, samples were kept at 4 °C.</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#SaturdaySCS1">Saturday, 08/09/2018</BUTTON><DIV id="SaturdaySCS1" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>08-09-2018</DD><DT>Operator</DT><DD>Susan Bouwmeester</DD></DL><H4>Restriction </H4><P>Restriction cloning was done for five different constructs. The following table shows an overview of what fragments were cut with what restriction enzymes. 
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (pSB1C3en) (uL)</TH><TH class="tableheaderadpbl">Volume (Linker1) (uL)</TH><TH class="tableheaderadpbl">Volume (Tn5) (uL)</TH><TH class="tableheaderadpbl">Volume (T7p.Tn5.T7t) (uL)</TH><TH class="tableheaderadpbl">Volume (pSB1C3en) (uL)</TH><TH class="tableheaderadpbl">Volume (fusionbb) (uL)</TH><TH class="tableheaderadpbl">Volume (Pacyc) (uL)</TH><TH class="tableheaderadpbl">Volume (fusionduet) </TH><TH class="tableheaderadpbl">Volume (fusionduet2)</TH></TR><TR class="tableunevenadpbl"><TD>10x CutSmart buffer    </TD><TD>2</TD><TD>2</TD><TD>2</TD><TD>2</TD><TD>2</TD><TD>2</TD><TD>2</TD><TD>2</TD><TD>2</TD></TR><TR class="tableunevenadpbl"><TD>Fragment</TD><TD>5.4 (~2000ng)</TD><TD>5.99 (~250ng)</TD><TD>2 (~1500ng)</TD><TD>10.6 (~1000ng)</TD><TD>3 (~2000ng)</TD><TD>10 (~3000ng)</TD><TD>10 (~1500ng)</TD><TD>10 (500ng)</TD><TD>10 (1200ng)</TD></TR><TR class="tableunevenadpbl"><TD>Enzyme </TD><TD>EcoRI 0.5uL</TD><TD>EcoRI 0.5uL</TD><TD>EcoRI 0.5uL</TD><TD>EcoRI 0.5uL</TD><TD>EcoRI 0.5uL</TD><TD>EcoRI 0.5uL</TD><TD>KpnI 0.5uL</TD><TD>KpnI 0.5uL</TD><TD>KpnI 0.5uL</TD></TR><TR class="tableunevenadpbl"><TD>Enzyme </TD><TD>PstI 0.5uL</TD><TD>PstI 0.5uL</TD><TD>PstI 0.5uL</TD><TD>PstI 0.5uL</TD><TD>PstI 0.5uL</TD><TD>PstI 0.5uL</TD><TD>NcoI 0.5uL</TD><TD>NcoI 0.5uL</TD><TD>NcoI 0.5uL</TD></TR><TR class="tableunevenadpbl"><TD>MilliQ</TD><TD> 11.6</TD><TD> 12</TD><TD> 15</TD><TD>6.4</TD><TD>14</TD><TD>7</TD><TD>7</TD><TD>7</TD><TD>7</TD></TR></TBODY></TABLE></CENTER><P>The samples were incubated for 4h at 37 °C. PCR Clean-Up is done, according to the protocol.</P><DL><DT>Date</DT><DD>08-09-2018</DD><DT>Operator</DT><DD>Susan Bouwmeester</DD></DL><H4>Nanodrop DNA Quantification</H4><P> 
    Nanodrop is done after restriction and PCR cleanup. The following isolates were quantified according to the Nanodrop DNA Quantification protocol. The concentration and purity of the DNA (260/280 and 260/230 ratios) were measured. 
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Isolate</TH><TH class="tableheaderadpbl">Concentration (ng/uL)</TH><TH class="tableheaderadpbl">260/280 ratio</TH><TH class="tableheaderadpbl">260/230 ratio</TH></TR><TR class="tableunevenadpbl"><TD>linker</TD><TD>3.3</TD><TD>1.7</TD><TD>0.2</TD></TR><TR class="tableevenadpbl"><TD>pSB1C3extranuc</TD><TD>39.9</TD><TD>1.8</TD><TD>0.9</TD></TR><TR class="tableunevenadpbl"><TD> Tn5</TD><TD>48.9</TD><TD>1.8</TD><TD>1.2</TD></TR><TR class="tableunevenadpbl"><TD>Pycduet</TD><TD>3.9</TD><TD>1.5</TD><TD>0.3</TD></TR><TR class="tableevenadpbl"><TD>pSB1C3</TD><TD>46.9</TD><TD>1.8</TD><TD>0.9</TD></TR><TR class="tableunevenadpbl"><TD> Fusionbb</TD><TD>54.1</TD><TD>1.8</TD><TD>1.3</TD></TR><TR class="tableunevenadpbl"><TD>Fusionduet1</TD><TD>6.2</TD><TD>1.5</TD><TD>0.3</TD></TR><TR class="tableevenadpbl"><TD>oldpSB1C3</TD><TD>46.9</TD><TD>1.8</TD><TD>0.9</TD></TR><TR class="tableunevenadpbl"><TD> T7p.Tn5.T7t</TD><TD>16.5</TD><TD>1.7</TD><TD>0.4</TD></TR><TR class="tableunevenadpbl"><TD>Fusionduet2</TD><TD>13.1</TD><TD>1.7</TD><TD>0.8</TD></TR></TBODY></TABLE></CENTER><DL><DT>Date</DT><DD>08/09/2018</DD><DT>Operator</DT><DD>Susan Bouwmeester</DD></DL><H4>Colony PCR</H4><P>After carrying out the transformation protocol on 06/09/2018, the plates were left for overnight incubation. In total five colonies were observed.</P><P>Colony PCR was done to verify propagation of pSB1C3-Tn5 and pSB1C3-T7p.Tn5.T7t after transformation from 06/09/2018. This plasmid contains Tn5 or T7.Tn5.T7, which can be checked with primers VF2 and VR
</P><CENTER><TABLE><TBODY><TR class="tableunevenadpbl"><TD>Template DNA</TD><TD>pSB1C3-Tn5</TD><TD>pSB1C3-T7p.Tn5.T7t</TD></TR><TR class="tableevenadpbl"><TD>Forward primer</TD><TD>VF2</TD><TD>VF2</TD></TR><TR class="tableunevenadpbl"><TD>Reverse primer</TD><TD>VR</TD><TD>VR</TD></TR><TR class="tableunevenadpbl"><TD>Expected amplicon size (bp)</TD><TD>~1700bp</TD><TD>~1800bp</TD></TR></TBODY></TABLE></CENTER><P>The thermocycler programme was installed to have an extension time of 2:20 and an annealing temperature of 52 °C. After the programme was finished, samples were kept at 4°C.</P><DL><DT>Date</DT><DD>08/09/2018</DD><DT>Operator</DT><DD>Susan Bouwmeester</DD></DL><H4>DNA Gel Electrophoresis</H4><P>The products of the colony PCR were run on a 0.8% agarose gel in TAE buffer. 5μL was pipetted into each lane. The gel was imaged using a gel documentation system with UV-light.</P> s
    
<B>Figure: Colony PCR pSB1C3-Tn5 and pSB1C3-T7p.Tn5.T7t</B> 0.8% agarose. Expected hight ~1700 bp. Lane 1: 500bp ladder; Lane 2: Colony1 (Tn5); Lane 3: Colony2 (Tn5); Lane 4: Colony 3 (Tn5); Lane 5: Colony 4 (Tn5); Lane 6: Colony 5 (T7p.Tn5.T7t) <DL><DT>Date</DT><DD>08/09/2018</DD><DT>Operator</DT><DD>Susan Bouwmeester</DD><H4>Gibson Assembly</H4><P>According to the displayed reaction mixture composition, Gibson Assembly was done to assemble the linker and pSB1C3en to obtain pSB1C3-linker, the fusion protein with pSB1C3 to obtain pSB1C3-fusion.
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL) pSB1C3-linker</TH><TH class="tableheaderadpbl">Volume (uL) pSB1C3-fusion</TH></TR><TR class="tableunevenadpbl"><TD>Gibson Assembly Mastermix 2X</TD><TD>10</TD><TD>10</TD></TR><TR class="tableevenadpbl"><TD>Fragment</TD><TD>2uL (50 ng)</TD><TD>6.8 uL(2000 ng)</TD></TR><TR class="tableunevenadpbl"><TD>Vector</TD><TD>2.7 uL (1000 ng)</TD><TD>2.3 uL (2107.9 ng)</TD></TR><TR class="tableunevenadpbl"><TD>MilliQ</TD><TD>5.3 uL</TD><TD>1.5 uL</TD></TR></TBODY></TABLE></CENTER><P>The samples were incubated at 50°C for 20 minutes, after which they were kept on ice for subsequent transformation.</P><DL><DT>Date</DT><DD>8/9/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>DNA Gel Electrophoresis</H4><P>PCR products for fusion protein construction were run on a 0.8% agarose gel in TAE buffer. 5μL was pipetted into each lane. The gel was imaged using a gel documentation system with UV-light.</P><P>The gel shows:</P><UL><LI>connecting Gblock 1 &amp; 2 did not work (lanes 2-17)</LI><LI>Amplifying gblock 3 &amp; 4, connected, did work without byproducts (lanes 18-34)</LI><LI>amplifying Gblock 1 on itself did not work (lanes 35-43)</LI><LI>pACYC backbone for cloning with Gblock 1 &amp; 2 was succesful (lanes 44-51)</LI><LI>pACYC backbone for cloning with Gblock 3 &amp; 4 was only successful in 1 sample (lane 52)</LI></UL><DL><DT>Date</DT><DD>8/9/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>PCR clean-up</H4><P>
    Based on the obtained gel, it was decided to PCR clean-up amplicons &quot;Gblock3&amp;4 gibson pACYC&quot; and &quot;pACYC backbone for Gblock 3&amp;4&quot; was subjected to PCR Clean-Up, according to the protocol. Elution was done in 40 µL Milli-Q.
</P><DL><DT>Date</DT><DD>8/9/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>Nanodrop DNA Quantification</H4><P> 
    The following isolates were quantified according to the Nanodrop DNA Quantification protocol. The concentration and purity of the DNA (260/280 and 260/230 ratios) were verified to be around 1.8~1.9. 
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Isolate</TH><TH class="tableheaderadpbl">Concentration (ng/uL)</TH></TR><TR class="tableunevenadpbl"><TD>Gblock3&amp;4 gibson pACYC</TD><TD>455</TD></TR><TR class="tableevenadpbl"><TD>Gblock3&amp;4 gibson pACYC</TD><TD>294</TD></TR><TR class="tableunevenadpbl"><TD>pACYC backbone for Gblock 3&amp;4</TD><TD>113</TD></TR></TBODY></TABLE></CENTER><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#SundaySCS1">Sunday, 09/09/2018</BUTTON><DIV id="SundaySCS1" class="modal fade" role="dialog"><DIV class="modal-dialog">S
    <DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>09/09/2018</DD><DT>Operator</DT><DD>Susan Bouwmeester</DD></DL><H4>Overnight culture </H4><P>An overnight culture of the five colonies as result of the transformation 05/09 and 06/09 is started. 3mL LBmedium with Cam resistance is used. Cultures overnight 37°C.</P><DL><DT>Date</DT><DD>9/9/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>PCR clean-up</H4><P>
    PCR products &quot;Gblock1&amp;2 Gibson pSB1C3&quot;, &quot;Gblock3&amp;4 Gibson pSB1C3&quot; and &quot;pACYC (for cloning with Gblock 1&amp;2)&quot; (all dating from 7/9/2018) were subjected to PCR Clean-Up, according to the protocol. Isolation was done by gel cut-out, due to byproducts visible on the gel (see 8/9/2018Elution was done in 40 µL Milli-Q.
</P><DL><DT>Date</DT><DD>9/9/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>Nanodrop DNA Quantification</H4><P> 
    The following isolates were quantified according to the Nanodrop DNA Quantification protocol. The concentration and purity of the DNA (260/280 and 260/230 ratios) were measured. 
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Isolate</TH><TH class="tableheaderadpbl">Concentration (ng/uL)</TH><TH class="tableheaderadpbl">260/280 ratio</TH><TH class="tableheaderadpbl">260/230 ratio</TH></TR><TR class="tableunevenadpbl"><TD>Gblock1&amp;2 Gibson pSB1C3</TD><TD>16</TD><TD>1.35</TD><TD>0.75</TD></TR><TR class="tableevenadpbl"><TD>Gblock1&amp;2 Gibson pSB1C3</TD><TD>15</TD><TD>1.8</TD><TD>0.6</TD></TR><TR class="tableunevenadpbl"><TD>Gblock1&amp;2 Gibson pSB1C3</TD><TD>13.5</TD><TD>2</TD><TD>0.31</TD></TR><TR class="tableevenadpbl"><TD>Gblock3&amp;4 Gibson pSB1C3</TD><TD>61</TD><TD>1.9</TD><TD>0.9</TD></TR><TR class="tableunevenadpbl"><TD>Gblock3&amp;4 Gibson pSB1C3</TD><TD>69</TD><TD>1.9</TD><TD>0.2</TD></TR><TR class="tableevenadpbl"><TD>pACYC (for cloning with Gblock 1&amp;2)</TD><TD>108.8</TD><TD>1.9</TD><TD>1.4</TD></TR><TR class="tableunevenadpbl"><TD>pACYC (for cloning with Gblock 1&amp;2)</TD><TD>145</TD><TD>1.9</TD><TD>1.5</TD></TR><TR class="tableevenadpbl"><TD>pACYC (for cloning with Gblock 1&amp;2)</TD><TD>163</TD><TD>1.9</TD><TD>1.6</TD></TR></TBODY></TABLE></CENTER><DL><DT>Date</DT><DD>9/9/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>High Fidelity PCR</H4><P> To amplify Gblock1&amp;2 Gibson pSB1C3 and Gblock3&amp;4 Gibson pSB1C3 (both ~6kb), high fidelity PCR was done using PCR cleanup isolates as template DNA. PCR mixtures were prepared, according to the High Fidelity PCR protocol.
    
    
Primer combination(s) and expected fragment(s)

</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Template</TH><TH class="tableheaderadpbl">Fw primer</TH><TH class="tableheaderadpbl">Rv primer</TH><TH class="tableheaderadpbl">expected size (bp) </TH></TR><TR class="tableunevenadpbl"><TD>Gblock1&amp;2 Gibson pSB1C3</TD><TD>054</TD><TD>057</TD><TD>~6kb</TD></TR><TR class="tableevenadpbl"><TD>Gblock3&amp;4 Gibson pSB1C3</TD><TD>055</TD><TD>056</TD><TD>~6kb</TD></TR></TBODY></TABLE></CENTER><P>The thermocycler programme was installed to have an extension time of 4 minutes and an annealing temperature of 65°C. After the programme was finished, samples were kept at 4 °C.</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV></DL></DIV></DIV></DIV></DIV></DIV></UL></FORM><FORM><LABEL for="WeekSCS2" class="subdropdown cadpbl">Week 2 10/09/2018 - 16/09/2018</LABEL><UL class="submenu"><DIV class="subcontent"><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#MondaySCS2">Monday, 10/09/2018</BUTTON><DIV id="MondaySCS2" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>10/9/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>DNA Gel Electrophoresis</H4><P>PCR products &quot;Gblock1&amp;2 Gibson pSB1C3&quot; and &quot;Gblock3&amp;4 Gibson pSB1C3&quot; (9/9/2018) were run on a 0.8% agarose gel in TAE buffer. 5μL was pipetted into each lane. The gel was imaged using a gel documentation system with UV-light.</P><P>According to this gel, neither of the two fragments is properly amplified.</P><DL><DT>Date</DT><DD>10/9/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>Gibson Assembly</H4><P>According to the displayed reaction mixture composition, Gibson Assembly was done to assemble Gblock 3&amp;4 (insert) into pACYC (vector).
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Amount</TH></TR><TR class="tableunevenadpbl"><TD>Gibson Assembly Mastermix 2X</TD><TD>10µl</TD></TR><TR class="tableevenadpbl"><TD>Vector</TD><TD>~0.35pmol</TD></TR><TR class="tableunevenadpbl"><TD>Fragment</TD><TD>~0.35pmol</TD></TR><TR class="tableunevenadpbl"><TD>MilliQ</TD><TD>until 20µl</TD></TR></TBODY></TABLE></CENTER><P>The samples were incubated at 50°C for 20 minutes, after which they were kept on ice for subsequent transformation.</P><DL><DT>Date</DT><DD>10/9/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4> Transformation of chemically competent cells </H4><P>The following transformations were carried out according to the protocol for transformation of chemically competent cells:
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Strain</TH><TH class="tableheaderadpbl">Plasmid &amp; size (bp) </TH><TH class="tableheaderadpbl">Cloning method</TH><TH class="tableheaderadpbl">Volume added (uL)</TH><TH class="tableheaderadpbl">Medium on which was plated</TH></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I>DH5α</TD><TD>pACYC-Gblock3&amp;4</TD><TD>Gibson Assembly</TD><TD>20</TD><TD>LB+Cam</TD></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I>DH5α</TD><TD>-</TD><TD>-</TD><TD>-</TD><TD>LB+Cam</TD></TR></TBODY></TABLE></CENTER><P>200 uL LB medium was added as recovery medium.
</P><DL><DT>Date</DT><DD>10/9/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>Electroporation</H4><P>The following transformations were carried out according to the protocol for electroporation:</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Strains</TH><TH class="tableheaderadpbl">Plasmid/fragment</TH><TH class="tableheaderadpbl">Amount</TH><TH class="tableheaderadpbl">Plate medium</TH></TR><TR class="tableunevenadpbl"><TD><EM>E. coli </EM>BL21DE3+pACYCdxCas9Tn5<I>lacZ</I>gRNA</TD><TD>MEpKantME</TD><TD>200ng</TD><TD>LB+X-gal+IPTG+Cam+Kan</TD></TR><TR class="tableevenadpbl"><TD><EM>E. coli</EM> BL21DE3+pACYCdxCas9Tn5<I>lacZ</I>gRNA</TD><TD>pKant</TD><TD>200ng</TD><TD>LB+X-gal+IPTG+Cam+Kan</TD></TR><TR class="tableunevenadpbl"><TD><EM>E. coli</EM> BL21DE3+pACYCdxCas9Tn5<I>lacZ</I>gRNA</TD><TD>-</TD><TD>-</TD><TD>LB+X-gal+IPTG+Cam+Kan</TD></TR></TBODY></TABLE></CENTER><P>200 μl LB medium+0.1mM IPTG was added as recovery medium.
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#WednesdaySCS1">Wednesday, 12/09/2018</BUTTON><DIV id="WednesdaySCS2" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>12/09/2018</DD><DT>Operator</DT><DD>Susan Bouwmeester</DD></DL><H4>High Fidelity PCR</H4><P> To amplify dxCas9 (~4207bp), high fidelity PCR was done using the fusionbb as template DNA and fw_bb_dxcas9 and rv_bb_dxcas9_2 as forward and reverse primers respectively. 6 50µL PCR mixtures were prepared, according to the High Fidelity PCR protocol.
    </P><P>The thermocycler programme was installed to have an extension time of 2:30 and an annealing temperature of 63 °C. After the programme was finished, samples were kept at 12 °C.</P><DL><DT>Date</DT><DD>12/09/18</DD><DT>Operator</DT><DD>Susan Bouwmeester</DD></DL><H4>Colony PCR</H4><P>Colony PCR was done to verify propagation of Tn5 and T7p.Tn5.T7t after transformation from 06/09 and 07/09 and plasmid isolation from 10/9.  Primers VF2 and VR were used. 
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL)</TH></TR><TR class="tableunevenadpbl"><TD>Template DNA</TD><TD>0.5</TD></TR><TR class="tableevenadpbl"><TD>Forward primer</TD><TD>0.6</TD></TR><TR class="tableunevenadpbl"><TD>Reverse primer</TD><TD>0.6</TD></TR><TR class="tableunevenadpbl"><TD>Expected amplicon size (bp)</TD><TD>~1700 bp</TD></TR></TBODY></TABLE></CENTER><P>5 tubes were prepared for colony PCR, containing purified plasmids as template DNA. The thermocycler programme was installed to have an extension time of 2:30 and an annealing temperature of 52 °C. After the programme was finished, samples were kept at 12°C.</P><DL><DT>Date</DT><DD>12/09/18</DD><DT>Operator</DT><DD>Susan Bouwmeester</DD></DL><H4>Colony PCR</H4><P>Colony PCR was done to verify propagation of the fusion protein for four colonies.  Primers dxcas9_mut2 and dxcas9_mut4 were used. Expected size is 1470bp.
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL)</TH></TR><TR class="tableunevenadpbl"><TD>Template DNA</TD><TD>0.5</TD></TR><TR class="tableevenadpbl"><TD>Forward primer</TD><TD>0.6</TD></TR><TR class="tableunevenadpbl"><TD>Reverse primer</TD><TD>0.6</TD></TR><TR class="tableunevenadpbl"><TD>Expected amplicon size (bp)</TD><TD>~1470 bp</TD></TR></TBODY></TABLE></CENTER><P>8 tubes were prepared for colony PCR, containing directly the colony or first boyled colony as template DNA. The thermocycler programme was installed to have an extension time of 2 minutes and an annealing temperature of 50 °C. After the programme was finished, samples were kept at 12°C.</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#ThursdaySCS2">Thursday, 13/09/2018</BUTTON><DIV id="ThursdaySCS2" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>13/09/2018</DD><DT>Operator</DT><DD>Susan Bouwmeester</DD></DL><H4> Transformation of chemically competent cells </H4><P>The following transformations were carried out according to the protocol for transformation of chemically competent cells. All transformations were previous made samples. 
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Strain</TH><TH class="tableheaderadpbl">Plasmid &amp; size (bp) </TH><TH class="tableheaderadpbl">Cloning method</TH><TH class="tableheaderadpbl">Volume added (uL)</TH><TH class="tableheaderadpbl">Medium on which was plated</TH></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I>DH5α</TD><TD>[pSB1C3-T7p.Tn5.T7t] (3605bp)</TD><TD>Restriction/ligation</TD><TD>18</TD><TD>LB Cam</TD></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I>DH5α</TD><TD>[pSB1C3-linker1] (~2000bp)</TD><TD>Restriction/ligation</TD><TD>18</TD><TD>LB Cam</TD></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I>DH5α</TD><TD>[pSB1C3-linker2] (~2000bp)</TD><TD>Restriction/ligation</TD><TD>18</TD><TD>LB Cam</TD></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I>DH5α</TD><TD>[pSB1C3-fusion1] (7699bp)</TD><TD>Restriction/ligation</TD><TD>18</TD><TD>LB Cam</TD></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I>DH5α</TD><TD>[pSB1C3-fusion2] (7699bp)</TD><TD>Restriction/ligation</TD><TD>18</TD><TD>LB Cam</TD></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I>DH5α</TD><TD>[pSB1C3-Tn5] (3499bp)</TD><TD>Restriction/ligation</TD><TD>18</TD><TD>LB Cam</TD></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I>DH5α</TD><TD>[pSB1C3-fusion] (7699)</TD><TD>Gibson Assembly</TD><TD>18</TD><TD>LB Cam</TD></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I>DH5α</TD><TD>[pSB1C3-linker] (~2000bp)</TD><TD>Gibson Assembly</TD><TD>18</TD><TD>LB Cam</TD></TR></TBODY></TABLE></CENTER><P>950 uL LB medium was added as recovery medium.
</P><DL><DT>Date</DT><DD>13/09/2018</DD><DT>Operator</DT><DD>Susan Bouwmeester</DD></DL><H4>DNA Gel Electrophoresis</H4><P> dxCas9 amplification product was run on a 0.8% agarose gel in TEA buffer. 5μL was pipetted into each lane. The gel was imaged using a gel documentation system with UV-light.</P><B>Figure dxCas9 amplification</B> 0.8% Agarose; Lane 1: 500bp ladder; Lane 2-7: PCR product dxCas9 amplification <P> Conclusion: dxCas9 is produced, but not pure. Gelcutout is needed. </P><DL><DT>Date</DT><DD>13/09/2018</DD><DT>Operator</DT><DD>Susan Bouwmeester</DD></DL><H4>DNA Gel Electrophoresis</H4><P> PCR Tn5 product from pure plasmid was run on a 0.8% agarose gel in TEA buffer. 5μL was pipetted into each lane. The gel was imaged using a gel documentation system with UV-light.</P><B>Figure PCR Tn5 and T7p.Tn5.T7t in pSB1C3 on pure plasmid</B> 0.8% agarose. Lane 1:Tn5 Colony 1; Lane 2: Tn5 Colony 2; Lane 3: Tn5 Colony 3; Lane 4: Tn5 Colony 4; Lane 5: T7p.Tn5.T7t Colony 5; Lane 6: 500bp ladder<P>No product, no correct Tn5 or T7p.Tn5.T7t integration</P><DL><DT>Date</DT><DD>13/09/2018</DD><DT>Operator</DT><DD>Susan Bouwmeester</DD></DL><H4>DNA Gel Electrophoresis</H4><P> Colony PCR of four colonies after transformation with pSB1C-fusion was run on a 0.8% agarose gel in TEA buffer. 5μL was pipetted into each lane. The gel was imaged using a gel documentation system with UV-light.</P><B>Figure Colony PCR pSB1C3-fusion</B>0.8% agarose. Expected size ~1500 bp. Lane 1: 500bp ladder; Lane 2: Fusion Colony 1; Lane 3: Fusion Colony 2; Lane 4: Fusion Colony 3; Lane 5: Fusion Colony 4; Lane 6: Fusion Colony 1 preboiled; Lane 7: Fusion Colony 2 preboiled; Lane 8: Fusion Colony 3 preboiled; Lane 9: Fusion Colony 4 preboiled; <P>Fusion Colony 4 preboiled have product. Start overnight culture in 3mL with cam. of this colony.</P><DL><DT>Date</DT><DD>13/09</DD><DT>Operator</DT><DD>Susan Bouwmeester</DD></DL><H4>High Fidelity PCR</H4><P> To amplify fusionbb (~6000bp) and fusionduet (~6000) , high fidelity PCR was done using fusionbb1 as template DNA and fw_BB_dxCas9 and rv_BB_Tn5 forward and reverse primers respectively for fusionbb and fw_3_dxCas9-L-Tn5_Gblock and rv_4_dxCas9-L-Tn5_Gblock forward and reverse primers respctively for fusionduet. 16 50µL PCR mixtures were prepared, 8 for fusionbb and 8 for fusionduet, according to the High Fidelity PCR protocol.
    
    
Primer combination(s) and expected fragment(s)

</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Sample</TH><TH class="tableheaderadpbl">Fw primer</TH><TH class="tableheaderadpbl">Rv primer</TH><TH class="tableheaderadpbl">expected fragment</TH><TH class="tableheaderadpbl">expected size (bp) </TH></TR><TR class="tableunevenadpbl"><TD>fusionbb</TD><TD>fw_bb_dxCas9</TD><TD>rv_bb_Tn5</TD><TD>fusionbb</TD><TD>~6000bp</TD></TR><TR class="tableunevenadpbl"><TD>fusionduet</TD><TD>fw_3_dxCas9-L-Tn5_Gblock</TD><TD>rv_4_dxCas9-L-Tn5_Gblock</TD><TD>fusionduet</TD><TD>~6000bp</TD></TR></TBODY></TABLE></CENTER><P>The thermocycler programme was installed to have an extension time of 4 minutes and an annealing temperature of 60 °C. After the programme was finished, samples were kept at 4 °C.</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#FridaySCS2">Friday, 14/09/2018</BUTTON><DIV id="FridaySCS2" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>14/09</DD><DT>Operator</DT><DD>Susan Bouwmeester</DD></DL><H4>High Fidelity PCR</H4><P> To amplify pSB1C3 linear , high fidelity PCR was done using linear pSB1C3 as template DNA and fw_BB_pSB1C3 and rv_BB_pSB1C3 forward and reverse primers respectively. 8 50µL PCR mixtures were prepared, according to the High Fidelity PCR protocol.
    
    
Primer combination(s) and expected fragment(s)

</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Sample</TH><TH class="tableheaderadpbl">Fw primer</TH><TH class="tableheaderadpbl">Rv primer</TH><TH class="tableheaderadpbl">expected fragment</TH><TH class="tableheaderadpbl">expected size (bp) </TH></TR><TR class="tableunevenadpbl"><TD>pSB1C3</TD><TD>fw_bb_pSB1C3</TD><TD>rv_bb_pSB1C3</TD><TD>pSB1C3</TD><TD>~2000bp</TD></TR></TBODY></TABLE></CENTER><P>The thermocycler programme was installed to have an extension time of 2:30 minutes and an annealing temperature of 60 °C. After the programme was finished, samples were kept at 4 °C.</P><DL><DT>Date</DT><DD>14/09/2018</DD><DT>Operator</DT><DD>Susan Bouwmeester</DD></DL><H4>DNA Gel Electrophoresis</H4><P>The products of the PCR 13/09 to amplify fusionbb and fusion duet was run on a 0.8 % agarose gel in TAE buffer. 3μL was pipetted into each lane. The gel was imaged using a gel documentation system with UV-light.</P><B>Figure PRC product fusionbb</B>  Lane 1: 500bp ladder; Lane 2-9: PCR product fusion bb <P> There is fusionbb made. There are a lot of side products made. There must be done a gelcut.</P><B>Figure PRC product fusionduet</B>  Lane 1: 500bp ladder; Lane 2-9: PCR product fusionduet <P> There is no fusionduet made</P><H4>DNA gel purification</H4><P>The fusionbb has a size of ~6000bp, but the PCR samples were contaminated with by-products. All the PCR product is loaded on a 0.8% agarose gel. DNA purification from agarose gel conform protocol
Elution was done in 30 µL Milli-Q.
</P><DL><DT>Date</DT><DD>14/09/2018</DD><DT>Operator</DT><DD>Susan Bouwmeester</DD></DL><H4>Plasmid Isolation</H4><P> 
    Overnight starter cultures of Tn5, T7p.Tn5.T7t, fusion are subjected to plasmid isolation following the Plasmid Isolation Protocol. Final elution was done with 30 μL of pre-warmed MilliQ.
</P><DL><DT>Date</DT><DD>14/09/2018</DD><DT>Operator</DT><DD>Susan Bouwmeester</DD></DL><H4>Nanodrop DNA Quantification</H4><P> 
    The following isolates were quantified according to the Nanodrop DNA Quantification protocol. The concentration and purity of the DNA (260/280 and 260/230 ratios) were measured. 
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Isolate</TH><TH class="tableheaderadpbl">Concentration (ng/uL)</TH><TH class="tableheaderadpbl">260/280 ratio</TH><TH class="tableheaderadpbl">260/230 ratio</TH></TR><TR class="tableunevenadpbl"><TD>Tn5 Col1</TD><TD>23.2</TD><TD>1.57</TD><TD>0.53</TD></TR><TR class="tableevenadpbl"><TD>Tn5 Col3</TD><TD>11.8</TD><TD>1.69</TD><TD>0.49</TD></TR><TR class="tableunevenadpbl"><TD>Tn5 Col4</TD><TD>133.5</TD><TD>1.52</TD><TD>0.58</TD></TR><TR class="tableunevenadpbl"><TD>T7p.Tn5.T7t Col5</TD><TD>118</TD><TD>1.7</TD><TD>1</TD></TR><TR class="tableunevenadpbl"><TD>T7p.Tn5.T7t Col5</TD><TD>118</TD><TD>1.7</TD><TD>1</TD></TR><TR class="tableunevenadpbl"><TD>T7p.Tn5.T7t OldCol6</TD><TD>315</TD><TD>1.87</TD><TD>1.9</TD></TR><TR class="tableevenadpbl"><TD>Fusion Col1</TD><TD>211</TD><TD>1.68</TD><TD>0.92</TD></TR><TR class="tableunevenadpbl"><TD>Fusion Col2</TD><TD>283.3</TD><TD>1.66</TD><TD>0.92</TD></TR><TR class="tableunevenadpbl"><TD>Fusin Col3</TD><TD>357.3</TD><TD>1.6</TD><TD>0.8</TD></TR><TR class="tableunevenadpbl"><TD>Fusion Col4</TD><TD>358.1</TD><TD>1.6</TD><TD>0.7</TD></TR><TR class="tableunevenadpbl"><TD>Fusion Col5</TD><TD>121.4</TD><TD>1.8</TD><TD>1.65</TD></TR></TBODY></TABLE></CENTER></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV></DIV></UL></FORM><FORM><LABEL for="WeekSCS3" class="subdropdown cadpbl">Week 3 17/09/2018 - 23/09/2018</LABEL><UL class="submenu"><DIV class="subcontent"><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#MondaySCS3">Monday, 17/09/2018</BUTTON><DIV id="MondaySCS3" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>17/09</DD><DT>Operator</DT><DD>Susan Bouwmeester</DD></DL><H4>Gibson Assembly</H4><P>According to the displayed reaction mixture composition, Gibson Assembly was done to assemble linker and pSB1C3 to obtain pSB1C3-linker; dxCas9 and pSB1C3 to obtain pSB1C3-dxCas9; fusionduet and PACYCduet to obtain PACYCduet-fusion .
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL)</TH></TR><TR class="tableunevenadpbl"><TD>Gibson Assembly Mastermix 2X</TD><TD>10</TD><TD>10</TD><TD>10</TD></TR><TR class="tableevenadpbl"><TD>Vector</TD><TD>pSB1C3en, 3 (1100 ng)</TD><TD>pSB1C3, 1.9 (1400 ng)</TD><TD>PACYCduet, 2.7 (2700 ng)</TD></TR><TR class="tableunevenadpbl"><TD>Fragment</TD><TD>linker, 3.6 (147 ng)</TD><TD>dxCas9, 7 (1400 ng)</TD><TD>fusionduet, 4.4 (2000)</TD></TR><TR class="tableunevenadpbl"><TD>MilliQ</TD><TD>3.4</TD><TD>1.1</TD><TD>2.9</TD></TR></TBODY></TABLE></CENTER><P>The samples were incubated at 50°C for 60 minutes, after which they were kept on ice for subsequent transformation.</P><DL><DT>Date</DT><DD>17/09/2018</DD><DT>Operator</DT><DD>Susan Bouwmeester</DD></DL><H4> Transformation of chemically competent cells </H4><P>The following transformations were carried out according to the protocol for transformation of chemically competent cells:
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Strain</TH><TH class="tableheaderadpbl">Plasmid &amp; size (bp) </TH><TH class="tableheaderadpbl">Cloning method</TH><TH class="tableheaderadpbl">Volume added (uL)</TH><TH class="tableheaderadpbl">Medium on which was plated</TH></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I>DH5α</TD><TD>[pSB1C3-linker] (~2000bp) </TD><TD>Gibson Assembly</TD><TD>2</TD><TD>LB Cam</TD></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I>DH5α</TD><TD>[pSB1C3-linker] (~2000bp) </TD><TD>Gibson Assembly</TD><TD>18</TD><TD>LB Cam</TD></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I>DH5α</TD><TD>[pSB1C3-dxCas9] (6217bp) </TD><TD>Gibson Assembly</TD><TD>2</TD><TD>LB Cam</TD></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I>DH5α</TD><TD>[pSB1C3-dxCas9] (6217bp) </TD><TD>Gibson Assembly</TD><TD>18</TD><TD>LB Cam</TD></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I>DH5α</TD><TD>[PACYC-fusion] (9432bp) </TD><TD>Gibson Assembly</TD><TD>2</TD><TD>LB Cam</TD></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I>DH5α</TD><TD>[PACYC-fusion] (9432bp)  </TD><TD>Gibson Assembly</TD><TD>18</TD><TD>LB Cam</TD></TR></TBODY></TABLE></CENTER><P>950 uL LB medium was added as recovery medium.
</P><DL><DT>Date</DT><DD>17/09/18</DD><DT>Operator</DT><DD>Susan Bouwmeester</DD></DL><H4>Colony PCR</H4><P>Colony PCR was done to verify propagation of pSB1C3-fusion and pSB1c3-Tn5.  This plasmid contains pSB1C3 and or the fusion protein or Tn5, which both can be checked with primers VR and VF2
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL)</TH></TR><TR class="tableevenadpbl"><TD>Template</TD><TD>Colony pSB1C3-fusion</TD><TD>Colony pSB1C3-Tn5</TD></TR><TR class="tableevenadpbl"><TD>Forward primer</TD><TD>0.6</TD><TD>0.6</TD></TR><TR class="tableunevenadpbl"><TD>Reverse primer</TD><TD>0.6</TD><TD>0.6</TD></TR><TR class="tableunevenadpbl"><TD>Expected amplicon size (bp)</TD><TD>7699</TD><TD>3499</TD></TR></TBODY></TABLE></CENTER><P>12 tubes were prepared for colony PCR. The thermocycler programme was installed to have an extension time of 2:30 for pSB1C3-Tn5 and 7:00 for pSB1C-fusion and an annealing temperature of 60 °C. After the programme was finished, samples were kept at 12°C.</P><H4>DNA Gel Electrophoresis</H4><P>The PCR product of the colony PCR was run on a 0.8% agarose gel in TAE buffer. 5μL was pipetted into each lane. The gel was imaged using a gel documentation system with UV-light.</P><B>Figure Colony PCR pSB1C3-fusion</B> expected size is 7699bp. 0.8% agarose. Lane 1: 500bp ladder; Lane 2: Fusion Col1; Lane 3: Fusion Col2; Lane 4: Fusion Col3; Lane 5: Fusion Col4; Lane 6: Fusion Col5; Lane 7: Negative control, GFP; Lane 8: Negative control, no template  <B>Figure Colony PCR pSB1C3-Tn5</B> expected size is 3499bp. 0.8% agarose. Lane1: Tn5 Col1; Lane 2: Tn5 Col2; Lane 3: Tn5 Col3; Lane 4: Tn5 Col4; Lane 5: Tn5 Col5; Lane 6: Negative control, GFP; Lane 7: Negative control, no template  <DL><DT>Date</DT><DD>17/09/18</DD><DT>Operator</DT><DD>Susan Bouwmeester</DD></DL><H4>High Fidelity PCR</H4><P> To eliminate the forbidden sites, high fidelity PCR was done using PACYCduet-fusionfs as template DNA and fw_dxCas9_mut4_BL and rv_dxCas9_mut4_BL as forward and reverse phosphorylated primers respectively. This makes the product compitable for blunt end ligation. 50µL PCR mixture were prepared, according to the High Fidelity PCR protocol. Expected size of the product PACYCduet-fusion-BL4 is 9432 bp. 
    </P><P>The thermocycler programme was installed to have an extension time of 5:00 and an annealing temperature of 63 °C. After the programme was finished, samples were kept at 12 °C.</P><DL><DT>Date</DT><DD>17/09/2018</DD><DT>Operator</DT><DD>Susan Bouwmeester</DD></DL><H4>Ligation</H4><P>In order to assemble cut fragments linker and pSB1C3en, Tn5 and pSB1C3, ligation was performed. This table gives an overview of which fragments were cut with which enzymes.

</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">DNA</TH><TH class="tableheaderadpbl">Size (bp) </TH><TH class="tableheaderadpbl">Cut with enzymes</TH></TR><TR class="tableunevenadpbl"><TD>Vector</TD><TD>~2000bp</TD><TD>EcoRI and PstI</TD></TR><TR class="tableevenadpbl"><TD>linker</TD><TD>~60bp</TD><TD>EcoRI and PstI</TD></TR><TR class="tableunevenadpbl"><TD>Tn5</TD><TD>~1500 bp</TD></TR></TBODY></TABLE></CENTER><P>A 29 uL ligation reaction mixture was prepared with the following composition</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume Linker1(uL) </TH><TH class="tableheaderadpbl">Volume Linker2</TH><TH class="tableheaderadpbl">Volume Tn51(uL) </TH><TH class="tableheaderadpbl">Volume Tn52</TH></TR><TR><TD>Ligase Buffer (10x) </TD><TD>2</TD><TD>2</TD><TD>2</TD><TD>2</TD></TR><TR class="tableevenadpbl"><TD>T4 DNA Ligase</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD>1</TD></TR><TR class="tableunevenadpbl"><TD>DNA vector (~100 ng) </TD><TD>pSB1C3en 2.5 uL</TD><TD>pSB1C3en 2.5 uL</TD><TD>pSB1C3 2.1 uL</TD><TD>pSB1C3 2.1 uL</TD></TR><TR class="tableunevenadpbl"><TD>DNA fragment</TD><TD>linker 2.6 uL (8.7ng)</TD><TD> linker 7.5 uL (26ng)</TD><TD>Tn5 4.4 uL</TD><TD>Tn5 10 uL</TD></TR><TR class="tableunevenadpbl"><TD>MilliQ</TD><TD>11.9 uL</TD><TD>7 uL</TD><TD>10.5 uL</TD><TD>4.9 uL</TD></TR></TBODY></TABLE></CENTER><P>The samples were incubated for overnight at room temperature for ligation. </P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#TuesdaySCS3">Tuesday, 18/09/2018</BUTTON><DIV id="TuesdaySCS3" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>18/09/2018</DD><DT>Operator</DT><DD>Susan Bouwmeester</DD></DL><H4>Restriction </H4><P>Restriction cloning was done for different constructs. The following table shows an overview of what fragment(s) was/were cut with what restriction enzyme(s).
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL) - pSB1C3 linear</TH><TH class="tableheaderadpbl">Volume (uL) - Tn5 </TH><TH class="tableheaderadpbl">Volume (uL) - fusionbb</TH><TH class="tableheaderadpbl">Volume (uL) - dxCas9</TH><TH class="tableheaderadpbl">Volume (uL) - linker </TH><TH class="tableheaderadpbl">Volume (uL) - pSB1C3en</TH><TH class="tableheaderadpbl">Volume (uL) - pSB1C3RFP</TH></TR><TR class="tableunevenadpbl"><TD>10x CutSmart buffer    </TD><TD>2</TD><TD>2</TD><TD>2</TD><TD>2</TD><TD>2</TD><TD>2</TD><TD>2</TD></TR><TR class="tableunevenadpbl"><TD>Fragment</TD><TD>2.7 (~2000 ng)</TD><TD>2 (~2000 ng)</TD><TD>10 (~600 ng)</TD><TD>10 (~2000 ng)</TD><TD>5 (~200 ng)</TD><TD>4.8 (~2000 ng)</TD><TD>17 (~2000 ng)</TD></TR><TR class="tableunevenadpbl"><TD>Enzyme (EcoRI)</TD><TD>0.5</TD><TD>0.5</TD><TD>0.5</TD><TD>0.5</TD><TD>0.5</TD><TD>0.5</TD><TD>0.5</TD></TR><TR class="tableunevenadpbl"><TD>Enzyme (PstI)</TD><TD>0.5</TD><TD>0.5</TD><TD>0.5</TD><TD>0.5</TD><TD>0.5</TD><TD>0.5</TD><TD>0.5</TD></TR><TR class="tableunevenadpbl"><TD>MilliQ</TD><TD> 14.3</TD><TD>15</TD><TD>7</TD><TD>7</TD><TD>13</TD><TD>12.2</TD><TD>0</TD></TR></TBODY></TABLE></CENTER><P>The samples were incubated for 2 hours in the incubator 37 °C and 2 hours in the shaker 37 °C , and followed by PCR cleanup. 

</P><H4>PCR clean-up</H4><P>
    PCR clean-up is according to the protocol for all the restricted framgents and PACYCduet-fusion-mut4. Elution was done in 30 µL Milli-Q.</P><DL><DT>Date</DT><DD>18/09/2018</DD><DT>Operator</DT><DD>Susan Bouwmeester</DD></DL><H4>DNA Gel Electrophoresis</H4><P>Restricted backbone was run on a 0.8% agarose gel in TAEuffer. 3μL was pipetted into each lane. The gel was imaged using a gel documentation system with UV-light.</P><B>Figure Restriction</B> Restriction check. 0.8% Agarose. Lane 1: pSB1C3-RFP restricted; Lane 2: linear pSB1C3 restriced; Lane 3: pSB1C3-RFP uncut; Lane 4: 500bp ladder <H4>Nanodrop DNA Quantification</H4><P> 
    After PCR cleanup, all te samples were measered with nanodrop quantification. Also two plasmids were measered. The following isolates were quantified according to the Nanodrop DNA Quantification protocol. The concentration and purity of the DNA (260/280 ratios) was measured. 
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Isolate</TH><TH class="tableheaderadpbl">Concentration (ng/uL)</TH><TH class="tableheaderadpbl">260/280 ratio</TH></TR><TR class="tableunevenadpbl"><TD>Tn5 plasmid 1</TD><TD>13.3</TD><TD>1.63</TD></TR><TR class="tableevenadpbl"><TD>Tn5 plasmid 2</TD><TD>100.5</TD><TD>1.86</TD></TR><TR class="tableunevenadpbl"><TD>linker</TD><TD>-</TD><TD>-</TD></TR><TR class="tableunevenadpbl"><TD>pSB1C3en</TD><TD>73.7</TD><TD>1.91</TD></TR><TR class="tableunevenadpbl"><TD>pSB1C3</TD><TD>24.4</TD><TD>2.00</TD></TR><TR class="tableevenadpbl"><TD>pSB1C3 RFP</TD><TD>10</TD><TD>1.99</TD></TR><TR class="tableunevenadpbl"><TD>dxCas9</TD><TD>34.8</TD><TD>1.84</TD></TR><TR class="tableunevenadpbl"><TD>Tn5 </TD><TD>73.7</TD><TD>1.91</TD></TR><TR class="tableevenadpbl"><TD>fusion</TD><TD>11.65</TD><TD>1.93</TD></TR><TR class="tableunevenadpbl"><TD>Bl4</TD><TD>68.5</TD><TD>1.96</TD></TR><TR class="tableunevenadpbl"><TD>Bl- </TD><TD>17.0</TD><TD>1.96</TD></TR></TBODY></TABLE></CENTER><DL><DT>Date</DT><DD>18/09/2018</DD><DT>Operator</DT><DD>Susan Bouwmeester</DD></DL><H4>Ligation</H4><P>In order to assemble cut several constructs, ligation was performed. </P><P>A 10ul ligation reaction mixture was prepared with the following composition</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">pSB1C3-linker Volume (uL) </TH><TH class="tableheaderadpbl">pSB1C3-Fusion Volume (uL) </TH><TH class="tableheaderadpbl">pSB1C3RFP-Fusion Volume (uL) </TH><TH class="tableheaderadpbl">pSB1C3-Tn5 Volume (uL) </TH><TH class="tableheaderadpbl">pSB1C3RFP-Tn5 Volume (uL) </TH><TH class="tableheaderadpbl">pSB1C3-dxCas9 Volume (uL) </TH><TH class="tableheaderadpbl">pSB1C3RFP-dxCas9 Volume (uL) </TH><TH class="tableheaderadpbl">PACYCduet-fusion-mut4 Volume (uL) </TH><TH class="tableheaderadpbl">PACYCduet-fusion (-) Volume (uL) </TH></TR><TR class="tableunevenadpbl"><TD>Ligase Buffer (10x) </TD><TD>2</TD><TD>2</TD><TD>2</TD><TD>2</TD><TD>2</TD><TD>2</TD><TD>2</TD><TD>2</TD><TD>2</TD></TR><TR class="tableevenadpbl"><TD>T4 DNA Ligase</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD>1</TD></TR><TR class="tableunevenadpbl"><TD>DNA vector </TD><TD>1.5 uL (~100ng) </TD><TD>8.5 uL (~200ng) </TD><TD>8.5 uL (~85ng)</TD><TD>4.2 uL (~100ng)</TD><TD>8.5 uL (~85ng)</TD><TD>6.4 uL (~150ng)</TD><TD>8.5 uL (~85ng)</TD><TD>1.5 uL (~100ng)</TD><TD>6.0 ul (~100ng)</TD></TR><TR class="tableunevenadpbl"><TD>DNA fragment </TD><TD>14 uL (~8.7ng)</TD><TD>8.5 uL (~100ng)</TD><TD>8.5 uL (~100ng)</TD><TD>3.4 uL (~225ng)</TD><TD>3.4 uL (~225ng)</TD><TD>10.6 uL (~100 ng)</TD><TD>8.5 uL (~100ng)</TD><TD>-</TD><TD>-</TD></TR><TR class="tableunevenadpbl"><TD>MilliQ</TD><TD>1.5 uL</TD><TD>0 uL</TD><TD>0 uL</TD><TD>9.4</TD><TD>5.1 uL</TD><TD>0 uL</TD><TD>0 uL</TD><TD>5.5 uL</TD><TD>11.0</TD></TR></TBODY></TABLE></CENTER><P>The samples were incubated overnight on room temperature for ligation. </P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#WednesdaySCS3">Wednesday, 19/09/2018</BUTTON><DIV id="WednesdaySCS3" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>19/09/2018</DD><DT>Operator</DT><DD>Susan Bouwmeester</DD></DL><H4>Colony PCR</H4><P>Colony PCR was done to verify propagation of pSB1C3-Tn5 after transformation in 2 colonies. This plasmid contains Tn5 and pSB1C3, which can be checked with primers VR and VF2.
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL)</TH></TR><TR class="tableevenadpbl"><TD>Forward primer</TD><TD>0.6</TD></TR><TR class="tableunevenadpbl"><TD>Reverse primer</TD><TD>0.6</TD></TR><TR class="tableunevenadpbl"><TD>Expected amplicon size (bp)</TD><TD>~1700 bp</TD></TR></TBODY></TABLE></CENTER><P>2 tubes were prepared for colony PCR. The thermocycler programme was installed to have an extension time of 2:30 and an annealing temperature of 52 °C. After the programme was finished, samples were kept at 12°C.</P><P> 
    Overnight starter cultures of PACYCduet-Fusion was subjected to plasmid isolation following the Plasmid Isolation Protocol. Final elution was done with 30 μL of pre-warmed MilliQ.
</P><DL><DT>Date</DT><DD>19/09/2018</DD><DT>Operator</DT><DD>Susan Bouwmeester</DD></DL><H4> Transformation of chemically competent cells </H4><P>Transformations were carried out according to the protocol for transformation of chemically competent cells. DNA used for the transformation is the result of overnight DNA Ligation of 17/09 and 18/09. 20uL ligation product is used. 
</P><H4>DNA Gel Electrophoresis</H4><P>The product of the colony-PCR was run on a 0.8% agarose gel in TAE buffer. 5μL was pipetted into each lane. The gel was imaged using a gel documentation system with UV-light.</P><B>Figure colonyPCR</B> Result colony PCR. Expected size ~1700bp. 0.8% Agarose. Lane 1: 500bp ladder; Lane 2: Colony 1, Tn5 8/9; Lane 3: Colony 2, Tn5 13/9 </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#ThursdaySCS3">Thursday, 20/09/2018</BUTTON><DIV id="ThursdaySCS3" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>20/09</DD><DT>Operator</DT><DD>Susan Bouwmeester</DD></DL><H4>DNA Gel Electrophoresis</H4><P>The different fragments were run on a 0.8% agarose gel in TAE buffer. 3μL was pipetted into each lane. The gel was imaged using a gel documentation system with UV-light.</P><B>Figure</B> Different fragments. 0.8% Agarose. Lane 1: 500 bp ladder; Lane 2: product of blunt end ligation PACYCduet-fusion-mut4 19/9 (plasmid); Lane 3: product of blunt end ligation 19/9 negative control (plasmid); Lane 4: 500bp ladder; Lane 5: pSB1C3-RFP (plasmid); Lane 6: pSB1C3 linear, expected size ~2000bp; Lane 7: pSB1C3en restricted with EcoRI and PstI<H4> Transformation Results </H4><P> After carrying out the transformation protocol on 19/09, the plates were left for overnight incubation. After ~14-18hr incubation, the following results were observed: </P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Strain</TH><TH class="tableheaderadpbl">DNA </TH><TH class="tableheaderadpbl">Medium</TH><TH class="tableheaderadpbl">Result</TH></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I> DH5α  </TD><TD>pSB1C3-linker </TD><TD>LB+Cam  </TD><TD>no growth</TD></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I> DH5α  </TD><TD>pSB1C3-Fusion </TD><TD>LB+Cam  </TD><TD>no growth</TD></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I> DH5α  </TD><TD>pSB1C3RFP-Fusion </TD><TD>LB+Cam  </TD><TD>only red colonies</TD></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I> DH5α  </TD><TD>pSB1C3-Tn5 </TD><TD>LB+Cam  </TD><TD>no growth</TD></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I> DH5α  </TD><TD>pSB1C3RFP-Tn5 </TD><TD>LB+Cam  </TD><TD>10 white colonies; several red colonies</TD></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I> DH5α  </TD><TD>pSB1C3-dxCas9 </TD><TD>LB+Cam  </TD><TD>no growth</TD></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I> DH5α  </TD><TD>pSB1C3RFP-dxCas9 </TD><TD>LB+Cam  </TD><TD>14 white colonies; several red colonies</TD></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I> DH5α  </TD><TD>PACYCduet-fusion-mut4 </TD><TD>LB+Cam  </TD><TD>~200 colonies</TD></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I> DH5α  </TD><TD>PACYCduet-fusion-mut4 </TD><TD>LB+Cam  </TD><TD>no colonies</TD></TR></TBODY></TABLE></CENTER></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#FridaySCS3">Friday, 21/09/2018</BUTTON><DIV id="FridaySCS3" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>21/09/2018</DD><DT>Operator</DT><DD>Susan Bouwmeester</DD></DL><H4>Colony PCR</H4><P>Colony PCR was done to verify propagation of pSB1C3-Tn5 in 10 colonies and pSB1C3-dxCas9 in 14 colonies after transformation 19/9. This plasmid contains Tn5 or dxCas9 and pSB1C3, which can be checked with primers VR and VF2.
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL)</TH></TR><TR class="tableevenadpbl"><TD>Forward primer</TD><TD>0.6</TD></TR><TR class="tableunevenadpbl"><TD>Reverse primer</TD><TD>0.6</TD></TR><TR class="tableunevenadpbl"><TD>Expected amplicon size (bp)</TD><TD>~1700 bp</TD></TR></TBODY></TABLE></CENTER><P>30 tubes were prepared for colony PCR. The thermocycler programme was installed to have an extension time of 2:30 for Tn5 and 5:00 for dxCas9 and an annealing temperature of 60 °C. After the programme was finished, samples were kept at 12°C.</P><DL><DT>Date</DT><DD>21/09/18</DD><DT>Operator</DT><DD>Susan Bouwmeester</DD></DL><H4>High Fidelity PCR</H4><P> To eliminate the forbidden sites, high fidelity PCR was done using PACYCduet-fusionfs as template DNA and fw_dxCas9_mut4_BL and rv_dxCas9_mut4_BL as forward and reverse phosphorylated primers respectively. This makes the product compitable for blunt end ligation. 50µL PCR mixture were prepared, according to the High Fidelity PCR protocol. Expected size of the product PACYCduet-fusion-BL4 is 9432 bp. 
    </P><P>The thermocycler programme was installed to have an extension time of 5:30 and a gradient in anealing temperature (55.1; 56.8; 58.7; 61.5; 63.7; 65.2) After the programme was finished, samples were kept at 12 °C.</P><H4>High Fidelity PCR</H4><P> To eliminate the forbidden sites, high fidelity PCR was done using PACYCduet-fusionfs as template DNA. To check the primers, all needed eliminations were performed parallel. The primers used can be found in the table. Phosphorylated primers were used. This makes the product compitable for blunt end ligation. 6x 50µL PCR mixture were prepared, according to the High Fidelity PCR protocol. Expected size of the product PACYCduet-fusion-mutation is 9432 bp. 
    </P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Primer</TH></TR><TR class="tableunevenadpbl"><TD>fw_dxCas9_mut1_BL</TD></TR><TR class="tableunevenadpbl"><TD>rv_dxCas9_mut1_BL</TD></TR><TR class="tableunevenadpbl"><TD>fw_dxCas9_mut2_BL</TD></TR><TR class="tableunevenadpbl"><TD>rv_dxCas9_mut2_BL</TD></TR><TR class="tableunevenadpbl"><TD>fw_dxCas9_mut3_BL</TD></TR><TR class="tableunevenadpbl"><TD>rv_dxCas9_mut3_BL</TD></TR><TR class="tableunevenadpbl"><TD>fw_dxCas9_mut4_BL</TD></TR><TR class="tableunevenadpbl"><TD>rv_dxCas9_mut4_BL</TD></TR><TR class="tableunevenadpbl"><TD>fw_dxCas9_mut5_BL</TD></TR><TR class="tableunevenadpbl"><TD>rv_dxCas9_mut5_BL</TD></TR></TBODY></TABLE></CENTER><P>The thermocycler programme was installed to have an extension time of 8:00 and an anealing temperature of 60°C. After the programme was finished, samples were kept at 12 °C.</P><H4>DNA Gel Electrophoresis</H4><P>The different fragments were run on a 0.8% agarose gel in TAE buffer. 3μL was pipetted into each lane. The gel was imaged using a gel documentation system with UV-light.</P><B>Figure dxCas9 colony PCR</B>0.8% Agarose. Expected size = ~4000bp. Lane 1: 500 bp ladder; Lane 2 t/m 14: Colony 1 t/m 13 pSB1C3-dxCas9; Lane 14: 500bp ladder; Lane 15 500bp ladder; Lane 16: Colony 15; Lane 17: Negative control GFP; Lane 18: Negative control, no template<B>Figure Tn5 colony PCR</B>0.8% Agarose. Lane 1: 500 bp ladder; Lane 2 t/m 11: Colony 1 t/m 10 pSB1C3-Tn5; Lane 11: Negative control GFP; Lane 12: Negative control, no template<P>Conclusion: No correct integrated dxCas9, 9 times possible integrated Tn5. </P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#SaturdaySCO3">Saturday, 22/09/2018</BUTTON><DIV id="SaturdaySCO3" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>18/09/2018</DD><DT>Operator</DT><DD>Susan Bouwmeester</DD></DL><H4>Restriction </H4><P>Restriction cloning was done for different constructs. The following table shows an overview of what fragment(s) was/were cut with what restriction enzyme(s).
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL) - fusionbb</TH><TH class="tableheaderadpbl">Volume (uL) - dxCas9</TH><TH class="tableheaderadpbl">Volume (uL) - linker </TH><TH class="tableheaderadpbl">Volume (uL) - pSB1C3en</TH><TH class="tableheaderadpbl">Volume (uL) - pSB1C3RFP</TH></TR><TR class="tableunevenadpbl"><TD>10x CutSmart buffer    </TD><TD>2</TD><TD>2</TD><TD>2</TD><TD>2</TD><TD>2</TD></TR><TR class="tableunevenadpbl"><TD>Fragment</TD><TD>16 (~976 ng)</TD><TD>5 (~1000 ng)</TD><TD>16 (~624 ng)</TD><TD>4.8 (~2000 ng)</TD><TD>16 (~1600 ng)</TD></TR><TR class="tableunevenadpbl"><TD>Enzyme (EcoRI-HF)</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD>1</TD></TR><TR class="tableunevenadpbl"><TD>Enzyme (PstI-HF)</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD>1</TD></TR><TR class="tableunevenadpbl"><TD>MilliQ</TD><TD> 0</TD><TD>11</TD><TD>0</TD><TD>11.2</TD><TD>0</TD></TR></TBODY></TABLE></CENTER><P>The samples were incubated for 3.5 hours in the incubator 37 °C, followed by PCR cleanup. The enzymes were heat inactivated for 10 minutes 80 °C. 

</P><H4>PCR clean-up</H4><P>
    PCR clean-up is according to the protocol for all the restricted framgents. Elution was done in two times 15uL Milli-Q.</P><H4>High Fidelity PCR</H4><P> To amplify dxCas9 (~4207bp), high fidelity PCR was done using the fusionbb as template DNA and fw_bb_dxcas9 and rv_bb_dxcas9_2 as forward and reverse primers respectively. 6 50µL PCR mixtures were prepared, according to the High Fidelity PCR protocol.
    </P><P>The thermocycler programme was installed to have an extension time of 2:30 and an annealing temperature of 63 °C. After the programme was finished, samples were kept at 12 °C.</P><H4>DNA Gel Electrophoresis</H4><P>The different PCR products from PACYCduet-fusionfs (21/9) were run on a 0.8% agarose gel in TAE buffer. 3μL was pipetted into each lane. The gel was imaged using a gel documentation system with UV-light.</P><B>Figure PCR PACYCduet-fusionfs</B>0.8% Agarose. Expected size = ~10000bp. Lane 1: 500 bp ladder; Lane 2: annealing temperature PCR 65.2°C; Lane 3: annealing temperature PCR 63.7°C; Lane 4: annealing temperature PCR 61.5°C; Lane 5: annealing temperature PCR 58.7°C; Lane 6: annealing temperature PCR 56.8°C; Lane 7: annealing temperature PCR 55.1°C; Lane 8: 500 bp ladder; Lane 9: primers mutation 1; Lane 10: primers mutation 2; Lane 11: primers mutation 3; Lane 12: primers mutation 4; Lane 13: primers mutation 5; Lane 14: primers mutation 6; Lane 15: negative control pSB1C3RFP
</DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV></DIV></UL></FORM><FORM><LABEL for="WeekSCS4" class="subdropdown cadpbl">Week 4 24/09/2018 - 30/09/2018</LABEL><UL class="submenu"><DIV class="subcontent"><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#MondaySCS4">Monday, 24/09/2018</BUTTON><DIV id="MondaySCS4" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>24/09/2018</DD><DT>Operator</DT><DD>Susan Bouwmeester</DD></DL><H4>Restriction </H4><P>Restriction cloning was done for pSB1C3 and the fusion protein. The following table shows an overview of what fragments were cut with what restriction enzymes.
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL) - pSB1C3 linear</TH><TH class="tableheaderadpbl">Volume (uL) - fusionbb</TH></TR><TR class="tableunevenadpbl"><TD>10x CutSmart buffer    </TD><TD>5</TD><TD>5</TD></TR><TR class="tableunevenadpbl"><TD>Fragment</TD><TD>42</TD><TD>13</TD></TR><TR class="tableunevenadpbl"><TD>Enzyme (EcoRI)</TD><TD>1.5</TD><TD>0.75</TD></TR><TR class="tableunevenadpbl"><TD>Enzyme (PstI)</TD><TD>1.5</TD><TD>0.75</TD></TR><TR class="tableunevenadpbl"><TD>MilliQ</TD><TD> 0</TD><TD>3.5</TD></TR></TBODY></TABLE></CENTER><P>The samples were incubated for 4.5 hours in the incubator 37 °C and followed by 20 minutes heat inactivation 80 °C. 

</P><H4>High Fidelity PCR</H4><P> To amplify fusionbb (~6000bp), high fidelity PCR was done using fusionbb1 as template DNA and fw_BB_dxCas9 and rv_BB_Tn5 forward and reverse primers respectively. 16 50µL PCR mixtures were prepared, according to the High Fidelity PCR protocol.
    </P><P>The thermocycler programme was installed to have an extension time of 6:30 minutes and an annealing temperature of 60 °C. After the programme was finished, samples were kept at 4 °C.</P><H4>DNA gel purification</H4><P>The fusionbb has a size of ~6000bp, but the PCR samples were contaminated with by-products. All the PCR product is loaded on a 0.8% agarose gel. DNA purification from agarose gel conform protocol
Elution was done in 30 µL Milli-Q.
</P><H4>PCR clean-up</H4><P>
    100 μL of PCR product PACYCduet-fusionfs-mut4 was subjected to PCR Clean-Up, according to the protocol. Elution was done in 2 times 15 µL Milli-Q.
</P><H4>DpnI Digestion</H4><P>To eliminate any residual template DNA, a DpnI digestion was performed with the following reaction mixture:
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL)</TH></TR><TR class="tableunevenadpbl"><TD>10x CutSmart buffer</TD><TD>4</TD></TR><TR class="tableevenadpbl"><TD>PCR product</TD><TD>23</TD></TR><TR class="tableunevenadpbl"><TD>Enzyme (DpnI)</TD><TD>1</TD></TR><TR class="tableunevenadpbl"><TD>MilliQ</TD><TD>12</TD></TR></TBODY></TABLE></CENTER><P>The reaction was incubated for 2 hours at 37 °C. 
Heat inactivation was performed for 20 minutes at 80 °C.</P><H4>Plasmid Isolation</H4><P> 
    Overnight starter cultures of pSB1C3-T7.Tn5.T7, pSB1C3-Tn5colony5 and pSB1C3-Tn5colony8 were subjected to plasmid isolation following the Plasmid Isolation Protocol. Final elution was done with two times 15 μL of pre-warmed MilliQ.
</P><H4>Nanodrop DNA Quantification</H4><P> 
    After plasmid isoaltion, all te samples were measered with nanodrop quantification. The following isolates were quantified according to the Nanodrop DNA Quantification protocol. The concentration and purity of the DNA (260/280 and 260/230 ratios) was measured. 
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Isolate</TH><TH class="tableheaderadpbl">Concentration (ng/uL)</TH><TH class="tableheaderadpbl">260/280 ratio</TH><TH class="tableheaderadpbl">230/280 ratio</TH></TR><TR class="tableunevenadpbl"><TD>pSB1C3-T7p.Tn5.T7t</TD><TD>134.8</TD><TD>1.74</TD><TD>1.81</TD></TR><TR class="tableevenadpbl"><TD>pSB1C3-Tn5Col5</TD><TD>146.7</TD><TD>1.96</TD><TD>1.8</TD></TR><TR class="tableunevenadpbl"><TD>pSB1C3-Tn5Col8</TD><TD>1.8</TD><TD>1.85</TD></TR></TBODY></TABLE></CENTER><H4>Sequence Verification </H4><H5>Option 1: Sequence from plasmid</H5><P> For verification of T7-Tn5.T7 biobrick and Tn5 biobrick, primers VF2 and VR were used for sequencing. The 10µL sequencing sample was prepared in a 1.5 mL tube as follows: 

</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">T7.Tn5.T7 Volume (uL)</TH><TH class="tableheaderadpbl">Tn5 Col5 Volume (uL)</TH><TH class="tableheaderadpbl">Tn5 Col8 Volume (uL)</TH><TH class="tableheaderadpbl">Final concentration</TH></TR><TR class="tableunevenadpbl"><TD>Plasmid DNA (500ng)</TD><TD>3.71</TD><TD>3.41</TD><TD>2.61</TD><TD>50 ng/uL</TD></TR><TR class="tableunevenadpbl"><TD>10uM primer XXX </TD><TD>2.5</TD><TD>2.5</TD><TD>2.5</TD><TD>25 pmol</TD></TR><TR class="tableunevenadpbl"><TD>MilliQ</TD><TD>3.79</TD><TD>4.09</TD><TD>4.89</TD></TR></TBODY></TABLE></CENTER><P>The result of the sequencing: Only Tn5Col5 and Tn5Col8 turned out to be correct. T7.Tn5.T7 missed a part. </P><H4>DNA gel purification</H4><P>The dxCas9bb has a size of ~4000bp, but the PCR samples were contaminated with by-products. All the PCR product is loaded on a 0.8% agarose gel. DNA purification from agarose gel conform protocol. Elution was done in two times 15 µL Milli-Q.
</P><H4>PCR clean-up</H4><P>
    Product of restriction of pSB1C3 and the fusion protein with EcoRI and PstI was subjected to PCR Clean-Up, according to the protocol. Elution was done in two times 15 µL Milli-Q.
</P><H4>Nanodrop DNA Quantification</H4><P> 
    After PCR cleanup, all te samples and some older restricted samples were measered with nanodrop quantification. The following isolates were quantified according to the Nanodrop DNA Quantification protocol. The concentration and purity of the DNA (260/280 and 230/280 ratios) was measured. 
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Isolate</TH><TH class="tableheaderadpbl">Concentration (ng/uL)</TH><TH class="tableheaderadpbl">260/280 ratio</TH><TH class="tableheaderadpbl">260/280 ratio</TH></TR><TR class="tableunevenadpbl"><TD>pSB1Cen 18/9</TD><TD>71.5</TD><TD>0.99</TD><TD>1.87</TD></TR><TR class="tableevenadpbl"><TD>Linker 18/9</TD><TD>7.3</TD><TD>0.23</TD><TD>1.42</TD></TR><TR class="tableunevenadpbl"><TD>pSB1C3en 22/9</TD><TD>68</TD><TD>1.38</TD><TD>1.88</TD></TR><TR class="tableunevenadpbl"><TD>pSB1C3RFP 22/9</TD><TD>85.3</TD><TD>0.84</TD><TD>1.82</TD></TR><TR class="tableunevenadpbl"><TD>dxCas9 22/9</TD><TD>17</TD><TD>0.39</TD><TD>1.79</TD></TR><TR class="tableevenadpbl"><TD>linker 22/9</TD><TD>7.7</TD><TD>0.25</TD><TD>1.70</TD></TR><TR class="tableunevenadpbl"><TD>fusion 22/9</TD><TD>11.3</TD><TD>0.46</TD><TD>1.79</TD></TR><TR class="tableunevenadpbl"><TD>pSB1C3RFP 24/9 </TD><TD>179.2</TD><TD>1.96</TD><TD>1.88</TD></TR><TR class="tableevenadpbl"><TD>fusion</TD><TD>32.8</TD><TD>1.17</TD><TD>1.83</TD></TR></TBODY></TABLE></CENTER><H4>Ligation</H4><P>In order to assemble several cut constructs, ligation was performed. </P><P>A 10ul ligation reaction mixture was prepared with the following composition</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">pSB1C3-Fusion Volume(uL) </TH><TH class="tableheaderadpbl">pSB1C3-dxCas9 Volume (uL) </TH><TH class="tableheaderadpbl">pSB1C3-linker(uL) </TH><TH class="tableheaderadpbl">pSB1C3-negative Volume (uL) </TH></TR><TR class="tableunevenadpbl"><TD>Ligase Buffer (10x) </TD><TD>2</TD><TD>2</TD><TD>2</TD><TD>2</TD></TR><TR class="tableevenadpbl"><TD>T4 DNA Ligase</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD>1</TD></TR><TR class="tableunevenadpbl"><TD>DNA vector </TD><TD>0.6 uL (~100ng) </TD><TD>0.6 uL (~100ng) </TD><TD>1.47 uL (~100ng)</TD><TD>0.6 uL (~100ng)</TD></TR><TR class="tableunevenadpbl"><TD>DNA fragment </TD><TD>17uL (~500ng)</TD><TD>16.4 uL (~270ng)</TD><TD>15.5 uL(~ng)</TD><TD>0 uL</TD></TR><TR class="tableunevenadpbl"><TD>MilliQ</TD><TD>0 uL</TD><TD>0.6 uL</TD><TD>0 uL</TD><TD>16.4</TD></TR></TBODY></TABLE></CENTER><P>The samples were incubated overnight on 16°C for ligation. After 13h 10minutes 90°C heat inactivation.</P><H4>PCR clean-up</H4><P>
    After DpnI digestion PACYCduet-fusionfs-mut4 was subjected to PCR Clean-Up, according to the protocol. Elution was done in 2 times 15 µL Milli-Q.
</P><H4>Ligation</H4><P>ligation was performed for blunt end ligation of PACYCduet-fusionfs-mut4. </P><P>A 10ul ligation reaction mixture was prepared with the following composition:</P><TABLE><CENTER><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">PACYCduet-fusionfs-mut4 </TH></TR><TR class="tableunevenadpbl"><TD>Ligase Buffer (10x) </TD><TD>2</TD></TR><TR class="tableevenadpbl"><TD>T4 DNA Ligase</TD><TD>1</TD></TR><TR class="tableunevenadpbl"><TD>DNA fragment </TD><TD>2.4 uL (~200ng) </TD></TR><TR class="tableunevenadpbl"><TD>MilliQ</TD><TD>14.6 uL</TD></TR></TBODY></CENTER></TABLE><P>The samples were incubated overnight on 16°C for ligation. After 13h 10minutes 90°C heat inactivation. </P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#TuesdaySCS4">Tuesday, 25/09/2018</BUTTON><DIV id="TuesdaySCS4" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>25/09/2018</DD><DT>Operator</DT><DD>Susan Bouwmeester</DD></DL><H4>High Fidelity PCR</H4><P> To amplify dxCas9 (~4207bp), high fidelity PCR was done using the dxCas9bb as template DNA and fw_bb_dxcas9 and rv_bb_dxcas9_2 as forward and reverse primers respectively. 6 50µL PCR mixtures were prepared, according to the High Fidelity PCR protocol.
    </P><P>The thermocycler programme was installed to have an extension time of 2:30 and an annealing temperature of 63 °C. After the programme was finished, samples were kept at 12 °C.</P><H4>DNA Gel Electrophoresis</H4><P>PCR product for the amplification of dxCas9bb was run on a 0.8% agarose gel in TAE buffer. 4μL was pipetted into the gel. The gel was imaged using a gel documentation system with UV-light.</P><B>Figure dxCas9 amplification</B>. Expected size (~4000bp). 0.8% agarose. Lane 1: 500bp ladder; Lane 2 PCR product <P>No dxCas9bb produced during the PCR. </P><H4>DNA Gel Electrophoresis</H4><P>PCR product for the amplification of fusionbb was run on a 0.8% agarose gel in TAE buffer. All PCR-product was loaded on the gel. The gel was imaged using a gel documentation system with UV-light.</P><B>Figure fusionbb amplification</B>. Expected size (~6000bp). 0.8% agarose. Lane 1: 500bp ladder; Lane 2t/m 9 PCR product fusionbb<P> fusionbb produced during the PCR. </P><H4>High Fidelity PCR</H4><P> To amplify linker(~60bp), high fidelity PCR was done using the previous produced linker as template DNA and fw_bb_linker and rv_bb_linker forward and reverse primers respectively. 8 50µL PCR mixtures were prepared, according to the High Fidelity PCR protocol.
    </P><P>The thermocycler programme was installed to have an extension time of 0:23 and an annealing temperature of 60 °C. After the programme was finished, samples were kept at 12 °C.</P><H4> Transformation of chemically competent cells </H4><P>The following transformations were carried out according to the protocol for transformation of chemically competent cells:
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Strain</TH><TH class="tableheaderadpbl">Plasmid </TH><TH class="tableheaderadpbl">Cloning method</TH><TH class="tableheaderadpbl">Volume added (uL)</TH><TH class="tableheaderadpbl">Medium on which was plated</TH></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I>DH5α</TD><TD>pSB1C3-fusion</TD><TD>restiction/ligation</TD><TD>2</TD><TD>LB Cam</TD></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I>DH5α</TD><TD>pSB1C3-fusion</TD><TD>restiction/ligation</TD><TD>18</TD><TD>LB Cam</TD></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I>DH5α</TD><TD>pSB1C3-dxCas9</TD><TD>restiction/ligation</TD><TD>2</TD><TD>LB Cam</TD></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I>DH5α</TD><TD>pSB1C3-dxCas9</TD><TD>restiction/ligation</TD><TD>18</TD><TD>LB Cam</TD></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I>DH5α</TD><TD>pSB1C3-linker</TD><TD>restiction/ligation</TD><TD>2</TD><TD>LB Cam</TD></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I>DH5α</TD><TD>pSB1C3-linker</TD><TD>restiction/ligation</TD><TD>18</TD><TD>LB Cam</TD></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I>DH5α</TD><TD>PACYCduet-fusionfs-mut4</TD><TD>restiction/ligation</TD><TD>2</TD><TD>LB Cam</TD></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I>DH5α</TD><TD>PACYCduet-fusionfs-negative</TD><TD>restiction/ligation</TD><TD>18</TD><TD>LB Cam</TD></TR></TBODY></TABLE></CENTER><P>800 uL LB medium was added as recovery medium.
</P><H4>High Fidelity PCR</H4><P> To amplify fusionbb (~6000bp), high fidelity PCR was done using fusionbb1 as template DNA and fw_BB_dxCas9 and rv_BB_Tn5 forward and reverse primers respectively. 4 50µL PCR mixtures were prepared, according to the High Fidelity PCR protocol.
    </P><P>The thermocycler programme was installed to have an extension time of 6:30 minutes and an annealing temperature of 70 °C. After the programme was finished, samples were kept at 4 °C.</P><H4>High Fidelity PCR</H4><P> To amplify dxCas9 (~4207bp), high fidelity PCR was done using the fusionbb as template DNA and fw_bb_dxcas9 and rv_bb_dxcas9_2 as forward and reverse primers respectively. 4 50µL PCR mixtures were prepared, according to the High Fidelity PCR protocol.
    </P><P>The thermocycler programme was installed to have an extension time of 3:00 and an annealing temperature of 63 °C. After the programme was finished, samples were kept at 12 °C.</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#WednesdaySCS4">Wednesday, 26/09/2018</BUTTON><DIV id="WednesdaySCS4" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>26/09/2018</DD><DT>Operator</DT><DD>Susan Bouwmeester</DD></DL><H4>DNA Gel Electrophoresis</H4><P>PCR product of the amplification of the linker was run on a 2% agarose gel in TAE buffer. 3μL was pipetted into each lane. The gel was imaged using a gel documentation system with UV-light.</P><B>Figure</B> linker amplification. 2% agarose. Expected size is ~60bp. Lane 1: 100bp ladder; Lane 2: PCR product linker <H4> Transformation Results </H4><P> After carrying out the transformation protocol on 25/09 the plates were left for overnight incubation. After ~14-18hr incubation, the following results were observed: </P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Strain</TH><TH class="tableheaderadpbl">DNA </TH><TH class="tableheaderadpbl">Medium</TH><TH class="tableheaderadpbl">Result</TH></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I> DH5α  </TD><TD>pSB1C3-dxCas9-1  </TD><TD>LB+Cam  </TD><TD>1 white; 8 red</TD></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I> DH5α  </TD><TD>pSB1C3-dxCas9-2  </TD><TD>LB+Cam  </TD><TD>16 red</TD></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I> DH5α  </TD><TD>pSB1C3-dxCas9-3  </TD><TD>LB+Cam  </TD><TD>1 white; ~100 red</TD></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I> DH5α  </TD><TD>pSB1C3-dxCas9-4  </TD><TD>LB+Cam  </TD><TD>~100 red</TD></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I> DH5α  </TD><TD>pSB1C3-fusion-1  </TD><TD>LB+Cam  </TD><TD>10 red</TD></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I> DH5α  </TD><TD>pSB1C3-fusion-2  </TD><TD>LB+Cam  </TD><TD>15 red</TD></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I> DH5α  </TD><TD>pSB1C3-fusion-3  </TD><TD>LB+Cam  </TD><TD>6 white; ~100 red</TD></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I> DH5α  </TD><TD>pSB1C3-dxCas9-4  </TD><TD>LB+Cam  </TD><TD>5 white; ~100 red</TD></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I> DH5α  </TD><TD>PACYCduet-fusionfs-mut4  </TD><TD>LB+Cam  </TD><TD>~100 </TD></TR></TBODY></TABLE></CENTER><H4>Colony PCR</H4><P>Colony PCR was done to verify propagation of pSB1C3-fusion in 2 colonies and pSB1C3-dxCas9 in 1 colony after transformation. This plasmid contains the fusion(~6000bp) or dxCas9 (~4000bp) and pSB1C3, which can be checked with primers VR and VF2.
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL)</TH></TR><TR class="tableevenadpbl"><TD>Forward primer</TD><TD>0.6</TD></TR><TR class="tableunevenadpbl"><TD>Reverse primer</TD><TD>0.6</TD></TR></TBODY></TABLE></CENTER><P>5 tubes were prepared for colony PCR. The thermocycler programme was installed to have an extension time of 8:00and an annealing temperature of 60 °C. After the programme was finished, samples were kept at 12°C.</P><H4>DNA Gel Electrophoresis</H4><P>PCR product of the colony PCR of dxCas9 and fusion was run on a 0.8% agarose gel in TAE buffer. 5μL was pipetted into each lane. The gel was imaged using a gel documentation system with UV-light.</P><B>Figure</B> Colony PCR fusion and dxCas9. 0.8% agarose. Expected size fusion ~6000 bp. Expected size dxCas9 ~4000bp. Lane 1: 500bp ladder; Lane 2: fusion colony 1; Lane 3: fusion colony 2; Lane 4: dxCas9 colony 1 <P>The amplified product colony PCR is around the correct hight. An overnight culture dxCas9 colony1 in 3mL with Cam is made.</P><H4>DNA Gel Electrophoresis</H4><P>PCR product of the colony PCR of dxCas9 and fusion was run on a 0.8% agarose gel in TAE buffer. 5μL was pipetted into each lane. The gel was imaged using a gel documentation system with UV-light.</P><B>Figure</B>Amplification dxCas9bb and fusionbb. 0.8% agarose. Expected size fusion ~6000 bp. Expected size dxCas9 ~4000bp. Lane 1: 500bp ladder; Lane 2: fusionbb; Lane 4: dxCas9bb <H4>DNA purification agarose gel </H4><P>The amplicon of interest has a size of ~4000bp or ~6000bp, but the PCR samples were contaminated with by-products. The products were loaded on a gel prior to <A href="https://2018.igem.org/Team:TUDelft/Experiments#DNApurificationfromgel-scroll" target="_blank">DNA purification from agarose gel conform protocol</A>.
Elution was done in 2 times 15 µL Milli-Q.
</P><H4>PCR clean-up</H4><P>200 ul of PCR product of the linker was subjected to PCR Clean-Up, according to the protocol. Elution was done in 30 µL Milli-Q.
</P><H4>Nanodrop DNA Quantification</H4><P> 
    After PCR cleanup, all te samples were measered with nanodrop quantification. The following isolates were quantified according to the Nanodrop DNA Quantification protocol. The concentration and purity of the DNA (260/280 ratios and 260/230) was measured. 
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Isolate</TH><TH class="tableheaderadpbl">Concentration (ng/uL)</TH><TH class="tableheaderadpbl">260/280 ratio</TH><TH class="tableheaderadpbl">260/230 ratio</TH></TR><TR class="tableunevenadpbl"><TD>linker</TD><TD>262.2</TD><TD>1.84</TD><TD>1.91</TD></TR><TR class="tableevenadpbl"><TD>dxCas9</TD><TD>111.4</TD><TD>1.84</TD><TD>1.77</TD></TR><TR class="tableunevenadpbl"><TD>fusion</TD><TD>88.8</TD><TD>1.88</TD><TD>1.66</TD></TR></TBODY></TABLE></CENTER><H4>High Fidelity PCR</H4><P> To amplify fusionbb (~6000bp), high fidelity PCR was done using fusionbb1 as template DNA and fw_BB_dxCas9 and rv_BB_Tn5 forward and reverse primers respectively. 8 50µL PCR mixtures were prepared, according to the High Fidelity PCR protocol.
    </P><P>The thermocycler programme was installed to have an extension time of 6:30 minutes and an annealing temperature of 70 °C. After the programme was finished, samples were kept at 4 °C.</P><H4>High Fidelity PCR</H4><P> To amplify dxCas9 (~4207bp), high fidelity PCR was done using the fusionbb as template DNA and fw_bb_dxcas9 and rv_bb_dxcas9_2 as forward and reverse primers respectively. 8 50µL PCR mixtures were prepared, according to the High Fidelity PCR protocol.
    </P><P>The thermocycler programme was installed to have an extension time of 3:00 and an annealing temperature of 63 °C. After the programme was finished, samples were kept at 12 °C.</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#ThursdaySCS4">Thursday, 27/09/2018</BUTTON><DIV id="ThursdaySCS4" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>27/09/2018</DD><DT>Operator</DT><DD>Susan Bouwmeester</DD></DL><H4>Plasmid Isolation</H4><P> 
    Overnight starter cultures of pSB1C3-dxCas9 subjected to plasmid isolation following the Plasmid Isolation Protocol. Final elution was done with two times 15 μL of pre-warmed MilliQ.
</P><H4>Nanodrop DNA Quantification</H4><P> 
    After plasmid isoaltion, the sample was measered with nanodrop quantification. The following isolates were quantified according to the Nanodrop DNA Quantification protocol. The concentration and purity of the DNA (260/280 and 260/230 ratios) was measured. 
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Isolate</TH><TH class="tableheaderadpbl">Concentration (ng/uL)</TH><TH class="tableheaderadpbl">260/280 ratio</TH><TH class="tableheaderadpbl">230/280 ratio</TH></TR><TR class="tableunevenadpbl"><TD>pSB1C3-dxCas9</TD><TD>292.7</TD><TD>1.87</TD><TD>2.09</TD></TR></TBODY></TABLE></CENTER><H4>Sequence Verification </H4><H5>Sequence from plasmid</H5><P> For verification of dxCas9 biobrick primers VF2, VR, fw_dxCas9_mut_EcoRI, fw_Tn5_beforeC, fw_dxCas9_mut2_GA and rv_dxCas9_mut3_GA were used for sequencing. The 10µL sequencing sample was prepared in a 1.5 mL tube as follows: 

</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL)</TH><TH class="tableheaderadpbl">Final concentration</TH></TR><TR class="tableunevenadpbl"><TD>Plasmid DNA (500ng)</TD><TD>1.71</TD><TD>50 ng/uL</TD></TR><TR class="tableunevenadpbl"><TD>10uM primer XXX </TD><TD>2.5</TD><TD>25 pmol</TD></TR><TR class="tableunevenadpbl"><TD>MilliQ</TD><TD>5.79</TD></TR></TBODY></TABLE></CENTER><P>The result of the sequencing: dxCas9 is only partly integrated. It is not the complete sequence for dxCas9bb.</P><H4>Colony PCR</H4><P>Colony PCR was done to verify propagation of pSB1C3-fusion in 12 colonies and after transformation. This plasmid contains the fusion (~6000bp) and pSB1C3, which can be checked with primers VR and VF2.
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL)</TH></TR><TR class="tableevenadpbl"><TD>Forward primer</TD><TD>0.6</TD></TR><TR class="tableunevenadpbl"><TD>Reverse primer</TD><TD>0.6</TD></TR></TBODY></TABLE></CENTER><P>12 tubes were prepared for colony PCR of 12 pSB1C3-fusion colonies. The thermocycler programme was installed to have an extension time of 8:00 and an annealing temperature of 60 °C. After the programme was finished, samples were kept at 12°C.</P><H4>DNA Gel Electrophoresis</H4><P>PCR product of the colony PCR of fusion was run on a 0.8% agarose gel in TAE buffer. 5μL was pipetted into each lane. The gel was imaged using a gel documentation system with UV-light.</P><B>Figure</B> Colony PCR dxCas9. 0.8% agarose. Expected size dxCas9 ~4000bp. Lane 1: 500bp ladder; Lane 2: colony 1; Lane 3: colony 2; Lane 4: colony 3; Lane 5: colony 4; Lane 6: colony 5; Lane 7: colony 6; Lane 8: colony 7; Lane 9: colony 8; Lane 10: colony 9; Lane 11: colony 10; Lane 12: colony 11; Lane 13: colony 12; Lane 14: Negative control, no template; Lane 15: 500bp ladder <P>The amplified product colony PCR is not around the correct hight. No succesful integration of dxCas9 in pSB1C3</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#FridaySCS4">Friday, 28/09/2018</BUTTON><DIV id="FridaySCS4" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>28/09/2018</DD><DT>Operator</DT><DD>Susan Bouwmeester</DD></DL><H4>DNA Gel Electrophoresis</H4><P>PCR product of PCR for fusion amplification was run on a 0.8% agarose gel in TAE buffer. 5μL was pipetted into each lane. The gel was imaged using a gel documentation system with UV-light.</P><B>Figure</B>Fusion amplification. 0.8% agarose. Expected size fusion ~6000bp. Lane 1: 500bp ladder; Lane 2: amplified fusion<H4>DNA gel purification</H4><P>The fusionbb has a size of ~6000bp, but the PCR samples were contaminated with by-products. All the PCR product is loaded on a 0.8% agarose gel. DNA purification from agarose gel conform protocol. Elution was done in two times 15 µL Milli-Q.
</P><H4>Nanodrop DNA Quantification</H4><P>   After PCR cleanup, all te samples and some older restricted samples were measered with nanodrop quantification. The following isolates were quantified according to the Nanodrop DNA Quantification protocol. The concentration and purity of the DNA (260/280 and 230/280 ratios) was measured. 
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Isolate</TH><TH class="tableheaderadpbl">Concentration (ng/uL)</TH><TH class="tableheaderadpbl">260/280 ratio</TH><TH class="tableheaderadpbl">260/280 ratio</TH></TR><TR class="tableunevenadpbl"><TD>fusionbb</TD><TD>173.2</TD><TD>1.87</TD><TD>0.8</TD></TR></TBODY></TABLE></CENTER><H4>Gibson Assembly</H4><P>According to the displayed reaction mixture composition, Gibson Assembly was done to assemble fusionbb and pSB1C3 to obtain pSB1C3-fusion.
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL)</TH></TR><TR class="tableunevenadpbl"><TD>Gibson Assembly Mastermix 2X</TD><TD>10</TD><TD>10</TD><TD>10</TD></TR><TR class="tableevenadpbl"><TD>Vector</TD><TD>pSB1C3, 0.3 (200 ng)</TD></TR><TR class="tableunevenadpbl"><TD>Fragment</TD><TD>fusionbb, 9.7 (1800)</TD></TR><TR class="tableunevenadpbl"><TD>MilliQ</TD><TD>3.4</TD><TD>1.1</TD><TD>2.9</TD></TR></TBODY></TABLE></CENTER><P>The samples were incubated at 55°C for 60 minutes, after which they were stored at -20°C.</P><H4>Restriction </H4><P>Restriction cloning was done for pSB1C3 and the fusion protein. The following table shows an overview of what fragments were cut with what restriction enzymes.
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL) - pSB1C3RFP</TH><TH class="tableheaderadpbl">Volume (uL) - fusionbb</TH></TR><TR class="tableunevenadpbl"><TD>10x CutSmart buffer    </TD><TD>5</TD><TD>5</TD></TR><TR class="tableunevenadpbl"><TD>Fragment</TD><TD>41 (~5000ng)</TD><TD>20.4 (~2000ng)</TD></TR><TR class="tableunevenadpbl"><TD>Enzyme (EcoRI)</TD><TD>2</TD><TD>2</TD></TR><TR class="tableunevenadpbl"><TD>Enzyme (PstI)</TD><TD>2</TD><TD>2</TD></TR><TR class="tableunevenadpbl"><TD>MilliQ</TD><TD> 0</TD><TD>20.6</TD></TR></TBODY></TABLE></CENTER><P>The samples were incubated for 4 hours in the incubator 37 °C and followed by 20 minutes heat inactivation 80 °C. 

</P><H4>Colony PCR</H4><P>Colony PCR was done to verify propagation of pSB1C3-fusion in 10 colonies and after transformation. This plasmid contains the fusion (~6000bp) and pSB1C3, which can be checked with primers VR and VF2.
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL)</TH></TR><TR class="tableevenadpbl"><TD>Forward primer</TD><TD>0.6</TD></TR><TR class="tableunevenadpbl"><TD>Reverse primer</TD><TD>0.6</TD></TR></TBODY></TABLE></CENTER><P>10 tubes were prepared for colony PCR of 10 pSB1C3-fusion colonies. The thermocycler programme was installed to have an extension time of 8:00 and an annealing temperature of 60 °C. After the programme was finished, samples were kept at 12°C.</P><H4>DNA Gel Electrophoresis</H4><P>PCR product of the colony PCR of fusion was run on a 0.8% agarose gel in TAE buffer. 5μL was pipetted into each lane. The gel was imaged using a gel documentation system with UV-light.</P><B>Figure</B> Colony PCR fusion. 0.8% agarose. Expected size fusion ~6000bp. Lane 1: 500bp ladder; Lane 2: colony 1; Lane 3: colony 2; Lane 4: colony 3; Lane 5: colony 4; Lane 6: colony 5; Lane 7: colony 6; Lane 8: colony 7; Lane 9: colony 8; Lane 10: colony 9; Lane 11: colony 10; Lane 12: Negative control, no template; <P>The amplified product colony PCR is not around the correct hight. No succesful integration of fusion in pSB1C3</P><H4>High Fidelity PCR</H4><P> To eliminate the forbidden sites, high fidelity PCR was done using PACYCduet-mut4-1, PACYCduet-mut4-2 and PACYCduet-mut4-3 as template DNA. Primer fw_dxCas9_mut5_BL and rv_dxCas9_mut5_BL were used. Phosphorylated primers were used. This makes the product compitable for blunt end ligation. 3x 50µL PCR mixture was prepared, according to the High Fidelity PCR protocol. Expected size of the product PACYCduet-fusion-mutation is 9432 bp. 
    </P><P>The thermocycler programme was installed to have an extension time of 8:00 and an anealing temperature of 60°C. After the programme was finished, samples were kept at 12 °C.</P><H4>DNA Gel Electrophoresis</H4><P>the PCR product of the amplification of PACYCduet-mut5 was run on a 0.8% agarose gel in TAE buffer to check the pCR reaction. 5μL was pipetted into each lane. The gel was imaged using a gel documentation system with UV-light.</P><B>Figure</B> Colony PCR fusion. 0.8% agarose. Expected size fusion ~10000bp. Lane 1: 500bp ladder; Lane 2: PACYCduet-mut5-1; Lane 3: PACYCduet-mut5-2; Lane 4: PACYCduet-mut5-3; <H4>High Fidelity PCR</H4><P> To amplify fusionbb (~6000bp), high fidelity PCR was done using fusionbb1 as template DNA and fw_BB_dxCas9 and rv_BB_Tn5 forward and reverse primers respectively. 8 50µL PCR mixtures were prepared, according to the High Fidelity PCR protocol.
    </P><P>The thermocycler programme was installed to have an extension time of 6:30 minutes and an annealing temperature of 70 °C. After the programme was finished, samples were kept at 4 °C.</P><H4>DNA Gel Electrophoresis</H4><P>Restriction product of fusion and pSB1C3RFP was run on a 0.8% agarose gel in TAE buffer. 5μL was pipetted into each lane. The gel was imaged using a gel documentation system with UV-light.</P><B>Figure</B>Restriction fusionbb and pSB1C3RFP. 0.8% agarose. Expected size fusion ~6000bp; Expected size pSB1C3 restricted is two fragments of ~2000bp and ~1500bp. Lane 1: 500bp ladder; Lane 2: restricted fusionbb; Lane 3: restricted pSB1C3RFP<H4>PCR clean-up</H4><P>PCR product of the PACYCduet-fusion-mut5 was subjected to PCR Clean-Up, according to the protocol. Elution was done in one time 15uL and one time 20uL Milli-Q.
</P><H4>Nanodrop DNA Quantification</H4><P> 
    After PCR cleanup, all te samples were measered with nanodrop quantification. The following isolates were quantified according to the Nanodrop DNA Quantification protocol. The concentration and purity of the DNA (260/280 ratios and 260/230) was measured. 
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Isolate</TH><TH class="tableheaderadpbl">Concentration (ng/uL)</TH><TH class="tableheaderadpbl">260/280 ratio</TH><TH class="tableheaderadpbl">260/230 ratio</TH></TR><TR class="tableunevenadpbl"><TD>PACYCduet-fusion-mut5a</TD><TD>100.7</TD><TD>1.97</TD><TD>0.15</TD></TR><TR class="tableevenadpbl"><TD>PACYCduet-fusion-mut5b</TD><TD>130.7</TD><TD>1.87</TD><TD>0.16</TD></TR><TR class="tableunevenadpbl"><TD>PACYCduet-fusion-mut5c</TD><TD>225.0</TD><TD>1.89</TD><TD>0.27</TD></TR></TBODY></TABLE></CENTER><H4>DpnI Digestion</H4><P>To eliminate any residual template DNA, a DpnI digestion was performed for PACYCduet-fusion-mut5a, PACYCduet-fusion-mut5b, PACYCduet-fusion-mut5c, with the following reaction mixture:
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL) PACYCduet-fusion-mut5a</TH><TH class="tableheaderadpbl">Volume (uL) PACYCduet-fusion-mut5b</TH><TH class="tableheaderadpbl">Volume (uL) PACYCduet-fusion-mut5c</TH></TR><TR class="tableunevenadpbl"><TD>10x CutSmart buffer</TD><TD>5</TD><TD>5</TD><TD>5</TD></TR><TR class="tableevenadpbl"><TD>PCR product</TD><TD>32</TD><TD>32</TD><TD>32</TD></TR><TR class="tableunevenadpbl"><TD>Enzyme (DpnI)</TD><TD>1.5</TD><TD>1.5</TD><TD>1.5</TD></TR><TR class="tableunevenadpbl"><TD>MilliQ</TD><TD>12</TD><TD>12</TD><TD>12</TD></TR></TBODY></TABLE></CENTER><P>The reaction was incubated for 4 hours at 37 °C. 
Heat inactivation was performed for 20 minutes at 80 °C.</P><H4>DNA Gel Electrophoresis</H4><P>Restriction product of fusion and pSB1C3RFP was run on a 0.8% agarose gel in TAE buffer. 5μL was pipetted into each lane. The gel was imaged using a gel documentation system with UV-light.</P><B>Figure</B>Restriction fusionbb and pSB1C3RFP. 0.8% agarose. Expected size fusion ~6000bp; Expected size pSB1C3 restricted is two fragments of ~2000bp and ~1500bp. Lane 1: 500bp ladder; Lane 2: restricted fusionbb; Lane 3: restricted pSB1C3RFP<H4>DNA Gel Electrophoresis</H4><P>Restriction product of fusion and pSB1C3RFP was run on a 0.8% agarose gel in TAE buffer. 5μL was pipetted into each lane. The gel was imaged using a gel documentation system with UV-light.</P><B>Figure</B> Amplification dxCas9. Expected size ~4000bp. 0.8% agarose. Expected size fusion ~6000bp; Expected size pSB1C3 restricted is two fragments of ~2000bp and ~1500bp. Lane 1: 500bp ladder; Lane 2: restricted fusionbb; Lane 3: restricted pSB1C3RFP</DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#SaturdaySCO4">Saturday, 29/09/2018</BUTTON><DIV id="SaturdaySCO4" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>29/09/2018</DD><DT>Operator</DT><DD>Susan Bouwmeester</DD></DL><H4>PCR clean-up</H4><P>DpnI digested PCR products of the PACYCduet-fusion-mut5 were subjected to PCR Clean-Up, according to the protocol. Elution was done in one time 15uL and one time 20uL Milli-Q.
</P><H4>DNA gel purification</H4><P>The EcoRI and PstI restricted fusionbb and pSB1C3 contains RFP and other flanks. To clean this, the restricted product is loaded on a 0.8% agarose gel. DNA purification from agarose gel conform protocol. Elution was done in one time 15 µL Milli-Q and one time 20 µL.
</P><H4>Nanodrop DNA Quantification</H4><P> 
    After PCR cleanup, all te samples were measered with nanodrop quantification. The following isolates were quantified according to the Nanodrop DNA Quantification protocol. The concentration and purity of the DNA (260/280 and 230/280 ratios) was measured. 
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Isolate</TH><TH class="tableheaderadpbl">Concentration (ng/uL)</TH><TH class="tableheaderadpbl">260/280 ratio</TH><TH class="tableheaderadpbl">260/280 ratio</TH></TR><TR class="tableunevenadpbl"><TD>PACYCduet-fusion-mut5-a</TD><TD>15.3</TD><TD>1.7</TD><TD>0.45</TD></TR><TR class="tableunevenadpbl"><TD>PACYCduet-fusion-mut5-b</TD><TD>15.1</TD><TD>1.8</TD><TD>0.72</TD></TR><TR class="tableunevenadpbl"><TD>PACYCduet-fusion-mut5-c</TD><TD>16.2</TD><TD>1.6</TD><TD>0.07</TD></TR><TR class="tableunevenadpbl"><TD>pSB1C3-restricted</TD><TD>57.4</TD><TD>1.88</TD><TD>1.35</TD></TR><TR class="tableunevenadpbl"><TD>fusion-resticted</TD><TD>26.7</TD><TD>1.5</TD><TD>0.78</TD></TR></TBODY></TABLE></CENTER><H4>Blunt end Ligation</H4><P>ligation was performed for blunt end ligation of PACYCduet-fusionfs-mut5. </P><P>A 10ul ligation reaction mixture was prepared with the following composition:</P><TABLE><CENTER><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">PACYCduet-fusion-mutation5 </TH></TR><TR class="tableunevenadpbl"><TD>Ligase Buffer (10x) </TD><TD>2</TD></TR><TR class="tableevenadpbl"><TD>T4 DNA Ligase</TD><TD>1</TD></TR><TR class="tableunevenadpbl"><TD>DNA fragment </TD><TD>2.4 uL (~200ng) </TD></TR><TR class="tableunevenadpbl"><TD>MilliQ</TD><TD>14.6 uL</TD></TR></TBODY></CENTER></TABLE><P>The samples were incubated overnight on 16°C for ligation. </P><H4>Ligation</H4><P>In order to assemble pSB1C3-fusion, ligation was performed. </P><P>A 10ul ligation reaction mixture was prepared with the following composition</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">pSB1C3-Fusion Volume(uL) </TH></TR><TR class="tableunevenadpbl"><TD>Ligase Buffer (10x) </TD><TD>2</TD></TR><TR class="tableevenadpbl"><TD>T4 DNA Ligase</TD><TD>1</TD></TR><TR class="tableunevenadpbl"><TD>DNA vector </TD><TD>0.9 uL (~100ng) </TD></TR><TR class="tableunevenadpbl"><TD>DNA fragment </TD><TD>16.1 uL (~500ng)</TD></TR><TR class="tableunevenadpbl"><TD>MilliQ</TD><TD>0 uL</TD></TR></TBODY></TABLE></CENTER><P>The samples were incubated overnight on 16°C for ligation.</P><H4>High Fidelity PCR</H4><P> To amplify fusionbb (~6000bp), high fidelity PCR was done using fusionbb1 as template DNA and fw_BB_dxCas9 and rv_BB_Tn5 forward and reverse primers respectively. 8 50µL PCR mixtures were prepared, according to the High Fidelity PCR protocol.
    </P><P>The thermocycler programme was installed to have an extension time of 6:30 minutes and an annealing temperature of 70 °C. After the programme was finished, samples were kept at 4 °C.</P><H4>High Fidelity PCR</H4><P> To amplify dxCas9 (~4207bp), high fidelity PCR was done using the fusionbb as template DNA and fw_bb_dxcas9 and rv_bb_dxcas9_2 as forward and reverse primers respectively. 8 50µL PCR mixtures were prepared, according to the High Fidelity PCR protocol.
    </P><P>The thermocycler programme was installed to have an extension time of 3:00 and an annealing temperature of 63 °C. After the programme was finished, samples were kept at 12 °C.</P><DL><DT>Date</DT><DD>29/09/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">Liquid Starter Culture Preparation</H4><P><I>E.coli</I> strain harbouring the plasmids pACYCDuet-1_Tn5, pACYCDuet-1 and pSB1C3 were used for inoculation of 10 mL liquid starter culture (Luria Broth, complemented with chloramphenicol) following the <A href="https://2018.igem.org/Team:TUDelft/Experiments#liquidstarterculture-scroll" class="adpbl" target="_blank">Liquid Starter Culture Protocol</A>.
</P><DL><DT>Date</DT><DD>29/09/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">High Fidelity PCR</H4><P>To amplify pSB1C3 backbone, dxCas9 (BioBrick cloning), dxCas9-Lin-Tn5 (BioBrick cloning), dxCas9 (pACYCDuet-1 cloning), dxCas9-Lin-Tn5 (pACYCDuet-1 cloning) high fidelity PCR was done. The template DNA are the Gblocks  3 and 4 combined (primerset 056 &amp; 057 and 056 &amp; 055) and pACYCDuet-1 (primerset 083 &amp; 084) and pSB1C3 (primerset 051 &amp; 052). Multiple 50µL PCR mixtures were prepared, according to the <A href="https://2018.igem.org/Team:TUDelft/Experiments#highfidelityPCR-scroll" target="_blank" class="adpbl">High Fidelity PCR protocol</A>. 
Primer combination(s) and expected fragment(s)
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Sample</TH><TH class="tableheaderadpbl">Fw primer</TH><TH class="tableheaderadpbl">Rv primer</TH><TH class="tableheaderadpbl">Expected fragment</TH><TH class="tableheaderadpbl">Expected size (bp) </TH></TR><TR class="tableunevenadpbl"><TD>1</TD><TD>051</TD><TD>052</TD><TD>pSB1C3</TD><TD>2068</TD></TR><TR class="tableevenadpbl"><TD>2</TD><TD>056</TD><TD>057</TD><TD>dxCas9 (BB)</TD><TD>4226</TD></TR><TR class="tableunevenadpbl"><TD>3</TD><TD>056</TD><TD>055</TD><TD>dxCas9-Lin-Tn5 (BB)</TD><TD>5703</TD></TR><TR class="tableunevenadpbl"><TD>4</TD><TD>001</TD><TD>002</TD><TD>dxCas9 (pACYCDuet-1)</TD><TD>4172</TD></TR><TR class="tableevenadpbl"><TD>5</TD><TD>079</TD><TD>082</TD><TD>Gblock 3 4</TD><TD>5897</TD></TR><TR class="tableunevenadpbl"><TD>6</TD><TD>083</TD><TD>084</TD><TD>pACYCDuet-1</TD><TD>3652</TD></TR></TBODY></TABLE></CENTER><P>The thermocycler programme was installed to have an extension time of 6 minutes 30 seconds and an annealing temperature of 60 °C. After the programme was finished, samples were kept at 4 °C.
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#SundaySCO4">Sunday, 30/09/2018</BUTTON><DIV id="SundaySCO4" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV></DIV></UL></FORM><H3 class="adpbl">October</H3><FORM><LABEL for="WeekSCO1" class="subdropdown cadpbl">Week 1 01/10/2018 - 07/10/2018</LABEL><UL class="submenu"><DIV class="subcontent"><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#MondaySCO1">Monday, 01/10/2018</BUTTON><DIV id="MondaySCO1" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>01/10/2018</DD><DT>Operator</DT><DD>Susan Bouwmeester</DD></DL><H4> Transformation of chemically competent cells </H4><P>The following transformations were carried out according to the protocol for transformation of chemically competent cells:
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Strain</TH><TH class="tableheaderadpbl">Plasmid </TH><TH class="tableheaderadpbl">Cloning method</TH><TH class="tableheaderadpbl">Volume added (uL)</TH><TH class="tableheaderadpbl">Medium on which was plated</TH></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I>DH5α</TD><TD>pSB1C3-fusion</TD><TD>restiction/ligation</TD><TD>2</TD><TD>LB Cam</TD></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I>DH5α</TD><TD>pSB1C3-fusion</TD><TD>restiction/ligation</TD><TD>18</TD><TD>LB Cam</TD></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I>DH5α</TD><TD>pSB1C3-fusion</TD><TD>Gibson Assembly</TD><TD>2</TD><TD>LB Cam</TD></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I>DH5α</TD><TD>pSB1C3-fusion</TD><TD>Gibson Assembly</TD><TD>18</TD><TD>LB Cam</TD></TR></TBODY></TABLE></CENTER><P>800 uL LB medium was added as recovery medium.
</P><H4>DNA purification agarose gel </H4><P>The dxCas9 and the fusion protein have a size of ~4000bp or ~6000bp, but the PCR samples were contaminated with by-products. The products were loaded on a gel prior to <A href="https://2018.igem.org/Team:TUDelft/Experiments#DNApurificationfromgel-scroll" target="_blank">DNA purification from agarose gel conform protocol</A>.
    Elution was done in 30 µL Milli-Q.</P><H4>Nanodrop DNA Quantification</H4><P> 
    After PCR cleanup, all te samples were measered with nanodrop quantification. The following isolates were quantified according to the Nanodrop DNA Quantification protocol. The concentration and purity of the DNA (260/280 and 230/280 ratios) was measured. 
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Isolate</TH><TH class="tableheaderadpbl">Concentration (ng/uL)</TH><TH class="tableheaderadpbl">260/280 ratio</TH><TH class="tableheaderadpbl">260/280 ratio</TH></TR><TR class="tableunevenadpbl"><TD>Fusion 1</TD><TD>72.7</TD><TD>1.84</TD><TD>0.14</TD></TR><TR class="tableunevenadpbl"><TD>Fusion 2</TD><TD>6.09</TD><TD>1.4</TD><TD>0.08</TD></TR><TR class="tableunevenadpbl"><TD>Fusion 3</TD><TD>64.9</TD><TD>1.84</TD><TD>0.07</TD></TR><TR class="tableunevenadpbl"><TD>Fusion 4</TD><TD>4.4</TD><TD>1.17</TD><TD>0.13</TD></TR><TR class="tableunevenadpbl"><TD>Fusion 5</TD><TD>23.2</TD><TD>2.1</TD><TD>0.13</TD></TR><TR class="tableunevenadpbl"><TD>dxCas9 1</TD><TD>279.7</TD><TD>1.85</TD><TD>1.21</TD></TR><TR class="tableunevenadpbl"><TD>dxCas9 2</TD><TD>3.17</TD><TD>1.11</TD><TD>0.24</TD></TR><TR class="tableunevenadpbl"><TD>dxCas9 3</TD><TD>120.1</TD><TD>1.87</TD><TD>1.09</TD></TR><TR class="tableunevenadpbl"><TD>dxCas9 4</TD><TD>2.5</TD><TD>1.26</TD><TD>0.32</TD></TR><TR class="tableunevenadpbl"><TD>dxCas9 5</TD><TD>36.6</TD><TD>1.9</TD><TD>0.9</TD></TR></TBODY></TABLE></CENTER><DL><DT>Date</DT><DD>01/10/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">Liquid Starter Culture Preparation</H4><P><I>E.coli</I> strain harbouring the plasmids pACYCDuet-1_dxCas9, pACYCDuet-1_dxCas9-Lin-Tn5, pSB1C3-mRFP, colony 3 pSB1C3_intron1, colony 8 pSB1C3_intron2, colony 10 pSB1C3_intron 1 and 2, pSB1C3_EPO and pSB1C3_gRNALacZ were used for inoculation of 4 times 10 mL per strain liquid starter culture (Luria Broth, complemented with chloramphenicol) following the <A href="https://2018.igem.org/Team:TUDelft/Experiments#liquidstarterculture-scroll" class="adpbl" target="_blank">Liquid Starter Culture Protocol</A>.
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#TuesdaySCO1">Tuesday, 02/10/2018</BUTTON><DIV id="TuesdaySCO1" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>02/10/2018</DD><DT>Operator</DT><DD>Susan Bouwmeester</DD></DL><H4>Gibson Assembly</H4><P>According to the displayed reaction mixture composition, Gibson Assembly was done to assemble pSB1C3-dxCas9.
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL) pSB1C3-dxCas9-1</TH><TH class="tableheaderadpbl">Volume (uL) pSB1C3-dxCas9-2</TH><TH class="tableheaderadpbl">Volume (uL) pSB1C3-dxCas9-3</TH></TR><TR class="tableunevenadpbl"><TD>Gibson Assembly Mastermix 2X</TD><TD>10</TD><TD>10</TD><TD>10</TD></TR><TR class="tableevenadpbl"><TD>Vector (pSB1C3)</TD><TD>1.5 (200 ng)</TD><TD>7.35 (980 ng)</TD><TD>2.5 (330 ng)</TD></TR><TR class="tableunevenadpbl"><TD>Fragment (dxCas9)</TD><TD>4.3 (1200 ng)</TD><TD>2.2 (600 ng)</TD><TD>7.08 (1900 ng)</TD></TR><TR class="tableunevenadpbl"><TD>MilliQ</TD><TD>4.2</TD><TD>0.45</TD><TD>0.42</TD></TR></TBODY></TABLE></CENTER><P>The samples were incubated at 55°C for 60 minutes, after which they were stored at -20°C.</P><H4>Restriction </H4><P>Restriction cloning was done for pSB1C3 and the dxCas9. The following table shows an overview of what fragments were cut with what restriction enzymes.
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL) - dxCas9bb</TH></TR><TR class="tableunevenadpbl"><TD>10x CutSmart buffer    </TD><TD>5</TD></TR><TR class="tableunevenadpbl"><TD>Fragment</TD><TD>30</TD></TR><TR class="tableunevenadpbl"><TD>Enzyme (EcoRI)</TD><TD>2</TD></TR><TR class="tableunevenadpbl"><TD>Enzyme (PstI)</TD><TD>2</TD></TR><TR class="tableunevenadpbl"><TD>MilliQ</TD><TD> 11</TD></TR></TBODY></TABLE></CENTER><P>The samples were incubated for 4 hours in the incubator 37 °C and followed by 20 minutes heat inactivation 80 °C. 

</P><H4>Colony PCR</H4><P>Colony PCR was done to verify propagation of pSB1C3-dxCas9 in 20 colonies after transformation. This plasmid contains the fusion (~6000bp) and pSB1C3, which can be checked with primers VR and VF2.
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL)</TH></TR><TR class="tableevenadpbl"><TD>Forward primer</TD><TD>0.6</TD></TR><TR class="tableunevenadpbl"><TD>Reverse primer</TD><TD>0.6</TD></TR></TBODY></TABLE></CENTER><P>20 tubes were prepared for colony PCR of pSB1C3-dxCas9 colonies. The thermocycler programme was installed to have an extension time of 5:00 and an annealing temperature of 60 °C. After the programme was finished, samples were kept at 12°C.</P><H4> Transformation of chemically competent cells </H4><P>The following transformations were carried out according to the protocol for transformation of chemically competent cells:
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Strain</TH><TH class="tableheaderadpbl">Plasmid </TH><TH class="tableheaderadpbl">Cloning method</TH><TH class="tableheaderadpbl">Volume added (uL)</TH><TH class="tableheaderadpbl">Medium on which was plated</TH></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I>DH5α</TD><TD>pSB1C3-dxCas9-1</TD><TD>Gibson Assembly</TD><TD>2</TD><TD>LB Cam</TD></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I>DH5α</TD><TD>pSB1C3-dxCas9-1</TD><TD>Gibson Assembly</TD><TD>18</TD><TD>LB Cam</TD></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I>DH5α</TD><TD>pSB1C3-dxCas9-2</TD><TD>Gibson Assembly</TD><TD>2</TD><TD>LB Cam</TD></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I>DH5α</TD><TD>pSB1C3-dxCas9-2</TD><TD>Gibson Assembly</TD><TD>18</TD><TD>LB Cam</TD></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I>DH5α</TD><TD>pSB1C3-dxCas9-3</TD><TD>Gibson Assembly</TD><TD>2</TD><TD>LB Cam</TD></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I>DH5α</TD><TD>pSB1C3-dxCas-9</TD><TD>Gibson Assembly</TD><TD>18</TD><TD>LB Cam</TD></TR></TBODY></TABLE></CENTER><P>800 uL LB medium was added as recovery medium.
</P><DL><DT>Date</DT><DD>02/10/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">Plasmid Isolation</H4><P> 
    Overnight starter cultures of <I>E.coli</I> strains harbouring the plasmids pACYCDuet-1_dxCas9, pACYCDuet-1_dxCas9-Lin-Tn5, pSB1C3-mRFP, colony 3 pSB1C3_intron1, colony 8 pSB1C3_intron2, colony 10 pSB1C3_intron 1 and 2, pSB1C3_EPO and pSB1C3_gRNALacZ  were subjected to plasmid isolation following the <A href="https://2018.igem.org/Team:TUDelft/Experiments#plasmidisolation-scroll" class="adpbl" target="_blank">Plasmid Isolation Protocol</A>. Final elution was done with 30 μL of pre-warmed MilliQ.
</P><DL><DT>Date</DT><DD>02/10/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">Nanodrop DNA Quantification</H4><P>The following isolates were quantified according to the <A href="https://2018.igem.org/Team:TUDelft/Experiments#NanodropDNA-scroll" class="adpbl" target="_blank">Nanodrop DNA Quantification protocol</A>. The concentration and purity of the DNA (260/280 and 260/230 ratios) were measured. </P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Isolate</TH><TH class="tableheaderadpbl">Concentration (ng/uL)</TH><TH class="tableheaderadpbl">260/280 ratio</TH><TH class="tableheaderadpbl">260/230 ratio</TH></TR><TR class="tableunevenadpbl"><TD>pACYCDuet-1_dxCas9 (1)</TD><TD>1447.8</TD><TD>2.06</TD><TD>2.17</TD></TR><TR class="tableevenadpbl"><TD>pACYCDuet-1_dxCas9 (2)</TD><TD>1867</TD><TD>2.09</TD><TD>2.38</TD></TR><TR class="tableunevenadpbl"><TD>pACYCDuet-1_dxCas9-Lin-Tn5 (1)</TD><TD>1256</TD><TD>2.06</TD><TD>2.10</TD></TR><TR class="tableevenadpbl"><TD>pACYCDuet-1_dxCas9-Lin-Tn5 (2)</TD><TD>1361.4</TD><TD>1.99</TD><TD>1.86</TD></TR><TR class="tableunevenadpbl"><TD>pSB1C3-mRFP (1)</TD><TD>738</TD><TD>1.98</TD><TD>2.12</TD></TR><TR class="tableevenadpbl"><TD>pSB1C3-mRFP (2)</TD><TD>737</TD><TD>2.01</TD><TD>2.18</TD></TR><TR class="tableunevenadpbl"><TD>pSB1C3_EPO_intron 1</TD><TD>495.9</TD><TD>1.83</TD><TD>1.72</TD></TR><TR class="tableevenadpbl"><TD>pSB1C3_EPO_intron 2</TD><TD>422.2</TD><TD>1.79</TD><TD>1.38</TD></TR><TR class="tableunevenadpbl"><TD>pSB1C3_EPO_intron 1 and 2</TD><TD>391.8</TD><TD>1.87</TD><TD>1.99</TD></TR><TR class="tableevenadpbl"><TD>pSB1C3_EPO</TD><TD>462.7</TD><TD>2.01</TD><TD>1.70</TD></TR><TR class="tableunevenadpbl"><TD>pSB1C3_LacZgRNA</TD><TD>347.6</TD><TD>2.01</TD><TD>1.68</TD></TR></TBODY></TABLE></CENTER><DL><DT>Date</DT><DD>30/08/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">PCR Clean-Up</H4><P>50 μL of each PCR mixture performed on 29/09/2018was subjected to PCR Clean-Up, according to the <A href="https://2018.igem.org/Team:TUDelft/Experiments#PCRcleanup-scroll" class="adpbl" target="_blank">protocol</A>. Elution was done in 30 µL Milli-Q.</P><DL><DT>Date</DT><DD>02/10/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">Nanodrop DNA Quantification</H4><P>The following isolates were quantified according to the <A href="https://2018.igem.org/Team:TUDelft/Experiments#NanodropDNA-scroll" class="adpbl" target="_blank">Nanodrop DNA Quantification protocol</A>. The concentration and purity of the DNA (260/280 and 260/230 ratios) were measured. </P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Isolate</TH><TH class="tableheaderadpbl">Concentration (ng/uL)</TH><TH class="tableheaderadpbl">260/280 ratio</TH><TH class="tableheaderadpbl">260/230 ratio</TH></TR><TR class="tableunevenadpbl"><TD>dxCas9 (pACYCDuet-1)</TD><TD>62.8</TD><TD>1.79</TD><TD>0.99</TD></TR><TR class="tableevenadpbl"><TD>dxCas9 (BioBrick)</TD><TD>197.1</TD><TD>1.84</TD><TD>2.09</TD></TR><TR class="tableunevenadpbl"><TD>dxCas9-Lin-Tn5 (pACYCDuet-1)</TD><TD>277.2</TD><TD>1.84</TD><TD>2.21</TD></TR><TR class="tableevenadpbl"><TD>dxCas9-Lin-Tn5 (BioBrick)</TD><TD>340.6</TD><TD>1.83</TD><TD>2.16</TD></TR><TR class="tableunevenadpbl"><TD>pSB1C3 backbone (1)</TD><TD>396.4</TD><TD>1.85</TD><TD>1.83</TD></TR><TR class="tableevenadpbl"><TD>pACYCDuet-1 backbone</TD><TD>553.1</TD><TD>1.84</TD><TD>2.19</TD></TR><TR class="tableunevenadpbl"><TD>pSB1C3_EPO_intron 1</TD><TD>495.9</TD><TD>1.83</TD><TD>1.72</TD></TR><TR class="tableevenadpbl"><TD>pSB1C3_EPO_intron 2</TD><TD>422.2</TD><TD>1.79</TD><TD>1.38</TD></TR><TR class="tableunevenadpbl"><TD>pSB1C3_EPO_intron 1 and 2</TD><TD>391.8</TD><TD>1.87</TD><TD>1.99</TD></TR><TR class="tableevenadpbl"><TD>pSB1C3_EPO</TD><TD>462.7</TD><TD>2.01</TD><TD>1.70</TD></TR><TR class="tableunevenadpbl"><TD>pSB1C3_LacZgRNA</TD><TD>347.6</TD><TD>2.01</TD><TD>1.68</TD></TR></TBODY></TABLE></CENTER></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#WednesdaySCO1">Wednesday, 03/10/2018</BUTTON><DIV id="WednesdaySCO1" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>03/10/2018</DD><DT>Operator</DT><DD>Susan Bouwmeester</DD></DL><H4>PCR clean-up</H4><P>
    PCR clean-up is done according to the protocol for pSB1C3, dxCas9fs and fusionfs previous made. Elution was done in 30 µL Milli-Q </P><H4>DNA Gel Electrophoresis</H4><P>PCR product of the colony PCR of fusion was run on a 0.8% agarose gel in TAE buffer. Only 4 colonies turned out to be white. A total of 9 colony PCR results were put on a gel. 5μL was pipetted into each lane. The gel was imaged using a gel documentation system with UV-light.</P><B>Figure</B> Colony PCR fusion. 0.8% agarose. Expected size fusion ~6000bp. Lane 1: 500bp ladder; Lane 2: colony 2; Lane 3: colony 5; Lane 4: colony 9; Lane 5: colony 14; Lane 6: positive control; Lane 7: Empty; Lane 8: colony 1 (red); Lane 9: colony 3 (red); Lane 10: colony 6 (red); Lane 11: colony 10 (red); Lane 12: colony 15 (red); Lane 13: negative control (no template) <P>The amplified product colony PCR is not around the correct hight. No succesful integration of fusion in pSB1C3</P><H4>Colony PCR</H4><P>Colony PCR was done to verify propagation of pSB1C3-dxCas9 in 8 colonies after transformation. This plasmid contains the dxcas9 (~4000bp) and pSB1C3, which can be checked with primers VR and VF2.
</P><P>20 tubes were prepared for colony PCR of pSB1C3-fusion colonies. The thermocycler programme was installed to have an extension time of 8:00 and an annealing temperature of 60 °C. After the programme was finished, samples were kept at 12°C.</P><H4>Restriction </H4><P>Restriction cloning was done for pSB1C3 and the fusion protein. The following table shows an overview of what fragments were cut with what restriction enzymes.
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL) - pSB1C3-linear</TH><TH class="tableheaderadpbl">Volume (uL) - fusionbb</TH><TH class="tableheaderadpbl">Volume (uL) - pSB1C3-RFP</TH><TH class="tableheaderadpbl">Volume (uL) - dxCas9bb</TH></TR><TR class="tableunevenadpbl"><TD>10x CutSmart buffer    </TD><TD>5</TD><TD>5</TD><TD>5</TD><TD>5</TD></TR><TR class="tableunevenadpbl"><TD>Fragment</TD><TD>41 (~3700ng)</TD><TD>41 (~3000ng)</TD><TD>20 (~8480 ng)</TD><TD>20 (~2220 ng)</TD></TR><TR class="tableunevenadpbl"><TD>Enzyme (EcoRI)</TD><TD>2</TD><TD>2</TD><TD>2</TD><TD>2</TD></TR><TR class="tableunevenadpbl"><TD>Enzyme (SpeI)</TD><TD>2</TD><TD>2</TD><TD>2</TD><TD>2</TD></TR><TR class="tableunevenadpbl"><TD>MilliQ</TD><TD> 0</TD><TD>0</TD><TD>21</TD><TD>21</TD></TR></TBODY></TABLE></CENTER><P>The samples were incubated for 4 hours in the incubator 37 °C. 

</P><H4>Nanodrop DNA Quantification</H4><P> 
    After PCR cleanup, all te samples were measered with nanodrop quantification. The following isolates were quantified according to the Nanodrop DNA Quantification protocol. The concentration was measured. 
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Isolate</TH><TH class="tableheaderadpbl">Concentration (ng/uL)</TH></TR><TR class="tableunevenadpbl"><TD>pSB1C3 (1.1.1)</TD><TD>2290</TD></TR><TR class="tableunevenadpbl"><TD>pSB1C3 (1.2.1)</TD><TD>2215</TD></TR><TR class="tableunevenadpbl"><TD>pSB1C3 (1.1.2)</TD><TD>430</TD></TR><TR class="tableunevenadpbl"><TD>pSB1C3 (1.2.2)</TD><TD>770.6</TD></TR><TR class="tableunevenadpbl"><TD>dxCas9 (2.1)</TD><TD>2541.4</TD></TR><TR class="tableunevenadpbl"><TD>dxCas9 (2.2)</TD><TD>690</TD></TR><TR class="tableunevenadpbl"><TD>dxCas9 (4.1)</TD><TD>1823</TD></TR><TR class="tableunevenadpbl"><TD>dxCas9 (4.2)</TD><TD>668.8</TD></TR></TBODY></TABLE></CENTER><H4>Gibson Assembly</H4><P>According to the displayed reaction mixture composition, Gibson Assembly was done to assemble pSB1C3-dxCas9fs and pSB1C3-fusionfs.
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL) pSB1C3-fusionfs-5</TH><TH class="tableheaderadpbl">Volume (uL) pSB1C3-fusionfs-10</TH><TH class="tableheaderadpbl">Volume (uL) pSB1C3-dxCas9fs-5</TH><TH class="tableheaderadpbl">Volume (uL) pSB1C3-dxCas9fs-10</TH></TR><TR class="tableunevenadpbl"><TD>Gibson Assembly Mastermix 2X</TD><TD>10</TD><TD>10</TD><TD>10</TD><TD>10</TD></TR><TR class="tableevenadpbl"><TD>Vector (pSB1C3)</TD><TD>1.5 (670 ng)</TD><TD>1 (140 ng)</TD><TD>1.5 (670 ng)</TD><TD>1 (140 ng)</TD></TR><TR class="tableunevenadpbl"><TD>Fragment</TD><TD>5.5 (1823 ng)</TD><TD>6.3 (668 ng)</TD><TD>2.6 (2541 ng)</TD><TD>4.1 (690 ng)</TD></TR><TR class="tableunevenadpbl"><TD>MilliQ</TD><TD>3</TD><TD>2.7</TD><TD>5.9</TD><TD>4.9</TD></TR></TBODY></TABLE></CENTER><P>The samples were incubated at 55°C for 60 minutes.</P><P>After cooling down, 1uL DpnI is added. The samples were incubated at 37°C for 60 minutes. Heatinactivation for 20 minutes at 60°C </P><H4> Transformation of chemically competent cells </H4><P>Transformations was carried out according to the protocol. The 2 and 18 uL of product of Gibson Assembly was transformed into <I>E. coli</I>DH5α   
</P><P>800 uL LB medium was added as recovery medium.
</P><H4>DNA gel purification</H4><P>DNA purification of fusionbb from agarose gel conform protocol. Elution was done in one time 15 µL Milli-Q and one time 20 µL.
</P><H4>Colony PCR</H4><P>9 tubes were prepared for colony PCR of pSB1C3-dxCas9 colonies. The thermocycler programme was installed to have an extension time of 5:00 and an annealing temperature of 60 °C. After the programme was finished, samples were kept at 12°C.</P><H4>PCR clean-up</H4><P>
    PCR clean-up is done according to the protocol for restriced pSB1C3-RFP, restricted fusion, restricted pSB1C3 and restricted dxCas9. Elution was done in 30 µL Milli-Q </P><H4>Nanodrop DNA Quantification</H4><P> 
    After PCR cleanup, all te samples were measered with nanodrop quantification. The following isolates were quantified according to the Nanodrop DNA Quantification protocol. The concentration and purity (280/260 and 230/260 ratio) was measured. 
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Isolate</TH><TH class="tableheaderadpbl">Concentration (ng/uL)</TH><TH class="tableheaderadpbl">280/260 ratio </TH><TH class="tableheaderadpbl">230/260 ratio</TH></TR><TR class="tableunevenadpbl"><TD>restricted pSB1C3RFP</TD><TD>98.6</TD><TD>1.6</TD><TD>1.8</TD></TR><TR class="tableunevenadpbl"><TD>restricted fusionbb</TD><TD>10.7</TD><TD>0.4</TD><TD>1.7</TD></TR><TR class="tableunevenadpbl"><TD>restricted pSB1C3lin</TD><TD>27.3</TD><TD>1</TD><TD>1.9</TD></TR><TR class="tableunevenadpbl"><TD>restricted dxCas9bb</TD><TD>15.0</TD><TD>0.6</TD><TD>1.68</TD></TR></TBODY></TABLE></CENTER><H4>Ligation</H4><P>In order to assemble pSB1C3-fusion and pSB1C3-dxCas9, ligation was performed. </P><P>A 10ul ligation reaction mixture was prepared with the following composition</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">pSB1C3-Fusion Volume(uL) </TH><TH class="tableheaderadpbl">pSB1C-dxCas9 </TH></TR><TR class="tableunevenadpbl"><TD>Ligase Buffer (10x) </TD><TD>2</TD><TD>2</TD></TR><TR class="tableevenadpbl"><TD>T4 DNA Ligase</TD><TD>1</TD><TD>1</TD></TR><TR class="tableunevenadpbl"><TD>DNA vector </TD><TD>0.25 uL (~25 ng) </TD><TD>0.40 uL (~40 ng)</TD></TR><TR class="tableunevenadpbl"><TD>DNA fragment </TD><TD>16.7 uL (~170 ng)</TD><TD>16.6 uL (~225 ng)</TD></TR><TR class="tableunevenadpbl"><TD>MilliQ</TD><TD>0 uL</TD><TD>0 uL</TD></TR></TBODY></TABLE></CENTER><P>The samples were incubated overnight on room temperature for ligation.</P><DL><DT>Date</DT><DD>03/10/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">Restriction</H4><P>The following table shows an overview of what fragment(s) were cut with what restriction enzyme(s) to obtain fragments Tn5-Lin-dxCas9 and pACYCDuet-1 for ligation.
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL)</TH><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL)</TH><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL)</TH><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL)</TH></TR><TR class="tableunevenadpbl"><TD>10x CutSmart</TD><TD>5</TD><TD>10x CutSmart</TD><TD>2</TD><TD>10x CutSmart</TD><TD>2</TD><TD>10x CutSmart</TD><TD>2</TD></TR><TR class="tableevenadpbl"><TD>pSB1C3-mRFP</TD><TD>3 (~2000 ng)</TD><TD>pSB1C3 backbone</TD><TD>6 (~2000 ng)</TD><TD>dxCas9</TD><TD>12 (~2000 ng)</TD><TD>dxCas9-Lin-Tn5</TD><TD>8 (~2000 ng)</TD></TR><TR class="tableunevenadpbl"><TD>Enzyme (EcoRI)</TD><TD>1</TD><TD>Enzyme (EcoRI)</TD><TD>1</TD><TD>Enzyme (EcoRI)</TD><TD>1</TD><TD>Enzyme (EcoRI)</TD><TD>1</TD></TR><TR class="tableevenadpbl"><TD>Enzyme (PstI)</TD><TD>1</TD><TD>Enzyme (PstI)</TD><TD>1</TD><TD>Enzyme (PstI)</TD><TD>1</TD><TD>Enzyme (PstI)</TD><TD>1</TD></TR><TR class="tableunevenadpbl"><TD>MilliQ</TD><TD>13</TD><TD>MilliQ</TD><TD>10</TD><TD>MilliQ</TD><TD>4</TD><TD>MilliQ</TD><TD>8</TD></TR></TBODY></TABLE></CENTER><P>The samples were incubated for 4 hours at 37 °C, and followed by 20 minute heat inactivation of the restriction enzyme(s). 
</P><DL><DT>Date</DT><DD>30/08/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">PCR Clean-Up</H4><P>20 μL of each restriction reaction mixture was subjected to PCR Clean-Up, according to the <A href="https://2018.igem.org/Team:TUDelft/Experiments#PCRcleanup-scroll" class="adpbl" target="_blank">protocol</A>. Elution was done in 30 µL Milli-Q.</P><DL><DT>Date</DT><DD>30/08/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">Nanodrop DNA Quantification</H4><P>The following isolates were quantified according to the <A href="https://2018.igem.org/Team:TUDelft/Experiments#NanodropDNA-scroll" class="adpbl" target="_blank">Nanodrop DNA Quantification protocol</A>. The concentration and purity of the DNA (260/280 and 260/230 ratios) were measured. </P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Isolate</TH><TH class="tableheaderadpbl">Concentration (ng/uL)</TH><TH class="tableheaderadpbl">260/280 ratio</TH><TH class="tableheaderadpbl">260/230 ratio</TH></TR><TR class="tableunevenadpbl"><TD>pSB1C3-mRFP cut with EcoRI and PstI</TD><TD>15.0</TD><TD>1.91</TD><TD>0.48</TD></TR><TR class="tableevenadpbl"><TD>pSB1C3 backbone cut with EcoRI and PstI</TD><TD>63.3</TD><TD>1.85</TD><TD>1.11</TD></TR><TR class="tableunevenadpbl"><TD>dxCas9 cut with EcoRI and PstI</TD><TD>52.3</TD><TD>1.85</TD><TD>0.75</TD></TR><TR class="tableevenadpbl"><TD>dxCas9-Lin-Tn5 cut with EcoRI and PstI</TD><TD>67.1</TD><TD>1.86</TD><TD>0.90</TD></TR></TBODY></TABLE></CENTER><DL><DT>Date</DT><DD>30/08/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">Ligation</H4><P>In order to assemble cut fragments pSB1C3 with dxCas9 or dxCas9-Lin-Tn5 ligation was performed. This table gives an overview of which fragments were cut with which enzymes.</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">DNA</TH><TH class="tableheaderadpbl">Size (bp) </TH><TH class="tableheaderadpbl">Cut with enzymes</TH></TR><TR class="tableunevenadpbl"><TD>pSB1C3</TD><TD>2068</TD><TD>EcoRI and PstI</TD></TR><TR class="tableevenadpbl"><TD>Fragment dxCas9</TD><TD>4226</TD><TD>EcoRI and PstI</TD></TR><TR class="tableunevenadpbl"><TD>Fragment dxCas9-Lin-Tn5</TD><TD>5703</TD><TD>EcoRI and PstI</TD></TR></TBODY></TABLE></CENTER><P>A 20 μL ligation reaction mixture was prepared with the following composition:</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (μL)</TH><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (μL)</TH></TR><TR class="tableunevenadpbl"><TD>Ligase Buffer (10X)</TD><TD>2</TD><TD>Ligase Buffer (10X)</TD><TD>2</TD></TR><TR class="tableevenadpbl"><TD>T4 DNA Ligase </TD><TD>1</TD><TD>T4 DNA Ligase </TD><TD>1</TD></TR><TR class="tableunevenadpbl"><TD>DNA vector pSB1C3 (~100 ng)</TD><TD>2</TD><TD>DNA vector pSB1C3 (~100 ng)</TD><TD>2</TD></TR><TR class="tableevenadpbl"><TD>DNA dxCas9 (~600 ng)</TD><TD>12</TD><TD>DNA dxCas9-Lin-Tn5 (~700 ng)</TD><TD>14</TD></TR><TR class="tableunevenadpbl"><TD>MilliQ</TD><TD>3</TD><TD>MilliQ</TD><TD>0</TD></TR></TBODY></TABLE></CENTER><P>The samples were incubated overnight at 4 °C for ligation.</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#ThursdaySCO1">Thursday, 04/10/2018</BUTTON><DIV id="ThursdaySCO1" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>04/10/2018</DD><DT>Operator</DT><DD>Susan Bouwmeester</DD></DL><H4>Nanodrop DNA Quantification</H4><P> 
  
    After PCR cleanup, all te samples were measered with nanodrop quantification. The following isolates were quantified according to the Nanodrop DNA Quantification protocol. The concentration and purity (280/260 and 230/260 ratio) was measured. 
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Isolate</TH><TH class="tableheaderadpbl">Concentration (ng/uL)</TH><TH class="tableheaderadpbl">280/260 ratio </TH><TH class="tableheaderadpbl">230/260 ratio</TH></TR><TR class="tableunevenadpbl"><TD>fusion A </TD><TD>230.4</TD><TD>1.88</TD><TD>1.87</TD></TR><TR class="tableunevenadpbl"><TD>fusion B</TD><TD>97.8</TD><TD>1.87</TD><TD>1.84</TD></TR></TBODY></TABLE></CENTER><H4>Restriction </H4><P>Restriction cloning was done for pSB1C3 and the fusion B. The following table shows an overview of what fragments were cut with what restriction enzymes. Restriction enzymes from the kitchen are used. 
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL) - pSB1C3</TH><TH class="tableheaderadpbl">Volume (uL) - fusion B</TH></TR><TR class="tableunevenadpbl"><TD>10x CutSmart buffer    </TD><TD>5</TD><TD>5</TD></TR><TR class="tableunevenadpbl"><TD>Fragment</TD><TD>25 (~2500 ng)</TD><TD>25 (~3500 ng)</TD></TR><TR class="tableunevenadpbl"><TD>Enzyme (EcoRI-HF)</TD><TD>2</TD><TD>2</TD></TR><TR class="tableunevenadpbl"><TD>Enzyme (PstI-HF)</TD><TD>2</TD><TD>2</TD></TR><TR class="tableunevenadpbl"><TD>MilliQ</TD><TD> 16</TD><TD>16</TD></TR></TBODY></TABLE></CENTER><P>The samples were incubated for 4 hours in the incubator 37 °C and followed by 20 minutes heat inactivation 80 °C. 

</P><H4>Colony PCR</H4><P>Colony PCR was done to verify propagation of pSB1C3-dxCas9 and pSB1C3-fusion in after transformation (transformation done 2/10 and 1/10). This plasmid contains the dxcas9 (~4000bp) or fusion (~6000bp) and pSB1C3 (~2000bp), which can be checked with primers VR and VF2.
</P><P>6 tubes were prepared for colony PCR of pSB1C3-fusion colonies. The thermocycler programme was installed to have an extension time of 10:00 and an annealing temperature of 60 °C. After the programme was finished, samples were kept at 12°C.</P><H4>Gibson Assembly</H4><P>According to the displayed reaction mixture composition, Gibson Assembly was done to assemble pSB1C3-dxCas9 and pSB1C3-fusion.
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL) pSB1C3-fusionfs-5</TH><TH class="tableheaderadpbl">Volume (uL) pSB1C3-fusionfs-10</TH></TR><TR class="tableunevenadpbl"><TD>Gibson Assembly Mastermix 2X</TD><TD>10</TD><TD>10</TD></TR><TR class="tableevenadpbl"><TD>Vector (pSB1C3)</TD><TD>0.34 (146 ng)</TD><TD>0.52 (72.8 ng)</TD></TR><TR class="tableunevenadpbl"><TD>Fragment (fusion)</TD><TD>9.66 (2200 ng)</TD><TD>9.48 (2200 ng)</TD></TR><TR class="tableunevenadpbl"><TD>MilliQ</TD><TD>0</TD><TD>0</TD></TR></TBODY></TABLE></CENTER><P>The samples were incubated at 55°C for 60 minutes.</P><P>After cooling down, 1uL DpnI is added. The samples were incubated at 37°C for 60 minutes. Heatinactivation for 20 minutes at 60°C </P><H4>PCR clean-up</H4><P>
    PCR clean-up is done according to the protocol for restricted fusion and pSB1C3. Elution was done in 30 µL Milli-Q </P><H4>Ligation</H4><P>In order to assemble pSB1C3-fusion, ligation was performed. </P><P>A 10ul ligation reaction mixture was prepared with the following composition</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">pSB1C3-Fusion Volume(uL) </TH><TH class="tableheaderadpbl">pSB1C3-Fusion Volume(uL) </TH></TR><TR class="tableunevenadpbl"><TD>Ligase Buffer (10x) </TD><TD>2</TD><TD>2</TD></TR><TR class="tableevenadpbl"><TD>T4 DNA Ligase</TD><TD>1</TD><TD>1</TD></TR><TR class="tableunevenadpbl"><TD>DNA vector </TD><TD>0.33 uL (~30 ng) </TD><TD>0.28 uL (~25 ng)</TD></TR><TR class="tableunevenadpbl"><TD>DNA fragment </TD><TD>15 uL (~200 ng)</TD><TD>15 uL (~750 ng)</TD></TR><TR class="tableunevenadpbl"><TD>MilliQ</TD><TD>2 uL</TD><TD>2 uL</TD></TR></TBODY></TABLE></CENTER><P>The samples were incubated overnight on room temperature for ligation.</P><H4>High Fidelity PCR</H4><P> To amplify fusionbb (~6000bp), high fidelity PCR was done using fusionbb1 as template DNA and fw_BB_dxCas9 and rv_BB_Tn5 forward and reverse primers respectively. 16 50µL PCR mixtures were prepared, according to the High Fidelity PCR protocol.
    </P><P>The thermocycler programme was installed to have an extension time of 6:30 minutes and an annealing temperature of 70 °C. After the programme was finished, samples were kept at 4 °C.</P><H4> Transformation of chemically competent cells </H4><P>The following transformations were carried out according to the protocol for transformation of chemically competent cells:
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Strain</TH><TH class="tableheaderadpbl">Plasmid </TH><TH class="tableheaderadpbl">Cloning method</TH><TH class="tableheaderadpbl">Volume added (uL)</TH><TH class="tableheaderadpbl">Medium on which was plated</TH></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I>DH5α NEB</TD><TD>pSB1C3-fusion</TD><TD>restiction/ligation (2/10)</TD><TD>2</TD><TD>LB Cam</TD></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I>DH5α NEB</TD><TD>pSB1C3-fusion</TD><TD>restiction/ligation (2/10)</TD><TD>18</TD><TD>LB Cam</TD></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I>DH5α NEB</TD><TD>pSB1C3-dxCas9</TD><TD>restiction/ligation (2/10)</TD><TD>2</TD><TD>LB Cam</TD></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I>DH5α NEB</TD><TD>pSB1C3-dxCas9</TD><TD>restiction/ligation (2/10)</TD><TD>18</TD><TD>LB Cam</TD></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I>DH5α NEB</TD><TD>pSB1C3-fusion-1</TD><TD>Gibson Assembly</TD><TD>2</TD><TD>LB Cam</TD></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I>DH5α NEB</TD><TD>pSB1C3-fusion-1</TD><TD>Gibson Assembly</TD><TD>18</TD><TD>LB Cam</TD></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I>DH5α NEB</TD><TD>pSB1C3-fusion-2</TD><TD>Gibson Assembly</TD><TD>2</TD><TD>LB Cam</TD></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I>DH5α NEB</TD><TD>pSB1C3-fusion-2</TD><TD>Gibson Assembly</TD><TD>18</TD><TD>LB Cam</TD></TR></TBODY></TABLE></CENTER><P>800 uL LB medium was added as recovery medium.
</P><DL><DT>Date</DT><DD>04/10/2018</DD><DT>Operator</DT><DD>Nicole Bennis</DD></DL><H4 class="adpbl">Transformation of chemically competent cells</H4><P>The following transformations were carried out according to the protocol for transformation of chemically competent cells:</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Strain</TH><TH class="tableheaderadpbl">Plasmid and size (bp)</TH><TH class="tableheaderadpbl">Cloning method</TH><TH class="tableheaderadpbl">Volume added (μl)</TH><TH class="tableheaderadpbl">Medium on which was plated</TH></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I> DH5α</TD><TD>pSB1C3_dxCas9 (6217 bp)</TD><TD>Restriction/Ligation</TD><TD>2.5</TD><TD>LB-Cam</TD></TR><TR class="tableevenadpbl"><TD><I>E.coli</I> DH5α</TD><TD>pSB1C3_dxCas9-Lin-Tn5 (7699 bp)</TD><TD>Restriction/Ligation</TD><TD>2.5</TD><TD>LB Cam</TD></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I> DH5α</TD><TD>pSB1C3 cut with EcoRI and PstI (neg control) (2060 bp)</TD><TD>Restriction/Ligation</TD><TD>2.5</TD><TD>LB Cam</TD></TR></TBODY></TABLE></CENTER><P>
200 μl LB medium was added as recovery medium.</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#FridaySCO1">Friday, 05/10/2018</BUTTON><DIV id="FridaySCO1" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>05/10/2018</DD><DT>Operator</DT><DD>Susan Bouwmeester</DD></DL><H4>Restriction </H4><P>DpnI digestion is done for pSB1C3, fusionfs and dxCas9fs. 
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL) pSB1C3</TH><TH class="tableheaderadpbl">Volume (uL) fusionfs</TH><TH class="tableheaderadpbl">Volume (uL) dxCas9fs</TH></TR><TR class="tableunevenadpbl"><TD>10x CutSmart buffer    </TD><TD>5</TD><TD>5</TD><TD>5</TD></TR><TR class="tableunevenadpbl"><TD>Fragment</TD><TD>42 </TD><TD>42 </TD><TD>42 </TD></TR><TR class="tableunevenadpbl"><TD>Enzyme (EcoRI)</TD><TD>2</TD><TD>2</TD><TD>2</TD><TD>2</TD></TR><TR class="tableunevenadpbl"><TD>Enzyme (DpnI)</TD><TD>3</TD><TD>3</TD><TD>3</TD></TR></TBODY></TABLE></CENTER><P>The samples were incubated for 4 hours in the incubator 37 °C. 

</P><H4>Colony PCR</H4><P>Colony PCR was done to verify propagation of pSB1C3-fusionfs in 20 colonies after transformation. This plasmid contains the fusionfs (~6000bp) and pSB1C3, which can be checked with primers VR and VF2.
</P><P>20 tubes were prepared for colony PCR of pSB1C3-fusionfs colonies. The thermocycler programme was installed to have an extension time of 8:00 and an annealing temperature of 60 °C. After the programme was finished, samples were kept at 12°C.</P><H4>Colony PCR</H4><P>Colony PCR was done to verify propagation of pSB1C3-fusionfs in 20 colonies after transformation. This plasmid contains the dxCas9fs (~4000bp) and pSB1C3, which can be checked with primers VR and VF2.
</P><P>20 tubes were prepared for colony PCR of pSB1C3-dxCas9fs colonies. The thermocycler programme was installed to have an extension time of 5:00 and an annealing temperature of 60 °C. After the programme was finished, samples were kept at 12°C.</P><H4>DNA Gel Electrophoresis</H4><P>PCR product of the colony PCR of fusion was run on a 0.8% agarose gel in TAE buffer. A total of 20 colony PCR results were put on a gel. 5μL was pipetted into each lane. The gel was imaged using a gel documentation system with UV-light.</P><B>Figure</B> Colony PCR fusionfs. 0.8% agarose. Expected size fusion ~6000bp. Lane 1: 200bp ladder; Lane 2: colony 1; Lane 3: colony 2; Lane 4: colony 3; Lane 5: colony 4; Lane 6: colony 5; Lane 7: colony 6; Lane 8: colony 7; Lane 9: colony 8; Lane 10: colony 9; Lane 11: colony 10; Lane 12: colony 11; Lane 13: colony 12; Lane 14: colony 13; Lane 15: colony 14; Lane 16: 200bp ladder. Lane 17: colony 15; Lane 18: colony 16; Lane 19: colony 17; Lane 20: colony 18; Lane 21: colony 19; Lane 22: colony 20 <P>Overnight culture is started for colony number 1, 2 and 8</P><H4>DNA Gel Electrophoresis</H4><P>PCR product of the colony PCR of dxCas9fs was run on a 0.8% agarose gel in TAE buffer. A total of 20 colony PCR results were put on a gel. 5μL was pipetted into each lane. The gel was imaged using a gel documentation system with UV-light.</P><B>Figure</B> Colony PCR dxCas9fs. 0.8% agarose. Expected size dxCas9 ~4000bp. Lane 1: 200bp ladder; Lane 2: colony 1; Lane 3: colony 2; Lane 4: colony 3; Lane 5: colony 4; Lane 6: colony 5; Lane 7: colony 6; Lane 8: colony 7; Lane 9: colony 8; Lane 10: colony 9; Lane 11: colony 10; Lane 12: colony 11; Lane 13: colony 12; Lane 14: colony 13; Lane 15: colony 14; Lane 16: 200bp ladder. Lane 17: colony 15; Lane 18: colony 16; Lane 19: colony 17; Lane 20: colony 18; Lane 21: colony 19; Lane 22: colony 20 <P>Overnight culture is started for colony number 1, 7, 8 and 15</P><H4>Colony PCR</H4><P>Colony PCR was done to verify propagation of pSB1C3-dxcas9 in 17 colonies after transformation. This plasmid contains the dxcas9 (~4000bp) and pSB1C3, which can be checked with primers VR and VF2.
</P><P>17 tubes were prepared for colony PCR of pSB1C3-dxcas9 colonies. The thermocycler programme was installed to have an extension time of 8:00 and an annealing temperature of 60 °C. After the programme was finished, samples were kept at 12°C.</P><H4>Colony PCR</H4><P>Colony PCR was done to verify propagation of pSB1C3-fusion in 33 colonies after transformation. This plasmid contains the fusion (~6000bp) and pSB1C3, which can be checked with primers VR and VF2.
</P><P>33 tubes were prepared for colony PCR of pSB1C3-fusion colonies. The thermocycler programme was installed to have an extension time of 8:00 and an annealing temperature of 60 °C. After the programme was finished, samples were kept at 12°C.</P><H4>DNA gel purification</H4><P>DNA purification of fusionbb from agarose gel conform protocol. Elution was done in one time 15 µL Milli-Q and one time 20 µL.
</P><H4>Nanodrop DNA Quantification</H4><P> 
    After PCR cleanup, all te samples were measered with nanodrop quantification. The following isolates were quantified according to the Nanodrop DNA Quantification protocol. The concentration and purity (280/260 and 230/260 ratio) was measured. 
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Isolate</TH><TH class="tableheaderadpbl">Concentration (ng/uL)</TH><TH class="tableheaderadpbl">280/260 ratio </TH><TH class="tableheaderadpbl">230/260 ratio</TH></TR><TR class="tableunevenadpbl"><TD>Cas</TD><TD>698.0</TD><TD>1.84</TD><TD>2.01</TD></TR><TR class="tableunevenadpbl"><TD>Cas 2</TD><TD>298.4</TD><TD>1.89</TD><TD>1.95</TD></TR><TR class="tableunevenadpbl"><TD>Fusion</TD><TD>345.2</TD><TD>1.88</TD><TD>1.76</TD></TR><TR class="tableunevenadpbl"><TD>Fusion 2</TD><TD>165.4</TD><TD>1.8</TD><TD>1.6</TD></TR></TBODY></TABLE></CENTER></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#SaturdaySCO1">Saturday, 06/10/2018</BUTTON><DIV id="SaturdaySCO1" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>06/10/2018</DD><DT>Operator</DT><DD>Susan Bouwmeester</DD></DL><H4>Restriction </H4><P>Restriction cloning was done for pSB1C3 and the fusion protein. The following table shows an overview of what fragments were cut with what restriction enzymes.
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL) - pSB1C3en</TH><TH class="tableheaderadpbl">Volume (uL) - linker</TH></TR><TR class="tableunevenadpbl"><TD>10x CutSmart buffer    </TD><TD>3</TD><TD>3</TD></TR><TR class="tableunevenadpbl"><TD>Fragment</TD><TD>8.5</TD><TD>8.5</TD></TR><TR class="tableunevenadpbl"><TD>Enzyme (EcoRI)</TD><TD>1</TD><TD>1</TD></TR><TR class="tableunevenadpbl"><TD>Enzyme (PstI)</TD><TD>1</TD><TD>1</TD></TR><TR class="tableunevenadpbl"><TD>MilliQ</TD><TD> 16.5</TD><TD>16.5</TD></TR></TBODY></TABLE></CENTER><P>The samples were incubated for 4 hours in the incubator 37 °C. 

</P><H4>Gibson Assembly</H4><P>According to the displayed reaction mixture composition, Gibson Assembly was done to assemble pSB1C3-linker.
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL) pSB1C3-linker-10</TH><TH class="tableheaderadpbl">Volume (uL) pSB1C3-linker-50</TH><TH class="tableheaderadpbl">Volume (uL) pSB1C3-linker-100</TH></TR><TR class="tableunevenadpbl"><TD>Gibson Assembly Mastermix 2X</TD><TD>10</TD><TD>10</TD><TD>10</TD></TR><TR class="tableevenadpbl"><TD>Vector (pSB1C3)</TD><TD>1.44 (600ng)</TD><TD>1.44 (600ng)</TD><TD>0.7 (300ng)</TD></TR><TR class="tableunevenadpbl"><TD>Fragment</TD><TD>1.45 (180 ng)</TD><TD>3.7 (900 ng)</TD><TD>3.7 (900 ng)</TD></TR><TR class="tableunevenadpbl"><TD>MilliQ</TD><TD>7.11</TD><TD>4.9</TD><TD>5.6</TD></TR></TBODY></TABLE></CENTER><P>The samples were incubated at 55°C for 20 minutes.</P><H4>Gibson Assembly</H4><P>According to the displayed reaction mixture composition, Gibson Assembly was done to assemble pSB1C3-dxCas9.
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL) pSB1C3-dxCas9-5</TH><TH class="tableheaderadpbl">Volume (uL) pSB1C3-dxCas9-10</TH><TH class="tableheaderadpbl">Volume (uL) pSB1C3-dxCas9-25</TH></TR><TR class="tableunevenadpbl"><TD>Gibson Assembly Mastermix 2X</TD><TD>10</TD><TD>10</TD><TD>10</TD></TR><TR class="tableevenadpbl"><TD>Vector (pSB1C3)</TD><TD>0.34 (576ng)</TD><TD>0.18 (300ng)</TD><TD>0.06 (50ng)</TD></TR><TR class="tableunevenadpbl"><TD>Fragment</TD><TD>8.2 (5760 ng)</TD><TD>8.6 (6000 ng)</TD><TD>7.1 (2560)/td&gt;
  </TD></TR><TR class="tableunevenadpbl"><TD>MilliQ</TD><TD>1.46</TD><TD>1.22</TD><TD>2.9</TD></TR></TBODY></TABLE></CENTER><P>The samples were incubated at 55°C for 60 minutes.</P><H4>Gibson Assembly</H4><P>According to the displayed reaction mixture composition, Gibson Assembly was done to assemble pSB1C3-fusion
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL) pSB1C3-fusion-5</TH><TH class="tableheaderadpbl">Volume (uL) pSB1C3-fusion-10</TH></TR><TR class="tableunevenadpbl"><TD>Gibson Assembly Mastermix 2X</TD><TD>10</TD><TD>10</TD></TR><TR class="tableevenadpbl"><TD>Vector (pSB1C3)</TD><TD>0.12 (200ng)</TD><TD>0.6 (100ng)</TD></TR><TR class="tableunevenadpbl"><TD>Fragment</TD><TD>8.6 (3000 ng)</TD><TD>8.6 (3000 ng)</TD></TR><TR class="tableunevenadpbl"><TD>MilliQ</TD><TD>1.3</TD><TD>0.8</TD></TR></TBODY></TABLE></CENTER><P>The samples were incubated at 55°C for 60 minutes.</P><H4> Transformation of chemically competent cells </H4><P>Transformations was carried out according to the protocol. The 2 and 18 uL of product of Gibson Assembly was transformed into <I>E. coli</I>DH5α   
</P><P>800 uL LB medium was added as recovery medium.
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#SundaySCO1">Sunday, 07/10/2018</BUTTON><DIV id="SundaySCO1" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>07/10/2018</DD><DT>Operator</DT><DD>Susan Bouwmeester</DD></DL><H4>Colony PCR</H4><P>Colony PCR was done to verify propagation of pSB1C3-linker in 10 colonies after transformation. This plasmid contains the linker (~60bp) and pSB1C3, which can be checked with primers VR and VF2.
</P><P>15 tubes were prepared for colony PCR of pSB1C3-linker colonies. The thermocycler programme was installed to have an extension time of 1:00 and an annealing temperature of 60 °C. After the programme was finished, samples were kept at 12°C.</P><H4>DNA Gel Electrophoresis</H4><P>PCR product of the colony PCR of linker was run on a 0.8% agarose gel in TAE buffer. A total of 10 colony PCR results were put on a gel. 5μL was pipetted into each lane. The gel was imaged using a gel documentation system with UV-light.</P><B>Figure</B> Colony PCR fusion. 0.8% agarose. Lane 1: 500bp ladder; Lane 2: colony 1; Lane 3: Colony 2; Lane 4: Colony 3; Lane 5: Colony 4; Lane 6: Colony 5; Lane 7: Colony 6; Lane 8: Colony 7; Lane 9: Colony 8; Lane 10; Colony 9; Lane 11; Colony 10 </DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV></DIV></UL></FORM><FORM><LABEL for="WeekSCO2" class="subdropdown cadpbl">Week 2 08/10/2018 - 14/10/2018</LABEL><UL class="submenu"><DIV class="subcontent"><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#MondaySCO2">Monday, 08/10/2018</BUTTON><DIV id="MondaySCO2" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>08/10/2018</DD><DT>Operator</DT><DD>Susan Bouwmeester</DD></DL><H4>Colony PCR</H4><P>Colony PCR was done to verify propagation of pSB1C3-linker in 8 colonies after transformation. This plasmid contains the linker (~60bp) and pSB1C3, which can be checked with primers VR and VF2.
</P><P>15 tubes were prepared for colony PCR of pSB1C3-linker colonies. The thermocycler programme was installed to have an extension time of 1:00 and an annealing temperature of 60 °C. After the programme was finished, samples were kept at 12°C.</P><H4>DNA Gel Electrophoresis</H4><P>PCR product of the colony PCR of linker was run on a 2% agarose gel in TAE buffer. A total of 7 colony PCR results were put on a gel. 5μL was pipetted into each lane. The gel was imaged using a gel documentation system with UV-light.</P><B>Figure</B> Colony PCR fusion. 0.8% agarose. Lane 1: 500bp ladder; Lane 2: colony 1; Lane 3: Colony 2; Lane 4: Colony 3; Lane 5: Colony 4; Lane 6: Colony 5; Lane 7: Colony 6; Lane 8: Colony 7; <P>Overnight culture of colony 1 is started.</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#TuesdaySCO2">Tuesday, 09/10/2018</BUTTON><DIV id="TuesdaySCO2" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>09/10/2018</DD><DT>Operator</DT><DD>Susan Bouwmeester</DD></DL><H4>Plasmid Isolation</H4><P> 
    Overnight starter culture of pSB1C3-linker was subjected to plasmid isolation following the Plasmid Isolation Protocol. Final elution was done with 30 μL of pre-warmed MilliQ.
</P><H4>Nanodrop DNA Quantification</H4><P> 
    After PCR cleanup, te sample was measered with nanodrop quantification. The following isolates were quantified according to the Nanodrop DNA Quantification protocol. The concentration and purity (280/260 and 230/260 ratio) was measured. 
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Isolate</TH><TH class="tableheaderadpbl">Concentration (ng/uL)</TH><TH class="tableheaderadpbl">280/260 ratio </TH><TH class="tableheaderadpbl">230/260 ratio</TH></TR><TR class="tableunevenadpbl"><TD>pSB1C3-linker</TD><TD>223.6</TD><TD>1.65</TD><TD>0.86</TD></TR></TBODY></TABLE></CENTER><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#WednesdaySCO1">Wednesday, 03/10/2018</BUTTON><DIV id="WednesdaySCO1" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#ThursdaySCO1">Thursday, 04/10/2018</BUTTON><DIV id="ThursdaySCO1" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#FridaySCO1">Friday, 05/10/2018</BUTTON><DIV id="FridaySCO1" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#SaturdayCO1">Saturday, 06/10/2018</BUTTON><DIV id="SaturdaySCO1" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#SaturdayCO1">Saturday, 06/10/2018</BUTTON><DIV id="SaturdaySCO1" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV></DIV></UL></FORM></DIV></DIV></DIV><DIV class="container nowbg" id="InVitro"><DIV class="row nowbg"><DIV class="col-xs-12 col-sm-3 col-lg-3"><NAV id="myScrollspy" class="hidden-xs hidden-sm"><UL class="nav nav-pills nav-stacked" data-spy="affix"><LI class="dropdown"><A class="dropdown-toggle adpbl" data-toggle="dropdown" href="#">July</A></LI><LI><A href="#JulyInVitro1" class="adpbl">Week: 02/07/2018 - 08/07/2018</A></LI><LI><A href="#JulyInVitro2" class="adpbl">Week: 09/07/2018 - 15/07/2018</A></LI><LI><A href="#JulyInVitro3" class="adpbl">Week: 16/07/2018 - 22/07/2018</A></LI></UL><LI class="dropdown"><A class="dropdown-toggle adpbl" data-toggle="dropdown" href="#">August</A></LI><LI><A href="#AugustInVitro1" class="adpbl">Week: 30/07/2018 - 05/08/2018</A></LI><LI><A href="#AugustInVitro2" class="adpbl">Week: 06/08/2018 - 12/08/2018</A></LI><LI><A href="#AugustInVitro3" class="adpbl">Week: 13/08/2018 - 19/08/2018</A></LI><LI><A href="#AugustInVitro4" class="adpbl">Week: 20/08/2018 - 26/08/2018</A></LI><LI><A href="#AugustInVitro5" class="adpbl">Week: 27/08/2018 - 02/09/2018</A></LI><LI class="dropdown"><A class="dropdown-toggle adpbl" data-toggle="dropdown" href="#">September</A></LI><LI><A href="#SeptemberInVitro1" class="adpbl">Week: 03/09/2018 - 09/09/2018</A></LI><LI><A href="#SeptemberInVitro2" class="adpbl">Week: 10/09/2018 - 16/09/2018</A></LI><LI><A href="#SeptemberInVitro3" class="adpbl">Week: 17/09/2018 - 23/09/2018</A></LI><LI><A href="#SeptemberInVitro4" class="adpbl">Week: 24/09/2018 - 30/09/2018</A></LI><LI class="drowdown"><A class="dropdown-toggle adpbl" data-toggle="dropdown" href="#">October</A></LI><LI><A href="#OctoberInVitro1" class="adpbl">Week: 01/10/2018 - 07/10/2018</A></LI><LI><A href="#OctoberInVitro2" class="adpbl">Week: 08/10/2018 - 14/10/2018</A></LI></NAV></DIV><DIV class="col-lg-8 col-md-8 col-sm-12 col-xs-12 nowbg" style="margin-right:20px;"><H3 class="adpbl">July</H3><FORM><LABEL for="WeekVTJ2" class="subdropdown cadpbl">Week 2 09/07/2018 - 15/07/2018</LABEL><UL class="submenu"><DIV class="subcontent"><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#TuesdayVTJ2">Tuesday, 10/07/2018</BUTTON><DIV id="TuesdayVTJ2" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>10/07/2018</DD><DT>Operator</DT><DD>Venda Mangkusaputra</DD></DL><H4>GBlock resuspension</H4><P>GBlock EPO was resuspended in 50 µL sterile TE buffer (10mM Tris, 1mM EDTA, pH8) to a final concentration of 10 ng/µL. GBlock sgRNA for EPO was resuspended in 25 µL sterile TE buffer (10mM Tris, 1mM EDTA, pH8) to a final concentration of 10 ng/µL.</P><DL><DT>Date</DT><DD>10/07/2018</DD><DT>Operator</DT><DD>Venda Mangkusaputra</DD></DL><H4>Primer working stock preparation</H4><P> The following primers were resuspended in sterile Milli-Q water, in order to make 100µM stock solutions.  
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Primer name</TH><TH class="tableheaderadpbl">Amount (nmol)</TH><TH class="tableheaderadpbl">Volume (uL)</TH></TR><TR class="tableunevenadpbl"><TD>033</TD><TD>25.8</TD><TD>258</TD></TR><TR class="tableevenadpbl"><TD>034</TD><TD>25.1</TD><TD>251</TD></TR><TR class="tableunevenadpbl"><TD> 038</TD><TD>32.6</TD><TD>326</TD></TR><TR class="tableunevenadpbl"><TD>039</TD><TD>32.0</TD><TD>320</TD></TR></TBODY></TABLE></CENTER><P> The primers in these 100 µM stocks were heated to 60 °C for 20 minutes and centrifuged  at maximum speed (~17,000 x g) for 2 minutes. 10µM primer working stock solutions were prepared by making 10x dilution aliquots with sterile Milli-Q, each in a total volume of 100µl. </P><DL><DT>Date</DT><DD>10/07/2018</DD><DT>Operator</DT><DD>Venda Mangkusaputra</DD></DL><H4>High Fidelity PCR</H4><P> To amplify EPO gBlocks (632bp), high fidelity PCR was done using EPO gBlocks as template DNA and 038 and 039 as forward and reverse primers respectively. To amplify sgRNA for EPO gBlocks (125bp), high fidelity PCR was done using sgRNA for EPO gBlocks as template DNA and 033 and 034 as forward and reverse primers respectively. 2 50µL PCR mixtures were prepared for each amplification of EPO and sgRNA for EPO gBlocks, according to the High Fidelity PCR protocol. 1 50µl of PCR mixtures with no template and primer 033 and 034 were prepared in parallel for negative control.
    
    
Primer combination(s) and expected fragment(s)

</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Sample</TH><TH class="tableheaderadpbl">fw primer</TH><TH class="tableheaderadpbl">rv primer</TH><TH class="tableheaderadpbl">expected fragment</TH><TH class="tableheaderadpbl">expected size (bp) </TH></TR><TR class="tableunevenadpbl"><TD>1</TD><TD>033</TD><TD>034</TD><TD>sgRNA for EPO gBlocks</TD><TD>125</TD></TR><TR class="tableevenadpbl"><TD>2</TD><TD>033</TD><TD>034</TD><TD>sgRNA for EPO gBlocks</TD><TD>125</TD></TR><TR class="tableunevenadpbl"><TD>3 </TD><TD>038</TD><TD>039</TD><TD>EPO gBlocks</TD><TD>632</TD></TR><TR class="tableunevenadpbl"><TD>4</TD><TD>038</TD><TD>039</TD><TD>EPO gBlocks</TD><TD>632</TD></TR><TR class="tableunevenadpbl"><TD>5 </TD><TD>033</TD><TD>034</TD><TD>negative control</TD><TD>none</TD></TR></TBODY></TABLE></CENTER><P>The thermocycler programme was installed to have an extension time of 16 seconds and an annealing temperature of 60°C. After the programme was finished, samples were run on 2% TAE agarose gel. Figure 1 displays the result of the gel imaged with GelDoc system.</P><B>Figure 1. 2% TAE Agarose gel of PCR amplification of gBlocks </B><I>. Lanes are (1) DNA Ladder, (2) and (3) amplified sgRNA template for EPO1 gBlocks, (4) and (5) amplified EPO gBlocks.
.  </I></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#ThursdayVTJ2">Thursday, 12/07/2018</BUTTON><DIV id="ThursdayVTJ2" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>12/07/2018</DD><DT>Operator</DT><DD>Venda Mangkusaputra</DD></DL><H4>sgRNA Transcription</H4><P>To produce sgRNAs that target the EPO gene, sgRNA transcription was done with sgRNA for EPO as a template. the following components were assembled at room temperature in the following order:
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL)</TH></TR><TR class="tableunevenadpbl"><TD>MilliQ</TD><TD>17.35 (fill to 30 µl) </TD></TR><TR class="tableevenadpbl"><TD>10x Reaction Buffer</TD><TD>2 uL</TD></TR><TR class="tableunevenadpbl"><TD>NTP (25mM each) </TD><TD>6 uL</TD></TR><TR class="tableunevenadpbl"><TD>sgRNA for EPO template</TD><TD>1.5 uL (0.2 - 1 µg) </TD></TR><TR class="tableunevenadpbl"><TD>RNase inhibitor</TD><TD>2 ul (1U/mL final) </TD></TR><TR class="tableunevenadpbl"><TD>T7 RNA polymerase </TD><TD>1 uL</TD></TR><TR class="tableunevenadpbl"><TD>MgCl<SUB>2</SUB></TD><TD>0.15 uL</TD></TR><TR class="tableunevenadpbl"><TD>Final Volume</TD><TD>30 uL </TD></TR></TBODY></TABLE></CENTER><P>The reaction was incubated at 37 °C overnight.</P><DL><DT>Date</DT><DD>12/07/2018</DD><DT>Operator</DT><DD>Venda Mangkusaputra</DD></DL><H4>Tn5 protein functionality testing: integration assay
</H4><P>Integration assay was performed to test the integration of adapter DNA by Tn5.</P><P>The sequence of the adapter used in this assay is shown on table 1.</P><CENTER><TABLE><CAPTION class="figadpbl"><B>Table 1:</B> Primer sequence to be annealed for the adapter. </CAPTION><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Name</TH><TH class="tableheaderadpbl">Sequence</TH></TR><TR class="tableunevenadpbl"><TH>Upper strand</TH><TD>CTAGCTGTCTCTTATACACATCTATTGGCGAAATGGATGAACTGTTTACCATGC</TD></TR><TR class="tableunevenadpbl"><TH>Lower strand</TH><TD>GCATGGTAAACAGTTCATCCATTTCGCCAATAGATGTGTATAAGAGACAGCTAG</TD></TR></TBODY></TABLE></CENTER><P>To anneal the two oligos, equamolar amount of each strand is mixed, incubated at 95°C and cooled slowly to room temperature. </P><P><B>Tn5:adapter DNA complex formation</B> 
Since this is the first time this experiment was done, the idea is to get the idea of the required amount of adapter needed in the reaction. Therefore 3 different adapter concentrations were added, namely 5µM, 25µM and 50µM. Tn5  was loaded with adapter DNA shown above by following the pipetting scheme shown on Table 2. All samples and reagents were kept on ice during preparation.
</P><CENTER><TABLE><CAPTION class="figadpbl"><B>Table 2:</B> Pipetting scheme for transposome formation. </CAPTION><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">5µM reaction (µl)</TH><TH class="tableheaderadpbl">25µM reaction (µl)</TH><TH class="tableheaderadpbl">50µM reaction (µl)</TH></TR><TR class="tableunevenadpbl"><TH>Buffer (250mM Tris, pH 7.5)</TH><TD>2.5</TD><TD>2.5</TD><TD>2.5</TD></TR><TR class="tableevenadpbl"><TH>Tn5 (0.5m/ml)</TH><TD>9</TD><TD>9</TD><TD>9</TD></TR><TR class="tableunevenadpbl"><TH>Adapter</TH><TD>2.5</TD><TD>2.5</TD><TD>2.5</TD></TR><TR class="tableevenadpbl"><TH>Glycerol</TH><TD>11</TD><TD>11</TD><TD>11</TD></TR><TR class="tableunevenadpbl"><TH>Total</TH><TD>25</TD><TD>25</TD><TD>25</TD></TR></TBODY></TABLE></CENTER><P>The samples were incubated at 37°C for 45 minutes.</P><P>Next, EPO (130ng/µl) was added in two different transposome:EPO volume ratios, namely 1:1 and 3:1 in total volume of 10µM. Finally 12.5mM final MgCl<SUB>2</SUB> concentration were added. The samples were incubated at 55°C for 10 minutes and stopped by adding final 1% SDS. The integration products were analysed on 2% TAE agarose gel. The gel was stained with sybersafe and imaged with GelDoc system. The resulting gel is shown on figure 1.
</P><B>Figure 1. 2% TAE Agarose gel of Tn5 integration assay. </B><I>. Lanes are (1) DNA ladder (100bp-1000bp), (2) 5µM adapter. 1:3 EPO:transposome, (3) 25µM adapter. 1:3 EPO:transposome, (4) 50µM adapter. 1:3 EPO:transposome, (6) 5µM adapter. 1:1 EPO:transposome, (7) 25µM adapter. 1:1 EPO:transposome, (8) 50µM adapter. 1:1 EPO:transposome.
 </I><P>Figure 1 suggested that the staining of the gel was poorly done as bands that was supposed to be present on the first and last lane were not visible. The smear on the rest of the lanes suggested the possible integration of adapter, creating bands of smaller sizes. However, more negative controls should be done to confirm this further. </P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#FridayVTJ2">Friday, 13/07/2018</BUTTON><DIV id="FridayVTJ2" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>13/07/2018</DD><DT>Operator</DT><DD>Venda Mangkusaputra</DD></DL><H4>RNA cleanup </H4><P> After the <I>in vitro</I> transcription, transcripts were isolated following the RNA Cleanup protocol. The isolated RNA were measured with nanodrop. The measure concentration of sgRNA for EPO is 11.3ng/µl.

</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV></DIV></UL></FORM><H3 class="adpbl">August</H3><FORM><LABEL for="WeekVTA2" class="subdropdown cadpbl">Week 2 06/08/2018 - 12/08/2018</LABEL><UL class="submenu"><DIV class="subcontent"><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#FridayVTA2">Friday, 10/07/2018</BUTTON><DIV id="FridayVTA2" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>10/08/2018</DD><DT>Operator</DT><DD>Venda Mangkusaputra</DD></DL><H4>Tn5 protein functionality testing 2: integration assay
</H4><P>Integration assay was performed to test the integration of adapter DNA by Tn5.</P><P>The sequence of the adapter used in this assay is shown on table 1.</P><CENTER><TABLE><CAPTION class="figadpbl"><B>Table 1:</B> Primer sequence to be annealed for the adapter. </CAPTION><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Name</TH><TH class="tableheaderadpbl">Sequence</TH></TR><TR class="tableunevenadpbl"><TH>Upper strand</TH><TD>CTAGCTGTCTCTTATACACATCTATTGGCGAAATGGATGAACTGTTTACCATGC</TD></TR><TR class="tableunevenadpbl"><TH>Lower strand</TH><TD>GCATGGTAAACAGTTCATCCATTTCGCCAATAGATGTGTATAAGAGACAGCTAG</TD></TR></TBODY></TABLE></CENTER><P><B>Tn5:adapter DNA complex formation</B> 
    From previous result, we learned that:</P><UL class="uladpbl"><LI>More negative control should be done to confirm integration</LI><LI>50µM adapter concentration was too high. In this experiment 10µM and 25µM adapter will be used.</LI><LI>50% glycerol might be too high. As the transposase already contain glycerol. No addition glycerol will be added in this experiment</LI></UL><P>
    Two different amount of Tn5  was loaded with adapter DNA by following the pipetting scheme shown on Table 2. All samples and reagents were kept on ice during preparation.
</P><CENTER><TABLE><CAPTION class="figadpbl"><B>Table 2:</B> Pipetting scheme for transposome formation. </CAPTION><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">No Tn5 (µl)</TH><TH class="tableheaderadpbl">Low Tn5  (µl)</TH><TH class="tableheaderadpbl">High Tn5 (µl)</TH></TR><TR class="tableunevenadpbl"><TH>Buffer (250mM Tris, pH 7.5)</TH><TD>2</TD><TD>2</TD><TD>2</TD></TR><TR class="tableevenadpbl"><TH>Tn5 (0.5mg/ml)</TH><TD>0</TD><TD>10</TD><TD>15</TD></TR><TR class="tableunevenadpbl"><TH>Adapter (10µM/25µM)</TH><TD>2.5</TD><TD>2.5</TD><TD>2.5</TD></TR><TR class="tableevenadpbl"><TH>Water</TH><TD>15.5</TD><TD>5.5</TD><TD>0.5</TD></TR><TR class="tableunevenadpbl"><TH>Total</TH><TD>25</TD><TD>25</TD><TD>25</TD></TR></TBODY></TABLE></CENTER><P>The samples were incubated at 37°C for 45 minutes.</P><P>Next, 1µl or 2µl of EPO (130ng/µl) was added to 5µl of each transposome reaction. 1µl of 125mM MgCl<SUB>2</SUB> was added. Water was added to make final volume of 10µM. The samples were incubated at 55°C for 10 minutes and stopped by adding final 1% SDS. The integration products were analysed on 2% TAE agarose gel. The gel was stained with sybersafe and imaged with GelDoc system. The resulting gel is shown on figure 1.
</P><B>Figure 1. TAE Agarose gel of PCR amplification of gBlocks. </B><I>. Lanes are (1) low adapter, low Tn5, (2) high adapter, low Tn5, (3) low EPO, low adapter, low Tn5, (4) low EPO, high adapter, low Tn5,  (5) low EPO, low adapter, high Tn5, (6) low EPO, high adapter, high Tn5, (7) DNA ladder, (8) low adapter, high Tn5, (9) high adapter, high Tn5, (10) high EPO, low adapter, low Tn5, (11) lohighw EPO, high adapter, low Tn5,  (12) high EPO, low adapter, high Tn5, (13) high EPO, high adapter, high Tn5, (14) high Tn5, (15) high EPO, high Tn5, (16) empty, (17) high Tn5, (18) high adapter, (19) empty, (20) DNA ladder.
  </I><P>Comparing our result with the negative control. It seemed like no (or very little) integration took place. </P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#SundayVTA2">Sunday, 11/08/2018</BUTTON><DIV id="SundayVTA2" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>12/08/2018</DD><DT>Operator</DT><DD>Venda Mangkusaputra</DD></DL><H4>gRNA Transcription</H4><P>For producing gRNAs that target EPO gene, gRNA transcription was done with sgRNA for EPO as a template. the following components were assembled at room temperature in the following order:
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL)</TH></TR><TR class="tableunevenadpbl"><TD>MilliQ</TD><TD>4.5ul </TD></TR><TR class="tableevenadpbl"><TD>10x Reaction Buffer</TD><TD>2 uL</TD></TR><TR class="tableunevenadpbl"><TD>NTP (25mM each) </TD><TD>4 uL</TD></TR><TR class="tableunevenadpbl"><TD>sgRNA for EPO</TD><TD> 5ul (0.3 µg) </TD></TR><TR class="tableunevenadpbl"><TD>RNase inhibitor</TD><TD>0.5ul (1U/mL final) </TD></TR><TR class="tableevenadpbl"><TD>DTT</TD><TD>2ul (5mM final) </TD></TR><TR class="tableunevenadpbl"><TD>T7 RNA polymerase </TD><TD>2 uL</TD></TR><TR class="tableunevenadpbl"><TD>Final Volume</TD><TD>20 uL </TD></TR></TBODY></TABLE></CENTER><P>The reaction was incubated at 37 °C overnight.</P><DL><DT>Date</DT><DD>12/08/2018</DD><DT>Operator</DT><DD>Venda Mangkusaputra</DD></DL><H4>Primer working stock preparation</H4><P> The following primers were resuspended in sterile Milli-Q water, in order to make 100µM stock solutions.  
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Primer name</TH><TH class="tableheaderadpbl">Amount (nmol)</TH><TH class="tableheaderadpbl">Volume (uL)</TH></TR><TR class="tableunevenadpbl"><TD>035</TD><TD>29.8</TD><TD>298</TD></TR></TBODY></TABLE></CENTER><P> The primers in these 100 µM stocks were heated to 60 °C for 20 minutes and centrifuged  at maximum speed (~17,000 x g) for 2 minutes. 10µM primer working stock solutions were prepared by making 10x dilution aliquots with sterile Milli-Q, each in a total volume of 100µl. </P><DL><DT>Date</DT><DD>12/08/2018</DD><DT>Operator</DT><DD>Venda Mangkusaputra</DD></DL><H4>High Fidelity PCR</H4><P> This PCR reaction was done to amplify sgRNA template for EPO. There are two different sgRNA that will target slightly different position of EPO cDNA. The two templates will be addressed as sgRNA for EPO1 and sgRNA for EPO2. To amplify  sgRNA for EPO1 and EPO2 (125bp), high fidelity PCR was done using sgRNA for EPO gBlocks as template DNA and either 033 and 034 or 035 and 034 as forward and reverse primers respectively. 2 50µL PCR mixtures were prepared for each amplification of sgRNA for EPO, according to the High Fidelity PCR protocol. 1 50µl of PCR mixtures with no template and primer 035 and 034 were prepared in parallel for negative control.
    
    
Primer combination(s) and expected fragment(s)

</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Sample</TH><TH class="tableheaderadpbl">fw primer</TH><TH class="tableheaderadpbl">rv primer</TH><TH class="tableheaderadpbl">expected fragment</TH><TH class="tableheaderadpbl">expected size (bp) </TH></TR><TR class="tableunevenadpbl"><TD>1</TD><TD>033</TD><TD>034</TD><TD>sgRNA for EPO1</TD><TD>125</TD></TR><TR class="tableevenadpbl"><TD>2</TD><TD>033</TD><TD>034</TD><TD>sgRNA for EPO1</TD><TD>125</TD></TR><TR class="tableunevenadpbl"><TD>3 </TD><TD>035</TD><TD>034</TD><TD>sgRNA for EPO2</TD><TD>632</TD></TR><TR class="tableunevenadpbl"><TD>4</TD><TD>035</TD><TD>034</TD><TD>sgRNA for EPO2</TD><TD>632</TD></TR><TR class="tableunevenadpbl"><TD>5 </TD><TD>035</TD><TD>034</TD><TD>negative control</TD><TD>none</TD></TR></TBODY></TABLE></CENTER><P>The thermocycler programme was installed to have an extension time of 16 seconds and an annealing temperature of 60°C. After the programme was finished, samples were run on 2% TAE agarose gel.Figure 1 displays the result of the gel imaged with GelDoc system.</P><B>Figure 1. 2% TAE Agarose gel of PCR products.</B><I>. Lanes are (1) DNA ladder, (2) and (3) amplified sgRNA template for EPO 1, (4) and (5) amplified sgRNA template for EPO2.
 </I></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV></DIV></UL></FORM><FORM><LABEL for="WeekVTA3" class="subdropdown cadpbl">Week 3 13/08/2018 - 19/08/2018</LABEL><UL class="submenu"><DIV class="subcontent"><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#MondayVTA3">Monday, 13/08/2018</BUTTON><DIV id="MondayVTA3" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>13/08/2018</DD><DT>Operator</DT><DD>Venda Mangkusaputra</DD></DL><H4>RNA cleanup </H4><P> After the <I>in vitro</I> transcription on the previous day, transcripts were isolated following the RNA Cleanup protocol.
    Following the clean up, the concentration of RNA is measured with nanodrop. The concentration achieved after clean up was 181.7 ng/µl. To visualize it, the samples before and after clean up, along with the template DNA were run on 2% TBE agarose gel.</P><P>
RNA loading buffer was added to samples following the RNA gel electrophoresis protocol. The agarose gel was run on 80 Voltage. The image of the gel is shown in Figure 1.
</P><B>Figure 1. 2% TBE Agarose gel of <I>in vitro</I> transcription products.  </B><I>. Lanes are (1) RNA ladder, (2) sgRNA for EPO 1 before RNA clean up, (3) sgRNA for EPO 1 after RNA clean up, (4) DNA ladder (5) sgRNA DNA template
 </I></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#WednesdayVTA3">Wednesday, 15/08/2018</BUTTON><DIV id="WednesdayVTA3" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>15/08/2018</DD><DT>Operator</DT><DD>Venda Mangkusaputra</DD></DL><H4>Tn5 protein functionality testing 3: integration assay
</H4><P>Integration assay was performed to test the integration of adapter DNA by Tn5.</P><P>The sequence of the adapter used in this assay is shown on table 1.</P><CENTER><TABLE><CAPTION class="figadpbl"><B>Table 1:</B> Primer sequence to be annealed for the adapter. </CAPTION><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Name</TH><TH class="tableheaderadpbl">Sequence</TH></TR><TR class="tableunevenadpbl"><TH>Short adapter upper strand</TH><TD>CTAGCTGTCTCTTATACACATCTATTGGCGAAATGGATGAACTGTTTACCATGC</TD></TR><TR class="tableunevenadpbl"><TH>Short adapter lower strand</TH><TD>GCATGGTAAACAGTTCATCCATTTCGCCAATAGATGTGTATAAGAGACAGCTAG</TD></TR><TR class="tableunevenadpbl"><TH>Long adapter upper strand</TH><TD>BBa_K2643011</TD></TR><TR class="tableunevenadpbl"><TH>Long adapter lower strand</TH><TD>BBa_K2643011</TD></TR></TBODY></TABLE></CENTER><P>From previous result, we learned that:</P><UL class="uladpbl"><LI>High amount of glycerol might be needed for integration since previous experiment without glycerol did not result in any integration.  </LI><LI>Since finally Tn5 has to function together with dxCas9, we will try to do the reaction in lower temperature, since dxCas9 might be denatured at 55°C.Therefore, in this experiment, the adapter loading and integration experiment will be done in one go, at 37°C for 2 hours.</LI><LI>New adapter will be used, which has ME sequence on both sides to test whether this type of integration (integration without splitting the fragment into two) would work better.</LI></UL><P>
    The reaction was done following the pipetting scheme shown on Table 2. All samples and reagents were kept on ice during preparation.
</P><CENTER><TABLE><CAPTION class="figadpbl"><B>Table 2:</B> Pipetting scheme for integration reaction. </CAPTION><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Reaction with short adapter (µl)</TH><TH class="tableheaderadpbl">Reaction with long adapter  (µl)</TH><TH class="tableheaderadpbl">No EPO, short adapter (µl)</TH><TH class="tableheaderadpbl">No EPO, long adapter(µl)</TH><TH class="tableheaderadpbl">No Tn5, short adapter (µl)</TH><TH class="tableheaderadpbl">No Tn5, long adapter (µl)</TH><TH class="tableheaderadpbl">No adapter (µl)</TH></TR><TR class="tableunevenadpbl"><TH>Buffer (250mM Tris, pH 7.5)</TH><TD>1</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD>1</TD></TR><TR class="tableevenadpbl"><TH>Tn5 (0.5m/ml)</TH><TD>3</TD><TD>3</TD><TD>3</TD><TD>3</TD><TD>0</TD><TD>0</TD><TD>3</TD></TR><TR class="tableunevenadpbl"><TH>Adapter</TH><TD>0.5</TD><TD>1.5</TD><TD>0.5</TD><TD>1.5</TD><TD>0.5</TD><TD>1.5</TD><TD>0</TD></TR><TR class="tableevenadpbl"><TH>Glycerol</TH><TD>2.5</TD><TD>2.5</TD><TD>2.5</TD><TD>2.5</TD><TD>2.5</TD><TD>2.5</TD><TD>2.5</TD></TR><TR class="tableunevenadpbl"><TH>MgCl<SUB>2</SUB> (125mM)</TH><TD>1</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD>1</TD></TR><TR class="tableevenadpbl"><TH>EPO (170ng/ul) &gt;</TH><TD>1</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD>1</TD></TR><TR class="tableunevenadpbl"><TH>Water</TH><TD>1</TD><TD>0</TD><TD>2</TD><TD>1</TD><TD>4</TD><TD>3</TD><TD>1.5</TD></TR><TR class="tableevenadpbl"><TH>Total</TH><TD>10</TD><TD>10</TD><TD>10</TD><TD>10</TD><TD>10</TD><TD>10</TD><TD>10</TD></TR></TBODY></TABLE></CENTER><P>The samples were incubated at 37°C for 2 hours.</P><P>The reaction was stopped by adding final 1% SDS. The integration products were analysed on 2% TAE agarose gel. The gel was stained with sybersafe and imaged with GelDoc system. The resulting gel is shown on figure 1.
</P><B>Figure 1. 2% TAE Agarose gel of Tn5 integration assay. </B><I>. Lanes are (1) integration with short adapter, (2) integration with long adapter, (3) integration with short adapter, no EPO, (4) integration with long adapter, no EPO, (5) integration with short adapter, no Tn5, (6) integration with long adapter, no Tn5
  </I><P>The presence of upper bands on lane xx and xx suggested that integration occurs. However, since some negative controls are missing, it might be difficult to draw any conclusive result. </P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#ThursdayVTA3">Thursday, 16/08/2018</BUTTON><DIV id="ThursdayVTA3" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>16/08/2018</DD><DT>Operator</DT><DD>Venda Mangkusaputra</DD></DL><H4>High Fidelity PCR</H4><P> PCR amplification for the long adapter (KanR adapter), high fidelity PCR was done using KanR adapter as template DNA and XXX and XXX as forward and reverse primers respectively. 10 50µl PCR mixtures were prepared, according to the High Fidelity PCR protocol.
    
    
Primer combination(s) and expected fragment(s)

</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Sample</TH><TH class="tableheaderadpbl">fw primer</TH><TH class="tableheaderadpbl">rv primer</TH><TH class="tableheaderadpbl">expected fragment</TH><TH class="tableheaderadpbl">expected size (bp) </TH></TR><TR class="tableunevenadpbl"><TD>1-10</TD><TD>IV005</TD><TD>IV006</TD><TD>KanR adapter</TD><TD>800</TD></TR></TBODY></TABLE></CENTER><P>The thermocycler programme was installed to have an extension time of 30 seconds and an annealing temperature of 60 °C. After the programme was finished, samples were run on 2% TAE agarose gel. Figure 1 displays the result of the gel imaged with GelDoc systemr.</P><B>Figure 1. 0.8% TAE Agarose gel of long adapter amplification. </B><I>. Lanes are (1) DNA ladder, (2-11) amplification of the long adapter
 </I></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#FridayVTA3">Friday, 17/07/2018</BUTTON><DIV id="FridayVTA3" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>17/08/2018</DD><DT>Operator</DT><DD>Venda Mangkusaputra</DD></DL><H4>Tn5 protein functionality testing 4: integration assay
</H4><P>Integration assay was performed to test the integration of adapter DNA by Tn5.</P><P>The sequence of the adapter used in this assay is shown on table 1.</P><CENTER><TABLE><CAPTION class="figadpbl"><B>Table 1:</B> Primer sequence to be annealed for the adapter. </CAPTION><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Name</TH><TH class="tableheaderadpbl">Sequence</TH></TR><TR class="tableunevenadpbl"><TH>Long adapter upper strand</TH><TD>BBa_K2643011</TD></TR><TR class="tableunevenadpbl"><TH>Long adapter lower strand</TH><TD>BBa_K2643011</TD></TR></TBODY></TABLE></CENTER><P>From previous result, we learned that:</P><UL class="uladpbl"><LI>Integration could be seen with the longer adapter.</LI><LI>More negative control has to be done</LI></UL><P>In this reaction, we tried reproduce the previous result. Moreover, the effect of Tn5 concentration was tested. Two different Tn5 batches was used, one with 0.5mg/ml concentration, one with 1.5mg/ml concentration.</P><P>
    The reaction was done following the pipetting scheme shown on Table 2. All samples and reagents were kept on ice during preparation.
</P><CENTER><TABLE><CAPTION class="figadpbl"><B>Table 2:</B> Pipetting scheme for integration reaction. </CAPTION><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Reaction (µl)</TH><TH class="tableheaderadpbl">Reaction without adapter  (µl)</TH><TH class="tableheaderadpbl">Reaction without EPO (µl)</TH></TR><TR class="tableunevenadpbl"><TH>Buffer (250mM Tris, pH 7.5)</TH><TD>12</TD><TD>8</TD><TD>8</TD></TR><TR class="tableevenadpbl"><TH>Adapter</TH><TD>12</TD><TD>0</TD><TD>8</TD></TR><TR class="tableunevenadpbl"><TH>Glycerol</TH><TD>30</TD><TD>20</TD><TD>20</TD></TR><TR class="tableevenadpbl"><TH>MgCl<SUB>2</SUB>(125mM)</TH><TD>12</TD><TD>8</TD><TD>8</TD></TR><TR class="tableunevenadpbl"><TH>EPO (170ng/ul) &gt;</TH><TD>6</TD><TD>4</TD><TD>0</TD></TR><TR class="tableevenadpbl"><TH>Water</TH><TD>0</TD><TD>8</TD><TD>4</TD></TR><TR class="tableunevenadpbl"><TH>Total</TH><TD>10</TD><TD>10</TD><TD>10</TD></TR></TBODY></TABLE></CENTER><P>12µl of each master mix shown on Table two was added to 0 / 3 / 5 / 8 µl of low concentration Tn5, or 8µl of high concentration Tn5. Water was added to each reaction to make final volume of 20µl. The samples were incubated at 37°C for 2 hours.</P><P>The reaction was stopped by adding final 1% SDS. The integration products were analysed on 2% TAE agarose gel. The gel was stained with sybersafe and imaged with GelDoc system. The resulting gel is shown on figure 1.
</P><B>Figure 1. 2% TAE Agarose gel of Tn5 integration assay. </B><I>. Lanes are (1) reaction, no Tn5, (2) reaction 1µM Tn5, (3) reaction 1.5µM Tn5, (4) reaction 3µM Tn5, (5) reaction 6µM Tn5, (6) DNA ladder, (7) DNA ladder, (8) 50nM EPO, (9) 2.5µM adapter, (10) 3µM Tn5, (11) 6µM, (12) no adapter, no Tn5, (13) no adapter, 1µM Tn5, (14) no adapter, 3µM Tn5, (15) no adapter, 6µM Tn5, (16) no EPO, no Tn5, (17) no EPO, 1µM Tn5, (18) no EPO, 3µM Tn5, (19) no EPO, 6µM Tn5, 
  </I><P>The presence of smearing, even when no adapter was added suggested that the Tn5 might be preloaded already with DNA.  </P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV></DIV></UL></FORM><FORM><LABEL for="WeekVTA4" class="subdropdown cadpbl">Week 4 20/08/2018 - 26/08/2018</LABEL><UL class="submenu"><DIV class="subcontent"><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#ThursdayVTA4">Thursday, 23/08/2018</BUTTON><DIV id="ThursdayVTA4" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>23/8/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>Nanodrop DNA Quantification</H4><P> 
    The following isolates were quantified according to the Nanodrop DNA Quantification protocol. The concentration and purity of the DNA (260/280 and 260/230 ratios) were measured. 
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Isolate</TH><TH class="tableheaderadpbl">Concentration (ng/uL)</TH><TH class="tableheaderadpbl">260/280 ratio</TH><TH class="tableheaderadpbl">260/230 ratio</TH></TR><TR class="tableunevenadpbl"><TD>colony 1 (EPO)</TD><TD>20.4</TD><TD>1.62</TD><TD>0.95</TD></TR><TR class="tableevenadpbl"><TD>colony 2 (EPO)</TD><TD>22</TD><TD>1.60</TD><TD>0.93</TD></TR></TBODY></TABLE></CENTER><DL><DT>Date</DT><DD>23/8/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>Sequence Verification </H4><H5>Sequence from plasmid</H5><P> For verification of EPO in pSB1C3, primers VF2 and VR were used for sequencing. The sequencing starts from the biobrick backbone in the direction of the insert. The 10µL sequencing sample was prepared in a 1.5 mL tube as follows: 

</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL)</TH><TH class="tableheaderadpbl">Final concentration</TH></TR><TR class="tableunevenadpbl"><TD>Plasmid DNA (500ng)</TD><TD>7.5</TD><TD>&gt;50 ng/uL</TD></TR><TR class="tableunevenadpbl"><TD>10uM primer</TD><TD>2.5</TD><TD>25 pmol</TD></TR></TBODY></TABLE></CENTER><P>Within 7 days, the sequence was retrievable online and alignment of colony 1 with the in silico designed molecule was compared by Venda, and showed 100% match of the 1050 bp downstream of the sequencing primer binding site. </P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV></DIV></UL></FORM><FORM><LABEL for="WeekVTA5" class="subdropdown cadpbl">Week 5 27/08/2018 - 02/09/2018</LABEL><UL class="submenu"><DIV class="subcontent"><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#MondayVTA5">Monday, 27/08/2018</BUTTON><DIV id="MondayVTA5" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>27/08/2018</DD><DT>Operator</DT><DD>Venda Mangkusaputra</DD></DL><H4>Tn5 protein functionality testing 5: integration assay
</H4><P>Integration assay was performed to test the integration of adapter DNA by Tn5.</P><P>The sequence of the adapter used in this assay is shown on table 1.</P><CENTER><TABLE><CAPTION class="figadpbl"><B>Table 1:</B> Primer sequence to be annealed for the adapter. </CAPTION><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Name</TH><TH class="tableheaderadpbl">Sequence</TH></TR><TR class="tableunevenadpbl"><TH>Long adapter lower strand</TH><TD>BBa_K2643011</TD></TR><TR class="tableunevenadpbl"><TH>Short adapter lower strand</TH><TD>BBa_K2643011</TD></TR></TBODY></TABLE></CENTER><P>From previous result, we learned that:</P><UL class="uladpbl"><LI>Our Tn5 might be preloaded. Therefore we treated our Tn5 with heparin column chromatography to get rid of all the DNA on it.</LI><LI>We will treat the reaction with protease to make sure that the protein will release the DNA and allow DNA to enter the gel.</LI></UL><P>
    The reaction was done following the pipetting scheme shown on Table 2. All samples and reagents were kept on ice during preparation.
</P><CENTER><TABLE><CAPTION class="figadpbl"><B>Table 2:</B> Pipetting scheme for integration reaction. </CAPTION><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">No EPO, No adapter (µl)</TH><TH class="tableheaderadpbl">Complete reaction  (µl)</TH><TH class="tableheaderadpbl">No adapter (µl)</TH><TH class="tableheaderadpbl">No EPO (µl)</TH></TR><TR class="tableunevenadpbl"><TH>Buffer (250mM Tris, 125mM MgCl<SUB>2</SUB>, pH 7.5)</TH><TD>4</TD><TD>4</TD><TD>4</TD><TD>4</TD></TR><TR class="tableevenadpbl"><TH>Adapter</TH><TD>0</TD><TD>4</TD><TD>0</TD><TD>4</TD></TR><TR class="tableunevenadpbl"><TH>Glycerol</TH><TD>4</TD><TD>4</TD><TD>4</TD><TD>4</TD></TR><TR class="tableevenadpbl"><TH>EPO (170ng/ul) &gt;</TH><TD>0</TD><TD>4</TD><TD>4</TD><TD>0</TD></TR><TR class="tableunevenadpbl"><TH>Water</TH><TD>20</TD><TD>12</TD><TD>16</TD><TD>16</TD></TR><TR class="tableevenadpbl"><TH>Total</TH><TD>28</TD><TD>28</TD><TD>28</TD><TD>28</TD></TR></TBODY></TABLE></CENTER><P>14µl of each master mix shown on Table 1 was added to 6µl of Tn5 (1.8mg/ml) or water. The samples were incubated at 37°C for 2 hours. 1µl of proteinase K were added to one set of the reaction</P><P>The integration products were analysed on 2% TAE agarose gel. The gel was stained with sybersafe and imaged with GelDoc system. The resulting gel is shown on figure 1.
</P><B>Figure 1. 2% TAE Agarose gel of Tn5 integration assay. </B><I>. Lanes are (1) DNA ladder, (2) buffer, (3) Tn5 (2.5µM), (4) Tn5(2.5µM) + protease K, (5) reaction without Tn5, (6)  reaction with Tn5 (2.5µM), (7) reaction with Tn5(2.5µM) + protease K, (8) DNA ladder, (9) reaction without Tn5, without adapter (10)  reaction without adapter, with Tn5 (2.5µM), (11) reaction without adapter, with Tn5(2.5µM) + protease K, (12) reaction without Tn5, without EPO (13)  reaction without EPO, with Tn5 (2.5µM), (14) reaction without EPO, with Tn5(2.5µM) + protease K, (15) EPO only, (16) adapter only.
  </I><P>When Tn5 was added All bands are missing, which could be the result of two possible reasonings. First, our Tn5 protein might be contaminated with DNase, degrading all of our DNA. Second, our Tn5 might be really active that it degrade our DNA to very small pieces that already ran off the gel.</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#ThursdayVTA5">Thursday, 30/08/2018</BUTTON><DIV id="ThursdayVTA5" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>30/8/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>Cell banking</H4><P>Strain <I>E. coli</I> DH5alpha pSB1C3-EPO was stocked (colony 1) and stored at -80<SUP>o</SUP>C following the cell banking protocol.
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV></DIV></UL></FORM><H3 class="adpbl">SEPTEMBER</H3><FORM><LABEL for="WeekVTS1" class="subdropdown cadpbl">Week 1 03/09/2018 - 09/09/2018</LABEL><UL class="submenu"><DIV class="subcontent"><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#ThursdayVTS1">Thursday, 06/09/2018</BUTTON><DIV id="ThursdayVTS1" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>06/09/2018</DD><DT>Operator</DT><DD>Venda Mangkusaputra</DD></DL><H4>Tn5 functionality testing: Electrophoretic Mobility Shift Assay
</H4><P>Since the Tn5 integration assay has been showing inconsistent result. We took a step further and first focus on checking whether our Tn5 is able to load the adapter DNA. EMSA was performed to test this. 
</P><P>The components of the reagents can be found on the protocol. The sequence of the adapter used in this assay is shown on table 1.</P><CENTER><TABLE><CAPTION class="figadpbl"><B>Table 1:</B> Primer sequence to be annealed for the adapter. Note that 5’ of the upper strand of the adapter has to be phosphorylated for successful integration. </CAPTION><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Name</TH><TH class="tableheaderadpbl">Sequence</TH></TR><TR class="tableunevenadpbl"><TH>Adapter upper strand</TH><TD>P-CTGTCTCTTATACACATCT</TD></TR><TR class="tableunevenadpbl"><TH>Adapter lower strand</TH><TD>GCATGGTAAACAGTTCATCCATTTCGCCAATAGATGTGTATAAGAGACAG</TD></TR></TBODY></TABLE></CENTER><P><B>Tn5:adapter DNA complex formation</B> 
Tn5 was loaded with adapter DNA shown above by following the pipetting scheme shown on Table 2. All samples and reagents were kept on ice during preparation.
</P><CENTER><TABLE><CAPTION class="figadpbl"><B>Table 2:</B> Pipetting scheme for transposome formation. </CAPTION><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Reaction (µl)</TH><TH class="tableheaderadpbl">No adapter (µl)</TH></TR><TR class="tableunevenadpbl"><TH>Buffer</TH><TD>1</TD><TD>1</TD></TR><TR class="tableevenadpbl"><TH>Tn5 (~8µM)</TH><TD>0/1/2/3/5/8</TD><TD>8</TD></TR><TR class="tableunevenadpbl"><TH>Adapter (25µM)</TH><TD>1</TD><TD>0</TD></TR><TR class="tableevenadpbl"><TH>Water</TH><TD>8/7/6/5/3/0</TD><TD>0</TD></TR><TR class="tableunevenadpbl"><TH>Total</TH><TD>10</TD><TD>10</TD></TR></TBODY></TABLE></CENTER><P>The reagents were pipetted in the following order: Water, 10x functionality buffer, Tn5 or dxCas9-Tn5, and adapter. The samples were incubate at 23°C for 1 hour and analysed on 8% native TBE polyacrylamide gel. The resulting gel is either stained with ethidium bromide and imaged with GelDoc system, or directly imaged with Typhoon Imaging System. The resulting gels are shown on figure 1.</P><B>Figure 1. % TBE native PAGE on Tn5 EMSA. </B><I>.  Lanes are (1) DNA ladder, (2) 2.5µM adapter + 0µM Tn5 (3) 2.5µM adapter + 0µM Tn5, (4) 2.5µM adapter + 0.8µM Tn5, (5) 2.5µM adapter + 1.5µM Tn5, (6) 2.5µM adapter + 2.5µM Tn5, (7) 2.5µM adapter + 4µM Tn5 , (8) 2.5µM adapter + 6µM Tn5, (9) 2.5µM adapter + 6µM Tn5 + SDS, (10) 0µM adapter + 6µM Tn5, (11) 0µM adapter + 6µM Tn5, (12) 2.5µg protein control albumin, (13) 5µg protein control albumin, (14) protein ladder, (15) 2.5µM adapter
.  </I><P>In order to see the position of the protein in the gel, and therefore confirmed the mobility shift further, the gel were stained with silver quest staining. The resulting gel is shown on figure 2. </P><DIV><B>Figure 2. % TBE native PAGE on Tn5 EMSA with silver stain.  </B><I>. Lanes are (1) DNA ladder, (2) 2.5µM adapter + 0µM Tn5 (3) 2.5µM adapter + 0µM Tn5, (4) 2.5µM adapter + 0.8µM Tn5, (5) 2.5µM adapter + 1.5µM Tn5, (6) 2.5µM adapter + 2.5µM Tn5, (7) 2.5µM adapter + 4µM Tn5 , (8) 2.5µM adapter + 6µM Tn5, (9) 2.5µM adapter + 6µM Tn5 + SDS, (10) 0µM adapter + 6µM Tn5, (11) 0µM adapter + 6µM Tn5, (12) 2.5µg protein control albumin, (13) 5µg protein control albumin, (14) protein ladder, (15) 2.5µM adapter

  </I></DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV></DIV></UL></FORM><FORM><LABEL for="WeekVTS2" class="subdropdown cadpbl">Week 2 10/09/2018 - 16/09/2018</LABEL><UL class="submenu"><DIV class="subcontent"><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#ThursdayVTS2">Thursday, 13/09/2018</BUTTON><DIV id="ThursdayVTS2" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>13/09/2018</DD><DT>Operator</DT><DD>Venda Mangkusaputra</DD></DL><H4>Fusion protein functionality testing: Electrophoretic Mobility Shift Assay
</H4><P>From the previous test, it was shown that Tn5 by itself were able to load the adapter DNA. Since the Tn5 portion of the fusion protein might behave differently, we did the same experiment with the fusion protein.
</P><P>The components of the reagents can be found on the protocol. The sequence of the adapter used in this assay is shown on table 1.</P><CENTER><TABLE><CAPTION class="figadpbl"><B>Table 1:</B> Primer sequence to be annealed for the adapter. Note that 5’ of the upper strand of the adapter has to be phosphorylated for successful integration. </CAPTION><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Name</TH><TH class="tableheaderadpbl">Sequence</TH></TR><TR class="tableunevenadpbl"><TH>Adapter upper strand</TH><TD>P-CTGTCTCTTATACACATCT</TD></TR><TR class="tableunevenadpbl"><TH>Adapter lower strand</TH><TD>GCATGGTAAACAGTTCATCCATTTCGCCAATAGATGTGTATAAGAGACAG</TD></TR></TBODY></TABLE></CENTER><P><B>Fusion protein:adapter DNA complex formation</B> 
Fusion protein was loaded with adapter DNA shown above by following the pipetting scheme shown on Table 2. All samples and reagents were kept on ice during preparation.
</P><CENTER><TABLE><CAPTION class="figadpbl"><B>Table 2:</B> Pipetting scheme for transposome formation. </CAPTION><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Reaction (µl)</TH><TH class="tableheaderadpbl">No adapter (µl)</TH></TR><TR class="tableunevenadpbl"><TH>Buffer</TH><TD>1</TD><TD>1</TD></TR><TR class="tableevenadpbl"><TH>Fusion protein(~3µM)</TH><TD>0/0.5/2/4/8</TD><TD>8</TD></TR><TR class="tableunevenadpbl"><TH>Adapter (25µM)</TH><TD>1</TD><TD>0</TD></TR><TR class="tableevenadpbl"><TH>Water</TH><TD>8/7.5/6/2/0</TD><TD>0</TD></TR><TR class="tableunevenadpbl"><TH>Total</TH><TD>10</TD><TD>10</TD></TR></TBODY></TABLE></CENTER><P>The reagents were pipetted in the following order: Water, 10x functionality buffer, fusion protein and adapter. The samples were incubate at 23°C for 1 hour and analysed on 5% native TBE polyacrylamide gel. One of the 8µl fusion protein reaction was treated with heat (90<SUP>o</SUP>C) to denature the fusion protein. The resulting gel is either stained with ethidium bromide and imaged with GelDoc system, or directly imaged with Typhoon Imaging System. The resulting gels are shown on figure 1.</P><B>Figure 1. 5% TBE native PAGE on Tn5 EMSA stained with EtBr, imaged by a) GelDoc and b) Typhoon Imaging.  </B><I>. Lanes are (1) DNA ladder, (2)2.5µM fusion protein, (3) 2.5µM adapter + 0µM fusion protein (4) 2.5µM adapter + 0µM fusion protein, (5) 2.5µM adapter + 0.15µM fusion protein, (6) 2.5µM adapter + 0.5µM fusion protein, (7) 2.5µM adapter + 1.5µM fusion protein, (8) 2.5µM adapter + 2.5µM fusion protein, (9) 2.5µM adapter + 2.5µM fusion protein + heat, (10) 2.5µM adapter
.  </I><P>The observed higher bands on figure 1 confirmed the ability of the fusion protein to load the adapter DNA.</P><P>In order to see the position of the protein in the gel, and therefore confirmed the mobility shift further, the gel were stained with silver quest staining. The resulting gel is shown on figure 2. It can be seen the observed bands on Figure 2 is consistent with the ones shown on Figure 1</P><B>Figure 2. 5% TBE native PAGE on Tn5 EMSA with silver stain, imaged by a) GelDoc and b) Typhoon Imaging.  </B><I>. Lanes are (1) DNA ladder, (2)2.5µM fusion protein, (3) 2.5µM adapter + 0µM fusion protein (4) 2.5µM adapter + 0µM fusion protein, (5) 2.5µM adapter + 0.15µM fusion protein, (6) 2.5µM adapter + 0.5µM fusion protein, (7) 2.5µM adapter + 1.5µM fusion protein, (8) 2.5µM adapter + 2.5µM fusion protein, (9) 2.5µM adapter + 2.5µM fusion protein + heat, (10) 2.5µM adapter
  </I></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV></DIV></UL></FORM><FORM><LABEL for="WeekVTS3" class="subdropdown cadpbl">Week 3 17/09/2018 - 23/09/2018</LABEL><UL class="submenu"><DIV class="subcontent"><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#ThursdayVTS3">Thursday, 19/09/2018</BUTTON><DIV id="ThursdayVTS3" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>19/09/2018</DD><DT>Operator</DT><DD>Venda and Monique</DD></DL><H4>Fusion protein functionality testing: integration assay
</H4><P>EMSA on both the Tn5 and dxCas9 portion of the fusion showed that these portion individually were able to load adapter or sgRNA and EPO. Following this, integration assay was performed to test the integration of adapter DNA by fusion protein.</P><P>The components of the reagents can be found on the protocol. The sequence of the adapter used in this assay is shown on table 1.</P><CENTER><TABLE><CAPTION class="figadpbl"><B>Table 1:</B> Primer sequence to be annealed for the adapter. Note that 5’ of the upper strand of the adapter has to be phosphorylated for successful integration. </CAPTION><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Name</TH><TH class="tableheaderadpbl">Sequence</TH></TR><TR class="tableunevenadpbl"><TH>Adapter upper strand</TH><TD>P-CTGTCTCTTATACACATCT</TD></TR><TR class="tableunevenadpbl"><TH>Adapter lower strand</TH><TD>GCATGGTAAACAGTTCATCCATTTCGCCAATAGATGTGTATAAGAGACAG</TD></TR></TBODY></TABLE></CENTER><P><B>Fusion protein:adapter DNA complex formation</B> 
Tn5-dxCas9 protein was loaded with adapter DNA shown above by following the pipetting scheme shown on Table 2. All samples and reagents were kept on ice during preparation.
</P><CENTER><TABLE><CAPTION class="figadpbl"><B>Table 2:</B> Pipetting scheme for transposome formation. </CAPTION><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">1x Fusion (µl)</TH><TH class="tableheaderadpbl">1x Negative (no adapter) (µl)</TH></TR><TR class="tableunevenadpbl"><TH>Buffer</TH><TD>1</TD><TD>1</TD></TR><TR class="tableevenadpbl"><TH>Fusion (~3µM)</TH><TD>0/4/6/8</TD><TD>8</TD></TR><TR class="tableunevenadpbl"><TH>Adapter (25µM)</TH><TD>1</TD><TD>0</TD></TR><TR class="tableevenadpbl"><TH>Water</TH><TD>8/4/2/0</TD><TD>0</TD></TR><TR class="tableunevenadpbl"><TH>Total</TH><TD>10</TD><TD>10</TD></TR></TBODY></TABLE></CENTER><P>The reagents were pipetted in the following order: Water, 10x functionality buffer, Tn5 or dxCas9-Tn5, and adapter. The samples were incubated at 23°C for 1 hour.</P><P>To each of the transposome reaction, reaction with sgRNA was done in parallel with the negative control where no sgRNA were added. To add appropriate amount of sgRNA, the pipetting scheme on Table 3 was followed. sgRNA J2.1 signifies version 1 of sgRNA targeting juntion 2 of EPO gene:
</P><CENTER><TABLE><CAPTION class="figadpbl"><B>Table 3:</B> Pipetting scheme for sgRNA loading.</CAPTION><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">With sgRNA J2.1 (µl)</TH><TH class="tableheaderadpbl">Without sgRNA (µl)</TH></TR><TR class="tableunevenadpbl"><TH>Transposome</TH><TD>10</TD><TD>10</TD></TR><TR class="tableevenadpbl"><TH>sgRNA (100ng/µl)</TH><TD>2</TD><TD>0</TD></TR><TR class="tableunevenadpbl"><TH>MgCl<SUB>2</SUB> Buffer</TH><TD>0.5</TD><TD>0.5</TD></TR><TR class="tableevenadpbl"><TH>Water</TH><TD>0</TD><TD>2</TD></TR></TBODY></TABLE></CENTER><P>The samples were incubated at 37°C for 10 minutes.
Target DNA (EPO cDNA) was added to make final concentration of ~5ng/µl.</P><P>The samples were incubated at 37°C for 1 hour and analysed on 5% TBE native PAGE </P><DIV class="col-lg-12 col-md-12 col-sm-12 col-xs-12" nowbg=""><FIGCAPTURE class="figjnnbl"><B>Figure 1:</B>  TBE native PAGE stained with EtBr and imaged by GelDoc (left) or directly imaged with Typhoon Imaging (right). The gel was loaded as follow: Lane 1 molecular ladder (bp), lane 2 0um fusion w/out EPO, lane 3 2.1um fusion w/out EPO, lane 4 0uM fusion w/o gRNA (1:1) w/ EPO, lane 5 1uM fusion w/o gRNA (1:1) w/ EPO, lane 6 1.6uM fusion w/o gRNA (1:1) w/ EPO, lane 7 2.1uM fusion w/o gRNA (1:1) w/ EPO, lane 8 2.1uM fusion no adapter, lane 9 0uM fusion w gRNA (1:1) w/ EPO, lane 10 1uM fusion w/ gRNA (1:1) w/ EPO, lane 11 1.6uM fusion w/ gRNA (1:1) w/ EPO, lane 12 2.1uM fusion w/ gRNA (1:1) w/ EPO, lane 13 2.1uM fusion no adapter, lane 14 2.1uM fusion w/o gRNA (boiled), lane 15 2.1uM fusion w/ gRNA (boiled). 
      </FIGCAPTURE></DIV></DIV><P>Figure 1 shows that no integration were visible as there is not visible bands other than the ones also found on the negative controls. Further evaluation is required. </P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#SaturdayVTS3">Saturday, 22/09/2018</BUTTON><DIV id="SaturdayVTS3" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>22/09/2018</DD><DT>Operator</DT><DD>Venda and Monique</DD></DL><H4>Fusion protein functionality testing: integration assay
</H4><P>Previous integration assay did not show any integration product on the gel. In this section, we will perform PCR amplication with primer sets that will specifically bind to integration product (primer will bind on the target DNA on one side, and to the adapter on the other side). Since this is the first time this type of experiment was carried out. Different variables were tested:</P><UL class="uladpbl"><LI>Two different reaction time were tested, namely 1 and 16 hours to see the effect of reaction duration on integration. </LI><LI>Some of the reaction were stopped with heat or SDS to release DNA from the protein. PCR clean up were carried out right after the integration to get rid of excess SDS before PCR.</LI><LI>For the rest of the reaction, PCR reaction was done directly after integration.</LI><LI>The PCR product was either directly run on the gel, or was first subjected to PCR clean up to get rid of small background DNA fragment before running it on the gel.</LI><LI>Second PCR was done to reamplify the first PCR product, in case very little product was visualised.</LI></UL><P>The components of the reagents can be found on the protocol. The sequence of the adapter used in this assay is shown on table 1.</P><CENTER><TABLE><CAPTION class="figadpbl"><B>Table 1:</B> Primer sequence to be annealed for the adapter. Note that 5’ of the upper strand of the adapter has to be phosphorylated for successful integration. </CAPTION><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Name</TH><TH class="tableheaderadpbl">Sequence</TH></TR><TR class="tableunevenadpbl"><TH>Adapter upper strand</TH><TD>P-CTGTCTCTTATACACATCT</TD></TR><TR class="tableunevenadpbl"><TH>Adapter lower strand</TH><TD>GCATGGTAAACAGTTCATCCATTTCGCCAATAGATGTGTATAAGAGACAG</TD></TR></TBODY></TABLE></CENTER><P><B>Fusion protein:adapter DNA complex formation</B> 
Tn5 or fusion protein was loaded with adapter DNA shown above by following the pipetting scheme shown on Table 2. All samples and reagents were kept on ice during preparation.
</P><CENTER><TABLE><CAPTION class="figadpbl"><B>Table 2:</B> Pipetting scheme for transposome formation. </CAPTION><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Reaction (µl)</TH><TH class="tableheaderadpbl">Reaction no fusion protein (µl)</TH><TH class="tableheaderadpbl">Reaction no adapter (µl)</TH></TR><TR class="tableunevenadpbl"><TH>Buffer</TH><TD>12</TD><TD>12</TD><TD>12</TD></TR><TR class="tableevenadpbl"><TH>Fusion (~3µM)</TH><TD>96</TD><TD>0</TD><TD>16</TD></TR><TR class="tableunevenadpbl"><TH>Adapter (25µM)</TH><TD>12</TD><TD>12</TD><TD>0</TD></TR><TR class="tableevenadpbl"><TH>Water</TH><TD>0</TD><TD>96</TD><TD>2</TD></TR><TR class="tableunevenadpbl"><TH>Total</TH><TD>120</TD><TD>120</TD><TD>20</TD></TR></TBODY></TABLE></CENTER><P>The reagents were pipetted in the following order: Water, 10x functionality buffer, Tn5 or dxCas9-Tn5, and adapter. The samples were incubated at 23°C for 1 hour.</P><P>
To add appropriate amount of sgRNA, the pipetting scheme on Table 3 was followed. sgRNA J2.1 signifies version 1 of sgRNA targeting juntion 2 of EPO gene:
</P><CENTER><TABLE><CAPTION class="figadpbl"><B>Table 3:</B> Pipetting scheme for sgRNA loading.</CAPTION><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Reaction with sgRNA J2.1 (µl)</TH><TH class="tableheaderadpbl">Reaction no fusion protein (µl)</TH><TH class="tableheaderadpbl">Reaction no adapter (µl)</TH></TR><TR class="tableunevenadpbl"><TH>Transposome</TH><TD>120</TD><TD>120</TD><TD>20</TD></TR><TR class="tableevenadpbl"><TH>sgRNA (100ng/µl)</TH><TD>24</TD><TD>24</TD><TD>4</TD></TR><TR class="tableunevenadpbl"><TH>MgCl<SUB>2</SUB> Buffer</TH><TD>6</TD><TD>6</TD><TD>1</TD></TR></TBODY></TABLE></CENTER><P>The samples were incubated at 37°C for 10 minutes. 
Target DNA (EPO cDNA) was added to make final concentration of ~5ng/µl.
</P><P>The samples were incubated at 37°C for 1 hour. 10µl of samples were taken to be analysed on 5% native TBE polyacrylamide gel (Figure 1). Based on Figure 1a, only uncleaved target DNA at 635bp and Donor DNA at 50pb are observed. No integration or high order complexes are visible. On Figure 1b, it showed that after 16h integration time, the target DNA disappears when Fusion protein is present, and significant smearing is observed, indicating functionality of the fusion protein. No specific cleavage bands nor big complex bands are observed.
</P><B>Figure 1. 2% TAE Agarose gel of Tn5 integration assay. </B><I>. Lanes are (1) DNA ladder (100bp-1000bp), (2) 5µM adapter. 1:3 EPO:transposome, (3) 25µM adapter. 1:3 EPO:transposome, (4) 50µM adapter. 1:3 EPO:transposome, (6) 5µM adapter. 1:1 EPO:transposome, (7) 25µM adapter. 1:1 EPO:transposome, (8) 50µM adapter. 1:1 EPO:transposome.
.  </I><P>25µl of samples were taken and deactivated by either only heat or heat + SDS. This was followed by PCR clean up. </P><P>
To amplify the integration products, PCR was done, both to the cleaned up sample and to the original itegration product. PCR mix and cycle can be found on protocol.

The PCR products were analysed on 5% native TBE polyacrylamide gel. The resulting gels are shown on Figure 2.
</P><B>Figure 2. 5% TBE native PAGE on fusion protein integration assay after PCR of 1 hour of integration duration. </B><I>Primer set 1 are Fw EPO + Rv adapter. Primer set 2 are Rv EPO + Rv EPO. Lanes are (1) DNA ladder, (2) PCR with primer set 1 (3) PCR with primer set 2, (4) No fusion, primer set 1, (5)no fusion, primer set 2, (6) DNA ladder, (7) PCR after PCR clean up with primer set 1, (8) PCR after PCR clean up with primer set 2, (9) without primer, PCR after PCR clean up with primer set 1, (10) without primer, PCR after PCR clean up with primer set 2, (11) protease treated PCR with primer set 1, (12) protease treated PCR with primer set 2, (13) no fusion, protease treated PCR with primer set 1, (14) no fusion, protease treated PCR with primer set 2.
  </I><P>Figure 2 shows 2 interesting PCR products observed (lane 2 and 3). Note the 2 PCR products are not observed in the negative controls (lane 4 and 5). No PCR products observed for lane 1-14 (when PCR clean up was done before PCR), most likely due to too little product, which got lost during PCR clean up.
</P><P>A second PCR was done from the product of the first PCR to amplify the product further. The PCR product wan run on 5% TBE native PAGE shown on Figure 3.</P><P>A more intense band are shown after the second PCR cycle. More intense unspecific bands are shown more on the negative control</P><B>Figure 3. 5% TBE native PAGE on fusion protein integration assay after second PCR of 1 hour (a) and 16 hour (b) of integration duration. </B><I>. Primer set 1 are Fw EPO + Rv adapter. Primer set 2 are Rv EPO + Rv EPO. Lanes are (1) DNA ladder, (2) PCR with primer set 1 (3) PCR with primer set 2, (4) No fusion, primer set 1, (5)no fusion, primer set 2, (6) PCR after PCR clean up with primer set 1, (7) PCR after PCR clean up with primer set 2, (8) without primer, PCR after PCR clean up with primer set 1, (9) without primer, PCR after PCR clean up with primer set 2, (10) protease treated PCR with primer set 1, (11) protease treated PCR with primer set 2, (12) no fusion, protease treated PCR with primer set 1, (13) no fusion, protease treated PCR with primer set 2, (14) DNA ladder. </I><P>The same steps were done with the 16 hours incubated samples. The results of the first PCR are shown on Figure 4.</P><P>Figure 4 shows consistent result between the 1h and 16h integration duration.</P><B>Figure 4. 5% TBE native PAGE on fusion protein integration assay after PCR after 16 hours of incubation. </B><I>Primer set 1 are Fw EPO + Rv adapter. Primer set 2 are Rv EPO + Rv EPO. Lanes are (1) PCR with primer set 1, EPO 4ng/µl (2) PCR with primer set 2, EPO 4ng/µl, (3) No fusion, primer set 1, EPO 4ng/µl, (4)no fusion, primer set 2, EPO 4ng/µl, (5) DNA ladder, (6) PCR with primer set 2, EPO 40ng/µl, (7) PCR with primer set 2, EPO 40ng/µl, (8) No fusion, primer set 1, EPO 40ng/µl, (9)no fusion, primer set 2, EPO 40ng/µl.
  </I><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV></DIV></DIV></UL></FORM><FORM><LABEL for="WeekVTS4" class="subdropdown cadpbl">Week 4 24/09/2018 - 30/09/2018</LABEL><UL class="submenu"><DIV class="subcontent"><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#FridayVTS4">Friday, 28/09/2018</BUTTON><DIV id="FridayVTS4" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>28/09/2018</DD><DT>Operator</DT><DD>Venda and Monique</DD></DL><H4>Fusion protein functionality testing: integration assay
</H4><P>From previous reaction, we saw that we were able to amplify some integration product, eventhough it was difficult to see whether or not it's specifically targetted. Therefore, we included more negative control in this reaction. Moreover, it was shown than performing PCR clean up prior to PCR leads no amplification of the integration product, perhaps due to the loss of products that might have been in very little amount during clean up process. Therefore, the following will be implemented in this reaction </P><UL class="uladpbl"><LI>The course of integration over time will be observed by stoping the integration reaction at 4 different time points, namely 1 hour, 4 hours, 8 hour, and over night.</LI><LI> PCR clean up procedure will not be implemented prior or post PCR reaction.</LI><LI>Second round PCR will be not implemented. </LI><LI>More negative control will be implemented. This includes reaction without adapter (to see whether there will be integration), without EPO (to see whether there will still be amplification product), without sgRNA (to compare the supposedly targetted integration guided by sgRNA vs non targetted integration), reaction with Tn5 (to compare the supposedly targetted integration by our fusion protien vs random integration by Tn5) </LI></UL><P>The components of the reagents can be found on the protocol. The sequence of the adapter used in this assay is shown on table 1.</P><CENTER><TABLE><CAPTION class="figadpbl"><B>Table 1:</B> Primer sequence to be annealed for the adapter. Note that 5’ of the upper strand of the adapter has to be phosphorylated for successful integration. </CAPTION><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Name</TH><TH class="tableheaderadpbl">Sequence</TH></TR><TR class="tableunevenadpbl"><TH>Adapter upper strand</TH><TD>P-CTGTCTCTTATACACATCT</TD></TR><TR class="tableunevenadpbl"><TH>Adapter lower strand</TH><TD>GCATGGTAAACAGTTCATCCATTTCGCCAATAGATGTGTATAAGAGACAG</TD></TR></TBODY></TABLE></CENTER><P><B>Fusion protein:adapter DNA complex formation</B> 
Tn5 or fusion protein was loaded with adapter DNA shown above by following the pipetting scheme shown on Table 2. All samples and reagents were kept on ice during preparation.
</P><CENTER><TABLE><CAPTION class="figadpbl"><B>Table 2:</B> Pipetting scheme for transposome formation. </CAPTION><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Reaction with fusion protein (µl)</TH><TH class="tableheaderadpbl">Reaction no fusion protein (µl)</TH><TH class="tableheaderadpbl">Reaction no adapter (µl)
       </TH><TH class="tableheaderadpbl">Reaction with Tn5 (µl)
      </TH></TR><TR class="tableunevenadpbl"><TH>Buffer</TH><TD>5.2</TD><TD>1.8</TD><TD>1.8</TD><TD>1.8</TD></TR><TR class="tableevenadpbl"><TH>Fusion (~3µM)</TH><TD>41.6</TD><TD>0</TD><TD>14.4</TD><TD>0</TD></TR><TR class="tableunevenadpbl"><TH>Tn5 (~8µM)</TH><TD>0</TD><TD>0</TD><TD>0</TD><TD>5.4</TD></TR><TR class="tableevenadpbl"><TH>Adapter (25µM)</TH><TD>5.2</TD><TD>1.8</TD><TD>0</TD><TD>1.8</TD></TR><TR class="tableunevenadpbl"><TH>Water</TH><TD>0</TD><TD>14.4</TD><TD>1.8</TD><TD>9</TD></TR><TR class="tableevenadpbl"><TH>Total</TH><TD>52</TD><TD>18</TD><TD>18</TD><TD>18</TD></TR></TBODY></TABLE></CENTER><P>The reagents were pipetted in the following order: Water, 10x functionality buffer, Tn5 or dxCas9-Tn5, and adapter. The samples were incubated at 23°C for 1 hour.</P><P>
To add appropriate amount of sgRNA, the pipetting scheme on Table 3 was followed. sgRNA J2.1 signifies version 1 of sgRNA targeting juntion 2 of EPO gene:
</P><CENTER><TABLE><CAPTION class="figadpbl"><B>Table 3:</B> Pipetting scheme for sgRNA loading.</CAPTION><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Reaction with sgRNA J2.1(µl)</TH><TH class="tableheaderadpbl">Reaction no sgRNA/with Tn5 (µl)</TH><TH class="tableheaderadpbl">Reaction no fusion/no adapter with sgRNA J2.1(µl)</TH></TR><TR class="tableunevenadpbl"><TH>Transposome</TH><TD>35</TD><TD>18</TD><TD>18</TD></TR><TR class="tableevenadpbl"><TH>sgRNA (100ng/µl)</TH><TD>7</TD><TD>0</TD><TD>3.6</TD></TR><TR class="tableunevenadpbl"><TH>MgCl<SUB>2</SUB> Buffer</TH><TD>1.75</TD><TD>0.9</TD><TD>0.9</TD></TR><TR class="tableevenadpbl"><TH>Water</TH><TD>0</TD><TD>3.6</TD><TD>0</TD></TR></TBODY></TABLE></CENTER><P>The samples were incubated at 37°C for 10 minutes. 
Target DNA (EPO cDNA) was added to make final concentration of ~5ng/µl. One negative control was done where water instead of EPO cDNA was added.
</P><P>The samples were incubated at 37°C. At incubation time = 1h, 4h, 8h, and over night, 2.5 µl of sample were taken from each reaction and was subjected to PCR. PCR mix and cycle can be found on protocol.

The PCR products were analysed on 5% native TBE polyacrylamide gel. The resulting gel is either stained with ethidium bromide and imaged with GelDoc system, or directly imaged with Typhoon Imaging System. The resulting gels are shown on Figure 1.
</P><B>Figure 1. 5% TBE native PAGE stained by EtBr and imaged by GelDoc system.Integration reaction after 1 hours.  </B><I>Lanes are (1) DNA ladder (100-1000bp), (2) amplified product with fw EPO (3) amplified product with rv EPO, (4) amplified product with fw EPO without sgRNA, (5) amplified product with rv EPO without sgRNA, (6) amplified product with fw EPO without fusion protein, (7) amplified product with rv EPO without fusion protein, (8) amplified product with fw EPO without adapter DNA, (9) amplified product with rv EPO without adapter DNA, (10) amplified product with fw EPO with Tn5, (11) amplified product with rv EPO with Tn5, (12) amplified product with fw EPO without target DNA, (13) amplified product 2 without target DNA.
  </I><P>Figure 1 showed the result of PCR. Hyphothetically, sgRNA would steer the fusion protein to the target EPO (~600bp), specifically to the area close to the 160th bp. When specific integration occurs, target EPO will be split into two fragments of around  280 and 480. PCR reactions were done with primer sets that would amplify these fragments seperately. Lane 2 shows the PCR of the first fragment. A very intense band were indeed present around 230bp, confirming the presence of this integration product. Lane 3 shows the PCR of second fragment. An intense band were also present around 480bp, which also confirmed the presence of the second integration product. On the negative control lanes (without sgRNA or with Tn5), random bands were present, and none of these intense bands shown on lane 2 and 3 were observed here . This confirmed further that targeted integration indeed occur due to our fusion protein that is loaded with sgRNA. The absence of amplification bands on the other negative controls confirmed our result further. </P><P>Results of the longer incubation time shows similar result. However, it seems like the specific bands loses intensity with increasing time, suggesting perhaps re-integration events done by excess fusion protein. The higher intensity of unspecific lower bands over time strengthen this analysis further. Figure 2,3 and 4 displays the result of PCR after 4, 8, and overnight incubation time</P><B>Figure 2. 5% TBE native PAGE stained by EtBr and imaged by GelDoc system. Integration reaction after 4 hours. </B><I> Lanes are (1) DNA ladder (100-1000bp), (2) amplified product with fw EPO (3) amplified product with rv EPO, (4) amplified product with fw EPO without sgRNA, (5) amplified product with rv EPO without sgRNA, (6) amplified product with fw EPO without fusion protein, (7) amplified product with rv EPO without fusion protein, (8) amplified product with fw EPO without adapter DNA, (9) amplified product with rv EPO without adapter DNA, (10) amplified product with fw EPO with Tn5, (11) amplified product with rv EPO with Tn5, (12) amplified product with fw EPO without target DNA, (13) amplified product 2 without target DNA
.  </I><B>Figure 3. 5% TBE native PAGE stained by EtBr and imaged by GelDoc system. Integration reaction after 8 hours. </B><I>. Lanes are (1) DNA ladder (100-1000bp), (2) amplified product with fw EPO (3) amplified product with rv EPO, (4) amplified product with fw EPO without sgRNA, (5) amplified product with rv EPO without sgRNA, (6) amplified product with fw EPO without fusion protein, (7) amplified product with rv EPO without fusion protein, (8) amplified product with fw EPO without adapter DNA, (9) amplified product with rv EPO without adapter DNA, (10) amplified product with fw EPO with Tn5, (11) amplified product with rv EPO with Tn5, (12) amplified product with fw EPO without target DNA, (13) amplified product 2 without target DNA
.  </I><B>Figure 4. 5% TBE native PAGE stained by EtBr and imaged by GelDoc system. Integration reaction overnight. </B><I>. Lanes are (1) amplified product with fw EPO (2) amplified product with rv EPO, (3) amplified product with fw EPO without sgRNA, (4) amplified product with rv EPO without sgRNA, (5) DNA ladder (100-1000bp), (6) amplified product with fw EPO without fusion protein, (7) amplified product with rv EPO without fusion protein, (8) amplified product with fw EPO without adapter DNA, (9) amplified product with rv EPO without adapter DNA, (10) amplified product with fw EPO with Tn5, (11) amplified product with rv EPO with Tn5, (12) amplified product with fw EPO without target DNA, (13) amplified product 2 without target DNA
.  </I></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#SundayVTS4">Sunday, 30/09/2018</BUTTON><DIV id="SundayVTS4" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>30/09/2018</DD><DT>Operator</DT><DD>Venda and Monique</DD></DL><H4>Fusion protein functionality testing: integration assay
</H4><P>From previous reaction, we saw that we were able to amplify the specific integration product that we expected, which, together with the results of the negative control, confirmed the ability of our fusion protein to perform specific integration. In this experiment, we would like to confirm this further by using a longer target DNA. If specific integration indeed occur, one of the new amplified fragment should also get longer by the same increase. Moreover, more sgRNA were tested to check whether other parts of the target EPO is also accessible. The following will therefore be done:</P><UL class="uladpbl"><LI>Incubation time of 1 hour will be implemented</LI><LI> 3 different EPO fragments were used. The original EPO cDNA, longer EPO with &quot;intron&quot; on the third junction, longer EPO with &quot;intron&quot; on the fourth junction</LI><LI>4 different sgRNA was used, targeting junction 1-4</LI><LI>reaction without gRNA, and reaction with Tn5 will be done in parallel as negative controls</LI></UL><P>The components of the reagents can be found on the protocol. The sequence of the adapter used in this assay is shown on table 1.</P><CENTER><TABLE><CAPTION class="figadpbl"><B>Table 1:</B> Primer sequence to be annealed for the adapter. Note that 5’ of the upper strand of the adapter has to be phosphorylated for successful integration. </CAPTION><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Name</TH><TH class="tableheaderadpbl">Sequence</TH></TR><TR class="tableunevenadpbl"><TH>Adapter upper strand</TH><TD>P-CTGTCTCTTATACACATCT</TD></TR><TR class="tableunevenadpbl"><TH>Adapter lower strand</TH><TD>GCATGGTAAACAGTTCATCCATTTCGCCAATAGATGTGTATAAGAGACAG</TD></TR></TBODY></TABLE></CENTER><P><B>Fusion protein:adapter DNA complex formation</B> 
Tn5 or fusion protein was loaded with adapter DNA shown above by following the pipetting scheme shown on Table 2. All samples and reagents were kept on ice during preparation.
</P><CENTER><TABLE><CAPTION class="figadpbl"><B>Table 2:</B> Pipetting scheme for transposome formation. </CAPTION><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Reaction (µl)</TH><TH class="tableheaderadpbl">Reaction no fusion protein (µl)</TH><TH class="tableheaderadpbl">Reaction with Tn5 (µl)</TH></TR><TR class="tableunevenadpbl"><TH>Buffer</TH><TD>4.5</TD><TD>1</TD><TD>1</TD></TR><TR class="tableevenadpbl"><TH>Fusion (~3µM)</TH><TD>36</TD><TD>0</TD><TD>0</TD></TR><TR class="tableevenadpbl"><TH>Tn5 (~8µM)</TH><TD>0</TD><TD>0</TD><TD>3</TD></TR><TR class="tableunevenadpbl"><TH>Adapter (25µM)</TH><TD>4.5</TD><TD>1</TD><TD>1</TD></TR><TR class="tableevenadpbl"><TH>Water</TH><TD>0</TD><TD>8</TD><TD>5</TD></TR><TR class="tableunevenadpbl"><TH>Total</TH><TD>45</TD><TD>10</TD><TD>10</TD></TR></TBODY></TABLE></CENTER><P>The reagents were pipetted in the following order: Water, 10x functionality buffer, Tn5 or dxCas9-Tn5, and adapter. The samples were incubated at 23°C for 1 hour.</P><P>
To add appropriate amount of sgRNA, the pipetting scheme on Table 3 was followed:
</P><CENTER><TABLE><CAPTION class="figadpbl"><B>Table 3:</B> Pipetting scheme for sgRNA loading. sgRNA J2.1 signifies version 1 of sgRNA targeting juntion 2 of EPO gene, etc. </CAPTION><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">sgRNA J1.1 (µl)</TH><TH class="tableheaderadpbl">sgRNA J2.1 (µl)</TH><TH class="tableheaderadpbl">sgRNA J3.1 (µl)</TH><TH class="tableheaderadpbl">sgRNA J4.1 (µl)</TH><TH class="tableheaderadpbl">no sgRNA (µl)</TH><TH class="tableheaderadpbl">no fusion, J2.1 (µl)</TH><TH class="tableheaderadpbl">Tn5 (µl)</TH></TR><TR class="tableunevenadpbl"><TH>Transposome</TH><TD>4</TD><TD>12</TD><TD>8</TD><TD>8</TD><TD>12</TD><TD>10</TD><TD>10</TD></TR><TR class="tableevenadpbl"><TH>sgRNA (100ng/µl)</TH><TD>0.8</TD><TD>2.4</TD><TD>1.6</TD><TD>1.6</TD><TD>0</TD><TD>2</TD><TD>0</TD></TR><TR class="tableunevenadpbl"><TH>MgCl<SUB>2</SUB> Buffer</TH><TD>0.2</TD><TD>0.6</TD><TD>0.4</TD><TD>0.4</TD><TD>0.6</TD><TD>0.5</TD><TD>0.5</TD></TR><TR class="tableevenadpbl"><TH>Water</TH><TD>0</TD><TD>0</TD><TD>0</TD><TD>0.9</TD><TD>2.4</TD><TD>0</TD><TD>2</TD></TR></TBODY></TABLE></CENTER><P>The samples were incubated at 37°C for 10 minutes. 
Target DNA (EPO cDNA) was added to make final concentration of ~5ng/µl. Three different types of EPO are present. The following reactions were done:</P><UL class="uladpbl"><LI>Normal EPO was targetted by sgRNAJ1.1, J2.1, J3.1, J4.1 to see whether all junctions are equality accessible</LI><LI>EPO with intron on Junction 3 was targetted by sgRNA 2.1 to see the increase of the size of the second fragment in comparison to the previous successfull experiments.</LI><LI>EPO with intron on Junction 3 was targetted by J3.1 to see whether EPO can be targetted even though the target site is disrupted by an intron sequence. </LI><LI>EPO with intron on Junction 4 was targetted by J4.1 to see whether EPO can be targetted even though the target site is disrupted by an intron sequence.</LI><LI>As a negative control reaction without sgRNA with normal EPO or EPO with intron on junction 3 or 4 were carried out in parallel</LI></UL><P>The samples were incubated at 37°C. At incubation time = 30 mins, 2.5 µl of sample from reaction with normal EPO + sgRNA J2.1 were taken along with the necessary negative control reactions to see whether we can decrease off target even more by deacreasing reaction time. These samples were then subjected to PCR based on the protocol. At incubation time=1 hours, 2.5µl sample from each reaction were subjected to PCR. PCR mix and cycle can be found on protocol.

The PCR products were analysed on 5% native TBE polyacrylamide gel. The resulting gel is either stained with ethidium bromide and imaged with GelDoc system. The resulting gels for intergration with sgRNA J1.1 - J4.1 are shown on figure 1-4.
</P><B>Figure 1. 5% TBE Native PAGE of integration with J1.1. </B><I> Primer sets 1 will amplify the first integration product. Primer sets 2 will amplify the second integration product. Lanes are (1) DNA ladder, (2) sgRNA J1.1, with primer set 1, (3) sgRNA J1.1, with primer set 2, (4) DNA ladder, (5) no sgRNA with primer set 1, (6) no sgRNA with primer set 2.
  </I><P>Figure 1 showed  uncleaved target DNA at 635bp and no integration fragments at 148bp and 548bp are observed (lane 2 and 3). Very little background is observed from ~150bp to ~50bp, probably due to off target events from the fusion protein. 
</P><B>Figure 2. 5% TBE Native PAGE of integration with J2.1.  </B><I>Primer sets 1 will amplify the first integration product. Primer sets 2 will amplify the second integration product. Lanes are (1) DNA ladder, (2) 30 mins, sgRNA J2.1, with primer set 1, with EPO (3) 30 mins, sgRNA J2.1, with primer set 2, with EPO ,(4) 1h, sgRNA J2.1, with primer set 1, with EPO (5) 1h, sgRNA J2.1, with primer set 2, with EPO , (6) 1h, sgRNA J2.1, with primer set 1, with EPO_intron3 (7) 1h, sgRNA J2.1, with primer set 2, with EPO_intron3, (8) DNA ladder, (9) no sgRNA, primer set 1, with EPO (10) no sgRNA, primer set 2, with EPO, (11) no sgRNA, primer set 1, with EPO_intron3 (12) no sgRNA, primer set 2, with EPO_intron3
  </I><P>Figure 2 Uncleaved target DNA at 635bp and 1247bp and 2 integration fragments at 280bp and 480/1068bp are observed (lane 1-7). Note the 2 PCR products are not observed in the negative controls (lane 9-12). Very little background is observed from ~150bp to ~50bp, probably due to off target events from the fusion protein. Note with the intron at junction 3, the integrated fragment is 1063, and the target DNA is 1247, causing the bands to un very close to each other. 
</P><B>Figure 3. 5% TBE Native PAGE of integration with J3.1.  </B><I>Primer sets 1 will amplify the first integration product. Primer sets 2 will amplify the second integration product. Lanes are (1) DNA ladder, (2) 1h, sgRNA J3.1, with primer set 1, with EPO (3) 1h, sgRNA J3.1, with primer set 2, with EPO , (4) 1h, sgRNA J3.1, with primer set 1, with EPO_intron3 (5) 1h, sgRNA J4.1, with primer set 2, with EPO_intron3, (6)DNA Ladder, (7) no sgRNA, primer set 1, with EPO (8) no sgRNA, primer set 2, with EPO, (9) no sgRNA, primer set 1, with EPO_intron3 (10) no sgRNA, primer set 2, with EPO_intron3.
  </I><P>Figure 3 snows uncleaved target DNA at 635bp and 1247bp and no specific integration products can be seen, and no difference in bands can be observed between the sample with and without gRNA (lane 2,3 vs lane 7,8 and lane 4,5 vs lane 9,10).
</P><B>Figure 4. 5% TBE Native PAGE of integration with J4.1.  </B><I>Primer sets 1 will amplify the first integration product. Primer sets 2 will amplify the second integration product. Lanes are (1) DNA ladder, (2) 1h, sgRNA J4.1, with primer set 1, with EPO (3) 1h, sgRNA J4.1, with primer set 2, with EPO , (4) 1h, sgRNA J4.1, with primer set 1, with EPO_intron4 (5) 1h, sgRNA J4.1, with primer set 2, with EPO_intron4, (6) no sgRNA, primer set 1, with EPO (7) no sgRNA, primer set 2, with EPO, (8) no sgRNA, primer set 1, with EPO_intron4 (9) no sgRNA, primer set 2, with EPO_intron4
  </I><P>Figure 4 snows uncleaved target DNA at 635bp and no integration fragments at 284bp and 448/1053bp are observed (lane 2-5). Very little background is observed from ~150bp to ~50bp, probably due to off target events from the fusion protein. EPO_I3 in late 5 and 9 has an interesting amplification pattern. 

</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV></DIV></UL></FORM><H3 class="adpbl">October</H3><FORM><LABEL for="WeekVTO1" class="subdropdown cadpbl">Week 3 15/10/2018 - 17/10/2018</LABEL><UL class="submenu"><DIV class="subcontent"><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#MondayVTO1">Monday, 15/10/2018</BUTTON><DIV id="MondayVTO1" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>15/10/2018</DD><DT>Operator</DT><DD>Venda Mangkusaputra</DD></DL><H4>Sequence Verification of Integration Products </H4><P> In order to confirm that the integration products of the functionality test were the ones we expected, we attempted to sequence verify them. We load the whole PCR product visualized on lane 2 and 3 from functionality assay on 28/09/2018 on 2% agarose gel and isolated the bands from the gel and purify them. Since a very low concentration was achieved, another round of PCR was done using the isolated fragment as template. We cleaned up the PCR product and achieved concentration of 250ng/µl for fragment 1, and 50ng/µl for fragment 2. </P><P>We sent these PCR products individually for sequencing. The alignment result of this sequence is shown on Figure 1.</P><I>Sequence alignment of integration product (two red arrows) with the target EPO. Pink box represent the sgRNA binding site.
  </I><P> Figure 1 shows the sequence alignment between the two fragments and the target EPO gene.From the sequence alignment, we can confirm that integration occured upstream the sgRNA binding site. The overlapping alignment between the two products  suggested that integration does not always occur at the exact base pair position (~10bp variation), which might be the result of the flexible linker between dxCas9 and Tn5.
</P><P>Next, we used pGEM T-Easy cloning method to clone these fragments on individual plasmid to be able to sequence verify them individually. By doing this we will be able to see the variations within that band individually. The protocol for pGEM T-Easy can be found on the protocol page. We used 2µl of 10x diluted sample of isolated fragment 1 or 1µl of isolated fragment 2 to be cloned into the plasmid. We were able to get white colonies and isolated their plasmid, but however, the sequence verification result did not come on time.</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV></DIV></UL></FORM></DIV></DIV></DIV><DIV class="container nowbg" id="InVivo"><DIV class="row nowbg"><DIV class="col-xs-12 col-sm-3 col-lg-3"><NAV id="myScrollspy" class="hidden-xs hidden-sm"><UL class="nav nav-pills nav-stacked" data-spy="affix"><LI class="dropdown"><A class="dropdown-toggle adpbl" data-toggle="dropdown" href="#">July</A></LI><LI><A href="#JulyInVivo3" class="adpbl">Week: 16/07/2018 - 22/07/2018</A></LI></UL><LI class="dropdown"><A class="dropdown-toggle adpbl" data-toggle="dropdown" href="#">August</A></LI><LI><A href="#AugustInVivo1" class="adpbl">Week: 30/07/2018 - 05/08/2018</A></LI><LI><A href="#AugustInVivo2" class="adpbl">Week: 06/08/2018 - 12/08/2018</A></LI><LI><A href="#AugustInVivo3" class="adpbl">Week: 13/08/2018 - 19/08/2018</A></LI><LI><A href="#AugustInVivo4" class="adpbl">Week: 20/08/2018 - 26/08/2018</A></LI><LI><A href="#AugustInVivo5" class="adpbl">Week: 27/08/2018 - 02/09/2018</A></LI><LI class="dropdown"><A class="dropdown-toggle adpbl" data-toggle="dropdown" href="#">September</A></LI><LI><A href="#SeptemberInVivo1" class="adpbl">Week: 03/09/2018 - 09/09/2018</A></LI><LI><A href="#SeptemberInVivo2" class="adpbl">Week: 10/09/2018 - 16/09/2018</A></LI><LI><A href="#SeptemberInVivo3" class="adpbl">Week: 17/09/2018 - 23/09/2018</A></LI><LI><A href="#SeptemberInVivo4" class="adpbl">Week: 24/09/2018 - 30/09/2018</A></LI></NAV></DIV><DIV class="col-lg-8 col-md-8 col-sm-12 col-xs-12 nowbg" style="margin-right:20px;"><H3 class="adpbl">July</H3><FORM><LABEL for="WeekVVJ3" class="subdropdown cadpbl">Week 3 16/07/2018 - 23/07/2018</LABEL><UL class="submenu"><DIV class="subcontent"><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#TuesdayVVJ1">Tuesday, 17/07/2018</BUTTON><DIV id="TuesdayVVJ1" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>17/7/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>High Fidelity PCR</H4><P> The following high fidelity PCR reactions were carried out:
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Amplicon</TH><TH class="tableheaderadpbl">Size (bp)</TH><TH class="tableheaderadpbl">Fw primer</TH><TH class="tableheaderadpbl">Rv primer</TH><TH class="tableheaderadpbl">template</TH><TH class="tableheaderadpbl">Elongation time (sec)</TH><TH class="tableheaderadpbl">Annealing temperature (°C)</TH></TR><TR class="tableunevenadpbl"><TD>LacZ-gRNA</TD><TD>314</TD><TD>IV003</TD><TD>IV004</TD><TD>Gblock</TD><TD>60</TD><TD>60</TD></TR><TR class="tableevenadpbl"><TD>KanR CDS</TD><TD>862</TD><TD>IV005</TD><TD>IV006</TD><TD>pSB1K3</TD><TD>60</TD><TD>60</TD></TR></TBODY></TABLE></CENTER><P>PCR mixtures were prepared, according to the High Fidelity PCR protocol. After the programme was finished, samples were kept at 4°C.</P><DL><DT>Date</DT><DD>17/7/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>PCR clean-up</H4><P>
PCR products were subjected to PCR Clean-Up, according to the protocol. Elution was done in 50 µL Milli-Q.
</P><DL><DT>Date</DT><DD>17/7/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>Nanodrop DNA Quantification</H4><P> 
    The following isolates were quantified according to the Nanodrop DNA Quantification protocol. The concentration and purity of the DNA (260/280 and 260/230 ratios) were measured and verified, but not registered. 
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Isolate</TH><TH class="tableheaderadpbl">Concentration (ng/uL)
  </TH></TR><TR class="tableunevenadpbl"><TD>LacZ-gRNA</TD><TD>85</TD></TR><TR class="tableevenadpbl"><TD>KanR cassette</TD><TD>209</TD></TR></TBODY></TABLE></CENTER></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV></DIV></UL></FORM><H3 class="adpbl">AUGUST</H3><FORM><LABEL for="WeekVVA2" class="subdropdown cadpbl">Week 2 06/08/2018 - 12/08/2018</LABEL><UL class="submenu"><DIV class="subcontent"><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#FridayVVA2">Friday, 10/08/2018</BUTTON><DIV id="FridayVVA2" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>10/8/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>GBlock resuspension</H4><P>GBlock <I>lacZ</I> gRNA (314bp) was resuspended according to the instructions provided by the supplier (IDT DNA).</P><DL><DT>Date</DT><DD>10/8/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>High Fidelity PCR</H4><P> To amplify GBlock <I>lacZ</I> gRNA  (314bp) and the Kanamycin resistance gene (KanR), high fidelity PCR was done according to the High Fidelity PCR protocol.
    
The following primer combination(s) were predicted to result in the following expected amplicon length(s)
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Template</TH><TH class="tableheaderadpbl">Fw primer</TH><TH class="tableheaderadpbl">Rv primer</TH><TH class="tableheaderadpbl">expected size (bp) </TH></TR><TR class="tableunevenadpbl"><TD><I>lacZ</I> Gblock</TD><TD>IV003</TD><TD>IV004</TD><TD>314</TD></TR><TR class="tableevenadpbl"><TD>pSB1K3</TD><TD>IV005</TD><TD>IV006</TD><TD>862</TD></TR></TBODY></TABLE></CENTER><P>The thermocycler programme was installed to have an extension time of 30sec and an annealing temperature of 60 °C. After the programme was finished, samples were kept at 4 °C.</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#SaturdayVVA2">Saturday, 11/08/2018</BUTTON><DIV id="SaturdayVVA2" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>11/8/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>DNA Gel Electrophoresis</H4><P>PCR samples from 10/8/2018 was run on a 0.8% agarose gel in TAE buffer. 5μL was pipetted into each lane. The gel was imaged using a gel documentation system with UV-light.</P><P>This gel shows the KanR gene with ME flanks (862bp) was successfully PCRed from pSB1K3 (lanes 5-7). Amplification of the Gblock gives a very faint band at the expected size (314bp) for <I>lacZ </I> gRNA (lanes 2-4).</P><DL><DT>Date</DT><DD>11/8/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>High Fidelity PCR</H4><P> To amplify the <I>lacZ</I> gRNA (314bp) and linearizing pACYC-dxCas9-LinkerTn5 (9388bp), high fidelity PCR was done. 50µL PCR mixtures were prepared, according to the High Fidelity PCR protocol.
    </P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Template</TH><TH class="tableheaderadpbl">Fw primer</TH><TH class="tableheaderadpbl">Rv primer</TH><TH class="tableheaderadpbl">expected size (bp)</TH><TH class="tableheaderadpbl">elongation time</TH></TR><TR class="tableunevenadpbl"><TD><I>lacZ</I> gRNA Gblock</TD><TD>IV003</TD><TD>IV004</TD><TD>314</TD><TD>15 sec</TD></TR><TR class="tableevenadpbl"><TD>pACYC-dxCas9-LinkerTn5</TD><TD>IV001</TD><TD>IV002</TD><TD>9388</TD><TD>5 min</TD></TR></TBODY></TABLE></CENTER><P>The thermocycler programme was installed to have an annealing temperature of 60 °C. After the programme was finished, samples were kept at 4 °C.</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#SundayVVA2">Sunday, 12/08/2018</BUTTON><DIV id="SundayVVA2" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>12/8/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>DNA Gel Electrophoresis</H4><P>PCR products from 11/8/2018 was run on a 0.8% agarose gel in TAE buffer. 5μL was pipetted into each lane. The gel was imaged using a gel documentation system with UV-light.</P><P>This gel shows that pACYC-dxCas9-LinkerTn5 was successfully linearized (lane 8, 9 and 10), as there is a strong amplicon signal far above 5kbp, and we expected 9388bp. Amplification of the Gblock showed very faint bands at the expected size (314bp) (lane 2, 3 and 4).</P><DL><DT>Date</DT><DD>12/8/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>PCR clean-up</H4><P>
    PCR samples from 10/8/2018 (<I>lacZ</I> gRNA and KanR) and 11/8/2018 (<I>lacZ</I> gRNA and linearized pACYC-dxCas9Tn5) were subjected to PCR Clean-Up, according to the protocol. Elution was done in 40 µL Milli-Q (2 times 20).
</P><DL><DT>Date</DT><DD>12/8/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>Nanodrop DNA Quantification</H4><P> 
    The following isolates were quantified according to the Nanodrop DNA Quantification protocol. The concentration and purity of the DNA (260/280 and 260/230 ratios) were measured. 
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Isolate</TH><TH class="tableheaderadpbl">Sample date</TH><TH class="tableheaderadpbl">Concentration (ng/uL)</TH><TH class="tableheaderadpbl">260/280 ratio</TH><TH class="tableheaderadpbl">260/230 ratio</TH></TR><TR class="tableunevenadpbl"><TD><I>lacZ</I> gRNA</TD><TD>10/8/2018</TD><TD>180.6</TD><TD>1.9</TD><TD>2.0</TD></TR><TR class="tableevenadpbl"><TD>ME flanked KanR gene</TD><TD>10/8/2018</TD><TD>220</TD><TD>1.8</TD><TD>2.1</TD></TR><TR class="tableunevenadpbl"><TD><I>lacZ</I> gRNA</TD><TD>11/8/2018</TD><TD>155</TD><TD>1.8</TD><TD>2.0</TD></TR><TR class="tableevenadpbl"><TD>linearized pACYCdxCas9Tn5</TD><TD>11/8/2018</TD><TD>220</TD><TD>1.8</TD><TD>2.1</TD></TR></TBODY></TABLE></CENTER><DL><DT>Date</DT><DD>12/8/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>Gibson Assembly</H4><P>According to the displayed reaction mixture composition, Gibson Assembly was done to assemble <I>lacZ</I> gRNA (insert) and linearized pACYC-dxCas9-LinkerTn5 (vector) to obtain pACYC-dxCas9-LinkerTn5-<I>lacZ</I>gRNA (9622bp).
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL)</TH></TR><TR class="tableunevenadpbl"><TD>Gibson Assembly Mastermix 2X</TD><TD>10</TD></TR><TR class="tableevenadpbl"><TD>Vector</TD><TD>9.4 (2 µg)</TD></TR><TR class="tableunevenadpbl"><TD>Insert</TD><TD>0.6 (100 ng)</TD></TR></TBODY></TABLE></CENTER><P>The samples were incubated at 50°C for 20 minutes, after which they were kept at 4°C.</P><DL><DT>Date</DT><DD>12/8/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>Solid Medium Preparation</H4><P>
Solid Luria Broth (LB) medium was prepared, complemented with chloramphenicol (Cam). Plates were stored at 4°C.
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV></DIV></UL></FORM><FORM><LABEL for="WeekVVA3" class="subdropdown cadpbl">Week 3 13/08/2018 - 19/08/2018</LABEL><UL class="submenu"><DIV class="subcontent"><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#MondayVVA3">Monday, 13/08/2018</BUTTON><DIV id="MondayVVA3" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>13/8/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4> Transformation of chemically competent cells </H4><P>The following transformations were carried out according to the protocol for transformation of chemically competent cells:
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Strain</TH><TH class="tableheaderadpbl">Plasmid &amp; size (bp) </TH><TH class="tableheaderadpbl">Cloning method</TH><TH class="tableheaderadpbl">Volume added (uL)</TH><TH class="tableheaderadpbl">Medium on which was plated</TH></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I>DH5α</TD><TD>pACYC-dxCas9-LinkerTn5-<I>lacZ</I>gRNA (9622bp)</TD><TD>Gibson Assembly</TD><TD>10µl</TD><TD>LB+Cam</TD></TR><TR class="tableevenadpbl"><TD><I>E.coli</I>DH5α</TD><TD>pACYC-dxCas9-LinkerTn5-<I>lacZ</I>gRNA (9622bp)</TD><TD>Gibson Assembly</TD><TD>6µl</TD><TD>LB+Cam</TD></TR><TR class="tableevenadpbl"><TD><I>E.coli</I>DH5α</TD><TD>linearized pACYC-dxCas9-LinkerTn5</TD><TD><I>neg. control</I></TD><TD>2µl</TD><TD>LB+Cam</TD></TR></TBODY></TABLE></CENTER><P>200 µL LB medium was added as recovery medium.
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#TuesdayVVA3">Tuesday, 14/08/2018</BUTTON><DIV id="TuesdayVVA3" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>14/8/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4> Transformation Results </H4><P> After carrying out the transformation protocol on 13/8/2018, the plates were left for overnight incubation. After ~14-18hr incubation, the following results were observed: </P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Strain</TH><TH class="tableheaderadpbl">DNA </TH><TH class="tableheaderadpbl">Medium</TH><TH class="tableheaderadpbl">Result</TH></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I>DH5α</TD><TD>10µl pACYC-dxCas9-LinkerTn5-<I>lacZ</I>gRNA (9622bp)</TD><TD>LB+Cam</TD><TD>growth</TD></TR><TR class="tableevenadpbl"><TD><I>E.coli</I>DH5α</TD><TD>6µl pACYC-dxCas9-LinkerTn5-<I>lacZ</I>gRNA (9622bp)</TD><TD>LB+Cam</TD><TD>growth</TD></TR><TR class="tableevenadpbl"><TD><I>E.coli</I>DH5α</TD><TD>linearized pACYC-dxCas9-LinkerTn5</TD><TD><I>LB+Cam</I></TD><TD>less growth</TD><P>The negative control shows that there is still instact plasmid template, or self-ligated plasmid present in the transformation mixture provided. However, since the amount of colonies is less, there is a strong indicatin that the pACYC-dxCas9-LinkerTn5-<I>lacZ</I>gRNA might be constructed. It was decided to proceed with colony PCR.</P></TR></TBODY></TABLE></CENTER><DL><DT>Date</DT><DD>14/8/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>Colony PCR</H4><P>Colony PCR was done to verify propagation of pACYC-dxCas9-LinkerTn5-<I>lacZ</I>gRNA after transformation (13/8/2018). This plasmid contains a fragment with dxCas9-LinkerTn5-<I>lacZ</I>gRNA, which can be checked with primers 018 and IV004.
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL)</TH></TR><TR class="tableunevenadpbl"><TD>Template DNA</TD><TD>picked colony</TD></TR><TR class="tableevenadpbl"><TD>Forward primer</TD><TD>018</TD></TR><TR class="tableunevenadpbl"><TD>Reverse primer</TD><TD>IV004</TD></TR><TR class="tableunevenadpbl"><TD>Expected amplicon size (bp)</TD><TD>2148</TD></TR></TBODY></TABLE></CENTER><P>14 tubes were prepared for colony PCR, containing picked colony as template DNA. The thermocycler programme was installed to have an extension time of 2.5 minutes and an annealing temperature of 60 °C. After the programme was finished, samples were kept at 4°C.</P><DL><DT>Date</DT><DD>14/8/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>DNA Gel Electrophoresis</H4><P>Colony PCR samples were run on a 0.8% agarose gel in TAE buffer. 5μL was pipetted into each lane. The gel was imaged using a gel documentation system with UV-light.</P><P>An amplicon of 2148bp was expected for this PCR reaction. For almost all colonies, we observe a band around this height. Not all bands seem to be at exactly the same height. Based on this gel, it was decided to inoculate colony 1, 2, 3 and 4</P><DL><DT>Date</DT><DD>14/8/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>Liquid Starter Culture Preparation</H4><P>
Colonies 1, 2, 3 and 4 from the plates from 13/8/2018 were used for inoculation of 10mL liquid starter culture (Luria Broth, complemented with Chloramphenicol) following the Liquid Starter Culture Protocol.
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#WednesdayVVA3">Wednesday, 15/08/2018</BUTTON><DIV id="WednesdayVVA3" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>15/8/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>Plasmid Isolation</H4><P> 
    Overnight starter cultures of colonies 1, 2, 3, 4 (plates 13/8/2018) were subjected to plasmid isolation following the Plasmid Isolation Protocol. Final elution was done with 30 μL of pre-warmed MilliQ.
</P><DL><DT>Date</DT><DD>15/8/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>Nanodrop DNA Quantification</H4><P> 
    The following isolates were quantified according to the Nanodrop DNA Quantification protocol. The concentration and purity of the DNA (260/280 and 260/230 ratios) were measured and verified to be around 1.8~1.9. 
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Isolate</TH><TH class="tableheaderadpbl">Concentration (ng/uL)</TH></TR><TR class="tableunevenadpbl"><TD>colony 1</TD><TD>32</TD></TR><TR class="tableevenadpbl"><TD>colony 2</TD><TD>35</TD></TR><TR class="tableunevenadpbl"><TD>colony 3</TD><TD>41</TD></TR><TR class="tableevenadpbl"><TD>colony 4</TD><TD>36</TD></TR></TBODY></TABLE></CENTER><DL><DT>Date</DT><DD>15/8/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>Sequence Verification from plasmid</H4><P> For verification of insertion of <I>lacZ</I> gRNA into pACYCdxCas9Tn5, primers IV003 and IV004 were used for sequencing. Only plasmid isolates from colony 1 and 2 were sent for sequencing. The sequencing starts from the homology flanks, in the direction of the <I>lacZ</I> gRNA. The 10µL sequencing sample was prepared in a 1.5 mL tube as follows: 

</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL)</TH><TH class="tableheaderadpbl">Final concentration</TH></TR><TR class="tableunevenadpbl"><TD>Plasmid DNA (500ng)</TD><TD>7.5</TD><TD>50 ng/µL</TD></TR><TR class="tableevenadpbl"><TD>10µM primer </TD><TD>2.5</TD><TD>25 pmol</TD></TR></TBODY></TABLE></CENTER><P>Within 7 days, the sequence was retrievable online and alignment with the <I>in silico</I> designed molecule showed 100% match of the 1050 bp downstream of the sequencing primer binding site. This was only the case for the plasmid isolate from colony 1. <B>This means the pACYC-dxCas9-LinkerTn5-<I>lacZ</I>gRNA construct is sequence verified.</B></P><DL><DT>Date</DT><DD>15/8/2015</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4> Transformation of chemically competent cells </H4><P>The following transformations were carried out according to the protocol for transformation of chemically competent cells:
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Strain</TH><TH class="tableheaderadpbl">Plasmid </TH><TH class="tableheaderadpbl">Volume added (µL)</TH><TH class="tableheaderadpbl">Medium on which was plated</TH></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I>DH5α</TD><TD>pSB1K3</TD><TD>5</TD><TD>LB+Kan</TD></TR><TR class="tableevenadpbl"><TD><I>E.coli</I>DH5α</TD><TD>- (neg. control)</TD><TD>-</TD><TD>LB+Kan</TD></TR></TBODY></TABLE></CENTER><P>200 uL LB medium was added as recovery medium.</P><DL><DT>Date</DT><DD>15/8/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4> Transformation Results </H4><P> After carrying out the transformation protocol in the morning, the plates were left for overday incubation in the stove. After ~12hr incubation, the following results were observed: </P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Strain</TH><TH class="tableheaderadpbl">DNA </TH><TH class="tableheaderadpbl">Medium</TH><TH class="tableheaderadpbl">Result</TH></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I> DH5α  </TD><TD>pSB1K3  </TD><TD>LB+Kan  </TD><TD>colony growth</TD></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I> DH5α  </TD><TD>-  </TD><TD>LB+Kan  </TD><TD>no growth</TD></TR></TBODY></TABLE></CENTER><DL><DT>Date</DT><DD>15/8/2015</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>Liquid Starter Culture Preparation</H4><P>
1 colony from the LB+Kan was used for inoculation of 5mL liquid starter culture (Luria Broth, complemented with Kanamycin) following the Liquid Starter Culture Protocol.
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#ThursdayVVA3">Thursday, 16/08/2018</BUTTON><DIV id="ThursdayVVA3" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>16/8/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>Plasmid Isolation</H4><P> 
    Overnight starter cultures of pSB1K3 (in DH5alpha) were subjected to plasmid isolation following the Plasmid Isolation Protocol. Final elution was done with 30μL of pre-warmed MilliQ.
</P><DL><DT>Date</DT><DD>16/8/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>Nanodrop DNA Quantification</H4><P> 
    The following isolates were quantified according to the Nanodrop DNA Quantification protocol. The concentration and purity of the DNA (260/280 and 260/230 ratios) were verified to be around 1.8~1.9. 
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Isolate</TH><TH class="tableheaderadpbl">Concentration (ng/uL)</TH></TR><TR class="tableunevenadpbl"><TD>pSB1K3</TD><TD>37</TD></TR></TBODY></TABLE></CENTER><DL><DT>Date</DT><DD>16/8/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>Solid Medium Preparation</H4><P>
Solid Luria Broth (LB) medium was prepared, complemented with Kanamycin (Kan), Chloramphenicol (Cam), IPTG and X-gal, following the following steps:
    <OL><LI>Warm 250mL LB agar.</LI><LI>Mix 50mg X-gal in 2.5mL DMSO.</LI><LI>Add X-gal solution to LB agar. </LI><LI>Add 2.5mL 100mM IPTG to LB agar.</LI><LI>Add 250µL Kan (1000x stock solution) &amp; 250µL Cam (1000x stock solution).</LI><LI>Pour plates: cool down while protecting from light.</LI><LI>store at 4°C, protected from light. </LI></OL></P><DL><DT>Date</DT><DD>16/8/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>Liquid Starter Culture Preparation</H4><P>
    Strain BL21DE3 used for inoculation of 10mL liquid starter culture (Luria Broth, <B>not complemented with antibiotics</B>) following the Liquid Starter Culture Protocol.
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#FridayVVA3">Friday, 17/08/2018</BUTTON><DIV id="FridayVVA3" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>17/8/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>Genomic DNA Isolation</H4><P>Starter culture(s) of BL21DE3 were subjected to genomic DNA isolation, following the <A href="https://2018.igem.org/Team:TUDelft/Experiments"> genomic DNA isolation protocol.</A></P> 
 Vigorous shaking (step 5) was done for 30 minutes, instead of 10 minutes.
</DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV></DIV></UL></FORM><FORM><LABEL for="WeekVVA4" class="subdropdown cadpbl">Week 4 20/08/2018 - 26/08/2018</LABEL><UL class="submenu"><DIV class="subcontent"><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#MondayVVA4">Monday, 20/08/2018</BUTTON><DIV id="MondayVVA4" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>20/8/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>Solid Medium Preparation</H4><P>
Solid Luria Broth (LB) medium was prepared, complemented with Cam. 40 plates were poured and stored at 4°C.
</P><DL><DT>Date</DT><DD>20/8/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4> Transformation of chemically competent cells </H4><P>The following transformations were carried out according to the protocol for transformation of chemically competent cells:
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Strain</TH><TH class="tableheaderadpbl">pACYC-dxCas9Tn5<I>lacZ</I>gRNA</TH><TH class="tableheaderadpbl">ME flanked KanR</TH><TH class="tableheaderadpbl">Medium on which was plated</TH></TR><TR class="tableunevenadpbl"><TD><I>E. coli </I>BL21DE3</TD><TD>4µl (~120ng)</TD><TD>10µl (~2µg)</TD><TD><UL><LI>LB+Cam+Kan+IPTG+X-gal</LI><LI>LB+Cam</LI></UL></TD></TR><TR class="tableunevenadpbl"><TD><I>E. coli </I>BL21DE3</TD><TD>4µl (~120ng)</TD><TD>-</TD><TD><UL><LI>LB+Cam+Kan+IPTG+X-gal</LI><LI>LB+Cam</LI></UL></TD></TR></TBODY></TABLE></CENTER><P>200 µL LB medium was added as recovery medium.
</P><DL><DT>Date</DT><DD>20/8/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>Liquid Starter Culture Preparation</H4><P>
Strain DH5alpha+pACYC-dxCas9Tn5<I>lacZ</I>gRNA were used for inoculation of 10mL liquid starter culture (Luria Broth, complemented with Chloramphenicol) following the Liquid Starter Culture Protocol.
</P><DL><DT>Date</DT><DD>20/08/2018</DD><DT>Operator</DT><DD>Venda Mangkusaputra</DD></DL><H4>Restriction </H4><P>Restriction cloning was done to assemble EPO and sgRNA for LacZ into biobrick backbone. The following table shows an overview of which fragments were cut with the appropriate restriction enzymes.
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Biobrick backbone (uL)</TH><TH class="tableheaderadpbl">EPO (ul)</TH><TH class="tableheaderadpbl">sgRNA for LacZ(ul)</TH></TR><TR class="tableunevenadpbl"><TD>10x CutSmart buffer    </TD><TD>2</TD><TD>2</TD><TD>2</TD></TR><TR class="tableunevenadpbl"><TD>Fragment</TD><TD>9</TD><TD>6</TD><TD>6</TD></TR><TR class="tableunevenadpbl"><TD>EcoRI</TD><TD>0.4</TD><TD>0.4</TD><TD>0.4</TD></TR><TR class="tableunevenadpbl"><TD>PstI</TD><TD>0.4</TD><TD>0.4</TD><TD>0.4</TD></TR><TR class="tableunevenadpbl"><TD>MilliQ</TD><TD> 8.2</TD><TD>11.2</TD><TD>11.2</TD></TR></TBODY></TABLE></CENTER><P>The samples were incubated for 4 hours at 37 °C. DNA was isolated by following PCR clean up kit protocol.
        DNA was quantified by nanodrop. The following concentrations were achieved:</P><UL class="uladpbl"><LI>Biobrick backbone=40ng/ul</LI><LI>EPO=25ng/ul</LI><LI>sgRNA for LacZ=25ng/ul</LI></UL><DL><DT>Date</DT><DD>20/08/2018</DD><DT>Operator</DT><DD>Venda Mangkusaputra</DD></DL><H4>Ligation</H4><P>In order to assemble biobrick inserts into the biobrick backbone, ligation was performed. The following pipetting scheme was followed

</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">EPO (uL) </TH><TH class="tableheaderadpbl">sgRNA (uL) </TH></TR><TR class="tableunevenadpbl"><TD>Ligase Buffer (10x) </TD><TD>2</TD><TD>2</TD></TR><TR class="tableevenadpbl"><TD>T4 DNA Ligase</TD><TD>1</TD><TD>1</TD></TR><TR class="tableunevenadpbl"><TD>DNA vector (~100 ng) </TD><TD>4</TD><TD>4</TD></TR><TR class="tableunevenadpbl"><TD>DNA fragment (XXX ng) </TD><TD>13</TD><TD>13</TD></TR></TBODY></TABLE></CENTER><P>The samples were incubated for 4 hours at room temperature. </P><DL><DT>Date</DT><DD>20/08/2018</DD><DT>Operator</DT><DD>Venda Mangkusaputra</DD></DL><H4> Transformation of chemically competent cells </H4><P>The following transformations for EPO and sgRNA for LacZ biobrick were carried out according to the protocol for transformation of chemically competent cells:
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Strain</TH><TH class="tableheaderadpbl">Plasmid </TH><TH class="tableheaderadpbl">Cloning method</TH><TH class="tableheaderadpbl">Volume added (uL)</TH><TH class="tableheaderadpbl">Medium on which was plated</TH></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I>DH5α </TD><TD>EPO BB</TD><TD>Restriction-ligation</TD><TD>6µl</TD><TD>LB-cam</TD></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I>DH5α </TD><TD>sgRNA for LacZ BB</TD><TD>Restriction-ligation</TD><TD>6µl</TD><TD>LB-cam</TD></TR></TBODY></TABLE></CENTER><P>200 uL LB medium was added as recovery medium.
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#TuesdayVVA4">Tuesday, 21/08/2018</BUTTON><DIV id="TuesdayVVA4" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>21/8/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4> Transformation Results </H4><P> After carrying out the transformation protocol on 20/8/2018, the plates were left for overnight incubation. After ~14-18hr incubation, the following results were observed: </P><P>All colonies on X-gal plates were white, possibly suggesting that LacZ was disrupted and kanR is present in the cell. However, to exclude false positives, a crucial control is required: BL21DE3 on LB+X-Gal+IPTG, to reassure that the parental strain has intact <I>lacZ</I> in the first place.</P><DL><DT>Date</DT><DD>21/8/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>Plasmid Isolation</H4><P> 
    Overnight starter cultures of DH5alpha+pACYC-dxCas9Tn5<I>lacZ</I>gRNA were subjected to plasmid isolation following the Plasmid Isolation Protocol. Final elution was done with 30 μL of pre-warmed MilliQ.
</P><DL><DT>Date</DT><DD>21/8/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>Nanodrop DNA Quantification</H4><P> 
    The following isolates were quantified according to the Nanodrop DNA Quantification protocol. The concentration and purity of the DNA (260/280 and 260/230 ratios) were measured. 
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Isolate</TH><TH class="tableheaderadpbl">Concentration (ng/µL)</TH><TH class="tableheaderadpbl">260/280 ratio</TH><TH class="tableheaderadpbl">260/230 ratio</TH></TR><TR class="tableunevenadpbl"><TD>pACYC-dxCas9Tn5<I>lacZ</I>gRNA</TD><TD>159</TD><TD>1.85</TD><TD>1.94</TD></TR><TR class="tableevenadpbl"><TD>pACYC-dxCas9Tn5<I>lacZ</I>gRNA</TD><TD>179</TD><TD>1.84</TD><TD>1.75</TD></TR><TR class="tableunevenadpbl"><TD>pACYC-dxCas9Tn5<I>lacZ</I>gRNA</TD><TD>152</TD><TD>1.84</TD><TD>1.86</TD></TR></TBODY></TABLE></CENTER><DL><DT>Date</DT><DD>21/8/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4> Transformation of chemically competent cells </H4><P>The following transformations were carried out according to the protocol for transformation of chemically competent cells:
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Strain</TH><TH class="tableheaderadpbl">Plasmid &amp; size (bp) </TH><TH class="tableheaderadpbl">Volume added (uL)</TH><TH class="tableheaderadpbl">Medium on which was plated</TH></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I>BL21DE3</TD><TD>pACYCdxCas9Tn5 + pSB1K3</TD><TD>4 each</TD><TD>LB+Cam+Kan+X-gal+IPTG</TD></TR></TBODY></TABLE></CENTER><P>200 µL LB medium was added as recovery medium.
</P><DL><DT>Date</DT><DD>21/8/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>Liquid Starter Culture Preparation</H4><P>
Four colonies from the plate from 20/8/2018 were used for inoculation of 5mL liquid starter culture (Luria Broth, complemented with chlorampenicol and kanamycin) following the Liquid Starter Culture Protocol.
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#WednesdayVVA4">Wednesday, 22/08/2018</BUTTON><DIV id="WednesdayVVA4" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>22/8/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4> Transformation Results </H4><P> After carrying out the transformation protocol on 21/8/2018, the plates were left for overnight incubation. After ~14-18hr incubation, the following results were observed: </P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Strain</TH><TH class="tableheaderadpbl">DNA </TH><TH class="tableheaderadpbl">Medium</TH><TH class="tableheaderadpbl">Result</TH></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I> BL21DE3  </TD><TD>pACYCdxCas9Tn5 + pSB1K3  </TD><TD>LB+Cam+Kan+X-gal+IPTG  </TD><TD>11 white colonies</TD></TR></TBODY></TABLE></CENTER><P>pSB1K3 is supposed to give a red colony phenotype. The pSB1K3 is suspected to be contaminated with other kanamycin (an)other resistant plasmid(s)</P><DL><DT>Date</DT><DD>22/8/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>Genomic DNA Isolation</H4><P>Starter culture(s) of inoculated colonies in LB+Cam+Kan were subjected to genomic DNA isolation, following the <A href="https://2018.igem.org/Team:TUDelft/Experiments"> genomic DNA isolation protocol.</A></P><DL><DT>Date</DT><DD>22/8/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>Nanodrop DNA Quantification</H4><P> 
    The following isolates were quantified according to the Nanodrop DNA Quantification protocol. The concentration and purity of the DNA (260/280 and 260/230 ratios) were measured. 
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Isolate</TH><TH class="tableheaderadpbl">Concentration (ng/uL)</TH><TH class="tableheaderadpbl">260/280 ratio</TH><TH class="tableheaderadpbl">260/230 ratio</TH></TR><TR class="tableunevenadpbl"><TD>colony 1</TD><TD>39</TD><TD>1.9</TD><TD>1.9</TD></TR><TR class="tableevenadpbl"><TD>colony 2</TD><TD>46</TD><TD>1.9</TD><TD>1.3</TD></TR><TR class="tableunevenadpbl"><TD>colony 3</TD><TD>40</TD><TD>1.9</TD><TD>1.2</TD></TR><TR class="tableevenadpbl"><TD>colony 4</TD><TD>52</TD><TD>1.8</TD><TD>1.5</TD></TR></TBODY></TABLE></CENTER><DL><DT>Date</DT><DD>22/8/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>Primer working stock preparation</H4><P> The following primers were resuspended in sterile Milli-Q water as directed by the manufaturer, in order to make 100µM stock solutions.  
</P><P> The primers in these 100 µM stocks were heated to 60 °C for 20 minutes and centrifuged  at maximum speed (~17,000 x g) for 2 minutes. 10µM primer working stock solutions were prepared by making 10x dilution aliquots with sterile Milli-Q, each in a total volume of 100µl. </P><DL><DT>Date</DT><DD>22/8/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>Solid Medium Preparation</H4><P>
The following solid media were prepared:
    <UL><LI>Luria Broth (LB) + X-gal + IPTG</LI><LI>Luria Broth (LB) + X-gal + IPTG + Cam</LI><LI>Luria Broth (LB) + X-gal + IPTG + Cam + Kan</LI></UL>
Final concentrations in these media are:
     <UL><LI>0.1µmol IPTG per mL medium</LI><LI>40 µg X-gal per mL medium</LI></UL></P><DL><DT>Date</DT><DD>22/8/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>Colony PCR</H4><P>Colony PCR was done to verify integration of KanR after transformation from 20/8/2018. This can be checked with primers IV007 &amp; IV008.
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH></TR><TR class="tableunevenadpbl"><TD>Template DNA</TD><TD>Isolated gDNA</TD></TR><TR class="tableevenadpbl"><TD>Forward primer</TD><TD>IV007</TD></TR><TR class="tableunevenadpbl"><TD>Reverse primer</TD><TD>IV008</TD></TR><TR class="tableunevenadpbl"><TD>Expected amplicon size (bp)</TD><TD>5680*</TD></TR></TBODY></TABLE></CENTER><P><I>*5680bp only amplifies if the KanR integrated in the lacZ locus. If the locus was not disrupted, 4818bp amplicons are expected. as a positive control, BL21DE3 gDNA was used as template.</I></P><P>5 tubes were prepared for colony PCR, containing isolated gDNA as template DNA. The thermocycler programme was installed to have an extension time of 6.5 minutes and an annealing temperature of 60°C. After the programme was finished, samples were kept at 4°C.</P><DL><DT>Date</DT><DD>22/08/2018</DD><DT>Operator</DT><DD>Venda Mangkusaputra</DD></DL><H4>Colony PCR</H4><P>Colony PCR was done to verify the success of EPO and sgRNA for LacZ biobrick. 3 different colonies from each plate were subjected to colony PCR. Two different primer sets were used. Biobrick primers 031 and 032 were used to verify the size of the insert. Primers 038 and 039 were used to verify the presence of EPO fragment within the biobrick backbone. Primers IV003 and IV004 were used to verify the presence of sgRNA for LacZ fragment within the biobrick backbone. 
    
</P><P>The PCR components and program can be found on the colony PCR protocol page. The PCR products were run on 2% TAE agarose gel shown on Figure 1. Upper lanes of Figure 1 shows amplification of insert with primers that anneal to the biobrick backbone flanking the insert (031 and 032). Lower lanes on Figure 1 shows amplification of insert with internal primers. All and all, the results confirmed insertion of the correct insert length.  </P><B>Figure 2% TAE Agarose gel of colony PCR</B>.<I> Lanes are (1-3) colony 1-3 harboring EPO Biobrick (BB) amplified by biobrick primers 031 and 032, (4-6) colony 1-3 harboring sgRNA DNA template for LacZ Biobrick (BB) amplified by biobrick primers 031 and 032, (7) DNA ladder, (8-10) colony 1-3 harboring EPO Biobrick (BB) amplified by internal primers 038 and 039, (11-13) colony 1-3 harboring sgRNA DNA template for LacZ Biobrick (BB) amplified by internal primers IV003 and IV004, (14) DNA ladder.</I></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#ThursdayVVA4">Thursday, 23/08/2018</BUTTON><DIV id="ThursdayVVA4" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>23/8/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>DNA Gel Electrophoresis</H4><P>gDNA PCR samples from 22/8/2018 were run on a 0.8% agarose gel in TAE buffer. 5μL was pipetted into each lane. The gel was imaged using a gel documentation system with UV-light.</P><P>This gel shows various bands, but no integration (5680bp) nor intact <I>lacZ</I> (4818bp). This suggests the parental strain is contaminated with a <I>lacZ</I>-deficient strain. This goes along with the absence of any expected blue colony phenotype from the transformation done on 20/8/2018.</P><DL><DT>Date</DT><DD>23/8/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>Plasmid Isolation</H4><P> 
    Overnight starter cultures of 2 colonies with BioBrick (pSB1C3-<I>lacZ</I>gRNA) were subjected to plasmid isolation following the Plasmid Isolation Protocol. Final elution was done with 30 μL of pre-warmed MilliQ.
</P><DL><DT>Date</DT><DD>23/8/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>Nanodrop DNA Quantification</H4><P> 
    The following isolates were quantified according to the Nanodrop DNA Quantification protocol. The concentration and purity of the DNA (260/280 and 260/230 ratios) were measured. 
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Isolate</TH><TH class="tableheaderadpbl">Concentration (ng/uL)</TH><TH class="tableheaderadpbl">260/280 ratio</TH><TH class="tableheaderadpbl">260/230 ratio</TH></TR><TR class="tableunevenadpbl"><TD>colony 1 (<I>lacZ</I>gRNa)</TD><TD>17</TD><TD>1.63</TD><TD>0.95</TD></TR><TR class="tableevenadpbl"><TD>colony 2 (<I>lacZ</I>gRNa)</TD><TD>23</TD><TD>1.63</TD><TD>0.88</TD></TR></TBODY></TABLE></CENTER><DL><DT>Date</DT><DD>23/8/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>Sequence Verification </H4><H5>Sequence from plasmid</H5><P> For verification of <I>lacZ</I> in pSB1C3, primers VF2 and VR were used for sequencing. The sequencing starts from the biobrick backbone in the direction of the insert. The 10µL sequencing sample was prepared in a 1.5 mL tube as follows: 

</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL)</TH><TH class="tableheaderadpbl">Final concentration</TH></TR><TR class="tableunevenadpbl"><TD>Plasmid DNA (500ng)</TD><TD>7.5</TD><TD>&gt;50 ng/uL</TD></TR><TR class="tableunevenadpbl"><TD>10uM primer</TD><TD>2.5</TD><TD>25 pmol</TD></TR></TBODY></TABLE></CENTER><P>Within 7 days, the sequence was retrievable online and alignment of colony 1 with the <I>in silico</I> designed molecule showed 100% match of the 1050 bp downstream of the sequencing primer binding site. </P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#FridayVVA4">Friday, 24/08/2018</BUTTON><DIV id="FridayVVA4" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>24/8/2018</DD><DT>Operator</DT><DD>Kavish Kohabir &amp; Alexander</DD></DL><H4>Chemically Competent Cells Preparation
</H4><P>Chemically competent cells were made of <I>E. coli</I> DH5alpha and <I>E. coli</I> BL21DE3, following the <A href="https://2018.igem.org/Team:TUDelft/Experiments#chemcomptcellsprep-scroll">Chemically Competent Cells Preparation protocol</A>.</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV></DIV></UL></FORM><FORM><LABEL for="WeekVVA5" class="subdropdown cadpbl">Week 5 27/08/2018 - 02/09/2018</LABEL><UL class="submenu"><DIV class="subcontent"><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#MondayVVA5">Monday, 27/08/2018</BUTTON><DIV id="MondayVVA5" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>27/8/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>High Fidelity PCR</H4><P> To amplify the KanR cassette* (~1kb), high fidelity PCR was done using pSB1K3 as template DNA and IV009 and IV010 as forward and reverse primers respectively. 8 50µL PCR mixtures were prepared, according to the High Fidelity PCR protocol.
    <I>NOTE: previously (10/8/2018), the KanR <B>gene</B> was used for amplification. This is not the same as the <B>whole cassette</B>, which includes the promoter and terminator.</I>
Primer combination(s) and expected fragment(s)

</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Sample</TH><TH class="tableheaderadpbl">Fw primer</TH><TH class="tableheaderadpbl">Rv primer</TH><TH class="tableheaderadpbl">expected size (bp) </TH></TR><TR class="tableunevenadpbl"><TD>ME-flanked KanR cassette (MEpKantME)</TD><TD>IV009</TD><TD>IV010</TD><TD>1060</TD></TR><TR class="tableevenadpbl"><TD>KanR cassette (pKant)</TD><TD>IV011</TD><TD>IV012</TD><TD>1014</TD></TR></TBODY></TABLE></CENTER><P>The thermocycler programme was installed to have an extension time of 45sec and an annealing temperature of 60 °C. After the programme was finished, samples were kept at 4 °C.</P><DL><DT>Date</DT><DD>27/8/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>DNA Gel Electrophoresis</H4><P>MEpKantME and pKant PCR samples were run on a 0.8% agarose gel in TAE buffer. 5μL was pipetted into each lane. The gel was imaged using a gel documentation system with UV-light.
    The gel was completely empty, only the ladder was visible. It is not clear what exactly went wrong. Perhaps make more offers to the PCR gods?</P><DL><DT>Date</DT><DD>27/8/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4> Testing new stock of BL21DE3 cells </H4><P>Previously (23/8/2018), it was observed that the BL21DE3 strain stock might be contaminated with other Kanamycin resistant cells. The following inoculations were carried out with both that old stock, and a fresh new stock, obtained from dr. Theo van Laar.
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">BL21DE3 stock</TH><TH class="tableheaderadpbl">Medium</TH></TR><TR class="tableunevenadpbl"><TD>old</TD><TD>LB+Cam</TD></TR><TR class="tableevenadpbl"><TD>new</TD><TD>LB+Cam</TD></TR><TR class="tableunevenadpbl"><TD>old</TD><TD>LB+Kan</TD></TR><TR class="tableevenadpbl"><TD>new</TD><TD>LB+Kan</TD></TR><TR class="tableunevenadpbl"><TD>old</TD><TD>LB+IPTG+X-gal*</TD></TR><TR class="tableevenadpbl"><TD>new</TD><TD>LB+IPTG+X-gal*</TD></TR></TBODY></TABLE></CENTER><I>*LB+IPT+X-gal plates from 22/8/2018 were used.</I></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#TuesdayVVA5">Tuesday, 28/08/2018</BUTTON><DIV id="TuesdayVVA5" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>28/8/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>High Fidelity PCR</H4><P> To amplify the KanR cassette* (~1kb), high fidelity PCR was done using pSB1K3 as template DNA and IV009 and IV010 as forward and reverse primers respectively. 8 50µL PCR mixtures were prepared, according to the High Fidelity PCR protocol.
    <I>NOTE: previously (10/8/2018), the KanR <B>gene</B> was used for amplification. This is not the same as the <B>whole cassette</B>, which includes the promoter and terminator.</I>
Primer combination(s) and expected fragment(s)

</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Sample</TH><TH class="tableheaderadpbl">Fw primer</TH><TH class="tableheaderadpbl">Rv primer</TH><TH class="tableheaderadpbl">expected size (bp) </TH><TH class="tableheaderadpbl">extension time </TH></TR><TR class="tableunevenadpbl"><TD>ME-flanked KanR cassette (MEpKantME)</TD><TD>IV009</TD><TD>IV010</TD><TD>1060</TD><TD>40sec</TD></TR><TR class="tableevenadpbl"><TD>KanR cassette (pKant)</TD><TD>IV011</TD><TD>IV012</TD><TD>1014</TD><TD>40sec</TD></TR><TR class="tableunevenadpbl"><TD>gDNA PCR</TD><TD>IV007</TD><TD>IV008</TD><TD>5680*</TD><TD>6.5min</TD></TR></TBODY></TABLE></CENTER>     
 *5680bp should appear only in case of KanR integration in the <I>lacZ</I> locus. Otherwise, this should result in 4818bp bands (intact <I>lacZ</I>).  
<P>The thermocycler programme was installed to have an annealing temperature of 65 °C. After the programme was finished, samples were kept at 4 °C.</P><DL><DT>Date</DT><DD>28/8/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4> Inoculation Results </H4><P> After inoculating BL21DE3 controls on solid medium, the plates were left for overnight incubation. After ~14-18hr incubation, the following results were observed: </P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">BL21DE3 stock</TH><TH class="tableheaderadpbl">Medium</TH><TH class="tableheaderadpbl">Result</TH></TR><TR class="tableunevenadpbl"><TD>old</TD><TD>LB+Cam</TD><TD>white colony growth</TD></TR><TR class="tableevenadpbl"><TD>new</TD><TD>LB+Cam</TD><TD>no colony growth</TD></TR><TR class="tableunevenadpbl"><TD>old</TD><TD>LB+Kan</TD><TD>white colony growth</TD></TR><TR class="tableevenadpbl"><TD>new</TD><TD>LB+Kan</TD><TD>no colony growth</TD></TR><TR class="tableunevenadpbl"><TD>old</TD><TD>LB+IPTG+X-gal*</TD><TD>blue colony growth</TD></TR><TR class="tableevenadpbl"><TD>new</TD><TD>LB+IPTG+X-gal*</TD><TD>blue colony growth</TD></TR></TBODY></TABLE></CENTER><I>*LB+IPT+X-gal plates from 22/8/2018 were used.</I><P>Based on these results, it was concluded that the old BL21DE3 batch already was contaminated with kanamycin and chloramphenicol resistance. The new batch does not show this, and therefore meets the requirements to continue <I>in vivo</I> screen work. Also, the plates poured on 22/8/2018 show good viability and chromogenic conversion of X-gal.</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#WednesdayVVA5">Wednesday, 29/08/2018</BUTTON><DIV id="WednesdayVVA5" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>29/8/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>DNA Gel Electrophoresis</H4><P>PCR samples from 28/8/2018 were run on a 0.8% agarose gel in TAE buffer. 5 μL was pipetted into each lane. The gel was imaged using a gel documentation system with UV-light.
    Again, the obtained gel was completely empty, only showing the reference ladders. It is not sure why this failed, but could have to do with too high annealing temperature. A gradient PCR could sort this out.</P><DL><DT>Date</DT><DD>29/8/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4> Transformation of chemically competent cells </H4><P>The following transformations were carried out according to the protocol for transformation of chemically competent cells:
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Strain</TH><TH class="tableheaderadpbl">Plasmid &amp; size (bp) </TH><TH class="tableheaderadpbl">Volume added (uL)</TH><TH class="tableheaderadpbl">Medium on which was plated</TH></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I>BL21DE3</TD><TD>pACYC-dxCas9Tn5<I>lacZ</I>gRNA (9622bp)</TD><TD>1µl (~150ng)</TD><TD>LB+Cam</TD></TR></TBODY></TABLE></CENTER><P>200 µL LB medium was added as recovery medium.
</P><DL><DT>Date</DT><DD>29/8/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>High Fidelity PCR</H4><P> To amplify the KanR cassette* (~1kb), high fidelity PCR was done using pSB1K3 as template DNA and IV009 and IV010 as forward and reverse primers respectively. Also, another attempt with IV007 and IV008 on gDNA was done to get a positive control band at 4818bp. 25µL PCR mixtures were prepared, according to the High Fidelity PCR protocol.
    </P><P>All samples contained 1µL of DMSO. The thermocycler programme was installed to have an extension time of 1 minute/kb amplicon and an annealing temperature ranging from 55 to 67°C (55; 56.1; 57.8; 59.8; 62.5; 64.7; 66.2; 67). After the programme was finished, samples were kept at 4 °C.</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#ThursdayVVA5">Thursday, 30/08/2018</BUTTON><DIV id="ThursdayVVA5" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>30/8/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>DNA Gel Electrophoresis</H4><P>PCR samples from 29/8/2018 and 30/8/2018 were run on a 0.8% agarose gel in TAE buffer. 5μL was pipetted into each lane. The gel was imaged using a gel documentation system with UV-light.</P><DL><DT>Date</DT><DD>30/8/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>Colony PCR</H4><P>Colony PCR was done to verify propagation of pACYC-dxCas9Tn5<I>lacZ</I>gRNA after transformation from 29/8/2018. This plasmid contains <I>lacZ</I> gRNA, which can be checked with primers 018 &amp; IV004.
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL)</TH></TR><TR class="tableunevenadpbl"><TD>Template DNA</TD><TD>picked colony</TD></TR><TR class="tableevenadpbl"><TD>Forward primer</TD><TD>018</TD></TR><TR class="tableunevenadpbl"><TD>Reverse primer</TD><TD>IV004</TD></TR><TR class="tableunevenadpbl"><TD>Expected amplicon size (bp)</TD><TD>2148</TD></TR></TBODY></TABLE></CENTER><P>10 tubes were prepared for colony PCR, containing picked colony as template DNA. 1 positive control (already isolated pACYC-dxCas9Tn5<I>lacZ</I>gRNA) was included. The thermocycler programme was installed to have an extension time of 2.5min and an annealing temperature of 60°C. After the programme was finished, samples were kept at 4°C.</P><DL><DT>Date</DT><DD>30/8/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>DNA Gel Electrophoresis</H4><P>Colony PCR samples was run on a 0.8% agarose gel in TAE buffer. 5μL was pipetted into each lane. The gel was imaged using a gel documentation system with UV-light.</P><P>This gel all tested colonies contain the <I>lacZ</I> gRNA next to Tn5 and dxCas9 (amplicon = 2148bp). it was decided to subject colony 1 to further analysis.</P><DL><DT>Date</DT><DD>30/8/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>Liquid Starter Culture Preparation</H4><P>
Colony 1 (BL21DE3+pACYC-dxCas9Tn5<I>lacZ</I>gRNA) of the plate from 29/8/2018 was used for inoculation of 60mL liquid starter culture (Luria Broth, complemented with chloramphenicol) following the Liquid Starter Culture Protocol.
</P><DL><DT>Date</DT><DD>30/8/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>Cell banking</H4><P>Strain <I>E. coli</I> DH5alpha pSB1C3-<I>lacZ</I>gRNA (colony 1) was stocked and stored at -80<SUP>o</SUP>C following the cell banking protocol.
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#FridayVVA5">Friday, 31/08/2018</BUTTON><DIV id="FridayVVA5" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>31/8/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>High Fidelity Genomic PCR</H4><P> To amplify the <I>lacZ</I> region (~4.8kb; no integration), high fidelity PCR was done using isolated BL21DE3 gDNA as template DNA and IV007 and IV008 as forward and reverse primers respectively.  25µL PCR mixtures were prepared, according to the High Fidelity PCR protocol.
    </P><P>The thermocycler programme was installed to have an extension time of 3min and an annealing temperature range from 60°C to 65°C. After the programme was finished, samples were kept at 4 °C.</P><DL><DT>Date</DT><DD>31/8/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>Cell banking</H4><P>Strain(s) BL21DE3+pACYCdxCas9Tn5<I>lacZ</I>gRNA was stocked and stored at -80<SUP>o</SUP>C following the cell banking protocol.
</P><DL><DT>Date</DT><DD>31/8/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>Electrocompetent Cell Preparation</H4><P>60mL of <I>E. coli</I> BL21DE3+pACYCdxCas9Tn5<I>lacZ</I>gRNA was used for making electrocompetent cell preparation protocol. Washing steps were only done with cold, sterile MilliQ. The final resuspension was done with 50% glycerol.
</P><DL><DT>Date</DT><DD>31/8/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>Genomic Colony PCR</H4><P>Genomic colony PCR was done as an approach to verify whether primers IV007 &amp; IV008 give a ~4.8kb amplicon. 
</P><P>8 tubes were prepared for colony PCR, containing supernatant of boiled picked blue colonies (24/8/2018) as template DNA. The thermocycler programme was installed to have an extension time of 6min and an annealing temperature of 60 °C. After the programme was finished, samples were kept at 4°C.</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#SundayVVA5">Sunday, 02/09/2018</BUTTON><DIV id="SundayVVA5" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>2/9/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>DNA Gel Electrophoresis</H4><P>PCR samples from 31/8/2018 were run on a 0.8% agarose gel in TAE buffer. 5μL was pipetted into each lane. The gel was imaged using a gel documentation system with UV-light.</P><P>This gel shows that colony PCR from boiled colony as template does not work, not even for 1 of the 8 screened colonies. However, when using gDNA as template, the expected 4.8kb amplicon shows for annewaling temperatures lower than 63°C. From 61°C onwards, there is also another amplicon, as byproduct.</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV></DIV></UL></FORM><H3 class="adpbl">SEPTEMBER</H3><FORM><LABEL for="WeekVVS1" class="subdropdown cadpbl">Week 1 03/09/2018 - 09/09/2018</LABEL><UL class="submenu"><DIV class="subcontent"><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#ThursdayVVS1">Thursday, 06/09/2018</BUTTON><DIV id="ThursdayVVS1" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>6/9/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>Electroporation</H4><P>The following transformations were carried out according to the protocol for electroporation:</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Strains</TH><TH class="tableheaderadpbl">Plasmid/fragment</TH><TH class="tableheaderadpbl">Plate medium</TH></TR><TR class="tableunevenadpbl"><TD><EM>E. coli</EM> BL21DE3 + pACYCdxCas9Tn5<I>lacZ</I>gRNA</TD><TD>pSB1K3</TD><TD>LB+Kan+Cam+X-gal+IPTG</TD></TR><TR class="tableunevenadpbl"><TD><EM>E. coli</EM> BL21DE3 + pACYCdxCas9Tn5<I>lacZ</I>gRNA</TD><TD>-</TD><TD>LB+Kan+Cam+X-gal+IPTG</TD></TR><TR class="tableunevenadpbl"><TD><EM>E. coli</EM> BL21DE3 + pACYCdxCas9Tn5<I>lacZ</I>gRNA</TD><TD>-</TD><TD>LB+Cam</TD></TR></TBODY></TABLE></CENTER><P>200 μl LB medium was added as recovery medium.
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#FridayVVS1">Friday, 07/09/2018</BUTTON><DIV id="FridayVVS1" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>7/9/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4> Transformation Results </H4><P> After carrying out the transformation protocol on 6/9/2018, the plates were left for overnight incubation. After ~14-18hr incubation, the following results were observed: </P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Strains</TH><TH class="tableheaderadpbl">Plasmid/fragment</TH><TH class="tableheaderadpbl">Plate medium</TH><TH class="tableheaderadpbl">Result</TH></TR><TR class="tableunevenadpbl"><TD><EM>E. coli</EM> BL21DE3 + pACYCdxCas9Tn5<I>lacZ</I>gRNA</TD><TD>pSB1K3</TD><TD>LB+Kan+Cam+X-gal+IPTG</TD><TD>no growth</TD></TR><TR class="tableunevenadpbl"><TD><EM>E. coli</EM> BL21DE3 + pACYCdxCas9Tn5<I>lacZ</I>gRNA</TD><TD>-</TD><TD>LB+Kan+Cam+X-gal+IPTG</TD><TD>no growth</TD></TR><TR class="tableunevenadpbl"><TD><EM>E. coli</EM> BL21DE3 + pACYCdxCas9Tn5<I>lacZ</I>gRNA</TD><TD>-</TD><TD>LB+Cam</TD><TD>white colony growth</TD></TR><TR class="tableunevenadpbl"><TD><EM>E. coli</EM> BL21DE3</TD><TD>-</TD><TD>LB+Cam</TD><TD>no growth</TD></TR></TBODY></TABLE></CENTER><P>The transformation results show that the bacth of competent cells indeed already contains the plasmid pACYCdxCas9Tn5<I>lacZ</I>gRNA. Transformation with the pSB1K3 plasmid does not seem to confer kanamycin resistance. Perhaps, something is wrong with the pSB1K3 isolate. </P><DL><DT>Date</DT><DD>7/9/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4> Transformation of chemically competent cells </H4><P>To determine the feasibility of <I>E. coli</I> BL21DE3 hosting 2 plasmids (pSB1K3 &amp; pACYC-derived), the following transformations were carried out according to the protocol for transformation of chemically competent cells:
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Strain</TH><TH class="tableheaderadpbl">Plasmid</TH><TH class="tableheaderadpbl">Volume added (uL)</TH><TH class="tableheaderadpbl">Medium</TH></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I>BL21DE3</TD><TD>pSB1K3</TD><TD>2</TD><TD>LB+Kan</TD></TR><TR class="tableevenadpbl"><TD><I>E.coli</I>BL21</TD><TD>pSB1K3+pACYC-DUET</TD><TD>2 each</TD><TD>LB+Kan+Cam</TD></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I>BL21DE3</TD><TD>-</TD><TD>-</TD><TD>LB+Kan</TD></TR><TR class="tableevenadpbl"><TD><I>E.coli</I>BL21</TD><TD>-</TD><TD>-</TD><TD>LB+Kan+Cam</TD></TR></TBODY></TABLE></CENTER><P>200 µL LB medium was added as recovery medium. 50µL was plated. 
</P><DL><DT>Date</DT><DD>7/9/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>High Fidelity PCR</H4><P> To amplify KanR cassette with (MEpKantME*) and without (pKant*) MEs (~1kb), high fidelity PCR was done using the following forward and reverse primers. 50µL PCR mixtures were prepared, according to the High Fidelity PCR protocol.
    <I>*pKant = promotor-Kan-terminator</I></P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Template</TH><TH class="tableheaderadpbl">Fw primer</TH><TH class="tableheaderadpbl">Rv primer</TH><TH class="tableheaderadpbl">expected fragment</TH><TH class="tableheaderadpbl">expected size (bp) </TH></TR><TR class="tableunevenadpbl"><TD>pSB1K3</TD><TD>IV013</TD><TD>IV014</TD><TD>MEpKantME</TD><TD>~1kb</TD></TR><TR class="tableevenadpbl"><TD>pSB1K3</TD><TD>IV015</TD><TD>IV016</TD><TD>pKant</TD><TD>~1kb</TD></TR></TBODY></TABLE></CENTER><P>The thermocycler programme was installed to have an extension time of 30sec and an annealing temperature of 60 °C. After the programme was finished, samples were kept at 4 °C.</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#SaturdayVVS1">Saturday, 08/09/2018</BUTTON><DIV id="SaturdayVVS1" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>8/9/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>Cell banking</H4><P>DH5alpha+pSB1K3 and DH5alpha+pSB1C3 were stocked and stored at -80<SUP>o</SUP>C following the cell banking protocol.
</P><DL><DT>Date</DT><DD>8/9/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>DNA Gel Electrophoresis</H4><P>PCR products for MEpKantME and pKant were run on a 0.8% agarose gel in TAE buffer. 5μL was pipetted into each lane. The gel was imaged using a gel documentation system with UV-light.</P><P>The gel shows multiple bands and indicates the plasmid sample may be contaminated with other DNA. This is the case, regardless of flanking with MEs or not.</P><DL><DT>Date</DT><DD>8/9/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>High Fidelity PCR</H4><P> To amplify KanR cassette with (MEpKantME*) and without (pKant*) MEs (~1kb), high fidelity PCR was done using the following forward and reverse primers. 50µL PCR mixtures were prepared, according to the High Fidelity PCR protocol. The same preparation was done as on 7/9/2018, but now with a gradient of annealing temperatures.
    
    
Primer combination(s) and expected fragment(s) are identical to as on 7/9/2018. Based on previous experiments, it suspected that the usual pSB1K3 plasmid sample is contaminated, so it was decided to work with a different vial (isolated in 2010, by iGEM Team TU Delft).
</P><P>The thermocycler programme was installed to have an extension time of 45sec and an annealing temperature range from 65°C to 72°C. After the programme was finished, samples were kept at 4 °C.</P><DL><DT>Date</DT><DD>8/9/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>DNA Gel Electrophoresis</H4><P>KanR cassette PCR fragments (with and without MEs) were run on a 0.8% agarose gel in TAE buffer. 5μL was pipetted into each lane. The gel was imaged using a gel documentation system with UV-light.</P><P>This gel shows that the initial template used indeed did not contain pur pSB1K3. The 2010 isolate was a useful alternative to this, showing that the problematics was due to template contamination.</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#SundayVVS1">Sunday, 09/09/2018</BUTTON><DIV id="SundayVVS1" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>9/9/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4> Transformation of chemically competent cells </H4><P>The following transformations were carried out according to the protocol for transformation of chemically competent cells:
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Strain</TH><TH class="tableheaderadpbl">Plasmid </TH><TH class="tableheaderadpbl">Volume added (µL)</TH><TH class="tableheaderadpbl">Medium on which was plated</TH></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I>DH5α</TD><TD>pSB1K3</TD><TD>1 (~350ng)</TD><TD>LB+Kan</TD></TR><TR class="tableevenadpbl"><TD><I>E.coli</I>DH5α</TD><TD>-</TD><TD>-</TD><TD>LB+Kan</TD></TR></TBODY></TABLE></CENTER><P>200 µL LB medium was added as recovery medium.
</P><DL><DT>Date</DT><DD>9/9/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4> Inoculation of <I>in vivo</I> assay controls </H4><P>The following transformations were carried out according to the protocol for transformation of chemically competent cells:
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Strain</TH><TH class="tableheaderadpbl">Plasmid </TH><TH class="tableheaderadpbl">Medium </TH></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I> BL21DE3</TD><TD>pACYCdxCas9Tn5<I>lacZ</I>gRNA</TD><TD>LB+Kan</TD></TR><TR class="tableevenadpbl"><TD><I>E.coli</I> BL21DE3</TD><TD>pACYCdxCas9Tn5<I>lacZ</I>gRNA + pSB1K3</TD><TD>LB+Kan</TD></TR></TBODY></TABLE></CENTER><P>200 µL LB medium was added as recovery medium.
</P><DL><DT>Date</DT><DD>9/9/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>High Fidelity PCR</H4><P> To amplify MEpKantME (~1kb, entails full expressional cassette for kanamycin resistance, flanked with MEs), high fidelity PCR was done using pSB1K3 as template DNA and IV013 and IV014 as forward and reverse primers respectively. PCR mixtures were prepared, according to the High Fidelity PCR protocol.

</P><P>The thermocycler programme was installed to have an extension time of 45sec and an annealing temperature of 65°C. After the programme was finished, samples were kept at 4 °C.</P><DL><DT>Date</DT><DD>9/9/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>DNA Gel Electrophoresis</H4><P>MEpKantME PCR samples were run on a 0.8% agarose gel in TAE buffer. 5μL
    was pipetted into each lane. The gel was imaged using a gel documentation system with UV-light.</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV></DIV></UL></FORM><FORM><LABEL for="WeekVVS2" class="subdropdown cadpbl">Week 2 10/09/2018 - 16/09/2018</LABEL><UL class="submenu"><DIV class="subcontent"><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#WednesdayVVS2">Wednesday, 12/09/2018</BUTTON><DIV id="WednesdayVVS2" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>12/9/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>Liquid Starter Culture Preparation</H4><P>
Strain DH5alpha+pSB1K3 (red phenotype)
    and BL21DE3+pACYCdxCas9Tn5<I>lacZ</I>gRNA were used for inoculation of 10mL liquid starter culture (Luria Broth, complemented with kanamycin for pSB1K3, and chloramphenicol for pACYC-derived vectors) following the Liquid Starter Culture Protocol.
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#ThursdayVVS2">Thursday, 13/09/2018</BUTTON><DIV id="ThursdayVVS2" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>13/9/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>Plasmid Isolation</H4><P> 
    Overnight starter cultures of DH5alpha + pSB1K3 were subjected to plasmid isolation following the Plasmid Isolation Protocol. Final elution was done with 30 μL of pre-warmed MilliQ.
</P><DL><DT>Date</DT><DD>13/9/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>Nanodrop DNA Quantification</H4><P> 
    The following isolates were quantified according to the Nanodrop DNA Quantification protocol. The concentration and purity of the DNA (260/280 and 260/230 ratios) were measured. 
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Isolate</TH><TH class="tableheaderadpbl">Concentration (ng/uL)</TH><TH class="tableheaderadpbl">260/280 ratio</TH><TH class="tableheaderadpbl">260/230 ratio</TH></TR><TR class="tableunevenadpbl"><TD>pSB1K3</TD><TD>125</TD><TD>1.9</TD><TD>2.0</TD></TR><TR class="tableevenadpbl"><TD>pSB1K3</TD><TD>130</TD><TD>1.9</TD><TD>2.1</TD></TR></TBODY></TABLE></CENTER><DL><DT>Date</DT><DD>13/9/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>High Fidelity PCR</H4><P> To amplify MEpKantME &amp; pKant (~1kb), high fidelity PCR was done using freshly isolated pSB1K3 as template DNA. PCR mixtures were prepared, according to the High Fidelity PCR protocol.
    
    
Primer combination(s) and expected fragment(s)

</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Sample</TH><TH class="tableheaderadpbl">Fw primer</TH><TH class="tableheaderadpbl">Rv primer</TH><TH class="tableheaderadpbl">Template</TH><TH class="tableheaderadpbl">expected size (bp) </TH></TR><TR class="tableunevenadpbl"><TD>MEpKantME</TD><TD>IV013</TD><TD>IV014</TD><TD>pSB1K3</TD><TD>~1kb</TD></TR><TR class="tableevenadpbl"><TD>pKant</TD><TD>IV015</TD><TD>IV016</TD><TD>pSB1K3</TD><TD>~1kb</TD></TR></TBODY></TABLE></CENTER><P>The thermocycler programme was installed to have an extension time of 45sec and an annealing temperature of 65°C. After the programme was finished, samples were kept at 4 °C.</P><DL><DT>Date</DT><DD>13/9/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4> Transformation of chemically competent cells </H4><P>The following transformations were carried out according to the protocol for transformation of chemically competent cells:
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Strain</TH><TH class="tableheaderadpbl">Plasmid &amp; size (bp) </TH><TH class="tableheaderadpbl">Medium</TH></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I>BL21DE3</TD><TD>pACYCdxCas9Tn5<I>lacZ</I>gRNA</TD><TD>LB+Cam</TD></TR><TR class="tableevenadpbl"><TD><I>E.coli</I>BL21DE3</TD><TD>pACYCTn5</TD><TD>LB+Cam</TD></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I>BL21DE3</TD><TD>pACYCdxCas9Tn5</TD><TD>LB+Cam</TD></TR></TBODY></TABLE></CENTER><P>200uL LB medium was added as recovery medium.
</P><DL><DT>Date</DT><DD>13/9/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>DNA Gel Electrophoresis</H4><P>PCR samples for kanamycin expression cassettes were run on a 0.8% agarose gel in TAE buffer. 5μL was pipetted into each lane. The gel was imaged using a gel documentation system with UV-light.</P><P>This gel shows that, finally, the kanamycin cassettes (each ~1000bp) are appropriately amplified, without byproducts. This means the PCR protocol &amp; template used on 13/9/2018 is correct.</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#FridayVVS2">Friday, 14/09/2018</BUTTON><DIV id="FridayVVS2" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>14/9/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>Colony PCR</H4><P>Colony PCR was done to verify propagation of pACYCTn5 and pACYCdxCas9Tn5 after transformation from 13/9/2018. These plasmids contain the T7 expression system, which can be checked with primers 029 &amp; 030.
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL)</TH></TR><TR class="tableunevenadpbl"><TD>Template DNA</TD><TD>picked colony</TD></TR><TR class="tableevenadpbl"><TD>Forward primer</TD><TD>029</TD></TR><TR class="tableunevenadpbl"><TD>Reverse primer</TD><TD>030</TD></TR><TR class="tableunevenadpbl"><TD>Expected amplicon size</TD><TD>pACYCTn5: ~1.5kb
      pACYCdxCas9Tn5: ~6.2kb</TD></TR></TBODY></TABLE></CENTER><P>Tubes were prepared for colony PCR, containing picked as template DNA. The thermocycler programme was installed to have an extension time of 6.5min and an annealing temperature of 60 °C. After the programme was finished, samples were kept at 4°C.</P><DL><DT>Date</DT><DD>14/9/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>DNA Gel Electrophoresis</H4><P>Colony PCR samples were run on a 0.8% agarose gel in TAE buffer. 5μL was pipetted into each lane. The gel was imaged using a gel documentation system with UV-light.</P><P>The gel shows that BL21DE3 was successfully transformed with pACYCdxCas9Tn5 (lane 2) and also with pACYCTn5 (lane 5). DH5alpha was not rendered with a pACYC-Gblock3&amp;4 insert (lane 6 &amp; 7), as the expected size would be 6kb.</P><DL><DT>Date</DT><DD>14/9/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>Liquid Starter Culture Preparation</H4><P>
    The following strains were used for inoculation of 30mL liquid starter cultures:</P><UL><LI>BL21DE3 + pACYCdxCas9Tn5<I>lacZ</I>gRNA</LI><LI>BL21DE3 + pACYCdxCas9Tn5</LI><LI>BL21DE3 + pAYCTn5</LI></UL><P>Preparation of the LB media (Luria Broth, complemented with cloramphenicol) was done following the Liquid Starter Culture Protocol.
        The cultures were grown until OD0.5, prior to making them electrocompetent.
</P><DL><DT>Date</DT><DD>14/9/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>Electrocompetent Cell Preparation</H4><P>The following strains were made electrocompetent: </P><UL><LI>BL21DE3 + pACYCdxCas9Tn5<I>lacZ</I>gRNA</LI><LI>BL21DE3 + pACYCdxCas9Tn5</LI><LI>BL21DE3 + pAYCTn5</LI></UL>.<P> Washing steps were only done with cold, sterile MilliQ. The final resuspension was done with 50% glycerol.
</P><DL><DT>Date</DT><DD>14/9/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>PCR clean-up</H4><P>
    MEpKantME &amp; pKant PCR products from 13/9/2018 were subjected to PCR Clean-Up, according to the protocol. Elution was done in 40 µL Milli-Q (2x20µl per column).
</P><DL><DT>Date</DT><DD>14/9/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>Nanodrop DNA Quantification</H4><P> 
    The following isolates were quantified according to the Nanodrop DNA Quantification protocol. The concentration and purity of the DNA (260/280 and 260/230 ratios) were measured. 
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Isolate</TH><TH class="tableheaderadpbl">Concentration (ng/uL)</TH><TH class="tableheaderadpbl">260/280 ratio</TH><TH class="tableheaderadpbl">260/230 ratio</TH></TR><TR class="tableunevenadpbl"><TD>MEpKantME</TD><TD>993</TD><TD>1.85</TD><TD>2.15</TD></TR><TR class="tableevenadpbl"><TD>MEpKant</TD><TD>1153</TD><TD>1.87</TD><TD>2.29</TD></TR><TR class="tableunevenadpbl"><TD>pKant</TD><TD>823</TD><TD>1.78</TD><TD>2.06</TD></TR></TBODY></TABLE></CENTER><DL><DT>Date</DT><DD>14/9/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>Solid Medium Preparation</H4><P>The following solid media were prepared:</P><UL><LI>Luria Broth (LB) + X-gal + IPTG</LI><LI>Luria Broth (LB) + X-gal + IPTG + Cam</LI><LI>Luria Broth (LB) + X-gal + IPTG + Cam + Kan</LI></UL><P>Final concentrations in these media are:
     <UL><LI>0.1µmol IPTG per mL medium</LI><LI>40 µg X-gal per mL medium</LI></UL></P><DL><DT>Date</DT><DD>14/9/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>DpnI Digestion</H4><P>To eliminate any residual template DNA, a DpnI digestion was performed with the following reaction mixture:
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL)</TH></TR><TR class="tableunevenadpbl"><TD>10x CutSmart buffer</TD><TD>5</TD></TR><TR class="tableevenadpbl"><TD>PCR product</TD><TD>40</TD></TR><TR class="tableunevenadpbl"><TD>Enzyme (DpnI)</TD><TD>1</TD></TR><TR class="tableunevenadpbl"><TD>MilliQ</TD><TD>4</TD></TR></TBODY></TABLE></CENTER><P>The reaction was incubated for 4 hours at 37 °C. 
Heat inactivation was performed for 30 minutes at 80 °C.</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#SaturdayVVS2">Saturday, 15/09/2018</BUTTON><DIV id="SaturdayVVS2" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>15/9/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>DNA Gel Electrophoresis</H4><P>Kanamycin PCR samples were run on a 0.8% agarose gel in TAE buffer. 1μL was pipetted into each lane. The gel was imaged using a gel documentation system with UV-light.</P><DL><DT>Date</DT><DD>15/9/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>DpnI Digest Clean-up</H4><P>
DpnI digested kanamycin PCR products were subjected to Clean-Up, according to the protocol. Elution was done in 40 µL Milli-Q.
</P><DL><DT>Date</DT><DD>15/9/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>Nanodrop DNA Quantification</H4><P> 
    The following isolates were quantified according to the Nanodrop DNA Quantification protocol. The concentration and purity of the DNA (260/280 and 260/230 ratios) were measured. 
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Isolate</TH><TH class="tableheaderadpbl">Concentration (ng/uL)</TH><TH class="tableheaderadpbl">260/280 ratio</TH><TH class="tableheaderadpbl">260/230 ratio</TH></TR><TR class="tableunevenadpbl"><TD>MEpKantME<SUB>DpnI</SUB></TD><TD>1537.3</TD><TD>1.86</TD><TD>2.29</TD></TR><TR class="tableevenadpbl"><TD>pKant<SUB>DpnI</SUB></TD><TD>735</TD><TD>1.82</TD><TD>2.18</TD></TR></TBODY></TABLE></CENTER><DL><DT>Date</DT><DD>15/9/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>Electroporation</H4><P>The following transformations were carried out according to the protocol for electroporation:</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Strain</TH><TH class="tableheaderadpbl">Plasmid/fragment size (bp)</TH><TH class="tableheaderadpbl">Medium*</TH></TR><TR class="tableunevenadpbl"><TD><EM>E. coli</EM> DH5alpha</TD><TD>pACYC-DUET</TD><TD>2</TD></TR><TR class="tableevenadpbl"><TD><EM>E. coli</EM> BL21DE3</TD><TD>-</TD><TD>4</TD></TR><TR class="tableunevenadpbl"><TD><EM>E. coli</EM> BL21DE3</TD><TD>-</TD><TD>5</TD></TR><TR class="tableevenadpbl"><TD><EM>E. coli</EM> BL21DE3</TD><TD>-</TD><TD>3</TD></TR><TR class="tableunevenadpbl"><TD><EM>E. coli</EM> BL21DE3 + pACYCdxCas9Tn5<I>lacZ</I>gRNA</TD><TD>-</TD><TD>1 &amp; 2</TD></TR><TR class="tableevenadpbl"><TD><EM>E. coli</EM> BL21DE3 + pACYCdxCas9Tn5<I>lacZ</I>gRNA</TD><TD>pKant<SUB>DpnI</SUB></TD><TD>1 &amp; 4</TD></TR><TR class="tableunevenadpbl"><TD><EM>E. coli</EM> BL21DE3 + pACYCdxCas9Tn5<I>lacZ</I>gRNA</TD><TD>MEpKantME<SUB>DpnI</SUB></TD><TD>1 &amp; 4</TD></TR><TR class="tableevenadpbl"><TD><EM>E. coli</EM> BL21DE3 + pACYCdxCas9Tn5<I>lacZ</I>gRNA</TD><TD>pSB1K3</TD><TD>1 &amp; 4</TD></TR><TR class="tableunevenadpbl"><TD><EM>E. coli</EM> BL21DE3 + pACYCTn5</TD><TD>-</TD><TD>1 &amp; 2</TD></TR><TR class="tableevenadpbl"><TD><EM>E. coli</EM> BL21DE3 + pACYCTn5</TD><TD>pKant<SUB>DpnI</SUB></TD><TD>1 &amp; 4</TD></TR><TR class="tableunevenadpbl"><TD><EM>E. coli</EM> BL21DE3 + pACYCTn5</TD><TD>MEpKantME<SUB>DpnI</SUB></TD><TD>1 &amp; 4</TD></TR><TR class="tableevenadpbl"><TD><EM>E. coli</EM> BL21DE3 + pACYCTn5</TD><TD>pSB1K3</TD><TD>1 &amp; 4</TD></TR></TBODY></TABLE></CENTER><I>*Some samples were plated on several media. The media are displayed with the corresponding numbers:</I><OL><LI>LB+Xgal+IPTG+Cam+Kan</LI><LI>LB+Xgal+IPTG+Cam</LI><LI>LB+Xgal+IPTG</LI><LI>LB+Kan</LI><LI>LB+Cam</LI></OL><P>Of every linear fragment used, 200ng was electroporated. Of every plasmid, 50ng was electroporated. 200 μl LB medium was added as recovery medium.
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#SundayVVS2">Sunday, 16/09/2018</BUTTON><DIV id="SundayVVS2" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>16/9/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4> Transformation Results </H4><P> After carrying out the transformation protocol on 14/9/2018, the plates were left for overnight incubation. After ~14-18hr incubation, the following results were observed: </P><I>*Some samples were plated on several media. The media are displayed with the corresponding numbers:</I><OL><LI>LB+Xgal+IPTG+Cam+Kan</LI><LI>LB+Xgal+IPTG+Cam</LI><LI>LB+Xgal+IPTG</LI><LI>LB+Kan</LI><LI>LB+Cam</LI></OL><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Strain</TH><TH class="tableheaderadpbl">Plasmid/fragment size (bp)</TH><TH class="tableheaderadpbl">Medium*</TH><TH class="tableheaderadpbl">Observation</TH></TR><TR class="tableunevenadpbl"><TD><EM>E. coli</EM> DH5alpha</TD><TD>pACYC-DUET</TD><TD>2</TD><TD>white phenotype</TD></TR><TR class="tableevenadpbl"><TD><EM>E. coli</EM> BL21DE3</TD><TD>-</TD><TD>4</TD><TD>no growth </TD></TR><TR class="tableunevenadpbl"><TD><EM>E. coli</EM> BL21DE3</TD><TD>-</TD><TD>5</TD><TD>no growth</TD></TR><TR class="tableevenadpbl"><TD><EM>E. coli</EM> BL21DE3</TD><TD>-</TD><TD>3</TD></TR><TR class="tableunevenadpbl"><TD><EM>E. coli</EM> BL21DE3 + pACYCdxCas9Tn5<I>lacZ</I>gRNA</TD><TD>-</TD><TD>1</TD></TR><TR class="tableevenadpbl"><TD><EM>E. coli</EM> BL21DE3 + pACYCdxCas9Tn5<I>lacZ</I>gRNA</TD><TD>-</TD><TD>2</TD></TR><TR class="tableunevenadpbl"><TD><EM>E. coli</EM> BL21DE3 + pACYCdxCas9Tn5<I>lacZ</I>gRNA</TD><TD>pKant<SUB>DpnI</SUB></TD><TD>1</TD><TD>no growth</TD></TR><TR class="tableevenadpbl"><TD><EM>E. coli</EM> BL21DE3 + pACYCdxCas9Tn5<I>lacZ</I>gRNA</TD><TD>pKant<SUB>DpnI</SUB></TD><TD>4</TD><TD>no growth</TD></TR><TR class="tableunevenadpbl"><TD><EM>E. coli</EM> BL21DE3 + pACYCdxCas9Tn5<I>lacZ</I>gRNA</TD><TD>MEpKantME<SUB>DpnI</SUB></TD><TD>1</TD><TD>no growth</TD></TR><TR class="tableevenadpbl"><TD><EM>E. coli</EM> BL21DE3 + pACYCdxCas9Tn5<I>lacZ</I>gRNA</TD><TD>MEpKantME<SUB>DpnI</SUB></TD><TD>4</TD><TD>no growth</TD></TR><TR class="tableunevenadpbl"><TD><EM>E. coli</EM> BL21DE3 + pACYCdxCas9Tn5<I>lacZ</I>gRNA</TD><TD>pSB1K3</TD><TD>1</TD></TR><TR class="tableevenadpbl"><TD><EM>E. coli</EM> BL21DE3 + pACYCdxCas9Tn5<I>lacZ</I>gRNA</TD><TD>pSB1K3</TD><TD>4</TD></TR><TR class="tableunevenadpbl"><TD><EM>E. coli</EM> BL21DE3 + pACYCTn5</TD><TD>-</TD><TD>1</TD></TR><TR class="tableevenadpbl"><TD><EM>E. coli</EM> BL21DE3 + pACYCTn5</TD><TD>-</TD><TD>2</TD></TR><TR class="tableunevenadpbl"><TD><EM>E. coli</EM> BL21DE3 + pACYCTn5</TD><TD>pKant<SUB>DpnI</SUB></TD><TD>1</TD><TD>no growth</TD></TR><TR class="tableevenadpbl"><TD><EM>E. coli</EM> BL21DE3 + pACYCTn5</TD><TD>pKant<SUB>DpnI</SUB></TD><TD>4</TD><TD>no growth</TD></TR><TR class="tableunevenadpbl"><TD><EM>E. coli</EM> BL21DE3 + pACYCTn5</TD><TD>MEpKantME<SUB>DpnI</SUB></TD><TD>1</TD><TD>no growth</TD></TR><TR class="tableevenadpbl"><TD><EM>E. coli</EM> BL21DE3 + pACYCTn5</TD><TD>MEpKantME<SUB>DpnI</SUB></TD><TD>4</TD><TD>no growth</TD></TR><TR class="tableunevenadpbl"><TD><EM>E. coli</EM> BL21DE3 + pACYCTn5</TD><TD>pSB1K3</TD><TD>1</TD></TR><TR class="tableevenadpbl"><TD><EM>E. coli</EM> BL21DE3 + pACYCTn5</TD><TD>pSB1K3</TD><TD>4</TD></TR></TBODY></TABLE></CENTER></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV></DIV></UL></FORM><FORM><LABEL for="WeekVVS3" class="subdropdown cadpbl">Week 3 17/09/2018 - 23/09/2018</LABEL><UL class="submenu"><DIV class="subcontent"><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#MondayVVS3">Monday, 17/09/2018</BUTTON><DIV id="MondayVVS3" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>17/9/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>Solid Medium Preparation</H4><P>
The following solid media were prepared:
    <UL><LI>Luria Broth (LB) + X-gal + IPTG</LI><LI>Luria Broth (LB) + X-gal + IPTG + Cam</LI><LI>Luria Broth (LB) + X-gal + IPTG + Cam + Kan</LI></UL>
Final concentrations in these media are:
     <UL><LI>0.1µmol IPTG per mL medium</LI><LI>40 µg X-gal per mL medium</LI></UL></P><DL><DT>Date</DT><DD>17/9/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>Liquid Starter Culture Preparation</H4><P>
A colony of BL21DE3+pACYCdxCas9Tn5<I>lacZ</I>gRNA was used for inoculation of 60mL liquid starter culture (Luria Broth, complemented with chloramphenicol) following the Liquid Starter Culture Protocol.
</P><P>
The idea was to inoculate a white phenotype colony (BL21DE3 + pACYCTn5 + MEpKantME, plated on LB+Xgal+IPTG+Cam+Kan, 15/9/2018). But the phenotype has changed over two days to form a red colony. <B>This was the first indication of what was going to become a hard time. Red colony phenotype formation is latent and can develop over 5 days.</B></P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#WednesdayVVS3">Wednesday, 19/09/2018</BUTTON><DIV id="WednesdayVVS3" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>19/9/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>Liquid Starter Culture Preparation</H4><P>
    The following strains were used for inoculation of 30mL liquid starter cultures:</P><UL><LI>BL21DE3 + pACYCdxCas9Tn5<I>lacZ</I>gRNA</LI></UL><P>Preparation of the LB media (Luria Broth, complemented with cloramphenicol) was done following the Liquid Starter Culture Protocol.
        The cultures were grown until OD0.5, prior to making them electrocompetent. The starter culture included IPTG induction from OD0.2 onwards.
</P><DL><DT>Date</DT><DD>19/9/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>Electrocompetent Cell Preparation</H4><P>The following strains were made electrocompetent: </P><UL><LI>BL21DE3 + pACYCdxCas9Tn5<I>lacZ</I>gRNA</LI><LI>BL21DE3 + pACYCdxCas9Tn5</LI><LI>BL21DE3 + pAYCTn5</LI></UL>.<P> Washing steps were only done with cold, sterile MilliQ. The final resuspension was done with 50% glycerol.
</P><DL><DT>Date</DT><DD>19/9/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>Electroporation</H4><P>As a second attempt, with 800ng-1µg DNA instead of 200ng (compared to 15/9/2018), the following transformations were carried out according to the protocol for electroporation. <I>It was assumed that some of the controls are not necessesary to repeat, since they have repeatedly shown to be consistent.</I></P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Strain</TH><TH class="tableheaderadpbl">Plasmid/fragment size (bp)</TH><TH class="tableheaderadpbl">Medium*</TH></TR><TR class="tableunevenadpbl"><TD><EM>E. coli</EM> BL21DE3 + pACYCdxCas9Tn5<I>lacZ</I>gRNA</TD><TD>-</TD><TD>1 &amp; 2</TD></TR><TR class="tableevenadpbl"><TD><EM>E. coli</EM> BL21DE3 + pACYCdxCas9Tn5<I>lacZ</I>gRNA</TD><TD>pKant<SUB>DpnI</SUB></TD><TD>1, 2 &amp; 4</TD></TR><TR class="tableunevenadpbl"><TD><EM>E. coli</EM> BL21DE3 + pACYCdxCas9Tn5<I>lacZ</I>gRNA</TD><TD>MEpKantME<SUB>DpnI</SUB></TD><TD>1, 2 &amp; 4</TD></TR><TR class="tableevenadpbl"><TD><EM>E. coli</EM> BL21DE3 + pACYCdxCas9Tn5<I>lacZ</I>gRNA</TD><TD>pSB1K3</TD><TD>1, 2 &amp; 4</TD></TR></TBODY></TABLE></CENTER><I>*Some samples were plated on several media. The media are displayed with the corresponding numbers:</I><OL><LI>LB+Xgal+IPTG+Cam+Kan</LI><LI>LB+Xgal+IPTG+Cam</LI><LI>LB+Xgal+IPTG</LI><LI>LB+Kan</LI><LI>LB+Cam</LI></OL><P>Of every linear fragment used, 200ng was electroporated. Of every plasmid, 50ng was electroporated. 200 μl LB medium was added as recovery medium with 0.1mM IPTG.
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#ThursdayVVS3">Thursday, 20/09/2018</BUTTON><DIV id="ThursdayVVS3" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>20/9/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4> Transformation Results </H4><P> After carrying out the transformation protocol on 19/9/2018, the plates were left for overnight incubation. After ~14-18hr incubation, the following results were observed.  <I>*Some samples were plated on several media. The media are displayed with the corresponding numbers:</I></P><OL><LI>LB+Xgal+IPTG+Cam+Kan</LI><LI>LB+Xgal+IPTG+Cam</LI><LI>LB+Xgal+IPTG</LI><LI>LB+Kan</LI><LI>LB+Cam</LI></OL><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Strain</TH><TH class="tableheaderadpbl">Plasmid/fragment size (bp)</TH><TH class="tableheaderadpbl">Medium*</TH><TH class="tableheaderadpbl">Observation</TH></TR><TR class="tableunevenadpbl"><TD><EM>E. coli</EM> BL21DE3 + pACYCdxCas9Tn5<I>lacZ</I>gRNA</TD><TD>-</TD><TD>1</TD></TR><TR class="tableevenadpbl"><TD><EM>E. coli</EM> BL21DE3 + pACYCdxCas9Tn5<I>lacZ</I>gRNA</TD><TD>-</TD><TD>2</TD></TR><TR class="tableunevenadpbl"><TD><EM>E. coli</EM> BL21DE3 + pACYCdxCas9Tn5<I>lacZ</I>gRNA</TD><TD>pKant<SUB>DpnI</SUB></TD><TD>1</TD></TR><TR class="tableevenadpbl"><TD><EM>E. coli</EM> BL21DE3 + pACYCdxCas9Tn5<I>lacZ</I>gRNA</TD><TD>pKant<SUB>DpnI</SUB></TD><TD>2</TD></TR><TR class="tableunevenadpbl"><TD><EM>E. coli</EM> BL21DE3 + pACYCdxCas9Tn5<I>lacZ</I>gRNA</TD><TD>pKant<SUB>DpnI</SUB></TD><TD>4</TD></TR><TR class="tableevenadpbl"><TD><EM>E. coli</EM> BL21DE3 + pACYCdxCas9Tn5<I>lacZ</I>gRNA</TD><TD>MEpKantME<SUB>DpnI</SUB></TD><TD>1</TD></TR><TR class="tableunevenadpbl"><TD><EM>E. coli</EM> BL21DE3 + pACYCdxCas9Tn5<I>lacZ</I>gRNA</TD><TD>MEpKantME<SUB>DpnI</SUB></TD><TD>2</TD></TR><TR class="tableevenadpbl"><TD><EM>E. coli</EM> BL21DE3 + pACYCdxCas9Tn5<I>lacZ</I>gRNA</TD><TD>MEpKantME<SUB>DpnI</SUB></TD><TD>4</TD></TR><TR class="tableunevenadpbl"><TD><EM>E. coli</EM> BL21DE3 + pACYCdxCas9Tn5<I>lacZ</I>gRNA</TD><TD>pSB1K3</TD><TD>1</TD></TR><TR class="tableevenadpbl"><TD><EM>E. coli</EM> BL21DE3 + pACYCdxCas9Tn5<I>lacZ</I>gRNA</TD><TD>pSB1K3</TD><TD>2</TD></TR><TR class="tableunevenadpbl"><TD><EM>E. coli</EM> BL21DE3 + pACYCdxCas9Tn5<I>lacZ</I>gRNA</TD><TD>pSB1K3</TD><TD>4</TD></TR></TBODY></TABLE></CENTER></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#FridayVVS3">Friday, 21/09/2018</BUTTON><DIV id="FridayVVS3" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>21/9/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>High Fidelity PCR</H4><P> To amplify kanamycin with ME flanks and biobrick flanks (~1kb) for placing it into a biobrick, high fidelity PCR was done using pSB1K3 as template DNA and IV017 &amp; IV018 as forward and reverse primers respectively. PCR mixtures were prepared, according to the High Fidelity PCR protocol.
    
    
</P><P>The thermocycler programme was installed to have an extension time of 50sec and an annealing temperature of 65°C. After the programme was finished, samples were kept at 4 °C.</P><DL><DT>Date</DT><DD>21/9/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>DNA Gel Electrophoresis</H4><P>Biobrick flanked MepKantME was run on a 0.8% agarose gel in TAE buffer. 5μL was pipetted into each lane. The gel was imaged using a gel documentation system with UV-light.</P><DL><DT>Date</DT><DD>21/9/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>PCR clean-up</H4><P>
    Biobrick flanked PCR products were subjected to PCR Clean-Up, according to the protocol. Elution was done in 40 µL Milli-Q.
</P><DL><DT>Date</DT><DD>21/9/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>Nanodrop DNA Quantification</H4><P> 
    The following isolates were quantified according to the Nanodrop DNA Quantification protocol. The concentration and purity of the DNA (260/280 and 260/230 ratios) were measured. 
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Isolate</TH><TH class="tableheaderadpbl">Concentration (ng/uL)</TH><TH class="tableheaderadpbl">260/280 ratio</TH><TH class="tableheaderadpbl">260/230 ratio</TH></TR><TR class="tableunevenadpbl"><TD>Biobrick flanked MEpKantME</TD><TD>620</TD><TD>2.25</TD><TD>1.9</TD></TR></TBODY></TABLE></CENTER></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#SundayVVS3">Sunday, 23/09/2018</BUTTON><DIV id="SundayVVS3" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>23/9/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>Restriction </H4><P>Restriction cloning was done for assembling MEpKantME in pSB1C3. The following table shows an overview of what fragment(s) was/were cut with what restriction enzyme(s).
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL)</TH><TH class="tableheaderadpbl">Volume (uL)</TH></TR><TR class="tableunevenadpbl"><TD>10x CutSmart buffer    </TD><TD>2</TD><TD>2</TD></TR><TR class="tableunevenadpbl"><TD>Fragment</TD><TD>3.5µl (2µg)
          Biobrick flanked MEpKantME</TD><TD>15µl (2µg)
          pSB1C3</TD></TR><TR class="tableunevenadpbl"><TD>Enzyme</TD><TD>0.4µl EcoRI</TD><TD>0.4µl EcoRI</TD></TR><TR class="tableunevenadpbl"><TD>Enzyme</TD><TD>0.4µl SpeI</TD><TD>0.4µl XbaI</TD></TR><TR class="tableunevenadpbl"><TD>MilliQ</TD><TD> 13.7</TD><TD> 2.2</TD></TR></TBODY></TABLE></CENTER><P>The samples were incubated for 4 hour(s) at 37 °C, and were not heat inactivated.

</P><DL><DT>Date</DT><DD>23/9/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>PCR clean-up</H4><P>
    restricted PCR products were subjected to PCR Clean-Up, according to the protocol. Elution was done in 30 µL Milli-Q.
</P><DL><DT>Date</DT><DD>23/9/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>Nanodrop DNA Quantification</H4><P> 
    The following isolates were quantified according to the Nanodrop DNA Quantification protocol. The concentration and purity of the DNA (260/280 and 260/230 ratios) were measured. 
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Isolate</TH><TH class="tableheaderadpbl">Concentration (ng/uL)</TH><TH class="tableheaderadpbl">260/280 ratio</TH><TH class="tableheaderadpbl">260/230 ratio</TH></TR><TR class="tableunevenadpbl"><TD>BBMEpKantMEBB<SUB>EcoRI/SpeI</SUB></TD><TD>75</TD><TD>1.9</TD><TD>1.5</TD></TR><TR class="tableevenadpbl"><TD>BBMEpKantMEBB<SUB>EcoRI/SpeI</SUB></TD><TD>1.9</TD><TD>1.7</TD></TR></TBODY></TABLE></CENTER><DL><DT>Date</DT><DD>23/9/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>Ligation</H4><P>In order to assemble cut fragments ligation was performed. A 20µl ligation reaction mixture was prepared with the following composition</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL) </TH></TR><TR class="tableunevenadpbl"><TD>Ligase Buffer (10x) </TD><TD>2</TD></TR><TR class="tableevenadpbl"><TD>T4 DNA Ligase</TD><TD>1</TD></TR><TR class="tableunevenadpbl"><TD>DNA vector (~100 ng) </TD><TD>4.6</TD></TR><TR class="tableunevenadpbl"><TD>DNA fragment (~500 ng) </TD><TD>6.5</TD></TR><TR class="tableunevenadpbl"><TD>MilliQ</TD><TD>5.9</TD></TR></TBODY></TABLE></CENTER><P>The samples were incubated overnight at 4°C for ligation. </P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV></DIV></UL></FORM><FORM><LABEL for="WeekVVS4" class="subdropdown cadpbl">Week 4 24/09/2018 - 30/09/2018</LABEL><UL class="submenu"><DIV class="subcontent"><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#MondayVVS4">Monday, 24/09/2018</BUTTON><DIV id="MondayVVS4" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>24/9/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4> Transformation of chemically competent cells </H4><P>The following transformations were carried out according to the protocol for transformation of chemically competent cells:
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Strain</TH><TH class="tableheaderadpbl">Plasmid &amp; size (bp) </TH><TH class="tableheaderadpbl">Cloning method</TH><TH class="tableheaderadpbl">Volume added (uL)</TH><TH class="tableheaderadpbl">Medium on which was plated</TH></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I>DH5α</TD><TD>MEpKantME in pSB1C3*</TD><TD>restriction-ligation</TD><TD>2</TD><TD>LB+Cam</TD></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I>DH5α</TD><TD>MEpKantME in pSB1C3*</TD><TD>restriction-ligation</TD><TD>2</TD><TD>LB+Cam+Kan</TD></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I>DH5α</TD><TD>MEpKantME in pSB1C3*</TD><TD>restriction-ligation</TD><TD>18</TD><TD>LB+Cam</TD></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I>DH5α</TD><TD>MEpKantME in pSB1C3*</TD><TD>restriction-ligation</TD><TD>18</TD><TD>LB+Cam+Kan</TD></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I>DH5α</TD><TD>-</TD><TD>-</TD><TD>-</TD><TD>LB+Cam</TD></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I>DH5α</TD><TD>-</TD><TD>-</TD><TD>-</TD><TD>LB+Cam+Kan</TD></TR></TBODY></TABLE></CENTER>     
  *default RFP is NOT cut out of the backbone.  <P>200 uL LB medium was added as recovery medium.
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#TuesdayVVS4">Tuesday, 25/09/2018</BUTTON><DIV id="TuesdayVVS4" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>25/9/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4> Transformation Results </H4><P> After carrying out the transformation protocol on 24/9/2018, the plates were left for overnight incubation. After ~14-18hr incubation, the following results were observed: </P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Strain</TH><TH class="tableheaderadpbl">Plasmid/insert</TH><TH class="tableheaderadpbl">Cloning method</TH><TH class="tableheaderadpbl">Medium</TH><TH class="tableheaderadpbl">Result</TH></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I>DH5α</TD><TD>MEpKantME in pSB1C3*</TD><TD>restriction-ligation</TD><TD>LB+Cam</TD></TR><TR class="tableevenadpbl"><TD><I>E.coli</I>DH5α</TD><TD>MEpKantME in pSB1C3*</TD><TD>restriction-ligation</TD><TD>LB+Cam+Kan</TD></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I>DH5α</TD><TD>-</TD><TD>-</TD><TD>LB+Cam</TD></TR><TR class="tableevenadpbl"><TD><I>E.coli</I>DH5α</TD><TD>-</TD><TD>-</TD><TD>LB+Cam+Kan</TD><TD>No growth</TD></TR></TBODY></TABLE></CENTER><DL><DT>Date</DT><DD>25/9/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>Colony PCR</H4><P>Colony PCR was done to verify construction f the right plasmid, and propagation thereof. This plasmid contains ME-flanked kanamycin and RFP, placed in a pSB1C3 backbone, which can be checked with primers VF2 and VR.
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL)</TH></TR><TR class="tableunevenadpbl"><TD>Template DNA</TD><TD>picked colony</TD></TR><TR class="tableevenadpbl"><TD>Forward primer</TD><TD>VF2</TD></TR><TR class="tableunevenadpbl"><TD>Reverse primer</TD><TD>VR</TD></TR><TR class="tableunevenadpbl"><TD>Expected amplicon size (bp)</TD><TD>2454*</TD></TR></TBODY></TABLE></CENTER><I>*We expect 1383bp in case of no placement of MEpKantME in the vector. </I><P>Tubes were prepared for colony PCR, containing picked colony as template DNA. The thermocycler programme was installed to have an extension time of 3 minutes and an annealing temperature of 60°C. After the programme was finished, samples were kept at 4°C.</P><DL><DT>Date</DT><DD>25/9/2018dd&gt;
   </DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>DNA Gel Electrophoresis</H4><P>colony PCR samples were run on a 0.8% agarose gel in TAE buffer. 5μL was pipetted into each lane. The gel was imaged using a gel documentation system with UV-light.</P><P>Based on this gel, it was decided to continue with colony 4 &amp; 6 for further aclaration. </P><DL><DT>Date</DT><DD>25/9/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>Liquid Starter Culture Preparation</H4><P>
Colony 4 &amp; 6 of the transformant plate (24/9/2018) were used for inoculation of 10mL liquid starter culture (Luria Broth, complemented with chloramphenicol AND kanamycin) following the Liquid Starter Culture Protocol.
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#WednesdayVVS4">Wednesday, 26/09/2018</BUTTON><DIV id="WednesdayVVS4" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>26/9/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>Plasmid Isolation</H4><P> 
    Overnight starter cultures of colony 4 &amp; 6 were subjected to plasmid isolation following the Plasmid Isolation Protocol. Final elution was done with 30 μL of pre-warmed MilliQ.
</P><DL><DT>Date</DT><DD>26/9/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>Nanodrop DNA Quantification</H4><P> 
    The following isolates were quantified according to the Nanodrop DNA Quantification protocol. The concentration and purity of the DNA (260/280 and 260/230 ratios) were verified to be around 1.8~1.9. 
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Isolate</TH><TH class="tableheaderadpbl">Concentration (ng/uL)</TH></TR><TR class="tableunevenadpbl"><TD>pSB1C3-RFP-MEpKantME colony 4</TD><TD>264</TD></TR><TR class="tableevenadpbl"><TD>pSB1C3-RFP-MEpKantME colony 4</TD><TD>250</TD></TR><TR class="tableunevenadpbl"><TD>pSB1C3-RFP-MEpKantME colony 6</TD><TD>252</TD></TR><TR class="tableevenadpbl"><TD>pSB1C3-RFP-MEpKantME colony 6</TD><TD>229</TD></TR></TBODY></TABLE></CENTER><DL><DT>Date</DT><DD>26/9/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>Sequence Verification </H4><H5>Option 1: Sequence from plasmid</H5><P> For verification of MEpKantME being inserted into the pSB1C3, next to RFP, primers VF2, IV020 and VR were used for sequencing. The sequencing starts from the biobrick backone. The 10µL sequencing sample was prepared in a 1.5 mL tube as follows: 

</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL)</TH><TH class="tableheaderadpbl">Final concentration</TH></TR><TR class="tableunevenadpbl"><TD>Plasmid DNA (500ng)</TD><TD>3</TD><TD>50 ng/uL</TD></TR><TR class="tableunevenadpbl"><TD>10uM primer</TD><TD>2.5</TD><TD>25 pmol</TD></TR><TR class="tableunevenadpbl"><TD>MilliQ</TD><TD>4.5</TD><TD>-</TD></TR></TBODY></TABLE></CENTER><P>Within 7 days, the sequence was retrievable online and alignment with the <I>in silico</I> designed molecule showed 100% match of the 1050 bp downstream of the sequencing primer binding site. </P><DL><DT>Date</DT><DD>26/9/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>High Fidelity PCR</H4><P> To amplify MEpKantME with long arms (3560bp), high fidelity PCR was done using pSB1C3-RFP-MEpKantME as template DNA and IV023 &amp; VR as forward and reverse primers respectively. PCR mixtures were prepared, according to the High Fidelity PCR protocol.</P><P>The thermocycler programme was installed to have an extension time of 3.5 minutes and an annealing temperature of 60 °C. After the programme was finished, samples were kept at 4 °C.</P><DL><DT>Date</DT><DD>26/9/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>DNA Gel Electrophoresis</H4><P>The extended-arm MEpKantME fragments were run on a 0.8% agarose gel in TAE buffer. 3μL was pipetted into each lane. The gel was imaged using a gel documentation system with UV-light.</P><P>This gel shows that this amplification worked as designed. The Kanamyin cassette is now protected better against exonuclease degradation of the ME flanks. (approximately 1200bp spacer arms).</P><DL><DT>Date</DT><DD>26/9/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>PCR clean-up</H4><P>
    PCR products of long-armed MEpKantME were subjected to PCR Clean-Up, according to the protocol. Elution was done in 40 µL Milli-Q.
</P><DL><DT>Date</DT><DD>26/9/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>Nanodrop DNA Quantification</H4><P> 
    The following isolates were quantified according to the Nanodrop DNA Quantification protocol. The concentration and purity of the DNA (260/280 and 260/230 ratios) were measured. 
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Isolate</TH><TH class="tableheaderadpbl">Concentration (ng/uL)</TH><TH class="tableheaderadpbl">260/280 ratio</TH><TH class="tableheaderadpbl">260/230 ratio</TH></TR><TR class="tableunevenadpbl"><TD>long-armed MEpKantME (colony 4)</TD><TD>1342</TD><TD>1.87</TD><TD>2.30</TD></TR><TR class="tableevenadpbl"><TD>long-armed MEpKantME (colony 6)</TD><TD>1506</TD><TD>1.86</TD><TD>2.29</TD></TR></TBODY></TABLE></CENTER><DL><DT>Date</DT><DD>26/9/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>DpnI Digestion</H4><P>To eliminate any residual template DNA, a DpnI digestion was performed with the following reaction mixture:
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL)</TH></TR><TR class="tableunevenadpbl"><TD>10x CutSmart buffer</TD><TD>5</TD></TR><TR class="tableevenadpbl"><TD>PCR product</TD><TD>34 (~45-50µg)</TD></TR><TR class="tableunevenadpbl"><TD>Enzyme (DpnI)</TD><TD>1</TD></TR><TR class="tableunevenadpbl"><TD>MilliQ</TD><TD>10</TD></TR></TBODY></TABLE></CENTER><P>The reaction was incubated overnight at 37 °C. No heat inactivation was performed.</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#ThursdayVVS4">Thursday, 27/09/2018</BUTTON><DIV id="ThursdayVVS4" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>27/9/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>PCR clean-up</H4><P>
    50 μL of restriction product was subjected to PCR Clean-Up, according to the protocol. Elution was done in 40 µL Milli-Q.
</P><DL><DT>Date</DT><DD>27/9/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>Nanodrop DNA Quantification</H4><P> 
    The following isolates were quantified according to the Nanodrop DNA Quantification protocol. The concentration and purity of the DNA (260/280 and 260/230 ratios) were measured. 
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Isolate</TH><TH class="tableheaderadpbl">Concentration (ng/uL)</TH><TH class="tableheaderadpbl">260/280 ratio</TH><TH class="tableheaderadpbl">260/230 ratio</TH></TR><TR class="tableunevenadpbl"><TD>long-armed MEpKantME<SUB>DpnI</SUB> (colony 4)</TD><TD>496</TD><TD>1.86</TD><TD>2.24</TD></TR><TR class="tableevenadpbl"><TD>long-armed MEpKantME<SUB>DpnI</SUB> (colony 6)</TD><TD>588</TD><TD>1.85</TD><TD>2.22</TD></TR></TBODY></TABLE></CENTER><DL><DT>Date</DT><DD>27/9/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>Electroporation</H4><P>The following transformations were carried out according to the protocol for electroporation:</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Strains</TH><TH class="tableheaderadpbl">Plasmid/fragment size (bp)</TH><TH class="tableheaderadpbl">Amount added</TH><TH class="tableheaderadpbl">Plate medium</TH></TR><TR class="tableunevenadpbl"><TD><EM>E. coli</EM> BL21DE3</TD><TD>pSB1C3-MEpKantME-RFP</TD><TD>50ng</TD><TD>1 &amp; 2</TD></TR><TR class="tableevenadpbl"><TD><EM>E. coli</EM>  BL21DE3</TD><TD>long-armed MEpKantME<SUB>DpnI</SUB></TD><TD>1µg</TD><TD>1 &amp; 2</TD></TR><TR class="tableunevenadpbl"><TD><EM>E. coli</EM> BL21DE3 + pACYCdxCas9Tn5<I>lacZ</I>gRNA</TD><TD>pSB1C3-MEpKantME-RFP</TD><TD>50ng</TD><TD>1, 2 &amp; 3</TD></TR><TR class="tableevenadpbl"><TD><EM>E. coli</EM> BL21DE3 + pACYCdxCas9Tn5<I>lacZ</I>gRNA</TD><TD>long-armed MEpKantME<SUB>DpnI</SUB></TD><TD>1µg</TD><TD>1, 2 &amp; 3</TD></TR></TBODY></TABLE></CENTER><I>*Some samples were plated on several media. The media are displayed with the corresponding numbers:</I><OL><LI>LB+Xgal+IPTG+Cam+Kan</LI><LI>LB+Xgal+IPTG+Cam</LI><LI>LB+Kan</LI></OL><P>200μl LB medium was added as recovery medium.
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#FridayVVS4">Friday, 28/09/2018</BUTTON><DIV id="FridayVVS4" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>28/9/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4> Transformation Results </H4><P> After carrying out the transformation protocol on 27/9/2018, the plates were left for overnight incubation. After ~14-18hr incubation, the following results were observed: </P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Strains</TH><TH class="tableheaderadpbl">Plasmid/fragment size (bp)</TH><TH class="tableheaderadpbl">Plate medium</TH><TH class="tableheaderadpbl">Observation</TH></TR><TR class="tableunevenadpbl"><TD><EM>E. coli</EM> BL21DE3</TD><TD>pSB1C3-MEpKantME-RFP</TD><TD>1</TD></TR><TR class="tableunevenadpbl"><TD><EM>E. coli</EM> BL21DE3</TD><TD>pSB1C3-MEpKantME-RFP</TD><TD>2</TD></TR><TR class="tableevenadpbl"><TD><EM>E. coli</EM>  BL21DE3</TD><TD>long-armed MEpKantME<SUB>DpnI</SUB></TD><TD>1</TD></TR><TR class="tableevenadpbl"><TD><EM>E. coli</EM>  BL21DE3</TD><TD>long-armed MEpKantME<SUB>DpnI</SUB></TD><TD>2</TD></TR><TR class="tableunevenadpbl"><TD><EM>E. coli</EM> BL21DE3 + pACYCdxCas9Tn5<I>lacZ</I>gRNA</TD><TD>pSB1C3-MEpKantME-RFP</TD><TD>1</TD></TR><TR class="tableunevenadpbl"><TD><EM>E. coli</EM> BL21DE3 + pACYCdxCas9Tn5<I>lacZ</I>gRNA</TD><TD>pSB1C3-MEpKantME-RFP</TD><TD>2</TD></TR><TR class="tableunevenadpbl"><TD><EM>E. coli</EM> BL21DE3 + pACYCdxCas9Tn5<I>lacZ</I>gRNA</TD><TD>pSB1C3-MEpKantME-RFP</TD><TD>3</TD></TR><TR class="tableevenadpbl"><TD><EM>E. coli</EM> BL21DE3 + pACYCdxCas9Tn5<I>lacZ</I>gRNA</TD><TD>long-armed MEpKantME<SUB>DpnI</SUB></TD><TD>1</TD></TR><TR class="tableevenadpbl"><TD><EM>E. coli</EM> BL21DE3 + pACYCdxCas9Tn5<I>lacZ</I>gRNA</TD><TD>long-armed MEpKantME<SUB>DpnI</SUB></TD><TD>2</TD></TR><TR class="tableevenadpbl"><TD><EM>E. coli</EM> BL21DE3 + pACYCdxCas9Tn5<I>lacZ</I>gRNA</TD><TD>long-armed MEpKantME<SUB>DpnI</SUB></TD><TD>3</TD></TR></TBODY></TABLE></CENTER><I>*Some samples were plated on several media. The media are displayed with the corresponding numbers:</I><OL><LI>LB+Xgal+IPTG+Cam+Kan</LI><LI>LB+Xgal+IPTG+Cam</LI><LI>LB+Kan</LI></OL> 
    Some of the colonies had white phenotype when observing them after 14-18h incubation. However, judging from previous observations, it is not excluded that RFP might come to expression at a later stage. Nevertheless, these colonies were inoculated for further identification.

<DL><DT>Date</DT><DD>28/9/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>Liquid Starter Culture Preparation</H4><P>
White colonies from LB+Cam+Kan+IPTG+Xgal plates were used for inoculation of 10mL liquid starter culture (Luria Broth, complemented with chloramphenicol &amp; kanamycin) following the Liquid Starter Culture Protocol.
</P><DL><DT>Date</DT><DD>28/9/2018</DD><DT>Operator</DT><DD>Kavish Kohabir</DD></DL><H4>Colony PCR</H4><P>Colony PCR was done to verify whether <I>lacZ</I> was really disrupted. The following rimer setup was used:</P><P>With this setup, the following amplicon sizes are expected:</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Primers</TH><TH class="tableheaderadpbl">No integration</TH><TH class="tableheaderadpbl">Integration</TH></TR><TR class="tableunevenadpbl"><TD>IV007 &amp; IV019</TD><TD>no band</TD><TD>?*</TD></TR><TR class="tableevenadpbl"><TD>IV020 &amp; IV008</TD><TD>no band</TD><TD>?*</TD></TR><TR class="tableunevenadpbl"><TD>IV007 &amp; IV022</TD><TD>2180bp</TD><TD>3218bp</TD></TR><TR class="tableunevenadpbl"><TD>IV021 &amp; IV008</TD><TD>2658bp</TD><TD>3696bp</TD></TR></TBODY></TABLE></CENTER>     
    *This amplicon size depends on the integration site of the donor DNA. The maximum expected size is 5kb.
<P>23 colonies were used for doing 4 PCR reactions each + a control (isolated parental BL21DE3 gDNA for each PCR; 92 reactions total. Resuspende colony served as template DNA for each reaction. The thermocycler programme was installed to have an extension time of 5min and an annealing temperature of 60 °C. After the programme was finished, samples were kept at 4°C.</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV></DIV></UL></FORM></DIV></DIV></DIV><DIV class="container nowbg" id="Purification"><DIV class="row nowbg"><DIV class="col-xs-12 col-sm-3 col-lg-3"><NAV id="myScrollspy" class="hidden-xs hidden-sm"><UL class="nav nav-pills nav-stacked" data-spy="affix"><LI class="dropdown"><A class="dropdown-toggle adpbl" data-toggle="dropdown" href="#">June</A></LI><LI><A href="#JunePurification4" class="adpbl">Week: 25/06/2018 - 01/06/2018</A></LI></UL><LI class="dropdown"><A class="dropdown-toggle adpbl" data-toggle="dropdown" href="#">July</A></LI><LI><A href="#JulyPurification1" class="adpbl">Week: 02/07/2018 - 08/07/2018</A></LI><LI><A href="#JulyPurification2" class="adpbl">Week: 09/07/2018 - 15/07/2018</A></LI><LI><A href="#JulyPurification3" class="adpbl">Week: 16/07/2018 - 22/07/2018</A></LI><LI><A href="#JulyPurification4" class="adpbl">Week: 23/07/2018 - 29/07/2018</A></LI><LI class="dropdown"><A class="dropdown-toggle adpbl" data-toggle="dropdown" href="#">August</A></LI><LI><A href="#AugustPurification1" class="adpbl">Week: 30/07/2018 - 05/08/2018</A></LI><LI><A href="#AugustPurification2" class="adpbl">Week: 06/08/2018 - 12/08/2018</A></LI><LI><A href="#AugustPurification3" class="adpbl">Week: 13/08/2018 - 19/08/2018</A></LI><LI><A href="#AugustPurification4" class="adpbl">Week: 20/08/2018 - 27/08/2018</A></LI><LI><A href="#AugustPurification5" class="adpbl">Week: 28/08/2018 - 02/09/2018</A></LI><LI class="dropdown"><A class="dropdown-toggle adpbl" data-toggle="dropdown" href="#">September</A></LI><LI><A href="#SeptemberPurification1" class="adpbl">Week: 03/09/2018 - 09/09/2018</A></LI><LI><A href="#SeptemberPurification2" class="adpbl">Week: 10/09/2018 - 16/09/2018</A></LI><LI><A href="#SeptemberPurification3" class="adpbl">Week: 17/09/2018 - 23/09/2018</A></LI><LI><A href="#SeptemberPurification4" class="adpbl">Week: 24/09/2018 - 30/09/2018</A></LI><LI class="drowdown"><A class="dropdown-toggle adpbl" data-toggle="dropdown" href="#">October</A></LI><LI><A href="#OctoberPurification1" class="adpbl">Week: 01/10/2018 - 07/10/2018</A></LI><LI><A href="#OctoberPurification2" class="adpbl">Week: 08/10/2018 - xx/10/2018</A></LI></NAV></DIV><DIV class="col-lg-8 col-md-8 col-sm-12 col-xs-12 nowbg" style="margin-right:20px;"><H3 class="adpbl">June</H3><FORM><LABEL for="WeekPF4" class="subdropdown cadpbl">Week 4 25/06/2018 - 01/07/2018</LABEL><UL class="submenu"><DIV class="subcontent"><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#FridayPF4">Friday, 29/06/2018</BUTTON><DIV id="FridayPF4" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>29/06/2018</DD><DT>Operator</DT><DD>Monique de Leeuw</DD></DL><H4>Objective</H4><P>Tn5 was expressed and purified to produce protein for <I>in vitro</I> functionality evaluation.
    </P><H4>Protein expression - <I>Seed Culture</I></H4><P>A seed culture was made to start Upstream Processing (USP) of Tn5 protein. A colony with Tn5_3 strain was used for inoculation of 10mL liquid starter culture (Luria Broth, complemented with 34 μg/mL chloramphenicol) following the Liquid Starter Culture Protocol.
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#SaturdayPF4">Saturday, 30/06/2018</BUTTON><DIV id="SaturdayPF4" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>30/06/2018</DD><DT>Operator</DT><DD>Monique de Leeuw</DD></DL><H4>Protein Expression - <I>Inoculation &amp; Induction</I></H4><P>The upstream processing of the Tn5 protein continued. Inoculated 1L liquid media (Luria Broth, complemented with 34 μg/mL chloramphenicol) with the seed culture and grown to an OD<SUB>600</SUB> of 2. 
    </P><CENTER><TABLE><CAPTION class="figadpbl"><B>Table :</B> Growth represented by OD<SUB>600</SUB> over time</CAPTION><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Time</TH><TH class="tableheaderadpbl">OD<SUB>600</SUB></TH></TR><TR class="tableunevenadpbl"><TD>12:00am</TD><TD>0.09</TD></TR><TR class="tableunevenadpbl"><TD>1:00pm</TD><TD>0.2</TD></TR><TR class="tableunevenadpbl"><TD>2:00pm</TD><TD>0.49</TD></TR></TBODY></TABLE></CENTER><P>After the culture OD<SUB>600</SUB> reached 0.49, the Protein Expression protocol was followed. After the expression is completed, the cells were transferred to Downstream Processing (DSP).
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#SundayPF4">Sunday, 01/07/2018</BUTTON><DIV id="SundayPF4" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>01/07/2018</DD><DT>Operator</DT><DD>Monique de Leeuw</DD></DL><H4>Protein Purification - <I>Cell lysis</I></H4><P>The downstream processing of the Tn5 protein started. Harvest the cells according to the purification protocol. 5.71g of cells of Tn5 protein were resuspended in Lysis buffer (20 mM Tris-HCl, 250 mM NaCl, 10% glycerol, pH 8).
        
    After resuspension the following additives were added:
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Additive</TH><TH class="tableheaderadpbl">Stock Concentration</TH><TH class="tableheaderadpbl"> Volume added</TH></TR><TR class="tableunevenadpbl"><TD>DTT</TD><TD>0.5M</TD><TD>80uL</TD></TR><TR class="tableunevenadpbl"><TD>EDTA-free Complete protease inhibitor tablets</TD><TD>n/a</TD><TD>1tablets</TD></TR><TR class="tableunevenadpbl"><TD>imidazole</TD><TD>2M</TD><TD>400uL</TD></TR></TBODY></TABLE></CENTER><P> The cells were lysed and clarified according to the the purification protocol.
</P><H4> Tris-Glycine Polyacrylamide Gel </H4><P>In order to analyse the expression of the Tn5 protein, a precast Tris-Glycine polyacrylamide gel was run with a 6% stacking gel and 12% separation gel according to the SDS PAGE Tris-Glycine Polyacrylamide Gel protocol. The polyacrylamide gel ran for 90min at 50volt. After the was completed, the gel was stained with SimplyBlue SafeStain and imaged using the GelDoc system. The result of the gel is shown below:
        
<B>Figure expression of Tn5: </B><I>12% SDS PAGE of Tn5 (expected size 53.3kDa). Lane 1 molecular ladder (kDa), lane clarified lysate, lane 3 cell debris. 
.  </I></P><H4> Protein Purification - <I>Nickel Affinty Chromatography</I></H4><P>1mL of 50% His-Select Ni resin, was washed according to the Purification protocol. 33.5mL clarified lysate of Tn5 protein was added to the washed His-Select Ni resin and incubated with gentle mixing for 60minutes at 4 ℃. The His-Select Ni resin was washed with washing buffer (20 mM Tris-HCl, 250 mM NaCl, 20mM Imidazole, 10% glycerol, 1mM DTT, pH8), and eluded with with 6* 500μL (6CV) of elution buffer (20 mM Tris-HCl, 250 mM NaCl, 250 mM Imidazole, 10% glycerol, 1mM DTT, pH8) at 4 ℃, according to the Purification protocol. 
</P><H4> Tris-Glycine Polyacrylamide Gel </H4><P>In order to analyse the nickel affinty chromatography of the Tn5 protein, a precast Tris-Glycine polyacrylamide gel was run with a 6% stacking gel and 12% separation gel according to the SDS PAGE Tris-Glycine Polyacrylamide Gel protocol.The polyacrylamide gel ran for 90min at 50volt. After the was completed, the gel was stained with SimplyBlue SafeStain and imaged using the GelDoc system. The result of the gel is shown below:
        
<B>Figure Nickel Affinity Chromatography. </B><I> 12% SDS of Tn5 (expected size 53.3kDa). Lane 1 molecular ladder (kDa), lane clarified lysate, lane 3 cell debris, lane 4 flow through, lane 5 wash1, lane 6 wash2, lane 7 Elution 1, lane 8 Elution 2, lane 9 Elution 3, lane 10 Elution 4.</I></P><P>No Tn5 was observed in the lysate or the elution fraction. At first, we thought an error occurred during expression, however after thorough analysis we identified our mistake. There was a miss communication between stain construction and bioprocess as a result the plasmid was in the incorrect host organism. The Tn5 plasmid was still in DH5 alpha. Going forward, all transformation into expression host will be covered by the bioprocess team, to avoid this mistake in the future. </P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV></DIV></UL></FORM><H3 class="adpbl">July</H3><FORM><LABEL for="WeekPFJ1" class="subdropdown cadpbl">Week 1 02/07/2018 - 08/07/2018</LABEL><UL class="submenu"><DIV class="subcontent"><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#MondayPFJ1">Monday, 02/07/2018</BUTTON><DIV id="MondayPFJ1" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>02/07/2018</DD><DT>Operator</DT><DD>Monique de Leeuw</DD></DL><H4> Transformation of chemically competent cells </H4><P>The Tn5_3 plasmid was transfomred into BL21AI for Tn5 protein production. The following transformations were carried out according to the protocol for transformation of chemically competent cells:
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Strain</TH><TH class="tableheaderadpbl">Plasmid &amp; size (bp) </TH><TH class="tableheaderadpbl">Cloning method</TH><TH class="tableheaderadpbl">Volume added (uL)</TH><TH class="tableheaderadpbl">Medium on which was plated</TH></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I>BL21AI</TD><TD>Tn5.3 (5210bp)</TD><TD>Heat shock</TD><TD>1</TD><TD>LB chloramphenicol</TD></TR></TBODY></TABLE></CENTER><P>200 uL LB medium was added as recovery medium.
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#TuesdayPFJ1">Tuesday, 03/07/2018</BUTTON><DIV id="TuesdayPFJ1" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>03/07/2018</DD><DT>Operator</DT><DD>Monique de Leeuw</DD></DL><H4>Objective</H4><P>Tn5 was expressed and purified to produce protein for <I>in vitro</I> functionality evaluation.
    </P><H4>Protein expression - <I>Seed Culture</I></H4><P>A seed culture was made to start Upstream Processing (USP) of Tn5 protein. A colony with Tn5_3 strain was used for inoculation of 10mL liquid starter culture (Luria Broth, complemented with 34 μg/mL chloramphenicol) following the Liquid Starter Culture Protocol.
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#WednesdayPFJ1">Wednesday, 04/07/2018</BUTTON><DIV id="WednesdayPFJ1" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>04/07/2018</DD><DT>Operator</DT><DD>Monique de Leeuw</DD></DL><H4>Protein Expression - <I>Inoculation &amp; Induction</I></H4><P>The upstream processing of the Tn5 protein continued. Inoculated 1L liquid media (Luria Broth, complemented with 34 μg/mL chloramphenicol) with the seed culture and grown to an OD<SUB>600</SUB> of 2. 
    </P><CENTER><TABLE><CAPTION class="figadpbl"><B>Table :</B> Growth represented by OD<SUB>600</SUB> over time</CAPTION><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Time</TH><TH class="tableheaderadpbl">OD<SUB>600</SUB></TH></TR><TR class="tableunevenadpbl"><TD>10:30am</TD><TD>0.12</TD></TR><TR class="tableunevenadpbl"><TD>11:30am</TD><TD>0.25</TD></TR><TR class="tableunevenadpbl"><TD>12:30am</TD><TD>0.56</TD></TR></TBODY></TABLE></CENTER><P>After the culture OD<SUB>600</SUB> reached 0.56, the Protein Expression protocol was followed. After the expression is completed, the cells were transferred to Downstream Processing (DSP).
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#ThursdayPFJ1">Thursday, 05/07/2018</BUTTON><DIV id="ThursdayPFJ1" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>05/07/2018</DD><DT>Operator</DT><DD>Monique de Leeuw</DD></DL><H4>Protein Purification - <I>Cell lysis</I></H4><P>The downstream processing of the Tn5 protein started. Harvest the cells according to the purification protocol. 5.87g of cells of Tn5 protein were resuspended in Lysis buffer (20 mM Tris-HCl, 250 mM NaCl, 10% glycerol, pH 8).
        
    After resuspension the following additives were added:
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Additive</TH><TH class="tableheaderadpbl">Stock Concentration</TH><TH class="tableheaderadpbl"> Volume added</TH></TR><TR class="tableunevenadpbl"><TD>DTT</TD><TD>0.5M</TD><TD>82uL</TD></TR><TR class="tableunevenadpbl"><TD>EDTA-free Complete protease inhibitor tablets</TD><TD>n/a</TD><TD>1tablets</TD></TR><TR class="tableunevenadpbl"><TD>imidazole</TD><TD>2M</TD><TD>411uL</TD></TR></TBODY></TABLE></CENTER><P> The cells were lysed and clarified according to the the purification protocol.
</P><H4> Tris-Glycine Polyacrylamide Gel </H4><P>In order to analyse the expression of the Tn5 protein, a precast Tris-Glycine polyacrylamide gel was run with a 6% stacking gel and 12% separation gel according to the SDS PAGE Tris-Glycine Polyacrylamide Gel protocol. The polyacrylamide gel ran for 90min at 50volt. After the was completed, the gel was stained with SimplyBlue SafeStain and imaged using the GelDoc system. The result of the gel is shown below:
        
<B>Figure expression of Tn5: </B><I>12% SDS PAGE of Tn5 (expected size 53.3kDa). Lane 1 molecular ladder (kDa), lane 2 whole lysate, lane 3 clarified lysate. 
.  </I></P><H4> Protein Purification - <I>Nickel Affinty Chromatography</I></H4><P>1mL of 50% His-Select Ni resin, was washed according to the Purification protocol. 33.5mL clarified lysate of Tn5 protein was added to the washed His-Select Ni resin and incubated with gentle mixing for 60minutes at 4 ℃. The His-Select Ni resin was washed with washing buffer (20 mM Tris-HCl, 250 mM NaCl, 20mM Imidazole, 10% glycerol, 1mM DTT, pH8), and eluded with with 6* 500μL (6CV) of elution buffer (20 mM Tris-HCl, 250 mM NaCl, 250 mM Imidazole, 10% glycerol, 1mM DTT, pH8) at 4 ℃, according to the Purification protocol. 
</P><H4> Tris-Glycine Polyacrylamide Gel </H4><P>In order to analyse the nickel affinty chromatography of the Tn5 protein, a precast Tris-Glycine polyacrylamide gel was run with a 6% stacking gel and 12% separation gel according to the SDS PAGE Tris-Glycine Polyacrylamide Gel protocol.The polyacrylamide gel ran for 90min at 50volt. After the was completed, the gel was stained with SimplyBlue SafeStain and imaged using the GelDoc system. The result of the gel is shown below:
        
<B>Figure Nickel Affinity Chromatography. </B><I> 12% SDS of Tn5 (expected size 53.3kDa). Lane 1 molecular ladder (kDa), lane 2 whole lysate, lane 3 clarified lysate, lane 4 flow through, lane 5 wash1, lane 6 Elution 1, lane 7 Elution 2, lane 8 Elution 3, lane 9 Elution 4 , lane 10 elution 5</I></P><P>A significant amount of Tn5 was observed in the nickel chromatography elution fractions (lane 6-9), however the Tn5 protein was also observed in the flowthrough and wash fractions (lane 4,5). It is recommended to optimize the nickel chromatography to reduce these losses, however a bigger consern is a double band for Tn5 is observed. One at the expected weight, and one slightly higher. It was unclear why the Tn5 would run higher. As a following step, elution fraction 1 (lane 6) was run under different denaturing conditions to fully denature the protein.</P><H4> Tris-Glycine Polyacrylamide Gel </H4><P>In order to analyse the degradation of the Tn5 protein, a precast Tris-Glycine polyacrylamide gel was run with a 6% stacking gel and 12% separation gel according to the SDS PAGE Tris-Glycine Polyacrylamide Gel protocol.The polyacrylamide gel ran for 90min at 50volt. After the was completed, the gel was stained with SimplyBlue SafeStain and imaged using the GelDoc system. The result of the gel is shown below:
        
<B>Figure Different sample gel preparation conditions. </B><I> 12% SDS PAGE of Tn5 (expected size 53.3kDa)  Lane 1 molecular ladder (kDa), lane 2 950C 30min, lane 3 950C 15min, lane 4 750C 7min, lane 5 No boiling.</I></P><P>  A double band for Tn5 observed for all conditions, both bands from the 12% gel were excised and sent for mass spectrometry. The mass spectrometry confirmed that both bands were full length Tn5 with both the N and C terminal intact. This concluded that there was not any protease digestion and that the results are potentially due to a gel artifact. As a result, the purification will continue.</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV></DIV></UL></FORM><FORM><LABEL for="WeekPFJ2" class="subdropdown cadpbl">Week 2 09/07/2018 - 15/07/2018</LABEL><UL class="submenu"><DIV class="subcontent"><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#MondayPFJ2">Monday, 09/07/2018</BUTTON><DIV id="MondayPFJ2" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>09/07/2018</DD><DT>Operator</DT><DD>Janine Nijenhuis</DD></DL><H4>Objective</H4><P>Tn5 was expressed and purified to produce protein for <I>in vitro</I> functionality evaluation.
    </P><H4>Protein expression - <I>Seed Culture</I></H4><P>A seed culture was made to start Upstream Processing (USP) of Tn5. A colony with Tn5 strain was used for inoculation of 40mL liquid starter culture (Luria Broth, complemented with 34 μg/mL chloramphenicol) following the Liquid Starter Culture Protocol.
</P><DL><DT>Date</DT><DD>07/09/2018</DD><DT>Operator</DT><DD>Monique de Leeuw</DD></DL><H4> Protein Purification - <I>Heparin Chromatography</I></H4><P>The Tn5 protein was further purified with Heparin chromatography. The 1mL HiTrap Heparin HP column was connected to the AKTA pure in the cold room at 4 ℃. 2.5mL of nickel chromatography elution fraction 1-5 from Tn5 protein was injected into the injection loop and the chromatography was followed according to the Purification Protocol.
</P><H4> Tris-Glycine Polyacrylamide Gel </H4><P>In order to analyse the heparin chromatography of the Tn5 protein, a precast Tris-Glycine polyacrylamide gel was run with a 6% stacking gel and 20% separation gel according to the SDS PAGE Tris-Glycine Polyacrylamide Gel protocol.The polyacrylamide gel ran for 90min at 50volt. After the was completed, the gel was stained with SimplyBlue SafeStain and imaged using the GelDoc system. The result of the gel is shown below:
        
<B>Figure Heparin Chromatography. </B><I> : 20% SDS PAGE of Tn5 (expected size 53.3kDa). Lane 1 molecular ladder (kDa), lane 2 starting material, lane 3 flow through, lane 4 wash, lane 5 elution 1, lane 6 elution 2, lane 7 elution 3, lane 8 elution 4, lane 9 elution 5, lane 10 elution 6, lane 11 elution 7, lane 12 elution 8, lane 13 elution 9, lane 14 elution 10, lane 15 elution 11.  </I></P><P>The heparin chromatography was successful. No losses were observed in the flowthrough or wash (ane 3-4) of the run, resulting in high recoveries. The first 4 elution fractions (lane 5-8) contained some low molecular weight contaminating proteins and therefore only elution fraction 6 to 11 (lane 10 to 15) were pooled.</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#TuesdayPFJ2">Tuesday, 10/07/2018</BUTTON><DIV id="TuesdayPFJ2" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>10/07/2018</DD><DT>Operator</DT><DD>Janine Nijenhuis</DD></DL><H4>Protein Expression - <I>Inoculation &amp; Induction</I></H4><P>The upstream processing of the Tn5 continued. Inoculated 800 mL liquid media (Luria Broth, complemented with 34 μg/mL chloramphenicol) with the seed culture and grown to an OD<SUB>600</SUB> of 0.6. 
    </P><CENTER><TABLE><CAPTION class="figadpbl"><B>Table :</B> Growth represented by OD<SUB>600</SUB> over time</CAPTION><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Time</TH><TH class="tableheaderadpbl">OD<SUB>600</SUB></TH></TR><TR class="tableunevenadpbl"><TD>09:23</TD><TD>0.01</TD></TR><TR class="tableunevenadpbl"><TD>10:20</TD><TD>0.07</TD></TR><TR class="tableunevenadpbl"><TD>11:18</TD><TD>0.25</TD></TR><TR class="tableunevenadpbl"><TD>12:01</TD><TD>0.61</TD></TR></TBODY></TABLE></CENTER><P>After the culture OD<SUB>600</SUB> reached 0.61, the Protein Expression protocol was followed. After the expression is completed, the cells were transferred to Downstream Processing (DSP).
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#WednesdayPFJ2">Wednesday, 11/07/2018</BUTTON><DIV id="WednesdayPFJ2" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>11/07/2018</DD><DT>Operator</DT><DD>Janine Nijenhuis</DD></DL><H4>Protein Purification - <I>Cell lysis</I></H4><P>The downstream processing of the Tn5 protein started. Harvest the cells according to the purification protocol. The cells of Tn5 protein were resuspended in 30 mL Lysis buffer (20 mM Tris-HCl, 250 mM NaCl, 10% glycerol, pH 8).
        
    After resuspension the following additives were added:
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Additive</TH><TH class="tableheaderadpbl">Stock Concentration</TH><TH class="tableheaderadpbl"> Volume added</TH></TR><TR class="tableunevenadpbl"><TD>DTT</TD><TD>0.5M</TD><TD>120uL</TD></TR><TR class="tableunevenadpbl"><TD>EDTA-free Complete protease inhibitor tablets</TD><TD>n/a</TD><TD>1 tablet</TD></TR><TR class="tableunevenadpbl"><TD>imidazole</TD><TD>2M</TD><TD>75uL</TD></TR></TBODY></TABLE></CENTER><P> The cells were lysed and clarified according to the the purification protocol.
</P><H4> Protein Purification - <I>Nickel Affinty Chromatography</I></H4><P>1mL of 50% His-Select Ni resin, was washed according to the Purification protocol. Clarified lysate of Tn5 protein was added to the washed His-Select Ni resin and incubated with gentle mixing for 60minutes at 4 ℃. The His-Select Ni resin was washed with washing buffer (20 mM Tris-HCl, 250 mM NaCl, 20mM Imidazole, 10% glycerol, 1mM DTT, pH8), and eluded with with 6* 500μL (6CV) of elution buffer (20 mM Tris-HCl, 250 mM NaCl, 250 mM Imidazole, 10% glycerol, 1mM DTT, pH8) at 4 ℃, according to the Purification protocol. 
</P><H4> Tris-Glycine Polyacrylamide Gel </H4><P>In order to analyse the nickel affinty chromatography of the Tn5 protein, a precast Tris-Glycine polyacrylamide gel was run with a 6% stacking gel and 20% separation gel according to the SDS PAGE Tris-Glycine Polyacrylamide Gel protocol.The polyacrylamide gel ran for 90min at 50volt. After the was completed, the gel was stained with SimplyBlue SafeStain and imaged using the GelDoc system. The result of the gel is shown below:
        
<B>Figure Nickel Affinity Chromatography. </B><I> 20% SDS PAGE of Tn5 (expected size 53.3kDa) Lane 1 molecular ladder (kDa), lane 2 whole lysate, lane 3 clarified lysate, lane 4 cell debris, lane 5 not bound lysate, lane 6  flow through, lane 7 wash, lane 8 elution 1, lane 9 elution 2, lane 10 elution 3, lane 11 elution 4, lane 12 elution 5, lane 13 elution 6.</I></P><P>The nickel chromatography was not completely successful, but there is some purification of Tn5. The Tn5 did not completely bind to the nickel resin and was observed in the flowthrough and wash (lane 5-7). It is recommended to further optimize the nickel chromatography to further reduce losses, and reduce contaminating proteins due to non-specific binding to the nickel resin (lane 8-10). However for our purposes, the losses were tolerated. Elution fractions 1 to 3 were pooled (lane 8-10).</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#ThursdayPFJ2">Thursday, 12/07/2018</BUTTON><DIV id="ThursdayPFJ2" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>12/07/2018</DD><DT>Operator</DT><DD>Monique de Leeuw</DD></DL><H4> Transformation of chemically competent cells </H4><P>The dxcas9_19 plasmid was transfomred into BL21AI for dxCas9 protein production. The following transformations were carried out according to the protocol for transformation of chemically competent cells:
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Strain</TH><TH class="tableheaderadpbl">Plasmid &amp; size (bp) </TH><TH class="tableheaderadpbl">Cloning method</TH><TH class="tableheaderadpbl">Volume added (uL)</TH><TH class="tableheaderadpbl">Medium on which was plated</TH></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I>BL21AI</TD><TD>dxcas9_19 (7967bp)</TD><TD>Heat shock</TD><TD>1</TD><TD>LB chloramphenicol</TD></TR></TBODY></TABLE></CENTER><P>200 uL LB medium was added as recovery medium.
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#SundayPFJ2">Sunday, 15/07/2018</BUTTON><DIV id="SundayPFJ2" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>15/07/2018</DD><DT>Operator</DT><DD>Monique de Leeuw</DD></DL><H4>Objective</H4><P>dxCas9 was expressed and purified to produce protein for <I>in vitro</I> functionality evaluation.
    </P><H4>Protein expression - <I>Seed Culture</I></H4><P>A seed culture was made to start Upstream Processing (USP) of dxCas9 protein. A colony with dxcas9-linker-Tn5_9 strain was used for inoculation of 20mL liquid starter culture (Luria Broth, complemented with 34 μg/mL chloramphenicol) following the Liquid Starter Culture Protocol.
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV></DIV></UL></FORM><FORM><LABEL for="WeekPFJ3" class="subdropdown cadpbl">Week 3 16/07/2018 - 22/07/2018</LABEL><UL class="submenu"><DIV class="subcontent"><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#MondayPFJ3">Monday, 16/07/2018</BUTTON><DIV id="MondayPFJ3" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>16/07/2018</DD><DT>Operator</DT><DD>Janine Nijenhuis</DD></DL><H4>Objective</H4><P>Tn5 was expressed and purified to produce protein for <I>in vitro</I> functionality evaluation.
    </P><H4>Protein expression - <I>Seed Culture</I></H4><P>A seed culture was made to start Upstream Processing (USP) of Tn5. A colony with Tn5 strain was used for inoculation of 40mL liquid starter culture (Luria Broth, complemented with 34 μg/mL chloramphenicol) following the Liquid Starter Culture Protocol.
</P><DL><DT>Date</DT><DD>16/07/2018</DD><DT>Operator</DT><DD>Monique de Leeuw</DD></DL><H4>Protein Expression - <I>Inoculation &amp; Induction</I></H4><P>The upstream processing of the dxCas9 protein continued. Inoculated 1L liquid media (Luria Broth, complemented with 34 μg/mL chloramphenicol) with the seed culture and grown to an OD<SUB>600</SUB> of 2. 
    </P><CENTER><TABLE><CAPTION class="figadpbl"><B>Table :</B> Growth represented by OD<SUB>600</SUB> over time</CAPTION><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Time</TH><TH class="tableheaderadpbl">OD<SUB>600</SUB></TH></TR><TR class="tableunevenadpbl"><TD>11:00am</TD><TD>0.14</TD></TR><TR class="tableunevenadpbl"><TD>12:10am</TD><TD>0.34</TD></TR><TR class="tableunevenadpbl"><TD>1:10am</TD><TD>0.54</TD></TR></TBODY></TABLE></CENTER><P>After the culture OD<SUB>600</SUB> reached 0.54, the Protein Expression protocol was followed. After the expression is completed, the cells were transferred to Downstream Processing (DSP).
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#TuesdayPFJ3">Tuesday, 17/07/2018</BUTTON><DIV id="TuesdayPFJ3" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>17/07/2018</DD><DT>Operator</DT><DD>Janine Nijenhuis</DD></DL><H4>Protein Expression - <I>Inoculation &amp; Induction</I></H4><P>The upstream processing of the Tn5 continued. Inoculated 800 mL liquid media (Luria Broth, complemented with 34 μg/mL chloramphenicol) with the seed culture and grown to an OD<SUB>600</SUB> of 0.6. 
    </P><CENTER><TABLE><CAPTION class="figadpbl"><B>Table :</B> Growth represented by OD<SUB>600</SUB> over time</CAPTION><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Time</TH><TH class="tableheaderadpbl">OD<SUB>600</SUB></TH></TR><TR class="tableunevenadpbl"><TD>12:20</TD><TD>0.23</TD></TR><TR class="tableunevenadpbl"><TD>13:05</TD><TD>0.25</TD></TR><TR class="tableunevenadpbl"><TD>14:10</TD><TD>0.64</TD></TR></TBODY></TABLE></CENTER><P>After the culture OD<SUB>600</SUB> reached 0.64, the Protein Expression protocol was followed. After the expression is completed, the cells were transferred to Downstream Processing (DSP).
</P><DL><DT>Date</DT><DD>24/08/2018</DD><DT>Operator</DT><DD>Monique de Leeuw</DD></DL><H4>Protein Purification - <I>Cell lysis</I></H4><P>The downstream processing of the dxCas9 protein started. Harvest the cells according to the purification protocol. 9.4g of cells of dxCas9 protein were resuspended in Lysis buffer (20 mM Tris-HCl, 250 mM NaCl, 10% glycerol, pH 8).
        
    After resuspension the following additives were added:
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Additive</TH><TH class="tableheaderadpbl">Stock Concentration</TH><TH class="tableheaderadpbl"> Volume added</TH></TR><TR class="tableunevenadpbl"><TD>DTT</TD><TD>0.5M</TD><TD>131.6uL</TD></TR><TR class="tableunevenadpbl"><TD>EDTA-free Complete protease inhibitor tablets</TD><TD>n/a</TD><TD>1tablets</TD></TR><TR class="tableunevenadpbl"><TD>imidazole</TD><TD>2M</TD><TD>658uL</TD></TR></TBODY></TABLE></CENTER><P> The cells were lysed and clarified according to the the purification protocol.
</P><H4> Tris-Glycine Polyacrylamide Gel </H4><P>In order to analyse the expression of the dxCas9 protein, a precast Tris-Glycine polyacrylamide gel was run with a 6% stacking gel and 12% separation gel according to the SDS PAGE Tris-Glycine Polyacrylamide Gel protocol. The polyacrylamide gel ran for 90min at 50volt. After the was completed, the gel was stained with SimplyBlue SafeStain and imaged using the GelDoc system. The result of the gel is shown below:
        
<B>Figure dxCas9 Expression: </B><I>12% SDS PAGE of dxCas9 (expected size 158.3kDa). Lane 1 molecular ladder (kDa), Lane 1 molecular ladder (kDa), lane 2 whole lysate, lane 3 clarified lysate.  </I></P><H4> Protein Purification - <I>Nickel Affinty Chromatography</I></H4><P>1mL of 50% His-Select Ni resin, was washed according to the Purification protocol. 51.2mL clarified lysate of dxCas9 protein was added to the washed His-Select Ni resin and incubated with gentle mixing for 60minutes at 4 ℃. The His-Select Ni resin was washed with washing buffer (20 mM Tris-HCl, 250 mM NaCl, 20mM Imidazole, 10% glycerol, 1mM DTT, pH8), and eluded with with 6* 500μL (6CV) of elution buffer (20 mM Tris-HCl, 250 mM NaCl, 250 mM Imidazole, 10% glycerol, 1mM DTT, pH8) at 4 ℃, according to the Purification protocol. 
</P><H4> Tris-Glycine Polyacrylamide Gel </H4><P>In order to analyse the nickel affinty chromatography of the dxCas9 protein, a precast Tris-Glycine polyacrylamide gel was run with a 6% stacking gel and 8% separation gel according to the SDS PAGE Tris-Glycine Polyacrylamide Gel protocol.The polyacrylamide gel ran for 90min at 50volt. After the was completed, the gel was stained with SimplyBlue SafeStain and imaged using the GelDoc system. The result of the gel is shown below:
        
<B>Figure Nickel Affinity Chromatography. </B><I> 12% SDS PAGE of dxCas9 (expected size 158.3kDa) Lane 1 molecular ladder (kDa), lane 2 whole lysate, lane 3 clarified lysate, lane 4 flow through 1, lane 5 flow through 2, lane 6  flow through 3, lane 7 wash 1, lane 8 elution 1, lane 9 elution 2, lane 10 elution 3, lane 11 elution 4, lane 12 elution 5, lane 13 elution 6, lane 14 elution 1.  
</I></P><P>The nickel chromatography was not successful. The dxCas9 did not bind to the nickel resin and was observed in the flowthrough and wash (lane 4-7). It is recommended to optimize the nickel chromatography by reducing the amount of imidazole in the binding buffer to 5mM for the next experiment.</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#WednesdayPFJ3">Wednesday, 18/07/2018</BUTTON><DIV id="WednesdayPFJ3" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>18/07/2018</DD><DT>Operator</DT><DD>Janine Nijenhuis</DD></DL><H4>Protein Purification - <I>Cell lysis</I></H4><P>The downstream processing of the Tn5 protein started. Harvest the cells according to the purification protocol. The cells of Tn5 protein were resuspended in 30 mL of Lysis buffer (20 mM Tris-HCl, 250 mM NaCl, 10% glycerol, pH 8).
        
    After resuspension the following additives were added:
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Additive</TH><TH class="tableheaderadpbl">Stock Concentration</TH><TH class="tableheaderadpbl"> Volume added</TH></TR><TR class="tableunevenadpbl"><TD>DTT</TD><TD>0.5M</TD><TD>60uL</TD></TR><TR class="tableunevenadpbl"><TD>EDTA-free Complete protease inhibitor tablets</TD><TD>n/a</TD><TD>1 tablet</TD></TR><TR class="tableunevenadpbl"><TD>imidazole</TD><TD>2M</TD><TD>75uL</TD></TR></TBODY></TABLE></CENTER><P> The cells were lysed and clarified according to the the purification protocol.
</P><H4> Protein Purification - <I>Nickel Affinty Chromatography</I></H4><P>1mL of 50% His-Select Ni resin, was washed according to the Purification protocol. Clarified lysate of Tn5 protein was added to the washed His-Select Ni resin and incubated with gentle mixing for 60minutes at 4 ℃. The His-Select Ni resin was washed with washing buffer (20 mM Tris-HCl, 250 mM NaCl, 20mM Imidazole, 10% glycerol, 1mM DTT, pH8), and eluded with with 6* 500μL (6CV) of elution buffer (20 mM Tris-HCl, 250 mM NaCl, 250 mM Imidazole, 10% glycerol, 1mM DTT, pH8) at 4 ℃, according to the Purification protocol. 
</P><H4> Tris-Glycine Polyacrylamide Gel </H4><P>In order to analyse the nickel affinty chromatography of the Tn5 protein, a precast Tris-Glycine polyacrylamide gel was run with a 6% stacking gel and 8% separation gel according to the SDS PAGE Tris-Glycine Polyacrylamide Gel protocol.The polyacrylamide gel ran for 90min at 50volt. After the was completed, the gel was stained with SimplyBlue SafeStain. Sadly no picture was taken. However, the purification was succesfull.
    
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV></DIV></UL></FORM><H3 class="adpbl">August</H3><FORM><LABEL for="WeekPFA2" class="subdropdown cadpbl">Week 2 06/08/2018 - 12/08/2018</LABEL><UL class="submenu"><DIV class="subcontent"><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#MondayPFA2">Monday, 06/08/2018</BUTTON><DIV id="MondayPFA2" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>06/08/2018</DD><DT>Operator</DT><DD>Monique de Leeuw</DD></DL><H4> Transformation of chemically competent cells </H4><P>The dxcas9_19 plasmid was transfomred into BL21AI for dxCas9 protein production. The following transformations were carried out according to the protocol for transformation of chemically competent cells:
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Strain</TH><TH class="tableheaderadpbl">Plasmid &amp; size (bp) </TH><TH class="tableheaderadpbl">Cloning method</TH><TH class="tableheaderadpbl">Volume added (uL)</TH><TH class="tableheaderadpbl">Medium on which was plated</TH></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I>BL21AI</TD><TD>dxcas9_19 (7967bp)</TD><TD>Heat shock</TD><TD>1</TD><TD>LB chloramphenicol</TD></TR></TBODY></TABLE></CENTER><P>200 uL LB medium was added as recovery medium.
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#TuesdayPFA2">Tuesday, 07/08/2018</BUTTON><DIV id="TuesdayPFA2" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>07/08/2018</DD><DT>Operator</DT><DD>Monique de Leeuw</DD></DL><H4>Objective</H4><P>dxCas9 was expressed and purified to produce protein for <I>in vitro</I> functionality evaluation.
    </P><H4>Protein expression - <I>Seed Culture</I></H4><P>A seed culture was made to start Upstream Processing (USP) of dxCas9 protein. A colony with dxcas9-linker-Tn5_9 strain was used for inoculation of 20mL liquid starter culture (Luria Broth, complemented with 34 μg/mL chloramphenicol) following the Liquid Starter Culture Protocol.
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#WednesdayPFA2">Wednesday, 08/08/2018</BUTTON><DIV id="WednesdayPFA2" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>08/08/2018</DD><DT>Operator</DT><DD>Janine Nijenhuis</DD></DL><H4>Objective</H4><P>dxCas9-Tn5 was expressed and purified to produce protein for <I>in vitro</I> functionality evaluation.
    </P><H4>Protein expression - <I>Seed Culture</I></H4><P>A seed culture was made to start Upstream Processing (USP) of dxCas9-Tn5. A colony with dxCas9-Tn5_9 strain was used for inoculation of 40mL liquid starter culture (Luria Broth, complemented with 34 μg/mL chloramphenicol) following the Liquid Starter Culture Protocol.
</P><DL><DT>Date</DT><DD>08/08/2018</DD><DT>Operator</DT><DD>Monique de Leeuw</DD></DL><H4>Protein Expression - <I>Inoculation &amp; Induction</I></H4><P>The upstream processing of the dxCas9 protein continued. Inoculated 1L liquid media (Luria Broth, complemented with 34 μg/mL chloramphenicol) with the seed culture and grown to an OD<SUB>600</SUB> of 2. 
    </P><CENTER><TABLE><CAPTION class="figadpbl"><B>Table :</B> Growth represented by OD<SUB>600</SUB> over time</CAPTION><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Time</TH><TH class="tableheaderadpbl">OD<SUB>600</SUB></TH></TR><TR class="tableunevenadpbl"><TD>10:30am</TD><TD>0.20</TD></TR><TR class="tableunevenadpbl"><TD>11:30am</TD><TD>0.38</TD></TR><TR class="tableunevenadpbl"><TD>12:00am</TD><TD>0.52</TD></TR></TBODY></TABLE></CENTER><P>After the culture OD<SUB>600</SUB> reached 0.52, the Protein Expression protocol was followed. After the expression is completed, the cells were transferred to Downstream Processing (DSP).
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#ThursdayPFA2">Thursday, 09/08/2018</BUTTON><DIV id="ThursdayPFA2" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>09/08/2018</DD><DT>Operator</DT><DD>Janine Nijenhuis</DD></DL><H4>Protein Expression - <I>Inoculation &amp; Induction</I></H4><P>The upstream processing of the dxCas9-Tn5 continued. Inoculated 2L liquid media (Luria Broth, complemented with 34 μg/mL chloramphenicol) with the seed culture and grown to an OD<SUB>600</SUB> of 0.6. 
    </P><CENTER><TABLE><CAPTION class="figadpbl"><B>Table :</B> Growth represented by OD<SUB>600</SUB> over time</CAPTION><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Time</TH><TH class="tableheaderadpbl">OD<SUB>600</SUB></TH></TR><TR class="tableunevenadpbl"><TD>12:50</TD><TD>0.23</TD></TR><TR class="tableunevenadpbl"><TD>13:30</TD><TD>0.25</TD></TR><TR class="tableunevenadpbl"><TD>14:10</TD><TD>0.47</TD></TR><TR class="tableunevenadpbl"><TD>14:20</TD><TD>0.58</TD></TR></TBODY></TABLE></CENTER><P>After the culture OD<SUB>600</SUB> reached 0.58, the Protein Expression protocol was followed. After the expression is completed, the cells were transferred to Downstream Processing (DSP).
</P><DL><DT>Date</DT><DD>09/08/2018</DD><DT>Operator</DT><DD>Monique de Leeuw</DD></DL><H4>Protein Purification - <I>Cell lysis</I></H4><P>The downstream processing of the dxCas9 protein started. Harvest the cells according to the purification protocol. 4.4g of cells of dxCas9 protein were resuspended in Lysis buffer (20 mM Tris-HCl, 250 mM NaCl, 10% glycerol, pH 8).
        
    After resuspension the following additives were added:
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Additive</TH><TH class="tableheaderadpbl">Stock Concentration</TH><TH class="tableheaderadpbl"> Volume added</TH></TR><TR class="tableunevenadpbl"><TD>DTT</TD><TD>0.5M</TD><TD>61.6uL</TD></TR><TR class="tableunevenadpbl"><TD>EDTA-free Complete protease inhibitor tablets</TD><TD>n/a</TD><TD>1tablets</TD></TR><TR class="tableunevenadpbl"><TD>imidazole</TD><TD>2M</TD><TD>77uL</TD></TR></TBODY></TABLE></CENTER><P> The cells were lysed and clarified according to the the purification protocol.
</P><H4> Tris-Glycine Polyacrylamide Gel </H4><P>In order to analyse the expression of the dxCas9 protein, a precast Tris-Glycine polyacrylamide gel was run with a 6% stacking gel and 12% separation gel according to the SDS PAGE Tris-Glycine Polyacrylamide Gel protocol. The polyacrylamide gel ran for 90min at 50volt. After the was completed, the gel was stained with SimplyBlue SafeStain and imaged using the GelDoc system. The result of the gel is shown below:
        
<B>Figure dxCas9 Expression: </B><I>12% SDS PAGE of dxCas9 (expected size 158.3kDa). Lane 1 molecular ladder (kDa), lane 2 pre-induced, lane 3 whole lysate, lane 4 clarified lysate, lane 5 cell debris.  </I></P><H4> Protein Purification - <I>Nickel Affinty Chromatography</I></H4><P>1mL of 50% His-Select Ni resin, was washed according to the Purification protocol. 29mL clarified lysate of dxCas9 protein was added to the washed His-Select Ni resin and incubated with gentle mixing for 60minutes at 4 ℃. The His-Select Ni resin was washed with washing buffer (20 mM Tris-HCl, 250 mM NaCl, 5mM Imidazole, 10% glycerol, 1mM DTT, pH8), and eluded with with 6* 500μL (6CV) of elution buffer (20 mM Tris-HCl, 250 mM NaCl, 250 mM Imidazole, 10% glycerol, 1mM DTT, pH8) at 4 ℃, according to the Purification protocol. 
</P><H4> Tris-Glycine Polyacrylamide Gel </H4><P>In order to analyse the nickel affinty chromatography of the dxCas9 protein, a precast Tris-Glycine polyacrylamide gel was run with a 6% stacking gel and 8% separation gel according to the SDS PAGE Tris-Glycine Polyacrylamide Gel protocol.The polyacrylamide gel ran for 90min at 50volt. After the was completed, the gel was stained with SimplyBlue SafeStain and imaged using the GelDoc system. The result of the gel is shown below:
        
<B>Figure Nickel Affinity Chromatography. </B><I> 12% SDS PAGE of dxCas9 (expected size 158.3kDa) Lane 1 molecular ladder (kDa), lane 2 pre-induction, lane 3 whole lysate, lane 4 clarified lysate, lane 5 cell debris, lane 6  flow through, lane 7 wash 1, lane 8 wash 2, lane 9 elution 1, lane 10 elution 2, lane 11 elution 3, lane 12 elution 4, lane 13 elution 5, lane 14 elution 6.</I></P><P>The nickel chromatography was not completely successful, but improved significantly after an the concentration of imidazole in the binding buffer was reduced to 5mM. The dxCas9 did not completely bind to the nickel resin and was observed in the flowthrough and wash (lane 6-8). It is recommended to further optimize the nickel chromatography to further reduce losses, and reduce contaminating proteins due to non-specific binding to the nickel resin (lane 9-10). However for our purposes, the losses were tolerated, and only elution fractions 3 to 6 were pooled (lane 11-14). Additionally, a double band for dxCas9 is observed. The purification will be continued however, we will look deeper into this observation. Is this double band due to degradation, or some other interaction with the dxcas9? We are working with technicians on performing mass spectrometry. </P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#FridayPFA2">Friday, 10/07/2018</BUTTON><DIV id="FridayPFA2" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>10/08/2018</DD><DT>Operator</DT><DD>Janine Nijenhuis</DD></DL><H4>Protein Purification - <I>Cell lysis</I></H4><P>The downstream processing of the dxCas9-Tn5 protein started. Harvest the cells according to the purification protocol. The cells of dxCas9-Tn5 protein were resuspended in 30 mL of Lysis buffer (20 mM Tris-HCl, 250 mM NaCl, 10% glycerol, pH 8).
        
    After resuspension the following additives were added:
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Additive</TH><TH class="tableheaderadpbl">Stock Concentration</TH><TH class="tableheaderadpbl"> Volume added</TH></TR><TR class="tableunevenadpbl"><TD>DTT</TD><TD>0.5M</TD><TD>60uL</TD></TR><TR class="tableunevenadpbl"><TD>EDTA-free Complete protease inhibitor tablets</TD><TD>n/a</TD><TD>1 tablet</TD></TR><TR class="tableunevenadpbl"><TD>imidazole</TD><TD>2M</TD><TD>75uL</TD></TR></TBODY></TABLE></CENTER><P> The cells were lysed and clarified according to the the purification protocol.
</P><H4> Protein Purification - <I>Nickel Affinty Chromatography</I></H4><P>1mL of 50% His-Select Ni resin, was washed according to the Purification protocol. Clarified lysate of dxCas9-Tn5 protein was added to the washed His-Select Ni resin and incubated with gentle mixing for 60minutes at 4 ℃. The His-Select Ni resin was washed with washing buffer (20 mM Tris-HCl, 250 mM NaCl, 5mM Imidazole, 10% glycerol, 1mM DTT, pH8), and eluded with with 6* 500μL (6CV) of elution buffer (20 mM Tris-HCl, 250 mM NaCl, 250 mM Imidazole, 10% glycerol, 1mM DTT, pH8) at 4 ℃, according to the Purification protocol. 
</P><H4> Tris-Glycine Polyacrylamide Gel </H4><P>In order to analyse the nickel affinty chromatography of the dxCas9-Tn5 protein, a precast Tris-Glycine polyacrylamide gel was run with a 6% stacking gel and 8% separation gel according to the SDS PAGE Tris-Glycine Polyacrylamide Gel protocol.The polyacrylamide gel ran for 90min at 50volt. After the was completed, the gel was stained with SimplyBlue SafeStain. Sadly no picture was taken. The results showed that the fusion protein would not bind to the column material. It was decided to do another purification, but with a lower concentration of Imidazole.
</P><DL><DT>Date</DT><DD>10/08/2018</DD><DT>Operator</DT><DD>Monique de Leeuw</DD></DL><H4> Protein Purification - <I>Heparin Chromatography</I></H4><P>The dxCas9 protein was further purified with Heparin chromatography. The 1mL HiTrap Heparin HP column was connected to the AKTA pure in the cold room at 4 ℃. 2mL of nickel chromatography elution fraction 3-6 from dxCas9 protein was injected into the injection loop and the chromatography was followed according to the Purification Protocol.
</P><H4> Tris-Glycine Polyacrylamide Gel </H4><P>In order to analyse the heparin chromatography of the dxCas9 protein, a precast Tris-Glycine polyacrylamide gel was run with a 6% stacking gel and 12% separation gel according to the SDS PAGE Tris-Glycine Polyacrylamide Gel protocol.The polyacrylamide gel ran for 90min at 50volt. After the was completed, the gel was stained with SimplyBlue SafeStain and imaged using the GelDoc system. The result of the gel is shown below:
        
<B>Figure Heparin Chromatography. </B><I> 12% SDS PAGE of dxCas9 (expected size 158.3kDa). Lane 1 molecular ladder (kDa), lane 2 starting material, lane 3 flow through, lane 4 wash, lane 5 elution 1, lane 6 elution 2, lane 7 elution 3, lane 8 elution 4, lane 9 elution 5, lane 10 elution 6, lane 11 elution 7, lane 12 elution 8, lane 13 elution 9, lane 14 elution 10, lane 15 elution 11. </I></P><P>The heparin chromatography was successful. The protein recoveries were high, with no losses in the flowthrough and wash (lane 3-4). The first 4 elution fractions (lane 5-8) contained some low molecular weight contaminating proteins, as a result only elution fraction 5 to 10 (lane 9 to 14) were pooled. Again the double band for dxCas9 was observed on the 12% SDS PAGE, which we have seen with all purification.</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#SaturdayPFA2">Saturday, 11/08/2018</BUTTON><DIV id="SaturdayPFA2" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>11/08/2018</DD><DT>Operator</DT><DD>Monique de Leeuw</DD></DL><H4>Protein Purification - <EM>Dialysis</EM></H4><P>Finally the dxCas9 protein is dialysed and concentrated. 3mL of sample (Heparin chromatography elution fraction 5-10 of dxCas9 protein) were dialysed according to the purification protocol with dialysis buffer (20 mM Tris-HCl, 100 mM KCl, 5% glycerol, 1mM beta-Mercaptoethanol, pH 7.5). After dialysis, the sample of dxCas9 protein was stored at -20℃, according to the purification protocol.</P><P>In order to analyse the concentrated dxCas9 protein, a precast Tris-Glycine polyacrylamide gel was run with a 6% stacking gel and 8% separation gel according to the SDS PAGE Tris-Glycine Polyacrylamide Gel protocol. The polyacrylamide gel ran for 90min at 50volt. After the was completed, the gel was stained with SimplyBlue SafeStain and imaged using the GelDoc system. The result of the gel is shown below:
        
<B>Figure Final protein: </B><I>8% SDS PAGE of dxCas9 (expected size 158.3kDa). Lane 1 molecular ladder (kDa), lane 2 final dxCas9 post dialysis.</I></P><P> Note the dxCas9 is only one band on the 8% PAGE. The purification will be continued however, we will look deeper into this observation. What causes this double band? Is there degradation, or some other interaction with the dxcas9? Also what happened during the final dialysis step to remove the double band? Trace amounts of contaminating background is observed, however dxCas9 is significantly more in quantity and the background was tolerated for functionality testing. The final concentration of the solution was determined according to the BCA protocol on our wiki and resulted in a concentration of 0.46mg/mL or 2.88µM. The </P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#SundayPFA2">Sunday, 12/08/2018</BUTTON><DIV id="SundayPFA2" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>12/08/2018</DD><DT>Operator</DT><DD>Janine Nijenhuis</DD></DL><H4>Objective</H4><P>dxCas9-Tn5 was expressed and purified to produce protein for <I>in vitro</I> functionality evaluation.
    </P><H4>Protein expression - <I>Seed Culture</I></H4><P>A seed culture was made to start Upstream Processing (USP) of dxCas9-Tn5. Two colonies with dxCas9-Tn5_9 strain were used for inoculation of two 20mL liquid starter culture (Luria Broth, complemented with 34 μg/mL chloramphenicol) following the Liquid Starter Culture Protocol.
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV></DIV></UL></FORM><FORM><LABEL for="WeekPFA3" class="subdropdown cadpbl">Week 3 13/08/2018 - 19/08/2018</LABEL><UL class="submenu"><DIV class="subcontent"><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#MondayPFA3">Monday, 13/08/2018</BUTTON><DIV id="MondayPFA3" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>13/08/2018</DD><DT>Operator</DT><DD>Janine Nijenhuis</DD></DL><H4>Protein Expression - <I>Inoculation &amp; Induction</I></H4><P>The upstream processing of the dxCas9-Tn5 continued. Inoculated 2L liquid media (Luria Broth, complemented with 34 μg/mL chloramphenicol) with the seed culture and grown to an OD<SUB>600</SUB> of 0.6. 
    </P><CENTER><TABLE><CAPTION class="figadpbl"><B>Table :</B> Growth represented by OD<SUB>600</SUB> over time</CAPTION><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Time</TH><TH class="tableheaderadpbl">Col 1 OD<SUB>600</SUB></TH><TH class="tableheaderadpbl">Col 2 OD<SUB>600</SUB></TH></TR><TR class="tableunevenadpbl"><TD>12:00</TD><TD>0.06</TD><TD>0.05</TD></TR><TR class="tableunevenadpbl"><TD>12:50</TD><TD>0.08</TD><TD>0.08</TD></TR><TR class="tableunevenadpbl"><TD>13:50</TD><TD>0.28</TD><TD>0.30</TD></TR><TR class="tableunevenadpbl"><TD>14:30</TD><TD>0.56</TD><TD>0.56</TD></TR></TBODY></TABLE></CENTER><P>After both cultures OD<SUB>600</SUB> reached 0.56, the Protein Expression protocol was followed. After the expression is completed, the cells were transferred to Downstream Processing (DSP).
</P><DL><DT>Date</DT><DD>13/08/2018</DD><DT>Operator</DT><DD>Monique de Leeuw</DD></DL><H4> Transformation of chemically competent cells </H4><P>The dxcas9_19 plasmid was transfomred into BL21AI for dxCas9 protein production. The following transformations were carried out according to the protocol for transformation of chemically competent cells:
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Strain</TH><TH class="tableheaderadpbl">Plasmid &amp; size (bp) </TH><TH class="tableheaderadpbl">Cloning method</TH><TH class="tableheaderadpbl">Volume added (uL)</TH><TH class="tableheaderadpbl">Medium on which was plated</TH></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I>BL21AI</TD><TD>dxcas9_19 (7967bp)</TD><TD>Heat shock</TD><TD>1</TD><TD>LB chloramphenicol</TD></TR></TBODY></TABLE></CENTER><P>200 uL LB medium was added as recovery medium.
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#TuesdayPFA3">Tuesday, 14/08/2018</BUTTON><DIV id="TuesdayPFA3" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>14/08/2018</DD><DT>Operator</DT><DD>Janine Nijenhuis</DD></DL><H4>Protein Purification - <I>Cell lysis</I></H4><P>The downstream processing of the dxCas9-Tn5 protein started. Harvest the cells of the cultures of col 1 and col 2 according to the purification protocol. The cells of both cultures with dxCas9-Tn5 protein were resuspended in 24 mL of Lysis buffer (20 mM Tris-HCl, 250 mM NaCl, 10% glycerol, pH 8).
        
    After resuspension the following additives were added for both cultures:
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Additive</TH><TH class="tableheaderadpbl">Stock Concentration</TH><TH class="tableheaderadpbl"> Volume added</TH></TR><TR class="tableunevenadpbl"><TD>DTT</TD><TD>0.5M</TD><TD>60uL</TD></TR><TR class="tableunevenadpbl"><TD>EDTA-free Complete protease inhibitor tablets</TD><TD>n/a</TD><TD>1 tablet</TD></TR><TR class="tableunevenadpbl"><TD>imidazole</TD><TD>2M</TD><TD>75uL</TD></TR></TBODY></TABLE></CENTER><P> The cells were lysed and clarified according to the the purification protocol.
</P><H4> Protein Purification - <I>Nickel Affinty Chromatography</I></H4><P>Two times 1mL of 50% His-Select Ni resin, was washed according to the Purification protocol. Clarified lysate of dxCas9-Tn5 protein of each colony was added to one of the washed His-Select Ni resin and incubated with gentle mixing for 60minutes at 4 ℃. The His-Select Ni resins were washed with washing buffer (20 mM Tris-HCl, 250 mM NaCl, 10% glycerol, 1mM DTT, pH8) and 0.5mM Imidazole for Col 1 and 0mM Imidazole for Col 2, and eluded with with 6* 500μL (6CV) of elution buffer (20 mM Tris-HCl, 250 mM NaCl, 250 mM Imidazole, 10% glycerol, 1mM DTT, pH8) at 4 ℃, according to the Purification protocol. 
</P><H4> Tris-Glycine Polyacrylamide Gel </H4><P>In order to analyse the nickel affinty chromatography of the dxCas9-Tn5 protein of both colonies, two precast Tris-Glycine polyacrylamide gel were run with a 6% stacking gel and 8% separation gel according to the SDS PAGE Tris-Glycine Polyacrylamide Gel protocol.The polyacrylamide gel ran for 90min at 50volt. After the was completed, the gel was stained with SimplyBlue SafeStain and imaged using the GelDoc system. The result of the gel is shown below:
        
<B>Figure Nickel Affinity Chromatography Col 1. </B><I> 8% SDS PAGE of dxCas9-Tn5 (expected size 214.7kDa). Lane 1 high range molecular ladder (kDa), lane 2 whole lysate, lane 3 clarified lysate, lane 4 cell debris, lane 5 flow through, lane 6 wash, lane 7 elution 1, lane 8 elution 2, lane 9 elution 3, lane 10 elution 4, lane 11 elution 5, lane 12 elution 6.</I><B>Figure Nickel Affinity Chromatography Col 2. </B><I> 8% SDS PAGE of dxCas9-Tn5 (expected size 214.7kDa). Lane 1 high range molecular ladder (kDa), lane 2 whole lysate, lane 3 clarified lysate, lane 4 cell debris, lane 5 flow through, lane 6 wash, lane 7 elution 1, lane 8 elution 2, lane 9 elution 3, lane 10 elution 4, lane 11 elution 5, lane 12 elution 6, lane 13 cell debris.</I></P><P>As the two figures show, even with decreasing the concentration of Imidazole, no purification of dxCas9-Tn5 happened. The fusion protein broke apart. This can already be seen in the samples before purification (figure col 1 lane 2-5, figure col 2 lane 2-5). The conclusion is that the fusion protein probably breaks at the linke, due to two bands at approximatly the same height as dxCas9 and Tn5. Another purification will be done, now with a gradient elution.</P><DL><DT>Date</DT><DD>14/08/2018</DD><DT>Operator</DT><DD>Monique de Leeuw</DD></DL><H4>Objective</H4><P>dxCas9 was expressed and purified to produce samples for mass spectrometry and other analysis to identify why there are two bands for the dxCas9 protein.
    </P><H4>Protein expression - <I>Seed Culture</I></H4><P>A seed culture was made to start Upstream Processing (USP) of dxCas9 protein. A colony with dxcas9-linker-Tn5_9 strain was used for inoculation of 20mL liquid starter culture (Luria Broth, complemented with 34 μg/mL chloramphenicol) following the Liquid Starter Culture Protocol.
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#WednesdayPFA3">Wednesday, 15/08/2018</BUTTON><DIV id="WednesdayPFA3" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>15/08/2018</DD><DT>Operator</DT><DD>Monique de Leeuw</DD></DL><H4>Protein Expression - <I>Inoculation &amp; Induction</I></H4><P>The upstream processing of the dxCas9 protein continued. Inoculated 12L liquid media (Luria Broth, complemented with 34 μg/mL chloramphenicol) with the seed culture and grown to an OD<SUB>600</SUB> of 2. 
    </P><CENTER><TABLE><CAPTION class="figadpbl"><B>Table :</B> Growth represented by OD<SUB>600</SUB> over time</CAPTION><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Time</TH><TH class="tableheaderadpbl">OD<SUB>600</SUB></TH></TR><TR class="tableunevenadpbl"><TD>10:30am</TD><TD>0.16</TD></TR><TR class="tableunevenadpbl"><TD>11:30am</TD><TD>0.34</TD></TR><TR class="tableunevenadpbl"><TD>12:00am</TD><TD>0.56</TD></TR></TBODY></TABLE></CENTER><P>After the culture OD<SUB>600</SUB> reached 0.56, the Protein Expression protocol was followed. After the expression is completed, the cells were transferred to Downstream Processing (DSP).
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#ThursdayPFA3">Thursday, 16/08/2018</BUTTON><DIV id="ThursdayPFA3" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>16/08/2018</DD><DT>Operator</DT><DD>Monique de Leeuw</DD></DL><H4>Protein Purification - <I>Cell lysis</I></H4><P>The downstream processing of the dxCas9 protein started. Harvest the cells according to the purification protocol. 8.4g of cells of dxCas9 protein were resuspended in Lysis buffer (20 mM Tris-HCl, 250 mM NaCl, 10% glycerol, pH 8).
        
    After resuspension the following additives were added:
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Additive</TH><TH class="tableheaderadpbl">Stock Concentration</TH><TH class="tableheaderadpbl"> Volume added</TH></TR><TR class="tableunevenadpbl"><TD>DTT</TD><TD>0.5M</TD><TD>117.6uL</TD></TR><TR class="tableunevenadpbl"><TD>EDTA-free Complete protease inhibitor tablets</TD><TD>n/a</TD><TD>1tablets</TD></TR><TR class="tableunevenadpbl"><TD>imidazole</TD><TD>2M</TD><TD>147uL</TD></TR></TBODY></TABLE></CENTER><P> The cells were lysed and clarified according to the the purification protocol.
</P><H4> Tris-Glycine Polyacrylamide Gel </H4><P>In order to analyse the expression of the dxCas9 protein, a precast Tris-Glycine polyacrylamide gel was run with a 6% stacking gel and 12% separation gel according to the SDS PAGE Tris-Glycine Polyacrylamide Gel protocol. The polyacrylamide gel ran for 90min at 50volt. After the was completed, the gel was stained with SimplyBlue SafeStain and imaged using the GelDoc system. The result of the gel is shown below:
        
<B>Figure dxCas9 Expression: </B><I>12% SDS PAGE of dxCas9 (expected size 158.3kDa). Lane 1 molecular ladder (kDa), lane 2 pre-induction, lane 3 whole lysate, lane 4 clarified lysate.  </I></P><H4> Protein Purification - <I>Nickel Affinty Chromatography</I></H4><P>1mL of 50% His-Select Ni resin, was washed according to the Purification protocol. 49mL clarified lysate of dxCas9 protein was added to the washed His-Select Ni resin and incubated with gentle mixing for 60minutes at 4 ℃. The His-Select Ni resin was washed with washing buffer (20 mM Tris-HCl, 250 mM NaCl, 5mM Imidazole, 10% glycerol, 1mM DTT, pH8), and eluded with with 6* 500μL (6CV) of elution buffer (20 mM Tris-HCl, 250 mM NaCl, 250 mM Imidazole, 10% glycerol, 1mM DTT, pH8) at 4 ℃, according to the Purification protocol. 
</P><H4> Tris-Glycine Polyacrylamide Gel </H4><P>In order to analyse the nickel affinty chromatography of the dxCas9 protein, a precast Tris-Glycine polyacrylamide gel was run with a 6% stacking gel and 12% separation gel according to the SDS PAGE Tris-Glycine Polyacrylamide Gel protocol.The polyacrylamide gel ran for 90min at 50volt. After the was completed, the gel was stained with SimplyBlue SafeStain and imaged using the GelDoc system. The result of the gel is shown below:
        
<B>Figure Nickel Affinity Chromatography. </B><I> 12% SDS PAGE of dxCas9 (expected size 158.3kDa) Lane 1 molecular ladder (kDa), lane 2 pre-induction, lane 3 whole lysate, lane 4 clarified lysate, lane 5 flow through, lane 6  elution 1, lane 7 elution 2, lane 8 elution 3.</I></P><P>The nickel chromatography was reproducible, and again a double band for dxCas9 is observed. One at the expected weight, and one slightly higher. It was unclear why the dxCas9 would run higher. As a following step, elution fraction 1 (lane 6) was run on a 6% SDS PAGE gel under different denaturing conditions for better resolution.</P><H4> Tris-Glycine Polyacrylamide Gel </H4><P>In order to analyse the degradation of the dxCas9 protein, a precast Tris-Glycine polyacrylamide gel was run with a 6% stacking gel and 6% separation gel according to the SDS PAGE Tris-Glycine Polyacrylamide Gel protocol.The polyacrylamide gel ran for 90min at 50volt. After the was completed, the gel was stained with SimplyBlue SafeStain and imaged using the GelDoc system. The result of the gel is shown below:
        
<B>Figure Different sample gel preparation conditions. </B><I> 12% SDS PAGE of dxCas9 (expected size 158.3kDa)  Lane 1 molecular ladder (kDa), lane 2 95 deg C 60min, lane 3 95 deg C 30min, lane 4 95 deg C 15min, lane 5 95 deg C 5min, lane 75 deg C 7min, lane 7 No boiling, lane 8 molecular ladder (kDa).</I></P><P>  On the 6% tris-glycine SDS PAGE a single band for dxCas9 was observed for all conditions, suggesting that the double band might be some artifact of the 12% SDS PAGE. To further confirm these results, both bands from the 12% gel were excised and sent for mass spectrometry. The mass spectrometry confirmed that both bands were full length dxCas9 with both the N and C terminal intact. This concluded that there was not any protease digestion and that the results are potentially due to a gel artifact. As a result, the purification will continue.</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#SundayPFA3">Sunday, 19/08/2018</BUTTON><DIV id="SundayPFA3" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>19/08/2018</DD><DT>Operator</DT><DD>Janine Nijenhuis</DD></DL><H4>Objective</H4><P>dxCas9-Tn5 was expressed and purified to produce protein for <I>in vitro</I> functionality evaluation.
    </P><H4>Protein expression - <I>Seed Culture</I></H4><P>A seed culture was made to start Upstream Processing (USP) of dxCas9-Tn5. A colony with dxCas9-Tn5_9 strain was used for inoculation of 40mL liquid starter culture (Luria Broth, complemented with 34 μg/mL chloramphenicol) following the Liquid Starter Culture Protocol.
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV></DIV></UL></FORM><FORM><LABEL for="WeekPFA4" class="subdropdown cadpbl">Week 4 20/08/2018 - 26/08/2018</LABEL><UL class="submenu"><DIV class="subcontent"><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#MondayPFA4">Monday, 20/08/2018</BUTTON><DIV id="MondayPFA4" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>20/08/2018</DD><DT>Operator</DT><DD>Janine Nijenhuis</DD></DL><H4>Protein Expression - <I>Inoculation &amp; Induction</I></H4><P>The upstream processing of the dxCas9-Tn5 continued. Inoculated 2L liquid media (Luria Broth, complemented with 34 μg/mL chloramphenicol) with the seed culture and grown to an OD<SUB>600</SUB> of 0.3. 
    </P><CENTER><TABLE><CAPTION class="figadpbl"><B>Table :</B> Growth represented by OD<SUB>600</SUB> over time</CAPTION><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Time</TH><TH class="tableheaderadpbl">OD<SUB>600</SUB></TH></TR><TR class="tableunevenadpbl"><TD>11:00</TD><TD>0.06</TD></TR><TR class="tableunevenadpbl"><TD>12:20</TD><TD>0.10</TD></TR><TR class="tableunevenadpbl"><TD>13:25</TD><TD>0.35</TD></TR></TBODY></TABLE></CENTER><P>After OD<SUB>600</SUB> reached 0.35, the Protein Expression protocol was followed. After the expression is completed, the cells were transferred to Downstream Processing (DSP).
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#TuesdayPFA4">Tuesday, 21/08/2018</BUTTON><DIV id="TuesdayPFA4" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>21/08/2018</DD><DT>Operator</DT><DD>Janine Nijenhuis</DD></DL><H4>Protein Purification - <I>Cell lysis</I></H4><P>The downstream processing of the dxCas9-Tn5 protein started. Harvest the cells according to the purification protocol. The cells with dxCas9-Tn5 protein were resuspended in 30 mL of Lysis buffer (20 mM Tris-HCl, 250 mM NaCl, 10% glycerol, pH 8).
        
    After resuspension the following additives were added for both cultures:
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Additive</TH><TH class="tableheaderadpbl">Stock Concentration</TH><TH class="tableheaderadpbl"> Volume added</TH></TR><TR class="tableunevenadpbl"><TD>DTT</TD><TD>0.5M</TD><TD>60uL</TD></TR><TR class="tableunevenadpbl"><TD>EDTA-free Complete protease inhibitor tablets</TD><TD>n/a</TD><TD>1 tablet</TD></TR><TR class="tableunevenadpbl"><TD>imidazole</TD><TD>2M</TD><TD>75uL</TD></TR></TBODY></TABLE></CENTER><P> The cells were lysed and clarified according to the the purification protocol.
</P><H4> Protein Purification - <I>Nickel Affinty Chromatography</I></H4><P>1mL of 50% His-Select Ni resin, was washed according to the Purification protocol. Clarified lysate of dxCas9-Tn5 protein was added to the washed His-Select Ni resin and incubated with gentle mixing for 60minutes at 4 ℃. The His-Select Ni resins were washed with washing buffer (20 mM Tris-HCl, 250 mM NaCl, 10% glycerol, 0.5mM Imidazole, 1mM DTT, pH8), and eluded with with 9* 500μL (6CV) of elution buffer with an increasing concentration of Imidazole (starting from 0.5mM to 250mM) (20 mM Tris-HCl, 250 mM NaCl, 10% glycerol, 1mM DTT, pH8) at 4 ℃, according to the Purification protocol. 
</P><H4> Tris-Glycine Polyacrylamide Gel </H4><P>In order to analyse the nickel affinty chromatography of the dxCas9-Tn5 protein, a precast Tris-Glycine polyacrylamide gel was run with a 6% stacking gel and 8% separation gel according to the SDS PAGE Tris-Glycine Polyacrylamide Gel protocol.The polyacrylamide gel ran for 90min at 50volt. After the was completed, the gel was stained with SimplyBlue SafeStain and imaged using the GelDoc system. The result of the gel is shown below:
        
<B>Figure Nickel Affinity Chromatography. </B><I> 8% SDS PAGE of dxCas9-Tn5 (expected size 214.7kDa). Lane 1 high range molecular ladder (kDa), lane 2 whole lysate, lane 3 cell debris, lane 4 clarified lysate, lane 5 flow through, lane 6 wash, lane 7 elution 1, lane 8 elution 2, lane 9 elution 3, lane 10 elution 4, lane 11 elution 5, lane 12 elution 6, lane 13 elution 7, lane 14 elution 8, lane 15 elution 9.</I></P><P>As the figure shows, even with a increasing concentration of Imidazole during elution, the fusion protein breaks down. This can be seen in elution fractions 3-8 (lane 9-14). The conclusion is that another method needs to be thought of to purify the fusion protein.</P><DL><DT>Date</DT><DD>21/08/2018</DD><DT>Operator</DT><DD>Monique de Leeuw</DD></DL><H4> Transformation of chemically competent cells </H4><P>The dxcas9-linker-Tn5_9 plasmid was transfomred into BL21AI for fusion protein production. The following transformations were carried out according to the protocol for transformation of chemically competent cells:
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Strain</TH><TH class="tableheaderadpbl">Plasmid &amp; size (bp) </TH><TH class="tableheaderadpbl">Cloning method</TH><TH class="tableheaderadpbl">Volume added (uL)</TH><TH class="tableheaderadpbl">Medium on which was plated</TH></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I>BL21AI</TD><TD>dxcas9-linker-Tn5_9 (9432bp)</TD><TD>Heat shock</TD><TD>1</TD><TD>LB chloramphenicol</TD></TR></TBODY></TABLE></CENTER><P>200 uL LB medium was added as recovery medium.
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#WednesdayPFA4">Wednesday, 22/08/2018</BUTTON><DIV id="WednesdayPFA4" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>22/08/2018</DD><DT>Operator</DT><DD>Monique de Leeuw</DD></DL><H4>Objective</H4><P>Fusion was expressed and purified to produce protein for <I>in vitro</I> functionality evaluation.
    </P><H4>Protein expression - <I>Seed Culture</I></H4><P>A seed culture was made to start Upstream Processing (USP) of fusion protein. A colony with dxcas9_linker-Tn5_9 strain was used for inoculation of 20mL liquid starter culture (Luria Broth, complemented with 34 μg/mL chloramphenicol) following the Liquid Starter Culture Protocol.
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#ThursdayPFA4">Thursday, 23/08/2018</BUTTON><DIV id="ThursdayPFA4" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>23/08/2018</DD><DT>Operator</DT><DD>Monique de Leeuw</DD></DL><H4>Protein Expression - <I>Inoculation &amp; Induction</I></H4><P>The upstream processing of the fusion protein continued. Inoculated 2L liquid media (Luria Broth, complemented with 34 μg/mL chloramphenicol) with the seed culture and grown to an OD<SUB>600</SUB> of 2. 
    </P><CENTER><TABLE><CAPTION class="figadpbl"><B>Table :</B> Growth represented by OD<SUB>600</SUB> over time</CAPTION><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Time</TH><TH class="tableheaderadpbl">OD<SUB>600</SUB></TH></TR><TR class="tableunevenadpbl"><TD>10:50am</TD><TD>0.28</TD></TR><TR class="tableunevenadpbl"><TD>11:25am</TD><TD>0.34</TD></TR><TR class="tableunevenadpbl"><TD>12:00am</TD><TD>0.58</TD></TR></TBODY></TABLE></CENTER><P>After the culture OD<SUB>600</SUB> reached 0.58, the Protein Expression protocol was followed. After the expression is completed, the cells were transferred to Downstream Processing (DSP).
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#FridayPFA4">Friday, 24/08/2018</BUTTON><DIV id="FridayPFA4" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>24/08/2018</DD><DT>Operator</DT><DD>Monique de Leeuw</DD></DL><H4>Protein Purification - <I>Cell lysis</I></H4><P>The downstream processing of the fusion protein started. Harvest the cells according to the purification protocol. 7.6g of cells of fusion protein were resuspended in Lysis buffer (20 mM Tris-HCl, 250 mM NaCl, 10% glycerol, pH 7.5).
        
    After resuspension the following additives were added:
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Additive</TH><TH class="tableheaderadpbl">Stock Concentration</TH><TH class="tableheaderadpbl"> Volume added</TH></TR><TR class="tableunevenadpbl"><TD>DTT</TD><TD>0.5M</TD><TD>106uL</TD></TR><TR class="tableunevenadpbl"><TD>EDTA-free Complete protease inhibitor tablets</TD><TD>n/a</TD><TD>1tablets</TD></TR><TR class="tableunevenadpbl"><TD>PMSF</TD><TD>0.2M</TD><TD>266uL</TD></TR></TBODY></TABLE></CENTER><P> The cells were lysed and clarified according to the the purification protocol.
</P><H4> Tris-Glycine Polyacrylamide Gel </H4><P>In order to analyse the expression of the fusion protein, a precast Tris-Glycine polyacrylamide gel was run with a 6% stacking gel and 12% separation gel according to the SDS PAGE Tris-Glycine Polyacrylamide Gel protocol. The polyacrylamide gel ran for 90min at 50volt. After the was completed, the gel was stained with SimplyBlue SafeStain and imaged using the GelDoc system. The result of the gel is shown below:
        
<B>Figure Fusion Expression: </B><I>12% SDS PAGE of fusion (expected size 214.7kDa). Lane 1 molecular ladder (kDa), lane 2 whole lysate, lane 3 cell debris, lane 5 clarified lysate. </I></P><H4> Protein Purification - <I>Heparin Chromatography</I></H4><P>The fusion protein was further purified with Heparin chromatography. The 1mL HiTrap Heparin HP column was connected to the AKTA pure in the cold room at 4 ℃. 42.4mL of crude lysate from fusion protein was injected into the injection loop and the chromatography was followed according to the Purification Protocol.
</P><H4> Tris-Glycine Polyacrylamide Gel </H4><P>In order to analyse the heparin chromatography of the fusion protein, a precast Tris-Glycine polyacrylamide gel was run with a 6% stacking gel and 8% separation gel according to the SDS PAGE Tris-Glycine Polyacrylamide Gel protocol.The polyacrylamide gel ran for 90min at 50volt. After the was completed, the gel was stained with SimplyBlue SafeStain and imaged using the GelDoc system. The result of the gel is shown below:
        
<B>Figure Heparin Chromatography. </B><I> 8% SDS PAGE of fusion (expected size 214.7kDa). Lane 1 molecular ladder (kDa), lane 2 elution 1, lane 3 elution 2, lane 4 elution 3, lane 5 elution 4, lane 6 elution 5, lane 7 elution 6, lane 8 elution 7, lane 9 elution 8, lane 10 elution 9, lane 11 elution 10, lane 12 elution 11, lane 13 elution 12. </I></P><P>Degradation was suspected for our fusion protein. Results seem to suggest that the fusion protein is being cleaved at the linker, resulting in both the dxCas9 and the fusion protein being present in our sample. The heparin chromatography was relatively successful at separating the two populations. The dxCas9 has a lower affinity to the heparin resin as a result it elutes at lower concentrations of salt (lane 2-10). As the concentration of salt increases less dxCas9 and more fusion elutes (lane 6-10). At high levels of salt the mostly fusion elutes (lane 11-13). Only these last three elution fractions are pooled (lane 11-13). Thus losses are tolerated, to obtain a relatively pure fusion fraction which can be further purified by MonoQ.</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#SaturdayPFA4">Saturday, 25/08/2018</BUTTON><DIV id="SaturdayPFA4" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>25/08/2018</DD><DT>Operator</DT><DD>Monique de Leeuw</DD></DL><H4>Protein Purification <I>- MonoQ Chromatography</I></H4><P>The fusion protein was further purified with MonoQ chromatography. The 1mL MonoQ 5/50 GL column was connected to the AKTA pure in the cold room at 4 ℃.  6mL of sample (4x diluted heparin chromatography elution fraction 10-12 of fusion protein) was injected into the injection loop and the chromatography was followed according to the Purification Protocol. </P><H4> Tris-Glycine Polyacrylamide Gel </H4><P>In order to analyse monoQ of the fusion protein, a precast Tris-Glycine polyacrylamide gel was run with a 6% stacking gel and 8% separation gel according to the SDS PAGE Tris-Glycine Polyacrylamide Gel protocol. The polyacrylamide gel ran for 90min at 50volt. After the was completed, the gel was stained with SimplyBlue SafeStain and imaged using the GelDoc system. The result of the gel is shown below:
        
<B>Figure MonoQ chromatography: </B><I>8% SDS PAGE of fusion (expected size 214.7kDa). Lane 1 molecular ladder (kDa), lane 2 starting material, lane 3 flow through, lane 4 wash, lane 5 elution 1, lane 6 elution 2, lane 7 elution 3, lane 8 elution 4, lane 9 elution 5, lane 10 elution 6, lane 11 elution 7, lane 12 elution 8, lane 13 elution 9, lane 14 elution 10, lane 15 elution 11. </I></P><P>The elution peak is broad (lane 6-14), however, the fractions are relatively pure and the dxCas9 contamination was mostly eliminated. These 9 fractions were pooled and further purified.</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#SundayPFA4">Sunday, 26/08/2018</BUTTON><DIV id="SundayPFA4" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>26/08/2018</DD><DT>Operator</DT><DD>Monique de Leeuw</DD></DL><H4>Protein Purification - <EM>Dialysis</EM></H4><P>Finally the fusion protein is dialysed and concentrated. 4.5mL of sample (MonoQ chromatography elution fraction 2-10 of fusion protein) were dialysed according to the purification protocol with dialysis buffer (20 mM Tris-HCl, 100 mM KCl, 5% glycerol, 1mM beta-Mercaptoethanol, pH 7.5).</P><H4>Protein Purification - <EM>Concentration</EM></H4><P>After dialysis, 4.5mL of sample of fusion protein was concentrated to a final volume of 0.6mL and  stored at -20℃, according to the purification protocol. </P><P>In order to analyse the concentrated fusion protein, a precast Tris-Glycine polyacrylamide gel was run with a 6% stacking gel and 8% separation gel according to the SDS PAGE Tris-Glycine Polyacrylamide Gel protocol. The polyacrylamide gel ran for 90min at 50volt. After the was completed, the gel was stained with SimplyBlue SafeStain and imaged using the GelDoc system. The result of the gel is shown below:
        
<B>Figure Concentration: </B><I>8% SDS PAGE of fusion (expected size 214.7kDa). Lane 1 molecular ladder (kDa), lane 2 Crude lysate, lane 3 final fusion post concentration 9uL, lane 4 final fusion post concentration 4.5uL, lane 5 final fusion post concentration 2.25uL. </I></P><P> Trace amounts of contaminating background was observed, however, the fusion was significantly more in quantity and the background was tolerated for functionality testing. The final concentration of the solution was determined according to the BCA protocol and resulted in a concentration of 0.57mg/mL (2.64µM).</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV></DIV></UL></FORM><FORM><LABEL for="WeekPFA5" class="subdropdown cadpbl">Week 5 27/08/2018 - 02/09/2018</LABEL><UL class="submenu"><DIV class="subcontent"><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#MondayPFA5">Monday, 27/08/2018</BUTTON><DIV id="MondayPFA5" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>27/08/2018</DD><DT>Operator</DT><DD>Janine Nijenhuis</DD></DL><H4>Objective</H4><P>Tn5 was expressed and purified to produce protein for <I>in vitro</I> functionality evaluation.
    </P><H4>Protein expression - <I>Seed Culture</I></H4><P>A seed culture was made to start Upstream Processing (USP) of Tn5. A colony with Tn5 strain was used for inoculation of 40mL liquid starter culture (Luria Broth, complemented with 34 μg/mL chloramphenicol) following the Liquid Starter Culture Protocol.
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#TuesdayPFA5">Tuesday, 28/08/2018</BUTTON><DIV id="TuesdayPFA5" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>28/08/2018</DD><DT>Operator</DT><DD>Janine Nijenhuis</DD></DL><H4>Protein Expression - <I>Inoculation &amp; Induction</I></H4><P>The upstream processing of the Tn5 continued. Inoculated 1L liquid media (Luria Broth, complemented with 34 μg/mL chloramphenicol) with the seed culture and grown to an OD<SUB>600</SUB> of 0.3. 
    </P><CENTER><TABLE><CAPTION class="figadpbl"><B>Table :</B> Growth represented by OD<SUB>600</SUB> over time</CAPTION><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Time</TH><TH class="tableheaderadpbl">OD<SUB>600</SUB></TH></TR><TR class="tableunevenadpbl"><TD>12:55</TD><TD>0.04</TD></TR><TR class="tableunevenadpbl"><TD>14:25</TD><TD>0.11</TD></TR><TR class="tableunevenadpbl"><TD>15:30</TD><TD>0.48</TD></TR></TBODY></TABLE></CENTER><P>After OD<SUB>600</SUB> reached 0.48, the Protein Expression protocol was followed. After the expression is completed, the cells were transferred to Downstream Processing (DSP).
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#WednesdayPFA5">Wednesday, 29/08/2018</BUTTON><DIV id="WednesdayPFA5" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>29/08/2018</DD><DT>Operator</DT><DD>Janine Nijenhuis</DD></DL><H4>Protein Purification - <I>Cell lysis</I></H4><P>The downstream processing of the Tn5 protein started. Harvest the cells according to the purification protocol. The cells with Tn5 protein were resuspended in 30 mL of Lysis buffer (20 mM Tris-HCl, 250 mM NaCl, 10% glycerol, pH 8).
        
    After resuspension the following additives were added for both cultures:
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Additive</TH><TH class="tableheaderadpbl">Stock Concentration</TH><TH class="tableheaderadpbl"> Volume added</TH></TR><TR class="tableunevenadpbl"><TD>DTT</TD><TD>0.5M</TD><TD>60uL</TD></TR><TR class="tableunevenadpbl"><TD>EDTA-free Complete protease inhibitor tablets</TD><TD>n/a</TD><TD>1 tablet</TD></TR><TR class="tableunevenadpbl"><TD>imidazole</TD><TD>2M</TD><TD>75uL</TD></TR></TBODY></TABLE></CENTER><P> The cells were lysed and clarified according to the the purification protocol.
</P><H4> Protein Purification - <I>Nickel Affinty Chromatography</I></H4><P>1mL of 50% His-Select Ni resin, was washed according to the Purification protocol. Clarified lysate of Tn5 protein was added to the washed His-Select Ni resin and incubated with gentle mixing for 60minutes at 4 ℃. The His-Select Ni resins were washed with washing buffer (20 mM Tris-HCl, 250 mM NaCl, 10% glycerol, 20mM imidazole, 1mM DTT, pH8), and eluded with with 6* 500μL (6CV) of elution buffer (20 mM Tris-HCl, 250 mM NaCl, 10% glycerol, 250mM imidazole, 1mM DTT, pH8) at 4 ℃, according to the Purification protocol. 
</P><H4> Tris-Glycine Polyacrylamide Gel </H4><P>In order to analyse the nickel affinty chromatography of the Tn5 protein, a precast Tris-Glycine polyacrylamide gel was run with a 6% stacking gel and 8% separation gel according to the SDS PAGE Tris-Glycine Polyacrylamide Gel protocol.The polyacrylamide gel ran for 90min at 50volt. After the was completed, the gel was stained with SimplyBlue SafeStain and imaged using the GelDoc system. The result of the gel is shown below:
        
<B>Figure Nickel Affinity Chromatography. </B><I> 8% SDS PAGE of Tn5 (expected size 53.3kDa). Lane 1 high range molecular ladder (kDa), lane 2 post induction, lane 3 whole lysate, lane 4 cell debris, lane 5 clarified lysate, lane 6 flow through, lane 7 wash, lane 8 elution 1, lane 9 elution 2, lane 10 elution 3, lane 11 elution 4, lane 12 elution 5, lane 13 elution 6.</I></P><P>The nickel chromatography was not completely successful, but there is some purification of Tn5. The Tn5 did not completely bind to the nickel resin and was observed in the flowthrough (lane 6). It is recommended to further optimize the nickel chromatography to further reduce losses, and reduce contaminating proteins due to non-specific binding to the nickel resin (lane 8). However for our purposes, the losses were tolerated.</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV></DIV></UL></FORM><H3 class="adpbl">September</H3><FORM><LABEL for="WeekPFS1" class="subdropdown cadpbl">Week 1 03/09/2018 - 09/09/2018</LABEL><UL class="submenu"><DIV class="subcontent"><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#MondayPFS1">Monday, 03/09/2018</BUTTON><DIV id="MondayPFS1" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>03/09/2018</DD><DT>Operator</DT><DD>Monique de Leeuw</DD></DL><H4> Transformation of chemically competent cells </H4><P>The Tn5_3 plasmid was transfomred into BL21AI for Tn5 protein production. The following transformations were carried out according to the protocol for transformation of chemically competent cells:
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Strain</TH><TH class="tableheaderadpbl">Plasmid &amp; size (bp) </TH><TH class="tableheaderadpbl">Cloning method</TH><TH class="tableheaderadpbl">Volume added (uL)</TH><TH class="tableheaderadpbl">Medium on which was plated</TH></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I>BL21AI</TD><TD>Tn5.3 (5210bp)</TD><TD>Heat shock</TD><TD>1</TD><TD>LB chloramphenicol</TD></TR></TBODY></TABLE></CENTER><P>200 uL LB medium was added as recovery medium.
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#TuesdayPFS1">Tuesday, 04/09/2018</BUTTON><DIV id="TuesdayPFS1" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>04/09/2018</DD><DT>Operator</DT><DD>Monique de Leeuw</DD></DL><H4>Objective</H4><P>Tn5 was expressed and purified to produce protein for <I>in vitro</I> functionality evaluation.
    </P><H4>Protein expression - <I>Seed Culture</I></H4><P>A seed culture was made to start Upstream Processing (USP) of Tn5 protein. A colony with Tn5_3 strain was used for inoculation of 10mL liquid starter culture (Luria Broth, complemented with 34 μg/mL chloramphenicol) following the Liquid Starter Culture Protocol.
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#WednesdayPFS1">Wednesday, 05/09/2018</BUTTON><DIV id="WednesdayPFS1" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>05/09/2018</DD><DT>Operator</DT><DD>Janine Nijenhuis</DD></DL><H4>Objective</H4><P>Tn5 was expressed and purified to produce protein for <I>in vitro</I> functionality evaluation.
    </P><H4>Protein expression - <I>Seed Culture</I></H4><P>A seed culture was made to start Upstream Processing (USP) of Tn5. A colony with Tn5 strain was used for inoculation of 100mL liquid starter culture (Luria Broth, complemented with 34 μg/mL chloramphenicol) following the Liquid Starter Culture Protocol.
</P><DL><DT>Date</DT><DD>05/09/2018</DD><DT>Operator</DT><DD>Monique de Leeuw</DD></DL><H4>Protein Expression - <I>Inoculation &amp; Induction</I></H4><P>The upstream processing of the Tn5 protein continued. Inoculated 1L liquid media (Luria Broth, complemented with 34 μg/mL chloramphenicol) with the seed culture and grown to an OD<SUB>600</SUB> of 2. 
    </P><CENTER><TABLE><CAPTION class="figadpbl"><B>Table :</B> Growth represented by OD<SUB>600</SUB> over time</CAPTION><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Time</TH><TH class="tableheaderadpbl">OD<SUB>600</SUB></TH></TR><TR class="tableunevenadpbl"><TD>10:30am</TD><TD>0.22</TD></TR><TR class="tableunevenadpbl"><TD>11:30am</TD><TD>0.35</TD></TR><TR class="tableunevenadpbl"><TD>12:00am</TD><TD>0.52</TD></TR></TBODY></TABLE></CENTER><P>After the culture OD<SUB>600</SUB> reached 0.52, the Protein Expression protocol was followed. After the expression is completed, the cells were transferred to Downstream Processing (DSP).
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#ThursdayPFS1">Thursday, 06/09/2018</BUTTON><DIV id="ThursdayPFS1" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>06/09/2018</DD><DT>Operator</DT><DD>Janine Nijenhuis</DD></DL><H4>Protein Expression - <I>Inoculation &amp; Induction</I></H4><P>The upstream processing of the Tn5 continued. Inoculated 3L liquid media (Luria Broth, complemented with 34 μg/mL chloramphenicol) with the seed culture and grown to an OD<SUB>600</SUB> of 0.3. 
    </P><CENTER><TABLE><CAPTION class="figadpbl"><B>Table :</B> Growth represented by OD<SUB>600</SUB> over time</CAPTION><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Time</TH><TH class="tableheaderadpbl">Flask 1 OD<SUB>600</SUB></TH><TH class="tableheaderadpbl">Flask 2 OD<SUB>600</SUB></TH><TH class="tableheaderadpbl">Flask 3 OD<SUB>600</SUB></TH></TR><TR class="tableunevenadpbl"><TD>12:50</TD><TD>0.03</TD><TD>0.05</TD><TD>0.05</TD></TR><TR class="tableunevenadpbl"><TD>14:25</TD><TD>0.05</TD><TD>0.07</TD><TD>0.08</TD></TR><TR class="tableunevenadpbl"><TD>15:20</TD><TD>0.14</TD><TD>0.18</TD><TD>0.17</TD></TR><TR class="tableunevenadpbl"><TD>15:40</TD><TD>0.27</TD><TD>0.33</TD><TD>0.26</TD></TR><TR class="tableunevenadpbl"><TD>16:00</TD><TD>0.37</TD><TD>0.40</TD><TD>0.39</TD></TR></TBODY></TABLE></CENTER><P>After the three cultures reached an OD<SUB>600</SUB> of respectively 0.37, 0.4 and 0.39, the Protein Expression protocol was followed. After the expression is completed, the cells were transferred to Downstream Processing (DSP).
</P><DL><DT>Date</DT><DD>06/09/2018</DD><DT>Operator</DT><DD>Monique de Leeuw</DD></DL><H4>Protein Purification - <I>Cell lysis</I></H4><P>The downstream processing of the Tn5 protein started. Harvest the cells according to the purification protocol. 6.17g of cells of Tn5 protein were resuspended in Lysis buffer (20 mM Tris-HCl, 250 mM NaCl, 10% glycerol, pH 8).
        
    After resuspension the following additives were added:
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Additive</TH><TH class="tableheaderadpbl">Stock Concentration</TH><TH class="tableheaderadpbl"> Volume added</TH></TR><TR class="tableunevenadpbl"><TD>DTT</TD><TD>0.5M</TD><TD>86uL</TD></TR><TR class="tableunevenadpbl"><TD>EDTA-free Complete protease inhibitor tablets</TD><TD>n/a</TD><TD>1tablets</TD></TR><TR class="tableunevenadpbl"><TD>imidazole</TD><TD>2M</TD><TD>432uL</TD></TR></TBODY></TABLE></CENTER><P> The cells were lysed and clarified according to the the purification protocol.
</P><H4> Tris-Glycine Polyacrylamide Gel </H4><P>In order to analyse the expression of the Tn5 protein, a precast Tris-Glycine polyacrylamide gel was run with a 6% stacking gel and 12% separation gel according to the SDS PAGE Tris-Glycine Polyacrylamide Gel protocol. The polyacrylamide gel ran for 90min at 50volt. After the was completed, the gel was stained with SimplyBlue SafeStain and imaged using the GelDoc system. The result of the gel is shown below:
        
<B>Figure expression of Tn5: </B><I>12% SDS PAGE of Tn5 (expected size 53.3kDa). Lane 1 molecular ladder (kDa), lane 2 pre-Induction, lane 3 whole lysate, lane 4 clarified lysate. 
.  </I></P><H4> Protein Purification - <I>Nickel Affinty Chromatography</I></H4><P>1mL of 50% His-Select Ni resin, was washed according to the Purification protocol. 33.5mL clarified lysate of Tn5 protein was added to the washed His-Select Ni resin and incubated with gentle mixing for 60minutes at 4 ℃. The His-Select Ni resin was washed with washing buffer (20 mM Tris-HCl, 250 mM NaCl, 20mM Imidazole, 10% glycerol, 1mM DTT, pH8), and eluded with with 6* 500μL (6CV) of elution buffer (20 mM Tris-HCl, 250 mM NaCl, 250 mM Imidazole, 10% glycerol, 1mM DTT, pH8) at 4 ℃, according to the Purification protocol. 
</P><H4> Tris-Glycine Polyacrylamide Gel </H4><P>In order to analyse the nickel affinty chromatography of the Tn5 protein, a precast Tris-Glycine polyacrylamide gel was run with a 6% stacking gel and 12% separation gel according to the SDS PAGE Tris-Glycine Polyacrylamide Gel protocol.The polyacrylamide gel ran for 90min at 50volt. After the was completed, the gel was stained with SimplyBlue SafeStain and imaged using the GelDoc system. The result of the gel is shown below:
        
<B>Figure Nickel Affinity Chromatography. </B><I> 12% SDS of Tn5 (expected size 53.3kDa). Lane 1 molecular ladder (kDa), lane 2 whole lysate, lane 3 clarified lysate, lane 4 cell debris, lane 5 flow through 1, lane 6  flow through 2, lane 7 wash 1, lane 8 wash 2, lane 9 wash 3, lane 10 elution 1, lane 11 elution 2, lane 12 elution 3, lane 13 elution 4, lane 14 elution 5, lane 15 elution 6.</I></P><P>A significant amount of Tn5 was observed in the nickel chromatography elution fractions (lane 10-14), however the Tn5 protein was also observed in the flowthrough and wash elution fractions (lane 5-9). It is recommended to optimize the nickel chromatography to reduce these losses. For our purposes, the Tn5 losses were tolerated because the amounts that we obtained were sufficient to proceed to functionality testing of the Tn5. To increase the Tn5 amount, the first five elution fractions were pooled (lane 10-14).</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#FridayPFS1">Friday, 07/09/2018</BUTTON><DIV id="FridayPFS1" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>07/09/2018</DD><DT>Operator</DT><DD>Janine Nijenhuis</DD></DL><H4>Protein Purification - <I>Cell lysis</I></H4><P>The downstream processing of the Tn5 protein started. Harvest the cells according to the purification protocol. The three cultures were combined to one. The French Press was however out of order, so the cell pellet is stored at -20 ℃.</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV></DIV></UL></FORM><FORM><LABEL for="WeekPFS2" class="subdropdown cadpbl">Week 2 10/09/2018 - 16/09/2018</LABEL><UL class="submenu"><DIV class="subcontent"><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#WednesdayPFS2">Wednesday, 12/09/2018</BUTTON><DIV id="WednesdayPFS2" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>12/09/2018</DD><DT>Operator</DT><DD>Janine Nijenhuis</DD></DL><H4>Protein Purification - <I>Cell lysis</I></H4><P>The downstream processing of the Tn5 protein started. Harvest the cells according to the purification protocol. The cells with Tn5 protein were resuspended in 72 mL of Lysis buffer (20 mM Tris-HCl, 250 mM NaCl, 10% glycerol, pH 8).
        
    After resuspension the following additives were added for both cultures:
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Additive</TH><TH class="tableheaderadpbl">Stock Concentration</TH><TH class="tableheaderadpbl"> Volume added</TH></TR><TR class="tableunevenadpbl"><TD>beta-mercaptoethanol</TD><TD>0.5M</TD><TD>5.04uL</TD></TR><TR class="tableunevenadpbl"><TD>EDTA-free Complete protease inhibitor tablets</TD><TD>n/a</TD><TD>2 tablet</TD></TR><TR class="tableunevenadpbl"><TD>imidazole</TD><TD>2M</TD><TD>180uL</TD></TR></TBODY></TABLE></CENTER><P> The cells were lysed and clarified according to the the purification protocol.
</P><H4> Protein Purification - <I>Nickel Affinty Chromatography</I></H4><P>1mL of 50% His-Select Ni resin, was washed according to the Purification protocol. Clarified lysate of Tn5 protein was added to the washed His-Select Ni resin and incubated with gentle mixing for 60minutes at 4 ℃. The His-Select Ni resins were washed with washing buffer (20 mM Tris-HCl, 250 mM NaCl, 10% glycerol, 20mM imidazole, 1mM DTT, pH8), and eluded with with 6* 500μL (6CV) of elution buffer (20 mM Tris-HCl, 250 mM NaCl, 10% glycerol, 250mM imidazole, 1mM DTT, pH8) at 4 ℃, according to the Purification protocol. 
</P><H4> Tris-Glycine Polyacrylamide Gel </H4><P>In order to analyse the nickel affinty chromatography of the Tn5 protein, a precast Tris-Glycine polyacrylamide gel was run with a 6% stacking gel and 8% separation gel according to the SDS PAGE Tris-Glycine Polyacrylamide Gel protocol.The polyacrylamide gel ran for 90min at 50volt. After the was completed, the gel was stained with SimplyBlue SafeStain and imaged using the GelDoc system. The result of the gel is shown below:
        
<B>Figure Nickel Affinity Chromatography. </B><I> 8% SDS PAGE of Tn5 (expected size 53.3kDa). Lane 1 high range molecular ladder (kDa), lane 2 whole lysate, lane 3 clarified lysate, lane 4 cell debris, lane 5 flow through, lane 6 wash, lane 7 elution 1, lane 8 elution 2, lane 9 elution 3, lane 10 elution 4, lane 11 elution 5, lane 12 elution 6.</I></P><P>The nickel chromatography was not completely successful, but there is some purification of Tn5. The Tn5 did not completely bind to the nickel resin and was observed in the flowthrough and wash (lane 6-7). It is recommended to further optimize the nickel chromatography to further reduce losses, and reduce contaminating proteins due to non-specific binding to the nickel resin (lane 9-10). However for our purposes, the losses were tolerated.</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV></DIV></UL></FORM><FORM><LABEL for="WeekPFS4" class="subdropdown cadpbl">Week 4 24/09/2018 - 30/09/2018</LABEL><UL class="submenu"><DIV class="subcontent"><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#MondayPFS4">Monday, 24/09/2018</BUTTON><DIV id="MondayPFS4" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>24/09/2018</DD><DT>Operator</DT><DD>Janine Nijenhuis</DD></DL><H4>Objective</H4><P>dxCas9-Tn5 was expressed and purified to produce protein for <I>in vitro</I> functionality evaluation.
    </P><H4>Protein expression - <I>Seed Culture</I></H4><P>A seed culture was made to start Upstream Processing (USP) of dxCas9-Tn5. A colony with dxCas9-Tn5_9 strain was used for inoculation of 100mL liquid starter culture (Luria Broth, complemented with 34 μg/mL chloramphenicol) following the Liquid Starter Culture Protocol.
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#TuesdayPFS4">Tuesday, 25/09/2018</BUTTON><DIV id="TuesdayPFS4" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>25/09/2018</DD><DT>Operator</DT><DD>Janine Nijenhuis</DD></DL><H4>Protein Expression - <I>Inoculation &amp; Induction</I></H4><P>The upstream processing of the dxCas9-Tn5 continued. Inoculated 5L liquid media (Luria Broth, complemented with 34 μg/mL chloramphenicol) with the seed culture and grown to an OD<SUB>600</SUB> of 0.3. 
    </P><CENTER><TABLE><CAPTION class="figadpbl"><B>Table :</B> Growth represented by OD<SUB>600</SUB> over time</CAPTION><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Time</TH><TH class="tableheaderadpbl">Flask 1 OD<SUB>600</SUB></TH><TH class="tableheaderadpbl">Flask 2 OD<SUB>600</SUB></TH><TH class="tableheaderadpbl">Flask 3 OD<SUB>600</SUB></TH></TR><TR class="tableunevenadpbl"><TD>11:30</TD><TD>0.03</TD><TD>0.08</TD><TD>0.04</TD></TR><TR class="tableunevenadpbl"><TD>13:50</TD><TD>0.30</TD><TD>0.22</TD><TD>0.22</TD></TR><TR class="tableunevenadpbl"><TD>14:10</TD><TD>X</TD><TD>0.37</TD><TD>0.34</TD></TR></TBODY></TABLE></CENTER><P>After the three cultures reached an OD<SUB>600</SUB> of respectively 0.30, 0.37 and 0.34, the Protein Expression protocol was followed. After the expression is completed, the cells were transferred to Downstream Processing (DSP).
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#WednesdayPFS4">Wednesday, 26/09/2018</BUTTON><DIV id="WednesdayPFS4" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>26/09/2018</DD><DT>Operator</DT><DD>Janine Nijenhuis</DD></DL><H4>Protein Purification - <I>Cell lysis</I></H4><P>The downstream processing of the dxCas9-Tn5 protein started. Harvest the cells according to the purification protocol. The cells with dxCas9-Tn5 protein were resuspended in a total volume of 150 mL of Lysis buffer (20 mM Tris-HCl, 250 mM NaCl, 10% glycerol, pH 7.5).
        
    After resuspension the following additives were added for both cultures:
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Additive</TH><TH class="tableheaderadpbl">Stock Concentration</TH><TH class="tableheaderadpbl"> Volume added</TH></TR><TR class="tableunevenadpbl"><TD>beta-mercaptoethanol</TD><TD>14.3M</TD><TD>10.5uL</TD></TR><TR class="tableunevenadpbl"><TD>EDTA-free Complete protease inhibitor tablets</TD><TD>n/a</TD><TD>3 tablets</TD></TR><TR class="tableunevenadpbl"><TD>imidazole</TD><TD>2M</TD><TD>375uL</TD></TR><TR class="tableunevenadpbl"><TD>PMSF</TD><TD>0.2M</TD><TD>3mL</TD></TR></TBODY></TABLE></CENTER><P> The cells were lysed and clarified according to the the purification protocol. However, something went wrong, and instead of a clear suspension, only foam came out of the homogenizer. Because it looked like denatured proteins, it is decided to start again with the purification, because it cannot be said for sure the fusion protein will work.
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#ThursdayPFS4">Thursday, 27/09/2018</BUTTON><DIV id="ThursdayPFS4" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>27/09/2018</DD><DT>Operator</DT><DD>Janine Nijenhuis</DD></DL><H4>Objective</H4><P>dxCas9-Tn5 was expressed and purified to produce protein for <I>in vitro</I> functionality evaluation.
    </P><H4>Protein expression - <I>Seed Culture</I></H4><P>A seed culture was made to start Upstream Processing (USP) of dxCas9-Tn5. A colony with dxCas9-Tn5_9 strain was used for inoculation of 100mL liquid starter culture (Luria Broth, complemented with 34 μg/mL chloramphenicol) following the Liquid Starter Culture Protocol.
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#FridayPFS4">Friday, 28/09/2018</BUTTON><DIV id="FridayPFS4" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>28/09/2018</DD><DT>Operator</DT><DD>Janine Nijenhuis</DD></DL><H4>Protein Expression - <I>Inoculation &amp; Induction</I></H4><P>The upstream processing of the dxCas9-Tn5 continued. Inoculated 3L liquid media (Luria Broth, complemented with 34 μg/mL chloramphenicol) with the seed culture and grown to an OD<SUB>600</SUB> of 0.3. 
    </P><CENTER><TABLE><CAPTION class="figadpbl"><B>Table :</B> Growth represented by OD<SUB>600</SUB> over time</CAPTION><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Flask number OD<SUB>600</SUB></TH><TH class="tableheaderadpbl">13:50</TH><TH class="tableheaderadpbl">15:30</TH><TH class="tableheaderadpbl">16:25</TH></TR><TR class="tableunevenadpbl"><TD>1</TD><TD>0.11</TD><TD>0.18</TD><TD>0.57</TD></TR><TR class="tableunevenadpbl"><TD>2</TD><TD>0.09</TD><TD>0.11</TD><TD>0.38</TD></TR><TR class="tableunevenadpbl"><TD>3</TD><TD>0.07</TD><TD>0.16</TD><TD>0.55</TD></TR><TR class="tableunevenadpbl"><TD>4</TD><TD>0.07</TD><TD>0.16</TD><TD>0.56</TD></TR><TR class="tableunevenadpbl"><TD>5</TD><TD>0.05</TD><TD>0.15</TD><TD>0.54</TD></TR><TR class="tableunevenadpbl"><TD>6</TD><TD>0.06</TD><TD>0.16</TD><TD>0.54</TD></TR><TR class="tableunevenadpbl"><TD>7</TD><TD>0.06</TD><TD>0.17</TD><TD>0.54</TD></TR></TBODY></TABLE></CENTER><P>After the seven cultures all reached an OD<SUB>600</SUB> higher than 0.3, the Protein Expression protocol was followed. After the expression is completed, the cells were transferred to Downstream Processing (DSP).
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV></DIV></UL></FORM><H3 class="adpbl">October</H3><FORM><LABEL for="WeekPFO1" class="subdropdown cadpbl">Week 1 01/10/2018 - 07/10/2018</LABEL><UL class="submenu"><DIV class="subcontent"><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#MondayPFO1">Monday, 01/10/2018</BUTTON><DIV id="MondayPFO1" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>01/10/2018</DD><DT>Operator</DT><DD>Monique de Leeuw</DD></DL><H4> Transformation of chemically competent cells </H4><P>The dxcas9-linker-Tn5_9 plasmid was transfomred into BL21AI for fusion protein production. The following transformations were carried out according to the protocol for transformation of chemically competent cells:
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Strain</TH><TH class="tableheaderadpbl">Plasmid &amp; size (bp) </TH><TH class="tableheaderadpbl">Cloning method</TH><TH class="tableheaderadpbl">Volume added (uL)</TH><TH class="tableheaderadpbl">Medium on which was plated</TH></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I>BL21AI</TD><TD>dxcas9-linker-Tn5_9 (9432bp)</TD><TD>Heat shock</TD><TD>1</TD><TD>LB chloramphenicol</TD></TR></TBODY></TABLE></CENTER><P>200 uL LB medium was added as recovery medium.
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#TuesdayPFO1">Tuesday, 02/10/2018</BUTTON><DIV id="TuesdayPFO1" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>02/10/2018</DD><DT>Operator</DT><DD>Monique de Leeuw</DD></DL><H4>Objective</H4><P>Fusion was expressed and purified to produce protein for <I>in vitro</I> functionality evaluation.
    </P><H4>Protein expression - <I>Seed Culture</I></H4><P>A seed culture was made to start Upstream Processing (USP) of fusion protein. A colony with dxcas9_linker-Tn5_9 strain was used for inoculation of 20mL liquid starter culture (Luria Broth, complemented with 34 μg/mL chloramphenicol) following the Liquid Starter Culture Protocol.
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#WednesdayPFO1">Wednesday, 03/10/2018</BUTTON><DIV id="WednesdayPFO1" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>03/10/2018</DD><DT>Operator</DT><DD>Monique de Leeuw</DD></DL><H4>Protein Expression - <I>Inoculation &amp; Induction</I></H4><P>The upstream processing of the fusion protein continued. Inoculated 2L liquid media (Luria Broth, complemented with 34 μg/mL chloramphenicol) with the seed culture and grown to an OD<SUB>600</SUB> of 2. 
    </P><CENTER><TABLE><CAPTION class="figadpbl"><B>Table :</B> Growth represented by OD<SUB>600</SUB> over time</CAPTION><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Time</TH><TH class="tableheaderadpbl">OD<SUB>600</SUB></TH></TR><TR class="tableunevenadpbl"><TD>10:50am</TD><TD>0.08</TD></TR><TR class="tableunevenadpbl"><TD>11:25am</TD><TD>0.32</TD></TR><TR class="tableunevenadpbl"><TD>12:00am</TD><TD>0.58</TD></TR></TBODY></TABLE></CENTER><P>After the culture OD<SUB>600</SUB> reached 0.58, the Protein Expression protocol was followed. After the expression is completed, the cells were transferred to Downstream Processing (DSP).
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#ThursdayPFO1">Thursday, 04/10/2018</BUTTON><DIV id="ThursdayPFO1" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>04/10/2018</DD><DT>Operator</DT><DD>Monique de Leeuw</DD></DL><H4>Protein Purification - <I>Cell lysis</I></H4><P>The downstream processing of the fusion protein started. Harvest the cells according to the purification protocol. 7.2g of cells of fusion protein were resuspended in Lysis buffer (20 mM Tris-HCl, 250 mM NaCl, 10% glycerol, pH 7.5).
        
    After resuspension the following additives were added:
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Additive</TH><TH class="tableheaderadpbl">Stock Concentration</TH><TH class="tableheaderadpbl"> Volume added</TH></TR><TR class="tableunevenadpbl"><TD>DTT</TD><TD>0.5M</TD><TD>101uL</TD></TR><TR class="tableunevenadpbl"><TD>EDTA-free Complete protease inhibitor tablets</TD><TD>n/a</TD><TD>1tablets</TD></TR><TR class="tableunevenadpbl"><TD>PMSF</TD><TD>0.2M</TD><TD>252uL</TD></TR></TBODY></TABLE></CENTER><P> The cells were lysed and clarified according to the the purification protocol.
</P><H4> Tris-Glycine Polyacrylamide Gel </H4><P>In order to analyse the expression of the fusion protein, a precast Tris-Glycine polyacrylamide gel was run with a 6% stacking gel and 12% separation gel according to the SDS PAGE Tris-Glycine Polyacrylamide Gel protocol. The polyacrylamide gel ran for 90min at 50volt. After the was completed, the gel was stained with SimplyBlue SafeStain and imaged using the GelDoc system. The result of the gel is shown below:
        
<B>Figure Fusion Expression: </B><I>12% SDS PAGE of fusion (expected size 214.7kDa).  Lane 1 Whole lysate, lane 2 clarified, lane 3 flow through 1, lane 4 flow through 2 , lane 5 molecular ladder (kDa). </I></P><H4> Protein Purification - <I>Heparin Chromatography</I></H4><P>The fusion protein was further purified with Heparin chromatography. The 1mL HiTrap Heparin HP column was connected to the AKTA pure in the cold room at 4 ℃. 45.2mL of crude lysate from fusion protein was injected into the injection loop and the chromatography was followed according to the Purification Protocol.
</P><H4> Tris-Glycine Polyacrylamide Gel </H4><P>In order to analyse the heparin chromatography of the fusion protein, a precast Tris-Glycine polyacrylamide gel was run with a 6% stacking gel and 8% separation gel according to the SDS PAGE Tris-Glycine Polyacrylamide Gel protocol.The polyacrylamide gel ran for 90min at 50volt. After the was completed, the gel was stained with SimplyBlue SafeStain and imaged using the GelDoc system. The result of the gel is shown below:
        
<B>Figure Heparin Chromatography #1. </B><I> 8% SDS PAGE of fusion (expected size 214.7kDa). Lane 1 Whole lysate, lane 2 clarified, lane 3 flow through 1, lane 4 flow through 2 , lane 5 molecular ladder (kDa), lane 6 wash, lane 7 elution 1, lane 8 elution 2, lane 9 elution 3, lane 10 elution 4, lane 11 elution 5, lane 12 elution 6, lane 13 elution 7, lane 14 elution 8, lane 15 elution 9. </I><B>Figure Heparin Chromatography #2. </B><I> 8% SDS PAGE of fusion (expected size 214.7kDa). Lane 1 molecular weight (kDa), lane 2 elution 10, lane 3 elution 11, lane 4 elution 12 , lane 5 elution 13, lane 6 elution14, lane 7 elution 15, lane 8 elution 16, lane 9 elution 17, lane 10 elution 18, lane 11 elution 19, lane 12 elution 20, lane 13 elution 21, lane 14 elution 22, lane 15 elution 23. </I></P>
8-15
        <P>Degradation of our fusion protein is still significant. Results seem to suggest that the fusion protein is being cleaved at the linker, resulting in both the dxCas9 and the fusion protein being present in our sample. The heparin chromatography was relatively successful at separating the two populations. The dxCas9 has a lower affinity to the heparin resin as a result it elutes at lower concentrations of salt. As the concentration of salt increases less dxCas9 and more fusion elutes (gel1, lane 9-15 and gel2, lane 1-7). At high levels of salt the mostly fusion elutes (gel2, lane 8-15). Only these last eight elution fractions are pooled. Thus losses are tolerated, to obtain a relatively pure fusion fraction which can be further purified by MonoQ.</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#FridayPFO1">Friday, 05/10/2018</BUTTON><DIV id="FridayPFO1" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>05/10/2018</DD><DT>Operator</DT><DD>Monique de Leeuw</DD></DL><H4>Protein Purification <I>- MonoQ Chromatography</I></H4><P>The fusion protein was further purified with MonoQ chromatography. The 1mL MonoQ 5/50 GL column was connected to the AKTA pure in the cold room at 4 ℃.  16mL of sample (4x diluted heparin chromatography elution fraction 16-23 of fusion protein) was injected into the injection loop and the chromatography was followed according to the Purification Protocol. </P><H4> Tris-Glycine Polyacrylamide Gel </H4><P>In order to analyse monoQ of the fusion protein, a precast Tris-Glycine polyacrylamide gel was run with a 6% stacking gel and 8% separation gel according to the SDS PAGE Tris-Glycine Polyacrylamide Gel protocol. The polyacrylamide gel ran for 90min at 50volt. After the was completed, the gel was stained with SimplyBlue SafeStain and imaged using the GelDoc system. The result of the gel is shown below:
        
<B>Figure MonoQ chromatography: </B><I>8% SDS PAGE of fusion (expected size 214.7kDa). Lane 1 molecular ladder (kDa), lane 2 starting material, lane 3 flow through, lane 4 wash, lane 5 elution 1, lane 6 elution 2, lane 7 elution 3, lane 8 elution 4, lane 9 elution 5, lane 10 elution 6, lane 11 elution 7, lane 12 elution 8, lane 13 elution 9, lane 14 elution 10, lane 15 elution 11.  </I></P><P>The elution peak is broad (lane 5-13), however, the fractions are relatively pure and the dxCas9 contamination was mostly eliminated. These 9 fractions were pooled and further purified.</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#SaturdayPFO1">Saturday, 06/10/2018</BUTTON><DIV id="SaturdayPFO1" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>06/10/2018</DD><DT>Operator</DT><DD>Monique de Leeuw</DD></DL><H4>Protein Purification - <EM>Dialysis</EM></H4><P>Finally the fusion protein is dialysed and concentrated. 4.5mL of sample (MonoQ chromatography elution fraction 2-10 of fusion protein) were dialysed according to the purification protocol with dialysis buffer (20 mM Tris-HCl, 100 mM KCl, 5% glycerol, 1mM beta-Mercaptoethanol, pH 7.5).</P><H4>Protein Purification - <EM>Concentration</EM></H4><P>After dialysis, 4.5mL of sample of fusion protein was concentrated to a final volume of 0.6mL and  stored at -20℃, according to the purification protocol. Trace amounts of contaminating background was observed, however, the fusion was significantly more in quantity and the background was tolerated for functionality testing. The final concentration of the solution was determined according to the BCA protocol and resulted in a concentration of 0.56mg/mL.</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV></DIV></UL></FORM></DIV></DIV></DIV><DIV class="container nowbg" id="Sequencing"><DIV class="row nowbg"><DIV class="col-xs-12 col-sm-3 col-lg-3"><NAV id="myScrollspy" class="hidden-xs hidden-sm"><UL class="nav nav-pills nav-stacked" data-spy="affix"><LI class="dropdown"><A class="dropdown-toggle adpbl" data-toggle="dropdown" href="#">September</A></LI><LI><A href="#SeptemberSequencing1" class="adpbl">Week: 03/09/2018 - 09/09/2018</A></LI><LI><A href="#SeptemberSequencing2" class="adpbl">Week: 10/09/2018 - 16/09/2018</A></LI><LI><A href="#SeptemberSequencing3" class="adpbl">Week: 17/09/2018 - 23/09/2018</A></LI><LI><A href="#SeptemberSequencing4" class="adpbl">Week: 24/09/2018 - 30/09/2018</A></LI></UL><LI class="drowdown"><A class="dropdown-toggle adpbl" data-toggle="dropdown" href="#">October</A></LI><LI><A href="#OctoberSequencing1" class="adpbl">Week: 01/10/2018 - 07/10/2018</A></LI><LI><A href="#OctoberSequencing2" class="adpbl">Week: 08/10/2018 - 14/10/2018</A></LI></NAV></DIV><DIV class="col-lg-8 col-md-8 col-sm-12 col-xs-12 nowbg" style="margin-right:20px;"><H3 class="adpbl">September</H3><FORM><LABEL for="WeekSQS1" class="subdropdown cadpbl">Week 2 10/09/2018 - 16/09/2018</LABEL><UL class="submenu"><DIV class="subcontent"><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#WednesdaySQS1">Wednesday, 12/09/2018</BUTTON><DIV id="WednesdaySQS1" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>12/09/2018</DD><DT>Operator</DT><DD>Timothy Páez</DD></DL><H4>High Fidelity PCR</H4><P> With the objective to generate <I>EPO</I> biobrick with a Kanamycin fragment as intron, high fidelity PCR was done to amplify EPO biobrick from junction 2 between introns. Primers 103 and 104 were used to PCR amplify the backbone. At the same time, primers 105 and 106 were used to PCR amplify a fragment from Kanamycin from the plasmid pSB1K3. Primer sequences can be found at: <A href="https://2018.igem.org/Team:TUDelft/Primers" target="_blank"> Primer sequences</A>. 
    50 µL PCR mixtures were prepared, according to the High Fidelity PCR protocol.
    
    
    </P><P>The thermocycler programme was installed to have an extension time of 1.5 minutes and an annealing temperature of 65 °C. After the programme was finished, samples were kept at 4 °C.</P><DL><DT>Date</DT><DD>12/09/2018</DD><DT>Operator</DT><DD>Timothy Páez</DD></DL><H4>DNA Gel Electrophoresis</H4><P>PCR products of <I>EPO</I> plasmid and kanamycin PCR were run on a 0.8 % agarose gel in TAE buffer. 
    - 5 μL of PCR product was pipetted into each lane with 1 µL of loading buffer.
    - 4 µL of 500 bp DNA ladder 
    - 80 V for 30 minutes 
    - The gel was imaged using a gel documentation system with UV-light.</P><FIGURE><FIGCAPTION class="adpbl">Figure 1. Gel electrophoresis from EPO plasmid and Kanamycin fragment PCR. Lanes are (1-2) Amplification of EPO backbone, (3) negative control, (L) 500 bp ladder, (4-5) amplification of Kanamycin fragment and (6) negative control. </FIGCAPTION></FIGURE><DL><DT>Date</DT><DD>12/09/2018</DD><DT>Operator</DT><DD>Timothy Páez</DD></DL><H4>Gibson Assembly</H4><P>According to the displayed reaction mixture composition, Gibson Assembly was done to assemble Kanamycin fragment and Epo backbone to obtain EPO+Kan intron plasmid (3.2 kbp).
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Component</TH><TH class="tableheaderadpbl">Volume (uL)</TH></TR><TR class="tableunevenadpbl"><TD>Gibson Assembly Mastermix 2X</TD><TD>10</TD></TR><TR class="tableevenadpbl"><TD>Vector</TD><TD>EPO biobrick (100 ng)</TD></TR><TR class="tableunevenadpbl"><TD>Fragment</TD><TD>Kanamycin fragment (75 ng)</TD></TR></TBODY></TABLE></CENTER><P>The samples were incubated at 50°C for 60 minutes, after which they were kept on ice for subsequent transformation.</P><DL><DT>Date</DT><DD>12/09/2018</DD><DT>Operator</DT><DD>Timothy Páez</DD></DL><H4> Transformation of chemically competent cells </H4><P>Using the Gibson assembly reaction, the following transformations were carried out according to the protocol for transformation of chemically competent cells:
</P><CENTER><TABLE><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Strain</TH><TH class="tableheaderadpbl">Plasmid &amp; size (bp) </TH><TH class="tableheaderadpbl">Cloning method</TH><TH class="tableheaderadpbl">Volume added (uL)</TH><TH class="tableheaderadpbl">Medium on which was plated</TH></TR><TR class="tableunevenadpbl"><TD><I>E.coli</I>DH5αlpha</TD><TD>EPO backbone (2600 bp)</TD><TD>Restriction Ligation</TD><TD>10 µL</TD><TD>LB with Chloroamphenicol</TD></TR></TBODY></TABLE></CENTER><P>400 µL LB medium was added as recovery medium.
</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#FridaySQS1">Friday, 14/10/2018</BUTTON><DIV id="FridaySQS1" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>19/09/2018</DD><DT>Operator</DT><DD>Timothy Páez</DD></DL><H4>Colony PCR</H4><P>Colony PCR was done to verify propagation of <I>Escherichia coli</I> DH5 alpha cells after transformation with Gibson assembly product from 12_09_2018 containing EPO+Kan intron plasmid. This insertion can be checked with primers 103 and 104 (<I>EPO</I> amplification), Primer sequences can be found at: <A href="https://2018.igem.org/Team:TUDelft/Primers" target="_blank"> Primer sequences</A>.
</P><DL><DT>Date</DT><DD>14/09/2018</DD><DT>Operator</DT><DD>Timothy Páez</DD></DL><H4>DNA Gel Electrophoresis</H4><P>PCR products of <I>EPO+kan</I> intron plasmid and kanamycin PCR were run on a 0.8 % agarose gel in TAE buffer. 
    - 5 μL of PCR product was pipetted into each lane with no loading buffer.
    - 4 µL of 200 bp DNA ladder 
    - 80 V for 30 minutes 
    - The gel was imaged using a gel documentation system with UV-light.</P><FIGURE><FIGCAPTION class="adpbl">Figure 1. Gel electrophoresis from EPO+Kan intron plasmid PCR. Lanes are (1-4) Amplification of EPO backbone with kanamycin fragment intron, (L) 200 bp ladder and (5-6) negative control of only EPO plasmid. </FIGCAPTION></FIGURE></DIV></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV></UL></FORM><H3 class="adpbl">October</H3><FORM><LABEL for="WeekSQO1" class="subdropdown cadpbl">Week 2 08/10/2018 - 14/10/2018</LABEL><UL class="submenu"><DIV class="subcontent"><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#MondaySQO1">Monday, 08/10/2018</BUTTON><DIV id="MondaySQO1" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>08/10/2018</DD><DT>Operator</DT><DD>Timothy Páez</DD></DL><H4>Library preparation for MinION sequencing</H4><H5>Sequencing of sample: Lambda DNA</H5><P> Lambda DNA suplemented in the Rapid Sequencing kit from ONT MinION (RAD004) was library preped and sequenced following the protocol indicated for MinION sequencing. Protocols can be accessed here: <A href="https://2018.igem.org/Team:TUDelft/Experiments" target="_blank"> Protocols</A>.
</P><H6>Sequencing stats</H6><FIGURE><FIGCAPTION class="adpbl">Figure 1. Read length histogram for Lambda DNA sequencing run during 6 hours with MinION in Flow Cell FloMin106 with RAD004 kit for library preparation. </FIGCAPTION></FIGURE><H6>Data processing</H6><P> After sequencing run, the following analysis were performed:
    - Albacore software for base calling.
    - <I>De novo</I> assembly of reads with Canu for 40 kb genome size.
The assembly resulted in 1 contig that was aligned to a known reference sequence with &gt; 99% identity.
    </P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#WednesdaySQO1">Wednesday, 10/10/2018</BUTTON><DIV id="WednesdaySQO1" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>08/10/2018</DD><DT>Operator</DT><DD>Timothy Páez</DD></DL><H4>Library preparation for MinION sequencing</H4><H5>Sequencing of sample: Mix of DNA molecules at equimolar concentrations</H5><P> A solution with different types of DNA molecules was prepared before library preparation. This solution contained equimolar concentrations of different molecules, defined below:
    - EPO PCR amplification: 600 bp.
    - EPO + Kanamycin amplification: 1200 bp.
    - EPO Biobrick plasmid: 2600 bp
    - pACYC plasmid: 4000 bp  - EPO+Amp plasmid:2800bp
    This solution was library prepared library preped following the protocol indicated for MinION sequencing. Protocols can be accessed here: <A href="https://2018.igem.org/Team:TUDelft/Experiments" target="_blank"> Protocols</A>. After the fragmentation step 1, a new DNA molecule was added to asess weather unadapted DNA is sequenced or not. This sequence was:
    - dxCas9 PCR amplification: 4000 bp</P><H6>Sequencing stats</H6><FIGURE><FIGCAPTION class="adpbl">Figure 1. Read length histogram for Lambda DNA sequencing run during 6 hours with MinION in Flow Cell FloMin106 with RAD004 kit for library preparation. </FIGCAPTION></FIGURE><H6>Data processing</H6><P> After sequencing run, the following analysis were performed:
    - Albacore software for base calling.
    - Samptools multiple alignment to reference molecules
    - Samtools sort alignment, stats, view, and export as fasta mapped files. </P><H6>Sequencing stats</H6><FIGURE><FIGCAPTION class="adpbl">Figure 1. Read length histogram for Lambda DNA sequencing run during 6 hours with MinION in Flow Cell FloMin106 with RAD004 kit for library preparation. </FIGCAPTION></FIGURE></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV><BUTTON type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#ThursdaySQO1">Sunday, 14/10/2018</BUTTON><DIV id="ThursdaySQO1" class="modal fade" role="dialog"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><DL><DT>Date</DT><DD>14/10/2018</DD><DT>Operator</DT><DD>Timothy Páez</DD></DL><H4>Fusion protein functionality testing: integration of sequencing adapters on EPO backbone and Tn5 DNA
</H4><P>We aimed to perform the integration of adapters with our fusion protein on EPO biobrick circular DNA plasmid. </P><P>The components of the reagents can be found on the protocol. The sequence of the adapter used in this assay were sequencing adapters specified  at  <A href="https://2018.igem.org/Team:TUDelft/Primers" target="_blank"> Sequences</A> page.</P><P><B>Fusion protein:adapter DNA complex formation</B> 
Tn5 or fusion protein was loaded with adapter DNA shown above by following the pipetting scheme shown on Table 2. All samples and reagents were kept on ice during preparation.
</P><CENTER><TABLE><CAPTION class="figadpbl"><B>Table 2:</B> Pipetting scheme for transposome formation. </CAPTION><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Reaction with fusion protein (µl)</TH><TH class="tableheaderadpbl">Reaction no fusion protein (µl)</TH></TR><TR class="tableunevenadpbl"><TH>Buffer</TH><TD>1</TD><TD>0.6</TD></TR><TR class="tableevenadpbl"><TH>Fusion (~3µM)</TH><TD>8</TD><TD>0</TD></TR><TR class="tableunevenadpbl"><TH>Tn5 (~8µM)</TH><TD>0</TD><TD>0</TD></TR><TR class="tableevenadpbl"><TH>Adapter (25µM)</TH><TD>1</TD><TD>0.6</TD></TR><TR class="tableunevenadpbl"><TH>Water</TH><TD>0</TD><TD>4.8</TD></TR><TR class="tableevenadpbl"><TH>Total</TH><TD>10</TD><TD>6</TD></TR></TBODY></TABLE></CENTER><P>The reagents were pipetted in the following order: Water, 10x functionality buffer,  dxCas9-Tn5, and adapter. The samples were incubated at 37°C for 1 hour.</P><P>
To add appropriate amount of sgRNA, the pipetting scheme on Table 3 was followed. sgRNA J2.1 signifies version 1 of sgRNA targeting juntion 2 of EPO gene:
</P><CENTER><TABLE><CAPTION class="figadpbl"><B>Table 3:</B> Pipetting scheme for sgRNA loading.</CAPTION><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderadpbl">Reaction with sgRNA J2.1(µl)</TH><TH class="tableheaderadpbl">Reaction no sgRNA(µl)</TH></TR><TR class="tableunevenadpbl"><TH>Transposome</TH><TD>4</TD><TD>4</TD></TR><TR class="tableevenadpbl"><TH>sgRNA (100ng/µl)</TH><TD>0.8</TD><TD>0</TD></TR><TR class="tableunevenadpbl"><TH>MgCl<SUB>2</SUB> Buffer</TH><TD>0.2</TD><TD>0.2</TD></TR><TR class="tableevenadpbl"><TH>Water</TH><TD>0</TD><TD>0.8</TD></TR></TBODY></TABLE></CENTER><P>The samples were incubated at 37°C for 10 minutes. 
Target DNA (EPO cDNA) was added to make final concentration of ~5ng/µl. One negative control was done where water instead of EPO cDNA was added.
</P><P>The samples were incubated at 37°C. At incubation time = 1h. 2.5 µl of sample were taken from each reaction and was subjected to PCR. PCR mix and cycle can be found on protocol.

The PCR products were analysed on 5% native TBE polyacrylamide gel. The resulting gel is either stained with ethidium bromide and imaged with GelDoc system, or directly imaged with Typhoon Imaging System.
</P><B>Figure 1. 5% TBE native PAGE stained by EtBr and imaged by GelDoc system.Integration reaction after 1 hour.  </B><I>Lanes are (L) DNA ladder (100-1000bp), (1) amplified product with fw EPO (2) amplified product with rv EPO, (3) amplified product with fw EPO without sgRNA, (4) amplified product with rv EPO without sgRNA, (5) amplified product with fw EPO without fusion protein, (6) amplified product with rv EPO without fusion protein, (7) amplified product with fw EPO without adapter DNA, (8) amplified product with rv EPO without adapter DNA
  </I><P>Figure 1 did not show integration of adapters on plasmid DNA. The reaction was repeated with plasmid linearization before the integration. </P><P>Integration on linearized plasmid worked.</P></DIV><DIV class="modal-footer"><BUTTON type="button" class="btn btn-default" data-dismiss="modal">Close</BUTTON></DIV></DIV></DIV></DIV></DIV></UL></FORM></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>