Team:TUDelft/Wetlab/Protocols
Antibiotic Stock Solution
Preparation of a 1000X antibiotic stock solution:
Weigh X grams of the specific antibiotic of interest.
Resolve in 95% ethanol (Chloramphenicol) or MilliQ (other antibiotics).
BCA Protein Quantification
This protocol is based on the Pierce BCA protein assay kit by Thermo Scientific protocol.
Prepare a set of protein standards using one 2mg/mL Albumin Standard (BSA) ampule according to the table below:
NOTE: Use the same diluent as the samples. The expected working range = 20-2000µg/mL.
Vial
Volume of MilliQ (µL)
Source of BSA
Volume of source BSA (µL)
Final BSA concentration (µg/µL)
A
0
Stock
300
2000
B
125
Stock
375
1500
C
325
Stock
325
1000
D
175
Vial B
175
750
E
325
Vial C
325
500
F
325
Vial E
325
250
G
325
Vial F
325
125
H
400
Vial G
100
25
I
400
n/a
0
0
Determine the amount of total volume of working reagent (WR) required by using the following formula:
Total volume WR = (# standards + # unknowns) × (# replicates) × (200 µL)
NOTE: With pipetting you always take a bit extra (e.g. 10%), so not exactly the amount needed for x samples
Prepare the BCA working reagent by mixing 50 parts of BCA Reagent A with 1 part of BCA Reagent B (50:1, Reagent A:B).
NOTE: The WR is stable for several days when stored in a closed container at room temperature.
Pipette 25µL of each standard or unknown sample replicate into a microplate well.
Add 200µL of the WR to each well and mix plate thoroughly.
Cover plate and incubate at 37 °C for 30 minutes.
Cool plate to room temperature.
Measure the absorbance at or near 562nm on a plate reader.
Blunt End Ligation
Thaw the ligase buffer on ice, to prevent damaging the ATP
For phosphorylation, prepare a sample a sample as follows:
Component
Volume (µL)
10x Ligase buffer
2.5
T4 DNA Ligase
1
ATP
2.5
T4 PNK enzyme
1
DNA vector
X (~250 ng)
DNA fragment
Y*
MilliQ
Up to 25 µL
* NOTE: The desired vector:insert ratio will be 1:3. Use a ligation calculator to calculate the amounts of vector DNA and insert DNA to be added.
Incubate for at one hour at 37 °C.
Cancel phosphorylation process by incubating for 30 minutes at 65 °C.
Add 1 µL of Ligase T4 and incubate for at least four hours at 4 °C.
Blood Sample Preparation
Centrifuge the whole blood (9mL EDTA anticoagulated tube) at 1200 x g for 10 minutes.
Pipet of the plasma carefully (approximately 4 to 4.5mL) and put in a separate tube. When you only want the cfDNA and not the white blood cells be sure to not disturb the buffycoat with the pipet.
Centrifuge the plasma at 24000 x g for 20 minutes.
Take of the plasma without disturbing the pallet and aliquot into 1.5mL tubes, 1mL per tube.
Store at -20 °C.
For every follow up steps frozen aliquots can be taken from the freezer and thawed at room temperature. When thawed, the samples should be directly processed. Avoid thawing and freezing cycles, since this will cause degradation of cfDNA.
F. Thurik (2016). Cell-free placental DNA beyond Down syndrome.
Cell Banking
NOTE: All work is performed under asceptic conditions.
For one cryostock, take a 1.5mL sample from an overnight liquid culture .
Centrifuge the 2mL tubes at 2000rpm for 10 min.
Decant the supernatant without disturbing the pellet.
Add fresh sterile LB medium to the pellet, 1/3 volume of the starting volume of the culture.
Completely resuspended the pellet by vortexing the tube.
Add sterile 80% glycerol solution, the same volume as fresh LB in step 4.
Mix by vortexing.
Make a 1mL aliquot in cryotubes and label it with the cell type, plasmid type, protein type, operator and date.
Store the vials at -80 ºC and update the inventory.
Chemically Competent Cells Preparation
This protocol s over 3 days of execution time, starting from a -80 °C mother stock. Throughout the protocol, it is recommended to work under aseptic conditions in order to prevent contamination risks.
Day 1
Keep the -80 °C strain stock of interest on ice.
Streak the strain on solid selective medium and incubate overnight at 37 °C while shaking.
Day 2
Prepare a 10mL liquid starter culture with one of the colonies that grew on the selective plate. Let the culture grow overnight at 37 °C, shaking at 180rpm.
Sterilise solutions of 100 mM CaCl2 and of 100 mM CaCl2 + 15 % Glycerol in advance.
NOTE: Volumes depend on the total culture volume to be prepared in step 5. For example, if 1L is used in step 7, 300mL of 100mM CaCl2 and 10mL of 100mM CaCl2 + 15% glycerol will be used.
Day 3
Inoculate 1:100 of overnight culture in the desired volume of LB with antibiotic when required (eg. 10mL of culture in 1L of LB).
Incubate in a shaker at 37 °C, 180rpm to OD600nm ~0.4-0.6 (measure OD600nm every 30 minutes).
Harvests the cells by centrifugation at 4000 x g for 5 minutes in centrifuge tubes, decant supernatant.
Resuspend cells by gently pipetting 1/5 (of the volume of LB from step 5) of ice-cold 100 mM CaCl2 and incubate on ice for 20 minutes.
Pellet the cells by centrifugation at 4000 x g for 5 minutes in centrifuge tubes, decant supernatant.
Resuspend cells by gently pipetting 1/10 (of the volume of LB from step 5) of ice-cold 100 mM CaCl2 and incubate on ice for 60 minutes.
Pellet the cells by centrifugation at 4000 x g for 5 minutes in centrifuge tubes, decant supernatant.
Resuspend cells by gently pipetting 1/100 (of the volume of LB from step 5) of ice-cold 100 mM CaCl2 + 15% glycerol and keep on ice.
Chemical competent cells can either immediately be used for heat shock transformation or stored in aliquots of 50µL in microcentrifuge tubes at -80 °C.
Chemically Competent Cells Transformation
Get as many aliquots of competent cells (50µL) from the -80 °C freezer as transformations to be done and put them on ice for 10-15 min.
NOTE: Do not forget positive and negative controls (no DNA). If commercial competent cells (highly efficient) are used, an aliquot of 50µL can be split in two equal volumes of 25 µL and used for two transformations.
Add DNA of interest to the 50µL of competent cells.
Gibson Assembly mixture: 5µL
Ligation mixture: 5µL
Plasmids: 10 ng, volume depends on the plasmid concentration
Incubate on ice for 10-20 minutes.
Heat shock at 42 °C for exactly 45 seconds.
Add 200µL of hand-warm LB-medium.
Incubate at 37 °C with shaking (250 rpm) for 1 hour.
Plate the cells on LB-agar plates with the correct antibiotic to select for the introduced plasmid .  50-75µL can be plated on one plate. The remaining cell culture can be pelleted and resuspended in some residual supernatant prior to plating.
NOTE: When adding more than 200µL LB medium during step 5, plate out 100µL on selective medium. Then, briefly centrifuge the remaining cell culture and plate out the cell pellet on selective medium.
Incubate plates at 37 °C overnight.
NOTE: Alternatively, you can incubate over the weekend, leaving the plate on the bench.
Colony PCR
Under aseptic conditions, pick a colony, resuspend it in 10µL of MilliQ water.
NOTE: A picked colony cannot be used again; it is recommended to restreak on a 'back-up'-plate with the same pipet tips and incubate it overnight at 37 °C.
Incubate the resuspended colony at 90 °C for 10 min. Spin the suspension down and use the supernatant as template DNA for the PCR.
NOTE: Instead of separate boiling prior to PCR, this step can be incorporated in the PCR program. The initial denaturation step at 98 °C should then be prolonged to 5 minutes.
Make sure every PCR reaction is composed as follows:
Component
Volume (µL)
Final concentration
Gotaq 5X buffer*
10
1X
10mM dNTPs
1
200µM
Primer forward (10µM)
1
200nM
Primer reverse (10µM)
1
200nM
Boiled colony supernatant
5
Gotaq polymerase (5U/µL)
0.2
20U/mL
MilliQ
31.8
* NOTE: Use Gotaq Buffer Green when it is required to run a verification gel afterwards.
Add 5µL of supernatant of colony mixture to each PCR tube.
Close all tubes thoroughly and place them in a thermocycler with the following protocol:
Step
Time (s)
Temperature (°C)
Initial denaturation
150
98
Denaturation
60
94
Annealing
60
Tann*
Extension
60 sec per kb DNA
72
Final extension
600
72
Hold
∞
4
*NOTE: The annealing temperature (Tann) is dependent on the melting temperature (Tm) of the primers used. It is recommended to have Tann = Tm - 5 °C.
The PCR product(s) can be checked on a agarose gel. In order to do so, cast a gel and prepare the samples according to the DNA electrophoresis protocol.
Dextrin-Capped Gold Nanoparticles (d-AuNPs) Generation
The generation of dextrin-capped gold nanoparticles generation was performed according to the protocol described by Anderson et al. 2011 .
Prepare the following solutions:
20mM gold chloride salt (HAuCl4) in MilliQ. Store at 4 ºC.
25g/L dextrin in MilliQ.
10% (w/v) sodium carbonate (Na2CO3) in MilliQ.
dH2O pH=9, adjusted with sodium hydroxide (NaOH).
Add 25mL of the dextrin solution (25g/L) to a sterile 250mL flask.
Add 5mL of the Gold Chloride salt solution (20mM).
Adjust pH of the solution to pH=9 with 10% (w/v) Na2CO3 by checking the pH with pH indicator strips.
Complete the reaction mixture by adding dH2O pH=9 up to a total reaction volume of 50mL.
Incubate the flask at 50 ºC in the dark with continuous shaking (250rpm)  for 3 hours.
Measure the absorbance spectrum of the sample every 20 minutes.
NOTE: Change in color to red is the final indication of the ion Au+3 reduction to Au0.
When the reaction mixture shows a clear shift in color to red, stop the reaction and store the dextrin-capped gold nanoparticle batch at room temperature in a closed glass container in the dark.
References:
Anderson, M.J. et al., 2011. One step alkaline synthesis of biocompatible gold nanoparticles using dextrin as capping agent. Journal of Nanoparticles Research, 13, pp2843–2851
Dextrin-Capped Gold Nanoparticles (d-AuNPs) Testing
Evaluating of the functionality of dextrin-capped gold nanoparticles is performed in a transparent 96 well plate with flat bottom. There are several analyses being conducted, which are:
Salt stability
ssDNA induced stability.
ssDNA and target DNA stability.
The following table indicates different solutions needed for each type of analysis:
Analysis
d-AuNPs (µL)
ssDNAp (µL)
NaCl solution (X mM) (µL)
dsDNA target (µL)
MilliQ / Hybridization buffer (µL)
Salt stability
20
0
20
0
20
ssDNA induced stability
20
4
20
0
16
ssDNA and target dsDNA induced stability
20
4
20
100
6
For each specific functionality testing reaction, perform the following steps directly in the wells of the 96 wells plate, unless indicated otherwise.
Salt Stability
Add 20µL of MilliQ.
Add 20µL of NaCl solutions (varying concentrations).
Use one well as salt blank by adding 20µL MilliQ instead of NaCl solution.
Add 20µL of d-AuNPs.
Mix gently by pipetting and incubate at 21 ºC during 10 minutes.
Quantify the visible absorption spectrum of the solutions or measure absorbance at 520nm and 620nm.
ssDNAp Induced Stability
Add 4µL of ssDNAp (1µM).
Use one wells as ssDNAp blank by adding 4 µL MilliQ instead of ssDNAp.
Add 16µL of MilliQ.
Add 20µL of NaCl solutions (varying concentrations).
Use one well as salt blank by adding 20µL MilliQ instead of NaCl solution.
Add 20µL of d-AuNPs.
Mix gently by pipetting and incubate at 21 ºC during 10 minutes.
Quantify the visible absorption spectrum of the solutions or measure absorbance at 520nm and 620nm.
ssDNAp and dsDNA Target Induced Stability
Prepare the following reaction in a PCR tube:
Add 4µL of ssDNAp (1µM).
Use one well as ssDNAp blank by adding 4µL MilliQ instead of ssDNAp.
Add 10µL of dsDNA target (~1nM).
Add 6µL of Hybridization buffer.
Mix gently by pipetting and add tubes on thermal cycler with the following program:
Step
Time (s)
Temperature (°C)
Denaturation
300
95
Annealing
60
57
Relaxation
600
20
Add 20µL of the reaction mixture into the wells of 96 well plate
Add 20µL of NaCl solutions (varying concentrations or defined concentration based on previous results).
Use one well as salt blank by adding 20µL MilliQ instead of NaCl solution.
Add 20µL of d-AuNPs.
Mix gently by pipetting and incubate at 21 ºC during 10 minutes.
Quantify the visible absorption spectrum of the solutions or measure absorbance at 520nm and 620nm.
DNA Gel Electrophoresis
Gel Electrophoresis and DNA staining makes use of mutagenic chemicals like EtBr, SYBR Safe or any other DNA staining. Wear protection (gloves) when carrying out this protocol and work in an assigned area for this work to prevent contamination of the rest of the lab.
For a 0.8% (w/v) gel in TAE, weigh agarose powder for a 0.8% (w/v) gel.
E.g. weigh 1.6g and add to 200mL TAE buffer for a 0.8% gel.
Add weighed agarose in TAE buffer (1X) and warm the solution (in a microwave) until it is completely dissolved. Let the solution cool down to hand warm temperature.
NOTE: Make sure the lid is not completely closed to avoid possible exploding of the glass bottle.
Pour gel in gel tray and mix well with SYBR Safe. For a small gel (~ 40mL solution) add 1µL of SYBR Safe; for a large gel (~80mL) add 2µL of SYBR Safe. Add a comb to create wells for the samples. Allow the agarose to solidify (approximately 20 minutes).
NOTE: For better mixing, add SYBR Safe to bottle instead of tray.
Transfer the gel to the electrophoresis cell, remove the combs and cover the gel  in TAE buffer (1X).
NOTE: Mind the direction of DNA migration when placing the gel in the cell.
Prepare the electrophoresis samples by adding Nucleic Acid Loading Buffer conform the manufacturer’s instructions.
Load the molecular weight marker (DNA ladder) in the first well according to manufacturer’s instructions (generally 3-5µL) and load 5-10µL of the dyed samples in the other wells.
NOTE: Do not contaminate the loading buffer and ladder with SYBR Safe! Do not touch it while wearing a glove.
Connect the cables of the gel tray following the colour code and run at 80-110V for 40-60 min.
NOTE: Mind the direction of DNA migration when placing the lid on the cell.
NOTE: Time and voltage depend on the density of the gel and the length of the lane in the gel.
DNA Purification from Agarose Gel
This protocol is based on the Wizard® SV Gel and PCR Clean-Up System of Promega Corporation.
Excise a proper band containing the DNA from gel and place the excised gel slice in a 1.5 mL Eppendorf tube.
Weigh the excised gel slice.
Add 10µL Membrane Binding Solution per 10 mg of gel slice. Vortex and incubate at 50-65 °C until gel slice is completely dissolved.
Insert the SV Minicolumn into the Collection Tube and label both of them according to the labelling of your samples.
Transfer the dissolved gel mixture to the Minicolumn assembly. Incubate the Minicolumn at room temperature for 1 minute.
Centrifuge the SV Minicolumn at maximum speed for 1 minute.
Discard the flow through and reinsert the SV Minicolumn into the Collection Tube.
Add 700µL of Membrane Wash Solution.
NOTE: Upon prior use, dilute the solution with 95% ethanol following the manufacturer's’ instructions.
Centrifuge the SV Minicolumn assembly at maximum speed for 1 minute.
Discard the flow through and reinsert the SV Minicolumn into the Collection Tube.
Repeat the washing step with 500µL of Membrane Wash Solution and centrifuge for 5 minutes at maximum speed.
Once the Collection Tube is empty, centrifuge the Minicolumn assembly at maximum speed for 1 minute with the Minicolumn assembly lid open to allow full evaporation of ethanol
NOTE: Leaving the column at room temperature ameliorates evaporation of residual ethanol traces.
Transfer the SV Minicolumn to a clean labelled 1.5mL Eppendorf tube.
Add 50µL of pre-warmed MilliQ directly to the centre of the SV Minicolumn, without touching the membrane with the pipette tip.
NOTE: Use 30µL when higher final concentrations of DNA are required or when small quantities of DNA are suspected.
Incubate the SV Minicolumn at room temperature for 5 minutes.
Centrifuge at maximum speed for 1 minute.
Discard the SV Minicolumn, cap the tube containing the eluted DNA, measure the DNA concentration and keep the DNA at 4 °C (for immediate use) or -20 °C (for storage).
DpnI Digestion
Our DpnI digestions were performed with New England Biolabs DpnI (20.000 units/mL) , which is compatible with CutSmart Buffer by the same manufacturer.
Prepare a sample in a 0.5mL microcentrifuge tube as follows:
Component
Volume (µL)
10x CutSmart buffer (NEB)
4
Purified PCR product
30
Restriction Enzyme DpnI
1
MilliQ
5
Incubate for 1.5 hours at 37 °C.
Heat inactivate the enzyme by incubating at 80 °C for 20 minutes.
dxCas9 Protein Functionality Testing: Mobility Shift
This protocol describes how the functionality of the gRNA:dxCas9:DNA complex or the gRNA:dxCas9-Tn5:DNA complex is evaluated to bind to the target DNA using a mobility shift assay. The protocol consists of two parts: binding of dxCas9(-Tn5) to the gRNA and binding of the gRNA:dxCas9(-Tn5) to the target DNA
gRNA:dxCas9(-Tn5) complex formation
NOTE: all samples and reagents are kept on ice during preparation.
Load the purified dxCas9 or dxCas9-Tn5 protein with gRNA provided by Arbor Biotechnologies by combining the following components:
1-100nM dxCas9 or dxCas9-Tn5
1.6-160nM gRNA
NOTE: the gRNA and dxCas9 or dxCas9-Tn5 is mixed in a 1:1.6 molar ratio.
10X functionality buffer (20mM Tris-HCl, 100mM KCl, 5% v/v glycerol, 1mM beta-mercaptoethanol, 10mM MgCl2 pH 7.5)
MilliQ (add up to a final volume of 10µL)
Pipette the reagents in the following order in a 0.5mL tube: MilliQ, 10X functionality buffer, dxCas9 or dxCas9-Tn5 protein and gRNA.
Incubate at 37 °C for 10 minutes.
gRNA:dxCas9(-Tn5):Target DNA complex formation
Add 1nM Target DNA (our case EPO cDNA) to the gRNA:dxCas9 or gRNA:dxCas9-Tn5 complex.
Incubate at 37 °C for 1 hour.
Analyse the samples on a 5% native TBE polyacrylamide gel to observe the mobility shift.
dxCas9 Protein Functionality Testing: Trypsin Resistance
This protocol describes how the trypsin resistance of the gRNA-loaded dxCas9 and dxCas9-Tn5 proteins were evaluated due to a protein conformational change. The protocol consists of two parts: binding of dxCas9(-Tn5) to the gRNA and the trypsin resistance assay
gRNA:dxCas9(-Tn5) complex formation
NOTE: all samples and reagents are kept on ice during preparation.
Load the purified dxCas9 or dxCas9-Tn5 protein with gRNA provided by Arbor Biotechnologies by adding the following components:
1-100nM dxCas9 or dxCas9-Tn5
1.6-160nM gRNA
NOTE: the gRNA and dxCas9 or dxCas9-Tn5 is mixed in a 1:1.6 molar ratio.
10X functionality buffer (20mM Tris-HCl, 100mM KCl, 5% v/v glycerol, 1mM beta-mercaptoethanol, 10mM MgCl2 pH 7.5)
MilliQ (add up to a final volume of 10µL)
Pipette the reagents in the following order in a 0.5mL tube: MilliQ, 10X functionality buffer, dxCas9 or dxCas9-Tn5 protein and gRNA.
Incubate at 37 °C for 10 minutes.
Trypsin Resistance Assay
Add 0.1-10nM Trypsin to the gRNA:dxCas9 or gRNA:dxCas9-Tn5 complex.
Incubate at 23 °C for 30 minutes.
Analyse the samples on a 8% SDS Tris-Glycine polyacrylamide gel to observe the trypsin resistance.
Electrocompetent Cells Preparation
This protocol spans over 3 days of execution time, starting from a -80 °C mother stock. Throughout the protocol, it is recommended to work under aseptic conditions in order to prevent contamination risks.
Day 1
Sterilize the required 50mL MilliQ and 250µL glycerol in advance.
Keep the -80 °C strain stock of interest on ice.
Streak the strain on solid selective medium and incubate overnight at 37 °C.
Day 2
Prepare a 10mL liquid starter culture with one of the colonies that grew on the selective plate. Let the culture grow overnight at 37 °C, shaking at 180rpm.
Day 3
Use 0.5mL of the starter culture to inoculate 35mL selective liquid medium.
Grow at 37°C while shaking 250 rpm till an OD600 of ~0.5.
Centrifuge for 10 minutes at 4 °C at 3900rpm.
Discard the supernatant and resuspend pellet in 20mL cold MilliQ.
Centrifuge for 10 minutes at 4 °C at 3900rpm.
Discard the supernatant and resuspend pellet in 20mL cold MilliQ.
Centrifuge for 10 minutes at 4 °C at 3900rpm.
Discard supernatant and resuspend in 200µL 50% glycerol.
Prepare aliquots of 50µL.
Either transform the electrocompetent cells straight away or store the electrocompetent cells at -80 °C.
Electrocompetent Cells Transformation
NOTE: This protocol allows for a single transformation.
Thaw a 50µL aliquot of electrocompetent cells on ice.
Add ~200 ng DNA to the cells and keep on ice for 20 minutes.
NOTE: in case of purified plasmid DNA, 1ng DNA is enough.
Transfer all the content to an electro-shock cuvette.
Electro-shock the cells with the Electro Cell Manipulator at 2.5kV.
Immediately add 0.2-1mL of recovery medium (eg. SOC-medium). Resuspend and transfer to a 1.5mL tube.
Incubate at 37 °C at 250rpm for 1 hour.
Plate the cells on solid medium with appropriate antibiotics and incubate overnight at 37 °C.
G-Block Resuspension
This protocol is based on the instructions for G-Block resuspension given by Integrated DNA Technologies (IDT) .
Centrifuge the tube containing the gBlock for 3−5 seconds (>3,000 x g) to pellet the material to the bottom of the tube.
Add an appropriate volume of sterile MilliQ to the tube for a desired final concentration. The required volume of MilliQ can be read from the table below (and the label on the IDT tube that contains your fragment):
Final concentration (ng/µL)
µL MilliQ to add to 250ng
µL MilliQ to add to 250ng
µL MilliQ to add to 250ng
10
25
50
100
20
not recommended
25
50
50
not recommended
10
20
Incubate at 50 °C for 20 minutes.
Briefly vortex and centrifuge.
Store the resuspended gBlock at -20 °C.
Genomic DNA Isolation
This protocol is based on MO BIO Laboratories Inc. UltraClean® Microbial DNA Isolation Kit. For more information on the composition of the MD solutions, and for safety precautions, we recommend consulting MO BIO Laboratories’ kit manual. .
Add 1.8 ml of microbial (bacteria, yeast) culture to a 2mL Collection Tube (provided) and centrifuge at 10,000 x g for 30 seconds at room temperature. Decant the supernatant and spin the tubes at 10,000 x g for 30 seconds at room temperature and completely remove the media supernatant with a pipette tip. NOTE: Based on the type of microbial culture, it may be necessary to centrifuge longer than 30 seconds.
Resuspend the cell pellet in 300µl of MicroBead Solution and gently vortex to mix. Transfer resuspended cells to MicroBead Tube.
Check Solution MD1. If Solution MD1 is precipitated, heat the solution at 60 °C until the precipitate has dissolved. Add 50µl of Solution MD1 to the MicroBead Tube.
Optional: To increase yields, to minimize DNA shearing, or for difficult cells, see Alternative lysis methods in the “Hints & Troubleshooting Guide” section before continuing.
Secure MicroBead Tubes horizontally using the MO BIO Vortex Adapter tube holder for the vortex (MO BIO Catalog# 13000-V1) or secure tubes horizontally on a flat-bed vortex pad with tape. Vortex at maximum speed for 10 minutes. (See “Hints & Troubleshooting Guide” for less DNA shearing).
Make sure the 2mL MicroBead Tubes rotate freely in the centrifuge without rubbing. Centrifuge the tubes at 10,000 x g for 30 seconds at room temperature.
CAUTION: Be sure not to exceed 10,000 x g or tubes may break.
Transfer the supernatant to a clean 2mL Collection Tube (provided). Expect 300-350µl of supernatant.
Add 100 µl of Solution MD2, to the supernatant. Vortex for 5 seconds. Then incubate at 4 °C for 5 minutes.
Centrifuge the tubes at room temperature for 1 minute at 10,000 x g.
Avoiding the pellet, transfer the entire volume of supernatant to a clean 2mL Collection Tube (provided). Expect approximately 450µL in volume.
Shake to mix Solution MD3 before use. Add 900 µl of Solution MD3 to the supernatant and vortex for 5 seconds.
Load about 700µL into the Spin Filter and centrifuge at 10,000 x g for 30 seconds at room temperature. Discard the flow through, add the remaining supernatant to the Spin Filter, and centrifuge at 10,000 x g for 30 seconds at room temperature. NOTE: A total of 2 to 3 loads for each sample processed are required. Discard all flow through liquid.
Add 300µL of Solution MD4 and centrifuge at room temperature for 30 seconds at 10,000 x g.
Discard the flow through.
Centrifuge at room temperature for 1 minute at 10,000 x g.
Being careful not to splash liquid on the spin filter basket, place Spin Filter in a new 2mL Collection Tube (provided)
Add 50µL of Solution MD5 to the center of the white filter membrane.
Centrifuge at room temperature for 30 seconds at 10,000 x g.
Discard Spin Filter. The DNA in the tube is now ready for any downstream application. No further steps are required. Store the DNA at -20 °C until further use.
Gibson Assembly
This Gibson Assembly protocol is based on the protocol provided by New England Biolabs .
Thaw 10µL of 2X Gibson Assembly mastermix (New England Biolabs) on ice.
Add backbone and insert; depending on the assembly, assembly pieces can be added in a predetermined ratio (recommended is ratio 1:3 mol vector over mol insert, so not a 1:3 ratio based on weight). Do not exceed the total volume of 10µL.
NOTE: An online ligation calculator can subsequently be used to calculate the amount of the assembly pieces that is required.
If applicable, fill up the reaction volume to 20µL with MilliQ.
Component
Volume (µL)
Gibson Assembly Master Mix 2X (NEB)
10
Vector
X (? ng)
Insert fragment
Y (? ng)
MilliQ
10-X-Y
NOTE: The vector should have compatible overhangs with the insert(s)
Incubate the assembly reaction at 50 °C for 60 minutes and place on ice for subsequent chemical transformation or electroporation . Otherwise, store at -20 °C.
High Fidelity PCR
For each PCR in 50µL reaction volume, make sure the composition is as follows:
Component
Volume (µL)
Final concentration
5X Phusion HF buffer
10
1X
10mM dNTPs
1
200µM
Primer forward (10µM)
2.5
200nM
Primer reverse (10µM)
2.5
200nM
DNA template
~10ng to 250ng*
Phusion polymerase
0.5
20U/mL
MilliQ
up to 50µL
* NOTE: The 10ng - 250ng of template DNA are approximate, choose a volume that works fine for all your samples.
NOTE: High fidelity PCR with Phusion polymerase can be optimized per case by adding 3% of DMSO.
Close all tubes thoroughly and place them in a thermocycler with the following protocol:
Step
Temperature (°C)
Time (s)
Initial denaturation
98
30
Denaturation
98
10
Annealing
Tann*
15
Extension
72
15-30 sec per kb DNA
Final extension
72
300
Hold
4
∞
* NOTE: The annealing temperature (Tann) is dependent on the melting temperature (Tm) of the primers used. It is recommended to have Tann = Tm - 5 °C.
The PCR product(s) can be checked on agarose gel. In order to do so, cast a gel and prepare the samples according to the DNA electrophoresis protocol.
Ligation
Thaw the ligase buffer on ice, to prevent damaging the ATP.
Prepare a sample as follows:
Component
Volume (µL)
10X Ligase buffer
2
T4 DNA Ligase
1
DNA vector
X (~100ng)
DNA fragment
Y*
MilliQ
17-X-Y
* NOTE: The desired vector:insert ratio will be 1:3. Use a ligation calculator to calculate the amounts of vector DNA and insert DNA to be added.
Incubate for at least one hour at 4 °C. Optimally, incubate overnight at 4 °C.
Liquid Medium Preparation
Dissolve Luria Broth powder in water according to instructions by manufacturer.
Heat sterilize (121 °C) the medium in the autoclave.
After cooling down, add the required antibiotic under aseptic conditions.
NOTE: With our antibiotic stock solutions of 1000x we used 1µL of antibiotic solution per mL of LB-agar.
Store the medium at 4 °C if complemented with antibiotics.
Liquid Starter Culture (10mL)
Label as many 15mL Falcon tubes/Erlenmeyer flasks as the number of colonies you want to grow.
Under aseptic conditions, distribute 10mL of liquid medium (LB or SOC) for each starter culture.
NOTE: Supply with relevant antibiotics.
Under aseptic conditions, pick a colony with the inoculation loop and swirl it in the starter culture to inoculate.
NOTE: A picked colony cannot be used again; it is recommended to restreak on a 'back-up'-plate and incubate it overnight at 37 °C.
Grow the liquid cultures at 37 °C overnight, shaking at 250rpm.
MinION Flow Cell Washing
This protocol describes the steps to be taken to wash a flow cell after performing a sequencing run.
Thaw components of the Wash kit (WKE_1012) at room temperature.
After the sequencing run has stopped, add 150µL of Buffer A to the priming port (without opening the SpotOn port).
Incubate for 10 minutes at room temperature.
For using immediately:
Add 500 µL of buffer B slowly on the priming port.
Slowly take out the waste solution from the waste chamber.
The flow cell is ready for priming and adding a new DNA library.
For storage:
Add 500 µL of storage buffer slowly on the priming port.
Slowly take out the waste solution from the waste chamber.
Store flow cell at 4 ºC.
Nanodrop DNA quantification
This protocol intends to quantify DNA amount, as well as the purity, in DNA purification or extraction samples, through spectroscopy. Make sure the Nanodrop machinery is installed to measure at wavelengths for dsDNA (260nm) and impurities at 230nm and 280nm.
Clean the measurement surface with a piece of tissue and ethanol.
Use 1µL of sterile MilliQ as a blank.
Clean the measurement surface with a piece of tissue.
Use 1µL of a sample to measure its concentration.
NOTE: It is best to measure the same sample in triplo and use the average value. If you have multiple samples, clean the measurement surface in between measurements.
When done measuring, wipe the measurement surface with a tissue and MilliQ.
Nanodrop Protein quantification
This protocol intends to quantify protein amount, as well as the purity, in purification or extraction samples, through spectroscopy. Make sure the Nanodrop machinery is installed to measure at wavelengths for 280nm and 260nm.
Clean the measurement surface with a piece of tissue and ethanol.
Use 2µL of sterile MilliQ as a blank.
Clean the measurement surface with a piece of tissue.
Use 2µL of a sample to measure its concentration.
NOTE: It is best to measure the same sample in triplo and use the average value. If you have multiple samples, clean the measurement surface in between measurements.
When done measuring, wipe the measurement surface with a tissue and MilliQ.
Nanodrop RNA quantification
This protocol intends to quantify RNA amount, as well as the purity, in purification or extraction samples, through spectroscopy. Make sure the Nanodrop machinery is installed to measure at wavelengths for ssRNA (260nm) and impurities at 230nm and 280nm.
Clean the measurement surface with a piece of tissue and ethanol.
Use 1µL of sterile MilliQ as a blank.
Clean the measurement surface with a piece of tissue.
Use 1µL of a sample to measure its concentration.
NOTE: It is best to measure the same sample in triplo and use the average value. If you have multiple samples, clean the measurement surface in between measurements.
When done measuring, wipe the measurement surface with a tissue and MilliQ.
Native TBE Polyacrylamide Gel: Ethidium Bromide Staining
A native PAGE electrophoresis is used to separate DNA based on their size by using an electric current. This protocol describes how to prepare a native PAGE gels, how to prepare samples and how to run the electrophoresis.
NOTE: In this protocol we use a 5% separation gel (this is for DNA samples between 50 bp and 1,000 bp).
Preparing the native PAGE gel
Prepare the following solutions for 2 gels (total 12mL):
Component
Volume
40% Acrylamide
1.5mL
10X TBE buffer
1.2mL
10% APS
120µL
TEMED
12µL
MilliQ
9.168mL
NOTE: Polymerization will start when 10% APS and TEMED are added.
Assemble the casting station and pour the separation gel between the glass plates.
Immediately place the gel comb on top of the gel.
Allow the gel to solidify, then remove the plates from the casting station.
The gels are ready for usage, or can be stored at 4 °C for later usage.
Preparing samples
Add 2μL protein 6X Loading Dye to 10μL of sample.
Running PAGE gel
Run the gel at 150V and 25mA.
The gel is finished when the purple line is at the end of the gel.
Processing PAGE gel: Ethidium bromide staining
Turn off the power pack before opening the gel box.
Remove the gel from the cassette and place the gel in a clean staining tray of the appropriate size filled with a layer of MilliQ or TBE buffer.
Gently allow the gel to slide from the glass plate into the water.
Add 1µL of Ethidium bromide into the water.
Allow the gel to shake on a moving platform for 10 minutes.
Take a picture with the Gel doc system/Typhoon.
Native TBE Polyacrylamide Gel: Silver Staining
A native PAGE electrophoresis is used to separate DNA based on their size by using an electric current. This protocol describes how to prepare a native PAGE gels, how to prepare samples and how to run the electrophoresis.
NOTE: In this protocol we use a 5% separation gel (this is for DNA samples between 50 bp and 1,000 bp).
Preparing te native PAGE gel
Prepare the following solutions for 2 gels (total 12mL):
Component
Volume
40% Acrylamide
1.5mL
10X TBE buffer
1.2mL
10% APS
120µL
TEMED
12µL
MilliQ
9.168mL
NOTE: Polymerization will start when 10% APS and TEMED are added.
Assemble the casting station and pour the separation gel between the glass plates.
Immediately place the gel comb on top of the gel.
Allow the gel to solidify, then remove the plates from the casting station.
The gels are ready for usage, or can be stored at 4 °C for later usage.
Preparing samples
Add 2μL protein 6X Loading Dye to 10μL of sample.
Running PAGE gel
Run the gel at 150V and 25mA.
The gel is finished when the purple line is at the end of the gel.
Processing PAGE gel: Silver staining
Turn off the power pack before opening the gel box.
Remove the gel from the cassette and place the gel in a clean staining tray of the appropriate size filled with a layer of MilliQ or TBE buffer.
Gently allow the gel to slide from the glass plate into the water.
Discard the water.
Prepare the following solutions for silver staining with SilverQuest Silver staining kit.
Solution
Components
Fixing solution
10mL Acetic Acid
40mL Ethanol
50mL MilliQ
Sensitizing solution
30mL Ethanol
10mL Sensitizer
60mL MilliQ
Staining solution
1mL Silver Stainer
100mL MilliQ
Developing solution
10mL Developer
1 drop Developer Enhancer
90mL MilliQ
NOTE: All incubations are performed on a rotary shaker rotating at a speed of 1 revolution/second at room temperature.
Cover the gel with 100mL fixing solution for one hour with gentle rotation.
NOTE: The gel can be stored in the fixative overnight if there is not enough time to complete the staining protocol.
Decant the fixative solution and wash the gel in 30% ethanol for 10 minutes.
Decant the ethanol and wash the gel in 100mL of Sensitizing solution for 10 minutes.
Decant the Sensitizing solution and wash the gel in 100mL of 30% ethanol for 10 minutes.
Decant the 30% ethanol and wash the gel in 100mL of ultrapure water for 10 minutes.
Decant the water and indicate the gel in 100mL of Staining solution for 15 minutes.
Decant the Staining solution and wash the gel with 100mL of ultrapure water for 20–60 seconds.
Note: Washing the gel for more than a minute will remove silver ions from the gel resulting in decreased sensitivity.
Incubate the gel in 100mL of Developing solution for 4–8 minutes until bands start to appear and the desired band intensity is reached.
After the appropriate staining intensity is achieved, immediately add 10 mL of Stopper directly to the gel still immersed in Developing solution.
Gently agitate the gel for 10 minutes.
Note: The color changes from pink to colorless indicating that the development has stopped.
Decant the Stopper solution and wash the gel with 100 mL of ultrapure water for 10 minutes.
Take a picture with the Gel doc system.
PCR Clean-Up
This protocol is based on the Wizard® SV Gel and PCR Clean-Up System of Promega Corporation.
Add an equal volume of Membrane Binding Solution to the volume of PCR product.
Insert the SV Minicolumn into Collection Tube and label both of them according to the labelling of your samples.
Transfer the dissolved gel mixture to the Minicolumn assembly. Incubate the Minicolumn at room temperature for 1 minute.
Centrifuge the SV Minicolumn at maximum speed for 1 minute.
Discard the flow through and reinsert the SV Minicolumn into the Collection Tube.
Add 700µL of Membrane Wash Solution.
NOTE: Upon prior use, dilute the solution with 95% ethanol following the manufacturer's’ instructions.
Centrifuge the SV Minicolumn assembly at maximum speed for 1 minute.
Discard the flow through and reinsert the SV Minicolumn into the Collection Tube.
Repeat the washing step with 500µL of Membrane Wash Solution and centrifuge for 5 minutes at maximum speed.
Once the Collection Tube is empty, centrifuge the Minicolumn assembly at maximum speed for 1 minute with the microcentrifuge lid open to allow ethanol full evaporation.
NOTE: Leaving the column at room temperature ameliorates evaporation of residual ethanol traces.
Transfer the SV Minicolumn to a clean labelled 1.5mL Eppendorf tube.
Add 50µL of pre-warmed MilliQ directly to the centre of the SV Minicolumn, without touching the membrane with the pipette tip.
NOTE: Use 30µL when higher final concentrations of DNA are required or when small quantities of DNA are suspected.
Incubate the SV Minicolumn at room temperature for 5 minutes.
Centrifuge at maximum speed for 1 minute.
Discard the SV Minicolumn, cap the tube containing the eluted DNA and keep the DNA at 4 °C (for immediate use) or -20 °C (for storage).
p-GEM T-Easy Cloning
The protocol is based on Promega’s pGEM T-Easy Vector System technical manual.
pGEM T-Easy vector system can be used to clone PCR amplification products by Taq polymerase on separate plasmids. The following steps are done to complete the protocol: Briefly centrifuge the pGEM.-T Easy Vector and insert DNA tubes.
Set up ligation reactions in the following order:
5µl 2x Rapid Ligation Buffer, T4 DNA Ligase
1µl pGEM-T Easy vector
Xµl PCR product (to obtain 3:1 insert:vector molar ratio)
1µl T4 DNA Ligase
Xµl MilliQ to 10µl
Incubate overnight at 4C.
Transform ligates into competent cells according to the chemical competent cell transformation protocol .
Blue/white colony screening to chose for colonies with inserts.
To calculate the amount of PCR product needed, the following formula can be used:
Plasmid Isolation
This protocol is based on the protocol supplied with the Promega PureYield™ Plasmid Miniprep Kit of Promega Corporation.
Centrifuge 1.5-3mL of a liquid (starter) culture for 1 minute in a 1.5mL microcentrifuge tube at maximum speed.
Discard the supernatant and resuspend in 600µL of MilliQ.
Add 100µL of Cell Lysis Buffer, and mix by inverting the tube.
Add 350µL of cold (4–8 °C) Neutralization Solution, and mix thoroughly by inverting.
Centrifuge at maximum speed in a microcentrifuge for 3 minutes.
Transfer the supernatant (~900µl) to a PureYield™ Minicolumn without disturbing the cell debris pellet.
Place the minicolumn into a Collection Tube and centrifuge at maximum speed in a microcentrifuge for 15 seconds.
Discard the flow through, and place the minicolumn into the same Collection Tube.
Add 200µl of Endotoxin Removal Wash (ERB) to the minicolumn. Centrifuge at maximum speed in a microcentrifuge for 15 seconds.
Add 400µl of Column Wash Solution (CWC) to the minicolumn. Centrifuge at maximum speed in a microcentrifuge for 30 seconds.
Transfer the minicolumn to a clean, labelled 1.5 ml Eppendorf tube, then add 30µl of sterile MilliQ directly to the minicolumn matrix. Incubate for 1 minute at room temperature.
Centrifuge for 15 seconds to elute the plasmid DNA. Discart the column and label the tube. The isolate can be stored at -20 °C or directly used.
Primer Working Stock
This protocol is based on the instructions for primer resuspension given by Integrated DNA Technologies (IDT).
Centrifuge the tube containing the primer for 3−5 seconds (>3,000 x g) to pellet the material to the bottom of the tube.
Dissolve the DNA material in sterile MilliQ according to the supplier IDT. This creates a 100µM stock solution.
Heat the primer stock solution to 65 °C for 20 minutes.
Centrifuge the primer stock solution at maximum speed (~17,000 x g) for 2 minutes.
Prepare a 10X diluted work solution (10µM) by dilution with sterile MilliQ.
Store both Stock and Work solutions at -20 °C.
Protein Expression
All work was performed within a sterile field created by a bunsen burner flame.
Cell Culture
Inoculate a single colony from an agar plate containing transformed cells in a 10 mL starter culture with appropriate antibiotic selection marker.
Let the seed culture grow overnight at 37 °C with 180 rpm rotation.
Measure the OD600 to confirm growth (an OD600 of around 2 is to be expected).
Prepare 1L of media with the required antibiotics.
Inoculate the media with 10 mL of the seed culture (1:100 ratio).
Let the culture grow until an OD600 of 0.5.
When the required OD is reached, put the culture on ice for 30 minutes.
After the 30 minutes, induce the expression of the protein by adding 1M IPTG to a final concentration of 1mM and 20% arabinose to a final concentration 0.2% arabinose.
Grow the cells for 16 hours ot 18 °C and 180rpm rotation.
Cell Harvest
Harvest the liquid culture by centrifugation at 5200g and 4 °C for 15 minutes.
Discard the supernatant and weigh the pellet. Resuspend the pellet with 6mL PBS/g cells.
Centrifuge the cells 15 minutes at 5200 g and 4 °C, and again discard supernatant.
The cells are now directly passed off to downstream processing.
Protein Purification dxCas9
This protocol describes the optimized downstream processing of dxCas9.
Cell Lysis
Retrieve a washed cell pellet from upstream processing according to the protein expression protocol.
NOTE: All following steps in this purification protocol are done at 4 °C.
Resuspend the pellet in lysis buffer (20mM Tris-HCL, 250mM NaCl, 10% v/v glycerol, 1mM DTT, 5mM imidazole, pH 8.0) and one protease inhibitor tablet per 50mL lysis buffer. Make sure pellet is fully resuspended.
Lyse the cells using a high-pressure homogenizer (French Press) (2 rounds at 1 kbar).
Clarification
Clarify the lysate via centrifugation for 45 min at 16,000 g.
NOTE: The dxCas9 is now dissolved in the supernatant and the pellet contains cell debris.
Nickel Affinity Chromatography
Perform Nickel affinity chromatography with a gravity column.
NOTE: 1mL of 50% His-Select Ni resin per 15mL clarified lysate for high expression proteins.
Resuspend 1mL of 50% Hisselect Nickel Affinity gel in 10mL (20CV) water, spinning down at 3220g for 1 min and discard the supernatant. Repeat this three times.
Equilibrate the washed resin with 10mL (20CV) of equilibration buffer (20mM Tris-HCl, 250mM NaCl, 5mM Imidazole, 10% v/v glycerol, 1mM DTT, pH8), spinning down at 3220g for 1 min and discard the supernatant.
Add the Hisselect column material to the clarified lysate and incubate for one hour with gentle mixing.
Spin down the mixture, for 1 min at 2000 g.
NOTE: This mixture contains dxCas9 bound to Hisselect Nickel resin.
Discard the supernatant by pipetting and load the resin onto a gravity column.
Wash the resin five times in 2mL (20CV) of washing buffer (20mM Tris-HCl, 250mM NaCl, 5mM Imidazole, 10% v/v glycerol, 1mM DTT, pH8) and collect the flowthrough.
Elute the dxCas9 from the resin with six fractions of 500μL of elution buffer (20mM Tris-HCl, 250mM NaCl, 1mM DTT and 250mM imidazole, pH 8.0), and collect in differently labeled tubes.
Measure the A280 of all six elution fractions at the Nanodrop, to roughly estimate the protein concentrations.
Load the all the samples including the elution fractions onto an SDS 8% Tris-Glycine PAGE gel to confirm presence of the protein.
Pool the fraction that contains the dxCas9.
Heparin Chromatography
Perform heparin chromatography on the AKTA pure with a 1 mL HiTrap Heparin HP column.
Set the following settings:
Flow rate: 0.5mL/min
Maximum column pressure: 0.5MPa
Perform the following method on the AKTA.
Chromatography step
Buffer
Column Volume (CV)
Volume (mL)
Fractionation
Column washing
Water
20
20
0 Fractions
Column equilibrating
20mM Tris-HCl, 250mM NaCl, 10% v/v glycerol, 1mM DTT, pH 7.5
20
20
0 Fractions
Sample loading
Pooled dxCas9 fractions
1-20
1-20
1 Fraction in the appropriate tube
Washing
20mM Tris-HCl, 250mM NaCl, 10% v/v glycerol, 1mM DTT, pH 7.5
20
20
1 Fraction in a 50mL Falcon tube
Eluting
20mM Tris-HCl, 1.5M NaCl, 10% v/v glycerol, 1mM DTT, pH 7.5
30
30
60 Fractions in a 1mL tube
Column Washing
Water
20
20
0 Fractions
Column Storage
20% ethanol
20
20
0 Fractions
Based on the A280 seen on the chromatogram select desired elution fractions that contains protein.
Load the all the samples including the desired elution fractions onto an SDS 8% Tris-Glycine PAGE gel to confirm presence of the protein.
Dialysis
Pool the fraction that contain the dxCas9.
Prepare 2L of dialysis buffer (20mM Tris-HCl, 100mM KCl, 5% v/v glycerol, 1mM beta-mercaptoethanol and pH 7.5).
Inject the pooled fractions (0.5-3mL) into a pierce 10,000 Molecular weight cut off cassette.
Buffer exchange by dialysis of the pooled fractions for 1 hour in 1L dialysis buffer.
Replace the dialysis buffer with fresh buffer and repeat dialysis for 1 hour.
Remove the dxCas9 solution out of the dialysis cassette and measure the concentration with the Pierce BCA protein assay kit by Thermo Scientific.
Aliquote the dxCas9 solution in 25µL aliquots and freeze at -20℃ for functionality testing assays.
Protein Purification Tn5
This protocol describes the optimized downstream processing of Tn5.
Cell Lysis
Retrieve a washed cell pellet from upstream processing according to the protein expression protocol.
NOTE: All following steps in this purification protocol are done at 4 °C.
Resuspend the pellet in lysis buffer (20mM Tris-HCL, 250mM NaCl, 10% v/v glycerol, 1mM DTT, 5mM imidazole, pH 8.0) and one protease inhibitor tablet per 50mL lysis buffer. Make sure pellet is fully resuspended.
Lyse the cells using a high-pressure homogenizer (French Press) (2 rounds at 1 kbar).
Clarification
Clarify the lysate via centrifugation for 45 min at 16,000 g.
NOTE: The Tn5 is now dissolved in the supernatant and the pellet contains cell debris.
Nickel Affinity Chromatography
Perform Nickel affinity chromatography with a gravity column.
NOTE: 1mL of 50% His-Select Ni resin per 15mL clarified lysate for high expression proteins.
Resuspend 1mL of 50% Hisselect Nickel Affinity gel in 10mL (20CV) water, spinning down at 3220 g for 1 min and discard the supernatant. Repeat this three times.
Equilibrate the washed resin with 10mL (20CV) of equilibration buffer (20mM Tris-HCl, 250mM NaCl, 5mM Imidazole, 10% v/v glycerol, 1mM DTT, pH8), spinning down at 3220 g for 1 min and discard the supernatant.
Add the Hisselect column material to the clarified lysate and incubate for one hour with gentle mixing.
Spin down the mixture, for 1 min at 2000 g.
NOTE: This mixture contains Tn5 bound to Hisselect Nickel resin.
Discard the supernatant by pipetting and load the resin onto a gravity column.
Wash the resin five times in 2 mL (20CV) of washing buffer (20mM Tris-HCl, 250mM NaCl, 5mM Imidazole, 10% v/v glycerol, 1mM DTT, pH8) and collect the flowthrough.
Elute the Tn5 from the resin with six fractions of 500μL of elution buffer (20mM Tris-HCl, 250mM NaCl, 1mM DTT and 250mM imidazole, pH 8.0), and collect in differently labeled tubes.
Measure the A280 of all six elution fractions at the Nanodrop, to roughly estimate the protein concentrations.
Load the all the samples including the elution fractions onto an SDS 12% Tris-Glycine PAGE gel to confirm presence of the protein.
Pool the fractions that contain the Tn5.
Heparin Chromatography
Perform heparin chromatography on the AKTA pure with a 1 mL HiTrap Heparin HP column.
Set the following settings:
Flow rate: 0.5mL/min
Maximum column pressure: 0.5 MPa
Perform the following method on the AKTA.
Chromatography step
Buffer
Column Volume (CV)
Volume (mL)
Fractionation
Column washing
Water
20
20
0 Fractions
Column equilibrating
20mM Tris-HCl, 250mM NaCl, 10% v/v glycerol, 1mM DTT, pH 7.5
20
20
0 Fractions
Sample loading
Pooled Tn5 fractions
1-20
1-20
1 Fraction in the appropriate tube
Washing
20mM Tris-HCl, 250mM NaCl, 10% v/v glycerol, 1mM DTT, pH 7.5
20
20
1 Fraction in a 50mL Falcon tube
Eluting
20mM Tris-HCl, 1.5M NaCl, 10% v/v glycerol, 1mM DTT, pH 7.5
30
30
60 Fractions in a 1 mL tube
Column Washing
Water
20
20
0 Fractions
Column Storage
20% ethanol
20
20
0 Fractions
Based on the A280 seen on the chromatogram select desired elution fractions that contains protein.
Load the all the samples including the desired elution fractions onto an SDS 12% Tris-Glycine PAGE gel to confirm presence of the protein.
Dialysis
Pool the fraction that contain the Tn5.
Prepare 2L of dialysis buffer (20mM Tris-HCl, 100mM KCl, 5% v/v glycerol, 1mM beta-mercaptoethanol and pH 7.5).
Inject the pooled fractions (0.5-3mL) into a pierce 10,000 Molecular weight cut off cassette.
Buffer exchange by dialysis of the pooled fractions for 1 hour in 1L dialysis buffer.
Replace the dialysis buffer with fresh buffer and repeat dialysis for 1 hour.
Remove the Tn5 solution out of the dialysis cassette.
Protein Concentration
Wash a Amicon ultra-15mL centrifugal filter units with an 10,000 molecular weight cutoff with MilliQ water by centrifuging at 4,000 × g for approximately 10–20 minutes.
Add the Tn5 sample to the filter and centrifuged at 4,000 × g for approximately 10–20 minutes.
Measure the Tn5 protein concentration with the Pierce BCA protein assay kit by Thermo Scientific.
Aliquote the Tn5 solution in 25µL aliquots and freeze at -20℃ for functionality testing assays.
Protein Purification dxCas9-Tn5
This protocol describes how the downstream processing of dxCas9-Tn5 was performed.
Cell Lysis
Retrieve a washed cell pellet from upstream processing according to the protein expression protocol.
NOTE: All following steps in this purification protocol are done at 4 °C.
Resuspend the pellet in lysis buffer (20mM Tris-HCL, 250mM NaCl, 1mM DTT, 10% v/v glycerol, 1mM PMSF, pH 7.5) and one protease inhibitor tablet per 50mL lysis buffer. Make sure pellet is fully resuspended.
Lyse the cells using a high-pressure homogenizer (French Press) (2 rounds at 1 kbar).
Clarification
Clarify the lysate via centrifugation for 45 min at 16,000 g.
NOTE: The dxCas9-Tn5 is now dissolved in the supernatant and the pellet contains cell debris.
Filter the clarified lysate with a 0.45µm filter.
Heparin Chromatography
Perform heparin chromatography on the AKTA pure with a 1mL HiTrap Heparin HP column.
Set the following settings:
Flow rate: 0.5mL/min
Maximum column pressure: 0.5 MPa
Perform the following method on the AKTA.
Chromatography step
Buffer
Column Volume (CV)
Volume (mL)
Fractionation
Column washing
Water
20
20
0 Fractions
Column equilibrating
20mM Tris-HCl, 250mM NaCl, 10% v/v glycerol, 1mM PMSF, pH 7.5
20
20
0 Fractions
Sample loading
Crude dxCas9-Tn5 extract
1-50
1-50
1 Fraction in the appropriate tube
Washing
20mM Tris-HCl, 250mM NaCl, 10% v/v glycerol, 1mM PMSF, 1mM DTT, pH 7.5
20
20
1 Fraction in a 50 mL Falcon tube
Eluting
20mM Tris-HCl, 1.5M NaCl, 10% v/v glycerol, 1mM PMSF, 1mM DTT, pH 7.5
30
30
60 Fractions in a 1mL tube
Column Washing
Water
20
20
0 Fractions
Column Storage
20% ethanol
20
20
0 Fractions
Based on the A280 seen on the chromatogram select desired elution fractions that contains protein.
Load the all the samples including the desired elution fractions onto an SDS 8% Tris-Glycine PAGE gel to confirm presence of the protein.
MonoQ Chromatography
Perform anionic exchange chromatography on the AKTA pure with a 1mL MonoQ 5/50 GL column on the crude extract.
Set the following settings:
Flow rate: 0.5mL/min
Maximum column pressure: 20 MPa
Follow the following method on the AKTA
Chromatography step
Buffer
Column Volume (CV)
Volume (mL)
Fractionation
Column washing
Water
20
20
0 Fractions
Column equilibrating
20mM Tris-HCl, 250mM NaCl, 10% v/v glycerol, 1mM PMSF, 1mM DTT, pH 8.5
20
20
0 Fractions
Sample loading
Pooled fractions diluted 4 times with 20mM Tris-HCl, 0mM NaCl, 10% v/v glycerol, 1mM PMSF, 1mM DTT, pH 8.5
1-20
1-20
1 Fraction in the appropriate tube
Washing
20mM Tris-HCl, 250mM NaCl, 10% v/v glycerol, 1mM PMSF, 1mM DTT, pH 8.5
20
20
1 Fraction in a 50 mL Falcon tube
Eluting
20mM Tris-HCl, 1.5M NaCl, 10% v/v glycerol, 1mM PMSF, 1mM DTT, pH 8.5
30
30
60 Fractions in a 1 mL tube
Column Washing
Water
20
20
0 Fractions
Column Storage
20% ethanol
20
20
0 Fractions
Based on the A280 seen on the chromatogram select desired elution fractions that contains protein.
Load the all the samples including the desired elution fractions onto an SDS 8% Tris-Glycine PAGE gel to confirm presence of the protein.
Dialysis
Pool the fraction that contain the dxCas9-L-Tn5.
Prepare 2L of dialysis buffer (20mM Tris-HCl, 100mM KCl, 5% v/v glycerol, 1mM beta-mercaptoethanol, 1mM PMSF and pH 7.5).
Inject the pooled fractions (0.5-3mL) into a pierce 10,000 Molecular weight cut off cassette.
Buffer exchange by dialysis of the pooled fractions for 1 hour in 1L dialysis buffer.
Replace the dialysis buffer with fresh buffer and repeat dialysis for 1 hour.
Remove the dxCas9-Tn5 solution out of the dialysis cassette.
Protein Concentration
Wash a Amicon ultra-15mL centrifugal filter units with an 10,000 molecular weight cutoff with MilliQ water by centrifuging at 4,000 × g for approximately 10–20 minutes.
Add the dxCas9-Tn5 sample to the filter and centrifuged at 4,000 × g for approximately 10–20 minutes.
Measure the dxCas9-Tn5 protein concentration with the Pierce BCA protein assay kit by Thermo Scientific.
Aliquote the dxCas9-Tn5 solution in 25µL aliquots and freeze at -20℃ for functionality testing assays.
Qubit High Sensitivity dsDNA quantification
This protocol is based on the manual for the Qubit dsDNA High Sensitivity Assay Kit provided by Thermofischer Scientific.
Set up the required amount of tubes for your samples and add two for the calibration standards.
Label the lids. Do not label the sides of the tubes, since this will interfere with the measurements.
Prepare the Qubit working solution.
n = amount of sample + two standards
Dilute n µL of dsDNA HS Reagent 1:200 in n * 199µL dsDNA HS Buffer. Use a clean plastic tube each time you prepare a working solution. Do not use glass containers.
Add 190µL of the working solution to the tubes for the two standards.
Add 10µL of the standards to the according tubes.Mix by vortexing 2-3 seconds and be careful not to create bubbles.
Add 180-199µL of working solution to the sample tubes, dependent on the amount of sample to be measured. The amount of sample used for the measurement can range from 1 to 20µL. Dependent on this amount the added volume of working solution should be adapted, so that the final volume is always 200µL.
Add each sample to the according assay tube. Mix by vortexing 2-3 seconds.
Allow all tubes to incubate at room temperature for 2 minutes.
Read the standards and samples following the Qubit instrument instructions.
Restriction
Decide on which restriction enzyme(s) to cut with. Check online (NEB) what buffer the enzyme(s) work(s) in . For most of the restriction enzymes, the CutSmart buffer 10X can be used.
Prepare a 20µL sample as follows:
Component
Volume (µL)
10X CutSmart buffer (NEB)
2
Fragment (~1-2μg)
X
Restriction Enzyme(s)
1 per enzyme
MilliQ
18 - #RE added - X)
Incubate for 4 hours at 37 °C.
Inactivate the restriction enzyme(s) by heating to 65 °C for 10 minutes.
NOTE: This last step can be skipped if the sample is evaluated on gel electrophoresis immediately.
NOTE: Some enzymes are not inactivated by increasing the temperature to 65 °C.
NOTE: DNA Clean Up is recommended for subsequent cloning strategies.
RNA Clean Up
This protocol is based on the QIAGEN RNeasy MinElute Cleanup Kit suited for volumes smaller than 100µL.
Adjust the transcripts volume to 100µl with RNase-free water.
Add 350µL Buffer RLT and mix.
Add 250µL of absolute ethanol.
Transfer the sample to an RNeasy MinElute spin column placed in a 2mL collection tube. Close the lid and centrifuge for 15 seconds at minimum speed 8000x g.
Discard the flow-through and place the spin column in a new 2mL collection tube.
Add 500µL Buffer RPE to the spin column. Close the lid and centrifuge for 15 seconds at minimum speed 8000x g.
Discard the flow-through and place the spin column back to the same 2mL collection tube.
Add 500µL 80% ethanol to the spin column. Close the lid and centrifuge for 2 minutes at minimum speed 8000x g.
Discard the flow-through and place the spin column in a new 2mL collection tube.
Open the lid and centrifuge at full speed for 5 minutes. Discard the flow through and the collection tube.
Place the spin column in a new 1.5mL collection tube. Add 14µL RNase-free water to the center of the spin column.
Close the lid and centrifuge for 1 minute at full speed to elute the RNA.
Store the purified RNA at -20 °C
RNA Gel Electrophoresis
Add 1:1 volume ratio of a purified RNA sample to 2X RNA loading buffer.
Boil at 90 °C for 5 minutes to denature potential secondary structures and temporarily inactivate RNAse.
Directly put on ice for ~2 minutes.
Spin down and electrophorese on a 2% agarose gel (in TBE buffer) for visualization with the gel documentation system using UV light.
SDS-PAGE Electrophoresis
An SDS-PAGE electrophoresis is used to separate proteins based on their size by using an electric current. This protocol describes how to prepare SDS-PAGE gels, how to prepare samples and how to run the electrophoresis.
Preparing of SDS-PAGE Gel
NOTE: An SDS-PAGE gel consist of a stacking gel and a separation gel. In this protocol we use a 6% Stack gel and a 12% separation gel (this is for protein samples between 15 kDa and 250 kDa).
Prepare the following solutions for 2 gels:
Component
Volume 6% stacking gel
Volume 12% separation gel
40% acrylamide
750 µL
3 mL
0.25 M Tris-HCl pH 6.8
2.5 mL
-
0.55 M Tris-HCl pH 8.8
-
5 mL
10% SDS
50 µL
100 µL
10% APS
50 µL
100 µL
TEMED
5 µL
10 µL
MilliQ
1.645 mL
1.79 mL
Total volume
5 mL
10 mL
NOTE: Polymerization will start when 10% APS and TEMED are added.
Assemble the casting station and pour the separation gel between the glass plates leaving enough space for the stacking gel.
Add 75% ethanol on top of the solidifying stacking gel.
When the gel is solidified, remove the ethanol and pour the stacking gel on top of the separation gel and immediately place the gel comb on top of it.
Allow the stacking gel to solidify, then remove the plates from the casting station.
The gels are ready for usage, or can be stored at 4 °C in liquid for later usage.
Preparing samples
Add 5μL protein Loading Dye to 20μL of protein sample.
Incubate the samples for 10 minutes at 90 °C.
Running SDS-PAGE gel
Run the gel at 150 V and 25 mA.
The gel is finished when the blue line is at the end of the gel.
Processing SDS-PAGE gel
Turn off the power pack before opening the gel box.
Remove the gel from the cassette and place the gel in a clean staining tray of the appropriate size filled with a layer of MilliQ.
Gently allow the gel to slide from the glass plate into the water.
Microwave the gel for 40 seconds on low heat.
Allow the gel to shake on a moving platform for 2 minutes.
Discard the water.
Add clean MilliQ water and repeat step 15-17 two times.
Cover the gel with a layer of SimplyBlue SafeStain.
Let the gel stain on a moving plate for approximately 20 minutes.
Discard the SimplyBlue SafeStain in the waste container.
Cover the gel with MilliQ water and allow the gel to shake on a moving platform until the gel is destained.
NOTE: the water can be replaced frequently to remove all the stain.
Take a picture with the Gel doc system.
Sequencing with MinION
This protocol is based on the protocol supplied with the MinION device from Oxford Nanopore Technologies .
Flow cell QC and check
Connect MinION device to a USB 3.0 port of a computer with recommended 1 Terabyte free space.
Open MinKNOW GUI from desktop and execute quality control (QC) of the flow cell.
NOTE: The number of available active pores to work with Flow cell from ONT is > 800 pores.
Library preparation rapid sequencing (lambda DNA)
Thaw the components of the kit for Rapid Sequencing on ice and keep on ice.
Add 2.5µL of DNA to 7.5µL of FRA (fragmentation mix).
Mix the content well by inversion and spin down.
Incubate tube at 30 ºC for 1 minute and then at 80 ºC for another minute.
After incubation, add 1µL of RAP (Rapid Adapter) to the tube and incubate for 5 minutes at room temperature.
Priming of Flow Cell
Turn the lid of the Flow Cell clockwise in order to make the priming port visible.
Remove bubbles from the priming port (not by directly pipetting but by changing pipette volume).
Pipette 30µL of Flush Tether into a tube of Flush Buffer and mix well by pipetting (priming mix).
Add 800µL of this priming mix to the priming port.
Loading the SpotON Flow Cell
Prepare the following reaction mixture:
Component
Volume (µL)
Sequencing Buffer (SQB)
34
Loading Beads (LB)
25.5
Nuclease-free MilliQ
4.5
Prepared DNA library
11
Open the SpotON port making the sample port available.
Add 200µL of priming mix.
Add 75µL of sample in the sample port in a dropwise fashion, allowing every drop to enter the port before adding another drop.
Close the priming port and replace the MinION lid.
Sequencing run and base calling
Start the sequencing run once the sample is loaded on the Flow cell using the Desktop Agent.
Check the progression of the upload and download of files, together with the network speed.
Close down MinKNOW and Desktop Agent
The run is stopped once the live base calling shows no DNA (or very little DNA) is being sequenced.
Quit Desktop Agent and MinKNOW and disconnect MinION.
To store the flow cell for a next run, follow the Wash Kit MinION protocol.
Sequence Verification
For sequence verification, the sequencing platform EZ-Seq established by Macrogen was used. This protocol is based on their instructions.
Prepare a sequencing sample in a 1.5 mL microcentrifuge tube, labelled with a Macrogen sequencing sticker (QR-code) as follows:
Component
Volume (µL)
DNA
X (~500 ng is required*)
Primer (10 µM)
2.5 (final concentration 5-10 pmol/µL)
Sterile MilliQ
10-X
* NOTE: The volume depends on the concentration of the sample.
Make sure to keep the QR-code and note down what sample it is affiliated with. Deliver the tubes for sequencing to the manufacturer and await results.
Results are evaluated in silico by aligning expected sequences with obtained sequences and calculating the match percentage.
Serum DNA Extraction
This protocol is an adaptation of the original QIAGEN QIAamp DNA Blood Mini Kit protocol .
Important points before starting
All centrifugation steps are carried out at room temperature (15–25°C).
Use carrier DNA if the sample contains <10,000 genome equivalents (see page 17).
200 μl of whole blood yields 3–12 μg of DNA. Preparation of buffy coat (see page 18) is recommended if a higher yield is required.
Things to do before starting
Equilibrate samples to room temperature (15–25°C).
Heat a water bath or heating block to 56°C for use in step 4.
Equilibrate Buffer AE or distilled water to room temperature for elution in step 11.
Ensure that Buffer AW1, Buffer AW2, and QIAGEN Protease have been prepared according to the instructions on page 16.
If a precipitate has formed in Buffer AL, dissolve by incubating at 56°C.
Serum DNA Extraction Procedure
Pipet 10 μl QIAGEN Protease (or proteinase K) into the bottom of two 1.5 ml microcentrifuge tubes.
Add 500 μl sample (plasma or serum) to the microcentrifuge tubes.
NOTE: It is possible to add QIAGEN Protease (or proteinase K) to samples that have already been dispensed into microcentrifuge tubes. In this case, it is important to ensure proper mixing after adding the enzyme.
Add 500 μl Buffer AL to the sample (the same amount as the sample). Mix by pulse-vortexing for 15 s. It is a quite syrupy consistency. To ensure efficient lysis, it is essential that the sample and Buffer AL are mixed thoroughly to yield a homogeneous solution.
NOTE: Do not add QIAGEN Protease or proteinase K directly to Buffer AL.
Incubate at 56°C for 10 min. DNA yield reaches a maximum after lysis for 10 min at 56°C. Longer incubation times have no effect on yield or quality of the purified DNA.
Briefly centrifuge the 1.5 ml microcentrifuge tube to remove drops from the inside of the lid.
Add 500 μl ethanol (96–100%) to the sample (the same amount as the sample). Pipet it into the tube via the side of tube. This will let the foam of the centrifugation go away. The ethanol will let the proteins precipitate. Mix again by pulse-vortexing.
Carefully apply 600 μl of the mixture from step 6 to the QIAamp Mini spin column (in a 2 ml collection tube) without wetting the rim (pipet via the side at the top of the tube). Close the cap, and centrifuge at 6000 x g (8000 rpm) for 1 min. Discard the filtrate en place the QIAamp Mini spin column in the same 2 ml collection tube. Repeat this 5 times untill the 3mL of the mixture is used (split over 2 1.5 ml microcentrifugation tubes). After the last time centrifugation place the QIAamp Mini spin column in a clean 2 ml collection tube (provided), and discard the tube containing the filtrate.
Close each spin column to avoid aerosol formation during centrifugation. Centrifugation is performed at 6000 x g (8000 rpm) to reduce noise. Centrifugation at full speed will not affect the yield or purity of the DNA. If the lysate has not completely passed through the column after centrifugation, centrifuge again at higher speed until the QIAamp Mini spin column is empty. Note: When preparing DNA from buffy coat or lymphocytes, centrifugation at full speed is recommended to avoid clogging.
Carefully open the QIAamp Mini spin column and add 500 μl Buffer AW1 without wetting the rim. Close the cap and centrifuge at 6000 x g (8000 rpm) for 1 min. Place the QIAamp Mini spin column in a clean 2 ml collection tube (provided), and discard the collection tube containing the filtrate.*
NOTE: It is not necessary to increase the volume of Buffer AW1 if the original sample volume is larger than 200 μl.
*NOTE: Flow-through contains Buffer AL or Buffer AW1 and is therefore not compatible with bleach. See page 6 for safety information.
Carefully open the QIAamp Mini spin column and add 500 μl Buffer AW2 without wetting the rim. Close the cap and centrifuge at full speed (20,000 x g; 14,000 rpm) for 3 min.
Place the QIAamp Mini spin column in a new 1.5 ml microcentrifuge tube (not provided) and discard the old collection tube with the filtrate. Centrifuge at full speed for 1 min.
This step helps to eliminate the chance of possible Buffer AW2 carryover.
Place the QIAamp Mini spin column in a clean 1.5 ml microcentrifuge tube (not provided), and discard the collection tube containing the filtrate. Carefully open the QIAamp Mini spin column and add 60 μl Buffer AE or distilled (MilliQ) water. Incubate at room temperature (15–25°C) for 5 min, and then centrifuge at 6000 x g (8000 rpm) for 1 min. Take the eluate from the eppendorf tube and put it again on top of the filter. Centrifuge a second time at 14.000 rpm for 1 min.
sgRNA Transcription
This protocol is based on the standard RNA synthesis of the New England Biolabs T7 RNA Polymerase.
The following components should be assembled at room temperature in the following order:
Component
Volume (µL)
10x Reaction Buffer
2
NTP (each 25mM)
4
Template DNA
X (0.2 - 1 µg)
RNase Inhibitor
1 U/µL (final)
DTT
5 mM (final)
T7 RNA Polymerase
2
MilliQ
add up to 20 µL
The reaction is incubated at 37oC for 4-16 hours.
The following sequence is used as the template DNA, in which “n” can be substituted to match the target DNA.
TAATACGACTCACTATAGGnnnnnnnnnnnnnnnnnnnGTTTTAGAGCTAGAAATAGCAAGTTAAAATAAGGCTAGTCCGTTATCAACTTGAAAAAGTGGCACCGAGTCGGTGCTTTT
Clean up sgRNA product(s) according to the RNA Clean Up protocol.
Solid Medium Preparation
Dissolve the LB-agar powder in sterile MilliQ as directed by the manufacturer’s protocol.
Autoclave the LB-agar solution (121 °C).
Cool the agar down to hand-warm temperature.
Supplement the medium with any relevant antibiotics and/or inducers etc.
NOTE: With our antibiotic stock solutions of 1000x we used 1 µL of antibiotic solution per mL of LB-agar.
Under aseptic conditions, pour the medium in empty Petri dishes.
Close the Petri dishes and let the agar solidify at room temperature.
Store the Petri dishes upside down at 4 °C.
NOTE: When complimenting medium with X-gal, be sure to store the plates protected from light.
Tn5 or dxCas9-Tn5 Fusion Protein Functionality Testing: Mobility Shift
This protocol describes how to evaluate the mobility shift of the Tn5:adapter DNA complex or the dxCas9-Tn5:adapter DNA complex.
Adapter DNA Sequence
To create 25µM adapter DNA molecule. Two primers shown on table 1 were annealed to each other as follow:
Add 25µl of each primer (from 100µM stock)
Add 50µl of MilliQ water
Place tube in 90-95 °C heating block and leave for 3-5 minutes to denature any secondary structure
Remove the heating block from the heat source allowing for slow cooling to room temperature (~45 minutes)
Table 1. Sequence of the adapter DNA. Note that 5’ of the upper strand of the adapter has to be phosphorylated for successful integration.
Name
Sequence (5’ - 3’)
Adapter upper strand
P-CTGTCTCTTATACACATCT
Adapter lower strand
GCATGGTAAACAGTTCATCCATTTCGCCAATAGATGTGTATAAGAGACAG
Tn5:adapter DNA or dxCas9-Tn5:adapter DNA complex (transposome) formation
Load the purified Tn5 or dxCas9-Tn5 protein with adapter DNA shown above. Note that all samples and reagents are kept on ice during preparation.
Calculate all the volumes of the reagents required:
1-6µM Tn5 or dxCas9-Tn5
2.5µM adapter
Functionality buffer
dxCas9-Tn5 buffer: 20mM Tris-HCl, 100mM KCl, 5% v/v glycerol, 1mM beta-mercaptoethanol, 10mM MgCl2 pH 7.5
Tn5 buffer: 25mM Tris-HCl, 500mM NaCl, 1mM DTT, 0.1mM EDTA, pH 7.5
Water to bring the final volume to 10µl
Pipette the reagents in the following order: Water, 10x functionality buffer, Tn5 or Tn5-dxCas9 and adapter
Incubate at 23 °C for 1 hour.
NOTE: The total reaction volume can be adjusted as long as molar ratios of the reagents are kept constant.
Analyse the samples on a 8% native TBE polyacrylamide gel to observe the mobility shift
Tn5 or dxCas9-Tn5 Fusion Protein Functionality Testing: Integration
This protocol describes how to evaluate the integration of adapter DNA by Tn5 or the dxCas9-Tn5 fusion protein.
Adapter DNA Sequence
To create 25µM adapter DNA molecule. Two primers shown on table 1 were annealed to each other as follow:
Add 25µl of each primer (from 100µM stock)
Add 50µl of MilliQ water
Place tube in 90-95 °C heating block and leave for 3-5 minutes to denature any secondary structure
Remove the heating block from the heat source allowing for slow cooling to room temperature (~45 minutes)
This protocol describes how to evaluate the integration of the Tn5 or the Tn5-dxCas9.
Table 1. Sequence of the adapter DNA. Note that 5’ of the upper strand of the adapter has to be phosphorylated for successful integration.
Name
Sequence (5’ - 3’)
Adapter upper strand
P-CTGTCTCTTATACACATCT
Adapter lower strand
GCATGGTAAACAGTTCATCCATTTCGCCAATAGATGTGTATAAGAGACAG
Tn5:adapter DNA or dxCas9-Tn5:adapter DNA complex (transposome) formation
Load the purified Tn5 or dxCas9-Tn5 protein with adapter DNA shown above. Note that all samples and reagents are kept on ice during preparation.
Calculate all the volumes of the reagents required:
2.5µM Tn5 or Tn5-dxCas9
2.5µM adapter
Functionality buffer (20mM Tris-HCl, 100mM KCl, 5% v/v glycerol, 1mM beta-mercaptoethanol, 0.1mM EDTA pH 7.5)
Water to bring the final volume to 10µl
Pipette the reagents in the following order: Water, 10X functionality buffer, Tn5 or dxCas9-Tn5 and adapter.
Incubate at 23 °C for 1 hour.
NOTE: The total reaction volume can be adjusted as long as molar ratios of the reagents are kept constant.
gRNA:Transposome complex formation (only for fusion protein)
Load dxCas9-Tn5:adapter complex with gRNA provided by Arbor Biotechnologies.
NOTE: all samples and reagents are kept on ice during preparation.
Add the following reagents to 10µl transposome
2µl sgRNA (100ng/µl stock solution) to make 1:1 sgRNA:transposome molar ratio
0.5µl buffer (add concentration)
Incubate at 37 °C for 10 minutes.
Target DNA integration
Add 50ng target DNA, in our case EPO cDNA to 12.5µl of gRNA:Transposome complex.
Incubate at 37 °C for 1 hour.
Amplification of integration products by PCR
Targeted integration of adapter DNA to EPO cDNA (632bp) will result in two fragments of ~250bp and ~450bp. To visualize these fragments, PCR was done to amplify the two products separately. The sequence of the primers are found on table 2. Primer set 1 will amplify the first fragment, while primer set 2 will amplify the second fragment (figure 1).
Table 2. Primer lists used for the functionality assay
Primer set
Fw primer
Information
Rv primer
Information
1
Fw EPO
GATCGAATTCGCGGCCGCTTC
This will bind to the start of the EPO gene (target gene)
Rv adapter
GCATGGTAAACAGTTCATCCATTTCGCCAATAGATGTGTATAAGAGACAG
This will bind to the end of the adapter
2
Rv adapter GCATGGTAAACAGTTCATCCATTTCGCCAATAGATGTGTATAAGAGACAG
This will bind to the end of the adapter (start of the product)
Rv EPO
CGATTCTGCAGCGGCCGCTAC
This will bind to the end of EPO gene (target gene)
Figure 1. Depiction of the binding location of the primer sets 1 (left) and 2 (right) for functionality assay PCR.
The PCR reaction composition is shown in Table 3.
Table 3. Pipetting scheme for PCR of integration products.
Component
Volume (µL)
5X GoTaq buffer
4
MgCl2 (25µM)
2
10 mM dNTPs
0.4
Primer forward (10µM)
0.4
Primer reverse (10µM)
0.4
GoTaq polymerase
0.1
DNA template
2.5
MilliQ
up to 20µL
Close all tubes thoroughly and place them in a thermocycler with the following protocol:
Step
Time (s)
Temperature (°C)
Initial denaturation
150
98
Denaturation
60
94
Annealing
60
60
Extension
60
72
Final extension
480
72
Hold
∞
20
The PCR product are  analysed on 5% native TBE polyacrylamide gel. The resulting gel is either stained with ethidium bromide and imaged with GelDoc system, or directly imaged with Typhoon Imaging System. In order to see the position of the protein in the gel the gel can be stained with silver quest staining.
