<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_TUDelft_Model_Barcoding skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:TUDelft/Model/Barcoding</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container-fluid nowbg"><DIV class="row nowbg"><DIV class="col-lg-3 col-md-3"><NAV id="myScrollspy" class="hidden-xs hidden-sm"><UL class="nav nav-pills nav-stacked" data-spy="affix"><LI class="active"><A href="#introduction" class="jnnbl">1. Introduction</A></LI><LI><A href="#multiplexing" class="jnnbl">2. Multiplexing</A></LI><LI><A href="#webtool" class="jnnbl">3. Webtool</A></LI><LI><A href="#application" class="jnnbl">4. Applications</A></LI><LI><A href="#References" class="jnnbl">References</A></LI></UL></NAV></DIV><DIV class="col-lg-8 col-md-9 col-sm-12 col-xs-12"><H1 class="jnnbl">1. Introduction</H1><P>
Sequencing the entire genome of an athlete is relatively expensive and time-consuming. If we would sequence the genome of each 918 medal winners at the Olympics Games in the Rio de Janeiro of 2016, it would have taken at total of 1836 hours (2 days per athlete) and 35802 flowcells (39 flowcells per athlete) (<A class="jnnbl" href="#References">Jain, <I>et al.</I> 2018</A>). Minimizing sequencing time and costs is already accomplished by implementing our targeted sequencing approach as it only requires one flowcell per athlete. However, this is not the limit that we aim for as speed can always be increased. Prof. Hagan Bayley from Oxford University, one of the founders of Oxford Nanopore Technologies Ltd., Oxford UK, suggested that multiplexing with additional barcoding would solve both problems: time and costs. Therefore, we implemented the possibility for barcoding in our detection method. In order to do so, we developed an iGEM barcoding tool by improving upon an existing iGEM DNA translation tool developed by <A href="https://2014.igem.org/Team:Aberdeen_Scotland/DNA" class="jnnbl" target="_blank">iGEM Aberdeen Scotland, 2014</A>.      
</P><H1 class="jnnbl">2. Multiplexing by Barcoding</H1><P>
Multiplexing is the simultaneous sequencing of multiple samples from different athletes using one sequencing device. In order to be able to assign the output sequences to the corresponding athlete, the output sequence should be labeled with an athlete specific mark. However, the sequence should be marked in such a way that the label does not disturb or interrupt the sequencing. Therefore, we decided to label the DNA sample with an athlete specific sequence: the DNA barcode (<A class="jnnbl" href="#References">Bayliss <I>et al</I>, 2017</A>). 
The barcode is a DNA sequence that is placed between the mosaic elements for the transposase and the Oxford Nanopore adapter sequence. This barcoded sequencing adapter is picked up by the  <A href="https://2018.igem.org/Team:TUDelft/Description#ourMethod3" class="jnnbl" target="_blank">Tn5-dxCas9 fusion protein</A> and integrated into the target DNA of a specific athlete. When the DNA is sequenced by pore sequencing, the barcode is readable in the sequencing output and is used to assign the DNA directly to the corresponding athlete <B class="jnnbl">(figure 1)</B>.
</P><DIV class="containerslideshow"><DIV class="mySlidesslideshow"><DIV class="numbertextslideshowjnnbl">1 / 6</DIV></DIV><DIV class="mySlidesslideshow"><DIV class="numbertextslideshowjnnbl">2 / 6</DIV></DIV><DIV class="mySlidesslideshow"><DIV class="numbertextslideshowjnnbl">3 / 6</DIV></DIV><DIV class="mySlidesslideshow"><DIV class="numbertextslideshowjnnbl">4 / 6</DIV></DIV><DIV class="mySlidesslideshow"><DIV class="numbertextslideshowjnnbl">5 / 6</DIV></DIV><DIV class="mySlidesslideshow"><DIV class="numbertextslideshowjnnbl">6 / 6</DIV></DIV><A class="prevslideshowjnnbl" onclick="plusSlides(-1)">❮</A><A class="nextslideshowjnnbl" onclick="plusSlides(1)">❯</A></DIV><FIGCAPTURE class="jnnbl">Figure 1. Multiplexing of samples in Targeted Next Generation Sequencing by using barcodes.</FIGCAPTURE><P>
Every athlete needs the personal DNA embedded barcode. The minimal barcode length we need to generate a unique barcode for every person on Earth is 17 nucleotides as 4<SUP>17</SUP> is more than twice the world population. Implementing this 17 nucleotide barcode will be sufficient to mark samples of multiple athletes and multiplex in one sequencing run. 
However, characterization by personal identification barcodes, for example name, date of birth, sports etc, rather than a unique identifier requires extensive barcoding. The barcode sequence should be extended due to the fact that 4 nucleotides cannot cover the whole alphabet, numbers and other characters. Therefore, we assigned sets of three nucleotides to a specific letter, number or other character (such as a dot (.), comma (,), semicolon (;) etc.). Converting the characters of the barcode into DNA allows labelling of the targeted DNA to be sequenced. Converting the DNA strand back into the athlete details after sequencing allows multiplexing of many athlete's samples.

Translating an athlete specific barcode into DNA is performed by extending the iGEM DNA translation tool developed by <A href="https://2014.igem.org/Team:Aberdeen_Scotland/DNA" class="jnnbl" target="_blank"> iGEM Aberdeen Scotland, 2014</A> to include more characters. Additionally, the start and stop codons are removed from the DNA translation tool since we do not require protein production. Instead, the mosaic element and sequencing adapter are implemented to flank the barcoding sequence.
</P><H1 class="jnnbl">3. Webtool</H1><DIV id="pcontent" class="backgroundbarcoding"><DIV class="input_text"><P>Name, date of birth and sport of the athlete:</P><P class="textDNAbarcoding"><SUB>The maximum size of the barcode is 50 characters. You can include alphabetic letters, number and the characters [space] [.] [!] [?] [-] [_] [@] [#] [$] [%] [^] [&amp;] [*] [/] [:] [,] [~] and [=]. </SUB></P></DIV><DIV class="output_dna"><P>Your 5' to 3' barcode sequence is:</P><P>Your 3' to 5' barcode sequence is:</P></DIV><DIV class="output_dna"><P>The barcode is now implemented in between the mosaic element and the adapter sequences:</P><DIV style="overflow-x:auto;"><TABLE><TBODY><TR class="headerDNAbarcoding"><TD class="adpbl">Mosaic element</TD><TD class="jnnbl">Barcode</TD><TD class="grnbl">Sequencing Adapter</TD></TR><TR class="DNAbarcoding"><TD class="adpblbarcoding">P’CTGTCTCTTATACACATCT</TD><TD class="grnblbarcoding">TCGTTATGCATTGACTTGCTTCA</TD></TR><TR class="DNAbarcoding"><TD class="adpblbarcoding">  GACAGAGAATATGTGTAGA</TD><TD class="grnblbarcoding">AGCAATACGTAACTGAACGAAGTACATTAAAAAAAAAAGGTTAAACACCCAAGCAGACGCC</TD></TR></TBODY></TABLE></DIV></DIV></DIV><H1 class="jnnbl">4. Applications</H1><P>
Implementating the barcode system into our Targeted Next Generation Sequencing platform allows multiplexing of samples, reducing sequencing costs and sequencing time. The WADA can choose between a 17 nucleotide anonymous barcode or longer barcode with human-readable information. The 17 nucleotide barcode is anonymous and can have the preference due to athletes privacy. Additionally, the barcode sequence only needs to be 17 nucleotides long and to cover already 2.7 times the world population. However, the athlete describing barcode can be desired when the sample are required to be labelled this way.
Besides the area of gene doping detection, multiplexing in combination with targeted sequencing can be applied to virus detection, strain identification, food quality measurements and any other application that requires target specific sequencing.
</P><H1 class="jnnbl">References</H1><OL class="jnnbl" type="1"><LI><A class="jnnbl" href="https://academic.oup.com/gigascience/article/6/3/gix001/3051930" target="_blank"> Bayliss, S. C., Hunt, V. L., Yokoyama, M., Thorpe, H. A., &amp; Feil, E. J. (2017). The use of Oxford Nanopore native barcoding for complete genome assembly. GigaScience, 6(3), 1–6. http://doi.org/10.1093/gigascience/gix001</A></LI><LI><A class="jnnbl" href="https://www.nature.com/articles/nbt.4060" target="_blank"> Jain, M. <I>et al.</I> (2018). Nanopore sequencing and assembly of a human genome with ultra-long reads. Nature Biotechnology, 36, 338–345.</A></LI></OL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>