<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_TUDelft_Model_HBK skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:TUDelft/Model/HBK</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container-fluid"><DIV class="row"><DIV class="col-xs-12 col-sm-3 col-lg-3"><NAV id="myScrollspy" class="hidden-xs hidden-sm"><UL class="nav nav-pills nav-stacked" data-spy="affix"><LI class="active"><A href="#section1" class="jnnbl">Overview</A></LI><LI><A href="#section2" class="jnnbl">1. Approach</A></LI><LI class="dropdown"><A class="dropdown-toggle jnnbl" data-toggle="dropdown" href="#">2. Model Design</A></LI><LI><A href="#modeldesign" class="jnnbl">2. Model Design</A></LI><LI><A href="#sgRNA-Model" class="jnnbl">2.1 sgRNA Model</A></LI><LI><A href="#Gene-Doping-Model" class="jnnbl">2.2 Gene Doping Model</A></LI></UL><LI class="dropdown"><A class="dropdown-toggle jnnbl" data-toggle="dropdown" href="#">3. Results</A></LI><LI><A href="#Results" class="jnnbl">3. Results</A></LI><LI><A href="#sgRNA-Model-Results" class="jnnbl">3.1 sgRNA Model</A></LI><LI><A href="#Gene-Doping-Model-Results" class="jnnbl">3.2 Gene Doping Model</A></LI><LI><A href="#Downloads" class="jnnbl">4. Downloads</A></LI><LI><A href="#References" class="jnnbl">References</A></LI></NAV></DIV><DIV class="col-xs-12 col-sm-8 col-lg-8"><DIV id="section1"><H1 class="jnnbl">Overview</H1><P>Gene doping is the administration of exogenous genetic material for performance enhancement. Detection of gene doping requires identifying target sequences and detection windows. Our targeted sequencing method includes our novel dxCas9-Tn5 fusion protein and requires a minimal set of guide RNAs aligning with gene doping sequences. As there are 10<SUP>104</SUP> possible codon variations in our proof of concept target, the EPO gene, it is not feasible to target all possible sequences in practice. We therefore implemented a search function to identify areas with minimal variation within the sequence, which reduced the testing set to twelve sgRNAs. Second, we modeled the process of infection and degradation of gene doping DNA in blood. From this model, we provided the laboratory with the time dependent concentration of the target DNA. Based on our wetlab sensitivity analyses, the model predicts that with microdosing, our detection method could effectively catch gene dopers.</P></DIV><DIV id="section2"><H1 class="jnnbl">1. Approach</H1><P>We identified a threat in detecting gene doping that lays in the possibility of modifying the genetic sequence of a gene without changing the protein sequence produced. This allows gene dopers to creatively modify their DNA sequence in several possible combinations and complicates the design of detection methods. To combat this, the exon-exon junctions with the smallest possible variation need to be identified and sgRNA sequences generated to cover all possible combinations of one gene (<B class="jnnbl">figure 1</B>).</P><FIGURE><FIGCAPTURE class="figjnnbl"><B>Figure 1.</B>  Modified doping gene with introns removed. Exon-exon junctions are the target sites for the sgRNAs in our fusion protein. </FIGCAPTURE></FIGURE><P>Once the testing set of sgRNAs is generated for our fusion protein, we determined the time during which the detection of the gene doping DNA in an athlete’s blood sample is possible. This was accomplished by modelling gene doping administration. We considered the entire process of gene doping to fully understand the underlying mechanisms of gene doping and its effect on the athlete. With our chosen model gene being the human erythropoietin (EPO) gene based on the recommendation by <A href="https://2018.igem.org/Team:TUDelft/Human_Practices#influencers" class="jnnbl" target="_blank">Prof. Hidde Haisma</A> from Groningen University, we included the EPO dependent production of red blood cells in our model (<B class="jnnbl">figure 2</B>).</P><FIGURE><FIGCAPTURE class="figjnnbl"><B>Figure 2.</B> A representation of EPO based gene doping. Viral vectors infect kidney cells, increasing their production of EPO. The increased concentration of EPO in the blood leads to an increased production of red blood cells. The endogenous production of EPO is inversely related to the red blood cell count.</FIGCAPTURE></FIGURE><BUTTON class="collapsible cjnnbl">Vectors in Gene Doping</BUTTON><DIV class="content"><P>A vector is a DNA molecule used as a vehicle to carry foreign genetic material into another cell. Once, there it can be expressed and replicated. The effect and detection of gene doping is highly dependent on the vectors that are being used. In the early stages of our project we spoke with <A href="https://2018.igem.org/Team:TUDelft/Human_Practices#influencers" class="jnnbl" target="_blank">Prof. Hidde Haisma</A> from Groningen University, a gene doping expert, who told us the main vectors he would expect athletes to use. He suspect that the most common vectors are plasmids and adenoviruses, because of their relative safety compared to vectors that integrate the DNA into the genome of the cell. Integrating vectors, such as the retrovirus, present the threat of insertional mutagenesis which can lead to the development of cancer. <B class="jnnbl">Table 1</B> shows our analysis of possible gene doping vectors and some of their properties based on the work of <A class="jnnbl" href="#References">Ratko <I>et al</I>. 2003</A>.</P><CENTER><TABLE><CAPTION class="figjnnbl"><B>Table 1.</B> Gene doping vectors advantages and disadvantages. </CAPTION><TBODY><TR class="tableheaderjnnbl"><TH class="tableheaderjnnbl">Vector</TH><TH class="tableheaderjnnbl">Advantages</TH><TH class="tableheaderjnnbl">Disadvantages</TH></TR><TR class="tableunevenjnnbl"><TD>Plasmid</TD><TD>Relatively safe 
                                        Generally low immune response 
                                        Low cost and easy large quantity production 
                                        Variable transgene insertion upto ±20 kb (<A class="jnnbl" href="#References">Lodish <I>et al</I>. 2000</A>)    Long storage (<A class="jnnbl" href="#References">Munier <I>et al</I>. 2005</A>, <A class="jnnbl" href="#References">Kircheis <I>et al</I>. 2001</A>, <A class="jnnbl" href="#References">Li and Huang 2000</A>)</TD><TD>Very low transfection efficiency (<A class="jnnbl" href="#References">Bergen <I>et al</I>. 2008</A>) 
                                        No targeting 
                                        Transient expression</TD></TR><TR class="tableevenjnnbl"><TD>Adenovirus</TD><TD>High transduction efficiency 
                                        Transduces proliferating and nonproliferating cells 
                                        Transduces many cell types 
                                        Easy Production 
                                        Very high titers (1012 pfu/mL)</TD><TD>No targeting 
                                        Transient expression 
                                        Limited insert size: 4–5 kb 
                                        Immune-related toxicity with repeated administration 
                                        Potential replication competence</TD></TR><TR class="tableevenjnnbl"><TD>Adeno-associated virus
                                    </TD><TD>Continued expression 
                                        No viral genes</TD><TD>No targeting 
                                        Difficult production 
                                        Not characterized well 
                                        Potential Insertional Mutagenesis 
                                        Limited insert size: 5kb</TD></TR><TR class="tableevenjnnbl"><TD>Lentivirus</TD><TD>Transduces proliferating and nonproliferating cells 
                                        Prolonged expression 
                                        Relatively high titers (106–107 pfu/mL)</TD><TD>Integrating virus 
                                        Clinical experience limited 
                                        Difficult to manufacture and store 
                                        Limited insert size: 8 kb</TD></TR><TR class="tableevenjnnbl"><TD>Retrovirus</TD><TD>Relatively high titers (106–107 pfu/mL) 
                                        Prolonged stable expression 
                                        Larger insert size: 9–12 kb </TD><TD>Inefficient transduction 
                                        Integrating virus 
                                        Insertional mutagenesis 
                                        Broad cell tropism 
                                        No targeting 
                                        Potential replication competence</TD></TR></TBODY></TABLE></CENTER><P>Each vector has its own benefits and drawbacks. Plasmid vectors, such as non-viral DNA vectors, have several advantages over viral vectors. Virus production is expensive (<A class="jnnbl" href="#References">Templeton <I>et al</I>. 2002</A>, <A class="jnnbl" href="#References">Nagasaki and Shinkai, 2007</A>) and safety of viral transfection remains a concern after several deaths (<A class="jnnbl" href="#References">McCormack<I>et al</I>. 2004</A>, <A class="jnnbl" href="#References">Hacein-Bey-Abina <I>et al</I>. 2008</A>). However, plasmid vectors have a very low transfection efficiency (<A class="jnnbl" href="#References">Murakami <I>et al</I>. 2011</A>), especially compared to adenoviruses that have been shown to have a 95% transfection efficiency in hepatocytes (<A class="jnnbl" href="#References">Huard <I>et al</I>. 1995</A>, <A class="jnnbl" href="#References">Sullivan <I>et al</I> 1997</A>). Transfection efficiency of plasmids can be improved through methods such as in vivo electroporation (<A class="jnnbl" href="#References">Ataka <I>et al</I>. 2003</A>). However, this method requires the insertion of electrode needles into the athlete to increase transfection efficiency. This is more invasive to the athlete than a single injection of vector particles.  Therefore, adenoviruses are seen as the most likely transfection method for gene doping at this stage and even more so in the near future. Hence, most of the numerical values were based on this type of vector.</P></DIV></DIV><H1 class="jnnbl">2. Model Design</H1><DIV id="sgRNA-Model"><H3 class="jnnbl">2.1 sgRNA Array Model </H3><P>The four input variables of the model are the coding sequence of a gene, the protospacer adjacent motif (PAM) of the CRISPR-Cas protein, the length of the sgRNA, and the CRISPR-Cas-dependent identity between sgRNA seed and target (off target effect). An overview of the generated algorithm is shown in <B class="jnnbl">figure 3</B>.</P><FIGURE><FIGCAPTURE class="figjnnbl"><B>Figure 3.</B>  Overview of the algorithm used to generate sgRNA array for detection of gene doping with targeted sequencing.
                            </FIGCAPTURE></FIGURE><BUTTON class="collapsible cjnnbl">Detailed Steps of the sgRNA Algorithm</BUTTON><DIV class="content"><P>A step-by-step overview of the algorithm used for determination and creation of sgRNAs is given below:</P><OL type="1"><LI>Convert coding sequence into numerical sequence (A=1, T=2, C=3, G=4).</LI><FIGURE><FIGCAPTURE class="figjnnbl"><B>Figure 4.</B>  Algorithm to convert coding sequence into numerical sequence in an ‘for’-‘if’ loop.
                                    </FIGCAPTURE></FIGURE><LI>Translate genetic sequence into amino acid sequence.</LI><FIGURE><FIGCAPTURE class="figjnnbl"><B>Figure 5.</B> Algorithm used  to translate numeric DNA sequence into amino acid sequence in gene doping.
                                    </FIGCAPTURE></FIGURE><LI>Based on the amino acid sequence, determine the number of possible different codons that will code for the same amino acid.</LI><FIGURE><FIGCAPTURE class="figjnnbl"><B>Figure 6.</B> Algorithm used to determine the possible number of synonymous codons for each amino acid position in the gene.
                                    </FIGCAPTURE></FIGURE><LI>Determine positions of PAM sequence close to the exon-exon junctions.</LI><OL type="I"><LI>Analyze the total number of sgRNAs necessary for each PAM sequence to cover all possibilities of synonymous mutations (product of possible codons).</LI><LI>Determined by the identity necessary between sgRNA and target sequence.</LI></OL><FIGURE><FIGCAPTURE class="figjnnbl"><B>Figure 7.</B> Algorithm used to find possible PAM sequences on the gene.
                                    </FIGCAPTURE></FIGURE><LI>Gather the number of all PAM positions in the gene and determine the total number of sgRNAs necessary, based on the codon variations.</LI><FIGURE><FIGCAPTURE class="figjnnbl"><B>Figure 8.</B> Algorithm used to determine the total number of possible variations of codons behind a specific PAM sequence.
                                    </FIGCAPTURE></FIGURE><LI>Select the PAM sequence that has the minimal number of sgRNAs possible and generate sgRNAs. </LI><FIGURE><FIGCAPTURE class="figjnnbl"><B>Figure 9.</B> Algorithm used to choose the position with smallest variation possible behind the PAM sequence and generate the sgRNA_array for this position. 
                                    </FIGCAPTURE></FIGURE><LI>Output the position of PAM sequence in the gene and all the sgRNAs’ sequence.</LI></OL></DIV></DIV><DIV id="Gene-Doping-Model" l=""><H3 class="jnnbl">2.2 Gene Doping Model</H3><P>First, we modeled the transit of the injected gene doping viral vectors from the injection site to the target cells. Both, intramuscular (IM) and intravenous (IV) injection methods were considered.</P><FIGURE><FIGCAPTURE class="figjnnbl"><B>Figure 10.</B>  Pharmacological compartment models for both IV and IM injections. The arrows indicate viral vector transfer from one body compartment to another and the kinetic rate constant (k) associated with the transfer.</FIGCAPTURE></FIGURE><BUTTON class="collapsible cjnnbl">Detailed Compartment Model</BUTTON><DIV class="content"><P>Our model starts from the point of injection, which is either intramuscularly (IM) or intravenously (IV). The advantage of intravenous injections is that the viral vectors immediately enter blood circulation. The means that more vectors reach the target kidney cells before they are degraded. However, intravenous injections require a qualified doctor to administer and can lead to vein damage such as phlebitis. Intramuscular injection does not have these issues. On top of this, they can supply relatively large volumes of the gene doping DNA as the muscles have larger uptake capacity than the veins. The major disadvantage of intramuscular injection compared to intravenous injections is the added degradation of the vectors in the muscle, leading to larger doses being required for the same effect as an IV dose. Finally, intramuscular injections can cause local swelling, drainage and severe pain at the site of injection. Nevertheless, we take both ways of administration into account.</P><P>Pharmacological compartment models are developed to understand the distribution of drugs administered to the human body by oral or, intramuscular, or intravenous routes (<A class="jnnbl" href="#References">Khanday <I>et al</I>. 2017</A>). They are formulated based on diffusion processes using Fick’s principle and law of mass action. The same diffusion processes affect the administered adenoviral vectors. We start with pharmacological compartment models for both injection types as displayed in Figure 10. Here it is assumed that the mixing of the vectors with blood is instantaneous, based on an article by <A class="jnnbl" href="#References">Tarr <I>et al</I>. 1933</A>). Prof. Beltman, Assistant Professor Biomedical Modelling at Leiden University, later agreed with this assumption.</P><P>There are multiple tissues producing EPO, including liver, brain and kidney tissue. The main cell population responsible for EPO production are the interstitial fibroblasts in the kidney, spanning an average population of approximately 100 million cells, which produce more than 80% of the EPO in blood (<A class="jnnbl" href="#References">Weidemann &amp; Johnson 2009</A>).</P><P>From the compartment models in <B class="jnnbl">figure 10</B>, equations 1 and 2 can be derived for the intravenous administration.</P><P>$$\frac{d[c_{blood}]}{dt} = -(k_{blood}+kel_{blood})[c_{blood}]+k_{tissue}[c_{tissue}]\tag{1}$$</P><P>$$\frac{d[c_{tissue}]}{dt} = k_{blood}[c_{blood}]-(k_{bind\,uptake}+k_{tissue})[c_{tissue}] \tag{2}$$</P><P>Similarly, equations 3, 4, and 5 can be derived for the intramuscular administration. In intramuscular administration, the viral vectors must first diffuse out of the muscle and enter the bloodstream. While in the muscle, the muscle macrophages break down and eliminate the viral vectors. </P><P>$$\frac{d[c_{muscle}]}{dt} = -(k_{muscle}+kel_{muscle})[c_{muscle}]\tag{3}$$</P><P>$$\frac{d[c_{blood}]}{dt} = -(k_{blood}+kel_{blood})[c_{blood}]+k_{tissue}[c_{tissue}]+k_{muscle}[c_{muscle}] \tag{4}$$</P><P>$$\frac{d[c_{tissue}]}{dt} = k_{blood}[c_{blood}]-(k_{bind\,uptake}+k_{tissue})[c_{tissue}] \tag{5}$$</P><P>The initial values and constants used are shown in <B class="jnnbl">tables 2 and 3</B>, respectively in the next dropdown.</P></DIV><BUTTON class="collapsible cjnnbl">Initial Values and Constants in the Compartment Model</BUTTON><DIV class="content"><CENTER><TABLE width="100%"><CAPTION class="figjnnbl"><B>Table 2.</B> Overview of the initial values used in the human body model for an adenoviral vector. </CAPTION><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderjnnbl">Constants</TH><TH class="tableheaderjnnbl">Values</TH><TH class="tableheaderjnnbl">Meaning</TH></TR><TR class="tableunevenjnnbl"><TD>$$[c_{blood} (t=0)]$$</TD><TD>94 billion [#/mL] for IV Single Dose 
                                        64 billion [#/mL] followed by smaller doses of 18 billion vectors every 20 days for IV Microdosing 
                                        0 [#/mL]  for IM
                                    </TD><TD>Initial injection of vectors intravenously</TD></TR><TR class="tableevenjnnbl"><TD>$$[c_{tissue} (t=0)]$$</TD><TD>0 [#/mL]</TD><TD>Initial vector concentration in in tissue</TD></TR><TR class="tableevenjnnbl"><TD>$$[c_{muscle} (t=0)]$$</TD><TD>141 billion [#/mL] for IM
                                        96 billion vectors followed by smaller doses of 27 billion vectors every 20 days for IM Microdosing 
                                        0 [#/mL]  for IV</TD><TD>Initial injection of vectors intramuscularly</TD></TR></TBODY></TABLE></CENTER><CENTER><TABLE width="100%"><CAPTION class="figjnnbl"><B>Table 3.</B> Overview of the constants used in the human body model for an adenoviral vector. </CAPTION><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderjnnbl">Rate Constants</TH><TH class="tableheaderjnnbl">Values (days<SUP>-1</SUP>)</TH><TH class="tableheaderjnnbl">Meaning</TH><TH class="tableheaderjnnbl">Source</TH></TR><TR class="tableevenjnnbl"><TD>$$k_{tissue}$$</TD><TD>1440</TD><TD>Vector displacement from tissue to blood</TD><TD>Estimate based on mean blood circulation time</TD></TR><TR class="tableevenjnnbl"><TD>$$k_{muscle}$$</TD><TD>1440</TD><TD>Vector displacement from muscle to blood in IM injections</TD><TD>Estimate based on mean blood circulation time</TD></TR><TR class="tableevenjnnbl"><TD>$$k_{blood}$$</TD><TD>1440</TD><TD>Vector displacement from blood to tissue</TD><TD>Estimate based on mean blood circulation time</TD></TR><TR class="tableevenjnnbl"><TD>$$kel_{blood}$$</TD><TD>720</TD><TD>Elimination of viral vectors from the blood</TD><TD><A class="jnnbl" href="#References">Ganesan <I>et al</I>. 2011</A></TD></TR><TR class="tableevenjnnbl"><TD>$$kel_{muscle}$$</TD><TD>720</TD><TD>Elimination of viral vectors from the muscle</TD><TD><A class="jnnbl" href="#References">Ganesan <I>et al</I>. 2011</A></TD></TR><TR class="tableevenjnnbl"><TD>$$k_{bind\,uptake}$$</TD><TD>8.64</TD><TD>Endosomal uptake</TD><TD><A class="jnnbl" href="#References">Varga <I>et al</I>. 2005</A></TD></TR></TBODY></TABLE></CENTER></DIV><P>Upon reaching the target kidney cells, the gene doping viral vectors infect the cells. The infection process and production of gene doping EPO was modeled through a series of kinetic equation. Kidney cells have a mean lifespan of around 57 days. Upon their death, the infected cells release the gene doping DNA back into the bloodstream as cell free DNA (cfDNA). This increases the detection window of the gene doping DNA. </P><FIGURE><FIGCAPTURE class="figjnnbl"><B>Figure 11.</B>  Cellular uptake of gene doping vectors. Figure was inspired by <A class="jnnbl" href="#References">Varga <I>et al</I>. 2005</A>.</FIGCAPTURE></FIGURE><BUTTON class="collapsible cjnnbl">Detailed Intracellular Gene Delivery Model</BUTTON><DIV class="content"><H4 class="jnnbl">Cellular uptake</H4><P>After the uptake of the vectors in the tissue, the stage of cellular uptake ensues. In this process we modelled multiple steps as indicated in Figure 11 according to a model first developed by <A class="jnnbl" href="#References">Varga <I>et al</I>. 2005</A>. The cellular uptake of gene doping vectors as depicted in Figure 11 can be dissected into multiple steps described by a set of coupled differential equations, for which the constants are given in Table 3.</P><P>First, the complex is taken up by endocytosis after which it is either degraded or taken up, as represented by equations 6 and 7.</P><P>$$\frac{d[vesicle]}{dt} = k_{bind\,uptake}[c_{tissue}]-(k_{Escape}+k_{deg\,vesicle})[vesicle] \tag{6}$$</P><P>$$\frac{d[complex\,intracell]}{dt} = k_{escape}[vesicle]-k_{unpack}[complex\,intracell]-k_{bind\,vector}[complex\,intracell]\tag{7}$$</P><P>Second, vector dissociation and either degradation or nuclear target complex binding takes place in either dissociated or complexed form, as given by equations 8, 9 and 10.</P><P>$$\frac{d[plasmid]}{dt} = k_{unpack}[complex\,intracell]-k_{bind \,plasmid}[plasmid]-k_{deg}[plasmid]\tag{8}$$</P><P>$$\frac{d[plasmid\,bound]}{dt} = k_{bind \,plasmid}[plasmid]-k_{NPC}[plasmid\,bound] \tag{9}$$</P><P>$$\frac{d[complex\,bound]}{dt} = k_{bind \,vector}[complex\,intracell]-k_{NPC}[complex\,bound] \tag{10}$$</P><P>Subsequently, transport to the inner part of the nucleus is believed to take place through first binding to a nuclear pore complex (NPC) and finally inside the nucleus dissociation of the nuclear target complex takes place. This is represented by equations 11 till 16.</P><P>$$\frac{d[complex\,boundNPC]}{dt} = k_{NPC}[complex\,bound]-k{in}[complex\,boundNPC] \tag{11}$$</P><P>$$\frac{d[complex\,bound\,nucleus]}{dt} = k{in}[complex\,boundNPC]-k_{dissociation}[complex\,bound\,nucleus] \tag{12}$$</P><P>$$\frac{d[complex\,nucleus]}{dt} = k_{dissociation}[complex\,bound\,nucleus] - k_{unpack2}[complex\,nucleus] \tag{13}$$</P><P>$$\frac{d[plasmid\,boundNPC]}{dt} = k_{NPC}[plasmid\,bound] -k_{in2}[plasmid\,boundNPC] \tag{14}$$</P><P>$$\frac{d[plasmid\,bound\,nucleus]}{dt} =k_{in2}[plasmid\,boundNPC]-k{kissociation2}[plasmid\,bound\,nucleus] \tag{15}$$</P><P>$$\frac{d[plasmid\,nucleus]}{dt} =k{kissociation2}[plasmid\,bound\,nucleus] + k_{unpack2}[complex\,nucleus]$$</P><P>$$- k_{cell\,death}[plasmid\,nucleus] \tag{16}$$</P><H4 class="jnnbl">Detection of cell free doping DNA</H4><P>Apart from the effect of the gene doping EPO on the production of red blood cells, the purpose of the model is to determine the dynamics of the detectable cfDNA concentration in the blood. Cell free doping DNA is released from dying infected cells and circulates in the blood where it is assumed to degrade at the same rate as natural cfDNA.</P><P>$$\frac{d[Doping\,DNA]}{dt} =k_{cell\,death}[plasmid\,nucleus]-kel_{cfDNA}[Doping\,DNA] \tag{17}$$</P><P>Equation 17 provides us with a detection window for which we assume that we can detect both, DNA left in the tissue and bloodstream after injection, and DNA released after cell death (kcelldeath). Any other degradation terms or transient expression we incorporated in the constants used. Based on the above model we obtained the concentration developments of cfDNA in time for both intramuscular and intravenous injections and the estimated detection windows where we assumed an estimated detection limit of 100 copies DNA. The concentrations of DNA over time were used in the laboratory for our sample preparation to mimic real life detection potential. The cell death rate constant is directly linked to the average lifetime of renal interstitial fibroblasts, which we estimated to be around 57 days based upon measurements in chicks by <A class="jnnbl" href="#References">Weissmanshomer <I>et al</I>. 1975</A>.</P><P>According to <A class="jnnbl" href="#References">Haller <I>et al</I>. 2018</A>, the expected concentration of doping cell free DNA may be higher for athletes in endurance and intermittent sports. Haller <I>et al</I>. found a 22.7 fold increase in venous cfDNA concentrations in footballers after a professional football match. Given the high amount of training top level athletes endure, this finding leads us to believe that the detection window might be even longer than our model predicts.</P><H4 class="jnnbl">The Protein Effect</H4><P>Lastly, the uptaken DNA can be translated into proteins according to equation 18, after which it can be exported to the extracellular environment according to equation 19.</P><P>$$\frac{d[protein]}{dt} =k_{protein}[plasmid\,nucleus]-k_{deg\,protein}[protein]-k_{export}[protein] \tag{18}$$</P><P>$$\frac{d[protein\,extracellular]}{dt} =k_{export}[protein]-k_{deg\,protein\,extracellular}[protein\,extracellular]\tag{19}$$</P></DIV><BUTTON class="collapsible cjnnbl">Constants for Intracellular Gene Delivery Model</BUTTON><DIV class="content"><CENTER><TABLE><CAPTION class="figjnnbl"><B>Table 4.</B> Overview of the constants used in the human body model for an adenoviral vector. </CAPTION><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderjnnbl">Rate Constants (Ad5)</TH><TH class="tableheaderjnnbl">Values (days<SUP>-1</SUP>)</TH><TH class="tableheaderjnnbl">Meaning</TH><TH class="tableheaderjnnbl">Source</TH></TR><TR class="tableevenjnnbl"><TD>$$k_{bind\,uptake}$$</TD><TD>8.64</TD><TD>Endosomal uptake</TD><TD><A class="jnnbl" href="#References">Varga <I>et al</I>. 2005</A></TD></TR><TR class="tableevenjnnbl"><TD>$$k_{deg\,vesicle}$$</TD><TD>28.8</TD><TD>Degradation of complex within uptake vesicle</TD><TD><A class="jnnbl" href="#References">Varga <I>et al</I>. 2005</A></TD></TR><TR class="tableevenjnnbl"><TD>$$k_{escape}$$</TD><TD>23.0</TD><TD>Complex movement from endosome to intracellular</TD><TD><A class="jnnbl" href="#References">Varga <I>et al</I>. 2005</A></TD></TR><TR class="tableevenjnnbl"><TD>$$k_{bind\,vector}$$</TD><TD>144</TD><TD>Binding of gene delivery vector to compound targeting for the nucleus</TD><TD><A class="jnnbl" href="#References">Varga <I>et al</I>. 2005</A></TD></TR><TR class="tableevenjnnbl"><TD>$$k_{unpack}, k_{unpack2}$$</TD><TD>144</TD><TD>Plasmid detaches from vector either in cytoplasm(1) or in the nucleus(2)</TD><TD><A class="jnnbl" href="#References">Varga <I>et al</I>. 2005</A></TD></TR><TR class="tableevenjnnbl"><TD>$$k_{deg}$$</TD><TD>7.2</TD><TD>Degradation of unbound plasmid in the cytoplasm</TD><TD><A class="jnnbl" href="#References">Lechardeur <I>et al</I>. 1999</A></TD></TR><TR class="tableevenjnnbl"><TD>$$k_{bind\,plasmid}$$</TD><TD>2.88</TD><TD>Binding of plasmid to compound targeting for the nucleus</TD><TD><A class="jnnbl" href="#References">Varga et al. 2001</A></TD></TR><TR class="tableevenjnnbl"><TD>$$k_{NPC}$$</TD><TD>1.44*10<SUP>6</SUP></TD><TD>Binding formed complexes to Nuclear Pore Complex</TD><TD><A class="jnnbl" href="#References">Vacik <I>et al</I>. 1999</A><A class="jnnbl" href="#References">Wilson <I>et al</I>. 1999</A><A class="jnnbl" href="#References">Chan <I>et al</I>. 1999</A><A class="jnnbl" href="#References">Dean <I>et al</I>. 1997</A></TD></TR><TR class="tableevenjnnbl"><TD>$$k_{in}, k_{in2}$$</TD><TD>2.88</TD><TD>Uptake nucleus through Nuclear Pore Complex</TD><TD><A class="jnnbl" href="#References">Varga <I>et al</I>. 2001</A></TD></TR><TR class="tableevenjnnbl"><TD>$$k_{dissociation}, k_{dissociation2}$$</TD><TD>1.44*10<SUP>6</SUP></TD><TD>Dissociation from the NPC targeting compound</TD><TD><A class="jnnbl" href="#References">Moroianu <I>et al</I>. 1996</A></TD></TR><TR class="tableevenjnnbl"><TD>$$k_{protein}$$</TD><TD>14.4</TD><TD>Protein production from plasmid</TD><TD><A class="jnnbl" href="#References">Schaffer <I>et al</I>. 1998</A></TD></TR><TR class="tableevenjnnbl"><TD>$$k_{degprot}$$</TD><TD>1.04</TD><TD>Cytoplasmic degradation of the protein</TD><TD><A class="jnnbl" href="#References">Fuertinger <I>et al</I>. 2012</A></TD></TR><TR class="tableevenjnnbl"><TD>$$k_{export}$$</TD><TD>1.44*10<SUP>6</SUP></TD><TD>Export protein to extracellular environment</TD><TD>Estimate</TD></TR><TR class="tableevenjnnbl"><TD>$$k_{deg\,protein\,extracellular}$$</TD><TD>1.04</TD><TD>Degradation of EPO protein in blood</TD><TD><A class="jnnbl" href="#References">Fuertinger <I>et al</I>. 2012</A></TD></TR><TR class="tableevenjnnbl"><TD>$$k_{cell\,death}$$</TD><TD>0.0167</TD><TD>Average death rate of renal interstitial fibroblast</TD><TD>Estimate based on chicks; <A class="jnnbl" href="#References">Weissmanshomer <I>et al</I>. 1975</A></TD></TR><TR class="tableevenjnnbl"><TD>$$kel_{cfDNA}$$</TD><TD>100</TD><TD>Clearance of cfDNA from the blood</TD><TD><A class="jnnbl" href="#References">Alegre <I>et al</I>. 2015</A></TD></TR></TBODY></TABLE></CENTER></DIV><P>The EPO from the infected cells is released into the bloodstream. The EPO reaches the bone marrow where it stimulates red blood cell production through erythropoiesis. Red blood cells begin as stem cells and go through a series of cell differentiations before maturing into red blood cells (<B class="jnnbl">figure 12</B>).</P><FIGURE><FIGCAPTURE class="figjnnbl"><B>Figure 12.</B>  Cell stages in red blood cell production. Stage 1 is the BFU-E cell stage. Stage 2 is the CFU-E cell stage. The proliferation rate of CFU-E cells depends on the concentration of EPO. Stage 3 is the erythroblasts stage. Stage 4 is the marrow reticulocytes stage. The length of time cells stay in this stage is dependent on the concentration of EPO. Stage 5 is the circulating red blood cell (RBC) stage. Neocytolysis of RBCs is triggered when the concentration of EPO drops below a certain threshold. The red blood cell count acts as a feedback loop that determines the endogenous production of EPO.</FIGCAPTURE></FIGURE><P>EPO promotes the proliferation of CFU-E cells. Increases in EPO levels lead to faster proliferation rates of CFU-E cells. Increases in EPO levels reduces the marrow transit time of cells for marrow reticulocytes, releasing the cells into the blood in a shorter time frame. In the blood, the reticulocytes mature into red blood cells which increase the oxygen carrying capacity of blood. If the partial partial pressure of oxygen in the blood drops below normal due to an excess of red blood cells, the endogenous production of EPO in the kidneys decrease. A low enough concentration of EPO in the blood will trigger macrophages to phagocytose young red blood cells. The process is referred to as neocytolysis and allows the body to quickly respond to changes environmental changes including changes in altitude (<A class="jnnbl" href="#References">Rice <I>et al</I>. 2005</A>).</P><BUTTON class="collapsible cjnnbl">Erythropoiesis: The Production of Red Blood Cells</BUTTON><DIV class="content"><P>With the doping DNA degradation and doping EPO formation determined, the effect of EPO on erythropoiesis, the process which produces red blood cells, is determined. We developed a model using an anemia EPO treatment model by <A href="#References" class="jnnbl">Fuertinger <EM>et al.</EM> 2012</A> as reference.</P><P>Red blood cells begin as stem cells and progress into different cell stages as they age. As progenitor and precursor cells age, they proliferate or undergo apoptosis at a rate dependent on the cell stage they are in. The resulting growth or decay rates may either be constant or dependent on the concentration of EPO in the blood. Burst-Forming Unit-Erythroid (BFU-E) cells have a very small number of EPO receptors. EPO concentration has no effect on BFU-E proliferation, their proliferation is assumed to be constant. After leaving the stem cell stage, cells stay in the BFU-E stage for 7 days, after which they enter the Colony-Forming Unit-Erythroid (CFU-E) stage. In this stage, the cells divide at a higher rate than in the BFU-E cell stage. The CFU-E cells have a large number of EPO receptors and are strongly dependent on EPO for their survival. Their rate of apoptosis is inversely related to the concentration of EPO. Under normal conditions within the human body, a large portion of CFU-E generated do not survive. As the concentration of EPO in the blood increases, the number of cells which survive increases.</P><P>After spending 6 days as CFU-E cells, the cells enter the erythroblasts stage. Here, the number of EPO receptors decline. During this stage, there is no evidence that additional divisions occur when production of EPO increases. For this reason we assume that the proliferation of erythroblasts is constant (<A class="jnnbl" href="#References">Lichtman <I>et al</I>. 2005</A>).</P><P>The cells stay in the erythroblasts stage for 5 days until they stop dividing, extrude their nuclei and mitochondria, and become marrow reticulocytes. Marrow reticulocytes no longer proliferate and their mortality rate is inversely dependent on iron concentration in the plasma. Since we assume that athletes have a sufficient iron supply, a constant apoptosis rate for marrow reticulocytes is assumed. The time cells stay in the reticulocytes stage is between 0.75 and 3 days. An increase in EPO concentration shortens the marrow transit time of reticulocytes.</P><P>Once reticulocytes are released from the bone marrow and enter the blood, they mature into erythrocytes (red blood cells) within 1-3 days. Reticulocytes have a hemoglobin content of around 27.5 ± 2.8 pg per cell and, red blood cells have a hemoglobin content of around 26.4 ± 2.4 pg per cell (<A class="jnnbl" href="#References">Fishbane <I>et al</I>. 1997</A>). Due to the similar ability of blood reticulocytes and red blood cells to carry oxygen, when red blood cells are discussed, we refer to both blood reticulocytes and mature red blood cells.  The lifespan of red blood cells (RBCs) in healthy human adults is about 120 days before their components are recycled by microphages (<A class="jnnbl" href="#References">Jandl 1987</A>). Over the course of this time a small number of RBCs die due to random daily breakdown or, internal or external bleeding. This is taken into account with a small apoptosis rate for RBCs. Adults have a red blood cell count ranging from about 20 to 30 trillion. Women have a blood cell count range of 3.5-5.5 trillion cells per liter, while men have a range of 4.3-5.9 trillion cells per liter (<A class="jnnbl" href="#References">Dean 2005</A>). The average red blood cell count is estimated to be 24.98 trillion by <A class="jnnbl" href="#References">Lichtman <I>et al</I>. 2005</A>. The entire process, from stem cell to red blood cell recycling by microphages, takes 141 days.</P><P>The endogenous release of EPO is inversely related to the partial pressure of oxygen in the blood. The partial pressure of oxygen in the blood is proportionally related to the number of circulating red blood cells. An increase in the red blood cell population in blood will decrease endogenous EPO production. If the concentration of EPO in the blood falls below a certain level (9.8 mU/ml in the case of this model), neocytolysis is triggered. Neocytolysisis the selective lysis of young red blood cells by the body to allow it to decrease its red blood count at a higher rate and reach the desired partial pressure of oxygen in the blood.</P></DIV><P>The series of partial differential equations (PDEs) that describe red blood production was modeled with a simplified linear age population method which provided similar accuracy to more computationally intensive PDE solvers. The red blood cell production model is then combined with the compartment and infection model to determine the effects of EPO gene doping on red blood cell count.</P><BUTTON class="collapsible cjnnbl">Comprehensive cell cycle model for RBCs in the context of EPO  </BUTTON><DIV class="content"><P>The process of red blood cell formation, from stem cell to red blood cell can be modelled in time through a series of partial differential equations based on the age of the cells. Each of the cell stages (BFU-E, CFU-E, erythroblasts, marrow reticulocytes and red blood cells) is represented by a partial differential equation. The general form of the equation is shown in <B class="jnnbl">figure 13</B>.</P><FIGURE><FIGCAPTURE class="figjnnbl"><B>Figure 13.</B>  Breakdown of the general PDE used to model the various cell stages that occur in the process of red blood cell production.</FIGCAPTURE></FIGURE><P>Starting at cell age 0 days, when stem cells become BFU-E cells, the age span up until cell age 141 days. When red blood cells are recycled by macrophages, the cells can be broken up into populations of cells at a certain age. For every cell stage, the population density of cells at a given maturity and time can modelled as a population mesh. A population mesh is a numerical estimator that breaks down the process of red blood cell production into populations at a given age. A mesh point is the population of cells of a specific age. When modelled as a mesh, the following assumptions of the relation between a change in time and a change in maturity can be made.</P><B><I class="jnnbl">Assumptions</I></B><OL type="1"><LI>For a population of cells u of age μ and at time t, a change in time Δt will result in an equal change in maturity Δμ, assuming that the maturation velocity \(vs(E(t))=1\).</LI><LI>The maturation velocity is 1 for every stage except the marrow reticulocytes, where the time cells stay in this stage depends on the concentration of EPO.</LI></OL><P>The commitment of stems cells to becoming red blood cells is an irreversible event (<A class="jnnbl" href="#References">Fuertinger <I>et al</I>. 2012</A>). Cells cannot regress back to a previous cell type or switch to a different differentiation pathway. For this reason, assumption 1 can be made. A change in time will lead to an equal change in the age of the cell. As a result, the finer you make the population mesh the smaller the time step is. This increases the accuracy of the model as the cells react to changes in EPO concentration in time faster. Once they reach the age at which they differentiate into another cell stage, the resulting growth and decay rates that govern them will change.</P><FIGURE><FIGCAPTURE class="figjnnbl"><B>Figure 14.</B>  Transition of cells from one stage to another through differentiation. (a) Follows one population set of reticulocyte cells as they age over time and transition to marrow reticulocytes. (b) Shows the result of a simulation of the population density of cells every 0.1 days of maturity at a steady state EPO concentration of 9.8 mU/ml.</FIGCAPTURE></FIGURE><P>When a time step occurs, the cells in each mesh point (\(u_n\)) experience growth or decay based on the stage they are in. The new population of cells is then shifted to the next age mesh point. The general form of the equation that governs the change of population density at each time step is below.</P><P>$$ u_n = u_{n-1}\times e^{(\beta-\alpha(EPO(t)) \times \Delta t} \tag{20}$$</P><P>$$ u_{n+1} = u_{n}\times e^{(\beta-\alpha(EPO(t)) \times \Delta t} \tag{21}$$</P><P>$$ \Delta t = \mu_{n+1} - \mu_n = \Delta \mu \tag{22}$$</P><P>The dynamics of the mesh in response to a gene doping injection can be seen in <B class="jnnbl">figure 15</B>.</P><FIGURE><FIGCAPTURE class="figjnnbl"><B>Figure 15.</B>  An animation of the dynamic red blood cell formation model. The injection 141 billion copies of the EPO cDNA in adenoviruses occurs at day 0. The infected cells produce extra EPO leading to a maximum concentration of EPO of 31.3 mU/ml (3.2 times more that the steady state EPO concentration of 9.8 mU/ml).</FIGCAPTURE></FIGURE><P>To calculate the total population of each stage, the trapezoid rule was used to calculate the area under the population mesh and estimate the total population.</P><H4 class="jnnbl">BFU-E Cells</H4><P>10<SUP>8</SUP> stem cells become BFU-E cells on day 0. BFU-E cells grow exponentially as they age. Their proliferation rate is not dependent on EPO. As result, the population of BFU-E cells grows at a constant exponential rate as they age.</P><P>$$ BFU{-}E_n = BFU{-}E_{n-1}\times e^{\beta_{BFU-E} \times \Delta t} \tag{23}$$</P><P>$$ BFU{-}E_{n+1} = BFU{-}E_{n}\times e^{\beta_{BFU-E} \times \Delta t} \tag{24}$$</P><P>Cells committed to become red blood cells continue to grow in this manner until they reach the age of 7 days. At this point, the BFU-E differentiate into CFU-E cells.</P><H4 class="jnnbl">CFU-E Cells</H4><P>CFU-E cells are strongly dependent on EPO for their survival.</P><P>$$ CFU{-}E_n = CFU{-}E_{n-1}\times e^{(\beta_{CFU{-}E}-\alpha_{CFU{-}E}(EPO(t))) \times \Delta t} \tag{25}$$</P><P>$$ CFU{-}E_{n+1} = CFU{-}E_{n}\times e^{(\beta_{CFU{-}E}-\alpha_{CFU{-}E}(EPO(t))) \times \Delta t} \tag{26}$$</P><P>The equation shows that the apoptosis rate of CFU-E cells is dependent on the concentration of EPO at a given time. As the concentration of EPO increases, the apoptosis rate of CFU-E cells decreases.</P><FIGURE><FIGCAPTURE class="figjnnbl"><B>Figure 16.</B>  The effect of EPO on the apoptosis of CFU-E cells.</FIGCAPTURE></FIGURE><P>The logistic equation that governs the apoptosis rate of CFU-E is seen in the equation below.</P><P>$$\alpha_{CFU{-}E}(EPO(t))= \frac{(a_1 - b_1)}{1+e^{k_1 \times EPO(t) - c_1}}+b_1 \tag{27}$$</P><P>The CFU-E stage starts once cells reach the age of 7 days and continues until cells differentiate into Erythroblasts at an age of 13 days.</P><H4 class="jnnbl">Erythroblasts</H4><P>The proliferation of erythroblasts does not depend on the concentration EPO. However, the population of cells entering the erythroblast stage at an age of 13 days depends on the concentration of EPO due to the CFU-E stage coming before.</P><P>$$ Erythroblasts_n = Erythroblasts_{n-1}\times e^{\beta_{Erythroblasts} \times \Delta t} \tag{28}$$</P><P>$$ Erythroblasts_{n+1} = Erythroblasts_{n}\times e^{\beta_{Erythroblasts} \times \Delta t} \tag{29}$$</P><P>Upon reaching the age of 18 days, erythroblast cells differentiate into marrow reticulocytes.</P><H4 class="jnnbl">Marrow Reticulocytes</H4><P>For the marrow reticulocytes, where the marrow transit time varies with EPO concentration, the same mesh method is used. Even with sufficient iron supply, a constant fraction of marrow reticulocytes is phagocytosed (<A class="jnnbl" href="#References">Fuertinger <I>et al</I>. 2012</A>). This is represented by a constant apoptosis rate independent of EPO concentration.</P><P>$$ Reticulocytes_n = Reticulocytes_{n-1}\times e^{-\alpha_{Reticulocytes} \times \Delta t} \tag{30}$$</P><P>$$ Reticulocytes_{n+1} = Reticulocytes_{n}\times e^{-\alpha_{Reticulocytes} \times \Delta t} \tag{31}$$</P><P>The amount time it takes for marrow reticulocytes to leave the bone marrow and enter the blood decreases as the concentration of EPO increases. A faster marrow transit time means marrow reticulocytes spend more time as blood reticulocytes. The oldest marrow reticulocytes will become RBCs first when marrow transit time decreases. To account for the change in transit time, the stage boundary between reticulocytes and red blood cells is allowed to shift depending on the concentration of EPO.</P><FIGURE><FIGCAPTURE class="figjnnbl"><B>Figure 17.</B>  The effect of EPO on reticulocyte marrow transit time.</FIGCAPTURE></FIGURE><P>The logistic equation that governs the marrow transit time can be seen below.</P><P>$$ \mu_{Reticulocytes,max}(EPO(t)) = a_2 - \frac{b_2}{1+e^{k_2 \times EPO(t) - c_2}} \tag{32}$$</P><FIGURE><FIGCAPTURE class="figjnnbl"><B>Figure 18.</B>  The effect of EPO concentration on marrow transit time. The injection 141 billion copies of the EPO cDNA in adenoviruses occurs at day 0. The infected cells produce extra EPO leading to a maximum concentration of EPO of 31.3 mU/ml (3.2 times more that the steady state EPO concentration of 9.8 mU/ml). As EPO concentration in the blood increases, the marrow transit time will decrease, shifting the boundary between marrow reticulocytes and circulating red blood cells to the left. A decrease in EPO concentration increases the marrow transit time and shifts the boundary to the right. The oscillations in the marrow transit time are due to the oscillations in the EPO concentration caused by gene doping.</FIGCAPTURE></FIGURE><P>Reticulocytes will stay in the bone marrow until age 18.75 to 21 days after which they leave the bone marrow and circulate in the blood, where they mature into red blood cells.</P><H4 class="jnnbl">Red Blood Cells and Blood Reticulocytes</H4><P> There is a fixed rate of random daily breakdown of red blood cells due to random daily breakdown or, internal or external bleeding. This is represented by a constant apoptosis rate independent of EPO concentration.</P><P>$$ RBCs_n = RBCs_{n-1}\times e^{-\alpha_{RBCs}(EPO(t), \, \mu_{RBCs}) \times \Delta t} \tag{33}$$</P><P>$$ RBCs_{n+1} = RBCs_{n}\times e^{-\alpha_{RBCs}(EPO(t),\, \mu_{RBCs}) \times \Delta t} \tag{34}$$</P><P>Between the ages of of 35 to 42 days, young red blood cells will undergo neocytolysis if the concentration of EPO in the blood drops below 9.8 mU/ml. The neocytolysis rate will increase as EPO levels drop until the level of EPO reach 3.3 mU/ml, when it maximum.</P><P>$$\alpha_{RBCs}(EPO(t), \mu_{RBCs}) = \alpha_{RBCs, \, r} + min\Big(\frac{c_E}{EPO(t)^{k_E}}, \, b_E \Big),$$
                                $$\, for \, EPO(t) &lt; \tau_E, \, 35 \, days \leq \mu_{RBCs} \leq 42 \, days \tag{35}$$</P><P>$$\alpha_{RBCs}(EPO(t), \mu_{RBCs}) = \alpha_{RBCs, \, r}, \, otherwise \tag{36}$$</P><H4 class="jnnbl">Feedbackloop via EPO</H4><P>The natural release of EPO from the kidneys depends on the partial pressure of oxygen in the blood. If the partial pressure of oxygen in the blood decreases due to a lack of red blood cells, more EPO is released. This leads to an increase in red blood cell production, leading to a higher population of circulating red blood cells and a higher partial pressure of oxygen. The partial pressure of oxygen and the number of circulating red blood cell are therefore assumed to be proportional.</P><P>The amount \(E^{end}_{in}(t)\)  of EPO released by the kidney can be estimated by the use of the total population of red blood cells \(RBCs(t)\), which consist of all circulating red blood cells. The equation that governs the natural release of EPO uses a scaled red blood cell count with TBVbeing the total blood volume, and is shown below:</P><P>$$ EPO^{end}_{in}(t) = \bigg(\frac{(a_3 - b_3)}{1+e^{k_3 \times \tilde{M}(t) - c_3}}+b_3\bigg)\ \times \frac{1}{TBV} \tag{37} $$</P><P>$$ \tilde{M}(t) = 10^{-8} \times \frac{RBCs(t)}{TBV} \tag{38} $$</P><FIGURE><FIGCAPTURE class="figjnnbl"><B>Figure 19.</B> The effect of red blood cell population on the release of EPO from the kidneys. The units of EPO released from kidneys is mU per ml per day.</FIGCAPTURE></FIGURE><P>The behavior of the endogenous EPO concentration Eend(t) in plasma is modeled by the following ordinary differential equation:</P><P>$$\frac{dEPO_{end}(t)}{dt} = EPO^{end}_{in}(t) - k_{deg\,protein\,extracellular}EPO_{end}(t) \tag{39}$$</P><P>When gene doping has been administered, the concentration of gene doping EPO at a given time is given by Equation 19. It is assumed that the degradation of natural EPO and gene doping EPO occurs at the same rate. The overall concentration of EPO in plasma consists of the naturally produced erythropoietin and the administered gene doping EPO:</P><P>$$ EPO(t) = EPO_{end}(t) + EPO_{doping}(t) \tag{40}$$</P></DIV><BUTTON class="collapsible cjnnbl">Overview of the constants used in the red blood cell production model</BUTTON><DIV class="content"><P>The parameters used in the model are based on the parameters derived in <A class="jnnbl" href="#References">Fuertinger <I>et al</I>. 2012</A>, except for a3 and b3 which are derived from <A class="jnnbl" href="#References"> Roberts, 2011</A>.</P><CENTER><TABLE><CAPTION class="figjnnbl"><B>Table 4.</B> Table of the parameters used in the red blood cell production model, their values, their units, and their meaning. </CAPTION><TBODY><TR class="tableheaderadpbl"><TH class="tableheaderjnnbl">Parameters</TH><TH class="tableheaderjnnbl">Values</TH><TH class="tableheaderjnnbl">Units</TH><TH class="tableheaderjnnbl">Meaning</TH></TR><TR class="tableevenjnnbl"><TD>$$\beta_{BFU-E}$$</TD><TD>0.2</TD><TD>1/day</TD><TD>Proliferation rate for BFU-E cells</TD></TR><TR class="tableevenjnnbl"><TD>$$\beta_{CFU-E}$$</TD><TD>0.57</TD><TD>1/day</TD><TD>Proliferation rate for CFU-E cells</TD></TR><TR class="tableevenjnnbl"><TD>$$\beta_{Erythroblasts}$$</TD><TD>1.024</TD><TD>1/day</TD><TD>Proliferation rate for erythroblasts</TD></TR><TR class="tableevenjnnbl"><TD>$$\mu_{BFU-E, \, max}$$</TD><TD>7</TD><TD>Days</TD><TD>Maximal maturity for BFU-E cells</TD></TR><TR class="tableevenjnnbl"><TD>$$\mu_{CFU-E, \, min}$$</TD><TD>7</TD><TD>Days</TD><TD>Minimal maturity for CFU-E cells</TD></TR><TR class="tableevenjnnbl"><TD>$$\mu_{CFU-E, \, max}$$</TD><TD>13</TD><TD>Days</TD><TD>Maximal maturity for CFU-E cells</TD></TR><TR class="tableevenjnnbl"><TD>$$\mu_{Erythroblasts, \, min}$$</TD><TD>13</TD><TD>Days</TD><TD>Minimal maturity for erythroblasts</TD></TR><TR class="tableevenjnnbl"><TD>$$\mu_{Erythroblasts, \, max}$$</TD><TD>18</TD><TD>Days</TD><TD>Maximal maturity for erythroblasts</TD></TR><TR class="tableevenjnnbl"><TD>$$\mu_{Reticulocytes, \, max}(EPO(t))$$</TD><TD>18.75 to 21</TD><TD>Days</TD><TD>Maximal maturity for marrow reticulocytes</TD></TR><TR class="tableevenjnnbl"><TD>$$\alpha_{Reticulocytes}$$</TD><TD>0.089</TD><TD>1/day</TD><TD>Rate of ineffective erythropoiesis in the marrow reticulocytes stage</TD></TR><TR class="tableevenjnnbl"><TD>$$\alpha_{RBCs}$$</TD><TD>0.005</TD><TD>1/day</TD><TD>Intrinsic mortality rate for erythrocytes</TD></TR><TR class="tableevenjnnbl"><TD>$$a_1, \, b_1$$</TD><TD>0.35, 0.07</TD><TD>1/day</TD><TD>Constants for the sigmoid apoptosis rate for CFU-E cells</TD></TR><TR class="tableevenjnnbl"><TD>$$c_1, \, k_1$$</TD><TD>3, 0.14</TD><TD>Dimensionless, ml/mU</TD><TD>Constants for the sigmoid apoptosis rate for CFU-E cells</TD></TR><TR class="tableevenjnnbl"><TD>$$a_2, \, b_2$$</TD><TD>3.225, 2.475</TD><TD>Days</TD><TD>Constants for the sigmoid maturation velocity/marrow transit time for marrow reticulocytes</TD></TR><TR class="tableevenjnnbl"><TD>$$c_2, \, k_2$$</TD><TD>2.3, 0.2</TD><TD>Dimensionless, ml/mU</TD><TD>Constants for the sigmoid maturation velocity/marrow transit time for marrow reticulocytes</TD></TR><TR class="tableevenjnnbl"><TD>$$a_3, \, b_3$$</TD><TD>9.1, 0.2</TD><TD>Dimensionless, ml/mU</TD><TD>Constants for the sigmoid function governing the release of EPO from the kidneys </TD></TR><TR class="tableevenjnnbl"><TD>$$\mu_{RBCs, \, neocytolysis \, min}$$</TD><TD>35</TD><TD>Days</TD><TD>Lower bound of erythrocytes which are possibly exposed to neocytolysis</TD></TR><TR class="tableevenjnnbl"><TD>$$\mu_{RBCs, \, neocytolysis \, max}$$</TD><TD>42</TD><TD>Days</TD><TD>Upper bound of erythrocytes which are possibly exposed to neocytolysis</TD></TR><TR class="tableevenjnnbl"><TD>$$\mu_{RBCs, \, max}$$</TD><TD>141</TD><TD>Days</TD><TD>Maximal life span for red blood cells</TD></TR><TR class="tableevenjnnbl"><TD>$$b_E$$</TD><TD>0.1</TD><TD>1/day</TD><TD>Constant in the mortality rate for red blood cells</TD></TR><TR class="tableevenjnnbl"><TD>$$c_E$$</TD><TD>3.5</TD><TD>mU<SUP>3</SUP>/(ml<SUP>3</SUP>\(\times\)day)</TD><TD>Constant in the mortality rate for red blood cells</TD></TR><TR class="tableevenjnnbl"><TD>$$k_E$$</TD><TD>3</TD><TD>Dimensionless</TD><TD>Exponent in the mortality rate for red blood cells</TD></TR><TR class="tableevenjnnbl"><TD>$$\tau_E$$</TD><TD>9.8</TD><TD>mU/ml</TD><TD>EPO threshold for neocytolysis</TD></TR><TR class="tableevenjnnbl"><TD>$$k_{deg\,protein\,extracellular}$$</TD><TD>1.04</TD><TD>1/day</TD><TD>Degradation rate of EPO in the blood</TD></TR><TR class="tableevenjnnbl"><TD>$$S_0$$</TD><TD>10<SUP>8</SUP></TD><TD>1/day</TD><TD>Rate at which cells are committing to the erythroid lineage</TD></TR><TR class="tableevenjnnbl"><TD>$$TBV$$</TD><TD>5000</TD><TD>ml</TD><TD>Total blood volume</TD></TR></TBODY></TABLE></CENTER></DIV></DIV><H1 class="jnnbl">3. Results</H1><DIV id="sgRNA-Model-Results"><H3 class="jnnbl">3.1 sgRNA Array Model </H3><P>The main purpose of our algorithm is creating a tool to generate an array of sgRNAs necessary to screen for gene doping with our novel targeted sequencing platform. The model works with any input gene . It was tested for the EPO gene, as EPO is our main target for gene doping. We worked with the following information:</P><UL class="uljnnbl"><LI>Gene cds sequence: Human EPO cds  (GenBank: BC143225.1)</LI><LI>Type of Cas: dxCas9</LI><LI>PAM sequence: NG</LI><LI>sgRNA length: 20 bp of target sgRNA</LI><LI>Off target possibility (seed to target): 10 bp (50 % adjacent to PAM should be identical)</LI></UL><P>As seen in <B class="jnnbl">figure 20</B>, the algorithm detected several PAM sequences close to exon-exon junctions and found the minimum number of necessary guides for each one.</P><FIGURE><FIGCAPTURE class="figjnnbl"><B>Figure 20.</B> Results from the model to find the possible PAM sequences and the minimum number of sgRNAs needed at each exon-exon junction in the EPO gene coding sequence.</FIGCAPTURE></FIGURE><P>Figure 20 clearly shows that in junction 3 there is the optimal PAM sequence with smallest number of sgRNAs possible. This way, the algorithm generates the sgRNAs and gives an output with the position of such sgRNA. In this specific case, there were two PAM sequences near Junction 3 that had the same minimal number of possible sgRNAs necessary (12 guides each). The algorithm does not ignore one or the other, but outputs both options (or more if available). This array can be used to generate the library of sgRNAs for targeted next generation sequencing of gene doping.
                        </P></DIV><DIV id="Gene-Doping-Model-Results"><H3 class="jnnbl">3.2 Gene Doping Model </H3><P>The concentration of EPO gene doping DNA in blood increases rapidly in the first 1.5 days after injection before decreasing exponentially. Due to the rapid clearance of the adenoviral vectors from the blood and muscle, the majority of the infection events occurs right after injection. As more infected cells die, the number of infected cells decreases. This decreases the amount of doping DNA released into the blood over time, as seen in <B class="jnnbl">figure 21</B>. </P><FIGURE><FIGCAPTURE class="figjnnbl"><B>Figure 21.</B> Concentration of doping DNA in the blood over time after a single intramuscular injection of 141 billion viral vectors. The estimated detection limit of 1000 copies per mL of blood is based on the loss of DNA that occurs during sample preparation and targeted sequencing preparation </FIGCAPTURE></FIGURE><P>Using regression, the half-life of EPO gene doping DNA in blood was determined to be around 41 days.. While the half-life of cfDNA in blood is around 10 minutes, the slow release of the gene doping DNA from the dying infected cells significantly increases the detection window. EPO gene doping cDNA administered to cynomolgus macaques using adeno-associated viral vectors was detectable for up to 57 weeks after injection in infected white blood cells (<A class="jnnbl" href="#References">Ni <I>et al</I>. 2011</A>). The concentrations ranged from 333 to 500 copier per mL. While the target cell of the study was muscle cells, the DNA was found in white blood cells due to off target infections. Lymphocytes, a type of white blood cell, can live up to several months (<A class="jnnbl" href="#References">Tough and Sprent, 1995</A>). Our detection window could be further increased in the future if we moved away from detected doping DNA in plasma and isolated white blood cells instead.</P><P>Microdosing was determined to be the best doping method for doping athletes. The benefit of this method is that it avoids detection through the biological passport. Assuming that the athlete begins the treatment prior to becoming a professional athlete and continues microdosing during, their red blood cell count would appear to be constant and naturally high.</P><FIGURE><FIGCAPTURE class="figjnnbl"><B>Figure 22.</B> Effect of EPO gene doping using IM microdosing on red blood cell count. The initial dose is 96 billion vectors followed by smaller doses of 27 billion vectors every 20 days. The black horizontal lines are the range of EPO concentrations found in healthy adults. The red line indicates the red blood cell count at which the risk of stroke to due blood clot becomes significant.
                            </FIGCAPTURE></FIGURE><BUTTON class="collapsible cjnnbl">Comparison of a Large Single Dose and Microdosing for IM and IV Administration</BUTTON><DIV class="content"><P>The red blood cell production model is combined with the compartment and infection model to determine the effects on EPO gene doping on red blood cell count. Intravenous (IV) and intramuscular (IM) injection are compared, and two methods for gene doping are tested. The first consists of a single large injection while the second consists of a large initial dose followed by smaller doses every 20 days.</P><H4 class="jnnbl">Large Single Dose Gene Doping</H4><FIGURE><FIGCAPTURE class="figjnnbl"><B>Figure 23.</B> Comparison of intravenous (a) and intramuscular injection (b) at day 0 on red blood cell production. The black line is the total EPO (doping and natural) present in the blood over time. The red line is the number of red blood cells in circulation over time. The black horizontal dashed lines indicates the natural range of EPO levels. The red horizontal dashed line indicates the red blood cell count at which the risk of stroke due to a blood clot becomes significant. In (a), 94 billion Ad5 vectors containing the EPO doping gene are injected at day 0. In (b), 141 billion Ad5 vectors are required to achieve the same EPO and red blood cell production as the IV injection.
                                </FIGCAPTURE></FIGURE><P>IV and IM exhibit similar behavior with regards to EPO and red blood cell production. However, IM injection requires more vectors than IV for the same response due to the vectors being destroyed in the muscle and having to pass into the bloodstream. The detection limit of the doping DNA for large dose administration is 261 days after administration. The effects of EPO gene doping on red blood cell count is prevalent until 136 days after injection. The red blood cell count passes 30 trillion cells 19 days after injection.  It reaches its maximum count of 35.9 trillion cells 57 days after injection before dropping below 30 trillion 136 days after injection. Assuming the athlete is trying to maximize his or her performance for a competition, the gene doping test could be conducted during the competition and the doping athlete would be caught. Gene doping athletes are also at risk of being caught through the athletes biological passport system implemented by WADA. The biological passport is an individual electronic record of a professional athlete which profiles biological markers of doping.  Every time an athlete is tested for doping, his blood cell count is recorded. Any large fluctuations in red blood cell count would flag the athlete as a potential blood doper. Another problem with single dose administration is that red blood cell count falls below normal levels after the doping EPO wears off. This causes a decrease in the athlete’s performance following gene doping. For these reasons, athletes are likely to avoid single dose methods in favor of multidosing.</P><H4 class="jnnbl">Microdosing Gene Doping</H4><P>In multidose gene doping, an initial large dose of gene doping EPO DNA is injected to rapidly increase the red blood cell count. Following this, smaller doses are given to maintain the desired red blood cell count. This can be seen in Figure 22. For IV administration, an initial dose of 64 billion vectors is used followed by smaller doses of 18 billion vectors every 20 days.  For IM administration, an initial dose of 96 billion vectors is used followed by smaller doses of 27 billion vectors every 20 days.</P></DIV><P>The downside of this method is the requirement for constant microdosing. Repeated injections would cause noticeable damage to the veins. The athlete would then be required to disguise the injection sites or perform the IV microdosing in parts of the body normally covered. For this reason athletes would likely favor IM injection to IV injection as IM is less invasive. </P><FIGURE><FIGCAPTURE class="figjnnbl"><B>Figure 24.</B> Concentration of EPO gene doping DNA in the blood during IM microdosing with an initial dose is 96 billion vectors followed by smaller doses of 27 billion vectors every 20 days.
                            </FIGCAPTURE></FIGURE><P>Though the microdoses are smaller, the copies of DNA stay above our detection limit (40,000 to 50,000 fragments per mL) due to IM injections occuring every 20 days. While the athlete would bypass detection through the biological passport, they would be detected by our gene doping detection method.</P><BUTTON class="collapsible cjnnbl">Supplementary Results: Effect of Steady State EPO concentration on Steady State Red Blood Cell Count</BUTTON><DIV class="content"><P>To determine the quantity of EPO gene plasmids to be injected, the effect of the steady state concentration of EPO on the steady state red blood cell count is determined. This is achieved by removing the feedback loop that causes the red blood cell count to have an effect on EPO production. EPO concentration can then be set at a desired level and the PDEs used to model the system can be reduced to solvable interconnected ODEs. The resulting equations are shown below, where \(EPO_{SS}\) is the set steady state EPO concentration in the blood.</P><H4 class="jnnbl">BFU-E Cells</H4><P>$$ BFU{-}E(\mu_{BFU{-}E}) = S_0\times e^{\beta_{BFU-E} \times \mu_{BFU{-}E}} \tag{41}$$</P><H4 class="jnnbl">CFU-E Cells</H4><P>$$ CFU{-}E(\mu_{CFU{-}E}) = BFU{-}E(\mu_{BFU{-}E, \, max}) \times e^{(\beta_{BFU-E}-\alpha_{CFU{-}E}(EPO_{SS})) \times (\mu_{CFU{-}E} - \mu_{BFU{-}E, \, max})} \tag{42}$$</P><H4 class="jnnbl">Erythroblasts</H4><P>$$ Erythroblasts(\mu_{Erythroblasts}) = CFU{-}E(\mu_{CFU{-}E, \, max})\times e^{\beta_{Erythroblasts} \times (\mu_{Erythroblasts}-\mu_{CFU{-}E, \, max})} \tag{43}$$</P><H4 class="jnnbl">Marrow Reticulocytes</H4><P>$$ Reticulocytes(\mu_{Reticulocytes}) = Erythroblasts(\mu_{Erythroblasts, \, max})\times e^{-\alpha_{Reticulocytes} \times (\mu_{Reticulocytes}-\mu_{Erythroblasts, \, max})} \tag{44}$$</P><H4 class="jnnbl">Red Blood Cells and Blood Reticulocytes</H4><P>$$ RBCs(\mu_{RBCs}) = Reticulocytes(\mu_{Reticulocytes, \, max})\times e^{-\alpha_{RBCs}(EPO_{SS}, \, \mu_{RBCs}) \times (\mu_{RBCs}-\mu_{Reticulocytes, \, max})} \tag{45}$$</P><H4 class="jnnbl">Results of Steady State Analysis</H4><FIGURE><FIGCAPTURE class="figjnnbl"><B>Figure 25.</B> Effect of steady state EPO concentration in plasma on the steady state red blood cell count. The blue vertical line indicates the neocytolysis trigger, when EPO drops below 9.8 mU/ml. The black vertical line indicates the EPO level where neocytolysis reaches its maximum rate. Below this EPO level, the neocytolysis level stays a 0.1/day. The red horizontal line indicates the red blood cell count above which the risk of stroke due to blood clots becomes prevalent. The stroke risk occurs at a steady state EPO concentration of 18.3 mU/ml. 
                                </FIGCAPTURE></FIGURE><P>When the EPO feedback loop is incorporated, the time dependent model converged to an EPO concentration of 9.8 mU/ml, just above the neocytolysis trigger. In the steady state model, the red blood cell count is 24.72 trillion total circulating cells at an EPO concentration of 9.8 mU/ml. The error between the steady state model and the estimated count by <A class="jnnbl" href="#References">Lichtman <I>et al</I>. 2005</A> of 24.98 trillion  is 1.05%. Due to the large range of red blood cell counts and the low error between the model steady state and the expected average, the model parameters were not optimized to exactly match the expected red blood cell count.</P></DIV><BUTTON class="collapsible cjnnbl">Supplementary Results: Effect of Population Mesh Density on Steady State RBC Population</BUTTON><DIV class="content"><P>The density of the population mesh determines how many mesh points each day is split into. A mesh density of 10 means there are 10 points per day (\(\Delta t = \Delta \mu = \) 0.1 days), while a mesh density of 100 means there are 100 points per day (\(\Delta t = \Delta \mu = \) 0.01 days). Figure 26 shows that increasing the density of the population mesh results in a reduction in the difference (error) between the steady state solution and the dynamic steady state solution. It also shows that increasing the mesh density increases the run time of the model exponentially.</P><FIGURE><FIGCAPTURE class="figjnnbl"><B>Figure 26.</B> Effect of mesh density on run time and error reduction. (a) Shows the reduction in error due to the increase of mesh density. (b) Shows the effect of mesh density on the model run time. (c) Shows the error reduction plotted against the model run time. 
                                </FIGCAPTURE></FIGURE><P>The error in the dynamic model with a mesh density of 100 and the steady state model is 0.55%. This is an acceptable degree of error between the two models. The error in the dynamic model is reduced by 0.015% by increasing the population mesh from 100 to 200. The run time of the model is increased from about 2.5 minutes to around 15 minutes when increasing the mesh density from 100 to 200. For these reasons the mesh density was kept at 100 throughout the EPO gene doping tests.</P></DIV><BUTTON class="collapsible cjnnbl">Supplementary Results: Effect of Initial Population Mesh on Dynamic Steady State RBC Population</BUTTON><DIV class="content"><P>The red blood cell feedback loop on EPO release from the kidneys was implemented to determine the dynamic effect of changing EPO concentration. The initial population of the mesh was varied to determine is the starting conditions had an effect on the final steady state. The initial mesh was set to 0, steady state values, a population of 10<SUP>10</SUP> at each maturity point, and a population of 10<SUP>11</SUP>  at each maturity point. Negative populations were not considered.</P><FIGURE><FIGCAPTURE class="figjnnbl"><B>Figure 27:</B> Effect of the initial population in the population mesh on the final steady state of the dynamic model. 
                                </FIGCAPTURE></FIGURE><P>For every initial population mesh, with the population density set to 100, the dynamic model reached a steady state of 24.59 trillion red blood cells. Due to the combination of feedback loops in the model, the red blood cell count will reach the same steady state regardless of the initial population in the mesh. To change the steady state of the dynamic model, the parameters in the feedback functions would need to be modified until a desired steady state red blood cell count is reached. This was not done in our model.</P></DIV></DIV><DIV id="Downloads"><H1 class="jnnbl">4. Download</H1><CENTER><BUTTON class="cjnnbl " style="width:100%"><A href="https://static.igem.org/mediawiki/2018/b/be/T--TUDelft--Gene_Doping_Model_Files.zip"> DOWNLOAD THE MODEL FILES </A></BUTTON></CENTER><DIV id="References"><H1 class="jnnbl">References<OL class="jnnbl" type="1"><LI><A class="jnnbl" href="https://www.sciencedirect.com/science/article/pii/S0065242314000419" target="_blank"> Alegre, E., Sammamed, M., Fernández-Landázuri, S., Zubiri, L., &amp; González, Á. (2015). Circulating Biomarkers in Malignant Melanoma. Advances in Clinical Chemistry, 47-89. doi:10.1016/bs.acc.2014.12.002</A></LI><LI><A class="jnnbl" href="https://www.karger.com/article/FullText/72913" target="_blank"> Ataka, K., Maruyama, H., Neichi, T., Miyazaki, J., &amp; Gejyo, F. (2003). Effects of Erythropoietin-Gene Electrotransfer in Rats with Adenine-Induced Renal Failure. American Journal of Nephrology, 23(5), 315-323. doi:10.1159/000072913</A></LI><LI><A class="jnnbl" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2292496/" target="_blank"> Bergen, J. M., Park, I., Horner, P. J., &amp; Pun, S. H. (2007). Nonviral Approaches for Neuronal Delivery of Nucleic Acids. Pharmaceutical Research, 25(5), 983-998. doi:10.1007/s11095-007-9439-5</A></LI><LI><A class="jnnbl" href="https://www.liebertpub.com/doi/10.1089/10430349950017699" target="_blank"> Chan, Chee Kai, and David A. Jans. “Enhancement of Polylysine-Mediated Transferrinfection by Nuclear Localization Sequences: Polylysine Does Not Function as a Nuclear Localization Sequence.” Human Gene Therapy, vol. 10, no. 10, 1999, pp. 1695–1702., doi:10.1089/10430349950017699.</A></LI><LI><A class="jnnbl" href="https://www.sciencedirect.com/science/article/pii/S0014482796934278?via%3Dihub" target="_blank"> Dean, D. (1997). Import of Plasmid DNA into the Nucleus Is Sequence Specific. Experimental Cell Research, 230(2), 293-302. doi:10.1006/excr.1996.3427</A></LI><LI><A class="jnnbl" href="https://www.ncbi.nlm.nih.gov/books/NBK2267/" target="_blank"> Dean, L. (2005). Blood groups and red cell antigens. Bethesda, MD: NCBI.</A></LI><LI><A class="jnnbl" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3402043/" target="_blank"> Elliott S., Sinclair A.M. (2012). The effect of erythropoietin on normal and neoplastic cells. Biologics. 6: 163–89. doi:10.2147/BTT.S32281.</A></LI><LI><A class="jnnbl" href="https://www.nejm.org/doi/full/10.1056/NEJM198701083160203" target="_blank"> Eschbach J.W., Egrie J.C., Downing M.R., Browne J.K., Adamson J.W. (1987). Correction of the anemia of end-stage renal disease with recombinant human erythropoietin. Results of a combined phase I and II clinical trial. The New England Journal of Medicine. 316 (2): 73–8. doi:10.1056/NEJM198701083160203.</A></LI><LI><A class="jnnbl" href="https://www.sciencedirect.com/science/article/pii/S0085253815601636?via%3Dihub" target="_blank"> Fishbane, S., Galgano, C., Langley, R. C., Canfield, W., &amp; Maesaka, J. K. (1997). Reticulocyte hemoglobin content in the evaluation of iron status of hemodialysis patients. Kidney International, 52(1), 217-222. doi:10.1038/ki.1997.323</A></LI><LI><A class="jnnbl" href="https://link.springer.com/article/10.1007/s00285-012-0530-0" target="_blank"> Fuertinger, D. H., Kappel, F., Thijssen, S., Levin, N. W., &amp; Kotanko, P. (2012). A model of erythropoiesis in adults with sufficient iron availability. Journal of Mathematical Biology, 66(6), 1209-1240. doi:10.1007/s00285-012-0530-0.</A></LI><LI><A class="jnnbl" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3182912/" target="_blank"> Ganesan, L. P., Mohanty, S., Kim, J., Clark, K. R., Robinson, J. M., &amp; Anderson, C. L. (2011). Rapid and Efficient Clearance of Blood-borne Virus by Liver Sinusoidal Endothelium. PLoS Pathogens, 7(9). doi:10.1371/journal.ppat.1002281</A></LI><LI><A class="jnnbl" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2496963/" target="_blank"> Hacein-Bey-Abina, S., Garrigue, A., Wang, G.P., Soulier, J., Lim, A.,Morillon, E., Clappier, E., Caccavelli, L., Delabesse, E., Beldjord, K., <I>et al</I>. (2008). Insertional oncogenesis in 4 patients after retrovirus-mediated gene therapy of SCID-X1. J. Clin. Investig. 118, 3132–3142.</A></LI><LI><A class="jnnbl" href="https://www.elsevier.com/books/guyton-and-hall-textbook-of-medical-physiology/hall/978-0-8089-2400-5" target="_blank"> Hall, John (2011). Guyton and Hall textbook of medical physiology (12th ed.). Philadelphia, Pa.: Saunders/Elsevier. pp. 286–287. ISBN 978-1-4160-4574-8.</A></LI><LI><A class="jnnbl" href="https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0191915" target="_blank"> Haller, N. <I>et al</I>. (2018). Circulating, cell-free DNA as a marker for exercise load in intermittent sports. PLOS ONE. 13. e0191915. 10.1371/journal.pone.0191915.</A></LI><LI><A class="jnnbl" href="https://www.ncbi.nlm.nih.gov/pubmed/7719927" target="_blank"> Huard, J., <I>et al</I>. (1995). The route of administration is a major determinant of the transduction efficiency of rat tissues by adenoviral recombinants. Gene Ther., 2, pp. 107-115.</A></LI><LI><A class="jnnbl" href="https://www.amazon.com/Blood-Textbook-Hematology-James-Jandl/dp/0316457310" target="_blank"> Jandl, J.H. (1987) Blood. Textbook of Hematology. Little, Brown and Company, Boston.</A></LI><LI><A class="jnnbl" href="https://www.sciencedirect.com/science/article/pii/S2090506816300136" target="_blank"> Khanday, M., Rafiq, A., &amp; Nazir, K. (2017). Mathematical models for drug diffusion through the compartments of blood and tissue medium. Alexandria Journal of Medicine, 53(3), 245-249. doi:10.1016/j.ajme.2016.03.005</A></LI><LI><A class="jnnbl" href="https://www.sciencedirect.com/science/article/pii/S0169409X01002022?via%3Dihub" target="_blank"> Kircheis, R., Wightman, L., &amp; Wagner, E. (2001). Design and gene delivery activity of modified polyethylenimines. Advanced Drug Delivery Reviews, 53(3), 341-358. doi:10.1016/s0169-409x(01)00202-2</A></LI><LI><A class="jnnbl" href="https://www.nature.com/articles/3300867" target="_blank"> Lechardeur, D. <I>et al</I>. (1999). Metabolic instability of plasmid DNA in the cytosol: a potential barrier to gene transfer. Gene Ther, 6, pp. 482-497.</A></LI><LI><A class="jnnbl" href="https://www.amazon.com/Williams-Hematology-Eighth-Kenneth-Kaushansky/dp/0071621512" target="_blank"> Lichtman, M.A., Beutler, E., Kipps, T.J., Seligsohn, U., Kaushansky, K., Prchal, J.T. (eds) (2005) Williams hematology, 7th edn. McGraw-Hill, New York.</A></LI><LI><A class="jnnbl" href="https://www.nature.com/articles/3301110" target="_blank"> Li, S., &amp; Huang, L. (2000). Nonviral gene therapy: Promises and challenges. Gene Therapy, 7(1), 31-34. doi:10.1038/sj.gt.3301110</A></LI><LI><A class="jnnbl" href="https://www.ncbi.nlm.nih.gov/books/NBK21475/" target="_blank"> Lodish, H., Berk, A., Zipursky, S.L., <I>et al</I>. (2000). Molecular Biology of the Cell 4th Edition. New York: W. H. Freeman.</A></LI><LI><A class="jnnbl" href="https://www.nejm.org/doi/full/10.1056/NEJMra032207" target="_blank"> McCormack, M.P.; Rabbitts, T.H. (2004). Activation of the T-cell oncogene LMO2 after gene therapy for X-linked severe combined immunodeficiency. N. Engl. J. Med. 350, 913–922.</A></LI><LI><A class="jnnbl" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC38935/" target="_blank"> Moroianu, J., Blobel, G., &amp; Radu, A. (1996). Nuclear protein import: Ran-GTP dissociates the karyopherin alphabeta heterodimer by displacing alpha from an overlapping binding site on beta. Proceedings of the National Academy of Sciences, 93(14), 7059-7062. doi:10.1073/pnas.93.14.7059</A></LI><LI><A class="jnnbl" href="https://www.sciencedirect.com/science/article/pii/S0927776505001323?via%3Dihub" target="_blank"> Munier, S., Messai, I., Delair, T., Verrier, B., &amp; Ataman-Önal, Y. (2005). Cationic PLA nanoparticles for DNA delivery: Comparison of three surface polycations for DNA binding, protection and transfection properties. Colloids and Surfaces B: Biointerfaces, 43(3-4), 163-173. doi:10.1016/j.colsurfb.2005.05.001</A></LI><LI><A class="jnnbl" href="http://www.eurekaselect.com/75646/article" target="_blank"> Murakami, T. <I>et al</I>. (2011). Plasmid DNA Gene Therapy by Electroporation: Principles and Recent Advances. Current Gene Therapy. 11(6):447-56.</A></LI><LI><A class="jnnbl" href="https://link.springer.com/article/10.1007/s10847-007-9303-6" target="_blank"> Nagasaki, T., Shinkai, S. (2007). The concept of molecular machinery is useful for design of stimuli-responsive gene delivery systems in the mammalian cell. J. Incl. Phenom. Macrocycl. Chem., 58, pp. 205-219.</A></LI><LI><A class="jnnbl" href="https://www.nature.com/articles/gt201119" target="_blank"> Ni, W., Guiner, C. L., Gernoux, G., Penaud-Budloo, M., Moullier, P., &amp; Snyder, R. O. (2011). Longevity of rAAV vector and plasmid DNA in blood after intramuscular injection in nonhuman primates: Implications for gene doping. Gene Therapy, 18(7), 709-718. doi:10.1038/gt.2011.19</A></LI><LI><A class="jnnbl" href="https://www.sciencedirect.com/science/article/pii/S0009898110007059" target="_blank"> Owen, W. E., &amp; Roberts, W. L. (2011). Performance characteristics of a new Immulite® 2000 system erythropoietin assay. Clinica Chimica Acta, 412(5-6), 480-482. doi:10.1016/j.cca.2010.11.023</A></LI><LI><A class="jnnbl" href="http://www.bloodjournal.org/content/116/4/625.long?sso-checked=true" target="_blank"> Pillay, J. <I>et al</I>. (2010). In vivo labelling with 2H2O reveals a human neutrophil lifespan of 5.4 days. Blood. 116 (4): 625-7. doi:10.1182/blood-2010-01-259028. PMID 20410504.</A></LI><LI><A class="jnnbl" href="https://www.sciencedirect.com/science/article/pii/S0002934303004479?via%3Dihub" target="_blank"> Ratko,T.A., Cummings,J.P., Blebea,J., Matuszewski, K.A. (2003). Clinical gene therapy for non malignant disease. Am. J. Med., 115, pp. 560-569.</A></LI><LI><A class="jnnbl" href="http://annals.org/aim/article-abstract/714407/neocytolysis-descent-from-altitude-newly-recognized-mechanism-control-red-cell?volume=134&amp;issue=8&amp;page=652" target="_blank"> Rice, L., Ruiz, W., Driscoll, T., Whitley, C. E., Tapia, R., Hachey, D. L., . . . Alfrey, C. P. (2001). Neocytolysis on Descent from Altitude: A Newly Recognized Mechanism for the Control of Red Cell Mass. Annals of Internal Medicine, 134(8), 652. doi:10.7326/0003-4819-134-8-200104170-00010</A></LI><LI><A class="jnnbl" href="http://www.jbc.org/content/273/43/28004.full" target="_blank">Schaffer, D. V., &amp; Lauffenburger, D. A. (1998). Optimization of Cell Surface Binding Enhances Efficiency and Specificity of Molecular Conjugate Gene Delivery. Journal of Biological Chemistry, 273(43), 28004-28009. doi:10.1074/jbc.273.43.28004</A></LI><LI><A class="jnnbl" href="https://www.liebertpub.com/doi/abs/10.1089/hum.1997.8.10-1195?url_ver=Z39.88-2003&amp;rfr_id=ori:rid:crossref.org&amp;rfr_dat=cr_pub%3dpubmed" target="_blank"> Sullivan, D.E., <I>et al</I>.  (1997). Liver-directed gene transfer into non-human primates. Hum. Gene Ther., 8, pp. 1195-1206.</A></LI><LI><A class="jnnbl" href="https://www.sciencedirect.com/science/article/pii/S0002870333901398" target="_blank"> Tarr, L., Oppenheimer, B., &amp; Sager, R. V. (1933). The circulation time in various clinical conditions determined by the use of sodium dehydrocholate. American Heart Journal, 8(6), 766-786. doi:10.1016/s0002-8703(33)90139-8</A></LI><LI><A class="jnnbl" href="http://www.bioscirep.org/content/22/2/283.long" target="_blank"> Templeton, N.S. <I>et al</I>. (2002). Cationic liposome-mediated gene delivery in vivo. Biosci. Rep., 22, pp. 283-295.</A></LI><LI><A class="jnnbl" href="https://link.springer.com/article/10.1007/BF02918494" target="_blank"> Tough, D. F., &amp; Sprent, J. (1995). Lifespan of lymphocytes. Immunologic Research, 14(1), 1-12. doi:10.1007/bf02918494</A></LI><LI><A class="jnnbl" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4408932/" target="_blank"> Vacik, J., Dean, B. S., Zimmer, W. E., &amp; Dean, D. A. (1999). Cell-specific nuclear import of plasmid DNA. Gene Therapy, 6(6), 1006-1014. doi:10.1038/sj.gt.3300924</A></LI><LI><A class="jnnbl" href="https://www.sciencedirect.com/science/article/pii/S1525001601904757?via%3Dihub" target="_blank"> Varga, C.M., <I>et al</I>. (2001). Quantitative Analysis of Synthetic Gene Delivery Vector Design Properties. Cell, Molecular Therapy. Volume 4, Issue 5, November 2001, Pages 438-446. https://doi.org/10.1006/mthe.2001.0475.</A></LI><LI><A class="jnnbl" href="https://www.nature.com/articles/3302495" target="_blank"> Varga, C.M., Tedford, N.C., Thomas, M., Klibanov, A.M., Griffith, L.G. and Lauffenburger, D.A. Quantitative comparison of polyethylenimine formulations and adenoviral vectors in terms of intracellular gene delivery processes. (2005). Gene Ther 12: 1023-1032.</A></LI><LI><A class="jnnbl" href="https://www.sciencedirect.com/science/article/pii/S008525381553771X?via%3Dihub" target="_blank"> Weidemann, A., &amp; Johnson, R. S. (2009). Nonrenal regulation of EPO synthesis. Kidney International, 75(7), 682-688. doi:10.1038/ki.2008.687</A></LI><LI><A class="jnnbl" href="https://www.sciencedirect.com/science/article/pii/0047637475900172" target="_blank"> Weissmanshomer, P. <I>et al</I>. (1975). Chick embryo fibroblasts senescence in vitro: Pattern of cell division and life span as a function of cell density. Mechanisms of Ageing and Development. 4 (2): 159-166. doi:10.1016/0047-6374(75)90017-2. PMID 1152547.</A></LI><LI><A class="jnnbl" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4397984/" target="_blank"> Wilson, G. L., Dean, B. S., Wang, G., &amp; Dean, D. A. (1999). Nuclear Import of Plasmid DNA in Digitonin-permeabilized Cells Requires Both Cytoplasmic Factors and Specific DNA Sequences. Journal of Biological Chemistry, 274(31), 22025-22032. doi:10.1074/jbc.274.31.22025</A></LI></OL></H1></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>