<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_KAIST-Korea_Projects_Modeling_Section1"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:KAIST-Korea/Projects/Modeling/Section1">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:KAIST-Korea/Projects/Modeling/Section1&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:KAIST-Korea/Projects/Modeling/Section1&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:KAIST-Korea/Projects/Modeling/Section1&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:KAIST-Korea/Projects/Modeling/Section1" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:KAIST-Korea/Projects/Modeling/Section1</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><CENTER><DIV id="ModelingMenuBar"><TABLE style="width:100%;" cellspacing="0" border="0" cellpadding="0"><TBODY><TR><TD><A href="/Team:KAIST-Korea/Projects/Modeling2" title="Team:KAIST-Korea/Projects/Modeling2">Overview</A></TD><TD><STRONG class="selflink">Section1</STRONG></TD><TD><A href="/Team:KAIST-Korea/Projects/Modeling/Section2" title="Team:KAIST-Korea/Projects/Modeling/Section2">Section2</A></TD><TD><A href="/Team:KAIST-Korea/Projects/Modeling/Section3" title="Team:KAIST-Korea/Projects/Modeling/Section3">Section3</A></TD><TD><A href="/Team:KAIST-Korea/Projects/Modeling/Section4" title="Team:KAIST-Korea/Projects/Modeling/Section4">Section4</A></TD><TD><A href="/Team:KAIST-Korea/Projects/Modeling/Section5" title="Team:KAIST-Korea/Projects/Modeling/Section5">Section5</A></TD><TD><A href="/Team:KAIST-Korea/Projects/Modeling/Section6" title="Team:KAIST-Korea/Projects/Modeling/Section6">Gallery</A></TD></TR></TBODY></TABLE></DIV></CENTER><DIV id="kaistcontent"><H2><SPAN class="mw-headline" id="Quorum_Production_by_the_Brush_E._coli"><B>Quorum Production by the Brush E. coli</B></SPAN></H2><P>Mathematical modeling is essential in qualitatively describing de novo genetic circuits that frequently arise in synthetic biology. We can use such models for two objectives: (1) predicting the behavior of combinations of BioBrick parts designed for the synthetic circuit that performs some task, and (2) choosing the appropriate promoter and ribosome binding site (RBS) with suitable strengths for the circuit. Also, it will serve as a reference for others who use the BioBrick in the future. In summary, the model and the computer simulation are our beginning point for making testable predictions about the behavior of our system. We construct a computational model describing the genetic network encompassing relevant signal transduction pathways in order to help build E.coli that can draw pictures!
</P><H3><SPAN class="mw-headline" id="Modeling_E.coli_Type_I._.28Brush_E.coli.29">Modeling E.coli Type I. (Brush E.coli)</SPAN></H3><H4><SPAN class="mw-headline" id="Modeling_Approach"><B>Modeling Approach</B></SPAN></H4><DL><DD>There are several known Quorum sensing (QS) networks. All known QS networks operate as an “on-off” gene expression switch by controlling the level of a certain transcription factor whose expression is suppressed in the “off” state and is strongly induced in the “on” state.<SUP>[2]</SUP> Usually, the intracellular network that is controlled by the quorum sensing remains in the “off” state until the quorum reaches a certain concentration. After quorum reaches the threshold concentration, the genetic circuit changes its state into “on” state and activates the expression of the relevant genes. In this model, we hypothesized that the typical E.coli cell volume is ~7.0×10<SUP>-16</SUP>L and cells are freely permeable to quorums. We used a standard chemical kinetic approach based on the mass-action rate law. The kinetic parameters used in our model are based on the published data. The rate constants were taken from several papers on the mathematical modeling of quorum sensing pathways.
</DD></DL><H4><SPAN class="mw-headline" id="Model"><B>Model</B></SPAN></H4><DL><DD>Before moving on to our model system, let us review how to model a general case of protein production from a single gene.
</DD></DL><H5><SPAN class="mw-headline" id="Protein_Production_of_a_Single_Gene_Modeling"><B>Protein Production of a Single Gene Modeling</B></SPAN></H5><DL><DD>The actual protein production from a single gene is composed of complex processes. However, in this model, protein production is simplified into two processes: transcription and translation. 
</DD></DL><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:725px;"><DIV class="thumbcaption">Fig 1.The schematic diagram of protein production modeling The model can be represented by these chemical reactions with corresponding ordinary differential equations (ODEs):</DIV></DIV></DIV></DIV><DL><DD>where p<SUB>m</SUB> is the mRNA production rate constant, and p<SUB>p</SUB> is the protein production rate constant. Although the choice of parameters depends on many factors such as the gene of interest and the internal and external environment of gene expression, the commonly accepted estimation of parameters is sufficient for our gene expression model. Therefore, we choose the average transcription/translation rate constants which are taken from published data. <SUP>[1]</SUP></DD></DL><DL><DD>The degradation rate of the mRNA and protein can be calculated from
</DD></DL><DL><DD>where t<SUB>1/2</SUB>is the degradation half-life in minutes. From this equation, we can calculate k<SUB>md</SUB>(mRNA degradation rate), k<SUB>pd</SUB>(protein degradation rate). The values are in the constants table. [3. Constants Table of this page] Based on these facts and the law of mass action, we can write these equations:
</DD></DL><DL><DD>The above models an average protein production from a single gene. However, in synthetic biology, we can control the transcription rate and the translation rate by appropriately changing the promoter and RBS parts. Therefore, we can represent the rate constants differently:
</DD></DL><DL><DD>where χ<SUB>p</SUB> is the strength ratio of the promoter relative to the average rate of transcription. Also, χ<SUB>R</SUB> is the strength ratio of RBS relative to the average rate of translation. It is determined by the promoter (or RBS) BioBrick part that we use.
</DD></DL><DL><DD>We will refer to the aforementioned protein production rate of a single gene model to build our own models. If specific values for the production and degradation rates exist for a certain protein, we will use those values. Otherwise, we will use a general, averaged value like above model. Let’s jump right into the details of the model. 
</DD></DL><H5><SPAN class="mw-headline" id="Modeling_of_Brush_.28Type_II.29_E.coli"><B>Modeling of Brush (Type II) E.coli</B></SPAN></H5><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:802px;"><DIV class="thumbcaption">Fig 2. The schematic diagram of Brush E.coli modeling.</DIV></DIV></DIV></DIV><DL><DD>The above schematic diagram represents the genetic circuit implmented in the Brush E.coli after it has been genetically recombined by the Cre-Lox system[HYPERLINK TO THE WIKIPEDIA]. The Cre-Lox recombination system is commonly used to knockout a specific gene conditionally, in which a gene is only knocked out at a specific time. By using this Cre-Lox recombination system, one of the four quorum sensing systems is chosen and installed in Brush E.coli (In this model, the Lux based quorum sensing system is installed). In our model, we will only discuss the system embedded with the Lux system and assume that the model is applicable to the other three installations because they operate with an almost identical mechanism and have similar reaction constants. Now, we briefly explain each component of the model in the following paragraphs.
</DD></DL><OL><LI>DNA1 translates LacI constituitively. As a derivative of the lac operon, LacI binds to the gene lacO to suppress the expression of the genes downstream relative to lacO. Although four LacI must bind to lacO for this to occur, we assume one is sufficient so that we can simplify the analysis.
</LI></OL><OL><LI>DNA2 produces LuxI, the component of the Lux based quorum sensing. LacI suppresses expression of DNA2 by binding to the LacO which is upstream to the luxI gene. As a result, DNA2 is not translated until we add IPTG.
</LI></OL><OL><LI>When we add IPTG, it binds with LacI. IPTG acts as a competitive inhibitor to the binding of LacI to lacO; IPTG prevents LacI from binding to DNA2, clearing way for the RNA Polymerase to begin transcription. That is, IPTG induces the translation of luxI in DNA2; it is an inducing signal that turns the switch on.<SUP>[2]</SUP> In this model, we assumed that IPTG is applied in the form of a step function.
</LI></OL><DL><DD>In order to describe the behaviour of the circuit with a deterministic model, we used the mass action rate laws in part A to represent changes of each quantity with respect to time in a set of ordinary differential equations. By solving these equations, we can obtain the number of each variable at a specific time. (Note, we used the Michaelis-Menten equation to account for the affinity difference of LacI for both for LacO and IPTG. Detailed explanation will follow.)
</DD></DL><CENTER>* Refer to the constants table for the constants</CENTER><DL><DD>The brackets represent the number of molecules of the species enclosed between them. For instance, [LacI]<SUB>T</SUB> indicates the number of LacI in the cell, [LacI]<SUB>Free</SUB> the number of LacI molecules not bound to an IPTG. The equation for [LacI]<SUB>Free</SUB> was derived from the Michaelis-Menten equation. For the derivation, refer to page 241-245 of the reference <SUP>[11]</SUP>.
</DD></DL><DL><DD>Because of the proportion of LacI molecules that form a complex with IPTG, only [LacI]<SUB>Free</SUB> LacI molecules can bind to DNA2. When there are the same number of repressor molecules, the promoter activity is given by the following equation, derived from the equation for the chemical equilibrium.
</DD></DL><DL><DD>where k5 is the transcription rate of DNA2 (refer to page 248-250 of the reference <SUP>[11]</SUP> for details). Using this equation, we can obtain the rate equations for mRNA2 and LuxI.
</DD></DL><DL><DD>By solving these ordinary equations, we can model the response of the genetically engineered E.coli to IPTG and also obtain the number of LuxI in the cell. Using the latter we now find out the number of quorum molecules produced by Brush E.coli per unit time, as well as the amount diffused to nearby cells.
</DD></DL><H5><SPAN class="mw-headline" id="Modeling_of_quorum_production.2Fdiffusion_rate"><B>Modeling of quorum production/diffusion rate</B></SPAN></H5><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:802px;"><DIV class="thumbcaption">Fig 3. The schematic diagram of quorum production/diffusion rate modeling</DIV></DIV></DIV></DIV><DL><DD>By the Michaelis-Menten equation, the enzymatic production rate is
</DD></DL><DL><DD>The reaction rate increases with increasing substrate concentration <SUP>[S]</SUP>, and asymptotically approaches its maximum rate V<SUB>max</SUB>, attained when all enzymes are bound to the substrate. In this model, we assumed that the substrate (Butyl-ACP) is in excess so that all enzymes are bound to a substrate. Then, we can write V<SUB>max</SUB> = k<SUB>cat</SUB>[E]<SUB>0</SUB>, where k<SUB>cat</SUB> is the rate constant of the enzyme reaction and [E]<SUB>0</SUB> the enzyme concentration. Therefore, the rate of quorum production is equal to k<SUB>14</SUB>×[LuxI]. Additionally, it is important to consider the amount of quorum that passively diffuses out of the cell. The diffusion rate depends on the rate constant and the concentration gradient of quorum across the cell membrane. Since quorum continuously diffuses away from the cell, the extracellular quorum concentration is lower than the intercellular concentration. Assuming that the diffusion rate only depends on the intracellular concentration, the rate equation can be written as
</DD></DL><DL><DD>By the Michaelis-Menten equation, The enzymatic production rate is
</DD></DL><H3><SPAN class="mw-headline" id="Results_.26_Conclusion">Results &amp; Conclusion</SPAN></H3><DL><DD>The set of ODEs in model was solved with MATLAB.[THE SOURCE CODE] The results below have been rounded to integers using the floor function. This is because molecules only exist in whole numbers. We simulated the behavior of Brush E.coli for two cases. In the first case, IPTG was applied at t=400 min with the concentration of 100,000 molecules per cell which is about 100 uM. In the second case, IPTG was applied at t=400 min with the concentration of 1,000,000 molecules per cell which is about 1 mM. Figures 4 (a) and (c) are the results for the first case, and figures 4 (b) and (d) are the results for the second case. For the detailed description of each result, refer to the caption of the figures.
</DD></DL><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:802px;"><DIV class="thumbcaption">Fig 4. The number of gene expression products of Brush E.coli. The IPTG induction occurs at t=400 min for all results. For figures (a) and (c) the concentration of IPTG is 100,000 molecules/cell and for figures (b) and (d) the concentration is 1,000,000 molecules/cell. For the figures (c) and (d), the result indicates the rate of change of quorum both inside and outside the cell.</DIV></DIV></DIV></DIV><DL><DD>From the results, we can see that [LacI]<SUB>Free</SUB> diminishes at the moment IPTG is induced, initiating the translation of DNA2. This induces the production of LuxI proteins, which reaches its saturation point about 200 min after induction. The saturation concentration of LuxI protein was 15 molecules per cell for the first case and 150 molecules per cell for the second case.
</DD><DD>Using the saturation concentrations of LuxI, we calculated the production/diffusion rate of the quorum. In the first case, approximately 54 quorum molecules were produced per minute and diffused towards neighboring cells (The value of d[Quorum]out/dt indicates this rate). In the second case, around 516 molecules of quorum are produced and dispersed per minute.
</DD><DD>In conclusion, we were able to achieve our objective by finding the rate at which Brush E.coli produce the quorum. The rate of quorum production depends on the initial amount of IPTG applied. Application of 100 uM of IPTG results in the production of 54 quorum molecules per minute, which would diffuse into Dyestuff E.coli to activate its genetic circuit. In the next modeling, we will discuss about the diffusion of these quorum molecules. [HYPERLINK TO THE NEXT MODELING PAGE] 
</DD></DL><H3><SPAN class="mw-headline" id="Constants_Table">Constants Table</SPAN></H3><TABLE border="1" cellpadding="5" cellspacing="0" style="margin: 1em auto 1em auto;text-align: center;"><TBODY><TR><TH style="width: 100px;" scope="col"> Constant
</TH><TH style="width: 300px;" scope="col"> Legend
</TH><TH style="width: 100px;" scope="col"> Value
</TH><TH style="width: 100px;" scope="col"> References
</TH></TR><TR><TD> p<SUB>m</SUB></TD><TD> Production rate of mRNA (Average)
</TD><TD> 2/min
</TD><TD rowspan="9"> [1]
</TD></TR><TR><TD> p<SUB>p</SUB></TD><TD> Production rate of protein (Average)
</TD><TD> 0.1/min
</TD></TR><TR><TD> k<SUB>md</SUB></TD><TD> mRNA degradation rate (Average)
</TD><TD> 0.139/min
</TD></TR><TR><TD> k<SUB>pd</SUB></TD><TD> mRNA degradation rate (Average)
</TD><TD> 0.0116/min
</TD></TR><TR><TD> k<SUB>1</SUB></TD><TD> Transcriptin of lacI gene
</TD><TD> p<SUB>m</SUB>χ<SUB>1</SUB></TD></TR><TR><TD> k<SUB>2</SUB></TD><TD> mRNA of lacI degradation
</TD><TD> k<SUB>md</SUB> = 0.139/min
</TD></TR><TR><TD> k<SUB>3</SUB></TD><TD> Translation of lacI mRNA
</TD><TD> p<SUB>p</SUB>χ<SUB>2</SUB></TD></TR><TR><TD> k<SUB>4</SUB></TD><TD> Protein degradation of LacI
</TD><TD> k<SUB>pd</SUB> = 0.0116/min
</TD></TR><TR><TD> k<SUB>5</SUB></TD><TD> Transcription of luxI gene
</TD><TD> p<SUB>m</SUB>χ<SUB>3</SUB></TD></TR><TR><TD> k<SUB>6</SUB></TD><TD> mRNA of luxI degradation
</TD><TD> 0.36/min
</TD><TD> [3],[4]
</TD></TR><TR><TD> k<SUB>7</SUB></TD><TD> Translation of luxI mRNA
</TD><TD> p<SUB>p</SUB>χ<SUB>4</SUB></TD></TR><TR><TD> k<SUB>8</SUB></TD><TD> Protein degradation of LuxI
</TD><TD> 6*10<SUP>-3</SUP>/min
</TD><TD> [3], [5]
</TD></TR><TR><TD> K<SUB>x</SUB></TD><TD> IPTG - LacI4 binding dissociation constant
</TD><TD> 548 molecules
</TD><TD> [6]
</TD></TR><TR><TD> K<SUB>d</SUB></TD><TD> LacI - DNA (LacO site) binding dissociation constant
</TD><TD> 4.22*10<SUP>-4</SUP> molecules
</TD><TD> [7]
</TD></TR><TR><TD> k<SUB>9</SUB></TD><TD> Enzymatic production rate of autoinducer catalyzed by LuxI
</TD><TD> 3.6/min
</TD><TD> [3], [8]
</TD></TR><TR><TD> k<SUB>10</SUB></TD><TD> Diffusion rate of the autoinducer (Quorum) across the cell
</TD><TD> 1.07*10<SUP>-2</SUP>/min
</TD><TD> [9]
</TD></TR><TR><TD>χ<SUB>1</SUB></TD><TD> The strength of DNA1 promoter relative to average transcription rate of DNA.
</TD><TD> 0.2
</TD><TD> Controllable
</TD></TR><TR><TD>χ<SUB>2</SUB></TD><TD> The strength of mRNA1 RBS relative to average translation of mRNA.
</TD><TD> 0.5
</TD><TD> Controllable
</TD></TR><TR><TD>χ<SUB>3</SUB></TD><TD> The strength of DNA2 promoter relative to average transcription rate of DNA.
</TD><TD> 20
</TD><TD> Controllable
</TD></TR><TR><TD>χ<SUB>4</SUB></TD><TD> The strength of mRNA2 RBS relative to average translation rate of mRNA.
</TD><TD> 1
</TD><TD> Controllable
</TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="References">References</SPAN></H3><DL><DD>[1]. Tae Jun Lee, Dennis Tu, Chee Meng Tan, and Lingchong You, Systems Bioinformatics: An Engineering Case-Based Approach, Artech House, Chapter 6, 159-178 (2007)
</DD><DD>[2]. Kavita lyer Ramalingam, Jonathan T.Tomshine, Jennifer A. Maynard, Yiannis N. Kaznessis, Forward engineering of synthetic bio-logical And gates, Biochemical Engineering Journal, 47, 38-47 (2009)
</DD><DD>[3]. A.B. Goryachev, D.J. Toh, T. Lee, Systems analysis of a quorum sensing network: Design constraints imposed by the functional requirements, network topology and kinetic constants, Bio Systems (2005)
</DD><DD>[4]. G. Hambraeus, C. von Wachenfeldt, L. Hederstedt, Genome-wide survey of mRNA half-lives in Bacillus subtilis identifies extremely stable mRNAs, Genomics, 269, 706-714 (2003)
</DD><DD>[5]. Julie M. Pratt, June Petty, Isabel Riba-Garcia, Duncan H. L. Robertson, Simon J. Gaskell, Stephen G. Oliver, and Robert J. Beynon, Dynamics of protein turnover, a missing dimension in proteomics, Molecular &amp; Cellular Proteomics, 1, 579-591 (2002)
</DD><DD>[6]. Y. Setty, A. E. Mayo, M. G. Surette, and U. Alon, Detailed map of a cis-regulatory input function, PNAS, 100, 7702-7707 (2003)
</DD><DD>[7]. Joakim Lundeberg, Johan wahlberg, and Mathias Uhlen, Affinity Purification of Specific DNA Fragments Using a lac Repressor Fusion Protein, Gene Analysis Techniques, 7, 47-52 (1990)
</DD><DD>[8]. Matthew R. Parsek, Dale L. Val, Brian L. Hanzelka, John E. Cronan Jr., and E. P. Greenberg, Acyl homoserine-lactone quorum-sensing signal generation, Proc. NatI. Acad. Sci. USA (PNAS), 96, 4360-4365 (1999)
</DD><DD>[9]. Lin gchong You, Robert Sidney Cox Ill, Ron Weiss, Frances H. Arnold, Programmed population control by cell-cell communication and regulated killing, letters to nature (2004)
</DD><DD>[10]. Darren J. Wilkinson, Stochastic Modeling for System Biology, Chapman &amp; Hall/CRC, Chapter 1 and 6, 1-18 and 139-161 (2006)
</DD><DD>[11]. Uri Alon, An Introduction to Systems Biology: Design Principles of Biological Circuits, Chapman &amp; Hall/CRC, Chapter 2 and Appendix A, 5-25 and 241-252 (2007) 
</DD></DL></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:KAIST-Korea/Projects/Modeling/Section1">http://2011.igem.org/Team:KAIST-Korea/Projects/Modeling/Section1</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:KAIST-Korea/Projects/Modeling/Section1" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:KAIST-Korea/Projects/Modeling/Section1" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:KAIST-Korea/Projects/Modeling/Section1&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:KAIST-Korea/Projects/Modeling/Section1&amp;oldid=73969" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>