"
Page
Discussion
View source
History
teams
Log in
Â 
Team:uOttawa/team
From 2014.igem.org
iGEMuOttawa
project dry lab policy team
uOttawa iGEM
Meet the team Safety Notebook Collaborations Attributions
Meet our awesome lab members and the individual sub-teams that make up uOttawa iGEM 2014.
See uOttawa's safety data and safety forms.
Follow our team through every phase of the project, from brainstorming to testing.
Read about our Waterloo and Queens collaborations.
We list the key people that made uOttawa iGEM 2014 a success.
Meet the team
TOP: Tommy Xu, Brent Weatherall, Danny Salem, Matt Chandrawan, Yara Abou-Hamde, Abdus Anwar, Katie Harriman
MIDDLE: Lloyd Mai, Kaitlin Kharas, Nicholas Huang, Sarah Mohand-Said, Linda Dam, Joanne Joseph, Alex Huluta, Peter Doan
BOTTOM: Huy Tran, Dylan Siriwardena, Jenna Khawas, Cory Lefebvre, Martin Hanzel
Wet lab
Dylan Siriwardena, Alex Tzahristos, Shihab Sawar, Sarah Mohand-Said, Martin Hanzel
Dry lab
Lloyd Mai, Peter Doan, Matt Chandrawan
Modelling
Cory Lefebvre, Alex Huluta, Joey Irani, Danny Salem
Policy and Practices
TOP: Abdus Anwar, Nick Huang, Alex Huluta, Katie Harriman
BOTTOM: Joanne Joseph, Yara Abou-Hamde, Jenna Khawas, Kaitlin Kharas, Linda Dam
Graphic Design
Cory Lefebvre, Sarah Mohand-Said, Jenna Khawas, Huy Tran
Shihab
I'm bringing sexy back
Safety
Would any of your project ideas raise safety issues?
Researcher safety: Our lab is a relatively safe one, as the only organisms we use are E. coli and S. cerevisiae, which are considered level 1 Biosafety Containment Agents so are not pathogenic. We do handle Bunsen burners, chloroform and phenol. Therefore, the largest risks that are presented within the lab include potential fires or exposure to fumes. However, the chloroform and phenol are only ever handled in a fume hood so the risk of exposure to fumes is minimized. In addition, all members must pass the university courses for WHMIS, general laboratory safety, biosafety and autoclave handling prior to entering in the lab.
Public safety: The strains that we use in the lab are non-pathogenic. Although we do genetically modify the organisms that we use, they never could possibly make it into the external environment due to the fact that we use decontamination techniques such as autoclaving and bleaching. The organisms that we modify may contain genes that are less advantageous than those seen in wild type strains, as we occasionally knock out genes that have an impact on the metabolism of the organism, such as genes that pertain to adenine metabolism. There are no risks to the biosecurity of different groups of people. The modifications that we make to the organisms that we use in the lab could not possibly cause the organisms to become pathogenic.
Environmental safety: The genes that we modify in the organisms that we use typically impact their metabolic processes such that they are not as "fit" as the same strains in the external environment. The wastes that we dispose are autoclaved first before they are disposed of. The phenol and chloroform waste that our lab produces is shipped to another facility that is designed to handle chemical waste. Overall, the amounts of phenol and chloroform that are used within the lab are so small that there should not be any environmental repercussions if they were to escape from the lab without being sent to a chemical handling facility first.
In order to reduce these risks, we only work with organisms that are of a level one biosafety containment, meaning that they are non-pathogenic. In order to fully assure that the organisms that we do use within the lab do not make their way out of it, we decontaminate any live samples that are used once we are finished with them. This also includes antibiotics, which we sterilize in order denature them so that they do not have any harmful effects on the environment. We do so through the use of sterilization techniques such as autoclaving. Our lab in general is very secure (ie. An access card is required to get into the lab, any side-entrances into the building are locked after hours, and there are security guards stationed within the building,) so there is a very low chance that any malicious persons will obtain access to the genetically modified organisms that are present within our lab.
View the About our Lab and Safety sheets for uOttawa iGEM 2014.
Is there a local biosafety group, committee, or review board at your institution?
The University of Ottawa has a Biosafety Committee that works with the Office of Risk Management and the Vice President of Research to ensure that all biosafety regulations are met. A lengthy description of their role can be found here .
Our lab falls under BSL-1, meaning that no special design and practice features are necessary as we are not working with organisms which are dangerous if ingested or airborne. The sections of the guide dealing with recombinant DNA and genetic manipulation state that most recombinant DNA work is safe, but containment level and pathogenicity should be considered. Containment level is addressed above, and none of the strains we worked with was pathogenic. Therefore, we did not need to discuss our project with the biosafety committee.
All lab members are required to take both a biosafety course and a lab safety course prior to getting their access card for the lab. Each course consists of a full day of lecture outlining risks and necessary precautionary measures followed by a take home exam that is marked by the lecturer. Upon passing the exam, lab members received certificates to verify that they had passed and could work in the lab.
In addition, the Government of Canada has issued a lengthy set of guidelines pertaining to biosafety.
Notebook
January
Recruitment begins for the uOttawa 2014 iGEM team.
March
March 2nd to March 8th Leaders for each of the sub-teams were selected. Project brainstorming began, and will continue until May.
March 26th to April 1st: Research into cancer-fighting project based on MMPs, and cell memory are pursued.
April
April 11th to April 18th: Research into tri-stable switches and cellular decision making began after a conversation with Dr. Mads Kaern. Other ideas turned out to be non-viable within the four-month research window.
April 19th to April 26th:Initial designs for the tri-stable network proposed.
April 27th to May 3rd:Lab prepared by Dylan Siriwardena for incoming new students. Reagents were prepared, and necessary strains streaked out from cold stock.
May
May 4th to May 10th: Training has begun! Upon finishing the winter semester, the wet lab team began its training in the lab. The new members of the lab who did not have lab experience learned various techniques that would be used over the course of the entire summer, such as PCR, genomic extractions, and transforming constructs into yeast cells. Theory was also taught this week.
May 11th to May 17th: Successful PCRs were achieved by all members. The wet lab team moved onto creating dimers and multimers of PCR amplicons.
May 18th to May 24th: Training construction complete, transformations and confirmations were conducted for the rest of the month. Confirmation includes both PCR and flow cytometry.
May 25th to May 31st: Training completed. Research and understanding of the tri-stable switch was focused on this week, to ensure all wet lab members had a solid understanding. Additional meetings with Dr. Kaern as well as a the construction of a research proposal has purified the many ideas into two solid designs.
June
June 1st to June 7th: Research into pigments was completed, to be used as a reporter and a possible paint application. Primers were designed and ordered to construct the various promoters needed in both designs. Much of this month was dedicated to running PCR after PCR in order to obtain a high concentration and quantity of the fragments that make up the final construct that we were to create. We even had an all-nighter during the month in which the entire night was dedicated to running PCRs!
June 8th to June 14th: Primers arrived, construction of test constructs with each promoter is started. Since many of the sections of the promoters are common, pGalTx was first obtained, and then modified to get the variety of activating and repressing regions. Lex constitutive promoters were also designed.
June 15th to June 21st: It was found that the constitutive promoter we were using to express our repressor was too weak (mrp7). In order to remedy that, we are creating a new strain with a strain with a much stronger promoter (pADH1) in the hope of visualizing strong repression by hindrance.
June 22nd to June 28th: Success! Repression by hindrance was shown to strongly repress in pGalTx, with pADH1 driving rTTA. Other test constructs coming along, with the first few promoters constructs being completed, and the next round being started.
June 29th to July 5th: Second round of promoters are completed and are being confirmed. Promoters with varying numbers of sites were now being constructed in order to have promoters with sufficient expression. It was found that modifying so close the TATA box significantly reduced promoter activity.
July
July 6th to July 12th: More transformations of promoter constructs that we had created into different strains of yeast that were previously created in the lab. In doing so, we could appreciate the effects that the already modified genes in the various strains of yeast would have on the regulation of the promoters. Within our spare time of course, we amplified more of the same monomers that we had used to create the tested constructs, so that if worse came to worst we could easily re-create the construct again or modify it in numerous ways.
July 13th to July 19th: Many constructs confirmed from previous transformations, full gradients were conducted in order to characterise the first round of promoters created.
July 20th to July 26th: Final round of promoters were completed and transformed. Testing of colonies continues.
July 27th to August 2nd: We had achieved the final promoter test strains that we had sought after for the entire summer thus far. Of course we knew it would probably need to be modified and possibly be re-created if the testing of it proved it to be feeble. Thus, upon transforming it into different strains of yeast we tested many colonies and used many different drug concentrations over the course of our analysis of the final construct. We created entire new constructs just in case anything were to fail.
August
August 3rd to August 9th: Problem with our phusion was found, accounting for failed PCRs for the past week. On the bright side PCR talent has increased due to the many failures and redos.
August 10th to August 16th: Final confirmations of all the promoter constructs with both PCR and flow cytometry. All promoters now fully characterised. Beginning of final construction of tri-stable switch.
August 17th to August 23th: PCRs and preliminary transformations were preformed for the two designs for the tri-stable switch.
August 24th to August 30th: Constructs were tested and confirmed. Only a few strains appeared to confirm, but another round of transformation was required to be sure.
August 31st to September 6th: With school starting up again, we each spent a significantly less amount of time in the lab, and most of the time that we spent in the lab was concentrated on testing out the constructs that we had obtained during August. Much larger drug concentrations were performed within our flow sessions, so that we could present the modelling team with sufficient data for the figures that they were to create. September overall was largely a month of transformations, testing failures, reconstructions, transformations, and then more disappointments.
September
September 7th to September 13th: All previous constructs failed to flow confirm, have to troubleshoot which parts are non-functional. Entirely new constructs which contained new monomers, and were entirely different from the constructs that we had worked with previously. We also spent some time frustratingly creating the BioBricks that were to be submitted to the iGEM HQ with long-expired endonucleases.
September 14th to September 20th: Transformations were tested, and GFP constructs confirmed. Sadly anything BFP-related along with rTTA did not. BFP will be removed and replaced, as well as the rTTA, meaning more construction.
September 21th to September 27th: Transformations failed again, rTTA will not PCR out properly, and we believe the promoter driving BFP maybe the problem.
September 28th to October 4th: Crunch time for the wet lab. Although we did have enough data to at least present the intricacies of the tri-stable switch, we really wanted to establish one ourselves. The problem is, with all of the time spent in testing colonies, we had little time left to start over again if such a thing was needed. With testing of the construct failing, we decided to shift our primary focus to gathering together all of the data that we did manage to retrieve over the course of the summer so that our wiki could be well completed.
October
October 5th to October 11th: Transformations all failed to confirm last week, last chance to build the final construct with the time left. Ran the all-nighter to end all all-nighters. Biobricks purified and sent in this week on Tuesday.
October 12th to October 18th: Wiki freeze is approaching! Last data on promoters is collected and write ups for the wiki are completed.
Attributions and Collaborations
uOttawa
The wet lab undergraduate team consists of Dylan Siriwardena, Shihab Sarwar, Alexandra Tzahristos, Sarah Mohand-Said and Martin Hanzel. All members helped construct all biobricks, promoter test constructs, and the final network. Flow cytometry was conducted by all members, with a majority done by Shihab Sarwar and Dylan Siriwardena. Martin Hanzel constructed and tested all E. coli strains used for the Measurement interlab-study. Dylan Siriwardena completed the network designs for the tri-stable switch, with the significant assistance of team advisor Ian Roney and supervisor Dr. Mads Kaern.
The many test strains created for each novel promoter were based upon Ian41, a strain created by Ian Roney. Ian Roney also provided significant assistance with training, day-to-day troubleshooting, and technical advice on many protocols. Our lab technician, Mila Tepliakova, assisted in training, gave indispensible advice on improving our methods, and provided stock solutions throughout the year. Finally, Dr. Mads Kaern provided the team with resources, including reagents, machines, and access to a flow cytometer, as well as providing extensive advice and support.
In fact, the entire Kaern lab provided support for the undergraduate wet lab, including Ian Roney, Afnan Azizi, Hilary Pheonix, Nada Elnour, and Vaibhav Gupta.
Dry lab:The entirety of the web application "Brick Builder" has been created from scratch by Mohammed Chamma, Kevin Rutkay, Peter Doan, Matthew Chandrawan, and Khama Hastick. They are the ones who have enabled the completion of Brick Builder. Mohammed set up the basics of the web app and helped bring iGEM's search tools into Brick Builder. Kevin set the basics of presenting search results and saved parts in the "Brick Bin" in a table. Peter translated RFC methods into computer programming codes and created a GUI for the construction of parts. Matthew helped reorganise the tables and paginated them so as to easily sort and search for parts. Khama created a tool tip for ease of understanding the functions of Brick Builder.
The members of the modeling team Danny Salem, Joey Irani, Alex Huluta and Cory Lefebvre designed, implemented and analyzed the models of our system. Cory Lefebvre was responsible for the design of the deterministic ODE models with the help of Sam Hirniak and John Drake from the Waterloo iGEM team, and Danny Salem was responsible for the design of the stochastic model. Joey Irani performed the model fitting of promoter data from the wet lab and parameterization.  Joey Irani, Alex Huluta and Cory Lefebvre performed stability and sensitivity analyses of the deterministic models and Danny Salem did the same for the stochastic models.
Policy and Practices: Members who were involved in the Letâs Talk Science collaboration in various capacities include: Abdus Anwar, Jenna Khawas, Nicholas Huang, Yara Abou-Hamde, Kaitlin Kharas, Peter Doan and Curtis Quan. All in-class presentations were done by Abdus Anwar. Members of the team who led activities at the various fairs and festivals (as well as at the Childrenâs Hospital of Eastern Ontario) include: Yara Abou-Hamde, Nicholas Huang, Joanne Joseph, Kaitlin Kharas, Abdus Anwar, Irene Harmsen, Cory Lefebvre, Alexandra Huluta, Peter Doan, Curtis Quan and Daniel Tesolin. Members of the team who prepared activities and/or presented as part of the Enrichment Mini-Course Program include: Yara Abou-Hamde, Joanne Joseph, Marina Kidisyuk, Joey Irani, Sarah Mohand-Said and Nicholas Huang. The research for the synthetic biology course was done by: Yara Abou-Hamde, Irene Harmsen, Kaitlin Kharas, Joanne Joseph and Lloyd Mai. Survey was designed and data analyzed by Yara Abou-Hamde. Abdus Anwar represented the policy team at the oGEM conference. The patent policy project is being led by Katie Harriman and Yara Abou-Hamde. Other significant contributions were made by Caitlin Johnston, Fiatsogbe Dzuali, Stephanie Gran-Ruaz and Linda Dam.
Collaborations
Waterloo collaboration
This year, we had the opportunity to collaborate with the iGEM team from the University of Waterloo. We were asked to aid in the  construction of three large DNA cassettes into one plasmid.
In order to do so, the uOttawa iGEM team designed a set of primers (6 total) to amplify out each of the DNA cassettes, with overhangs to be used for our labs version of overlap extension PCR. A protocol was written for the uWaterloo team. We also were able to discuss and advise its design and construction.
In return, Sam Hirniak and John Drake from the uWaterloo team helped us design ODE deterministic models for our inducible switch.
Queens collaboration
For the Queens iGEM team, we sent the fusion inducer GEV (gal4 binding domain, human estrogen receptor, and VP16 transactivator) to aid in their design, accompanied with detailed sequence data. We also provided some advice on its use.
Retrieved from " http://2014.igem.org/Team:uOttawa/team "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
