<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_uOttawa_drylab"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:uOttawa/drylab">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:uOttawa/drylab&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:uOttawa/drylab&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:uOttawa/drylab&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:uOttawa/drylab" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:uOttawa/drylab</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><DIV class="row"><A href="https://2014.igem.org/Team:uOttawa" class="title"><SPAN class="igem">iGEM</SPAN>uOttawa</A><NAV><A href="/Team:uOttawa/project">project</A><A class="current" href="/Team:uOttawa/drylab">dry lab</A><A href="/Team:uOttawa/policy">policy</A><A href="/Team:uOttawa/team">team</A></NAV></DIV><DIV class="row"><NAV class="sidebar"><H1>Software and Modelling</H1><A href="#" data-pane="modelling">Modelling</A><A href="#" data-pane="eqns">Equations and Parameters</A><A href="#" data-pane="results">Model results</A><A href="#" data-pane="refs">References</A><A href="#" data-pane="bricklayer">Brick Builder</A><P id="desc-modelling">
                See how we modelled the tristable switch to gain valuable insight on the behaviour of this network.
            </P><P id="desc-eqns">
                Our model includes many complex systems of differential equations. Take a look at the math behind our model.
            </P><P id="desc-results">
                View our modelling results and see how well our model compared to empirical results.
            </P><P id="desc-refs">
                A list of articles on which our research and model is based. If you want to know more about our model, this is the place to start.
            </P><P id="desc-bricklayer">
                uOttawa created a software package to intelligently query the biobrick registry and help with assembling genetic constructs.
            </P></NAV><DIV class="content"><DIV class="pane" id="pane-modelling"><H1>Modelling</H1><P>What do we do in computational biological modeling? Simply put, we translate biology into mathematics and back again. It may be to predict, to confirm or to study. Whatever the reason may be, as modelers, our goal is to turn the biology into mathematical terms, which are more easily manipulated, tested and analyzed, and deduce the biological meaning from the results. Above all, our main goal is to work with and support the members conducting the biological research in the laboratory.</P><P>The system we designed and built this year is known as a tristable switch (a more detailed description of the biology can be found in the <A href="/Team:uOttawa/project">project section</A>).  In essence, the system is a two-gene construct with mutual repression and self-activation that should result in three stable states. A diagram of the system is seen in Figure 1. The two genes code for transcriptional factors, GEV and rtTA, which act as self-activators and repressors in the presence of activator molecules (beta-estradiol and aTc, respectively). These small molecules, in effect, act as inducible switches which allow them to control which transcriptional factor is active. Our objectives for the model are to design mathematical models representative of the system in order to 1) predict the stable points of the systems and at what concentrations of beta-estradiol and aTc they occur and 2) to study the dynamics of inducing the switch between stable states. But what does it mean to have stable states?</P><FIGURE><P>Simplified visual diagram of genetic construct. Mutual inhibition and self activation are mediated by beta-estradiol and aTc when GEV and rtTa are involved, respectively.</P></FIGURE><P>In a biological context, stable states refer to states of stable gene expression levels. This means that expression levels tend to progress until it reaches a certain stable expression level, depending on the initial conditions. These states also are resistant to modifications of expression by changing regulatory proteins, loss of genetic product, etc. In a modeling context, it refers to the trajectory of the system (or group) of equations to converge to a steady or stable state. Mathematically speaking, stable states occur at intersections of nullclines, but simply, they attract the trajectory of the system of equations. To analyze model stability, one can use phase plane and bifurcation analyses. Phase plane analyses are used to analyze the dynamics of the stability of the system of equations, at a given set of parameters, and bifurcation analyses are used to analyze the change in stability with regards to modifications of parameters.</P><FIGURE><P>Phase plane analysis of GEV and rTTA.</P></FIGURE><P>In a phase plane analysis, we investigate the stability of two components (molecules like GEV and rtTA) with respect to each other by plotting the amount of the first component (GEV) against the amount of the second (rtTa). For an example, take Figure 2. Each line represents a single simulation, with a unique set of initial conditions (starting amounts) where the blue progressing to red in the line represents the progress of the simulation from the beginning to the end. The phase plane, in essence, is a summary of multiple simulations of the system of equations with different initial conditions. This system of equations tend to progress to three stable states (the groupings of red ends). Stable states are also known as attractor sites or &quot;sinks&quot;, because, like a sink, these sites pull in the trajectories of the components as seen by the phase plane.</P><FIGURE><P>Bifurcation analysis of GEV and rTTA with respect to parameter <I>k9</I>.</P></FIGURE><P>In bifurcation analyses, we investigate the change in stable states with regards to the parameters in order to test robustness (the range of each parameter where the model can still predict tristability) and sensitivity (which parameter can cause the most change in the stable states). The bifurcation diagram is very similar to phase planes; in essence, a phase plane is conducted at each parameter value. At each value, multiple simulations of the system equations are conducted but only the last several values of the simulation (red regions in phase plane) are plotted. One can observe the progress of the stable points as one modifies the parameter. Figure 3 best demonstrates the concept of bifurcation analysis. In this example, we can say that the stable state with high expression of GEV is greatly affected by the parameter. </P><P>We just went through why we are modeling and how we're going to use it. The question now is what we are going to use to model? For modeling biological systems, one of the first choices is whether to use a deterministic or stochastic model. In essence, a deterministic model assumes that all variables of the model can account for the majority of the biological behavior of the system, ignoring the inherent variability of the system. Stochastic models take this variability into account by introducing a random factor, but this often makes the system difficult to analyze. We decided to pursue the design of an ordinary differential equation (ODE) deterministic model and a Gillespie-based stochastic model.</P></DIV><DIV class="pane" id="pane-eqns" hidden=""><H1>Equations and parameters</H1><H2>Deterministic model</H2><P>Our main objectives with the model are to demonstrate tristability and to study the inducibility of the switch, so we designed ordinary differential equation (ODE) models to achieve the two objectives. ODE models refer to the ordinary differential equation function used in the system which are functions where the derivative of a component is defined with respect to itself (e.g. dx/dt = ax +b, where a and b are constants). In our model, we are restricting it to two equations, each one representing the change in amount of GEV or rtTa with respect to time (dGEV/dt and drtTa/dt), in order to simplify the analysis of stability. In essence, both equations represent the change in expression of the two genes over time and these equations are then solved at each time point to give the expression level at said time point.</P><P>Assumptions:</P><UL><LI>Ignoring spatial dimensions (nuclear import of protein, nuclear export of mRNA, etc.)</LI><LI>Grouping of transcription and translation rates into general expression rate</LI><LI>Ignoring degradation of mRNA</LI></UL><FIGURE><P>Graphical representation of how each equation was constructed.</P></FIGURE><P>For each design of the construct, there were two groups of equations: one for showing tristability and one for studying inducible switch. Within each group, there are three sets of equations, each set handles the terms of self-activation and inhibition differently. Every equation in the ODE models are based on what happens to the promoters driving the expression of GEV and rtTa. There will always be a basal expression rate and maximal expression rate affected by the repression and activation caused by the transcriptional factor.</P><FIGURE><P>Schematic diagram of design one of genetic construct</P></FIGURE><P>The next three sets of equations represent design one solely for the purpose of demonstrating tristability. The reason for three different sets of equations is to determine which set of assumptions best represents the genetic construct:</P><FIGURE><P>Equations for demonstrating tristability of design one assuming multiplicative terms.</P></FIGURE><P>Equations (1) and (2) assume that the terms for repression and self-activation are multiplicative. The parameters pertaining to the promoters are: <I>a</I>, which represents the basal expression rate, and <I>b</I>, which represents the maximal expression rate, for pGEVtx in equation (1) and pTREgx  in equation (2). The parameters pertaining to the transcriptional factors are <I>k</I>, which represents the degradation rate of proteins, <I>K</I>, which represents the dissociation constant of the transcriptional factor to DNA, and <I>n</I>, which represents the Hill coefficient of binding for the transcriptional factor to DNA. <I>K</I> and <I>n</I> are also specific to the promoter, so the values of <I>n</I> and <I>K</I> for GEV or rtTA are different between equations (1) and (2).</P><FIGURE><P>Equations for demonstrating tristability of design one assuming additive terms.</P></FIGURE><P>Equations (3) and (4) are similar to equations (1) and (2), except it’s now assumed that the terms of repression and self-activation are additive rather than multiplicative. All parameters are the same to equations (1) and (2), save for <I>d</I>, which represents an arbitrary self-activation strength factor.</P><FIGURE><P>Equations for demonstrating tristability of design one assuming singular terms.</P></FIGURE><P>Equations (5) and (6) assume that repression and self-activation should be treated as a singular term rather than separate terms. All parameters are the same to equations (1) and (2).</P><P>To include the inducible elements of beta-estradiol and aTc to the equations, we assumed that the law of mass action applied and replaced the variables of GEV or rTTA with <I>c<SUB>GEV</SUB> * (GEV * Be)</I> or <I>c<SUB>rTTA</SUB> * (rTTA * aTc)</I>, respectively. <I>Be</I> and <I>aTc</I> represent the amount of beta-estradiol and aTc added to the system [molecules] and the parameter <I>c</I> represents the binding constant of the factor and small molecule binding reaction (<I>c = K - 1</I>). Other than that addition, equations (7)-(12) are similar to equations (1)-(6), where equations (7) and (8) assume multiplicative terms, equations (9) and (10) assume additive terms and equations (11) and (12) assume singular terms.</P><FIGURE><P>Equations for inducible switch of design one assuming multiplicative terms for design two.</P></FIGURE><FIGURE><P>Equations for inducible switch of design one assuming additive terms for design two.</P></FIGURE><FIGURE><P>Equations for inducible switch of design one assuming singular terms for design two.</P></FIGURE><FIGURE><P>Schematic of design two for genetic construct. Similar to design one, except for the addition of pMRP7 that drives rTTA and GEV expression.</P></FIGURE><P>For the second design of the system, there is an addition of the promoter pMRP7 that drives the expression of GEV and rTTA. The addition of a second promoter results in the addition of a second basal expression rate (a2) in each equation from design one. The basal and basal expression rates for pGEVtx (equations (13),(15),(17),(19),(21) and (23)) and for pTREgx (equations (14),(16),(18),(20),(22) and (24)) are now represented by a1 and b1, respectively. This also includes the arbitrary self-activation factor (d1) in equations (15), (16), (21) and (22).</P><FIGURE><P>Equations for demonstrating tristability of design two assuming multiplicative terms.</P></FIGURE><FIGURE><P>Equations for demonstrating tristability of design two assuming additive terms.</P></FIGURE><FIGURE><P>Equations for demonstrating tristability of design two assuming singular terms.</P></FIGURE><FIGURE><P>Equations for inducible switch of design two assuming multiplicative terms.</P></FIGURE><FIGURE><P>Equations for inducible switch of design two assuming additive terms.</P></FIGURE><FIGURE><P>Equations for inducible switch of design two assuming singular terms.</P></FIGURE><H2>Stochastic Design</H2><P>A stochastic model is different from a deterministic model. In Biology, Stochasticity is an attribute of most, if not all, biological systems. A stochastic system is one where the the subsequent state is calculated probabilistically. This is simply another way of saying that there is noise and randomness associated with most things in life. In gene expression, where noise causes lots of &quot;randomness&quot;, stochastic methods of modelling are very useful to account for that &quot;randomness&quot;.</P><P>There are many methods of stochastic modelling and each differ in the exact step where the &quot;random&quot; is added to the model. In the case of the Direct Method, it occurs in reaction determination. In the case of the Tau Leap Method, it occurs in the determination of the time-step. This step is called the Monte Carlo step, named after the famous casino in Monaco, because a random number is generated and used in a calculation. Our stochastic model used the Direct Method since it is simpler to code, and simpler to understand. Unfortunately, it is also pretty computationally inefficient.</P><P>The first step is producing an intimidating flow chart that looks something like this:</P><P>or this:</P><P>These are our flowcharts for the two designs of our construct. The important thing to keep in mind when making a flocwchart is how complex or simple you want to make your model. Often, certain reaction can absorbed into a single one or could even be ignored, it all depends on what you need your model to do. An example of this would be combining transcription and translation into one reaction or keeping them separate if the complexity is necessary for your model.</P><P>Once the flowchart is ready, you must then write rate equations for each reaction, keeping in mind molar ratios. It’s important to remember that things like a promoter binding to a DNA sequence is actually two equations, one for the binding and one for the dissociation. After the equations are completed, the hard part is over and you can start coding.</P><P>First you must initialize the variables and vectors that you will store the populations of each molecule/complex. Assign your initial populations to each variable. Next you copy the rate equations you had done previously into the script. Following this, we must determine which equation is going to occur for the timestep. This is where the Monte Carlo step occurs in the Direct Method of the Gillespie Algorithm. Calculate the sum of all the rates calculated earlier and generate a random number between 0 and that sum. To determine which equation occurs, you find which interval the random number finds itself in. For example, if you have three equations that have rates: 1,2,3. So the cumulative rate is 6, and let’s say the random number you generate is 4.5. 4.5 lies in the interval [3,6] which is between the rates of reaction 2 and 3. This means the reaction that occurs in this time interval is reaction 3. The random number is also used to calculate the timestep for the reaction.</P><P>Once this is done, it’s just a matter of updating the populations of each molecule based on the reaction chosen in the previous step. Usually only a small subset of the molecules changes in population since only one reaction occurs. After this update, you repeat all the steps starting with the rate calculations.</P><H3>Stochastic Parameters</H3><TABLE style="border-top: 1px solid rgb(100, 150, 20); border-bottom: 1px solid rgb(100, 150, 20); width:100%;"><TBODY><TR style="font-weight: bold; border-bottom: 1px solid rgb(100, 150, 20);"><TD>Parameter</TD><TD>Design 1</TD><TD>Design 2</TD></TR><TR><TD>Basal expression rate pGevTx (protein/min)</TD><TD>a1 = 0</TD><TD>ag1 = 0</TD></TR><TR><TD>Maximal expression rate pGevTx (protein/min)</TD><TD>b1 = 4.53E9</TD><TD>bg1 = 4.53E9</TD></TR><TR><TD>Basal expression rate pTreGx (protein/min)</TD><TD>a2 = 0</TD><TD>ar1 = 0</TD></TR><TR><TD>Maximal expression rate pTreGx (protein/min)</TD><TD>b2 = 6.84E6</TD><TD>br1 = 6.84E6</TD></TR><TR><TD>Basal expression rate pmrp7 (protein/min) (1)</TD><TD>NA</TD><TD>a2 = 1E6</TD></TR><TR><TD>GEV degradation (min-1) (2)</TD><TD>k1 = 5.833E-3</TD><TD>k1 = 5.833E-3</TD></TR><TR><TD>rTTA degradation (min-1) (2)</TD><TD>k2 = 5.833E-3</TD><TD>k2 = 5.833E-3</TD></TR><TR><TD>Binding constant GEV-beta-estradiol (molecule-1) (3)</TD><TD>c1 = 6E21</TD><TD>c1 = 6E21</TD></TR><TR><TD>Binding constant rTTA-aTc (molecule-1) (3)</TD><TD>c1 = 6E21</TD><TD>c1 = 6E21</TD></TR><TR><TD>Dissociation constant GEV-pTreGx (molecule-1)</TD><TD>K2GEV = 7.795E19</TD><TD>K2GEV = 7.795E19</TD></TR><TR><TD>Dissociation constant rTTA-pTreGx (molecule-1)</TD><TD>K2ATC = 5.608E19</TD><TD>K2ATC = 5.608E19</TD></TR><TR><TD>Hill coefficient GEV-pGevTx</TD><TD>n1GEV = 1.1326</TD><TD>n1GEV = 1.1326</TD></TR><TR><TD>Hill coefficient rTTA-pGevTx</TD><TD>n1ATC = 1.0082</TD><TD>n1ATC = 1.0082</TD></TR><TR><TD>Hill coefficient GEV-pTreGx</TD><TD>n2GEV = 0.2581</TD><TD>n2GEV = 0.2581</TD></TR><TR><TD>Hill coefficient rTTA-pTreGx</TD><TD>n2ATC = 0.9887</TD><TD>n2ATC = 0.9887</TD></TR></TBODY></TABLE><P>
                    (1) = Values were approximated
                    (2) = GEV and rTTA degradation values were general protein degradation values
                    (3) = Value of binding constant GEV-beta-estradiol from [2]
                    (4) = Value of binding constant rTTA-aTc from [3]
                </P><P> </P><TABLE style="border-top: 1px solid rgb(100, 150, 20); border-bottom: 1px solid rgb(100, 150, 20); width:100%;"><TBODY><TR style="font-weight: bold; border-bottom: 1px solid rgb(100, 150, 20);"><TD>Parameter</TD><TD>Design 1</TD><TD>Design 2</TD></TR><TR><TD>Constitutive binding rate (molecule-1 * min-1)</TD><TD>k1 = 1</TD><TD>k1 = 1</TD></TR><TR><TD>Constitutuve dissociation rate(min-1)</TD><TD>NA</TD><TD>k1<SUB>2</SUB> = 0.1</TD></TR><TR><TD>Constitutuve transcription rate(min-1)</TD><TD>NA</TD><TD>k2 = 1.5</TD></TR><TR><TD>Translation rate (min-1)</TD><TD>k4 = 1</TD><TD>k3 = 0.5</TD></TR><TR><TD>Drug activation rate (molecule-1 * min-1)</TD><TD>NA</TD><TD>k4 = 1</TD></TR><TR><TD>Inhibitor binding rate (molecule-1 * min-1)</TD><TD>k1 = 1</TD><TD>k4 = 1</TD></TR><TR><TD>Inhibitor dissociation rate (min-1)</TD><TD>NA</TD><TD>k5<SUB>2</SUB> = 0.1</TD></TR><TR><TD>Inhibited transcription rate (min-1)</TD><TD>k3 = 0.5</TD><TD>k6 = 0.1</TD></TR><TR><TD>Activator binding rate (molecule-1 * min-1)</TD><TD>k1 = 1</TD><TD>k7 = 1</TD></TR><TR><TD>Activator dissociation rate (min-1)</TD><TD>NA</TD><TD>k7<SUB>2</SUB> = 0.1</TD></TR><TR><TD>Activated transcription rate (min-1)</TD><TD>k2 = 1.5</TD><TD>k8 = 1</TD></TR><TR><TD>GFP/BFP transcription or translation rate (min-1)</TD><TD>k6 = 0.5</TD><TD>k9 = 1</TD></TR><TR><TD>Degradation rate (min-1)</TD><TD>k1<SUB>2</SUB> = 1</TD><TD>k10 = 0.5</TD></TR><TR><TD>Basal rate (min-1)</TD><TD>b = 1</TD><TD>b = 1</TD></TR></TBODY></TABLE><P>Parameters are relative to the basal rate of 1.</P></DIV><DIV class="pane" id="pane-results" hidden=""><H1>Results</H1><P>The deterministic models were tested by conducting phase plane analyses on each set of equations for each design (an explanation for phase plane analysis can be found on the introduction page). With phase plane analyses, one can observe the stability of the system of equations (group of equations) and how many stable states exist. For our purposes, we expect to see three stable states (three distinct red areas) in the diagrams and the amounts of GEV and rtTa associated with the stable states. The current model data for design one demonstrate that the system is monostable (only one stable state). Each set of equations (multiplicative, additive and singular terms, which are described in the equations page) were tested and yielded very similar results. The multiplicative and singular term equations stabilized around no expression of GEV and rtTA (GEV=0, rtTa=0) and the additive equations stabilized around 8x1011 GEV molecules and somewhere in between 1x108¬ and 1x105 rtTa molecules (not 0 rtTA molecules given the log transformed phase plane analysis).</P><FIGURE class="full"><P>Phase plane and log-phase plane diagrams of the three sets of equations (Multiplicative, Additive and Singular terms) for design one of tristable switch. Each line represents one simulation of the equations with a unique set of initial conditions and the progress of the simulation is denoted by the progress from blue (beginning) to red (end). All simulations were run for 4000 steps and all parameters of the equations may be found in parameters page.LEFT: phase plane analyses of the multiplicative equations MIDDLE: phase plane analyses of the additive equations. RIGHT: phase plane analyses of the singular term equations. TOP ROW: non-transformed phase plane analyses. BOTTOM ROW: log transformed (log(rtTa) vs log(GEV)) phase plane analyses.</P></FIGURE><P>The results for the analyses of the model for design two were very similar to those of design one showing only monostability of the system. The difference between the designs was the position of the stable states in the multiplicative and singular term equations. In design one, those equations stabilized at no expression of GEV and rtTa (GEV=0, rtTa=0), but for the second design, they stabilized at equal expression of both, around 2x108 molecules of GEV and rtTa. Both non-transformed and log transformed phase planes suggest that tristability may not be obtained with the current promoters and system designs being used in the wet lab, but the model needs to be validated with experimental results before accepting these results.</P><FIGURE class="full"><P>Phase plane and log-phase plane diagrams of the three sets of equations (Multiplicative, Additive and Singular terms) for design two of tristable switch. See previous figure for a legend.</P></FIGURE><P>The stochastic models were tested by conducting individual simulations with identical starting amounts of GEV and rtTa, but varied the initial amount of beta-estradiol and aTc. Three conditions of initial activator molecules amounts were tested: beta-estradiol &gt; aTc, beta-estradiol = aTc, beta-estradiol &lt; aTc. Each condition resulted in different and distinct stable states where the initial amount of activator molecule dictated which protein would be more expressed. In the case of higher beta-estradiol amount, the amount of GEV found in the nucleus was higher than the amount of rtTa and vice versa for the case of lower beta-estradiol amount. In the case of equal amounts, there were roughly equal amounts of GEV and rtTa transcriptional factors found in the nucleus. These results are promising and suggest that it might be possible to obtain tristability with the design one construct in vitro.</P><FIGURE class="full"><P>Individual simulations of stochastic model for design one of the construct. The amount of GEV (green) and rtTa (blue) found in the nucleus was plotted against time of the simulation (minutes). The simulations were run with 50 beta-estradiol and 5 aTc molecules (LEFT), 50 beta-estradiol and 50 aTc molecules (MIDDLE) and 5 beta-estradiol and 50 aTc molecules (RIGHT). All simulations were implemented with identical parameters found on the parameter page. </P></FIGURE><P>The results for the stochastic model of design two are similar to those obtained from the models of design one. The three conditions (beta-estradiol &gt; aTc, beta-estradiol = aTc, beta-estradiol &lt; aTc) resulted in three distinct stable states with respective amounts of GEV and rtTa. These results are promising and suggest that it might be possible to obtain tristability with the design two construct in vitro. </P><FIGURE class="full"><P>Individual simulations of stochastic model for design one of the construct. The amount of GEV (green) and rtTa (blue) found in the nucleus was plotted against time of the simulation (minutes). The simulations were run with 50 beta-estradiol and 5 aTc molecules (RIGHT), 50 beta-estradiol and 50 aTc molecules (MIDDLE) and 5 beta-estradiol and 50 aTc molecules (LEFT). All simulations were implemented with identical parameters found on the parameter page.</P></FIGURE><P>Although the deterministic models demonstrate that obtaining a tristable system with the promoters currently being used may not be possible, the stochastic models do show a possibility of using the system to attain three stable states. The issue is that model data is not reliable unless the models have been validated with experimental data from in vitro, so the immediate next step is to validate our model results with experimental results from the constructs. With validation, the parameters of the system can be modified to better reflect the in vitro system. Following validation and model optimization, more rigorous stability analyses can be conducted on the deterministic and stochastic models to obtain a better insight of tristability. Once the system is shown to be tristable, then analyses can be conducted to study the dynamics of the inducible switch.</P><H2>Additional data</H2><P>Download additional figures in the following documents:</P><UL><LI><A href="https://static.igem.org/mediawiki/2014/9/92/Uo2014-design1results.pdf" target="_blank">Additional design 1 results</A></LI><LI><A href="https://static.igem.org/mediawiki/2014/b/b4/Uo2014-stochastic1dbifurcations.pdf" target="_blank">Stochastic model 1D bifurcations</A></LI><LI><A href="https://static.igem.org/mediawiki/2014/1/15/Uo2014-stochastic2dbifurcations.pdf" target="_blank">Stochastic model 2D bifurcations</A></LI><LI><A href="https://static.igem.org/mediawiki/2014/5/59/Uo2014-stochasticphase.pdf" target="_blank">Stochastic model phase diagrams</A></LI><LI><A href="https://static.igem.org/mediawiki/2014/b/b1/Uo2014-stochasticpopulation.pdf" target="_blank">Stochastic model population vs time charts</A></LI><LI><A href="https://static.igem.org/mediawiki/2014/4/41/Uo2014-modaux.pdf" target="_blank">Auxiliary data</A></LI></UL></DIV><DIV class="pane" id="pane-refs" hidden=""><H1>References</H1><P> Orth, Peter, Dirk Schnappinger, Phaik-Eng Sum, George A. Ellestad, Wolfgang Hillen, Wolfram Saenger, and Winfried Hinrichs. &quot;Crystal Structure of the Tet Repressor in Complex with a Novel Tetracycline, 9-(N, N-dimethylglycylamido)-6-demethyl-6-deoxy-tetracycline1.&quot; Journal of Molecular Biology 285.2 (1999): 455-61. Print.</P><P>Takahashi, M. Kinetic and Equilibrium Characterization of the Tet Repressor-Tetracycline complex by Fluorescence Measurements. J.Mol.Bio (1986) 187: 341-348 </P><P>McIsaac, R., et al. Synthetic gene expression perturbation systems with rapid tunable single-gene specificity in yeast. Nuc.Acids.Research (2013) 41(4) doi:10.1093/nar/gks1313 </P><P>McIsaac, R., et al. Fast-acting and nearly gratuitous induction of gene expression and protein depletion in Saccahromyces cerevisae. Molecular Biology of the Cell (2011) 22(4) : 4447-4459 doi:10.1093/nar/gks1313</P><P>Sneppen K., Krishna, S., Semesy, S. Simplified models of biological networks. Annu.Rev.Biophys (2010), 39:43-59</P></DIV><DIV class="pane" id="pane-bricklayer" hidden=""><H1>Brick Builder <SPAN style="font-size:0.6em"> the ultimate DNA assembly tool</SPAN></H1><P>The Brick Builder software is a web-based application that allows teams to virtually engineer their DNA constructs and design primers for assembly - all in one place. Users can query and select parts from the Bio Brick database, and then design a construct using their chosen parts. Brick Builder will generate the primers necessary to build the desired construct using the user’s choice of compatible assembly method and melting temperature range, among other optional parameters.</P><H2>Search</H2><P>Searching could not be simpler. Using iGEM's provided API tools, users can search for parts directly from the parts registry with up-to-date information.</P><H2>Search Results</H2><P>Sometimes a search can result in hundreds of parts being displayed, making it difficult to find the perfect one. With the new implementation of pagination, users can browse for particular parts by sorting them by their properties.</P><H2>Brick Bin</H2><P>Users can save selected parts into their very own unique Brick Bin. Here, users can review their saved parts and remove any nonessential parts from their Brick Bin in preparation for construction.</P><H2>Construction</H2><P>The Construct Bin displays the parts intended for construction. Users are able to add, delete and reorder the parts from the start of the construct (top of the list) to the end (bottom of the list). When users are satisfied with the construct, a simple click will generate the primers required for that specific construct.</P></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:uOttawa/drylab">http://2014.igem.org/Team:uOttawa/drylab</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:uOttawa/drylab" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:uOttawa/drylab" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:uOttawa/drylab&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:uOttawa/drylab&amp;oldid=396672" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>