<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_uOttawa_project"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:uOttawa/project">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:uOttawa/project&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:uOttawa/project&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:uOttawa/project&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:uOttawa/project" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:uOttawa/project</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><DIV class="row"><A href="https://2014.igem.org/Team:uOttawa" class="title"><SPAN class="igem">iGEM</SPAN>uOttawa</A><NAV><A class="current" href="/Team:uOttawa/project">project</A><A href="/Team:uOttawa/drylab">dry lab</A><A href="/Team:uOttawa/policy">policy</A><A href="/Team:uOttawa/team">team</A></NAV></DIV><DIV class="row"><NAV class="sidebar"><H1>Project</H1><A href="#" data-pane="project">The Project</A><A href="#" data-pane="tristable">The Tri-Stable Switch</A><A href="#" data-pane="promoters">Promoters</A><A href="#" data-pane="wres">Results</A><A href="#" data-pane="bricks">Biobricks</A><A href="#" data-pane="interlab">Interlab Study</A><A href="#" data-pane="wrefs">References</A><P id="desc-project">
                Read an introduction about this year's project.
            </P><P id="desc-tristable">
                We describe our approaches in creating the proposed tri-stable switch network.
            </P><P id="desc-promoters">
                uOttawa designed a handful of complex and dual-input yeast promoters for this year's project. Read about them here.
            </P><P id="desc-wres">
                See comparisons and characterization of each promoter.
            </P><P id="desc-bricks">
                Self explanatory.
            </P><P id="desc-interlab">
                View uOttawa's results for the iGEM measurement track's Interlab Study.
            </P><P id="desc-wrefs">
                Want more info on the tristable switch and dual-input promoters? These papers are an excellent place to start.
            </P></NAV><DIV class="content"><DIV class="pane" id="pane-project"><H1>The Project</H1><H2>Engineering Fate: Cellular decision making and the Tri-Stable switch</H2><P>Throughout our lives, individual cells make vital decisions that directly affect us. From deciding how to develop, when stem cells differentiation into various cell types, to when to die, as mutated or erroneous cells undergo apoptosis.</P><P><B>We decided to examine how cells make those decisions.</B></P><P>It was hypothesized that a unique tri-stable switch controlled stem cell differentiation, with the three states being  an arbitrary state A, B and AB,  where both states coexists stably (AB).</P><FIGURE><P>Design adapted from Sui Huang, 2009.</P></FIGURE><P>For instance, if state A produced blue marbles and B red marbles, the three states would look like this:</P><P>Now instead of marbles, lets image A and B as cell types like liver cells and heart cells, and the AB state the undifferentiated state! This of course is a massive over simplification, as A and B in nature are likely transcription factors. Yet, it helps to visualize this switch as such.</P><FIGURE><P>(TOP) Human Blastocyst from early development. (LEFT) Human liver cells. (RIGHT) Human heart cells.</P></FIGURE><P>This is a primary example of cellular decision making. The 2014 uOttwa iGEM team chose to build this decision making pathway. To do so we created a <B>novel</B> form of <B>gene regulation</B> using <B>activators</B> as <B>repressors</B>.</P><H2>Potential Applications</H2><P>Why build such a system? Understanding how this genetic network works and being able to model its behaviour may shed light on how exactly stem cells differentiate. More importantly, it will allow us to engineer cells that implement this synthetic decision-making pathway, and use it in an application such as logic gates.</P><P>Alternatively, we may use this system as a unique cellular detector. If A and B are reporters driven by promoters that are sensitive to small molecules such as phosphorous and nitrogen, these cells can monitor the balance between those two. The balance between those two is an important indicator of human pollution, which is indicated by high levels of phosphorous.  If one spikes higher than the other, the cell will enter either the A or B state, which would be indicated by their respective reporters. If both spike, it will remain in the AB and indicate an equilibrium.</P></DIV><DIV class="pane" id="pane-tristable" hidden=""><H1>The Tristable Switch</H1><H2>Background</H2><P>In multiple papers, including Sui Huang’s 2009 paper, tri-stability was predicted from a typical bi-stable switch with self-activation, as shown below:</P><P>An example of how it may be used in differentiation is the pathway between stem cells, and tropho-ectoderm and inner cell mass (Huang 2009). It is important to mention that this pathway, along with many similar ones with the same architecture is embedded in larger regulatory networks.</P><FIGURE><P>An example of a tri-stable switch in nature. S<SUB>A</SUB>, S<SUB>B</SUB> and S<SUB>B</SUB> stand for state A, B and C, where state C is the undifferentiated AB state. Figure from Huang et al. 2007.</P></FIGURE><P>However, this pathway could be used in many ways if recreated synthetically, as our team has attempted. </P><H2>Our designs</H2><P>In order to implement this network, A and B appear to have to work as both an activator (to activate themselves) and a repressor (of the other state). Thus, we had to design a system where A and B can function as both. In brief, activator binding sites were placed 10bp away from the TATA box, causing steric hindrance of the TATA binding protein. This is explained in more depth, along with accompanied data in the <B>Promoters</B> section.</P><P>For our system to work, two controllable transcription activators were selected, GEV and rtTA. GEV is a fusion protein of the gal4 DNA binding domain (G), a human estrogen receptor subunit (E), and VP16 (V), a viral trans-activator. To function, two GEV molecules must dimerize with a beta-estradiol molecule, localizing the GEV to the nucleus and allowing for activation. As such, how much GEV is active can be controlled via estradiol concentration. Similarly, rtTA is the tetR binding domain with a VP16 trans-activator, which requires anhydrotetracycline (aTc) in order to function. Thus, we can control how much of each trans-activator is active at any given time by varying the concentration of these small molecules. This is important for our design, as it allows us to test the functionality of our promoters, and examine our system from various 'start points' or states.</P><P>To create this system, we engineered two designs, both of which are based upon this core interaction:</P><FIGURE><P>Tri-stable involved portion of our network. Activation arrows are shown in green, and repression arrows in red. GEV and rtTA are transcriptional activators. pGALtx is a promoter activated by GEV and repressed by rtTA. pTREgx is a promoter activated by rTTA and repressed by GEV. tPGK1 is a transcriptional terminator.</P></FIGURE><P>Thus, in order to create this system, we had to design novel promoters to drive GEVE and rtTA, along with promoters to drive reports with similar expression levels. Their design, construction, and testing are described in the Promoters section</P><P>We then created two designs with this core.</P><H2>Design 1</H2><P>This design is simply the core, with selection and reporter cassettes, along with DNA overhangs to transform the construct into <I>Saccharomyces cerevisiae.</I></P><FIGURE><P>Full network design. On top of tri-stable portion described above, selection cassettes were added for transformation, surrounded in red (KanMX and NatMX). Reported cassettes driving GFP and BFP were added to track the activity and amount of the transcriptional activators. Ade2 and Ade4 are DNA overhands for the adenine 2 and 4 gene respectively, used for transformation into the yeast genome.</P></FIGURE><P>One of the concerns around this design is the fact that it has no basal activator present, depending upon basal transcriptional noise to start the system. It may have been necessary to give the system a kick start of activator or have some basal level of both activators. To solve this problem, design two was created. However, a strain of yeast containing GEV being self-activated, and a reporter produced a strong signal, as shown below:</P><FIGURE><P>Partial Network for self-activation test construct. The construction of this strain also contains other pieces of design one, but they were proved to be non-functional.</P></FIGURE><H2>Self-activation</H2><P>In a strain with the gal4 protein knocked, we were able to show GFP expression from just the transcriptional noise from pGALtx, which is driving GEV. This validates design 1. </P><FIGURE><P>GFP expression of GEV noise self-activated system. GFP is driven by a pGAL promoter with 4 activating gal4 sites, so increasing estradiol results in increasing activation. GEV is driven by pGALtx and the Gal4 protein is knocked out, so all activation is a result of leaky expression by pGALtx.</P></FIGURE><H2>Design 2</H2><P>This design contains the same network as design one, but adds the constitutive promotion of each of the activators, to have a basal level of each.</P><FIGURE><P>Constitutive promotion by the native yeast promoter pMRP7 of each transcriptional activator. Gal U and Gal D represent DNA overhangs for transformation into the Gal4 locus, His is the histidine selection cassette, and tPGK1 is a transcriptional terminator.</P></FIGURE><P>Unfortunately, we were unable to complete construction of either design due to problems with construction and transformation. However, we were able to test each functional part independently, so completion is now simply a matter of assembly.</P><H2>Future Directions</H2><P>We hope to finish construction of both designs, and test them to see if we can show three states. This can be done by taking state 'snap-shots' by adding specific amounts of estradiol and aTc and noting expression, or by modulating the amount of small molecules added over time, to move from one state to another.</P></DIV><DIV class="pane" id="pane-promoters" hidden=""><H1>Promoters</H1><P>The bulk of our project lay in the design, construction, and testing of the many promoters found in our designs. The aim was to create a set of dual input promoters that can be activated by one activator, and repressed by the other. Single input promoters were also needed for driving reporters, and were considered in other experimental designs. Promoters will be activated or repressed with GEV (gal4 binding) and rtTA (tet binding), both of which are described in detail in the tri-stable section. </P><P>All promoters were based upon the pGAL promoter found natively in Saccharomyces cerevisiae. Activating sites were placed where gal4 sites or mig1 sites already existed, in the hope of not disturbing the activity of this strong promoter. Repressing sites were placed 10 base pairs downstream of the TATA box, which was shown by Ellis et al. 2009 to retain promoter activity, as they placed repressing sites 10bp away from the TATA box.</P><H2>Repression by hindrance</H2><P>In short, transcription is repressed by sterically hindering the binding of the TATA binding protein to the TATA box. Thus, despite the bound proteins being an activator complex, transcription is prevented. Our data proves that this repression is not only strong and robust once activator concentration passes a certain saturation point. Activators only have a positive effect at very low functional concentrations, and even then it is minute. Along with very steep repression after a point, this indicates that this form of repression requires a certain saturation point before taking effect.</P><P>Legend of promoter sites:
                    <SPAN style="margin: 0px 7px; background: rgb(78, 166, 60); color: white; padding: 2px 4px;">Gal4 site</SPAN><SPAN style="margin-right: 7px; background: rgb(250, 180, 50); color: black; padding: 2px 4px;">TetO site</SPAN><SPAN style="background: #111; color: white; padding: 2px 4px;">TATA box</SPAN></P><H2>pTreGx</H2><P>This dual input promoter has four upstream activating sequences (UAS) and two repressing sequences. The four UAS sites are tetO binding sites that can bind to the tetracycline responsive activator protein, rtTA (reverse tetracycline-controlled transactivator), to induce transcription. The third and fourth GAL4 binding sites of the native pGAL1 promoter are replaced with tetO sites in this version and the first two GAL4 sites are replaced with random sequences with identical C-G content . The Mig1 sequences that are native to the pGAL1 promoter are replaced with two tetO sites. The two repressing sequences are binding sites for the GAL4 DNA binding domain proximal to the TATA box, causing transcriptional repression by steric hindrance and prevention of transcription machinery assembly at the promoter.</P><P>In cells expressing rtTA and GEV (GAL4 binding domain-human estrogen receptor-VP16 activator domain), this promoter can be used to drive transcription of a downstream gene by the addition of aTc (anhydrotetracycline). The level of transcription can be modulated or repressed with the addition of beta-estradiol.</P><H2>pTre(4)</H2><P>This single input promoter has four upstream activating sequences (UAS). The four UAS sites are tetO binding sites that can bind to the tetracycline responsive activator protein, rtTA (reverse tetracycline-controlled transactivator), to induce transcription. The third and fourth GAL4 binding sites of the native pGAL1 promoter are replaced with tetO sites in this version and the first two GAL4 sites are replaced with a random sequence. The Mig1 sequences that are native to the pGAL1 promoter are replaced with two tetO sites.</P><P>In cells expressing rtTA, this promoter can be used to drive transcription of a downstream gene by the addition of aTc (anhydrotetracycline).</P><H2>pTre(2)</H2><P>This single input promoter has two upstream activating sequences (UAS). The third and fourth GAL4 binding site of the native pGAL1 promoter has been replaced with tetO binding sites in this version and the first and second GAL4 sites have been replaced with random sequences with identical C-G content. The Mig1 sequences that are native to the pGAL1 promoter are removed to allow transcriptional activation of the promoter in the presence of glucose in the cellular growth medium.</P><P>In cells expressing rtTA, this promoter can be used to drive transcription of a downstream gene by the addition of aTc (anhydrotetracycline). This is a weakly activating promoter.</P><H2>pGal</H2><P>This is the native pGAL1 promoter in S. cerevisiae with its Mig1 sites scrambled. This is a single input promoter that has four upstream activating sequences (UAS), which are GAL4 binding elements. The Mig1 sites are binding sites for proteins responsible for transcriptional repression of downstream genes to this and other promoters in S. cerevisiae in the presence of glucose. The removal of the Mig1 sites allows transcriptional activation of the promoter in the presence of glucose in the cellular growth medium.</P><H2>pGalTx</H2><P>This is a dual input promoter that has four upstream activating sequences (UAS), which are GAL4 binding elements, and two repressing sequences proximal to the TATA box, which are tetO sites. The proximity of the tetO sites to the TATA box causes transcriptional repression by steric hindrance and prevention of transcription machinery assembly at the promoter. The Mig1 sequences that are native to the pGAL1 promoter are scrambled to prevent transcriptional repression in the presence of glucose. </P><P>In cells expressing rtTA and GEV (GAL4 binding domain-human estrogen receptor-VP16 activator domain), this promoter can be used to drive transcription of a downstream gene by the addition of Beta-estradiol. The level of transcription can be modulated or repressed with the addition of aTc (anhydrotetracycline).</P><P>This promoter was originally designed and tested by Tom Ellis et al. 2009.</P></DIV><DIV class="pane" id="pane-wres" hidden=""><H1>Results</H1><P>The bulk of our project lay in designing, testing and characterizing the key promoters within our designs. Below are a summary of the results we obtained throughout the summer.</P><H2>Repression by Activators</H2><P>The characterization of repression via the binding of hindering activators was an integral part of our project. Below are characterizations of the two promoters we modified and/or designed for our project. For all the following graphs, the X and Y axes show small molecule concentration, which can either represent increased activation or repression, depending on the promoter. Below is an example of strong repression by rtTA. </P><FIGURE><P>Characterization of pGALtx via dual drug induction. pGALtx has four activating gal4 sites upstream of the TATA box and 2 repressing tet sites downstream. Estradiol then represents promoter activation and aTc promoter repression. rtTA is driven by a strong constitutive promoter. GFP is represented by arbitrary fluorescence units obtained from our flow cytometer. </P></FIGURE><P>However, when we first characterised these promoters, we were using a weaker constitutive promoter to drive the repressing activator. By using a strong repressor we got dramatically increased repression, indicating a certain saturation point is required of activator for repression to be robust.</P><FIGURE class="full"><P>Four activator site pGALtx under different repressor saturation. pGALtx has 4 activating gal4 sites and 2 repressing tet sites. On the left, rtTA is driven by the weak constitutive promoter pMRP7 and on the right rtTA is driven by the strong constitutive promoter pADH1. aTc represents the amount of functional repressing activator, while estradiol the amount of functional activating activator.  </P></FIGURE><H2>Number of Activating Sites</H2><P>The number of activating sites were also varied. It was found that at least four sites were required to have significant enough expression. Below is a comparison between pTRE promoters, one with 4 sites the other 2 sites. As one can see, expression jumps from 4 fluorescence units to 30 at maximum.</P><FIGURE><P>Comparison of activation by two gal site and 4 gal site pgALtx. On the left is pGALtx with only two activating gal4 sites, and on the right is pGALtx with four activating gal4 sites. The activation by GEV is examined here, with estradiol drving activation. rtTA is behind a weak promoter, so repression is minimal or non-existent. Any increases in activation with aTc is likely due to auto-fluorescence caused by the drug. </P></FIGURE><H2>Promoter Characterization</H2><P>We in fact  characterised all the main promoters used in our designs. Below are the remaining promoters. All repression is shown with strong constitutive activation. </P><P><B> pGAL </B></P><FIGURE><P>Characterization of pGAL via dual drug induction. pGAL has no repressing sites and 4 activating gal4 sites, so increasing estradiol increases activation while aTc has no effect beyond auto-fluorescence. This particular data set has GEV being driven by pMRP7, a week constitutive promoter.</P></FIGURE><P><B> pTRE(4)gx </B></P><FIGURE><P>Characterization of pTREgx via dual drug induction. pTREgx has 4 activating tetO sites and 2 repressing gal4 sites 10bp away from the TATA box. Thus, activation increases with aTc concentration and repression increases with estradiol concentration.  </P></FIGURE><P><B> pTRE(2)gx </B></P><FIGURE><P>Characterization of pTREgx via dual drug induction. pTREgx has 2 activating tetO sites and 2 repressing gal4 sites 10bp away from the TATA box. Thus, activation increases with aTc concentration and repression increases with estradiol concentration. Repression is minimal as GEV is only weakly promoted by pMRP7, unlike the pTRE(4)gx data, where a strong promoter pADH1 is driving GEV </P></FIGURE><P><B> pTRE(4) </B></P><FIGURE><P>Characterization of pTRE via dual drug induction. pTRE has 4 activating tetO sites and not repressing sites, so increasing aTc increases activation. Estradiol has no effect beyond auto-fluorescence.   </P></FIGURE><P><B> pTRE(2) </B></P><FIGURE><P>(LEFT) The pTre promoter shows no activation upon addition of aTc (anhydrotetracycline)in cells expressing the activator, rtTA (reverse tetracycline-controlled transactivator), with the weak constitutive promoter, pMRP7. (RIGHT) In cells expressing rtTA with the strong constitutive promoter, pADH1, there is weak activation with increasing aTc concentration. Increasing concentrations of estradiol was added to compare its effects on activation with the other cognate promoters made. No increase in activation is seen with increasing estradiol due to the absence of GAL4 UAS's in this promoter. </P></FIGURE><H2>Additional data</H2><P>View the <A href="https://static.igem.org/mediawiki/2014/1/1b/Uo2014-raw.pdf" target="_blank">raw flow cytometry data</A>.</P></DIV><DIV class="pane" id="pane-bricks" hidden=""><H1>Submitted biobricks</H1><DIV id="partsTableContainer" class="row parts text-center">&lt;groupparts&gt;iGEM2014 uOttawa&lt;/groupparts&gt;</DIV></DIV><DIV class="pane" id="pane-interlab" hidden=""><H1>Interlab Study</H1><P>The goal of the Interlab Study was to collect and compare fluorescence data for 3 devices:</P><UL><LI><I>BBa_I20260</I> in <I>pSB3K3</I>, which came pre-assembled in the iGEM distribution and expresses GFP behind a strong promoter <I>BBa_J23101</I>;</LI><LI><I>BBa_23101</I> + <I>BBa_E0240</I> in <I>pSB1C3</I>, which expresses GFP behind a strong promoter;</LI><LI>and <I>BBa_J23115</I> + <I>BBa_E0240</I> in <I>pSB1C3</I>, which expresses GFP behind a weaker promoter.</LI></UL><P>Applicable constructs were assembled using standard biobrick assembly by inserting the <I>BBa_E0240</I> part into the plasmid containing the upstream promoter. Colonies were grown on appropriate selection and screened for protein expression on a flow cytometer (Beckman Coulter CyAn with 488 nm laser and FITC filters). Positive colonies were then re-grown in straight LB media for 8 hours in a shaking incubator at 37 C and 200 RPM, then measured in the same flow cytometer.</P><P>The protocol for flow cytometry involved suspending cells in 50 mM sodium citrate to a final concentration of OD<SUB>600</SUB> = 0.03. Cells were then inoculated into a 96-well plate and run on the cytometer.</P><P>The cytometer was calibrated against a negative control and Beckman Coulter <I>Flow-Set Pro Fluorospheres</I>. These controls are included in the data. The fluorospheres served as a reference to which to compare fluorescence of cells.</P><H2>Confirmation</H2><P>DNA sequencing was not available for these constructs. However, a PCR confirmation was performed from a miniprep of positive colonies. Each yielded predicable bands at about 1000 bp, which is the length of all three constructs.</P><H2>Extra credit: cell-to-cell variation</H2><P>Flow cytometry allows hundreds to hundreds of thousands of events to be captured in a relatively short time. Since each event ideally represents a single cell, it is trivial to obtain results for multiple cells very quickly. Here, we found that the fluorescence of cells formed quite well-defined peaks distinct from other constructs that were tested.</P><P>However, the nature of flow cytometry also includes noise and debris in the result set. Thus, flow cytometry results must be taken with a grain of salt. A single event holds no significance on its own, and due to the wide noise profile, taking the standard deviation of a single trial yields a very large value. We tested multiple colonies and cell lines for each construct and compared the arithmetic means for each in order to obtain reliable results.</P><H2>Results and Interlab Study form</H2><A href="https://static.igem.org/mediawiki/2014/c/cd/Uo2014-interlab.pdf">Download</A> our interlab study form and results.
            </DIV><DIV class="pane" id="pane-wrefs" hidden=""><H1>References</H1><P>Ellis, T., Wang, X., &amp; Collins, J. (2009). Diversity-based, model-guided construction of synthetic gene networks with predicted functions. Nature Biotechnology, 27(5): 465-471.</P><P>
Wang, Y., Huang, C., Tung, S., &amp; Lin, Y. (2000). Competition with TATA Box-Binding Protein for Binding to the TATA Box Implicated in Human Cytomegalovirus IE2-Mediated Transcriptional Repression of Cellular Promoters. DNA and Cell Biology, 19(10): 613-619.</P><P>
Brachman, C., Davies, A., Cost, G., Caputo, E., Li, J., Hieter, P., &amp; Boeke, J. (1998). Designer Deletion Strains derived from Saccharomyces cerevisiae S288C: A Useful set of Strains and Plasmids for PCR-mediated Gene Disruption and Other Applications. Yeast, 14: 115-132.</P><P>
Balázsi, G., Van Oudenaarden, A., &amp; Collins, J. (2011). Cellular Decision Making and Biological Noise: From Microbes to Mammals. Cel,l 144(6): 910-925.</P><P>
Huang, S. (2009). Reprogramming cell fates: Reconciling rarity with robustness. Bioessays, 31(5): 546-560.</P><P>
Way, J., Collins, J., Keasling, J., &amp; Silver, P. (2014). Integrating Biological Redesign: Where Synthetic Biology Came From and Where It Needs to Go. Cell, 157(1): 151-161.</P><P>
Dueber, J., Mirsky, E., &amp; Lim, W. (2007). Engineering synthetic signaling proteins with ultrasensitive input/output control. Nature Biotechnology, 25(6): 660-662.

</P><P>Huang S, Guo YP, May G, Enver T. “Bifurcation dynamics in lineage-commitment in bipotent progenitor cells.” Developmental Biology (2007). 305:695-713.</P></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:uOttawa/project">http://2014.igem.org/Team:uOttawa/project</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:uOttawa/project" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:uOttawa/project" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:uOttawa/project&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:uOttawa/project&amp;oldid=401614" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>