<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_NYU_Abu_Dhabi_Description skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:NYU Abu Dhabi/Description</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="content"><DIV class="content-inside"><H6><B>Description</B></H6><DIV class="section-block"><CENTER><DIV class="tab"><BUTTON class="tablinks" onclick="openTab(event, 'Key Achievments')" id="defaultOpen">Background</BUTTON><BUTTON class="tablinks2" onclick="openTab(event, 'biology')">Methodology</BUTTON><BUTTON class="tablinks3" onclick="openTab(event, 'engineering')">Significance</BUTTON></DIV></CENTER><DIV id="Key Achievments" class="tabcontent"><ARTICLE><CENTER><H4><CENTER><U>Abstract</U></CENTER></H4><H2><B>Pathogene: A portable, low-cost, microfluidic lab-on-a-chip based device for rapid detection of multiple foodborne pathogens
</B></H2><H2>Despite regulations in place to ensure the distribution of safe food, foodborne diseases (FBDs) remain a global concern. To address the worldwide challenge of FBDs, we have devised a customizable device for the simultaneous detection of multiple food-borne pathogens (FBPs). The device detects specific DNA sequences associated with four FBPs: Campylobacter, Listeria monocytogenes, Salmonella, and Vibrio cholerae using the isothermal amplification techniques: recombinase polymerase amplification (RPA) and loop-mediated isothermal amplification (LAMP). The use of isothermal techniques allows the device to be more portable and cost-effective compared to conventional PCR systems, while the use of microfluidics allows for multiplexing and rapid high-throughput screening. The parameters of the device such as the number of pathogens, and amplification and detection methods can be customized as required. This novel lab-on-a-chip based device is rapid, portable, affordable, sensitive, specific, and customizable, making it ideal for resource-limited settings and point-of-care testing.
</H2><H4><CENTER><U>Background</U></CENTER></H4><H2>Food safety and prevention of foodborne illness remains an important plight of members of the international community and more specifically the scientific community. Even though most countries have food safety regulations and laws, it is still a challenge to ensure that all food companies and manufacturers, and not just the large-scale ones, are compliant to these rules and sell food that is safe to consume. Often, the challenges that the small-scale food producers or everyday consumers face in deciding whether food is safe to consume are challenges of accessibility, affordability, and portability of the methods of detection. Thus, this project aims to build on and improve on last year’s NYU Abu Dhabi iGEM project that detected the presence of STEC in food. They created a device that was a cost-effective and portable. This year’s team hopes to build on it by developing a service of customizable devices to suit anyone’s need. The team started by analysing different target groups and realized that there is a huge need for a quick   detection of food pathogens in many different areas (See more in target group info under ideation). We focused on making our service wide by providing different techniques (LAMP and RPA), waiting time, different visualization methods (fluorescence and color change) and detection of various different pathogens that cause foodborne illnesses. We made sure that the device is efficient and portable and  guided this year’s project based on WHO’s ASSURED (Affordable, Sensitive, Specific, User-friendly, Rapid, Robust, Equipment free, and Delivered to those who need it) ideal features for diagnostic tests (1).
</H2><H2>The team has selected four pathogens to be investigated: <I>Campylobacter, Listeria monocytogenes, Salmonella,</I> and <I>Vibrio cholerae</I>. All of these pathogens are most widely spread and are significant threats to human health around the world. Detailed information about each pathogen, why they are a significant concern, their mode of action, geographical locations most affected, the symptoms of infection, and a bacteria-specific gene that was selected by the team can be found below:
</H2><H5><I>What is it?</I></H5><H2><I>Campylobacter</I> is a bacteria genus with 17 known species and 6 subspecies. The two species <I>C.jejuni</I> and <I>C.coli</I> are the most commonly associated with human diseases. Their structure is mainly spiral and &quot;S&quot; shaped, or curved and rod shaped (1). <I>Campylobacter Jejuni</I> functions by arresting the cell cycle in G2 phase using Cytolethal Distending Toxin (4).
</H2><H5><I>Why are they a major concern? Which parts of the world are the most affected?</I></H5><H2>Yearly, 550 million people are infected with diarrheal diseases (1). <I>Campylobacter</I> is one of the four main causes of diarrheal diseases and is the most common cause of gastroenteritis, in this case called Campylobacteriosis, an infection in the stomach and small intestine (1). Campylobacteriosis is a zoonosis, meaning the source of its transmittance to humans is animals.
</H2><H5><I>What are the symptoms of the infection?</I></H5><H2>The infection has symptoms that last 3 to 6 days such as diarrhea, nausea, vomiting, headaches, and abdominal pain that appear 2 to 4 days after an infection (1). Infections by <I>Campylobacter</I> are usually caused by injected contaminated undercooked poultry, or contaminated water. Such infections can be fatal in young children, elderly and immunosuppressed patients, such as those with diseases like AIDs. In developing countries there is a high incidence of <I>Campylobacter</I> infections in children under two and are sometimes fatal (1). In rare cases, a <I>Campylobacter</I> infection can lead to other diseases and disorders such as hepatitis, pancreatitis, miscarriages and Guillain-Barré syndrome (2). Guillain-Barré syndrome is a neurodegenerative disorder that occurs due to the immune system attacking the peripheral nervous system, including the cranial nerves, the spinal nerves and the autonomic nervous system (3).
</H2><H5><I>Gene of choice. Why?</I></H5><H2><I>hipO</I> gene codes for the hippurate hydrolase in <I>Campylobacter jejuni</I>. It was chosen due its specificity to the organism, the lack of toxicity and previous research projects that have used this gene to detect the pathogen by PCR (5, 6).
</H2><H5><U>References:</U></H5><H2>1. Campylobacter (2018) World Health Organization. Available at: http://www.who.int/mediacentre/factsheets/fs255/en/ [Accessed April 18, 2018].</H2><H2>2. Hofreuter D (2006) Unique Features of a Highly Pathogenic Campylobacter jejuni Strain. American Society for Microbiology 74 no. 8 4694-4707.</H2><H2>3. Yuki  N &amp; Hartung  H-P (2012) Guillain–Barré Syndrome. New England Journal of Medicine 366(24):2294-2304. </H2><H2>4. Whitehouse CA (1998) Campylobacter jejuni Cytolethal Distending Toxin Causes a G2-Phase Cell Cycle Block. American Society for Microbiology 66 no. 5 1934-1940.</H2><H2>5. Ayaz, N. D., Goncuoglu, M., Cakmak, O., &amp; Erol, I. (2016). Comparison of hipO and ceuE gene based PCR assays for the detection of Campylobacter jejuni. J Clin Microbiol Biochem Technol 2 (1): 006, 8(006).</H2><H2>6. Caner, V., Cokal, Y., Cetin, C., Sen, A., &amp; Karagenc, N. (2008). The detection of hipO gene by real-time PCR in thermophilic Campylobacter spp. with very weak and negative reaction of hippurate hydrolysis. Antonie Van Leeuwenhoek, 94(4), 527. </H2><H5><I>What is it?</I></H5><H2><I>Listeria Monocytogenes</I> is a rod-shaped, food-borne, pathogenic species of gram-positive bacteria that ultimately infects both animals and humans, culminating in the disease Listeriosis. <I>L. monocytogenes</I> are characterised by their outstanding ability not only to survive in harsh conditions but also proliferate, these conditions include pH environments of 4.3-9.6, temperatures between 1-45°C and saline environments (less than 10%) (1-2). Three out of the total 13 identified serotypes of the bacterium, 1/2a, 1/2b and 4b are responsible for most of the cases of listeriosis, albeit all 13 serovars have the pathogenicity to cause listeriosis (3).</H2><H5><I>Why are they a major concern? Which parts of the world are the most affected?</I></H5><H2><I>L. Monocytogenes</I> infections do not usually lead to serious listeriosis in healthy adults, however of the susceptible population of infants, elderly and pregnant women who contract invasive listeriosis, there is a significantly high mortality rate of 20-30% (4). This high mortality rate is especially concerning considering that the bacterium persists in a diverse range of environments and foods thus triggering sporadic outbreaks all over the world. According to the World Health Organisation, as of March 2018, the largest listeria outbreak is occurring in South Africa and has seen 978 cases of laboratory-confirmed listeriosis, of which 183 have resulted in a fatal outcome (5). Due to the bacterium’s robust nature and commonness, outbreaks are a common global occurrence and are non-region specific thus making them a serious international public health threat.</H2><H5><I>What are the symptoms of the infection?</I></H5><H2>The motility of <I>L. Monocytogenes</I> is reliant on their flagella, active at relatively low temperatures and so they are able to travel from food sources such as raw or unpasteurised milk and milk products, raw and processed vegetables, fruit and meat to food-contact surfaces, especially in food-processing and handling environments. From the source, humans ingest the bacteria orally after which the bacterium travels directly from cell to cell through the intestinal mucosa where dendritic cells under Peyer’s patches are infected and promote bacterial spread to blood and lymph (2-3). The majority vulnerable population of neonates and foetuses (~40%) are infected at birth or in-utero from the mother, prompting abortions and stillbirths (6). </H2><H2>Two different forms of the disease manifest after infection by <I>L. Monocytogenes</I> depending on the general health and immune strength of the infected. Healthy adults who contract the disease may suffer a mild non-invasive gastrointestinal illness including symptoms of fever, diarrhoea and vomiting with an incubation period between 18 and 20 hours and is usually self-resolving. Infected elderly, pregnant women, cancer patients and others affected by immunosuppression are more likely to contract an invasive septicaemic or neuropathic form of the disease resulting in a combination of the following set of symptoms: fever, malaise, fatigue, abdominal pain, ataxia, seizures, meningitis. This invasive form has an incubation period of 3-90 days (2).</H2><H5><I>Gene of choice. Why?</I></H5><H2><I>Lmo0773</I> gene codes for a lmo0773 protein, which is involved in the transcription regulation. The gene was chosen as it is not involved in any toxic activities in the bacteria and is specific for <I>Listeria monocytogenes</I> (7, 8).</H2><H5><U>References:</U></H5><H2>1. Bowman JP, Bittencourt CR, &amp; Ross T (2008) Differential gene expression of Listeria monocytogenes during high hydrostatic pressure processing. Microbiology 154(2):462-475.</H2><H2>2. Roberts A &amp; Wiedmann M (2003) Pathogen, host and environmental factors contributing to the pathogenesis of listeriosis. Cellular and Molecular Life Sciences CMLS 60(5):904-918.</H2><H2>3. World Health Organisation (2012) Foodborne Pathogenic Microorganisms and Natural Toxins. Food and Drug Administration.</H2><H2>4. Anonymous (2004) Risk assessment of Listeria monocytogenes in ready-to-eat foods. in Technical Report (World Health Organisation, Rome, Italy).</H2><H2>5. Anonymous (2018) Listeriosis- South Africa. Emergencies preparedness, response.</H2><H2>6. Farber J &amp; Peterkin P (1991) Listeria monocytogenes, a food-borne pathogen. Microbiological reviews 55(3):476-511.</H2><H2>7. Chatterjee, S. S., Hossain, H., Otten, S., Kuenne, C., Kuchmina, K., Machata, S., ... &amp; Hain, T. (2006). Intracellular gene expression profile of Listeria monocytogenes. Infection and immunity, 74(2), 1323-1338.</H2><H2>8. Glaser, P., Frangeul, L., Buchrieser, C., Rusniok, C., Amend, A., Baquero, F., ... &amp; Charbit, A. (2001). Comparative genomics of Listeria species. Science, 294(5543), 849-852.</H2><H5><I>What is it?</I></H5><H2><I>Salmonella</I> is a bacteria that causes diarrheal diseases in organisms. It is a gram negative rods and belongs to the Enterobacteriaceae family. There are two major species, <I>Salmonella bongori</I> and <I>Salmonella enterica</I>, and over 2500 different serotypes have been identified within the two species. Of the various serotypes, <I>Salmonella enterica</I> serotype Enteritidis and <I>Salmonella enterica</I> serotype Typhimurium are two of the most important and prevalent types that affect human around the world. <I>Salmonella</I> is a hardy bacteria that can survive under unfavorable circumstances: it can survive several weeks in a dry environment and can survive several months in water (1). </H2><H5><I>Why are they a major concern? Which parts of the world are the most affected?</I></H5><H2>According to WHO, 1 in 10 people fall ill each year because of foodborne diseases, and salmonella is one of the four major causes of diarrheal diseases throughout the world (1). <I>Salmonella</I> infection is usually caused by consuming raw or undercooked meat, poultry, eggs, egg products or contaminated raw fruits and vegetables.<I> Salmonella</I> can be exterminated by cooking and pasteurization (2).</H2><H5><I>What are the symptoms of the infection?</I></H5><H2>Depending on the serotype of <I>Salmonella</I> infected, two types of illnesses might result, either nontyphoidal salmonellosis or typhoid fever. For nontyphoidal salmonellosis, the illness is developed through the penetration and passage of <I>Salmonella</I> organisms from gut lumen into epithelium of small intestine. Evidence have shown that enterotoxin (toxin affecting the intestines) may be produced within enterocytes (a cell of the intestinal lining). For typhoid fever, the illness is developed through the penetration and passage of typhoid Salmonella organisms from gut lumen into epithelium of small intestine and further into the bloodstream. Once the bacteria enter the bloodstream, it may be carried to other sites of the body. There is also evidence that enterotoxin may be produced within the cells of the intestinal lining (3). </H2><H2>The disease caused by salmonella is called Salmonellosis. It is usually characterized by acute fever, abdominal pain, diarrhea, nausea, and vomiting. The symptoms of salmonellosis is relatively mild: some people infected don’t exhibit any signs of infection while others can recover without specific medical treatment (2). However, for those with weaker immune system, especially in children or the elderly, the associated dehydration can become severe and life-threatening. Symptoms of the disease occurs 6–72 hours after the ingestion of <I>Salmonella</I>, and the illness usually lasts 2–7 days. </H2><H5><I>Gene of choice. Why?</I></H5><H2>The <I>invA</I> gene codes for the invasion gene involved in the invasion of the intestinal epithelium cells. This gene is specific to all <I>Salmonella</I> serotypes and alone does not cause toxicity. Previous studies have also utilized this gene for detection of <I>Salmonella</I> by PCR and LAMP (4-6).</H2><H5><U>References:</U></H5><H2>1. Salmonella (non-typhoidal) (2018) World Health Organization. Available at: http://www.who.int/mediacentre/factsheets/fs139/en/ [Accessed April 18, 2018].</H2><H2>2. Salmonella infection - Symptoms and causes (2018) Mayo Clinic. Available at: https://www.mayoclinic.org/diseases-conditions/salmonella/symptoms-causes/syc-20355329 [Accessed April 18, 2018].</H2><H2>3. Center for Food S &amp; Applied N, The bad bug book.</H2><H2>4. Rahn K, et al. (1992) Original article: Amplification of an invA gene sequence of Salmonella typhimurium by polymerase chain reaction as a specific method of detection of Salmonella. Molecular and Cellular Probes 6:271-279.</H2><H2>5. Chiu CH &amp; Ou JT (1996) Rapid Identification of Salmonella Serovars in Feces by Specific Detection of Virulence Genes, invA and spvC, by an Enrichment Broth Culture-Multiplex PCR Combination Assay.  (AMERICAN SOCIETY FOR MICROBIOLOGY, United States), p 2619.</H2><H2>6. Hara-Kudo, Y., Yoshino, M., Kojima, T., &amp; Ikedo, M. (2005). Loop-mediated isothermal amplification for the rapid detection of Salmonella. FEMS microbiology letters, 253(1), 155-161.
</H2><H5><I>What is it?</I></H5><H2><I>Vibrio cholerae</I> is a comma-shaped gram-negative bacteria responsible for the pandemic disease cholera (1). Out of numerous pathogenic and non-pathogenic strains, the most wide sweeping one responsible for the the disease is the <I>V. cholerae</I> serotype O1 El Tor N16961 strain (2). Containing a genomic island of pathogenicity and lysogenized with phage DNA, <I>V. cholerae</I> is made pathogenic by the integration of the genes of a virus into the bacterial genome. <I>V. cholerae</I> serogroup O1 does not have a polysaccharide capsule, but serogroup O139 does contain a polysaccharide capsule made up of &quot;N-acetylglucosamine, N-acetylquinovosamine (QuiNAc), galacturonic acid (GalA), and galactose and two residues of 3,6-dideoxyxylohexose (Xylhex)&quot; (3).
</H2><H5><I>Why are they a major concern? Which parts of the world are the most affected?</I></H5><H2>Cholera is an acute diarrhoeal disease that can cause death within hours in the absence of treatments (3). According to WHO data, each year there are 1.3 million to 4.0 million cases of cholera, and 21000 to 143000 deaths worldwide due to cholera (4).  Although Cholera is rare in industrialized nations such as the United States, globally, Cholera cases have increased steadily since 2005 and many places including Africa, Southeast Asia, and Haiti are still affected (5). It is a serious public health threat especially in economically reduced areas with high population density and lack of proper water and sewage treatment facilities.</H2><H5><I>What are the symptoms of the infection?</I></H5><H2><I>V. cholerae</I> infects the intestine and rapidly increases the production of mucous. This causes diarrhea and vomiting which lead to extreme dehydration and, if not treated, death in 50-70% of the cases. Most people infected with <I>V. cholerae</I> do not develop any symptoms. Among people who develop symptoms, most have mild symptoms, while few develop acute watery diarrhoea associated with severe dehydration.  Transmission is usually through the fecal-oral route of contaminated food or water caused by poor sanitation. After <I>V. cholera</I> enters the human body through the ingestion of contaminated food or water, it enters the intestine, embeds itself in the villi of absorptive intestinal cells, and releases the cholera toxin. </H2><H5><I>Gene of choice. Why?</I></H5><H2>The <I>gbpA</I> gene in Vibrio cholerae codes for the N-acetyl glucosamine-binding protein A (GbpA), which is a chitin-binding protein involved in the attachment of <I>V. cholerae</I> to environmental chitin surfaces and human intestinal cells (6). The gbpA gene has been found to be consistently present and highly conserved in <I>V. cholerae. GbpA</I> has been used as a target gene for the species-specific detection of <I>V. Cholerae</I>  O1, O139, Non-O1/Non-O139 (6).</H2><H5><U>References:</U></H5><H2>1. Heidelberg, J.F., Eisen, J.A., Nelson, W.C., et al. DNA sequence of both chromosomes of the cholera pathogen Vibrio cholerae. Nature. (2000);406 (6795), 477-483.</H2><H2>2. Center for Disease Control, Coordinating Center for Infectious Diseases / Division of Bacterial and Mycotic Diseases, October 6, 2005. Available at: http://www.cdc.gov/ncidod/dbmd/diseaseinfo/cholera_g.htm. </H2><H2>3. Preston, L.M., XU, Q., Johnson, J.A., Joseph, Maneval Jr., D.R., Husain, K., Reddy, G.P., Bush, C.A. AND Morris Jr., J.G. Preliminary Structure Determination of the Capsular Polysaccharide of Vibrio cholerae O139 Bengal Al1837. Journal of Bacteriology. (Feb. 1995); p. 835–838/</H2><H2>4. Cholera. World Health Organization(2017). Available at: http://www.who.int/mediacentre/factsheets/fs107/en/.</H2><H2>5. Cholera - Vibrio cholerae infection. Centers for Disease Control and Prevention(2016). Available at: https://www.cdc.gov/cholera/general/index.html. </H2><H2>6. Vezzulli L, et al. (2015) gbpA as a novel qPCR target for the species-specific detection of Vibrio cholerae O1, O139, non-O1/non-O139 in environmental, stool, and historical continuous plankton recorder samples. PloS one 10(4):e0123983.</H2></CENTER></ARTICLE></DIV><DIV id="biology" class="tabcontent"><ARTICLE><CENTER><H2>For the creation of a fast, reliable and robust detection device, two main branches of methodology were considered: biological detection of the pathogen and the engineering of the device itself that would allow for multiple pathogen detection and would yield clear results. For the biological detection, gene amplification and colorimetric and fluorescent dyes that change color in the presence of DNA were considered. The three main techniques chosen were Polymerase Chain Reaction (PCR), Loop-mediated Isothermal Amplification (LAMP) and Recombinase Polymerase Amplification (RPA). PCR was used as a control to compare and assess the quality of the amplification. LAMP and RPA were used as potential means for rapid and reliable DNA amplification. Each technique was assessed individually and its advantages and disadvantages in comparison to the other alternatives were evaluated. For the engineering of the device itself, a novel microfluidics technique was considered. The methodology presents a review of this technique, assessing its compatibility with LAMP and RPA.
	</H2><H4><CENTER><U><B>Polymerase Chain Reaction</B></U></CENTER></H4><H2>Polymerase chain reaction (PCR) is a technique used to amplify a specific DNA segment. The targeted sequence is duplicated in every cycle which leads to an exponential increase in the number of copies, producing millions after two dozen repetitions. It is seen as an easy, cheap and reliable way to produce large quantities of specific DNA sequences.</H2><H2>PCR is widely used in molecular biology due to its several advantages. One of its strongest advantages is its high sensitivity (2). A single copy of the targeted sequence is easily detectable and duplicated. DNA from a single cell provides enough material to produce billions of copies of the specified sequence in a short amount of time. Typically, the cycle is repeated 25 to 35 times which produces tens of millions of copies in 2-4 hours out of just one or two DNA copies. Moreover, PCR is relatively cheap to run (3). However, PCR does have several disadvantages when compared to LAMP and RPA. The high sensitivity of PCR can be a disadvantage due to the possibility of a contaminant DNA strand (possibly from a previous experiment) leading to the production of unwanted duplicates. PCR also requires expensive and sophisticated thermocyclers to ramp the temperature up and down. Furthermore, it is less resistant to inhibitors present in complex samples when compared to other techniques such as LAMP.
		</H2><H2>These properties make it that PCR was not the method chosen for this project due to the longer time needed to run it compared to LAMP, the thermocycling step, requiring multiple temperature, as well as higher costs associated with it. Instead, it served as a control procedure for confirming that the faster processes (LAMP and RPA) are working correctly. PCR remains one of the best method for testing due to its high reliability, which makes it one of the most commonly used methods for DNA amplification today (3).</H2><H4><CENTER><U><B>Loop-mediated Isothermal Amplification</B></U></CENTER></H4><H2>Loop-mediated Isothermal Amplification (LAMP) is a highly specific, efficient and rapid DNA amplification technique that uses 4-6 primers that bind to 6-8 distinct regions of target DNA. This technique was shown to be more specific than colony PCR without the need for heat lysis or centrifugation steps (4).
</H2><H2>Unlike PCR, LAMP is an isothermal process which does not require the use of expensive thermocyclers. This technique allows for simple and easy detection of selected genes as it produces stem-loop structures of DNA which enable a more rapid selection process. The time required for the assay is typically less than 30 minutes, which is substantially lower when compared to PCR. LAMP also provides a higher yield of the product: on average 10-20 µg of DNA is obtained, which is 50-100 times more than the 0.2 µg obtained from PCR. Therefore, detection with a colorimetric or fluorescent dye that binds to DNA is far more feasible. Furthermore, LAMP uses 4-6 primers which gives its specificity for its target sequence. Thus it is not affected by the alternative nucleic acid amplification methods of background interference (4, 6). However, LAMP does involve several disadvantages, especially when compared to another promising technique RPA. One of the main issues is that several parameters in the technique have to be optimized to achieve high-standard assay performance, which involves assay temperature, enzyme concentration, primer design, and concentration among others. It is more time consuming to design the primers as they are more complex and longer than PCR or RPA primers. Finally, as opposed to both PCR and RPA it is challenging to multiplex LAMP reaction (4-6).
</H2><H4><CENTER><U><B>Recombinase Polymerase Amplification</B></U></CENTER></H4><H2>Recombinase polymerase amplification (RPA) is an isothermal DNA amplification method in which amplification reaction can be completed in 10 to 20 minutes at 24°C to 45°C.
</H2><H2>Among isothermal amplification methods RPA is considered the most suitable method to be used in point of care testing (POCT). The RPA process employs three enzymes-a recombinases, a single-stranded DNA-binding protein (SSB) and a strand-displacing polymerase. The recombinase is capable of pairing oligonucleotide primers with a homologous sequence in the target DNA. SSB then binds to the displaced strand of DNA and prevents the dissociation of primers. Finally, the  strand displacing polymerase begins DNA synthesis where the primer has bound to the target DNA. With the use of two opposing primers, exponential amplification of the target sequence with RPA can be achieved at a constant temperature in 10–20 min. The RPA product can be measured in real-time using different probes with fluorescence detection device (9).
</H2><H2>RPA does not use thermal cycling and works by producing amplicons incubated at 37 to 42-degree Celsius. When compared to LAMP it requires less complicated primer design. It is also suitable for detecting pathogens because it has resistance to inhibitors and has a lesser sensitivity to the ambient temperature (11). Moreover, in many cases, RPA does not need rigorous sample preparation. Laura C Bonney et al. prepared samples by aliquoting and refrigerating the urine that was tested with RPA later (12). Another advantage of the RPA assay is that the reaction can be conducted in a single tube and result in ‘real-time by the including a fluorescent probe (13).
</H2><H4><CENTER><U><B>Microfluidics</B></U></CENTER></H4><H2>Microfluidics is the interdisciplinary technique used for achieving multiplexing and rapid high-throughput screening. The two sub-words -<I>fluidics</I> and <I>micro</I>- suggest the use of fluids and small size of the device, which satisfies the goals of any low-cost, field-maneuverable, diagnostic tool. Microfluidic devices are often called biochips, or lab-on-a-chip, because of their history in advancing bio-assay techniques based on features which can be used for continuous sampling and on-the-ground testing of the presence of toxins (15-16).
</H2><H2>As one considers using LAMP or RPA as techniques of amplifying DNA to be detected inside these small devices, microfluidic chambers are ideal. Currently developing research suggests improved methods for detecting and diagnosing pathogens with a fluorescent output. Zeming et al. have developed fluorescent label-free detection on a pillar array that uses microbeads to stabilize the proteins for analysis while decreasing cost and increasing efficiency of identifying a result (16). The drawback of this new approach is that it has only been tested on albumin and vesicles, although the principles suggest it should work with DNA as well. Another group has used microfluidics coupled with PCR to achieve simultaneous detection of the influenza virus gene, Anthrax PA gene, and botulinum toxin, all under 10 minutes, while also being built for field deployment (17). We see a possible area for improvement here. Our use of LAMP or RPA is faster and more reliable than PCR, but RPA does not require the heating and cooling of the DNA chamber (18).
	</H2><H2>The previous methodology for the detection of Shiga-toxin relied on inserting the DNA inside the device, which was then amplified by LAMP (heated up to 80 degrees), and then detected by concentration-dependent fluorescence (molecular beacon probes) under UV light. The new approach has implemented the microfluidic chambers, in the sense that channels for different pathogens would be independent of one another, providing a continuous and versatile sampling.
</H2><H5><U>References:</U></H5><H2>1. Wassenegger, Michael. &quot;Advantages and disadvantages of using PCR techniques to characterize transgenic plants.&quot; Molecular biotechnology 17.1 (2001): 73-82.
 </H2><H2>2. National Collaborating Centre for Women's and Children's Health (UK). Bacterial Meningitis and Meningococcal Septicaemia: Management of Bacterial Meningitis and Meningococcal Septicaemia in Children and Young People Younger than 16 Years in Primary and Secondary Care. London: RCOG Press; 2010. (NICE Clinical Guidelines, No. 102.) Appendix I, Cost effectiveness of polymerase chain reaction for diagnosis in suspected meningococcal disease. Available from: https://www.ncbi.nlm.nih.gov/books/NBK83068/
</H2><H2>3. Strom C, Rechitsky S, Wolf G, Verlinsky Y (1994) Reliability of polymerase chain reaction (PCR) analysis of single cells for preimplantation genetic diagnosis. Journal of Assisted Reproduction and Genetics 11:55-62.
</H2><H2>4. Notomi T, et al. (2000) Loop-mediated isothermal amplification of DNA. Nucleic acids research 28(12):e63-e63.</H2><H2>5. Dieffenbach C, Lowe T, &amp; Dveksler G (1993) General concepts for PCR primer design. PCR Methods Appl 3(3):S30-S37.
</H2><H2>6. Wang X, Seo DJ, Lee MH, &amp; Choi C (2014) Comparison of Conventional PCR, Multiplex PCR, and Loop-Mediated Isothermal Amplification Assays for Rapid Detection of Arcobacter Species. Journal of Clinical Microbiology 52(2):557-563.</H2><H2>7. Team NYU Abu Dhabi: Protocols (2017) Available at: https://2017.igem.org/Team:NYU_Abu_Dhabi/Protocols.
</H2><H2>8. Gao W, et al. (2016) Recombinase Polymerase Amplification-Based Assay for Rapid Detection of Listeria monocytogenes in Food Samples. Food Analytical Methods10(6):1972–1981.
</H2><H2>9. Wang, Jianchang, et al. “Recombinase Polymerase Amplification Assay-A Simple, Fast and Cost-Effective Alternative to Real Time PCR for Specific Detection of Feline Herpesvirus-1.” PLOS ONE, Public Library of Science, doi.org/10.1371/journal.pone.0166903. </H2><H2>10. Recombinase Polymerase Amplification, or RPA, is the breakthrough, isothermal replacement to PCR RPA - The versatile replacement to PCR. Available at: https://www.twistdx.co.uk/en/rpa.</H2><H2>11. Chao, Chien-Chung, et al. “Development of Recombinase Polymerase Amplification Assays for Detection of Orientia Tsutsugamushi or Rickettsia Typhi.” PLOS Neglected Tropical Diseases, vol. 9, no. 7, 2015, doi:10.1371/journal.pntd.0003884. </H2><H2>12. Bonney, Laura C., et al. “A Recombinase Polymerase Amplification Assay for Rapid Detection of Crimean-Congo Haemorrhagic Fever Virus Infection.” PLOS Neglected Tropical Diseases, Public Library of Science, doi.org/10.1371/journal.pntd.0006013.</H2><H2>13. Moore, Matthew D, and Lee-Ann Jaykus. “Recombinase Polymerase Amplification: a Promising Point-of-Care Detection Method for Enteric Viruses.” Future Virology, vol. 12, no. 8, 2017, pp. 421–429., doi:10.2217/fvl-2017-0034.</H2><H2>14. Kirby, B.J. (2010). Micro- and Nanoscale Fluid Mechanics: Transport in Microfluidic Devices. Cambridge University Press.</H2><H2>15. Whitesides GM (2006) The origins and the future of microfluidics. Nature 442(7101):368–373.</H2><H2>16. Zeming KK, Salafi T, Shikha S, &amp; Zhang Y (2018) Fluorescent label-free quantitative detection of nano-sized bioparticles using a pillar array. Nature communications 9(1):1254.</H2><H2>17. Saito M, et al. (2018) Field-deployable rapid multiple biosensing system for detection of chemical and biological warfare agents. Microsystems &amp; Nanoengineering 4:17083.</H2><H2>18. Wan L, et al. (2017) A digital microfluidic system for loop-mediated isothermal amplification and sequence specific pathogen detection. Scientific Reports 7(1):14586.</H2></CENTER></ARTICLE></DIV><DIV id="engineering" class="tabcontent"><ARTICLE><CENTER><H2>Access to safe and nutritious food is key to good health and development. Although food regulations already exist throughout most jurisdictions, there is still a great need to reduce the prevalence of foodborne diseases (FBD). An estimated 600 million people fall ill from FBDs every year, with about 420 000 dying from them (1). Such statistics reveal a great need for better FBD prevention and detection mechanisms. This is especially true for small-scale businesses and customers, which need quick, easy to use, reliable and affordable FBD detection devices.
</H2><H2>Devices that can detect specific nucleic acid sequences can be tailored to detect a specific DNA sequence associated with a particular FBD. Unfortunately, most of the devices remain in the prototype stage and are not available to the average consumer or are very costly to use. Even worse, of the available devices, most of them fail to meet all the necessary criteria for widespread implementation in the prevention of FBDs. Taking the increasing need of rapid tests in low-resource settings, WHO has summarized the ideal features of such tests under the acronym ASSURED (Affordable, Sensitive, Specific, User-friendly, Rapid, Robust, Equipment-free and Delivered to those who need it) (2). An example of a device failing to meet the ASSURED criteria is the recently published “mobile platform for multiplexed detection and differentiation of disease-specific nucleic acid sequences” prototype, which is capable of performing ten simultaneous DNA amplification through Loop-mediated Isothermal Amplification (LAMP) to detect target DNA sequences (3). This device is then paired with a smartphone for data interpretation, and the results are displayed and reported to an online database (3). However, the prototype suffers several drawbacks in its implementation. Although it can provide results relatively quickly (~30 min), it requires sample preparation by centrifugation and heating, which are not readily available and hard to perform in the field. These manual steps significantly reduce the ease of use and portability. Furthermore, the cost of the components for the instrument amounts to approximately 550 USD, which consumers may not be willing or able to pay (3). Our project seeks to address these shortcomings.
</H2><H2>Last year, the NYU Abu Dhabi 2017 team created a working prototype to detect Shiga toxin-producing <I>E. coli</I> in a sample of food. Similar to other prototypes in literature, it used LAMP to amplify the genetic material and reported the results by the fluorescence of the sample. However, the prototype required the consumer to prepare their samples outside the reaction vessel by mixing them in the tubes and then transferring them directly into wells, it was only streamed to detect 1 pathogen and made use of only LAMP technique. Therefore, this year’s team improved on the work already achieved by iGEM 2017 team. This year’s team made sure to  make a device that meets the WHO-designed ASSURED criteria. For Affordable the team minimized the costs by using an inexpensive yet robust material to build the device and offers both options the use of the cheaper colorimetric dyes and the more expensive but reliable fluorescent dyes to detect the DNA amplification. For Sensitive and Specific the team assed PCR, LAMP and RPA and compared their sensitivity and specificity results, allowing the customer to choose which technique they prefer based on their needs. Rapid, Equipment-free the team to assessed LAMP and RPA techniques and showed that both are sensitive, reliable, fast DNA amplification methods that only require a constant supply of heat to maintain a specific temperature, which the team integrated in their device with the option to adjust the temperature based on the technique of choice. For Affordable, Specific and Robust the team applied microfluidics into the chip to provide channels for detection of distinct pathogens and used a durable material that allow the manufacture of low cost and high impact resistance chips. For Affordable, Robust and User-Friendly, the device was designed to enhance its portability, efficiency, cost, and ease of use. For Delivered the team expanded the number of FBDs the device can detect, thus allowing the device to be of use in all parts of the world.
</H2><H5><U>References:</U></H5><H2>1. Havelaar AH, et al.(2015) World Health Organization Global Estimates and Regional Comparisons of the Burden of Foodborne Disease in 2010. PLOS Medicine 12(12):e1001923.
</H2><H2>2. A guide to aid the selection of diagnostic tests (2017) World Health Organization. Available at: http://www.who.int/bulletin/volumes/95/9/16-187468/en/.
</H2><H2>3. Chen W, et al.(2017) Mobile Platform for Multiplexed Detection and Differentiation of Disease-Specific Nucleic Acid Sequences, Using Microfluidic Loop-Mediated Isothermal Amplification and Smartphone Detection. Analytical Chemistry 89(21):11219-11226.
</H2></CENTER></ARTICLE></DIV></DIV></DIV><FOOTER class="footer"><DIV class="columns"><H3>Sponsors:</H3></DIV></FOOTER><DIV id="app"><NAV style="padding-bottom:1px;" class="navbar navbar-expand-lg fixed-top navbar-light bg-light"><DIV class="collapse navbar-collapse" id="navbarSupportedContent"><UL class="navbar-nav mr-auto"><LI class="nav-item"><A class="nav-link" href="https://2018.igem.org/Team:NYU_Abu_Dhabi">Home</A></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
					 People
				 </A><DIV class="dropdown-menu" aria-labelledby="navbarDropdown"><A class="dropdown-item" href="https://2018.igem.org/Team:NYU_Abu_Dhabi/Team">Team</A><A class="dropdown-item" href="https://2018.igem.org/Team:NYU_Abu_Dhabi/Attributions">Attributions</A><A class="dropdown-item" href="https://2018.igem.org/Team:NYU_Abu_Dhabi/Sponsors">Sponsors</A><A class="dropdown-item" href="https://2018.igem.org/Team:NYU_Abu_Dhabi/About_Us">About Us</A></DIV></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
					 Project
				 </A><DIV class="dropdown-menu" aria-labelledby="navbarDropdown"><A class="dropdown-item" href="https://2018.igem.org/Team:NYU_Abu_Dhabi/Description">Description</A><A class="dropdown-item" href="https://2018.igem.org/Team:NYU_Abu_Dhabi/Demonstrate">Demonstration</A><A class="dropdown-item" href="https://2018.igem.org/Team:NYU_Abu_Dhabi/Results">Results</A><A class="dropdown-item" href="https://2018.igem.org/Team:NYU_Abu_Dhabi/Parts">Parts</A><A class="dropdown-item" href="https://2018.igem.org/Team:NYU_Abu_Dhabi/Safety">Safety</A><A class="dropdown-item" href="https://2018.igem.org/Team:NYU_Abu_Dhabi/InterLab">Interlab</A><A class="dropdown-item" href="https://2018.igem.org/Team:NYU_Abu_Dhabi/Model">Modelling</A><A class="dropdown-item" href="https://2018.igem.org/Team:NYU_Abu_Dhabi/Medal_Requirements">Medal Requirements</A></DIV></LI><LI class="nav-item"><A class="nav-link" href="https://2018.igem.org/Team:NYU_Abu_Dhabi/Hardware">Design</A></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
					 Lab notebooks
				 </A><DIV class="dropdown-menu" aria-labelledby="navbarDropdown"><A class="dropdown-item" href="https://2018.igem.org/Team:NYU_Abu_Dhabi/Biology">Biology</A><A class="dropdown-item" href="https://2018.igem.org/Team:NYU_Abu_Dhabi/Ideation">Ideation</A><A class="dropdown-item" href="https://2018.igem.org/Team:NYU_Abu_Dhabi/Engineering">Engineering</A></DIV></LI><LI class="nav-item"><A class="nav-link" href="https://2018.igem.org/Team:NYU_Abu_Dhabi/Entrepreneurship">Entrepreneurship</A></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
					 Human practices
				 </A><DIV class="dropdown-menu" aria-labelledby="navbarDropdown"><A class="dropdown-item" href="https://2018.igem.org/Team:NYU_Abu_Dhabi/Public_Engagement">Education and Public Engagement</A><A class="dropdown-item" href="https://2018.igem.org/Team:NYU_Abu_Dhabi/Human_Practices">Integrated Human Practices</A><A class="dropdown-item" href="https://2018.igem.org/Team:NYU_Abu_Dhabi/Home_Outreach">Home Country Outreach</A><A class="dropdown-item" href="https://2018.igem.org/Team:NYU_Abu_Dhabi/Social_Media">Social Media</A></DIV></LI><LI class="nav-item"><A class="nav-link" href="https://2018.igem.org/Team:NYU_Abu_Dhabi/Collaborations">Collaborations</A></LI></UL></DIV></NAV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>