Team:JNFLS/Protocols
Home
Team
Team Member
Collaboration
PROJECT
Background
Design
Protocols
Model
Results
Interlab
Demonstrate
Notebook
Achievement
Attribution
PARTS
SAFETY
HUMAN PRACTICE
Silver Human Practice
Integrated And Gold
Education&Engagement
Judging Form
Protocols
PCR truncation of HCVC gene
Primers:
Enzyme digestion of pcoldII vector plasmid
select 1μL plasmid
add 5ul green buffer
add 1ul Xhol
add 1ul PstI
add distilled water to 20μL
water bath at 37 degrees for 10 min
Ligation
Bacterial transformation
Remove cells from -80°C freezer and thaw in hand
Add 1-5 μl containing 1 pg-100 ng of plasmid DNA to the cell mixture. Carefully flick the tube 4-5 times to mix cells and DNA. Do not vortex
Place the mixture on ice for 2 minutes. Do not mix
Heat shock at exactly 42°C for exactly 30 seconds. Do not mix.
Place on ice for 2 minutes. Do not mix.
Pipette 250 μl of room temperature SOC into the mixture. Immediately spread 50-100 μl onto a selection plate and incubate overnight at 37-42°C. NOTE: Selection using antibiotics other than ampicillin may require some outgrowth before plating on selective media. Colonies develop faster at temperatures above 37°C, however some constructs may be unstable at elevated temperatures.
Colony PCR
Conditions：
Gel Recycling
Run electrophoresis separation of the target DNA fragment
Cut off the target DNA fragments, as far as possible the unwanted cut off
The cut of the glue into the 1.5m | plastic centrifuge tube, said the quality of the plastic
Add the same amount of Binding Buffer(XP2), that is, add 0.3m| liquid if 0.3g is weighed
Heat in a 50-60 degree bath for a few minutes until all the glue is melted, stirring to mix.
Put the HiBind DNA Mini Column into the 2ml collection tube
Add the HiBindDNA Mini Column with less than 700ul from the DNA solution in step 5
Centrifuge 60s at 12000rpm at room temperature
Discard the waste liquid and recycle the collection pipe
Repeat steps 7-9 until all samples are transferred to column
Add 300ul Binding Buffer
Centrifuge 60s at 12000rpm at room temperature
Discard the waste liquid and recycle the collection pipe
Add the 700ul SPW Wash Buffer
Centrifuge 60s at 12000rpm at room temperature
Discard the waste liquid and recycle the collection pipe once again in steps 14-16
Centrifuge the empty HiBind DNA Mini Column 1 2000rmp for 2 minutes
The Hibind DNA Mini Column was transferred to 1.5m| plastic centrifuge tube
Add 15-30ul Elution Buffer to the center of the membrane
Let sit at room temperature for 2 minutes
Centrifugation for 12000rmp at room temperature for 1 minute
plasmid extraction from positive clonesusing an extraction kit
Decant or aspirate and discard the culture media.Centrifuge at 10,000x g for 1 minute at room temperature
Add 250 uL Solution I/RNase A. Vortex or pipet up and down to mix thoroughly.Complete resuspension of cell pellet is vital for obtaining good yields.
Transfer suspension into a new 15 mL microcentrifuge tube.
Add250 uL Solution IL Invert and gently rotate the tube several times to obtain a dear lysate. A 2-3minute incubation may be necessary.
Add 350uL Solution I Immediately invert several times until a flocculent white precipitate form.
Centrifuge at maximum speed (213,000 x g) for 10 minutes. A compact white pellet will form. Promptly proceed to the next step.
Insert a HiBinde DNA Mini Column into a 2 mL Collection Tube.
Transfer the cleared supernatant from Step 8 by CAREFULLY aspirating it into the HiBind* DNA Mini Column. Be careful not to disturb the pellet and that no cellular debris is transferred to the HiBind" DNA Mini Column.
Centrifuge at maximum speed for 1 minute.
Discard the filtrate and reuse the collection tube.
Add 500 uL HBC Buffer
centrifuge at maximum speed for 1 minute.
Discard the filtrate and reuse collection tube.
Add 700uL DNA Wash buffer.
Centrifuge at maximum speed for 1 minute.
Discard the filtrate and reuse the collection tube.
Centrifuge the empty HiBind" DNA Mini Column for 2 minutes at maximum speed to dry the column matrix.
Transfer the HiBind DNA Mini Column to a clean 1.5 mL microcentrifuge tube.
Add 30100 uL Elution Buffer or sterile deionized water directly to the center of the column membrane.
Let sit at room temperature for 1 minute.
Centrifuge at maximum speed for 1 minute. Note: This represents approximately 70% of bound DNA. An optional second elution will yield any residual DNA, though at a lower concentration
Store DNA at -20degrees.
Purification of recombinant protein with His tag
1. buffer preparation:
Lysis Buffer 1 (under native condition)
0.5mM Tris-HCl
0.5M NaCl
5%(w/v) glycerol
10mM Imidazole
100mg(1mg/ml) lysozyme (Purification of 6xHis-tagged proteins from E.coli)
1% Nonidet P40 (NP40 = Igepal CA-630)
0.25% Tween 20 （or Triton 100）
0.02% NaN3 (Optional)
2 tablets of protease inhibitor cocktail (EDTA free, Recommended)
200 μg(2μg/ml) RNase A (Optional) 1mg (10μg/ml) DNase1 (Optional)
50 mM NaF (Optional)
1mM Na3VO4 (Optional)
+ ddH2O to 100ml, adjust pH to 8.0 using NaOH
Lysis Buffer 2(under denature condition)
50 mM Tris-Cl
8 M Urea (or 6 M Gu-HCl)
10mM Imidazole
0.05% Tween 20
Adjust pH to 8.0 using NaOH
Dialysis/Tev cleavage Buffer
50 mM Tris-Cl, pH8.0
100 mM NaCl (depends on solubility of protein)
2.5% glycerol
0.5mM EDTA
0.5mM DTT or TCEP
Buffer A：
50mM Tris-HCl
0.5M NaCl
5% glycerol
0.05% Tween 20
Adjust pH to 8.0 using high-concentration HCl
Buffer B（washing buffer）：
li50mM Tris-Cl /li
0.5M NaCl 5% glycerol
0.05% Tween 20
500mM imidazole
Adjust pH to 8.0 using high-concentration HCl
Buffer C（imidazole gradient washing buffer）：
Prepare with buffer A and buffer B at different ratio: A 90ml, B 10ml
Sample treatment：
fully suspend centrifuge-collected bacteria at the ratio wet bacteria per gram/ 4~5ml Lysis Buffer，and place overnight in -80℃ fridge
thaw the bacteria at 4℃ and suspend the bacteria
break the bacteria in two turns, which contain 5s of ultrasonic breaking, 5s of cooling for each cycle and run 10min per turn, at 400W.
centrifuge at 4℃, 12000rpm for 30min，collect supernatant liquid, and adjust the pH with NaOH to 8.0
take 1ml Ni-NTA agarose gel and place it into 15ml centrifuge tube, then centrifuge at 2000 rpm for 2 min
take the supernatant liquid；add 5ml Lysis Buffer, centrifuge at low speed, then take the supernatant liquid.
add4-5 ml bacteria lysate，and rotate at 4℃ at low speed for 1h
Load the chromatographic column
Wash the chromatographic column with buffer C to wash off protein in the column
Improvement on purification method
Imp1: Add 1% N-Lauryol-Sarcosine (weak denaturant) into buffer for purification. Other procedures and reagents are the same as those shown above.
Imp2: Dialyze the purified protein using 3k dialysis membrane (the buffer is pbs 50% glycerol)
Imp3: Dilute dialyzed protein with 50 mM tris and 100mM NaCl with the proportion of 1：1
SDS-PAGE
Prepare the separation gel (10%). Mix in the following order:
After adding TEMED and APS to the SDS-PAGE separation gel solution, the gel will polymerize quickly, so add these two reagents when ready to pour.
Pour gel, leaving ∼2 cm below the bottom of the comb for the stacking gel. Make sure to remove bubbles.
Layer the top of the gel with isopropanol. This will help to remove bubbles at the top of the gel and will also keep the polymerized gel from drying out.  o In ∼30 min, the gel should be completely polymerized.
Remove the isopropanol and wash out the remaining traces of isopropanol with distilled water.
Prepare the stacking gel (4%). Mix in the following order:
Pour stacking gel on top of the separation gel.
Add combs to make wells. In ∼30 min, the stacking gel should become completely polymerized.
Clamp gel into apparatus, and fill both buffer chambers with gel running buffer according to the instructions for the specific apparatus.
Load samples and molecular mass protein markers into wells for separation by electrophoresis.
Aptamer & Rolling PCR experiment
Sequences needed
Reagents:
Phi29 DNA polymerase Biomics Co., LTD T4 DNA Ligase Biomics Co., LTD 10×BSA(0.05%) ：NEB Co., LTD 10×T4 DNA ligase buffer and 10×BSA (0.05%)）and dNTPs: Dalianbao Biotech Co., LTD 10×annealing buffer： 500mM NaCl，100mM Tris-HCl(PH8.0) 1mM EDTA NEB 10X phi29 DNA Polymerase Reaction Buffer：500 mM Tris-HCl 、100 mM MgCl2 、100 mM (NH4)2SO4 、40 mM DTT 、(pH 7.5 @ 25°C) TAKARA 10X T4 DNA Ligase Reaction Buffer ：500 mM Tris-HCl 、100 mM MgCl2 、10 mM ATP 、100 mM DTT 、(pH 7.5 @ 25°C)
Examination of the reaction of designed primer, probe, and cDNA
Add 2.5 µL 10 µM Aptamer Circle Probe, 2.5 µL 0.1 µM cDNA, 1uL 10×annealing buffer, 4µL ddH2O to the 10 uL reaction system，anneal at 95 °C for 10 min, then
Let cool naturally. cDNA and aptamer are not added in the negative control group. Aptamer is added in the inhibition group to form competitive and complementary binding.
2.5ul annealed product (2.5uM final concentration) is taken; 1ul Reaction Buffer, 1ul 10 x T4 DNA ligase, and ddH2O 5.5ul are added in the product; the reaction is run at 22 ° C for 2 hours.
5ul product of the ligase reaction in Step 2 is taken; 10×phi29 DNA polymerase Reaction Buffer 2 ul, phi29 DNA polymerase 1 ul, BSA 0.5ul, 2.5mM dNTP 1ul, and ddH2O 10.5 ul are added; the reaction is run at 37°C for 2 hours.
Take 4 ul reacted mixture，run 1%AGE，and observe whether rolling circle amplification products exist.
Continue the reaction for 16 hours (overnight), take 4ul reacted mixture again, run 1%AGE, and observe the impact of reacting time on the reaction.
Experiment of competition-based rolling circle amplification
The annealing experiment is carried out according to the same method of experiment #1, as follows:
Ligation，the protocol is consistent with #1
Rolling PCR, the protocol is consistent with #1
5ul product of the reaction mentioned above is taken; 10×phi29 DNA polymerase Reaction Buffer 2 ul, phi29 DNA polymerase 1 ul, BSA 0.5ul, 2.5mM dNTP 1ul, and ddH2O 10.5 ul are added; the reaction is run at 37°C for 2 hours.
Take 4 ul reacted mixture，run 1%AGE，and observe whether rolling circle amplification products exist.
Rolling PCR, the protocol is consistent with #1
Further experiment on rolling circle amplification
20 ul system
The above solution was diluted with multiple ratio of 1:5, 1:25, and 1:125, resulting in four parallel reactions of different concentrations
Water bath at 37℃ for 30 minutes.
Then add 2.5ul circle probe with different dilutions, and bathe at 67℃ for 30 minutes.
8ul of the reaction mixture is taken for ligation reaction.
20ul ligation reaction:
2ul of the above ligation product was used for the following rolling circle amplification reaction.
Reference：
[1] https://international.neb.com/protocols/2012/05/24/5-minute-transformation-protocol
[2] Saiki R.K. et al. (1985). Science. 230, 1350-1354.
[3] Powell, L.M. et al. (1987). Cell. 50, 831-840.
[4] Sun, Y., Hegamyer, G. and Colburn, N. (1993). Biotechniques. 15, 372-374.
[5] Sarkar, G., Kapelner, S. and Sommer, S.S. (1990). Nucleic Acids Res.. 18, 7465.
[6] SDS-PAGE Gel. (2015). Cold Spring Harbor Protocols , 2015(7), pdb.rec087908. Retrieved from http://cshprotocols.cshlp.org/content/2015/7/pdb.rec087908.short
[7] Li Shengtao. The expression and purification of the truncated HCV core protein (HCV Core125) and its antibody preparation [D]. Kunming university of technology,2012.
[8] Wu Xianbo. Cloning, expression of a gene fragment encoding HCV core antigen and purification, antigenecity analysis of the recombinant protein [D]. First military medical university of the people's liberation army,2003.
[9] Zhang Songbai, Zheng Liying, Hu xia, Shen Guangyu, Liu Xuewen, Shen Guoli, Yu Ruqin. Highly sensitive fluorescent aptasensor for thrombin detection based on competition triggered rolling circle amplification [J]. Chinese Journal of Analytical Chemistry,2015,43(11):1688-1694.
[10] Zhou hui. Studies on competitive mechanism triggered signal amplification based aptasensors [D]. Hunan university,2009.
[11] Dean F B, Nelson J R, Giesler T L, et al. Rapid amplification of plasmid and phage DNA using phi29 DNA polymerase and multiply-primed rolling circle amplification[J]. Genome research, 2001, 11(6): 1095-1099.
[12] Shi S, Yu X, Gao Y, et al. Inhibition of hepatitis C virus production by aptamers against the core protein[J]. Journal of virology, 2014, 88(4): 1990-1999.
[13] Banér J, Nilsson M, Mendel-Hartvig M, et al. Signal amplification of padlock probes by rolling circle replication[J]. Nucleic acids research, 1998, 26(22): 5073-5078.
