<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_JNFLS_Protocols skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:JNFLS/Protocols</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container-fluid"><DIV id="navbar" class="navbar-collapse collapse"><UL class="nav navbar-nav navbar-right" style="margin-top: 15px;"><LI><A class="dropdown-toggle active" href="https://2018.igem.org/Team:JNFLS">Home</A></LI><LI class="dropdown"><A class="dropdown-toggle active" href="https://2018.igem.org/Team:JNFLS/Team">Team </A></LI><LI><A href="https://2018.igem.org/Team:JNFLS/Team">Team Member</A></LI><LI><A href="https://2018.igem.org/Team:JNFLS/Collaborations">Collaboration</A></LI></UL><LI class="dropdown"><A class="dropdown-toggle active" href="https://2018.igem.org/Team:JNFLS/PROJECT">PROJECT </A></LI><LI><A href="https://2018.igem.org/Team:JNFLS/Description">Background</A></LI><LI><A href="https://2018.igem.org/Team:JNFLS/Design">Design</A></LI><LI><A href="https://2018.igem.org/Team:JNFLS/Protocols">Protocols</A></LI><LI><A href="https://2018.igem.org/Team:JNFLS/Model">Model</A></LI><LI><A href="https://2018.igem.org/Team:JNFLS/Results">Results</A></LI><LI><A href="https://2018.igem.org/Team:JNFLS/Interlab">Interlab</A></LI><LI><A href="https://2018.igem.org/Team:JNFLS/Demonstrate">Demonstrate</A></LI><LI><A href="https://2018.igem.org/Team:JNFLS/NoteBook">Notebook</A></LI><LI><A href="https://2018.igem.org/Team:JNFLS/Achievement">Achievement</A></LI><LI><A href="https://2018.igem.org/Team:JNFLS/Attributions">Attribution</A></LI><LI class="dropdown"><A class="a" href="https://2018.igem.org/Team:JNFLS/Parts">PARTS </A></LI><LI><A class="dropdown-toggle active" href="https://2018.igem.org/Team:JNFLS/SAFETY">SAFETY </A></LI><LI class="dropdown"><A class="dropdown-toggle active" href="https://2018.igem.org/Team:JNFLS/HUMAN_PRACTICES">HUMAN PRACTICE </A></LI><LI><A href="https://2018.igem.org/Team:JNFLS/Silver_Human_Practice">Silver Human Practice</A></LI><LI><A href="https://2018.igem.org/Team:JNFLS/Integrated_and_Gold">Integrated And Gold</A></LI><LI><A href="https://2018.igem.org/Team:JNFLS/Public_Engagement">Education&amp;Engagement</A></LI><LI><A class="dropdown-toggle active" href="https://igem.org/2018_Judging_Form?team=JNFLS">Judging Form</A></LI></DIV></DIV><DIV class="center"><H1 style="text-align:center;color:#FFFFFF;background-color:#A9A9A9">Protocols</H1></DIV><DIV class="column full_size"><H2>PCR truncation of HCVC gene</H2><H4>Primers:</H4><H2>Enzyme digestion of pcoldII vector plasmid</H2><OL><LI>select 1μL plasmid </LI><LI>add 5ul green buffer</LI><LI>add 1ul Xhol</LI><LI>add 1ul PstI</LI><LI>add distilled water to 20μL</LI><LI>water bath at 37 degrees for 10 min</LI><H2>Ligation</H2><H2>Bacterial transformation</H2><OL><LI>Remove cells from -80°C freezer and thaw in hand</LI><LI>Add 1-5 μl containing 1 pg-100 ng of plasmid DNA to the cell mixture. Carefully flick the tube 4-5 times to mix cells and DNA. Do not vortex</LI><LI>Place the mixture on ice for 2 minutes. Do not mix</LI><LI>Heat shock at exactly 42°C for exactly 30 seconds. Do not mix.</LI><LI>Place on ice for 2 minutes. Do not mix.</LI><LI>Pipette 250 μl of room temperature SOC into the mixture. Immediately spread 50-100 μl onto a selection plate and incubate overnight at 37-42°C. NOTE: Selection using antibiotics other than ampicillin may require some outgrowth before plating on selective media. Colonies develop faster at temperatures above 37°C, however some constructs may be unstable at elevated temperatures.</LI><H2>Colony PCR</H2><H4>Conditions：</H4><H2>Gel Recycling</H2><OL><LI>Run electrophoresis separation of the target DNA fragment</LI><LI>Cut off the target DNA fragments, as far as possible the unwanted cut off</LI><LI>The cut of the glue into the 1.5m | plastic centrifuge tube, said the quality of the plastic</LI><LI>Add the same amount of Binding Buffer(XP2), that is, add 0.3m| liquid if 0.3g is weighed</LI><LI>Heat in a 50-60 degree bath for a few minutes until all the glue is melted, stirring to mix.</LI><LI>Put the HiBind DNA Mini Column into the 2ml collection tube</LI><LI>Add the HiBindDNA Mini Column with less than 700ul from the DNA solution in step 5</LI><LI>Centrifuge 60s at 12000rpm at room temperature</LI><LI>Discard the waste liquid and recycle the collection pipe</LI><LI>Repeat steps 7-9 until all samples are transferred to column</LI><LI>Add 300ul Binding Buffer</LI><LI>Centrifuge 60s at 12000rpm at room temperature</LI><LI>Discard the waste liquid and recycle the collection pipe</LI><LI>Add the 700ul SPW Wash Buffer</LI><LI>Centrifuge 60s at 12000rpm at room temperature</LI><LI>Discard the waste liquid and recycle the collection pipe once again in steps 14-16</LI><LI>Centrifuge the empty HiBind DNA Mini Column 1 2000rmp for 2 minutes</LI><LI>The Hibind DNA Mini Column was transferred to 1.5m| plastic centrifuge tube</LI><LI>Add 15-30ul Elution Buffer to the center of the membrane</LI><LI>Let sit at room temperature for 2 minutes</LI><LI>Centrifugation for 12000rmp at room temperature for 1 minute</LI></OL><H2>plasmid extraction from positive clonesusing an extraction kit</H2><OL><LI>Decant or aspirate and discard the culture media.Centrifuge at 10,000x g for 1 minute at room temperature</LI><LI>Add 250 uL Solution I/RNase A. Vortex or pipet up and down to mix thoroughly.Complete resuspension of cell pellet is vital for obtaining good yields.</LI><LI>Transfer suspension into a new 15 mL microcentrifuge tube.</LI><LI>Add250 uL Solution IL Invert and gently rotate the tube several times to obtain a dear lysate. A 2-3minute incubation may be necessary.</LI><LI>Add 350uL Solution I Immediately invert several times until a flocculent white precipitate form.</LI><LI>Centrifuge at maximum speed (213,000 x g) for 10 minutes. A compact white pellet will form. Promptly proceed to the next step.</LI><LI>Insert a HiBinde DNA Mini Column into a 2 mL Collection Tube.</LI><LI>Transfer the cleared supernatant from Step 8 by CAREFULLY aspirating it into the HiBind* DNA Mini Column. Be careful not to disturb the pellet and that no cellular debris is transferred to the HiBind&quot; DNA Mini Column.</LI><LI>Centrifuge at maximum speed for 1 minute.</LI><LI>Discard the filtrate and reuse the collection tube.</LI><LI>Add 500 uL HBC Buffer</LI><LI>centrifuge at maximum speed for 1 minute.</LI><LI>Discard the filtrate and reuse collection tube.</LI><LI>Add 700uL DNA Wash buffer.</LI><LI>Centrifuge at maximum speed for 1 minute.</LI><LI>Discard the filtrate and reuse the collection tube.</LI><LI>Centrifuge the empty HiBind&quot; DNA Mini Column for 2 minutes at maximum speed to dry the column matrix.</LI><LI>Transfer the HiBind DNA Mini Column to a clean 1.5 mL microcentrifuge tube.</LI><LI>Add 30100 uL Elution Buffer or sterile deionized water directly to the center of the column membrane.</LI><LI>Let sit at room temperature for 1 minute.</LI><LI>Centrifuge at maximum speed for 1 minute. Note: This represents approximately 70% of bound DNA. An optional second elution will yield any residual DNA, though at a lower concentration</LI><LI>Store DNA at -20degrees.</LI></OL><H2>Purification of recombinant protein with His tag</H2><UL><H6>1. buffer preparation:</H6><B> Lysis Buffer 1 (under native condition) </B><LI><B><B>0.5mM Tris-HCl </B></B></LI><LI>0.5M NaCl </LI><LI>5%(w/v) glycerol </LI><LI>10mM Imidazole </LI><LI>100mg(1mg/ml) lysozyme (Purification of 6xHis-tagged proteins from E.coli) </LI><LI>1% Nonidet P40 (NP40 = Igepal CA-630) </LI><LI>0.25% Tween 20 （or Triton 100） </LI><LI>0.02% NaN3 (Optional) </LI><LI>2 tablets of protease inhibitor cocktail (EDTA free, Recommended) </LI><LI>200 μg(2μg/ml) RNase A (Optional) 1mg (10μg/ml) DNase1 (Optional) </LI><LI>50 mM NaF (Optional) </LI><LI>1mM Na3VO4 (Optional) </LI><LI>+ ddH2O to 100ml, adjust pH to 8.0 using NaOH </LI></UL><UL><B>Lysis Buffer 2(under denature condition) </B><LI>50 mM Tris-Cl </LI><LI>8 M Urea (or 6 M Gu-HCl) </LI><LI>10mM Imidazole </LI><LI>0.05% Tween 20 </LI><LI>Adjust pH to 8.0 using NaOH </LI></UL><UL><B>Dialysis/Tev cleavage Buffer </B><LI>50 mM Tris-Cl, pH8.0 </LI><LI>100 mM NaCl (depends on solubility of protein) </LI><LI>2.5% glycerol </LI><LI>0.5mM EDTA </LI><LI>0.5mM DTT or TCEP </LI></UL><UL><B>Buffer A： </B><LI>50mM Tris-HCl </LI><LI>0.5M NaCl </LI><LI>5% glycerol </LI><LI>0.05% Tween 20 </LI><LI>Adjust pH to 8.0 using high-concentration HCl </LI></UL><UL><B>Buffer B（washing buffer）： </B><LI>li50mM Tris-Cl /li</LI><LI>0.5M NaCl 5% glycerol </LI><LI>0.05% Tween 20 </LI><LI>500mM imidazole </LI><LI>Adjust pH to 8.0 using high-concentration HCl </LI></UL><UL><B>Buffer C（imidazole gradient washing buffer）：</B><LI>Prepare with buffer A and buffer B at different ratio: A 90ml, B 10ml</LI></UL><UL><B>Sample treatment： </B><LI>fully suspend centrifuge-collected bacteria at the ratio wet bacteria per gram/ 4~5ml Lysis Buffer，and place overnight in -80℃ fridge</LI><LI>thaw the bacteria at 4℃ and suspend the bacteria</LI><LI>break the bacteria in two turns, which contain 5s of ultrasonic breaking, 5s of cooling for each cycle and run 10min per turn, at 400W.</LI><LI>centrifuge at 4℃, 12000rpm for 30min，collect supernatant liquid, and adjust the pH with NaOH to 8.0</LI><LI>take 1ml Ni-NTA agarose gel and place it into 15ml centrifuge tube, then centrifuge at 2000 rpm for 2 min</LI><LI>take the supernatant liquid；add 5ml Lysis Buffer, centrifuge at low speed, then take the supernatant liquid.</LI><LI>add4-5 ml bacteria lysate，and rotate at 4℃ at low speed for 1h</LI><LI>Load the chromatographic column</LI><LI>Wash the chromatographic column with buffer C to wash off protein in the column</LI></UL><UL><H2>Improvement on purification method</H2><UL><LI>Imp1: Add 1% N-Lauryol-Sarcosine (weak denaturant) into buffer for purification. Other procedures and reagents are the same as those shown above.</LI><LI>Imp2: Dialyze the purified protein using 3k dialysis membrane (the buffer is pbs 50% glycerol)</LI><LI>Imp3: Dilute dialyzed protein with 50 mM tris and 100mM NaCl with the proportion of 1：1</LI></UL><H2>SDS-PAGE</H2><OL><LI>Prepare the separation gel (10%). Mix in the following order:</LI><LI>After adding TEMED and APS to the SDS-PAGE separation gel solution, the gel will polymerize quickly, so add these two reagents when ready to pour.</LI><LI>Pour gel, leaving ∼2 cm below the bottom of the comb for the stacking gel. Make sure to remove bubbles.</LI><LI>Layer the top of the gel with isopropanol. This will help to remove bubbles at the top of the gel and will also keep the polymerized gel from drying out.
	o In ∼30 min, the gel should be completely polymerized.
	</LI><LI>Remove the isopropanol and wash out the remaining traces of isopropanol with distilled water.</LI><LI>Prepare the stacking gel (4%). Mix in the following order:</LI><LI>Pour stacking gel on top of the separation gel.</LI><LI>Add combs to make wells. In ∼30 min, the stacking gel should become completely polymerized. </LI><LI>Clamp gel into apparatus, and fill both buffer chambers with gel running buffer according to the instructions for the specific apparatus.</LI><LI>Load samples and molecular mass protein markers into wells for separation by electrophoresis.</LI></OL><OL><H2>Aptamer &amp; Rolling PCR experiment </H2><OL><B>Sequences needed</B><H4>Reagents: </H4><B>Phi29 DNA polymerase Biomics Co., LTD</B><B>T4 DNA Ligase Biomics Co., LTD</B><B>10×BSA(0.05%) ：NEB Co., LTD </B><B>10×T4 DNA ligase buffer and 10×BSA (0.05%)）and dNTPs: Dalianbao Biotech Co., LTD</B><B>10×annealing buffer： 500mM NaCl，100mM Tris-HCl(PH8.0) 1mM EDTA</B><B>NEB 10X phi29 DNA Polymerase Reaction Buffer：500 mM Tris-HCl 、100 mM MgCl2 、100 mM (NH4)2SO4 、40 mM DTT 、(pH 7.5 @ 25°C)</B><B>TAKARA 10X T4 DNA Ligase Reaction Buffer ：500 mM Tris-HCl 、100 mM MgCl2 、10 mM ATP 、100 mM DTT 、(pH 7.5 @ 25°C)</B></OL><OL><H2>Examination of the reaction of designed primer, probe, and cDNA</H2><LI>Add 2.5 µL 10 µM Aptamer Circle Probe, 2.5 µL 0.1 µM cDNA, 1uL 10×annealing buffer, 4µL ddH2O to the 10 uL reaction system，anneal at 95 °C for 10 min, then </LI><LI>Let cool naturally. cDNA and aptamer are not added in the negative control group. Aptamer is added in the inhibition group to form competitive and complementary binding. </LI><LI>2.5ul annealed product (2.5uM final concentration) is taken; 1ul Reaction Buffer, 1ul 10 x T4 DNA ligase, and ddH2O 5.5ul are added in the product; the reaction is run at 22 ° C for 2 hours. </LI><LI>5ul product of the ligase reaction in Step 2 is taken; 10×phi29 DNA polymerase Reaction Buffer 2 ul, phi29 DNA polymerase 1 ul, BSA 0.5ul, 2.5mM dNTP 1ul, and ddH2O 10.5 ul are added; the reaction is run at 37°C for 2 hours. </LI><LI>Take 4 ul reacted mixture，run 1%AGE，and observe whether rolling circle amplification products exist. </LI><LI>Continue the reaction for 16 hours (overnight), take 4ul reacted mixture again, run 1%AGE, and observe the impact of reacting time on the reaction. </LI></OL><OL><H2>Experiment of competition-based rolling circle amplification</H2><B>The annealing experiment is carried out according to the same method of experiment #1, as follows:</B><B>Ligation，the protocol is consistent with #1</B><B>Rolling PCR, the protocol is consistent with #1</B><LI>5ul product of the reaction mentioned above is taken; 10×phi29 DNA polymerase Reaction Buffer 2 ul, phi29 DNA polymerase 1 ul, BSA 0.5ul, 2.5mM dNTP 1ul, and ddH2O 10.5 ul are added; the reaction is run at 37°C for 2 hours. </LI><LI>Take 4 ul reacted mixture，run 1%AGE，and observe whether rolling circle amplification products exist. </LI><LI>Rolling PCR, the protocol is consistent with #1</LI></OL><OL><H2>Further experiment on rolling circle amplification</H2><B>20 ul system</B><LI>The above solution was diluted with multiple ratio of 1:5, 1:25, and 1:125, resulting in four parallel reactions of different concentrations</LI><LI>Water bath at 37℃ for 30 minutes.</LI><LI>Then add 2.5ul circle probe with different dilutions, and bathe at 67℃ for 30 minutes.</LI><LI>8ul of the reaction mixture is taken for ligation reaction.</LI><B>20ul ligation reaction:</B><LI>2ul of the above ligation product was used for the following rolling circle amplification reaction.</LI></OL><OL><H3>Reference：</H3><P>[1] https://international.neb.com/protocols/2012/05/24/5-minute-transformation-protocol</P><P>[2] Saiki R.K. et al. (1985). Science. 230, 1350-1354.</P><P>[3] Powell, L.M. et al. (1987). Cell. 50, 831-840.</P><P>[4] Sun, Y., Hegamyer, G. and Colburn, N. (1993). Biotechniques. 15, 372-374.</P><P>[5] Sarkar, G., Kapelner, S. and Sommer, S.S. (1990). Nucleic Acids Res.. 18, 7465.</P><P>[6] SDS-PAGE Gel. (2015). Cold Spring Harbor Protocols , 2015(7), pdb.rec087908. Retrieved from http://cshprotocols.cshlp.org/content/2015/7/pdb.rec087908.short</P><P>[7] Li Shengtao. The expression and purification of the truncated HCV core protein (HCV Core125) and its antibody preparation [D]. Kunming university of technology,2012.</P><P>[8] Wu Xianbo. Cloning, expression of a gene fragment encoding HCV core antigen and purification, antigenecity analysis of the recombinant protein [D]. First military medical university of the people's liberation army,2003.</P><P>[9] Zhang Songbai, Zheng Liying, Hu xia, Shen Guangyu, Liu Xuewen, Shen Guoli, Yu Ruqin. Highly sensitive fluorescent aptasensor for thrombin detection based on competition triggered rolling circle amplification [J]. Chinese Journal of Analytical Chemistry,2015,43(11):1688-1694.</P><P>[10] Zhou hui. Studies on competitive mechanism triggered signal amplification based aptasensors [D]. Hunan university,2009.</P><P>[11] Dean F B, Nelson J R, Giesler T L, et al. Rapid amplification of plasmid and phage DNA using phi29 DNA polymerase and multiply-primed rolling circle amplification[J]. Genome research, 2001, 11(6): 1095-1099.</P><P>[12] Shi S, Yu X, Gao Y, et al. Inhibition of hepatitis C virus production by aptamers against the core protein[J]. Journal of virology, 2014, 88(4): 1990-1999.</P><P>[13] Banér J, Nilsson M, Mendel-Hartvig M, et al. Signal amplification of padlock probes by rolling circle replication[J]. Nucleic acids research, 1998, 26(22): 5073-5078.
	</P></OL></OL></UL></OL></OL></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>