Team:JNFLS/Results
Home
Team
Team Member
Collaboration
PROJECT
Background
Design
Protocols
Model
Results
Interlab
Demonstrate
Notebook
Achievement
Attribution
PARTS
SAFETY
HUMAN PRACTICE
Silver Human Practice
Integrated And Gold
Education&Engagement
Judging Form
Results
1. Construction of recombinant plasmid of HCV Core protein
1.1 Codon optimization of HCV core protein
Fig.1 Part sequence of original (left) or optimized (right) HCV core protein at website http://gcua.schoedl.de/sequential_v2.htmloptimized sequence of HCV core gene was showed as follows:
GGTACCATGAGTACCAATCCGAAACCGCAGCGCAAAACCAAACGTAATACCAATCGTCGTCCGGAAGATGTTAAATTTCCGGGCGGCGGTCAGATTGTGGG
CGGCGTTTATCTGCTGCCGCGTCGTGGCCCGCGTCTGGGTGTTCGTACCACCCGTAAAACCAGTGAACGCAGTCAGCCGCGCGGCCGCCGTCAACCTATTC
CGAAAGATCGTCGCAGTACCGGCAAAGCCTGGGGCAAACCGGGCCGTCCGTGGCCTCTGTATGGTAATGAAGGTCTGGGCTGGGCCGGTTGGCTGCTGAGC
CCTCGTGGTAGTCGTCCGAGTTGGGGCCCGACCGATCCGCGTCATCGCAGTCGTAATGTGGGTAAAGTGATTGATACCCTGACCTGTGGCTTTGCAGATCT
GATGGGCTATATTCCGGTGGTTGGCGCACCGCTGAGCGGTGCAGCACGCGCAGTTGCACATGGCGTTCGTGTTCTGGAAGATGGTGTTAATTATGCCACCG
GCAATCTGCCGGGCTTTCCGTTTAGTATTTTTCTGCTGGCCCTGCTGAGCTGTATTACCGTGCCGGTGAGCGCCCTGCAG
1.2 PCR and confirmation of HCV C gene
For expression efficiently, the optimized and truncated HCV C genes, O173 (173AA) and O120 (120AA), were amplified using PCR method. The primers and PCR results are showed as below:
Fig.2 Recycled enzyme-digested PCR products.
1.3 O173 and O120 genes were cloned into pColdII vector
Fig.3 Before ligation of PCR products and pColdII, they were ldigested by restriction enzymes.
1.4 Identification of pColdII-O120 and pColdII-O173 recombinant plasmid
After ligation reactions, the products were transformed into the competent cell DH5α. We used colony PCR method to select the positive clones, and then send them to the company for sequencing. Colony PCR results were showed as below:
Fig.4 Colony PCR products to identify the recombinant plasmids pColdII-O120 and pColdII-O173
Fig.5 Plasmid extraction for positive recombinant pColdII-O120 and pColdII-O173
2. HCV C Protein Expression and Purification
2.1 Expression of HCV C protein in E.coli
Fig.6 SDS-PAGE results showing the expression of HCV C protein in E.coli
From the SDS-PAGE rsults, we can see HCV C-O120 was expressed in the lysate of the complete bacteria, but the expression quantity was not too much. However, the HCV C-O173 wasn’t expressed.
2.2 Induced Expression of HCV C protein
For expression HCV C-O120 and HCV C-O173, they were inserted into another vector pcold-GFPuv, which can express the fusion protein GFP-HCV C-O120 and GFP-HCV C-O173. From the centrifuge pellets of bateria we can see the fusion proteins expressed very well.
Fig.7 The centrifuge pellets of bacteria showing the induced expression of fusion HCV C protein in E.coli
Fig.8 SDS-PAGE results showing the induced expression of HCV C-O173 and HCV C-O120, which were marked by GFP
2.3 Purification of HCV C protein
Fig.9 The induced expression Proteins were purified, then run electrophoresis.
3. Aptamer & Rolling PCR experiment
3.1 Sequences involved in rolling PCR
In order to detect the sensitivity and specificity of the aptamer in rolling PCR, we used the aptamer of HCV C7, which was screened previously, to perform the rolling PCR. Sequences used in the Rolling PCR are showed in the followings:
(The 5' end of the circle probe is modified by a phosphate group, and its terminal sequences are able to hybridize with the cDNA. The italics part of the nucleic acid aptamer is able to complementarily hybridize with cDNA; also, the underlined part of HCV C7 aptamer is able to specifically bind to the target protein HCV C. Both of the circle probe and HCV C7 aptamer can bind specifically with cDNA, forming a competitive relationship.)
Fig.10 Schematic diagram of cDNA with Circle Probe
Fig.11 Schematic diagram of cDNA with HCV C aptamer
3.2 Examination reaction of designed circle probe, cDNA and aptamer
1.Add 2.5 µL 10 µM Aptamer Circle Probe, 2.5 µL 0.1 µM cDNA, 1uL 10×annealing buffer, 4µL ddH2O to the 10 uL reaction system，anneal at 95 °C for 10 min, then let it cool naturally. cDNA and aptamer are not added in the negative control group.
Dilution anealing product by 100×, so the probes concentration is 0.1µM, then perform ligation reaction with the diluted anealing product, and rolling PCR reaction as follows:
Take the ligation product 1ul for each tube, 10×phi29 DNA Polymerase Reaction Buffer 2 ul, phi29 DNA Polymerase 1 ul，BSA 0.4ul, 2.5mM dNTP 0.6ul, primer 1ul, ddH2O 14 ul, put the mix at 37°C, 2h, then run electrophoresis, the result was showed as Fig.12. From the rolling PCR results below, we can see that the designed circle probe, cDNA and HCV C aptamer worked well.
Fig.12 Rolling PCR products using designed circle probe, cDNA and HCV C aptamer (without HCV C protein)
3.3 Experiment of competition-based rolling PCR
Prepared 3 replicates of the reaction system as the following table, each replicate is ready for one kind of cDNA 1-3.
Then each reaction system is added continuously cDNA and circle probe, as the below table shows.
Finally, all above reaction products were performed ligation and rolling PCR reactions as described previously, ran electrophoresis. The results are shown as below:
Fig.13 Competition-based rolling PCR products using circle probe, cDNA and HCV C aptamer with purified HCV C protein.
In general, the rolling PCR basically generated nothing at 37℃ annealing; it was better at 67℃ annealing, and the result was best at 94℃ annealing. In terms of the length of cDNA, cDNA#2 and cDNA#3 were shorter, the annealing temperature was lower, and the results were better. The high-temperature annealing after competitive hybridization showed that the rolling PCR products showed improved characteristics in turn with the increasing concentration of protein. In summary, at 94℃ annealing, cDNA#2 and 9ul protein combination could achieve the best result.
