<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_JNFLS_Experiments skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:JNFLS/Experiments</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container-fluid"><DIV id="navbar" class="navbar-collapse collapse"><UL class="nav navbar-nav navbar-right" style="margin-top: 15px;"><LI><A class="dropdown-toggle active" href="https://2018.igem.org/Team:JNFLS">Home</A></LI><LI class="dropdown"><A class="dropdown-toggle active" href="https://2018.igem.org/Team:JNFLS/Team">Team </A></LI><LI><A href="https://2018.igem.org/Team:JNFLS/Team">Team Member</A></LI><LI><A href="https://2018.igem.org/Team:JNFLS/Collaborations">Collaboration</A></LI></UL><LI class="dropdown"><A class="dropdown-toggle active" href="https://2018.igem.org/Team:JNFLS/PROJECT">PROJECT </A></LI><LI><A href="https://2018.igem.org/Team:JNFLS/Description">Background</A></LI><LI><A href="https://2018.igem.org/Team:JNFLS/Design">Design</A></LI><LI><A href="https://2018.igem.org/Team:JNFLS/Protocols">Protocols</A></LI><LI><A href="https://2018.igem.org/Team:JNFLS/Model">Model</A></LI><LI><A href="https://2018.igem.org/Team:JNFLS/Results">Results</A></LI><LI><A href="https://2018.igem.org/Team:JNFLS/Interlab">Interlab</A></LI><LI><A href="https://2018.igem.org/Team:JNFLS/Demonstrate">Demonstrate</A></LI><LI><A href="https://2018.igem.org/Team:JNFLS/NoteBook">Notebook</A></LI><LI><A href="https://2018.igem.org/Team:JNFLS/Achievement">Achievement</A></LI><LI><A href="https://2018.igem.org/Team:JNFLS/Attributions">Attribution</A></LI><LI class="dropdown"><A class="a" href="https://2018.igem.org/Team:JNFLS/Parts">PARTS </A></LI><LI><A class="dropdown-toggle active" href="https://2018.igem.org/Team:JNFLS/SAFETY">SAFETY </A></LI><LI class="dropdown"><A class="dropdown-toggle active" href="https://2018.igem.org/Team:JNFLS/HUMAN_PRACTICES">HUMAN PRACTICE </A></LI><LI><A href="https://2018.igem.org/Team:JNFLS/Silver_Human_Practice">Silver Human Practice</A></LI><LI><A href="https://2018.igem.org/Team:JNFLS/Integrated_and_Gold">Integrated And Gold</A></LI><LI><A href="https://2018.igem.org/Team:JNFLS/Public_Engagement">Education&amp;Engagement</A></LI><LI><A class="dropdown-toggle active" href="https://igem.org/2018_Judging_Form?team=JNFLS">Judging Form</A></LI></DIV></DIV><DIV class="column full_size"><H1 style="text-align:center;color:#FFFFFF;background-color:#A9A9A9">Protocols</H1><H3>Ligation</H3><OL><LI>Perform agarose gel/ethidium bromide electrophoresis to fractionate DNA fragments.
Any type or grade of agarose may be used. However, it is strongly recommended that fresh TAE buffer or TBE buffer be used as running buffer. Do not reuse running buffer as its pH will increase and reduce yields.
</LI><LI>When adequate separation of bands has occurred, carefully excise the DNA fragment of interest using a wide, clean, sharp scalpel. Minimize the size of the gel slice by removing extra agarose.</LI><LI>Determine the appropriate volume of the gel slice by weighing it in a clean 15 mL microcentrifuge tube. Assuming a density of 1 g/mL, the volume of gel is derived as follows: a gel slice of mass 0.3 g will have a volume of 0.3 mL</LI><LI>Add1 volume Binding Buffer (XP2).</LI><LI> Incubate at 50-60C for 7 minutes or until the gel has completely melted. Vortex or shake the tube every 2-3 minutes.</LI><LI>Insert a HiBind* DNA Mini Column in a 2 mL Collection Tube.</LI><LI>Add no more than 700 uL DNA/agarose solution from Step5 to the HiBind DNA Mini Column.</LI><LI>Centrifuge at 10，000＊g for 1 minute at room temperature.</LI><LI>Discard the filtrate and reuse collection tube.</LI><LI>Repeat Steps 7-9 until all of the sample has been transferred to the column.</LI><LI> Add 300 uL Binding Buffer (XP2).</LI><LI> Centrifuge at maximum speed (&gt;=13,000 x g) for 1 minute at room temperature.</LI><LI> Discard the filtrate and reuse collection tube.</LI><LI>Add 700 uL SPW Wash Buffer.</LI><LI> Centrifuge at maximum speed for 1 minute at room temperature.</LI><LI>Discard the filtrate and reuse collection tube.</LI><LI>Centrifuge the empty HiBind DNA Mini Column for 2 minutes at maximum speed to
dry the column matrix.
Note: It is important to dry the HiBind DNA Mini Column matrix before elution.Residual ethanol may interfere with downstream applications.
</LI><LI>Transfer the HiBind DNA Mini Column to a clean 15 mL microcentrifuge tube.</LI><LI>Add 15-30 uL Elution Buffer or deionized water directly to the center of the column
membrane.
</LI><LI> Let sit at room temperature for 2 minutes.</LI><LI> Centrifuge at maximum speed for 1 minute.</LI><LI>Store DNA at -20”C.</LI></OL></DIV><DIV class="two_thirds_size"><H3>Plasmid extraction</H3><OL><LI> Isolate a single colony from a freshly streaked selective plate, and inoculate a culture of 1- 5 mLLB medium containing the appropriate selective antibiotic. lnocubate for~12-16hours at 37C with vigorous shaking (~ 300 rpm). Use a 10-20 mL culture tube or a flask with a volume of at least 4 times the volume of the culture. It is strongly recommended that an endA negative strain of E. coli be used for routine plasmid isolation. Examples of such strains include DH5a* and JM1O9.</LI><LI> Centrifuge at 10,000x g for 1 minute at room temperature.</LI><LI>Decant or aspirate and discard the culture media.</LI><LI> Add 250 uL Solution I/RNase A. Vortex or pipet up and down to mix thoroughly.Complete resuspension of cell pellet is vital for obtaining good yields.</LI><LI>Transfer suspension into a new 15 mL microcentrifuge tube.</LI><LI>Add250 uL Solution IL Invert and gently rotate the tube several times to obtain a dear lysate. A 2-3minute incubation may be necessary.</LI><LI>Add 350uL Solution I Immediately invert several times until a flocculent white precipitate forms.</LI><LI>Centrifuge at maximum speed (213,000 x g) for 10 minutes. A compact white pellet will form. Promptly proceed to the next step.</LI><LI>Insert a HiBinde DNA Mini Column into a 2 mL Collection Tube.</LI><LI>Transfer the cleared supernatant from Step 8 by CAREFULLY aspirating it into the HiBind* DNA Mini Column. Be careful not to disturb the pellet and that no cellular debris is transferred to the HiBind&quot; DNA Mini Column.</LI><LI>Centrifuge at maximum speed for 1 minute.</LI><LI> Discard the filtrate and reuse the collection tube.</LI><LI>Add 500 uL HBC Buffer.</LI><LI> centrifuge at maximum speed for 1 minute.</LI><LI>Discard the filtrate and reuse collection tube.</LI><LI>Add 700uL DNA Wash buffer.</LI><LI>Centrifuge at maximum speed for 1 minute.</LI><LI>Discard the filtrate and reuse the collection tube.</LI><LI>Centrifuge the empty HiBind&quot; DNA Mini Column for 2 minutes at maximum speed to
dry the column matrix.</LI><LI>Transfer the HiBind DNA Mini Column to a clean 1.5 mL microcentrifuge tube.</LI><LI>Add 30100 uL Elution Buffer or sterile deionized water directly to the center of the
column membrane.</LI><LI>Let sit at room temperature for 1 minute.</LI><LI> Centrifuge at maximum speed for 1 minute.
Note: This represents approximately 70% of bound DNA. An optional second elution will yield any residual DNA, though at a lower concentration</LI><LI>Store DNA at -20degrees.</LI></OL><H3>Transformation</H3><OL><LI>Select the plasmid needed on part and repeat with 10 uL water to dissolve the DNA</LI><LI>All were sucked out and put into 1.5m| plastic centrifuge tube</LI><LI>Place on ice for 30 minutes, mix twice </LI><LI>Put in 42 degrees water bath and heat for 90s</LI><LI>Place on ice for 2 minutes</LI><LI>Add 500ul antibiotic free medium and resuscitate for 40-60 minutes. </LI><LI>Put the bacteria into the culture medium with the head of the spear, and spread with the applicator</LI><LI>Put in 37 degrees of shock culture</LI><H3>Restrictive endonuclease digestion</H3><OL><LI>select plasmid(J04500) for 40ul</LI><LI>add 10ul green buffer</LI><LI>add 46ul distilled water</LI><LI>add 2ul Spe1</LI><LI>add 2ul Pst 1</LI><LI>water bath at 37 degrees for 10 min</LI></OL><OL><LI>select plasmid(K592009) for 40ul</LI><LI>add 10ul green buffer</LI><LI>add 46ul distilled water</LI><LI>add 2ul Xba 1</LI><LI>add 2ul Pst 1</LI><LI>water bath at 37 degrees for 10 min</LI></OL><H3>Gel recycling</H3><OL><LI>Run electrophoresis separation of the target DNA fragment</LI><LI>Cut off the target DNA fragments, as far as possible the unwanted cut off</LI><LI>The cut of the glue into the 1.5m | plastic centrifuge tube, said the quality of the plastic</LI><LI>Add the same amount of Binding Buffer(XP2), that is, add 0.3m| liquid if 0.3g is weighed</LI><LI>Heat in a 50-60 degree bath for a few minutes until all the glue is melted, stirring to mix.</LI><LI>Put the HiBind DNA Mini Column into the 2ml collection tube</LI><LI>Add the HiBindDNA Mini Column with less than 700ul from the DNA solution in step 5</LI><LI>Centrifuge 60s at 12000rpm at room temperature</LI><LI>Discard the waste liquid and recycle the collection pipe</LI><LI>Repeat steps 7-9 until all samples are transferred to column</LI><LI>Add 300ul Binding Buffer</LI><LI>Centrifuge 60s at 12000rpm at room temperature</LI><LI>Discard the waste liquid and recycle the collection pipe</LI><LI>Add the 700ul SPW Wash Buffer</LI><LI>Centrifuge 60s at 12000rpm at room temperature</LI><LI>Discard the waste liquid and recycle the collection pipe once again in steps 14-16</LI><LI>Centrifuge the empty HiBind DNA Mini Column 1 2000rmp for 2 minutes</LI><LI>The Hibind DNA Mini Column was transferred to 1.5m| plastic centrifuge tube</LI><LI>Add 15-30ul Elution Buffer to the center of the membrane</LI><LI>Let sit at room temperature for 2 minutes</LI><LI>Centrifugation for 12000rmp at room temperature for 1 minute</LI></OL><H2 style="text-align:center;color:#FFFFFF;background-color:#A9A9A9">Experiments</H2><H2 style="text-align:center;color:#FFFFFF;background-color:#A9A9A9">Core protein expression </H2><H3>1.materials</H3><OL><LI> Target gene HCVC, and the codon-optimized sequence connected to the pColdII vector. 
GGTACCATGAGTACCAATCCGAAACCGCAGCGCAAAACCAAACGTAATACCAATCGTCGTCCGGAAGATG

TTAAATTTCCGGGCGGCGGTCAGATTGTGGGCGGCGTTTATCTGCTGCCGCGTCGTGGCCCGCGTCTGGG

TGTTCGTACCACCCGTAAAACCAGTGAACGCAGTCAGCCGCGCGGCCGCCGTCAACCTATTCCGAAAGA

TCGTCGCAGTACCGGCAAAGCCTGGGGCAAACCGGGCCGTCCGTGGCCTCTGTATGGTAATGAAGGTCT

GGGCTGGGCCGGTTGGCTGCTGAGCCCTCGTGGTAGTCGTCCGAGTTGGGGCCCGACCGATCCGCGTCA

TCGCAGTCGTAATGTGGGTAAAGTGATTGATACCCTGACCTGTGGCTTTGCAGATCTGATGGGCTATAT

TCCGGTGGTTGGCGCACCGCTGAGCGGTGCAGCACGCGCAGTTGCACATGGCGTTCGTGTTCTGGAAGAT

GGTGTTAATTATGCCACCGGCAATCTGCCGGGCTTTCCGTTTAGTATTTTTCTGCTGGCCCTGCTGAGCT

GTATTACCGTGCCGGTGAGCGCCCTGCAG
</LI><LI>genetic map of truncated gene HCVCO173 and HCVCO120</LI><LI>Instruments：
</LI><LI>PCR machine: ABI 2700</LI><LI>Centrifuge machine: Eppendorf 5415D</LI><LI>Dyy-6c electrophoresis machine: Beijing liuyi instrument factory</LI><LI>Dycp-31dn electrophoresis tank: Beijing liuyi instrument factory</LI><LI>SC760 gel electrophoresis image collector: Shanghai shanfu scientific instrument co. LTD</LI><LI>37 ℃ constant temperature incubator: VWR co., LTD</LI><LI>Super clean bench: suzhou purification equipment factory</LI><LI>Reagents：
</LI><LI>Plasmid extraction kit: nantong Biomics</LI><LI>Agarose gel recycle kit: nantong Biomics</LI><LI>Taq DNA polymerase: nantong Biomics</LI><LI>Pfu DNA polymerase: nantong Biomics</LI><LI>dNTPs: Shanghai migrant worker</LI><LI>T4 DNA Ligase: nantong Biomics Biomics</LI><LI>10x T4 DNA Ligase Buffer: nantong Biomics</LI><LI>Endonuclease XhoI: NEB</LI><LI>Endonuclease pstI: NEB</LI><LI>Buffer 3: NEB</LI><LI>BSA: NEB</LI><LI>1 KB plus DNA ladder: Tiangen</LI><LI>The primers were all synthesized by nantong Biomics</LI></OL><H3>2.Experiment procedures ：</H3><OL><LI>Plasmid design</LI><LI>RT-PCR and electrophoresis examination
<H4>Primers: </H4><P>Ligating site: Xhol+pstI</P><H4>PCR program：</H4><B>Condition:</B></LI><LI>95℃,5min</LI><LI>95℃, 30s</LI><LI>56℃,30s      </LI><LI>72℃,30min</LI><LI>72℃,2min</LI><LI>repeat 1-5 for 25cycles</LI></OL><B> Electrophoresis examination：</B><H3>3. Subcloning the gene into pColdII vector plasmid</H3><P>Through analysis of sequencing results by DNAStar SeqMan software, recombinant plasmids whose sequencing results were consistent with the original sequence were screened out.</P><P>Lane 1 and 2: recycled enzyme-digested PCR
 product: O120 and O173 (400bp  550bp)
Lane M: Marker
(400bp  500bp  850bp  1250bp 1750bp)
</P><B>c.Result of examination</B><P>Lane 1: Linear pCold II
Lane 2, 3: Enzyme-digested PCR product of O120 and O173 
Lane M: 1kb plus DNA Ladder </P><B>d．Recycle of pColdII plasmid and target gene</B><P>Lane 1 and 2: recycled enzyme-digested PCR product: O120 and O173 (400bp  550bp)
Lane M: Marker
(400bp  500bp  850bp  1250bp 1750bp)
</P><B>e．Ligation </B><B>f．Bacterial transformation Heat-shock method.</B><B>g．Colony PCR</B><P>pCold-F：ACGCCATATCGCCGAAAGG</P><P>pCold-R：TGGCAGGGATCTTAGATTCTG</P><B>4. Protein purification</B><P>Stage 1</P><P>SDS-PAGE result</P><P> Result of induced expression</P><P>Expression products of HCVCO173 and HCVCO120, which were marked by GFP, increased after codon optimization.</P><P>SDS-PAGE of Induced expression:     </P><P>Stage 2</P><P>At present, the problem is that the previously purified antigens are so weak (in the supernatant liquid) that more relevant experiments cannot be carried out in the hope of improving the prokaryotic expression system to obtain relevant proteins. As a result, more downstream experiments can be carried out, and there is currently no way to buy ready HCV core proteins from companies to speed up our experiments.</P><P>The final lane may have few target proteins for downstream experiments. Our current strategy is to change the form of fusion proteins as shown below.</P><P>The gene is planned to be inserted into the following plasmid vector; its fusion protein label is MBP. Meanwhile, His tag is moved further from core protein gene, because the isoelectric point of core protein is too high. It is hoped that the current difficult situation can be changed.</P><H3>Aptamer &amp; Rolling PCR experiment</H3><B>Sequences</B><P>*TB (Thrombin)</P><P>The sequence of nucleic acid sequences in this experiment is synthesized by Biomics biotechnology Co., LTD. The 5' end of the padlock probe is modified by a phosphate group, and the red-colored sequence is able to hybridize with the cDNA, and form a competitive relationship with the ligand. </P><P>The italics part of the nucleic acid aptamer is able to complementarily hybridize with cDNA; also, the underlined part of HCV C7 aptamer is able to specifically bind to the target protein.</P><H4>Schematic diagram of cDNA and Circle Probe</H4><B>1)	Circle Probe for TB aptamer</B><B>2)	cDNA and TB aptamer (complementary)</B><B>3)	Circle Probe for HCVC7 aptamer</B><B>4)	HCV Core aptamer</B><UL><B>Reagents: </B><LI> Phi29 DNA polymerase   Biomics Co., LTD</LI><LI> T4 DNA Ligase         Biomics Co., LTD</LI><LI>10×BSA(0.05%) ：NEB Co., LTD</LI><LI> 10×T4 DNA ligase buffer and 10×BSA (0.05%)）and dNTPs: Dalianbao Biotech Co., LTD</LI><LI> 10×annealing buffer： 500mM NaCl，100mM Tris-HCl(PH8.0) 1mM EDTA</LI><LI> NEB  10X phi29 DNA Polymerase Reaction Buffer：500 mM Tris-HCl 、100 mM MgCl2 、100 mM (NH4)2SO4 、40 mM DTT 、(pH 7.5 @ 25°C)</LI><LI> TAKARA 10X T4 DNA Ligase Reaction Buffer ：500 mM Tris-HCl 、100 mM MgCl2 、10 mM ATP 、100 mM DTT 、(pH 7.5 @ 25°C)</LI></UL><H3>#1  Examination of the reaction of designed primer, probe, and cDNA</H3><OL><LI>  2.5 µL 10 µM Aptamer Circle Probe, 2.5 µL 0.1 µM cDNA, 1uL 10×annealing buffer, 4µL  ddH2O to the 10 uL reaction system，anneal at 95 °C for 10 min, then let it cool naturally. cDNA and aptamer are not added in the negative control group. Aptamer is added in the inhibition group to form competitive and complementary binding. </LI><LI> 2.5ul annealed product (2.5uM final concentration) is taken; 1ul Reaction Buffer, 1ul 10 x T4 DNA ligase, and ddH2O 5.5ul are added in the product; the reaction is run at 22 ° C for 2 hours.  </LI><LI> 5ul product of the ligase reaction in Step 2 is taken; 10×phi29 DNA polymerase Reaction Buffer 2 ul, phi29 DNA polymerase 1 ul, BSA 0.5ul, 2.5mM dNTP 1ul, and ddH2O 10.5 ul are added; the reaction is run at 37°C for 2 hours. </LI><LI> Take 4 ul reacted mixture，run 1%AGE，and observe whether rolling circle amplification products exist. </LI><LI> Continue the reaction for 16 hours(overnight), take 4ul reacted mixture again, run 1%AGE, and observe the impact of reacting time on the reaction.  </LI></OL><P>Two parallel reactions were performed, respectively with different batches of enzymes. The 8 samples on the left and the 8 samples on the right showed consistency, and the parallel lines were good, but the amplification total products were different due to the different activity of the two batches of enzymes, but the amplification trend was consistent.</P><H4>TB aptamer and the establishment of its competitive reaction system</H4><P>A large number of rolling ring amplification products, such as reaction 1 and 2, can be obtained by the hybridization of TB padlock probes with both types of cDNA. The rolling circle amplification products cannot be produced in the reaction system without adding cDNA, such as the reaction 4. When an equal proportion of aptamer DNA is added to the reaction system to compete for the hybridization of cDNA, because the padlock probe cannot hybridize with enough cDNA and connect to cyclize, it cannot generate a clear rolling circle amplification product; such as reaction 3.</P><H4>HCV aptamer padlock probe and its ligation and rolling circle amplification</H4><P>After the hybridization with HCV padlock probe, a large number of rolling circle amplification products can be obtained, such as reaction 5, 6, and 7. The rolled ring amplification products cannot be produced in the reaction system without adding cDNA, such as reaction 8.</P><P>The result of overnight reaction is shown below:</P><P>The eight reactions on the left were parallel. DNA degradation occurred when the amplification result of TB padlock probe was not very strong, probably because the Phi29 enzyme were contaminated by DNase. However, the HCV padlock probe remained consistent with the 2-hour reaction because of its strong reaction amplification results, and the experimental results showed no significant difference from the 2-hour reaction. </P><P>The eight reactions on the left are also parallel; because of the good quality and high activity of the Phi29 enzyme, TB and HCV padlock probe rolling circle amplifications are consistent with the results of 2 hours reaction. Among the reactions, HCV padlock probe showed a strong reaction amplification result and some tailing accumulation of reaction products, but the overall trend remained consistent with the 2-hour reaction result.</P><P>To sum up, the suggested reaction time is 2 hours, which is consistent with the reports in the reference.</P><H3>#2、Experiment of competition-based rolling circle amplification</H3><P>The annealing experiment is carried out according to the same method of experiment #1, as follows:</P><B>Ligation，the protocol is consistent with #1</B><B>Rolling PCR, the protocol is consistent with #1</B><P>1.  5ul product of the reaction mentioned above is taken; 10×phi29 DNA polymerase Reaction Buffer 2 ul, phi29 DNA polymerase 1 ul, BSA 0.5ul, 2.5mM dNTP 1ul, and ddH2O 10.5 ul are added; the reaction is run at 37°C for 2 hours. </P><P>2.  Take 4 ul reacted mixture，run 1%AGE，and observe whether rolling circle amplification products exist. </P><P> We also did two parallel reactions, using different T4 Ligase; the eight reactions on the left used T4 DNA Ligase from General Biology System Co., LTD, and the eight reactions on the right used T4 DNA Ligase from Biomics Biotech Co., LTD. </P><P> Experimental results show that cDNA3 can better promote the effect of padlock probe, and the rolled circle amplification products are the most abundant, which is consistent with experimental results. When there is a competitive aptamer, cDNA cannot effectively form sufficient double strands and ligate with the padlock probe, so the rolling circle amplification product is basically absent. When purified HCV core protein is added to the reaction, part of the cDNA can be released, thus contributing to circle amplification of the padlock probe. This trend is concentration dependent: the number of circle amplification products in the eighth lane is significantly higher than that in the seventh lane.</P><P>The T4 Ligase activity of the two different companies was slightly different, but the experimental results were well parallel and the trend was consistent.</P><H4>Further experiment</H4><P> The experimental method was unchanged, and the concentration gradient of the ligation products in the original reaction condition was diluted to detect the sensitivity of the probe, cDNA and at a lower concentration. When performing PCR, the reaction was too strong and sometimes inhibited. When the ligation product was diluted, the reaction products tended to be stable, of which 25 times was the best effect and 125 times was still effective. Therefore, the concentration of the probe can be as low as 100 times, i.e. 0.1uM, which is consistent with the report from the reference.</P><P> Psl333 eukaryotic expression plasmid was transfected into the 293 cell, and the protein was collected and extracted for experiment.</P><P> *Transfected granulocyte extract was positive protein P sample and non-transfected granulocyte extract was negative protein N sample</P><P>The above solution was diluted with multiple ratio of 1:5, 1:25, and 1:125, resulting in four parallel reactions of different concentrations</P><P>Water bath at 37℃ for 30 minutes.</P><P>Then add 2.5ul circle probe with different dilutions, and bathe at 67℃ for 30 minutes.</P><P>8ul of the reaction mixture is taken for ligation reaction.</P></OL></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>