<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Hannover_Parts"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Hannover/Parts">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Hannover/Parts&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Hannover/Parts&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Hannover/Parts&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Hannover/Parts" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Hannover/Parts</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><H1>Results / Parts</H1><P class="text">Our goal was to engineer a transgenic plant, that
is able to bind heavy metals like cadmium, arsenic, zinc and copper. Therefore, metal binding proteins were fused (called Top 4 metal binding protein, T4MBP). To avoid cytotoxic effects on the plant caused by the heavy metals, we decided to place T4MBP to the cell exterior. With a secretion signal (Expa4) the T4MBP is directed to the extracellular space. The protein is tethered to the cell surface by using a cellulose binding domain (CBD). Plant cells have a cell wall which consists of cellulose. Via the cellulose binding domain the protein is attached to the cell wall.</P><CENTER><DIV id="groupparts" style="min-height:100px;width:700px;"><TABLE class=" pgrouptable tablesorter" cellspacing="0" cellpadding="0" style="width: 100%;"><THEAD><TR><TH style="width: 87px;">Name</TH><TH colspan="" style="width: 80px">Type</TH><TH colspan="" style="width: auto;">Description</TH><TH colspan="" style="width: 150px">Designer</TH><TH colspan="" style="width: 50px">Length</TH></TR></THEAD><TBODY><TR><TD><A class="noul_link part_link" href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1478002">BBa_K1478002</A></TD><TD>Coding</TD><TD>Top 4 metal binding protein. Binds zinc, arsenic, copper and cadmium.</TD><TD width="100px">Fabian Frömling</TD><TD align="right">909</TD></TR></TBODY></TABLE></DIV></CENTER><H1>Top 4 metal binding protein. Binds zinc, arsenic, copper and cadmium <A href="http://parts.igem.org/Part:BBa_K1478002">(BBa_K1478002)</A></H1><P class="text">Multiple domain protein. Domains bind zinc, arsenic, copper and cadmium. Sequences derived from metallothioneins and transcription factors of different organisms. Arsenic (CAA06729) and cadmium (AAK84863) are full length metallothioneins. Copper (AAA34542) domain is partial. Zinc (CAD88267) binding region derived from a transcription factor binding domain. Different protein binding regions are separated with glycerin-serine spacers to improve domain folding. Metal binding regions consist of disulfide bonds. For expression in <I>Escherichia coli</I> disulfide bond forming strains are useful (e. g. Origami). Sequence was optimized for <I>Escherichia coli</I> and <I>Arabidopsis thaliana</I> codon usage.sequenced all forbidden restriction-sites have been removedprearrangement and bioinformatics <A href="https://2014.igem.org/Team:Hannover/Background_bioinformatics" target="_blank">here...</A></P><H4>Analyzes</H4><P class="text">Figures 1  and 2 show the heavy metal binding efficiency difference between wildtype bacteria without added heavy metal and bacteria plus T4MBP and added heavy metal. The samples were analysed via ICP-OES. <A href="https://2014.igem.org/Team:Hannover/SpectrometryResults">More Information about the analysis and results</A></P><CENTER><TABLE border="0"><TBODY><TR><TD width="330px">Fig. 1: Measured heavy metals in dry bacteria pellets of wildtype bacteria (WT) without zinc (Zn) and for bacteria with T4MBP plus zinc.</TD><TD width="330px">Fig. 2: Measured heavy metals in dry bacteria pellets of wildtype bacteria (WT) without Cadmium (Cd) and for bacteria with T4MBP plus cadmium.</TD><TD width="330px">Fig. 3: Measured heavy metals in dry bacteria pellets of wildtype bacteria (WT) without cadmium (Cd) and for bacteria with T4MBP plus cadmium. Measurement was done with mass spectrometry.</TD></TR></TBODY></TABLE></CENTER><P class="text">Figure 1 shows that through expressed T4MBP about four times more zinc can be bound to bacteria than to the normal wildtype. A difference of binding cadmium of about 3 times more by T4MBP than without can be seen in figure 2. Additional analyses via mass spectrometry showed 3 times enhanced cadmium binding as well (fig. 3).</P><P class="text"><B> We therefore conclude:</B></P><OL><LI>The expression and right folding of T4MBP works.</LI><LI>Bacteria with expressed T4MBP bind effectively more heavy metals out of the surrounding than wildtype bacteria.</LI><LI>The binding of heavy metals works best for zinc, second cadmium and third copper (among of its lethal effect).</LI><LI>For arsenic there is no statement possible, but in consideration of points 1-3 we assume that expressed T4MBP will bind arsenic too.</LI></OL><H1>Expa4 secretion signal, localizes to extracellular space <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1478000">(BBa_K1478000)</A></H1><P class="text">Coding region for first 20 aminoacids of plant protein Exansin4. Secretion signal for plants, when fused to a protein directs the protein to the extracellular space. Derived from the signal sequence of Expansin4 protein from <I>Arabidopsis thaliana</I> (NP_181500). The sequence was optimized for <I>Escherichia coli</I> and <I>Arabidopsis thaliana</I> codon usage. Analyzes showed that the part works. sequenced all forbidden restriction-sites have been removed </P><H1>Cellulose binding domain <A href="http://parts.igem.org/Part:BBa_K1478001">(BBa_K1478001)</A></H1><P class="text">Coding region for cellulose binding domain. When fused to a protein, protein attaches to cellulose. Domain derived from cellulose binding protein from <I>Chlostridium cellulovorans</I> (WP_013291799). Sequence was optimized for <I>Escherichia coli</I> and <I>Arabidopsis thaliana</I> codon usage.sequenced all forbidden restriction-sites have been removedprearrangement and bioinformatics <A href="https://2014.igem.org/Team:Hannover/Background_bioinformatics" target="_blank">here...</A></P><H4>Analyzes BBa_K1478000 and BBa_K1478001</H4><P class="text">Bioinformatic analyses based on the raw protein sequence (20 aa) predicts the secretion of the protein with 50 % probability (fig. 1).</P><CENTER><TABLE><TBODY><TR><TD width="400px"><P class="text">Fig. 1: Prediction of subcellular localization of Expa4 Prediction of the raw sequence of Expa4 (20 aa) by TargetP. Len: protein length; cTP: chloroplast; mTP: mitochondria; SP: secretory pathway. Probability of 100 % is 1.0.</P></TD></TR></TBODY></TABLE></CENTER><P class="text">For further analyses a GFP-fusion protein was designed. Therefore,  Expa4 was cloned in frame with GFP and the cellulose binding domain. This coding sequence was cloned into a vector backbone and was used for plant transformation. The GFP signal was detected via confocal microscopy in plant cells. In large eucaryotic cells it’s possible to analyze the subcellular localization with fusion proteins [1]. A prediction via bioinformatics showed nearly 100 % probability for the secretion of the fusion protein (fig. 2).</P><CENTER><TABLE><TBODY><TR><TD width="400px"><P class="text">Fig. 2: Prediction of subcellular localization of fusion protein Expa4:GFP:CBD Prediction of the raw sequence of fusion protein (372 aa) by TargetP. Len: protein length; cTP: chloroplast; mTP: mitochondria; SP: secretory pathway. Probability of 100 % is 1.0.</P></TD></TR></TBODY></TABLE></CENTER><P class="text">Figure 3 shows detected fluorescence of transformed <I>Nicotiana tabacum</I> cells. A GFP control without protein fusion is shown in lane A-D. The signal is specific for a cytosolic protein, which is indicated by chloroplast surrounded with GFP-Signal. In contrast the plasmamembranemarker (E-H) shows a specific signal at the periphery. The signal doesn’t surround the chloroplast, which shows that it is located at the cell cover. Because it’s a marker, it’s location is already known. The analyzed construct is shown in I-K. It contrast to cytoplasmic GFP, the signal appears to be in the exterior. The signal appears to diffuse between the cells, at the cell wall. This indicates that the secretion signal and the tethering at the cell wall via CBD might work.</P><CENTER><TABLE><TBODY><TR><TD width="400px"><P class="text">Fig. 3: Confocal detection of fluorescence from transformed plant cells (replicate 1). Transformed cells of <I>Nicotiana tabacum</I> were analyzed via confocal microscopy. Column A-I: Red channel, shows chlorophyll’s autofluorescence which shows the chloroplasts. Column B-J: Green channel showing GFP. Column C-K: Red and green channel merged. Column D-L: Pseudo transmission detection. Lane A-D shows a transformed cell with raw GFP. Lane E-H shows a plasmamembranemarker (Nelson <I>et al.</I> 2007). Lane I-L shows the detection of the analyzed construct: Expa4-GFP-CBD.</P></TD></TR></TBODY></TABLE></CENTER><P class="text">1. Cutler, S. R.; Ehrhardt, D. W.; Griffitts, J. S.; Somerville, C. R. (2000): Random GFP::cDNA fusions enable visualization of subcellular structures in cells of Arabidopsis at a high frequency. In Proceedings of the National Academy of Sciences 97 (7), pp. 3718–3723. DOI: 10.1073/pnas.97.7.3718.
2. Nelson, B. K.; Cai, X.; Nebenfuhr, A. (2007): A multicolored set of in vivo organelle markers for co-localization studies in Arabidopsis and other plants. In Plant J 51 (6), pp. 1126–1136. DOI: 10.1111/j.1365-313X.2007.03212.x.</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Hannover/Parts">http://2014.igem.org/Team:Hannover/Parts</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Hannover/Parts" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Hannover/Parts" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Hannover/Parts&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Hannover/Parts&amp;oldid=367840" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>