<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Nevada_Notebook_Weeks1316"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Nevada/Notebook/Weeks1316">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Nevada/Notebook/Weeks1316&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Nevada/Notebook/Weeks1316&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Nevada/Notebook/Weeks1316&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Nevada/Notebook/Weeks1316" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Nevada/Notebook/Weeks1316</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><DIV id="absHeader"><DIV id="banner-box"><DIV id="banner-title-box"><DIV class="banner-text-title">
So Happy Together</DIV><DIV class="banner-subtext">
A Cooperative Relationship between Cyanobacteria and E. Coli
for production of biofuels
</DIV></DIV><DIV id="banner-buttons" class="buttons-text"><UL class="top-level"><LI>  <A href="/Team:Nevada">Home</A>  </LI><LI>
|  <A href="/Team:Nevada/Team">Team</A>  </LI><LI><A href="/Team:Nevada/Team">Nevada's Team</A></LI><LI><A href="/Team:Nevada/gallery">Gallery</A></LI></UL><LI>
|  <A href="/Team:Nevada/Project">Project</A>  </LI><LI><A href="/Team:Nevada/Project/Background">Introduction</A></LI><LI><A href="/Team:Nevada/Project/Ecoli">E. coli Project</A></LI><LI><A href="/Team:Nevada/Project/Cyano">Cyanobacteria Project</A></LI><LI><A href="/Team:Nevada/Project/Assay">Assay Development</A></LI><LI><A href="/Team:Nevada/Project/Co-Cult">Co-Cultivation</A></LI><LI><A href="/Team:Nevada/Project/Results">Results Summary</A></LI><LI><A href="/Team:Nevada/Reference">References</A></LI><LI>
|  <A href="/Team:Nevada/Parts">Parts</A>  </LI><LI>
|  <A href="/Team:Nevada/Notebook/Weeks14">Notebook</A>  </LI><LI><A href="/Team:Nevada/Notebook/Weeks14">Weeks 1-4</A></LI><LI><A href="/Team:Nevada/Notebook/Weeks58">Weeks 5-8</A></LI><LI><A href="/Team:Nevada/Notebook/Weeks912">Weeks 9-12</A></LI><LI><A href="/Team:Nevada/Notebook/Weeks1316">Weeks 13-16</A></LI><LI>
|  <A href="">Human Practices</A>  </LI><LI><A href="/Team:Nevada/iGEMcollaborators">iGEM Collaborators</A></LI><LI><A href="/Team:Nevada/Outreach">Outreach</A></LI><LI><A href="/Team:Nevada/Contributions">Contributions</A></LI><LI><A href="/Team:Nevada/Fundraising">Fundraising</A></LI><LI>
|  <A href="/Team:Nevada/Safety">Safety</A>  </LI></DIV></DIV></DIV><DIV id="Nevada-Notebook-MainContent"><DIV id="left-minor-content"><FONT color="red"><U><FONT size="4">E. Coli</FONT></U>
Jovanna, Megan, Lauren, Sam, Dafne, Tutku &amp; Destiny
The E.coli group is responsible for making the constructs which will produce fatty acids and ethanol.</FONT><FONT color="blue"><U><FONT size="4">Cyano</FONT></U>
Megan, Dru, Vadim, Marguerite, Laura, David, Jen, Ron &amp; Chris
The cyano group is responsible for increasing glucose production and secretion in cyanobacteria.</FONT><FONT color="green"><U><FONT size="4">Assay</FONT></U>
Casey, Sam, Vadim, Majid, Megan, and Jovanna
The assay group is responsible for testing and verifying the products made by our system.</FONT><FONT color="black"><U><FONT size="4">Media</FONT></U>
Bryson &amp; Matt
The media group is responsible for creating the best possible co-cultivation system for E.coli and cyanobacteria. They are also responsible for creating a co-cultivation apparatus.
<FONT size="4"><B> Description </B>:</FONT><DIV id="descDef" style="text-align:justify;">
BTE-Bay Laurel (Plant) thioesterase gene- produces 12 and 14 carbon medium chain fatty acids that can be converted to biofuels. 

PDC/ADH- pyruvate decarboxylase/aldehyde dehydrogenase operon. This part can be used to produce ethanol. PDC converts pyruvate to acetaldehyde. Alcohol dehydrogenase can then turn acetaldehyde into ethanol. 

ThiE-Thiamine E synthesizing gene in cyanobacteria. The enzyme is usually referred to as thiamine monophosphate pyrophosphorylase. Important for vitamin B synthesis in cyanobacteria. Knockout of this gene will allow for production of an auxotroph in cyanobacteria. 
sigma70 promoter- constitutive promoter in E.coli.

trc promoter- inducible promoter in E.coli. It is not sensitive to changes in glucose concentrations.

GLF- glucose facilitator gene responsible for transporting glucose out of the cyanobacteria.

INV- invertase gene responsible for breaking down sucrose into glucose and fructose. Gene is from cyanobacteria.
</DIV></FONT></DIV></DIV><DIV id="right-major-content" style="overflow:auto;"><A href="https://2011.igem.org/Team:Nevada/Notebook/Weeks14"><FONT size="4">Weeks 1-4</FONT></A>   <A href="https://2011.igem.org/Team:Nevada/Notebook/Weeks58"><FONT size="4">Weeks 5-8</FONT></A>   <A href="https://2011.igem.org/Team:Nevada/Notebook/Weeks912"><FONT size="4">Weeks 9-12</FONT></A>   <A href="https://2011.igem.org/Team:Nevada/Notebook/Weeks1316"><FONT size="4">Weeks 13-END</FONT></A>   <DIV class="nbtext" align="left"><P><FONT size="6">Calender Weeks 13-END</FONT></P><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Week_13_-_August_22nd-28th"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Week 13 - August 22nd-28th</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Week_14_-_August_29th-_September_4th"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Week 14 - August 29th- September 4th</SPAN></A></LI><LI class="toclevel-1 tocsection-3"><A href="#Week_15_-_September_5th-11th"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Week 15 - September 5th-11th</SPAN></A></LI><LI class="toclevel-1 tocsection-4"><A href="#Week_16_-_September_12th-18th"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Week 16 - September 12th-18th</SPAN></A></LI><LI class="toclevel-1 tocsection-5"><A href="#Week_17_-_September_19th-28th"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Week 17 - September 19th-28th</SPAN></A></LI></UL></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Week_13_-_August_22nd-28th"><B>Week 13 - August 22nd-28th</B></SPAN></H2><P><FONT color="red">E. Coli <U>Trc Promoter</U>
LT:The protocol from week #11 was repeated but the transformation reaction was plated onto LB-CHL plates for proper selection. Results: The BTE digest was successful as confirmed by the gel. The trc promoter digest had the same inconclusive results as seen preciously. Colonies successfully grew from the ligation and transformation. </FONT></P><P><U>Pyruvate Decarboxylase &amp; Alcohol Dehydrogenase</U>
JC:  To test viability of PDC, found information regarding Aldehyde detection plates.  Since PDC converts Pyruvate to Acetaldehyde, these plates could be a quick way to test for the presence of aldehydes in our constructs.  Growing s70/PDC/ADH in NEB B10 cells yielded bright pink colonies (aldehydes present).  
<U>Bay Laurel Thioesterase</U></P><P>MT: Five cultures of BTE coding region in the Topo vector (pUC57) were grown, and minipreped. The BTE samples and pSB1C3  were digested with Xba I and Pst I and ligated. Ligation was transformed using NEB Beta cells. Colonies were selected to culture overnight and minipreped, digested to check for correct band size.
</P><P>A Free Fatty Acid Assay was done on BTE with the promoter sigma 70. A sample was also ran through a Gas chromatographer  to check the length of hydrocabons we are producing. Results are in the assay section.
</P><P>BTE with sigma 70 promoter and pSB1C3 were digested with EcoRI and PstI and ligated, transformed, and cultured.</P><P><FONT color="blue">
Cyano </FONT></P><P><U>ThiE/GLF:</U></P><P>VG:All the colonies only grew on CMR, except for colonies 14 and 19, which grew on both AMP and CMR. The first twenty colonies that grew on CMR plates only were minipreped using the Qiagen protocol and the sequencing for GLF was checked using blast. It appeared that there was a high chance of GLF being in the pUC57 vector. Thus, we decided to digest GLF from the pUC57 vector and pSB1C3 with EcoRI and SpeI and trying another ligation. 
0.5mg of the first twenty CMR only colonies from the INV/pSB1C3 ligation (Elaine) were digested with EcoRI HIFI and PstI, along with GLF (5) in pUC57 and pSBIC3. Colonies 9, 11, and 15 on the gel looked correct, showing two bands at 2000bp and 1600bp. They will be minipreped and checked further with EcoRI and SpeI digests, EcoRI digests only, and sent in for sequencing. The GLF digest did not show up on the gel. Thus, we will digest GLF again along with pSB1C3 and try a ligation. 
</P><P><U>AGP/INV:</U></P><P>Procedure: KNR was PCR amplified from pUC4K vector in order to produce a large amount of the cassette for use in gibson assembly of the AGP knock-out/INV operon. KNR was amplified from 1 ng/ul pUC4K using the AGP_KNR forward and KNR_petBD reverse primers and an initial annealing temperature of 57 degrees C.  Successful PCR was confirmed on 0.7% agarose gel. Gibson assembly was performed to construct KNR+petBD and petBD+INV, with the rational that either construct could be used to assemble the entire AGP knockout. Equimolar ratios of each part were used for both reactions, and the master mix was prepared as outlined by University of Cambridge. Gibson assembly products could not be verified by electrophoresis so PCR was performed on each reaction in order overcome the possibility of small reaction yields. Potential KNR+petBD was amplified using AGP_KNR forward and petBD_INV reverse primers and an initial annealing temperature of 57 degrees C. Potential petBD+INV was amplified using petBD_INV forward and INV_AGP reverse primers with an initial annealing temperature of 64 degrees C. Gibson assembly was verified on 0.7% agarose gel at 13.8 V/cm for 60 minutes.

Results: No bands were observed for the PCR of the Gibson Assembly reactions. This indicates that either gibson assembly failed, or purification and PCR of the gibson assembly products failed. Thus, neither KNR+petBD nor petBD+INV were constructed via gibson assembly.
</P><P><FONT color="green">
Enzymology</FONT></P><P>CL:This week we used the GC to analyze the fatty acids being produced by one of the BTE IQ cell samples (BTE13).  The fatty acids were extracted from cell supernatent using hexane and tested with a carbon disulfide solvent.  We used FAME standards ranging from C:8-C:19 as well as a C:19 internal standard for comparison.  Results proved positive as we were looking for C:12 FA's and there were C:8, C:10 and C:12 FA's present in the sample.  We suspect that the small chains could possibly be the larger C:12 chain being metabolized and we plan to further analyzed this data.  This was just a test or a pre-run and we did not use a negative control or test very many samples.  We plan to do a definite run next week on four new BTE samples as well as negative controls of wild-type IQ cells and a known concentration of a C:17 internal standard will be used for quantification.  Also this week we performed our third ethanol assay using the BioSystems Ethanol assay kit on 4 of the ADH/PDC IQ cell cultures.  We retrieved positive results for one of the samples (ACH/PDC17) with an ethanol production of 3.4mM.  
</P><P>
Media
BW:We thought to check the pH of BG-11 with casaminos to determine if this addition lowered the pH noticeably.  Whereas BG-11 has a pH of 6.7, after adding casaminos the pH drops to 5.7.  The pH of our media will be adjusted to 6.7 in future experiments.

MB:Fabrication of the light slides and mount was completed and installed on the base. Transfer pump was mounted and all hose connections installed. All electrical was run through entire apparatus. Testing began using food coloring in E coli. chamber and clear water in the cyanobacteria side. Hose clamps were first used then o-rings to secure the dialysis tubing to the inner glass tube. Dialysis tubing was still forming a small leak at the o-ring seal.
</P><H2><SPAN class="mw-headline" id="Week_14_-_August_29th-_September_4th"><B>Week 14 - August 29th- September 4th</B></SPAN></H2><P><FONT color="red">E. Coli <U>Trc Promoter</U>
LT:Colonies #1-15 were selected from the BTE/trc promoter transformation, inoculated in TB-CHL, and minipreped. Colonies #16-45 were selected and used for colony PCR using VF and VR primers. Half the PCR products were loaded onto a 0.7% gel to check the products. Results: There is no DNA present on the gel. It appears that a mistake was made during preparation of the PCR reaction. Only the standards were visible. </FONT></P><P><U>Pyruvate Decarboxylase &amp; Alcohol Dehydrogenase</U>
JC:  To again test PDC and ADH activity, s70/PDC/ADH was transformed into NEB High Expression Iq cells.  These cells were plated on the aldehyde detection plates and demonstrated a more intense pink colony than the NEB B10 cells. 
</P><P><U>Bay Laurel Thioesterase</U></P><P>MT: The cultures of the transformation of BTE with sigma 70 promoter in pSB1C3 were minipreped, digested with EcoRI and PstI, and checked on a 1% gel. Four of the five had full digestion. Sample #4 was sent for sequencing.</P><P><FONT color="blue">
Cyano </FONT></P><P><U>ThiE/GLF:</U></P><P>VG:0.5mg of colonies 9, 11, and 15 were digested with EcoRI and PstI and EcoRI only and 0.25mg of pSB1C3 were digested with EcoRI and PstI. The digests appeared correct for colonies 9 and 11. These samples were sent in for sequencing to the Nevada Genomics Center. GLF in the ampicillin resistant pUC57 vector was successfully digested with EcoRI and PstI and ligated with digested pSB1C3 vector. Digested GLF and pSB1C3 were also run on a 1.2% agarose gel and the gene and vector fragments were cut out of the gel, purified, and ligated. Both ligations were transformed in NEB10β E.coli cells and checked with restriction digests prior to sequencing. 
2 Liters of media were also prepared to make more AMP and CMR plates.
</P><P><U>AGP/INV:</U></P><P>Procedure: Gibson assembly was reattempted to construct KNR+petBD+INV (Attempts 4 and 5). Taking the advice from University of Cambridge, phusion polymerase was added last to the reactions, and reactions were verified on 0.7% agarose gel at 13.8V per cm. The electrophoresis data indicated that both Gibson assembly attempts were unsuccesful (No bands observed). In response to the consecutive failed Gibson assembly attempts, Sequence and Ligation Independent Cloning Technique (SLIC) was used to assemble the KNR+petBD+INV operon. This technique involves incubating the individual parts (KNR, petBD, and INV) with T4 DNA polymerase prior to the addition of Taq DNA ligase and nucleotides in order to chew back the overlapping sequence prior to assembling the construct. The SLIC product was PCR amplified using the AGP_KNR forward and INV_AGP reverse primers and an initial annealing temperature of 60 degrees C. The SLIC product was verified on 0.7% agarose gel at 13.8 V per cm.
</P><P>
Results: The electrophoresis data indicated that the SLIC assembly attempt was unsuccessful (No bands observed). Thus, the KNR+petBD+INV operon has yet to be assembled.
</P><P><FONT color="green">
Enzymology</FONT></P><P>
Media
We reran the experiment from week 12, in which we looked at the effect of glucose and ammonium chloride concentrations on I<SUP>q</SUP> cells grown in BG-11.  This time we set the pH of all media to 6.7 with 0.1 M NaOH and cultured our samples in triplicate.  The results confirmed what we had found in week 12.  With the addition of glucose, the cultures grew to an OD at 600 nm of more than 60% greater than our standard curve.
</P><P>The co-cultivation apparatus has arrived this week, and we can begin going over how to clean it and perform our first tests with it.</P><H2><SPAN class="mw-headline" id="Week_15_-_September_5th-11th"><B>Week 15 - September 5th-11th</B></SPAN></H2><P><FONT color="red">E. Coli <U>Trc Promoter</U>
LT:BTE in puc57 was recultured in LB-Amp broth from glycerol stocks.  Cultures were minipreped and quantitated using Nanodrop spectrophotometry. 
The trc promoter was isolated from pTRC99A plasmid using PCR. Products were purified using the QIAquick PCR purification protocol and quantitated using Nanodrop spectrophotometry. Products were verified by running them on a 1.2% agarose gel. Results: The PCR reaction was successful. There is a band corresponding to the 100bp fragment that we expected to see. </FONT></P><P><U>Pyruvate Decarboxylase &amp; Alcohol Dehydrogenase</U>
JC:  
Since 70/PDC/ADH construct is producing aldehydes at this point (PDC is working), prepared for submission to iGEM.  The construct was digested and ligated into pSB1C3 and transformed into NEB10 β cells. Selected single colonies from LB-Chloramphenicol plates and single colony streaked onto LB-Chl and LB-Amp plates to perform plasmid check. Grew liquid cultures in LB-Chl and performed minipreps and nanodrop analysis.
</P><P>To confirm s70/PDC/ADH insertion into pSB1C3, 0.5ug of s70/PDC/ADH/pSB1C3 was digested with EcoRI and PstI. Digestions were run on a 1% agarose gel, and bands obtained confirmed s70/PDC/ADH (3089bp) and pSB1C3 (2070bp). 
</P><P>s70/PDC/ADH/pSB1C3 was sent to Nevada Genomics Center for sequencing. 
</P><P><FONT color="blue">
Cyano </FONT></P><P><U>ThiE/GLF:</U></P><P>VG:The digest ligation of pSB1C3 and GLF grew red and white colonies. Fifteen white colonies were selected, line streaked, and minipreped. 0.5mg of all fifteen samples were digested with EcoRI and PstI and EcoRI only to confirm a band at 1605 bp for GLF and a second band at 2070 bp for pSB1C3. The samples were run on a 1% agarose gel. All the digested samples appear correct, except for sample six. Thus, we can send any of the remaining fourteen samples to the Nevada Genomics Center for sequencing.
</P><P><U>AGP/INV:</U></P><P>Procedure: AGP was PCR amplified from <I>Synechocystis PCC 6803</I> genomic DNA using the AGP forward and reverse primers and an initial annealing temperature of 57 degrees C. PCR was confirmed successful on 0.7% agarose gel at 13.8V cm. In order to construct the vector backbone for gibson assembly of the AGP knockout/INV operon, the AGP PCR product was topocloned into the TOPOIIPCRBlunt vector (Invitrogen), transformed into DH5α cells, and plated on LB-KNN/2% X-Gal. Successful transformants were selected and incubated in liquid cultures of LB-KNN broth, and plasmid DNA was isolated using a miniprep kit(QIAgen). Plasmid DNA was digested with PstI and SpI at a final concentration of 25 ng/ul. Restriction digests were verified on 0.7 % agarose gel at 13.8V/cm 
</P><P>Results: The electrophoresis data confirmed successful topocloning of agp into the TOPIIBlunt vector for 3 out of the 6 colonies selected. Bands were observed at the expected size for the vector (3.5 kb) and agp (1320 bp). Thus, these plasmid samples will be used for subsequent PCR to obtain the linearized agp/vector backbone for use in Gibson assembly. Also, the sequencing data for INV was checked and confirmed the presence of INV in pSB1C3. Thus, the DNA will be submitted as a part to the iGEM Registry of Biological Parts.
</P><P>Enzymology

Media
An experiment was done to determine what levels of glucose would be able to give us a noticeable increase in growth for I<SUP>q</SUP> cells grown in BG-11.  Past experiments have shown that the minimum concentration we've used (10 mM) is as effective as higher concentrations, so we wished to see how much lower we could get the concentration, on the chance that our cyanobacteria supplies less than 10 mM glucose.  Glucose concentrations tested were: 10 mM, 5.0 mM, 2.5 mM, 1.0 mM and 0.50 mM.  The experiment was run in triplicate with the pH of all media adjusted to 6.7.  
</P><P>The growth curves indicated that 5 mM glucose consistently produced even greater cell growth than 10 mM and growth slowly dropped off with lower concentrations after that point.  However, even at 0.50 mM, there was an increase of roughly 10% in the OD at 600 nm after 24 hours.
</P><H2><SPAN class="mw-headline" id="Week_16_-_September_12th-18th"><B>Week 16 - September 12th-18th</B></SPAN></H2><P><FONT color="red">E. Coli </FONT></P><P><U>Pyruvate Decarboxylase &amp; Alcohol Dehydrogenase</U>
JC:  The EnzyChrom Ethanol Detection Kit was used again to test ethanol production in s70/PDC/ADH in NEB Iq E. coli cells.  The cultures were grown over 24h with the addition of 2% glucose, to test the effects of glucose on E. coli growth.  With these constructs, 0.02% Ethanol was detected.
</P><P><U>Bay Laurel Thioesterase</U></P><P>MT: The sequencing results of sigma 70/ BTE in pSB1C3 showed a succseful clone and is ready for submission to iGEM. 
A Free Fatty Acid Assay was done on the BTE with sigma 70 promter in Iq cells. The samples were also ran through a Gas Chromotagrapher to determine concentration of C-12 fatty acids. ( For more specific results please look in Enymology section.) <FONT color="blue">
Cyano </FONT></P><P><U>ThiE/GLF:</U></P><P>VG:The sequencing data for GLF was checked and confirmed the presence of GLF in pSB1C3. Thus, we will submit the DNA as a part to the iGEM Registry of Biological Parts.
A temperature gradient from 50-70C was run for a PCR reaction of GLF.  This gradient did not yeild satisfactory results.</P><P><U>AGP/INV:</U></P><P>Procedure: PCR amplification was used to linearize the agp/vector backbone. Three PCR attempts were made using either &quot;back-to-back&quot; primers (i.e. AGP_KNR forward and KN_AGP reverse) or INV_AGP forward and AGP_KNR reverse primers. PCR was executed with an initial annealing temperature of 60 degrees C and verified on 0.7 % agarose gel at 13.8 V/cm.
</P><P>Results: The electrophoresis indicated that the agp/vector was not linearized by PCR. No band was observed at the expected size of agp+topo vector of 4268 bp.
</P><P><FONT color="green">
Enzymology
CL: This week we performed our third FFA assay once again using the FFA kit from BioAssay Systems.  We tested 8 samples (BTE with Sigma70 promoter in Iq cells)along with two negative controls (Sigma70 in Iq cells).  Results proved positive as sample absorbency was significantly higher than those of the negative conrols (0.069 vs 0.037).  We tested the positive samples on the GC, but did not obtain conclusive results as none of the samples had fatty acids present. We believe there was a protocol error as we most like evaporated the samples too far and evaporated our sample FA's along with the extraction solvent.  We will prep for another GC test.    </FONT>
Media
We began testing the apparatus to ensure that there are no leaks in the system and that cross-contamination cannot occur between the chambers.  We ran the apparatus overnight with a culture of I<SUP>q</SUP> cells in LB with 34 µg/mL chloramphenicol.  No contamination was seen after 8 hours.  However, after 30 hours, <I>E. coli</I> had spread to the cyano chamber.  Our best guess is that contamination occurred 18-24 hours into the experiment.  Indicating that the o-rings are not providing quite a tight enough seal between the dialysis tubing and its glass fitting.  But the water pump did survive overnight, and no leaks were observed.  The apparatus will be sterilized and we will attempt to grow a culture without contamination again.
</P><P>Made 3 liters of BG-11 with 50 mM glucose for growing a large culture of <I>Synechocystis</I>.  By the next day, the vessel we were holding the BG-11 in had ruptured.  Remaining media was transferred to three 1 liter bottles and re-autoclaved.  This media was used to start two 500 mL cultures of cyano.  </P><H2><SPAN class="mw-headline" id="Week_17_-_September_19th-28th"><B>Week 17 - September 19th-28th</B></SPAN></H2><P><FONT color="red">E. Coli <U>Trc Promoter</U>
LT:Psb1C3 was digested using Eco RI and Pst I, incubated for 1 hour at 37°C, and heat deactivated at 80°C for 20 minutes. The restriction digest was run on a 0.7% agarose gel. Results: The digest was not complete. There is plasmid DNA cut with only one of the two enzymes. However, most of the plasmid DNA was successfully digested with a band at approximately 2040bp and 900 corresponding to the plasmid and red fluorescent protein. 
Psb1C3 and the trc PCR products were ligated using sticky end ligation in a 1:1 ratio and 10:1 (plasmid:insert) ratios then transformed into IQ cells. Results: The transformation was unsuccessful because a chloramphenicol vector was transformed into chloramphenicol resistant cells. 
</FONT></P><P><U>Pyruvate Decarboxylase &amp; Alcohol Dehydrogenase</U>
JC:  To further improve ethanol production, we contacted UniPavia, who in 2009 was able to attain 2% ethanol production.  They used 10% glucose and induced cultures with 1% ethanol (to improve ADH activity).  

The EnzyChrom Ethanol Detection Kit was used again to test ethanol production in s70/PDC/ADH in NEB Iq E. coli cells. To mimic the growth medium of UniPavia 2009 iGEM team, cultures were grown over 48h with the addition of 10% glucose, to test the effects of glucose on E. coli growth and induced with 1% ethanol at 24hours. With these constructs, 0.018% Ethanol was detected. 
<FONT color="blue">
Cyano </FONT></P><P><U>ThiE/GLF:</U></P><P>VG:GLF and INV were added to the Nevada 2011 iGEM team page as parts and a DNA sample of each was sent to Meagan Lizarazo at MIT.
A new temperature gradient was run for GLF from 40-60C.  This yielded bands that were extracted, gel purified, and ran through another pcr reaction on another gradient from 40-65C.  This was checked on a 1% agarose gel.  The gel yielded two promising bands.  These were extracted and purified with a gel extraction kit.  Then another pcr reaction was set up for these with annealing temperatures of 55C (x5) and 50C (x30).</P><P><U>AGP/INV:</U></P><P>Procedure: An alternative strategy for constructing KNR+petBD was attempted using PCR to create &quot;bridges&quot; between the two gene segments. PCR was performed on pSB1C3 containing petBD and pUC4K containing KNR using picomolar amounts of their common &quot;bridge&quot; primers (KNR_petBD reverse and forward). The PCR reaction was verified on 0.7% agarose gel at 60 V.
</P><P>
Results: The electrophoresis data for the bridge PCR of KNR+petBD was inconclusive. A band was observed at around 1.3-1.5 kb for the reaction which is consistent with the expected size of 1520 bp. However, the gel had too low of a concentration for the 200 bp separation required to differentiate between KNR and KNR+petBD. Thus, there has been no successful gene segment constructions from the individual parts as of yet. 
<FONT color="green">
Enzymology
This week we performed our final ethanol assay once again using the BioAssay systems Ethanol Assay kit.  Two sample of ADH/PDC/sigma70 in Iq cells were tested along with two negative controls (BTE/Sigma70 in Iq cells).  Results proved positive as there was a calculated average ethanol production of 0.019% ethanol for the samples compared to 0.006% in the negative controls.</FONT>
Media
The apparatus was disassembled and cleaned.  All glassware and rubber fittings were autoclaved.  The water pump and piping were bleached.  The apparatus was reassembled, this time with the dialysis tubing just prior to the o-rings cinched with dental floss to further prevent <I>E. coli</I> from escaping.  It was then filled with BG-11, 0.20% casaminos and 50 mM glucose.  100 mLs of an established <I>Synechocystis</I> culture was introduced to the cyanobacteria chamber and 10 mLs of an overnight I<SUP>q</SUP> culture was introduced to the <I>E. coli</I> chamber.  Overnight contamination was observed.  The contaminant was a bright blue color and it was thought that somehow the cyanobacteria had made it into the <I>E. coli</I> chamber, although it is unknown how this could occur.
</P><P>It was discovered that all of our cyanobacteria cultures had died.  Another experiment was performed with only <I>E. coli</I>.  Because we were still having problems with cross-contamination, we placed heat-shrink tubing around the o-ring sittings before autoclaving the glass support for the dialysis tubing.  It was reassembled and again filled with BG-11 supplemented with casaminos and 50 mM glucose.  No contamination was observed overnight.  However, our <I>E. coli</I> culture turned bright blue overnight, with a peak absorption of 400 nm determined by spectrophotometry.  Experiments will need to be done with antibiotics in the future to rule out contamination.
</P><P>A growth curve was also performed this week to determine if σ<SUP>70</SUP>/BTE in I<SUP>q</SUP> cells would grow normally.  Samples were also collected and spun down for testing on a fatty acid assay and GC.  The results showed that there was no inhibition of growth relative to our control (σ<SUP>70</SUP> without BTE).
</P><P>Media was also made for toxicity tests of I<SUP>q</SUP> cells in BG-11 with ethanol.  However, ethanol evaporated from the media before tests could be done, invalidating any data that could have been collected.</P></DIV></DIV></DIV><DIV id="bottom-content"><DIV id="sponser-banner-content"><DIV class="Sponser-text">SPONSORS</DIV></DIV><DIV id="sitemap-banner-content"><DIV id="subNav"><DIV class="subNavText1">
Navigation
</DIV><DIV class="subNavText2"><A href="/Team:Nevada">
Home
</A>     <A href="/Team:Nevada/Team">
Team
</A>     <A href="/Team:Nevada/Project">
Project
</A>     <A href="/Team:Nevada/Parts">
Parts
</A>     <A href="/Team:Nevada/Notebook/Weeks14">
Notebook
</A>     <A href="/Team:Nevada/Outreach">
Outreach
</A>     <A href="/Team:Nevada/Contributions">
Contribution
</A>     <A href="/Team:Nevada/Fundraising">
Fundraising
</A>     <A href="/Team:Nevada/Reference">
References
</A>     <A href="/Team:Nevada/Safety">
Safety
</A></DIV></DIV></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:Nevada/Notebook/Weeks1316">http://2011.igem.org/Team:Nevada/Notebook/Weeks1316</A>&quot;</DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Nevada/Notebook/Weeks1316" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Nevada/Notebook/Weeks1316" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Nevada/Notebook/Weeks1316&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Nevada/Notebook/Weeks1316&amp;oldid=202925" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>