"
Page
Discussion
View source
History
teams
Log in
 
Team:Nevada/Project/Cyano
From 2011.igem.org
So Happy Together
A Cooperative Relationship between Cyanobacteria and E. Coli
for production of biofuels
   Home   
|   Team   
Nevada's Team
Gallery
|   Project   
Background
E. coli Project
Cyanobacteria Project
Assay Development
Co-Cultivation
Results Summary
References
|   Parts   
|   Notebook   
Weeks 1-4
Weeks 5-8
Weeks 9-12
Weeks 13-16
|   Human Practices   
iGEM Collaborators
Outreach
Contributions
Fundraising
|   Safety   
Contents
1 Cyanobacteria
1.1 Introduction
2 Goals and Specific Tasks for Cyanobacteria
2.1 Engineer Synechocystis to overproduce hexose sugars
2.1.1 AGP K.O./INV Construct Design
2.1.2 AGP K.O./INV Construct Components
2.2 Gibson Assembly of AGP K.O./INV Components
2.3 Engineer Synechocystis to secrete hexose sugars to the medium
2.3.1 ThiE K.O./GLF Construct Design
2.3.2 ThiE K.O./GLF Construct Components
2.4 Gibson Assembly of ThiE K.O./GLF Components
3 Results
4 Future Directions
Cyanobacteria
Introduction
Cyanobacteria are gram negative photosynthetic bacteria that have the unique ability to also fix atmospheric nitrogen gas into a biologically useful form. As such they are one of the few truly autotrophic organisms in regards to carbon and nitrogen nutrition. In this project we are engineering the cyanobacterium, Synechocystis PCC 6803, to provide glucose as a carbon source for a biofuel producing E. coli strain. The rationale behind this effort is that if successful we will be able to decrease the cost of growth medium and develop a self-sustaining feeder system that can be adapted to existing microbial fermentation systems that have been optimized for the production of biofuels and other high value compounds. Synechocystis PCC6803 was chosen for this project because it is widely used as a model cyanobacterial system. As such it, it has numerous resources available to the investigator including facile and established transformation protocols and a completely sequenced genome. The work being done on this project will set the foundation for basic research into microbial community interactions and also applied research with real world industrial relevance.
Goals and Specific Tasks for Cyanobacteria
The goal of this project is to engineer the cyanobacterium strain Synechocystis PCC6803 to produce and secrete hexose sugars that can be utilized by a biofuel producing E. coli strain. To achieve this goal the Cyanobacteria team has been assigned following two tasks:
1) Engineer Synechocystis to overproduce the hexose
2) Engineer Synechocystis to secrete hexose sugar to the medium
Engineer Synechocystis to overproduce hexose sugars
Cyanobacteria use photosynthesis to provide energy and carbon skeletons for anabolic processes. During the day excess fixed carbon can be converted to the polysaccharide, glycogen and stored for later use.  To engineer Synechocystis to overproduce hexose sugars, we will divert carbon away from the glycogen biosynthetic pathway and towards hexose sugar production. To achieve this goal we will create a null mutation in the gene encoding ADP glucose pyrophosphorylase (AGP). AGP specifically converts glucose to ADP-glucose which is the monomeric precursor to glycogen. Synechocystis AGP knockout mutants have been reported to no longer produce glycogen, but instead accumulate high levels of sucrose (Miao et al (2003) FEMS Letters 218: 71-77). Fortunately, sucrose can easily be converted to the hexose sugars, glucose and fructose, in a reaction catalyzed by the enzyme invertase (INV). Therefore, we will introduce and overexpress the INV gene in the AGP mutant background.
AGP K.O./INV Construct Design
We will simultaneously create the AGP mutant and the INV overexpressing line by inserting the INV expression construct into the coding region of the Synechocystis AGP gene. This will be accomplished by creating an AGP gene replacement construct in which the INV overexpression gene cassette and a kanamycin resistance gene cassette are inserted into the coding region of a subcloned AGP gene. This dysfunctional AGP construct will then be transformed into Synechocystis and through homologous recombination, replace the native AGP gene.
AGP K.O./INV Construct Components
ADP Glucose Pyrophosphorylase (AGP) Gene: The slr1176 open reading frame was PCR amplified from total Synechocystis PCC6803 genomic DNA and subcloned into pSB1a3. Forward and reverse primers were designed with 5’ extensions containing iGEM prefix and suffix sequences.
Invertase (INV) Gene+double terminator: The INV gene was synthesized based on the sequence of a Zymonomas mobilis invertase gene described by Neiderholtmeyer et al. (Applied and Environmental Microbiology (2010) 76: 3462) which was codon optimized for expression in cyanobacteria. The gene was also synthesized with a double transcriptional terminator designed from sequences described for iGEM part BBa_B0015. The synthetic gene was subsequently subcloned into pSB1C and has been designated BBa_K558006.
petBD promoter+RBS: The strong light inducible petBD promoter will be used to drive the expression of invertase gene in Synechocystis. The petBD+RBS (BBa_K390015) was generously provided to us by the Utah State iGEM team.
Kanamyacin resistance cassette (KanR): The kanamycin resistance cassette was PCR amplified from pUC4K (Taylor and Rose (1988) NAR 16:358). This cassette includes the neomycin phosphotransferase 2 gene with its own constitutive promoter and transcriptional terminator. This cassette is frequently used to select for positive transformants in Synechocystis.
Gibson Assembly of AGP K.O./INV Components
Primer Design: Forward and reverse primers for each DNA part were designed with 20 base pair overlapping sequences with the upstream and downstream flanking segments. This will create 40 base pair overlaps between neighboring parts in the construct.
PCR: Inv, KnR, and petBD+RBS were amplified from BBa_K558006(Nevada), pUC4K, and K390015(Utah State) respectively under standard PCR conditions. AGP in pSB1A3 will be amplified to include the vector. This will be used as the backbone for Gibson Assembly.
Assembly of Parts: The above parts will be assembled into the following construct
Primers developed for PCR and Gibson Assembly for INV operon. Overlapping DNA sequences can be created through PCR by adding twenty base-pairs of the end of your first genetic sequence to the beginning of your second genetic sequence. A blunt-end 40 base-pair sequence is then created that is ideal for Gibson Assembly.
Transformation: Synechocystis will be naturally transformed via homologous recombination.
Engineer Synechocystis to secrete hexose sugars to the medium
Synechocystis lacks the ability to secrete hexose sugars. Therefore it was necessary to introduce a sugar transporter through genetic engineering. Previous studies (Nedierholtmeyer et al (2010) Applied and Environmental Microbiology (2010) 76: 3462) showed that by expressing the Zymonomas mobilis glucose facilitative transporter (GLF) gene in Synechococcus, glucose and fructose were detected in the culture medium at levels as high as 30 uM and 150 uM respectively. Based on these results we decided to express GLF in Synechocystis. Our hope is that by expressing GLF in the AGP KO/INV overexpressing line, we will be able to produce enough glucose to the surrounding medium to support the E. coli biofuel production.
ThiE K.O./GLF Construct Design
To integrate the GLF gene into the Synechocystis genome, we have decided to use an alternative insertion site. In this case we will insert the GLF overexpression gene cassette along with a chloramphenicol resistance cassette (CamR) into the coding region of the thiamin monophosphate pyrophosphorylase (ThiE) gene. The rationale for creating a ThiE knockout mutant is that it will lead to the creation of an auxotophic mutant that will only survive when the grown in the presence of thiamin (Vitamin B1). Therefore, the transgenic Synechocystis will not be able to survive outside laboratory and the chances of environmental contamination will be decreased. We will simultaneously create the ThiE mutant and the GLF overexpressing line by creating an ThiE gene replacement construct in which the GLF overexpression gene cassette and a chloramphenicol resistance gene cassette are inserted into the coding region of a subcloned ThiE gene. This dysfunctional ThiE construct will then be transformed into Synechocystis and through homologous recombination, replace the native ThiE gene.
ThiE K.O./GLF Construct Components
ThiE Gene: The sll0635 open reading frame was PCR amplified from total Synechocystis PCC6803 genomic DNA and subcloned into pSB1a3. Forward and reverse primers were designed with 5’ extensions containing iGEM prefix and suffix sequences.
Glucose facilitative transporter (GLF) Gene+double terminator: The GLF gene was synthesized based on the sequence of a Zymonomas mobilis invertase gene described by Neiderholtmeyer et al. (Applied and Environmental Microbiology (2010) 76: 3462) which was codon optimized for expression in cyanobacteria. The gene was also synthesized with a double transcriptional terminator designed from sequences described for iGEM part BBa_B0005.
petBD promoter+RBS: The strong light inducible petBD promoter will be used to drive the expression of invertase gene in Synechocystis. The petBD+RBS (BBa_K390015) was generously provided to us by the Utah State iGEM team.
Chloramphenicol resistance cassette (CamR): The chloramphenicol resistance cassette was PCR amplified from pSB1C3. This cassette includes the chloramphenicol acetyltransferase gene with its own constitutive promoter and transcriptional terminator. This cassette is frequently used to select for positive transformants in Synechocystis.
Gibson Assembly of ThiE K.O./GLF Components
Primer Design: Forward and reverse primers for each DNA part were designed with 20 base pair overlapping sequences with the upstream and downstream flanking segments. This will create 40 base pair overlaps between neighboring parts in the construct.
PCR: GLF, CamR, and petBD+RBS were amplified from BBa_K558005(Nevada), pSB1C3, and K390015(Utah State) respectively under standard PCR conditions. ThiE in pSB1A3 will be amplified to include the vector. This will be used as the backbone for Gibson Assembly.
Assembly of Parts: The above parts will be assembled into the following construct
Primers developed for PCR and Gibson Assembly for GLF operon. Overlapping DNA sequences can be created through PCR by adding twenty base-pairs of the end of your first genetic sequence to the beginning of your second genetic sequence. A blunt-end 40 base-pair sequence is then created that is ideal for Gibson Assembly.
Results
Figure 1. Confirmation of invertase operon parts. PCR was performed to amplify petBD+RBS, KanR, and inv with 20 bp flanking overlap regions.  Equimolar volumes of each PCR product were mixed and ran on 0.7% agarose gel at 110 V for 60 minutes (lanes 2-9). 0.5 ug Kb+ standard was loaded into lanes 1 and 10.
Figure 2. Confirmation of agp in TopoIIBluntPCR vector. Agp was ligated into TopoIIBluntPCR vector (Invitrogen). 0.25 ug of each ligation was digested with SpI and PstI and ran on 0.7% agarose gel at 110 V for 60 minutes (lanes 2-8). 0.5 ug Kb+ standard was loaded into lanes 1 and 9.  Lanes 3,5, and 8 confirm successful ligation.
Figure 3. Agarose gel electrophoresis of cleaved and uncut thiamin monophosphate pyrophosphorylase (ThiE) DNA. 0.5 ug of DNA samples were first isolated before a small portion was cleaved using EcoRI and Pst I restriction enzymes. Four cut samples and uncut DNA was separated by electrophoresis in a 1% agarose gel and visualized under a UV light. The results show a successful ligation of ThiE.
Figure 4. Agarose gel electrophoresis of PCR amplified chloramphenicol acetyltransferase (CmR) DNA. 5 ng of DNA was first amplified using PCR. The DNA product was separated by electrophoresis in a 1% agarose gel and visualized under a UV light. The results show a successful amplification of CmR.
Figure 5. Agarose gel electrophoresis of PCR amplified photosynthetic cytochrome b6 and subunit 4  (petBD) DNA. 4 ng of DNA was first amplified using PCR. The DNA product was separated by electrophoresis in a 1.5% agarose gel and visualized under a UV light. The results show a successful amplification of petBD.
Figure 6. Agarose gel electrophoresis of PCR amplified Zymonomas mobilis glucose facilitative transporter (GLF) DNA. Three different samples of 10 ng of GLF DNA was first amplified using PCR. The DNA products was separated by electrophoresis in a 1% agarose gel and visualized under a UV light. The results show a successful amplification of GLF.
Future Directions
Each of the gene parts for the assembly of both the AGP KO/inv operon and the ThiE KO/GLF operon have been isolated and confirmed by gel electrophoresis. The primary goal now is to successfully construct each operon by Gibson assembly or SLIC. Once the operons have been confirmed transformation of Synechocystis will be carried out in two successive steps: first with the AGP KO/inv operon and, secondly, with the ThiE KO/GLF operon.
SPONSORS
Navigation
Home       Team       Project       Parts       Notebook       Outreach       Contribution       Fundraising       References       Safety
Retrieved from " http://2011.igem.org/Team:Nevada/Project/Cyano "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
