"
Page
Discussion
View source
History
teams
Log in
 
Team:Nevada/Notebook/Weeks912
From 2011.igem.org
So Happy Together
A Cooperative Relationship between Cyanobacteria and E. Coli
for production of biofuels
   Home   
|   Team   
Nevada's Team
Gallery
|   Project   
Introduction
E. coli Project
Cyanobacteria Project
Assay Development
Co-Cultivation
Results Summary
References
|   Parts   
|   Notebook   
Weeks 1-4
Weeks 5-8
Weeks 9-12
Weeks 13-16
|   Human Practices   
iGEM Collaborators
Outreach
Contributions
Fundraising
|   Safety   
E. Coli
Jovanna, Megan, Lauren, Sam, Dafne, Tutku & Destiny
The E.coli group is responsible for making the constructs which will produce fatty acids and ethanol.
Cyano
Megan, Dru, Vadim, Marguerite, Laura, David, Jen, Ron & Chris
The cyano group is responsible for increasing glucose production and secretion in cyanobacteria.
Assay
Casey, Sam, Vadim, Majid, Megan, and Jovanna
The assay group is responsible for testing and verifying the products made by our system.
Media
Bryson & Matt
The media group is responsible for creating the best possible co-cultivation system for E.coli and cyanobacteria. They are also responsible for creating a co-cultivation apparatus.
Description :
BTE-Bay Laurel (Plant) thioesterase gene- produces 12 and 14 carbon medium chain fatty acids that can be converted to biofuels.
PDC/ADH- pyruvate decarboxylase/aldehyde dehydrogenase operon. This part can be used to produce ethanol. PDC converts pyruvate to acetaldehyde. Alcohol dehydrogenase can then turn acetaldehyde into ethanol.
ThiE-Thiamine E synthesizing gene in cyanobacteria. The enzyme is usually referred to as thiamine monophosphate pyrophosphorylase. Important for vitamin B synthesis in cyanobacteria. Knockout of this gene will allow for production of an auxotroph in cyanobacteria.
sigma70 promoter- constitutive promoter in E.coli.
trc promoter- inducible promoter in E.coli. It is not sensitive to changes in glucose concentrations.
GLF- glucose facilitator gene responsible for transporting glucose out of the cyanobacteria.
INV- invertase gene responsible for breaking down sucrose into glucose and fructose. Gene is from cyanobacteria.
Weeks 1-4     Weeks 5-8     Weeks 9-12     Weeks 13-END    
Calender Weeks 9-12
Contents
1 Week 9 - July 25th-31st
2 Week 10 - August 1st-7th
3 Week 11 - August 8th-14th
4 Week 12 - August 15th-21st
Week 9 - July 25th-31st
E. Coli
Trc Promoter
LT:Four colonies (1,4,9,& 15) from the initial 15 were selected and inoculated in LB-Amp broth. Cultures were purified using the QIAquick miniprep kit and analyzed using nanodrop spectrophotometry.  Plamid DNA was then submitted to the Nevada Genomics Center for sequencing using the M13 Forward and Reverse primers. Results: The sequencing data confirms the TRC promoter was successfully cloned into the TOPO2.1 PCR vector!
Colonies #1,4,9, & 15 were re-cultured in LB-Amp broth, purified using the QIAquick miniprep kit, and analyzed using nanodrop spectrophotometry.  An Eco RI and Spe I digest was set up and run on a 1.2% gel to isolate the TRC promoter in preparation for ligation with the BTE and ADH/PDC genes.  Results: The restriction digest showed large amounts of star activity and no 100b.p. band. There is a dominant band at approximately 3.9 kb which is what we expected for the plasmid size however, due to poor imaging it is impossible to see a 100bp band corresponding to the Trc promoter on the gel.
The digest above was repeated with minor changes. The DNA was digested at 37 degrees Celsius instead of digesting overnight at room temperature and high fidelity Eco RI was used. Results: There is a band at approximately 3.9kb however, the gel was run for too long and it appears that any 100bp band ran off the gel.
Bay Laurel Thioesterase
MT: Primers were ordered for amplification of the BTE coding region
Cyano
Thie/GLF:
7/25/11:  Chloramphenicol vector was digested with EcoRI and checked on 0.7% gel.  The results for this gel were good.  ThiE/Topo Vector was digest with EcoRI and PstI and checked on a 0.7% gel.  The gel was not very clean, so we will re-run tomorrow (7/26).   The PetBD and GLF PCR was done at 63.5C.  Positive control/primer controls were added to the PCR reaction. After confirmation of ThiE it will be put into plasmid PSB1C3 and sequenced.    7/26/11:  Gels for PCR products of PetBD and GLF were run with no results.  We believe this is due to errors in the master mix.   ThiE digest was re-run on a 0.7% gel.  The gel showed that the ThiE was not digested.  (7/27).   Ran PCR reaction on ThiE gel purification 7/18/11 with positive control and primer control at 54.3C for Topo Cloning tomorrow (7/27). 7/27/11:  Ran gel on + control, Primer control and ThiE to be used for Topo cloning.  Gel was completely empty aside from ladders.  Did digest on ThiE in Topo Vector with ecoRI.  Results showed that we have an empty topo Vector on all three digests (#'s 2, 3 and 5).
AGP/INV:
Procedure: KnR was to be dropped out from the pUC4k vector using an EcoRI digest. KnR would then be purified from the gel in order to use as a template for the PCR of KnR for gibson assembly. .5 ug of pUC4k was digested with EcoRI and incubated at 37 degrees C for 1 hour. The digest was ran on .7% gel at 110 V for 1 hour.  Results: The EcoRI digest was unsuccessful as indicated by cut plasmid being identical to the uncut control. Thus, the EcoRI enzyme most likely needs to be reordered.
VG:2mg of Invertase was digested with EcoRI and PstI and run on a 1% agarose gel. The INV fragment was then cut from the gel and purified. The purified INV DNA and pSB1C3 were ligated at a 2:1 ratio of gene to plasmid. The ligation was then transformed in NEB10β E.coli cells. Unfortunately, the colonies were pink, suggesting that the RFP was not digested out of the pSB1C3 vector prior to the ligation. Thus, the resulting colonies were probably all pSB1C3 without the INV insert. 1 mg of pSB1C3 vector DNA was then digested with EcoRI and PstI to drop out the RFP, which is 800bp in length. The digest was not successful, since the gel didn’t show a fragment at 800bp.
Enzymology
CL:This week we performed our time course growth on the BTE9 sample.  We took culture samples every two hours and grew an overnight culture as well.  We performed the FFA assay on every time course sample using the BioAssay Systems FFA assay kit with slight modifications to the manufactures protocol.  The results were negative for all samples which could have possibly been due to improper storage of the assay kit.  Another assay kit was ordered and samples are to be further tested.
Media
The goal of this week was to find a standard growth curve for cultures grown in BG-11 supplemented with tryptone.  Because tryptone acts as an additional source of both carbon and nitrogen, we need to determine a tryptone concentration that negates the auxotrophies of our strains but is still deficient in the amount of carbon provided.  By maintaining a moderate carbon deficiency, we will still be able to test for successful passage of glucose from cyanobacteria to E. coli.  For the standard curve, we are looking for a tryptone concentration that will produce an OD of nearly 1.000 at 600 nm after 24 hours.
BW: Media for this experiment was made, both in 250 mL flasks and 16x25 mm culture tubes.  This was done to again confirm that culture tubes did not provide sufficient aeration during incubation.  The experiment to find a standard curve will be done next week, and media will be stored in the cold-room until then.
MB: Meeting with IGEM members to discuss apparatus requirements and design criteria. Criteria outline was all materials had to be easily sterilized, a pump needed to transfer E coli. fluid through dialysis tube in order to allow adequate interaction of the two medias.  Provided were two chromatography tubes that would act as the chambers for the different media they were suspended on a chemistry stand. Oxygen needed to be provided to both chambers of media. The chamber containing cyanobacteria required lighting to provide photosynthesis. During the meeting group members began brainstorming for ideas on what materials are suitable and basic design concepts. Post meeting research began into materials to be ordered and combining ideas to create a suitable apparatus. Plans were drawn up and revised to represent the planned apparatus.
Week 10 - August 1st-7th
E. Coli
Trc Promter
LT:Colonies #1,4,9, & 15 were re-cultured in LB-Amp broth and purified using the QIAquick miniprep kit. DNA was concentrated using salt solution and ethanol, resuspended in 10ul sterile water, and analyzed using nanodrop spectrophotometry. Results: DNA concentration was unsuccessful. Nanodrop spectrophotometry indicated less than 10ng/ul of DNA in each sample.
Colonies #1,4,9, & 15 were re-cultured in TB-Amp broth to increase DNA concentrations, purified using the QIAquick miniprep kit, and analyzed using nanodrop spectrophotometry. A HF Eco RI and Pst I digest was set up, digested for an hour, and run on a 1.2% gel. Both uncut and cut samples of DNA were loaded onto the gel. Results: There is no visible band at 100bp like we expected. There are bands greater than 3.5kb corresponding to the TOPO2.1 PCR vector. It appears that the digest was unsuccessful, however it is possible that the trc promoter was digested out but isn’t visible on the gel due to its small size.
Pyruvate Decarboxylase & Alcohol Dehydrogenase
JC:  PDC/ADH was re-digested with XbaI/PstI and run on a 1% agarose gel to confirm digestion.  Bands obtained confirmed PDC/ADH (3054bp), and pSB1C3 (2070bp).  PDC/ADH-XbaI/PstI and σ70/pSB1A3-SpeI/PstI were re-ligated and transformed into NEB10 β cells.
Bay Laurel Thioesterase <br MT: BTE coding region was amplified using PCR with taq polymerase.  T.A. Topo cloning was done on the PCR product, but transformation was not successful.
Cyano
ThiE/GLF:
After ordering newly designed primers, this week focused mostly on PCR amplification of: AGP, Kan Resistance, the promoter pETBD and Invertase. The initial amplification of AGP (ADP-glucose-pyrophosphorylase)(8/1) was performed using old primers and the fragment observed was approx. 700 bp too short. Transformation of the vector pUC4k into E. coli was performed to produce a large amount of vector for the PCR of Kan Resistance. Digestion of pUC4k with EcoRI yielded the correct 1301 bp fragment indicating the presence of Kan R in the vector. 8/3 - 8/4 PCR results showed successful amplification of Kan Resistance and pETBD, 1341 and 219 bp respectively.
Created new sterile LB stock.  Poured new plates with LB + Ampicillin, LB + Chloraphenicol, and LB + Kanamycin.  Ran PCR of cyanobacteria genomic DNA to isolate ThiE at 54.3C.  Gel for ThiE isolation showed bands at the appropriate length, just over 1kb.  Purified ThiE from the gel.  Nanodrop gave readings of an average of 7.45ng/ul.  Transformation of cultures will be done later in the week with blue/white selection.  Waiting for new primers, as soon as they come in to the lab we will begin isolation of PetBD, GLF and Chloraphenicol.  Set up PCR reactions for promoter petBD and ThiE with positive control.  PetBD was successful, but with the wrong primers.  ThiE still showing two distinct bands, will perform gel purification to try to clear up. Received new primers for PetBD and Chloamphenicol.  Ran PCR with new primers and petBD was successful and is now ready for Gibson assembly.  Chloramphenicol was not successful, will run a temperature gradient on it next week. Performed gel purification on ThiE and ran on a 1% gel.  Also performed PCR reactions for GLF and Chloramphenicol and ran on a 1% gel.  Only band present was for GLF, but not strong or clear.
AGP/INV:
VG:1 mg of pSB1C3 was again digested with EcoRI and PstI and run on a 1% agarose gel. This time an 800bp fragment (RFP) was successfully cut out of the vector. 0.1mg of gel purified INV and .05mg of gel purified pSB1C3 were ligated and transformed in NEB10β E.coli cells. Unfortunately, no colonies grew. The transformation of the gel purified INV and pSB1C3 was repeated in NEB10β E.coli cells. Unfortunately, no colonies grew again. Thus, we decided to digest INV out of the pUC57 vector and ligate it with digested pSB1C3 vector.  GLF samples 3 and 4 were sent in for sequencing with the new primers. 0.5 mg of INV and 0.5 mg of pSB1C3 were digested with EcoRI and PstI. The digests were then run on a 0.7% agarose gel. The gel showed a successful digest of both INV and pSB1C3. The digested DNA was then concentrated with 3M sodium acetate and 95% ethanol. Unfortunately, the DNA pellets were lost during the concentration process.
Enzymology
CL:We began to focus on proof of ethanol production as the E.coli part modified to produce ADH and PDC was ready to be tested.  We purchased an Ethanol Assay Kit once again from BioAssay Systems and performed the assay on eight different cell lines plus two negative controls (sigma70 promoters) and four standards (included in the kit).  Results proved to be negative as there was no visible color change in the samples and there absorbencies were no higher than that of the negative control.  We opt to take a step back and determine where the problem was.  The sequencing report determined that both the ADH and the PDC genes were present, but were they being expressed? We chose to use aldehyde detecting plates to determine if the PDC enzyme was begin produced.  We also decided to write up a protocol to lyse the modified cells themselves and assay for ADH activity instead of ethanol presence.
Media
BW: Readings on contents of tryptone showed that it contains carbohydrates.  This presented a problem, as we need an amino acid supplement that is cleaner and does not introduce additional carbon sources.  Casamino acids were chosen as an alternative, as they provide a similar amino acid content to tryptone, but are free of carbohydrates and contain greater quantities of free amino acids and small peptides.
A test was done to confirm that casamino acids would be the better alternative for a standard curve and to find the appropriate concentration.  0.25%, 0.50%, and 1.0% tryptone in BG-11 were compared with 0.25%, 0.50%, and 1.0% casamino acids in BG-11.  This experiment was done using 10-ß cells.
The results showed that 0.25% casaminos are sufficient to produce the desired standard curve of roughly 1.0.  This means that casamino acids more adequately provide the amino acids needed by 10-ß and are a more pure source of them.  The data also indicated that there was no noticeable difference between using flasks with 100 ml cultures and culture tubes.  Future experiments will be done in culture tubes to increase the number of samples that can be done at any one time.
MB: Selected materials were ordered and collected to begin construction. All the aluminum for the base of the apparatus was plasma cut to selected size, including all cutouts required for added components and features. The oxygen pump was designed and built to meet size requirements in the base. The stand was fastened to the base along side oxygen pump.
Week 11 - August 8th-14th
E. Coli
Trc Promoter
LT:The Eco RI and Spe I digest from week 10 was repeated with new controls: uncut, Eco RI only digest, and Spe I only digest. Half the digest was run on a 1.2% gel and the other was saved for ligation. Results: Again there is no visible, 100bp band corresponding to the trc promoter.
The BTE gene in pSB1C3 was digested using Eco RI and Xba I. This allows trc to be placed in front of the gene. Half the digest was run on 1.0% gel and the other half was saved for ligation. Results: The gel began to run towards the top of the gel instead of the bottom. The cords were changed and the gel ran in the right direction. The bands were fuzzy but the digest still appeared successful.
Pyruvate Decarboxylase & Alcohol Dehydrogenase
JC:  Single colonies from LB-Amp plates were selected and tested on LB-Amp and LB-Chlor plates.  Liquid cultures grown in LB-Amp and minipreps and nanodrop analysis performed.
To verify presence of PDC/ADH genes with the σ70 constitutive promoter, 0.5ug of PDC/ADH/σ70/pSB1A3 was digested with EcoRI and PstI.  Digests were run on a 1% agarose gel, and bands confirmed σ70/PDC/ADH (3089bp) and pSB1A3 (2155bp).
Bay Laurel Thioesterase
MT: BTE coding region was amplified using PCR with phusion polymerase.  Zero Blunt Topo cloning was done on the PCR product, but transformation was not successful.
Cyano
ThiE/GLF:
Re-ran a PCR reaction for ThiE, GLF, and a temperature gradient on Chloramphenicol.  Found optimal temperature for Chloramphenicol to be 55.4C.  Will perform a gel purification on this band as well as the band for GLF.  ThiE was still not present, will make more genomic DNA to try and isolate fresh samples. Ran a PCR reaction on GLF and Chloramphenicol.  Used gel purification of GLF at 38ng.  Ran another sample of GLF from a previous PCR reaction at a concentration of 45ng.  For the chloramphenicol gel purification we used a concentration of 38ng. We also ran a Chloramphenicol reaction from the plasmid at a concentration of 50ng.  Ran GLF reactions at 67.6C and Chloramphenicol at 55.4C. The positive control was run at 57.7C and had a concentration of 100ng.  The gel for these reactions will be run tomorrow (8/10).  We did attempt to isolate genomic DNA from cyanobacteria cells, however we were unsuccessful.  We will re-PCR a previous successful ThiE PCR reaction and try to purify ThiE from that. Ran gel from PCR reactions done yesterday, CmR looks great! CmR is now ready for Gibson.  PCR of ThiE as looks good, we can perform a gel extraction of this sample for the next step. Tomorrow we will do a PCR of at higher DNA concentrations to see if stronger results can be produced.  Attempted to run another PCR reaction of several different GLF samples with double the amount of DNA (200 ng) and no reactions created a viable reaction. So we are starting over by recombining Genescript GLF stock with new competent cells and culturing overnight. Five single colonies that tested posted with blue/white selection were removed from the overnight cultures and were then cultured in LB medium overnight in a 37 C incubator with 300 rpm shaking . The five colonies were purified using Miniprep before they were nanodropped. GLF – 1 43 ng/ul GLF – 2 67.62 ng/ul GLF – 3 2 ng/ul GLF – 4 2 ng/ul GLF – 5 68.80 ng/ul
GLF samples were then digested with EcoRI and Pst I and were left to digest overnight at room temperature
AGP/INV:
Successful PCR of Kan R and pETBD the previous week allowed us to move forward and perform larger, 100 ul volume reactions with these two genes so that we would have a large stock with which Gibson assembly could be performed as well as a back-stock from which any further PCR of the genes could be performed. 8/8 PCR of AGP from Synechocystis yielded the correct fragment at 1360 bp. Liquid cultures of E. coli transformed with pUC57 were cultured for a large supply of vector containing the synthetic Invertase (INV) gene. Both PCR attempts (8/10 - 8/11) were unsuccessful at selected annealing temperatures.
VG:INV and pSB1C3 were digested again with EcoRI and PstI and checked on a 0.7% agarose gel. The digests appeared correct, with 2 bands for cut INV at 1700bp and 2700bp (pUC57). The digested pSB1C3 product showed one band at 2000bp and a second at 800bp (RFP). The digests were ligated overnight and transformed in NEB10β E. coli cells. White colonies were observed. These colonies were streaked on AMP and CMR plates. Only the CMR plates showed growth. Thus, we decided to perform colony PCR for the samples. The PCR products were checked on 0.7% agarose gels. Only one colony (4) showed a band which may have been Invertase. The positive pSB1C3 control didn’t show up. Thus, we decided to do restriction digests with EcoRI and PstI to see if the INV gene was present in the pSB1C3 vector.
Enzymology
CL:This week we ran the ADH activity assay testing five different ADH/PDC cell lines and one positive (ADH @ 0.75u/ml) and negative (sigma70) standard.  We used a standard lysis buffer to lyse the cells and lysate was used as the enzyme additive for the assay.  Unfortunately, there was still no sign of ADH activity.  Although the positive control tested positive in both assay and lysis buffer, we decided to view sources with similar experiments to get an idea of what conditions we should be running the cell lysis and assay at.  Also prepped was an alcohol oxidase assay which will be used to detect ethanol (if we get that far)!
Media
This week an experiment was done to determine the effect of glucose and ammonium chloride in the presence of casamino acids.  LB and BG-11 without additives were used as the positive and negative controls respectively, with 0.25% casaminos in BG-11 as our standard curve.  A set of samples in BG-11 with 0.25% casaminos and varying levels of glucose were tested at 10 mM, 25 mM and 50 mM concentrations.  Another set was tested in BG-11 with 0.25% casaminos and varying amounts of NH4Cl at 0.5 mg/ML, 1.0 mg/mL, and 2.0 mg/mL concentrations.  Finally, samples were tested in the presence of glucose and ammonium chloride in the following quantities: 10 mM glucose + 0.5 mg/mL NH4Cl, 10 mM glucose + 1.0 mg/mL NH4Cl, 25 mM glucose + 0.5 mg/mL NH4Cl, 25 mM glucose + 1.0 NH4Cl.  10-ß cells were again used.
The results showed that, in the case of this cell line, glucose increased the growth rate but reduced the carrying capacity of the media by 20-30%, relative to the BG-11 + 0.25% casamino standard curve. Samples cultured in the presence of ammonium chloride showed an increased carrying capacity of just over 10%, with the minimal amount tested - 0.5 mg/mL - being no less effective than higher concentrations.  Oddly, those samples that were grown in the presence of glucose and ammonium chloride behaved identically to those that were grown in the presence of glucose alone.
Week 12 - August 15th-21st
E. Coli
Trc Promoter
LT: Despite inconclusive gel results the BTE gene was ligated with the trc promoter using sticky end ligation and transformed into NEB 10 B cells. Transformation was plated onto LB-Amp plates. Results: Colonies grew, however, chloramphenicol plates should have been used to select for colonies containing the trc promoter in front of the BTE gene in the pSB1C3 vector.
Pyruvate Decarboxylase & Alcohol Dehydrogenase
JC:  s70/PDC/ADH was tested for Ethanol production using EnzyChrom Ethanol Detection Kit (BioAssay Systems).  No samples yielded results above background.
ADH enzymatic assay was performed on s70/PDC/ADH constructs.  The presence of ADH was not verified.
Bay Laurel Thioesterase
MT: Topo cloning was successful with a new Zero Blunt Topo cloning kit. Topo cloning done with the PCR product amplified with phusion polymerase, and correct selection.
Cyano
ThiE/GLF:
Digested GLF was then run through a 1% agraose gels with results showing up at the expected size. ThiE from PCR preformed on 8/10 was amplified using Taq polymerase. Using Taq creates “sticky” ends and it is hoped that this will aid us in recombination into our PSB vector. GLF from 8/15 was extracted from the gel and a PCR reaction was performed using Phusion.  PCR reactions of both the ThiE and the GLF were run through a 1% gel. GLF has bands in the desired regions  and ThiE has a very faint band at the desired location. Since the ThiE band is so faint, we will use all of our remaining DNA for a another gel and if that too behaves in the expected manner we will extract it for TOPO cloning.
GLF samples labeled 1-5 from 8/13 and 8/14 were sent to the Nevada Genomics Center for Sequencing. This will confirm that we have GLF and not our sister gene, Invertase (previous samples of GLF and Invertase were confused and since they are the same size and cut with the same restriction enzymes we wanted to be sure that this really was the correct gene since it is behaving during PCR).  The ThiE DNA was then run through a 1% agarose gel and a visible band was seen. This band was then extracted for cloning.
ThiE was placed in competent cells and TOPO cloned. Five single ThiE colonies that tested posted with blue/white selection were removed from the overnight cultures and were then cultured in LB medium overnight in a 37 C incubator with 300 rpm shaking .
AGP/INV:
VG:48 colonies were minipreped and digested with EcoRI and PstI. The digests were run on a 0.7% agarose gel. The gel turned out blank. 0.5mg of minipreped colony 4 were digested with EcoRI and PstI and checked on a 0.7% agarose gel. The gel came out very blurry. The digests were done again and run on a 0.7% agarose gel. This time the digests appeared to be incomplete. We decided to miniprep the first twenty colonies from the pSB1C3/INV ligation in NEB10β E. coli cells. Afterwards, we digested the first twenty colonies and five samples of colony four with EcoRI and PstI and run the products on a 1% agarose gel. Once again we had an incomplete digest.  The first 5 colonies from the pSB1C3/INV ligation in NEB10β E.coli cells were single colony streaked. The isolated colonies were cultured and single line streaked on AMP and CMR plates. Elaine’s ligation of pSB1C3/INV was transformed in NEB10β E.coli cells.  The first 5 colonies from the pSB1C3/INV ligation in NEB10β E.coli cells were minipreped and 0.5mg of pSB1C3 uncut, 0.5mg of pSB1C3 cut with EcoRI and PstI, 0.5mg of pSB1C3 cut with EcoRI only, 0.17446 mg of colony 1 cut with EcoRI and PstI, and 0.17446 mg of colony 1 cut with EcoRI only, rest of colony 1 uncut, 0.2mg of colonies 2-5 cut with EcoRI and PstI and 0.2mg of colonies 2-5 cut with EcoRI only and rest of colonies 2-5 uncut DNA were all run on a 1% agarose gel. The gel showed incomplete digests. Thus, fifty of Elaine’s ligation colonies were streaked on AMP and CMR plates.
Enzymology
CL:This week was our third FFA assay using the new FA assay kit from BioAssay systems testing fresh samples from different cell lines than from before.  Three of the five cultures tested positive for fatty acids. Fresh overnight cultures of these cell lines will be re-assayed along with two sigma70 negative controls.  We performed our final FFA assay (using the assay kit) on 4 samples that had previously tested positive plus three new samples and two negative (sigma70) controls.  All seven samples tested positive with significantly higher absorbencies (α to concentration) than background and the negative controls.  Those samples, plus two fresh negative controls were prepped for GC analysis (extracted with hexane), and GC analysis is scheduled for next week.
We also ran two assays on the ADH/PDC samples.  The first assay was the ADH activity assay using lysed cell samples.  The results proved to be inconclusive as there was no ADH detected but the lysis buffer did not appear to be stable and therefore most likely interfered with final readings.  We then used an Enzychrome Ethanol Assay kit (BioAssay Systems) which yielded negative results as sample reading were similar to background and negative control readings, no ethanol was detected.  The E. coli team is now trying to transform the ADH/PDC construct into high expression Iq cells for further testing.
Media
The first experiment for this week was to test Iq cells grown in BG-11 in the presence of casamino acids to find a standard curve similar to that for 10-ß.  Cultures were grown with 0.1%, 0.25%, 0.50%, and 1.0% concentrations of casamino acids.  After 24 hours, the culture grown with 0.25% casaminos had an OD at 600 nm of 1.174, indicating that this concentration was just slightly greater than desired for a standard curve.  A concentration of 0.20% casaminos will be used with Iq cells in future experiments with this cell line.
Next, an experiment was done to determine the effect of glucose and ammonium chloride on the growth of Iq cells in BG-11 with 0.20% casamino acids.  The concentrations of glucose and ammonium chloride tested were identical to those used with 10-ß cells in week 11.  Unexpectedly, Iq cells reacted quite differently than did the 10-ß cells.  Iq showed a positive response to glucose, with a slightly decreased lag phase and increased carrying capacity.  Iq showed no response to ammonium chloride.
MB:Glasswork begun; several renditions were required for the dialysis tube mount. Also air vents at top were made. Glass chamber caps were drilled and threaded for transfer tubing connections and oxygen check valves. Base assembly was finalized and external components began being assembled to top of the base.
SPONSORS
Navigation
Home       Team       Project       Parts       Notebook       Outreach       Contribution       Fundraising       References       Safety
Retrieved from " http://2011.igem.org/Team:Nevada/Notebook/Weeks912 "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
