<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Nevada_Notebook_Weeks58"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Nevada/Notebook/Weeks58">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Nevada/Notebook/Weeks58&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Nevada/Notebook/Weeks58&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Nevada/Notebook/Weeks58&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Nevada/Notebook/Weeks58" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Nevada/Notebook/Weeks58</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><DIV id="absHeader"><DIV id="banner-box"><DIV id="banner-title-box"><DIV class="banner-text-title">
So Happy Together</DIV><DIV class="banner-subtext">
A Cooperative Relationship between Cyanobacteria and E. Coli
for production of biofuels
</DIV></DIV><DIV id="banner-buttons" class="buttons-text"><UL class="top-level"><LI>  <A href="/Team:Nevada">Home</A>  </LI><LI>
|  <A href="/Team:Nevada/Team">Team</A>  </LI><LI><A href="/Team:Nevada/Team">Nevada's Team</A></LI><LI><A href="/Team:Nevada/gallery">Gallery</A></LI></UL><LI>
|  <A href="/Team:Nevada/Project">Project</A>  </LI><LI><A href="/Team:Nevada/Project/Background">Introduction</A></LI><LI><A href="/Team:Nevada/Project/Ecoli">E. coli Project</A></LI><LI><A href="/Team:Nevada/Project/Cyano">Cyanobacteria Project</A></LI><LI><A href="/Team:Nevada/Project/Assay">Assay Development</A></LI><LI><A href="/Team:Nevada/Project/Co-Cult">Co-Cultivation</A></LI><LI><A href="/Team:Nevada/Project/Results">Results Summary</A></LI><LI><A href="/Team:Nevada/Reference">References</A></LI><LI>
|  <A href="/Team:Nevada/Parts">Parts</A>  </LI><LI>
|  <A href="/Team:Nevada/Notebook/Weeks14">Notebook</A>  </LI><LI><A href="/Team:Nevada/Notebook/Weeks14">Weeks 1-4</A></LI><LI><A href="/Team:Nevada/Notebook/Weeks58">Weeks 5-8</A></LI><LI><A href="/Team:Nevada/Notebook/Weeks912">Weeks 9-12</A></LI><LI><A href="/Team:Nevada/Notebook/Weeks1316">Weeks 13-16</A></LI><LI>
|  <A href="">Human Practices</A>  </LI><LI><A href="/Team:Nevada/iGEMcollaborators">iGEM Collaborators</A></LI><LI><A href="/Team:Nevada/Outreach">Outreach</A></LI><LI><A href="/Team:Nevada/Contributions">Contributions</A></LI><LI><A href="/Team:Nevada/Fundraising">Fundraising</A></LI><LI>
|  <A href="/Team:Nevada/Safety">Safety</A>  </LI></DIV></DIV></DIV><DIV id="Nevada-Notebook-MainContent"><DIV id="left-minor-content"><FONT color="red"><U><FONT size="4">E. Coli</FONT></U>
Jovanna, Megan, Lauren, Sam, Dafne, Tutku &amp; Destiny
The E.coli group is responsible for making the constructs which will produce fatty acids and ethanol.</FONT><FONT color="blue"><U><FONT size="4">Cyano</FONT></U>
Megan, Dru, Vadim, Marguerite, Laura, David, Jen, Ron &amp; Chris
The cyano group is responsible for increasing glucose production and secretion in cyanobacteria.</FONT><FONT color="green"><U><FONT size="4">Assay</FONT></U>
Casey, Sam, Vadim, Majid, Megan, and Jovanna
The assay group is responsible for testing and verifying the products made by our system.</FONT><FONT color="black"><U><FONT size="4">Media</FONT></U>
Bryson &amp; Matt
The media group is responsible for creating the best possible co-cultivation system for E.coli and cyanobacteria. They are also responsible for creating a co-cultivation apparatus.
<FONT size="4"><B> Description </B>:</FONT><DIV id="descDef" style="text-align:justify;">
BTE-Bay Laurel (Plant) thioesterase gene- produces 12 and 14 carbon medium chain fatty acids that can be converted to biofuels. 

PDC/ADH- pyruvate decarboxylase/aldehyde dehydrogenase operon. This part can be used to produce ethanol. PDC converts pyruvate to acetaldehyde. Alcohol dehydrogenase can then turn acetaldehyde into ethanol. 

ThiE-Thiamine E synthesizing gene in cyanobacteria. The enzyme is usually referred to as thiamine monophosphate pyrophosphorylase. Important for vitamin B synthesis in cyanobacteria. Knockout of this gene will allow for production of an auxotroph in cyanobacteria. 
sigma70 promoter- constitutive promoter in E.coli.

trc promoter- inducible promoter in E.coli. It is not sensitive to changes in glucose concentrations.

GLF- glucose facilitator gene responsible for transporting glucose out of the cyanobacteria.

INV- invertase gene responsible for breaking down sucrose into glucose and fructose. Gene is from cyanobacteria.
</DIV></FONT></DIV></DIV><DIV id="right-major-content" style="overflow:auto;"><A href="https://2011.igem.org/Team:Nevada/Notebook/Weeks14"><FONT size="4">Weeks 1-4</FONT></A>   <A href="https://2011.igem.org/Team:Nevada/Notebook/Weeks58"><FONT size="4">Weeks 5-8</FONT></A>   <A href="https://2011.igem.org/Team:Nevada/Notebook/Weeks912"><FONT size="4">Weeks 9-12</FONT></A>   <A href="https://2011.igem.org/Team:Nevada/Notebook/Weeks912"><FONT size="4">Weeks 13-END</FONT></A>   <DIV class="nbtext" align="left"><P><FONT size="6">Calender Weeks 5-8</FONT></P><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Week_5_-_June_27th-_July_3rd"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Week 5 - June 27th- July 3rd</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Week_6_-_July_4th-10th"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Week 6 - July 4th-10th</SPAN></A></LI><LI class="toclevel-1 tocsection-3"><A href="#Week_7_-_July_11th-17th"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Week 7 - July 11th-17th</SPAN></A></LI><LI class="toclevel-1 tocsection-4"><A href="#Week_8_-_July_18th-24th"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Week 8 - July 18th-24th</SPAN></A></LI></UL></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Week_5_-_June_27th-_July_3rd"><B>Week 5 - June 27th- July 3rd</B></SPAN></H2><P><FONT color="red">E. Coli <U>Trc Promoter</U>
LT: Trc promoter was selected for expression of the ADH/PDC and Thioesterase (BTE) genes.  Trc promoter was located in the commercial pTRC99A plasmid and transformed into NEB10 β cells. Four colonies were selected from Ampicillin resistant plates and cultured in LB-Amp broth. The cultures were purified using QIAprep Spin Miniprep  protocol and the DNA was quantitated using Nanodrop spectrophotometry.  
Trc promoter was isolated from pTRC99A plasmid using PCR. 

Trc promoter was isolated from pTRC99A plasmid using PCR. 
</FONT></P><P><U>Pyruvate Decarboxylase &amp; Alcohol Dehydrogenase</U>
JC:  To insert PDC/ADH genes into the σ70 constitutive promoter plasmid, 0.5ug of σ70/pSB1A3 (2190bp) was digested with SpeI and PstI and 0.5ug of PDC/ADH/pSB1C3 (5124bp) was digested with XbaI and PstI.  These digestions cut out the red fluorescent protein (rfp) tag on pSB1A3 (rfp=885bp) to aid in colony selection.  Digestions were run on a 1% agarose gel, and bands obtained confirmed PDC/ADH (3054bp), pSB1C3 (2070bp), σ70/pSB1A3 (2190), and rfp (885bp).  Incomplete digestion of PDC/ADH/pSB1C3 was apparent for sample #8, so the sample was re-digested before ligation.

PDC/ADH-XbaI/PstI and σ70/pSB1A3-SpeI/PstI were ligated and transformed into NEB10 β cells.  Single colonies from LB-Ampicillin plates were selected and tested on LB-Amp and LB-Chlor plates.  No usable colonies were obtained (all were light pink and grew on both LB-Chl and LB-Amp plates).
</P><P><U>Bay Laurel Thioesterase</U></P><P>MT: Four colonies of Part J sigma 70 contiuative promoters was cultured and digested with SpeI and PstI. 3-2 and 3-3 cell lines of Bay Laurel Thioesterase with RBS and terminator (BTE) were digested with XbaI and PstI.  All cell lines of sigma 70 and BTE were shown to be fully digested by check with 1% agarose gel.  Colony 1 of sigma 70 and 3-2 of BTE were chosen to be ligated together in the J61002 Amp R plasmid that sigma 70 came in. The ligation was transformed in NEB Beta cells and spread on Amp R plates. Twenty-five white colonies were chosen from the transformation and streaked on Amp200 R plates cross-selected on Cm34 R plates. 17/25 of the colonies remained white and none of the colonies grow on the chloramphenicol plates.  Ten colonies were chosen for culture, and minipreped.
</P><P><FONT color="blue">
Cyano 
Procedure: DH5α cells were transformed with the promoters K390015 (petBD+RBS), K390016, and K390017; the integration vector K654056; GLF; and INV. All genes were contained in the plasmid pSB1C3. Transformants were selected for on chloramphenicol plates and cultured in LBccm broth. Plasmid DNA was isolated from the transformants using a QIAgen miniprep kit. Isolated plasmid was digested with EcoRI and PstI to ensure that the correct plasmid transformations took place.
</FONT></P><P>Results/Discussion: The EcoRI and PstI digests did not confirm successful plasmid transformations. This seems to be due to a defect in the restriction enzymes. In order to advance in our research, these isolated plasmids will be submitted for sequencing.<FONT color="green">
Enzymology
CL:The week we conducted the test assay for ethanol detection. Simple one-step assay takes ethanol and NAD+ into acetylaldahyde and NADH with use of alcohol Deh (ADH) enzyme. Final NADH concentration can be determined using its molar extinction coefficient and absorbance at 340.0 nm, and is proportionate to initial ethanol concetration.  Ethanol concentrations ranging from 0.05mM-0.25mM were assayed with 173U/mL ADH.  and absorbances were measured at 340.0 nm.
</FONT></P><P>Results: Final NADH concentrations were calculated to be off by a factor of ten due to unexpectedly low absorbancies. Assay was repeated with increased ethanol concentrations (0.5-10mM), but absorbancies remained lower than expected.
</P><P>Discussion: It was determined that this assay was now sensitive enough for the range of ethanol concentrations that will need to be detected in our project, therefore opt to use a more sensitive Alcohol Oxidase enzyme instead.  The alternative enzyme was ordered and will be stored until testing.  A rough draft protocol for the alcohol oxidase assay was also written up and buffers were prepped. 
Media
Growth of BL-21 cells was tested in hopes of finding a cell line that could grow in BG-11 with minimal supplements.  BL-21 cells showed only very slight increases in growth relative to 10-ß.
</P><P>Next, cell growth in BG-11 was tested with varying amounts of NH<SUB>4</SUB>Cl, in hopes that we hadn't seen desirable levels of growth due to ammonium chloride concentrations that either weren't sufficient or were toxic.  Cultures were grown in triplicate, with 10 mL cultures in 16x25 mm culture tubes, in an effort to analyze more samples quickly.  Results showed that varying the ammonium chloride levels anywhere between 0.5-3.0 mg/mL (9.35-56.1 mM) had little noticeable effect.</P><H2><SPAN class="mw-headline" id="Week_6_-_July_4th-10th"><B>Week 6 - July 4th-10th</B></SPAN></H2><P><FONT color="red">E. Coli <U>Trc Promoter</U>
LT:PCR products were cloned into PCR2.1 TOPO vector and transformed into One Shot Top 10 Chemically competent cells using the TOPO TA cloning protocol.  Twenty-five colonies were selected from Ampicillin resistant plates streaked on new LB –Amp plates and cultured in LB-Amp broth. QIAprep Spin Miniprep protocol was performed on the cultures and DNA was quantitated using nanodrop spectrophotometry. Results: Nanodrop analysis indicated high levels of RNA contamination. P1 buffer didn’t work properly.
Ten new colonies (#1-10) were streaked and inoculated in LB-Amp broth, purified using the Qiaprep Spin Miniprep protocol, and nanodropped.  To confirm that TRC was correctly inserted into the PCR2.1 TOPO vector, an Eco RI and Pst I digest was set up and run on a 1.0% gel. Results: Trc was not successfully inserted into the PCR 2.1 TOPO vector. Bands in the gel correspond to the TOPO vector without insert. </FONT></P><P><U>Pyruvate Decarboxylase &amp; Alcohol Dehydrogenase</U>
JC:  PDC/ADH was re-digested with XbaI/PstI and run on a 1% agarose gel to confirm digestion.  Bands obtained confirmed PDC/ADH (3054bp), and pSB1C3 (2070bp).  PDC/ADH-XbaI/PstI and σ70/pSB1A3-SpeI/PstI were re-ligated and transformed into NEB10 β cells.
</P><P><U>Bay Laurel Thioesterase</U></P><P>MT: The ten cultures of sigma 70 promoter ligated with BTE into the J61002 Amp R plasmid were digested with EcoRI and PstI. 8/10 of the samples had a band at 2000bp (J61002) and 1200bp (BTE) on a 1% agarose gel. They were compared to BTE/ pSB1C3 digest with EcoRI and PstI, and you can see a slight shift in the size of the BTE suggesting that successful ligation of the sigma 70 with BTE. Six of the samples were sent to the Nevada Genomic Center for sequencing in the forward direction. All six had the entire sigma 70 promoter and the beginning of BTE.
</P><P><FONT color="blue">
Cyano 
Procedure: Submitted isolated pSB1C3 containing K390015, K390016, K390017, K654056, INV, and GLF for sequencing. Prepared 2L of BG11 media for future Synechocystis growth experiments
</FONT></P><P>Results/Discussion: The sequencing data indicated the successful isolation of all genes. Thus, for creating the invertase operon via gibson assembly, K390015 will be used to PCR out the petBD+RBS (light inducible promoter) and INV will be used to PCR out invertase containing the correct overlapping flanking regions. K39005 will also be used to PCR promoter for the GLF operon. K654056 can be used as an integration vector for transforming Synechocystis PCC 6803.
</P><P><FONT color="green">
Enzymology
CL: This week we ordered a Free Fatty Acid Assay Kit from BioSystems as we anticipate that our E. coli modified to produce and secrete free fatty acids part is complete.  While I waited for my kit to arrive I participated in a Western Blot used to confirm a His tag on our BTE (Bay Laurel Esterase) gene.   
</FONT></P><P>Media</P>
Autoclaved and cleaned flasks for future experiments.<P>
Prepared media to perform last experiment (varying levels of ammonium chloride) again, with 100 mL cultures.  It was thought that using 16x25 mm culture tubes may not provide sufficient aeration, so the decision was made to go back to flasks.    
</P><H2><SPAN class="mw-headline" id="Week_7_-_July_11th-17th"><B>Week 7 - July 11th-17th</B></SPAN></H2><P><FONT color="red">E. Coli <U>Trc Promoter</U>
LT:Fifteen new colonies (#11-25) were recultured, purified using the QIAprep Spin Miniprep protocol, digested with Eco RI and Pst I and run on a 1% gel. (See week 6.) Results:  Trc was not successfully inserted into the PCR 2.1 TOPO vector. Bands in the gel correspond to the TOPO vector without insert. 
LB- Amp plates were prepared with x-gal. Cultures #1-25 were streaked and incubated overnight for blue/white selection.  Results: All colonies were white.  
All colonies were white so control cells were spread on the x-gal plates. Results:  The control cells appeared very light blue. The x-gal blue/white selection wasn’t helpful for selecting transformed versus non-transformed colonies.  </FONT></P><P><U>Pyruvate Decarboxylase &amp; Alcohol Dehydrogenase</U>
JC:  Single colonies from LB-Amp plates were selected and tested on LB-Amp and LB-Chlor plates.  Liquid cultures grown in LB-Amp and minipreps and nanodrop analysis performed.

To verify presence of PDC/ADH genes with the σ70 constitutive promoter, 0.5ug of PDC/ADH/σ70/pSB1A3 was digested with EcoRI and PstI.  Digests were run on a 1% agarose gel, and bands confirmed σ70/PDC/ADH (3089bp) and pSB1A3 (2155bp).
<FONT color="blue">
Cyano 
Procedure: Synechocystis PCC 6803 genomic DNA was isolated using a lysis, denaturation, and ethanol precipitation. This genomic DNA was used to PCR out agp and ThiE for the creation of knockouts of the respective genes using gibson assembly. PCR was performed on CmR, petBD, GLF, and INV from the isolated pSB1C3 plasmids. PCR was also performed on KnR from pUC4K. These PCR reactions were performed using the gibson assembly primers and master mix was prepared with phusion DNA polymerase. DNA was present in 1 ng/ul reaction. All PCR reactions were run on .7% agarose gel at 110 V for 60 minutes.</FONT></P><P>Results/Discussion: Bands were expected at 4231 bp for agp, 1061 bp for KnR, 219 bp for petBD, 1711 bp for INV, 1548 bp for glf, 894 bp for CmR, and 1031 bp for ThiE. None of these bands appeared on the gel. Thus, PCR failed for all genes and the experiments need to be redone. Since the primers are quite large for gibson assembly parts, the optimum initial annealing temperature for the first three PCR cycles needs to be determined for all genes.
</P><P>VG:One liter of LB liquid media and stock solutions of ampicillin were made. Ampicillin was added to the LB liquid media and ampicillin plates were poured for future use. Invertase (INV) and glucose facilitator gene (GLF) contained in the pUC57 vector colonies were streaked on ampicillin and chloromphenicol plates to check for growth.
<FONT color="green">
Enzymology
CL:Still waiting for the Free Fatty Acid Assay kit to arrive.  Continued to help and clean up around the lab.   </FONT>
Media
We ordered a new cell line, NEB Express I<SUP>q</SUP>, for testing against 10-ß.  NEB tech support recommended the I<SUP>q</SUP> strain as an alternative to 10-ß, claiming that it had no auxotrophies and should grow without amino acid supplements.  A test was done in which I<SUP>q</SUP> and 10-ß were grown in BG-11 without ammonium chloride and with 1.0 mg/mL (18.7 mM) ammonium chloride.  In both cases the media contained 50 mM glucose.  This test was done with 100 mL cultures in 250 mL Erlenmeyer flasks.  The results indicated that I<SUP>q</SUP> cells could not grow with ammonium chloride as the only nitrogen source. </P><H2><SPAN class="mw-headline" id="Week_8_-_July_18th-24th"><B>Week 8 - July 18th-24th</B></SPAN></H2><P><FONT color="red">E. Coli <U>Trc Promoter</U>
LT:Half of the PCR products were loaded onto a 1.2% gel to verify that the Trc promoter was successfully isolated. Results: TRC was successfully isolated by PCR.

Remaining PCR products were combined and run on a 1.2% gel. The band correspond to the Trc promoter was cut out and purified using the QIAquick gel extraction of PCR products protocol.  Nanodrop analysis was then used to quantitate the amount of DNA present. 

Trc promoter was cloned into the TOPO2.1 PCR vector, transformed into One Shot Top 10 Chemically Competent cells, and streaked onto LB-Amp plates with x-gal.  Control cells were also plated on LB-Amp plates with x-gal. Results:  The transformation seems to have been successful. There were both dark blue and white colonies.  The control cells contained only blue colonies!

Fifteen white colonies (#1-15) were selected. Half the colony was single colony streaked onto LB-Amp plates for purification and the other half was inoculated into LB-Amp broth. The cultures were purified using QIAquick miniprep and analyzed using the nanodrop. An Eco RI and Pst I restriction digest was set up  and run on a 1.2% gel to confirm that the transformation was successful.  Results: The restriction digest did not confirm a successful digest. However, 4 colonies were selected from colonies #1-15 to continue. 
</FONT></P><P><FONT color="blue">
Cyano </FONT></P><P><U>ThiE/GLF:</U></P><P>7/17/11 – PCR temp gradient was created for ThiE solution, used New England Biolabs phusion high-fidelity DNA polymerase protocol. Temp gradient ranges from 72-50 degrees Celsius. Controls of only DNA and only Primer were run with the samples. 10 ul samples were run for each temp gradient7/18/11 - PCR results were run through a 1% agarose gel. Best results came from ‘F’ which ran at 54.3 Celsius. The DNA from this band was extracted and purified using QIAGEN Gel Extraction Kit7/19/11 – PCR 50 ul of ThiE at 54.3 Celsius for backup stock. TOPO cloned ThiE that was extracted from gel yesterday. Plated TOPO cells at 100 ul, 75 ul, 50 ul and 25 ul  concentrations then placed in incubator overnightPCR reactions only need one control, of just the primers without DNA.7/20/11 – 5 colonies of TOPO bacteria grew overnight. Cultured all 5 for purification and confirmation of proper insert (ThiE) Ran gel to confirm that 50 ul ThiE PCR worked, nice clean band at 1 kb marker, saved the rest of the 50 ul stock in -20 C freezer for further use. PCR done on  the additional pieces (CmR, PetBD and GLF) used same temp gradient as done with ThiE. Will run a gel of these tomorrow to see what optimal temp is for each sample. If clean bands are seen, we will perform a gel purification and re-PCR those samples. Spoke with Shintani, these additional pieces should just be amplified through PCR until a clean stock is created, then this stock will be used in Gibson.7/21/11 – Miniprep purified TOPO cultures. Sample “1” did not bloom in culture but grew on stock plate. Purified “2-5” and will save “1” as a backup if no other  colonies have the correct insert.  Ran temp gradient samples through gels.GLF had very light bands at 67.6 and 63.5 Celsius temps, the average temp of those two is 65.5 – will try to run more DNA at this temp to see if we can get stronger bands. Maybe increase DNA concentration in solution.*Shintani gave me his master stock solution of GLF (in puc vector) for next run of PCR. Diluted 1:9 of his solution and stored in freezer in PCR tube (with orange label) will try to PCR this stock at 65.5.PetBD worked really well at several temps but 63.5 Celsius appears to be the best temp, strong band that was very clean. This worked out so well I might try just running a 50 ul solution at this temp and saving it as backup stock. Confirm the same clean band with a 5 ul run of solution through gel.CmR did not work at all  I was told K56 was the integration vector sent from Utah State but was unable to confirm that in any of my team’s notebooks. Will double check with David today and see if he has that same number in his note book. If it is correct, maybe need to play with DNA concentration and temps again.*E. coli group has extra CmR, will ask Megan to bring over some of this stock (pSB1C3 – iGEM vector) for next PCR. Shintani thinks that maybe vials were switched in the beginning (KmR PCR also not working) or something went wrong during purification.With better stock will run temp gradient PCR on CmR again.7/22/11:  Gel for GLF at 65.5C yielded no bands.  Ran PCR of GLF at 63.5C.  Faint bands, attempted gel purification, will run confirmation gel on Monday.  Ran PCR for PETBD (50ul), which also yielded no bands.  Will run another gel for a shorter time period on Monday.  Will discuss options/next steps on Monday with the group.
</P><P><U> AGP/INV:</U></P><P>Procedure: PCR was reperformed on KnR, petBD, INV, and agp with a temperature gradient in order to determine the optimum initial annealing temperature for the different parts of the agp knockout-invertase operon. PCR reactions contained phusion DNA polymerase and DNA concentrations of 1 ng/ul reaction. The initial primer annealing temperatures used were 72.0, 70.3, 67.6, 63.5, 58.2, 54.3, 51.6, and 50.0 degrees Celsius. KnR, INV, and agp PCR products were run on .7% agarose gel at 110 V for 1 hour, and petBD PCR product was run on 1.5% agarose gel at 110 V for 1 hour.
</P><P>Results: No bands were observed at the expected sizes (1360 bp for agp, 1341 bp for KnR, 209 for petBD, and 1768 bp for INV). Thus, the gibson assembly primers are to be reconstructed and reordered for future PCR reactions. 
</P><P>VG:GLF (1605bp) and INV (1728bp) in the pUC57 (2710bp) ampicillin resistant vector were minipreped and 0.5mg of each sample of GLF and INV in the pUC57 ampicillin resistant vector were digested with EcoRI and PstI restriction enzymes and checked on a 1% agarose gel. GLF was properly digested showing bands at about 2000 bp and 1600 bp. Also, the GLF samples grew on chloromphenicol plates. Thus, two GLF samples were sent for sequencing at the Nevada Genomics Center to check if GLF was present in the pSB1C3 vector. The INV digest was not successful, producing a single band for all samples.
Samples of GLF and INV in the pUC57 ampicillin resistant vector were obtained from Dr. Shintani and transformed in DH5α E.coli cells on AMP plates. The transformation was successful. As a result, 10 cultures were made for GLF and 10 cultures for INV in 2ml of LB-AMP liquid broth. The sequencing results for GLF from were checked. The sequencing was not complete. Thus, new primers were designed for further sequencing. 
The GLF and INV in the pUC57 cultures were minipreped and 0.5mg of all 10 GLF and INV minipreps were digested with EcoRI and PstI and checked along with 0.25mg of uncut GLF and INV on a 1% agarose gel. All digested samples for GLF showed two bands at 2700 bp and 1600bp. Samples 3 and 10 showed a third band at about 5000bp. All digested samples for INV showed two bands at 2700 bp and 1700bp. Samples 2 and 3 showed an extra third band at about 5000bp and sample 4 showed four bands, with the fourth band being above the 5000 bp band. Overall, eight GLF and seven INV digested samples appeared correct. 
</P><P><FONT color="green">
Enzymology
CL: This week we received the FFA assay kit and tested two BTE/PDC transformed cultures (+ 4 negative sigma70 controls); pellets and supernatants both were tested.  Results were positive for two of the cultures (BTE9 and BTE10) and results suggested that fatty acids were present in the supernatant as opposed to the cell itself.  We decided to do a time course growth on one of the identified cultures (BTE9)in order to further analyze production of fatty acids through out the culture growth cycle.   </FONT>
Media
We received a new incubator to allow for greater numbers of 250 mL flasks.  This will also let us keep one incubator at 30° C.
</P><P>Because we have been getting poor results for growth of all of our <I>E. coli</I> strains in BG-11 media, we ordered fresh BG-11 stock from Sigma Aldrich and made a new stock solution of 1.0 M glucose.  10-ß plates were restreaked on new LB plates.  We also made a 10% w/v tryptone stock for overcoming auxotrophies in the <I>E. coli</I> strains.  
</P><P>A growth curve was performed to determine if 1% w/v tryptone was enough to induce sufficient growth in either 10-ß or I<SUP>q</SUP> cells.  Results showed that the addition of 1% tryptone to BG-11 allowed for growth nearly comparable to that of LB, in the case of both strains.
</P></DIV></DIV></DIV><DIV id="bottom-content"><DIV id="sponser-banner-content"><DIV class="Sponser-text">SPONSORS</DIV></DIV><DIV id="sitemap-banner-content"><DIV id="subNav"><DIV class="subNavText1">
Navigation
</DIV><DIV class="subNavText2"><A href="/Team:Nevada">
Home
</A>     <A href="/Team:Nevada/Team">
Team
</A>     <A href="/Team:Nevada/Project">
Project
</A>     <A href="/Team:Nevada/Parts">
Parts
</A>     <A href="/Team:Nevada/Notebook/Weeks14">
Notebook
</A>     <A href="/Team:Nevada/Outreach">
Outreach
</A>     <A href="/Team:Nevada/Contributions">
Contribution
</A>     <A href="/Team:Nevada/Fundraising">
Fundraising
</A>     <A href="/Team:Nevada/Reference">
References
</A>     <A href="/Team:Nevada/Safety">
Safety
</A></DIV></DIV></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:Nevada/Notebook/Weeks58">http://2011.igem.org/Team:Nevada/Notebook/Weeks58</A>&quot;</DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Nevada/Notebook/Weeks58" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Nevada/Notebook/Weeks58" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Nevada/Notebook/Weeks58&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Nevada/Notebook/Weeks58&amp;oldid=200973" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>