<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Nevada_Project"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Nevada/Project">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Nevada/Project&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Nevada/Project&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Nevada/Project&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Nevada/Project" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Nevada/Project</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><DIV id="absHeader"><DIV id="banner-box"><DIV id="banner-title-box"><DIV class="banner-text-title">
So Happy Together</DIV><DIV class="banner-subtext">
A Cooperative Relationship between Cyanobacteria and E. Coli
for production of biofuels
</DIV></DIV><DIV id="banner-buttons" class="buttons-text"><UL class="top-level"><LI>  <A href="/Team:Nevada">Home</A>  </LI><LI>
|  <A href="/Team:Nevada/Team">Team</A>  </LI><LI><A href="/Team:Nevada/Team">Nevada's Team</A></LI><LI><A href="/Team:Nevada/gallery">Gallery</A></LI></UL><LI>
|  <A href="/Team:Nevada/Project">Project</A>  </LI><LI><A href="/Team:Nevada/Project/Background">Background</A></LI><LI><A href="/Team:Nevada/Project/Ecoli">E. coli Project</A></LI><LI><A href="/Team:Nevada/Project/Cyano">Cyanobacteria Project</A></LI><LI><A href="/Team:Nevada/Project/Assay">Assay Development</A></LI><LI><A href="/Team:Nevada/Project/Co-Cult">Co-Cultivation</A></LI><LI><A href="/Team:Nevada/Project/Results">Results Summary</A></LI><LI><A href="/Team:Nevada/Reference">References</A></LI><LI>
|  <A href="/Team:Nevada/Parts">Parts</A>  </LI><LI>
|  <A href="/Team:Nevada/Notebook/Weeks14">Notebook</A>  </LI><LI><A href="/Team:Nevada/Notebook/Weeks14">Weeks 1-4</A></LI><LI><A href="/Team:Nevada/Notebook/Weeks58">Weeks 5-8</A></LI><LI><A href="/Team:Nevada/Notebook/Weeks912">Weeks 9-12</A></LI><LI><A href="/Team:Nevada/Notebook/Weeks1316">Weeks 13-16</A></LI><LI>
|  <A href="">Human Practices</A>  </LI><LI><A href="/Team:Nevada/iGEMcollaborators">iGEM Collaborators</A></LI><LI><A href="/Team:Nevada/Outreach">Outreach</A></LI><LI><A href="/Team:Nevada/Contributions">Contributions</A></LI><LI><A href="/Team:Nevada/Fundraising">Fundraising</A></LI><LI>
|  <A href="/Team:Nevada/Safety">Safety</A>  </LI></DIV></DIV></DIV><DIV id="Nevada-MainContent"><DIV id="top-team-content" style="text-align:justify; font-size:16px;">

A Cooperative Relationship between Cyanobacteria and E.Coli
for production of Biofuels
<TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Introduction"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Introduction</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#E._coli_Project"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">E. coli Project</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Production_of_Medium_Chain_Fatty_Acids_as_a_Biofuel"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Production of Medium Chain Fatty Acids as a Biofuel</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Production_of_Ethanol_as_a_Biofuel"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">Production of Ethanol as a Biofuel</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Construction_of_a_Glucose_Insensitive_Inducible_Promoter_in_E._coli"><SPAN class="tocnumber">2.3</SPAN><SPAN class="toctext">Construction of a Glucose Insensitive Inducible Promoter in E. coli</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-6"><A href="#Cyanobacteria"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Cyanobacteria</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#Introduction_2"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">Introduction</SPAN></A></LI><LI class="toclevel-1 tocsection-8"><A href="#Goals_and_Specific_Tasks_for_Cyanobacteria"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Goals and Specific Tasks for Cyanobacteria</SPAN></A></LI><LI class="toclevel-2 tocsection-9"><A href="#Engineer_Synechocystis_to_overproduce_hexose_sugars"><SPAN class="tocnumber">4.1</SPAN><SPAN class="toctext">Engineer Synechocystis to overproduce hexose sugars</SPAN></A></LI><LI class="toclevel-3 tocsection-10"><A href="#AGP_K.O..2FINV_Construct_Design"><SPAN class="tocnumber">4.1.1</SPAN><SPAN class="toctext">AGP K.O./INV Construct Design</SPAN></A></LI><LI class="toclevel-3 tocsection-11"><A href="#AGP_K.O..2FINV_Construct_Components"><SPAN class="tocnumber">4.1.2</SPAN><SPAN class="toctext">AGP K.O./INV Construct Components</SPAN></A></LI><LI class="toclevel-2 tocsection-12"><A href="#Gibson_Assembly_of_AGP_K.O..2FINV_Components"><SPAN class="tocnumber">4.2</SPAN><SPAN class="toctext">Gibson Assembly of AGP K.O./INV Components</SPAN></A></LI><LI class="toclevel-2 tocsection-13"><A href="#Engineer_Synechocystis_to_secrete_hexose_sugars_to_the_medium"><SPAN class="tocnumber">4.3</SPAN><SPAN class="toctext">Engineer Synechocystis to secrete hexose sugars to the medium</SPAN></A></LI><LI class="toclevel-3 tocsection-14"><A href="#ThiE_K.O..2FGLF_Construct_Design"><SPAN class="tocnumber">4.3.1</SPAN><SPAN class="toctext">ThiE K.O./GLF Construct Design</SPAN></A></LI><LI class="toclevel-3 tocsection-15"><A href="#ThiE_K.O..2FGLF_Construct_Components"><SPAN class="tocnumber">4.3.2</SPAN><SPAN class="toctext">ThiE K.O./GLF Construct Components</SPAN></A></LI><LI class="toclevel-2 tocsection-16"><A href="#Gibson_Assembly_of_ThiE_K.O..2FGLF_Components"><SPAN class="tocnumber">4.4</SPAN><SPAN class="toctext">Gibson Assembly of ThiE K.O./GLF Components</SPAN></A></LI><LI class="toclevel-1 tocsection-17"><A href="#Results"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Results</SPAN></A></LI><LI class="toclevel-1 tocsection-18"><A href="#Future_Directions"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">Future Directions</SPAN></A></LI><LI class="toclevel-1 tocsection-19"><A href="#Assay_Development"><SPAN class="tocnumber">7</SPAN><SPAN class="toctext">Assay Development</SPAN></A></LI><LI class="toclevel-2 tocsection-20"><A href="#Ethanol_Production"><SPAN class="tocnumber">7.1</SPAN><SPAN class="toctext">Ethanol Production</SPAN></A></LI><LI class="toclevel-1 tocsection-21"><A href="#Co-cultivation"><SPAN class="tocnumber">8</SPAN><SPAN class="toctext">Co-cultivation</SPAN></A></LI><LI class="toclevel-1 tocsection-22"><A href="#Choice_of_Growth_Medium"><SPAN class="tocnumber">9</SPAN><SPAN class="toctext">Choice of Growth Medium</SPAN></A></LI><LI class="toclevel-1 tocsection-23"><A href="#Results_of_Growth_Curves"><SPAN class="tocnumber">10</SPAN><SPAN class="toctext">Results of Growth Curves</SPAN></A></LI><LI class="toclevel-2 tocsection-24"><A href="#Growth_in_BG-11_.2B_NH4Cl"><SPAN class="tocnumber">10.1</SPAN><SPAN class="toctext">Growth in BG-11 + NH<SUB>4</SUB>Cl</SPAN></A></LI><LI class="toclevel-2 tocsection-25"><A href="#Growth_in_BG-11_.2B_Amino_Acids"><SPAN class="tocnumber">10.2</SPAN><SPAN class="toctext">Growth in BG-11 + Amino Acids</SPAN></A></LI><LI class="toclevel-2 tocsection-26"><A href="#Detection_of_Glucose"><SPAN class="tocnumber">10.3</SPAN><SPAN class="toctext">Detection of Glucose</SPAN></A></LI><LI class="toclevel-1 tocsection-27"><A href="#Apparatus_Components"><SPAN class="tocnumber">11</SPAN><SPAN class="toctext">Apparatus Components</SPAN></A></LI></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Introduction"><B>Introduction</B></SPAN></H2><P>In light of the growing energy crisis, much research has been devoted to finding economical means of producing renewable fuels. Traditional methods for obtaining biofuels have relied mainly on the fermentation of agricultural crops. However, there are a number of problems with this approach: the reduction in land available for food production, relatively low levels of CO2 biofixation, and large biomass requirements. Our project aims to overcome these problems by utilizing E. coli for the production of biodiesel (C-12 fatty acids) and bioethanol. In the past there have been a number of examples of biofuel production in E. coli; however 30-40% of production cost is based on media costs (Galbe et al., 2007). Our project will surmount these high production costs by engineering the cyanobacteria, Synechocystis PCC 6803, to secrete large quantities of glucose that will feed our biofuel-producing E. coli. Cyanobacteria and E. coli will be co-cultivated in an apparatus that allows for the mutual transfer of carbon to produce biofuels. Not only will this project provide an efficient means for producing biofuels without the need for a carbon source, but it will also create a novel cooperative system between bacterial species that may have further industrial implications.
</P><H2><SPAN class="mw-headline" id="E._coli_Project"><I><I>E. coli</I> Project</I></SPAN></H2><P>The goal of the <I>E. coli</I> team was to transform <I>E. coli</I> with genes that produce medium chain fatty acids and ethanol, both of which can be used as biofuels. In addition, the group wanted to develop an <I>E. coli</I> glucose insensitive inducible promoter that would not be down-regulated in the presence of high glucose concentrations. In our system glucose levels produced by Cyanobacteria may be unpredictable; therefore having an <I>E. coli</I> inducible promoter that is glucose-insensitive will provide more control over expression of the biofuel genes.  
</P><P><U>Pathway for Producing Biofuels and Ethanol in <I>E. coli</I></U></P><H3><SPAN class="mw-headline" id="Production_of_Medium_Chain_Fatty_Acids_as_a_Biofuel"><B>Production of Medium Chain Fatty Acids as a Biofuel</B></SPAN></H3><P><U>Bay Laurel Thioesterase Cloning Strategy</U>
Bay Laurel thioesterase (<I>BTE</I>) is a gene that is naturally found in the bay leaves (<I>Umbellularia californica</I>)and used as a spice in cooking. The BTE produces medium chain 12-carbon and 14-carbon fatty acids (Voelker et al.,1996). The Nevada team changed the codon sequence for BTE based on the publication by Welch et. al. “Design Parameters to Control Synthetic Gene Expression in <I>Escherichia coli</I>” by changing the codons that are more common in plants to codons common in <I>E. coli</I>. The newly synthesized intermediate part (BBa_K558003) also contained an <I>E. coli</I> ribosome binding sequence, a His tag sequence on the 3’ end of the coding sequence and an iGEM registry double terminator sequence (BBa_B0014).
</P><P>The constitutive promoter sigma 70 from the Registry of Standard Biological Parts (BBa_J23101) was classically cloned in front of the synthesized Bay Laurel thioesterase gene to create the Nevada BTE generator part (BBa_K558007).
Many tests were used to determine the expression of the Bay Laurel thioesterase including SDS-PAGE, Western blot, colorimetric assay and gas chromatography.  The one assay that was most successful for measuring free fatty acid content in the growth media was purchased through Bioassay Systems (EnzyChrom™ Free Fatty Acid Assay Kit www.bioassaysys.com). T7 Express Iq <I>E. coli</I> with the <I>BTE</I> generator produced up to 125 uM free fatty acid compared to control line which produced 50 uM fatty acid.
</P><P><U>Free Fatty Acid Activity</U>
Fatty Acid production was tested using EnzyChrom Free Fatty Acid Assay Kit (BioAssay Systems). The graphs below show the concentration of free fatty acid produced by Bay Laurel Thioesterase transformed in NEB 10 β and NEB Iq cell strains. 
</P><P>The NEB Iq E. coli cell line transformed with the sigma 70-BTE generator produced 125uM free fatty acid compared to the negative control sigma 70-red fluorescent protein (50uM) and the media only control (40uM). Bay Laurel Thioesterase transformed into NEB 10 β cells showed a positive increase of 115uM of fatty acids compared to negative controls of 40uM and 70uM. Based on this data we can produce up to 2.5 fold more free fatty acids with cell lines producing the Bay Laurel Thioesterase then without.
</P><P>
Preliminary tests for fatty acid length showed C-12 in the Bay Laurel Thioesterase transformed cells, but more chromatography test need to be done in order to confirm the production of C-12 and C-14 medium chained fatty acids. </P><H3><SPAN class="mw-headline" id="Production_of_Ethanol_as_a_Biofuel"><B>Production of Ethanol as a Biofuel</B></SPAN></H3><P><U>Pyruvate Decarboxylase and Alcohol Dehydrogenase Cloning Strategy</U>
In order to produce ethanol efficiently in <I>E. coli</I>, is was necessary to transform <I>E. coli</I> with both the pyruvate decarboxylase gene to produce an acetaldehyde intermediate, and an alcohol dehydrogenase gene that will take acetaldehyde and convert it to ethanol. The Nevada team designed and synthesized an operon without a promoter (intermediate part k558000) containing pyruvate decarboxylase and alcohol dehydrogenase coding regions along with a standard E. coli ribosome binding site and double terminator (BBa_B0014). The pyruvate decarboxylase (pdc) and alcohol dehydrogenase (adh) coding region sequences were based on the Zymomonas mobilis sequences (Ingram et. al. 1987) and work conducted by UNIPV-Pavia (iGEM 2009 Part K173016; K173017) and Utah State (iGEM 2009 M11041; M11042). The goal was to express this pdc/adh operon under the control of the constitutive promoter sigma70 (iGEM part J23101), or a glucose insensitive inducible promoter, trc (see details below) and test for expression of functional genes.
</P><P>The sigma 70-pdc/adh generator (k558001) was assembled using classic cloning techniques. It was then transformed into two E. coli competent cell lines, NEB 10-Beta (New England BioLabs) and T7 Express Iq (New England BioLabs). Colonies were screened through restriction digest analysis, DNA sequencing and tested for enzymatic activity.</P><P><U>Pyruvate Decarboxylase Activity</U></P><P>Pyruvate decarboxylase activity was assayed using aldehyde indicator plates (Ingram et. al. 1987) to test for a Schiff Base reaction to detect the presence of acetaldehyde, a substrate for ethanol production. The figure below shows pink colonies after an overnight incubation on Schiff base plates. The streaked colony on the top of the plate is a negative control cell line (white-light pink). The streaked colony on the right are a typical example (pink color) of sigma 70-pdc/adh generator in the NEB 10-Beta cells. The cells on the left (dark pink) and represent what is typically seen for the sigma 70-pdc/adh generator expressed in T7 Express Iq cell lines. From the results shown here and other experiments, we  concluded that the NEB T7 Express Iq cells produced higher levels of the acetaldehyde intermediate. </P><P><U>Alcohol Dehydrogenase Activity</U></P><P>Ethanol production was tested using EnzyChrom Ethanol Assay Kit (BioAssay Systems). The figure below shows percent ethanol production in T7 Express Iq cells (NEB) in 2% versus 10% glucose medium over 24 and 48 hours. The sample grown in 2% Glucose over 24 hours yielded 0.02% ethanol, and the sample grown in 10% Glucose over 48 hours yielded 0.018% ethanol. From the results shown here, and other experiments, we have concluded that our modified <I>E. coli</I> is in fact producing ethanol.  Furthermore, we have also concluded that increasing amounts of glucose does not lead to higher ethanol production. It is possible that these lines are only producing low levels of alcohol dehydrogenase and we are seeing a buildup of acetaldehyde in these same lines. It may be necessary to re-evaluate the original operon and re-engineer it to improve the adh gene thereby increasing production of the alcohol dehydrogenase enzyme.</P><H3><SPAN class="mw-headline" id="Construction_of_a_Glucose_Insensitive_Inducible_Promoter_in_E._coli"><B>Construction of a Glucose Insensitive Inducible Promoter in <I>E. coli</I></B></SPAN></H3>
Although the constitutively expressed Escherichia coli sigma 70 promoter (J23101) has proven to be adequate for expression of our fatty acid and alcohol genes, the hope was to construct a new promoter for the iGEM registry that was inducible, but not sensitive to glucose or other sugars. The inducible promoter, trc, was a promising option.  This promoter originated as a fusion between the trp (tryptophan inducible/glucose repressible) promoter and the lac (lactose inducible/glucose repressible) promoter (De Boer et. al. 1983). The trc fusion promoter was engineered in such a way that gene expression can be up-regulated in the presence of IPTG, but expression is unaffected by fluctuations in glucose concentration. <P>
In spite of the encouraging research, trc has proven to be a complicated promoter to work with in the lab. With only 62 base pairs it is hard obtain definitive visual results and manipulate such small fragments between plasmids, cells, etc.  There has been little success with this small fragment. Many techniques have aimed at transforming trc into pSB1C3 iGEM plasmid for submission to the registry for use by future teams and placing the promoter in front of both genes in various plasmids including pUC vectors, TOPO vectors, as well as iGEM vectors. Different techniques, enzymes, and protocols were used without success. One of the biggest challenges when using trc is that agarose gel electrophoresis is of little use for quick verification. The 100bp (promoter plus iGEM prefix and suffix) shift is nearly impossible to visualize on the gel and sequencing is often needed to confirm results which takes a few days to obtain results.  Despite the frustration that we have had using the trc promoter, we think it would be a valuable part for the registry. 
</P><H2><SPAN class="mw-headline" id="Cyanobacteria"><B>Cyanobacteria</B></SPAN></H2><H3><SPAN class="mw-headline" id="Introduction_2"><B>Introduction</B></SPAN></H3><P>Cyanobacteria are gram negative photosynthetic bacteria that have the unique ability to also fix atmospheric nitrogen gas into a biologically useful form. As such they are one of the few truly autotrophic organisms in regards to carbon and nitrogen nutrition. In this project we are engineering the cyanobacterium, <I>Synechocystis PCC 6803</I>, to provide glucose as a carbon source for a biofuel producing <I>E. coli</I> strain. The rationale behind this effort is that if successful we will be able to decrease the cost of growth medium and develop a self-sustaining feeder system that can be adapted to existing microbial fermentation systems that have been optimized for the production of biofuels and other high value compounds. <I>Synechocystis PCC6803</I> was chosen for this project because it is widely used as a model cyanobacterial system. As such it, it has numerous resources available to the investigator including facile and established transformation protocols and a completely sequenced genome. The work being done on this project will set the foundation for basic research into microbial community interactions and also applied research with real world industrial relevance. 
</P><H2><SPAN class="mw-headline" id="Goals_and_Specific_Tasks_for_Cyanobacteria"><B>Goals and Specific Tasks for Cyanobacteria</B></SPAN></H2><P>The goal of this project is to engineer the cyanobacterium strain Synechocystis PCC6803 to produce and secrete hexose sugars that can be utilized by a biofuel producing E. coli strain. To achieve this goal the Cyanobacteria team has been assigned following two tasks: </P><P>1)	Engineer Synechocystis to overproduce the hexose 
2)	Engineer Synechocystis to secrete hexose sugar to the medium
</P><H3><SPAN class="mw-headline" id="Engineer_Synechocystis_to_overproduce_hexose_sugars"><B>Engineer Synechocystis to overproduce hexose sugars</B></SPAN></H3><P>
Cyanobacteria use photosynthesis to provide energy and carbon skeletons for anabolic processes. During the day excess fixed carbon can be converted to the polysaccharide, glycogen and stored for later use.  To engineer Synechocystis to overproduce hexose sugars, we will divert carbon away from the glycogen biosynthetic pathway and towards hexose sugar production. To achieve this goal we will create a null mutation in the gene encoding ADP glucose pyrophosphorylase (AGP). AGP specifically converts glucose to ADP-glucose which is the monomeric precursor to glycogen. Synechocystis AGP knockout mutants have been reported to no longer produce glycogen, but instead accumulate high levels of sucrose (Miao et al (2003) FEMS Letters 218: 71-77). Fortunately, sucrose can easily be converted to the hexose sugars, glucose and fructose, in a reaction catalyzed by the enzyme invertase (INV). Therefore, we will introduce and overexpress the INV gene in the AGP mutant background.</P><H4><SPAN class="mw-headline" id="AGP_K.O..2FINV_Construct_Design"><B>AGP K.O./INV Construct Design</B></SPAN></H4><P>
We will simultaneously create the AGP mutant and the INV overexpressing line by inserting the INV expression construct into the coding region of the Synechocystis AGP gene. This will be accomplished by creating an AGP gene replacement construct in which the INV overexpression gene cassette and a kanamycin resistance gene cassette are inserted into the coding region of a subcloned AGP gene. This dysfunctional AGP construct will then be transformed into Synechocystis and through homologous recombination, replace the native AGP gene.</P><H4><SPAN class="mw-headline" id="AGP_K.O..2FINV_Construct_Components"><B>AGP K.O./INV Construct Components</B></SPAN></H4><P><B>ADP Glucose Pyrophosphorylase (AGP) Gene:</B> The slr1176 open reading frame was PCR amplified from total Synechocystis PCC6803 genomic DNA and subcloned into pSB1a3. Forward and reverse primers were designed with 5’ extensions containing iGEM prefix and suffix sequences.</P><P><B>Invertase (INV) Gene+double terminator:</B> The INV gene was synthesized based on the sequence of a Zymonomas mobilis invertase gene described by Neiderholtmeyer et al. (Applied and Environmental Microbiology (2010) 76: 3462) which was codon optimized for expression in cyanobacteria. The gene was also synthesized with a double transcriptional terminator designed from sequences described for iGEM part BBa_B0015. The synthetic gene was subsequently subcloned into pSB1C and has been designated BBa_K558006.</P><P><B>petBD promoter+RBS:</B> The strong light inducible petBD promoter will be used to drive the expression of invertase gene in Synechocystis. The petBD+RBS (BBa_K390015) was generously provided to us by the Utah State iGEM team.</P><P><B>Kanamyacin resistance cassette (KanR):</B> The kanamycin resistance cassette was PCR amplified from pUC4K (Taylor and Rose (1988) NAR 16:358). This cassette includes the neomycin phosphotransferase 2 gene with its own constitutive promoter and transcriptional terminator. This cassette is frequently used to select for positive transformants in Synechocystis. </P><H3><SPAN class="mw-headline" id="Gibson_Assembly_of_AGP_K.O..2FINV_Components"><B>Gibson Assembly of AGP K.O./INV Components</B></SPAN></H3><P><B>Primer Design:</B>  Forward and reverse primers for each DNA part were designed with 20 base pair overlapping sequences with the upstream and downstream flanking segments. This will create 40 base pair overlaps between neighboring parts in the construct.</P><P><B>PCR:</B>  Inv, KnR, and petBD+RBS were amplified from BBa_K558006(Nevada), pUC4K, and K390015(Utah State) respectively under standard PCR conditions. AGP in pSB1A3 will be amplified to include the vector. This will be used as the backbone for Gibson Assembly.
</P><P><B>Assembly of Parts:</B>  The above parts will be assembled into the following construct
</P><CENTER>Primers developed for PCR and Gibson Assembly for INV operon. Overlapping DNA sequences can be created through PCR by adding twenty base-pairs of the end of your first genetic sequence to the beginning of your second genetic sequence. A blunt-end 40 base-pair sequence is then created that is ideal for Gibson Assembly.</CENTER><P><B>Transformation:</B> Synechocystis will be naturally transformed via homologous recombination.</P><H3><SPAN class="mw-headline" id="Engineer_Synechocystis_to_secrete_hexose_sugars_to_the_medium"><B>Engineer <I>Synechocystis</I> to secrete hexose sugars to the medium</B></SPAN></H3><P>
Synechocystis lacks the ability to secrete hexose sugars. Therefore it was necessary to introduce a sugar transporter through genetic engineering. Previous studies (Nedierholtmeyer et al (2010) Applied and Environmental Microbiology (2010) 76: 3462) showed that by expressing the Zymonomas mobilis glucose facilitative transporter (GLF) gene in Synechococcus, glucose and fructose were detected in the culture medium at levels as high as 30 uM and 150 uM respectively. Based on these results we decided to express GLF in Synechocystis. Our hope is that by expressing GLF in the AGP KO/INV overexpressing line, we will be able to produce enough glucose to the surrounding medium to support the E. coli biofuel production.</P><H4><SPAN class="mw-headline" id="ThiE_K.O..2FGLF_Construct_Design"><B>ThiE K.O./GLF Construct Design</B></SPAN></H4><P>
To integrate the GLF gene into the Synechocystis genome, we have decided to use an alternative insertion site. In this case we will insert the GLF overexpression gene cassette along with a chloramphenicol resistance cassette (CamR) into the coding region of the thiamin monophosphate pyrophosphorylase (ThiE) gene. The rationale for creating a ThiE knockout mutant is that it will lead to the creation of an auxotophic mutant that will only survive when the grown in the presence of thiamin (Vitamin B1). Therefore, the transgenic Synechocystis will not be able to survive outside laboratory and the chances of environmental contamination will be decreased. We will simultaneously create the ThiE mutant and the GLF overexpressing line by creating an ThiE gene replacement construct in which the GLF overexpression gene cassette and a chloramphenicol resistance gene cassette are inserted into the coding region of a subcloned ThiE gene. This dysfunctional ThiE construct will then be transformed into Synechocystis and through homologous recombination, replace the native ThiE gene. 
</P><H4><SPAN class="mw-headline" id="ThiE_K.O..2FGLF_Construct_Components"><B>ThiE K.O./GLF Construct Components</B></SPAN></H4><P><B>ThiE Gene:</B> The sll0635 open reading frame was PCR amplified from total Synechocystis PCC6803 genomic DNA and subcloned into pSB1a3. Forward and reverse primers were designed with 5’ extensions containing iGEM prefix and suffix sequences.</P><P><B>Glucose facilitative transporter (GLF) Gene+double terminator:</B> The GLF gene was synthesized based on the sequence of a Zymonomas mobilis invertase gene described by Neiderholtmeyer et al. (Applied and Environmental Microbiology (2010) 76: 3462) which was codon optimized for expression in cyanobacteria. The gene was also synthesized with a double transcriptional terminator designed from sequences described for iGEM part BBa_B0005.</P><P><B>petBD promoter+RBS:</B> The strong light inducible petBD promoter will be used to drive the expression of invertase gene in Synechocystis. The petBD+RBS (BBa_K390015) was generously provided to us by the Utah State iGEM team.</P><P><B>Chloramphenicol resistance cassette (CamR):</B> The chloramphenicol resistance cassette was PCR amplified from pSB1C3. This cassette includes the chloramphenicol acetyltransferase gene with its own constitutive promoter and transcriptional terminator. This cassette is frequently used to select for positive transformants in Synechocystis. </P><H3><SPAN class="mw-headline" id="Gibson_Assembly_of_ThiE_K.O..2FGLF_Components"><B>Gibson Assembly of ThiE K.O./GLF Components</B></SPAN></H3><P><B>Primer Design:</B>  Forward and reverse primers for each DNA part were designed with 20 base pair overlapping sequences with the upstream and downstream flanking segments. This will create 40 base pair overlaps between neighboring parts in the construct.</P><P><B>PCR:</B>  GLF, CamR, and petBD+RBS were amplified from BBa_K558005(Nevada), pSB1C3, and K390015(Utah State) respectively under standard PCR conditions. ThiE in pSB1A3 will be amplified to include the vector. This will be used as the backbone for Gibson Assembly.
</P><P><B>Assembly of Parts:</B>  The above parts will be assembled into the following construct
</P><CENTER>Primers developed for PCR and Gibson Assembly for GLF operon. Overlapping DNA sequences can be created through PCR by adding twenty base-pairs of the end of your first genetic sequence to the beginning of your second genetic sequence. A blunt-end 40 base-pair sequence is then created that is ideal for Gibson Assembly.</CENTER><H2><SPAN class="mw-headline" id="Results"><B>Results</B></SPAN></H2><P><B>Figure 1. Confirmation of invertase operon parts.</B>  PCR was performed to amplify petBD+RBS, KanR, and inv with 20 bp flanking overlap regions.  Equimolar volumes of each PCR product were mixed and ran on 0.7% agarose gel at 110 V for 60 minutes (lanes 2-9). 0.5 ug Kb+ standard was loaded into lanes 1 and 10.
</P><P><B>Figure 2. Confirmation of agp in TopoIIBluntPCR vector.</B> Agp was ligated into TopoIIBluntPCR vector (Invitrogen). 0.25 ug of each ligation was digested with SpI and PstI and ran on 0.7% agarose gel at 110 V for 60 minutes (lanes 2-8). 0.5 ug Kb+ standard was loaded into lanes 1 and 9.  Lanes 3,5, and 8 confirm successful ligation.
</P><P><B>Figure 3. Agarose gel electrophoresis of cleaved and uncut thiamin monophosphate pyrophosphorylase (ThiE) DNA.</B> 0.5 ug of DNA samples were first isolated before a small portion was cleaved using EcoRI and Pst I restriction enzymes. Four cut samples and uncut DNA was separated by electrophoresis in a 1% agarose gel and visualized under a UV light. The results show a successful ligation of ThiE.
</P><P><B>Figure 4. Agarose gel electrophoresis of PCR amplified chloramphenicol acetyltransferase (CmR) DNA. </B>5 ng of DNA was first amplified using PCR. The DNA product was separated by electrophoresis in a 1% agarose gel and visualized under a UV light. The results show a successful amplification of CmR.</P><P><B>Figure 5. Agarose gel electrophoresis of PCR amplified photosynthetic cytochrome b6 and subunit 4  (petBD) DNA.</B> 4 ng of DNA was first amplified using PCR. The DNA product was separated by electrophoresis in a 1.5% agarose gel and visualized under a UV light. The results show a successful amplification of petBD.
</P><P><B>Figure 6. Agarose gel electrophoresis of PCR amplified Zymonomas mobilis glucose facilitative transporter (GLF) DNA.</B> Three different samples of 10 ng of GLF DNA was first amplified using PCR. The DNA products was separated by electrophoresis in a 1% agarose gel and visualized under a UV light. The results show a successful amplification of GLF. 
</P><H2><SPAN class="mw-headline" id="Future_Directions"><B>Future Directions</B></SPAN></H2><P>
Each of the gene parts for the assembly of both the AGP KO/inv operon and the ThiE KO/GLF operon have been isolated and confirmed by gel electrophoresis. The primary goal now is to successfully construct each operon by Gibson assembly or SLIC. Once the operons have been confirmed transformation of Synechocystis will be carried out in two successive steps: first with the AGP KO/inv operon and, secondly, with the ThiE KO/GLF operon.
</P><H2><SPAN class="mw-headline" id="Assay_Development"><B>Assay Development</B></SPAN></H2><P>
Throughout the duration of this project, enzymatic assays were used to confirm the function of the genes used to transform E. coli and quantify the secretion of biofuel products. A hexokinase assay was used to measure glucose/fructose secretions from cyanobacteria, and oxidase/peroxidase assays were used to quantify free fatty acid and ethanol production from E. coli.

Most techniques for directly determining the concentrations of sugars, alcohols and fatty acids in culture media involve multiple steps, and may not distinguish between, for example, different types of sugars. We overcame these problems by using enzyme assays which, upon the addition of the sugars, alcohols, or fatty acids, result in the formation of other compounds which can be measured directly.

For each assay, samples of Synechocystis or E.coli medium were taken, and centrifuged to remove particulates. These samples were then added directly to assay mixtures containing enzymes and additional substrates necessary for the formation of compounds whose concentrations we could measure directly.

Coupled enzyme assays for Glucose, Fructose and Sucrose uses enzymes to catalyze a series of reactions, resulting in the formation of NADH, which we can measure directly in the spectrophotometer.
<B>Assay Description:</B> Invertase enzyme will be directly added to sample media to split sucrose into D-glucose and D-fructose, which are then added to a Hexokinase/Glucose-6-phosphate DeH assay mix, which will produce one NADH molecule for every one glucose molecule added.  NADH can be measure on the spectrophotometer at 340 nm and can be quantitated using Beer’s law and the NADH extinction coefficient (shown above).  Because the assay is glucose specific, the first reading will quantitate glucose present, then a Phosphoglucose isomerase enzyme will be added to the assay mix to convert fructose-6-phosphate into glucose-6-phosphate and a second reading will be taken, the increase in absorbance will be used to quantitate fructose present.

It should be noted that fructose is produced naturally by wild-type Synechocystis. It was therefore necessary to measure constitutive fructose production in wild-type cultures.
<B>Fatty Acid Production</B> was measured using the EnzyChrom Free Fatty Acid Assay Kit from Bioassay Systems according to the manufacture’s protocol. This kit uses a three step assay.
<B>Assay Description:</B> Fatty acids are enzymatically converted to acyl CoA and then to peroxide.  A Peroxidase then uses the resulting peroxide to oxidize a dye substrate forming a pink colored product with optical density (O.D.) at 570 nm. 

</P><H3><SPAN class="mw-headline" id="Ethanol_Production"><B>Ethanol Production</B></SPAN></H3><P>Ethanol Production was measured using a similar protocol to the free fatty acid assay:
<B>Assay Description:</B> Alcohol Oxidase converts primary alcohols like ethanol and diatomic oxygen into a formaldehyde and a peroxide, respectively.  The peroxide is then converted into two molecules of water by a peroxidase using an ABTS substrate as an electron donor.  The resulting oxidized ABTS will absorb at 405nm.  There is a 1:1 ration of ethanol to oxidized ABTS molecules; therefore we can use the molar extinction coefficient of oxidized ABTS in order to quantitate the amount of ethanol originally present.  
<B>The fatty acid and ethanol assays were quantitated by generating standard curves of absorbance vs substrate concentration using known quantities of palmitic acid or ethanol standards.</B></P><H2><SPAN class="mw-headline" id="Co-cultivation"><B>Co-cultivation</B></SPAN></H2><P>     
Co-culturing multiple bacterial strains is inherently complicated. The two species must co-exist while at the same time competing for resources. Over time the growth rate of two co-cultured species will establish equilibrium. However, conditions will not be optimal for either individual species. In our case, we want to grow the <I>E. coli</I> in a culture that is being provided with glucose from the cyanobacterium, <I>Synechocystis PCC6803</I>. It is clear that <I>E. coli</I> growth will be limited by the productivity of the cyanobacterium. Therefore, we need to develop a system that will optimize <I>Synechocystis</I> growth. One of the major considerations in cyanobacteria growth is the availability of sufficient light to optimize rates of photosynthesis. This is particularly important in our system since we are also depending on photosynthesis for the production of glucose to feed <I>E. coli</I>. While it is possible to grow <I>E. coli</I> and cyanobacteria in the same culture vessel (<I>Niederholtmeyer et al.</I>, 2010) Applied and Environmental Microbiology 76: 3462-3466), the photosynthetic efficiency of the system will be limited by light blockage caused by <I>E. coli</I>. For this reason, we have developed an apparatus that physically partitions the two bacterial species from each other, while still allowing for the free exchange of growth medium. In this way, we can ensure high photosynthetic rates for <I>Synechocystis</I> and total accessibility by <I>E. coli</I> to the cyanobacterial produced glucose.</P><H2><SPAN class="mw-headline" id="Choice_of_Growth_Medium"><B>Choice of Growth Medium</B></SPAN></H2><P>     
BG-11 must be used for culturing <I>Synechocystis</I>.  However, this medium does not provide adequate nourishment to <I>E. coli</I>.  Thus, this project must determine what additives to BG-11 will foster growth of both organisms.  The supplemented media must allow for growth of <I>E. coli</I> comparable to LB but must not inhibit <I>Synechocystis</I> with toxic levels of solutes. </P><P>     
Initial tests will determine if BG-11 has an inherently sufficient nitrogen supply for <I>E. coli</I>.  Previous research has shown that NH<SUB>4</SUB><SUP>+</SUP> is more effective than nitrates and nitrites when used by <I>E. coli</I> as a nitrogen source (<I>Strevett, K.A., and Chen, G.,</I> 2003).  Therefor, testing will be done with the growth of NEB 10-ß cells in BG-11 with NH<SUB>4</SUB>Cl as an additional source of nitrogen.  The growth in this media will be compared with that of BG-11 alone.  It must then be shown that <I>Synechocystis PCC6803</I> can tolerate the levels of ammonium chloride required.  If ammonium chloride cannot be used, nitrites and nitrates may then be used as a backup option.</P><P>     
Once a source of nitrogen has been found, experiments will be done to find a standard curve.  This curve will give us a baseline we can compare with when determining if our <I>E. coli</I> is being successfully fed by glucose produced in cyanobacteria.  The ideal media will produce an optical density of 1.00, at 600 nm, after 24 hours.  When glucose is added to this media, we should see a greater extent of growth, giving a positive result for co-cultivation.</P><H2><SPAN class="mw-headline" id="Results_of_Growth_Curves"><B>Results of Growth Curves</B></SPAN></H2><H3><SPAN class="mw-headline" id="Growth_in_BG-11_.2B_NH4Cl"><B>Growth in BG-11 + NH<SUB>4</SUB>Cl</B></SPAN></H3><P>     
It was quickly found that 10-ß growth was limited in BG-11 due to one or more auxotrophies.  Upon contacting NEB tech support, it was confirmed that 10-ß cells are deficient in leucine synthesis.  Thus preventing them from growing in BG-11 that is not supplemented with this amino acid.  However, further testing with BG-11 and leucine indicated that 10-ß still could not grow significantly in this media.</P><P>     
Testing was then done with other cell lines, in an attempt to find a cell line that was not inhibited by amino acid auxotrophies.  BL-21 cells were first used, with no luck.  NEB tech support was again contacted and asked for a cell line that was not deficient for any amino acid synthesis.  NEB recommended Express I<SUP>q</SUP> cells, claiming that they were not auxotrophic and would be ideal for expression studies using the pTRC promoter.  Unfortunately, it was found that even I<SUP>q</SUP> cells were not capable of significant growth in media lacking amino acid supplements.</P><H3><SPAN class="mw-headline" id="Growth_in_BG-11_.2B_Amino_Acids"><B>Growth in BG-11 + Amino Acids</B></SPAN></H3><P>     
Upon finding that none of the cell lines available to the team were capable of growing in BG-11 supplemented with ammonium chloride as the only nitrogen source, sources of amino acids were looked at.  The first of these sources was tryptone, a trypsin digest of casein providing all amino acids.  Our studies showed that a concentration of 0.5%  w/v tryptone produced our desired standard curve, with an average OD at 600 nm of 0.992 (n=2), 25 hours after inoculation with an overnight culture.  This experiment was done using 10-ß cells.  Readings on the contents of tryptone showed that it contained unacceptable amounts of sugars (<I>Biotech Solabia Group</I>).  This would interfere with our studies into the effects of adding glucose to our media.</P><P>     
The next source of amino acids tested was casamino acids, another product of casein.  However, casaminos provide greater amounts of free amino acids and small peptides and do not contain notable quantities of sugars (<I>BD Difco</I>).  Our experiments produced the desired standard curve with a concentration of 0.25% w/v casaminos, in the case of 10-ß, and 0.20% w/v for I<SUP>q</SUP>.
</P><H3><SPAN class="mw-headline" id="Detection_of_Glucose"><B>Detection of Glucose</B></SPAN></H3><P>     
With standard curves found for both 10-ß and I<SUP>q</SUP>, using casaminos, experiments were done to determine if we could induce greater growth with the addition of glucose.  Our initial experiment was with 10-ß and glucose was added in 10, 25 and 50 mM amounts.  The data showed that 10-ß responded negatively to added glucose, giving 600 nm ODs of greater than 35% less than our standard curve at all concentrations.</P><P>     
An identical experiment was performed with I<SUP>q</SUP> cells.  In this case, glucose increased growth by up to 38% (at 25 mM glucose) over the standard curve after 25 hours.  With a positive result, the experiment was repeated in triplicate.  The results confirmed those of the initial experiment.  In the presence of 10 mM glucose, the average OD at 600 nm, after 24 hours, was 67.9% (n=3) greater than our standard curve.</P><P>     
Having confirmed that I<SUP>q</SUP> cells respond positively to glucose in BG-11 media with casamino acids, we wanted to look at the levels of glucose we would need to feed our <I>E. coli</I>.  Other researchers have shown that <I>Synechocystis PCC6803</I><I>agp</I> knockouts are capable of producing 10 mM sucrose (<I>Zhao, N., et al.</I> 2003).  Another group has shown that introducing <I>invA</I> to cyanobacteria can convert nearly all cellular sucrose to fructose and glucose (<I>Niederholtmeyer et al.</I>, 2010).  On the notion that 10 mM sucrose can be produced by knocking out <I>agp</I> and all sucrose can be converted to glucose, it was hypothesized that our maximum glucose output from <I>Synechocystis</I> would be 10 mM.  Our previous experiments had shown that there was no noticeable difference in growth increases between 10, 25 and 50 mM glucose, with 10 mM actually showing the greatest increases.  If we could detect even lower amounts than our hypothesized maximum, we would be ready to begin co-culturing.  An experiment was done, in triplicate, in which glucose was tested in BG-11 in the following decreasing concentrations: 10, 5.0, 2.5, 1.0, 0.50 mM.  Even at concentrations as low as 2.5 mM I<SUP>q</SUP> cells showed increases in growth over the standard curve of up to 60%.  Even at concentrations of 500 µM, growth was increased more than 10% greater. 
With confirmation that such small amounts of glucose can successfully nourish <I>E. coli</I>, it was time to begin testing the co-culturing apparatus.
</P><H2><SPAN class="mw-headline" id="Apparatus_Components"><B>Apparatus Components</B></SPAN></H2><P><B>Culture Vessels</B>    The actual chambers are made from modified chromatography columns. They are Pyrex glass tubes with removable plastics caps. Each chamber has a double seal to prevent leaking.  Both of these are supported by a ring stand with clamps.</P><P><B>Transfer Pump</B>     
One main thing required to have an effective apparatus is a pump to transfer one bacteria solution to the other chamber into a permeable membrane. It needs to be able to transfer enough volume to fill the dialysis tube completely but not create too much pressure that the tubing would burst. An external pump was chosen to reduce the number of components that would touch the bacteria and lesson the chances of contamination. It also reduces the amount of parts to be cleaned. On our apparatus the <I>E. coli</I> is gravity fed to the transfer pump inlet, then pumped through clear vinyl tubing to the dialysis tube and returned back to the <I>E. coli</I> chamber. It pumps approximately 25 gallons/hr.</P><P><B>Semipermeable Membrane</B>     
The dialysis tubing allows regulated bacteria interaction, meaning glucose from the Cyanobacteria can transfer over to the <I>E. coli</I> but the two bacteria never contact. Something was needed to give the dialysis tubing rigidity and also allow effective flow of <I>E. coli</I> through the inside of the tubing. The dialysis tubing also needed something to seal the ends to prevent contamination. A borosilicate tube was taken and had bubbled ends installed for the sealing surface to the dialysis tubing. The glass tubing then had small holes placed in it with a wall in the middle. This made one end an inlet where the media would enter the glass tube flow out of the small holes into the dialysis tubing filling it on its way to the other end. From there it would exit through the other set of small holes and return to the vinyl tubing and the second chamber.  To seal the dialysis tubing to the glass a piece of shrink tube was placed over the dialysis tubing compressing it against the glass. Next a rubber o-ring was placed over the shrink tube to provide an extra seal.</P><P><B>Oxygenation</B>     
For proper growth, each chamber needs a constant flow of oxygen. A four-channel variable aquarium oxygen pump, inside the apparatus base, oxygenates the Cyanobacteria and <I>E. coli</I>. Each chamber is fed from the bottom through two lines, check valves were installed to prevent the bacteria from draining down the tubing into the oxygen pump. On the top of the base is the adjustment knob to control the oxygen flow. Hand blown glass bubblers, made from borosilicate glass, were attached to the top of each chamber to allow proper venting without releasing the bacteria out. </P><P><B>Illumination</B>    
In order for photosynthesis to take place some sort of artificial light is necessary. Two T5 14W fluorescent bulbs, each connected to a slider that allows them to be independently adjusted to vary the amount of light as needed in the Cyanobacteria chamber.</P><P><B>Controller</B>     
The base is constructed of 6 panels of 6061 aluminum tig(GTAW) welded together. Its dimensions are approximately sixteen inches long, fourteen inches wide, and three inches tall. All electrical switches and wiring are inside with the oxygen pump and rings stand base to keep bacteria out and make it easier to clean.
</P></DIV></DIV><DIV id="bottom-content"><DIV id="sponser-banner-content"><DIV class="Sponser-text">SPONSORS</DIV></DIV><DIV id="sitemap-banner-content"><DIV id="subNav"><DIV class="subNavText1">
Navigation
</DIV><DIV class="subNavText2"><A href="/Team:Nevada">
Home
</A>     <A href="/Team:Nevada/Team">
Team
</A>     <A href="/Team:Nevada/Project">
Project
</A>     <A href="/Team:Nevada/Parts">
Parts
</A>     <A href="/Team:Nevada/Notebook/Weeks14">
Notebook
</A>     <A href="/Team:Nevada/Outreach">
Outreach
</A>     <A href="/Team:Nevada/Contributions">
Contribution
</A>     <A href="/Team:Nevada/Fundraising">
Fundraising
</A>     <A href="/Team:Nevada/Reference">
References
</A>     <A href="/Team:Nevada/Safety">
Safety
</A></DIV></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:Nevada/Project">http://2011.igem.org/Team:Nevada/Project</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Nevada/Project" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Nevada/Project" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Nevada/Project&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Nevada/Project&amp;oldid=203085" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>