"
Page
Discussion
View source
History
teams
Log in
 
Team:SCUT-China/Project/Synthetic TKL
From 2014.igem.org
Home
Team
Members
Attributions
Sponsors
Collaborators
Notebook
Protocol
Project
Background
Overview
Polyketide Synthesis
Chassis
PPS
Docking Domain
Have A Try
Modeling
Overview
Simulation
Results
Data Page
Parts
Judging Criteria
Policy&Practices
Overview
Campus Activity
Public Outreach
Social Media
Life Science Community
Survey
Evaluation
Safety
Background
Overview
Polyketide Synthesis
Overview Design Results References
Chassis
Overview Design Result References
PPS
Part1 Design Result Conclusion
Docking Domain
Overview Design Result Conclusion
Have A Try
Background Design Operation
Polyketide Synthesis
Overview
In this project, we propose that DNA coding for polyketide synthase   domains can be standardized and reconstructed to generate chimeric   polyketide synthases that produce new-to-nature polyketides, through   which we can dramatically expand the library of antibiotic candidates.   In order to demonstrate the feasibility of our proposal, we conducted   “proof of concept” experiments to synthesize a simple polyketide—  triketide lactone 1B—using a truncated erythromycin-producing   polyketide synthase assembled with biobricks constructed by   standardizing essential domains required for the synthesis of the   triketide lactone. The domains we standardized are the loading domain,   domains of DEBS1 and the thioesterase (TE) domain from DEBS3 (Fig. 1).   The loading domain is required for the direct loading of propionyl-CoA   to form a starter unit for triketide lactone. DEBS1 is a large   multifunctional polypeptide composed of two modules that incorporate   (2S)-methylmalonyl-CoA to form extender units. The TE domain catalyzes   the intramolecular cyclization of the triketide to form the target   triketide lactone. [1]The target product is expected to be detected   and identified from the culture broth.
Design
Candidate selection
Biosynthesis of polyketides like erythromycin, tacrolimus and many   others are guided by modular PKS genes. These genes encode large   enzymes consisting of modules of domains, forming an "assembly line"   to extend the ketide units to form polyketides. Among the known PKS   genes, the genes coding for erythromycin producing polyketide   synthethase have been extensively and deeply studied, and there are   numerous information sources. Therefore, we chose a truncated   erythromycin-producing polyketide synthase which has been proved to   produce a simple triketide lactone 1B.[2]
Standardization of domains
Figuring Out the Boundary
With the idea of engineering PKS genes to be interchangeable domains,   we firstly develop certain design principles for the engineering of   PKS genes. Although many PKSs are highly modular proteins consisting   of modules of domains, figuring out the boundary of each domain should   be carefully considered because slight changes may result in lost of   function.To achieve this goal, we searched for more information from two   database ASMPKS (Analysis System for Modular Polyketide Synthases:( http://gate.smallsoft.co.kr:8008/~hstae/asmpks/pks_prediction.pl ) and MAPSI (Management and Analysis for Polyketide Synthase type I: ( http://gate.smallsoft.co.kr:8008/pks/ ). According to the resource   provided by these websites, we separated the sequences of DEBS1 with   centering on domains.
Standardization of Interchangeable Domains
Once the boundaries of the domains have been defined, standard prefix   or suffix should be assigned to each end of the domains to make them   interchangeable. The standardization of sufficient numbers of domains   from numerous organisms would allow customized replacement,   rearrangement, and/or recombining of different domains to produce   novel polyketides. In this project, we tested customized synthesis of   triketide lactones by replacing the loading domain and swapping KR+ACP   domains between modules to test the compatibility of the standardized   domains.
The “proof-of-concept” product--triketide lactone 1B
Assemble standardized domains
To validate our proposal we attempted to express the varient --  DEBS1+TE in E.coli. [3] Initially, we tried to amplify the coding   sequences for each individual domain from Saccharopolyspora erythraea genomic DNA by PCR using primers consisting of RFC 23. However, after   laborious work of weeks, we had failed to obtain proper PCR products   due to the high G+C content (≥70%) of the interest DNA sequences.   Afterward, to achieve our goal in a more efficient way, we contracted   GENEWIZ, an international genomics service company, to synthesize the   genes. With the coding genes under the control of T7 promoter and   double T7 terminator, we successfully assembled three constructs for   our design.
We were able to assemble all of our constructs and continued our work   with our synthetic DEBS1+TE, DEBS1’+TE (KR of module2 in DEBS1 was   replaced with KR of module1) and RFP+DEBS1+TE (DEBS1 was fused with   red fluorescent protein) by transforming them into the E. coli strain   BAP1. (Fig. 2)
If the right product triketide lactone can be generated, we can   conclude that the standardized domains can work relatively   independently and generate normal product.
Also, we changed the loading domains which selected different   substrates resulting in product of different polyketides.
Furthermore, we can shuffle the domains as units between different   DEBSs.
Induction of PKSs
12-48 hours after induction with IPTG, culture broth samples of the   recombinant strain expressing the chimeric polyketide synthetases were   collected and subjected for further analysis.
Results
1. DNA Electrophoresis
We used restriction enzyme digestion and/or PCR to validate the   constructs (Fig. 3). The constructs carrying DEBS1+TE and DEBS1’+TE   were isolated and digested with EcoRI (Fig. 3, lane 1 and 2) or double   digested with EcoRI+PstI (data not shown). The construct carrying RFP  +DEBS1+TE was validated with PCR (Fig. 3 lane 3) and EcoRI+PstI double   digestion (data not shown). The results showed that all of the three    constructs were successfully created.
2. SDS-PAGE
DEBS1 is a large protein of 370kDa. We performed SDS-PAGE to confirm   the expression of the recombiant DEBS1+TE and DEBS1’+TE. However,   none of DEBS1+TE, DEBS1’+TE (data not shown) and RFP+DEBS1+TE (Fig.   4) could be detected after Coomassie blue staining.
3.RT-PCR validation on transcriptional level
After failure of detecting target proteins using SDS-PAGE, we used   RT-PCR to examine the expression of the target genes on   transcriptional level. The results were shown in Fig. 5. We used 16S   rRNA gene as an house keeping gene to compare the expression level of   target genes. The results showed that DEBS1 gene was successfully   expressed, albeit the transcription level was relatively low compared   with other target genes (acc, pcc and sfp) whose transcription are   controlled by T7 promoter as well. The low transcription level of   DEBS1 gene is in consistent with our SDS-PAGE results. This result   might due to the large size of DEBS1, which could hamper both the   transcriptional and translational efficiency.
4.Fluorescence detection analyses
To further confirm the expression of DEBS1, we monitored the   fluorescence of RFP under Zeiss LSM710 confocal microscope (Fig. 6).   The results showed that after excitation with lazer source, red   fluorescence was monitored in the RFP+DEBS1+TE transformant, while in   the BL21 strain, no fluorescence was detected. These results indicate   that RFP+DEBS1+TE was successfully expressed in the transformant.
Figure6: Results of fluorescence detection
5.UPLC-MS/MS Detection of TKL 1B
To test if RFP+DEBS1+TE was functionally expressed, the transformant   was grown in Luria Broth (LB) liquid medium and induced with IPTG.   After 48h of induction, cells were harvested and the supernatant was   used for the extraction of the product TKL1B. After extraction, the   samples were concentrated by 5 times and subjected into a triple   quadrupole mass spectrometry and monitored under an MRM mode using the   parent ion /product ion=173.1/155.1 channel. The results were shown in   Fig. 7. The retention time of the target product in the UPLC column   was 6.42 min, using water/methanol gradient elution protocol. We   monitored a strong response at m/z = 155.1 ion pair in the RFP  +DEBS1+TE transformant sample. The ion pair of 173.1/155.1 is specific   to the expected product TKL1B, and is in consistent with previous   studies. These results demonstrated that TKL1B was successfully   produced from the transformant.[4]
Conclusion
In this project, we realized the design by standardizing domains from   the gene cluster of DEBSs. To our knowledge, it was the first time to   assemble domains as individual units.
We performed separations of the coding sequence resulted in   standardized domains and assembled these domains in correct order for   generating our target product triketide lactone.[5] Evaluation of   recombinant DEBS+TE expression was performed by RT-PCR analyses,   fluorescence microscopy analyses, SDS-PAGE and final polyketide   product was detected using Mass Spectrometry. Our results demonstrated   that standardized domains could express correctly folded and   posttranslationally modified modular PKSs, we concluded that domains   in PKSs can work relatively independently. Furthermore, RFP may   influence the function of protein, but in our project the product   could be generated via BAP1 harboring plasmids with RFP-DEBS1-TE.   Also, there is one scar (Thr-Arg) every two parts produced in every   ligations because of RFC 23 .We had worried these additional amino   acids may affect the expression of our synthetase. However, the   complete construction with 20 additional amino acids of RFP-DEBS1-TE,   expressed well from the IPTG-inducible promoter present in pSB1C3 in   E. coli strain BAP1. According to these results, we demonstrated the   compatibility as well as interchangeability between different   standardized domains. [6]
Based on investigation of standardized domains coming from semi-  rational separation, the obtained results represented an important   milestone toward the ultimate goal of creating novel PKSs by   assembling standardized domains.Additionally, these results showed   a new approach to generating triketide lactone, which can be an   important source of advanced intermediates of complex polyketides.[7]
References
[1] Keatinge-Clay, Adrian T. "The structures of type I polyketide   synthases."Natural product reports 29.10 (2012): 1050-1073.
[2] Menzella, Hugo G., John R. Carney, and Daniel V. Santi. "Rational   design and assembly of synthetic trimodular polyketide synthases."   Chemistry & biology 14.2 (2007): 143-151.
[3]Pfeifer, Blaine A., et al. "Biosynthesis of complex polyketides in   a metabolically engineered strain of E. coli." Science 291.5509   (2001): 1790-1792.
[4] Menzella, Hugo G., et al. "Combinatorial polyketide biosynthesis   by de novo design and rearrangement of modular polyketide synthase   genes." Nature biotechnology 23.9 (2005): 1171-1176.
[5] Jiang, Ming, and Blaine A. Pfeifer. "Metabolic and pathway   engineering to influence native and altered erythromycin production   through E. coli." Metabolic engineering 19 (2013): 42-49.
[6] Yan, John, et al. "Functional Modular Dissection of DEBS1-TE   Changes Triketide Lactone Ratios and Provides Insight into Acyl Group   Loading, Hydrolysis, and ACP Transfer." Biochemistry 51.46 (2012):   9333-9341.
[7] Hughes, Diarmaid. "Exploiting genomics, genetics and chemistry to   combat antibiotic resistance." Nature reviews genetics 4.6 (2003):   432-441.
Retrieved from " http://2014.igem.org/Team:SCUT-China/Project/Synthetic_TKL "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
