<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_SCUT_Project_System_Construction_Co2_Fixation"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:SCUT/Project/System_Construction/Co2_Fixation">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:SCUT/Project/System_Construction/Co2_Fixation&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:SCUT/Project/System_Construction/Co2_Fixation&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:SCUT/Project/System_Construction/Co2_Fixation&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:SCUT/Project/System_Construction/Co2_Fixation" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:SCUT/Project/System Construction/Co2 Fixation</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><DIV id="menu"><DIV class="menubar"><UL><LI id="home" title="Home"><A href="https://2014.igem.org/Team:SCUT">HOME</A></LI><LI id="team" title="Team">TEAM
</LI><LI><A href="https://2014.igem.org/Team:SCUT/Team/Members">Members</A></LI><LI><A href="https://2014.igem.org/Team:SCUT/Team/Notebook">Notebook</A></LI><LI><A href="https://2014.igem.org/Team:SCUT/Team/Attributions">Attributions</A></LI><LI><A href="https://2014.igem.org/Team:SCUT/Team/Acknowledgement">Acknowledgements</A></LI><LI><A href="https://2014.igem.org/Team:SCUT/Team/Our University">Our University</A></LI></UL><LI id="project" title="Project">PROJECT
</LI><LI><A href="https://2014.igem.org/Team:SCUT/Project/Overview">Overview</A></LI><LI><A href="https://2014.igem.org/Team:SCUT/Project/System_Construction/Co2_Fixation">System Construction</A></LI><LI><A href="https://2014.igem.org/Team:SCUT/Project/Analysis and Discussion">Analysis and Discussion</A></LI><LI><A href="https://2014.igem.org/Team:SCUT/Project/Other Work">Other Work</A></LI><LI id="model" title="Modeling">MODELING
</LI><LI><A href="https://2014.igem.org/Team:SCUT/Model/Overview">Overview</A></LI><LI><A href="https://2014.igem.org/Team:SCUT/Model/Rubisco_simulation">Carbon dioxide fixed simulation</A></LI><LI><A href="https://2014.igem.org/Team:SCUT/Model/N-butanol_simulation">n-butanol simulation</A></LI><LI><A href="https://2014.igem.org/Team:SCUT/Model/Tips_for_other_teams">Tips for other teams</A></LI><LI id="data" title="Data Page">DATA PAGE
</LI><LI><A href="https://2014.igem.org/Team:SCUT/DP/Biobricks">Biobricks</A></LI><LI><A href="https://2014.igem.org/Team:SCUT/DP/Achievements">Achievements</A></LI><LI><A href="https://2014.igem.org/Team:SCUT/DP/Protocol">Protocol</A></LI><LI id="hp" title="Human Practice"><A href="https://2014.igem.org/Team:SCUT/HP">HUMAN PRACTICE</A></LI><LI><A href="https://2014.igem.org/Team:SCUT/HP/Timeline">Timeline</A></LI><LI><A href="https://2014.igem.org/Team:SCUT/HP/Visit and interview">Magazine</A></LI><LI><A href="https://2014.igem.org/Team:SCUT/HP/Practical analysis">Survey and Data Analysis</A></LI><LI id="safety" title="Safety"><A href="https://2014.igem.org/Team:SCUT/Safety">SAFETY</A></LI></DIV></DIV><DIV id="pro_sc_co2"><DIV id="combine"><DIV id="left"><DIV class="navibody navibody1"><P>Overview&gt;&gt;&gt;</P></DIV><DIV class="navibody navibody2" id="show"><DIV class="navihead navihead2_1"><A href="https://2014.igem.org/Team:SCUT/Project/System_Construction/CO&lt;sub&gt;2&lt;/sub&gt; Fixation" id="special" title="A link"><P>CO<SUB>2</SUB> Fixation&gt;&gt;&gt;</P></A></DIV><DIV class="navibody navibody2_1" id="show_1"><P onclick="scroll_1()">Introduction</P><P onclick="scroll_2()">Design</P><P onclick="scroll_3()">Construction</P><P onclick="scroll_4()">Leading Peptide Test</P><P onclick="scroll_5()">References</P></DIV><DIV class="navihead navihead2_2"><A href="https://2014.igem.org/Team:SCUT/Project/System_Construction/n-Butanol Prod" id="special" title="A link"><P>n-Butanol Prod&gt;&gt;&gt;</P></A></DIV><DIV class="navibody navibody2_2" id="navibody2_2_2"><P>Introduction</P><P>Design</P><P>Construction</P><P>Localization Test</P><P>References</P></DIV></DIV><DIV class="navibody navibody3"><P>Introduction</P><P>Simulation</P><P>Reference</P></DIV><DIV class="navibody navibody5"><P>Outline</P><P>Reference</P></DIV></DIV><DIV class="mainbody mainbody1" id="label_1"><P class="atop"><SPAN>Introduction</SPAN></P><P>The components of the enzyme machinery anchored onto the outer mitochondrial membrane were selected within the known endogenous metabolic enzymes of S. cerevisiae and other exogenous enzymes, considering the high availability of ATP and CO<SUB>2</SUB> around mitochodria. The selected enzymes were PRK ( phosphoribulokinase ), RuBisCo ( Ribulose-1,5-bisphosphate carboxylase ) and CA ( carbonic anhydrase ) that working sequentially improve the carbon sequestration of the yeast，or in other words, increase the yield of pyruvate thereby increasing butanol production. To maximize the use of ATP and CO<SUB>2</SUB>, a selected leading peptide (Tom22) was used to anchor three exogenous scaffold proteins (GBD, SH3 and PDZ) able to bind the above-mentioned enzymes within a limited distance from the membrane.
</P><P style="margin-top:10px;"><B>Figure 1 | The overview of our project.</B></P><SPAN class="xiaobiaoti">PRK</SPAN><P>
A phosphoribulokinase is an enzyme that catalyzes the chemical reaction:
<B>ATP + D-ribulose 5-phosphate → ADP + D-ribulose 1,5-bisphosphate</B>
This enzyme belongs to the family of transferases, specifically those transferring phosphorus-containing groups (phosphotransferases) with an alcohol group as acceptor. This enzyme participates in carbon fixation. As is stated in previous research, increased expression levels of PKR in yeast result in an overall positive effect on the ethanol yield. In the meantime, though, these would bring a small metabolic burden to the host cell. Therefore, its expression level should be controlled.

The PRK employed in this pathway is from Spinacia oleracea, catalyting the first reaction of this pathway. 
</P><SPAN class="xiaobiaoti">RuBisCo</SPAN><P>
Ribulose-1,5-bisphosphate carboxylase/oxygenase, commonly known by the abbreviation RuBisCO, is an enzyme involved in the first major step of carbon fixation, a process by which atmospheric carbon dioxide is converted by plants to energy-rich molecules such as glucose. 
<B>Ribulose 1,5-bisphosphate + CO<SUB>2</SUB> → 2x 3-phosphoglycerate</B>
In cyanobacteria and many chemolithoauto-trophic bacteria, most if not all of RuBisCo is packaged in protein microcompartments called carboxysomes. It is probably the most abundant protein on Earth. However, RuBisCO also catalyses a reaction between ribulose-1,5-bisphosphate and molecular oxygen (O<SUB>2</SUB>) instead of carbon dioxide (CO<SUB>2</SUB>). Thus, something must be taken to increase concentration of the substrate of interest (CO<SUB>2</SUB>)to bring a high yield. 

The RuBisCo we use in this pathway, cbbs, is a prokaryotic form-Ⅱ RuBisCo from Thiobacillus denitrificans and used for catalyting the second reaction of this pathway
</P><SPAN class="xiaobiaoti">CA</SPAN><P>Carbonic anhydrases (or carbonate dehydratases) form a family of enzymes that catalyze the rapid interconversion of carbon dioxide
and water to bicarbonate and protons (or vice versa), a reversible reaction that occurs relatively slowly in the absence of a catalyst. The active site of most carbonic anhydrases contains a zincion; they are therefore classified as metalloenzymes.
The reaction catalyzed by carbonic anhydrase is:
<B>H<SUB>2</SUB>CO<SUB>3</SUB> → CO<SUB>2</SUB> + H<SUB>2</SUB>O</B>

The reaction rate of carbonic anhydrase is one of the fastest of all enzymes, and its rate is typically limited by the diffusion rate of its substrates. The epsilon(ε) class of CAs occurs exclusively in bacteria in a few chemolithotrophs and marine cyanobacteria that contain cso-carboxysomes.

The CA we use is a component of the carboxysome shell of Halothiobacillus neapolitanus c2.
</P><SPAN class="xiaobiaoti">Chaperons</SPAN><P>It is stated that Functional expression of RuBisCo would be strongly stimulated in the presence of its chaperons. Compared with  the endogenous chaperon couple of S.cerevisiae, Hsp60/Hsp10, existing in mitochondrial matrix, EroGL/EroGS, two E.coli protein-folding chaperones, are much more efficient since RuBisCo expression requires their activity in the cytosol. 
</P><P style="margin-top:10px;"><B>
Figure 2 | The schematic of mechanism of the molecular chaperons GroES and GroEL.</B> Firstly, an ATP binds with GroEL and the ATP-GroEL immediately captures the other unit GroES and there comes the GroEL-GroES-ATP complex.With the help of the complex, the unfolded RuBisCo protein can then properly fold and make a difference in our project.
</P><SPAN class="xiaobiaoti">Scaffold Protein</SPAN><P>
Scaffold protein, without enzymatic activity, can combine with two or more proteins which increases the efficiency of interaction between individual partner molecules by the simple tethering mechanism. Beyond that, these proteins can also exert complex allosteric control over their partners as well as themselves.

Obviously, scaffold proteins offer a simple, flexible strategy for regulating selectivity in pathways, shaping output behaviors, and achieving new responses from preexisting signaling components. Scaffold proteins hence have been exploited by evolution, pathogens and cellular engineers to reshape cellular behavior.</P><P style="margin-top:10px;"><B>Figure 3 | </B>As shown in the picture, we construct the scaffold protein fusing with the Tom22 which is the member of the Tom family. Tom family is the protein complex that helps the protein including leading peptide enter into the mitochondria. We utilized the embed protein Tom22 to make our scaffold protein (the scaffold protein fuse with the Tom22).
</P><SPAN class="xiaobiaoti">Leading Peptides</SPAN><P>
In Saccharomyces cerevisiae, Tom protein complex is positioned on the outer mitochondria membrane. Almost all precursor proteins transported into mitochondria must cross the Tom protein complex. Tom family, including Tom20, Tom22, Tom40, Tom5, Tom6 and Tom7. Tom20 and Tom70, are the initial receptors of Tom complex. Tom22 is anchored to the outer membrane by its hydrophobic segment in the middle of the sequence and it can carry the precursor protein into the channel formed by Tom40, the β-barrier protein which precursor protein can cross, while Tom 5, Tom 6, Tom7 subunits play an important role in stabilize the Tom complex.
</P></DIV><DIV class="mainbody" id="label_2"><P class="atop"><SPAN>Design</SPAN></P><P>
Apparently, when happen in a limited space, reactions accelerate owing to bigger probability of collisions between molecules. To utilize this nature, we recruited scaffold proteins. In spite of the fact that there are endogenous scaffold proteins in S.cerevisia, we used GBD, SH3 and PDZ, three exogenous scaffold proteins so as to avoid competing with some endogenous metabolic pathways. By this way, we aimed to minimize the negative effect on metabolisms of host cells and hence to optimize our system.
</P><P>
We construct different radios of scaffold proteins, GBD(x), SH3(y) and PDZ (z), of which can combine with PRK, RuBisCo and CA. By altering the number of the domains, we hope to obtain the most appropriate scale of scaffold proteins and get the best result. In view of the high efficiency of PRK, we chose x=1 at the first attempt (#GBD: #SH3: #PDZ, x: y: z), and will choose the optimal radio or construct other ratios according to the output of ethanol.
</P><P>
The design of scaffold protein is based on the Tom family. As shown in the figure, there is a protein complex called Tom family located on the outer mitochondria membrane. Tom family which contains Tom20, Tom22, Tom40, Tom5, Tom6 and Tom7 subunits can help the protein synthesized in the cytoplasm with presequence enter into the mitochondria. Because Tom22 is anchored to the outer membrane by its hydrophobic segment in the middle of the sequence, exposing the N-terminal and C-terminal domains to the cytosol and the outer membrane, intermembrane space(IMS). Thus, we decide to design fusion protein containing target proteins and Tom22. We hope we can successfully build up the scaffolding protein binding on the outer mitochondria membrane and help finish the rest of jobs.
</P><P>
It is stated in the background that increased expression level of PRK would bring bad effects to the host cells, which means a something should be taken into the construction of PRK recombinant molecule for regulating its expression level. Hence, we recruited a commonly used inducible promoter, pGal1 for initiation of translation. In the meantime, we quantified Gal1 promoter activity to find out the optimal point/range of induction dose (shown in <A href="https://2014.igem.org/Team:SCUT/Project/Other_Work">Quantification of Promoter Activities</A>).
</P></DIV><DIV class="mainbody" id="label_3"><P class="atop"><SPAN>Construction</SPAN></P><P>The five genes encoding these six enzymes were cloned into two different plasmids-----YEplac181 and YEp352, which can coexist in one cell.Considering our multiple-enzyme system, it is necessary to use those compatible plasmids. Most genes were synthesized by Genewiz: Hbd, Crt, AdhE2 (C. beijerinckii), and Ccr ((Streptomyces collinus). While S. cerevisiae gene: Erg10 were cloned from genomic DNA. And all enzymes were under control of GAL1 promoter with CYC1 or ADH1 terminator.
We combine not only our enzymes PRK, RuBisCo and CA with the plasmid Yeplac181 but also the GBD,SH3 and PDZ with the plasmid Yep352.Both of them will transform into one cell. We've construct different scales in order to find out the best scale to fit our pathway enzyme such as PRK, RuBisCo and CA</P><P>
These are constructions of our pathway and scaffold protein.
</P><P style="margin-top:10px;"><B>Figure 4 |</B> We made our construction in plasmid Yeplac181 and Yep352.The construction of PRK, RuBisCo and CA were built up in the plasmid Yeplac181. Moreover, the FP + ligand was also constructed in the plasmid Yeplac181.Yeplac181 and Yep352 , which can coexist in one cell.Considering our multiple-enzyme system, it is necessary to use those compatible plasmids so we added the GBD+SH3+PDZ scaffold protein sequence in the plasmid YEp352.
</P><P style="margin-top:10px;"><B>Fiugre 5 | </B>The SCUT A is the pathway that the RuBisCo contains the molecule chaperone in contrast with the SCUT B which lacks the molecule chaperone. The SCUT 1-1-Y means the construction includes the molecule chaperone and can fix on the outer mitochondria membrane successfully. It also means that 1-1-Y includes two device, one is the GBD, SH3, PDZ scaffold protein with different scale(1-1 means the SH3:PDZ ) ,the other is the pathway of the carbon dioxide fixation whether it owns the molecule chaperone. On the contrary the 1-1-N is the oppisite.
</P><P style="margin-top:10px;"><B>Figure 6 | At the pre-experiment</B>, to test out whether CA can make a difference or not, we had two constrast recombinant groups, one(group Ⅰ) consisting of PRK-RuBisCo(in YEplac181) and GroES-GroEL(in YEp352), the other（group Ⅱ） consisting of PRK-RuBisCo-CA(in YEplac181) and GroES-GroEL(in YEp352) transfered into competent yeast cells. The colonies were first cultured in YPD media and then cultured in media containing galactose as carbon source, during which the yeast underwent a fermentation process and samples were taken out at a certain time for analysis.
As shown below, at the beginning of the fermentation process, ethanol concentrations of contrast groups are higher than that of wild type cells（group 0）. With time going by, for group 0 and group Ⅰ ethanol concentrations increased first and then decreased. For group Ⅱ , on the contrary, ethanol concentration decreased first and then increased. 
</P></DIV><DIV class="mainbody" id="label_4"><P class="atop"><SPAN>Leading Peptide Testing</SPAN></P><P>We construct the BFP, YFP, and GFP fusing with the GBD, SH3 and PDZ ligand in order to check whether the ligands with targeted protein can bind to the domains of GBD, SH3 and PDZ as well as the scaffold protein can position on the outer mitochondria membrane or not. If it works, the fluorescent protein can gather in the same site in the cell.
</P><P style="margin-top:10px;"><B>Figure 7 | </B>This is the Co-localization of BFP, YFP and GFP in mitochondria. We made he fluorescent protein fusing with the GBD, SH3 and PDZ protein ligand so that they can bind to the domains of GBD, SH3 and PDZ protein(GBD:SH3:PDZ=1:4:3). Now we see that in picture 1, it's obviously that the BFP is much brighter in somewhere than other site. And the picture 2 is the bright filed of our cell. Similarly, there is no doubt that in pictrue 3 and 4, the GFP and YFP has been gathered in several places of the cell. Picture 5 is the same cell that we dye the mitochondria. In picture 6, we gather all the fluorescent protein and the dyed mitochondria in the same image. As shown in the picture, BFP ,GFP and YFP has been overlapped with each other and they gather in same place of the cell. So, In short, We can come to a conclusion that With the help of Tom22 , we've successfully construct the scaffold protein on the surface of outer mitochondrial 
membrane.
</P><P style="margin-top:10px;"><B>Figure 8 | </B> The same results come from cotransformation of fluorescence protein and scaffold proteins in different ratio (GBD:SH3:PDZ=1:4:3 and GBD:SH3:PDZ=1:2:3) shows successfully colocalizing.
</P></DIV><DIV class="mainbody" id="label_5"><P class="atop"><SPAN>References</SPAN></P><P>[1]   Hillel K. Brandes‡, Fred C. Hartman§¶, Tse-Yuan S. Lu§, et al. : <B>Efficient Expression of the Gene for Spinach Phosphoribulokinase in Pichia pastoris and Utilization of the Recombinant Enzyme to Explore the Role of Regulatory Cysteinyl Residues by Site-directed Mutagenesis*.</B> The Journal of Biological Chemistry 1996 Vol. 271, No. 11, Issue of March 15, pp. 6490–6496.
</P><P>
[2]   F. Robert Tabita, Sriram Satagopan, Thomas E. Hanson, et al. : <B>Distinct form I, II, III, and IV Rubisco proteins from the three kingdoms of life provide clues about Rubisco evolution and structure/function relationships.</B> Journal of Experimental Botany 2008 Vol. 59, No. 7, pp. 1515–1524.
</P><P>
[3]   Christopher R. Somerville and Shauna C. Somerville: <B>Cloning and Expression of the Rhodospirillum rubrum Ribulosebisphosphate Carboxylase Gene in E. coil.</B> Mol Gen Genet 1984 193:214-219
</P><P>
[4]   Víctor Guadalupe-Medina, H Wouter Wisselink, Marijke AH Luttik, et al. : <B>Carbon dioxide fixation by Calvin-Cycle enzymes improves ethanol yield in yeast.</B> Biotechnology for Biofuels 2013 6:125.
</P><P>
[5]   Walter Bonaccia, Poh K. Tengb, Bruno Afonsoa, et al. : <B>Modularity of a carbon-fixing protein organelle.</B> PNAS 2012 vol. 109 , no. 2: 478-483.
</P><P>
[6]   Jian Qiu, Lena-Sophie Wenz, Ralf M. Zerbes, et al. : <B>Coupling of Mitochondrial Import and Export Translocases by Receptor-Mediated Supercomplex Formation.</B> Cell 2013 154, 596–608.
</P><P>
[7]   Birgitta M. GEIER', IIermann SCHAGGER', Claus ORTWEIN&quot;, et al. : <B>Kinetic properties and ligand binding of the eleven-subunit cytochrome-c oxidase from Saccharomyces cerevisiae isolated.</B> Eur. J. Biochem. 227, 296-302 (1995). Eur. J. Biochem 1995 227, 296-302.
</P><P>
[8]   MA Jun, SUN Fei: <B>Translocation of Mitochondrial Proteins.</B> ACTA BIOPHYSICA SINICA 2010 Vol.26 No.10: 880-893
</P><P>
[9]   John E Dueber, Gabriel C Wu, G Reza Malmirchegini, et al. : <B>Synthetic protein scaffolds provide modular control.</B> NATURE BIOTECHNOLOGY 2009 Vol. 27 No. 8
</P><P>
[10]  Matthew C. Good, Jesse G. Zalatan, Wendell A. Lim† : <B>Scaffold Proteins: Hubs for Controlling the Flow of Cellular.</B> Science 2011 332, 680.
</P><P>
[11]  Tae SeokMoona, JohnE.Dueber, EricShiue, et al. : <B>Use of modular, synthetic scaffolds forim proved production of glucaricacid in engineered E. coli over metabolic flux.</B> Metabolic Engineering 2010 12 298–305. </P></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:SCUT/Project/System_Construction/Co2_Fixation">http://2014.igem.org/Team:SCUT/Project/System_Construction/Co2_Fixation</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:SCUT/Project/System_Construction/Co2_Fixation" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:SCUT/Project/System_Construction/Co2_Fixation" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:SCUT/Project/System_Construction/Co2_Fixation&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:SCUT/Project/System_Construction/Co2_Fixation&amp;oldid=403189" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></DIV></BODY></HTML>