"
Page
Discussion
View source
History
teams
Log in
 
Team:SCUT/Project/Other Work
From 2014.igem.org
HOME
TEAM
Members
Notebook
Attributions
Acknowledgements
Our University
PROJECT
Overview
System Construction
Analysis and Discussion
Other Work
MODELING
Overview
Carbon dioxide fixed simulation
n-butanol simulation
Tips for other teams
DATA PAGE
Biobricks
Achievements
Protocol
HUMAN PRACTICE
Timeline
Magazine
Survey and Data Analysis
SAFETY
Overview>>>
CO2 Fixation>>>
n-Butanol Prod>>>
Introduction
Simulation
Reference
TQPA
Leading Peptides
Testing and Quantification of Promoter Activities
Overview
This is an additional part of our project, designed to testing and quantify promoter activities for having moderate expression level of target protein, or more detailedly saying, for knowing if TEF2 and TDH3 promoter work and for better control of induction of Gal1 promoter, the only inducible promoter used in our project. We employed GFPmut3b ( BBa_K294055 ), one of the most widely used fluorescence protein, as a reporter. After making a comparison between FCM and ELIASA, we chose the latter one for quantification of fluorescence since it was more convenient to manipulate and was accurate enough for comparing promoter activities. Considering that the fermentation process will span a period of time from several hours to several days, we did not have timed induction of Gal1 promoter included in quantification process.
Design and Result
With employing GFPmut3b ( BBa_K294055 ) as a reporter protein, we designed three similar devices only differing in promoters as shown below.
Promoter Testing
Plasmids carrying the above-mentioned devices were transformed into competent S.cerevisiae cells. Transformed cells were plated on 2% agar in media A (yeast nitrogen base, MET, LYS, HIS, URA), complemented with 2% glucose and cultured at 30℃ for 48h. The colonies were then inoculated in the liquid media A complemented with 2% glucose and were grown at 30℃ overnight. For those contained TEF2  or TDH3 promoter, culture was stopped and samples were taken out for testing. For those contained Gal1 promoter, glucose was removed from media with washing cells by PBS till they reached the exponential phase. Cells were then pelleted and diluted and cultured on liquid medium containing galactose concentration of 2% w/v overnight. Samples were then taken out for testing. The result is shown below.
Figure 1 丨 The results show that all the promoters work.
Galactose Dose Response of Gal1 Promoter
Plasmid carrying the device containing Gal1 promoter was transformed into competent S.cerevisiae cells. Transformed cells were plated on 2% agar in media A (yeast nitrogen base, MET, LYS, HIS, URA), complemented with 2% glucose and cultured at 30℃ for 48h. The colonies were then inoculated in the liquid media A complemented with 2% glucose and were grown at 30℃. Culture was stopped and glucose was removed from media with washing cells by PBS till they reached the exponential phase. Cells were then pelleted and diluted and cultured on liquid medium containing galactose concentrations between 0.25% and 2% w/v overnight. Samples were taken out for quantification.
In quantification process, concentrations of liquid media containing cells were measured at λ600nm and then liquid media were removed and cells were washed twice and refolded to a OD600 = 1.0 concentration by Buffer ( 0.05 M NaH2PO4, 0.1 M NaCl, 0.5 M Imidazole ) for ELIASA analysis. Fluorescence intensity of the refolded cells were then measured by ELIASA (Tecan, Infinite M200 and i-Control)) with an excitation wavelength of 488 nm (the suggested one is 501 nm), and an emission filter of 520 nm (the suggested one is 511nm) to quantify fluorescence intensities. The result is shown below.
Figure 2 丨 Dose responsiveness of the Gal1 promoter to galactose induction
The graph above summarises the ELIASA data, and shows that the intensity of GFP expressing cells induced by low or high dose of galactose are higher then those induced by the middle doses. This is similar to the result given by iGEM 2010 team, Aberdeen_Scotland. We therefore plan to set more experiment groups between these doses to figure out what happened.
Leading Peptides for Future Plan
What we plan to do next is to find the other pathways in the yeast and make it highly effective by taking advantages of the micro environments. But frist of all, how to make the protein import into its own station and work smoothly?  Fortunately, we find some leading peptides to be our “guides”. As we know, different locations need different “guides”. This summer, we have constructed 6 leading peptides successfully in the yeast. In other words, we have 6 another possibility to take advantages of the micro environments.We chose those 8 membrane from a list of about ten candidates based on the following factor:1.It can be easily targeted in yeast.2.Less reaction over it is not allowed.3.Visible distinction from others.So far，we have found the following leading peptides.
1.Erv25p
We characterized a novel protein termed Erv25p that have benn discovered on ER-derived transport vesicles, and it is required for efficient ER to golgi transport. Erv25p is a single membrane spanning segment derived from yeast, and a 12-amino acid N-terminal sequence exposed to the cytoplasm.
Figure 3丨Direct targeting of proteins to ER丨The gene Erv25p( BBa_K1462940 ) was obtained from S. Cerevisiae genome by PCR technique.BFP was fused to its C-terminus as reporter.The biobrick is constructed as follow to verify this leading peptide.
2.CTS1-1A N-terminal 12-amino acid serve as a typical cleavable signal sequence. The targeting sequence derived from yeast can be located at cytoderm.
Figure 4 丨 Direct targeting of proteins to cytoderm丨We got this short targeting sequence( BBa_K1462930 ) by gene synthesis .GFP was fused to its C-terminus as reporter. The biobrick is constructed as follow to verify this leading peptide.
3.CIIC It’s a 12-amino acid C-terminal sequence separated from the Ras protein,which is the leading peptide of Plasma Membrane.
Figure 5 丨Direct targeting of proteins to Plasma Membrane丨CIIC( BBa_K1462850 ) target to We got this short targeting sequence by gene synthesis .CFP was fused to its N-terminus as reporter.The biobrick is constructed as follow to verify this leading peptide. iGEM12_Berkeley also use a same sequence. BBa_K900005
4.H2A2 The localizing protein(H2A2)( BBa_K1462870 ) is the major structural protein of chromosomes. And the C-terminal protein fusion to it will be located at nucleus.The gene H2A2 was obtained from S. Cerevisiae genome by PCR technique.CFP was fused to its C-terminus as reporter. The biobrick is constructed as follow to verify this leading peptide.
5.ZRC1 The localizing protein(ZRC1）( BBa_K1462860 )is a protein of vacuolar membrane zinc transporter.And a C-terminal protein fusion to it will be located at vacuolar membrance.The gene ZRC1 was obtained from S. Cerevisiae genome .YFP was fused to its C-terminus as reporter. The biobrick is constructed as follow to verify this leading peptide.
6.PTS There is a mechanism about cargo translocation in the peroxisomes, the matrix proteins are posttranslationally targeted to peroxisomes from the cytosol by peroxisomal targeting signals (PTSs). These signals include the predominantly used PTS1( BBa_K1462910 ) and the less prevalent PTS2, which are recognized by the soluble import receptors PEX5 and PEX7, respectively. We obtained this nine-bp-leading peptide by gene synthesis .GFP  was fused to its C-terminus as reporter. The biobrick is constructed as follow to verify this leading peptide.
References
[1]   Michael J. KurandaS and Phillips W. Robbins : Chitinase Is Required for Cell Separation during Growtho f Saccharomyces cerevisiae". The Journal of Biological Chemistry 1991 by The American Society for Biochemistry and Molecular Biology, Inc.
[2]   William J. Belden and Charles Barlowe J. Biol. Chem.: Reticulum to Golgi Transport That Is Required for Efficient Endoplasmic Vesicles, Forms a Complex with Emp24p Erv25p, a Component of COPII-coated. The Journal of Biological Chemistry 1996, 271:26939-26946.  doi: 10.1074/jbc.271.43.26939
[3]   Jennifer J. Smith and John D. Aitchison: Peroxisomes take shape NATURE REVIEWS 98109-5219, USA.  doi: 10.1074/jbc.271.43.26939
Retrieved from " http://2014.igem.org/Team:SCUT/Project/Other_Work "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
