<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Lund_Design_Experiments skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Lund/Design/Experiments</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar-collapse" aria-expanded="false"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse" id="navbar-collapse"><UL class="nav navbar-nav navbar-right"><LI id="li_home"><A href="/Team:Lund">Home</A></LI><LI id="li_team"><A href="/Team:Lund/People">People</A></LI><LI id="li_project" class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Project </A></LI><LI><A href="/Team:Lund/Description">Description</A></LI><LI><A href="/Team:Lund/Design">Design</A></LI><LI><A href="/Team:Lund/Applied_Design">Product Design</A></LI><LI><A href="/Team:Lund/Results">Results</A></LI><LI><A href="/Team:Lund/InterLab">InterLab</A></LI><LI><A href="/Team:Lund/Parts">Parts</A></LI><LI><A href="/Team:Lund/Experiments">Protocols</A></LI><LI><A href="/Team:Lund/Safety">Safety</A></LI><LI><A href="/Team:Lund/Notebook">Notebook</A></LI></UL><LI id="li_model"><A href="/Team:Lund/Model">Model</A></LI><LI id="li_hp" class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Practices </A></LI><LI><A href="/Team:Lund/Human_Practices">Integrated</A></LI><LI><A href="/Team:Lund/Public_Engagement">Engagement</A></LI></DIV></DIV><DIV id="design-page"><HEADER class="design-landing"><DIV class="container"><H1 class="design-landing-texts">Design</H1></DIV></HEADER><DIV class="container"><UL class="nav nav-pills nav-auto-stacked"><LI role="presentation"><A href="/Team:Lund/Design">Overview</A></LI><LI role="presentation"><A href="/Team:Lund/Design/Applications">Applications</A></LI><LI role="presentation"><A href="/Team:Lund/Design/Theory">Theory</A></LI><LI role="presentation"><A href="/Team:Lund/Design/Design">Design</A></LI><LI role="presentation" class="active"><A href="/Team:Lund/Design/Experiments">Experiments</A></LI></UL></DIV><DIV class="content-wrapper under-nav text-layout"><DIV class="container"><DIV class="row"><DIV class="col-lg-8"><SECTION><H3 class="section-heading">Overview</H3><P>
            In this section we present and discuss the experimental plan we set up in order to test our idea of co-expressing <I>Vitreoscilla</I> hemoglobin (VHb) to increase protein yields. We chose to perform all VHb-related experiments using the iGEM standard plasmid pSB1C3 and for the proof of concept we decided to use green fluorescent protein (GFP). The reasoning behind this choice was two-fold. First, the protein is an excellent reporter as it is easy to detect and measure. Secondly, the additional supply of oxygen provided by the hemoglobin should help the chromophore mature faster. This provides a second measure to quantify the presence and effect of the hemoglobin by studying the evolution of fluorescence intensity over time for the different constructs.
          </P></SECTION><SECTION><H3 class="section-heading">Plasmid construction - VHb &amp; GFP</H3><P>
            Initially, we planned to order both the VHb constructs and the GFP construct directly from Integrated DNA Technologies (IDT). However, as the latter contained multiple repeated sequences, it could not be synthesized and we therefore decided to instead use Bba_J364000 that was provided in the iGEM distribution kit.
          </P><P>   Constructs like the one in <I>fig. 1</I> were obtained by cloning the VHb biobrick, as a gBlock fragment, into pSB1C3. It was first attempted according to IDTs protocol for gBlocks. For this reason, all seven fragments were digested with XbaI and SpeI. However, due to lack of experience in handling gBlock fragments, the clonings did not succeed. After that, the protocol from iGEM Academy was tried, still with no success. In the meantime, our stock of gBlock fragments was starting to run out. Therefore, other strategies such as Gibson Assembly and standard restriction enzyme cloning became our main alternatives.

          </P><FIGURE><FIGCAPTION>Figure 1: VHb assembly in pSB1C3 by conventional digestion-ligation technique.</FIGCAPTION></FIGURE><P>
            Primers were designed to amplify the remaining biobrick stock and to create the overlaps required for Gibson assembly. Restriction enzyme cloning and Gibson assembly were applied in parallel to obtain successful results as fast as possible. As can be seen in <I>fig. 1</I> BBa_J04450, obtained from the iGEM distribution kit, was used as the backbone source after amplification in <I>E. coli</I> TG1. <I>Fig. 2</I> shows how primers were designed to create the overlapping sequences for Gibson assembly. The backbone fragment utilized was a linearized plasmid from the iGEM distribution kit. The backbone fragment, pSB1C3, contained the prefix and suffix, meanwhile the prefix and suffix of the VHb gBlock fragments had been digested as mentioned before.
          </P><FIGURE><FIGCAPTION>Figure 2: VHb cloning strategy through Gibson assembly</FIGCAPTION></FIGURE><P>
            Once the VHb biobricks were successfully cloned, the inserts were screened to ensure that the constructs were successfully ligated into the backbones. This was done through colony PCR, PCR amplification on purified plasmids and digestion with restriction enzymes. The results from these methods were analyzed by agarose gel electrophoresis and ultimately positive results were obtained for all constructs.
          </P><P>
            Once the VHb biobricks were confirmed to be successfully inserted, they needed to be combined with GFP for co-expression. 3A assembly was the first method of choice, performed according to  the protocol from the iGEM registry. However, with lack of results we modified the protocol slightly and proceeded with Gibson assembly and conventional restriction enzyme cloning as shown in <I>fig. 3</I>. Since the sticky ends created by XbaI and SpeI are very similar, GFP can be ligated into the VHb plasmid by cutting the suffix site of the plasmid with both rSpeI and PstI. Fragments were purified by gel extraction to increase the chances of successful ligation. In parallel, Gibson assembly was attempted. The design of the necessary overlapping fragments is shown in <I>fig. 4</I>.
          </P><FIGURE><FIGCAPTION>Figure 3: VHb-GFP assembly through conventional restriction enzyme cloning.</FIGCAPTION></FIGURE><P><I>Fig. 4</I> depicts how primers to obtain the overlapping fragments were designed. However, only one VHb-GFP construct (K2602026) out of six was successfully assembled through Gibson assembly.
          </P><FIGURE><FIGCAPTION>Figure 4: VHb cloning strategy using Gibson assembly.</FIGCAPTION></FIGURE><P>
            The final biobricks were characterized and confirmed via a combination of sequencing, colony PCR, PCR amplification, restriction enzyme digestion, as well as measuring GFP fluorescence and observing the red color originating from the hemoglobin in the cell pellets. During further analysis, as described further down on this page, biobricks were additionally confirmed by absorbance measurements and flow cytometry.
          </P></SECTION><SECTION><H3 class="section-heading">Batch cultivation</H3><P>
            Batch cultivations were carried out in erlenmeyer flasks under various conditions. Both Lysogeny broth (LB) and Terrific broth (TB) were used as means of growing the cells under varying nutritional conditions. The idea here was that while VHb enhances oxygen utilization, there might also be an interaction with respect to nutritional availability as an increased metabolic activity demands additional oxygen. Also one could make the opposite argument that low nutrient conditions puts further stress on the cells, which might cause VHb to either deliver oxygen or act directly as an antioxidant. Ultimately, by talking to experts within the field of biopharmaceutical production as a part of our <A href="/Team:Lund/Human_Practices">integrated practices</A>, we were taught that while LB is an excellent and cheap medium, TB is more commonly used within the industry as it generally yields higher cell densities. Obviously, the additional cost of having a more expensive medium is heavily outweighed by a higher yield of a valuable product.
          </P><P>
            Further conditions that were modified include the free headspace volume in the cultivation flasks as a way of restricting the oxygen availability. This was accomplished by simply changing the headspace to culture volume ratio and using flasks with or without baffles. By restricting the oxygen uptake, cell cultures expressing VHb should thrive compared to those without. The procedure was advised after a long <A href="/Team:Lund/Human_Practices">Skype meeting</A> with Prof. Benjamin C. Stark, an important figure within the development of VHb technology.

          </P><P>
            Finally, cells were also grown with 5-aminolevulinic acid (ALA) to further stimulate the synthesis of porphyrins. In addition, some cultures were flushed with carbon monoxide (CO), which is commonly done when expressing human hemoglobin as it promotes the assembly of dimers or tetramers [1].
          </P></SECTION><SECTION><H3 class="section-heading">Analysis of growth curves</H3><P>
            In order to compare the growth rates between cells expressing different amounts of hemoglobin, cultivation experiments were carried out with samples taken at regular time intervals. The optical density was immediately measured by spectrophotometry. In total three controls were used: cells with no plasmid, with plasmid containing only the ribosome binding site R0010 (“empty plasmid”), and with a plasmid expressing only GFP. The control with an empty plasmid was used to make a fair comparison as the burden associated with a plasmid is twofold. First, the cell has to express the target proteins from within the plasmid. Secondly, the additional genes within the plasmid, such as the ones accounting for antibiotic resistance, also need to be expressed.
          </P></SECTION><SECTION><H3 class="section-heading">Proof of concept: Production of GFP</H3><P>
            The principle behind our proof of concept experiment is simple: if VHb has an effect on the production of recombinant GFP, there should be an observable change in fluorescence intensity. The fluorescence intensity was measured by flow cytometry and absorbance was measured by spectrophotometry. The advantage of using a flow cytometer over a plate reader as in the InterLab study is that in flow cytometry, the fluorescence intensity is measured for each individual cell. This means that it is possible to determine whether effects of VHb on GFP production are caused by a change in overall cell density or a change within the individual cells. It also provides the additional advantage of giving much more exact information about the concentration of cells than what can be obtained by OD<SUB>600</SUB>. Whilst, through spectrophotometry and insight of its extinction coefficient, the concentration of not only GFP but also VHb produced after disruption of the cells can be determined at the same time and in more straightforward way. Moreover, the correlation between the VHb expression level and the GFP absorbance could be clearly seen.
          </P></SECTION><SECTION><H3 class="section-heading"> Expression of human hemoglobin mutants</H3><P>
            Based on our model, we had selected five different point mutations for the beta chain of adult human hemoglobin (HbA) which we wanted to express and study the oxygen affinity of. The wild-type DNA of HbA was acquired from our supervisor, Dr. Nélida Leiva Eriksson, in a pET-Duet plasmid.The beta chain mutants were ordered as gBlocks from IDT and amplified through PCR. The beta chain sequences in the plasmid was then replaced by the mutated one by digesting of both plasmid and gBlocks with NdeI and XhoI, agarose gel separation of plasmid and wild-type beta chain and ligation of plasmid and mutated beta chain. Four out of the five mutations were successfully cloned and transformed into <I>E. coli</I> BL21(DE3), example plasmid shown in <I>fig. 5</I>.
          </P><FIGURE><FIGCAPTION>Figure 5: Simplified drawing of the pET-Duet plasmid used for HbA mutations.</FIGCAPTION></FIGURE><P>
            The cultivations of HbA mutants were carried out in 2 L baffled flasks with 500 mL TB media. When cultures reached OD<SUB>620</SUB> ≥ 2.0, ALA and Isopropyl-Beta-D-Thiogalactoside (IPTG) were added to final concentrations of 0.3 mM and 0.1 mM respectively for induction of HbA expression. Cultures were also bubbled with CO for approximately 15 seconds.
          </P><P>
            As the aim of this study was to investigate the oxygen affinity of the HbA mutants, we planned to send our purified HbA mutants for oxygen affinity measurements. In addition, spectrophotometry and SDS-PAGE were used to confirm functional hemoglobins.
          </P></SECTION><SECTION><H3 class="section-heading">Case study: Production of protein A</H3><P>
            After screening the promoters using GFP, we wanted to select some of them for co-expression with a protein of industrial interest. As a part of our <A href="/Team:Lund/Human_Practices">integrated practices</A>, Protein A was recommended to us by Laila Sakhnini, industrial PhD student at NovoNordisk. She also provided us with the sequence for the Z domain of the protein, which is the domain that binds to the Fc region of antibodies. This property is what makes protein A an important protein within the pharmaceutical industry, where it is used for the purification of therapeutic monoclonal antibodies.
          </P><P>
            As protein A is native to <I>Staphylococcus aureus</I>, we first codon optimized the sequence for expression in <I>E. coli</I>. We then designed our construct in the same way as we did for VHb. The DNA sequence was purchased as a gBlock from IDT and inserted into pSB1C3 via conventional restriction enzyme cloning, as this strategy had proven most successful for us throughout the project. We amplified the gBlock through PCR and then divided it into two different reactions: one digestion with XbaI and SpeI to be ligated into pSB1C3, and one with XbaI and PstI to be ligated into pSB1C3 already containing VHb.
          </P><P>
            However, only the former of the two reactions succeeded. Therefore, we modified our strategy to that shown in <I>fig. 6</I>. pSB1C3 containing protein A was digested with both EcoRI and XbaI in its prefix. VHb was digested with EcoRI and SpeI and then ligated into the plasmid containing protein A. Finally, the insert was confirmed as described previously for VHb.
          </P><FIGURE><FIGCAPTION>Figure 6: VHb-Protein A assembly through conventional restriction enzyme cloning.</FIGCAPTION></FIGURE><P>
            With protein A, we needed a different method to determine the yield rather than flow cytometry. We decided to use SDS-PAGE for confirmation of expression and to analyze protein solubility.
          </P></SECTION><SECTION><H3 class="section-heading">Case study: Reactor upscaling</H3><P>
            The objective of the up-scaling was to evaluate how the VHb-GFP system works on a larger scale. This was done to better emulate industrial scenarios as recommended to us by our contacts in the <A href="/Team:Lund/Human_Practices">biopharmaceutical industry</A>. We wanted to compare the production of GFP with and without VHb co-expression, so for the first cultivation we used the VHB-GFP composite part BBa_K2602020, since it, based on previous results from our screening of the promoters, indicated the highest production of GFP in 80 % headspace. The second fermentation used the GFP composite part BBa_J364000 from the measurement kit.

          </P><P>
            The department of Pure and Applied Biochemistry at Lund university has substantial experience working with hemoglobin, mainly of human origin but also from <I>Vitreoscilla</I>. Therefore we decided to follow their fed-batch protocol for expression of adult human hemoglobin (HbA), which was provided to us by our supervisor, Dr. Nélida Leiva Eriksson. We cultivated the cells in 2 L medium based on Davis Minimal Broth for approximately 31 hours, feeding with glucose when needed. We monitored dissolved oxygen, base addition, pH and took regular samples to measure cell density and glucose concentration in the medium.
          </P></SECTION><DIV class="row"><DIV class="col-md-12"><H3 class="section-heading">References</H3><BUTTON id="refBtn" class="btn btn-default" type="button" data-toggle="collapse" data-target="#ref" aria-expanded="false" aria-controls="ref">
          Show
        </BUTTON><DIV class="collapse" id="ref"><OL class="references">
            [1] Hernan, R. A. and Sligar, S. G. (1995). Tetrameric Hemoglobin Expressed in <I>Escherichia coli</I>. The Journal of Biological Chemistry 270(44), 26257-26264.
          </OL></DIV></DIV></DIV></DIV></DIV></DIV><FOOTER class="footer"><SECTION class="site-map container"><DIV class="row"><DIV class="col-sm-3"><H5>Home</H5><UL><LI><A href="/Team:Lund">Home</A></LI></UL><H5>Team</H5><UL><LI><A href="/Team:Lund/Team">Team</A></LI><LI><A href="/Team:Lund/Collaborations">Collaborations</A></LI><LI><A href="/Team:Lund/Attributions">Attributions</A></LI></UL></DIV><DIV class="col-sm-3"><H5>Project</H5><UL><LI><A href="/Team:Lund/Description">Description</A></LI><LI><A href="/Team:Lund/Design">Design</A></LI><LI><A href="/Team:Lund/Results">Results</A></LI><LI><A href="/Team:Lund/InterLab">InterLab</A></LI><LI><A href="/Team:Lund/Parts">Parts</A></LI><LI><A href="/Team:Lund/Experiments">Protocol</A></LI><LI><A href="/Team:Lund/Safety">Safety</A></LI><LI><A href="/Team:Lund/Notebook">Notebook</A></LI></UL></DIV><DIV class="col-sm-3"><H5>Model</H5><UL><LI><A href="/Team:Lund/Model">Landing page</A></LI><LI><A href="/Team:Lund/Model/Hemoglobin">Hemoglobin</A></LI><LI><A href="/Team:Lund/Model/GrowthCurves">Growth curves</A></LI></UL></DIV><DIV class="col-sm-3"><H5>Practices</H5><UL><LI><A href="/Team:Lund/Human_Practices">Integrated</A></LI><LI><A href="/Team:Lund/Public_Engagement">Engagement</A></LI></UL></DIV></DIV></SECTION><SECTION class="container"><DIV class="row"><DIV class="contact col-sm-3"><H4>Contact</H4><A>Sölvegatan 22 A</A><A>223 62 Lund</A><A> Sweden</A><A href="mailto:igemlund@gmail.com">igemlund@gmail.com</A></DIV><DIV class="social-media col-sm-3 col-sm-offset-3"><H4>Stay connected</H4><UL><LI><A href="https://www.facebook.com/igemlund" target="_blank">
              Facebook
            </A></LI><LI><A href="https://www.instagram.com/igem_lund" target="_blank">
              Instagram
            </A></LI><LI><A href="https://twitter.com/iGEM_Lund" target="_blank">
              Twitter
            </A></LI></UL></DIV></DIV></SECTION><SECTION class="copyright container">©2018 iGEM Lund
  </SECTION></FOOTER></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>