Team:ECUST/Project
HOMEPAGE
Project
Description
Experiment
Demonstrate
HARDWARE
PARTS
MODEL
Human Practices
Human Practices
Integrated Practices
Public Engagement
Collaborations
InterLab
Lab
Team
Attributions
Notebook
Safety
Why do we focus on hydrogen?
With the rising of the world population and the higher average lifespan resulting from the development of science and technology, more energy is required to meet people’s growing demand. However, existing mineral resources on the earth, mainly known as coals, oil and gas, are limited and non-renewable. In order to solve this dilemma, we must find ways to obtain green energy resources with the assistance of modern technology to maintain sustainable development. Hydrogen energy, as a kind of green energy, is an efficient fuel and can be widely used in chemical synthesis, smelting process, etc. As a new energy, hydrogen energy development and utilization is still in its infancy.
How do we obtain hydrogen?
Now several methods are used for the production of hydrogen energy, including water electrolysis, coal gasification, heavy oil and natural gas water vapor catalytic conversion, etc. The relatively green method is water electrolysis, which means hydrogen can be produced by pyrolysis of water with the usage of light-resourced battery[1]. Additionally, we found that there are many photosynthetic bacteria in the biological world with the corresponding nitrogenase and hydrogenase which can use light in the appropriate conditions to produce hydrogen [2]. The main mechanism is that the nitrogenase can turn ATP, protons and electrons into H2, while ATPs come from the photosynthetic phosphorylation, part of the protons are from the tricarboxylic acid cycle, and other parts are from the ATP synthase. The electrons use organic acid (glucose, succinic acid, malic acid) as the donor[3].
Opportunities and challenges we face
The use of photosynthetic bacteria has a good application prospect. They use organic acids as electron donor to produce hydrogen, and the organic wastewater from such as food factories can be used during this process.
But during the process of light fermentation,there are two major problems we have to face.
First,  photoelectric conversion efficiency of photosynthetic bacteria are relatively low. Rhodobacter sphaeroides 2.4.1 use photosynthetic reaction center to absorb electrons and by photoion separation they produce electronics. The photochemical conversion efficiency is only 8.4% at 522 nm and 19% at 860 nm.
Second, a reasonable photobioreactor makes it possible to maximize the use of light energy by photosynthetic bacteria, but actually there's no photobioreactor that could meet the demand of Rhodobacter sphaeroides 2.4.1,  which is one of the challenges we face.
What are we doing?
Improve photoelectric conversion efficiency
We used Rhodobacter sphaeroides 2.4.1 as a chassis organism for this study. Rhodobacter sphaeroides 2.4.1, as a type of non-sulfur purple photosynthetic bacteria have their unique absorption spectrum. We hope to broaden its absorption spectrum to enhance its total photon absorption.
Using the principle of fluorescence resonance energy transfer, we fused the fluorescent protein sYFP2 (excitation wavelength 517 nm, emission wavelength 529 nm) to the H subunit of the RC complex of Rhodobacter sphaeroides 2.4.1 in order to enhance the ability of the bulb to absorb photons at 517 nm. At the same time we knocked out crtB to eliminate the effect of carotenoids on photon absorption covered by sYFP2.
The prediction of the RC complex and the sYFP2 distance by protein homology modeling shows that the method is potentially feasible.
To learn more about homology modeling ， please click here.
Design a new type of photobioreactor
Then, we designed a new type of photobioreactor, the idea of which is to combine the light source with the mixing system. We used the latest LED light technology, and gave full play to its merits, including small size, energy saving and environmental protection. With the help of brush on the top and other ancillary equipment, the light source can rotate with the paddles at the same time, which will not affect the flow field and the mixing effect of the photobioreactor.
Click here to see the video. "Weijie Chen said: 'Let there be light.'"
To learn more about our photobioreactor， please click here.
Reference:
[1]Blankenship R E, Sayre R T. Comparing photosynthetic and photovoltaic efficiencies and recognizing the potential for improvement.[J]. Science, 2011, 332(6031):805.
[2]Das D, Veziroǧlu T N. Hydrogen production by biological processes: a survey of literature[J]. International Journal of Hydrogen Energy, 2001, 26(1):13-28.
[3]邓文武. 外加电场辅助质子传递供类球红细菌光合产氢研究[D]. 西南大学, 2010.
[4]Miyake J. The science of biohydrogen. Biohydrogen [M]. Manoa: University of Hawaii, 1998.7-18
HOMEPAGE
Project
Description
Experiment
Demonstrate
HARDWARE
PARTS
MODEL
Human Practices
Human Practices
Integrated Practices
Public Engagement
Collaborations
InterLab
Lab
Team
Attributions
Notebook
Safety
Review
Through overall modelling of our project, we finally decided to adopt fluorescent protein sYFP2 as our Light-Harvester. By fusing sYFP2 to H subunit (one part of photosynthesis reaction center of Rhodobacter sphaeroides 2.4.1), Rhodobacter sphaeroides 2.4.1 would have an extra photon absorption at around 517 nm. From Figure 1, we discovered that Rhodobacter sphaeroides 2.4.1 had absorption peaks at 450-550nm (provided by carotenoids) and 800-900nm (provided by chlorophyll). Since the absorption peaks of carotenoids and sYFP2 may overlap to some extent, we needed to knock out gene crtB (phytoene synthase), making it unable for Rhodobacter sphaeroides 2.4.1 to synthesize carotenoids.
Figure 1. Room temperature absorption spectrum of membranes from wild type normalised to 590 nm
So we chose Rhodobacter sphaeroides 2.4.1 as our chassis, we constructed a plasmid for knocking out crtB based on pDM4 and another plasmid for fusing sYFP2 to H-subunit (the relative code gene of H subunit is puhA). After that, we constructed an inducible pIND4. Based on it, we built another plasmid for cytoplasmic expression of sYFP2. Then by conjugation or electro-transformation of three plasmids constructed above, we respectively got three kinds of recombinant strains we needed.Finally, we would carry out some subsequent determination including absorption spectrum, growth curve and hydrogen production.
Figure 2. Work flow of our project about experiment in wetlab
 Description
1. Knock in (sYFP2)/ Knock out(crtB)
Figure 3. Work flow of Knock in/out experiment
1.1 Gibson assembly
Since the gene fragments related to gene knock-out and knock-in involved assembly of three fragments, we adopted Gibson Assembly to assemble upstream fragment, downstream fragment and sYFP2 onto the plasmid.
Figure 4. Two devices used in Knock in and Knock out experiment.
1.1.1 Main Protocols
PCR From Genome or Plasmids
Materials:
5xl Gxl Buffer
Primer star Gxl（DNA polymerase）
dNTP
ddH2O
Rhodobacter sphaeroides 2.4.1 genome
plasmid backbone with sYFP2 gene
Forward/Reverse Primer：
crtBUF1
GAGCTCAGGTTACCCGCATGCAAGATCTATACACGTTCTATGCGCTCTCG
crtBUR1
AGGCCGACTGCAAGATCC
crtBUF2
GGATCTTGCAGTCGGCCT
crtBUR2
TGAGAACCTACATTCCGCGGCAAGCCTTT
crtBDF
CCGCGGAATGTAGGTTCTCATGAAGGTATACCGG
crtBDR
CCCTCGAGTACGCGTCACTAGTGGGGCCCTGATCAGGTAGGGCACGAACA
puhAYFPUF
GAGCTCAGGTTACCCGCATGCAAGATCTATACGGCCACAACAAGATCAAGC
puhAYFPUR
TGCTCACCATGGCGTATTCGGCCAGCATCG
YFPF
CGAATACGCCATGGTGAGCAAGGGCGA
YFPR
CATGCGGGGATCATTACTTGTACAGCTCGTC
puhAYFPDF
CAAGTAATGATCCCCGCATGGCGCGGCCC
puhAYFPDR
CCCTCGAGTACGCGTCACTAGTGGGGCCCTCATCCGCAGGGCGATGGTA
Method:
PCR program:
PCR System（50 μL）：
5xl Gxl Buffer
10 μL
dNTP
4 μL
Gxl polymerase
1 μL
Forward primer
2 μL
Reverse primer
2 μL
template
1 μL
Gibson assembly
Materials:
PCR fragment (concentration requirement> 100 ng / μL)
Linearized plasmids (concentration requirements> 100 ng / μL, can be prepared by digestion or reverse PCR)
1.33x ITA reagent
Method:
Add all kinds of PCR fragments and linear plasmids of the same mass to 1.33x ITA reagents (7.5 µL), making total volume 10 µL;
Keep the system at 50°C for 1 hour;
Transform constructed plasmids into E.coli-sm10 and select positive colony whit chloromycetin resistance.
PCR From Colony
Materials:
10xl Taq Buffer
Taq polymerase
dNTP
ddH2O
Forward/Reverse Primer：
pDM4-F
aacaagccagggatgtaacgc
pDM4-R
tccagtggcttctgtttcta
Method:
Select monoclonal strains from LB plate with chloromycetin resistance, and incubate cells in LB media for 2 hours;
Test the monoclonal strains by PCR;
PCR system（10μL）：
5xl Taq Buffer
2 µL
dNTP
0.8μL
Taq polymerase
0.2 µL
Forward primer
0.5 µL
Reverse primer
0.5 µL
template
1 µL
ddH2O
5 µL
PCR procedure
Examine the results by electrophoresis. If positive, incubate cells in 5 mL LB media with chloromycetin resistance overnight. Preserve the stains with 20% glycerol.
1.2 Conjugation
In order to knock out/in gene, we used a special  plasmid(pDM4). This plasmid contains gene Mob, replicator oriV and gene SacB .Gene mob can be used to facilitate the use of bonding method of transformation. Replicator oriV couldn't replicate in the globular bacteria (Erythrocytes abscess λπ factor ) , thus single clones could be selected by chloromycetin resistance of the plasmid . Finally , through the gene SacB on pDM4 , sucrose was screened for secondary recombination .
1.2.1 Main Protocols：
Conjugation
Materials:
Rhodobacter sphaeroides 2.4.1
E.coli-sm10
Joint film
LB medium
PBS buffer
Method:
E.coli-sm10, Rhodobacter sphaeroides 2.4.1 were incubated overnight for 12 h. Add the corresponding antibiotics to donor bacteria .
E.coli-sm10, Rhodobacter sphaeroides 2.4.1 secondary culture, add the appropriate antibiotic into the donor bacteria medium, shake to the logarithmic growth phase.
Take 1 mL from both E.coli-sm10, Rhodobacter sphaeroides 2.4.1 washed with PBS and mixed. Take 25 μL of mixture and drop in the dried adhesive film. Dry out and cultivate overnight.
Wash the bacteria moss on the joint film with 1 mL of medium or PBS and take 100 μL of that to coat plate. The results were observed after several hours of culture.
After one screening, it was induced with TSB containing 10% sucrose and then subjected to secondary screening on a plate containing 10% sucrose free of resistance.
Validate gene knock out and gene knock in by priming pairs on the genome.
2. Expression of sYFP2 in cytoplasm
Figure 5. Work flow of expression experiment.
We wanted to figure out whether sYFP2 (with its codons optimized) could be stably expressed in the cytoplasm of Rhodobacter sphaeroides 2.4.1 and took it as an experimental control group.
2.1 Reconstruction of PIND4
Prof. Gaoyi Tan provided us with a reproducible plasmid pIND4 in Rhodobacter sphaeroides 2.4.1. In order to make the expression controllable, we inserted the repressor protein LacIq and its corresponding operon into the plasmid by reconstructing the plasmid and transfered it into a lactose operon-inducible plasmid.
Figure 6. The devices used in improvement of pIND4
2.1.1 Main Protocols：
PCR From Plasmids
Materials:
5xl Gxl Buffer
Primer star Gxl（DNA polymerase）
dNTP
ddH2O
pIND4 plasmid
pMCS-eq plasmid
Forward/Reverse Primer：
F1
CTGGCCTTTTGCTCACATGTTCTTTCCTGCGTTATCCCCT
R1
CGAGCTCGAATTCGCGCGGCCCTGCATTAA
F2
GCCGCGCGAATTCGAGCTCGGTACCGACGT
R2
GGTGGTGAATGTGAAACCAGTAACGTTATACGAT
F3
CTGGTTTCACATTCACCACCCTGAATTGACTCT
R3
AGATCTGGATCCTCCCATGGTTAATTTCTCCTCTTTAATT
Method:
Mentioned above.
Gibson Assembly
Mentioned above. (see 1.1.1)
2.2 Clone optimization with unoptimized sYFP2
2.2.1 Use the jcat site for codon optimization
sYFP2（Part:BBa_K864100）
sYFP2（Part:BBa_K2308003)
ATGGTTAGCAAGGGCGAAGAACTTTTTACAGGCGTAGTACCGATCTTA GTTGAATTAGACGGCGACGTTAACGGTCATAAGTTTAGCGTGAGCGGTGAG GGTGAAGGTGACGCAACTTACGGCAAGCTGACCCTGAAGCTGATTTGCACG ACGGGTAAGCTGCCGGTCCCGTGGCCTACCCTGGTCACGACCTTGGGTTAT GGCGTTCAGTGTTTCGCGCGTTATCCGGACCACATGAAACAACACGATTTCT TTAAGAGCGCGATGCCAGAAGGCTATGTGCAGGAGCGTACGATCTTTTTCAA AGACGACGGTAACTACAAGACGCGTGCCGAAGTCAAATTCGAAGGCGACAC CCTGGTGAATCGCATTGAGCTGAAGGGTATTGATTTCAAAGAGGATGGCAAT ATCCTGGGTCACAAGCTGGAGTACAATTACAATTCCCACAACGTTTACATCA CCGCAGATAAACAGAAAAATGGCATCAAAGCGAATTTCAAAATCCGTCACAA CATTGAGGACGGTGGTGTTCAACTGGCGGATCATTACCAGCAAAACACCCC GATTGGTGACGGTCCGGTCCTGTTGCCGGATAACCATTATCTGTCTTACCAA AGCAAACTGAGCAAAGATCCGAACGAGAAGCGCGACCACATGGTGCTGCTG GAGTTTGTGACCGCTGCCGGTATTACCCTGGGTATGGATGAGCTGTATAAATGA
ATGGTTAGCAAGGGCGAAGAACTTTTTACAGGCGTAGTACCGATCTTAG TTGAATTAGACGGCGACGTTAACGGTCATAAGTTTAGCGTGAGCGGTGAGG GTGAAGGTGACGCAACTTACGGCAAGCTGACCCTGAAGCTGATTTGCACGA CGGGTAAGCTGCCGGTCCCGTGGCCTACCCTGGTCACGACCTTGGGTTAT GGCGTTCAGTGTTTCGCGCGTTATCCGGACCACATGAAACAACACGATTTC TTTAAGAGCGCGATGCCAGAAGGCTATGTGCAGGAGCGTACGATCTTTTTC AAAGACGACGGTAACTACAAGACGCGTGCCGAAGTCAAATTCGAAGGCGA CACCCTGGTGAATCGCATTGAGCTGAAGGGTATTGATTTCAAAGAGGATGG CAATATCCTGGGTCACAAGCTGGAGTACAATTACAATTCCCACAACGTTTAC ATCACCGCAGATAAACAGAAAAATGGCATCAAAGCGAATTTCAAAATCCGTC ACAACATTGAGGACGGTGGTGTTCAACTGGCGGATCATTACCAGCAAAACA CCCCGATTGGTGACGGTCCGGTCCTGTTGCCGGATAACCATTATCTGTCTT ACCAAAGCAAACTGAGCAAAGATCCGAACGAGAAGCGCGACCACATGGTG CTGCTGGAGTTTGTGACCGCTGCCGGTATTACCCTGGGTATGGATGAGCTG TATAAATGA
Figure 7. The devices used in expression of sYFP2
2.2.2 Main protocols：
Restriction Digestion
Materials:
Enzyme NcoI and BamHI
10xBuffer（NEB buffer 1.1 2.1 3.1 star）
PCR DNA
ddH2O
pIND4 plasmid DNA
Method:
Configuration of double enzyme digestion system：
Digestion System（50 μL）：
NcoI
1 μL
BamHI
1 μL
10 x Buffer
5 μL  (NEB's buffer score 1.1 2.1 3.1 star specific see enzyme instructions select the cutting efficiency and asterisk the best activity)
ddH2O
added as appropriate
DNA
1 μg
keep the system at 37℃ for 15min (specific reaction time see the instructions).
Inactivate restriction enzyme at 80℃ for 20 minutes.
Ligation
Materials:
T4 DNA ligase
10x buffer
Plasmid DNA
PCR DNA
ddH2O
Method:
According to the external source and carrier concentration ratio of 5: 1 configuration connection system 20μL
Ligation System（20 μL）：
10 x buffer
2 μL
T4 DNA ligase
1 μL
plasmid DNA
X μL
PCR DNA
Y μL
Incubate the mixture at 16℃ for 1 h
Pipet 10 μL solution from the connection mixture for transformation into E.coli-sm10
Electroporation Transformation
Materials:
10% glycerol
2mm Electroporation cup
Kanr plate
ddH2O
Method:
32℃ overnight culture and transfer, until the OD grows to about 2.5
10 mL of bacteria at 5000 rpm for 5 min to collect all the cells
2 mL sterile water pumped retrogradesum (washed away from the medium ion residue, centrifuged at 5000 rpm for 2 min)
200 μL 10% glycerol resuspend the bacteria
Mixing 100 μL of competent and 100ng plasmid
Pipetting 100 μL mixture to 2mm electric rotor shock with 1.5Kv (voltage index curve mode, capacitance 25uF, resistance 200Ω)
600 μL TSB mixed and moved into the EP tube
32℃, 220rpm recovery 2h
5000 rpm after centrifugation, remove the 400 μL supernatant, the remaining bacteria coated on kanr resistant plate
culture 2-3d observation results
3. Final performance
After obtaining three engineering bacteria, we first examined the presence of fluorescence with fluorescence microscope in order to determine whether sYFP2 is well expressed. Then by measuring the absorption spectrum, we checked whether there is an additional peak at 517 nm near the absorption peak. Finally, we carried out photosynthetic growth curve measurements and hydrogen production experiments to test for additional ATP or H2 production.
3.1 Spectrum or imaging analysis
3.1.1  Room temperature and 77K absorption spectrums were recorded using Clariostar-430-9903 in the spectral range between 300 and 900nm（5nm width）.
3.1.2  Fluorescence intensity were recorded using Clariostar-430-9903 with 497nm excitation wavelength and 540nm emission wavelength.
3.1.3  Fluorescence images were taken with an inverted ﬂuorescence microscope
3.2 Photosynthetic growth curve analysis
Cells were cultured in TSB medium. After overnight incubation, the OD700 was 0.02 and then the photosynthetic growth curve of 0-24 h was recorded. The cells were given sufficient oxygen. The light source was white LED lamp or green LED lamp (main wavelength For 520nm), the illuminance parameters measured by the illuminometer are shown below
Figure 8. The illuminance of LED bulb used in experiment
3.3 Hydrogen production analysis
Materials:
For 1 L fermentation media：
Succinic acid disodium salt
5.49 g
NaCl
0.4 g
MgS04·7H20
0.2 g
CaCl2·2H20
0.05 g
L-glutamate
5 mol
KH2PO4
1.0 g
K2HPO4 ·2H20
1.572 g
Yeast extract
1.0 g
Solution of trace element
1.0 mL
Solution of growth factor
1.0 mL
For 100 mL solution of trace element:：
MnSO4 · 4H2O
0.21 g
H3BO3
0.28 g
Cu(NO3)2 · 7H2O
0.004 g
ZnSO4 · 7H2O
0.024 g
Na2MoO4 · 2H2O
0.075 g
For 100 mL solution of growth factor:
biotin
0.01 g
thiamine HCl
0.5 g
nicotinic acid
1.0 g
Main protocol:
Rhodobacter sphaeroides 2.4.1 were cultured in TSB medium for 48h
Took 10 mL bacterial suspension and removed the clear supernatant extract after demission at 3500rpm for 5 minutes
Resuspended the bacteria in 10 mL fermentation medium and transferred them into a new conical flask
Added 90 mL fermentation medium into the flask with 100 μL solution of growth factor, 100 μL solution of trace element, 30ng/μL kanamycin and 100 μL IPTG (making the final concentration 800 μL)
Introduced argon to the flask through rubber tube for 5 min and wrapped the bottleneck with plastic membrane in case of gaseous escape
Pressed the bottleneck with rubber stopper and sealed it with parafilm
Cultured the bacteria in the flask which was exposed to 4,000lx under anaerobic condition with nutrient at 32℃
Collected the gas based on water gas displacing principle using pinhead through rubber stopper and the pinhead should be covered with Vaseline
Drew bacterial suspension with syringe and measured OD every 24 hours. After each measurement, the pinhead should be covered with Vaseline.
Figure 9. The pictures of hydrogen production
HOMEPAGE
Project
Description
Experiment
Demonstrate
HARDWARE
PARTS
MODEL
Human Practices
Human Practices
Integrated Practices
Public Engagement
Collaborations
InterLab
Lab
Team
Attributions
Notebook
Safety
