<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_ECUST_Exp skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:ECUST/Exp</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container-fluid"><DIV id="navbar" class="navbar-collapse collapse" style="font-family: 'Quicksand', sans-serif"><UL class="nav navbar-nav navbar-right"><LI><A class="dropdown-toggle active" href="https://2017.igem.org/Team:ECUST">HOMEPAGE </A></LI><LI class="dropdown"><A class="dropdown-toggle active" href="https://2017.igem.org/Team:ECUST/Description">Project </A></LI><LI><A href="https://2017.igem.org/Team:ECUST/Description">Description</A></LI><LI><A href="https://2017.igem.org/Team:ECUST/Experiment">Experiment</A></LI><LI><A href="https://2017.igem.org/Team:ECUST/Demonstrate">Demonstrate</A></LI></UL><LI><A class="dropdown-toggle active" href="https://2017.igem.org/Team:ECUST/Hardware">HARDWARE </A></LI><LI><A class="dropdown-toggle active" href="https://2017.igem.org/Team:ECUST/Parts">PARTS </A></LI><LI><A class="dropdown-toggle active" href="https://2017.igem.org/Team:ECUST/Model">MODEL </A></LI><LI class="dropdown"><A class="dropdown-toggle active" href="https://2017.igem.org/Team:ECUST/Human_Practices">Human Practices </A></LI><LI><A href="https://2017.igem.org/Team:ECUST/HP/Hp">Human Practices</A></LI><LI><A href="https://2017.igem.org/Team:ECUST/HP/Integrated">Integrated Practices</A></LI><LI><A href="https://2017.igem.org/Team:ECUST/Engagement">Public Engagement</A></LI><LI><A href="https://2017.igem.org/Team:ECUST/HP/Collaboration">Collaborations</A></LI><LI><A class="dropdown-toggle active" href="https://2017.igem.org/Team:ECUST/InterLab">InterLab</A></LI><LI class="dropdown"><A class="dropdown-toggle active" href="https://2017.igem.org/Team:ECUST/Lab/Team">Lab </A></LI><LI><A href="https://2017.igem.org/Team:ECUST/Lab/Team">Team</A></LI><LI><A href="https://2017.igem.org/Team:ECUST/Attributions">Attributions</A></LI><LI><A href="https://2017.igem.org/Team:ECUST/Lab/Notebook">Notebook</A></LI><LI><A href="https://2017.igem.org/Team:ECUST/Lab/Safety">Safety</A></LI></DIV></DIV><DIV class="col-lg-10 col-centered" style="margin-left: 10%;"><DIV class="row"><H2>Review</H2></DIV><DIV class="row style1"><P>Through <A href="https://2017.igem.org/Team:ECUST/Model">overall modelling</A> of our project, we finally decided to adopted fluorescent protein SYPF2 as our Light-Harvester. By the expression of SYPF2 and H-subunit (from Rhodobacter Sphaeroides’ reaction center) fusion protein, Rhodobacter Sphaeroides have an extra photon absorption at around 514nm, and we can discover from Figure 1. that Rhodobacter Sphaeroides have absorption peaks at 450-550nm (provided by carotenoids) and 800-900nm (provided by chlorophyll). Since the absorption peaks of carotenoids and SYPF2 may overlap to some extent, we need to knock out crtB (phytoene synthase), making it unable for Rhodobacter Sphaeroides to synthesize carotenoids.</P></DIV><CENTER><DIV style="font-size:10px;"><P>Figure 1：Room temperature absorption spectra of membranes from wild type normalised to 590 nm</P></DIV></CENTER><DIV class="row style1"><P>So we chose Rhodobacter Sphaeroides 2.4.1 as our chassis, and after codon optimization of SYPF2, we constructed a plasmid for knocking out crtB based on pDM4 and another plasmid for fusing SYPF2 and H-subunit (the relative code gene of H-subunit is puhA). After that, we constructed an inducible expression pIND4 and another plasmid for cytoplasmic expression of SYPF2 based on that. Then we got three kinds of recombinant strains we need by conjugation or electro-transformation.Finally, we would carry out some subsequent determination including absorption spectrum, growth curve and hydrogen production.</P></DIV><CENTER><DIV style="font-size:10px;"><P>Figure 2：Work flow of our project about experiment in wetlab</P></DIV></CENTER><DIV class="row style1"><P>Finally, we will characterize our engineering bacteria, including absorption spectra, photosynthetic growth curves and hydrogen production experiments.</P></DIV><DIV class="row"><SPECIALH2><STRONG> Description</STRONG></SPECIALH2><H3>1. Knock in (SYPF2)/ Knock out(crtB)</H3><CENTER><DIV style="font-size:10px;"><P>Figure 3：Work flow of Knock in/out experiment</P></DIV></CENTER><H4>1.1 Gibson assembly</H4></DIV><DIV class="row style1"><P>Since the gene fragments related to gene knock-out and knock-in involves assembly of three fragments, we adopted Gibson Assembly to assemble upstream fragment, downstream fragment and SYPF2 onto the plasmid.</P></DIV><CENTER><DIV style="font-size:10px;"><P>Figure 4：Two devices used in Knock in and Knock out experiment.</P></DIV></CENTER><H5>1.1.1 Main Protocols</H5><DIV class="panel-group" id="accordion" role="tablist" aria-multiselectable="true"><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A role="button" data-toggle="collapse" data-parent="#accordion" href="#collapse1" aria-expanded="false" aria-controls="signup-collapse"><DIV><DIV class="col-md-11"> PCR From Genome or Plasmids</DIV></DIV></A></H4></DIV><DIV id="collapse1" class="panel-collapse collapse" role="tabpanel" aria-labelledby="general"><DIV class="panel-body"><P><B>Materials:</B></P><P>
                5xl Gxl Buffer    
                Primer star Gxl（DNA polymerase）
                dNTP
                ddH<SUB>2</SUB>O
                Rhodobacter Sphaeroides 2.4.1 genome
                plasmid backbone with sYFP2 gene</P><P>Forward/Reverse Primer：</P><DIV class="bs-docs-section"><DIV class="row"><DIV class="col-md-12"><DIV class="bs-example table-responsive"><TABLE class="table table-striped table-bordered table-hover"><TBODY><TR><TD>crtBUF1</TD><TD>GAGCTCAGGTTACCCGCATGCAAGATCTATACACGTTCTATGCGCTCTCG</TD></TR><TR><TD>crtBUR1</TD><TD>AGGCCGACTGCAAGATCC</TD></TR><TR><TD>crtBUF2</TD><TD>GGATCTTGCAGTCGGCCT</TD></TR><TR><TD>crtBUR2</TD><TD>TGAGAACCTACATTCCGCGGCAAGCCTTT</TD></TR><TR><TD>crtBDF</TD><TD>CCGCGGAATGTAGGTTCTCATGAAGGTATACCGG</TD></TR><TR><TD>crtBDR</TD><TD>CCCTCGAGTACGCGTCACTAGTGGGGCCCTGATCAGGTAGGGCACGAACA</TD></TR><TR><TD>puhAYFPUF</TD><TD>GAGCTCAGGTTACCCGCATGCAAGATCTATACGGCCACAACAAGATCAAGC</TD></TR><TR><TD>puhAYFPUR</TD><TD>TGCTCACCATGGCGTATTCGGCCAGCATCG</TD></TR><TR><TD>YFPF</TD><TD>CGAATACGCCATGGTGAGCAAGGGCGA</TD></TR><TR><TD>YFPR</TD><TD>CATGCGGGGATCATTACTTGTACAGCTCGTC</TD></TR><TR><TD>puhAYFPDF</TD><TD>CAAGTAATGATCCCCGCATGGCGCGGCCC</TD></TR><TR><TD>puhAYFPDR</TD><TD>CCCTCGAGTACGCGTCACTAGTGGGGCCCTCATCCGCAGGGCGATGGTA</TD></TR></TBODY></TABLE></DIV></DIV></DIV></DIV><P><B>Method:</B></P><P>PCR program:</P><P><B>PCR System（50ul）：</B></P><DIV class="bs-docs-section"><DIV class="row"><DIV class="col-md-12"><DIV class="bs-example table-responsive"><TABLE class="table table-striped table-bordered table-hover"><TBODY><TR><TD>5xl Gxl Buffer</TD><TD>10 ul</TD></TR><TR><TD>dNTP</TD><TD>4 ul</TD></TR><TR><TD>Gxl polymerase</TD><TD>1 ul</TD></TR><TR><TD>Forward primer</TD><TD>2 ul</TD></TR><TR><TD>Reverse primer</TD><TD>2 ul</TD></TR><TR><TD>template</TD><TD>1 ul</TD></TR></TBODY></TABLE></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="panel-group" id="accordion"><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A role="button" data-toggle="collapse" data-parent="#accordion" href="#collapse2" aria-expanded="false" aria-controls="collapse2"><DIV><DIV class="col-md-11"> Gibson assembly</DIV></DIV></A></H4></DIV><DIV id="collapse2" class="panel-collapse collapse" role="tabpanel" aria-labelledby="general"><DIV class="panel-body"><P><B>Materials:</B></P><P>
                PCR fragment (concentration requirement&gt; 100 ng / ul)    
                Linearized plasmids (concentration requirements&gt; 100 ng / ul, can be prepared by digestion or reverse PCR)
                1.33x ITA reagent</P><P><B>Method:</B></P><OL style="font-size:16px;"><LI>Add all kinds of PCR fragments and linear plasmids of the same mass to 1.33x ITA reagents (7.5 µL), making total volume 10 µL</LI><LI>Keep the system at 50°C for 1h</LI><LI>Transform 10 µL ligation liquid and select using chl resistance</LI></OL></DIV></DIV></DIV></DIV><DIV class="panel-group" id="accordion"><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A role="button" data-toggle="collapse" data-parent="#accordion" href="#collapse3" aria-expanded="false" aria-controls="collapse3"><DIV><DIV class="col-md-11"> PCR From Colony</DIV></DIV></A></H4></DIV><DIV id="collapse3" class="panel-collapse collapse "><DIV class="panel-body"><P><B>Materials:</B></P><P>
                10xl Taq Buffer     
                Taq polymerase
                dNTP
                ddH<SUB>2</SUB>O</P><P>Forward/Reverse Primer：</P><DIV class="bs-docs-section"><DIV class="row"><DIV class="col-md-12"><DIV class="bs-example table-responsive"><TABLE class="table table-striped table-bordered table-hover"><TBODY><TR><TD>pDM4-F</TD><TD style="text-transform:uppercase;">aacaagccagggatgtaacgc</TD></TR><TR><TD>pDM4-R</TD><TD style="text-transform:uppercase;">tccagtggcttctgtttcta</TD></TR></TBODY></TABLE></DIV></DIV></DIV></DIV><P><B>Method:</B></P><OL style="font-size:16px;"><LI>Select monoclonal strains to LB plate with chl resistance, and carry out PCR after culture for 2h</LI><LI>PCR</LI><DIV id="myTabContent" class="tab-content"><DIV class="tab-pane fade" id="Method"><P>PCR system（10ul）：</P><TABLE class="table table-striped table-bordered table-hover"><TBODY><TR class="warning"><TD>5xl Taq Buffer</TD><TD>2 µL</TD></TR><TR class="warning"><TD>dNTP</TD><TD>0.8ul µL</TD></TR><TR class="warning"><TD>Taq polymerase</TD><TD>0.2 µL</TD></TR><TR class="warning"><TD>Forward primer</TD><TD>0.5 µL</TD></TR><TR class="warning"><TD>Reverse primer</TD><TD>0.5 µL</TD></TR><TR class="warning"><TD>template</TD><TD>1 µL</TD></TR><TR class="warning"><TD>ddH<SUB>2</SUB>O</TD><TD>5 µL</TD></TR></TBODY></TABLE><P>PCR procedure</P></DIV></DIV><LI>Examine the results using electrophoresis. If positive, inoculate 5mL to LB plat with chl resistance for overnight culture. Preserve the stains with 15% glycerol.</LI></OL></DIV></DIV></DIV></DIV><H4>1.2 Conjugation</H4><DIV class="row style1"><P>In order to achieve gene knockout / knocking, we use a special pDM4 plasmid, the plasmid has a mob element, which can be used to facilitate the use of bonding method of transformation, in addition to pDM4 on the replicator oriV, so within the globular bacteria can not be copied (Erythrocytes abscess λπ factor), can be re-recombined by chl resistance on the plasmid. Finally, through the SacB gene on pDM4, sucrose was screened for secondary recombination.</P></DIV><H5>1.2.1 Main Protocols：</H5><DIV class="panel-group" id="accordion"><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A role="button" data-toggle="collapse" data-parent="#accordion" href="#collapse4" aria-expanded="false" aria-controls="collapse4"><DIV><DIV class="col-md-11"> Conjugation</DIV></DIV></A></H4></DIV><DIV id="collapse4" class="panel-collapse collapse" role="tabpanel" aria-labelledby="general"><DIV class="panel-body"><P><B>Materials:</B></P><P>
                Rhodobacter Sphaeroides
				Ecoli-sm10
				Joint film
				LB medium
				PBS buffer</P><P><B>Method:</B></P><OL style="font-size:16px;"><LI>Ecoli-sm10, Rhodobacter Sphaeroides were incubated overnight for 12 h. Add the corresponding antibiotics to donor bacteria .</LI><LI> Ecoli-sm10, Rhodobacter Sphaeroides secondary culture, add the appropriate antibiotic into the donor bacteria medium, shake to the logarithmic growth phase.</LI><LI> Take Ecoli-sm10, Rhodobacter Sphaeroides, each 1ml wash with PBS and mix them, take 25μl drop in the dried adhesive film, dry. Cultivate overnight.</LI><LI> Wash the bacteria moss on the joint membrane with 1 ml of medium or PBS and take 100 μl of the coated plate. The results were observed after several hours of culture.</LI><LI>After one screening, it was induced with TSB containing 10% sucrose and then subjected to secondary screening on a plate containing 10% sucrose free of resistance.</LI><LI>Validate gene knockout and gene knocking by priming pairs on the genome.</LI></OL></DIV></DIV></DIV></DIV><H3>2. Expression of sYFP2 in cytoplasm</H3><CENTER><DIV style="font-size:10px;"><P>Figure 5：Work flow of expression experiment.</P></DIV></CENTER><DIV class="row style1"><P>As an experimental control group, and whether the optimized post-sYFP2 can be expressed in the cytoplasmic case (as a prerequisite for fusion expression), we hope to be able to express sYFP2 stably in the cytoplasm of Rhodobacter Sphaeroides.</P></DIV><H4>2.1 Reconstruction of PIND4</H4><DIV class="row style1"><P>Prof. Gaoyi Tan provided us with a copy plasmid pIND4 in Rhodobacter Sphaeroides, but it was a constant expression (without induction). To make the expression controllable, we inserted the repressor protein LacIq and its corresponding operon into the plasmid by reconstructing the plasmid , transfer it into a lactose operon-inducible plasmid.</P></DIV><CENTER><DIV style="font-size:10px;"><P>Figure 6：The devices used in improvement of pIND4</P></DIV></CENTER><H5>2.1.1 Main Protocols：</H5><DIV class="panel-group" id="accordion" role="tablist" aria-multiselectable="true"><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A role="button" data-toggle="collapse" data-parent="#accordion" href="#collapse5" aria-expanded="false" aria-controls="collapse5"><DIV><DIV class="col-md-11">PCR From Plasmids</DIV></DIV></A></H4></DIV><DIV id="collapse5" class="panel-collapse collapse" role="tabpanel" aria-labelledby="general"><DIV class="panel-body"><P><B>Materials:</B></P><P>
                5xl Gxl Buffer    
                Primer star Gxl（DNA polymerase）
                dNTP
                ddH<SUB>2</SUB>O
                pIND4 plasmid
                pMCS-eq plasmid</P><P>Forward/Reverse Primer：</P><DIV class="bs-docs-section"><DIV class="row"><DIV class="col-md-12"><DIV class="bs-example table-responsive"><TABLE class="table table-striped table-bordered table-hover"><TBODY><TR><TD>F1</TD><TD>CTGGCCTTTTGCTCACATGTTCTTTCCTGCGTTATCCCCT</TD></TR><TR><TD>R1</TD><TD>CGAGCTCGAATTCGCGCGGCCCTGCATTAA</TD></TR><TR><TD>F2</TD><TD>GCCGCGCGAATTCGAGCTCGGTACCGACGT</TD></TR><TR><TD>R2</TD><TD>GGTGGTGAATGTGAAACCAGTAACGTTATACGAT</TD></TR><TR><TD>F3</TD><TD>CTGGTTTCACATTCACCACCCTGAATTGACTCT</TD></TR><TR><TD>R3</TD><TD>AGATCTGGATCCTCCCATGGTTAATTTCTCCTCTTTAATT</TD></TR></TBODY></TABLE></DIV></DIV></DIV></DIV><P><B>Method:</B></P><P>Mentioned above.</P></DIV></DIV></DIV></DIV><DIV class="panel-group" id="accordion" role="tablist" aria-multiselectable="true"><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A role="button" data-toggle="collapse" data-parent="#accordion" href="#collapse6" aria-expanded="false" aria-controls="collapse6"><DIV><DIV class="col-md-11">PCR From Plasmids</DIV></DIV></A></H4></DIV><DIV id="collapse6" class="panel-collapse collapse" role="tabpanel" aria-labelledby="general"><DIV class="panel-body"><P>Mentioned above.</P></DIV></DIV></DIV></DIV><H4>2.2 Clone optimization with unoptimized sYFP2</H4><H5>2.2.1 Use the jcat site for codon optimization</H5><CENTER><DIV style="font-size:10px;"><P>Figure 7：The devices used in expression of sYFP2</P></DIV></CENTER><H5>2.2.2 Main protocols：</H5><DIV class="panel-group" id="accordion" role="tablist" aria-multiselectable="true"><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A role="button" data-toggle="collapse" data-parent="#accordion" href="#collapse7" aria-expanded="false" aria-controls="collapse7"><DIV><DIV class="col-md-11">PCR From Plasmids</DIV></DIV></A></H4></DIV><DIV id="collapse7" class="panel-collapse collapse" role="tabpanel" aria-labelledby="general"><DIV class="panel-body"><P><B>Materials:</B></P><P>
                Enzyme NcoI and BamHI    
                10xBuffer（NEB buffer 1.1 2.1 3.1 star）
                PCR DNA
                ddH<SUB>2</SUB>O
                pIND4 plasmid DNA</P><P><B>Method:</B></P><OL style="font-size:16px;"><LI>Configuration of double enzyme digestion system(50ul)：</LI><P>
						NcoI   1ul
						BamHI   1ul
						10xBuffer 5ul (NEB's buffer score 1.1 2.1 3.1 star specific see enzyme instructions select the cutting efficiency and asterisk the best activity)
						ddH<SUB>2</SUB>O is added as appropriate
						DNA 1ug</P><LI>the reaction temperature of 37 ℃ 15min (specific reaction time see the instructions).</LI><LI>After the end of the reaction 80 ℃ 20min inactivation.</LI></OL></DIV></DIV></DIV></DIV><DIV class="panel-group" id="accordion" role="tablist" aria-multiselectable="true"><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A role="button" data-toggle="collapse" data-parent="#accordion" href="#collapse8" aria-expanded="false" aria-controls="collapse8"><DIV><DIV class="col-md-11">Ligation</DIV></DIV></A></H4></DIV><DIV id="collapse8" class="panel-collapse collapse" role="tabpanel" aria-labelledby="general"><DIV class="panel-body"><P><B>Materials:</B></P><P>
				T4 DNA ligase
				10x buffer
				Plasmid DNA
				PCR DNA
				ddH<SUB>2</SUB>O</P><P><B>Method:</B></P><OL style="font-size:16px;"><LI>According to the external source and carrier concentration ratio of 5: 1 configuration connection system 20ul</LI><P>
						system 20ul
						10xbuffer 2ul
						T4 DNA ligase 1ul
						plasmid DNA Xul
						PCR DNA Yul</P><LI>16 ℃for 1 h</LI><LI>The connection solution to take 10ul to transform, with kanr resistance screening positive bacteria</LI></OL></DIV></DIV></DIV></DIV><DIV class="panel-group" id="accordion" role="tablist" aria-multiselectable="true"><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A role="button" data-toggle="collapse" data-parent="#accordion" href="#collapse9" aria-expanded="false" aria-controls="collapse9"><DIV><DIV class="col-md-11">Electroporation Transformation</DIV></DIV></A></H4></DIV><DIV id="collapse9" class="panel-collapse collapse" role="tabpanel" aria-labelledby="general"><DIV class="panel-body"><P><B>Materials:</B></P><P>
				10% glycerol
				2mm Electroporation cup
				Kanr plate
				ddH<SUB>2</SUB>O</P><P><B>Method:</B></P><OL style="font-size:16px;"><LI>32℃ overnight culture and transfer, until the OD grows to about 2.5</LI><LI>10 ml of bacteria at 5000 rpm for 5 min to collect all the cells</LI><LI>2ml sterile water pumped retrogradesum (washed away from the medium ion residue, centrifuged at 5000 rpm for 2 min)</LI><LI>with 200ul 10% glycerol resuspend the bacteria</LI><LI>100ul of competent and 100ng plasmid mixed</LI><LI>by adding 2mm electric rotor 1.5Kv shock (voltage index curve mode, capacitance 25uF, resistance 200Ω)</LI><LI>600 ulTSB mixed and moved into the EP tube</LI><LI>32℃, 220rpm recovery 2h</LI><LI>5000 rpm after centrifugation, remove the 400ul supernatant, the remaining bacteria coated on kanr resistant plate</LI><LI>culture 2-3d observation results</LI></OL></DIV></DIV></DIV></DIV><H3>3. Final performance</H3><DIV class=" row style1"><P>After obtaining three engineering bacteria, we first examined whether the presence of fluorescence in the fluorescence microscope to determine whether the success of the gene, and through the absorption spectrum to check whether there is an additional 514nm near the absorption peak. Finally, we performed photosynthetic growth curve measurements and hydrogen production experiments to test for additional ATP or H2 production.</P></DIV><H4>3.1Spectra or imaging analysis</H4><H5>3.1.1  Room temperature and 77K absorption spectra were recorded using Clariostar-430-9903 in the spectral range between 300 and 900nm（5nm width）.</H5><H5>3.1.2  Fluorescence intensity were recorded using Clariostar-430-9903 with 497nm excitation wavelength and 540nm emission wavelength.</H5><H5>3.1.3  Fluorescence images were taken with an inverted ﬂuorescence microscope</H5><H4>3.2Photosynthetic growth curve analysis</H4><DIV class="row style1"><P>Cells were cultured in TSB medium. After overnight incubation, the OD<SUB>700</SUB> was 0.02 and then the photosynthetic growth curve of 0-24 h was recorded. The cells were given sufficient oxygen. The light source was white LED lamp or green LED lamp (main wavelength For 520nm), the illuminance parameters measured by the illuminometer are shown below</P></DIV><CENTER><DIV style="font-size:10px;"><P>Figure 8：The illuminance of LED bulb used in experiment</P></DIV></CENTER><H4>3.3Hydrogen production analysis</H4><DIV class="panel-group" id="accordion" role="tablist" aria-multiselectable="true"><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A role="button" data-toggle="collapse" data-parent="#accordion" href="#collapse10" aria-expanded="false" aria-controls="collapse10"><DIV><DIV class="col-md-11">Hydrogen production analysis</DIV></DIV></A></H4></DIV><DIV id="collapse10" class="panel-collapse collapse" role="tabpanel" aria-labelledby="general"><DIV class="panel-body"><P><B>Materials:</B></P><P>
				Media for fermentation ：<B>For 1L media:</B>
				Succinic acid disodium salt   5.49g
				NaCl   0.4g
				MgS04·7H20   0.2g
				CaCL2·2H20   0.05g
				L-glutamate 5mol
				KH2PO4   1.0g
				K2HPO4 ·2H20   1.572g
				Yeast extract fermentation   1.0g
				Solution of trace element    1.0ml
				Solution of growth factor     1.0ml
				For 100ml solution of trace element:
				0.21 g  MnSO4 · 4H2O
				0.28 g  H3BO3
				0.004 g  Cu(NO3)2 · 7H2O,
				0.024 g  ZnSO4 · 7H2O,
				0.075 g  Na2MoO4 · 2H2O<B>For 100ml solution of growth factor:</B>
				0.01g biotin
				0.5g thiamine HCl
				1.0g nicotinic acid</P><P><B>Main protocol:</B></P><OL style="font-size:16px;"><LI>Rhodobacter Sphaeroides were cultured in TSB medium for 48h</LI><LI>Took 10ml bacterial suspension and removed the clear supernatant extract after demission at 3500rpm for 5 minutes</LI><LI>Resuspended the bacteria in 10ml fermentation medium and transferred them into a new conical flask</LI><LI>Added 90ml fermentation medium into the flask with 100μL solution of growth factor, 100μL solution of trace element, 30ng/μL kanamycin and 100μL IPTG (making the final concentration 800μL) </LI><LI>Introduced argon to the flask through rubber tube for 5 min and wrapped the bottleneck with plastic membrane in case of gaseous escape</LI><LI>Pressed the bottleneck with rubber stopper and sealed it with parafilm</LI><LI>Cultured the bacteria in the flask which was exposed to 4,000lx under anaerobic condition with nutrient at 32℃ </LI><LI>Collected the gas based on water gas displacing principle using pinhead through rubber stopper and the pinhead should be covered with Vaseline</LI><LI>Drew bacterial suspension with syringe and measured OD every 24 hours. After each measurement, the pinhead should be covered with Vaseline.</LI></OL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>