Team:ECUST/Project/Experiment
<partinfo>BBa_K2308003 short</partinfo>
This part is a coding gene of sYFP2 (super yellow fluorescent protein), and the codes are optimized for expression in Rhodobacter Sphaeroides.
This part is an improvement of part (<partinfo>BBa_K864100</partinfo>)
Sequence and Features <partinfo>BBa_K2308003 SequenceAndFeatures</partinfo>
<partinfo>BBa_K2308003 parameters</partinfo>
sYFP2(BBa_K2308003) is the key part in our experiment. It is used both for fusion expression (see BBa_K2308002) and inducible cytoplasm expression (combined with BBa_K2308016), and the host cell is Rhodobacter sphaeroides 2.4.1.
Figure 1：PCR test of BBa_K2308003(optimized sYFP2) and original sYFP2
In inducible cytoplasm expression experiment, we used part BBa_K2308016 to turn the Rhodobacter sphaedoides 2.4.1 into inducible strains, and IPTG was added （800 μM in final volume）when the OD700 of the strain was about 0.4（grown for about 24h）.
Figure 2：fluorescent image of sYFP2 (original) and sYFP2(optimized)
Figure 3：Fluorescence intensity of WT, sYFP2(optimized), sYFP2(unoptimized) strains.
It is obvious that after the codon optimization, sYFP2 can be better expressed in Rhodobacter sphaeroides 2.4.1.
Review
Through overall modelling of our project, we finally decided to adopted fluorescent protein SYPF2 as our Light-Harvester. By the expression of SYPF2 and H-subunit (from Rhodobacter Sphaeroides’ reaction center) fusion protein, Rhodobacter Sphaeroides have an extra photon absorption at around 514nm, and we can discover from Figure 1. that Rhodobacter Sphaeroides have absorption peaks at 450-550nm (provided by carotenoids) and 800-900nm (provided by chlorophyll). Since the absorption peaks of carotenoids and SYPF2 may overlap to some extent, we need to knock out crtB (phytoene synthase), making it unable for Rhodobacter Sphaeroides to synthesize carotenoids.
Figure 1：Room temperature absorption spectra of membranes from wild type normalised to 590 nm
So we chose Rhodobacter Sphaeroides 2.4.1 as our chassis, and after codon optimization of SYPF2, we constructed a plasmid for knocking out crtB based on pDM4 and another plasmid for fusing SYPF2 and H-subunit (the relative code gene of H-subunit is puhA). After that, we constructed an inducible expression pIND4 and another plasmid for cytoplasmic expression of SYPF2 based on that. Then we got three kinds of recombinant strains we need by conjugation or electro-transformation.Finally, we would carry out some subsequent determination including absorption spectrum, growth curve and hydrogen production.
Figure 2：Work flow of our project about experiment in wetlab
Finally, we will characterize our engineering bacteria, including absorption spectra, photosynthetic growth curves and hydrogen production experiments.
 Description
1. Knock in (SYPF2)/ Knock out(crtB)
Figure 3：Work flow of Knock in/out experiment
1.1 Gibson assembly
Since the gene fragments related to gene knock-out and knock-in involves assembly of three fragments, we adopted Gibson Assembly to assemble upstream fragment, downstream fragment and SYPF2 onto the plasmid.
Figure 4：Two devices used in Knock in and Knock out experiment.
1.1.1 Main Protocols
PCR From Genome or Plasmids
Materials:
5xl Gxl Buffer
Primer star Gxl（DNA polymerase）
dNTP
ddH2O
Rhodobacter Sphaeroides 2.4.1 genome
plasmid backbone with sYFP2 gene
Forward/Reverse Primer：
crtBUF1
GAGCTCAGGTTACCCGCATGCAAGATCTATACACGTTCTATGCGCTCTCG
crtBUR1
AGGCCGACTGCAAGATCC
crtBUF2
GGATCTTGCAGTCGGCCT
crtBUR2
TGAGAACCTACATTCCGCGGCAAGCCTTT
crtBDF
CCGCGGAATGTAGGTTCTCATGAAGGTATACCGG
crtBDR
CCCTCGAGTACGCGTCACTAGTGGGGCCCTGATCAGGTAGGGCACGAACA
puhAYFPUF
GAGCTCAGGTTACCCGCATGCAAGATCTATACGGCCACAACAAGATCAAGC
puhAYFPUR
TGCTCACCATGGCGTATTCGGCCAGCATCG
YFPF
CGAATACGCCATGGTGAGCAAGGGCGA
YFPR
CATGCGGGGATCATTACTTGTACAGCTCGTC
puhAYFPDF
CAAGTAATGATCCCCGCATGGCGCGGCCC
puhAYFPDR
CCCTCGAGTACGCGTCACTAGTGGGGCCCTCATCCGCAGGGCGATGGTA
Method:
PCR program:
PCR System（50ul）：
5xl Gxl Buffer
10 ul
dNTP
4 ul
Gxl polymerase
1 ul
Forward primer
2 ul
Reverse primer
2 ul
template
1 ul
Gibson assembly
Materials:
PCR fragment (concentration requirement> 100 ng / ul)
Linearized plasmids (concentration requirements> 100 ng / ul, can be prepared by digestion or reverse PCR)
1.33x ITA reagent
Method:
Add all kinds of PCR fragments and linear plasmids of the same mass to 1.33x ITA reagents (7.5 µL), making total volume 10 µL
Keep the system at 50°C for 1h
Transform 10 µL ligation liquid and select using chl resistance
PCR From Colony
Materials:
10xl Taq Buffer
Taq polymerase
dNTP
ddH2O
Forward/Reverse Primer：
pDM4-F
aacaagccagggatgtaacgc
pDM4-R
tccagtggcttctgtttcta
Method:
Select monoclonal strains to LB plate with chl resistance, and carry out PCR after culture for 2h
PCR
PCR system（10ul）：
5xl Taq Buffer
2 µL
dNTP
0.8ul µL
Taq polymerase
0.2 µL
Forward primer
0.5 µL
Reverse primer
0.5 µL
template
1 µL
ddH2O
5 µL
PCR procedure
Examine the results using electrophoresis. If positive, inoculate 5mL to LB plat with chl resistance for overnight culture. Preserve the stains with 15% glycerol.
1.2 Conjugation
In order to achieve gene knockout / knocking, we use a special pDM4 plasmid, the plasmid has a mob element, which can be used to facilitate the use of bonding method of transformation, in addition to pDM4 on the replicator oriV, so within the globular bacteria can not be copied (Erythrocytes abscess λπ factor), can be re-recombined by chl resistance on the plasmid. Finally, through the SacB gene on pDM4, sucrose was screened for secondary recombination.
1.2.1 Main Protocols：
Conjugation
Materials:
Rhodobacter Sphaeroides
Ecoli-sm10
Joint film
LB medium
PBS buffer
Method:
Ecoli-sm10, Rhodobacter Sphaeroides were incubated overnight for 12 h. Add the corresponding antibiotics to donor bacteria .
Ecoli-sm10, Rhodobacter Sphaeroides secondary culture, add the appropriate antibiotic into the donor bacteria medium, shake to the logarithmic growth phase.
Take Ecoli-sm10, Rhodobacter Sphaeroides, each 1ml wash with PBS and mix them, take 25μl drop in the dried adhesive film, dry. Cultivate overnight.
Wash the bacteria moss on the joint membrane with 1 ml of medium or PBS and take 100 μl of the coated plate. The results were observed after several hours of culture.
After one screening, it was induced with TSB containing 10% sucrose and then subjected to secondary screening on a plate containing 10% sucrose free of resistance.
Validate gene knockout and gene knocking by priming pairs on the genome.
2. Expression of sYFP2 in cytoplasm
Figure 5：Work flow of expression experiment.
As an experimental control group, and whether the optimized post-sYFP2 can be expressed in the cytoplasmic case (as a prerequisite for fusion expression), we hope to be able to express sYFP2 stably in the cytoplasm of Rhodobacter Sphaeroides.
2.1 Reconstruction of PIND4
Prof. Gaoyi Tan provided us with a copy plasmid pIND4 in Rhodobacter Sphaeroides, but it was a constant expression (without induction). To make the expression controllable, we inserted the repressor protein LacIq and its corresponding operon into the plasmid by reconstructing the plasmid , transfer it into a lactose operon-inducible plasmid.
Figure 6：The devices used in improvement of pIND4
2.1.1 Main Protocols：
PCR From Plasmids
Materials:
5xl Gxl Buffer
Primer star Gxl（DNA polymerase）
dNTP
ddH2O
pIND4 plasmid
pMCS-eq plasmid
Forward/Reverse Primer：
F1
CTGGCCTTTTGCTCACATGTTCTTTCCTGCGTTATCCCCT
R1
CGAGCTCGAATTCGCGCGGCCCTGCATTAA
F2
GCCGCGCGAATTCGAGCTCGGTACCGACGT
R2
GGTGGTGAATGTGAAACCAGTAACGTTATACGAT
F3
CTGGTTTCACATTCACCACCCTGAATTGACTCT
R3
AGATCTGGATCCTCCCATGGTTAATTTCTCCTCTTTAATT
Method:
Mentioned above.
PCR From Plasmids
Mentioned above.
2.2 Clone optimization with unoptimized sYFP2
2.2.1 Use the jcat site for codon optimization
Figure 7：The devices used in expression of sYFP2
2.2.2 Main protocols：
PCR From Plasmids
Materials:
Enzyme NcoI and BamHI
10xBuffer（NEB buffer 1.1 2.1 3.1 star）
PCR DNA
ddH2O
pIND4 plasmid DNA
Method:
Configuration of double enzyme digestion system(50ul)：
NcoI   1ul
BamHI   1ul
10xBuffer 5ul (NEB's buffer score 1.1 2.1 3.1 star specific see enzyme instructions select the cutting efficiency and asterisk the best activity)
ddH2O is added as appropriate
DNA 1ug
the reaction temperature of 37 ℃ 15min (specific reaction time see the instructions).
After the end of the reaction 80 ℃ 20min inactivation.
Ligation
Materials:
T4 DNA ligase
10x buffer
Plasmid DNA
PCR DNA
ddH2O
Method:
According to the external source and carrier concentration ratio of 5: 1 configuration connection system 20ul
system 20ul
10xbuffer 2ul
T4 DNA ligase 1ul
plasmid DNA Xul
PCR DNA Yul
16 ℃for 1 h
The connection solution to take 10ul to transform, with kanr resistance screening positive bacteria
Electroporation Transformation
Materials:
10% glycerol
2mm Electroporation cup
Kanr plate
ddH2O
Method:
32℃ overnight culture and transfer, until the OD grows to about 2.5
10 ml of bacteria at 5000 rpm for 5 min to collect all the cells
2ml sterile water pumped retrogradesum (washed away from the medium ion residue, centrifuged at 5000 rpm for 2 min)
with 200ul 10% glycerol resuspend the bacteria
100ul of competent and 100ng plasmid mixed
by adding 2mm electric rotor 1.5Kv shock (voltage index curve mode, capacitance 25uF, resistance 200Ω)
600 ulTSB mixed and moved into the EP tube
32℃, 220rpm recovery 2h
5000 rpm after centrifugation, remove the 400ul supernatant, the remaining bacteria coated on kanr resistant plate
culture 2-3d observation results
3. Final performance
After obtaining three engineering bacteria, we first examined whether the presence of fluorescence in the fluorescence microscope to determine whether the success of the gene, and through the absorption spectrum to check whether there is an additional 514nm near the absorption peak. Finally, we performed photosynthetic growth curve measurements and hydrogen production experiments to test for additional ATP or H2 production.
3.1Spectra or imaging analysis
3.1.1  Room temperature and 77K absorption spectra were recorded using Clariostar-430-9903 in the spectral range between 300 and 900nm（5nm width）.
3.1.2  Fluorescence intensity were recorded using Clariostar-430-9903 with 497nm excitation wavelength and 540nm emission wavelength.
3.1.3  Fluorescence images were taken with an inverted ﬂuorescence microscope
3.2Photosynthetic growth curve analysis
Cells were cultured in TSB medium. After overnight incubation, the OD700 was 0.02 and then the photosynthetic growth curve of 0-24 h was recorded. The cells were given sufficient oxygen. The light source was white LED lamp or green LED lamp (main wavelength For 520nm), the illuminance parameters measured by the illuminometer are shown below
Figure 8：The illuminance of LED bulb used in experiment
3.3Hydrogen production analysis
Hydrogen production analysis
Materials:
Media for fermentation ：
For 1L media:
Succinic acid disodium salt   5.49g
NaCl   0.4g
MgS04·7H20   0.2g
CaCL2·2H20   0.05g
L-glutamate 5mol
KH2PO4   1.0g
K2HPO4 ·2H20   1.572g
Yeast extract fermentation   1.0g
Solution of trace element    1.0ml
Solution of growth factor     1.0ml
For 100ml solution of trace element:
0.21 g  MnSO4 · 4H2O
0.28 g  H3BO3
0.004 g  Cu(NO3)2 · 7H2O,
0.024 g  ZnSO4 · 7H2O,
0.075 g  Na2MoO4 · 2H2O
For 100ml solution of growth factor:
0.01g biotin
0.5g thiamine HCl
1.0g nicotinic acid
Main protocol:
Rhodobacter Sphaeroides were cultured in TSB medium for 48h
Took 10ml bacterial suspension and removed the clear supernatant extract after demission at 3500rpm for 5 minutes
Resuspended the bacteria in 10ml fermentation medium and transferred them into a new conical flask
Added 90ml fermentation medium into the flask with 100μL solution of growth factor, 100μL solution of trace element, 30ng/μL kanamycin and 100μL IPTG (making the final concentration 800μL)
Introduced argon to the flask through rubber tube for 5 min and wrapped the bottleneck with plastic membrane in case of gaseous escape
Pressed the bottleneck with rubber stopper and sealed it with parafilm
Cultured the bacteria in the flask which was exposed to 4,000lx under anaerobic condition with nutrient at 32℃
Collected the gas based on water gas displacing principle using pinhead through rubber stopper and the pinhead should be covered with Vaseline
Drew bacterial suspension with syringe and measured OD every 24 hours. After each measurement, the pinhead should be covered with Vaseline.
