<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_ECUST_Project skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:ECUST/Project</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container-fluid"><DIV id="navbar" class="navbar-collapse collapse" style="font-family: 'Quicksand', sans-serif"><UL class="nav navbar-nav navbar-right"><LI><A class="dropdown-toggle active" href="https://2017.igem.org/Team:ECUST">HOMEPAGE </A></LI><LI class="dropdown"><A class="dropdown-toggle active" href="https://2017.igem.org/Team:ECUST/Description">Project </A></LI><LI><A href="https://2017.igem.org/Team:ECUST/Description">Description</A></LI><LI><A href="https://2017.igem.org/Team:ECUST/Experiment">Experiment</A></LI><LI><A href="https://2017.igem.org/Team:ECUST/Demonstrate">Demonstrate</A></LI></UL><LI><A class="dropdown-toggle active" href="https://2017.igem.org/Team:ECUST/Hardware">HARDWARE </A></LI><LI><A class="dropdown-toggle active" href="https://2017.igem.org/Team:ECUST/Parts">PARTS </A></LI><LI><A class="dropdown-toggle active" href="https://2017.igem.org/Team:ECUST/Model">MODEL </A></LI><LI class="dropdown"><A class="dropdown-toggle active" href="https://2017.igem.org/Team:ECUST/Human_Practices">Human Practices </A></LI><LI><A href="https://2017.igem.org/Team:ECUST/HP/Hp">Human Practices</A></LI><LI><A href="https://2017.igem.org/Team:ECUST/HP/Integrated">Integrated Practices</A></LI><LI><A href="https://2017.igem.org/Team:ECUST/Engagement">Public Engagement</A></LI><LI><A href="https://2017.igem.org/Team:ECUST/HP/Collaboration">Collaborations</A></LI><LI><A class="dropdown-toggle active" href="https://2017.igem.org/Team:ECUST/InterLab">InterLab</A></LI><LI class="dropdown"><A class="dropdown-toggle active" href="https://2017.igem.org/Team:ECUST/Lab/Team">Lab </A></LI><LI><A href="https://2017.igem.org/Team:ECUST/Lab/Team">Team</A></LI><LI><A href="https://2017.igem.org/Team:ECUST/Attributions">Attributions</A></LI><LI><A href="https://2017.igem.org/Team:ECUST/Lab/Notebook">Notebook</A></LI><LI><A href="https://2017.igem.org/Team:ECUST/Lab/Safety">Safety</A></LI></DIV></DIV><DIV class="row"><DIV class="col-md-10"><DIV class="container"><DIV class="row"><H2>Why do we focus on hydrogen?</H2></DIV><DIV class="row style1"><P>With the rising of the world population and the higher average lifespan resulting from the development of science and technology, more energy is required to meet people’s growing demand. However, existing mineral resources on the earth, mainly known as coals, oil and gas, are limited and non-renewable. In order to solve this dilemma, we must find ways to obtain green energy resources with the assistance of modern technology to maintain sustainable development. Hydrogen energy, as a kind of green energy, is an efficient fuel and can be widely used in chemical synthesis, smelting process, etc. As a new energy, hydrogen energy development and utilization is still in its infancy.</P></DIV><DIV class="row"><H2>How do we obtain hydrogen?</H2></DIV><DIV class="row style1"><P>Now several methods are used for the production of hydrogen energy, including water electrolysis, coal gasification, heavy oil and natural gas water vapor catalytic conversion, etc. The relatively green method is water electrolysis, which means hydrogen can be produced by pyrolysis of water with the usage of light-resourced battery<SUP>[1]</SUP>. Additionally, we found that there are many photosynthetic bacteria in the biological world with the corresponding nitrogenase and hydrogenase which can use light in the appropriate conditions to produce hydrogen<SUP> [2]</SUP>. The main mechanism is that the nitrogenase can turn ATP, protons and electrons into H<SUB>2</SUB>, while ATPs come from the photosynthetic phosphorylation, part of the protons are from the tricarboxylic acid cycle, and other parts are from the ATP synthase. The electrons use organic acid (glucose, succinic acid, malic acid) as the donor<SUP>[3]</SUP>.</P></DIV><DIV class="row"><H2>Opportunities and challenges we face</H2></DIV><DIV class="row style1"><P>The use of photosynthetic bacteria has a good application prospect. They use organic acids as electron donor to produce hydrogen, and the organic wastewater from such as food factories can be used during this process.</P></DIV><DIV class="row style1"><P>But during the process of light fermentation,there are two major problems we have to face.</P><P>First,  photoelectric conversion efficiency of photosynthetic bacteria are relatively low. <I>Rhodobacter sphaeroides 2.4.1</I> use photosynthetic reaction center to absorb electrons and by photoion separation they produce electronics. The photochemical conversion efficiency is only 8.4% at 522 nm and 19% at 860 nm.</P></DIV><DIV class="row style1"><P>Second, a reasonable photobioreactor makes it possible to maximize the use of light energy by photosynthetic bacteria, but actually there's no photobioreactor that could meet the demand of <I>Rhodobacter sphaeroides 2.4.1</I>,  which is one of the challenges we face.</P></DIV><DIV class="row"><H2>What are we doing?</H2><H3>Improve photoelectric conversion efficiency</H3></DIV><DIV class="row style1"><P>We used <I>Rhodobacter sphaeroides 2.4.1</I> as a chassis organism for this study. <I>Rhodobacter sphaeroides 2.4.1</I>, as a type of non-sulfur purple photosynthetic bacteria have their unique absorption spectrum. We hope to broaden its absorption spectrum to enhance its total photon absorption.</P></DIV><DIV class="row style"><P>Using the principle of fluorescence resonance energy transfer, we fused the fluorescent protein sYFP2 (excitation wavelength 517 nm, emission wavelength 529 nm) to the H subunit of the RC complex of <I>Rhodobacter sphaeroides 2.4.1</I> in order to enhance the ability of the bulb to absorb photons at 517 nm. At the same time we knocked out <I>crtB</I> to eliminate the effect of carotenoids on photon absorption covered by sYFP2. </P></DIV><DIV class="row style1"><P>The prediction of the RC complex and the sYFP2 distance by protein homology modeling shows that the method is potentially feasible. </P></DIV><P><FONT color="#0000FF"><I> To learn more about homology modeling ，<A href="https://2017.igem.org/Team:ECUST/Part/Theory">please click here.</A></I></FONT></P><DIV class="row"><H3>Design a new type of photobioreactor</H3></DIV><DIV class="row style1"><P>Then, we designed a new type of photobioreactor, the idea of which is to combine the light source with the mixing system. We used the latest LED light technology, and gave full play to its merits, including small size, energy saving and environmental protection. With the help of brush on the top and other ancillary equipment, the light source can rotate with the paddles at the same time, which will not affect the flow field and the mixing effect of the photobioreactor. </P></DIV><CENTER><P><A href="https://static.igem.org/mediawiki/2017/4/42/SHIPING2.mp4 ">Click here to see the video. &quot;Weijie Chen said: 'Let there be light.'&quot;</A></P></CENTER><P><FONT color="#0000FF"><I> To learn more about our photobioreactor，<A href="https://2017.igem.org/Team:ECUST/Hardware">please click here.</A></I></FONT></P><DIV class="row"><H3>Reference:</H3><DIV class="style1"><P>[1]Blankenship R E, Sayre R T. Comparing photosynthetic and photovoltaic efficiencies and recognizing the potential for improvement.[J]. Science, 2011, 332(6031):805.</P><P>[2]Das D, Veziroǧlu T N. Hydrogen production by biological processes: a survey of literature[J]. International Journal of Hydrogen Energy, 2001, 26(1):13-28.</P><P>[3]邓文武. 外加电场辅助质子传递供类球红细菌光合产氢研究[D]. 西南大学, 2010.</P><P>[4]Miyake J. The science of biohydrogen. Biohydrogen [M]. Manoa: University of Hawaii, 1998.7-18</P></DIV></DIV></DIV></DIV></DIV><NAV class="navbar navbar-default navbar-fixed-top" style=" padding-top: 15px;"><DIV class="container-fluid"><DIV id="navbar" class="navbar-collapse collapse" style="font-family: 'Quicksand', sans-serif"><UL class="nav navbar-nav navbar-right"><LI><A class="dropdown-toggle active" href="https://2017.igem.org/Team:ECUST">HOMEPAGE </A></LI><LI class="dropdown"><A class="dropdown-toggle active" href="https://2017.igem.org/Team:ECUST/Description">Project </A></LI><LI><A href="https://2017.igem.org/Team:ECUST/Description">Description</A></LI><LI><A href="https://2017.igem.org/Team:ECUST/Experiment">Experiment</A></LI><LI><A href="https://2017.igem.org/Team:ECUST/Demonstrate">Demonstrate</A></LI></UL><LI><A class="dropdown-toggle active" href="https://2017.igem.org/Team:ECUST/Hardware">HARDWARE </A></LI><LI><A class="dropdown-toggle active" href="https://2017.igem.org/Team:ECUST/Parts">PARTS </A></LI><LI><A class="dropdown-toggle active" href="https://2017.igem.org/Team:ECUST/Model">MODEL </A></LI><LI class="dropdown"><A class="dropdown-toggle active" href="https://2017.igem.org/Team:ECUST/Human_Practices">Human Practices </A></LI><LI><A href="https://2017.igem.org/Team:ECUST/HP/Hp">Human Practices</A></LI><LI><A href="https://2017.igem.org/Team:ECUST/HP/Integrated">Integrated Practices</A></LI><LI><A href="https://2017.igem.org/Team:ECUST/Engagement">Public Engagement</A></LI><LI><A href="https://2017.igem.org/Team:ECUST/HP/Collaboration">Collaborations</A></LI><LI><A class="dropdown-toggle active" href="https://2017.igem.org/Team:ECUST/InterLab">InterLab</A></LI><LI class="dropdown"><A class="dropdown-toggle active" href="https://2017.igem.org/Team:ECUST/Lab/Team">Lab </A></LI><LI><A href="https://2017.igem.org/Team:ECUST/Lab/Team">Team</A></LI><LI><A href="https://2017.igem.org/Team:ECUST/Attributions">Attributions</A></LI><LI><A href="https://2017.igem.org/Team:ECUST/Lab/Notebook">Notebook</A></LI><LI><A href="https://2017.igem.org/Team:ECUST/Lab/Safety">Safety</A></LI></DIV></DIV></NAV><DIV class="col-lg-10 col-centered" style="margin-left: 10%;"><DIV class="row"><H2>Review</H2></DIV><DIV class="row style1"><P>Through <A href="https://2017.igem.org/Team:ECUST/Model">overall modelling</A> of our project, we finally decided to adopt fluorescent protein sYFP2 as our Light-Harvester. By fusing sYFP2 to H subunit (one part of photosynthesis reaction center of <I>Rhodobacter sphaeroides 2.4.1</I>), <I>Rhodobacter sphaeroides 2.4.1</I> would have an extra photon absorption at around 517 nm. From Figure 1, we discovered that <I>Rhodobacter sphaeroides 2.4.1</I> had absorption peaks at 450-550nm (provided by carotenoids) and 800-900nm (provided by chlorophyll). Since the absorption peaks of carotenoids and sYFP2 may overlap to some extent, we needed to knock out gene <I>crtB</I> (phytoene synthase), making it unable for <I>Rhodobacter sphaeroides 2.4.1</I> to synthesize carotenoids.</P></DIV><CENTER><DIV style="font-size:10px;"><P>Figure 1. Room temperature absorption spectrum of membranes from wild type normalised to 590 nm</P></DIV></CENTER><DIV class="row style1"><P>So we chose <I>Rhodobacter sphaeroides 2.4.1</I> as our chassis, we constructed a plasmid for knocking out <I>crtB</I> based on pDM4 and another plasmid for fusing sYFP2 to H-subunit (the relative code gene of H subunit is <I>puhA</I>). After that, we constructed an inducible pIND4. Based on it, we built another plasmid for cytoplasmic expression of sYFP2. Then by conjugation or electro-transformation of three plasmids constructed above, we respectively got three kinds of recombinant strains we needed.Finally, we would carry out some subsequent determination including absorption spectrum, growth curve and hydrogen production.</P></DIV><CENTER><DIV style="font-size:10px;"><P>Figure 2. Work flow of our project about experiment in wetlab</P></DIV></CENTER><DIV class="row"><SPECIALH2><STRONG> Description</STRONG></SPECIALH2><H3>1. Knock in (sYFP2)/ Knock out(<I>crtB</I>)</H3><CENTER><DIV style="font-size:10px;"><P>Figure 3. Work flow of Knock in/out experiment</P></DIV></CENTER><H4>1.1 Gibson assembly</H4></DIV><DIV class="row style1"><P>Since the gene fragments related to gene knock-out and knock-in involved assembly of three fragments, we adopted Gibson Assembly to assemble upstream fragment, downstream fragment and sYFP2 onto the plasmid.</P></DIV><CENTER><DIV style="font-size:10px;"><P>Figure 4. Two devices used in Knock in and Knock out experiment.</P></DIV></CENTER><H5>1.1.1 Main Protocols</H5><DIV class="panel-group" id="accordion" role="tablist" aria-multiselectable="true"><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A role="button" data-toggle="collapse" data-parent="#accordion" href="#collapse1" aria-expanded="false" aria-controls="signup-collapse"><DIV><DIV class="col-md-11"> PCR From Genome or Plasmids</DIV></DIV></A></H4></DIV><DIV id="collapse1" class="panel-collapse collapse" role="tabpanel" aria-labelledby="general"><DIV class="panel-body"><P><B>Materials:</B></P><P>
                5xl Gxl Buffer    
                Primer star Gxl（DNA polymerase）
                dNTP
                ddH<SUB>2</SUB>O<I>Rhodobacter sphaeroides 2.4.1</I> genome
                plasmid backbone with sYFP2 gene</P><P>Forward/Reverse Primer：</P><DIV class="bs-docs-section"><DIV class="row"><DIV class="col-md-12"><DIV class="bs-example table-responsive"><TABLE class="table table-striped table-bordered table-hover"><TBODY><TR><TD>crtBUF1</TD><TD>GAGCTCAGGTTACCCGCATGCAAGATCTATACACGTTCTATGCGCTCTCG</TD></TR><TR><TD>crtBUR1</TD><TD>AGGCCGACTGCAAGATCC</TD></TR><TR><TD>crtBUF2</TD><TD>GGATCTTGCAGTCGGCCT</TD></TR><TR><TD>crtBUR2</TD><TD>TGAGAACCTACATTCCGCGGCAAGCCTTT</TD></TR><TR><TD>crtBDF</TD><TD>CCGCGGAATGTAGGTTCTCATGAAGGTATACCGG</TD></TR><TR><TD>crtBDR</TD><TD>CCCTCGAGTACGCGTCACTAGTGGGGCCCTGATCAGGTAGGGCACGAACA</TD></TR><TR><TD>puhAYFPUF</TD><TD>GAGCTCAGGTTACCCGCATGCAAGATCTATACGGCCACAACAAGATCAAGC</TD></TR><TR><TD>puhAYFPUR</TD><TD>TGCTCACCATGGCGTATTCGGCCAGCATCG</TD></TR><TR><TD>YFPF</TD><TD>CGAATACGCCATGGTGAGCAAGGGCGA</TD></TR><TR><TD>YFPR</TD><TD>CATGCGGGGATCATTACTTGTACAGCTCGTC</TD></TR><TR><TD>puhAYFPDF</TD><TD>CAAGTAATGATCCCCGCATGGCGCGGCCC</TD></TR><TR><TD>puhAYFPDR</TD><TD>CCCTCGAGTACGCGTCACTAGTGGGGCCCTCATCCGCAGGGCGATGGTA</TD></TR></TBODY></TABLE></DIV></DIV></DIV></DIV><P><B>Method:</B></P><P>PCR program:</P><P><B>PCR System（50 μL）：</B></P><DIV class="bs-docs-section"><DIV class="row"><DIV class="col-md-12"><DIV class="bs-example table-responsive"><TABLE class="table table-striped table-bordered table-hover"><TBODY><TR><TD>5xl Gxl Buffer</TD><TD>10 μL</TD></TR><TR><TD>dNTP</TD><TD>4 μL</TD></TR><TR><TD>Gxl polymerase</TD><TD>1 μL</TD></TR><TR><TD>Forward primer</TD><TD>2 μL</TD></TR><TR><TD>Reverse primer</TD><TD>2 μL</TD></TR><TR><TD>template</TD><TD>1 μL</TD></TR></TBODY></TABLE></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="panel-group" id="accordion"><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A role="button" data-toggle="collapse" data-parent="#accordion" href="#collapse2" aria-expanded="false" aria-controls="collapse2"><DIV><DIV class="col-md-11"> Gibson assembly</DIV></DIV></A></H4></DIV><DIV id="collapse2" class="panel-collapse collapse" role="tabpanel" aria-labelledby="general"><DIV class="panel-body"><P><B>Materials:</B></P><P>
                PCR fragment (concentration requirement&gt; 100 ng / μL)    
                Linearized plasmids (concentration requirements&gt; 100 ng / μL, can be prepared by digestion or reverse PCR)
                1.33x ITA reagent</P><P><B>Method:</B></P><OL style="font-size:16px;"><LI>Add all kinds of PCR fragments and linear plasmids of the same mass to 1.33x ITA reagents (7.5 µL), making total volume 10 µL;</LI><LI>Keep the system at 50°C for 1 hour;</LI><LI>Transform constructed plasmids into <I>E.coli-sm10</I> and select positive colony whit chloromycetin resistance.</LI></OL></DIV></DIV></DIV></DIV><DIV class="panel-group" id="accordion"><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A role="button" data-toggle="collapse" data-parent="#accordion" href="#collapse3" aria-expanded="false" aria-controls="collapse3"><DIV><DIV class="col-md-11"> PCR From Colony</DIV></DIV></A></H4></DIV><DIV id="collapse3" class="panel-collapse collapse "><DIV class="panel-body"><P><B>Materials:</B></P><P>
                10xl Taq Buffer     
                Taq polymerase
                dNTP
                ddH<SUB>2</SUB>O</P><P>Forward/Reverse Primer：</P><DIV class="bs-docs-section"><DIV class="row"><DIV class="col-md-12"><DIV class="bs-example table-responsive"><TABLE class="table table-striped table-bordered table-hover"><TBODY><TR><TD>pDM4-F</TD><TD style="text-transform:uppercase;">aacaagccagggatgtaacgc</TD></TR><TR><TD>pDM4-R</TD><TD style="text-transform:uppercase;">tccagtggcttctgtttcta</TD></TR></TBODY></TABLE></DIV></DIV></DIV></DIV><P><B>Method:</B></P><OL style="font-size:16px;"><LI>Select monoclonal strains from LB plate with chloromycetin resistance, and incubate cells in LB media for 2 hours;</LI><LI>Test the monoclonal strains by PCR;</LI><DIV id="myTabContent" class="tab-content"><DIV class="tab-pane fade" id="Method"><P>PCR system（10μL）：</P><TABLE class="table table-striped table-bordered table-hover"><TBODY><TR class="warning"><TD>5xl Taq Buffer</TD><TD>2 µL</TD></TR><TR class="warning"><TD>dNTP</TD><TD>0.8μL</TD></TR><TR class="warning"><TD>Taq polymerase</TD><TD>0.2 µL</TD></TR><TR class="warning"><TD>Forward primer</TD><TD>0.5 µL</TD></TR><TR class="warning"><TD>Reverse primer</TD><TD>0.5 µL</TD></TR><TR class="warning"><TD>template</TD><TD>1 µL</TD></TR><TR class="warning"><TD>ddH<SUB>2</SUB>O</TD><TD>5 µL</TD></TR></TBODY></TABLE><P>PCR procedure</P></DIV></DIV><LI>Examine the results by electrophoresis. If positive, incubate cells in 5 mL LB media with chloromycetin resistance overnight. Preserve the stains with 20% glycerol.</LI></OL></DIV></DIV></DIV></DIV><H4>1.2 Conjugation</H4><DIV class="row style1"><P>In order to knock out/in gene, we used a special  plasmid(pDM4). This plasmid contains gene <I>Mob</I>, replicator <I>oriV</I> and gene <I>SacB</I> .Gene <I>mob</I> can be used to facilitate the use of bonding method of transformation. Replicator oriV couldn't replicate in the globular bacteria (Erythrocytes abscess λπ factor ) , thus single clones could be selected by chloromycetin resistance of the plasmid . Finally , through the gene <I>SacB</I> on pDM4 , sucrose was screened for secondary recombination .</P></DIV><H5>1.2.1 Main Protocols：</H5><DIV class="panel-group" id="accordion"><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A role="button" data-toggle="collapse" data-parent="#accordion" href="#collapse4" aria-expanded="false" aria-controls="collapse4"><DIV><DIV class="col-md-11"> Conjugation</DIV></DIV></A></H4></DIV><DIV id="collapse4" class="panel-collapse collapse" role="tabpanel" aria-labelledby="general"><DIV class="panel-body"><P><B>Materials:</B></P><P><I>Rhodobacter sphaeroides 2.4.1</I><I>E.coli-sm10</I>
        Joint film
        LB medium
        PBS buffer</P><P><B>Method:</B></P><OL style="font-size:16px;"><LI><I>E.coli-sm10</I>, <I>Rhodobacter sphaeroides 2.4.1</I> were incubated overnight for 12 h. Add the corresponding antibiotics to donor bacteria .</LI><LI><I>E.coli-sm10</I>, <I>Rhodobacter sphaeroides 2.4.1</I> secondary culture, add the appropriate antibiotic into the donor bacteria medium, shake to the logarithmic growth phase.</LI><LI> Take 1 mL from both <I>E.coli-sm10</I>, <I>Rhodobacter sphaeroides 2.4.1</I>  washed with PBS and mixed. Take 25 μL of mixture and drop in the dried adhesive film. Dry out and cultivate overnight.</LI><LI> Wash the bacteria moss on the joint film with 1 mL of medium or PBS and take 100 μL of that to coat plate. The results were observed after several hours of culture.</LI><LI>After one screening, it was induced with TSB containing 10% sucrose and then subjected to secondary screening on a plate containing 10% sucrose free of resistance.</LI><LI>Validate gene knock out and gene knock in by priming pairs on the genome.</LI></OL></DIV></DIV></DIV></DIV><H3>2. Expression of sYFP2 in cytoplasm</H3><CENTER><DIV style="font-size:10px;"><P>Figure 5. Work flow of expression experiment.</P></DIV></CENTER><DIV class="row style1"><P>We wanted to figure out whether sYFP2 (with its codons optimized) could be stably expressed in the cytoplasm of <I>Rhodobacter sphaeroides 2.4.1</I> and took it as an experimental control group.</P></DIV><H4>2.1 Reconstruction of PIND4</H4><DIV class="row style1"><P>Prof. Gaoyi Tan provided us with a reproducible plasmid pIND4 in <I>Rhodobacter sphaeroides 2.4.1</I>. In order to make the expression controllable, we inserted the repressor protein LacIq and its corresponding operon into the plasmid by reconstructing the plasmid and transfered it into a lactose operon-inducible plasmid.</P></DIV><CENTER><DIV style="font-size:10px;"><P>Figure 6. The devices used in improvement of pIND4</P></DIV></CENTER><H5>2.1.1 Main Protocols：</H5><DIV class="panel-group" id="accordion" role="tablist" aria-multiselectable="true"><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A role="button" data-toggle="collapse" data-parent="#accordion" href="#collapse5" aria-expanded="false" aria-controls="collapse5"><DIV><DIV class="col-md-11">PCR From Plasmids</DIV></DIV></A></H4></DIV><DIV id="collapse5" class="panel-collapse collapse" role="tabpanel" aria-labelledby="general"><DIV class="panel-body"><P><B>Materials:</B></P><P>
                5xl Gxl Buffer    
                Primer star Gxl（DNA polymerase）
                dNTP
                ddH<SUB>2</SUB>O
                pIND4 plasmid
                pMCS-eq plasmid</P><P>Forward/Reverse Primer：</P><DIV class="bs-docs-section"><DIV class="row"><DIV class="col-md-12"><DIV class="bs-example table-responsive"><TABLE class="table table-striped table-bordered table-hover"><TBODY><TR><TD>F1</TD><TD>CTGGCCTTTTGCTCACATGTTCTTTCCTGCGTTATCCCCT</TD></TR><TR><TD>R1</TD><TD>CGAGCTCGAATTCGCGCGGCCCTGCATTAA</TD></TR><TR><TD>F2</TD><TD>GCCGCGCGAATTCGAGCTCGGTACCGACGT</TD></TR><TR><TD>R2</TD><TD>GGTGGTGAATGTGAAACCAGTAACGTTATACGAT</TD></TR><TR><TD>F3</TD><TD>CTGGTTTCACATTCACCACCCTGAATTGACTCT</TD></TR><TR><TD>R3</TD><TD>AGATCTGGATCCTCCCATGGTTAATTTCTCCTCTTTAATT</TD></TR></TBODY></TABLE></DIV></DIV></DIV></DIV><P><B>Method:</B></P><P>Mentioned above.</P></DIV></DIV></DIV></DIV><DIV class="panel-group" id="accordion" role="tablist" aria-multiselectable="true"><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A role="button" data-toggle="collapse" data-parent="#accordion" href="#collapse6" aria-expanded="false" aria-controls="collapse6"><DIV><DIV class="col-md-11">Gibson Assembly</DIV></DIV></A></H4></DIV><DIV id="collapse6" class="panel-collapse collapse" role="tabpanel" aria-labelledby="general"><DIV class="panel-body"><P>Mentioned above. (see 1.1.1)</P></DIV></DIV></DIV></DIV><H4>2.2 Clone optimization with unoptimized sYFP2</H4><H5>2.2.1 Use the jcat site for codon optimization</H5><DIV class="bs-example table-responsive"><TABLE class="table table-striped table-bordered table-hover" style="word-wrap: break-word;"><THEAD><TR><TH>sYFP2（Part:BBa_K864100）</TH><TH>sYFP2（Part:BBa_K2308003)</TH></TR></THEAD><TBODY><TR><TD>ATGGTTAGCAAGGGCGAAGAACTTTTTACAGGCGTAGTACCGATCTTA
GTTGAATTAGACGGCGACGTTAACGGTCATAAGTTTAGCGTGAGCGGTGAG
GGTGAAGGTGACGCAACTTACGGCAAGCTGACCCTGAAGCTGATTTGCACG
ACGGGTAAGCTGCCGGTCCCGTGGCCTACCCTGGTCACGACCTTGGGTTAT
GGCGTTCAGTGTTTCGCGCGTTATCCGGACCACATGAAACAACACGATTTCT
TTAAGAGCGCGATGCCAGAAGGCTATGTGCAGGAGCGTACGATCTTTTTCAA
AGACGACGGTAACTACAAGACGCGTGCCGAAGTCAAATTCGAAGGCGACAC
CCTGGTGAATCGCATTGAGCTGAAGGGTATTGATTTCAAAGAGGATGGCAAT
ATCCTGGGTCACAAGCTGGAGTACAATTACAATTCCCACAACGTTTACATCA
CCGCAGATAAACAGAAAAATGGCATCAAAGCGAATTTCAAAATCCGTCACAA
CATTGAGGACGGTGGTGTTCAACTGGCGGATCATTACCAGCAAAACACCCC
GATTGGTGACGGTCCGGTCCTGTTGCCGGATAACCATTATCTGTCTTACCAA
AGCAAACTGAGCAAAGATCCGAACGAGAAGCGCGACCACATGGTGCTGCTG
GAGTTTGTGACCGCTGCCGGTATTACCCTGGGTATGGATGAGCTGTATAAATGA</TD><TD>ATGGTTAGCAAGGGCGAAGAACTTTTTACAGGCGTAGTACCGATCTTAG
TTGAATTAGACGGCGACGTTAACGGTCATAAGTTTAGCGTGAGCGGTGAGG
GTGAAGGTGACGCAACTTACGGCAAGCTGACCCTGAAGCTGATTTGCACGA
CGGGTAAGCTGCCGGTCCCGTGGCCTACCCTGGTCACGACCTTGGGTTAT
GGCGTTCAGTGTTTCGCGCGTTATCCGGACCACATGAAACAACACGATTTC
TTTAAGAGCGCGATGCCAGAAGGCTATGTGCAGGAGCGTACGATCTTTTTC
AAAGACGACGGTAACTACAAGACGCGTGCCGAAGTCAAATTCGAAGGCGA
CACCCTGGTGAATCGCATTGAGCTGAAGGGTATTGATTTCAAAGAGGATGG
CAATATCCTGGGTCACAAGCTGGAGTACAATTACAATTCCCACAACGTTTAC
ATCACCGCAGATAAACAGAAAAATGGCATCAAAGCGAATTTCAAAATCCGTC
ACAACATTGAGGACGGTGGTGTTCAACTGGCGGATCATTACCAGCAAAACA
CCCCGATTGGTGACGGTCCGGTCCTGTTGCCGGATAACCATTATCTGTCTT
ACCAAAGCAAACTGAGCAAAGATCCGAACGAGAAGCGCGACCACATGGTG
CTGCTGGAGTTTGTGACCGCTGCCGGTATTACCCTGGGTATGGATGAGCTG
TATAAATGA</TD></TR></TBODY></TABLE></DIV><CENTER><DIV style="font-size:10px;"><P>Figure 7. The devices used in expression of sYFP2</P></DIV></CENTER><H5>2.2.2 Main protocols：</H5><DIV class="panel-group" id="accordion" role="tablist" aria-multiselectable="true"><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A role="button" data-toggle="collapse" data-parent="#accordion" href="#collapse7" aria-expanded="false" aria-controls="collapse7"><DIV><DIV class="col-md-11">Restriction Digestion</DIV></DIV></A></H4></DIV><DIV id="collapse7" class="panel-collapse collapse" role="tabpanel" aria-labelledby="general"><DIV class="panel-body"><P><B>Materials:</B></P><P>
                Enzyme NcoI and BamHI    
                10xBuffer（NEB buffer 1.1 2.1 3.1 star）
                PCR DNA    ddH<SUB>2</SUB>O
                pIND4 plasmid DNA</P><P><B>Method:</B></P><OL style="font-size:16px;"><LI>Configuration of double enzyme digestion system：</LI><P><B>Digestion System（50 μL）：</B></P><DIV class="bs-docs-section"><DIV class="row"><DIV class="col-md-12"><DIV class="bs-example table-responsive"><TABLE class="table table-striped table-bordered table-hover"><TBODY><TR><TD>NcoI</TD><TD>1 μL</TD></TR><TR><TD>BamHI</TD><TD>1 μL</TD></TR><TR><TD>10 x Buffer </TD><TD>5 μL  (NEB's buffer score 1.1 2.1 3.1 star specific see enzyme instructions select the cutting efficiency and asterisk the best activity)</TD></TR><TR><TD>ddH<SUB>2</SUB>O</TD><TD>added as appropriate</TD></TR><TR><TD>DNA</TD><TD>1 μg</TD></TR></TBODY></TABLE></DIV></DIV></DIV></DIV><LI>keep the system at 37℃ for 15min (specific reaction time see the instructions).</LI><LI>Inactivate restriction enzyme at 80℃ for 20 minutes.</LI></OL></DIV></DIV></DIV></DIV><DIV class="panel-group" id="accordion" role="tablist" aria-multiselectable="true"><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A role="button" data-toggle="collapse" data-parent="#accordion" href="#collapse8" aria-expanded="false" aria-controls="collapse8"><DIV><DIV class="col-md-11">Ligation</DIV></DIV></A></H4></DIV><DIV id="collapse8" class="panel-collapse collapse" role="tabpanel" aria-labelledby="general"><DIV class="panel-body"><P><B>Materials:</B></P><P>
        T4 DNA ligase
        10x buffer
        Plasmid DNA
        PCR DNA
        ddH<SUB>2</SUB>O</P><P><B>Method:</B></P><OL style="font-size:16px;"><LI>According to the external source and carrier concentration ratio of 5: 1 configuration connection system 20μL</LI><P><B>Ligation System（20 μL）：</B></P><DIV class="bs-docs-section"><DIV class="row"><DIV class="col-md-12"><DIV class="bs-example table-responsive"><TABLE class="table table-striped table-bordered table-hover"><TBODY><TR><TD>10 x buffer</TD><TD>2 μL</TD></TR><TR><TD>T4 DNA ligase</TD><TD>1 μL</TD></TR><TR><TD>plasmid DNA</TD><TD>X μL</TD></TR><TR><TD>PCR DNA</TD><TD>Y μL</TD></TR></TBODY></TABLE></DIV></DIV></DIV></DIV><P><LI>Incubate the mixture at 16℃ for 1 h</LI><LI>Pipet 10 μL solution from the connection mixture for transformation into <I>E.coli-sm10</I></LI></P></OL></DIV></DIV></DIV></DIV><DIV class="panel-group" id="accordion" role="tablist" aria-multiselectable="true"><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A role="button" data-toggle="collapse" data-parent="#accordion" href="#collapse9" aria-expanded="false" aria-controls="collapse9"><DIV><DIV class="col-md-11">Electroporation Transformation</DIV></DIV></A></H4></DIV><DIV id="collapse9" class="panel-collapse collapse" role="tabpanel" aria-labelledby="general"><DIV class="panel-body"><P><B>Materials:</B></P><P>
        10% glycerol
        2mm Electroporation cup
        Kan<SUP>r</SUP> plate
        ddH<SUB>2</SUB>O</P><P><B>Method:</B></P><OL style="font-size:16px;"><LI>32℃ overnight culture and transfer, until the OD grows to about 2.5</LI><LI>10 mL of bacteria at 5000 rpm for 5 min to collect all the cells</LI><LI>2 mL sterile water pumped retrogradesum (washed away from the medium ion residue, centrifuged at 5000 rpm for 2 min)</LI><LI>200 μL 10% glycerol resuspend the bacteria</LI><LI>Mixing 100 μL of competent and 100ng plasmid</LI><LI>Pipetting 100 μL mixture to 2mm electric rotor shock with 1.5Kv (voltage index curve mode, capacitance 25uF, resistance 200Ω)</LI><LI>600 μL TSB mixed and moved into the EP tube</LI><LI>32℃, 220rpm recovery 2h</LI><LI>5000 rpm after centrifugation, remove the 400 μL supernatant, the remaining bacteria coated on kan<SUP>r</SUP> resistant plate</LI><LI>culture 2-3d observation results</LI></OL></DIV></DIV></DIV></DIV><H3>3. Final performance</H3><DIV class=" row style1"><P>After obtaining three engineering bacteria, we first examined the presence of fluorescence with fluorescence microscope in order to determine whether sYFP2 is well expressed. Then by measuring the absorption spectrum, we checked whether there is an additional peak at 517 nm near the absorption peak. Finally, we carried out photosynthetic growth curve measurements and hydrogen production experiments to test for additional ATP or H2 production.</P></DIV><DIV class="panel-group" id="accordion" role="tablist" aria-multiselectable="true"><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A role="button" data-toggle="collapse" data-parent="#accordion" href="#collapse12" aria-expanded="false" aria-controls="collapse12"><DIV><DIV class="col-md-11">3.1 Spectrum or imaging analysis</DIV></DIV></A></H4></DIV><DIV id="collapse12" class="panel-collapse collapse" role="tabpanel" aria-labelledby="general"><DIV class="panel-body"><DIV class="row style1"><SPECIALH5>3.1.1  Room temperature and 77K absorption spectrums were recorded using Clariostar-430-9903 in the spectral range between 300 and 900nm（5nm width）.</SPECIALH5><SPECIALH5>3.1.2  Fluorescence intensity were recorded using Clariostar-430-9903 with 497nm excitation wavelength and 540nm emission wavelength.<SPECIALH5>3.1.3  Fluorescence images were taken with an inverted ﬂuorescence microscope</SPECIALH5></SPECIALH5></DIV></DIV></DIV></DIV></DIV><DIV class="panel-group" id="accordion" role="tablist" aria-multiselectable="true"><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A role="button" data-toggle="collapse" data-parent="#accordion" href="#collapse11" aria-expanded="false" aria-controls="collapse11"><DIV><DIV class="col-md-11">3.2 Photosynthetic growth curve analysis</DIV></DIV></A></H4></DIV><DIV id="collapse11" class="panel-collapse collapse" role="tabpanel" aria-labelledby="general"><DIV class="panel-body"><DIV class="row style1"><P>Cells were cultured in TSB medium. After overnight incubation, the OD<SUB>700</SUB> was 0.02 and then the photosynthetic growth curve of 0-24 h was recorded. The cells were given sufficient oxygen. The light source was white LED lamp or green LED lamp (main wavelength For 520nm), the illuminance parameters measured by the illuminometer are shown below</P></DIV><CENTER><DIV style="font-size:10px;"><P>Figure 8. The illuminance of LED bulb used in experiment</P></DIV></CENTER></DIV></DIV></DIV></DIV><DIV class="panel-group" id="accordion" role="tablist" aria-multiselectable="true"><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A role="button" data-toggle="collapse" data-parent="#accordion" href="#collapse10" aria-expanded="false" aria-controls="collapse10"><DIV><DIV class="col-md-11">3.3 Hydrogen production analysis</DIV></DIV></A></H4></DIV><DIV id="collapse10" class="panel-collapse collapse" role="tabpanel" aria-labelledby="general"><DIV class="panel-body"><DIV class="row style1"><P><B>Materials:</B></P><P><B>For 1 L fermentation media：</B></P><DIV class="bs-docs-section"><DIV class="row"><DIV class="col-md-12"><DIV class="bs-example table-responsive"><TABLE class="table table-striped table-bordered table-hover"><TBODY><TR><TD> Succinic acid disodium salt </TD><TD>5.49 g</TD></TR><TR><TD>NaCl</TD><TD>0.4 g</TD></TR><TR><TD>MgS0<SUB>4</SUB>·7H<SUB>2</SUB>0</TD><TD>0.2 g</TD></TR><TR><TD>CaCl<SUB>2</SUB>·2H<SUB>2</SUB>0</TD><TD>0.05 g</TD></TR><TR><TD>L-glutamate</TD><TD>5 mol</TD></TR><TR><TD>KH<SUB>2</SUB>PO<SUB>4</SUB></TD><TD>1.0 g</TD></TR><TR><TD> K<SUB>2</SUB>HPO<SUB>4</SUB> ·2H<SUB>2</SUB>0</TD><TD>1.572 g</TD></TR><TR><TD>  Yeast extract</TD><TD>1.0 g</TD></TR><TR><TD> Solution of trace element </TD><TD>1.0 mL</TD></TR><TR><TD>Solution of growth factor  </TD><TD>1.0 mL</TD></TR></TBODY></TABLE></DIV></DIV></DIV></DIV><P><B>For 100 mL solution of trace element:：</B></P><DIV class="bs-docs-section"><DIV class="row"><DIV class="col-md-12"><DIV class="bs-example table-responsive"><TABLE class="table table-striped table-bordered table-hover"><TBODY><TR><TD>MnSO<SUB>4</SUB> · 4H<SUB>2</SUB>O </TD><TD>0.21 g</TD></TR><TR><TD>H<SUB>3</SUB>BO<SUB>3</SUB></TD><TD>0.28 g</TD></TR><TR><TD>Cu(NO<SUB>3</SUB>)<SUB>2</SUB> · 7H<SUB>2</SUB>O</TD><TD>0.004 g</TD></TR><TR><TD>ZnSO<SUB>4</SUB> · 7H<SUB>2</SUB>O</TD><TD>0.024 g </TD></TR><TR><TD>Na<SUB>2</SUB>MoO<SUB>4</SUB> · 2H<SUB>2</SUB>O</TD><TD>0.075 g</TD></TR></TBODY></TABLE></DIV></DIV></DIV></DIV><P><B>For 100 mL solution of growth factor:</B></P><DIV class="bs-docs-section"><DIV class="row"><DIV class="col-md-12"><DIV class="bs-example table-responsive"><TABLE class="table table-striped table-bordered table-hover"><TBODY><TR><TD>biotin</TD><TD>0.01 g</TD></TR><TR><TD>thiamine HCl</TD><TD>0.5 g</TD></TR><TR><TD>nicotinic acid</TD><TD>1.0 g</TD></TR></TBODY></TABLE></DIV></DIV></DIV></DIV><P><B>Main protocol:</B></P><OL style="font-size:16px;"><LI><I>Rhodobacter sphaeroides 2.4.1</I> were cultured in TSB medium for 48h</LI><LI>Took 10 mL bacterial suspension and removed the clear supernatant extract after demission at 3500rpm for 5 minutes</LI><LI>Resuspended the bacteria in 10 mL fermentation medium and transferred them into a new conical flask</LI><LI>Added 90 mL fermentation medium into the flask with 100 μL solution of growth factor, 100 μL solution of trace element, 30ng/μL kanamycin and 100 μL IPTG (making the final concentration 800 μL) </LI><LI>Introduced argon to the flask through rubber tube for 5 min and wrapped the bottleneck with plastic membrane in case of gaseous escape</LI><LI>Pressed the bottleneck with rubber stopper and sealed it with parafilm</LI><LI>Cultured the bacteria in the flask which was exposed to 4,000lx under anaerobic condition with nutrient at 32℃ </LI><LI>Collected the gas based on water gas displacing principle using pinhead through rubber stopper and the pinhead should be covered with Vaseline</LI><LI>Drew bacterial suspension with syringe and measured OD every 24 hours. After each measurement, the pinhead should be covered with Vaseline.</LI></OL><CENTER><CENTER><DIV style="font-size:10px;"><P>Figure 9. The pictures of hydrogen production</P></DIV></CENTER></CENTER></DIV></DIV></DIV></DIV></DIV></DIV></DIV><NAV class="navbar navbar-default navbar-fixed-top" style=" padding-top: 15px;"><DIV class="container-fluid"><DIV id="navbar" class="navbar-collapse collapse" style="font-family: 'Quicksand', sans-serif"><UL class="nav navbar-nav navbar-right"><LI><A class="dropdown-toggle active" href="https://2017.igem.org/Team:ECUST">HOMEPAGE </A></LI><LI class="dropdown"><A class="dropdown-toggle active" href="https://2017.igem.org/Team:ECUST/Description">Project </A></LI><LI><A href="https://2017.igem.org/Team:ECUST/Description">Description</A></LI><LI><A href="https://2017.igem.org/Team:ECUST/Experiment">Experiment</A></LI><LI><A href="https://2017.igem.org/Team:ECUST/Demonstrate">Demonstrate</A></LI></UL><LI><A class="dropdown-toggle active" href="https://2017.igem.org/Team:ECUST/Hardware">HARDWARE </A></LI><LI><A class="dropdown-toggle active" href="https://2017.igem.org/Team:ECUST/Parts">PARTS </A></LI><LI><A class="dropdown-toggle active" href="https://2017.igem.org/Team:ECUST/Model">MODEL </A></LI><LI class="dropdown"><A class="dropdown-toggle active" href="https://2017.igem.org/Team:ECUST/Human_Practices">Human Practices </A></LI><LI><A href="https://2017.igem.org/Team:ECUST/HP/Hp">Human Practices</A></LI><LI><A href="https://2017.igem.org/Team:ECUST/HP/Integrated">Integrated Practices</A></LI><LI><A href="https://2017.igem.org/Team:ECUST/Engagement">Public Engagement</A></LI><LI><A href="https://2017.igem.org/Team:ECUST/HP/Collaboration">Collaborations</A></LI><LI><A class="dropdown-toggle active" href="https://2017.igem.org/Team:ECUST/InterLab">InterLab</A></LI><LI class="dropdown"><A class="dropdown-toggle active" href="https://2017.igem.org/Team:ECUST/Lab/Team">Lab </A></LI><LI><A href="https://2017.igem.org/Team:ECUST/Lab/Team">Team</A></LI><LI><A href="https://2017.igem.org/Team:ECUST/Attributions">Attributions</A></LI><LI><A href="https://2017.igem.org/Team:ECUST/Lab/Notebook">Notebook</A></LI><LI><A href="https://2017.igem.org/Team:ECUST/Lab/Safety">Safety</A></LI></DIV></DIV></NAV></DIV></DIV></DIV></DIV></BODY></HTML>