"
Page
Discussion
View source
History
teams
Log in
 
Team:Groningen/Sensor
From 2012.igem.org
home
team
about us
official profile
gallery
attributions
acknowledgements
contact
project
abstract
data page
sticker
volatiles
sensor
pigment
construct
kill switch
modeling
notebook
parts
our biobricks
improvements
in development
safety
general safety
in the lab
public safety
environment
food safety
human
practice
overview
summerschool
stop food waste
market research
festivals
presentations
collaboration
media coverage
sponsors
our sponsors
sponsorship levels
Sensor
We took a long journey in finding a sensor for spoiled meat. After a literature research, we decided to find a     spoilage sensor experimentally. Read about our quest for "PBADmeat" below.
A literature study about the alsT promoter
The promoter that was initially chosen for the food warden construct was the alsT promoter. This gene is one of several genes that is regulated by TnrA.     TnrA is a regulator in B. subtilis that is triggered by the availability of nitrogen sources. It regulates a few genes and one of them that is     repressed is alsT. TnrA is only active when the amount of nitrogen is low. Since alsT is repressed by TnrA, we suggested that it gets activated as soon    as TnrA is depleted due to the availability of nitrogen (Kayumov et al., 2008). There are a few components in rotting meat that contain nitrogen.     One of these components is ammonia, a highly volatile and abundant compound. These volatiles would trigger the depletion of TnrA, removing it from the     alsT promoter, activating the meat warden construct.
We identified this promoter-regulator system only by performing a literature study. And it was given that there were only a few components in rotting meat    that contain nitrogen. One of these components is ammonia, a highly volatile and abundant compound. These volatiles would trigger the depletion of TnrA,     removing it from the alsT promoter, activating the pBAD meat promoters. However, we were not completely convinced that our whole project should depend     on one promoter, so we asked our modeler to give us more insight into the regulator TnrA and its behavior when exposed to nitrogen sources. Unfortunately,     the results on the modeling page were disappointing…
Reference
Kayumov, A. Heinrich, A. Sharipova, M. Iljinskaya, O. Forchhammer, K. Inactivation of the general transcription factor TnrA in Bacillus subtilis by proteolysis,    Microbiology (2008), 154, 2348–2355.
Finding PBADmeat by transcriptome analysis
With the literature research reaching a dead end and the weeks passing by, we had to come up with a different strategy for finding a sensor for rotting meat volatiles.
We decided to see whether Bacillus subtilis sp. 168 would react to spoiled meat itself. This had two major reasons: the first is that Bacillus subtilis naturally inhabits an environment (especially soil), where it is plausible that it can detect certain metabolites     from other organisms. Furthermore, if we would find a promoter upregulated in B. subtilis, this would make the construct inside our chassis    more reliable than when we would have to transfer a whole uptake system in the not-so-easily-tweaked B. subtilis.
Microarray analysis
We compare the difference in transcription level of all genes (the transcriptome) of Bacillus subtilis at a certain time point,     at defined conditions. We compared the difference in transcription in the exponential growth phase of B. subtilis subjected to fresh and to rotten meat.
From both test conditions, we harvested the mRNA of Bacillus subtilis. With reverse transcriptase, the mRNA was copied into cDNA. Half of the cDNA     was labeled with a green dye, the other half with a red dye. The cDNA of both conditions was brought onto a microarray slide: one red, the other green.     On this slides, small parts (probes) of DNA are put at a known position. The cDNA hybridized with the DNA probes. Because of the colored label, we could     define which genes were transcribed in the green condition, and which genes were transcribed in the red condition. The ratio green:red is a measure for the     up- or downregulation of the genes. On each slide there are three technical replicates to increase the significance of the data. Using a second slide we     swapped the dyes bound to de cDNA of both conditions, in order to rule out any error in dye binding and verifying the acquired data. We also did the whole     experiment in duplo to add significance to the data and avoiding artifacts. The entire workflow is shown below:
Experimental setup
How did we measure the transcriptome of Bacillus subtilis subjected to meat volatiles? We had to grow Bacillus subtilis in the presence of meat,     without touching the meat or interfering with the growth of Bacillus subtilis. We tried a few setups (also fermentors). In the end we decided to use     simple closed system, with a pot with rotten meat (rotten in fridge for 7 days), from which the air (filtered) goes through bacterial culture, stirred with a     magnetic stirrer. The air is pumped through the system with a simple peristaltic pump. The whole system was running at 37 degrees Celsius, note: both the fresh    and rotted meat is placed on ice, this prevented the rotting of the fresh meat during the experiment.
Experimental setup for meat testing. Volatiles of rotten or fresh meat were flushed through a culture of Bacillus subtilis for two hours.     After this, RNA was harvested for further analysis. Check the system in real life below!
Growth conditions
We inoculated Bacillus subtilis from plate and let it grow over night in Luria Broth, in a normal shake flask (37 degrees Celsius).     We diluted the culture to OD(600) = 0.3 and let it grow in a normal shake flask (37 degrees Celsius) until the OD(600) was 0.8 (approximately 2 hours).    At that point, the cultures was diluted to OD(600) = 0.1 and put it in our experimental setup. The setup was ran for two hours, until the OD(600) = 0.8.
Analysis
We analyzed the slide images using ArrayPro 4.5 (Media Cybernetics Inc., Silver Spring, MD). The obtained data was processed and normalized (Lowess normalization)     using the in-house software of the Molecular Genetics group MicroPrep (Van Hijum et. al, 2003). A statistical analysis was done using the webservice CyberT     (Baldi et. al, 2001) A gene was considered differentially expressed when the Bayes p value was lower than 0.001 and the difference in expression (the fold)    was >2 or <-2. The genes obtained using the CyberT analysis were ordered by locus tag and fold. This list was processed further using different analysis tools.     With use of MolGen’s in-house software Genome2d (Baerends et. al, 2004), operons with expression difference were identified and figures showing these operons were made.
Results
We found 19 upregulated operons. We had to rule many of them out, because they were somehow involved in stress-reactions and regulated by general transcription     factors like SigmaB. This left us with five upregulated operons. Of this, we took the two highest upregulated  operons, the NarK-fnr and sboA-sboX-Alb operon.
NarK-fnr is regulated by the redox regulator Fnr. In Bacillus subtilis, Fnr induces the expression of the narGHJI operon as well as NarK,     which are both highly upregulated in our bacteria exposed to meat. The fnr gene is known to be induced in an anaerobic environment in the presence of nitrate.     Since the experimental setup we used in the control (fresh meat) as well as the target (spoiled meat) is a closed system, there is a shortage of oxygen in both situations.     The reason that the operon is upregulated should therefore be a difference in nitrate or other, unknown reactions due to the rotten meat.
Both fnr and sboA-sboX-Alb are regulated by the ResDE signal transduction system. It is thought that this system only activates anaerobically induced     genes in the presence of nitrite. This might explain the difference between the target and control.
encodes the production of an antibiotic compound called subtilosin. It is produced by Bacillus subtilis at anaerobic conditions and at very high cell     densities. Apart from regulation by ResDE, it can also be regulated by the Spo0A system.
Expression details of downregulated operons WapA-yxxG and mntC-mntB-mntA.
There were also many downregulated operons. Most of these operons where general stress responses, but there were two interesting operons involved    in high salt concentrations (mntC-mntB-mntA and salt stress: WapA-yxxG). The first is repressed at high Mn(II) concentrations by MntR,    while it is normally activated by SigmaB and TnrA (see the AlsT section). The second is a two-component system which is repressed at high salt concentrations    by DegU-P. WapA is a cellwall-associated protein which is known to be highly repressed in the presence of 0.7 M disodium succinate. It is thought that     another unknown repressor controls the downregulation of wapA as well. These downregulated operons could be interesting for future implementation into    a multi-colored system (see our development page ).
Testing the promoters
We tested the promoters using our micro-array setup. You can find the test and the results on our construct page .
Reference
Reents H., Münch R., Dammeyer T., Jahn D., Härtig E. (2005).The Fnr Regulon of Bacillus subtilis.  Journal of Bacteriology, 188(3):1103-1112
Nakano M. M., Zheng G., Zuber P. (2000). Dual control of sboa-alb operon expression by the Spo0 and ResDE systems of signal transduction under anaerobic conditions in Bacillus subtilis. Journal of Bacteriology, 181(11): 3274-3277
Dartois V. Débarbouillé M., Kunst F., Rapoport G. (1998). Characterization of a novel member of the DegS-DegU regulon affected by salt stress in Bacillus subtilis. Journal of Bacteriology, 180(7): 1855-1861
Serizawa M., Kodama K, Yamamoto Hl, Kobayashi K., Oqasawara N., Sekiquchi J. (2005) Functional analysis of the YvrGHb two-component system of Bacillus subtilis: identification of the regulated genes by DNA microarray and northern blot analyses. Bioscience, biotechnology and biochemistry, 69(11): 2155-2169
Baldi , Lond, A. D. (2001): "A Bayesian framework for the analysis of microarray expression data: regularized t-test and statistical inferences of gene changes", Bioinformatics 17(6):509-519.
Baerends R, Smits W, De Jong A, Hamoen L, Kok J, Kuipers O (2004). "Genome2D: a visualization tool for the rapid analysis of bacterial transcriptome data." Genome Biol, 5(5):R37.
Our sponsors:
Retrieved from " http://2012.igem.org/Team:Groningen/Sensor "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
