"
Page
Discussion
View source
History
teams
Log in
 
Team:Groningen/Modeling
From 2012.igem.org
home
team
about us
official profile
gallery
attributions
acknowledgements
contact
project
abstract
data page
sticker
volatiles
sensor
pigment
construct
kill switch
modeling
notebook
parts
our biobricks
improvements
in development
safety
general safety
in the lab
public safety
environment
food safety
human
practice
overview
summerschool
stop food waste
market research
festivals
presentations
collaboration
media coverage
sponsors
our sponsors
sponsorship levels
Modeling
What is modeling? This year it can mean building a model to describe the growth curve, it can mean describing a potential signaling pathway,    or, it can mean an attempt at building an organism-scale model of B. subtilis. Each of these definitions is realized through a separate    initiative. If you came directly to this page from the Project menu, then you may be more interested in the techniques and algorithms used in the models.    Skip ahead to the presentation below, and feel free to request a copy of the source code. If you navigated to this page from the Growth section of the     Sticker page, then you're probably expecting some sort of computer simulation predicting real-world growth behavior. Head on down to the Quantification of    Initial Medium Composition. However, if you're interested in learning how to balance complexity with usefulness, read it all. You will see that    complexity is introduced only where necessary (FBA to fill in literature gaps and discover relationships, the organism-scale model to try and discover other    behavior from the GC and gene-expression data). Unless you're designing a Rube Goldberg machine, it is always best to keep things as simple as possible.
Initiatives
This year, the modeling portion of the project was tasked with providing tangible support for the rest of the team.     Theory is all well and good, but a project with such a limited timeframe required focus. The result was three major initiatives:
1. Quantify the initial medium composition which would:
a. Germinate the spores
b. Raise the population to reporter-required levels within the smallest possible spoilage time-frame
c. Maintain an active population at that level for a prolonged period of time, avoiding the onset of dormancy or sporulation
2. The creation of a succinct explanation of TnrA activation, according to current literature.
3. The creation of a dynamic model for B. subtilis suitable for flux balance analysis which responds to environmental cues.
Quantification of Initial Medium Composition
Purpose
The B. subtilis within the sticker must go through distinct phases under defined time constraints. The only way to control the amount of biomass, and its behavior, is through the initial medium concentration. The containment unit does not contain any time-release capsules for providing a controlled level of nutrients.
Methods
Refresh the page if the presentation does not appear. Alternatively, click here . In compliance with the wiki freeze, Google Drive places an unalterable timestamp on the document.     Access granted upon request as this requires a specific email address.
Yes, you can make this presentation full screen. Click the button between the slide number and the cog.
Constrained FBA Model
The constrained model used for flux balance/variability analysis was developed by adding thermodynamic data from the eQuilibrator     database to iBsu1103 developed by Henry et al.. The article provided the model in SBML format. Subsystem information was    contained in the article's supplementary information. libSBML was used to load the SBML model into MATLAB. Custom functions were    used to (1) assign subsystems to reactions, (2) correlate thermodynamic data with metabolites, (3) to calculate the Gibbs     energies of reaction, and (4) to convert the model to GDX; the format required for running FBA and FVA in GAMS.
The resulting model was composed thusly:
Compartments:
Cytosol and Extracellular
Metabolites:
1138 (852 have listed Gibbs energy of formation)
Reactions:
1681 = 1157 intracompartmental (763 with Gibbs energy of reaction) + 244 exchange with medium + 280 transport between compartments
Reaction Subsystems:
Amino Acids and Derivatives
Biomass
Carbohydrates
Cell Wall and Capsule
Cofactors, Vitamins, Prosthetic Groups, Pigments
Exchange
Fatty Acids and Lipids
Macromolecular Synthesis
Membrane Transport
Metabolism of Aromatic Compounds
Nucleosides and Nucleotides
Sulfur Metabolism
Results
A picture is worth a thousand words. Below you will see the spreadsheet which encapsulates all of the relationships and calculations into a user-friendly format.
But, what does it mean? Well, lets take a look at the Input section.
Temperature:
The general operating temperature of the device in degrees Celsius
OD at the Start of Log Growth:
This value is modified till the Final OD value is close to the desired value (explained below)
Initial Amount of Potassium:
Milligrams of potassium at the start
We have the intermediate values either used to calculate the rest, or to simply provide extra information:
Number of Spores:
This is a measure of the number of spores which should be contained in the sticker
Generation Time:
Doubling time in minutes
Biomass Production:
Rate of biomass production in millimolar / grams of dry weight biomass / hour
Potassium Uptake:
Rate of potassium uptake in millimolar / grams of dry weight biomass / hour
Length of Exponential Growth:
Number of hours for which the cells are happily growing exponentially
And, we have the direct values of interest:
Final OD:
This is an experimentally maximized value at which (1) there is enough biomass for the reporter to be visible,        and (2) the growth stops because of a lack of potassium not because of cell-crowding        (we must ensure volatile transport is not hindered by crowding)
Required Glucose:
Total amount of glucose required to support growth to the final OD, in grams
Minimum NH4:
Minimum total amount of nitrogen required to support growth to the final OD, in grams
So, what is missing? The endpoints of the active sensing region. When does it start? When does it end?     As stated in the slides above, this region starts when there is enough biomass for the reporter to be visible.     We know the reporter works, we have seen it. However, the relationship between reporter opacity and amount of biomass has not been quantified.
Also, if you look to the top of this page, this initiative should also adjust the initial amounts to avoid dormancy or sporulation at the final OD.     This is quite the sticky wicket. It requires extensive testing for our specific conditions. The best we can find from sporulation studies in literature is that    you can expect B. subtilis to start producing spores 5 hours after the end of the exponential growth phase.
References
A. Flamholz, E. Noor, A. Bar-Even, and R. Milo, “eQuilibrator--the biochemical thermodynamics calculator,” Nucleic Acids Research, vol. 40, no. D1, pp. D770–D775, Nov. 2011.
C. S. Henry, J. F. Zinner, M. P. Cohoon, and R. L. Stevens, “iBsu1103: a new genome-scale metabolic model of Bacillus subtilis based on SEED annotations,” Genome Biology, vol. 10, no. 6, p. R69, Jun. 2009.
B. J. Bornstein, S. M. Keating, A. Jouraku, and M. Hucka, “LibSBML: an API Library for SBML,” Bioinformatics, vol. 24, no. 6, pp. 880–881, Feb. 2008.
E. Fischer and U. Sauer, “Large-scale in vivo flux analysis shows rigidity and suboptimal performance of Bacillus subtilis metabolism,” Nature Genetics, vol. 37, no. 6, pp. 636–640, May 2005.
J-W. Veening, W. k. Smits, L. w. Hamoen, and O. p. Kuipers, “Single cell analysis of gene expression patterns of competence development and initiation of sporulation in Bacillus subtilis grown on chemically defined media,” Journal of Applied Microbiology, vol. 101, no. 3, pp. 531–541, 2006.
M. Dauner, T. Storni, and U. Sauer, “Bacillus Subtilis Metabolism and Energetics in Carbon-Limited and Excess-Carbon Chemostat Culture,” J. Bacteriol., vol. 183, no. 24, pp. 7308–7317, Dec. 2001.
TnrA Activation
Purpose
Prior to the successful microarray experiment conducted by the wetwork team, the proposed volatile sensing mechanism     tied reporter activation to the metabolism of ammonium/ammonia (NH4/NH3+). The nitrogen metabolism in B. subtilis is a convoluted mesh of reactions and (in)activation complexes mostly controlled by the TnrA transcription factor.     In order to observe the effect of NH4 uptake on the TnrA, it was necessary to have a concise behavioral diagram.     Unfortunately, no such diagram existed. The diagram for nitrogen metabolism on the KEGG database identified most    of what was involved, but did so in an unclear manner.
Results
The figure below used the behavioral information from literature to highlight the active and inactive pathways during     ammonium uptake. In terms of the project, the creation of this diagram brought to light a critical problem for using     TnrA to sense extracellular NH4/NH3+; TnrA is only active when glutamine synthetase (GS) is actively converting NH4 to     glutamine. This in turn is regulated by cell growth, not the amount of NH4/NH3+ present. However, in one article it was    observed that GS is also active under conditions of nitrogen limitation. If the medium contained the precise amount of    glutamine necessary to support the initial growth phase, then the glutamine would be depleted at the sensing phase.    GS would activate to try and create more glutamine. GS would only deactivate when the amount of extracellular NH4/NH3+     reached a level to remove the lack of nitrogen as factor limiting further growth.
The real result? If this sensing pathway was selected, then the wetwork team would have to find the necessary levels of     glutamine to exploit the edge of nitrogen limitation. Possible, but not exactly the easiest mechanism to implement for the consumer market.
References
KEGG Database, "Nitrogen Metabolism - Bacillus subtilis", Kanehisa Laboratories, Last Modified: July 23, 2010. http://www.kegg.jp/kegg-bin/show_pathway?org_name=bsu&mapno=00910
SubtiWiki, http://subtiwiki.uni-goettingen.de/wiki/index.php
K. Gunka and F. M. Commichau, “Control of glutamate homeostasis in Bacillus subtilis: a complex interplay between ammonium assimilation, glutamate biosynthesis and degradation,” Molecular Microbiology, vol. 85, no. 2, pp. 213–224, 2012.
N. Doroshchuk, M. Gelfand, and D. Rodionov, “Regulation of nitrogen metabolism in gram-positive bacteria,” Molecular Biology, vol. 40, no. 5, pp. 829–836, 2006.
Study Guide, Chem153C, University of California, Los Angeles.
http://vohweb.chem.ucla.edu/voh/classes%5Cspring10%5C153CID28%5C11AminoAcidBiosynthesisSQA.pdf
Dynamic Model
Purpose
For our purposes, the dynamic model seeks to provide an explanation for the gene expression observed in the microarray experiment.    However, for the scientific community in general, a dynamic model would be enable phenotype prediction over time under varying     environmental conditions. Such a model would also be able to predict phenotypes for knock-out mutant strains. Probabilistic     integrative modeling (PROM) uses gene expression data across a wide variety of environmental conditions to quantify the link     between the transcriptional-regulatory network and the stoichiometric metabolite-reaction matrix. In other words, PROM modulates     the reaction fluxes by looking at the likelihood a given reaction is active given the current state of gene expression. This method    has been shown to accurately predict growth phenotypes in knock-out strains of E. coli and M. tuberculosis. Unfortunately,    PROM does not consider the current environmental cues. It considers a multitude of prior environmental cues through the gene expression     data. As such, it is really only useful for predicting fluxes in knock-out strains. Despite this limitation, it is still a great     starting point for building a dynamic model as it considers both the transcriptional-regulatory network and a constraint-based     stoichiometric model. It does not rely on the sparse kinetic parameters used in ODE models.
Enter integrative dynamic flux balance analysis (idFBA); it combines the signaling, transcriptional-regulatory, and metabolic networks.
Results
The flowchart below shows the progress of the feasibility study. We were able to build a suitable metabolic model, and to find the information    necesssary for implementing the transcriptional-regulatory network using PROM. We were even able to obtain some idFBA scripts from an author of    the article who happened to be a supervisor to the University of Virginia iGEM team. Unfortunately, the singalling network was a mystery. Sure,     Groningen studies this network in B. subtilis quite extensively, but there still are too many gaps to be able to predict the Wetwork team's    microarray data from their gas chromatography data.
This initiative did not progress past the feasibility study.
References
N. E. Lewis, H. Nagarajan, and B. O. Palsson, “Constraining the metabolic genotype–phenotype relationship using a phylogeny of in silico methods,” Nature Reviews Microbiology, vol. 10, no. 4, pp. 291–305, Apr. 2012.
S. Chandrasekaran and N. D. Price, “Probabilistic integrative modeling of genome-scale metabolic and regulatory networks in Escherichia coli and Mycobacterium tuberculosis,” PNAS, vol. 107, no. 41, pp. 17845–17850, Oct. 2010.
J. Min Lee, E. P. Gianchandani, J. A. Eddy, and J. A. Papin, “Dynamic Analysis of Integrated Signaling, Metabolic, and Regulatory Networks,” PLoS Comput Biol, vol. 4, no. 5, p. e1000086, May 2008.
Our sponsors:
Retrieved from " http://2012.igem.org/Team:Groningen/Modeling "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
