<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Groningen_Wetwork"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Groningen/Wetwork">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Groningen/Wetwork&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Groningen/Wetwork&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Groningen/Wetwork&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Groningen/Wetwork" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Groningen/Wetwork</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><DIV class="expand-down"><UL id="navigation"><LI class="sub" id="menuhome"><A href="https://2012.igem.org/Team:Groningen" title="Home" rel="nofollow"><SPAN>home</SPAN></A></LI><LI class="sub" id="menuteam"><A href="https://2012.igem.org/Team:Groningen/Team" title="Team" rel="nofollow"><SPAN>team</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Groningen/Team">about us</A></LI><LI><A href="https://igem.org/Team.cgi?id=818" target="_blank">official profile</A></LI><LI><A href="https://2012.igem.org/Team:Groningen/Gallery">gallery</A></LI><LI><A href="https://2012.igem.org/Team:Groningen/Attributions">attributions</A></LI><LI><A href="https://2012.igem.org/Team:Groningen/acknowledgements">acknowledgements</A></LI><LI><A href="https://2012.igem.org/Team:Groningen/Contact">contact</A></LI></UL><LI class="sub" id="menuproject"><A href="https://2012.igem.org/Team:Groningen/Project" title="Project" rel="nofollow"><SPAN>project</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Groningen/Project">abstract</A></LI><LI><A href="https://2012.igem.org/Team:Groningen/Data_page">data page</A></LI><LI><A href="https://2012.igem.org/Team:Groningen/Sticker">sticker</A></LI><LI><A href="https://2012.igem.org/Team:Groningen/volatiles">volatiles</A></LI><LI><A href="https://2012.igem.org/Team:Groningen/Sensor">sensor</A></LI><LI><A href="https://2012.igem.org/Team:Groningen/pigmentproduction">pigment</A></LI><LI><A href="https://2012.igem.org/Team:Groningen/Construct">construct</A></LI><LI><A href="https://2012.igem.org/Team:Groningen/Kill_Switch">kill switch</A></LI><LI><A href="https://2012.igem.org/Team:Groningen/Modeling">modeling</A></LI><LI><A href="https://2012.igem.org/Team:Groningen/Notebook">notebook</A></LI><LI class="sub" id="menuparts"><A href="https://2012.igem.org/Team:Groningen/OurBiobrick" title="Parts" rel="nofollow"><SPAN>parts</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Groningen/OurBiobrick">our biobricks</A></LI><LI><A href="https://2012.igem.org/Team:Groningen/parts_fail">improvements</A></LI><LI><A href="https://2012.igem.org/Team:Groningen/in_development">in development</A></LI><LI class="sub" id="menusafety"><A href="https://2012.igem.org/Team:Groningen/Safety" title="Safety" rel="nofollow"><SPAN>safety</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Groningen/Safety">general safety</A></LI><LI><A href="https://2012.igem.org/Team:Groningen/10com">in the lab</A></LI><LI><A href="https://2012.igem.org/Team:Groningen/publicsafety">public safety</A></LI><LI><A href="https://2012.igem.org/Team:Groningen/environment">environment</A></LI><LI><A href="https://2012.igem.org/Team:Groningen/foodsafety">food safety</A></LI><LI class="sub" id="menupr"><A href="https://2012.igem.org/Team:Groningen/overview" title="Public Relation" rel="nofollow"><SPAN>humanpractice</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Groningen/overview">overview</A></LI><LI><A href="https://2012.igem.org/Team:Groningen/summer_school">summerschool</A></LI><LI><A href="https://2012.igem.org/Team:Groningen/Stop_the_food_waste_initiative">stop food waste</A></LI><LI><A href="https://2012.igem.org/Team:Groningen/market_research">market research</A></LI><LI><A href="https://2012.igem.org/Team:Groningen/Festivals">festivals</A></LI><LI><A href="https://2012.igem.org/Team:Groningen/symposia">presentations</A></LI><LI><A href="https://2012.igem.org/Team:Groningen/international_cooperation">collaboration</A></LI><LI><A href="https://2012.igem.org/Team:Groningen/Media_coverage">media coverage</A></LI><LI class="sub" id="menusponsors"><A href="https://2012.igem.org/Team:Groningen/Sponsors" title="Sponsors" rel="nofollow"><SPAN>sponsors</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Groningen/our_sponsors">our sponsors</A></LI><LI><A href="https://2012.igem.org/Team:Groningen/sponsor_levels">sponsorship levels</A></LI></DIV><DIV class="cte"><DIV class="ctd"><Z1>Wetwork</Z1></DIV></DIV><DIV class="sticker2"><DIV class="sticker1"><STICKERP><A href="https://2012.igem.org/Team:Groningen/rotting"><FONT color="#ffffff">sticker</FONT></A></STICKERP></DIV></DIV><DIV class="Germination2"><DIV class="Germination1"><GERMINATIONP><A href="https://2012.igem.org/Team:Groningen/germinationandsporulation"><FONT color="#ffffff">germination</FONT></A></GERMINATIONP></DIV></DIV><DIV class="PBADmeat12"><DIV class="PBADmeat11"><PBADMEAT1P><A href="https://2012.igem.org/Team:Groningen/volatiles"><FONT color="#ffffff">volatiles</FONT></A></PBADMEAT1P></DIV></DIV><DIV class="PBADmeat22"><DIV class="PBADmeat21"><PBADMEAT2P><A href="https://2012.igem.org/Team:Groningen/identication"><FONT color="#ffffff">identification</FONT></A></PBADMEAT2P></DIV></DIV><DIV class="Pigments2"><DIV class="Pigments1"><PIGMENTSP><A href="https://2012.igem.org/Team:Groningen/PBADmeat"><FONT color="#ffffff">pigment</FONT></A></PIGMENTSP></DIV></DIV><DIV class="Construct2"><DIV class="Construct1"><CONSTRUCTP><A href="https://2012.igem.org/Team:Groningen/pigmentproduction"><FONT color="#ffffff">construct</FONT></A></CONSTRUCTP></DIV></DIV><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#When_is_food_rotten.3F"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">When is food rotten?</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Identification_of_bad_meat_volalites_by_Gas_Chromatography-Mass_Spectometry_.28GC-MS.29"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Identification of bad meat volalites by Gas Chromatography-Mass Spectometry (GC-MS)</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Introduction_to_GC-MS"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Introduction to GC-MS</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Organic_solvents_measurements"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">Organic solvents measurements</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-5"><A href="#Identification_of_BADmeat_promoters_by_transcriptome_analysis"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Identification of BADmeat promoters by transcriptome analysis</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Introduction_to_the_microarray_technique"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">Introduction to the microarray technique</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#Experimental_setup"><SPAN class="tocnumber">3.2</SPAN><SPAN class="toctext">Experimental setup</SPAN></A></LI><LI class="toclevel-1 tocsection-8"><A href="#Results"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Results</SPAN></A></LI><LI class="toclevel-2 tocsection-9"><A href="#Overview_of_overexpressed_and_repressed_genes"><SPAN class="tocnumber">4.1</SPAN><SPAN class="toctext">Overview of overexpressed and repressed genes</SPAN></A></LI><LI class="toclevel-1 tocsection-10"><A href="#Testing_Constructs"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Testing Constructs</SPAN></A></LI><LI class="toclevel-1 tocsection-11"><A href="#Failed_BioBricks"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">Failed BioBricks</SPAN></A></LI><LI class="toclevel-2 tocsection-12"><A href="#BBa_K090403"><SPAN class="tocnumber">6.1</SPAN><SPAN class="toctext">BBa_K090403</SPAN></A></LI><LI class="toclevel-2 tocsection-13"><A href="#BBa_I742123"><SPAN class="tocnumber">6.2</SPAN><SPAN class="toctext">BBa_I742123</SPAN></A></LI></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="When_is_food_rotten.3F">When is food rotten?</SPAN></H2><P>To make a rotting sensor, we have to have a defenition of rotten meat. For this, we used the guidelines of the European Union (2006, <A href="http://www.imik.org/wettelijke_context/Europese_hygienerichtlijn_en_microbiologische_criteria.pdf" class="external text" rel="nofollow">source (in Dutch))</A>, doing a simple Total Aerobic Microbial Count test. With this test, one can estimate the amount of colony forming units (CFU) per gram meat. 
</P><P>We used 70% pork, 30 % beef minced meat from our local supermarket. We incubated the meat closed from air, in portions of 1 gram at room temperature, and tested the TAMC at timepoints 0, 3, 5, 7 and 24 hours. The test has been done in triplo.
</P><P>To see the working of our own inbuilt rotting sensor, Elbrich bravely tested the smell and look of meat for 5 hours. According to this we humans smell bad meat pretty well too. Side note: the meat has been exposed to air many times so it could be smelled. The color of the meat changed but little: it turned a bit greyer.
</P><H2><SPAN class="mw-headline" id="Identification_of_bad_meat_volalites_by_Gas_Chromatography-Mass_Spectometry_.28GC-MS.29">Identification of bad meat volalites by Gas Chromatography-Mass Spectometry (GC-MS)</SPAN></H2><H3><SPAN class="mw-headline" id="Introduction_to_GC-MS">Introduction to GC-MS</SPAN></H3><P>With Gas Chromatography-Mass Spectrometry one can separate and identify the different volatiles present in the rotten meat. The University of Groningen has a large database of compounds available so we can identify what the GC-MS data mean. Drawback of the GC-MS is that the compounds that are identified from the rotten meat, will be destroyed during the measurements. If the GC-MS measurements succeed, it should give qualitative data, which is difficult due to the large diversity of volatiles present. We want to thank Monique Smith from Bio Organic Chemistry for her valuable help. 
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Arjan and Tom at work with the GC-MS</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Sample bottles with rotten minced meat</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">All our samples, ready to be measured. The volatiles will be taken up by the syringe in the left corner (red, to the left)</DIV></DIV></DIV></DIV><P>The first step of separation in the GC is based on the fact that different compounds have different boiling points. Therefore in  complete evaporation of the compound should happen all at once and at high temperature. 
</P><P>Later, in the GC oven, the compounds will be condensated and different temperatures will make separation possible. The colums used for GC were HP-5 and HP-1. 
</P><P>Then the measurement continues with Mass Spectrometry. This is a technique to separate compounds based on their mass charge ratio. Our machine uses electron impact, ionization in vacuum and quadrapole separation. Reliability of this measurement depends on the range of the spectrum, reaction(s) with other molecules and how small the differences in molecule structure are. 
</P><H3><SPAN class="mw-headline" id="Organic_solvents_measurements">Organic solvents measurements</SPAN></H3><P>The first measurements with only volatiles gave no reliable results, therefore we decided to dissolve the rotten meat into organic solvents to ensure more compounds are available for measurement.
</P><P><B>Toluene</B></P><P><B>Hexane</B></P><P><B>Methanol</B></P><P><B>Dichloromethane</B></P><H2><SPAN class="mw-headline" id="Identification_of_BADmeat_promoters_by_transcriptome_analysis">Identification of BADmeat promoters by transcriptome analysis</SPAN></H2><H3><SPAN class="mw-headline" id="Introduction_to_the_microarray_technique">Introduction to the microarray technique</SPAN></H3><P>This technique encompasses a comparison between two conditions. These can be a knock-out strain and WT, but also the same strain on a different “diet”. In our case, <I>Bacillus subtilis</I> was subjected to volatiles of fresh vs. bad meat at a certain OD. We use OD 0.8-1.0, representing the late exponential phase,as this should yield much RNA. One isolates all mRNA (the transcriptome) and with reverse transcriptase, the RNA is copied into cDNA. The cDNA is tagged with a fluorescent dye, one condition red, the other green. Both conditions are then brought onto a microarray slide, where DNA of each gene of the organism of interest (in our case, <I>Bacillus</I>) is attached in separate wells (these are called “probes”). When the cDNA is brought onto the slide, it will hybridize with the probes. After fluorescence measurement, one will see a black dot when the gene is not transcribed in both conditions, an orange/yellow one when the gene is transcribed in both, and a green or red one when just one condition causes the transcription of the gene. The ratio of red/green will be a measure for differential expression.
</P><TABLE align="center"><TBODY><TR><TD>Fig. w1. Introduction to the Microarray.
</TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Experimental_setup">Experimental setup</SPAN></H3><P>How do we measure the transcriptome of <I>Bacillus subtilis</I> subjected to meat volatiles? <I>Bacillus</I> should be able to grow without many restrictions to maximize volatiles uptake. --&gt; tried a few setups (also fermentors). In the end: a simple closed system, with a pot with rotten meat (rotten in fridge for 14 days), from which the air (filtered) goes through bacillus in LB medium, stirred with a magnetic stirrer. The air is pumped through the system with a simple peristaltic pump.
</P><TABLE align="center"><TBODY><TR><TD>Fig. w2. Experimental set up for determining the volatiles produced by rotting meat.
</TD></TR></TBODY></TABLE><P><B>Growth conditions <I>Bacillus</I>: volatiles/oxygen depletion</B></P><P>Apart from the microarray, we’ll do this experiment again measuring the OD to see if there is a difference in growth rate for <I>Bacillus</I>.
</P><H2><SPAN class="mw-headline" id="Results">Results</SPAN></H2><H3><SPAN class="mw-headline" id="Overview_of_overexpressed_and_repressed_genes">Overview of overexpressed and repressed genes</SPAN></H3><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:682px;"><DIV class="thumbcaption">Fig. w3. Part of the <I>Bacillus subtilis sp 168</I> genome. Genes depicted as arrows, upregulated genes are green, repressed genes pink. The color intensity is a measure for the fold of upregulation/repression</DIV></DIV></DIV></DIV><P>This picture, generated with MolGen's software Genome2d, is giving an idea of the amount of up- and downregulated genes and operons in <I>Bacillus subtilis</I> when subjected to rotten meat volatiles. 
</P><P>Two highly upregulated operons (upto 40 fold!), and thereby candidates for delivering our &quot;Pbadmeat&quot; are shown in the table below.
</P><H2><SPAN class="mw-headline" id="Testing_Constructs">Testing Constructs</SPAN></H2><P>We designed a construct for testing the different parts of our device. More information will follow, but until now, you can have a peak of the design on our office blackboard!
</P><H2><SPAN class="mw-headline" id="Failed_BioBricks">Failed BioBricks</SPAN></H2><H3><SPAN class="mw-headline" id="BBa_K090403">BBa_K090403</SPAN></H3><P><B>Name</B>  Gram-positive Shuttle Vector for Chromosomal Integration
</P><P><B>Intended Purpose</B>  Plasmid backbone for cloning in <I>Bacillus subtilis</I></P><P><B>Testing Protocol</B> Restriction analysis (single cut)
</P><P><B>Results</B> The restriction pattern showed that this biobrick is not as described in the parts registry. The cut plasmid is shorter than the expected size.
</P><H3><SPAN class="mw-headline" id="BBa_I742123">BBa_I742123</SPAN></H3><P><B>Name</B> multi-host vector pTG262 converted to BioBrick vector
</P><P><B>Intended Purpose</B> plasmid backbone for cloning in <I>Bacilus subtilis</I></P><P><B>Testing Protocol</B> restriction analysis (single cut) and restriction analysis (dual cut)
</P><P><B>Results</B> the restriction enzymes cut at illegal site(s)
</P><DIV class="cta"><H1>Our sponsors:</H1><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:Groningen/Wetwork">http://2012.igem.org/Team:Groningen/Wetwork</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Groningen/Wetwork" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Groningen/Wetwork" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Groningen/Wetwork&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Groningen/Wetwork&amp;oldid=122201" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></BODY></HTML>