<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_BGIC-Union_Result skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:BGIC-Union/Result</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Team：BGIC-Union</TITLE></P><DIV class="column_full_size top1 divfont"><NAV class="navbar navbar-inverse navbar-static-top" role="navigation"><DIV class="container-fluid"><DIV class="navbar-header"><A class="navbar-brand" href="https://2017.igem.org/Team:BGIC-Union">BGIC-Union</A></DIV><DIV><UL class="nav navbar-nav navbar-center"><LI class="active"><A href="https://2017.igem.org/Team:BGIC-Union"><B>HOME</B></A></LI><LI><A href="https://2017.igem.org/Team:BGIC-Union/Team">Team</A></LI><LI><A href="https://2017.igem.org/Team:BGIC-Union/Team">members</A></LI><LI><A href="https://2017.igem.org/Team:BGIC-Union/Collaborations">collaborations</A></LI><LI><A href="https://2017.igem.org/Team:BGIC-Union/Sponsors">sponsors</A></LI><LI><A href="https://2017.igem.org/Team:BGIC-Union/Attributions">attributions</A></LI></UL><LI><A href="https://2017.igem.org/Team:BGIC-Union/Project">Project</A></LI><LI><A href="https://2017.igem.org/Team:BGIC-Union/Description">description</A></LI><LI><A href="https://2017.igem.org/Team:BGIC-Union/Design">design</A></LI><LI><A href="https://2017.igem.org/Team:BGIC-Union/Result">results</A></LI><LI><A href="https://2017.igem.org/Team:BGIC-Union/Model">model</A></LI><LI><A href="https://2017.igem.org/Team:BGIC-Union/Demonstrate">demonstrate</A></LI><LI><A href="https://2017.igem.org/Team:BGIC-Union/Improve">improve</A></LI><LI><A href="https://2017.igem.org/Team:BGIC-Union/Lab">Lab</A></LI><LI><A href="https://2017.igem.org/Team:BGIC-Union/Experiments">experiments</A></LI><LI><A href="https://2017.igem.org/Team:BGIC-Union/Notebook">notebook</A></LI><LI><A href="https://2017.igem.org/Team:BGIC-Union/InterLab">InterLab</A></LI><LI><A href="https://2017.igem.org/Team:BGIC-Union/Parts">Parts</A></LI><LI><A href="https://2017.igem.org/Team:BGIC-Union/Parts #BasicParts">basic parts</A></LI><LI><A href="https://2017.igem.org/Team:BGIC-Union/Composite_Parts #CompositeParts">composite parts</A></LI><LI><A href="https://2017.igem.org/Team:BGIC-Union/Parts_Collection #Collection">parts collection</A></LI><LI><A href="https://2017.igem.org/Team:BGIC-Union/Practices-Design">Practices-Design</A></LI><LI><A href="https://2017.igem.org/Team:BGIC-Union/HP/Silver">silver</A></LI><LI><A href="https://2017.igem.org/Team:BGIC-Union/HP/Gold_Integrated">gold</A></LI><LI><A href="https://2017.igem.org/Team:BGIC-Union/Engagement">public engagement</A></LI><LI><A href="https://2017.igem.org/Team:BGIC-Union/Applied_Design">applied design</A></LI><LI><A href="https://2017.igem.org/Team:BGIC-Union/Entrepreneurship">entrepreneurship</A></LI><LI class="dropdown "><A href="https://2017.igem.org/Team:BGIC-Union/Safety">Safety </A></LI><LI class="dropdown "><A href="https://2017.igem.org/Team:BGIC-Union/Evaluation">EVALUATION </A></LI></DIV></DIV></NAV><SECTION><DIV class="container"><H2 class="dabiaoti">
                1.Expression of PC report system</H2><SPAN class="xiaobiaoti">T7 PC report system:</SPAN><H6 class="imgbiaoti">Figure 1. Plasmid of NT7-dCas9 on pet28a backbone. <SPAN class="imgzhengwen">Our expression plasmid on pet28a backbone of NT7-dCas9 includes a T7 promotor, a lac repressor, the NT7-Linker-dCas9 coding sequence, a 6 X His-Tag and a T7 Terminator.</SPAN></H6><H6 class="imgbiaoti">Figure 2. Plasmid of CT7-dCas9 on pet28a backbone. <SPAN class="imgzhengwen">Our expression plasmid of CT7-dCas9 on pet28a backbone includes a T7 promotor, a lac repressor, the CT7-Linker-dCas9 coding sequence, a 6 X His-Tag and a T7 Terminator.</SPAN></H6><H3 class="xiaobiaoti">Fluc PC report system:</H3><H6 class="imgbiaoti">Figure 3. Plasmid of Nluc-dCas9 on pet28a backbone. <SPAN class="imgzhengwen">Our expression plasmid of Nluc-dCas9 on pet28a backbone includes a T7 promotor, a lac repressor, the Nfluc-Linker-dCas9 coding sequence, a 6 X His-Tag and a T7 Terminator.</SPAN></H6><H6 class="imgbiaoti">Figure 4. Plasmid of Cfluc-dCas9 on pet28a backbone. <SPAN class="imgzhengwen"> Our expression plasmid of Cfluc-dCas9 on pet28a backbone includes a T7 promotor, a lac repressor, the Cfluc-Linker-dCas9 coding sequence, a 6 X His-Tag and a T7 Terminator.</SPAN></H6>We construct both two parts for T7 PC report system and luciferase PC report system. Luciferase PC report system was created to testify its ability and compare its efficiency with ours. 
            Each plasmid is constructed with a T7 promotor, a lac repressor, a RBS and the coding sequence followed by a 6 X His-Tag on PET28a backbone with kanamycin resistance gene. A T7 promotor and a T7 terminator, which can start and stop transcription in the presence with T7 RNA polymerase, are installed at start and end of the part respectively. Lac operator is added for the induction of protein because it can silence the genes behind it unless IPTG is introduced. A 6 X polyhistidine–tag(His - Tag) is included after the coding sequence in order to purify the protein. <H4 class="xiaobiaoti">Cloning: </H4>
            The coding sequence of T7 polymerase is acquired from the Eco.li BL21(DE3), whose genome is genetically modified to include the coding sequence of T7 polymerase. 
            The coding sequence of luciferase is acquired from the control DNA template, pBESTluc™ Vector containing the eukaryotic firefly luciferase gene in E. coli S30 Extract System for Circular DNA, Promega. 
            The coding sequence of dCas9 is acquired from the plasmid pX335-dCas9 plasmid ordered from Addgene. 
            Half of the linker is adhered to the end of NT7,CT7,Nfluc, Cfluc, and half of linker is adhered to the start of dCas9 by modification of 5’ end in primers of PCR. In order to connect the sequence more easily, we rewrite the coding sequence of Linker from GGGGSGGGGS into GGGGGSGGGS in order to create a BamHI restrictive endonuclease site. 
            Likewise, the His-Tag sequence is also adhered to end of dCas9 by modification of 5’ end in primers of PCR. 
            Then the NT7-dCas9-his, CT7-dCas9-his, Nfluc-dCas9-his and Cfluc-dCas9-his coding sequences are inserted into a PET28a plasmid with T7 promotor, lac operator, RBS and T7 terminator by Golden Gate Assembly.
            <H4 class="xiaobiaoti">Protein induction and purification</H4>To analyze the function the PC report system, the target protein should be induced and purified from bacteria. 

            In order to induce the protein, we first transform all the pET28a plasmids into Eco.li strain BL21(DE3), whose genome is genetically modified to contain the coding sequence for T7 polymerase. After preserving the bacteria, we underwent the following procedures.
            <H4 class="xiaobiaoti">Pre-Culture</H4>
            Inoculate all the preserved bacteria into 5ml of LB broth.
            Incubate the samples in the shaker for one night.
            <H4 class="xiaobiaoti">Seed-Culture</H4>
            Inoculate 20ul previous bacteria liquid into new tubes with 5ml of LB broth. Measure and record the OD600 value of each sample in the Nanodrop. In order to find out the best IPTG concentration for induction, we tested a gradient of IPTG concentration. When the OD600 value reaches 0.4 – 0.6, add appropriate amount of IPTG so that the final concentration is in the gradient of 100uM, 200uM, 300uM, 400uM. 
            Incubate the samples in the shaker in 24°C for 19 hours.
            
            We then conducted the extraction of protein by using a commercial Kit.( xTractor Buffer Kit, Clontech). We centrifuged samples to collect the bacteria pellet and resuspend them in buffer provided by the kit together with lysozyme and DNase I. The lysate with target protein was collected from the centrifuged cell debris. Then we resuspend the cell debris in the same buffer and same volume with clear lysate. Then a series of polyacrylamide gel electrophoresis(PAGE) were conducted to find out the best induction concentration. Sometime, when the concentration of IPTG is too high, excessive amount of target protein will force bacteria to transform them into inclusion body protein which can not be purified. Thus the rule of thumb is to find out the concentration with least target protein in cell debris solution and with highest target protein in supernate. All of the four proteins, their molecular weights are around 200KD. With all the four concentration(100uM, 200uM, 300uM, 400uM), we concluded that 200uM is the best induction concentration for four of them.
            <H6 class="imgbiaoti">Figure 5. PAGE of Nfluc-dCas9. <SPAN class="imgzhengwen">The molecular weight of this protein is around 200KD. Alternative of Supernate and sediment with a gradient of IPTG concentrations(100uM, 200uM, 300uM, 400uM) were tested to find the best induction concentration of IPTG.</SPAN></H6><H6 class="imgbiaoti">Figure 6. PAGE of Cfluc-dCas9. <SPAN class="imgzhengwen">The molecular weight of this protein is around 200KD. Alternative of Supernate and sediment with a gradient of IPTG concentrations(100uM, 200uM, 300uM, 400uM) were tested to find the best induction concentration of IPTG.</SPAN></H6><H6 class="imgbiaoti">Figure 7. PAGE of NT7-dCas9. <SPAN class="imgzhengwen">The molecular weight of this protein is around 200KD. Alternative of Supernate and sediment with a gradient of IPTG concentrations(100uM, 200uM, 300uM, 400uM) were tested to find the best induction concentration of IPTG.</SPAN></H6><H6 class="imgbiaoti">Figure 8. PAGE of CT7-dCas9. <SPAN class="imgzhengwen">The molecular weight of this protein is around 200KD. Alternative of Supernate and sediment with a gradient of IPTG concentrations(100uM, 200uM, 300uM, 400uM) were tested to find the best induction concentration of IPTG.</SPAN></H6>
            After determining that we have successfully induced all of the four proteins, we adopted a commercial kit to purify the samples. (Capturem™ His-Tagged Purification Miniprep Kit, Clontech) 

            His-Tag is a series of histidine amino acids that can bind itself with the nickel column of tube provided by the kit. Because of this characteristic, the target protein with his-tag can be separated from Flowthrough. Then we were able to use elution buffer containing imidazole to wash the nickel column to collect target protein. 
            A PAGE was conducted to confirm the efficiency of purification.<H6 class="imgbiaoti">Figure 9. The PAGE of NT7-dCas9, CT7-dCas9, Nfluc-dCas9,Cfluc-dCas9 purified solution and Flowthrough. <SPAN class="imgzhengwen">The red circle indicated the expected protein.</SPAN></H6>
            As the figure shows, we have successfully purified the protein.
            
            However, proteins cannot work with the presence of imidazole. We therefore used a commercial kit to conduct a buffer exchange(Pierce™ Protein Concentrators PES, 30K MWCO, Thermo Fisher). After turns of washing, the protein were transferred from imidazole solution to working solution (20 mM HEPES, 150 mM KCl, pH 7.5).

            da
            <H3 class="dabiaoti">
                2.Production and purification of RNA</H3>
            sgRNA guides the dCas9 protein to its destination. We intended to translate sgRNA in vitro. In order to achieve it, a cassette of T7-promotor-sgRNA-T7 terminator is indispensable and we planed to insert it into psb1c3 plasmid and submit them to IGEM registry as our new Biobricks.
            
            A sgRNA generator comprises of a 20bp guiding sequence and a 77bp scaffold sequence. Together with a T7 promotor and T7 terminator and restrictive enzyme site prepared for insertion, the length for each sgRNA is 213bp. <H6 class="imgbiaoti">Figure 10. The plasmid of sgRNA generator. <SPAN class="imgzhengwen">The sgRNA generator Biobrick is composed of a T7 promotor, coding sequence for sgRNA and a T7 terminator.</SPAN></H6>
            First, we mixed all the six oligo DNAs with PCR mix. A standard PCR was conducted with the annealing temperature suggested by DNAWorks.
            Second, the outer primers was added to the purified product of the first step.
            Finally, run agarose gel analysis and collect the product.<H6 class="imgbiaoti">Figure 11. <SPAN class="imgzhengwen">The electrophoresis of second amplification using the using the oligo DNA in the start and the end of the sequence as primers. The desired length should be around 200bp.</SPAN></H6>

            After restrictive enzyme digestion, all of the seven sgRNA were inserted into psb1c3 plasmid. All of the plasmids were then transformed in to Eco.li strain DH5a and amplified. Sequencing of the samples was done by BGI and most of them were successful.
            
            We also conducted restrictive enzyme check and PCR check to determine its accuracy. <H6 class="imgbiaoti">Figure 12. PCR check of seven sgRNA generators. <SPAN class="imgzhengwen">The PCR check is conducting using VF and VR as primers. Marker is noted as M which has a ladder of 200bp. Control is a J23119 biobrick and noted as C. PCR result of control should be about 350bp. VA23 stands for sgRNA generator for EML4-ALK Variant A 23 and so on. The length of the PCR result of sgRNA generator should be about 500bp.</SPAN></H6>


            We used MEGAscript<SUP>™</SUP> T7 Transcription Kit to yield sgRNA in vitro.
            A PCR was conducted to obtain the linear T7 promotor – sgRNA – T7 terminator cassette. The PCR product was then mixed with T7 RNA polymerase, ribonucleotides and reaction buffer.
            After six hours of incubation at 37 degrees centigrade, DNase was introduced to remove the template DNA.
            
            We used a commercial miRNA purification kit to separate the transcribed sgRNA.(Ambion<SUP>®</SUP> mirVana<SUP>™</SUP> miRNA Isolation Kit)
            After turns of wash and purification, the final RNA product diluted in nuclease-free water can reach the concentration of 1500 ng/ul or higher.


            Also, a RNA electrophoresis was conducted to examine the existence of sgRNA.
            <H6 class="imgbiaoti">Figure 13. Native RNA electrophoresis of all the seven sgRNA. <SPAN class="imgzhengwen">Because our RNA electrophoresis is native electrophoresis, the each RNA won’t concentrate to a band since they may form higher structure. The only purpose we did it was to ensure we had successfully obtain the sgRNA.</SPAN></H6>
            The sgRNA samples were then stored in -20 degrees centigrade.
            <H3 class="dabiaoti">3.NASBA</H3>
            Nucleic acid sequence based amplification (NASBA) is a method in molecular biology that is used to amplify RNA sequences under a constant temperature. We decide to add this procedure to counter with the extremely low concentration of ctDNA in blood.
            
            We conducted NASBA with a gradient of diluted Target DNA to demonstrate the success of NASBA. 
            Procedures: 
            1.	Dilute the target sequence(both Variant A and Variant B) in to 15nM、1.5nM、0.15nM、15pM、1.5pM、0.15pM. 
            2.	Mix T7 polymerase buffer, AMV RT buffer, rNTPs, dNTPs, DEPC water and T7 polymerase together. 
            3.	Process the diluted DNA in 65 degrees centigrade for 5mins and then retrieve to 41 degree centigrade. 
            4.	Mix the NASBA system with diluted DNA. 
            5.	Gel electrophoresis. 

            We compared the PCR result with NASBA result. <H6 class="imgbiaoti">Figure 14. Comparison between PCR result and NASBA result with a gradient of initial concentrations(Variant A). <SPAN class="imgzhengwen">The initial concentrations are 15nM, 1.5nM, 0.15Nm, 15pM, 1.5pM and 0.15pM.</SPAN></H6><H6 class="imgbiaoti">Figure 15. Comparison between PCR result and NASBA result with a gradient of initial concentrations(Variant B). <SPAN class="imgzhengwen">The initial concentrations are 15nM, 1.5nM, 0.15Nm, 15pM, 1.5pM and 0.15pM.</SPAN></H6>

            As the figures show, we have successfully amplified the target sequence both by PCR or NASBA. It means that the NASBA amplification ability can apply to sample with concentration 0.15pM or higher. Also, the relative low amplification rate makes NASBA method easier to regulate the final concentration. The rate of amplification is essential since the efficiency of PC report system will reduce if the concentration of target DNA increases to a certain threshold as suggested by our modeling.
            
            Thus, NASBA can substitute PCR because 1) based on our modeling, too much target DNA concentration will reduce the efficiency of the system and NASBA is easier to regulate than PCR 2) it is easier to conduct and does not require experimental machine.




            </DIV></SECTION></DIV><SECTION><DIV id="footer"><DIV class="container"><DIV class="row"><DIV class="col-lg-4" style="font-family: Arial"><P style="font-family: Arial">F12 BGI-College，146 Beishan Road，Yantian District 518083
                         Shenzhen，Guangdong Province, China，Asia
                        Phone:+86-755-36307888
                        Email: info@genomics.cn
                        Fax:+86-755-36307273</P></DIV><DIV class="col-lg-4"><P style="font-family: Arial"><A href="https://2017.igem.org/Team:BGIC-Union/Applied_Design">Applied Design</A><A href="https://2017.igem.org/Team:BGIC-Union/Entrepreneurship">Entreprenuership</A><A href="https://2017.igem.org/Team:BGIC-Union/Gold_Integrated">Integrated HP</A><A href="https://2017.igem.org/Team:BGIC-Union/Engagement">Education &amp; Public Engagement</A><A href="https://2017.igem.org/Team:BGIC-Union/Model">Model</A><A href="https://2017.igem.org/Team:BGIC-Union/Evaluation">EVALUATION</A></P></DIV><DIV class="col-lg-4"><H4 style="font-family: Arial">BGIC-COLLEGE 2017</H4><P><I class="fa-twitter" style="font-family: Arial">TWITTER:@BGIC-Union</I></P></DIV></DIV></DIV></DIV></SECTION></DIV></DIV></DIV></DIV></DIV></BODY></HTML>