<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Buenos_Aires_Results_Bb1"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Buenos_Aires/Results/Bb1">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Buenos_Aires/Results/Bb1&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Buenos_Aires/Results/Bb1&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Buenos_Aires/Results/Bb1&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Buenos_Aires/Results/Bb1" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Buenos Aires/Results/Bb1</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><UL id="ldd_menu" class="ldd_menu"><LI><SPAN>iGEM BsAs</SPAN></LI><LI class="ldd_heading">The team</LI><LI><A href="https://igem.org/Team.cgi?id=792">Official roster</A></LI><LI><A href="https://2012.igem.org/Team:Buenos_Aires/Team/Members">Members</A></LI><LI><A href="https://2012.igem.org/Team:Buenos_Aires/Team/BsAs">Where we come from</A></LI></UL><UL><LI class="ldd_heading">Safety</LI><LI><A href="https://2012.igem.org/Team:Buenos_Aires/Safety">Key questions</A></LI></UL><UL><LI class="ldd_heading">Attributions</LI><LI><A href="https://2012.igem.org/Team:Buenos_Aires/Attributions">Attributions</A></LI></UL><UL><LI class="ldd_heading">Sponsors</LI><LI><A href="https://2012.igem.org/Team:Buenos_Aires/Sponsors">Institutional</A></LI></UL><A class="ldd_subfoot" href="https://2012.igem.org/Team:Buenos_Aires/DataPage#Data_Page">Data Page</A></DIV><LI><SPAN>Project</SPAN></LI><LI class="ldd_heading">Synthetic ecology</LI><LI><A href="https://2012.igem.org/Team:Buenos_Aires/Project">Overview</A></LI><LI><A href="https://2012.igem.org/Team:Buenos_Aires/Project#Motivation">Motivation</A></LI><LI><A href="https://2012.igem.org/Team:Buenos_Aires/Project#Design">Design</A></LI><LI><A href="https://2012.igem.org/Team:Buenos_Aires/Project#Possible_applications">Possible applications</A></LI><UL><LI class="ldd_heading">Schemes</LI><LI><A href="https://2012.igem.org/Team:Buenos_Aires/Project/Schemes">Crossfeeding</A></LI><LI><A href="https://2012.igem.org/Team:Buenos_Aires/Project/Schemes#Independent_Population_Control">Independent Population Control</A></LI><LI><A href="https://2012.igem.org/Team:Buenos_Aires/Project/Schemes#Cross-population_control">Cross-population control</A></LI><LI><A href="https://2012.igem.org/Team:Buenos_Aires/Project/Schemes#Stochastic_State_Transitions">Stochastic State Transitions</A></LI></UL><UL><LI class="ldd_heading">Modeling</LI><LI><A href="https://2012.igem.org/Team:Buenos_Aires/Project/Model">Modeling synEcology</A></LI><LI><A href="https://2012.igem.org/Team:Buenos_Aires/Project/ModelAdvance">Advanced modeling</A></LI></UL><A class="ldd_subfoot" href="https://2012.igem.org/Team:Buenos_Aires/DataPage#Data_Page">Data Page</A></DIV><LI><SPAN>Results and Devices</SPAN></LI><LI class="ldd_heading">Strains characterization</LI><LI><A href="https://2012.igem.org/Team:Buenos_Aires/Results/Strains#Description_of_strains">Strains description</A></LI><LI><A href="https://2012.igem.org/Team:Buenos_Aires/Results/Strains#Measurment_of_strains_fluorescence">Fluorescence screening</A></LI><LI><A href="https://2012.igem.org/Team:Buenos_Aires/Results/Strains#Screening_of_strain_proportion">Screening of strain proportion</A></LI><LI><A href="https://2012.igem.org/Team:Buenos_Aires/Results/Strains#Auxotrophy_confirmation">Auxotrophy confirmation</A></LI><LI><A href="https://2012.igem.org/Team:Buenos_Aires/Results/Strains#Coculture_in_liquid_medium">Coculture in liquid medium</A></LI><LI><A href="https://2012.igem.org/Team:Buenos_Aires/Results/Strains#Coculture_in_Agar_and_Revertant_mutation_control">Revertant control</A></LI><LI><A href="https://2012.igem.org/Team:Buenos_Aires/Results/Strains#Measurement_of_Trp_in_medium_and_Basal_Production">Trp basal production</A></LI><LI><A href="https://2012.igem.org/Team:Buenos_Aires/Results/Strains#Growth_dependence_on_the_Trp_and_His_concentrations">Growth dependence </A></LI><UL><LI class="ldd_heading">BBs design</LI><LI><A href="https://2012.igem.org/Team:Buenos_Aires/Results/Bb1">Trp/His export devices</A></LI><LI><A href="https://2012.igem.org/Team:Buenos_Aires/Results/Bb2">Backup devices</A></LI><LI><A href="https://2012.igem.org/Team:Buenos_Aires/Results/Cloning">Preparing for sending</A></LI></UL><UL><LI class="ldd_heading">Testing</LI><LI><A href="https://2012.igem.org/Team:Buenos_Aires/Results/DevicesTesting">BioBricks testing</A></LI><LI><A href="https://2012.igem.org/Team:Buenos_Aires/Results/SynEcoTesting">SynEco testing</A></LI></UL><A class="ldd_subfoot" href="https://2012.igem.org/Team:Buenos_Aires/DataPage#Data_Page">Data Page</A></DIV><LI><SPAN>Human practices</SPAN></LI><LI class="ldd_heading">Garage Lab</LI><LI><A href="https://2012.igem.org/Team:Buenos_Aires/HP/GarageLab">Spreading the word</A></LI><LI><A href="https://2012.igem.org/Team:Buenos_Aires/HP/GarageLab#Future_plans">Solving local problems</A></LI><UL><LI class="ldd_heading">EMBO</LI><LI><A href="https://2012.igem.org/Team:Buenos_Aires/HP/EMBO#EMBO_Course_Buenos_Aires_2012">Seeding SynBio in Latin America</A></LI></UL><A class="ldd_subfoot" href="https://2012.igem.org/Team:Buenos_Aires/DataPage#Data_Page">Data Page</A><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#PoPS_-.3E_His.2FTrp_rich_peptide_export_devices"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">PoPS -&gt; His/Trp rich peptide export devices</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Kozak_Sequence"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Kozak Sequence</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Signal_Peptide"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Signal Peptide</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Trojan_peptide"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Trojan peptide</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Payloads"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">Payloads</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-6"><A href="#Implementation"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Implementation</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#Selection_of_restriction_sites"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Selection of restriction sites</SPAN></A></LI><LI class="toclevel-3 tocsection-8"><A href="#Considering_Yeast_Expression_Plasmids"><SPAN class="tocnumber">2.1.1</SPAN><SPAN class="toctext">Considering Yeast Expression Plasmids</SPAN></A></LI><LI class="toclevel-3 tocsection-9"><A href="#Considering_RFC10_assembly_standard_restriction_enzymes"><SPAN class="tocnumber">2.1.2</SPAN><SPAN class="toctext">Considering RFC10 assembly standard restriction enzymes</SPAN></A></LI><LI class="toclevel-3 tocsection-10"><A href="#Final_selection"><SPAN class="tocnumber">2.1.3</SPAN><SPAN class="toctext">Final selection</SPAN></A></LI><LI class="toclevel-3 tocsection-11"><A href="#References"><SPAN class="tocnumber">2.1.4</SPAN><SPAN class="toctext">References</SPAN></A></LI></TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="PoPS_-.3E_His.2FTrp_rich_peptide_export_devices"> PoPS -&gt; His/Trp rich peptide export devices  </SPAN></H1><P>In order for the cross-feeding scheme to work we need each strain to export the amino acid they produce (either Histidine or Tryptophan). To achieve this we created a devices design to secrete to the medium an His (or Trp) rich peptides.
</P><TABLE style="width: 100%"><TBODY><TR><TD align="center"><B>Basic DNA structure of the devices with their constitutive parts</B></TD></TR></TBODY></TABLE><UL><LI><B>Kozak</B> consensus sequence for initiation of translation.
</LI><LI><B>Signal peptide</B> that targets the product of the gene for secretion.
</LI><LI><B>Trojan peptide</B> to increase internalization in target cell.
</LI><LI><B>Payload</B>: this is the exported amino acid rich domain of the protein.
</LI></UL><P>The input of the devices are <I>PoPS</I> and the output is secreted amino acids, so the devices are <I>PoPS -&gt; exported AA</I> transducers. In principle any PoPS generating part can be used.
</P><P>
We built 2 <B>His-export devices</B>, and 2 <B>Trp-export devices</B>:
</P><UL><LI> His-export I (&lt;partinfo&gt;BBa_K792009&lt;/partinfo&gt;)
</LI><LI> Trp export I (&lt;partinfo&gt;BBa_K792010&lt;/partinfo&gt;)
</LI><LI> His-export II (&lt;partinfo&gt;BBa_K792011&lt;/partinfo&gt;)
</LI><LI> Trp export II (&lt;partinfo&gt;BBa_K792012&lt;/partinfo&gt;)
</LI></UL><P>
To achive this, we had to create several <B>new basic biobricks</B>:
</P><UL><LI> Kozak sequence from yeast α-factor mating pheromone (MFα1) (&lt;partinfo&gt;BBa_K792001&lt;/partinfo&gt;)
</LI><LI> Secretion tag from yeast α-factor mating pheromone (MFα1) (&lt;partinfo&gt;BBa_K792002&lt;/partinfo&gt;)
</LI><LI> HIV TAT penetratin (&lt;partinfo&gt;BBa_K792003&lt;/partinfo&gt;)
</LI><LI> Polyarginine trojan peptide (&lt;partinfo&gt;BBa_K792004&lt;/partinfo&gt;)
</LI><LI> PolyHa, a Histidine rich peptide (His-Tag) (&lt;partinfo&gt;BBa_K792005&lt;/partinfo&gt;)
</LI><LI> TrpZipper2, a Tryptophan rich peptide water soluble and monomeric (&lt;partinfo&gt;BBa_K792006&lt;/partinfo&gt;)
</LI><LI> PolyHb, a stable Histidine rich peptide designed by us (&lt;partinfo&gt;BBa_K792007&lt;/partinfo&gt;)
</LI><LI> PolyWb, a stable Tryptophan rich peptide designed by us(&lt;partinfo&gt;BBa_K792008&lt;/partinfo&gt;)
</LI></UL><P>Details about how we create these new basic biobricks can be found in the next sections. More details can be found in their registry entries also.
</P><P>
The next table summaries  each <I>export device</I> composition.
</P><TABLE class="wikitable" width="80%"><TBODY><TR><TH scope="row" style="background: #7ac5e8"><B>Device</B></TH><TH scope="row" style="background: #7ac5e8"><B>Kozak</B></TH><TH scope="row" style="background: #7ac5e8"><B>Signal peptide</B></TH><TH scope="row" style="background: #7ac5e8"><B>Trojan peptide</B></TH><TH scope="row" style="background: #7ac5e8"><B>Payload</B></TH></TR><TR><TH scope="row" style="background: #CCCCCC"> His-export I (&lt;partinfo&gt;BBa_K792009&lt;/partinfo&gt;)
</TH><TD> MFα1 [-12;6] (&lt;partinfo&gt;BBa_K792001&lt;/partinfo&gt;)
</TD><TD> MFα1 secretion tag (&lt;partinfo&gt;BBa_K792002&lt;/partinfo&gt;)
</TD><TD> TAT penetratin (&lt;partinfo&gt;BBa_K792003&lt;/partinfo&gt;)
</TD><TD> PoliHa (HisTag) (&lt;partinfo&gt;BBa_K792005&lt;/partinfo&gt;)
</TD></TR><TR><TH scope="row" style="background: #CCCCCC"> Trp-export I (&lt;partinfo&gt;BBa_K792010&lt;/partinfo&gt;)
</TH><TD> MFα1 [-12;6] (&lt;partinfo&gt;BBa_K792001&lt;/partinfo&gt;)
</TD><TD> MFα1 secretion tag (&lt;partinfo&gt;BBa_K792002&lt;/partinfo&gt;)
</TD><TD> TAT penetratin (&lt;partinfo&gt;BBa_K792003&lt;/partinfo&gt;)
</TD><TD> TrpZipper2 (&lt;partinfo&gt;BBa_K792006&lt;/partinfo&gt;)
</TD></TR><TR><TH scope="row" style="background: #CCCCCC"> His-export II (&lt;partinfo&gt;BBa_K792011&lt;/partinfo&gt;)
</TH><TD>&lt;partinfo&gt;BBa_J63003&lt;/partinfo&gt;</TD><TD>&lt;partinfo&gt;BBa_K416003&lt;/partinfo&gt;</TD><TD> Polyarginine (&lt;partinfo&gt;BBa_K792004&lt;/partinfo&gt;)
</TD><TD> PolyHb (&lt;partinfo&gt;BBa_K792007&lt;/partinfo&gt;)
</TD></TR><TR><TH scope="row" style="background: #CCCCCC"> Trp-export II (&lt;partinfo&gt;BBa_K792012&lt;/partinfo&gt;)
</TH><TD>&lt;partinfo&gt;BBa_J63003&lt;/partinfo&gt;</TD><TD>&lt;partinfo&gt;BBa_K416003&lt;/partinfo&gt;</TD><TD> Polyarginine (&lt;partinfo&gt;BBa_K792004&lt;/partinfo&gt;)
</TD><TD> PolyWb (&lt;partinfo&gt;BBa_K792008&lt;/partinfo&gt;)
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Kozak_Sequence"> Kozak Sequence </SPAN></H2><TABLE><TBODY><TR><TD style="width: 75%"> The Kozak sequence is the eukaryotic analog to the bacterial RBS, it is required for efficient initiation of translation. There is  only one yeast Kozak sequence in the registry (part <A href="http://partsregistry.org/Part:BBa_J63003" class="external text" rel="nofollow">BBa_J63003</A>, distributed in the <A href="http://partsregistry.org/assembly/libraries.cgi?id=42" class="external text" rel="nofollow">2012 kit</A>).  Note that this sequence codes for a glutamic acid (E) after the start codon.
<P>We decided to create a new part (&lt;partinfo&gt;BBa_K792001&lt;/partinfo&gt;) using the 5’UTR of the <A href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006108" class="external text" rel="nofollow">MFα1</A> gene of yeast, partly because we used the signal peptide from this gene (see below). This gene is efficiently translated in yeast, and therefore it stands to reason that translation is efficiently initiated on its mRNA. 
</P></TD><TD style="width: 25%" align="center"><TABLE class="wikitable"><CAPTION> Kozak consensus parts
</CAPTION></TABLE></TD></TR><TR><TH scope="row" style="background: #7ac5e8">Part
</TH><TH scope="row" style="background: #7ac5e8">DNA Sequence
</TH></TR><TR><TD><A href="http://partsregistry.org/Part:BBa_J63003" class="external text" rel="nofollow">BBa_J63003</A></TD><TD>CCCGCCGCCACCATGGAG
</TD></TR><TR><TD>&lt;partinfo&gt;BBa_K792001&lt;/partinfo&gt;</TD><TD>ACGATTAAAAGAATGAGA
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Signal_Peptide"> Signal Peptide </SPAN></H2><TABLE><TBODY><TR><TD style="width: 75%"> The signal peptides target proteins for secretion, effectively exporting the payload.  This peptides are cleaved once the protein is in the lumen of the ER, so they won't have any further relevance.
<P>
Part &lt;partinfo&gt;BBa_K416003&lt;/partinfo&gt; of the registry codes for a yeast signal peptide, based on [Clements 1991]. As an alternative we designed a second part (&lt;partinfo&gt;BBa_K792002&lt;/partinfo&gt;) coding for another signal papetide, from the yeast α-mating factor gene (<A href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006108" class="external text" rel="nofollow">MFα1</A>) [Waters et al 1987]. This part is likely to work well when combined with the Kozak sequence from the same gene (&lt;partinfo&gt;BBa_K792001&lt;/partinfo&gt;, see above), as the natural 5' end of the MFα1 transcript is reconstituted. Also, because it is a yeast gene, it can be used as is, without any optimization. 
</P></TD><TD style="width: 25%" align="center"><TABLE class="wikitable"><CAPTION> Secretion tag parts
</CAPTION></TABLE></TD></TR><TR><TH scope="row" style="background: #7ac5e8">Part
</TH></TR><TR><TD>&lt;partinfo&gt;BBa_K416003&lt;/partinfo&gt;</TD><TD> already in registry
</TD></TR><TR><TD>&lt;partinfo&gt;BBa_K792002&lt;/partinfo&gt;</TD><TD> our contribution
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Trojan_peptide"> Trojan peptide </SPAN></H2><TABLE><TBODY><TR><TD> Trojan peptides are short (15aa) sequences that penetrate through the plasma membrane inside the cell without the need of any receptor or endocitosis process [Derossi 1998].  We want to use them to increase the efficiency with which the payload enters the target cell. Ideally, they should not contain Trp or His, as those are the relevant amino acids for exportation. Two good candidates are the penetratin from the HIV TAT protein, and polyarginine [Jones et al 2005].
</TD></TR><TR><TD><TABLE class="wikitable"><CAPTION> Primary protein structure for penetratins
</CAPTION></TABLE></TD></TR><TR><TH scope="row" style="background: #7ac5e8">Penetratin
</TH><TH scope="row" style="background: #7ac5e8">Residue sequence
</TH></TR><TR><TD>TAT
</TD><TD>YGRKKRRQRRR
</TD></TR><TR><TD>polyarginine
</TD><TD>RRRRRRRRRRR
</TD></TR></TBODY></TABLE> This proteins are not from yeast, so we needed to retro-translate them and codon-optimize them for expression in yeast. To do this we used the R package <A href="http://www.bioconductor.org/packages/2.10/bioc/html/GeneGA.html" class="external text" rel="nofollow">geneGA</A>, that takes into account codon usage and messenger secondary structure in the optimization process.
<TABLE class="wikitable"><CAPTION> Trojan peptide parts
</CAPTION><TBODY><TR><TH scope="row" style="background: #7ac5e8">Penetratin
</TH><TH scope="row" style="background: #7ac5e8">Part
</TH></TR><TR><TD>TAT
</TD><TD>&lt;partinfo&gt;BBa_K792003&lt;/partinfo&gt;</TD></TR><TR><TD>polyarginine
</TD><TD>&lt;partinfo&gt;BBa_K792004&lt;/partinfo&gt;</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Payloads"> Payloads </SPAN></H2><TABLE><TBODY><TR><TD>The payloads are the elements of the synthetic gene that code for the “amino acid rich” region of the secreted protein. By “a.a. rich” we mean, rich in the amino acid we want to export, Trp or His in our case. These domains should be soluble enough not to cause precipitation of the protein, and should be relatively stable not to be degraded before they are actually secreted from the cell.
<P>
We have used and contributed 4 new payload biobricks.
</P><UL><LI><B>His-tag</B>, a polyhistidine-tag normally used for protein purification protocols, or as an epitope for commercial antibodies.
</LI><LI><B>TrpZipper2</B>, a small peptide that folds into a beta-hairpin secondary structure. The indole rings of the Trp form a hydrophobic core. The protein is water soluble and monomeric [Cochran 2001].  
</LI><LI><B>PolyHb</B> and <B>PolyWb</B>, Histidine and Tryptophan rich peptides design by us for this project.
</LI></UL></TD><TD style="width: 25%" align="center"><TABLE class="wikitable"><CAPTION> Payloads parts
</CAPTION></TABLE></TD></TR><TR><TH scope="row" style="background: #7ac5e8">Payload
</TH><TH scope="row" style="background: #7ac5e8">Part
</TH></TR><TR><TD>His tag
</TD><TD>&lt;partinfo&gt;BBa_K792005&lt;/partinfo&gt;</TD></TR><TR><TD>TrpZipper2
</TD><TD>&lt;partinfo&gt;BBa_K792006&lt;/partinfo&gt;</TD></TR><TR><TD>PolyHb
</TD><TD>&lt;partinfo&gt;BBa_K792007&lt;/partinfo&gt;</TD></TR><TR><TD>PolyWb
</TD><TD>&lt;partinfo&gt;BBa_K792008&lt;/partinfo&gt;</TD></TR></TBODY></TABLE><TABLE><TBODY><TR><TD><B>PolyHb</B> and <B>PolyWb</B> were designed taking into acount the following consideration:
<OL><LI> avoided repeating the same residue in tandem to minimize local tRNA depletion
</LI><LI> avoided tryptophan in tandem because of their low solubility
</LI><LI> we included glycine to avoid the formation of rigid structures 
</LI><LI> included acidic and basic amino acids to increase solubility
</LI></OL></TD><TD><TABLE class="wikitable"><CAPTION> Payloads parts and protein sequences
</CAPTION></TABLE></TD></TR><TR><TH scope="row" style="background: #7ac5e8">Payload
</TH><TH scope="row" style="background: #7ac5e8">Residue sequence
</TH></TR><TR><TD>His tag
</TD><TD>HNHNHNHNHNHN
</TD></TR><TR><TD>TrpZipper2
</TD><TD>SWTWENGKWTWK
</TD></TR><TR><TD>PolyHb
</TD><TD>HGDHDGHGKHKG HGDHDGHGKHKG HGDHDGHGKHKG
</TD></TR><TR><TD>PolyWb
</TD><TD>WGDWDGWGKWKG WGDWDGWGKWKG WGDWDGWGKWKG
</TD></TR></TBODY></TABLE><P>Retro-translating and optimizing for yeast (as explained above), we obtained the final sequences (see the registry for  more details).
</P><H1><SPAN class="mw-headline" id="Implementation"> Implementation  </SPAN></H1><P><B>Due to time and resources limitation</B>, we decided to simplify the construction (assembly) process as much as possible. The <B>devices were ordered as a whole</B> (as gBlocks gene fragments) instead of obtaining each constitutive biobrick part and then assembling them. Although this goes against the standard part base approach, it <B>saved both time and money</B>. We plan to obtain each constitutive part from the devices by PCR with suffix/prefix containing primers, as a contribution to the registry.   
</P><P><B>We decided to use yeast expression plasmids with repressible or constitutive promoters</B>, to drive the expression of our devices. This decision was taken because such plasmids were readily available to us, they had adequate selection markers and they are a fairly standard approach in yeast genetics.    
</P><TABLE style="width: 100%"><TBODY><TR><TD align="center"><B>Figure 1: DNA structure we sent to synthesis</B></TD></TR></TBODY></TABLE><P>Although the prefix and suffix are not considered part of the device, they were included in DNA structure we the ordered for easy future assembly.
Because the entire ORF is contained within the prefix and suffix, no care for in-frame assembly has to be taken. We used the original <I>RFC10 BioBrick standard</I>.
</P><P>In addition, <B>we included convenient restriction sites (RS) for directional cloning into the yeast expression vectors</B> (<I>RS1</I> and <I>RS3</I>). <I>RS2</I> will allow easy removal of the sequence coding for the trojan peptide, by restriction and re-ligation. 
</P><H2><SPAN class="mw-headline" id="Selection_of_restriction_sites"> Selection of restriction sites </SPAN></H2><P>We yet have to choose exactly which restriction enzymes we are going to use, so we can know the final DNA sequence to include in the synthesis request.
</P><H3><SPAN class="mw-headline" id="Considering_Yeast_Expression_Plasmids"> Considering Yeast Expression Plasmids </SPAN></H3><TABLE><TBODY><TR><TD width="50%">To determine which restriction sites to use for cloning (<I>RS1</I> and <I>RS3</I> in Figure 1), <B>we need to know the <I>MCS</I> of the expression plasmids we are going to use</B>.
<P>One of these plasmids will probably of the pCM180-5 series, which are centromeric plasmids with TRP1 marker, and with a doxycycline repressible promoter [Gari et al 1996].  
</P><P>
Comparing Figure 3 and Figure 4, <I>BamHI</I> and <I>NotI</I> appear in both <I>MCS</I> in the same order, so they are good candidates for <I>RS1</I> and <I>RS3</I> (Figure 1) respectively.
</P><P>
We will probably need to clone the construct in a general purpose plasmid for manipulation. For instance, to remove the trojan we need to clone the construct into a plasmid, cut it with the RE of RS2, precipitate the DNA (to get rid of the trojan fragment), and religate. For this to work we have to make sure that there is no RS2 in the vector. A common vector for this is pBluescript, wich has the MCS  shown in Figure 5.
</P></TD><TD><TABLE style="width: 100%"><CAPTION><B>Figure 3: Multiple Cloning Site (MCS) of the pCM180 series plasmids'</B></CAPTION></TABLE></TD></TR></TBODY></TABLE><TABLE style="width: 100%"><CAPTION><B>Figure 4: pEG202 MCS sequence and restriction sites</B></CAPTION></TABLE><P>The other plasmid we might use is <I>pEG202</I>, with a 2μ ori, HIS3 marker and a constitutive promoter (PADH1).
</P><TABLE style="width: 100%"><CAPTION><B>Figure 5: pBluescript Multiple Cloning Site</B></CAPTION></TABLE><H3><SPAN class="mw-headline" id="Considering_RFC10_assembly_standard_restriction_enzymes"> Considering RFC10 assembly standard restriction enzymes </SPAN></H3><P>The <B>restriction sites used for RS1-3  have to be different from the ones used in the BioBricks standard</B>. The standard RE for BioBricks are EcoRI, NotI, XbaI, SpeI and PstI.
</P><P>Unfortunately, <I>NotI</I> was our candidate for <I>RS3</I>, so we have a problem here. There are different solutions. We can either use two restriction sites instead of <I>RS3</I>, one for each plasmid, or we can change the BioBrick standard to something like RFC[21] (Berkeley standard) that has no NotI restriction site. 
</P><P>An other option would be to use the restriction sites in the prefix and suffix to clone the construct into the expression plasmid. This is appealing because we don´t need new REs. Anyhow we would need to include a RS in the 5’ end to be able to directionally clone into pCM180. The new design would look something like Figure 6.
</P><TABLE style="width: 100%"><CAPTION><B>Figure 6: Alternative scheme for the restriction sites</B></CAPTION></TABLE><TABLE><TBODY><TR><TD> If we make RS4 a BamHI site, we can directionally clone the construct into both plasmids (pCM180 and pEG202 ) by cutting with BamHI and NotI. In this design we would not need RS1 and RS3, but we can include them just in case we need to clone them into an other vector.
<P>Regarding RS2 we need a restriction enzyme that produces cohesive ends, codes for acceptable amino acids, is easily available and not used in an other part of the construct. Some candidate RE are listed in Table 9. 
</P></TD><TD width="40%" align="center"><TABLE class="wikitable"><CAPTION><B>Table 9. Candidate restriction enzymes for RS2</B></CAPTION></TABLE></TD></TR><TR><TH scope="row" style="background: #7ac5e8">R. Enzyme
</TH><TH scope="row" style="background: #7ac5e8">R. Site sequence
</TH><TH scope="row" style="background: #7ac5e8">Overhang
</TH><TH scope="row" style="background: #7ac5e8">Codes for
</TH></TR><TR><TD>HindIII
</TD><TD>A/AGCTT
</TD><TD>AGCT
</TD><TD>Lys-Leu (KL)
</TD></TR><TR><TD>XhoI
</TD><TD>C/TCGAG
</TD><TD>TCGA
</TD><TD>Leu-Glu (LE)
</TD></TR></TBODY></TABLE><P>Probably any of them will work, but the trojan peptide needs to be basic so the HindIII site looks better suited. 
If we want to remove the trojan, we will have to clone the construct into a vector with no HindIII site. One way to do this is to clone it into pBluescript in such a way that the HindIII restriction site of the MCS is removed. 
</P><P>Looking at Figure 5 we can see that if we cut pBluescript with XhoI and PstI, the HindIII site is removed.  If we make RS1 -&gt; XhoI (which is easily available) we can cut the construct with these same enzymes and directionally clone it into pBluescript.
</P><P>Most likely we wont use RS3, but we can assign it a restriction site just in case. For example NcoI could be used instead of NotI to do the directional cloning into pEG202.
</P><H3><SPAN class="mw-headline" id="Final_selection"> Final selection </SPAN></H3><TABLE><TBODY><TR valign="top"><TD width="55%"><P>Taking in count everything we mentioned above, this is the final selection restriction enzymes we are going to use. Refer to <I>Figure 1</I> to check localization of each restriction site.
</P></TD></TR><TR><TH scope="row" style="background: #7ac5e8">RS#
</TH><TH scope="row" style="background: #7ac5e8">R. Enzyme
</TH><TH scope="row" style="background: #7ac5e8">R. Site Sequence
</TH></TR><TR><TD>RS1
</TD><TD>XhoI
</TD><TD>C/TCGAG
</TD></TR><TR><TD>RS2
</TD><TD>HindIII
</TD><TD>A/AGCTT
</TD></TR><TR><TD>RS3
</TD><TD>NcoI
</TD><TD>C/CATGG
</TD></TR><TR><TD>RS4
</TD><TD>BamHI
</TD><TD>G/GATCC
</TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="References"> References </SPAN></H3><UL><LI>Clements, J. M., G. H. Catlin, et al. (1991). &quot;Secretion of human epidermal growth factor from Saccharomyces cerevisiae using synthetic leader sequences.&quot; Gene 106(2): 267-271.
</LI><LI>Cochran, A. G., N. J. Skelton, et al. (2001). &quot;Tryptophan zippers: stable, monomeric beta -hairpins.&quot; Proc Natl Acad Sci U S A 98(10): 5578-5583.
</LI><LI>Derossi, D., G. Chassaing, et al. (1998). &quot;Trojan peptides: the penetratin system for intracellular delivery.&quot; Trends Cell Biol 8(2): 84-87.
</LI><LI>Jones, S. W., R. Christison, et al. (2005). &quot;Characterisation of cell-penetrating peptide-mediated peptide delivery.&quot; Br J Pharmacol 145(8): 1093-1102.
</LI><LI>Waters, M. G., E. A. Evans, et al. (1988). &quot;Prepro-alpha-factor has a cleavable signal sequence.&quot; J Biol Chem 263(13): 6209-6214.
</LI></UL><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:Buenos_Aires/Results/Bb1">http://2012.igem.org/Team:Buenos_Aires/Results/Bb1</A>&quot;</DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Buenos_Aires/Results/Bb1" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Buenos_Aires/Results/Bb1" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Buenos_Aires/Results/Bb1&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Buenos_Aires/Results/Bb1&amp;oldid=234274" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>