"
Page
Discussion
View source
History
teams
Log in
 
Team:Wageningen UR/notebook/journal/model
From 2014.igem.org
Wageningen UR iGEM 2014
 
 
 
Overview
Background
Solution
Results
Achievements
BioBricks
Project
Fungal Sensing
Fungal Inhibition
Wetlab
Greenhouse
Kill-Switch
Kill-Switch
Characterization
Gene Transfer
Gene Transfer
Low Copy Plasmids
Model
Promoter Design
System Cost
System Stability
Interlab Study
Outreach
Policy
Collaboration
Synenergene
Interviews
National Meetup
Media
Team
Members
Students
Supervisors
Gallery
Attributions
Acknowledgements
Contact
Safety
Environment
Laboratory
Notebook
Journals
Protocols
Interesting Links
Modelers journal
All the included pathways and metabolites used in the metabolic model can be found here .
All the necessary scripts for running the Genomic-Scale Metabolic Model "System Cost" can be found here .
All the necessary scripts for running the Dynamic Model "Promoter Design" and "System Performance" can be found here .
+ expand all
June
Week 4
Designing of the biosynthesis reactions of Pyoverdine.
Found an article with full description about the Pyoverdine of P. putida KT2440, structure formule is also included. Divided the biosynthesis of pyoverdine in 8 reactions, included the modified amino acids and added these to the model. The pathway can be found in pathways and metabolites .
July
Week 1
Pyoverdine added to the GSMM. The next step was to check the parameters for the production of Pyoverdine. The Pyoverdine production was tested against nitrogen uptake, glucose uptake and biomass production. These fluxes were tested in a 4D plot to check its dependency's. The next thing  was validating these maximum production fluxes against data from in vitro experiments to get a reference. One article can be found about the production kinetics of pyoverdine production but the dimension are not usable. So  these experiments have to be performed here in the lab. After validation of the model and setting new constraints in the model, we started on the next fungal growth inhibitor, that was the metabolite DAPG. The biosynthesis pathway was found in literature. So these have were added to the model divided in 4 reactions . pathways and metabolites
Week 2
Mainly focusing on executing the experiments that were setup last week.
the production values of DAPG are tested against the nitrogen uptake and sulfate uptake. This is take a reference against the metabolites that have an increased requirement of these elements. As expected the change of the uptakes did not change the production of DAPG as you expect from the structure formule. Next thing to do is writing a script that tests the DAPG production against the total production pool of CoA to see if there is a big change in the amount of CoA needed correlated with growth rate.
Week 3
This week the pyoverdine experiment has started which was mainly preparation of all the media(M9 Protocol). In the wait steps of this experiment the analysing of the data continued  with the DAPG production kinetics and for the results for the metabolic stress that is induced by this fungal growth inhibitor.
August
Week 1-4
This month the pyoverdine experiment was performed. with the aim to get biologically relevant production range for P. putida. For this experiment the pyoverdine measurment protocol was used. With quantitative data of the biomass concentration and the pyoverdine concentration, the pyoverdine production rate was determined as 150e-3 mmol gDW-1hr-1.
September
Week 1
Writing report for the toolbox. In this toolbox the experiment is included with the title: Determination of pyoverdin production kinetics by P. Putida KT2440.
Week 2
Focused on adding proteins to the model. A test was performed to check for a difference if proteins are added with their protein sequences and proteins described as in the model for biomass production. Conclusion is that is does not have a great impact on the growth rate. With this knowledge we choose for adding the proteins with their protein sequences to approximate the calculations for BananaGuard realistic as possible.
Week 3
This week all the growth fungal inhibitors and Kill-switch are combined in one model. For the fungal growth inhibitors: Pyoverdin and Pyocin, the production kinetics are known as well as for the Kill-switch (production value calculated from input model design) for the other 2 fungal growth inhibitors there is still a lot of literature search required to get correct production values. also the model has been updated to an irreversible model to be able to minimize the total flux trough the model. Because of this we can get rid of existing loops in a Flux balance Analysis (FBA).
Week 4
The Toxin/anti-Toxin part of BananaGuard is added to the model. A literature research has been performed to find all the remaining unknown fungal growth inhibitor production rates. The new extended model is tested and improved to run the simulations faster. Also multiple carbon sources are now possible and it is possible to change the oxygen uptake rate.
October
Week 1
Most the simulations have run this week. Simulations have run for glucose and the carbon composition as found in the exudates of the banana roots. Other important simulations were the parameter analysing. This is simulation is performed with all parameters combined and separately. A single last simulation was performed for two variables, glucose and oxygen, to check for its dependency of these uptake rates. But it has failed so far because of the forced uptake in certain ratio's (wrong ratio gives infeasible FBA results).
Week 2
The simulation with glucose and oxygen as variables could not run properly in a minimized flux model. This has been changed back to a  model with a normal structure. This fix could cause minor differences in the results but was necessary to perform this simulation.
The only thing left to do was making the plots for the wiki’s. This is a time consuming process. In the waiting time of the simulations I focused on writing the metabolic modeling part.
Questions about the metabolic modeling part can be send to walter.dekoster@wur.nl
Dynamic model
The first month of iGEM was dedicated to the making of the statistical mechanics model. The experimentalists needed the optimized configuration of repressor binding sites for the Kill-switch. Most of the time spend on the model went into literature research considering the code to be fairly simple. The last three months were spend on building a stochastic model of BananaGuard. This model was more complex. The runtimes for a single point on the plots could sometimes take a day. For one of the results in the population dynamics the number of simulations had to be reduced to 5000.  For reproduction  purposes 7 scripts have been added which can be used or modified to reproduce the results. Because the raw data files range from several hundred megabytes to gigabytes they could not be uploaded. Because there were a large amount of scripts 7 have been chosen that can be worked with as a base. Should there be any questions feel free to contact me or one of the iGEM team members. All the models are made in python, Ananconda 2.7.
Important to note: Adding stochastisity to the equations and attempting to solve it using the standard odeint solver might cause the solver to crash.  bob.vansluijs@wur.nl
iGEM Wageningen UR 2014. Send us an email , or follow us on Twitter and Facebook !
For more information about iGEM, check out their website and Twitter !
Retrieved from " http://2014.igem.org/Team:Wageningen_UR/notebook/journal/model "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
