<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Wageningen_UR_notebook_journal_inhibition"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Wageningen_UR/notebook/journal/inhibition">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Wageningen_UR/notebook/journal/inhibition&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Wageningen_UR/notebook/journal/inhibition&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Wageningen_UR/notebook/journal/inhibition&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Wageningen_UR/notebook/journal/inhibition" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Wageningen UR/notebook/journal/inhibition</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><P><TITLE>Wageningen UR iGEM 2014 </TITLE></P><DIV id="wrapper"><DIV id="innerwrap"><DIV id="left"><P> </P><DIV id="inner-left"><P> </P><DIV class="sidebar-container"><P> </P></DIV></DIV></DIV><DIV id="center"><DIV class="navmenubkg"><DIV id="nav_wrapper"><DIV id="ca-menu"><UL><LI><A href="https://2014.igem.org/Team:Wageningen_UR/overview"><DIV class="ca-content">Overview</DIV></A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/overview/background">Background</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/overview/solution">Solution</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/overview/results">Results</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/overview/achievements">Achievements</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/overview/parts">BioBricks</A></LI></UL><LI><A href="https://2014.igem.org/Team:Wageningen_UR/project"><DIV class="ca-content">Project</DIV></A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/project/fungal_sensing">Fungal Sensing</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/project/fungal_inhibition">Fungal Inhibition</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/project/fungal_inhibition" class="up_menu_fix">Wetlab</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/project/greenhouse">Greenhouse</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/project/kill-switch">Kill-Switch</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/project/kill-switch" class="up_menu_fix">Kill-Switch</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/project/characterization">Characterization</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/project/gene_transfer">Gene Transfer</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/project/gene_transfer" class="up_menu_fix">Gene Transfer</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/project/low_copy">Low Copy Plasmids</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/project/model_overview">Model</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/project/model#design1" class="up_menu_fix">Promoter Design</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/project/model#cost1">System Cost</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/project/model#performance1">System Stability</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/project/interlab">Interlab Study</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/outreach"><DIV class="ca-content">Outreach</DIV></A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/outreach/policy">Policy</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/outreach/collaboration">Collaboration</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/outreach/synenergene">Synenergene</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/outreach/interviews">Interviews</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/outreach/meetup">National Meetup</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/outreach/media">Media</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/team"><DIV class="ca-content">Team</DIV></A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/team/members">Members</A></LI><LI class="up_menu_fix"><A href="https://2014.igem.org/Team:Wageningen_UR/team/members#students">Students</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/team/members#supervisors">Supervisors</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/team/gallery">Gallery</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/team/attributions">Attributions</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/team/acknowledgements">Acknowledgements</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/team/contact">Contact</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/safety"><DIV class="ca-content">Safety</DIV></A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/safety/environment">Environment</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/safety/laboratory">Laboratory</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/notebook"><DIV class="ca-content">Notebook</DIV></A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/notebook/journal">Journals</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/notebook/protocols">Protocols</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/notebook/links">Interesting Links</A></LI></DIV></DIV></DIV><DIV class="center_content"><H1>Fungal inhibition and greenhouse journal</H1><DIV class="journal"><DIV class="container" style="margin-top:100px;"><DIV id="timelineContainer" class="timelineContainer"><DIV class="timelineToggle"><P><A class="expandAll">+ expand all</A></P></DIV><DIV class="timelineMajor"><H2 class="timelineMajorMarker"><SPAN>May</SPAN></H2><DL class="timelineMinor"><DT id="02w01"><A>Week 4</A></DT><DD class="timelineEvent" id="02w01EX" style="display:none;"><P>Tranformed <I>E. coli</I> with BioBricks psb1C3, psb1A3, BBa_B0034. Grew transformants in 10 ml LB medium containing correct antibiotic and miniprepped them using a Genejet miniprep kit.
</P></DD></DL></DIV><DIV class="timelineMajor"><H2 class="timelineMajorMarker"><SPAN>June</SPAN></H2><DL class="timelineMinor"><DT id="02w02"><A>Week 1</A></DT><DD class="timelineEvent" id="02w02EX" style="display:none;"><P>PCR methyl-γ-lyase with NEB  Q5 polymerase.  Used as templates:  1. Camembert cheese 2. <I>Bevibacterium</I> strain isolated from sponge. Ran gel with 1 kb NEB ladder. Expected product size: 1.5 kbp.
Primers: 
Fw: 5’-GTTTCTTCGAATTCGCGGCCGCTTCTAGATGAGTATCACCCAGAACG-3'
Rev: 5’- GTTTCTTCCTGCAGCGGCCGCTACTAGTATTATTATTCATACCGTTGCTACAGG-3'
<FIGURE><FIGCAPTION>PCR methionine-gamma-lyase</FIGCAPTION></FIGURE></P></DD></DL><DL class="timelineMinor"><DT id="02w03"><A>Week 2</A></DT><DD class="timelineEvent" id="02w03EX" style="display:none;"><P>Colony PCR with Q5 polymerase, using <I>P. putida</I> colonies as template. Ran gel with 100 bp NEB ladder. Expected bands size were around 500 bp.
Primers:
Fw: 5’-GTTTCTTCGAATTCGCGGCCGCTTCTAGATGGCGGAACAACTATCCACAAGTAAG
Rev: 5’- GTTTCTTCCTGCAGCGGCCGCTACTAGTATTATTATCAGGCCTGGCGACTGGC<FIGURE><FIGCAPTION>PCR PfrI</FIGCAPTION></FIGURE></P><P>Extract pfrI from gel, digested it with SpeI and xbaI.  Ligated pfrI with BBa_B0034 (digested with SpeI).</P></DD></DL><DL class="timelineMinor"><DT id="02w04"><A>Week 3-4</A></DT><DD class="timelineEvent" id="02w04EX" style="display:none;"><P>Transformed chemical competent <I>E. coli</I> cells </P><UL><LI>Ligation mix methyl-γ-lyase + BBa_B0034</LI><LI>Ligation mix pfrI + BBa_B0034</LI><LI>psb1T3</LI><LI>psb1K3</LI><LI>positive control: puc19</LI><LI>negative control: nothing</LI></UL><P>Psb1T3, psb1K3 transformants were grown in 10ml and then miniprepped.
Colony PCR with methyl-γ-lyase + BBa_B0034 transformant colonies using Taq polymerase. Primers were VF2 and a reverse primer from methyl-γ-lyase.  Ran gel with 1 kb NEB ladder. Is expect a band of around 1.5 kpb to appear if there was correct insertion of the product into the plasmid.<FIGURE><FIGCAPTION>Colony PCR Methionine-gamma-lyase</FIGCAPTION></FIGURE>
Sample 1, 2, 3 and 5 seem to have the correct insert. So glycerol stocks were made for sample 1 and 2.
</P></DD></DL></DIV><DIV class="timelineMajor"><H2 class="timelineMajorMarker"><SPAN>July</SPAN></H2><DL class="timelineMinor"><DT id="02w05"><A>Week 1</A></DT><DD class="timelineEvent" id="02w05EX" style="display:none;"><P>Sequencing data showed that Metionine-γ-lyase was inserted correctly together with Bba_b0034 (RBS), pfrI + B_0034 transformants were grown in 10 ml LB containing antibiotic and miniprepped. Plasmids were digested with EcoRI &amp; SpeI. This was done because insert could enter in 2 ways, need to check for the correct insert.  Correct insert would give us 2132 bp and 566 bp, the incorrect insert will give us 5675 bp and 23 bp. Ran it with 1 kb NEB ladder.</P><FIGURE><FIGCAPTION>Schematic drawing of digestion and ligation strategy</FIGCAPTION></FIGURE><FIGURE><FIGCAPTION>PfrI digest with EcoRI &amp; SpeI</FIGCAPTION></FIGURE></DD></DL><DL class="timelineMinor"><DT id="02w06"><A>Week 2</A></DT><DD class="timelineEvent" id="02w06EX" style="display:none;"><P>Grew <I>E. coli</I> containing SEVA plasmid 256, 581 and 434 in LB medium with correct antibiotic. Miniprepped them the day after.  Want to create SEVA plasmid that has pBBR1 ori, <I>lacIq-Ptrc</I> promoter and km or tet resistance. SEVA plasmids were digest with PacI and SpeI in order to get the cargo out, want to change the cargo and insert the correct cargo which is the lacIq-Ptrc (in SEVA 434) to backbones from 256 (Km resistant)0 and 581 (tet resistant). For SEVA 256 and 581 alkaline phosphatase was added in the digestion mixture. </P><FIGURE><FIGCAPTION>SEVA plasmid</FIGCAPTION></FIGURE><P>
	PCR was done with plasmid SEVA 434 using Q5 poly from NEB with primers:
T<SUB>0</SUB> term rev: 	5’-CTGGATTCTCACCAATAAAAAACG-3’
T<SUB>1</SUB> term fw: 	5’-TCTAGGGCGGCGGAT-3’
T<SUB>m</SUB> was 60°C, expected band of around 1700 bp 
Ran 5 μl of it through gel with a 1 kb plus GeneRuler ladder.
</P><FIGURE><FIGCAPTION>lacIq-Ptrc PCR using SEVA 434 as template</FIGCAPTION></FIGURE><P>
PCR purified rest of the PCR samples. Digested lacIq-Ptrc with PacI and SpeI then ligated with SEVA 581 (tet) and SEVA256 (km)  backbones. Did overnight ligation then afterwards ligase was deactivated by putting ligation mixture in 70°C for 10 min. 
</P></DD></DL><DL class="timelineMinor"><DT id="02w07"><A>Week 3</A></DT><DD class="timelineEvent" id="02w07EX" style="display:none;"><P>PCR phlABCDE gene from <I>Psuedomonas protegens</I> pf-5 with Q5 polymerase and a tm of 63°C.
Primers are: 
Fw: 5’-GTTTCTTCGAATTCGCGGCCGCTTCTAGAGTGGAATCAAGAGGAAAATGAACGTG-3’
Rev: 5’-GTTTCTTCCTGCAGCGGCCGCTACTAGTATTATTACTAGTCCTTCAGGGGCAAG-3’
Expected a band of around 5.5 kb. Gel was ran with a 2-log NEB ladder.</P><FIGURE><FIGCAPTION>PCR <I>phl (DAPG)</I> gene cluster</FIGCAPTION></FIGURE><P>Transformed commercially competent <I>E. coli</I> cells with ligation mix that containing:</P><UL><LI>LacIq digested with PacI &amp; SpeI</LI><LI>SEVA 256 digested with PacI &amp; SpeI</LI></UL><P>Grew a couple of transformants in 10 ml LB, miniprepped them the day after. Digested the plasmids with PacI &amp; SpeI. Expect a band of 1.5 kb and 5.5 kb. Ran gel with a 2-log NEB ladder.</P><FIGURE><FIGCAPTION>SEVA transformants digest.</FIGCAPTION></FIGURE>
All transformants seemed to be correct, took #2 to be shuttle vector. Renamed it SEVA 254m contains a kanymcin resistance.
<P>
Digested the following:<UL><LI>SEVA 254 with EcoRI &amp; SpeI</LI><LI>SEVA 254 with xbaI &amp; SpeI</LI><LI>Methionine-γ-lyase with EcoRI &amp; SpeI</LI><LI>PfrI with EcoRI &amp; SpeI</LI><LI>PhlABCDE with xbaI &amp; SpeI</LI></UL></P><P>Ran the digestions thru gel with a 2-log NEB ladder. <I>phlABCDE</I> was not run through gel because it was digested using a PCR product and not a plasmid. </P><FIGURE><FIGCAPTION>Gel extracted SEVA 254 from gel and column purified MgL and PfrI digestions. MgL and PfrI was column purified in order to increase a higher final concentration of DNA in the end. </FIGCAPTION></FIGURE><P>
Ligation was done for the following: <UL><LI>SEVA 254 digested with EcoRI &amp; SpeI + Methionine-γ-lyase digested with EcoRI &amp; SpeI</LI><LI>SEVA 254 digested with xbaI &amp; SpeI + PhlABCDE digested with xbaI &amp; SpeI</LI><LI>SEVA 254 digested with EcoRI &amp; SpeI + PfrI  digested with EcoRI &amp; SpeI</LI></UL></P><P>
Transformed in commercially competent <I>E. coli</I> cell using the ligations that were done above. 
Chitinase sequence transformation to pSB1A3 into <I>E. coli</I> (unsuccesful attempt).</P></DD></DL><DL class="timelineMinor"><DT id="02w08"><A>Week 4</A></DT><DD class="timelineEvent" id="02w08EX" style="display:none;"><P>Transformation of SEVA 254 containing genes was successful. Did a colony PCR for Methionine-γ-lyase and <I>PfrI</I> transformants. 
Using same primers that was used in order to amplify the gene. Used Taq polymerase with tm of 64°C. Ran PCR with a 2-log NEB ladder. Upper lane is Methionine-γ-lyase and lower lane is <I>PfrI</I>. Positive and negative control are indicated as + and – respectively. + control was for Methionine-γ-lyase <I>B. linens</I> and <I>PfrI</I> was <I>P. putida</I> KT2440.</P><FIGURE><FIGCAPTION>pfri colony PCR</FIGCAPTION></FIGURE><P>Made glycerol stocks for #4 and 5 of Methionine-γ-lyase and pfrI.
Miniprepped and digest <I>phlABCDE</I> in SEVA 254 with xbaI and SpeI. If it cuts, then gene was inserted in the correct direction. Ran gel with 2-log NEB ladder.</P><FIGURE><FIGCAPTION>SEVA254 DAPG digest</FIGCAPTION></FIGURE>
Took #2 and made glycerol stocks.
<P>
Transformed P.putida KT2440 using electroporation:</P><UL><LI>SEVA 254 phlACDE</LI><LI>SEVA 254 methionine-γ-lyase</LI><LI>SEVA 254 PfrI</LI></UL><P>Found out that methionine-γ-lyase gene has frameshift and then was then ordered synthetically. </P></DD></DL><DL class="timelineMinor"><DT id="02w09"><A>Week 5</A></DT><DD class="timelineEvent" id="02w09EX" style="display:none;"><P>Did colony PCR with <I>P. putida</I> transformants containing SEVA 254 <I>PfrI</I>. Ran PCR with 2-log NEB ladder. + control was a SEVA 254 <I>PfrI</I> plasmid.</P></DD></DL></DIV><DIV class="timelineMajor"><H2 class="timelineMajorMarker"><SPAN>August</SPAN></H2><DL class="timelineMinor"><DT id="02w10"><A>Week 1</A></DT><DD class="timelineEvent" id="02w10EX" style="display:none;"><P>Did colony PCR with <I>P. putida</I> transformants containing SEVA 254 <I>phlABCDE</I>. Ran PCR with 2-log NEB ladder. Positive control was a SEVA 254 <I>phlABCDE</I> plasmid. Used A the reverse primer of the gene a forward primer that it in the end gene where the product forms is around 1 kb. Annealing temperature used was: 52°C .These primes were chosen because we had troubles with colony PCR-ing &gt;5kb. 
Primers used:
Fw: 5’-GTGCTGGTGACCTCGATTGC-3’
Rev:5’-GTTTCTTCCTGCAGCGGCCGCTACTAGTATTATTA CTAGTCCTTCAGGGGCAAG-3’</P><FIGURE><FIGCAPTION>Colony PCR DAPG</FIGCAPTION></FIGURE>
Continued with number 6.

<P>Chitinase sequence transformation to pSB1A3 into <I>E. coli</I> (unsuccesful attempt).
Primers for colony PCR:
FW: VF2 primer
Rev:5'-AGGTGCTGCAGGAGCGTCATGGCTGA
and
FW: 5'GTGGATAAGATCGATTTTGCATCAA
Rev:5'-AGGTGCTGCAGGAGCGTCATGGCTGA
</P></DD></DL><DL class="timelineMinor"><DT id="02w11"><A>Week 2</A></DT><DD class="timelineEvent" id="02w11EX" style="display:none;"><P>Set up HPLC machine in order to detect 2,4-DAPG. 
Ran 2,4-DAPG standard in HPLC. ALso did a colony PCR with PfrI tranformants
</P><FIGURE><FIGCAPTION>Colony PCR pfri</FIGCAPTION></FIGURE>
Continued with number 3 and 4.
                                                     </DD></DL><DL class="timelineMinor"><DT id="02w12"><A>Week 3</A></DT><DD class="timelineEvent" id="02w12EX" style="display:none;"><P>Synethetic methionine-γ-lyase arrived. Grew transformed plasmid to <I>E. coli</I>. Grew transformants and mini-prepped the day after. 
Digested the following:</P><UL><LI>SEVA 254 EcoRI &amp; PstI</LI><LI>synthetic methionine-γ-lyase in IDT vector EcoRI &amp; PstI</LI></UL><FIGURE><FIGCAPTION>Colony PCR SEVA Methionine-gamma-lyase(MgL)</FIGCAPTION></FIGURE>
 Lower band of MgL and backbone band were gel extracted and then ligated together. Transformed in <I>E. coli</I>.  Ran a colony PCR (Taq pol) with the only transformant and ran gel with 2-log NEB ladder. + control was methionine-γ-lyase in IDT vector.
 <FIGURE><FIGCAPTION>Digest SEVA MgL</FIGCAPTION></FIGURE>
Transformant is correct, grew it in liq LB, mini-prepped day after. Transformed in <I>P. putida</I> KT2440. 

<P>Chitinase transformation into <I>P. putida</I> in pSEVA254 (succesfully). <FIGURE><FIGCAPTION>SnapGene scheme</FIGCAPTION></FIGURE><FIGURE><FIGCAPTION>PCR colony of <I>P. putida</I>. Expected bands were ~750 bp which were prooved by sequencing.</FIGCAPTION></FIGURE></P></DD></DL><DL class="timelineMinor"><DT id="02w13"><A>Week 4</A></DT><DD class="timelineEvent" id="02w13EX" style="display:none;"><P>Did Pyoverdine growth experiments (see protocols).
														</P></DD></DL></DIV><DIV class="timelineMajor"><H2 class="timelineMajorMarker"><SPAN>September</SPAN></H2><DL class="timelineMinor"><DT id="02w14"><A>Week 1</A></DT><DD class="timelineEvent" id="02w14EX" style="display:none;"><H3>Greenhouse</H3><P>Banana plants were transfered to soil.
													</P><H3>Fungal inhibition</H3><P>Did a growth experiment with <I>phl P. putida</I> transformants (contains <I>phl</I> gene cluster in order to produce 2,4-DAPG) in 100ml medium in 1L erlenmeyer flasks.  Done in duplo’s.</P><UL><LI>WT <I>P. putida</I> KT2440</LI><LI><I>P. protegens</I> pf-5</LI><LI><I>P. putida</I> containing <I>phl</I> gene cluster (0 mM IPTG)</LI><LI><I>P. putida</I> containing <I>phl</I>  gene cluster (0.1mM IPTG)</LI><LI><I>P. putida</I> containing <I>phl</I>  gene cluster (1mM IPTG)</LI><LI><I>E. coli</I> DH5-α containing <I>phl</I>  gene cluster (1mM IPTG)</LI></UL><P>More information can be found in protocols (link to protocols). Sampled 40ml at t=24h and t=48h.
Extracted and concentrated samples using ethyl acetate.
Digested synthetic Methionine-γ-lyase in IDT best fit and psb1C3 vector with EcoRI and PstI. Purified digestion and ligated overnight. 
Transformed plasmids <I>Pfri</I> (psb1C3) and Methionine-γ-lyase (psb1C3) to <I>E. coli</I> DH5-α commercial competent cells. 
Also transformed SEVA 254 containing synthetic Methionine-γ-lyase to <I>P. putida</I> KT2440.

Chitinase sequence transformation to pSB1A3 into <I>E. coli</I> (unsuccesful attempt).</P></DD></DL><DL class="timelineMinor"><DT id="02w15"><A>Week 2</A></DT><DD class="timelineEvent" id="02w15EX" style="display:none;"><P>Ran a colony PCR with <I>E. coli</I> transforamants with <I>PfrI</I> (psb1C3) and methionine-γ-lyase (psb1C3). Ran gel with 2-log NEB ladder. </P><FIGURE><FIGCAPTION>Mgl, PfrI colony PCR</FIGCAPTION></FIGURE>
Upper lane is from methionine-γ-lyase (psb1C3) and lower lane is <I>PfrI</I> (psb1C3). Expected around 1.5kb for methionine-γ-lyase and around 600bp for <I>PfrI</I>. Numbers 1 and 2 were continued with both methionine-γ-lyase (psb1C3) and <I>PfrI</I> (psb1C3).
														<P>Grew and miniprepped these transformants. methionine-γ-lyase (psb1C3) plasmids were digested with EcoRI and PstI. Ran gel with 2-log NEB ladder. Expected a 1.5kb band (insert) with a 2.2 kb backbone.</P><FIGURE><FIGCAPTION>MgL EcoRI and PstI digestion</FIGCAPTION></FIGURE>
Both plasmids are correct with 1.5kb insert band and a 2.2kb backbone, 3-4kb band are uncut plasmids.
	
<P>Also did a mutagenesis with <I>PfrI</I> (psb1C3) plasmid with Q5 polymerase using primers: 
Fw:   5’-CGATTCACCATTACTGCAAGCCTTCGTCGACAAC-3’
Rev: 5’-GTTGTCGACGAAGGCTTGCAGTAATGGTGAATCG-3’
Afterwards ran a gel for the mutagenesis. </P><FIGURE><FIGCAPTION>mutagenesis</FIGCAPTION></FIGURE>
Both plasmids are correct with 1.5kb insert band and a 2.2kb backbone, 3-4kb band are uncut plasmids.

<P>Mutagenesis was succesfull with expected band size of 2.8kb and nothing on negative control. Afterwards added DpnI to purified mutagenesis plasmid and transformed into <I>E. coli</I> competent cells. 
Transformed <I>P. putida</I> with an empty SEVA 254 plasmid. 

Also repeated 2,4-DAPG growth experiment in 100ml with 1L erlenmeyers. This time with:</P><UL><LI>1x <I>E. coli</I> wtih <I>phl</I> gene</LI><LI>2x DH5-α</LI></UL><P>Sampled 40 ml at t=24h and t=48h. Extracted and concentrated with ethylacetate. 
In addition grew wild <I>P. putida</I> KT2440 on 20μg/ml Kanamycin to see if a lower antibiotic was possible, and there was no growth in 20μg/ml Kanamycin so it is possible to select <I>P. putida</I> with 20μg/ml Kanamycin.
Grew in duplo’s the following on LB agar plates in order to get 2,4-DAPG production with +/- anitibiotic and 1mM IPTG. </P><UL><LI>WT <I>P. putida</I> KT2440</LI><LI><I>P. putida</I> with <I>phl</I> gene cluster</LI><LI><I>P. protegens</I> pf-5</LI><LI>DH5-α</LI><LI><I>E. coli</I> with <I>phl</I> gene</LI></UL><P>Grew them all for 48 hours before harvesting the agar plates.

Directed mutagenesis of Chitinase in pSEVA254 performed 3 times with different annealing temperatures (60°C, 50°C and 70°C) and different times of elongation (30 seconds per kb - 50 seconds per kb). None of them worked.</P></DD></DL><DL class="timelineMinor"><DT id="02w16"><A>Week 3</A></DT><DD class="timelineEvent" id="02w16EX" style="display:none;"><H3>Greenhouse</H3><P>Banana plants were transfered to bigger pots.
													</P><H3>Fungal inhibition</H3><P>Cut agar in little pieces and extracted first with acetone and then with ethyl acetate. Prepared all HPLC samples and ran them in HPLC. Check <A href=" https://static.igem.org/mediawiki/2014/0/07/Wageningen_UR_protocols_DAPGprotocols.pdf " class="soft_link">protocols</A> for more details.</P><P>Co-inoculated an in vivo agar assay with <I>P. putida</I> and <I>Fusarium oxysporum cubense (foc)</I>  TR4 on LB agar (in duplo’s)</P><UL><LI>WT <I>P. putida</I> KT2440 +<I>foc</I></LI><LI><I>P. putida</I> (<I>phl</I> gene cluster)+ <I>foc</I></LI><LI><I>P. putida</I> (PfrI) + <I>foc</I></LI></UL><P>Did a growth experiment using P.putida with Methionine-γ-lyase in 50ml liquid medium (250ml erlenmeyer flask).  Cultures were induced ~0.5 OD600 and cells were harvested after 3 hours. Enzymes were isolated using sonfication. Check <A href=" https://static.igem.org/mediawiki/2014/4/4b/Wageningen_UR_protocols_MgLgrowth.pdf" class="soft_link">protocols</A> for more details. 
The following strains were grown (in duplo’s):</P><UL><LI>WT <I>P. putida</I> KT2440 </LI><LI><I>P. putida </I>with M-γ-lyase (1mM IPTG) </LI><LI><I>P. putida </I>with M-γ-lyase (0.1mM IPTG) </LI><LI><I>P. putida</I> with M-γ-lyase (0mM IPTG) </LI></UL><P>Chitinase sequence transformation to pSB1C3 into <I>E. coli</I> (unsuccesful attempt).</P></DD></DL><DL class="timelineMinor"><DT id="02w17"><A>Week 4</A></DT><DD class="timelineEvent" id="02w17EX" style="display:none;"><P>Plated 2,4-DAPG and DMDS on komada plates with concentrations being and inoculated <I>Fusarium</I> TR4 on these plates. Also co-inoculated <I>P. putida</I> and <I>Fusarium (foc)</I>.

Did a pyoverdine growth experiment.</P></DD></DL></DIV><DIV class="timelineMajor"><H2 class="timelineMajorMarker"><SPAN>October</SPAN></H2><DL class="timelineMinor"><DT id="02w18"><A>Week 1</A></DT><DD class="timelineEvent" id="02w18EX" style="display:none;"><H3>Greenhouse</H3><P>Banana plants were inoculated with 5 ml (OD600 of 0.25) of Pseudomonas putida expressing fungal growth inhibitors.
												 2 days later banana plants were inoculated with 5 ml (1 mio spores per ml) of Fusarium and 2 inoculated maize kernels per plant.
													</P><H3>Fungal inhibition</H3><P>Did a pyoverdine growth experiment. And also co-inoculated <I>P. putida</I> with <I>Fusarium</I> on agar plates for in vivo assay.</P></DD></DL><DL class="timelineMinor"><DT id="02w19"><A>Week 2</A></DT><DD class="timelineEvent" id="02w19EX" style="display:none;"><P>Plated 2,4-DAPG and DMDS on komada plates with concentrations being and inoculated <I>Fusarium</I> TR4 on these plates. Also co-inoculated <I>P. putida</I> and <I>Fusarium (foc)</I>.

Did a pyoverdine growth experiment.
Performed a Chitinase growth experiment in <I>P. putida</I>.</P></DD></DL><DL class="timelineMinor"><DT id="02w20"><A>Week 3</A></DT><DD class="timelineEvent" id="02w20EX" style="display:none;"><P>In co-inoculated more <I>Fusarium</I> with <I>P. putida</I>  on agar plates. Prepared inoculum for green house experiments. Inoculated them in the morning with overnight cultures, induced them at OD ~0.5, harvest after 4 hours and afterwards dilute them with LB medium to an OD of 0.25.
Did a colony PCR with methionine-γ-lyase <I>P. putida</I> transformants, these were done with primers: 
FW: 5’- TGTGGAATTGTGAGCGGATA-3’
Rev: 5’- GGTCAGCGATAGGCACCTC-3’

Found out that tranformants used for enzyme isolation in previous growth experiments were correct.
</P></DD></DL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV id="footer"><P>iGEM Wageningen UR 2014. Send us an <A href="mailto:igemwageningen@gmail.com" class="soft_link">email</A>, or follow us on <A class="soft_link" href="http://twitter.com/igemwageningen" target="_blank">Twitter</A> and <A class="soft_link" href="http://www.facebook.com/iGEMWageningen" target="_blank">Facebook</A>!
			
			For more information about iGEM, check out their <A class="soft_link" href="https://igem.org">website</A> and <A class="soft_link" href="http://twitter.com/igem">Twitter</A>!</P></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Wageningen_UR/notebook/journal/inhibition">http://2014.igem.org/Team:Wageningen_UR/notebook/journal/inhibition</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Wageningen_UR/notebook/journal/inhibition" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Wageningen_UR/notebook/journal/inhibition" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Wageningen_UR/notebook/journal/inhibition&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Wageningen_UR/notebook/journal/inhibition&amp;oldid=400079" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>