<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Wageningen_UR_notebook_journal_resistance"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Wageningen_UR/notebook/journal/resistance">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Wageningen_UR/notebook/journal/resistance&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Wageningen_UR/notebook/journal/resistance&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Wageningen_UR/notebook/journal/resistance&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Wageningen_UR/notebook/journal/resistance" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Wageningen UR/notebook/journal/resistance</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><P><TITLE>Wageningen UR iGEM 2014 </TITLE></P><DIV id="wrapper"><DIV id="innerwrap"><DIV id="left"><P> </P><DIV id="inner-left"><P> </P><DIV class="sidebar-container"><P> </P></DIV></DIV></DIV><DIV id="center"><DIV class="navmenubkg"><DIV id="nav_wrapper"><DIV id="ca-menu"><UL><LI><A href="https://2014.igem.org/Team:Wageningen_UR/overview"><DIV class="ca-content">Overview</DIV></A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/overview/background">Background</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/overview/solution">Solution</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/overview/results">Results</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/overview/achievements">Achievements</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/overview/parts">BioBricks</A></LI></UL><LI><A href="https://2014.igem.org/Team:Wageningen_UR/project"><DIV class="ca-content">Project</DIV></A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/project/fungal_sensing">Fungal Sensing</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/project/fungal_inhibition">Fungal Inhibition</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/project/fungal_inhibition" class="up_menu_fix">Wetlab</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/project/greenhouse">Greenhouse</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/project/kill-switch">Kill-Switch</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/project/kill-switch" class="up_menu_fix">Kill-Switch</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/project/characterization">Characterization</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/project/gene_transfer">Gene Transfer</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/project/gene_transfer" class="up_menu_fix">Gene Transfer</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/project/low_copy">Low Copy Plasmids</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/project/model_overview">Model</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/project/model#design1" class="up_menu_fix">Promoter Design</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/project/model#cost1">System Cost</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/project/model#performance1">System Stability</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/project/interlab">Interlab Study</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/outreach"><DIV class="ca-content">Outreach</DIV></A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/outreach/policy">Policy</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/outreach/collaboration">Collaboration</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/outreach/synenergene">Synenergene</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/outreach/interviews">Interviews</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/outreach/meetup">National Meetup</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/outreach/media">Media</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/team"><DIV class="ca-content">Team</DIV></A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/team/members">Members</A></LI><LI class="up_menu_fix"><A href="https://2014.igem.org/Team:Wageningen_UR/team/members#students">Students</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/team/members#supervisors">Supervisors</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/team/gallery">Gallery</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/team/attributions">Attributions</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/team/acknowledgements">Acknowledgements</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/team/contact">Contact</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/safety"><DIV class="ca-content">Safety</DIV></A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/safety/environment">Environment</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/safety/laboratory">Laboratory</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/notebook"><DIV class="ca-content">Notebook</DIV></A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/notebook/journal">Journals</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/notebook/protocols">Protocols</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/notebook/links">Interesting Links</A></LI></DIV></DIV></DIV><DIV class="center_content"><H1>Fungal resistance journal</H1><P>All transformations were done in <I>E. coli</I>. Preparation of the backbones was done in DH5alfa, while for transformations with the clusters JM109 was used.</P><DIV class="journal"><DIV class="container" style="margin-top:100px;"><DIV id="timelineContainer" class="timelineContainer"><DIV class="timelineToggle"><P><A class="expandAll">+ expand all</A></P></DIV><DIV class="timelineMajor"><H2 class="timelineMajorMarker"><SPAN>May</SPAN></H2><DL class="timelineMinor"><DT id="02w1"><A>Growth experiments</A></DT><DD class="timelineEvent" id="02w1EX" style="display:none;"><P>Growth experiments were conducted with fusaric acid concentrations ranging from 0 to 250 ug/ml were performed on <I>Eschericia coli (Strains DH5alfa, BL21 and JM109), Bacillus subtilis</I> and <I>Pseudomonas putida</I>. After 18 hours of incubation at 37 degress celcius, OD 600 measurements were taken.</P><FIGURE><FIGCAPTION>
                                                        Figure 1. Both <I>B. subtilis</I> and all of the <I>E. coli</I> strains show a strong response to increasing doses of fusaric acid in the medium. <I>P. putida</I>, however, does not show any decrease in growth for the concentrations used. Therefore another experiment was done with higher concentrations for <I>P. putida</I>.
                                                        </FIGCAPTION></FIGURE><P><I><B>NOTE: After ordering new fusaric acid later in the project, it was found out that the fusaric acid used in this experiment was not the same strength, with the old one most likely being (partially) degraded. Therefore data from this experiment is not used.</B></I></P></DD></DL></DIV><DIV class="timelineMajor"><H2 class="timelineMajorMarker"><SPAN>June</SPAN></H2><DL class="timelineMinor"><DT id="02w2"><A>HPLC</A></DT><DD class="timelineEvent" id="02w2EX" style="display:none;"><P>Since some soil bacteria are known to be able to degrade fusaric acid (FA), a HPLC experiment was started to test <I>P. putida</I>(PP) for fusaric acid degradation and carbon utilization. M9 media was used with glucose/fusaric acid(250ug/ml) or both as a carbon source.  </P><P>For the HPLC the following settings were used.  </P><P><UL><LI>Column: Polaris C18A</LI><LI>Eluent: Acetonitril</LI><LI>Temperature: 35 degrees Celsius</LI><LI>Flow speed: 0.5 ml/min</LI><LI>Detection: UV (260 - 280nm)</LI></UL></P><FIGURE><FIGCAPTION>
                                                        Figure 2. Left: The HPLC method used delivered a good calibration curve, but peak resolvance was not optimal, with the FA peak having overlap with several smaller peaks in the samples with <I>P. putida</I>. Therefore it was decided to repeat the experiment to improve the HPLC method. 
                                                        Right: Initial results on fusaric acid breakdown showed signs of breakdown, but because peak resolvance was not optimal and the measurements were not done in duplo more data was needed to draw conclusions.                                                      
                                                        </FIGCAPTION></FIGURE><P>Based on the initial results a more elaborate experiment was started in duplo. Six samples would be grown in duplo:</P><UL><LI>A. Negative control for growth on M9 without carbon source. (PP+, Glucose-, FA-)</LI><LI>B. Positive control for growth on M9 with carbon source (glucose).(PP+, Glucose+, FA-)</LI><LI>C. Positive control for growth on M9 with carbon source and fusaric acid. (PP+, Glucose+, FA-)</LI><LI>D. Test for PP growth on FA as carbon source.(PP+, Glucose-, FA+)</LI><LI>E. E.	Negative control for contamination and FA stability. (PP+, Glucose-, FA-)</LI><LI>F. Negative control for contamination. (PP-, Glucose+, FA-)</LI></UL></DD><DT id="02w3"><A>Growth experiments</A></DT><DD class="timelineEvent" id="02w3EX" style="display:none;"><P>Using an automated platereader, time based growth was measured on <I>P. putida</I>, to see the response to different concentrations of fusaric acid.</P><FIGURE><FIGCAPTION>
                                                        Figure 3. <I>P. putida</I> growth followed for 12 hours grown in LB medium. The OD was measured every 15 minutes at 600nm. Plates were incubated shaking at 30 degrees Celsius. Concentrations of fusaric acid are noted in the legend in ug/ml. 
                                                        At concentrations higher then 500 <I>P. putida</I> ceases to grow.
                                                        </FIGCAPTION></FIGURE></DD></DL></DIV><DIV class="timelineMajor"><H2 class="timelineMajorMarker"><SPAN>August</SPAN></H2><DL class="timelineMinor"><DT id="02w4"><A>Isolation of genes</A></DT><DD class="timelineEvent" id="02w4EX" style="display:none;"><P>All gene clusters were isolated using <A href="https://static.igem.org/mediawiki/2014/e/eb/Wageningen_UR_protocols_Pcr.pdf" target="_blank" class="soft_link">PCR</A>. The following primers were used. Temperatures were calculated using <A href="http://tmcalculator.neb.com/" class="soft_link" target="blank">NEB Tm Calculator.</A> If the initial temperature did not work, the Tm was lowered by 3 degrees and a duplo with 1 ul of DMSO was done.</P><H3><I>P. putida</I> - PP1263-1266</H3><UL><LI>FP: GTTTCTTCGAATTCGCGGCCGCTTCTAGAGATGCCGCGTCGCATCATC</LI><LI>RP: GTTTCTTCCTGCAGCGGCCGCTACTAGTATTATTATCAACCTTCGCCAGGCTTGAC</LI></UL><H3><I>Klebsiella oxycota</I> - FDT-123</H3><UL><LI>FP: GTTTCTTCGAATTCGCGGCCGCTTCTAGAGATGCTCGCCTATTACGTTGC</LI><LI>RP: GTTTCTTCCTGCAGCGGCCGCTACTAGTATTATTACTATGGGTGGATAGCCTGAGC</LI></UL><H3><I>Pseudomonas cepacia</I> - FusABCDE</H3><UL><LI>FP: GTTTCTTCGAATTCGCGGCCGCTTCTAGAGGGAGAAAATCATGCAGTCTC</LI><LI>RP: GTTTCTTCCTGCAGCGGCCGCTACTAGTATTATTATTACGACTTCTTCTGCTTGTCC</LI></UL><H3><I>Stenotrophomonas maltophilia</I> - FuaABC</H3><UL><LI>FP: GTTTCTTCGAATTCGCGGCCGCTTCTAGAGATTGATTGTCATTGTAGCGAAATTC</LI><LI>RP: GTTTCTTCCTGCAGCGGCCGCTACTAGTACGGTAATTTCCTGAACAGAC</LI></UL><P>No PCR results could be obtained for this cluster, possibly the DNA was degraded during the time spent shipping. Since the other PCRs did give results, it was chosen not to continue with this gene cluster</P></DD><DT id="02w5"><A>Preparation of backbone</A></DT><DD class="timelineEvent" id="02w5EX" style="display:none;"><P>Initially <A href="http://parts.igem.org/Part:pSB1C3" class="soft_link" target="blank">pSB1C3</A> was chosen as a backbone. To serve as a repressible promotor <A href="http://parts.igem.org/Part:BBa_J04500" class="soft_link" target="blank">J04500</A> was incorporated between the EcoRI and the XbaI site, because all the gene clusters to be integrated contain illegal PstI sites. Xba and SpeI were used to insert the gene clusters into the backbone vector.
                                                    </P><FIGURE><FIGCAPTION>Figure 4. Example of the plasmid layout. BBa_J04500 is inserted between EcoRI and XbaI while the FDT cluster is inserted between XbaI and SpeI. Standard three point ligation could not be used because of illegal PstI sites.</FIGCAPTION></FIGURE></DD></DL></DIV><DIV class="timelineMajor"><H2 class="timelineMajorMarker"><SPAN>September</SPAN></H2><DL class="timelineMinor"><DT id="02w6"><A>Transformations</A></DT><DD class="timelineEvent" id="02w6EX" style="display:none;"><P>All transformations using the fusaric acid resistance clusters proved unsuccesfull. Any colonies obtained were proven by colony PCR or restriction digestion to not contain any inserts. Since the expression of membrane proteins can often be toxic, it was considered that the leakiness of the <I>pLac</I> promoter in combination with the high copy pSB1C3 plasmid might prove toxic for our host organism. Therefore it was decided to construct low copy backbones using pSB3K3 and pSB4K5.</P></DD><DT id="02w7"><A>Preparation of low copy backbones</A></DT><DD class="timelineEvent" id="02w7EX" style="display:none;"><P>To prevent toxicity from leaky expression of potentially toxic membrane proteins low copy number backbones pSB3K3 and pSB4K5 were digested with EcoRI and XbaI to insert J04500 according to standard protocols.</P></DD><DT id="02w8"><A>FDT-123 PCR of seperate genes</A></DT><DD class="timelineEvent" id="02w7EX" style="display:none;"><P>As an alternate way to see if the proteins that were tried to be expressed were toxic, it was decided to isolate the three genes from the FDT-123 cluster seperatly. This could give insight in which of the genes would cause the potential toxicity.
													
													Primers used were:</P><UL><LI>AEX05579.1 FP - GTTTCTTCGAATTCGCGGCCGCTTCTAGAGCCACACTGGGGTTATTCGCATGAAATC</LI><LI>AEX05580.1 FP - GTTTCTTCGAATTCGCGGCCGCTTCTAGAGCGATCTGCACTGCGCGTTAC</LI><LI>AEX05580.1 RP - GTTTCTTCCTGCAGCGGCCGCTACTAGTATTATTAGGCCGCTACCGCTACTGC</LI><LI>AEX05581.1 RP - GTTTCTTCCTGCAGCGGCCGCTACTAGTATTATTATCATTTATCATGGGTGGGTTCCTCGTTC</LI></UL></DD></DL></DIV><DIV class="timelineMajor"><H2 class="timelineMajorMarker"><SPAN>October</SPAN></H2><DL class="timelineMinor"><DT id="02w9"><A>Transformations with low copy backbones</A></DT><DD class="timelineEvent" id="02w6EX" style="display:none;"><P>Using the low copy backbones transformations results were obtained. The transformation of the FDT-123 cluster in pSB4K5 initially gave positive results on colony pcr, but after miniprepping and digestion the plasmid seemed empty. Two of the transformants were sent for sequencing. Insert containing the FDT-123 FP and RP was inserted with a palindromic sequence of &quot;ATCGATGCTA&quot; or &quot;TAGCATCGAT&quot; was confirmed by sequencing. The PP cluster gave similar results but with a sequence of &quot;TTGTCCT&quot;. Since these sequences are palindromic this might be an indication that the cluster is toxic and recombination with deletion of the cluster is the only viable way to survive.</P><P>For the seperate genes of the FDT clusters, succesfull transformants were obtained for FDT-2 and FDT-3 and confirmed by sequencing. FDT-1 did not result in any succesfull tranformants leading to the conclusion that this is most likely the gene responsible for the toxicity of the cluster in <I>E. coli</I>.</P><P>Because of the problems getting the whole gene clusters expressed in combination with the high ammount of illegal restriction sites in the clusters, it was decided not to continue with the resistance 
                                                    </P></DD><DT id="02w10"><A>Further HPLC experiments</A></DT><DD class="timelineEvent" id="02w7EX" style="display:none;"><P>Several more tries were done to characterize the possible fusaric acid breakdown of <I>P. putida</I>, however, several technical issues with the HPLC were had so no further results have been obtained yet.</P></DD></DL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV id="footer"><P>iGEM Wageningen UR 2014. Send us an <A href="mailto:igemwageningen@gmail.com" class="soft_link">email</A>, or follow us on <A class="soft_link" href="http://twitter.com/igemwageningen" target="_blank">Twitter</A> and <A class="soft_link" href="http://www.facebook.com/iGEMWageningen" target="_blank">Facebook</A>!
			
			For more information about iGEM, check out their <A class="soft_link" href="https://igem.org">website</A> and <A class="soft_link" href="http://twitter.com/igem">Twitter</A>!</P></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Wageningen_UR/notebook/journal/resistance">http://2014.igem.org/Team:Wageningen_UR/notebook/journal/resistance</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Wageningen_UR/notebook/journal/resistance" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Wageningen_UR/notebook/journal/resistance" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Wageningen_UR/notebook/journal/resistance&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Wageningen_UR/notebook/journal/resistance&amp;oldid=387527" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>