"
Page
Discussion
View source
History
teams
Log in
 
Team:Wageningen UR/notebook/journal/inhibition
From 2014.igem.org
Wageningen UR iGEM 2014
 
 
 
Overview
Background
Solution
Results
Achievements
BioBricks
Project
Fungal Sensing
Fungal Inhibition
Wetlab
Greenhouse
Kill-Switch
Kill-Switch
Characterization
Gene Transfer
Gene Transfer
Low Copy Plasmids
Model
Promoter Design
System Cost
System Stability
Interlab Study
Outreach
Policy
Collaboration
Synenergene
Interviews
National Meetup
Media
Team
Members
Students
Supervisors
Gallery
Attributions
Acknowledgements
Contact
Safety
Environment
Laboratory
Notebook
Journals
Protocols
Interesting Links
Fungal inhibition and greenhouse journal
+ expand all
May
Week 4
Tranformed E. coli with BioBricks psb1C3, psb1A3, BBa_B0034. Grew transformants in 10 ml LB medium containing correct antibiotic and miniprepped them using a Genejet miniprep kit.
June
Week 1
PCR methyl-γ-lyase with NEB  Q5 polymerase.  Used as templates:
1. Camembert cheese 2.
Bevibacterium strain isolated from sponge. Ran gel with 1 kb NEB ladder. Expected product size: 1.5 kbp.
Primers:
Fw: 5’-GTTTCTTCGAATTCGCGGCCGCTTCTAGATGAGTATCACCCAGAACG-3'
Rev: 5’- GTTTCTTCCTGCAGCGGCCGCTACTAGTATTATTATTCATACCGTTGCTACAGG-3'
PCR methionine-gamma-lyase
Week 2
Colony PCR with Q5 polymerase, using P. putida colonies as template. Ran gel with 100 bp NEB ladder. Expected bands size were around 500 bp.
Primers:
Fw: 5’-GTTTCTTCGAATTCGCGGCCGCTTCTAGATGGCGGAACAACTATCCACAAGTAAG
Rev: 5’- GTTTCTTCCTGCAGCGGCCGCTACTAGTATTATTATCAGGCCTGGCGACTGGC
PCR PfrI
Extract pfrI from gel, digested it with SpeI and xbaI.  Ligated pfrI with BBa_B0034 (digested with SpeI).
Week 3-4
Transformed chemical competent E. coli cells
Ligation mix methyl-γ-lyase + BBa_B0034
Ligation mix pfrI + BBa_B0034
psb1T3
psb1K3
positive control: puc19
negative control: nothing
Psb1T3, psb1K3 transformants were grown in 10ml and then miniprepped.
Colony PCR with methyl-γ-lyase + BBa_B0034 transformant colonies using Taq polymerase. Primers were VF2 and a reverse primer from methyl-γ-lyase.  Ran gel with 1 kb NEB ladder. Is expect a band of around 1.5 kpb to appear if there was correct insertion of the product into the plasmid.
Colony PCR Methionine-gamma-lyase
Sample 1, 2, 3 and 5 seem to have the correct insert. So glycerol stocks were made for sample 1 and 2.
July
Week 1
Sequencing data showed that Metionine-γ-lyase was inserted correctly together with Bba_b0034 (RBS), pfrI + B_0034 transformants were grown in 10 ml LB containing antibiotic and miniprepped. Plasmids were digested with EcoRI & SpeI. This was done because insert could enter in 2 ways, need to check for the correct insert.  Correct insert would give us 2132 bp and 566 bp, the incorrect insert will give us 5675 bp and 23 bp. Ran it with 1 kb NEB ladder.
Schematic drawing of digestion and ligation strategy
PfrI digest with EcoRI & SpeI
Week 2
Grew E. coli containing SEVA plasmid 256, 581 and 434 in LB medium with correct antibiotic. Miniprepped them the day after.  Want to create SEVA plasmid that has pBBR1 ori, lacIq-Ptrc promoter and km or tet resistance. SEVA plasmids were digest with PacI and SpeI in order to get the cargo out, want to change the cargo and insert the correct cargo which is the lacIq-Ptrc (in SEVA 434) to backbones from 256 (Km resistant)0 and 581 (tet resistant). For SEVA 256 and 581 alkaline phosphatase was added in the digestion mixture.
SEVA plasmid
PCR was done with plasmid SEVA 434 using Q5 poly from NEB with primers:
T0 term rev:  5’-CTGGATTCTCACCAATAAAAAACG-3’
T1 term fw:  5’-TCTAGGGCGGCGGAT-3’
Tm was 60°C, expected band of around 1700 bp  Ran 5 μl of it through gel with a 1 kb plus GeneRuler ladder.
lacIq-Ptrc PCR using SEVA 434 as template
PCR purified rest of the PCR samples. Digested lacIq-Ptrc with PacI and SpeI then ligated with SEVA 581 (tet) and SEVA256 (km)  backbones. Did overnight ligation then afterwards ligase was deactivated by putting ligation mixture in 70°C for 10 min.
Week 3
PCR phlABCDE gene from Psuedomonas protegens pf-5 with Q5 polymerase and a tm of 63°C.
Primers are:
Fw: 5’-GTTTCTTCGAATTCGCGGCCGCTTCTAGAGTGGAATCAAGAGGAAAATGAACGTG-3’
Rev: 5’-GTTTCTTCCTGCAGCGGCCGCTACTAGTATTATTACTAGTCCTTCAGGGGCAAG-3’
Expected a band of around 5.5 kb. Gel was ran with a 2-log NEB ladder.
PCR phl (DAPG) gene cluster
Transformed commercially competent E. coli cells with ligation mix that containing:
LacIq digested with PacI & SpeI
SEVA 256 digested with PacI & SpeI
Grew a couple of transformants in 10 ml LB, miniprepped them the day after. Digested the plasmids with PacI & SpeI. Expect a band of 1.5 kb and 5.5 kb. Ran gel with a 2-log NEB ladder.
SEVA transformants digest.
All transformants seemed to be correct, took #2 to be shuttle vector. Renamed it SEVA 254m contains a kanymcin resistance.
Digested the following:
SEVA 254 with EcoRI & SpeI
SEVA 254 with xbaI & SpeI
Methionine-γ-lyase with EcoRI & SpeI
PfrI with EcoRI & SpeI
PhlABCDE with xbaI & SpeI
Ran the digestions thru gel with a 2-log NEB ladder. phlABCDE was not run through gel because it was digested using a PCR product and not a plasmid.
Gel extracted SEVA 254 from gel and column purified MgL and PfrI digestions. MgL and PfrI was column purified in order to increase a higher final concentration of DNA in the end.
Ligation was done for the following:
SEVA 254 digested with EcoRI & SpeI + Methionine-γ-lyase digested with EcoRI & SpeI
SEVA 254 digested with xbaI & SpeI + PhlABCDE digested with xbaI & SpeI
SEVA 254 digested with EcoRI & SpeI + PfrI  digested with EcoRI & SpeI
Transformed in commercially competent E. coli cell using the ligations that were done above.
Chitinase sequence transformation to pSB1A3 into E. coli (unsuccesful attempt).
Week 4
Transformation of SEVA 254 containing genes was successful. Did a colony PCR for Methionine-γ-lyase and PfrI transformants.
Using same primers that was used in order to amplify the gene. Used Taq polymerase with tm of 64°C. Ran PCR with a 2-log NEB ladder. Upper lane is Methionine-γ-lyase and lower lane is PfrI. Positive and negative control are indicated as + and – respectively. + control was for Methionine-γ-lyase B. linens and PfrI was P. putida KT2440.
pfri colony PCR
Made glycerol stocks for #4 and 5 of Methionine-γ-lyase and pfrI. Miniprepped and digest phlABCDE in SEVA 254 with xbaI and SpeI. If it cuts, then gene was inserted in the correct direction. Ran gel with 2-log NEB ladder.
SEVA254 DAPG digest
Took #2 and made glycerol stocks.
Transformed P.putida KT2440 using electroporation:
SEVA 254 phlACDE
SEVA 254 methionine-γ-lyase
SEVA 254 PfrI
Found out that methionine-γ-lyase gene has frameshift and then was then ordered synthetically.
Week 5
Did colony PCR with P. putida transformants containing SEVA 254 PfrI. Ran PCR with 2-log NEB ladder. + control was a SEVA 254 PfrI plasmid.
August
Week 1
Did colony PCR with P. putida transformants containing SEVA 254 phlABCDE. Ran PCR with 2-log NEB ladder. Positive control was a SEVA 254 phlABCDE plasmid. Used A the reverse primer of the gene a forward primer that it in the end gene where the product forms is around 1 kb. Annealing temperature used was: 52°C .These primes were chosen because we had troubles with colony PCR-ing >5kb.
Primers used:
Fw: 5’-GTGCTGGTGACCTCGATTGC-3’
Rev:5’-GTTTCTTCCTGCAGCGGCCGCTACTAGTATTATTA CTAGTCCTTCAGGGGCAAG-3’
Colony PCR DAPG
Continued with number 6.
Chitinase sequence transformation to pSB1A3 into E. coli (unsuccesful attempt). Primers for colony PCR:
FW: VF2 primer
Rev:5'-AGGTGCTGCAGGAGCGTCATGGCTGA
and
FW: 5'GTGGATAAGATCGATTTTGCATCAA
Rev:5'-AGGTGCTGCAGGAGCGTCATGGCTGA
Week 2
Set up HPLC machine in order to detect 2,4-DAPG.  Ran 2,4-DAPG standard in HPLC. ALso did a colony PCR with PfrI tranformants
Colony PCR pfri
Continued with number 3 and 4.
Week 3
Synethetic methionine-γ-lyase arrived. Grew transformed plasmid to E. coli. Grew transformants and mini-prepped the day after.
Digested the following:
SEVA 254 EcoRI & PstI
synthetic methionine-γ-lyase in IDT vector EcoRI & PstI
Colony PCR SEVA Methionine-gamma-lyase(MgL)
Lower band of MgL and backbone band were gel extracted and then ligated together. Transformed in E. coli.  Ran a colony PCR (Taq pol) with the only transformant and ran gel with 2-log NEB ladder. + control was methionine-γ-lyase in IDT vector.
Digest SEVA MgL
Transformant is correct, grew it in liq LB, mini-prepped day after. Transformed in P. putida KT2440.
Chitinase transformation into P. putida in pSEVA254 (succesfully).
SnapGene scheme
PCR colony of P. putida. Expected bands were ~750 bp which were prooved by sequencing.
Week 4
Did Pyoverdine growth experiments (see protocols).
September
Week 1
Greenhouse
Banana plants were transfered to soil.
Fungal inhibition
Did a growth experiment with phl P. putida transformants (contains phl gene cluster in order to produce 2,4-DAPG) in 100ml medium in 1L erlenmeyer flasks.  Done in duplo’s.
WT P. putida KT2440
P. protegens pf-5
P. putida containing phl gene cluster (0 mM IPTG)
P. putida containing phl gene cluster (0.1mM IPTG)
P. putida containing phl gene cluster (1mM IPTG)
E. coli DH5-α containing phl gene cluster (1mM IPTG)
More information can be found in protocols (link to protocols). Sampled 40ml at t=24h and t=48h. Extracted and concentrated samples using ethyl acetate.
Digested synthetic Methionine-γ-lyase in IDT best fit and psb1C3 vector with EcoRI and PstI. Purified digestion and ligated overnight.
Transformed plasmids Pfri (psb1C3) and Methionine-γ-lyase (psb1C3) to E. coli DH5-α commercial competent cells.
Also transformed SEVA 254 containing synthetic Methionine-γ-lyase to P. putida KT2440.
Chitinase sequence transformation to pSB1A3 into E. coli (unsuccesful attempt).
Week 2
Ran a colony PCR with E. coli transforamants with PfrI (psb1C3) and methionine-γ-lyase (psb1C3). Ran gel with 2-log NEB ladder.
Mgl, PfrI colony PCR
Upper lane is from methionine-γ-lyase (psb1C3) and lower lane is PfrI (psb1C3). Expected around 1.5kb for methionine-γ-lyase and around 600bp for PfrI. Numbers 1 and 2 were continued with both methionine-γ-lyase (psb1C3) and PfrI (psb1C3).
Grew and miniprepped these transformants. methionine-γ-lyase (psb1C3) plasmids were digested with EcoRI and PstI. Ran gel with 2-log NEB ladder. Expected a 1.5kb band (insert) with a 2.2 kb backbone.
MgL EcoRI and PstI digestion
Both plasmids are correct with 1.5kb insert band and a 2.2kb backbone, 3-4kb band are uncut plasmids.
Also did a mutagenesis with PfrI (psb1C3) plasmid with Q5 polymerase using primers:
Fw:   5’-CGATTCACCATTACTGCAAGCCTTCGTCGACAAC-3’
Rev: 5’-GTTGTCGACGAAGGCTTGCAGTAATGGTGAATCG-3’
Afterwards ran a gel for the mutagenesis.
mutagenesis
Both plasmids are correct with 1.5kb insert band and a 2.2kb backbone, 3-4kb band are uncut plasmids.
Mutagenesis was succesfull with expected band size of 2.8kb and nothing on negative control. Afterwards added DpnI to purified mutagenesis plasmid and transformed into E. coli competent cells.
Transformed P. putida with an empty SEVA 254 plasmid.
Also repeated 2,4-DAPG growth experiment in 100ml with 1L erlenmeyers. This time with:
1x E. coli wtih phl gene
2x DH5-α
Sampled 40 ml at t=24h and t=48h. Extracted and concentrated with ethylacetate.
In addition grew wild P. putida KT2440 on 20μg/ml Kanamycin to see if a lower antibiotic was possible, and there was no growth in 20μg/ml Kanamycin so it is possible to select P. putida with 20μg/ml Kanamycin.
Grew in duplo’s the following on LB agar plates in order to get 2,4-DAPG production with +/- anitibiotic and 1mM IPTG.
WT P. putida KT2440
P. putida with phl gene cluster
P. protegens pf-5
DH5-α
E. coli with phl gene
Grew them all for 48 hours before harvesting the agar plates.
Directed mutagenesis of Chitinase in pSEVA254 performed 3 times with different annealing temperatures (60°C, 50°C and 70°C) and different times of elongation (30 seconds per kb - 50 seconds per kb). None of them worked.
Week 3
Greenhouse
Banana plants were transfered to bigger pots.
Fungal inhibition
Cut agar in little pieces and extracted first with acetone and then with ethyl acetate. Prepared all HPLC samples and ran them in HPLC. Check protocols for more details.
Co-inoculated an in vivo agar assay with P. putida and Fusarium oxysporum cubense (foc) TR4 on LB agar (in duplo’s)
WT P. putida KT2440 +foc
P. putida (phl gene cluster)+ foc
P. putida (PfrI) + foc
Did a growth experiment using P.putida with Methionine-γ-lyase in 50ml liquid medium (250ml erlenmeyer flask).  Cultures were induced ~0.5 OD600 and cells were harvested after 3 hours. Enzymes were isolated using sonfication. Check protocols for more details.  The following strains were grown (in duplo’s):
WT P. putida KT2440
P. putida with M-γ-lyase (1mM IPTG)
P. putida with M-γ-lyase (0.1mM IPTG)
P. putida with M-γ-lyase (0mM IPTG)
Chitinase sequence transformation to pSB1C3 into E. coli (unsuccesful attempt).
Week 4
Plated 2,4-DAPG and DMDS on komada plates with concentrations being and inoculated Fusarium TR4 on these plates. Also co-inoculated P. putida and Fusarium (foc).
Did a pyoverdine growth experiment.
October
Week 1
Greenhouse
Banana plants were inoculated with 5 ml (OD600 of 0.25) of Pseudomonas putida expressing fungal growth inhibitors.              2 days later banana plants were inoculated with 5 ml (1 mio spores per ml) of Fusarium and 2 inoculated maize kernels per plant.
Fungal inhibition
Did a pyoverdine growth experiment. And also co-inoculated P. putida with Fusarium on agar plates for in vivo assay.
Week 2
Plated 2,4-DAPG and DMDS on komada plates with concentrations being and inoculated Fusarium TR4 on these plates. Also co-inoculated P. putida and Fusarium (foc).  Did a pyoverdine growth experiment.
Performed a Chitinase growth experiment in P. putida.
Week 3
In co-inoculated more Fusarium with P. putida on agar plates. Prepared inoculum for green house experiments. Inoculated them in the morning with overnight cultures, induced them at OD ~0.5, harvest after 4 hours and afterwards dilute them with LB medium to an OD of 0.25.
Did a colony PCR with methionine-γ-lyase P. putida transformants, these were done with primers:
FW: 5’- TGTGGAATTGTGAGCGGATA-3’
Rev: 5’- GGTCAGCGATAGGCACCTC-3’
Found out that tranformants used for enzyme isolation in previous growth experiments were correct.
iGEM Wageningen UR 2014. Send us an email , or follow us on Twitter and Facebook !
For more information about iGEM, check out their website and Twitter !
Retrieved from " http://2014.igem.org/Team:Wageningen_UR/notebook/journal/inhibition "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
