"
Page
Discussion
View source
History
teams
Log in
 
Team:Wageningen UR/project/fungal sensing
From 2014.igem.org
Wageningen UR iGEM 2014
 
 
 
Sensing
FA Sensing
Overview
Approach
Discussion
Future
Resistance
Approach
Results
Future
BioBricks
References
Overview
Background
Solution
Results
Achievements
BioBricks
Project
Fungal Sensing
Fungal Inhibition
Wetlab
Greenhouse
Kill-Switch
Kill-Switch
Characterization
Gene Transfer
Gene Transfer
Low Copy Plasmids
Model
Promoter Design
System Cost
System Stability
Interlab Study
Outreach
Policy
Collaboration
Synenergene
Interviews
National Meetup
Media
Team
Members
Students
Supervisors
Gallery
Attributions
Acknowledgements
Contact
Safety
Environment
Laboratory
Notebook
Journals
Protocols
Interesting Links
Fungal Sensing
When we designed BananaGuard, we wanted it to be effective, safe and specific in fighting Fusarium oxysporum f. sp. cubense (foc). This part describes how our product is specific in only targeting this fungus and not affecting other soil fungi.
The pathogenic fungus F. oxysporum secretes fusaric acid, a toxin needed to invade the banana plant via the roots. Pseudomonas putida, which serves us as a platform for the development of our biological control agent. Here the engineered Pseudomonas putida needs to be able to sense fusaric acid and activate gene expression accordingly. Resistance to fusaric acid is also crucial in this process and we found that Pseudomonas putida already possesses a good base level of resistance [1]. To be able to transfer our system to different platforms, we aim to introduce fusaric acid resistance in Escherichia coli.
Fusaric Acid Sensing
Overview
Resistance to fusaric acid is found in several microorganisms, such as Stenotrophomonas maltophilia [3] and Pseudomonas putida [4]. This resistance to the toxic compound is due to a fusaric acid efflux pump, which pumps the acid out of the cell. After doing a literature search, we found out that there are a few proteins encoded in the genome of P. putida KT2440 which should confer fusaric acid resistance based on in silico prediction. The corresponding genes are all found placed in an operon cluster starting from pp1263 to pp1266 [4]. Upstream of this region and transcribed to the opposite direction we found the pp1262 gene whose predicted function is a LysR-type regulator.
Based on the above predictions and the gene topology, we hypothesized that since fusaric acid resistance from this cluster has not been confirmed through experiments it was worth to try to check if the theoretical regulator pp1262 can control the expression of the gene cluster through a promoter in the intergenic area. Based on this hypothesis, expression of the pp1262 protein should repress transcription of the operon cluster in the absence of fusaric acid. On the other side, when fusaric acid is present, repression should be relieved. To further support that this is how the system works and that fusaric acid does not activate the regulator leading to higher transcription rates of the resistance cluster, we expect that when the regulator is overexpressed in P. putida, it causes lower fusaric acid resistance because of lower depression rate of the promoter (Figure 1). Hence, we isolated pp1262 and the intergenic region and put it into a BioBrick, effectively acting together as a fusaric acid inducible promoter (FAiP).
Figure 1. Hypothetical fusaric acid efflux pump operon present in the genome of Pseudomonas putida KT2440. When fusaric acid enters the cell, the inhibition of the efflux pump is inhibited and the efflux pump is transcribed, resulting in the excretion of fusaric acid out of the cell. For the sensing part of this iGEM project, only the detection part is used.
We decided to test and characterize the fusaric acid sensitivity and expression of this part by using different levels of fusaric acid. The promoter was tested using GFP and RFP reporter genes ( BBa_I13507 , BBa_I13504 , BBa_J23100 ) as output. Each BioBrick part contains our promoter, an RBS, a reporter gene and two terminators (Figure 2). The new constructs were expressed in P. putida to test for expression in the presence of fusaric acid.
Figure 2. The complete construct containing both our fusaric acid inducible promoter and a reporter gene.
Approach
To isolate the pp1262 gene and the downstream intergenic region, two promoters containing the RFC10 parts were used in a PCR reaction on the genomic DNA of P. putida KT2440. The product was ligated into pSB1C3 and transformed into chemically competent E. coli cells. The part was digested again and ligated upstream of several different reporter genes. BBa_I13507, which contains an RBS, mRFP and two terminators was successfully transformed. However, after sequencing it became clear that this BioBrick did not contain mRFP, making it unsuitable for this experiment . This problem has been reported to the registry. BBa_I13504 - which contains an RBS, GFP and two terminators - was used for further experiments. This reporter construct combined with our promoter part was ligated and transformed into E. coli. Sequencing confirmed the successful assembly. This construct should only show green fluorescence in the presence of fusaric acid, and no fluorescence when fusaric acid is absent.
The E. coli cells, containing the construct were grown on agar plates containing different concentrations of fusaric acid. At 140µM, there was no significant difference between the transformed cells and the control cells. At higher concentrations the cells died, due to their inability to pump the acid out of the cell. Since small amounts of fusaric acid are lethal to E. coli, this is not suitable as a host to test the expression.
The construct was transformed into a different backbone (pSB1K3) and inserted into electrocompetent P. putida cells. The transformed cells were again grown on plates containing different concentrations of fusaric acid. A slight difference in colour could be observed between the transformed cells and the wild-type cells, indicating that the promoter is only active when fusaric acid is present (see Figure 3).
Figure 3. Both plates contain 4 strains of transformed P. putida, containing the BBa_K1493003 . The right plate contains 50µg/ml kanamycin and 50µg/ml fusaric acid, whereas the left plate only contains 50µg/ml kanamycin. A slightly increased fluorescence can be observed in the right plate, indicating the promoter is induced by the fusaric acid.
A growth experiment was performed in a 96-wells plate, containing 0, 55, 140, 285, 565 and 1415µM (0, 10, 25, 50, 100 and 250µg/ml) fusaric acid and both transformed and wild-type P. putida cells. After 18h, at a concentration of 285µM fusaric acid, the transformed cells showed more fluorescence than the wild-type cells, while both strains showed no fluorescence when no fusaric acid was added, indicating that the promoter is not active when no fusaric acid is present.
The wild-type P. putida cells showed some fluorescence as well. We hypothesized that the stress caused by fusaric acid caused the excretion of additional fluorescent compounds, such as pyoverdine. Also, we found that LB medium has some auto-fluorescence. Since GFP does not include an excretion tag, we solved this problem by removing the supernatant by washing the cells with PBS and measuring the clean cells afterwards.
After washing with PBS, the wild-type cells showed no significant fluorescence, whereas the transformed cells clearly did, (see Figure 4.) indicating that the promoter is functional in this experiment.
Figure 4. Both P. putida strains are grown in LB medium with 285µM fusaric acid and washed with PBS. GFP fluorescence was checked for 6 samples of each, at 395nm excitation and 501 nm emission. This experiment shows that the promoter is active in the presence of fusaric acid. However, the value of the fluorescence is a dimentionless unit relative to the wild-type, which means a characterized promoter with a downstream GFP gene should be added to the experiment as a positive control to characterize this fusaric acid induced promoter.
Futhermore, the WT cells showed growth on all concentrations of fusaric acid, whereas the transformed cells did not survive a fusaric acid concentration of 565µM and higher (see Figure 5), which supports our hypothesis that the inserted vector decreased the fusaric acid resistance of our transformed P. putida cells.
Figure 5. Fusaric Acid does not seem to influence the growth of the wild-type P. putida, whereas the transformed cells do not survive a concentration 565µM or higher.
A new growth experiment was set up, using M9 medium instead of LB medium, to prevent fluorescent background noise of the medium. In this experiment both the WT P. putida and P. putida containing BBa_K1493000 are grown in a 96-wells plate at 0, 85, 170, 255, 340 and 425µM (0, 15, 30, 45, 60 and 75µg/ml) fusaric acid. Furthermore, E. coli DH5α cells (WT and BBa BBa_K741002 ) were also grown in the same plate but without fusaric acid. The transformed E. coli has a well characterized promoter (pLac) with the same GFP gene downstream. (See Figure 6.) By comparing the fluorescence of our fusaric acid induced promoter at different fusaric acid concentrations to this constitutive promoter, characterization is performed. pLac is used as a constitutive promoter, which has shown activity without induction of IPTG .
Figure 6.
*Significantly different from WT with p<0.05
The measurement is based on GFP fluorescence in P. putida at increased concentrations of fusaric acid to prove and characterize the activity of the fusaric acid induced promoter, BBa_K1493000 . For comparison, the well characterized pLac promoter ( BBa_K741002 , uninduced by IPTG) was used to quantify the activity of this promoter at different concentrations of fusaric acid. Our fusaric acid inducible promoter does not respond to low concentrations up to 170µM. From 255µM and up, the activity increases. The maximum measured activity of the promoter is 0.21 RPU at 425µM.
Discussion
Up to now, the regulation of this fusaric acid efflux pump was only theoretical. The literature [4,5] merely suggested a possibility in the regulation of the efflux pump yet no data is shown whatsoever but from these results the following conclusions can be drawn.
The hypothesis that the combination of the inhibiting LysR-type gene[5] and the promoter in the intergenic region is indeed fusaric acid inducible is supported by the results of the growth experiment in Figure 6. To activate the promoter, the concentration of fusaric acid has to pass a certain threshold.
To further support this hypothesis, it was observed that the transformed P. putida cells did not survive a fusaric acid concentration of 565µM or higher, likely because of over-expression of the inhibiting gene pp1262, which would not only inhibit the activity of the GFP but also of the original fusaric acid efflux pump in the original genome of P. putida, therefore decreasing the resistance of the host.
These experiments do not only give some good indications that the functioning of this promoter is indeed regulated by the induction of fusaric acid, but they also show that this regulator could actually be used for other purposes, such as in our case the detection of Fusarium oxysporum and producing anti-fungal agents in its presence. A fusaric acid detection system has not been developed before now, and the implications of this part could very likely exceed this iGEM project.
Future work
During this iGEM project, we also tried to isolate the promoter from the original P. putida KT2440 genome without the inhibiting gene, but we were unsuccessful to do so. However, while performing mutagenesis on the BioBrick to remove the illegal restriction site PstI, an unintentional insert was introduced, causing a frame shift in the inhibiting gene, possibly disabling its function, leaving the promoter uninhibited. When this part would be inserted into P. putida, the over-expression of the inhibiting gene would be gone. Note that using this part is only possible in P. putida for this reason.
By introducing this part to P. putida and growing it at the same time as the tested transformed P. putida used in the previous experiments, the fusaric acid resistance can be compared between the two strains. Hypothetically, the new strain could have a higher resistance, because the overexpression of the inhibiting gene is dysfunctional and therefore cannot block the fusaric acid efflux pump in the original genome. We did not have time to test this yet.
Resistance
Approach
Different gene cluster known to be related to fusaric acid resistance will be isolated from multiple different organisms. FusABCDE from Burkholderia cepacia [1], FDT-123 from Klebsiella oxytoca [2], FuaABC from Stenotrophomonas maltophilia [3] and PP1263-5 from Pseudomonas putida itself. These clusters will then be expressed under an IPTG inducible promoter (BBa_J04500) or a ramose inducible promoter (BBa_K914003) in order to test the resistance levels. Since all of the clusters contain multiple illegal PstI sites, it was decided to insert the cluster directly in the SpeI site of the biobrick suffix.
PP1263-5 from P. putida could be isolated from the wild type strain used at our own facilities. The original strains referred to in the research papers for the other three organisms were not available. Therefore homologues genes, with high levels of sequence identity (>98%) were identified and obtained from microbial collections.
Results
All clusters but FuaABC were successfully isolated by PCR but cloning proved to be more difficult. Initial attempts to ligate the cluster behind BBa_J04500 in psb1C3 and transform it into E. Coli yielded no results. Although colony PCR showed a positive colony in some cases of the FDT cluster, plasmids isolated from these colonies were not of the right size to contain the inserts. Sequencing one of these plasmids with the VR and VF2 primers showed insertion of a small fragment containing the end and start of the primer used to PCR the cluster and a small palindromic sequence of 8bp. (ATCGATGCTA)
Membrane proteins can potentially be toxic when expressed at a too high rate. Therefore the construct was inserted in to a low copy number backbone (pSB3K3 / pSB4K5). Additionally the three genes of the smallest cluster (FDT-123) were separately expressed to see if one of them was the cause of the possible toxicity.
Successful transformants were obtained in pSB4K5 for the whole FDT cluster, FDT-2 and FDT-3 under a pLac promoter. Once again the whole FDT cluster showed up successful on colony PCR, but failed to show the expected results when a restriction digest was done on isolated plasmids. Sequencing showed again inclusion of both the primers and a small palindromic sequence (TAGCATCGAT/ATCGATGCTA).
Given that both FDT-2 and FDT-3 were successfully cloned but the whole cluster or FDT-1 separately did not yield any results,  FDT-1 , which has homologues proteins in all 3 clusters is the most likely cause for toxicity.
Future Work
Expression of the complete cluster has not yet been achieved. More research is required to understand the function of the different parts of the resistance cluster. Ideally functionality could be proven using a FA sensitive knockout strain of P. putida itself with which could be restored by inserting the putative resistance cluster. After achieving successful expression, more insight could be achieved using protein interaction studies or GFP tagged proteins to determine localization.
BioBricks
BBa_K1493000 , BBa_K1493002 , BBa_K1493003
Continue to Fungal Inhibition >>
References
Utsumi, R., et al., Molecular cloning and characterization of the fusaric acid-resistance gene from Pseudomonas cepacia. Agricultural and biological chemistry, 1991. 55(7): p. 1913-8.
Toyoda, H., et al., DNA Sequence of Genes for Detoxification of Fusaric Acid, a Wilt-inducing Agent Produced by Fusarium Species. Journal of Phytopathology, 1991. 133(4): p. 265-277.
Hu, R.-M., et al., An Inducible Fusaric Acid Tripartite Efflux Pump Contributes to the Fusaric Acid Resistance in Stenotrophomonas maltophilia. PLoS ONE, 2012. 7(12): p. e51053.
Nelson, K.E., et al., Complete genome sequence and comparative analysis of the metabolically versatile Pseudomonas putida KT2440. Environmental microbiology, 2002. 4(12): p. 799-808.
Maddocks, S.E. and P.C.F. Oyston, Structure and function of the LysR-type transcriptional regulator (LTTR) family proteins. Microbiology (Reading, England), 2008. 154(Pt 12): p. 3609-23.
iGEM Wageningen UR 2014. Send us an email , or follow us on Twitter and Facebook !
For more information about iGEM, check out their website and Twitter !
Retrieved from " http://2014.igem.org/Team:Wageningen_UR/project/fungal_sensing "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
