<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Wageningen_UR_project_model_overview"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Wageningen_UR/project/model_overview">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Wageningen_UR/project/model_overview&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Wageningen_UR/project/model_overview&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Wageningen_UR/project/model_overview&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Wageningen_UR/project/model_overview" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Wageningen UR/project/model overview</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><P><TITLE>Wageningen UR iGEM 2014 </TITLE></P><DIV id="wrapper"><DIV id="innerwrap"><DIV id="left"><P> </P><DIV id="inner-left"><P> </P><DIV class="sidebar-container"><P> </P><SIDENAV><DIV class="menu-head"><H4><A href="#header">Modeling</A></H4></DIV><LI class="menu-item"><A href="#model_overview">Overview</A></LI><LI class="menu-item"><A href="#model_key_results">Key Results</A></LI><LI class="menu-item"><A href="#model_implications">Implications</A></LI><LI class="menu-item"><A href="https://2014.igem.org/Team:Wageningen_UR/project/model#design1">Read More</A></LI></SIDENAV></DIV></DIV></DIV><DIV id="center"><DIV class="navmenubkg"><DIV id="nav_wrapper"><DIV id="ca-menu"><UL><LI><A href="https://2014.igem.org/Team:Wageningen_UR/overview"><DIV class="ca-content">Overview</DIV></A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/overview/background">Background</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/overview/solution">Solution</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/overview/results">Results</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/overview/achievements">Achievements</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/overview/parts">BioBricks</A></LI></UL><LI><A href="https://2014.igem.org/Team:Wageningen_UR/project"><DIV class="ca-content">Project</DIV></A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/project/fungal_sensing">Fungal Sensing</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/project/fungal_inhibition">Fungal Inhibition</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/project/fungal_inhibition" class="up_menu_fix">Wetlab</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/project/greenhouse">Greenhouse</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/project/kill-switch">Kill-Switch</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/project/kill-switch" class="up_menu_fix">Kill-Switch</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/project/characterization">Characterization</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/project/gene_transfer">Gene Transfer</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/project/gene_transfer" class="up_menu_fix">Gene Transfer</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/project/low_copy">Low Copy Plasmids</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/project/model_overview">Model</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/project/model#design1" class="up_menu_fix">Promoter Design</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/project/model#cost1">System Cost</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/project/model#performance1">System Stability</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/project/interlab">Interlab Study</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/outreach"><DIV class="ca-content">Outreach</DIV></A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/outreach/policy">Policy</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/outreach/collaboration">Collaboration</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/outreach/synenergene">Synenergene</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/outreach/interviews">Interviews</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/outreach/meetup">National Meetup</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/outreach/media">Media</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/team"><DIV class="ca-content">Team</DIV></A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/team/members">Members</A></LI><LI class="up_menu_fix"><A href="https://2014.igem.org/Team:Wageningen_UR/team/members#students">Students</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/team/members#supervisors">Supervisors</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/team/gallery">Gallery</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/team/attributions">Attributions</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/team/acknowledgements">Acknowledgements</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/team/contact">Contact</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/safety"><DIV class="ca-content">Safety</DIV></A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/safety/environment">Environment</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/safety/laboratory">Laboratory</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/notebook"><DIV class="ca-content">Notebook</DIV></A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/notebook/journal">Journals</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/notebook/protocols">Protocols</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/notebook/links">Interesting Links</A></LI></DIV></DIV></DIV><DIV class="center_content"><SECTION id="model_overview"><H1>Modeling Overview</H1><P>
The performance of BananaGuard as biological control agent is defined by <B>three</B> factors: its ability to <B>destroy or inhibit</B><I>Fusarium oxysporum</I> upon detection, its ability to <B>compete</B> with other micro-organisms in the soil and the <B>stability</B> of the genetic circuit.</P><P> 

If the fungal growth inhibitors cannot destroy or inhibit <I>F. oxysporum</I>, the bananaplants will die. This aspect of BananaGuards performance is being tested <A href="https://2014.igem.org/Team:Wageningen_UR/project/fungal_inhibition" class="soft_link">in vitro</A>. This leaves two  remaining questions to be answered:</P><P><B>Can the BananaGuard system survive in the presence of competing micro-organisms?</B></P><P><B>Is the genetic circuit robust to biological fluctuations?</B></P><P>The performance and stability of the genetic circuit is crucial for BananaGuard to work. To prevent <I>F. oxysporum</I> from functioning, the fungal growth inhibitors in BananaGuard need to be produced when the fungi are present. This requires two aspects of the system to function properly: the toggle-switch within the Kill-switch system needs to be stable and the fusaric acid dependent promoter needs to activate the system in the presence of fusaric acid.

</P><P> Optimizing <A href="https://2014.igem.org/Team:Wageningen_UR/project/model#design1" class="soft_link"><B>promoter design</B></A> 
constitutes the first step of BananaGuard development. Different configurations of promoter elements on the promoters of the Kill-switch were modeled using statistical mechanics. Parameters such as repressor binding strengths and repressor production rates were changed in each simulation. By scoring Kill-switch activation, the promoter element configuration with the most simulations showing a bi-stable switch were considered to be most likely to work. Including the anti-gene transfer inhibition effect on cell growth rates and the fusaric acid dependent promoter, this optimized version of the BananaGuard system was subsequently modeled over a population of dividing cells, assessing both its stability and performance in the form of a stochastic model.  


</P><P>Running thousands of simulations incorporating <B>noise terms</B>  and <B>promoter leakiness,</B><A href="https://2014.igem.org/Team:Wageningen_UR/project/model#performance1" class="soft_link">the robustness of the BananaGuard system</A> was quantified. Stochasticity in the system caused by cell division resulted in fractions of fast and slow growing cell populations with either stable or unstable Kill-switches. The same was considered for system performance; upon <B>induction</B> by fusaric acid a portion of the cell population activated their Kill-switch or remained in the same state. In order to make the simulations as accurate as possible a parameter for the maximum growth rate of BananaGuard was required. This parameter was obtained from a metabolic model that described the effects of competition between the BananaGuard system and micro-organisms in the rhizosphere. 

</P><P>To ascertain our hosts competitiveness the rate of cell division needs to be investigated on a metabolic level. Cell growth rate is a key factor in the competition between our engineered <I>Pseudomonas putida</I> and the wildtype <I>P. putida</I> found in the rhizosphere of the banana plant roots [1,2]. The integrated genetic circuit uses metabolic resources that would otherwise be dedicated to cellular maintenance or growth. Should this divergence of resources be too large our host will be outcompeted. A genome scale <A href="https://2014.igem.org/Team:Wageningen_UR/project/model#cost1" class="soft_link">metabolic model</A> was extended, investigating and outlining the cost of introducing our synthetic genes on a metabolic level.
</P><FIGURE><FIGCAPTION>Figure 1. Schematic overview of modeling indicating overlapping parts.</FIGCAPTION></FIGURE></SECTION><SECTION id="model_key_results"><H2>Key Results</H2><H3>Promoter Design</H3><P>
Before the optimal configuration of repressor binding sites could be identified, the behaviour of the system had to be assigned scores representing a functioning or non-functioning Kill-switch. Three distinct behaviours could be identified:
</P><FIGURE><FIGCAPTION style="text-align:left">0: The Kill-switch does not work, the toggle switch is out of balance and does not function. </FIGCAPTION></FIGURE><FIGURE><FIGCAPTION style="text-align:left">1: The system performs less efficiently, though the toggle switch changes state, the toxin promoter is leaky. </FIGCAPTION></FIGURE><FIGURE><FIGCAPTION style="text-align:left">2: The system performs to design, after a fusaric acid input the toggle switch changes state and toxin is produced when  CIλ leaves the system.</FIGCAPTION></FIGURE><P>
In total eight (A,B,..,H) different configurations of repressor binding sites were tested (Figure 2). Two: F and H, were found to be moderately robust towards variations in binding strengths and production rates. H was chosen as a base to design new promoters for the Kill-switch. The practical implications for the construction of the Kill-switch were as follows: two TetR repressor binding sites on the LacI promoter, two LacI and one CIλ repressor binding site on the TetR promoter, two TetR and one CIλ repressor binding site on the toxin promoter. Based on the results of the model the experimentalists ordered oligonucleotides and assembled six promoters. Two Kill-switches were built in parallel using both the registry parts and the designed parts. The stability of the optimized BananaGuard system was modeled in the stochastic model over cell division.
</P><P><FIGURE><FIGCAPTION>Figure 2. Colormaps indicating functioning and non-functioning systems. Each letter represents different repressor binding site configurations. Each small square within the color maps represents a score for a simulation of the system with a unique set of parameters. The colors correspond to a working system (red), leaky system (green) and a non-functioning system (blue).</FIGCAPTION></FIGURE></P><P><B>More information and results of promoter design can be found <A href="https://2014.igem.org/Team:Wageningen_UR/project/model#design1" class="soft_link"><B>here</B></A></B></P><H3>Cost</H3><P>
A major concern for the efficacy of BananaGuard was its ability to compete with other micro-organisms in the soil. 
To check if BananaGuard is still viable two states need to be discerned, a <B>resting state</B> and an <B>active state</B>. In its resting state the TetR protein represses the LacI promoter and thereby the production of fungal growth inhibitors. In its active state fusaric acid induces CIλ lambda production resulting in a change of state where LacI represses the TetR promoter and fungal growth inhibitors are produced. A comparison was made between BananaGuard and the wild type <I>P. putida</I> investigating both these states. The results of this 
comparison outline the impact of the synthetic pathway on its metabolism and the metabolic stress when the production of the fungal growth inhibitors has been activated. To obtain the most realistic result the carbon composition and ratio of the banana roots exudates are used. Figure 1 depicts how BananaGuard compares to the wild type P. putida depending on the carbon uptake rate. For bacteria in the rhizosphere of the banana roots a carbon uptake rate of 4 mmol gDW-1hr-1 was determined. This shows a growth rate of &gt;99% compared to the wild type for the resting state and still over 50% for the active state. This indicates that BananaGuard is not outcompeted by other rhizosphere-populating micro-organisms and that metabolic stress is not a bottleneck for the production of fungal growth inhibitors in our activated system. The average growth rate obtained in this model has been used in the stochastic model assessing the stability and performance of BananaGuard.
</P><FIGURE><FIGCAPTION><B>Figure 3.</B> The relative growth rate compared to the wild type <I>P. putida</I> for different carbon uptake rates. The realistic solution is with the banana exudates as carbon source and the other solution is with glucose as reference. The expected carbon uptake rate of <I>P. putida</I> in the rhizosphere is indicated with a transparent red colour.</FIGCAPTION></FIGURE><P><B>More information and results of the system cost can be found <A href="https://2014.igem.org/Team:Wageningen_UR/project/model#cost1" class="soft_link"><B>here</B></A></B></P><H3>Performance</H3><FIGURE><FIGCAPTION><B>Figure 4.</B> Two histrograms showing the effect of leaky promoters on the system and the performance of the system upon induction by fusaric acid.
<B>(A)</B> For a maximum growth rate corresponding to a division time of 180 minutes and a basal CIλ production of 50 nM/min or higher destabilizes the kill-switch. The population dynamics are affected. Low protein dilution due to slow growth causes the Kill-switch to leak toxin. Higher basal production rates compensate, increasing the average growth rate but also the instability. A total of 5000 simulation were run.<B>(B)</B> For a maximum growth rate of 180 minutes 98% of the Kill-switches activate, longer division times activate the cells more effectively. A total of 20000 simulations were run. </FIGCAPTION></FIGURE><P>
Two histogram plots (figure. 4) illustrate the key results of the stochastic model. The stability and distribution of populations with different basal production rates of CIλ and the activation of the system upon fusaric acid induction.
Large distribution of division times can be seen in the population. The population dynamics are effected by free toxins from the toxin-antitoxin system. For a lower maximum growth rate and therefore a lower protein dilution rate the fluctuations in the level of free toxins is attributed to more than just stochasticity. Because a toxin is included on the kill-switch its promoter is subject to leakiness when a low amount of repressor protein is present. With lower dilution rates, low amounts of CIλ are able to build up slower over time. The CIλ protein represses both the toxin and the TetR promoter (whose protein TetR also represses the toxin) on the toggle switch. This results in lower levels of TetR for which the CIλ cannot compensate over the course of its build-up. A deficiency that translates in a leaky toxin promoter causing a portion of the cells to distribute over longer division times.
<B>Figure 4.A</B> indicates that a basal CIλ production rate of 50nM or higher will be devastating to the stability of the genetic circuit in a large portion of the population whilst also shifting the average growth rate. The long division times caused by the leaking toxins, which in turn is caused by the slow CIλ build-up, results in the Kill-switch to change state. A low basal production rate of CIλ would result in faster growth and more stability.<B>Figure 4.B</B> shows that slower growth rates activate the system more efficiently because CIλ is not diluted as much. Given the growth rate obtained from the metabolic model an activation efficiency of 98% is achieved. This means that the vast majority of our the BananaGuard population will activate.
</P><P><B>More information and results about the stability and performance of the system can be found <A href="https://2014.igem.org/Team:Wageningen_UR/project/model#performance1" class="soft_link"><B>here</B></A></B></P></SECTION><SECTION id="model_implications"><H2>Implications for the experimentalist and the system</H2><P><B>Promoter Design:</B> The statistical mechanics model has led to the experimentalists decision to opt for a new set of designed promoters and build two kill-switches in parallel. The model has predicted the <A href="https://2014.igem.org/Team:Wageningen_UR/project/model#results1" class="soft_link">newly designed promoters</A> to result in a system with higher stability. 
</P><P><B>Cost:</B>
 To check if BananaGuard is still viable in its resting and active state (activated by fusaric acid), a comparison was made between BananaGuard and the wild type <I>P. Putida</I>. The results of this comparison indicates the impact of the synthetic pathway on its metabolism and the metabolic stress when fungal growth inhibitor production has been activated. The <A href="https://2014.igem.org/Team:Wageningen_UR/project/model#results2" class="soft_link">results</A> for both the resting and active state show that BananaGuard is not outcompeted by  other rhizosphere-populating microorganisms and that metabolic stress is not a bottleneck of the fungal growth inhibitor production relatively. This means that for our application it is not necessary to change the proteins in the <A href="https://2014.igem.org/Team:Wageningen_UR/project/kill-switch" class="soft_link">kill switch</A> or produce less or different <A href="https://2014.igem.org/Team:Wageningen_UR/project/fungal_inhibition" class="soft_link">fungal growth inhibitors</A>. 
The determined growth rate for BananaGuard at a doubling time of 3 h<SUP>-1</SUP> is applied in the stochastic model assessing stability and performance.
</P><P><B>Performance:</B> The <A href="https://2014.igem.org/Team:Wageningen_UR/project/model#results3" class="soft_link">results</A> have shown that leakiness of promoters on the input/output plasmid (CIλ and Zeta-toxin) can be detrimental to the performance of BananaGuard. This aspect can be improved  upon when developed further. 
</P><P>
The activation of our system is limited by the production of CIλ and not the influx of fusaric acid. The <A href=" https://2014.igem.org/Team:Wageningen_UR/project/fungal_sensing" class="soft_link"> fusaric acid detection </A>experiments  have shown the fusaric acid dependent promoter is activated with a limited amount of fusaric acid present. The activation occurs more efficiently if the cell growth rate is low. The model has provided insight on parts of the system that can be improved. Whether it be leaky promoters or the systems dependency on the build up of CIλ, alterations can be made further optimizing BananaGuard. 
</P></SECTION><P>
More information about the following models can be found here:
<UL><LI><A href="https://2014.igem.org/Team:Wageningen_UR/project/model#design1" class="soft_link"><B>Promoter Design</B></A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/project/model#cost1" class="soft_link"><B>System Cost</B></A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/project/model#performance1" class="soft_link"><B>System Performance</B></A></LI></UL></P></DIV></DIV></DIV></DIV><DIV id="footer"><P>iGEM Wageningen UR 2014. Send us an <A href="mailto:igemwageningen@gmail.com" class="soft_link">email</A>, or follow us on <A class="soft_link" href="http://twitter.com/igemwageningen" target="_blank">Twitter</A> and <A class="soft_link" href="http://www.facebook.com/iGEMWageningen" target="_blank">Facebook</A>!
			
			For more information about iGEM, check out their <A class="soft_link" href="https://igem.org">website</A> and <A class="soft_link" href="http://twitter.com/igem">Twitter</A>!</P></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Wageningen_UR/project/model_overview">http://2014.igem.org/Team:Wageningen_UR/project/model_overview</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Wageningen_UR/project/model_overview" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Wageningen_UR/project/model_overview" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Wageningen_UR/project/model_overview&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Wageningen_UR/project/model_overview&amp;oldid=400323" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>