"
Page
Discussion
View source
History
teams
Log in
 
Team:Wageningen UR/project/kill-switch
From 2014.igem.org
Wageningen UR iGEM 2014
 
 
 
Kill-Switch
Introduction
Design
Input-Output
Toggle-Switch
Promoter Design
Results
Input-Output
Toggle-Switch
Promoter Design
BioBricks
References
Overview
Background
Solution
Results
Achievements
BioBricks
Project
Fungal Sensing
Fungal Inhibition
Wetlab
Greenhouse
Kill-Switch
Kill-Switch
Characterization
Gene Transfer
Gene Transfer
Low Copy Plasmids
Model
Promoter Design
System Cost
System Stability
Interlab Study
Outreach
Policy
Collaboration
Synenergene
Interviews
National Meetup
Media
Team
Members
Students
Supervisors
Gallery
Attributions
Acknowledgements
Contact
Safety
Environment
Laboratory
Notebook
Journals
Protocols
Interesting Links
Kill-Switch
Introduction
This project aims to protect the natural balance within the soil and provides maximal safety. We engineered a regulatory system that will only activate stable expression of fungal growth inhibitors when needed and will self-destruct the bacteria when the bacterium fulfilled its purpose. With this system we reassure that there are as little GMO bacteria in the soil as possible and that they are only active when BananaGuard is needed. The Kill-switch system will reassure regulated expression by providing a stable expression of the fungal inhibitors that are activated only during exposure to fusaric acid. After exposure to fusaric acid the Kill-switch will have a stable expression of toxins that will kill the bacteria. This reassures that the bacterium removes itself when the threat of Fusarium oxysporum is gone. The designed Kill-switch system consists of two plasmids, an input/output plasmid and a toggle switch plasmid. The input/output plasmid harbours the fungal sensing sensor (input) and the toxin gene (output). The toggle switch regulates the desired gene expression.
To create such a Kill-switch, we worked on the following projects :
Design and test the Kill-switch with promoters from the registry.
Characterize promoters of the Kill-switch by a new characterization method based on a rhamnose inducible promoter.
Design, assemble and test new promoters that are repressible by multiple repressors for an improved Kill-switch based on modelling predictions .
In the end we were able to construct a functional input/output plasmid, a toggle switch that is likely to be stable and develop a new way to characterize promoters. With this new method we characterized the PR promoter with multiple operator sites: cIOR1, cIOR2 and LacOR1 ( BBa_K909012 ), which we will refer to as pCI/Lac and determined its Relative Promoter Unit (RPU). You can find the parts that we made here .
During the project there was a close collaboration with the modelling part of our project. The system was modelled and its conclusions were used to design the system and new promoters.
The Regulatory System Design
The Kill-switch regulatory system consist of a two plasmid system as can be seen in Figure 1. One will contain the input system, the L-rhamnose-inducible promoter (pRha) regulating the CIλ repressor from bacteriophage lambda (CIλ), and the output pCI/Tet regulating the toxin to kill the bacterium, currently replaced by the reporter gene gfp . The other plasmid contains a genetic toggle switch based on the principle of Gardner et al, 2000 [1].
Figure 1. The overview of the Kill-switch regulatory system showing all possible repressions and a rhamnose input on the upper left and a GFP output on the upper right. pRha is the L-rhamnose inducible promoter, pCI/Tet is a promoter repressed by CIλ and TetR, pCI/Lac is a promoter repressed by CIλ and LacI, pTet is the Tet promoter repressed by TetR.
The toggle switch functions as a memory system for the input signal. In the final system pRha will be exchanged for the fusaric acid inducible promoter and the reporter gene gfp will be replaced by a toxin that will kill the cell. We used the CIλ, TetR and LacI repressors, because they are proven to work and well characterized [2]. The promoters we used were the only promoters that could be double repressed by our selection of repressors. pRha was chosen due to its concentration specific expression and gfp due to its usefulness as reporter gene.  The function of the Kill-switch can be explained in three states. These states are shown Figure 2-4 and the the video.
Your browser does not support this video. Please watch it on YouTube or upgrade your browser.
Visualisation of the Kill-switch circuit. Watch this video on youtube.
In the first state, as shown in the pictures (2-4) and the video, only the repressor TetR will be expressed, repressing pCIλ/Tet and pTet. Initially, the toggle switch will be set to this state by Isopropyl β-D-1-thiogalactopyranoside (IPTG) inhibiting the binding of LacI to the promoter.
Figure 2. Initial state of the kill switch. LacI is not repressing the pCIλ/Lac repressible promoter, as IPTG is added for setting the toggle switch into this state. Subsequently, TetR represses the toxin production and the production of LacI.
For the final design, in the second state of the Kill-switch the fusaric acid inducible promoter is active upon F. oxysporum recognition. However, because the fusaric acid promoter was not yet avalible we replaced the fusaric acid inducible promoter by a rhamnose inducible promoter and pRha. When pRha is induced by rhamnose, repressor CIλ is expressed, switching the circuit to its second state therefore CIλ represses pCIλ/Tet and pCIλ/Lac. This leads to a switch within the toggle switch. TetR can no longer repress pTet, therefore LacI is expressed, keeping the toggle switch in its final state, in which TetR is no longer expressed.
Figure 3. The second state of the regulatory system. Rhamnose recognition leads to expression of CIλ and therefore to suppression of TetR, initiation the toggle switch to flip state. Toxin production is repressed by CIλ.
Figure 4. Toxin expression. When Rhamnose is not sensed anymore GFP expression is not repressed.
When the organism cannot sense rhamnose/fusaric acid anymore the third state of the Kill-switch is induced. In this final state the pCIλ/Tet promoter is not repressed by the CIλ repressor or the TetR repressor, leading to the expression of GFP or the toxin in the final design of the Kill-switch system.  Besides the self-destruction system the Kill-switch circuit has lots of potential applications. The regulatory properties of this system can be utilized for any genetic system that needs to memorize the input signal and express the output gene when this signal is gone. The input promoter can be changed into the sensing promoter of choice. Likewise, the output gene can be changed giving the system a great flexibility. An even stronger improvement would be the addition intercellular communication, making a collective synchronized memory possible, which enables simple control over vast populations of bacteria equipped with this system.
The Input Output Plasmid
The input/output plasmid is the plasmid that can sense rhamnose and produce GFP or toxin, see Figure 5.
Figure 5. The overview of the kill switch regulatory system. The input output plasmid is circled.
The input/output plasmid is the plasmid with the rhamnose promoter, the CIλ repressor, CItet promoter and the GFP or toxins see figure 5 and 6. It is assembled with the standard BioBrick assembly method . The end product is assembled into a low copy number plasmid to reduce the metabolic stress, enable faster reactions to input repressors, lower the GFP production so that it can be measured more accurately and to reduce the possibility that leakiness in toxin production will kill the cell. Another low copy plasmid than pSB3K3 can also be used.
Figure 6. A schematic overview of the input/output plasmid. The first light green arrow is the rhamose promoter (Prhamnose) that is activated by the presence of rhamnose in the media. The second green arrow is the tetracycline CIʎ promoter (Ptet/CI) that is repressed by the tetR repressor and the CIʎ repressor. The yellow block represents the CIʎ repressor gene, and the Green block represents the GFP gene.
Input-Output plasmid results
Figure 7 shows cells carrying the pSB3K3 plasmid containing the Kill-switch input output plasmid ( BBa_K1493560 ) that were incubated on LB agar plates containing 0.2% rhamnose and no rhamnose overnight at 37°C.
Figure 7. Plates with streaks of colonies with the input output plasmid in duplo. The top plates are plates without rhamnose and the bottom plates are plates with rhamnose.
On plates without rhamnose the construct should express GFP since the rhamnose promoter is not active so there is no repression of the promoter expressing GFP. On the plates with rhamnose colonies should not express GFP since the rhamnose promoter should be active and producing repressors that repress the promoter that produces GFP. Under UV light the colonies on the plate with no rhamnose shows green fluorescence. The colonies induced by rhamnose don’t show a green colour. From the picture in Figure 7 we can conclude that the production of GFP is repressed by the repressor protein CIλ.
Since this plasmid is also used to characterize the pCI/Tet promoter ( see rhamnose mediated characterization ) it also gives a first suggestion that this method works.
The Toggle Switch Plasmid
The toggle switch is a memory system consisting out of the PCIlac promoter , TetR gene , Tet promoter and LacI repressor , see figure 8 and 9.
Figure 8. The overview of the kill switch regulatory system. The toggle switch plasmid is circled.
It is assembled with the standard BioBrick assembly method . The end product is assembled into a low copy number plasmid. Another low copy plasmid than pSB3K3 can also be used. To test the toggle switch a GFP was added behind the TetR gene.
Figure 9. A schematic picture of the toggle switch with a reporter. This construct contains the assembled BioBricks BBa_K1493702 and BBa_K1493703 . The tetR gene (blue) codes for the tetracyclin repressor protein TetR, which binds the promoter pTet. The lacI gene (Red) codes for the lac repressor protein LacI, which binds to the lac operator site of pCI/Lac. Binding the promoter inhibits transcription downstream from the promoter sequence. LacI inhibits the expression TetR and TetR inhibits the expression of LacI and GFP. The chemical compound aTc induces pTet and IPTG induces pCI/Lac.
In the active state the promoter pTet is on, expressing the lac repressor protein LacI, which binds to pCI/Lac and inhibits the production of TetR, keeping it in the same state. Besides the lac gene, a second gene is arranged downstream of the tet promoter: the gfp gene. We expect to see high fluorescence values for the active state, since both LacI and GFP will be expressed. In the resting state only TetR will be produced, repressing pTet, thus no GFP is expressed resulting in low fluorescence. The active state can be induced by adding the chemical anhydrotetracycline (aTc). aTc is an inhibitor of TetR, which induces the toggle switch to start expressing LacI and GFP. The resting state is induced by the chemical Isopropyl β-D-1-thiogalactopyranoside (IPTG), which is an inhibitor of LacI.
Toggle-Switch results
To determine if the toggle switch is functional and the repressor proteins produced under the promoters inhibit the gene expression of each other the system was characterized. Three cultures were prepared in M9 medium in duplo, inoculated with Escherichia coli strain NEB5α carrying the toggle switch with reporter. The first containing 500 ng/ml aTc to get the cells into the active state and the second inducing the resting state by 2 mM IPTG as described by Gardner et al[1]. A third culture is grown without inducer, evoking a random state. E. coli carrying pSB3K3 with CI and lacI with no promoter was used as an auto fluorescence control. Cultures were grown overnight and fluorescence was measured the next morning using a plate reader (395 excitation, 509 emission). Cells did not grow well in M9 with 2mM IPTG, as the OD was low (OD 600 nm of 0.3) compared to OD 600nm 0.6-0.7 of the cultures grown with ATC or with no inducer, after overnight incubation.
Figure 10. The relative fluorescence unit of each toggle switch state. Fluorescence is measured in duplo of cell cultures carrying the pSB3K3 plasmid with the toggle switch construct ( BBa_K1493702 , BBa_K1493703 ) grown in M9 medium containing 500 ng/ml aTc (green), 2 mM IPTG (red) and with no inducer added to the medium (blue).
Figure 10 shows that the cultures grown in M9 with 500 ng/ml aTc have an average relative fluorescence unit of 7300. The cultures grown with 2 mM IPTG give an average fluorescence unit of 1000. The cells grown in M9 with no inducer give a fluorescence of 5200 RFU. These values indicate that the toggle switch is functional as it has a high fluorescence when grown with aTc, which means it is in its active state. The resting state is reached with low fluorescence when grown with IPTG. We can conclude from the RFU value of the cultures grown with no inducer that the culture has a mix of both states. Thus, the toggle switch is able to choose a random state and no state is significantly more preferred than the other . This is in contrast to what the model predicted.
Promoter Design
The model of the system we build predicted that the system would not be stable with the promoter configurations we had been using so far. These promoters, see figure 11, where promoters from the registry with the same operator sites that we needed, but configured in such a way that it was likely they would not work. This was predicted by the model of the kilswitch . Therefore promoters with another configuration of the operator sites of the promoter would be necessary. With the results of the model we build 6 new promoters based on the sequences of Elowiz et al 2007 [2].
Figure 11. The overview of the kill switch regulatory system. The promoters are circled .
Promoters have 3 different regions that are separated by the -34 and -10 RNA polymerase binding sites, see figure 12. In the article Elowiz et al 2007 [3] these are called the distal, core and proximal region.
Figure 12. A schematic representation of promoter design. The promoter region is divided into three sections: the distal, core and proximal region separated by the -34 and -10 RNA polymerase binding sites. The distal, core and proximal regions can contain different operator sites such as the tetR (orange), LacI (yellow) and CI (blue) operator site or no operator site (empty site). We made 6 different combinations of these operator sites providing two CI/Lac promoters, two CI/Tet promoters and tow Tet promoters.
Binding sites on the core, distal and proximal regions have different strengths, generally the core region is the strongest followed by the proximal region and finally the distal region [3] [4]. By putting repressor operator sites on different positions you can influence the strength of the repression [3, 4] and place the most important repressor sites on the strongest regions. In theory systems would be tuneable with this method. Here we give this theory a try by designing promoters for a toggle switch that have operator sites at the same places so that the toggle switch is as balanced as possible. The most stable setup was modelled and the results can be found here . The promoters from the registry where not balanced in this way, so we made new promoters with the sequences provided in the article of Elowiz et al 2007 [3]. In the article a set of different sequences for the proximal, core and distal regions was used that can be combined in different combinations. We used these sequences to design 6 different promoters in silico/. The operator configurations of the 6 promoters are listed below:
Empty site, TetR site, TetR site (-- Tet Tet)
TetR site, empty site, TetR site (Tet – Tet)
CIλ site, LacI site, LacI site (CI Lac Lac)
TetR site, CIλ site, TetR site (Tet CI Tet)
CIλ site, TetR site, TetR site (CI Tet Tet)
LacI site, CIλ site, Lac site (Lac CI Lac)
We then ordered two single stranded sequences that had an overlapping region, and used these to make a double stranded promoter with prefix and suffix so they can be assembled according to BioBrick Standard 10. These promoters where ten assembled into pSB1C3 and other plasmids to make the Kill-switch system.
Promoter Design results
We were able to assemble the promoters with GFP, to characterize their functionality after rhamnose induction. All promoters except for promoter 3 (CI Lac Lac) showed GFP expression. See also the plates depicted in Figure 13.
Figure 13. New constructed promoters upstream GFP, expressed in E. coli. DH5alfa. There are 6 plates with each tow streaks of two different colonies, except for the plate where P4 is plated, that is a streak of only one colony. where you can see streaks of different colonies with promoter 1-6 expressing GFP (P1-P6). The only promoter that does not seem to express GFP is promoter 3 (P3).
Figure 13 shows all the promoters (P1-6) with GFP on ampicillin plates exposed to UV light. On each plate are colonies that are fluorescent suggesting correct constructs in these bacteria, except for promoter 3. After sequencing all parts had the expected sequences so promoter 3 is either a very weak promoter or the sequence used is not working. Future work with a plate reader could give some more conclusive results.
Future work
Future work: In the last month we made some characterization plasmids to characterize the registry promoters and the new set of promoters, but due to a lack of time we were not able to finish them all or to characterize them all. In future work they could be finished and the new and registry promoters could be fully characterized. More experiences with this new type of characterization could also further optimize the protocol and the rhamnose concentrations that should be used. When fully developed this could become a standard method to characterize repressible promoters, since it is able to characterize non chemically inducible promoters with different levels of rhamnose. The final regulatory system was only a few steps away from completion, but is due to a lack of time not finished. In future work this system could be finished, tested and characterized. When fully finished this system could become a major basis for safety systems and regulated expression. It can be implemented as a Kill-switch as would be the case for BananaGuard or as regulator for production in a bioreactor where the bacteria only produce products when the rhamnose is cleared out of the media.
Parts summary
New promoter parts
Table 1. promoter biobricks. The first column contains the biobrick codes the second column explains the function of the biobrick and the third column schematically shows the order of the repressor operator sites within the promoter. The most right column shows a short codename for these promoters.
BioBrick code
Feature
Operator Configuration
Code
BBa_K1493801
Promoter repressed by TetR
-- Tet Tet
P1
BBa_K1493802
Promoter repressed by TetR
Tet -- Tet
P2
BBa_K1493803
Promoter repressed by CIλ and LacI
CI Lac Lac
P3
BBa_K1493804
Promoter repressed by CIλ and TetR
Tet CI Tet
P4
BBa_K1493805
Promoter repressed by CIλ and TetR
CI Tet Tet
P5
BBa_K1493806
Promoter repressed by CIλ and LacI
Lac CI Lac
P6
The promoters from the registry
Table 2. Promoters used from the registry to build the Kill-switch system that later were replaced with new promoters in the system.
BioBrick code
Feature
Used for?
BBa_K909012
Promoter repressed by LacI and CIλ
In-output plasmid
BBa_K909013
Promoter repressed by TetR and CIλ
Toggle switch plasmid
BBa_R0040
Promoter repressed by TetR
Toggle switch plasmid
Rhamnose mediated characterization parts
Table 3. All parts that were made for the rhamnose mediated characterization and send to the registry for the Kill-switch project. The first column contains the BioBrick code the second column explains the function of the BioBrick and the last column explains where it is used for in the Kill-switch project.
BioBrick code
Feature
Used for?
BBa_K1493520
Rhamnose promoter with lacI
To characterize promoters with a LacI operator
BBa_K1493540
Rhamnose promoter with CIλ and lacI
To characterize promoters with an CIλ operator and an LacI operator
BBa_K1493703
Promoter represed by CIλ and LacI with TetR
To characterize promoters an CIλ operator and an TetR operator
BBa_K1493504
Tet promoter with GFP
To characterize the Tet promoter
BBa_K1493501
Rhamnose promoter with GFP
To characterize the rhamnose promoter
BBa_K1493560
Rhamnose promoter with CIλ and pCI/Tet with GFP
To characterize the pCI/Tet promoter for the CIλ repressor
BBa_K1493530
Rhamnose promoter with TetR
To characterize promoters with a TetR operator
BBa_K1493502
pCI/Lac with GFP
To characterize the pCI/Lac promoter
BBa_K1493503
pCI/Tet with GFP
To characterize the pCI/Tet promoter
BBa_K1493510
Rhamnose promoter with CIλ
To characterize promoters with a CIλ operator
BBa_K1493701
pCI/Lac with TetR and GFP
To characterize the toggle switch
BBa_K1493702
Tet promoter with LacI and GFP
To characterize the toggle switch
Parts we used
Table 4. All parts that we used from the registry in the Kill-switch project. The first column contains the BioBrick code, the second explains the function of the BioBrick and the last column explains where it is used for in the Kill-switch project.
BioBrick code
Feature
Used for?
BBa_K909012
Promoter repressed by LacI and CIλ
Input/output plasmid
BBa_K909013
Promoter repressed by TetR and CIλ
Toggle switch plasmid
BBa_R0040
Promoter repressed by TetR
Toggle switch plasmid
BBa_K914003
Rhamnose inducible promoter
Input/output plasmid
BBa_I13504
CIλ repressor
Input/output plasmid
BBa_P0440
TetR repressor
Toggle switch plasmid
BBa_P0412
LacI repressor
toggle switch plasmid
BBa_I13504
GFP
Reporter
BBa_I13504
CIλ
Repression of promoter CI/Tet and CI/Lac
BBa_P0440
TetR
Repression of promoter CI/Tet and pTet
BBa_R0010
Promoter that is repressed by LacI
Promoter reference
Continue to Characterization >>
References
Gardner, T.S., C.R. Cantor, and J.J. Collins, Construction of a genetic toggle switch in Escherichia coli. Nature, 2000. 403(6767): p. 339-42.
Mitchel Lewis (2005) The Lac repressor. C. R. Biologies 328 (2005) 521–548.
Cox, R.S., 3rd, M.G. Surette, and M.B. Elowitz, Programming gene expression with combinatorial promoters. Mol Syst Biol, 2007. 3: p. 145.
Lanzer, M. and H. Bujard, Promoters largely determine the efficiency of repressor action. Proc Natl Acad Sci U S A, 1988. 85(23): p. 8973-7.
iGEM Wageningen UR 2014. Send us an email , or follow us on Twitter and Facebook !
For more information about iGEM, check out their website and Twitter !
Retrieved from " http://2014.igem.org/Team:Wageningen_UR/project/kill-switch "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
