"
Page
Discussion
View source
History
teams
Log in
 
Team:Wageningen UR/notebook/journal/sensing
From 2014.igem.org
Wageningen UR iGEM 2014
 
 
 
Overview
Background
Solution
Results
Achievements
BioBricks
Project
Fungal Sensing
Fungal Inhibition
Wetlab
Greenhouse
Kill-Switch
Kill-Switch
Characterization
Gene Transfer
Gene Transfer
Low Copy Plasmids
Model
Promoter Design
System Cost
System Stability
Interlab Study
Outreach
Policy
Collaboration
Synenergene
Interviews
National Meetup
Media
Team
Members
Students
Supervisors
Gallery
Attributions
Acknowledgements
Contact
Safety
Environment
Laboratory
Notebook
Journals
Protocols
Interesting Links
Fungal sensing journal
+ expand all
June
Week 1-4
Performed colony PCR on P. putida with the following primer combinations:
Reaction 1: P01_RFC10_pp1262_fwd + P03_RFC10_FAdP_rev
Reaction 2: P01_RFC10_pp1262_fwd + p04_RFC10_FAdP_Delta-RBS_rev
Reaction 3: P01_RFC10_pp1262_fwd + p05_RFC10_FAdP_Delta-RBS2_rev
Reaction 4: P01_RFC10_pp1262_fwd + p06_RFC10_FAdP_Delta-RBS3_rev
Reaction 5: p02_RFC10_FAdP_fwd + p03_RFC10_FAdP_rev
Reaction 6: p02_RFC10_FAdP_fwd + p04_RFC10_FAdP_Delta-RBS_rev
Reaction 7: p02_RFC10_FAdP_fwd + p05_RFC10_FAdP_Delta-RBS2_rev
Reaction 8: p02_RFC10_FAdP_fwd + p06_RFC10_FAdP_Delta-RBS3_rev
Only Reaction 1 and 4 gave a clear result.
Performed colony PCR on P. putida with the following primer combinations:
Reaction 1: P01_RFC10_pp1262_fwd + P03_RFC10_FAdP_rev
Reaction 2: P01_RFC10_pp1262_fwd + p04_RFC10_FAdP_Delta-RBS_rev
Reaction 3: P01_RFC10_pp1262_fwd + p05_RFC10_FAdP_Delta-RBS2_rev
Reaction 4: P01_RFC10_pp1262_fwd + p06_RFC10_FAdP_Delta-RBS3_rev
Changed the Ta from 63 to 60, for better annealing. All 4 reactions gave a band in result. The product was isolated with a DNA purification kit.
July
Week 1-4
Purified PCR products were treated with the wrong restriction enzymes, resulting in a worthless ligation. PCR was done again to repeat the experiment using correct enzymes.
Performed colony PCR on P. putida with the following primer combinations:
Reaction 1: P01_RFC10_pp1262_fwd + P03_RFC10_FAdP_rev
Reaction 2: P01_RFC10_pp1262_fwd + p04_RFC10_FAdP_Delta-RBS_rev
Reaction 3: P01_RFC10_pp1262_fwd + p05_RFC10_FAdP_Delta-RBS2_rev
Reaction 4: P01_RFC10_pp1262_fwd + p06_RFC10_FAdP_Delta-RBS3_rev
All PCR reactions resulted in a positive band. The product was isolated with a DNA purification kit, using 50 ul of elution buffer.
Concentrations according to the nanodrop:
J01: 25ng/ul
J02: 15ng/ul
J03: 15ng/ul
J04: 15ng/ul
Products are restriction digested:
J01: 500/25 = 20 ul
J02: 500/15 = 33 ul
J03: 500/15 = 33 ul
J04: 500/15 = 33 ul
pSB1C3: 500/80 = 6.25 ul
Added 1 ul EcoRI, 1 ul SpeI, 5 ul buffer and MQ to a total of 50 ul.
Digested for 60 minutes at 37°C
Denatured for 20 minutes at 80°C
Put all the parts (digested and undigested) on gel. In J01-J04, no difference in band size was detectable, because the cut off part is only a few bp. The vector showed 1 band when uncut and 2 bands when cut.
Added J01-J04 to the cut vector pSB1C3 and ligated:
10 minutes at room temperature
20 minutes denature at 80°C
The ligated parts now contain either mRFP or J01-J04.
These vectors are used for chemical transformation, using a heatshock of 42°C on chemically competent E. coli cells. After 1 hours of growing in SOC medium, 35 ul and 175 ul was plated containing Cm antibiotics, then incubated at 37°C.
Picked white colonies from all plates, dipped them into PCR mix to check for inserts (same primers as before, to detect the insert itself, p01-P06). Every picked colony was grown over night in LB containing Cm antibiotics in the 37°C climate room rotating at 160 rpm.
PCR showed only inserts in 3 of the J01 colonies and 1 of the J03 colonies. Both strains are grown overnight again for a glycerol stock.
The J03 insert seemed to be mRFP, since the colony expressed red color. J01 was made into a glycerol stock.
Picked different colonies for J02-J04 from the same plates and performed colony PCR. Strains were grown in LB medium. PCR gave an inconclusiv result. PCR was repeated in the morning, with the non-red strains: Only 1 strain of J04 contained the insert.
Used miniprep to isolate the plasmids of the successful J01 and J04 strains.   Nanodropping showed a concentration of 110ng/ul and 85ng/ul, respectively. Both samples are sent for sequencing.
PCR is performed on putida, to obtain J02 and J03 once more and try to isolate those as well for transformation purposes.
Reaction 1: P01_RFC10_pp1262_fwd + p04_RFC10_FAdP_Delta-RBS_rev
Reaction 2: P01_RFC10_pp1262_fwd + p05_RFC10_FAdP_Delta-RBS2_rev
Reaction 1 and 2 will be put on gel
Inoculated 5ml + 5ul Cm with the successful J04 strain to make glycerol stock. Incubation at 37°C and 160 RPM.
J02 and J03 showed vague bands. The products were ligated and transformed into E. coli. Transformation was not successful.
The sequences of BBa_J01 and BBa_J04 showed that J01 was the right insert of pp1262 and intergenic region (including RBS). J04 had the right sequence except for an inconsistency in the promoter region. This is probably a result of a mistake in the primer design.
Cut the isolated BBa_J01 and BBa_J04 vectors with EcoRI and SpeI. Added these restriction enzymes to BBa_J23100, a biobrick containing a reporter gene, with its active promoter between E & S. By inserting the J01 or J04 promoter, the reporter should in theory only work if the added promoter is active. By using alkaline phosphatase, the chance for self-ligation became much smaller.
After using the ligated products for transformation, many white colonies formed. 4 of each were picked, PCR’ed (with VF2 and VR primers) and grown overnight in 5ml LB medium + Amp. The PCR showed inserts in 1 of each colony (1.2 and 4.1). Also made stocks of these two transformed strains.
Performed the following PCR
p01 + p04, Q5, J01+J23100
VF2 + VR, Q5, J01+J23100
p01 + p04, Q5, J04+J23100
VF2 + VR, Q5, J04+J23100
p01 + p04, Q5, BBa_J01 (positive control)
VF2 + VR, Q5, BBa_J01 (positive control)
p01 + p04, Q5 (negative control)
VF2 + VR, Q5 (negative control)
Also performed a gradient PCR for J02 and J03, ranging from Ta 56 to 66
The band of the J01 part has the same size as the band of the J01 part + mRFP part from the J23100 biobrick. This indicates something is wrong. To find out what happened, restriction ligations are performed on J01, BBa_J23100 and J01R (the J01 part in the J23100 reporter plasmid) with restriction enzymes E-S, S-P and E-S. The results of J01 and BBa_J23100 are as expected. However, the J01R gives questionable results. To gain further insight into this vector, PCR is run with all combinations of the primers VF2, VR, p01forward and p03reverse. All experiments give a band. To be sure what happened, both J01R and J04R are sent for sequencing with primers VF2 and VR.
A new approach is started, using the BBa_I13507 biobrick (RBS, mRFP and 2 terminators in a pSB1A2 plasmid). J01 and J04 are cut with E&S, I13507 with E&X. Both parts are ligated together.
August
Week 1-4
Ligation products are transformed into E. coli chemically. All plates had colonies. Picked 4 colonies of each plate, performed colony PCR and made liquid cultures. Miniprepped two overnight cultures which had an insert, according to the colony PCR insert check. The parts are sent for sequencing.
J1I13 (J01 + I13507 in pSB1A2) -> 150 ng/ul
J4I13 (J04 + I13507 in pSB1A2) -> 160 ng/ul
Innoculated J1I13 and J4I13 on plates with LB + Amp and LB + Amp + 25ul FA and put in the 37°C incubator. If the biobrick is built correctly and the promoter is indeed FA dependant, the plate with FA would only be colored red, whereas the negative control without FA would stay white. However, both plates stayed white. Still made a -80 stock of both.
Made putida kt2440 electrocompetent. Put in the following vectors:
J1I13 (put on Amp plate)
J4I13 (put on Amp plate)
BBa_1C3 (put on Cm plate, positive control)
no insert (put on Amp plate, negative control)
Made a new biobrick: BBa_I13504, a biobrick containing an RBS, RFP and two terminators. Backbone is pSB1A2.
All P. putida plates had overgrowth, including the negative control. P. putida seems to be resistant to Amp and Cm. Redid the experiment with 5x concentrated Amp concentration. Still lots of colonies on the negative control. So, a different backbone is required for putida.
Miniprepped BBa_I13504 -> 190 ng/ul.
Did a triple ligation with the following bricks:
J01/J04, E+S
I13504, X+P
pSB1K3, E+P
Gel purified all three parts, then put the J01/J04 insert, the I13504 insert and the pSB1K3 backbone together into one ligation mix. Ligated 30 minutes and O/N. Transformed the 30 minute ligation mix and put on Kan gels. No colonies. Transformed the O/N ligation mix in commercial competent cells. Put on Kan gels. No colonies. Transformed just the pSB1K3 backbone in E. coli cells and put on a Kan plate. No colonies. This indicates something is wrong with the plates or the backbone.
Decided to first make a construct with J01/J04 and I13504 in pSB1A2 (J1I4 and J4I4)
Cut J01/J04 with E+S (added Alkaline Phospathase)
Cut I13504 with E+S (column purified to remove insert)
Ligated together and transformed into E. coli. Put over weekend at room temperature. Found overgrowth. Made new Kan and Amp plates.
September
Week 1-4
Transformed J1I4 & J4I4 in E. coli, put on the new amp plates. Picked colonies of each, grew in liquid cultures containing Amp and miniprepped to obtain the plasmids J1I4 & J4I4. Grew E. coli containing these plasmids on both plates with and without 25ug/ml FA. No GFP was visible with the eye.
Started restriction ligation to make J1I4K3 & J4I4K3
J1I4 – E+S
J4I4 – E+S
pSB1K3 – E + S
Put on gel for extraction. However, J1I4 showed only 1 band at 2kb. J4I4 showed an insert of only 1.2 kb. A different restriction enzyme is used, PvuI, which is present once in the backbone. This showed that the insert was the same size as the insert, hence only 1 band was visable. Miniprepped to obtain more J4I4.
Disgested J1I4 & J4I4 with E+S, pSB1K3 with E+S+alkaline phosphatase. Ligated and chemically trnasformed E. coli with the ligation product on Kan plates O/N. After mini-prepping, transformed into putida. When put on plates with both FA and no FA, the plate with FA showed a little more fluorescence. Growth experiments are required to investigate this further.
iGEM Wageningen UR 2014. Send us an email , or follow us on Twitter and Facebook !
For more information about iGEM, check out their website and Twitter !
Retrieved from " http://2014.igem.org/Team:Wageningen_UR/notebook/journal/sensing "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
