<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Wageningen_UR_project_fungal_inhibition"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Wageningen_UR/project/fungal_inhibition">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Wageningen_UR/project/fungal_inhibition&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Wageningen_UR/project/fungal_inhibition&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Wageningen_UR/project/fungal_inhibition&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Wageningen_UR/project/fungal_inhibition" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Wageningen UR/project/fungal inhibition</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><P><TITLE>Wageningen UR iGEM 2014 </TITLE></P><DIV id="wrapper"><DIV id="innerwrap"><DIV id="left"><P> </P><DIV id="inner-left"><P> </P><DIV class="sidebar-container"><P> </P><SIDENAV><DIV class="menu-head"><H4><A href="#header">Fungal  Inhibition</A></H4></DIV><LI class="menu-item"><A href="#overview">Overview</A></LI><LI class="menu-item"><A href="#results">Results</A></LI><LI class="menu-item"><A href="#conclusion">Conclusion</A></LI><LI class="menu-item"><A href="#future">Future work</A></LI><LI class="menu-item"><A href="#biobricks">BioBricks</A></LI><LI class="menu-item"><A href="#reference">References</A></LI></SIDENAV></DIV></DIV></DIV><DIV id="center"><DIV class="navmenubkg"><DIV id="nav_wrapper"><DIV id="ca-menu"><UL><LI><A href="https://2014.igem.org/Team:Wageningen_UR/overview"><DIV class="ca-content">Overview</DIV></A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/overview/background">Background</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/overview/solution">Solution</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/overview/results">Results</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/overview/achievements">Achievements</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/overview/parts">BioBricks</A></LI></UL><LI><A href="https://2014.igem.org/Team:Wageningen_UR/project"><DIV class="ca-content">Project</DIV></A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/project/fungal_sensing">Fungal Sensing</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/project/fungal_inhibition">Fungal Inhibition</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/project/fungal_inhibition" class="up_menu_fix">Wetlab</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/project/greenhouse">Greenhouse</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/project/kill-switch">Kill-Switch</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/project/kill-switch" class="up_menu_fix">Kill-Switch</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/project/characterization">Characterization</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/project/gene_transfer">Gene Transfer</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/project/gene_transfer" class="up_menu_fix">Gene Transfer</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/project/low_copy">Low Copy Plasmids</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/project/model_overview">Model</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/project/model#design1" class="up_menu_fix">Promoter Design</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/project/model#cost1">System Cost</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/project/model#performance1">System Stability</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/project/interlab">Interlab Study</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/outreach"><DIV class="ca-content">Outreach</DIV></A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/outreach/policy">Policy</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/outreach/collaboration">Collaboration</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/outreach/synenergene">Synenergene</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/outreach/interviews">Interviews</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/outreach/meetup">National Meetup</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/outreach/media">Media</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/team"><DIV class="ca-content">Team</DIV></A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/team/members">Members</A></LI><LI class="up_menu_fix"><A href="https://2014.igem.org/Team:Wageningen_UR/team/members#students">Students</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/team/members#supervisors">Supervisors</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/team/gallery">Gallery</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/team/attributions">Attributions</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/team/acknowledgements">Acknowledgements</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/team/contact">Contact</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/safety"><DIV class="ca-content">Safety</DIV></A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/safety/environment">Environment</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/safety/laboratory">Laboratory</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/notebook"><DIV class="ca-content">Notebook</DIV></A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/notebook/journal">Journals</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/notebook/protocols">Protocols</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/notebook/links">Interesting Links</A></LI></DIV></DIV></DIV><DIV class="center_content"><H1>Fungal Inhibition</H1><SECTION id="overview"><H2>Overview</H2><P>In order to inhibit <I>Fusarium oxysporum cubense</I> growth, several fungal growth inhibitors will be produced when fusaric acid is sensed. Those being 2,4-DAPG, chitinase, DMDS, DMTS and pyoverdine. 2,4-DAPG or 2,4-Diacetylphloroglucinol is an antibiotic against plant pathogens[1]. Chitinase is a lytic enzyme that breaks down fungal cell walls. Dimethyldisulfide (DMDS) and dimethyltrisulfide (DMTS) are sulfur by-products produced by <I>Pseudomonas putida</I> and have been shown to stimulate plant growth and inhibit <I>F.oxyposrum</I> respectively[2,3]. And lastly, pyoverdines are iron-chelating compounds that are produced when iron is limited in order to scavenge for iron to induce iron competition. With these fungal growth inhibitors our engineered <I>P. putida</I> will be able  to better inhibit <I>F. oxysporum</I>.
			</P><H3>2,4-Diacetylphloroglucinol(2,4-DAPG)</H3><P>2,4-DAPG, full name 2,4-diacetylphloroglucinol, is an antibiotic that is widely used in the agricultural industry against pathogens. It is a broad spectrum antibiotic that has been shown to play a key role in the biological control of various plant pathogens including <I>F. oxysporum</I>[4]. In addition to that, 2,4-DAPG has also shown to induce systematic resistance in plants[5]. Since <I>P. putida</I> does not produce DAPG by itself, a gene cluster obtained from <I>Pseudomonas fluorescens</I> was introduced into <I>P. putida</I>. The <I>phl</I> gene cluster contains eight genes, from <I>phlA</I> to <I>phlH</I>. Gene cluster <I>phlABCDE</I> was used for this project as literature has been shown that <I>phlABCD</I> are the DAPG synthesis genes(see figure 1) and <I>phlE</I> codes for an efflux pump[6,7]. <I>phlABCD</I> has been expressed before in <I>P. putida</I> and has been shown to produce 2,4-DAPG[8].</P><FIGURE><FIGCAPTION style="font-size:11px;font-weight:bold">Figure 1. 2,4-DAPG synthesis pathway (Loper 2009)
			<H3>Dimethyldisulfide (DMDS) and dimethyltrisulfide (DMTS)</H3><P>When breaking down methionine for ammonium, methanethiol gets formed as a by-product. Then methanethiol gets oxidized into dimethyldisulphide (DMDS) and dimethyltrisulphide (DMTS) (see figure 2). DMTS was shown to have an inhibitory effect on <I>F. oxysporum</I> with an inhibition at the start of 78,  but then slowly declined to 26%[3]. DMDS is used as plant growth promoter and at the same time also has been shown a slight inhibition to <I>F. oxysporum</I> of 10%[2]. Both DMTS and DMDS are naturally produced in <I>P. putida</I> but a higher production is desired. One thing that causes low yield of DMDS is the low affinity of methionine-γ-lyase to methionine, which means less conversion of methanethiol, which then leads to low DMTS and DMDS production. So,  we decided to overexpress an enzyme that has a higher affinity towards methionine and will therefore increase DMDS and DMTS production, this enyzme is a methionine-γ-lyase from <I>Brevibacterium linens</I>, which has been shown to increase DMDS production in <I>Lactocossus lactis</I>[9]. This gene was codon-optimized and synthetsized by IDT for <I>P. putida</I>.</P><FIGURE><FIGCAPTION style="font-size:11px;font-weight:bold">Figure 2. Dimethyldisulfate (DMDS) and dimethyltrisulfate pathway.
			<H3>Pyoverdine</H3><P>Pyoverdines are siderophores produced by <I>P. putida</I>[10,11]. Siderophores are green/yellow fluorescent compounds that have high affinity to iron(III), which by scavenging free iron can in the end lead to iron limitation. Due to iron starvation, the growth of pathogenic fungi and bacteria in the rhizosphere will be restricted[12]. It was shown that there was a direct correlation of siderophore production and their inhibition to germination of chlamydospores of <I>F. oxysporum</I>[13]. In addition to that, siderophores have also been shown to induce resistance in radish plants[14]. However the effects of siderophore decreases when the disease incidence increases above 74%[15]. <I>P. putida</I> WCS358 is able to produce another sort of siderophore, pseudobactin 358 (PBS358), which has been shown to be involved in inhibition of <I>Fusarium</I>[16]. Pyoverdine production is iron dependent as it is regulated by a ferric uptake regulator protein (Fur)[17], see figure 4. The Fur protein inhibits expression of PfrI and PfrI is a transcription activator that is needed for activation of genes involved in pyoverdine synthesis[18]. So an overexpression was done of PfrI, which we expected to result in  pyoverdine production, even when in an iron abundant environment.</P><FIGURE><FIGCAPTION style="font-size:11px;font-weight:bold">Figure 3. Pyoverdine structure[19] 
			<FIGURE><FIGCAPTION style="font-size:11px;font-weight:bold">Figure 4. Fur regulation of siderophore genes[20] 
			<H3>Chitinase </H3><P>Chitinase is a hydrolytic enzyme that breaks down hydrolytic bonds in chitin and is produced in both bacteria and plants. Chitinase has been shown to be useful in biological control against fungi[21,22]. In bacteria their function is to attack shellfish animals or fungi, and degrade their chitin cell walls. In plants they are known as pathogen related proteins that are involved in the induced systematic resistance of plants in order to defend themselves against pathogens. <I>P. putida</I> KT2440 has a lytic enzyme  <A class="soft_link" href="http://www.uniprot.org/uniprot/Q88ID3"><I>PP3066</I></A> that is predicted to have chitinase activity, so this gene was overexpressed to increase chitinase activity.</P></FIGCAPTION><SECTION id="results"><H2>Results</H2><H3>2,4-Diacetylphloroglucinol(2,4-DAPG)</H3><P><I>PhlABCDE</I> was successfully cloned and put into plasmid pSEVA254[23], which has a kanamycin resistance, a RFS1010 ori and a <I>lacIq</I> with a P<I>trc</I> promoter. A SEVA plasmid was used instead of the iGEM plasmid because it has been shown to work in <I>P. putida</I>, it contains a LacI that controls an IPTG inducible Ptrc promoter and it has terminators behind the gene insert. Both <I>E. coli</I> and <I>P. putida</I> KT2440 were transformed with a pSEVA containing the <I>phlABCDE</I> gene cluster. This strain was further referred to as 2,4-DAPG <I>P. putida</I>. Succesfull transformants were verified via colony PCR <A class="soft_link" href="https://2014.igem.org/Team:Wageningen_UR/notebook/journal">(see journal)</A>. During colony PCR of <I>P. putida</I>, we were not able to get an expected 5.4kbp band. However, when we used a primer pair that forms a 1kbp product, we could see positive transformants. For detection of 2,4-DAPG we successfully set up a High-performance liquid chromatography (HPLC) method <A class="soft_link" href="https://static.igem.org/mediawiki/2014/0/07/Wageningen_UR_protocols_DAPGprotocols.pdf">(see protocol)</A>. Standards were detected and a standard curve was made. However, verifying production of 2,4-DAPG via HPLC turned out to be difficult due to a lot of background noise of the samples, even after an extraction step. </P><FIGURE><FIGCAPTION>Figure 5. 2,4-DAPG HPLC standard curve.
			</FIGCAPTION></FIGURE><P>In order to test the inhibition of pure 2,4-DAPG against <I>F. oxysporum</I>, an assay was done on Komada agar plates. Different concentrations of pure 2,4-DAPG were plated and then inoculated with a 5mm <I>F. oxysporum</I> plug. Plates were then incubated at 25°C for several days.</P><FIGURE><FIGCAPTION>Figure 6. Inhibition assay with pure 2,4-DAPG concentrations (0, 25, 50, 100 and 200ug/ml) inoculated with 5mm <I>F. oxysporum</I> TR4 plug. t=6 days, upper row= face down plates, lower row= face up plates .
			</FIGCAPTION></FIGURE><P>From figure 6 it can be seen that with an increase of 2,4-DAPG there is a decrease in <I>F. oxysporum</I> growth. However when it reached to 100ug/ml results were not reproducible with one plate having more growth than the other. Afterwards, another inhibition experiment was set up to test 2,4-DAPG concentration from 100-400ug/ml. However here spores (3.87E+07 spores/ml) were used as an inoculum, instead of a <I>F. oxysporum</I> plug, due to unavailability of a fresh <I>F. oxysporum</I> plug on agar plate. Plates were then incubated in 25°C for several days.</P><FIGURE><FIGCAPTION>Figure 7. Inhibition assay with pure 2,4-DAPG with <I>F. oxysporum</I>TR4 spores (3.87E+07 spores/ml) with concentration range (100,200 and 400 ug/ml), t=5 days, upper row= face down plates, lower row= face up plates.
			</FIGCAPTION></FIGURE><P>When comparing figure 6 and 7 it can be seen that when inoculated with spores even in 5 days there is absolutely no growth of <I>F. oxysporum</I> at a concentration of 100ug/ml 2,4-DAPG, whereas when for the <I>F. oxysporum</I> plug there is growth after 6 days with mixed results at the same concentration of 100ug/ml 2,4-DAPG. After testing inhibition with pure 2,4-DAPG, another inhibition experiments was done using our 2,4-DAPG <I>P. putida</I>, containing the <I>phl</I> gene cluster for 2,4-DAPG production. An overnight 2,4-DAPG <I>P. putida</I> liquid culture was spread on a LB agar plate containing 1mM IPTG for induction, grown overnight at 30°C and then inoculated with a 5mm <I>F. oxysporum</I> plug.</P><FIGURE><FIGCAPTION>Figure 8. Inhibition profiles of <I>F. oxysporum</I>TR4 by 2,4-DAPG <I>P. putida</I>. A 5mm <I>F. oxysporum</I> TR4 plug was used, tests done in duplo (upper &amp; lower row), t=7 days. Red circle indicates area occupated by <I>F. oxysporum</I> growth. Foc control=only <I>F. oxysporum</I> TR4, Wildtype= <I>P. putida</I> KT2440 and DAPG=2,4-DAPG <I>P. putida</I> containing <I>phlABCDE</I> gene cluster.
			</FIGCAPTION></FIGURE><P>In figure 8, plate pictures were modified with a red circle to better visualize the fungal disc. It can be seen that the wild type <I>P. putida</I> KT2440 already inhibits <I>F. oxysporum</I> very well. Our 2,4-DAPG <I>P. putida</I> seems to show a slightly smaller <I>F. oxysporum</I> disc when comparing it with the wild type. It is very hard to see a big difference between the wild type and our 2,4-DAPG <I>P. putida</I>, as the wild type has been shown to already inhibit <I>F. oxysporum</I> greatly when comparing wild type and <I>foc</I> control. Please note that <I>phlABCDE</I> was not yet made into the BioBrick standard, due to too many illegal sites and insufficient time to remove them.</P><H3>Dimethyldisulfide (DMDS) and dimethyltrisulfide (DMTS) </H3><P>Methionine-γ-lyase was successfully made into a BioBrick standard format <A class="soft_link" href="http://parts.igem.org/Part:BBa_K1493300"> (Bba_K1493300)</A>. It was then put into plasmid pSEVA254 plasmid, which was transformed in both <I>E. coli</I> and <I>P. putida</I> KT2440. Succesfull transformants were verified via colony PCR for <I>E. coli</I> and <I>P. putida</I>. Transformants are further referred to as DMDS/DMTS <I>P. putida</I>. DMDS/DMTS <I>P. putida</I> was grown, induced and harvested after 3 hours. Cell free extracts were obtained via sonification and were then used for an assay. DMDS and DMTS compounds were supposed to be measured via Gas Chromatography (GC)<A class="soft_link" href="https://static.igem.org/mediawiki/2014/4/4b/Wageningen_UR_protocols_MgLgrowth.pdf">(see protocol)</A>. But it was in the end not possible due to unexpected problems with the GC. However an inhibition assay was done co-inoculating our DMDS/DMTS <I>P. putida</I> with <I>F. oxysporum</I>. </P><FIGURE><FIGCAPTION>Figure 9. Inhibition profiles of <I>F. oxysporum</I> TR4 by DMDS/DMTS <I>P. putida</I>. A 5mm <I>F. oxysporum</I> TR4 plug was used, tests done in duplo (upper &amp; lower row), t=7 days. Red circle indicates area occupated by <I>F. oxysporum</I> growth. Foc control=only <I>F. oxysporum</I> TR4, Wildtype= <I>P. putida</I> KT2440 and 
			DMDS/DMTS=DMDS/DMTS <I>P. putida</I> overexpressing Methionine-gamma-lyase.
			</FIGCAPTION></FIGURE><P>In figure 9 it can be seen when comparing our DMDS/DMTS <I>P. putida</I> the wild type <I>P. putida</I> there is a much smaller fungal disc, probably cause by a  higher inhibition of our DMDS/DMTS <I>P. putida</I>. </P><H3>Pyoverdine</H3><P>The <I>pfri</I> gene was successfully made into a BioBrick standard format <A class="soft_link" href="http://parts.igem.org/Part:BBa_K1493200"> (Bba_K1493200)</A>, validated and characterized. <I>pfri</I> was cloned and put into plasmid pSEVA254. Then transformed in both <I>E. coli</I> and <I>P. putida</I> KT2440. Successful transformants were verified via colony PCR. And are further referred to as pyoverdine <I>P. putida</I>. Afterwards growth experiments were done in <A class="soft_link" href="https://static.igem.org/mediawiki/2014/d/d0/Wageningen_UR_protocols_Mediabuffers.pdf">minimal M9 medium</A> supplemented with iron (31μM) and the pyoverdine compound was measured using spectrophotomery. When grown overnight it was possible to see that pyoverdine <I>P. putida</I> were slightly greener when compared to the control. Control used here was a <I>P. putida</I> containing an empty pSEVA254 plasmid.</P><FIGURE><FIGCAPTION>Figure 10. Pyoverdine spectrum(350-400nm)OD corrected average from biological duplicate.
			</FIGCAPTION></FIGURE><P>In figure 10 it can be seen that the peak (400nm) of pyoverdine <I>P. putida</I> is higher than the peak from a <I>P. putida</I> containing an empty plasmid.</P><FIGURE><FIGCAPTION>Figure 3. Pyoverdine production in M9 medium supplemented with 31μM iron with error bars,OD corrected. empty plasmid=  <I>P. putida</I> containing an empty plasmid for control.
							</FIGCAPTION></FIGURE><P>When looking only at the aborbance at 400nm (figure 11), it can be seen that there is a 4-fold increase of pyoverdine production when <I>pfri</I> is overexpressed in P. putida. In addition to this experiment, inhibition assay was also done with pyoverdine <I>P. putida</I> co-inoculated with a 5mm <I>F. oxysporum</I> plug. </P><FIGURE><FIGCAPTION>Figure 12. Inhibition profiles of <I>F. oxysporum</I> TR4 by pyoverdine <I>P. putida</I>. A 5mm <I>F. oxysporum</I> TR4 plug was used, tests done in duplo (upper &amp; lower row), t=7 days. Red circle indicates area occupated by <I>F. oxysporum</I> growth. Foc control=only <I>F. oxysporum</I> TR4, Wild type= <I>P. putida</I> KT2440 and 
			pyoverdine= pyoverdine <I>P. putida</I> with an overexpression of <I>pfri</I>.
			</FIGCAPTION></FIGURE><P>In figure 12 it can be seen that pyoverdine <I>P. putida</I> has a smaller <I>F. oxysporum</I> disk when compared to the wild type. Meaning it's inhibition is increased but only ever so slightly.</P><H3>Chitinase</H3><P><I>PP3066</I> was succesfully cloned into broad host range plasmid (pSEVA254) and transformed into <I>P. putida</I>. Successful tranformants were verified via colony PCR. And are further referred to at chitinase <I>P. putida</I>. Chitinase <I>P. putida</I> was grown, induced and harvested 3 hours after induction. Cell free extracted were obtained via sonification of cells (see protocols), sadly there was not enough time to do a chitinase assay. Chitinase <I>P. putida</I> was co-inoculated with Fusarium in an inhibition assay, results in figure 13.</P><FIGURE><FIGCAPTION>Figure 13. Inhibition profiles of <I>F. oxysporum</I> TR4 by chitinase <I>P. putida</I>. A 5mm <I>F. oxysporum</I> TR4 plug was used, tests done in duplo (upper &amp; lower row), t=7 days. Red circle indicates area occupated by <I>F. oxysporum</I> growth. Foc control=<I>Fusarium oxysporum cubense</I> TR4, Wild type= <I>P. putida</I> KT2440 and chitinase=chitinase <I>P. putida</I> with an overexpression of <I>PP3066</I> predicted to have chitinase activity.
			</FIGCAPTION></FIGURE><P>In figure 13, it can be seen that chitinase <I>P. putida</I> seem to have a smaller fungal disc as when compared to the wild type.</P><H3>All fungal growth inhibitors together</H3><P>In addition to testing just one tranformant with <I>F. oxysporum</I> TR4, two culture mixes were made. 
			<OL><LI>Chitinase, DMDS/DMTS and pyoverdine</LI><LI>2,4-DAPG, Chitinase, DMDS/DMTS and pyoverdine</LI></OL></P><P>These two culture mix were then spread on agar plates using overnight liquid culture, grown overnight in 30°C and co-inoculated with <I>F. oxysporum</I> the following day. 
			<FIGURE><FIGCAPTION>Figure 14. Inhibition profiles of <I>F. oxysporum</I> TR4 by pyoverdine <I>P. putida</I>. A 5mm <I>F. oxysporum</I> TR4 plug was used, tests done in duplo (upper &amp; lower row), t=7 days. Red circle indicates area occupated by <I>F. oxysporum</I> growth. Foc control=<I>Fusarium oxysporum cubense</I> TR4, WT=wild type <I>P. putida</I> KT2440, chitinase+DMDS/DMDS+pyoverdine= culture mix 1, mix(all 4)= culture mix 2.
			</FIGCAPTION></FIGURE></P><P>In figure 14, chit+DMDS/DMDS+pyoverdine seems to have a better inhibitory effect when compared to the wild type as it can be seen that the fungal disc is smaller than the one of the wild type. For the culture mix that contains all four, mix(all 4) the fungal disc is only slightly smaller when compared to the wild type. The latter we cannot explain, and shows more and different assay are needed to thoroughly investigate the combined inhibition by our engineered <I>P. putida</I> strains.</P></SECTION><SECTION id="conclusion"><H2>Conclusion</H2><P>We  engineered <I>P. putida</I> with four different genes on four different pSEVa plasmids, coding for fungal growth inhibitor resistance, in order to increase its inhibitory effects against <I>F. oxysporum</I>. All four different transformants were obtained and have been further tested. We tested the <I>P.  putida</I> transformants in inhibition experiments with <I>F. oxysporum</I> to detect growth inhibition. Additionally for some fungal growth inhibitors, we tested specific production or the effect of the pure fungal growth inhibitor only on <I>F. oxysporum</I> growth.</P><P>Out of the 4 fungal growth inhibitors, methionine-γ-lyase and <I>pfri</I> were already made into BioBricks format, <A class="soft_link" href="http://parts.igem.org/Part:BBa_K1493300"> Bba_K1493300</A> and <A class="soft_link" href="http://parts.igem.org/Part:BBa_K1493200"> Bba_K1493200</A> respectively. With both BioBricks validated, and for PfrI characterized. PfrI has been shown to give a four-fold increase of pyoverdine production in the pressence of iron in the medium.</P><P>Even though production of 2,4-DAPG could not clearly be verified via HPLC because of extraction problems in the assay. But it was also shown in inhibition assays that pure 2,4-DAPG is indeed a good fungal growth inhibitor against <I>F. oxysporum</I> TR4. With decreased growth of <I>F. oxysporum</I> when there is an increase of 2,4-DAPG in the plates (figure 6 and 7).</P><P>In general,  it was hard to distinguish the increased inhibition effect of the fungal growth inhibitor producing <I>P. putida</I> against <I>F. oxysporum</I>. This is because the <I>P.  putida</I> chassis we have chosen is already very good at inhibiting <I>F. oxysporum</I> naturally, which probably makes it hard to distinguish increased growth inhibition by our synthetic,  fungal growth inhibitor producing <I>P. putida</I> strains. However, with the DMDS/DMTS production strain, we have a clear indication that there is an increase of growth inhibition of <I>F. oxysporum</I> (figure 9) and with others, producing 2,4-DAPG, chitinase or pyoverdine, we can say that there is an indication of a slight increase of growth inhibition (figure 8, 12 and 13) on top of the natural inhibition.</P></SECTION><SECTION id="future"><H2>Future work</H2><P>For future work, it would be nice if all the fungal growth inhibitor genes are coupled together behind the fusaric acid promoter in order to test production rate that is induced by fusaric acid. Also to test the strains in the <A href="https://2014.igem.org/Team:Wageningen_UR/project/greenhouse" class="soft_link">greenhouse</A> with actual banana plants, by a co-inoculating the plant roots it with <I>F. oxysporum</I> and our engineered strains. This could demonstrate if the production of the fungal growth inhibitors can actually affect to <I> F. oxysporum</I> in the soil. Other works would be to try other fungal growth inhibitor genes to increase inhibition by using other fungal growth inhibitor compounds such as phenazine-1-carboxylic acid which was shown to also have fungal growth inhibitor effects[24].  </P><SECTION id="biobricks"><H2> BioBricks </H2><P><B><A href="http://parts.igem.org/Part:BBa_K1493002" class="soft_link"> BBa_K1493002</A></B> and <B><A href="http://parts.igem.org/Part:BBa_K1493003" class="soft_link">BBa_K1493003</A></B></P></SECTION><P style="float:right"><B>Continue to <A href="https://2014.igem.org/Team:Wageningen_UR/project/greenhouse" class="soft_link">Greenhouse &gt;&gt;</A></B></P><H2> References </H2><OL class="references"><LI>Loper, H. G. a. J. E. (2009). &quot;Genomics of secondary metabolite production by Pseudomonas spp.&quot; from http://pubs.rsc.org/en/content/articlehtml/2009/np/b817075b#cit62.</LI><LI>Meldau, D. G., S. Meldau, L. H. Hoang, S. Underberg, H. Wunsche and I. T. Baldwin (2013). &quot;Dimethyl disulfide produced by the naturally associated bacterium bacillus sp B55 promotes Nicotiana attenuata growth by enhancing sulfur nutrition.&quot; Plant Cell 25(7): 2731-2747.</LI><LI>Zhang, H., A. Mallik and R. S. Zeng (2013). &quot;Control of Panama disease of banana by rotating and intercropping with Chinese chive (Allium tuberosum Rottler): role of plant volatiles.&quot; J Chem Ecol 39(2): 243-252.</LI><LI>Belgrove, A. (2007). Biological Control of Fusarium Oxysporum F.sp. Cubense Using Non-pathogenic F. Oxysporum Endophytes, University of Pretoria.</LI><LI>Rezzonico, F., M. Zala, C. Keel, B. Duffy, Y. Moënne-Loccoz and G. Défago (2007). &quot;Is the ability of biocontrol fluorescent pseudomonads to produce the antifungal metabolite 2,4-diacetylphloroglucinol really synonymous with higher plant protection?&quot; New Phytologist 173(4): 861-872.</LI><LI>Bangera, M. G. and L. S. Thomashow (1999). &quot;Identification and characterization of a gene cluster for synthesis of the polyketide antibiotic 2,4-diacetylphloroglucinol from Pseudomonas fluorescens Q2-87.&quot; J Bacteriol 181(10): 3155-3163.</LI><LI>Abbas, A., J. E. McGuire, D. Crowley, C. Baysse, M. Dow and F. O'Gara (2004). &quot;The putative permease PhlE of Pseudomonas fluorescens F113 has a role in 2,4-diacetylphloroglucinol resistance and in general stress tolerance.&quot; Microbiology 150(Pt 7): 2443-2450.</LI><LI>Bakker, P. H. M., D. M. Glandorf, M. Viebahn, T. M. Ouwens, E. Smit, P. Leeflang, K. Wernars, L. Thomashow, J. Thomas-Oates and L. van Loon (2002). &quot;Effects of Pseudomonas putida modified to produce phenazine-1-carboxylic acid and 2,4-diacetylphloroglucinol on the microflora of field grown wheat.&quot;  81(1-4): 617-624.</LI><LI>Hanniffy, S. B., M. Philo, C. Pelaez, M. J. Gasson, T. Requena and M. C. Martinez-Cuesta (2009). &quot;Heterologous production of methionine-gamma-lyase from Brevibacterium linens in Lactococcus lactis and formation of volatile sulfur compounds.&quot; Appl Environ Microbiol 75(8): 2326-2332.</LI><LI>23.	Ravel, J. and P. Cornelis (2003). &quot;Genomics of pyoverdine-mediated iron uptake in pseudomonads.&quot; Trends in Microbiology 11(5): 195-200.</LI><LI>Matthijs, S., G. Laus, J. M. Meyer, K. Abbaspour-Tehrani, M. Schafer, H. Budzikiewicz and P. Cornelis (2009). &quot;Siderophore-mediated iron acquisition in the entomopathogenic bacterium Pseudomonas entomophila L48 and its close relative Pseudomonas putida KT2440.&quot; Biometals 22(6): 951-964.</LI><LI>Duijff, B., J. Meijer, P. H. M. Bakker and B. Schippers (1993). &quot;Siderophore-mediated competition for iron and induced resistance in the suppression of fusarium wilt of carnation by fluorescent Pseudomonas spp.&quot;  99(5-6): 277-289.</LI><LI>Elad, Y. and R. Baker (1985). &quot;The role of competition for iron and carbon in suppression of chlamydospore germination of Fusarium spp. by Pseudomonas spp.&quot; Phytopathology 75(9): 1053-1059.</LI><LI>Nagarajkumar, M., R. Bhaskaran and R. Velazhahan (2004). &quot;Involvement of secondary metabolites and extracellular lytic enzymes produced by Pseudomonas fluorescens in inhibition of Rhizoctonia solani, the rice sheath blight pathogen.&quot; Microbiological Research 159(1): 73-81.</LI><LI>Duijff, B. J., P. A. H. M. Bakker and B. Schippers (1994). &quot;Suppression of fusarium wilt of carnation by Pseudomonas putida WCS358 at different levels of disease incidence and iron availability.&quot; Biocontrol Science and Technology 4(3): 279-288.</LI><LI>Lemanceau, P., P. A. Bakker, W. J. De Kogel, C. Alabouvette and B. Schippers (1992). &quot;Effect of pseudobactin 358 production by Pseudomonas putida WCS358 on suppression of fusarium wilt of carnations by nonpathogenic Fusarium oxysporum Fo47.&quot; Applied and Environmental Microbiology 58(9): 2978-2982.</LI><LI>Dos Santos, V. A. P. M., S. Heim, E. R. B. Moore, M. Strätz and K. N. Timmis (2004). &quot;Insights into the genomic basis of niche specificity of Pseudomonas putida KT2440.&quot; Environmental Microbiology 6(12): 1264-1286.</LI><LI>Venturi, V., C. Ottevanger, M. Bracke and P. Weisbeek (1995). &quot;Iron regulation of siderophore biosynthesis and transport in Pseudomonas putida WCS358: involvement of a transcriptional activator and of the Fur protein.&quot; Molecular Microbiology 15(6): 1081-1093.</LI><LI>Atkinson, R. A., A. L. M. Salah El Din, B. Kieffer, J.-F. Lefèvre and M. A. Abdallah (1998). &quot;Bacterial Iron Transport: 1H NMR Determination of the Three-Dimensional Structure of the Gallium Complex of Pyoverdin G4R, the Peptidic Siderophore of Pseudomonas putida G4R†,‡.&quot; Biochemistry 37(45): 15965-15973.</LI><LI>Venturi, V., P. Weisbeek and M. Koster (1995). &quot;Gene regulation of siderophore-mediated iron acquisition in Pseudomonas: not only the Fur repressor.&quot; Mol Microbiol 17(4): 603-610.</LI><LI>Mauch, F., B. Mauch-Mani and T. Boller (1988). &quot;Antifungal Hydrolases in Pea Tissue: II. Inhibition of Fungal Growth by Combinations of Chitinase and β-1,3-Glucanase.&quot; Plant Physiology 88(3): 936-942.</LI><LI>Herrera-Estrella, A. and I. Chet (1999). &quot;Chitinases in biological control.&quot; Exs 87: 171-184.</LI><LI>Silva-Rocha, R., E. Martínez-García, B. Calles, M. Chavarría, A. Arce-Rodríguez, A. de las Heras, A. D. Páez-Espino, G. Durante-Rodríguez, J. Kim and P. I. Nikel (2013). &quot;The Standard European Vector Architecture (SEVA): a coherent platform for the analysis and deployment of complex </LI><LI>Upadhyay, A., &amp; Srivastava, S. (2011). Phenazine-1-carboxylic acid is a more important contributor to biocontrol<I> Fusarium oxysporum</I> than pyrrolnitrin in <I> Pseudomonas fluorescens</I> strain Psd. Microbiological research, 166(4), 323-335.</LI></OL></SECTION></FIGURE></FIGCAPTION></FIGURE></FIGCAPTION></FIGURE></FIGCAPTION></FIGURE></SECTION></DIV></DIV></DIV></DIV><DIV id="footer"><P>iGEM Wageningen UR 2014. Send us an <A href="mailto:igemwageningen@gmail.com" class="soft_link">email</A>, or follow us on <A class="soft_link" href="http://twitter.com/igemwageningen" target="_blank">Twitter</A> and <A class="soft_link" href="http://www.facebook.com/iGEMWageningen" target="_blank">Facebook</A>!
			
			For more information about iGEM, check out their <A class="soft_link" href="https://igem.org">website</A> and <A class="soft_link" href="http://twitter.com/igem">Twitter</A>!</P></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Wageningen_UR/project/fungal_inhibition">http://2014.igem.org/Team:Wageningen_UR/project/fungal_inhibition</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Wageningen_UR/project/fungal_inhibition" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Wageningen_UR/project/fungal_inhibition" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Wageningen_UR/project/fungal_inhibition&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Wageningen_UR/project/fungal_inhibition&amp;oldid=398564" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>