<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Wageningen_UR_project_fungal_sensing"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Wageningen_UR/project/fungal_sensing">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Wageningen_UR/project/fungal_sensing&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Wageningen_UR/project/fungal_sensing&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Wageningen_UR/project/fungal_sensing&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Wageningen_UR/project/fungal_sensing" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Wageningen UR/project/fungal sensing</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><P><TITLE>Wageningen UR iGEM 2014 </TITLE></P><DIV id="wrapper"><DIV id="innerwrap"><DIV id="left"><P> </P><DIV id="inner-left"><P> </P><DIV class="sidebar-container"><P> </P><SIDENAV><DIV class="menu-head"><H4><A href="#header">Sensing</A></H4></DIV><LI class="menu-item"><A href="#fasensing">FA Sensing</A></LI><LI><A href="#overview1">Overview</A></LI><LI><A href="#approach1">Approach</A></LI><LI><A href="#results1">Discussion</A></LI><LI><A href="#future1">Future</A></LI><LI class="menu-item"><A href="#resistance">Resistance</A></LI><LI><A href="#approach2">Approach</A></LI><LI><A href="#results2">Results</A></LI><LI><A href="#future2">Future</A></LI><LI class="menu-item"><A href="#biobricks">BioBricks</A></LI><LI class="menu-item"><A href="#reference">References</A></LI></SIDENAV></DIV></DIV></DIV><DIV id="center"><DIV class="navmenubkg"><DIV id="nav_wrapper"><DIV id="ca-menu"><UL><LI><A href="https://2014.igem.org/Team:Wageningen_UR/overview"><DIV class="ca-content">Overview</DIV></A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/overview/background">Background</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/overview/solution">Solution</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/overview/results">Results</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/overview/achievements">Achievements</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/overview/parts">BioBricks</A></LI></UL><LI><A href="https://2014.igem.org/Team:Wageningen_UR/project"><DIV class="ca-content">Project</DIV></A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/project/fungal_sensing">Fungal Sensing</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/project/fungal_inhibition">Fungal Inhibition</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/project/fungal_inhibition" class="up_menu_fix">Wetlab</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/project/greenhouse">Greenhouse</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/project/kill-switch">Kill-Switch</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/project/kill-switch" class="up_menu_fix">Kill-Switch</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/project/characterization">Characterization</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/project/gene_transfer">Gene Transfer</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/project/gene_transfer" class="up_menu_fix">Gene Transfer</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/project/low_copy">Low Copy Plasmids</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/project/model_overview">Model</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/project/model#design1" class="up_menu_fix">Promoter Design</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/project/model#cost1">System Cost</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/project/model#performance1">System Stability</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/project/interlab">Interlab Study</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/outreach"><DIV class="ca-content">Outreach</DIV></A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/outreach/policy">Policy</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/outreach/collaboration">Collaboration</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/outreach/synenergene">Synenergene</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/outreach/interviews">Interviews</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/outreach/meetup">National Meetup</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/outreach/media">Media</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/team"><DIV class="ca-content">Team</DIV></A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/team/members">Members</A></LI><LI class="up_menu_fix"><A href="https://2014.igem.org/Team:Wageningen_UR/team/members#students">Students</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/team/members#supervisors">Supervisors</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/team/gallery">Gallery</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/team/attributions">Attributions</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/team/acknowledgements">Acknowledgements</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/team/contact">Contact</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/safety"><DIV class="ca-content">Safety</DIV></A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/safety/environment">Environment</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/safety/laboratory">Laboratory</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/notebook"><DIV class="ca-content">Notebook</DIV></A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/notebook/journal">Journals</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/notebook/protocols">Protocols</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/notebook/links">Interesting Links</A></LI></DIV></DIV></DIV><DIV class="center_content"><H1>Fungal Sensing</H1><P>When we designed BananaGuard, we wanted it to be <B>effective</B>, <B>safe</B> and <B>specific</B> in fighting <I>Fusarium oxysporum f. sp. cubense (foc)</I>. This part describes how our product is specific in only targeting this fungus and not affecting other soil fungi. The pathogenic fungus <I>F. oxysporum</I> secretes fusaric acid, a toxin needed to invade the banana plant via the roots. <I>Pseudomonas putida</I>, which serves us as a platform for the development of our biological control agent. Here the engineered <I>Pseudomonas putida</I> needs to be able to sense fusaric acid and activate gene expression accordingly. Resistance to fusaric acid is also crucial in this process and we found that <I>Pseudomonas putida</I> already possesses a good base level of resistance [1]. To be able to transfer our system to different platforms, we aim to introduce fusaric acid resistance in <I>Escherichia coli</I>.</P><SECTION id="fasensing"><H2>Fusaric Acid Sensing</H2><SECTION id="overview1"><H3>Overview</H3></SECTION><P>Resistance to fusaric acid is found in several microorganisms, such as <I>Stenotrophomonas maltophilia</I> [3] and <I>Pseudomonas putida</I> [4]. This resistance to the toxic compound is due to a fusaric acid efflux pump, which pumps the acid out of the cell. After doing a literature search, we found out that there are a few proteins encoded in the genome of <I>P. putida</I> KT2440 which should confer fusaric acid resistance based on <I>in silico</I> prediction. The corresponding genes are all found placed in an operon cluster starting from pp1263 to pp1266 [4]. Upstream of this region and transcribed to the opposite direction we found the pp1262 gene whose predicted function is a LysR-type regulator.
Based on the above predictions and the gene topology, we hypothesized that since fusaric acid resistance from this cluster has not been confirmed through experiments it was worth to try to check if the theoretical regulator pp1262 can control the expression of the gene cluster through a promoter in the intergenic area. Based on this hypothesis, expression of the pp1262 protein should repress transcription of the operon cluster in the absence of fusaric acid. On the other side, when fusaric acid is present, repression should be relieved. To further support that this is how the system works and that fusaric acid does not activate the regulator leading to higher transcription rates of the resistance cluster, we expect that when the regulator is overexpressed in <I>P. putida</I>, it causes lower fusaric acid resistance because of lower depression rate of the promoter (Figure 1). Hence, we isolated pp1262 and the intergenic region and put it into a BioBrick, effectively acting together as a fusaric acid inducible promoter (FAiP).
</P><FIGURE><FIGCAPTION>Figure 1. Hypothetical fusaric acid efflux pump operon present in the genome of <I>Pseudomonas putida</I> KT2440. When fusaric acid enters the cell, the inhibition of the efflux pump is inhibited and the efflux pump is transcribed, resulting in the excretion of fusaric acid out of the cell. For the sensing part of this iGEM project, only the detection part is used.</FIGCAPTION></FIGURE><P>We decided to test and characterize the fusaric acid sensitivity and expression of this part by using different levels of fusaric acid. The promoter was tested using GFP and RFP reporter genes (<A class="soft_link" href="http://parts.igem.org/Part:BBa_I13507">BBa_I13507</A>, <A class="soft_link" href="http://parts.igem.org/Part:BBa_I13504">BBa_I13504</A>, <A class="soft_link" href="http://parts.igem.org/Part:BBa_J23100">BBa_J23100</A>) as output. Each BioBrick part contains our promoter, an RBS, a reporter gene and two terminators (Figure 2). The new constructs were expressed in <I>P. putida</I> to test for expression in the presence of fusaric acid. </P><FIGURE><FIGCAPTION>Figure 2. The complete construct containing both our fusaric acid inducible promoter and a reporter gene.</FIGCAPTION></FIGURE><SECTION id="approach1"><H3>Approach</H3></SECTION><P>To isolate the pp1262 gene and the downstream intergenic region, two promoters containing the RFC10 parts were used in a PCR reaction on the genomic DNA of <I>P. putida</I> KT2440. The product was ligated into pSB1C3 and transformed into chemically competent <I>E. coli</I> cells. The part was digested again and ligated upstream of several different reporter genes. BBa_I13507, which contains an RBS, mRFP and two terminators was successfully transformed. However, after sequencing it became clear that this BioBrick did not contain mRFP, making it <A class="soft_link" href="http://parts.igem.org/Part:BBa_I13507:Experience#User_Reviews">unsuitable for this experiment</A>. This problem has been reported to the registry. BBa_I13504 - which contains an RBS, GFP and two terminators - was used for further experiments. This reporter construct combined with our promoter part was ligated and transformed into <I>E. coli</I>. Sequencing confirmed the successful assembly. This construct should only show green fluorescence in the presence of fusaric acid, and no fluorescence when fusaric acid is absent. 
The <I>E. coli</I> cells, containing the construct were grown on agar plates containing different concentrations of fusaric acid. At 140µM, there was no significant difference between the transformed cells and the control cells. At higher concentrations the cells died, due to their inability to pump the acid out of the cell. Since small amounts of fusaric acid are lethal to <I>E. coli</I>, this is not suitable as a host to test the expression.
The construct was transformed into a different backbone (pSB1K3) and inserted into electrocompetent <I>P. putida</I> cells. The transformed cells were again grown on plates containing different concentrations of fusaric acid. A slight difference in colour could be observed between the transformed cells and the wild-type cells, indicating that the promoter is only active when fusaric acid is present (see Figure 3). </P><FIGURE><FIGCAPTION>Figure 3. Both plates contain 4 strains of transformed <I>P. putida</I>, containing the <A class="soft_link" href="http://parts.igem.org/Part:BBa_K1493003">BBa_K1493003</A>. The right plate contains 50µg/ml kanamycin and 50µg/ml fusaric acid, whereas the left plate only contains 50µg/ml kanamycin. A slightly increased fluorescence can be observed in the right plate, indicating the promoter is induced by the fusaric acid.</FIGCAPTION></FIGURE><P>
A growth experiment was performed in a 96-wells plate, containing 0, 55, 140, 285, 565 and 1415µM (0, 10, 25, 50, 100 and 250µg/ml) fusaric acid and both transformed and wild-type <I>P. putida</I> cells. After 18h, at a concentration of 285µM fusaric acid, the transformed cells showed more fluorescence than the wild-type cells, while both strains showed no fluorescence when no fusaric acid was added, indicating that the promoter is not active when no fusaric acid is present. 
The wild-type <I>P. putida</I> cells showed some fluorescence as well. We hypothesized that the stress caused by fusaric acid caused the excretion of additional fluorescent compounds, such as pyoverdine. Also, we found that LB medium has some auto-fluorescence. Since GFP does not include an excretion tag, we solved this problem by removing the supernatant by washing the cells with PBS and measuring the clean cells afterwards. 
After washing with PBS, the wild-type cells showed no significant fluorescence, whereas the transformed cells clearly did, (see Figure 4.) indicating that the promoter is functional in this experiment. </P><FIGURE><FIGCAPTION>Figure 4. Both <I>P. putida</I> strains are grown in LB medium with 285µM fusaric acid and washed with PBS. GFP fluorescence was checked for 6 samples of each, at 395nm excitation and 501 nm emission. This experiment shows that the promoter is active in the presence of fusaric acid. However, the value of the fluorescence is a dimentionless unit relative to the wild-type, which means a characterized promoter with a downstream GFP gene should be added to the experiment as a positive control to characterize this fusaric acid induced promoter.</FIGCAPTION></FIGURE><P>Futhermore, the WT cells showed growth on all concentrations of fusaric acid, whereas the transformed cells did not survive a fusaric acid concentration of 565µM and higher (see Figure 5), which supports our hypothesis that the inserted vector decreased the fusaric acid resistance of our transformed <I>P. putida</I> cells.</P><FIGURE><FIGCAPTION>Figure 5. Fusaric Acid does not seem to influence the growth of the wild-type <I>P. putida</I>, whereas the transformed cells do not survive a concentration 565µM or higher.</FIGCAPTION></FIGURE><P>A new growth experiment was set up, using M9 medium instead of LB medium, to prevent fluorescent background noise of the medium. In this experiment both the WT <I>P. putida</I> and <I>P. putida</I> containing <A class="soft_link" href="http://parts.igem.org/Part:BBa_K1493000">BBa_K1493000</A> are grown in a 96-wells plate at 0, 85, 170, 255, 340 and 425µM (0, 15, 30, 45, 60 and 75µg/ml) fusaric acid. Furthermore, <I>E. coli</I> DH5α cells (WT and <A class="soft_link" href="http://parts.igem.org/Part:BBa_K741002">BBa BBa_K741002</A>) were also grown in the same plate but without fusaric acid. The transformed <I>E. coli</I> has a well characterized promoter (pLac) with the same GFP gene downstream. (See Figure 6.) By comparing the fluorescence of our fusaric acid induced promoter at different fusaric acid concentrations to this constitutive promoter, characterization is performed. pLac is used as a constitutive promoter, which has shown activity <A class="soft_link" href="http://parts.igem.org/Part:BBa_R0010:Experience">without induction of IPTG</A>.</P><FIGURE><FIGCAPTION>Figure 6. *Significantly different from WT with p&lt;0.05The measurement is based on GFP fluorescence in <I>P. putida</I> at increased concentrations of fusaric acid to prove and characterize the activity of the fusaric acid induced promoter, <A class="soft_link" href="http://parts.igem.org/Part:BBa_K1493000">BBa_K1493000</A>. For comparison, the well characterized pLac promoter (<A class="soft_link" href="http://parts.igem.org/Part:BBa_K741002">BBa_K741002</A>, uninduced by IPTG) was used to quantify the activity of this promoter at different concentrations of fusaric acid. Our fusaric acid inducible promoter does not respond to low concentrations up to 170µM. From 255µM and up, the activity increases. The maximum measured activity of the promoter is 0.21 RPU at 425µM. </FIGCAPTION></FIGURE><SECTION id="results1"><H3>Discussion</H3></SECTION><P>Up to now, the regulation of this fusaric acid efflux pump was only theoretical. The literature [4,5] merely suggested a possibility in the regulation of the efflux pump yet no data is shown whatsoever but from these results the following conclusions can be drawn.
The hypothesis that the combination of the inhibiting LysR-type gene[5] and the promoter in the intergenic region is indeed fusaric acid inducible is supported by the results of the growth experiment in Figure 6. To activate the promoter, the concentration of fusaric acid has to pass a certain threshold.
To further support this hypothesis, it was observed that the transformed <I>P. putida</I> cells did not survive a fusaric acid concentration of 565µM or higher, likely because of over-expression of the inhibiting gene pp1262, which would not only inhibit the activity of the GFP but also of the original fusaric acid efflux pump in the original genome of <I>P. putida</I>, therefore decreasing the resistance of the host.
These experiments do not only give some good indications that the functioning of this promoter is indeed regulated by the induction of fusaric acid, but they also show that this regulator could actually be used for other purposes, such as in our case the detection of <I>Fusarium oxysporum</I> and producing anti-fungal agents in its presence. A fusaric acid detection system has not been developed before now, and the implications of this part could very likely exceed this iGEM project. </P><SECTION id="future1"><H3>Future work</H3></SECTION><P>During this iGEM project, we also tried to isolate the promoter from the original <I>P. putida</I> KT2440 genome without the inhibiting gene, but we were unsuccessful to do so. However, while performing mutagenesis on the BioBrick to remove the illegal restriction site PstI, an unintentional insert was introduced, causing a frame shift in the inhibiting gene, possibly disabling its function, leaving the promoter uninhibited. When this part would be inserted into <I>P. putida</I>, the over-expression of the inhibiting gene would be gone. Note that using this part is only possible in <I>P. putida</I> for this reason.

By introducing this part to <I>P. putida</I> and growing it at the same time as the tested transformed <I>P. putida</I> used in the previous experiments, the fusaric acid resistance can be compared between the two strains. Hypothetically, the new strain could have a higher resistance, because the overexpression of the inhibiting gene is dysfunctional and therefore cannot block the fusaric acid efflux pump in the original genome. We did not have time to test this yet.</P></SECTION><SECTION id="resistance"><H2>Resistance</H2><SECTION id="approach2"><H3>Approach</H3></SECTION><P>
Different gene cluster known to be related to fusaric acid resistance will be isolated from multiple different organisms. FusABCDE from <I>Burkholderia cepacia</I> [1], FDT-123 from <I>Klebsiella oxytoca</I> [2], FuaABC from <I>Stenotrophomonas maltophilia</I> [3] and PP1263-5 from <I>Pseudomonas putida</I> itself. These clusters will then be expressed under an IPTG inducible promoter (BBa_J04500) or a ramose inducible promoter (BBa_K914003) in order to test the resistance levels. Since all of the clusters contain multiple illegal PstI sites, it was decided to insert the cluster directly in the SpeI site of the biobrick suffix. 
PP1263-5 from <I>P. putida</I> could be isolated from the wild type strain used at our own facilities. The original strains referred to in the research papers for the other three organisms were not available. Therefore homologues genes, with high levels of sequence identity (&gt;98%) were identified and obtained from microbial collections. 
</P><SECTION id="results2"><H3>Results</H3></SECTION><P>
All clusters but FuaABC were successfully isolated by PCR but cloning proved to be more difficult. Initial attempts to ligate the cluster behind BBa_J04500 in psb1C3 and transform it into E. Coli yielded no results. Although colony PCR showed a positive colony in some cases of the FDT cluster, plasmids isolated from these colonies were not of the right size to contain the inserts. Sequencing one of these plasmids with the VR and VF2 primers showed insertion of a small fragment containing the end and start of the primer used to PCR the cluster and a small palindromic sequence of 8bp. (ATCGATGCTA) 
Membrane proteins can potentially be toxic when expressed at a too high rate. Therefore the construct was inserted in to a low copy number backbone (pSB3K3 / pSB4K5). Additionally the three genes of the smallest cluster (FDT-123) were separately expressed to see if one of them was the cause of the possible toxicity. 
Successful transformants were obtained in pSB4K5 for the whole FDT cluster, FDT-2 and FDT-3 under a pLac promoter. Once again the whole FDT cluster showed up successful on colony PCR, but failed to show the expected results when a restriction digest was done on isolated plasmids. Sequencing showed again inclusion of both the primers and a small palindromic sequence (TAGCATCGAT/ATCGATGCTA). 
Given that both FDT-2 and FDT-3 were successfully cloned but the whole cluster or FDT-1 separately did not yield any results,  FDT-1 , which has homologues proteins in all 3 clusters is the most likely cause for toxicity.
</P><SECTION id="future2"><H3>Future Work</H3></SECTION><P>
Expression of the complete cluster has not yet been achieved. More research is required to understand the function of the different parts of the resistance cluster. Ideally functionality could be proven using a FA sensitive knockout strain of <I>P. putida</I> itself with which could be restored by inserting the putative resistance cluster. After achieving successful expression, more insight could be achieved using protein interaction studies or GFP tagged proteins to determine localization.
</P></SECTION><SECTION id="biobricks"><H2>BioBricks</H2><P><A class="soft_link" href="http://parts.igem.org/Part:BBa_K1493000">BBa_K1493000</A>, <A class="soft_link" href="http://parts.igem.org/Part:BBa_K1493002">BBa_K1493002</A>, <A class="soft_link" href="http://parts.igem.org/Part:BBa_K1493003">BBa_K1493003</A></P><P style="float:right"><B>Continue to <A href="https://2014.igem.org/Team:Wageningen_UR/project/fungal_inhibition" class="soft_link">Fungal Inhibition &gt;&gt;</A></B></P><H2>References</H2></SECTION><P><OL class="references"><LI>Utsumi, R., et al., Molecular cloning and characterization of the fusaric acid-resistance gene from Pseudomonas cepacia. Agricultural and biological chemistry, 1991. 55(7): p. 1913-8.</LI><LI>Toyoda, H., et al., DNA Sequence of Genes for Detoxification of Fusaric Acid, a Wilt-inducing Agent Produced by Fusarium Species. Journal of Phytopathology, 1991. 133(4): p. 265-277.</LI><LI>Hu, R.-M., et al., An Inducible Fusaric Acid Tripartite Efflux Pump Contributes to the Fusaric Acid Resistance in Stenotrophomonas maltophilia. PLoS ONE, 2012. 7(12): p. e51053.</LI><LI>Nelson, K.E., et al., Complete genome sequence and comparative analysis of the metabolically versatile Pseudomonas putida KT2440. Environmental microbiology, 2002. 4(12): p. 799-808.</LI><LI>Maddocks, S.E. and P.C.F. Oyston, Structure and function of the LysR-type transcriptional regulator (LTTR) family proteins. Microbiology (Reading, England), 2008. 154(Pt 12): p. 3609-23.</LI></OL></P></DIV></DIV></DIV></DIV><DIV id="footer"><P>iGEM Wageningen UR 2014. Send us an <A href="mailto:igemwageningen@gmail.com" class="soft_link">email</A>, or follow us on <A class="soft_link" href="http://twitter.com/igemwageningen" target="_blank">Twitter</A> and <A class="soft_link" href="http://www.facebook.com/iGEMWageningen" target="_blank">Facebook</A>!
			
			For more information about iGEM, check out their <A class="soft_link" href="https://igem.org">website</A> and <A class="soft_link" href="http://twitter.com/igem">Twitter</A>!</P></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Wageningen_UR/project/fungal_sensing">http://2014.igem.org/Team:Wageningen_UR/project/fungal_sensing</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Wageningen_UR/project/fungal_sensing" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Wageningen_UR/project/fungal_sensing" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Wageningen_UR/project/fungal_sensing&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Wageningen_UR/project/fungal_sensing&amp;oldid=394941" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>