"
Page
Discussion
View source
History
teams
Log in
 
Team:Wageningen UR/notebook/journal/resistance
From 2014.igem.org
Wageningen UR iGEM 2014
 
 
 
Overview
Background
Solution
Results
Achievements
BioBricks
Project
Fungal Sensing
Fungal Inhibition
Wetlab
Greenhouse
Kill-Switch
Kill-Switch
Characterization
Gene Transfer
Gene Transfer
Low Copy Plasmids
Model
Promoter Design
System Cost
System Stability
Interlab Study
Outreach
Policy
Collaboration
Synenergene
Interviews
National Meetup
Media
Team
Members
Students
Supervisors
Gallery
Attributions
Acknowledgements
Contact
Safety
Environment
Laboratory
Notebook
Journals
Protocols
Interesting Links
Fungal resistance journal
All transformations were done in E. coli. Preparation of the backbones was done in DH5alfa, while for transformations with the clusters JM109 was used.
+ expand all
May
Growth experiments
Growth experiments were conducted with fusaric acid concentrations ranging from 0 to 250 ug/ml were performed on Eschericia coli (Strains DH5alfa, BL21 and JM109), Bacillus subtilis and Pseudomonas putida. After 18 hours of incubation at 37 degress celcius, OD 600 measurements were taken.
Figure 1. Both B. subtilis and all of the E. coli strains show a strong response to increasing doses of fusaric acid in the medium. P. putida, however, does not show any decrease in growth for the concentrations used. Therefore another experiment was done with higher concentrations for P. putida.
NOTE: After ordering new fusaric acid later in the project, it was found out that the fusaric acid used in this experiment was not the same strength, with the old one most likely being (partially) degraded. Therefore data from this experiment is not used.
June
HPLC
Since some soil bacteria are known to be able to degrade fusaric acid (FA), a HPLC experiment was started to test P. putida(PP) for fusaric acid degradation and carbon utilization. M9 media was used with glucose/fusaric acid(250ug/ml) or both as a carbon source.
For the HPLC the following settings were used.
Column: Polaris C18A
Eluent: Acetonitril
Temperature: 35 degrees Celsius
Flow speed: 0.5 ml/min
Detection: UV (260 - 280nm)
Figure 2. Left: The HPLC method used delivered a good calibration curve, but peak resolvance was not optimal, with the FA peak having overlap with several smaller peaks in the samples with P. putida. Therefore it was decided to repeat the experiment to improve the HPLC method. Right: Initial results on fusaric acid breakdown showed signs of breakdown, but because peak resolvance was not optimal and the measurements were not done in duplo more data was needed to draw conclusions.
Based on the initial results a more elaborate experiment was started in duplo. Six samples would be grown in duplo:
A. Negative control for growth on M9 without carbon source. (PP+, Glucose-, FA-)
B. Positive control for growth on M9 with carbon source (glucose).(PP+, Glucose+, FA-)
C. Positive control for growth on M9 with carbon source and fusaric acid. (PP+, Glucose+, FA-)
D. Test for PP growth on FA as carbon source.(PP+, Glucose-, FA+)
E. E. Negative control for contamination and FA stability. (PP+, Glucose-, FA-)
F. Negative control for contamination. (PP-, Glucose+, FA-)
Growth experiments
Using an automated platereader, time based growth was measured on P. putida, to see the response to different concentrations of fusaric acid.
Figure 3. P. putida growth followed for 12 hours grown in LB medium. The OD was measured every 15 minutes at 600nm. Plates were incubated shaking at 30 degrees Celsius. Concentrations of fusaric acid are noted in the legend in ug/ml. At concentrations higher then 500 P. putida ceases to grow.
August
Isolation of genes
All gene clusters were isolated using PCR . The following primers were used. Temperatures were calculated using NEB Tm Calculator. If the initial temperature did not work, the Tm was lowered by 3 degrees and a duplo with 1 ul of DMSO was done.
P. putida - PP1263-1266
FP: GTTTCTTCGAATTCGCGGCCGCTTCTAGAGATGCCGCGTCGCATCATC
RP: GTTTCTTCCTGCAGCGGCCGCTACTAGTATTATTATCAACCTTCGCCAGGCTTGAC
Klebsiella oxycota - FDT-123
FP: GTTTCTTCGAATTCGCGGCCGCTTCTAGAGATGCTCGCCTATTACGTTGC
RP: GTTTCTTCCTGCAGCGGCCGCTACTAGTATTATTACTATGGGTGGATAGCCTGAGC
Pseudomonas cepacia - FusABCDE
FP: GTTTCTTCGAATTCGCGGCCGCTTCTAGAGGGAGAAAATCATGCAGTCTC
RP: GTTTCTTCCTGCAGCGGCCGCTACTAGTATTATTATTACGACTTCTTCTGCTTGTCC
Stenotrophomonas maltophilia - FuaABC
FP: GTTTCTTCGAATTCGCGGCCGCTTCTAGAGATTGATTGTCATTGTAGCGAAATTC
RP: GTTTCTTCCTGCAGCGGCCGCTACTAGTACGGTAATTTCCTGAACAGAC
No PCR results could be obtained for this cluster, possibly the DNA was degraded during the time spent shipping. Since the other PCRs did give results, it was chosen not to continue with this gene cluster
Preparation of backbone
Initially pSB1C3 was chosen as a backbone. To serve as a repressible promotor J04500 was incorporated between the EcoRI and the XbaI site, because all the gene clusters to be integrated contain illegal PstI sites. Xba and SpeI were used to insert the gene clusters into the backbone vector.
Figure 4. Example of the plasmid layout. BBa_J04500 is inserted between EcoRI and XbaI while the FDT cluster is inserted between XbaI and SpeI. Standard three point ligation could not be used because of illegal PstI sites.
September
Transformations
All transformations using the fusaric acid resistance clusters proved unsuccesfull. Any colonies obtained were proven by colony PCR or restriction digestion to not contain any inserts. Since the expression of membrane proteins can often be toxic, it was considered that the leakiness of the pLac promoter in combination with the high copy pSB1C3 plasmid might prove toxic for our host organism. Therefore it was decided to construct low copy backbones using pSB3K3 and pSB4K5.
Preparation of low copy backbones
To prevent toxicity from leaky expression of potentially toxic membrane proteins low copy number backbones pSB3K3 and pSB4K5 were digested with EcoRI and XbaI to insert J04500 according to standard protocols.
FDT-123 PCR of seperate genes
As an alternate way to see if the proteins that were tried to be expressed were toxic, it was decided to isolate the three genes from the FDT-123 cluster seperatly. This could give insight in which of the genes would cause the potential toxicity. Primers used were:
AEX05579.1 FP - GTTTCTTCGAATTCGCGGCCGCTTCTAGAGCCACACTGGGGTTATTCGCATGAAATC
AEX05580.1 FP - GTTTCTTCGAATTCGCGGCCGCTTCTAGAGCGATCTGCACTGCGCGTTAC
AEX05580.1 RP - GTTTCTTCCTGCAGCGGCCGCTACTAGTATTATTAGGCCGCTACCGCTACTGC
AEX05581.1 RP - GTTTCTTCCTGCAGCGGCCGCTACTAGTATTATTATCATTTATCATGGGTGGGTTCCTCGTTC
October
Transformations with low copy backbones
Using the low copy backbones transformations results were obtained. The transformation of the FDT-123 cluster in pSB4K5 initially gave positive results on colony pcr, but after miniprepping and digestion the plasmid seemed empty. Two of the transformants were sent for sequencing. Insert containing the FDT-123 FP and RP was inserted with a palindromic sequence of "ATCGATGCTA" or "TAGCATCGAT" was confirmed by sequencing. The PP cluster gave similar results but with a sequence of "TTGTCCT". Since these sequences are palindromic this might be an indication that the cluster is toxic and recombination with deletion of the cluster is the only viable way to survive.
For the seperate genes of the FDT clusters, succesfull transformants were obtained for FDT-2 and FDT-3 and confirmed by sequencing. FDT-1 did not result in any succesfull tranformants leading to the conclusion that this is most likely the gene responsible for the toxicity of the cluster in E. coli.
Because of the problems getting the whole gene clusters expressed in combination with the high ammount of illegal restriction sites in the clusters, it was decided not to continue with the resistance
Further HPLC experiments
Several more tries were done to characterize the possible fusaric acid breakdown of P. putida, however, several technical issues with the HPLC were had so no further results have been obtained yet.
iGEM Wageningen UR 2014. Send us an email , or follow us on Twitter and Facebook !
For more information about iGEM, check out their website and Twitter !
Retrieved from " http://2014.igem.org/Team:Wageningen_UR/notebook/journal/resistance "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
