<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Wageningen_UR_project_model"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Wageningen_UR/project/model">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Wageningen_UR/project/model&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Wageningen_UR/project/model&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Wageningen_UR/project/model&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Wageningen_UR/project/model" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Wageningen UR/project/model</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><P><TITLE>Wageningen UR iGEM 2014 </TITLE></P><DIV id="wrapper"><DIV id="innerwrap"><DIV id="left"><P> </P><DIV id="inner-left"><P> </P><DIV class="sidebar-container"><P> </P><SIDENAV><DIV class="menu-head"><H4><A href="#header">Model</A></H4></DIV><LI class="menu-item"><A href="#design1">Promoters</A></LI><LI><A href="#introduction1">Introduction</A></LI><LI><A href="#approach1">Approach</A></LI><LI><A href="#results1">Results</A></LI><LI class="menu-item"><A href="#cost1">Cost</A></LI><LI><A href="#introduction2">Introduction</A></LI><LI><A href="#approach2">Approach</A></LI><LI><A href="#results2">Results</A></LI><LI class="menu-item"><A href="#performance">Performance</A></LI><LI><A href="#introduction3">Introduction</A></LI><LI><A href="#approach3">Approach</A></LI><LI><A href="#results3">Results</A></LI><LI class="menu-item"><A href="#conclusion">Conclusion</A></LI><DIV class="menu-head"><H4><A href="#appendix1">Appendix</A></H4></DIV><LI class="menu-item"><A href="#parameters1">Parameters</A></LI><LI><A href="#parameters41">Promoters</A></LI><LI><A href="#parameters42">Cost</A></LI><LI><A href="#parameters43">Performance</A></LI><LI class="menu-item"><A href="#allrefs">References</A></LI></SIDENAV></DIV></DIV></DIV><DIV id="center"><DIV class="navmenubkg"><DIV id="nav_wrapper"><DIV id="ca-menu"><UL><LI><A href="https://2014.igem.org/Team:Wageningen_UR/overview"><DIV class="ca-content">Overview</DIV></A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/overview/background">Background</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/overview/solution">Solution</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/overview/results">Results</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/overview/achievements">Achievements</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/overview/parts">BioBricks</A></LI></UL><LI><A href="https://2014.igem.org/Team:Wageningen_UR/project"><DIV class="ca-content">Project</DIV></A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/project/fungal_sensing">Fungal Sensing</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/project/fungal_inhibition">Fungal Inhibition</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/project/fungal_inhibition" class="up_menu_fix">Wetlab</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/project/greenhouse">Greenhouse</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/project/kill-switch">Kill-Switch</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/project/kill-switch" class="up_menu_fix">Kill-Switch</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/project/characterization">Characterization</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/project/gene_transfer">Gene Transfer</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/project/gene_transfer" class="up_menu_fix">Gene Transfer</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/project/low_copy">Low Copy Plasmids</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/project/model_overview">Model</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/project/model#design1" class="up_menu_fix">Promoter Design</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/project/model#cost1">System Cost</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/project/model#performance1">System Stability</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/project/interlab">Interlab Study</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/outreach"><DIV class="ca-content">Outreach</DIV></A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/outreach/policy">Policy</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/outreach/collaboration">Collaboration</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/outreach/synenergene">Synenergene</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/outreach/interviews">Interviews</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/outreach/meetup">National Meetup</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/outreach/media">Media</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/team"><DIV class="ca-content">Team</DIV></A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/team/members">Members</A></LI><LI class="up_menu_fix"><A href="https://2014.igem.org/Team:Wageningen_UR/team/members#students">Students</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/team/members#supervisors">Supervisors</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/team/gallery">Gallery</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/team/attributions">Attributions</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/team/acknowledgements">Acknowledgements</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/team/contact">Contact</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/safety"><DIV class="ca-content">Safety</DIV></A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/safety/environment">Environment</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/safety/laboratory">Laboratory</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/notebook"><DIV class="ca-content">Notebook</DIV></A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/notebook/journal">Journals</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/notebook/protocols">Protocols</A></LI><LI><A href="https://2014.igem.org/Team:Wageningen_UR/notebook/links">Interesting Links</A></LI></DIV></DIV></DIV><DIV class="center_content"><H2>Modeling Overview</H2><P>
The performance of BananaGuard as biological control agent is defined by <B>three</B> factors: its ability to <B>destroy or inhibit</B><I>Fusarium oxysporum</I> upon detection, its ability to <B>compete</B> with other micro-organisms in the soil and the <B>stability</B> of the genetic circuit.</P><P> 

If the fungal growth inhibitors cannot destroy or inhibit <I>F. oxysporum</I>, the bananaplants will die. This aspect of BananaGuards performance is being tested <A href="https://2014.igem.org/Team:Wageningen_UR/project/fungal_inhibition" class="soft_link">in vitro</A>. This leaves two  remaining questions to be answered:</P><P><B>Can the BananaGuard system survive in the presence of competing micro-organisms?</B></P><P><B>Is the genetic circuit robust to biological fluctuations?</B></P><P>The performance and stability of the genetic circuit is crucial for BananaGuard to work. To prevent <I>F. oxysporum</I> from functioning, the fungal growth inhibitors in BananaGuard need to be produced when the fungi are present. This requires two aspects of the system to function properly: the toggle-switch within the Kill-switch system needs to be stable and the fusaric acid dependent promoter needs to activate the system in the presence of fusaric acid.

</P><P> Optimizing <A href="https://2014.igem.org/Team:Wageningen_UR/project/model#design1" class="soft_link"><B>promoter design</B></A> 
constitutes the first step of BananaGuard development. Different configurations of promoter elements on the promoters of the Kill-switch were modeled using statistical mechanics. Parameters such as repressor binding strengths and repressor production rates were changed in each simulation. By scoring Kill-switch activation, the promoter element configuration with the most simulations showing a bi-stable switch were considered to be most likely to work. Including the anti-gene transfer inhibition effect on cell growth rates and the fusaric acid dependent promoter, this optimized version of the BananaGuard system was subsequently modeled over a population of dividing cells, assessing both its stability and performance in the form of a stochastic model.  


</P><P>Running thousands of simulations incorporating <B>noise terms</B>  and <B>promoter leakiness,</B><A href="https://2014.igem.org/Team:Wageningen_UR/project/model#performance1" class="soft_link">the robustness of the BananaGuard system</A> was quantified. Stochasticity in the system caused by cell division resulted in fractions of fast and slow growing cell populations with either stable or unstable Kill-switches. The same was considered for system performance; upon <B>induction</B> by fusaric acid a portion of the cell population activated their Kill-switch or remained in the same state. In order to make the simulations as accurate as possible a parameter for the maximum growth rate of BananaGuard was required. This parameter was obtained from a metabolic model that described the effects of competition between the BananaGuard system and micro-organisms in the rhizosphere. 

</P><P>To ascertain our hosts competitiveness the rate of cell division needs to be investigated on a metabolic level. Cell growth rate is a key factor in the competition between our engineered <I>Pseudomonas putida</I> and the wildtype <I>P. putida</I> found in the rhizosphere of the banana plant roots [1,2]. The integrated genetic circuit uses metabolic resources that would otherwise be dedicated to cellular maintenance or growth. Should this divergence of resources be too large our host will be outcompeted. A genome scale <A href="https://2014.igem.org/Team:Wageningen_UR/project/model#cost1" class="soft_link">metabolic model</A> was extended, investigating and outlining the cost of introducing our synthetic genes on a metabolic level.
</P><FIGURE><FIGCAPTION>Figure 1. Schematic overview of the modeling project indicating overlapping parts.</FIGCAPTION></FIGURE><SECTION id="design1"><H1>Kill-switch promoter design</H1><H2 id="introduction1">Introduction</H2><P>
						Promoter design for a stable <A href="https://2014.igem.org/Team:Wageningen_UR/project/kill-switch" class="soft_link">Kill-switch</A> involves balancing of different regulatory components. Two elements, the number of repressor binding sites and the position of those binding sites with respect to the gene, control two key variables: the chance a repressor binds to the regulatory region and the affinity with which it binds. These variables, in turn, determine the effectiveness of the repressor protein [1,2]. 
It has been shown that repressor proteins are more effective when there are more repressor binding sites in front of a gene, confirming that the increased chance of repressor protein binding leads to a significant increase in its effectiveness [1,2]. With respect to binding affinity, it has been shown for TetR proteins that a repressor binding site positioned in front of the open reading frame is more effective than those positioned at the back [6].
Because of slight variations in the position of a repressor binding site or variations in experimental conditions, a range of different repressor binding strength values are reported for the same repressor (sometimes varying several orders of magnitude)[4,5].
This means that case specific designs cannot be modeled accurately but it is possible to estimate the likelihood that the designed system will work. By taking all the possible repressor binding site combinations and varying the different binding affinities (thereby simulating their position) and production rates, the robustness of the system can determined given the repressor binding site combinations(figure 2). Statistical mechanics is an ideal tool because it can incorporate both repressor binding strength and binding sites. Continue to learn more about the statistical mechanics approach or view the <A href="https://2014.igem.org/Team:Wageningen_UR/project/model#results1" class="soft_link">results</A>.
</P><FIGURE><FIGCAPTION><B>Figure 2.</B> The figure shows the kill-switch with arbitrarily chosen repressor binding sites. (I) Altering the position of the TetR repressor binding sites change the effectiveness [6]; a large space of basepairs between the binding site and the LacI will result in weaker repression [5]. (II) Adding a LacI repressor binding site will increase the chance that LacI binds to the promoter region. </FIGCAPTION></FIGURE><H2 id="approach1">Theory and experimental design</H2><H3>Statistical Mechanics</H3><P>
Statistical mechanics provides a way to understand how the macroscopic properties of a system arise from the individual components within this system. In effect there are two states to consider in statistical mechanic descriptions. A macrostate specifies a system in terms of quantities that average over the microscopic constituents of the system. For example if you consider a closed container of gas (figure 3), quantities for the macrostate would be pressure or volume. This means that these macrostate quantities only work if you consider a system with a very large number of particles. Talking about a macrostate of a single molecule makes no sense since the terms are inherently descriptive of large systems. A microstate, however, does describe the properties of a single molecule e.g. kinetic energy and position. The macro- and micro-states come together in the key concept of statistical mechanics. A lot of different microstates can result in the same macrostate, but characterizing the macrostate imposes restraints on the possible microstates [7]. 
</P><FIGURE><FIGCAPTION><B>Figure 3.</B> The image represents the macro and microstates applied in statistical mechanics. The box can be considered a macrostate e.g. volume, temperature, pressure. The molecules with different microstates e.g. kinetic energy, direction. The molecules determine the pressure and temperature but a given temperature provides information on the kinetic energy a molecule is most likely to have.</FIGCAPTION></FIGURE><H3>Modeling Biological Systems using Statistical Mechanics </H3><P>
						The logic of macrostates and microstates can also be applied to cells, replacing the container of the previous example with the cell and the different particle constituents with RNA polymerases and repressor proteins. The condition and production states of the various proteins can be considered a macrostate.
Modeling with statistical mechanics provides an insight in transcription regulation of a genetic circuit by generating ‘parameter-free’ predictions for the levels of gene expression. Assuming transcription of a gene to be proportional to the chance that a RNA polymerase binds to the promoter of interest a statistical mechanics model has two microstate outcomes [1,2]:
</P><P><B>Non-specific binding:</B> A state where none of the polymerases are bound to the gene of interest and distributed among non-specific binding sites . 
</P><P><B>Specific binding:</B> A state where the promoter of the gene of interest is occupied and the remaining polymerases are distributed among the non-specific binding sites. 

In order to determine the probability that either state will be realized we need to know the total number of possible microstates and the chance that a particular microstate will be achieved. For the non-specific binding by RNA polymerases combinatorics can be used [1]. The following equations will describe step for step how statistical mechanics can model biology, starting with combinatorics.
</P><P><MATH xmlns:MML="http://www.w3.org/1998/Math/MathML" xmlns:M="http://schemas.openxmlformats.org/officeDocument/2006/math" display="block"><MFRAC><MROW><MSUB><MROW><MI>N</MI></MROW><MROW><MI>n</MI><MI>s</MI></MROW></MSUB><MO>!</MO></MROW><MROW><MI>P</MI><MO>!</MO><MFENCED separators="|"><MROW><MSUB><MROW><MI>N</MI></MROW><MROW><MI>n</MI><MI>s</MI></MROW></MSUB><MO>-</MO><MI>P</MI></MROW></MFENCED><MO>!</MO></MROW></MFRAC><MO>=</MO><MI>N</MI><MI>u</MI><MI>m</MI><MI>b</MI><MI>e</MI><MI>r</MI><MI>o</MI><MI>f</MI><MI>n</MI><MI>o</MI><MI>n</MI><MO>-</MO><MI>s</MI><MI>p</MI><MI>e</MI><MI>c</MI><MI>i</MI><MI>f</MI><MI>i</MI><MI>c</MI><MI>b</MI><MI>i</MI><MI>n</MI><MI>d</MI><MI>i</MI><MI>n</MI><MI>g</MI><MI>p</MI><MI>o</MI><MI>s</MI><MI>s</MI><MI>i</MI><MI>b</MI><MI>i</MI><MI>l</MI><MI>i</MI><MI>t</MI><MI>i</MI><MI>e</MI><MI>s</MI></MATH></P><P>
Where P is the number of RNA polymerases and N<SUB>ns</SUB> the number of non-specific binding sites. Because there is a difference in the binding strength between specific and non-specific binding sites a Boltzmann weight needs to be included [1]:
</P><P><MATH xmlns:MML="http://www.w3.org/1998/Math/MathML" xmlns:M="http://schemas.openxmlformats.org/officeDocument/2006/math" display="block"><MI>Z</MI><MFENCED separators="|"><MROW><MI>P</MI></MROW></MFENCED><MO>=</MO><MFRAC><MROW><MSUB><MROW><MI>N</MI></MROW><MROW><MI>n</MI><MI>s</MI></MROW></MSUB><MO>!</MO></MROW><MROW><MI>P</MI><MO>!</MO><MFENCED separators="|"><MROW><MSUB><MROW><MI>N</MI></MROW><MROW><MI>n</MI><MI>s</MI></MROW></MSUB><MO>-</MO><MI>P</MI></MROW></MFENCED><MO>!</MO></MROW></MFRAC><MSUP><MROW><MI>e</MI></MROW><MROW><MO>-</MO><MFRAC><MROW><MI>P</MI><MSUBSUP><MROW><MI>ε</MI></MROW><MROW><MI>p</MI><MI>d</MI></MROW><MROW><MI>N</MI><MI>S</MI></MROW></MSUBSUP></MROW><MROW><MSUB><MROW><MI>K</MI></MROW><MROW><MI>b</MI></MROW></MSUB><MI>T</MI></MROW></MFRAC></MROW></MSUP></MATH></P><P>
To describes difference between strength of specific and non-specific binding a Boltzmann weight is added. For the non-specific binding of RNA polymerases the Boltzmann weight ε<SUB>pd</SUB><SUP>NS</SUP> is going to be very low compared to specific binding, compensating for the high number of binding site arrangements. In order to get the total statistical weight the microstate for RNA polymerase bound to the promoter needs to be added [1,2].
</P><P><MATH xmlns:MML="http://www.w3.org/1998/Math/MathML" xmlns:M="http://schemas.openxmlformats.org/officeDocument/2006/math" display="block"><MSUB><MROW><MI>Z</MI></MROW><MROW><MI>t</MI><MI>o</MI><MI>t</MI></MROW></MSUB><MFENCED separators="|"><MROW><MI>p</MI></MROW></MFENCED><MO>=</MO><MI>Z</MI><MFENCED separators="|"><MROW><MI>P</MI></MROW></MFENCED><MO>+</MO><MI>Z</MI><MFENCED separators="|"><MROW><MI>P</MI><MO>-</MO><MN>1</MN></MROW></MFENCED><MSUP><MROW><MI>e</MI></MROW><MROW><MO>-</MO><MFRAC><MROW><MSUBSUP><MROW><MI>ε</MI></MROW><MROW><MI>p</MI><MI>d</MI></MROW><MROW><MI>S</MI></MROW></MSUBSUP></MROW><MROW><MSUB><MROW><MI>K</MI></MROW><MROW><MI>b</MI></MROW></MSUB><MI>T</MI></MROW></MFRAC></MROW></MSUP></MATH></P><P>
ε<SUB>pd</SUB><SUP>S</SUP> is the specific binding energy of the RNA polymerase binding to the promoter. To obtain the chance an RNA polymerase is bound we divide the specific binding chance by the total chance [1,2].
</P><P><MATH xmlns:MML="http://www.w3.org/1998/Math/MathML" xmlns:M="http://schemas.openxmlformats.org/officeDocument/2006/math" display="block"><MI>P</MI><MI>b</MI><MI>o</MI><MI>u</MI><MI>n</MI><MI>d</MI><MO>=</MO><MFRAC><MROW><MI>Z</MI><MO>(</MO><MI>P</MI><MO>-</MO><MN>1</MN><MO>)</MO><MSUP><MROW><MI>e</MI></MROW><MROW><MO>-</MO><MFRAC><MROW><MSUBSUP><MROW><MI>ε</MI></MROW><MROW><MI>p</MI><MI>d</MI></MROW><MROW><MI>S</MI></MROW></MSUBSUP></MROW><MROW><MSUB><MROW><MI>K</MI></MROW><MROW><MI>b</MI></MROW></MSUB><MI>T</MI></MROW></MFRAC></MROW></MSUP></MROW><MROW><MSUB><MROW><MI>Z</MI></MROW><MROW><MI>t</MI><MI>o</MI><MI>t</MI></MROW></MSUB><MFENCED separators="|"><MROW><MI>p</MI></MROW></MFENCED></MROW></MFRAC></MATH></P><P>
By dividing top and bottom by Z(P<SUP>-1</SUP>)e<SUP>(-(ε<SUB>pd</SUB><SUP>S</SUP>)/(K<SUB>b</SUB>T))</SUP> the equation takes the form of a Hill function where the assumption was made that the amount of polymerases is a lot smaller than the amount of non-specific binding sites [1,2].
</P><P><MATH xmlns:MML="http://www.w3.org/1998/Math/MathML" xmlns:M="http://schemas.openxmlformats.org/officeDocument/2006/math" display="block"><MI>P</MI><MI>b</MI><MI>o</MI><MI>u</MI><MI>n</MI><MI>d</MI><MO>=</MO><MFRAC><MROW><MN>1</MN></MROW><MROW><MN>1</MN><MO>+</MO><MFRAC><MROW><MSUB><MROW><MI>N</MI></MROW><MROW><MI>n</MI><MI>s</MI></MROW></MSUB></MROW><MROW><MI>P</MI></MROW></MFRAC><MSUP><MROW><MI>e</MI></MROW><MROW><MO>-</MO><MFRAC><MROW><MO>∆</MO><MSUBSUP><MROW><MI>ε</MI></MROW><MROW><MI>p</MI><MI>d</MI></MROW><MROW><MI>S</MI></MROW></MSUBSUP></MROW><MROW><MSUB><MROW><MI>K</MI></MROW><MROW><MI>b</MI></MROW></MSUB><MI>T</MI></MROW></MFRAC></MROW></MSUP></MROW></MFRAC></MATH></P><P>
In this form a fold change of mRNA expression can be correlated to the chance that a polymerase binds (N<SUB>ns</SUB>/P e<SUP>(-(∆ε<SUB>pd</SUB><SUP>S</SUP>)/(K<SUB>b</SUB>T))</SUP>). If the term describing RNA polymerase binding equals zero, the the chance that a RNA polymerase binds is equal to one and there is no change in gene expression. This will most likely be the case in a system without interference. Most biological systems, however, are dependent on regulators, repression is the main method of gene regulation in our designed circuit (see <A href="https://2014.igem.org/Team:Wageningen_UR/project/kill-switch" class="soft_link">Kill-switch</A>). The effect of repressors can be taken up in previous equation by adding the microstate ‘repressor protein bound’ as a regulatory region to the total statistical weight (see equation below). The term ‘repressor binding’ states that if a repressor is bound a polymerase cannot bind. The equation assumes a high basal level of expression in the absence of repressors. Note that like the previous equation, it takes the form of a Hill function, the gene expression fold change is dependent on the number of repressors [1,2].
</P><P><MATH xmlns:MML="http://www.w3.org/1998/Math/MathML" xmlns:M="http://schemas.openxmlformats.org/officeDocument/2006/math" display="block"><MI>F</MI><MI>o</MI><MI>l</MI><MI>d</MI><MI>C</MI><MI>h</MI><MI>a</MI><MI>n</MI><MI>g</MI><MI>e</MI><MO>=</MO><MFRAC><MROW><MI>P</MI><MI>b</MI><MI>o</MI><MI>u</MI><MI>n</MI><MI>d</MI><MO>(</MO><MI>R</MI><MO>≠</MO><MN>0</MN><MO>)</MO></MROW><MROW><MI>P</MI><MI>b</MI><MI>o</MI><MI>u</MI><MI>n</MI><MI>d</MI><MO>(</MO><MI>R</MI><MO>=</MO><MN>0</MN><MO>)</MO></MROW></MFRAC><MO>=</MO><MFRAC><MROW><MN>1</MN><MO>+</MO><MFRAC><MROW><MI>P</MI></MROW><MROW><MSUB><MROW><MI>N</MI></MROW><MROW><MI>n</MI><MI>s</MI></MROW></MSUB></MROW></MFRAC><MSUP><MROW><MI>e</MI></MROW><MROW><MO>-</MO><MFRAC><MROW><MO>∆</MO><MSUBSUP><MROW><MI>ε</MI></MROW><MROW><MI>p</MI><MI>d</MI></MROW><MROW><MI>S</MI></MROW></MSUBSUP></MROW><MROW><MSUB><MROW><MI>K</MI></MROW><MROW><MI>b</MI></MROW></MSUB><MI>T</MI></MROW></MFRAC></MROW></MSUP></MROW><MROW><MN>1</MN><MO>+</MO><MFRAC><MROW><MI>P</MI></MROW><MROW><MSUB><MROW><MI>N</MI></MROW><MROW><MI>n</MI><MI>s</MI></MROW></MSUB></MROW></MFRAC><MSUP><MROW><MI>e</MI></MROW><MROW><MO>-</MO><MFRAC><MROW><MO>∆</MO><MSUBSUP><MROW><MI>ε</MI></MROW><MROW><MI>p</MI><MI>d</MI></MROW><MROW><MI>S</MI></MROW></MSUBSUP></MROW><MROW><MSUB><MROW><MI>K</MI></MROW><MROW><MI>b</MI></MROW></MSUB><MI>T</MI></MROW></MFRAC></MROW></MSUP><MO>+</MO><MO>(</MO><MFRAC><MROW><MI>R</MI></MROW><MROW><MSUB><MROW><MI>N</MI></MROW><MROW><MI>n</MI><MI>s</MI></MROW></MSUB></MROW></MFRAC><MSUP><MROW><MI>e</MI></MROW><MROW><MO>-</MO><MFRAC><MROW><MO>∆</MO><MSUBSUP><MROW><MI>ε</MI></MROW><MROW><MI>d</MI></MROW><MROW><MI>r</MI></MROW></MSUBSUP></MROW><MROW><MSUB><MROW><MI>K</MI></MROW><MROW><MI>b</MI></MROW></MSUB><MI>T</MI></MROW></MFRAC></MROW></MSUP><MSUP><MROW><MO>)</MO></MROW><MROW><MI>n</MI></MROW></MSUP></MROW></MFRAC></MATH></P><P>
R is the number of repressor proteins and ∆ε_d<SUP>r</SUP> the specific binding energy of the repressor proteins, the same combinatorics rules apply repressor binding as for polymerase binding.
n is the number of repression regulators should those regulators have the same binding energy. Because they act independently on whether or not an RNA polymerase can bind, the regulatory values are multiplied.
The equation can now be incorporated into the model describing the Kill-Switch by tying production rates to the fold change of gene expression (K<SUB>prod</SUB> x Fold change), i.e. the production rate will decrease proportionally to changes in gene expression.  
</P><H3>Model Equations</H3><P>
Ordinary differential equations describe the genetic circuit, three of based on the statistical mechanics approach. The Deg<SUB>protein</SUB> and Dil<SUB>div</SUB> terms represent the half-life and dilution rate of the proteins. Because the third equation (the CIλ production) relies on an external input and is not dependent on any of the other equations, the ordinary differential equations uses Michaelis-Menten kinetics to simulate a fusaric acid input. In every simulation fusaric acid is inputted at t = 10 hours [1,2,10].
</P><P><MATH xmlns:MML="http://www.w3.org/1998/Math/MathML" xmlns:M="http://schemas.openxmlformats.org/officeDocument/2006/math"><MFRAC><MROW><MI mathvariant="bold-italic">d</MI><MI mathvariant="bold-italic">L</MI><MI mathvariant="bold-italic">a</MI><MI mathvariant="bold-italic">c</MI><MI mathvariant="bold-italic">I</MI></MROW><MROW><MI mathvariant="bold-italic">d</MI><MI mathvariant="bold-italic">t</MI></MROW></MFRAC><MO>=</MO><MSUB><MROW><MI mathvariant="bold-italic">K</MI></MROW><MROW><MI mathvariant="bold-italic">p</MI><MI mathvariant="bold-italic">r</MI><MI mathvariant="bold-italic">o</MI><MSUB><MROW><MI mathvariant="bold-italic">d</MI></MROW><MROW><MI mathvariant="bold-italic">L</MI><MI mathvariant="bold-italic">a</MI><MI mathvariant="bold-italic">c</MI><MI mathvariant="bold-italic">I</MI></MROW></MSUB></MROW></MSUB><MFRAC><MROW><MN>1</MN><MO>+</MO><MFRAC><MROW><MI mathvariant="bold-italic">P</MI></MROW><MROW><MSUB><MROW><MI mathvariant="bold-italic">N</MI></MROW><MROW><MI mathvariant="bold-italic">n</MI><MI mathvariant="bold-italic">s</MI></MROW></MSUB></MROW></MFRAC><MSUP><MROW><MI mathvariant="bold-italic">e</MI></MROW><MROW><MO>-</MO><MFRAC><MROW><MO>∆</MO><MSUBSUP><MROW><MI mathvariant="bold-italic">ε</MI></MROW><MROW><MI mathvariant="bold-italic">p</MI><MI mathvariant="bold-italic">d</MI></MROW><MROW><MI mathvariant="bold-italic">S</MI></MROW></MSUBSUP></MROW><MROW><MSUB><MROW><MI mathvariant="bold-italic">K</MI></MROW><MROW><MI mathvariant="bold-italic">b</MI></MROW></MSUB><MI mathvariant="bold-italic">T</MI></MROW></MFRAC></MROW></MSUP></MROW><MROW><MN>1</MN><MO>+</MO><MFRAC><MROW><MI mathvariant="bold-italic">P</MI></MROW><MROW><MSUB><MROW><MI mathvariant="bold-italic">N</MI></MROW><MROW><MI mathvariant="bold-italic">n</MI><MI mathvariant="bold-italic">s</MI></MROW></MSUB></MROW></MFRAC><MSUP><MROW><MI mathvariant="bold-italic">e</MI></MROW><MROW><MO>-</MO><MFRAC><MROW><MO>∆</MO><MSUBSUP><MROW><MI mathvariant="bold-italic">ε</MI></MROW><MROW><MI mathvariant="bold-italic">p</MI><MI mathvariant="bold-italic">d</MI></MROW><MROW><MI mathvariant="bold-italic">S</MI></MROW></MSUBSUP></MROW><MROW><MSUB><MROW><MI mathvariant="bold-italic">K</MI></MROW><MROW><MI mathvariant="bold-italic">b</MI></MROW></MSUB><MI mathvariant="bold-italic">T</MI></MROW></MFRAC></MROW></MSUP><MO>+</MO><MO>(</MO><MFRAC><MROW><MI mathvariant="bold-italic">A</MI><MI mathvariant="bold-italic">V</MI><MI mathvariant="bold-italic">T</MI><MI mathvariant="bold-italic">e</MI><MI mathvariant="bold-italic">t</MI><MI mathvariant="bold-italic">R</MI></MROW><MROW><MSUB><MROW><MI mathvariant="bold-italic">N</MI></MROW><MROW><MI mathvariant="bold-italic">n</MI><MI mathvariant="bold-italic">s</MI></MROW></MSUB></MROW></MFRAC><MSUP><MROW><MI mathvariant="bold-italic">e</MI></MROW><MROW><MO>-</MO><MFRAC><MROW><MO>∆</MO><MSUBSUP><MROW><MI mathvariant="bold-italic">ε</MI></MROW><MROW><MI mathvariant="bold-italic">d</MI></MROW><MROW><MI mathvariant="bold-italic">T</MI><MI mathvariant="bold-italic">e</MI><MI mathvariant="bold-italic">t</MI><MI mathvariant="bold-italic">R</MI><MO>_</MO><MI mathvariant="bold-italic">L</MI><MI mathvariant="bold-italic">a</MI><MI mathvariant="bold-italic">c</MI></MROW></MSUBSUP></MROW><MROW><MSUB><MROW><MI mathvariant="bold-italic">K</MI></MROW><MROW><MI mathvariant="bold-italic">b</MI></MROW></MSUB><MI mathvariant="bold-italic">T</MI></MROW></MFRAC></MROW></MSUP><MSUP><MROW><MO>)</MO></MROW><MROW><MI mathvariant="bold-italic">n</MI><MI mathvariant="bold-italic">T</MI><MI mathvariant="bold-italic">e</MI><MI mathvariant="bold-italic">t</MI><MO>_</MO><MI mathvariant="bold-italic">L</MI><MI mathvariant="bold-italic">a</MI><MI mathvariant="bold-italic">c</MI></MROW></MSUP></MROW></MFRAC><MSUB><MROW><MO>-</MO><MI mathvariant="bold-italic">D</MI><MI mathvariant="bold-italic">e</MI><MI mathvariant="bold-italic">g</MI></MROW><MROW><MI mathvariant="bold-italic">L</MI><MI mathvariant="bold-italic">a</MI><MI mathvariant="bold-italic">c</MI><MI mathvariant="bold-italic">I</MI></MROW></MSUB><MI mathvariant="bold-italic">L</MI><MI mathvariant="bold-italic">a</MI><MI mathvariant="bold-italic">c</MI><MI mathvariant="bold-italic">I</MI><MO>-</MO><MSUB><MROW><MI mathvariant="bold-italic">D</MI><MI mathvariant="bold-italic">i</MI><MI mathvariant="bold-italic">l</MI></MROW><MROW><MI mathvariant="bold-italic">d</MI><MI mathvariant="bold-italic">i</MI><MI mathvariant="bold-italic">v</MI></MROW></MSUB><MI mathvariant="bold-italic">L</MI><MI mathvariant="bold-italic">a</MI><MI mathvariant="bold-italic">c</MI><MI mathvariant="bold-italic">I</MI></MATH></P><P><MATH xmlns:MML="http://www.w3.org/1998/Math/MathML" xmlns:M="http://schemas.openxmlformats.org/officeDocument/2006/math"><MFRAC><MROW><MI mathvariant="bold-italic">d</MI><MI mathvariant="bold-italic">L</MI><MI mathvariant="bold-italic">a</MI><MI mathvariant="bold-italic">c</MI><MI mathvariant="bold-italic">I</MI></MROW><MROW><MI mathvariant="bold-italic">d</MI><MI mathvariant="bold-italic">t</MI></MROW></MFRAC><MO>=</MO><MSUB><MROW><MI mathvariant="bold-italic">K</MI></MROW><MROW><MI mathvariant="bold-italic">p</MI><MI mathvariant="bold-italic">r</MI><MI mathvariant="bold-italic">o</MI><MSUB><MROW><MI mathvariant="bold-italic">d</MI></MROW><MROW><MI mathvariant="bold-italic">L</MI><MI mathvariant="bold-italic">a</MI><MI mathvariant="bold-italic">c</MI><MI mathvariant="bold-italic">I</MI></MROW></MSUB></MROW></MSUB><MFRAC><MROW><MN>1</MN><MO>+</MO><MFRAC><MROW><MI mathvariant="bold-italic">P</MI></MROW><MROW><MSUB><MROW><MI mathvariant="bold-italic">N</MI></MROW><MROW><MI mathvariant="bold-italic">n</MI><MI mathvariant="bold-italic">s</MI></MROW></MSUB></MROW></MFRAC><MSUP><MROW><MI mathvariant="bold-italic">e</MI></MROW><MROW><MO>-</MO><MFRAC><MROW><MO>∆</MO><MSUBSUP><MROW><MI mathvariant="bold-italic">ε</MI></MROW><MROW><MI mathvariant="bold-italic">p</MI><MI mathvariant="bold-italic">d</MI></MROW><MROW><MI mathvariant="bold-italic">S</MI></MROW></MSUBSUP></MROW><MROW><MSUB><MROW><MI mathvariant="bold-italic">K</MI></MROW><MROW><MI mathvariant="bold-italic">b</MI></MROW></MSUB><MI mathvariant="bold-italic">T</MI></MROW></MFRAC></MROW></MSUP></MROW><MROW><MN>1</MN><MO>+</MO><MFRAC><MROW><MI mathvariant="bold-italic">P</MI></MROW><MROW><MSUB><MROW><MI mathvariant="bold-italic">N</MI></MROW><MROW><MI mathvariant="bold-italic">n</MI><MI mathvariant="bold-italic">s</MI></MROW></MSUB></MROW></MFRAC><MSUP><MROW><MI mathvariant="bold-italic">e</MI></MROW><MROW><MO>-</MO><MFRAC><MROW><MO>∆</MO><MSUBSUP><MROW><MI mathvariant="bold-italic">ε</MI></MROW><MROW><MI mathvariant="bold-italic">p</MI><MI mathvariant="bold-italic">d</MI></MROW><MROW><MI mathvariant="bold-italic">S</MI></MROW></MSUBSUP></MROW><MROW><MSUB><MROW><MI mathvariant="bold-italic">K</MI></MROW><MROW><MI mathvariant="bold-italic">b</MI></MROW></MSUB><MI mathvariant="bold-italic">T</MI></MROW></MFRAC></MROW></MSUP><MO>+</MO><MO>(</MO><MFRAC><MROW><MI mathvariant="bold-italic">A</MI><MI mathvariant="bold-italic">V</MI><MI mathvariant="bold-italic">T</MI><MI mathvariant="bold-italic">e</MI><MI mathvariant="bold-italic">t</MI><MI mathvariant="bold-italic">R</MI></MROW><MROW><MSUB><MROW><MI mathvariant="bold-italic">N</MI></MROW><MROW><MI mathvariant="bold-italic">n</MI><MI mathvariant="bold-italic">s</MI></MROW></MSUB></MROW></MFRAC><MSUP><MROW><MI mathvariant="bold-italic">e</MI></MROW><MROW><MO>-</MO><MFRAC><MROW><MO>∆</MO><MSUBSUP><MROW><MI mathvariant="bold-italic">ε</MI></MROW><MROW><MI mathvariant="bold-italic">d</MI></MROW><MROW><MI mathvariant="bold-italic">T</MI><MI mathvariant="bold-italic">e</MI><MI mathvariant="bold-italic">t</MI><MI mathvariant="bold-italic">R</MI><MO>_</MO><MI mathvariant="bold-italic">L</MI><MI mathvariant="bold-italic">a</MI><MI mathvariant="bold-italic">c</MI></MROW></MSUBSUP></MROW><MROW><MSUB><MROW><MI mathvariant="bold-italic">K</MI></MROW><MROW><MI mathvariant="bold-italic">b</MI></MROW></MSUB><MI mathvariant="bold-italic">T</MI></MROW></MFRAC></MROW></MSUP><MSUP><MROW><MO>)</MO></MROW><MROW><MI mathvariant="bold-italic">n</MI><MI mathvariant="bold-italic">T</MI><MI mathvariant="bold-italic">e</MI><MI mathvariant="bold-italic">t</MI><MO>_</MO><MI mathvariant="bold-italic">L</MI><MI mathvariant="bold-italic">a</MI><MI mathvariant="bold-italic">c</MI></MROW></MSUP></MROW></MFRAC><MSUB><MROW><MO>-</MO><MI mathvariant="bold-italic">D</MI><MI mathvariant="bold-italic">e</MI><MI mathvariant="bold-italic">g</MI></MROW><MROW><MI mathvariant="bold-italic">L</MI><MI mathvariant="bold-italic">a</MI><MI mathvariant="bold-italic">c</MI><MI mathvariant="bold-italic">I</MI></MROW></MSUB><MI mathvariant="bold-italic">L</MI><MI mathvariant="bold-italic">a</MI><MI mathvariant="bold-italic">c</MI><MI mathvariant="bold-italic">I</MI><MO>-</MO><MSUB><MROW><MI mathvariant="bold-italic">D</MI><MI mathvariant="bold-italic">i</MI><MI mathvariant="bold-italic">l</MI></MROW><MROW><MI mathvariant="bold-italic">d</MI><MI mathvariant="bold-italic">i</MI><MI mathvariant="bold-italic">v</MI></MROW></MSUB><MI mathvariant="bold-italic">L</MI><MI mathvariant="bold-italic">a</MI><MI mathvariant="bold-italic">c</MI><MI mathvariant="bold-italic">I</MI></MATH></P><P><MATH xmlns:MML="http://www.w3.org/1998/Math/MathML" xmlns:M="http://schemas.openxmlformats.org/officeDocument/2006/math"><MFRAC><MROW><MI mathvariant="bold-italic">d</MI><MI mathvariant="bold-italic">C</MI><MI mathvariant="bold-italic">I</MI><MI mathvariant="bold-italic">λ</MI></MROW><MROW><MI mathvariant="bold-italic">d</MI><MI mathvariant="bold-italic">t</MI></MROW></MFRAC><MO>=</MO><MSUB><MROW><MI mathvariant="bold-italic">K</MI></MROW><MROW><MI mathvariant="bold-italic">p</MI><MI mathvariant="bold-italic">r</MI><MI mathvariant="bold-italic">o</MI><MSUB><MROW><MI mathvariant="bold-italic">d</MI></MROW><MROW><MI mathvariant="bold-italic">C</MI><MI mathvariant="bold-italic">I</MI><MI mathvariant="bold-italic">λ</MI></MROW></MSUB></MROW></MSUB><MFRAC><MROW><MO>[</MO><MI mathvariant="bold-italic">f</MI><MI mathvariant="bold-italic">u</MI><MI mathvariant="bold-italic">s</MI><MI mathvariant="bold-italic">a</MI><MI mathvariant="bold-italic">r</MI><MI mathvariant="bold-italic">i</MI><MI mathvariant="bold-italic">c</MI><MI mathvariant="bold-italic">a</MI><MI mathvariant="bold-italic">c</MI><MI mathvariant="bold-italic">i</MI><MI mathvariant="bold-italic">d</MI><MO>]</MO></MROW><MROW><MFENCED open="[" close="]" separators="|"><MROW><MI mathvariant="bold-italic">f</MI><MI mathvariant="bold-italic">u</MI><MI mathvariant="bold-italic">s</MI><MI mathvariant="bold-italic">a</MI><MI mathvariant="bold-italic">r</MI><MI mathvariant="bold-italic">i</MI><MI mathvariant="bold-italic">c</MI><MI mathvariant="bold-italic">a</MI><MI mathvariant="bold-italic">c</MI><MI mathvariant="bold-italic">i</MI><MI mathvariant="bold-italic">d</MI></MROW></MFENCED><MO>+</MO><MSUB><MROW><MI mathvariant="bold-italic">K</MI></MROW><MROW><MI mathvariant="bold-italic">d</MI></MROW></MSUB></MROW></MFRAC><MO>-</MO><MSUB><MROW><MI mathvariant="bold-italic">d</MI><MI mathvariant="bold-italic">e</MI><MI mathvariant="bold-italic">g</MI></MROW><MROW><MI mathvariant="bold-italic">C</MI><MI mathvariant="bold-italic">I</MI><MI mathvariant="bold-italic">λ</MI></MROW></MSUB><MI mathvariant="bold-italic">C</MI><MI mathvariant="bold-italic">I</MI><MI mathvariant="bold-italic">λ</MI><MO>-</MO><MSUB><MROW><MI mathvariant="bold-italic">D</MI><MI mathvariant="bold-italic">i</MI><MI mathvariant="bold-italic">l</MI></MROW><MROW><MI mathvariant="bold-italic">d</MI><MI mathvariant="bold-italic">i</MI><MI mathvariant="bold-italic">v</MI></MROW></MSUB><MI mathvariant="bold-italic">C</MI><MI mathvariant="bold-italic">I</MI><MI mathvariant="bold-italic">λ</MI></MATH></P><P><MATH xmlns:MML="http://www.w3.org/1998/Math/MathML" xmlns:M="http://schemas.openxmlformats.org/officeDocument/2006/math"><MFRAC><MROW><MI mathvariant="bold-italic">d</MI><MI mathvariant="bold-italic">T</MI><MI mathvariant="bold-italic">o</MI><MI mathvariant="bold-italic">x</MI><MI mathvariant="bold-italic">i</MI><MI mathvariant="bold-italic">n</MI></MROW><MROW><MI mathvariant="bold-italic">d</MI><MI mathvariant="bold-italic">t</MI></MROW></MFRAC><MO>=</MO><MSUB><MROW><MI mathvariant="bold-italic">K</MI></MROW><MROW><MI mathvariant="bold-italic">p</MI><MI mathvariant="bold-italic">r</MI><MI mathvariant="bold-italic">o</MI><MSUB><MROW><MI mathvariant="bold-italic">d</MI></MROW><MROW><MI mathvariant="bold-italic">L</MI><MI mathvariant="bold-italic">a</MI><MI mathvariant="bold-italic">c</MI><MI mathvariant="bold-italic">I</MI></MROW></MSUB></MROW></MSUB><MFRAC><MROW><MN>1</MN><MO>+</MO><MFRAC><MROW><MI mathvariant="bold-italic">P</MI></MROW><MROW><MSUB><MROW><MI mathvariant="bold-italic">N</MI></MROW><MROW><MI mathvariant="bold-italic">n</MI><MI mathvariant="bold-italic">s</MI></MROW></MSUB></MROW></MFRAC><MSUP><MROW><MI mathvariant="bold-italic">e</MI></MROW><MROW><MO>-</MO><MFRAC><MROW><MO>∆</MO><MSUBSUP><MROW><MI mathvariant="bold-italic">ε</MI></MROW><MROW><MI mathvariant="bold-italic">p</MI><MI mathvariant="bold-italic">d</MI></MROW><MROW><MI mathvariant="bold-italic">S</MI></MROW></MSUBSUP></MROW><MROW><MSUB><MROW><MI mathvariant="bold-italic">K</MI></MROW><MROW><MI mathvariant="bold-italic">b</MI></MROW></MSUB><MI mathvariant="bold-italic">T</MI></MROW></MFRAC></MROW></MSUP></MROW><MROW><MN>1</MN><MO>+</MO><MFRAC><MROW><MI mathvariant="bold-italic">P</MI></MROW><MROW><MSUB><MROW><MI mathvariant="bold-italic">N</MI></MROW><MROW><MI mathvariant="bold-italic">n</MI><MI mathvariant="bold-italic">s</MI></MROW></MSUB></MROW></MFRAC><MSUP><MROW><MI mathvariant="bold-italic">e</MI></MROW><MROW><MO>-</MO><MFRAC><MROW><MO>∆</MO><MSUBSUP><MROW><MI mathvariant="bold-italic">ε</MI></MROW><MROW><MI mathvariant="bold-italic">p</MI><MI mathvariant="bold-italic">d</MI></MROW><MROW><MI mathvariant="bold-italic">S</MI></MROW></MSUBSUP></MROW><MROW><MSUB><MROW><MI mathvariant="bold-italic">K</MI></MROW><MROW><MI mathvariant="bold-italic">b</MI></MROW></MSUB><MI mathvariant="bold-italic">T</MI></MROW></MFRAC></MROW></MSUP><MO>+</MO><MO>(</MO><MFRAC><MROW><MI mathvariant="bold-italic">A</MI><MI mathvariant="bold-italic">V</MI><MI mathvariant="bold-italic">T</MI><MI mathvariant="bold-italic">e</MI><MI mathvariant="bold-italic">t</MI><MI mathvariant="bold-italic">R</MI></MROW><MROW><MSUB><MROW><MI mathvariant="bold-italic">N</MI></MROW><MROW><MI mathvariant="bold-italic">n</MI><MI mathvariant="bold-italic">s</MI></MROW></MSUB></MROW></MFRAC><MSUP><MROW><MI mathvariant="bold-italic">e</MI></MROW><MROW><MO>-</MO><MFRAC><MROW><MO>∆</MO><MSUBSUP><MROW><MI mathvariant="bold-italic">ε</MI></MROW><MROW><MI mathvariant="bold-italic">d</MI></MROW><MROW><MI mathvariant="bold-italic">T</MI><MI mathvariant="bold-italic">e</MI><MI mathvariant="bold-italic">t</MI><MSUB><MROW><MI mathvariant="bold-italic">R</MI></MROW><MROW><MI mathvariant="bold-italic">t</MI><MI mathvariant="bold-italic">o</MI><MI mathvariant="bold-italic">x</MI><MI mathvariant="bold-italic">i</MI><MI mathvariant="bold-italic">n</MI></MROW></MSUB></MROW></MSUBSUP></MROW><MROW><MSUB><MROW><MI mathvariant="bold-italic">K</MI></MROW><MROW><MI mathvariant="bold-italic">b</MI></MROW></MSUB><MI mathvariant="bold-italic">T</MI></MROW></MFRAC></MROW></MSUP><MSUP><MROW><MO>)</MO></MROW><MROW><MI mathvariant="bold-italic">n</MI><MI mathvariant="bold-italic">T</MI><MI mathvariant="bold-italic">e</MI><MSUB><MROW><MI mathvariant="bold-italic">t</MI></MROW><MROW><MI mathvariant="bold-italic">T</MI><MI mathvariant="bold-italic">o</MI><MI mathvariant="bold-italic">x</MI><MI mathvariant="bold-italic">i</MI><MI mathvariant="bold-italic">n</MI></MROW></MSUB></MROW></MSUP><MO>(</MO><MFRAC><MROW><MI mathvariant="bold-italic">A</MI><MI mathvariant="bold-italic">V</MI><MI mathvariant="bold-italic">C</MI><MI mathvariant="bold-italic">I</MI><MI mathvariant="bold-italic">λ</MI></MROW><MROW><MSUB><MROW><MI mathvariant="bold-italic">N</MI></MROW><MROW><MI mathvariant="bold-italic">n</MI><MI mathvariant="bold-italic">s</MI></MROW></MSUB></MROW></MFRAC><MSUP><MROW><MI mathvariant="bold-italic">e</MI></MROW><MROW><MO>-</MO><MFRAC><MROW><MO>∆</MO><MSUBSUP><MROW><MI mathvariant="bold-italic">ε</MI></MROW><MROW><MI mathvariant="bold-italic">d</MI></MROW><MROW><MI mathvariant="bold-italic">C</MI><MI mathvariant="bold-italic">I</MI><MI mathvariant="bold-italic">λ</MI><MI>t</MI><MI>o</MI><MI>x</MI><MI>i</MI><MI>n</MI></MROW></MSUBSUP></MROW><MROW><MSUB><MROW><MI mathvariant="bold-italic">K</MI></MROW><MROW><MI mathvariant="bold-italic">b</MI></MROW></MSUB><MI mathvariant="bold-italic">T</MI></MROW></MFRAC></MROW></MSUP><MSUP><MROW><MO>)</MO></MROW><MROW><MI mathvariant="bold-italic">n</MI><MI mathvariant="bold-italic">C</MI><MI mathvariant="bold-italic">I</MI><MI mathvariant="bold-italic">λ</MI><MO>_</MO><MI mathvariant="bold-italic">G</MI><MI mathvariant="bold-italic">F</MI><MI mathvariant="bold-italic">P</MI></MROW></MSUP></MROW></MFRAC><MSUB><MROW><MO>-</MO><MI mathvariant="bold-italic">d</MI><MI mathvariant="bold-italic">e</MI><MI mathvariant="bold-italic">g</MI></MROW><MROW><MI mathvariant="bold-italic">T</MI><MI mathvariant="bold-italic">o</MI><MI mathvariant="bold-italic">x</MI><MI mathvariant="bold-italic">i</MI><MI mathvariant="bold-italic">n</MI></MROW></MSUB><MI mathvariant="bold-italic">T</MI><MI mathvariant="bold-italic">o</MI><MI mathvariant="bold-italic">x</MI><MI mathvariant="bold-italic">i</MI><MI mathvariant="bold-italic">n</MI><MO>-</MO><MSUB><MROW><MI mathvariant="bold-italic">D</MI><MI mathvariant="bold-italic">i</MI><MI mathvariant="bold-italic">l</MI></MROW><MROW><MI mathvariant="bold-italic">d</MI><MI mathvariant="bold-italic">i</MI><MI mathvariant="bold-italic">v</MI><MI mathvariant="bold-italic">t</MI><MI mathvariant="bold-italic">o</MI><MI mathvariant="bold-italic">x</MI><MI mathvariant="bold-italic">i</MI><MI mathvariant="bold-italic">n</MI></MROW></MSUB></MATH></P><H3>Software used</H3><H3><P>
All models were made in Python (Python software foundation, Delaware, US) version 2.71 for Mac OS X version 10.5.8. Solving of ODE systems was done using the standard soln function in Python 
</P></H3><H2 id="results1">Results</H2><H3>System Behaviour and Optimal Repressor Binding Site Configuration</H3><P>
Before the optimal configuration of repressor binding sites could be identified, the behaviour of the system had to be assigned scores representing a functioning or non-functioning Kill-switch. Three distinct behaviours could be identified:
</P><FIGURE><FIGCAPTION style="text-align:left">0: The Kill-switch does not work, the toggle switch is out of balance and does not function. </FIGCAPTION></FIGURE><FIGURE><FIGCAPTION style="text-align:left">1: The system performs less efficiently, though the toggle switch changes state, the toxin promoter is leaky. </FIGCAPTION></FIGURE><FIGURE><FIGCAPTION style="text-align:left">2: The system performs to design, after a fusaric acid input the toggle switch changes state and toxin is produced when  CIλ leaves the system.</FIGCAPTION></FIGURE><P>
In total eight (A,B,..,H) different configurations of repressor binding sites were tested (Figure 4). Two: F and H, were found to be produce Kill-switch systems robust towards variations in binding strengths and production rates. The result of the best scoring configuration H has been magnified (figure 5). The repressor binding site configuration corresponding to this map has the highest chance of working in an experimental set-up. The x-axis shows the fold change in protein production, the y-axis has increased binding strengths for individual repressors by an order of magnitude each row. The color map indicates that increases in repressor binding strengths can be compensated for by increased production rates of either the target gene or decreased production of the repressor itself. The practical implications for constructing the an optimized version of the Kill-switch were as follows: two TetR repressor binding sites on the LacI promoter, two LacI and one CIλ repressor binding site on the TetR promoter, two TetR and one CIλ repressor binding site on the toxin promoter.
</P><FIGURE><FIGCAPTION><B>Figure 4.</B> Color maps indicating functioning and non-functioning systems. Each letter represents different repressor binding site configurations. Each small square within the color maps represents a score for a simulation of the system with a unique set of parameters. The colors correspond to a working system (2:red), leaky system (1:green) and a non-functioning system (0:blue). The table underneath shows the configuration of the repressor binding sites. The type of repressor binding site is specified followed by the promoter it is located on. The letters correspond to the colormaps </FIGCAPTION></FIGURE><FIGURE><FIGCAPTION><B>Figure 5.</B> The best scoring repressor binding site configuration H. Each small square within the colour maps represents a score for a simulation of the system with a unique set of parameters. The x-axis shows the production rates in uM/min. The Y-axis has the repressor binding strengths. Using the lowest value found in literature [11], each row the value for th repressor binding strength increases by an order of magnitude.</FIGCAPTION></FIGURE><H3>Statistical Mechanics in Practice</H3><P>
There were two main reasons for using statistical mechanics:<B>(I)</B> Statistical mechanics provides a way describe the system in terms of the number of repressors and repressor binding sites, their binding affinity and the position of the repressor binding on the promoter [1,2].<B>(II)</B> In contrast to Michaelis-Menten kinetics, statistical mechanics offers more freedom studying the behavior of differently designed promoters.Michaelis-Menten requires the dissociation constants of the repressor proteins to be experimentally determined [1,2]. 
Based on the results of the model the experimentalists ordered oligonucleotides and assembled six promoters. Building two Kill-switches in parallel the chance of constructing a stable system increases. The assumption that the system is optimized and stable has been incorporated in the model quantifying stability .
</P></SECTION><SECTION id="cost1"><H1>System Cost</H1><H2 id="introduction2">Introduction</H2><P>
BananaGuard was designed without knowing whether our engineered <I>Pseudomonas putida</I> would be outcompeted by other bacteria and fungi found in the rhizosphere of banana roots [12, 13].  For a successful application it is crucial that our strain can survive and compete near the banana plant roots. This seemed challenging as our system consists of multiple constitutively expressed genes controlling the <A href="https://2014.igem.org/Team:Wageningen_UR/project/kill-switch" class="soft_link">kill switch</A> and <A href="https://2014.igem.org/Team:Wageningen_UR/project/gene_transfer" class="soft_link">toxin-antitoxin</A> systems. Moreover, when the system is switched to the active state by <A href="https://2014.igem.org/Team:Wageningen_UR/project/fungal_sensing#wetlab" class="soft_link">sensing fusaric acid</A>, BananaGuard produces multiple <A href="https://2014.igem.org/Team:Wageningen_UR/project/fungal_sensing#wetlab" class="soft_link">fungal growth inhibitor enzymes and compounds</A>. The The results display the intrinsic behavior of Kill-switch stability and performance.integrated synthetic pathways use metabolic resources that would otherwise be dedicated to cellular maintenance or growth. The synthetic pathways therefore reduce the bacteria's potential to sustain themselves in the rhizosphere. Because of this reduced potential we wondered to what degree <I>P. putida</I> metabolism was affected by the introduced synthetic genes and if need be, to what extent the system could be changed to lessen the impact. To approach this problem we used a genome-scale metabolic model (GSMM) [14]. Our GSMM extended by the BananaGuard system predicts the interaction of our integrated plasmids with genes originating from endogenous pathways. With this prediction of the metabolic stress we gain insight in the metabolic burden caused by the integration of our plasmids. This will give an indication about  BananaGuard if it will lose its capability to compete with other rhizosphere-populating micro-organisms in its resting state.

Continue to learn more about a genomic-scale metabolic model or view the <A href="https://2014.igem.org/Team:Wageningen_UR/project/model#results2" class="soft_link">results</A>. 
</P><H2 id="approach2">Theory and experimental design</H2><P>
In order to answer our questions with a GSMM we need a model that:
<UL><LI>contains a comprehensive representation of <I>P. putida</I> metabolism</LI><LI>is capable of predicting growth rates in various environments</LI><LI>includes all reactions that are important for the BananaGuard system</LI></UL></P><P>
For this reason a comparison was made between various available GSMMs for <I>P. putida</I>. 
We took a in-depth look in two models that were up-to-date, published and verified: iJP962 and iJN746[15,16]. 
For the necessary work to model the metabolic stress in <I>P. putida</I>, the best starting model needs to be chosen which has the most potential for accurately modeling the metabolic stress of our system.
</P><FIGURE><FIGCAPTION>Table 1. Comparison between iJP962 and iJN746</FIGCAPTION><P>
According to the previously stated requirements,the model iJP962 is most suitable for modeling our system. The biggest difference between these models shown in table 1 is the number of compartments.  But this is not necessarily an advantage. Many of the reactions are over the internal membrane and not functional for the core intermediate metabolites. Added to this, iJN746 has a focus on compounds that are in a different class of metabolites than the fungal growth inhibitors we chose for our system [27]. Therefore iJP962 is more representative due to more reactions and more metabolites in a broader spectrum. The primary category of concern for fungal growth inhibitor production is the amino acid metabolism and iJP962 has the most reactions in this category [14,15,17]. neither model included all the specific reactions, metabolites and genes that are necessary for modeling the fungal growth inhibitor metabolites. These reactions need to be added manually to the GSMM.
</P><H3>Extending the model </H3><P>
Traditionally, metabolic models only predict metabolic fluxes. Although this is directly applicable to the metabolites that we want to produce, such as DMDS, this does not suffice to answer our questions regarding the impact of the whole system. Therefore, we added the production of all the proteins of interest as if they are metabolites. For this we used the protein sequences to determine the correct stoichiometric ratio of amino acids [18]. The other side-substrates/products are in the same ratio as described for production of proteins for the biomass reaction in the GSMM. In the model <A href="https://2014.igem.org/Team:Wageningen_UR/project/model" class="soft_link">Promoter Design</A> the protein production rate is estimated.
</P><P>
As our metabolites of interest (DAPG, DMDS, DMTS) were not yet included in the GSMM, we had to deduce their production pathways based on databases and literature information [19,20,21,22](<A href="https://static.igem.org/mediawiki/2014/7/7d/Wageningen_UR_protocols_Metabolicmodelingpathways.pdf" target="blank" class="soft_link">pathways and metabolites</A>). Furthermore, we took special care that all added pathways are stoichiometrically balanced, which is essential for proper and reliable integration in a GSMM. To get the most realistic results we need to set a biologically relevant production rate. This can be done by estimating a production range with <I>in vivo</I> data from literature.
</P><H3> Programs and toolboxes</H3><P>
Matlab[23] was used in combination with the Cobra toolbox[24] for handling the model. The Cobra toolbox is designed for working with genome-scale metabolic models. To solve the flux balance analysis a linear program solver is needed. Gurobi[25] was chosen because of its compatibility with Matlab and the Cobra toolbox.
</P><H3> Flux Balance Analysis </H3><P>
Flux Balance Analysis (FBA) is a framework for investigating the theoretical capabilities of a GSMM and can be solved using a linear programming solver. This is possible because of the linear objective function and linear constraints[4,6].  
FBA is based on the following assumptions:
</P><OL><LI>Steady state: The fluxes are considered to have attained a static equilibrium value and do not change through time, this means that accumulation or dissipation is not possible. </LI><LI>Thermodynamic constraints: (ir)reversibility of reactions.</LI><LI>No enzyme saturation: The maximum flux for every reaction is not dependent on the concentration of the enzymes used for that reaction. </LI></OL><P>
The FBA method uses a representation of the metabolic reaction network in the form of a stoichiometric matrix (S) where : 
<UL><LI>Each column corresponds to a reaction R<SUB>i</SUB></LI><LI>Each row corresponds to a metabolite C<SUB>j</SUB></LI></UL></P><P>
The definition of S is : 
</P><MATH xmlns:MML="http://www.w3.org/1998/Math/MathML" xmlns:M="http://schemas.openxmlformats.org/officeDocument/2006/math" display="block"><MSUB><MROW><MI>S</MI></MROW><MROW><MI>i</MI><MO>,</MO><MI>j</MI></MROW></MSUB><MO>=</MO><MI>k</MI><MI> </MI><MI> </MI><MI>s</MI><MO>.</MO><MI>t</MI><MO>.</MO><MI> </MI><MI> </MI><MFENCED open="{" close="" separators="|"><MROW><MTABLE><MTR><MTD><MROW><MI> </MI><MI> </MI><MI>k</MI><MO>&lt;</MO><MI> </MI><MI> </MI><MN>0</MN><MI> </MI><MI> </MI><MO>→</MO><MSUB><MROW><MI>C</MI></MROW><MROW><MI>j</MI></MROW></MSUB><MI> </MI><MI> </MI><MI>i</MI><MI>s</MI><MI> </MI><MI> </MI><MI>a</MI><MI> </MI><MI> </MI><MI>s</MI><MI>u</MI><MI>b</MI><MI>s</MI><MI>t</MI><MI>r</MI><MI>a</MI><MI>t</MI><MI>e</MI><MI> </MI><MI> </MI><MI>o</MI><MI>f</MI><MI> </MI><MI> </MI><MI>r</MI><MI>e</MI><MI>a</MI><MI>c</MI><MI>t</MI><MI>i</MI><MI>o</MI><MI>n</MI><MI> </MI><MI> </MI><MSUB><MROW><MI>R</MI></MROW><MROW><MI>i</MI></MROW></MSUB><MI> </MI><MI> </MI><MI>w</MI><MI>i</MI><MI>t</MI><MI>h</MI><MI> </MI><MI> </MI><MI>s</MI><MI>t</MI><MI>o</MI><MI>i</MI><MI>c</MI><MI>h</MI><MI>i</MI><MI>o</MI><MI>m</MI><MI>e</MI><MI>t</MI><MI>r</MI><MI>y</MI><MI> </MI><MI> </MI><MI>k</MI></MROW></MTD></MTR><MTR><MTD><MROW><MI>k</MI><MO>&gt;</MO><MI> </MI><MI> </MI><MN>0</MN><MI> </MI><MI> </MI><MO>→</MO><MSUB><MROW><MI>C</MI></MROW><MROW><MI>j</MI></MROW></MSUB><MI> </MI><MI> </MI><MI>i</MI><MI>s</MI><MI> </MI><MI> </MI><MI>a</MI><MI> </MI><MI> </MI><MI>p</MI><MI>r</MI><MI>o</MI><MI>d</MI><MI>u</MI><MI>c</MI><MI>t</MI><MI> </MI><MI> </MI><MI>o</MI><MI>f</MI><MI> </MI><MI> </MI><MI>r</MI><MI>e</MI><MI>a</MI><MI>c</MI><MI>t</MI><MI>i</MI><MI>o</MI><MI>n</MI><MI> </MI><MI> </MI><MSUB><MROW><MI>R</MI></MROW><MROW><MI>i</MI></MROW></MSUB><MI> </MI><MI> </MI><MI>w</MI><MI>i</MI><MI>t</MI><MI>h</MI><MI> </MI><MI> </MI><MI>s</MI><MI>t</MI><MI>o</MI><MI>i</MI><MI>c</MI><MI>h</MI><MI>i</MI><MI>o</MI><MI>m</MI><MI>e</MI><MI>t</MI><MI>r</MI><MI>y</MI><MI> </MI><MI> </MI><MI>k</MI></MROW></MTD></MTR><MTR><MTD><MROW><MI>k</MI><MO>=</MO><MN>0</MN><MI> </MI><MI> </MI><MI> </MI><MI> </MI><MO>→</MO><MSUB><MROW><MI>C</MI></MROW><MROW><MI>j</MI></MROW></MSUB><MI> </MI><MI> </MI><MI>d</MI><MI>o</MI><MI>e</MI><MI>s</MI><MI> </MI><MI> </MI><MI>n</MI><MI>o</MI><MI>t</MI><MI> </MI><MI> </MI><MI>p</MI><MI>a</MI><MI>r</MI><MI>t</MI><MI>i</MI><MI>c</MI><MI>i</MI><MI>p</MI><MI>a</MI><MI>t</MI><MI>e</MI><MI> </MI><MI> </MI><MI>i</MI><MI>n</MI><MI> </MI><MI> </MI><MI>r</MI><MI>e</MI><MI>a</MI><MI>c</MI><MI>t</MI><MI>i</MI><MI>o</MI><MI>n</MI><MI> </MI><MI> </MI><MSUB><MROW><MI>R</MI></MROW><MROW><MI>i</MI></MROW></MSUB></MROW></MTD></MTR></MTABLE></MROW></MFENCED></MATH><P>
The FBA problem is then formulated as a maximization or minimization problem under the determined constraints: 
</P><MATH xmlns:MML="http://www.w3.org/1998/Math/MathML" xmlns:M="http://schemas.openxmlformats.org/officeDocument/2006/math" display="block"><MFENCED open="{" close="" separators="|"><MROW><MTABLE><MTR><MTD><MROW><MI>M</MI><MI>a</MI><MI>x</MI><MI>i</MI><MI>m</MI><MI>i</MI><MI>z</MI><MI>e</MI><MI> </MI><MSUP><MROW><MI mathvariant="bold-italic">c</MI></MROW><MROW><MI>T</MI></MROW></MSUP><MI>v</MI></MROW></MTD></MTR><MTR><MTD><MROW><MI>S</MI><MI>u</MI><MI>b</MI><MI>j</MI><MI>e</MI><MI>c</MI><MI>t</MI><MI> </MI><MI>t</MI><MI>o</MI><MI> </MI><MI>S</MI><MI>v</MI><MO>=</MO><MN>0</MN></MROW></MTD></MTR><MTR><MTD><MROW><MI>L</MI><MI>o</MI><MI>w</MI><MI>e</MI><MI>r</MI><MI>b</MI><MI>o</MI><MI>u</MI><MI>n</MI><MI>d</MI><MO>≤</MO><MI>v</MI><MO>≤</MO><MI>U</MI><MI>p</MI><MI>p</MI><MI>e</MI><MI>r</MI><MI>b</MI><MI>o</MI><MI>u</MI><MI>n</MI><MI>d</MI></MROW></MTD></MTR></MTABLE></MROW></MFENCED></MATH><P>
Where : 
<UL><LI><B>v</B> is the vector of to be determined reaction fluxes </LI><LI><B>c</B> is a vector of constants defining the objective function </LI><LI>S is the stoichiometric matrix </LI><LI><B>Lowerbound</B> and <B>Upperbound</B> are vectors of constraints indicating, minimal and maximal flux values for each reaction</LI></UL></P><H3> Media composition</H3><P>
GSMMs consider all metabolic possibilities given the available substrates. 
By changing the substrates and their uptake rates in the media you can dictate what’s available in the model. However, for most  metabolites the uptake rates are unknown. This can be solved setting these fluxes as unconstrained in the GSMM. This also enforces decision to not include all root components but only the most prevalent. You don’t want that rare compounds have a huge impact on the model predictions. This does imply that the model predictions could be slightly negative: with including the other available compounds the system may work even better.
</P><P>
To get the most realistic approximation for BananaGuard in the soil the M9 media composition was used with carbon as limiting nutrient. The carbon source is based on the exudates of banana roots and consists of 34% sucrose and 66% divided over multiple amino acids[15, 27].   
</P><H2 id="results2">Results</H2><P><I>P. putida</I> naturally habits the rhizosphere of plant roots.
To check if BananaGuard is still viable in its resting and active state, a comparison is important between the wild type <I>P. putida</I> and BananaGuard. If the growth rate of BananaGuard approximates the growth rate of wild type <I>P. putida</I> in the resting state it is safe to assume that the disadvantage of the introduced synthetic pathway does not have a huge impact on competitiveness with other rhizosphere-populating microorganisms. This means that BananaGuard should be able to populate the rhizosphere of the banana roots.   
For the active state it is not essential to compete with other micro-organisms but the capability to produce all the fungal growth inhibitors with a constant rate around the roots, without overloading a metabolic pathway and causing shortage of intermediates. 
</P><P>
We aim for the best approximation of the environment of the banana roots. To do this we need to estimate the carbon uptake rate of <I>P. Putida</I>. Because these units are not a standard in literature we have to calculate them. Within this GSMM it is possible to calculate the growth rate with known nutrient uptake rates. This can be reversed, so if we know the the doubling time of <I>P. putida</I> in the rhizosphere the carbon uptake rate can be calculated. This doubling time is 3 hours resulting in a growth rate of 0.23 h<SUP>-1</SUP> [1]. From this we can calculate the carbon uptake rate in the rhizosphere which is around 4 mmol gDW<SUP>-1</SUP>hr<SUP>-1</SUP>.
</P><P>
Figure 5 depicts how the growth rate of BananaGuard compares to the growth rate of the wild type <I>P. putida</I> depending on the carbon uptake rate of the organism in the resting state and active state of the kill switch. Glucose was chosen as carbon source reference, because of the efficient degradation of glucose.
BananaGuard is still able to grow at almost max growth rate (&gt;99%) compared to the wild type in the resting state. We can conclude from this that BananaGuard is not outcompeted by other rhizosphere-populating microorganisms because of the synthetic pathway.
When the kill switch is activated by fusaric acid, BananaGuard is still able to growth with 50-65% of the maximum growth of the wild type (Figure 5). This indicates that metabolic stress is not a bottleneck for the production of fungal growth inhibitors in our fusaric acid activated system.
</P><P>
Next we want to know if our conclusions made for glucose are also valid for the environment close to the banana roots. To approach this realistic as possible the carbon source is changed to have the same ratio of carbohydrates and amino acids as found in the exudates of banana roots [26]. This new carbon composition causes a reduced growth rate because of less efficient degradation of these compounds.
In figure 5 the relative growth rate is shown for the resting and active state based on root exudate composition. The relative growth rate of BananaGuard around the roots has slightly decreased compared to glucose as carbon source but is still above 45%. This indicates  it still has the capacity to grow and maintain the constant fungal growth inhibitor production with one condition that BananaGuard needs to stay close to the roots to maintain its carbon uptake rate. From this we can conclude that these changes in the carbon composition do not change our previously made conclusions .
</P><FIGURE><FIGCAPTION>Figure 5. The relative growth rate compared to the wild type <I>P. putida</I> for different carbon uptake rates.  Two different carbon sources banana exudates and and glucose as reference. The expected carbon uptake rate of <I>P. putida</I> in the rhizosphere is indicated with transparent red.</FIGCAPTION></FIGURE><H3> Oxygen necessity in the soil </H3><P><I>P. putida</I> is a strict aerobic bacteria[13]. This means that  the bacteria cannot grow without oxygen available. Another problem could be that it might not be possible anymore to sustain the fungal growth inhibitor production or other anaerobic bacteria can outcompete BananGuard. Figure 6 was made to check the dependancy for oxygen. In figure 6 the growth rate in the active state is plotted as a function of the oxygen and carbon availability. This showed a sharp slope for the oxygen uptake rate and and the growth rate for BananaGuard so this could be a huge bottleneck for BananaGuard. However, after a literature research we concluded that the depletion of oxygen should not be a problem because there is a layer of oxygen present closely to the roots [17]. 
This is advantageous for BananaGuard, because this small oxygen layer  and root exudates  contains BananaGuard to the banana roots. This result is important for BananaGuard because it is designed to protect the banana roots from <I>F. oxysporum</I> invasion.
</P><FIGURE><FIGCAPTION>Figure 6. Dependacy of oxygen and carbon uptake rates for the relative growth rate of BananaGuard in the active state. Colors indicate the height of the growth rate. Note(1) that this is oxygen uptake rate and not oxygen availability, if the oxygen uptake rate is to high it needs to get rid of the oxygen this causes a decreased growth rate. Note(2) if the wild type growth rate is 0 h<SUP>-1</SUP>, the relative growth is 0 h<SUP>-1</SUP> as well.</FIGCAPTION></FIGURE><H3> Parameter analysis results </H3><P>
The parameters used to obtain the above-mentioned results are all based on available literature and published experimental data, except for the pyoverdin production rate. This parameter has been determined from growth experiments with <I>P. putida</I> (<A href="”https://2014.igem.org/Team:Wageningen_UR/notebook/journal&quot;" target="”blank”" class="soft_link">notebook</A>. So every parameter is biologically relevant, but because different studies report different values an estimation of a realistic production value has been calculated. This means that the parameter can fluctuate in a small range. To compensate this a parameter analysis has been performed.
This analysis shows the possible fluctuations of the modeled system and which parameter has the greatest impact on the system. These parameters have been checked over a range, dependent on the accuracy of the calculated parameter. The same analysis has been performed for all the fluctuations in the parameter set combined.
Figure 7 shows that Pyoverdine and the protein production rate influences the growth rate of BananaGuard the most. From this analysis we can conclude that even in the worst case scenario BananaGuard still performs its task around the banana roots. 
</P><FIGURE><FIGCAPTION>Figure 7. Different parameters and their influence on the relative growth rate compared with the WT <I>P. putida</I>. The tetragon indicates the lowest growth rate, the mean of all growth rates and the maximum growth rate of the resting state (red) and active state (green).
All: all parameters combined, Protein production: [0.1-0.4 μM min<SUP>-1</SUP>], Non-growth maintenance:[5-9mmol gDW<SUP>-1</SUP>hr<SUP>-1</SUP>], Cell density[0.035-0.047 gDW l<SUP>-1</SUP>], Pyoverdine[100-200 μmol gDW<SUP>-1</SUP>hr<SUP>-1</SUP>], DAPG[10-50 μmol gDW<SUP>-1</SUP>hr<SUP>-1</SUP>], DMDS[0.08-8 μmol gDW<SUP>-1</SUP>hr<SUP>-1</SUP>]
</FIGCAPTION></FIGURE></FIGURE><H1 id="performance">System Performance</H1><H2 id="introduction3">Introduction</H2><P>In order to gain insight in the performance of our biological control agent in the soil the optimized system was modelled over time. Focusing on the robustness of the Kill-switch in the presence of fluctuating protein concentrations and the activation of the Kll-switch upon a fusaric acid influx, the following system characteristics were taken into account:            
As a bi-stable system the kill-switch is subject to both extrinsic noise factors e.g. cell division and intrinsic noise e.g. fluctuations in repressor proteins. This affects the stability of the toggle-switch within the Kill-switch and thereby the system as a whole. Large fluctuations in the toggle-switch repressor protein concentrations will cause the Kill-switch to change state intermittently <B>(Noise)</B>.
The unwanted production of CIλ caused by leaky promoters is another factor contributing to kill-switch instability. A high basal production rate for CIλ will weigh the toggle-switch in favour of the fungal growth inhibitor producing LacI state <B>(Leakiness)</B>.
A state that is vital for our biological control agent to work but only when induced in the presence of <I>F. oxysporum</I> oxysporum <B>(Induction)</B>.
To that end noise, leakiness and an influx of fusaric acid are the focal points in a stochastic model of our genetic circuit. The model aims to quantify Kill-switch stability and characterize BananaGuards performance in the presence of <I>F. oxysporum</I>. The model takes into account cell division dividing the cellular contents the moment the cell has doubled in size. The average growth rate provided by the metabolic model and the impact of anti-gene transfer toxin antitoxin system on cell growth rate (figure 8) [34,35,37].
</P><P><FIGURE><FIGCAPTION><B>Figure 8.</B> A schematic overview of the components contained within the model: inductuction of the fusaric acid promoter and thereby the system, stability of the kill-switch and impact of the toxin anti-toxin system on cell growth and population dynamics. .</FIGCAPTION></FIGURE></P><H2 id="approach3">Theory and experimental design</H2><H3> Stochastic Modeling and Toxin-Antitoxin Systems </H3><P>
The performance of BananaGuard is defined by its ability to destroy <I>F. oxysporum</I> upon detection, its ability to compete with other micro-organisms and the stability of the genetic circuit.
The ability to destroy <I>F. oxysporum</I> has been tested in vitro. The biological control agents’ ability to compete with other soil born micro-organisms has been modeled in the metabolic model. The stability of the genetic switch will be explored in the following model, there are <B>three</B> key factors in assessing this aspect of the biological control agent: <B> Noise, Leakiness and Induction</B> (figure 9)
</P><FIGURE><FIGCAPTION><B>Figure 9.</B> A schematic representation of the reengineered <I>P. putida</I>.<B>(I)</B> Noise can destabilize bistable systems, causing the kill-switch to change state. <B>(II)</B> A leaky promoter can destabalize the toggleswitch by weighing it to the LacI state. <B>(III)</B> the induction of the genetic circuit by fusaric acid determines the performance of BananaGuard as a biological control agent.</FIGCAPTION></FIGURE><P><B>Noise (I)</B>, or variability of gene expression is of two kinds:
'intrinsic' noise derives from the natural variability in the way that molecules interact. This follows the observed difference in protein expression levels for identically regulated biological systems.
'Extrinsic' noise arises from random fluctuations in the environment or differences in the cellular sate. This comes in part from local pressure and temperature variations or the unequal distribution of cellular organelles after cell division1.
Our biological control agent will have to perform in these noisy environments. Recently, more and more experimental evidence in the form of bimodal population distributions indicates that noise plays a very important role in the switching of bistable systems. In bi-stable systems given a sizeable noise term, prolonged exposure to a single state increases the chance of a state change. These phenomena could be detrimental to the functioning of our biological control agent. If a large part of the BananaGuard population changes from the resting state (not induced by fusaric acid) to the fungal growth production state without an external input, the system would not function.
The following model therefore aims to quantify the stability of our biological control agent. If a large part of the population is robust to noisy environments we can confirm the functionality of the system [34,35].
</P><P><B>Leakiness (II)</B> is another factor to contributing to the instability of our bi-stable system. Most toggle switch dependent systems do not include a third interaction factor interfering with the stability of one state. The repressed CIλ on the input/output plasmid of the Kill-switch will not be completely repressed. Biologically a repressor does not stay bound, there is a constant turnover of the repressor protein on the promoters. If in the very short timeframe of promoter non-occupancy a RNA polymerase binds, CIλ will be transcribed. The binding of RNA polymerases occurs intermittently and leads to a basal production rate of CIλ. A high enough basal production rate weighs the toggle-switch in favour of the fungal inhibiting LacI state, in effect stopping the biological control agent from functioning. The model therefore aims to investigate the tolerance levels for the leakiness of CIλ and characterize the conditions in which it has the greatest impact [37].
</P><P><B>Induction (III)</B> ties into the performance aspect of our biological control. The biological control agent has to perform in the presence of <I>F. oxysporum</I>. It is imperative that the Kill-switch system is activated upon an influx of fusaric acid. In the soil the influx of fusaric acid will not be continuous but rather interlaced with periods low induction. Most likely leading to an increasing but haggard production of CIλ. The model therefore aims to characterize the optimal conditions for the successful activation of the fungal growth inhibitor producing state [34].
</P><H3> Toxin-Antitoxin Systems </H3><P>
Toxin-antitoxin systems in micro-organisms can act as a survival mechanism for micro-organisms in stressful environments. Operating as an hysteric switch, a small percentage of the population enters a toxin producing state. In this state protein production and cell growth are inhibited causing the cells to become persistor cells. This ‘metabolic inhibition’ makes these persistor cells robust to unfavorable conditions in the environment. Fluctuations in the toxins produced interfere with both the growth rate and protein translation of our biological control agent. This is therefore an important aspect of the stochastic model. Combining the Kill-switch with the toxin antitoxin system in a model over cell division will give insight in the effects of varying cell division times on the different destabilizing factors and the induction of the biological control agent by fusaric acid. It will also outline the population dynamics of our biological control agent. It is important to note that our chassis <I>P. putida</I> already has six native toxin-antitoxin systems. These have been incorporated into the model [38,39].
</P><H3> Model Equations </H3><P>
The model consists of eleven differential equations. The first six equations describe the dynamics of the different repressor proteins, in the form of a Hill function. All proteins have an equation describing RNA transcription (equations) and translation (<B>term I</B>), protein degradation (<B>term II</B>) and the stochastic noise term (d = 0.2) (<B>term III</B>). The term below the translation describes the impact of the free proteolytic proteins from the toxin anti-toxin system. Should the level of free toxin increase the value below the fraction will reduce protein translation efficacy [10,38,40,41].
</P><P><MATH xmlns:MML="http://www.w3.org/1998/Math/MathML" xmlns:M="http://schemas.openxmlformats.org/officeDocument/2006/math"><MFRAC><MROW><MI mathvariant="bold-italic">d</MI><MI mathvariant="bold-italic">L</MI><MI mathvariant="bold-italic">a</MI><MI mathvariant="bold-italic">c</MI><MSUB><MROW><MI mathvariant="bold-italic">I</MI></MROW><MROW><MI mathvariant="bold-italic">R</MI><MI mathvariant="bold-italic">N</MI><MI mathvariant="bold-italic">A</MI></MROW></MSUB></MROW><MROW><MI mathvariant="bold-italic">d</MI><MI mathvariant="bold-italic">t</MI></MROW></MFRAC><MO>=</MO><MFRAC><MROW><MI mathvariant="bold-italic">P</MI><MI mathvariant="bold-italic">r</MI><MI mathvariant="bold-italic">o</MI><MSUB><MROW><MI mathvariant="bold-italic">d</MI></MROW><MROW><MI mathvariant="bold-italic">L</MI><MI mathvariant="bold-italic">a</MI><MI mathvariant="bold-italic">c</MI><MI mathvariant="bold-italic">I</MI><MO>_</MO><MI mathvariant="bold-italic">R</MI><MI mathvariant="bold-italic">N</MI><MI mathvariant="bold-italic">A</MI></MROW></MSUB></MROW><MROW><MN>1</MN><MO>+</MO><MO>(</MO><MFRAC><MROW><MI mathvariant="bold-italic">T</MI><MI mathvariant="bold-italic">e</MI><MI mathvariant="bold-italic">t</MI><MI mathvariant="bold-italic">R</MI></MROW><MROW><MSUB><MROW><MI mathvariant="bold-italic">K</MI></MROW><MROW><MI mathvariant="bold-italic">T</MI><MI mathvariant="bold-italic">e</MI><MI mathvariant="bold-italic">t</MI><MI mathvariant="bold-italic">R</MI></MROW></MSUB></MROW></MFRAC><MSUP><MROW><MO>)</MO></MROW><MROW><MSUB><MROW><MI mathvariant="bold-italic">n</MI></MROW><MROW><MI mathvariant="bold-italic">T</MI><MI mathvariant="bold-italic">e</MI><MI mathvariant="bold-italic">t</MI><MI mathvariant="bold-italic">R</MI></MROW></MSUB></MROW></MSUP></MROW></MFRAC><MO>-</MO><MI mathvariant="bold-italic">d</MI><MI mathvariant="bold-italic">e</MI><MSUB><MROW><MI mathvariant="bold-italic">g</MI></MROW><MROW><MI mathvariant="bold-italic">L</MI><MI mathvariant="bold-italic">a</MI><MI mathvariant="bold-italic">c</MI><MI mathvariant="bold-italic">I</MI><MO>_</MO><MI mathvariant="bold-italic">R</MI><MI mathvariant="bold-italic">N</MI><MI mathvariant="bold-italic">A</MI></MROW></MSUB><MI mathvariant="bold-italic">L</MI><MI mathvariant="bold-italic">a</MI><MI mathvariant="bold-italic">c</MI><MSUB><MROW><MI mathvariant="bold-italic">I</MI></MROW><MROW><MI mathvariant="bold-italic">R</MI><MI mathvariant="bold-italic">N</MI><MI mathvariant="bold-italic">A</MI></MROW></MSUB></MATH></P><P><MATH xmlns:MML="http://www.w3.org/1998/Math/MathML" xmlns:M="http://schemas.openxmlformats.org/officeDocument/2006/math"><MFRAC><MROW><MI mathvariant="bold-italic">d</MI><MI mathvariant="bold-italic">L</MI><MI mathvariant="bold-italic">a</MI><MI mathvariant="bold-italic">c</MI><MI mathvariant="bold-italic">I</MI></MROW><MROW><MI mathvariant="bold-italic">d</MI><MI mathvariant="bold-italic">t</MI></MROW></MFRAC><MO>=</MO><MFRAC><MROW><MI mathvariant="bold-italic">T</MI><MI mathvariant="bold-italic">r</MI><MI mathvariant="bold-italic">a</MI><MI mathvariant="bold-italic">n</MI><MSUB><MROW><MI mathvariant="bold-italic">s</MI></MROW><MROW><MI mathvariant="bold-italic">L</MI><MI mathvariant="bold-italic">a</MI><MI mathvariant="bold-italic">c</MI><MI mathvariant="bold-italic">I</MI><MO>_</MO><MI mathvariant="bold-italic">R</MI><MI mathvariant="bold-italic">N</MI><MI mathvariant="bold-italic">A</MI></MROW></MSUB><MI mathvariant="bold-italic">L</MI><MI mathvariant="bold-italic">a</MI><MI mathvariant="bold-italic">c</MI><MSUB><MROW><MI mathvariant="bold-italic">I</MI></MROW><MROW><MI mathvariant="bold-italic">R</MI><MI mathvariant="bold-italic">N</MI><MI mathvariant="bold-italic">A</MI></MROW></MSUB></MROW><MROW><MN>1</MN><MO>+</MO><MFRAC><MROW><MI mathvariant="bold-italic">T</MI><MI mathvariant="bold-italic">o</MI><MI mathvariant="bold-italic">x</MI><MI mathvariant="bold-italic">i</MI><MSUP><MROW><MI mathvariant="bold-italic">n</MI></MROW><MROW><MI mathvariant="bold-italic">n</MI></MROW></MSUP></MROW><MROW><MI mathvariant="bold-italic">K</MI><MSUBSUP><MROW><MI mathvariant="bold-italic">T</MI></MROW><MROW><MN>2</MN></MROW><MROW><MI mathvariant="bold-italic">n</MI></MROW></MSUBSUP></MROW></MFRAC></MROW></MFRAC><MO>-</MO><MI mathvariant="bold-italic">d</MI><MI mathvariant="bold-italic">e</MI><MSUB><MROW><MI mathvariant="bold-italic">g</MI></MROW><MROW><MI mathvariant="bold-italic">L</MI><MI mathvariant="bold-italic">a</MI><MI mathvariant="bold-italic">c</MI><MI mathvariant="bold-italic">I</MI></MROW></MSUB><MI mathvariant="bold-italic">L</MI><MI mathvariant="bold-italic">a</MI><MI mathvariant="bold-italic">c</MI><MI mathvariant="bold-italic">I</MI><MO>+</MO><MSUB><MROW><MI mathvariant="bold-italic">ζ</MI></MROW><MROW><MI mathvariant="bold-italic">L</MI><MI mathvariant="bold-italic">a</MI><MI mathvariant="bold-italic">c</MI><MI mathvariant="bold-italic">I</MI></MROW></MSUB></MATH></P><P><MATH xmlns:MML="http://www.w3.org/1998/Math/MathML" xmlns:M="http://schemas.openxmlformats.org/officeDocument/2006/math"><MFRAC><MROW><MI mathvariant="bold-italic">d</MI><MSUB><MROW><MI mathvariant="bold-italic">T</MI><MI mathvariant="bold-italic">e</MI><MI mathvariant="bold-italic">t</MI><MI mathvariant="bold-italic">R</MI></MROW><MROW><MI mathvariant="bold-italic">R</MI><MI mathvariant="bold-italic">N</MI><MI mathvariant="bold-italic">A</MI></MROW></MSUB></MROW><MROW><MI mathvariant="bold-italic">d</MI><MI mathvariant="bold-italic">t</MI></MROW></MFRAC><MO>=</MO><MFRAC><MROW><MI mathvariant="bold-italic">P</MI><MI mathvariant="bold-italic">r</MI><MI mathvariant="bold-italic">o</MI><MSUB><MROW><MI mathvariant="bold-italic">d</MI></MROW><MROW><MI mathvariant="bold-italic">T</MI><MI mathvariant="bold-italic">e</MI><MI mathvariant="bold-italic">t</MI><MI mathvariant="bold-italic">R</MI></MROW></MSUB></MROW><MROW><MN>1</MN><MO>+</MO><MO>(</MO><MFRAC><MROW><MI mathvariant="bold-italic">L</MI><MI mathvariant="bold-italic">a</MI><MI mathvariant="bold-italic">c</MI><MI mathvariant="bold-italic">I</MI></MROW><MROW><MSUB><MROW><MI mathvariant="bold-italic">K</MI></MROW><MROW><MI mathvariant="bold-italic">L</MI><MI mathvariant="bold-italic">a</MI><MI mathvariant="bold-italic">c</MI><MI mathvariant="bold-italic">I</MI></MROW></MSUB></MROW></MFRAC><MSUP><MROW><MO>)</MO></MROW><MROW><MSUB><MROW><MI mathvariant="bold-italic">n</MI></MROW><MROW><MI mathvariant="bold-italic">T</MI><MI mathvariant="bold-italic">e</MI><MI mathvariant="bold-italic">t</MI><MI mathvariant="bold-italic">R</MI></MROW></MSUB></MROW></MSUP><MO>+</MO><MO>(</MO><MFRAC><MROW><MI mathvariant="bold-italic">λ</MI><MI mathvariant="bold-italic">C</MI><MI mathvariant="bold-italic">I</MI></MROW><MROW><MSUB><MROW><MI mathvariant="bold-italic">K</MI></MROW><MROW><MI mathvariant="bold-italic">C</MI><MI mathvariant="bold-italic">I</MI><MI mathvariant="bold-italic">λ</MI></MROW></MSUB></MROW></MFRAC><MSUP><MROW><MO>)</MO></MROW><MROW><MSUB><MROW><MI mathvariant="bold-italic">n</MI></MROW><MROW><MI mathvariant="bold-italic">C</MI><MI mathvariant="bold-italic">I</MI><MI mathvariant="bold-italic">λ</MI></MROW></MSUB></MROW></MSUP></MROW></MFRAC><MO>-</MO><MI mathvariant="bold-italic">d</MI><MI mathvariant="bold-italic">e</MI><MSUB><MROW><MI mathvariant="bold-italic">g</MI></MROW><MROW><MI mathvariant="bold-italic">T</MI><MI mathvariant="bold-italic">e</MI><MI mathvariant="bold-italic">t</MI><MI mathvariant="bold-italic">R</MI><MO>_</MO><MI mathvariant="bold-italic">R</MI><MI mathvariant="bold-italic">N</MI><MI mathvariant="bold-italic">A</MI></MROW></MSUB><MI mathvariant="bold-italic">T</MI><MI mathvariant="bold-italic">e</MI><MSUB><MROW><MI mathvariant="bold-italic">t</MI><MI mathvariant="bold-italic">R</MI></MROW><MROW><MI mathvariant="bold-italic">R</MI><MI mathvariant="bold-italic">N</MI><MI mathvariant="bold-italic">A</MI></MROW></MSUB></MATH></P><P><MATH xmlns:MML="http://www.w3.org/1998/Math/MathML" xmlns:M="http://schemas.openxmlformats.org/officeDocument/2006/math"><MFRAC><MROW><MI mathvariant="bold-italic">d</MI><MI mathvariant="bold-italic">T</MI><MI mathvariant="bold-italic">e</MI><MI mathvariant="bold-italic">t</MI><MI mathvariant="bold-italic">R</MI></MROW><MROW><MI mathvariant="bold-italic">d</MI><MI mathvariant="bold-italic">t</MI></MROW></MFRAC><MO>=</MO><MFRAC><MROW><MI mathvariant="bold-italic">T</MI><MI mathvariant="bold-italic">r</MI><MI mathvariant="bold-italic">a</MI><MI mathvariant="bold-italic">n</MI><MSUB><MROW><MI mathvariant="bold-italic">s</MI></MROW><MROW><MI mathvariant="bold-italic">T</MI><MI mathvariant="bold-italic">e</MI><MI mathvariant="bold-italic">t</MI><MI mathvariant="bold-italic">R</MI><MO>_</MO><MI mathvariant="bold-italic">R</MI><MI mathvariant="bold-italic">N</MI><MI mathvariant="bold-italic">A</MI></MROW></MSUB><MI mathvariant="bold-italic">T</MI><MI mathvariant="bold-italic">e</MI><MI mathvariant="bold-italic">t</MI><MSUB><MROW><MI mathvariant="bold-italic">R</MI></MROW><MROW><MI mathvariant="bold-italic">R</MI><MI mathvariant="bold-italic">N</MI><MI mathvariant="bold-italic">A</MI></MROW></MSUB></MROW><MROW><MN>1</MN><MO>+</MO><MFRAC><MROW><MI mathvariant="bold-italic">T</MI><MI mathvariant="bold-italic">o</MI><MI mathvariant="bold-italic">x</MI><MI mathvariant="bold-italic">i</MI><MSUP><MROW><MI mathvariant="bold-italic">n</MI></MROW><MROW><MI mathvariant="bold-italic">n</MI></MROW></MSUP></MROW><MROW><MI mathvariant="bold-italic">K</MI><MSUBSUP><MROW><MI mathvariant="bold-italic">T</MI></MROW><MROW><MN>2</MN></MROW><MROW><MI mathvariant="bold-italic">n</MI></MROW></MSUBSUP></MROW></MFRAC></MROW></MFRAC><MO>-</MO><MI mathvariant="bold-italic">d</MI><MI mathvariant="bold-italic">e</MI><MSUB><MROW><MI mathvariant="bold-italic">g</MI></MROW><MROW><MI mathvariant="bold-italic">T</MI><MI mathvariant="bold-italic">e</MI><MI mathvariant="bold-italic">t</MI><MI mathvariant="bold-italic">R</MI></MROW></MSUB><MI mathvariant="bold-italic">T</MI><MI mathvariant="bold-italic">e</MI><MI mathvariant="bold-italic">t</MI><MI mathvariant="bold-italic">R</MI><MO>+</MO><MSUB><MROW><MI mathvariant="bold-italic">ζ</MI></MROW><MROW><MI mathvariant="bold-italic">T</MI><MI mathvariant="bold-italic">e</MI><MI mathvariant="bold-italic">t</MI><MI mathvariant="bold-italic">R</MI></MROW></MSUB></MATH></P><P><MATH xmlns:MML="http://www.w3.org/1998/Math/MathML" xmlns:M="http://schemas.openxmlformats.org/officeDocument/2006/math"><MFRAC><MROW><MI mathvariant="bold-italic">d</MI><MI mathvariant="bold-italic">C</MI><MSUB><MROW><MI mathvariant="bold-italic">I</MI><MI mathvariant="bold-italic">λ</MI></MROW><MROW><MI mathvariant="bold-italic">R</MI><MI mathvariant="bold-italic">N</MI><MI mathvariant="bold-italic">A</MI></MROW></MSUB></MROW><MROW><MI mathvariant="bold-italic">d</MI><MI mathvariant="bold-italic">t</MI></MROW></MFRAC><MO>=</MO><MFRAC><MROW><MI mathvariant="bold-italic">F</MI><MI mathvariant="bold-italic">u</MI><MI mathvariant="bold-italic">s</MI><MI mathvariant="bold-italic">a</MI><MI mathvariant="bold-italic">r</MI><MI mathvariant="bold-italic">i</MI><MI mathvariant="bold-italic">c</MI><MI mathvariant="bold-italic">A</MI><MI mathvariant="bold-italic">c</MI><MI mathvariant="bold-italic">i</MI><MI mathvariant="bold-italic">d</MI><MI mathvariant="bold-italic">P</MI><MI mathvariant="bold-italic">r</MI><MI mathvariant="bold-italic">o</MI><MSUB><MROW><MI mathvariant="bold-italic">d</MI></MROW><MROW><MI mathvariant="bold-italic">C</MI><MI mathvariant="bold-italic">I</MI><MI mathvariant="bold-italic">λ</MI></MROW></MSUB></MROW><MROW><MI mathvariant="bold-italic">F</MI><MI mathvariant="bold-italic">u</MI><MI mathvariant="bold-italic">s</MI><MI mathvariant="bold-italic">a</MI><MI mathvariant="bold-italic">r</MI><MI mathvariant="bold-italic">i</MI><MI mathvariant="bold-italic">c</MI><MI mathvariant="bold-italic">A</MI><MI mathvariant="bold-italic">c</MI><MI mathvariant="bold-italic">i</MI><MI mathvariant="bold-italic">d</MI><MSUB><MROW><MI mathvariant="bold-italic">K</MI></MROW><MROW><MI mathvariant="bold-italic">F</MI><MI mathvariant="bold-italic">u</MI><MI mathvariant="bold-italic">s</MI><MI mathvariant="bold-italic">a</MI><MI mathvariant="bold-italic">r</MI><MI mathvariant="bold-italic">i</MI><MI mathvariant="bold-italic">c</MI><MI mathvariant="bold-italic">A</MI><MI mathvariant="bold-italic">c</MI><MI mathvariant="bold-italic">i</MI><MI mathvariant="bold-italic">d</MI></MROW></MSUB></MROW></MFRAC><MO>-</MO><MI mathvariant="bold-italic">d</MI><MI mathvariant="bold-italic">e</MI><MSUB><MROW><MI mathvariant="bold-italic">g</MI></MROW><MROW><MI mathvariant="bold-italic">C</MI><MI mathvariant="bold-italic">I</MI><MI mathvariant="bold-italic">λ</MI><MO>_</MO><MI mathvariant="bold-italic">R</MI><MI mathvariant="bold-italic">N</MI><MI mathvariant="bold-italic">A</MI></MROW></MSUB><MI mathvariant="bold-italic">C</MI><MI mathvariant="bold-italic">I</MI><MI mathvariant="bold-italic">λ</MI><MO>_</MO><MI mathvariant="bold-italic">R</MI><MI mathvariant="bold-italic">N</MI><MI mathvariant="bold-italic">A</MI></MATH></P><P><MATH xmlns:MML="http://www.w3.org/1998/Math/MathML" xmlns:M="http://schemas.openxmlformats.org/officeDocument/2006/math"><MFRAC><MROW><MI mathvariant="bold-italic">d</MI><MI mathvariant="bold-italic">C</MI><MI mathvariant="bold-italic">I</MI><MI mathvariant="bold-italic">λ</MI></MROW><MROW><MI mathvariant="bold-italic">d</MI><MI mathvariant="bold-italic">t</MI></MROW></MFRAC><MO>=</MO><MFRAC><MROW><MI mathvariant="bold-italic">T</MI><MI mathvariant="bold-italic">r</MI><MI mathvariant="bold-italic">a</MI><MI mathvariant="bold-italic">n</MI><MSUB><MROW><MI mathvariant="bold-italic">s</MI></MROW><MROW><MI mathvariant="bold-italic">C</MI><MI mathvariant="bold-italic">I</MI><MI mathvariant="bold-italic">λ</MI><MO>_</MO><MI mathvariant="bold-italic">R</MI><MI mathvariant="bold-italic">N</MI><MI mathvariant="bold-italic">A</MI></MROW></MSUB><MI mathvariant="bold-italic">C</MI><MSUB><MROW><MI mathvariant="bold-italic">I</MI><MI mathvariant="bold-italic">λ</MI></MROW><MROW><MI mathvariant="bold-italic">R</MI><MI mathvariant="bold-italic">N</MI><MI mathvariant="bold-italic">A</MI></MROW></MSUB></MROW><MROW><MN>1</MN><MO>+</MO><MFRAC><MROW><MI mathvariant="bold-italic">T</MI><MI mathvariant="bold-italic">o</MI><MI mathvariant="bold-italic">x</MI><MI mathvariant="bold-italic">i</MI><MSUP><MROW><MI mathvariant="bold-italic">n</MI></MROW><MROW><MI mathvariant="bold-italic">n</MI></MROW></MSUP></MROW><MROW><MI mathvariant="bold-italic">K</MI><MSUBSUP><MROW><MI mathvariant="bold-italic">T</MI></MROW><MROW><MN>2</MN></MROW><MROW><MI mathvariant="bold-italic">n</MI></MROW></MSUBSUP></MROW></MFRAC></MROW></MFRAC><MO>-</MO><MI mathvariant="bold-italic">d</MI><MI mathvariant="bold-italic">e</MI><MSUB><MROW><MI mathvariant="bold-italic">g</MI></MROW><MROW><MI mathvariant="bold-italic">C</MI><MI mathvariant="bold-italic">I</MI><MI mathvariant="bold-italic">λ</MI><MO>_</MO><MI mathvariant="bold-italic">R</MI><MI mathvariant="bold-italic">N</MI><MI mathvariant="bold-italic">A</MI></MROW></MSUB><MI mathvariant="bold-italic">C</MI><MI mathvariant="bold-italic">I</MI><MI mathvariant="bold-italic">λ</MI><MO>_</MO><MI mathvariant="bold-italic">R</MI><MI mathvariant="bold-italic">N</MI><MI mathvariant="bold-italic">A</MI></MATH></P><P>
The toxin anti-toxin systems can be described by two equations taken from the paper ‘Molecular mechanisms of multiple toxin–antitoxin systems are coordinated to govern the persister phenotype’ by Rick A. Fasani and Michael A. Savageau (2013). They proposed a mathematical description of a hysteric switch, linking it to fluctuating levels in free toxins and antitoxins and the growth rate. If the levels of the free toxin increases the growth rate of the cell decreases. Protein dilution caused by the growth rate was the main factor of protein degradation. This means that if a balanced is tipped the cell changes state and becomes a slow growing persistor cell.
The three equations that follow describe the production of toxin RNA as part of the Kill-switch and free toxins and antitoxins(<B>term I</B>), active degradation (<B>term II</B>), and stochastic fluctuations (<B>term III</B>). The equation describing free toxins has a RNA translation term added. The production terms for the free toxins and antitoxins have two components in the denominator. The first term describes the inhibition of protein synthesis caused by the free toxins. The second term describes auto repression of the toxin antitoxin genes by the toxin anti-toxin complexes that are formed [38].
</P><P><MATH xmlns:MML="http://www.w3.org/1998/Math/MathML" xmlns:M="http://schemas.openxmlformats.org/officeDocument/2006/math"><MFRAC><MROW><MI mathvariant="bold-italic">d</MI><MI mathvariant="bold-italic">T</MI><MI mathvariant="bold-italic">o</MI><MI mathvariant="bold-italic">x</MI><MI mathvariant="bold-italic">i</MI><MSUB><MROW><MI mathvariant="bold-italic">n</MI></MROW><MROW><MI mathvariant="bold-italic">R</MI><MI mathvariant="bold-italic">N</MI><MI mathvariant="bold-italic">A</MI></MROW></MSUB></MROW><MROW><MI mathvariant="bold-italic">d</MI><MI mathvariant="bold-italic">t</MI></MROW></MFRAC><MO>=</MO><MFRAC><MROW><MI mathvariant="bold-italic">P</MI><MI mathvariant="bold-italic">r</MI><MI mathvariant="bold-italic">o</MI><MSUB><MROW><MI mathvariant="bold-italic">d</MI></MROW><MROW><MI mathvariant="bold-italic">T</MI><MI mathvariant="bold-italic">o</MI><MI mathvariant="bold-italic">x</MI><MI mathvariant="bold-italic">i</MI><MI mathvariant="bold-italic">n</MI></MROW></MSUB></MROW><MROW><MN>1</MN><MO>+</MO><MO>(</MO><MFRAC><MROW><MI mathvariant="bold-italic">C</MI><MI mathvariant="bold-italic">I</MI><MI mathvariant="bold-italic">λ</MI></MROW><MROW><MSUB><MROW><MI mathvariant="bold-italic">K</MI></MROW><MROW><MI mathvariant="bold-italic">C</MI><MI mathvariant="bold-italic">I</MI><MI mathvariant="bold-italic">λ</MI></MROW></MSUB></MROW></MFRAC><MSUP><MROW><MO>)</MO></MROW><MROW><MSUB><MROW><MI mathvariant="bold-italic">n</MI></MROW><MROW><MI mathvariant="bold-italic">C</MI><MI mathvariant="bold-italic">I</MI><MI mathvariant="bold-italic">λ</MI></MROW></MSUB></MROW></MSUP><MO>+</MO><MO>(</MO><MFRAC><MROW><MI mathvariant="bold-italic">T</MI><MI mathvariant="bold-italic">e</MI><MI mathvariant="bold-italic">t</MI><MI mathvariant="bold-italic">R</MI></MROW><MROW><MSUB><MROW><MI mathvariant="bold-italic">K</MI></MROW><MROW><MI mathvariant="bold-italic">T</MI><MI mathvariant="bold-italic">e</MI><MI mathvariant="bold-italic">t</MI><MI mathvariant="bold-italic">R</MI></MROW></MSUB></MROW></MFRAC><MSUP><MROW><MO>)</MO></MROW><MROW><MSUB><MROW><MI mathvariant="bold-italic">n</MI></MROW><MROW><MI mathvariant="bold-italic">T</MI><MI mathvariant="bold-italic">e</MI><MI mathvariant="bold-italic">t</MI><MI mathvariant="bold-italic">R</MI></MROW></MSUB></MROW></MSUP></MROW></MFRAC><MO>-</MO><MI mathvariant="bold-italic">d</MI><MI mathvariant="bold-italic">e</MI><MSUB><MROW><MI mathvariant="bold-italic">g</MI></MROW><MROW><MI mathvariant="bold-italic">T</MI><MI mathvariant="bold-italic">o</MI><MSUB><MROW><MI mathvariant="bold-italic">x</MI></MROW><MROW><MI mathvariant="bold-italic">R</MI><MI mathvariant="bold-italic">N</MI><MI mathvariant="bold-italic">A</MI></MROW></MSUB></MROW></MSUB><MI mathvariant="bold-italic">T</MI><MI mathvariant="bold-italic">o</MI><MI mathvariant="bold-italic">x</MI><MI mathvariant="bold-italic">i</MI><MSUB><MROW><MI mathvariant="bold-italic">n</MI></MROW><MROW><MI mathvariant="bold-italic">R</MI><MI mathvariant="bold-italic">N</MI><MI mathvariant="bold-italic">A</MI></MROW></MSUB></MATH></P><P><MATH xmlns:MML="http://www.w3.org/1998/Math/MathML" xmlns:M="http://schemas.openxmlformats.org/officeDocument/2006/math"><MFRAC><MROW><MI mathvariant="bold-italic">d</MI><MI mathvariant="bold-italic">T</MI><MI mathvariant="bold-italic">o</MI><MI mathvariant="bold-italic">x</MI><MI mathvariant="bold-italic">i</MI><MI mathvariant="bold-italic">n</MI></MROW><MROW><MI mathvariant="bold-italic">d</MI><MI mathvariant="bold-italic">t</MI></MROW></MFRAC><MO>=</MO><MFRAC><MROW><MI mathvariant="bold-italic">α</MI></MROW><MROW><MO>(</MO><MN>1</MN><MO>+</MO><MFRAC><MROW><MI mathvariant="bold-italic">T</MI><MI mathvariant="bold-italic">o</MI><MI mathvariant="bold-italic">x</MI><MI mathvariant="bold-italic">i</MI><MSUP><MROW><MI mathvariant="bold-italic">n</MI></MROW><MROW><MI mathvariant="bold-italic">n</MI></MROW></MSUP></MROW><MROW><MI mathvariant="bold-italic">K</MI><MSUBSUP><MROW><MI mathvariant="bold-italic">T</MI></MROW><MROW><MN>2</MN></MROW><MROW><MI mathvariant="bold-italic">n</MI></MROW></MSUBSUP></MROW></MFRAC><MO>)</MO><MO>(</MO><MN>1</MN><MO>+</MO><MFRAC><MROW><MI mathvariant="bold-italic">A</MI><MI mathvariant="bold-italic">n</MI><MI mathvariant="bold-italic">t</MI><MI mathvariant="bold-italic">i</MI><MI mathvariant="bold-italic">T</MI><MI mathvariant="bold-italic">o</MI><MI mathvariant="bold-italic">x</MI><MI mathvariant="bold-italic">i</MI><MSUP><MROW><MI mathvariant="bold-italic">n</MI></MROW><MROW><MI mathvariant="bold-italic">s</MI></MROW></MSUP></MROW><MROW><MI mathvariant="bold-italic">K</MI><MSUBSUP><MROW><MI mathvariant="bold-italic">p</MI></MROW><MROW><MN>1</MN></MROW><MROW><MI mathvariant="bold-italic">s</MI></MROW></MSUBSUP></MROW></MFRAC><MO>+</MO><MO>(</MO><MN>2</MN><MFRAC><MROW><MI mathvariant="bold-italic">A</MI><MI mathvariant="bold-italic">n</MI><MI mathvariant="bold-italic">t</MI><MI mathvariant="bold-italic">i</MI><MI mathvariant="bold-italic">t</MI><MI mathvariant="bold-italic">o</MI><MI mathvariant="bold-italic">x</MI><MI mathvariant="bold-italic">i</MI><MSUP><MROW><MI mathvariant="bold-italic">n</MI></MROW><MROW><MI mathvariant="bold-italic">s</MI></MROW></MSUP><MI mathvariant="bold-italic">T</MI><MI mathvariant="bold-italic">o</MI><MI mathvariant="bold-italic">x</MI></MROW><MROW><MI mathvariant="bold-italic">K</MI><MSUBSUP><MROW><MI mathvariant="bold-italic">p</MI></MROW><MROW><MN>2</MN></MROW><MROW><MI mathvariant="bold-italic">s</MI></MROW></MSUBSUP></MROW></MFRAC><MSUP><MROW><MO>)</MO></MROW><MROW><MI mathvariant="bold-italic">P</MI></MROW></MSUP><MO>+</MO><MFENCED separators="|"><MROW><MFRAC><MROW><MI mathvariant="bold-italic">A</MI><MI mathvariant="bold-italic">n</MI><MI mathvariant="bold-italic">t</MI><MI mathvariant="bold-italic">i</MI><MI mathvariant="bold-italic">t</MI><MI mathvariant="bold-italic">o</MI><MI mathvariant="bold-italic">x</MI><MI mathvariant="bold-italic">i</MI><MSUP><MROW><MI mathvariant="bold-italic">n</MI></MROW><MROW><MI mathvariant="bold-italic">s</MI></MROW></MSUP><MI mathvariant="bold-italic">T</MI><MI mathvariant="bold-italic">o</MI><MSUP><MROW><MI mathvariant="bold-italic">x</MI></MROW><MROW><MI mathvariant="bold-italic">s</MI></MROW></MSUP></MROW><MROW><MI mathvariant="bold-italic">K</MI><MSUBSUP><MROW><MI mathvariant="bold-italic">p</MI></MROW><MROW><MN>1</MN></MROW><MROW><MI mathvariant="bold-italic">s</MI></MROW></MSUBSUP><MI mathvariant="bold-italic">K</MI><MSUP><MROW><MI mathvariant="bold-italic">H</MI></MROW><MROW><MI mathvariant="bold-italic">s</MI></MROW></MSUP></MROW></MFRAC></MROW></MFENCED><MO>)</MO></MROW></MFRAC><MO>+</MO><MFRAC><MROW><MI mathvariant="bold-italic">T</MI><MI mathvariant="bold-italic">r</MI><MI mathvariant="bold-italic">a</MI><MI mathvariant="bold-italic">n</MI><MSUB><MROW><MI mathvariant="bold-italic">s</MI></MROW><MROW><MI mathvariant="bold-italic">T</MI><MI mathvariant="bold-italic">o</MI><MI mathvariant="bold-italic">x</MI><MI mathvariant="bold-italic">i</MI><MI mathvariant="bold-italic">n</MI><MO>_</MO><MI mathvariant="bold-italic">R</MI><MI mathvariant="bold-italic">N</MI><MI mathvariant="bold-italic">A</MI></MROW></MSUB><MI mathvariant="bold-italic">T</MI><MI mathvariant="bold-italic">o</MI><MI mathvariant="bold-italic">x</MI><MI mathvariant="bold-italic">i</MI><MI mathvariant="bold-italic">n</MI></MROW><MROW><MN>1</MN><MO>+</MO><MFRAC><MROW><MI mathvariant="bold-italic">T</MI><MI mathvariant="bold-italic">o</MI><MI mathvariant="bold-italic">x</MI><MI mathvariant="bold-italic">i</MI><MSUP><MROW><MI mathvariant="bold-italic">n</MI></MROW><MROW><MI mathvariant="bold-italic">n</MI></MROW></MSUP></MROW><MROW><MI mathvariant="bold-italic">K</MI><MSUBSUP><MROW><MI mathvariant="bold-italic">T</MI></MROW><MROW><MN>2</MN></MROW><MROW><MI mathvariant="bold-italic">n</MI></MROW></MSUBSUP></MROW></MFRAC></MROW></MFRAC><MO>-</MO><MI mathvariant="bold-italic">d</MI><MI mathvariant="bold-italic">e</MI><MSUB><MROW><MI mathvariant="bold-italic">g</MI></MROW><MROW><MI mathvariant="bold-italic">T</MI><MI mathvariant="bold-italic">o</MI><MI mathvariant="bold-italic">x</MI><MI mathvariant="bold-italic">i</MI><MI mathvariant="bold-italic">n</MI></MROW></MSUB><MO>+</MO><MSUB><MROW><MI mathvariant="bold-italic">ζ</MI></MROW><MROW><MI mathvariant="bold-italic">T</MI><MI mathvariant="bold-italic">o</MI><MI mathvariant="bold-italic">x</MI><MI mathvariant="bold-italic">i</MI><MI mathvariant="bold-italic">n</MI></MROW></MSUB></MATH></P><P><MATH xmlns:MML="http://www.w3.org/1998/Math/MathML" xmlns:M="http://schemas.openxmlformats.org/officeDocument/2006/math"><MFRAC><MROW><MI mathvariant="bold-italic">d</MI><MI mathvariant="bold-italic">A</MI><MI mathvariant="bold-italic">n</MI><MI mathvariant="bold-italic">t</MI><MI mathvariant="bold-italic">i</MI><MI mathvariant="bold-italic">T</MI><MI mathvariant="bold-italic">o</MI><MI mathvariant="bold-italic">x</MI><MI mathvariant="bold-italic">i</MI><MI mathvariant="bold-italic">n</MI></MROW><MROW><MI mathvariant="bold-italic">d</MI><MI mathvariant="bold-italic">t</MI></MROW></MFRAC><MO>=</MO><MFRAC><MROW><MI mathvariant="bold-italic">α</MI><MI mathvariant="bold-italic">σ</MI></MROW><MROW><MO>(</MO><MN>1</MN><MO>+</MO><MFRAC><MROW><MI mathvariant="bold-italic">T</MI><MI mathvariant="bold-italic">o</MI><MI mathvariant="bold-italic">x</MI><MI mathvariant="bold-italic">i</MI><MSUP><MROW><MI mathvariant="bold-italic">n</MI></MROW><MROW><MI mathvariant="bold-italic">n</MI></MROW></MSUP></MROW><MROW><MI mathvariant="bold-italic">K</MI><MSUBSUP><MROW><MI mathvariant="bold-italic">T</MI></MROW><MROW><MN>2</MN></MROW><MROW><MI mathvariant="bold-italic">n</MI></MROW></MSUBSUP></MROW></MFRAC><MO>)</MO><MO>(</MO><MN>1</MN><MO>+</MO><MFRAC><MROW><MI mathvariant="bold-italic">A</MI><MI mathvariant="bold-italic">n</MI><MI mathvariant="bold-italic">t</MI><MI mathvariant="bold-italic">i</MI><MI mathvariant="bold-italic">T</MI><MI mathvariant="bold-italic">o</MI><MI mathvariant="bold-italic">x</MI><MI mathvariant="bold-italic">i</MI><MSUP><MROW><MI mathvariant="bold-italic">n</MI></MROW><MROW><MI mathvariant="bold-italic">s</MI></MROW></MSUP></MROW><MROW><MI mathvariant="bold-italic">K</MI><MSUBSUP><MROW><MI mathvariant="bold-italic">p</MI></MROW><MROW><MN>1</MN></MROW><MROW><MI mathvariant="bold-italic">s</MI></MROW></MSUBSUP></MROW></MFRAC><MO>+</MO><MO>(</MO><MN>2</MN><MFRAC><MROW><MI mathvariant="bold-italic">A</MI><MI mathvariant="bold-italic">n</MI><MI mathvariant="bold-italic">t</MI><MI mathvariant="bold-italic">i</MI><MI mathvariant="bold-italic">t</MI><MI mathvariant="bold-italic">o</MI><MI mathvariant="bold-italic">x</MI><MI mathvariant="bold-italic">i</MI><MSUP><MROW><MI mathvariant="bold-italic">n</MI></MROW><MROW><MI mathvariant="bold-italic">s</MI></MROW></MSUP><MI mathvariant="bold-italic">T</MI><MI mathvariant="bold-italic">o</MI><MI mathvariant="bold-italic">x</MI></MROW><MROW><MI mathvariant="bold-italic">K</MI><MSUBSUP><MROW><MI mathvariant="bold-italic">p</MI></MROW><MROW><MN>2</MN></MROW><MROW><MI mathvariant="bold-italic">s</MI></MROW></MSUBSUP></MROW></MFRAC><MSUP><MROW><MO>)</MO></MROW><MROW><MI mathvariant="bold-italic">P</MI></MROW></MSUP><MO>+</MO><MFENCED separators="|"><MROW><MFRAC><MROW><MI mathvariant="bold-italic">A</MI><MI mathvariant="bold-italic">n</MI><MI mathvariant="bold-italic">t</MI><MI mathvariant="bold-italic">i</MI><MI mathvariant="bold-italic">t</MI><MI mathvariant="bold-italic">o</MI><MI mathvariant="bold-italic">x</MI><MI mathvariant="bold-italic">i</MI><MSUP><MROW><MI mathvariant="bold-italic">n</MI></MROW><MROW><MI mathvariant="bold-italic">s</MI></MROW></MSUP><MI mathvariant="bold-italic">T</MI><MI mathvariant="bold-italic">o</MI><MSUP><MROW><MI mathvariant="bold-italic">x</MI></MROW><MROW><MI mathvariant="bold-italic">s</MI></MROW></MSUP></MROW><MROW><MI mathvariant="bold-italic">K</MI><MSUBSUP><MROW><MI mathvariant="bold-italic">p</MI></MROW><MROW><MN>1</MN></MROW><MROW><MI mathvariant="bold-italic">s</MI></MROW></MSUBSUP><MI mathvariant="bold-italic">K</MI><MSUP><MROW><MI mathvariant="bold-italic">H</MI></MROW><MROW><MI mathvariant="bold-italic">s</MI></MROW></MSUP></MROW></MFRAC></MROW></MFENCED><MO>)</MO></MROW></MFRAC><MO>-</MO><MI mathvariant="bold-italic">d</MI><MI mathvariant="bold-italic">e</MI><MSUB><MROW><MI mathvariant="bold-italic">g</MI></MROW><MROW><MI mathvariant="bold-italic">A</MI><MI mathvariant="bold-italic">n</MI><MI mathvariant="bold-italic">t</MI><MI mathvariant="bold-italic">i</MI><MI mathvariant="bold-italic">T</MI><MI mathvariant="bold-italic">o</MI><MI mathvariant="bold-italic">x</MI><MI mathvariant="bold-italic">i</MI><MI mathvariant="bold-italic">n</MI></MROW></MSUB><MI mathvariant="bold-italic">A</MI><MI mathvariant="bold-italic">n</MI><MI mathvariant="bold-italic">t</MI><MI mathvariant="bold-italic">i</MI><MI mathvariant="bold-italic">T</MI><MI mathvariant="bold-italic">o</MI><MI mathvariant="bold-italic">x</MI><MI mathvariant="bold-italic">i</MI><MI mathvariant="bold-italic">n</MI><MO>+</MO><MSUB><MROW><MI mathvariant="bold-italic">ζ</MI></MROW><MROW><MI mathvariant="bold-italic">A</MI><MI mathvariant="bold-italic">n</MI><MI mathvariant="bold-italic">t</MI><MI mathvariant="bold-italic">i</MI><MI mathvariant="bold-italic">T</MI><MI mathvariant="bold-italic">o</MI><MI mathvariant="bold-italic">x</MI><MI mathvariant="bold-italic">i</MI><MI mathvariant="bold-italic">n</MI></MROW></MSUB></MATH></P><P>
dV/dt represents cell growth. Cells grow exponentially at a rate of µ_max, the stochastic fluctuation of free toxin levels inhibit maximum growth. The model states that given an initial volume of 1, cell volume will increase according to µV where µ is µ_max divided by the free toxin inhibition term [38].
The moment the cell has doubled in size (volume = 2), the cell contents are divided over the daughter cells following a binomial distribution. The simulation is restarted following one of these daughter cells. Should the kill-switch change state the initial conditions are reset to those at the start of the simulation [42].
</P><P><MATH xmlns:MML="http://www.w3.org/1998/Math/MathML" xmlns:M="http://schemas.openxmlformats.org/officeDocument/2006/math" display="block"><MFRAC><MROW><MI mathvariant="bold-italic">d</MI><MI mathvariant="bold-italic">V</MI></MROW><MROW><MI mathvariant="bold-italic">d</MI><MI mathvariant="bold-italic">t</MI></MROW></MFRAC><MO>=</MO><MFRAC><MROW><MSUB><MROW><MI mathvariant="bold-italic">µ</MI></MROW><MROW><MI mathvariant="bold-italic">m</MI><MI mathvariant="bold-italic">a</MI><MI mathvariant="bold-italic">x</MI></MROW></MSUB><MI mathvariant="bold-italic">V</MI></MROW><MROW><MN>1</MN><MO>+</MO><MFRAC><MROW><MI mathvariant="bold-italic">T</MI><MI mathvariant="bold-italic">o</MI><MI mathvariant="bold-italic">x</MI><MI mathvariant="bold-italic">i</MI><MSUP><MROW><MI mathvariant="bold-italic">n</MI></MROW><MROW><MI mathvariant="bold-italic">n</MI></MROW></MSUP></MROW><MROW><MI mathvariant="bold-italic">K</MI><MSUBSUP><MROW><MI mathvariant="bold-italic">T</MI></MROW><MROW><MN>1</MN></MROW><MROW><MI mathvariant="bold-italic">n</MI></MROW></MSUBSUP></MROW></MFRAC></MROW></MFRAC></MATH></P><P>
The last equation describes the concentration of fusaric acid in the cell. Not much is known about fusaric acid dynamics besides measured quantities in the soil after infection. The fusaric acid detector experiments have given an indication with respect to the kinetics of the fusaric acid promoter. The required number of fusaric acid molecules for promoter activation is quite low, especially compared to the levels of fusaric acid that have been measured in the soil after infection. The influx is stochastic and is weighed such that the concentration increases over time. This randomized increasing influx represents the varying influx of fusaric acid into our BananaGuard population.
</P><P><MATH xmlns:MML="http://www.w3.org/1998/Math/MathML" xmlns:M="http://schemas.openxmlformats.org/officeDocument/2006/math" display="block"><MFRAC><MROW><MI mathvariant="bold-italic">d</MI><MI mathvariant="bold-italic">F</MI><MI mathvariant="bold-italic">u</MI><MI mathvariant="bold-italic">s</MI><MI mathvariant="bold-italic">a</MI><MI mathvariant="bold-italic">r</MI><MI mathvariant="bold-italic">i</MI><MI mathvariant="bold-italic">c</MI><MI mathvariant="bold-italic">A</MI><MI mathvariant="bold-italic">c</MI><MI mathvariant="bold-italic">i</MI><MI mathvariant="bold-italic">d</MI></MROW><MROW><MI mathvariant="bold-italic">d</MI><MI mathvariant="bold-italic">t</MI></MROW></MFRAC><MO>=</MO><MSUB><MROW><MI mathvariant="bold-italic">ζ</MI></MROW><MROW><MI mathvariant="bold-italic">I</MI><MI mathvariant="bold-italic">n</MI><MI mathvariant="bold-italic">f</MI><MI mathvariant="bold-italic">l</MI><MI mathvariant="bold-italic">u</MI><MI mathvariant="bold-italic">x</MI></MROW></MSUB><MO>-</MO><MI mathvariant="bold-italic">d</MI><MI mathvariant="bold-italic">e</MI><MSUB><MROW><MI mathvariant="bold-italic">g</MI></MROW><MROW><MI mathvariant="bold-italic">F</MI><MI mathvariant="bold-italic">u</MI><MI mathvariant="bold-italic">s</MI><MI mathvariant="bold-italic">a</MI><MI mathvariant="bold-italic">r</MI><MI mathvariant="bold-italic">i</MI><MI mathvariant="bold-italic">c</MI><MI mathvariant="bold-italic">A</MI><MI mathvariant="bold-italic">c</MI><MI mathvariant="bold-italic">i</MI><MI mathvariant="bold-italic">d</MI></MROW></MSUB><MI mathvariant="bold-italic">F</MI><MI mathvariant="bold-italic">u</MI><MI mathvariant="bold-italic">s</MI><MI mathvariant="bold-italic">a</MI><MI mathvariant="bold-italic">r</MI><MI mathvariant="bold-italic">i</MI><MI mathvariant="bold-italic">c</MI><MI mathvariant="bold-italic">A</MI><MI mathvariant="bold-italic">c</MI><MI mathvariant="bold-italic">i</MI><MI mathvariant="bold-italic">d</MI></MATH></P><H3> Model Sensitivity </H3><P>
Below a table for the parameter tolerances for the maintenance bi-stability of the kill-switch and functionality in the toxin antitoxin system. A local sensitivity analysis was done in order to find the ranges of the bi-stable region for this parameter set by running simulation in a set state of the Kill-Switch. If the parameters cause instability, the state at the end if the simulation changed. The values for the toxin-antitoxin tolerances were published by Rick A. Fasani and Michael A. Savageau (2013). The indicate the parameter ranges where hysteric switching occurs
</P><FIGURE><FIGCAPTION>Table 2. The parameter tolerances for the maintenance of a bi-stable Kill-switch and hysteric switching in the toxin-antitoxin system .</FIGCAPTION></FIGURE><H3>Software used</H3><H3><P>
All models were made in Python (Python software foundation, Delaware, US) version 2.71 for Mac OS X version 10.5.8. Solving of ODE systems was done using the standard soln function in Python 
</P></H3><H2 id="results3">Results</H2><P>
The results follow the trend of Noise, Leakiness and input. The first set of figures describe the Kill-switch system over cell division without the presence of the toxin anti-toxin system, this means the cell division times are equal for all simulations. The results display the intrinsic behavior of Kill-switch stability and performance.
The second set of figures follows the model with the toxin-antitoxin system included. The varying growth and protein production rates caused by fluctuations in the free toxin concentration do impact the functionality but the overall system performs in line with the results obtained (Figure 10) [38].

</P><FIGURE><FIGCAPTION><B>Figure 10. Plots showing the effect of noise and leakiness on the stability of the Kill-switch and the performance of the Killswitch upon induction by fusric acid. Every plot point represent a percentage of unstable (leakyness and noise) or functioning (induction) kill-switches over 10000 simulations.
							<B>(A)</B> The plot shows the impact of a normally distributed noise term over a range of 120 to 240 nM/min for the toggle switch proteins LacI and TetR. 5% of the Kill-switches becomes unstable given a noise term of 200 nM/min and a 45 minute division time. Should the division time be doubled to 90 minutes less than half a percent becomes unstable.
							<B>(B)</B> The plot shows the impact of different CIλ protein basal production rates on
							toggle-switch stability. The noise term for the toggle-switch proteins LacI and TetR is 200nM. This corresponds to 5% and 0.5% instability due to noise (figure. 1A). The plots show a linear increase in instability for increasing CIλ basal production rates given a cell division time of 45 minutes. Proportionally, the increase in instability is a lot higher given a doubling in cell division time umax to 90 minutes. This can be atributed to a lower dilution rate allowing for a build-up of CIλ increasing instability.
							<B>(C)</B> The plot shows the performance levels of the kill-switch upon induction by fusaric acid. Simulated with four different fusaric acid inputs and four different growth rates. Testing solely the performance no leakiness was taken into account, a noise term of 200nM was added for the toggle-switch proteins LacI and TetR. The figure show the influx of fusaric acid is not limiting to the performance of the Kill-switch. The repeated doubling of the maximum growth rate doubles performance. Indicating that the build-up and therefore production rate of CIλ is limiting to the functioning of the Kill-switch.</B></FIGCAPTION></FIGURE><P><B>Noise, A:</B> The significant increase in stability given a longer cell division time coincides with the fact that rapid protein dilution adds to the instability of the Kill-switches. Biologically the ratio between the protein levels at steady state has decreased, the chance that the noise term will have an impact therefore increases, resulting in a higher number of Kill-switches that have changed state. <B>Leakiness, B:</B> The significant increase in instability given a longer cell division and a leaky promoter can be explained in a reversed but similar manner to figure 1A. Lower dilution rates due to increased division times increases the amount of CIλ in the cell. Biologically, a more effective repression of the TetR promoter occurs due higher levels of CIλ. This lowers the steady state levels of the TetR protein decreasing the ratio of the protein levels at steady state, increasing the chance that the noise term will have an impact.<B>Induction, C:</B> The plots correspond production rate and the activation of the CIλ repressor protein to be limiting to the performance of the kill-switch. Similar to the effect of the basal production of CIλ, the increased division times and thereby lowered dilution rates allow CIλ to accumulate. The accumulation gradually lowers the steady state ratios of the toggle switch proteins until the system changes state. The results indicate that for a parameter set aiming to describe low copy plasmid numbers, the performance of our biological control agent genetic circuitry is tied to its growth rate.
</P><P>
Overall the results for the Kill-switch modeled over cell division have shown that a fast growth rate affects the ‘basal’ stability of the genetic circuit. High dilution rates increase the chance of instability due to noise. Leakiness and induction in and of the system are more likely to be effective at a slow growth rates. In both cases the build-up of CIλ over time is key in accessing the systems instability and performance.
</P><H3> The stability and performance of the Biological Control Agent over Cell Division </H3><P>
The following results will outline the following:
</P><P><B>(I)</B> The impact of the toxin-antitoxin anti-gene gene transfer system on cell growth.
</P><P><B>(II)</B> The impact of the toxin antitoxin on promoter leakiness and induction.
</P><P>
For both Leakiness and induction, the maximum and the predicted growth rate was simulated. The prior in order to ascertain the general behavior of the system, the latter in order to get a more accurate picture of our biological control agents performance in the soil (figure 11).
The effect of just noise on the stability of the biological control agent has been neglected because its effect was statistically insignificant.
</P><FIGURE><FIGCAPTION><B>Figure 11. The figures show the stability given a leaky promoter . Taking the form of a histogram, the y-axis shows the number of cell divisions that have occurred at a specific time point. The X-axis shows the number of minutes it takes for the cell to divide. The light green bars indicate a functioning system and a specific cell division time, coloured bars indicate non-functioning systems corresponding to a specific basal production rate.
							<B>(A)</B> For a maximum growth rate corresponding to a division time of 45 minutes, a basal CIλ production of 50 nM/min or higher destabilizes the kill-switch in slow growing cells. The frequency of division times within the population of the cells is spread out. A spread over longer divsion times can be observed after an initial peak at 60 minutes. Protein dilution due to cell division is high, the population dynamics is not impacted. A total of 20000 simulation were run. 
							(B) For a maximum growth rate corresponding to a division time of 180 minutes a basal CIλ production of 50 nM/min or higher destabilizes the Kill-switch. The population dynamics are affected. Low protein dilution due to slow growth causes the Kill-switch to leak toxin. Higher basal production rates compensate, increasing the average growth rate but also the instability. A total of 5000 simulation were run.
 </B></FIGCAPTION></FIGURE><P><B>Leakiness, A:</B> The population dynamics are effected by free toxins from the toxin-antitoxin system. For a lower maximum growth rate and therefore a lower protein dilution rate the fluctuations in the level of free toxins is attributed to more than just stochasticity. Because a toxin is included on the kill-switch its promoter is subject to leakiness with a low amount of repressor protein present. With lower dilution rates, low amounts of CIλ are able to build up slower over time. The CIλ protein represses both the toxin and the TetR promoter on the toggle switch. This results in lower levels of TetR for which the CIλ cannot compensate over the course of its build-up. A deficiency that translates in a leaky toxin promoter causing a portion of the cells to distribute over longer division times.
The effect of leakiness on the stability of the genetic circuit corresponds to the results obtained in figure 1.B. Over the total number simulations only 13% of the kill-switches became unstable given a basal production rate larger than 50nM and a maximum growth rate of 45 minutes. As expected all instances of instability occurred in the slow growing portion of the cell population.
</P><P><B>Leakiness, B:</B> Because the population of cells averaged a significantly higher growth rate the percentage of unstable kill-switches increased proportionally too, once again confirming increased instability for lengthy cell divisions cycles. With respect population dynamics, the same logic that is applied to figure 1B applies to figure 1A. the effect however of a leaky CIλ can clearly be seen. A combination of an included toxin on the kill-switch, a promoter subject to leakiness and Lower dilution rates increase the levels of free toxin. Low basal production levels for CIλ increases TetR repression, alleviating toxin repression for which the CIλ cannot compensate if its concentration is to low. Higher levels of basal CIλ production reduce the levels of free toxins speeding up cell growth but decreasing stability. In order to avoid this problem the basal production rate of CIλ needs to be measured and adjusted if necessary, values lower than 50nM for the basal production appear to have no significant effect on the stability of the genetic circuit. For a low basal production of CIλ entails a low basal production of toxin.
 </P><FIGURE><FIGCAPTION><B>Figure 12.</B><B>A</B>The figures show the performance of our biological control agent upon induction. All plots take the form of histograms, the y-axis shows the number of cell divisions that have occurred at a specific time point over 20000 simulations. The X-axis shows the number of minutes it takes for the cell to divide. The green bars indicate a functioning system and a specific cell division time, red bars indicate non-functioning systems given a specific cell division time. Two maximum growth rates uMax were investigated 45 minutes and 180 minutes.
							(A) For a maximum growth rate of 45 minutes 36% of the kill-switches activate, longer division times activate the cells more effectively. (B) For a maximum growth rate of 180 minutes 98% of the kill-switches activate, longer division times activate the cells more effectively. </FIGCAPTION></FIGURE><P><B>Induction, A and B:</B> The population dynamics for an induced system varies slightly. The same logic applies with respect to leaky toxins but the time-spans are different. Because the build-up of CIλ occurs faster in an induced system compared to a system with a basal transcription, the time frame in which the toxin promoter leaks is smaller. This results in faster average growth rate compared to a situation with leaky promoters.
Induction in and of itself works in accordance with the results obtained in figure 1. Slower growth rates activate the system more efficiently if a maximum growth rate 36% of the kill-switches is assumed, given the growth rate obtained from the metabolic model an efficiency of 98% is achieved.</P></SECTION><SECTION id="conclusion"><H1>Conclusion</H1><P>
The performance of BananaGuard as a biological control agent is defined by its ability to destroy <I>F. oxysporum</I> upon detection, its ability to compete with other micro-organisms and the stability of the genetic circuit.
The ability to destroy <I>F. oxysporum</I> has been tested in vitro. The modeling has therefore been used to predict the answer to the remaining two questions. 
 Can the BananaGuard system survive in the presence of competing micro-organisms?

Is the genetic circuit robust to biological fluctuations?
 
We have demonstrated that different modeling approaches can be taken and integrated with both the lab and one another. The statistical mechanics model, modeling <B>promoter design</B> has shown that a prediction regarding the functionality of the kill-switch can be made with respect to the different constituents on the promoter. The predicted configuration of repressor binding sites has been used in designing a second, optimized version of the kill-switch.
The model estimating the <B>cost</B> of the introduced genetic circuit has shown that in its resting state, BananaGuard is capable of competing with other micro-organisms in the soil on a metabolic level. The average growth rate obtained in this model has been used in the final model describing the <B>stability and performance</B> of BananaGuard. The results obtained have characterized several aspects of the kill-switch and the toxin anti-toxin system. First, noise can impact the stability of the kill-switch to a certain degree. Secondly, different basal production levels of CIλ can have different effects on population dynamics, cell growth and the stability of the kill-switch, a point of attention for final construct of the system. Finally, the kill-switch will perform with 98% efficiency given the slow growth rate in the soil predicted by the metabolic model.
</P></SECTION><P style="float:right"><B>Continue to <A href="https://2014.igem.org/Team:Wageningen_UR/project/interlab" class="soft_link"> Interlab Study &gt;&gt;</A></B></P><H1 id="appendix1">Appendix</H1><SECTION id="parameters1"><H2>Table of parameters</H2><H3 id="parameters41">Promoter Design</H3><P><UL class="table"><TABLE width="800px"><TBODY><TR><TH> Designation </TH><TH> Value </TH><TH> Description </TH><TH> Reference </TH></TR><TR><TD>K<SUB>prod<SUB>LacI</SUB></SUB></TD><TD>1 µM min<SUP>-1</SUP></TD><TD>LacI production rate</TD><TD>[9]</TD></TR><TR><TD>K<SUB>prod<SUB>TetR</SUB></SUB></TD><TD>1 µM min<SUP>-1</SUP></TD><TD>TetR production rate</TD><TD>[9]</TD></TR><TR><TD>K<SUB>prod<SUB>CIλ</SUB></SUB></TD><TD>1 µM min<SUP>-1</SUP></TD><TD>CIλ production rate</TD><TD>[9]</TD></TR><TR><TD>K<SUB>prod<SUB>RNAtoxin</SUB></SUB></TD><TD>0.75 µM min<SUP>-1</SUP></TD><TD>Toxin production rate</TD><TD>[9]</TD></TR><TR><TD>Dil</TD><TD>0.023 min<SUP>-1</SUP></TD><TD>Degradation due to cell division</TD><TD>[9]</TD></TR><TR><TD>deg<SUB>LacI</SUB></TD><TD>0.034 min<SUP>-1</SUP></TD><TD>Degradation (Half-life LacI)</TD><TD>[3,9]</TD></TR><TR><TD>deg<SUB>TetR</SUB></TD><TD>0.023 min<SUP>-1</SUP></TD><TD>Degradation (Half-life TetR)</TD><TD>[4,9,43]</TD></TR><TR><TD>deg<SUB>CIλ</SUB></TD><TD>0.0011 min<SUP>-1</SUP></TD><TD>Degradation (Half-life CIλ)</TD><TD>[4,9,43]</TD></TR><TR><TD>deg<SUB>Toxin</SUB></TD><TD>0.0025 min<SUP>-1</SUP></TD><TD>Degradation (Half-life Toxin)</TD><TD>[5]</TD></TR><TR><TD>A</TD><TD>6.02e<SUP>23</SUP></TD><TD>Avogadro’s number</TD><TD>[9]</TD></TR><TR><TD>V</TD><TD>1.5e<SUP>-16</SUP> dm<SUP>-3</SUP></TD><TD>Cell volume</TD><TD>[9]</TD></TR><TR><TD>P</TD><TD>1900</TD><TD>Number of RNA polymerases</TD><TD>[9]</TD></TR><TR><TD>N<SUB>ns</SUB></TD><TD>500,000</TD><TD>Non-specific binding sites</TD><TD>[1,2,9]</TD></TR><TR><TD>K<SUB>b</SUB></TD><TD>1.38e<SUP>-23</SUP>m<SUP>2</SUP>kg s<SUP>-</SUP></TD><TD>Boltzmann constant</TD><TD>[9]</TD></TR><TR><TD>T</TD><TD>298</TD><TD>Temperature in Kelvin</TD><TD>-</TD></TR><TR><TD>∆ε<SUB>pd</SUB><SUP>S</SUP></TD><TD>0.01 nM</TD><TD>Energy of the state, RNA polymerases</TD><TD>[1,2]</TD></TR><TR><TD>∆ε<SUB>d</SUB><SUP>TetR<SUB>LacI</SUB></SUP></TD><TD>100 pM</TD><TD>Energy of the state, TetR repressor</TD><TD>[4,6,9]</TD></TR><TR><TD>∆ε<SUB>d</SUB><SUP>TetR<SUB>Toxin</SUB></SUP></TD><TD>100 pM</TD><TD>Energy of the state, TetR repressor</TD><TD>[4,6,9]</TD></TR><TR><TD>∆ε<SUB>d</SUB><SUP>LacI<SUB>TetR</SUB></SUP></TD><TD>1 pM</TD><TD>Energy of the state, LacI repressor</TD><TD>[3,9,43]</TD></TR><TR><TD>∆ε<SUB>d</SUB><SUP>CIλ<SUB>TetR</SUB></SUP></TD><TD>1 pM</TD><TD>Energy of the state, CIλ repressor</TD><TD>[4,9]</TD></TR><TR><TD>∆ε<SUB>d</SUB><SUP>CIλ<SUB>Toxin</SUB></SUP></TD><TD>1 pM</TD><TD>Energy of the state, CIλ repressor</TD><TD>[4,9]</TD></TR><TR><TD>nTet<SUB>LacI</SUB></TD><TD>-</TD><TD>Repressor binding sites TetR binding PLacI</TD><TD>-</TD></TR><TR><TD>nTet<SUB>Toxin</SUB></TD><TD>-</TD><TD>Repressor binding sites TetR binding PToxin</TD><TD>-</TD></TR><TR><TD>nCIλ<SUB>Toxin</SUB></TD><TD>-</TD><TD>Repressor binding sites CIλ binding PToxin</TD><TD>-</TD></TR><TR><TD>nCIλ<SUB>TetR</SUB></TD><TD>-</TD><TD>Repressor binding sites CIλ binding PTetR</TD><TD>-</TD></TR><TR><TD>nLacI</TD><TD>-</TD><TD>Repressor binding sites Laci binding PTetR</TD><TD>-</TD></TR></TBODY></TABLE></UL></P><H3 id="parameters42">System Cost<P><UL class="table"><TABLE width="800px"><TBODY><TR><TH> Designation </TH><TH> Value </TH><TH> Description </TH><TH> Reference </TH></TR><TR><TD> model iJP962</TD><TD>-</TD><TD>Genomic-scale metabolic model</TD><TD>[6,18]</TD></TR><TR><TD>carbonsource</TD><TD>-</TD><TD>Fructose, Glucose, Alanine, Asparagine, Aspartate, L-Leucine, Serine, L-Threonine, L-Proline, D-Glutamate</TD><TD>[15]</TD></TR><TR><TD>glucuptake</TD><TD>4.43 mmol gDW<SUP>-1</SUP>hr<SUP>-1</SUP></TD><TD>Carbon uptake rate</TD><TD>[1]</TD></TR><TR><TD> NGAM </TD><TD>3.96 mmol gDW<SUP>-1</SUP>hr<SUP>-1</SUP></TD><TD>Non-growth maintenance factor</TD><TD>[6]</TD></TR><TR><TD> KS_rate </TD><TD>0.2 μM min<SUP>-1</SUP></TD><TD>Protein production rate</TD><TD><A href="https://2014.igem.org/Team:Wageningen_UR/project/model#design1" class="soft_link">Promoter Design</A></TD></TR><TR><TD> TAT_rate </TD><TD>1 nM min<SUP>-1</SUP></TD><TD>Toxin/anti-toxin production rate</TD><TD><A href="https://2014.igem.org/Team:Wageningen_UR/project/model#design1" class="soft_link">Promoter Design</A></TD></TR><TR><TD> Cell density </TD><TD>0.040 gDW l<SUP>-1</SUP></TD><TD>gDW/OD600=0.1</TD><TD>[20]</TD></TR><TR><TD> ncells </TD><TD>1e<SUP>8</SUP> cells ml<SUP>-1</SUP></TD><TD>cells/OD600=0.1</TD><TD>[21]</TD></TR><TR><TD> Chitinase </TD><TD>0.2 μM min<SUP>-1</SUP></TD><TD>Chitinase production rate</TD><TD>[7] and <A href="https://2014.igem.org/Team:Wageningen_UR/project/model#design1" class="soft_link">Promoter Design</A></TD></TR><TR><TD> Pyoverdine </TD><TD>150 μmol gDW<SUP>-1</SUP>hr<SUP>-1</SUP></TD><TD>Pyoverdine production rate</TD><TD>Experiment</TD></TR><TR><TD> DAPG </TD><TD>25 μmol gDW<SUP>-1</SUP>hr<SUP>-1</SUP></TD><TD>DAPG production rate</TD><TD>[8]</TD></TR><TR><TD> DMDS </TD><TD>0.845 μmol gDW<SUP>-1</SUP>hr<SUP>-1</SUP></TD><TD>DMDS production rate</TD><TD>[9]</TD></TR><TR><TD> Nplasmids </TD><TD>10 plasmids cell<SUP>-1</SUP></TD><TD>Number of plasmids per cell (low copy number)</TD><TD>[23]</TD></TR><TR><TD> plasmidTD </TD><TD>0.3 h<SUP>-1</SUP></TD><TD>Time needed for at least a doubling of the number of plasmids per cell</TD><TD>[1]</TD></TR><TR><TD>TDrhizo </TD><TD>3 hr </TD><TD>Doubling time <I>P. putida</I> in the rhizosphere</TD><TD>[1]</TD></TR></TBODY></TABLE></UL></P></H3><H3 id="parameters43"> System Performance</H3><P><UL class="table"><TABLE width="800px"><TBODY><TR><TH> Designation </TH><TH> Value </TH><TH> Description </TH><TH> Reference </TH></TR><TR><TD>K<SUB>prod RNA<SUB>LacI</SUB></SUB></TD><TD>0.75 µM min<SUP>-1</SUP></TD><TD>LacI production rate</TD><TD>[9,11]</TD></TR><TR><TD>K<SUB>prod RNA<SUB>TetR</SUB></SUB></TD><TD>0.75 µM min<SUP>-1</SUP></TD><TD>TetR production rate</TD><TD>[9,11]</TD></TR><TR><TD>K<SUB>prod RNA<SUB>CIλ</SUB></SUB></TD><TD>0.75 µM min<SUP>-1</SUP></TD><TD>CIλ production rate</TD><TD>[9,11]</TD></TR><TR><TD>K<SUB>prod<SUB>RNAtoxin</SUB></SUB></TD><TD>0.75 µM min<SUP>-1</SUP></TD><TD>Toxin production rate</TD><TD>[9,11]</TD></TR><TR><TD>deg<SUB>ToxinRNA</SUB></TD><TD>0.178 min<SUP>-1</SUP></TD><TD>Degradation (Half-life Toxin RNA)</TD><TD>[5]</TD></TR><TR><TD>deg<SUB>RNA</SUB></TD><TD>0.82 min<SUP>-1</SUP></TD><TD>Degradation (Half-life RNA)</TD><TD>[45]</TD></TR><TR><TD>deg<SUB>LacI</SUB></TD><TD>0.034 min<SUP>-1</SUP></TD><TD>Degradation (Half-life LacI)</TD><TD>[3,44]</TD></TR><TR><TD>deg<SUB>TetR</SUB></TD><TD>0.023 min<SUP>-1</SUP></TD><TD>Degradation (Half-life TetR)</TD><TD>[43]</TD></TR><TR><TD>deg<SUB>CIλ</SUB></TD><TD>0.011 min<SUP>-1</SUP></TD><TD>Degradation (Half-life CIλ)</TD></TR><TR><TD>deg<SUB>Toxin</SUB></TD><TD>0.000155 min<SUP>-1</SUP></TD><TD>Degradation (Half-life Toxin)</TD><TD>[38]</TD></TR><TR><TD>deg<SUB>Anti-Toxin</SUB></TD><TD>0.0011 min<SUP>-1</SUP></TD><TD>Degradation (Half-life Anti-Toxin)</TD><TD>[38]</TD></TR><TR><TD>Trans<SUB>LacI</SUB></TD><TD>0.12 µM min<SUP>-1</SUP></TD><TD>Translation Rate LacI</TD><TD>[9,45]</TD></TR><TR><TD>Trans<SUB>TetR</SUB></TD><TD>0.08 µM min<SUP>-1</SUP></TD><TD>Translation Rate TetR</TD><TD>[9,45]</TD></TR><TR><TD>Trans<SUB>CIλ</SUB></TD><TD>0.12 µM min<SUP>-1</SUP></TD><TD>Translation Rate CIλ</TD><TD>[9,45]</TD></TR><TR><TD>Trans<SUB>Zeta</SUB></TD><TD>0.12 µM min<SUP>-1</SUP></TD><TD>Translation Rate Zeta</TD><TD>[9,45]</TD></TR><TR><TD>K<SUB>TetR</SUB></TD><TD>4</TD><TD>Dissociation constant (TetR)</TD><TD>[4,6,9,43]</TD></TR><TR><TD>K<SUB>LacI</SUB></TD><TD>1.75</TD><TD>Dissociation constant (LacI)</TD><TD>[3,9,11]</TD></TR><TR><TD>K<SUB>CIλ</SUB></TD><TD>4</TD><TD>Dissociation constant (CIλ)</TD><TD>[4,43]</TD></TR><TR><TD>N<SUB>TetR</SUB></TD><TD>3</TD><TD>Hill constant(TetR)</TD><TD>[1,2,9,10]</TD></TR><TR><TD>N<SUB>LacI</SUB></TD><TD>2</TD><TD>Hill constant(LacI)</TD><TD>[3,9,43]</TD></TR><TR><TD>K<SUB>CIλ</SUB></TD><TD>2</TD><TD>Hill constant(CIλ)</TD><TD>[4]</TD></TR><TR><TD>α</TD><TD>1 nM min<SUP>-1</SUP></TD><TD>Free Toxin-Antitoxin production</TD><TD>[38]</TD></TR><TR><TD>σ</TD><TD>10</TD><TD>Antitoxin factor</TD><TD>[38]</TD></TR><TR><TD>μ<SUB>max</SUB></TD><TD>0.0153-0.0038</TD><TD>Maximum cell growth rate</TD><TD><B>Metabolic Model</B></TD></TR><TR><TD>KP<SUB>1</SUB></TD><TD>1000 nM</TD><TD>Dissociation constant(Toxin Repression)</TD><TD>[38]</TD></TR><TR><TD>KP<SUB>2</SUB></TD><TD>10 nM</TD><TD>Dissociation constant(antiToxin Repression)</TD><TD>[38]</TD></TR><TR><TD>KT<SUB>1</SUB></TD><TD>10</TD><TD>Dissociation constant(Growth inhibition)</TD><TD>[38]</TD></TR><TR><TD>KT<SUB>2</SUB></TD><TD>100</TD><TD>Dissociation constant(Translation inhibition)</TD><TD>[38]</TD></TR><TR><TD>n</TD><TD>6</TD><TD>Half of the total number of Toxin-Antitoxin systems</TD><TD>[38]</TD></TR><TR><TD>p</TD><TD>2</TD><TD>scaling factor</TD><TD>[38]</TD></TR><TR><TD>KH</TD><TD>1000 nM</TD><TD>Dissociation constant(Antitoxin)</TD><TD>[38]</TD></TR></TBODY></TABLE></UL></P></SECTION><SECTION id="allrefs"><H2>References</H2><OL class="references"><H4>Promoter Design</H4><LI>Bintu L, Buchler NE, Garcia HG, Gerland U, Hwa T, Kondev J,Kuhlman T, Phillips R (2005): Transcriptional regulation by the numbers: models. Curr Opin Genet Dev, 15:125-135.</LI><LI> Bintu L, Buchler NE, Garcia HG, Gerland U, Hwa T, Kondev J,Kuhlman T, Phillips R (2005): Transcriptional regulation by the numbers: application. Curr Opin Genet Dev (2), 15:125-135.</LI><LI>Mitchel Lewis (2005) The Lac repressor. C. R. Biologies 328 (2005) 521–548.</LI><LI>Subhayu Basu1, Yoram Gerchman, Cynthia H. Collins3, Frances H. Arnold&amp; Ron Weiss,2: A synthetic multicellular system for programmed pattern formation. Nature 434 ,28 April 2005, 1130-113.</LI><LI>Falcon C.M and Matthews K.S. (2000) Operator DNA sequence Variation Enhances High Affinity Binding by Hinge Helix Mutants of Lactose Repressor Protein. Biochemistry. 39, 11074-11084.</LI><LI> Christian Berens , Dirk Schnappinger and Wolfgang Hillen ; The Role of the Variable Region in Tet Repressor for Inducibility by Tetracycline, Biological Chemisttry Volume 272, Number 11, Issue of March 14, 1997 pp. 6936-6942.</LI><LI>ME346A Introduction to Statistical Mechanics – Wei Cai – Stanford University – Win 2011, Homepage; micro.stanford.edu/.</LI><LI>Lee T.I, Young R.A: Transcription of eukaryotic protein-coding genes. Annu Rev Genet 2000, 34:77-137. p.109.</LI><LI>Milo et al. Nucl. Acids Res. (2010) 38: D750-D753.</LI><LI>Timothy S. Gardner, Charles R. Cantor1 &amp; James J. Collins; Construction of a genetic toggle switch inEscherichia coli, 20 January 2000, Nature 403, 339-342.</LI><LI>iGEM, Aberdeen Schotland 2009, homepage: igem.org.</LI><H4>System Cost</H4><LI>Bennett, R. A., &amp; Lynch, J. M. (1981). Bacterial growth and development in the rhizosphere of gnotobiotic cereal plants. Journal of General Microbiology, 125(1), 95-102.</LI><LI>Molina, L., Ramos, C., Duque, E., Ronchel, M. C., Garcı́a, J. M., Wyke, L., &amp; Ramos, J. L. (2000). Survival of&lt; i&gt; Pseudomonas putida KT2440 in soil and in the rhizosphere of plants under greenhouse and environmental conditions. Soil Biology and Biochemistry, 32(3), 315-321.</LI><LI>Puchałka J, Oberhardt MA, Godinho M, Bielecka A, Regenhardt D, et al. (2008) Genome-Scale Reconstruction and Analysis of the a KT2440 Metabolic Network Facilitates Applications in Biotechnology. PLoS Comput Biol 4(10): e1000210. doi: 10.1371/journal.pcbi.1000210. </LI><LI>Oberhardt, M. A., Puchałka, J., Dos Santos, V. A. M., &amp; Papin, J. A. (2011). Reconciliation of genome-scale metabolic reconstructions for comparative systems analysis. PLoS computational biology, 7(3), e1001116.</LI><LI>Nogales, J., Palsson, B. Ø., &amp; Thiele, I. (2008). A genome-scale metabolic reconstruction of Pseudomonas putida KT2440: iJN746 as a cell factory. BMC systems biology, 2(1), 79.</LI><LI>van Duuren, J. B., Pucha, J., Mars, A. E., Bücker, R., Eggink, G., Wittmann, C., &amp; dos Santos, V. A. (2013). Reconciling in vivo and in silico key biological parameters of Pseudomonas putida KT2440 during growth on glucose under carbon-limited condition. BMC biotechnology, 13(1), 93.</LI><LI>The UniProt Consortium,Activities at the Universal Protein Resource (UniProt),[Pyocin,Q88ID3].</LI><LI>Schnider-Keel, U., Seematter, A., Maurhofer, M., Blumer, C., Duffy, B., Gigot-Bonnefoy, C., ... &amp; Keel, C. (2000). Autoinduction of 2, 4-diacetylphloroglucinol biosynthesis in the biocontrol agent Pseudomonas fluorescensCHA0 and repression by the bacterial metabolites salicylate and pyoluteorin. Journal of Bacteriology, 182(5), 1215-1225.</LI><LI>Arfi, K., Landaud, S., &amp; Bonnarme, P. (2006). Evidence for distinct L-methionine catabolic pathways in the yeast Geotrichum candidum and the bacterium Brevibacterium linens. Applied and environmental microbiology, 72(3), 2155-2162.</LI><LI>Salah El Din, A. L. M., Kyslík, P., Stephan, D., &amp; Abdallah, M. A. (1997). Bacterial iron transport: Structure elucidation by FAB-MS and by 2D NMR (H<SUP> 1</SUP>,C<SUP> 13</SUP>,N<SUP> 15</SUP>) of pyoverdin G4R, a peptidic siderophore produced by a nitrogen-fixing strain of<I> Pseudomonas putida</I>. Tetrahedron, 53(37), 12539-12552.</LI><LI>Boukhalfa, H., Reilly, S. D., Michalczyk, R., Iyer, S., &amp; Neu, M. P. (2006). Iron (III) coordination properties of a pyoverdin siderophore produced by Pseudomonas putida ATCC 33015. Inorganic chemistry, 45(14), 5607-5616.</LI><LI><A href="http://www.mathworks.nl/products/matlab/" class="soft_link" target="_blank">Matlab</A></LI><LI><A href="http://opencobra.sourceforge.net/openCOBRA/Welcome.html" class="soft_link" target="_blank">Cobra Toolbox</A></LI><LI><A href="http://www.gurobi.com/" class="soft_link" target="_blank">Gurobi</A></LI><LI>Buxton, E. W. (1962). Root exudates from banana and their relationship to strains of the Fusarium causing Panama wilt. Annals of Applied Biology, 50(2), 269-282.</LI><LI>Bais, H. P., Weir, T. L., Perry, L. G., Gilroy, S., &amp; Vivanco, J. M. (2006). The role of root exudates in rhizosphere interactions with plants and other organisms. Annu. Rev. Plant Biol., 57, 233-266.</LI><LI>Armstrong, W., &amp; Drew, M. C. (2002). Root growth and metabolism under oxygen deficiency. Plant roots: the hidden half, 3, 729-761.</LI><LI>Oberhardt, M.A., J. Puchalka, V.A.P. Martins dos Santos, and J.A. Papin. 2011. Reconciliation of genome-scale metabolic reconstructions for comparative systems analysis. PLoS Computational Biology, 7(3).</LI><LI>BNID 107924, Milo et al 2010.</LI><LI>BNID 100985, Milo et al 2010.</LI><LI>Gross, H., &amp; Loper, J. E. (2009). Genomics of secondary metabolite production by Pseudomonas spp. Natural product reports, 26(11), 1408-1446.</LI><LI>Stoker, N. G., Fairweathe, N. F., &amp; Spratt, B. G. (1982). Versatile low-copy-number plasmid vectors for cloning in Escherichia coli. Gene, 18(3), 335-341.</LI><H4>System Design</H4><LI>Shu C-C, Chatterjee A, Dunny G, Hu W-S, Ramkrishna D (2011) Bistability versus Bimodal Distributions in Gene Regulatory Processes from Population Balance. PLoS Comput Biol 7(8): e1002140.</LI><LI>Volfson, D. &amp; Marciniak, J. et al.(2005) Origins of extrinsic variability in eukaryotic gene expression. Nature 21 December 2005</LI><LI>Martinez Arias, A. &amp; Hayward (2006), P. Filtering transcriptional noise during development: concepts and mechanisms. Nature Rev. Genet. 7, 34–44</LI><LI>Pallavi Penumetcha, Kin Lau, Xiao Zhu, Kelly Davis, Todd T. Eckdahl, A. Malcolm Campbell.  (2010) Improving the Lac System for Synthetic Biology. BIOS. 7-15</LI><LI>R.A. Fasani, M.A. Savageau  Molecular mechanisms of multiple toxin-antitoxin systems are coordinated to govern the persister phenotype (2013) .  Proc. Natl. Acad. Sci. USA, 110, pp. E2528–E2537</LI><LI>Etienne Maisonneuve, Kenn Gerdes (2014), Molecular Mechanisms Underlying Bacterial Persisters. Cell Volume 157, Issue 3,  Pages 539–548</LI><LI>Rué, P., &amp; Garcia-Ojalvo, J. (2013). Modeling gene expression in time and space. Annual review of biophysics, 42, 605-627.</LI><LI>Swain, P. S. (2004). Efficient attenuation of stochasticity in gene expression through post-transcriptional control. Journal of molecular biology, 344(4), 965-976</LI><LI>Gonze, D. (2013). Modeling the effect of cell division on genetic oscillators. Journal of theoretical biology, 325, 22-33.</LI><LI>Braun, D., Basu, S., &amp; Weiss, R. (2005, March). Parameter estimation for two synthetic gene networks: a case study. In Acoustics, Speech, and Signal Processing, 2005. Proceedings.(ICASSP'05). IEEE International Conference on (Vol. 5, pp. v-769). IEEE.</LI><LI>Purcell, O., Grierson, C. S., Bernardo, M., &amp; Savery, N. J. (2012). Temperature dependence of ssrA-tag mediated protein degradation. J. Biol. Eng, 6(10).</LI><LI>Alon, U. (2006). An introduction to systems biology: design principles of biological circuits. CRC press.
Nevozhay, D., Adams, R. M., Murphy, K. F., Josić, K., &amp; Balázsi, G. (2009). Negative autoregulation linearizes the dose–response and suppresses the heterogeneity of gene expression. Proceedings of the National Academy of Sciences, 106(13), 5123-5128.</LI><LI>Braun, D., Basu, S., &amp; Weiss, R. (2005, March). Parameter estimation for two synthetic gene networks: a case study. In Acoustics, Speech, and Signal Processing, 2005. Proceedings.(ICASSP'05). IEEE International Conference on (Vol. 5, pp. v-769). IEEE.</LI></OL></SECTION></DIV></DIV></DIV></DIV><DIV id="footer"><P>iGEM Wageningen UR 2014. Send us an <A href="mailto:igemwageningen@gmail.com" class="soft_link">email</A>, or follow us on <A class="soft_link" href="http://twitter.com/igemwageningen" target="_blank">Twitter</A> and <A class="soft_link" href="http://www.facebook.com/iGEMWageningen" target="_blank">Facebook</A>!
			
			For more information about iGEM, check out their <A class="soft_link" href="https://igem.org">website</A> and <A class="soft_link" href="http://twitter.com/igem">Twitter</A>!</P></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Wageningen_UR/project/model">http://2014.igem.org/Team:Wageningen_UR/project/model</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Wageningen_UR/project/model" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Wageningen_UR/project/model" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Wageningen_UR/project/model&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Wageningen_UR/project/model&amp;oldid=399502" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>