"
Page
Discussion
View source
History
teams
Log in
 
Team:Wageningen UR/overview/results
From 2014.igem.org
Wageningen UR iGEM 2014
 
 
 
Results
Sensing
Inhibition
Kill-Switch
System Model
Greenhouse
In short
Overview
Background
Solution
Results
Achievements
BioBricks
Project
Fungal Sensing
Fungal Inhibition
Wetlab
Greenhouse
Kill-Switch
Kill-Switch
Characterization
Gene Transfer
Gene Transfer
Low Copy Plasmids
Model
Promoter Design
System Cost
System Stability
Interlab Study
Outreach
Policy
Collaboration
Synenergene
Interviews
National Meetup
Media
Team
Members
Students
Supervisors
Gallery
Attributions
Acknowledgements
Contact
Safety
Environment
Laboratory
Notebook
Journals
Protocols
Interesting Links
Results
In this section we present the key results of the BananaGuard project (figure 1). By combining both experimental and modelling techniques, we have designed, implemented and tested a system that protects banana plants from Fusarium oxysporum. The BananaGuard system senses fusaric acid and, in response, produces fungal growth inhibitors to prevent infection of the banana plant. In addition we have also implemented a Kill-switch that disables our system when fusaric acid is not present. For this, we have optimized the circuit design, assessed the potential of BananaGuard in the soil, and analyzed the robustness of the system using different mathematical models.
Figure 1: The BananaGuard time line. After application, BananaGuard will: 1. detect F. oxysporum based on fusaric acid secretion; 2. produce fungal growth inhibitors, and; 3. destroy itself when F. oxysporum cannot be detected anymore.
Sensing
While fusaric acid dependent protection systems have been observed in micro-organisms and predicted in silico after genome sequencing, no evidence of a validated fusaric acid dependent promoter could be found in the literature. In this project we characterised and validated such a promoter. A putative fusaric acid dependent promoter along with its hypothesized regulator (isolated from Pseudomonas putida KT2440 (P. putida)) were cloned in front of GFP (BBa_E0040) , transformed in P. putida KT2440. Promoter function was quantified by measuring fluorescence in the presence of different fusaric acid concentrations (figure 2). We were able to validate and characterize a novel fusaric acid dependent promoter (Bba_K1493000) .
Figure 6.
*Significantly different from WT with p<0.05
The measurement is based on GFP fluorescence in P. putida at increased concentrations of fusaric acid to prove and characterize the activity of the fusaric acid induced promoter, BBa_K1493000 . For comparison, the well characterized pLac promoter ( BBa_K741002 , uninduced by IPTG) was used to quantify the activity of this promoter at different concentrations of fusaric acid. Our fusaric acid inducible promoter does not respond to low concentrations up to 170µM. From 255µM and up, the activity increases. The maximum measured activity of the promoter is 0.21 RPU at 425µM.
For more information, read fungal sensing .
Inhibition
Upon sensing fusaric acid, three genes and a gene cluster will be activated that will lead to production of fungal growth inhibitors. They were cloned behind an IPTG inducible promoter. Those genes and their function being:
phlABCDE gene cluster, able to produce 2,4-Diacetylphloroglucinol(2,4-DAPG)
Methionine-γ-lyase, Dimethyldisulfide (DMDS) and dimethyltrisulfide (DMTS)
PfrI, produce pyoverdine in presence of iron
Chitinase, overexpresses chitinase activity
Methionine-γ-lyase and PfrI were both made into BiobBicks, Bba_K1493300 and Bba_K1493200 respectively. With both BiobBicks validated, and for PfrI characterized. PfrI has shown to give a four fold increase of pyoverdine production in the presence of iron (31μM) in the growth medium. Pyoverdine is a compound that chelates iron and is naturally only produced when there is no iron available.
Figure 3. Pyoverdine production in M9 medium supplemented with 31μM iron with error bars,OD corrected. empty plasmid= P. putida containing an empty plasmid for control.
All transformants were co-inoculated with Fusarium oxysporum cubense TR4 on agar plates in order to test their inhibition ability. Controls used were plates inoculated with F. oxysporum in the presence and absence of wild type P. putida KT2440.
Figure 4.In vivo assay, P. putida co-inoculated with F. oxysporum. Red circle indicates area occupated by F. oxysporum growth. Foc control=Fusarium oxysporum cubense TR4, WT=wild type P. putida KT2440, DAPG=P. putida containing phlABCDE gene cluster, MgL=P. putida containing methionine-γ-lyase, chitinase=P. putida overexpressing chitinase, Pyoverdine= P. putida overexpressing pfri and mix(all 4)=all 4 tranformants mixed.
In general, it was hard to distinguish the increased inhibition effect of the fungal growth inhibitors producing P. putida against F. oxysporum. This is because the P. putida chassis we have chosen is already very good at inhibiting F. oxysporum naturally, which probably makes it hard to observe increased growth inhibition by our synthetic, growth inhibitor producing P. putida strains. However, with the Methionine-γ-lyase(MgL) strain, we have a clear indication that there is an enhanced growth inhibition of F. oxysporum (figure 4).We can say that there is an indication of a slight increase of growth inhibition, on top of the natural inhibitionin, the 2,4-DAPG, chitinase and pyoverdine producing strains. For more information, read fungal inhibiton .
Kill-Switch
Once fungal growth inhibitors are produced and F. oxysporum is no longer in the soil BananaGuard has done it's job and is no longer needed in the soil. Therefore we have implemented a Kill-switch into our system, which works like a toggle switch that senses when fusaric acid is around, and when it has dissipated toxins will be produced that eliminate BananaGuard. Toxins will be produced that eliminate BananaGuard itself, with the kill switch regulatory system in figure 4.
Figure 5. the overview of the Kill-switch regulatory system showing al possible repressions. To simplify things in wetlab rhamnose input (white dots) is used instead of fusaric acid and GFP output (green dots) is used instead of toxins.
CIλ (induced by rhamnose) has shown to suppress the pcIλ/Tet promoter inhibiting GFP production when induced with rhamnose (figure 5).         In addition, CIλ and lacI has also shown to suppress GFP production of the pCIλ/lacI promoter (figure 6). A toggle-switch was constructed ( Bba_K1493702 , Bba_K1493703 ) containing pCIλ/lacI promoter + TetR together with pTet + LacI + GFP. After establishing that induction by IPTG leads to adequately low GFP expression (the off-state), whereas induction by aTc results in high GFP expression (the on-state).The non induced state of the toggle switch shows a fluorescence value between the two induced states, indicating that it might be stable. For more information, read Kill-switch .
Figure 6. CIλ induced by rhamnose suppressing pCIλ/Tet promoter expressing GFP (input output plasmid. Plates were done in duplicate. Top plates are plates without rhamnose and the bottom plates are plates with rhamnose.
Figure 7. A) The average RFU values of E. Coli carrying a pSB3K3 plasmid containing the BioBricks pRha CIλ and pCI/lac gfp. B) The average RFU values of E. Coli carrying a pSB3K3 plasmid containing the BioBricks pRha lacI and pCI/lac gfp. Cells were grown in M9 medium with 2% glycerol and induced with 0%, 0.001%, 0.01%, 0.05% or 0.2% L-rhamnose or 0.2% glucose at t=0. Fluorescence was measured over time and data of time point 8.13 are shown in the graphs. Rhamnose concentrations of 0.001% and 0.01% have no substantial effect on fluorescence, compared to 0% rhamnose. Cells grown in 0.05% and 0.2% rhamnose show a lower RFU value compared to 0% rhamnose indicating that the pCI/lac is repressed by the repressor protein regulated by the rhamnose promoter. 0.2% glucose has an effect on the RFU, as the values are lower than 0% rhamnose.
In order to characterize promoters that are non-inducible we developed a new characterization method that is based on a rhamnose inducible promter. We used this system to characterize a promoter repressed cIλ repressor and LacI repressor (figure 7.) For more information, read rhamnose characterization .
Figure 8. The relative fluorescence unit of each toggle switch state. Fluorescence is measured in duplicate of cell cultures carrying the pSB3K3 plasmid with the toggle switch construct (BBa_K1493702, BBa_K1493703) grown in M9 medium containing 500 ng/ml aTc (green), 2 mM IPTG (red) and with no inducer added to the medium (blue).
Promoter design model
The kill-switch design is relatively intricate and therefore requires in silico analysis in order to test and improve its architecture. In order to perform such analysis we exploited statistical mechanics to derive a model of the promoter system. Not unexpectedly, the new insight obtained strongly favoured some adaptations to the current design, which included reallocation of promoters as well as parallel placement of an additional kill-switch, which according to the predictions would yield a more stable system. For more information, read model kill-switch promoter design .
Figure 9. Colour maps indicating functioning and non-functioning systems. Each letter represents different repressor binding site configurations. Each small square within the colour maps represents a score for a simulation of the system with a unique set of parameters. The colours correspond to the previously given description.
2: The system performs to design; after a rhamnose input the toggle switch changes state and GFP is produced when CIλ leaves the system
1: The system performs less efficiently; though the toggle switch changes state, the GFP promoter is leaky
0: The system does not work; the toggle switch is out of balance and does not function, the system favours either LacI or TetR
Back to the lab
With new output from the promoter model, new promoters were made with different inhibitor binding sites with the BioBrick standard in mind. These promoters were then assembled in front of GFP and all 5 out of the 6 promoters we have designed have shown to work ( Bba_K1493801 , Bba_K1493802 , Bba_K1493804 , Bba_K1493805 and Bba_K1493806 ), by showing GFP expression (see figure 10). For more information, read promoter design results .
Figure 10. New constructed promoters upstream GFP, expressed in E. coli. DH5alfa. There are 6 plates with each two streaks of two different colonies, except for the plate where P4 is plated, that is a streak of only one colony. where you can see streaks of different colonies with promoter 1-6 expressing GFP (P1-P6). The only promoter that does not seem to express GFP is promoter 3 (P3).
System model
Cost
Having the whole system in P. putida is great however there is always metabolic stress in everything that we want P. putida to produce. Therefore another model was developed to predict the cost of the whole system, using a genome-scale constraint based metabolic model. The model indicates that the metabolic stress introduced by fungal growth inhibitors production should not pose a bottleneck. For more information read system cost .
Figure 11. The relative growth rate compared to the wild type P. putida for different carbon uptake rates. Two different carbon sources are tested, banana exudates and and glucose as reference. The expected carbon uptake rate of P. putida in the rhizosphere is indicated with transparent red.
Performance
Knowing that P. putida is able to cope with the whole system, the next objective is to assess the performance of the system; will the kill-switch function according to our expectations? Will the kill-switch kill P. putida in advance of performing its intended role as a fungicide due to imbalance of the toxin anti-toxin system?In order to answer these questions we created a stochastic whole system model, incorporating metabolic stress, leakiness of each individual promoter and the toxin anti-toxin syswtem. The results of this analysis are depicted in figure 4.
Figure 12. Two histrograms showing the effect of leaky promoters on the system and the performance of the system upon induction by fusaric acid.(A) For a Maximum growth rate of 180 minutes the stability of the kill-switch a basal CIλ production of 50 nM/min or higher destabilizes the kill-switch. The population dynamics are affected. Low protein dilution due to slow growth causes the Kill-switch to leak toxin. Higher basal production rates compensate, increasing the average growth rate but also the instability. A total of 5000 simulation were run.(B) For a maximum growth rate of 180 minutes 98% of the kill-switches activate, longer division times activate the cells more effectively. A total of 20000 simulations were run
The stochastic model (figure 11) has shown that different basal production levels of CIλ can have different effects on population dynamics, cell growth and the stability of the kill-switch, a point of attention for final construct of the system. Finally, the kill-switch will perform with 98% efficiency given the slow growth rate in the soil predicted by the metabolic model. For more information read model performance .
Greenhouse
We established a collaboration with the Plant Research International group of Wageningen, which gave us the unique opportunity to test the system, not only against F. oxysporum, but in a setting that mimics the situation outside the lab as closely as possible with banana plants. At present we have banana plants in the greenhouse (figure 12) that have been inoculated with our engineered P. putida and which were also infected with F. oxysporum. However, plants grow at a much slower pace than bacteria, so results were not possible to be obtained before the wiki-freeze (see greenhouse ).
Figure 13: Banana plants two weeks after inoculation.
In short
We as an iGEM team have achieved quite a lot during these couple of months. Here is a short list of what we have achieved:
Validated and characterized a novel working fusaric acid dependent promoter
Proved that pyoverdine can be produced in an environment with iron
Improve inhibition of P. putida towards F. oxysporum
Show the proof of concept of the Kill-Switch using the input-output plasmid system
Extensively characterized the rhamnose promoter
Developed and used a new characterization method based on the rhamnose promoter
Create a toggle switch that can be activated
Combined modelling and wetlab results for promoter design
Have made new promoters and validated their function for future use in iGEM!
Built a model that predicts the metabolic cost of BananaGuard on P. putida
Constructed a model that shows the performance of our whole BananaGuard system
In parallel with obtaining results from the lab, we have also been working towards implementing the BananaGuard system in greenhouse-grown banana plants. Will our engineered P. putida win the fight against F. oxysporum? Is it strong enough to survive in the complex soil rhizosphere? Will it save the banana plants? Sadly, the results were not here before the wiki-freeze, however we will present them at the jamboree! So stay tuned and come to our presentation!
Continue to Achievements >>
iGEM Wageningen UR 2014. Send us an email , or follow us on Twitter and Facebook !
For more information about iGEM, check out their website and Twitter !
Retrieved from " http://2014.igem.org/Team:Wageningen_UR/overview/results "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
