<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Guelph_Project"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:Guelph/Project">Page               </A></LI><LI><A href="/Talk:Team:Guelph/Project">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Guelph/Project&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Guelph/Project&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Guelph/Project" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Guelph/Project</H1><DIV id="bodyContent"><H3 id="siteSub">From 2008.igem.org</H3><TABLE cellpadding="3" cellspacing="5" border="4" width="960px"><TBODY><TR><TH style="font-size:140%; text-align:center; background-color:#ffff00; border-width:0px; padding:3px;"><A href="/Team:Guelph" title="Team:Guelph"><FONT color="#cd0000">Home</FONT></A></TH><TH style="font-size:140%;text-align:center; background-color:#ffff00; border-width:0px; padding:3px;"><A href="/Team:Guelph/Team" title="Team:Guelph/Team"><FONT color="#cd0000">The Team</FONT></A></TH><TH style="font-size:140%;text-align:center; background-color:#ffff00; border-width:0px; padding:3px;"><STRONG class="selflink"><FONT color="#cd0000">The Project</FONT></STRONG></TH><TH style="font-size:140%;text-align:center; background-color:#ffff00; border-width:0px; padding:3px;"><A href="/Team:Guelph/Parts" title="Team:Guelph/Parts"><FONT color="#cd0000">Parts</FONT></A></TH><TH style="font-size:140%;text-align:center; background-color:#ffff00; border-width:0px; padding:3px;"><A href="/Team:Guelph/Notebook" title="Team:Guelph/Notebook"><FONT color="#cd0000">Notebook</FONT></A></TH><TH style="font-size:140%;text-align:center; background-color:#ffff00; border-width:0px; padding:3px;"><A href="/Team:Guelph/Results" title="Team:Guelph/Results"><FONT color="#cd0000">Results</FONT></A></TH><TH style="font-size:140%;text-align:center; background-color:#ffff00; border-width:0px; padding:3px;"><A href="/Team:Guelph/Links" title="Team:Guelph/Links"><FONT color="#cd0000">Links</FONT></A></TH></TR></TBODY></TABLE><TABLE align="justify"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Title"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Title</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Project_Abstract"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Project Abstract</SPAN></A></LI><LI class="toclevel-1 tocsection-3"><A href="#Overall_project"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Overall project</SPAN></A></LI><LI class="toclevel-1 tocsection-4"><A href="#Project_Details"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Project Details</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Background_and_Justification"><SPAN class="tocnumber">4.1</SPAN><SPAN class="toctext">Background and Justification</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#The_Experiments"><SPAN class="tocnumber">4.2</SPAN><SPAN class="toctext">The Experiments</SPAN></A></LI></UL></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Title"><B>Title</B></SPAN></H2><P><B>Reprogramming microbes to cater to or silence their hosts - beta carotene</B>
production and RNAi delivery
</P><H2><SPAN class="mw-headline" id="Project_Abstract"><B>Project Abstract</B></SPAN></H2><P>Humans contain billions of microbes which help digest our food and produce vitamins to suppliment 
our diet, while plants such as corn harbour microbes within their tissues 
which can extend the metabolic capacity of their host. In an attempt to 
exploit these patterns of microbial habitation, we built constructs which were inserted into                         the broad host range plasmid pDSK-GFPuv to contain either a synthetic operon of metabolic genes 
belonging to the soil microbe <I>Erwinia uredovora</I>, or Biobrick compatible 
RNAi constructs targetting the plant expression of either GFP or corn TB1 genes. These plasmids were
to be electroporated into either probiotic Escherichia coli Nissle 1917 or endophytic 
Klebsiella pneumonii 342. Assays will then show whether a genetically modified 
enteric microbe could be made to produce vitamin A in a modelled human intestine, 
or whether a common corn endophyte could stably express and deliver RNAi 
signals against expression of GFP and corn TB1 genes while living inside a growing corn plant.
</P><H2><SPAN class="mw-headline" id="Overall_project"><B>Overall project</B></SPAN></H2><P>Microbes are found in every nook and cranny of the planet, and multicellular organisms are no exception. Plants and animals are found to contain huge numbers of bacteria and fungi that help with nutrient absorption, producing beneficial compounds, fighting off pathogens, or often are pathogens themselves. We are interested in taking advantage of some of these microbes to deliver transgenic payloads for the benefit or modification of the host organism. These might be called GM endosymbionts. On the human side, we would like to introduce the carotendoid metabolic genes from a well studied soil microbe called <I>Erwinia urodovora</I> into human probiotic microbes which will survive and colonize the intestine for stable production of the essential human nutrient, pro-vitamin A. Time permitting, we will attempt to enhance carotenoid accumulation by increasing plasma membrane sink by overexpression of the fumarate reductase operon. 
</P><P>Millions of humans suffer from vitamin A deficiencies across the world, resulting in blindness and death which could be mitigated by symbitic production of this important vitamin. We will also attempt to get corn endosymbionts to produce pro-vitamin A, a function usually performed by the plant itself. In both cases, the ethical and practical implications of this technology will affect how it might ever be implemented, so we are very excited to be collaborating with the Calgary Ethics team to ruminate on the possibilities and pitfalls of this bit of synthetic biology. 
</P><P>A more basic wet lab project we are doing will focus on RNAi signal delivery by a corn plant endosymbiont to silence corn genes. Most euaryotes react to double stranded RNA by copying it and chopping up any mRNA with the same sequence, while prokaryotes do not seem to possess the same response. Since bacteria like <I>K. pneumonii</I> live in large stable populations within corn plants, it is believed that as individual bacteria grow, die and lyse within the plant host, they will release RNAi transcripts into the sensitive host during the entire life cycle of the plant, which will silence the targeted gene and show a phenotype indicating gene function. Bacterial Induced Gene Silencing (BIGS) will be a useful, quick, and stable alternative for plant functional genomic research. This will be an enableing technology for scientists studying plant gene function, as well as a Biobrick registry value adding tool; any Biobrick compliant peice of DNA will be clonable into a double stranded RNA orientation that could be expressed by bacteria (or plants for that matter).
</P><H2><SPAN class="mw-headline" id="Project_Details"> Project Details</SPAN></H2><H3><SPAN class="mw-headline" id="Background_and_Justification"> Background and Justification </SPAN></H3><P>1) Beta-carotene production by endosymbionts: For socioeconomic and environmental reasons, this might be advantageous. To a large extent, this will be answered by our collaborators at iGEM Calgary Ethics, so stay posted for their results and paper. Their results are posted on their wiki site here : <A href="https://2008.igem.org/Team:Calgary_Ethics/Collaboration" class="external free" rel="nofollow">https://2008.igem.org/Team:Calgary_Ethics/Collaboration</A></P><P>
2) BIGS for functional genomics: Functional genomics is an area of genetics that uses genomic information to help understand gene function. Currently, the most straightforward way to do this for any given gene is to analyze a single gene mutant to discover that organism's deficiency, and thus the gene's normal function. If you're lucky, you can find a natural mutant, or make one using random mutagenisis. However, in complex eukaryotes with tens of thousands of genes, it is impossible to expect that you can get knock outs for any particular gene you want to study. As of a decade ago, it began to be possible to use RNAi to selectively knock out your particular gene by making a transgenic organism expressing double stranded RNA with the same sequence of the target gene. For a eukaryote like corn, this makes it possible to knock out one particular gene you want to study, but it does require about a year for that transgenic corn to be made. An alternative in some eukaryotes is Viral Induced Gene Silencing, or VIGS, whereby the double stranded RNA is transiently delivered into the organism by a virus. VIGS has only been developed for certain eukaryotes however (not including corn), and even so is only a transient way to silence a gene, giving a scientist only a portion of the eukaryote's life cycle where gene function might be studied. We ask ourselves, can a system be made which could effectively silence eukaryotic genes using RNAi, but more quickly and cheaply than making a transgenic eukaryote and more stably and long term than by using VIGS? Our idea is that a normal endosymbiont of an organism could do this if it can be coaxed to produce this RNAi signal. Read on to find out how we'll test this idea...
</P><H3><SPAN class="mw-headline" id="The_Experiments"> The Experiments </SPAN></H3><P>1) Beta carotene producing intestinal microbes and plant endophytes
</P><P>
Tentative cloning strategy: 
</P><P>So what is the stategy we should follow? Initially this 
Hydrogen synthetic operon paper was inspiring us: 
</P><P><A href="http://www.ncbi.nlm.nih.gov/pubmed/17996187" class="external free" rel="nofollow">http://www.ncbi.nlm.nih.gov/pubmed/17996187</A></P><P>
Its specific for a bacterial synthetic 
operon and if we followed it we wouldn't have to rework things too hard in order to follow it.
</P><P>
However, this was not a perfectly Biobrick compliant strategy, and there were unecessary extra steps we'd have to follow so we reworked our strategy. Instead the strategy we followed depended on sequentially adding genes to an operon by cutting the a biobrick plasmid containing CrtE with SpeI and PstI, and adding each consecutive XbaI / PstI cut gene via ligation. The last gene will have transcription terminators on the end. Next we would add promoters, and uing EcoRI and XbaI, then cut the transcriptional unit out and transfer to other plasmids using EcoRI and PstI.  
</P><P>The promoter that first interested us was the strong constitutive promoter from 
herbicide tolerant Amaranthus weeds which is unregulated (and therefore constitutive) in prokaryotes.  To get this we should PCR up the promoter from the pDSK-GFPuv plasmid (introducing an SpeI site after the native NdeI), and put it into BBa_E0240 to test promoter activity by visually checking GFP levels. 
</P><P>
With a strong, constitutive, biobricked promoter having an NdeI on the start codon followed by an SpeI site, we're ready to build the operon. The gene order for the operon has already been optimized. This paper explains we should have E-B-I-Y. Check it out:
</P><P><A href="http://aem.asm.org/cgi/content/abstract/AEM.02268-06v1" class="external free" rel="nofollow">http://aem.asm.org/cgi/content/abstract/AEM.02268-06v1</A></P><P>
The initial gene, crt-E, on plasmid p3-10-10 will be amplified and appropriate flanking , XbaI and NdeI (on or before the start codon) and SpeI (after the stop codon) restriction sites introduced. There is also a ribosome binding site present in front of each of the carotenoid genes, and thus we will conserve them in our cloning strategy.  We will cut the plasmid with XbaI and SpeI then dephosphorylate it with antartic alkaline phosphatase, kindly provided by our sponsor New England Biolabs. Likewise, the PCR product will be cut with XbaI and SpeI, and ligated to the dephosphorylated plasmid. 
</P><P>
The Biobricked crt-B PCR product will also be flanked with XbaI and SpeI, and ligated into its own pSB1A2 plasmid where it will attain the EcoRI and PstI sites as well. To insert this crt-B after crt-E, we will cut it with XbaI and Pst, while cutting the plasmid with SpeI and PstI. This will destroy the SpeI site on the plasmid and the XbaI site on the PCR product, but these complementary sticky ends have the benefit of decreasing the rate of background ligations; the complementary sticky ends act as 'heat seeking missiles', seeking out their complementary mates and ignoring non-complementary overhangs or blunt end. The PstI site will be conserved for future use. 
</P><P>
Next comes Crt-I. This is done in the same way as above, as it is also flanked with XbaI and PstI with an internal SpeI site. Once again, the biobricked open reading frame is cut with XbaI and PstI, while the plasmid is cut with SpeI and PstI. Ligate. 
</P><P><I>(note: Crt-I has two PstI sites which really should be removed but don't have to if we don't have time - we have a technique we'd like to try to do this using some sort of nested PCR - there are a couple primers in the list</I>
which could be used to use to do this - in the end we just got the Biobricked gene from iGEM Edinburgh so all good)
</P><P>
At this point we will attempt to add other genes to the sequence, such as the fumarate reductase operon since its overexpression results in increased cell membrane, which is where lipid soluble carotenoids are stored. Here's the link the paper on fumarate reductase overexpression: 
</P><P><A href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=215469" class="external free" rel="nofollow">http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=215469</A></P><P>
In plants it has been shown that sink strength is a limiting factor to carotenoid accumulation, so giving bacteria more carotenoid storage space may result in much more carotenoid accumulation. Here's a discussion about engineering plant sinks for increased carotenoid accumulation: 
</P><P><A href="http://www.isb.vt.edu/articles/feb0802.htm" class="external free" rel="nofollow">http://www.isb.vt.edu/articles/feb0802.htm</A></P><P>These two complete operons (both ending in the transcriptional terminator containing GFP biobrick BBa_E0240) will then be put under control of the PSBA promoter or the arabinose inducible promoter, and the whole transcriptional cassette cut out to be inserted into broad host range plasmids like pDKS-GFPuv or pTG262 which will be electroporated into biologically relevant microbes such as probiotic lactobacilli, corn endophytic <I>Klebsiella pneumonii</I> 342,  or <I>E. coli</I> Nissle 1917 which is sold in some countries as a probiotic for intestinal health. We will then test these microbes for beta carotene production (which ever we end up being able to get to work) in conditions simulating those we are interested in, such as human intestines. Take a look at our strategy :
</P><P>Our strategy : 
</P><P>2) RNAi inducing corn endophytes (BIGS)
</P><P>Functional genomics is the study of gene function in an organism through (genomics) informed study of transcription, translation, interactions, and phenotype. A common way to do this is by making mutants and assaying for different phenotypes; a very convenient way to accomplish this is by using RNA interferance (RNAi). Usually this is done by making transgenic plants expressing RNAi constructs, and is stable but limited by the difficulty and length of time it takes to produce the transgenic plant. In the case of corn, this means almost a full year before you can test your gene function.  
</P><P>
An alternative is called viral induced gene silencing (VIGS) which can uses a plant specific virus to deliver and transiently silence the gene. An obvious limitation here is that it is a transient process even if it has been developed for a particular plant, which it has not been for corn. 
</P><P>
We propose a happy medium which would be relatively easy and cheap, stable during the whole lifecycle of the plant, Biobrick compatible (allowing RNAi constructs to be made from any biobrick compatible fragment) and importantly, quick and effective. Bacterial Induced Gene Silencing (BIGS) will employ an endophytic microbe (lives mutualistically inside a plant) to deliver an RNAi signal against a plant gene in order to study that gene's function. We are hoping that as these microbes die, they will release these molecules into the medium, and since plants are very sensitive to these double stranded RNA molecules, they will sense AND systemically amplify the signal across the entire plant body. 
</P><P>
There are relevant precedants for this area of research, but it has never before been done using endophytes in plants which could theorhetically produce this signal during the entire life cycle of the host. Here are some relevant publications supporting the potential of our work which we will not further discuss: 
</P><P>
Spraying RNAi E. Coli onto plants to protect against viruses - <A href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=153545" class="external free" rel="nofollow">http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=153545</A></P><P>Using E. coli to deliver RNAi signals to tumors in mouse colons - <A href="http://www.ncbi.nlm.nih.gov/books/bv.fcgi?indexed=google&amp;rid=eurekah.section.75458" class="external free" rel="nofollow">http://www.ncbi.nlm.nih.gov/books/bv.fcgi?indexed=google&amp;rid=eurekah.section.75458</A></P><P>
We are planning to order a 200 bp <I>Zea mays</I> Actin1 intron construct from our sponsoring DNA synthesis company, IDT, which will consist of shuffled Biobrick ends (XbaI and PstI will switch spots) which will allow RNAi compatibility with any pre-existing Biobrick in the registry. We will program the ribosome binding site sequence AGGAGG into the middle of the intron, as well as the 3' and 5' ends of the construct to improve RNA stability by ribosome binding. We will also program NdeI and AvrII (compatible end to XbaI and SpeI) restriction sites into the intron where we will clone a GFP gene for monitoring of RNAi construct transcription in vivo. The strong synthetic transcription termination signal from Biobrick BBa_B1006 will also be programmed into the construct.  Cloning will take place with the antisense biobrick going in first using NotI and SpeI sites, followed by the sense construct and XbaI and EcoRI restriction sites. WIth both orientations in place the constructs will be ligated to the strong constitutive PbsA promotor in the broad host range proprietary vector pDSK-GFPuv and then electroporated into the corn endophyte, <I>Klebsiella pneumonii</I> 342. GFP silencing will be tested by innoculation of <I>Arabidopsis</I> plants constitutively expressing GFP, while corn gene silencing activity will be monitored by testing endophyte silencing efficiency of the TB1 gene whose expression inhibits branch formation in corn plants. 
</P><P>Our strategy: </P></TABLE><DIV class="printfooter">
Retrieved from &quot;<A href="http://2008.igem.org/Team:Guelph/Project">http://2008.igem.org/Team:Guelph/Project</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Guelph/Project" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Guelph/Project" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Guelph/Project&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Guelph/Project&amp;oldid=110826" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2008.igem.org:Privacy_policy" title="2008.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2008.igem.org:General_disclaimer" title="2008.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>