"
Page
Discussion
View source
History
teams
Log in
Â 
Team:Guelph/carotenoids
From 2008.igem.org
Tentative cloning strategy:
So what is the stategy we should follow? AThis is found in that Hydrogen synthetic operon paper.
http://www.ncbi.nlm.nih.gov/pubmed/17996187
Its specific for a bacterial synthetic operon and we don't have to rework things too hard in order to follow it.
I am making a Biobrick vector with a strong constitutive promoter from herbicide tolerant Amaranthus weeds which is unregulated (and therefore constitutive) in prokaryotes. To do this we should PCR up the promoter from the pDSK-GFPuv plasmid, and put it into this:
http://partsregistry.org/wiki/index.php?title=Part:pSB1AK3
Likewise, I am PCRing up and inserting a restriction site optimized GFP into the plasmid. Then, we're ready to clone the carotenoid genes into the NdeI space in between promoter and GFP.
The gene order for the operon has already been optimized. This paper explains we should have E-B-I-Y. Check it out:
http://aem.asm.org/cgi/content/abstract/AEM.02268-06v1
So, the initial gene, crt-E on plasmid p3-10-10 will be amplified and appropriate flanking NdeI restriction sites introduced, alongside an internal ribosome binding site after the stop codon of the gene, followed by StuI (blunt RE) and AvrII. Cut the PCR product and the plasmid with NdeI, dephosphorylate the plasmid and ligate.
Now, to join Crt-B in there, cut it with SwaI (blunt RE) and AvrII, while cutting the plasmid with StuI and AvrII. Ligate. Again, there is a "AGGAGG" RBS directly after the stop codon which will promote translation of the following ORF. (note: CrtB has an internal StuI site which will have to be removed - there is a primer I included for this purpose, and would be used for PCR mediated mutagenesis and amplification followed by cutting with SmlI and StuI and ligating into the original PCR product)
Next comes Crt-I. This is done in the same way as above. cut it with SwaI and AvrII, while cutting the plasmid with StuI and AvrII. Ligate. (note: Crt-I has two PstI sites which really should be removed but don't have to if we don't have time - I have a technique I'd like to try to do this using some sort of nested PCR - there are a couple primers in the list which could be used to use to do this)
Finally, we insert Crt-Y. Cut it (PCR product) with SwaI and AvrII, while cutting the plasmid with StuI and AvrII. Ligate.
The GFP at the end should hopefully work as a reporter to let us know the transcript is OK, and the RBS sites are all lined up properly. If we want to add any more genes, we use the same protocol with SwaI, StuI and AvrII. Since these are all plasmid based PCRs, I expect it will be easier to get strong amplification than if we were using genomic DNA. In any case, I hope we could use proof reading enzymes for these PCRs - I have recently been turned on to Phusion from NEB. What do you use?
Retrieved from " http://2008.igem.org/Team:Guelph/carotenoids "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
