"
Page
Discussion
View source
History
teams
Log in
 
Team:NJAU China/Wet Labs
From 2014.igem.org
Home
IGEM
Home
Project
Background
System Design
Safety
Achievement
Wet Lab
Model
Protocol/Notebooks
Biobricks
Practice
Exhibitions and Workshops
Questionnaire Survey
Collaboration
Team
Acknowledgement
Inside Campus&Arrtibutions
Outside Campus
Protocol
Adjective Experiments
Fundamental System
Facilitating System
Adjective Experiments
Besides the experiments to build our system, we also did some adjective experiments to testify the harmless of copper ions to organisms and the efficiency about GSH compound with copper ions.
Will GSH detoxify Cu2+ in water or will GSH become the new pollution?
Cu2+ can cause great damage to many kinds of aquatic organisms especially alga. It has been reported that Cu2+ under 1ppm can affect the growth of alga. We use alga to measure the harm of copper ions in water, hoping to get a more accurate results.
Chlamydomonas reinhardtii, a common alga which can find in most flesh water, is chosen as the alga's represent. And the trend of relationship among cupric ions, GSH and Chlamydomonas reinhardtii are showed below.
A. High level of GSH suppress the alga. However, the dose-effect relationship doesn't look like normal toxin. It must affect the nutrition in medium.TAP medium contains Hutner's trace elements, which must be oxidized for one week. So, GSH as an antioxidant can reduce Hutner's trace elements' effect.
B. With no GSH or CuSO4 ,alga grows in the most fast rate.
C. CuSO4 as an algicide is highly toxic to algas.But it shows that Chlamydomonas reinhardtii are more tolerant to copper ions than the normal alga.
As a result, GSH does have significant detoxification effect!
Fundamental System
Senor Choice
For the sensor part, we select three genes, copA, cueO and marRAB as our candidates of sensor. They are all indirectly or directly regulated by copper ions in E.coli's chromosome. Here we use the promoters of them to link with a GFP reporter ( BBa_E0840) to construct three testing parts(Table1).Then we designed two experiments to compare their sensitivity and specificity individually, for choosing a more suitable sensor.
Relative Fluorescence Intensity
When we test our parts, we found the same fault of those three testing constructs that all the parts can generate the fluorescence without copper. We presumed that it relates to vector's high-copy property(we used the pSB1C3 as our vector).So we adopted Relative Fluorescence Intensity to process our experimental data for showing the relation between copper and three parts. Relative fluorescence intensity=experimental group fluorescence intensity/corresponding control group fluorescence intensity. Control groups indicate that the E.coli contains the same plasmid as corresponding experimental group without treating with copper.
Sensitivity Experiment
In sensitivity experiment, we measured the fluorescence output of three parts in different copper concentration(0.25mg/L 0.5mg/L 1.0mg/L 2.0mg/L 4.0mg/L 8.0mg/L) each 20 min within 3 hours. And we found the intensity of GFP can enhanced even in the low copper concentration under the control of copA promoter. Adversely, two other testing parts ,cueO promoter and marO, are hardly to be induced at the same concentration (Fig 2.1.3).However, after long period of time(8 hours) and in relative high concentration of copper（5mg/L）,these two testing parts could generate much more fluorescence than control group（Fig 2.1.4）.
The experiment shows that copA promoter is more sensitive, specially copper concentration over 1mg/L (Fig 2.1.3-A).Moreover in the higher level of copper, the part displayed higher intensity fluorescence. But other two promoters from sensing copper to making response had a "delay phase"(Fig 2.1.3-B,C and Fig.2),they can function in longer time and higher concentration, which may result from their role of conferring intrinsic copper homeostasis is subordinate to copA.
(A) GFP is under the control of the copA promoter. (B) GFP is under the control of the cueO promoter.(C) GFP is under the control of the marO. The control group and testing groups all cultured at 37 centigrade.
Specificity Experiment
In another experiment for researching the specificity of three promoters, we tested our candidates with six different and the same concentration(4mg/L)metals(Al3+,Li2+,Mn2+,Pb2+,Zn2+,,Cu2+,) each 3 hours. As a result, the copA promoter was failed to elicited by other five metals other than copper(Figure 2.1.5-A).Although the maro can make response to several metals, it seems to be repressed not to be induced(Figure 2.1.5-B). However, cueO promoter was almost inactive for all six metals(Fig 2.1.5-C).
Considering complicated environment usually contains various metals, we set this experiment and reveal that copA promoter is more suitable than other promoters because it is highly selective.
Getting Coding Genes --- gshF
According to Streptococcus thermophilus's gshF gene sequence of Streptococcus thermophilus SIIM 218 published by Genbank, we used gshF primer to amplify the target gene. The result on 1% agarose gel electrophoresis showed that there is clarity band around 2265bp which was predicted. As shown in Figure three, Target fragments were purified and were connected to the amplified plasmid 18T then they were sent to the company to be sequenced. The result shows that homology of Streptococcus thermophilus SIIM 218 gshF is 99%. Sequence results are as follows. However from the first gene of following gene sequence and form the 38th in the sequence of Streptococcus thermophilus SIIM 218 are totally coincide, which is believed that the several mismatch is because the starting signal is too low to be recognized.（Figure 2.1.6）
Besides, we use enzyme cut and enzyme linked to add in front of the in front of the gshF and undergo certain conditions to get plenty of GSH which can effectively combine heavy metal to control the heavy metal pollution figure 2.1.7 and figure 2.1.8.
Facilitating System
Promoter vgb
Pvgb, promoter of the facilitating part to active CRISPERi system, has been installed into plasmid and transform into E.coli. To test the efficiency and conditions can induce this promoter, we incubated the E.coli which have Pvgb and GFP inserted in plasmid under 37℃ and sealed conditions, every two hours since inoculated tested the concentration of oxygen and the florescence intensity of E.coli to further dig up the relationship between the expression of Pvgb and O2 concentration. Some manipulation must be done on our raw data to get the result. Considering the change in E.coli population, we should first divide the fluorescence signal by E.coli concentration. An assumption is made that the change rate of Pvgb expression is influenced by O2 concentration. Because fluorescence signal is an accumulation signal since the experiment is carried out. The change rate of fluorescence is the derivation of fluorescence signal by time.
Since the data points are few and scattered, we have to utilize interpolation to generate more data points between the data points we got from experiment, and here spline interpolation is used.
Then we plot the Change Rate – O2 concentration image and do a quadratic regression. 3σ principle is utilized to remove some abnormal data from our experiment.
After interpolation the data points are intensive enough, so that we use the difference between two interfacing points as the estimation of derivation at that point:
From the chart we can see that with the concentration of dissolved oxygen decreased, the rate of GFP expression increased and reached maximum at last. Except the radioactivehalf-life of GFP, the negative speed under high concentration of dissolved oxygen revealed that there might exist a mechanism to decompose green florescence protein.
CRISPERi System
The design and assembly of sgRNA
In purpose of the contracture of sgRNA, we should choose a specific N20 sequence from the CopA gene and then conjoin N20 with conservative sgRNA sequence.
The chosen of N20 :
ACATCCGCC TGCTCAACATC
The contracture of sgRNA
Strain and material
Plasmid pCB003 from Shanghai Chinese Academy of Sciences. The structure of pCB003 is as shown in the following figure 2.1.12.
1.two designed N20  single stranded fragments are combined into double chain by the way of  annealing with PCR Amplifier
2.To synthesize sgRNA, we use sgRNA F and sgRNA R plasmid pCB003 as primer and as a template for PCR The length of PCR product is approximately 100 bp
Reaction system :     GXL buffer （mg2+plus）     4.0 µl     dNTP mixture               1.6µl     sgRNA F（10µmol）        0.5µl     sgRNA R（10µmol）        0.5µl     pCB003                  1.0µl     GXL polymerase            0.4µl                          ddH2O                     12µl     Reactiom program:     Tm:52 ℃  10 cycle     Tm:62℃   24cycle
The productof step2 PCR product was purified by Extraction of DNA from agarose gel.
3.Step 1 (N20) product and step2 (sgRNA) product as a template and to OLF and sgRNAR as overlap PCR primers are to synthesize N20 and sgRNA fragment
4.Cut the overlap of PCR products with XbaI and SpeI enzyme digestion, enzyme cut over 85 ℃ for 20 min for inactivation
Reaction system :     GXL buffer （mg2+plus）     4.0 µl     dNTP mixture               1.6µl     sgRNA F（10µmol）        0.5µl     sgRNA R（10µmol）        0.5µl     step3 PCR product          1.0µl     GXL polymerase            0.4µl      ddH2O                     12µl     reactiom program:     Tm:49 ℃   5 cycle     Tm:60 ℃   24cycle
5.Cut BBa_J23100 plasmid with SpeI enzyme and then make the product to take off the phosphorylation
6.Combine step4 product with step5 product using T4 ligase
7. Purify the step6 product
8. Use colony PCR to verify the enzyme product
9. Pick a single colony to culture and then use XbaI & SpeI enzyme to cut the mention of plasmid
10. Sequence the final product (BBa_J23100 + N20sgRNA plasmid)
Validation of the result:
To make sure the correctness of sequence, we have verified the final product. The results have shown below:
1. Colony PCR
2. Sequence the final product
Compare the origin sequence from NCBI with the result from sequencing company.Through the verification result ,we completely ensure that our sequence is right. To some extent ,this part could achieve guiding the dcas9 protein to combine with the copA gene on the E. coli chromosome purpose.
[1]Christopher Rensing , Gregor Grass. (2003) Escherichia coli mechanisms of copper homeostasis in a changing environment. FEMS Microbiology Reviews, 27: 197~213
[2] Lei S. Qi, Matthew H. Larson, Luke A. Gilbert, Jennifer A. Doudna, Jonathan Weissman, Adam P. Arkin, and Wendell A. Lim. (2013) Repurposing CRISPR as an RNA-Guided
Retrieved from " http://2014.igem.org/Team:NJAU_China/Wet_Labs "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
