<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_NJAU_China_Wet_Labs"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:NJAU_China/Wet_Labs">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:NJAU_China/Wet_Labs&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:NJAU_China/Wet_Labs&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:NJAU_China/Wet_Labs&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:NJAU_China/Wet_Labs" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:NJAU China/Wet Labs</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><P><TITLE>Home</TITLE></P><DIV id="myheader"><H1 id="my_h1"><A href="#">IGEM</A></H1><DIV id="my_nav"><UL><LI><A href="https://2014.igem.org/Team:NJAU_China">Home</A></LI><LI id="project_li"><A href="#">Project</A><DL id="project"><DD><A href="https://2014.igem.org/Team:NJAU_China/Background">Background</A></DD><DD><A href="https://2014.igem.org/Team:NJAU_China/System Design">System Design</A></DD><DD><A href="https://2014.igem.org/Team:NJAU_China/Safety">Safety</A></DD></DL></LI><LI id="achievement_li"><A href="#">Achievement</A><DL id="achievement"><DD><A href="https://2014.igem.org/Team:NJAU_China/Wet Labs">Wet Lab</A></DD><DD><A href="https://2014.igem.org/Team:NJAU_China/Modeling">Model</A></DD><DD><A href="https://2014.igem.org/Team:NJAU_China/Protocol/Notebooks">Protocol/Notebooks</A></DD><DD><A href="https://2014.igem.org/Team:NJAU_China/Biobricks">Biobricks</A></DD></DL></LI><LI id="prctice_li"><A href="#">Practice</A><DL id="prctice"><DD><A href="https://2014.igem.org/Team:NJAU_China/iGEM and Synthetic Biology Exhibition">Exhibitions and Workshops</A></DD><DD><A href="https://2014.igem.org/Team:NJAU_China/Questionnaire Survey">Questionnaire Survey</A></DD><DD><A href="https://2014.igem.org/Team:NJAU_China/Collaboration">Collaboration</A></DD></DL></LI><LI><A href="https://2014.igem.org/Team:NJAU_China/Team">Team</A></LI><LI id="ackonwledgement_li"><A href="#">Acknowledgement</A><DL id="ackonwledgement"><DD><A href="https://2014.igem.org/Team:NJAU_China/Inside Campus">Inside Campus&amp;Arrtibutions</A></DD><DD><A href="https://2014.igem.org/Team:NJAU_China/Outside Campus">Outside Campus</A></DD></DL></LI></UL></DIV></DIV><DIV id="my_contents"><DIV id="main"><DIV class="p_left2"><UL id="PKU_subsubmenu" class="vertical" onmouseout="listReset();"><LI class="bar">Protocol</LI><LI class="trunk" onmouseover="listTrigger(0);"><A href="#Adjective Experiments">Adjective Experiments</A></LI><LI class="trunk" onmouseover="listTrigger(0);"><A href="#Fundamental System">Fundamental System</A></LI><LI class="trunk" onmouseover="listTrigger(0);"><A href="#Facilitating System">Facilitating System</A></LI></UL></DIV></DIV><DIV id="my_con"><H1><A name="Adjective Experiments">Adjective Experiments</A></H1><P>Besides the experiments to build our system, we also did some adjective experiments to testify the harmless of copper ions to organisms and the efficiency about GSH compound with copper ions.</P><H3>Will GSH detoxify Cu<SUP>2+</SUP> in water or will GSH become the new pollution? </H3><P>Cu<SUP>2+</SUP> can cause great damage to many kinds of aquatic organisms especially alga. It has been reported that Cu<SUP>2+</SUP> under 1ppm can affect the growth of alga. We use alga to measure the harm of copper ions in water, hoping to get a more accurate results.</P><P><I>Chlamydomonas reinhardtii</I>, a common alga which can find in most flesh water, is chosen as the alga's represent. And the trend of relationship among cupric ions, GSH and <I>Chlamydomonas reinhardtii</I> are showed below.</P><P>A.	High level of GSH suppress the alga. However, the dose-effect relationship doesn't look like normal toxin. It must affect the nutrition in medium.TAP medium contains Hutner's trace elements, which must be oxidized for one week. So, GSH as an antioxidant can reduce Hutner's trace elements' effect.</P><P>B.	With no GSH or CuSO<SUB>4</SUB> ,alga grows in the most fast rate.</P><P>C.	CuSO<SUB>4</SUB> as an algicide is highly toxic to algas.But it shows that <I>Chlamydomonas reinhardtii</I> are more tolerant to copper ions than the normal alga.</P><H3>As a result, GSH does have significant detoxification effect!</H3><H1><A name="Fundamental System">Fundamental System</A></H1><H3>Senor Choice</H3><P>For the sensor part, we select three genes, <I>cop</I>A, <I>cue</I>O and <I>mar</I>RAB as our candidates of sensor. They are all indirectly or directly regulated by copper ions in E.coli's chromosome. Here we use the promoters of them to link with a GFP reporter ( BBa_E0840) to construct three testing parts(Table1).Then we designed two experiments to compare their sensitivity and specificity individually, for choosing a more suitable sensor.</P><H3>Relative Fluorescence Intensity</H3><P>When we test our parts, we found the same fault of those three testing constructs that all the parts can generate the fluorescence without copper. We presumed that it relates to vector's high-copy property(we used the pSB1C3 as our vector).So we adopted Relative Fluorescence Intensity to process our experimental data for showing the relation between copper and three parts.
Relative fluorescence intensity=experimental group fluorescence intensity/corresponding control group fluorescence intensity. Control groups indicate that the E.coli contains the same plasmid as corresponding experimental group without treating with copper.</P><H3>Sensitivity Experiment</H3><P>In sensitivity experiment, we measured the fluorescence output of three parts in different copper concentration(0.25mg/L 0.5mg/L 1.0mg/L 2.0mg/L 4.0mg/L 8.0mg/L) each 20 min within 3 hours. And we found the intensity of GFP can enhanced even in the low copper concentration under the control of <I>cop</I>A promoter. Adversely, two other testing parts ,<I>cue</I>O promoter and <I>mar</I>O, are hardly to be induced at the same concentration (Fig 2.1.3).However, after long period of time(8 hours) and in relative high concentration of copper（5mg/L）,these two testing parts could generate much more fluorescence than control group（Fig 2.1.4）.</P><P>The experiment shows that <I>cop</I>A promoter is more sensitive, specially copper concentration over 1mg/L (Fig 2.1.3-A).Moreover in the higher level of copper, the part displayed higher intensity fluorescence. But other two promoters from sensing copper to making response had a &quot;delay phase&quot;(Fig 2.1.3-B,C and Fig.2),they can function in longer time and higher concentration, which may result from their role of conferring intrinsic copper homeostasis is subordinate to <I>cop</I>A.</P><P>(A) GFP is under the control of the <I>cop</I>A promoter. (B) GFP is under the control of the <I>cue</I>O promoter.(C) GFP is under the control of the <I>mar</I>O. The control group and testing groups all cultured at 37 centigrade.</P><H3>Specificity Experiment</H3><P>In another experiment for researching the specificity of three promoters, we tested our candidates with six different and the same concentration(4mg/L)metals(Al<SUP>3+</SUP>,<SUP>Li2+</SUP>,Mn<SUP>2+</SUP>,Pb<SUP>2+</SUP>,Zn<SUP>2+</SUP>,,Cu<SUP>2+</SUP>,) each 3 hours. As a result, the <I>cop</I>A promoter was failed to elicited by other five metals other than copper(Figure 2.1.5-A).Although the maro can make response to several metals, it seems to be repressed not to be induced(Figure 2.1.5-B). However, <I>cue</I>O promoter was almost inactive for all six metals(Fig 2.1.5-C).</P><P>Considering complicated environment usually contains various metals, we set this experiment and reveal that <I>cop</I>A promoter is more suitable than other promoters because it is highly selective.</P><H1><A name="Getting Coding Genes --- &lt;i&gt;gsh&lt;/i&gt;F">Getting Coding Genes --- <I>gsh</I>F</A></H1><P>According to Streptococcus thermophilus's <I>gsh</I>F gene sequence of <I>Streptococcus thermophilus</I> SIIM 218 published by Genbank, we used <I>gsh</I>F primer to amplify the target gene. The result on 1% agarose gel electrophoresis showed that there is clarity band around 2265bp which was predicted. As shown in Figure three, Target fragments were purified and were connected to the amplified plasmid 18T then they were sent to the company to be sequenced. The result shows that homology of <I>Streptococcus thermophilus</I> SIIM 218 <I>gsh</I>F is 99%. Sequence results are as follows. However from the first gene of following gene sequence and form the 38th in the sequence of <I>Streptococcus thermophilus</I> SIIM 218 are totally coincide, which is believed that the several mismatch is because the starting signal is too low to be recognized.（Figure 2.1.6）</P><P>Besides, we use enzyme cut and enzyme linked to add in front of the in front of the <I>gsh</I>F and undergo certain conditions to get plenty of GSH which can effectively combine heavy metal to control the heavy metal pollution figure 2.1.7 and figure 2.1.8. </P><H1><A name="Facilitating System">Facilitating System</A></H1><H3>Promoter <I>vgb</I></H3><P>P<I>vgb</I>, promoter of the facilitating part to active CRISPERi system, has been installed into plasmid and transform into <I>E.coli</I>. To test the efficiency and conditions can induce this promoter, we incubated the <I>E.coli</I> which have P<I>vgb</I> and GFP inserted in plasmid under 37℃ and sealed conditions, every two hours since inoculated tested the concentration of oxygen and the florescence intensity of E.coli to further dig up the relationship between the expression of P<I>vgb</I> and O<SUB>2</SUB> concentration. Some manipulation must be done on our raw data to get the result. Considering the change in <I>E.coli</I> population, we should first divide the fluorescence signal by <I>E.coli</I> concentration. An assumption is made that the change rate of P<I>vgb</I> expression is influenced by O<SUB>2</SUB> concentration. Because fluorescence signal is an accumulation signal since the experiment is carried out. The change rate of fluorescence is the derivation of fluorescence signal by time.</P><P>Since the data points are few and scattered, we have to utilize interpolation to generate more data points between the data points we got from experiment, and here spline interpolation is used. </P><P>Then we plot the Change Rate – O<SUB>2</SUB> concentration image and do a quadratic regression. 3σ principle is utilized to remove some abnormal data from our experiment.</P><P>After interpolation the data points are intensive enough, so that we use the difference between two interfacing points as the estimation of derivation at that point:</P><P>From the chart we can see that with the concentration of dissolved oxygen decreased, the rate of GFP expression increased and reached maximum at last. Except the radioactivehalf-life of GFP, the negative speed under high concentration of dissolved oxygen revealed that there might exist a mechanism to decompose green florescence protein.</P><H1><A name="CRISPERi System">CRISPERi System</A></H1><H3>The design and assembly of sgRNA</H3><P>In purpose of the contracture of sgRNA, we should choose a specific N20 sequence from the <I>Cop</I>A gene and then conjoin N20 with conservative sgRNA sequence.</P><H3>The chosen of N20 :</H3><H3>ACATCCGCC TGCTCAACATC</H3><H3>The contracture of sgRNA</H3><H4>Strain and material</H4><P>Plasmid pCB003 from Shanghai Chinese Academy of Sciences. The structure of pCB003 is as shown in the following figure 2.1.12.</P><P>1.two designed N20  single stranded fragments are combined into double chain by the way of  annealing with PCR Amplifier</P><P>2.To synthesize sgRNA, we use sgRNA F and sgRNA R plasmid pCB003 as primer and as a template for PCR
The length of PCR product is approximately 100 bp</P><PRE>
    Reaction system :
    GXL buffer （mg2+plus）     4.0 µl
    dNTP mixture               1.6µl
    sgRNA F（10µmol）        0.5µl
    sgRNA R（10µmol）        0.5µl
    pCB003                  1.0µl
    GXL polymerase            0.4µl                     
    ddH2O                     12µl
    Reactiom program:
    Tm:52 ℃  10 cycle
    Tm:62℃   24cycle
    </PRE><P>The productof step2 PCR product was purified by Extraction of DNA from agarose gel.</P><P>3.Step 1 (N20) product and step2 (sgRNA) product as a template and to OLF and sgRNAR as overlap PCR primers are to synthesize N20 and sgRNA fragment</P><P> 4.Cut the overlap of PCR products with XbaI and SpeI enzyme digestion, enzyme cut over 85 ℃ for 20 min for inactivation</P><PRE>Reaction system :
    GXL buffer （mg2+plus）     4.0 µl
    dNTP mixture               1.6µl
    sgRNA F（10µmol）        0.5µl
    sgRNA R（10µmol）        0.5µl
    step3 PCR product          1.0µl
    GXL polymerase            0.4µl 
    ddH2O                     12µl
    reactiom program:
    Tm:49 ℃   5 cycle
    Tm:60 ℃   24cycle
    </PRE><P>5.Cut BBa_J23100 plasmid with SpeI enzyme and then make the product to take off the phosphorylation    </P><P>6.Combine step4 product with step5 product using T4 ligase</P><P>7. Purify the step6 product </P><P>8. Use colony PCR to verify the enzyme product</P><P>9. Pick a single colony to culture and then use XbaI &amp; SpeI enzyme to cut the mention of plasmid </P><P>10. Sequence the final product (BBa_J23100 + N20sgRNA plasmid)</P><P>Validation of the result:	</P><P>To make sure the correctness of sequence, we have verified the final product. The results have shown below:</P><P>1.	Colony PCR</P><P>2.	Sequence the final product</P><P>Compare the origin sequence from NCBI with the result from sequencing company.Through the verification result ,we completely ensure that our sequence is right. To some extent ,this part could achieve guiding the dcas9 protein to combine with the copA gene on the E. coli chromosome purpose.
    </P><P>[1]Christopher Rensing , Gregor Grass. (2003) Escherichia coli mechanisms of copper homeostasis in a changing environment. FEMS Microbiology Reviews, 27: 197~213</P><P>[2] Lei S. Qi, Matthew H. Larson, Luke A. Gilbert, Jennifer A. Doudna, Jonathan Weissman, Adam P. Arkin, and Wendell A. Lim. (2013) Repurposing CRISPR as an RNA-Guided</P></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:NJAU_China/Wet_Labs">http://2014.igem.org/Team:NJAU_China/Wet_Labs</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:NJAU_China/Wet_Labs" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:NJAU_China/Wet_Labs" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:NJAU_China/Wet_Labs&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:NJAU_China/Wet_Labs&amp;oldid=368360" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>