<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_NJAU_China_System_Design"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:NJAU_China/System_Design">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:NJAU_China/System_Design&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:NJAU_China/System_Design&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:NJAU_China/System_Design&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:NJAU_China/System_Design" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:NJAU China/System Design</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><P><TITLE>Home</TITLE></P><DIV id="myheader"><H1 id="my_h1"><A href="#">IGEM</A></H1><DIV id="my_nav"><UL><LI><A href="https://2014.igem.org/Team:NJAU_China">Home</A></LI><LI id="project_li"><A href="#">Project</A><DL id="project"><DD><A href="https://2014.igem.org/Team:NJAU_China/Background">Background</A></DD><DD><A href="https://2014.igem.org/Team:NJAU_China/System Design">System Design</A></DD><DD><A href="https://2014.igem.org/Team:NJAU_China/Safety">Safety</A></DD></DL></LI><LI id="achievement_li"><A href="#">Achievement</A><DL id="achievement"><DD><A href="https://2014.igem.org/Team:NJAU_China/Wet Labs">Wet Lab</A></DD><DD><A href="https://2014.igem.org/Team:NJAU_China/Modeling">Model</A></DD><DD><A href="https://2014.igem.org/Team:NJAU_China/Protocol/Notebooks">Protocol/Notebooks</A></DD><DD><A href="https://2014.igem.org/Team:NJAU_China/Biobricks">Biobricks</A></DD></DL></LI><LI id="prctice_li"><A href="#">Practice</A><DL id="prctice"><DD><A href="https://2014.igem.org/Team:NJAU_China/iGEM and Synthetic Biology Exhibition">Exhibitions and Workshops</A></DD><DD><A href="https://2014.igem.org/Team:NJAU_China/Questionnaire Survey">Questionnaire Survey</A></DD><DD><A href="https://2014.igem.org/Team:NJAU_China/Collaboration">Collaboration</A></DD></DL></LI><LI><A href="https://2014.igem.org/Team:NJAU_China/Team">Team</A></LI><LI id="ackonwledgement_li"><A href="#">Acknowledgement</A><DL id="ackonwledgement"><DD><A href="https://2014.igem.org/Team:NJAU_China/Inside Campus">Inside Campus</A></DD><DD><A href="https://2014.igem.org/Team:NJAU_China/Inside Campus">Inside Campus&amp;Arrtibutions</A></DD></DL></LI></UL></DIV></DIV><DIV id="my_contents"><DIV id="main"><DIV class="p_left2"><UL id="PKU_subsubmenu" class="vertical" onmouseout="listReset();"><LI class="bar">System Design</LI><LI class="trunk" onmouseover="listTrigger(0);"><A href="#Fundamental Part">Fundamental Part</A></LI><LI class="trunk" onmouseover="listTrigger(0);"><A href="#Facilitating Part">Facilitating Part</A></LI></UL></DIV></DIV><DIV id="my_con"><P> To cope with the copper solution, we hope to build a system which can be activated automatically when pollution occurred, efficient and intelligent.</P><P>We design two basic parts to form a whole system, fundamental part and facilitating part :</P><H1><A name="Fundamental Part">1.	Fundamental Part</A></H1><P>There are sensor and <I>gsh</I>F in this part.</P><P>For engineering an bio-machine to decrease the copper toxic automatically, we need a sensitive copper sensor to make CAM boy do rapid response once detecting the concentration of copper is excessive.</P><P><I>marO</I> and mar locus</P><P>E.coli can tolerate or resist several extracellular stress through conferring intrinsic mechanism such as multiple antibiotic resistance (mar) locus in chromosome<SUP>[1]</SUP>. Two transcriptional units, <I>mar</I>C and <I>mar</I>RAB, dividedly flanked operator <I>mar</I>O in the mar locus<SUP> [2] </SUP>.The <I>mar</I>RAB has been uncovered to be involved in antibiotic resistance and various cellular stress response. Meanwhile the MarR ,which is expressed by <I>mar</I>R and binds to <I>mar</I>O, can negatively regulate <I>mar</I>RAB by repressing its promoter<SUP> [3] </SUP> (Figure 1.2.3).There are two sites targeted by MarO in <I>mar</I>O has been deduced through DNA footprinting<SUP> [4] </SUP>.Recently the copper has been demonstrated as a signal molecule to directly trigger the dissociation of MarR from DNA by redox reaction inside E.coli ,and in this reaction the copper(II) oxidize the MarR to become a dimer linked by disulfide bonds <SUP> [5] </SUP> (Figure 1.2.4).</P><P>The cue system</P><P>Moreover the E.coli has astonishing tolerance to copper in relation to several dependent or independent systems mediating the homeostasis. The <I>cop</I>A and <I>cue</I>O are both in the same <I>cue</I> (Cu efflux) system. They are activated by a copper-responsive protein CueR (Cu efflux regulator) by targeting at their promoters <SUP> [6] </SUP>The CopA is a ATPase which is responsible to efflux of copper(I) in cells ,and it seems to become a critical method of mediating the homeostasis in high concentration of copper<SUP> [7,8] </SUP>. Meanwhile the CueO is another important factor conferring the intracellular copper tolerance of E.coli, and it is a multi-copper oxidase protecting periplasm from damage of copper<SUP> [9] </SUP>.</P><P>Glutathione exist in two forms in bacteria, the reduced Glutathione (known as GSH) and oxidized Glutathione (known as GSSG) and the reduced Glutathione is the majority group and play the significant role in organism . The glutathione often refers to reduced glutathione.</P><P>GSH is a natural bioactive peptide with an important physiological functions, form by l-glutamic acid, l-cysteine and glycine linked by peptide bond. Its molecular formula is C10H17O6SN3 and its chemical structure is shown in figure 1.2.5. Glutathione cysteine side chain has a active sulfhydryl group, which is the basis of many important physiological functions structure, combining with free radicals in human body. Free radicals can be directly reduced to metabolic acids easily and accelerate the excretion of free radicals, thereby reduce free radical damage to vital organs.  It is formed a antioxidant network, with a variety of antioxidant enzymes, reacting with various reactive oxygen species (ROS) to protect hemoglobin and metals sulfur proteins form oxidation<SUP> [10] </SUP>.</P><P>Metals, especially the characteristics of heavy metal with a strong pro-s and sulfhydryl (-SH) is an important functional group of GSH molecule structure, it can combine metal ions (M2 +) to form nontoxic or less toxic complexes to detoxificate.    Also GSH, under heavy metal ions stress-induced situation, produce free radicals (-OH) for redox reaction, which deduce oxidative damage. A major physiological function of GSH is protecting sulfhydryl contained in membrane protein and enzymes from oxidation <SUP> [11] </SUP>. Therefore, the GSH-metal interactions are critical part of detoxification.  Using the 1.0 m g/l copper ion solution on suspensions of Escherichia coli for 4 hours, the result of the average killing rate is 61.90%. </P><P>Heavy metal pollution has become an urgent issue. Using the secreting GSH secreted by engineering bacteria to remediate the metals heavy metals, this approach provides direction to reduce heavy metal pollution. After the induction of certain conditions, glutathione synthesis gene (<I>gsh</I>F) which is transformed into the bacteria can secret GSH and GSH can combine metal ions reducing the toxic effects of heavy metals on Escherichia coli leading to the reduce the accumulation level of heavy metals（Fig 1.2.6）.</P><P>In vivo, GSH synthesis is a two-step sequential enzyme reaction. Two key enzyme are γ-glutamyl cysteine synthetase (γ-GCS) and Glutathione synthetase (GS) and gamma-glutamyl cysteine synthetase is the rate-limiting enzyme. In recent years, Streptococcus agalactiae was reported by Janowiak can synthesis GSH and code for γ-glutamyl cysteine synthetase and Glutathione synthetase activity bifunctional enzyme genes which named GshF. This enzyme is γ--GCS-GS. In Streptococcus agalactiae, GshF activity is not confined by the negative feedback of GSH and there is no accumulation of intermediate products. </P><H1><A name="Facilitating Part">2. Facilitating Part</A></H1><P>In this part, promoter vgb and CRISPERi are included:</P><P>As a matter of fact, during the process of copper compound with glutathione, there are other reactions also exists. Although the copper-glutathione complex, the product of the chemical reaction, has been reported to be very stable in aqueous solutions, the Cu(I)–glutathione complex continually reacts with molecular oxygen to generate superoxide anions. Here's the mechanism of these reactions:</P><P>It has been reported that the Cu(I)–glutathione complex, present in a pre-incubated (3:1) GSH plus Cu<SUP>2+</SUP> mixture, reacts continually with molecular oxygen to generate superoxide anions<SUP> [12] </SUP>.</P><P>Promoter <I>vgb</I> is oxygen-dependent promoter which can be maximally induced under microaerobic conditions. Webster (1988) and Bailey (1988) isolated the DNA fragment including the <I>vgb</I> gene, which encodes VHb protein and its native promoter respectively from <I>Vitreoscilla</I><SUP> [13] </SUP>. It is found in freshwater sediments and cow dung, where oxygen availability is limited. In various organisms, VHb, coding by <I>vgb</I> gene, was shown to improve microaerobic cell growth and enhance oxygen dependent product formation in various organisms.</P><P>From now on, promoter <I>vgb</I> has been characterized in E. coli and has been shown to be functional in heterologous hosts. Transcriptional activity of P<I>vgb</I> in E. coli is positively modulated by CRP (cAMP receptor protein) and FNR (Fumarate nitrate reduction regulator) <SUP> [14] </SUP>. Using P<I>vgb</I> to activate CRISPERi system to repress expression of CopA, a Cu(I)-translocating efflux pump, can regulating autonomously and make sure the efficiency of GSH dealing with the copper pollution.</P><P>Targeted gene regulation on a genome-wide scale is a powerful strategy for interrogating, perturbing and engineering cellular systems. Here, we develop a method for controlling gene expression based on Cas9, an RNA-guided DNA endonuclease from a type II CRISPR system. We show that a catalytically dead Cas9 lacking endonuclease activity, when coexpressed with a guide RNA, generates a DNA recognition complex that can specifically interfere with transcriptional elongation, RNA polymerase binding, or transcription factor binding. This system, which we call CRISPR interference (CRISPRi), can efficiently repress expression of targeted genes in Escherichia coli, with no detectable off-target effects. This RNA-guided DNA recognition platform provides a simple approach for selectively perturbing gene expression on a genome-wide scale<SUP> [15] </SUP></P><P>The <I>cop</I>A gene product, a putative copper-translocating P-type ATPase, has been shown to be involved in copper resistance in Escherichia coli. Some evidences indicate that CopA is a Cu(I)-translocating efflux pump that is similar to the copper pumps<SUP> [16] </SUP>.</P><P>The CRISPRi system consists of an inducible Cas9 protein and a designed sgRNA chimera. The dCas9 contains mutations of the RuvC1 and HNH nuclease domains. The sgRNA chimera contains three functional domains, as described below.</P><P>The minimal interference system consists of a single protein and a designed sgRNA chimera. The sgRNA chimera consists of three domains (boxed region): a 20 nt complementary region for specific DNA binding, a 42 nt hairpin for Cas9 binding (Cas9 handle), and a 40 nt transcription terminator derived from <I>S. pyogenes</I>. The wild-type Cas9 protein contains the nuclease activity. The dCas9 protein is defective in nuclease activity.</P><P>The wild-type Cas9 protein binds to the sgRNA and forms a protein-RNA complex. The complex binds to specific DNA targets by Watson-Crick base pairing between the sgRNA and the DNA target. In the case of wild-type Cas9, the DNA will be cleaved due to the nuclease activity of the Cas9 protein. We hypothesize that the dCas9 is still able to form a complex with the sgRNA and bind to specific DNA target. When the targeting occurs on the protein-coding region, it could block RNA polymerase and transcript elongation <SUP> [17] </SUP>.</P><P>If the blueprint is broken or deficient ,we cannot make a whole model .similarly, the mRNA elongation process is interrupted ,the copper-translocating P-type ATPase won't be compounded. Consequently, the possibility of cuprous ions in cytoplasm shipped out will reduce. Because of the induced quality of cuprous ions , the copper ions are accumulated in E.coli.</P><P>References:</P><P> [1] George, A. M., &amp; Levy, S. B. (1983). Gene in the major cotransduction gap of the Escherichia coli K-12 linkage map required for the expression of chromosomal resistance to tetracycline and other antibiotics. Journal of bacteriology, 155(2), 541-548.</P><P>[2] Cohen, S. P., Hächler, H., &amp; Levy, S. B. (1993). Genetic and functional analysis of the multiple antibiotic resistance (mar) locus in Escherichia coli.Journal of bacteriology, 175(5), 1484-1492.</P><P>[3] Alekshun, M. N., &amp; Levy, S. B. (1997). Regulation of chromosomally mediated multiple antibiotic resistance: the mar regulon. Antimicrobial agents and chemotherapy, 41(10), 2067.</P><P>[4] Martin, R. G., &amp; Rosner, J. L. (1995). Binding of purified multiple antibiotic-resistance repressor protein (MarR) to mar operator sequences. Proceedings of the National Academy of Sciences, 92(12), 5456-5460.</P><P>[5] Hao, Z., Lou, H., Zhu, R., Zhu, J., Zhang, D., Zhao, B. S., ... &amp; Chen, P. R. (2013). The multiple antibiotic resistance regulator MarR is a copper sensor in Escherichia coli. Nature chemical biology.</P><P>[6] Outten, F. W., Outten, C. E., Hale, J., &amp; O'Halloran, T. V. (2000). Transcriptional Activation of an Escherichia coli Copper Efflux Regulon by the Chromosomal MerR Homologue, CueR. Journal of Biological Chemistry,275(40), 31024-31029.</P><P>[7] Rensing, C., Fan, B., Sharma, R., Mitra, B., &amp; Rosen, B. P. (2000). CopA: an Escherichia coli Cu (I)-translocating P-type ATPase. Proceedings of the National Academy of Sciences, 97(2), 652-656.</P><P>[8] Outten, F. W., Huffman, D. L., Hale, J. A., &amp; O'Halloran, T. V. (2001). The Independent cue and cusSystems Confer Copper Tolerance during Aerobic and Anaerobic Growth in Escherichia coli. Journal of Biological Chemistry,276(33), 30670-30677.</P><P>[9] Grass, G., &amp; Rensing, C. (2001). CueO Is a Multi-copper Oxidase That Confers Copper Tolerance in Escherichia coli. Biochemical and biophysical research communications, 286(5), 902-908.</P><P>[10] Bode AM, Liang HQ, Green EH, et al. Ascorbic acid recycling in Nb2 lymphoma cell: implications for tumor progression [J].Free. Radic. Biol. Med. 1999, 26:136-47.</P><P>[11] P.B. Tchounwou, C. Newsome , J. Williams , etal, Copper-induced cytotoxicity and transcriptional activationof stress genes in human liver carcinoma cells [J]. Met. Ions. Biol.Med., 2008, 10:285–290.）</P><P>[12] Speisky H, Gómez M, Carrasco-Pozo C, et al. Cu (I)–Glutathione complex: A potential source of superoxide radicals generation[J]. Bioorganic &amp; medicinal chemistry, 2008, 16(13): 6568-6574.</P><P>[13] Webster, D. A. (1988). Structure and function of bacterial hemoglobin and related proteins.
Adv. Inorg. Biochem. 7, 245–265.</P><P>[14] Khosla, C., and Bailey, J. E. (1989). Characterization of the oxygen-dependent promoter of
the Vitreoscilla hemoglobin gene in Escherichia coli. J. Bacteriol. 171, 5995–6004.</P><P>[15] Lei S. Qi, Matthew H. Larson, Luke A. Gilbert, Jennifer A. Doudna, Jonathan S. Weissman, Adam P. Arkin, and Wendell A. Lim. (2013). Repurposing CRISPR as an RNA-Guided Platform for Sequence-Specific Control of Gene Expression. Cell 152, 1173-1183.</P><P>[16] Martin Jinek, Krzysztof Chylinski, Ines Fonfara, Michael Hauer, Jennifer A. Doudna, Emmanuelle Charpentier. (2012). A Programmable Dual-RNA–Guided DNA Endonuclease in Adaptive Bacterial Immunity. Science 337, 816-820.</P><P>[17] Christopher Rensing, Bin Fan, Rakesh Sharma, Bharati Mitra, Barry P. Rosen.(2000). CopA: An Escherichia coli Cu(I)-translocating P-type ATPase. PNAS 97, 652-656.</P></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:NJAU_China/System_Design">http://2014.igem.org/Team:NJAU_China/System_Design</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:NJAU_China/System_Design" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:NJAU_China/System_Design" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:NJAU_China/System_Design&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:NJAU_China/System_Design&amp;oldid=365895" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>