<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_SUSTech_Shenzhen_Model2 skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:SUSTech Shenzhen/Model2</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Team SUSTC-Shenzhen</TITLE></P><DIV class="container"><NAV class="navbar navbar-default navbar-fixed-top" role="navigation" id="index-nav-main"><DIV class="container"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav"><LI><A href="https://2017.igem.org/Team:SUSTech_Shenzhen"><SPAN>Home</SPAN></A></LI><LI class=" dropdown"><A href="#" data-toggle="dropdown"><SPAN>View</SPAN></A></LI><LI><A href="https://2017.igem.org/Team:SUSTech_Shenzhen/Description">Description</A></LI><LI><A href="https://2017.igem.org/Team:SUSTech_Shenzhen/Safety">Safety</A></LI></UL><LI class="dropdown"><A href="#" data-toggle="dropdown"><SPAN>Project</SPAN></A></LI><LI><A href="https://2017.igem.org/Team:SUSTech_Shenzhen/Design">Design</A></LI><LI><A href="https://2017.igem.org/Team:SUSTech_Shenzhen/Demonstrate">Demonstrate</A></LI><LI><A href="https://2017.igem.org/Team:SUSTech_Shenzhen/Results">Results</A></LI><LI><A href="https://2017.igem.org/Team:SUSTech_Shenzhen/Notebook">Notebook</A></LI><LI><A href="https://2017.igem.org/Team:SUSTech_Shenzhen/Experiments">Experiments</A></LI><LI class="dropdown"><A href="#" data-toggle="dropdown"><SPAN>Model</SPAN></A></LI><LI><A href="https://2017.igem.org/Team:SUSTech_Shenzhen/Model">Model</A></LI><LI><A href="https://2017.igem.org/Team:SUSTech_Shenzhen/Chemical Diffusion Model">Chemical Diffusion</A></LI><LI><A href="https://2017.igem.org/Team:SUSTech_Shenzhen/Worms_Locomotion Model">Worms Locomotion</A></LI><LI><A href="https://2017.igem.org/Team:SUSTech_Shenzhen/Genetic Probability Model">Genetic Probability</A></LI><LI><A href="https://2017.igem.org/Team:SUSTech_Shenzhen/Neuron Network Model">Neuron Network</A></LI><LI class="dropdown"><A href="#" data-toggle="dropdown"><SPAN>Hardware</SPAN></A></LI><LI><A href="https://2017.igem.org/Team:SUSTech_Shenzhen/Hardware">Overview</A></LI><LI><A href="https://2017.igem.org/Team:SUSTech_Shenzhen/Hardware/Light">Light Modulator</A></LI><LI><A href="https://2017.igem.org/Team:SUSTech_Shenzhen/Hardware/Microfluidics">Microfluidics</A></LI><LI class="dropdown"><A href="#" data-toggle="dropdown"><SPAN>Contribution</SPAN></A></LI><LI><A href="https://2017.igem.org/Team:SUSTech_Shenzhen/Parts">Parts</A></LI><LI><A href="https://2017.igem.org/Team:SUSTech_Shenzhen/Contribution">Characterization</A></LI><LI><A href="https://2017.igem.org/Team:SUSTech_Shenzhen/Improve">Improvement</A></LI><LI><A href="https://2017.igem.org/Team:SUSTech_Shenzhen/InterLab">InterLab</A></LI><LI class="dropdown"><A href="#" data-toggle="dropdown"><SPAN>Human Practice</SPAN></A></LI><LI><A href="https://2017.igem.org/Team:SUSTech_Shenzhen/Human_Practices">Overview</A></LI><LI><A href="https://2017.igem.org/Team:SUSTech_Shenzhen/HP/Silver">HP Silver</A></LI><LI><A href="https://2017.igem.org/Team:SUSTech_Shenzhen/HP/Gold_Integrated">Gold_Integrated</A></LI><LI><A href="https://2017.igem.org/Team:SUSTech_Shenzhen/Collaborations">Collaborations</A></LI><LI class=" dropdown"><A href="#" data-toggle="dropdown"><SPAN>Team</SPAN></A></LI><LI><A href="https://2017.igem.org/Team:SUSTech_Shenzhen/Team">Members</A></LI><LI><A href="https://2017.igem.org/Team:SUSTech_Shenzhen/Attributions">Attribution</A></LI></DIV></DIV></NAV></DIV><DIV class="bs-docs-header"><DIV class="container"><DIV class="row"><DIV class="col-xs-10"><H1>

Genetic Probability

        </H1><P>

Model

        </P></DIV></DIV></DIV></DIV><DIV id="main-content" class="container"><DIV class="col-md-3" id="document-sidebar"><DIV id="toc" class="toc"><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#The_successful_rate_of_miniMos_injection"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">The successful rate of miniMos injection</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#The_stable_inheritance_rate"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">The stable inheritance rate</SPAN></A></LI><LI class="toclevel-1 tocsection-3"><A href="#The_ideal_result"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">The ideal result</SPAN></A></LI><LI class="toclevel-1 tocsection-4"><A href="#The_effect_of_crossing_over"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">The effect of crossing over</SPAN></A></LI><LI class="toclevel-1 tocsection-5"><A href="#The_fitness_of_our_model_with_our_real_data"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">The fitness of our model with our real data</SPAN></A></LI><LI class="toclevel-1 tocsection-6"><A href="#References"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">References</SPAN></A></LI></UL></DIV></DIV><DIV class="col-md-9" id="document-body" style=""><H2>Genetic probability model</H2><P>In our project, we aim to get <I>C. elegans</I> strains with the preference for blue lights and the aversion for red lights by miniMos injection. Therefore, we build a miniMos injection model in order to simulate and estimate numbers of injected worms that can expressed two phenotypes. 
</P><DIV class="row"><DIV class="col-md-8 col-md-offset-2"><DIV class="thumbnail"><DIV class="caption"><B>Fig 2.1  The overview of miniMos Transfection Model</B></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-8 col-md-offset-2"><DIV class="thumbnail"><DIV class="caption"><B>Fig 2.12  The overview of microinjection</B></DIV></DIV></DIV></DIV><H2><SPAN class="mw-headline" id="The_successful_rate_of_miniMos_injection">The successful rate of miniMos injection</SPAN></H2><P>First we need to consider the successful rate of miniMos system. The microinjection needs accurate operations to inject plasmids into <I>C. elegans</I> gonads, which allows plasmids to enter the embryos and maintain the phenotypes. Therefore, accurate manual operations can lead to a promising result. According to the miniMos protocol, the probability of correct injection is approximately 60% excluding manual errors, and about 10% of F1 can be rescued, meaning that the plasmid expression rate(transposon transfers target gene into chromosomes) is about 10%.<SUP id="cite_ref-1" class="reference"><A href="#cite_note-1">[1]</A></SUP>
So the successful rate is:
</P><P>accurate rate of manual operation * plasmid expression rate = 60% * 10% = 6%.
</P><DIV class="row"><DIV class="col-md-10 col-md-offset-1"><DIV class="thumbnail"><DIV class="caption"><B>Fig 2.2. The MiniMos</B></DIV></DIV></DIV></DIV><H2><SPAN class="mw-headline" id="The_stable_inheritance_rate">The stable inheritance rate</SPAN></H2><P>After got heterozygote F1 via miniMos injection, our next step is to get stable expression worms in F2 generation. During the F2 generation, based on the Mendel's laws, if two genes are located on different chromosomes, their alleles will segregate independently during the formation of gametes. And Mendel calculated the probability of getting stable expression F2, the ideal rate is <SPAN>\frac{1}{16}</SPAN>.
Therefore, in our experiment, the prospecting result of getting target strains is also <SPAN>\frac{1}{16}</SPAN>. 
</P><H2><SPAN class="mw-headline" id="The_ideal_result">The ideal result</SPAN></H2><P>Therefore, the overall probability of getting the target strain is:
(accurate rate of manual operation ) * (plasmid expression rate) * ( F2 stable expression rate ) * 100% = 60% * 10% * <SPAN>\frac{1}{16}</SPAN>*100% = 0.004.
</P><DIV class="row"><DIV class="col-md-8 col-md-offset-2"><DIV class="thumbnail"><DIV class="caption"><B>Fig 2.1  The possibility of target strains by Mendel's laws</B></DIV></DIV></DIV></DIV><H2><SPAN class="mw-headline" id="The_effect_of_crossing_over">The effect of crossing over</SPAN></H2><P>However, this probability does not consider the chance of crossing over. Crossing over is the exchange of genetic materials between homologous chromosomes and results in recombinant chromosomes during sexual reproduction.<SUP id="cite_ref-2" class="reference"><A href="#cite_note-2">[2]</A></SUP> Therefore, crossing over may affect the rate of stable inheritance by exchanging genetic materials. According to the formula, the rate of crossing over is:
(numbers of recombinant genotype offspring)/(numbers of total offspring)*100%
</P><P>Therefore, the possible rate of crossing over in our experiment is: <SPAN>\frac{10}{354}</SPAN>=<SPAN>\frac{5}{177}</SPAN></P><P>And the final rate isÂ : 0.004*(1-<SPAN>\frac{5}{177}</SPAN>)=<SPAN>\frac{86}{22125}</SPAN>=0.3%
</P><P><SPAN class="katex-block">\0.004\times ( 1-frac{5}{177} )=\frac{86}{22125}=0.3 \times</SPAN></P><H2><SPAN class="mw-headline" id="The_fitness_of_our_model_with_our_real_data">The fitness of our model with our real data</SPAN></H2><P>Based on our model, we calculate the probability of getting the genetic stable worms in order to precisely implement our experiments. More importantly, our modeling reliability has been tested and confirmed by comparing the calculated result with raw results from the experiments.
1)	The rate of miniMos injection is about 6%, that is six worms with correct insertions within 100 worms. And in reality, we actually obey this probability. In our project, we injected about 500 worms in total, and finally got about 25 worms with expected insertions. 
2)	The probability of getting stable inheritance F2 is <SPAN>\frac{1}{16}</SPAN> in theory, and we found 2 plates of worms showed both GFP and RFP at the site of neurons among 40 F2. 
Overall, these results showed that our model is a good guidance to explain our results, it is feasible.
</P><H1><SPAN class="mw-headline" id="References">References</SPAN></H1><OL class="references"><LI id="cite_note-1"><SPAN class="mw-cite-backlink"><A href="#cite_ref-1">â</A></SPAN><SPAN class="reference-text"><A rel="nofollow" class="external free" href="http://www.wormbuilder.org/minimos/commentsfaq/">http://www.wormbuilder.org/minimos/commentsfaq/</A></SPAN></LI><LI id="cite_note-2"><SPAN class="mw-cite-backlink"><A href="#cite_ref-2">â</A></SPAN><SPAN class="reference-text">Griffiths, AJF; Gelbart, WM; Miller, JH; et al. (1999). &quot;Modern Genetic Analysis: Mitotic Crossing-Over&quot;. New York: W. H. Freeman.</SPAN></LI></OL></DIV></DIV><FOOTER class="footer bs-docs-footer col-md-12" role="contentinfo"><DIV class="container"><P>Made by  from the elegans.Inc in SUSTech_Shenzhen.</P><P>Licensed under <A href="http://creativecommons.org/licenses/by/4.0/">CC BY 4.0</A>.</P><UL class="bs-docs-footer-links muted"><LI>Â©2017 SUSTech_Shenzhen</LI></UL></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>