<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Stockholm_Description skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Stockholm/Description</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="page-wrapper"><CENTER><DIV id="header-wrapper"><DIV id="header" class="container"><NAV id="nav"><DIV class="title noselect">iGEM STOCKHOLM</DIV><UL><LI><A class="icon-home" href="https://2015.igem.org/Team:Stockholm"><SPAN>Home</SPAN></A></LI><LI><A class="icon-users" href="https://2015.igem.org/Team:Stockholm/Team"><SPAN>Team</SPAN></A></LI><LI><A class="icon-lab" href="https://2015.igem.org/Team:Stockholm/Description"><SPAN>Project</SPAN></A></LI><LI><A class="icon-clipboard" href="https://2015.igem.org/Team:Stockholm/Results"><SPAN>Results</SPAN></A></LI><LI><A class="icon-bubbles2" href="https://2015.igem.org/Team:Stockholm/Practices"><SPAN>Practices</SPAN></A></LI><LI><A class="icon-earth" href="https://2015.igem.org/Team:Stockholm/Collaborations"><SPAN>Collaborations</SPAN></A></LI><LI><A class="icon-gift" href="https://2015.igem.org/Team:Stockholm/Attributions"><SPAN>Attributions</SPAN></A></LI></UL></NAV></DIV></DIV></CENTER></DIV><DIV class="container-fluid content-container"><DIV id="sticky"><UL class="nav nav-tabs pillnav"><LI role="presentation" class="active"><A href="#"><B> Project</B></A></LI><LI role="presentation" class="clickable anchor_link"><A href="#abstract">Abstract</A></LI><LI role="presentation" class="clickable anchor_link"><A href="#recognition">Recognition</A></LI><LI role="presentation" class="clickable anchor_link"><A href="#signaling">Signaling</A></LI><LI role="presentation" class="clickable anchor_link"><A href="#readout">Read-out</A></LI><LI role="presentation" class="clickable anchor_link"><A href="#modeling">Modeling</A></LI></UL></DIV><H2>Project Abstract </H2><P>Early disease diagnosis still represents one of the most crucial issues in modern medicine. Despite great advances in high-throughput technologies and the discovery of several hundred new disease biomarkers, we are currently still unable to detect diseases specifically and sensitively when they are most vulnerable, at the beginning. We want to develop a transferable, biological detection system for different diseases using HER-2, a breast cancer biomarker, for a proof-of-prinicple. Therefore, we created a novel class of chimeric receptors based on the bacterial osmoregulator EnvZ. We substituted a part of the periplasmic domain of EnvZ with an exchangable, but highly-specific protein binder, an Affibody molecule. Upon antigen binding, the receptor would trigger the OmpR signaling cascade leading to the production of quorum sensing molecules, which activate transcription of a fluorescent signal in a separated read-out strain. Taken together, this system could be a transferable method for biomarker detection. 
</P><H2>Biomarker recognition </H2><P>Presence of certain proteins in blood or urine indicates disease . Most of these proteins are too large to pass through  the membrane of bacteria without active transport. Thus, a bacterial assay to detect protein biomarkers must transport them into the cell or detect them with a receptor in the membrane. In this project, we wanted to construct a chimeric receptor for biomarker detection.</P><H4>Background to the idea</H4><P>An ideal system for detection of protein biomarkers is specific, sensitive, transferable and cheap. This means that an ideal receptor should bind to the target biomarker with high specificity and affinity. To make the system transferable to different biomarkers, it should be possible to modify the receptor to change its affinity. Furthermore, binding should trigger an intracellular cascade that is suitable for a read-out system.</P><P>With these aims in mind, we were inspired by Chimeric antigen receptors (CARs), a proposed therapy for cancer. In their simplest form, CARs are fusions of single-chain variable fragments from monoclonal antibodies and the transmembrane and cytoplasmic parts of CD3 receptors. Treatment with CARs modifies T-cells from the patient to express receptors which are specific to a certain biomarker. These T-cells are then reintroduced into the patient to recognize and kill cancer cells.. If a suitable scaffold receptor exists in bacteria it may be possible to develop a family of Bacterial antigen receptors (BARs). BARs would not be a treatment but a detection system for biomarkers in blood or urine samples.</P><P>In prokaryotes, two-component regulatory systems are a common mechanism for signal transduction through the membrane. Most often, a membrane-bound histidine kinase activates in response to changes in the environment. This activates response regulators by phosphorylation. One such system in <I>Escherichia coli</I> is the receptor EnvZ and its response regulator OmpR. The EnvZ/OmpR system regulates different expression of the outer membrane porin proteins OmpF and OmpC.</P><H4>EnvZ as scaffold for chimeric receptors</H4><P>The function of the periplasmic region of EnvZ is poorly understood and its ligand is not known. However, the region responsible for signal transduction has been characterized and is located between the transmembrane helix and the histidine kinase domain. This region, known as the HAMP-domain, is present in many two-component regulatory system receptors.. EnvZ stood out as an interesting candidate for BAR since it has been used for chimeric receptors before. Most successful examples are chimeras of receptors that both have HAMP domains, but in at least one case this was not required for signal transduction. In 2005, Christopher A. Voigt et al. made a light sensitive EnvZ fusion with the photoreceptor cph1 which does not contain a HAMP domain, proving that such chimeras are possible.</P><P>A common strategy for chimeric receptors is to characterize many possible constructs.  While it is difficult to find a functioning construct, recent research by Roger Draheim et al. has shown that receptors with the HAMP-domain can be tuned by moving aromatic residues next to the HAMP domain. By using this “aromatic tuning” it is sometimes possible to find a sweet spot where conformational change in the binding domain results in signal transduction. This recent research points towards EnvZ as a good candidate for chimeric receptors in synthetic biology.</P><H4>Using Affibody Molecules to bind biomarkers</H4><P>To use EnvZ to recognize protein biomarkers we had to find a suitable binding domain. Both the C and N terminals of EnvZ are located in the cytoplasm, so a binding domain could not just be fused to the end of EnvZ. This meant that we needed to find a binding domain that was likely to be stable, fold well and bind even when fused to another protein on both ends. Antibody fragments and molecules that mimic antibodies are often used to bind the protein biomarkers. Many families of molecules fit this description. We chose to work with Affibody molecules, an antibody mimetic first described at KTH, the Royal Institute of Technology, and further developed by Stockholm based company Affibody AB. The difference between antibody fragments and Affibody molecules in the receptor are shown in Figure 1.</P><DIV class="figure-text"><P> Figure 1: Illustrations of Chimeric Antigen Receptor (CAR) and Bacterial Antigen Receptor (BAR).</P></DIV><H4>Designing the chimeric receptor</H4><P>Affibody molecules are stable and easy to express in bacteria. The original Affibody scaffold is based on the Z-domain of Protein A in <I>Staphylococcus aureus</I>. Affibody molecules are only 58 amino acids long and structured as a three-helix bundle. By randomizing 13 amino acids in two of the helices, Affibody molecules with specific binding affinity can be found. We chose to work with the Affibody molecule ZHER2:342 that binds HER2, a biomarker for certain aggressive types of breast cancer. The sequence of ZHER2:342 has been published and used in chimeric proteins by many independent researchers . Affibody molecules with affinity for different proteins are of similar structural configuration and it has been suggested that they behave in a similar way upon binding . It is then also likely that a working system with ZHER2:342 can be adapted for other biomarkers.</P><P>To find a BAR that would activate upon binding we had to find a good location to insert the ZHER2:342 into EnvZ. Since EnvZ forms homodimers in the inner membrane <A href="#ref-recognition5"><SUP class="">[3]</SUP></A> and we wanted the function to be as similar to native EnvZ as possible, we needed to research its secondary structure. When expressed in the membrane, the N-terminus of EnvZ begins with 16 amino acids in the cytosol followed by a transmembrane helix. The following periplasmic domain is 123 amino acids long and largely uncharacterized. A second transmembrane helix then leads back into the cytosol to the signal transducing HAMP domain, followed by a histidine kinase. The structure and function of EnvZ are outlined in Figure 2.</P><DIV class="figure-text"><P> Figure 2: An overview of the structure and function of EnvZ.</P></DIV><P>Since the secondary structure of the periplasmic domain has not been determined experimentally, we had to rely on predictions. A first prediction was made using the RaptorX protein prediction server . We then consulted with researcher Roger Draheim who had run a prediction of the periplasmic domain with the Phyre2 tool . These predictions were very similar, and the Figure below is based on the Phyre2 model and rendered with PyMol . Based on this prediction, Draheim suggested that the domain was structurally similar TIpB, another bacterial receptor found in <I>H. pylori</I>. In TIpB, two alpha-helices are responsible for dimerization. The rest of the periplasmic region in TIpB is composed of beta strands, coils and a single helix which forms a ligand binding domain . Based on this, we decided to insert the genetic construct for ZHER2:342 into EnvZ, as shown in Figure 3. This chimeric receptor is hereby referred to as EnvZ-Affibody chimera.</P><P>Four constructs of the chimeric receptor were designed. In the Figure below, regions that were replaced with ZHER2:342 in each construct are colored red.</P><DIV class="figure-text"><P> Figure 3: Bacterial antigen receptor (BAR) constructs 1, 2 and 3 with the parts on EnvZ replaced with Z:HER2:342 marked in red.</P></DIV><P> From the Figure 3, it can be seen that construct 1 was created by replacing part of the PAS domain with the affibody Z:HER2:342. This resulted in 62 residues in EnvZ were exchanged for 52 residues in the affibody. The PAS Beta sheet/Coil I region was kept as this was thought to effect dimerization. Unlike construct 1, construct 2 was created by first adding a GSGGGSG linker to the C-terminal of the affibody molecule. After this the affibody+linker replaced in the same part as for construct 1, resulting in 62 residues being exchanged for 66 residues. </P><P>The final construct (BAR3) was created by replacing a smaller part of the PAS domain. This resulted in 27 residues in EnvZ were exchanged for 58 residues in the affibody. </P><DIV class="figure-text"><P>Figure 4: A and B: Periplasmic domain of EnvZ from amino acid 42 to 158. Regions replaced with Affibody colored red, where A corresponds to BAR 1 and BAR 2 and B corresponds to BAR 3. C: Affibody ZHER2:342. 3D-structures were generated with PyMol.</P></DIV><H4>Choosing the host</H4><P>EnvZ is expressed in the inner membrane and activates OmpR in the cytosol. This presented a problem since the HER2 biomarker is too large to permeate the outer membrane of <I>E.coli</I>. Early in the project we believed that we could use only the epitope which the ZHER2:342 binds to. We later realized that this would not be possible since Affibody molecules do not bind denatured proteins.</P><P>Working with the osmolarity sensor EnvZ requires a distinct and detailed study of the activation of the downstream signaling cascade upon changes in osmolarity. Fortunately, we could work in close connection with our advisor Roger Draheim, an EnvZ expert, who could send us a specific EnvZ reporter strain.</P><P>His EBP30 <I>E.coli</I> strain is an EnvZ- derivate from K-12 MG1655, which contains the transcriptional fusion proteins OmpC-CFP and OmpF-YFP (Figure 4A; see schematic). In normal osmolarity conditions, the fusion protein OmpF-YFP is constitutively expressed whereas OmpC-CFP stays silent. However, changing the osmolarity conditions will lead to increase expression of OmpC-CFP and a decrease in OmpF-YFP (Figure 4B; Nørholm, von Heijne &amp; Draheim (2015) S1). Due to this reporter system we can observe the activation of the EnvZ-OmpR signaling cascade. The knockout of EnvZ has been introduced by homologues recombination and insertion of a Kanamycin selection marker.</P><P>In conclusion, the ΔEnvZ EBP30 strain with its genomically encoded fusion protein reporter system represents the perfect model to study the activation and the signal behavior of our chimeric receptors.</P><P>We then chose between two alternatives. One was to express the system in a gram positive host. This would circumvent the problem since gram positive bacteria lack an outer membrane. This solution would be preferable and should be the aim for future applications of the system. However, since all previous work on EnvZ had been done in <I>E.coli</I>, getting the entire EnvZ-OmpR pathway to work in another host could well be a project of its own. We also had to consider that the EBP30 strain we wanted to use for testing  is gram negative. An option we were left with was then to create spheroplasts by removing the outer membrane of the <I>E.coli</I> before adding the HER2 biomarker to the sample. We decided to go with the option of creating spheroplasts over changing to gram positive since this introduced fewer unknown variables when expressing the receptor in the membrane.</P><P>If spheroplasts are used to prove our concept, they must be able to express proteins to produce a read-out signal. Investigating this thus became a part of our project.</P><H4>Stipulating hypotheses</H4><TABLE class="table table-hover hypotheses-table"><THEAD><TR><TH class="col-md-1 table-head">#</TH><TH class="col-md-7 table-head">Hypothesis</TH><TH class="col-md-2 table-head">Status</TH><TH class="col-md-2">Experiments</TH></TR></THEAD><TBODY><TR><TD class="col-md-1 ">1</TD><TD class="col-md-7 ">A construct of the EnvZ-Affibody chimera can be successfully expressed in <I>E.coli</I></TD><TD class="col-md-2 ">Positive</TD><TD class="col-md-2"><A type="button" class="btn btn-default btn-sm" href="https://2015.igem.org/Team:Stockholm/Results#hyp1">
Go to experiments
</A></TD></TR><TR><TD class="col-md-1 ">2</TD><TD class="col-md-7 ">The construct is expressed in the inner membrane</TD><TD class="col-md-2 ">Inconclusive</TD><TD class="col-md-2"><A type="button" class="btn btn-default btn-sm" href="https://2015.igem.org/Team:Stockholm/Results#hyp2">
Go to experiments
</A></TD></TR><TR><TD class="col-md-1 ">3</TD><TD class="col-md-7 ">A construct of the EnvZ-Affibody chimera protein binds the HER2 protein</TD><TD class="col-md-2 ">Positive</TD><TD class="col-md-2"><A type="button" class="btn btn-default btn-sm" href="https://2015.igem.org/Team:Stockholm/Results#hyp3">
Go to experiments
</A></TD></TR><TR><TD class="col-md-1 ">4</TD><TD class="col-md-7 ">A construct of the EnvZ-Affibody chimera protein is unresponsive to changes in osmolarity
</TD><TD class="col-md-2 ">Negative</TD><TD class="col-md-2"><A type="button" class="btn btn-default btn-sm" href="https://2015.igem.org/Team:Stockholm/Results#hyp4">
Go to experiments
</A></TD></TR><TR><TD class="col-md-1 ">5</TD><TD class="col-md-7 ">The readout can be activated in spheroplast <I>E.coli</I></TD><TD class="col-md-2 ">Inconclusive</TD><TD class="col-md-2"><A type="button" class="btn btn-default btn-sm" href="https://2015.igem.org/Team:Stockholm/Results#hyp5">
Go to experiments
</A></TD></TR><TR><TD class="col-md-1 ">6</TD><TD class="col-md-7 ">A construct of the EnvZ-Affibody chimera protein phosphorylates OmpR when it has bound the HER2 protein</TD><TD class="col-md-2 ">Not started</TD><TD class="col-md-2"><A type="button" class="btn btn-default btn-sm" href="https://2015.igem.org/Team:Stockholm/Results#hyp6">
Go to experiments
</A></TD></TR></TBODY></TABLE><H4>Experimental plan</H4><P>The experimental plan for the BAR was divided over the different hypotheses we aimed to prove for each of the constructs. The first hypothesis handles the successful expression of a  BAR construct in <I>E. coli</I>. For this western blots were deemed the most suitable as we could measure low concentration of our constructs with great accuracy. To confirm the correct location of the protein in the inner membrane we did flow-cytometric measurements. For this we had to construct spheroplasts (gram positive cells were the outer membrane and cell wall has been removed) to gain access to the periplasm and tag our constructs. For testing the affinity of our construct towards HER2, three different experiments were formulated. In the first one an IMAC column would be used to bind in recombinant HER2 through a polyhistidine tag added to the protein. The HER2-inbound column would then act as a affinity column binding the constructs. In the second experiment an ELISA would be performed by first binding HER2 to the plate and then the construct (sandwich method). The last experiment was a FACS analysis on <I>E.coli</I> spheroplasts. We would also like to prove that our construct was unresponsive to osmolarity changes as this would interfere with any signal we get from binding HER2. For this a CFP/YFP read-out strain was transformed and tested for fluorescence at different osmolarities. Finally we would test if the read-out system could be activated in spheroplast <I>E.coli</I>. This would verify the that the link between the BAR and the intracellular signaling would be functional. </P><H2>Intracellular signal transduction</H2><P>If activation of the histidine kinase on the BAR is successful, it will phosphorylate the response regulator OmpR. The aim then becomes to carry this intracellular signal from the receptor to the readout strain, together with quantification of the signal. The main idea is to build/assemble a BioBrick with two OmpR-dependent quorum synthases that will produce two different quorum sensing molecules depending on the levels of phosphorylated OmpR inside the cell. Finally, these quorum sensing molecules are detected by the readout strain and quantified accordingly.</P><DIV class="figure-text"><P>Figure 5: Signaling pathway for ABBBA</P></DIV><P>The whole signaling system is divided into two parts: i) Fluorescence signaling system (consists of OmpR-Regulated-RFP, MicF-Regulated-GFP and OmpR-Regulated-RFP/GFP) to check if the BioBricks are functional and ii) Quorum sensing signaling system (OmpR-Regulated-RhlI, MicF-Regulated-LuxI and OmpR-Regulated-RhlI/LuxI) to replace the fluorescence output with the quorum sensing output as final outcome.</P><H4>Regulating protein expression with OmpR</H4><P>When EnvZ (histidine kinase) exhibits its kinase property, it leads to phosphorylation of OmpR which is an osmoregulatory protein in <I>E.coli</I>. Activation of EnvZ- OmpR two-component regulatory system is the main focus of this signaling pathway. Phosphorylated OmpR regulates differential transcription of two porin responsing genes (ompC and ompF) Activation of the BAR can either lead to kinase activity or phosphatase activity of EnvZ (depending on the conformational change in the receptor and the intracellular osmolarity) which in turn leads to dephosphorylation of OmpR inside the cell. When the osmolarity is low, phosphatase activity of EnvZ is pronounced which leads to increased expression of ompF gene and vice versa. Our first goal was to express RFP which is OmpR regulated. See OmpR-Regulated-RFP in Figure 6.</P><H4>Silencing protein expression with MicF RNA</H4><P>The micF gene has been shown to regulate post-transcriptional expression of ompF gene. The micF gene encodes an antisense RNA which binds to its target region in ompF gene, leading to inhibition of translation . Taken this fact into consideration, we tried to incorporate MicF Target (MicF-T) and GFP in one plasmid in order to express micF-Regulated-GFP. See MicF-Regulated-GFP in Figure 6.</P><H4>Showing differential expression of RFP and GFP</H4><P>This is the final part of the fluorescence signaling system. In order to investigate both the effects of phosphorylated and dephosphorylated OmpR, this part is created. The main idea is to introduce micF RNA in a plasmid, together with OmpR-Regulated-RFP and MicF-Regulated-GFP. In high intracellular osmolarity condition, micF binds to MicF-T in MicF-Regulated-GFP and silence production of GFP (Red colonies are pronounced here). In contrast, GFP is pronounced in low osmolarity when micF is not binding to MicF-T. Thus, it makes the whole system sensitive to both kinase and phosphatase activity in the BAR. See OmpR-Regulated-GFP/RFP in Figure 6.</P><DIV class="figure-text"><P> Figure 6: Signaling biobricks: OmpR-Regulated-RFP, MicF-Regulated-GFP and OmpR-Regulated-RFP/GFP.</P></DIV><H4>Replacing RFP and GFP with quorum sensing molecules</H4><P>After showing that differential expression regulated by OmpR is possible, we want to extend this system to production of quorum sensing molecules. Each fluorescent protein is replaced by a  quorum synthase, keeping the backbone and other parts same. See OmpR-Regulated-RhII and MicF-Regulated-LuxI in Figure 7. OmpR-Regulated-RhII is basically similar to OmpR-Regulated-RFP where RFP is replaced by RhlI synthase which produces OHHL. In MicF-Regulated-LuxI, GFP of MicF-Regulated-GFP is similarly replaced with BHL producing quorum synthase that is LuxI.</P><H4>Putting it all together</H4><P>This is our final product in the signaling pathway which is a combination of OmpR-Regulated-RhII, MicF-Regulated-LuxI and micF RNA. The final goal of this project in the signaling section is to achieve a highly sensitive on-off switch mechanism in the OmpR dependent regulatory system depending on the kinase and the phosphatase activity of the BAR. See OmpR-Regulated-RhII/LuxI in Figure 7.</P><DIV class="figure-text"><P> Figure 7: Signaling biobricks: OmpR-Regulated-RhlI, MicF-Regulated-LuxI and OmpR-Regulated-RhlI/LuxI.</P></DIV><H4>Stipulating hypotheses</H4><TABLE class="table table-hover hypotheses-table"><THEAD><TR><TH class="col-md-1 table-head">#</TH><TH class="col-md-7 table-head">Hypothesis</TH><TH class="col-md-2 table-head">Status</TH><TH class="col-md-2">Experiments</TH></TR></THEAD><TBODY><TR><TD class="col-md-1 ">7</TD><TD class="col-md-7 ">Expression of OmpR-Regulated-RFP leads to OmpR dependent production of red fluorescence protein (RFP)</TD><TD class="col-md-2 ">Positive</TD><TD class="col-md-2"><A type="button" class="btn btn-default btn-sm" href="https://2015.igem.org/Team:Stockholm/Results#hyp7">
Go to experiments
</A></TD></TR><TR><TD class="col-md-1 ">8</TD><TD class="col-md-7 ">Expression of MicF-Regulated-GFP leads to constitutive expression of green fluorescence protein (GFP) which can be silenced with MicF RNA</TD><TD class="col-md-2 ">Inconclusive</TD><TD class="col-md-2"><A type="button" class="btn btn-default btn-sm" href="https://2015.igem.org/Team:Stockholm/Results#hyp8">
Go to experiments
</A></TD></TR><TR><TD class="col-md-1 ">9</TD><TD class="col-md-7 ">Expression of OmpR-Regulated-GFP/RFP leads to OmpR dependent regulation of RFP/GFP production</TD><TD class="col-md-2 ">Inconclusive</TD><TD class="col-md-2"><A type="button" class="btn btn-default btn-sm" href="https://2015.igem.org/Team:Stockholm/Results#hyp9">
Go to experiments
</A></TD></TR><TR><TD class="col-md-1 ">10</TD><TD class="col-md-7 ">Expression of OmpR-Regulated-RhII leads to OmpR dependent expression of quorum sensing molecule BHL</TD><TD class="col-md-2 ">Positive</TD><TD class="col-md-2"><A type="button" class="btn btn-default btn-sm" href="https://2015.igem.org/Team:Stockholm/Results#hyp10">
Go to experiments
</A></TD></TR><TR><TD class="col-md-1 ">11</TD><TD class="col-md-7 ">Expression of MicF-Regulated-LuxI leads to constitutive expression of quorum sensing molecule OHHL which can be silenced with MicF RNA</TD><TD class="col-md-2 ">Not started</TD><TD class="col-md-2"><A type="button" class="btn btn-default btn-sm" href="https://2015.igem.org/Team:Stockholm/Results#hyp11">
Go to experiments
</A></TD></TR><TR><TD class="col-md-1 ">12</TD><TD class="col-md-7 ">Expression of OmpR-Regulated-RhII/LuxI leads to OmpR dependent regulation of BHL/OHHL production</TD><TD class="col-md-2 ">Not started</TD><TD class="col-md-2"><A type="button" class="btn btn-default btn-sm" href="https://2015.igem.org/Team:Stockholm/Results#hyp12">
Go to experiments
</A></TD></TR></TBODY></TABLE><H4>Experimental plan</H4><P>An experimental plan for intracellular signal transduction was set in order to prove the above-mentioned hypotheses. At first, appropriate fluorescent proteins (where their wavelengths do not overlap) and quorum sensing molecules (where they have minimum interaction with each other) were chosen based on the literature data and the previous experiences. We thought of transforming all the needed BioBrick parts one by one from the iGEM distribution kit in order to clone them later on and at the same time we ordered the other necessary parts from IDT. <I>In silico</I> part design was done in Snapgene software. Subsequently, we planned to do the cloning and parts characterization in parallel. We decided to perform osmolarity experiments with different concentrations of sucrose in the culture media. We also considered repeating our experiments at least three times to have some statistical data points and to show their reproducibility.</P><H2>Readout – a second detector strain</H2><P>Readout is a critical part of a biomarker detection assay. Used clinically this step determines whether or not the method can be trusted. A simple readout system needs an easy and cheap assay without the need of any extra expenses which at the same time provides high accuracy.</P><P>In conjunction with the signaling part of the project, the readout of the system was constructed in a separate strain of <I>E.coli</I>. This was chosen for several reasons: to make it easier to create the systems in parallel; to make it a more general system which could be used for other readout purposes; and to potentially achieve amplification through the use of quorum sensing.</P><H4>Positive and negative fluorescent readout</H4><P>In order to have a simple and cheap method, a fluorescence readout was chosen. We decided to express two different kinds of fluorescent proteins. One colour would be produced by the bacteria if the biomarker is bound to the receptor another one no biomarker is bound. This would make it possible to see that the system was working and provide an easy way to distinguish positive/negative readout for diagnosis. </P><H4>Quorum sensing (QS) as a modular approach</H4><P>Quorum sensing (QS) system use communication between bacteria through quorum signalling molecules (QSMs) (N-Acyl homoserine lactones or AHL used in our experiments). If expressed by one bacterial strain, the other bacterial strain can detect and respond accordingly. Biofilm formation is an example of where quorum sensing is used endogenously.</P><DIV class="figure-text"><P> Figure 8: An overview of the ABBBA read-out system.</P></DIV><P>QS was chosen to be used as it would provide a modular approach where the main bacterial strain would signal to a secondary bacterial strain stimulating it to express a certain fluorescent protein. The fluorescence producing bacteria could potentially be used for various different purposes. Another speculation was whether the use of quorum sensing potentially could achieve amplification for the signal.</P><H4>Biobricks for QS</H4><P>The two biobricks chosen for the readout system were <A href="http://parts.igem.org/Part:BBa_K1157006">BBa_K1157006</A> and <A href="http://parts.igem.org/Part:BBa_T9002">BBa_T9002</A>:</P><P>They share the same basic layout, where the QS receptor is coupled with a constitutive promoter and the fluorescent gene is coupled with a promoter activated by the complex of QS receptor and QSM.</P><P>The biobricks were chosen as the QSMs are of similar lengths, while the structures were different enough to ensure that there would be little to no crosstalk between the systems. https://2013.igem.org/Team:Concordia/Interface</P><H4>Stipulating hypotheses</H4><TABLE class="table table-hover hypotheses-table"><THEAD><TR><TH class="col-md-1 table-head">#</TH><TH class="col-md-7 table-head">Hypothesis</TH><TH class="col-md-2 table-head">Status</TH><TH class="col-md-2">Experiments</TH></TR></THEAD><TBODY><TR><TD class="col-md-1 ">13</TD><TD class="col-md-7 ">In the presence of BHL, BBa_K1157006 expresses mCherry (RFP)</TD><TD class="col-md-2 ">Inconclusive</TD><TD class="col-md-2"><A type="button" class="btn btn-default btn-sm" href="https://2015.igem.org/Team:Stockholm/Results#hyp13">
Go to experiments
</A></TD></TR><TR><TD class="col-md-1 ">14</TD><TD class="col-md-7 ">In the presence of OHHL, T9002 expresses GFP</TD><TD class="col-md-2 ">Negative</TD><TD class="col-md-2"><A type="button" class="btn btn-default btn-sm" href="https://2015.igem.org/Team:Stockholm/Results#hyp14">
Go to experiments
</A></TD></TR><TR><TD class="col-md-1 ">15</TD><TD class="col-md-7 ">Expression of the different quorum sensing molecules from the main bacteria is enough to start the expression of the fluorescent proteins</TD><TD class="col-md-2 ">Not started</TD><TD class="col-md-2"><A type="button" class="btn btn-default btn-sm" href="https://2015.igem.org/Team:Stockholm/Results#hyp15">
Go to experiments
</A></TD></TR></TBODY></TABLE><H4>Experimental plan</H4><P>There are several biobricks which produce fluorescent proteins when exposed to different QSMs. Two different colored proteins were chosen as well as two different QSMs. Colours were chosen to have well separated spectra and the QSMs to not  interfere. The two biobricks would be tested separately with their corresponding and non-specific QSMs to determine regular levels and possible interference. The two biobricks would then be cloned together and again characterized with each QSMs. Different levels of each QSMs simultaneously would also be characterised, as this scenario would be likely in the full system. </P><H2>Modeling the EnvZ/OmpR pathway</H2><P>To predict the correlation between the concentration of activated EnvZ and signal intensity, we modeled the mechanism from EnvZ to OmpC and OmpF. Depending on the osmolarity level, the output is either OmpC  (in the case of high osmolarity) or OmpF (low osmolarity). By determining this connection we can then adjust the EnvZ amount to get the desired product, OmpC.  After the OmpC translation, the two quorum sensing molecules (QSMs), BHL and RhlR, are activated to further produce GFP. By analyzing the ratio at which OmpC triggers the QSMs, we can achieve a GFP production that is easy to detect. With this data we would, in theory, be able to link back to the EnvZ concentration. This means that a detectable level of the biomarker can be received.</P><P>The process by which we achieved these results was mostly researching the subjects and using Simbiology, an application in Matlab to both design and simulate models. For the third (release of QSMs) and fourth part (GFP expression) of our project we started with the same methods as previously but realized early on that we needed to develop our own equation. For this we expanded our knowledge in Matlab and also contacted iGEM Technion from 2014 for support.</P><P>The hypothesis for the first part (EnvZ -&gt; OmpC/OmpF) was that EnvZ is proportional to the OmpR-P production but, at the same time, a part of the OmpR-P is consumed.</P><H4>References</H4><TABLE class="table table-condensed reference-table"><THEAD><TR><TH class="col-md-1 table-head">#</TH><TH class="col-md-10 table-head">Reference</TH></TR></THEAD><TBODY><TR><TD class="col-md-1 ">1</TD><TD class="col-md-10 "><A href="http://jnci.oxfordjournals.org/content/101/1/8.full">
Schmidt, C. (2009). Urine biomarkers may someday detect even distant tumors. Journal of the National Cancer Institute, 101(1), 8-10.</A></TD></TR><TR><TD class="col-md-1 ">2</TD><TD class="col-md-10 "><A href="http://cancerdiscovery.aacrjournals.org/content/3/4/388.short">
Sadelain, M., Brentjens, R., &amp; Rivière, I. (2013). The basic principles of chimeric antigen receptor design. Cancer discovery, 3(4), 388-398.</A></TD></TR><TR><TD class="col-md-1 ">3</TD><TD class="col-md-10 "><A href="http://www.ncbi.nlm.nih.gov/pubmed/20690824?dopt=Abstract">
Parkinson, J. S. (2010). Signaling mechanisms of HAMP domains in chemoreceptors and sensor kinases. Annual review of microbiology, 64, 101-122.</A></TD></TR><TR><TD class="col-md-1 ">4</TD><TD class="col-md-10 "><A href="https://books.google.se/books?id=mIqqjShSJFIC&amp;lpg=PA212&amp;ots=YSLpkZSpVr&amp;dq=Cph1%20hamp&amp;pg=PA212#v=onepage&amp;q=Cph1%20hamp&amp;f=true">
Collin, M., and Schuch, R. (2009). Bacterial Sensing and Signalling.</A></TD></TR><TR><TD class="col-md-1 ">5</TD><TD class="col-md-10 "><A href="http://www.nature.com/nature/journal/v438/n7067/abs/nature04405.html">
Levskaya, A., Chevalier, A. A., Tabor, J. J., Simpson, Z. B., Lavery, L. A., Levy, M., ... &amp; Voigt, C. A. (2005). Synthetic biology: engineering Escherichia coli to see light. Nature, 438(7067), 441-442.</A></TD></TR><TR><TD class="col-md-1 ">6</TD><TD class="col-md-10 "><A href="http://www.ncbi.nlm.nih.gov/pubmed/25162177">
Nørholm MH, Von heijne G, Draheim RR. Forcing the issue: aromatic tuning facilitates stimulus-independent modulation of a two-component signaling circuit. ACS Synth Biol. 2015;4(4):474-81.</A></TD></TR><TR><TD class="col-md-1 ">7</TD><TD class="col-md-10 "><A href="http://www.pnas.org/content/107/34/15039.full">
Eigenbrot, C., Ultsch, M., Dubnovitsky, A., Abrahmsén, L., &amp; Härd, T. (2010). Structural basis for high-affinity HER2 receptor binding by an engineered protein. Proceedings of the National Academy of Sciences, 107(34), 15039-15044.</A></TD></TR><TR><TD class="col-md-1 ">8</TD><TD class="col-md-10 "><A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC404300/">
Högbom, M., Eklund, M., Nygren, P. Å., &amp; Nordlund, P. (2003). Structural basis for recognition by an in vitro evolved affibody. Proceedings of the National Academy of Sciences, 100(6), 3191-3196.</A></TD></TR><TR><TD class="col-md-1 ">9</TD><TD class="col-md-10 "><A href="http://raptorx.uchicago.edu/about/">
Morten Källberg, Haipeng Wang, Sheng Wang, Jian Peng, Zhiyong Wang, Hui Lu &amp; Jinbo Xu. Template-based protein structure modeling using the RaptorX web server. Nature Protocols 7, 1511–1522, 2012.</A></TD></TR><TR><TD class="col-md-1 ">10</TD><TD class="col-md-10 "><A href="http://www.nature.com/nprot/journal/v10/n6/abs/nprot.2015.053.html">
Kelley, L. A., Mezulis, S., Yates, C. M., Wass, M. N., &amp; Sternberg, M. J. (2015). The Phyre2 web portal for protein modeling, prediction and analysis. Nature protocols, 10(6), 845-858.</A></TD></TR><TR><TD class="col-md-1 ">11</TD><TD class="col-md-10 "><A href="https://www.pymol.org/citing">
The PyMOL Molecular Graphics System, Version 1.7.4 Schrödinger, LLC.</A></TD></TR><TR><TD class="col-md-1 ">12</TD><TD class="col-md-10 "><A href="http://www.sciencedirect.com/science/article/pii/S0969212612001815">
Sweeney, E. G., Henderson, J. N., Goers, J., Wreden, C., Hicks, K. G., Foster, J. K., ... &amp; Guillemin, K. (2012). Structure and proposed mechanism for the pH-sensing Helicobacter pylori chemoreceptor TlpB. Structure, 20(7), 1177-1188.</A></TD></TR><TR><TD class="col-md-1 ">13</TD><TD class="col-md-10 "><A href="http://www.sciencedirect.com/science/article/pii/S0966842X00017078">
Buckler, D. R., Anand, G. S., &amp; Stock, A. M. (2000). Response-regulator phosphorylation and activation: a two-way street?. Trends in microbiology, 8(4), 153-155.</A></TD></TR><TR><TD class="col-md-1 ">14</TD><TD class="col-md-10 "><A href="http://www.jbc.org/content/277/27/24155.full">
Cai, S. J., &amp; Inouye, M. (2002). EnvZ-OmpR interaction and osmoregulation in <I>Escherichia coli</I>. Journal of Biological Chemistry, 277(27), 24155-24161.</A></TD></TR><TR><TD class="col-md-1 ">15</TD><TD class="col-md-10 "><A href="http://www.sciencedirect.com/science/article/pii/S0022283601950296">
Delihas, N., &amp; Forst, S. (2001). MicF: an antisense RNA gene involved in response of Escherichia coli to global stress factors. Journal of molecular biology, 313(1), 1-12.</A></TD></TR></TBODY></TABLE></DIV><DIV class="footer-container"><DIV class="row title-row"><DIV class="col-md-10 col-sm-10 col-xs-10 footer-title">iGEM Stockholm is made possible with support from:</DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>