<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Stockholm_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Stockholm/Results</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="page-wrapper"><CENTER><DIV id="header-wrapper"><DIV id="header" class="container"><NAV id="nav"><DIV class="title noselect">iGEM STOCKHOLM</DIV><UL><LI><A class="icon-home" href="https://2015.igem.org/Team:Stockholm"><SPAN>Home</SPAN></A></LI><LI><A class="icon-users" href="https://2015.igem.org/Team:Stockholm/Team"><SPAN>Team</SPAN></A></LI><LI><A class="icon-lab" href="https://2015.igem.org/Team:Stockholm/Description"><SPAN>Project</SPAN></A></LI><LI><A class="icon-clipboard" href="https://2015.igem.org/Team:Stockholm/Results"><SPAN>Results</SPAN></A></LI><LI><A class="icon-bubbles2" href="https://2015.igem.org/Team:Stockholm/Practices"><SPAN>Practices</SPAN></A></LI><LI><A class="icon-earth" href="https://2015.igem.org/Team:Stockholm/Collaborations"><SPAN>Collaborations</SPAN></A></LI><LI><A class="icon-gift" href="https://2015.igem.org/Team:Stockholm/Attributions"><SPAN>Attributions</SPAN></A></LI></UL></NAV></DIV></DIV></CENTER></DIV><DIV class="container-fluid content-container"><DIV id="sticky"><UL class="nav nav-tabs pillnav"><LI role="presentation" class="active"><A href="#"><B> Results</B></A></LI><LI role="presentation" class="dropdown anchor_link"><A class="dropdown-toggle" data-toggle="dropdown" href="#" role="button" aria-haspopup="true" aria-expanded="false">
Lab results </A></LI><LI role="presentation" class="clickable anchor_link"><A href="#recognition">Recognition</A></LI><LI role="presentation" class="clickable anchor_link"><A href="#signaling">Signaling</A></LI><LI role="presentation" class="clickable anchor_link"><A href="#readout">Read-out</A></LI><LI role="presentation" class="clickable anchor_link"><A href="https://2015.igem.org/Team:Stockholm/Measurement">Inter-lab measurements</A></LI><LI role="presentation" class="clickable anchor_link"><A href="https://2015.igem.org/Team:Stockholm/Safety">Safety</A></LI></UL><LI role="presentation" class="clickable anchor_link"><A href="https://2015.igem.org/Team:Stockholm/Modeling">Modeling</A></LI><LI role="presentation" class="clickable anchor_link"><A href="https://2015.igem.org/Team:Stockholm/Medal">Medals</A></LI><LI role="presentation" class="clickable anchor_link"><A href="https://2015.igem.org/Team:Stockholm/Parts">Parts</A></LI><LI role="presentation" class="clickable anchor_link"><A href="https://2015.igem.org/Team:Stockholm/Protocols">Protocols</A></LI></DIV><H2 id="#">Methods and results</H2><P>Our team had access to labs from the beginning of June to the middle of September. During this time, we tested the parts and hypotheses formulated in the project description. In the sections below, each hypothesis is listed along with the experiments we designed to test it. Some hypotheses remain untested due to time constraints. In the table below, all hypotheses are listed again for sake of clarity. At the end of each hypothesis, you can find the contact details of the person reponsible for the experiment. If you have any questions or concerns, you can contact them. </P><P>This style of presentation came out of our integrated human practices work on research transparency in the iGEM community and was informed by a survey answered by 44 teams, and by evaluating 19 gold winning overgrad wikis from 2013 and 2014. See our <A href="https://2015.igem.org/Team:Stockholm/Ethics">report on transparency and negative results in the iGEM community</A> to find out more and <A href="https://2015.igem.org/Team:Stockholm/Wikipledge">read our pledge to see our suggestions for future wikis</A>.</P><H2>Hypotheses</H2><H4>Biomarker recognition hypotheses</H4><TABLE class="table table-hover hypotheses-table"><THEAD><TR><TH class="col-md-1 table-head">#</TH><TH class="col-md-7 table-head">Hypothesis</TH><TH class="col-md-2 table-head">Status</TH><TH class="col-md-2">Experiments</TH></TR></THEAD><TBODY><TR><TD class="col-md-1 ">1</TD><TD class="col-md-7 ">A construct of the EnvZ-Affibody chimera can be successfully expressed in <I>E. coli</I></TD><TD class="col-md-2 ">Positive</TD><TD class="col-md-2"><A type="button" class="btn btn-default btn-sm" href="#hyp1">
Go to experiments
</A></TD></TR><TR><TD class="col-md-1 ">2</TD><TD class="col-md-7 ">The construct is expressed in the inner membrane</TD><TD class="col-md-2 ">Inconclusive</TD><TD class="col-md-2"><A type="button" class="btn btn-default btn-sm" href="#hyp2">
Go to experiments
</A></TD></TR><TR><TD class="col-md-1 ">3</TD><TD class="col-md-7 ">A construct of the EnvZ-Affibody chimera protein binds the HER2 protein</TD><TD class="col-md-2 ">Positive</TD><TD class="col-md-2"><A type="button" class="btn btn-default btn-sm" href="#hyp3">
Go to experiments
</A></TD></TR><TR><TD class="col-md-1 ">4</TD><TD class="col-md-7 ">A construct of the EnvZ-Affibody chimera protein is unresponsive to changes in osmolarity</TD><TD class="col-md-2 ">Negative</TD><TD class="col-md-2"><A type="button" class="btn btn-default btn-sm" href="#hyp4">
Go to experiments
</A></TD></TR><TR><TD class="col-md-1 ">5</TD><TD class="col-md-7 ">The readout can be activated in spheroplast <I>E. coli</I></TD><TD class="col-md-2 ">Inconclusive</TD><TD class="col-md-2"><A type="button" class="btn btn-default btn-sm" href="#hyp5">
Go to experiments
</A></TD></TR><TR><TD class="col-md-1 ">6</TD><TD class="col-md-7 ">A construct of the EnvZ-Affibody chimera protein phosphorylates OmpR when it has bound the HER2 protein</TD><TD class="col-md-2 ">Not started</TD><TD class="col-md-2"><A type="button" class="btn btn-default btn-sm" href="#hyp6">
Go to experiments
</A></TD></TR></TBODY></TABLE><H4>Intracellular signal transduction hypotheses</H4><TABLE class="table table-hover hypotheses-table"><THEAD><TR><TH class="col-md-1 table-head">#</TH><TH class="col-md-7 table-head">Hypothesis</TH><TH class="col-md-2 table-head">Status</TH><TH class="col-md-2">Experiments</TH></TR></THEAD><TBODY><TR><TD class="col-md-1 ">7</TD><TD class="col-md-7 ">Expression of OmpR-Regulated-RFP leads to OmpR dependent production of red fluorescent protein (RFP)</TD><TD class="col-md-2 ">Positive</TD><TD class="col-md-2"><A type="button" class="btn btn-default btn-sm" href="#hyp7">
Go to experiments
</A></TD></TR><TR><TD class="col-md-1 ">8</TD><TD class="col-md-7 ">Expression of MicF-Regulated-GFP leads to constitutive expression of green fluorescent protein (GFP) which can be silenced with MicF RNA</TD><TD class="col-md-2 ">Inconclusive</TD><TD class="col-md-2"><A type="button" class="btn btn-default btn-sm" href="#hyp8">
Go to experiments
</A></TD></TR><TR><TD class="col-md-1 ">9</TD><TD class="col-md-7 ">Expression of OmpR-Regulated-GFP/RFP leads to OmpR dependent regulation of RFP/GFP production</TD><TD class="col-md-2 ">Inconclusive</TD><TD class="col-md-2"><A type="button" class="btn btn-default btn-sm" href="#hyp9">
Go to experiments
</A></TD></TR><TR><TD class="col-md-1 ">10</TD><TD class="col-md-7 ">Expression of OmpR-Regulated-RhII leads to OmpR dependent expression of quorum sensing molecule BHL</TD><TD class="col-md-2 ">Positive</TD><TD class="col-md-2"><A type="button" class="btn btn-default btn-sm" href="#hyp10">
Go to experiments
</A></TD></TR><TR><TD class="col-md-1 ">11</TD><TD class="col-md-7 ">Expression of MicF-Regulated-LuxI leads to constitutive expression of quorum sensing molecule OHHL which can be silenced with MicF RNA</TD><TD class="col-md-2 ">Not started</TD><TD class="col-md-2"><A type="button" class="btn btn-default btn-sm" href="#hyp11">
Go to experiments
</A></TD></TR><TR><TD class="col-md-1 ">12</TD><TD class="col-md-7 ">Expression of OmpR-Regulated-RhII/LuxI leads to OmpR dependent regulation of BHL/OHHL production</TD><TD class="col-md-2 ">Not started</TD><TD class="col-md-2"><A type="button" class="btn btn-default btn-sm" href="#hyp12">
Go to experiments
</A></TD></TR></TBODY></TABLE><H4>Read-out hypotheses</H4><TABLE class="table table-hover hypotheses-table"><THEAD><TR><TH class="col-md-1 table-head">#</TH><TH class="col-md-7 table-head">Hypothesis</TH><TH class="col-md-2 table-head">Status</TH><TH class="col-md-2">Experiments</TH></TR></THEAD><TBODY><TR><TD class="col-md-1 ">13</TD><TD class="col-md-7 ">In the presence of BHL, BBa_K1157006 expresses mCherry (RFP)</TD><TD class="col-md-2 ">Inconclusive</TD><TD class="col-md-2"><A type="button" class="btn btn-default btn-sm" href="#hyp13">
Go to experiments
</A></TD></TR><TR><TD class="col-md-1 ">14</TD><TD class="col-md-7 ">In the presence of OHHL, T9002 expresses GFP</TD><TD class="col-md-2 ">Negative</TD><TD class="col-md-2"><A type="button" class="btn btn-default btn-sm" href="#hyp14">
Go to experiments
</A></TD></TR><TR><TD class="col-md-1 ">15</TD><TD class="col-md-7 ">Expression of the different quorum sensing molecules from the main bacteria is enough to start the expression of the fluorescent proteins</TD><TD class="col-md-2 ">Not started</TD><TD class="col-md-2"><A type="button" class="btn btn-default btn-sm" href="#hyp15">
Go to experiments
</A></TD></TR></TBODY></TABLE><H2 id="recognition">Biomarker recognition</H2><H4>HYPOTHESIS 1: A construct of the EnvZ-Affibody chimera can be successfully expressed in <I>E. coli</I></H4><P><B>Introduction:</B> After having modeled and synthesized our Bacterial Antigen Receptor constructs (BAR1, BAR2 and BAR3), we performed western blots (WB) to show the expression of our EnvZ-Affibody chimera protein in <I>E. coli</I> TOP10 strain. In order to check the expression of our constructs, we used antibodies targeting different parts of our chimeric receptor. The first was an anti-V5 antibody, against the C-terminal V5 tag of the EnvZ receptor. The second was an anti-affibody antibody to check the presence and expression of our BAR constructs.</P><P><B>Method:</B> BAR1, BAR2, BAR3, EnvZ wildtype (wt), and the empty plasmid control were transformed in TOP10 cells. Those constructs were verified by SANGER sequencing (SUPREMERUN by GATC Biotech). Singles colonies were picked and cultures in log phase were induced overnight with 25 µM IPTG. Western blots were performed for each sample using a mouse monoclonal antibody against the V5 tag (R960-CUS from Thermo Fisher Scientific) and a polyclonal goat antibody against the Affibody molecule (Goat Anti-Affibody® IgG from Affibody AB)  according to the <A href="https://static.igem.org/mediawiki/2015/f/fb/STHLM_WB_protocol.pdf">western blot protocol</A>. For each experiment, the membrane was afterwards stripped in mild conditions according to the <A href="https://static.igem.org/mediawiki/2015/7/7d/STHLM_Stripping_protocol.pdf"> stripping protocol</A>. The membrane was then incubated with a control protein: a mouse monoclonal antibody against DnaK (ADI-SPA-880-D from Enzo). All the secondary antibodies used in these experiments were coupled to horseradish peroxidase. The imaging was acquired with a BioRad Gel DOc  XR+ Imager. </P><P><B>Results:</B> The three constructs and the wild type EnvZ in the IPTG inducible low copy number plasmid pRD400 have successfully been expressed (Figure 1A). </P><DIV class="figure-text"><P> Figure 1: EnvZ-Affibody chimera can be expressed in TOP10 <I>E. coli</I> upon IPTG induction.</P></DIV><P> In Figure 1, the wild type as well as the constructs can be observed as homodimer with an approximate size of 100kDa (black triangle) and monomer with an approximate size of 50kDa (black circle). Further analysis, using an anti-Affibody antibody, confirmed that only our constructs strains are expressing the desired BAR constructs (black star) (Figure 1B). The faint bands at around 25kDa (black square) are presumably caused by some unspecific binding. The loading control DnaK  shows in all WB lanes proving that all wells have been loaded equally with whole cell lysates of the different strains.</P><P><B>Conclusion for hypothesis 1:</B> The EnvZ-chimera proteins BAR1, BAR2 and BAR3  have successfully been expressed in <I>E. coli</I> TOP10. 
Contact person: Manon Ricard, <A href="mailto:manon.ricard@yahoo.fr">manon.ricard@yahoo.fr</A></P><H4>HYPOTHESIS 2: The construct is expressed in the inner membrane</H4><P><B>Introduction:</B> As our chimeric receptor derives from the osmoregulator EnvZ which is situated in the inner membrane, we are suspecting that our BARs are going to have the same subcellular location. In order to confirm its localisation in the inner membrane of our chimeric receptor, we performed a FACS analysis comparing spheroplast (gram-negative cells with removed outer membrane and cell wall) to untreated cells. </P><P><B>Method:</B> For the FACS analysis, we used the following strains: TOP10-pRD400-EnvZ-V5 and the TOP10-pRD400-BAR2-V5 which were induced in log phase (OD<SUB>600 nm</SUB> ~0.3) with 25 µM IPTG and incubated overnight. From the overnight cultures, one sample of each strain was treated following the<A href="https://static.igem.org/mediawiki/2015/7/7b/STHLM_Spheroplast_protocol.pdf"> spheroplast protocol</A>, whereas one sample of each strain was left untreated. All samples were then incubated with a fluorescently-labeled Alexa 488 anti-affibody antibody. Finally, the samples were analysed using Beckman Coulter Gallios FACS machine (<A href="https://static.igem.org/mediawiki/2015/e/e3/STHLM_FACS_protocol.pdf">FACS protocol</A>). Machine was set to a 11 V (FSC), 993 V (SSC) and 354 V for the direct Alexa488 anti-affibody antibody.</P><P><B>Results:</B> Cells have been resuspended in PBS and then gated accordingly for the <I>E. Coli</I> population. Interestingly, the spheroplast populations seems to be differentiate from the broader untreated <I>E. Coli</I> population (Figure 2B). Therefore, the gate setting could be kept the same for all samples test. The gated population has been consequently analyzed for their Alexa488 levels. Hereby, we used the untreated strains as control to adjust the voltage accordingly to have signal peak on the left side of the histogram. We observe that only the TOP10-pRD400-EnvZ Spheroplast samples express a distinct positive colony, whereas the TOP10-pRD400-BAR2 spheroplast population show in a perfect overlay with the negative population for Alexa488 (Figure 2A). </P><P> For this observation, we do have two plausible explanations: (1) We created only a spheroplast with the pRD400-EnvZ-V5 strain, hence, the positive population is due unspecific binding or (2) we have mixed up the samples between the spheroplast pRD400-EnvZ and pRD400-BAR2. We tried to reproduce the results and failed. We only observed negative populations.</P><DIV class="figure-text"><P>Figure 2: FACS analysis of pRD400-BAR2-V5 and pRD400-EnvZ-V5 expressing TOP10 strains and their corresponding spheroplasts.</P></DIV><P><B>Conclusion for hypothesis 2:</B> For this observation, we do have two plausible explanations: (1) We created only a spheroplast with the pRD400-EnvZ-V5 strain, hence, the positive population is due unspecific binding or (2) a human error by mixing up the samples between the spheroplast pRD400-EnvZ and pRD400-BAR2.  Contact person: Felix Richter, <A href="mailoto:felix.richter@stud.ki.se">felix.richter@stud.ki.se</A></P><H4>HYPOTHESIS 3: A construct of the EnvZ-Affibody chimera protein binds the HER2 protein</H4><P><B>Introduction:</B> To test whether interaction between our BAR construct and HER2 was taking place, affinity based experiments were performed. The affinity was primarily tested using column experiments to separate the BAR construct from a cell lysate. For this experiment BAR constructs 1 and 2 were tested by running them in a IMAC column with inbound recombinant HER2 which would hopefully capture the constructs through the affibody. Also, an ELISA experiment was attempted for construct 2 with HER2 fixed to the ELISA plate. Additionally a FACS analysis was performed on spheroplasts by fluorescently tagging the constructs in the inner membrane incubated with HER2. </P><TABLE class="table table-hover hypotheses-table"><THEAD><TR><TH class="col-md-1 table-head">#</TH><TH class="col-md-7 table-head">Experiments for hypothesis 3</TH><TH class="col-md-2 table-head">Status</TH></TR></THEAD><TBODY><TR><TD class="col-md-1 ">1</TD><TD class="col-md-7 "> Can affinity be proved using affinity separation in an IMAC column?</TD><TD class="col-md-2">Unfinished</TD><TD class="col-md-2"><A href="#hyp3exp1" type="button" class="btn btn-default btn-sm">
Go to experiment
</A></TD></TR><TR><TD class="col-md-1 ">2</TD><TD class="col-md-7 ">Could interaction be proven using ELISA?</TD><TD class="col-md-2 ">Positive</TD><TD class="col-md-2"><A href="#hyp3exp2" type="button" class="btn btn-default btn-sm">
Go to experiment
</A></TD></TR><TR><TD class="col-md-1 ">3</TD><TD class="col-md-7 ">Could interaction be proven for the constructs in spheroblasts using FACS?</TD><TD class="col-md-2 ">Unfinished</TD><TD class="col-md-2"><A href="#hyp3exp3" type="button" class="btn btn-default btn-sm">
Go to experiment
</A></TD></TR></TBODY></TABLE><H4>Experiment 1: Can affinity be proved using affinity separation in an IMAC column?</H4><P><B>Method:</B>  For this experiment the constructs were in the pRD400 backbone transformed into TOP10 <I>E. coli</I>. Sample of chosen construct were grown overnight at 37 degrees in LB medium with 100 µg/ml ampicillin. Culture diluted to OD<SUB>600 nm</SUB>~0.3 before induction with 25 µM IPTG. 5 mL  of bacterial culture with an approximate OD of 0.3  were lysed using a 6X SDS lysis buffer (without bromophenol blue). Consequently the cells were thawed and frozen for 10 minutes each in three cycles.</P><P>From the lysates, the protein concentrations were determined by measuring absorbance at 280 nm. Lysates were then prepared in solutions of 2-10 mg. Recombinant His6-HER2 (from R&amp;D Systems) was used to make a temporary affinity column using the his-tag as a anchor. rHER2 was applied in amounts between 5-10 µg to a Talon-Co<SUP>2+</SUP> matix. From running the <A href="https://static.igem.org/mediawiki/2015/8/86/STHLM_Affinity_separation_protocol.pdf"> affinity separation</A> a number of samples were collected. To verify the content of these fractions a dot blot was made according to the following <A href="https://static.igem.org/mediawiki/2015/1/17/STHLM_DotBlot_protocol.pdf">protocol</A>. </P><P>In the dot blot both HER2 and the applied cell lysate was used to control the presence of HER2 and BAR respectively. For each experiment two membranes were prepared with the same samples to be treated with the different antibodies (anti-HER2 and anti-V5). To increase concentration of these samples, freeze-drying for 2h was tried once for BAR construct 2.  </P><P><B>Results:</B>From Figure 3 the anti-HER2 treated membranes display the HER2 control in comparable intensities for both constructs. In the elution fractions the HER2 is found on both membranes but is absent from the flow-through and washing samples. This confirms that the binding and elution of HER2 have gone according to plan.  </P><DIV class="figure-text"><P>Figure 3: Dot blot membrane images for BAR construct 1 treated with anti-HER2 and anti-V5 respectively. </P></DIV><P>For Figure 3 samples were applied from top to bottom in decreasing amounts. HER2 was applied in amounts 5 ng, 1 ng, 0.2 ng. BAR1 control ,i.e. the lysate, was applied in approximate amounts of 45 µg, 9µg, and 1.8 µg. The samples tested were flow-through from applying HER2 (HER2 flow), washout from applying the lysate (BAR1 wash) and elution fractions 1, 4, 7, 10, 12 and 14.</P><P> In the anti-V5 treated membranes dots are only present in the positive control (BAR1 lysate). The lack of BAR1 from both the fraction before elution (washing) and in the elution fractions suggest the protein was applied in too low concentration to be visualized. A concentrating step should therefore be added to the method to fully asses affinity through this experiment.</P><H4>Experiment 2: Could interaction be proven using ELISA?</H4><P><B>Method:</B> Another way of showing possible interaction between proteins is the performance of an Enzyme-linked immunosorbant assay (ELISA) following the <A href="https://static.igem.org/mediawiki/2015/6/68/STHLM_ELISA_new_protocol.pdf">ELISA protocol</A>. Whole cell protein lysates from three different strains have been used: TOP10-pRD400-BAR2-V5, TOP10-pRD400-EnvZ-V5 and TOP10-pEB (empty plasmid control). Therefore we coated a 96-well ELISA plate (Costar 3690) with two different concentrations of HER-2, 3 µg/mL &amp; 5ug/mL (R&amp;D Systems NP_004439). After we removed unbound HER-2, we applied whole cell lysate samples extracted by using a 6X SDS buffer and incubated the samples for 1,5 hours at 37°C under permanent agitation (150rpm). We specifically targeted retained receptor by applying a mouse anti-V5 antibody  (R960-CUS from Thermo Fisher Scientific). We revealed the presence of our receptor bound to the HER-2 by the corresponding HRP-conjugated secondary antibody. The plate was measured at an OD of 450 nm using a CLARIOstar plate reader (BMG Labtech). </P><P><B>Results:</B></P><DIV class="figure-text"><P>Figure 4: Interaction of HER2 with the solublilized chimeric receptor BAR2 using ELISA assay. ‘*’ signifies significant P value. ns : P &gt; 0.05 (not significant), ‘*’ : P ≤ 0.05.</P></DIV><P> In Figure 4, all obtained signals have been processed by normalizing against the empty-plasmid control. pRD400-EnvZ-V5 has been used as a negative control and as expected, it does not show any difference from the empty-plasmid control. However, we observe a 50% increase in absorbance in samples with the pRD400-BAR2-V5 lysate. This general trend can be observed in both coating concentrations (3 µg/mL and 5 µg/mL). The results represented above indicate that BAR2 is actively retained by HER-2 coated surfaces.</P><H4>Experiment 3: Could interaction be proven for the constructs in spheroplasts using FACS?</H4><P><B>Method:</B> In order to assess whether our chimeric receptor will be capable of binding HER-2 in its original subcellular localization, we first removed the outer membrane and cell wall of TOP10 cells containing either the plasmids pRD400-EnvZ-V5 or pRD400-BAR2-V5 in order to create spheroplasts. Therefore we were following the <A href="https://static.igem.org/mediawiki/2015/7/7b/STHLM_Spheroplast_protocol.pdf">spheroplast protocol</A>. For the FACS analysis we used both strains as spheroplasts and as a control in their untreated form. All four samples have from now on be treated exactly the same. We first incubated all samples with 0.1µg/mL of recombinant His-tagged HER-2 (R&amp;D Systems NP_004439) and then consequently revealed the bound His-tagged HER-2 using a Penta-His Alexa647 conjugated antibody (QIAGEN, Cat# 35370). The staining procedure is described in the <A href="https://static.igem.org/mediawiki/2015/e/e3/STHLM_FACS_protocol.pdf">FACS protocol</A>. For exciting the fluorophore we used a 640 nm laser and a bandpass filter at 560 mn (± 20 nm). Machine was set to 11 V (FSC), 993 V (SSC) and 772 V for the anti-penta-His Alexa647 antibody. </P><P><B>Results:</B></P><DIV class="figure-text"><P>Figure 5: HER-2 binding capacity of BAR-2 in spheroplast and untreated cells.</P></DIV><P>In the FACS analysis performed (Figure 5), we were unable to show the binding of HER-2 to our BAR2 construct [Explain]. We have to argue whether the technique however represents the appropriate measure to investigate the question (notably in the short time we had left to test this hypothesis). Despite the technical issues which are coming along in producing a spheroplast and applying a FACS staining, we would need to improve the protocol as it would give us information about the functioning of BARs in a physiological condition. </P><P><B>Conclusion for hypothesis 3:</B> Despite having two experiments in which we have not seen any interaction between HER-2 and the BAR receptor, we have reproducible results using the way less complex ELISA protocol to proof interaction of HER-2 with BAR2. <B>We therefore conclude that binding capacity of the integrated Affibody molecule is maintained.</B> Contact person: Pontus Höjer, <A href="mailto:phojer@kth.se">phojer@kth.se</A></P><H4>HYPOTHESIS 4: A construct of the EnvZ-Affibody chimera protein is unresponsive to changes in osmolarity</H4><P><B>Introduction:</B> EnvZ is originally a receptor in <I>E. coli</I> which is sensible to osmolarity. In our three EnvZ-Affibody chimeras (BAR1, BAR2 and BAR3) a part of the periplasmic region has been replaced by the affibody. We are therefore expecting our constructs to not be reacting to changes in osmolarity. This was tested by performing osmolarity tests on our constructs.   </P><P><B>Method:</B> For this experiment the constructs BAR1, BAR2  were in the pRD400 backbone and transformed into <I>E. coli</I> EPB30. pRD400 (WT) was used as a positive control. pEB5 which is an empty plasmid in the same pRD400 backbone was used as a negative control. When the samples were in log phase, they were induced with 25 µM IPTG and cultured overnight  in high and low osmolarity media. Osmolarity tests were performed with triplicates and according to the <A href="https://static.igem.org/mediawiki/2015/0/05/STHLM_Alternative_Osmolarity_protocol.pdf">protocol</A>. </P><P><B>Results:</B>In Figure 6, the graph shows that the constructs cultured in high osmolarity medium expressed more CFP than constructs cultured in low osmolarity. These values therefore show that our constructs are still osmolarity dependent. YFP values obtained during this experiment were off and inconclusive. </P><DIV class="figure-text"><P>Figure 6: CFP fluorescence of the different samples in low osmolarity medium (0% sucrose) and high osmolarity medium (15% sucrose). ‘*’ signifies significant P value. ns : P &gt; 0.05 (not significant), ‘*’ : P ≤ 0.05, ‘**’ : P ≤ 0.01, ‘***’ : P ≤ 0.001. </P></DIV><P><B>Conclusion for hypothesis 4:</B> We did not manage to prove that spheroplasts are able to express proteins and a read-out signal and cannot conclude at this point as the data was inconclusive. Further experiments should be made such as some gram-staining in order to determine the presence or not of spheroplasts. 
Contact person: Manon Ricard, <A href="mailto:manon.ricard@yahoo.fr">manon.ricard@yahoo.fr</A></P><H4>HYPOTHESIS 5: The readout can be activated in spheroplast <I>E. coli</I></H4><P><B>Introduction:</B> EnvZ is expressed in the inner membrane of gram-negative <I>E. coli </I> and it needs to bind a protein, HER2, that cannot reach the receptor due to the outer membrane. We therefore considered the possibility to create spheroplasts by removing the outer membrane of <I>E. coli</I> before adding the HER2 biomarker to the sample as an attractive option. We consequently wanted to show that spheroplasts are able to express proteins and a readout signal. We investigated this by comparing the fluorescent signal given by spheroplast and non spheroplast. </P><P><B>Method:</B> In this experiment, the BL21-pBAD-GFP strain, which contains a plasmid with GFP controlled by an arabinose promoter, was used. Half of the samples were induced overnight with 6% arabinose, whereas the other half were left non induced. From the overnight cultures, half of the induced samples and half of the non induced samples were  treated according to the <A href="https://static.igem.org/mediawiki/2015/7/7b/STHLM_Spheroplast_protocol.pdf">spheroplast protocol</A>, whereas the other halves was left untreated. Fluorescence analysis using plate reader was then performed on all the samples at 4°C (temperature at which spheroplasts should be the most stable).  The excitation wavelength used was set at 488 nm and the emission wavelength at 509 nm. </P><P><B>Results:</B> In Figure 7, even if the spheroplast samples seem to be less fluorescent, the values are statistically not significantly different. Furthermore, this graph does not enable to draw any conclusions. Both spheroplasts and non-spheroplasts do not seem to be effectively induced which could be explained by a non functional strain. We were additionally unable to assess neither the presence nor the viability of the spheroplast. </P><DIV class="figure-text"><P>Figure 7: Difference in fluorescence between induced and non-induced samples in both spheroplast and non spheroplast <I>E. coli</I>. ns : P &gt; 0.05 (not significant).</P></DIV><P><B>Conclusion for hypothesis 5:</B> We did not manage to prove that spheroplasts are able to express proteins and a readout signal and cannot conclude at this point as the data was inconclusive. Further experiments should be made such as gram-staining in order to determine the presence or not of spheroplasts. 
Contact person: Manon Ricard, <A href="mailto:manon.ricard@yahoo.fr">manon.ricard@yahoo.fr</A></P><H4>HYPOTHESIS 6: A construct of the EnvZ-Affibody chimera protein phosphorylates OmpR when it has bound the HER2 protein</H4><P>Due to time restraints this part of the project was never reached and focus was instead put on the parts of the project which were showing more promising results. Contact person: Felix Richter, <A href="mailto:felix.richter@stud.ki.se">felix.richter@stud.ki.se</A></P><H2 id="signaling">Intracellular signal transduction</H2><H4>HYPOTHESIS 7: Expression of OmpR-Regulated-RFP leads to OmpR dependent production of red fluorescent protein (RFP)</H4><P>After activation of the BAR receptor, OmpR gets phosphorylated. Our first goal was to express RFP which is OmpR regulated. There was already a BioBrick (BBa_M30011) in the iGEM distribution kit that we used to fulfil our goal. As EnvZ deficient strain (BW25113 strain) does not contain EnvZ, it will not have any intracellular interference of endogenous OmpR activation with that of OmpR-Regulated-RFP. In high osmolarity, EnvZ exhibits kinase activity, leading to enormous amount of phosphorylated OmpR inside the cell which activates transcription of RFP containing gene in OmpR-Regulated-RFP.</P><TABLE class="table table-hover hypotheses-table"><THEAD><TR><TH class="col-md-1 table-head">#</TH><TH class="col-md-7 table-head">Experiments for hypothesis 7</TH><TH class="col-md-2 table-head">Status</TH></TR></THEAD><TBODY><TR><TD class="col-md-1 ">1</TD><TD class="col-md-7 "> Can we express RFP in Top10 and BW25113 <I>E. coli</I> bacteria?</TD><TD class="col-md-2">Positive</TD><TD class="col-md-2"><A href="#hyp7exp1" type="button" class="btn btn-default btn-sm">
Go to experiment
</A></TD></TR><TR><TD class="col-md-1 ">2</TD><TD class="col-md-7 ">Is OmpR-Regulated-RFP actually OmpR dependent?</TD><TD class="col-md-2 ">Positive</TD><TD class="col-md-2"><A href="#hyp7exp2" type="button" class="btn btn-default btn-sm">
Go to experiment
</A></TD></TR><TR><TD class="col-md-1 ">3</TD><TD class="col-md-7 ">Can we characterize OmpR-Regulated-RFP even further?</TD><TD class="col-md-2 ">Positive</TD><TD class="col-md-2"><A href="#hyp7exp3" type="button" class="btn btn-default btn-sm">
Go to experiment
</A></TD></TR></TBODY></TABLE><H4>Experiment 1: Can we express RFP in Top10 and BW25113 <I>E. coli</I> bacteria?</H4><P><B>Method:</B> TOP10 and BW25113 (EnvZ deficient strain) chemo-competent <I>E. coli</I> bacteria were transformed with BBa_M30011 (See <A href="https://static.igem.org/mediawiki/2015/c/cc/STHLM_Transformation_protocol.pdf">Transformation protocol </A>) after extraction from the iGEM distribution kit.</P><P><B>Results:</B> RFP was successfully expressed in both the strains. Figure 8 shows red colonies with BBa_M30011 in BW25113 strain.</P><DIV class="figure-text"><P>Figure 8: BW25113 <I>E. coli</I> transformed with OmpR-Regulated-RFP (BBa_M30011).</P></DIV><H4>Experiment 2: Is OmpR-Regulated-RFP actually OmpR dependent?</H4><P><B>Method:</B> Osmolarity test was performed in triplicates, first in TOP10 <I>E. coli</I> containing BBa_M30011 followed by in the BW25113 <I>E. coli</I> strain. Bacteria were grown in minimal media with four different concentrations of sucrose (0%, 5%, 10% and 15%). RFP was then measured in the fluorescence plate reader following the <A href="https://static.igem.org/mediawiki/2015/9/91/STHLM_Osmolarity_protocol.pdf">osmolarity protocol</A>. An osmolarity experiment setup is shown in Figure 9.</P><DIV class="figure-text"><P>Figure 9: Preparation of osmolarity experiment with the positive (BBa_J04450) and the negative controls (BBa_K1766005) to the right top and the right bottom, respectively.</P></DIV><P><B>Results:</B> Figure 10 depicts osmolarity dependence of OmpR-Regulated-RFP BioBrick. TOP10 bacteria had increasing order of red fluorescence (in terms of fold change) depending on the increasing osmolarity level whereas BW25113 showed minor change in fluorescence intensity as expected.</P><DIV class="figure-text"><P>Figure 10: Fluorescence of TOP10 and BW25113 transformed with BBa_M30011 and cultured at different osmolarities. Statistical analysis was done by Student’s t-test and standard error bars of fold changes are shown in the bar diagram. Changes in the fluorescence intensities in 5% and 15% were significant in TOP10, and that of 10% and 15% were significant in BW25113 when compared to 0%. ‘*’ signifies significant P value. ns : P &gt; 0.05 (not significant), ‘*’ : P ≤ 0.05, ‘**’ : P ≤ 0.01, ‘***’ : P ≤ 0.001, ‘****’ : P ≤ 0.0001.</P></DIV><P>These results together show that the BioBrick is OmpR controlled and osmolarity dependent.</P><H4>Experiment 3: Can we characterize OmpR-Regulated-RFP even further?</H4><P><B>Method:</B> The same BioBrick (BBa_M30011) was cloned in the low copy number plasmid (pSB3C5) on Roger Draheim's advice. This was done to have better and more detailed characterization of this part. This time the osmolarity test was designed differently. Here, TOP10 <I>E. coli</I> bacteria were grown in four replicates in special osmo media (LB + Sucrose)  (See <A href="https://static.igem.org/mediawiki/2015/0/05/STHLM_Alternative_Osmolarity_protocol.pdf">Alternative osmolarity protocol</A>) instead of minimal media.</P><P><B>Results:</B> The Y axis in Figure 11 shows average fluorescence/OD instead of fluorescence fold changes. These are compiled data from the Experiment 2 (BBa_M30011 in pSB1C3 in TOP10 strain) and Experiment 3 (BBa_M30011 in pSB3C5 in TOP10 strain). The standard deviation between the samples was much smaller in low copy number plasmid. It can be explained by the use of new osmo media where the cells were not stressed. The fluorescence changes were more consistent in LB osmo media than the minimal media where the cells were under more stress. OmpR dependent change in the low copy number plasmid is less pronounced than the conventional one (pSB1C3) in this Figure. Possible explanation could be due to the composition stoichiometry in the cell . 
The absolute fluorescence was lower in pSB3C5 as expected. However, fluorescence fold change (compared to 0%) remained roughly the same in pSB3C5 and pSB1C3. </P><DIV class="figure-text"><P>Figure 11: Comparison between BBa_M30011 fluorescence in high (pSB1C3) and low (pSB3C5) copy number plasmid.‘*’ signifies significant P value. ns : P &gt; 0.05 (not significant), ‘*’ : P ≤ 0.05, ‘**’ : P ≤ 0.01, ‘***’ : P ≤ 0.001, ‘****’ : P ≤ 0.0001.</P></DIV><H4>Conclusion for hypothesis 7</H4><P>OmpR-Regulated-RFP was expressed in <I>E. coli</I> and it is characterized comprehensively. It was proven to be osmolarity dependent. The results are more stable and reliable when cultured in LB media and in low copy number plasmid. Contact person: Utsa Karmakar, <A href="mailto:utsa.karmakar@stud.ki.se">utsa.karmakar@stud.ki.se</A></P><H4>HYPOTHESIS 8: Expression of MicF-Regulated-GFP leads to constitutive expression of green fluorescent protein (GFP) which can be silenced with MicF RNA</H4><P>MicF RNA has been shown to regulate post-transcriptional expression of the ompF gene. The micF gene encodes an antisense RNA which binds to its target region in the ompF gene. This leads to inhibition of translation . Taking this fact into consideration, we tried to incorporate the micF target (micF-T) and GFP in one plasmid in order to express micF regulated GFP. </P><TABLE class="table table-hover hypotheses-table"><THEAD><TR><TH class="col-md-1 table-head">#</TH><TH class="col-md-7 table-head">Experiments for hypothesis 8</TH><TH class="col-md-2 table-head">Status</TH></TR></THEAD><TBODY><TR><TD class="col-md-1 ">1</TD><TD class="col-md-7 "> Can MicF-Regulated-GFP 1 express GFP in an osmolarity dependent manner?</TD><TD class="col-md-2">Negative</TD><TD class="col-md-2"><A href="#hyp8exp1" type="button" class="btn btn-default btn-sm">
Go to experiment
</A></TD></TR><TR><TD class="col-md-1 ">2</TD><TD class="col-md-7 ">Can MicF-Regulated-GFP 2 express GFP in an osmolarity dependent manner?</TD><TD class="col-md-2 ">Unfinished</TD><TD class="col-md-2"><A href="#hyp8exp2" type="button" class="btn btn-default btn-sm">
Go to experiment
</A></TD></TR></TBODY></TABLE><H4>Experiment 1: Can MicF-Regulated-GFP 1 express GFP in an osmolarity dependent manner?</H4><P><B>Method:</B> Osmolarity tests were performed on TOP10 <I>E. coli</I> transformed with MicF-Regulated-GFP 1, according to the <A href="https://static.igem.org/mediawiki/2015/0/05/STHLM_Alternative_Osmolarity_protocol.pdf">alternative osmolarity protocol</A>. This was done to characterize the osmolarity dependence of the construct’s GFP production i.e. the micF inhibitory effect.  <I>E. coli</I> transformed with BBa_K608011 was used as a positive control. BBa_K1766005 was used as a negative control.</P><P><B>Results:</B> Sequencing showed successful cloning of MicF-Regulated-GFP 1. Osmolarity testing showed that no significant fluorescence was achieved (See Figure 12).</P><DIV class="figure-text"><P> Figure 12: Relative fluorescence of <I>E. coli</I> transformed with MicF-Regulated-GFP 1 cultured at different osmolarities. ns : P &gt; 0.05 (not significant). </P></DIV><P><B>Conclusion: </B>A conclusive result that GFP is not produced made us believe that the design of the construct was faulty. After analyzing the gene we found a reading frame within the micF target gene which ran into two stop codons before reaching the GFP reading frame. We decided to try to make a fusion protein instead with the micF target included in the GFP.</P><H4>Experiment 2: Can MicF-Regulated-GFP 2 express GFP in an osmolarity dependent manner?</H4><P><B>Method:</B> Figure 13 shows the illustration of MicF-regulated-GFP 2. To create this construct a site directed mutagenesis was performed on MicF-regulated-GFP 1. Mutagenesis primers were designed and ordered from Integrated DNA Technologies according to  with the scars removed and the restriction site of BsmI introduced so that screening could be performed with restriction analysis.  The <A href="https://static.igem.org/mediawiki/2015/1/12/STHLM_SiteDirected_mut_protocol.pdf">Site-directed mutagenesis protocol</A> was made based on the product sheet recommendation on the polymerase used. The selection was enabled by the fact that the cells with the new construct had visible fluorescence.  </P><DIV class="figure-text"><P> Figure 13: Plasmid illustration of repaired MicF-Regulated-GFP 2 </P></DIV><P><B>Results: </B> All picked colonies after transformation with the mutagenesis product showed positive result on the restriction analysis and confirmation was made by sequencing. Fluorescence was visible under UV light.</P><P><B>Conclusion for hypothesis 8:</B> The only conclusion made is that the new construct produces fluorescence.
Next step: To test if the MicF-Regulated-GFP 2 construct actually works as intended, with the micF dependent inhibition, an osmolarity test would be conducted. The test would be performed on TOP10 <I>E. coli</I> as well as on the EnvZ deficient strain BW25113. Both strains would be  transformed with the MicF-Regulated-GFP 2 fusion protein in a low copy number plasmid. It could also be combined on the same plasmid with the micF gene which is ompR regulated.  Contact person: Linnea Österberg, <A href="mailto:linneaos@kth.se">linneaos@kth.se</A></P><H4>HYPOTHESIS 9: Expression of OmpR-Regulated-GFP/RFP leads to OmpR dependent regulation of RFP/GFP production</H4><P><B>Introduction: </B>This is the final part of the fluorescence signaling system. This part is created in order to investigate both the effects of phosphorylated and dephosphorylated OmpR. The main idea is to introduce micF RNA in a plasmid, together with OmpR-Regulated-RFP and MicF-Regulated-GFP. In high intracellular osmolarity condition, micF binds to micF-T in MicF-Regulated-GFP and silences production of GFP (Red colonies are pronounced here). In contrast, GFP is pronounced in low osmolarity when micF is not binding to micF-T. Thus, it makes the whole system sensitive to both kinase and phosphatase activity in the BAR.</P><P><B>Methods: </B>Osmolarity tests should be performed according to the <A href="https://static.igem.org/mediawiki/2015/0/05/STHLM_Alternative_Osmolarity_protocol.pdf">Alternative osmolarity protocol</A> to characterize the construct’s GFP and RFP's osmolality dependence. We want to see if the GFP production is inhibited by the OmpR dependent micF and if the RFP production is induced by phosphorylated OmpR.</P><P><B>Results: </B>MicF was also cloned together with the OmpR dependent promoter in pSB1C3. The part  was confirmed by sequencing but not characterized. The rest of the construction on this part was not attempted due to lack of time. Thus, its characterization is left for future study.</P><P><B>Conclusion for hypothesis 9: </B>No conclusions could be drawn as we could not perform any experiment to assemble the parts. Contact person: Utsa Karmakar, <A href="mailto:utsa.karmakar@stud.ki.se">utsa.karmakar@stud.ki.se</A></P><H4>HYPOTHESIS 10: Expression of OmpR-Regulated-RhII leads to OmpR dependent expression of quorum sensing molecule BHL</H4><P><B>Introduction: </B>RhlI is a quorum synthase which produces the quorum sensing molecule N-butyl-homoserine lactone (BHL). In this construct RhlI is under the control of an OmpR dependent promoter. OmpR is activated when it is phosphorylated by the EnvZ receptor. In wild type <I>E. coli</I> this will occur at high osmolarity conditions but not at low osmolarity. To show that the OmpR-Regulated-RhlI BioBrick (BBa_K1766005) works as expected we wanted to prove that it produces BHL in an OmpR dependent manner.</P><P><B>Method: </B> To show that expression of OmpR-Regulated-RhlI is controlled by OmpR we performed an osmolarity bioassay. We used <I>Chromobacterium violaceum</I> in soft agar as a BHL reporter. In the presence of BHL the <I>C. violaceum</I> lab strain CV026 produces a  violet pigment called violacein.  
<I>E. coli</I> transformed with OmpR-Regulated-RhlI were cultured overnight in high and low osmolarity media. The cultures or supernatants  were then applied to separate wells on the bioassay plates.  <I>E. coli</I> transformed with RhlI generator (BBa_K082035) were used as a positive control. Untransformed <I>E. coli</I> were used as a negative control (See <A href="https://static.igem.org/mediawiki/2015/7/75/STHLM_AHL_Osmolarity_protocol.pdf">AHL osmolarity dependence protocol</A>).</P><P><B>Results for supernatant plates: </B>On all plates, weak purple pigmentation could be seen around one well. This well contained supernatant from high osmolarity cultures of OmpR-Regulated-RhlI. No purple pigmentation could be seen around the other wells (as shown in Figure 14).</P><DIV class="figure-text"><P>Figure 14: Bacterial culture supernatant applied to  plates containing <I>C. violaceum</I> in soft agar.</P></DIV><P><B>Results for bacterial culture plates: </B>Purple pigmentation could be seen around all wells except the negative control. There was visibly less pigmentation  around the positive control compared to the OmpR-Regulated-RhlI samples, shown in Figure 15. The diameter of the purple pigmentation  around the high and the low osmolarity samples was measured and compared. The high osmolarity samples, on average, had a 23% bigger radius than the low osmolarity samples (as depicted in Figure 16).
Statistical analysis was performed by Student's t-test on the average of the biological replicates. The total number of biological and technical replicates were three and four, respectively. The obtained P value was 0.00053 which is statistically significant.</P><DIV class="figure-text"><P>Left: Figure 15: Bacterial cultures applied to  plates containing <I>C. violaceum</I> in soft agar. Right: Figure 16: Violacein induction in <I>C. violaceum</I>.  Comparison between <I>E. coli</I> transformed with OmpR-Regulated-RhlI, cultured at high and low osmolarity. ‘*’ signifies significant P value. ns : P &gt; 0.05 (not significant), ‘*’ : P ≤ 0.05, ‘**’ : P ≤ 0.01, ‘***’ : P ≤ 0.001.</P></DIV><P><B>Conclusion for hypothesis 10:</B>OmpR-Regulated-RhlI expresses functional RhlI and synthesizes BHL in an OmpR dependent manner.
To characterize the construct further a fluorescent reporter strain could be used.  Contact person: Sarah Wideman, <A href="mailto:sarah.wideman@stud.ki.se">sarah.wideman@stud.ki.se</A></P><H4>HYPOTHESIS 11: Expression of MicF-Regulated-LuxI leads to constitutive expression of quorum sensing molecule OHHL which can be silenced with micF RNA.</H4><P><B>Introduction: </B>LuxI is a quorum synthase which produces 3-oxo-hexanoyl homoserine lactones (OHHL). In this construct LuxI has been fused to the ompF 5’UTR. This is the target for micF regulatory RNA. In wild type <I>E. coli</I> micF will inhibit translation of LuxI at high osmolarity conditions. We therefore expect MicF-Regulated-LuxI to produce more OHHL at low osmolarity conditions than at high osmolarity.</P><P><B>Methods: </B>To characterize MicF-Regulated-LuxI we intended to perform osmolarity tests together with an OHHL reporter strain.</P><P><B>Results: </B>No results were obtained.</P><P><B>Conclusion for hypothesis 11:</B> Due to time limitations we were unable to construct this BioBrick and could not perform the necessary experiments to prove our hypothesis. Thus, it is left for future study.  Contact person: Linnea Österberg, <A href="mailto:linneaos@kth.se">linneaos@kth.se</A></P><H4>HYPOTHESIS 12: Expression of OmpR-Regulated-RhlI/LuxI leads to OmpR dependent regulation of BHL/OHHL production</H4><P><B>Introduction: </B>Our idea was to build this part as our final product in the signaling pathway. The theoretical goal was to have more production of BHL in high osmolarity condition and more OHHL production in low osmolarity condition in the same bacteria, similar to OmpR-Regulated-GFP/RFP.</P><P><B>Results: </B>No results were obtained.</P><P><B>Conclusion for hypothesis 12: </B>Due to limited time of the project, we could not perform the necessary experiments to prove our hypothesis. Thus, it is left for future study. Contact person: Sarah Wideman, <A href="mailto:sarah.wideman@stud.ki.se">sarah.wideman@stud.ki.se</A></P><H2 id="readout">Read-out from the assay – a second detector strain</H2><H4>HYPOTHESIS 13: In the presence of N-butyl-homoserine lactone (BHL), BBa_K1157006 expresses mCherry (RFP)</H4><TABLE class="table table-hover hypotheses-table"><THEAD><TR><TH class="col-md-1 table-head">#</TH><TH class="col-md-7 table-head">Experiments for hypothesis 13</TH><TH class="col-md-2 table-head">Status</TH></TR></THEAD><TBODY><TR><TD class="col-md-1 ">1</TD><TD class="col-md-7 "> We have an easily accessible source of BHL
</TD><TD class="col-md-2">Negative</TD><TD class="col-md-2"><A href="#hyp13exp1" type="button" class="btn btn-default btn-sm">
Go to experiment
</A></TD></TR><TR><TD class="col-md-1 ">2</TD><TD class="col-md-7 ">Testing the functionality of BBa_K082035 </TD><TD class="col-md-2 ">Positive</TD><TD class="col-md-2"><A href="#hyp13exp2" type="button" class="btn btn-default btn-sm">
Go to experiment
</A></TD></TR><TR><TD class="col-md-1 ">3</TD><TD class="col-md-7 ">Testing the functionality of BBa_K1157006 </TD><TD class="col-md-2 ">Inconclusive</TD><TD class="col-md-2"><A href="#hyp13exp3" type="button" class="btn btn-default btn-sm">
Go to experiment
</A></TD></TR><TR><TD class="col-md-1 ">4</TD><TD class="col-md-7 ">Characterizing BBa_K1157006 </TD><TD class="col-md-2 ">Negative</TD><TD class="col-md-2"><A href="#hyp13exp4" type="button" class="btn btn-default btn-sm">
Go to experiment
</A></TD></TR></TBODY></TABLE><H4>Experiment 1: We have an easily accessible source of BHL</H4><P><B>Introduction:</B> Known sources of BHL were required in order to test functionality of BBa_1157006. Research was focused to either design or find a previous BioBrick which produces BHL in order to have an established and easily accessible source of BHL. A BioBrick for this purpose was found in the registry and we decided to clone it ourselves according to the <A href="https://static.igem.org/mediawiki/2015/f/f6/STHLM_3A_assembly_protocol.pdf">3A assembly protocol</A> from parts already present in the 2015 kit [LINK]. 

</P><P><B>Method:</B>  Promoter, RBS, RhlI, and double terminator BioBricks were transformed in TOP10 <I>E. coli </I> using the <A href="https://static.igem.org/mediawiki/2015/c/cc/STHLM_Transformation_protocol.pdf">transformation protocol</A> and were supposed to be assembled using the <A href="https://static.igem.org/mediawiki/2015/f/f6/STHLM_3A_assembly_protocol.pdf">3A assembly protocol</A>.</P><P><B>Results:</B> During the initial lab weeks there were some trouble with transformation, later addressed to not using KCM buffer. The BBa_K082035 was ordered from IDT to speed up the process after repeated failed transformations and transformation troubleshooting were done in parallel.</P><P><B>Conclusion:</B> BBa_K082035 was ordered instead of cloned.</P><H4>Experiment 2: Testing the functionality of BBa_K082035 </H4><P><B>Introduction:</B> To test if BBa_K082035 works properly, <I>Chromobacterium violaceum</I> was acquired from one of our supervisors (Ute Römling) labs, as <I>C. violaceum</I> changes color to violet when exposed to different AHLs.  </P><P><B>Method:</B><I>E. coli</I> transformed with BBa_K082035 were grown together with <I>C. violaceum</I> on normal agar plates without antibiotics. The bacteria were spread according to Figure 17.</P><P><B>Results:</B> The plates showed a slight violet color change shown in Figure 17. </P><DIV class="figure-text"><P>Figure 17: Plates showing <I>E. coli</I> with BBa_K082035 spread in horizontal streaks <I>C. violaceum</I> spread as an M over the <I>E. coli</I>. The <I>C. violaceum</I> shows a slight violet color change.</P></DIV><P><B>Conclusion:</B> Only two plates were prepared which may not be enough to draw any conclusions from, however, later experiments done by the signalling team have shown that BBa_K082035 works as intended.</P><H4>Experiment 3: Testing the functionality of BBa_K1157006</H4><P><B>Method:</B><I>E. coli</I> transformed with BBa_K1157006 were grown together with bacteria transformed with BBa_K082035 on normal agar plates without antibiotics. The bacteria were spread in the same way as in experiment 2.</P><P><B>Results:</B> The plates showed a slight red color change showed in Figure 18. </P><DIV class="figure-text"><P>Figure 18: Plate showing <I>E. coli</I> with BBa_K082035 spread in horizontal streaks <I>E. coli</I> with BBa_1157006 spread as an M on top. A slight red color change is seen, as expected.</P></DIV><P><B>Conclusion:</B> Only two plates were prepared which may not be enough to draw any conclusions from, even though the two plates showed the correct color change. The results were however not reproducible at a later stage.</P><H4>Experiment 4: Characterizing BBa_K1157006</H4><P><B>Introduction:</B> Along with the recombinant/enzymatic production of BHL, more controlled levels of the molecule were desired in order to characterize the system. The molecule was therefore ordered as a purified solid and stock solutions with concentrations of 10 M in DMSO was prepared.</P><P><B>Method:</B> For each experiment fresh solutions in was prepared in ddH2O in the following concentrations: 100 mM; 10 mM; 0.1 mM; 0.01 mM. We tested these different concentrations in 3 repetitions first in agar plates with the bacteria with BBa_K1157006. Then we tested these different concentrations in the liquid LB cultures with 3 repetitions for each sample. Both plates and liquid cultures were incubated in 37°C for 24h.</P><P><B>Results:</B> No color change was observed for any of the experiments. It seems that our bacterial strain did not respond to any concentrations of BHL in agar or liquid cultures. </P><P><B>Conclusion for hypothesis 13:</B> Based on our results, this part of our experiment did not work as intended. Due to the negative result of our experiment with pure BHL and lack of the time, we decided not to follow up our research in this direction. If there was more time we would however try to optimize this part of the research to obtain a good protocol. After optimization we would try to optimize this in the bacterial strain obtained from the signalling team. 
 Contact person: Maximilian Karlander, <A href="mailto:mkarla@kth.se">mkarla@kth.se</A></P><H4>HYPOTHESIS 14: In the presence of 3-oxo-hexanoyl homoserine lactone (OHHL), T9002 expresses GFP</H4><TABLE class="table table-hover hypotheses-table"><THEAD><TR><TH class="col-md-1 table-head">#</TH><TH class="col-md-7 table-head">Experiments for hypothesis 14</TH><TH class="col-md-2 table-head">Status</TH></TR></THEAD><TBODY><TR><TD class="col-md-1 ">1</TD><TD class="col-md-7 "> Easily accessible source of OHHL</TD><TD class="col-md-2">Negative</TD><TD class="col-md-2"><A href="#hyp14exp1" type="button" class="btn btn-default btn-sm">
Go to experiment
</A></TD></TR><TR><TD class="col-md-1 ">2</TD><TD class="col-md-7 ">Testing the functionality of BBa_K082029  </TD><TD class="col-md-2 ">Not started</TD><TD class="col-md-2"><A href="#hyp14exp2" type="button" class="btn btn-default btn-sm">
Go to experiment
</A></TD></TR><TR><TD class="col-md-1 ">3</TD><TD class="col-md-7 ">Characterizing BBa_T9002 </TD><TD class="col-md-2 ">Negative</TD><TD class="col-md-2"><A href="#hyp14exp3" type="button" class="btn btn-default btn-sm">
Go to experiment
</A></TD></TR></TBODY></TABLE><H4>Experiment 1: Easily accessible source of OHHL</H4><P><B>Introduction:</B>  Known sources of OHHL were also required. Research was focused to either design or find a previous BioBrick which produces OHHL in order to have an established and easily accessible source of OHHL. A BioBrick for this purpose was found in the registry but it was decided to try to clone it ourselves according to the iGEM <A href="https://static.igem.org/mediawiki/2015/f/f6/STHLM_3A_assembly_protocol.pdf">3A assembly protocol</A> from parts already present in the 2015 kit. 

</P><P><B>Method:</B> Promoter, RBS, RhlI, and double terminator BioBricks were transformed and were supposed to be assembled using the <A href="https://static.igem.org/mediawiki/2015/f/f6/STHLM_3A_assembly_protocol.pdf">3A assembly protocol</A>.</P><P><B>Results:</B> The cloning did not work as intended for a week of trials (restriction analysis showed an empty plasmid with neither reporter/RFP nor expected clone. This prompted us to once again order the BioBrick instead. Unfortunately, the ordered BioBrick didn’t grow at all in any antibiotic.</P><P><B>Conclusion:</B> Several attempts of getting the received bacteria to grow were made without a single colony. Either the received bacteria had died, or the received agar didn’t contain any bacteria with the BioBrick to begin with. </P><H4>Experiment 2: Testing the functionality of BBa_K082029 </H4><P><B>Introduction:</B>  To test if BBa_K082029 works properly, yet again the <I>C. violaceum</I> was thought to be used. Due to the problems with BBa_K082029 this was not done as intended.</P><P><B>Method:</B> The method intended to be used was the same as for experiment one for hypothesis 13.</P><P><B>Results:</B> No results were achieved from this experiment.</P><P><B>Conclusion:</B> No conclusions can be drawn. </P><H4>Experiment 3: Characterizing BBa_T9002</H4><P><B>Introduction:</B>  Along with the recombinant/enzymatic production of OHHL, more controlled levels of the molecule were desired in order to characterize the system. The molecule was therefore ordered as a purified solid and stock solutions with concentrations of 10 M in DMSO was prepared.</P><P><B>Method:</B> For each experiment fresh solutions in was prepared in ddH2O in the following concentrations: 100 mM; 10 mM; 0.1 mM; 0.01 mM. We tested these different concentrations in 3 repetitions first in agar plates with the bacteria with BBa_T9002. Then we tested these different concentrations in the liquid LB cultures with 3 repetitions for each sample. Both plates and liquid cultures were incubated in 37°C for 24h.</P><P><B>Results:</B> No color change was observed for any of the experiments. It seems that our bacterial strain did not respond to any concentrations of OHHL in agar or liquid cultures.</P><P><B>Conclusion of hypothesis 14:</B> Based on our results, this part of our experiment did not work as intended. Due to the negative result of our experiment with pure OHHL and lack of the time, we decided not to follow up our research in this direction. If there was more time we would however try to optimize this part of the research to obtain a good protocol. After optimization we would try to optimize this in the bacterial strain obtained from the signalling team.  Contact person: Karol Kugiejko, <A href="mailto:karol.kugiejko@stud.ki.se">karol.kugiejko@stud.ki.se</A></P><H4>HYPOTHESIS 15: Expression of the different quorum sensing molecules from the biomarker recognizing bacteria is enough to start the expression of the fluorescent proteins</H4><P>Due to time restraints this part of the project was never reached and focus was instead put on the parts of the project which were showing more promising results. Contact person: Maximilian Karlander, <A href="mailto:mkarla@kth.se">mkarla@kth.se</A></P><H4>References</H4><TABLE class="table table-condensed reference-table"><THEAD><TR><TH class="col-md-1 table-head">#</TH><TH class="col-md-10 table-head">Reference</TH></TR></THEAD><TBODY><TR><TD class="col-md-1 ">1</TD><TD class="col-md-10 "><A href="http://www.sciencedirect.com/science/article/pii/S1096717600901617">
Jones, K. L., Kim, S. W., &amp; Keasling, J. D. (2000). Low-copy plasmids can perform as well as or better than high-copy plasmids for metabolic engineering of bacteria. Metabolic engineering, 2(4), 328-338.</A></TD></TR><TR><TD class="col-md-1 ">2</TD><TD class="col-md-10 "><A href="http://www.sciencedirect.com/science/article/pii/S0022283601950296">
Delihas, N., &amp; Forst, S. (2001). MicF: an antisense RNA gene involved in response of Escherichia coli to global stress factors. Journal of molecular biology, 313(1), 1-12.</A></TD></TR><TR><TD class="col-md-1 ">3</TD><TD class="col-md-10 "><A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2762917/">
Laible, M., &amp; Boonrod, K. (2009). Homemade site directed mutagenesis of whole plasmids. Journal of visualized experiments: JoVE, (27).</A></TD></TR></TBODY></TABLE></DIV><FOOTER><DIV class="footer-container"><DIV class="row title-row"><DIV class="col-md-10 col-sm-10 col-xs-10 footer-title">iGEM Stockholm is made possible with support from:</DIV></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></BODY></HTML>