<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Bulgaria_DESIGN skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Bulgaria/DESIGN</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="quickfind"><BUTTON class="dropbtn">LEARN MORE</BUTTON><DIV class="quickfind-content"><A href="#WHOUSE">PROJECT USE</A><A href="#SAFE">IS IT SAFE</A><A href="#USEDSTRAINS">USED ORGANISM STRAINS</A><A href="#RISKS">POTENTIAL RISKS</A><A href="#WASTE">WASTE TREATMENT</A><A href="#EMERGENCY">EMERGENCY REACTIONS</A><A href="#EQUIPMENT">PROTECTIVE EQUIPMENT</A><A href="#OTHERS">OTHER RULES</A></DIV></DIV><BUTTON onclick="topFunction()" id="myBtn" title="Go to top">Top</BUTTON><DIV class="contentHolder" id="topTitle"><A id="pagetitle">PROJECT DESIGN</A></DIV></DIV><DIV class="contentHolder" id="PROJECTDESIGN"><H1>PRINCIPLES</H1><P>
Our main goal was to create a novel CRISPR-based system that can detect disease
causing mutations without the use of DNA sequencing. We focused our attention on two
different approaches: </P><P><STRONG>1) CRISPR-Cas9 – RFLP: </STRONG>The idea is to use the wild type CRISPR-Cas9 system with
appropriate gRNAs. These complexes can cut in vitro PCR products only if a given
mutation is present. The output can be visualized on a simple agarose gel without the need
of sequencing. </P><P><STRONG>2) Paired CRISPR-dCas9 sensors: </STRONG>Split fragments from a reporter protein can be fused to
a pair of dCas9 enzymes. If both inactivated nucleases recognize their specific targets, the
split protein fragments will be in close enough proximity to each other, re-assembling the
reporter molecule. In this case, our system will generate a specific output signal –
fluorescence, luminescence or a colorful enzymatic reaction. </P><P>
In order for it to be time and cost efficient, we focused our efforts on approach one.
The design of paired dCas9 sensors will, hopefully, be completed next year due to severe
financial limitations.
</P></DIV><DIV class="contentHolder" id="PROJECTDESIGN"><H1>STEP-BY-STEP EXPLANATION OF OUR DESIGN</H1><P><STRONG>Selection of model disease: </STRONG>We selected cystic fibrosis as a model disease for our
system, since it is the most common genetic condition in Bulgaria and Europe as a whole.
Many Bulgarian patients that live in remote regions and/or small towns and villages are
diagnosed far later, compared to other European countries. In the light of these facts, a
novel system for molecular diagnostics of cystic fibrosis will prove incredibly useful.
Moreover, a single mutation (del F508) in the CFTR gene is responsible for approximately
70% of the cases. This variant affects three nucleotides located right next to a NGG region
that can serve as a PAM site for different CRISPR-Cas9 systems. This makes the del F508
mutation the best candidate for our system. </P><P><STRONG>Design of gRNAs: </STRONG>The specific location of the delta F508 mutation allowed us to design
two different gRNAs: one that targets the wild type locus and another to recognize the
mutated DNA sequence. Both molecules differ in 3 nucleotides located just next to the
PAM region. According to literature, differences in this region have severe effect on the
ability of Cas9 to recognize and cut its targets. </P><P><STRONG>Design of gRNA expression vectors: </STRONG>To test our system in vivo, we cloned both gRNAs
into a pSB1K3-gRNA vector (a kanamycin-resistant version of part BBa_K2515002). Since
the combination of a gRNA targeting a sequence in the E. coli genome and Cas9 is lethal
for these bacteria, we checked both our constructs using a BLAST search against the E.
coli genomic sequence. The gRNA that targets the delta F508 mutation showed no
significant similarities, so we concentrated on it for our in vivo experiments. For in vitro
studies, the off-target effects are not a concern. </P><P><STRONG>Cas9 system: </STRONG>We used a plasmid for Cas9 expression from Addgene. It has a weak
constitutive promoter and a chloramphenicol resistance cassette. Our gRNA construct was
then transferred to the pCas9 strain via the process of transformation, followed by selection
on both antibiotics. </P><P><STRONG>Cas9 chromosome integration system: </STRONG>In order to reduce the total plasmid number, we
created an E. coli strain with a Cas9 expression cassette integrated into its genome. The
Cas9 expression is under the control of an arabinose-induced promoter. The Tn7
transposon-based system was selected specifically for the integration step. </P><P><STRONG>In vivo check of the gRNA specificity: </STRONG>The specificity of the gRNA is critical for the
proper functioning of our del F508 detection system. To experimentally validate this aspect,
we used a gBlock fragment with the corresponding CFTR gene region, containing the
mutation of interest. This part was cloned into our vector, pSB1K3-gRNA, and the resulting
construct was validated via colony PCR. Next, we transferred that plasmid into pCas9
containing competent cells. After a two-hour period of outgrowth at 37 o C, aliquots were
plated on petri dishes with chloramphenicol and chloramphenicol plus kanamycin. The
colony numbers were analyzed on the next day. If the gRNA is active against its target
sequence, one would expect no (or just a few) colonies on the dish with kanamycin, since
Cas9 will eliminate the gRNA-producing vector. If the gRNA cannot recognize the analysed
target, many colonies will grow on the plate, since the transformation step is performed
with a purified super-coiled plasmid. Using this approach, we aimed to design a gRNA
molecule that targets the mutated version of the CTFR gene sequence at one hand, and
cannot recognize the wild type on the other. </P><P><STRONG>In vitro mutation detection: </STRONG>Specific PCR primers were designed to be able to amplify a
part of the CTFR gene (approximately 1 kb in size). An in vitro Cas9 digestion reaction is
performed, using purified Cas9 nuclease protein and a specific gRNA. The results of this
reaction are analyzed on agarose gel electrophoresis.
</P></DIV><DIV class="contentHolder" id="PROJECTDESIGN"><H1>EXPERIMENTAL PLAN TO TEST OUR DESIGN</H1><P><STRONG>Design of gRNAs: </STRONG>Two specific 20-base long gRNAs were manually designed using the
closest possible PAM region, located just next to del F508. They will be analyzed with in
vivo and in vitro experiments. </P><P><STRONG>Cloning of the gRNAs: </STRONG>We used the vector BBa_K2515002, designed by iGEM Bulgaria
2017 team. The gRNA expression cassette was digested with EcoRI and PstI and then
subcloned into a pSB1K3 standard vector to obtain the pSB1K3-gRNA expression system.
Positive clones were verified by colony PCR. </P><P><STRONG>gRNA activity and specificity: </STRONG>These properties were analyzed via the addition of the
targeting region (with and without the mutation) to the gRNA expression vectors. If the
gRNA is active against the given sequence, the plasmid is eliminated when the bacteria
have pCas9 inside them. </P><P><STRONG>In vitro mutation detection: </STRONG>PCR amplified regions from the CTFR gene from patients
with del F508 and healthy controls will be treated with Cas9 + gRNA in vitro. The result can
be monitored on agarose gel.
</P></DIV><DIV class="navigation" id="nav"><A href="https://2018.igem.org/Team:Bulgaria">HOME</A><DIV class="dropdown"><BUTTON class="dropNAVbtn">TEAM 
      </BUTTON><DIV class="dropdown-content"><A href="https://2018.igem.org/Team:Bulgaria/theteam">MEMBERS</A><A href="https://2018.igem.org/Team:Bulgaria/attrib">ATTRIBUTIONS</A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropNAVbtn">PROJECT 
      </BUTTON><DIV class="dropdown-content"><A href="https://2018.igem.org/Team:Bulgaria/theproject">DESCRIPTION</A><A href="https://2018.igem.org/Team:Bulgaria/DESIGN" id="activeMenu">DESIGN</A><A href="https://2018.igem.org/Team:Bulgaria/PROTOCOLS">PROTOCOLS</A><A href="https://2018.igem.org/Team:Bulgaria/LABBOOK">LAB BOOK</A><A href="https://2018.igem.org/Team:Bulgaria/RESULTS">RESULTS</A><A href="https://2018.igem.org/Team:Bulgaria/CONTRIBUTION">CONTRIBUTION</A><A href="https://2018.igem.org/Team:Bulgaria/IMPROVE">IMPROVE</A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropNAVbtn">HUMAN PRACTICES 
      </BUTTON><DIV class="dropdown-content"><A href="https://2018.igem.org/Team:Bulgaria/publicEng">PUBLIC ENGAGEMENTS</A><A href="https://2018.igem.org/Team:Bulgaria/IntHP">INTEGRATED HUMAN PRACTICES</A><A href="https://2018.igem.org/Team:Bulgaria/survey">SURVEY</A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropNAVbtn">COLLABORATIONS
      </BUTTON><DIV class="dropdown-content"><A href="https://2018.igem.org/Team:Bulgaria/COLLABMEETINGS">MEETINGS</A><A href="https://2018.igem.org/Team:Bulgaria/COLLABSURVEYS">SURVEYS</A><A href="https://2018.igem.org/Team:Bulgaria/COLLABOTHERS">OTHERS</A><A href="https://2018.igem.org/Team:Bulgaria/COLLABfollow">#FollowMeToiGEM</A></DIV></DIV><A href="https://2018.igem.org/Team:Bulgaria/parts">PARTS</A><A href="https://2018.igem.org/Team:Bulgaria/safety">SAFETY</A><DIV class="dropdown"><BUTTON class="dropNAVbtn">MEDAL CRITERIA
      </BUTTON><DIV class="dropdown-content"><A href="https://2018.igem.org/Team:Bulgaria/MEDALBRONZE">BRONZE</A><A href="https://2018.igem.org/Team:Bulgaria/MEDALSILVER">SILVER</A><A href="https://2018.igem.org/Team:Bulgaria/MEDALGOLD">GOLD</A><A href="https://2018.igem.org/Team:Bulgaria/MEDALAWARD">SPECIAL AWARD</A></DIV></DIV><A href="https://igem.org/2018_Judging_Form?id=2847" id="specialMenu">JUDGING FORM</A><A href="javascript:void(0);" class="icon" onclick="burgermenu()">
	MENU
  </A></DIV></DIV></DIV></DIV></DIV></BODY></HTML>