Team:Bulgaria/RESULTS
LEARN MORE
PROJECT USE IS IT SAFE USED ORGANISM STRAINS POTENTIAL RISKS WASTE TREATMENT EMERGENCY REACTIONS PROTECTIVE EQUIPMENT OTHER RULES
Top
PROJECT RESULTS
We identified a putative PAM site right next to the del F508 amino acid position in the CTFR gene. Having this in mind, we designed two gRNAs – one that recognizes this mutation and another, that targets the wild-type sequence. To clone them, we first sub- cloned the gRNA expression cassette from part BBa_K2515002 into pSB1K3. Positive clones were identified via colony PCR.
To confirm the functionality of this vector, we cloned a gRNA targeting the mutS gene (involved in DNA repair processes) in it. The gRNA sequence and the oligo pair was kindly provided by the iGEM Bulgaria 2017 team. Next, a positive clone was used for plasmid DNA isolation and re-transformation in E. coli cells, that contain pCas9 vector (it produces wild-type Cas9 protein). No colonies were observed on the petri dishes, which indicates that our vector produces functionally active gRNAs.
After that, we cloned our two gRNAs and isolated positive clones. Both molecules were checked with BLAST search against the E. coli genome. The obtained results indicated that the gRNA targeting the del F508 mutation sequence has no sequence homology to regions in the bacterial genome. We used this observation to design a scheme for in vivo activity and specificity tests. To do that, a CTFR gene fragment (containing the F508 region) was ordered as two gBlocks – one with del F508 and another, with the wild-type sequence. Both of these fragments were cloned into the pSB1K3-gRNA vector with our gRNA, using the standard BioBrick assembly rules.
Positive clones were isolated, then re-transformed into E. coli cells containing a vector that expresses Cas9 under the control of a weak constitutive promoter. After a period of outgrowth, equal aliquots were plated on two dishes – one with chloramphenicol and one with chloramphenicol plus kanamycin. If the gRNA can recognize the target sequencing into the gene fragment, it will eliminate the plasmid that carries it and its antibiotic resistance will, therefore, be lost.
To further improve our setup, we designed a novel BioBrick part, that contains the cer region. This sequence is known to stabilize plasmids due to its interaction with Xer. Both these compounds form a multimer resolution system, that was first identified into the multicopy plasmid ColE1. We cloned this part into pSB1C3 and then tested the stability of the resulting construct via cultivation without an antibiotic and then plating on dishes with chloramphenicol. The cer-containing vector demonstrated improved stability, hence this part being integrated into our system for gRNA test.
Our experiments indicated that the gRNA recognizes the mutated CTFR gene sequence and shows almost no effect on the wild-type sequence. We concluded that this gRNA is promising enough to be tested in in vitro cleavage experiments for CRISPR-RFLP detection of del F508.
We ordered the Cas9 protein and both gRNAs needed for the in vitro test (from IDT). A primer pair that generates PCR products of approximately 1000 bp was designed. This product contains the mutation region and will be used in our experiments. Unfortunately, we needed DNA from a patient with del F508 as a matrix to test our approach. This compound was not provided to us in time for the Giant Jamboree, thus hindering our work for now. We are planning on finishing the experiment next year.
HOME
TEAM
MEMBERS ATTRIBUTIONS
PROJECT
DESCRIPTION DESIGN PROTOCOLS LAB BOOK RESULTS CONTRIBUTION IMPROVE
HUMAN PRACTICES
PUBLIC ENGAGEMENTS INTEGRATED HUMAN PRACTICES SURVEY
COLLABORATIONS
MEETINGS SURVEYS OTHERS #FollowMeToiGEM
PARTS SAFETY
MEDAL CRITERIA
BRONZE SILVER GOLD SPECIAL AWARD
JUDGING FORM MENU
