Team:NAIT Edmonton/Parts
Team NAIT 2015
home
team
project
human practices
notebook
team
bios
attributions
official team profile
project
description
experiment and protocols
parts and results
modeling
human practices
community outreach
policy and practices
collaborations
notebook
lab safety
achievements
log book
Parts and Results
Check out the new parts we made for the registry:
BBa_K1787001 - MBL
Bba_K1787000 - Cys-Thr
Data Analysis
The sequences that were ordered had a design flaw on them. The restriction sites were not on the correct place and were incompatible with pSB1C3. In order to fix them, we decided to create primers that would insert a point mutation in the restriction sites and, at the same time, add the correct iGEM prefix and suffix to the sequences.
A gradient PCR was conducted in order to determine the best annealing temperature (Ta) for the primers. It was observed that Ta=61ºC was optimum. A new PCR comprising all our sequences was done and, after using a PCR Purification kit, an agarose gel showed that out of 23 sequences, we were able to purify 18 (see Figure 2). Nonetheless, one of the sequences contained an illegal restriction site within itself and it was discarded.
The sequences, now containing an iGEM compatible prefix and suffix, needed a vector and a host organism so as to be expressed. T7 expression systems can produce a high copy of a Protein of Interest (POI), provided that the DNA sequence that encodes for the polypeptide is next to a T7 promoter. With regards to the vector, it was noted that BioBrick K1321338 (BBa_ K1321338), found on the iGEM 2015 Kit Plate #5, contained a T7 promoter and a Ribosome Binding Site. And pertaining the host organism, BL21 (DE3), a T7 E.coli expression strain was chosen as the system to synthesize our POIs
After transforming cells with BBa_K1321338, obtaining the plasmid (i.e., pSB1C3 with the BioBrick. Note: pSB1C3 contains a chloramphenicol resistant gene) via a MiniPrep Kit, and confirming its presence by running a sample in an agarose gel, a digestion with SpeI and PstI, and subsequent dephosphorylation was conducted so as to linearize the plasmid and prepare it for ligation to the sequences. Simultaneously, the sequences were digested with XbaI and PstI (see Figure 3).
Ligation was performed using T4 DNA Ligase; subsequently, E. coli BL21 (DE3) was transformed with the ligation products and plated in LB + chloramphenicol agar. 34 plates (two per sequence of interest) were left 24 hours at 37ºC to ensure optimum growth of transformed cells. Grown colonies were accounted for and numbered (see Figure 4).
pSB1C3 Forward and Reverse Primers were used to confirm that the colonies actually contained the sequence. A gradient PCR showed that the optimum Ta was 57ºC, so a colony PCR was made following such parameter and an agarose gel was done to verify outcome of the PCR.
Data showed that, out of three hundred and two colonies, only five contained an insert (see Figure 5). Other colonies contained the plasmid without an insertion. This was concluded because the Forward and Reverse Primers amplify a fragment 339 bp long. The divergent colonies amplified longer fragments and, out of those six, only two had the size that was expected. These were sequences ‘Cys-Thr’ and ‘MLAB’ (see Figure 6, this figure only shows significant gels).
These five positive colonies were cultured in 200mL of LB media with chloramphenicol and induced with IPTG to increase protein expression. The cells were pelleted and proteins were isolated using a NiNTA Kit.  The samples were ran in a SDS-PAGE (see Figure 6). Only the sample containing the protein encoded by sequence ‘Cys-Thr’ showed bands. Additionally, the bands were ~19kDa in size, the expected size for this particular protein.
Preliminary data shows that this engineered protein is able to show a different colouration after silver staining (see Figure 8)
This preliminary data supports our original hypothesis that certain amino acid configurations within a protein alter band colouration post-silver staining.  We are currently looking into new combinations of amino acid motifs in the hopes of generating the colours blue, red, green, purple, orange, and shards of grey.
home
team
project
human practices
notebook
