<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_NAIT_Edmonton_Desc skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:NAIT Edmonton/Desc</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Team NAIT 2015</TITLE></P><DIV class="nav"><DIV tabindex="0" class="onclick-menu"><UL class="onclick-menu-content" style="list-style:none;"><LI><A href="https://2015.igem.org/Team:NAIT_Edmonton">home</A></LI><LI><A href="https://2015.igem.org/Team:NAIT_Edmonton/Team">team</A></LI><LI><A href="https://2015.igem.org/Team:NAIT_Edmonton/Project">project</A></LI><LI><A href="https://2015.igem.org/Team:NAIT_Edmonton/Human">human practices</A></LI><LI><A href="https://2015.igem.org/Team:NAIT_Edmonton/Notebook">notebook</A></LI></UL></DIV><DIV class="navigation"><UL><LI><A href="https://2015.igem.org/Team:NAIT_Edmonton/Team">team</A></LI><LI><A href="https://2015.igem.org/Team:NAIT_Edmonton/Bios">bios</A></LI><LI><A href="https://2015.igem.org/Team:NAIT_Edmonton/Attributions">attributions</A></LI><LI><A href="https://igem.org/Team.cgi?id=1787">official team profile</A></LI></UL><LI><A href="https://2015.igem.org/Team:NAIT_Edmonton/Project">project</A></LI><LI><A href="https://2015.igem.org/Team:NAIT_Edmonton/Desc">description</A></LI><LI><A href="https://2015.igem.org/Team:NAIT_Edmonton/Protocols">experiment and protocols</A></LI><LI><A href="https://2015.igem.org/Team:NAIT_Edmonton/Results">parts and results</A></LI><LI><A href="https://2015.igem.org/Team:NAIT_Edmonton/Modeling">modeling</A></LI><LI><A href="https://2015.igem.org/Team:NAIT_Edmonton/Human">human practices</A></LI><LI><A href="https://2015.igem.org/Team:NAIT_Edmonton/Outreach">community outreach</A></LI><LI><A href="https://2015.igem.org/Team:NAIT_Edmonton/Practices">policy and practices</A></LI><LI><A href="https://2015.igem.org/Team:NAIT_Edmonton/Collaborations">collaborations</A></LI><LI><A href="https://2015.igem.org/Team:NAIT_Edmonton/Notebook">notebook</A></LI><LI><A href="https://2015.igem.org/Team:NAIT_Edmonton/Safety">lab safety</A></LI><LI><A href="https://2015.igem.org/Team:NAIT_Edmonton/Achievements">achievements</A></LI><LI><A href="https://2015.igem.org/Team:NAIT_Edmonton/Logbook">log book</A></LI></DIV></DIV><DIV id="wrap"><DIV class="wrapper"><CENTER><DIV class="top_slogan">Development and Characterization of Protein Motifs to Generate Colours upon Interaction with Silver Staining Reagents</DIV></CENTER><H2><B>Abstract: </B>SDS-PAGE is a very popular technique used to separate proteins based on their size. Embedded proteins, invisible to the naked eye, are then visualized by staining. Among the various staining techniques, silver staining is easy to perform and highly sensitive. However, the outcome is a series of monochromatic protein bands. Previously, we observed that some proteins inherently produce different hues post-staining. We hypothesized that specific amino acid configurations yield coloured bands after reacting with silver staining reagents. To test our hypothesis, we created numerous amino acid motifs to elucidate the sequences that would generate specific colours following silver staining. Our findings will let us generate a molecular weight marker with the innate capacity of providing users colour-coded bands post-staining without the use of impregnating dyes. Our technology will also pave the way for new types of colorimetric assays using synthetic proteins</H2><DIV class="accordion"><DIV class="accordion-section"><A class="accordion-section-title" href="#accordion-1" style="background-color:#f96040">Background</A><DIV id="accordion-1" class="accordion-section-content">
The structural and functional study of the proteins expressed by a genome is 

called proteomics. This relatively novel science uses different methodologies in order to 

separate and identify specific proteins of interest. Among these techniques, SDS-PAGE 

plays an essential role due to its high sensitivity, low sample volume requirement, and 

high popularity. Negatively charged proteins migrate towards the positive electrode 

according to their size and charge. Smaller proteins migrate further in a given amount of 

time. As proteins are separated in this manner, users load molecular weight standards 

to estimate the size (in kDa) of the proteins present in their sample. Once the proteins of 

a single sample have been isolated and are embedded in the polyacrylamide (PA) gel 

matrix, staining procedures are used to visualize them.<P style="font-size:10px">Source: http://upload.wikimedia.org/wikipedia/commons/4/46/SDS-PAGE_Electrophoresis.png</P><P style="font-size:15;">Organic dyes, such as Coomassie blue, can be used for this purpose; 

nevertheless, their low sensitivity and a detection range that goes from 1 to 50 ng can 

be a challenge for detecting low abundance proteins (Jin, Huang, Yoo, &amp; Choi, 2006). A 

higher sensitivity can be achieved by fluorescent staining techniques (from 0.1 to 10 

ng.); however, UV instruments are necessary in order to read the data (Jin et al., 2006). 

The most sensitive method up to date is radiolabeling, but the requirement of hazardous 

isotopes and their complex management makes it a complicated procedure (Jin et al., 

2006). Silver staining is a method that offers great sensitivity and an easy to handle 

protocol, thus making it one of the most commonly used staining methods. </P><P><FONT size="1px">[1] Bio-rad.com, 'Coomassie Stains', 2015. [Online]. Available: http://www.bio-rad.com/webroot/web/images/lsr/products/electrophoresis/product_overlay_content/global/lsr_biosafe_coomasie_gel.jpg. [Accessed: 14- Jun- 2015].</FONT></P></DIV></DIV><DIV class="accordion-section"><A class="accordion-section-title" href="#accordion-2" style="background-color:#FBB252">The Problem</A><DIV id="accordion-2" class="accordion-section-content"><P>However, there are limitations to using the silver staining technique. The primary issue with the technique is that the users lose the ability to use the color coded molecular weight marker as a reference post-staining. To combat this issue, researchers poke holes into the polyacrylamide gel so that they can retain their molecular weight ladder reference points. However, by doing this we ruin the integrity of the gel making the staining process much more likely to damage or destroy the fragile gel. </P><CENTER><P style="font-size:10px">Source:http://labs.mmg.pitt.edu/gjoerup/silver_stain.jpg</P></CENTER><P>Researchers who have used the technique in the past have also expressed concerns on the time it takes to develop the gel and stain; the possibility of “over-staining” the gel, and the generation of false positives due to the sensitivity of the technique. (An example would be the solutions reacting with particles in the air to provide an unneeded background on the gel) </P></DIV></DIV><DIV class="accordion-section"><A class="accordion-section-title" href="#accordion-3" style="background-color: #f7e133">Our Goal and Solution</A><DIV id="accordion-3" class="accordion-section-content"><P>Our goal is to develop a marker that, when interacting with the reagents used in the staining protocol, will stain in colour and in specific positions. This technology will aid in the identification of the protein(s) of interest post-staining. In order to do so, we will investigate how specific amino acids react with silver staining reagents. After determining specific motifs, we aim to create novel proteins that contain an excess of that motif, particular amino acid and/or chemical modifications that will generate a specific color after treating it with silver staining reagents. To obtain such proteins, we will introduce novel nucleotide sequences into a plasmid by in vitro transcription translation. <I>E. coli</I> cells with expression vectors will then be transformed with the new plasmids.</P><P>By testing different techniques to develop and stain the gels we hope to see if the expressed concerns are still an issue, namely time taken to develop the gels and stain them effectively. To investigate this we will use BioRad’s recommended protocol, our protocol developed in-house, and a technique documented in <I>Electrophoresis.</I></P></DIV></DIV><DIV class="accordion-section"><A class="accordion-section-title" href="#accordion-5" style="background-color:#afcf64">Why Does this Matter?</A><DIV id="accordion-5" class="accordion-section-content"><P>With the current technique used today, researchers poke holes into the PA gel so that they can retain their molecular weight ladder reference points. Not only does this take time to do, but it also ruins the integrity of the gel making the staining process much more likely to damage the fragile gel.</P></DIV></DIV></DIV></DIV></DIV><DIV class="footer"><DIV class="footer_content"><DIV class="footer_left"><UL class="footer_menu"><LI><A href="https://2015.igem.org/Team:NAIT_Edmonton">home</A></LI><LI><A href="https://2015.igem.org/Team:NAIT_Edmonton/Team">team</A></LI><LI><A href="https://2015.igem.org/Team:NAIT_Edmonton/Project">project</A></LI><LI><A href="https://2015.igem.org/Team:NAIT_Edmonton/Human">human practices</A></LI><LI><A href="https://2015.igem.org/Team:NAIT_Edmonton/Notebook">notebook</A></LI></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>