"
Page
Discussion
View source
History
teams
Log in
 
Team:KULeuven/Model/Output
From 2008.igem.org
  dock/undock dropdown  
Home
Reports
Calendar
Project abstract
Meet the team
Leave a message
Mail us
The Team
Research Labs and Groups Students Instructors Advisors Pictures
The Project
Summary Components
Input Output Filter InverTimer Reset Cell Death Memory
Safety End Evaluation Literature Brainstorm
Ethics
Introduction Three A's
Defining the three A's Developments in modern physics Developments in biology
Concerns & Issues
Defining synthetic biology A framework Concerns Issues The need for answers
Biological Robotics
KULeuven iGEM 2008 project Biological robotics
Afterword References
Submitted Parts
Sandbox
Modeling
Overview Kinetic Constants Components
Output Filter InverTimer Reset Cell Death Memory
Full Model Sensitivity Analysis Multi-cell Model Diffusion
Data Analysis
Overview New Parts
GFP (LVA-tag) Other
Components
Input Output Other
Software
Multi-cell Toolbox Simbiology2LaTeX Toolbox
Notebook
Protocols Reports
Daily Weekly
Lab Data
Freezer Primers Ligation
Tools Press Acknowledgments
Contents
1 Output
1.1 Position in the system
1.2 Describing the system
1.2.1 ODE's
1.2.2 Parameters
1.3 Models
1.3.1 CellDesigner (SBML file)
1.3.2 Matlab (SBML file)
1.4 Simulations
1.5 Sensitivity Analysis
1.6 References
Output
Position in the system
The output system is one of three modules directly linked to the input. The output system is a simple gene regulation, of which transcription is repressed by TetR - and can be activated by anhydrotetracyclin. The output signal is GFP (green fluorescent protein).
In the applied project of dealing with Crohn's disease this would be replaced by the drug necessary to cure the local inflammation. The drug concentration would be proportional to the amount of local inflammation sensed.
Describing the system
see also: Project:Output
ODE's
Parameters
Parameter values (Output)
Name
Value
Comments
Reference
Degradation Rates
dGFP
dLVA = 2.814E-4 s-1
LVA-tag reduces lifetime to 40 minutes
[1] [4]
dmRNA_GFP
0.0023 s-1
[2]
Transcription Rates
TetR_var_transcr_rate
p(TetR) dependent
(GFP) between 5E-5 and 0.0125 s-1 ~ [aTc]
[3]
Translation Rates
kGFP
0.167 s-1
translation rate for B0032 RBS (0.3 relative efficiency)
[5]
Models
CellDesigner ( SBML file )
Matlab ( SBML file )
Simulations
Several simulations were conducted. The following two show the transient respons and eventual saturation of the output system to a low (left figure) and a high (right figure) input signal (TetR = 5E-5 vs 0.0125 s-1). There is a great distinction in the number of GFP molecules between the two input signals. All graphs have amounts (number of molecules in the cell) plotted vs time, measured in seconds.
The next figure shows the switching behaviour of the output system to an light signal turning on and off:
Sensitivity Analysis
References
Bibliography
[1]
“ETHZ/Parameters - IGEM07”; https://2007.igem.org/ETHZ/Parameters.
 
[2]
J.A. Bernstein et al., “Global analysis of mRNA decay and abundance in Escherichia coli at single-gene resolution using two-color fluorescent DNA microarrays,” Proceedings of the National Academy of Sciences of the United States of America,  vol. 99, Jul. 2002, pp. 9697–9702.
 
[3]
M. Bon, S.J. McGowan, and P.R. Cook, “Many expressed genes in bacteria and yeast are transcribed only once per cell cycle,” FASEB J.,  vol. 20, Aug. 2006, pp. 1721-1723.
 
[4]
J.B. Andersen et al., “New Unstable Variants of Green Fluorescent Protein for Studies of Transient Gene Expression in Bacteria,” Applied and Environmental Microbiology,  vol. 64, Jun. 1998, pp. 2240–2246.
 
[5]
“Part:BBa B0032 - partsregistry.org”; http://partsregistry.org/Part:BBa_B0032.
 
Retrieved from " http://2008.igem.org/Team:KULeuven/Model/Output "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
