"
Page
Discussion
View source
History
teams
Log in
 
Team:KULeuven/Model/Inverter
From 2008.igem.org
  dock/undock dropdown  
Home
Reports
Calendar
Project abstract
Meet the team
Leave a message
Mail us
The Team
Research Labs and Groups Students Instructors Advisors Pictures
The Project
Summary Components
Input Output Filter InverTimer Reset Cell Death Memory
Safety End Evaluation Literature Brainstorm
Ethics
Introduction Three A's
Defining the three A's Developments in modern physics Developments in biology
Concerns & Issues
Defining synthetic biology A framework Concerns Issues The need for answers
Biological Robotics
KULeuven iGEM 2008 project Biological robotics
Afterword References
Submitted Parts
Sandbox
Modeling
Overview Kinetic Constants Components
Output Filter InverTimer Reset Cell Death Memory
Full Model Sensitivity Analysis Multi-cell Model Diffusion
Data Analysis
Overview New Parts
GFP (LVA-tag) Other
Components
Input Output Other
Software
Multi-cell Toolbox Simbiology2LaTeX Toolbox
Notebook
Protocols Reports
Daily Weekly
Lab Data
Freezer Primers Ligation
Tools Press Acknowledgments
Contents
1 Invertimer
1.1 Position in the system
1.2 Describing the system
1.2.1 ODE's
1.2.2 Parameters
1.3 Models
1.3.1 CellDesigner (SBML file)
1.3.2 Matlab (SBML file)
1.4 Simulations
1.5 References
Invertimer
Position in the system
The InverTimer subsystem receives its input from the filter, T7. The InverTimer's function is to produce HSL when no input is present, so a low T7 input gives rise to a high HSL output and vice versa. The production of HSL means that the cell will start a timer that eventually will be used in the celldeath-subsystem to produce ccdB. In this way the cell will die off if no desease remains present.
Describing the system
see also: Project:InverTimer
ODE's
Parameters
Parameter values (Inverter)
Name
Value
Comments
Reference
Degradation Rates
dLuxI
dLVA = 2.814E-4 s-1
LVA-tag reduces lifetime to 40 minutes
[2] [7]
dRNA_LuxI
0.0025 s-1
[5]
dLuxI_antimRNA
0.0045303737 s-1
estimate: because this RNA isn't translated, it degrades faster
[5]
dLacI
dLVA = 2.814E-4 s-1
LVA-tag reduces lifetime to 40 minutes
[2] [7]
dclosed mRNA LacI
0.0046209812 s-1
estimate: because this mRNA isn't translated, it degrades faster
[5]
dopen mRNA LacI
0.0023104906 s-1
[5]
dopen mRNA LacI complex
0.0023104906 s-1
[5]
dHSL
1.02E-6 s-1
very stable in the medium, lifetime around 185h
[11]
LuxI catalysis
kcat
0.0166666667 s-1
Estimated to be about 90% of Vmax in LB medium.
[4]
T7 Transcription
KT7
421
dissociation constant, recalculated to remove units
[10]
kmax
0.044 s-1
maximal T7 transcription rate
[10]
Key-Lock constants
Keq 1
0,015 [M]
between closed and open T7 mRNA, modeled for competition, experimental
[2]
Keq 2
0.0212 [M]
between closed T7 mRNA and key unlocked mRNA complex, modeled for competition, experimental
[2]
kdis2
0.00416 s-1
derived from experimental values
[2]
kcomplex2
0.00237 s-1
derived from experimental values
[2]
kclosed
500 s-1
derived from experimental values
[2]
kopen
7.5 s-1
derived from experimental values
[2]
LacI repression
KLacI
1.0E-10 M-1
Dissociation constant
[3]
nLacI
2.0
Hill coefficient for LacI
[3]
k_trans_LacI
0.0025 s-1
Estimated maximal transcription rate from R0011
[9]
Antisense LuxI
k_complex3
0.00237 s-1
rate constant for formation of asRNA - LuxI mRNA duplex
[3]
KmRNA_LuxI:antisense_mRNA
4.22E14
Complex of LuxI mRNA with antisense mRNA
[1]
Translation Rates
ktransl LuxI
0.167 s-1
translation rate for B0032 RBS (0.3 relative efficiency)
[8]
ktransl LacI
0.167 s-1
estimate: lock defined translation rate for LacI
[2]
Models
CellDesigner ( SBML file )
Matlab ( SBML file )
Simulations
Time span
Input (TetR)
Results
A
0.0125 s-1
The amount LacI increases from state zero to state one because both mRNA_RIBOKEY and pT7_tag are present. This results in a repression of LuxI which decreases to zero: the input signal (TetR) is inverted.
B
5E-5 s-1
The amount LacI decreases back to state zero. The amount LuxI remains the same (state zero).
C
5E-5 s-1
LuxI changes from state zero to state one. Time span B and C form together the transient behaviour of the inverter when the input signal changes from one to zero.
D
5E-5 s-1
LuxI remains in state one: the input signal is once again inverted.
E
0.0125 s-1
A short pulse of 1000 seconds has a influence a steep decrease of LuxI.
F & G
5E-5 s-1
During time span F and G, LuxI decreases further for a while and increases back to state one.
H
5E-5 s-1
LuxI is back in state one.
The simulation shows a working inverter (left figure). A small disadvantage is the transient behaviour of the inverter: a small pulse of 1000 seconds results in a transient behaviour of +- 30000 seconds. Also for a long pulse (10000 seconds) is a long transient behaviour noticeable (40000 seconds). The effect of the inverter on the timer aspect is visuable in the right figure: a long pulse ( from 10000 till 11000) resets the timer (HSL drecreases till zero). After this pulse and the transient behaviour of the inverter, the timer restarts counting. The short pulse (from 200000 till 201000 seconds) only partially resets the timer.
All graphs have amounts (number of molecules in the cell) plotted vs time, measured in seconds.
References
Bibliography
[1]
A. E G H Wagner and R W Simons, “Antisense RNA Control in Bacteria, Phages, and Plasmids,” Nov. 2003; http://arjournals.annualreviews.org/doi/abs/10.1146/annurev.mi.48.100194.003433.
 
[2]
“Berkeley2006-RiboregulatorsMain - IGEM”; http://parts2.mit.edu/wiki/index.php/Berkeley2006-RiboregulatorsMain.
 
[3]
“ETHZ/Parameters - IGEM07”; https://2007.igem.org/ETHZ/Parameters.
 
[4]
“Generation of cell-to-cell signals in quorum sensing: acyl homoserine lactone synthase activity of a purified Vibrio fischeri LuxI protein,” Sep. 1996; http://www.pnas.org/content/93/18/9505.
 
[5]
J.A. Bernstein et al., “Global analysis of mRNA decay and abundance in Escherichia coli at single-gene resolution using two-color fluorescent DNA microarrays,” Proceedings of the National Academy of Sciences of the United States of America,  vol. 99, Jul. 2002, pp. 9697–9702.
 
[6]
W. Hsieh et al., “Influence of sequence and distance between two operators on interaction with the lac repressor,” J. Biol. Chem.,  vol. 262, Oct. 1987, pp. 14583-14591.
 
[7]
J.B. Andersen et al., “New Unstable Variants of Green Fluorescent Protein for Studies of Transient Gene Expression in Bacteria,” Applied and Environmental Microbiology,  vol. 64, Jun. 1998, pp. 2240–2246.
 
[8]
“Part:BBa B0032 - partsregistry.org”; http://partsregistry.org/Part:BBa_B0032.
 
[9]
“Part:BBa R0011 - partsregistry.org”; http://partsregistry.org/Part:BBa_R0011.
 
[10]
G.M. Skinner et al., “Promoter Binding, Initiation, and Elongation By Bacteriophage T7 RNA Polymerase: A SINGLE-MOLECULE VIEW OF THE TRANSCRIPTION CYCLE,” J. Biol. Chem.,  vol. 279, Jan. 2004, pp. 3239-3244.
 
[11]
Y. Wang and J.R. Leadbetter, “Rapid Acyl-Homoserine Lactone Quorum Signal Biodegradation in Diverse Soils,” Appl. Environ. Microbiol.,  vol. 71, Mar. 2005, pp. 1291-1299.
 
Retrieved from " http://2008.igem.org/Team:KULeuven/Model/Inverter "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
