"
Page
Discussion
View source
History
teams
Log in
 
Team:KULeuven/Data/Other Parts
From 2008.igem.org
  dock/undock dropdown  
Home
Reports
Calendar
Project abstract
Meet the team
Leave a message
Mail us
The Team
Research Labs and Groups Students Instructors Advisors Pictures
The Project
Summary Components
Input Output Filter InverTimer Reset Cell Death Memory
Safety End Evaluation Literature Brainstorm
Ethics
Introduction Three A's
Defining the three A's Developments in modern physics Developments in biology
Concerns & Issues
Defining synthetic biology A framework Concerns Issues The need for answers
Biological Robotics
KULeuven iGEM 2008 project Biological robotics
Afterword References
Submitted Parts
Sandbox
Modeling
Overview Kinetic Constants Components
Output Filter InverTimer Reset Cell Death Memory
Full Model Sensitivity Analysis Multi-cell Model Diffusion
Data Analysis
Overview New Parts
GFP (LVA-tag) Other
Components
Input Output Other
Software
Multi-cell Toolbox Simbiology2LaTeX Toolbox
Notebook
Protocols Reports
Daily Weekly
Lab Data
Freezer Primers Ligation
Tools Press Acknowledgments
Contents
1 Introduction
2 Antisense LuxI
2.1 Construct & protocol
2.2 Results
3 T7 RNA polymerase (with and without UmuD)
3.1 Construct & protocol
3.2 Results
4 Hybrid promoter
4.1 Construct & protocol
4.2 Results
5 ccdB
5.1 Construct & protocol
5.2 Results
Introduction
As Dr.Coli is a very extensive project, there was not sufficient time to test all the new parts that we created. This page contains a summary of how our new basic parts can be tested (the construct that have to be made and the actual testing protocol). It will also give an overview of the results we had so far. We worked as much in parallel as possible, building different parts and subsystems at the same time. This way we could still have some nice results, even when some of the parts didn't succeed. All the new parts introduced in our project were properly built. We leave the actual testing of these parts to the next year's teams, our advisors or you :-).
Antisense LuxI
Parts Registry:K145013
Construct & protocol
Antisense LuxI (asLuxI) is an RNA molecule that can bind to the mRNA of luxI, thus repressing the translation of the LuxI enzyme. Testing this new part would not be very easy. One would need two rather complex systems :
System 1:
+
+
+
System 2:
+
+
+
+
In system 1 LuxR is produced constitutively. On its own LuxR will not do much, but when it is bound to HSL it can activate the transcription of gfp from the lux promoter (R0062). HSL is a molecule that can be generated by LuxI, and the production of LuxI depends on the presence or absence of aTc. So, when no aTc (to relieve TetR repression) is present, there will be no LuxI and therefore no HSL, and no active LuxR. This means no production of GFP. When aTc is added to the system, LuxI will be produced and so will HSL. HSL can then bind to LuxR and start the transcription of GFP (fluorescence should become detectable).
System 2 works very much in the same way, but now we also have the constitutive production of antisense LuxI. This means that, when aTc is added to the system, transcription of luxI will occur. However, the protein LuxI should not be produced in high amounts because asLuxI will bind the mRNA of luxI. No LuxI means no HSL, and no active LuxR, so no GFP. In system 2 the production of GFP will never occur, with or without aTc.
To test this, one should build these two systems and transform them into E. coli cells. After that you can prepare liquid cultures of the 2 systems. Then you should add aTc (100 ng/ml) and follow the fluorescence of the liquid cultures in time (e.g. with FACS). After incubation and if all goes well, you should see highly fluorescent cells with system 1, while the fluorescence of system 2 should remain low.
Results
Antisense LuxI ( K145013 ) was constructed with PCR. The template was the plasmid containing the luxI BioBrick ( C0061 ). We used the following primers :
Forward Primer: CATCAGGAAT TCGCGGCCGC TTCTAGTTAT TAAGCTACTA AAGCGTAGTT TTC
Forward Primer: CTGCAGCGGC CGCTACTAGT AATGACTATA ATGATAAAAA AATCGG
This PCR succeeded very well and we then cut the PCR product with EcoRI and SpeI. This digest was consecutively ligated into a pSB1A2 plasmid with T4 DNA ligase. This plasmid was then electroporated into TOP10 cells and these cells were grown on agar plates containing ampicillin. Everything went well and sequencing showed that part K145013 was built correctly !
T7 RNA polymerase (with and without UmuD)
Parts Registry:K145014 and Parts Registry:K145001
Construct & protocol
T7 RNA polymerase is an enzyme that will start transcription from a T7 promoter. We use this part in our filter and there it is necessary that this enzyme is degraded quickly. Therefore we also constructed a T7 polymerase with an UmuD tag. This tag will render T7 RNA polymerase susceptible to degradation by certain proteases. To test whether this tag really makes T7 polymerase degrade faster, one should build the following constructs:
Construct 1 (no tag):
+
Construct 2 (UmuD tag):
+
In these two constructs, TetR is constitutively produced and this will repress the production of T7 RNA polymerase from the tet promoter (R0040). Adding aTc will render TetR inactive and T7 RNA polymerase will be produced. So, in order to test the degradation of the enzymes, you should grow liquid cultures containing aTc of the cells with construct 1 or construct 2. These cells will start to produce T7 RNA polymerase or T7 RNA polymerase with UmuD. When these cells have been grown overnight, you can transfer them to another medium that does not contain aTc. The production of T7 polymerase will then stop and you can now measure the degradation rate of the T7 RNA polymerases. To do this, you should harvest some of the cells at several time intervals. Then you should isolate the proteins from the cells and determine the concentration of T7 RNA polymerase using anti-T7 antibodies. The tag works if the concentration of T7 RNA polymerase with UmuD tag decreases much faster in time than the concentration of normal T7 RNA polymerase.
Results
We constructed both T7 RNA polymerases with a PCR reaction. To built the T7 RNA polymerase without the tag ( K145001 ), we did a PCR with pfx polymerase on the plasmid of I712022 as template and with the following primers:
Forward primer: ATGAATTCGC GGCCGCTTCT AGATGAACAC GATTAACATC GC
Reverse primer: CTGCAGCGGC CGCTACTAGT ATTATTACGC GAACGCGAAG TCCG
To attach an UmuD tag to the T7 RNA polymerase ( K145014 ) we did a two-step PCR. The first PCR was done with I712022 as a template and the following primers:
Forward primer: CCGCTATTTA GCGATCTTGT TCAGTGTGGC TTTCCTTCAC CGATGAACAC GATTAACATC GC
Reverse primer: CTGCAGCGGC CGCTACTAGT ATTATTACGC GAACGCGAAG TCCG
The second step of the PCR was performed on the unpurified PCR product of step one and with these two primers:
Forward primer: ATGAATTCGC GGCCGCTTCT AGATGTTGTT TATCAAGCCT GCGGATCTCC GCGAAATTGT GACTTTTCCG
CTATTTAGCG ATCTTGTTCA G
Reverse primer: CTGCAGCGGC CGCTACTAGT ATTATTACGC GAACGCGAAG TCCG
It took very long, but eventually the two PCRs succeeded. These PCR products were then cut with EcoRI and XbaI, and these digests were ligated into pSB1A2 plasmids. These plasmids were electroporated into TOP10 cells and the obtained colonies were miniprepped. Sequencing showed that both PCRs succeeded very well and that we made the correct constructs ( K145001 , K145014 ).
The first part of the test constructs (J23116+B0034+C0040+B0015) is actually our input and this was properly built, see here . We also tried to build the second part (R0040+B0032+T7polymerase+B0015), but various attempts to ligate the T7 polymerase genes to B0015 failed.
Hybrid promoter
Parts Registry:K145150
Construct & protocol
The hybrid promoter (HPr) is a part that was completely designed by ourselves and it is used in our cell death mechanism. This is a promoter that can be regulated in two ways. On the one hand it can be activated by a complex of LuxR-HSL and on the other hand it can be repressed by cII P22. In order to test if this promoter works properly, we designed two seperate tests.
The first test is designed to see if the LuxR-HSL complex can activate transcription from this promoter. We would need the construct that is shown above. This construct will constitutively produce LuxR. LuxR on its own cannot activate transcription from the hybrid promoter ( K145150 ) and so production of GFP will be only basal. However, when we add HSL, the transcription of gfp should become activated. So, if you want to test whether the hybrid promoter can be activated, you would have to make a liquid culture of the cells with the above construct. Then you can add HSL to the liquid culture and start measuring the fluorescence of the cells at different time intervals (e.g. with FACS). If all goes well, the fluorescence should increase upon addition of HSL.
The second test is designed to see if cII P22 can repress transcription from the hybrid promoter, even in the presence of a LuxR-HSL complex. Therefore we should build the construct shown above. In this construct we have constitutive production of TetR, which will repress the production of cII P22. When aTc is added to the cells, TetR is inactivated and cII P22 is produced. This will then act as a repressor for the hybrid promoter, thus completely blocking the expression of GFP. We also have constitutive production of LuxR, but this shouldn’t do anything. It should only activate the expression of GFP when HSL is present and cII P22 is absent. So, if you want to test the hybrid promoter, you should prepare some liquid cultures of cells containing this construct and add various amounts of aTc and/or HSL. Then you can see what the effect is of these molecules by measuring the fluorescence at different time intervals (e.g. with FACS). If the hybrid promoter works properly, the results should look like this:
Molecules added
Fluorescence
Molecules added
Fluorescence
nothing
no
HSL + aTc
no
only HSL
yes
only aTc
no
Results
The hybrid promoter itself was built by an end-filling reaction using the Klenow fragment (polymerase). The primers that were used in this end-filling reaction and that make up the promoter were the following:
Forward primer: CATCATGAAT TCGCGGCCGC TTCTAGACCT GTAGGATCGT ACAGGTTTAC TAAAGATTCC TTTAG
Reverse primer: TACTAGTAGC GGCCGCTGCA GGAAATTAAA GAACACTTAA ATTATAAACT AAAGGAATCT TTAG
When this was finished, we digested it with EcoRI and SpeI and ligated it into a pSB1A2 plasmid. This plasmid was later on electroporated into E.coli cells and then miniprepped. We also tried to make the construct for the first test and we actually performed this test, but it failed. Sequencing then showed that our construct didn’t contain all the desired parts. We also tried to sequence the hybrid promoter, but this failed twice.
ccdB
Parts Registry:K145151
Construct & protocol
In our project we want Dr.Coli to eliminate himself when he is not needed anymore. We do this because we don’t want useless bacteria in our gut or the environment. One way to achieve this is by using the protein CcdB ( K145151 ). This protein is toxic to E.coli cells (without CcdA) and therefore very suitable. Constructing and testing this part would be relatively simple. To test whether our CcdB can cause cell death, we should construct the following BioBrick:
+
In this construct the production of CcdB is inhibited by the constitutive production of the Tet repressor. Adding aTc however, will inactivate this repressor and CcdB will be produced, causing the cells to die. To test the function of CcdB you should prepare three types of agar plates:
one plate should be inoculated with Top10 cells containing the construct and aTc should be present
another plate should be inoculated with the same cells containing the same construct, but aTc should not be present
yet another plate should be inoculated with empty Top10 cells (no plasmid) and aTc should be present
The results of this test should be that the first plate (CcdB induced by aTc) contains considerably fewer colonies – preferably none – than the other to plates. If you want to be sure that the lack of colonies is not just a coincidence, you should repeat this test a few times. When you always get the same results (no colonies on the first plate), then this CcdB construct works.
Results
The ccdB gene was obtained by PCR on the plasmid of BioBRick P1010 as a template with the following primers:
Forward primer: CATCATGAAT TCGCGGCCGC TTCTAGATGC AGTTTAAGGT TTACACC
Reverse primer: CTGCAGCGGC CGCTACTAGT ATTATATTCC CCAGAACATC AGG
This PCR didn’t cause any problems and the PCR product was cut with EcoRI and SpeI. This digest was then ligated into pSB1A2 cells and subsequently electroporated into Top10 cells and miniprepped. Sequencing showed that CcdB was constructed correctly. We also started to build the test construct, but unfortunately we couldn’t finish it.
Retrieved from " http://2008.igem.org/Team:KULeuven/Data/Other_Parts "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
