<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_KULeuven_Model_Cell_Death"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:KULeuven/Model/Cell_Death">Page               </A></LI><LI><A href="/Talk:Team:KULeuven/Model/Cell_Death">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:KULeuven/Model/Cell_Death&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:KULeuven/Model/Cell_Death&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:KULeuven/Model/Cell_Death" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:KULeuven/Model/Cell Death</H1><DIV id="bodyContent"><H3 id="siteSub">From 2008.igem.org</H3><DIV id="ddToggle">  dock/undock dropdown  </DIV><DIV id="header"><DIV id="utilNav"><UL><LI id="homepage"><A title="Return to the KULeuven iGEM 2008 homepage" href="https://2008.igem.org/Team:KULeuven">Home</A></LI><LI id="reports"><A title="Keep track of our progress throughout the summer" href="https://2008.igem.org/Team:KULeuven/Meeting_Reports">Reports</A></LI><LI id="calendar"><A title="View important events" href="https://2008.igem.org/Team:KULeuven/Meeting_Calendar">Calendar</A></LI></UL></DIV><DIV id="suppNav"><UL><LI id="abstract"><A title="See what's Dr.Coli all about" href="https://2008.igem.org/Team:KULeuven/Project"><STRONG>Project</STRONG> abstract</A></LI><LI id="team"><A title="List of all our team participants" href="https://2008.igem.org/Team:KULeuven/Team/Students"><STRONG>Meet</STRONG> the team</A></LI><LI id="guestbook"><A title="Take a peek at our guestbook" href="https://2008.igem.org/Team:KULeuven/Guestbook"><STRONG>Leave</STRONG> a message</A></LI><LI id="mail"><A title="Questions, want to cooperate with our team or drop by? Let us know" href="mailto:igem@kuleuven.be?SUBJECT=IGEM%202008"><STRONG>Mail</STRONG> us</A></LI></UL></DIV><DIV id="mainNav"><UL id="ddNav"><LI><A>The Team</A><DIV><A href="https://2008.igem.org/Team:KULeuven/Team/LabsandGroups">Research Labs and Groups</A><A href="https://2008.igem.org/Team:KULeuven/Team/Students">Students</A><A href="https://2008.igem.org/Team:KULeuven/Team/Instructors">Instructors</A><A href="https://2008.igem.org/Team:KULeuven/Team/Advisors">Advisors</A><A href="https://2008.igem.org/Team:KULeuven/Team/Pictures">Pictures</A></DIV></LI><LI><A>The Project</A><DIV><A href="https://2008.igem.org/Team:KULeuven/Project">Summary</A><SPAN><A>Components</A><DIV><A href="https://2008.igem.org/Team:KULeuven/Project/Input">Input</A><A href="https://2008.igem.org/Team:KULeuven/Project/Output">Output</A><A href="https://2008.igem.org/Team:KULeuven/Project/Filter">Filter</A><A href="https://2008.igem.org/Team:KULeuven/Project/Inverter">InverTimer</A><A href="https://2008.igem.org/Team:KULeuven/Project/Reset">Reset</A><A href="https://2008.igem.org/Team:KULeuven/Project/CellDeath">Cell Death</A><A href="https://2008.igem.org/Team:KULeuven/Project/Memory">Memory</A></DIV></SPAN><A href="https://2008.igem.org/Team:KULeuven/Safety">Safety</A><A href="https://2008.igem.org/Team:KULeuven/Evaluation">End Evaluation</A><A href="https://2008.igem.org/Team:KULeuven/Literature">Literature</A><A href="https://2008.igem.org/Team:KULeuven/Brainstorm">Brainstorm</A></DIV></LI><LI><A>Ethics</A><DIV><A href="https://2008.igem.org/Team:KULeuven/Ethics/Introduction">Introduction</A><SPAN><A>Three A's</A><DIV><A href="https://2008.igem.org/Team:KULeuven/Ethics/Threeas/Defining">Defining the three A's</A><A href="https://2008.igem.org/Team:KULeuven/Ethics/Threeas/Physics">Developments in modern physics</A><A href="https://2008.igem.org/Team:KULeuven/Ethics/Threeas/Biology">Developments in biology</A></DIV></SPAN><SPAN><A>Concerns &amp; Issues</A><DIV><A href="https://2008.igem.org/Team:KULeuven/Ethics/Concerns/Defining">Defining synthetic biology</A><A href="https://2008.igem.org/Team:KULeuven/Ethics/Concerns/Framework">A framework</A><A href="https://2008.igem.org/Team:KULeuven/Ethics/Concerns/Concerns">Concerns</A><A href="https://2008.igem.org/Team:KULeuven/Ethics/Concerns/Issues">Issues</A><A href="https://2008.igem.org/Team:KULeuven/Ethics/Concerns/Answers">The need for answers</A></DIV></SPAN><SPAN><A>Biological Robotics</A><DIV><A href="https://2008.igem.org/Team:KULeuven/Ethics/Robotics/KULeuven">KULeuven iGEM 2008 project</A><A href="https://2008.igem.org/Team:KULeuven/Ethics/Robotics/Robotics">Biological robotics</A></DIV></SPAN><A href="https://2008.igem.org/Team:KULeuven/Ethics/Afterword">Afterword</A><A href="https://2008.igem.org/Team:KULeuven/Ethics/References">References</A></DIV></LI><LI><A>Submitted Parts</A><DIV><A href="http://partsregistry.org/cgi/partsdb/pgroup.cgi?pgroup=iGEM2008&amp;group=KULeuven">Sandbox</A></DIV></LI><LI><A>Modeling</A><DIV><A href="https://2008.igem.org/Team:KULeuven/Model/Overview">Overview</A><A href="https://2008.igem.org/Team:KULeuven/Model/KineticConstants">Kinetic Constants</A><SPAN><A>Components</A><DIV><A href="https://2008.igem.org/Team:KULeuven/Model/Output">Output</A><A href="https://2008.igem.org/Team:KULeuven/Model/Filter">Filter</A><A href="https://2008.igem.org/Team:KULeuven/Model/Inverter">InverTimer</A><A href="https://2008.igem.org/Team:KULeuven/Model/Reset">Reset</A><A href="https://2008.igem.org/Team:KULeuven/Model/CellDeath">Cell Death</A><A href="https://2008.igem.org/Team:KULeuven/Model/Memory">Memory</A></DIV></SPAN><A href="https://2008.igem.org/Team:KULeuven/Model/FullModel">Full Model</A><A href="https://2008.igem.org/Team:KULeuven/Model/Sensitivity">Sensitivity Analysis</A><A href="https://2008.igem.org/Team:KULeuven/Model/MultiCell">Multi-cell Model</A><A href="https://2008.igem.org/Team:KULeuven/Model/Diffusion">Diffusion</A></DIV></LI><LI><A>Data Analysis</A><DIV><A href="https://2008.igem.org/Team:KULeuven/Data/Overview">Overview</A><SPAN><A>New Parts</A><DIV><A href="https://2008.igem.org/Team:KULeuven/Data/GFP">GFP (LVA-tag)</A><A href="https://2008.igem.org/Team:KULeuven/Data/Other_Parts">Other</A></DIV></SPAN><SPAN><A>Components</A><DIV><A href="https://2008.igem.org/Team:KULeuven/Data/Input">Input</A><A href="https://2008.igem.org/Team:KULeuven/Data/Output">Output</A><A href="https://2008.igem.org/Team:KULeuven/Data/Other_Components">Other</A></DIV></SPAN></DIV></LI><LI><A>Software</A><DIV><A href="https://2008.igem.org/Team:KULeuven/Software/MultiCell">Multi-cell Toolbox</A><A href="https://2008.igem.org/Team:KULeuven/Software/Simbiology2LaTeX">Simbiology2LaTeX Toolbox</A></DIV></LI><LI><A>Notebook</A><DIV><A href="https://2008.igem.org/Team:KULeuven/Protocols">Protocols</A><SPAN><A>Reports</A><DIV><A href="https://2008.igem.org/Team:KULeuven/Meeting Reports">Daily</A><A href="https://2008.igem.org/Team:KULeuven/Weekly Meetings">Weekly</A></DIV></SPAN><SPAN><A>Lab Data</A><DIV><A href="https://2008.igem.org/Team:KULeuven/Freezer">Freezer</A><A href="https://2008.igem.org/Team:KULeuven/Primers">Primers</A><A href="https://2008.igem.org/Team:KULeuven/Ligation">Ligation</A></DIV></SPAN><A href="https://2008.igem.org/Team:KULeuven/Tools">Tools</A><A href="https://2008.igem.org/Team:KULeuven/Press">Press</A><A href="https://2008.igem.org/Team:KULeuven/Acknowledgments">Acknowledgments</A></DIV></LI></UL></DIV></DIV><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Cell_Death"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Cell Death</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Position_in_the_system"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Position in the system</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Describing_the_system"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Describing the system</SPAN></A></LI><LI class="toclevel-3 tocsection-4"><A href="#ODE.27s"><SPAN class="tocnumber">1.2.1</SPAN><SPAN class="toctext">ODE's</SPAN></A></LI><LI class="toclevel-3 tocsection-5"><A href="#Parameters"><SPAN class="tocnumber">1.2.2</SPAN><SPAN class="toctext">Parameters</SPAN></A></LI></UL><LI class="toclevel-2 tocsection-6"><A href="#Models"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Models</SPAN></A></LI><LI class="toclevel-3 tocsection-7"><A href="#CellDesigner_.28SBML_file.29"><SPAN class="tocnumber">1.3.1</SPAN><SPAN class="toctext">CellDesigner (SBML file)</SPAN></A></LI><LI class="toclevel-3 tocsection-8"><A href="#Matlab_.28SBML_file.29"><SPAN class="tocnumber">1.3.2</SPAN><SPAN class="toctext">Matlab (SBML file)</SPAN></A></LI><LI class="toclevel-1 tocsection-9"><A href="#New_Cell_Death"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">New Cell Death</SPAN></A></LI><LI class="toclevel-2 tocsection-10"><A href="#Extensions_to_previous_system"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Extensions to previous system</SPAN></A></LI><LI class="toclevel-2 tocsection-11"><A href="#Describing_the_system_2"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">Describing the system</SPAN></A></LI><LI class="toclevel-3 tocsection-12"><A href="#ODE.27s_2"><SPAN class="tocnumber">2.2.1</SPAN><SPAN class="toctext">ODE's</SPAN></A></LI><LI class="toclevel-3 tocsection-13"><A href="#Parameters_2"><SPAN class="tocnumber">2.2.2</SPAN><SPAN class="toctext">Parameters</SPAN></A></LI><LI class="toclevel-2 tocsection-14"><A href="#Models_2"><SPAN class="tocnumber">2.3</SPAN><SPAN class="toctext">Models</SPAN></A></LI><LI class="toclevel-3 tocsection-15"><A href="#CellDesigner_.28SBML_file.29_2"><SPAN class="tocnumber">2.3.1</SPAN><SPAN class="toctext">CellDesigner (SBML file)</SPAN></A></LI><LI class="toclevel-3 tocsection-16"><A href="#Matlab_.28SBML_file.29_2"><SPAN class="tocnumber">2.3.2</SPAN><SPAN class="toctext">Matlab (SBML file)</SPAN></A></LI><LI class="toclevel-2 tocsection-17"><A href="#Simulations"><SPAN class="tocnumber">2.4</SPAN><SPAN class="toctext">Simulations</SPAN></A></LI><LI class="toclevel-2 tocsection-18"><A href="#References"><SPAN class="tocnumber">2.5</SPAN><SPAN class="toctext">References</SPAN></A></LI></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Cell_Death">Cell Death</SPAN></H2><H3><SPAN class="mw-headline" id="Position_in_the_system"> Position in the system </SPAN></H3><P>The Cell Death subsystem receives input from two other subsystems, namely:
</P><UL><LI><A href="/Team:KULeuven/Model/Inverter" title="Team:KULeuven/Model/Inverter"> Inverter</A></LI><LI><A href="/Team:KULeuven/Model/Pulse_Generator" title="Team:KULeuven/Model/Pulse Generator"> Pulse Generator</A></LI></UL><P>LuxR is the component repressing the regulation of CcdB, the toxic product causing cell death. There the LuxR production is constitutive, no protein controls the gene regulation of LuxR, but the amount of LuxR available to repress the transcripion of the CcdB gene is controlled by HSL (Homoserine lactone).
</P><P>If the inverter subsystem produces HSL (occurs when no light is detectable), this will forms a complex with LuxR. This will diminish the amount of LuxR available to repress the CcdB transcription and initiate cell death. When waiting long enough the amount of HSL becomes critical.
</P><P>If however the pulse generator becomes active (by the filter), it will produce a pulse of lactonase, which will then bind to the HSL, reacting to an hydroxy-acid. As opposed to HSL, this hydroxy-acid will no longer form a complex with LuxR. This increase in LuxR lowers the CcdB production. The challenge is to generate a pulse of lactonase high enough to neutralise all HSL present in the cell.
</P><H3><SPAN class="mw-headline" id="Describing_the_system"> Describing the system </SPAN></H3><H4><SPAN class="mw-headline" id="ODE.27s"> ODE's </SPAN></H4><H4><SPAN class="mw-headline" id="Parameters"> Parameters </SPAN></H4><P>Attention: old system, old parameters!
</P><TABLE width="80%" style="border: 1px solid #003E81; background-color: #EEFFFF;"><CAPTION><I>Parameter values (Cell Death)</I></CAPTION><TBODY><TR><TH width="15%"> Name
</TH><TH width="15%"> Value
</TH><TH width="40%"> Comments
</TH><TH width="10%"> Reference
</TH></TR><TR><TH colspan="4" style="border-bottom: 1px solid #003E81;"> Degradation Rates
</TH></TR><TR><TD> d<SUB>LuxR</SUB></TD><TD> 0.0010 s<SUP>-1</SUP></TD></TR><TR><TD> d<SUB>LuxR_HSL</SUB></TD><TD> 0.0010 s<SUP>-1</SUP></TD><TD> complex of HSL and LuxR degrades, giving back HSL
</TD></TR><TR><TD> d<SUB>RNA_LuxR</SUB></TD><TD> 0.00227 s<SUP>-1</SUP></TD></TR><TR><TD> d<SUB>CcdB</SUB></TD><TD> 7.7E-5 s<SUP>-1</SUP></TD></TR><TR><TD> d<SUB>RNA_CcdB</SUB></TD><TD> 0.00231 s<SUP>-1</SUP></TD></TR><TR><TD> d<SUB>HSL</SUB></TD><TD> 1.02E-6 s<SUP>-1</SUP></TD></TR><TR><TH colspan="4" style="border-bottom: 1px solid #003E81;"> Association/Dissociation/Reaction Rates
</TH></TR><TR><TD> k<SUB>ass</SUB>(HSL+LuxR)
</TD><TD> 1E6 s<SUP>-1</SUP></TD><TD> association rate of HSL with LuxR. Chosen to be relatively (to the other rate constants) high and such that K<SUB>diss</SUB> (HSL + LuxR) equals 10<SUP>-6</SUP></TD></TR><TR><TD> k<SUB>diss</SUB> (HSL+LuxR)
</TD><TD> 1 s<SUP>-1</SUP></TD><TD> dissociation rate of the HSL-LuxR complex
</TD></TR><TR><TD> k<SUB>ass</SUB> (HSL+lactonase)
</TD><TD> 1E6 s<SUP>-1</SUP></TD><TD> association rate of HSL with lactonase
</TD></TR><TR><TD> k<SUB>diss</SUB> (HSL+lactonase)
</TD><TD> 446.5 s<SUP>-1</SUP></TD><TD> dissociation rate of the HSL-lactonase complex
</TD></TR><TR><TD> k<SUB>cat</SUB> (HSL:hydroxy-acid)
</TD><TD> 29 s<SUP>-1</SUP></TD><TD> catalytic transformation of HSL to an hydroxy-acid, lactonase is the enzyme
</TD></TR><TR><TH colspan="4" style="border-bottom: 1px solid #003E81;"> Dissociation Constants
</TH></TR><TR><TD> K<SUB>HSL_LuxR</SUB></TD><TD> 1E-6 [M]/L
</TD><TD> k<SUB>diss</SUB> / k<SUB>ass</SUB> (HSL+LuxR)
</TD></TR><TR><TD> K<SUB>HSL_LuxR</SUB></TD><TD> 4.05E-6 [M]/L
</TD><TD> binding HSL_LuxR on LuxPromotor
</TD></TR><TR><TH colspan="4" style="border-bottom: 1px solid #003E81;"> Hill Cooperativity
</TH></TR><TR><TD> n<SUB>HSL_LuxR</SUB></TD><TD> 2.08
</TD></TR><TR><TH colspan="4" style="border-bottom: 1px solid #003E81;"> Transcription Rates
</TH></TR><TR><TD> k<SUB>mRNA_LuxR</SUB> (constitutive promotor)
</TD><TD> 0.025 s<SUP>-1</SUP></TD><TD> see <A href="/Team:KULeuven/Model/KineticConstants#Constitutive_promoters" title="Team:KULeuven/Model/KineticConstants" class="mw-redirect"> Constitutive Promotors &amp; E. coli transcription Rates</A></TD></TR><TR><TD> k<SUB>mRNA_CcdB</SUB></TD><TD> 0.025 s<SUP>-1</SUP></TD><TD> maximal transcription rate for CcdB RNA (no LuxR repressor present)
</TD></TR></TBODY></TABLE><P><A href="http://www.sciencedirect.com/science?_ob=MImg&amp;_imagekey=B6T39-3Y6HKD6-BK-1&amp;_cdi=4941&amp;_user=877992&amp;_orig=search&amp;_coverDate=08%2F30%2F1995&amp;_sk=998379998&amp;view=c&amp;wchp=dGLbVtz-zSkzS&amp;md5=49efd14150c71e668eabdef225220ce3&amp;ie=/sdarticle.pdf" class="external text" rel="nofollow">This paper</A> says that the synthesis of only a few molecules of CcdB protein is probably lethal.
</P><P>All references for parameters can be found below, in the new section.
</P><H3><SPAN class="mw-headline" id="Models"> Models </SPAN></H3><H4><SPAN class="mw-headline" id="CellDesigner_.28SBML_file.29"> CellDesigner (<A href="https://static.igem.org/mediawiki/2008/3/37/CellDeath_CellDesigner.zip" class="external text" rel="nofollow">SBML file</A>)</SPAN></H4><H4><SPAN class="mw-headline" id="Matlab_.28SBML_file.29"> Matlab (<A href="https://static.igem.org/mediawiki/2008/4/45/CellDeath_Matlab.zip" class="external text" rel="nofollow">SBML file</A>)</SPAN></H4><P>Remark: not yet up to date to latest (final) version 
</P><H2><SPAN class="mw-headline" id="New_Cell_Death"> New Cell Death</SPAN></H2><H3><SPAN class="mw-headline" id="Extensions_to_previous_system">Extensions to previous system</SPAN></H3><P>During the summer we switched from the above system to a new one which you can see just below, take a look at this figure as it will help you understand the regulation that is present. This new system has a few novelties. 
</P><UL><LI> First of all, transcription begins at a new hybrid promoter we made: <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K145150" class="external text" rel="nofollow"><B>BBa_K145150</B></A>. This promoter is repressed by c2 P22, which is produced by the memory in the OFF state, making premature activation and cell death impossible. Besides this repression, the promoter is activated by the HSL-LuxR complex originating from a previously activated timer. The promoter behaves as shown schematically below. 
</LI></UL><UL><LI> Second, LuxR is now no longer constitutively produced but is placed behind the beforementioned hybrid promoter. This construction mimicks more closely the natural system where LuxR is upregulated when a threshold amount of HSL is present. Plus it also increases the time it takes to activate ccdB, lengthening the timer. The system will now auto-activate if enough HSL is present and the memory is in the ON state.
</LI></UL><UL><LI> Third, the ccdB coding region is also downstream of the hybrid promoter and is thus also subject to the regulation explained above; c2 P22 repression and HSL-LuxR auto-activation. One difference is that the polymerase must first read through a bad terminator with about 60% efficiency before reaching this coding region. Another difference is in the ribosome binding sites preceding both coding regions. Where LuxR can be translated from a RBS with a relative efficiency of 1.00, the ccdB frame can only be read from a 0.01 efficiency RBS.
</LI></UL><H3><SPAN class="mw-headline" id="Describing_the_system_2">Describing the system</SPAN></H3><P>see also: <A href="https://2008.igem.org/Team:KULeuven/Project/CellDeath" class="external text" rel="nofollow">Project:Cell Death</A></P><H4><SPAN class="mw-headline" id="ODE.27s_2">ODE's</SPAN></H4><H4><SPAN class="mw-headline" id="Parameters_2"> Parameters </SPAN></H4><TABLE width="80%" style="border: 1px solid #003E81; background-color: #EEFFFF;"><CAPTION><I>Parameter values (Cell Death)</I></CAPTION><TBODY><TR><TH width="15%"> Name
</TH><TH width="15%"> Value
</TH><TH width="40%"> Comments
</TH><TH width="10%"> Reference
</TH></TR><TR><TH colspan="4" style="border-bottom: 1px solid #003E81;"> Degradation Rates
</TH></TR><TR><TD> d<SUB>LuxR</SUB></TD><TD> 9.627044174E-5 s<SUP>-1</SUP></TD><TD> no LVA tag, so longer lifetime (t<SUB>1/2</SUB> = 2h)
</TD></TR><TR><TD> d<SUB>complex</SUB></TD><TD> 9.627044174E-5 s<SUP>-1</SUP></TD><TD> complex of HSL and LuxR degrades, giving back HSL
</TD></TR><TR><TD> d<SUB>RNA_LuxR</SUB></TD><TD> 0.00227 s<SUP>-1</SUP></TD><TD><A href="https://2008.igem.org/Team:KULeuven/Model/CellDeath#References" class="external text" rel="nofollow">[3]</A></TD></TR><TR><TD> d<SUB>ccdB</SUB></TD><TD> 7.7E-5 s<SUP>-1</SUP></TD><TD> stable in the absence of ccdA
</TD><TD><A href="https://2008.igem.org/Team:KULeuven/Model/CellDeath#References" class="external text" rel="nofollow">[4]</A></TD></TR><TR><TD> d<SUB>RNA_ccdB</SUB></TD><TD> 0.00231 s<SUP>-1</SUP></TD><TD><A href="https://2008.igem.org/Team:KULeuven/Model/CellDeath#References" class="external text" rel="nofollow">[3]</A></TD></TR><TR><TD> d<SUB>HSL</SUB></TD><TD> 1.02E-6 s<SUP>-1</SUP></TD><TD> very stable in the medium, average lifetime of 185h
</TD><TD><A href="https://2008.igem.org/Team:KULeuven/Model/CellDeath#References" class="external text" rel="nofollow">[10]</A></TD></TR><TR><TH colspan="4" style="border-bottom: 1px solid #003E81;"> Association/Dissociation/Reaction Rates
</TH></TR><TR><TD> k<SUB>ass</SUB> (HSL+LuxR)
</TD><TD> 0.002372 s<SUP>-1</SUP></TD><TD> association rate of HSL with LuxR (estimate from K<SUB>M</SUB> but recalculated to remove molar dimension)
</TD><TD><A href="https://2008.igem.org/Team:KULeuven/Model/CellDeath#References" class="external text" rel="nofollow">[2]</A></TD></TR><TR><TD> k<SUB>diss</SUB> (HSL-LuxR)
</TD><TD> 1.0 s<SUP>-1</SUP></TD><TD> dissociation rate of the HSL-LuxR complex (estimate from K<SUB>M</SUB>)
</TD><TD><A href="https://2008.igem.org/Team:KULeuven/Model/CellDeath#References" class="external text" rel="nofollow">[2]</A></TD></TR><TR><TD> k<SUB>ass</SUB> (HSL+lactonase)
</TD><TD> 0.002372 s<SUP>-1</SUP></TD><TD> association rate of HSL with lactonase (estimate from K<SUB>M</SUB> but recalculated to remove molar dimension)
</TD><TD><A href="https://2008.igem.org/Team:KULeuven/Model/CellDeath#References" class="external text" rel="nofollow">[11]</A></TD></TR><TR><TD> k<SUB>diss</SUB> (HSL-lactonase)
</TD><TD> 4470.0 s<SUP>-1</SUP></TD><TD> dissociation rate of the HSL-lactonase complex (estimate from K<SUB>d</SUB>)
</TD><TD><A href="https://2008.igem.org/Team:KULeuven/Model/CellDeath#References" class="external text" rel="nofollow">[11]</A></TD></TR><TR><TD> k<SUB>cat</SUB> (HSL&gt;&gt;hydroxy-acid)
</TD><TD> 29 s<SUP>-1</SUP></TD><TD> lactonase catalyzed transformation of HSL to a hydroxy-acid
</TD><TD><A href="https://2008.igem.org/Team:KULeuven/Model/CellDeath#References" class="external text" rel="nofollow">[11]</A></TD></TR><TR><TH colspan="4" style="border-bottom: 1px solid #003E81;"> Dissociation Constants
</TH></TR><TR><TD> K<SUB>HSL_LuxR</SUB></TD><TD> 1E-6 [M]
</TD><TD> HSL binding to LuxR
</TD><TD><A href="https://2008.igem.org/Team:KULeuven/Model/CellDeath#References" class="external text" rel="nofollow">[2]</A></TD></TR><TR><TD> K<SUB>HSL_LuxR-promoter</SUB></TD><TD> 4.05E-6 [M]
</TD><TD> binding of HSL_LuxR complex to the Lux Promotor
</TD><TD><A href="https://2008.igem.org/Team:KULeuven/Model/CellDeath#References" class="external text" rel="nofollow">[2]</A></TD></TR><TR><TD> K<SUB>HSL_lactonase</SUB></TD><TD> 4.47E-3 [M]
</TD><TD> K<SUB>M</SUB> of lactonase with 3OC6HSL
</TD><TD><A href="https://2008.igem.org/Team:KULeuven/Model/CellDeath#References" class="external text" rel="nofollow">[11]</A></TD></TR><TR><TH colspan="4" style="border-bottom: 1px solid #003E81;"> Hybrid Promoter
</TH></TR><TR><TD> K<SUB>C2 P22</SUB></TD><TD> 2.6E-10 [M]
</TD><TD> Dissociation constant for P22 c2 promoter binding
</TD><TD><A href="https://2008.igem.org/Team:KULeuven/Model/CellDeath#References" class="external text" rel="nofollow">[6]</A></TD></TR><TR><TD> K<SUB>HSL_LuxR</SUB></TD><TD> 4.05E-6 [M]
</TD><TD> Dissociation constant for HSL-LuxR complex promoter binding
</TD><TD><A href="https://2008.igem.org/Team:KULeuven/Model/CellDeath#References" class="external text" rel="nofollow">[1] [12]</A></TD></TR><TR><TD> Hill (P22 c2)
</TD><TD> 2
</TD><TD> Hill coefficient for P22 c2 binding
</TD><TD><A href="https://2008.igem.org/Team:KULeuven/Model/CellDeath#References" class="external text" rel="nofollow">[6]</A></TD></TR><TR><TD> Hill (HSL-LuxR)
</TD><TD> 2
</TD><TD> Hill coefficient for HSL-LuxR binding
</TD><TD><A href="https://2008.igem.org/Team:KULeuven/Model/CellDeath#References" class="external text" rel="nofollow">[1] [12]</A></TD></TR><TR><TD> k<SUB>leaky</SUB></TD><TD> 5E-4 s<SUP>-1</SUP></TD><TD> transcription rate from the hybrid promoter in the <B>unactivated</B> ON state
</TD><TD><A href="https://2008.igem.org/Team:KULeuven/Model/CellDeath#References" class="external text" rel="nofollow">[1]</A></TD></TR><TR><TD> k<SUB>max</SUB></TD><TD> 0.0030  s<SUP>-1</SUP></TD><TD> maximal transcription rate for the fully activated ON hybrid promoter (might be higher)
</TD><TD><A href="https://2008.igem.org/Team:KULeuven/Model/CellDeath#References" class="external text" rel="nofollow">[5]</A></TD></TR><TR><TD> Terminator eff
</TD><TD> 60.8%
</TD><TD> percentage of transcription termination at the B0014 double terminator in front of ccdB
</TD><TD><A href="https://2008.igem.org/Team:KULeuven/Model/CellDeath#References" class="external text" rel="nofollow">[7]</A></TD></TR><TR><TH colspan="4" style="border-bottom: 1px solid #003E81;"> Translation Rates
</TH></TR><TR><TD> k<SUB>LuxR translation</SUB></TD><TD> 0.556 s<SUP>-1</SUP></TD><TD> Translation rate for LuxR, B0034 RBS (relative efficiency 1.0)
</TD><TD><A href="https://2008.igem.org/Team:KULeuven/Model/CellDeath#References" class="external text" rel="nofollow">[9]</A></TD></TR><TR><TD> k<SUB>ccdB translation</SUB></TD><TD> 0.00556 s<SUP>-1</SUP></TD><TD> Translation rate for ccdB, B0033 RBS (relative efficiency 0.01)
</TD><TD><A href="https://2008.igem.org/Team:KULeuven/Model/CellDeath#References" class="external text" rel="nofollow">[9]</A></TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Models_2">Models</SPAN></H3><H4><SPAN class="mw-headline" id="CellDesigner_.28SBML_file.29_2"> CellDesigner (<A href="https://static.igem.org/mediawiki/2008/1/14/Celldeath_CellDesigner_final.zip" class="external text" rel="nofollow">SBML file</A>)</SPAN></H4><H4><SPAN class="mw-headline" id="Matlab_.28SBML_file.29_2"> Matlab (<A href="https://static.igem.org/mediawiki/2008/3/36/CellDeath_Matlab_Final.zip" class="external text" rel="nofollow">SBML file</A>)</SPAN></H4><H3><SPAN class="mw-headline" id="Simulations"> Simulations </SPAN></H3><P>To simulate the cell death, the input signal TetR is all the time low (5E-5 s<SUP>-1</SUP>) except from 50000 till 80000 seconds,from 180000 till 190000 seconds and from 250000 till 270000 seconds (0.0125 s<SUP>-1</SUP>). The left figure shows us that a high input signal will lower the amount of HSL (the HSL will be converted into hydroxy acid by lactonase). This will increase the amount of free LuxR because there is not enough HSL to form the HSL-LuxR complex. During the pulse, the right figure shows us an increase of the amount of CcdB. Whenthe memory switches from state zero to state one, the amount of CIIP22 decreases which makes it possible to produce a small amount of CcdB (background signal). 
When the input signal is low again, the amount of HSL increases. This extra HSL will form with LuxR the complex which lowers the amount of free LuxR. The higher amount of the complex promotes the production of CcdB which increases to a (hopefully) deathly level. 
A new pulse lowers the amount of HSL and the complex and increases the amount of LuxR for the same reasons as before. The second pulse is smal in length and decreases the amount of CcdB only partially. The third pulse is long enough for a total reset of the entire system.
</P><P>All graphs have amounts (number of molecules in the cell) plotted vs time, measured in seconds. 
</P><H3><SPAN class="mw-headline" id="References"> References </SPAN></H3><P><TITLE>Bibliography</TITLE><TABLE style="border-collapse:collapse;line-height:1.1em;"><TBODY><TR style="vertical-align:top;"><TD>[1]</TD><TD style="padding-left:4pt;">L.C.M. Antunes et al., “A Mutational Analysis Defines Vibrio fischeri LuxR Binding Sites,” <SPAN style="font-style:italic;">J. Bacteriol.</SPAN>, Dec. 2007, pp. JB.01443-07. </TD></TR><TR><TD colspan="2"> </TD></TR><TR style="vertical-align:top;"><TD>[2]</TD><TD style="padding-left:4pt;">N. Qin et al., “Analysis of LuxR Regulon Gene Expression during Quorum Sensing in Vibrio fischeri,” <SPAN style="font-style:italic;">J. Bacteriol.</SPAN>,  vol. 189, Jun. 2007, pp. 4127-4134. </TD></TR><TR><TD colspan="2"> </TD></TR><TR style="vertical-align:top;"><TD>[3]</TD><TD style="padding-left:4pt;">J.A. Bernstein et al., “Global analysis of mRNA decay and abundance in Escherichia coli at single-gene resolution using two-color fluorescent DNA microarrays,” <SPAN style="font-style:italic;">Proceedings of the National Academy of Sciences of the United States of America</SPAN>,  vol. 99, Jul. 2002, pp. 9697–9702. </TD></TR><TR><TD colspan="2"> </TD></TR><TR style="vertical-align:top;"><TD>[4]</TD><TD style="padding-left:4pt;">L. Van Melderen, P. Bernard, and M. Couturier, “Lon-dependent proteolysis of CcdA is the key control for activation of CcdB in plasmid-free segregant bacteria,” <SPAN style="font-style:italic;">Molecular Microbiology</SPAN>,  vol. 11, Mar. 1994, pp. 1151-7. </TD></TR><TR><TD colspan="2"> </TD></TR><TR style="vertical-align:top;"><TD>[5]</TD><TD style="padding-left:4pt;">M. Bon, S.J. McGowan, and P.R. Cook, “Many expressed genes in bacteria and yeast are transcribed only once per cell cycle,” <SPAN style="font-style:italic;">FASEB J.</SPAN>,  vol. 20, Aug. 2006, pp. 1721-1723. </TD></TR><TR><TD colspan="2"> </TD></TR><TR style="vertical-align:top;"><TD>[6]</TD><TD style="padding-left:4pt;">J. De Anda, A. Poteete, and R. Sauer, “P22 c2 repressor. Domain structure and function,” <SPAN style="font-style:italic;">J. Biol. Chem.</SPAN>,  vol. 258, Sep. 1983, pp. 10536-10542. </TD></TR><TR><TD colspan="2"> </TD></TR><TR style="vertical-align:top;"><TD>[7]</TD><TD style="padding-left:4pt;">“Part:BBa B0014 - partsregistry.org”; http://partsregistry.org/Part:BBa_B0014.</TD></TR><TR><TD colspan="2"> </TD></TR><TR style="vertical-align:top;"><TD>[8]</TD><TD style="padding-left:4pt;">“Part:BBa B0033 - partsregistry.org”; http://partsregistry.org/Part:BBa_B0033.</TD></TR><TR><TD colspan="2"> </TD></TR><TR style="vertical-align:top;"><TD>[9]</TD><TD style="padding-left:4pt;">“Part:BBa B0034 - partsregistry.org”; http://partsregistry.org/Part:BBa_B0034.</TD></TR><TR><TD colspan="2"> </TD></TR><TR style="vertical-align:top;"><TD>[10]</TD><TD style="padding-left:4pt;">Y. Wang and J.R. Leadbetter, “Rapid Acyl-Homoserine Lactone Quorum Signal Biodegradation in Diverse Soils,” <SPAN style="font-style:italic;">Appl. Environ. Microbiol.</SPAN>,  vol. 71, Mar. 2005, pp. 1291-1299. </TD></TR><TR><TD colspan="2"> </TD></TR><TR style="vertical-align:top;"><TD>[11]</TD><TD style="padding-left:4pt;">L. Wang et al., “Specificity and enzyme kinetics of the quorum-quenching AHL-lactonase,” <SPAN style="font-style:italic;">J. Biol. Chem.</SPAN>, Jan. 2004, p. M311194200. </TD></TR><TR><TD colspan="2"> </TD></TR><TR style="vertical-align:top;"><TD>[12]</TD><TD style="padding-left:4pt;">“Tokyo/AHL assay - IGEM07”; https://2007.igem.org/Tokyo/AHL_assay.</TD></TR><TR><TD colspan="2"> </TD></TR></TBODY></TABLE></P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2008.igem.org/Team:KULeuven/Model/Cell_Death">http://2008.igem.org/Team:KULeuven/Model/Cell_Death</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:KULeuven/Model/Cell_Death" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:KULeuven/Model/Cell_Death" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:KULeuven/Model/Cell_Death&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:KULeuven/Model/Cell_Death&amp;oldid=51146" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2008.igem.org:Privacy_policy" title="2008.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2008.igem.org:General_disclaimer" title="2008.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>