<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_KULeuven_Model_Memory"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:KULeuven/Model/Memory">Page               </A></LI><LI><A href="/Talk:Team:KULeuven/Model/Memory">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:KULeuven/Model/Memory&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:KULeuven/Model/Memory&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:KULeuven/Model/Memory" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:KULeuven/Model/Memory</H1><DIV id="bodyContent"><H3 id="siteSub">From 2008.igem.org</H3><DIV id="ddToggle">  dock/undock dropdown  </DIV><DIV id="header"><DIV id="utilNav"><UL><LI id="homepage"><A title="Return to the KULeuven iGEM 2008 homepage" href="https://2008.igem.org/Team:KULeuven">Home</A></LI><LI id="reports"><A title="Keep track of our progress throughout the summer" href="https://2008.igem.org/Team:KULeuven/Meeting_Reports">Reports</A></LI><LI id="calendar"><A title="View important events" href="https://2008.igem.org/Team:KULeuven/Meeting_Calendar">Calendar</A></LI></UL></DIV><DIV id="suppNav"><UL><LI id="abstract"><A title="See what's Dr.Coli all about" href="https://2008.igem.org/Team:KULeuven/Project"><STRONG>Project</STRONG> abstract</A></LI><LI id="team"><A title="List of all our team participants" href="https://2008.igem.org/Team:KULeuven/Team/Students"><STRONG>Meet</STRONG> the team</A></LI><LI id="guestbook"><A title="Take a peek at our guestbook" href="https://2008.igem.org/Team:KULeuven/Guestbook"><STRONG>Leave</STRONG> a message</A></LI><LI id="mail"><A title="Questions, want to cooperate with our team or drop by? Let us know" href="mailto:igem@kuleuven.be?SUBJECT=IGEM%202008"><STRONG>Mail</STRONG> us</A></LI></UL></DIV><DIV id="mainNav"><UL id="ddNav"><LI><A>The Team</A><DIV><A href="https://2008.igem.org/Team:KULeuven/Team/LabsandGroups">Research Labs and Groups</A><A href="https://2008.igem.org/Team:KULeuven/Team/Students">Students</A><A href="https://2008.igem.org/Team:KULeuven/Team/Instructors">Instructors</A><A href="https://2008.igem.org/Team:KULeuven/Team/Advisors">Advisors</A><A href="https://2008.igem.org/Team:KULeuven/Team/Pictures">Pictures</A></DIV></LI><LI><A>The Project</A><DIV><A href="https://2008.igem.org/Team:KULeuven/Project">Summary</A><SPAN><A>Components</A><DIV><A href="https://2008.igem.org/Team:KULeuven/Project/Input">Input</A><A href="https://2008.igem.org/Team:KULeuven/Project/Output">Output</A><A href="https://2008.igem.org/Team:KULeuven/Project/Filter">Filter</A><A href="https://2008.igem.org/Team:KULeuven/Project/Inverter">InverTimer</A><A href="https://2008.igem.org/Team:KULeuven/Project/Reset">Reset</A><A href="https://2008.igem.org/Team:KULeuven/Project/CellDeath">Cell Death</A><A href="https://2008.igem.org/Team:KULeuven/Project/Memory">Memory</A></DIV></SPAN><A href="https://2008.igem.org/Team:KULeuven/Safety">Safety</A><A href="https://2008.igem.org/Team:KULeuven/Evaluation">End Evaluation</A><A href="https://2008.igem.org/Team:KULeuven/Literature">Literature</A><A href="https://2008.igem.org/Team:KULeuven/Brainstorm">Brainstorm</A></DIV></LI><LI><A>Ethics</A><DIV><A href="https://2008.igem.org/Team:KULeuven/Ethics/Introduction">Introduction</A><SPAN><A>Three A's</A><DIV><A href="https://2008.igem.org/Team:KULeuven/Ethics/Threeas/Defining">Defining the three A's</A><A href="https://2008.igem.org/Team:KULeuven/Ethics/Threeas/Physics">Developments in modern physics</A><A href="https://2008.igem.org/Team:KULeuven/Ethics/Threeas/Biology">Developments in biology</A></DIV></SPAN><SPAN><A>Concerns &amp; Issues</A><DIV><A href="https://2008.igem.org/Team:KULeuven/Ethics/Concerns/Defining">Defining synthetic biology</A><A href="https://2008.igem.org/Team:KULeuven/Ethics/Concerns/Framework">A framework</A><A href="https://2008.igem.org/Team:KULeuven/Ethics/Concerns/Concerns">Concerns</A><A href="https://2008.igem.org/Team:KULeuven/Ethics/Concerns/Issues">Issues</A><A href="https://2008.igem.org/Team:KULeuven/Ethics/Concerns/Answers">The need for answers</A></DIV></SPAN><SPAN><A>Biological Robotics</A><DIV><A href="https://2008.igem.org/Team:KULeuven/Ethics/Robotics/KULeuven">KULeuven iGEM 2008 project</A><A href="https://2008.igem.org/Team:KULeuven/Ethics/Robotics/Robotics">Biological robotics</A></DIV></SPAN><A href="https://2008.igem.org/Team:KULeuven/Ethics/Afterword">Afterword</A><A href="https://2008.igem.org/Team:KULeuven/Ethics/References">References</A></DIV></LI><LI><A>Submitted Parts</A><DIV><A href="http://partsregistry.org/cgi/partsdb/pgroup.cgi?pgroup=iGEM2008&amp;group=KULeuven">Sandbox</A></DIV></LI><LI><A>Modeling</A><DIV><A href="https://2008.igem.org/Team:KULeuven/Model/Overview">Overview</A><A href="https://2008.igem.org/Team:KULeuven/Model/KineticConstants">Kinetic Constants</A><SPAN><A>Components</A><DIV><A href="https://2008.igem.org/Team:KULeuven/Model/Output">Output</A><A href="https://2008.igem.org/Team:KULeuven/Model/Filter">Filter</A><A href="https://2008.igem.org/Team:KULeuven/Model/Inverter">InverTimer</A><A href="https://2008.igem.org/Team:KULeuven/Model/Reset">Reset</A><A href="https://2008.igem.org/Team:KULeuven/Model/CellDeath">Cell Death</A><A href="https://2008.igem.org/Team:KULeuven/Model/Memory">Memory</A></DIV></SPAN><A href="https://2008.igem.org/Team:KULeuven/Model/FullModel">Full Model</A><A href="https://2008.igem.org/Team:KULeuven/Model/Sensitivity">Sensitivity Analysis</A><A href="https://2008.igem.org/Team:KULeuven/Model/MultiCell">Multi-cell Model</A><A href="https://2008.igem.org/Team:KULeuven/Model/Diffusion">Diffusion</A></DIV></LI><LI><A>Data Analysis</A><DIV><A href="https://2008.igem.org/Team:KULeuven/Data/Overview">Overview</A><SPAN><A>New Parts</A><DIV><A href="https://2008.igem.org/Team:KULeuven/Data/GFP">GFP (LVA-tag)</A><A href="https://2008.igem.org/Team:KULeuven/Data/Other_Parts">Other</A></DIV></SPAN><SPAN><A>Components</A><DIV><A href="https://2008.igem.org/Team:KULeuven/Data/Input">Input</A><A href="https://2008.igem.org/Team:KULeuven/Data/Output">Output</A><A href="https://2008.igem.org/Team:KULeuven/Data/Other_Components">Other</A></DIV></SPAN></DIV></LI><LI><A>Software</A><DIV><A href="https://2008.igem.org/Team:KULeuven/Software/MultiCell">Multi-cell Toolbox</A><A href="https://2008.igem.org/Team:KULeuven/Software/Simbiology2LaTeX">Simbiology2LaTeX Toolbox</A></DIV></LI><LI><A>Notebook</A><DIV><A href="https://2008.igem.org/Team:KULeuven/Protocols">Protocols</A><SPAN><A>Reports</A><DIV><A href="https://2008.igem.org/Team:KULeuven/Meeting Reports">Daily</A><A href="https://2008.igem.org/Team:KULeuven/Weekly Meetings">Weekly</A></DIV></SPAN><SPAN><A>Lab Data</A><DIV><A href="https://2008.igem.org/Team:KULeuven/Freezer">Freezer</A><A href="https://2008.igem.org/Team:KULeuven/Primers">Primers</A><A href="https://2008.igem.org/Team:KULeuven/Ligation">Ligation</A></DIV></SPAN><A href="https://2008.igem.org/Team:KULeuven/Tools">Tools</A><A href="https://2008.igem.org/Team:KULeuven/Press">Press</A><A href="https://2008.igem.org/Team:KULeuven/Acknowledgments">Acknowledgments</A></DIV></LI></UL></DIV></DIV><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Memory"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Memory</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Position_in_the_system"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Position in the system</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Describing_the_system"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Describing the system</SPAN></A></LI><LI class="toclevel-3 tocsection-4"><A href="#ODE.27s"><SPAN class="tocnumber">1.2.1</SPAN><SPAN class="toctext">ODE's</SPAN></A></LI><LI class="toclevel-3 tocsection-5"><A href="#Parameters"><SPAN class="tocnumber">1.2.2</SPAN><SPAN class="toctext">Parameters</SPAN></A></LI></UL><LI class="toclevel-2 tocsection-6"><A href="#Models"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Models</SPAN></A></LI><LI class="toclevel-3 tocsection-7"><A href="#CellDesigner_.28SBML_file.29"><SPAN class="tocnumber">1.3.1</SPAN><SPAN class="toctext">CellDesigner (SBML file)</SPAN></A></LI><LI class="toclevel-3 tocsection-8"><A href="#Matlab"><SPAN class="tocnumber">1.3.2</SPAN><SPAN class="toctext">Matlab</SPAN></A></LI><LI class="toclevel-2 tocsection-9"><A href="#Problem"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">Problem</SPAN></A></LI><LI class="toclevel-2 tocsection-10"><A href="#Alternative"><SPAN class="tocnumber">1.5</SPAN><SPAN class="toctext">Alternative</SPAN></A></LI><LI class="toclevel-2 tocsection-11"><A href="#References"><SPAN class="tocnumber">1.6</SPAN><SPAN class="toctext">References</SPAN></A></LI><LI class="toclevel-1 tocsection-12"><A href="#Memory_-_episode_2"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Memory - episode 2</SPAN></A></LI><LI class="toclevel-2 tocsection-13"><A href="#Position_in_the_system_2"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Position in the system</SPAN></A></LI><LI class="toclevel-2 tocsection-14"><A href="#Describing_the_system_2"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">Describing the system</SPAN></A></LI><LI class="toclevel-3 tocsection-15"><A href="#ODE.27s_2"><SPAN class="tocnumber">2.2.1</SPAN><SPAN class="toctext">ODE's</SPAN></A></LI><LI class="toclevel-3 tocsection-16"><A href="#Parameters_2"><SPAN class="tocnumber">2.2.2</SPAN><SPAN class="toctext">Parameters</SPAN></A></LI><LI class="toclevel-2 tocsection-17"><A href="#Models_2"><SPAN class="tocnumber">2.3</SPAN><SPAN class="toctext">Models</SPAN></A></LI><LI class="toclevel-3 tocsection-18"><A href="#CellDesigner.28SBML_file.29"><SPAN class="tocnumber">2.3.1</SPAN><SPAN class="toctext">CellDesigner(SBML file)</SPAN></A></LI><LI class="toclevel-3 tocsection-19"><A href="#Matlab_.28SBML_file.29"><SPAN class="tocnumber">2.3.2</SPAN><SPAN class="toctext">Matlab (SBML file)</SPAN></A></LI><LI class="toclevel-2 tocsection-20"><A href="#Simulations"><SPAN class="tocnumber">2.4</SPAN><SPAN class="toctext">Simulations</SPAN></A></LI><LI class="toclevel-2 tocsection-21"><A href="#Mathematical_Analysis_.28Maple_file.29"><SPAN class="tocnumber">2.5</SPAN><SPAN class="toctext">Mathematical Analysis (Maple file)</SPAN></A></LI><LI class="toclevel-2 tocsection-22"><A href="#References_2"><SPAN class="tocnumber">2.6</SPAN><SPAN class="toctext">References</SPAN></A></LI></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Memory"> Memory </SPAN></H2><H3><SPAN class="mw-headline" id="Position_in_the_system"> Position in the system </SPAN></H3><P>This system must activate the cell death system after one light pulse. As long as there is no light, there is no P2ogr, no CIIP22 and a lot of antimRNA_LuxI. The antimRNA_LuxI blocks the cell death system. When light is turned on OmpF increases. This causes P2ogr and CIIP22 to increase and antimRNA_LuxI to decrease. This activates the system. When light is turned off, the P2ogr concentration is large enough to maintain itself. This way antimRNA doesn't increase. The system stays activated.
</P><H3><SPAN class="mw-headline" id="Describing_the_system"> Describing the system </SPAN></H3><H4><SPAN class="mw-headline" id="ODE.27s"> ODE's </SPAN></H4><H4><SPAN class="mw-headline" id="Parameters"> Parameters </SPAN></H4><TABLE width="80%" style="border: 1px solid #003E81; background-color: #EEFFFF;"><CAPTION><I>Parameter values (Memory)</I></CAPTION><TBODY><TR><TH width="15%"> Name
</TH><TH width="15%"> Value
</TH><TH width="40%"> Comments
</TH><TH width="10%"> Reference
</TH></TR><TR><TH colspan="4" style="border-bottom: 1px solid #003E81;"> Degradation Rates
</TH></TR><TR><TD> d<SUB>P2ogr</SUB></TD><TD> 0.002265 s<SUP>-1</SUP></TD><TD><A href="https://2008.igem.org/Team:KULeuven/Model/Memory#References" class="external text" rel="nofollow">[1]</A></TD></TR><TR><TD> d<SUB>RNA_P2ogr</SUB></TD><TD> 0.002265 s<SUP>-1</SUP></TD><TD><A href="https://2008.igem.org/Team:KULeuven/Model/Memory#References" class="external text" rel="nofollow">[2]</A></TD></TR><TR><TD> d<SUB>P22CII</SUB></TD><TD> 0.002311 s<SUP>-1</SUP></TD><TD> This value is too low. The correct value is used in the final model.
</TD><TD><A href="https://2008.igem.org/Team:KULeuven/Model/Memory#References" class="external text" rel="nofollow">[3]</A></TD></TR><TR><TD> d<SUB>RNA_P22CII</SUB></TD><TD> 0,0022651 s<SUP>-1</SUP></TD><TD><A href="https://2008.igem.org/Team:KULeuven/Model/Memory#References" class="external text" rel="nofollow">[2]</A></TD></TR><TR><TD> d<SUB>antimRNA_luxI</SUB></TD><TD> 0.0045303 s<SUP>-1</SUP></TD><TD> estimate: because this RNA isn't translated, it degrades faster
</TD><TD><A href="https://2008.igem.org/Team:KULeuven/Model/Memory#References" class="external text" rel="nofollow">[2]</A></TD></TR><TR><TH colspan="4" style="border-bottom: 1px solid #003E81;"> Transcription Rates
</TH></TR><TR><TD> k<SUB>P2ogr</SUB></TD><TD> 0.0125 s<SUP>-1</SUP></TD><TD> estimate
</TD><TD><A href="https://2008.igem.org/Team:KULeuven/Model/Memory#References" class="external text" rel="nofollow">[4]</A></TD></TR><TR><TD> k<SUB>P22CII</SUB></TD><TD> 0.0125 s<SUP>-1</SUP></TD><TD> estimate
</TD><TD><A href="https://2008.igem.org/Team:KULeuven/Model/Memory#References" class="external text" rel="nofollow">[4]</A></TD></TR><TR><TD> k<SUB>AntimRNA_LuxI</SUB></TD><TD> 0.0094 s<SUP>-1</SUP></TD><TD> estimate
</TD><TD><A href="https://2008.igem.org/Team:KULeuven/Model/Memory#References" class="external text" rel="nofollow">[5]</A></TD></TR><TR><TH colspan="4" style="border-bottom: 1px solid #003E81;"> Dissociation Constants
</TH></TR><TR><TD> K<SUB>P2ogr</SUB></TD><TD> 4.2156
</TD><TD> Used in two reactions for activator control at the transcription of P2ogr mRNA and CIIP22 mRNA
</TD><TD><A href="https://2008.igem.org/Team:KULeuven/Model/Memory#References" class="external text" rel="nofollow">[6]</A></TD></TR><TR><TD> K<SUB>R0053_P22CII</SUB></TD><TD> 0.1099
</TD><TD><A href="https://2008.igem.org/Team:KULeuven/Model/Memory#References" class="external text" rel="nofollow">[3]</A></TD></TR><TR><TH colspan="4" style="border-bottom: 1px solid #003E81;"> Hill Cooperativity
</TH></TR><TR><TD> n</TD><TD> 2
</TD><TD> Used for all reactions throughout the memory submodel using Hill kinetics
</TD><TD><A href="https://2008.igem.org/Team:KULeuven/Model/Memory#References" class="external text" rel="nofollow">[1]</A></TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Models"> Models </SPAN></H3><H4><SPAN class="mw-headline" id="CellDesigner_.28SBML_file.29"> CellDesigner (<A href="https://static.igem.org/mediawiki/2008/4/49/Memory_CellDesigner.zip" class="external text" rel="nofollow">SBML file</A>)</SPAN></H4><H4><SPAN class="mw-headline" id="Matlab"> Matlab </SPAN></H4><H3><SPAN class="mw-headline" id="Problem"> Problem </SPAN></H3><P>The OmpF promoter is not ideal. When there is no light the transcription rate is still 0.00005 s<SUP>-1</SUP>. This means that P2ogr will slowly build up, activating the system. In this case the memory is in 0-state when the stationary state isn't reached yet. The 1-state is the stationary state. So the system automatically ends up in state 1 after some time (300s). This can be seen in the figure below.
</P><DL><DD><B>Figure: CIIP22(purple), P2ogr(green), AntimRNA(pink).</B></DD></DL><P>The system can only stay in 0-state for 300 sec. This makes it completely useless.
</P><H3><SPAN class="mw-headline" id="Alternative"> Alternative </SPAN></H3><P>In the previous system the 0-state isn't actively maintained. It's just 'not stationary state'. So we need to search mathematical system that has 2 stationary states. A possible solution is given below.
</P><DL><DD><B>Figure: Part representation of alternative system</B></DD></DL><P>When this system starts Rep build up because Rep represses the Act promoter better then Act represses the Rep promoter. The Rep concentration stays high and the Act stays low. This is the 0-state. When there is light, the OmpF promoter is activated and the Act concentration is increased. This represses Rep promoter. The Rep concentration decreases and the Act promoter is activated. The Act concentration keeps increasing. When the light pulse ends the Act concentration is high enough to repress the Rep promoter, Act concentration stays high and Rep concentration stays low. This is the 1-state.
</P><P>CellDesigner gives the following simulation when OmpF transcription rate changes from 0.0001 to 0.01 at t=6000 sec for 2000 sec.
</P><DL><DD><B>Figure: Celldesigner simulation of the alterative system. Act(grey), Rep(yellow) </B></DD></DL><P>The OmpF peak causes the Act concentration to rise and the Rep to decrease. The high Act concentration keeps the Rep concentration low. This causes the Act concentration to stay high.
</P><H3><SPAN class="mw-headline" id="References"> References </SPAN></H3><P><TITLE>Bibliography</TITLE><TABLE style="border-collapse:collapse;line-height:1.1em;"><TBODY><TR style="vertical-align:top;"><TD>[1]</TD><TD style="padding-left:4pt;">“ETHZ/Parameters - IGEM07”; https://2007.igem.org/ETHZ/Parameters.</TD></TR><TR><TD colspan="2"> </TD></TR><TR style="vertical-align:top;"><TD>[2]</TD><TD style="padding-left:4pt;">J.A. Bernstein et al., “Global analysis of mRNA decay and abundance in Escherichia coli at single-gene resolution using two-color fluorescent DNA microarrays,” <SPAN style="font-style:italic;">Proceedings of the National Academy of Sciences of the United States of America</SPAN>,  vol. 99, Jul. 2002, pp. 9697–9702. </TD></TR><TR><TD colspan="2"> </TD></TR><TR style="vertical-align:top;"><TD>[3]</TD><TD style="padding-left:4pt;">J. De Anda, A. Poteete, and R. Sauer, “P22 c2 repressor. Domain structure and function,” <SPAN style="font-style:italic;">J. Biol. Chem.</SPAN>,  vol. 258, Sep. 1983, pp. 10536-10542. </TD></TR><TR><TD colspan="2"> </TD></TR><TR style="vertical-align:top;"><TD>[4]</TD><TD style="padding-left:4pt;">“Part:BBa I746364 - partsregistry.org”; http://partsregistry.org/Part:BBa_I746364.</TD></TR><TR><TD colspan="2"> </TD></TR><TR style="vertical-align:top;"><TD>[5]</TD><TD style="padding-left:4pt;">“Part:BBa R0053 - partsregistry.org”; http://partsregistry.org/Part:BBa_R0053.</TD></TR><TR><TD colspan="2"> </TD></TR><TR style="vertical-align:top;"><TD>[6]</TD><TD style="padding-left:4pt;">D. Kornitzer, S. Altuvia, and A.B. Oppenheim, “The activity of the CIII regulator of lambdoid bacteriophages resides within a 24-amino acid protein domain.,” <SPAN style="font-style:italic;">Proceedings of the National Academy of Sciences of the United States of America</SPAN>,  vol. 88, Jun. 1991, pp. 5217–5221. </TD></TR><TR><TD colspan="2"> </TD></TR></TBODY></TABLE></P><H2><SPAN class="mw-headline" id="Memory_-_episode_2">Memory - episode 2</SPAN></H2><H3><SPAN class="mw-headline" id="Position_in_the_system_2"> Position in the system </SPAN></H3><P>The Memory must keep the <A href="https://2008.igem.org/Team:KULeuven/Model/Cell_Death" class="external text" rel="nofollow">Cell Death</A> inactivated untill a decent input signal has been received. When there is no input signal, c2 P22 will take control and repress cI 434 and <A href="https://2008.igem.org/Team:KULeuven/Model/Cell_Death" class="external text" rel="nofollow">Cell Death</A>. This is the OFF state in which the memory can remain indefinitely unless an input signal emerges. This causes production of cI 434 (without LVA tag) whih will repress c2 P22 production, allowing cI 434 production to start from the c2 P22 repressible promoter. cI 434 will also start repressing the antisense LuxI production, enabling the <A href="https://2008.igem.org/Team:KULeuven/Model/Inverter" class="external text" rel="nofollow">InverTimer</A> to do its work. After an input signal of about 1000 seconds, enough of the c2 P22 has disappeared in order to make the cI 434 production (LVA) self-sufficient. At this point, the memory has reached the ON state in which it will remain. For a more elaborate description of the Memory's biomolecular workings, please see the <A href="https://2008.igem.org/Team:KULeuven/Project/Memory" class="external text" rel="nofollow">Project/Memory</A> page.
</P><H3><SPAN class="mw-headline" id="Describing_the_system_2"> Describing the system </SPAN></H3><P>This is the same figure as the alternative, but this time filled in for the new system. It conventiently shows that the system is based upon the alternative suggested above while laying out the workings and concretisations in the form of the different BioBricks implemented. Rep is now the c2 P22 repressor while Act is represented by cI 434.
</P><P>The output of this system is antisense RNA production that represses the LuxI translation. c2 P22 has also got a function in repressing <A href="https://2008.igem.org/Team:KULeuven/Model/Cell_Death" class="external text" rel="nofollow">Cell Death</A></P><H4><SPAN class="mw-headline" id="ODE.27s_2">ODE's</SPAN></H4><H4><SPAN class="mw-headline" id="Parameters_2"> Parameters </SPAN></H4><TABLE width="80%" style="border: 1px solid #003E81; background-color: #EEFFFF;"><CAPTION><I>Parameter values (New Memory)</I></CAPTION><TBODY><TR><TH width="15%"> Name
</TH><TH width="15%"> Value
</TH><TH width="40%"> Comments
</TH><TH width="10%"> Reference
</TH></TR><TR><TH colspan="4" style="border-bottom: 1px solid #003E81;"> Degradation Rates
</TH></TR><TR><TD> d<SUB>cIIP22</SUB></TD><TD> d<SUB>LVA</SUB> = 2.814E-4 s<SUP>-1</SUP></TD><TD> LVA-tag reduces lifetime to 40 minutes
</TD><TD><A href="https://2008.igem.org/Team:KULeuven/Model/Memory#References" class="external text" rel="nofollow">[5]</A></TD></TR><TR><TD> d<SUB>mRNA_cIIP22</SUB></TD><TD> 0.0023104906 s<SUP>-1</SUP></TD><TD><A href="https://2008.igem.org/Team:KULeuven/Model/Memory#References" class="external text" rel="nofollow">[3]</A></TD></TR><TR><TD> d<SUB>cI434_LVA</SUB></TD><TD> d<SUB>LVA</SUB> = 2.814E-4 s<SUP>-1</SUP></TD><TD> LVA-tag reduces lifetime to 40 minutes
</TD><TD><A href="https://2008.igem.org/Team:KULeuven/Model/Memory#References" class="external text" rel="nofollow">[5]</A></TD></TR><TR><TD> d<SUB>mRNA_cI434_LVA</SUB></TD><TD> 0.0023104906 s<SUP>-1</SUP></TD><TD><A href="https://2008.igem.org/Team:KULeuven/Model/Memory#References" class="external text" rel="nofollow">[3]</A></TD></TR><TR><TD> d<SUB>cI434</SUB></TD><TD> 9.627044174E-5 s<SUP>-1</SUP></TD><TD> no LVA tag, so longer lifetime (t<SUB>1/2</SUB> = 2h)
</TD></TR><TR><TD> d<SUB>mRNA_cI434</SUB></TD><TD> 0.0023104906 s<SUP>-1</SUP></TD><TD><A href="https://2008.igem.org/Team:KULeuven/Model/Memory#References" class="external text" rel="nofollow">[3]</A></TD></TR><TR><TD> d<SUB>asRNA luxI</SUB></TD><TD> 0.00462 s<SUP>-1</SUP></TD><TD> estimate: because this RNA isn't translated, it degrades faster
</TD><TD><A href="https://2008.igem.org/Team:KULeuven/Model/Memory#References" class="external text" rel="nofollow">[3]</A></TD></TR><TR><TD> d<SUB>LuxI asRNA complex</SUB></TD><TD> 0.00462 s<SUP>-1</SUP></TD><TD> estimate: because this RNA isn't translated, it degrades faster
</TD><TD><A href="https://2008.igem.org/Team:KULeuven/Model/Memory#References" class="external text" rel="nofollow">[3]</A></TD></TR><TR><TH colspan="4" style="border-bottom: 1px solid #003E81;"> c2 P22 transcription (cI 434 regulated)
</TH></TR><TR><TD> k<SUB>transcr</SUB></TD><TD> 0.0125 s<SUP>-1</SUP></TD><TD> estimate of maximal transcription rate, strong promoter
</TD><TD><A href="https://2008.igem.org/Team:KULeuven/Model/Memory#References" class="external text" rel="nofollow">[10]</A></TD></TR><TR><TD> K<SUB>m</SUB> (cI 434)
</TD><TD> 0.8708
</TD><TD> Recalculated to remove [M] dimension from K<SUB>m</SUB>=2E-9 [M]
</TD><TD><A href="https://2008.igem.org/Team:KULeuven/Model/Memory#References" class="external text" rel="nofollow">[12]</A></TD></TR><TR><TH colspan="4" style="border-bottom: 1px solid #003E81;"> cI 434 transcription (c2 P22 regulated)
</TH></TR><TR><TD> k<SUB>transcr</SUB></TD><TD> 0.0040 s<SUP>-1</SUP></TD><TD> estimate of maximal transcription rate, weak-medium promoter
</TD><TD><A href="https://2008.igem.org/Team:KULeuven/Model/Memory#References" class="external text" rel="nofollow">[11]</A></TD></TR><TR><TD> K<SUB>m</SUB> (c2 P22)
</TD><TD> 0.1099
</TD><TD> Recalculated to remove [M]<SUP>2</SUP> dimension from K<SUB>app</SUB>=6.8E-20[M]<SUP>2</SUP></TD><TD><A href="https://2008.igem.org/Team:KULeuven/Model/Memory#References" class="external text" rel="nofollow">[6]</A></TD></TR><TR><TH colspan="4" style="border-bottom: 1px solid #003E81;"> TetR regulated promoter
</TH></TR><TR><TD> TetR_var_transcr_rate
</TD><TD> p(TetR) dependent
</TD><TD> (RiboKey) between 5E-5 and 0.0125 s<SUP>-1</SUP> ~ [aTc]
</TD><TD><A href="https://2008.igem.org/Team:KULeuven/Model/Memory#References" class="external text" rel="nofollow">[4] [9]</A></TD></TR><TR><TH colspan="4" style="border-bottom: 1px solid #003E81;"> Translation rates
</TH></TR><TR><TD> k<SUB>transl_cI434</SUB></TD><TD> 0.0055555 s<SUP>-1</SUP></TD><TD> translation rate for B0033 RBS (0.01 relative efficiency)
</TD><TD><A href="https://2008.igem.org/Team:KULeuven/Model/Memory#References" class="external text" rel="nofollow">[8]</A></TD></TR><TR><TD> k<SUB>transl_cI434_lva</SUB></TD><TD> 0.0388889 s<SUP>-1</SUP></TD><TD> translation rate for B0031 RBS (0.07 relative efficiency)
</TD><TD><A href="https://2008.igem.org/Team:KULeuven/Model/Memory#References" class="external text" rel="nofollow">[7]</A></TD></TR><TR><TD> k<SUB>transl_cIIP22</SUB></TD><TD> 0.0055555 s<SUP>-1</SUP></TD><TD> translation rate for B0033 RBS (0.01 relative efficiency)
</TD><TD><A href="https://2008.igem.org/Team:KULeuven/Model/Memory#References" class="external text" rel="nofollow">[8]</A></TD></TR><TR><TH colspan="4" style="border-bottom: 1px solid #003E81;"> Hill Cooperativity
</TH></TR><TR><TD> n</TD><TD> 2
</TD><TD> Used for all reactions throughout the memory submodel using Hill kinetics
</TD><TD><A href="https://2008.igem.org/Team:KULeuven/Model/Memory#References" class="external text" rel="nofollow">[2]</A></TD></TR><TR><TH colspan="4" style="border-bottom: 1px solid #003E81;"> Antisense LuxI
</TH></TR><TR><TD> k_complex3
</TD><TD> 0.00237 s<SUP>-1</SUP></TD><TD> rate constant for formation of asRNA - LuxI mRNA duplex
</TD><TD><A href="https://2008.igem.org/Team:KULeuven/Model/Memory#References" class="external text" rel="nofollow">[1]</A></TD></TR><TR><TD> K<SUB>mRNA_LuxI:antisense_mRNA</SUB></TD><TD> 4.22E-14 [M]
</TD><TD> Dissociation constant for complex of LuxI mRNA with antisense mRNA
</TD><TD><A href="https://2008.igem.org/Team:KULeuven/Model/Memory#References" class="external text" rel="nofollow">[1]</A></TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Models_2">Models</SPAN></H3><H4><SPAN class="mw-headline" id="CellDesigner.28SBML_file.29"> CellDesigner(<A href="https://static.igem.org/mediawiki/2008/4/49/Memory_CellDesigner.zip" class="external text" rel="nofollow">SBML file</A>)</SPAN></H4><H4><SPAN class="mw-headline" id="Matlab_.28SBML_file.29"> Matlab (<A href="https://static.igem.org/mediawiki/2008/1/1f/Memory_Matlab.zip" class="external text" rel="nofollow">SBML file</A>)</SPAN></H4><H3><SPAN class="mw-headline" id="Simulations"> Simulations </SPAN></H3><P>Multiple simulations have been performed to see if the memory works. If we give a pulse of TetR from
5E-5 to 0.0125 for 10000 sec, we see that the memory switches from its 0-state to its 1-state (left figure): 
cIIP22 amount drops and the cI434 increases. The high cI434 amount will repress the RNA production. If the
pulse lasts for only 1000 sec, the memory remains in its 0-state (middle figure). Simulations show the
minimum time span needed to switch the memory is approximately 1300 sec. The right figure shows us that a sequence
of two pulses of only 1000 sec can also make the memory switch.
</P><P>The graphs have amounts (number of molecules in the cell) plotted vs time, measured in seconds. 
</P><H3><SPAN class="mw-headline" id="Mathematical_Analysis_.28Maple_file.29"> Mathematical Analysis (<A href="https://static.igem.org/mediawiki/2008/8/87/Memory_Maple.zip" class="external text" rel="nofollow">Maple file</A>)</SPAN></H3><P>The existence of the two stable states of the memory (cI434_LVA high and cIIP22 low / cI434_LVA low and cIIP22 high) can be mathematically proven.
</P><P>First we define the equilibrium points of the following differential equation system:
</P><P>The equilibrium points are defined as the points for which all the derivatives are zero. Solving this non-linear system for [OmpF] equal to 0.00005 results in finding the roots of the equation 
</P><P>This equation has three real zeros ([CI434_LVA = 0.002435815407], [CI434_LVA = 11.52604711], [CI434_LVA = 233.6410112]) and two conjugated imaginary zeros ([CI434_LVA = -8.242012862+11.11131953*I], [CI434_LVA = -8.242012862-11.11131953*I]) as can be seen in the following figure:
</P><P>The real roots of the system for all the variables are:
</P><TABLE class="wikitable" style="text-align:center"><CAPTION>Real Roots
</CAPTION><TBODY><TR><TH> Zero 1 </TH><TH> Zero 2 </TH><TH> Zero 3
</TH></TR><TR><TH> [CI434]
</TH><TD> 1.248827120</TD><TD> 1.248827120</TD><TD> 1.248827120
</TD></TR><TR><TH> [CI434-LVA]
</TH><TD> 0.002435815407</TD><TD> 11.52604711</TD><TD> 233.6410112
</TD></TR><TR><TH> [mRNA<SUB>CI434-LVA</SUB>]
</TH><TD> 0.00001804584283</TD><TD> 0.08539121399</TD><TD> 1.730939445
</TD></TR><TR><TH> [CIIP22]
</TH><TD> 34.03962262</TD><TD> 0.4824866257 </TD><TD> 0.001433763049
</TD></TR><TR><TH> [mRNA<SUB>CIIP22</SUB>]
</TH><TD> 1.765288021 </TD><TD> 0.02502165991 </TD><TD> 0.00007435466497
</TD></TR><TR><TH> [asRNA<SUB>CI434</SUB>]
</TH><TD> 0.02164042563 </TD><TD> 0.02164042563 </TD><TD> 0.02164042563
</TD></TR></TBODY></TABLE><P>The stability of the three real zeros is defined by the eigenvalues of the Jacobian of the differential equation system. This Jacobian equals 
</P><P>The eigenvalues for <B>Zero 1</B> are all negative (-0.2334056227e-2, -0.2286362516e-2, -0.3122380843e-3, -0.2645443732e-3, -0.9627000000e-4, -0.2310490600e-2). This is also the case for <B>Zero 3</B> (-0.2319019326e-2, -0.2301889325e-2, -0.2795812737e-3, -0.2967112750e-3, -0.9627000000e-4, -0.2310490600e-2). These two zero's represent the two stable states of the memory system. <B>Zero 2</B> has one positive eigenvalue (0.0002015903724) and is therefore unstable. 
</P><P>The following figure shows some trajectories in the phase plane ([CIIP22],[CI434-LVA]): there is a clear boundary between the two stable equilibrium points (the green dots) that goes through the unstable equilibrium point (the red dot). This boundary divides the two dimensional phase plane in two separate basins of attraction.
</P><H3><SPAN class="mw-headline" id="References_2"> References </SPAN></H3><P><TITLE>Bibliography</TITLE><TABLE style="border-collapse:collapse;line-height:1.1em;"><TBODY><TR style="vertical-align:top;"><TD>[1]</TD><TD style="padding-left:4pt;">A. E G H Wagner and R W Simons, “Antisense RNA Control in Bacteria, Phages, and Plasmids,” Nov. 2003; http://arjournals.annualreviews.org/doi/abs/10.1146/annurev.mi.48.100194.003433.</TD></TR><TR><TD colspan="2"> </TD></TR><TR style="vertical-align:top;"><TD>[2]</TD><TD style="padding-left:4pt;">“ETHZ/Parameters - IGEM07”; https://2007.igem.org/ETHZ/Parameters.</TD></TR><TR><TD colspan="2"> </TD></TR><TR style="vertical-align:top;"><TD>[3]</TD><TD style="padding-left:4pt;">J.A. Bernstein et al., “Global analysis of mRNA decay and abundance in Escherichia coli at single-gene resolution using two-color fluorescent DNA microarrays,” <SPAN style="font-style:italic;">Proceedings of the National Academy of Sciences of the United States of America</SPAN>,  vol. 99, Jul. 2002, pp. 9697–9702. </TD></TR><TR><TD colspan="2"> </TD></TR><TR style="vertical-align:top;"><TD>[4]</TD><TD style="padding-left:4pt;">M. Bon, S.J. McGowan, and P.R. Cook, “Many expressed genes in bacteria and yeast are transcribed only once per cell cycle,” <SPAN style="font-style:italic;">FASEB J.</SPAN>,  vol. 20, Aug. 2006, pp. 1721-1723. </TD></TR><TR><TD colspan="2"> </TD></TR><TR style="vertical-align:top;"><TD>[5]</TD><TD style="padding-left:4pt;">J.B. Andersen et al., “New Unstable Variants of Green Fluorescent Protein for Studies of Transient Gene Expression in Bacteria,” <SPAN style="font-style:italic;">Applied and Environmental Microbiology</SPAN>,  vol. 64, Jun. 1998, pp. 2240–2246. </TD></TR><TR><TD colspan="2"> </TD></TR><TR style="vertical-align:top;"><TD>[6]</TD><TD style="padding-left:4pt;">J. De Anda, A. Poteete, and R. Sauer, “P22 c2 repressor. Domain structure and function,” <SPAN style="font-style:italic;">J. Biol. Chem.</SPAN>,  vol. 258, Sep. 1983, pp. 10536-10542. </TD></TR><TR><TD colspan="2"> </TD></TR><TR style="vertical-align:top;"><TD>[7]</TD><TD style="padding-left:4pt;">“Part:BBa B0031 - partsregistry.org”; http://partsregistry.org/wiki/index.php?title=Part:BBa_B0031.</TD></TR><TR><TD colspan="2"> </TD></TR><TR style="vertical-align:top;"><TD>[8]</TD><TD style="padding-left:4pt;">“Part:BBa B0033 - partsregistry.org”; http://partsregistry.org/Part:BBa_B0033.</TD></TR><TR><TD colspan="2"> </TD></TR><TR style="vertical-align:top;"><TD>[9]</TD><TD style="padding-left:4pt;">“Part:BBa R0040 - partsregistry.org”; http://partsregistry.org/Part:BBa_R0040.</TD></TR><TR><TD colspan="2"> </TD></TR><TR style="vertical-align:top;"><TD>[10]</TD><TD style="padding-left:4pt;">“Part:BBa R0052 - partsregistry.org”; http://partsregistry.org/Part:BBa_R0052.</TD></TR><TR><TD colspan="2"> </TD></TR><TR style="vertical-align:top;"><TD>[11]</TD><TD style="padding-left:4pt;">“Part:BBa R0053 - partsregistry.org”; http://partsregistry.org/Part:BBa_R0053.</TD></TR><TR><TD colspan="2"> </TD></TR><TR style="vertical-align:top;"><TD>[12]</TD><TD style="padding-left:4pt;">B.C. McCabe, D.R. Pawlowski, and G.B. Koudelka, “The Bacteriophage 434 Repressor Dimer Preferentially Undergoes Autoproteolysis by an Intramolecular Mechanism,” <SPAN style="font-style:italic;">J. Bacteriol.</SPAN>,  vol. 187, Aug. 2005, pp. 5624-5630. </TD></TR><TR><TD colspan="2"> </TD></TR></TBODY></TABLE></P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2008.igem.org/Team:KULeuven/Model/Memory">http://2008.igem.org/Team:KULeuven/Model/Memory</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:KULeuven/Model/Memory" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:KULeuven/Model/Memory" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:KULeuven/Model/Memory&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:KULeuven/Model/Memory&amp;oldid=97746" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2008.igem.org:Privacy_policy" title="2008.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2008.igem.org:General_disclaimer" title="2008.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>