"
Page
Discussion
View source
History
teams
Log in
 
Team:KULeuven/Model/Reset
From 2008.igem.org
  dock/undock dropdown  
Home
Reports
Calendar
Project abstract
Meet the team
Leave a message
Mail us
The Team
Research Labs and Groups Students Instructors Advisors Pictures
The Project
Summary Components
Input Output Filter InverTimer Reset Cell Death Memory
Safety End Evaluation Literature Brainstorm
Ethics
Introduction Three A's
Defining the three A's Developments in modern physics Developments in biology
Concerns & Issues
Defining synthetic biology A framework Concerns Issues The need for answers
Biological Robotics
KULeuven iGEM 2008 project Biological robotics
Afterword References
Submitted Parts
Sandbox
Modeling
Overview Kinetic Constants Components
Output Filter InverTimer Reset Cell Death Memory
Full Model Sensitivity Analysis Multi-cell Model Diffusion
Data Analysis
Overview New Parts
GFP (LVA-tag) Other
Components
Input Output Other
Software
Multi-cell Toolbox Simbiology2LaTeX Toolbox
Notebook
Protocols Reports
Daily Weekly
Lab Data
Freezer Primers Ligation
Tools Press Acknowledgments
Contents
1 Pulse Generator
1.1 Position in the system
1.2 Describing the system
1.2.1 ODE's
1.2.2 Parameters
1.3 Models
1.3.1 CellDesigner (SBML file)
1.3.2 Matlab
1.4 Problem
2 Constant Lactonase Production
2.1 Position in the system
2.2 Describing the system
2.2.1 ODE's
2.2.2 Parameters
2.3 Models
2.3.1 CellDesigner (SBML file)
2.3.2 Matlab (SBML file)
2.4 Simulation
2.5 References
Pulse Generator
Position in the system
The Pulse Generator-subsystem is directly linked to the Filter.
When the filter indicates that the input is zero (there is no desease), the system will (ideally) produce no lactonase. As soon as the output of the filter is one, the subsystem will produce a pulse of lactonase which will be high enough to 'remove' all HSL present in the system and in that way reset the timer.
Describing the system
ODE's
NOT AVAILABLE
Parameters
Parameter values (Pulse Generator)
Name
Value
Comments
Reference
Degradation rates
dRNA_cI
0.00462 s-1
dcI
7.0E-4 s-1
dRNA_Lac
0.00231 s-1
dLac
2.888E-4 s-1
dRNA_Ribokey:cI
0.00231 s-1
Dissociation constants
KRibokey:cI
0.00212
kass/kdiss for the Ribokey cI complex
KcI
0.00337
binding cI on cI-Promotor
Transcription rates
kRNA_cI
0.025 s-1
maximal transcription rate RNA cI (no cI repressor present)
kRNA_Lac
0.025 s-1
Translation rates
kcI
0.167 s-1
kLac
0.167 s-1
RBS is B0032 (efficiency 0.3)
Hill cooperativity
ncI
2.0
All references for parameters can be found below, in the new section.
Models
CellDesigner ( SBML file )
Matlab
Problem
The idea of a pulsgenerator as reset mechanism doesn't meet the black-box requirements for the following reasons:
it takes too long before the proposed system generates a pulse-like event
the pulse itself is too long
a constant lactonase production sequence generates enough lactonase to reset the timer
Constant Lactonase Production
Position in the system
The Constant Lactonase Production-system is directly linked to the Filter .
When the filter indicates that the input is zero (there is no desease), the system will (ideally) produce no lactonase. As soon as the output of the filter is one, the system starts producing lactonase and remains doing this untill the light goes off again. In this way all the HSL-molecules that are present will be 'removed' and the timer is reset.
Describing the system
see also: Project:Reset
ODE's
Parameters
Parameter values Constant Lactonase Production
Name
Value
Comments
Reference
Degradation rates
daiiA
dLVA = 2.814E-4 s-1
LVA-tag reduces lifetime to 40 minutes
[3]
dclosed mRNA aiiA
0.0046209812 s-1
estimate: because this RNA isn't translated, it degrades faster
[2]
dopen mRNA aiiA
0.0023104906 s-1
[2]
dmRNA aiiA complex
0.0023104906 s-1
[2]
T7 Transcription
KT7
421
dissociation constant, recalculated to remove units
[4]
kmax
0.044 s-1
maximal T7 transcription rate
[4]
Key-Lock constants
Keq 1
0,015 [M]
between closed and open Lactonase mRNA, modeled for competition, experimental
[1]
Keq 2
0.0212 [M]
between closed Lactonase mRNA and key unlocked mRNA complex, modeled for competition, experimental
[1]
kdis1
0.00416 s-1
estimate: derived from experimental values
[1]
kcomplex1
0.00237 s-1
estimate: derived from experimental values
[1]
kclosed
500 s-1
estimate: derived from experimental values
[1]
kopen
7.5 s-1
estimate: derived from experimental values
[1]
ktranslation
0.167 s-1
lock defined translation rate for Lactonase
[1]
Models
CellDesigner ( SBML file )
Matlab ( SBML file )
Simulation
During the first 50000 seconds, the input signal TetR (no aTc) is held constant at 5E-5 s-1. This results in a background signal of lactonase which will convert a small part of the HSL into hydroxy acid. From 50000 till 80000 seconds, the input signal is at its maximum value: 0.0125 s-1 (aTc added). This results in an increase of lactonase which converts almost every HSL molecule into the hydroxy acid. The timer is set to 0. After a delay of approximately 20000 seconds, the amount of HSL starts to increase again: the clock is ticking. A shorter pulse of TetR (1000 seconds) only partially resets the timer: not all the HSL is converted into hydroxy acid. The graph has amounts (number of molecules in the cell) plotted vs time, measured in seconds.
References
Bibliography
[1]
“Berkeley2006-RiboregulatorsMain - IGEM”; http://parts2.mit.edu/wiki/index.php/Berkeley2006-RiboregulatorsMain.
 
[2]
J.A. Bernstein et al., “Global analysis of mRNA decay and abundance in Escherichia coli at single-gene resolution using two-color fluorescent DNA microarrays,” Proceedings of the National Academy of Sciences of the United States of America,  vol. 99, Jul. 2002, pp. 9697–9702.
 
[3]
J.B. Andersen et al., “New Unstable Variants of Green Fluorescent Protein for Studies of Transient Gene Expression in Bacteria,” Applied and Environmental Microbiology,  vol. 64, Jun. 1998, pp. 2240–2246.
 
[4]
G.M. Skinner et al., “Promoter Binding, Initiation, and Elongation By Bacteriophage T7 RNA Polymerase: A SINGLE-MOLECULE VIEW OF THE TRANSCRIPTION CYCLE,” J. Biol. Chem.,  vol. 279, Jan. 2004, pp. 3239-3244.
 
Retrieved from " http://2008.igem.org/Team:KULeuven/Model/Reset "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
