"
Page
Discussion
View source
History
teams
Log in
 
Team:KULeuven/Model/Memory
From 2008.igem.org
  dock/undock dropdown  
Home
Reports
Calendar
Project abstract
Meet the team
Leave a message
Mail us
The Team
Research Labs and Groups Students Instructors Advisors Pictures
The Project
Summary Components
Input Output Filter InverTimer Reset Cell Death Memory
Safety End Evaluation Literature Brainstorm
Ethics
Introduction Three A's
Defining the three A's Developments in modern physics Developments in biology
Concerns & Issues
Defining synthetic biology A framework Concerns Issues The need for answers
Biological Robotics
KULeuven iGEM 2008 project Biological robotics
Afterword References
Submitted Parts
Sandbox
Modeling
Overview Kinetic Constants Components
Output Filter InverTimer Reset Cell Death Memory
Full Model Sensitivity Analysis Multi-cell Model Diffusion
Data Analysis
Overview New Parts
GFP (LVA-tag) Other
Components
Input Output Other
Software
Multi-cell Toolbox Simbiology2LaTeX Toolbox
Notebook
Protocols Reports
Daily Weekly
Lab Data
Freezer Primers Ligation
Tools Press Acknowledgments
Contents
1 Memory
1.1 Position in the system
1.2 Describing the system
1.2.1 ODE's
1.2.2 Parameters
1.3 Models
1.3.1 CellDesigner (SBML file)
1.3.2 Matlab
1.4 Problem
1.5 Alternative
1.6 References
2 Memory - episode 2
2.1 Position in the system
2.2 Describing the system
2.2.1 ODE's
2.2.2 Parameters
2.3 Models
2.3.1 CellDesigner(SBML file)
2.3.2 Matlab (SBML file)
2.4 Simulations
2.5 Mathematical Analysis (Maple file)
2.6 References
Memory
Position in the system
This system must activate the cell death system after one light pulse. As long as there is no light, there is no P2ogr, no CIIP22 and a lot of antimRNA_LuxI. The antimRNA_LuxI blocks the cell death system. When light is turned on OmpF increases. This causes P2ogr and CIIP22 to increase and antimRNA_LuxI to decrease. This activates the system. When light is turned off, the P2ogr concentration is large enough to maintain itself. This way antimRNA doesn't increase. The system stays activated.
Describing the system
ODE's
Parameters
Parameter values (Memory)
Name
Value
Comments
Reference
Degradation Rates
dP2ogr
0.002265 s-1
[1]
dRNA_P2ogr
0.002265 s-1
[2]
dP22CII
0.002311 s-1
This value is too low. The correct value is used in the final model.
[3]
dRNA_P22CII
0,0022651 s-1
[2]
dantimRNA_luxI
0.0045303 s-1
estimate: because this RNA isn't translated, it degrades faster
[2]
Transcription Rates
kP2ogr
0.0125 s-1
estimate
[4]
kP22CII
0.0125 s-1
estimate
[4]
kAntimRNA_LuxI
0.0094 s-1
estimate
[5]
Dissociation Constants
KP2ogr
4.2156
Used in two reactions for activator control at the transcription of P2ogr mRNA and CIIP22 mRNA
[6]
KR0053_P22CII
0.1099
[3]
Hill Cooperativity
n
2
Used for all reactions throughout the memory submodel using Hill kinetics
[1]
Models
CellDesigner ( SBML file )
Matlab
Problem
The OmpF promoter is not ideal. When there is no light the transcription rate is still 0.00005 s-1. This means that P2ogr will slowly build up, activating the system. In this case the memory is in 0-state when the stationary state isn't reached yet. The 1-state is the stationary state. So the system automatically ends up in state 1 after some time (300s). This can be seen in the figure below.
Figure: CIIP22(purple), P2ogr(green), AntimRNA(pink).
The system can only stay in 0-state for 300 sec. This makes it completely useless.
Alternative
In the previous system the 0-state isn't actively maintained. It's just 'not stationary state'. So we need to search mathematical system that has 2 stationary states. A possible solution is given below.
Figure: Part representation of alternative system
When this system starts Rep build up because Rep represses the Act promoter better then Act represses the Rep promoter. The Rep concentration stays high and the Act stays low. This is the 0-state. When there is light, the OmpF promoter is activated and the Act concentration is increased. This represses Rep promoter. The Rep concentration decreases and the Act promoter is activated. The Act concentration keeps increasing. When the light pulse ends the Act concentration is high enough to repress the Rep promoter, Act concentration stays high and Rep concentration stays low. This is the 1-state.
CellDesigner gives the following simulation when OmpF transcription rate changes from 0.0001 to 0.01 at t=6000 sec for 2000 sec.
Figure: Celldesigner simulation of the alterative system. Act(grey), Rep(yellow)
The OmpF peak causes the Act concentration to rise and the Rep to decrease. The high Act concentration keeps the Rep concentration low. This causes the Act concentration to stay high.
References
Bibliography
[1]
“ETHZ/Parameters - IGEM07”; https://2007.igem.org/ETHZ/Parameters.
 
[2]
J.A. Bernstein et al., “Global analysis of mRNA decay and abundance in Escherichia coli at single-gene resolution using two-color fluorescent DNA microarrays,” Proceedings of the National Academy of Sciences of the United States of America,  vol. 99, Jul. 2002, pp. 9697–9702.
 
[3]
J. De Anda, A. Poteete, and R. Sauer, “P22 c2 repressor. Domain structure and function,” J. Biol. Chem.,  vol. 258, Sep. 1983, pp. 10536-10542.
 
[4]
“Part:BBa I746364 - partsregistry.org”; http://partsregistry.org/Part:BBa_I746364.
 
[5]
“Part:BBa R0053 - partsregistry.org”; http://partsregistry.org/Part:BBa_R0053.
 
[6]
D. Kornitzer, S. Altuvia, and A.B. Oppenheim, “The activity of the CIII regulator of lambdoid bacteriophages resides within a 24-amino acid protein domain.,” Proceedings of the National Academy of Sciences of the United States of America,  vol. 88, Jun. 1991, pp. 5217–5221.
 
Memory - episode 2
Position in the system
The Memory must keep the Cell Death inactivated untill a decent input signal has been received. When there is no input signal, c2 P22 will take control and repress cI 434 and Cell Death . This is the OFF state in which the memory can remain indefinitely unless an input signal emerges. This causes production of cI 434 (without LVA tag) whih will repress c2 P22 production, allowing cI 434 production to start from the c2 P22 repressible promoter. cI 434 will also start repressing the antisense LuxI production, enabling the InverTimer to do its work. After an input signal of about 1000 seconds, enough of the c2 P22 has disappeared in order to make the cI 434 production (LVA) self-sufficient. At this point, the memory has reached the ON state in which it will remain. For a more elaborate description of the Memory's biomolecular workings, please see the Project/Memory page.
Describing the system
This is the same figure as the alternative, but this time filled in for the new system. It conventiently shows that the system is based upon the alternative suggested above while laying out the workings and concretisations in the form of the different BioBricks implemented. Rep is now the c2 P22 repressor while Act is represented by cI 434.
The output of this system is antisense RNA production that represses the LuxI translation. c2 P22 has also got a function in repressing Cell Death
ODE's
Parameters
Parameter values (New Memory)
Name
Value
Comments
Reference
Degradation Rates
dcIIP22
dLVA = 2.814E-4 s-1
LVA-tag reduces lifetime to 40 minutes
[5]
dmRNA_cIIP22
0.0023104906 s-1
[3]
dcI434_LVA
dLVA = 2.814E-4 s-1
LVA-tag reduces lifetime to 40 minutes
[5]
dmRNA_cI434_LVA
0.0023104906 s-1
[3]
dcI434
9.627044174E-5 s-1
no LVA tag, so longer lifetime (t1/2 = 2h)
dmRNA_cI434
0.0023104906 s-1
[3]
dasRNA luxI
0.00462 s-1
estimate: because this RNA isn't translated, it degrades faster
[3]
dLuxI asRNA complex
0.00462 s-1
estimate: because this RNA isn't translated, it degrades faster
[3]
c2 P22 transcription (cI 434 regulated)
ktranscr
0.0125 s-1
estimate of maximal transcription rate, strong promoter
[10]
Km (cI 434)
0.8708
Recalculated to remove [M] dimension from Km=2E-9 [M]
[12]
cI 434 transcription (c2 P22 regulated)
ktranscr
0.0040 s-1
estimate of maximal transcription rate, weak-medium promoter
[11]
Km (c2 P22)
0.1099
Recalculated to remove [M]2 dimension from Kapp=6.8E-20[M]2
[6]
TetR regulated promoter
TetR_var_transcr_rate
p(TetR) dependent
(RiboKey) between 5E-5 and 0.0125 s-1 ~ [aTc]
[4] [9]
Translation rates
ktransl_cI434
0.0055555 s-1
translation rate for B0033 RBS (0.01 relative efficiency)
[8]
ktransl_cI434_lva
0.0388889 s-1
translation rate for B0031 RBS (0.07 relative efficiency)
[7]
ktransl_cIIP22
0.0055555 s-1
translation rate for B0033 RBS (0.01 relative efficiency)
[8]
Hill Cooperativity
n
2
Used for all reactions throughout the memory submodel using Hill kinetics
[2]
Antisense LuxI
k_complex3
0.00237 s-1
rate constant for formation of asRNA - LuxI mRNA duplex
[1]
KmRNA_LuxI:antisense_mRNA
4.22E-14 [M]
Dissociation constant for complex of LuxI mRNA with antisense mRNA
[1]
Models
CellDesigner( SBML file )
Matlab ( SBML file )
Simulations
Multiple simulations have been performed to see if the memory works. If we give a pulse of TetR from 5E-5 to 0.0125 for 10000 sec, we see that the memory switches from its 0-state to its 1-state (left figure):  cIIP22 amount drops and the cI434 increases. The high cI434 amount will repress the RNA production. If the pulse lasts for only 1000 sec, the memory remains in its 0-state (middle figure). Simulations show the minimum time span needed to switch the memory is approximately 1300 sec. The right figure shows us that a sequence of two pulses of only 1000 sec can also make the memory switch.
The graphs have amounts (number of molecules in the cell) plotted vs time, measured in seconds.
Mathematical Analysis ( Maple file )
The existence of the two stable states of the memory (cI434_LVA high and cIIP22 low / cI434_LVA low and cIIP22 high) can be mathematically proven.
First we define the equilibrium points of the following differential equation system:
The equilibrium points are defined as the points for which all the derivatives are zero. Solving this non-linear system for [OmpF] equal to 0.00005 results in finding the roots of the equation
This equation has three real zeros ([CI434_LVA = 0.002435815407], [CI434_LVA = 11.52604711], [CI434_LVA = 233.6410112]) and two conjugated imaginary zeros ([CI434_LVA = -8.242012862+11.11131953*I], [CI434_LVA = -8.242012862-11.11131953*I]) as can be seen in the following figure:
The real roots of the system for all the variables are:
Real Roots
Zero 1
Zero 2
Zero 3
[CI434]
1.248827120
1.248827120
1.248827120
[CI434-LVA]
0.002435815407
11.52604711
233.6410112
[mRNACI434-LVA]
0.00001804584283
0.08539121399
1.730939445
[CIIP22]
34.03962262
0.4824866257
0.001433763049
[mRNACIIP22]
1.765288021
0.02502165991
0.00007435466497
[asRNACI434]
0.02164042563
0.02164042563
0.02164042563
The stability of the three real zeros is defined by the eigenvalues of the Jacobian of the differential equation system. This Jacobian equals
The eigenvalues for Zero 1 are all negative (-0.2334056227e-2, -0.2286362516e-2, -0.3122380843e-3, -0.2645443732e-3, -0.9627000000e-4, -0.2310490600e-2). This is also the case for Zero 3 (-0.2319019326e-2, -0.2301889325e-2, -0.2795812737e-3, -0.2967112750e-3, -0.9627000000e-4, -0.2310490600e-2). These two zero's represent the two stable states of the memory system. Zero 2 has one positive eigenvalue (0.0002015903724) and is therefore unstable.
The following figure shows some trajectories in the phase plane ([CIIP22],[CI434-LVA]): there is a clear boundary between the two stable equilibrium points (the green dots) that goes through the unstable equilibrium point (the red dot). This boundary divides the two dimensional phase plane in two separate basins of attraction.
References
Bibliography
[1]
A. E G H Wagner and R W Simons, “Antisense RNA Control in Bacteria, Phages, and Plasmids,” Nov. 2003; http://arjournals.annualreviews.org/doi/abs/10.1146/annurev.mi.48.100194.003433.
 
[2]
“ETHZ/Parameters - IGEM07”; https://2007.igem.org/ETHZ/Parameters.
 
[3]
J.A. Bernstein et al., “Global analysis of mRNA decay and abundance in Escherichia coli at single-gene resolution using two-color fluorescent DNA microarrays,” Proceedings of the National Academy of Sciences of the United States of America,  vol. 99, Jul. 2002, pp. 9697–9702.
 
[4]
M. Bon, S.J. McGowan, and P.R. Cook, “Many expressed genes in bacteria and yeast are transcribed only once per cell cycle,” FASEB J.,  vol. 20, Aug. 2006, pp. 1721-1723.
 
[5]
J.B. Andersen et al., “New Unstable Variants of Green Fluorescent Protein for Studies of Transient Gene Expression in Bacteria,” Applied and Environmental Microbiology,  vol. 64, Jun. 1998, pp. 2240–2246.
 
[6]
J. De Anda, A. Poteete, and R. Sauer, “P22 c2 repressor. Domain structure and function,” J. Biol. Chem.,  vol. 258, Sep. 1983, pp. 10536-10542.
 
[7]
“Part:BBa B0031 - partsregistry.org”; http://partsregistry.org/wiki/index.php?title=Part:BBa_B0031.
 
[8]
“Part:BBa B0033 - partsregistry.org”; http://partsregistry.org/Part:BBa_B0033.
 
[9]
“Part:BBa R0040 - partsregistry.org”; http://partsregistry.org/Part:BBa_R0040.
 
[10]
“Part:BBa R0052 - partsregistry.org”; http://partsregistry.org/Part:BBa_R0052.
 
[11]
“Part:BBa R0053 - partsregistry.org”; http://partsregistry.org/Part:BBa_R0053.
 
[12]
B.C. McCabe, D.R. Pawlowski, and G.B. Koudelka, “The Bacteriophage 434 Repressor Dimer Preferentially Undergoes Autoproteolysis by an Intramolecular Mechanism,” J. Bacteriol.,  vol. 187, Aug. 2005, pp. 5624-5630.
 
Retrieved from " http://2008.igem.org/Team:KULeuven/Model/Memory "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
