"
Page
Discussion
View source
History
teams
Log in
 
Team:KULeuven/Model/Cell Death
From 2008.igem.org
(Redirected from Team:KULeuven/Model/CellDeath )
  dock/undock dropdown  
Home
Reports
Calendar
Project abstract
Meet the team
Leave a message
Mail us
The Team
Research Labs and Groups Students Instructors Advisors Pictures
The Project
Summary Components
Input Output Filter InverTimer Reset Cell Death Memory
Safety End Evaluation Literature Brainstorm
Ethics
Introduction Three A's
Defining the three A's Developments in modern physics Developments in biology
Concerns & Issues
Defining synthetic biology A framework Concerns Issues The need for answers
Biological Robotics
KULeuven iGEM 2008 project Biological robotics
Afterword References
Submitted Parts
Sandbox
Modeling
Overview Kinetic Constants Components
Output Filter InverTimer Reset Cell Death Memory
Full Model Sensitivity Analysis Multi-cell Model Diffusion
Data Analysis
Overview New Parts
GFP (LVA-tag) Other
Components
Input Output Other
Software
Multi-cell Toolbox Simbiology2LaTeX Toolbox
Notebook
Protocols Reports
Daily Weekly
Lab Data
Freezer Primers Ligation
Tools Press Acknowledgments
Contents
1 Cell Death
1.1 Position in the system
1.2 Describing the system
1.2.1 ODE's
1.2.2 Parameters
1.3 Models
1.3.1 CellDesigner (SBML file)
1.3.2 Matlab (SBML file)
2 New Cell Death
2.1 Extensions to previous system
2.2 Describing the system
2.2.1 ODE's
2.2.2 Parameters
2.3 Models
2.3.1 CellDesigner (SBML file)
2.3.2 Matlab (SBML file)
2.4 Simulations
2.5 References
Cell Death
Position in the system
The Cell Death subsystem receives input from two other subsystems, namely:
Inverter
Pulse Generator
LuxR is the component repressing the regulation of CcdB, the toxic product causing cell death. There the LuxR production is constitutive, no protein controls the gene regulation of LuxR, but the amount of LuxR available to repress the transcripion of the CcdB gene is controlled by HSL (Homoserine lactone).
If the inverter subsystem produces HSL (occurs when no light is detectable), this will forms a complex with LuxR. This will diminish the amount of LuxR available to repress the CcdB transcription and initiate cell death. When waiting long enough the amount of HSL becomes critical.
If however the pulse generator becomes active (by the filter), it will produce a pulse of lactonase, which will then bind to the HSL, reacting to an hydroxy-acid. As opposed to HSL, this hydroxy-acid will no longer form a complex with LuxR. This increase in LuxR lowers the CcdB production. The challenge is to generate a pulse of lactonase high enough to neutralise all HSL present in the cell.
Describing the system
ODE's
Parameters
Attention: old system, old parameters!
Parameter values (Cell Death)
Name
Value
Comments
Reference
Degradation Rates
dLuxR
0.0010 s-1
dLuxR_HSL
0.0010 s-1
complex of HSL and LuxR degrades, giving back HSL
dRNA_LuxR
0.00227 s-1
dCcdB
7.7E-5 s-1
dRNA_CcdB
0.00231 s-1
dHSL
1.02E-6 s-1
Association/Dissociation/Reaction Rates
kass (HSL+LuxR)
1E6 s-1
association rate of HSL with LuxR. Chosen to be relatively (to the other rate constants) high and such that Kdiss (HSL + LuxR) equals 10-6
kdiss (HSL+LuxR)
1 s-1
dissociation rate of the HSL-LuxR complex
kass (HSL+lactonase)
1E6 s-1
association rate of HSL with lactonase
kdiss (HSL+lactonase)
446.5 s-1
dissociation rate of the HSL-lactonase complex
kcat (HSL:hydroxy-acid)
29 s-1
catalytic transformation of HSL to an hydroxy-acid, lactonase is the enzyme
Dissociation Constants
KHSL_LuxR
1E-6 [M]/L
kdiss / kass (HSL+LuxR)
KHSL_LuxR
4.05E-6 [M]/L
binding HSL_LuxR on LuxPromotor
Hill Cooperativity
nHSL_LuxR
2.08
Transcription Rates
kmRNA_LuxR (constitutive promotor)
0.025 s-1
see Constitutive Promotors & E. coli transcription Rates
kmRNA_CcdB
0.025 s-1
maximal transcription rate for CcdB RNA (no LuxR repressor present)
This paper says that the synthesis of only a few molecules of CcdB protein is probably lethal.
All references for parameters can be found below, in the new section.
Models
CellDesigner ( SBML file )
Matlab ( SBML file )
Remark: not yet up to date to latest (final) version
New Cell Death
Extensions to previous system
During the summer we switched from the above system to a new one which you can see just below, take a look at this figure as it will help you understand the regulation that is present. This new system has a few novelties.
First of all, transcription begins at a new hybrid promoter we made: BBa_K145150 . This promoter is repressed by c2 P22, which is produced by the memory in the OFF state, making premature activation and cell death impossible. Besides this repression, the promoter is activated by the HSL-LuxR complex originating from a previously activated timer. The promoter behaves as shown schematically below.
Second, LuxR is now no longer constitutively produced but is placed behind the beforementioned hybrid promoter. This construction mimicks more closely the natural system where LuxR is upregulated when a threshold amount of HSL is present. Plus it also increases the time it takes to activate ccdB, lengthening the timer. The system will now auto-activate if enough HSL is present and the memory is in the ON state.
Third, the ccdB coding region is also downstream of the hybrid promoter and is thus also subject to the regulation explained above; c2 P22 repression and HSL-LuxR auto-activation. One difference is that the polymerase must first read through a bad terminator with about 60% efficiency before reaching this coding region. Another difference is in the ribosome binding sites preceding both coding regions. Where LuxR can be translated from a RBS with a relative efficiency of 1.00, the ccdB frame can only be read from a 0.01 efficiency RBS.
Describing the system
see also: Project:Cell Death
ODE's
Parameters
Parameter values (Cell Death)
Name
Value
Comments
Reference
Degradation Rates
dLuxR
9.627044174E-5 s-1
no LVA tag, so longer lifetime (t1/2 = 2h)
dcomplex
9.627044174E-5 s-1
complex of HSL and LuxR degrades, giving back HSL
dRNA_LuxR
0.00227 s-1
[3]
dccdB
7.7E-5 s-1
stable in the absence of ccdA
[4]
dRNA_ccdB
0.00231 s-1
[3]
dHSL
1.02E-6 s-1
very stable in the medium, average lifetime of 185h
[10]
Association/Dissociation/Reaction Rates
kass (HSL+LuxR)
0.002372 s-1
association rate of HSL with LuxR (estimate from KM but recalculated to remove molar dimension)
[2]
kdiss (HSL-LuxR)
1.0 s-1
dissociation rate of the HSL-LuxR complex (estimate from KM)
[2]
kass (HSL+lactonase)
0.002372 s-1
association rate of HSL with lactonase (estimate from KM but recalculated to remove molar dimension)
[11]
kdiss (HSL-lactonase)
4470.0 s-1
dissociation rate of the HSL-lactonase complex (estimate from Kd)
[11]
kcat (HSL>>hydroxy-acid)
29 s-1
lactonase catalyzed transformation of HSL to a hydroxy-acid
[11]
Dissociation Constants
KHSL_LuxR
1E-6 [M]
HSL binding to LuxR
[2]
KHSL_LuxR-promoter
4.05E-6 [M]
binding of HSL_LuxR complex to the Lux Promotor
[2]
KHSL_lactonase
4.47E-3 [M]
KM of lactonase with 3OC6HSL
[11]
Hybrid Promoter
KC2 P22
2.6E-10 [M]
Dissociation constant for P22 c2 promoter binding
[6]
KHSL_LuxR
4.05E-6 [M]
Dissociation constant for HSL-LuxR complex promoter binding
[1] [12]
Hill (P22 c2)
2
Hill coefficient for P22 c2 binding
[6]
Hill (HSL-LuxR)
2
Hill coefficient for HSL-LuxR binding
[1] [12]
kleaky
5E-4 s-1
transcription rate from the hybrid promoter in the unactivated ON state
[1]
kmax
0.0030  s-1
maximal transcription rate for the fully activated ON hybrid promoter (might be higher)
[5]
Terminator eff
60.8%
percentage of transcription termination at the B0014 double terminator in front of ccdB
[7]
Translation Rates
kLuxR translation
0.556 s-1
Translation rate for LuxR, B0034 RBS (relative efficiency 1.0)
[9]
kccdB translation
0.00556 s-1
Translation rate for ccdB, B0033 RBS (relative efficiency 0.01)
[9]
Models
CellDesigner ( SBML file )
Matlab ( SBML file )
Simulations
To simulate the cell death, the input signal TetR is all the time low (5E-5 s-1) except from 50000 till 80000 seconds,from 180000 till 190000 seconds and from 250000 till 270000 seconds (0.0125 s-1). The left figure shows us that a high input signal will lower the amount of HSL (the HSL will be converted into hydroxy acid by lactonase). This will increase the amount of free LuxR because there is not enough HSL to form the HSL-LuxR complex. During the pulse, the right figure shows us an increase of the amount of CcdB. Whenthe memory switches from state zero to state one, the amount of CIIP22 decreases which makes it possible to produce a small amount of CcdB (background signal).
When the input signal is low again, the amount of HSL increases. This extra HSL will form with LuxR the complex which lowers the amount of free LuxR. The higher amount of the complex promotes the production of CcdB which increases to a (hopefully) deathly level.
A new pulse lowers the amount of HSL and the complex and increases the amount of LuxR for the same reasons as before. The second pulse is smal in length and decreases the amount of CcdB only partially. The third pulse is long enough for a total reset of the entire system.
All graphs have amounts (number of molecules in the cell) plotted vs time, measured in seconds.
References
Bibliography
[1]
L.C.M. Antunes et al., “A Mutational Analysis Defines Vibrio fischeri LuxR Binding Sites,” J. Bacteriol., Dec. 2007, pp. JB.01443-07.
 
[2]
N. Qin et al., “Analysis of LuxR Regulon Gene Expression during Quorum Sensing in Vibrio fischeri,” J. Bacteriol.,  vol. 189, Jun. 2007, pp. 4127-4134.
 
[3]
J.A. Bernstein et al., “Global analysis of mRNA decay and abundance in Escherichia coli at single-gene resolution using two-color fluorescent DNA microarrays,” Proceedings of the National Academy of Sciences of the United States of America,  vol. 99, Jul. 2002, pp. 9697–9702.
 
[4]
L. Van Melderen, P. Bernard, and M. Couturier, “Lon-dependent proteolysis of CcdA is the key control for activation of CcdB in plasmid-free segregant bacteria,” Molecular Microbiology,  vol. 11, Mar. 1994, pp. 1151-7.
 
[5]
M. Bon, S.J. McGowan, and P.R. Cook, “Many expressed genes in bacteria and yeast are transcribed only once per cell cycle,” FASEB J.,  vol. 20, Aug. 2006, pp. 1721-1723.
 
[6]
J. De Anda, A. Poteete, and R. Sauer, “P22 c2 repressor. Domain structure and function,” J. Biol. Chem.,  vol. 258, Sep. 1983, pp. 10536-10542.
 
[7]
“Part:BBa B0014 - partsregistry.org”; http://partsregistry.org/Part:BBa_B0014.
 
[8]
“Part:BBa B0033 - partsregistry.org”; http://partsregistry.org/Part:BBa_B0033.
 
[9]
“Part:BBa B0034 - partsregistry.org”; http://partsregistry.org/Part:BBa_B0034.
 
[10]
Y. Wang and J.R. Leadbetter, “Rapid Acyl-Homoserine Lactone Quorum Signal Biodegradation in Diverse Soils,” Appl. Environ. Microbiol.,  vol. 71, Mar. 2005, pp. 1291-1299.
 
[11]
L. Wang et al., “Specificity and enzyme kinetics of the quorum-quenching AHL-lactonase,” J. Biol. Chem., Jan. 2004, p. M311194200.
 
[12]
“Tokyo/AHL assay - IGEM07”; https://2007.igem.org/Tokyo/AHL_assay.
 
Retrieved from " http://2008.igem.org/Team:KULeuven/Model/Cell_Death "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
