"
Page
Discussion
View source
History
teams
Log in
 
Team:KULeuven/Model/Kinetic Constants
From 2008.igem.org
  dock/undock dropdown  
Home
Reports
Calendar
Project abstract
Meet the team
Leave a message
Mail us
The Team
Research Labs and Groups Students Instructors Advisors Pictures
The Project
Summary Components
Input Output Filter InverTimer Reset Cell Death Memory
Safety End Evaluation Literature Brainstorm
Ethics
Introduction Three A's
Defining the three A's Developments in modern physics Developments in biology
Concerns & Issues
Defining synthetic biology A framework Concerns Issues The need for answers
Biological Robotics
KULeuven iGEM 2008 project Biological robotics
Afterword References
Submitted Parts
Sandbox
Modeling
Overview Kinetic Constants Components
Output Filter InverTimer Reset Cell Death Memory
Full Model Sensitivity Analysis Multi-cell Model Diffusion
Data Analysis
Overview New Parts
GFP (LVA-tag) Other
Components
Input Output Other
Software
Multi-cell Toolbox Simbiology2LaTeX Toolbox
Notebook
Protocols Reports
Daily Weekly
Lab Data
Freezer Primers Ligation
Tools Press Acknowledgments
This page contains various references to literature with several parameters that helped us get started on the model. More references, along with parameters can be found under the corresponding components sections.
Contents
1 ETHZ list of parameters
2 mRNA decay
3 T7 RNAP
4 LacI - LuxI
5 LuxI, LuxR, mRNALuxI, mRNALuxR decay
6 HSL stuff
7 Diffusion
8 OmpR, OmpF
9 Psid met P2ogr promotor
10 Constitutive promoters
11 E. coli transcription rates
ETHZ list of parameters
Bibliography
[1]
“ETHZ/Parameters - IGEM07”; https://2007.igem.org/ETHZ/Parameters.
 
mRNA decay
Bibliography
[1]
J.A. Bernstein et al., “Global analysis of mRNA decay and abundance in Escherichia coli at single-gene resolution using two-color fluorescent DNA microarrays,” Proceedings of the National Academy of Sciences of the United States of America,  vol. 99, Jul. 2002, pp. 9697–9702.
 
T7 RNAP
Bibliography
[1]
“IGEM:Tsinghua/2007/Projects/RAP - OpenWetWare”; http://www.openwetware.org/wiki/IGEM:Tsinghua/2007/Projects/RAP.
 
[2]
R. Ikeda, A. Lin, and J. Clarke, “Initiation of transcription by T7 RNA polymerase as its natural promoters,” J. Biol. Chem.,  vol. 267, Feb. 1992, pp. 2640-2649.
 
[3]
R. Bandwar et al., “Kinetic and Thermodynamic Basis of Promoter Strength: Multiple Steps of Transcription Initiation by T7 RNA Polymerase Are Modulated by the Promoter Sequence,” Biochemistry,  vol. 41, Mar. 2002, pp. 3586-3595.
 
[4]
G.M. Skinner et al., “Promoter Binding, Initiation, and Elongation By Bacteriophage T7 RNA Polymerase: A SINGLE-MOLECULE VIEW OF THE TRANSCRIPTION CYCLE,” J. Biol. Chem.,  vol. 279, Jan. 2004, pp. 3239-3244.
 
[5]
V.S. Anand and S.S. Patel, “Transient State Kinetics of Transcription Elongation by T7 RNA Polymerase,” J. Biol. Chem.,  vol. 281, Nov. 2006, pp. 35677-35685.
 
LacI - LuxI
Bibliography
[1]
M.B. Elowitz and S. Leibler, “A synthetic oscillatory network of transcriptional regulators,” Nature,  vol. 403, Jan. 2000, pp. 335-338.
 
[2]
“Modeling a synthetic multicellular clock: Repressilators coupled by quorum sensing,” Jul. 2004; http://www.pnas.org/content/101/30/10955.full.
 
LuxI, LuxR, mRNALuxI, mRNALuxR decay
Bibliography
[1]
“Generation of cell-to-cell signals in quorum sensing: acyl homoserine lactone synthase activity of a purified Vibrio fischeri LuxI protein,” Sep. 1996; http://www.pnas.org/content/93/18/9505.
 
[2]
L. Chen et al., “Noise-induced cooperative behavior in a multicell system,” Bioinformatics,  vol. 21, Jun. 2005, pp. 2722-2729.
 
HSL stuff
Bibliography
[1]
“Acyl homoserine-lactone quorum-sensing signal generation,” Apr. 1999; http://www.pnas.org/content/96/8/4360.full.
 
[2]
P. Nilsson et al., “Kinetics of the AHL Regulatory System in a Model Biofilm System: How Many Bacteria Constitute a "Quorum"?,” Journal of Molecular Biology,  vol. 309, Jun. 2001, pp. 631-640.
 
[3]
Y. Wang and J.R. Leadbetter, “Rapid Acyl-Homoserine Lactone Quorum Signal Biodegradation in Diverse Soils,” Appl. Environ. Microbiol.,  vol. 71, Mar. 2005, pp. 1291-1299.
 
[4]
L. Wang et al., “Specificity and Enzyme Kinetics of the Quorum-quenching N-Acyl Homoserine Lactone Lactonase (AHL-lactonase),” J. Biol. Chem.,  vol. 279, Apr. 2004, pp. 13645-13651.
 
Diffusion
Bibliography
[1]
P.S. Stewart, “Diffusion in Biofilms,” J. Bacteriol.,  vol. 185, Mar. 2003, pp. 1485-1491.
 
OmpR, OmpF
Bibliography
[1]
K.J. Huang, J.L. Schieberl, and M.M. Igo, “A distant upstream site involved in the negative regulation of the Escherichia coli ompF gene.,” Journal of Bacteriology,  vol. 176, Mar. 1994, pp. 1309–1315.
 
[2]
K.V. Srividhya and S. Krishnaswamy, “A simulation model of Escherichia coli osmoregulatory switch using E-CELL system,” BMC Microbiology,  vol. 4, 2004, p. 44.
 
[3]
S.A. Forst, J. Delgado, and M. Inouye, “DNA-binding properties of the transcription activator (OmpR) for the upstream sequences of ompF in Escherichia coli are altered by envZ mutations and medium osmolarity.,” Journal of Bacteriology,  vol. 171, Jun. 1989, pp. 2949–2955.
 
[4]
N. Ramani, M. Hedeshian, and M. Freundlich, “micF antisense RNA has a major role in osmoregulation of OmpF in Escherichia coli.,” Journal of Bacteriology,  vol. 176, Aug. 1994, pp. 5005–5010.
 
[5]
T. Yoshida et al., “Transcription Regulation of ompF and ompC by a Single Transcription Factor, OmpR,” J. Biol. Chem.,  vol. 281, Jun. 2006, pp. 17114-17123.
 
[6]
S. Tokishita et al., “Transmembrane signal transduction and osmoregulation in Escherichia coli. Functional importance of the periplasmic domain of the membrane- located protein kinase, EnvZ,” J. Biol. Chem.,  vol. 266, Apr. 1991, pp. 6780-6785.
 
Psid met P2ogr promotor
Bibliography
[1]
B. Julien and R. Calendar, “Bacteriophage PSP3 and phiR73 activator proteins: analysis of promoter specificities.,” Journal of Bacteriology,  vol. 178, Oct. 1996, pp. 5668–5675.
 
[2]
“Cambridge/Amplifier project - IGEM07”; https://2007.igem.org/Cambridge/Amplifier_project.
 
Constitutive promoters
Estimated transcription rate for J23105: [1]
Scale other transcription rate with table in parts registry.
Estimate the rate of transcription from a constitutive promotor family member.
Estimates for the rate of transcription from the constitutive promotor family members. X is the GFP fluorescence in arbitrary units according to the Registry. Y is the number of mRNA's produced per second from that promotor
E. coli transcription rates
Bibliography
[1]
M. Bon, S.J. McGowan, and P.R. Cook, “Many expressed genes in bacteria and yeast are transcribed only once per cell cycle,” FASEB J.,  vol. 20, Aug. 2006, pp. 1721-1723.
 
Retrieved from " http://2008.igem.org/Team:KULeuven/Model/Kinetic_Constants "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
