<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_MIT_Protocols"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:MIT/Protocols">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:MIT/Protocols&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:MIT/Protocols&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:MIT/Protocols&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:MIT/Protocols" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:MIT/Protocols</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><DIV style="margin-top:-50px;Background-color:#FFF"> </DIV></DIV><TABLE class="nav"><TBODY><TR><TD><A href="https://2014.igem.org/Team:MIT">Home</A></TD><TD><A href="https://2014.igem.org/Team:MIT/Project"> Our Project</A><UL><LI><A href="https://2014.igem.org/Team:MIT/Project">Motivation</A></LI><LI><A href="https://2014.igem.org/Team:MIT/Protein_sensor">Native Receptor</A></LI><LI><A href="https://2014.igem.org/Team:MIT/BCR">B-Cell Receptor</A></LI><LI><A href="https://2014.igem.org/Team:MIT/miRNA">miRNA Detection</A></LI><LI><A href="https://2014.igem.org/Team:MIT/Treatment">Treatment Module</A></LI><LI><A href="https://2014.igem.org/Team:MIT/Delivery">Delivery</A></LI><LI><A href="https://2014.igem.org/Team:MIT/Modeling">Modeling</A></LI></UL></TD><TD><A href="https://2014.igem.org/Team:MIT/Parts"> Lab Work</A><UL><LI><A href="https://2014.igem.org/Team:MIT/Parts">Parts</A></LI><LI><A href="https://2014.igem.org/Team:MIT/Safety">Safety</A></LI><LI><A href="https://2014.igem.org/Team:MIT/Notebook">Notebook</A></LI><LI><A href="https://2014.igem.org/Team:MIT/Protocols">Protocols</A></LI></UL></TD><TD><A href="https://2014.igem.org/Team:MIT/Outreach"> Outreach</A><UL><LI><A href="https://2014.igem.org/Team:MIT/Outreach">Outreach</A></LI><LI><A href="https://2014.igem.org/Team:MIT/2014.igem.org/Team:MIT/Interviews">Interviews</A></LI><LI><A href="https://2014.igem.org/Wiki/2014.igem.org/Team:MIT/Survey">Survey</A></LI><LI><A href="https://2014.igem.org/Team:MIT/Manifold">DIY Vacuum Manifold</A></LI></UL></TD><TD><A href="https://2014.igem.org/Team:MIT/Team"> About Us</A><UL><LI><A href="https://2014.igem.org/Team:MIT/Team">Student Bios</A></LI><LI><A href="https://2014.igem.org/Team:MIT/Attributions">Acknowledgements</A></LI><LI><A href="https://2014.igem.org/Team:MIT/Actually Attributions">Attributions</A></LI><LI><A href="https://igem.org/Team.cgi?year=2014&amp;team_name=MIT">Official iGEM page</A></LI></UL></TD><TD><A href="https://2014.igem.org/Team:MIT/Medals">Medals</A></TD></TR></TBODY></TABLE><H3 align="center" style="font-size:42px; color:teal"><B> PROTOCOLS </B></H3><P style="font-size:12px" align="center"><I>Attributions: MIT iGEM 2014, MIT iGEM 2013</I></P><CENTER><TABLE width="85%" border="solid;"><TBODY><TR><TH width="25%">Cloning</TH><TH width="25%">Tissue Culture</TH><TH width="25%">Miscellaneous Lab</TH><TH width="25%">Non-Lab</TH></TR><TR><TD><A href="#gels">GELS</A><A href="#GOLDEN_GATE">GOLDEN GATE</A><A href="#LR_GATEWAY">LR GATEWAY</A><A href="#MAKNG_LIQUID_CULTURE">MAKNG LIQUID CULTURE</A><A href="#MIDI_PREP">MIDI PREP</A><A href="#MINI_Prep">MINI Prep</A><A href="#PCR">PCR</A><A href="#RESUSPENDING_PCR_PRIMERS">RESUSPENDING PCR PRIMERS</A><A href="#RESTRICTION_DIGEST">RESTRICTION DIGEST</A><A href="#TRANSFORMATION">TRANSFORMATION</A><A href="#BP_REACTION">BP REACTION</A><A href="#ANNEALING_AND_KINASING_OLIGOS">ANNEALING AND KINASING OLIGOS</A><A href="#MAKING_CELL_STOCKS">MAKING CELL STOCKS</A><A href="#MAKING_SOC">MAKING SOC</A><A href="#POURING_GG_DONOR_PLATES">POURING GG DONOR PLATES</A><A href="#PREPARING_LB_AGAR_PLATES">PREPARING LB AGAR PLATES</A></TD><TD><A href="#MAKINGCELLCULTUREMEDIA">MAKING CELL CULTURE MEDIA</A><A href="#SPLITTINGCELLSANDSEEDINGPLATES">SPLITTING CELLS AND SEEDING PLATES</A><A href="#TRANSFECTIONUSINGLIPOFECTAMINELTX">TRANSFECTION (USING LIPOFECTAMINE LTX)</A><A href="#TRANSFECTIONUSINGLIPOFECTAMINE2000">TRANSFECTION (USING LIPOFECTAMINE 2000)</A><A href="#FACSCELLPREP24WELLPLATE">FACS CELL PREP (24 WELL PLATE)</A><A href="#COTRANSFECTIONOFHEK293WITHLIPOFECTAMINE200024WELLPLATE">CO-TRANSFECTION OF HEK293 WITH LIPOFECTAMINE 2000 (24 WELL PLATE)</A><A href="#INDIRECTFLOWCYTOMETRY">INDIRECT FLOW CYTOMETRY</A><A href="#FIXATION">FIXATION</A><A href="#IMMUNOSTAININGFORFLUORESCENTMICROSCOPY">IMMUNOSTAINING FOR FLUORESCENT MICROSCOPY</A><A href="#GELATINTREATMENTFORGLASSPLATES">GELATIN TREATMENT FOR GLASS PLATES</A><A href="#BETAAMYLOIDSTREPTAVIDINPROTOCOLFORCYTOMETRY">BETA AMYLOID STREPTAVIDIN PROTOCOL FOR CYTOMETRY</A><A href="#BETAAMYLOIDSTREPTAVIDINSTAININGFORLIVEMICROSCOPY">BETA AMYLOID STREPTAVIDIN STAINING FOR LIVE MICROSCOPY</A><A href="#ATTACHINGBCELLSUSINGPOLY-L-LYSINE">ATTACHING B CELLS USING POLY-L-LYSINE</A></TD><TD><A href="#WESTERNBLOT">WESTERN BLOT</A><A href="#OLIGOMERIZINGBETAAMYLOID">OLIGOMERIZING BETA AMYLOID</A></TD><TD><A href="#CHECKINGPRIMERTBWITHNEB">CHECKING PRIMER TB WITH NEB</A><A href="#LOCATINGAQCUTSITEWITHINAGENE">LOCATING A Q CUT SITE WITHIN A GENE</A><A href="#MUTATINGDNASEQUENCESTOGETRIDOFCUTSITES">MUTATING DNA SEQUENCES TO GET RID OF CUT SITES</A><A href="#FUSIONPROTEINSANDGENEIOUS">FUSION PROTEINS AND GENEIOUS</A><A href="#cookies">MIT iGEM cookie preparation</A></TD></TR></TBODY></TABLE></CENTER><DIV style="padding:20px"><H2>FUSION PROTEINS AND GENEIOUS</H2><A href="#top">return to top</A>
Fusion proteins (Using GoldenGate):
Add Q sites using primers.
Make sure to add at least 1 extra nucleotide before the BsaI site (6 is standard). Make sure that the extra nucleotides at the end of the primers are NOT complementary.
Make sure to add/keep the Kozak sequence.
Make sure to get rid of the stop codon of the first protein (all proteins except the last protein).
Make sure the two proteins are in frame.
Check binding site melting temperature at the NEB website. Check hairpin structures in Geneious.
If the BsaI cleavage is scarless, then there should not be a frameshift. (Someone verify this.)

<H2>MUTATING DNA SEQUENCES TO GET RID OF CUT SITES</H2><A href="#top">return to top</A>
You are ordering DNA to use with a Golden Gate reaction (using BsaI) to make entry vectors.  But... your sequence has BsaI cut sites - oh no!  Little fear, you might be able to mutate the codons that make up the BsaI site to remove those sites without changing the amino acid sequence.
Look for BsaI cut sites in your sequence using Geneious.  Use Restriction Analysis to find the BsaI sites (under Advanced) and apply them to your sequence.
Select the CDS annotation for your sequence, and check the BsaI sites to see in which codons they occur.
Look for alternate codons that code for the same amino acid.  Make sure that the codon that you end up with after mutating has a higher frequency than the existing codon.  You can find a table of frequencies for humans here: http://www.genscript.com/cgi-bin/tools/codon_freq_table
If you can't find a codon with a higher frequency, then use a lower frequency codon but you might have to mutate it back (i.e., site-directed mutagenesis*) after doing the Golden Gate reaction.
Here are the changes that we made (4/29/2014):
PirB
VVS: GTG GTC TCC -&gt; GTG GTG TCC
GTC frequency: 0.24
GTG frequency: 0.47
LVS:  CTG GTC TCA -&gt; CTG GTG TCA
GTC frequency: 0.24
GTG frequency: 0.47
ET:  GAG ACC -&gt; GAA ACC
GAG frequency: 0.58
GAA frequency: 0.42
WARNING: We replaced with a lower frequency codon
LilrB2
VVS: GTG GTC TCC -&gt; GTG GTG TCC
GTC frequency: 0.24
GTG frequency: 0.47
*If you really care, you can mutate these sites back using site-directed mutagenesis (e.g. this kit: http://www.genomics.agilent.com/en/Site-Directed-Mutagenesis/QuikChange-Lightning/?cid=AG-PT-175&amp;tabId=AG-PR-1162)

<H2>LOCATING A Q CUT SITE WITHIN A GENE</H2><A href="#top">return to top</A>
When ordering gBlocks, one must be mindful that the desired DNA does not exceed the maximum size of 2kb. In the case where the gene of interest is close to, or longer than, 2,000 base pairs (for example, the PirB gene is &gt;2,700 base pairs in length), it is necessary to find an appropriate cut site such that 2 (or more) gBlocks may be ordered, and subsequently ligated to reconstruct the whole gene.
The process of identifying and selecting an appropriate cut site within a gene is as follows:
Find the midway mark of the gene. For PirB, this would be around base pairs 1,350 - 1,360.
Identify the 4bp sequence for the Q1 cut site.
Scan the middle region of the gene for this sequence (consider +/- 100bp from the midpoint). For Q1 in PirB, look for 'AGGT' between bases 1,250 to 1,460.
If found, then this is an appropriate cut site: use as the ligation site when constructing pENTR (the entry plasmid).
If not found, repeat the procedure for Q2, then Q3, and so on, until an appropriate cut site is found.
Once an appropriate cut site is located, split the gene into two files, and order each &quot;half&quot; in a separate gBlock.

<H2>CHECKING PRIMER TB WITH NEB</H2><A href="#top">return to top</A>
Link to Tm Calculator: https://www.neb.com/tools-and-resources/interactive-tools/tm-calculator
Product Group: Phusion
Product: Phusion High-Fidelity PCR Master Mix (HF Buffer)
Primer Conc. (nM): 500
For the primer sequences, input only the binding region and not the entire primer.
For PCR protocol: https://www.neb.com/protocols/2012/09/06/protocol-phusion-high-fidelity-pcr-master-mix-with-hf-buffer-m0531

<H2>OLIGOMERIZING BETA AMYLOID</H2><A href="#top">return to top</A>
Derived from the Abcam website: http://www.abcam.com/amyloid-beta-peptide-1-42-human-ab120301.html  This protocol is for 100 µg polypeptide.
In a fume hood, resuspend 100 µg polypeptide in 100 µL neat HFIP (1,1,1,3,3,3-hexafluoro-2-propanol).
Incubate at room temperature for 1 hour, vortexing briefly every 10 minutes at a moderate speed.
Sonicate in a sonicating water bath on high for 10 minutes.
In a fume hood, place the HFIP/peptide solution under a gentle stream of dry nitrogen gas.  Continue until the HFIP evaporates.
Resuspend in 83 µl neat DMSO.  Incubate at room temperature for 12 minutes.
Aliquot 10 µl volumes into individual 700 µl microcentrifuge tubes.  Store at -80°C.
Oligomerization
Retrieve a single 10 µl aliquot from the -80.  Resuspend in 90 µl PBS for 100 µl total solution at a final concentration of 25 µM. 
Incubate 2 hours at room temperature to allow for peptide aggregation.  
If necessary, remove the insoluble portion by centrifuging at 16,000 xg for 15 minutes.
The solution should be stable at 4° for at least a week.
NOTE: The non-biotinylated ABetan (white tubes, labelled A{Beta}) is in 20 µl aliquots; resuspend in 180 µl PBS for the same working concentration. 
As per the LilrB2 paper , fig 1H, LilrB2/Beta amyloid binding gets close to saturation at about 500 nM.  This is 10 µl of 25 µM working stock diluted into 500 µl of media (in one well of a 24-well plate.)
Resources (alternate protocols, etc)
Reconstituting Peptide from Package:
http://www.anaspec.com/products/product.asp?id=30304
Add 1% NH4OH directly to powder. Add 70-80 uL NH4OH per 1 mg of the peptide. 
Dilute this solution in 1xPBS to your working concentration (should be less than 1mg/mL).
Gently vortex to mix. 
Oligomerizing (LilrB2 Paper)
Sonicate for 30 seconds (
Dilute in 1X PBS to a concentration of 100 uM
Incubate at 22C for 16 hours 
Incubate at 4C for 24 hours
Centrifuge at 16000 x g for 15 minutes 
Collect the supernatant as oligomerized 
Oligomerizing (Other protocols)
HFIP Treatment - to ensure monomerization (Abcam protocol)
http://www.abcam.com/amyloid-beta-peptide-1-42-human-ab120301.html
-Dissolve peptide in 100% HFIP. Final concentration = 1mg/mL.
-Incubate at room temperature for 1 hour with occasional vortexing.
-Sonicate for 10 minutes in a water bath sonicator (optional according to some protocols).
-Aliquot into microcentrifuge tubes.
-Dry HFIP-peptide solution under a gentle stream of N2 gas (some protocols leave it to dry in a fume hood - no N2). 
-Store at -80C. Do not freeze thaw.
Dissolve an aliquot in 100% DMSO to 5mM.
Add to ice-cold F12 medium (DMEM?) to a concentration of 100 uM.
Incubate at 4C for 24 hours. 
Centrifuge at 14000 x g for 10 minutes. Supernatant is a mixture of monomers and oligomers. 
This protocol (steps 2-5) is primarily from a paper cited by several other papers, including (indirectly) the LilrB2 paper:
http://onlinelibrary.wiley.com/enhanced/doi/10.1046/j.1471-4159.2001.00592.x/
(HFIP monomerizes http://www.anaspec.com/products/product.asp?id=48257)
Verifying Identity of Supernatant: 
Western blot 
 Math
How much beta amyloid per aliquot?
100 µl aliquot size * 25 µmol/L beta amyloid concentration * (L/10^6 µl) * (1000 nmol / µmol) = 2.5 nmol / aliquot
How many µg is that?
2.5 nmol * (4853.9 g/mol) * (mol/10^9 nmol) * (10^6 µg/g) = 12.1 µg in 10 µL DMSO.
So, 100 µg gets us 8 aliquots of 12.1 ug in 80 µl DMSO

<H2>WESTERN BLOT</H2><A href="#top">return to top</A>
Cell Lysis
For a 24 well plate:
Aspirate media.
Add 500 uL of cold PBS to each well.
Aspirate.
Add 200 uL of trypsin.
Incubate for 90 seconds.
Neutralize with 800 uL of cold complete media.
Pipette up and down to disperse clumps.
Transfer contents of each well to 2 mL labelled eppendorf tubes.
Centrifuge 2500 RPM for 8 minutes.
Aspirate supernatant. 
Add 500 uL of lysis buffer. 
Incubate at 4C with agitation for 30 minutes. 
Centrifuge at 12000 RPM for 20 minutes. 
Transfer supernatant to new labelled tubes and store at 4C (or on ice). 
Discard pellet. 
Denaturing Protein 
Add 50uL of betamercaptoethanol (reducing agent) to 950uL of Laemmli dye.
Add Laemmli dye + betamercaptoethanol to lysate so that it dilutes to 1x (equal volumes of Laemmli and lysate if Laemmli is 2x).  For one gel, this means that for each sample add 10uL dye to 10uL sample.
Transfer to heat block set to 98C and boil for 5 minutes. 
Store at 4C (reboil if reloading).
BCA Assay (to check protein concentration)
For more details: https://www.piercenet.com/instructions/2160412.pdf
For a sample Excel spreadsheet, see: BCR WB BCA 2014-8-1.xls
For a 20mL solution of working reagent, add 10mL Solution A, 9.6mL Solution B, and 0.4mL Solution C to a 50mL tube.
In a 96 well plate, make three rows of protein standards.  Add 150uL of protein standard solution and 150uL of working reagent to each well.  To make the protein standards:<TABLE><TBODY><TR><TD>2. Vial</TD><TD>Volume of Diluent</TD><TD>Volume and Source of BSA</TD><TD>Final BSA Concentration</TD></TR><TR><TD>A</TD><TD>4.5 mL</TD><TD>0.5 mL of Stock</TD><TD>200ug/mL</TD></TR><TR><TD>B</TD><TD>8.0 mL</TD><TD>2.0 mL of vial A dilution</TD><TD>40ug/mL</TD></TR><TR><TD>C</TD><TD>4.0 mL</TD><TD>4.0 mL of vial B dilution</TD><TD>20ug/mL</TD></TR><TR><TD>D</TD><TD>4.0mL</TD><TD>4.0 mL of vial C dilution</TD><TD>10ug/mL</TD></TR><TR><TD>E</TD><TD>4.0mL</TD><TD>4.0 mL of vial D dilution</TD><TD>5ug/mL</TD></TR><TR><TD>F</TD><TD>4.0mL</TD><TD>4.0 mL of vial E dilution</TD><TD>2.5ug/mL</TD></TR><TR><TD>G</TD><TD>4.8mL</TD><TD>3.2 mL of vial F dilution</TD><TD>1ug/mL</TD></TR><TR><TD>H</TD><TD>4.0mL</TD><TD>4.0 mL of vial G dilution</TD><TD>0.5ug/mL</TD></TR><TR><TD>I</TD><TD>8.0mL</TD><TD>0</TD><TD>0ug/mL (blank)</TD></TR><TABLE>
Add dilutions of samples to a final volume of 150uL in separate wells, and add 150uL of working reagent to each of those sample dilutions.  Do at least 2 replicates per dilution.  For a seeded 24 well plate, dilutions of 75uL sample + 75uL dH2O + 150uL working reagent and 25uL sample + 125uL dH2O + 150uL working reagent tend to result in concentrations that fall within the range of the protein standards.  However, when seeding plates with larger wells one should adjust the dilutions accordingly so that the protein concentrations fall within the range of the protein standards.
Cover the plate with a sticky air-tight seal, being sure to close off each well individually so that wells don't contaminate one another.
Incubate the plate at 37C, slow shaking, for 2 hours.  Alternately, shake for 30 seconds and then incubate standing for 2 hours at 37C.
Cool the plate to room temperature.
Use a plate centrifuge (balanced with another plate) to spin condensation/displaced liquid back into the wells.
Measure the absorbance at 562nm on a plate reader.
BCA assay working reagent requires special disposal.  Make sure to dispose it in a plastic bottle rather than dumping down the drain.
Take the average of the replicates.  Subtract from these averages the average blank reading (standard I).
Plot a standard curve by plotting the average blank-corrected reading for each standard versus its concentration in ug/mL.  Concentration should be on the x axis and absorbance on the y axis.  Then, fit a polynomial (degree 2) curve to the data.
Use the equation for the polynomial curve to calculate back the concentrations for the average blank-corrected absorbances of each sample.  You will need to solve for x in order to find the concentration (y=absorbance, x=concentration).  Make sure to use the correct solution to that equation.  For example, one equation should give higher concentrations for samples with lower absorbances - this is NOT the correct equation to use.  Also, don't just use the equation that Excel gives on the chart because that it is not always precise enough to give accurate concentrations.  Use the LINEST function to find more exact values for the coefficients of the curve fit.
Find the concentration of the initial undiluted sample based on the concentrations of the average absorbances for each dilution of sample.
Running an SDS-PAGE gel
Use BioRad gel.
Load gel into gel running machine.
Add 1000 mL of running buffer to both the inside and outside compartments. Make sure the buffer covers the top of the wells.
Add 5uL of ladder to the first and last wells of the gel.  
Add 16uL of sample to each well.  This should correspond to about 10ug of protein, but down to 2ug can still give some signal.
Run gel at 200V for 30 minutes (or until the dye front has reached the bottom of the gel.
Remove the gel from the running machine and then remove the gel from its plastic encasement, being careful not to rip the gel.  Cut off the top left corner of the gel.
Transfer
For more details: http://www.bio-rad.com/webroot/web/pdf/lsr/literature/M1703940.pdf
Cut two sheets of extra thick blotting paper and one sheet of nitrocellulose membrane to the size of the gel (one set of 2xblotting paper/1xmembrane per gel being blotted).  Cut off the top left corner of the membrane (so that you can orient it relative to the gel).  Also remember that the membrane comes packaged in a protective blue paper - that's not part of the membrane.
Soak the blotting paper and in transfer buffer.  Equilibrate the membrane in transfer buffer for 15-30 minutes.  Equilibrate the gel in transfer buffer for 15 minutes.  Don't let the gel equilibrate for too long or the proteins might diffuse out.
Put a piece of extra thick blotting paper on the anode and roll a pipette over the surface to get rid of bubbles.  Throughout the loading of blotting paper/membrane/gel try not to get the anode too wet because it makes transfer less efficient.
Put the membrane on top of the blotting paper and roll a pipette over it to get rid of bubbles.
Put the gel on top of the membrane (make sure that it's aligned in the center).  Roll a pipette over it to get rid of the bubbles and make sure that the gel is lying perfectly flat.
Put the second piece of extra thick blotting paper on top of the gel and remove air bubbles with the pipette.
Place the cathode and safety cover on the unit.
Run at 10V for 15 minutes.
Open the transfer machine and dispose of blotting paper and gel.  You can tell that you got successful transfer to the membrane if the ladder now shows up on the membrane.
Incubations
Wash the membrane in PBS for 5 minutes, shaking (40mL of PBS required for incubating in a pipette tip box lid).
Incubate the membrane with blocking buffer (preferably 5% BSA) for 1 hour, shaking.  5% BSA is made by adding powdered BSA (in Deepak's fridge) to PBST.  Make 100mL (5g BSA to 100mL PBST).  It is difficult to suspend BSA in solution so be sure to vortex and to filter before adding the buffer to the membrane.  Use a vacuum filter for the filtration.
Save the blocking buffer (it can keep for up to a week).
Wash three times for 5 minutes in PBST.
Incubate the membrane with primary antibody solution for 1 hour, shaking.  (Use antibodies diluted in PBST.  Dilute according to manufacturer's recommendations.  Start at the low end of the dilutions.  For polyclonal antibodies like our anti-GFP consider using an even lower dilution.)
Save the antibody solution (it can keep for up to a week).
Wash three times for 5 minutes in PBST.
Incubate the membrane with secondary antibody solution for 1 hour, shaking.  Be sure to cover this because the secondary antibodies are photosensitive.  (Use antibodies diluted in PBST.  Dilute 1/15000.)
Save the antibody solution (it can keep for up to a week).
Wash three times for 5 minutes in PBST.
Image.  Leave the membrane in PBST when taking to image so that it does not dry out (dried out portions show up on the blot reading).
Lysis Buffer 
1% NP40 
0.1% tween 20
1x HALT protease inhibitor 
150 mM NaCl
50 mM Tris <H2>ATTACHING B CELLS USING POLY-L-LYSINE</H2><A href="#top">return to top</A>
DILUTE POLY L LYSINE to 0.1 mg/mL
Pipette 200 uL into each well
Rock plate gently to ensure an even coating
wait 5 minutes
gently aspirate the Poly-L-Lysine
rinse 3x with sterile water
add B-Cell Culture<H2>BETA AMYLOID STREPTAVIDIN STAINING FOR LIVE MICROSCOPY</H2><A href="#top">return to top</A>
aspirate off media
add 500 uL PBS and aspirate
Add 500 uL complete media
add beta amyloid to 500 nM or other (500 nM=10)
incubate in the incubator for 2hr
aspirate media
add 500 uL warm PBS and aspirate
add 500 uL warm  complete media
add streptavidin to a 10 ug/mL (5uL for our aliquot)
Incubate in the dark for 1hr 30min
aspirate media 
add 500 uL of PBS
Microscopy time!<H2>BETA AMYLOID STREPTAVIDIN PROTOCOL FOR CYTOMETRY</H2><A href="#top">return to top</A>
Aspirate off media.
Rinse wells with 500uL of PBS/versene. Aspirate.
Add 200uL of trypsin/EDTA to each well. Incubate at room temperature for 90 seconds, then neutralize with 800uL of warm complete media
Triturate cells to remove clumps (pipette up and down)
Transfer suspended cells to 15 mL Falcon Tubes.
Centrifuge cells 5min 2500 rpm at 23°C.
Resuspend cells in warm PBS, 10% FBS, 1% sodium azide to a concentration of 5X10^6 cells per mL with a total volume of 1000 uL
NOTE: DO NOT ASPIRATE SOLUTIONS CONTAINING SODIUM AZIDE. Pipette waste into an empty bottle and dispose of it appropriately later. 
Add Beta Amyloid at 500 nM concentration (or other)
Incubate at 37C for 1hr 30min
Move all warm reagents to the refrigerator
Move to cold room for 30 min
Centrifuge cells 5min 2500 rpm at 4°C
Resuspend cells in 1000 uL ice-cold PBS, 10% FBS, 1% sodium azide
Add Streptavidin to a 10 ug/mL (5 uL for our aliquot)
Incubate in the cold room in the dark for 1hr 30 min
Centrifuge cells 5min 2500 rpm 4°C
Resuspend cells in 1000 uL of ice cold PBS, 10% FBS, 1% sodium azide
Count cells on a hemacytometer (not all of samples, just a few.)  Dilute to ~1e6 cells/ml with ice-cold  PBS, 10% FBS, 1% sodium azide. 
Transfer suspension to cytometry tubes.
Run flow cytometry.<H2>GELATIN TREATMENT FOR GLASS PLATES</H2><A href="#top">return to top</A>
Many cultured cell types do not typically adhere well to glass bottom plates. Adding a thin layer of gelatin can help the cells to adhere better.
Gelatin preparation should be done before transfection is started (before you start seeding the cells). 
Find the 0.1X bottle of gelatin. It is already at the proper concentration. It is usually kept above the TC room centrifuge. Otherwise, possibly check the Cold Room if the TC room was recently cleaned.
Put glass cover slips in wells
Pipette gelatin into each well to be used of the plate such that the bottom of the well is fully coated with gelatin. 
-For a 24 well plate, add 0.5 to 1.0 mL gelatin per well
-For other plates, scale appropriately --&gt;  glass 35 mm dishes would be at least 1.5 mL 
-More recommendations by Millipore: http://www.millipore.com/userguides/tech1/mcproto045 
Swirl plate (figure eight or shake forward/side) gently to evenly spread the gelatin. (Make sure the bottom of each well is covered!)
Let gelatin set for 20 minutes. 
Just prior to seeding the cells, gently aspirate the gelatin from each well. (Make sure you remove all the liquid but don't scrape the bottom of the plate -- tilt the plate to make it easier)
Seed plates<H2>IMMUNOSTAINING FOR FLUORESCENT MICROSCOPY</H2><A href="#top">return to top</A>
Make sure you perform this procedure BEFORE THE CELLS BECOME CONFLUENT.  If they are confluent, they'll sheet off the coverslips and then you won't be able to stain them.
Aspirate media leaving cells (be careful not to aspirate any of the cells)
Gently add 500 uL of Fixation buffer (drizzled down the sides of the wells).  Wait for 20 minutes at room temperature.
Use the pipette to aspirate the Fixation Buffer.  DO NOT ASPIRATE WITH THE VACUUM; the fixation buffer contains paraformaldehyde, which must be treated as hazardous waste.  Instead transfer the fixation buffer to the formaldehyde waste container.
Add 500 uL PBS; transfer the 24-well plate to the benchtop.
Squirt some water on the bench, then lay down several sheets of parafilm side-by-side.  In the steps below, pipette the solution onto a spot on the parafilm (it will form a &quot;bubble&quot;), then float the coverslip with the cells upside-down on the solution.  When you're done, aspirate the liquid off of the parafilm.
CAREFULLY fish the coverslips out of the wells. Wash the coverslips with 200 µl PBS for 5 minutes.
Permeabilize the cells with 100 µl 0.2% Triton X-100 for 15 minutes.
Block the cells with 200 µl PBS+4% BSA for at least 20 minutes (longer is better)
Prepare 25 µl primary antibody solution for each coverslip: PBS + 4% BSA + antibodies
Incubate cells with 25µl primary antibody solution for 60 minutes.  Be careful that they don't dry out!  Saturate a kimwipe or paper towel with water, then put the the wipe in a cover over the incubating coverslips.  (The top of a cryobox works well.)
Wash the coverslips 3x: 200 µl PBS, 5 minutes each.
Prepare 25 µl secondary antibody solution: PBS + 4% BSA + 2° antibody + DAPI (diluted 1:500, ~1 ug/ml)
Incubate the cells as above (30 minutes); this time, make the cover light-tight (again, use the top of a cryobox.)
Wash the coverslips 3x: 200 µl PBS, 5 minutes each.
Pipette 20 µl of embedding solution on a microscope slide.  Slowly lower the sample onto it, taking care to avoid bubbles!
Store the slide flat, in the dark, overnight to let the embedding solution cure.<H2>FIXATION</H2><A href="#top">return to top</A>
If you need to wait longer than 1 hour before analysis, you may need to fix the cells after step 5. This can preserve them for several days (this will stabilize the light scatter and inactivate most biohazardous agents). Controls will required fixation using the same procedure. Cells should not be fixed if they need to remain viable. There are several methods available. The fixation for different antigens will require optimization by the user.
Paraformaldehyde 0.01% to 1% for 10-15 minutes only, 100 µl per sample.
Acetone or methanol:N/B polystyrene/plastic tubes are not suitable for use with acetone
Add 1ml ice cold acetone to each sample.
Mix gently. Place at -20oC for 5-10 minutes.
Centrifuge, wash twice in PBS 1% BSA. (Do not add sodium azide to buffers if you are concerned with recovering cell function e.g. if cells are to be collected for functional assays. It inhibits metabolic activity)
For more information visit http://www.abcam.com/protocols

<H2>INDIRECT FLOW CYTOMETRY</H2><A href="#top">return to top</A>
Indirect flow cytometry (FACS) protocol. General procedure for flow cytometry using a primary antibody and conjugated secondary antibody. Indirect labelling requires two incubation steps, firstly with a primary antibody then with a compatible secondary antibody. The secondary (and not the primary) antibody has the fluorescent dye (FITC, PE, Cy5, etc.) conjugated. Please note that this is a general protocol and you may need to adapt it for your applications.
General Procedure:
Harvest and wash the cells then determine the total cell number. Cells are usually stained in polystyrene round bottom 12 x 75 mm 2 Falcon tubes. However, they can be stained in any container for which you have an appropriate centrifuge e.g. test tubes, eppendorf tubes, and 96 well round bottomed microtiter plates. In general,cells should be spun down hard enough that the supernatant fluid can be removed with little loss of cells, but not so hard that the cells are difficult to resuspend.It is always useful to check the viability of the cells which should be around 95% not less than 90%.
Resuspend the cells to approximately 1-5 x 10 6 cells/ml in ice cold PBS, 10% FCS, 1% sodium azide. (Use ice cold reagents/solutions and at 4oC as low temperature and presence of sodium azide prevent the modulation and internalization of surface antigens which can produce a loss of fluorescence intensity)
Add 100 µl of cell suspension to each tube.
Add 0.1-10 µg/ml of the primary antibody. Dilutions, if necessary, should be made in 3% BSA/PBS.
Incubate for at least 30 min at room temperature or 4oC in the dark.
Wash the cells 3-times by centrifugation at 400 g for 5 min and resuspend them in ice cold PBS. You may need to adjust the conditions of the centrifugation (the force and the time) for the cell types used.
Dilute the fluorochrome-labeled secondary antibody in 3% BSA/PBS at the optimal dilution (according to the manufacturer’s instructions) and then resuspend the cells in this solution.
Incubate for at least 20-30 minutes at room temperature of 4oC. This incubation must be done in the dark.
Wash the cells 3 X by centrifugation at 400 g for 5 min and resuspend them in ice cold PBS, 3% BSA, 1% sodium azide.
Store the cell suspension immediately at 4°C in the dark.
Analysis: For best results, analyze the cells on the flow cytometer as soon as possible.We recommend analysis on the same day. For extended storage (16 hr) as well as for greater flexibility in planning time on the cytometer, resuspend cells in 1% paraformaldehyde to prevent deterioration.<H2>CO-TRANSFECTION OF HEK293 WITH LIPOFECTAMINE 2000 (24 WELL PLATE)</H2><A href="#top">return to top</A>
From Invitrogen
Remember to include the proper positive and negative controls in your experiment.
One day before transfection, plate cells in the appropriate amount of growth medium without antibiotics such that they will be 80-90% confluent at the time of transfection.
For each transfection sample, prepare DNA-RNAi molecule-Lipofectamine 2000 complexes as follows.
-Dilute 1 ug of DNA and 100 pmoles (0.0001 umoles) of RNAi in 50 uL DMEM. Mix gently.
-Mix Lipofectamine 2000 gently before use, then dilute 0.5-1.5 uL in 50uL DMEM. Mix gently and incubate for 5 minutes at room temperature.
-After the 5 minute incubation, combine the diluted DNA and RNAi molecule with the diluted Lipofectamine 2000. Mix gently and incubate for 20 minutes at room temperature to allow complex formation to occur. The solution may appear cloudy, but this will not impede the transfection.
Add the DNA-RNAi molecule-Lipofectamine 2000 complexes to each well containing cells and medium. Mix gently by rocking the plate back and forth.
Incubate the cells at 37°C in a CO2 incubator until you are ready to harvest cells. Removal of complexes or media change is not required; however, growth medium may be replaced after 4-6 hours without loss of transfection activity.<H2>FACS CELL PREP (24 WELL PLATE)</H2><A href="#top">return to top</A>
MATERIALS: Ensure that trypsin is completely thawed, but PBS/versene and complete media are cold (do NOT place in warm water bath; take them straight from the fridge).
Aspirate off media.
Rinse wells with 500uL of PBS/versene. Aspirate.
Add 200uL of trypsin/EDTA to each well. Incubate at room temperature for 90 seconds, then neutralize with 800uL of cold complete media.
Triturate (pipette up and down) to disperse clumps. Transfer to cytometry tubes. Place tubes in ice (cells must be kept cold; in stasis).
Number the wells and the corresponding tubes (the FACS machine deals with numbered samples better). Record the contents of the wells to which the numbers correspond.
Count the cells from 2-3 tubes, using the hemacytometer (typically 2x10^6 per mL). Add PBS to dilute the cells to 1x10^6 per mL (typically, an additional 1mL is required).
Run FACS.<H2>TRANSFECTION (USING LIPOFECTAMINE 2000)</H2><A href="#top">return to top</A>
Calculate the volume of DNA you are adding to each well based on the concentrations of the midiprepped DNA and the mass (ng) of each plasmid you want to add. Total DNA transfected per well should be 1ug.
Label a DNA dilution tube (2mL eppendorf) for each well you are planning to transfect. These tubes will have a total volume of 50uL.
Based on the volumes of calculated in Step 1, add enough volume of DMEM to each dilution tube such that once you add your DNA you will have a total volume of 50uL.
i.e. Volume of DMEM = 50uL - Volume of DNA
Add the volume of DNA calculated in Step 1 to each DNA dilution tube. 
Add 50uL of DMEM to new 2mL eppendorf tubes (same number of tubes as DNA dilution tubes).
Add to each new tube 2uL of Lipofectamine 2000
Incubate for 5-10 minutes.
Add 50 uL of the Lipofectamine+DMEM mix to each of the DNA dilution tubes.
Incubate for 30 minutes. 
Transfer 100uL of each DNA+Lipofectamine+DMEM mix to its corresponding well. 
Put transfected plate back into 37C incubator. <H2>TRANSFECTION (USING LIPOFECTAMINE LTX)</H2><A href="#top">return to top</A>
One day before transfecting, seed cells (in a well plate) so that they are 70-90% confluent at time of transfection.
Required Materials: Eppendorf tubes, Opti-MEM Medium, PLUS Reagent, Lipofectamine LTX Reagent.
Dilute Lipofectamine in Opti-MEM.  Wait 5-10 min.
Dilute DNA in Opti-MEM, then add PLUS.
Add diluted DNA to diluted Lipofectamine in a 1:1 ratio, to form DNA-lipid complex. 
Incubate at room temperature for 30 minutes.
Add DNA-lipid complex to cells (wells).
Incubate cells for 1-3 days at 37C.
Visualize/analyze transfected cells (flow cytometry).<TABLE><TBODY><TR><TD>STEP</TD><TD>COMPONENT</TD><TD>6-well</TD><TD>24-well</TD><TD>96-well</TD></TR><TR><TD>1</TD><TD>Adherent Cells</TD><TD>0.25-1 x 10^6</TD><TD>0.5-2 x 10^5</TD><TD>1-4 x 10^4</TD></TR><TR><TD>3</TD><TD>Opti-MEM Medium</TD><TD>150uL x 4</TD><TD>25uL</TD><TD>25uL x 4</TD></TR><TR><TD>Lipofectamine LTX Reagent</TD><TD>6, 9, 12, 15 uL</TD><TD>1uL</TD><TD>1, 1.5, 2, 2.5 uL</TD></TR><TR><TD>4</TD><TD>Opti-MEM Medium</TD><TD>700uL</TD><TD>25uL</TD><TD>125uL</TD></TR><TR><TD>DNA (0.5-5 ug/uL)</TD><TD>14ug</TD><TD>1ug (total)</TD><TD>2.5ug</TD></TR><TR><TD>PLUS Reagent</TD><TD>14uL</TD><TD>0.5uL</TD><TD>2.5uL</TD></TR><TR><TD>5</TD><TD>Diluted DNA (with PLUS)</TD><TD>150uL</TD><TD>26uL</TD><TD>25uL</TD></TR><TR><TD>Diluted Lipofectamine LTX</TD><TD>150uL</TD><TD>26uL</TD><TD>25uL</TD></TR><TR><TD>7</TD><TD>DNA-lipid complex (per well)</TD><TD>250uL</TD><TD>52uL</TD><TD>10uL</TD></TR><TR><TD>FINAL DNA (PER WELL)**</TD><TD>2500ng</TD><TD>1000ng</TD><TD>100ng</TD></TR></TBODY></TABLE><H2>SPLITTING CELLS AND SEEDING PLATES</H2><A href="#top">return to top</A>
(WELL PLATES SHOULD BE SEEDED ONE DAY PRIOR TO TRANSFECTING)
Prior to transfection, the cells should be 70-90% confluent
Aspirate off the media from the dish the cells are growing in
Add 3mL of PBS/versene, aspirate
Add 2.5 mL of trypsin
Incubate for 2 minutes maximum (do not over trypsinize)
Add 9.5mL of complete media to inhibit the trypsin
Resuspend in a conical tube
Spin down in the centrifuge at 2500 RPM for 5 minutes at room temperature (20-25 C)
Aspirate off the media (be sure not to aspirate the pellet at the bottom!) and resuspend in 10mL of complete media.
Pipette 10-15 uL of culture from the conical tube onto a hemocytometer
Take the hemocytometer to the microscope and count the number of cells in a 4X4 grid (or take an average from multiple 4X4 grids)
The number of cells you count (if you are looking at a large 4X4 grid) is the amount per 0.0001 mL (every 100 cells you count = 1 million cells/ MILLI LITER)
Dilute a sample of the suspension to the correct cell concentration
-A new culture plate needs 1 million cells in 10-12 mL media
-A 24 well plate needs ~50,000 (attractene) or ~100,000 (Lipofectamine) cells in ~0.5mL media per well.
Pipette the necessary volume into each well of your plate for however many wells you need
Tilt/swirl to evenly distribute cells within the wells.
SEEDING PLATES (RAMOS CELLS):
Use modified RPMI-1640 media (it has some added components - the bottle has Brian's initials on it and it's on a shelf in the fridge).
Pipette cells from plate into 50mL conical tube.
Centrifuge at 2500 RPM for 5 minutes at room temperature.
Aspirate off media and resuspend in 10mL Ramos media.
Add 200,000 cells to about 10mL media in a plate (roughly 1:10 split every 3 days).<H2>MAKING CELL CULTURE MEDIA</H2><A href="#top">return to top</A>
Precautions: Make sure that all equipment is functioning as expected first, so that you do not get caught unaware in the middle of a protocol. Media is to be made in the hood. Wash EVERYTHING that goes into the hood. As much as the water is changed in the hot water bath and is blue from an antimicrobial, it is still nasty. We don't want whatever could be in there growing in our culture media, or with our cells. 
Materials: DMEM, Fetal Bovine Serum (FBS), Penicilin/Streptomycin (Pen/Strep + L-Glu), Non-Essential Amino Acid
Protocol:
Get a filter column from the cabinet outside the Tissue Culture Room
Attach the vacuum tube to the nozzle 
Add 400mL of DMEM to the filter column
Add 50mL of FBS
Add 5mL of 100x Penicillin-Streptomycin-Glutamine
Add 5mL of 100x Non Essential Amino Acids
Let the liquid run through
Add the final 35mL of DMEM to wash through any of the ingredients above through the filter. <H2>PREPARING LB AGAR PLATES</H2><A href="#top">return to top</A>
17.5 grams of LB Agar powder makes 500 mL.
1. Weigh 17.5 grams of LB Agar powder, add into sterilized Pyrex container.
2. Fill with sterile water up to 500 mL mark.
3. Cap loosely and tape with autoclave tape.
4. Autoclave for 40 minutes (see an instructor).
5. Wait until cool enough that it doesn't hurt to touch.
If starting from solid agar in the fridge, heat in microwave (cap unscrewed) 3min on high, 5 min on 30% (till it melts completely)
Wait till cool enough so that you can touch it for a minute (60 degree Celsius)
6. Add appropriate volume of Amp or Kan (1000X). For 500 mL of LB Agar, add 500 uL of each antibiotic needed.
7. Swirl to mix, use pipette to transfer 20 mL into each plate. Avoid bubbles. 
8. Allow plates to cool and solidify. Mark antibiotic on side of plate. 
If LB Agar solidifies before being poured, microwave to liquefy again.<H2>POURING GG DONOR PLATES</H2><A href="#top">return to top</A>
Start from solid agar in the fridge or start from powder and follow the protocol.
If starting from solid agar in the fridge, heat in microwave (cap unscrewed) 3min on high, 5 min on 30% (till it melts completely).
Wait till cool enough so that you can touch it for a minute (60 degree Celsius).
Add appropriate volume of Kan (1000X). For 500 mL of LB Agar, add 500 uL of Kan.
Add appropriate volume of X-Gal (1000X). For 500 mL of LB Agar, add 500 uL of X-Gal. (X-Gal is solid at 4 degrees, so take it out to thaw.)
Swirl to mix, use pipette to transfer 20 mL into each plate. Avoid bubbles. 
Allow plates to cool and solidify. Mark antibiotic on side of plate. 
If LB Agar solidifies before being poured, microwave to liquefy again.

<H2>MAKING SOC</H2><A href="#top">return to top</A>
S.O.C is made by
dissolving 0.5 ml of 20% glucose in
25 ml of SOB<H2>MAKING CELL STOCKS</H2><A href="#top">return to top</A>
Acquire 500 uL of freshly grown cells in liquid culture. This usually comes from the leftover culture that was grown for the miniprep. Fresh Culture can also be grown by inoculating a single colony in TB with appropriate antibiotic marker.
Add 500 uL of culture to microtube.
Add 500 uL of filter sterilized 50% v/v glycerol in water to culture.
Label the tube!
Place in appropriate box in -80C freezer.
For anything important, make two cell stocks (by splitting or by doing the above twice) and store one in the backup cell stock box (Starting 6/28).

<H2>ANNEALING AND KINASING OLIGOS</H2><A href="#top">return to top</A>
1) Create 100uM stock solutions of each oligo
2) Create a 20uL solution of the following:
2 uL of 100uM stock solution of top oligo
2 uL of 100uM stock solution of complementary bottom oligo
2 uL T4 DNA ligase buffer
1 uL T4 poly nuclease kinase (T4PNK)
13 uL water
3) Put in thermocycler and run the following program:
37°C for 30 mins
95°C for 5 min
Ramp to 25°C by 0.1% (0.1°C per second) and keep at 25°C for 5 min
End at 4°C

<H2>BP REACTION</H2><A href="#top">return to top</A>
1. Thaw BP clonase on ice
2. Mix 0.5 uL BP clonase with 1 uL appropriate pDONOR (typically ~150 ng/uL) and 1 uL attB-flanked PCR fragment (typically ~50 ng/uL).
3. Incubate at room temperature for 1-1.5 hours.
4. Ad 0.5 uL Proteinase K.
5. Incubate at 37 C 20 min.
6. Transform 1 uL into 10G competent cells.
7. Outgrow w/ SOC 1hr.
8. Plate on Kan, 30C overnight.

<H2>TRANSFORMATION</H2><A href="#top">return to top</A>
Make sure that the incubator (30/37C) and heat block (42C) are ON.
-Put water in the wells of the 42°C heat block.
Make sure required antibiotic plates are present.  Make sure you're using the right antibiotic plates for your plasmid's resistance!
-Warm plates to 37°C.  Cold plates reduce transformation efficiency by an order of magnitude.
-Also warm 500 µl SOC per transformation.
Take the DNA out of --20 freezer, let it thaw.
-Vortex DNA to mix, then spin down.  Make sure it is completely thawed out!
Make sure that all of the required reagents/DNA etc are present at the site of transformation before you take the cells out of the -80.
Thaw the competent cells on ice for 3-4 min.
-You want to add your DNA right as the last bit of cells' ice melts.  Even if it's still a little slushy, that's okay.
Add 1-2 µl of DNA into the comp cells.  Stir with a pipette tip a few times, then put right back on ice.
-If you're transforming the result of a reaction (GG, LR, etc) add 1-2 µl of the reaction.  Don't add more: many of these reactions have additives that screws up transformation.
-If you're transforming plasmid DNA (from a miniprep), either (a) dilute it out so you add only ~10 ng of DNA, or (b) plate only 10 µl of the outgrowth – else you'll get a lawn!  Super-coiled DNA transforms super-efficiently.
Incubate the cells on ice for 30-40 min.
Heat shock the cells for EXACTLY 30 sec at 42 C water bath.
Place back on ice for 2 min.
Add 450 ul of SOC (37° to RT) medium to each tube (S.O.C is made by dissolving 0.5 ml of 20% glucose in 25 ml of SOB. Make sure that the SOC is clear and not cloudy/ contaminated.)
Shake the tubes at 37 C, 280 rpm for 60 min.
Plate 100 µl for a reaction product, or 10 µl in a 100 µl puddle of water for a supercoiled plasmid.
Incubate plates upside down overnight at 37 C or 16-18h at 30C.
Can leave the cells in the incubator for up to 18 hours but no more.


<H2>RESTRICTION DIGEST</H2><A href="#top">return to top</A>
Digestion Protocol
20 ul total
500-1000 ng DNA
1 ul enzyme
2 ul enzyme buffer
fill the rest with water
Pipette up and down thoroughly 
37 degrees for 1-3 hours
4-5 ul loading dye
gel + ladder!
From NEB:
One unit is defined as the amount of enzyme required to digest 1 µg of ? DNA in 1 hour at 37°C in a total reaction volume of 50 µl
So check calculate how much DNA you have and use the right amount of enzyme. Or more.

<H2>RESUSPENDING PCR PRIMERS</H2><A href="#top">return to top</A>
From UTexas protocol
Materials:
TE Buffer
Primer
You will first make a 100 uM master stock
1: spin down primer tube incase there is some primer stuck in cap
2: Calculate amount of buffer needed- multiply nmoles of primer by 10
3: add this many uL of TE to the primer tube
4: gently mix and allow to sit for 10 minutes before making a working stock

<H2>PCR</H2><A href="#top">return to top</A>
Appending Prefix and Suffix
Things to keep in mind!
Annealing temperature of primers (Tm) should be around 60 C
Check the secondary structure of the primers before you order them!
-no individual secondary structures i.e. hairpins
-no heterostructure with the forward and reverse primers together
-free energy of primers should be greater than -4 kCal 
-GC content should be around 50% (40-60% is okay)
If using Phusion Master Mix, use this protocol: https://www.neb.com/protocols/2012/09/06/protocol-phusion-high-fidelity-pcr-master-mix-with-hf-buffer-m0531
Dilute your DNA to the following concentrations:
Template:0.1 - 1 ng/ul
Forward Primer:10 uM
Reverse Primer:10 uM
Set up a small box (e.g. empty pipette tip box) with ice and water. Your DNA and polymerase mix will go into this box before going into the the thermocycler in order to limit endonucelase activity. 
Add the following DNA to a labeled 0.6ml PCR tube 
DNA:Volume
Template:1 uL
Forward Primer:500 nM
Reverse Primer:500 nM
Program the thermocycler as follows
Temperature:Time
98:30s
PAUSE
98:5s
Tm:15
72:(15s)x(#kb)
72:5m
4:forever
Wait for thermocycler to heat up 
Add 22.5uL of polymerase mix (Phusion Master Mix) to your DNA. Mix well and spin down. Transfer tubes to ice as soon as possible. 
Once the thermocycler has heated up to the right temperature (it should be paused at 98C), add tubes to thermocycler and resume PCR program. 
Calculating Reaction Conditions
Use idtdna.com or VectorNTI to calculate melting temperatures of primers
without common overhangs (base pairs 30 to end when read 5' to 3').
PRIMER
Tm
FW
RV
Phusion elongates at a rate of 1kb (1000bp) per 15s. Look up the length of the 
gene of interest and calculate time of elongation.
You should get your Tm from NEB.
If you get the melting temperature of your primer from Genious, the annealing
temperature will be that number minus 2.
Assembling Reaction
Get 0.6mL PCR tubes (not the strip tubes).
Get primers for gene of interest. Resuspend if necessary.
Thaw Phusion supermix on ice.
Add the following (in order):
VOLUME:REAGENT
22.5uL (for 35 cycles):Phusion Supermix
2uL:5uM Forward Primer
2uL:5uM Reverse Primer
1uL:Template DNA(~150ng)
Programming The Thermocycler
(In the 3rd floor thermocycler, the PCR program is named &quot;PHUSION&quot;)
Initial Denaturation: 98C for 5min
LOOP: 30-35 cycles
CYCLE: 
Denaturation: 98C for 10s
Annealing: calculated temperature (typically 55-65C) for 30s
Elongation: 72C for 15s per kb
Final Elongation: 72C for 10min
Store: 4C



<H2>MINI PREP</H2><A href="#top">return to top</A>
Before you Start:
Optional: Add LyseBlue reagent to Buffer P1 at a ratio of 1 to 1000 and mix (If you're the one adding, initial top and check the box on cap)
Add the provided RNase A solution to Buffer P1, mix, and store at 4C. (One vial of RNase A per bottle of Buffer P1 to give final concentration of 100ug/mL. If you're the one adding, initial top and check box on cap. Buffer P1 will be in the fridge)
Add ethanol (96-100%) to Buffer PE before use (see bottle label for volume) and then check mark on cap.
Check Buffers P2 and N3 for precipitates, if any redissolve by placing in water bath at 37C Do NOT vortex.
Steps
Pellet 1-5mL of bacterial overnight culture by centrifugation at &gt;8000 rpm for 3 min at room temperature (15-25C; use 2ml microcentrifuge collection tubes). Decant all the liquid and add 1 ml of the culture into the corresponding tube. Make sure not to mix up the tries.
Resuspend pelleted bacterial cells in 250 uL Buffer P1 and transfer to microcentrifuge tube.
Add 250uL Buffer P2 and mix thoroughly by inverting tube 4-6 times. Do NOT vortex. Mixture turns blue. Do NOT allow this lysis reaction to proceed for more than 5 min.
Add 350uL of Buffer N3 and mix IMMEDIATELY and thoroughly by inverting tube 4-6 times. Do NOT vortex. Mixture is now colorless.
Centrifuge for 10min at 13,000 rpm in table-top centrifuge.
Apply the supernatant to a QIAprep spin column by decanting or pipetting. Do NOT get any of the sticky precipitate.
Centrifuge for 30 - 60s at 13000rpm. Discard flow-through.
Wash the QIAprep column by adding 0.5 mL Buffer PB.
Centrifuge for 30 - 60s at 13000rpm. Discard flow-through.
Wash the QIAprep column by adding 0.75 mL Buffer PE.
Centrifuge for 30 - 60s at 13000rpm. Discard flow-through.
Centrifuge for 1 min to remove residual was buffer.
Place the QIAprep column in a clean 1.5mL microcentrifuge tube. To elute DNA, add 50uL Buffer EB to center of each column. Be careful NOT to pierce column.
Let stand for 1 minute.
Centrifuge for 60s at 13000rpm.
Remove column and discard, tube now contains DNA.
Go to NanoDrop and spec DNA.<H2>MIDI PREP</H2><A href="#top">return to top</A>
NOTES BEFORE STARTING:
Add the provided RNase A solution to Buffer P1, mix, and store at 4C. (One vial of RNase A per bottle of Buffer P1 to give final concentration of 100ug/mL. If you're the one adding, initial top and check box on cap. Buffer P1 will be in the fridge)
Optional: Add LyseBlue reagent to Buffer P2 at a ratio of 1 to 1000 and mix (If you're the one adding, initial top and check the box on cap)
Add ethanol (96-100%) to Buffer PE before use (see bottle label for volume) and then check mark on cap.
Check Buffers P2 and N3 for precipitates, if any redissolve by placing in water bath at 37C. Do NOT vortex.
STEPS:
Harvest bacterial culture by centrifuging at 6000 x g for 15 min at 4C (in 50ml conicals).
Completely resuspend pelleted bacteria in 4ml Buffer P1.
Add 4ml Buffer P2, gently mix by inverting until the lysate appears viscous, and incubate at room temperature (15-25C) for 3 min. If LyseBlue reagent had been added, the cell suspension will turn blue.
Place the QIAfilter Cartridge into a new and suitable tube, allowing space for addition of Buffer BB.
Add 4 ml Buffer S3 to the lysate, and mix by inverting 4-6 times. If LyseBlue reagent has been added, mix the solution until it is completely colorless.
Transfer the lysate to the QIAfilter Cartridge and incubate at room temperature for 10 min. The DNA is now stable; this is an appropriate place in the protocol to pause (if necessary).
During incubation, place QIAGEN plasmid Plus spin columns into the QIAvac 24 Plus. Insert Tube Extenders into each column.
Gently insert the plunger into the QIAfilter Cartridge and filter the cell lysate into the tube.
Add 2 ml Buffer BB to the cleared lysate, and mix by inverting 4-6 times.
Transfer lysate to a QIAGEN Plasmid Plus spin column on the QIAvac 24 plus.
Apply approximately -300 mbar vacuum until the liquid has been drawn through all columns.
To wash DNA, add 0.7 ml Buffer ETR and apply vacuum until the liquid has been drawn through all columns.
To further wash the DNA, add 0.7 ml Buffer PE and apply vacuum until the liquid has been drawn through all columns.
To completely remove the residual wash buffer, centrifuge the column at 10,000 x g (9,700 rpm) for 1 min in a tabletop microcentrifuge.
Place the QIAGEN Plasmid Plus spin column into a clean 1.5 ml tube. To elute the DNA, add 200 ul Buffer EB or water to the center of the QIAGEN Plasmid Plus spin column, let it stand for &gt; 1 min, and centrifuge for 1 min. 


<H2>MAKING LIQUID CULTURE</H2><A href="#top">return to top</A>
MINI-PREP:
Prepare culture in a 15mL, round bottom tube.
Add 3-5mL LB
Add 3-5uL (respectively) of antibiotic (Ampicillin, Kanamycin, etc.)
Pick colony using a pipette tip. Eject tip into tube (tip should remain in tube).
MIDI-PREP:
Prepare culture in a 500mL Erlenmeyer flask.
Add 50mL LB
Add 50uL of antibiotic (Ampicillin, Kanamycin, etc.)
Pick colony using a pipette tip. Eject tip into tube (tip should remain in tube).<H2>LR GATEWAY</H2><A href="#top">return to top</A>
Pro Tips
Run at the larger scale
ALWAYS kill it with PRoteinase K
Transform 4 uL of the reaction
Use ALL the transformation tricks
Plate ALL of it
ALWAYS run a PUC19 control for transformations
Instructions for newbies:
1.5 uL of 10 fM Dest
1.5 uL of 5 fM promoter
1.5 uL of 5 fM gene
1.5 uL of H2O
1.5 uL of LR clonase
Pipette up and down.  Incubate at room temperature overnight.
Next day:  Add 1.5 uL of proteinase K.  Incubate for 15 minutes at 37C, then transform.
Instructions:
USE 3x VOLUME OF EVERYTHING (at least for now) 
Use nanodrop to measure concentration of pEntry vectors, make 5 femtomolar working solution of each pEntry.
Excel sheet set up to calculate the necessary volumes
LR (concentration) calculations.xlsx
Sample calculation:Combine into 1 aliquot:
1uL of 5 fmol of pENTR_L4_Promoter_R1
1uL of 5 fmol pENTR_L1_Gene_L2
1uL of 10 fmol pDEST_R4_R2
3uL Total
WARNING: KEEP ALIQUOTED LR CLONASE MIX AT -80 AT ALL TIMES!
Add 0.5uL LR Clonase Enzyme
Additionally, remember to mix your reactions well after all elements have been added (use a 10 uL pipette set to 3 or 4 uL, then just pipette up and down gently)
Leave at room temperature for minimum of 16 hours, maximum of 24 hours.
transform just 1 uL of the reaction (This should result in around 50-500 colonies (more often than not closer to 500) with high (~90-95%) efficiency)
or storage -20C freezer.<H2>GOLDEN GATE</H2><A href="#top">return to top</A>
50 ng of each piece of DNA being joined
Use nanodrop to find concentration in ng/ul, then divide 50 by that concentration to find the required volume of DNA:
Conc: x ng/uL
Vol: 50/x uL
NOTE: If GGDonr is too concentrated, dilute it with EB or water.
NOTE: Ligase buffer does not like to be freeze-thawed, so use one-time-use aliquots.
x1 uL of DNA1
x2 uL of DNA2
y uL (100ng) Donor
2ul 10X T4 Ligase Buffer
2ul 10X BSA
1ul BsaI (enzyme) HC (high concentration)
1ul T4 Ligase (enzyme) HC (high concentration)
fill to 20uL with SDIH20 (put water in before the buffer and enzymes)
-------------
20ul total
(NOTE: Make sure that Buffer and Enzyme added last, enzyme after buffer)
Take a p20, set it to 10uL and then pipet up and down.
THERMOCYCLER:
(Protocol EBGG)
37C for 5min
Part 1
50X:
37C for 2.5min
4C for 0.5min
16C for 5.5min
Part 2
37C for 10 min
80C for 20 min
4C hold (for 8+ hours)
(Check protocol by looking up the paper or other online GG protocols)




GELS<H2>Gels</H2><A href="#top">return to top</A>
Preparing the Gel
Dissolve UltraPure agarose to a final concentration of 1%(by mass) in TAE buffer in a glass bottle.
Heat the solution in the microwave with frequent stirring to dissolve the agarose homogenously. ~1 minute/200ml solution
Place the solution in a warm water bath for 5 mins.
Add 10 µl SYBRSafe (1:10000) per 100 ml of the solution and mix well.
Pour 50ml of solution per small gel tray. (the gel trays and combs should be pre-cleaned with water and wiped dry).
-Note for combs: 15-well combs hold about 6 ul liquid, 12-well combs hold about 15 ul, 8-well combs hold about 20 ul
-Taping two 8-well comb wells together results in a well that holds up to 100 ul
-Taping three 8-well comb wells together result in a well that holds up to 200 ul
Use 120ml per large gel tray. [need to update amounts]
For the small set: small trays hold 20ml, large trays hold 50ml
Wait for the gels to solidify. ~15 mins
Label and store at 4C.
Running the Gel
When doing gel extraction, it is important to run both an analytical gel (to view under UV) and an extraction gel (from which bands are excised). UV damages DNA, and so we dont want to expose our extracted DNA.
Analytical Gel:
The analytical gel should have between 20 and 100 ng of DNA in each well. It should be an exact copy of the extraction gel with respect to position, voltage, and run time.
Extraction Gel:
This should be the rest of the digestion(s).
The analytical and extraction gels can technically be part of the same physical gel. Make sure to separate with a razor blade before imaging.
Refer to Gel Prep protocol above to determine the amounts of liquid to load for the specific well.
Appropriate Hyperladder to be used for PCR product which is linear. Usually Hyperladder I will be used.
While casting gel, add two sets of lanes; use one set to load an analytical gel. 
Add 2ul gel loading buffer (Orange G 6X; it helps DNA sink into the bottom of the well) to DNA.
Make sure there is enough 1xTAE in the plate holder.
Load 5.0ul of appropriate hyperladder to one of the lanes.
Load appropriate amount of DNA - As much as possible! Usually 15-18ul - (mixed with the buffer) in each well.
Set the timer and voltage to 100V and 25 min.
Analytical Gel Annotation
The following things need to be added to the analytical gel image BEFORE it is posted to the wiki:
Label each lane with part number and amount of DNA loaded
Label each band with length and proposed identification
Include wt% agarose, run time, and voltage
Gel Extraction Protocol using Zymo kit (preferred if available)
Place the extraction gel on the blue light table.
Cut out the appropriate bands. Place into 2mL microtube(s). Try to cut out as small a piece as possible while still getting all the DNA.
Weigh gel slice (tare with empty microtube). Add 3 volumes of ADB buffer per mg of gel (so a 100mg gel gets 300 uL of ADB buffer).
Incubate at 55C for 10 minutes. Make sure that the gel is completely dissolved.
Add dissolved gel solution to Zymo column in collection tube. Max volume is 800 uL at a time.
Spin 14000 rpm for 30 sec.
Discard liquid in collection tube.
Repeat step 5-7 if had more than 800 uL dissolve gel.
Add 200 uL DNA wash buffer.
Spin 14000 rpm 30 seconds.
Discard liquid in collection tube.
Add 200 uL DNA wash buffer
Spin 14000 rpm 1 min.
Discard liquid in collection tube.
Spin 14000 rpm 1 min one more time (dry spin).
Discard collection tube (but not the column).
(Optional: 2nd dry spin into clean collection tube.)
Place column in a clean labeled microtube.
Add 10 uL (min 6 uL for higher DNA concentration) of sterile DDH2O to top of column. Water should be pipetted directly onto center of filter.
Incubate at RT 1 min (or longer).
Spin 1 min at 14000 rpm. Discard the column.
Measure the concentration on the nanodrop. (You may recover the 1uL from the nanodrop if needed.)
Gel Extraction Protocol using QIAquick Gel Extraction Kit:
Cut the gel to separate analytical and extraction gel; place analytical gel in UV illuminator. 
Look at the gel under low wavelength UV (high wavelengths will denature DNA). Quickly take a polaroid image and shut OFF the UV. 
Cut extraction gel under white light; avoid UV illuminating the extraction gel as this drastically decreases the DNA yield. If necessary, stain with Methyl Blue. 
Place the cut bands in 2ml Eppendorf tubes; Weigh slices; No more than 400mg per tube
Add 3 volumes (6 volumes if you are afraid of getting a low yield) of Buffer QG to 1 volume of gel (100mg ~ 100ul)
Incubate at 50C for 10min or until gel is dissolved; vortex every 2-3 min
Confirm that color of mixture is yellow (if not, add 10ul of 3M NaAc, pH 5.0)
Add 1 gel volume of isopropanol
Add max of 770ul to QIAquick column and centrifuge for 1 min (max speed, ~13,000rpm, RT)
Run flow-through over column one more time.
After the second time, discard flow-through and place column back in tube.
If needed, add rest of mixture to same tube (up to additional 770ul), spin, and discard flow-through
Add 500uL of Buffer QG to column and centrifuge for 1 min (wash).
Wash: add 0.75ml Buffer PE (make sure that the buffer has ethanol added to it) to column.  Let stand for 2-5 minutes and then centrifuge for 1 min
Discard flow-through &amp; centrifuge for 1 min
Place column into clean Eppendorf tube
Add 50ul Buffer EB or water to center of membrane.  Make sure to use warm EB (50C).  (Use 30uL if worried about low concentration.)
Let stand at RT for 4 min
Centrifuge for 1 min
Measure the concentration using the UV spectrophotometer.
Pro Tips
You don't need 2 lanes if you aren't putting your gel under UV light (the blue light and SYBR safe is fine)
You can up the IPA to 1/4 of the total volume
Warm EB (50 mL conical filled w/ water, plop the tube inside, put it in the heat block)
Don't let it stand at room temperature, you can do it at 5 degrees (heat block)
Gel Extraction Protocol using QIAgen MinElute Kit:
Cut the gel to separate analytical and extraction gel; place analytical gel in UV illuminator. 
Look at the gel under low wavelength UV (high wavelengths will denature DNA). Quickly take a polaroid image and shut OFF the UV. 
Cut extraction gel under white light; avoid UV illuminating the extraction gel as this drastically decreases the DNA yield. If necessary, stain with Methyl Blue. 
Place the cut bands in 2ml Eppendorf tubes; Weigh slices; No more than 300mg per tube
Add 3 volumes of Buffer QG to 1 volume of gel (100mg ~ 100ul)
Incubate at 50C for 10min or until gel is dissolved; vortex every 2-3 min
Confirm that color of mixture is yellow (if not, add 10ul of 3M NaAc, pH 5.0)
Add 1 gel volume of isopropanol
Add max of 800ul to MinElute column and centrifuge for 1 min (speed &gt;= 10,000 G, RT)
Discard flow-through and place column back in tube.
If needed, add rest of mixture to same tube (up to additional 770ul), spin, and discard flow-through
Add 500 uL of buffer QG and spin column for 1 min and discard flow-through
Wash: add 0.75ml Buffer PE(make sure that the buffer has ethanol added to it) to column and centrifuge for 1 min
Discard flow-through &amp; centrifuge for 1 min
Place column into clean Eppendorf tube
Add 10ul Buffer EB (10 mM TrisCl,pH 8.5) or water to center of membrane
Let stand at RT for 1 min
Centrifuge for 1 min
Measure the concentration using the UV spectrophotometer.<H2>MIT iGEM cookie preparation</H2><A href="#top">return to top</A>
Reagents required:
1 1/2 cups (3 sticks) unsalted butter
1 cup sugar
2 large egg yolks
3 3/4 cups sifted all-purpose flour
1/4 teaspoon salt
1 tablespoon pure vanilla extract
Safety note: wear gloves, this procedure involves touching the dough with your hands, a lot.  Human tissue is minimum BSL 2 because it can contain contagious pathogens.
Make sure oven is set to 350°F
Melt 3 sticks butter
Add butter and 1C sugar to mixing bowl
mix sugar and butter thoroughly
Add 3.75c all purpose flour
Add .25 tsp salt
Add 1.33Csugar
Add 0.75c cocoa powder
Add 1Tbsp vanilla extract
Add 2 egg yolks
mix until consistent and cohesive, this will likely require kneading.
for all dough
place some amount of dough onto cookie sheet and flatten with palm of hand, should be &gt;3&quot; in diameter and ~.25&quot; thick and flat (no texture/bumps)
press outline cutter into dough and apply pressure around perimeter to ensure a good cut
--The dough should just make it to the angled part of the cookie cutter, if not, adjust height on next cookie
remove cookie cutter
removed excess dough
get more dough and repeat cookie blanking until cookie sheet is full
press stamp into each cookie by pushing on two opposite edges, examine result to see if enough pressure was applied.  A cookie can be stamped twice if you're careful.
bake cookies on sheet for 9 minutes
(at this point, running multiple parallel sheets is suggested)
after baking, use a spatula to move cookies onto drying rack
--This is the easiest step to break cookies at, they almost always break teeth off the gears.  Make sure that cookies are rotated and aligned such that all gear teeth are supported and they do not fall in between wires on rack.
Wait as long as possible for cookies to cool completely before transferring to a storage container as cookie structural integrity is inversely proportional to temperature.  Having a fan blow on cookies to help cooling is suggested.
<DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:MIT/Protocols">http://2014.igem.org/Team:MIT/Protocols</A>&quot;</DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:MIT/Protocols" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:MIT/Protocols" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:MIT/Protocols&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:MIT/Protocols&amp;oldid=400218" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></TABLE></TBODY></TABLE></DIV></DIV></DIV></BODY></HTML>