<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_UT-Tokyo_Counter_Project_Result"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:UT-Tokyo/Counter/Project/Result">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:UT-Tokyo/Counter/Project/Result&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:UT-Tokyo/Counter/Project/Result&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:UT-Tokyo/Counter/Project/Result&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:UT-Tokyo/Counter/Project/Result" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:UT-Tokyo/Counter/Project/Result</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><DIV id="Result-1"><P>&lt;img src=&quot;&quot; class = &quot;contTitle&quot; /&gt;</P><H3><SPAN class="mw-headline" id="Influence_on_growth_of_E._coli">Influence on growth of E. coli</SPAN></H3><P>We chose two sigma factors, ecf20 (BBa_K1461004) and ecf11 (BBa_K1461005), which have no influence on the growth of E. coli based on a previous research[1]. Then, we can expect that there will be no defference between the growth curve of the cells which express sigma factors and do not.</P><P>&lt;img src=&quot;&quot; class=&quot;figure&quot; /&gt;&lt;legend&gt;<B>Fig. 1</B> The growth curve of the cells which express sigma factors and do not. Arabinose was added 3 hours after the inoculation. &quot;None&quot; means pSB1C3 plasmid without insert sequence. Error bar shows average ± SE. N = 5. &lt;/legend&gt;</P><P>However, as seen in Figure 1, we found that ecf20 had repressive influence on growth of E. coli. This result, which did not meet that of the previous research, may be caused by the difference of strain of E.coli or the medium used in the measurement. We used MG1655 and M9 medium, while the previous researchers used DH10b and LB medium.</P><P>Although we found that ecf20 repressed the growth of E. coli, the influence may not be so large as to change general behaviour of our device. In the presence of ecf20, E. coli's growth rate became slower and the maximum concentration became lower a little, but not critically. </P><H3><SPAN class="mw-headline" id="Sigma_factor_and_its_corresponding_promoter">Sigma factor and its corresponding promoter</SPAN></H3><P>In order to test whether the sigma factors activate only their corresponding promoters, and whether the promoters was activated only by their corresponding sigma factors, we introduced various combinations of sigma factor (induced by arabinose) and promoter (reported by GFP) into E. coli.</P><P>&lt;img src=&quot;&quot; class=&quot;figure&quot; /&gt;&lt;img src=&quot;&quot; class=&quot;figure&quot; /&gt;&lt;legend&gt;<B>Fig. 2</B> GFP fluorescence reporting the activities of various combinations of sigma factors and promoters. GFP was activated by 488 nm excitation laser and the activity was measured by 514 nm emission laser.  Error bar shows average ± SE. N = 3. &lt;/legend&gt;</P><P>As seen in Figure 2, ecf20 and ecf11 activated only their corresponding promoters. Leak from the promoters was also not observed. The result of the sample whose sigma factor was not induced by arabinose (ara-), which had little difference from the one induced by arabinose (ara+), may be caused by the leak of pBAD (BBa_I13453). The amount of the sigma factor leaked from pBAD may be over the switch-point of sigmoid curve indicating relation between sigma factor and its corresponding promoter. Thus, even if the amount of sigma factor changes by arabinose induction, corresponding activity of promoter may not change.</P></DIV><DIV id="Result-2"><P>&lt;img src=&quot;&quot; class = &quot;contTitle&quot; /&gt;</P><P>We tested whether the anti-sigmas (induced by IPTG) repress the activation of the promoters by their corresponding sigma factors.</P><P>&lt;img src=&quot;&quot; class=&quot;figure&quot; /&gt;&lt;img src=&quot;&quot; class = &quot;figure&quot; /&gt;&lt;legend&gt;<B>Fig. 3</B> GFP fluorescence coded by the promoters activated by their corresponding sigma factors under the existence/abstance of their corresponding anti-sigma factors. GFP was activated by 488 nm excitation laser and the activity was measured by 514 nm emission laser. Error bar shows average ± SE. N = 3. &lt;/legend&gt;</P><P>Unexpectedly, the culture of cells containing the genes of the reset function showed no difference from that without reset function. This result may indicate that the sigma factors were expressed more than the anti-sigma factors could repress their activation of the corresponding promoters.</P></DIV><DIV id="Result-3"><P>&lt;img src=&quot;&quot; class=&quot;contTitle&quot; /&gt;</P><P>In order to test the riboregulator system, we constructed a genetic circuit containing constitutively transcribed crRNA coding GFP and taRNA (arabinose induced). We make the genetic circuit coding only crRNA, not coding taRNA, as control. We can expect the cells will express GFP by arabinose induction. </P><P>&lt;img src=&quot;&quot; class=&quot;figure&quot; /&gt;&lt;img src = &quot;&quot; class = &quot;figure&quot; /&gt;&lt;legend&gt;<B>Fig. 4</B> GFP fluorescence coded by constitutively transcribed crRNA under exitence/absence of taRNA. &quot;None&quot; means pSB1C3 with no insert sequence as negative control. GFP was activated by 501 nm excitation laser and the activity was measured by 514 nm emission laser. Error bar shows average ± SE. N = 5. &lt;/legend&gt;</P><P>The difference between the samples was not significant, which means the riboregulator system did not work. Several factors can be suggested for the reason why this device did not work, such as the incubation temperature, the amount of transcribed taRNA relative to that of crRNA. (Although we tried at various temperature, we could not see any fluorescence.)</P></DIV><DIV id="Result-4"><P>&lt;img src=&quot;&quot; class = &quot;contTitle&quot; /&gt;</P><P>To test whether the positive feedback of sigma factor can keep the sigma factors expressed after the arabinose induction was stopped, we added arabinose for some length of time to the cultures of cells containing/not containing positive feedback of sigma factors.</P><P>&lt;img src=&quot;&quot; class = &quot;figure&quot; /&gt;&lt;img src=&quot;&quot; class = &quot;figure&quot; /&gt;&lt;legend&gt;<B>Fig. 5</B> GFP fluorescence coded by the promoter and the corresponding sigma factor with/without positive feedback of the sigma factor. GFP was excited by 488 nm excitation laser and the activity was measured by 514 nm emission laser. N = 1. &lt;/legend&gt;</P><P>Unexpectedly, the genetic circuit with positive feedback showed no fluorescence. The sequence of this construct was not confirmed (since the degradation tag was added by PCR at the final step of DNA assembly and the primers for sequencing had not been prepared before deadline), and we thought some mutations made this result.</P></DIV><DIV id="Result-5"><P>&lt;img src=&quot;&quot; class = &quot;contTitle&quot; /&gt;</P><P>From figure1, we suggest that even so low concentration of sigma factors as the leak from pBAD can activate the corresponding promoters. For the realization of our device, change of the promoter coding sigma factor or the backbone whose parameters are appropriate may be necessary. The strain of the host cell is also thought to be one of the important factors. These selections are the future works.</P><H3><SPAN class="mw-headline" id="Reference">Reference</SPAN></H3><P>[1]]Rhodius, Virgil A., et al. &quot;Design of orthogonal genetic switches based on a crosstalk map of σs, anti‐σs, and promoters.&quot; Molecular systems biology 9.1 (2013).</P></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:UT-Tokyo/Counter/Project/Result">http://2014.igem.org/Team:UT-Tokyo/Counter/Project/Result</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:UT-Tokyo/Counter/Project/Result" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:UT-Tokyo/Counter/Project/Result" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:UT-Tokyo/Counter/Project/Result&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:UT-Tokyo/Counter/Project/Result&amp;oldid=379724" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>