<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_UT-Tokyo_Counter_Project_Modeling"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:UT-Tokyo/Counter/Project/Modeling">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:UT-Tokyo/Counter/Project/Modeling&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:UT-Tokyo/Counter/Project/Modeling&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:UT-Tokyo/Counter/Project/Modeling&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:UT-Tokyo/Counter/Project/Modeling" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:UT-Tokyo/Counter/Project/Modeling</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><DIV id="Modeling-1"><P>&lt;img src = &quot;&quot; class = &quot;contTitle&quot; /&gt;</P><P>Modeling is an attempt to describe, in a precise way, an understanding of the elements of a system of interest, their states, and their interactions with other elements.</P><P>The purpose of our modeling team is to peel back the layer of appearance of the device to reveal it's underlying nature. We tried to improve the device, cooperating with the experiment team. To achieve our goal, we have developed three fundamental themes. These three themes divide the modeling part into three parts. At the beginning, we confirmed whether our circuit realizes a reaction:this for part 1. When it comes to extending the counter to a device which can count a lot of numbers, there is a possibility that the device skips the next state due to the leak expression of sigma factor. So we examined what influences the leak of sigma factor have on the state-transit device:this for part 2. Thirdly, as an implementation of our genetic circuit, we designed new construct freeing the degree of induce-time and checked how the circuit works:this for part 3. Finally, we discussed what would be appropriate modeling, frequent issue to attack, in order to find the best strategy of modeling and wrote how we constructed our model:this Guide for modeling.</P><P>In Part1(deterministic model, stochastic model), we considered whether the device outputs much higher fluorescence when 2nd-induced than that of 1st-induced. In addition, we checked the key function of our device, reset function. Two approaches was made.</P><P>・Deterministic model:In this model,intermolecular interactions such as DNA-protein interactions and protein-ligand interactions are described as differential equations and concentrations of product (multimolecular complex) can be calculated by those of reactants. This model is intuitive, simple and hence popular to estimate the result of experiment.</P><P>・Stochastic model:Due to a cell's small volume (down to a few femtoliters for bacterial cells) and typically low bimolecular concentrations, the absolute number of molecules of a given signaling species in a cell may be quite small (typically on the order of 100 to 103). At such low numbers of reacting molecules, the reactions are heavily influenced by chance events, such as the chance encounter of reactant molecules. So we also used the stochastic model. The most common formulation of stochastic models for biochemical networks is the chemical master equation (CME). We used Gillespie algorithm to solve CME. Other method to solve CME is mentioned in Part4.</P><P>In Part2(Leaky expression analysis), taking the influences of the leak of sigma factor into consideration, we designed model which can change the induce time. In the end, we concluded that the leak of sigma-factor can be ignored when the induce time is much longer than the time of degradation of sigma-factor.</P><P>In Part3(Improvement of counter), the current sigma-Re-counter depends much on pulse length, so we devised genetic circuits that would not be affected by the pulse length of the arabinose induction. We modeled this construct to test if it can be realized.</P></DIV><DIV id="Modeling-2"><P>&lt;img src = &quot;&quot; class = &quot;contTitle&quot; /&gt;</P><P>First construction we came up with about the counter is described as following.</P><P>&lt;img src = &quot;&quot; class = &quot;figure&quot; /&gt;</P><P>These generators have following functions:</P><P>・production of cis-repressed mRNA of a sigma factor by constitutive promoter.</P><P>・production of taRNA by the 1st arabinose induction and activating of cis-repressed mRNA by taRNA.</P><P>・positive feedback loops formed by sigma factors.</P><P>・production of cis-repressed mRNA of GFP by sigma factors.</P><P>・activating mRNA of GFP by the 2nd arabinose induction.</P><P>・degradation of sigma factors by induction of IPTG.</P><P>The counter consists of these functions. Our first work is confirming theoretically that these functions cooperate with each other, GFP expresses significantly only by 2nd arabinose induction and the counter is reset by IPTG.</P><P>In this simulation, we used two types of models, the deterministic model and the stochastic model. First, we simulated the counter by the deterministic model, which is popular and intuitive. Second, we simulated it by the stochastic model, which takes paucity and fluctuation of reactants into account.</P><P>Formulation of models and results of simulations are described below.</P><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1"><A href="#Deterministic_Model"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Deterministic Model</SPAN></A></LI><LI class="toclevel-2"><A href="#Formulation_of_the_Model"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Formulation of the Model</SPAN></A></LI><LI class="toclevel-2"><A href="#Parameter"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Parameter</SPAN></A></LI><LI class="toclevel-2"><A href="#Simulations"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Simulations</SPAN></A></LI><LI class="toclevel-2"><A href="#Sensitivity_analysis"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">Sensitivity analysis</SPAN></A></LI></UL><LI class="toclevel-1"><A href="#Stochastic_Model"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Stochastic Model</SPAN></A></LI><LI class="toclevel-2"><A href="#Formulation_of_the_Model_2"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Formulation of the Model</SPAN></A></LI><LI class="toclevel-2"><A href="#Parameter_2"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">Parameter</SPAN></A></LI><LI class="toclevel-2"><A href="#Simulations_2"><SPAN class="tocnumber">2.3</SPAN><SPAN class="toctext">Simulations</SPAN></A></LI><LI class="toclevel-1"><A href="#What_is_modeling"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">What is modeling</SPAN></A></LI><LI class="toclevel-1"><A href="#Aims_of_modeling"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Aims of modeling</SPAN></A></LI><LI class="toclevel-1"><A href="#Stochastic_Approaches"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Stochastic Approaches</SPAN></A></LI><LI class="toclevel-1"><A href="#Stochastic_Formulation_and_Markov_Process"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">Stochastic Formulation and Markov Process</SPAN></A></LI><LI class="toclevel-1"><A href="#References"><SPAN class="tocnumber">7</SPAN><SPAN class="toctext">References</SPAN></A></LI></TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Deterministic_Model">Deterministic Model</SPAN></H3><H4><SPAN class="mw-headline" id="Formulation_of_the_Model">Formulation of the Model</SPAN></H4><P>First of all, we constructed the deterministic model to estimate the behavior of the counter. In this model, biochemical reactions are described as differential equations and concentration of reaction product can be calculated by those of reactants. This model is intuitive, simple and hence popular to estimate the result of experiment. We were able to get some parameters for modeling of counter from previous works.[→ see parameter section]</P><P>We had simplified the construction of the mathematical model before described time evolution in which concentrations of mRNAs and proteins change as differential equations. First, we regarded that the reaction between taRNA(trans-activating RNA) and crRNA(cis-repressor RNA) in riboregulator is much faster than that of transcription or translation and equilibrium reaction. This diminution of parameters enable us to use the equilibrium constant as a parameter and prevent us from overfitting when we adapt this model to raw data.</P><P>equation (1):</P><P>&lt;img src = &quot;&quot;  height=&quot;50px&quot; class = &quot;math&quot; /&gt;</P><P>So, the concentrations of mRNAs and the coupling of taRNA and crRNA are represented as stated above
. Subscript means the gene encoded by the mRNA. We regarded that the affinities between crRNA and taRNA on different mRNAs were equal. The dissociation constant of equilibrium reaction was therefore shown as following.</P><P>equation (2):</P><P>&lt;img src = &quot;&quot; height=&quot;100px&quot; class = &quot;math&quot; /&gt;</P><P>Using dissociation constant, concentrations of reaction products such as [mcr<SUB>cr-σ</SUB>] could be described as function of those of taRNA and mRNA of σ and GFP. We put X, A and B as the total quantity of taRNA, sigma and GFP, respectively.</P><P>equation (3):</P><P>&lt;img src = &quot;&quot; height=&quot;20px&quot; class = &quot;math&quot; /&gt;</P><P>equation (4):</P><P>&lt;img src = &quot;&quot; height=&quot;20px&quot; class = &quot;math&quot; /&gt;</P><P>equation (5):</P><P>&lt;img src = &quot;&quot; height=&quot;20px&quot; class = &quot;math&quot; /&gt;</P><P>Using these equations((3)-(5)) and equilibrium constant, concentrations of free taRNA, free mRNAs and taRNA-mRNA complexes are described as following. These are all of simplifications.</P><P>equation (6):</P><P>&lt;img src = &quot;&quot; height=&quot;45px&quot; class = &quot;math&quot; /&gt;</P><P>equation (7):</P><P>&lt;img src = &quot;&quot; height=&quot;45px&quot; class = &quot;math&quot; /&gt;</P><P>equation (8):</P><P>&lt;img src = &quot;&quot; height=&quot;45px&quot; class = &quot;math&quot; /&gt;</P><P>equation (9):</P><P>&lt;img src = &quot;&quot; height=45px&quot; class = &quot;math&quot; /&gt;</P><P>equation (10):</P><P>&lt;img src = &quot;&quot; height=&quot;44px&quot; class = &quot;math&quot; /&gt;</P><P>Finally, we built up differential equations about concentrations of reaction products including mRNA of sigma which has no riboregulator. (It makes positive feedback loop.) This model assumed relationship between promoter and the amount of transcriptional product increasing per unit time. The amount is in proportion to the number of promoter if the promoter expressed constitutively and can be approximated by Hill equation if the inducer controlled its promoter. It was also assumed that the degradation of proteins and RNAs followed first order kinetics. Some of used parameters were cited from references.[1]~[9]</P><P>We aimed to determine parameters about sigma through experiment and used provisional parameter determined in reference to other promotor.</P><P>equation (11):</P><P>&lt;img src = &quot;&quot; height=&quot;76px&quot; class = &quot;math&quot; /&gt;</P><P>equation (12):</P><P>&lt;img src = &quot;&quot; height=&quot;43px&quot; class = &quot;math&quot; /&gt;</P><P>equation (13):</P><P>&lt;img src = &quot;&quot; height=&quot;45px&quot; class = &quot;math&quot; /&gt;</P><P>The transcriptional activity of mRNA encoding sigma factor initiated by positive feedback loop and 
that of mRNA encoding anti-sigma induced by IPTG are described as following.</P><P>equation (14):</P><P>&lt;img src = &quot;&quot; height=&quot;46px&quot; class = &quot;math&quot; /&gt;</P><P>equation (15):</P><P>&lt;img src = &quot;&quot; height=&quot;43px&quot; class = &quot;math&quot; /&gt;</P><P>In our project, IPTG induction was aimed at enough production of anti-sigma to reset the counter and the sensitivity of <I>P<SUB>lac</SUB></I>(lactose promoter) was not our main interest. Therefore, we used simple equation,(15) to describe how lactose promoter behave. <I>P<SUB>lac</SUB></I> depend on the concentration of IPTG but we regarded it as a fixed number in this modeling.</P><P>Taking into account that translation coincide with transcription in prokaryotes, the linearity between the concentration of transcriptional product and the change rate of the amount of translational product was assumed. this linearity does not depend on the kind of translational product. We also assumed that anti-sigma combine with sigma and form inert matter irreversibly, and the reaction velocity of that is proportional to product of these.</P><P>equation (16):</P><P>&lt;img src = &quot;&quot; height=&quot;44px&quot; class = &quot;math&quot; /&gt;</P><P>equation (17):</P><P>&lt;img src = &quot;&quot; height=&quot;44px&quot; class = &quot;math&quot; /&gt;</P><P>equation (18):</P><P>&lt;img src = &quot;&quot; height=&quot;44px&quot; class = &quot;math&quot; /&gt;</P><P>Using above-mentioned differential equations, we simulated behavior of the counter by Euler's method.</P><H4><SPAN class="mw-headline" id="Parameter">Parameter</SPAN></H4><P>Here, we explain how we determined the parameters of the deterministic model. PoPS (polymerase per second) of <I>P<SUB>Const</SUB></I> is 0.03[3], so its promoter activity is 0.03/(6.0× 10<SUP>23</SUP>× 1.0× 10<SUP>-15</SUP>)[M/sec] = 0.051[nM/sec]. The switch point and the hill coefficient of <I>P<SUB>BAD</SUB></I> are written in [4]. PoPS of <I>P<SUB>BAD</SUB></I> is 5/60[5] , so its RPU (relative promoter unit) is (5/60)/(0.03) = 2.78. We set the RPU of <I>P<SUB>lac</SUB></I> as 2 when IPTG is induced. We don't consider the leak expression from <I>P<SUB>lac</SUB></I>.</P><P>The average half life of mRNA is 2-5 min[1], so we set the degradation rate of mRNA as 0.010[/sec]. The half life of GFP is infinite[9], so we set the degradation of GFP as 0.0[/sec]. The degradation rate of sigma factor[20] is fast. We set it as 0.001[/sec]. The degradation rate of anti-sigma is unknown, We set it as 6.0× 10<SUP>-6</SUP>, the average degradation rate of protein[11]. The equilibrium constant of the equations (1)is 80.0[nM][2]. The reaction rate of the association of sigma and anti-sigma is unknown. We assumed this reaction is fast and we set it as 10.0[/M sec]. The copy number of the plasmid is 100~300[7][8] , so we set as 200. When the number of lacZ mRNA is 62, the protein synthesis rate is 20[/sec][11], and we set the translational rate as 20/62 = 0.32, assuming it does not change from the kind of mRNA.</P><P>The summary of the parameters of this model is given in Table 1.</P><P>&lt;img src = &quot;&quot; class = &quot;figure&quot; /&gt;&lt;legend&gt;<B>Table 1</B>&lt;/legend&gt;</P><H4><SPAN class="mw-headline" id="Simulations">Simulations</SPAN></H4><P>&lt;img src = &quot;&quot; class = &quot;math&quot; /&gt;&lt;legend&gt;<B>Fig.1</B>&lt;/legend&gt;</P><P>arabinose induction time: 5000-5120[sec], 15000-15120[sec], IPTG induction time: t &gt; 22500[sec]</P><P>The unit of vertical axis is [nM], and that of the horizontal axis is [sec]. We can see GFP expression only after 2nd-induction and rapid degradation of sigma after IPTG induction. These mean that the counter shows collect behavior theoretically.</P><H4><SPAN class="mw-headline" id="Sensitivity_analysis">Sensitivity analysis</SPAN></H4><P>Sensitivity analysis is an examination of output value changing when a certain parameter changes.</P><P>By conducting sensitivity analysis, we could know how each parameter affects the system. Results of some specific parameters are shown below. The parameters we changed are shown in Horizontal axis. The vertical axis is the ratio of GFP that is expressed in the first induction to GFP expressed over the entire experiment. When we changed one parameter, the other parameters were fixed.</P><P>&lt;img src = &quot;&quot; height=&quot;200px&quot; class = &quot;math&quot; /&gt;&lt;legend&gt;<B>Fig.2</B> Horizontal axis : pulse length of the arabinose induction&lt;/legend&gt;&lt;img src = &quot;&quot; height=&quot;200px&quot; class = &quot;math&quot; /&gt;&lt;legend&gt;<B>Fig.3</B> Horizontal axis : K<SUB>σ</SUB>&lt;/legend&gt;&lt;img src = &quot;&quot; height=&quot;200px&quot; class = &quot;math&quot; /&gt;&lt;legend&gt;<B>Fig.4</B> Horizontal axis :V<SUB>σ</SUB>&lt;/legend&gt;</P><P>Comparing Fig.2 with Fig.3,  K<SUB>σ</SUB> seems to have less influence on the system than the pulse length of the arabinose induction. From Fig.4, V<SUB>σ</SUB> shouldn't be so large or so small to minimize the leak_GFP/GFP.</P><H3><SPAN class="mw-headline" id="Stochastic_Model">Stochastic Model</SPAN></H3><H4><SPAN class="mw-headline" id="Formulation_of_the_Model_2">Formulation of the Model</SPAN></H4><P>If there are a lot of molecules, modeling usually uses ordinary differential equations, but some in vivo reactions involve only a few molecules. For example, transcription involves the cell's genomic DNA which is one copy or plasmids which are about 200 copies [7][8] in a cell of <I> Escherichia coli</I>. The average size of a cell of <I>E. coli</I> is about 1.0× 10<SUP>-15</SUP>[L][15], so the concentration of DNA is about 1.7[nM] and the concentration of plasmids is about 200 times of it. This is obviously weak. Reactions like this are well affected by fluctuations due to the reactants's limited copy numbers. So, we need to take this fluctuations into our modeling which is derived from stochastic methods. We also introduce delay effect.</P><P>First we explain about the Gillespie algorithm which is often used in stochastic simulations. In the Gillespie algorithm, we treated not the concentration of molecules but the number of them. Reactions are also viewed as discrete, essentially instantaneous physical events. What we have to determine when using the Gillespie algorithm is (A) when the next reaction is going to occur and (B) which type of the reaction it will be. Looking more closely at the Gillespie algorithm by the next set of reaction formulas:</P><P>equation (19):</P><P>&lt;img src = &quot;&quot; height=&quot;48px&quot; class = &quot;math&quot; /&gt;</P><P>Let n<SUB>1</SUB>, n<SUB>2</SUB>, and n<SUB>3</SUB> denote the respective copy number of the components X<SUB>1</SUB>, X<SUB>2</SUB>, and X<SUB>3</SUB>. Notice that they are all integer. First we have to determine how easily each reactions could happen. It depends on the number of components copied. In stochastic simulations, we often determine the parameter called stochastic rate constant, which is often written as &quot;c<I>. We assume that each possible combinations of reactant molecules have the same probability c per unit time to react. In other words, c× dt gives the probability that a particular combination of reactant molecules will react in a short time interval [t,t+dt). We call the stochastic rate constant of a reaction j, c<SUB>j</SUB>. Considering the all combinations of reactant molecules, the probability that the reaction 0 occur in [t,t+dt) is c× n<SUB>1</SUB>× n<SUB>2</SUB>. We now define the propensity function as the function of which product with dt gives the probability that a particular reaction will occur in the next infinitesimal time dt, which is often written as &quot;a</I>. Later on, the propensity function of a reaction j is a<SUB>j</SUB>. Following the equation:</P><P>equation (20):</P><P>&lt;img src = &quot;&quot; height=&quot;15px&quot; class = &quot;math&quot; /&gt;</P><P>Notice that c<SUB>j</SUB> is invariant parameter, but a<SUB>j</SUB> changes as the state changes. In the same way, a<SUB>1</SUB> = c<SUB>1</SUB>× n<SUB>3</SUB>.</P><P>First we answer the question (A) when is the next reaction going to occur? Now, to simplify the situation we assume the situation that only the reaction 0 occurs. Set the time as 0, and define P(t) as the probability that the reaction 0 doesn't occur in [0,t). Then from the definition of a,we obtain the equation; P(t+dt) = P(t)(1-a× dt). (Because the probability that the reaction 0 doesn't occur in [0,t+dt) is the product of the probability that the reaction 0 doesn't occur in [0,t) with the probability that the reaction 0 doesn't occur in [t,t+dt).) Using P(t+dt) = P(t) + dP(t)/dt, we get :</P><P>equation (21):</P><P>&lt;img src = &quot;&quot; height=&quot;50px&quot; class = &quot;math&quot; /&gt;</P><P>Because the probability that the reaction0 doesn't occur in a 0 second interval is zero; P(0)=1. Solving the above ordinary differential equation we get :</P><P>equation (22):</P><P>&lt;img src = &quot;&quot; height=&quot;24px&quot; class = &quot;math&quot; /&gt;</P><P> If r<SUB>1</SUB> is a uniform number from [0,1], the time of the next reaction should be determined by solving P(t) = r<SUB>1</SUB>. Using (22), we get t =  -a<SUB>0</SUB>/log r<SUB>1</SUB>.</P><P>Now we suppose there is N types of reactions. Let a<SUB>1</SUB>,a<SUB>2</SUB>,…,a<SUB>N</SUB> denote the respective propensity function of reaction 1,2,…,N. From previous method;</P><P>equation (23):</P><P>&lt;img src = &quot;&quot; height=&quot;70px&quot; class = &quot;math&quot; /&gt;</P><P>Let dt be so small that we can ignore the term of higher than two orders of dt. The equation(23) becomes:</P><P>equation (24):</P><P>&lt;img src = &quot;&quot; height=&quot;70px&quot; class = &quot;math&quot; /&gt;</P><P>Solving (4) (a = Σ_{j=1}^{N} a<SUB>j</SUB>)</P><P>equation (25):</P><P>&lt;img src = &quot;&quot; height=&quot;23px&quot; class = &quot;math&quot; /&gt;</P><P>Setting τ as the time of the next reaction, we get:</P><P>equation (26):</P><P>&lt;img src = &quot;&quot; height=&quot;45px&quot; class = &quot;math&quot; /&gt;</P><P>Second we answer the question (B) what types of the reaction will it be? We determined the time of the next reaction, so what we have left to do is to determine what kind of reaction occurs. Some people may feel queer, but in the Gillespie algorithm, first the time of next reaction will be determined, and second the kind of reaction will be determined. It is natural to determine that the probability that the reaction j occurs is a<SUB>j</SUB>/a. If r<SUB>2</SUB> is a uniform number from [0,1], j is the only number that meets below in equations:</P><P>equation (27):</P><P>&lt;img src = &quot;&quot; height=&quot;45px&quot; class = &quot;math&quot; /&gt;</P><P>In the case a<SUB>0</SUB> ≧ a× r<SUB>2</SUB>, the reaction that occurred is reaction 0.</P><P>Now we can run the Gillespie algorithm by following the next steps.(t<SUB>MAX</SUB> is the finish time of the simulation.)1.Initialize the system at t = 0 with initial numbers of molecules for each spices, n<SUB>0</SUB>,… ,n<SUB>s</SUB>2.For each j = 0,1,…,r, calculate a<SUB>j</SUB>(n) based on the current state n using (20)3.Calculate the exit rate a(n) = Σ_{j=0}^{r} a&lt;/sub&gt;j&lt;/sub&gt;(n).4.Compute a sample tau of the time until the next time using (26)5.Update the time t = t +τ6.Compute a sample j of the reaction index using (27)7.Update the state n according to the reaction j.8.If t &lt; t<SUB>MAX</SUB>, return to Step 2</P><P>Stochastic rate constant can be determined by the parameters we used in the deterministic model (if we modeled the reaction in the deterministic model) . If there are a lot of reactant molecules, stochastic simulations have to show similar results as those of deterministic simulations. For this reason, stochastic rate constant, c, can be calculated from the chemical reaction rate constant, k. See [10] if you want to know the deriving process. Here we just write the result.</P><P>For a unimolecular reaction, c numerically equals to k, whereas for a bimolecular reaction, c equals to k/N<SUB>A</SUB>V if the species of the reactant molecules are different, or 2k/N<SUB>A</SUB>V  if they are the same.V is the volume of the system and N<SUB>A</SUB> is the Avogadro's constant.</P><P>However, these results should not be taken to imply that the mathematical forms of the propensity functions are just heuristic extrapolations. The propensity functions are grounded in molecular physics, and the formulas of deterministic chemical kinetics are approximate consequences of the formulas of stochastic chemical kinetics, not the other way around.</P><P>The Gillespie algorithm is so clear and useful that it is often used. However, this algorithm is not suitable for describing transcriptions and translations because they are very slow and complex reactions involving many kinds of reactant molecules. If we treat transcription from plasmids as one reaction, assuming the copy number of plasmids as 200, then the propensity function a equals to the stochastic rate constant multiplied by 200 (200× c). So it will take about one of a two hundred times of an average transcription time to finish one transcription. Of course, in the time scale of average transcription time it is not a big problem, but this may not be good for simulating, like in our project, the system that uses the time for transcriptions and translations cannot be shortened. We introduce time-delay into the Gillespie algorithm based on [12]~[14]. The mathematical correctness of this algorithm is proved in [14]. Time-delay means treating reactions as following:</P><P>&lt;img src = &quot;&quot; height=&quot;48px&quot; class = &quot;math&quot; /&gt;</P><P>Furthermore, transcriptions and translations are too complex to list up all of the reactions step by step. Therefore it is better to treat them as time-delay than reaction formulas.</P><P>Now we begin to model our project, sigma Re-counter. In our model, there are five types of reactions: transcription, translation, an association and disassociation of crRNA and taRNA, an association and disassociation of sigma and anti-sigma, and degradation. We introduce time-delay into only transcription and translation. Then, we explain how we treat these three reactions in general.</P><P>We explain transcription's model of mRNA[11]. When RNA polymerase binds to the promoter region, first they take the RNAP・promoter close complex. At this state, the complex can dissociate. But with a certain probability, the close complex turn to the open complex which doesn't dissociate. After the RNA polymerase and the promoter region take the open complex, the elongation of mRNA starts. In this state, the promoter region is cleared and RBS(Ribosome binding site) is synthesized, so we model as reaction3'.  It is difficult to model the elongation of mRNA by elementary reaction formula because it is a complex reaction, so we model by time-delay. Then the reaction formula of transcription can be described as following's reactions:</P><P>&lt;img src = &quot;&quot; height=&quot;137px&quot; class = &quot;math&quot; /&gt;</P><P>combining reaction3' and reaction3<I>, we get:</I></P><P>&lt;img src = &quot;&quot; height=&quot;20px&quot; class = &quot;math&quot; /&gt;</P><P>In the above reaction formula, DNA denotes the promoter region, and mRNA denotes RBS.</P><P>In our project, we have to model transcription of taRNA. Because taRNA functions only when it is completely transcribed, we model as following reactions:</P><P>&lt;img src = &quot;&quot; height=&quot;107px&quot; class = &quot;math&quot; /&gt;</P><P>We refer to the translational model [8]. Similarly to the transcriptional model we model as following;</P><P>&lt;img src = &quot;&quot; height=&quot;106px&quot; class = &quot;math&quot; /&gt;</P><P>combining reaction2' and reaction2<I>, we get:</I></P><P>&lt;img src = &quot;&quot; height=&quot;20px&quot; class = &quot;math&quot; /&gt;</P><P>We model the association and disassociation of crRNA and taRNA as a reversible reaction:</P><P>&lt;img src = &quot;&quot; height=&quot;45px&quot; class = &quot;math&quot; /&gt;</P><P>We know that anti-sigma directly affect sigma[20], but a large part of their relation is still unknown. From their direct relation, We model as following reactions:</P><P>&lt;img src = &quot;&quot; height=&quot;45px&quot; class = &quot;math&quot; /&gt;</P><P>We model the degradation as the following reaction:</P><P>&lt;img src = &quot;&quot; height=&quot;18px&quot; class = &quot;math&quot; /&gt;</P><P>We can conclude that reaction formulas of our model are as follows:</P><P>&lt;img src = &quot;&quot; class = &quot;figure&quot; /&gt;</P><H4><SPAN class="mw-headline" id="Parameter_2">Parameter</SPAN></H4><P> We got the parameters mainly from [11]. The summary of the parameters of this model is given in the Table 2 (α = 1/N<SUB>A</SUB> V, N<SUB>A</SUB> is  Avogadro constant, and V is the volume of <I>E. coli</I>.):</P><P>&lt;img src = &quot;&quot; class = &quot;figure&quot; /&gt;&lt;legend&gt;<B>Table 2</B>&lt;/legend&gt;</P><P> We estimated τ from the fact that average translational speed is 20 aa/sec, and average transcriptional speed is 40 bp/sec.[18][19] τ<SUB>1</SUB> = 40.0,τ<SUB>2</SUB> = 40.0,τ<SUB>3</SUB> = 40.0, τ<SUB>4</SUB> = 40.0,τ<SUB>5</SUB> = 40.0,τ<SUB>6</SUB> = 40.0, τ<SUB>7</SUB> = 50.0,τ<SUB>8</SUB> = 40.0,τ<SUB>9</SUB> = 40.0.</P><H4><SPAN class="mw-headline" id="Simulations_2">Simulations</SPAN></H4><P>Before executing every step of the Gillespie algorithm, we set the number of RNAP molecules and ribosomes as 35 and 350[11].</P><P>&lt;img src = &quot;&quot; height=&quot;200px&quot; class = &quot;math&quot; /&gt;&lt;legend&gt;<B>Fig.5</B>&lt;/legend&gt;&lt;img src = &quot;&quot; height=&quot;200px&quot; class = &quot;math&quot; /&gt;&lt;legend&gt;<B>Fig.6</B>&lt;/legend&gt;</P><P>Expression of GFP and sigma when arabinose is added at  t = 5000-5120, 15000-15120. Figure 5 is the time series of GFP, and figure 6 is the time series of sigma. From  figure 6, we can confirm that the system is reset.</P></DIV><DIV id="Modeling-3"><P>&lt;img src = &quot;&quot; class = &quot;contTitle&quot; /&gt;</P><P>As written in the Project part, we can imagine an expansion of the sigma-recounter, which can count a lot of numbers. One of the most important problems arising from the expansion is that a single induction may inadvertently increase the count number 2 or more times. We will explain this by considering the 3 counter. Below is the construct of the 3 counter. </P><P>&lt;img src = &quot;&quot; class = &quot;figure&quot; /&gt;</P><P>When arabinose is induced for the first time, sigma A will be expressed and its positive feedback loop will be put in motion. On the second induction, sigma B will be expressed, and third, GFP will be expressed. </P><P>There is a problem that occurs in the first induction. Not only sigma A but also a little amount of sigma B will be expressed in the first induction, as a portion of sigma A precociously stimulates the next step in the counting sequence. The sigma B positive feedback loop circuit exemplifies that little expression. Because of these two characteristics, sigma B has possibility of being highly expressed before the second induction; in other words, it counts 2. We did not have to model this in our main construct, because GFP does not have its positive feedback loop circuit.</P><P>Therefore we modeled if the counting can accurately be conducted. We also analyzed parameter sensitivity on the pulse length of arabinose induction.</P><P>&lt;img src = &quot;&quot;  class = &quot;math&quot; /&gt;&lt;legend&gt;<B>Fig.7</B> The model of 3 counter. Expression of sigmaA, sigmaB, GFP when arabinose is added at t= 10000/7, 30000/7, 50000/7. The pulse length is t = 60.)The unit of vertical axis is [nM], and that of the horizontal axis is [sec].&lt;/legend&gt;</P><P>Figure 7 shows that the first count can precisely be conducted even if a small amount of sigma B is expressed.</P><P>&lt;img src = &quot;&quot; class = &quot;math&quot; /&gt;&lt;legend&gt;<B>Fig.8</B> Parameter sensitivity on the pulse length of arabinose induction. Vertical axis is the ratio of GFP that is expressed in the second induction to GFP that is expressed over the entire experiment. Horizontal axis is the pulse length of arabinose induction.&lt;/legend&gt;</P><P>Figure 8 shows how strong the leaky expression is during the second induction event. There seems robust relation between the pulse length and whether it succeeded in counting. There seems to be three patterns depending on the pulse length. This simulation result can be explained by assuming that the positive feedback loop of sigma is put into motion when the concentration of sigma exceeds a threshold. When the pulse length of arabinose is too short and sigma A is expressed less than the threshold value, the sigma A concentration will head to zero after the induction. Therefore, in any induction event, the count number remains unchanged, and very few GFP is expressed. This is the t = 1 ~ 42 pattern in Figure 8. When the pulse length is correct, sigma A is expressed more than the threshold value of sigma A while sigma B was expressed less than the threshold value of sigma B. This time, the count is accurately conducted and GFP is expressed significantly in the third induction. This is the t = 43~117 pattern of Figure 8.When the pulse length is too long, sigma B is expressed more than the threshold value. This time, the first induction made 2 counts, which is the situation we were concerned about.</P><P>Thus, the pulse length plays an important role in 3 counter.</P><P>In fact, this behavior can be explained by an easy dynamical system analysis. 
</P></DIV><DIV id="Modeling-4"><P>&lt;img src = &quot;&quot; class = &quot;contTitle&quot; /&gt;</P><P>As we have seen in the previous sections, current sigma-recounter depends much on the pulse length of the arabinose induction; when the pulse length is too long, it would count 2 or more (if there is). We will show below the 2 counter's figure that the pulse length is hypothesized to be too long.</P><P>&lt;img src = &quot;&quot;  class = &quot;math&quot; /&gt;&lt;legend&gt;<B>Fig.9</B>The model of 2 counter when the pulse length is long. Unit of vertical axis is [nM], and that of horizontal axis is [sec]. induction time: 20000-40000[sec], 60000-80000[sec]&lt;/legend&gt;</P><P>However, with a little improvement, our counter would not count more than 1 by a single pulse, as long as the pulse length is long enough for it to count. You want to make a construct that is not affected by the pulse length. Shown below is such an improved construct.</P><P>&lt;img src = &quot;&quot; class = &quot;figure&quot; /&gt;</P><P>X and Y are substances that both are required to activate <I>P<SUB>X&amp;Y</SUB></I> promoter.</P><P>Before arabinose induction, <I>P<SUB>Tet</SUB></I> and <I>P<SUB>Const</SUB></I> express  Y and crRBS-sigma. After arabinose induction, <I>P<SUB>BAD</SUB></I> becomes activated and TetR and X are expressed. X binds to Y and the transcription of taRNA from <I>P<SUB>X&amp;Y</SUB></I> occur, which leads to counting. At that time, expression of Y is repressed by TetR and the amount of Y, which has short half life, decreases rapidly. Thus, <I>P<SUB>X&amp;Y</SUB></I> is again repressed, the amount of taRNA decreases, and the counter never counts more than 1. You might be afraid that <I>P<SUB>X&amp;Y</SUB></I> also begins transcription of taRNA when the induction ends; however, supposed degradation of X is faster than that of TetR, it will not occur. When the induction ends, X first degrades while still a lot of TetR remain and Y is not abundant. Since <I>P<SUB>Tet</SUB></I> has a sigmoidal transcriptional response, the production rate of Y will change little even if the concentration of TetR decrease a little. When TetR degrades so much that it finishes repression of Y, most of X have already decomposed, and <I>P<SUB>X&amp;Y</SUB></I> will not be activated to begin transcription of taRNA.</P><P>We modeled this construct to test if it can be realized. We did not model resetting this time, either.</P><P>&lt;img src = &quot;&quot; class = &quot;math&quot; /&gt;</P>&lt;legend&gt;<B>Fig.10</B> The improved 2 counter. The induction was modeled to be conducted just the same as the non-improved version. Units of vertical axis and horizontal axis are the same, too.&lt;/legend&gt;<P> Although the pulse length is long, counts are accurately conducted. Thus, theoretically, the counter works independently of the pulse length is suggested to be available. Only thing we have to do is to research for the substances that satisfy these conditions!</P></DIV><DIV id="Modeling-5"><P>&lt;img src = &quot;&quot; class = &quot;contTitle&quot; /&gt;</P><H3><SPAN class="mw-headline" id="What_is_modeling">What is modeling</SPAN></H3><P>The model should be sufficiently detailed and precise so that it can in principle be used to simulate the behavior of the system on a computer. </P><P>In the context of molecular cell biology, a model may describe the mechanisms involved in transcription, translation, gene regulation, cellular signaling, DNA damage and repair processes, homeostatic processes, the cell cycle, or apoptosis. Indeed any biochemical mechanism of interest can, in principle, be modeled. At a higher level, modeling may be used to describe the functioning of a tissue, organ, or even an entire organism. At still higher levels, models can be used to describe the behavior and time evolution of populations of individual organisms.</P><P>The first issue to confront when embarking on a modeling project is to describe on exactly which features to include in the model, and in particular, the level of detail model is intended to capture. Interacting with other cells and/or its environment, the cell realizes four key functions: growth, proliferation, apoptosis, and differentiation. The processes that realize these functions of a cell can be further organized into three processes levels: gene regulation, signaltransduction and metabolism. So, a model of an entire organism is unlikely to describe the detailed functioning of every individual cell, but a model of a cell is likely to include a variety of very detailed description of key cellular processes. Even then, however, a model of a cell is unlikely to contain details of every single gene and protein.</P><P>Indeed, really accurate modeling of the process would require a model far more detailed and complex than most biologists would be comfortable with, using molecular dynamic simulations that explicitly manage the position and momentum of every molecule in the system.</P><P>The &quot;art&quot; of building a good model is to capture the essential features of the biology without burdening the model with non-essential details. Every model is to some extent a  simplification of the biology, but models are valuable because they take ideas that might have been expressed verbally or diagrammatically and make them more explicit, so that they can begin to be understood in a quantitative rather than purely qualitative way.</P><H3><SPAN class="mw-headline" id="Aims_of_modeling">Aims of modeling</SPAN></H3><P>The features of a model depend very much on the aims of the modeling exercise. We therefore need to consider why people model and what they hope to achieve by so doing. Often the most basic aim is to make clear the current state of knowledge regarding a particular system, by attempting to be precise about the elements involved and the interactions between them. Doing this can be a particularly effective way of highlighting gaps in understanding. In addition, having a detailed model of a system allows people to test that their understanding of a system is correct, by seeing if the implications of their models are consistent with observed experimental data. In practice, this model validation stage is central to the systems biology approach. However this work will often represent only the initial stage of the modeling process. Once people have a model they are happy with, they often want to use their models predictively, by conducting &quot;virtual experiments&quot; that might be difficult, time-consuming, or impossible to do in the lab. Such experiments may uncover important indirect relationships between the model components that would be hard to predict otherwise. An additional goal of modern biological modeling is to pool a number of small models of well-understood mechanisms into a large model in order to investigate the effect of interactions between the model components. Models can also be extremely useful for informing the design and analysis of complex biological experiments.</P><P>In summary, modeling and computer simulation are becoming increasingly important in post-genomic biology for integrating knowledge and experimental data and making testable predictions about the behavior of complex biological systems.</P><H3><SPAN class="mw-headline" id="Stochastic_Approaches">Stochastic Approaches</SPAN></H3><P>The most common formulation of stochastic models for biochemical networks is the chemical master equation (CME). The analytical nature of the early stochastic approaches was highly complicated and, in some cases, intractable so that they received little attention in the biochemical community. Later, the situation changed with the increasing computational power of modern computers. And finally Gillespie presented an ground-breaking algorithm for numerically generating sample trajectories of the abundances of chemical species in chemical reaction networks. The so-called &quot;stochastic simulation algorithm,&quot; or &quot;Gillespie algorithm,&quot; can easily be implemented in any programming or scripting language that has a pseudorandom number generator. Several software packages implementing the algorithm have been developed. Differerent stochastic approaches and their interrelationships are depicted in Figure.</P><P>&lt;img src=&quot;&quot; class=&quot;figure&quot; /&gt;</P><P>For large biochemical systems, with many species and reactions, stochastic simulations (based on the original Gillespie algorithm) become computationally demanding. Recent years have seen a large interest in improving the efficiency/speed of stochastic simulations by modification/approximation of the original Gillespie algorithm. These improvements include the &quot;next reaction&quot; method of Gibson and Bruck, the &quot;τ-leap&quot; method and its various improvements and generalizations and the &quot;maximal time step method&quot;, which combines the next reaction and the τ-leap methods.</P><P>While stochastic simulations are a practical way to realize the CME, analytical approximations offer more insights into the influence of noise on cell function. Formally, the CME is a continuous-time discrete-state Markov process. For gaining intuitive insight and a quick characterization of fluctuations in biochemical networks, the CME is usually approximated analytically in different ways, including the frequently used chemical Langevin equation (CLE), the linear noise approximation (LNA), and the two-moment approximation (2MA).</P><P>The traditional Langevin approach is based on the assumption that the time-rate of abundance (copy number or concentration) or the flux of a component can be decomposed into a deterministic flux and  a Langevin noise term, which is a Gaussian (white noise) process with zero mean and amplitude determined by the system dynamics. This separation of noise from system dynamics may be a reasonable assumption for external noise that arises from the interaction of the system with the other systems (such as the environment), but cannot be assumed for the internal noise that arises from within the system. Internal noise is not something that can be isolated from the system, because it results from the discrete nature of the underlying molecular events. Any noise term in the model must be derived from the system dynamics and cannot be presupposed in an ad hoc manner. However, the CLE does not suffer from above criticism because Gillespie derived it from the CME description. The CLE allows much faster simulations compared to the exact stochastic simulation algorithm (SSA) and its variants. The CLE is a stochastic different equation (dealing directly with random variables rather than moments) and has no direct way of representing the mean and (co)variance and the coupling between the two. That does not imply the CLE, like the LNA, which has the same mean as the solution of deterministic model, ignores the coupling.</P><P>Markov processes form the basis of the vast majority of stochastic models of dynamical systems. At the center of a stochastic analysis is the Chapman-Kolmogorov equation (CKE), which describes the evolution of a Markov process over time. From the CKE stem three equations of practical importance: the master equation for jump Markov processes, the Fokker-Planck equation for continuous Markov processes, and the differential Chapman-Kolmogorov equation (dCKE) for processes made up both the continuous and jump parts.</P><H3><SPAN class="mw-headline" id="Stochastic_Formulation_and_Markov_Process">Stochastic Formulation and Markov Process</SPAN></H3><P>Since the occurrence of reactions involves discrete and random events at the microscopic level, it is impossible to deterministically predict the progress of reactions in terms of the macroscopic variables (observables).</P><P>Our goal is to determine how the process N(t) of copy numbers evolves in time. Starting at time t=0 from some initial state N(0), every sample path of the process remains in state N(0) for a random amount of time W<SUB>1</SUB> until the occurrence of a reaction takes process to a new state N(W<SUB>1</SUB>); it remains in state N(W<SUB>1</SUB>) for another random amount of time W<SUB>2</SUB> until the occurrence of another reaction takes the process to a new state N(W<SUB>1</SUB>+W<SUB>2</SUB>), and so on. In other words, the time-dependent copy number N(t) is a jump process.</P><P>The stochastic process N(t) is characterized by a collection of state probabilities and transition probabilities. The state probability P(n,t)=Pr[N(t)=n] is the probability that the process N(t) is the state n at a time t. The transition probability Pr[N(t<SUB>0</SUB>+t)=n|N(t<SUB>0</SUB>)=m] is the conditional probability that process N(t) has moved from state m to state n during the time interval [t<SUB>0</SUB>,t<SUB>0</SUB>+t]. The analysis of a stochastic process becomes greatly simplified when the above transition probability depends on (i) the starting state m but not on the states before time t<SUB>0</SUB> and (ii) the interval length t but not on the. Property (i) is the well-known Markov process. The process holding property (ii) is said to be homogeneous process.</P><H3><SPAN class="mw-headline" id="References">References</SPAN></H3><P>[1] Uri Alon『An introduction to Systems Biology: Design Principles od Biological Circuits』</P><P>[2] Isaacs, Farren J., et al. &quot;Engineered riboregulators enable post-transcriptional control of gene expression.&quot; Nature biotechnology 22.7 (2004): 841-847.</P><P>[3] Kelly, Jason R., et al. &quot;Measuring the activity of BioBrick promoters using an in vivo reference standard.&quot; Journal of biological engineering 3.1 (2009): 4.</P><P>[4] Part:BBa I13453 &lt;<A href="http://parts.igem.org/Part:BBa_I13453" class="external free" rel="nofollow">http://parts.igem.org/Part:BBa_I13453</A>&gt; ( We finally accessed on 2014/8/20)</P><P>[5]Megerle, Judith A., et al. &quot;Timing and dynamics of single cell gene expression in the arabinose utilization system.&quot; Biophysical journal 95.4 (2008): 2103-2115.</P><P>[6] iGEM Kyoto 2010 &lt;<A href="https://2010.igem.org/Team:Kyoto/Project/Goal_A" class="external free" rel="nofollow">https://2010.igem.org/Team:Kyoto/Project/Goal_A</A>&gt;</P><P>[7] pSB1A2 &lt;<A href="http://parts.igem.org/Part:pSB1A2" class="external free" rel="nofollow">http://parts.igem.org/Part:pSB1A2</A>&gt; ( We finally accessed on 2014/8/20)</P><P>[8] pSB1C3 &lt;<A href="http://parts.igem.org/Part:pSB1C3" class="external free" rel="nofollow">http://parts.igem.org/Part:pSB1C3</A>&gt; ( We finally accessed on 2014/8/20)</P><P>[9] Andersen, Jens Bo, et al. &quot;New unstable variants of green fluorescent protein for studies of transient gene expression in bacteria.&quot; Applied and environmental microbiology 64.6 (1998): 2240-2246.</P><P>[10] Gillespie, Daniel T. &quot;A general method for numerically simulating the stochastic time evolution of coupled chemical reactions.&quot; Journal of computational physics 22.4 (1976): 403-434.</P><P>[11] Kierzek, Andrzej M., Jolanta Zaim, and Piotr Zielenkiewicz. &quot;The effect of transcription and translation initiation frequencies on the stochastic fluctuations in prokaryotic gene expression.&quot; Journal of Biological Chemistry 276.11 (2001): 8165-8172.</P><P>[12]Roussel, Marc R., and Rui Zhu. &quot;Validation of an algorithm for delay stochastic simulation of transcription and translation in prokaryotic gene expression.&quot; Physical biology 3.4 (2006): 274.</P><P>[13] Ribeiro, Andre S. &quot;Stochastic and delayed stochastic models of gene expression and regulation.&quot; Mathematical biosciences 223.1 (2010): 1-11.</P><P>[14] Schlicht, Robert, and Gerhard Winkler. &quot;A delay stochastic process with applications in molecular biology.&quot; Journal of mathematical biology 57.5 (2008): 613-648.</P><P>[15] Santillán, Moisés, and Michael C. Mackey. &quot;Influence of Catabolite Repression and Inducer Exclusion on the Bistable Behavior of the&lt; i&gt; lac&lt;/i&gt; Operon.&quot; Biophysical journal 86.3 (2004): 1282-1292.</P><P>[16] D.J.Wilkinson.Stochastic Modelling for Systems Biology.Mathematical &amp; Computational Biology.Chapman &amp; Hall/CRC, London, Apr. 2006. <A href="/Special:BookSources/1584885408" class="internal mw-magiclink-isbn">ISBN 1584885408</A></P><P>[17] Mukhtar Ullah &amp; Olaf Wolkenhauer Stochastic Approaches for Systems Biology.</P><P>[18] Bionumbers &lt;<A href="http://bionumbers.hms.harvard.edu/bionumber.aspx?&amp;id=100059&amp;ver=30&amp;trm=translation" class="external free" rel="nofollow">http://bionumbers.hms.harvard.edu/bionumber.aspx?&amp;id=100059&amp;ver=30&amp;trm=translation</A>&gt; ( We finally accessed on 2014/8/20)</P><P>[19] Bionumbers &lt;<A href="http://bionumbers.hms.harvard.edu/bionumber.aspx?&amp;id=100059&amp;ver=30&amp;trm=translation" class="external free" rel="nofollow">http://bionumbers.hms.harvard.edu/bionumber.aspx?&amp;id=100059&amp;ver=30&amp;trm=translation</A>&gt; ( We finally accessed on 2014/8/20)</P><P>[20]Staron, Anna. Phylogenetic and functional analyses of stress-responsive bacterial transmembrane signal transducing systems. Diss. lmu, 2012.</P></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:UT-Tokyo/Counter/Project/Modeling">http://2014.igem.org/Team:UT-Tokyo/Counter/Project/Modeling</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:UT-Tokyo/Counter/Project/Modeling" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:UT-Tokyo/Counter/Project/Modeling" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:UT-Tokyo/Counter/Project/Modeling&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:UT-Tokyo/Counter/Project/Modeling&amp;oldid=400333" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>