<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Austin_UTexas_Demonstrate skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Austin UTexas/Demonstrate</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="menu_wrap"><NAV class="menu"><UL class="clearfix"><LI><A href="https://2018.igem.org/Team:Austin_UTexas">HOME</A></LI><LI><A href="">TEAM <SPAN class="arrow">▼</SPAN></A></LI><LI><A href="https://2018.igem.org/Team:Austin_UTexas/Team">Meet the Team!</A></LI><LI><A href="https://2018.igem.org/Team:Austin_UTexas/Collaborations">  Collaborations </A></LI></UL><LI><A href="">PROJECT  <SPAN class="arrow">▼</SPAN></A></LI><LI><A href="https://2018.igem.org/Team:Austin_UTexas/Description">Description </A></LI><LI><A href="https://2018.igem.org/Team:Austin_UTexas/Results">Results</A></LI><LI><A href="https://2018.igem.org/Team:Austin_UTexas/Kit/Contents">Kit Contents</A></LI><LI><A href="https://2018.igem.org/Team:Austin_UTexas/InterLab">InterLab</A></LI><LI><A href="https://2018.igem.org/Team:Austin_UTexas/Demonstrate">Demonstrate</A></LI><LI><A href="https://2018.igem.org/Team:Austin_UTexas/Improve">Improve</A></LI><LI><A href="https://2018.igem.org/Team:Austin_UTexas/Attributions">Attributions</A></LI><LI><A href="https://2018.igem.org/Team:Austin_UTexas/Parts">PARTS</A></LI><LI><A href="https://2018.igem.org/Team:Austin_UTexas/Safety"> SAFETY </A></LI><LI><A href=""> HUMAN PRACTICES<SPAN class="arrow">▼</SPAN></A></LI><LI><A href="https://2018.igem.org/Team:Austin_UTexas/HP/Silver">Silver HP</A></LI><LI><A href="https://2018.igem.org/Team:Austin_UTexas/HP/Gold_Integrated">Integrated and Gold</A></LI></NAV></DIV><DIV class="content_wrapper"><DIV class="background"><H1>Demonstrate</H1><DIV class="floatleft"><FIGURE><FIGCAPTION><B>Figure 1:</B> From left to right: Cultures containing BCP, PinkCP, PCP, GFP, E2C, RCP, or GCP.</FIGCAPTION></FIGURE></DIV><P><B>The Broad Host Range Kit is not simply a set of parts and plasmids, but rather a set of tools designed to inform researchers about the functionality of broad host origins of replication in non-model organisms.</B> Once the assembly plasmids were synthesized, we needed to demonstrate that the kit can be applied to non-model organisms and produce results that are easy to understand. To simplify the process of validating compatible parts, color reporters and barcodes are included as indicators of different assembly plasmids. This allows us to transform multiple plasmids in a single reaction, dubbed the &quot;One Tube Method&quot;.</P><P>To prove our kit is capable of testing multiple plasmids at a time in non-model bacteria, four types of experiments were organized. First, part plasmids containing the reporter protein coding sequences were plated together to determine which would be good candidates for integration into bridge parts. These coding sequences will be paired with specific origins of replication, allowing us to determine if the plasmids were replicated and expressed by the host. This experiment also provides an example for how a plate may look, when all the full assemblies are transformed into a non-model bacteria. Transforming all the plasmids into a single sample of bacteria at the same time is the quickest way to test many plasmids at once. Second, we tested the one tube method in <I>E. coli</I> via electroporation. During the third experiment, we electroporated the same one tube mixtures into <I>Vibrio natriegens</I> to begin testing kit design in non-model species. Lastly, we conjugated a single assembly plasmid from an auxotrophic strain of <I>E. coli</I> to <I>Serratia marcescens</I> to confirm that the Origin of Transfer (part type seven), which is responsible for conjugation, is functional. Then the eight plasmid mixture was transformed into the same auxotrophic strain of <I>E. coli</I> in preparation for conjugation with multiple plasmids. <B>These experiments were instructive, and though some specifics need to be altered, we ultimately determined that the execution of the kit was effective </B>.</P><H3> Demonstration of Colored Reporters </H3><DIV class="floatright"><FIGURE><FIGCAPTION><B>Figure 2:</B> Chromoprotein expected results demonstration. The above photos show an LB only plate with the chromoproteins BCP, GFP, GCP, PCP, pink, and E2C. RCP was present on the plate however is not easily identified in the picture and is therefore not shown. </FIGCAPTION></FIGURE></DIV><P>We plated a mixture containing 7 different chromoproteins to confirm that our selected colors would be different enough for us to identify the different plasmids that they represent. We grew overnight cultures of the chromoproteins from glycerol stocks. 100 µL of each O/N culture was then mixed together in a tube and diluted. This was streaked out on an LB only plate and incubated overnight at 37°C. Since some chromoproteins take longer to be strongly expressed, the plate in Figure 2 does not reflect all of the chromoproteins. The plate on the left in Figure 2 shows the plate on a white background, making it easier to identify the E2-crimson colonies. The plate on the right in Figure 2 is on a black background and under a blue light. Some chromoproteins, like E2C and GFP, are more easily identified under UV light because they fluoresce. <B>In both of the plates in Figure 2, the different chromoprotein colors can be clearly seen, indicating that the chromoprotein genes would serve as effective reporter genes to differentiate the plasmids.</B></P><DIV class="floatright"><FIGURE><FIGCAPTION><B>Table 1:</B> The completed assembly plasmids along with their antibiotic resistance gene, origin of replication, reporter, and barcode. </FIGCAPTION></FIGURE></DIV><H3>Demonstrating the One Tube Method</H3><H4>Electroporation into <I>E. coli</I></H4><DIV class="floatright"><FIGURE><FIGCAPTION><B>Figure 3:</B> Transformation of the One Tube reactions into <I>E. coli</I> via electroporation. Above are plates containing the transformants from a one-tube reaction that were successful. The plate on the far right and left are from One Tube 1, and the plate in the middle is from one tube 2. This includes the KAN plate that had 3 assemblies, and the CAM plate which has 2 assemblies (See Table 1 on the Results - Assembly Plasmids page for more details). Within this reaction, many different assembly plasmids are added to a competent cell mixture. The various reporter genes help to distinguish which assemblies were successfully transformed. This can be visualized in the far-right plate, featuring both E2-crimson (red) and GFP (green) colonies.</FIGCAPTION></FIGURE></DIV><P>At the core of the Broad Host Range Kit is the One Tube Method of testing plasmids in a host bacteria.<B> The One Tube Method involves transforming multiple assemblies at once in a single reaction.</B> The majority of people who get our kit will begin their experiments transforming a mixture of all the plasmids, testing the origins of replication, into a sample of their host bacteria. The idea is that through selective plating and use of visual reporters, researchers will quickly be able to determine which origins of replication work in the non-model organisms. This is the first step in building a plasmid to genetically engineer an organism. Results will be confirmed by one of two ways: assessing the phenotype produced by the reporter gene or sequencing at the barcode region. The reporter is either a specific fluorescent protein or chromoprotein, and each color designates a particular origin of replication. At the DNA level, a unique, non-coding Barcode sequence does the same thing, specifying which origin is present. We created our one tube mixture of initial assemblies by calculating equimolar concentrations.</P><DIV class="floatright"><FIGURE><FIGCAPTION><B>Figure 4:</B> Collaboration with Rice University — One Tube Reactions. The plates above represent Rice’s attempts to utilize the one tube reactions. As seen in Figure 3, the plates exhibit multiple phenotypes implying that multiple assemblies have been successfully transformed. As in Figure 3, <I>E. coli</I> was plated onto LB plates with varying antibiotics.</FIGCAPTION></FIGURE></DIV><P>Figure 3 represents our first successful trials to see if the one tube method would work. The first mixture had 8 different assemblies (refer to Table 1 in the Results - Assembly Plasmids page for the 8 assemblies) and bacteria transformed with it was plated on 4 different plates, each with a different antibiotic to select for <I>E. coli</I> containing a plasmid with the corresponding antibiotic resistance. The p15a and pMB1 origins have each been paired with a green fluorescent protein in this example, while the pAMB1 origin is paired with E2 crimson, which appears black in normal light and red under UV. This mixture is a preliminary version of what the one tube reaction in our kit will contain, and tests the methodology of combining all plasmids, rather than segregating by antibiotic resistance. <I>E. coli</I> only grew with active reporter genes on the KAN and CAM plates, although there should have been growth also on the CRB and TET plates. We believe we did not see growth on CRB and TET because the DNA concentration might have been too low or they may have needed to incubate longer. The second One Tube mixture (Figure 3, middle) contained only 3 plasmids with the same antibiotic resistance, pAMB1+KAN, pMB1+KAN, and p15A+KAN, and was plated on a single KAN plate. This was done to focus on the effect of transforming multiple plasmids with the same antibiotic resistance. <B>These tests allowed us to determine that the colored reporter system was an effective method of determining which plasmids were present in the bacteria.</B> However, the absence of colonies on TET and CRB plates showed us that we may need higher concentrations of some plasmids relative to others to generate more obvious results.</P><P>Rice University performed the same initial tests independently. Their One tube had 8 different assemblies, as described above. This reaction was plated on 4 different plates, each with a different antibiotic. This was to ensure only the <I>E. coli</I> that had picked up each specific assembly would grow on each plate. The results in Figure 4 show that <I>E. coli</I> grew with active reporter genes on the KAN and CAM as well as on the CRB plate. Their CRB plate was left to grow longer than ours, which might be why they saw positive results. TET was not plated because Rice University did not have the antibiotic available. The second One Tube contained was also used by Rice and this was plated on a single KAN plate. <B>This collaborative effort allowed us to see that results produced in other labs would be comparable to ours, and the Rice team was able to provide feedback on how we could improve the protocol we provided them.  This feedback was integrated into our project design.</B></P><H4>Electroporation into <I>Vibrio natriegens</I></H4><DIV class="floatright"><FIGURE><FIGCAPTION><B>Figure 5:</B> Transformation of One Tube Reaction into <I>Vibrio natriegens</I>. Using a one tube plasmid mix, non-model organism, <I>V. natriegens</I> was transformed. Growth can be seen on this kanamycin-containing LB plate. It should be noted that two distinct colony morphologies are present. Neither showed distinct colored or fluorescent phenotypes — some colonies were lightly colored.</FIGCAPTION></FIGURE></DIV><P>Though we built the plasmid kit in <I>E. coli</I> because it is a reliable and easy to use model organism, the kit is actually designed for non-model organisms. To ensure that the plasmids and method of transforming many plasmids at once was applicable outside of <I>E. coli</I>, we began testing the same preliminary version of the one tube mixture in other host organisms. The assemblies found in Table 1 were transformed into <I>V. natriegens</I>, specifically we used Vmax cells. The recoveries were plated on media containing KAN, CRB, or CAM. Separately, we also performed a transformation that involved a positive control provided by the Vmax kit. We received this positive control from the Barrick lab who told us that the quality of controls was not optimal. Vmax cells are more sensitive to ampicillin/carbenicillin than our <I>E. coli</I>, and Vmax™ Express has natural resistance to kanamycin. Therefore, we used the recommended concentration for the selection of kanamycin-resistant colonies: 100 µg/mL of kanamycin. Colony growth was seen only in KAN and CAM plates, however, there was no clear presence of the reporter colors on either plate. Additionally, the KAN plate has two size colonies growing (Figure 5).</P><P>Transformation into <I>V. natriegens</I> was only evident in the KAN plate, on which three assembly plasmids containing the appropriate antibiotic resistance gene can grow. The protocol for transforming into Vmax cells (electrocompetent <I>V. natriegens</I>) cites the presence of two types of colonies as potential evidence for successful transformation, the smaller size colonies perhaps being the ones with the extra burden of maintaining one of our plasmids. To see the protocol,
<A href="https://www.sgidna.com/documentation/Vmax/Vmax_Express_40026_100517-Rev1.6.pdf">click here.</A> There were hints of color in the smaller colonies, but what was observed could not be considered conclusive evidence of reporter gene expression. This could be due to the choice of promoter for these completed assemblies. Up until this point, we had been making our assemblies with the GlpT promoter. However, this promoter does not have a particularly broad host range, which may explain why the transformations did not appear to be as successful. <B>The lack of reporter expression in <I>V. natriegens</I> has led us to begin transitioning to using the CP25 promoter in our assemblies and bridge parts, because it is considerably more broad host range and has a higher expression rate<SUP>1</SUP>.</B></P><P>We have several options for next steps. These involve picking colonies and plating them on new plates to see if they grow normally, and/or sequencing colonies for their barcodes which are specific to origins. This will allow us to check which origins are working. From there we may retransform for further verification.</P><H3>Demonstrations of Transformation </H3><H4>Electroporation into Mu-Free Donor <I>E. coli</I></H4><DIV class="floatleft"><FIGURE><FIGCAPTION><B>Figure 6:</B> Transformation of One Tube Reaction into Mu-Free donor <I>E. coli</I>. The above photos show LB plates with varying types of antibiotics. MFD <I>E. coli</I> were electroporated, recovered in SOC media, and plated. Although few reporters can be seen (sGFP dominates each plate except on CRB), it is possible that multiple transformations succeeded.</FIGCAPTION></FIGURE></DIV><P>In order to test the BHR kit’s function in a diverse array of organisms, we demonstrated how a one tube plasmid mix, which contained multiple assembly plasmids, could be transformed into DAP auxotrophic Mu-Free Donor (MFD) <I>E. coli</I> cells. <B>These transformed Mu-Free Donors would ultimately be used to conjugate with non-model organisms in which electroporation is not a viable option for transformation.</B> When attempting to conjugate with a target non-model organism, donor cells carrying the BHR plasmids must first be created. Rather than performing an independent transformation with the donor cells for each assembly plasmid, it is more time-efficient to perform a single procedure that transforms all of the assembly plasmids into the donor cells that is followed by a selection of the recovered cells. In this demonstration, we show the how the one tube transformation can easily be applied to Mu-Free Donors for conjugation.</P><P>Electrocompetent MFU donors were electroporated with a single mix of DNA containing eight assembly plasmids, as described above and in Table 1. The antibiotics were added to the media at a 1:1000 ratio, and the media was supplemented with DAP at 6 µL DAP per ml of media. The electrocompetent MFD cells were transformed via electroporation on settings suitable for <I>E. coli</I> and allowed to recover for an hour at 37°C. 100 µL of the recovery was plated on each selective plate and incubated overnight. The results of this demonstration are shown in Figure 6. <B>These transformed cells can be used in future conjugations using the Broad Host Range kit in non-model organisms.</B></P><H4> Conjugation into <I>S. marcescens</I></H4><DIV class="floatleft"><FIGURE><FIGCAPTION><B>Figure 7:</B> Demonstration via conjugation into <I>Serratia marcescens</I>. Depicted above are two plates containing LB and kanamycin on which <I>S. marcescens</I> is growing. The top plate contains <I>S. marcescens</I> conjugated with BHR902, one of our assembly plasmids. The bottom plate contains the control with plasmids conjugated into the host cells. The left side shows the plates under normal light; the right side shows the same plates under a high energy blue light. </FIGCAPTION></FIGURE></DIV><P>When building the plasmid kit in <I>E. coli</I>, a model organism, electroporation was a convenient avenue for transforming plasmids into the host. However, electroporation protocols can vary for different bacteria species, and many may have no protocol developed for them at all. Rather than assuming the burden of developing one, researchers frequently rely on conjugation. This is the transfer of a plasmid from one species of bacteria to the other, which commonly happens in nature. We electroporated a GFP-expressing assembly plasmid (BHR 902) from our kit into a strain of <I>E. coli</I>, called a Mu-free donor, that is a DAP auxotroph. This means DAP has to be added to the growth media for the bacteria to survive. We then grew these bacteria along with <I>S. marcescens</I> at different ratios. Samples from the dual cultures were then grown on media without DAP and with antibiotic, selecting for only <I>S. marcescens</I>. <B>The <I>S. marcescens</I> that did grow on the media contained the plasmid, expressing the green fluorescence and the antibiotic resistance gene.</B></P><P>The results of the conjugation are shown in Figure 7. Although the green coloring of <I>S. marcescens</I> with the BHR kit plasmid (top) under UV light (right) is not as strong as some of the controls, the bacteria is clearly expressing the green phenotype, which it does not naturally have. Figure 7 contains bacteria that was streaked onto media without additional nutrients, which was then picked and re-streaked to ensure that no nutrients or colonies without plasmids remained. The first round of selective plating killed off the colonies of MFD <I>E. coli</I> cells because the plates lacked the extra nutrients needed for their growth, so they were not included in the pictures above. Since the untransformed <I>S. marcescens</I> and Top 10 <I>E. coli</I> did not have the antibiotic resistance the plasmids coded for, they died on the media because it contained KAN antibiotic. The control plasmid contained a GFP and KAN resistance as well. However, it appears to have allowed the MFD strain to survive on the selective media, as traces can be seen in one of the control plasmid conjugation sections, as well as the sections where MFD plus the control plasmid was plated alone. However, because all the negative controls for the <I>S. marcescens</I> conjugation with a plasmid from the BHR kit died on the selective media, the results are not assumed to be invalidated. <B>Ultimately, because the <I>S. marcescens</I> expressed the GFP from the BHR plasmid, we concluded that the origin of transfer (part type 7) was functional, allowing us to conjugate into other bacteria.</B></P><LI> Jensen PR, Hammer K. The sequence of spacers between the consensus sequences modulates the strength of prokaryotic promoters. Appl Environ Microbiol. 64: 82–87 (1998).</LI></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>