<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Austin_UTexas_HP_Gold_Integrated skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Austin UTexas/HP/Gold Integrated</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="menu_wrap"><NAV class="menu"><UL class="clearfix"><LI><A href="https://2018.igem.org/Team:Austin_UTexas">HOME</A></LI><LI><A href="">TEAM <SPAN class="arrow">▼</SPAN></A></LI><LI><A href="https://2018.igem.org/Team:Austin_UTexas/Team">Meet the Team!</A></LI><LI><A href="https://2018.igem.org/Team:Austin_UTexas/Collaborations">  Collaborations </A></LI></UL><LI><A href="">PROJECT  <SPAN class="arrow">▼</SPAN></A></LI><LI><A href="https://2018.igem.org/Team:Austin_UTexas/Description">Description </A></LI><LI><A href="https://2018.igem.org/Team:Austin_UTexas/Results">Results</A></LI><LI><A href="https://2018.igem.org/Team:Austin_UTexas/Kit/Contents">Kit Contents</A></LI><LI><A href="https://2018.igem.org/Team:Austin_UTexas/InterLab">InterLab</A></LI><LI><A href="https://2018.igem.org/Team:Austin_UTexas/Demonstrate">Demonstrate</A></LI><LI><A href="https://2018.igem.org/Team:Austin_UTexas/Improve">Improve</A></LI><LI><A href="https://2018.igem.org/Team:Austin_UTexas/Attributions">Attributions</A></LI><LI><A href="https://2018.igem.org/Team:Austin_UTexas/Parts">PARTS</A></LI><LI><A href="https://2018.igem.org/Team:Austin_UTexas/Safety"> SAFETY </A></LI><LI><A href=""> HUMAN PRACTICES<SPAN class="arrow">▼</SPAN></A></LI><LI><A href="https://2018.igem.org/Team:Austin_UTexas/HP/Silver">Silver HP</A></LI><LI><A href="https://2018.igem.org/Team:Austin_UTexas/HP/Gold_Integrated">Integrated and Gold</A></LI></NAV></DIV><DIV class="content_wrapper"><DIV class="background"><H2>Integrated Human Practices</H2><DIV class="column full_size"><H3> Optimization of Golden Gate Assembly </H3><P><B>Many of the teams we talked to, including Texas Tech and Rice University, either did not use Golden Gate Assembly (GGA) or did not use it as their main technique for molecular cloning and were not intimately familiar with the process. Given our experience with GGA, we decided to make our kit simpler to use.  We designed standardized bridging sequences that reduced the number of parts in a GGA reaction, increasing the likelihood of a successful assembly. We also improved the expression of certain chromoproteins to allow them to express more quickly and brighter. Through these two improvements, we streamlined the kit, increasing its efficiency and potential to be used by amateurs and professionals alike. </B></P><H4> 1 - 5 Bridging Sequences </H4><P> Golden Gate Assembly is a molecular cloning method that uses modular, interchangeable parts. Parts are categorized into types according to their function. In order to make a full assembly plasmid, containing multiple parts of interest, such as a promoter, a specific coding sequence, and a specific origin, each part must first be put into a standardized entry vector. This entry vector is pYTK001, a GFP-dropout with a colE one origin of replication and a chloramphenicol resistance gene. Parts are cloned from a template plasmid using primers that add BsmBI and BsaI restriction sites to the ends of the PCR products. These products are then made into parts plasmids, using BsmBI and T4 DNA ligase. The end product is a self-replicating plasmid that acts to store the part for use in a larger assembly. The BsaI sites on the insertion sequence are retained for future assemblies using multiple part plasmids.</P><P>Multiple part plasmids are combined in a Golden Gate Assembly reaction. Each part is digested with BsaI and ligated back together. However, the success of reaction, that is when all part types are successfully ligated together to create an entire plasmid, decreases as the number of parts increases. Therefore, larger, more sophisticated plasmids tend to be difficult to make. </P><DIV class="floatleft"><FIGURE><FIGCAPTION><B> Figure 1: The original 9 part assembly versus an improved 5 part assembly.</B> Top: schematic showing the required parts that we had originally planned to use for making our fully assembled plasmid. Bottom: The revised assembly plan, which makes use of a 1-5 Bridge. The 1-5 Bridge reduces the number of parts in a BsaI GGA reaction by combining many of the non-variant part types into a single part prior to the BsaI assembly reaction.</FIGCAPTION></FIGURE></DIV><P> Our solution was to create a standardized bridging sequence that contained part types 1 through 5 to make the number of overall BsaI containing parts in the final reaction smaller. Instead of designing primers that contain both BsaI and BsmBI sites on both ends, the 1-5 bridge only contains a BsaI sequence on the Type 1 Forward primer and on the Type 5 reverse primer. All other parts in the bridge contain only BsmBI sites. Therefore, when the parts are combined in a BsmBI Golden Gate reaction, a single part that can be used in BsaI assembly is formed.</P><H4> Better Reporter Genes </H4><P><B>Many of our chromoproteins, such as Red chromoprotein and E2-Crimson, express weakly or take days to portray the desired phenotype. To make them more useful reporters, we coupled the sequence to a strong, constitutive promoter. This created brighter colors that expressed faster </B></P><P> When attempting to make assemblies, our experiments were often delayed when we attempted to use certain fluorescent proteins or chromoproteins because they either did not express, expressed weakly, or took many days to become visible.</P><P> To solve this problem, we extracted a strong promoter, CP25, and a red chromoprotein from pSL1, with primers that added BsmBI and BsaI restriction sites. We then inserted that sequence into pYTK001 to make a part plasmid. Currently, we are attempting to make assemblies with this part. 
</P><DIV class="floatright"><FIGURE><FIGCAPTION><B>Figure 2:</B> Red chromoprotein coupled to a strong promoter inserted into an entry vector and expressed in <I>E. coli</I>.</FIGCAPTION></FIGURE></DIV><H4>Integration of Feedback from Synthetic Biology Experts</H4><P>Dr. Brian Renda of Gingko Bioworks emphasized that our kit would be limited by the ability of the plasmids to be inserted into the bacteria of interest as many can not be transformed with standard protocols. Therefore, we transformed the assembly plasmids into a strain of E. coli that can act as a plasmid donor in conjugations.</P><P>In addressing Dr.Davies' suggestion that we show the kit can work in multiple organisms, we attempted to express a variety of our assemblies in non-model organisms. These organisms included: </P><UL><LI><I>Serratia marcescens </I></LI><LI><I>Serratia symbiotica </I></LI><LI><I>Staphylococcus epidermidis</I> ATCC 35984 </LI><LI><I>Staphylococcus epidermidis</I> ATCC 12228 </LI><LI><I> Psuedomonas chloritidismutans</I></LI><LI><I>Vibrio natriegens </I></LI><LI><I>Lactobacillus plantarum</I></LI></UL><P>Success was only confirmed in a few of these organisms. However, these experiments were informative in that they demonstrated the variety of transformation and growth procedures necessary to genetically manipulate non-model organisms and underscored the strengths and weaknesses of our kit. </P><P>See <A href="https://2018.igem.org/Team:Austin_UTexas/Demonstrate">Demonstrate</A> Page </P><H3> One Tube Reaction Feedback </H3><H4>Rice and Texas Tech University</H4><P><B> After discussing our kit with members of the Rice University and Texas Tech University iGEM Teams, we designed methods to simplify Golden Gate Assembly reactions, redesigned our presentation of the kit, and amended our protocol for potential users.</B></P><P>Members of our iGEM Team gave the Rice University iGEM Team and the Texas Tech iGEM Team a One Tube reaction to test in their lab. One reaction contained all our confirmed full assemblies and the other contained our assemblies for a single antibiotic, kanamycin. After performing the one tube reaction, Rice University sent us the pictures found in Figure 2. Texas Tech was unable to perform the transformation, but provided a critique of the protocol sent with the tubes.</P><DIV class="floatleft"><FIGURE><FIGCAPTION><B>Figure 3:</B> These are the results of Rice's One Tube electroporation with <I>E. coli</I>.</FIGCAPTION></FIGURE></DIV><P>While the E2-Crimson, shuttle vector assemblies did not initially express when our team performed the one tube reaction, Rice saw blue colonies on their plates, which acted as a secondary confirmation of the Shuttle vector origin. They also gave us feedback that we incorporated into our protocol and experimental design. They mentioned that it would be helpful to have the expected colony phenotype for each assembly in the protocol. Instead of the one tube reaction, they would have preferred  that each assembly was sent in a separate tube which they believe would have been more amenable to the heat shock protocol they used.</P><P> From their and Texas Tech's feedback, we made the following changes to our protocol: </P><UL><LI> Explicitly stated the purpose of using the One Tube Reaction at the beginning of the procedure, as well as the reasoning for providing both a tube containing all assemblies and a tube with just the assemblies for a specific antibiotic. </LI><LI> Listed the expected colony phenotypes for each assembly. </LI><LI> Explored and implemented methods to make transformation efficient for labs which do not use electroporation. For example, we transformed MuFDs with our One Tube reaction for use in conjugations. </LI><LI> Included the expected bacterial range of each origin in the protocol. </LI></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>