<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Austin_UTexas_Protocols skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Austin UTexas/Protocols</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="menu_wrap"><NAV class="menu"><UL class="clearfix"><LI><A href="https://2018.igem.org/Team:Austin_UTexas">HOME</A></LI><LI><A href="">TEAM <SPAN class="arrow">▼</SPAN></A></LI><LI><A href="https://2018.igem.org/Team:Austin_UTexas/Team">Meet the Team!</A></LI><LI><A href="https://2018.igem.org/Team:Austin_UTexas/Collaborations">  Collaborations </A></LI></UL><LI><A href="">PROJECT  <SPAN class="arrow">▼</SPAN></A></LI><LI><A href="https://2018.igem.org/Team:Austin_UTexas/Description">Description </A></LI><LI><A href="https://2018.igem.org/Team:Austin_UTexas/Results">Results</A></LI><LI><A href="https://2018.igem.org/Team:Austin_UTexas/Kit/Contents">Kit Contents</A></LI><LI><A href="https://2018.igem.org/Team:Austin_UTexas/InterLab">InterLab</A></LI><LI><A href="https://2018.igem.org/Team:Austin_UTexas/Demonstrate">Demonstrate</A></LI><LI><A href="https://2018.igem.org/Team:Austin_UTexas/Improve">Improve</A></LI><LI><A href="https://2018.igem.org/Team:Austin_UTexas/Attributions">Attributions</A></LI><LI><A href="https://2018.igem.org/Team:Austin_UTexas/Parts">PARTS</A></LI><LI><A href="https://2018.igem.org/Team:Austin_UTexas/Safety"> SAFETY </A></LI><LI><A href=""> HUMAN PRACTICES<SPAN class="arrow">▼</SPAN></A></LI><LI><A href="https://2018.igem.org/Team:Austin_UTexas/HP/Silver">Silver HP</A></LI><LI><A href="https://2018.igem.org/Team:Austin_UTexas/HP/Gold_Integrated">Integrated and Gold</A></LI></NAV></DIV><DIV class="content_wrapper"><DIV class="background"><DIV class="column full_size"><H1>Protocols</H1><P>Hello! You have received the Broad Host Range Plasmid Kit, made by the 2018 UT iGEM team. We hope this makes your experience trying to genetically engineer your bacteria with plasmids relatively quick and easy. We have designed plasmids to test several broad host range origins of replication in any bacterial species. We have modeled this protocol off our own experiences at the lab bench and hope that, should you need guidance, we have provided it through this procedure. This protocol has been designed in chronological order, starting when it arrives, helping you interpret results, and links showing you how to build your own cassette plasmid. You, of course, will have different bacteria or equipment that us, so we expect that you may need to deviate from this protocol. Our ultimate goal for this kit is to get you one step closer to your goal. Happy experimenting!</P><P>We are sending you a one tube reaction, containing all the plasmids we have designed to test different origins of replication (as well as part plasmids so that you can customize your own assembly plasmids once you have learned which origins work and which do not). Remember, once DNA is in solution it should always be kept on ice or in a -200F freezer.</P><H3>Transformation through Electroporation</H3><P>Electroporation is a very reliable method for many of the bacteria we use in our lab: <I>Lactobacillus plantarum</I>, <I>E. coli</I>, and <I>Psuedomonas chlorar</I>. It is also beneficial for transforming DNA into the Mu-free strain of <I>E. coli
</I> we use as the donor species in conjugations. The following protocol is the one we use for <I>E. coli</I>. However, not all species respond well to electroporation, and protocol can vary for some species. Please check to see what transformation techniques are commonly used in your bacteria. If no electroporation or chemical transformation exists for this species, the best option is a conjugation protocol. If you are using your own protocol please skip to <B>Plating</B>.</P><H4>Materials</H4><DIV class="rowz"><DIV class="columnz" style="background-color:#808080;"><UL><LI>One Tube Mixture</LI><LI>Electrocompetent Cells</LI><LI>500 µL SOC Media</LI></UL></DIV><DIV class="columnz" style="background-color:#808080;"><UL><LI>37°C Shaker Incubator</LI><LI>Electro-Cuvette</LI><LI>Micropipettes and Tips</LI><LI>Microcentrifuge Tubes</LI><P>First add 1 µL of your DNA from the DNA mixture tube and add it to your tube of electrocompetent cells. Then pipette your cells into a chilled e-cuvette. Be sure to wipe condensation of the side of the cuvette before placing it in the pulser. Pulse the cuvette and immediately add 500 µL SOC to the cuvette and pipette up and down to mix. It is very important to add the SOC immediately after the pulse to ensure the cells survive. Transfer the contents of the cuvette into a microcentrifuge tube and place it in a 37°C Shaker for an hour to allow the cells to recover.</P><H3>Plating of Cells</H3><P>Plating is a standard procedure. However, we have included the procedure because this step will give you your results. Once plated, single bacteria grow into colonies of cells displaying a homogeneous phenotype. If you know your bacteria has an innate tolerance to one of the following bacteria, there is no need to plate on media with that antibiotic, as the transformed cells cannot be selected for.</P><H4>Materials</H4><DIV class="rowz"><DIV class="columnz" style="background-color:#808080;"><UL><LI>LB Agar Media</LI><LI>Petri Dishes</LI><LI>Spreader</LI><LI>Electroporation Recovery</LI><LI>Bunsen Burner</LI></UL></DIV><DIV class="columnz" style="background-color:#808080;"><UL><LI>37°C Shaker Incubator</LI><LI>Antibiotic(s)</LI><LI>Micropipettes and Tips</LI><LI>70% Ethanol</LI><P>The LB/Agar first needs to be melted in a microwave at 50% power until completely melted. From here on out sterile conditions. Once melted, one of the antibiotics should be added to the LB/agar. The amount of antibiotic added needs to make a ratio of 1000:1. So if there is 50 mL of LB/agar then 50 µL of antibiotic needs to be added to the solution. After adding the antibiotic, pour 25 mL LB/agar into labeled petri dishes and leave the lid open while under a flame to let solidify. Once solid, pipette 100 µL of your transformed cells onto the plate and spread the cells evenly around the plate with a sterilized spreader. Once again allow the cells to dry with the lid open. Once dry, place the plate in a 37°C incubator overnight.</P><H1>Picturesss</H1><TABLE><TBODY><TR><TH>Gene Name</TH><TH>Color under Normal Light</TH><TH>Color under UV Light</TH><TH>Origin of Replication</TH></TR><TR><TD>Red Chromoprotein</TD><TD>Red</TD><TD>Red</TD><TD>pMB1</TD></TR><TR><TD>Green Fluorescent Protein</TD><TD>Green</TD><TD>Green</TD><TD>p15a</TD></TR><TR><TD>E2 Crimson</TD><TD>Blue-Gray</TD><TD>Red</TD><TD>pAMB1+ColE1</TD></TR></TBODY></TABLE><P>Each reporter gene is synonymous to the presence of one specific origin of replication. The table above shows which marker is associated with which origin of replication. If there are colonies with no colors, you could still sequence the plasmid with primers included in the kit to get the barcode and determine which origin worked. Some colors take longer to show up so waiting another day could allow more color to be expressed.</P><H3>Picking Colonies to Make Overnight Cultures</H3><P>Picking colonies can help confirm that a colony is expressing a whole plasmid. Where as there can be a bit of uncertainty when the bacteria are grown on a plate, an organism must have an immunity to the antibiotic in the liquid media to grow. </P><H4>Materials</H4><DIV class="rowz"><DIV class="columnz" style="background-color:#808080;"><UL><LI>LB Media</LI><LI>Serological Pipettes</LI><LI>Culture Tubes</LI><LI>Electroporation Recovery</LI><LI>Bunsen Burner</LI></UL></DIV><DIV class="columnz" style="background-color:#808080;"><UL><LI>37°C Incubator</LI><LI>Antibiotic(s)</LI><LI>Micropipettes and Tips</LI><LI>70% Ethanol</LI><LI>Recovery Plates</LI><P>Once you have allowed your colonies to grow overnight you have to start an overnight culture. By picking a single colony from a plate and letting it grow overnight we can generate a test tube full of clones of a single cell and therefore all the same DNA. First add 5 mL of LB media into a labeled test tube. Then add 5 µL of your antibiotic.  You must then pick which colony on the plate you want to culture. A good colony to pick should display a chromoprotein or fluorescent protein and be a good size. (Examples are shown above) Once you’ve decided on a colony, use a micropipette to gently scrape the colony of the plate and then discard the tip into the test tube to inoculate the LB media. Then place the test tube in a 37°C shaker overnight.</P><P>Turbid liquid cultures that do not express the reporter protein can indicate contamination, a species’ innate resistance to the antibiotic, or a host’s incompatibility with the plasmid’s promoter region. Indicative of the first two possibilities, are large numbers of colorless colonies on the plate (addressed in the plate results section). If neither is the case, then promoter incompatibility is possible, and the liquid culture should be miniprepped and sequenced (reference section on sequencing).</P><H1>Picture</H1><H3>Glycerol Stocks</H3><P>Once the liquid cultures have incubated for about a day, the culture must be miniprepped and turned into a glycerol stock so the plasmids can be preserved and used in future experiments. Making a glycerol stock of the overnight culture allows you to store your plasmid in the bacteria for long periods of time. <B>Miniprepping</B> the cells isolate the plasmid and allow it to be used in future transformations or cloning reactions. The reagents for miniprepping are sold in kits along with procedures. For that reason, the protocol is not listed here.</P><H4>Materials</H4><DIV class="rowz"><DIV class="columnz" style="background-color:#808080;"><UL><LI>Cryovials</LI><LI>Overnight Cultures</LI><LI>80% Glycerol</LI></UL></DIV><DIV class="columnz" style="background-color:#808080;"><UL><LI>-80°C Freezer</LI><LI>Micropipettes and Tips</LI><P>Glycerol stocks should be made of each overnight culture to preserve the cells for future use. First pipette 250 µL of glycerol in the cryovial. Then add 1000 µL of the O/N culture into the cryovial. Mix the cryovial using a vortex or by inverting the tube a few times. The tube can then be placed in a -80°C freezer for future use.</P><H3>Golden Gate Assembly</H3><FIGURE><FIGCAPTION>The image on the left describes the workflow of Golden Gate Assembly. The figure on the left figure is taken from &quot;A Highly Characterized Yeast Toolkit for Modular, Multipart Assembly&quot; 2015 Lee, Et. al.  The image on the right depicts the various overhangs that are used for Golden Gate Assembly. The figure on the right is adapted from the <A href="http://barricklab.org/twiki/bin/view/Lab/ProtocolsBTKDesignANewPart">Barrick Lab Website</A></FIGCAPTION></FIGURE><H4>Creating a New Part Plasmid</H4><P>Protocol adapted from the <A href="http://barricklab.org/twiki/bin/view/Lab/ProtocolsBTKDesignANewPart">Barrick Lab Website</A></P><P><B>Method 1: Synthesizing dsDNA containing the required overhang regions </B></P><P>When ordering a double-stranded piece of DNA to be synthesized, you can just append the sites needed for cloning in your order. Use this protocol when ordering a gBlock from IDT, for example.</P><P>5'-GCATCGTCTCATCGGTCTCAXXXX YOUR PART YYYYTGAGACCTGAGACGGCAT-3'
3'-CGTAGCAGAGTAGCCAGAGTxxxx your part yyyyACTCTGGACTCTGCCGTA-5' </P><P>Replace XXXX with the prefix-F sequence for the part type you are designing ( see the table and notes above). Similarly, replace YYYY with the suffix-F sequence for the part. 
In general, the amount of DNA synthesized is sufficient for cloning into the entry vector without further PCR amplification of a gBlock.</P><P><B>Method 2: Amplifying a sequence with primers that add the required overhang regions</B></P><P>You need to order two primers that will anneal to your desired part sequence and contain overhang sequences necessary for proper Golden Gate Assembly. </P><P>For easy primer design you can use the shiny app here: <A href="https://spleonard1.shinyapps.io/paRting/">Link</A> made by Sean Leonard of the Barrick Lab.
Otherwise reference the table in the diagram at the top of the page to add the necessary prefix (XXXX)/suffix (yyyy) to the primer templates below.</P><P><B>Primer 1 (forward)</B>
5'-GCATCGTCTCATCGGTCTCAXXXXUUUUUUUUUUUUUUUUUUUU-3'<B>Primer 2 (reverse)</B>
5'-ATGCCGTCTCAGGTCTCAyyyyddddddddddddddddddd-3'
&quot;UUUUU&quot; = Upstream annealing region
&quot;ddddd&quot; = Downsream annealing region (reverse complement)</P><P><B>CAUTION</B> In Primer 2, the priming site (dddd...) must be the reverse-complement of your part and you must use the suffix-R sequence for the Golden Gate part overlap because this is on the other strand. 
The overlap with the template can vary from the 20 base pairs that are shown according to the normal rules for designing good PCR primers. If calculating melting temperatures, be sure to only include the overlap region in your calculations, not the stuff that is being added to the ends!</P><P>If you chose to use amplification of a DNA template to create a new part plasmid, then use the protocol below</P><H5>PCR for Part Amplification with Primers</H5><P>Protocol adapted from the <A href="http://barricklab.org/twiki/bin/view/Lab/ProtocolsStandardPCR">Barrick Lab Website</A></P><H4>Materials</H4><DIV class="rowz"><DIV class="columnz" style="background-color:#808080;"><UL><LI>PCR Tubes</LI><LI>Molecular Grade Water</LI><LI>Forward and Reverse Primers</LI><LI>5x HF Buffer</LI><LI>DNA Template</LI></UL></DIV><DIV class="columnz" style="background-color:#808080;"><UL><LI>Phusion Polymerase</LI><LI>dNTPs</LI><LI>MgCl<SUB>2</SUB></LI><LI>DMSO</LI><LI>Thermal Cycler</LI><DIV class="column"><TABLE><TBODY><TR><TH>Template</TH><TH>Forward Primer</TH><TH>Reverse Primer</TH><TH>5x HF Buffer</TH><TH>10 mM dNTPs</TH><TH>Phusion</TH><TH>MgCl<SUB>2</SUB>/th&gt; 
    </TH><TH>DMSO</TH><TH>ddH<SUB>2</SUB>O</TH></TR><TR><TD>&lt;250 ng</TD><TD>0.5µM</TD><TD>0.5µM</TD><TD>1x</TD><TD>200µM</TD><TD>1 Unit/50µL</TD><TD>0.5µM</TD><TD>0.5µM</TD><TD>-</TD></TR></TBODY></TABLE></DIV><H3>Conjugation</H3><P>Some non-model organisms cannot be transformed through electroporation, in which case, conjugation may be a more viable option for the transfer our plasmids into a organism. The conjugation will occur between a donor strain that carries the desired plasmid from the BHR kit and the recipient strain of interest. Follow the one tube electroporation protocol to produce multiple donor strains, each carrying a different plasmid from the kit. To test each plasmid in your non-model organism, a separate conjugation should be performed between each donor and your target bacteria. This protocol outlines how to perform a conjugation between DAP (diaminopimelic acid) - auxotrophic E. coli with a standard recipient strain. This protocol is adapted from the general protocol developed by the Barrick lab’s Sean Leonard. Some aspects of this protocol may require modification for conjugation with non-model recipient strains.</P><H6>Materials</H6><UL><LI><B>Donor strain</B> - We use Mu Free Donors (MFDpir), a DAP auxotroph that contains chromosome-integrated conjugation transfer machinery. The donor strain should carry one plasmid from the kit for conjugative transfer.</LI><LI><B>Recipient strain</B> - The recipient strain’s antibiotic resistance should be well characterized in order to select for transconjugant bacteria after conjugation.</LI><LI><B>Non-selective plates</B> - These plates should contain DAP and media suitable for both donor and recipient strains.</LI><LI><B>Selective plates</B>- These plates should contain an antibiotic which the recipient strain is naturally susceptible to and should corresponds with the selective marker on the plasmid. These plates should select against the donor strain and thus should contain no DAP.</LI></UL><P>Produce overnight cultures of the donor and recipient strains. The donor strain should be grown in the presence of DAP and an antibiotic corresponding with the selective marker on its plasmid. Spin down 1 mL of donor strain culture at 3000 rpm for 5 minutes. Pour out the supernatant, wash with PBS, and repeat. This process removes excess antibiotics from the donor strain. Repeat the spinning and washing procedure for the recipient strain. After washing, measure the OD of the recipient and donor cells and mix the cells at a 1:1 ratio. This ratio may be modified depending on the growth rate of the recipient strain relative to the donor strain. Without spreading, plate 100 µL of the donor-recipient mixture on a non-selective plate and incubate overnight. Collect the conjugation mixture and wash in 1 mL of PBS, spin down the cells, and repeat at least once more in order to remove excess DAP. Plate the 100 µL of the washed cells on a selective plate and incubate overnight. A secondary selection may be necessary in order to fully select against the donor cells. Pick transconjugant colonies of the recipient strain and confirm the transfer of the plasmid through PCR and barcode sequencing.</P></UL></DIV></DIV></UL></DIV></DIV></UL></DIV></DIV></UL></DIV></DIV></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>