"
Page
Discussion
View source
History
teams
Log in
 
Team:UEA-JIC Norwich/Nittygritty-algae
From 2011.igem.org
University of East Anglia-JIC
UNIVERSITY OF EAST ANGLIA-JOHN INNES CENTRE
Home
Team Students | Advisors | Acknowledgements |
Project Project Overview | Algae | Moss | Methods | Lab Safety | Attributions |
Results Registry Overview | Data |
Human Practice Interviews | Outreach | UK Team Meetup | Media | School Work |
Lab Journal Week 1 | Week 2 | Week 3 | Week 4 | Week 5 | Week 6 | Week 7 | Week 8 | Week 9 | Week 10 | Week 11 | Week 12 | Week 13 | Week 14 | Week 15 |
Software
Photo Gallery
Sponsors
ALGAE
We have preformed two different types of protocol; glass bead and electroporation to transform the algal species Chlamydomonas reinhardtii. This is a single celled, photosynthetic eukaryote. We have been using the specific strain CC-4350 cw15-302 mt+. This strain is biflagellate with a high transformation frequency, due in part to its lack of a cell wall. Its genome has been sequenced, allowing us to research its codon bias.
There are twenty common amino acids, but over sixty codon configurations. So, each amino acid can be coded for by multiple codons. The code is thus said to be degenerate. However, most organisms display a preference for one codon or another, and so prefer to express a given amino acid by a certain codon. This is known as the codon bias. We researched the codon bias for Chlamydomonas reinhardtii, and are using this information to optimise the Biobricks we wish to use for expression in C. reinhardtii.
Chlamydomonas Fact File
Name:Chlamydomonas reinhardtii
Attributes
Can be transformed by a variety of methods – electroporation; the bacterium Agrobacterium tumorfaciens; glass beads; or by a biolistic particle delivery system (gene gun)
Eukaryotic photosynthetic organism – therefore its post translational modifications will more closely reflect those seen in plants and other higher organisms when compared to, for example, E.coli
Difficulties of Use
Many strains have a cell wall, and therefore prove difficult to transform
Growing time of around a week in cultures or plates
Low transformation frequency due to genome integration of plasmids
Growth Conditions
Requires TAP (tris-acetone phosphate) media
Must be grown in sunlight
Must be grown at 25°C
Must be grown in an incubator shaker to ensure adequate aeration of the media with carbon dioxide
Stock solution
Volume
Component
Concentration in stock Solution
Concentration in final media
Tris base
2.42g
H2NC(CH2OH)3
2.00 . 10-2M
TAP-salts (Beijerinck salts)
25mL
NH4Cl MgSO4 . 7H2O CaCl2 . 2H2O
15 g . L-1 4 g . L-1 2 g . L-1
7.00 . 10-3 M 8.30 . 10-4 M 4.50 . 10-4 M
Phosphate solution
1mL
K2HPO4 KH2PO4
28.8 g . 100 mL-1 14.4 g . 100 mL-1
1.65 . 10-3 M 1.05 . 10-3M
Hunter’s trace Elements
1mL
5.00 g . 100 mL-1 2.20 g . 100 mL-1 1.14 g . 100 mL-1 0.50 g . 100 mL-1 0.50 g . 100 mL-1 0.16 g . 100 mL-1 0.16 g . 100 mL-1 0.11 g . 100 mL-1
5.00 g . 100 mL-1 2.20 g . 100 mL-1 1.14 g . 100 mL-1 0.50 g . 100 mL-1 0.50 g . 100 mL-1 0.16 g . 100 mL-1 0.16 g . 100 mL-1 0.11 g . 100 mL-1
1.34 . 10-4 M 1.36 . 10-4 M 1.84 . 10-4 M 4.00 . 10-5 M 3.29 . 10-5 M 1.23 . 10-5 M 1.00 . 10-5 M 4.44 . 10-6 M
Acetic Acid
1mL
CH3COOH
Method:
Dissolve Na2EDTA . 2H2O in 100 mL dd-H2O by heating to 60-80 °C, then adjust pH with KOH to 5.0. Add all trace elements separately and check the pH value constantly. The pH value should not increase above 6.8, otherwise MnSO4 may precipitate. Let the solution stand at 4 °C; when the colour changes from orange to red after approx. 2 weeks, filter it, split it and store at -20 °C teflon or polycarbonate containers (do not use glass containers for trace elements as these tend to adsorb to the glass surface). After addition of acetic acid the pH should range at about 7.0.
Adjust medium to final pH of 6.0 or as desired with acetic acid and autoclave at 121 °C for 20 min.
For stock cultures on agar slants add 1.0-1.3 % Agar (e.g. purified high strength, 1000 g . cm-²) to prepared medium before autoclaving.
The genome integration of plasmids lowers the transformation frequency of Chlamydomonas, as it is difficult to direct the specific site the DNA will integrate into. This raises the risk of integration within a gene vital for the organisms survival, thus killing the organism. The presence of a cell wall potentially hinders the transformations of many strains, but strains are available that are cell wall deficient (such as the CC-4350 cw15-302 mt+ strain we are using). Using such a strain negates the problem.
Plasmid Design:
The genome integration of plasmids lowers the transformation frequency of Chlamydomonas, as it is difficult to direct the specific site the DNA will integrate into. This raises the risk of integration within a gene vital for the organisms survival, thus killing the organism. The presence of a cell wall potentially hinders the transformations of many strains, but strains are available that are cell wall deficient (such as the CC-4350 cw15-302 mt+ strain we are using). Using such a strain negates the problem.
References:
1. http://cccryo.fraunhofer.de/sources/files/medien/TAP.pdf
Retrieved from " http://2011.igem.org/Team:UEA-JIC_Norwich/Nittygritty-algae "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
