"
Page
Discussion
View source
History
teams
Log in
 
Team:UEA-JIC Norwich/Nittygritty-moss
From 2011.igem.org
University of East Anglia-JIC
UNIVERSITY OF EAST ANGLIA-JOHN INNES CENTRE
Home
Team Students | Advisors | Acknowledgements |
Project Project Overview | Algae | Moss | Methods | Lab Safety | Attributions |
Results Registry Overview | Data |
Human Practice Interviews | Outreach | UK Team Meetup | Media | School Work |
Lab Journal Week 1 | Week 2 | Week 3 | Week 4 | Week 5 | Week 6 | Week 7 | Week 8 | Week 9 | Week 10 | Week 11 | Week 12 | Week 13 | Week 14 | Week 15 |
Software
Photo Gallery
Sponsors
Moss
We have decided to work with Physcomitrella patens ( P. patens ) with the intention of encouraging its use within iGEM. P patens itself has benefits for use in iGEM. It's already a model organism in plant research which can be transformed with inexpensive, widely available equipment using PEG (poly ethylene glycol) mediated methods and, provided a thorough project and experimental plan is constructed early, it can be transformed and grown within the ten week time period of a typical iGEM project. The full genome has been sequenced and there is a lot of existing data to support iGEM teams who wish to work with it.
A significant benefit to moss is that its transcription pathways are well conserved and it doesn’t suffer from codon bias, in theory providing a greater success rate in translation and synthesis from genes that have been derived from other organisms. If this is found to be correct, work with moss could avoid costly and time consuming gene optimisation and synthesis, which we believe is currently seen as a significant restriction on plant work in iGEM.
Physcomitrella Fact File:
Name: Physcomitrella patens
Attributes:
No codon bias
Eukaryotic photosynthetic organism – therefore its post translational modifications will more closely reflect those seen in plants and other higher organisms when compared to, for example, E.coli
Genome is fully sequenced, well documented growth methods and transformation protocols
No genomic integration of plasmid
Difficulties of Use:
Low transformation frequency (though higher than Chlamydomonas)
Takes around a month to grow
Growth Conditions:
Requires Knop’s media
Must be grown at 37°C
Must be grown in sunlight (in light for 16 hours, darkness for 8 hours)
Recipe for PPNH4[1]:
1.84 mM KH2PO4
3.4 mM Ca(NO3)2
1 mM MgSO4
2.72 mM Diammonium tartrate
54 μM FeSO4.7H2O
9.93 μM H3BO3
1.97 μM MnCl2.4H2O
0.23 μM CoCl2.6H2O
0.19 μM ZnSO4. 7H2O
0.22 μM CuSO4. 5H2O
0.10 μM Na2MoO4.2H2O
0.168 μM KI
0.8% agar for solid medium
6% mannitol
All of the above ingredients have been autoclaved, then 10 mL of 1 M CaCl2 (autoclaved) has been added before pouring the plates.
The lack of a codon bias means that P. patens should be able to express genes from all organisms equally well, with no optimal preference for the specific codons used to code for any amino acid, and a similar lack of preference for a specific stop codon. The ability of P. patens to retain plasmids episomally is also an advantage. This lack of genome integration gives it a higher transformation frequency in comparison to algal species.
The lengthy growing time of P. patens is a major disadvantage, but if properly planned the impact of this on projects can be minimised. Once transformed, the growth of colonies after around a week serves as proof of the uptake of a given plasmid. From the time colonies are first observed relevant tests and experiments could be conducted in accordance with the Biobrick transformed into the moss. In this way, it is not necessarily vital to wait a month for the full growth.
[1]Liu, Y., Vidali, L. 2011. Efficient Polyethylene Glycol (PEG) Mediated Transformation of the Moss Physcomitrella patens. J. Vis. Exp. (50), e2560, DOI: 10.3791/2560.
Retrieved from " http://2011.igem.org/Team:UEA-JIC_Norwich/Nittygritty-moss "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
