"
Page
Discussion
View source
History
teams
Log in
 
Team:Leicester/test
From 2013.igem.org
Recycling
The first of the three streams was focused into amplifying the genes of the tod operon present in the Pseudomonas putida F1 strain. The tod operon is known to degrade toluene. Toluene is degraded by mediation of a dioxygenase. The toluene structure is similar to the styrene monomer structure of polystyrene. The team’s objective was to amplify and modify the dioxygenase genes responsible for toluene degradation so it would accept polystyrene.
The todC1C2BA genes present in the tod operon encode the toluene dioxygenase (TDO), which is capable of oxidizing over a 100 substrates. The catalytic oxygenase is encoded by todC1C2. I was determined the presence of a gap directly into the active site, that could be widen to fit polystyrene. Modelling carried out by the Leicester 2012 iGEM team using the programme Pymol, they modified amino acids present in the gap to amino acids with smaller side chains to widen the gap that could allow polystyrene into the active site. The residues that they modified were Met220 -> Ala220, Val421 -> Ala421, Tyr422 -> Leu422 and Tyr266 -> Val266.
We received the Pseudomonas Putida strain F1. We grew the bacteria overnight on plates, using PCR we amplified the genes tod C, X, F, G and B of the tod operon, also the 16s gene was amplified to confirm that we had the right strain of P.putida. The amplified 16s was sent to be sequenced and sequence was aligned using BLAST to the genomic sequence (NC_009512) of Pseudomonas putida. The alignment confirmed that we had the F1 strain. We proceed to carry out a fusion PCR for the tod genes F, X and B.
We made 3 tod genes into biobricks: tod X, F and B. The genes were ligated to the iGEM supplied backbone (pSB1C3), performed a restriction digest on both. The ligations were transformed into DH5α.  After the colonies were picked and put on for an overnight culture, the isolated plasmids were sent to be sequences. Sequencing confirmed that the genes were cloned into the backbone and the restrictions sites required by iGEM were present. Further double digestions were carried out to confirm that no mutations occurred.  (pictures of the gel and plates).
Retrieved from " http://2013.igem.org/Team:Leicester/test "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
