<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Leicester_Notebook"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Leicester/Notebook">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Leicester/Notebook&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Leicester/Notebook&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Leicester/Notebook&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Leicester/Notebook" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Leicester/Notebook</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><P><TITLE>iGEM Leicester Test Page 2012</TITLE></P><DIV id="container"><DIV id="menu" align="center"><UL class="nav" id="nav-one"><LI><A href="/Team:Leicester">Home</A></LI><LI><A href="/Team:Leicester/Team">Team</A></LI><LI><A href="/Team:Leicester/Project">Project</A></LI><LI><A href="https://igem.org/Team.cgi?year=2013">Official Team Profile</A></LI><LI><A href="/Team:Leicester/Parts">Parts Submitted to Registry</A></LI><LI><A href="/Team:Leicester/Modeling">Human Practices</A></LI><LI><A href="/Team:Leicester/Notebook">Notebook</A></LI><LI><A href="/Team:Leicester/Safety">Safety</A></LI><LI><A href="/Team:Leicester/Attributions">Attributions</A></LI></UL></DIV><DIV id="contentArea"><H1><FONT size="6"><B>Notebook</B></FONT></H1><DIV id="countrytabs"><TABLE><TBODY><TR><TH colspan="7">July</TH></TR><TR><TH>M</TH><TH>T</TH><TH>W</TH><TH>T</TH><TH>F</TH><TH>S</TH><TH>S</TH></TR><TR><TD><A href="#" rel="Jul1">1</A></TD><TD><A href="#" rel="Jul2">2</A></TD><TD><A href="#" rel="Jul3">3</A></TD><TD><A href="#" rel="Jul4">4</A></TD><TD><A href="#" rel="Jul5">5</A></TD><TD><A href="#" rel="Jul6">6</A></TD><TD><A href="#" rel="Jul7">7</A></TD></TR><TR><TD><A href="#" rel="Jul8">8</A></TD><TD><A href="#" rel="Jul9">9</A></TD><TD><A href="#" rel="Jul10">10</A></TD><TD><A href="#" rel="Jul11">11</A></TD><TD><A href="#" rel="Jul12">12</A></TD><TD><A href="#" rel="Jul13">13</A></TD><TD><A href="#" rel="Jul14">14</A></TD></TR><TR><TD><A href="#" rel="Jul15">15</A></TD><TD><A href="#" rel="Jul16">16</A></TD><TD><A href="#" rel="Jul17">17</A></TD><TD><A href="#" rel="Jul18">18</A></TD><TD><A href="#" rel="Jul19">19</A></TD><TD><A href="#" rel="Jul20">20</A></TD><TD><A href="#" rel="Jul21">21</A></TD></TR><TR><TD><A href="#" rel="Jul22">22</A></TD><TD><A href="#" rel="Jul23">23</A></TD><TD><A href="#" rel="Jul24">24</A></TD><TD><A href="#" rel="Jul25">25</A></TD><TD><A href="#" rel="Jul26">26</A></TD><TD><A href="#" rel="Jul27">27</A></TD><TD><A href="#" rel="Jul28">28</A></TD></TR><TR><TD><A href="#" rel="Jul29">29</A></TD><TD><A href="#" rel="Jul30">30</A></TD><TD><A href="#" rel="Jul31">31</A></TD></TR></TBODY></TABLE><TABLE><TBODY><TR><TH colspan="7">August</TH></TR><TR><TH>M</TH><TH>T</TH><TH>W</TH><TH>T</TH><TH>F</TH><TH>S</TH><TH>S</TH></TR><TR><TD><A href="#" rel="Aug1">1</A></TD><TD><A href="#" rel="Aug2">2</A></TD><TD><A href="#" rel="Aug3">3</A></TD><TD><A href="#" rel="Aug4">4</A></TD></TR><TR><TD><A href="#" rel="Aug5">5</A></TD><TD><A href="#" rel="Aug6">6</A></TD><TD><A href="#" rel="Aug7">7</A></TD><TD><A href="#" rel="Aug8">8</A></TD><TD><A href="#" rel="Aug9">9</A></TD><TD><A href="#" rel="Aug10">10</A></TD><TD><A href="#" rel="Aug11">11</A></TD></TR><TR><TD><A href="#" rel="Aug12">12</A></TD><TD><A href="#" rel="Aug13">13</A></TD><TD><A href="#" rel="Aug14">14</A></TD><TD><A href="#" rel="Aug15">15</A></TD><TD><A href="#" rel="Aug16">16</A></TD><TD><A href="#" rel="Aug17">17</A></TD><TD><A href="#" rel="Aug18">18</A></TD></TR><TR><TD><A href="#" rel="Aug19">19</A></TD><TD><A href="#" rel="Aug20">20</A></TD><TD><A href="#" rel="Aug21">21</A></TD><TD><A href="#" rel="Aug22">22</A></TD><TD><A href="#" rel="Aug23">23</A></TD><TD><A href="#" rel="Aug24">24</A></TD><TD><A href="#" rel="Aug25">25</A></TD></TR><TR><TD><A href="#" rel="Aug26">26</A></TD><TD><A href="#" rel="Aug27">27</A></TD><TD><A href="#" rel="Aug28">28</A></TD><TD><A href="#" rel="Aug29">29</A></TD><TD><A href="#" rel="Aug30">30</A></TD><TD><A href="#" rel="Aug31">31</A></TD></TR></TBODY></TABLE><TABLE><TBODY><TR><TH colspan="7">September</TH></TR><TR><TH>M</TH><TH>T</TH><TH>W</TH><TH>T</TH><TH>F</TH><TH>S</TH><TH>S</TH></TR><TR><TD><A href="#" rel="Sep1">1</A></TD></TR><TR><TD><A href="#" rel="Sep2">2</A></TD><TD><A href="#" rel="Sep3">3</A></TD><TD><A href="#" rel="Sep4">4</A></TD><TD><A href="#" rel="Sep5">5</A></TD><TD><A href="#" rel="Sep6">6</A></TD><TD><A href="#" rel="Sep7">7</A></TD><TD><A href="#" rel="Sep8">8</A></TD></TR><TR><TD><A href="#" rel="Sep9">9</A></TD><TD><A href="#" rel="Sep10">10</A></TD><TD><A href="#" rel="Sep11">11</A></TD><TD><A href="#" rel="Sep12">12</A></TD><TD><A href="#" rel="Sep13">13</A></TD><TD><A href="#" rel="Sep14">14</A></TD><TD><A href="#" rel="Sep15">15</A></TD></TR><TR><TD><A href="#" rel="Sep16">16</A></TD><TD><A href="#" rel="Sep17">17</A></TD><TD><A href="#" rel="Sep18">18</A></TD><TD><A href="#" rel="Sep19">19</A></TD><TD><A href="#" rel="Sep20">20</A></TD><TD><A href="#" rel="Sep21">21</A></TD><TD><A href="#" rel="Sep22">22</A></TD></TR><TR><TD><A href="#" rel="Sep23">23</A></TD><TD><A href="#" rel="Sep24">24</A></TD><TD><A href="#" rel="Sep25">25</A></TD><TD><A href="#" rel="Sep26">26</A></TD><TD><A href="#" rel="Sep27">27</A></TD><TD><A href="#" rel="Sep28">28</A></TD><TD><A href="#" rel="Sep29">29</A></TD><TD><A href="#" rel="Sep30">30</A></TD></TR></TBODY></TABLE></DIV><DIV style="border-top:1px solid gray; width:860px; margin-bottom: 1em;"><DIV id="Jul11"><H3 class="nb">Making chloramphenicol agar plates</H3>
        400ml agar 
		melt in microwave
		Add 800ul of chloramphenicol to make 25ug/ml
		Pour out into 20 petri dishes
		Leave overnight to set on the bench</DIV><DIV id="Jul12"><H3 class="nb">Transformation of RFP control plasmid</H3>
        Keep competent cells on ice
		Add 50ul of cells to 4 tubes
		Add 1ul of resuspended DNA to 2 tubes
		Add 1ul RFP to the other 2 tubes
		Incubate for 30mins
		Heat shock at 42 degrees for 60secs
		Incubate on ice for 5mins
        Add 200ul of SOC media
		Incubate at 37degrees for 2hrs
		Spread on 6 plates:
		Positive control (normal plate)
	Negative control (No plasmid)
	20ul of 50pg/ul
	200ul of 50pg/ul
	20ul of 10pg/ul
	200ul of 10pg/ul
	leave plates on bench overnight</DIV><DIV id="Jul15"><H3 class="nb">Count colonies on agar plates</H3><TABLE><TBODY><TR><TH>Plate</TH><TH>Colonies</TH></TR><TR><TD>Viability control</TD><TD>Covered</TD></TR><TR><TD>Negative control</TD><TD>1 contaminant</TD></TR><TR><TD>20ul of 50pg/ul</TD><TD>135</TD></TR><TR><TD>20ul of 10pg/ul</TD><TD>96</TD></TR><TR><TD>200ul of 50pg/ul</TD><TD>~1600</TD></TR><TR><TD>200ul of 10pg/ul</TD><TD>944</TD></TR></TBODY></TABLE>
        Therefore there was 1.2x10^8 cfu/ug
        <H3 class="nb">Replate to get individual colonies</H3>
        Take an individual colonies
		Streak out onto new plate
		Sterilize loop using bunsen burner
		Incubate at 37 overnight</DIV><DIV id="Jul16"><H3>Incubate in broth</H3>
        4ml of luria broth in 6 universals
		Add 8ul of chloramphenicol
		Pick single colony and put in broth (sterile technique)
		Incubate overnight at 37 in shaker<H3>Make agarose gel</H3>
        20ml of 5xTBE
		80ml of water
		0.8g of agarose
		(0.8% gel)
		boil till all agarose dissolved
		Cool
		Add 5ul of ethidium bromide
		Pour gel<H3>Make buffer</H3>  50ml of 5xTBE
		200ml of water</DIV><DIV id="Jul17"><H3>Isolating RFP plasmid</H3>
     Add 1.5ml of overnight culture to epindorph 
	Centrifuge for 2mins 
	Remove supernatent 
	Repeat 
	Use thermo scientific mini kit 
	Measure concentration using nanodrop<H3>Double digest</H3>
     50ng of DNA 
	2ul of 3.1 buffer 
	1ul of each restriction (XbaI and PstIin the 6 samples, EcoRI and PstI in the 6 samples, 12 samples in total) 
	Water to make total volume to 20ul 
	Incubate at 60mins at 37 
	20mins at 80</DIV><DIV id="Jul18"><H3>Single digest</H3>
        Digesting RFP plasmid again using a single restriction enzyme- XbaI
		Digesting DNA samples 1-6
		200ng of DNA is run
		Volumes for digest:<TABLE><TBODY><TR><TH>Sample nr</TH><TH>1</TH><TH>2</TH><TH>3</TH><TH>4</TH><TH>5</TH><TH>6</TH></TR><TR><TD>DNA</TD><TD>0.7</TD><TD>0.7</TD><TD>0.8</TD><TD>1.3</TD><TD>1.2</TD><TD>2.6</TD></TR><TR><TD>Buffer</TD><TD>2</TD><TD>2</TD><TD>2</TD><TD>2</TD><TD>2</TD><TD>2</TD></TR><TR><TD>Water</TD><TD>16.3</TD><TD>16.3</TD><TD>16.2</TD><TD>15.7</TD><TD>15.8</TD><TD>14.4</TD></TR><TR><TD>Xbal</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD>1</TD></TR></TBODY></TABLE>
        
        Buffer used was cutsmart 
	Incubate samples at 37C for 1h 
	2ul of orange G dye was added to each samples 
	Samples were then frozen overnight to run on gel on Monday</DIV><DIV id="Jul22"><H3>Make agarose gel</H3>
    	20ml 5xTBE
		80ml water
		1g of agarose
		Boil in microwave 1 min at a time
		Cool
		Add 5ul ethidium bromide<H3>Running single digested DNA from Friday</H3>
        Gel didn't show the expected bands and was disregarded</DIV><DIV id="Jul25"><H3>Isolating Limonene biobrick</H3>
     Use Bioline isolate II plasmid mini kit
     </DIV><DIV id="Jul26"><H3>no title</H3><TABLE><TBODY><TR><TH>Sample</TH><TH>Plasmid Volume(ul)</TH><TH>DNA Concerntration(ng/nl)</TH></TR><TR><TD>1.1</TD><TD>40</TD><TD>131.8</TD></TR><TR><TD>1.2</TD><TD>40</TD><TD>77.1</TD></TR><TR><TD>1.3</TD><TD>46</TD><TD>56.8</TD></TR><TR><TD>1.4</TD><TD>39</TD><TD>109</TD></TR><TR><TD>1.5</TD><TD>41</TD><TD>129.1</TD></TR><TR><TD>1.6</TD><TD>32</TD><TD>93.3</TD></TR></TBODY></TABLE>
     measured DNA concentration using nanodrop
     <H3>Restriction enzyme digest of the DNA samples</H3>
     enzymes: XbaI, PstI
master mix:
Buffer - 14ul
H2O - 88.2ul
XbaI - 1.4ul
PstI - 1.4ul
Total volume - 105ul
Each sample contained 5ul of DNA and 15ul of the master mix.
The total amount of DNA in each sample was 200ng
To calculate it we divided 200ng by the concentration of each sample
The rest of the volume was made up with water to give a total of 5ul.
The samples were then incubated in a water bath at 37C for an hour<H3>Agorose Gel preparation</H3>
20ml of 5xTBE
80ml of water
0.8g of agarose
Boil for 3 minutes, 1 minute at a time.
Let it cool and add 5ul of ethidium bromide.
Pour and let it set for 30 mins.
Adding dye and marker
Used orange g loading dye.
2ul to each sample
Used 1kb ladder, 5ul per 1 marker well.
Run the gel
Gel picture</DIV><DIV id="Jul29"><H3>Ran the gel from the 26/07/2013 again for 45min</H3><H3>Digestion of plasmid backbone (pSB1C3)</H3>
        Three different reactions
Master Mix (changes were made from the protocol)
5ul NEB Buffer 3.1
0.5ul of EcoRI
0.5ul of PstI
19ul of dH2O
For each reaction add:
4ul of linearized backbone
4ul of enzyme master mix
digest at 37C for 30min
heat kill at 80C for 20min<H3>Digestion of the limonene biobrick plasmid backbone (BBa_K118025)</H3>
Using clone 1.1 purified plasmid DNA
For the reaction adding:
5ul NEB Buffer 3.1
2ul of EcoRI
2ul of PstI
2.5ul of dH2O
38.5ul of DNA
digest at 37C for 30min
heat kill at 80C for 20min<H3>Ligation of insert to vector</H3>
For this experiment, two separate samples will be run; a control and the actual ligation.
Calculate appropriate vector:insert ratio and convert to a 1:1 ratio
Add the following substances
11.1ul of dH2O
1ul of QS Ligase *
5ul of 4xQS Buffer (vortex before use)
Mix thoroughly by pipetting
Incubate at room temperature for 5 min to create cohesive ends
Run 2.5-5ul of the ligation mixture onto an agarose gel to check ligation efficiency against a known marker. Add 1ul of dye to aid the visualization of results.
Transform it into competent cells (E.coli)
*For the control, no QS Ligase is added but its equivalent volume (1ul) is made up by dH2O. 
The results from the gel electrophoresis show four lanes; two 1kb markers on either end (the first and forth lane), one control (the second lane) and the actual ligation results (the third lane). The control has two bands which shows that it has not been ligated, one band is around the 2kb area which is lightly-staining and one band is heavy-staining around the 5kb area. The actual ligation shows a lightly-staining smear, conclusive with the formation of a plasmid. The results are sufficiently good to carry on with the next phase of the experiment - the transformation.
     </DIV><DIV id="Jul30"><H3>Transformation of the limonene Biobrick (BBa_K118025)</H3>
        Ligated with the pSB1C3 vector (restriction sites EcoRI, PstI)
Protocol:
Start thawing the competent cells on ice
Seperate 4 pre-chilled eppendorf tubes;
Resistance/Viability test
Ligation experiment with 10ul DNA
Ligation experiment with 5ul DNA
Positive control using 1ul RFP
Add 50ul of thawed competent cells and then the aforementioned volume of DNA to each eppendorf tube.
For each eppendorf tube, pipette the solution gently to mix. Ensure the cells are kept on ice.
Close the tubes and incubate the cells on ice for 30 minutes
Heat shock the cells by immersion in a pre-heated water bath at 42'C for 60 seconds
Incubate the cells on ice for 5 minutes
Add the 200ul of SOC (SOB + 0.4g glucose) media to each tube
Incubate the cells at 37'C for 2 hours
Plate onto petri dishes (ensure your petri dishes are labelled correctly):
For eppendorf 1, plate 100ul onto a chloroamphenicol petri dish and another 100ul of the solution onto a LB (luria broth) petri dish
For eppendorf 2, plate 200ul and 20ul onto two separate chloroamphenicol petri dishes
For eppendorf 3, plate 200ul and 20ul onto two separate chloroamphenicol petri dishes
For eppendorf 4, plate 200ul and 20ul onto two separate chloroamphenicol petri dishes
Incubate the plates at 37'C overnight</DIV><DIV id="Jul31"><H3>Results from the Transformation of limonene Biobrick (BBa_K118025) to the pSB1C3 vector</H3><OL><LI>The Viability test showed colony growth on it, as expected</LI><LI>The Resistance test showed no colony growth, as expected</LI><LI>The ligation experiment with 10ul DNA showed no results for both the 20ul and 200ul plates. This was not expected.</LI><LI>The ligation experiment with 5ul DNA showed no results for both the 20ul and 200ul plates. This was not expected.</LI><LI>The positive RFP control showed red colonies, as expected.</LI></OL>
            Discussion: The results of the viability test indicates that the cells used were viable, and the resistance test indicated that the these cells did not have any intrinsic resistance to the chloroamphenicol antibiotic. The positive RFP control showed red colonies, which implies that the transformation was successful. Since our ligations showed no results, it can be assumed that there was a problem with the ligation experiment itself.
Running the gel for Limonene biobrick digests (BBa_K118025) again for separation of the 2kb and 5kb fragment.

Extracted the 5kb fragment using Bioline gel extraction kit.
     </DIV><DIV id="Aug1"><H3>Digestion of the chlorophenicol plasmid vector pSB1C3</H3>
        Three different reactions
Master Mix (changes were made from the protocol)
5ul NEB Buffer 3.1
0.5ul of EcoRI
0.5ul of PstI
19ul of dH2O
For each reaction add:
4ul of linearized backbone
4ul of enzyme master mix
digest at 37C for 30min
heat kill at 80C for 20min<H3>Ligation of the plasmid pSB1C3 with the purified limonene biobrick (plasmid pSB1C3)</H3>
For this experiment, two separate samples will be run; a control and the actual ligation.
Calculate appropriate vector:insert ratio and convert to a 1:1 ratio
volume of vector - 2ul
volume of insert - 0.8ul
Add the following substances
11.2ul of dH2O
1ul of QS Ligase *
5ul of 4xQS Buffer (vortex before use)
Mix thoroughly by pipetting
Incubate at room temperature for 5 min to create cohesive ends
Run 2.5-5ul of the ligation mixture onto an agarose gel to check ligation efficiency against a known marker. Add 1ul of dye to aid the visualization of results.
Transform it into competent cells (E.coli)
*For the control, no QS Ligase is added but its equivalent volume (1ul) is made up by dH2O.<H3>Transformation of the plasmid pSB1C3 and the limonene biobrick BBa_K118025</H3>
Start thawing the competent cells on ice
Seperate 4 pre-chilled eppendorf tubes;<OL><LI>Resistance/Viability test</LI><LI>Ligation experiment with 10ul DNA</LI><LI>Ligation experiment with 5ul DNA</LI><LI>Positive control using 1ul RFP</LI></OL>	
Add 50ul of thawed competent cells and then the aforementioned volume of DNA to each eppendorf tube.
For each eppendorf tube, pipette the solution gently to mix. Ensure the cells are kept on ice.
Close the tubes and incubate the cells on ice for 30 minutes
Heat shock the cells by immersion in a pre-heated water bath at 42'C for 60 seconds
Incubate the cells on ice for 5 minutes
Add the 200ul of SOC (SOB + 0.4g glucose) media to each tube
Incubate the cells at 37'C for 2 hours
Plate onto petri dishes (ensure your petri dishes are labelled correctly):<OL><LI>For eppendorf 1, plate 100ul onto a chloroamphenicol petri dish and another 100ul of the solution onto a LB (luria broth) petri dish</LI><LI>For eppendorf 2, plate 200ul and 20ul onto two separate chloroamphenicol petri dishes</LI><LI>For eppendorf 3, plate 200ul and 20ul onto two separate chloroamphenicol petri dishes</LI><LI>For eppendorf 4, plate 200ul and 20ul onto two separate chloroamphenicol petri dishes</LI></OL>
Incubate the plates at 37'C overnight<H3>Gel electrophoresis</H3>
Gel electrophoresis was done to confirm the presence of the purified DNA.
5ul of DNA was run, but after electrophoresis no band was present.
We assumed this was down to pippeting error and thus carried on with the transformation.</DIV><DIV id="Aug2"><H3>Results from the transformation plates</H3>
        On the 01/08/2013 we plated the transformations containing the biobrick BBa_K118025.
		Our transformations were successful
        the plates contained different volumes of the transformation: 20ul and 200ul
the plate to test viability shows growth as expected
the plate to test resistance shows no growth as expected
there were two different different transformations that contained different volumes of ligated DNA, which were plated into different plates.
The 10ul ligated DNA plates showed growth
The RFP control plates showed growth as expected
the plates with the ligated DNA did not show growth as the RFP plates, this could be because the cells did not take up as much plasmid as the optimum RFP plasmid or not all the plasmid were ligated with the biobrick.
It is also important to notice that there is not much different in the number of colonies between the 5ul and 10ul plates.<H3>Next steps</H3>The presence of the limonene biobrick in the ligated product has to be confirmed.
This would be done in 2 ways
a) Measurement of volatile organic compounds (VOC) - 3 separate groups of cells would be prepared for this experiment.
The first group would be transformed cells containing the ligated product.
The second group would be transformed cells containing only circularised pSB1C3 vector
The third group would just be competent cells
b) Miniprep of transformed cells containing the ligated product (from above results)<H3>Digestion of pSB1C3 backbone</H3>
Master Mix (changes were made from the protocol)
5ul cut smart Buffer
0.5ul of XbaI
0.5ul of SpeI
19ul of dH2O
For each reaction add:
4ul of linearized backbone
4ul of enzyme master mix
digest at 37C for 30min
heat kill at 80C for 20min
this digestion was done as control to test if our colonies with the limonene biobrick is expressing limonene.<H3>Circularisation of pSB1C3</H3>
For this experiment, two separate samples will be run; a control and the actual ligation.
Add the following substances
12ul of dH2O
1ul of QS Ligase*
2ul of the digested vector pSB1C3
5ul of 4xQS Buffer (vortex before use)
Mix thoroughly by pipetting
Incubate at room temperature for 5 min to create cohesive ends
Run 2.5-5ul of the ligation mixture onto an agarose gel to check ligation efficiency against a known marker. Add 5ul of dye to aid the visualization of results.
Transform it into competent cells (E.coli)<H3>Gel Electrophoresis</H3>
Gel was run in order to confirm that the cirularisation experiment had worked.
The linearised pSB1C3 vector was used as a control.
The results showed that the circularisation experiment worked, as the linearised vector is represented on the gel while the circularised vector was cloudy and ran off the gel.<H3>Transformation of circularised pSB1C3 vector</H3>
After confirming the presence of the circularised vector, it was transformed into competent cells
This was left to incubate overnight at 37 degrees Celsius<H3>Reculturing of transformed cells</H3>
The transformed cells containing the ligated product from yesterday's experiment were recultured onto 2 agar plates.
The 2 plates were divided into 12 squares each.
Each square was streaked with colonies that were randomly chosen from the 5ul and 10ul plates.
The plates were then left on the bench for overnight incubation<H3>Incubation in broth</H3>
The next day the some colonies from the recultured plates were incubated in broth.
This was done in preparation of the miniprep.</DIV><DIV id="Aug5"><H3>Isolation of ligated plasmid</H3>
        The ligated plasmid- chloramphenicol backbone and limonene biobrick, were isolated from cells that were incubated overnight in broth by miniprep.
Protocols from both Bioline kit and Thermoscientific were used for the miniprep.
there were 6 tubes containing the plasmid.
The plasmid concentration in each of the 6 tubes were measured using nanodrop.
200ng of plasmid from each of the 6 different tube was used for digestion.<TABLE><TBODY><TR><TH>sample</TH><TH>volume(ul)</TH><TH>concentration (ng/ul)</TH><TH>yield(ug)</TH></TR><TR><TD>5.1</TD><TD>99</TD><TD>75</TD><TD>7.4</TD></TR><TR><TD>5.2</TD><TD>100</TD><TD>44.2</TD><TD>4.4</TD></TR><TR><TD>5.3</TD><TD>98</TD><TD>46.7</TD><TD>4.6</TD></TR><TR><TD>10.1</TD><TD>46</TD><TD>61.8</TD><TD>2.84</TD></TR><TR><TD>10.2</TD><TD>44</TD><TD>77.1</TD><TD>3.39</TD></TR><TR><TD>10.3</TD><TD>47</TD><TD>13.7</TD><TD>0.64</TD></TR></TBODY></TABLE><H3>Digestion of ligated plasmid</H3>
Master Mix (changes were made from the protocol)
2ul of NEB Buffer
1ul of EcoRI
1ul of PstI
varying concentration of dH2O and plasmid depending on overall concentration of plasmid in each tube.
Final volume of each of the 6 tubes was 20ul.<H3>Gel Electrophoresis</H3>
The product of digestion was run on agarose gel
This was done in order to confirm the presence of the limonene biobrick
The lanes are in the following order:
Marker, Sample 5.1, Sample 5.2, Sample 5.3, Sample 10.1, Sample 10.2, Sample 10.3, Marker.
We were expecting to see a 2kb and 5kb band. These bands appeared for samples 5.1, 10.1,10.2 and 10.3. To further determine the composition of samples, we did a single digest on the samples.<H3>Single digest</H3>
For samples 5.1, 10.1, 10.2 and 10.3 two single digests each were made with EcoRI and PstI, giving a total of 8 samples.
The samples are run on 0.8% agarose gel.</DIV><DIV id="Aug6"><H3>Running of an agarose gel</H3>
     Running the gel for digests of samples 5.1, 10.1, 10.2 and 10.3 from the previous day.
expectation-7kb band, confirming the presence of the limonene biobrick (5kb) and chloramphenicol backbone (2kb)
The gel shows the presence of 7kb bands for both digests of samples 10.1, 10.2 and the 5.1 sample, which was expected.
No bands appeared for samples 5.1.
For samples 10.3 there is a 2kb band, indicating that with the ligation, the chloroamphenicol circulized.<H3>Making chloroamphenicol agar plates</H3>
Making additional plates
400ml of agar
Melt in microwave
3mins on low -&gt; shake
Repeat
2mins on low -&gt; shake
Repeat
Add 800ul of chloroamphenicol at a concentration of 25 ul/ml
Pour out onto 20 petri dishes
Leave to set on bench<H3>Plating out samples</H3>
Plating out samples 10.1, 10.2, 10.3 and 5.1 onto chloroamphenicol plates.
Grow overnight at 37C</DIV><DIV id="Aug7"><H3>Dillution of Herring sperm DNA</H3>
        Dilutions were done in 3 sets:
Samples 1-3 were diluted in 90ul of water, where 10ul of stock DNA was added from an overnight dilution of 100um of Herring sperm DNA in 900ul of water.
Samples 1s-3s were also diluted in 90ul of water, except the first 10um of DNA was taken from an undiluted stock Herring sperm DNA.
Samples 1 5/45- 3 5/45 were from a known concentration of DNA stock, where 5ul of DNA was added to 45ul of water.<TABLE><TBODY><TR><TH>sample</TH><TH>concentration (ng/ul)</TH><TH>260/280	</TH><TH>260/230</TH></TR><TR><TD>1kb</TD><TD>534</TD><TD>1.82</TD><TD>1.95</TD></TR><TR><TD>1 5/45</TD><TD>52.7</TD><TD>1.77</TD><TD>2.05</TD></TR><TR><TD>2 5/45</TD><TD>4.6</TD><TD>1.46</TD><TD>1.88</TD></TR><TR><TD>3 5/45</TD><TD>0.1</TD><TD>0.24</TD><TD>-0.29</TD></TR><TR><TD>Overnight</TD></TR><TR><TD>1 in 10</TD><TD>24.8</TD><TD>1.8</TD><TD>2.42</TD></TR><TR><TD>1</TD><TD>2.1</TD><TD>1.67</TD><TD>22.44</TD></TR><TR><TD>2</TD><TD>0.8</TD><TD>2.87</TD><TD>-3.99</TD></TR><TR><TD>Fresh Stock</TD></TR><TR><TD>1s</TD><TD>5341.8</TD><TD>1.78</TD><TD>2.15</TD></TR><TR><TD>2s</TD><TD>440.0</TD><TD>1.80</TD><TD>2.23</TD></TR><TR><TD>3s</TD><TD>45.1</TD><TD>1.83</TD><TD>2.26</TD></TR></TBODY></TABLE>
concentration of DNA was measured using nanodrop.<H3>Gel electrophoresis of Dilluted Herring sperm DNA</H3>
Lanes are: marker, 1 5/45, 2 5/45, 3 5/45, 1, 2, 3, 1s, 2s, 3s, lambda NT, lambda
Results are as expected. Lanes show that samples 1, 2 and 3 were degraded overnight, suggesting nuclease activity. Samples 1s, 2s and 3s are the fresh Herring sperm stock and show no degradation.</DIV><DIV id="Aug8"><H3>Making samples for DNA sequencing</H3><H3>Making chloroamphenicol/IPTG plates</H3>
        400ml of agar
Boil for 3min x2
Boil for 2min x2
Cool to 40C
Add 800ul of chloroamphenicol at 25ug/ml
Add 400ul IPTG at 0.15mM
Makes 20 petri dishes<H3>Streaked out samples of the limonene/pSB1C3 ligated plasmid</H3>
Streaked samples 5.1, 10.1, 10.2 and 10.3 onto the chloroamphenicol/IPTG plates
IPTG was used to induce the lac operon, and therefore induce limonene synthesis
Plates were labelled using special symbols by only one team member to ensure double-blinded test<H3>Made polystyrene strips</H3>
--&gt; include video here
<H3>Overnight culture preparation</H3>
To make glycerol broths, so sample can be frozen and preserved
Use four universal tubes, add:
4ml Luria Broth
8ul chloroamphenicol
Add bacteria from the original plates (sample 5.1, 10.1, 10.2 and 10.3 into a single tube each respectively)
*Each sample was also re-streaked on a chloroamphenicol plate to maintain current stock.</DIV><DIV id="Aug9"><H3>Isolating plasmid</H3>
        From overnight culture, took 3ml of culture and centrifuged into pellet of samples 5.1, 10.1, 10.2, 10.3
Isolated the plasmid using omega bio-tek Plasmid mini kit I
Concentrations from nano drop are shown in the table below<TABLE><TBODY><TR><TH>Sample</TH><TH>Volume(ul)</TH><TH>Conc.(ng/ul)</TH><TH>260/280</TH><TH>260/230</TH></TR><TR><TD>5.1</TD><TD>92</TD><TD>65.8</TD><TD>1.87</TD><TD>1.75</TD></TR><TR><TD>10.1</TD><TD>88</TD><TD>36.9</TD><TD>1.80</TD><TD>1.77</TD></TR><TR><TD>10.2</TD><TD>89</TD><TD>48.3</TD><TD>1.79</TD><TD>1.98</TD></TR><TR><TD>10.3</TD><TD>94</TD><TD>17.4</TD><TD>1.84</TD><TD>1.59</TD></TR></TBODY></TABLE><H3>Digesting the plasmids for restriction mapping</H3>
    Digesting 200ng of DNA with SacI
1ul of SacI
2ul of NEB buffer 1.1
DNA volumes added for samples 5.1 to 10.3:
3ul; 5ul; 4ul; 11.5ul
Water added for samples 5.1 to 10.3:
14ul; 12ul; 13ul; 5.5ul
Digestion in 37C water bath for 30mins
Heat kill at 80C for 20mins<H3>Running an agarose gel</H3>
Add 5ul of 6x orange G to each sample
1kb marker is used
Wells are loaded in the following order:
5ul of marker, 25ul of samples 5.1; 10.1; 10.2; 10.3, 5ul of marker
The lanes are as expected.
Samples 5.1, 10.1 and 10.3 were expected to show 3 fragments, sized 2070, 1890 and 1529bp. This is what is shown on the gel.
Sample 10.3 only has one SacI restriction site and when cut, a band of 2070bp is expected. That is also shown on the gel.<H3>Glycerol stocks</H3>
Took 750ul from overnight culture and centrifuged at full speed for ~2 minutes
Supernatant was removed
Pellet resuspended in 375ul of HMFM
Samples were then frozen at -80<H3>Sonicating Herring sperm DNA</H3>
The herring Sperm was to viscous, so to reduce viscosity the sonicator was used
1ml of core DNA used in 2 tubes, 1 control and 1 to go in the sonicator
The control tube was left on the bench at room temperature
The 2nd tube was put in the sonicator for the folowing times: 2mins, 2mins, 5mins, 5mins, 20mins, 20mins, 30mins
Each time both tubes were filmed to compare viscosity</DIV><DIV id="Aug12"><H3>Sonicating herring sperm DNA</H3>
        Pipetting out sonicated samples every 30mins
Pipette 4ul of DNA into Son 1-6 every 30mins
Dilute 1ul of DNA from Son 1-6 in 9ul of 1xTE buffer to tubes 10xdil 1-6
Vortex each dilution thoroughly and centrifuge briefly.
Keep tubes on ice<H3>Running an agarose gel for sonication analysis</H3>
To tubes 1-6, add 14 ul of sterile water and 5 ul of orange G dye and 1ul of DNA from Dil x10 1-6 appropriately. Centrifuge briefly. Keep on ice until loading.
Use 5 ul of 1kb marker on both ends
Add buffer so that wells in the tray and the tray itself are covered
Run at 80 V for an hour and a half (gel is 1.2%)</DIV><DIV id="Aug13"><H3>Making agar, chloroamphenicol and chloroamphenicol/IPTG plates</H3>
        	One container of 400 ml Luria Agar was melted down in the microwave. This is enough to make 20 agar petri dishes.
This was poured straight into 4 petri dishes to make agar plates
The remaining agar had 0.64 ul of chloroamphenicol added to it (concentration = 24 ug/ml), and was mixed thoroughly
The resulting solution was poured into 4 petri dishes, to make 4 chloroamphenicol agar plates
The remaining agar solution had 1.5 ul of IPTG added to it (concentration = 24 ug/ml)
This was mixed thoroughly, and then used to make up the remaining 12 chloroamphenicol/IPTG plates.<H3>Agarose gel preparation</H3>
20 ml of 5 x TBE, 80 ml of distilled water and 0.8 g of agarose was added into a container.
The solution was boiled for 3 minutes, 1 minute at a time.
The solution was left to cool.
5 ul of ethidium bromide was added to the solution.
The solution was then poured into the gel plate and left to set for 30 minutes.</DIV><DIV id="Aug14"><H3>Dilutions of Herring Sperm DNA</H3>
        Add 9ul of TE buffer to 6 epindorph tubes
Pipette 1ul of each sonicated DNA sample 1-6 into each buffer for a 10x dilution
vortex thoroughly and centrifuge briefly<H3>Running dilutions on 0.8% gel</H3>
1ul of dilution was put in 14ul of water and 5ul of orange G, to make up a total volume of 20ul for each sample.
5ul of 1kb ladder was used on both ends as a marker<H3>Testing for limonene</H3>
Took sample plates to chemistry lab
     </DIV><DIV id="Aug15"><H3>Serial dilution of herring sperm DNA</H3>
        100ul DNA and 900ul TE to make master mix
5 independent samples 10ul each were taken from master mix
Each independent sample was diluted twice in 90ul of TE
20 dilution samples were then nanodropped to check dilution accuracy<TABLE><TBODY><TR><TH>Sample nr</TH><TH>Concerntration(ng/ul)</TH><TH>260/280</TH><TH>260/230</TH></TR><TR><TD>nr. 1/1</TD><TD>529.9</TD><TD>1.81</TD><TD>2.26</TD></TR><TR><TD>nr. 1/2</TD><TD>52.1</TD><TD>1.86</TD><TD>2.32</TD></TR><TR><TD>nr. 2/1</TD><TD>454.3</TD><TD>1.81</TD><TD>2.28</TD></TR><TR><TD>nr. 2/2</TD><TD>46</TD><TD>1.82</TD><TD>2.23</TD></TR><TR><TD>nr. 3/1</TD><TD>457.3</TD><TD>1.83</TD><TD>2.28</TD></TR><TR><TD>nr. 3/2</TD><TD>529.9</TD><TD>1.81</TD><TD>2.26</TD></TR><TR><TD>nr. 4/1</TD><TD>522.8</TD><TD>1.82</TD><TD>2.27</TD></TR><TR><TD>nr. 4/2</TD><TD>47.3</TD><TD>1.87</TD><TD>2.26</TD></TR><TR><TD>nr. 5/1</TD><TD>481.4</TD><TD>1.82</TD><TD>2.28</TD></TR><TR><TD>nr. 5/2</TD><TD>40.8</TD><TD>1.84</TD><TD>2.35</TD></TR></TBODY></TABLE>

Dilutions show accuracy
Master dilution is sonicated and every 30mins a 30ul sample is taken to dilute twice (10ul of DNA in 90ul of TE buffer)
6 30ul samples are taken
Samples are then run on a gel</DIV><DIV id="Aug16"><H3>Running the gel from sonication series</H3>
        Samples were taken from sonicated tube every 30min intervals, a total of 6 samples.
Each was diluted 100 fold to get a concentration in the range of 40-50ng/ul
5ul of DNA was taken from the dilution, to which 10ul of sterile water and 5ul of orange G was added
Samples were run against a 1kb ladder and control herring sperm DNA, which was not sonicated.</DIV><DIV id="Aug19"><H3>Denaturing sonicated DNA</H3>
        	Denaturing DNA by heat shock for 10mins in a 95C heat block
Adding 5ul of OG dye<H3>Running both denatured and sonicated samples on gel</H3>
    For just the sonication samples 1-6, 5ul of OG and 10ul of sterile water was added to 5ul of DNA.
5ul of 1kb ladder was used for 3 wells each.
    The last 6 wells after ladder are the denatured DNA samples.
We were expecting to see fragmentation for the denatured samples.
There is no significant difference in between the sonicated samples and the sonicated and denatured ones.
There is a slight decrease in bands depending on the sonication time interval.
Suggestions are to use ultra-sonification to view the effects focused just on the sample.</DIV><DIV id="Aug20"><H3>Making nutrient agar plates and broth</H3>
     Making up broth and agar solution to grow P. putida in.
For agar plates:
9.2g of nutrient agar medium
400ml of distilled water
For broth:
3.2g of nutrient broth
400ml of distilled water
Both were autoclaved for 45mins
Agar was poured onto 16 plates<H3>Plating out bacteria</H3>
     Freeze dried P. putida was rehydrated using 1ml of nutrient broth
The culture was aseptically transferred back into the broth containing tube
Mix
From that tube, colonies were streaked onto 6 plates and left to incubate over night at 30C<H3>Growing bacteria in broth</H3>
     100ml of autoclaved broth was added to 2 flasks each
The broth tube, containing the bacteria, was divided between flasks
Flask were left to incubate at 30C on a shaker overnight</DIV><DIV id="Aug21"><H3>Reculturing Bacteria</H3>
     The P. putida grew overnight, which proved that we had viable cells.
The bacteria was then re-streaked to single colonies and left to grow overnight in a 37C incubator.<H3>Isolation of P. putida DNA</H3>
     8 1.5ml samples were isolated from each of the 2 broths that were prepared yesterday.
The first few steps of CTAB protocol was used to prep cells for DNA isolation;
The 8 samples were spun down at 10000 rpm for 5 mins.
The supernatant was discarded.
cells were then resuspended in 1ml of TE.
740ul of this new mixture was transferred to 8 new eppendorf tubes.
20ul of lysozyme was added to each of these 8 samples.
A maxwell robot was then used to complete the DNA extraction process.<H3>Hydrodynamic shearing of Herring Sperm DNA</H3>
     A 20G syringe was used ~24 times to squirt the herring sperm DNA in order to make it less viscous.
30G syringe was also used
This method seems to work better than the sonification.</DIV><DIV id="Aug22"><H3>Measuring the concentrations of isolated P. putida DNA from the previous day</H3>
        Measuring concentration using nanodrop
		Blanking against buffer<TABLE><TBODY><TR><TH>Sample nr.</TH><TH>Concerntration ng/ul</TH><TH>260/280</TH><TH>260/230</TH></TR><TR><TD>2</TD><TD>44</TD><TD>1.8</TD><TD>1.15</TD></TR><TR><TD>4</TD><TD>30.6</TD><TD>1.8</TD><TD>1.29</TD></TR><TR><TD>6</TD><TD>42.3</TD><TD>1.81</TD><TD>1.05</TD></TR><TR><TD>8</TD><TD>35.6</TD><TD>1.85</TD><TD>1.26</TD></TR><TR><TD>10</TD><TD>38.2</TD><TD>1.84</TD><TD>1.28</TD></TR><TR><TD>12</TD><TD>40.6</TD><TD>1.83</TD><TD>1.45</TD></TR><TR><TD>14</TD><TD>39.9</TD><TD>1.88</TD><TD>1.47</TD></TR><TR><TD>16</TD><TD>18.5</TD><TD>1.17</TD><TD>0.67</TD></TR></TBODY></TABLE><H3>Preparing samples to run on 1% gel</H3>
        500ng of each of the isolated P. putida DNA sample is run on gel
Volumes are adjusted to 20ul per well accordingly (includes DNA, sterile H2O and OG)
5ul of orange G dye is added to each sample
5ul of 1kb ladders are used
For sheared DNA, dilutions were loaded
1ul of DNA was diluted in 9ul of 1xTE buffer to make up 10x dilution
1ul from 10x dilution was put in 9ul of 1xTE buffer to make up 100x dilution
Dilutions were nanodropped and the concentration for 100 fold dilution was 121 ng/ul
200ng of herring sperm DNA was run on gel
One lane was left for control, non-sheared DNA, also 100 fold dilution<H3>Running gel of P. putida DNA and Herring sperm DNA</H3>
Samples in lanes goes as follows: 1 Kb ladder, 2, 4, 6, 8, 10, 12, 14, 16, 1 kb ladder, 100x dilution of sheared herring sperm DNA, control 100x diluted herring sperm DNA<H3>Running sonicated herring sperm DNA on 0.8% gel</H3>
3 lanes for running:
Sheared and sonicated
Unsheared and sonicated
Unsheared and unsonicated
5ul of 1kb marker is used
The DNA samples are diluted 100 fold and 200ng of sample is run.
Volume of sterile water is adjusted appropriately to make up 20ul of sample per well, including 5ul of orange G dye
Not visible on the computer file, but picture shows a very slight decrease in sheared/sonificated and unsheared/sonicated samples vs the unsheared/unsonificated control
The difference is not significant, but the viscosity has changed when mechanically spreading the DNA on polystyrene, though it is not that evident on the gel.<H3>Making up a sheared/sonicated dilution to leave on bench overnight</H3>
500ul of sheared/sonicated DNA
500ul of 1xTE
Mix
Leave on bench overnight<H3>Incubating sheared/sonicated dilution at 37C on a shaking rack</H3>
10ul of 1xTE added to 2ul of sheared/sonicated DNA
Mix
Incubated overnight at 37C on a shaking rack</DIV><DIV id="Aug23"><H3>Nanodropping overnight incubations of herring sperm DNA</H3>
        Table representing the concentrations of herring sperm DNA before and after different treatments<TABLE><TBODY><TR><TH>Sample</TH><TH>Concerntration</TH></TR><TR><TD>Sheared/Unsonicated (s.u.s)</TD><TD>17900ng/ul=17.9mg/ml</TD></TR><TR><TD>S.u.s 10fold dil.</TD><TD>1790.1ng/ul</TD></TR><TR><TD>S.u.s 100fold dil</TD><TD>152.6ng/ul</TD></TR><TR><TD>Sheared/Sonicated (s.s)</TD><TD>17667ng/ul=17.6mg/ml</TD></TR><TR><TD>S.s 10fold dil.</TD><TD>1766.7ng/ul</TD></TR><TR><TD>S.s 100fold dil.</TD><TD>156.2ng/ul</TD></TR><TR><TD>Sheared/sonicated overnight</TD><TD>Concentration in the tube=Average of the two samples=16.9mg/ml</TD></TR><TR><TD>500ul(DNA) in 500ul(1xTE)</TD><TD>14485.9ng/ul</TD></TR><TR><TD>2ml(DNA) in 10ml(1xTE)</TD><TD>3883ng/ul</TD></TR></TBODY></TABLE>
        Concentrations decrease with dilutions
The stock concentration in sheared/sonicated tube varies slightly, when derived from measured dilution concentrations that were subjected to different treatments (i.e. overnight incubation), though they orignated from the same tube
Difference in concentration is not too significant though: proposed 17.6mg/ml vs 16.9mg/ml<H3>Running overnight dilutions on gel</H3>
Lanes are as folllows: 1kb ladder, dilution 500ul DNA in 500 1xTE, dilution 2ml DNA in 10ml 1xTE, control unsheared/unsonicated 100 fold dilution.<H3>Running 16s PCR of P. putida</H3>
Lanes are as follows: 100kb ladder, pcr samples 1, 2, 3, 4, 100kb ladder
<H3>Gel Extraction of PCR</H3>
Cut out the bright band
Use Thermo scientific geneJET extraction kit</DIV><DIV id="Aug27"><H3>Running DNA from purified gel</H3>
        Lanes are as follows: 1kb ladder, samples 1+2, samples 3+4, 1kb ladder<H3>Sent samples for sequencing</H3>
        Sent sample to PNACL to be sequenced<H3>First trial of polystyrene combustion with DNA</H3>
        Tested with DNA covered strips and non DNA covered strips
Tested with 1, 2, 3, 6 strips at a time.
Burned with bunsen burner against a ruler to measure rate
+DNA burned with yellow flame, black smoke and left carbon residue
-DNA appeared to melt with into the more compact form of polystyrene and flame did not catch
Experiment is being redesigned to improve it as results were not as expected</DIV><DIV id="Aug28"><H3>Gel electrophoresis</H3>
        Two different sets of gel were run due to the different fragment size of the PCR product.
Smaller fragments were run on a 2% agarose gel whilst larger fragments were run on a 1% gel.<H3>DNA extraction from gel</H3>
        Extracting previously run DNA from gel using Thermo Scientific GeneJet gel extraction kit and following its protocol.
		Concentrations of DNA were measured by nanodrop
		Sample B (Tod C gene) was not purified correctly, another PCR is needed. (Nanodrop shows contamination)
		Table representing the concentration and volume of DNA isolated from samples<TABLE><TBODY><TR><TH>Sample</TH><TH>Volume</TH><TH>Concerntration ng/ul</TH><TH>260/280</TH><TH>260/230</TH></TR><TR><TD>A</TD><TD>48</TD><TD>19.3</TD><TD>2.02</TD><TD>0.03</TD></TR><TR><TD>C</TD><TD>50</TD><TD>27.7</TD><TD>1.88</TD><TD>0.04</TD></TR><TR><TD>D</TD><TD>49</TD><TD>26.9</TD><TD>1.71</TD><TD>0.05</TD></TR><TR><TD>E</TD><TD>50</TD><TD>20.5</TD><TD>1.98</TD><TD>0.03</TD></TR></TBODY></TABLE></DIV><DIV id="Aug29"><H3>Make 2% agarose gel</H3><H3>Run Tod operon ligation genes on the gel</H3>
        1.5h at 70V
        <H3>Isolate the DNA from gel</H3>
        DNA isolated using Bioline Isolate II PCR and gel kit and following its protocol
        <H3>Measuring concentration of DNA using nanodrop</H3><TABLE><TBODY><TR><TH>Sample</TH><TH>Volume</TH><TH>Concerntration ng/ul</TH><TH>260/280</TH><TH>260/230</TH></TR><TR><TD>A1</TD><TD>30</TD><TD>14.7</TD><TD>1.81</TD><TD>1.41</TD></TR><TR><TD>A2</TD><TD>31</TD><TD>13</TD><TD>1.71</TD><TD>1.43</TD></TR><TR><TD>B1</TD><TD>30</TD><TD>23.6</TD><TD>1.93</TD><TD>1.02</TD></TR><TR><TD>B2</TD><TD>29</TD><TD>21.9</TD><TD>1.88</TD><TD>1.33</TD></TR><TR><TD>C1</TD><TD>29</TD><TD>14.7</TD><TD>1.83</TD><TD>0.63</TD></TR><TR><TD>C2</TD><TD>30</TD><TD>15.5</TD><TD>1.86</TD><TD>0.64</TD></TR></TBODY></TABLE></DIV><DIV id="Aug30"><H3>Preparing samples to be sent to be sequenced by PNACL</H3><H3>Repeating the combustion experiment</H3>
        Blocks of both DNA and no DNA coating were made
Blocks are thicker than last experiment to observe the difference
Both DNA and no DNA blocks were sprayed with water and dried for equal treatment conditions<H3>Second attempt at polystyrene combustion with DNA</H3>
        Larger strips were used
DNA was spread on some and water was spread on some other
Strips were left to dry on the bench
some strips were cut from a block of polystyrene which was made with DNA
Each strip was burnt with a bunsen burner with a ruler and filmed to compare the rate of burning
Results were not as expected, DNA appeared to make no difference except a blacker remnant</DIV><DIV id="Sep2"><H3>Preparation of IPTG and Chloramphenicol plates</H3>
        The IPTG/Chloramphenicol plates that were prepared earlier on (08/08/13) for the volatile organic compound (VOC) test were found to be contaminated.
Hence new plates had to be made using the below protocol;
PROTOCOL:
x2 400ml of already prepared and sterilized agar was obtained from the media kitchen.
The agar was microwaved using the lowest power level at 2-3 mins intervals until completely melted.
The x2 400ml agar were then placed in the 37 degrees room to cool.
After cooling, 800ul of cholramphenicol and 400ul of IPTG were added to one bottle whilst only 800ul of chloramphenicol was added to the second bottle.
The content of both agar bottles was then poured into petri dishes.
3 (chloramphenicol+ IPTG) plates were streaked with 5.1 ligated product (limonene biobrick and chloramphenicol backbone) from earlier and 3 others were streaked with 10.2 ligated product.
RFP colonies were streaked on 3 chloramphenicol resistance plates.
All 9 plates were Incubated overnight in a 37 degrees room.<H3>Ligation of PCR products (TOD operon) using the Promega pGEM-T Vector System</H3>
        The samples ligated were A (TODX), C (TODF) and E (TOBG)
The promega protocol was followed and it can be found at http://www.promega.co.uk/resources/protocols/technical-manuals/0/pgem-t-and-pgem-t-easy-vector-systems-protocol/
To calculate the amount of DNA insert needed for each sample, we used the promega biomath calculator (www.promega.com/biomath). We used the 1:3 ratio of vector to insert.
Size of the PCR inserts (add 56bp to account for the primer)
TodX 544 + 56= 600
TodF 460+56= 516
TobG 807 + 56=863

Volume of DNA insert for each sample
A - 1.6ul
C - 1.1ul
E - 1.5ul
The samples were incubate at 4 degrees overnight to be transformed tomorrow morning.<H3>Overnight Broth</H3>
        RFP cells were incubated in luria broth in a 37 degrees shaker incubator
The cells would be minipreped tomorrow to serve as a backbone for the ligation of the TOD biobbricks.
How to make the broth
Using a 15ml centrifuge tube add:
5ml of luria broth
5ul of Chloramphenicol<H3>Preparation of Ampicillin Resistance plates</H3>
        Ampicillin resistance plates were made by adding 400ul of Ampicillin to 400ml of agar.
This would be used to grow the transformants of samples A,C and E (above) as pGEM-T Vector has AmpR gene.<H3>Volatile organic compound (VOC) experiment of limonene biobrick/pBS1C3 backbone</H3>
        9 plates were taken over to our chemistry department to measure the expression of limonene.
The equipment used for this assay is a Proton Transfer Reaction- Time of Flight Mass Spectrometer.
PLATES
3 plates of 5.1(see 05/08/13) ligated product
3 plates of 10.2 (see 05/08/13)ligated product
3 luria agar plates ( to be used as control, for blanking the equipment)</DIV><DIV id="Sep3"><H3>Miniprep of cells</H3>
        The pSB1C3 RFP cells that were incubated in luria broth overnight yesterday, were minipreped using the OMEGA bio-tek protocol, in order to isolate plasmid of interest.
Nanodrop results<TABLE><TBODY><TR><TH>Sample</TH><TH>Volume</TH><TH>Concerntration ng/ul</TH><TH>260/280</TH><TH>260/230</TH></TR><TR><TD>1</TD><TD>40</TD><TD>56.2</TD><TD>1.89</TD><TD>1.95</TD></TR><TR><TD>2</TD><TD>40</TD><TD>62.3</TD><TD>1.88</TD><TD>2.02</TD></TR></TBODY></TABLE><H3>Double Digest of the pSB1C3 miniprep</H3>
        Samples 1 and 2 were transfered to new eppendorf tubes
In each tube it was added
39ul of DNA (miniprep)
1ul of XbaI
2ul of SpeI
6ul of cut smart buffer
12ul of sterile water
The samples were incubated at 37 degrees water buffer After the samples were incubated at 80 degrees to kill the enzymes<H3>Transformation of TOD genes ligations done on the 02/09/13 using the pGEM-T Vector System I-A3600 (changes were made to the protocol)</H3>
        Centrifuge the ligations reactions briefly.
Add 5ul of each ligation reaction to a sterile 1.5ml
Thaw the competent cells on ice. Mix the cells by gently flicking the tube.
Carefully transfer 100ul of the competent cells to the ligation reaction tubes.
Incubate the tubes on ice for 20 minutes
Heat-shock the cells for 45-50 seconds in a water bath at exactly 42C degrees. Do not Shake. Immediately return the tubes to ice for 2 minutes.
Add 950ul of room temperature SOC medium to the ligation reaction transformations. incubate for 1.5 hours at 37C with shaking.<H3>Plating out the transformants</H3>
        The transformants would be plated out on Ampicillin resistant plates that were prepared yesterday.
40ul of X-Gal and 20ul of of IPTG were streaked respectively unto the already made plates. (except the viability and resistance test plates)
Below  is a list of the plates that were made;
1)x2 plates of positive control (no PCR product)- 20ul and 200ul plates 2)x2 plates of Background control- 20ul and 200ul 3)x2 plates of ligated product 1 (L1) - A- 20ul and 200ul plates 4)x2 plates of L2- C -20ul and 200ul plates 5)x2 plates of L3- E - 20ul and 200ul plates 6)x2 transformation control - (50pg of puc19) - 20ul and 200ul plates 7)x1 Viability control (only luria broth)- 200ul of cells 8)x1 resistance control (Only Amp agar) - 200ul of cells
All 14 plates were incubated overnight at 37 degrees centigrade.
     </DIV><DIV id="Sep4"><H3>Transformation results</H3>
        From the above results,colonies were found growing on the resistance control plate,
which indicates that our competent cells already express ampicillin resistance, but this is not meant to be so. 
Hence in order to confirm that our results are reliable;
colonies would be randomly selected from each of the two plates (20ul and 200ul) for the 3 TOD genes
These colonies would be cultured in luria broth + Amp. This broth would be incubated overnight in a 37 degrees shaker incubator.<H3>Gel Electrophoresis</H3>
        The double digest (pSB1C3) from yesterday was run on a 1% gel.
This was done in order to confirm that the digest worked and also to purify the pSB1C3 vector.
From left to right
Sample 1.1 - track 2
Sample 1.2 - track 3
Sample 2.1 - track 4
Sample 2.2 - track 5
The 2 Kb bands are the pSB1C3 backbone
The 1 Kb bands are the RFP biobrick
The two higher bands in tracks 2 and 3 are plasmids that were not digested.<H3>Gel purification of the pSB1C3 backbone and RFP biobrick</H3>        
The Zymoclean Gel DNA recovery Kit was used to perform the purification and its protocol was followed.
However the elution step was changed to 12ul of elution buffer, and this step was repeated.
Nanodrop of the samples<TABLE><TBODY><TR><TH>Sample</TH><TH>Volume</TH><TH>Concerntration ng/ul</TH><TH>260/280</TH><TH>260/230</TH></TR><TR><TD>pSB1C3 1</TD><TD>20</TD><TD>34</TD><TD>1.67</TD><TD>0.92</TD></TR><TR><TD>pSB1C3 2</TD><TD>40</TD><TD>62.3</TD><TD>1.88</TD><TD>2.02</TD></TR><TR><TD>RFP 1</TD><TD>20</TD><TD>28.8</TD><TD>1.57</TD><TD>0.71</TD></TR><TR><TD>RFP 2</TD><TD>18.8</TD><TD>22.2</TD><TD>1.79</TD><TD>1.84</TD></TR></TBODY></TABLE><H3>Double digest of TOD genes PCR products</H3>
        Samples A (TodX), C (TodF) and E (ToBG) were digested with the enzymes XbaI and SpeI.
        Protocol<TABLE><TBODY><TR><TH>Sample</TH><TH>A</TH><TH>C</TH><TH>E</TH></TR><TR><TD>Volume (ul)</TD><TD>20</TD><TD>23.5</TD><TD>21</TD></TR><TR><TD>Spel (ul)</TD><TD>1</TD><TD>1</TD><TD>1</TD></TR><TR><TD>Xbal</TD><TD>0.5</TD><TD>0.5</TD><TD>0.5</TD></TR><TR><TD>Custmart Buffer</TD><TD>6</TD><TD>6</TD><TD>6</TD></TR><TR><TD>5m Tris Hcl (ul)</TD><TD>32.5</TD><TD>29</TD><TD>31.5</TD></TR></TBODY></TABLE>
        
        The samples were incubated at 37C for 90 minutes
Heat kill the enzymes by incubating the samples at 80C for 20 minutes<H3>Ligation of the TOD genes (X, F and ToBG) to the pSB1C3 backbone</H3>
        The pGEM-T Vector System was used, However changes were made to the protocol. The promega biomath caculator (www.promega.com/biomath) was used to work out the ratio of insert to vector. The DNA concentration of the TOD genes can be found at Lab work 28/09/2013. The DNA concentration of the pSB1C3 and the RFP biobrick can be found above. The amount of vector DNA was added at 50ng.<TABLE><TBODY><TR><TH>Samples</TH><TH>A (TodX)</TH><TH>C (TodF)</TH><TH>E (ToBG)</TH><TH>RFP-sample 2 (positive control)</TH><TH>Background Control</TH></TR><TR><TD>X Rapod ligation buffer (ul)</TD><TD>5</TD><TD>5</TD><TD>5</TD><TD>5</TD><TD>5</TD></TR><TR><TD>vector (pSB1c3-sample1) (ul)</TD><TD>1.6</TD><TD>1.6</TD><TD>1.6</TD><TD>1.6</TD><TD>1.6</TD></TR><TR><TD>T4 DNA ligase (ul)</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD>1</TD></TR><TR><TD>PCR product (ul)</TD><TD>2.3</TD><TD>1.4</TD><TD>3.1</TD><TD>3.6</TD><TD>-</TD></TR><TR><TD>Deionized water for the final volume of 12 ul</TD><TD>2.1</TD><TD>3</TD><TD>1.3</TD><TD>0.8</TD><TD>4.4</TD></TR></TBODY></TABLE><H3>Isolating microbes from CSE kits</H3>
        We received some CSE kits from one of our participating schools.
It is to be noted that the polystyrene was not significantly degraded.
Distilled water was run down each of the 2 sides of the kit (the polystyrene and miscellanous sides)into 2 separate petri dishes.
a spreader was used to spread the sample from each of the petri dishes onto luria agar plates.
Thus each of the 2 samples from each CSE kit was incubated at 3 different temperatures (room temperature, 30 degrees and 37 degrees)
Because we received 5 CSE kits, 30 plates were incubated.
     </DIV><DIV id="Sep5"><H3>Transformation of pSB1C3/TOD genes</H3>
        Centrifuge the ligation
Add 6 ul of each ligation reaction to a sterile 1.5ml tube on ice
Thaw the cells on ice
Carefully transfer 50ul of cells to the ligation reaction. Gently flick the tubes and incubate on ice for 20 minutes.
Heat shock the cells for 45 seconds in a 42'C water bath. Do not shake. Immediately return the tubes to ice for 2 minutes.
Add 950 ul of SOC medium (at room temperature) to the ligation reactions. Incubate for 1.5 hours at 37'C on a shaker.
The controls for this experiment are the resistance and viability controls. 
One tube has 50 ul of highly competent cells and SOC medium added to it.
The samples A (TodX), C (TodF), E(ToBG), positive control, and background control were plated on chlorophenicol resistant agar plates, at the volumes of 20ul and 200ul. A 100ul of only highly efficient cells and SOC medium was plated in a chlorophenicol plate and another 100ul was plated on a LB plate.<H3>Isolate plasmid from overnight broth</H3>
        The plasmids were isolated using the Omega Plasmid Mini Kit 1, and its protocol was followed.
Nanodrop results<TABLE><TBODY><TR><TH>Sample</TH><TH>Volume</TH><TH>Concerntration ng/ul</TH><TH>260/280</TH><TH>260/230</TH></TR><TR><TD>TodX 1</TD><TD>39</TD><TD>37.8</TD><TD>2.02</TD><TD>1.72</TD></TR><TR><TD>TodX 2</TD><TD>43.8</TD><TD>59.1</TD><TD>1.9</TD><TD>1.74</TD></TR><TR><TD>TodF 1</TD><TD>42</TD><TD>100.4</TD><TD>1.9</TD><TD>2.02</TD></TR><TR><TD>TodF 2</TD><TD>40.5</TD><TD>26.2</TD><TD>1.92</TD><TD>1.66</TD></TR><TR><TD>ToBG 1</TD><TD>45</TD><TD>53.2</TD><TD>1.94</TD><TD>1.67</TD></TR><TR><TD>ToBG 2</TD><TD>47</TD><TD>21.6</TD><TD>2.00</TD><TD>1.43</TD></TR></TBODY></TABLE><H3>Double digest of isolated plasmid with SpeI and XbaI</H3>
        Master mix for 12 reactions
36 ul of cut smart buffer
1.2 ul of XbaI
2.4 of SpeI
44.4 ul of 5mTris Hcl
Protocol<TABLE><TBODY><TR><TH>Sample</TH><TH>DNA (ul)</TH><TH>Master Mix (ul)</TH><TH>5mTris Hcl (ul) added to final volume of 30ul</TH></TR><TR><TD>TodX 1</TD><TD>13.2</TD><TD>9</TD><TD>7.8</TD></TR><TR><TD>TodX 2</TD><TD>8.5</TD><TD>9</TD><TD>12.5</TD></TR><TR><TD>TodF 1</TD><TD>8.5</TD><TD>9</TD><TD>19</TD></TR><TR><TD>TodF 2</TD><TD>5</TD><TD>9</TD><TD>2</TD></TR><TR><TD>ToBG 1</TD><TD>9.5</TD><TD>9</TD><TD>11.6</TD></TR><TR><TD>ToBG 2</TD><TD>23</TD><TD>9</TD><TD>0</TD></TR></TBODY></TABLE></DIV><DIV id="Sep6"><H3>Results from double digest of isolated plasmid with SpeI and XbaI</H3>
        Experiment conducted on the 05/09/13<H3>Results from the transformation of pSB1C3/TOD genes</H3>
        Experiment conducted on the 05/09/13
The positive RFP control showed colony growth on both the 20 ul and the 200 ul plates, but showed a mix of red and white colonies.
The background control also showed colony growth, which was not expected.
Upon further investigation, it was discovered the XbaI and SpeI have complementary sticky ends which could result in re-circularisation of the vector backbone to itself. This explains the presence of white colonies on the RFP positive control and the growth on the background control. To identify if this has occurred, another double digest will be carried out.
The samples A (TodX), C (TodF) and E(ToBG) all showed colony growth for the 20 ul and the 200 ul plates.</DIV><DIV id="Sep9"><H3>Miniprep of overnight broth</H3>
        Following on from 06/09/13 result,overnight broth was prepared yesterday from the single colonies that had grown on each of the 2 plates shown on 06/09/13.
The broth, which contains cells that have been transformed with pSB1C3/TOD genes, was minipreped using the Omega Bio-Tek kit.
This would enable us to confirm which cells have the recircularised plasmid and those with the ligated product.<H3>Measurement of DNA concentration</H3><TABLE><TBODY><TR><TH>Sample</TH><TH>Volume</TH><TH>Concerntration ng/ul</TH><TH>260/280</TH><TH>260/230</TH></TR><TR><TD>TOD X1</TD><TD>31.6</TD><TD>86.5</TD><TD>1.87</TD><TD>1.92</TD></TR><TR><TD>TOD X2</TD><TD>31.7</TD><TD>58.7</TD><TD>1.89</TD><TD>1.26</TD></TR><TR><TD>TOD X3</TD><TD>32.7</TD><TD>56.6</TD><TD>1.91</TD><TD>1.88</TD></TR><TR><TD>TOD X4</TD><TD>31.6</TD><TD>55.8</TD><TD>1.88</TD><TD>1.65</TD></TR><TR><TD>TOD X5</TD><TD>31.7</TD><TD>44.4</TD><TD>1.87</TD><TD>1.78</TD></TR><TR><TD>TOD X6</TD><TD>31.7</TD><TD>70.5</TD><TD>1.88</TD><TD>1.88</TD></TR><TR><TH>Sample</TH><TH>Volume</TH><TH>Concerntration ng/ul</TH><TH>260/280</TH><TH>260/230</TH></TR><TR><TD>TOD F1</TD><TD>32.4</TD><TD>39.6</TD><TD>1.91</TD><TD>1.59</TD></TR><TR><TD>TOD F2</TD><TD>33.7</TD><TD>49.4</TD><TD>1.91</TD><TD>1.65</TD></TR><TR><TD>TOD F3</TD><TD>32.3</TD><TD>47.4</TD><TD>1.89</TD><TD>1.71</TD></TR><TR><TD>TOD F4</TD><TD>31.6</TD><TD>43.7</TD><TD>1.94</TD><TD>1.73</TD></TR><TR><TD>TOD F5</TD><TD>36.5</TD><TD>57</TD><TD>1.89</TD><TD>1.50</TD></TR><TR><TD>TOD F6</TD><TD>38</TD><TD>59.4</TD><TD>1.89</TD><TD>1.61</TD></TR><TR><TD>TOD G1</TD><TD>36.8</TD><TD>72.5</TD><TD>1.89</TD><TD>1.51</TD></TR><TR><TD>TOD G2</TD><TD>44.7</TD><TD>1.90</TD><TD>1.90</TD><TD>1.65</TD></TR><TR><TD>TOD G3</TD><TD>33.7</TD><TD>50.2</TD><TD>1.90</TD><TD>1.46</TD></TR><TR><TD>TOD G4</TD><TD>41.9</TD><TD>55</TD><TD>1.93</TD><TD>1.35</TD></TR><TR><TD>TOD G5</TD><TD>34</TD><TD>43</TD><TD>1.96</TD><TD>1.54</TD></TR><TR><TD>TOD G6</TD><TD>35</TD><TD>48</TD><TD>1.89</TD><TD>1.72</TD></TR></TBODY></TABLE>
     DNA concentration was measured using Nanodrop.
        <H3>Double Digestion of pSB1C3/TOD genes</H3>
        The pSB1C3/TOD genes were double digested with (XbaI and SpeI) &amp; (EcoRI and PstI)
The master mix for the (XbaI and SpeI) was;
60ul of cut smart buffer 5ul of XbaI 10ul of SpeI 273ul of 5M Tris
The master mix for the (EcoRI and PstI) was;
60ul of cut smart buffer 5ul of XbaI 5ul of SpeI 278ul of 5M Tris
All 36 tubes were incubated overnight in 37 degrees room.
Tomorrow the plasmids that were isolated from the miniprep would be run on a 1% agarose gel to distinguish between cells that have the pSB1C3/TOD genes from those that have the recircularised pSB1C3 vector.
Meanwhile the plasmids from the miniprep were sent for sequencing this afternoon.<H3>Dissolution of polystyrene in Limonene</H3>
        This experiment was set up to decide how much limonene can economically dissolve polystyrene.
MATERIALS USED
20ml solution of 25% limonene.
8cm x 5cm x 2cm polystyrene block (thus volume of 80cm^3)
RESULT
It took ~33 minutes for the block to be completely dissolved</DIV><DIV id="Sep10"><H3>Dissolution of polystyrene</H3>
        a 20ml solution of 50% Limonene was made to determine if the dissolution of polystyrene from 09/09/13 would occur faster.
as predicted, the 8cm x 5cm x 2cm PS block was completely dissolved in 9 minutes.<H3>Gel electrophoresis</H3>
        The samples from the double digest on 09/09/13 were run to confirm if the digestion worked.
From the results, we would also be able to distinguish between cells that have been successfully transformed with the ligated pSB1C3/TOD genes from those with the recircularised pSB1C3.<B>Results of the Gel electrophoresis</B>
TODX Gel
Wells order: TODX1 (SpeI/XbaI), TODX1 (EcoRI/PstI); TODX2 (SpeI/XbaI), TODX2 (EcoRI/PstI); TODX3 (SpeI/XbaI), TODX3 (EcoRI/PstI); TODX4 (SpeI/XbaI), TODX4 (EcoRI/PstI); TODX5 (SpeI/XbaI), TODX5 (EcoRI/PstI); TODX6 (SpeI/XbaI), TODX6 (EcoRI/PstI).
The expected results were that the we would see the TODX genes drop off in separate band in all the digestions in its respective size (600bp). However as it can be seen in the picture above only the samples cut with SpeI and XbaI had two fragments and they are the wrong size (~ 1Kb). The samples TODX (1, 2 and 6) were sent to be sequenced before the digestions and it showed that the pSB1C3 plasmid re-circularized. This is because the two enzymes have complementary sticky ends.
TODF
Wells order: TODF1 (SpeI/XbaI), TODF1 (EcoRI/PstI); TODF2 (SpeI/XbaI), TODF2 (EcoRI/PstI); TODF3 (SpeI/XbaI), TODF3 (EcoRI/PstI); TODF4 (SpeI/XbaI), TODF4 (EcoRI/PstI); TODF5 (SpeI/XbaI), TODF5 (EcoRI/PstI); TODF6 (SpeI/XbaI), TODF6 (EcoRI/PstI).
This shows the same results as TODX above.
TOBG
Wells order: TOBG1 (SpeI/XbaI), TOBG1 (EcoRI/PstI); TOBG2 (SpeI/XbaI), TOBG2 (EcoRI/PstI); TOBG3 (SpeI/XbaI), TOBG3 (EcoRI/PstI); TOBG4 (SpeI/XbaI), TOBG4 (EcoRI/PstI); TOBG5 (SpeI/XbaI), TOBG5 (EcoRI/PstI); TOBG6 (SpeI/XbaI), TOBG6 (EcoRI/PstI).
This shows the same results as TODX above.<H3>Nanodrop of the pGEM-T plasmids that were ligated with the TOD operon genes and the pSB1C3 backbone</H3><TABLE><TBODY><TR><TH>Sample</TH><TH>Volume</TH><TH>Concerntration ng/ul</TH><TH>260/280</TH><TH>260/230</TH></TR><TR><TD>TodX 1</TD><TD>25.8</TD><TD>37.4</TD><TD>1.84</TD><TD>1.52</TD></TR><TR><TD>TodX 2</TD><TD>35</TD><TD>61.3</TD><TD>1.82</TD><TD>1.74</TD></TR><TR><TD>TodF 1</TD><TD>37.8</TD><TD>99.4</TD><TD>1.83</TD><TD>1.99</TD></TR><TR><TD>TodF 2</TD><TD>22.7</TD><TD>25.7</TD><TD>1.76</TD><TD>1.60</TD></TR><TR><TD>ToBG 1</TD><TD>31.4</TD><TD>51.7</TD><TD>1.84</TD><TD>1.57</TD></TR><TR><TD>ToBG 2</TD><TD>20.1</TD><TD>21.9</TD><TD>1.83</TD><TD>1.26</TD></TR><TR><TD>pSB1C3 1</TD><TD>10.3</TD><TD>31.8</TD><TD>1.65</TD><TD>1.44</TD></TR><TR><TD>pSB1C3 1</TD><TD>19</TD><TD>40.9</TD><TD>1.81</TD><TD>1.71</TD></TR></TBODY></TABLE><H3>Double Digestions of the plasmids above</H3><TABLE><TBODY><TR><TH>Sample</TH><TH>DNA Volume</TH><TH>Buffer</TH><TH>Xbal</TH><TH>Pstl</TH><TH>dH2O to reach final volume of 60 ul</TH></TR><TR><TD>TodX </TD><TD>45.5</TD><TD>6</TD><TD>1</TD><TD>1</TD><TD>6.5</TD></TR><TR><TD>TodF </TD><TD>49.6</TD><TD>6</TD><TD>1</TD><TD>1</TD><TD>2.4</TD></TR><TR><TD>TodG</TD><TD>42.9</TD><TD>6</TD><TD>1</TD><TD>1</TD><TD>9.1</TD></TR><TR><TD>pSB1C3</TD><TD>25</TD><TD>6</TD><TD>1</TD><TD>1</TD><TD>27</TD></TR></TBODY></TABLE></DIV><DIV id="Sep11"><H3>Gel electropheresis of the double digestion performed in the 10/09/13</H3>
     	The picture above shows that the double digestion hasn't worked as expected. Different reasons of why it hasn't worked were discussed and the reason agreed was that the PStI enzyme was not working properly.
     </DIV><DIV id="Sep12"><H3>Isolate plasmid from overnight culture</H3>
        Using Omega mini prep kit protocol
Samples:
TODF 1,2,3,
TODX 1, 2, 3
TOBG 1, 2, 3,
RFP 1, 2, 3<TABLE><TBODY><TR><TH>Sample</TH><TH>Volume (ul)</TH><TH>Concerntration (ng/ul)</TH></TR><TR><TD>TODF1</TD><TD>43</TD><TD>120</TD></TR><TR><TD>TODF2</TD><TD>43</TD><TD>98.8</TD></TR><TR><TD>TODX1</TD><TD>42</TD><TD>399.3</TD></TR><TR><TD>TODX2</TD><TD>44</TD><TD>442.2</TD></TR><TR><TD>TODX3</TD><TD>43</TD><TD>417.5</TD></TR><TR><TD>TODG1</TD><TD>43</TD><TD>262.4</TD></TR><TR><TD>TODG2</TD><TD>47</TD><TD>319.2</TD></TR><TR><TD>TODG3</TD><TD>41</TD><TD>338.4</TD></TR><TR><TD>RFP1</TD><TD>41</TD><TD>77.7</TD></TR><TR><TD>RFP2</TD><TD>41</TD><TD>32.7</TD></TR><TR><TD>RFP3</TD><TD>42</TD><TD>59</TD></TR></TBODY></TABLE><H3>Glycerol stocks</H3>
        Took 750ul from overnight culture and centrifuged at full speed for ~2 minutes
Supernatant was removed
Pellet resuspended in 375ul of HMFM
Samples were then frozen at -80<H3>Gel Purification of the pGEM-T pasmid double digestions ran in the 11/09/13</H3>
        The gel was purified by using the The Zymoclean Gel DNA recovery Kit and its protocol was followed.
Nanodrop Results
         <TABLE><TBODY><TR><TH>Sample</TH><TH>Volume</TH><TH>Concerntration ng/ul</TH><TH>260/280</TH><TH>260/230</TH></TR><TR><TD>TodX</TD><TD>48.5</TD><TD>7.1</TD><TD>1.83</TD><TD>0.72</TD></TR><TR><TD>Todf</TD><TD>45.5</TD><TD>21.7</TD><TD>1.89</TD><TD>1.19</TD></TR><TR><TD>TodBG/td&gt;
                </TD><TD>47.7</TD><TD>9</TD><TD>1.77</TD><TD>0.70</TD></TR><TR><TD>pSB1C3 (RFP)</TD><TD>46.2</TD><TD>7.3</TD><TD>1.81</TD><TD>0.16</TD></TR></TBODY></TABLE><H3>Digestion of the samples purified with the enzyme PstI-HF and XbaI</H3>
        Two single digestions were set up to test if one of the enzymes was not working properly.
        <TABLE><TBODY><TR><TH>DNA volume (ul)</TH><TH>Volume</TH><TH>Custmart Buffer</TH><TH>enzyme volume</TH><TH>dH20 added to total volume of 60 ul</TH></TR><TR><TD>TODX</TD><TD>23</TD><TD>6</TD><TD>1</TD><TD>30</TD></TR><TR><TD>TODF</TD><TD>22.5</TD><TD>6</TD><TD>1</TD><TD>30.5</TD></TR><TR><TD>TOBG</TD><TD>22</TD><TD>6</TD><TD>1</TD><TD>31</TD></TR><TR><TD>pSB1C3 (RFP)</TD><TD>20.5</TD><TD>6</TD><TD>1</TD><TD>32.5</TD></TR></TBODY></TABLE><B>Gel image of the above double digestion</B>
        Gel order: TODX, TODX, TODF, TODF, TOBG, TOBG, pSB1C3, pSB1C3 (XbaI); TODX, TODF, TOBG, pSB1C3 (PstI)
The picture above shows that the assumption that one of the enzymes did not working properly was wrong, as it shows the same result as the gel run on the 11/09/13.<H3>Double digestion of the isolated pGEM plasmids ligated with the TOD operon genes (X, F and TOBG)</H3>
     	As the digestions with the plasmids that were stored in the freezer did not work new digestins were set up with the isolated plasmids from the overnight culture.
The pGEM-T plasmid contain the restrictions sites for PstI and SpeI, which are also present in the amplified TOD operon gene and the pSB1C3 plasmid.
To avoid the recircularization of the pSB1C3 backbone when ligating the genes with this plasmid, the double digestion was set up with the enzymes PstI and XbaI.
Another double digestion with EcoRI and SpeI was set up to test if the restrictions sites present in the TOD genes are not mutated. If the bands in the gel shows a linear band, it can be assumed that the enzymes cut the pGEM plasmids in its PstI and SpeI restriction sites.<B>Master Mix of the PstI/XbaI double digestion</B>
30 ul of cut samart buffer
5 ul of PstI
5 ul of XbaI
58 ul of dH2O
9.8 ul of the master mix were added to each reaction.
2 ug DNA of each sample were digested.
Protocol<TABLE><TBODY><TR><TH>Sample</TH><TH>DNA volume (ul)</TH><TH>dH20 (ul) addded to the final volume of 30 ul</TH></TR><TR><TD>TODF1</TD><TD>16.5</TD><TD>3.5</TD></TR><TR><TD>TODF2</TD><TD>20.2</TD><TD>0</TD></TR><TR><TD>TODX1</TD><TD>5</TD><TD>15.2</TD></TR><TR><TD>TODX2</TD><TD>4.2</TD><TD>16</TD></TR><TR><TD>TODFX3</TD><TD>4.8</TD><TD>15.4</TD></TR><TR><TD>TODFG1</TD><TD>7.6</TD><TD>12.6</TD></TR><TR><TD>TODFG2</TD><TD>6.3</TD><TD>13.9</TD></TR><TR><TD>TODFG3</TD><TD>5.9</TD><TD>14.3</TD></TR></TBODY></TABLE><B>Master Mix of the EcoRI/SpeI double digestion</B>
Only samples TODX1, TODX2 and TODX3 were used for this digestion
15 ul of cutsmart buffer
2.5 ul of EcoRI
2.5 ul of SpeI
106 ul of dH2O
25 ul of Master Mix was added to each reaction.<TABLE><TBODY><TR><TH>Sample</TH><TH>DNA volume (ul)</TH><TH>dH20 (ul) addded to the final volume of 30 ul</TH></TR><TR><TD>TODX1</TD><TD>5</TD><TD>25</TD></TR><TR><TD>TODX2</TD><TD>4.2</TD><TD>0.8</TD></TR><TR><TD>TODFX3</TD><TD>4.8</TD><TD>0.2</TD></TR></TBODY></TABLE>
The digestion was left overnight in the 37C room.
Heat kill at 80C for 20 minutes.<H3>Chemistry Lab Results</H3>
     	Two experiments were run in the chemistry lab.
Polystyrene Combustion
Two pieces of normal packaging polystyrene were burnt, which showed the flammability of polystyrene.
After two blocks of polystyrene made form the sugar given to the Leicester iGEM team by Jablite. One of them had DNA incorporated and the other had no DNA. We were able to establish that the polystyrene sugar provided to us from Jablite already had fire retardant chemicals incorporated, which did not allow us to test if DNA could work as a fire retardant for polystyrene.
Polystyrene dissolution in limonene</DIV><DIV id="Sep13"><H3>Gel electropheresis of pGEM-T vectors with the TOD genes</H3>
        Gel order: TodX1, TodX2, TodX3, TodF1, TodF2, TobG1, TobG2, TobG3 (all cut with (XbaI/PstI); TodX1, TodX2, TodX3 (this were cut with EcoRI/SpeI)
The gel shows that once again the expected genes were not cut out from the plasmid. This could be because the restriction sites were mutated when they were cloned.
     </DIV><DIV id="Sep16"><H3>Digesting pSB1C3 backbone</H3>
        Digesting for ligation with the already digested Tod operon PCR products
Following the iGEM official protocol
Using EcoRI-HF and PstI-HF
Exluding the DpnI from the master mix and adjusting the volume of water accordingly
Protocol can be found at: [1]<H3>Ligation of digested pSB1C3 backbone to Tod F, X and Tob G</H3>
Using Promega protocol to which changes were made
<TABLE><TBODY><TR><TH>Reagents</TH><TH>Tod F</TH><TH>Tod X</TH><TH>Tod G</TH><TH>Postive Control</TH><TH>Background Control</TH></TR><TR><TD>2x Rapid Ligation buffer</TD><TD>5ul</TD><TD>5ul</TD><TD>5ul</TD><TD>5ul</TD><TD>5ul</TD></TR><TR><TD>pSB1C3 digest (50ng)</TD><TD>2ul</TD><TD>2ul</TD><TD>2ul</TD><TD>2ul</TD><TD>2ul</TD></TR><TR><TD>PCR Product</TD><TD>3.3ul</TD><TD>3ul</TD><TD>3.8ul</TD><TD>-</TD><TD>-</TD></TR><TR><TD>Control insert DNA</TD><TD>-</TD><TD>-</TD><TD>-</TD><TD>2ul</TD><TD>-</TD></TR><TR><TD>T4 DNA ligase</TD><TD>1ul</TD><TD>1ul</TD><TD>1ul</TD><TD>1ul</TD><TD>1ul</TD></TR><TR><TD>Deinoized water for final volume of 10ul</TD><TD>-</TD><TD>-</TD><TD>-</TD><TD>-</TD><TD>1ul</TD></TR></TBODY></TABLE>
Mix reactions by pipetting. Incubate the reactions for 1h at room temperature.
Control insert DNA from the transformation efficiency kit provided by iGEM<H3>Transformation of TOD genes ligations with the digested pSB1C3 using the pGEM-T Vector System I-A3600 (changes were made to the protocol)</H3>
Centrifuge the ligations reactions briefly.
Add 5ul of each ligation reaction to a sterile 1.5ml
Thaw the competent cells on ice. Mix the cells by gently flicking the tube.
Carefully transfer 50ul of the competent cells to the ligation reaction tubes.
Incubate the tubes on ice for 20 minutes
Heat-shock the cells for 45-50 seconds in a water bath at exactly 42C degrees. Do not Shake. Immediately return the tubes to ice for 2 minutes.
Add 900ul of room temperature SOC medium to the ligation reaction transformations. incubate for 1.5 hours at 37C with shaking.<H3>Digestion of the TOD genes PCR products</H3>
The PCR products of TODX, TODF and TOBG were digested with the enzymes EcoRI-HF and PstI-HF.
Protocol<TABLE><TBODY><TR><TH>Sample</TH><TH>DNA volume (ul)</TH><TH>custmart buffer (ul)</TH><TH>EcoRl (ul)</TH><TH>Pstl (ul)</TH><TH>dH20 (ul)</TH></TR><TR><TD>TODX</TD><TD>8.7</TD><TD>2</TD><TD>0.5</TD><TD>0.5</TD><TD>8.3</TD></TR><TR><TD>TODF</TD><TD>8.3</TD><TD>2</TD><TD>0.5</TD><TD>0.5</TD><TD>8.7</TD></TR><TR><TD>TOBG</TD><TD>8</TD><TD>2</TD><TD>0.5</TD><TD>0.5</TD><TD>9</TD></TR></TBODY></TABLE>
The samples were incubated for 1 hour at 37C.
Heat kill the enzymes at 80C for 20 minutes.<H3>Running the backbone and RFP on gel</H3>
on 1% gel to check if the gel extraction worked.
gel shows that extraction worked
2kb band representing the backbone is visible as well as 1kb representing the RFP<H3>Digestion of the pGEM plasmid gel extracted on the 13/09/13</H3>
The samples were digested with the enzymes EcoRI-HF and PstI-HF
Master Mix for 8 reactions
16 ul of cutsmart buffer
4 ul of EcoRI
4 ul of PstI
88 ul of dH2O
14 ul of master mix was added to each reaction</DIV><DIV id="Sep17"><H3>Results from previous day</H3>
        Background control plate had growth, which was not expected. This suggests that pSB1C3 backbone religated.
It was also thought that the Tod genes used were not from the PCR fusion experiment.
Same experiment done again, with a new set of Tod genes from a PCR fusion.
For digestion of backbone, already digested backbone from 01/08/2013 was used, which was digested with EcoRI and PstI.<H3>Incubation of bacteria from the plates of previous day for mini prep</H3>
12 single colonies were picked from each plate and put into 15ml of broth
36 tubes were left incubating overnight at 37C<H3>Sending pGEM-T vector with Tod insert clones to sequencing by PNACL</H3><H3>Digestion of new fusion PCR Tod genes</H3>
Digestion with EcorI-HF and PstI-HF using Cutsmart buffer
500ng of DNA was digested from the following Tod DNA concentrations:<TABLE><TBODY><TR><TH>Sample</TH><TH>Concerntration ng/ul</TH><TH>260/280</TH><TH>260/230</TH></TR><TR><TD>TodX</TD><TD>49.5</TD><TD>1.89</TD><TD>1.99</TD></TR><TR><TD>Todf</TD><TD>33.9</TD><TD>1.98</TD><TD>1.68</TD></TR><TR><TD>Tod B</TD><TD>37.4</TD><TD>1.82</TD><TD>1.70</TD></TR></TBODY></TABLE>
     Volumes for double digest:
     <TABLE><TBODY><TR><TH>Sample</TH><TH>DNA</TH><TH>Buffer</TH><TH>Water</TH><TH>EcorI-HF</TH><TH>PstI-HF</TH></TR><TR><TD>Tod X</TD><TD>10.1ul</TD><TD>3ul</TD><TD>15.9</TD><TD>0.5</TD><TD>0.5</TD></TR><TR><TD>Tod F</TD><TD>13.4ul</TD><TD>3ul</TD><TD>12.6</TD><TD>0.5</TD><TD>0.5</TD></TR><TR><TD>Tod B</TD><TD>14.7ul</TD><TD>3ul</TD><TD>11.3</TD><TD>0.5</TD><TD>0.5</TD></TR></TBODY></TABLE>
     Incubation at 37C for 30min
Heat kill at 80C for 20min<H3>Ligation of Tod genes and pSB1C3 backbone</H3>
For background control, just the pSB1C3 backbone was used
For positive control, RFP plasmid from transformation efficiency kit provide by iGEM was used
Quick stick ligase was used and a different protocol from previous day
Bioline Quick Stick Ligase protocol:
Combine the vector and the insert in the appropriate ratio to make up no more than 100ng of DNA
Adjust volume to 14ul with ddH2O
Add 1ul of QS ligase
Add 5ul of 4x QS Buffer (vortex before use)
Mix thoroughly before pipetting
Incubate at room temperature for 5min
The volumes for ligation:<TABLE><TBODY><TR><TH>Sample</TH><TH>DNA ul</TH><TH>Vector(pSB1C3) ul</TH><TH>QS ligase ul</TH><TH>Buffer ul</TH><TH>H20 ul</TH></TR><TR><TD>Tod X</TD><TD>0.9</TD><TD>2</TD><TD>1</TD><TD>5</TD><TD>5.1</TD></TR><TR><TD>Tod F</TD><TD>0.8</TD><TD>2</TD><TD>1</TD><TD>5</TD><TD>5.2</TD></TR><TR><TD>Tod B</TD><TD>1</TD><TD>2</TD><TD>1</TD><TD>5</TD><TD>5</TD></TR><TR><TD>Postive Control</TD><TD>1</TD><TD>2</TD><TD>1</TD><TD>5</TD><TD>5</TD></TR><TR><TD>Background Control</TD><TD>1</TD><TD>2</TD><TD>1</TD><TD>5</TD><TD>5</TD></TR></TBODY></TABLE><H3>Transformation of TOD genes ligations with the digested pSB1C3 using the pGEM-T Vector System I-A3600 (changes were made to the protocol)</H3>
Centrifuge the ligations reactions briefly.
Add 5ul of each ligation reaction to a sterile 1.5ml
Thaw the competent cells on ice. Mix the cells by gently flicking the tube.
Carefully transfer 50ul of the competent cells to the ligation reaction tubes.
Incubate the tubes on ice for 20 minutes
Heat-shock the cells for 45-50 seconds in a water bath at exactly 42C degrees. Do not Shake. Immediately return the tubes to ice for 2 minutes.
Add 900ul of room temperature SOC medium to the ligation reaction transformations. incubate for 1 hour at 37C with shaking.<H3>Plating out transformants on chloroamphenicol plates</H3>
Plating out a total of 10 plates, 2 for each sample
20ul and 200ul were plated from each sample
Plates were left over night at 37C.
     </DIV><DIV id="Sep18"><H3>Making and running an agarose gel</H3>
     Making a 3% agarose gel to run Tod PCR fusion genes from 17/09
This is to determine that the digestion works
Running the gel with digested and undigested samples to observe the difference
Digested samples were taken from the digestions the Tod genes from previous day
100ng of both samples were run against a 100kb ladder
Rows are as follow:100bp ladder, undigested x, digested x, undigested b, digested b, undigested f, digested f

For selection of individual colonies
     </DIV><DIV id="Sep19"><H3>DNA isolation from PCR products</H3>
        Isolating DNA using Omega nucleic acid purification kit and following its protocol.
Isolated DNA from all of the samples B and F and samples X3, 5, 6, 7, 8, 10, 12.
Selection was made by looking at gel pictures and selecting samples with visible bands.<H3>Nanodropping the samples for concentration</H3><TABLE><TBODY><TR><TH>Sample nr</TH><TH>Volume ul</TH><TH>Concerntration ng/ul</TH><TH>260/280</TH><TH>260/230</TH></TR><TR><TD>X3</TD><TD>41</TD><TD>11.3</TD><TD>1.87</TD><TD>1.29</TD></TR><TR><TD>X5</TD><TD>47</TD><TD>12.2</TD><TD>1.78</TD><TD>1.22</TD></TR><TR><TD>X6</TD><TD>34</TD><TD>12.9</TD><TD>1.79</TD><TD>1.22</TD></TR><TR><TD>X7</TD><TD>45</TD><TD>8</TD><TD>1.71</TD><TD>0.96</TD></TR><TR><TD>X8</TD><TD>43</TD><TD>10.5</TD><TD>1.88</TD><TD>1.2</TD></TR><TR><TD>X10</TD><TD>44</TD><TD>9</TD><TD>1.88</TD><TD>1.2</TD></TR><TR><TD>X12</TD><TD>45</TD><TD>12.8</TD><TD>1.70</TD><TD>1.40</TD></TR><TR><TD>F1</TD><TD>44.5</TD><TD>11.2</TD><TD>1.69</TD><TD>1.22</TD></TR><TR><TD>F2</TD><TD>44</TD><TD>9.1</TD><TD>1.88</TD><TD>1.20</TD></TR><TR><TD>F3</TD><TD>45.2</TD><TD>10.6</TD><TD>1.77</TD><TD>0.99</TD></TR><TR><TD>F4</TD><TD>43.7</TD><TD>9.7</TD><TD>1.82</TD><TD>1.10</TD></TR><TR><TD>F5</TD><TD>45.2</TD><TD>10.6</TD><TD>1.67</TD><TD>1.19</TD></TR><TR><TD>F6</TD><TD>45.4</TD><TD>9.6</TD><TD>1.82</TD><TD>0.85</TD></TR><TR><TD>F7</TD><TD>46</TD><TD>11</TD><TD>1.76</TD><TD>1.14</TD></TR><TR><TD>F8</TD><TD>46</TD><TD>11.7</TD><TD>1.73</TD><TD>1.24</TD></TR><TR><TD>F9</TD><TD>42.4</TD><TD>10.5</TD><TD>1.83</TD><TD>1.32</TD></TR><TR><TD>F10</TD><TD>46</TD><TD>11</TD><TD>1.93</TD><TD>1.39</TD></TR><TR><TD>F11</TD><TD>45.5</TD><TD>10.3</TD><TD>1.82</TD><TD>1.18</TD></TR><TR><TD>F12</TD><TD>45.5</TD><TD>14.9</TD><TD>1.83</TD><TD>1.32</TD></TR><TR><TD>B1</TD><TD>45</TD><TD>6.9</TD><TD>1.56</TD><TD>0.86</TD></TR><TR><TD>B2</TD><TD>46</TD><TD>10.5</TD><TD>1.64</TD><TD>1.17</TD></TR><TR><TD>B3</TD><TD>46</TD><TD>10</TD><TD>1.78</TD><TD>1.41</TD></TR><TR><TD>B4</TD><TD>47</TD><TD>11.5</TD><TD>1.70</TD><TD>1.35</TD></TR><TR><TD>B5</TD><TD>47</TD><TD>10.4</TD><TD>1.84</TD><TD>1.25</TD></TR><TR><TD>B6</TD><TD>47</TD><TD>8.6</TD><TD>1.96</TD><TD>1.33</TD></TR><TR><TD>B7</TD><TD>47</TD><TD>9.7</TD><TD>1.74</TD><TD>1.01</TD></TR><TR><TD>B8</TD><TD>46</TD><TD>12.4</TD><TD>1.58</TD><TD>1.22</TD></TR><TR><TD>B9</TD><TD>45</TD><TD>10.5</TD><TD>1.58</TD><TD>1.05</TD></TR><TR><TD>B10</TD><TD>46</TD><TD>11.1</TD><TD>1.73</TD><TD>1.32</TD></TR><TR><TD>B11</TD><TD>46</TD><TD>10.2</TD><TD>1.87</TD><TD>1.56</TD></TR><TR><TD>B12</TD><TD>46</TD><TD>12.1</TD><TD>1.75</TD><TD>1.38</TD></TR></TBODY></TABLE><H3>Sending isolated DNA to be sequenced by PNACL</H3>
Sending 4 samples from each operon that showed highest DNA concentration: B5, 4, 9, 12; F8, 1, 10, 12; X3, 5, 6, 12.<H3>Preparing overnight broths</H3>
Preparing broths from individual colonie plates from previous day
Selecting 4 colonies from each that showed highest DNA concentration from nanodrop: B5, 4, 9, 12; F8, 1, 10, 12; X3, 5, 6, 12.</DIV><DIV id="Sep20"><H3>Miniprep from overnight broths</H3>
        DNA is isolated using Omega Miniprep kit and its protocol is followed
        <H3>Nanodropping the isolated DNA</H3><TABLE><TBODY><TR><TH>Sample</TH><TH>Volume</TH><TH>Concerntration ng/ul</TH><TH>260/280</TH><TH>260/230</TH></TR><TR><TD>X12</TD><TD>90</TD><TD>13.5</TD><TD>1.86</TD><TD>1.38</TD></TR><TR><TD>X3</TD><TD>43</TD><TD>13.6</TD><TD>2.12</TD><TD>1.34</TD></TR><TR><TD>X5</TD><TD>45</TD><TD>16</TD><TD>2.08</TD><TD>1.64</TD></TR><TR><TD>X6</TD><TD>44</TD><TD>19.1</TD><TD>2.04</TD><TD>1.59</TD></TR><TR><TD>B4</TD><TD>93</TD><TD>11.1</TD><TD>1.73</TD><TD>1.28</TD></TR><TR><TD>B5</TD><TD>87</TD><TD>9.5</TD><TD>1.99</TD><TD>1.34</TD></TR><TR><TD>B9</TD><TD>43</TD><TD>13.1</TD><TD>2.14</TD><TD>1.43</TD></TR><TR><TD>B12</TD><TD>43</TD><TD>14.9</TD><TD>1.99</TD><TD>1.34</TD></TR><TR><TD>F1</TD><TD>92</TD><TD>17.6</TD><TD>1.91</TD><TD>1.15</TD></TR><TR><TD>F8</TD><TD>93</TD><TD>17.6</TD><TD>1.85</TD><TD>1.70</TD></TR><TR><TD>F10</TD><TD>91</TD><TD>12.4</TD><TD>1.99</TD><TD>1.30</TD></TR><TR><TD>F12</TD><TD>41</TD><TD>18.5</TD><TD>2.06</TD><TD>1.71</TD></TR></TBODY></TABLE><H3>Double digest of isolated DNA</H3>
        Digesting with EcorI-HF/PstI-HF and XbaI/PstI to see if DNA incorporated insert has the biobrick prefix and suffix present.
Gel to be run in the 23/09/13
     </DIV><DIV id="Sep23"><H3>Gel Results from the digestion performed on the 20/09/13</H3>
        Gel Order: X3 (E/P), X3 (X/P); X5 (E/P), X5 (X/P); X6 (E/P), X6 (X/P); X12 (E/P), X12 (X/P); F1 (E/P), F1 (X/P);
The gel shows that digestions were successful as every sample dropped the inserted gene.

Gel Order:
F8 (E/P), F8 (X/P); F10 (E/P), F10 (X/P); F12 (E/P), F12 (X/P); B4 (E/P), B4 (X/P); B5 (E/P), B5 (X/P); B9 (E/P), B9 (X/P); B12 (E/P), B12 (X/P). 
Same result as the gel above
<H3>Evaporation of isolated plasmids and re-suspended in 1xTE</H3>
The samples were evaporated with a vacuum evaporator.
The samples were re-suspended in 20 ul of 1xTE.<TABLE><TBODY><TR><TH>Sample</TH><TH>Volume</TH><TH>Concerntration ng/ul</TH><TH>260/280</TH><TH>260/230</TH></TR><TR><TD>X3</TD><TD>20</TD><TD>12.1</TD><TD>1.49</TD><TD>1.44</TD></TR><TR><TD>X5</TD><TD>20</TD><TD>15.6</TD><TD>1.63</TD><TD>1.45</TD></TR><TR><TD>X6</TD><TD>20</TD><TD>25.2</TD><TD>1.63</TD><TD>1.46</TD></TR><TR><TD>B4</TD><TD>20</TD><TD>12.1</TD><TD>1.49</TD><TD>1.44</TD></TR><TR><TD>B9</TD><TD>20</TD><TD>16.3</TD><TD>1.63</TD><TD>1.45</TD></TR><TR><TD>B12</TD><TD>20</TD><TD>26.3</TD><TD>1.87</TD><TD>1.05</TD></TR><TR><TD>F1</TD><TD>20</TD><TD>38.9</TD><TD>1.83</TD><TD>1.57</TD></TR><TR><TD>F10</TD><TD>20</TD><TD>33.6</TD><TD>1.67</TD><TD>1.4</TD></TR><TR><TD>F12</TD><TD>20</TD><TD>6.5</TD><TD>1.48</TD><TD>1.10</TD></TR></TBODY></TABLE><H3>Luria Agar plates</H3>
     Luria agar plates were made for the plating of the CSE kits that arrived.

     </DIV><DIV id="Sep24"><H3>Plating out the CSE kits</H3>
        The CSE kits that arrived yesterday were plated out. The polystyrene and the miscellaneous part were plated on two plates,one was incubated at room temperature and the other at 37C.
        <H3>Overnight cultures</H3>
        Overnight cultures of the biobricks samples were set up to prepare glycerol cultures tomorrow. The samples were F1, F10, F12, X3,X5, X6, B4, B9 and B12. These samples were sequenced and they had no mutations.
     </DIV><DIV id="Sep25"><H3>Glycerol Stocks</H3>
     Glycerol stocks were made from the samples F1, F10, F12, X3,X5, X6, B4, B9 and B12. These samples were sequenced and they had no mutations.
     </DIV></DIV><DIV><A href="https://2013.igem.org/wiki/index.php?title=Team:Leicester/Notebook&amp;action=edit">EDIT</A></DIV></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:Leicester/Notebook">http://2013.igem.org/Team:Leicester/Notebook</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Leicester/Notebook" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Leicester/Notebook" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Leicester/Notebook&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Leicester/Notebook&amp;oldid=310655" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>