"
Page
Discussion
View source
History
teams
Log in
 
Team:Leicester/Project
From 2013.igem.org
iGEM Leicester Test Page 2012
Home
Team
Project
Official Team Profile
Parts Submitted to Registry
Human Practises
Notebook
Safety
Attributions
Our Project
Polystyrene is a multifaceted material - light, largely chemically and biologically inert, and easily shaped, but also a highly visible pollutant that locks up valuable hydrocarbons. In 2013 the Leicester iGEM team are retaining their focus on polystyrene but will diversify our approach. We have three streams involving recycling, re-using and re-purposing polystyrene.
Recycling – This stream involves building on the work of the 2012 team in environmental prospecting for genes involved in polystyrene degradation. Currently we have isolated organisms from environmentally exposed polystyrene, and are progressing in our aim of adapting the toluene degradation pathway of Pseudomonas species.
Re-using - Consumer 3D printers are now a reality and use a variety of thermoplastics (PS and ABS, for example). While the technology is constantly finding new uses, most involve using virgin plastics, which is not good for the environment. Recycled polystyrene can be used for 3D printing, but has an additional advantage as it is soluble in limonene (an environmentally friendly solvent), but ABS is not. Complex 3D printed shapes require removable support structures - one solution is to print the supports using PS and the object in ABS, with the PS being removed by limonene dissolution. We are adapting limonene biosynthesis biobricks developed in previous iGEM competitions to enable genetically engineered machine biological "finishing" of 3D printed objects.
Re-purposing (DNA!) - Expanded polystyrene (EPS) is a great insulator, used by the construction industry to make our homes warmer, using less energy. EPS insulation is required to be flame retardant, which is currently achieved by incorporating environmentally polluting halogenated hydrocarbons. Recently DNA has been shown to be an effective flame retardant, but is expensive to produce. We are building a genetically engineered machine with inducible endoreplication (over-replication of DNA). This should yield DNA cheap enough to burn and when added to EPS make it flame retardant and environmentally friendly.
Recycling
The first of the three streams was focused on amplifying and modifying the genes of the tod operon found in the Pseudomonas putida F1 strain. The dioxygenase enzyme encoded by the tod operon degrades toluene which has a similar structure to the styrene monomer of polystyrene. Hence the main objective was to modify the dioxygenase genes responsible for toluene degradation so that polystyrene can fit in the active site of the enzyme.
The todC1C2BA genes encode the toluene dioxygenase (TDO) enzyme which is capable of oxidizing more than 100 substrates albeit the catalytic oxygenase is encoded only by todC1C2. Modelling carried out by the Leicester 2012 iGEM team using the programme Pymol, suggested that the gap in the active site of the TDO enzyme could be widen to fit polystyrene. They modified amino acids present in the gap by reducing their side chains in order to widen the gap that could allow polystyrene into the active site. The residues that they modified were Met220 -> Ala220, Val421 -> Ala421, Tyr422 -> Leu422 and Tyr266 -> Val266.
Modified Toluene 2,3-dioxygenase enzyme with a similar view to the image above. Again, the pink residues are the active site residues. The white residues are those that have been modified. (left) The polystyrene substrate has been omitted for clarity, but note the gap is much bigger now (modified from 3EN1). (right) The polystyrene is present here: note the very tight fit into the gap
A side view of the modified enzyme. Note that the terminal phenyl group is in the active site
This view is just of the toluene 2,3-dioxygenase active site (blue), overlapped with the modified version (pink). Notice how similar the active site shape is, which means that potentially, the active site could still catalyse the dioxygenase reaction.
The Pseudomonas Putida strain F1 was grown in overnight broth. Miniprep procedures and gel purification were used to isolate and purify bacterial DNA. Polymerase chain reaction (PCR) was then used to amplify the tod C, X, F, G and B genes as well as the 16s rRNA. Thus we were able to confirm that we had the correct Pseudomonas strain using 16S rRNA gene sequencing. Therefore we proceeded to carry out a fusion PCR on the tod F, X and B genes.
These 3 tod genes were made into biobricks which were ligated to the iGEM supplied backbone (pSB1C3).  A restriction digest was performed on the fusion PCR products (biobricks) and pSB1C3. Each of the biobricks were subsequently ligated to pSB1C3 and transformed into highly competent cells (DH5α).  Some of the resulting colonies were then incubated in overnight broth. The isolation and sequencing of their plasmids confirmed that the genes were successfully cloned into the backbone containing the restrictions sites as stipulated by iGEM. In addition, double digestions were carried out to confirm the absence of mutations on the restrictions sites.
The gel picture shows the result of the double digestions. Well 2 is the digestion with EcoRI and PstI, well 3 is the double digestion with XbaI and SpeI. This shows that the restrictions sites are present in our biobricks.
We successfully made biobricks of todF BBa_K1095000 , todX BBa_K1095001 and todB BBa_K1095002 . Future Leicester iGEM team can continue to build biobricks of the other genes of the tod operon, especially todC1C2 genes that encodes the catalytic site of the TDO (toluene dioxygenase) enzyme, and proceed to characterise the biobricks created by this team.
Re-using
The second stream was centered on re-using polystyrene in consumer 3D printers. The 3D printer technology requires a support structure which is made prior to printing the image. This support material then needs to be disposed of which can be challenging. Previous studies have shown that limonene can be used to dissolve polystyrene and from the iGEM registry we know that a limonene biobrick exists. Therefore the team decided to use the biobrick to try to produce limonene-expressing bacteria which would be a cheaper method of synthesizing the solvent.
Limonene is a colourless hydrocarbon present in the rinds of most citrus fruits. The natural sources (lemons, oranges, etc.) produce the R-limonene, (+)-limonene, which gives citrus fruits their unique smell. It is widely used as a flavouring in food manufacturing and also  commonly in household products such as tooth paste.
The 2013 iGEM Leicester team carried out experiments to show that limonene is an effective polystyrene solvent. We were able to demonstrate that a 50% solution of limonene in a 20ml solution dissolves an 8cm x 5cm x 2cm block of polystyrene in approximately 9 minutes (video below).
limonene disolution from Quezia Toe on Vimeo .
Our idea was to characterise the limonene biobrick (BBa_K118025) created by the Edinburgh 2008 iGEM team. Firstly we shuffled the biobrick into the pSB1C3 backbone. The aim was to introduce the pSB1C3 backbone, is to separate the colonies that have incorporated the limonene biobrick, because those would be the ones present on a chloramphenicol plate.
We digested the backbone and the biobrick with EcoRI and PstI restriction enzymes. Digestions were run on a gel to confirm expected fragment sizes. Quick stick ligase was then used to ligate both genes with corresponding sites. Ligated DNA was transformed into E. coli and cells were grown overnight. Colonies that were present on the chloramphenicol plates were successfully transformed with the ligated plasmid. To confirm this overnight broth was made from the colonies whose DNA was isolated using mini prep. The isolated DNA was then cut with EcoRI and PstI and fragment sizes representative of the backbone and biobrick were visualized using gel electrophoresis.
The limonene biobrick supplied had a Lac operon preceding the limonene synthase genes. We aimed to induced the production of limonene  by growing bacteria with the incorporated ligated DNA (transformants) on chloramphenicol and IPTG plates. Volatile organic compound (VOC) was carried out in order to detect the presence on limonene on the plates. This assay was carried out using a Proton Transfer Reaction- Time of Flight Mass Spectrometer which has a parts per billion sensitivity. This instrument detects protonated compounds; for instance limonene has a molecular mass of 136 therefore a sharp peak should be seen at 137 and a fragmentation peak at 81. From the graphs below, these 2 important peaks that would indicate the presence of limonene were absent, which suggested that limonene was not synthesized.
The graph above shows no peaks at 137 and 81 (left).  (right) The instrument: Proton Transfer Reaction - Time of Flight Mass Spectrometer
Further statistical tests were carried out using ‘R’programme in order to arrive at conclusive results. The data obtained from the test had a non-normal frequency distribution (positively skewed) which was confirmed using Shapiro-Wilk normality test. Therefore non-parametric tests (Spearman correlation and Kendall tau tests)were used for statistical analysis
The non-parametric test used included; Spearman correlation and Kendall tau tests.
Samples 1,2 & 3 = RFP plates
Samples 4,5 & 6 = 10.2 plates
Samples 7,8 & 9 = 5.1 plates
Correlation was made between each of the RFP plates and the limonene biobrick/pBS1C3 plates (there were 9 plates in total; 3 RFP and 6 limonene biobrick transformant plates) in the following order:
a) Samples 1, 4 and 7
b) Samples 2, 5 and 8
c) Samples 3, 6 and 9
After carrying out both tests (Spearman and Kendall) on groups a,b and c, p-values were < 0.05. As shown below (Spearman and Kendall respectively);
a) Between 1& 4 - p-value= 6.412e-10 and 2.53e-07
Between 1 & 7- p-value = 9.82e-13 and 4.558e-08
b) Between 2 & 5- p-value = 1.534e-09 and 3.1e-07
Between 2 & 8- p-value = 3.579e-12 and 5.655e-08
c) Between 3 & 6- p-value=8.359e-13 and 3.81e-08
Between 3 & 9 - p-value=3.566e-12 and 7.004e-08
Thus the null hypothesis (There is no correlation between the RFP plates and limonene biobrick/shipping vector) was rejected.
In conclusion, it seems that the stats tests cannot detect limonene expression too! Hence the correlation between the RFP (control plates) and the limonene biobrick plates. This suggests either a fault in the limonene biobrick or a disfunctional promoter. Unfortunately due to time constraint we were unable to run further testing to determinate the cause of the lack of limonene synthesis.
Re-purposing
Expanded polystyrene is widely used as insulation in the construction industry. In some applications the insulation is required to be flame retardant, which is achieved by incorporating halogenated hydrocarbons – an environmental pollutant. DNA has been shown to be flame-retardant ; however it is very expensive to produce. We proposed to build a genetically engineered machine to cheaply generate DNA for incorporation into flame retardant polystyrene.
Firstly, experiments were carried out to show that polystyrene is indeed highly flammable (video below).
igem polystyrene (no) from Quezia Toe on Vimeo .
We then designed experiments to determine if coating polystyrene with DNA could change its fire retardancy in comparison to a non-coated polystyrene block. Herring sperm DNA was used at a concentration of 13.4 mg/ml. Mechanical processes such as sonification and hydrodynamic shearing were used to make the DNA more spreadable.
Initial combustion tests were carried out using DNA-coated paper-thin strips of polystyrene which were compared with water-coated polystyrene strips shown in the video below.
iGEM combustion from Quezia Toe on Vimeo
It is evident from the video that the addition of DNA aided the flammability of the polystyrene. These results were not as expected; hence we decided to carry out a second test by modifying experimental conditions.
The second test was carried out using blocks of polystyrene instead of paper-thin strips. This time around DNA was incorporated into the block instead of being spread on it.  This new  video recording (below) brought to our attention the possibility that the polystyrene we were using might in fact be coated in HBCD. It was later confirmed that the polystyrene sugar supplied to us was in fact incorporated in HBCD.
Block Combustion from Quezia Toe on Vimeo .
Unfortunately due to time constraint we were unable to aquire the non-HBCD incorporated polystyrene sugar to use for further tests. This can be used as a starting point for future research albeit we propose that the cotton experiment should be replicated in order to confirm that DNA is able to give flame-retardancy property to materials.
Bibliography
Alongi, J., et al, - DNA: a novel, green, natural flame retardant and suppressant for cotton. - The Royal Society of Chemistry.
Atiq, N., Ahmed, S., Ali, M., Andleeb, S., Ahmad, B., Robson, G., 2010. Isolation and identification of polystyrene biodegrading bacteria from soil. African Journal of Microbiology Research. 4(14), 1537-1541
Parales, R.E., Parales, J.V., Pelletier, D.A., Ditty, J.L., 2008. Diversity of microbial toluene degradation pathways. Advances in Applied Microbiology. 64, 1-73, 2 p following 264.
[edit]
Retrieved from " http://2013.igem.org/Team:Leicester/Project "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
