<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_IIT_Delhi_project skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:IIT Delhi/project</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="wrapper"><DIV id="header"><NAV><DIV class="iconsDiv" style="margin-left:29%;"><UL class="smenu"><LI><A href="https://2015.igem.org/Team:IIT_Delhi/project_background">BACKGROUND</A></LI><LI><A href="https://2015.igem.org/Team:IIT_Delhi/project_solution">SOLUTION</A></LI><LI><A href="https://2015.igem.org/Team:IIT_Delhi/project_research">RESEARCH</A></LI><LI><A href="https://2015.igem.org/Team:IIT_Delhi/project_prototype">PROTOTYPE</A></LI><LI><A href="https://2015.igem.org/Team:IIT_Delhi/project_conclusion">CONCLUSION</A></LI></UL></DIV><DIV class="iconsDiv" style="margin-left:36%;"><UL class="smenu"><LI><A href="https://2015.igem.org/Team:IIT_Delhi/notebook_summary">SUMMARY</A></LI><LI><A href="https://2015.igem.org/Team:IIT_Delhi/notebook_timeline">TIMELINE</A></LI><LI><A href="https://2015.igem.org/Team:IIT_Delhi/notebook_album">ALBUM</A></LI></UL></DIV><DIV class="iconsDiv" style="margin-left:43%;"><UL class="smenu"><LI><A href="https://2015.igem.org/Team:IIT_Delhi/results_parts">PARTS SUBMITTED</A></LI><LI><A href="https://2015.igem.org/Team:IIT_Delhi/results_judging">JUDGING CRITERION</A></LI><LI><A href="https://2015.igem.org/Team:IIT_Delhi/results_attributes">ATTRIBUTES</A></LI></UL></DIV><DIV class="iconsDiv" style="margin-left:50%;"><UL class="smenu"><LI><A href="https://2015.igem.org/Team:IIT_Delhi/modelling_introduction">INTRODUCION</A></LI><LI><A href="https://2015.igem.org/Team:IIT_Delhi/modelling_mathematical">MATHEMATICAL MODEL</A></LI><LI><A href="https://2015.igem.org/Team:IIT_Delhi/modelling_simulation">SIMULATION</A></LI><LI><A href="https://2015.igem.org/Team:IIT_Delhi/modelling_interpretation">INTERPRETATION</A></LI></UL></DIV><DIV class="iconsDiv" style="margin-left:57%;"><UL class="smenu"><LI><A href="https://2015.igem.org/Team:IIT_Delhi/team_ourstory">OUR STORY</A></LI><LI><A href="https://2015.igem.org/Team:IIT_Delhi/team_mentors">MENTORS</A></LI><LI><A href="https://2015.igem.org/Team:IIT_Delhi/team_members">MEMBERS</A></LI><LI><A href="https://2015.igem.org/Team:IIT_Delhi/team_sponsors">SPONSORS</A></LI><LI><A href="https://2015.igem.org/Team:IIT_Delhi/team_carpediem">CARPE DIEM</A></LI></UL></DIV><DIV class="iconsDiv" style="margin-left:64%;"><UL class="smenu"><LI><A href="https://2015.igem.org/Team:IIT_Delhi/collaboration_iiser">IISER</A></LI><LI><A href="https://2015.igem.org/Team:IIT_Delhi/collaboration_enactus">ENACTUS</A></LI><LI><A href="https://2015.igem.org/Team:IIT_Delhi/collaboration_outreach">OUTREACH</A></LI></UL></DIV><DIV class="iconsDiv" style="margin-left:71%;"><UL class="smenu"><LI><A href="https://2015.igem.org/Team:IIT_Delhi/ethics_orientation">ORIENTATION</A></LI><LI><A href="https://2015.igem.org/Team:IIT_Delhi/ethics_openhouse">OPEN HOUSE</A></LI><LI><A href="https://2015.igem.org/Team:IIT_Delhi/ethics_tryst">TRYST</A></LI><LI><A href="https://2015.igem.org/Team:IIT_Delhi/ethics_surveys">SURVEYS</A></LI></UL></DIV></NAV><DIV class="box"><UL id="ldd_menu" class="ldd_menu" style="position:absolute;"><LI><SPAN id="site">SiteMap</SPAN></LI><LI class="ldd_heading"><A style="color:white;" href="https://2015.igem.org/Team:IIT_Delhi/abstract">ABSTRACT</A></LI><LI><A href="https://2015.igem.org/Team:IIT_Delhi/abstract#ourproject">OUR PROJECT</A></LI><LI><A href="https://2015.igem.org/Team:IIT_Delhi/abstract#categories">CATEGORIES</A></LI><LI><A href="https://2015.igem.org/Team:IIT_Delhi/abstract#sponsors">SPONSORS</A></LI></UL><UL><LI class="ldd_heading"><A style="color:white;" href="https://2015.igem.org/Team:IIT_Delhi/project">PROJECT</A></LI><LI><A href="https://2015.igem.org/Team:IIT_Delhi/project#research">RESEARCH</A></LI><LI><A href="https://2015.igem.org/Team:IIT_Delhi/project#prototype">PROTOTYPE</A></LI><LI><A href="https://2015.igem.org/Team:IIT_Delhi/project#conclusion">CONCLUSION</A></LI></UL><UL><LI class="ldd_heading"><A style="color:white;" href="https://2015.igem.org/Team:IIT_Delhi/notebook">NOTEBOOK</A></LI><LI><A href="https://2015.igem.org/Team:IIT_Delhi/notebook#summary/">SUMMARY</A></LI><LI><A href="https://2015.igem.org/Team:IIT_Delhi/notebook#timeline">TIMELINE</A></LI><LI><A href="https://2015.igem.org/Team:IIT_Delhi/notebook#album">ALBUM</A></LI></UL><UL><LI class="ldd_heading"><A style="color:white;" href="https://2015.igem.org/Team:IIT_Delhi/results">RESULTS</A></LI><LI><A href="https://2015.igem.org/Team:IIT_Delhi/results#parts">PARTS SUBMITTED</A></LI><LI><A href="https://2015.igem.org/Team:IIT_Delhi/results#judging">JUDGING CRITERION</A></LI><LI><A href="https://2015.igem.org/Team:IIT_Delhi/results#attributions">ATTRIBUTIONS</A></LI></UL><UL><LI class="ldd_heading"><A style="color:white;" href="https://2015.igem.org/Team:IIT_Delhi/modelling">MODELLING</A></LI><LI><A href="https://2015.igem.org/Team:IIT_Delhi/modelling#introduction">INTRODUCTION</A></LI><LI><A href="https://2015.igem.org/Team:IIT_Delhi/modelling#mathematical">MATHEMATICAL MODEL</A></LI><LI><A href="https://2015.igem.org/Team:IIT_Delhi/modelling#simulation">SIMULATION</A></LI><LI><A href="https://2015.igem.org/Team:IIT_Delhi/modelling#interpretation">INTERPRETATION</A></LI></UL><UL><LI class="ldd_heading"><A style="color:white;" href="https://2015.igem.org/Team:IIT_Delhi/team">TEAM</A></LI><LI><A href="https://2015.igem.org/Team:IIT_Delhi/team#ourstory">OUR STORY</A></LI><LI><A href="https://2015.igem.org/Team:IIT_Delhi/team#mentors">MENTORS</A></LI><LI><A href="https://2015.igem.org/Team:IIT_Delhi/team#members">MEMBERS</A></LI><LI><A href="https://2015.igem.org/Team:IIT_Delhi/team#carpediem">CARPE DIEM</A></LI></UL><UL><LI class="ldd_heading"><A style="color:white;" href="https://2015.igem.org/Team:IIT_Delhi/human_practices">ETHICS</A></LI><LI><A href="https://2015.igem.org/Team:IIT_Delhi/human_practices#collaborations">COLLABORATIONS</A></LI><LI><A href="https://2015.igem.org/Team:IIT_Delhi/human_practices#outreach">OUTREACH</A></LI><LI><A href="https://2015.igem.org/Team:IIT_Delhi/human_practices#documentation">POLLUTION</A></LI><LI><A href="https://2015.igem.org/Team:IIT_Delhi/human_practices#casestudy">CASE STUDY</A></LI></UL><UL><LI class="ldd_heading"><A style="color:white;" href="https://2015.igem.org/Team:IIT_Delhi/ethics">HUMAN PRACTICES</A></LI><LI><A href="https://2015.igem.org/Team:IIT_Delhi/ethics#orientation">ORIENTATION</A></LI><LI><A href="https://2015.igem.org/Team:IIT_Delhi/ethics#openhouse">OPEN HOUSE</A></LI><LI><A href="https://2015.igem.org/Team:IIT_Delhi/ethics#tryst">TRYST</A></LI><LI><A href="https://2015.igem.org/Team:IIT_Delhi/ethics#surveys">SURVEYS</A></LI></UL></DIV></DIV></DIV></DIV><DIV id="leftNav"><H1 style="color:white;font-family:Verdana, Geneva, sans-serif;font-size:230%;text-align:center;">PROJECT</H1><NAV class="snav"><UL class="mnmenu"><LI><A href="#research">Research</A></LI><LI><A href="#prototype">Prototype</A></LI><LI><A href="#conclusion">Conclusion</A></LI></UL></NAV></DIV><DIV id="wikicontent"><DIV class="container con"><TABLE><TBODY><TR id="research"><TD class="det" align="left" style="vertical-align:top;"><H1 style="font-family:'Trebuchet MS', 'Lucida Grande', 'Lucida Sans Unicode', 'Lucida Sans', Tahoma, sans-serif;color:white;font-size:250%;padding-top:18px;">Research</H1><H2 style="font-family: 'Roboto', sans-serif;color:white;font-size:150%;vertical-align:text-top;">


Since our target was to reduce air pollution, we targeted genes that act on NOx and SOx gases. Research was done extensively and the following genes were found to be best fitting for our project-
<FONT color="#1569C7"><B>I - NrfA (Nitrite reductase)-</B></FONT>

Since Nitric oxide NO is a major greenhouse gas, the reduction of NO from the air is an important task to address global warming concerns. NO is found primarily in motor exhausts, and Oxides of nitrogen (NOX) are key oxidants that have a role in the photochemical production of ozone, whilst also being linked with an increase in the oxidising capacity of the atmosphere.
 
The nitrite reductase NrfA is encoded within the nrfA operon that is found in enteric bacteria such as E. coli. The primary function of NrfA is to enable the use of nitrite as a respiratory electron acceptor, forming ammonia as a final product. Nitric oxide (NO) is a proposed intermediate in the six-electron reduction of nitrite and NrfA can use exogenous NO as a substrate. This enzyme is a periplasmic protein and therefore contains a signal sequence to redirect it to the periplasmic space after the protein is formed. While no intermediates of the reaction are released, NrfA is also able to reduce various oxides other than NO such as, hydroxylamine (H2NOH), and nitrous oxide (N2O), but notably also sulfite(SO32-), providing the only known direct link between the nitrogen and sulfur cycles. 
 
The pathway of the nrfA Gene is still not completely understood or characterised. However, the overall reaction is given by:</H2></TD></TR></TBODY></TABLE></DIV>


Thus, the six electron reduction of NO2- to NH3 is catalysed by the nitrite reductase nrfA. Also, another important product of the reaction, NAD(P)+ act as reducing equivalents that are an inherent part of the electron transport system in the host.

The nrfA gene is suppressed at higher levels of nitrite or nitrate. Steady-state gene expression studies revealed a differential pattern of nitrite reductase gene expression where optimal nrfA-lacZ expression occurred only at low to intermediate levels of nitrate. 

The reason for this repression at high nitrite concentrations is the presence of two regulators of the nrfA gene, NarL and NarP, in the E.coli K-12 genome (from where this gene has been isolated). While either NarL or NarP was able to induce nrfA-lacZ expression in response to low levels of nitrate, only NarL could repress at high nitrate levels.
 
This problem could therefore, be solved, by expressing the nitrite reductase gene under a constitutive strong promoter, which is what we have done. 
 
Escherichia coli possesses two biochemically distinct nitrite reductase enzymes encoded by the nrfABCDEFG and nirBDC operons. The NirB nitrite reductase is a soluble siroheme-containing enzyme that uses NADH as an electron donor to reduce nitrite in the cytoplasm. The NrfA nitrite reductase is a membrane-associated respiratory enzyme that couples to the membrane-associated formate-oxidizing enzymes via quinones in order to generate membrane potential. The abundance of each enzyme is elevated during anaerobic cell growth conditions when either nitrate and/or nitrite is present[7]. Nitrite, the substrate for the two enzymes, must either be encountered environmentally or generated by the cell from nitrate reduction by one of the three E. coli nitrate reductases. <B>Gene construction &amp; cloning: </B><P>The general cloning strategy for NrfA consists of the following steps.
<LI>1.	The biobrick BBa_K1153001 (NrfA), previously submitted by KENT iGEM 2013 team, was obtained from iGEM repository.
</LI><LI>2.	Another biobrick consisting of the Promoter and Ribosome Binding Site BBa_K880005 (P+RBS) was also procured.
</LI><LI>3.	(P+RBS) was doubly digested by EcoRI and SpeI enzymes to yield Eco-(P+RBS)-Spe construct.
</LI><LI>4.	(NrfA) was doubly digested by XbaI and PstI to yield Xba-(NrfA)-Pst construct.
</LI><LI>5.	A linearized vector was prepared by double-digestion with EcoRI and PstI.
</LI><LI>6.	The 3A assembly was performed to clone the desired construct as shown below into the linearized backbone.
<CENTER>EcoRI-XbaI-(P+RBS)-(NrfA)-SpeI-PstI</CENTER></LI></P><FONT color="#1569C7"><B>II - NosZ (Nitrous Oxide Reductase)-</B></FONT>
Nitrous oxide is a potent greenhouse gas, whose atmospheric concentration has been increasing since the introduction of the Haber Bosch process led to the widespread use of nitrogenous fertilizers. One of the pathways to its destruction is reduction to molecular nitrogen by the enzyme nitrous oxide reductase found in denitrifying bacteria.
 
NosZ, a gene coding for nitrous oxide reductase, isolated from Pseudomonas aeruginosa PAO1, is known to convert N2O to harmless Nitrogen gas (N2). It has two copper centers, a binuclear CuA center, similar to the one found in cytochrome c oxidase, and the CuZ center, a unique tetranuclear copper center now known to possess either one or two sulfide bridges[3]. 

The reduction of nitrous oxide to molecular nitrogen requires two protons and two electrons, according to the equation
N2O + 2H+ + 2e− → N2 + H2O, E°′ (pH 7.0) = 1.35 V; ΔG0′ = −339.5 kJ/mol                
This is a challenging reaction to be catalyzed by a metalloenzyme, as nitrous oxide is not just kinetically inert to decomposition but also a poor transition metal ligand, due to its weak σ-donating and π-accepting properties.
Although, there have been some reports of other enzymes, such as nitrogenase and a multicopper oxidase from Pyrobaculum aerophilum, able to catalyze this reaction in vitro, the only one, whose primary role is the reduction of nitrous oxide, is N2O reductase (NosZ) 
 
The complete denitrification pathway is shown below- Gene construction &amp; cloning:
<P>The general cloning strategy for P+RBS+NosZ+SYFP consists of the following steps.
<LI>1.	The biobrick BBa_K1866001 (NosZ+SYFP) was prepared by Team IIT Delhi iGEM 2015.
</LI><LI>2.	Another biobrick consisting of the Promoter and Ribosome Binding Site BB_K880005 (P+RBS) was also procured.
</LI><LI>3.	(P+RBS) was doubly digested by EcoRI and SpeI enzymes to yield Eco-(P+RBS)-Spe construct.
</LI><LI>4.	(NosZ+SYFP) was doubly digested by EcoRI and XbaI to yield EcoRI-(NosZ+SYFP)-XbaI construct.
</LI><LI>5.	Ligation was set up to obtain the desired construct as shown below.
<CENTER>EcoRI-XbaI-(P+RBS)-(NosZ+SYFP)-SpeI-PstI</CENTER></LI></P><FONT color="#1569C7"><B>III - SQR (Sulfide Quinone Reductase)-</B></FONT>

The SQR gene codes for sulfide quinone reductase, an enzyme catalysing the conversion of H2S to elemental Sulfur(S). The initial plan was to clone this gene downstream of the CysI gene from Pseudomonas aeruginosa, which codes for sulfite reductase, catalysing the conversion of sulfite to hydrogen sulfide (H2S).  

Sulfur dioxide is a gas. It is invisible and has a nasty, sharp smell. It reacts easily with other substances to form harmful compounds, such as sulfuric acid, sulfurous acid and sulfate particles. 
About 99% of the sulfur dioxide in air comes from human sources. The main source of sulfur dioxide in the air is industrial activity that processes materials that contain sulfur, eg the generation of electricity from coal, oil or gas that contains sulfur. Some mineral ores also contain sulfur, and sulfur dioxide is released when they are processed. In addition, industrial activities that burn fossil fuels containing sulfur can be important sources of sulfur dioxide. 
Sulfur dioxide is also present in motor vehicle emissions, as the result of fuel combustion. In the past, motor vehicle exhaust was an important, but not the main, source of sulfur dioxide in air. However, this is no longer the case. 
 
Sulfur dioxide affects human health when it is breathed in. It irritates the nose, throat, and airways to cause coughing, wheezing, shortness of breath, or a tight feeling around the chest. The effects of sulfur dioxide are felt very quickly and most people would feel the worst symptoms in 10 or 15 minutes after breathing it in. 

Those most at risk of developing problems if they are exposed to sulfur dioxide are people with asthma or similar conditions.


The CysI gene converts sulfur dioxide, SO2 to hydrogen sulfide, H2S. However, hydrogen sulfide in itself is a harmful gas. The U.S. Environmental Protection Agency (U.S. EPA) identifies the most serious hazardous waste sites in the nation. U.S. EPA then includes these sites on the National Priorities List (NPL) and targets them for federal clean-up activities. U.S. EPA has found hydrogen sulfide in at least 35 of the 1,689 current or former NPL sites. Hence, cloning the SQR gene is important, as it will convert the toxic waste gas hydrogen sulfide to sulfur, which is a solid product and can therefore be separated out with ease.

<B> Gene construction &amp; cloning:</B><P>The general cloning strategy for NrfA consists of the following steps.
<LI>1.	The biobrick BBa_K896000 (Sqr), previously submitted by NYMU iGEM 2012 team, was obtained from iGEM repository.
</LI><LI>2.	Another biobrick consisting of the Promoter and Ribosome Binding Site BBa_K880005 (P+RBS) was also procured.
</LI><LI>3.	(P+RBS) was doubly digested by EcoRI and SpeI enzymes to yield Eco-(P+RBS)-Spe construct.
</LI><LI>4.	(Sqr) was doubly digested by XbaI and PstI to yield Xba-(Sqr)-Pst construct.
</LI><LI>5.	A linearized vector was prepared by double-digestion with EcoRI and PstI.
</LI><LI>6.	The 3A assembly was performed to clone the desired construct as shown below into the linearized backbone.
<CENTER>EcoRI-XbaI-(P+RBS)-(Sqr)-SpeI-PstI</CENTER></LI></P><B>Confirmation of the clones: </B><P>We performed a double digestion by EcoRI and PstI to release the cloned fragment from the recombinant vector. The digestion products were resolved on an Agarose Gel by Electrophoresis and the size of the fragment confirmed the correct clone.</P><H2>Figure3: Clone confirmation </H2><FONT color="#1569C7"><B>IV - The Problem:Heme-</B></FONT> 
The biggest problem which we encountered was that the heme synthesis was much less as compared to our overexpressed protein. 
Heme : 
Heme (iron-protoporphyrin IX) prosthetic groups perform a wide range of functions in nature including
<LI>1)	electron transfer</LI><LI>2)	oxygen transport and storage</LI><LI>3)	catalysis</LI><LI>4)	gas sensing</LI><LI>5)	gene regulation</LI>
Types of Heme :
 The most common types of heme are heme b and heme c.
 Heme b: It is iron-protoporphyrin IX that is bound noncovalently to protein 
Heme c: heme c has two covalent thioether bonds formed between Cys side chains and the heme vinyl groups at positions 2 and 4 . The stereochemistry of heme attachment is the same in all known examples of heme c, and the vinyl groups at positions 2 and 4 are attached to the N- and C-terminal Cys of the CXXCH motif, respectively.


Other, less common, derivatives of heme include heme d1, present in cytochrome cd1 nitrite reductase, heme a, found in cytochrome c oxidase, and the related heme o, found in some bacterial oxidases.
From a chemical perspective, hemes b and c are very similar to each other and thus are not expected to display significant inherent differences in electronic structure or reduction potential.
 From a biosynthetic perspective, however, these types of heme are quite different; because heme c is synthesized from heme b, the use of heme c requires a greater investment from the organism. 
The heme which is found in NrfA is Heme c 
Heme Biosynthesis Pathway :



Incomplete heme incorporation into recombinant protein has been a frequently encountered problem .A population of protein will fold without heme cofactor under condition where folding outspaces heme delivery 


Cytochrome c biogenesis: 
Natural modiﬁcations to heme occur after the ferrous iron (Fe2) is inserted into the porphyrin by ferrochelatase, the last enzyme in heme biosynthesis

 
CCm complex : It is a complex of Transmembrane protein which helps in Heme maturation .From modelling we believe that by upregulating CCm complex of proteins we can increase heme incorporation into our proteins. 


Technique to improve heme incorporation:
<LI>1)	Supplementing the growth media with δ-Aminolevulinic acid a precursor to C5 heme biosynthesis pathway, increases levels of heme biosynthesis and thereby heme incorporation into some proteins. But the end product heme acts as a feedback regulator of ALA synthase thereby inhibiting its production. </LI><LI>To avoid this we thought of Expressing plasmid encoded mouse ALA synthase which is not inhibited by heme ,thereby increasing cellular heme production. </LI><LI>2)	We can also supplement the growth medium with hemin . But most of the E-coli strains do not possess an efficient heme transport system and thus uptake of hemin solely relies on diffusion through the cell membrane.
So to avoid this problem we thought of incorporating ChuA gene into E.coli
ChuA :It is basically a heme-transport protein which helps in utilisation of Hemin </LI><FONT color="#1569C7"><B>V - Sulphite Reductase(Sir)-</B></FONT> 

NADPH-sulfite reductase (SiR) of Escherichia coli catalyzes the reduction of sulfite to sulfide and is required for synthesis of L-cysteine from inorganic sulfate
The native enzyme has a subunit structure a8B4 , where a8 is a flavoprotein (SiR-FP) containing both flavin adenine dinucleotide and flavin mononucleotide and P is a hemoprotein (SiR-HP) containing an Fe4S4 centre and a single molecule of siroheme
Electron flow between these cofactors proceeds from NADPH to flavin adenine dinucleotide to flavin mononucleotide in the flavoprotein, then to a closely coupled Fe4S4 siroheme center in the hemoprotein, and finally from siroheme to sulphiteThe SiR-FP and SiR-HP components of SiR are encoded by cysJ and cysl, respectively. These genes are contiguous and together with cysH, the gene for 3'-phosphoadenosine 5'-phosphosulfate sulfotransferase, comprise an operon with the gene order promoter-cysJ-cysI-cysH
cysJIH operon is part of the positively regulated cysteine regulon (15) and requires sulfur limitation
CysB protein, and either O-acetyl-L-serine or N-acetyl-L-serine for expression
SiR activity is also dependent on cysG, which encodes a uroporphyrinogen III methyltransferase necessary for the synthesis of siroheme
 This gene is located more than 10 min away from cysJIH on the chromosomal map (34) and is not tightly regulated as part of the cysteine regulatory 

<FONT color="#1569C7"><B>VI - Characterization-</B></FONT><B>Poly Acrylamide Gel Electrophoresis</B>
1.	To characterise the Promoter + RBS + nrfA biobrick (Bba_K1866000), we first ran the total protein content of our recombinant E.coli DH5α culture (into which our pSB1A3 plasmid containing the biobrick had been cloned) on an SDS-Polyacrylamide Gel. The result showed over expression in the range of 55 kDa, which is roughly equal to the size of our protein.
2.	Periplasmic protein SDS-PAGE—following the over expression obtained from the Poly Acrylamide gel electrophoresis, we extracted the protein content from the periplasmic space using the periplasm fractionation protocol. This was run on another polyacrylamide gel, and we saw over expression on this gel as well, on the appropriate range.<B>Growth experiments</B>
3.	Minimal media growth- Following a research paper by Clarke et al., a culture of our cloned E.coli DH5α culture was grown on a minimal media containing 5% Luria Broth (by volume), 40mM Potassium Nitrate, 20mM Fumarate and 0.4% glycerol for 16 hours, in anaerobic conditions. The cells were then pelleted by centrifugation and were then to be sub cultured. This experiment failed, even though we saw a pellet after centrifugation following growth. However, we later realised that the pellet was not of our cells, but of ampicillin, which had formed some sort of white precipitate with the media. Hence, this resulted in a failure.


4.	pH Monitoring (LB Growth)- Cultures of our clones were grown anaerobically in 5ml of luria broth, along with standard DH5α cells, used as control. These were then subcultured into 50 ml cultures, also grown anaerobically, with different concentrations of Sodium Nitrite. After 16 hours of growth, the cultures were taken out and the pH of the solution was taken for all different concentrations of Nitrite. The experiment showed results as expected, with the pH of our clones being higher than the control, which can be attributed to the presence of excess ammonia in the solution. We also saw that at concentrations of Nitrite greater than 2mM, the pH showed a sharp decline. This could be due to the fact that high concentrations of nitrite (NO¬2-) become toxic for the cells, due to which cell death occurs. 

5.	Indophenol test (LB Growth)- Cultures of our clones were grown anaerobically in 5ml in luria broth, along with standard DH5α cells, used as control. These were then subcultured into 50 ml cultures, also grown anaerobically, with different concentrations of Sodium Nitrite. To a 1ml aliquot, 40 microlitre phenol, 40 micolitre sodium nitroprusside and 100 ul of oxidising reagent (tri- sodium citrate + sodium hypochlorite) was added, and absorbance was measured at 540 nm. These OD values were plotted against DH5α culture, used as control for the experiments.

a.	Monitoring with time- For a concentration of 1mM Nitrite, multiple cultures were inoculated. After regular intervals of time, the indophenol test was carried out, and OD540 values were noted. This experiment was successful, as it showed an increase of OD values with increase in time, before finally saturating. The OD values were more than that of DH5α cultures throughout, pointing to the fact that more ammonia was being formed in our clone cultures.

b.	Monitoring with different nitrite concentrations- 50 ml Cultures were inoculated with different concentrations of Sodium nitrite. After 16 hours of growth, the indophenol test was carried out, and OD Values were taken. This experiment was also successful, with OD values showing a trend similar to the one above. At high concentrations of Sodium nitrite, the OD values showed a sharp decline, which can again be attributed to high concentrations of nitrite (&gt;2mM) being toxic to the cells.

6.	Nessler’s test (LB growth)- Cultures of our clones were grown anaerobically in 5ml of luria broth, along with standard DH5α cells, used as control. These were then subcultured into 50 ml cultures, also grown anaerobically, with different concentrations of Sodium Nitrite. After 16 hours of growth, the cultures were taken out and nessler’s reagent was added to a 1ml aliquot. Nessler’s reagent forms a reddish brown precipitate on reacting with ammonia, and is extremely sensitive. This culture was then centrifuged at maximum speed (13,400 rpm) for 10 minutes, and the media was decanted and dried out, leaving only the red ammonia pellet. This was then weighed against DH5α culture (given the same treatment), using the DH5α treated sample as blank (mass of ammonia pellet taken = 0). This experiment was also successful, showing trends as expected. At high nitrite concentrations, the pellet size reduced drastically (the reason being toxicity at high nitrite concentration).<FONT color="#1569C7"><B>VII - References-</B></FONT><LI>1.	Clarke, TA., Mills, PC. et al (2008). Escherichia coli cytochrome c nitrite reductase NItalic textrfA.. Methods in Enzymology. 437: 63-77. </LI><LI>2.	Structure and function of formate-dependent cytochrome c nitrite reductase, NrfA, Einsle O, Methoda in Enzymology </LI><LI>3.	Review, Nitrous oxide reductase Sofia R. Pauleta, Simone Dell’Acqua, Isabel Moura </LI><LI>4.	Air quality fact sheet, Department of the Environment and Heritage, 2005 </LI><LI>5.	The nrfA and nirB Nitrite Reductase Operons inEscherichia coli Are Expressed Differently in Response to Nitrate than to Nitrite, Henian Wang and Robert P. Gunsalus </LI><LI> 6.	Cole J. (1996) Nitrate reduction to ammonia by enteric bacteria: redundancy, or a strategy for survival during oxygen starvation? FEMS Microbiol. Lett. 136:1–11 </LI><LI>7.	Page L., Griffiths L., Cole J. A. (1990) Different physiological roles for two independent pathways for nitrite reduction to ammonia by enteric bacteria.Arch. Microbiol. 154:349–354 </LI><LI>8.	Toxic substances portal, Public Health Statement for Hydrogen Sulphide </LI><LI>9.	Optimization of the heme biosynthesis pathway for the production of 5-aminolevulinic acid in Escherichia coli, Junli Zhang Zhen Kang Jian Chen Guocheng Du </LI><LI>10.	The Chemistry and Biochemistry of Heme c: Functional Bases for Covalent Attachment, Sarah E. J. Bowman and Kara L. Bre</LI><LI>11.	ChuA – outer membrane heme receptor, Torres,A.G et.al. , Okele , I.N .et.al ,Naggy , G et.al</LI><LI>12.	Cytochrome c Biogenesis: Mechanisms for Covalent Modiﬁcations and Trafﬁcking of Heme and for Heme-Iron Redox Control Robert G. Kranz,1 Cynthia Richard-Fogal,1 John-Stephen Taylor,2 and Elaine R. Frawley1 </LI><LI>13.	High-Level Expression of Escherichia coli NADPH-Sulfite Reductase: Requirement for a Cloned cysG Plasmid To Overcome Limiting Siroheme Cofactor, Jer-yuarn wu, lewis m. Siegel, and nicholas m. Kredich </LI><TABLE><TBODY><TR id="prototype"><TD class="det"><H1 style="font-family:'Trebuchet MS', 'Lucida Grande', 'Lucida Sans Unicode', 'Lucida Sans', Tahoma, sans-serif;color:white;font-size:250%;padding-top:18px;">Prototype</H1><H2 style="font-family: 'Roboto', sans-serif;color:white;font-size:150%;vertical-align:text-top;">
												Why the &quot;Pollution Crusader Device&quot;?  
The alarming levels of pollution in Delhi are not hidden from any of us. Delhi has been ranked the first most polluted metropolis in the world, and is one of the most heavily polluted cities in India. In the period between 2000-2011, PM10 levels in Delhi's air jumped by as much as 47 per cent. 
Hence we came up with idea of using biology to cater to this problem. To come with a solution that is both sustainable and effective. We started with making Eco.coli that could reduce (NO)x emissions. 
We further instigated the problem and found that one of the largest contributions to rising pollution in Delhi is the exhaust from automobiles and other devices using petrol and diesel as fuel. The main components of Diesel engine exhaust are: 
Respirable Suspended Particulate Matter: <LI>NOx</LI><LI>SOx</LI><LI>CO2 </LI><LI>CO </LI>
All these emissions are responsible for causing harmful health impacts on living beings, and deteriorate the environmental conditions of planet earth. 
RSPM and NOx when combined, form a deadly combination which drastically affects human health. 
RSPM when inhaled acts like an abrasive and wears the lining of respiratory tract exposing soft spots where NOx reacts and causes severe issues. So we decided to eliminate soot and NOx as the primary objective. 
Analysis of soot: 
Soot is of two types: hydrophilic as well as hydrophobic. Hydrophilic soot dissolves in water, whereas hydrophobic soot dissolves in organic solvents like alcohol. To get rid of both the hydrophilic and hydrophobic components, we decided to use a solution of acetic acid in water as the solvent. Acetic acid, being an organic solvent, dissolves the hydrophobic part, and water, dissolves the hydrophobic soot. Using acetic acid, however, offers an added advantage as well. Being an acid, it reduces the PH of the medium and hence decreases the solubility of acidic NOx and SOx, from the exhaust into this solvent. Thus, the tank containing this solvent dissolves soot and NOx, Sox gases move out with the exhaust. The exhaust out of this tank is now passed into the desooter tank. The desooter tank has the following main design components: <LI>Water tank </LI><LI>Fresh water inlet</LI><LI>Exit of tar </LI><LI>Level meter </LI><LI>Sparger</LI></H2></TD></TR></TBODY></TABLE></DIV>
 
The exhaust gases enter the desooter tank at high temperature(200-250 C), and hence they cause the water in the tank to evaporate when they pass through it. Hence, a level meter is required to keep a check on the water level left in the tank, and hence supply fresh water accordingly through the fresh water inlet.  
A provision of tar exit is needed to remove this tar slurry continuously out of the tank. Spargers are needed to form bubbles of exhaust gas in the mixture. Blowers are also attached to the system to compensate for the back pressure generated in the system at each step.  
Now, we needed to condense the water vapor which flow along with exhaust gases out of this tank. This was required so that these excess water vapor do not clog the silica gel at the outlet of the tank. For this, we attach a heat exchanger at the outlet of the tank. It’s a cross flow type heat exchanger, with which we did some slight innovations to increase the efficiency of the condensation process. Cold water flows through the exchanger pipes and steam flows on the outside. We here filled the outside of the pipe portion of the exchanger with metal ball bearings. These ball bearings, through conduction, cool down to acquire the temperature(similar to) of the flow pipes and hence increase the surface area available to the vapor for condensation, speeding up the condensation process. The remaining exhaust, congaing NOx, Sox and some amount of water vapor now passes over the silica gel tank at the outlet of this desooter tank.  
Studies showed that NOx and SOx, in the presence of water vapor, react over Alumina-silica gel(SiO2.Al203) to lead to an oxidation reduction reaction forming NO and H2SO4. This was a remarkable discovery as the NRFA bacteria we are working with, effectively reduce NO into harmless forms, and hence we needed NO as the output from this tank. Thus, the NO rich gas from this tank is passed into the next tank containing NRFA bacteria for further action. H2SO4 produced is extracted and used for industrial purposes.  <A href="https://grabcad.com/library/prototype-iitd-1/files/Assem12.SLDASM"><FONT size="5"><STRONG>Click here to view the 3D model</STRONG></FONT></A><B> Prototype Results: </B><P> Tests were carried out on the prototype, taking in exhaust from a diesel burner and passing it through the system. The desooter tank worked even better than expected, collecting soot out from the exhaust (which was weighed after each run). NOx sensors placed at the edge of the silica gel tube confirmed that the gel was catalysing the reaction that we intended to do, converting NO2 to NO.

FUTURE SCOPE OF THE PROJECT AND TARGETS TO BE ACHIEVED: 
 
To optimize the design of the desooter tank and the entire contraption, to increase efficiency and reduce the cost of the process.  
To ultimately make the process so cheap and efficient that it can replace SCR in big engines, industries and power plants. 
Use the soot (obtained as dissolved in water) to make ink to ultimately make the process self-sustainable and bring down the processing cost to negligible amounts by accounting for commercial selling of this ink obtained. This will also further reduce environmental repercussions as soot now is converted into a useful product, rather than being dumped into the environment 
To work on developing and optimizing the desooter tank design to reduce power consumption in the blower, as well as the heat exchanger.  
Hence, through optimization, we aim to finally reach a design which gets the engine to work efficiently at even higher trade-off points. Conventionally, diesel engines are operated at temperature ranges which are not very high, as operation at high temperature leads to higher amount of NOx in the exhaust. Thus, they are always operated at a trade-off temperature lower than these high temperature values, to reduce NOx emission, however, also leading to a decrease in the engine’s efficiency, and increased amounts of CO and soot emission (due to more unburnt C content in the exhaust now). SC-R’s are not efficient enough, and thus, do not lead to a significant increase in the trade-off temperature. Thus, if an efficient and cost effective design of this NRFA driven pollution crusader device is realized, it would bring about a remarkable change in the automobile and diesel engine industry, and increase the efficiency of these devices, while reducing both the cost, and carbon footprints on the environment.	
 </P><H1 style="font-family:'Trebuchet MS', 'Lucida Grande', 'Lucida Sans Unicode', 'Lucida Sans', Tahoma, sans-serif;color:white;font-size:250%;padding-top:18px;">Conclusion</H1><H2 style="font-family: 'Roboto', sans-serif;color:white;font-size:150%;vertical-align:text-top;">

We were able to materialize our dream of designing the perfect &quot;Pollution Crusader&quot; prototype model, using the genetically engineered Eco.coli bacteria, to a very good extent, if not fully.
 
From the viewpoint of Synthetic Biology, we succeeded in obtaining the desired clones. Indeed, we were able to express relevant genes as per our requirements. We have been successful in cloning a total of 5 relevant bio bricks:<LI>   BBa_K1866006 (P + RBS + NrfA) </LI><LI>   BBa_K1866007 (NosZ + SYFP) </LI><LI>BBa_K1866008 (P + RBS + NrfA + SYFP) </LI><LI>BBa_K1866009 (P + RBS + SQR) </LI><LI>BBa_K1866010 (P + RBS + NosZ + SYFP)</LI>
  
We have submitted these parts in the iGEM registry for future use by any iGEM team. 
 
While working in the project, towards the end, the main complexity we faced was of Heme protein. It is so because most of the enzymes we were using were Cytochrome C Hemo proteins, which require a Heme group for thein 100% activity. While scavenging the research papers, we found out that nobody, till date had found a perfect solution for the over expression of Heme protein. Under the constraints of time, we managed to compile the literature, designed the perfect strategy and using Copasi (modelling tool), we found some novel pathways to regulate and manipulate the increase in Heme production and hence the activity of the Heme proteins.  We did manage to extensively characterize the NrfA cloned gene, which served as the nitrate reductase. We found out that even without using any extracellular or intracellular Heme over production, our clones were highly efficient in reducing the nitrite and subsequently converting it into ammonia. Finally, we came to the conclusion that while expressing NrfA in a constitutive fashion, there was increased conversion of NOx to ammonia. 
 
In the mechanical half of the project, we designed a prototype which in effect served as the perfect “Pollution Crusader”.  We tested the model against the exhaust emitted by a diesel engine. Although the efficiency wasn’t that high, it did show positive results. We observed considerable reduction in the amount of NOx and SOx reduced. In a nutshell, our prototype model is capable of removing the harmful soot particles, gases like NOx and SOx and also the particulate matter present in the exhaust gases of the various engines being used. The prototype model designed will not only serve as an alternative for the existing technology, but has the potential to bring revolutionary changes in the upcoming days.

</H2></DIV></DIV></DIV></BODY></HTML>