<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Paris_Bettencourt_Newsletter1"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Paris_Bettencourt/Newsletter1">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Paris_Bettencourt/Newsletter1&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Paris_Bettencourt/Newsletter1&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Paris_Bettencourt/Newsletter1&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Paris_Bettencourt/Newsletter1" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Paris Bettencourt/Newsletter1</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><DIV class="navmenubkg"><DIV id="navmenubox"><UL id="nav"><LI><A href="https://2014.igem.org/Team:Paris_Bettencourt/Project">Project</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Project/Odor_Library">SmellThe Roses</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Project/TMAU">SomethingFishy</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Project/Foot_Odor">GoodyTwo Shoes</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Project/Eliminate_Smell">Don'tSweat It</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Project/Old_People_Smell">Teen Spirit</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Project/Art">Design Practice </A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Project/Bioinformatics">Bioinformatics</A></LI></UL><LI><A href="https://2014.igem.org/Team:Paris_Bettencourt/Achievement">Achievements</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Results">Results</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Project/Interlab_Study">Interlab Study</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Medal">MedalFulfilment</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Parts">Parts</A></LI><LI><A href="https://2014.igem.org/Team:Paris_Bettencourt/Policy">Policy </A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Safety">Safety</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Consent_Form">Consent Form</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Patenting">Patenting</A></LI><LI><A href="https://2014.igem.org/Team:Paris_Bettencourt/Human_Practice">Practices</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/MOOC">MOOC</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Citizen_Science">CitizenScience</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Facts_Smell">The Senseof Smell</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Facts_Micriobiome">The HumanMicriobiome</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Minder">Minder</A></LI><LI><A href="https://2014.igem.org/Team:Paris_Bettencourt/Collaboration_Main">Collaborations </A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Collaboration">Collaborations </A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/The_Smell_Game">The SmellGame</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Newsletter">Newsletters</A></LI><LI><A href="https://2014.igem.org/Team:Paris_Bettencourt/Notebook_Main">Notebook</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Notebook">Daily Note</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Protocols">Protocols</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Bibliograpy">Bibliography</A></LI><LI><A href="https://2014.igem.org/Team:Paris_Bettencourt/Team_Main">Team </A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Team">Our Team </A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Attributions">Attributions</A></LI><LI><A class="liste" href="https://igem.org/Team.cgi?year=2014&amp;team_name=Paris_Bettencourt">Official TeamProfile </A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Acknowledgements">Acknowledgements</A></LI><DIV id="twnoticon" class="sidelogos"><A class="twitter-timeline" width="250" height="400" data-dnt="true" href="https://twitter.com/iGEM_Paris" data-widget-id="368144501119795201">@iGEM_Paris</A></DIV></DIV></DIV></DIV><P id="top"><A id="link" href="#topheader">Back to top</A></P><DIV id="shadow"><UL id="menu-accordeon"><LI><A href="https://2014.igem.org/Team:Paris_Bettencourt/Newsletter">Newsletter</A></LI><LI><A href="https://2014.igem.org/Team:Paris_Bettencourt/Newsletter1">- Newsletter 1</A></LI><LI><A href="https://2014.igem.org/Team:Paris_Bettencourt/Newsletter2">- Newsletter 2</A></LI><LI><A href="https://2014.igem.org/Team:Paris_Bettencourt/Newsletter3">- Newsletter 3</A></LI><LI><A href="https://2014.igem.org/Team:Paris_Bettencourt/Newsletter4">- Newsletter 4</A></LI></UL></DIV><DIV id="map1"><TABLE id="tablelist"><TBODY><TR><TD id="case3">iGEM 2014Weekly newsletter</TD><TD id="case3">ETH ZürichGoettingenGothenburgParis Bettencourt 2014</TD><TD id="case3">Polytechnic University of ValenciaTJU-China iGEM TeamUI-IndonesiaUSTC_China iGEM team</TD></TR></TBODY></TABLE></DIV><DIV><H5>Newsletter n°1</H5><H6>from July 28th to August 10th 2014</H6><P id="intro"><B>Hi, all the iGEMers.</B>
			I am happy to announce the first issue of our newsletter. This biweekly newsletter is cofounded by Paris Bettencourt iGEM team and Xiamen University iGEM team. Currently, we have 9 teams participating and are expecting more for the later issues. The newsletter has two major parts: project update and team member introduction. The teams should be writing about one of the parts every two weeks. We hope that through this activity, the teams will be able to keep better record of their own projects, gain feedback from their peers and get to know other young synthetic biologists. There will also be special issues with interviews of iGEM alumni and past advisors from college teams.</P><P id="intro">This first issue covers the general information of both the project and the introduction of each team. It also includes a question component, which includes the topics that each team wants to get feedback on from other teams. We highly encourage everyone to think about the questions and provide insights. Please send your thoughts to one member of the team and we will collect them a week from today.</P><P id="intro"><B>We wish the best luck to everyone competing in iGEM and hope this newsletter can be a valuable resource.</B></P></DIV><DIV id="eth"><H5>Newsletter n°1</H5><H6>from July 28th to August 10th 2014</H6><H4>ETH Zürich</H4><TABLE id="table"><TBODY><TR><TD id="case1"><P><B>OUR TEAM</B></P></TD><TD id="case2"><P id="text"><B>ETH Zürich is taking part to iGEM since it was opened to the
				European universities (for the second edition).</B></P><P id="text">The team has known several successes in the Information
				Processing track of the competition. This year, we are seven highly
				motivated students from different backgrounds, aiming to rock the
				Information Processing track once again. The interaction between
				wet lab and dry lab is crucial for our team.
				</P></TD></TR><TR><TD id="case1"><P><B>OUR PROJECT</B>
				Make a Sierpinski triangle pattern
				appear in a grid
				Conjugate quorum sensing and
				logic gates in bacterial colonies
				Implement an XOR gate in an E.
				coli
				Characterize integrases (retrieve
				missing parameters)
				Study quorum sensing mechanism
				aiming to lower the leakiness
				Be able to predict accurately the
				system’s behavior.</P><P><I>Questions it raises :
				How can a complex pattern emerge from a
				simple logic rule?</I></P><P>What is to be considered as simple or
				complex?</P><P>How can leakiness, noise and lack of
				robustness in biological systems be
				handled?</P></TD><TD id="case2"><P id="text"><B>Emergence of complex patterns in nature is a fascinating
				phenomenon, which is not fully understood yet. Some sea snail
				shells indeed present natural mosaics, made out of only two
				colors seemingly following a simple rule to form a pattern.</B></P><P id="text">Our project Mosaicoli aims to implement a similar complex
				behavior. It challenges the high noise and the low robustness
				shown by biological systems. This cellular automaton consists of
				a regular grid of E. coli colonies. Either in an ON or OFF state,
				each colony interacts with its neighbours according to a fixed logic
				rule. Mosaicoli exploits quorum sensing, integrase logic gates
				and microfluidics to generate the Sierpinski triangle pattern.
				Communication between colonies is achieved by AHL molecules
				and subsequent updates in a cell state are determined by a discrete
				XOR integrase gate. The final state is read out via fluorescence,
				which collectively depicts a complex pattern. Mosaicoli deals with
				the leakiness of biological subunits to create a reliable complex
				entity, which can be considered as a first step towards the biological
				computer.</P></TD></TR></TBODY></TABLE><DIV id="media"><P><A id="link" href="https://twitter.com/ETH_iGEM">@ETH_iGEM</A><A id="link" href="https://www.facebook.com/iGEM.ETH.Zurich">iGEM ETH Zürich</A><A id="link">igem2014@bsse.ethz.com</A></P></DIV></DIV><DIV id="Goettingen"><H5>Newsletter n°1</H5><H6>from July 28th to August 10th 2014</H6><H4>Goettingen</H4><TABLE id="table"><TBODY><TR><TD id="case1"><P><B>OUR TEAM</B></P></TD><TD id="case2"><P id="text"><B>University of Göttingen, Germany.</B></P><P id="text">We come from different countries: Germany, China, Iran, Lebanon,
					India and Mexico. The group formed when the team of 2013 gave a
					presentation of their work and how they did in the Jamboree (they
					won the award for the best presentation in the overgrad division at
					the World Championship).</P></TD></TR><TR><TD id="case1"><P><B>OUR PROJECT</B></P><P><I>Questions it raises :
				Is it possible to select a short peptide with
				affinity towards fungal surface proteins out
				of a short peptide library?
				</I></P></TD><TD id="case2"><P id="text"><B>We intend to develop a fast screening technique for different
				invasive fungi, such as Aspergillus fumigatus, Candida albicans
				and Aspergillus nidulans.</B></P><P id="text">The idea is to tag the invasive fungal cells with a special protein
				marker that allows an easy detection of fungi. By modifying the
				protein marker with a proper signal molecule, we will be able to
				accomplish a fast and direct elimination of invasive fungi.</P><P id="text">We want to prove the principle that it’s possible to obtain short
				peptides with affinity towards a protein of interest through a highthroughput
				screening method. The proteins of interest in our project
				are fungal surface proteins, but they can be any other protein, such
				as tumor markers or other pathogenic microorganism surface
				proteins.</P><P id="text">We also want to develop a diagnostic tool with those peptides.
				Here, our methodology is straightforward and it is mainly a proofof-
				principle: we will tag a GFP marker and check for fluorescence.
				However, if this is successful, in the long term and after other sorts
				of chemical modifications, these peptides can be further applied
				as diagnostic and even therapeutic tools.</P><P id="text">We are running for the Health and Medicine track.</P></TD></TR></TBODY></TABLE><DIV id="media"><P><A id="link" href="https://twitter.com/iGEMGoe">@iGEMGoe</A><A id="link">igem2014@uni-goettingen.de</A></P></DIV></DIV><DIV id="Gothenburg"><H5>Newsletter n°1</H5><H6>from July 28th to August 10th 2014</H6><H4>Gothenburg</H4><TABLE id="table"><TBODY><TR><TD id="case1"><P><B>OUR TEAM</B></P></TD><TD id="case2"><P id="text"><B>The Gothenburg team is currently working in the Systems and
					Synthetic Biology group, in the Department of Chemical and
					Biological Engineering at Chalmers.</B></P><P id="text">Our team reflects the diversity of Chalmers and the iGEM competition;
					from the total of 10 team members, 5 of us are not Swedish, with
					different nationalities such as Brazilian and Indian. We are all
					students at Chalmers, mostly at undergraduate level, and with
					different backgrounds. Being the second Chalmers iGEM team is
					both challenging and rewarding. We are excited and inspired by the
					experience of the iGEM competition and having fun while learning!</P></TD></TR><TR><TD id="case1"><P><B>OUR PROJECT</B></P><P>Our team goal with the iGEM
					project is to construct a yeast
					generation counter. The idea is
					that each time the cell divides
					a different florescent protein is
					produced. Therefore by examining
					the cell under a microscope or in a
					flow cytometer one can determine
					how many times the cell has
					divided.</P></TD><TD id="case2"><P id="text"><B>Nowadays the determination of replicative age of yeast cells is
				done by counting the budding scars of each cell in a microscope,
				a process time consuming and not effective.</B></P><P id="text">Our goal would be achieved by constructing a logical AND gate in
				the cell where the input signals consist of a cyclin activated dCas9-
				VP64 and a guide RNA (gRNA) signal from the previous cell cycle.
				The output response consist of a different fluorescent protein and
				a new gRNA molecule.</P><P id="text">dCas9-VP64 is an engineered turntable transcription factor only
				active when dimerized with an interchangeable gRNA molecule.
				The gRNA also determines the specificity of the transcription
				factor which enables the dCas9-VP64 to activate different
				genes depending on the sequence of the gRNA molecule and
				the promoter. gRNA consists of two parts, a scaffold and a 20
				bp Specificity Determinant Sequence (SDS) on the 5’ end. The
				scaffold constitutes the majority of the gRNA molecule and
				gives it its structure whereas the SDS binds to the target site in
				the gene promoter. Cyclins are proteins that are involved in the
				progression of the cell cycle, therefore activated at specific times
				[5]. To mimic the specific production pattern of cyclins the dCas9-
				VP64 gene is placed under the control of a yeast cyclin promoter.
				Therefore the Cas9 will be produced in the G1 phase. When the
				Cas9 and gRNA dimerize and the transcription factor is activated,
				it in turn activates the transcription of a new fluorescent protein
				and a new gRNA molecule to act as a memory for the next cycle.
				Once this age counter is implemented, it would be possible to sort
				cells according to their replicative age automatically with a flow
				cytometer device.</P></TD></TR></TBODY></TABLE><DIV id="media"><P><A id="link">pacheco@student.chalmers.se</A></P></DIV></DIV><DIV id="Paris-Bettencourt"><DIV><H5>Newsletter n°1</H5><H6>from July 28th to August 10th 2014</H6><H4>PARIS BETTENCOURT</H4><TABLE id="table"><TBODY><TR><TD id="case1"><P><B>OUR TEAM</B></P></TD><TD id="case2"><P id="text"><B>Paris-Bettencourt iGEM team is part of the Center for Research
					and Interdisciplinarity (CRI) in Paris.</B></P><P id="text">It was the first French iGEM team, and has long history of success
					in the competition. The team won the World Jamboree Grand
					Prize Award last year with the Fight Tuberculosis with Modern
					Weapons project. When deciding on a theme, above all else, we
					value creativity and the potential of our project to benefit society.</P></TD></TR><TR><TD id="case1"><P><B>OUR PROJECT</B></P><P>Treating strong body odor using
				probiotics, producing a low cost
				perfume produced by bacteria
				and setting up a database of
				microbiome correlated to body
				odor</P><P><I>Questions it raises :
				What kind of regulation would you like to
				see for GMO food/products?
				(Ex: mandatory labeling)</I></P><P>Would you buy a perfume that is produced
				by E. coli? If not, why?</P><P>What would you like to know about body
				odor?
				(Ex: does your body odor affect who is
				sexual attracted to you?)</P></TD><TD id="case2"><P id="text"><B>This year, the Paris-Bettencourt team has decided to explore a
				topic that has rarely been approached by synthetic biologists in
				this competition: odor.</B></P><P id="text">We are currently working on several projects related to odor:
				Engineering bacteria to produce an odor library
				Designing probiotics and cosmetic products aimed to help people
				with strong body odor.
				Treating genetic diseases leading certain persons to have strong
				malodor.
				Launching a citizen science project aiming to correlate armpit
				microbiome data with malodor.</P><P id="text">As for human practice, we are focusing this year on open-source
				biology and hope to get contribution from not only team members,
				but also the public, through collecting samples, crowdsourcing
				and updating of project development. At the same time, we are
				building a MOOC course for high school students. As popular as
				iGEM is above undergraduates and graduates, it is still quite new
				in high school. This course will include an introduction of Synthetic
				Biology, the social concern with it and work students through the
				important steps to form an iGEM team.
				In addition to the scientific contribution to iGEM, we are aiming
				at improving scientific communication and opening the project to
				the general public. To facilitate open science the team is building
				an online collaborative platform to involve citizen scientists in the
				process. We believe, everyone has something to contribute to the
				development of science!</P></TD></TR></TBODY></TABLE><DIV id="media"><P><A id="link" href="https://twitter.com/iGEM_Paris">@iGEM_Paris</A><A id="link" href="https://www.facebook.com/ParisiGEM2013">iGEM Paris Bettencourt</A><A id="link">2014.igem.Parisbettencourt@gmail.com</A></P></DIV></DIV><DIV id="Valencia"><DIV><H5>Newsletter n°1</H5><H6>from July 28th to August 10th 2014</H6><H4>Polytechnic University of Valencia</H4><TABLE id="table"><TBODY><TR><TD id="case1"><P><B>OUR TEAM</B></P></TD><TD id="case2"><P id="text"><B>We are a team uniquely composed by students from the
					Polythecnic University of Valencia</B>with the support of the Spanish
					National Research Council. In this iGEM edition we dare to work
					with plants and hope our work will encourage other teams to do
					so in the future.</P></TD></TR><TR><TD id="case1"><P><B>OUR PROJECT</B></P><P>Avoiding damage in crops caused
				by pests, using pheromoneproducing
				plants that disrupt
				insects mating and therefore avoid
				laying and damage caused by
				larvae.</P><P><I>Questions it raises :
				In what fields do you think that using
				Genetically Modified Organisms (GMOs)
				could improve our lifestyle?
				(Ex: Safer drugs production, bioremediation)</I></P><P>Would you accept using GMOs for noneating
				purposes? If not, why not?</P><P>What do you want to know about crop
				protection against pests?
				(Ex: How do insects behave in crops?)</P></TD><TD id="case2"><P id="text"><B>This year, our team is developing an environmental friendly
				approach to control insect pests: The Sexy Plant.</B></P><P id="text">The gist of the project we are working on is focused on the production
				of insect pheromones in the plant Nicotiana benthamiana. Our
				main efforts are aimed at avoiding the insect laying and the
				damage caused by insect larvae in crops. Since pests are one of
				the main causes of damage and the use of pesticides is getting
				more and more unpopular due to their environmental effects, our
				goal is to release the sufficient amount of pheromones into the air
				to cause mating disruption among insects, thus, male insects will
				not be able to find and mate with female insects, in the same way a
				soldier dressed in camouflage is difficult to be seen in a rainforest.</P><P id="text">As for human practices, we are creating surveys about GMOs and
				Synthetic Biology in collaboration with other iGEM teams from
				different countries. This way, surveys from all the teams can reach
				a higher number of people with different regional culture, so the
				results are more global.</P><P id="text">We know the importance of an early introduction in science.
				For that reason, we have carried out activities to increase kid’s
				interest in science in our university summer school and we are
				creating a project to increase interest in iGEM for High School in
				collaboration with more iGEM teams.</P></TD></TR></TBODY></TABLE><DIV id="media"><P><A id="link" href="https://twitter.com/ValenciaUPViGEM">@ValenciaUPViGEM</A><A id="link" href="https://es-es.facebook.com/ValenciaUPViGEM2014">Valencia UPV iGEM 2014</A><A id="link">valencia.upv.igem@gmail.com</A></P></DIV></DIV><DIV id="TJU-China"><DIV><H5>Newsletter n°1</H5><H6>from July 28th to August 10th 2014</H6><H4>TJU-China iGEM Team</H4><TABLE id="table"><TBODY><TR><TD id="case1"><P><B>OUR TEAM</B></P></TD><TD id="case2"><P id="text">Tianjin University IGEM team is one of the first entries to attend the IGEM
					competition in China. We started our IGEM journey from 2007 and we won a
					gold medal on our first try. Last year, our project AlkSensor won the gold medal
					and Best Poster award.
					With the increasing times of successful or failed attempts in the competition.
					We have accumulated much meaningful experience in the field of synthetic
					biology. With all of the experiences and joint efforts, a more united, efficient and
					passionate team is built up.</P><P id="text">This year we form a team of 16 students from the school of chemical engineering.
					Not only seeking help from our instructor, we are planning to launch a series of
					interchange activities with other teams and Chinese Academy of Sciences. We
					also pay attention to the potential impact of our project to our society. With our
					joint efforts, we hope, by exploring the relationship between science and life, we
					can make the world more wonderful.</P></TD></TR><TR><TD id="case1"><P><B>OUR PROJECT</B></P><P>An efficient method to transduce biological sense into electric signal.</P><P>To detect the concentration of certain micromolecules quantitively in almost real time.</P><P><I>Questions it raises :
				1.If we set up this platform above, what aspects can it apply to?</I></P><P>2,What methods can be applied to real-time
				detection and analysis to some molecules
				that are difficult to detect in traditional
				ways?</P><P>3.How to maintain the stable structure of curli fiber when it functioned?</P></TD><TD id="case2"><P id="text"><B>Biosensors are becoming more and more indispensable tools in
				life science, medicine, chemistry and biotechnology, which are
				greatly enhanced by synthetic biology.</B></P><P id="text">Yet in the bio-sensing procedure, the limitations of signal output
				methods restrict the usage and practicality of biosensors. Wildly
				used reporters such as chromophore and fluorescent protein
				which requires laboratorial equipments to measure often result in
				the loss of accuracy and Immediacy.
				Thus our project is focus on the bio-signal transformation to
				diversify output methods of biosensor. In account of the high
				maturity of silicon analysis, it will be a great advantage to transduce
				biological sense into electric signal.
				Following this idea, we design a transducer which can convert the
				change of gene expression level that stimulated by inducer directly
				into the electric signal via inductive synthesis（or destroy）of
				nanowire between the electrodes. Curli fiber, a well characterized
				amyloid fiber which forms β-sheet-rich amyloid fibers, seems
				to be an ideal foundation of our nanowire structure. CsgA, the
				modified morphon protein of curli fiber with the ability to bind
				nano gold particles, will play the role as the conductor.
				Once we synthesize conductive curli fiber, there are two approach
				to carry out the target sensing, by inductively synthesizing the
				conductive fiber between the electrodes to turn on a switch for
				connecting a open circuit, or inductively expressing the gene
				coding the curli-degrading enzyme to destruct the pre-synthesized
				nanowire. The change of the electric current shall represent the
				concentration of inducer.</P></TD></TR></TBODY></TABLE><DIV id="media"><P><A id="link">tjuigem2014@163.com</A></P></DIV></DIV><DIV id="UI-Indonesia"><DIV><H5>Newsletter n°1</H5><H6>from July 28th to August 10th 2014</H6><H4>UI-Indonesia</H4><TABLE id="table"><TBODY><TR><TD id="case1"><P><B>OUR TEAM</B></P></TD><TD id="case2"><P id="text">We met for the first time in January 2014 and we’re all
					undergraduate students from multidisciplinary backgrounds—
					medicine, biology, bioengineering, and also chemical
					engineering.</P></TD></TR><TR><TD id="case1"><P><B>OUR PROJECT</B></P></TD><TD id="case2"><P id="text"><B>Cholerae is major cause of death in children during first year
				of life in tropical country. Cholera is tropical neglected disease
				because it causes significant illness more than one billion people
				and cholera outbreak rises when the disaster comes, because of
				bad water sanitation.</B></P><P id="text">In high cell density, Vibrio cholera formed biofilm to protect them
				from milieu exterior disturbance therefore antibiotic and other
				drugs treatment can not be effective against cholera infection. Here,
				we proposed a novel strategy to degrade cholera biofilm by using
				principles of synthetic biology. Our team genetically engineered
				E. coli to specifically detect quorum sensing (QS) molecule from
				Vibrio cholera. We use E.coli because it is normally found in human
				intestine. Mutagenic E.coli able to detect specific quorum sensing
				molecule by expressed receptor which is naturally expressed in
				transmembrane domain of Vibrio cholera, this receptor called
				CqsS receptor, a group of histidine kinases receptor. During
				biofilm formation Vibrio cholera will secret quorum sensing
				molecule called Vibrio cholera autoinducer 1 (CAI-1) which is bind
				to CqsS receptor in E.coli. CAI-1 molecule act as “find me signal”
				for E.coli, at the same time the binding of QS molecule to receptor
				in E.coli, it will active motile gene, CheZ which is enable E.coli
				migrate to source of infection, besides activate motile gene, it will
				also activate killing gene which is enable E.coli producing biofilm
				enzyme degrading matrix such as: alpha amylase, substilisin, and
				nuclease. After release enzyme degrading matrix our E.coli will
				secrete peptide 1018, an innate immunity peptide from human
				cathelicidin which is successfully expressed in E.coli as novel
				antibiotic peptide to kill Vibrio cholera directly. By interconnecting
				the principles of pathogen-directed motility with the dual system
				degrading and killing biofilm, we gained significantly develop
				killing activity to combat biofilm formation and further severe
				infection.</P></TD></TR></TBODY></TABLE><DIV id="media"><P><A id="link">ui.igem2014@gmail.com</A></P></DIV></DIV><DIV id="USTC_China"><DIV><H5>Newsletter n°1</H5><H6>from July 28th to August 10th 2014</H6><H4>USTC_China iGEM team</H4><TABLE id="table"><TBODY><TR><TD id="case1"><P><B>OUR TEAM</B></P></TD><TD id="case2"><P id="text"><B>Our team, founded in 2007, joined the iGEM competition as one
					of the earliest teams in China and we got the 2nd runner up and
					fundamental advance project that year.</B></P><P id="text">Nowadays, we still spare no time trying to make our project come
					true and propagate the biological knowledge to public.</P></TD></TR><TR><TD id="case1"><P><B>OUR PROJECT</B></P><P>-Figure out the optimum conditions
				of C.Crescentus development,
				transformation and expression.
				-Develop the holdfast in C.Crescentus.
				-Extract the gene regulating the
				expression of holdfast in C.Crescentus
				and build parts with the regulating
				system.
				-Construct the light sensing-response
				system.
				-Guide the accomplished system into
				E.coli and C.Crescentus.
				-Pre-test the function and efficiency of
				ribozymes.
				-Plan the industrialization and further
				development in biological imaging
				industry.
				-Design the projector specialized for
				biological imaging</P><P><I>Questions it raises :</I></P><P>Do you wanna try photographic film
				produced by bacteria?</P><P>What do you care most when using this
				product?</P><P>How do you like the development of other
				fields’ technology into biological system?</P><P>Will you believe the future when everything
				we use and require is produced by
				organism?</P></TD><TD id="case2"><P id="text"><B>This year, our team attempt to combine the photographic film
				with biological system triggered by much careful investigation
				of bacteria imaging study and light inducible response both in
				recent research and iGEM team work.</B></P><P id="text">We would like to call the system as Fast and Accurate Biological
				Chromatic Sensing-Imaging System(FABCSIS) by respectively
				joint red-green-blue (RGB) light sensing proteins with RGB
				florescent proteins into a new pedestal Caulobacter Crescentus,
				a kind of bacterium possessing adhesive holdfast which we can
				utilize for more stable expression in distinct surroundings like
				the water flow. And we are determinant to improve the system
				with ribozymes regulation to enhance the accuracy and speed of
				perception and response. At last, we will construct a customized
				projector to measure the efficiency of system and make a plan for
				the possibility of industrialization in the future. The FABCSIS, as we
				wish, will realize the chromatic imaging system firstly held in the
				media, regarding each individual as a pixel just like photographing.
				We believe our work will drive the further study and development
				of imaging system in other living organisms and one day biological
				photographic films will appear in everybody’s daily life to change
				the world for its lowcost and high resolution.</P><P id="text">This year, we attempt to improve the technology in biological
				system and hope our job could influence the development and
				study of biological imaging chromatically. Besides, we seized
				many opportunities to introducing genetic engineering and
				synthetic biology to students who are intrigued in science, parents
				and people who are worried about GMO without any biological
				conception. We hope one day in China, everyone could make a
				decision scientifically with knowledge and judgement with our tiny
				efforts.</P></TD></TR></TBODY></TABLE><DIV id="media"><P><A id="link">stark@mail.ustc.edu.cn</A></P></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Paris_Bettencourt/Newsletter1">http://2014.igem.org/Team:Paris_Bettencourt/Newsletter1</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Paris_Bettencourt/Newsletter1" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Paris_Bettencourt/Newsletter1" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Paris_Bettencourt/Newsletter1&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Paris_Bettencourt/Newsletter1&amp;oldid=118678" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>