<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Paris_Bettencourt_Protocols"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Paris_Bettencourt/Protocols">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Paris_Bettencourt/Protocols&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Paris_Bettencourt/Protocols&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Paris_Bettencourt/Protocols&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Paris_Bettencourt/Protocols" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Paris Bettencourt/Protocols</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><DIV class="navmenubkg"><DIV id="navmenubox"><UL id="nav"><LI><A href="https://2014.igem.org/Team:Paris_Bettencourt/Project">Project</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Project/Odor_Library">SmellThe Roses</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Project/TMAU">SomethingFishy</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Project/Foot_Odor">GoodyTwo Shoes</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Project/Eliminate_Smell">Don'tSweat It</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Project/Old_People_Smell">Teen Spirit</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Project/Art">Design Practice </A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Project/Bioinformatics">Bioinformatics</A></LI></UL><LI><A href="https://2014.igem.org/Team:Paris_Bettencourt/Achievement">Achievements</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Results">Results</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Project/Interlab_Study">Interlab Study</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Medal">MedalFulfilment</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Parts">Parts</A></LI><LI><A href="https://2014.igem.org/Team:Paris_Bettencourt/Policy">Policy </A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Safety">Safety</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Consent_Form">Consent Form</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Patenting">Patenting</A></LI><LI><A href="https://2014.igem.org/Team:Paris_Bettencourt/Human_Practice">Practices</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/MOOC">MOOC</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Citizen_Science">CitizenScience</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Facts_Smell">The Senseof Smell</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Facts_Micriobiome">The HumanMicriobiome</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Minder">Minder</A></LI><LI><A href="https://2014.igem.org/Team:Paris_Bettencourt/Collaboration_Main">Collaborations </A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Collaboration">Collaborations </A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/The_Smell_Game">The SmellGame</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Newsletter">Newsletters</A></LI><LI><A href="https://2014.igem.org/Team:Paris_Bettencourt/Notebook_Main">Notebook</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Notebook">Daily Note</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Protocols">Protocols</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Bibliograpy">Bibliography</A></LI><LI><A href="https://2014.igem.org/Team:Paris_Bettencourt/Team_Main">Team </A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Team">Our Team </A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Attributions">Attributions</A></LI><LI><A class="liste" href="https://igem.org/Team.cgi?year=2014&amp;team_name=Paris_Bettencourt">Official TeamProfile </A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Acknowledgements">Acknowledgements</A></LI><DIV id="twnoticon" class="sidelogos"><A class="twitter-timeline" width="250" height="400" data-dnt="true" href="https://twitter.com/iGEM_Paris" data-widget-id="368144501119795201">@iGEM_Paris</A></DIV></DIV></DIV></DIV><DIV id="shadow"><UL id="menu-accordeon"><LI><A onclick="showprot('#prot1')">Heat Shock Transformation of <I>E. coli</I></A></LI><LI><A onclick="showprot('#prot2')">CaCl2 Competent Cells</A></LI><LI><A onclick="showprot('#prot3')">Electroporation</A></LI><LI><A onclick="showprot('#prot4')">Miniprep</A></LI><LI><A onclick="showprot('#prot5')">PCR Purification</A></LI><LI><A onclick="showprot('#prot6')">Gel Purification</A></LI><LI><A onclick="showprot('#prot7')">Colony PCR</A></LI><LI><A onclick="showprot('#prot8')">Glycerol Stocks</A></LI><LI><A onclick="showprot('#prot9')">Ligation</A></LI><LI><A onclick="showprot('#prot10')">Isolation of indigo</A></LI><LI><A onclick="showprot('#prot11')">Transformation of <I>Corynebacterium</I></A></LI><LI><A onclick="showprot('#prot12')">DNA Isolation</A></LI></UL></DIV><DIV id="corps"><DIV id="prot1" class="protocolactive"><H3> Heat Shock Transformation of <I>E. coli</I></H3><P>This protocol can be used to transform chemically competent (i.e. from CaCl2) with a miniprepped plasmid or a ligation product.</P><H5>Note: Never vortex competent cells. Mix cells by gentle shaking.</H5><P><OL><LI>Thaw competent cells on ice. These can be prepared using the CaCl2 protocol.</LI><LI>Place 20 ul of cells in a pre-chilled Eppendorf tube.
	    </LI><LI><U>For an Intact Vector:</U> Add 0.5 ul or less to the chilled cells</LI><LI><U>For a Ligation Product:</U> Add 2-3 ul to the chilled cells.</LI><LI>Mix gently by flicking the tube.</LI><LI>Chill on ice for 10 minutes. <EM>This step is optional, but can improve yields when transforming a ligation product.</EM></LI><LI>Heat shock at 42 °C for 30 seconds.</LI><LI>Return to ice for 2 minutes.</LI><LI>Add 200 ul LB medium and recover the cells by shaking at 37 °C.
	    Another rich medium can substitute for the recovery.
	    The recovery time varies with the antibiotic selection.
	    Ampicillin: 15-30 minutes
	    Kanamycin or Spectinomycin: 30-60 minutes
	    Chloramphenicol: 60-120 minutes 
	  </LI><LI>Plate out the cells on selective LB.
	    Use glass beads to spread the cells.
	    The volume of cells plated depends on what is being transformed.</LI><LI><U>For an Intact Vector:</U> High transformation efficiencies are expected. Plating out 10 ul of recovered cells should produce many colonies.</LI><LI><U>For a Ligation Product:</U> Lower transformation efficiencies are expected. Therefore you can plate the entire 200 ul volume of recovered cells.</LI>
	    Note: 200 ul is the maximum volume of liquid that an LB plate can absorb.
	  <LI>Incubate at 37 °C. Transformants should appear within 12 hrs.</LI></OL></P></DIV><DIV id="prot2" class="protocol"><H3>CaCl2 Competent Cells</H3><P>This protocol makes 4 ml of competent cells, and can be easily scaled up to make more. The cells are typically stored in 110 ul aliquots, so this will make about 35 tubes. A typical transformation uses 20 ul of cells.</P><H5> Note: Never vortex competent cells. Resuspend by pipetting with large Pasteur pipettes.</H5><P><OL><LI><B><U>The night before</U></B>, inoculate a 5 ml culture and grow overnight with selection.</LI><LI><B><U>The day of</U></B> the experiment dilute cells ~ 1:200 into selective media.For this example add 250 ul to 50 ml of selective media.Note: The protocol is easily scaled to increase the number of cells.</LI><LI> Grow the cells to an OD600 of 0.6 – 0.7.
	    Use a large flask, 500ml, for good aeration.
	    Use a baffled flask for fastest growth.
	    This takes about 3 hours depending on the cells.
	    Medium-heavy cloudiness by eye is fine.</LI><LI>Spin down the cells at 4 ºC, 4000 rpm, 15 minutes.
	    Note: Keep the cells at 4 ºC from now on.</LI><LI>Resuspend cells in 15 ml, ice-cold 100 mM CaCl2. 
	    Leave on ice 4 hours to overnight.</LI><LI>Spin down the cells at 4 ºC, 4000 rpm, 15 minutes.</LI><LI>Resuspend cells in 4 ml, ice-cold 100 mM CaCl2 + 15% glycerol.</LI><LI>Aliquot into pre-chilled Eppendorf tubes. Use immediately or store at -80ºC.
	    Note: Frozen cells are only good once.Do not refreeze cells once thawed.</LI></OL></P></DIV><DIV id="prot3" class="protocol"><H3> Electroporation </H3><H4>Preparation of Electrocompetent Cells</H4><P>
	Competent cells should never be vortexted, as this will cause them to lyse
	and release salts into the media. Resuspend cells by pipeting up and down with a large
	pasteur pipet. Once they are chilled, cells should be continuously cold.
	<OL><LI>The night before the transformation, start an overnight culture of cells.
	    5 ml LB Amp.
	  </LI><LI>The day of the transformation, dilute the cells 100X.
	    100 ml LB Amp.
	    Grow at 30°C for about 90 minutes.
	  </LI><LI>Harvest the cells.
	    When the cells reach an OD600 of between 0.6 and 0.8.
	    Split the culture into 2x 50 ml falcon tubes, on ice.
	    Centrifuge at 4 °C for 10 min at 4000 rpm.
	  </LI><LI>Wash and combine the cells.
	    Remove the supernatant.
	    Resuspend the cells in 2x 25 ml of ice cold water.
	    Combine the volumes in a single 50 ml falcon tube.
	  </LI><LI>Wash the cells 2 more times.
	    Centrifuge at 4 °C for 10 min at 4000 rpm.
	    Resuspend in 50 ml of ice cold water.
	    Repeat.
	  </LI><LI>Wash and concentrate the cells for electroporation.
	    Centrifuge at 4 °C for 10 min at 4000 rpm.
	    Resuspend in 1-2 ml of ice cold water.
	    We will use 200 ul of washed cells per transformation.
	  </LI></OL></P><H4>Dialysis of PCR or Digestion Products</H4><P>
	DNA for electroporation must be free of salts to avoid arcing.
	<OL><LI>Float a filter in a Petri dish filled with water.
	    Millipore membrane filter 0.025 uM.</LI><LI>Pipet one drop of PCR product onto the filter.
	    200 ng is needed per transformation.
	    20 - 100 ul fits well on one filter.</LI><LI>Collect the drop after 30 - 45 minutes.
	    The volume will change, but the DNA is not lost.
	  </LI></OL></P></DIV><DIV id="prot4" class="protocol"><H3>Miniprep using <I>Thermo Scientific GeneJET Plasmid Miniprep Kit</I></H3><P>All centrifugations should be carried out in a table-top microcentrifuge at sup12000 x g
	<OL><LI>Resuspend the pelleted cells in 250 ul of the Resuspension Solution. Transfer the cell suspension to a microcentrifuge tube. The bacteria should be resuspended completely by vortexing or pipetting up and down until no cell clumps remain.<B><I>Note</I></B>. Ensure RNase A has been added to the Resuspension Solution.
	  </LI><LI>Add 250 ul of the Lysis Solution and mix thoroughly by inverting the tube 4-6 times until the solution becomes viscous and slightly clear.<B><I>Note</I></B>. Do not vortex to avoid shearing of chromosomal DNA. Do not incubate for more than 5 min to avoid denaturation of supercoiled plasmid DNA.</LI><LI>Add 350 ul of the Neutralization Solution and mix immediately and thoroughly by inverting the tube 4-6 times.<B><I>Note</I></B>.
	    It is important to mix thoroughly and gently after the addition of the Neutralization Solution to avoid localized precipitation of bacterial cell debris. The neutralized bacterial lysate should become cloudy.
	  </LI><LI>Centrifuge for 5 min to pellet cell debris and chromosomal DNA. 
	  </LI><LI>Transfer the supernatant to the supplied GeneJET spin column by decanting or pipetting. Avoid disturbing or transferring the white precipitate. 
	  </LI><LI>Centrifuge for 1 min. Discard the flow-through and place the column back into the same collection tube.<B><I>Note</I></B>.Do not add bleach to the flow-through.
	  </LI><LI>Add 500 ul of the Wash Solution (diluted with ethanol) to the GeneJET spin column. Centrifuge for 30-60 seconds and discard the flow-through. Place the column back into the same collection tube. 
	  </LI><LI>Repeat the wash procedure (step 7) using 500 ul of the Wash Solution. 
	  </LI><LI>Discard the flow-through and centrifuge for an additional 1 min to remove residual Wash Solution. This step is essential to avoid residual ethanol in plasmid preps. 
	  </LI><LI>Transfer the GeneJETspin column into a fresh 1.5 ml microcentrifuge tube. Add 50 ul of the Elution Bufferto the center of GeneJET spin column membrane to elute the plasmid DNA. Take care not to contact the membrane with the pipette tip. Incubate for 2 min at room tempera ture and centrifuge for 2 min.<B><I>Note</I></B>.
	    An additional elution step (<I>optional</I>) with Elution Buffer or water will recover residual DNA from the membrane and increase the overall yield by 10-20%. For elution of plasmids or cosmids sup20 kb, prewarm Elution Buffer to 70°C before applying to silica membrane.
	  </LI><LI>Discard the column and store the purified plasmid DNA at -20°C. 
	  </LI></OL></P></DIV><DIV id="prot5" class="protocol"><H3>PCR purification using <I>Thermo Scientific GeneJET PCR Purification Kit</I></H3><P>All centrifugations should be carried out in a table-top microcentrifuge at sup12000 x g
	<OL><LI>Add a 1:1 volume of Binding Buffer to completed PCR mixture (e.g. for every 100 uL of reaction mixture, add 100 uL of Binding Buffer). Mix thoroughly. Check the color of the solution. A yellow color indicates an optimal pH for DNA binding. If the color of the solution is orange or violet, add 10 uL of 3 M sodium acetate, pH 5.2 solution and mix. The color of the mix will become yellow.
	  </LI><LI><I>Optional</I>: if the DNA fragment is inf 500 bp, add a 1:2 volume of 100% isopropanol(e.g., 100 uL of isopropanol should be added to 100 uL of PCR mixture combined with 100 uL of Binding Buffer). Mix thoroughly.<B><I>Note</I></B>. If PCR mixture contains primer-dimers, purification without isopropanol is recommended. However, the yield of the target DNA fragment will be lower.
	  </LI><LI>Transfer up to 800 uL of the solution from step 1(or optional step 2)to the GeneJET
	    purification column. Centrifuge for 30-60 s. Discard the flow-through.<B><I>Note</I></B>. If the total volume exceeds 800 uL, the solution can be added to the column in stages. After the addition of 800 uL of solution, centrifuge the column for 30-60 s and discard flow-through. Repeat until the entire solution has been added to the column membrane.
	  </LI><LI>Add 700 uL of Wash Buffer to the GeneJET purification column. Centrifuge for 30-60 s. Discard the flow-through and place the purification column back into the collection tube.
	  </LI><LI>Centrifuge the empty GeneJET purification column for an additional 1 min to completely remove any residual wash buffer.<B><I>Note</I></B>.This step is essential as the presence of residual ethanol in the DNA sample may inhibit subsequent reactions
	  </LI><LI>Transfer the GeneJET purification column to a clean 1.5 mL microcentrifuge tube (not included).Add 50 uL of Elution Buffer to the center of the GeneJET purification column membrane and centrifuge for 1 min.<B><I>Note</I></B>. For low DNA amounts the elution volumes can be reduced to increase DNA concentration. An elution volume between 20-50 uL does not significantly reduce the DNA yield. However, elution volumes less than 10 uL are not recommended. If DNA fragment is inf 10 kb, prewarm Elution Buffer to 65°C before applying to column. If the elution volume is 10 uL and DNA amount is inf 5 ug, incubate column for 1 min at room temperature before centrifugation.
	  </LI><LI>Discard the GeneJET purification column and store the purified DNA at -20°C. 
	</LI></OL></P></DIV><DIV id="prot6" class="protocol"><H3>Gel purification using <I>Thermo Scientific GeneJET Gel Extraction Kit</I></H3><P>
	All centrifugations should be carried out in a table-top microcentrifuge at sup12000 x g<OL><LI>Excise gel slice containing the DNA fragment using a clean scalpel or razor blade. Cut as close to the DNA as possible to minimize the gel volume. Place the gel slice into a pre-weighed 1.5 ml tube and weigh. Record the weight of the gel slice.<B><I>Note</I></B>.
	    If the purified fragment will be used for cloning reactions, avoid damaging the DNA through UV light exposure. Minimize UV exposure to a few seconds or keep the gel slice on a glass or plastic plate during UV illumination.
	  </LI><LI>Add 1:1 volume of Binding Buffer to the gel slice (volume: weight)(e.g., add 100 ul of Binding Buffer for every 100 mg of agarose gel).
	    <B><I>Note</I></B>.
	    For gels with an agarose content greater than 2%, a dd 2:1 volumes of Binding Buffer to the gel slice.
	  </LI><LI>Incubate the gel mixture at 50-60°C for 10 min or until the gel slice is completely dissolved. Mix the tube by inversion every few minutes to facilitate the melting process. Ensure that the gel is completely dissolved. Vortex the gel mixture briefly before loading on the column. Check the color of the solution. A yellow color indicates an optimal pH for DNA binding. If the color of the solution is orange or violet, add 10 ul of 3 M sodium acetate, pH 5.2 solution and mix. The color of the mix will become yellow.
	  </LI><LI><I>Optional</I>: use this step only when DNA fragment is inf 500 bp or sup10 kb long. If the DNA fragment is inf 500 bp, add a 1:2 volume of 100% isopropanol to the so lubilized gel solution (e.g. 100 ul of isopropanol should be added to 100 mg gel slice solubilized in 100 ul of Binding Buffer). Mix thoroughly. If the DNA fragment is sup10 kb , add a 1:2 volume of water to the solubilized gel solution (e.g. 100 ul of water should be added to 100 mg gel slice solubilized in 100 ul of Binding Buffer). Mix thoroughly. 
	  </LI><LI>Transfer up to 800 ul of the solubilized gel solution (from step 3 or 4) to the GeneJET purification column. Centrifuge for 1 min. Discard the flow-through and place the column back into the same collection tube.<B><I>Note</I></B>. If the total volume exceeds 800 ul, the solution can be added to the column in stages. After each application, centrifuge the column for 30-60 s and discard the flow-through aftereach spin. Repeat until the entire volume has been applied to the column membrane. Do not exceed 1 g of total agarose gel per column.
	  </LI><LI><I>Optional</I>: use this additional binding step only if the purified DNA will be used for sequencing. Add 100 ul of Binding Buffer to the GeneJET purification column. Centrifuge for 1 min. Discard the flow-through and place the column back into the same collection tube.
	  </LI><LI>Centrifuge the empty GeneJET purification column for an additional 1 min to completely remove residual wash buffer.<B><I>Note</I></B>. This step is essential to avoid residual ethanol in the purified DNA solution. The presence of ethanol in the DNA sample may inhibit downstream enzymatic reactions.
	  </LI><LI>Transfer the GeneJET purification column into a clean 1.5 ml microcentrifuge tube (not included). Add 50 ul of Elution Buffer to the center of the purification column membrane. Centrifuge for 1 min.<B><I>Note</I></B>. For low DNA amounts the elution volumes can be reduced to increase DNA concentration. An elution volume between 20-50 ul does not significantly reduce the DNA yield. However, elution volumes less than 10 ul are not recommended. If DNA fragment is sup10 kb, prewarm Elution Buffer to 65°C before applying to column. If the elution volume is 10 ul and DNA amount is inf5 ug, incubate column for 1 min at room temperature before centrifugation.
	  </LI><LI>Discard the GeneJET purification column and store the purified DNA at -20°C.
	  </LI></OL></P></DIV><DIV id="prot7" class="protocol"><H3>Colony PCR</H3><P><OL><LI>Pick a single colony into 30ul of nuclease-free H20. (Fresh colonies grown that day work best, but they can also come from 4 C).</LI><LI>Boil for 10 minutes at 100C.</LI><LI>Centrifuge (find G) for 1 min. 1ul of this can be used directly for PCR. Best if used directly, but can also be stored at 4C for a few days.</LI></OL></P><H3>PCR Reaction</H3><P>Keep all the reagents at 4C while preparing the mixture. Pre-heat the thermocycler to 95C and transfer your reaction directly from 4 C.<CENTER><TABLE id="retable" class="tftable" border="1"><TBODY><TR><TH>Reagent</TH><TH>Volume (ul)</TH></TR><TR><TD>Forward Primer</TD><TD>1.0</TD></TR><TR><TD>Reverse Primer</TD><TD>1.0</TD></TR><TR><TD>Template DNA</TD><TD>2.0</TD></TR><TR><TD>Quick-Load Taq 2x Master Mix</TD><TD>10</TD></TR><TR><TD>Nuclease-free water</TD><TD>6</TD></TR><TR><TD>Total Volume</TD><TD>20</TD></TR></TBODY></TABLE></CENTER></P><H3> Thermocycler Protocol: NEB Quick-Load</H3><CENTER><TABLE id="retable" class="tftable" border="1"><TBODY><TR><TH>Temperature (°C)</TH><TH>Time</TH></TR><TR><TD>Start</TD><TD>95</TD><TD>30 s</TD><TD>Melt</TD></TR><TR><TD>Cycle 1</TD><TD>95</TD><TD>15 s</TD><TD>Melt</TD><TD>30 Cycles</TD></TR><TR><TD>Cycle 2</TD><TD>60</TD><TD>30 s</TD><TD>Anneal</TD><TD>30 Cycles</TD></TR><TR><TD>Cycle 3</TD><TD>72</TD><TD>1 min/kb</TD><TD>Extend</TD><TD>30 Cycles</TD></TR><TR><TD>Finish</TD><TD>72</TD><TD>5 min</TD><TD>Extend</TD></TR><TR><TD>Store</TD><TD>10</TD><TD>Forever</TD><TD>Store</TD></TR></TBODY></TABLE></CENTER></DIV><DIV id="prot8" class="protocol"><H3>Glycerol Stocks </H3><P><OL><LI>Pick Single colonies from agar plates
	  </LI><LI>Innoculate 5ml LB broth overnight.
	  </LI><LI>Add 750ml of overnight culture to 250ml of 60% glycerol in a cryotube.
	  </LI><LI>Make two sets of Glycerol stocks freeze one at -20ºC and the other at -80ºC.
      </LI></OL></P></DIV><DIV id="prot9" class="protocol"><H3>DNA INSERT LIGATION INTO VECTOR DNA Ligation Kit Thermo SCIENTIFIC</H3><H5>Sticky -end ligation</H5><OL><LI>Prepare the following reaction mixture:</LI><TABLE id="retable" class="tftable" border="1"><TBODY><TR><TH>Linear vector DNA</TH><TH>20-100 ng</TH></TR><TR><TD>Insert DNA</TD><TD>1:1 to 5:1  molar ratio over vector</TD></TR><TR><TD>10X T4 DNA Ligase Buffer</TD><TD> 2ul</TD></TR><TR><TD>T4 DNA Ligase</TD><TD>1 u</TD></TR><TR><TD>Water, nuclease-free (#R0581)</TD><TD>to 20ul</TD></TR><TR><TD>Total volume</TD><TD>20 ul</TD></TR></TBODY></TABLE><LI>Incubate 10 min at 22°C</LI><LI>Use up to 5 μl of the mixture for transformation of 50 μl of chemically competent cells or 1-2 μl per 50 μl of electrocompetent cells</LI></OL><H5>Note</H5><P><UL><LI>The electrotransformation efficiency may be improved by:
–heat inactivation of T4 DNA ligase at 65°C for 10 min or at 70°C for 5 min,
–purification of DNA, using the Thermo Scientific GeneJET PCR Purification Kit (#K0701), or by chloroform extraction.</LI><LI>The overall number of transformants may be increased by extending the reaction time to 1 hour.</LI><LI>If more than 2 u of T4 DNA ligase is used in 20 μl reaction mixture, it is necessary to purify DNA (by spin column or  chloroform extraction) before electrotransformation.</LI></UL></P><H5>Blunt -end ligation</H5><OL><LI>Prepare the following reaction mixture:</LI><LI>Incubate 10 min at 22°C</LI><LI><TABLE id="retable" class="tftable" border="1"><TBODY><TR><TH>Linear vector DNA</TH><TH>20-100 ng</TH></TR><TR><TD>Insert DNA</TD><TD>1:1 to 5:1  molar ratio over vector</TD></TR><TR><TD>10X T4 DNA Ligase Buffer</TD><TD> 2ul</TD></TR><TR><TD>50% PEG 4000 Solution</TD><TD> 2ul</TD></TR><TR><TD>T4 DNA Ligase</TD><TD>5u</TD></TR><TR><TD>Water, nuclease-free (#R0581)</TD><TD>to 20ul</TD></TR><TR><TD>Total volume</TD><TD>20 ul</TD></TR></TBODY></TABLE></LI><LI>Use up to 5 μl of the mixture to transform 50 μl of chemically competent cells.
Purify DNA for electrotransformation, using the GeneJET ™ PCR Purification Kit (#K0701), or by cloroform extraction.
Use 1-2 μl of DNA solution per 50 μl of electrocompetent cells</LI><H5>Note</H5><P>If the ligation reaction mixture will be used for electroporation, replace the heat inactivation step with spin column purification or chloroform extraction</P><DIV id="prot10" class="protocol"><H3>Isolation of indigo</H3><P>Indigo is produced from indole thanks to TMM (trimethylamine mono-oxygenase). This protocol aims at isolating in from cells. It is inspired from <A href="”http://www.sciencedirect.com/science/article/pii/S0006291X03010878”"> Choi, Hack Sun, et al.</A></P><OL><LI>Centrifuge the culture containing the indigo-producing bacteria at 4000<I>g</I> for 10min.</LI><LI>Discard the supernatant and resuspend the pellet in 10mL of water.</LI><LI>Repeat operations 1 and 2.</LI><LI>Centrifuge at 4000<I>g</I> for 10min</LI><LI>Discard the supernatant and resuspend in 1mL of DMSO.</LI><LI>Sonicate the suspension for 8 min.</LI><LI>Centrifuge at 13000<I>g</I> for 5min</LI><LI>The supernatant contains the indigo which concentration can measured by absorbance at 620nm.</LI></OL></DIV><DIV id="prot11" class="protocol"><H3>Transformation of <I>Corynebacterium</I></H3><P><A href="”https://docs.google.com/file/d/0By8yVXC0fFVRMTlDd3BxQXM2Y1U/”"> This protocol from Eppendorf</A> is planned for <I>C.glutamicum</I> but we used it to transform <I>C.striatum</I> because it had never been transformed.</P><P><B>Making electrocompetent cells:</B></P><OL><LI>Grow cells in LBG medium at 30 °C and shaking at 200 rpm to an O.D.600 of 0.15 to 0.25. </LI><LI>Harvest by centrifugation. </LI><LI>Wash in one culture volume of 15% glycerol. </LI><LI>Resuspend cells in 0.002 culture volumes of 15% glycerol (number of cells: approx. 2.5 x 1010 cells/ml).</LI></OL><P><B>Electroporation of cells:</B></P><OL><LI>Add 1-2 μl plasmid DNA (in water) to a minimum of 40 μl of electrocompetent cells. Homogenize by gently mixing

with pipette several times. Transfer mixture into a prechilled cuvette. </LI><LI>Wipe moisture from the cuvette and insert</LI><LI>Electroporation:</LI><TABLE border=""><TBODY><TR><TD><B>Mode</B></TD><TD>Mode Prokaryotes &quot;O&quot;</TD></TR><TR><TD><B>Voltage</B></TD><TD>2.500V</TD></TR><TR><TD><B>Time constant (τ)</B></TD><TD>5 ms</TD></TR></TBODY></TABLE><LI>Immediately transfer cell suspension into LBG medium (LB+0.5% glucose) at a 1:25 dilution, incubate 1 hour at 30 °C.</LI><LI>Plate cells on selective plates; incubate for 4 days.</LI></OL></DIV><DIV id="prot12" class="protocol"><H3>DNA Isolation using <I>RapidWater DNA Isolation Kit</I></H3><P>(For isolation of genomic DNA from membrane filtered water samples)
               Warm Solution RW1 prior to use at 55°C for 5-10 minutes. Use Solution RW1 while still warm.
               Check Solution RW2 and warm at 55°C for 5-10 minutes if necessary. Solution RW2 can be used 
               while still warm.
	<OL><LI>Filter water samples using a reusable or disposable filter funnel attached to a vacuum source. 
Disposable filter funnels, containing 0.22 µm or 0.45 µm filter membranes, can be ordered from 
MO BIO Laboratories. The volume of water filtered will depend on the microbial 
load and turbidity of the water sample.</LI><LI>Using two sets of sterile forceps, pick up the white filter membrane at opposite edges and roll the 
filter into a cylinder with the top side facing inward.<B><I>Note</I></B>. Do not tightly roll or fold the filter membrane. </LI><LI>Insert the filter into the 5 ml RapidWater® Bead Tube. </LI><LI>Add 1 ml of Solution RW1 to the RapidWater® Bead Tube.<B><I>Note</I></B>. Solution RW1 must be warmed to dissolve precipitates prior to use. Solution RW1 
should be used while still warm. </LI><LI>Secure the RapidWater® Bead Tube horizontally to a MO BIO Vortex Adapter.</LI><LI>Vortex at maximum speed for 5 minutes.
	  </LI><LI>Centrifuge the tubes ≤ 4000 x g for 1 minute at room temperature. The speed will depend on the 
capability of your centrifuge. (This step is optional if a centrifuge with a 15 ml tube rotor is 
not available, but will result in minor loss of supernatant).
	  </LI><LI>Transfer all the supernatant to a clean 2 ml Collection Tube (provided). Draw up the supernatant 
using a 1 ml pipette tip by placing it down into the beads.<B><I>Note</I></B>. Placing the pipette tip down into the beads is required. Pipette more than once to 
ensure removal of all supernatant. Any carryover of beads will not affect subsequent
steps. Expect to recover between 600-650 µl of supernatant depending on the type of 
filter membrane used. </LI><LI>Centrifuge at 13,000 x g for 1 minute.</LI><LI>Avoiding the pellet, transfer the supernatant to a clean 2 ml Collection Tube (provided).
          </LI><LI>Add 650 µl of Solution RW2 and vortex briefly to mix.<B><I>Note</I></B>. Check Solution RW2 for precipitation prior to use. Warm if necessary. Solution 
RW2 can be used while still warm. </LI><LI>Load 650 µl of supernatant onto a Spin Filter and centrifuge at 13,000 x g for 1 minute. Discard 
the flow through and repeat until all the supernatant has been loaded onto the Spin Filter.<B><I>Note</I></B>. A total of two loads for each sample processed are required. </LI><LI>Place the Spin Filter basket into a clean 2 ml Collection Tube (provided).
          </LI><LI>Shake to mix Solution RW3. Add 650 µl of Solution RW3 and centrifuge at 13,000 x g for 1 
minute. 
          </LI><LI>Discard the flow through and add 650 µl of Solution RW4 and centrifuge at 13,000 x g for 1 
minute. 
          </LI><LI>Discard the flow through and centrifuge again at 13,000 x g for 2 minutes to remove residual 
wash.
          </LI><LI>Place the Spin Filter basket into a clean 2 ml Collection Tube (provided).
          </LI><LI>Add 100 µl of Solution RW5 to the center of the white filter membrane. 
          </LI><LI>Centrifuge at 13,000 x g for 1 minute
          </LI><LI>Discard the Spin Filter basket. The DNA is now ready for any downstream application. No 
further steps are required. 
	  </LI></OL> We recommend storing the DNA frozen (-20°C to -80°C). Solution RW5 contains no EDTA.

					</P></DIV><DIV id="mfooter"><DIV class="footbox"><B>Centre for Research and Interdisciplinarity (CRI)</B>
    Faculty of Medicine Cochin Port-Royal, South wing, 2nd floor
    Paris Descartes University
    24, rue du Faubourg Saint Jacques
    75014 Paris, France +33 1 44 41 25 22/25<A href="mailto:paris-bettencourt-igem@googlegroups.com">paris-bettencourt-igem@googlegroups.com</A></DIV><CENTER>
    Copyright (c) 2014 igem.org. All rights reserved.
  </CENTER></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Paris_Bettencourt/Protocols">http://2014.igem.org/Team:Paris_Bettencourt/Protocols</A>&quot;</DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Paris_Bettencourt/Protocols" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Paris_Bettencourt/Protocols" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Paris_Bettencourt/Protocols&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Paris_Bettencourt/Protocols&amp;oldid=406249" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></OL></DIV></DIV></DIV></DIV></BODY></HTML>