"
Page
Discussion
View source
History
teams
Log in
 
Team:Paris Bettencourt/Notebook/Odor Library
From 2014.igem.org
Project
SmellThe Roses
SomethingFishy
GoodyTwo Shoes
Don'tSweat It
Teen Spirit
Design Practice
Bioinformatics
Achievements
Results
Interlab Study
MedalFulfilment
Parts
Policy
Safety
Consent Form
Patenting
Practices
MOOC
CitizenScience
The Senseof Smell
The HumanMicriobiome
Minder
Collaborations
Collaborations
The SmellGame
Newsletters
Notebook
Daily Note
Protocols
Bibliography
Team
Our Team
Attributions
Official TeamProfile
Acknowledgements
@iGEM_Paris
Smell The Roses
Something Fishy
Don't Sweat It
Goody Two Shoes
Teen Spirit
Interlab Study
June
July
August
September
October
Notebook
June
June 26th
Goal
Design plasmid for limonene synthase
Procedure
Used MG1665 strain, the RBS with the highest initial rate and the sequence of r-limonene synthase from genBank. Added polyhistidine tail to the construct. Submitted the construct to Jake.
Results
Designed plasmid for limonene synthase using the software geneious. Found a biobricks part containing this sequence. For the purpose of saving budget, we would first transform this standard part into cell and later modify it.
June 27th
Goal
Transform the bioBricks limonene synthase(BBa_I742111) part into e.coli. We found that the bioBricks kit for 2014 contain the limonene synthase sequence in plate 4 position 3I.
Procedure
Start thawing the competent cells(Used Christina's competent cells) on ice.
Add 50 µL of thawed competent cells into pre-chilled 2ml tube, and another 50µL into a 2ml tube, labelled for your control.
Add 2 µL of the resuspended DNA to the 2ml tube. Pipet up and down a few times, gently. Make sure to keep the competent cells on ice.
Close tubes and incubate the cells on ice for 30 minutes.
Heat shock the cells by immersion in a pre-heated water bath at 42ºC for 60 seconds.
Incubate the cells on ice for 5 minutes.
Incubate the cells at 37ºC for 2 hours while the tubes are rotating or shaking. Important: 2 hour recovery time helps in transformation efficiency, especially for plasmid backbones with antibiotic resistance other than ampicillin.
Label two petri dishes with LB agar and the appropriate antibiotic(s) with the part number, plasmid backbone, and antibiotic resistance. Plate 20 µl and 200 µl of the transformation onto the dishes, and spread. This helps ensure that you will be able to pick out a single colony.
For the control, label two petri dishes with LB agar (CM). Plate 20 µl and 200 µl of the transformation onto the dishes, and spread.
You can pick a single colony, make a glycerol stock, grow up a cell culture and miniprep.
Results
Transformed the plasmid containing limonene synthase sequence from biobricks into E.Coli. Colonies seen on both plates. Glycerol Stock made: PB. 018 and PB. 019
June 28th
Goal
Made chemical competent cell following standard protocol with MG1665 strains.
Procedure
The night before, inoculate a 5 ml culture and grow overnight with selection.
The day of the experiment dilute cells ~ 1:200 into selective media. For this example add 250 ul to 50 ml of selective media. Note: The protocol is easily scaled to increase the number of cells.
Grow the cells to an OD600 of 0.6 – 0.7.  Use a large flask, 500ml, for good aeration.  Use a baffled flask for fastest growth.  This takes about 3 hours depending on the cells.  Medium-heavy cloudiness by eye is fine.
Spin down the cells at 4 ºC, 4000 rpm, 15 minutes. Note: Keep the cells at 4 ºC from now on.
Resuspend cells in 15 ml, ice-cold 100 mM CaCl2. Leave on ice 4 hours to overnight.
Spin down the cells at 4 ºC, 4000 rpm, 15 minutes.
Resuspend cells in 4 ml, ice-cold 100 mM CaCl2 + 15% glycerol.
Aliquot into pre-chilled Eppendorf tubes. Use immediately or store at -80ºC.
Note: Frozen cells are only good once.Do not refreeze cells once thawed.
Results
The transformed cell were grown for 20 hours and the transformation was successful. The plates were put into the 4 degree fridge for making stocks on Monday.
July
July 1st
Goal
To design the primers and gBlock to modify the BioBricks part BBa_I742111.
Procedure
decide the major steps of modification
to PCR the biobricks construct using primers that have Agel restriction site sequences.
to digest the PCR product
to PCR the gBlock with Promoter, RBS and restriction sites (Xbal and Agel)
to digest the PCR product
to ligate the two sequences together
2. Design the primers for PCR and the gBlock including promoter, RBS and restriction sites using the software Geneious.
Results
fwd primer for BioBricks part PCR: 0PB.015
rev primer for BioBricks: 0PB.016
gBlock with promoter, RBS and restriction sites: 0PB. 019
fwd primer for gBlock PCR: 0PB.30
rev primer for gBlock PCR: 0PB.31
(We ended up ordering the gBlock as synthetic gene sequence, because it is too long for oligos and too short for gBlocks. Oh well.)
July 7th
Goal
to extract plasmid of biobricks I742111 from the transformed cells.
Procedure
Growth the cell for two days
Centrifuge the tube for 15 minutes with 4000 rps
Follow the standard protocol of Thermo Scientific miniprep kit
Label the centrifuge tube and store it in the -20 freezer. (Syl miniprep)
Results
We grew two tubes of glycerol stocks. However living cells were only seen in one of them, which was used for miniprep.
Plasmid stored in the box after the procedure. PCR would be done tomorrow.
July 8th
Goal
Amplify limonene synthase gene sequence for E.coli
Fw primer: oPB.015
Rv primer: oPB.016
Procedure
Reagent
Volume
1x
Nuclease-free water
71 ul
5x Phusion HF Buffer
20 ul
10 mM dNTPs
2 ul
Forward Primer (10 uM)
1 ul
Reverse Primer (10 uM)
1 ul
Template Plasmid
1 ul
Phusion DNA Polymerase
1 ul
DMSO
3 ul
Total Volume
100 ul
Temperature
Time
Function
start
98 C
30 sec
melt
cycle 1
98 C
10 sec
melt
cycle 2
51 C
30 sec
anneal
cycle 3
72 C
30 sec
extend
finish
72 C
5 min
extend
blind
10 C
forever
blind
Results
PCR product at the expected length.
July 9th
Goal
To run the Gel of the PCR product to confirm that the segment is at the right length.
to digest the segment using AgeI and XbaI. Purify the digested product.
Procedure
Take the PCR tubes out of the machine.
Mixing the dye with PCR products, from each tube.
Run the gel for an hour and half using 100bp plus DNA ruler
Observe under UV light
Digest in the 37 degree incubator for 10 minutes using AgeI and XbaI.
Reagent
Volume
Nuclease-free water
27.5 ul
10x FD Buffer
5 ul
DNA
12.5 ul
AgeI
2.5 ul
Xbal
2.5 ul
Total Volume
50 ul
Purify following the protocol of digestion purification kit. Used warm NF water for diluting DNA at the last step.
Results
The PCR product is at expected length. Purified product has a very low concentration(2.5ng/ul), meaning the PCR product was a mixture of multiple segments and didn't give ideal result. Therefore proceed to PCR again.
July 16th
Goal
Amplify limonene synthase gene sequence for E.coli
Fw primer: oPB.015
Rv primer: oPB.016
Procedure
Reagent
Volume
1x
Nuclease-free water
71 ul
5x Phusion HF Buffer
20 ul
10 mM dNTPs
2 ul
Forward Primer (10 uM)
1 ul
Reverse Primer (10 uM)
1 ul
Template Plasmid
1 ul
Phusion DNA Polymerase
1 ul
DMSO
3 ul
Total Volume
100 ul
Temperature
Time
Function
start
98 C
30 sec
melt
cycle 1
98 C
10 sec
melt
cycle 2
51 C
30 sec
anneal
cycle 3
72 C
2.5 min
extend
finish
72 C
5 min
extend
blind
10 C
forever
blind
Results
It worked!
Goal
: transform the PB19 synthase part into e.coli.
Procedure
Start thawing the competent cells(Used Christina's competent cells) on ice.
Add 50 µL of thawed competent cells into pre-chilled 2ml tube, and another 50µL into a 2ml tube, labelled for your control.
Add 2 µL of the resuspended DNA to the 2ml tube. Pipet up and down a few times, gently. Make sure to keep the competent cells on ice.
Close tubes and incubate the cells on ice for 30 minutes.
Heat shock the cells by immersion in a pre-heated water bath at 42ºC for 60 seconds.
Incubate the cells on ice for 5 minutes.
Incubate the cells at 37ºC for 2 hours while the tubes are rotating or shaking. Important: 2 hour recovery time helps in transformation efficiency, especially for plasmid backbones with antibiotic resistance other than ampicillin.
Label two petri dishes with LB agar and the appropriate antibiotic(s) with the part number, plasmid backbone, and antibiotic resistance. Plate 20 µl and 200 µl of the transformation onto the dishes, and spread. This helps ensure that you will be able to pick out a single colony.
For the control, label two petri dishes with LB agar (CM). Plate 20 µl and 200 µl of the transformation onto the dishes, and spread.
You can pick a single colony, make a glycerol stock, grow up a cell culture and miniprep.
Results
Colonies seen on both plates.
July 17th
Goal
to extract plasmid of PB19 from the transformed cells.
Procedure
Growth the cell overnight
Centrifuge the tube for 15 minutes with 4000 rps
Follow the standard protocol of Thermo Scientific miniprep kit
Label the centrifuge tube and store it in the -20 freezer. (Syl mini prep)
Results
Tube 1: 40 ng/ul
Tube 2: 30ng/ul
Goal
to digest the segment using AgeI and XbaI. Purify the digested product.
Procedure
Digest in the 37 degree incubator for 10 minutes using AgeI and XbaI.
Reagent
Volume
Nuclease-free water
27.5 ul
10x FD Buffer
5 ul
DNA
12.5 ul
AgeI
2.5 ul
Xbal
2.5 ul
Total Volume
50 ul
Purify following the protocol of digestion purification kit. Used warm NF water for diluting DNA at the last step
Results
Concentration after purification: 12.2 ng/ul
July 23rd
Goal
: to achieve the great great goal of magnifying the RBS and promoter before digestion, purification and ligation.
Procedure
we did a nano drop of the mini prep plasmid, and it is 460ng/ul. The curve looks good. We diluted 1:500
Results
a fainted band was seen. However, the concentration after purification is below 5 ng/ul, too low to be used to ligation. Therefore, we decided to design the sequence as two oligos, with Xbal and Agel sticky ends.
August
August 4th
Goal
to dilute received oilgos, to Phosphorylate them, to Anneal them and to ligation is with Limonene vector
Procedure
Dilute the oilgo to 100 uM, the stock
Take 10 ul out of the 100 uM stock and dilute to make 10 uM working stock
Mix:
2 uL 10 µM sense oligo
2 uL 10 µM anti-sense oligo
2 uL 10 x PNK (polynucleotide kinase) buffer A
2 uL 10mM ATP
1 uL T4 polynucleotide kinase (PNK)
10 uL distilled water
to give 20 uL total volume
Incubate at 37C for 30 mins
Place in boiling water bath for 2 min, then remove water bath from the heat source and allow the reaction (still in the water bath) to cool to room temperature. (Labelled Annealing in the box)
ligation
Mix :
8 ul of vector (12 ng/ul)
1 ul of insert
0.5 ul of ligase enzyme
2 ul of ligase buffer
8.5 ul of distilled water
total 20 ul
put under 22 degree for 30 minutes
put under 16 degree for 30 minutes
put in 4 degree overnight
Results
colonies were seen after transforming and plating.
August 10th
Goal
to sequence the ligated plasmid for limonene construct
Procedure
Dilute the mini prep product 1: 2 (Give us concentration 70ng/ul and 80ng/ul)
design primers that are about 200 bp away upstream and downstream from the inserted sequence
Send them to GATC
Results
It appears that the old plasmid recirculate itself.
Further Steps
regrow 6 more colonies, minipreped and did analytical PCR with NotI and AgeI Colony 1a appears to have the complete plasmid.
August 20th
Goal
to PCR ilvBN gene from MG1655
Procedure
Picked 2 colonies from the plate MG1655 115+GFP
Inoculate them each in 50 ul of NF H2O
Boiled for 3 mins at 98 degree
Use the solution as DNA template
Follow standard PCR protocol from this step on
Reagent
Volume
1x
Nuclease-free water
63 ul
5x Phusion HF Buffer
20 ul
10 mM dNTPs
2 ul
Forward Primer (10 uM)
5 ul
Reverse Primer (10 uM)
5 ul
Template Plasmid
1 ul
Phusion DNA Polymerase
1 ul
DMSO
3 ul
Total Volume
100 ul
Temperature
Time
Function
start
98 C
30 sec
melt
cycle 1
98 C
10 sec
melt
cycle 2
51 C
30 sec
anneal
cycle 3
72 C
2 min
extend
finish
72 C
5 min
extend
blind
10 C
forever
blind
Results
Band seen on both sample (from 2 colonies)
August 21st
Goal
to Digest the PCR product (ilvBN) from yesterday and to digest pET-DUET1 as vector
Procedure
Vector (pET-DUET1)
Reagent
Volume
Nuclease-free water
38 ul
10x FD Buffer
5 ul
DNA
2 ul
EcoRI
2 ul
NcoI
2 ul
FastAP
1 ul
Total Volume
50 ul
ilvBN
Reagent
Volume
Nuclease-free water
39 ul
10x FD Buffer
5 ul
DNA
2 ul
EcoRI
2 ul
NcoI
2 ul
Total Volume
50 ul
Incubate at 37 degree for 2 hours. Purification using the PCR purification kit.
Results
24 ng/ul for C2 and 12 ng/ul for C1 (Will use C2 PCR purified product for digestion.)
August 25th
Goal
to verify the insert of RBS and promoter is in the limonene biobrick part.
Procedure
digest
Reagent
Volume
Nuclease-free water
19 ul
10x FD Buffer
3 ul
DNA
1 ul
AgeI
1 ul
Xbal
1 ul
Total Volume
25 ul
(Miniprep concentration: 500 for N.1, 400 for N.2 and N.3) Expected length: 3750 and 3740 or 3840 and 3740
Results
(with 100bp plus DNA ladder)
(the control sample didn’t get digested. Therefore, we can’t say for sure if the ligation worked. However, we decided to go ahead with sequencing and see the result.)
August 26th
Goal
To ligate ilvBN gene with PETDUET 1
Procedure
Reagent
Volume
Nuclease-free water
3 ul
Ligase Buffer
2 ul
ligase
1 ul
Insert(2132 bp, 10ng/ul)
4 ul
Vector(5381bp, 18ng.ul)
10 ul
Total Volume
20 ul
25 degree for 1 hour and then heat shock transformation.
Results
colonies seen. Picked 4 colonies for analytical digestion. Bands seen at expected length.
August 27th
Goal
to ligate the aldB gblock with pSB1C3
Procedure
Digested and Purified PCR product, concentration: 40 ng/ul Digested and purified vector pSB1C3: 20 ng/ul
Reagent
Volume
Nuclease-free water
10 ul
Ligase Buffer
2 ul
ligase
1 ul
Insert(2132 bp, 10ng/ul)
4 ul
Vector(5381bp, 18ng.ul)
4.5 ul
Total Volume
20 ul
20 degree for 5 hours, followed by heat shock transformation into NEB strain.
Results
Blurring band seen on gel after colony PCR, will process to mini prep and sequencing
5/06  Odourless coli
11/06 Odourless coli
tnaA deletion mutant from Keio Collection
Location: 63:E:9
Streaked on LBA+Kan
We have glycerol stock sPB.000
5/06 Banana and Wintergreen E. coli Transformation
Goal:
Transform NEB Turbo E. coli competent cells (à la Jake) using standard heat shock transformation protocol (à la Jake).
The generators do not include promoters, therefore there is no expression of the odour generating enzymes.
I will transform the following parts:
pPB.003 BBa_R0040 2013  3  5E  pSB1C3
pPB.004 BBa_J45199 2013  4  6I   pSB1C3
pPB.005 BBa_J45119 2012  2  5B  pSB1AT3
Procedure:
Resuspend DNA from registry plates using 10 uL of ddH2O. Make sure to resuspend GOOD.
Follow protocol below. Using 1 uL of DNA.
Protocol: Heat Shock Transformation of E. coli 1) Place 20 ul of cells in a pre-chilled Eppendorf tube.2) Add 1 ul of DNA to the chilled cells 3) Mix gently by flicking the tube.4) Chill on ice for 10 minutes.
5) Heat shock at 42 °C for 30 seconds.
6) Return to ice for 2 minutes.
7) Add 200 ul LB medium and recover the cells by shaking at 37 °C.Another rich medium can substitute for the recovery.
The recovery time varies with the antibiotic selection. Tetracycline: 60-120 minutesChloramphenicol: 60-120 minutes 8) Plate out 20uL of the cells on selective LB. And 200 uL on another plate. Use glass beads to spread the cells.
6/06 Transformations
Yesterday's plates do not have colonies.
ToDo:
Plate the 200 uL from yesterday's tube containing recovered cells.
pPB.003 BBa_R0040   LBA+Cm
pPB.004 BBa_J45199  LBA+Cm
pPB.005 BBa_J45119  LBA+Tet
Transform:
pPB.003 BBa_R0040: Constitutive TetR promoter
Resuspended DNA from 2013:3:5E
pSB1C3  Chloramphenicol resistance
pPB.004 BBa_J45199: Banana generator without promoter
Resuspended DNA from 2013:4:6I
pSB1C3 Chloramphenicol resistance
Requires promoter and 5mM isoamyl alcohol
pPB.005 BBa_J45119: Wintergreen generator without promoter
Resuspended DNA from 2012:2:5B
pSB1AT3 Ampicillin and Tetracycline resistance
Requires promoter and 2 mM Salicylic acid
USED, NOT AVAILABLE BBa_J45200: Constitutive Banana generator
2012:2:5F
pSB1AT3 Ampicillin and Tetracycline resistance
Requires 5mM isoamyl alcohol
USED, NOT AVAILABLE BBa_J45120: Constitutive Wintergreen generator
2012:2:5D
pSB1AT3 Ampicillin and Tetracycline resistance
Requires 2 mM Salicylic acid
Protocol: Heat Shock Transformation of E. coli
Plate fresh transformations:
pPB.003 BBa_R0040   LBA+Cm
BBa_J45119  LBA+Tet
BBa_J45199  LBA+Cm
Streak NEB Turbo in LBA for singles (tomorrow set an overnight for competent cell production).
7/06 Single colonies from Transformations
Took single colonies, started overnight cultures.
8/06 Single colonies from Transformations
Left the overnight cultures… will need to dilute for mini prepping.
9/06 Single colonies from Transformations
No growth. Picked single colonies and started overnights again.
10/06 Minipreps
Minipreps:
pPB.003: BBa_R0040
pPB.004: BBa_J45199
pPB.005 BBa_J45119
All samples are at high concentrations ~100 ng/uL
26/06 Digestion set up
Digestion:
FD Restriction Digestion
Reagent
Volume (uL)
NF ddH2O
17
Buffer
3
FD Enzyme I
1
FD Enzyme II
1
DNA
3
Total Volume
25
Cut pPB.003 with SpeI & PstI
Expected fragments:
2.1 kb
18  bp
Cut pPB.004 with XbaI and PstI
Expected fragments:
~ 2 kb
1.765 kb
Cut pPB.005 with XbaI and PstI
Expected fragments:
3.4 kb
1.26 kb
I performed a purification with the PCR cleanup kit and stored the samples at -20ºC.
27/06 SAGE  set up
Sage:
SAGE on thick gel (3 mm) 1% agarose should be fine.
There should be two bands for pPB.004 and pPB.005.
Gel extraction:
Cut out the second band of pPB.004 and pPB.005 (1.8 kb and 1.3 kb respectively).
Weigh the agarose fragments (100 mg gel : 300 uL QG Buffer)
Max amount is 400 mg
Solubilize agarose completely by incubating at 50ºC for 10 min in water bath. Vortexing the tube every 3 minutes helps.
Color of mixture should be yellow. Otherwise add 10 uL of 3 M sodium acetate pH 5.
Add the equivalent of 1 gel volume of isopronanol and mix the tube (although for fragments between 500 bp and 4 kb this should matter).
Place column in a 2 mL collection tube
Load sample into column. Centrifuge 1 min @ 13 000 rpm.
Discard flowthrough and repeat adding 0.5 mL of QG buffer.
Discard flowthorugh and wash with 0.75 mL of PE Buffer. Let the column sit for 5 min  prior centrifugation.
Discard flowthrough and centrifuge again to ensure all residual ethanol is gone.
Place the column into a clean 1.5 mL Epp and elute by adding 50 uL of NF-ddH2O. Make sure to wait 2 min before centrifugation.
Store at 4ºC overnight or -20ºC for longer periods.
27/06 Ligation set up
July
7/07 We start again...
Digestion:
FD Restriction Digestion
Reagent
Volume (uL)
NF ddH2O
17
Buffer
3
FD Enzyme I
1
FD Enzyme II
1
DNA
3
Total Volume
25
Cut pPB.003 with SpeI & PstI
Expected fragments:
2.1 kb
18  bp
Cut pPB.004 with XbaI and PstI
Expected fragments:
~ 2 kb
1.765 kb
Cut pPB.005 with XbaI and PstI
Expected fragments:
3.4 kb
1.26 kb
SAGE
SAGE on thick gel (3 mm) 1% agarose should be fine.
There should be two bands for pPB.004 and pPB.005.
WEIRD GEL: Very dim bands after staining for 7 minutes in EtBr. The gel was ran at 100 mV but it took more than 60 minutes for the indicator marker dye to reach half migration. FAILED.
14/07 Digestion-product gel extraction and ligation
Use the remaining purified digestion products. Run a gel and purify the appropriate bands.
Cut pPB.003 with SpeI & PstI
Expected fragments:
2.1 kb
18  bp
Cut pPB.004 with XbaI and PstI
Expected fragments:
~ 2 kb
1.765 kb
Cut pPB.005 with XbaI and PstI
Expected fragments:
3.4 kb
1.26 kb
Gel was good. The gel extraction not. Things got messy with the blade and the transilluminator, I chopped off half of the wrong band.
17/07/2014 Restriction Digestion set up
Digestion:
FD Restriction Digestion
Reagent
Volume (uL)
NF ddH2O
75
Buffer
10
FD Enzyme I
2.5
FD Enzyme II
2.5
DNA
10
Total Volume
100
Incubate at 37ºC with moderate shaking for ~1 hour.
Cut pPB.003 with SpeI & PstI
Expected fragments:
2.1 kb
18  bp
Cut pPB.004 with XbaI and PstI
Expected fragments:
~ 2 kb
1.765 kb
Cut pPB.005 with XbaI and PstI
Expected fragments:
3.4 kb
1.26 kb
Cut pPB.014 with SpeI and PstI
(For enhanced Banana pPB.016 and Wintergreen pPB.015)
Expected fragments:
~2.1 kb
~18 kb
21/07/2014  Ligation  set up
Ligation:
Fermentas T4 DNA Ligation
Reagent
Vector (pPB.003)
25 ng
Insert
75 ng
T4 DNA Ligase Buffer 10X
1 uL
T4 DNA Ligase
0.5 uL
NF-ddH2O
as needed
Total
10 uL
Fermentas T4 DNA Ligation
Reagent
Vector
2 uL
Insert
2 uL
NF ddH2O
4.5 uL
T4 DNA Ligase Buffer
1 uL
T4 DNA Ligase
0.5 uL
Total
10 uL
Incubate @22ºC for 10 minutes.
Transform 20 uL of NEB Turbo competent cells with 5 uL of ligation product.
No colonies. Failed experiment.
pPB.016 Banana Generator under Anderson Promoter
pPB.016: Banana generator AndersonPromoter (BBa_J23108)
Requires 5mM isoamyl alcohol
7/07 Transformations
I will transform the following parts:
pPB.014 BBa_J23108 2014  4  4C  pSB1C3
Resuspend DNA from registry plates using 10 uL of ddH2O. Make sure to resuspend GOOD.
Follow protocol below. Using 1 uL of DNA.
Protocol: Heat Shock Transformation of E. coli 1) Place 20 ul of cells in a pre-chilled Eppendorf tube.
2) Add 1 ul of DNA to the chilled cells
3) Mix gently by flicking the tube.
4) Chill on ice for 10 minutes. 5) Heat shock at 42 °C for 30 seconds.
6) Return to ice for 2 minutes.
7) Add 200 ul LB medium and recover the cells by shaking at 37 °C. Another rich medium can substitute for the recovery.
The recovery time varies with the antibiotic selection.
Tetracycline: 60-120 minutes
Chloramphenicol: 60-120 minutes
8) Plate out 20uL of the cells on selective LB. And 200 uL on another plate.
Use glass beads to spread the cells
9) Count colonies after incubation.
Transform:
Constitutive Promoter
BBa_J23108: Constitutive Promoter from Anderson's Library
Resuspended DNA from 2014:4:4C
pSB1C3  Chloramphenicol resistance
BBa_J45199: Banana generator without promoter
Resuspended DNA from 2013:4:6I
pSB1C3 Chloramphenicol resistance
Requires promoter and 5mM isoamyl alcohol
Protocol: Heat Shock Transformation of E. coli
1) Place 20 ul of cells in a pre-chilled Eppendorf tube.
2) Add 2 ul of DNA to the chilled cells
3) Mix gently by flicking the tube.
4) Chill on ice for 10 minutes.
5) Heat shock at 42 °C for 30 seconds.
6) Return to ice for 2 minutes.
7) Add 200 ul LB medium and recover the cells by shaking at 37 °C. Another rich medium can substitute for the recovery.
The recovery time varies with the antibiotic selection.
Tetracycline: 60-120 minutes
Chloramphenicol: 60-120 minutes
8) Plate out 20uL of the cells on selective LB. And 200 uL on another plate.
Use glass beads to spread the cells
9) Count colonies after incubation.
Plate fresh transformations:
BBa_R0040   LBA+Cm
BBa_J45119  LBA+Tet
BBa_J45199  LBA+Cm
Streak NEB Turbo in LBA for singles (tomorrow set an overnight for competent cell production).
Send email asking for salicylic acid and isoamyl alcohol.
8/07 Single colonies from Transformations
Done
8/07 Minipreps
Done
8/07 Digestion from miniprep set up
Digestion:
FD Restriction Digestion
Reagent
Volume (uL)
NF ddH2O
17
Buffer
3
FD Enzyme I
1
FD Enzyme II
1
DNA
3
Total Volume
25
Cut pPB.014 with SpeI & PstI
Expected fragments:
~2 kb
~18  bp
Cut pPB.004 with XbaI and PstI
Expected fragments:
~ 2 kb
1.765 kb
8/07 SAGE  set up
Sage:
SAGE on thick gel (3 mm) 1% agarose should be fine.
There should be two bands for pPB.004.
Gel extraction:
Cut out the second band of pPB.004 and pPB.005 (1.8 kb and 1.3 kb respectively).
Weigh the agarose fragments (100 mg gel : 300 uL QG Buffer)
Max amount is 400 mg
Solubilize agarose completely by incubating at 50ºC for 10 min in water bath. Vortexing the tube every 3 minutes helps.
Color of mixture should be yellow. Otherwise add 10 uL of 3 M sodium acetate pH 5.
Add the equivalent of 1 gel volume of isopronanol and mix the tube (although for fragments between 500 bp and 4 kb this should matter).
Place column in a 2 mL collection tube
Load sample into column. Centrifuge 1 min @ 13 000 rpm.
Discard flowthrough and repeat adding 0.5 mL of QG buffer.
Discard flowthorugh and wash with 0.75 mL of PE Buffer. Let the column sit for 5 min  prior centrifugation.
Discard flowthrough and centrifuge again to ensure all residual ethanol is gone.
Place the column into a clean 1.5 mL Epp and elute by adding 50 uL of NF-ddH2O. Make sure to wait 2 min before centrifugation.
Store at 4ºC overnight or -20ºC for longer periods.
Ligation set up
Fermentas T4 DNA Ligation
Reagent
Vector (pPB.003)
25 ng
Insert (pPB.014)
75 ng
T4 DNA Ligase Buffer 10X
1 uL
T4 DNA Ligase
0.5 uL
NF-ddH2O
as needed
Total
10 uL
Incubate vector @22ºC for 60-120 minutes and insert for 0.5-1 hour.
Transform 20 uL of NEB Turbo competent cells with 5 uL of ligation product.
Recover for 60 minutes on LB.
Plate on selective LBA+Cm plates. Incubate at 37ºC, colonies should appear overnight or within 24 hours max.
Recovered the cells for 45 minutes: No colonies (27/06)
Re-plated remaining recovered cells (overnight @ room temperature): No colonies (27/06)
<Repeated transformation (29/06): No colonies.
pPB.016 Enhanced Wintergreen Generator
pPB.015: Enhanced Wintergreen generator Salis Promoter (BBa_23108)
Requires 2 mM Salicylic acid
8/07 Digestion from miniprep set up
FD Restriction Digestion
Reagent
Volume (uL)
NF ddH2O
17
Buffer
3
FD Enzyme I
1
FD Enzyme II
1
DNA
3
Total Volume
25
Cut pPB.014 with SpeI & PstI
Expected fragments:
~2 kb
~18  bp
Cut pPB.005 with XbaI and PstI
Expected fragments:
3.4 kb
1.26 kb
8/07 SAGE  set up
Sage:
SAGE on thick gel (3 mm) 1% agarose should be fine.
There should be two bands for pPB.005.
Gel extraction:
Cut out the second band of pPB.004 and pPB.005 (1.8 kb and 1.3 kb respectively).
Weigh the agarose fragments (100 mg gel : 300 uL QG Buffer)
Max amount is 400 mg
Solubilize agarose completely by incubating at 50ºC for 10 min in water bath. Vortexing the tube every 3 minutes helps.
Color of mixture should be yellow. Otherwise add 10 uL of 3 M sodium acetate pH 5.
Add the equivalent of 1 gel volume of isopronanol and mix the tube (although for fragments between 500 bp and 4 kb this should matter).
Place column in a 2 mL collection tube
Load sample into column. Centrifuge 1 min @ 13 000 rpm.
Discard flowthrough and repeat adding 0.5 mL of QG buffer.
Discard flowthorugh and wash with 0.75 mL of PE Buffer. Let the column sit for 5 min  prior centrifugation.
Discard flowthrough and centrifuge again to ensure all residual ethanol is gone.
Place the column into a clean 1.5 mL Epp and elute by adding 50 uL of NF-ddH2O. Make sure to wait 2 min before centrifugation.
Store at 4ºC overnight or -20ºC for longer periods.
Ligation set up
Fermentas T4 DNA Ligation
Reagent
Vector (pPB.014)
25 ng
Insert (pPB.005)
75 ng
T4 DNA Ligase Buffer 10X
1 uL
T4 DNA Ligase
0.5 uL
NF-ddH2O
as needed
Total
10 uL
Incubate @22ºC for 10 minutes.
Transform 20 uL of NEB Turbo competent cells with 5 uL of ligation product.
Recover for 60 minutes on LB.
Plate on selective LBA+Cm plates. Incubate at 37ºC, colonies should appear overnight or within 24 hours max.
No colonies.
Check-up after vacations:
2/09/14:
Run gel of old samples
pPB.003   TetR Promoter
pPB.004    RBS-ATF1
pPB.005    RBS-BMST1
pPB.014    BBa_J23108
SAGE showed very faint bands suggesting a low [DNA] ( I used 4 uL of QIAGEN gel purification products).
Start 5 mL overnights of single colonies from Estefania’s plates.
pPB.015: Anderson’s Promoter + BMST1
pPB.016: Anderson’s Promoter + ATF1
pPB.022 (BBa_J45004) : BMST1 from Registry 2012 kit
3/09/14:
The gel purified fragments I have should be:
pPB.003 (Digested with SpeI & PstI) ~2.1 kb
pPB.004 (Digested with XbaI & PstI) ~1.765 kb
pPB.005 (Digested with XbaI & PstI) ~1.26 kb
pPB.014 (Digested with SpeI & PstI) ~2 kb
5/09/14:
Analytical digestion of Estefania’s pPB.015 and pPB.016 with NheI and PstI.
The Anderson’s promoter has two NheI sites. A restriction digestion with NheI and PstI should yield the following fragments:
pPB.015: 27 bp, 1.2 kb, 2.06 kb
pPB.016: 27 bp, 1.11 kb, 2.06 kb
FD Restriction Digestion
Reagent
Volume (uL)
NF ddH2O
17
Buffer
3
FD NheI
1
FD PstI
1
DNA
3
Total Volume
25
Incubate at 37ºC for at least 1 hour.
Ligation setup:
Fermentas T4 DNA Ligation
Reagent
Vector (pPB.003 or pPB.014)
25 ng
Insert (pPB.004 or pPB.005)
75 ng
T4 DNA Ligase Buffer 10X
1 uL
T4 DNA Ligase
0.5 uL
NF-ddH2O
as needed
Total
10 uL
Incubate @22ºC for 10 minutes.
Transform 20 uL of NEB Turbo competent cells with 5 uL of ligation product.
Recover for 60 minutes on LB.
Plate on selective LBA+Cm plates. Incubate at 37ºC, colonies should appear overnight or within 24 hours max.
No colonies.
Banana Fix
The sequencing results from 16/09/2014 show that the promoter got inserted between the EcoRI and XbaI restriction sites.  It is supposed to be downstream of XbaI. Also, the promoter sequence seems to be erroneous on the assembled plasmid (pPB.016) but not on the plasmid containing the promoter (pPB.014).  I will clone it again.
1.- Restriction digestion
Cut 300 ng of pPB.014 with SpeI & PstI
Expected fragments:
~2 kb
~18  bp
Cut 300 ng of pPB.004 with XbaI and PstI
Expected fragments:
~ 2 kb
1.765 kb
FD Restriction Digestion
Reagent
Volume (uL)
NF ddH2O
13.5
Buffer
4
FD Enzyme I
1.5
FD Enzyme II
1.5
DNA
3
Fast AP
1.5 uL
Incubate at 37ºC for at least one hour.
pPB.004 Digestion Product has to be gel purified. Extract the lower band of the gel.
pPB.014 Purify with PCR cleanup kit.
2.- Purification
Run a gel with a lot of pPB.004 (~20 uL of digestion product) for purification using gel excision kit. Purify pPB.014 with the PCR cleanup kit.
3.- Ligation
Fermentas T4 DNA Ligation
Reagent
Vector (pPB.003)
25 ng
Insert
75 ng
T4 DNA Ligase Buffer 10X
1 uL
T4 DNA Ligase
0.5 uL
NF-ddH2O
as needed
Total
10 uL
Incubate @22ºC for 10 minutes.
Transform 20 uL of NEB Turbo competent cells with 5 uL of ligation product.
Recover for 60 minutes on LB.
Plate on selective LBA+Cm plates. Incubate at 37ºC, colonies should appear overnight or within 24 hours max.
4.- Sequencing
September 17,2014
The sequenced samples from Estefania (EST pPB.015) do not contain matching repeats to the reference sequence (pPB.015). However, the old samples I was working with (pPB.005) have a match with the original mint construct taken from the registry kit. In other words, the pPB.005 contains a BMST1 CDS without promoter, and pPB.015 has a non matching sequence (corresponds to limonene enzyme…). My guess is that the wrong samples were sent for sequencing due to mislabeling or magic.
September 23, 2014
Fix it.
I digested some pPB.014 (Fig.1) with SpeI and PstI. This will open up a cohesive end downstream of the BBa_J23108 promoter. Alkaline phosphatase is added to the reaction mix to dephosphorylate the 5’ end of the fragment, this will help to prevent vector re-circularisation. The ATF1 promoter-less generator will be cut out from pPB.004 by restriction digestion with XbaI and PstI. Complementary cohesive ends will be formed upstream of the BBa_B0030 RBS (XbaI) and downstream of the BBa_0012 transcriptional terminator (PstI). This fragment (Fig.2: 1) will be inserted into the linearised pPB.014 (Fig.2: 4) by ligation with T4 DNA ligase after a gel purification.The resulting plasmid will contain the promoter upstream of the RBS, enabling constitutive expression of ATF1, and thus production of banana smell when transformed cells are grown in presence of isoamyl alcohol.
Fig.1: Constitutive promoter from the Anderson’s promoter library (BBa_J23108) in a pSB1C3 vector with standard BioBrick format.
Fig.2: Restriction digestion products:
1.- RBS (BBa_B0030), ATF1 CDS, and transcriptional terminators (BBa_B0010 and BBa_B0012). This will be purified by gel excision.
2.- Fragment of pSB1C3 vector.
3.- Fragment between SpeI and PstI sites (BioBrick suffix)
4.- Linearised pSB1C3 with appropriate cohesive ends for inserting fragment 1.
Fig.3: Final assembled plasmid containing a constitutive ATF1 expression system in a pSB1C3 vector with standard BioBrick format.
gBlock PCR Troubleshooting
Reagent
Volume
1x
4x
Nuclease-free water
71 ul
284 ul
5x Phusion HF Buffer
20 ul
80 ul
10 mM dNTPs
2 ul
8 ul
Forward Primer (10 uM)
1 ul
4 ul
Reverse Primer (10 uM)
1 ul
4 ul
Template DNA (Plasmid, 1:10 dilution)
1 ul
4 ul
DMSO
3 ul
12 ul
Phusion DNA Polymerase
1 ul
4 ul
Total Volume
100 ul
Thermocycler Protocol: NEB Phusion
Temp
Time
Start
98 °C
30 sec
Melt
Cycle 1
98 °C
10 sec
Melt
35 Cycles
Cycle 2
52 °C
30 sec
Anneal
Cycle 3
72 °C
40 sec
Extend
Finish
72 °C
5 min
Extend
Store
10 °C
Forever
Store
LIGATE EVERYTHING
Ligation
[pPB.004]= 8.5 ng/uL
[pPB.014]= 2.6 ng/uL
Banana and Mint ligations:
pPB.004 (XbaI PstI) + pPB.014 (SpeI PstI)
pPB.005 (XbaI PstI) + pPB.014 (SpeI PstI)
Controls:
Intact pPB.014
pPB.014 (SpeI PstI)
Ligase + pPB.014 (SpeI PstI)
Ligase + NF dH2O
Digested 30 ng of gPB.005 and 30 ng of gPB.006 with EcoRI and PstI. Both of them were column purified using the QIAGEN kit. They were eluted in 10 uL of NF dH2O. Some linearised pSB1C3 was prepared by PCR with oPB.081 and oPB.082, the product was column purified. A restriction digestion of pPB.003 with EcoRI and PstI was done in parallel and gel purified.
[gPB.005]= 2.2 ng/uL
[gPB.006]= 2.3 ng/uL
gBlocks Mint and Geosmin:
gPB.003 (EcoRI PstI) + pSB1C3 (EcoRI PstI)
gPB.005 (EcoRI PstI) + pSB1C3 (EcoRI PstI)
gPB.006 (EcoRI PstI) + pSB1C3 (EcoRI PstI)
PCR Vector:
Controls:
Intact pSB1C3 (PCR Product)
pSB1C3 (EcoRI PstI)
Ligase + pSB1C3 (EcoRI PstI)
Ligase + NF dH2O
Keep insert+vector volume below 4 uL.
Fermentas T4 DNA Ligation
Reagent
Vector (pPB.003)
25 ng
Insert
75 ng
T4 DNA Ligase Buffer 10X
1 uL
T4 DNA Ligase
0.5 uL
NF-ddH2O
as needed
Total
10 uL
Incubate @22ºC for 10 minutes.
Transform 20 uL of NEB Turbo competent cells with 5 uL of ligation product.
Recover for 60 minutes on LB.
Plate on selective LBA+Cm plates. Incubate at 37ºC, colonies should appear overnight or within 24 hours max.
The first 6 samples are PCR products of gPB.005 (BMST1) with JW.340 and JW.341. One uL of 1:10 dilution from the gBlock stock was used as template for the first set of three samples (~1 ng of DNA). The volume of 10 uM primers was 1, 1, and 0.5 uL. One uL directly from the stock was used as template for the second set (~10 ng of DNA).  The volume of 10 uM primers was 0.5, 1, and 1.5 uL respectively.  The last two lanes before the ladder are the PCR products of 1uL of  BBa_J45119 (~140 ng)  with oPB.081 and oPB.082. I got confused and thought this biobrick was in pSB1C3 vector, but I was wrong. I expected a band of ~ 2 kb, instead there is bands of ~3.3 kb which correspond to the length of pSB1AT3.
The second half of the gel follow the same scheme as the first one except that the DNA template is gPB.006 (Geosmin) and the vectors are from a 1:10 dilution of a 140 ng/uL BBa_J45119 mini prep (~14 ng) instead. The used program is:
Thermocycler Protocol: NEB Phusion
Temp
Time
Start
98 °C
30 sec
Melt
Cycle 1
98 °C
10 sec
Melt
35 Cycles
Cycle 2
52 °C
30 sec
Anneal
Cycle 3
72 °C
60 sec
Extend
Finish
72 °C
10 min
Extend
Store
10 °C
Forever
Store
-
I will repeat the PCR for the gPB.003 (oPB.013 and oPB.014), gPB.005, gPB.006 (JW.340 and JW.341) using the following conditions:
Phusion MasterMix
Reagent
Volume (uL)
DNA
1
Fwd (10 uM)
2.5
Rev (10 uM)
2.5
DMSO
1.5
MM
25
H2O
17.5
50
I will do each sample by duplicate varying only the [DNA template] using 1 uL directly from stock tubes and a 1:10 dilution as well. For the pSB1C3 (oPB.081 and oPB.082) I will use 1 uL of pPB.003 (RBS) or pPB.004 (Banana generator) at 1:10 and 1:100 dilutions.
The program will be the same except that I will increase the annealing temperature to 55ºC.
gPB.003: 1307 bp
gPB.005: 1211 bp
gPB.006: 1238 bp
pSB1C3: 2070 bp
The PCR was repeated with a 58ºC annealing temperature:
29/09/2014
Colony PCR of pPB.015  and pPB.016 BL21 colonies:
Phusion MasterMix
Final concentration
Reagent
Volume (uL)
DNA
1
< 250
Fwd (10 uM)
1
0.5 uM
Rev (10 uM)
1
0.5 uM
DMSO
0.6
3%
MM
10
1X
H2O
6.4
20
Purification/ligation of digested gBlocks:
Keep insert+vector volume below 4 uL.
Fermentas T4 DNA Ligation
Reagent
Vector (pPB.003)
25 ng
Insert
75 ng
T4 DNA Ligase Buffer 10X
1 uL
T4 DNA Ligase
0.5 uL
NF-ddH2O
as needed
Total
10 uL
Digestion of gBlocks:
Digestion of vectors
Plamid DNA
20
10x FD Buffer
10
FD Enzyme  I
2.5
FD Enzyme II
2.5
FastAP
2.5
NF H2O
62.5
Total
100
Incubate @ 37ºC for 2-4 hours with shaking.
Digestion of inserts
Plamid DNA
16
10x FD Buffer
2
FD Enzyme I
1
FD Enzyme II
1
NF H2O
0
Total
20
Incubate @ 37ºC for 0.5-1 hours with shaking.
Hurrying
Heat shock:
5 uL of ligation product to 20 uL of NEB Turbo competent cells @42°C for 30 seconds.
2 min in ice.
200 uL of LB and recover for a couple of hours @ 37ºC.
*Prepare NEB Turbo overnight culture for electroporation in cae heat shock fails.
Colony PCR 2:
Phusion MasterMix
Final concentration
Reagent
Volume (uL)
DNA
1
< 250
Fwd (10 uM)
1
0.5 uM
Rev (10 uM)
1
0.5 uM
DMSO
0.6
3%
MM
10
1X
H2O
6.4
20
Thermocycler Protocol: NEB Phusion
Temp
Time
Start
98 °C
30 sec
Melt
Cycle 1
98 °C
10 sec
Melt
35 Cycles
Cycle 2
55 °C
30 sec
Anneal
Cycle 3
72 °C
60 sec
Extend
Finish
72 °C
10 min
Extend
Store
10 °C
Forever
Store
*Colony PCR pics are unavailable due to failure of USB key (files were corrupted and pictures became black).
1/10/2014
Submerged agarose gel electrophoresis of the PCR products showed a ~1.3 kb band on the sample from colony#2 (14+4). The expected product with oPB.006 and oPB.074 is ~2 kb. Phusion polymerase extends at a rate of ~ 100 bp/s, so the 60 s extension time should be good. However, the primers have a Tm of 51.8ºC and 49.7ºC respectively. Therefore, the annealing temperature might be too high. Run another colony PCR with the verification primers using an annealing temperature of 50.7ºC.
Thermocycler Protocol: NEB Phusion
Temp
Time
Start
98 °C
30 sec
Melt
Cycle 1
98 °C
10 sec
Melt
35 Cycles
Cycle 2
50 °C
30 sec
Anneal
Cycle 3
72 °C
60 sec
Extend
Finish
72 °C
10 min
Extend
Store
10 °C
Forever
Store
October
1/10/2014
pPB.015 (Mint) and pPB.016 (Banana) minipreps:
I used the Promega kit.
pSB1C3 and gBlock amplification by PCR using Phusion polymerase:
Thermocycler Protocol: NEB Phusion
Temp
Time
Start
98 °C
30 sec
Melt
Cycle 1
98 °C
10 sec
Melt
35 Cycles
Cycle 2
59 °C
30 sec
Anneal
Cycle 3
72 °C
60 sec
Extend
Finish
72 °C
10 min
Extend
Store
10 °C
Forever
Store
2/10/2014
To DO:
1.- Colony PCR with oPB.006 & oPB.074, annealing temperature= 50ºC.
2.- Nanodrop miniprep samples.
3.- Set analytical digestion of pPB.015 and pPB.016 with AvrII, BstXI, HindIII, and PstI.
Expected fragments:
300 bp
968 bp
1258 bp
2076 bp
4.- Send for sequencing if positive.
5.- gBlock PCR again.
6.- Set up electroporation stuff.
7.- Gel purify yesterday’s PCR products.
8.- Run a small volume SAGE of purified PCR products to confirm size.
9.- Digest old PCR products.
10.- Purify digestion products.
11.- Set up ligation reaction.
12.- Electroporate/Heat shock.
Protocol: E. coli Chromosomal Integration (à la Datsenko and Wanner)
Preparation of Electrocompetent Cells
Note: Competent cells should never be vortexted, as this will cause them to lyse and release salts into the media. Resuspend cells by pipeting up and down with a large pasteur pipet. Once they are chilled, cells should be continuously cold.
1) The night before the transformation, start an overnight culture of cells.
5 ml LB Amp.
2) The day of the transformation, dilute the cells 100X.
100 ml LB Amp.
Grow at 30°C for about 90 minutes.
3) Harvest the cells.
When the cells reach an OD600 of between 0.6 and 0.8.
Split the culture into 2x 50 ml falcon tubes, on ice.
Centrifuge at 4 °C for 10 min at 4000 rpm.
4) Wash and combine the cells.
Remove the supernatant.
Resuspend the cells in 2x 25 ml of ice cold water.
Combine the volumes in a single 50 ml falcon tube.
5) Wash the cells 2 more times.
Centrifuge at 4 °C for 10 min at 4000 rpm.
Resuspend in 50 ml of ice cold water.
Repeat.
6) Wash and concentrate the cells for electroporation.
Centrifuge at 4 °C for 10 min at 4000 rpm.
Resuspend in 1-2 ml of ice cold water.
We will use 200 ul of washed cells per transformation.
Dialysis of PCR or Digestion Products
Note: DNA for electroporation must be free of salts to avoid arcing.
1) Float a filter in a Petri dish filled with water.
Millipore membrane filter 0.025 uM.
2) Pipet one drop of PCR product onto the filter.
200 ng is needed per transformation.
20 - 100 ul fits well on one filter.
3) Collect the drop after 30 - 45 minutes.
The volume will change, but the DNA is not lost.
Protocol: Electroporation of E. coli
Note: This protocol uses 200 ul of cells and 200 ng of DNA.
The cuvette is 2 mm. We use electroporation setting EC2.
A time constant above 5 indicates a successful electroporation.
1) Prepare BD tubes with a pipette filled with LB at the interior of each tube (pipette supplied with the electroporation cuvette)
2) Test the purity of the electrocompetent cells.
Add 200 ul of washed cells to a cuvette.
3) Mix the cells and DNA in a cuvette.
200 ul of washed cells with 200 ng of PCR product.
Keep the cuvette on ice until just before the electroporation.
4) Preload a pipette with 1 ml of LB.
5) Pulse the cuvette with voltage.
Dry the electrodes with a kimwipe prior to loading.
Use the EC2 setting.
6) Listen for arcing.
A cracking sound means all the cells are dead.
Note the time constant: 5 is good, 5.8 is great.
7) Immediately recover the cells.
Add the 1 ml of preloaded LB and pipet up and down to mix.
Collect 1 ml of cells, some volume is lost in the cuvette.
8) Incubate 2 h at 37 °C with shaking.
9) Plate 100 ul of recovered cells on selective plates.
Use antibiotic appropriate to the part being integrated.
Let the other 900 ul rest overnight at room temperature.
10) Concentrate and plate the remaining cells
Spin down quickly and resuspend in 100 ul LB before plating.
Transformed cells should be incubated at 37 °C.
Colonies should appear 24-48 h after plating.
13.- Recover.
14.- Plate.
Start again
October 8, 2014
Banana BioBrick
[DNA] (ng/uL)
Volume (uL)
Amount (ng)
vector pPB.014
34
1
34
insert (pPB.004)
15
5
75
Buffer
2
T4 Ligase
1
9
H2O
11
Jasmine gBlock
[DNA] (ng/uL)
Volume (uL)
Amount (ng)
vector
4
12
48
insert (pPB.003)
5
19
92.4
1307
Buffer
6
T4 Ligase
3
40
gPB.003 PCR with oPB.013 & oPB.014. pSB1C3 PCR with oPB.081 and oPB.082.
Phusion MasterMix
Final concentration
Reagent
Volume (uL)
DNA
1
< 250
Fwd (10 uM)
1
0.5 uM
Rev (10 uM)
1
0.5 uM
DMSO
0.6
3%
MM
10
1X
H2O
6.4
20
Thermocycler Protocol: NEB Phusion
Temp
Time
Start
98 °C
30 sec
Melt
Cycle 1
98 °C
10 sec
Melt
35 Cycles
Cycle 2
59 °C
30 sec
Anneal
Cycle 3
72 °C
60 sec
Extend
Finish
72 °C
10 min
Extend
Store
10 °C
Forever
Store
Yesterday’s heat shock transformation of E. coli tnaA deletion mutant from Keio Collection with geosmin, JMT, BMST1, and ATF1 expression cassettes yielded colonies for all except ATF1.
Colony PCR
Phusion MasterMix
Final concentration
Reagent
Volume (uL)
DNA
1
< 250
Fwd (10 uM)
1
0.5 uM
Rev (10 uM)
1
0.5 uM
DMSO
0.6
3%
MM
10
1X
H2O
6.4
20
Expected product with oPB.006 and oPB.074 is ~2 kb. Phusion polymerase extends at a rate of ~ 150 bp/s, so the 60 s extension time should be good. The primers have a Tm of 51.8ºC and 49.7ºC respectively.
Thermocycler Protocol: NEB Phusion
Temp
Time
Start
98 °C
30 sec
Melt
Cycle 1
98 °C
10 sec
Melt
35 Cycles
Cycle 2
50.7 °C
30 sec
Anneal
Cycle 3
72 °C
60 sec
Extend
Finish
72 °C
10 min
Extend
Store
10 °C
Forever
Store
*Geosmin pPB.025 was successfully transformed.
Characterizations planning
Media Preparation:
M9 Minimal Media 0.4% Glucose. Bring to 1 L
Reagent
Qty
5x M9 Salts
200 mL
1M MgSO4
2 mL
20% Glucose Stock Solution
20 mL
1M CaCl2
0.1 mL
Complete AA mix
2 g
Banana:
5 mM isoamyl alcohol
Molecular Weight 88.15
Add 4.4075 g to 100 mL of sterile dH2O. This is a 1 M stock solution.
Use 50 uL in 9.95 mL of M9 Media.
Mint:
2 mM Salicylic acid
Molecular Weight 138.12
Add 6.906 g to 100 mL of dH2O. This is a 1 M stock solution.
Use 20 uL in 9.98 mL of M9 Media.
Geosmin:
FPP
Jasmine:
Jasmonic acid
Limonene:
FPP
Butter:
Acetoin
1.- Make an overnight culture.
2.- Dilute them 1:200 by duplicate.
3.- Treat one of the duplicates with substrate, leave the other without, and grow to exponential phase. (Make a control of selective media+substrate)
4.- Smell them when they reach mid-log phase.
5.- Add substrate to the untreated samples.
6.- Grow to stationary phase and smell.
Centre for Research and Interdisciplinarity (CRI) Faculty of Medicine Cochin Port-Royal, South wing, 2nd floor Paris Descartes University 24, rue du Faubourg Saint Jacques 75014 Paris, France
+33 1 44 41 25 22/25
paris-bettencourt-igem@googlegroups.com
Copyright (c) 2014 igem.org. All rights reserved.
Retrieved from " http://2014.igem.org/Team:Paris_Bettencourt/Notebook/Odor_Library "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
