"
Page
Discussion
View source
History
teams
Log in
 
Team:Paris Bettencourt/Project/Eliminate Smell
From 2014.igem.org
Project
SmellThe Roses
SomethingFishy
GoodyTwo Shoes
Don'tSweat It
Teen Spirit
Design Practice
Bioinformatics
Achievements
Results
Interlab Study
MedalFulfilment
Parts
Policy
Safety
Consent Form
Patenting
Practices
MOOC
CitizenScience
The Senseof Smell
The HumanMicriobiome
Minder
Collaborations
Collaborations
The SmellGame
Newsletters
Notebook
Daily Note
Protocols
Bibliography
Team
Our Team
Attributions
Official TeamProfile
Acknowledgements
@iGEM_Paris
  Gym Class Heroes
BACKGROUND
Sweat is odorless until it is metabolized by the bacteria of the skin. If the enzymes responsible for producing odorant molecules could be removed from the skin microbiome, body odor would be attenuated or eliminated. In this project we explore strategies to identify, target, and remove odor-related genes from a complex population.
AIMS
This project is subdivided into 3 approaches. First we seek to understand odor production by a single enzyme related to armpit odor. Next we develop a CRISPR-based tool for isolating precise loss-of-function mutants. Finally we use bioinformatics to identify potential gene targets to eliminate with our tool.
ACHIEVEMENTS
Expressed the odor gene agaA in E. coli (BBa_K1403002).
Designed and used CRISPRs to enrich for loss-of-function mutants 1000x in a mixed population.
Identified potential CRISPR targets as odor genes with variability hotspots in metagenome data.
Formulated a simple skin cream encorporating live bacteria.
Genes Affect Odor
A CRISPR/Cas System
The Probiotic Cream
Natural variants of odor genes
Figure 1. Enzymes responsible for body odor in the human axilla ( Tauch, 2013 et al., 2001 ).
Figure 2. Smell test. 14 people smelled two tubes of E. coli grown to saturation in LB. One culture carried synthetic agaA and other an empty vector control. 13 people out of 14 rated the E. coli carrying agaA as more smelly (pink) than the control(violet). Experiments were double-blind. Significance was confirmed by Chi square test (p-value = 0.001341).
Figure 3. Phylogenetic tree produced from 16S sequencing of Corynebacterium from human axillary samples. Corynebacterium was cultured on blood agar plates and the 16S region of selected colonies was sequenced. Sequencing results are presented here in a phylogenetic tree format. The tree was made by creating a multiple sequence alignment (MSA) using the CLUSTALW algorithm and then creating a neighbor-joining tree using the MSA. The length and order of the branches is not accurate due to the evolutionary closeness of the species sampled.
1. Genes and Bacterial Species Affect Odor
Introduction
Body odor is a complex combination of odorants, but the major components come from just a few known enzymes (Fig. 1). In the armpit, for example, a major source of "pungent" or "musky" odors is the enzyme AgaA expressed by Corynebacterium striatum ( Kligman et al., 1981 ). AgaA is an aminocylase that hydrolyzes 3-methyl-2-hexenoyl-glutamine (3M2H-gln) into 3-methyl-2-hexenoic acid (3M2H) and free glutamine. The substrate is secreted in sweat, and the product is a distinctive armpit odor.
Results
To understand the smell created by AgaA, we cloned the gene into E. coli (BBa_K1403002). A noticeable odor was produced by agaA-expressing E. coli grown in selective LB, described as "beer-like" or "cheese-like". We confirmed this observation with a formal double-blind smell test (Fig. 2). Although the native substrate of the enzyme is not present in LB, the enzyme is known to have a low specificity for the acyl group, and to act on a range of glutamine conjugates ( Natsch et al., 2003 ). These observations highlight the remarkable affect of single gene on the odor profile of a bacterial population.
We next investigated bacterial species involved in human body odor. We collected sweat samples from our team, from other iGEM teams, and during our public exibition at the Cité des Sciences. Samples were plated on blood agar with phosphomycin, a media selective for Corynebacterium. We identified 15 distinctive species by 16S sequencing (Fig. 4), each with a noticeably different smell.
Methods
1. Cloning agaA into E. coli
The AgaA enzyme from Corynebacterium ( Genbank: AF534871.1 ) was codon optimized using the IDT online tool (Integrated DNA Technologies, Belgium) for E. coli K12 and was commercially synthesized by IDT. PCR was performed to obtain the agaA construct using a forward primer that contained the Biobrick prefix and a RBS designed using the Salis online tool ; and a reverse primer that contained the BioBrick suffix. The PCR product was purified.
The agaA construct and pSB1C3 vector were digested using EcoRI and PstI. After, the agaA construct was purified using the PCR Purification Kit . The vector was gel extracted and purified. Both parts were ligated using Ligase T4.
Competent E. coli Neb Turbo were transformed with the ligation product by heat shock. Then, they were recovered during 2 hours in 200 uL of LB medium at 37ºC and plated in selective plates with LB and chloramphenicol in a concentration of 1 uL/mL. The transformation was confirmed by analytical digestion.
2. Find Corynebacterium in human skin samples.
After collecting human axilary samples, we cultured them in blood agar plates to select for Corynebacterium and performed 16S sequencing. We made a multiple sequence alignment (MSA) using the CLUSTAW algorithm and we created a neighbor-joining tree using the MSA. Finally, we obtained a phylogenetic tree for the Corynebacterium found in our samples (Fig. 3).
Figure 4. The designed CRISPR/Cas system targets the LacZ gene of a wild type and a mutant. The targeting sequence is 30 nucleotides wide and targets the wild type sequence of lacZ. The mutant strain has a deletion and a substitution at positions 1470 and 1473 respectively, decreasing the specificity of the CRISPR for it.
Figure 5. The designed CRISPR/Cas system is capable of discriminating small genomic differences. The wild type (LacZ+) and the mutant (LacZ-) were transformed with the non-targeting CRISPR (Random CRISPR) and the LacZ-targeting CRISPR (LacZ CRISPR). The non-targeting CRISPR resulted in comparable number of transformed cells among the wild type and the mutant. The LacZ-targeting CRISPR resulted in the killing of most of the wild type, but in the survival of a substantial number of the mutants (40% compared to the control non-targeting CRISPR).
Figure 6. The designed CRISPR/Cas system is capable of isolation a mutant from a mixed population. The wild type and the mutant strains were mixed right before electroporation. Since the wild type produces a blue color and the mutant stays colorless, we measured the final frequency of mutant by blue/white screening. The transformation with the non-targeting CRISPR resulted in a final proportion of the mutant proportional to the initial concentration. However, when transformed with the LacZ targeting CRISPR, the final proportion of mutant was almost at 100%. This property is maintained until a ratio of 1/106 after which the isolation loses efficacy.
Figure 9. We took microbiome samples of 12 volunteers, performed DNA extraction and PCR amplified for ldh, L-ldh, D-ldh and ackA. The samples from 5 volunteers were positive for the PCR and we Sanger sequenced them.
Figure 10. Hotspots found for enzymes D-Ldh in human microbiome axillary samples.
2. A CRISPR/Cas System to Enrich for Natural Mutants
Introduction
Removing genes (like agaA) from the bacteria that express them (like Corynebacterium), should attenuate their associated odor. Classical genetic tools can produce gene deletions. Odor-negative deletion mutants, applied to the skin in large numbers, could replace a wild-type strain by the principle of competitive exclusion. However, this approach faces two difficulties:
1) Gene deletions may reduce fitness, rendering the mutant unable to compete with the WT for a niche.
2)Genetically Modified Organisms (GMOs) face special regulations and limited public acceptance.
We propose an alternative strategy to isolate naturally occurring loss-of-function mutants from a complex microbiome. Natural mutants, if they are isolated from the microbiome itself, are likely to retain high fitness. Public acceptance is higher and biosafety concerns are reduced for non-GMO microbes.
The CRISPR-Cas system has been shown to target DNA sequences for cutting with single-base precision, providing means of targeting genome editing or killing specific pathogens ( Jiang et al,, 2013; Gomaa et al., 2014 ). Here we present a CRISPR-based protocol for isolating pre-existing mutants from a mixed population. Mutants that mismatch a CRISPR target sequence will survive transformation with a CRISPR-expressing plasmid and therefore be enriched in the post-transformation population. In principle, CRISPR-based mutant enrichment does not alter the existing mutant genome and the isolated organisms do not meet the conventional definition of GMOs.
Results
As a proof-of-principle, we designed a CRISPR system on a single plasmid with a tracrRNA targeting lacZ. The targeting sequence was designed to match the wild-type sequence, but not a loss-of-function mutant sequence with two point mutations (Fig. 4).
Our CRISPR system was capable of sequence-specific killing (Fig. 5). Transformation with the lacZ-targeting CRISPR significantly killed less than 98% of wild-type bacteria compared to a control CRISPR with a randomized tracrRNA. In contrast, about 40% of mutant bacteria survived transformation relative to control. This result showed that our construct could selectively kill wild-type bacteria.
We next used our construct to enrich for mutant bacteria in a mixed population (Fig. 6). Wild-type and lacZ- strains were mixed in measured proportions. Blue/white screening for lacZ activity was applied following transformation to measure the frequency of lacZ- mutants post-transformation. Even when lacZ- mutants represented only 1:106 of the initial population, they represented the large majority of the population post-transformation. In contrast, transformation with randomly-targeted CRISPR produced a linear proportionality between lacZ- frequency before and after transformation.
CRISPR-mediated cleavage may select for mutants, but it can also induce mutagenesis. To distinguish between these two possibilities we additionally labeled lacZ mutants with a plasmid expressing YFP. Wild-type cells were similarly labeled with CFP. This labeling allowed us to distinguish a CRISPR-selected mutant (lacZ-, YFP+) from a CRISPR-induced mutant (lacZ-, CFP+). We found CRISPR-induced mutagenesis to be rare, representing less than 10% of the total isolated mutants in most cases (Fig. 7).
To use this system in strains beyond E. coli, we ported the entire CRISPR/Cas system into the broad host-range vector pSEVA351. This new vector is also functional in E. coli (Fig. 8) but carries a viral replication origin that functions in many species. This vector should serve as a tool to enable CRISPR-mediated mutant enrichment in many species of the skin microbiome.
Methods
In order use a cloning strain which wouldn't be killed by the CRISPR, the strain BL21-AI has been chosen for all the clonings. However, prior to its use in cloning purposes, its chloramphenicol resistance has been flipped out by the use of the thermosensitive plasmid pCP20. The pCas9 plasmid was the CRISPR vector used. Its targeting sequence has been changed by the use of the enzyme Eco31I (BsaI) in regular cloning technique. Regarding the mixing of different strains for the mutant enrichment purpose, after the third washing, the strains were diluted to reach the same OD600 and then mixed in different proportion. They were then washed one more time and resuspended in 1mL of water before electroporation.
Figure 7. CRISPR induced mutagenesis is rare during mutant isolation. After labeling the wild type with a plasmid expressing CFP and the mutant with a plasmid expressing YFP, we transformed mix of the two strain at a ratio of 1:102. The blue/white screening was then compared to the fluorescence. Colorless colonies expressing CFP fluorescence were representing potential induced mutants. These potential induced mutants represented about 8% of the colorless colonies. However it was found that their colorless phenotype was to be imputed to the imperfect efficiency of the blue/white screening method. This was confirmed by re-streaking on X-gal and sequencing. The colonies were blue after re-streaking, and presented the expected sequence of a wild type.
Figure 8. Broad-host range CRISPR has a similar efficiency. After cloning the whole CRISPR/Cas system in the vector pSEVA351, we compared its discrimination efficiency to our previous CRISPR/Cas system. It appears the even if the transformation efficiency seems to be in general higher with the new vector, discrimination properties are conserved. Indeed there is about 45% of survival of the mutant when compared to the control non targeting CRISPR.
3. Sequence Identification of Natural Variants of Odor Genes
To identify potential natural sequence variants that might provide CRISPR targets, we amplified and sequenced odor-related genes from human sweat samples (Fig. 9). We focused on enzymes producing odorant fermentation products in Staphylococcus species like acetate kinase (ackA) and lactate dehydrogenase (ldh) ( Tauch, 2013 et al., 2001 ).
We extracted, amplified and sequenced 4 genes from 12 individual sweat samples. A range of synonymous and nonsynonymous mutations were detected (Fig. 10). Each of these mutations represents a CRISPR-selectable target, and some of them may have functional consequences for an individual odor profile.
Figure 12: Cream with Corynebacterium striatum.
4.Probiotic Cream: a Cure for Body Odor
We formulated a basic skin cream to package and deliver live bacteria as a cosmetic. The cream is a water-in-oil emulsion using jojoba oil as a base, beeswax as an emulsifier and soy milk as a bacterial culture medium.
The soy milk media forms emulsified droplets 100 μM in scale, in which bacteria are protected from dessication. We tested the cream with RFP-expressing E. coli (Video of Cream with fluorescent E. coli) and with wild-type C. striatum (Fig. 12).
Video of Cream with fluorescent E. coli.
Centre for Research and Interdisciplinarity (CRI) Faculty of Medicine Cochin Port-Royal, South wing, 2nd floor Paris Descartes University 24, rue du Faubourg Saint Jacques 75014 Paris, France
+33 1 44 41 25 22/25
paris-bettencourt-igem@googlegroups.com
Copyright (c) 2014 igem.org. All rights reserved.
Retrieved from " http://2014.igem.org/Team:Paris_Bettencourt/Project/Eliminate_Smell "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
