"
Page
Discussion
View source
History
teams
Log in
 
Team:Paris Bettencourt/Newsletter4
From 2014.igem.org
Project
SmellThe Roses
SomethingFishy
GoodyTwo Shoes
Don'tSweat It
Teen Spirit
Design Practice
Bioinformatics
Achievements
Results
Interlab Study
MedalFulfilment
Parts
Policy
Safety
Consent Form
Patenting
Practices
MOOC
CitizenScience
The Senseof Smell
The HumanMicriobiome
Minder
Collaborations
Collaborations
The SmellGame
Newsletters
Notebook
Daily Note
Protocols
Bibliography
Team
Our Team
Attributions
Official TeamProfile
Acknowledgements
@iGEM_Paris
Back to top
Newsletter
- Newsletter 1
- Newsletter 2
- Newsletter 3
- Newsletter 4
iGEM 2014Weekly newsletter
Newsletter n°4
from Sept 1st to Sept 15th 2014
Hi, all the iGEMers.
Last time when I wrote on this page of the newsletter, it was our first issue. Now, the summer is coming to the end and this is our fourth issue of the newsletter, also our last regular issue. Afterwards, we will replace it by special issues and review sessions, as i mentioned before. A couple schools have asked me about the timeline. I understand that school is starting and life is getting crazy. The student review of the judge question would be done by October and the peer review session would be in the first or second week of October. I will send out one more email with details and the rubrics in the coming three days.
I can’t thank you enough for being part of this community. I hope this newsletter has been and will continue to be a valuable resource for your team. In the process, we started with 8 teams and have had over 25 different teams from all over the world involved.
Again, thank you so much for all the cooperation. One thing I would like to emphasize and encourage you to do is to reach out to other teams on your own, ask for advise, talk about ideas, or just have a casual conversation. One thing I personally love about synthetic biology is how young, ambitious and creative everyone is.
Good luck with the rest of your project and journey to Boston! Watch out for the emails about the following special issues and let us know if you have specific things you would like to read or learn about!
Newsletter n°4
from Sept 1st to Sept 15th 2014
ETH ZÜRICH
OUR TEAM
We are the team from Zurich in Switzerland. This year, we are seven highly motivated students from different backgrounds, aiming to rock the Information Processing track once again. The interaction between wet lab and dry lab is crucial for our team.
PROJECT UPDATE
Questions to other teams:
For XMU China: Which mathematical principles do you think govern cell differentiation? Which kind of logic circuitry do you have in mind to relate chemotaxis to patterns?
Futher Plan:
We will be further characterizing the different modules of our design.
Advertisement&Plz help:
There is only one week left to complete our survey on complexity and try to win the collaboration badges (only 20 answers are needed to get the first badge). You can find our survey on our Facebook page, on our wiki and on our twitter. Thanks a lot for your help!
Our project is based on the conjugation of quorum sensing, integrasebased logic gates and diffusion. We have been investigating quorum sensing for the past few weeks. We want to find two orthogonal quorum sensing systems. Our system has two inputs, which correspond to two different homoserine lactones (HSL) concentrations. Each sensing system should be sensitive to only one HSL and should not react if the other HSL is added. We want to avoid a phenomenon called cross-talk.
We investigated three different systems: the Lux system, the Las system and the Rhl system. For each system, we built one plasmid consisting of the promoter, which can be induced by a specific signaling molecule, and of a gene coding for fast folding green fluorescent protein (GFP). Each colony was induced by different concentrations of its native HSL and also by different concentrations of HSL of the two other systems, in order to investigate cross-talk. The read-out is green fluorescence.
From the gathered experimental data, we can observe steady state, as well as the dynamic behavior of the inducing process, thanks to the fast folding property of GFP. We modeled the steady state as a Hill function of the native HSL + a constant modeling the leakiness + other Hill functions depending on other HSLs, if cross-talk was observed. The model allows a quantification of leakiness and cross-talk.
@ETH_iGEM
iGEM ETH Zürich
igem2014@bsse.ethz.com
Newsletter n°4
from Sept 1st to Sept 15th 2014
PROJECT UPDATE
Make a Sierpinski triangle pattern appear in a grid Conjugate quorum sensing and logic gates in bacterial colonies Implement an XOR gate in an E. coli Characterize integrases (retrieve missing parameters) Study quorum sensing mechanism aiming to lower the leakiness Be able to predict accurately the system’s behavior Questions for other teams:
We want to ask all the team how is their structure to work. Does everybody do everything? Do you have leader for each task?
We have three main groups: Lab, Model and HP. Each of the has their leader and students
At the lab we have very exiting news ☺ !! The part circled in red of the circuit is almost done. We are beginning to think about experiments to prove that the biobrick works.
We told you we had problems with the parameters in the mathematical model, but fortunately the parts of our system have been well studied and we were abled to solve the problem. Right now we are evaluating different variations of the circuit (Feed back, no feed back, different reporter…) in order to give the lab the best circuit possible.
Because we were working really hard on the lab last weeks we did not get the chance to participate in the third issue. But we still want you to meet one member of our group: his name is Skerlock coli.
Sherlock coli is a private investigator specialized in finding pathogenic bacteria. In this project he is helping us to find the bad guy Vibrio cholerae. For this purpose, Sherlock is using quorum-sensing molecules as his first lead and as soon he finds V. cholera he will give us a colorful announce.
Right now we are trying to give Sherlock the right tools to find V. cholerae. We have almost finished the circuit and we need to connect it with the phosphorylation cascade made by Peking 2011 Team.
NEXT STEP
The next steps are to prove the circuit and see what improvements can be done. In the Mathematical model the next step is the stochastic simulation.
Colombia_igem1
Colombia Team- iGEM
d.olivera1320@uniandes.edu.co
Newsletter n°4
from Sept 1st to Sept 15th 2014
GOETTINGEN
OUR TEAM
University of Göttingen, Germany. We come from different countries: Germany, China, Iran, Lebanon, India and Mexico.
PROJECT UPDATE
We’ve had some issues with the transformation efficiency kit. The efficiency with the provided reagents seems to be almost null. After a series of tests using control plasmids, different competent batches and different bacterial strains, we came to the conclusion that the shipping of the kit may have affected it. We’d like to let other teams be aware of that in case they are having the same issues.
We have also been working to generate 3D structures by homology modeling for some of the peptides that showed affinity towards fungal proteins. However, they still need to be assessed more thoroughly.
Predicted structure of one of our selected peptides. The portion in blue shows the scaffold (B1 domain of protein G) and the red portion is our inserted peptide. We’re using Modeller to generate the structures by homology modeling (https://salilab.org/modeller/).
@iGEMGoe
iGEM Team Goettingen
igem2014@uni-goettingen.de
Newsletter n°4
from Sept 1st to Sept 15th 2014
PARIS BETTENCOURT
OUR TEAM
Paris-Bettencourt iGEM team is part of the Center for Research and Interdisciplinarity (CRI) in Paris.
PROJECT UPDATE
Treating strong body odor using probiotics, producing a low cost perfume produced by bacteria and setting up a database of microbiome correlated to body odor
Questions it raises : Are you more likely to use a product whose ingredients you are familiar with and can even make on your own? Is the formulation of a cream like the one we’ve described above something that you would put on your skin?
One of our sub-projects has to do with the development of the formulation that our probiotic will be given in. We wanted to make a cream that used components of daily life in order to create something that consumers can make at home and with products they can trust. One such formulation we are looking at is the development of a cream that is an emulsion of beeswax, jojoba oil, and the medium that our bacteria are grown in. We conducted an experiment of such an emulsion with fluorescent E. coli and viewed this cream under a microscope and recorded the images in a time-lapse video (which can be viewed here: http://youtu.be/4IgNmKaGdbU).
GFP E.coli under microscope.
NEXT STEP
Since our final probiotic will be with a C. striatum strain, we want to see if this bacteria can also be grown in medium which is easily found in grocery stores or something that one interacts with in their daily life. It turns out that C. striatum can actually grow very well on soymilk. Therefore, our next steps are to determine if soymilk needs to be supplemented with other nutrients or if we need to have other steps (such as boiling or filtration) in order to create the media to grow our final C. striatum strain which can then be emulsified with the beeswax and jojoba oil for our final formulation of our probiotic cream.
@iGEM_Paris
iGEM Paris Bettencourt
2014.igem.Parisbettencourt@gmail.com
Newsletter n°4
from Sept 1st to Sept 15th 2014
POLYTECHNIC UNIVERSITY OF VALENCIA
OUR TEAM
We are a team uniquely composed by students from the Polythecnic University of Valencia with the support of the Spanish National Research Council. In this iGEM edition we dare to work with plants and hope our work will encourage other teams to do so in the future.
PROJECT UPDATE
Avoiding damage in crops caused by pests, using pheromoneproducing plants that disrupt insects mating and therefore avoid laying and damage caused by larvae.
During the last few weeks we had some amazing results. We finally obtained pheromone production satisfactorily and tested the specific expression in glandular trichomes using GFP as a reporter. The rest of the components of our circuits are still under development and should be finished soon. By the way, keep summoning the demons of the failed PCRs please, we’re gonna need some extra help with this.
NEXT STEP
In the next few weeks, we’re doing some extra trials combining our pre-existing elements and performing the firsts tests for the newly constructed elements. We are very happy about the results obtained so far and feel optimistic about testing the new components! You’ll hear about us soon.
@ValenciaUPViGEM
Valencia UPV iGEM 2014
valencia.upv.igem@gmail.com
Newsletter n°4
from Sept 1st to Sept 15th 2014
LIKA-CESAR-BRASIL
PROJECT UPDATE
In our project, we have a robotics system, which had a huge update this week. There exists a centrifuge, a heater system, and a reagents insertion, for blood treatment. To reach a functional centrifuge design, we analyzed the weight, size and materials. The machine needs to have a stabilized mass center, in order to not break itself and the material has to support high temperatures, because of the heating system.
Besides the robotic part, we are working and learning the transformation and biobricks parts. We needed to adjust your biobrick part, due some problems of shipping that we had with our genes. So now, we are running against the time, working day and night to get the transformation and expression in the deadline. Now we building an alert system of DNA/RNA extraction, using a combination of regulators and reporters to make the E.coli recognize the extraction as a quality control.
@lika_cesar
IGEM 2014 LIKA CESAR Brasil
likacesarbrasil@gmail.com
Newsletter n°4
from Sept 1st to Sept 15th 2014
THE FATBERGLARS
OUR PROJECT
QUESTIONS TO OTHER TEAMS:
WOULD ANY TEAM BE INTERESTED IN TRIALLING OUR LAB NOTATION? OR BE INTERESTED IN HELPING US DEVELOP IT FURTHER. LET US KNOW IF YOU ARE :)
The last two weeks have been hectic for the team as the end of our project draws near. In labs we have worked towards planning the assays to test our lipase and keratinase BioBricks, getting our BioBricks into submission vectors and producing a nutrient cartridge for use in the bioreactor we have designed. We went to a waste water treatment plant in North Yorkshire to see a fatberg with our own eyes, which definitely smelt worse than anything Paris Bettencourt or Polytechnic University of Valencia would have to deal with.
We have been working on an idea to create a standardised laboratory notation system to ensure iGEM protocols could be followed accurately and precisely. One member our team in particular has a little difficulty with English and have developed the notation to ensure procedures in labs will be carried out correctly. We have asked some other iGEM teams to trial the system to see if it work in practice and not just in theory. Other iGEM teams have also asked if they could contribute protocols to the system. It is possible that the system could be developed into an iGEM protocol language programme, but we may be constrained by time and resources.
Members of the team went to Young Synthetic Biologist 2.0 Conference held at the Wellcome Trust by UCL’s SynBioSoc and the UCL 2014 iGEM team. We met loads of other teams from the UK (and Denmark), we presented our work so far, exhibited a poster and attended workshops that gave us valuable insight into human practices, public outreach and ideas about characterising BioBricks. We intend to use the feedback given from other teams to improve and develop our project further, and we thank the hosts for their amazing meet-up.
NEXT STEP
We intend to finish our work in labs as soon as possible since the academic year starts again in Sheffield at the end of September. We are going to assay our keratinase BioBrick with feathers and measure.
@iGEMSheffield
IGEM Sheffield 2014
igem2014@sheffield.ac.ukm
Newsletter n°4
from Sept 1st to Sept 15th 2014
UCSF&UCB
OUR TEAM
We are from San Francisco, California, USA. This is a team with members from both UC Berkeley and UC San Francisco. We are competing in the Overgraduate group.
PROJECT UPDATE
Our goal is to understand how a group of cells uses sense and secrete communication to make a community decision. Therefore, we have been implementing the circuit we designed into yeast cells. Now that we have transformed many of the yeast strains necessary for our project, we are currently analyzing how each strain responds to an external stimulus, both individually and as a general population. Some issues have arisen, such as discovering that some yeast strains did not have the reporter gene RFP, which means that we will have to re-transform those strains. We will also proceed to the next step with the successful strains, which is to co-culture different yeast strains and observe whether their responses to the external stimulus differ from the responses when alone.
We have also been working on iGEM requirements. Our characterized parts have just been shipped to MIT, and you can expect to see our website up sometime soon!
jchsueh@berkeley.edu
Newsletter n°4
from Sept 1st to Sept 15th 2014
TECHNION-ISRAEL
PROJECT UPDATE
WE ARE PLANNING A BIOSENSOR FOR LOW CONCENTRATION SUBSTANCES THAT IS BASED ON A CLEVER GATE CONSTRUCT, COMMUNICATION BETWEEN BACTERIA AND SIGNAL FOCUSING – THIS IS WHERE THE AZOBENZENE COMES INTO PLAY.
Azobenzene is a photo-switchable molecule – when in dark the molecule is in trans isomer but when the molecule is exposed to short wavelengths of light it switches to cis isomer. This switch to cis isomer causes the azobenzene molecules to “stick” to one another. We plan on attaching the azobenzene molecules to the LPS of our E.coli and by that allowing the bacteria to group together and form a visible green aggregate. The transition to cis isomer happens when the molecules are exposed to UV light (which is harmful to DNA), but we have found that substitution of all four ortho positions with methoxy groups in an amidoazobenzene makes photoswitching possible using green light (530-560 nm).
The series of organic reactions are almost done! The next step is to attach the azobenzene to the bacteria and test the aggregate formation under a microscope (you can see updates in our Facebook page).
Faris, our chemist is creating azobenzene molecules, in a series of cuttingedge organic reactions, for a unique application in our biological system. - This is the first time ever that this molecule has been synthesized and it is being done in our lab! Faris was the first person to come up with the method of attaching the bacteria to each other using the azobenzene molecule.
igem14il@gmail.com
Newsletter n°4
from Sept 1st to Sept 15th 2014
PROJECT UPDATE
QUESTIONS TO OTHER TEAMS:
1.RED LIGHT SENSING AND IMAGING SYSTEM DIDN’T WORK AS WE OBSERVED THE TERMINATION OF GROWTH OF BACTERIA AFTER TRANSFORMATION. PARTS WE USED ARE K519039(SENSING RED LIGHT) AND P0451, R0051 AND I3502(PERCEIVE THE RED LIGHT AND EXPRESSION) AND THEY WERE PUT IN TWO PLASMIDS. IS THERE ANY TRICK TO LET IT WORK? WE ARE A LITTLE BIT DESPERATE. 2.HOW TO OPERATE A QUANTIFICATION OF CHROMOPROTEIN? 3.IS THERE ANY BETTER WAY TO GUIDE THE NEW SYSTEM INTO A NEW-USED BACTERIA C.CRESCENTUS？
1.Construction of light sensing-imaging system: Some color light didn’t work as we anticipated thus we are trying to reconstruct the whole system and measurement will be soon operated for test.
2.Construction of RNA based regulatory circuits: The main logical key-gate NOT, which is useful for substitution of subpart of our circuit has been constructed.
3.Testify the function of blue light sensing-imaging system: Blue light sensing-imaging system works very well.
4.Conjugation and electrotransformation test still on trying: The first try on guiding system into Caulobacter crescentus is a really hard work to do. Consequently, we are still working on it.
5.Propagation animation is producing: This time what we wanna to is to produce a stop-motion animation for propagation of our project. For the whole conception is quite hard to understand. Thus with this animation we are producing, audience will get the core idea of our project quickly and briefly.
NEXT STEP
1.Finish main content of Wiki and construction of main page of wiki. 2.Construct the projector and product with industrialization. 3.Further test of RNA regulatory parts.
USTC_China iGEM
stark@mail.ustc.edu.cn
Newsletter n°4
from Sept 1st to Sept 15th 2014
Tsinghua-A
PROJECT UPDATE
Question: 1,The function of our report system is largely limited by the instabilit y of biolog y organisms . The differences between each E.coli colonies lead to the totall y different appearance after the test. Can you provide us with some hints that can improve the robustness of our report system?
2,In the C’ sequence of our TALE protein , there is a Pst1 recognition site which makes it hard to assembl y with othe r iGEM stan dard pa rts. We hope we can change it with the degenerac y of the codons. Howe ver, it is a time-consuming wo rk if we do this by mutation PCR. So is there any better idea to change the endonuclease recognition site?
These two weeks, our work focused on the wet lab work. We assembled our wild TALE sequence with the Golden Gate assembly methods. Though the wild TALE protein coding sequence is more than 2000bp in length which is relative hard to assembly, we conquered this task. At the same time, we constructed an expression carrier for the TALE protein with iGEM standard parts. We designed two carriers for the TALE protein, one with constitutive promoter and one with regulative promoter. In addition to that, our report system, a system aiming at testing the binding ability of the TALE proteins we produced was also finished in the last two weeks.
On the other hand, our human practice group spared no effort to design our logo and wiki. We are also making essential preparation for our trip to the States.
PLAN FOR FUTURE STEPS
1. Test the reliability of our report system.
2. Finish the construction of our express carrier (the regulatory part).
3. Assembly the TALE protein sequence after our optimization.
4. Work in the Human practice section.
iGEM2014_TsinghuaA@126.com
Newsletter n°4
from Sept 1st to Sept 15th 2014
Tianjin University
PROJECT UPDATE
Question: 1. the specific protocol about how to use Congo red to stain curli fibre?
2. we used to consider making the curli fibre ca rry the current at a magnitu de of 10-12 , but we lac k a proper ammeter to detect the current at such a magnitu de. What if we increase the current to a magnitu de of 10-6, woul d the curli fibre (as conducti ve wire) be burnt off? Could somebody have some suggestion on increasing the current?
3. Does somebody have some tips on Gibson assembl y? With many thanks!
It is rather a rough time for our team. On account of some neglects and faults in designing the reverse primer, we had to restart the assembly of the building blocks tragically. So, our project now is faced with much difficulty arranging the time and accomplishing them in time. Since there is no time left to tolerant more unnecessary faults, we are scheduling specific experimental plans in two parallel groups and going to make a contrast so that we can choose a more complete one. In addition, we are trying to find specific protocols in further inducible expression.
PLAN FOR FUTURE STEPS
1.arranging two groups to building the standard blocks in the meantime
2. trying a new assembly method gibson assembly to improve the efficiency.
IGEM Tianjin 2014
tjuigem2014@163.com
Newsletter n°4
from Sept 1st to Sept 15th 2014
Tongji University
PROJECT UPDATE
Our project is about biobleaching and our work is now mainly about producing XynB, ArfB and ManA1.
We have got each expression vector for each protein. And XynB has been expressed successfully.
After we get these three proteins successfully produced, we will detect their activity and decolor efficiency. We will use DNS solution and draw standard curves to do some quantitative determination. These things will be talked about in later issue.
We have used XynB to do some decolor experiment which turned out to be quite effective. We will do further experiments to make the result confirmable.
PLAN FOR FUTURE STEPS
1. We are now planning to do a demonstrative experiment to show audience the using process and result of our products.
2. We are also trying to make some Chinese brush writing paper to arouse people’s interest in paper-making industry and they can focus on the opportunities to make an environmental contribution of our products.
1253225@tongji.edu.cn
Newsletter n°4
from Sept 1st to Sept 15th 2014
NJU-QIBEBT
PROJECT UPDATE
This year, we choose a project to produce fatty acid. We use GC-MS to detect the constituent and the concentration of the fatty acid.
Now, we are in Qingdao Institute of Bioenergy and Bioprocess Technology and we have already got some result.
Here I’m gonna to show you some result.
This is the MC result of the Fatty acids. Those fatty acids were produced by E.coli in M9 culture medium. Peak 11.624 is marker. As you can see, peaks are very clear. And we can get more detail with GS figures.
As you can see, we increase the production of fatty acid with 16/18 carbon.
More details are waiting for us to discovery. We hope to show you later.
In the next 2 weeks, we will get on with our experiments. Then we will do our human practice.
My question is that why some operons such as lac operon and ara operon are not work all the time.
@zhouyu1992nju
Yu Zhou
zhouyu1992nju@outlook.com
Newsletter n°4
from Sept 1st to Sept 15th 2014
NEAU-Harbin
OUR TEAM
Back in 2009, some intelligent students in the college of life science of NEAU learnt about iGEM and they suggested teachers to let them join the competition. Unfortunately, because of many reasons, they failed. However, they left the iGEM community of the college of life science to their juniors. So in 2012, four staffs of iGEM community knew that HIT joined iGEM since 2011. They participated HIT-Harbin and then for the love of iGEM and scientific research of life science, one of them joined the jamboree following the HIT-Harbin. Next they suggested teachers to let them join the competition and they got success! So NEAUHarbin was established. Now the members of NEAU-Harbin are almost from the college of life science of NEAU. But the iGEM community is planning to make the team stronger and better!
PROJECT UPDATE
Now we are still construct our standard expression vectors. We are trying to eliminate NotI restriction enzyme cutting site on our expression vectors. But we have already constructed 3 kinds of non-standard expression vectors with 3 kinds of chromo protein genes. And we transformed them into Agrobacterium and co-culture with A.Niger. But now we don’t see any colorful A.Niger. We deduce the chromo protein parts that iGEM HQ provide is codon optimized for prokaryote, not for mold. So we decide to codon optimize the chromo protein genes for mold and synthesize it in this week.
PLAN FOR FUTURE STEPS
Construct our vectors, and watch the results of the A.Niger’ color. Codon optimization.
igem@neau.edu.cn
Newsletter n°4
from Sept 1st to Sept 15th 2014
XMU_China
PROJECT UPDATE
So far we have accomplished two circuits, which endow the CL-1( E.coli engineered strain ) the ability of the chemotaxis. Then the engineering bacteria could be regulated by IPTG and L-Arabinose; pBAD and pLac promoters control the expression of Lac I and CheZ. Besides, when we use IPTG as an inducer and L-Arabinose as an inhibitor to take control of the chemotaxis of the engineering bacteria, we can get a good characterization in the semisolid plate.
What’more, we start to prepare the content of the wiki and fill in the biological brick.
PLAN FOR FUTURE STEPS
First, we still use the finished circuits to draw graphics. Second, we have found some problems of QS Promoter, which is connected with the project we did last year. So we will make use of this year’s project to improve the project last year. Therefore, we will build two circuits to compare the effects of last year’s project and this year’s project.
@XMUiGEM2014
xmuigem2014@gmail.com
Newsletter n°4
from Sept 1st to Sept 15th 2014
SYSU -China
PROJECT UPDATE
Our project is separated into 4 groups, namely B2H (Bacterial-two-hybrid), mutation, M13, and RNAT (RNA thermo). The work of each group is approximately in accordance with our schedule, and some of the details are stated as following: For B2H group, who aims to assemble a B2H system to activate a reporter gene by interaction of two compensative proteins, a major problem is that a mutated positive control for lower affinity shows, on the contrary, higher expression of reporter GFP. We have been working on this problem for days, and still need an answer.
For RNAT group, who aims to build a RNA thermal switch for system control, we have successfully certified the function of one of three prepared thermal switches, but the other two were somehow disable.
For M13 group and mutation group, a dramatic number of bugs are haunting the project, and thus we need some time to deal with them.
Additionally, this year we participate in the Interlab Study, a subordinate and independent section of Measurement Track. We have already accomplished all cloning work as well as procedure modification, and final statistics and report will be finished in few days.
PLAN FOR FUTURE STEPS
1. All bricks to be submitted must be recorded into normative document, and standard sample of each brick will be prepared in few days.
2. The style of our wiki will be decided at once and first edition of frame work will soon come out.
3. A detailed plan about which individual awards we should focus on should be taken into consideration, in that this is helpful to increase the competitive force of our wiki.
4. The final experiments and data collection will be done according to the plan of each group.
@IGEM SYSU-China
igem.sysu.china@gmail.com
Retrieved from " http://2014.igem.org/Team:Paris_Bettencourt/Newsletter4 "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
