<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Paris_Bettencourt_Project_Eliminate_Smell"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Paris_Bettencourt/Project/Eliminate_Smell">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Paris_Bettencourt/Project/Eliminate_Smell&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Paris_Bettencourt/Project/Eliminate_Smell&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Paris_Bettencourt/Project/Eliminate_Smell&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Paris_Bettencourt/Project/Eliminate_Smell" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Paris Bettencourt/Project/Eliminate Smell</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><DIV class="navmenubkg"><DIV id="navmenubox"><UL id="nav"><LI><A href="https://2014.igem.org/Team:Paris_Bettencourt/Project">Project</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Project/Odor_Library">SmellThe Roses</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Project/TMAU">SomethingFishy</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Project/Foot_Odor">GoodyTwo Shoes</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Project/Eliminate_Smell">Don'tSweat It</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Project/Old_People_Smell">Teen Spirit</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Project/Art">Design Practice </A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Project/Bioinformatics">Bioinformatics</A></LI></UL><LI><A href="https://2014.igem.org/Team:Paris_Bettencourt/Achievement">Achievements</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Results">Results</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Project/Interlab_Study">Interlab Study</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Medal">MedalFulfilment</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Parts">Parts</A></LI><LI><A href="https://2014.igem.org/Team:Paris_Bettencourt/Policy">Policy </A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Safety">Safety</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Consent_Form">Consent Form</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Patenting">Patenting</A></LI><LI><A href="https://2014.igem.org/Team:Paris_Bettencourt/Human_Practice">Practices</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/MOOC">MOOC</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Citizen_Science">CitizenScience</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Facts_Smell">The Senseof Smell</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Facts_Micriobiome">The HumanMicriobiome</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Minder">Minder</A></LI><LI><A href="https://2014.igem.org/Team:Paris_Bettencourt/Collaboration_Main">Collaborations </A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Collaboration">Collaborations </A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/The_Smell_Game">The SmellGame</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Newsletter">Newsletters</A></LI><LI><A href="https://2014.igem.org/Team:Paris_Bettencourt/Notebook_Main">Notebook</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Notebook">Daily Note</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Protocols">Protocols</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Bibliograpy">Bibliography</A></LI><LI><A href="https://2014.igem.org/Team:Paris_Bettencourt/Team_Main">Team </A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Team">Our Team </A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Attributions">Attributions</A></LI><LI><A class="liste" href="https://igem.org/Team.cgi?year=2014&amp;team_name=Paris_Bettencourt">Official TeamProfile </A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Acknowledgements">Acknowledgements</A></LI><DIV id="twnoticon" class="sidelogos"><A class="twitter-timeline" width="250" height="400" data-dnt="true" href="https://twitter.com/iGEM_Paris" data-widget-id="368144501119795201">@iGEM_Paris</A></DIV></DIV></DIV></DIV><DIV id="topheader"><A class="playlist" id="link" href="https://www.youtube.com/watch?v=bxV-OOIamyk" target="_blank">  Gym Class Heroes</A></DIV><DIV id="fond"><TABLE id="headtable"><TBODY><TR><TD><CENTER><B>BACKGROUND</B></CENTER><P class="text1">Sweat is odorless until it is metabolized by the bacteria of the skin. If the enzymes responsible for producing odorant molecules could be removed from the skin microbiome, body odor would be attenuated or eliminated. In this project we explore strategies to identify, target, and remove odor-related genes from a complex population. </P></TD><TD><CENTER><B>AIMS</B></CENTER><P class="text1">This project is subdivided into 3 approaches. First we seek to understand odor production by a single enzyme related to armpit odor. Next we develop a CRISPR-based tool for isolating precise loss-of-function mutants. Finally we use bioinformatics to identify potential gene targets to eliminate with our tool.</P></TD><TD><CENTER><B>ACHIEVEMENTS</B></CENTER><LI> Expressed the odor gene <I>agaA</I> in <I>E. coli</I> (BBa_K1403002).</LI><LI> Designed and used CRISPRs to enrich for loss-of-function mutants 1000x in a mixed population.</LI><LI> Identified potential CRISPR targets as odor genes with variability hotspots in metagenome data.</LI><LI> Formulated a simple skin cream encorporating live bacteria.</LI></TD></TR></TBODY></TABLE></DIV><TABLE id="tablelien"><TBODY><TR><TD><A href="#part1">Genes Affect Odor</A></TD><TD><A href="#part2">A CRISPR/Cas System</A></TD><TD><A href="#part3">The Probiotic Cream</A></TD><TD><A href="#part3">Natural variants of odor genes</A></TD></TR></TBODY></TABLE><DIV id="part1" class="project"><P class="text2"><SPAN class="legende"><B>Figure 1. Enzymes responsible for body odor in the human axilla (<A href="https://2014.igem.org/Team:Paris_Bettencourt/Bibliograpy">Tauch, 2013 <I>et al.</I>, 2001</A>). </B></SPAN><SPAN class="legende"><B>Figure 2. Smell test</B>. 14 people smelled two tubes of <I>E. coli</I> grown to saturation in LB. One culture carried synthetic <I>agaA</I> and other an empty vector control. 13 people out of 14 rated the <I>E. coli</I> carrying <I>agaA</I> as more smelly (pink) than the control(violet). Experiments were double-blind. Significance was confirmed by Chi square test (p-value = 0.001341).</SPAN><SPAN class="legende"><B>Figure 3. Phylogenetic tree produced from 16S sequencing of <I>Corynebacterium</I> from human axillary samples. </B><I>Corynebacterium</I> was cultured on blood agar plates and the 16S region of selected colonies was sequenced. Sequencing results are presented here in a phylogenetic tree format. The tree was made by creating a multiple sequence alignment (MSA) using the CLUSTALW algorithm and then creating a neighbor-joining tree using the MSA. The length and order of the branches is not accurate due to the evolutionary closeness of the species sampled.  </SPAN></P><H6>1. Genes and Bacterial Species Affect Odor</H6><P class="text1"><STRONG style="font-size: 125%;">Introduction</STRONG>

Body odor is a complex combination of odorants, but the major components come from just a few known enzymes (Fig. 1). In the armpit, for example, a major source of &quot;pungent&quot; or &quot;musky&quot; odors is the enzyme AgaA expressed by <I>Corynebacterium striatum</I> (<A href="https://2014.igem.org/Team:Paris_Bettencourt/Bibliograpy">Kligman <I>et al.</I>, 1981</A>). AgaA is an aminocylase that hydrolyzes 3-methyl-2-hexenoyl-glutamine (3M2H-gln) into 3-methyl-2-hexenoic acid (3M2H) and free glutamine. The substrate is secreted in sweat, and the product is a distinctive armpit odor.<STRONG style="font-size: 125%;">Results</STRONG>

To understand the smell created by AgaA, we cloned the gene into <I>E. coli</I> (BBa_K1403002). A noticeable odor was produced by <I>agaA</I>-expressing <I>E. coli</I> grown in selective LB, described as &quot;beer-like&quot; or &quot;cheese-like&quot;. We confirmed this observation with a formal double-blind smell test (Fig. 2). Although the native substrate of the enzyme is not present in LB, the enzyme is known to have a low specificity for the acyl group, and to act on a range of glutamine conjugates (<A href="https://2014.igem.org/Team:Paris_Bettencourt/Bibliograpy">Natsch <I>et al.</I>, 2003</A>). These observations highlight the remarkable affect of single gene on the odor profile of a bacterial population.

We next investigated bacterial species involved in human body odor. We collected sweat samples from our team, from other iGEM teams, and during our public exibition at the <I>Cité des Sciences</I>. Samples were plated on blood agar with phosphomycin, a media selective for <I>Corynebacterium</I>. We identified 15 distinctive species by 16S sequencing (Fig. 4), each with a noticeably different smell.<STRONG style="font-size: 125%;">Methods</STRONG><STRONG style="font-size: 110%;">1. Cloning <I>agaA</I> into <I>E. coli</I></STRONG>
The AgaA enzyme from <I>Corynebacterium</I> (<A href="http://www.ncbi.nlm.nih.gov/nuccore/AF534871.1">Genbank: AF534871.1</A>) was codon optimized using the <A href="http://eu.idtdna.com/CodonOpt">IDT online tool</A> (Integrated DNA Technologies, Belgium) for <I>E. coli</I> K12 and was commercially synthesized by IDT. PCR was performed to obtain the <I>agaA</I> construct using a forward primer that contained the Biobrick prefix and a RBS designed using the <A href="https://salis.psu.edu/software/">Salis online tool</A>; and a reverse primer that contained the BioBrick suffix. The PCR product was purified. 

The <I>agaA</I> construct and pSB1C3 vector were digested using EcoRI and PstI. After, the <I>agaA</I> construct was purified using the PCR Purification Kit . The vector was gel extracted and purified. Both parts were ligated using Ligase T4. 

Competent <I>E. coli</I> Neb Turbo were transformed with the ligation product by heat shock. Then, they were recovered during 2 hours in 200 uL of LB medium at 37ºC and plated in selective plates with LB and chloramphenicol in a concentration of 1 uL/mL. The transformation was confirmed by analytical digestion. <STRONG style="font-size: 110%;">2. Find <I>Corynebacterium</I> in human skin samples.</STRONG>
After collecting human axilary samples, we cultured them in blood agar plates to select for <I>Corynebacterium</I> and performed 16S sequencing. We made a multiple sequence alignment (MSA) using the CLUSTAW algorithm and we created a neighbor-joining tree using the MSA. Finally, we obtained a phylogenetic tree for the <I>Corynebacterium</I> found in our samples (Fig. 3).
 </P></DIV><DIV id="part2" class="project"><P class="text2"><SPAN class="legende"><B>Figure 4. The designed CRISPR/Cas system targets the LacZ gene of a wild type and a mutant. </B>The targeting sequence is 30 nucleotides wide and targets the wild type sequence of <I>lacZ</I>. The mutant strain has a deletion and a substitution at positions 1470 and 1473 respectively, decreasing the specificity of the CRISPR for it.  </SPAN><SPAN class="legende"><B>Figure 5. The designed CRISPR/Cas system is capable of discriminating small genomic differences. </B>The wild type (<I>LacZ+</I>) and the mutant (<I>LacZ-</I>) were transformed with the non-targeting CRISPR (Random CRISPR) and the <I>LacZ</I>-targeting CRISPR (<I>LacZ</I> CRISPR). The non-targeting CRISPR resulted in comparable number of transformed cells among the wild type and the mutant. The <I>LacZ</I>-targeting CRISPR resulted in the killing of most of the wild type, but in the survival of a substantial number of the mutants (40% compared to the control non-targeting CRISPR).  </SPAN><SPAN class="legende"><B>Figure 6. The designed CRISPR/Cas system is capable of isolation a mutant from a mixed population. </B>The wild type and the mutant strains were mixed right before electroporation. Since the wild type produces a blue color and the mutant stays colorless, we measured the final frequency of mutant by blue/white screening. The transformation with the non-targeting CRISPR resulted in a final proportion of the mutant proportional to the initial concentration. However, when transformed with the <I>LacZ</I> targeting CRISPR, the final proportion of mutant was almost at 100%. This property is maintained until a ratio of 1/10<SUP>6</SUP> after which the isolation loses efficacy.  </SPAN><SPAN class="legende"><B>Figure 9. We took microbiome samples of 12 volunteers, performed DNA extraction and PCR amplified for <I>ldh</I>, <I>L-ldh</I>, <I>D-ldh</I> and <I>ackA</I>. </B>The samples from 5 volunteers were positive for the PCR and we Sanger sequenced them. </SPAN><SPAN class="legende"><B>Figure 10. Hotspots found for enzymes <I>D-Ldh</I> in human microbiome axillary samples.</B></SPAN></P><H6>2. A CRISPR/Cas System to Enrich for Natural Mutants</H6><P class="text1"><STRONG style="font-size: 125%;">Introduction</STRONG>

Removing genes (like <I>agaA</I>) from the bacteria that express them (like <I>Corynebacterium</I>), should attenuate their associated odor. Classical genetic tools can produce gene deletions. Odor-negative deletion mutants, applied to the skin in large numbers, could replace a wild-type strain by the principle of competitive exclusion. However, this approach faces two difficulties: 
1) Gene deletions may reduce fitness, rendering the mutant unable to compete with the WT for a niche.
2)Genetically Modified Organisms (GMOs) face special regulations and limited public acceptance.

We propose an alternative strategy to isolate naturally occurring loss-of-function mutants from a complex microbiome. Natural mutants, if they are isolated from the microbiome itself, are likely to retain high fitness. Public acceptance is higher and biosafety concerns are reduced for non-GMO microbes. 

The CRISPR-Cas system has been shown to target DNA sequences for cutting with single-base precision, providing means of targeting genome editing or killing specific pathogens (<A href="https://2014.igem.org/Team:Paris_Bettencourt/Bibliograpy">Jiang <I>et al,</I>, 2013; Gomaa <I>et al.</I>, 2014</A>). Here we present a CRISPR-based protocol for isolating pre-existing mutants from a mixed population. Mutants that mismatch a CRISPR target sequence will survive transformation with a CRISPR-expressing plasmid and therefore be enriched in the post-transformation population. In principle, CRISPR-based mutant enrichment does not alter the existing mutant genome and the isolated organisms do not meet the conventional definition of GMOs. <STRONG style="font-size: 125%;">Results</STRONG>

As a proof-of-principle, we designed a CRISPR system on a single plasmid with a tracrRNA targeting lacZ. The targeting sequence was designed to match the wild-type sequence, but not a loss-of-function mutant sequence with two point mutations (Fig. 4).

Our CRISPR system was capable of sequence-specific killing (Fig. 5). Transformation with the <I>lacZ</I>-targeting CRISPR significantly killed less than 98% of wild-type bacteria compared to a control CRISPR with a randomized tracrRNA. In contrast, about 40% of mutant bacteria survived transformation relative to control. This result showed that our construct could selectively kill wild-type bacteria.

We next used our construct to enrich for mutant bacteria in a mixed population (Fig. 6). Wild-type and <I>lacZ</I>- strains were mixed in measured proportions. Blue/white screening for <I>lacZ</I> activity was applied following transformation to measure the frequency of <I>lacZ</I>- mutants post-transformation. Even when <I>lacZ</I>- mutants represented only 1:10<SUP>6</SUP> of the initial population, they represented the large majority of the population post-transformation. In contrast, transformation with randomly-targeted CRISPR produced a linear proportionality between <I>lacZ</I>- frequency before and after transformation.

CRISPR-mediated cleavage may select for mutants, but it can also induce mutagenesis. To distinguish between these two possibilities we additionally labeled lacZ mutants with a plasmid expressing YFP. Wild-type cells were similarly labeled with CFP. This labeling allowed us to distinguish a CRISPR-selected mutant (<I>lacZ-</I>, YFP+) from a CRISPR-induced mutant (<I>lacZ</I>-, CFP+). We found CRISPR-induced mutagenesis to be rare, representing less than 10% of the total isolated mutants in most cases (Fig. 7). 

To use this system in strains beyond <I>E. coli</I>, we ported the entire CRISPR/Cas system into the broad host-range vector pSEVA351. This new vector is also functional in <I>E. coli</I> (Fig. 8) but carries a viral replication origin that functions in many species. This vector should serve as a tool to enable CRISPR-mediated mutant enrichment in many species of the skin microbiome.<STRONG style="font-size: 125%;">Methods</STRONG>



In order use a cloning strain which wouldn't be killed by the CRISPR, the strain BL21-AI has been chosen for all the clonings. However, prior to its use in cloning purposes, its chloramphenicol resistance has been flipped out by the use of the thermosensitive plasmid pCP20.
The pCas9 plasmid was the CRISPR vector used. Its targeting sequence has been changed by the use of the enzyme Eco31I (BsaI) in regular cloning technique.

Regarding the mixing of different strains for the mutant enrichment purpose, after the third washing, the strains were diluted to reach the same OD600 and then mixed in different proportion. They were then washed one more time and resuspended in 1mL of water before electroporation.<SPAN class="legende"><B>Figure 7. CRISPR induced mutagenesis is rare during mutant isolation. </B>After labeling the wild type with a plasmid expressing CFP and the mutant with a plasmid expressing YFP, we transformed mix of the two strain at a ratio of 1:10<SUP>2</SUP>. The blue/white screening was then compared to the fluorescence. Colorless colonies expressing CFP fluorescence were representing potential induced mutants. These potential induced mutants represented about 8% of the colorless colonies. However it was found that their colorless phenotype was to be imputed to the imperfect efficiency of the blue/white screening method. This was confirmed by re-streaking on X-gal and sequencing. The colonies were blue after re-streaking, and presented the expected sequence of a wild type.  </SPAN><SPAN class="legende"><B>Figure 8. Broad-host range CRISPR has a similar efficiency. </B>After cloning the whole CRISPR/Cas system in the vector pSEVA351, we compared its discrimination efficiency to our previous CRISPR/Cas system. It appears the even if the transformation efficiency seems to be in general higher with the new vector, discrimination properties are conserved. Indeed there is about 45% of survival of the mutant when compared to the control non targeting CRISPR.  </SPAN></P></DIV><DIV id="part3" class="project"><H6>3. Sequence Identification of Natural Variants of Odor Genes</H6><P class="text1">
To identify potential natural sequence variants that might provide CRISPR targets, we amplified and sequenced odor-related genes from human sweat samples (Fig. 9). We focused on enzymes producing odorant fermentation products in <I>Staphylococcus</I> species like acetate kinase (<I>ackA</I>) and lactate dehydrogenase (<I>ldh</I>) (<A href="https://2014.igem.org/Team:Paris_Bettencourt/Bibliograpy">Tauch, 2013 <I>et al.</I>, 2001</A>).
We extracted, amplified and sequenced 4 genes from 12 individual sweat samples. A range of synonymous and nonsynonymous mutations were detected (Fig. 10). Each of these mutations represents a CRISPR-selectable target, and some of them may have functional consequences for an individual odor profile.
</P><P class="text2"><SPAN class="legende"><B>Figure 12: Cream with <I>Corynebacterium striatum</I>.</B></SPAN></P><H6>4.Probiotic Cream: a Cure for Body Odor</H6><P class="text1"> 

We formulated a basic skin cream to package and deliver live bacteria as a cosmetic. The cream is a water-in-oil emulsion using jojoba oil as a base, beeswax as an emulsifier and soy milk as a bacterial culture medium.
 
The soy milk media forms emulsified droplets 100 μM in scale, in which bacteria are protected from dessication. We tested the cream with RFP-expressing <I>E. coli</I> (Video of Cream with fluorescent <I>E. coli</I>) and with wild-type <I>C. striatum</I> (Fig. 12).
</P><B>Video of Cream with fluorescent <I>E. coli</I>.</B></DIV><DIV id="mfooter"><DIV class="footbox"><B>Centre for Research and Interdisciplinarity (CRI)</B>
    Faculty of Medicine Cochin Port-Royal, South wing, 2nd floor
    Paris Descartes University
    24, rue du Faubourg Saint Jacques
    75014 Paris, France +33 1 44 41 25 22/25<A href="mailto:paris-bettencourt-igem@googlegroups.com">paris-bettencourt-igem@googlegroups.com</A></DIV><CENTER>
    Copyright (c) 2014 igem.org. All rights reserved.
  </CENTER></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Paris_Bettencourt/Project/Eliminate_Smell">http://2014.igem.org/Team:Paris_Bettencourt/Project/Eliminate_Smell</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Paris_Bettencourt/Project/Eliminate_Smell" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Paris_Bettencourt/Project/Eliminate_Smell" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Paris_Bettencourt/Project/Eliminate_Smell&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Paris_Bettencourt/Project/Eliminate_Smell&amp;oldid=403732" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>