"
Page
Discussion
View source
History
teams
Log in
 
Team:Paris Bettencourt/Notebook/Eliminate Smell
From 2014.igem.org
Project
SmellThe Roses
SomethingFishy
GoodyTwo Shoes
Don'tSweat It
Teen Spirit
Design Practice
Bioinformatics
Achievements
Results
Interlab Study
MedalFulfilment
Parts
Policy
Safety
Consent Form
Patenting
Practices
MOOC
CitizenScience
The Senseof Smell
The HumanMicriobiome
Minder
Collaborations
Collaborations
The SmellGame
Newsletters
Notebook
Daily Note
Protocols
Bibliography
Team
Our Team
Attributions
Official TeamProfile
Acknowledgements
@iGEM_Paris
Smell The Roses
Something Fishy
Don't Sweat It
Goody Two Shoes
Teen Spirit
Interlab Study
June
July
August
September
October
Notebook
June
June 7th
Goal: Design oligos to change the targetting sequence of a CRISPR plasmid. This is performed in order to target the LacZ gene of E.coli
Procedure:
This is performed on Geneious:
Open the LacZ gene sequence on Geneious
Find a PAM sequence in the gene (NGG)
Copy the 30 nucleotide prior to the PAM region, and paste it as a new sequence. Convert it to oligo
Copy it and paste in another new sequence (you now have two times the 30 nucelotides). Convert it to oligo, and Reverse Complement one of the two sequences
Add to the sequence which isn't reverse complemented in the 5' end : AAAC
Add to the sequence which is reverse complemented in the 5' end : AAAAC
The two sequences should look this way :
The name of these two oligos are oPB.008 and oPB.009
A second set of oligos was designed to not be targeting the E.coli genome
A 30 nucleotides sequence was generated randomly and then blasted against bacteria genomes to make sure it was not complementary to any part of the E.coli genome
June 11th
﻿
Goal: Design plasmids that express agaA construct
Procedure:
Using Geneious, we first created the agaA construct:
Promoter BBa_J23108 from the Anderson's promoter collection
RBS from the RBS Calculator of Salis lab specific for E coli
BioBrick Prefix + Scar from the iGEM Parts webpage
agaA sequence codon optimized for E coli 12 (IDT online tool). We checked with Generious that there were no restriction sites for EcoRI, SpeI, ZbaI and PstI.
Histidine tag: 6 Histidines in C-terminal were added
Stop codon
BioBrick scar + BioBrick suffix from the iGEM Parts webpage
To amplify this construct, we created two oligos:
Name
Sequence (5'->3')
Notes
Binding position
oPB.001
TATAGAATTCGCGGCCGCTTCTAGAGTGACAGCTAGCTCAGTCCTAGG
agaA forward for BioBrick vector
1->23
oPB.002-BAD PRIMER
GAAGCATCATCACCATCACCACTGATACTAGTAGCGGCCGCTGCAGTTA
agaA REVERSE for BioBrick vector- NOT REVERSED
1272->1296
oPB.005
TTAACTGCAGCGGCCGCTACTAGTATCAGTGGTGATGGTGATGATGCTTC
agaA reverse for BioBrick vector
1272->1296
* We noticed the 16/06/2014 that oPB.002 was NOT reversed and we designed oPB.005
Tip: When creating oligos: we add 4 nt at the beginning and the end composed by AT, to make sure the enzyme will bind properly.
We chose two backbones for the construct:
pSB1C3 from the iGEM Parts to create a BioBrick.
pEC-XC99E, a E coli-C glutamicum shuttle plasmid
We designed two plasmids:
1. pPB.001: Biobrick submission of agaA expression construct
1) Plasmid pSB1C3: High copy BioBrick assembly plasmid. Constitutive expression. To use in E coli
2) agaA construct
2.pPB.002:agaA expression construct. Shuttle vector E coli-C glutamicum
1) Plasmid pEC-XC99E modified: Ptrc (IPTG inducible promoter) and lacI will be removed by PCR. Primers will be oPB.003 -that includes a PstI site- and     oPB.004 -that includes a XbaI site.
oPB.003
5'-CTGCAGATGCAAGCTTGGCTGTTTTGGCG-3'
PstI site + agaA forward for pEC-XC99E
oPB.004
5'-TCTAGACACCACCCTGAATTGACTCTCTTC-3'
XbaI site + aga reverse for pEC-XC99E
2) agaA construct
June 13th
Goal: Since most cloning strains contain the LacZ region that is targeted by the CRISPR/Cas system. The goal is to eliminate the chloramphenicol resistance from the strain BL21-AI (in order to have a proper cloning strain that will be used for the cloning of the CRISPR/Cas system)
Procedure:
The strain BL21-AI was transformed by regular Heat Shock protocol (with 42°C for 30 seconds) with the plasmid pCP20. The transformations were plated on ampicilin plates at 30°C for 24h. Colonies were picked and re-streaked on chloramphenicol plates and on ampicilin plates at 30°C for 24h. After confirming that there were no colonies on the chloramphenicol plates ; colonies from the ampicilin plates were picked and re-streaked on LB plates at 37°C for 24h. Colonies were picked and re-streaked on Chloramphenicol, Ampicilin and LB at 37°C for 24h.
Results:
After the last plating :
There was no colonies on the chloremphenicol plates suggesting that the original resistance from the strain has been eliminated
There was no colonies on the ampicilin plates suggesting that the thermosensitive plasmid has successfully been deactivated
There was colonies on the LB plates that could be used to make fresh glycerol stocks
June 16th
﻿
Goal: Design new reverse oligo for Biobrick construction
Procedure:
We noticed oPB.002 was NOT reverse and therefore could not be used. We created oPB.005 instead:
oPB.005
5'-TTAACTGCAGCGGCCGCTACTAGTATCAGTGGTGATGGTGATGATGCTTC-3'
agaA reverse for BioBrick vector
1272->1296
June 20th
Goal: Change the targeting sequence of the CRISPR/Cas system
Procedure:
1) Preparation of the vector
The following cells were taken from glycerol stock:
Strain (old name)
Carrying the plasmid (old name)
Resistance
New name
sSP004 (DH5alpha)
pM.001 - pCas9
Chl
sPB.008 carrying pPB.008
The cells were streaked on LB plates containing the proper antibiotic (Chl or Kan)
The cells were then incubated at 37°C for 24h
Single colonies were put in tubes containing 5mL of LB and 5uL of antibiotics
Tubes now containing liquid culture were incubated at 37°C with agitation at 200rpm overnight
New glycerol stocks were made using 500uL from liquid culture, mixed with 500ul of 50% glycerol
Cells went through regular miniprep protocol using Thermofisher kit
The pCas9 from miniprep was digested as follow :
3 uL of pCas9
2 uL of BsaI
5 uL of FastDigest Buffer
1 uL of 100X BSA
84 uL of dH20
Incubated 1h @ 37°C
The digested product was PCR purified through regular PCR purification protocol using Qiagen kit
2) Preparation of the oligos
20uL of the oligos from IDT were diluted in 180uL of NF water, as a working stock
The oligos were then phosphorylated as follow :
1 uL of oligo I
1 uL of oligo II
5 uL of 10X T4 Ligase Buffer
1 uL of T4 PNK
42 uL of dH20
Incubated 1h @ 37°C
The oligos were annealed, by adding 2.5 uL of 1M NaCl, and incubating 5min @ 95°C (slow cool down). They were then diluted 10 times with dH20
3) Ligation was performed as follows
2 uL of digested vector
2 uL of annealed oligos
4 uL of 10X T4 Ligase Buffer
2 uL of T4 Ligase
30 uL of dH20
Incubated 1h @ 37°C
Overnight at room temperature
4) Transformation was performed on the fliped-out BL21-AI strain using regular Heat Shock protocol
Results:
More than a 100 colonies were observed after 24h at 37°C.
June 23rd
﻿
Goal: PCR agaA gBlock with oPB.001 and oPB.005
Procedure:
We are going to PCR using the DreamTaq polymerase protocol the agaA gBlock with the forward primer oPB.001 and the reverse primer oPB.005. These two oligos include the BioBrick prefix and suffix. The aim is to add the prefix and suffix to the agaA sequence and amplify it.
We will set the parameters in the machine and run the reaction overnight.
June 24th
﻿
Goal: PCR purification of the gBlock
Procedure:
We will purify the PCR product -agaA gBlock that contains prefix-agaA-suffix- obtained on June 23rd following the QIAquick PCR purification kit standard protocol .
We will run a 1% agarose gel with the purified DNA following the standard protocol . Gels were made by Ihab and Antonio the 23rd of June following the same protocol.
Results
After running the gel, we exposed it in UV light. We could see the Gene Ruler, but there was no band in our sample.
We think that PCR did not work. Things to change:
Use fusion polymerase, as it is more reliable than the DreamTaq we used. DreamTaq is more for colony PCR.
Final concentration of 200 uL, to have more DNA
When PCR does not work we look at:
Annealing temperature: we start typically at 52ºC and can go down to 50ºC.
DMSO concentration: typically 3%. Avoids missmatching
Extension time, according to the manufacturer
Goal:PCR agaA gBlock with oPB.001 and oPB.005
Procedure:
We are going to PCR using the Fusion Polymerase for the agaA gBlock with the forward primer oPB.001 and the reverse primer oPB.005. These two oligos include the BioBrick prefix and suffix. The aim is to add the prefix and suffix to the agaA sequence and amplify it.
June 25th
Goal: Make sure the cloning was successfull by performing analytical digestion and sequencing
Procedure:
Liquid cultures were made out of colonies from the transformation
After 24h at 37°C, regular Miniprep protocol was followed
1) Analytical digestion was performed as follows:
4uL 10X Green Buffer
2uL of EcoRI
2uL of XbaI
6uL pCas9
26uL of dH20
Incubated for 30 min at 37°C in the thermocycler
2) A primer was designed 200 nucleotides upstream the region were the sequenced was changed
The miniprep plasmid and the primer were sent to a sequencing company
Results:
After gel running, it appears that the analytical digestion produced the right band sizes. Moreover, the sequencing results were matching the expected sequence.
Goal: PCR purification of the gBlock
Procedure:
We will purify the PCR product -agaA gBlock that contains prefix-agaA-suffix- obtained on June 24th following the QIAquick PCR purification kit standard protocol .
We will run a 1% agarose gel with the purified DNA following the standard protocol . Gels were made by Ihab and Antonio the 23rd of June following the same protocol.
Results:
- 1st raw: Gene Ruler 100 kb + 1uL Loading Dye
- 2nd and 3rd raw: purified PCR products + 1 uL of Loading Dye per 5 uL of sample. We saw the samples did not sink properly in the hole,     so we tried using more Loading Dye (raw 2 is tube A, raw 3 is tube B)
- 4th and 5th raw: 2 uL of Loading Dye for 5uL of sample (raw 4 is tube A, raw B is tube B)
We can see our construct around the 1300 bp line, so we know is the gBlock.
Me measured the DNA in the samples using the Nanodrop.
- Tube A: 100 ng/uL
- Tube B: 55 ng/uL. This is a little low. It is an unexpected result considering that we pepared both tubes at the same time and following the same     protocol.
June 26th
﻿
Goal:  Obtain agaA Biobrick (agaA gBlock digestion + ligation)
Procedure:
We will digest the purified agaA gBlock following the Thermo Fast Digestion Protocol for enzimes EcoRI and PstI. Then we will ligate it with the linearized vector pSB1C3. We want to obtain agaA Biobrick and transform E coli.
Digestion
1) We will use tube A (100 ng/uL). We added 20 uL of water + 5 uL Buffer + 20 uL DNA + 2.5 uL EcoRI + 2.5 uL PstI. We digested longer than indicated,     around 40 minutes, as we have used these enzymes before and work better this way.
2) Then we put it at 37ºC at the Isotemp for 5 min.
3) We will repeat for PSB1C3
These tubes were called Digested gBlock and Digested PSB1C3
Purification
4) We will purify both tubes using the QIAquick kit .
These tubes were called Dig + Pur gBlock and Dig + Pur PSB1C3
Ligation
﻿Following the Thermo T4 DNA Ligase protocol . Instead of leaving the tube at 22ºC for 10 min, we left it at room temperature for 45 min.
5) We will make a ligation. We will make a 1:3 proportion for the gBlock:vector. We added 3,3 uL of the linearized vector [25 ng/mL] and     2,5 uL of the insert DNA [100 ng/mL].
Purification
6) We will purify the product using the QIAquick kit .
This tube was called pPB.001
Goal: Transform E coli with agaA Biobrick
Procedure:
We will transform competent E coli that were previously made by Jake following his own protocol. We will use agaA Biobrick .
We will follow this Heat Shock protocol .
We will plate them in LBA+Cm. We will add 250 uL of Cm for 250 mL of LBA.
Results:
No colonies grew after 12h. We will leave them in the incubator some more time and repeat the Heat Shock in the meanwhile
June 27th
﻿
Goal: E coli transformation with agaA Biobrick
Procedure:
Because the plates cultured on June 26th have no colonies after 12h, we have decided to repeat the heat shock transformation. The mistake might be that we are using Cloroamphenicol, and we did not recover them for enough time. We recovered them during 50 min in the thermocycler.
Today we have repeated the same Heat Shock protocol , but we have recovered them during 2h in the incubator. The cells used were a new stock from Jake's competent E coli. The plasmid used is agaA Biobrick .
Then we will plate them in LBA+Cm
Results:
We still have no colonies in the plates.
What might not work:
- Enzimes (EcoRI and PstI) work, as Ihab and Antonio have tested them. However I am cutting a linearized plasmid, so if their efficiency is not very         high, they might not work for me.
- Ligase.
- Transformation. I doubt it
- Primers are not well designed
June 29th
﻿
Goal: Start culture to extract plasmid
Procedure:
We will start a culture to extract the plasmid PSB6A1 (Matt's stock, 2013). The aim is to cut a circularized plasmid with EcoRI and PstI and check             their efficiency.
We cultured the cells in LBA+Amp at 37ºC
June 30th
﻿
Goal: Miniprep to extract a plasmid (unfinished)
Procedure:
We started a culture on June 29th. We will make a mini prep following the Mini Prep protocol. The aim is to extract the plasmid PSB6A1 to cut it with EcoRI and PstI and see the efficiency of the cut.
Results:
We have found out that Antonio and Ihab have already tested PstI and EcoRI. They seem to work quite specifically .
Also, they haven't managed to construct the plasmid, or do the Golden Gate assembly . We believe this is due to the Ligase and/or the Ligation buffer.
Therefore, we will use a new enzyme and buffer, ligate and transform.
Most enzymes from last year do not seem to work. We shoud not use them and wait for the new ones.
July
July 1st
Goal: PCR purification of the gBlock
Procedure:
July 1st
** pEC-XC99E plasmids (Cm) from the Beilfield iGEM team arrived today. We stored them at 37ºC and Pierre-Luc will make a miniprep tomorrow **
Goal: Obtain agaA Biobrick
Procedure:
We will use the digested agaA gBlock and PSB1C3 obtained on June 26th .
Ligation
We will use a new Ligase from Promega (Jake's stock) for this experiment, as the former one (from iGEM's 2013 stock, Thermo) did not         seem to work.
We will follow the Promega T4 DNA Ligase protocol. It is recommended to leave the tubes at room temperature for 3h but instead we did it for 2h.
We will make a 1:3 proportion for the gBlock:vector. We added 3,3 uL of the linearized vector [25 ng/mL] and 2,5 uL of the insert DNA         [100 ng/mL].
Purification
We will purify the product using the QIAquick kit .
Tip: It is better to purify with water, as the Ligase might be sensitive to the Elution Buffer (EB), that has a lot of salts on it.             Last time (June 26th), we did use the EB Buffer.
This tube was called pPB.001
Goal: Transform E coli with agaA Biobrick
Procedure:
We will transform competent NEB turbo E coli made following the standard protocol . In step 5, we did not incubate them for 4 hours, but only until the density was optimal (around 30-45 min).
We will use agaA Biobrick .
We will follow this Heat Shock protocol .
Tip (From Matt): At the last step when we recover the cells, plate most of the 200 uL and leave the rest recovering overnight. Then plate them the next             day.
We will plate them in LBA+Cm. We will add 250 uL of Cm for 250 mL of LBA.
Results:
We are still not able to transform the cells. We went back to check the primers, and the oPB.001 that we have in the database and the one we commanded are     not the same.
We will re-do this process with the good oPB.001 when it arrives. At least now we have learnt all the techniques. Always re-check your plasmids, many     times.
July 3rd: Study the Strains
Goal: Make sure the studied strains EC307(+) and EC307(-) that were given by a collegue scientist have the right sequence. This was performed by colony PCR and sequencing
Procedure
Primers were designed with Geneious, 200 nucleotides upstream and dowstream of the targeted sequence of LacZ.
4 colony PCRs were performed : 2 on EC307(+) colonies and 2 on EC307(-) colonies.
The PCR products were sent for sequencing.
Results
The sequencing results matched the expected sequences.
July 4th
﻿
Goal: Design primers for smelling genes
Procedure:
Some genes responsible of odor belong to alternative fermentative pathway like: ackA, ldH, l-ldH, d-ldH We want to do different kind of sequencing for these genes. It's wy we want to design primers in order to get 500n pcr products.
July 2nd
Goal: Miniprep to extract pSEVA351 + Glycerol stock
Procedure:
We recieved a tube that contains pSEVA351 (sent by the CSIC ).We     cultured it the night before, and extract a colony. We put a mixture of this colony + 5ml of LB medium in 2 different tubes.
We will let it grow overnight at 37ºC.
- Tube 1: miniprep following the standard protocol. This tube will be pSEVA351 and it is in my own stock.
- Tube 2 : glycerol stock: 250 uL of glycerol 60% + 750 uL of the culture. We will have both a -20ºC and -80ºC stock.
This is the vector we will use to construct agaA in pSEVA .
July 4th
﻿
Goal: Design primers for smelling genes
Procedure:
In addition to agaA, some genes responsible of odor belong to alternative fermentative pathway like: ackA, ldH, l-ldH, d-ldH We want to do different kind of sequencing for these genes. It's wy we want to design primers in order to get 500n pcr products.
Name
Sequence (5'->3')
Notes
Strain
/TR>
oPB.046
AGATTGTAGATAGTCTCGAGTCCTCC
agaA forward for primer 1
Corynebacterium Striatum
oPB.047
GTCTTCGTTGAACACGTGCTGGG
agaA reverse for primer 1
Corynebacterium Striatum
oPB.048
CAAGTGGTTCCACCAGCACCC
agaA forward primer 2
Corynebacterium Striatum
oPB.049
ACGCCGCCGTTAGAAACCGG
agaA reverse for primer 2
Corynebacterium Striatum
oPB.066
GTAAATGCTGGTAGCTCATCTTTAAAGTTCC
ackA forward
Staphylococcus epidermidis
oPB.067
CGTATGCCATAGTCTTCATAGTAATGATATGG
ackA reverse
Staphylococcus epidermidis
oPB.068
AGAAGGACATGGCATTACGTTGGG
d-ldH forward
Staphylococcus epidermidis
oPB.069
AGGATATGCATCATAACCAACTACTCTCG
d-ldH reverse
Staphylococcus epidermidis
oPB.070
GATGGTTTCTTCTTAATTGCTGCAAACCC
l-ldH forward
Staphylococcus epidermidis
oPB.071
CCATTTTGATTGATAAGTGTTGGTAAGCC
l-ldH reverse
Staphylococcus epidermidis
oPB.036
AGTGTCACCATGTTCACCAA
ldH forward
Staphylococcus aureus
oPB.037
CAGTAGGTTCAAGCTACGCA
ldH reverse
Staphylococcus aureus
July 7th
1) odor exctraction
Protocol:
Samples: Jalke/Henry/Alexandre cotton pads (of the 01/07)(see sweat collection)
(1)  Cut a small piece of sample from Alex (A), put it in an eppendorf tube an add 600ul of PBS.
Vortex during 60 secondes.
Spin 1 minutes at 1000rpm (to put the cotton at the bottom).
Take supernatant with a 1ml syringe
Filter it (0.2ul)
FAILED: Not enought liquid to be used with a 0.20ul filter we've got
(2) Cut a small piece of sample from Alex (A), put it in an eppendorf tube an add 4000ul of water.
Vortex during 60 secondes.
Spin 1 minutes at 1000rpm (to put the cotton at the bottom).
Take supernatant with a 10ml syringe
Filter it (0.2ul)
==> Smell it: The liquid has a very subtil smell (compare with the patch which have a very strong smell) Maybe the patche was too small.
(3)Cut a big piece of sample from Alex (1/4) (A), put it in an eppendorf tube an add 4000ul of water.
Vortex during 60 secondes.
Spin 1 minutes at 1000rpm (to put the cotton at the bottom).
Take supernatant with a 10ml syringe
Filter it (0.2ul)
==> Smell it: The liquid has a very subtil smell (compare with the patch which have a very strong smell). It is very slightly stronger than the (2) but it is not enought.
Results: Sweat filtration doesn't seems to be efficient enough to capture the smell.
July 7th: Crispr Works
Goal: Determine is the designed CRISPR/Cas systems produce the expected results regarding bacteria survival (both of the strains should survive the non targeting CRISPR, and the mutant strain should survive to the LacZ targeting CRISPR more than the wild type).
Procedure
The strains were transformed by regular heat shock protocol (with 30 seconds at 42°C), either by the non targeting CRISPR or by the LacZ targeting CRISPR. The transformations products were plated on chloramphenicol plates at 37°C for 24h.
Results
Here we report the number of colonies of counting
Non targeting CRISPR
LacZ targeting CRISPR
Wild Type
145
3
Mutant
132
44
July 8th
Goal: Capture the smell (volatiles compounds)﻿ in a patch. This has to be sterile, the odour has to be strong and that can stay for 5 days at least.
Protocol:  Samples: Jalke/Henry/Alexandre cotton pads (of the 01/07)(see sweat collection) We used a protocol from: "Regulation of ovulation by human pheromones" and "Olfactory inﬂuences on the human menstrual cycle" Cut a medium piece form patch (from sample A). Put 500ul of ethanol (70%) and vortex (60 secondes) put immediatly the tube at -20°c. Wait for it to be frozen (2h) Take it out of freezer, wait 2-3 mins. Then all the ethanol should be evaporated, rub the piece on a new clean pad. Wait until the ethanol smell dissapear. Smell it Let the new cotton pad overnight in a box, smell it Results The cotton pad is smelling like sweat, the smell is quite strong (7 people smelled it and recognized a strong sweat odour) but there is also a small smell of ethanol. The longer you wait, smaller is the ethanol smell. The smell completely disappear overnight.
July 10th
Goal: Capture the smell (volatiles compounds)﻿ in a patch. This has to be sterile, the odour has to be strong and that can stay for 5 days at least. Use a solvent exctraction technic found in litterature.
Protocol:  Samples: Jalke/Henry/Alexandre cotton pads (of the 01/07)(see sweat collection) We used a protocol from: "Regulation of ovulation by human pheromones" and "Olfactory inﬂuences on the human menstrual cycle" Cut a medium piece form patch (from sample 1/2/3(3 is a whole pad)). Put 3ml of ethanol (70%) and vortex (60 secondes) put immediatly the tube at -20°c. (I've increased the amount of ethanol to be sure that the pad is sterilised)  I had a control: PAD frozen without ethanol. Wait for it to be frozen (2h) Take it out of freezer, wait 2-3 mins. (1)Then all the ethanol should be evaporated, rub the piece on a new clean pad. Wait until the ethanol smell dissapear. Smell it Let the new cotton pad overnight in a box, smell it (2)When ethanol is evaporated, put the pad in a close falcon tube (3)Then all the ethanol should be evaporated, rub the piece on a new clean pad. Wait until the ethanol smell dissapear. Smell it Let the new cotton pad overnight in a box, smell it
Results houldThe cotton pad smell too much like ethanol, we should find a perfect value for the amount of ethanol putted. Then the solvent exctraction look complicated, because it appear that the smell is going away quite fast, also it is not very good that we have to keep the pad, because when people will smell it, they will directly recognize a cotton pad.
July 10th: Crispr efficiency
Goal
In order assess the discrimination properties of the designed CRISPR/Cas system, the strains were transformed by electroporation (to have more colonies and therefore a more trustworthy counting of the colonies) in 5 replicates.
Procedure
The strains were washed with sterile water and resuspended in 1 mL after the final washing.
The electroporation was performed following the EC2 settings, 200uL were transformed with 2uL of dialysed miniprep.
2h of recover followed the transformation.
Cells were plated on chloramphenicol plates for 24h at 37°C.
Results
Here are reported the number of colonies after counting:
WT with LacZ Crispr
Mutant with LacZ Crispr
WT with Random Crispr
Mutant with Random Crispr
Replicate n°1
5
742
1845
1745
Replicate n°2
40
543
2398
2354
Replicate n°3
12
984
1549
1890
Replicate n°4
2
659
1740
1754
Replicate n°5
26
978
2176
1956
July 11th
Goal:  PCR of agaA gBlock with oPB.001 and oPB.005
Procedure:
We are going to PCR using the
Fusion Polymerase
for the agaA gBlock with the forward primer
oPB.001
(the good one this time!) and the reverse primer
oPB.005.
These two oligos include the BioBrick prefix and suffix. The aim is to add the prefix and suffix to the agaA sequence and amplify it.
July 12th and 13th
Goal: PCR purification of the gBlock
Procedure:
We will purify the PCR product -agaA gBlock that contains prefix-agaA-suffix- obtained on July 11th following the QIAquick PCR purification kit standard protocol .
We will run a 1% agarose gel at 50V with the purified DNA following the standard protocol .
The gel was put in a solution of Ethidium Bromide for 10 minutes.
Results:
-
1st raw:
Ready-to-use Gene Ruler (includes Loading Dye) 100 kb (5 uL)*
- 2nd raw: 5 uL purified PCR products + 1 uL of Loading Dye per 5 uL of sample (PCR product obtained the 11th of July and purified the         12th of July).
This is strange. I am confident that my PCR product is fine. I will repeat it with a different Gene
-
1st raw:
Ready-to-use Gene Ruler (includes Loading Dye) 100 kb (5 uL)*
- 2nd raw: 4uL Gene Ruler + 1 uL Loading Dye 100 kb*
- 3rd raw: 5 uL purified PCR products + 1 uL of Loading Dye per 5 uL of sample (PCR product obtained the 11th of July and purified the     12th of July).
*We used these two because Ready-to-Use seems not to work fine.
We see that it was the 'Ready-to-Use Gene Ruler' that did not work properly. Our PCR worked fine and our construct is about 300 bp. Even if                 this is not at all a good image, as we are only trying to check if the PCR runned properly I will not repeat it. We saw in the other gel that                 there is a clear band. In this gel we can see that this band is 300 bp.
Goal:Obtain agaA Biobrick
Procedure:
We will digest the purified agaA gBlock following the Thermo Fast Digestion Protocol for enzimes EcoRI and PstI. Then we will ligate it with the linearized vector pSB1C3. We want to obtain agaA Biobrick and transform E coli.
We measured the purified aga gBlock in the nanodrop, the concentration was 50 ng/mL. This is low. I do not know if it is low because I am measuring the     purification product instead of the direct PCR product.
Digestion
1) We added 20 uL of water + 5 uL Buffer + 20 uL DNA + 2.5 uL EcoRI + 2.5 uL PstI.
2) Then we put it at 37ºC at the Isotemp for 5 min.
This tubes was named Digested agaA gBlock
Purification
4) We will purify the tube using the QIAquick kit .
This tube was called Dig + Pur agaA gBlock
Ligation
We ligated Dig+Pur PSB1C﻿3 (June 26th) and Dig+Pur agaA gBlock (July 12th)
Following the Thermo T4 DNA Ligase protocol . Instead of leaving the tube at 22ºC for 10 min, we left it at room temperature for 45 min.
5) We will make a ligation. We will make a 1:3 proportion for the gBlock:vector. We added 3,3 uL of the linearized vector [25 ng/mL] and 7     uL of the insert DNA [50 ng/mL].
The protocol says to leave the ligation for 10 min, but we left if for 20 min at room temperature. The tube is called Ligation agaA Biobrick.
Goal: Transform E coli with agaA Biobrick
Procedure:
We will transform competent E coli that were previously made by Jake following his own protocol. We will use the agaA Biobrick.
We will follow this Heat Shock protocol . We made two transformations (two tubes following the same protocol)
We will plate one of the tubes in an LBA+Cm plate. We will leave the other tube in the incubator overnight and plate it the day after. For each tube, we     always plate 20 uL in one plate and 200 uL in another one.
Results:
On July 13th we can see what seems a colony in one of the plates. We will see tomorrow.
On July 14th we see colonies.
July 14th
Goal:Culture Corynebacterium glutamicum and Corynebacterium striatum
Procedure:
We will open the flask that arrived and culture Corynebacterium in Tryptophan Soy liquid medium at 37ºC overnight. After, we will plate them and culture single colonies to make competent cells and transform them. Corynebacterium glutamicum is sPB.006 and Corynebacterium striatum is sPB.007.
Goal:Obtain agaA in pSEVA (Digest pSEVA351 + ligate with agaA gBlock)
Procedure:
We will digest pSEVA351 following the Thermo Fast Digestion Protocol for enzimes EcoRI and PstI. Then we will ligate it with the agaA gBlock. We want to obtain agaA in pSEVA and transform Corynebacterium glutamicum and Corynebacterium striatum.
We measured the purified aga gBlock in the nanodrop, the concentration was 50 ng/mL. This is low. I do not know if it is low because I am measuring the         purification product instead of the direct PCR product.
Digestion
1) We added 20 uL of water + 5 uL Buffer + 20 uL DNA + 2.5 uL EcoRI + 2.5 uL PstI.
2) Then we put it at 37ºC at the Isotemp for 5 min.
This tube was named Digested pSEVA351
Purification
4) We will purify the tube using the QIAquick kit .
This tube was called Dig + Pur pSEVA351
Ligation
We ligated Dig+Pur agaA gBlock (July 12th) and Dig+Pur pSEVA351(July 14th)
Following the Thermo T4 DNA Ligase protocol . Instead of leaving the tube at 22ºC for 10 min, we left it at room temperature for 45 min.
5) We will make a ligation. We will make a 1:3 proportion for the gBlock:vector. We added 3,3 uL of the linearized vector [25 ng/mL]         and 7 uL of the insert DNA [50 ng/mL].
The protocol says to leave the ligation for 10 min, but we left if for 20 min at room temperature. The tube is called Ligation pPB.003.
July 16th
Goal: Colony PCR: Check agaA Biobrick transformation
Procedure:
Yesterday July 15th Juanma runned a colony PCR with oPB.001 and oPB.005 using the
Phusion Polymerase protocol
. Today we will run a 1% agarose gel with the purified DNA following the
standard protocol
.
Results:
There are no bands on the gel. This might happen with Colony PCR. To double-check, we will do a MiniPrep to extract the agaA Biobrick and run a PCR with oPB.001 and oPB.005.
Goal:Make competent Corynebacterium
Procedure:
We will use the eppendorf protocol ( https://docs.google.com/file/d/0By8yVXC0fFVRMTlDd3BxQXM2Y1U/edit ) for making competent cells, but when growing them overnight, we will follow Haynes and Britz protocol (The effect of growth conditions of Corynebacterium     glutamicum on the transformation frequency obtained by electroporation). We did this because its been proven that adding glycine to the medium overnight     increases the electrotransformation efficiency. This is the different part of the protocol:
Procedure:
Cells were grown for approximately 18 h at 30ºC in a rotary shaker at 200 r.p.m. in 400 ml LB supplemented with kanamycin (50 pg ml-l) and glycine         (2.5%).
July 17th
Goal:Check transformation of E coli with the agaA Biobrick (MiniPrep + PCR + Gel)
Procedure:
MiniPrep
We will extract agaA Biobrick by making a MiniPrep of the 8 colonies following the standard protocol.
PCR
We will run a PCR with oPB.001 and oPB.005 following the Phusion Polymerase protocol . And we will run a 1% agarose gel with the purified DNA following the standard protocol .
Results:
Again, no bands.
Goal:Make competent Corynebacterium + Transform Corynebacterium with pSEVA
Procedure:
We will use the eppenforf protocol (https://docs.google.com/file/d/0By8yVXC0fFVRMTlDd3BxQXM2Y1U/edit)
Results:
There are a lot of colonies for Corynebacterium striatum. In both plates. We will check that there is an insert. for that, we have started a culture (July 20th) and we will PCR it tomorrow.
July 18th
Goal:Check transformation of E coli with agaA Biobrick
Procedure:
Using the MiniPrep we got on July 17th, we will do a digestion analysis: (Jake's recipe)
- 5 uL plasmid
- 2 uL Fast Digest Green (has LD in it)
- 1 uL NotI or EcoRI+PstI. We used EcoRI+PstI as it was impossible to find NotI.
- 13uL of H2O
(Final volume of 20 uL)
We will incubate for 15-30 min at 37ºC. Then we will run 10 uL (our Fast Digest Green buffer already has Loadyng Dye in it, so no need to add more) in     an agarose 1% gel.
Results:
The gel is empty. We cannot even see the vector.
Goal:amplify agaA in Corynebacterium Striatum
Procedure:
500uL liquid culture of Corynebactereium striatum overnight DNA extraction with RapidWater DNA isolation kit x2 number one : 25 ng/uL number two : 28 ng/uL PCR with : oPB.046/oPB.047 (agaA primers 1) oPB.048/oPB.049 (agaA primers 2) PCR protocol for 500 nucleotides PCR products: Reagent for 1X Volume: Nuclease-free water : 35 uL 5x Phusion HF Buffer : 10 ul 10 mM dNTPs : 1 ul Forward Primer (10 uM) : 0.5 ul Reverse Primer (10 uM) : 0.5 ul Template DNA : 1 ul DMSO : 1.5 ul Phusion DNA Polymerase : 0.5 ul Thermocycler Protocol: NEB Phusion
Temperature (°C)
Time
Start
98
30 s
Melt
Cycle 1
98
10 s
Melt
35 Cycles
Cycle 2
50
30 s
Anneal
35 Cycles
Cycle 3
72
20 s
Extend
35 Cycles
Finish
72
5 min
Extend
Store
10
Forever
Store
Result: No band, maybe no agaA in Corynebacterium striatum ?
July 20th: Crispr specificity
Goal
Mix the the two strains in order to study the limit concentration of mutant that our CRISPR/Cas system could isolate.
Procedure
At the 3rd washing step, the strains were mixed in different ratios (from 50/50 to 1 / 10 to the minus 7).
Then the mixes were transformed with the regular electroporation protocol.
Results
Here are reported the ratio blue/white colonies:
LacZ Plate1
LacZ Plate2
LacZ Plate3
LacZ Plate4
Rand Plate1
Rand Plate2
Rand Plate3
Rand Plate4
10-7
0.15
0.16
0.16
0.23
5.00E-01
7.00E-01
 
2.00E-01
10-6
0.69
0.78
0.7
0.65
17
3
6
5
10-5
0.5
0.59
0.67
0.7
700
90
100
10
10-4
0.5
0.59
0.67
0.65
1,354
342
456
320
10-3
0.6
1
0.5
0.7
4,400
7,000
2,600
980
10-2
1
1
1
1
14,500
34,000
10,000
23,000
10-1
1
1
1
1
150,000
130,000
150,000
120,000
0.5
1
1
1
1
540,000
570,000
480,000
400,000
July 20th
Goal: Ligate agaA Biobrick + E coli transformation
Procedure:
Ligation
We will use the digested agaA gBlock and PSB1C3 obtained on June 26th .
We will follow the Thermo T4 DNA Ligase protocol . I think last ligation did not work because it was too hot in the lab. This time, I will leave the ligation product for 30 min in the dark room             (20ºC).
E coli transformation
We will transform competent NEB turbo E coli made following the standard protocol . In step 5, we did not incubate them for 4 hours, but only until the density was optimal (around 30-45 min).
We will use agaA Biobrick obtained today.
We will follow this Heat Shock protocol .
Tip (From Matt): At the last step when we recover the cells, plate most of the 200 uL and leave the rest recovering overnight. Then plate                         them the next day.
We will plate them in LBA+Cm.
Results:
There were no colonies 48h after.
August
August 1st
Goal:  Check E coli transformation with agaA Biobrick (miniprep + digestion)
Procedure:
Ligation
We will use the digested agaA gBlock and PSB1C3 obtained on June 26th .
We will follow the Thermo T4 DNA Ligase protocol . I think last ligation did not work because it was too hot in the lab. This time, I will leave the ligation product for 30 min in the dark room             (20ºC).
E coli transformation
We will transform competent NEB turbo E coli made following the standard protocol . In step 5, we did not incubate them for 4 hours, but only until the density was optimal (around 30-45 min).
We will use agaA Biobrick obtained today.
We will follow this Heat Shock protocol .
Tip (From Matt): At the last step when we recover the cells, plate most of the 200 uL and leave the rest recovering overnight. Then plate                         them the next day.
We will plate them in LBA+Cm.
Results:
There are colonies on the plate. We will culture, miniprep and digest them.
August 3rd
Goal: Transform the WT and the mutant with fluorescent plasmids in order to spot CRISPR induced mutants
Procedure:
The WT was transformed with a CFP plasmid, and the mutant with a YFP plasmid, by regular heat shock protocol (30 seconds at 42°C). Cells were recovered 1h at 37°C after transformation and plated on chloramphenicol+spectomycin plates 24h at 37°C. Plates were put under optical filters to check the fluorescence.
Results:
Colonies had the expected fluorescence according to the transformation.
August 4th
Goal: Amplify ackA, l-ldH, d-ldH and ldH in microbiome armpit samples.
DNA extraction from cotton swabs.
1st step : remove cotton from stick and immerse it in PBS.
2nd step : vortex for 20s
3rd step : follow RapidWater DNA isolation kit’s protocol
PCR on DNA extraction with :
oPB.066/oPB.067 (ackA)
oPB.068/oPB.069 (d-ldH)
oPB.070/oPB.071 (l-ldH)
oPB.036/oPB.037 (ldH)
It worked for the four genes.
Column 1 : ladder 1 kb
Column 2 : ackA
Column 3 : d-ldH
Column 4 : l-ldH
Column 7 : ldH
August 5th
Goal:  Check E coli transformation with agaA Biobrick and Corynebacterium transformation with pSEVA +
Procedure:
MiniPrep
We will extract the agaA Biobrick by making a MiniPrep of the 8 colonies + the new E coli transformation (July 20th) + Corynebacterium transformed with pSEVA     following the standard protocol.
In the nanodrop, we saw that the yield for the 8 colonies was way too low (10 ug/mL) so we discarded them. We will continue with the E coli transformed on     July 20th (37 ug/mL) + Corynebacterium transformed with pSEVA (117 ug/mL).
Digest
Using the MiniPrep we got on July 17th, we will do a digestion analysis: (Jake's recipe)
- 5 uL plasmid (for the Ecoli we used 10 uL and added less water)
- 2 uL Fast Digest Green (has LD in it)
- 1 uL NotI or EcoRI+PstI. We used EcoRI+PstI
- 13uL of H2O
(Final volume of 20 uL)
We will incubate for 15-30 min at 37ºC. Then we will run 10 uL (our Fast Digest Green buffer already has Loadyng Dye in it, so no need to add         more) in an agarose 1% gel.
Results:
-
1st raw:
Gene Ruler 100 kb + 1uL Loading Dye
- 2nd raw: Corynebacterium striatum transformed with pSEVA
- 3st raw: E. coli transformed with agaA Biobrick. However it weights 5000 kb. We go back to our protocol of August 1st and in fact what we have     here is E coli transformed with pSEVA.
Conclusions:
Looking at this gel:
- We should not have used the 100 kb rule but the 1 kb
- Corynebacterium has not been transformed
- When we run at 100 V, the gel looks worse
Goal: Corynebacterium striatum transformation with pSEVA
Procedure:
We will use the eppenforf protocol (https://docs.google.com/file/d/0By8yVXC0fFVRMTlDd3BxQXM2Y1U/edit)
Results:
There are a lot of colonies in the plate. We will strike them to pick single colonies and check the transformation.
Goal: Sequencing PCR product from 03/08 PCR to check if I amplified right genes
Procedure:
Remind : I amplified ackA, d-ldH, l-ldH, ldH.
August6th
Goal:  agaA Biobrick ligation
Procedure:
We are going to check our ligation product in a gel to see what is going on. Last time, the problem is that I used pSEVA to transform.
Ligation
We will use the digested agaA gBlock and pSB1C3.
We will follow the Thermo T4 DNA Ligase protocol .
We runned 5 uL of the digestion product in a 1% agarose gel. We used 2 uL to transform E coli. The rest of the tube was kept at 4ºC. If the             transformation goes wrong, we will use it tomorrow to transform again.
Goal:  Transform E coli with agaA Biobrick
Procedure:
We will transform competent NEB turbo E. coli made following the standard protocol . In step 5, we did not incubate them for 4 hours, but only until the density was optimal (around 30-45 min).
We will use agaA Biobrick .
We will follow this Heat Shock protocol .
Tip (From Matt): At the last step when we recover the cells, plate most of the 200 uL and leave the rest recovering overnight. Then plate them the next             day.
We will plate them in LBA+Cm. We will add 250 uL of Cm for 250 mL of LBA.
Results:
There are colonies in the plate. We made a sub culture to pick single colonies on August 6th. On August 7th we made a liquid culture to then miniprep it and check the transformation.
August 7th
Goal: Partially run the crispr specificty study (see July 20th) but also compare the blue/white screening and the fluorescence
Procedure:
The same protocol was used (see July 20th) but on the fluorescent strains. Blue and colorless colonies were counted, and their fluorescence was checked with an optical filter.
Results:
Here are reported the number of counted colonies regarding their color and their fluorescence.
LacZ targeting CRISPR
Non targeting CRISPR
Colorless colonies
Blue colonies
Colorless colonies
Blue colonies
CFP
17
15
16
16
23
26
24
27
20
21
23
26
875
873
870
839
YFP
247
256
239
230
0
0
0
0
11
12
15
12
0
0
0
0
The mismatches are to be studied in more details.
Goal:Obtain agaA Biobrick (agaA gBlock digestion + ligation)
Procedure:
We will digest the purified agaA gBlock following the Thermo Fast Digestion Protocol for enzimes EcoRI and PstI. Then we will ligate it with the linearized vector pSB1C3. We want to obtain agaA Biobrick and transform E. coli.
Digestion
1) We are using the agaA working stock. We added 20 uL of water + 5 uL Buffer + 20 uL DNA + 2.5 uL EcoRI + 2.5 uL PstI.
2) Then we put it at 37ºC at the Isotemp for 1h30 (instead of 5 min).
3) We will repeat for PSB1C3, using the 'plasmid DNA' protocol.
These tubes were called Digested gBlock and Digested PSB1C3
Purification
4) We will purify both tubes using the QIAquick kit .
These tubes were called Dig + Pur gBlock and Dig + Pur PSB1C3
Ligation
﻿Following the Thermo T4 DNA Ligase protocol . Instead of leaving the tube at 22ºC for 10 min, we left it at room temperature for 45 min.
5) We will make a ligation. We will make a 1:3 proportion for the gBlock:vector. We added 3,3 uL of the linearized vector [25 ng/mL] and     2,5 uL of the insert DNA [100 ng/mL].
Purification
6) We will purify the product using the QIAquick kit .
This tube was called agaA Biobrick
Results:
In the nanodrop, we have seen that the concentration of both our vector and insert is 2 ng/mL. This is very low for ligation. This might be the reason why     ligation does not work: not enough DNA.
Next step:
- Take PSB1C3 from the BioBrick parts, clone it into E coli
- Miniprep
- Use this PSB1C3 to obtain agaA Biobrick
Goal:Check transformation in Corynebacterium
Procedure:
We picked 2 single colonies and incubated them in LB at 37ºC overnight.
Goal:Glycerol stock for E coli with pSEVA
Procedure:
We added 750 uL of culture and 250 uL of Glycerol 60% and stock them in the common box at -20ºC and some other tubes at -80ºC.
August 8th, 10th and 11th
Goal:Check Corynebacterium transformation with pSEVA
Procedure:
MiniPrep
We will extract agaA Biobrick by making a MiniPrep of 2 colonies of Corynebacterium transformed with pSEVA following the standard protocol.
In the nanodrop, we saw that the yield for colony 1 was 80 ng/uL and colony 2 had 170 ng/uL
Digest
Using the MiniPrep we got on July 17th, we will do a digestion analysis: (Jake's recipe)
- 5 uL plasmid (for the colony 1 we used 10 uL and added less water)
- 2 uL Fast Digest Green (has LD in it)
- 1 uL NotI (we have checked that it makes two cuts in pSEVA, so we will see two bands)
- 13uL of H2O
(Final volume of 20 uL)
We will incubate for 15-30 min at 37ºC. Then we will run 10 uL (our Fast Digest Green buffer already has Loadyng Dye in it, so no need to add         more) in an agarose 1% gel.
Goal: Obtain PSB1C3 and clone it in E coli
Procedure:
We will take PSB1C3 from the DNA Kit Plate and clone it into E coli. Then, we will make a culture, grow it overnight and miniprep it tomorrow. This was, we will obtain PSB1C3 in a much higher         concentration than we had in the linearized version.
It is on the Spring 2014 Plate 4, 4B
DNA Kit Plate Instructions
Before you use the DNA in the Distribution Kit Plates, be sure to test the efficiency of your competent cells with the Transformation Efficiency Kit.
To use the DNA in the Distribution Kit, follow these instructions:
Note: There is an estimated 2-3ng of DNA in each well, following this protocol, assume that you are transforming with 200-300pg/ul
With a pipette tip, punch a hole through the foil cover into the corresponding well of the part that you want. Make sure you have properly oriented the plate . Do not remove the foil cover, as it could lead to cross contamination between the wells.
Pipette 10uL of dH2O (distilled water) into the well. Pipette up and down a few times and let sit for 5 minutes to make sure the dried DNA                     is fully resuspended. We recommend that you do not use TE to resuspend the dried DNA.
Transform 1ul of the resuspended DNA into your desired competent cells, plate your transformation with the appropriate antibiotic* and grow                     overnight.
Pick a single colony and inoculate broth (again, with the correct antibiotic) and grow for 16 hours.
Use the resulting culture to miniprep the DNA AND make your own glycerol stock (for further instruction on making a glycerol see this page ).                     We recommend using the miniprepped DNA to run QC tests, such as restriction digests and sequencing.
* To know which antibiotics to use, look at the plasmid that the part is in. The naming scheme for             plasmids is specifically designed to indicate antibiotic resistance.
Note: There is not enough DNA in each well to perform anything but transformations
Transformation
We will transform competent NEB turbo E coli made following the standard protocol . In step 5, we did not incubate them for 4 hours, but only until the density was optimal (around 30-45 min).
We will use PSB1C3 obtained today from the DNA Kit Plate.
We will follow this Heat Shock protocol .
Tip (From Matt): At the last step when we recover the cells, plate most of the 200 uL and leave the rest recovering overnight. Then plate them the                 next day.
We will plate them in LBA+Cm. We will add 250 uL of Cm for 250 mL of LBA. I forgot to do it on the afternoon, so I did it on the 9th and the 10th I         started a culture for miniprep.
Results:
div>     I have too many colonies and they do not look red, just redish. So two things:
- Strike a sample in another plate to see single colonies (August 10th) - Plate 1
- Do a new transformation (August 10th) - Plate 2
August 11th - We see colonies in both plates, but in Plate 1 they are bigger and express very intensly RFP. So we took a colony from this plate to     miniprep and extract PSB1C3.
Goal:amplifying and sequencing smell genes from a mix of 10 people’s DNA extraction and test control.
Procedure:
1> To use like control, I chose an available strain : staphylococcus capitis
I test on it these following primers :
oPB.066/oPB.067
oPB.068/oPB.069
oPB.070/oPB.071
oPB.036/oPB.037
Column 1 : ladder 100bp
Column 2 : ackA
Column 3 : l-ldH
Column 4 : d-ldH
I can’t amplify d-ldH in Staphylococcus capitis.
1st step: 10 people rub cotton swab on their armpit.
2nd step: After immersing the cotton in PBS I mix all PBS together in order to get a mix of samples.
3rd step: I follow the RapidWater DNA isolation kit protocol.
DNA nanodrop
Mix : 48 ng/uL
1 : 50 ng/uL
2 : 51 ng/uL
3 : 24 ng/uL
4 : 30 ng/uL
5 : 30 ng/uL
6 : 41 ng/uL
7 : 43 ng/uL
8 : 51 ng/uL
9 : 32 ng/uL
10 : 42 ng/uL
11 : 26 ng/uL
12 : 70 ng/uL
PCR for each sample with following primers :
oPB.066/oPB.067
oPB.070/oPB.071
oPB.068/oPB.069
oPB.036/oPB.037
We can notice both amplification : T and C. The reverse confirm that it's not a sequencing mistake. For example this mutation is non synonymous substitution which change Proline into Serine in 3D protein structure.
August 12th
Goal:agaA Biobrick with new PBS1C3
Procedure:
Today is fun day.
1) Miniprep E coli culture
We will extract PSB1C3 by making a MiniPrep of the colony of E coli we transformed following the standard protocol.
In the nanodrop, we saw that the yield was 253,3 ng/uL
2) Digest PBS1C3
We will digest the purified agaA gBlock following the Thermo Fast Digestion Protocol for enzimes EcoRI and PstI. Then we will ligate it with the linearized vector pSB1C3. We want to obtain agaA Biobrick and transform E coli.
We digested two tubes. We used the protocol for Genomic DNA because it has worked for some colleagues that are doing the same, and I         would like to have a big amount of final volume.
We will digest for 2h. We added 20 uL of PSB1C3 (we wanted to have 5 ug), so we used 10 uL less of water.
We called these tubes Dig PSB1C3.
3) Purify PBS1C3 with sticky ends in a gel
We run an agarose gel at 1%, cut the region of PSB1C3 and make a gel purification using     the kit.
4) Ligate PSB1C3 with the gBlock
﻿Following the Thermo T4 DNA Ligase protocol . Instead of leaving the tube at 22ºC for 10 min, we left it at room temperature for 45 min.
We will make a ligation. We will make a 1:3 proportion for the gBlock:vector. We added 6,6 uL of the PSB1C3 vector [11.3 ng/uL] and         2,5 uL of the insert DNA [100 ng/uL].
5) Purification
We will purify the product using the QIAquick kit .
This tube was called pPB.001
6) Transform agaA Biobrick in E coli
We will transform competent NEB turbo E coli made following the standard protocol . In step 5, we did not incubate them for 4 hours, but only until the density was optimal (around 30-45 min).
We will use agaA Biobrick obtained today.
We will follow this Heat Shock protocol .
Tip (From Matt): At the last step when we recover the cells, plate most of the 200 uL and leave the rest recovering overnight. Then plate them                     the next day.
We will plate them in LBA+Cm. We will add 250 uL of Cm for 250 mL of LBA.
Results:
No colonies were seen 12h after. We will incubate more time. So we will (August 13th):
- We will repeat the heat shock transformation with the ligation product that was left at 4ºC
- We will run a gel with the ligation product
August 15th and 16th
Because we cannot see any colonies on August 14th, we will:
- We will repeat the heat shock transformation with the ligation product that was left at 4ºC
- We will run a gel with the ligation product
If it does not work, we should try electroporation.
Goal:Check E coli transformation with agaA Biobrick
Procedure:
MiniPrep
We will extract the agaA Biobrick by making a MiniPrep following the standard protocol.
In the nanodrop, we saw that the yield was10 ug/mL.
Digest
Using the MiniPrep we got on August 13th, we will do a digestion analysis: (Jake's recipe)
- 5 uL plasmid (for the Ecoli we used 10 uL and added less water)
- 2 uL Fast Digest Green (has LD in it)
- 1 uL NotI or EcoRI+PstI. We used EcoRI+PstI as it was impossible to find NotI.
- 13uL of H2O
(Final volume of 20 uL)
We will incubate for 15-30 min at 37ºC. Then we will run 10 uL (our Fast Digest Green buffer already has Loadyng Dye in it, so no need to add         more) in an agarose 1% gel.
Results:
We can see the pSB1C3 but not agaA. We are going to make a culture of the second plate we plated, and try again with more colonies tomorrow.
August 17th
Goal:Check E coli transformation with the agaA Biobrick
Procedure:
MiniPrep
We will extract the agaA Biobrick by making a MiniPrep of 8 colonies of the second plate we plated (we left a part of our liagation at 4ºC overnight and then     transformed E coli) following the standard protocol.
Digest
Using the MiniPrep we got on August 13th, we will do a digestion analysis: (Jake's recipe)
- 5 uL plasmid
- 2 uL Fast Digest Green (has LD in it)
- 1 uL NotI or EcoRI+PstI. We used EcoRI+PstI
- 13uL of H2O
(Final volume of 20 uL)
We will incubate for 15-30 min at 37ºC. Then we will run 10 uL (our Fast Digest Green buffer already has Loadyng Dye in it, so no need to add         more) in an agarose 1% gel.
PCR
We also made a PCR to see the gBlock. We followed Jake's protocl for the Fusion Polymerase with the forward primer oPB.001 and the reverse primer oPB.005.
Results:
-We cannot see anything in the analysis by digestion.
-The gel of the PCR confirms the presence of our gBlock with agaA
- 1st raw: Gene Ruler 100 kbPlus + 1uL Loading Dye
- 2nd raw: Colonies from the second transformation (Ligation was left overnight at 4ºC). There are 7 single colonies that were miniprep. We PCR with oPB.001 and oPB.005 (for the gBlock that contains agaA). We can see a band that does not correspond with the gBlock at around 350bp.
August 21st
Goal: Study in more details the colorless colonies having a CFP fluorescence
Procedure:
Colony PCR were performed following the protocol used June 25th. The colony were also re-streaked on X-gal.
Results:
The sequences matched the expected sequence of a WT strain. The colonies were blue on X-gal. We can deduce unperfect efficiency of the blue-white screening and re-adjust the figure on Crispr efficiency.
September
September 3rd
Goal:Check Ligation of agaA gBlock in pSEVA // YFP and BFP in pSEVA
Procedure:
Digestion
We will digest v92 and V93 following Jake's recipe:
- Vector: 10 uL
- Green Buffer: 4 uL
- XbaI: 2 uL
- XhoI: 2 uL
- H20: 22 uL
Total: 40 uL
 
We will digest pSEVA (from my box 'pSEVA c1; 63 ug/uL) following Jake's recipe:
- Vector: 30 uL
- Buffer: 20 uL
- XbaI: 4 uL
- Sal1: 4 uL
- H20: 142 uL
Total: 200 uL
We will digest the gBlock containing agaA (from my box 'gBlock aga Dig+Pur') following Jake's recipe:
- Vector: 30 uL
- Buffer: 20 uL
- EcoRI: 4 uL
- PstI: 4 uL
- H20: 142 uL`'
Total: 200 uL
 
We will digest pSEVA (from my box 'pSEVA c2; 140 ug/uL) following Jake's recipe:
- Vector: 20 uL
- Buffer: 20 uL
- EcoRI: 4 uL
- Pst1: 4 uL
- FastAP: 4 uL
- H20: 142 uL
Total: 200 uL
 
We will incubate all tubes at 37ºC for 2 hours.
 
Purify
We will purify both tubes using the  QIAquick kit . 
 
These tubes were called agaA EP Dig + Pur, pSEVA EP Dig + Pur and pSEVA XS Dig+Pur
 
- agaA EP Dig+Pur: Tube 1: 5,2 ng/mL;/ Tube 2: 5 ng/uL
- pSEVA EP EP Dig+Pur: Tube 1: 18,6 ng/mL; Tube 2: 8 ng/uL
- pSEVA XS Dig+Pur: Tube 1: 10 ng/mL; Tube 2: 7 ng/uL
 
Ligate pSEVA with the gBlock
Following the  Thermo   T4 DNA Ligase protocol . 
 
We will make a ligation. We will make a 1:3 proportion for the gBlock:vector. We added 3 uL of the pSEVA EP vector [18,6 ng/uL, 22,51 fmols] and 3 uL of the insert DNA [5,2 ng/uL, 5,6 fmols].
 
We will leave the tube at 22ºC for 1h. Then the rest will be kept at 4ºC overnight.
 
This tube was called agaA in pSEVA
 
Results
I will take care of the pSEVA-gBlock ligation. This is the gel:
img src="https://static.igem.org/mediawiki/2014/7/7e/Gel_03-09_Dontsweatit_PB.JPG"
- 1st raw: Loading dye 1kb
- 2nd raw: sample of agaA EP Dig+Pur.
- 3rd raw: sample of pSEVA EP Dig+Pur.
- 4th raw: ligation product agaA in pSEVA (ligation of agaA EP Dig+Pur and pSEVA EP Dig+Pur)
 
Ligation is successful! Next time, we should add more DNA and decrease water.
 
September 4th •Design des oligos
Goal: Design with Geneious primers for PCR cloning in order to clone the CRISPR/Cas system into the pSeva vector.
Procedure
The primer design was made following the regular protocol (see 1st of June), but were added tails containing restriction sites for XbaI and SalI.
September 8th •Cloning
Goal: Clone the whole CRISPR/Cas system in the pSeva vector in order to be able to transform a broad range of strains.
Procedure
PCR were performed following the regular protocol.
PCR products were PCR purified, following the regular Qiagen PCR purification protocol.
Purified products were digested with XbaI and SalI, following the same protocol as the one followed the 25th of June.
Digestion products were PCR purified.
Purification products were used for ligation at room temperature for 2h, following this preparation :
1uL of T4 Ligase
2uL of Ligase Buffer
15uL of NF water
1uL of vector
1uL of insert
The ligation products were transformed by electroporation in BL21-AI, and plated on Chloramphenicol plates for 24h at 37°C.
Results
Around 500 colonies were found after 24h hours of growing.
September 16th
Goal:Transform agaA+pSEVA in E. coli
Procedure:
**There was a previous attempt where I directly transformed Corynebacterium with agaA+pSEVA. The rate of transformation of the product of a ligation is too low. First, we have to transform E. coli, that has a higher rate. Then we miniprep this plasmid and transform it in Corynebacterium.**
We will transform competent NEB turbo E coli made following the  standard protocol  . In step 5, we did not incubate them for 4 hours, but only until the density was optimal (around 30-45 min).
 
We will use agaA+pSEVA.
 
We will follow this  Heat Shock protocol . 
 
Tip (From Matt): At the last step when we recover the cells, plate most of the 200 uL and leave the rest recovering overnight. Then plate them the next day. 
    
We will plate them in LBA+Cm. We will add 250 uL of Cm for 250 mL of LBA.  
Results
Next day we had colonies. We started a culture to miniprep and check transformation.
September 17th
Goal:Check E. coli transformation withagaA+pSEVA
Procedure:
MiniPrep
We will extract agaA+pSEVA by making a MiniPrep following the  standard protocol.
 
 
Digest
We will do a digestion analysis: (Jake's recipe)
- 5 uL plasmid 
- 2 uL Fast Digest Green (has LD in it)
- 1 uL NotI or EcoRI+PstI. We used NotI
- 13uL of H2O
 
(Final volume of 20 uL)
 
We will incubate for 15-30 min at 37ºC. Then we will run 10 uL (our Fast Digest Green buffer already has Loadyng Dye in it, so no need to add more) in an agarose 1% gel.
 
Results
- 1st raw: Loadying Dye 1kb
- 2nd raw: sample from culture 1: we threw it away. We do not see anything!
- 3rd raw: sample from culture 2. Not the right band size
 
September 18th
Goal:Transform Corynebacterium striatum with agaA+pSEVA
Procedure:
We will use the eppendorf protocol ( https://docs.google.com/file/d/0By8yVXC0fFVRMTlDd3BxQXM2Y1U/edit ) for Corynebacterium electroporation.
 
Procedure:
     Cells were grown for approximately 18 h at 30ºC in a rotary shaker at 200 r.p.m. in 400 ml LB supplemented with kanamycin (50 pg ml-l) and glycine (2.5%).
Goal:Make competent Corynebacterium
Procedure:
 We will use the eppendorf protocol ( https://docs.google.com/file/d/0By8yVXC0fFVRMTlDd3BxQXM2Y1U/edit ) for making competent cells, but when growing them overnight, we will follow Haynes and Britz protocol (The effect of growth conditions of Corynebacterium glutamicum on the transformation frequency obtained by electroporation). We did this because its been proven that adding glycine to the medium overnight increases the electrotransformation efficiency. This is the different part of the protocol:
 
Procedure:
     Cells were grown for approximately 18 h at 30ºC in a rotary shaker at 200 r.p.m. in 400 ml LB supplemented with kanamycin (50 pg ml-l) and glycine (2.5%).
September 20th •Study of clones
Goal: Study if the cloning was effective.
Procedure
Liquid cultures were made out of colonies.
The liquid cultures were minipreped, and the minipreps were digested with HindIII and ran on a gel.
The minipreps were also sequenced.
Results
The expected bands appeared on the gel (around 6100, 2300, 1700).
The expected sequence resulted from the sequencing.
September 23rd
Goal:Obtan agaA Biobrick
Procedure:
Ligate PSB1C3 with the gBlock
 
Following the  Thermo   T4 DNA Ligase protocol . Instead of leaving the tube at 22ºC for 10 min, we left it at room temperature for 45 min.
 
We will use tubes PSB1C3 Dig+Pur and gBlock EP Dig+Pur
 
We will make a ligation. We will make a 1:3 proportion for the gBlock:vector. We added 6 uL of the PSB1C3 vector [11 ng/uL] and 6 uL of the insert DNA [5 ng/uL].
 
**Usually we would add 3 uL for each but we are doubling the concentration of DNA and decreasing the H20**
Results:
We runned a 1% agarose gel. The gel shows that there is nothing on the PSB1C3 tube. We will alplify PSB1C3 from Antonio's stock using oPB.082 and oPB.083 using.
Goal:PCR PSB1C3
Procedure:
 PCR of PSB1C3 from Antonio's box using the  Fusion Polymerase  and primers oPB.082 and oPB.083.
September 24th
Goal:Transform agaA+pSEVA in E. coli
Procedure:
We will purify the PCR product obtained on Sep 23rd following the  QIAquick PCR purification kit standard protocol .
Goal: Obtain agaA Biobrick (new PSB1C3 digestion + ligation)
Procedure:
We will digest the purified PSB1C3 following the  Thermo Fast Digestion Protocol  for enzimes EcoRI and PstI. Then we will ligate it with our agaA Dig+Pur. We want to obtain  agaA Biobrick  and transform E coli. 
 
Digestion
 
   1) We are using the PSB1C3 that we just PCRed and purified.
 
We will follow Jake's recipe:
- Vector: 30 uL
- Buffer: 20 uL
- EcoRI: 4 uL
- PstI: 4 uL
- H20: 142 uL
Total: 200 uL
 
    2) Then we put it at 37ºC at the Isotemp for 2h
 
This tube is called PSB1C3 Dig EP
 
Purification
 
    4) We will purify both tubes using the  QIAquick kit . 
 
These tubes were called Dig + Pur gBlock and Dig + Pur PSB1C3
 
Ligation
 
Following the  Thermo   T4 DNA Ligase protocol . Instead of leaving the tube at 22ºC for 10 min, we left it at room temperature for 1h.
 
    5) We will make a ligation. We will make a 1:3 proportion for the gBlock:vector. We added 3 uL of the PSB1C3 vector [2 ng/uL] and 10.5 uL of the insert DNA [5 ng/uL].
 
 
Goal:Transformation of Corynebacterium striatum with agaA +pSEVA
Procedure:
We will use the eppendorf protocol (https://docs.google.com/file/d/0By8yVXC0fFVRMTlDd3BxQXM2Y1U/edit) for Corynebacterium electroporation.      As a contol, we will also transform Corynebacterium striatum with V5 from Jake (pSEVA+cerulean).
Results:
September 25th
Goal:Old agaA Biobrick analytical digestion
Procedure:
We will check the tube of E coli with agaA Biobrick that we made in August 17th. We will do a miniprep and analytical digestion.
 
MiniPrep
We will extract the agaA Biobrick by making a MiniPrep following the  standard protocol.
 
In the nanodrop, we see that the concentration was 20 ng/uL (low)
 
Digest
Using the MiniPrep we got on August 13th, we will do a digestion analysis: (Jake's recipe)
- 10 uL plasmid (we would usually put 5 but the concentration is very low)
- 2 uL Fast Digest Green (has LD in it)
- 1 uL NotI or EcoRI+PstI. We used EcoRI+PstI 
- 7uL of H2O
(Final volume of 20 uL)
 
We will incubate for 15 min at 37ºC. Then we will run 10 uL (our Fast Digest Green buffer already has Loadyng Dye in it, so no need to add more) in an agarose 1% gel.
 
 
Results:
There is nothing in the gel
Goal:Old agaA Biobrick analytical digestion
Procedure:
    In case the old transformation analysis is negative (and because I have not been able to sequence verify it), we are making a new transformation. There are colonies in yesterday's plates, so we will start cultures to miniprep and do analytical digestion tomorrow.
 
Goal:Ligation agaA gBlock in pSEVA
Procedure:
Ligate pSEVA with the gBlock
Following the  Thermo   T4 DNA Ligase protocol . 
 
We will make a ligation. We will make a 1:3 proportion for the gBlock:vector. We added 3 uL of the pSEVA EP vector [18,6 ng/uL, 22,51 fmols] and 3 uL of the insert DNA [5,2 ng/uL, 5,6 fmols].
 
- 3uL pSeva
- 3uL agaA
-1 uL Buffer
- 0.5 uL Ligase
- 2.5 uL H2O
 
We will leave the tube at room temperature for 1h. Then the rest will be kept at 4ºC overnight.
 
This tube was called agaA+pSEVA
 
 
Procedure:
 
We will transform competent NEB turbo E coli made following the standard protocol . In step 5, we did not incubate them for 4 hours, but only until the density was optimal (around 30-45 min).
 
We will use agaA+pSEVA.
 
We will follow this  Heat Shock protocol . 
 
Tip (From Matt): At the last step when we recover the cells, plate most of the 200 uL and leave the rest recovering overnight. Then plate them the next day. 
    
We will plate them in LBA+Cm. We will add 250 uL of Cm for 250 mL of LBA.  
 
September 26th
Goal: agaA Biobrick miniprep and analytical digestion
Procedure:
  We will check the tube of E coli with the agaA Biobrick (4 different colonies). We will do a miniprep and analytical digestion.
 
MiniPrep
We will extract the agaA Biobrick of 4 different colonies by making a MiniPrep following the  standard protocol.
 
 
Digest
We will do a digestion analysis: (Jake's recipe)
- 5 uL plasmid
- 2 uL Fast Digest Green (has LD in it)
- 1 uL NotI or EcoRI+PstI. We used EcoRI+PstI 
- 12 uL of H2O
(Final volume of 20 uL)
 
We will incubate for 15 min at 37ºC. Then we will run 10 uL (our Fast Digest Green buffer already has Loadyng Dye in it, so no need to add more) in an agarose 1% gel.
 
Results:
We can see two bands. One around 2kb and other around 1.5 kb. This would be PSB1C3 and the gBlock. We will make a stock of colony 4.
September 27th
Goal:agaA+pSEVA miniprep and analytical digestion
Procedure:
    We will check the tube of E coli with agaA+pSEVA (4 different colonies). We will do a miniprep and analytical digestion.
 
 
MiniPrep
We will extract agaA+pSEVA of 4 different colonies by making a MiniPrep following the  standard protocol.
 
 
Digest
We will do a digestion analysis: (Jake's recipe)
- 5 uL plasmid
- 2 uL Fast Digest Green (has LD in it)
- 1 uL NotI or EcoRI+PstI. We used EcoRI+PstI 
- 12 uL of H2O
(Final volume of 20 uL)
 
We will incubate for 15 min at 37ºC. Then we will run 10 uL (our Fast Digest Green buffer already has Loadyng Dye in it, so no need to add more) in an agarose 1% gel.
 
Results:
We can see pSEVA band at 5000 bp and there is another band around 1500 bp. It is not a very strong band but is there. Also, the culture tube smells like body odor, so even if this band is not very strong, we think it is agaA. We will use the miniprep product of colony 1 to transform Corynebacterium striatum.
Goal:Transform Corynebacterium striatum with agaA+pSEVA
Procedure:
   We will use the eppendorf protocol ( https://docs.google.com/file/d/0By8yVXC0fFVRMTlDd3BxQXM2Y1U/edit ) for Corynebacterium electroporation.
    We will use the miniprep product that we just obtained (agaA+pSEVA colony 1). We will recover 2h instead of 1h, as for Cm resistance longer recovery times are advised.
 
September 27th •Killing Efficiency
Goal: Compare the efficacy of the new CRISPR vector with the previous one.
Procedure
The same expriment as July 10th was conducted, for the pCas9 and the pSeva vectors.
Results
WT with non targeting pCas9
Mutant with non targeting pCas9
Replicate 1
1845
1745
Replicate 2
2398
2354
Replicate 3
1549
1860
Replicate 4
1740
1754
Replicate 5
2176
1956
 
WT with non targeting pSeva
Mutant with non targeting pSeva
Replicate 1
2398
2230
Replicate 2
1876
1984
Replicate 3
2784
2093
Replicate 4
2009
2092
Replicate 5
1997
1700
 
WT with LacZ targeting pCas9
Mutant with LacZ targeting pCas9
Replicate 1
5
742
Replicate 2
40
543
Replicate 3
12
984
Replicate 4
2
659
Replicate 5
26
978
September 29th
The sequencing of agaA Biobrick and agaA+pSEVA was negative, so we will throw again everything and start from the very beggining
Digest pSB1C3 vector.
 
P28 (pSB1C3) at 160 ng/ul
Reagent
Volume
Plasmid
30 ul
H2O
135 ul
10x FastDigest Green Buffer
20 ul
FastDigest XbaI
5 ul
FastDigest PstI
5 ul
FastAP
5 ul
Total Volume
200 ul
Gel extract and purify. The vector concentration after purification was 6.5 ng/ul
PCR for agaA gBlock
 
Primers: oPB.001, oPB.005
Template gBlock agaA 1:50 dilution
Reagent
Volume
 
1x
4x
Nuclease-free water
71 ul
284 ul
5x Phusion HF Buffer
20 ul
80 ul
10 mM dNTPs
2 ul
8 ul
Forward Primer (10 uM)
1 ul
4 ul
Reverse Primer (10 uM)
1 ul
4 ul
Template DNA (Plasmid, 1:10 dilution)
1 ul
 4 ul
DMSO
3 ul
12 ul
Phusion DNA Polymerase
1 ul
4 ul
Total Volume
100 ul
 
Thermocycler Protocol: NEB Phusion
 
Temp
Time
 
 
Start
98 °C          
30 sec
Melt
 
 
Cycle 1          
98 °C          
10 sec
Melt
 
35 Cycles          
Cycle 2          
52 °C          
30 sec
Anneal
Cycle 3          
72 °C          
40 sec
Extend
 
Finish
72 °C          
5 min
Extend
 
Store
10 °C          
Forever
Store
         
PCR Product looks good
PCR purify gBlock agaA
The concentration is 80 ng/ul
Digest for cloning:
Reagent
Volume
PCR Product
30 ul
H2O
135 ul
10x FastDigest Buffer
20 ul
FastDigest XbaI
10 ul
FastDigest PstI
10 ul
 
 
Total Volume
200 ul
Resuspend in 50 ul.
The final concentration is 36 ng/ul 
 
 
Vector P28 (6 ng/ul)
Vector P41 diluted 1:10 (5 ng/ul)
Insert gBlock agaA (36 ng/ul)
Enzymes: XbaI, PstI
Reagent
Volume
Vector
12 ul
Insert
5 ul
H2O
0 ul
Fermentas
T4 Ligase Buffer
2 ul
Fermentas
T4 Ligase Enzyme   
1 ul
Total Volume
20 ul
Incubate for 10 min at 22C (manufacturer’s suggestion)
I ran our the ligation controls. The gel looks a little funny. I think it is overloaded. There is some evidence that the ligation worked.
Miniprep and analytical digest with XbaI, PstI. V96 is P28 (pSB1C3) + agaA I picked 6 colonies from the V96 transformation. Clones 4, 5 and 6 all produced the correct banding pattern. I chose Clone 6 because the miniprep had the highest concentraion.
We made a culture and sent it to GATC with the verification primers. The sequencing was positive.
Centre for Research and Interdisciplinarity (CRI) Faculty of Medicine Cochin Port-Royal, South wing, 2nd floor Paris Descartes University 24, rue du Faubourg Saint Jacques 75014 Paris, France
+33 1 44 41 25 22/25
paris-bettencourt-igem@googlegroups.com
Copyright (c) 2014 igem.org. All rights reserved.
Retrieved from " http://2014.igem.org/Team:Paris_Bettencourt/Notebook/Eliminate_Smell "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
