<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Paris_Bettencourt_Notebook_Eliminate_Smell"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Paris_Bettencourt/Notebook/Eliminate_Smell">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Paris_Bettencourt/Notebook/Eliminate_Smell&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Paris_Bettencourt/Notebook/Eliminate_Smell&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Paris_Bettencourt/Notebook/Eliminate_Smell&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Paris_Bettencourt/Notebook/Eliminate_Smell" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Paris Bettencourt/Notebook/Eliminate Smell</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><DIV class="navmenubkg"><DIV id="navmenubox"><UL id="nav"><LI><A href="https://2014.igem.org/Team:Paris_Bettencourt/Project">Project</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Project/Odor_Library">SmellThe Roses</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Project/TMAU">SomethingFishy</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Project/Foot_Odor">GoodyTwo Shoes</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Project/Eliminate_Smell">Don'tSweat It</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Project/Old_People_Smell">Teen Spirit</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Project/Art">Design Practice </A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Project/Bioinformatics">Bioinformatics</A></LI></UL><LI><A href="https://2014.igem.org/Team:Paris_Bettencourt/Achievement">Achievements</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Results">Results</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Project/Interlab_Study">Interlab Study</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Medal">MedalFulfilment</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Parts">Parts</A></LI><LI><A href="https://2014.igem.org/Team:Paris_Bettencourt/Policy">Policy </A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Safety">Safety</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Consent_Form">Consent Form</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Patenting">Patenting</A></LI><LI><A href="https://2014.igem.org/Team:Paris_Bettencourt/Human_Practice">Practices</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/MOOC">MOOC</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Citizen_Science">CitizenScience</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Facts_Smell">The Senseof Smell</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Facts_Micriobiome">The HumanMicriobiome</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Minder">Minder</A></LI><LI><A href="https://2014.igem.org/Team:Paris_Bettencourt/Collaboration_Main">Collaborations </A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Collaboration">Collaborations </A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/The_Smell_Game">The SmellGame</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Newsletter">Newsletters</A></LI><LI><A href="https://2014.igem.org/Team:Paris_Bettencourt/Notebook_Main">Notebook</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Notebook">Daily Note</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Protocols">Protocols</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Bibliograpy">Bibliography</A></LI><LI><A href="https://2014.igem.org/Team:Paris_Bettencourt/Team_Main">Team </A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Team">Our Team </A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Attributions">Attributions</A></LI><LI><A class="liste" href="https://igem.org/Team.cgi?year=2014&amp;team_name=Paris_Bettencourt">Official TeamProfile </A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Acknowledgements">Acknowledgements</A></LI><DIV id="twnoticon" class="sidelogos"><A class="twitter-timeline" width="250" height="400" data-dnt="true" href="https://twitter.com/iGEM_Paris" data-widget-id="368144501119795201">@iGEM_Paris</A></DIV></DIV></DIV></DIV><DIV id="moocdiv"><TABLE><TBODY><TR><TD><A href="https://2014.igem.org/Team:Paris_Bettencourt/Notebook/Odor_Library">Smell The Roses</A></TD><TD><A href="https://2014.igem.org/Team:Paris_Bettencourt/Notebook/TMAU">Something Fishy</A></TD><TD><A href="https://2014.igem.org/Team:Paris_Bettencourt/Notebook/Eliminate_Smell">Don't Sweat It</A></TD><TD><A href="https://2014.igem.org/Team:Paris_Bettencourt/Notebook/Foot_Odor">Goody Two Shoes</A></TD><TD><A href="https://2014.igem.org/Team:Paris_Bettencourt/Notebook/Old_People_Smell">Teen Spirit</A></TD><TD><A href="https://2014.igem.org/Team:Paris_Bettencourt/Notebook/Interlab_Study">Interlab Study</A></TD></TR></TBODY></TABLE></DIV><DIV id="intra"><UL><LI><A href="https://2014.igem.org/Team:Paris_Bettencourt/Notebook/Eliminate_Smell#June">June</A></LI><LI><A href="https://2014.igem.org/Team:Paris_Bettencourt/Notebook/Eliminate_Smell#July">July</A></LI><LI><A href="https://2014.igem.org/Team:Paris_Bettencourt/Notebook/Eliminate_Smell#August">August</A></LI><LI><A href="https://2014.igem.org/Team:Paris_Bettencourt/Notebook/Eliminate_Smell#September">September</A></LI><LI><A href="https://2014.igem.org/Team:Paris_Bettencourt/Notebook/Eliminate_Smell#October">October</A></LI></UL></DIV><DIV id="corps"><H3 id="haut">Notebook</H3><DIV id="June"><H4>June</H4><H5 id="date">June 7th</H5><H6>Goal: Design oligos to change the targetting sequence of a CRISPR plasmid. This is performed in order to target the LacZ gene of <I>E.coli</I></H6><H6>Procedure:</H6><P>This is performed on Geneious:

<UL><LI>Open the LacZ gene sequence on Geneious</LI><LI>Find a PAM sequence in the gene (NGG)</LI><LI>Copy the 30 nucleotide prior to the PAM region, and paste it as a new sequence. Convert it to oligo</LI><LI>Copy it and paste in another new sequence (you now have two times the 30 nucelotides). Convert it to oligo, and Reverse Complement one of the two sequences</LI><LI>Add to the sequence which isn't reverse complemented in the 5' end : AAAC</LI><LI>Add to the sequence which is reverse complemented in the 5' end : AAAAC</LI><LI>The two sequences should look this way :</LI></UL><UL><LI>The name of these two oligos are oPB.008 and oPB.009</LI><LI>A second set of oligos was designed to not be targeting the <I>E.coli</I> genome</LI><LI>A 30 nucleotides sequence was generated randomly and then blasted against bacteria genomes to make sure it was not complementary to any part of the <I>E.coli</I> genome</LI></UL></P><H5 id="date">June 11th</H5>
					
﻿<H6>Goal: Design plasmids that express agaA construct</H6><H6>Procedure:</H6><P>Using Geneious, we first created the agaA construct:

<UL><LI>Promoter BBa_J23108 from the Anderson's promoter collection</LI><LI>RBS from the RBS Calculator of Salis lab specific for E coli</LI><LI>BioBrick Prefix + Scar from the iGEM Parts webpage</LI><LI>agaA sequence codon optimized for E coli 12 (IDT online tool). We checked with Generious that there were no restriction sites for EcoRI, SpeI, ZbaI and PstI.</LI><LI>Histidine tag: 6 Histidines in C-terminal were added</LI><LI>Stop codon</LI><LI>BioBrick scar + BioBrick suffix from the iGEM Parts webpage</LI></UL></P><P>To amplify this construct, we created two oligos:</P><TABLE border=""><TBODY><TR><B><TD>Name</TD><TD>Sequence (5'-&gt;3')</TD><TD>Notes</TD><TD>Binding position</TD></B></TR><TR><TD><B>oPB.001</B></TD><TD>TATAGAATTCGCGGCCGCTTCTAGAGTGACAGCTAGCTCAGTCCTAGG</TD><TD>agaA forward for BioBrick vector</TD><TD>1-&gt;23</TD></TR><TR><TD><B>oPB.002-BAD PRIMER</B></TD><TD>GAAGCATCATCACCATCACCACTGATACTAGTAGCGGCCGCTGCAGTTA</TD><TD>agaA REVERSE for BioBrick vector- NOT REVERSED</TD><TD>1272-&gt;1296</TD></TR><TR><TD><B>oPB.005</B></TD><TD>TTAACTGCAGCGGCCGCTACTAGTATCAGTGGTGATGGTGATGATGCTTC</TD><TD>agaA reverse for BioBrick vector</TD><TD>1272-&gt;1296</TD></TR></TBODY></TABLE><P><I>* We noticed the 16/06/2014 that oPB.002 was NOT reversed and we designed oPB.005 </I></P><P><B>Tip:</B> When creating oligos: we add 4 nt at the beginning and the end composed by AT, to make sure the enzyme will bind properly.</P><P>We chose two backbones for the construct:</P><UL><LI>pSB1C3 from the iGEM Parts to create a BioBrick.</LI><LI>pEC-XC99E, a E coli-C glutamicum shuttle plasmid</LI></UL><P>We designed two plasmids: </P><P><B>    1. pPB.001:</B> Biobrick submission of agaA expression construct</P><P><U>1) Plasmid pSB1C3: </U>High copy BioBrick assembly plasmid. Constitutive expression. To use in E coli</P><P><U>2) agaA construct</U></P><P><B> 2.pPB.002:agaA expression construct. Shuttle vector E coli-C glutamicum</B></P><DIV>
    1) <U>Plasmid pEC-XC99E modified:</U> Ptrc (IPTG inducible promoter) and lacI will be removed by PCR. Primers will be oPB.003 -that includes a PstI site- and
    oPB.004 -that includes a XbaI site.
</DIV><DIV><TABLE dir="ltr" border="1" cellspacing="0" cellpadding="0"><TBODY><TR><TD colspan="1" rowspan="1">
                    oPB.003
                </TD><TD colspan="1" rowspan="1">
                    5'-CTGCAGATGCAAGCTTGGCTGTTTTGGCG-3'
                </TD><TD colspan="1" rowspan="1">
                    PstI site + agaA forward for pEC-XC99E
                </TD></TR><TR><TD colspan="1" rowspan="1">
                    oPB.004
                </TD><TD colspan="1" rowspan="1">
                    5'-TCTAGACACCACCCTGAATTGACTCTCTTC-3'
                </TD><TD colspan="1" rowspan="1">
                    XbaI site + aga reverse for pEC-XC99E
                </TD></TR></TBODY></TABLE></DIV><DIV><P>    2) <U>agaA construct</U></P></DIV><H5 id="date">June 13th</H5><H6>Goal: Since most cloning strains contain the LacZ region that is targeted by the CRISPR/Cas system. The goal is to eliminate the chloramphenicol resistance from the strain BL21-AI (in order to have a proper cloning strain that will be used for the cloning of the CRISPR/Cas system)</H6><H6>Procedure:</H6><P>
The strain BL21-AI was transformed by regular Heat Shock protocol (with 42°C for 30 seconds) with the plasmid pCP20.
The transformations were plated on ampicilin plates at 30°C for 24h.
Colonies were picked and re-streaked on chloramphenicol plates and on ampicilin plates at 30°C for 24h.
After confirming that there were no colonies on the chloramphenicol plates ; colonies from the ampicilin plates were picked and re-streaked on LB plates at 37°C for 24h.
Colonies were picked and re-streaked on Chloramphenicol, Ampicilin and LB at 37°C for 24h.
</P><H6>Results:</H6><P>After the last plating :
<UL><LI>There was no colonies on the chloremphenicol plates suggesting that the original resistance from the strain has been eliminated</LI><LI>There was no colonies on the ampicilin plates suggesting that the thermosensitive plasmid has successfully been deactivated</LI><LI>There was colonies on the LB plates that could be used to make fresh glycerol stocks</LI></UL></P><H5>June 16th</H5>
﻿<H6>Goal: Design new reverse oligo for Biobrick construction</H6><H6>Procedure:</H6><P>
    We noticed oPB.002 was NOT reverse and therefore could not be used. We created oPB.005 instead:
</P><TABLE dir="ltr" border="1" cellspacing="0" cellpadding="0"><TBODY><TR><TD colspan="1" rowspan="1">
                oPB.005
            </TD><TD colspan="1" rowspan="1">
                5'-TTAACTGCAGCGGCCGCTACTAGTATCAGTGGTGATGGTGATGATGCTTC-3'
            </TD><TD colspan="1" rowspan="1">
                agaA reverse for BioBrick vector
            </TD><TD colspan="4" rowspan="1">
                1272-&gt;1296
            </TD></TR></TBODY></TABLE><H5>June 20th</H5><H6>Goal: Change the targeting sequence of the CRISPR/Cas system</H6><H6>Procedure:</H6><P>
1) Preparation of the vector
<UL><LI>The following cells were taken from glycerol stock:</LI></UL></P><TABLE border=""><TBODY><TR><B><TD><B>Strain (old name)</B></TD><TD><B>Carrying the plasmid (old name)</B></TD><TD><B>Resistance</B></TD><TD><B>New name</B></TD></B></TR><TR><TD>sSP004 (DH5alpha)</TD><TD>pM.001 - pCas9</TD><TD>Chl</TD><TD>sPB.008 carrying pPB.008</TD></TR></TBODY></TABLE><P><UL><LI>The cells were streaked on LB plates containing the proper antibiotic (Chl or Kan)</LI><LI>The cells were then incubated at 37°C for 24h</LI><LI>Single colonies were put in tubes containing 5mL of LB and 5uL of antibiotics</LI><LI>Tubes now containing liquid culture were incubated at 37°C with agitation at 200rpm overnight</LI><LI>New glycerol stocks were made using 500uL from liquid culture, mixed with 500ul of 50% glycerol</LI><LI>Cells went through regular miniprep protocol using Thermofisher kit</LI><LI>The pCas9 from miniprep was digested as follow :
    </LI><LI>3 uL of pCas9</LI><LI>2 uL of BsaI</LI><LI>5 uL of FastDigest Buffer</LI><LI>1 uL of 100X BSA</LI><LI>84 uL of dH20</LI></UL><LI>Incubated 1h @ 37°C</LI><LI>The digested product was PCR purified through regular PCR purification protocol using Qiagen kit</LI></P><P>
2) Preparation of the oligos
<UL><LI>20uL of the oligos from IDT were diluted in 180uL of NF water, as a working stock</LI><LI>The oligos were then phosphorylated as follow :
    </LI><LI>1 uL of oligo I</LI><LI>1 uL of oligo II</LI><LI>5 uL of 10X T4 Ligase Buffer</LI><LI>1 uL of T4 PNK</LI><LI>42 uL of dH20</LI></UL><LI>Incubated 1h @ 37°C</LI><LI>The oligos were annealed, by adding 2.5 uL of 1M NaCl, and incubating 5min @ 95°C (slow cool down). They were then diluted 10 times with dH20</LI></P><P>
3) Ligation was performed as follows
<UL><LI>2 uL of digested vector</LI><LI>2 uL of annealed oligos</LI><LI>4 uL of 10X T4 Ligase Buffer</LI><LI>2 uL of T4 Ligase</LI><LI>30 uL of dH20</LI><LI>Incubated 1h @ 37°C</LI><LI>Overnight at room temperature</LI></UL></P><P>
4) Transformation was performed on the fliped-out BL21-AI strain using regular Heat Shock protocol
</P><H6>Results:</H6><P>
More than a 100 colonies were observed after 24h at 37°C.
</P><H5>June 23rd</H5>
﻿<H6>Goal: PCR agaA gBlock with oPB.001 and oPB.005</H6><H6>Procedure:</H6><DIV>
    We are going to <STRONG>PCR</STRONG> using the
    <A href="https://www.evernote.com/Home.action?__fp=ZloDp4AdW-c3yWPvuidLz-TPR6I9Jhx8&amp;username=iGEM-PB2014&amp;login=true&amp;_sourcePage=L90YhFqIo3HiMUD9T65RG_YvRLZ-1eYO3fqfqRu0fynRL_1nukNa4gH1t86pc1SP#st=p&amp;n=f47d72e7-4213-468f-989d-2e9620b0fe66" target="_blank" shape="rect">
        DreamTaq polymerase protocol
    </A>
    the agaA gBlock with the forward primer
    <A href="https://www.evernote.com/Home.action?__fp=IYz5qkyPGHc3yWPvuidLz-TPR6I9Jhx8&amp;username=iGEM-PB2014&amp;login=true&amp;_sourcePage=FOC90vQVhF_iMUD9T65RG_YvRLZ-1eYO3fqfqRu0fynRL_1nukNa4gH1t86pc1SP#b=596d48d5-2edd-48b4-b5e9-dc81ce1b9aab&amp;st=p&amp;n=caecc826-a5e8-4b0b-b8a8-14a5aad1eb7c" target="_blank" shape="rect">
        oPB.001
    </A>
    and the reverse primer
    <A href="https://www.evernote.com/Home.action?__fp=IYz5qkyPGHc3yWPvuidLz-TPR6I9Jhx8&amp;username=iGEM-PB2014&amp;login=true&amp;_sourcePage=FOC90vQVhF_iMUD9T65RG_YvRLZ-1eYO3fqfqRu0fynRL_1nukNa4gH1t86pc1SP#b=596d48d5-2edd-48b4-b5e9-dc81ce1b9aab&amp;st=p&amp;n=340942ec-952b-4a13-8f31-df72361990e4" target="_blank" shape="rect">
        oPB.005.
    </A>
    These two oligos include the BioBrick prefix and suffix. The aim is to add the prefix and suffix to the agaA sequence and amplify it.
</DIV><DIV>
    We will set the parameters in the machine and run the reaction overnight.
</DIV><H5>June 24th</H5>
﻿<H6>Goal: PCR purification of the gBlock</H6><H6>Procedure:</H6><DIV><DIV>
        We will purify the PCR product -agaA gBlock that contains prefix-agaA-suffix- obtained on
        <A href="https://www.evernote.com/Home.action?__fp=ZloDp4AdW-c3yWPvuidLz-TPR6I9Jhx8&amp;username=iGEM-PB2014&amp;login=true&amp;_sourcePage=L90YhFqIo3HiMUD9T65RG_YvRLZ-1eYO3fqfqRu0fynRL_1nukNa4gH1t86pc1SP#b=596d48d5-2edd-48b4-b5e9-dc81ce1b9aab&amp;st=p&amp;n=d5df8100-4e36-4a75-b532-5c6ff76613ea" target="_blank" shape="rect">
            June 23rd
        </A>
        following the
        <A href="https://www.evernote.com/Home.action?__fp=ZloDp4AdW-c3yWPvuidLz-TPR6I9Jhx8&amp;username=iGEM-PB2014&amp;login=true&amp;_sourcePage=L90YhFqIo3HiMUD9T65RG_YvRLZ-1eYO3fqfqRu0fynRL_1nukNa4gH1t86pc1SP#b=60f448df-afb5-4a4d-9310-6dea624f7ab3&amp;st=p&amp;n=9dd0d893-9195-4ae2-9bbb-3748ecff72c9" target="_blank" shape="rect">
            QIAquick PCR purification kit standard protocol
        </A>
        .
    </DIV><DIV>
        We will run a 1% agarose gel with the purified DNA following the
        <A href="https://www.evernote.com/Home.action?__fp=avuu_OOUw8I3yWPvuidLz-TPR6I9Jhx8&amp;username=iGEM-PB2014&amp;login=true&amp;_sourcePage=dd1SVp2N6cjiMUD9T65RG_YvRLZ-1eYO3fqfqRu0fynRL_1nukNa4gH1t86pc1SP#b=60f448df-afb5-4a4d-9310-6dea624f7ab3&amp;st=p&amp;n=228cf621-87a8-43ca-9aa9-7e11f82d83d0" target="_blank" shape="rect">
            standard protocol
        </A>
        . Gels were made by Ihab and Antonio the 23rd of June following the same protocol.
    </DIV></DIV><H6>Results</H6><DIV><DIV><DIV>
            After running the gel, we exposed it in UV light. We could see the Gene Ruler, but there was <STRONG>no band</STRONG> in our sample.
        </DIV><DIV>
            We think that PCR did not work. Things to change:
        </DIV><UL><LI>
                Use <STRONG>fusion polymerase</STRONG>, as it is more reliable than the DreamTaq we used. DreamTaq is more for colony PCR.
            </LI><LI>
                Final concentration of 200 uL, to have more DNA
            </LI><LI>
                When PCR does not work we look at:
                </LI><LI>
                        Annealing temperature: we start typically at 52ºC and can go down to 50ºC.
                    </LI><LI>
                        DMSO concentration: typically 3%. Avoids missmatching
                    </LI><LI>
                        Extension time, according to the manufacturer
                    </LI></UL></DIV></DIV><H6>Goal:PCR agaA gBlock with oPB.001 and oPB.005</H6><H6>Procedure:</H6><DIV><DIV><DIV><DIV>
                We are going to PCR using the
                <A href="https://www.evernote.com/Home.action?__fp=ZloDp4AdW-c3yWPvuidLz-TPR6I9Jhx8&amp;username=iGEM-PB2014&amp;login=true&amp;_sourcePage=L90YhFqIo3HiMUD9T65RG_YvRLZ-1eYO3fqfqRu0fynRL_1nukNa4gH1t86pc1SP#st=p&amp;n=65588d43-faa8-48c3-9114-884c43fad4d7" target="_blank" shape="rect">
                    Fusion Polymerase
                </A>
                for the agaA gBlock with the forward primer
                <A href="https://www.evernote.com/Home.action?__fp=IYz5qkyPGHc3yWPvuidLz-TPR6I9Jhx8&amp;username=iGEM-PB2014&amp;login=true&amp;_sourcePage=FOC90vQVhF_iMUD9T65RG_YvRLZ-1eYO3fqfqRu0fynRL_1nukNa4gH1t86pc1SP#b=596d48d5-2edd-48b4-b5e9-dc81ce1b9aab&amp;st=p&amp;n=caecc826-a5e8-4b0b-b8a8-14a5aad1eb7c" target="_blank" shape="rect">
                    oPB.001
                </A>
                and the reverse primer
                <A href="https://www.evernote.com/Home.action?__fp=IYz5qkyPGHc3yWPvuidLz-TPR6I9Jhx8&amp;username=iGEM-PB2014&amp;login=true&amp;_sourcePage=FOC90vQVhF_iMUD9T65RG_YvRLZ-1eYO3fqfqRu0fynRL_1nukNa4gH1t86pc1SP#b=596d48d5-2edd-48b4-b5e9-dc81ce1b9aab&amp;st=p&amp;n=340942ec-952b-4a13-8f31-df72361990e4" target="_blank" shape="rect">
                    oPB.005.
                </A>
                These two oligos include the BioBrick prefix and suffix. The aim is to add the prefix and suffix to the agaA sequence and amplify it.
            </DIV></DIV></DIV></DIV><H5>June 25th</H5><H6>Goal: Make sure the cloning was successfull by performing analytical digestion and sequencing</H6><H6>Procedure:</H6><P><UL><LI>Liquid cultures were made out of colonies from the transformation</LI><LI>After 24h at 37°C, regular Miniprep protocol was followed</LI></UL></P><P>
1) Analytical digestion was performed as follows:
<UL><LI>4uL 10X Green Buffer</LI><LI>2uL of EcoRI</LI><LI>2uL of XbaI</LI><LI>6uL pCas9</LI><LI>26uL of dH20</LI><LI>Incubated for 30 min at 37°C in the thermocycler</LI></UL></P><P>
2) A primer was designed 200 nucleotides upstream the region were the sequenced was changed
<UL><LI>The miniprep plasmid and the primer were sent to a sequencing company</LI></UL></P><H6>Results:</H6><P>
After gel running, it appears that the analytical digestion produced the right band sizes.
Moreover, the sequencing results were matching the expected sequence.
</P><H6>Goal: PCR purification of the gBlock</H6><H6>Procedure:</H6><DIV><DIV><DIV><DIV><DIV>
                    We will purify the PCR product -agaA gBlock that contains prefix-agaA-suffix- obtained on
                    <A href="https://www.evernote.com/Home.action#b=596d48d5-2edd-48b4-b5e9-dc81ce1b9aab&amp;st=p&amp;n=573c733b-ffca-46e9-9f73-d51ec730a15f" target="_blank" shape="rect">
                        June 24th
                    </A>
                    following the
                    <A href="https://www.evernote.com/Home.action?__fp=ZloDp4AdW-c3yWPvuidLz-TPR6I9Jhx8&amp;username=iGEM-PB2014&amp;login=true&amp;_sourcePage=L90YhFqIo3HiMUD9T65RG_YvRLZ-1eYO3fqfqRu0fynRL_1nukNa4gH1t86pc1SP#b=60f448df-afb5-4a4d-9310-6dea624f7ab3&amp;st=p&amp;n=9dd0d893-9195-4ae2-9bbb-3748ecff72c9" target="_blank" shape="rect">
                        QIAquick PCR purification kit standard protocol
                    </A>
                    .
                </DIV><DIV>
                    We will run a 1% agarose gel with the purified DNA following the
                    <A href="https://www.evernote.com/Home.action?__fp=avuu_OOUw8I3yWPvuidLz-TPR6I9Jhx8&amp;username=iGEM-PB2014&amp;login=true&amp;_sourcePage=dd1SVp2N6cjiMUD9T65RG_YvRLZ-1eYO3fqfqRu0fynRL_1nukNa4gH1t86pc1SP#b=60f448df-afb5-4a4d-9310-6dea624f7ab3&amp;st=p&amp;n=228cf621-87a8-43ca-9aa9-7e11f82d83d0" target="_blank" shape="rect">
                        standard protocol
                    </A>
                    . Gels were made by Ihab and Antonio the 23rd of June following the same protocol.
                </DIV></DIV></DIV></DIV></DIV><H6> Results: </H6><DIV>
    - <STRONG>1st raw:</STRONG>Gene Ruler 100 kb + 1uL Loading Dye
</DIV><DIV>
    - <STRONG>2nd and 3rd raw</STRONG>: purified PCR products + 1 uL of Loading Dye per 5 uL of sample. We saw the samples did not sink properly in the hole,
    so we tried using more Loading Dye (raw 2 is tube A, raw 3 is tube B)
</DIV><DIV>
    - <STRONG>4th and 5th raw:</STRONG> 2 uL of Loading Dye for 5uL of sample (raw 4 is tube A, raw B is tube B)
</DIV><DIV>
    We can see our construct around the 1300 bp line, so we know is the gBlock.
</DIV><DIV>
    Me measured the DNA in the samples using the Nanodrop.
</DIV><DIV>
    - Tube A: 100 ng/uL
</DIV><DIV>
    - Tube B: 55 ng/uL. This is a little low. It is an unexpected result considering that we pepared both tubes at the same time and following the same
    protocol.
    </DIV><H5>June 26th</H5>
﻿<H6>Goal:  Obtain agaA Biobrick (agaA gBlock digestion + ligation)</H6><H6>Procedure:</H6><DIV>
We will digest the purified agaA gBlock following the    <A href="http://www.thermoscientificbio.com/uploadedFiles/Resources/fast-digestion-dna.pdf" target="_blank" shape="rect">Thermo Fast Digestion Protocol</A>
    for enzimes <STRONG>EcoRI and PstI.</STRONG> Then we will ligate it with the linearized <STRONG>vector pSB1C3</STRONG>. We want to obtain
    <A href="https://www.evernote.com/Home.action?__fp=ZloDp4AdW-c3yWPvuidLz-TPR6I9Jhx8&amp;amp;username=iGEM-PB2014&amp;amp;login=true&amp;amp;_sourcePage=L90YhFqIo3HiMUD9T65RG_YvRLZ-1eYO3fqfqRu0fynRL_1nukNa4gH1t86pc1SP#b=596d48d5-2edd-48b4-b5e9-dc81ce1b9aab&amp;amp;st=p&amp;amp;n=c359be34-a042-4d01-a0bd-ebb7b81f3166" target="_blank" shape="rect">
       agaA Biobrick
    </A>
    and transform E coli.
</DIV><DIV><STRONG>Digestion</STRONG></DIV><DIV>
    1) We will use tube A (100 ng/uL). We added 20 uL of water + 5 uL Buffer + 20 uL DNA + 2.5 uL EcoRI + 2.5 uL PstI. We digested longer than indicated,
    around 40 minutes, as we have used these enzymes before and work better this way.
</DIV><DIV>
    2) Then we put it at 37ºC at the Isotemp for 5 min.
</DIV><DIV>
    3) We will repeat for PSB1C3
</DIV><DIV>
    These tubes were called Digested gBlock and Digested PSB1C3
</DIV><DIV><STRONG>Purification</STRONG></DIV><DIV>
    4) We will purify both tubes using the
    <A href="https://www.evernote.com/Home.action?__fp=ZloDp4AdW-c3yWPvuidLz-TPR6I9Jhx8&amp;amp;username=iGEM-PB2014&amp;amp;login=true&amp;amp;_sourcePage=L90YhFqIo3HiMUD9T65RG_YvRLZ-1eYO3fqfqRu0fynRL_1nukNa4gH1t86pc1SP#b=60f448df-afb5-4a4d-9310-6dea624f7ab3&amp;amp;st=p&amp;amp;n=9dd0d893-9195-4ae2-9bbb-3748ecff72c9" target="_blank" shape="rect">
        QIAquick kit
    </A>
    .
</DIV><DIV>
    These tubes were called Dig + Pur gBlock and Dig + Pur PSB1C3
</DIV><DIV><STRONG>Ligation</STRONG></DIV><DIV>﻿Following the    <A href="http://www.thermoscientificbio.com/uploadedfiles/resources/el001-product-information.pdf" target="_blank" shape="rect">Thermo</A><STRONG><A href="http://www.thermoscientificbio.com/uploadedfiles/resources/el001-product-information.pdf" target="_blank" shape="rect">
            T4 DNA Ligase protocol
        </A>
        .
    </STRONG>
    Instead of leaving the tube at 22ºC for 10 min, we left it at room temperature for 45 min.
</DIV><DIV>
    5) We will make a ligation. We will make a <STRONG>1:3</STRONG> proportion for the gBlock:vector. We added 3,3 uL of the linearized vector [25 ng/mL] and
    2,5 uL of the insert DNA [100 ng/mL].
</DIV><DIV><STRONG>Purification</STRONG></DIV><DIV>
    6) We will purify the product using the
    <A href="https://www.evernote.com/Home.action?__fp=ZloDp4AdW-c3yWPvuidLz-TPR6I9Jhx8&amp;amp;username=iGEM-PB2014&amp;amp;login=true&amp;amp;_sourcePage=L90YhFqIo3HiMUD9T65RG_YvRLZ-1eYO3fqfqRu0fynRL_1nukNa4gH1t86pc1SP#b=60f448df-afb5-4a4d-9310-6dea624f7ab3&amp;amp;st=p&amp;amp;n=9dd0d893-9195-4ae2-9bbb-3748ecff72c9" target="_blank" shape="rect">
        QIAquick kit
    </A>
    .
</DIV><DIV>
    This tube was called pPB.001
</DIV><H6>Goal: Transform E coli with agaA Biobrick</H6><H6>Procedure:</H6><DIV>
    We will transform competent E coli that were previously made by Jake following his own protocol. We will use
    <A href="https://www.evernote.com/Home.action#b=596d48d5-2edd-48b4-b5e9-dc81ce1b9aab&amp;amp;st=p&amp;amp;n=c359be34-a042-4d01-a0bd-ebb7b81f3166" target="_blank" shape="rect">
        agaA Biobrick
    </A>
    .
    </DIV><DIV>
    We will follow this
    <A href="https://www.evernote.com/Home.action?__fp=ZloDp4AdW-c3yWPvuidLz-TPR6I9Jhx8&amp;amp;username=iGEM-PB2014&amp;amp;login=true&amp;amp;_sourcePage=L90YhFqIo3HiMUD9T65RG_YvRLZ-1eYO3fqfqRu0fynRL_1nukNa4gH1t86pc1SP#b=60f448df-afb5-4a4d-9310-6dea624f7ab3&amp;amp;st=p&amp;amp;n=b48f12a9-a15f-45a2-a179-095516e07b3e" target="_blank" shape="rect">
        Heat Shock protocol
    </A>
    .
    </DIV><DIV>
    We will plate them in LBA+Cm. We will add 250 uL of Cm for 250 mL of LBA.
    </DIV><H6>Results:</H6><P>No colonies grew after 12h. We will leave them in the incubator some more time and repeat the Heat Shock in the meanwhile</P><H5>June 27th</H5>
﻿<H6>Goal: <I>E coli</I> transformation with agaA Biobrick</H6><H6>Procedure:</H6><DIV>
    Because the plates cultured on
    <A href="https://www.evernote.com/Home.action#b=596d48d5-2edd-48b4-b5e9-dc81ce1b9aab&amp;st=p&amp;n=db4278a7-4436-4a96-83c8-a23d6c57c65b" target="_blank" shape="rect">
        June 26th
    </A>
have no colonies after 12h, we have decided to repeat the heat shock transformation. The mistake might be that we are using Cloroamphenicol, and we    <STRONG>did not recover them for enough time</STRONG>. We recovered them during 50 min in the thermocycler.
</DIV><DIV>
    Today we have repeated the same
    <A href="https://www.evernote.com/Home.action#b=60f448df-afb5-4a4d-9310-6dea624f7ab3&amp;st=p&amp;n=b48f12a9-a15f-45a2-a179-095516e07b3e" target="_blank" shape="rect">
        Heat Shock protocol
    </A>
    , but we have recovered them during 2h in the incubator. The cells used were a new stock from Jake's competent E coli. The plasmid used is
    <A href="https://www.evernote.com/Home.action#b=596d48d5-2edd-48b4-b5e9-dc81ce1b9aab&amp;st=p&amp;n=c359be34-a042-4d01-a0bd-ebb7b81f3166" target="_blank" shape="rect">
        agaA Biobrick
    </A>
    .
</DIV><DIV>
    Then we will plate them in LBA+Cm
</DIV><H6>Results:</H6><DIV><DIV>
        We still have no colonies in the plates.
    </DIV><DIV>
        What might not work:
    </DIV><DIV>
        - Enzimes (EcoRI and PstI) work, as Ihab and Antonio have tested them. However I am cutting a linearized plasmid, so if their efficiency is not very
        high, they might not work for me.
    </DIV><DIV>
        - Ligase.
        </DIV><DIV>
        - Transformation. I doubt it
    </DIV><DIV>
        - Primers are not well designed
    </DIV></DIV><H5>June 29th</H5>
﻿<H6>Goal: Start culture to extract plasmid</H6><H6>Procedure:</H6><DIV><DIV><DIV>
            We will start a culture to extract the plasmid PSB6A1 (Matt's stock, 2013). The aim is to cut a circularized plasmid with EcoRI and PstI and check
            their efficiency.
            </DIV><DIV>
            We cultured the cells in LBA+Amp at 37ºC
        </DIV></DIV></DIV><H5>June 30th</H5>
﻿<H6>Goal: Miniprep to extract a plasmid (unfinished)</H6><H6>Procedure:</H6><P> We started a culture on June 29th. We will make a mini prep following the Mini Prep protocol. The aim is to extract the plasmid PSB6A1 to cut it with EcoRI and PstI and see the efficiency of the cut. </P><H6> Results: </H6><DIV><DIV><DIV><DIV>
                We have found out that Antonio and
                <A href="https://www.evernote.com/Home.action#b=fa182290-7cce-4560-82ad-0a0bcbd62a13&amp;st=p&amp;n=f48e633a-a1d8-455d-b6ca-6b6c67b9716f" target="_blank" shape="rect">
                    Ihab
                </A>
                have already tested PstI and EcoRI. They seem to <STRONG>work quite specifically</STRONG> .
                </DIV><DIV>
                Also, they haven't managed to construct the plasmid, or do the
                <A href="https://www.evernote.com/Home.action#b=fa182290-7cce-4560-82ad-0a0bcbd62a13&amp;st=p&amp;n=f6e77f48-48d4-409f-b6a4-f7be98430daa" target="_blank" shape="rect">
                    Golden Gate assembly
                </A>
                . We believe this is due to the <STRONG>Ligase and/or the Ligation buffer.</STRONG></DIV><DIV>
                Therefore, we will use a <STRONG>new enzyme and buffer</STRONG>, ligate and transform.
                </DIV><DIV>
                Most enzymes from last year do not seem to work. We shoud not use them and wait for the new ones.
            </DIV></DIV></DIV></DIV></DIV><DIV id="July"><H4>July</H4><H5>July 1st</H5><H6>Goal: PCR purification of the gBlock</H6><H6>Procedure:</H6><H5>July 1st</H5><I>** pEC-XC99E plasmids (Cm) from the Beilfield iGEM team arrived today. We stored them at 37ºC and Pierre-Luc will make a miniprep tomorrow **</I><H6>Goal: Obtain agaA Biobrick</H6><H6>Procedure:</H6><DIV>
    We will use the digested agaA gBlock and PSB1C3 obtained on
    <A href="https://www.evernote.com/Home.action#b=596d48d5-2edd-48b4-b5e9-dc81ce1b9aab&amp;st=p&amp;n=db4278a7-4436-4a96-83c8-a23d6c57c65b" target="_blank" shape="rect">
        June 26th
    </A>
    .
    </DIV><DIV><DIV><STRONG>Ligation</STRONG></DIV><DIV>
        We will use a <STRONG>new Ligase</STRONG> from Promega (Jake's stock) for this experiment, as the former one (from iGEM's 2013 stock, Thermo) did not
        seem to work.
        </DIV><DIV>
        We will follow the
        <A href="http://http//www.promega.com/~/media/files/resources/protocols/product%20information%20sheets/g/t4%20dna%20ligase%20blue%20white%20cloning%20qualified%20protocol.pdf" target="_blank" shape="rect">
            Promega T4 DNA Ligase protocol.
        </A>
        It is recommended to leave the tubes at room temperature for 3h but instead we did it for 2h.
    </DIV><DIV>
        We will make a <STRONG>1:3</STRONG> proportion for the gBlock:vector. We added 3,3 uL of the linearized vector [25 ng/mL] and 2,5 uL of the insert DNA
        [100 ng/mL].
    </DIV><DIV><STRONG>Purification</STRONG></DIV><DIV>
        We will purify the product using the
        <A href="https://www.evernote.com/Home.action?__fp=ZloDp4AdW-c3yWPvuidLz-TPR6I9Jhx8&amp;username=iGEM-PB2014&amp;login=true&amp;_sourcePage=L90YhFqIo3HiMUD9T65RG_YvRLZ-1eYO3fqfqRu0fynRL_1nukNa4gH1t86pc1SP#b=60f448df-afb5-4a4d-9310-6dea624f7ab3&amp;st=p&amp;n=9dd0d893-9195-4ae2-9bbb-3748ecff72c9" target="_blank" shape="rect">
            QIAquick kit
        </A>
        .
        </DIV><DIV><EM><STRONG>Tip:</STRONG>
            It is better to <STRONG>purify with water</STRONG>, as the Ligase might be sensitive to the Elution Buffer (EB), that has a lot of salts on it.
            Last time (June 26th), we did use the EB Buffer.
            </EM></DIV><DIV>
        This tube was called pPB.001
    </DIV></DIV><H6>Goal: Transform <I>E coli</I> with agaA Biobrick</H6><H6>Procedure:</H6><DIV><DIV>
        We will transform competent NEB turbo E coli made following the
        <A href="https://www.evernote.com/Home.action#b=60f448df-afb5-4a4d-9310-6dea624f7ab3&amp;st=p&amp;n=b4939a60-0f83-446a-a111-39d63eb7589c" target="_blank" shape="rect">
            standard protocol
        </A>
        . In step 5, we did not incubate them for 4 hours, but only until the density was optimal (around 30-45 min).
    </DIV><DIV>
        We will use
        <A href="https://www.evernote.com/Home.action#b=596d48d5-2edd-48b4-b5e9-dc81ce1b9aab&amp;st=p&amp;n=c359be34-a042-4d01-a0bd-ebb7b81f3166" target="_blank" shape="rect">
            agaA Biobrick
        </A>
        .
    </DIV><DIV>
        We will follow this
        <A href="https://www.evernote.com/Home.action?__fp=ZloDp4AdW-c3yWPvuidLz-TPR6I9Jhx8&amp;username=iGEM-PB2014&amp;login=true&amp;_sourcePage=L90YhFqIo3HiMUD9T65RG_YvRLZ-1eYO3fqfqRu0fynRL_1nukNa4gH1t86pc1SP#b=60f448df-afb5-4a4d-9310-6dea624f7ab3&amp;st=p&amp;n=b48f12a9-a15f-45a2-a179-095516e07b3e" target="_blank" shape="rect">
            Heat Shock protocol
        </A>
        .
        </DIV><DIV><EM><STRONG>Tip</STRONG>
            (From Matt): At the last step when we recover the cells, plate most of the 200 uL and leave the rest recovering overnight. Then plate them the next
            day.
            </EM></DIV></DIV><DIV>
    We will plate them in LBA+Cm. We will add 250 uL of Cm for 250 mL of LBA.
    </DIV><H6>Results:</H6><DIV>
    We are still not able to transform the cells. We went back to check the primers, and the oPB.001 that we have in the database and the one we commanded are
    not the same.
    </DIV><DIV>
    We will re-do this process with the good oPB.001 when it arrives. At least now we have learnt all the techniques. Always re-check your plasmids, many
    times.
</DIV><H5>July 3rd: Study the Strains</H5><H6>Goal: Make sure the studied strains EC307(+) and EC307(-) that were given by a collegue scientist have the right sequence. This was performed by colony PCR and sequencing </H6><H6>Procedure</H6><P><UL><LI>Primers were designed with Geneious, 200 nucleotides upstream and dowstream of the targeted sequence of LacZ.</LI><LI>4 colony PCRs were performed : 2 on EC307(+) colonies and 2 on EC307(-) colonies.</LI>The PCR products were sent for sequencing.</UL></P><H6>Results</H6><P>The sequencing results matched the expected sequences.</P><H5 id="date">July 4th</H5>

﻿<H6>Goal: Design primers for smelling genes</H6><H6>Procedure:</H6></DIV>
Some genes responsible of odor belong to alternative fermentative pathway like: ackA, ldH, l-ldH, d-ldH
We want to do different kind of sequencing for these genes. It's wy we want to design primers in order to get 500n pcr products.



<DIV><H5 id="date">July 2nd</H5><H6>Goal: Miniprep to extract pSEVA351 + Glycerol stock</H6><H6>Procedure:</H6><DIV>
    We recieved a tube that contains pSEVA351 (sent by the <A href="http://seva.cnb.csic.es/SEVA/Welcome.html" target="_blank" shape="rect">CSIC</A>).We
    cultured it the night before, and extract a colony. We put a mixture of this colony + 5ml of LB medium in 2 different tubes.
    </DIV><DIV>
    We will let it grow overnight at 37ºC.
    </DIV><DIV>
    -<STRONG> Tube 1:</STRONG> miniprep following the standard protocol. This tube will be pSEVA351 and it is in my own stock.
    </DIV><DIV>
    - <STRONG>Tube 2 :</STRONG> glycerol stock: 250 uL of glycerol 60% + 750 uL of the culture. We will have both a -20ºC and -80ºC stock.
    </DIV><DIV>
    This is the vector we will use to construct
    <A href="https://www.evernote.com/Home.action#b=596d48d5-2edd-48b4-b5e9-dc81ce1b9aab&amp;st=p&amp;n=c411c8c1-73e6-4f3f-835c-edc7b8826530" target="_blank" shape="rect">
        agaA in pSEVA
    </A>
    .
</DIV><H5 id="date">July 4th</H5>

﻿<H6>Goal: Design primers for smelling genes</H6><H6>Procedure:</H6></DIV>
In addition to agaA, some genes responsible of odor belong to alternative fermentative pathway like: ackA, ldH, l-ldH, d-ldH
We want to do different kind of sequencing for these genes. It's wy we want to design primers in order to get 500n pcr products.

<TABLE border=""><TBODY><TR><TD><B>Name</B></TD><TD><B>Sequence (5'-&gt;3')</B></TD><TD><B>Notes</B></TD><TD><B>Strain</B></TD>/TR&gt;
</TR><TR><TD><B>oPB.046</B></TD><TD><B>AGATTGTAGATAGTCTCGAGTCCTCC</B></TD><TD><B>agaA forward for primer 1</B></TD><TD><B>Corynebacterium Striatum</B></TD></TR><TR><TD><B>oPB.047</B></TD><TD><B>GTCTTCGTTGAACACGTGCTGGG</B></TD><TD><B>agaA reverse for primer 1</B></TD><TD><B>Corynebacterium Striatum</B></TD></TR><TR><TD><B>oPB.048</B></TD><TD><B>CAAGTGGTTCCACCAGCACCC</B></TD><TD><B>agaA forward primer 2</B></TD><TD><B>Corynebacterium Striatum</B></TD></TR><TR><TD><B>oPB.049</B></TD><TD><B>ACGCCGCCGTTAGAAACCGG</B></TD><TD><B>agaA reverse for primer 2</B></TD><TD><B>Corynebacterium Striatum</B></TD></TR><TR><TD><B>oPB.066</B></TD><TD><B>GTAAATGCTGGTAGCTCATCTTTAAAGTTCC</B></TD><TD><B>ackA forward</B></TD><TD><B>Staphylococcus epidermidis</B></TD></TR><TR><TD><B>oPB.067</B></TD><TD><B>CGTATGCCATAGTCTTCATAGTAATGATATGG</B></TD><TD><B>ackA reverse</B></TD><TD><B>Staphylococcus epidermidis</B></TD></TR><TR><TD><B>oPB.068</B></TD><TD><B>AGAAGGACATGGCATTACGTTGGG</B></TD><TD><B>d-ldH forward</B></TD><TD><B>Staphylococcus epidermidis</B></TD></TR><TR><TD><B>oPB.069</B></TD><TD><B>AGGATATGCATCATAACCAACTACTCTCG</B></TD><TD><B>d-ldH reverse</B></TD><TD><B>Staphylococcus epidermidis</B></TD></TR><TR><TD><B>oPB.070</B></TD><TD><B>GATGGTTTCTTCTTAATTGCTGCAAACCC</B></TD><TD><B>l-ldH forward</B></TD><TD><B>Staphylococcus epidermidis</B></TD></TR><TR><TD><B>oPB.071</B></TD><TD><B>CCATTTTGATTGATAAGTGTTGGTAAGCC</B></TD><TD><B>l-ldH reverse</B></TD><TD><B>Staphylococcus epidermidis</B></TD></TR><TR><TD><B>oPB.036</B></TD><TD><B>AGTGTCACCATGTTCACCAA</B></TD><TD><B>ldH forward</B></TD><TD><B>Staphylococcus aureus</B></TD></TR><TR><TD><B>oPB.037</B></TD><TD><B>CAGTAGGTTCAAGCTACGCA</B></TD><TD><B>ldH reverse</B></TD><TD><B>Staphylococcus aureus</B></TD></TR></TBODY></TABLE><H5 id="date">July 7th</H5><P id="text"><U>1) odor exctraction </U></P>


Protocol:
<P>Samples: Jalke/Henry/Alexandre cotton pads (of the 01/07)(see sweat collection)
(1)  Cut a small piece of sample from Alex (A), put it in an eppendorf tube an add 600ul of PBS. 
Vortex during 60 secondes. 
Spin 1 minutes at 1000rpm (to put the cotton at the bottom). 
Take supernatant with a 1ml syringe 
Filter it (0.2ul)</P>

FAILED: Not enought liquid to be used with a 0.20ul filter we've got<P>
(2) Cut a small piece of sample from Alex (A), put it in an eppendorf tube an add 4000ul of water. 
Vortex during 60 secondes. 
Spin 1 minutes at 1000rpm (to put the cotton at the bottom). 
Take supernatant with a 10ml syringe 
Filter it (0.2ul)
==&gt; Smell it: The liquid has a very subtil smell (compare with the patch which have a very strong smell) Maybe the patche was too small. </P><P>
(3)Cut a big piece of sample from Alex (1/4) (A), put it in an eppendorf tube an add 4000ul of water. 
Vortex during 60 secondes. 
Spin 1 minutes at 1000rpm (to put the cotton at the bottom). 
Take supernatant with a 10ml syringe 
Filter it (0.2ul)
==&gt; Smell it: The liquid has a very subtil smell (compare with the patch which have a very strong smell). It is very slightly stronger than the (2) but it is not enought. 

Results:
Sweat filtration doesn't seems to be efficient enough to capture the smell. </P><H5>July 7th: Crispr Works</H5><H6>Goal: Determine is the designed CRISPR/Cas systems produce the expected results regarding bacteria survival (both of the strains should survive the non targeting CRISPR, and the mutant strain should survive to the LacZ targeting CRISPR more than the wild type).</H6><H6>Procedure</H6><P>The strains were transformed by regular heat shock protocol (with 30 seconds at 42°C), either by the non targeting CRISPR or by the LacZ targeting CRISPR.
The transformations products were plated on chloramphenicol plates at 37°C for 24h.</P><H6>Results</H6><P>Here we report the number of colonies of counting</P><TABLE class="tableizer-table"><TBODY><TR class="tableizer-firstrow"><TH>Non targeting CRISPR</TH><TH>LacZ targeting CRISPR</TH></TR><TR><TD>Wild Type</TD><TD>145</TD><TD>3</TD></TR><TR><TD>Mutant</TD><TD>132</TD><TD>44</TD></TR></TBODY></TABLE><H5 id="date">July 8th</H5><P>Goal:
Capture the smell (volatiles compounds)﻿ in a patch. This has to be sterile, the odour has to be strong and that can stay for 5 days at least. </P><P>
Protocol: 
Samples: Jalke/Henry/Alexandre cotton pads (of the 01/07)(see sweat collection)
We used a protocol from: &quot;Regulation of ovulation
by human pheromones&quot; and &quot;Olfactory inﬂuences on the human menstrual cycle&quot;

Cut a medium piece form patch (from sample A). Put 500ul of ethanol (70%) and vortex (60 secondes) put immediatly the tube at -20°c. 
Wait for it to be frozen (2h) 

Take it out of freezer, wait 2-3 mins. 
Then all the ethanol should be evaporated, rub the piece on a new clean pad. 
Wait until the ethanol smell dissapear. 
Smell it
Let the new cotton pad overnight in a box, smell it
Results
The cotton pad is smelling like sweat, the smell is quite strong (7 people smelled it and recognized a strong sweat odour) but there is also a small smell of ethanol. 
The longer you wait, smaller is the ethanol smell. 
The smell completely disappear overnight. </P><H5 id="date">July 10th</H5>
Goal:
Capture the smell (volatiles compounds)﻿ in a patch. This has to be sterile, the odour has to be strong and that can stay for 5 days at least. Use a solvent exctraction technic found in litterature. <P>Protocol: 
Samples: Jalke/Henry/Alexandre cotton pads (of the 01/07)(see sweat collection)
We used a protocol from: &quot;Regulation of ovulation
by human pheromones&quot; and &quot;Olfactory inﬂuences on the human menstrual cycle&quot;

Cut a medium piece form patch (from sample 1/2/3(3 is a whole pad)). Put 3ml of ethanol (70%) and vortex (60 secondes) put immediatly the tube at -20°c. (I've increased the amount of ethanol to be sure that the pad is sterilised) 
I had a control: PAD frozen without ethanol.
Wait for it to be frozen (2h) 

Take it out of freezer, wait 2-3 mins. 
(1)Then all the ethanol should be evaporated, rub the piece on a new clean pad. 
Wait until the ethanol smell dissapear. 
Smell it
Let the new cotton pad overnight in a box, smell it


(2)When ethanol is evaporated, put the pad in a close falcon tube


(3)Then all the ethanol should be evaporated, rub the piece on a new clean pad. 
Wait until the ethanol smell dissapear. 
Smell it
Let the new cotton pad overnight in a box, smell it</P><P>Results
houldThe cotton pad smell too much like ethanol, we should find a perfect value for the amount of ethanol putted. Then the solvent exctraction look complicated, because it appear that the smell is going away quite fast, also it is not very good that we have to keep the pad, because when people will smell it, they will directly recognize a cotton pad.</P><H5>July 10th: Crispr efficiency</H5><H6>Goal</H6><P>In order assess the discrimination properties of the designed CRISPR/Cas system, the strains were transformed by electroporation (to have more colonies and therefore a more trustworthy counting of the colonies) in 5 replicates.</P><H6>Procedure</H6><P><UL><LI>The strains were washed with sterile water and resuspended in 1 mL after the final washing.</LI><LI>The electroporation was performed following the EC2 settings, 200uL were transformed with 2uL of dialysed miniprep.</LI><LI>2h of recover followed the transformation.</LI><LI>Cells were plated on chloramphenicol plates for 24h at 37°C.</LI></UL></P><H6>Results</H6><P>Here are reported the number of colonies after counting:</P><TABLE class="tableizer-table"><TBODY><TR class="tableizer-firstrow"><TH>WT with LacZ Crispr</TH><TH>Mutant with LacZ Crispr</TH><TH>WT with Random Crispr</TH><TH>Mutant with Random Crispr</TH></TR><TR><TD>Replicate n°1</TD><TD>5</TD><TD>742</TD><TD>1845</TD><TD>1745</TD></TR><TR><TD>Replicate n°2</TD><TD>40</TD><TD>543</TD><TD>2398</TD><TD>2354</TD></TR><TR><TD>Replicate n°3</TD><TD>12</TD><TD>984</TD><TD>1549</TD><TD>1890</TD></TR><TR><TD>Replicate n°4</TD><TD>2</TD><TD>659</TD><TD>1740</TD><TD>1754</TD></TR><TR><TD>Replicate n°5</TD><TD>26</TD><TD>978</TD><TD>2176</TD><TD>1956</TD></TR></TBODY></TABLE><H5 id="date">July 11th</H5><H6>Goal:  PCR of agaA gBlock with oPB.001 and oPB.005</H6><H6>Procedure:</H6><P>
    We are going to PCR using the
</P><A href="https://www.evernote.com/Home.action?__fp=ZloDp4AdW-c3yWPvuidLz-TPR6I9Jhx8&amp;username=iGEM-PB2014&amp;login=true&amp;_sourcePage=L90YhFqIo3HiMUD9T65RG_YvRLZ-1eYO3fqfqRu0fynRL_1nukNa4gH1t86pc1SP#st=p&amp;n=65588d43-faa8-48c3-9114-884c43fad4d7" target="_blank" shape="rect">
    Fusion Polymerase
</A><P>
    for the agaA gBlock with the forward primer
</P><A href="https://www.evernote.com/Home.action?__fp=IYz5qkyPGHc3yWPvuidLz-TPR6I9Jhx8&amp;username=iGEM-PB2014&amp;login=true&amp;_sourcePage=FOC90vQVhF_iMUD9T65RG_YvRLZ-1eYO3fqfqRu0fynRL_1nukNa4gH1t86pc1SP#b=596d48d5-2edd-48b4-b5e9-dc81ce1b9aab&amp;st=p&amp;n=caecc826-a5e8-4b0b-b8a8-14a5aad1eb7c" target="_blank" shape="rect">
    oPB.001
</A><P>
    (the good one this time!) and the reverse primer
</P><A href="https://www.evernote.com/Home.action?__fp=IYz5qkyPGHc3yWPvuidLz-TPR6I9Jhx8&amp;username=iGEM-PB2014&amp;login=true&amp;_sourcePage=FOC90vQVhF_iMUD9T65RG_YvRLZ-1eYO3fqfqRu0fynRL_1nukNa4gH1t86pc1SP#b=596d48d5-2edd-48b4-b5e9-dc81ce1b9aab&amp;st=p&amp;n=340942ec-952b-4a13-8f31-df72361990e4" target="_blank" shape="rect">
    oPB.005.
</A><P>
    These two oligos include the BioBrick prefix and suffix. The aim is to add the prefix and suffix to the agaA sequence and amplify it.
</P><H5 id="date">July 12th and 13th</H5><H6>Goal: PCR purification of the gBlock</H6><H6>Procedure:</H6><DIV>
    We will purify the PCR product -agaA gBlock that contains prefix-agaA-suffix- obtained on July 11th following the
    <A href="https://www.evernote.com/Home.action?__fp=ZloDp4AdW-c3yWPvuidLz-TPR6I9Jhx8&amp;username=iGEM-PB2014&amp;login=true&amp;_sourcePage=L90YhFqIo3HiMUD9T65RG_YvRLZ-1eYO3fqfqRu0fynRL_1nukNa4gH1t86pc1SP#b=60f448df-afb5-4a4d-9310-6dea624f7ab3&amp;st=p&amp;n=9dd0d893-9195-4ae2-9bbb-3748ecff72c9" target="_blank" shape="rect">
        QIAquick PCR purification kit standard protocol
    </A>
    .
</DIV><DIV>
    We will run a 1% agarose gel at 50V with the purified DNA following the
    <A href="https://www.evernote.com/Home.action?__fp=avuu_OOUw8I3yWPvuidLz-TPR6I9Jhx8&amp;username=iGEM-PB2014&amp;login=true&amp;_sourcePage=dd1SVp2N6cjiMUD9T65RG_YvRLZ-1eYO3fqfqRu0fynRL_1nukNa4gH1t86pc1SP#b=60f448df-afb5-4a4d-9310-6dea624f7ab3&amp;st=p&amp;n=228cf621-87a8-43ca-9aa9-7e11f82d83d0" target="_blank" shape="rect">
        standard protocol
    </A>
    .
    </DIV><DIV>
    The gel was put in a solution of Ethidium Bromide for 10 minutes.
</DIV><H6>Results:</H6><P>
    -
</P><STRONG>1st raw: </STRONG><P>
    Ready-to-use Gene Ruler (includes Loading Dye) 100 kb (5 uL)*
</P><DIV><DIV>
        - <STRONG>2nd raw</STRONG>: 5 uL purified PCR products + 1 uL of Loading Dye per 5 uL of sample (PCR product obtained the 11th of July and purified the
        12th of July).
    </DIV><DIV>
        This is strange. I am confident that my PCR product is fine. I will repeat it with a different Gene
    </DIV></DIV><P>
    -
</P><STRONG>1st raw: </STRONG><P>
    Ready-to-use Gene Ruler (includes Loading Dye) 100 kb (5 uL)*
</P><DIV>
    - <STRONG>2nd raw</STRONG><STRONG>: </STRONG>4uL Gene Ruler + 1 uL Loading Dye 100 kb*
</DIV><DIV>
    -<STRONG> 3rd raw</STRONG>: 5 uL purified PCR products + 1 uL of Loading Dye per 5 uL of sample (PCR product obtained the 11th of July and purified the
    12th of July).
</DIV><DIV><EM>
        *We used these two because Ready-to-Use seems not to work fine.
        </EM></DIV><DIV><DIV><DIV><DIV>
                We see that it was the 'Ready-to-Use Gene Ruler' that did not work properly. Our PCR worked fine and our construct is about 300 bp. Even if
                this is not at all a good image, as we are only trying to check if the PCR runned properly I will not repeat it. We saw in the other gel that
                there is a clear band. In this gel we can see that this band is 300 bp.
                </DIV></DIV></DIV></DIV><H6>Goal:Obtain agaA Biobrick</H6><H6>Procedure:</H6><DIV>
We will digest the purified agaA gBlock following the    <A href="http://www.thermoscientificbio.com/uploadedFiles/Resources/fast-digestion-dna.pdf" target="_blank" shape="rect">Thermo Fast Digestion Protocol</A>
    for enzimes <STRONG>EcoRI and PstI.</STRONG> Then we will ligate it with the linearized <STRONG>vector pSB1C3</STRONG>. We want to obtain
    <A href="https://www.evernote.com/Home.action?__fp=ZloDp4AdW-c3yWPvuidLz-TPR6I9Jhx8&amp;amp;username=iGEM-PB2014&amp;amp;login=true&amp;amp;_sourcePage=L90YhFqIo3HiMUD9T65RG_YvRLZ-1eYO3fqfqRu0fynRL_1nukNa4gH1t86pc1SP#b=596d48d5-2edd-48b4-b5e9-dc81ce1b9aab&amp;amp;st=p&amp;amp;n=c359be34-a042-4d01-a0bd-ebb7b81f3166" target="_blank" shape="rect">
        agaA Biobrick
    </A>
    and transform E coli.
</DIV><DIV>
    We measured the purified aga gBlock in the nanodrop, the concentration was 50 ng/mL. This is low. I do not know if it is low because I am measuring the
    purification product instead of the direct PCR product.
</DIV><DIV><STRONG>Digestion</STRONG></DIV><DIV>
    1) We added 20 uL of water + 5 uL Buffer + 20 uL DNA + 2.5 uL EcoRI + 2.5 uL PstI.
</DIV><DIV>
    2) Then we put it at 37ºC at the Isotemp for 5 min.
</DIV><DIV>
    This tubes was named Digested agaA gBlock
</DIV><DIV><STRONG>Purification</STRONG></DIV><DIV>
    4) We will purify the tube using the
    <A href="https://www.evernote.com/Home.action?__fp=ZloDp4AdW-c3yWPvuidLz-TPR6I9Jhx8&amp;amp;username=iGEM-PB2014&amp;amp;login=true&amp;amp;_sourcePage=L90YhFqIo3HiMUD9T65RG_YvRLZ-1eYO3fqfqRu0fynRL_1nukNa4gH1t86pc1SP#b=60f448df-afb5-4a4d-9310-6dea624f7ab3&amp;amp;st=p&amp;amp;n=9dd0d893-9195-4ae2-9bbb-3748ecff72c9" target="_blank" shape="rect">
        QIAquick kit
   </A>
    .
</DIV><DIV>
    This tube was called Dig + Pur agaA gBlock
</DIV><DIV><STRONG>Ligation</STRONG></DIV><DIV>
    We ligated Dig+Pur PSB1C﻿3 (June 26th) and Dig+Pur agaA gBlock (July 12th)
</DIV><DIV>
    Following the <A href="http://www.thermoscientificbio.com/uploadedfiles/resources/el001-product-information.pdf" target="_blank" shape="rect">Thermo</A><STRONG><A href="http://www.thermoscientificbio.com/uploadedfiles/resources/el001-product-information.pdf" target="_blank" shape="rect">
            T4 DNA Ligase protocol
        </A>
        .
    </STRONG>
    Instead of leaving the tube at 22ºC for 10 min, we left it at room temperature for 45 min.
</DIV><DIV>
    5) We will make a ligation. We will make a <STRONG>1:3</STRONG> proportion for the gBlock:vector. We added 3,3 uL of the linearized vector [25 ng/mL] and 7
    uL of the insert DNA [50 ng/mL].
</DIV><DIV>
    The protocol says to leave the ligation for 10 min, but we left if for 20 min at room temperature. The tube is called Ligation agaA Biobrick.
</DIV><H6>Goal: Transform <I>E coli</I> with agaA Biobrick</H6><H6>Procedure:</H6><DIV>
    We will transform competent E coli that were previously made by Jake following his own protocol. We will use the agaA Biobrick.
    </DIV><DIV>
    We will follow this
    <A href="https://www.evernote.com/Home.action?__fp=ZloDp4AdW-c3yWPvuidLz-TPR6I9Jhx8&amp;amp;username=iGEM-PB2014&amp;amp;login=true&amp;amp;_sourcePage=L90YhFqIo3HiMUD9T65RG_YvRLZ-1eYO3fqfqRu0fynRL_1nukNa4gH1t86pc1SP#b=60f448df-afb5-4a4d-9310-6dea624f7ab3&amp;amp;st=p&amp;amp;n=b48f12a9-a15f-45a2-a179-095516e07b3e" target="_blank" shape="rect">
        Heat Shock protocol
    </A>
    . We made two transformations (two tubes following the same protocol)
</DIV><DIV>
    We will plate one of the tubes in an LBA+Cm plate. We will leave the other tube in the incubator overnight and plate it the day after. For each tube, we
    always plate 20 uL in one plate and 200 uL in another one.
</DIV><H6>Results:</H6><P>     On July 13th we can see what seems a colony in one of the plates. We will see tomorrow.</P><P>On July 14th we see colonies.</P><H5 id="date">July 14th</H5><H6>Goal:Culture <I>Corynebacterium glutamicum</I> and <I>Corynebacterium striatum</I></H6><H6>Procedure:</H6><P>We will open the flask that arrived and culture Corynebacterium in Tryptophan Soy liquid medium at 37ºC overnight. After, we will plate them and culture single colonies to make competent cells and transform them. <I>Corynebacterium glutamicum</I> is sPB.006 and <I>Corynebacterium striatum</I> is sPB.007.
</P><H6>Goal:Obtain agaA in pSEVA (Digest pSEVA351 + ligate with agaA gBlock)</H6><H6>Procedure:</H6><DIV>
We will digest pSEVA351 following the    <A href="http://www.thermoscientificbio.com/uploadedFiles/Resources/fast-digestion-dna.pdf" target="_blank" shape="rect">Thermo Fast Digestion Protocol</A>
    for enzimes <STRONG>EcoRI and PstI.</STRONG> Then we will ligate it with <STRONG>the agaA gBlock</STRONG>. We want to obtain
    <A href="https://www.evernote.com/Home.action#b=596d48d5-2edd-48b4-b5e9-dc81ce1b9aab&amp;st=p&amp;n=c411c8c1-73e6-4f3f-835c-edc7b8826530" target="_blank" shape="rect">
       agaA in pSEVA
    </A>
    and transform <EM>Corynebacterium glutamicum</EM> and <EM>Corynebacterium striatum</EM>.
    <DIV>
        We measured the purified aga gBlock in the nanodrop, the concentration was 50 ng/mL. This is low. I do not know if it is low because I am measuring the
        purification product instead of the direct PCR product.
    </DIV><DIV><STRONG>Digestion</STRONG></DIV><DIV>
        1) We added 20 uL of water + 5 uL Buffer + 20 uL DNA + 2.5 uL EcoRI + 2.5 uL PstI.
    </DIV><DIV>
        2) Then we put it at 37ºC at the Isotemp for 5 min.
    </DIV><DIV>
        This tube was named Digested pSEVA351
        </DIV><DIV><STRONG>Purification</STRONG></DIV><DIV>
        4) We will purify the tube using the
        <A href="https://www.evernote.com/Home.action?__fp=ZloDp4AdW-c3yWPvuidLz-TPR6I9Jhx8&amp;amp;username=iGEM-PB2014&amp;amp;login=true&amp;amp;_sourcePage=L90YhFqIo3HiMUD9T65RG_YvRLZ-1eYO3fqfqRu0fynRL_1nukNa4gH1t86pc1SP#b=60f448df-afb5-4a4d-9310-6dea624f7ab3&amp;amp;st=p&amp;amp;n=9dd0d893-9195-4ae2-9bbb-3748ecff72c9" target="_blank" shape="rect">
            QIAquick kit
        </A>
        .
    </DIV><DIV>
        This tube was called Dig + Pur pSEVA351
    </DIV><DIV><STRONG>Ligation</STRONG></DIV><DIV>
        We ligated Dig+Pur agaA gBlock (July 12th) and Dig+Pur pSEVA351(July 14th)
    </DIV><DIV>
Following the        <A href="http://www.thermoscientificbio.com/uploadedfiles/resources/el001-product-information.pdf" target="_blank" shape="rect">Thermo</A><STRONG><A href="http://www.thermoscientificbio.com/uploadedfiles/resources/el001-product-information.pdf" target="_blank" shape="rect">
                T4 DNA Ligase protocol
            </A>
            .
        </STRONG>
        Instead of leaving the tube at 22ºC for 10 min, we left it at room temperature for 45 min.
    </DIV><DIV>
        5) We will make a ligation. We will make a <STRONG>1:3</STRONG> proportion for the gBlock:vector. We added 3,3 uL of the linearized vector [25 ng/mL]
        and 7 uL of the insert DNA [50 ng/mL].
    </DIV><DIV>
        The protocol says to leave the ligation for 10 min, but we left if for 20 min at room temperature. The tube is called Ligation pPB.003.
    </DIV></DIV><H5 id="date">July 16th</H5><H6>Goal: Colony PCR: Check agaA Biobrick transformation</H6><H6>Procedure:</H6><P>
    Yesterday July 15th Juanma runned a colony PCR with oPB.001 and oPB.005 using the
</P><A href="https://www.evernote.com/Home.action#b=60f448df-afb5-4a4d-9310-6dea624f7ab3&amp;st=p&amp;n=9df680b8-ce02-4c4c-be03-91cdbc73224b" target="_blank" shape="rect">
    Phusion Polymerase protocol
</A><P>
    . Today we will run a 1% agarose gel with the purified DNA following the
</P><A href="https://www.evernote.com/Home.action?__fp=avuu_OOUw8I3yWPvuidLz-TPR6I9Jhx8&amp;username=iGEM-PB2014&amp;login=true&amp;_sourcePage=dd1SVp2N6cjiMUD9T65RG_YvRLZ-1eYO3fqfqRu0fynRL_1nukNa4gH1t86pc1SP#b=60f448df-afb5-4a4d-9310-6dea624f7ab3&amp;st=p&amp;n=228cf621-87a8-43ca-9aa9-7e11f82d83d0" target="_blank" shape="rect">
    standard protocol
</A><P>
    .
</P><H6>Results: </H6>
     There are no bands on the gel. This might happen with Colony PCR. To double-check, we will do a MiniPrep to extract the agaA Biobrick and run a PCR with oPB.001 and oPB.005.
<H6>Goal:Make competent Corynebacterium</H6><H6>Procedure:</H6><DIV>
    We will use the eppendorf protocol (
    <A href="https://docs.google.com/file/d/0By8yVXC0fFVRMTlDd3BxQXM2Y1U/edit" target="_blank" shape="rect">
        https://docs.google.com/file/d/0By8yVXC0fFVRMTlDd3BxQXM2Y1U/edit
    </A>
    ) for making competent cells, but when growing them overnight, we will follow Haynes and Britz protocol (The effect of growth conditions of Corynebacterium
    glutamicum on the transformation frequency obtained by electroporation). We did this because its been proven that adding glycine to the medium overnight
    increases the electrotransformation efficiency. This is the different part of the protocol:
    </DIV><DIV><EM>Procedure:</EM></DIV><DIV><EM>
        Cells were grown for approximately 18 h at 30ºC in a rotary shaker at 200 r.p.m. in 400 ml LB supplemented with kanamycin (50 pg ml-l) and glycine
        (2.5%).
    </EM></DIV><H5 id="date">July 17th</H5><H6>Goal:Check transformation of E coli with the agaA Biobrick (MiniPrep + PCR + Gel)</H6><H6>Procedure:</H6><DIV><STRONG>MiniPrep</STRONG></DIV><DIV>
    We will extract agaA Biobrick by making a MiniPrep of the 8 colonies following the
    <A href="https://www.evernote.com/Home.action?__fp=ZloDp4AdW-c3yWPvuidLz-TPR6I9Jhx8&amp;username=iGEM-PB2014&amp;login=true&amp;_sourcePage=L90YhFqIo3HiMUD9T65RG_YvRLZ-1eYO3fqfqRu0fynRL_1nukNa4gH1t86pc1SP#b=60f448df-afb5-4a4d-9310-6dea624f7ab3&amp;st=p&amp;n=7103fd59-762f-4792-bb1a-280f7b7af3a3" target="_blank" shape="rect">
        standard protocol.
    </A></DIV><DIV><STRONG>PCR</STRONG></DIV><DIV>
    We will run a PCR with oPB.001 and oPB.005 following the
    <A href="https://www.evernote.com/Home.action#b=60f448df-afb5-4a4d-9310-6dea624f7ab3&amp;st=p&amp;n=9df680b8-ce02-4c4c-be03-91cdbc73224b" target="_blank" shape="rect">
        Phusion Polymerase protocol
    </A>
    . And we will run a 1% agarose gel with the purified DNA following the
    <A href="https://www.evernote.com/Home.action?__fp=avuu_OOUw8I3yWPvuidLz-TPR6I9Jhx8&amp;username=iGEM-PB2014&amp;login=true&amp;_sourcePage=dd1SVp2N6cjiMUD9T65RG_YvRLZ-1eYO3fqfqRu0fynRL_1nukNa4gH1t86pc1SP#b=60f448df-afb5-4a4d-9310-6dea624f7ab3&amp;st=p&amp;n=228cf621-87a8-43ca-9aa9-7e11f82d83d0" target="_blank" shape="rect">
        standard protocol
    </A>
    .
</DIV><H6>Results:</H6><P>Again, no bands. </P><H6>Goal:Make competent <I>Corynebacterium</I> + Transform <I>Corynebacterium</I> with pSEVA</H6><H6>Procedure:</H6><P>     We will use the eppenforf protocol (https://docs.google.com/file/d/0By8yVXC0fFVRMTlDd3BxQXM2Y1U/edit) </P><H6>Results:</H6><P>There are a lot of colonies for <I>Corynebacterium striatum</I>. In both plates. We will check that there is an insert. for that, we have started a culture (July 20th) and we will PCR it tomorrow.</P><H5 id="date">July 18th</H5><H6>Goal:Check transformation of <I>E coli</I> with agaA Biobrick </H6><H6>Procedure:</H6><DIV>
    Using the MiniPrep we got on July 17th, we will do a <STRONG>digestion analysis: </STRONG>(Jake's recipe)
    </DIV><DIV>
    - 5 uL plasmid
</DIV><DIV>
    - 2 uL Fast Digest Green (has LD in it)
</DIV><DIV>
    - 1 uL NotI or EcoRI+PstI. We used EcoRI+PstI as it was impossible to find NotI.
</DIV><DIV>
    - 13uL of H2O
</DIV><DIV>
    (Final volume of 20 uL)
</DIV><DIV>
    We will incubate for 15-30 min at 37ºC. Then we will run 10 uL (our Fast Digest Green buffer already has Loadyng Dye in it, so no need to add more) in
    an agarose 1% gel.
</DIV><H6>Results:</H6><P>The gel is empty. We cannot even see the vector.</P><H6>Goal:amplify agaA in Corynebacterium Striatum </H6><H6>Procedure:</H6>
500uL liquid culture of Corynebactereium striatum overnight
DNA extraction with RapidWater DNA isolation kit x2
number one : 25 ng/uL 
number two : 28 ng/uL 

PCR with :

oPB.046/oPB.047 (agaA primers 1)
oPB.048/oPB.049 (agaA primers 2)

PCR protocol for 500 nucleotides PCR products:


Reagent for 1X Volume:
Nuclease-free water : 35 uL 
5x Phusion HF Buffer : 10 ul 
10 mM dNTPs : 1 ul
Forward Primer (10 uM) : 0.5 ul
Reverse Primer (10 uM) : 0.5 ul
Template DNA : 1 ul
DMSO : 1.5 ul
Phusion DNA Polymerase : 0.5 ul

Thermocycler Protocol: NEB Phusion<CENTER><TABLE id="retable" class="tftable" border="1"><TBODY><TR><TH>Temperature (°C)</TH><TH>Time</TH></TR><TR><TD>Start</TD><TD>98</TD><TD>30 s</TD><TD>Melt</TD></TR><TR><TD>Cycle 1</TD><TD>98</TD><TD>10 s</TD><TD>Melt</TD><TD>35 Cycles</TD></TR><TR><TD>Cycle 2</TD><TD>50</TD><TD>30 s</TD><TD>Anneal</TD><TD>35 Cycles</TD></TR><TR><TD>Cycle 3</TD><TD>72</TD><TD>20 s</TD><TD>Extend</TD><TD>35 Cycles</TD></TR><TR><TD>Finish</TD><TD>72</TD><TD>5 min</TD><TD>Extend</TD></TR><TR><TD>Store</TD><TD>10</TD><TD>Forever</TD><TD>Store</TD></TR></TBODY></TABLE></CENTER>
Result: No band, maybe no agaA in Corynebacterium striatum ? <H5>July 20th: Crispr specificity</H5><H6>Goal</H6><P>Mix the the two strains in order to study the limit concentration of mutant that our CRISPR/Cas system could isolate.</P><H6>Procedure</H6><P>At the 3rd washing step, the strains were mixed in different ratios (from 50/50 to 1 / 10 to the minus 7).</P><P>Then the mixes were transformed with the regular electroporation protocol.</P><H6>Results</H6><P>Here are reported the ratio blue/white colonies:</P><TABLE class="tableizer-table"><TBODY><TR class="tableizer-firstrow"><TH>LacZ Plate1</TH><TH>LacZ Plate2</TH><TH>LacZ Plate3</TH><TH>LacZ Plate4</TH><TH>Rand Plate1</TH><TH>Rand Plate2</TH><TH>Rand Plate3</TH><TH>Rand Plate4</TH></TR><TR><TD>10-7</TD><TD>0.15</TD><TD>0.16</TD><TD>0.16</TD><TD>0.23</TD><TD>5.00E-01</TD><TD>7.00E-01</TD><TD> </TD><TD>2.00E-01</TD></TR><TR><TD>10-6</TD><TD>0.69</TD><TD>0.78</TD><TD>0.7</TD><TD>0.65</TD><TD>17</TD><TD>3</TD><TD>6</TD><TD>5</TD></TR><TR><TD>10-5</TD><TD>0.5</TD><TD>0.59</TD><TD>0.67</TD><TD>0.7</TD><TD>700</TD><TD>90</TD><TD>100</TD><TD>10</TD></TR><TR><TD>10-4</TD><TD>0.5</TD><TD>0.59</TD><TD>0.67</TD><TD>0.65</TD><TD>1,354</TD><TD>342</TD><TD>456</TD><TD>320</TD></TR><TR><TD>10-3</TD><TD>0.6</TD><TD>1</TD><TD>0.5</TD><TD>0.7</TD><TD>4,400</TD><TD>7,000</TD><TD>2,600</TD><TD>980</TD></TR><TR><TD>10-2</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD>14,500</TD><TD>34,000</TD><TD>10,000</TD><TD>23,000</TD></TR><TR><TD>10-1</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD>150,000</TD><TD>130,000</TD><TD>150,000</TD><TD>120,000</TD></TR><TR><TD>0.5</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD>540,000</TD><TD>570,000</TD><TD>480,000</TD><TD>400,000</TD></TR></TBODY></TABLE><H5 id="date">July 20th</H5><H6>Goal: Ligate agaA Biobrick + <I>E coli</I> transformation</H6><H6>Procedure:</H6><DIV><STRONG>
        Ligation
        </STRONG></DIV><DIV><DIV>
        We will use the digested agaA gBlock and PSB1C3 obtained on
        <A href="https://www.evernote.com/Home.action#b=596d48d5-2edd-48b4-b5e9-dc81ce1b9aab&amp;st=p&amp;n=db4278a7-4436-4a96-83c8-a23d6c57c65b" target="_blank" shape="rect">
            June 26th
        </A>
        .
    </DIV><DIV><DIV>
We will follow the            <A href="http://www.thermoscientificbio.com/uploadedfiles/resources/el001-product-information.pdf" target="_blank" shape="rect">Thermo</A><STRONG><A href="http://www.thermoscientificbio.com/uploadedfiles/resources/el001-product-information.pdf" target="_blank" shape="rect">
                    T4 DNA Ligase protocol
                </A>
                .
            </STRONG>
            I think last ligation did not work because it was too hot in the lab. This time, I will leave the ligation product for 30 min in the dark room
            (20ºC).
        </DIV><DIV><STRONG>E coli transformation</STRONG></DIV><DIV><DIV><DIV>
                    We will transform competent NEB turbo E coli made following the
                    <A href="https://www.evernote.com/Home.action#b=60f448df-afb5-4a4d-9310-6dea624f7ab3&amp;st=p&amp;n=b4939a60-0f83-446a-a111-39d63eb7589c" target="_blank" shape="rect">
                        standard protocol
                    </A>
                    . In step 5, we did not incubate them for 4 hours, but only until the density was optimal (around 30-45 min).
                </DIV><DIV>
                    We will use
                    <A href="https://www.evernote.com/Home.action#b=596d48d5-2edd-48b4-b5e9-dc81ce1b9aab&amp;st=p&amp;n=c359be34-a042-4d01-a0bd-ebb7b81f3166" target="_blank" shape="rect">
                        agaA Biobrick
 </A>
                    obtained today.
                </DIV><DIV>
                    We will follow this
                    <A href="https://www.evernote.com/Home.action?__fp=ZloDp4AdW-c3yWPvuidLz-TPR6I9Jhx8&amp;username=iGEM-PB2014&amp;login=true&amp;_sourcePage=L90YhFqIo3HiMUD9T65RG_YvRLZ-1eYO3fqfqRu0fynRL_1nukNa4gH1t86pc1SP#b=60f448df-afb5-4a4d-9310-6dea624f7ab3&amp;st=p&amp;n=b48f12a9-a15f-45a2-a179-095516e07b3e" target="_blank" shape="rect">
                        Heat Shock protocol
                    </A>
                    .
                    </DIV><DIV><EM><STRONG>Tip</STRONG>
                        (From Matt): At the last step when we recover the cells, plate most of the 200 uL and leave the rest recovering overnight. Then plate
                        them the next day.
                        </EM></DIV></DIV><DIV>
                We will plate them in LBA+Cm.
            </DIV></DIV></DIV></DIV><H6>Results:</H6><P>There were no colonies 48h after.</P><DIV id="August"><H4>August</H4><H5 id="date">August 1st</H5><H6>Goal:  Check <I>E coli</I> transformation with agaA Biobrick (miniprep + digestion)</H6><H6>Procedure:</H6><DIV><STRONG>
        Ligation
        </STRONG></DIV><DIV><DIV>
        We will use the digested agaA gBlock and PSB1C3 obtained on
        <A href="https://www.evernote.com/Home.action#b=596d48d5-2edd-48b4-b5e9-dc81ce1b9aab&amp;st=p&amp;n=db4278a7-4436-4a96-83c8-a23d6c57c65b" target="_blank" shape="rect">
            June 26th
        </A>
        .
    </DIV><DIV><DIV>
We will follow the            <A href="http://www.thermoscientificbio.com/uploadedfiles/resources/el001-product-information.pdf" target="_blank" shape="rect">Thermo</A><STRONG><A href="http://www.thermoscientificbio.com/uploadedfiles/resources/el001-product-information.pdf" target="_blank" shape="rect">
                    T4 DNA Ligase protocol
                </A>
                .
            </STRONG>
            I think last ligation did not work because it was too hot in the lab. This time, I will leave the ligation product for 30 min in the dark room
            (20ºC).
        </DIV><DIV><STRONG>E coli transformation</STRONG></DIV><DIV><DIV><DIV>
                    We will transform competent NEB turbo E coli made following the
                    <A href="https://www.evernote.com/Home.action#b=60f448df-afb5-4a4d-9310-6dea624f7ab3&amp;st=p&amp;n=b4939a60-0f83-446a-a111-39d63eb7589c" target="_blank" shape="rect">
                        standard protocol
                    </A>
                    . In step 5, we did not incubate them for 4 hours, but only until the density was optimal (around 30-45 min).
                </DIV><DIV>
                    We will use
                    <A href="https://www.evernote.com/Home.action#b=596d48d5-2edd-48b4-b5e9-dc81ce1b9aab&amp;st=p&amp;n=c359be34-a042-4d01-a0bd-ebb7b81f3166" target="_blank" shape="rect">
                        agaA Biobrick
                    </A>
                    obtained today.
                </DIV><DIV>
                    We will follow this
                    <A href="https://www.evernote.com/Home.action?__fp=ZloDp4AdW-c3yWPvuidLz-TPR6I9Jhx8&amp;username=iGEM-PB2014&amp;login=true&amp;_sourcePage=L90YhFqIo3HiMUD9T65RG_YvRLZ-1eYO3fqfqRu0fynRL_1nukNa4gH1t86pc1SP#b=60f448df-afb5-4a4d-9310-6dea624f7ab3&amp;st=p&amp;n=b48f12a9-a15f-45a2-a179-095516e07b3e" target="_blank" shape="rect">
                        Heat Shock protocol
                    </A>
                    .
                    </DIV><DIV><EM><STRONG>Tip</STRONG>
                        (From Matt): At the last step when we recover the cells, plate most of the 200 uL and leave the rest recovering overnight. Then plate
                        them the next day.
                        </EM></DIV></DIV><DIV>
                We will plate them in LBA+Cm.
            </DIV></DIV></DIV></DIV><H6>Results:</H6><P>There are colonies on the plate. We will culture, miniprep and digest them.</P><H5 id="date">August 3rd</H5><H6>Goal: Transform the WT and the mutant with fluorescent plasmids in order to spot CRISPR induced mutants</H6><H6>Procedure:</H6><P>
The WT was transformed with a CFP plasmid, and the mutant with a YFP plasmid, by regular heat shock protocol (30 seconds at 42°C).
Cells were recovered 1h at 37°C after transformation and plated on chloramphenicol+spectomycin plates 24h at 37°C.
Plates were put under optical filters to check the fluorescence.
</P><H6>Results:</H6><P>
Colonies had the expected fluorescence according to the transformation.
</P><H5 id="date">August 4th</H5><H6>Goal: Amplify ackA, l-ldH, d-ldH and ldH in microbiome armpit samples.</H6><P>
    DNA extraction from cotton swabs.
</P><P>
    1<SUP>st</SUP> step : remove cotton from stick and immerse it in PBS.
</P><P>
    2<SUP>nd</SUP> step : vortex for 20s
</P><P>
    3<SUP>rd</SUP> step : follow RapidWater DNA isolation kit’s protocol
</P><P>
    PCR on DNA extraction with :
</P><P>
    oPB.066/oPB.067 (ackA)
</P><P>
    oPB.068/oPB.069 (d-ldH)
</P><P>
    oPB.070/oPB.071 (l-ldH)
</P><P>
    oPB.036/oPB.037 (ldH)
</P><P>
    It worked for the four genes.
</P><P>
    Column 1 : ladder 1 kb
</P><P>
    Column 2 : ackA
</P><P>
    Column 3 : d-ldH
</P><P>
    Column 4 : l-ldH
</P><P>
    Column 7 : ldH
</P><H5 id="date">August 5th</H5><H6>Goal:  Check <I>E coli</I> transformation with agaA Biobrick and <I>Corynebacterium</I> transformation with pSEVA + </H6><H6>Procedure:</H6><DIV><STRONG>MiniPrep</STRONG></DIV><DIV>
    We will extract the agaA Biobrick by making a MiniPrep of the 8 colonies + the new E coli transformation (July 20th) + Corynebacterium transformed with pSEVA
    following the
    <A href="https://www.evernote.com/Home.action?__fp=ZloDp4AdW-c3yWPvuidLz-TPR6I9Jhx8&amp;username=iGEM-PB2014&amp;login=true&amp;_sourcePage=L90YhFqIo3HiMUD9T65RG_YvRLZ-1eYO3fqfqRu0fynRL_1nukNa4gH1t86pc1SP#b=60f448df-afb5-4a4d-9310-6dea624f7ab3&amp;st=p&amp;n=7103fd59-762f-4792-bb1a-280f7b7af3a3" target="_blank" shape="rect">
        standard protocol.
    </A></DIV><DIV>
    In the nanodrop, we saw that the yield for the 8 colonies was way too low (10 ug/mL) so we discarded them. We will continue with the E coli transformed on
    July 20th (37 ug/mL) + Corynebacterium transformed with pSEVA (117 ug/mL).
    </DIV><DIV><STRONG>Digest</STRONG></DIV><DIV><DIV>
        Using the MiniPrep we got on July 17th, we will do a <STRONG>digestion analysis: </STRONG>(Jake's recipe)
        </DIV><DIV>
        - 5 uL plasmid (for the Ecoli we used 10 uL and added less water)
        </DIV><DIV>
        - 2 uL Fast Digest Green (has LD in it)
    </DIV><DIV>
        - 1 uL NotI or EcoRI+PstI. We used EcoRI+PstI
    </DIV><DIV>
        - 13uL of H2O
    </DIV><DIV>
        (Final volume of 20 uL)
    </DIV><DIV>
        We will incubate for 15-30 min at 37ºC. Then we will run 10 uL (our Fast Digest Green buffer already has Loadyng Dye in it, so no need to add
        more) in an agarose 1% gel.
    </DIV></DIV><H6>Results:</H6><P>
    -
</P><STRONG>1st raw:</STRONG><P>
    Gene Ruler 100 kb + 1uL Loading Dye
</P><DIV>
    - <STRONG>2nd raw</STRONG>: Corynebacterium striatum transformed with pSEVA
    </DIV><DIV>
    - <STRONG>3st raw: </STRONG><I>E. coli</I> transformed with agaA Biobrick. However it weights 5000 kb. We go back to our protocol of August 1st and in fact what we have
    here is <STRONG>E coli transformed with pSEVA.</STRONG></DIV><H6>Conclusions: </H6><DIV>
    Looking at this gel:
    </DIV><DIV>
    - We should not have used the 100 kb rule but the 1 kb
    </DIV><DIV>
    - Corynebacterium has not been transformed
    </DIV><DIV>
    - When we run at 100 V, the gel looks worse
</DIV><H6>Goal: <I>Corynebacterium striatum</I> transformation with pSEVA</H6><H6>Procedure:</H6><P> We will use the eppenforf protocol (https://docs.google.com/file/d/0By8yVXC0fFVRMTlDd3BxQXM2Y1U/edit) </P><H6>Results:</H6><P>  There are a lot of colonies in the plate. We will strike them to pick single colonies and check the transformation.
 </P><H6>Goal: Sequencing PCR product from 03/08 PCR to check if I amplified right genes </H6><H6>Procedure:</H6>
Remind : I amplified ackA, d-ldH, l-ldH, ldH.<H5 id="date">August6th</H5><H6>Goal:  agaA Biobrick ligation </H6><H6>Procedure:</H6><DIV>
    We are going to check our ligation product in a gel to see what is going on. Last time, the problem is that I used pSEVA to transform.
    </DIV><DIV><DIV><DIV><STRONG>
                Ligation
                </STRONG></DIV><DIV>
            We will use the digested agaA gBlock and pSB1C3.
        </DIV><DIV>
We will follow the            <A href="http://www.thermoscientificbio.com/uploadedfiles/resources/el001-product-information.pdf" target="_blank" shape="rect">Thermo</A><STRONG><A href="http://www.thermoscientificbio.com/uploadedfiles/resources/el001-product-information.pdf" target="_blank" shape="rect">
                    T4 DNA Ligase protocol
                </A>
                .
            </STRONG></DIV><DIV>
            We runned 5 uL of the digestion product in a 1% agarose gel. We used 2 uL to transform E coli. The rest of the tube was kept at 4ºC. If the
            transformation goes wrong, we will use it tomorrow to transform again.
            </DIV></DIV></DIV><H6>Goal:  Transform <I>E coli</I> with agaA Biobrick</H6><H6>Procedure:</H6><DIV><DIV>
        We will transform competent NEB turbo <I>E. coli</I> made following the
        <A href="https://www.evernote.com/Home.action#b=60f448df-afb5-4a4d-9310-6dea624f7ab3&amp;st=p&amp;n=b4939a60-0f83-446a-a111-39d63eb7589c" target="_blank" shape="rect">
            standard protocol
        </A>
        . In step 5, we did not incubate them for 4 hours, but only until the density was optimal (around 30-45 min).
    </DIV><DIV>
        We will use
        <A href="https://www.evernote.com/Home.action#b=596d48d5-2edd-48b4-b5e9-dc81ce1b9aab&amp;st=p&amp;n=c359be34-a042-4d01-a0bd-ebb7b81f3166" target="_blank" shape="rect">
            agaA Biobrick
        </A>
        .
    </DIV><DIV>
        We will follow this
        <A href="https://www.evernote.com/Home.action?__fp=ZloDp4AdW-c3yWPvuidLz-TPR6I9Jhx8&amp;username=iGEM-PB2014&amp;login=true&amp;_sourcePage=L90YhFqIo3HiMUD9T65RG_YvRLZ-1eYO3fqfqRu0fynRL_1nukNa4gH1t86pc1SP#b=60f448df-afb5-4a4d-9310-6dea624f7ab3&amp;st=p&amp;n=b48f12a9-a15f-45a2-a179-095516e07b3e" target="_blank" shape="rect">
            Heat Shock protocol
        </A>
        .
        </DIV><DIV><EM><STRONG>Tip</STRONG>
            (From Matt): At the last step when we recover the cells, plate most of the 200 uL and leave the rest recovering overnight. Then plate them the next
            day.
            </EM></DIV></DIV><DIV>
    We will plate them in LBA+Cm. We will add 250 uL of Cm for 250 mL of LBA.
    </DIV><H6>Results:</H6><P>There are colonies in the plate. We made a sub culture to pick single colonies on August 6th. On August 7th we made a liquid culture to then miniprep it and check the transformation.</P><H5 id="date">August 7th</H5><H6>Goal: Partially run the crispr specificty study (see July 20th) but also compare the blue/white screening and the fluorescence</H6><H6>Procedure:</H6><P>
The same protocol was used (see July 20th) but on the fluorescent strains.
Blue and colorless colonies were counted, and their fluorescence was checked with an optical filter.
</P><H6>Results:</H6><P>
Here are reported the number of counted colonies regarding their color and their fluorescence.</P><DIV><TABLE dir="ltr" border="1" cellspacing="0" cellpadding="0"><TBODY><TR><TD colspan="8" rowspan="1">
                    LacZ targeting CRISPR
                </TD><TD colspan="8" rowspan="1">
                    Non targeting CRISPR
                </TD></TR><TR><TD colspan="4" rowspan="1">
                    Colorless colonies
                </TD><TD colspan="4" rowspan="1">
                    Blue colonies
                </TD><TD colspan="4" rowspan="1">
                    Colorless colonies
                </TD><TD colspan="4" rowspan="1">
                    Blue colonies
                </TD></TR><TR><TD colspan="1" rowspan="1">
                    CFP
                </TD><TD colspan="1" rowspan="1">
                    17
                </TD><TD colspan="1" rowspan="1">
                    15
                </TD><TD colspan="1" rowspan="1">         16
                </TD><TD colspan="1" rowspan="1">
                    16
                </TD><TD colspan="1" rowspan="1">
                    23
                </TD><TD colspan="1" rowspan="1">
                    26
                </TD><TD colspan="1" rowspan="1">
                    24
                </TD><TD colspan="1" rowspan="1">
                    27
                </TD><TD colspan="1" rowspan="1">
                    20
                </TD><TD colspan="1" rowspan="1">
                    21
                </TD><TD colspan="1" rowspan="1">
                    23
                </TD><TD colspan="1" rowspan="1">
                    26
                </TD><TD colspan="1" rowspan="1">
                    875
                </TD><TD colspan="1" rowspan="1">
                    873
                </TD><TD colspan="1" rowspan="1">
                    870
                </TD><TD colspan="1" rowspan="1">
                    839
                </TD></TR><TR><TD colspan="1" rowspan="1">
                    YFP
                </TD><TD colspan="1" rowspan="1">
                    247
                </TD><TD colspan="1" rowspan="1">
                    256
                </TD><TD colspan="1" rowspan="1">
                    239
                </TD><TD colspan="1" rowspan="1">
                    230
                </TD><TD colspan="1" rowspan="1">
                    0
                </TD><TD colspan="1" rowspan="1">
                    0
                </TD><TD colspan="1" rowspan="1">
                    0
                </TD><TD colspan="1" rowspan="1">
                    0
                </TD><TD colspan="1" rowspan="1">
                    11
                </TD><TD colspan="1" rowspan="1">
                    12
                </TD><TD colspan="1" rowspan="1">
                    15
                </TD><TD colspan="1" rowspan="1">
                    12
                </TD><TD colspan="1" rowspan="1">
                    0
                </TD><TD colspan="1" rowspan="1">
                    0
                </TD><TD colspan="1" rowspan="1">
                    0
                </TD><TD colspan="1" rowspan="1">
                    0
                </TD></TR></TBODY></TABLE></DIV>

The mismatches are to be studied in more details.
<H6>Goal:Obtain agaA Biobrick (agaA gBlock digestion + ligation)</H6><H6>Procedure:</H6><DIV>
We will digest the purified agaA gBlock following the    <A href="http://www.thermoscientificbio.com/uploadedFiles/Resources/fast-digestion-dna.pdf" target="_blank" shape="rect">Thermo Fast Digestion Protocol</A>
    for enzimes <STRONG>EcoRI and PstI.</STRONG> Then we will ligate it with the linearized <STRONG>vector pSB1C3</STRONG>. We want to obtain
    <A href="https://www.evernote.com/Home.action?__fp=ZloDp4AdW-c3yWPvuidLz-TPR6I9Jhx8&amp;amp;username=iGEM-PB2014&amp;amp;login=true&amp;amp;_sourcePage=L90YhFqIo3HiMUD9T65RG_YvRLZ-1eYO3fqfqRu0fynRL_1nukNa4gH1t86pc1SP#b=596d48d5-2edd-48b4-b5e9-dc81ce1b9aab&amp;amp;st=p&amp;amp;n=c359be34-a042-4d01-a0bd-ebb7b81f3166" target="_blank" shape="rect">
        agaA Biobrick
    </A>
    and transform <I>E. coli</I>.
</DIV><DIV><STRONG>Digestion</STRONG></DIV><DIV>
    1) We are using the agaA working stock. We added 20 uL of water + 5 uL Buffer + 20 uL DNA + 2.5 uL EcoRI + 2.5 uL PstI.
</DIV><DIV>
    2) Then we put it at 37ºC at the Isotemp for 1h30 (instead of 5 min).
</DIV><DIV>
    3) We will repeat for PSB1C3, using the 'plasmid DNA' protocol.
</DIV><DIV>
    These tubes were called Digested gBlock and Digested PSB1C3
</DIV><DIV><STRONG>Purification</STRONG></DIV><DIV>
    4) We will purify both tubes using the
    <A href="https://www.evernote.com/Home.action?__fp=ZloDp4AdW-c3yWPvuidLz-TPR6I9Jhx8&amp;amp;username=iGEM-PB2014&amp;amp;login=true&amp;amp;_sourcePage=L90YhFqIo3HiMUD9T65RG_YvRLZ-1eYO3fqfqRu0fynRL_1nukNa4gH1t86pc1SP#b=60f448df-afb5-4a4d-9310-6dea624f7ab3&amp;amp;st=p&amp;amp;n=9dd0d893-9195-4ae2-9bbb-3748ecff72c9" target="_blank" shape="rect">
        QIAquick kit
    </A>
    .
</DIV><DIV>
    These tubes were called Dig + Pur gBlock and Dig + Pur PSB1C3
</DIV><DIV><STRONG>Ligation</STRONG></DIV><DIV>﻿Following the    <A href="http://www.thermoscientificbio.com/uploadedfiles/resources/el001-product-information.pdf" target="_blank" shape="rect">Thermo</A><STRONG><A href="http://www.thermoscientificbio.com/uploadedfiles/resources/el001-product-information.pdf" target="_blank" shape="rect">
            T4 DNA Ligase protocol
        </A>
        .
    </STRONG>
    Instead of leaving the tube at 22ºC for 10 min, we left it at room temperature for 45 min.
</DIV><DIV>
    5) We will make a ligation. We will make a <STRONG>1:3</STRONG> proportion for the gBlock:vector. We added 3,3 uL of the linearized vector [25 ng/mL] and
    2,5 uL of the insert DNA [100 ng/mL].
</DIV><DIV><STRONG>Purification</STRONG></DIV><DIV>
    6) We will purify the product using the
    <A href="https://www.evernote.com/Home.action?__fp=ZloDp4AdW-c3yWPvuidLz-TPR6I9Jhx8&amp;amp;username=iGEM-PB2014&amp;amp;login=true&amp;amp;_sourcePage=L90YhFqIo3HiMUD9T65RG_YvRLZ-1eYO3fqfqRu0fynRL_1nukNa4gH1t86pc1SP#b=60f448df-afb5-4a4d-9310-6dea624f7ab3&amp;amp;st=p&amp;amp;n=9dd0d893-9195-4ae2-9bbb-3748ecff72c9" target="_blank" shape="rect">
        QIAquick kit
    </A>
    .
</DIV><DIV>
    This tube was called agaA Biobrick
</DIV><H6>Results:</H6><DIV>
    In the nanodrop, we have seen that the concentration of both our vector and insert is 2 ng/mL. This is very low for ligation. This might be the reason why
    ligation does not work: not enough DNA.
</DIV><DIV><STRONG> Next step:</STRONG></DIV><DIV>
    - Take PSB1C3 from the BioBrick parts, clone it into E coli
</DIV><DIV>
    - Miniprep
</DIV><DIV>
    - Use this PSB1C3 to obtain agaA Biobrick
</DIV><H6>Goal:Check transformation in <I>Corynebacterium</I></H6><H6>Procedure:</H6><P>We picked 2 single colonies and incubated them in LB at 37ºC overnight.</P><H6>Goal:Glycerol stock for <I>E coli with pSEVA</I></H6><H6>Procedure:</H6><P> We added 750 uL of culture and 250 uL of Glycerol 60% and stock them in the common box at -20ºC and some other tubes at -80ºC.</P><H5 id="date">August 8th, 10th and 11th</H5><H6>Goal:Check <I>Corynebacterium</I> transformation with pSEVA  </H6><H6>Procedure:</H6><DIV><STRONG>MiniPrep</STRONG></DIV><DIV>
    We will extract agaA Biobrick by making a MiniPrep of 2 colonies of Corynebacterium transformed with pSEVA following the
    <A href="https://www.evernote.com/Home.action?__fp=ZloDp4AdW-c3yWPvuidLz-TPR6I9Jhx8&amp;username=iGEM-PB2014&amp;login=true&amp;_sourcePage=L90YhFqIo3HiMUD9T65RG_YvRLZ-1eYO3fqfqRu0fynRL_1nukNa4gH1t86pc1SP#b=60f448df-afb5-4a4d-9310-6dea624f7ab3&amp;st=p&amp;n=7103fd59-762f-4792-bb1a-280f7b7af3a3" target="_blank" shape="rect">
        standard protocol.
    </A></DIV><DIV>
    In the nanodrop, we saw that the yield for colony 1 was 80 ng/uL and colony 2 had 170 ng/uL
    </DIV><DIV><STRONG>Digest</STRONG></DIV><DIV><DIV>
        Using the MiniPrep we got on July 17th, we will do a <STRONG>digestion analysis: </STRONG>(Jake's recipe)
        </DIV><DIV>
        - 5 uL plasmid (for the colony 1 we used 10 uL and added less water)
        </DIV><DIV>
        - 2 uL Fast Digest Green (has LD in it)
    </DIV><DIV>
        - 1 uL NotI (we have checked that it makes two cuts in pSEVA, so we will see two bands)
    </DIV><DIV>
        - 13uL of H2O
    </DIV><DIV>
        (Final volume of 20 uL)
    </DIV><DIV>
        We will incubate for 15-30 min at 37ºC. Then we will run 10 uL (our Fast Digest Green buffer already has Loadyng Dye in it, so no need to add
        more) in an agarose 1% gel.
    </DIV></DIV><H6>Goal: Obtain PSB1C3 and clone it in <I>E coli</I></H6><H6>Procedure:</H6><DIV><DIV>
We will take <A href="http://parts.igem.org/Part:BBa_J04450" target="_blank" shape="rect">PSB1C3 from the DNA Kit Plate</A> and clone it into        <EM>E coli</EM>. Then, we will make a culture, grow it overnight and miniprep it tomorrow. This was, we will obtain PSB1C3 in a much higher
        concentration than we had in the linearized version.
        </DIV><DIV>
        It is on the Spring 2014 Plate 4, 4B
    </DIV></DIV><DIV><H3>
        DNA Kit Plate Instructions
 </H3><P><STRONG>
            Before you use the DNA in the Distribution Kit Plates, be sure to test the efficiency of your competent cells with the
            <A href="http://parts.igem.org/Help:2014_DNA_Distribution#Transformation_Efficiency_Kit" target="_blank" shape="rect">
                Transformation Efficiency Kit.
            </A></STRONG></P><P>
        To use the DNA in the Distribution Kit, follow these instructions:
        <EM>Note: There is an estimated 2-3ng of DNA in each well, following this protocol, assume that you are transforming with 200-300pg/ul</EM></P><DL><DD><OL><LI>
                    With a pipette tip, punch a hole through the foil cover into the corresponding well of the part that you want.
                    <A href="http://parts.igem.org/Help:Spring_2011_DNA_distribution#DNA_Kit_Plate_Orientation" target="_blank" rel="nofollow" shape="rect">
                        Make sure you have properly oriented the plate
                    </A>
                    . Do not remove the foil cover, as it could lead to cross contamination between the wells.
                </LI><LI>
                    Pipette 10uL of dH2O (distilled water) into the well. Pipette up and down a few times and let sit for 5 minutes to make sure the dried DNA
                    is fully resuspended. We recommend that you do not use TE to resuspend the dried DNA.
                </LI><LI><A title="Help:Transformation Protocol" href="http://parts.igem.org/Help:Transformation_Protocol" target="_blank" shape="rect">
                        Transform
                    </A>
                    1ul of the resuspended DNA into your desired competent cells, plate your transformation with the appropriate antibiotic* and grow
                    overnight.
                </LI><LI>
                    Pick a single colony and inoculate broth (again, with the correct antibiotic) and grow for 16 hours.
                </LI><LI>
Use the resulting culture to                    <A href="http://openwetware.org/wiki/Miniprep/Kit-free_high-throughput_protocol" target="_blank" rel="nofollow" shape="rect">miniprep</A>
the DNA AND make your own glycerol stock (for further instruction on making a glycerol see                    <A href="http://openwetware.org/wiki/Endy:Making_a_long_term_stock_of_bacteria" target="_blank" rel="nofollow" shape="rect">this page</A>).
                    We recommend using the miniprepped DNA to run QC tests, such as restriction digests and sequencing.
                </LI></OL></DD></DL><P><EM>
* To know which antibiotics to use, look at the plasmid that the part is in. The            <A href="http://parts.igem.org/wiki/index.php/Help:Plasmids/Nomenclature" target="_blank" rel="nofollow" shape="rect">naming scheme</A> for
            plasmids is specifically designed to indicate antibiotic resistance.
        </EM></P><P><EM>
Note: There is not enough DNA in each well to perform anything but            <A title="Help:Transformation Protocol" href="http://parts.igem.org/Help:Transformation_Protocol" target="_blank" shape="rect">transformations</A></EM></P><P><STRONG>Transformation</STRONG></P></DIV><DIV><DIV><DIV>
            We will transform competent NEB turbo E coli made following the
            <A href="https://www.evernote.com/Home.action#b=60f448df-afb5-4a4d-9310-6dea624f7ab3&amp;st=p&amp;n=b4939a60-0f83-446a-a111-39d63eb7589c" target="_blank" shape="rect">
                standard protocol
            </A>
            . In step 5, we did not incubate them for 4 hours, but only until the density was optimal (around 30-45 min).
        </DIV><DIV>
            We will use PSB1C3 obtained today from the DNA Kit Plate.
        </DIV><DIV>
            We will follow this
            <A href="https://www.evernote.com/Home.action?__fp=ZloDp4AdW-c3yWPvuidLz-TPR6I9Jhx8&amp;username=iGEM-PB2014&amp;login=true&amp;_sourcePage=L90YhFqIo3HiMUD9T65RG_YvRLZ-1eYO3fqfqRu0fynRL_1nukNa4gH1t86pc1SP#b=60f448df-afb5-4a4d-9310-6dea624f7ab3&amp;st=p&amp;n=b48f12a9-a15f-45a2-a179-095516e07b3e" target="_blank" shape="rect">
                Heat Shock protocol
            </A>
            .
            </DIV><DIV><EM><STRONG>Tip</STRONG>
                (From Matt): At the last step when we recover the cells, plate most of the 200 uL and leave the rest recovering overnight. Then plate them the
                next day.
                </EM></DIV></DIV><DIV>
        We will plate them in LBA+Cm. We will add 250 uL of Cm for 250 mL of LBA. I forgot to do it on the afternoon, so I did it on the 9th and the 10th I
        started a culture for miniprep.
    </DIV></DIV><H6> Results: </H6>
div&gt;
    I have too many colonies and they do not look red, just redish. So two things:
</DIV><DIV>
    - Strike a sample in another plate to see single colonies (<EM>August 10th</EM>) - <STRONG>Plate 1</STRONG></DIV><DIV>
    - Do a new transformation (<EM>August 10th</EM>) - <STRONG>Plate 2</STRONG></DIV><DIV><EM> August 11th</EM>
    - We see colonies in both plates, but in <STRONG>Plate 1</STRONG> they are bigger and express very intensly RFP. So we took a colony from this plate to
    miniprep and extract PSB1C3.
</DIV><H6>Goal:amplifying and sequencing smell genes from a mix of 10 people’s DNA extraction and test control.</H6><H6>Procedure:</H6><PRE id="result-source"><P>1&gt; To use like control, I chose an
available strain : staphylococcus capitis</P><P>I test on it these following primers :</P><P> oPB.066/oPB.067   </P><P>oPB.068/oPB.069</P><P>oPB.070/oPB.071</P><P>oPB.036/oPB.037</P></PRE><P>
    Column 1 : ladder 100bp
</P><P>
    Column 2 : ackA
</P><P>
    Column 3 : l-ldH
</P><P>
    Column 4 : d-ldH
</P><P>
    I can’t amplify d-ldH in Staphylococcus capitis.
</P><P>
    1<SUP>st</SUP> step: 10 people rub cotton swab on their armpit.
</P><P>
    2<SUP>nd</SUP> step: After immersing the cotton in PBS I mix all PBS together in order to get a mix of samples.
</P><P>
    3<SUP>rd</SUP> step: I follow the RapidWater DNA isolation kit protocol.
</P><P>
    DNA nanodrop
</P><P>
    Mix : 48 ng/uL
</P><P>
    1 : 50 ng/uL
</P><P>
    2 : 51 ng/uL
</P><P>
    3 : 24 ng/uL
</P><P>
    4 : 30 ng/uL
</P><P>
    5 : 30 ng/uL
</P><P>
    6 : 41 ng/uL
</P><P>
    7 : 43 ng/uL
</P><P>
    8 : 51 ng/uL
</P><P>
    9 : 32 ng/uL
</P><P>
    10 : 42 ng/uL
</P><P>
    11 : 26 ng/uL
</P><P>
    12 : 70 ng/uL
</P><P>
    PCR for each sample with following primers :
</P><P>
    oPB.066/oPB.067
</P><P>
    oPB.070/oPB.071
</P><P>
    oPB.068/oPB.069
</P><P>
    oPB.036/oPB.037
</P>
We can notice both amplification : T and C. The reverse confirm that it's not a sequencing mistake. For example this mutation is non synonymous substitution which change Proline into Serine in 3D protein structure.
<H5 id="date">August 12th</H5><H6>Goal:agaA Biobrick with new PBS1C3</H6><H6>Procedure:</H6><DIV><DIV><STRONG>Today is fun day.</STRONG></DIV><DIV><DIV><STRONG>
                1) Miniprep E coli culture
                </STRONG></DIV></DIV><DIV>
        We will extract PSB1C3 by making a MiniPrep of the colony of<EM> E coli</EM> we transformed following the
        <A href="https://www.evernote.com/Home.action?__fp=ZloDp4AdW-c3yWPvuidLz-TPR6I9Jhx8&amp;username=iGEM-PB2014&amp;login=true&amp;_sourcePage=L90YhFqIo3HiMUD9T65RG_YvRLZ-1eYO3fqfqRu0fynRL_1nukNa4gH1t86pc1SP#b=60f448df-afb5-4a4d-9310-6dea624f7ab3&amp;st=p&amp;n=7103fd59-762f-4792-bb1a-280f7b7af3a3" target="_blank" shape="rect">
            standard protocol.
        </A></DIV><DIV>
        In the nanodrop, we saw that the yield was 253,3 ng/uL
        </DIV></DIV><DIV><STRONG>2) Digest PBS1C3</STRONG></DIV><DIV><DIV>
        We will digest the purified agaA gBlock following the
        <A href="http://www.thermoscientificbio.com/uploadedFiles/Resources/fast-digestion-dna.pdf" target="_blank" shape="rect">
            Thermo Fast Digestion Protocol
        </A>
        for enzimes <STRONG>EcoRI and PstI.</STRONG> Then we will ligate it with the linearized <STRONG>vector pSB1C3</STRONG>. We want to obtain
        <A href="https://www.evernote.com/Home.action?__fp=ZloDp4AdW-c3yWPvuidLz-TPR6I9Jhx8&amp;amp;username=iGEM-PB2014&amp;amp;login=true&amp;amp;_sourcePage=L90YhFqIo3HiMUD9T65RG_YvRLZ-1eYO3fqfqRu0fynRL_1nukNa4gH1t86pc1SP#b=596d48d5-2edd-48b4-b5e9-dc81ce1b9aab&amp;amp;st=p&amp;amp;n=c359be34-a042-4d01-a0bd-ebb7b81f3166" target="_blank" shape="rect">
            agaA Biobrick
        </A>
        and transform E coli.
    </DIV><DIV>
        We digested<STRONG> two tubes</STRONG>. We used the protocol for Genomic DNA because it has worked for some colleagues that are doing the same, and I
        would like to have a big amount of final volume.
    </DIV><DIV>
        We will digest for 2h. We added 20 uL of PSB1C3 (we wanted to have 5 ug), so we used 10 uL less of water.
    </DIV><DIV>
        We called these tubes Dig PSB1C3.
    </DIV></DIV><DIV><STRONG>3) Purify PBS1C3 with sticky ends in a gel</STRONG></DIV><DIV>
We run an agarose gel at 1%, cut the region of PSB1C3 and make a    <A href="http://www.thermoscientificbio.com/uploadedFiles/Resources/k069-product-information.pdf" target="_blank" shape="rect">gel purification</A> using
    the kit.
</DIV><DIV><STRONG>4) Ligate PSB1C3 with the gBlock</STRONG></DIV><DIV><DIV>﻿Following the        <A href="http://www.thermoscientificbio.com/uploadedfiles/resources/el001-product-information.pdf" target="_blank" shape="rect">Thermo</A><STRONG><A href="http://www.thermoscientificbio.com/uploadedfiles/resources/el001-product-information.pdf" target="_blank" shape="rect">
                T4 DNA Ligase protocol
            </A>
            .
        </STRONG>
        Instead of leaving the tube at 22ºC for 10 min, we left it at room temperature for 45 min.
    </DIV><DIV>
        We will make a ligation. We will make a <STRONG>1:3</STRONG> proportion for the gBlock:vector. We added 6,6 uL of the PSB1C3 vector [11.3 ng/uL] and
        2,5 uL of the insert DNA [100 ng/uL].
    </DIV><DIV><STRONG>5) Purification</STRONG></DIV><DIV>
        We will purify the product using the
        <A href="https://www.evernote.com/Home.action?__fp=ZloDp4AdW-c3yWPvuidLz-TPR6I9Jhx8&amp;amp;username=iGEM-PB2014&amp;amp;login=true&amp;amp;_sourcePage=L90YhFqIo3HiMUD9T65RG_YvRLZ-1eYO3fqfqRu0fynRL_1nukNa4gH1t86pc1SP#b=60f448df-afb5-4a4d-9310-6dea624f7ab3&amp;amp;st=p&amp;amp;n=9dd0d893-9195-4ae2-9bbb-3748ecff72c9" target="_blank" shape="rect">
            QIAquick kit
        </A>
        .
        </DIV><DIV>
        This tube was called pPB.001
    </DIV></DIV><DIV><STRONG>6) Transform agaA Biobrick in E coli</STRONG></DIV><DIV><DIV><DIV><DIV>
                We will transform competent NEB turbo E coli made following the
                <A href="https://www.evernote.com/Home.action#b=60f448df-afb5-4a4d-9310-6dea624f7ab3&amp;st=p&amp;n=b4939a60-0f83-446a-a111-39d63eb7589c" target="_blank" shape="rect">
                    standard protocol
                </A>
                . In step 5, we did not incubate them for 4 hours, but only until the density was optimal (around 30-45 min).
            </DIV><DIV>
                We will use
                <A href="https://www.evernote.com/Home.action#b=596d48d5-2edd-48b4-b5e9-dc81ce1b9aab&amp;st=p&amp;n=c359be34-a042-4d01-a0bd-ebb7b81f3166" target="_blank" shape="rect">
                   agaA Biobrick
                </A>
                obtained today.
            </DIV><DIV>
                We will follow this
                <A href="https://www.evernote.com/Home.action?__fp=ZloDp4AdW-c3yWPvuidLz-TPR6I9Jhx8&amp;username=iGEM-PB2014&amp;login=true&amp;_sourcePage=L90YhFqIo3HiMUD9T65RG_YvRLZ-1eYO3fqfqRu0fynRL_1nukNa4gH1t86pc1SP#b=60f448df-afb5-4a4d-9310-6dea624f7ab3&amp;st=p&amp;n=b48f12a9-a15f-45a2-a179-095516e07b3e" target="_blank" shape="rect">
                    Heat Shock protocol
                </A>
                .
                </DIV><DIV><EM><STRONG>Tip</STRONG>
                    (From Matt): At the last step when we recover the cells, plate most of the 200 uL and leave the rest recovering overnight. Then plate them
                    the next day.
                    </EM></DIV></DIV><DIV>
            We will plate them in LBA+Cm. We will add 250 uL of Cm for 250 mL of LBA.
        </DIV></DIV></DIV><H6>Results: </H6><DIV>
    No colonies were seen 12h after. We will incubate more time. So we will (August 13th):
</DIV><DIV>
    - We will repeat the heat shock transformation with the ligation product that was left at 4ºC
</DIV><DIV>
    - We will run a gel with the ligation product
</DIV><H5 id="date">August 15th and 16th</H5><DIV>
    Because we cannot see any colonies on August 14th, we will:
</DIV><DIV><DIV>
        - We will repeat the heat shock transformation with the ligation product that was left at 4ºC
    </DIV><DIV>
        - We will run a gel with the ligation product
    </DIV><DIV>
        If it does not work, we should try electroporation.
    </DIV></DIV><H6>Goal:Check <I>E coli</I> transformation with agaA Biobrick </H6><H6>Procedure:</H6><DIV><STRONG>MiniPrep</STRONG></DIV><DIV>
    We will extract the agaA Biobrick by making a MiniPrep following the
    <A href="https://www.evernote.com/Home.action?__fp=ZloDp4AdW-c3yWPvuidLz-TPR6I9Jhx8&amp;username=iGEM-PB2014&amp;login=true&amp;_sourcePage=L90YhFqIo3HiMUD9T65RG_YvRLZ-1eYO3fqfqRu0fynRL_1nukNa4gH1t86pc1SP#b=60f448df-afb5-4a4d-9310-6dea624f7ab3&amp;st=p&amp;n=7103fd59-762f-4792-bb1a-280f7b7af3a3" target="_blank" shape="rect">
        standard protocol.
    </A></DIV><DIV>
    In the nanodrop, we saw that the yield was10 ug/mL.
    </DIV><DIV><STRONG>Digest</STRONG></DIV><DIV><DIV>
        Using the MiniPrep we got on August 13th, we will do a <STRONG>digestion analysis: </STRONG>(Jake's recipe)
        </DIV><DIV>
        - 5 uL plasmid (for the Ecoli we used 10 uL and added less water)
        </DIV><DIV>
        - 2 uL Fast Digest Green (has LD in it)
    </DIV><DIV>
        - 1 uL NotI or EcoRI+PstI. We used EcoRI+PstI as it was impossible to find NotI.
    </DIV><DIV>
        - 13uL of H2O
    </DIV><DIV>
        (Final volume of 20 uL)
    </DIV><DIV>
        We will incubate for 15-30 min at 37ºC. Then we will run 10 uL (our Fast Digest Green buffer already has Loadyng Dye in it, so no need to add
        more) in an agarose 1% gel.
    </DIV></DIV><H6>Results:</H6><P>We can see the pSB1C3 but not agaA. We are going to make a culture of the second plate we plated, and try again with more colonies tomorrow.</P><H5 id="date">August 17th</H5><H6>Goal:Check E coli transformation with the agaA Biobrick </H6><H6>Procedure:</H6><DIV><STRONG>MiniPrep</STRONG></DIV><DIV>
    We will extract the agaA Biobrick by making a MiniPrep of 8 colonies of the second plate we plated (we left a part of our liagation at 4ºC overnight and then
    transformed E coli) following the
    <A href="https://www.evernote.com/Home.action?__fp=ZloDp4AdW-c3yWPvuidLz-TPR6I9Jhx8&amp;username=iGEM-PB2014&amp;login=true&amp;_sourcePage=L90YhFqIo3HiMUD9T65RG_YvRLZ-1eYO3fqfqRu0fynRL_1nukNa4gH1t86pc1SP#b=60f448df-afb5-4a4d-9310-6dea624f7ab3&amp;st=p&amp;n=7103fd59-762f-4792-bb1a-280f7b7af3a3" target="_blank" shape="rect">
        standard protocol.
    </A></DIV><DIV><STRONG>Digest</STRONG></DIV><DIV><DIV>
        Using the MiniPrep we got on August 13th, we will do a <STRONG>digestion analysis: </STRONG>(Jake's recipe)
        </DIV><DIV>
        - 5 uL plasmid
        </DIV><DIV>
        - 2 uL Fast Digest Green (has LD in it)
    </DIV><DIV>
        - 1 uL NotI or EcoRI+PstI. We used EcoRI+PstI
    </DIV><DIV>
        - 13uL of H2O
    </DIV><DIV>
        (Final volume of 20 uL)
    </DIV><DIV>
        We will incubate for 15-30 min at 37ºC. Then we will run 10 uL (our Fast Digest Green buffer already has Loadyng Dye in it, so no need to add
        more) in an agarose 1% gel.
    </DIV><DIV><STRONG>PCR</STRONG></DIV><DIV>
        We also made a PCR to see the gBlock. We followed Jake's protocl for the
        <A href="https://www.evernote.com/Home.action?__fp=ZloDp4AdW-c3yWPvuidLz-TPR6I9Jhx8&amp;username=iGEM-PB2014&amp;login=true&amp;_sourcePage=L90YhFqIo3HiMUD9T65RG_YvRLZ-1eYO3fqfqRu0fynRL_1nukNa4gH1t86pc1SP#st=p&amp;n=65588d43-faa8-48c3-9114-884c43fad4d7" target="_blank" shape="rect">
            Fusion Polymerase
        </A>
        with the forward primer
        <A href="https://www.evernote.com/Home.action?__fp=IYz5qkyPGHc3yWPvuidLz-TPR6I9Jhx8&amp;username=iGEM-PB2014&amp;login=true&amp;_sourcePage=FOC90vQVhF_iMUD9T65RG_YvRLZ-1eYO3fqfqRu0fynRL_1nukNa4gH1t86pc1SP#b=596d48d5-2edd-48b4-b5e9-dc81ce1b9aab&amp;st=p&amp;n=caecc826-a5e8-4b0b-b8a8-14a5aad1eb7c" target="_blank" shape="rect">
            oPB.001
        </A>
        and the reverse primer
        <A href="https://www.evernote.com/Home.action?__fp=IYz5qkyPGHc3yWPvuidLz-TPR6I9Jhx8&amp;username=iGEM-PB2014&amp;login=true&amp;_sourcePage=FOC90vQVhF_iMUD9T65RG_YvRLZ-1eYO3fqfqRu0fynRL_1nukNa4gH1t86pc1SP#b=596d48d5-2edd-48b4-b5e9-dc81ce1b9aab&amp;st=p&amp;n=340942ec-952b-4a13-8f31-df72361990e4" target="_blank" shape="rect">
            oPB.005.
        </A></DIV></DIV><H6>Results:</H6><DIV>
    -We cannot see anything in the analysis by digestion.
</DIV><DIV>
    -The gel of the PCR confirms the<STRONG> presence of our gBlock with agaA</STRONG></DIV><DIV><DIV>
    - <STRONG>1st raw: </STRONG>Gene Ruler 100 kbPlus + 1uL Loading Dye
</DIV><DIV><STRONG><STRONG> - <STRONG>2nd raw:</STRONG></STRONG></STRONG>
    Colonies from the second transformation (Ligation was left overnight at 4ºC). There are 7 single colonies that were miniprep. We PCR with oPB.001 and oPB.005 (for the gBlock that contains agaA). We can see a band that does not correspond with the gBlock at around 350bp.
</DIV></DIV><H5>August 21st</H5><H6>Goal: Study in more details the colorless colonies having a CFP fluorescence</H6><H6>Procedure:</H6><P>
Colony PCR were performed following the protocol used June 25th.
The colony were also re-streaked on X-gal.
</P><H6>Results:</H6><P>The sequences matched the expected sequence of a WT strain.
The colonies were blue on X-gal.
We can deduce unperfect efficiency of the blue-white screening and re-adjust the figure on Crispr efficiency.
</P><DIV id="September"><H4>September</H4><H5 id="date">September 3rd</H5><H6>Goal:Check Ligation of agaA gBlock in pSEVA // YFP and BFP in pSEVA </H6><H6>Procedure:</H6><DIV><DIV><DIV><STRONG>Digestion</STRONG></DIV><DIV><DIV><DIV>We will digest <STRONG>v92 and V93 </STRONG>following Jake's recipe:</DIV><DIV>- Vector: 10 uL</DIV><DIV>- Green Buffer: 4 uL</DIV><DIV>- XbaI: 2 uL</DIV><DIV>- XhoI: 2 uL</DIV><DIV>- H20: 22 uL</DIV><DIV><SPAN>Total:</SPAN> 40 uL</DIV><DIV> </DIV><DIV><DIV>We will digest <STRONG>pSEVA </STRONG>(from my box 'pSEVA c1; 63 ug/uL) following Jake's recipe:</DIV><DIV>- Vector: 30 uL</DIV><DIV>- Buffer: 20 uL</DIV><DIV>- XbaI: 4 uL</DIV><DIV>- Sal1: 4 uL</DIV><DIV>- H20: 142 uL</DIV><DIV><SPAN>Total</SPAN>: 200 uL</DIV><DIV>We will digest the <STRONG>gBlock containing agaA </STRONG>(from my box 'gBlock aga Dig+Pur') following Jake's recipe:</DIV><DIV>- Vector: 30 uL</DIV><DIV>- Buffer: 20 uL</DIV><DIV>- EcoRI: 4 uL</DIV><DIV>- PstI: 4 uL</DIV><DIV>- H20: 142 uL`'</DIV><DIV><SPAN>Total</SPAN>: 200 uL</DIV><DIV> </DIV><DIV><DIV>We will digest <STRONG>pSEVA </STRONG>(from my box 'pSEVA c2; 140 ug/uL) following Jake's recipe:</DIV><DIV>- Vector: 20 uL</DIV><DIV>- Buffer: 20 uL</DIV><DIV>- EcoRI: 4 uL</DIV><DIV>- Pst1: 4 uL</DIV><DIV>- FastAP: 4 uL</DIV><DIV>- H20: 142 uL</DIV><DIV><SPAN>Total</SPAN>: 200 uL</DIV><DIV> </DIV><DIV>We will incubate all tubes at 37ºC for 2 hours.</DIV></DIV></DIV></DIV><DIV> </DIV></DIV><DIV><STRONG>Purify</STRONG></DIV><DIV><DIV>We will purify both tubes using the <A href="https://www.evernote.com/Home.action?__fp=ZloDp4AdW-c3yWPvuidLz-TPR6I9Jhx8&amp;amp;username=iGEM-PB2014&amp;amp;login=true&amp;amp;_sourcePage=L90YhFqIo3HiMUD9T65RG_YvRLZ-1eYO3fqfqRu0fynRL_1nukNa4gH1t86pc1SP#b=60f448df-afb5-4a4d-9310-6dea624f7ab3&amp;amp;st=p&amp;amp;n=9dd0d893-9195-4ae2-9bbb-3748ecff72c9" shape="rect" target="_blank">QIAquick kit</A>. </DIV><DIV> </DIV><DIV>These tubes were called agaA EP <SPAN>Dig + Pur</SPAN><SPAN>, pSEVA EP </SPAN><SPAN>Dig + Pur and pSEVA XS Dig+Pur</SPAN></DIV><DIV><SPAN> </SPAN></DIV><DIV><SPAN>- agaA EP Dig+Pur:</SPAN> Tube 1: 5,2 ng/mL;/ Tube 2: 5 ng/uL</DIV><DIV><SPAN>- pSEVA EP EP Dig+Pur:</SPAN><SPAN> Tube 1: 18,6 ng/mL; Tube 2: 8 ng/uL</SPAN></DIV><DIV><SPAN><SPAN>- pSEVA XS Dig+Pur:</SPAN><SPAN> Tube 1: 10 ng/mL; Tube 2: 7 ng/uL</SPAN></SPAN></DIV><DIV><SPAN> </SPAN></DIV></DIV><DIV><STRONG>Ligate pSEVA with the gBlock</STRONG></DIV><DIV><DIV><SPAN>Following the </SPAN><A href="http://www.thermoscientificbio.com/uploadedfiles/resources/el001-product-information.pdf" shape="rect" target="_blank">Thermo</A> <STRONG><A href="http://www.thermoscientificbio.com/uploadedfiles/resources/el001-product-information.pdf" shape="rect" target="_blank">T4 DNA Ligase protocol</A>.</STRONG> </DIV><DIV> </DIV><DIV>We will make a ligation. We will make a <STRONG>1:3</STRONG> proportion for the gBlock:vector. We added 3 uL of the pSEVA EP vector [18,6 ng/uL, 22,51 fmols] and 3 uL of the insert DNA [5,2 ng/uL, 5,6 fmols].</DIV><DIV> </DIV><DIV>We will leave the tube at 22ºC for 1h. Then the rest will be kept at 4ºC overnight.</DIV><DIV> </DIV><DIV>This tube was called <SPAN>agaA in pSEVA</SPAN></DIV></DIV></DIV><DIV> </DIV></DIV><H6>Results</H6><DIV><SPAN>I will take care of the pSEVA-gBlock ligation. This is the gel:</SPAN></DIV></DIV>

img src=&quot;https://static.igem.org/mediawiki/2014/7/7e/Gel_03-09_Dontsweatit_PB.JPG&quot;

<DIV><SPAN><STRONG>- 1st raw</STRONG></SPAN>: Loading dye 1kb</DIV><DIV><SPAN><STRONG>- 2nd raw:</STRONG></SPAN> sample of <SPAN>agaA EP Dig+Pur.</SPAN></DIV><DIV><SPAN><STRONG>- 3rd raw:</STRONG></SPAN> sample of <SPAN>pSEVA EP Dig+Pur.</SPAN></DIV><DIV><SPAN><STRONG>- 4th raw:</STRONG></SPAN> ligation product agaA in pSEVA (ligation of <SPAN>agaA EP Dig</SPAN>+Pur and <SPAN>pSEVA EP Dig+Pur</SPAN>)</DIV><DIV> </DIV><DIV>Ligation is <STRONG>successful!</STRONG> Next time, we should add more DNA and decrease water.</DIV><P> </P><H5>September 4th •Design des oligos</H5><H6>Goal: Design with Geneious primers for PCR cloning in order to clone the CRISPR/Cas system into the pSeva vector.</H6><H6>Procedure</H6><P>The primer design was made following the regular protocol (see 1st of June), but were added tails containing restriction sites for XbaI and SalI.</P><H5>September 8th •Cloning</H5><H6>Goal: Clone the whole CRISPR/Cas system in the pSeva vector in order to be able to transform a broad range of strains.</H6><H6>Procedure</H6><P>PCR were performed following the regular protocol.</P><P>PCR products were PCR purified, following the regular Qiagen PCR purification protocol.</P><P>Purified products were digested with XbaI and SalI, following the same protocol as the one followed the 25th of June.</P><P>Digestion products were PCR purified.</P><P>Purification products were used for ligation at room temperature for 2h, following this preparation :
<UL><LI>1uL of T4 Ligase</LI><LI>2uL of Ligase Buffer</LI><LI>15uL of NF water</LI><LI>1uL of vector</LI><LI>1uL of insert</LI></UL></P><P>The ligation products were transformed by electroporation in BL21-AI, and plated on Chloramphenicol plates for 24h at 37°C.</P><H6>Results</H6><P>Around 500 colonies were found after 24h hours of growing.</P><H5 id="date">September 16th</H5><H6>Goal:Transform agaA+pSEVA in <I>E. coli </I></H6><H6>Procedure:</H6><P>**There was a previous attempt where I directly transformed Corynebacterium with agaA+pSEVA. The rate of transformation of the product of a ligation is too low. First, we have to transform E. coli, that has a higher rate. Then we miniprep this plasmid and transform it in Corynebacterium.**</P><DIV><DIV><SPAN>We will transform competent NEB turbo E coli made following the <A href="https://www.evernote.com/Home.action#b=60f448df-afb5-4a4d-9310-6dea624f7ab3&amp;st=p&amp;n=b4939a60-0f83-446a-a111-39d63eb7589c" shape="rect" target="_blank">standard protocol</A> . In step 5, we did not incubate them for 4 hours, but only until the density was optimal (around 30-45 min).</SPAN></DIV><DIV><SPAN> </SPAN></DIV><DIV><SPAN>We will use agaA+pSEVA.</SPAN></DIV><DIV><SPAN> </SPAN></DIV><DIV><SPAN>We will follow this <A href="https://www.evernote.com/Home.action?__fp=ZloDp4AdW-c3yWPvuidLz-TPR6I9Jhx8&amp;username=iGEM-PB2014&amp;login=true&amp;_sourcePage=L90YhFqIo3HiMUD9T65RG_YvRLZ-1eYO3fqfqRu0fynRL_1nukNa4gH1t86pc1SP#b=60f448df-afb5-4a4d-9310-6dea624f7ab3&amp;st=p&amp;n=b48f12a9-a15f-45a2-a179-095516e07b3e" shape="rect" target="_blank">Heat Shock protocol</A>. </SPAN></DIV><DIV><SPAN> </SPAN></DIV><DIV><SPAN><EM><STRONG><SPAN>Tip</SPAN></STRONG> (From Matt): At the last step when we recover the cells, plate most of the 200 uL and leave the rest recovering overnight. Then plate them the next day. </EM></SPAN></DIV></DIV><DIV><SPAN>    </SPAN></DIV><DIV><SPAN>We will plate them in LBA+Cm. We will add 250 uL of Cm for 250 mL of LBA.  </SPAN></DIV><H6>Results</H6><DIV>Next day we had colonies. We started a culture to miniprep and check transformation.</DIV><H5 id="date">September 17th</H5><H6>Goal:Check <I>E. coli</I> transformation withagaA+pSEVA </H6><H6>Procedure:</H6><DIV><STRONG>MiniPrep</STRONG></DIV><DIV>We will extract agaA+pSEVA by making a MiniPrep following the <A href="https://www.evernote.com/Home.action?__fp=ZloDp4AdW-c3yWPvuidLz-TPR6I9Jhx8&amp;username=iGEM-PB2014&amp;login=true&amp;_sourcePage=L90YhFqIo3HiMUD9T65RG_YvRLZ-1eYO3fqfqRu0fynRL_1nukNa4gH1t86pc1SP#b=60f448df-afb5-4a4d-9310-6dea624f7ab3&amp;st=p&amp;n=7103fd59-762f-4792-bb1a-280f7b7af3a3" shape="rect" target="_blank">standard protocol.</A></DIV><DIV> </DIV><DIV> </DIV><DIV><STRONG>Digest</STRONG></DIV><DIV><DIV>We will do a <STRONG>digestion analysis: </STRONG>(Jake's recipe)</DIV><DIV>- 5 uL plasmid </DIV><DIV>- 2 uL Fast Digest Green (has LD in it)</DIV><DIV>- 1 uL NotI or EcoRI+PstI. We used NotI</DIV><DIV>- 13uL of H2O</DIV><DIV> </DIV><DIV>(Final volume of 20 uL)</DIV><DIV> </DIV><DIV>We will incubate for 15-30 min at 37ºC. Then we will run 10 uL (our Fast Digest Green buffer already has Loadyng Dye in it, so no need to add more) in an agarose 1% gel.</DIV><DIV> </DIV></DIV><H6>Results</H6><H6><DIV><DIV><SPAN><STRONG>- 1st raw: </STRONG><SPAN>Loadying Dye 1kb</SPAN></SPAN></DIV><DIV><SPAN><STRONG>- 2nd raw:</STRONG></SPAN> sample from culture 1: we threw it away. We do not see anything!</DIV><DIV>- <SPAN><STRONG>3rd raw:</STRONG></SPAN> sample from culture 2. Not the right band size</DIV><P> </P></DIV></H6><H5 id="date">September 18th</H5><H6>Goal:Transform <I>Corynebacterium striatum</I> with agaA+pSEVA </H6><H6>Procedure:</H6><DIV><SPAN><SPAN><SPAN><SPAN>We will use the eppendorf protocol (<A href="https://docs.google.com/file/d/0By8yVXC0fFVRMTlDd3BxQXM2Y1U/edit" shape="rect" target="_blank">https://docs.google.com/file/d/0By8yVXC0fFVRMTlDd3BxQXM2Y1U/edit</A>) for <EM>Corynebacterium</EM> electroporation.</SPAN></SPAN></SPAN></SPAN></DIV><DIV><SPAN><SPAN><SPAN> </SPAN></SPAN></SPAN></DIV><DIV><EM><SPAN><SPAN><SPAN>Procedure:</SPAN></SPAN></SPAN></EM></DIV><DIV><SPAN><EM><SPAN><SPAN><SPAN>     Cells were grown for approximately 18 h at 30ºC in a rotary shaker at 200 r.p.m. in 400 ml LB supplemented with kanamycin (50 pg ml-l) and glycine (2.5%).</SPAN></SPAN></SPAN></EM></SPAN></DIV><H6>Goal:Make competent <I>Corynebacterium </I></H6><H6>Procedure: </H6><DIV><SPAN><SPAN><SPAN><SPAN> We will use the eppendorf protocol (<A href="https://docs.google.com/file/d/0By8yVXC0fFVRMTlDd3BxQXM2Y1U/edit" shape="rect" target="_blank">https://docs.google.com/file/d/0By8yVXC0fFVRMTlDd3BxQXM2Y1U/edit</A>) for making competent cells, but when growing them overnight, we will follow Haynes and Britz protoc</SPAN><SPAN>ol (The effect of growth conditions of Corynebacterium glutamicum on the </SPAN><SPAN>transformation frequency obtained by electroporation). We did this because its been proven that adding glycine to the medium overnight increases the electrotransformation efficiency. This is the different part of the protocol:</SPAN></SPAN></SPAN></SPAN></DIV><DIV><SPAN><SPAN><SPAN> </SPAN></SPAN></SPAN></DIV><DIV><EM><SPAN><SPAN><SPAN>Procedure:</SPAN></SPAN></SPAN></EM></DIV><DIV><SPAN><EM><SPAN><SPAN><SPAN>     Cells were grown for approximately 18 h at 30ºC in a rotary shaker at 200 r.p.m. in 400 ml LB supplemented with kanamycin (50 pg ml-l) and glycine (2.5%).</SPAN></SPAN></SPAN></EM></SPAN></DIV><H5>September 20th •Study of clones</H5><H6>Goal: Study if the cloning was effective.</H6><H6>Procedure</H6><P><UL><LI>Liquid cultures were made out of colonies.</LI><LI>The liquid cultures were minipreped, and the minipreps were digested with HindIII and ran on a gel.</LI><LI>The minipreps were also sequenced.</LI></UL></P><H6>Results</H6><P>The expected bands appeared on the gel (around 6100, 2300, 1700).</P><P>The expected sequence resulted from the sequencing.</P><H5 id="date">September 23rd</H5><H6>Goal:Obtan agaA Biobrick</H6><H6>Procedure:</H6><DIV><STRONG>Ligate PSB1C3 with the gBlock</STRONG></DIV><DIV><STRONG> </STRONG></DIV><DIV><DIV><SPAN>Following the </SPAN><A href="http://www.thermoscientificbio.com/uploadedfiles/resources/el001-product-information.pdf" shape="rect" target="_blank">Thermo</A> <STRONG><A href="http://www.thermoscientificbio.com/uploadedfiles/resources/el001-product-information.pdf" shape="rect" target="_blank">T4 DNA Ligase protocol</A>.</STRONG> Instead of leaving the tube at 22ºC for 10 min, we left it at room temperature for 45 min.</DIV><DIV> </DIV><DIV>We will use tubes<SPAN> PSB1C3 Dig+Pur</SPAN> and <SPAN>gBlock EP Dig+Pur</SPAN></DIV><DIV> </DIV><DIV>We will make a ligation. We will make a <STRONG>1:3</STRONG> proportion for the gBlock:vector. We added 6 uL of the PSB1C3 vector [11 ng/uL] and 6 uL of the insert DNA [5 ng/uL].</DIV><DIV> </DIV><DIV><EM>**Usually we would add 3 uL for each but we are doubling the concentration of DNA and decreasing the H20**</EM></DIV></DIV><H6>Results:</H6><P>We runned a 1% agarose gel. The gel shows that there is nothing on the PSB1C3 tube. We will alplify PSB1C3 from Antonio's stock using oPB.082 and oPB.083 using. </P><H6>Goal:PCR PSB1C3</H6><H6>Procedure:</H6><P><SPAN> PCR of PSB1C3 from Antonio's box using the </SPAN><A href="https://www.evernote.com/Home.action?__fp=ZloDp4AdW-c3yWPvuidLz-TPR6I9Jhx8&amp;username=iGEM-PB2014&amp;login=true&amp;_sourcePage=L90YhFqIo3HiMUD9T65RG_YvRLZ-1eYO3fqfqRu0fynRL_1nukNa4gH1t86pc1SP#st=p&amp;n=65588d43-faa8-48c3-9114-884c43fad4d7" shape="rect" target="_blank">Fusion Polymerase</A><SPAN> and primers oPB.082 and oPB.083.</SPAN></P><H5 id="date">September 24th</H5><H6>Goal:Transform agaA+pSEVA in <I>E. coli </I></H6><H6>Procedure:</H6><P><SPAN>We will purify the PCR product obtained on Sep 23rd following the </SPAN><A href="https://www.evernote.com/Home.action?__fp=ZloDp4AdW-c3yWPvuidLz-TPR6I9Jhx8&amp;username=iGEM-PB2014&amp;login=true&amp;_sourcePage=L90YhFqIo3HiMUD9T65RG_YvRLZ-1eYO3fqfqRu0fynRL_1nukNa4gH1t86pc1SP#b=60f448df-afb5-4a4d-9310-6dea624f7ab3&amp;st=p&amp;n=9dd0d893-9195-4ae2-9bbb-3748ecff72c9" shape="rect" target="_blank">QIAquick PCR purification kit standard protocol</A><SPAN>.</SPAN></P><H6>Goal: Obtain agaA Biobrick (new PSB1C3 digestion + ligation)</H6><H6>Procedure:</H6><DIV>We will digest the purified PSB1C3 following the <A href="http://www.thermoscientificbio.com/uploadedFiles/Resources/fast-digestion-dna.pdf" shape="rect" target="_blank">Thermo Fast Digestion Protocol</A> for enzimes <STRONG>EcoRI and PstI.</STRONG> Then we will ligate it with our <SPAN>agaA Dig+Pur</SPAN>. We want to obtain <A href="https://www.evernote.com/Home.action?__fp=ZloDp4AdW-c3yWPvuidLz-TPR6I9Jhx8&amp;amp;username=iGEM-PB2014&amp;amp;login=true&amp;amp;_sourcePage=L90YhFqIo3HiMUD9T65RG_YvRLZ-1eYO3fqfqRu0fynRL_1nukNa4gH1t86pc1SP#b=596d48d5-2edd-48b4-b5e9-dc81ce1b9aab&amp;amp;st=p&amp;amp;n=c359be34-a042-4d01-a0bd-ebb7b81f3166" shape="rect" target="_blank">agaA Biobrick</A> and transform E coli. </DIV><DIV> </DIV><DIV><STRONG>Digestion</STRONG></DIV><DIV> </DIV><DIV>  1) We are using the PSB1C3 that we just PCRed and purified.</DIV><DIV> </DIV><DIV>We will follow Jake's recipe:</DIV><DIV><DIV>- Vector: 30 uL</DIV><DIV>- Buffer: 20 uL</DIV><DIV>- EcoRI: 4 uL</DIV><DIV>- PstI: 4 uL</DIV><DIV>- H20: 142 uL</DIV><DIV><SPAN>Total</SPAN>: 200 uL</DIV><DIV> </DIV></DIV><DIV>   2) Then we put it at 37ºC at the Isotemp for 2h</DIV><DIV> </DIV><DIV>This tube is called <SPAN>PSB1C3 Dig EP</SPAN></DIV><DIV> </DIV><DIV><STRONG>Purification</STRONG></DIV><DIV> </DIV><DIV>   4) We will purify both tubes using the <A href="https://www.evernote.com/Home.action?__fp=ZloDp4AdW-c3yWPvuidLz-TPR6I9Jhx8&amp;amp;username=iGEM-PB2014&amp;amp;login=true&amp;amp;_sourcePage=L90YhFqIo3HiMUD9T65RG_YvRLZ-1eYO3fqfqRu0fynRL_1nukNa4gH1t86pc1SP#b=60f448df-afb5-4a4d-9310-6dea624f7ab3&amp;amp;st=p&amp;amp;n=9dd0d893-9195-4ae2-9bbb-3748ecff72c9" shape="rect" target="_blank">QIAquick kit</A>. </DIV><DIV> </DIV><DIV>These tubes were called <SPAN>Dig + Pur gBlock</SPAN> and <SPAN>Dig + Pur PSB1C3</SPAN></DIV><DIV> </DIV><DIV><STRONG>Ligation</STRONG></DIV><DIV><STRONG> </STRONG></DIV><DIV><SPAN>Following the </SPAN><A href="http://www.thermoscientificbio.com/uploadedfiles/resources/el001-product-information.pdf" shape="rect" target="_blank">Thermo</A> <STRONG><A href="http://www.thermoscientificbio.com/uploadedfiles/resources/el001-product-information.pdf" shape="rect" target="_blank">T4 DNA Ligase protocol</A>.</STRONG> Instead of leaving the tube at 22ºC for 10 min, we left it at room temperature for 1h.</DIV><DIV> </DIV><DIV>   5) We will make a ligation. We will make a <STRONG>1:3</STRONG> proportion for the gBlock:vector. <SPAN>We added 3 uL of the PSB1C3 vector [2 ng/uL] and 10.5 uL of the insert DNA [5 ng/uL].</SPAN></DIV><P><SPAN> </SPAN></P><DIV> </DIV><H6>Goal:Transformation of <I>Corynebacterium striatum</I> with agaA +pSEVA </H6><H6>Procedure:</H6><P>    We will use the eppendorf protocol (https://docs.google.com/file/d/0By8yVXC0fFVRMTlDd3BxQXM2Y1U/edit) for Corynebacterium electroporation.

    As a contol, we will also transform <I>Corynebacterium striatum</I> with V5 from Jake (pSEVA+cerulean).</P><H6>Results:</H6><H5 id="date">September 25th</H5><H6>Goal:Old agaA Biobrick analytical digestion</H6><H6>Procedure:</H6><DIV><DIV>We will check the tube of E coli with agaA Biobrick that we made in August 17th. We will do a <STRONG>miniprep</STRONG> and <STRONG>analytical digestion</STRONG>.</DIV><DIV> </DIV><DIV><DIV><STRONG>MiniPrep</STRONG></DIV><DIV>We will extract the agaA Biobrick by making a MiniPrep following the <A href="https://www.evernote.com/Home.action?__fp=ZloDp4AdW-c3yWPvuidLz-TPR6I9Jhx8&amp;username=iGEM-PB2014&amp;login=true&amp;_sourcePage=L90YhFqIo3HiMUD9T65RG_YvRLZ-1eYO3fqfqRu0fynRL_1nukNa4gH1t86pc1SP#b=60f448df-afb5-4a4d-9310-6dea624f7ab3&amp;st=p&amp;n=7103fd59-762f-4792-bb1a-280f7b7af3a3" shape="rect" target="_blank">standard protocol.</A></DIV><DIV> </DIV><DIV>In the nanodrop, we see that the concentration was 20 ng/uL (low)</DIV><DIV> </DIV><DIV><STRONG>Digest</STRONG></DIV><DIV><DIV>Using the MiniPrep we got on August 13th, we will do a <STRONG>digestion analysis: </STRONG>(Jake's recipe)</DIV><DIV>- 10 uL plasmid (we would usually put 5 but the concentration is very low)</DIV><DIV>- 2 uL Fast Digest Green (has LD in it)</DIV><DIV>- 1 uL NotI or EcoRI+PstI. We used EcoRI+PstI </DIV><DIV>- 7uL of H2O</DIV><DIV>(Final volume of 20 uL)</DIV><DIV> </DIV><DIV>We will incubate for 15 min at 37ºC. Then we will run 10 uL (our Fast Digest Green buffer already has Loadyng Dye in it, so no need to add more) in an agarose 1% gel.</DIV><P> </P></DIV></DIV></DIV><DIV> </DIV><H6>Results:</H6><P> There is nothing in the gel </P><H6>Goal:Old agaA Biobrick analytical digestion</H6><H6>Procedure:</H6><DIV><DIV>   In case the old transformation analysis is negative (and because I have not been able to sequence verify it), we are making a new transformation. There are colonies in yesterday's plates, so we will <STRONG>start cultures</STRONG> to miniprep and do analytical digestion tomorrow.</DIV><P> </P></DIV><H6>Goal:Ligation agaA gBlock in pSEVA</H6><H6>Procedure:</H6><DIV><STRONG>Ligate pSEVA with the gBlock</STRONG></DIV><DIV><DIV><SPAN>Following the </SPAN><A href="http://www.thermoscientificbio.com/uploadedfiles/resources/el001-product-information.pdf" shape="rect" target="_blank">Thermo</A> <STRONG><A href="http://www.thermoscientificbio.com/uploadedfiles/resources/el001-product-information.pdf" shape="rect" target="_blank">T4 DNA Ligase protocol</A>.</STRONG> </DIV><DIV> </DIV><DIV>We will make a ligation. We will make a <STRONG>1:3</STRONG> proportion for the gBlock:vector. We added 3 uL of the pSEVA EP vector [18,6 ng/uL, 22,51 fmols] and 3 uL of the insert DNA [5,2 ng/uL, 5,6 fmols].</DIV><DIV> </DIV><DIV>- 3uL pSeva</DIV><DIV>- 3uL agaA</DIV><DIV>-1 uL Buffer</DIV><DIV>- 0.5 uL Ligase</DIV><DIV>- 2.5 uL H2O</DIV><DIV> </DIV><DIV>We will leave the tube at room temperature for 1h. Then the rest will be kept at 4ºC overnight.</DIV><DIV> </DIV><DIV>This tube was called <SPAN>agaA+pSEVA</SPAN></DIV><DIV><SPAN> </SPAN></DIV><DIV> </DIV><DIV><DIV><SPAN><STRONG>Procedure:</STRONG></SPAN></DIV><DIV><STRONG> </STRONG></DIV><DIV><DIV><DIV><SPAN>We will transform competent NEB turbo E coli made following the standard protocol . In step 5, we did not incubate them for 4 hours, but only until the density was optimal (around 30-45 min).</SPAN></DIV><DIV><SPAN> </SPAN></DIV><DIV><SPAN>We will use agaA+pSEVA.</SPAN></DIV><DIV><SPAN> </SPAN></DIV><DIV><SPAN>We will follow this <A href="https://www.evernote.com/Home.action?__fp=ZloDp4AdW-c3yWPvuidLz-TPR6I9Jhx8&amp;username=iGEM-PB2014&amp;login=true&amp;_sourcePage=L90YhFqIo3HiMUD9T65RG_YvRLZ-1eYO3fqfqRu0fynRL_1nukNa4gH1t86pc1SP#b=60f448df-afb5-4a4d-9310-6dea624f7ab3&amp;st=p&amp;n=b48f12a9-a15f-45a2-a179-095516e07b3e" shape="rect" target="_blank">Heat Shock protocol</A>. </SPAN></DIV><DIV><SPAN> </SPAN></DIV><DIV><SPAN><EM><STRONG><SPAN>Tip</SPAN></STRONG> (From Matt): At the last step when we recover the cells, plate most of the 200 uL and leave the rest recovering overnight. Then plate them the next day. </EM></SPAN></DIV></DIV><DIV><SPAN>    </SPAN></DIV><DIV><SPAN>We will plate them in LBA+Cm. We will add 250 uL of Cm for 250 mL of LBA.  </SPAN></DIV><P><SPAN> </SPAN></P></DIV></DIV></DIV><H5 id="date">September 26th</H5><H6>Goal: agaA Biobrick miniprep and analytical digestion</H6><H6>Procedure:</H6><DIV>  We will check the tube of E coli with the agaA Biobrick (4 different colonies). We will do a <STRONG>miniprep</STRONG> and <STRONG>analytical digestion</STRONG>.</DIV><DIV> </DIV><DIV><DIV><STRONG>MiniPrep</STRONG></DIV><DIV>We will extract the agaA Biobrick of 4 different colonies by making a MiniPrep following the <A href="https://www.evernote.com/Home.action?__fp=ZloDp4AdW-c3yWPvuidLz-TPR6I9Jhx8&amp;username=iGEM-PB2014&amp;login=true&amp;_sourcePage=L90YhFqIo3HiMUD9T65RG_YvRLZ-1eYO3fqfqRu0fynRL_1nukNa4gH1t86pc1SP#b=60f448df-afb5-4a4d-9310-6dea624f7ab3&amp;st=p&amp;n=7103fd59-762f-4792-bb1a-280f7b7af3a3" shape="rect" target="_blank">standard protocol.</A></DIV><DIV> </DIV><DIV> </DIV><DIV><STRONG>Digest</STRONG></DIV><DIV><DIV>We will do a <STRONG>digestion analysis: </STRONG>(Jake's recipe)</DIV><DIV>- 5 uL plasmid</DIV><DIV>- 2 uL Fast Digest Green (has LD in it)</DIV><DIV>- 1 uL NotI or EcoRI+PstI. We used EcoRI+PstI </DIV><DIV>- 12 uL of H2O</DIV><DIV>(Final volume of 20 uL)</DIV><DIV> </DIV><DIV>We will incubate for 15 min at 37ºC. Then we will run 10 uL (our Fast Digest Green buffer already has Loadyng Dye in it, so no need to add more) in an agarose 1% gel.</DIV><P> </P></DIV></DIV><H6>Results:</H6><P> We can see two bands. One around 2kb and other around 1.5 kb. This would be PSB1C3 and the gBlock. We will make a stock of colony 4.</P><H5 id="date">September 27th</H5><H6>Goal:agaA+pSEVA miniprep and analytical digestion</H6><H6>Procedure:</H6><DIV><DIV>   We will check the tube of E coli with agaA+pSEVA (4 different colonies). We will do a <STRONG>miniprep</STRONG> and <STRONG>analytical digestion</STRONG>.</DIV><DIV> </DIV><DIV> </DIV><DIV><DIV><STRONG>MiniPrep</STRONG></DIV><DIV>We will extract agaA+pSEVA of 4 different colonies by making a MiniPrep following the <A href="https://www.evernote.com/Home.action?__fp=ZloDp4AdW-c3yWPvuidLz-TPR6I9Jhx8&amp;username=iGEM-PB2014&amp;login=true&amp;_sourcePage=L90YhFqIo3HiMUD9T65RG_YvRLZ-1eYO3fqfqRu0fynRL_1nukNa4gH1t86pc1SP#b=60f448df-afb5-4a4d-9310-6dea624f7ab3&amp;st=p&amp;n=7103fd59-762f-4792-bb1a-280f7b7af3a3" shape="rect" target="_blank">standard protocol.</A></DIV><DIV> </DIV><DIV> </DIV><DIV><STRONG>Digest</STRONG></DIV><DIV><DIV>We will do a <STRONG>digestion analysis: </STRONG>(Jake's recipe)</DIV><DIV>- 5 uL plasmid</DIV><DIV>- 2 uL Fast Digest Green (has LD in it)</DIV><DIV>- 1 uL NotI or EcoRI+PstI. We used EcoRI+PstI </DIV><DIV>- 12 uL of H2O</DIV><DIV>(Final volume of 20 uL)</DIV><DIV> </DIV><DIV>We will incubate for 15 min at 37ºC. Then we will run 10 uL (our Fast Digest Green buffer already has Loadyng Dye in it, so no need to add more) in an agarose 1% gel.</DIV></DIV></DIV></DIV><DIV><DIV><P> </P></DIV></DIV><H6>Results:</H6><H6><P>We can see pSEVA band at 5000 bp and there is another band around 1500 bp. It is not a very strong band but is there. Also, the culture tube smells like body odor, so even if this band is not very strong, we think it is agaA. We will use the miniprep product of colony 1 to transform <I>Corynebacterium striatum</I>.</P></H6><H6>Goal:Transform <I>Corynebacterium striatum</I> with agaA+pSEVA</H6><H6>Procedure:</H6><DIV><STRONG><SPAN><SPAN><SPAN>  </SPAN></SPAN></SPAN></STRONG><SPAN><SPAN><SPAN><SPAN> We will use the eppendorf protocol (<A href="https://docs.google.com/file/d/0By8yVXC0fFVRMTlDd3BxQXM2Y1U/edit" shape="rect" target="_blank">https://docs.google.com/file/d/0By8yVXC0fFVRMTlDd3BxQXM2Y1U/edit</A>) for <EM>Corynebacterium</EM> electroporation.</SPAN></SPAN></SPAN></SPAN></DIV><DIV><SPAN><SPAN><SPAN><SPAN>   We will use the miniprep product that we just obtained (agaA+pSEVA colony 1). We will recover 2h instead of 1h, as for Cm resistance longer recovery times are advised.</SPAN></SPAN></SPAN></SPAN></DIV><P><SPAN><SPAN><SPAN><SPAN> </SPAN></SPAN></SPAN></SPAN></P><H5>September 27th •Killing Efficiency</H5><H6>Goal: Compare the efficacy of the new CRISPR vector with the previous one.</H6><H6>Procedure</H6><P>The same expriment as July 10th was conducted, for the pCas9 and the pSeva vectors.</P><H6>Results</H6><TABLE class="tableizer-table"><TBODY><TR class="tableizer-firstrow"><TH>WT with non targeting pCas9</TH><TH>Mutant with non targeting pCas9</TH></TR><TR><TD>Replicate 1</TD><TD>1845</TD><TD>1745</TD></TR><TR><TD>Replicate 2</TD><TD>2398</TD><TD>2354</TD></TR><TR><TD>Replicate 3</TD><TD>1549</TD><TD>1860</TD></TR><TR><TD>Replicate 4</TD><TD>1740</TD><TD>1754</TD></TR><TR><TD>Replicate 5</TD><TD>2176</TD><TD>1956</TD></TR><TR><TD> </TD><TD>WT with non targeting pSeva</TD><TD>Mutant with non targeting pSeva</TD></TR><TR><TD>Replicate 1</TD><TD>2398</TD><TD>2230</TD></TR><TR><TD>Replicate 2</TD><TD>1876</TD><TD>1984</TD></TR><TR><TD>Replicate 3</TD><TD>2784</TD><TD>2093</TD></TR><TR><TD>Replicate 4</TD><TD>2009</TD><TD>2092</TD></TR><TR><TD>Replicate 5</TD><TD>1997</TD><TD>1700</TD></TR><TR><TD> </TD><TD>WT with LacZ targeting pCas9</TD><TD>Mutant with LacZ targeting pCas9</TD></TR><TR><TD>Replicate 1</TD><TD>5</TD><TD>742</TD></TR><TR><TD>Replicate 2</TD><TD>40</TD><TD>543</TD></TR><TR><TD>Replicate 3</TD><TD>12</TD><TD>984</TD></TR><TR><TD>Replicate 4</TD><TD>2</TD><TD>659</TD></TR><TR><TD>Replicate 5</TD><TD>26</TD><TD>978</TD></TR></TBODY></TABLE><H5 id="date">September 29th</H5><P>The sequencing of agaA Biobrick and agaA+pSEVA was negative, so we will throw again everything and start from the very beggining </P><DIV>Digest pSB1C3 vector.</DIV><DIV> </DIV><DIV><SPAN>P28 (pSB1C3) at 160 ng/ul</SPAN><TABLE cellspacing="0" cellpadding="0"><TBODY><TR><TD valign="middle"><SPAN><STRONG>Reagent</STRONG></SPAN></TD><TD valign="middle"><SPAN><STRONG>Volume</STRONG></SPAN></TD></TR><TR><TD valign="middle"><SPAN>Plasmid</SPAN></TD><TD valign="middle"><SPAN>30 ul</SPAN></TD></TR><TR><TD valign="middle"><SPAN>H2O</SPAN></TD><TD valign="middle"><SPAN>135 ul</SPAN></TD></TR><TR><TD valign="middle"><SPAN>10x FastDigest Green Buffer</SPAN></TD><TD valign="middle"><SPAN>20 ul</SPAN></TD></TR><TR><TD valign="middle"><SPAN>FastDigest XbaI</SPAN></TD><TD valign="middle"><SPAN>5 ul</SPAN></TD></TR><TR><TD valign="middle"><SPAN>FastDigest PstI</SPAN></TD><TD valign="middle"><SPAN>5 ul</SPAN></TD></TR><TR><TD valign="middle"><SPAN>FastAP</SPAN></TD><TD valign="middle"><SPAN>5 ul</SPAN></TD></TR><TR><TD valign="middle"><SPAN><STRONG>Total Volume</STRONG></SPAN></TD><TD valign="middle"><SPAN>200 ul</SPAN></TD></TR></TBODY></TABLE><SPAN>Gel extract and purify. The vector concentration after purification was 6.5 ng/ul</SPAN></DIV><DIV>PCR for agaA gBlock</DIV><DIV> </DIV><DIV><SPAN>Primers: oPB.001, oPB.005</SPAN><SPAN>Template gBlock agaA 1:50 dilution</SPAN></DIV><DIV><TABLE cellspacing="0" cellpadding="0"><TBODY><TR><TD valign="middle"><SPAN><STRONG>Reagent</STRONG></SPAN></TD><TD colspan="2" valign="middle"><SPAN><STRONG>Volume</STRONG></SPAN></TD></TR><TR><TD valign="middle"><SPAN> </SPAN></TD><TD valign="middle"><SPAN><STRONG>1x</STRONG></SPAN></TD><TD valign="top"><SPAN><STRONG>4x</STRONG></SPAN></TD></TR><TR><TD valign="middle"><SPAN>Nuclease-free water</SPAN></TD><TD valign="middle"><SPAN>71 ul</SPAN></TD><TD valign="top"><SPAN>284 ul</SPAN></TD></TR><TR><TD valign="middle"><SPAN>5x Phusion HF Buffer</SPAN></TD><TD valign="middle"><SPAN>20 ul</SPAN></TD><TD valign="top"><SPAN>80 ul</SPAN></TD></TR><TR><TD valign="middle"><SPAN>10 mM dNTPs</SPAN></TD><TD valign="middle"><SPAN>2 ul</SPAN></TD><TD valign="top"><SPAN>8 ul</SPAN></TD></TR><TR><TD valign="middle"><SPAN>Forward Primer (10 uM)</SPAN></TD><TD valign="middle"><SPAN>1 ul</SPAN></TD><TD valign="top"><SPAN>4 ul</SPAN></TD></TR><TR><TD valign="middle"><SPAN>Reverse Primer (10 uM)</SPAN></TD><TD valign="middle"><SPAN>1 ul</SPAN></TD><TD valign="top"><SPAN>4 ul</SPAN></TD></TR><TR><TD valign="middle"><SPAN>Template DNA (Plasmid, 1:10 dilution)</SPAN></TD><TD valign="middle"><SPAN>1 ul</SPAN></TD><TD valign="top"><SPAN> 4 ul</SPAN></TD></TR><TR><TD valign="middle"><SPAN>DMSO</SPAN></TD><TD valign="middle"><SPAN>3 ul</SPAN></TD><TD valign="top"><SPAN>12 ul</SPAN></TD></TR><TR><TD valign="middle"><SPAN>Phusion DNA Polymerase</SPAN></TD><TD valign="middle"><SPAN>1 ul</SPAN></TD><TD valign="top"><SPAN>4 ul</SPAN></TD></TR><TR><TD valign="middle"><SPAN><STRONG>Total Volume</STRONG></SPAN></TD><TD valign="middle"><SPAN>100 ul</SPAN></TD><TD valign="top"><SPAN> </SPAN></TD></TR></TBODY></TABLE><TABLE cellspacing="0" cellpadding="0"><TBODY><TR><TD colspan="5" valign="top"><SPAN><STRONG>Thermocycler Protocol: NEB Phusion</STRONG></SPAN></TD></TR><TR><TD valign="top"><SPAN> </SPAN></TD><TD valign="top"><SPAN>Temp</SPAN></TD><TD valign="top"><SPAN>Time</SPAN></TD><TD valign="top"><SPAN> </SPAN></TD><TD valign="top"><SPAN> </SPAN></TD></TR><TR><TD valign="top"><SPAN>Start</SPAN></TD><TD valign="top"><SPAN>98 °C          </SPAN></TD><TD valign="top"><SPAN>30 sec</SPAN></TD><TD valign="top"><SPAN>Melt</SPAN></TD><TD valign="top"><SPAN> </SPAN></TD></TR><TR><TD colspan="5" valign="top"><SPAN> </SPAN></TD></TR><TR><TD valign="top"><SPAN>Cycle 1          </SPAN></TD><TD valign="top"><SPAN>98 °C          </SPAN></TD><TD valign="top"><SPAN>10 sec</SPAN></TD><TD valign="top"><SPAN>Melt</SPAN></TD><TD rowspan="3" valign="top"><SPAN> </SPAN><SPAN>35 Cycles          </SPAN></TD></TR><TR><TD valign="top"><SPAN>Cycle 2          </SPAN></TD><TD valign="top"><SPAN>52 °C          </SPAN></TD><TD valign="top"><SPAN>30 sec</SPAN></TD><TD valign="top"><SPAN>Anneal</SPAN></TD></TR><TR><TD valign="top"><SPAN>Cycle 3          </SPAN></TD><TD valign="top"><SPAN>72 °C          </SPAN></TD><TD valign="top"><SPAN>40 sec</SPAN></TD><TD valign="top"><SPAN>Extend</SPAN></TD></TR><TR><TD colspan="5" valign="top"><SPAN> </SPAN></TD></TR><TR><TD valign="top"><SPAN>Finish</SPAN></TD><TD valign="top"><SPAN>72 °C          </SPAN></TD><TD valign="top"><SPAN>5 min</SPAN></TD><TD valign="top"><SPAN>Extend</SPAN></TD><TD valign="top"><SPAN> </SPAN></TD></TR><TR><TD valign="top"><SPAN>Store</SPAN></TD><TD valign="top"><SPAN>10 °C          </SPAN></TD><TD valign="top"><SPAN>Forever</SPAN></TD><TD valign="top"><SPAN>Store</SPAN></TD><TD valign="top"><SPAN>     </SPAN></TD></TR></TBODY></TABLE>PCR Product looks good</DIV><DIV><SPAN>PCR purify gBlock agaA</SPAN><SPAN>The concentration is 80 ng/ul</SPAN><SPAN>Digest for cloning:</SPAN><TABLE cellspacing="0" cellpadding="0"><TBODY><TR><TD valign="middle"><SPAN><STRONG>Reagent</STRONG></SPAN></TD><TD valign="middle"><SPAN><STRONG>Volume</STRONG></SPAN></TD></TR><TR><TD valign="middle"><SPAN>PCR Product</SPAN></TD><TD valign="middle"><SPAN>30 ul</SPAN></TD></TR><TR><TD valign="middle"><SPAN>H2O</SPAN></TD><TD valign="middle"><SPAN>135 ul</SPAN></TD></TR><TR><TD valign="middle"><SPAN>10x FastDigest Buffer</SPAN></TD><TD valign="middle"><SPAN>20 ul</SPAN></TD></TR><TR><TD valign="middle"><SPAN>FastDigest XbaI</SPAN></TD><TD valign="middle"><SPAN>10 ul</SPAN></TD></TR><TR><TD valign="middle"><SPAN>FastDigest PstI</SPAN></TD><TD valign="middle"><SPAN>10 ul</SPAN></TD></TR><TR><TD valign="middle"> </TD><TD valign="middle"> </TD></TR><TR><TD valign="middle"><SPAN><STRONG>Total Volume</STRONG></SPAN></TD><TD valign="middle"><SPAN>200 ul</SPAN></TD></TR></TBODY></TABLE><SPAN>Resuspend in 50 ul.</SPAN><SPAN>The final concentration is 36 ng/ul</SPAN> </DIV><DIV> </DIV><DIV> </DIV><DIV><SPAN>Vector P28 (6 ng/ul)</SPAN><SPAN>Vector P41 diluted 1:10 (5 ng/ul)</SPAN><SPAN>Insert gBlock agaA (36 ng/ul)</SPAN><SPAN>Enzymes: XbaI, PstI</SPAN><TABLE cellspacing="0" cellpadding="0"><TBODY><TR><TD valign="middle"><SPAN><STRONG>Reagent</STRONG></SPAN></TD><TD valign="middle"><SPAN><STRONG>Volume</STRONG></SPAN></TD></TR><TR><TD valign="middle"><SPAN>Vector</SPAN></TD><TD valign="middle"><SPAN>12 ul</SPAN></TD></TR><TR><TD valign="middle"><SPAN>Insert</SPAN></TD><TD valign="middle"><SPAN>5 ul</SPAN></TD></TR><TR><TD valign="middle"><SPAN>H</SPAN><SPAN><SUB>2</SUB></SPAN><SPAN>O</SPAN></TD><TD valign="middle"><SPAN>0 ul</SPAN></TD></TR><TR><TD valign="middle"><SPAN>Fermentas</SPAN><SPAN>T4 Ligase Buffer</SPAN></TD><TD valign="middle"><SPAN>2 ul</SPAN></TD></TR><TR><TD valign="middle"><SPAN>Fermentas</SPAN><SPAN>T4 Ligase Enzyme   </SPAN></TD><TD valign="middle"><SPAN>1 ul</SPAN></TD></TR><TR><TD valign="middle"><SPAN><STRONG>Total Volume</STRONG></SPAN></TD><TD valign="middle"><SPAN>20 ul</SPAN></TD></TR></TBODY></TABLE><SPAN>Incubate for 10 min at 22C (manufacturer’s suggestion)</SPAN><SPAN>I ran our the ligation controls. The gel looks a little funny. I think it is overloaded. There is some evidence that the ligation worked.</SPAN></DIV><P>Miniprep and analytical digest with XbaI, PstI.

V96 is P28 (pSB1C3) + agaA

I picked 6 colonies from the V96 transformation. Clones 4, 5 and 6 all produced the correct banding pattern. I chose Clone 6 because the miniprep had the highest concentraion.
</P><P>We made a culture and sent it to GATC with the verification primers. The sequencing was positive. </P></DIV></DIV></DIV><DIV id="mfooter"><DIV class="footbox"><B>Centre for Research and Interdisciplinarity (CRI)</B>
    Faculty of Medicine Cochin Port-Royal, South wing, 2nd floor
    Paris Descartes University
    24, rue du Faubourg Saint Jacques
    75014 Paris, France +33 1 44 41 25 22/25<A href="mailto:paris-bettencourt-igem@googlegroups.com">paris-bettencourt-igem@googlegroups.com</A></DIV><CENTER>
    Copyright (c) 2014 igem.org. All rights reserved.
  </CENTER></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Paris_Bettencourt/Notebook/Eliminate_Smell">http://2014.igem.org/Team:Paris_Bettencourt/Notebook/Eliminate_Smell</A>&quot;</DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Paris_Bettencourt/Notebook/Eliminate_Smell" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Paris_Bettencourt/Notebook/Eliminate_Smell" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Paris_Bettencourt/Notebook/Eliminate_Smell&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Paris_Bettencourt/Notebook/Eliminate_Smell&amp;oldid=399848" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>