"
Page
Discussion
View source
History
teams
Log in
 
Team:Paris Bettencourt/Project/Odor Library
From 2014.igem.org
Project
SmellThe Roses
SomethingFishy
GoodyTwo Shoes
Don'tSweat It
Teen Spirit
Design Practice
Bioinformatics
Achievements
Results
Interlab Study
MedalFulfilment
Parts
Policy
Safety
Consent Form
Patenting
Practices
MOOC
CitizenScience
The Senseof Smell
The HumanMicriobiome
Minder
Collaborations
Collaborations
The SmellGame
Newsletters
Notebook
Daily Note
Protocols
Bibliography
Team
Our Team
Attributions
Official TeamProfile
Acknowledgements
@iGEM_Paris
  Stop And Smell The Roses
BACKGROUND
Synthetic enzymes can produce odors that humans experience directly, without special instruments. The banana and wintergreeen smell BioBricks are iGEM icons, and a favorite way to introduce genetic engineering. An expanded library of easy-to-use odor enzymes would take synthetic biology to new audiences for creativity, beauty and fun!
AIMS
We standardized and simplified existing smell-producing BioBricks for banana, wintergreen, lemon and rain. Also, we created new BioBricks for the aromas of popcorn and jasmine. We made an odor wheel made out of genetic odors that follow a standard organization so that the general public can play with odor genes.
ACHIEVEMENTS
Created BioBricks coding for sequences for different enzymes to nullify the bad odor produced by E. coli.
We produced the smells that compose the main odor categories perceived by humans.
BioBricks submitted to be BioBrick registry: BBa_K1403003 , BBa_K1403006 , BBa_K1403009 , BBa_K1403012 , BBa_K1403017 , BBa_K1403019
Introduction
Results
Methods
Figure 1. Odor palette composed of BioBricks containing coding sequences for different enzymes known to catalyze reactions that yield volatile compounds with distinctive smells. We included odors with different tonalities in order to explore the aromas resulting from different combinations of smelly units. The tonality of each smell is categorized as butter, balminess, citrus, non-citrus fruit and herbal. These cover half of the main odor categories perceivable to human beings.
Introduction
All living things can detect chemical signals from the environment. In humans, the direct chemical sense takes the form of olfaction. Although our sense of smell is less sensitive than that of other mammals, it is still capable of detecting many compounds at concentrations lower than 1 part per million. Odors can trigger emotional responses and memory associations, making them a very direct and visceral way to experience the environment.
Bacterial enzymes can produce volatile compounds, including many that are surprising and fun. A generation of iGEMers has had the chance to experience bacteria that smell like banana or mint . The goal of this project is to expand the range of odors that can be created with synthetic enzymes. An easy-to-use, standardized genetic odor library will empower a new generation to play with synthetic biological smells.
Figure 2. Basic structure of the odor cassettes: iGEM prefix, constitutive promoter , synthetic ribosome-binding site, coding sequence with his tag, and iGEM suffix
Results
We created BioBricks coding for sequences of different enzymes to explore each of the odors described in the palette. These sequences are codon-optmized for produced expression in E. coli and produce the smells that compose the main odor categories perceived by humans.
Methods
Odorless E. coli:
The tnaA deletion mutant E. coli was taken from location 63:E:9 in the Keio collection. Standard microbiology techniques were used to culture it in liquid and solid media. Subsequently, a single colony was taken to make a batch of Ca2Cl chemically competent cells using.
BioBricks:
The coding sequences for BMST1 , ATF1 , LIMS1 and GGS were extracted from the Registry. The ATF1 and LIMS1 generator BioBricks were taken from previous iGEM distribution kits and were cloned into the pSB1C3 vector downstream of BBa_J23108 constitutive promoter. The BMST1, JMT, and GDS sequences were codon optimized for expression in E. coli. A synthetic ribosome-binding site that includes XmaI and AgeI flanking restriction sites was designed using the RBS calculator from the Salis’ lab. All BioBricks were assembled using standard molecular biology cloning techniques and 3A iGEM assembly.
Characterization:
Transformed E. coli was grown in M9 minimal media with 2% glucose and complete amino acid mix overnight with the appropriate substrate.
System
Enzyme
Substrate
Concentration
Product
Jasmine
JMT
Jasmonic acid
1 mM
Methyl jasmonate
Mint
BMST1
Salicylic acid
2 mM
Methyl salicylate
Banana
ATF1
Isoamyl alcohol
5 mM
Isoamyl acetate
Limonene
LIMS1
Farnesyl diphosphate
Natural
(+)-Limonene
Rain
G/G D-synthase
Farnesyl diphosphate
Natural
Geosmin
Flowers
BMST1
Benzoic acid
5 mM
Methyl benzoate
Butter
AldB
Acetolactic acid
-
Acetoin
Sweat
agaA
3-M-2-hexenoic acid
-
3-HO-3-Methylhexanoic acid
Table 1. Substrate concentrations (mM).
Centre for Research and Interdisciplinarity (CRI) Faculty of Medicine Cochin Port-Royal, South wing, 2nd floor Paris Descartes University 24, rue du Faubourg Saint Jacques 75014 Paris, France
+33 1 44 41 25 22/25
paris-bettencourt-igem@googlegroups.com
Copyright (c) 2014 igem.org. All rights reserved.
Retrieved from " http://2014.igem.org/Team:Paris_Bettencourt/Project/Odor_Library "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
