"
Page
Discussion
View source
History
teams
Log in
 
Team:Paris Bettencourt/Notebook/Interlab Study
From 2014.igem.org
Project
SmellThe Roses
SomethingFishy
GoodyTwo Shoes
Don'tSweat It
Teen Spirit
Design Practice
Bioinformatics
Achievements
Results
Interlab Study
MedalFulfilment
Parts
Policy
Safety
Consent Form
Patenting
Practices
MOOC
CitizenScience
The Senseof Smell
The HumanMicriobiome
Minder
Collaborations
Collaborations
The SmellGame
Newsletters
Notebook
Daily Note
Protocols
Bibliography
Team
Our Team
Attributions
Official TeamProfile
Acknowledgements
@iGEM_Paris
Smell The Roses
Something Fishy
Don't Sweat It
Goody Two Shoes
Teen Spirit
Interlab Study
June
July
August
September
October
Notebook
July
July 4th
We will participate in the iGEM interlab study
Device 1
Existing device: BBa_I20260 (J23101-B0032-E0040-B0015) in the pSB3K3 vector.
Kit location
Spring 2014 Plate 4, Well 18A
We use NEB Turbo cells to transform with the Biobrick
Biobrick extraction (iGEM protocol)
To use the DNA in the Distribution Kit you may follow these instructions:
With a pipette tip, punch a hole through the foil cover into the corresponding well of the Biobrick™-standard part that you want.
Pipette 10uL of dH2O (distilled water) into the well. Pipette up and down a few times and let sit for 5 minutes to make sure the dried DNA is fully resuspended.
Then we followed our protocol for Heat Shock transformation of E.coli
July 5th
Bacterias plated yesterday growed and made big and small colonies.I picked one small and one big colony from one plate in order to make liquid culture:
5mL of LB
5um Chlorophenicol
Inoculate a single colony
July 6th
I made a glycerol stock from the liquid culture following the standard protocol of the Glycerol stock and labelled it with a number G.22
July 7th
Devices 2 and 3
New device to be built by the iGEM team: BBa_J23101 + BBa_E0240 (B0032-E0040-B0015), must be built in the pSB1C3 backbone
Kit locations (Spring 2014)
BBa_J23101 (called BBa_K823005 when in pSB1C3): Plate 1, Well 20K
BBa_E0240 (in pSB1C3): Plate 2, Well 24B
New device to be built by the iGEM team: BBa_J23115 + BBa_E0240 (B0032-E0040-B0015), must be built in the pSB1C3 backbone
Kit locations (Spring 2014)
BBa_J23115 (called BBa_K823012 when in pSB1C3): Plate 1, Well 22I
BBa_E0240 (in pSB1C3): Plate 2, Well 24B
I followed iGEM Distribution Kit instructions and Heat Shock transformation of E.coli
July 8th
Successful transformations:
BBa_E0240
BBa_K823012
Unsuccessful transformation:
BBa_K823005
For the successful transformations I followed the Glycerol stock protocol
For the unsuccessful transformation I transformed NEB turbo again with the rest of Biobrick BBa_K823005 following again iGEM Distribution Kit instructions and Heat Shock transformation of E.coli
July 9th
I finished the Glycerol stock protocol and labelled G.23 for BBa_K823012 and G.24 for BBa_E0240
BBa_K823005 successfully transformed! I followed the Glycerol stock protocol
Minipreps Following the standard protocol for the miniprep I measured DNA content with the nanodrop. Digestion analysis: (Clovis's recipe) - 5 ug plasmid (given 211ng of BBa_E0240 - 15uL // given 387ng of BBa_K823012  - 30uL - 5 ul FD Buffer - 2.5 uL SpeI + 2.5 uL PstI (BBa_K823012) / 2.5 uL XbeI + 2.5 uL PstI (BBa_E0240) - complete with H2O (Final volume of 50 uL)
Result:DNA wasn't cut. Possible causes: - wrongly calculated volumes (BBa_K823012 shouldn't be 30uL) - too short digestion time
July 10th
Finished the Glycerol stock protocol and labelled it G.25
Minipreps Following the standard protocol for the miniprep I measured DNA content with the nanodrop. Digestion analysis: (Clovis's recipe) - 5 ug plasmid - 5 ul FD Buffer - 2.5 uL SpeI + 2.5 uL PstI (BBa_K823012) / 2.5 uL XbeI + 2.5 uL PstI (BBa_E0240) - complete with H2O (Final volume of 50 uL) In the final 30uL of the miniprep of  BBa_K823005  I didn't have enough of DNA.
July 11th
Minipreps [yes, again!]  Following the standard protocol fot the miniprep I measured DNA content with the nanodrop. Digestion analysis: (Clovis's recipe) - 5 ug plasmid - 5 ul FD Buff(given 211ng of BBa_E0240 - 21uL // given 144ng and 120ng of BBa_K823012 - 35uL // given 273ng of BBa_K823005 - 20uL - 2.5 uL SpeI + 2.5 uL PstI (BBa_K823012 and BBa_K823005) / 2.5 uL XbeI + 2.5 uL PstI (BBa_E0240) - complete with H2O (Final volume of 50 uL) Result: The gel showed that I mislabelled the constructs. There were 3 promoters (big and very small fragments) I will restart the experiment from the Biobricks
Biobricks have been taken from the -20C stock and I followed iGEM Distribution Kit instructions and Heat Shock transformation of E.coli
July 12th
Colonies grew for all 3 Biobricks. They have been put into a liquid culture in order to put them into the Glycerol stock
July 13th
The Glycerol stock have been finish and it is stocked in my box at -20C
July 14th
Minipreps Following the standard protocol for the miniprep Result: 2 tubes in the fridge -20C.
July 15th
Goal: Ligate and transform BBa_J23101 + BBa_E0240 and  BBa_J23115 + BBa_E0240
For both J23101 and J23115 : - BBa_E0240(GFP) 5 ul -BBa_J231011/BBa_J23115 1ul -Ligase 1ul -Buffer 4ul -NF water 9ul TOTAL 20ul
Incubation: 30min 24°C
The ligation products have been transformed following Heat Shock transformation forE.coli
July 16th
Both transformation showed up to be sucessfull. However, the contruct BBa_J23115 + BBa_E0240 did not seem to be fluorescent (observation through the blue filter)
August
August 4th
I decided to redo the construct BBa_J23115 + BBa_E0240
I inoculated bacterias from the glycerol stock (BBa_J23115 and BBa_E0240) into the culture 5ml LB overnight in 37C
August 5th
I prepared a minipreps following the standard protocol for the miniprep Then I digested -BBa_E0240 with XbeI and PstI to extract the GFP coding part -BBa_J23115 with SpeI and PstI to extract the promotor with the backbone I have run the digestion products on the elecrophoresis gel Result:the bands at around  876 bp for GFP and 2100 bp for the promoter + backbone
A gel of two digested biobrics pSB1C3 with GFP and pSB1C3 with the promoter
I performed gel extraction for the GFP fragment and I used some of the digestion product,which I didn't run on the gel, of promoter+backbone to perform a PCR purification . When the gel was running, I added some AP to that sample to avoid end's sticking.  I ligated the GFP with promoter+backbone part following the standard ligation kit in proportions 1:1, 3:1, 5:1. then Incubate at 22°C for 1h and at 16°C O/N
August 6th
I transformed the ligation product into E.coli following Heat Shock transformation
August 7th
Bacterias grew on the plates 1:1 and 3:1. However they are NOT fluorescent.
August 8th
Device three
I looked at the plates again and found some candidates, colonies slighty greener. I have inoculated them in LB 37C O/N.
Device one
I checked also the device one. My glycerol stock wasn't acttualy fluorecent. I decided to re-do it. Existing device: BBa_I20260 (J23101-B0032-E0040-B0015) in the pSB3K3 vector. We gonna use the BBa_I20260  located in plate 2, 17F iGEM_2012
We use NEB Turbo cells to transform with the Biobrick
Biobrick extraction (iGEM protocol)
To use the DNA in the Distribution Kit you may follow these instructions:
With a pipette tip, punch a hole through the foil cover into the corresponding well of the Biobrick™-standard part that you want.
Pipette 10uL of dH2O (distilled water) into the well. Pipette up and down a few times and let sit for 5 minutes to make sure the dried DNA is fully resuspended.
Then we followed our protocol for Heat Shock transformation of E.coli
August 9th
Device three
I spinned the colonies and looked at the sediment through the filter. Bacteria were fluorescent.
Device one
Transformed bacterias grew on the plates. I veryfied the fluorescence and the cells were green when observed through the filter. I've put tchem into a liquid culture : -5ml of LB -5um of Kn one colony from the plate. I made a glicerol stock replacing the wrong one
Thus, all three are acomplished.
Measurments
August 16th
Having all the three devices in the glycerol stock. Bacteries from glycerol stock were inoculated in 5mL LB broth with appropriate antibiotic and grown to saturation overnight (16h) at 37°C with shaking (220 rpm).
August 17th
Samples were diluted 100x (50um in 5 mL LB with appropriate antibiotic) and incubated for 2h at 37°C prior to measurement.
A plate, Greiner 96 Flat Bottom Black Polystyrene, has been loaded with 200um of cells in LB and 100um mineral oil. I used LB and LB+NEB (not fluorescent) as controls. I programed device to run for 18h
August 18th
Once the machine finished running, I have looked at the data and almost all wells were 'overread', which means I didn't calibrated the machine correctly. I also been told I shouldn't have put 100um of mineral oil. I have inoculated glycerol stock again in LB for O/N.
August 19th
Samples were diluted 100x (50um in 5 mL LB with appropriate antibiotic) and incubated for 2h at 37°C prior to measurement. A Greiner plate  has been loaded with 200um of cells in LB and 50um mineral oil. This time I calibrated for the optimal gain and 50 um of mineral oil.
August 20th
We had overread for one sample in the beginning. Apparently the optimisation of gain applies for all the plate and we used only 5 wells (LB, NEB, Device 1, Device 2, Device 3
I was told that I should have start with single colonies to make the measurments more accurate
I plated the glycerol stock
August 21th
I have put a single colony from each plate (Devices 1,2,3 and NEB) into LB with an appropriate antibiotic.
August 22th
Samples were diluted 100x (50um in 5 mL LB with appropriate antibiotic) and incubated for 2h at 37°C prior to measurement. A plate, Greiner 96 Flat Bottom Black Polystyrene, has been loaded with 150um of cells in LB and 30um mineral oil. This time I have used all rows. I made also as a control LB+Kn and LB+Cm.
August 23th
When Tecan stoped running I copied the data into my computer and started to work on it.
September
September 4th
I inoculated in LB the bacterias from glycerol stock for all three devices in order to do a miniprep next day and sequence it
September 5th
I made minipreps following the standard protocol for the miniprep .  I used a nenodrop to quantify the amout of DNA : 1st Device  350 ng/ul, 2nd Device: 309 ng/uL, 3rd Device 477ng/uLI have used http://www.idtdna.com/calc/dilution/ to calculate the corredt dilutions and fit the GATC requirements: Concentration: Plasmids: 30-100 ng/µl Custom primers: 10 pmol/µl Volume: DNA samples & primers: 20 µl each (sufficient for 8 reactions), for 96 well sequencing please send 50 µl primer in a single tube. Please send the DNA dissolved in water. The solution must not contain any EDTA. DNA and primers to be delivered in 1.5 ml reaction tubes or 96 well microtiter pl. I prepared samples and VR iGEM forward plasmid as required and send it to GATC company.
September 4th
We received the result of the sequencing. I have used Geneious software to map onto reference (engeenered plasmid). The sequenced devices mached with the engeneered ones.
Centre for Research and Interdisciplinarity (CRI) Faculty of Medicine Cochin Port-Royal, South wing, 2nd floor Paris Descartes University 24, rue du Faubourg Saint Jacques 75014 Paris, France
+33 1 44 41 25 22/25
paris-bettencourt-igem@googlegroups.com
Copyright (c) 2014 igem.org. All rights reserved.
Retrieved from " http://2014.igem.org/Team:Paris_Bettencourt/Notebook/Interlab_Study "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
