<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Paris_Bettencourt_Notebook_TMAU"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Paris_Bettencourt/Notebook/TMAU">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Paris_Bettencourt/Notebook/TMAU&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Paris_Bettencourt/Notebook/TMAU&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Paris_Bettencourt/Notebook/TMAU&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Paris_Bettencourt/Notebook/TMAU" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Paris Bettencourt/Notebook/TMAU</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><DIV class="navmenubkg"><DIV id="navmenubox"><UL id="nav"><LI><A href="https://2014.igem.org/Team:Paris_Bettencourt/Project">Project</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Project/Odor_Library">SmellThe Roses</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Project/TMAU">SomethingFishy</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Project/Foot_Odor">GoodyTwo Shoes</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Project/Eliminate_Smell">Don'tSweat It</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Project/Old_People_Smell">Teen Spirit</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Project/Art">Design Practice </A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Project/Bioinformatics">Bioinformatics</A></LI></UL><LI><A href="https://2014.igem.org/Team:Paris_Bettencourt/Achievement">Achievements</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Results">Results</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Project/Interlab_Study">Interlab Study</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Medal">MedalFulfilment</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Parts">Parts</A></LI><LI><A href="https://2014.igem.org/Team:Paris_Bettencourt/Policy">Policy </A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Safety">Safety</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Consent_Form">Consent Form</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Patenting">Patenting</A></LI><LI><A href="https://2014.igem.org/Team:Paris_Bettencourt/Human_Practice">Practices</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/MOOC">MOOC</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Citizen_Science">CitizenScience</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Facts_Smell">The Senseof Smell</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Facts_Micriobiome">The HumanMicriobiome</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Minder">Minder</A></LI><LI><A href="https://2014.igem.org/Team:Paris_Bettencourt/Collaboration_Main">Collaborations </A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Collaboration">Collaborations </A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/The_Smell_Game">The SmellGame</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Newsletter">Newsletters</A></LI><LI><A href="https://2014.igem.org/Team:Paris_Bettencourt/Notebook_Main">Notebook</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Notebook">Daily Note</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Protocols">Protocols</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Bibliograpy">Bibliography</A></LI><LI><A href="https://2014.igem.org/Team:Paris_Bettencourt/Team_Main">Team </A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Team">Our Team </A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Attributions">Attributions</A></LI><LI><A class="liste" href="https://igem.org/Team.cgi?year=2014&amp;team_name=Paris_Bettencourt">Official TeamProfile </A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Acknowledgements">Acknowledgements</A></LI><DIV id="twnoticon" class="sidelogos"><A class="twitter-timeline" width="250" height="400" data-dnt="true" href="https://twitter.com/iGEM_Paris" data-widget-id="368144501119795201">@iGEM_Paris</A></DIV></DIV></DIV></DIV><DIV id="moocdiv"><TABLE><TBODY><TR><TD><A href="https://2014.igem.org/Team:Paris_Bettencourt/Notebook/Odor_Library">Smell The Roses</A></TD><TD><A href="https://2014.igem.org/Team:Paris_Bettencourt/Notebook/TMAU">Something Fishy</A></TD><TD><A href="https://2014.igem.org/Team:Paris_Bettencourt/Notebook/Eliminate_Smell">Don't Sweat It</A></TD><TD><A href="https://2014.igem.org/Team:Paris_Bettencourt/Notebook/Foot_Odor">Goody Two Shoes</A></TD><TD><A href="https://2014.igem.org/Team:Paris_Bettencourt/Notebook/Old_People_Smell">Teen Spirit</A></TD><TD><A href="https://2014.igem.org/Team:Paris_Bettencourt/Notebook/Interlab_Study">Interlab Study</A></TD></TR></TBODY></TABLE></DIV><DIV id="intra"><UL><LI><A href="https://2014.igem.org/Team:Paris_Bettencourt/Notebook/TMAU#June">June</A></LI><LI><A href="https://2014.igem.org/Team:Paris_Bettencourt/Notebook/TMAU#July">July</A></LI><LI><A href="https://2014.igem.org/Team:Paris_Bettencourt/Notebook/TMAU#August">August</A></LI><LI><A href="https://2014.igem.org/Team:Paris_Bettencourt/Notebook/TMAU#September">September</A></LI><LI><A href="https://2014.igem.org/Team:Paris_Bettencourt/Notebook/TMAU#October">October</A></LI></UL></DIV><DIV id="corps"><H3 id="haut">Notebook</H3><DIV id="June"><H4>June</H4><H5 id="date">June 24th</H5><P>oPB.010 received
oPB.010
tmm Fw for BioBrick vector
TATAGAATTCGCGGCCGCTTCTAGAGCTGACAGCTAGCTCAGTCCTAG</P><H5 id="date">June 29th</H5><P>Goal: to PCR ilvBN gene from MG1655</P><P>Procedure: 
1. Picked 2 colonies from the plate MG1655 115+GFP
2. inoculate them each in 50 ul of NF H2O
3. Boiled for 3 mins at 98 degree
4. use the solution as DNA template
5. follow standard PCR protocol from this step on</P><P>Reagent Volume 1x 
Nuclease-free water 63 ul
5x Phusion HF Buffer 20 ul
10 mM dNTPs 2 ul
Forward Primer (10 uM) 5 ul
Reverse Primer (10 uM) 5 ul
Genomic DNA template 1 ul 
DMSO 3 ul
Phusion DNA Polymerase 1 ul
Total Volume 100 ul</P><P>Thermocycler protocol: NEB Phusion
 Temp Time 
Start 98 ° C 30 sec Melt 
35 Cycles
Cycle 1 98 ° C 10 sec Melt 
Cycle 2 52 ° C 30 sec Anneal
Cycle 3 72 ° C 2 minutes Extend
Finish 72 ° C 5 min Extend 
Blind 10 ° C Forever 
Blind</P><P>outcome:</P><P>bands seen at expected length.</P></DIV><DIV id="July"><H4>July</H4><H5 id="date">July 3rd</H5><P> We made a PCR of our tmm gBlock using this protocol:
					
<TABLE border=""><TBODY><TR><TD><B>Reagent</B></TD><TD><B>Volume</B></TD></TR><TR><TD>1X</TD><TD>4X</TD></TR><TR><TD>5X Phusion HF Buffer</TD><TD>20uL</TD><TD>80uL</TD></TR><TR><TD>10mM dNTPs</TD><TD>2uL</TD><TD>8uL</TD></TR><TR><TD>oPB.010 (forward primer)</TD><TD>1uL</TD><TD>4uL</TD></TR><TR><TD>oPB.012 (reverse primer)</TD><TD>1uL</TD><TD>4uL</TD></TR><TR><TD>DNA</TD><TD>1uL</TD><TD>4uL</TD></TR><TR><TD>DMSO</TD><TD>3uL</TD><TD>12uL</TD></TR><TR><TD>Phusion polymerase</TD><TD>1uL</TD><TD>4uL</TD></TR><TR><TD>Nuclease Free Water</TD><TD>71uL</TD><TD>284uL</TD></TR></TBODY></TABLE></P><P>We used this thermocycle:</P><TABLE border=""><TBODY><TR><TD><B>Temperature</B></TD><TD><B>Time</B></TD></TR><TR><TD><B>Start</B></TD><TD>98°C</TD><TD>30s</TD></TR><TR><TD><B>Cycle x35</B></TD><TD>98°C</TD><TD>10s</TD></TR><TR><TD><B>Cycle x35</B></TD><TD>52°C</TD><TD>30s</TD></TR><TR><TD><B>Cycle x35</B></TD><TD>72°C</TD><TD>30s</TD></TR><TR><TD><B>Finish</B></TD><TD>72°C</TD><TD>5min</TD></TR><TR><TD><B>Blind</B></TD><TD>10°C</TD><TD>-</TD></TR></TBODY></TABLE><P>After gel migration, we got this:</P><P> The PCR worked.</P><P>Miniprep using  <A href="https://2014.igem.org/Team:Paris_Bettencourt/Protocols#prot1">Thermo scientific kits</A></P><P> Digestion using EcoR1 and Pst1</P><P>Extracted plasmids from miniprep 22 uL 30uL
FD Buffer 5uL 5uL
Pst1 2.5uL 2.5uL
EcoR1 2.5uL 2.5uL
NF Water 18uL 10uL</P><P> Incubation for 5min 37°C and 2 hours at room temperature.</P><P>Ligation of gBlock into Vector</P><P>PCR product: tmm gBlock with oPB.010 and oPB.012
Vector: pSB1C3 linearized</P><P>Concentration:
-digested PSB1C3: 4.6ng/uL
-digested tmm: 13.7ng/uL</P><P> Cohesive Ends
5X Ligase Reaction Buffer 4 μl</P><P>Insert: Vector Molar Ratio 3:1</P><P>Vector Ends 5ul</P><P>Insert Ends 5ul</P><P>Total DNA 0.092ug(0.01-0.1 μg)</P><P>T4 DNA Ligase 0.1 unit</P><P>Autoclaved distilled water to 20 μl</P><P>Incubate at 24°C for O/N.</P><P>
Note: For optimal transformation, dilute the ligation reaction ≥ 5-fold, to at least 100 μl, before adding to competent cells</P><H5 id="date">July 5th</H5><P>Transformation of the part.</P><P>Strain: <I>E.coli</I> MG1655
plasmid: pPB.011
Heat Shock Transformation of E. coli</P><P>This protocol can be used to transform chemically competent (i.e. from CaCl2) with a miniprepped plasmid or a ligation product.</P><P>Note: Never vortex competent cells. Mix cells by gentle shaking.</P><P> No colonies after 24h, this might be due to the low concentration of reagents</P><H5 id="date">July 12th</H5><P id="text">We transformed NEB and MG1655 with our ligation product according to <A href="https://2014.igem.org/Team:Paris_Bettencourt/Protocols#prot1">this protocol</A></P><P>We finally had no colonies.</P><H5 id="date">July 10th</H5><P id="text"><U>1) Mini prep to extract plasmids (Qiagen)</U></P><P> I used <A href="https://2014.igem.org/Team:Paris_Bettencourt/Protocols#prot4"> this protocol</A>. I made 2 tubes and got 163.2 and 242 ng/uL as concentrations.
</P><P id="text"><U>2) Digestion of miniprep product with EcoR1 &amp; Pst1</U></P><TABLE border=""><TBODY><TR><TD><B>Tube 1 (163ng/uL)</B></TD><TD><B>Tube 2 (242ng/uL)</B></TD></TR><TR><TD><B>pSB1C3 from miniprep</B></TD><TD>22uL</TD><TD>30uL</TD></TR><TR><TD><B>10X FD Buffer</B></TD><TD>5uL</TD><TD>5uL</TD></TR><TR><TD><B>PstI</B></TD><TD>2.5uL</TD><TD>2.5uL</TD></TR><TR><TD><B>EcoRI</B></TD><TD>2.5uL</TD><TD>2.5uL</TD></TR><TR><TD><B>NF Water</B></TD><TD>18uL</TD><TD>10uL</TD></TR></TBODY></TABLE><P id="text"><U>3) Gel migration</U></P><P> I had the digested pSB1C3 run on a gel using 0.5X TBE (2H-50V). We clearly saw a band at 2kb corresponding to pSB1C3. I cut the gel and stored it at 4°C.</P><H5 id="date">July 11th</H5><P><U>1) Gel purification of digested pSB1C3</U></P><P> I used <A href="https://2014.igem.org/Team:Paris_Bettencourt/Protocols#prot6"> this protocol</A></P><P><I>Result: I got concentrations of 59.6 and 55.1 ng/uL.</I></P><P><U>2) Ligation of digested tmm and digested pSB1C3</U></P><P> I used <A href="http://nebiocalculator.neb.com/#!/ligation"> NEBiocalculator</A> to calculate the mass of insert and vector I had to put (Insert length: 1488 bp, Vector length: 2037 bp, Vector DNA mass: 0,3 ug, Ratio insert:vector: 2:1). I need to put 450ng of insert and 300 ng of vector.</P><TABLE border=""><TBODY><TR><TD><B>Volume</B></TD></TR><TR><TD><B>Insert (tmm)</B></TD><TD>20uL of 10.1 ng/uL and 20uL of 13.7 ng/uL</TD></TR><TR><TD><B>Vector (pSB1C3)</B></TD><TD>5uL of 59.6ng/uL</TD></TR><TR><TD><B>T4 Ligase</B></TD><TD>3uL</TD></TR><TR><TD><B>T4 Ligase Buffer</B></TD><TD>6uL</TD></TR><TR><TD><B>NF Water</B></TD><TD>6uL</TD></TR></TBODY></TABLE><P> I incubated overnight at room temperature</P><H5 id="date">July 14th</H5><H5 id="date">July 15th</H5><P><U>1) Gel extraction of pSB1C3 digested plasmid</U></P><P> I used <A href="https://2014.igem.org/Team:Paris_Bettencourt/Protocols#prot6">this protocol</A>. I finally got 2 tubes with 13.9 and 19.8ng/uL concentrations. </P><P><U>2) PCR purification</U></P><P>I purified the PCR product of July 14th using <A href="https://2014.igem.org/Team:Paris_Bettencourt/Protocols#prot5">this protocol</A>.</P><P><U>3) Ligation of tmm PCR product and pSB1C3</U></P><P>I used <A href="http://nebiocalculator.neb.com/#!/ligation">NEBiocalculator</A> to to calculate the needed amount of insert and vector with the values:</P><UL><LI> Insert length: 1488 bp</LI><LI>Vector length: 2037 bp</LI><LI> Vector DNA mass: 50 ng</LI><LI>Ratio insert: I made 1 ligation of ratio 1:1, 2:1 and 3:1 (insert:vector).</LI></UL><P>I made the ligation with the 13.9 ng/uL concentrated insert tube. I used this protocol:</P><TABLE border=""><TBODY><TR><TD><B>1:1</B></TD><TD><B>2:1</B></TD><TD><B>3:1</B></TD></TR><TR><TD><B>Digested tmm (insert)</B></TD><TD>1.5uL</TD><TD>3uL</TD><TD>4uL</TD></TR><TR><TD><B>Digested pSB1C3 (vector)</B></TD><TD>4uL</TD><TD>4uL</TD><TD>4uL</TD></TR><TR><TD><B>T4 Ligase</B></TD><TD>1uL</TD><TD>1uL</TD><TD>1uL</TD></TR><TR><TD><B>T4 Ligase Buffer</B></TD><TD>4uL</TD><TD>4uL</TD><TD>4uL</TD></TR><TR><TD><B>Nuclease Free water</B></TD><TD>9.5uL</TD><TD>8uL</TD><TD>7uL</TD></TR></TBODY></TABLE><P> I incubated the tubes at 24°C for 30min.</P><P><U>4) Transformation into NEB and MG1655</U></P><P>I used the <A href="https://2014.igem.org/Team:Paris_Bettencourt/Protocols#prot5">Heat Shock transformation protocol</A> to transform NEB and MG1655 with each one of the ligation products with ratios 1:1, 2:1 and 3:1 (I got 6 plates).</P><H5 id="date">July 16th</H5><P><U>1) Results of the transformations</U></P><P>We got no colonies in MG1655, big and small colonies for 1:1 and 2:1 ratios and only big colonies with 3:1 ratios. I prepared 5 liquid cultures in LB+Chloramphenicol: 1 for each type of colony for 1:1 and 2:1 ratios, and 1 for big colonies for 3:1 ratio. I incubated them at 37°C.</P><P><U>2) PCR colony</U></P><P>To check if the colonies developped in my plates have tmm gene, I made a PCR with <A href="https://2014.igem.org/Team:Paris_Bettencourt/Protocols#prot7">this protocol</A> and I ran a gel. I got nothing on it. I will do miniprep and digestion on July 17th to identify tmm construct into pSB1C3.</P><H5 id="date">July 17th</H5><P><U>1) Miniprep using each liquid culture of NEB+tmm-pSB1C3</U></P><P>I used  <A href="https://2014.igem.org/Team:Paris_Bettencourt/Protocols#prot4">this protocol</A>.</P><P><U>2) Digestion of the plasmids by EcoRI</U></P><P>Then I digested the miniprep product by EcoRI in order to get a linearised plasmid before running it on a gel.</P><TABLE border=""><TBODY><TR><TD><B>Reagent</B></TD><TD><B>Volume</B></TD></TR><TR><TD>Nuclease free water</TD><TD>12uL</TD></TR><TR><TD>10X Fast Digest Buffer</TD><TD>2uL</TD></TR><TR><TD>plasmid DNA</TD><TD>5uL</TD></TR><TR><TD>EcoRI</TD><TD>1uL</TD></TR><TR><TD><I>TOTAL</I></TD><TD>20uL</TD></TR></TBODY></TABLE><P>I incubated the tubes at 37°C during 30min.</P><P><U>3) Gel</U></P><P>I ran on a gel each one of the tubes (small and big colonies from 1:1 and 2:1 ratios and big colony from 3:1 ratio). But the plasmids may have not been digested enough because I got many bands.</P><H5 id="date">July 18th</H5><P>I will try again to identify tmm gene in my transformed bacteria.</P><P><U>1) Digestion of plasmids with EcoRI and PstI.</U></P><P>This time I digested by EcoRI and PstI to be able to directly identify tmm gene on the gel. I nanodroped the miniprep product from 17th. We need 1ug of DNA for the digestion.</P><TABLE border=""><TBODY><TR><TD><B>Concentration (ng/uL, given by Nanodrop)</B></TD><TD><B>Needed volume (uL)</B></TD></TR><TR><TD><B>big 1:1 ratio colony</B></TD><TD>316.4</TD><TD>3.5</TD></TR><TR><TD><B>small 2:1 ratio colony</B></TD><TD>156.9</TD><TD>7</TD></TR><TR><TD><B>big 2:1 ratio colony</B></TD><TD>207.6</TD><TD>5</TD></TR><TR><TD><B>big 3:1 ratio colony</B></TD><TD>207.6</TD><TD>4</TD></TR></TBODY></TABLE><P>Digestion protocol:</P><TABLE border=""><TBODY><TR><TD><B>Reagent</B></TD><TD><B>big 1:1 ratio colony</B></TD><TD><B>small 2:1 ratio colony</B></TD><TD><B>big 2:1 ratio colony</B></TD><TD><B>big 3:1 ratio colony</B></TD></TR><TR><TD>Nuclease free water</TD><TD>13.5uL</TD><TD>10uL</TD><TD>12uL</TD><TD>13uL</TD></TR><TR><TD>10X Fast Digest Buffer</TD><TD>2uL</TD><TD>2uL</TD><TD>2uL</TD><TD>2uL</TD></TR><TR><TD>plasmid DNA</TD><TD>3.5uL</TD><TD>7uL</TD><TD>5uL</TD><TD>4uL</TD></TR><TR><TD>EcoRI</TD><TD>0.5uL</TD><TD>0.5uL</TD><TD>0.5uL</TD><TD>0.5uL</TD></TR><TR><TD>PstI</TD><TD>0.5uL</TD><TD>0.5uL</TD><TD>0.5uL</TD><TD>0.5uL</TD></TR><TR><TD><I>TOTAL</I></TD><TD>20uL</TD><TD>20uL</TD><TD>20uL</TD><TD>20uL</TD></TR></TBODY></TABLE><P><U>1)Gel</U></P><P>I ran all the samples on a gel but I did not get the bands I expected. We have to try a new cloning.</P><H5 id="date">July 22nd</H5><P>I had prepared liquid cultures to be able to do minipreps.</P><P><U>1) Miniprep of liquid cultures to get pSB1C3</U></P><P>I used <A href="https://2014.igem.org/Team:Paris_Bettencourt/Protocols#prot4">this protocol</A>. The Nanodrop gave me a concentration of 317ng/uL.</P><P><U>2) Digestion of pSB1C3 and tmm PCR product by EcoRI and PstI.</U></P><UL><LI> pSB1C3:
<TABLE border=""><TBODY><TR><TD><B>Reagent</B></TD><TD><B>Volume</B></TD></TR><TR><TD>Nuclease free water</TD><TD>25uL</TD></TR><TR><TD>10X Fast Digest Buffer</TD><TD>5uL</TD></TR><TR><TD>plasmid DNA</TD><TD>14uL</TD></TR><TR><TD>EcoRI</TD><TD>2.5uL</TD></TR><TR><TD>PstI</TD><TD>2.5uL</TD></TR><TR><TD>Fast AP (phosphatase)</TD><TD>1uL</TD></TR><TR><TD><I>TOTAL</I></TD><TD>40uL</TD></TR></TBODY></TABLE></LI><LI>tmm PCR product
<TABLE border=""><TBODY><TR><TD><B>Reagent</B></TD><TD><B>Volume</B></TD></TR><TR><TD>Nuclease free water</TD><TD>27uL</TD></TR><TR><TD>10X Fast Digest Buffer</TD><TD>5uL</TD></TR><TR><TD>plasmid DNA</TD><TD>13uL</TD></TR><TR><TD>EcoRI</TD><TD>2.5uL</TD></TR><TR><TD>PstI</TD><TD>2.5uL</TD></TR><TR><TD><I>TOTAL</I></TD><TD>50uL</TD></TR></TBODY></TABLE></LI></UL><P><U>3)Gel</U></P><P> For pSB1C3 we can see two bands corresponding to 900bp and 2000bp (backbone) fragments. I will be able to extract the 2000bp fragment. But I got nothing for tmm.</P><P><U>4)Gel extraction</U></P>
I used <A href="https://2014.igem.org/Team:Paris_Bettencourt/Protocols#prot6">this protocol</A>.
<H5 id="date">July 29th</H5><P>We need tmm PCR product to be able to do a new cloning.</P><P><U>1) PCR of tmm gBlock</U></P><TABLE border=""><TBODY><TR><TD><B>Reagent</B></TD><TD><B>Volume</B></TD></TR><TR><TD>1X</TD></TR><TR><TD>5X Phusion HF Buffer</TD><TD>10uL</TD></TR><TR><TD>10mM dNTPs</TD><TD>1uL</TD></TR><TR><TD>oPB.010 (forward primer)</TD><TD>0.5uL</TD></TR><TR><TD>oPB.012 (reverse primer)</TD><TD>0.5uL</TD></TR><TR><TD>DNA</TD><TD>1uL</TD></TR><TR><TD>DMSO</TD><TD>1.5uL</TD></TR><TR><TD>Phusion polymerase</TD><TD>0.5uL</TD></TR><TR><TD>Nuclease Free Water</TD><TD>35uL</TD></TR></TBODY></TABLE><P>We used this thermocycle:</P><TABLE border=""><TBODY><TR><TD><B>Temperature</B></TD><TD><B>Time</B></TD></TR><TR><TD><B>Start</B></TD><TD>98°C</TD><TD>30s</TD></TR><TR><TD><B>Cycle x35</B></TD><TD>98°C</TD><TD>10s</TD></TR><TR><TD><B>Cycle x35</B></TD><TD>56°C for tube 1, 51°C for tubes 2 and 3</TD><TD>30s</TD></TR><TR><TD><B>Cycle x35</B></TD><TD>72°C</TD><TD>1min</TD></TR><TR><TD><B>Finish</B></TD><TD>72°C</TD><TD>5min</TD></TR><TR><TD><B>Blind</B></TD><TD>10°C</TD><TD>-</TD></TR></TBODY></TABLE><P><U>2) Gel</U></P><P>No bands were visible, even for the ladder. I tried a new one and got a few tiny bands for tube 3, particularly at 1500bp. I tried a gel extraction using <A href="https://2014.igem.org/Team:Paris_Bettencourt/Protocols#prot6">this protocol</A> and got 4.7ng/uL (unusable). I tried otherwise:</P><P><U>3) PCR purification</U></P><P> I puridied tube 3 according to <A href="https://2014.igem.org/Team:Paris_Bettencourt/Protocols#prot5">this protocol</A> and got a concentration of 11ng/uL which was not enough to do a digestion.</P><P><U>4) New PCR</U></P><TABLE border=""><TBODY><TR><TD><B>Reagent</B></TD><TD><B>Volume</B></TD></TR><TR><TD>1X</TD></TR><TR><TD>5X Phusion HF Buffer</TD><TD>10uL</TD></TR><TR><TD>10mM dNTPs</TD><TD>1uL</TD></TR><TR><TD>oPB.010 (forward primer)</TD><TD>0.5uL</TD></TR><TR><TD>oPB.012 (reverse primer)</TD><TD>0.5uL</TD></TR><TR><TD>DNA</TD><TD>1uL</TD></TR><TR><TD>DMSO</TD><TD>1.5uL</TD></TR><TR><TD>Phusion polymerase</TD><TD>0.5uL</TD></TR><TR><TD>Nuclease Free Water</TD><TD>35uL</TD></TR></TBODY></TABLE><P>We used this thermocycle:</P><TABLE border=""><TBODY><TR><TD><B>Temperature</B></TD><TD><B>Time</B></TD></TR><TR><TD><B>Start</B></TD><TD>98°C</TD><TD>30s</TD></TR><TR><TD><B>Cycle x35</B></TD><TD>98°C</TD><TD>10s</TD></TR><TR><TD><B>Cycle x35</B></TD><TD>55°C</TD><TD>30s</TD></TR><TR><TD><B>Cycle x35</B></TD><TD>72°C</TD><TD>1min</TD></TR><TR><TD><B>Finish</B></TD><TD>72°C</TD><TD>5min</TD></TR><TR><TD><B>Blind</B></TD><TD>10°C</TD><TD>-</TD></TR></TBODY></TABLE><P><U>3) PCR purification</U></P><P> I puridied tube 3 according to <A href="https://2014.igem.org/Team:Paris_Bettencourt/Protocols#prot5">this protocol</A>.</P></DIV><DIV id="August"><H4>August</H4><H5 id="date">August 1st</H5><P><U>1)Digestion of purified PCR product</U></P><TABLE border=""><TBODY><TR><TD><B>Reagent</B></TD><TD><B>Volume</B></TD></TR><TR><TD>Nuclease free water</TD><TD>20uL</TD></TR><TR><TD>10X Fast Digest Buffer</TD><TD>20uL</TD></TR><TR><TD>plasmid DNA</TD><TD>7uL</TD></TR><TR><TD>EcoRI</TD><TD>0.5uL</TD></TR><TR><TD>PstI</TD><TD>0.5uL</TD></TR><TR><TD><I>TOTAL</I></TD><TD>50uL</TD></TR></TBODY></TABLE><P> I incubated the digestion at 37°C during 1h.</P><P><U>2) Purification of digested tmm</U></P><P> I used <A href="https://2014.igem.org/Team:Paris_Bettencourt/Protocols#prot5">this protocol</A> and got a really low concentration, unusable for ligation.</P><P><U>3) New PCR</U></P><P>I prepared 4 tubes according to this protocol:</P><TABLE border=""><TBODY><TR><TD><B>Reagent</B></TD><TD><B>Volume</B></TD></TR><TR><TD>1X</TD></TR><TR><TD>5X Phusion HF Buffer</TD><TD>10uL</TD></TR><TR><TD>10mM dNTPs</TD><TD>1uL</TD></TR><TR><TD>oPB.010 (forward primer)</TD><TD>0.5uL</TD></TR><TR><TD>oPB.012 (reverse primer)</TD><TD>0.5uL</TD></TR><TR><TD>DNA</TD><TD>1uL</TD></TR><TR><TD>DMSO</TD><TD>1.5uL</TD></TR><TR><TD>Phusion polymerase</TD><TD>0.5uL</TD></TR><TR><TD>Nuclease Free Water</TD><TD>35uL</TD></TR></TBODY></TABLE><P>We used this thermocycle:</P><TABLE border=""><TBODY><TR><TD><B>Temperature</B></TD><TD><B>Time</B></TD></TR><TR><TD><B>Start</B></TD><TD>98°C</TD><TD>30s</TD></TR><TR><TD><B>Cycle x35</B></TD><TD>98°C</TD><TD>10s</TD></TR><TR><TD><B>Cycle x35</B></TD><TD>51°C</TD><TD>30s</TD></TR><TR><TD><B>Cycle x35</B></TD><TD>72°C</TD><TD>1min</TD></TR><TR><TD><B>Finish</B></TD><TD>72°C</TD><TD>5min</TD></TR><TR><TD><B>Blind</B></TD><TD>10°C</TD><TD>-</TD></TR></TBODY></TABLE><P><U>4) Gel</U></P><P>Nothing was visible on the gel.</P><P><U>5) New PCR</U></P><TABLE border=""><TBODY><TR><TD><B>Reagent</B></TD><TD><B>Volume</B></TD></TR><TR><TD>1X</TD></TR><TR><TD>5X Phusion HF Buffer</TD><TD>20uL</TD></TR><TR><TD>10mM dNTPs</TD><TD>2uL</TD></TR><TR><TD>oPB.010 (forward primer)</TD><TD>5uL</TD></TR><TR><TD>oPB.012 (reverse primer)</TD><TD>5uL</TD></TR><TR><TD>DNA</TD><TD>1uL</TD></TR><TR><TD>DMSO</TD><TD>3uL</TD></TR><TR><TD>Phusion polymerase</TD><TD>2uL</TD></TR><TR><TD>Nuclease Free Water</TD><TD>53uL</TD></TR></TBODY></TABLE><P>We used this thermocycle:</P><TABLE border=""><TBODY><TR><TD><B>Temperature</B></TD><TD><B>Time</B></TD></TR><TR><TD><B>Start</B></TD><TD>98°C</TD><TD>30s</TD></TR><TR><TD><B>Cycle x35</B></TD><TD>98°C</TD><TD>10s</TD></TR><TR><TD><B>Cycle x35</B></TD><TD>50°C</TD><TD>30s</TD></TR><TR><TD><B>Cycle x35</B></TD><TD>72°C</TD><TD>45s</TD></TR><TR><TD><B>Finish</B></TD><TD>72°C</TD><TD>5min</TD></TR><TR><TD><B>Blind</B></TD><TD>10°C</TD><TD>-</TD></TR></TBODY></TABLE><P><U>6) Gel</U></P><P>There was a diffused band next to 200bp. Maybe there is a problem of specificity of the primers. We will try a PCR with the very first protocol which worked.</P><H5 id="date">August 5th</H5><P><U>1) PCR of tmm gBlock</U></P><TABLE border=""><TBODY><TR><TD><B>Reagent</B></TD><TD><B>Volume</B></TD></TR><TR><TD>1X</TD></TR><TR><TD>5X Phusion HF Buffer</TD><TD>20uL</TD></TR><TR><TD>10mM dNTPs</TD><TD>2uL</TD></TR><TR><TD>oPB.010 (forward primer)</TD><TD>1uL</TD></TR><TR><TD>oPB.012 (reverse primer)</TD><TD>1uL</TD></TR><TR><TD>tmm DNA</TD><TD>1uL</TD></TR><TR><TD>DMSO</TD><TD>3uL</TD></TR><TR><TD>Phusion polymerase</TD><TD>1uL</TD></TR><TR><TD>Nuclease Free Water</TD><TD>71uL</TD></TR></TBODY></TABLE><P>We used this thermocycle:</P><TABLE border=""><TBODY><TR><TD><B>Temperature</B></TD><TD><B>Time</B></TD></TR><TR><TD><B>Start</B></TD><TD>98°C</TD><TD>30s</TD></TR><TR><TD><B>Cycle x35</B></TD><TD>98°C</TD><TD>10s</TD></TR><TR><TD><B>Cycle x35</B></TD><TD>52°C</TD><TD>30s</TD></TR><TR><TD><B>Cycle x35</B></TD><TD>72°C</TD><TD>30s</TD></TR><TR><TD><B>Finish</B></TD><TD>72°C</TD><TD>5min</TD></TR><TR><TD><B>Blind</B></TD><TD>10°C</TD><TD>-</TD></TR></TBODY></TABLE>
We still got nothing. We decided to design new primers.
<H5 id="date">August 6th</H5><P>We think that that the reverse primer (oPB012) might have a strong secondary structure and that it may prevent the PCR to work as expected. We designed modified versions of oPB010 and oPB012 respectively called oPB056 and oPB057, shorter then the first ones. We removed the first restriction site at each end making sure the other enzyme would still have enough nucleotides to bind. We also shortened the bind part to have a melting temperature around 52°C.</P>
oPB.010	TATAGAATTCGCGGCCGCTTCTAGAGCTGACAGCTAGCTCAGTCCTAG    

-&gt; oPB.056     CTTCTAGAGCTGACAGCTAGCTCAGTCC
oPB.012	TTAACTGCAGCGGCCGCTACTAGTATCAGTGGTGATGGTGATGATGGTTA 
-&gt; oPB.057	TACTAGTATCAGTGGTGATGGTGATGATG
<P><B>Note:</B>The digestion should now use <B>XbaI and SpeI</B> as EcorI and PstI were removed from the PCR product.</P><H5 id="date">August 13th</H5><P>We received our new primers (oPB.056 and oPB.057).</P><P><U>1) New PCR with different combinations or primers </U></P><P>We used this protocol:</P><TABLE border=""><TBODY><TR><TD><B>Reagent</B></TD><TD><B>Tube 1</B></TD><TD><B>Tube 2</B></TD><TD><B>Tube 3</B></TD><TD><B>Tube 4</B></TD></TR><TR><TD>5X Phusion HF Buffer</TD><TD>20uL</TD><TD>20uL</TD><TD>20uL</TD><TD>20uL</TD></TR><TR><TD>10mM dNTPs</TD><TD>2uL</TD><TD>2uL</TD><TD>2uL</TD><TD>2uL</TD></TR><TR><TD>tmm DNA</TD><TD>10L</TD><TD>10L</TD><TD>10L</TD><TD>10L</TD></TR><TR><TD>DMSO</TD><TD>3uL</TD><TD>3uL</TD><TD>3uL</TD><TD>3uL</TD></TR><TR><TD>Phusion polymerase</TD><TD>2uL</TD><TD>2uL</TD><TD>2uL</TD><TD>2uL</TD></TR><TR><TD>Nuclease Free Water</TD><TD>53uL</TD><TD>53uL</TD><TD>53uL</TD><TD>53uL</TD></TR><TR><TD>oPB.010</TD><TD>5uL</TD><TD>-</TD><TD>5uL</TD><TD>-</TD></TR><TR><TD>oPB.012</TD><TD>5uL</TD><TD>-</TD><TD>-</TD><TD>5uL</TD></TR><TR><TD>oPB.056</TD><TD>-</TD><TD>5uL</TD><TD>-</TD><TD>5uL</TD></TR><TR><TD>oPB.057</TD><TD>-</TD><TD>5uL</TD><TD>5uL</TD><TD>-</TD></TR><TR><TD><I>TOTAL</I></TD><TD>100uL</TD><TD>100uL</TD><TD>100uL</TD><TD>100uL</TD></TR></TBODY></TABLE><P>We used this thermocycle:</P><TABLE border=""><TBODY><TR><TD><B>Temperature</B></TD><TD><B>Time</B></TD></TR><TR><TD><B>Start</B></TD><TD>98°C</TD><TD>30s</TD></TR><TR><TD><B>Cycle x35</B></TD><TD>98°C</TD><TD>10s</TD></TR><TR><TD><B>Cycle x35</B></TD><TD>52°C</TD><TD>30s</TD></TR><TR><TD><B>Cycle x35</B></TD><TD>72°C</TD><TD>30s</TD></TR><TR><TD><B>Finish</B></TD><TD>72°C</TD><TD>5min</TD></TR><TR><TD><B>Blind</B></TD><TD>10°C</TD><TD>-</TD></TR></TBODY></TABLE><H5 id="date">August 14th</H5><P> After having ran the PCR product on a gel, only one PCR seemed to have worked and the yield was very low. We made a new PCR with the same protocol except a longer elongation time (2min) and it worked: after purification, we got a concentration of 584ng/uL.</P><P> We also digested tmm PCR product, pSB1C3 and pSEVA351 before doing a ligation:</P><TABLE border=""><TBODY><TR><TD>tmm</TD><TD>pSEVA351</TD><TD>pSB1C3</TD></TR><TR><TD>NF Water</TD><TD>18uL</TD><TD>14uL</TD><TD>27uL</TD></TR><TR><TD>10X FD Buffer</TD><TD>4uL</TD><TD>4uL</TD><TD>4uL</TD></TR><TR><TD>XbaI</TD><TD>1uL</TD><TD>1uL</TD><TD>1uL</TD></TR><TR><TD>SpeI</TD><TD>1uL</TD><TD>1uL</TD><TD>1uL</TD></TR><TR><TD>DNA</TD><TD>14uL</TD><TD>18uL</TD><TD>5uL</TD></TR><TR><TD>FastAP</TD><TD>2uL</TD><TD>2uL</TD><TD>2uL</TD></TR><TR><TD><I>TOTAL</I></TD><TD>40uL</TD><TD>40uL</TD><TD>40uL</TD></TR></TBODY></TABLE><H5 id="date">August 18th</H5><P>The digestion products were purified and we got these concentrations: 84.1ng/uL for digested tmm, 41.1ng/uL for digested pSEVA351 and 28.6ng/uL digested pSB1C3.</P><P>The amounts of vector and insert were calculated thanks to <I>NEBiocalculator</I> with a insert:vector ratio of 2:1</P><TABLE border=""><TBODY><TR><TD>tmm+pSEVA351</TD><TD>tmm+pSB1C3</TD></TR><TR><TD>NF Water</TD><TD>14.5uL</TD><TD>12.5uL</TD></TR><TR><TD>10X T4 Buffer</TD><TD>2uL</TD><TD>2uL</TD></TR><TR><TD>T4 ligase</TD><TD>1uL</TD><TD>1uL</TD></TR><TR><TD>tmm</TD><TD>0.5uL</TD><TD>1.5uL</TD></TR><TR><TD>Vector</TD><TD>2uL</TD><TD>3uL</TD></TR></TBODY></TABLE><P>The tubes were incubated at 22°C for 1H and overnight at 16°C.</P><H5 id="date">August 21st</H5><P><I>E.coli</I> competent cells (NEB turbo) were transformed with tmm+pSEVA351 and tmm+pSB1C3 ligation products using the <A href="https://2014.igem.org/Team:Paris_Bettencourt/Protocols#prot5">Heat Shock transformation protocol</A>. On August 22nd we got 4 colonies of NEB transformed with tmm+pSB1C3. </P><H5 id="date">August 25th</H5><P>A <A href="https://2014.igem.org/Team:Paris_Bettencourt/Protocols#prot7">colony PCR</A> was done with 2 different colonies but the result was not convincing.</P><H5 id="date">August 27th</H5><P>Miniprep and digestion were done using liquid culture of the 2 colonies.</P><TABLE border=""><TBODY><TR><TD>colony 1</TD><TD>colony 2</TD></TR><TR><TD>NF Water</TD><TD>14.5uL</TD><TD>12.5uL</TD></TR><TR><TD>10X FD Buffer</TD><TD>2uL</TD><TD>2uL</TD></TR><TR><TD>XbaI</TD><TD>1uL</TD><TD>1uL</TD></TR><TR><TD>SpeI</TD><TD>1uL</TD><TD>1uL</TD></TR><TR><TD>DNA</TD><TD>15uL</TD><TD>11uL</TD></TR><TR><TD>Fast AP</TD><TD>2uL</TD><TD>2uL</TD></TR><TR><TD><I>TOTAL</I></TD><TD>40uL</TD><TD>40uL</TD></TR></TBODY></TABLE><P> On the gel we can see a band at 1.5kb for colony 1 but not for colony 2. The cloning seems to have worked.</P></DIV><DIV id="September"><H4>September</H4><H5 id="date">September 1st</H5><P> tmm also has to be cloned using pSEVA351 so that we can transform into C.striatum.</P><P><I>E.coli</I> was transformed using tmm+pSEVA351 ligation product from August 18th with <A href="https://2014.igem.org/Team:Paris_Bettencourt/Protocols#prot3">electroporation</A>. It did not work on the first time.</P><H5 id="date">September 3rd</H5><P>A new ligation of tmm and pSEVA351 was done:</P><TABLE border=""><TBODY><TR><TD>NF Water</TD><TD>13.5uL</TD></TR><TR><TD>10X T4 Buffer</TD><TD>2uL</TD></TR><TR><TD>T4 ligase</TD><TD>1uL</TD></TR><TR><TD>tmm</TD><TD>1uL</TD></TR><TR><TD>pSEVA351</TD><TD>2.5uL</TD></TR></TBODY></TABLE><P>The tube was incubated 1H at 22°C and overnight at 4°C.</P><H5 id="date">September 4th</H5><P>A new electroporation was done with the same <A href="https://2014.igem.org/Team:Paris_Bettencourt/Protocols#prot3">protocol</A>.Some colonies developped. Two of them were transfered in LB+Chlormaphenicol and two glycerol stocks were made with the strains called sPB.054 and sPB.055. Analytical digestion was performed on these strains: miniprep and digestion using XbaI and SpeI. The gel showed nothing corresponding to <I>tmm</I>.</P><H5 id="date">September 25th</H5><P>The cloning was tried again from the beginning. 2 new pairs of primers were designed: 1 pair (oPB.087 and oPB.088) corresponding to pSB1C3 cloning including XbaI and PstI restriction sites and 1 pair (oPB.089 and oPB.090) corresponding to pSEVA315 cloning including XbaI and HindIII restriction sites. The gBlock was amplified using the first pair (PCR1) and the second pair (PCR2) of primers. The DNA template is <I>tmm</I> gBlock diluted 1:50.</P><TABLE border=""><TBODY><TR><TD><B>Reagent</B></TD><TD><B>Volume</B></TD></TR><TR><TD>1X</TD><TD>4X</TD></TR><TR><TD>5X Phusion HF Buffer</TD><TD>20uL</TD><TD>80uL</TD></TR><TR><TD>10mM dNTPs</TD><TD>2uL</TD><TD>8uL</TD></TR><TR><TD>Forward primer</TD><TD>1uL</TD><TD>4uL</TD></TR><TR><TD>Reverse primer</TD><TD>1uL</TD><TD>4uL</TD></TR><TR><TD>DNA</TD><TD>1uL</TD><TD>4uL</TD></TR><TR><TD>DMSO</TD><TD>3uL</TD><TD>12uL</TD></TR><TR><TD>Phusion polymerase</TD><TD>1uL</TD><TD>4uL</TD></TR><TR><TD>Nuclease Free Water</TD><TD>71uL</TD><TD>284uL</TD></TR></TBODY></TABLE><P>We used this thermocycle:</P><TABLE border=""><TBODY><TR><TD><B>Temperature</B></TD><TD><B>Time</B></TD></TR><TR><TD><B>Start</B></TD><TD>98°C</TD><TD>30s</TD></TR><TR><TD><B>Cycle x35</B></TD><TD>98°C</TD><TD>10s</TD></TR><TR><TD><B>Cycle x35</B></TD><TD>52°C</TD><TD>30s</TD></TR><TR><TD><B>Cycle x35</B></TD><TD>72°C</TD><TD>40s</TD></TR><TR><TD><B>Finish</B></TD><TD>72°C</TD><TD>5min</TD></TR><TR><TD><B>Blind</B></TD><TD>10°C</TD><TD>-</TD></TR></TBODY></TABLE><P>On the gel the bands look perfect. After purification, the concentration is 80ng/uL.</P><H5 id="date">September 26th</H5><P>The PCR products were digested:</P><P><U>1 - oPB.087 and oPB.088</U></P><P>Digest with XbaI, PstI. Product is 80 ng/ul.</P><TABLE border=""><TBODY><TR><TD><B>Reagent</B></TD><TD><B>Volume</B></TD></TR><TR><TD>Nuclease free water</TD><TD>135uL</TD></TR><TR><TD>10X Fast Digest Buffer</TD><TD>20uL</TD></TR><TR><TD>PCR product</TD><TD>30uL</TD></TR><TR><TD>FastDigest XbaI</TD><TD>10uL</TD></TR><TR><TD>FastDigest PstI</TD><TD>10uL</TD></TR><TR><TD><I>TOTAL</I></TD><TD>200uL</TD></TR></TBODY></TABLE><P> The tubes were incubated at 37°C for 1H. After purification, the concentration was 26ng/uL.</P><P><U>2 - oPB.089 and oPB.090</U></P><P>Digest with XbaI, HindIII. Product is 80 ng/ul.</P><TABLE border=""><TBODY><TR><TD><B>Reagent</B></TD><TD><B>Volume</B></TD></TR><TR><TD>Nuclease free water</TD><TD>135uL</TD></TR><TR><TD>10X Fast Digest Buffer</TD><TD>20uL</TD></TR><TR><TD>PCR product</TD><TD>30uL</TD></TR><TR><TD>FastDigest XbaI</TD><TD>10uL</TD></TR><TR><TD>FastDigest HindIII</TD><TD>10uL</TD></TR><TR><TD><I>TOTAL</I></TD><TD>200uL</TD></TR></TBODY></TABLE><P> The tubes were incubated at 37°C for 1H. After purification, the concentration was 33ng/uL.</P><P>Digestion of pSEVA315</P><TABLE border=""><TBODY><TR><TD><B>Reagent</B></TD><TD><B>Volume</B></TD></TR><TR><TD>Nuclease free water</TD><TD>155uL</TD></TR><TR><TD>10X Fast Digest Buffer</TD><TD>20uL</TD></TR><TR><TD>Plasmid</TD><TD>15uL</TD></TR><TR><TD>FastDigest XbaI</TD><TD>5uL</TD></TR><TR><TD>FastDigest HindIII</TD><TD>5uL</TD></TR><TR><TD>FastAP</TD><TD>5uL</TD></TR><TR><TD><I>TOTAL</I></TD><TD>200uL</TD></TR></TBODY></TABLE><P> The tubes were incubated at 37°C for 1H. After purification, the concentration was 38ng/uL.</P><H5 id="date">September 29th</H5><P><B>Ligation and transformation</B></P><TABLE border=""><TBODY><TR><TD><B>Volume</B></TD></TR><TR><TD><B>Insert (XbaI, PstI) 26ng/uL</B></TD><TD>3uL</TD></TR><TR><TD><B>Digested pSB1C3 (XbaI, PstI) 6.5ng/uL</B></TD><TD>12uL</TD></TR><TR><TD><B>T4 Ligase</B></TD><TD>1uL</TD></TR><TR><TD><B>T4 Ligase Buffer</B></TD><TD>2uL</TD></TR><TR><TD><B>NF Water</B></TD><TD>2uL</TD></TR><TR><TD><I>TOTAL</I></TD><TD>20uL</TD></TR></TBODY></TABLE><TABLE border=""><TBODY><TR><TD><B>Volume</B></TD></TR><TR><TD><B>Insert (XbaI, HindIII) 33ng/uL</B></TD><TD>2uL</TD></TR><TR><TD><B>Digested pSEVA315 (XbaI, HindIII) 38ng/uL</B></TD><TD>2uL</TD></TR><TR><TD><B>T4 Ligase</B></TD><TD>1uL</TD></TR><TR><TD><B>T4 Ligase Buffer</B></TD><TD>2uL</TD></TR><TR><TD><B>NF Water</B></TD><TD>13uL</TD></TR><TR><TD><I>TOTAL</I></TD><TD>20uL</TD></TR></TBODY></TABLE><P>The transformation was performed according to the <A href="https://2014.igem.org/Team:Paris_Bettencourt/Protocols#prot5">Heat Shock transformation protocol</A>.
</P><P>After 24h, we observed many colonies, including some dark ones which could be due to indigo produced by TMM. These dark colonies were transfered in order to miniprep them and do an analytical digestion.</P></DIV><DIV id="October"><H4>October</H4><H5 id="date">October 1st</H5><P>After miniprep, the plasmids were digested.</P><TABLE border=""><TBODY><TR><TD><B>Reagent</B></TD><TD><B>Volume</B></TD></TR><TR><TD>Nuclease free water</TD><TD>13uL</TD></TR><TR><TD>10X Fast Digest Buffer</TD><TD>2uL</TD></TR><TR><TD>Plasmid</TD><TD>3uL</TD></TR><TR><TD>FastDigest XbaI</TD><TD>1uL</TD></TR><TR><TD>FastDigest PstI or HindIII</TD><TD>1uL</TD></TR><TR><TD><I>TOTAL</I></TD><TD>20uL</TD></TR></TBODY></TABLE><P>On the gel, we can see bands at 1.5kb corresponding to <I>tmm</I> (<I>tmm</I>+pSB1C3 on the left and <I>tmm</I>+pSEVA315 on the right). The cloning worked.</P><P>Glycerol stocks were prepared with these strains called sPB.091 (pSB1C3) and sPB.092 (pSEVA315).</P><H5 id="date">October 8th</H5><P id="text">TMM activity was found in <I>E.coli</I> pSB1C3-TMM (tmm+) but not in <I>E.coli</I> pSB1C3 (tmm-). TMM does not only degrade trimethylamine into trimethylamine-N-oxide, but also converts indole into indigo. To measure the activity of TMM, the growth medium was supplemented with tryptophan, a precursor of indole, which is the substrate of TMM. After 14h of culture, cells were pelleted, washed with water twice, and then resuspended in DMSO and sonicated. TMM activity was determined by measuring the absorbance spectrum of bacterial extractions. Peaks at 620 nm were found in tmm+ cultures supplemented with tryptophan, which was identified as indigo according absorbance spectrum analysis.</P><P>Details: we made a overnight preculture of 2 strains (sPB.026: control and sPB.091: pSB1C3+tmm) in LB+Cm. Then, the cells were resuspended and diluted in 3 tubes of each to get a OD600 of 0.2 (beginning of the exponential phase). The resuspension solution was LB or LB supplemented with tryptophane (2g/L): 3 tubes +tmm +TRP, 3 tubes -tmm +TRP, 3 tubes +tmm -TRP and 3 tubes -tmm -TRP. After 14h of culture at 37°C, the cells were washed twice (centrifugation 4000<I>g</I> 10min and resuspension in 10mL of water) and resuspended in 1mL DMSO. After having sonicated the cells, the solution was centrifuged (4000<I>g</I> 10min) and the supernatant was transfered into cuvettes to measure the absorbance spectrum. The results are in <A href="https://static.igem.org/mediawiki/2014/b/b8/Indigo_spectrum_PB.xls">this file</A> .</P><H5 id="date">October 9th</H5><P> A dilution scale of TMA was prepared to determine what is the lowest concentration people can smell in order to know which concentration we have to use in the olfactory experiment. Concentrations from 1M to 1uM were prepared with serial dilutions. 10 people were asked to smell the tubes from the lowest concentration and to stop when they smell fish odor. The results show that most of people smell trimethylamine from to a concentration of 1mM.</P><H5 id="date">October 9th</H5><P>GC/MS analysis were performed in the chemistry lab of Paris-Diderot University. We prepared overnight precultures of TMM-expressing <I>E. coli</I> and <I>E. coli</I> expressing empty pSB1C3. After having diluted them in LB to get an OD600 of 0.2, we added 1mM of TMA and incubated the tubes for 36H at 37°C. Then we brought the tubes for the analysis and got these figures.</P><H5 id="date">October 15th</H5><P id="text"><I>C.striatum</I> was transformed with pSVEA351 containing <I>tmm</I> gene. according to <A href="https://2014.igem.org/Team:Paris_Bettencourt/Protocols#prot11">this protocol</A>. 70mL LB medium were supplemented with 7mL of 50% glucose solution. After 2h of recovery, the cells were plated on LB supplemented with chloramphenicol plates (one with 20uL and one with 200uL). Another plating was done the day after (20uL and 200uL). 
					</P><P id="text">Text</P></DIV></DIV><DIV id="mfooter"><DIV class="footbox"><B>Centre for Research and Interdisciplinarity (CRI)</B>
    Faculty of Medicine Cochin Port-Royal, South wing, 2nd floor
    Paris Descartes University
    24, rue du Faubourg Saint Jacques
    75014 Paris, France +33 1 44 41 25 22/25<A href="mailto:paris-bettencourt-igem@googlegroups.com">paris-bettencourt-igem@googlegroups.com</A></DIV><CENTER>
    Copyright (c) 2014 igem.org. All rights reserved.
  </CENTER></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Paris_Bettencourt/Notebook/TMAU">http://2014.igem.org/Team:Paris_Bettencourt/Notebook/TMAU</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Paris_Bettencourt/Notebook/TMAU" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Paris_Bettencourt/Notebook/TMAU" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Paris_Bettencourt/Notebook/TMAU&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Paris_Bettencourt/Notebook/TMAU&amp;oldid=395656" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>