"
Page
Discussion
View source
History
teams
Log in
 
Team:Paris Bettencourt/Notebook/TMAU
From 2014.igem.org
Project
SmellThe Roses
SomethingFishy
GoodyTwo Shoes
Don'tSweat It
Teen Spirit
Design Practice
Bioinformatics
Achievements
Results
Interlab Study
MedalFulfilment
Parts
Policy
Safety
Consent Form
Patenting
Practices
MOOC
CitizenScience
The Senseof Smell
The HumanMicriobiome
Minder
Collaborations
Collaborations
The SmellGame
Newsletters
Notebook
Daily Note
Protocols
Bibliography
Team
Our Team
Attributions
Official TeamProfile
Acknowledgements
@iGEM_Paris
Smell The Roses
Something Fishy
Don't Sweat It
Goody Two Shoes
Teen Spirit
Interlab Study
June
July
August
September
October
Notebook
June
June 24th
oPB.010 received
oPB.010
tmm Fw for BioBrick vector
TATAGAATTCGCGGCCGCTTCTAGAGCTGACAGCTAGCTCAGTCCTAG
June 29th
Goal: to PCR ilvBN gene from MG1655
Procedure:
1. Picked 2 colonies from the plate MG1655 115+GFP
2. inoculate them each in 50 ul of NF H2O
3. Boiled for 3 mins at 98 degree
4. use the solution as DNA template
5. follow standard PCR protocol from this step on
Reagent Volume 1x
Nuclease-free water 63 ul
5x Phusion HF Buffer 20 ul
10 mM dNTPs 2 ul
Forward Primer (10 uM) 5 ul
Reverse Primer (10 uM) 5 ul
Genomic DNA template 1 ul
DMSO 3 ul
Phusion DNA Polymerase 1 ul
Total Volume 100 ul
Thermocycler protocol: NEB Phusion
Temp Time
Start 98 ° C 30 sec Melt
35 Cycles
Cycle 1 98 ° C 10 sec Melt
Cycle 2 52 ° C 30 sec Anneal
Cycle 3 72 ° C 2 minutes Extend
Finish 72 ° C 5 min Extend
Blind 10 ° C Forever
Blind
outcome:
bands seen at expected length.
July
July 3rd
We made a PCR of our tmm gBlock using this protocol:
Reagent
Volume
1X
4X
5X Phusion HF Buffer
20uL
80uL
10mM dNTPs
2uL
8uL
oPB.010 (forward primer)
1uL
4uL
oPB.012 (reverse primer)
1uL
4uL
DNA
1uL
4uL
DMSO
3uL
12uL
Phusion polymerase
1uL
4uL
Nuclease Free Water
71uL
284uL
We used this thermocycle:
Temperature
Time
Start
98°C
30s
Cycle x35
98°C
10s
Cycle x35
52°C
30s
Cycle x35
72°C
30s
Finish
72°C
5min
Blind
10°C
-
After gel migration, we got this:
The PCR worked.
Miniprep using Thermo scientific kits
Digestion using EcoR1 and Pst1
Extracted plasmids from miniprep 22 uL 30uL
FD Buffer 5uL 5uL
Pst1 2.5uL 2.5uL
EcoR1 2.5uL 2.5uL
NF Water 18uL 10uL
Incubation for 5min 37°C and 2 hours at room temperature.
Ligation of gBlock into Vector
PCR product: tmm gBlock with oPB.010 and oPB.012
Vector: pSB1C3 linearized
Concentration:
-digested PSB1C3: 4.6ng/uL
-digested tmm: 13.7ng/uL
Cohesive Ends
5X Ligase Reaction Buffer 4 μl
Insert: Vector Molar Ratio 3:1
Vector Ends 5ul
Insert Ends 5ul
Total DNA 0.092ug(0.01-0.1 μg)
T4 DNA Ligase 0.1 unit
Autoclaved distilled water to 20 μl
Incubate at 24°C for O/N.
Note: For optimal transformation, dilute the ligation reaction ≥ 5-fold, to at least 100 μl, before adding to competent cells
July 5th
Transformation of the part.
Strain: E.coli MG1655
plasmid: pPB.011
Heat Shock Transformation of E. coli
This protocol can be used to transform chemically competent (i.e. from CaCl2) with a miniprepped plasmid or a ligation product.
Note: Never vortex competent cells. Mix cells by gentle shaking.
No colonies after 24h, this might be due to the low concentration of reagents
July 12th
We transformed NEB and MG1655 with our ligation product according to this protocol
We finally had no colonies.
July 10th
1) Mini prep to extract plasmids (Qiagen)
I used this protocol . I made 2 tubes and got 163.2 and 242 ng/uL as concentrations.
2) Digestion of miniprep product with EcoR1 & Pst1
Tube 1 (163ng/uL)
Tube 2 (242ng/uL)
pSB1C3 from miniprep
22uL
30uL
10X FD Buffer
5uL
5uL
PstI
2.5uL
2.5uL
EcoRI
2.5uL
2.5uL
NF Water
18uL
10uL
3) Gel migration
I had the digested pSB1C3 run on a gel using 0.5X TBE (2H-50V). We clearly saw a band at 2kb corresponding to pSB1C3. I cut the gel and stored it at 4°C.
July 11th
1) Gel purification of digested pSB1C3
I used this protocol
Result: I got concentrations of 59.6 and 55.1 ng/uL.
2) Ligation of digested tmm and digested pSB1C3
I used NEBiocalculator to calculate the mass of insert and vector I had to put (Insert length: 1488 bp, Vector length: 2037 bp, Vector DNA mass: 0,3 ug, Ratio insert:vector: 2:1). I need to put 450ng of insert and 300 ng of vector.
Volume
Insert (tmm)
20uL of 10.1 ng/uL and 20uL of 13.7 ng/uL
Vector (pSB1C3)
5uL of 59.6ng/uL
T4 Ligase
3uL
T4 Ligase Buffer
6uL
NF Water
6uL
I incubated overnight at room temperature
July 14th
July 15th
1) Gel extraction of pSB1C3 digested plasmid
I used this protocol . I finally got 2 tubes with 13.9 and 19.8ng/uL concentrations.
2) PCR purification
I purified the PCR product of July 14th using this protocol .
3) Ligation of tmm PCR product and pSB1C3
I used NEBiocalculator to to calculate the needed amount of insert and vector with the values:
Insert length: 1488 bp
Vector length: 2037 bp
Vector DNA mass: 50 ng
Ratio insert: I made 1 ligation of ratio 1:1, 2:1 and 3:1 (insert:vector).
I made the ligation with the 13.9 ng/uL concentrated insert tube. I used this protocol:
1:1
2:1
3:1
Digested tmm (insert)
1.5uL
3uL
4uL
Digested pSB1C3 (vector)
4uL
4uL
4uL
T4 Ligase
1uL
1uL
1uL
T4 Ligase Buffer
4uL
4uL
4uL
Nuclease Free water
9.5uL
8uL
7uL
I incubated the tubes at 24°C for 30min.
4) Transformation into NEB and MG1655
I used the Heat Shock transformation protocol to transform NEB and MG1655 with each one of the ligation products with ratios 1:1, 2:1 and 3:1 (I got 6 plates).
July 16th
1) Results of the transformations
We got no colonies in MG1655, big and small colonies for 1:1 and 2:1 ratios and only big colonies with 3:1 ratios. I prepared 5 liquid cultures in LB+Chloramphenicol: 1 for each type of colony for 1:1 and 2:1 ratios, and 1 for big colonies for 3:1 ratio. I incubated them at 37°C.
2) PCR colony
To check if the colonies developped in my plates have tmm gene, I made a PCR with this protocol and I ran a gel. I got nothing on it. I will do miniprep and digestion on July 17th to identify tmm construct into pSB1C3.
July 17th
1) Miniprep using each liquid culture of NEB+tmm-pSB1C3
I used this protocol .
2) Digestion of the plasmids by EcoRI
Then I digested the miniprep product by EcoRI in order to get a linearised plasmid before running it on a gel.
Reagent
Volume
Nuclease free water
12uL
10X Fast Digest Buffer
2uL
plasmid DNA
5uL
EcoRI
1uL
TOTAL
20uL
I incubated the tubes at 37°C during 30min.
3) Gel
I ran on a gel each one of the tubes (small and big colonies from 1:1 and 2:1 ratios and big colony from 3:1 ratio). But the plasmids may have not been digested enough because I got many bands.
July 18th
I will try again to identify tmm gene in my transformed bacteria.
1) Digestion of plasmids with EcoRI and PstI.
This time I digested by EcoRI and PstI to be able to directly identify tmm gene on the gel. I nanodroped the miniprep product from 17th. We need 1ug of DNA for the digestion.
Concentration (ng/uL, given by Nanodrop)
Needed volume (uL)
big 1:1 ratio colony
316.4
3.5
small 2:1 ratio colony
156.9
7
big 2:1 ratio colony
207.6
5
big 3:1 ratio colony
207.6
4
Digestion protocol:
Reagent
big 1:1 ratio colony
small 2:1 ratio colony
big 2:1 ratio colony
big 3:1 ratio colony
Nuclease free water
13.5uL
10uL
12uL
13uL
10X Fast Digest Buffer
2uL
2uL
2uL
2uL
plasmid DNA
3.5uL
7uL
5uL
4uL
EcoRI
0.5uL
0.5uL
0.5uL
0.5uL
PstI
0.5uL
0.5uL
0.5uL
0.5uL
TOTAL
20uL
20uL
20uL
20uL
1)Gel
I ran all the samples on a gel but I did not get the bands I expected. We have to try a new cloning.
July 22nd
I had prepared liquid cultures to be able to do minipreps.
1) Miniprep of liquid cultures to get pSB1C3
I used this protocol . The Nanodrop gave me a concentration of 317ng/uL.
2) Digestion of pSB1C3 and tmm PCR product by EcoRI and PstI.
pSB1C3:
Reagent
Volume
Nuclease free water
25uL
10X Fast Digest Buffer
5uL
plasmid DNA
14uL
EcoRI
2.5uL
PstI
2.5uL
Fast AP (phosphatase)
1uL
TOTAL
40uL
tmm PCR product
Reagent
Volume
Nuclease free water
27uL
10X Fast Digest Buffer
5uL
plasmid DNA
13uL
EcoRI
2.5uL
PstI
2.5uL
TOTAL
50uL
3)Gel
For pSB1C3 we can see two bands corresponding to 900bp and 2000bp (backbone) fragments. I will be able to extract the 2000bp fragment. But I got nothing for tmm.
4)Gel extraction
I used this protocol .
July 29th
We need tmm PCR product to be able to do a new cloning.
1) PCR of tmm gBlock
Reagent
Volume
1X
5X Phusion HF Buffer
10uL
10mM dNTPs
1uL
oPB.010 (forward primer)
0.5uL
oPB.012 (reverse primer)
0.5uL
DNA
1uL
DMSO
1.5uL
Phusion polymerase
0.5uL
Nuclease Free Water
35uL
We used this thermocycle:
Temperature
Time
Start
98°C
30s
Cycle x35
98°C
10s
Cycle x35
56°C for tube 1, 51°C for tubes 2 and 3
30s
Cycle x35
72°C
1min
Finish
72°C
5min
Blind
10°C
-
2) Gel
No bands were visible, even for the ladder. I tried a new one and got a few tiny bands for tube 3, particularly at 1500bp. I tried a gel extraction using this protocol and got 4.7ng/uL (unusable). I tried otherwise:
3) PCR purification
I puridied tube 3 according to this protocol and got a concentration of 11ng/uL which was not enough to do a digestion.
4) New PCR
Reagent
Volume
1X
5X Phusion HF Buffer
10uL
10mM dNTPs
1uL
oPB.010 (forward primer)
0.5uL
oPB.012 (reverse primer)
0.5uL
DNA
1uL
DMSO
1.5uL
Phusion polymerase
0.5uL
Nuclease Free Water
35uL
We used this thermocycle:
Temperature
Time
Start
98°C
30s
Cycle x35
98°C
10s
Cycle x35
55°C
30s
Cycle x35
72°C
1min
Finish
72°C
5min
Blind
10°C
-
3) PCR purification
I puridied tube 3 according to this protocol .
August
August 1st
1)Digestion of purified PCR product
Reagent
Volume
Nuclease free water
20uL
10X Fast Digest Buffer
20uL
plasmid DNA
7uL
EcoRI
0.5uL
PstI
0.5uL
TOTAL
50uL
I incubated the digestion at 37°C during 1h.
2) Purification of digested tmm
I used this protocol and got a really low concentration, unusable for ligation.
3) New PCR
I prepared 4 tubes according to this protocol:
Reagent
Volume
1X
5X Phusion HF Buffer
10uL
10mM dNTPs
1uL
oPB.010 (forward primer)
0.5uL
oPB.012 (reverse primer)
0.5uL
DNA
1uL
DMSO
1.5uL
Phusion polymerase
0.5uL
Nuclease Free Water
35uL
We used this thermocycle:
Temperature
Time
Start
98°C
30s
Cycle x35
98°C
10s
Cycle x35
51°C
30s
Cycle x35
72°C
1min
Finish
72°C
5min
Blind
10°C
-
4) Gel
Nothing was visible on the gel.
5) New PCR
Reagent
Volume
1X
5X Phusion HF Buffer
20uL
10mM dNTPs
2uL
oPB.010 (forward primer)
5uL
oPB.012 (reverse primer)
5uL
DNA
1uL
DMSO
3uL
Phusion polymerase
2uL
Nuclease Free Water
53uL
We used this thermocycle:
Temperature
Time
Start
98°C
30s
Cycle x35
98°C
10s
Cycle x35
50°C
30s
Cycle x35
72°C
45s
Finish
72°C
5min
Blind
10°C
-
6) Gel
There was a diffused band next to 200bp. Maybe there is a problem of specificity of the primers. We will try a PCR with the very first protocol which worked.
August 5th
1) PCR of tmm gBlock
Reagent
Volume
1X
5X Phusion HF Buffer
20uL
10mM dNTPs
2uL
oPB.010 (forward primer)
1uL
oPB.012 (reverse primer)
1uL
tmm DNA
1uL
DMSO
3uL
Phusion polymerase
1uL
Nuclease Free Water
71uL
We used this thermocycle:
Temperature
Time
Start
98°C
30s
Cycle x35
98°C
10s
Cycle x35
52°C
30s
Cycle x35
72°C
30s
Finish
72°C
5min
Blind
10°C
-
We still got nothing. We decided to design new primers.
August 6th
We think that that the reverse primer (oPB012) might have a strong secondary structure and that it may prevent the PCR to work as expected. We designed modified versions of oPB010 and oPB012 respectively called oPB056 and oPB057, shorter then the first ones. We removed the first restriction site at each end making sure the other enzyme would still have enough nucleotides to bind. We also shortened the bind part to have a melting temperature around 52°C.
oPB.010 TATAGAATTCGCGGCCGCTTCTAGAGCTGACAGCTAGCTCAGTCCTAG -> oPB.056     CTTCTAGAGCTGACAGCTAGCTCAGTCC oPB.012 TTAACTGCAGCGGCCGCTACTAGTATCAGTGGTGATGGTGATGATGGTTA -> oPB.057 TACTAGTATCAGTGGTGATGGTGATGATG
Note:The digestion should now use XbaI and SpeI as EcorI and PstI were removed from the PCR product.
August 13th
We received our new primers (oPB.056 and oPB.057).
1) New PCR with different combinations or primers
We used this protocol:
Reagent
Tube 1
Tube 2
Tube 3
Tube 4
5X Phusion HF Buffer
20uL
20uL
20uL
20uL
10mM dNTPs
2uL
2uL
2uL
2uL
tmm DNA
10L
10L
10L
10L
DMSO
3uL
3uL
3uL
3uL
Phusion polymerase
2uL
2uL
2uL
2uL
Nuclease Free Water
53uL
53uL
53uL
53uL
oPB.010
5uL
-
5uL
-
oPB.012
5uL
-
-
5uL
oPB.056
-
5uL
-
5uL
oPB.057
-
5uL
5uL
-
TOTAL
100uL
100uL
100uL
100uL
We used this thermocycle:
Temperature
Time
Start
98°C
30s
Cycle x35
98°C
10s
Cycle x35
52°C
30s
Cycle x35
72°C
30s
Finish
72°C
5min
Blind
10°C
-
August 14th
After having ran the PCR product on a gel, only one PCR seemed to have worked and the yield was very low. We made a new PCR with the same protocol except a longer elongation time (2min) and it worked: after purification, we got a concentration of 584ng/uL.
We also digested tmm PCR product, pSB1C3 and pSEVA351 before doing a ligation:
tmm
pSEVA351
pSB1C3
NF Water
18uL
14uL
27uL
10X FD Buffer
4uL
4uL
4uL
XbaI
1uL
1uL
1uL
SpeI
1uL
1uL
1uL
DNA
14uL
18uL
5uL
FastAP
2uL
2uL
2uL
TOTAL
40uL
40uL
40uL
August 18th
The digestion products were purified and we got these concentrations: 84.1ng/uL for digested tmm, 41.1ng/uL for digested pSEVA351 and 28.6ng/uL digested pSB1C3.
The amounts of vector and insert were calculated thanks to NEBiocalculator with a insert:vector ratio of 2:1
tmm+pSEVA351
tmm+pSB1C3
NF Water
14.5uL
12.5uL
10X T4 Buffer
2uL
2uL
T4 ligase
1uL
1uL
tmm
0.5uL
1.5uL
Vector
2uL
3uL
The tubes were incubated at 22°C for 1H and overnight at 16°C.
August 21st
E.coli competent cells (NEB turbo) were transformed with tmm+pSEVA351 and tmm+pSB1C3 ligation products using the Heat Shock transformation protocol . On August 22nd we got 4 colonies of NEB transformed with tmm+pSB1C3.
August 25th
A colony PCR was done with 2 different colonies but the result was not convincing.
August 27th
Miniprep and digestion were done using liquid culture of the 2 colonies.
colony 1
colony 2
NF Water
14.5uL
12.5uL
10X FD Buffer
2uL
2uL
XbaI
1uL
1uL
SpeI
1uL
1uL
DNA
15uL
11uL
Fast AP
2uL
2uL
TOTAL
40uL
40uL
On the gel we can see a band at 1.5kb for colony 1 but not for colony 2. The cloning seems to have worked.
September
September 1st
tmm also has to be cloned using pSEVA351 so that we can transform into C.striatum.
E.coli was transformed using tmm+pSEVA351 ligation product from August 18th with electroporation . It did not work on the first time.
September 3rd
A new ligation of tmm and pSEVA351 was done:
NF Water
13.5uL
10X T4 Buffer
2uL
T4 ligase
1uL
tmm
1uL
pSEVA351
2.5uL
The tube was incubated 1H at 22°C and overnight at 4°C.
September 4th
A new electroporation was done with the same protocol .Some colonies developped. Two of them were transfered in LB+Chlormaphenicol and two glycerol stocks were made with the strains called sPB.054 and sPB.055. Analytical digestion was performed on these strains: miniprep and digestion using XbaI and SpeI. The gel showed nothing corresponding to tmm.
September 25th
The cloning was tried again from the beginning. 2 new pairs of primers were designed: 1 pair (oPB.087 and oPB.088) corresponding to pSB1C3 cloning including XbaI and PstI restriction sites and 1 pair (oPB.089 and oPB.090) corresponding to pSEVA315 cloning including XbaI and HindIII restriction sites. The gBlock was amplified using the first pair (PCR1) and the second pair (PCR2) of primers. The DNA template is tmm gBlock diluted 1:50.
Reagent
Volume
1X
4X
5X Phusion HF Buffer
20uL
80uL
10mM dNTPs
2uL
8uL
Forward primer
1uL
4uL
Reverse primer
1uL
4uL
DNA
1uL
4uL
DMSO
3uL
12uL
Phusion polymerase
1uL
4uL
Nuclease Free Water
71uL
284uL
We used this thermocycle:
Temperature
Time
Start
98°C
30s
Cycle x35
98°C
10s
Cycle x35
52°C
30s
Cycle x35
72°C
40s
Finish
72°C
5min
Blind
10°C
-
On the gel the bands look perfect. After purification, the concentration is 80ng/uL.
September 26th
The PCR products were digested:
1 - oPB.087 and oPB.088
Digest with XbaI, PstI. Product is 80 ng/ul.
Reagent
Volume
Nuclease free water
135uL
10X Fast Digest Buffer
20uL
PCR product
30uL
FastDigest XbaI
10uL
FastDigest PstI
10uL
TOTAL
200uL
The tubes were incubated at 37°C for 1H. After purification, the concentration was 26ng/uL.
2 - oPB.089 and oPB.090
Digest with XbaI, HindIII. Product is 80 ng/ul.
Reagent
Volume
Nuclease free water
135uL
10X Fast Digest Buffer
20uL
PCR product
30uL
FastDigest XbaI
10uL
FastDigest HindIII
10uL
TOTAL
200uL
The tubes were incubated at 37°C for 1H. After purification, the concentration was 33ng/uL.
Digestion of pSEVA315
Reagent
Volume
Nuclease free water
155uL
10X Fast Digest Buffer
20uL
Plasmid
15uL
FastDigest XbaI
5uL
FastDigest HindIII
5uL
FastAP
5uL
TOTAL
200uL
The tubes were incubated at 37°C for 1H. After purification, the concentration was 38ng/uL.
September 29th
Ligation and transformation
Volume
Insert (XbaI, PstI) 26ng/uL
3uL
Digested pSB1C3 (XbaI, PstI) 6.5ng/uL
12uL
T4 Ligase
1uL
T4 Ligase Buffer
2uL
NF Water
2uL
TOTAL
20uL
Volume
Insert (XbaI, HindIII) 33ng/uL
2uL
Digested pSEVA315 (XbaI, HindIII) 38ng/uL
2uL
T4 Ligase
1uL
T4 Ligase Buffer
2uL
NF Water
13uL
TOTAL
20uL
The transformation was performed according to the Heat Shock transformation protocol .
After 24h, we observed many colonies, including some dark ones which could be due to indigo produced by TMM. These dark colonies were transfered in order to miniprep them and do an analytical digestion.
October
October 1st
After miniprep, the plasmids were digested.
Reagent
Volume
Nuclease free water
13uL
10X Fast Digest Buffer
2uL
Plasmid
3uL
FastDigest XbaI
1uL
FastDigest PstI or HindIII
1uL
TOTAL
20uL
On the gel, we can see bands at 1.5kb corresponding to tmm (tmm+pSB1C3 on the left and tmm+pSEVA315 on the right). The cloning worked.
Glycerol stocks were prepared with these strains called sPB.091 (pSB1C3) and sPB.092 (pSEVA315).
October 8th
TMM activity was found in E.coli pSB1C3-TMM (tmm+) but not in E.coli pSB1C3 (tmm-). TMM does not only degrade trimethylamine into trimethylamine-N-oxide, but also converts indole into indigo. To measure the activity of TMM, the growth medium was supplemented with tryptophan, a precursor of indole, which is the substrate of TMM. After 14h of culture, cells were pelleted, washed with water twice, and then resuspended in DMSO and sonicated. TMM activity was determined by measuring the absorbance spectrum of bacterial extractions. Peaks at 620 nm were found in tmm+ cultures supplemented with tryptophan, which was identified as indigo according absorbance spectrum analysis.
Details: we made a overnight preculture of 2 strains (sPB.026: control and sPB.091: pSB1C3+tmm) in LB+Cm. Then, the cells were resuspended and diluted in 3 tubes of each to get a OD600 of 0.2 (beginning of the exponential phase). The resuspension solution was LB or LB supplemented with tryptophane (2g/L): 3 tubes +tmm +TRP, 3 tubes -tmm +TRP, 3 tubes +tmm -TRP and 3 tubes -tmm -TRP. After 14h of culture at 37°C, the cells were washed twice (centrifugation 4000g 10min and resuspension in 10mL of water) and resuspended in 1mL DMSO. After having sonicated the cells, the solution was centrifuged (4000g 10min) and the supernatant was transfered into cuvettes to measure the absorbance spectrum. The results are in this file .
October 9th
A dilution scale of TMA was prepared to determine what is the lowest concentration people can smell in order to know which concentration we have to use in the olfactory experiment. Concentrations from 1M to 1uM were prepared with serial dilutions. 10 people were asked to smell the tubes from the lowest concentration and to stop when they smell fish odor. The results show that most of people smell trimethylamine from to a concentration of 1mM.
October 9th
GC/MS analysis were performed in the chemistry lab of Paris-Diderot University. We prepared overnight precultures of TMM-expressing E. coli and E. coli expressing empty pSB1C3. After having diluted them in LB to get an OD600 of 0.2, we added 1mM of TMA and incubated the tubes for 36H at 37°C. Then we brought the tubes for the analysis and got these figures.
October 15th
C.striatum was transformed with pSVEA351 containing tmm gene. according to this protocol . 70mL LB medium were supplemented with 7mL of 50% glucose solution. After 2h of recovery, the cells were plated on LB supplemented with chloramphenicol plates (one with 20uL and one with 200uL). Another plating was done the day after (20uL and 200uL).
Text
Centre for Research and Interdisciplinarity (CRI) Faculty of Medicine Cochin Port-Royal, South wing, 2nd floor Paris Descartes University 24, rue du Faubourg Saint Jacques 75014 Paris, France
+33 1 44 41 25 22/25
paris-bettencourt-igem@googlegroups.com
Copyright (c) 2014 igem.org. All rights reserved.
Retrieved from " http://2014.igem.org/Team:Paris_Bettencourt/Notebook/TMAU "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
