<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Paris_Bettencourt_Newsletter2"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Paris_Bettencourt/Newsletter2">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Paris_Bettencourt/Newsletter2&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Paris_Bettencourt/Newsletter2&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Paris_Bettencourt/Newsletter2&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Paris_Bettencourt/Newsletter2" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Paris Bettencourt/Newsletter2</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><DIV class="navmenubkg"><DIV id="navmenubox"><UL id="nav"><LI><A href="https://2014.igem.org/Team:Paris_Bettencourt/Project">Project</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Project/Odor_Library">SmellThe Roses</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Project/TMAU">SomethingFishy</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Project/Foot_Odor">GoodyTwo Shoes</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Project/Eliminate_Smell">Don'tSweat It</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Project/Old_People_Smell">Teen Spirit</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Project/Art">Design Practice </A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Project/Bioinformatics">Bioinformatics</A></LI></UL><LI><A href="https://2014.igem.org/Team:Paris_Bettencourt/Achievement">Achievements</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Results">Results</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Project/Interlab_Study">Interlab Study</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Medal">MedalFulfilment</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Parts">Parts</A></LI><LI><A href="https://2014.igem.org/Team:Paris_Bettencourt/Policy">Policy </A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Safety">Safety</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Consent_Form">Consent Form</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Patenting">Patenting</A></LI><LI><A href="https://2014.igem.org/Team:Paris_Bettencourt/Human_Practice">Practices</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/MOOC">MOOC</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Citizen_Science">CitizenScience</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Facts_Smell">The Senseof Smell</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Facts_Micriobiome">The HumanMicriobiome</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Minder">Minder</A></LI><LI><A href="https://2014.igem.org/Team:Paris_Bettencourt/Collaboration_Main">Collaborations </A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Collaboration">Collaborations </A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/The_Smell_Game">The SmellGame</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Newsletter">Newsletters</A></LI><LI><A href="https://2014.igem.org/Team:Paris_Bettencourt/Notebook_Main">Notebook</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Notebook">Daily Note</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Protocols">Protocols</A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Bibliograpy">Bibliography</A></LI><LI><A href="https://2014.igem.org/Team:Paris_Bettencourt/Team_Main">Team </A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Team">Our Team </A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Attributions">Attributions</A></LI><LI><A class="liste" href="https://igem.org/Team.cgi?year=2014&amp;team_name=Paris_Bettencourt">Official TeamProfile </A></LI><LI><A class="liste" href="https://2014.igem.org/Team:Paris_Bettencourt/Acknowledgements">Acknowledgements</A></LI><DIV id="twnoticon" class="sidelogos"><A class="twitter-timeline" width="250" height="400" data-dnt="true" href="https://twitter.com/iGEM_Paris" data-widget-id="368144501119795201">@iGEM_Paris</A></DIV></DIV></DIV></DIV><P id="top"><A id="link" href="#topheader">Back to top</A></P><DIV id="shadow"><UL id="menu-accordeon"><LI><A href="https://2014.igem.org/Team:Paris_Bettencourt/Newsletter">Newsletter</A></LI><LI><A href="https://2014.igem.org/Team:Paris_Bettencourt/Newsletter1">- Newsletter 1</A></LI><LI><A href="https://2014.igem.org/Team:Paris_Bettencourt/Newsletter2">- Newsletter 2</A></LI><LI><A href="https://2014.igem.org/Team:Paris_Bettencourt/Newsletter3">- Newsletter 3</A></LI><LI><A href="https://2014.igem.org/Team:Paris_Bettencourt/Newsletter4">- Newsletter 4</A></LI></UL></DIV><DIV id="map1"><TABLE id="tablelist"><TBODY><TR><TD id="case3">iGEM 2014Weekly newsletterIssue N°2</TD></TR></TBODY></TABLE></DIV><DIV id="colombia"><H5>Newsletter n°2</H5><H6>from August 11th to August 24th</H6><H4>Colombia</H4><TABLE id="table"><TBODY><TR><TD id="case1"><P><B>OUR TEAM</B></P></TD><TD id="case2"><P id="text"><B>We are iGEM Colombia Team, we are students from Universidad
								de los Andes in Bogotá, Colombia and this is our fourth year in
								the iGEM Competition.</B></P><P id="text">In 2012 the team won the Regional Jamboree with a bacteria
								capable of detecting a phytopathogen and alerting the plant of its
								presence. Students from biological sciences, medicine, physics,
								chemistry, computer and chemical engineering comprise our
								team. We have students from different ages: we have master
								students that help the smaller ones to learn the techniques and
								undergraduate students that go from freshman to senior year.
				</P></TD></TR><TR><TD id="case1"><P><B>OUR PROJECT</B></P></TD><TD id="case2"><P id="text">Cholera, caused by the bacterial pathogen Vibrio cholerae, has
				been a scourge to civilizations the world over since ancient times.
				Our project aims to use synthetic biology to develop a V. cholerae
				sensor using a new technique. We propose detecting V. cholerae’s
				species-specific quorum sensing molecule. If we rewire V.
				cholerae’s own quorum sensing mechanism in a harmless E. coli
				chassis, we can build a biosensor that gives a color output when it
				senses the pathogen.</P><P id="text">We have finished our deterministic model, but we still have
				problems with the parameters , as usual. At the lab we just finished
				InterLab parts and 3 biobricks of our own project … we are still
				working on more parts, specially Tetracycline Repressor from the
				registry, which has a strange molecular weight.</P><P id="text">We also had a Bake Sale this week and collected funds to go to
				the Jamboree. At our HP practices project we have received a lot
				of collaboration from many teams. THANK YOU SO MUCH. If you
				want to know what we are doing and want to collaborate, contact
				us.</P><P id="text">Our system is based in a phosphorylation cascade that is reversed
				in the presence of an Auto inductor (CAI-1). When there is V.
				Cholerae in the media CqsS senses it and changes it functions
				to phosphase. CqsS takes the phosphate from LuxU, which takes
				the phosphate from LuxO and turns off the production of TetR
				(Repressor of pTet). This process allows the production of the
				signal Amilcp and the activator TetA ,creating a positive feedback.
			</P></TD></TR></TBODY></TABLE><DIV id="media"><P><A id="link" href="https://twitter.com/Colombia_igem1">@Colombia_igem1</A><A id="link" href="https://www.facebook.com/colombia.igem?fref=nf">Colombia Team- iGEM</A><A id="link">d.olivera1320@uniandes.edu.co</A></P></DIV></DIV><DIV id="eth"><H5>Newsletter n°2</H5><H6>from August 11th to August 24th</H6><H4>ETH Zürich</H4><TABLE id="table"><TBODY><TR><TD id="case1"><P><B>PROJECT UPGRADE</B></P></TD><TD id="case2"><P id="text">Mosaicoli aims to investigate emergence of complex patterns by
					engineering a cellular automaton in bacteria. The design of our
					system was only a first step. While the wet lab worked hard to
					assemble our constructs, the dry lab focused on the deterministic
					model and its parameters. Finally, this week, we began to do our
					first experiments. They concern the quorum sensing module of
					our system. After fitting our deterministic model with the first
					results, we aim to investigate cross-talk between different AHLs
					and their diffusion.</P></TD></TR></TBODY></TABLE><DIV id="media"><P><A id="link" href="https://twitter.com/ETH_iGEM">@ETH_iGEM</A><A id="link" href="https://www.facebook.com/iGEM.ETH.Zurich">iGEM ETH Zürich</A><A id="link">igem2014@bsse.ethz.com</A></P></DIV></DIV><DIV id="goettingen"><H5>Newsletter n°2</H5><H6>from August 11th to August 24th</H6><H4>Goettingen</H4><TABLE id="table"><TBODY><TR><TD id="case1"><P><B>PROJECT UPGRADE</B></P></TD><TD id="case2"><P id="text"><B>We selected different surface proteins from Candida albicans,
					Aspergillus fumigatus, Aspergillus nidulans, Candida glabrata,
					Toxoplasma gondii and Saccharomyces cerevisiae and amplified
					them via PCR, avoiding glycosilated regions.</B></P><P id="text">So far, we have 18 amplified genes of interest, six of which are
					cloned in yeast and the rest is still in the sequencing stage. After
					the first round of the yeast two-hybrid assay, we have found some
					peptides that potentially bind the six genes cloned in yeast, which
					will be further tested for confirmation. We have also constructed a
					new peptide library.We presented these results at the symposium
					of the Goettingen Center for Molecular Biosciences and received
					feedback from the researchers of the institute.Now we are
					organizing the SynBio Day in our city on the 15th of August,
					following what was discussed at the team meetup held in Munich:
					we will have a boot in the city center of Goettingen where we will
					talk to pedestrians and answer the questions they may have about
					synthetic biology. We will also have some activities where people
					can win some prices from one of our sponsors.</P><P id="text">Many biochemical and biological events can take place when two
					molecules interact with each other. Our screening method is,
					generally speaking, a way to find new molecules with potential
					biological applications. This pipeline should allow other researchers
					to select novel biological parts that interact with their protein(s) of
					interest, and which can be further refined to increase specificity
					and modified to add functionalities.In our project, the selected
					peptides will be attached to a fluorescent tag and submitted to
					the biological part registry. However, the modifications are not
					restricted to a fluorescent tag: radioactively labeled compounds,
					attachment of different functional groups and immunoactive tags,
					among others, are also options that can be explored.</P></TD></TR></TBODY></TABLE><DIV id="media"><P><A id="link" href="https://twitter.com/iGEMGoe">@iGEMGoe</A><A id="link" href="https://www.facebook.com/pages/iGEM-Team-Goettingen/687557807970778">iGEM Team Goettingen</A><A id="link">igem2014@uni-goettingen.de</A></P></DIV></DIV><DIV id="lika"><H5>Newsletter n°2</H5><H6>from August 11th to August 24th</H6><H4>LIKA-CESAR-BRASIL</H4><TABLE id="table"><TBODY><TR><TD id="case1"><P><B>OUR TEAM</B></P></TD><TD id="case2"><P id="text"><B>For the first time on the iGEM competition, the LIKA-CESAR-BRASIL
					Team is a mix of students from different areas of knowledge.</B></P><P id="text">The
					formation of the team started with the willingness to innovate within
					the biology field of students of Laboratory of Immunopathology Keizo
					Asami (LIKA) and the Center for Advanced Studies and Systems of
					Recife (CESAR), which includes the participation of members from
					both institutes.</P></TD></TR><TR><TD id="case1"><P><B>OUR PROJECT</B></P></TD><TD id="case2"><P id="text"><B>The association LIKA-CESAR-BRASIL Team will show the
				competition an innovative project of biosensors for early
				detection of breast cancer.</B></P><P id="text">According to the World Health Organization, breast cancer is the
				most frequent type of cancer among the women worldwide, both in
				developing as well as developed countries. Most deaths from this
				disease occur in low and middle income, because the patients are
				diagnosed in advanced stages of the disease. Besides the delay
				in detecting the tumor, often the resolution on the most effective
				treatment is compromised by lack of technical and accurate
				diagnostic system.
				Currently, the diagnosis occurs using image techniques such
				mammography and magnetic resonance, which is very expensive
				and not portable. One way out of this problem would be the use of
				biosensors. This technology is capable of diagnosing infectious
				diseases, genetic, metabolic and others. A biosensor is a device that
				incorporates a biological element with a physical transudor to generate
				a measurable signal proportional to the analyte concentration.
				Joining LIKA’s scientific experience with C.E.S.A.R.’s technological
				skills, a the multidisciplinary team is developing a biosensor for the
				detection of breast cancer with the help of robotics. The idea is to build
				a system, which is able to process, and prepare small blood samples
				in an automated mode through help of bacteria. This system will be
				coupled to biosensors that are capable of detecting breast cancer
				in its early stages. The goal is to shorten the analysis of samples
				from patients that currently take 4-6 hours to less than 30 minutes.
				The automation of the entire process, together with the creation of a
				specific biosensor for breast cancer, will result in a future decrease in
				deaths from the disease, due to a more rapid and effective treatment.</P></TD></TR></TBODY></TABLE><DIV id="media"><P><A id="link" href="https://twitter.com/lika_cesar">@lika_cesar</A><A id="link" href="https://www.facebook.com/LIKACESARBRASIL">IGEM 2014 LIKA CESAR BRASIL</A><A id="link">likacesarbrasil@gmail.com</A></P></DIV></DIV><DIV id="Paris-Bettencourt"><DIV><H5>Newsletter n°2</H5><H6>from August 11th to August 24th</H6><H4>PARIS BETTENCOURT</H4><TABLE id="table"><TBODY><TR><TD id="case1"><P><B>PROJECT UPGRADE</B></P><P id="text">Plan for future steps:</P><P id="text">This is a small scale experiment,
					but it seems to lead us somewhere.
					As part of our Citizen Science
					project, we would like to project a
					horror movie in a local cinema and
					get at least 50 samples.</P></TD><TD id="case2"><P id="text"><B>Movies project – How our mood changes of body odor
				The aim of the project was to determine if body odor changes
				with our mood. We also wanted to determine if a 5 minute clip
				was enough to make a change in the smell or if it was necessary
				to watch a whole movie.</B></P><P id="text">We analyzed the odor of some volunteers before and after watching
				a clip and a movie that showed a specific emotion –fear, love,
				sadness, happiness. It was the first time they ever watched the
				movie.</P><P id="text">Before the clip, we showed a white screen during 30 seconds.
				Volunteers were asked to relax and empty their minds. After the
				clip, we showed a nature documentary (as it is supposed to set
				us in a neutral mood state). Before the movie, a white screen was
				shown again during 30 seconds.</P><P id="text">Other volunteers smelled the samples and answered:
				How pleasant is the smell? (0 the least – 5 the most)
				How strong is the smell? (0 the least – 5 the most)</P><P id="text">Results:
				Happy movie : Two people were asked to watch ‘Liar, Liar’. Results
				are not very conclusive. The volunteers told us the movie was not
				specially happy or funny.</P><P id="text">Romantic movie : One person was asked to watch ‘ Love actually’.
				After the movie, the smell is perceived as less strong and less
				pleasant.</P><P id="text">Sad movie : One volunteer watched ‘My sister’s keeper’. The smell
				after the movie is perceived as stronger and less pleasant.</P><P id="text">Horror movie : This is the smell of one volunteer, tested by three
				different people. After the movie ‘The silence of the lambs’, they
				all rated the body odor as less pleasant and stronger.</P><P id="text">We repeated the experiment with another volunteer, and asked
				two people to rate the smell. We had the same results. One person
				rated the sample over the scale (6) for the question ‘How strong is
				the smell?’ .</P></TD></TR></TBODY></TABLE><DIV id="media"><P><A id="link" href="https://twitter.com/iGEM_Paris">@iGEM_Paris</A><A id="link" href="https://www.facebook.com/ParisiGEM2013">iGEM Paris Bettencourt</A><A id="link">2014.igem.Parisbettencourt@gmail.com</A></P></DIV></DIV><DIV id="Valencia"><DIV><H5>Newsletter n°2</H5><H6>from August 11th to August 24th</H6><H4>Polytechnic University of Valencia</H4><TABLE id="table"><TBODY><TR><TD id="case1"><P><B>PROJECT UPGRADE</B></P></TD><TD id="case2"><P id="text"><B>The last two weeks have been very motivational for all of us.
				Our first provisional constructions for pheromones biosynthesis
				have been prepared and already tested in plant, our first partial
				results will come soon! In the meantime, multigenic and more
				complex constructions are being developed for future trials.</B></P><P id="text">We have started with the construction of a translator from
				GoldenBraid (the assembly system we are using) to Biobricks,
				so future iGEM teams can use our modules in Biobricks format.
				A specific promoter for glandular trichomes is a few steps away
				from being tested in plants and depending on the results, new
				approaches will be used. The design of our cooper-activated
				genetic switch is complete and we are waiting for the parts to
				arrive so we can start building it.</P><P id="text">We are also having some difficulties amplifying a gene we want
				to introduce into our system to produce an extra pheromone.
				Hopefully we will solve these problems soon so we can enrich the
				#SexyPlant with even more features. Please summon the demons
				of the failed PCRs and make them stop annoying us.</P><P id="text">We started a collaboration with NRP-UEA-Norwich team to create
				a common biosafety module, since both of our teams are working
				in N benthamiana. Our biosafety module will have a male sterility
				part, to prevent spreading of the genetic material, and identity
				preservation part which will confer the plant a different color,
				so it is easily distinguishable. This biosafety module will be fully
				compatible with Biobricks, so future teams working in plants can
				incorporate it in their systems.</P><P id="text">Plan for future steps:
				We will test new, more complex multigenic constructions in N
				benthamiana. Glandular trichome specific promoters will also be
				tested with GFP in N benthamiana. Parts from NRP-UEA-Norwich
				team and our own parts need to be adapted to each other in order
				to build the biosafety module.
				We hope the last gene of our set of pathways is finally amplified
				before the next newsletter is finished and that the translator is
				finally built. The sooner the better.
				Stay tuned for our updates!</P></TD></TR></TBODY></TABLE><DIV id="media"><P><A id="link" href="https://twitter.com/ValenciaUPViGEM">@ValenciaUPViGEM</A><A id="link" href="https://es-es.facebook.com/ValenciaUPViGEM2014">Valencia UPV iGEM 2014</A><A id="link">valencia.upv.igem@gmail.com</A></P></DIV></DIV><DIV id="Tongji"><DIV><H5>Newsletter n°2</H5><H6>from August 11th to August 24th</H6><H4>Tongji</H4><TABLE id="table"><TBODY><TR><TD id="case1"><P><B>OUR TEAM</B></P></TD><TD id="case2"><P id="text">Our team consists of students from biotechnology major and bioinformation
					major. The group formed when we got the information about the 2014 iGEM
					competition in winter vacation. We have close relationship with other universities
					in Shanghai, which offers us many chances to ask for help and assist other
					universities.</P></TD></TR><TR><TD id="case1"><P><B>OUR PROJECT</B></P></TD><TD id="case2"><P id="text"><B>I can talk about what has happened in the last two weeks.</B></P><P id="text">We did PCR with C. stercorarium as template to get gene of XynB.
				Then we connect it to T-vector.
				So we can build some T-vector which contained our target gene. We
				were using blue-white selection method for our experiment. The
				experiment went well and we have cultivated some E.coli which
				contained our target gene. We are going to do plasmid extraction
				and splice our target gene from T-vector to the expression vector.
				We also get some expression vector from another lab in our
				college. However, the amount was not enough for our project. So
				we did some common experiments to make it enough.
				Our project is about using an enzyme to biobleach paper pulp.</P><P id="text">Plan for future steps:
				We need to build an expression vector to verify our expectation
				and we will build vectors which is based on pSB1C3 for iGEM
				submitting.
				We want to set up a line chart about the bleaching effect of enzyme
				and the concentration of the enzyme. This task will be done by
				GAO Jian.
				We also want to call some paper factories and take a visit. In that
				case, We can find out what bleaching method is really being used
				in manufactory. And we will study the cost-effectiveness of our
				method.</P></TD></TR></TBODY></TABLE><DIV id="media"><P><A id="link">1253225@tongji.edu.cn</A></P></DIV></DIV><DIV id="Purdue"><DIV><H5>Newsletter n°2</H5><H6>from August 11th to August 24th</H6><H4>Purdue</H4><TABLE id="table"><TBODY><TR><TD id="case1"><P><B>OUR TEAM</B></P></TD><TD id="case2"><P id="text">2006 was the first year that Purdue participated in iGEM, entering the
					competition with seven undergraduate members and three advisers. Purdue
					continued to compete for the next four years, earning bronze medals almost
					every year but never anything more. Due to lack of interest, financial support,
					and time, Purdue did not field a team in the 2011 iGEM competition. Fortunately,
					the team was revived with enough interest and support to not only field a team
					in 2012, but receive a gold medal and advance to the international competition
					at MIT for the first time in the team’s history.
					In 2013, the Biomakers were ranked regional finalists with their project: “Back
					to the Basics of Synthetic Biology”. They also received the award for Best New
					Natural BioBrick, and advanced to the international competition!</P></TD></TR><TR><TD id="case1"><P><B>OUR PROJECT</B></P></TD><TD id="case2"><P id="text">This year the Purdue iGEM Team is tackling the problem of global
				malnutrition with our new project: Minecrobe! More than 870
				million people are malnourished according to the United Nations
				World Food Program, and the World Health Organization states
				that iro deficiency is one of the most common and widespread
				nutritional disorders in the world. We plan to utilize synthetic
				biology to combat this problem by engineering Bacillus subtillus
				to increase plants’ ability to uptake iron from the soil, through
				production of plant phytosiderophores.</P><P id="text">As the summer comes to a close, we’ve finished our main plant
				control experiment, we’re working on optimizing our transformation
				protocol for bacillus, and we’re waiting for our synthesized DNA to
				come in. The goal of the plant control experiment was to find out
				how corn and rice grow at different levels of iron available in the
				soil. We had 48 corn and 48 rice plants grow in a range from 0%
				to 150% recommended iron content, and grew them for 4 weeks.
				We took pH readings, height measurements, and chlorophyll
				measurements at regular intervals to gauge how the plants reacted
				to the varying levels of iron. At the end of the experiment we took
				leaf samples from each plant, and are planning on using ICP-MS
				to quantitatively find out how much iron the plants had absorbed.
				We’ll be able to use this data when we compare the plants grown
				with our modified bacillus in the future.</P><P id="text">Plan for future steps:
				In the coming weeks, are going to finalize our bacillus
				transformation protocol and then transform our constructs
				into the three strains we have in our lab. We then plan to do a
				siderophore assay to confirm that our constructs are working as
				expected, and then choose our final strain based on the quantity
				of phytosiderophores being produced. Once we have chosen our
				strain, we are going to start the final plant experiment to see how
				our modified Bacillus affects plant growth.</P></TD></TR></TBODY></TABLE><DIV id="media"><P><A id="link" href="https://twitter.com/PurdueBiomakers">@PurdueBiomakers</A><A id="link" href="https://www.facebook.com/PurdueBiomakers">PurdueBiomakers</A><A id="link">PurdueBiomakers@gmail.com</A></P></DIV></DIV><DIV id="StanfordBrownSpelman"><DIV><H5>Newsletter n°2</H5><H6>from August 11th to August 24th</H6><H4>StanfordBrownSpelman</H4><TABLE id="table"><TBODY><TR><TD id="case1"><P><B>OUR TEAM</B></P></TD><TD id="case2"><P id="text"><B>Our team has participated in the last four iGEM competitions.</B></P></TD></TR><TR><TD id="case1"><P><B>OUR PROJECT</B></P><P>With the exponential growth of interest
				in unmanned aerial vehicles (UAVs)
				and their vast array of applications in
				both space exploration and terrestrial
				uses such as the delivery of medicine
				and monitoring the environment, the
				2014 Stanford-Brown-Spelman iGEM
				team is pioneering the development of
				a fully biological UAV for scientific and
				humanitarian missions.</P></TD><TD id="case2"><P id="text"><B>We are working to engineer cells to synthesize cellulose
				acetate as a novel bioplastic, characterize biological methods of
				waterproofing the material, and program this material’s systemic
				biodegradation. In addition, we aim to use an “amberless”
				system to prevent horizontal gene transfer from live cells on the
				material to microorganisms in the flight environment.</B></P><P id="text">So far, we have: successfully transformed Gluconacetobacter
				hansenii, a cellulose-producing bacterium, with a series of
				promoters to test transformation efficiency before adding the
				acetylation genes; isolated novel waterproofing protein bands
				present in wasp nest material; transformed the cellulosedegrading
				genes into Escherichia coli; and we have confirmed that
				the amberless construct prevents protein expression in wild-type
				cells. In addition, as part of our human outreach project, we have
				been in touch with leaders in the fields of UAVs, synthetic biology,
				and earth sciences, and it is clear that biodegradable UAVs could
				have a significant impact on the industry.</P><P id="text">Plan for future steps:
				For the production of cellulose acetate, we plan to use directed
				evolution to select for those organisms which produce the polymer
				with highest acetate content.</P><P id="text">For our wasp protein waterproofing project, we plan to transform
				yeast with the genes for the cellulose waterproofing proteins and
				run a high throughput assay to isolate the colonies that produce
				the most water-resistant products.</P><P id="text">Finally, for our biodegradability project, we plan to link the
				production of our esterases and cellulases with a quorum sensing
				time-delay mechanism.</P></TD></TR></TBODY></TABLE><DIV id="media"><P><A id="link" href="https://twitter.com/SBSiGEM2014">@SBSiGEM2014</A><A id="link" href="https://www.facebook.com/pages/Stanford-Brown-iGEM/166210643442328">Stanford-Brown-Spelman</A><A id="link">sbsigem2014@googlegroups.com</A></P></DIV></DIV><DIV id="Sheffield"><DIV><H5>Newsletter n°2</H5><H6>from August 11th to August 24th</H6><H4>Sheffield</H4><TABLE id="table"><TBODY><TR><TD id="case1"><P><B>OUR TEAM</B></P></TD><TD id="case2"><P id="text">We are an interdisciplinary team of eight undergraduates, consisting of
					five engineers and three biologists hailing from four different countries.
					We were assembled earlier in the year by five PhD student advisors and
					having decided it was critical to tackle a real world problem, we asked
					the general public what sort of problems they faced on a regular basis.
					Many replies were made by people stating that they often had problems
					with pipe blockages, and from there our project began.</P><P id="text">We are the third team from Sheffield to enter the iGEM competition, and
					the first since 2010. We are aiming for victory in the manufacturing track!</P></TD></TR><TR><TD id="case1"><P><B>OUR PROJECT</B></P></TD><TD id="case2"><P id="text"><B>The accumulation of fat and hair deposits in sewer systems is
				increasingly problematic in urban areas, causing disruption to
				the public and costing water companies a lot of money.</B></P><P id="text">FOG (fats, oils and greases) pollutants mainly originate from
				homes and restaurants, who dispose of used cooking oil down the
				sink rather than in the bin. Our idea is to engineer E. coli to secrete
				lipase and keratinase enzymes to break down these deposits
				before the blockages have the chance to form.</P><P id="text">Plan for future steps:
				We are working on methods of inserting the bacteria into the pipe
				in a way that is both cost-effective and practical. We also intend to
				research where the responsibility for dealing with damage from
				environmental pollutants lies.</P></TD></TR></TBODY></TABLE><DIV id="media"><P><A id="link" href="https://twitter.com/iGEMSheffield">@igemsheffield</A><A id="link" href="https://www.facebook.com/iGEMSheffield2014">iGEMSheffield2014</A><A id="link">igem2014@sheffield.ac.uk</A></P></DIV></DIV><DIV id="SJTU-BioX-Shanghai"><DIV><H5>Newsletter n°2</H5><H6>from August 11th to August 24th</H6><H4>SJTU-BioX-Shanghai</H4><TABLE id="table"><TBODY><TR><TD id="case1"><P><B>OUR TEAM</B></P></TD><TD id="case2"><P id="text"><B>The iGEM team of SJTU is supported by Bio-X Institutes of Shanghai
					Jiao Tong University, founded in 2009. We were the Grand Prize
					Winner of Asia in 2012, and won Best New BioBrick Devices and
					gold medals in the past five years.</B></P><P id="text">Our team consists of undergraduates with biology background.
					Make science and technology serve production more effectively
					is our constent pursuit. This year, our team focus on protein
					multimerization system in cytomembrane and we hope MembRing
					can bring new ideas and greater efficiency to the relevant industrial
					practices.</P></TD></TR><TR><TD id="case1"><P><B>OUR PROJECT</B></P><P id="text">Use recognition of DNA-binding
				proteins and specific nucleic acid
				sequences to achieve proteins
				multimerization</P><P id="text">Remold connecters(plasmids) to
				achieve selective combination of
				different enzymes</P><P id="text">Explore the maximum number of
				enzymes that can be connect on a
				single connecter.</P></TD><TD id="case2"><P id="text"><B>This year, we bring MembRing – an annular protein
				multimerization system in cytomembrane.</B></P><P id="text">This system will help us achieve the goal of selective polymerization
				of enzymes. Polymerized enzymes, different from normal
				scattered condition, has much more opportunities to contact with
				the substrates. Therefore, our project aims to increase as well as
				control the efficiency of complex reactions.</P><P id="text">The basic function, polymerization, is carried out by a circular
				DNA and DNA-binding protein to polymerize enzymes. First, a
				fluorescent protein, a transmembrane domain, an enzyme and a
				DNA binding protein are connected in order, forming a complex
				that the FP locates outside the film while the enzyme and DNA
				binding protein are in cytoplasm. Then integrate specific matching
				DNA sequences onto an exogenous plasmid and co-transform
				this connecter with expression vectors into E.Coli to associate
				enzymes.</P><P id="text">In addition, we plan to make the enzymes selectable by remolding
				the connecter with different recognition sequences and cotransforming
				it with expression vectors. This will help the whole
				project to work in all kinds of practical applications.
				During this year, our team has already had many opportunities
				to interact with other teams. We find those communications
				inspiring to both sides. Thus, we welcome any kinds of comments
				or questions. Let’s work together to ake this year wonderful!</P></TD></TR></TBODY></TABLE><DIV id="media"><P><A id="link">sjtu.igem2014@gmail.com</A></P></DIV></DIV><DIV id="Technion-Israel"><DIV><H5>Newsletter n°2</H5><H6>from August 11th to August 24th</H6><H4>Technion Israel</H4><TABLE id="table"><TBODY><TR><TD id="case1"><P><B>OUR TEAM</B></P></TD><TD id="case2"><P id="text">11 engineering students and one design student form the 2014 iGEM
					team. This is only the second time the Technion is participating
					in iGEM, and the third year an Israeli team is taking part in the
					competition.
					We are passionate not only about science but also about educating
					children and youth in Israel from all walks of life.</P></TD></TR><TR><TD id="case1"><P><B>OUR PROJECT</B></P><P>With the exponential growth of interest
				in unmanned aerial vehicles (UAVs)
				and their vast array of applications in
				both space exploration and terrestrial
				uses such as the delivery of medicine
				and monitoring the environment, the
				2014 Stanford-Brown-Spelman iGEM
				team is pioneering the development of
				a fully biological UAV for scientific and
				humanitarian missions.</P></TD><TD id="case2"><P id="text">Detection of allergens and toxins in food and water is a very
				important issue, which is difficult to solve, mainly because the
				detection must be done at very low concentrations. Our team is
				paving the path to a cheap and accessible solution to this issue
				by planning a bio-sensor system that can accurately identify a
				substance of interest at extremely low concentrations.
				We are programming E.Coli bacteria to sense the presence of a
				specific substance and process the signal to produce a green light
				as well as signal to neighboring bacteria to follow suit, causing a
				chain reaction. This makes the green light visible to the naked eye,
				alerting the consumer of the presence of the substance.
				Along the way, we plan to build (and make available to future iGEM
				teams), many useful sub-systems which will include: a Histidine
				Kinase two component signaling system (which will detect new
				materials), 6 different double-repression toggle switches (along
				with the proper characterization), a chemically synthesized
				photo-switching molecule (activated by the green light produced
				by the bacteria themselves) which can connect cells, creating a
				synthetic biofilm, and a method of constructing genetic sequences
				with Gibson addresses which will make the high-throughput
				construction of complex biobricks possible.</P></TD></TR></TBODY></TABLE><DIV id="media"><P><A id="link" href="https://www.facebook.com/iGEM2014.Technion?fref=ts">Technion iGEM team 2014</A><A id="link">igem14il@gmail.com</A></P></DIV></DIV><DIV id="UCSF_&amp;_UCB"><DIV><H5>Newsletter n°2</H5><H6>from August 11th to August 24th</H6><H4>UCSF &amp; UCB</H4><TABLE id="table"><TBODY><TR><TD id="case1"><P><B>OUR TEAM</B></P></TD><TD id="case2"><P id="text">The UCSF iGEM program was created in 2007, with the belief that
					students regardless of age could participate in synthetic biology.
					This year, UCSF and UCB have combined forces to produce a team
					composed of five graduates from San Francisco’s Abraham Lincoln
					High School’s Biotechnology Program, two returning iGEM alumni,
					two undergraduates from UC Berkeley, and one exchange student
					from Peking University. While we all have different backgrounds
					and knowledge, we are united in our desire to learn, contribute,
					and succeed.</P></TD></TR><TR><TD id="case1"><P><B>OUR PROJECT</B></P></TD><TD id="case2"><P id="text">Project update (It can be a subpart of your project), what has
				happened, worked or failed in the past two weeks and why:
				For teams that were not in the first issue, please try to give a really
				really brief summary of your project.
				Summary:
				In nature, many organisms must make multicellular decisions,
				whether the cells are part of a community of unicellular organisms
				or an organ in a multicellular organism. In some cases, the
				decision reaches a consensus and all cells respond similarly, such
				as in quorum sensing by some species of bacteria. Decisions may
				also favor a diversified response, in which some cells respond
				strongly to a signal while others are downregulated or cease to
				respond. For instance, in the human immune system, even though
				all T cells initially react to an antigen, only those with a strong
				response proliferate and produce an immune response. Many of
				these decisions are facilitated by a signaling factor, which the
				cells sense and produce more of.</P><P id="text">Our goal is to understand how a group of cells uses sense and
				secrete communication to make a community decision. Therefore,
				we have chosen to pursue the Information Processing track.</P><P id="text">We are designing multicellular systems in which different yeast
				strains communicate with each other to form a community
				decision. By engineering systems in which strains with variable
				initial responses develop the same response, or strains with the
				same initial response diverge to develop a bimodal response, we
				can model interesting communication systems, such as those of
				the immune system, leading to more novel insights about how
				cells interact to produce a specific response.</P></TD></TR></TBODY></TABLE><DIV id="media"><P><A id="link" href="https://twitter.com/iGEM_UCSF">@ iGEM_UCSF</A><A id="link" href="https://www.facebook.com/pages/UCSF-IGEM/290769630936213">UCSF iGEM</A><A id="link">jchsueh@berkeley.edu</A></P></DIV></DIV><DIV id="USTC_China"><DIV><H5>Newsletter n°2</H5><H6>from August 11th to August 24th</H6><H4>USTC_China iGEM team</H4><TABLE id="table"><TBODY><TR><TD id="case1"><P><B>PROJECT UPDATE</B></P></TD><TD id="case2"><P id="text">In this week’s Newletter, we are going to use more graphs to
					illustrate our ideas and process. If you have any questions or
					advice, please just email us :)</P><P id="text">This week, we have been invited to the iGEM Conference in Taiwan,
					where we shared our work and results with many teams in Asia.
					That’s a great trip and we learned a lot from this meetup!</P><P id="text">This is the poster we used in the meetup which provides your more
					details about our project compared to last Newsletter.</P><P id="text">Currently, we are trying to figure out the optimum conditions of C.
					Crescentus development, transformation and expression, which
					will tell us how to develop the holdfast in C.Crescentus within
					control. So, we can extract the gene regulating the expression
					of holdfast in C. Crescentus and build parts with the regulating
					system.</P><P id="text">In the future, based on the current results, we will construct the
					light sensing-response system. And furthermore, to guide the
					accomplished system into E. coli and C. Crescentus as well as to
					test the function and efficiency of ribozymes.</P><P id="text">You may be curious what this little project can lead us to. Here is
					our answer:</P><P id="text">1. Plan the industrialization and further development in biological
					imaging industry
					2. Design the projector specialized for biological imaging</P><P id="text">Do you have more excellent ideas about this topic? Welcome to
					discuss with us!</P></TD></TR></TBODY></TABLE><DIV id="media"><P><A id="link">stark@mail.ustc.edu.cn</A></P></DIV></DIV><DIV id="XMU-China"><DIV><H5>Newsletter n°2</H5><H6>from August 11th to August 24th</H6><H4>XMU-China</H4><TABLE id="table"><TBODY><TR><TD id="case1"><P><B>OUR TEAM</B></P></TD><TD id="case2"><P id="text">Regardless of different personal traits and dispositions, we gathered here, in
					Xiamen University, striving for a common goal; In spite of various setbacks
					and difficulties, we united as a team, supporting each other and persevering in
					iGEM work.
					The 2014 iGEM team of Xiamen University is a great one with over 20 members
					at the present. We each, from a broad range of background, are dedicated to
					explore the way how to utilize the synthetic biology to promote social progress
					like seeking to address the environmental problems with a wiser biological
					method and improve the public understanding toward such a subject. The
					constitution of our team is sophomoric and junior college undergraduates,
					among which former one holds the majority. As a youthful and energetic group,
					we XMU iGEMers are clinging to various innovative attempts and expecting to
					develop long-term friendships with other teams in different area sincerefully.</P></TD></TR><TR><TD id="case1"><P><B>OUR PROJECT</B></P></TD><TD id="case2"><P id="text">This year, chemotaxis interests us a lot. We intend to reprogram
				the strain (CL-1) which lacks the CheZ gene to form mathematical
				patterns. CheZ gene belongs to chemotaxis family of E.coli, and
				protein CheZ can dephosphorylate CheY-P which allows E.coli to
				make smooth swimming. The ΔCheZ bacteria can’t dephosphorylate
				CheY-P, therefore the strain isn’t motile until CheZ is involved in.
				Based on this, the motility of E.coli can be precisely controlled by
				stimulus (such as IPTG). We try to construct logic gene circuits to
				make E.coli recognize environmental stimulation, thus by utilizing
				the controlled chemotaxis which named pseudotaxis we could
				command bacteria to form patterns such as ellipse, hyperbola
				and so on.
				The intrinsic motivation that drive us striving for the above project
				is that we want to simulate the process of stem cell differentiation.
				In the organs development, stem cells differentiate while aggregate
				together to form heart, liver and kidney what have precise shapes.
				We think there must be some mathematical principles that govern
				the differentiation process. By simulating differentiation, we
				want to get a closer understanding of the differentiation process.
				However, E.coli can’t sense as much stimulus as stem cell, we
				intend to utilize RNA aptamers what has the potential to response
				to almost all stimulus to cover that shortage. Thus, we can get a
				more precise stimulation.
				We also try to utilize pseudotaxis to accomplish some experimental
				meaning. As far as we know, the motile ability is proportional to
				the amount of protein CheZ in certain range. By measuring the
				average chemotaxis distance, we can get a precise evaluation on
				the RBS efficiency which is evaluated by fluorescence strength
				previously.</P></TD></TR></TBODY></TABLE><DIV id="media"><P><A id="link" href="https://twitter.com/XMUiGEM2014">@XMUiGEM2014</A><A id="link">xmuigem2014@gmail.com</A></P></DIV></DIV><DIV id="question"><DIV><H5>Newsletter n°2</H5><H6>from August 11th to August 24th</H6></DIV><TABLE id="table"><TBODY><TR><TD id="case2"><P id="text"><B>TEAM'S QUESTIONS</B></P></TD></TR><TR><TD id="case1"><P><B>From Goettingen</B></P></TD><TD id="case2"><P>To Valencia : Which insect pheromones are you using?
					To Paris Bettencourt : No doubt there are persons who may feel insecure
					because of their personal odor and who will benefit from your project, but
					it seems that you are categorizing a single and only kind of “malodor”.
					It’s interesting to consider that the attitude towards body odor is a
					culture-related trait. How can the concepts of “strong personal odor”
					and “malodor” be defined in a multicultural context without any cultural
					bias? Wouldn’t it be more politically correct to leave the users decide if
					they have “malodor”?</P></TD></TR><TR><TD id="case1"><P><B>From Paris Bettencourt</B></P></TD><TD id="case2"><P>Would any other iGEM teams like to repeat our experiment? They would
					project sad, happy and romantic movies in a controlled environment, and
					analyze the sweat samples by smelling them and rating the strength
					and pleasantness before and after.</P></TD></TR><TR><TD id="case1"><P><B>From Valencia</B></P></TD><TD id="case2"><P>How would you quantify the secretion of volatile compounds into the air?
					Would you use an alternative genetic switch to the cooper-activated one
					we’re using? Do you think our biosafety module is safe enough? If not,
					how would you improve it?</P></TD></TR><TR><TD id="case1"><P><B>From NJU-QIBEBT</B></P></TD><TD id="case2"><P>How to manage time for experiments.
					How to arrange different tasks for each team member.</P></TD></TR><TR><TD id="case1"><P><B>From Purdue</B></P></TD><TD id="case2"><P>Would you feel safe eating corn that was grown in soil with modified
					microbes in it, even if there was no threat to your health?</P></TD></TR><TR><TD id="case1"><P><B>From StanfordBrownSpelman</B></P></TD><TD id="case2"><P>How would you use a biological UAV?</P></TD></TR><TR><TD id="case1"><P><B>From Sheffield</B></P></TD><TD id="case2"><P>Who should be held responsible for repairing the damage caused by
					environmental pollutants (such as fats, oils and greases)?</P></TD></TR><TR><TD id="case1"><P><B>From SJTU</B></P></TD><TD id="case2"><P>How to build a fusion protein effectively?
					If your projects consider the use of polymerase system?</P></TD></TR><TR><TD id="case1"><P><B>From Technilon Israel</B></P></TD><TD id="case2"><P>What would you like our system to detect? What materials interest you?
					How do you envision a home kit for material detection? What things
					are important for you in such a product? Would you use a product with
					bacteria at home? How but outside your home?
					What kind of safety measures would you like the product to have?</P></TD></TR><TR><TD id="case1"><P><B>From UCSF UCSB</B></P></TD><TD id="case2"><P>How can simple cell signaling be used to model complex signaling
					systems on a larger level? How can we differentiate between different
					cell strains when they are intermixed? What are some ways to get more
					people interested in synthetic biology?</P></TD></TR><TR><TD id="case1"><P><B>From XMU-China</B></P></TD><TD id="case2"><P>We want to know that whether there is any easy methods to dye bacteria.
					Why some parts of RBS can’t be transformed ? Could amino acid be
					sterilized at above 120 degrees Celsius zero ? We want to form a complex
					mathematic pattern through the chemotaxis of E.coli,dose it make
					sense? In the absence of external forces, is that ture self-assembly
					(self-organized) gene circuits can only form a circular pattern.</P></TD></TR></TBODY></TABLE><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Paris_Bettencourt/Newsletter2">http://2014.igem.org/Team:Paris_Bettencourt/Newsletter2</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Paris_Bettencourt/Newsletter2" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Paris_Bettencourt/Newsletter2" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Paris_Bettencourt/Newsletter2&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Paris_Bettencourt/Newsletter2&amp;oldid=118681" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>