"
Page
Discussion
View source
History
teams
Log in
 
Team:NCTU Formosa/Project-sub2
From 2012.igem.org
Team:NCTU Formosa - 2012.igem.org
iGEM 2012
Official Team Profile
NCTU_Formosa
Home
Project
Introduction
Project Details
Conclusion
Optimization
Future Works
Modeling
Team
Members & Advisors
Host Lab
Gallery
Notes
Parts Submitted
Safety
Human Practice
Bio-camp
iGEM Cooperation
Popularization on Campus
Popular Science Creation
Surveys
L-shaped Folders
Gallery
Back to Top
Introduction
Introduction
Enzyme for Isobutanol
Enzyme for Isobutanol
Temperature Control System
Temperature Control System
Zinc Finger
Zinc Finger
Instrument
Instrument
Conclusion
Conclusion
Optimization
Optimization
Future Works
Future Works
Project details
Enzyme for isobutanol
According to the previous study, we use four enzymes to catalyze pyruvate to produce isobutanol. The genes are cloned from different bacteria and encode four enzymes─ AlsS, ilvC, ilvD, KivD.Figure 1 shows the overall pathway. As glucose can be catalyzed into pyruvate by glycolysis, we chose glucose as the starting point of our biosynthetic pathway. Then, pyruvate will be converted into isobutanol by the enzymes shown in Figure 2.
Figure 1.Isobutanol synthesis pathway.
Figure 2.We cloned four gene, ilvC, ilvD, AlsS, kivd. The figure above are the names, strains, length and point mutation of four genes.
Temperature control system
To allow E.coli to produce isobutanol efficiently,we introduced the low temperature releasing system (Figure 3.) in to our circuit (BBa_K887002) . The low temperature system could allow E.coli to produce the optimum production of isobutanol before being poisoned by isobutyaldehyde. The following picture is our system.
Figure 3.The idea of our low temperature release system.
First, we incubated E.coli in 37°C environment. After accumulating enough 2-ketoisovalerate , we move E.coli into 30°C environment. The accumulated non-toxic intermediate would be converted into the final product , isobutanol. Therefore, producing an efficient method to obtain the excellent biofuel.
Figure 4.Two circuits of our biobrick of temperature control system.
Figure 4 shows our biobrick. The most important gene of our biobrick is 37°C ribosome binding site gene. There are two circuits in our biobrick. The first circuit is the one encodes 37℃ ribosome binding site gene and the second circuit is the one that encodes kivD gene.
Now, let us introduce how our system works.
Figure 5.The system works in the 37℃ environment.
When being in 37°C environment, the first circuit will be translated and produce TetR protein to inhibit Ptet promoter. So, the second circuit will not be translated. Therefore, we can obtain the intermediate , 2-ketoisovalerate , at this step. (Figure 5.)
Figure 6.The system works in the 30℃ environment.
After having enough of 2-ketoisovalerate , we move E.coli into 30°C environment. This way, the ribosome will not bind the 37°C ribosome binding site and tetR genes will not be translated. Therefore, the second circuit will be translated successfully. In the end , we can get the isobutanol efficiently. (Figure 6.)
Result
Figure 7.Activate our E.coli overnight. Then, transfer it into the new medium with the microaerobic environment until OD600 reached 0.2. After that, measure the OD600 every 4 hours.
We did an experiment to prove the isobutanol is truly toxic to the E.coli. The data shows that the higher concentration of the isobutanol was in the medium, the lower OD600 value could be obtained. (Figure 7.)
We used the fluorescent protein to mark the second circuit of our biobrick. The data tells us that kivD enzyme under 37℃ environment had the lower expression than under 30℃ and the 25℃ environment.
According to Figure 8, our low temperature release system do truly work !
Figure 8.Mark the second circuit with the fluorescent protein to test the expression of kivD enzyme.
Zinc finger
Figure 9.This is the whole circuit in our project
Figure 9 shows the whole circuit of our project. We encoded four zinc fingers(show as blue Cylinder) in front of each enzyme(show as orange). Besides, we encoded DNA program in the second circuit.
Figure 10.The simple picture for zinc finger.
Zinc finger proteins contain a DNA binding domain and a functional domain. DNA binding domain could recognize specific DNA sequence, which called DNA program. Zinc fingers could tightly bind to specific DNA or RNA sequence. We replace the zinc fingers' functional domains with our enzymes to create fusion proteins. (Figure 10.) With the zinc finger's "hand", the enzyme could bind to the specific DNA program. By doing so, the enzymes would no longer disperse around the cell. Therefore the productivity of isobutanol will be higher.
Figure 11.The intermediate and enzyme in the pathway.
With this feature, we expected to build a production line to help us make isobutanol. We put the enzymes in order. (Figure 11.) When the intermediates are produced, it could have the next reaction as quickly as possible. The final product, isobutyraldehyde will be converted into isobutanol by ADH in E.coli.
Point mutation for avoiding frame shift
Figure 12.Point mutation for avoiding frame shift.
We found that there are only five nucleotides between HIVC and ilvD genes. (ATG are the first three nucleotides of the ilvD gene.) (Firgure 12.) According to the triplet nature of gene expression by codons, it would cause a frameshift mutation, which cause the condons code for incorrect amino acid.
In order to assemble a production line in E.coli, we have to transform DNA program as well as our fusion protein genes in to E.coli. The design of our biobricks has the same order of zinc finger as 2010 Slovenia iGEM team, so we decided to use their DNA program (BBa_K323066) instead of synthesizing one.
Point mutation for correcting DNA program
After cloning, we sent it to Genomics BioSci & Tech Co., Ltd. for sequencing. We found that there is a deletion of a base pair in the zif268 biding site. (Figure 13.)
Figure 13.Comparing with the sequence form the part wiki page, there is a missing base (pair) in the zif268 biding site, to be specific a deletion of 25 G. Image: NCBI BLAST.
So, we designed a complementary primer to insert the lost base pair. After PCR of point mutation, we sequence it again. Figure 14 shows the outcome.
Figure 14.There is a correct zif268 biding site in this DNA program. Image: NCBI BLAST.
We submit this corrected DNA program as BBa_K887011 . You can find out more information in its part wiki page.
Instrument
Figure 15.Our project is applied to the simple instrument. Besides, we also add a simple way to collect isobutanol.
(1)37°C
At the first step, we transformed the plasmid we designed into the E.coli ,and inoculated it in M9 medium- which containing 36 g/L glucose, 5 g/L yeast extract,100 μg/ml ampicillin, 30 μg/ml kanamycin, and 1,000th dilution of Trace Metal Mix A5 (2.86 g H3BO3, 1.81 g MnCl2 ⋅4H2O, 0.222 g ZnSO4 ⋅7H2O, 0.39 g Na2MoO4⋅2H2O, 0.079 g CuSO4⋅5H2O, 49.4 mg Co(NO3)2⋅6H2O per liter water)-into the first tank. Then, we culture the E.coli in 37°C  environment for three hours which means that we put the tank in the warm bath to let E.coli produce the intermediate,2-ketoisovalerate.
(2)30°C
Afterward, we put our E.coli into 30°C environment maintained by warm bath for 3 days incubation. Our low temperature control system would initiate expression of kivd which would convert 2-ketoisovalerate to isobutyraldehyde. Then, isobutyraldehyde would be converted into isobutanol by E.coli's own alcohol dehydrogenase(ADH).
(3) preliminary distillation
After incubating the “E.coline” in 30°C environment for three days, the concentration of isobutanol is high enough to be collected. We prepared two  flasks which contained half-filled cold water and each of them is equipped with a condenser. The three flasks were linked with pipes. One end of the pipe (air out) must be under the water level, so that the air would expose into water of the destined flask. We pumped air to strip the isobutanol to the flask for product collection. If isobutanol could be transferred from the fermentation flask, we expected the production rate could extend tremendously and the following condensate collector will obtain higher concentration of isobutanol than the previous fermentation flask. By having this higher concentrated isobutanol, isobutanol purification will be much more favorable to be conducted.
Figure 16.The actual image of the instrument.
Copyright © 2012 NCTU Formosa
Background image belongs to Yuri Kormin .
Website designed by Calvin Hue.
Retrieved from " http://2012.igem.org/Team:NCTU_Formosa/Project-sub2 "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
