"
Page
Discussion
View source
History
teams
Log in
 
Team:Bordeaux/Parts/BBa K1317002
From 2014.igem.org
Link to the registry of Standard Parts: http://parts.igem.org/Part:BBa_K1317002
Initial strategy :
We tried to assemble the gene coding for the SLPs from 8 oligonucleotids with homologous regions with the Gibson Assembly. The oligos were designed with the software DNA Works (Helix Systems)
First of all, consensus sequences for the spider silk were identified and our own protein was designed. Then, the nucleotide sequence using the peptide sequence was determined. We had to pay attention because our protein sequence is made of repeated motifs.
Table 1 : sequences of the 8 nucleotides
2 different methods were used with the Gibson Assembly1: in one step at 50°C or with cycles of denaturation at 95°C and annealing at 50°C (figure 1). The enzyme used was the Phusion® High Fidelity Polymerase.
Figure 1: Strategy of the Gibson Assembly to assemble the gene coding for the SLPs
Our 8 oligo were not assembled with these 2 methods, so another method was used : the PCR-Fusion.
This method is made of different steps using the Phusion® High Fidelity Polymerase (figure 2). In a first step fragments were joined two by two, then fragments 1-2 were joined to fragments 3-4 and a PCR is performed using fragments 1 and 4 as primers. The same method was used for fragments 5-6 and 7-8.
Finally, fragments 1-2-3-4 were assembled to fragments 5-6-7-8 and a PCR was also performed using the fragments 1 and 8 as primers.
Figure 2: PCR Fusion strategy to assemble the gene coding for the SLPs
This method was not successful because fragments 6 and 7 were unable to join. Therefore, new fragments were designed with a different homologous region. The fragment 8 that added only 2 nucleotides was suppressed and these 2  nucleotides were added on fragments 7.
Table 2: Sequence of the new fragments
Then, a new strategy was used (figure 3). The two first steps are common to the previous strategy but then, fragments 5-6 are joined to fragments 1-2-3-4 and the PCR is made with the fragments 1 and 6. Finally the fragment 7 is added and a PCR is also performed.
Fig 3 : Strategy to assemble the CDS for the SLPs using the new fragments 6 and 7
This method allowed the assembly of the 7 fragments (picture 4). A fragment of 318 bp was expected on the electrophoresis gel and this is what we got.
Figure 4 : Gel electrophoresis on 3% agarose
Reference :
[1] https://www.neb.com/tools-and-resources/feature-articles/gibson-assembly-building-a-synthetic-biology-toolset
[2] Shevchuk N.A., Bryksin A.V., Nusinovich Y.A., Cabello F.C., Sutherland M. et Ladisch S. Construction of long DNA molecules using long PCR-based fusion of several fragments simultaneously (2004) Nucleic Acids Res., 32(2), 19
[3] Michael B. Hinman, Justin A. Jones and Randolph V. Lewis. SYNTHETIC spider silk: a modular fiber. TIBTECH SEPTEMBER 2000 (Vol. 18) (PMID 10942961)
/p> /p> /p>
Retrieved from " http://2014.igem.org/Team:Bordeaux/Parts/BBa_K1317002 "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
