Team:Duke/Description
HOME
TEAM
PROJECT
Description
Experiments & Protocols
Results
Design
PARTS
Team Parts
Basic Parts
NOTEBOOK
ATTRIBUTIONS
COLLABORATIONS
HUMAN PRACTICES
SAFETY
MODELING
INTERLAB STUDY
SOFTWARE
Project Description
Overview
Our work over the summer fed into three main projects: a gene circuit to detect antibiotic resistance, a review of programmable cell deaths and the creation of a $100 thermocycler. The design of the antibiotic resistance detector was began but because of cloning problems was not finished. A review of programmable cell deaths produced a strong response from the Phi X 174 (K1835500), however Protegrin-I (K628000) could not be proven to inhibit cell growth from within the cell but further testing is required. Further, we recommend a variant on Protegrin, Cysteine-Deleted Protegrin (K1835502) which is still antimicrobial but does not have haemolytic activity. Our PCR machine, while still slow due to an underperforming voltage source, is ready for initial PCR efficiency testing.
Programmable Cell Death Screening
Programmable Cell Death
To complete the “if gate” function that our construct is designed to perform, there must be a “then” statement. One option for this would be a fluorescent protein reporter. That would simply tell us that the construct is detecting the specific sequence by fluorescing. However, we want it to actually do something when it detects the antibiotic resistance. With the goal of combating antibiotic resistance in mind, the most practical function for the “then” statement to perform is to initiate cell death.
Several different such programmable cell death genes were tested to compare how well they killed the bacteria so that the best candidate could be determined and implemented in the final construct. Six genes in total were examined, two of which already existed in the registry of standard biological parts, one of which another lab at Duke shared with us, and three of which we designed based on further research.  Each of these genes is discussed further below.
The original goal with these cell death parts was to adopt several cell death genes from the registry of standard biological parts and further characterize them by comparing their bacteriocidal capability. However, this proved to be more difficult than anticipated, as we had lot of trouble finding genes in the registry that would kill E. coli. The two that we have tested, BBa_K117000 and BBa_K628000, were the only two viable candidates that we came across. We wanted more parts to compare, so we did a little more investigation and were able to acquire or engineer the other four cell death genes that we attempted to test.
Programmable cell death genes are incredibly useful parts not just for our project, but also for many other applications, such as population control or selective markers. With that in mind, it is important for there to be plenty of data available to everyone on this class of gene so that teams can select a gene that caters more precisely to their specific interest. Thus, thoroughly characterizing the activity of the existing parts as well as the new parts became a main goal for us. In order to characterize the bacteriocidal capacity of each gene, we placed them downstream of a strong ribosome binding site and strong inducible lac promoter. To quantitatively measure the activity, we measured the optical density of cells using a spectrophotometer, comparing the  samples induced with isopropyl - D-1 thiogalactopyranoside (IPTG) at different levels vs. a control with no IPTG induction. At higher levels induction, the cell death genes are expressed more and killing the cells more, so the optical density should be lower. At a given level of induction, the greater the difference between the induced and uninduced samples, the more effective the gene is at killing cells. This was the basis of our comparison.
More information on the parts used are on the Basic Parts page.
Moving Forward
The E lysis gene was the only cell death part that convincingly showed that it killed bacteria to a significant degree. The other cell death parts did not show an appreciable difference between the induced and uninduced samples. These results prompted us to look further into the mechanism of action by each part. The fact that none of the antimicrobial peptides, both with and without cysteine residues, had any significant effect on the optical density of the cells makes more sense based on how exactly they cause cells to lyse. The peptide forms a complex with lipopolysaccharides on the outer membrane of the bacteria which causes the formation of pores in the membrane that kill the cell. However, the peptides are being made within the bacterium and thus have a very hard time accessing the lipopolysaccharides on the outside. With that in mind, producing just the antimicrobial peptides without an export mechanism would be expected to have very little antimicrobial activity.  E lysis protein’s success is reasonable, as well. The protein is derived from the genome of a bacteriophage, whose sole purpose is to infect bacteria, have them produce many new phages, and then make them lyse in order to release the reproduced phages. This lysis must be initiated from the inside of the cell in order for this whole mechanism to work. The fact that the protein inhibits the production of the cell wall works under this logic. The lack of results from K117000 are more mysterious. It could be that the lysis protein is less effective when it is not expressed in tandem with the other two components of the operon it comes from.
Going forward, we would like to actually incorporate our cell death genes into the overall construct. Due to its significant results, the E lysis protein is the best candidate for being included as the “then” statement.  There are also a couple of things that can be done in reference to the antimicrobial peptides.  Previous research has been done into the hemolytic and antibiotic abilities of cysteine-deleted tachyplesin, but there has only been investigation into cysteine-deleted protegrin’s antibiotic activity, and not its tendency to lyse human erythrocytes. Thus, one easy project that can be done in the future is to perform a hemolytic assay with CDP to see if, like CDT, its activity against human erythrocytes is abolished while its activity against bacteria is maintained. Additional research should also be done on mechanisms by which any or all of the antimicrobial peptides can be transported from the inside of the cell to the outside of the cell. One possible direction this could be taken is finding an ATP-binding cassette transporter that can export antimicrobial peptides. These are both possible future projects for our team and/or for another team who takes interest in this topic.
Thermocycler
Overview
The idea to design and build a thermocycler that could be built with parts found in and around the house was planted in our heads by our advisors. When an old PCR machine was salvaged from Duke Surplus, that idea became ambition. Was it even feasible for a team of biologists to create a machine that sold for upwards of $2000 for less than $200 and with parts that could be found for free or on Amazon?    A minimal amount of parts are required to make a PCR machine functional. Behind the heating and cooling of the PCR tube well (called “heating block”) is a Peltier chip. A Peltier chip’s function is quite simple – it heats up rapidly by stealing heat from the other side of the chip when current is sent through one way, and cools down rapidly when that current direction is reversed. The component responsible for controlling the direction of the current is an H-Bridge, which is very simply a parallel arrangement of four switches. Closing two switches causes the voltage from the DC power source to be defined one way while closing the other two switches causes the voltage to be switched. A thermistor or thermocouple is required to monitor the temperature of the heating block. Both allow you to convert a voltage into a temperature reading. The brains behind the machine is the Arduino board, which is responsible for processing the current temperature of the heating block and telling the H-Bridge which direction to send the current to get the heating block to the desired temperature.
Acknowledgements
Thank you to Ben’s dad, Patrick Hoover, for being able to find and acquire miscellaneous screws and capacitors and tools needed to assemble the prototypes.
Thank you to the University of Maryland iGEM team, with whom this project is closely tied. For further information on the extent of collaboration, read below.
Conceptualization
The original objective in designing a PCR machine was to design the kind of machine that anyone could look at the parts list for, buy whatever was needed online, and assemble it themselves according to a publicly designed blueprint. The goal, then, was akin to creating a DIY, college student rendition of OpenPCR’s open-sourced thermocycler design, which sells a 16 well thermal cycler for $650.
This task proved incredibly daunting, and the mission statement soon became proving that a functional thermal cycler could be built from parts found around the house.
Part List
Prototyping began by disassembling an old desktop computer for parts. Parts salvaged from this are as follows:
Power Supply – Capable of running 12A at 12V.  Printed details can be crudely seen below.
Heat Sink – For preventing the Peltier Chip from overheating on one side
Two Fans – to keep heat sinks cool (visible partly in above picture)
The only part salvaged from the old 1994 thermal cycler found in surplus was the heating block, whose design has not changed.
The rest of the parts were ordered from Amazon, and they include:
Arduino Mega2560
Thermal Compound (for attaching Peltier to heat sink)
VNH2SP30 Monster Motor Shield (H-Bridge)
Aluminum Heatsink
Peltier TEC1-12709
Prototype V1 – Shoebox Design (“The Testing Phase”)
Thermistor Testing
The first design was a completely disorganized mess of temporarily attached cables and individual components that all needed to be tested. The first component that needed to be calibrated was the thermistor. A pack of 100 NTC thermistors were bought from Amazon for less than $7. They were calibrated by the Steinhart-Hart equation, which goes as follows:
Where A, B, and C are constants unique to each thermistor that can be calculated as a system of equations by taking the resistance of the thermistor at three different and known temperatures.
The thermistor used in the PCR machine was calibrated on a kitchen stove with multimeter capable of reading temperature through its own thermistor attachment.
Peltier Chip Testing
Before further prototyping, it became necessary to test the Peltier chip to see if it could reach temperatures of 95°C.  The first TEC1-12706 (which can handle a maximum of 6A) could not reach the temperatures desired unless stacked, and when they were stacked were at high risk of shattering (cite: UMD iGEM team). With this foreknowledge, we went ahead and bought the TEC1-12709 model which is rated at 90W and can handle a maximum current of 9A (note: the openPCR project used a custom built Peltier chip that could handle a maximum of 86W and 7.8A. See openPCR.org BOM under Open Source Design). When plugged in to just the computer power supply, this new chip was drawing currents around 5A and heating a heating block twice as large as itself to 95°C in less than 1 min. Therefore, this chip has the power needed to conduct PCRs.
Motor Shield (H-Bridge) Testing
The first H-Bridge bought was an old L293D Arduino motor shield, capable of running the native voltage of the board (5V) and outputting 600mA per motor (max of 4 motors). We quickly realized this was an inadequate H-Bridge as we needed at least 5A for acceptable ramp up speeds. This problem was remedied when we came across the VNH2SP30 Monster Motor Driver on Amazon ( Linked Here ) which was rated for 14A of continuous current and up to 16V! The item was soon shipped and basic code implemented allowing the Arduino to control the voltage and direction of current through the PCR.
This Monster Motor Shield appeared at first to work great, as code was being tested to cycle through lower simulation temperatures continuously. It was able to make the Peltier reach 95°C many times, but it felt much slower than before. In the last week, we discovered that the website lied about the practical current use. The board shipped with a built in 4.12A current limiter. The unit also shipped with a defective VNH2SP30 H-Bridge. It failed to output a full 12V when the current was desired to flow in a counter-clockwise direction. Also, the shield has a safety to shut down all current when it senses potentially overheating. It has shut off countless times when it is just outputting 3A. For a unit rated at 14A, this is incredibly frustrating.
Brainstorming with UMD
Both of our teams decided separately that it would be cool to undertake the project of building a PCR machine for our team this summer. At the iGEM gathering hosted by the Maryland team at their university, we discovered the mutual progress. This started a series of scattered emails about updates and advice over the summer. While UMD sought novel and unique ways of solving the problems they ran into with weak Peltier chips, we stuck by our traditional H-Bridge and Peltier chip design as initial tests seemed very promising.
Summary
The Shoebox Design was a long-lived, frustrating phase of the prototyping process. Once each part of the machine was deemed suitable for incorporation into a more portable, permanent housing, the next phase of prototyping began.
Prototype V2 – Toolbox Design (“Functionality Phase”)
The goal of this design was to incorporate all the above components into a single compact and portable case. The case used was an old aluminum Remline toolbox my dad found sitting in the garage. After mounting the power supply to one side of the toolbox to maximize airflow in the case of a closed lid, the rest of the components were placed in a manner to balance the weight of the power supply without losing too much function.  Unnecessary cables were tucked away. The rest of the cables were neatly organized and unnecessary lengths trimmed down. An aluminum heat sink was attached to the VNH2SP30 chips and another fan incorporated to try and keep the motor shield cool, but even this is insufficient for how hot those chips get.
Current Status
We will be the first to admit that this design is still incredibly much a prototype. Right after assembling all the components into the toolbox, the heating block heated up to 95°C slowly but surely. After bringing the machine to college, where tools such as multimeters and oscilloscopes and soldering irons are reserved for classroom labs, the heating block refuses to cycle to 95°C. Sadly, we were not able to collect temperature over time data during the phase it was working. It can currently reach 78°C in just over a minute. We fear that assembling the H-Bridge into the toolbox has damaged it in some way, but have no way to test the voltage or current measurement.
Each functional element of the PCR machine can be reduced to a simple circuit diagram, which is hand drawn below.
Improvements
The one limiting factor in the success of the PCR machine is the Motor Shield. Replace this, and we have a truly functional thermocycler in a toolbox. Therefore, the first thing to fix is to try and bypass the faulty and frustratingly fickle Motor Shield with a self-built H-Bridge incorporating real high current relays. These could be attached to a breadboard or soldered together on top of a prototyping PCB (Printed Circuit Board). This should solve both the current limiting problem and overheating safety shut off problem of the Motor Shield.
The thermistor really ought to be calibrated with a temperature sensor more accurate than a food thermometer. Smaller thermistors should be bought and multiple should be embedded into the heating block and the temperatures of each averaged to receive a more accurate temperature of the whole block.
We purchased the Arduino Mega2560 for this project. However, so many digital I/O pins are entirely unnecessary for this machine. The cheaper Arduino Uno R3 would be more than sufficient for the machine.
Once faster and more reliable heating is achieved, the code should be modified to enable fine controls such as controlling the ramp up rate and preventing excess overshoot of the desired temperature.
A heating lid should eventually be added so that condensation does not appear on the PCR tubes. This can be accomplished with a couple of 10W 10Ω power resistors in series, or even a couple 100W 1Ω power resistors in parallel (<$6 on amazon). The circuit from the heating lid could be wired in series with the Peltiers to prevent overloading the power supply.
Antibiotic Detection
The fundamental idea of our project is to create an IF gate - if a certain genetic sequence is present on the target plasmid, transcribe a different gene. If the sequence is not present on the target plasmid, do not. We designed this circuit through using dCas9 through which you can create specific gRNAs that will bind to and deactivate a specific DNA sequence. We decided to use this circuit to tackle the issue of antibiotic resistance: if an antibiotic resistance gene (in our construct, we used Kanamycin resistance) is present in a plasmid, transcribe a programmable cell death gene to kill the cell. If no antibiotic resistance gene is present, nothing will happen.
Our assumptions for this design include that you must know the gene sequence for which you are targeting. The target plasmid must also have a high copy number in comparison to the dCas9 plasmid.
Selecting the Target sites
We decided to use multiple gRNAs in order to fully repress the gene. We chose sites at toward the beginning of the Kanamycin resistance gene, as earlier sites more effectively bind and inhibit the gene. These target sequences must be preceded by a Protospace Adjacent Motif (PAM)  which is required for the dCas9 to land. With Cas9 and dCas9, the PAM site is three nucleotides, NGG. The gRNA sequences were the reverse compliments of the gene segments we targeted.
Creating ‘Decoy’ Target Sites
Once we designed our gRNAs, we wanted to create the IF gate rather than just using dCas9 to repress the Kanamycin resistance gene. We wanted a reporter or a programmable cell death gene to be translated. We did this by implementing a construct of decoy binding sites. We used the same binding sites that we chose for the gRNAs to target and put them between the report promoter and the ribosomal binding site (RBS) on the dCas9 plasmid. However, the RBS must be exposed in order to translate the reporter protein. We put ‘spacers’ in between the different binding sites so the mRNA would form a hairpin and allow for a RBS to be exposed directly in front of the programmable cell death gene we wanted to be transcribed.
The predicted mRNA structure of the decoy site hairpin.
These added binding sites allow for the reporter gene to compete with the sites allow the Kanamycin resistance or other target gene. If the target plasmid was present, the gRNAs would attach to and deactivate the Kanamycin resistance gene on the target plasmid due to the relative high copy number of the target plasmid. This would allow for the decoy sites to hairpin and the programmable cell death gene to be transcribed. If the target plasmid was not present, the gRNAs would bind to the decoy binding sites and inhibit transcription of the programmable cell death gene.
Our Progress
This summer, we fully designed the construct including the gRNAs, dCas9 scaffolding, multiple programmable cell death genes, and the decoy binding site construct. However, we were never able to fully assemble the construct, and therefore do not have any definitive results on the effectiveness of the IF gate. Going forward, we hope to test the effective of the decoy hairpinning on transcription in isolation and in the overall antibiotic resistance circuit.
