<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Duke_Basic_Part skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Duke/Basic Part</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="mainContainer"><DIV id="menuContainer"><UL><LI><A href="https://2015.igem.org/Team:Duke">HOME</A></LI><LI><A href="https://2015.igem.org/Team:Duke/Team">TEAM</A></LI><LI><A href="#">PROJECT
            					</A></LI><LI><A href="https://2015.igem.org/Team:Duke/Description">Description</A></LI><LI><A href="https://2015.igem.org/Team:Duke/Experiments">Experiments &amp; Protocols</A></LI><LI><A href="https://2015.igem.org/Team:Duke/Results">Results</A></LI><LI><A href="https://2015.igem.org/Team:Duke/Design">Design</A></LI></UL><LI><A href="#">PARTS
            					</A></LI><LI><A href="https://2015.igem.org/Team:Duke/Parts">Team Parts</A></LI><LI><A href="https://2015.igem.org/Team:Duke/Basic_Part">Basic Parts</A></LI><LI><A href="https://2015.igem.org/Team:Duke/Notebook">NOTEBOOK</A></LI><LI><A href="https://2015.igem.org/Team:Duke/Attributions">ATTRIBUTIONS</A></LI><LI><A href="https://2015.igem.org/Team:Duke/Collaborations">COLLABORATIONS</A></LI><LI><A href="https://2015.igem.org/Team:Duke/Practices">HUMAN PRACTICES</A></LI><LI><A href="https://2015.igem.org/Team:Duke/Safety">SAFETY</A></LI><LI><A href="https://2015.igem.org/Team:Duke/Modeling">MODELING</A></LI><LI><A href="https://2015.igem.org/Team:Duke/Interlabstudy">INTERLAB STUDY</A></LI><LI><A href="https://2015.igem.org/Team:Duke/Software">SOFTWARE</A></LI></DIV><DIV id="contentContainer"><H2> Basic Parts</H2><H4>Part Table </H4>&lt;groupparts&gt;iGEM015 Duke&lt;/groupparts&gt;<H2>Cell Death Genes</H2><H3>BBa_K117000</H3><P>BBa_K117000 was originally designed and submitted by Nguyen Xuan Hung from Nanyang Technological University, Singapore, for the 2008 competition. It encodes only the cel gene, the third coding region in the tricistronic colicin-E7 operon which acts as a bacteriocin.  The purpose of cel gene is to induce cell lysis, allowing the ColE7 protein to diffuse out of the cell and kill other strains of bacteria. This biobrick was included in the plates supplied by iGEM.</P><H3>BBa_K628000</H3><P>BBa_K628000 was originally designed and submitted by Charles Thompson from St. Andrews University in Scotland for the 2011 competition. The gene encodes the antimicrobial peptide Protegrin-1, which is one of five protegrins found in porcine leukocytes. Protegrin-1 is an 18-amino acid-long peptide which forms a beta sheet thanks to two cysteine-cysteine disulfide bridges. It works by forming pores or channels in the cell membrane, resulting in the death of the bacterium. This mechanism is facilitated by the beta-sheet secondary structure. This biobrick was not provided so we had to have it synthesized for us.</P><H3>Bacteriophage ΦX174 E protein</H3><P>The phi X 174 bacteriophage is a phage that infects E. coli. The You Lab here at Duke shared with us a strain of bacteria carrying a plasmid which contained the phi X 174 bacteriophage’s E protein, which induces lysis in bacteria. Lysis is caused by a transmembrane tunnel created by the E protein.  It is believed that the transmembrane tunnel forms as a result of the E protein inhibiting peptidoglycan synthesis during cell growth.</P><P>One example of an application for this cell death gene is that employed by Lingchong You, et al. They designed a construct that contained the lysis gene as well as quorum sensing components, which caused populations oscillations over time.  The gene circuit did not work as anticipated, but the oscillations were a product of cell suicide caused by the E lysis gene.</P><P>The figure below graphically displays the E lysis protein’s effect on cell growth. The uninduced sample grew approximately four times as well as the fully induced sample.</P><H3>Tachyplesin I</H3><P>Tachyplesin I (TP-I) is a cysteine-rich antimicrobial peptide derived from the hemocytes of horseshoe crabs. It is very similar in structure, sequence, and function to Protegrin-1, containing the same disulfide bridges and resulting beta-sheet secondary structure which allows it to form pores in the membrane.  This antimicrobial peptide was the first gene that we designed. Similar to how St. Andrews created BBa_K117000, the gene sequence for TP-I was derived by translating the amino acid residues to their most common codon triplet, putting those triplets in order, and adding start and stop codons.</P><P>Further research into the action of tachyplesin and protegrin revealed that in addition to having antimicrobial ability, both peptides also exhibited some degree of hemolytic activity against human erythrocytes. In the interest of safety, we wanted to find a way to get rid of the hemolytic activity without abolishing the antimicrobial action. Some more digging led us to research that had been done on versions of both of these antimicrobial peptides in which all cysteine residues had been removed. For tachyplesin, it was found that while the deletion of cysteines somewhat decreased the peptides antibiotic capacity, it completely got rid of its ability to lyse human red blood cells. Likewise, the deletion of cysteines in protegrin was found to have little effect on its ability to kill bacteria, however there was no investigation into its hemolytic activity. Based on this information, we decided to develop genes for these cysteine-deleted antimicrobial peptides and investigate them further.</P><H3>Cysteine-Deleted Tachyplesin</H3><P>Cysteine-deleted Tachyplesin (CDT) is just as it sounds: it contains the same sequence as Tachyplesin-1 minus the four cysteine residues. The consequence of the lack of cysteine residues in CDT is that the peptide cannot form the disulfide bridges result in the permanent beta-sheet. Although CDT is only about half as effective as original Tachyplesin-I, the change  is likely responsible for the increase in the degree of specificity of action to bacteria and away from human erythrocytes which was noted in previous research. However, because of time constraints, CDT was not successfully BioBricked.</P><H3>Cysteine-Deleted Protegrin</H3><P>Cysteine-deleted Protegrin (CDP) contains the same sequence as Protegrin-1 minus the four cysteine residues. CDP’s lack of cysteine residues means it cannot form disulfide bridges to bring together the permanent beta-sheet secondary structure. Although the secondary structure is no longer permanent with the deletion of the cysteines, it can still be achieved transiently and maintained if it experiences the correct interactions. CDP is still capable of killing bacteria, which are the original intended target of these peptides and thus show a greater degree of binding specificity and affinity.</P></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>