<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Kyoto_GSDD_Results"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:Kyoto/GSDD/Results">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Kyoto/GSDD/Results&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Kyoto/GSDD/Results&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Kyoto/GSDD/Results&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Kyoto/GSDD/Results" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Kyoto/GSDD/Results</H1><DIV id="bodyContent"><H3 id="siteSub">From 2009.igem.org</H3><DIV id="topicpath"><OL><LI><A href="/Team:Kyoto" title="Team:Kyoto">Home</A></LI><LI><A href="/Team:Kyoto/GSDD" title="Team:Kyoto/GSDD" class="mw-redirect">GSDD</A></LI><LI>Reslts &amp; Discussion
</LI><LI><A href="/Team:Kyoto/Others#Team" title="Team:Kyoto/Others">Team</A></LI><LI><A href="/Team:Kyoto/Others#Sponsor" title="Team:Kyoto/Others">Sponsor</A></LI><LI><A href="/Team:Kyoto/Others#Acknowledgment" title="Team:Kyoto/Others">Acknowledgment</A></LI></OL></DIV><DIV id="leftnavi"><DIV id="navigation"><UL id="navi_GSDD"><LI><A href="https://2009.igem.org/Team:Kyoto/GSDD" title="Go to Project top.">GSDD</A></LI><LI class="nav_av"><A href="https://2009.igem.org/Team:Kyoto/GSDD/Abstract" title="Go to Abstract page.">Abstract</A></LI><LI class="nav_i"><A href="https://2009.igem.org/Team:Kyoto/GSDD/Introduction" title="Go to Introduction page.">Introduction</A></LI><LI class="nav_me"><A href="https://2009.igem.org/Team:Kyoto/GSDD/Mechanism" title="Go to Mechanism page.">Mechanism</A></LI><LI class="nav_g"><A href="https://2009.igem.org/Team:Kyoto/GSDD/Goals" title="Go to Goals page.">Goals</A></LI><LI class="nav_e"><A href="https://2009.igem.org/Team:Kyoto/GSDD/Experiment" title="Go to Experiment page.">Experiment</A></LI><LI class="nav_r"><A href="https://2009.igem.org/Team:Kyoto/GSDD/Results" title="Go to Results &amp; Discussion page.">Results &amp; Discussion</A></LI><LI class="nav_p"><A href="https://2009.igem.org/Team:Kyoto/GSDD/Parts" title="Go to Registered parts page.">Parts</A></LI><LI class="nav_mo"><A href="https://2009.igem.org/Team:Kyoto/GSDD/Modelling" title="Go to Modelling page.">Modelling</A></LI><LI class="nav_n"><A href="https://2009.igem.org/Team:Kyoto/GSDD/Notebook" title="Go to Notebook page.">Notebook</A></LI></UL><UL id="navi_CiC"><LI><A href="https://2009.igem.org/Team:Kyoto/CiC" title="Go to Project top.">Cells in Cells</A></LI><LI class="nav_i"><A href="https://2009.igem.org/Team:Kyoto/CiC/Introduction" title="Go to Introduction page.">Introduction</A></LI><LI class="nav_g"><A href="https://2009.igem.org/Team:Kyoto/CiC/Goals" title="Go to Goals page.">Goals</A></LI><LI class="nav_me"><A href="https://2009.igem.org/Team:Kyoto/CiC/Method" title="Go to Method page.">Method</A></LI><LI class="nav_e"><A href="https://2009.igem.org/Team:Kyoto/CiC/Experiment" title="Go to Experiment page.">Experiment</A></LI><LI class="nav_r"><A href="https://2009.igem.org/Team:Kyoto/CiC/Results" title="Go to Results &amp; Discussion page.">Results &amp; Discussion</A></LI><LI class="nav_p"><A href="https://2009.igem.org/Team:Kyoto/CiC/Parts" title="Go to Registered parts page.">Parts</A></LI><LI class="nav_n"><A href="https://2009.igem.org/Team:Kyoto/CiC/Notebook" title="Go to Notebook page.">Notebook</A></LI></UL><UL><LI class="nav_sa"><A href="https://2009.igem.org/Team:Kyoto/Safety" title="Go to SafetyIssue page.">Safety Issue</A></LI><LI class="nav_sq"><A href="https://2009.igem.org/Team:Kyoto/sequence" title="Go to Sequence page.">Sequence</A></LI></UL></DIV></DIV><DIV id="rightcontents"><H2><SPAN class="mw-headline" id="Experimental_Results">Experimental Results</SPAN></H2><H3><SPAN class="mw-headline" id="Result_of_subgoal-1.3B_Microgene_Polymerization_Reaction_.28MPR.29_for_constructing_repetitive_LacI_binding_sites">Result of subgoal-1; Microgene Polymerization Reaction (MPR) for constructing repetitive LacI binding sites</SPAN></H3><P>We achieved constructing repetitive sequence of LacI binding site. A certain length of LacI binding site is required in both ends of Timer vector. We expected that LacI binds to this site and protects the both ends of Timer vector from the degradation caused by exonuclease, so the both ends of Timer vector become shorter per duplication by the end replication problem. In short, the repetitive sequence counts the time.  Also, constructing various length of repetitive sequence was necessary because we did not evaluate the DNA stability by binding LacI. 
As a result, we constructed a set of repetitive sequence that contains multiple LacI binding site by MPR (Microgene Polymerization Reaction). MPR suits our experiment because various length of DNA are constructed at once without ligation.
</P><P>It was suggested that the temperature during MPR influences the length of the acquired sequence. Because we aimed to construct a set of repetitive DNA with different lengths, we investigated and optimized the temperature condition during MPR. After getting repetitive sequences, we inserted them into plasmid by flush end ligation, and designed the part that has repetitive sequence.
</P><H4><SPAN class="mw-headline" id="1.29_Experimental_conditions_in_MPR">1) Experimental conditions in MPR</SPAN></H4><P>We tried various conditions in the temperature X. The result indicated that the temperature during MPR influences the length of the acquired sequence. At last, we programmed 72 o C of the temperature X. 
Fig shows the repetitive sequence that we used for designing parts. As shown in Fig. 4, we succeeded in constructing the polymerized microgenes with different number of LacI binding sites.
</P><H4><SPAN class="mw-headline" id="2.29_Designing_the_repetitive_part_of_LacI_binding_site">2) Designing the repetitive part of LacI binding site</SPAN></H4><P>We can not embed restriction site in the both ends of repetitive sequence part because it is consists of repeating short DNA. Embedding restriction site in each short primer leads to the degradation into small parts when digesting by enzyme. Therefore, we inserted it into plasmid by blunt end ligation.  
The repetitive part was inserted into PSB1A2 which has blunt end. In this process, we phosphorylated PSB1A2, and dephosphorylated the repetetitive. We purified them before ligation.
	In conclusion, we successfully designed and constructed the repetitive sequences with the multiple protein binding sites by MPR. This is the simple method to construct any repetitive DNA sequence. Thus, it might be applicable to synthesize other repetitive sequences with different lengths in a tailor-made manner.
</P><P><B>blunt end clonig by Stu1 (BBa_K210014)</B></P><H3><SPAN class="mw-headline" id="Parts_sequence">Parts sequence</SPAN></H3><P>We sequenced features and parts which we constructed. The results of sequences are matched as we expect.
If you see the detailed data, click <A href="/Team:Kyoto/sequence" title="Team:Kyoto/sequence">here</A>.
</P><H3><SPAN class="mw-headline" id="Result_of_subgoal-2_Construction_of_Timer_vector_that_contains_LacI_generator_and_TIMER">Result of subgoal-2 Construction of Timer vector that contains LacI generator and TIMER</SPAN></H3><P><B>Timer vector #1600 for measurement (BBa_K210008)</B></P><H3><SPAN class="mw-headline" id="Result_of_subgoal-3_Construction_of_the_constitutive_LacI_expression_vector">Result of subgoal-3 Construction of the constitutive LacI expression vector</SPAN></H3><P><B>lacI generator for yeast (BBa_K210005)</B></P><H3><SPAN class="mw-headline" id="Result-4_Analysis_of_the_constructed_TIMER_vector_in_yeast">Result-4 Analysis of the constructed TIMER vector in yeast</SPAN></H3><P>No data.
</P><H2><SPAN class="mw-headline" id="Discussion">Discussion</SPAN></H2><H3><SPAN class="mw-headline" id="The_difference_between_our_designed_device_and_YAC">The difference between our designed device and YAC</SPAN></H3><P>The difference between our designed TIMER and YAC
In the case of YAC, both ends is composed of teromeric site. Thus, when some of the end sequences of them are deleted gradually by the end replication problem, the deleted sequences will be repaired. But in the case of our system, the deleted sequence will not be repaired because it does not contain the telomeric sites. Therefore, the length of the TIMER vector will become shorter as cell division proceeds. We assume that the most critical problem in this system is that we do not know exactly how many DNA bases are removed during each cell division and how differently the system works between each cell. We assume that about 100-200 base pairs (this length is same with the average primer sequence in the ragging chain replication of yeast cells) are removed during one cell division. Thus, we constructed a set of repetitive DNA sequences with the different number of LacI binding sites to optimize the condition in our system. 
</P><H3><SPAN class="mw-headline" id="Evaluation_for_the_inhibition_of_degradation">Evaluation for the inhibition of degradation</SPAN></H3><P>If we model Timer vector, we can upgrade the timer function, and apply Timer vector to more various uses. In order to model it, it is necessary to evaluate the time to that the repetitive sequence is completely lost and Timer vector is degraded. We need to analyze the length of DNA which becomes shorter during each cell division. 
</P><P>The both ends of Timer vector become short because of the end replication problem as already discussed. However, the both ends are also degraded by exonuclease because the LacI that binds to the repetitive sequences does not completely protect and stabilize the both end of Timer. Therefore, evaluation for the inhibition of degradation by LacI is important. 
</P><P>Thus, to evaluate the inhibition by LacI binding, we need to verify the length of linear DNA which becomes short not by the end replication problem but by exonuclease, after the Timer Vector is transformed.
</P><P>In previous work, it has been reported that a designed linear DNA was stabilized in E. coli. by the binding of lactose repressor to the both ends of the designed repeated binding sequences (REFERENCE). They quantified the inhibition efficiency by the protein expression level, by majoring the amount of fluorescence of GFP expressed from the linear DNA. They compared (a) and (b) below. They observed them in cell-free gene expression system, RTS (rapid translation system, Roche diagnostics), in the presence of the exonuclease. 
</P><DL><DD>(a) linear DNA which can express GFP
</DD><DD>(b) linear DNA which can express GFP and have LacI binding site in both ends
</DD></DL><P>After (a) and (b) were expressed, the amount of fluorescence in (b) was more than (a). In addition, when they increased the concentration of LacI in RTS, the difference of them became bigger. This work indicated that LacI in ends inhibit the degradation by exonuclease. In previous work, they quantified the inhibition not by the length of degraded DNA but by the expression level, and they used E coli. No studies have ever evaluated the length of DNA degraded by exonuclease and the length protected by LacI in Saccharomyces cerevisiae. We thought of two following ideas to investigate them. 
</P><P>The first idea is to observe linear DNA which have various length of LacI binding site in both ends, and measure the time to stop the expression of GFP. In this idea, much data is necessary to estimate the parameters to model it. It is thought to be difficult to estimate the parameters, because this data is supposed to vary because the end replication problem itself doesn’t happen evenly. So the second idea is supposed to be more realistic. It is to stop the duplication, and measure the length of linear DNA which is shortened only by exonuclease. After the Timer vector is transformed, we stop the duplication in some way. We can observe the expression of GFP, and the time to stop the expression. This enables us to quantify the inhibition by the length of degraded DNA, where the end replication problem doesn’t occur.  
</P><P>
To sum up, in order to model the Timer vector, we need to evaluate the length of DNA degraded by exonuclease besides the end replication problem. This is expected to be possible by some experiments noted above.
</P><H3><SPAN class="mw-headline" id="The_instability_aspect_of_linear_vector">The instability aspect of linear vector</SPAN></H3><P>The ends of DNA double-strand break are repaired by homologous recombination or nonhomologous end-joining after they are degraded by exonuclease inside the cell. Therefore, linear vector is supposed to be instable because there is a possibility of recombination with other DNA or circularization in addition to DNA degradation. Binding LacI or other DNA binding protein in both ends is expected to inhibit degradation, recombination and circularization. For example, telomeric DNA inside the cell has such a mechanism (REFERENCE).
Timer vector has LacI binding sites in both ends. However, there still may be the instability aspect of it. In order to reinforce this end protection, the following ideas are likely to be effective.
</P><OL><LI> Use other binding protein which has stronger binding force, instead of LacI
</LI><LI> Use mutant yeast with loss of function of a protein related DNA degradation 
</LI></OL><P>By adopting these ideas, suppression of the instability is supposed to be stronger. 
</P></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2009.igem.org/Team:Kyoto/GSDD/Results">http://2009.igem.org/Team:Kyoto/GSDD/Results</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Kyoto/GSDD/Results" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Kyoto/GSDD/Results" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Kyoto/GSDD/Results&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Kyoto/GSDD/Results&amp;oldid=166794" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2009.igem.org:Privacy_policy" title="2009.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2009.igem.org:General_disclaimer" title="2009.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>