<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Kyoto_CiC_Experiment"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:Kyoto/CiC/Experiment">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Kyoto/CiC/Experiment&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Kyoto/CiC/Experiment&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Kyoto/CiC/Experiment&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Kyoto/CiC/Experiment" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Kyoto/CiC/Experiment</H1><DIV id="bodyContent"><H3 id="siteSub">From 2009.igem.org</H3><DIV id="topicpath"><OL><LI><A href="/Team:Kyoto" title="Team:Kyoto">Home</A></LI><LI><A href="/Team:Kyoto/CiC" title="Team:Kyoto/CiC">Cells in Cells</A></LI><LI>Results &amp; Discussion
</LI><LI><A href="/Team:Kyoto/Others#Team" title="Team:Kyoto/Others">Team</A></LI><LI><A href="/Team:Kyoto/Others#Sponsor" title="Team:Kyoto/Others">Sponsor</A></LI><LI><A href="/Team:Kyoto/Others#Acknowledgment" title="Team:Kyoto/Others">Acknowledgment</A></LI></OL></DIV><DIV id="leftnavi"><DIV id="navigation"><UL id="navi_GSDD"><LI><A href="https://2009.igem.org/Team:Kyoto/GSDD" title="Go to Project top.">GSDD</A></LI><LI class="nav_av"><A href="https://2009.igem.org/Team:Kyoto/GSDD/Abstract" title="Go to Abstract page.">Abstract</A></LI><LI class="nav_i"><A href="https://2009.igem.org/Team:Kyoto/GSDD/Introduction" title="Go to Introduction page.">Introduction</A></LI><LI class="nav_me"><A href="https://2009.igem.org/Team:Kyoto/GSDD/Mechanism" title="Go to Mechanism page.">Mechanism</A></LI><LI class="nav_g"><A href="https://2009.igem.org/Team:Kyoto/GSDD/Goals" title="Go to Goals page.">Goals</A></LI><LI class="nav_e"><A href="https://2009.igem.org/Team:Kyoto/GSDD/Experiment" title="Go to Experiment page.">Experiment</A></LI><LI class="nav_r"><A href="https://2009.igem.org/Team:Kyoto/GSDD/Results" title="Go to Results &amp; Discussion page.">Results &amp; Discussion</A></LI><LI class="nav_p"><A href="https://2009.igem.org/Team:Kyoto/GSDD/Parts" title="Go to Registered parts page.">Parts</A></LI><LI class="nav_mo"><A href="https://2009.igem.org/Team:Kyoto/GSDD/Modelling" title="Go to Modelling page.">Modelling</A></LI><LI class="nav_n"><A href="https://2009.igem.org/Team:Kyoto/GSDD/Notebook" title="Go to Notebook page.">Notebook</A></LI></UL><UL id="navi_CiC"><LI><A href="https://2009.igem.org/Team:Kyoto/CiC" title="Go to Project top.">Cells in Cells</A></LI><LI class="nav_i"><A href="https://2009.igem.org/Team:Kyoto/CiC/Introduction" title="Go to Introduction page.">Introduction</A></LI><LI class="nav_g"><A href="https://2009.igem.org/Team:Kyoto/CiC/Goals" title="Go to Goals page.">Goals</A></LI><LI class="nav_me"><A href="https://2009.igem.org/Team:Kyoto/CiC/Method" title="Go to Method page.">Method</A></LI><LI class="nav_e"><A href="https://2009.igem.org/Team:Kyoto/CiC/Experiment" title="Go to Experiment page.">Experiment</A></LI><LI class="nav_r"><A href="https://2009.igem.org/Team:Kyoto/CiC/Results" title="Go to Results &amp; Discussion page.">Results &amp; Discussion</A></LI><LI class="nav_p"><A href="https://2009.igem.org/Team:Kyoto/CiC/Parts" title="Go to Registered parts page.">Parts</A></LI><LI class="nav_n"><A href="https://2009.igem.org/Team:Kyoto/CiC/Notebook" title="Go to Notebook page.">Notebook</A></LI></UL><UL><LI class="nav_sa"><A href="https://2009.igem.org/Team:Kyoto/Safety" title="Go to SafetyIssue page.">Safety Issue</A></LI><LI class="nav_sq"><A href="https://2009.igem.org/Team:Kyoto/sequence" title="Go to Sequence page.">Sequence</A></LI></UL></DIV></DIV><DIV id="rightcontents"><H2><SPAN class="mw-headline" id="Experiments">Experiments</SPAN></H2><P><B>Construction</B></P><H3><SPAN class="mw-headline" id="HIV-TAT::.28LALAAAA.293_expressing_vector_.28BBa_K210009.29">HIV-TAT::(LALAAAA)3 expressing vector (BBa_K210009)</SPAN></H3><TABLE class="table"><TBODY><TR><TH>Parts</TH><TH>Origin</TH><TH>Length /bp
</TH></TR><TR><TD>T7 promoter</TD><TD>BBa_I712074</TD><TD>46
</TD></TR><TR><TD>RBS+HIV-TAT+HIStag+(LALAAAA)3</TD><TD>201
</TD></TR><TR><TD>Terminator</TD><TD>BBa_B0015</TD><TD>129
</TD></TR></TBODY></TABLE><P>†RBS+HIV-TAT+HIStag+(LALAAAA)3 was made by elongation of primer dimer.
</P><P>Forward primer; 
</P><P>cggaattcgcggccgcttctagagaaagaggagaaatactagATGTATGGACGTAAAAAACGTCGTGGACGTCGTCGTGGCGGCGGTCAT
CATCATCATCACCATGGCGG
</P><P>Reverse primer;
</P><P>ctgcagcggccgctactagtaTTACGCGGCCGCCGCCAGGGCCAGCGCGGCCGCCGCCAGGGCCAGCGCGGCCGCCGCCAGG
GCCAGGCCACCGCCATGGTGATGATGATG
</P><H3><SPAN class="mw-headline" id="Signal_for_TIM23_complex::GFP_expressing_vector_.28BBa_K210010.29">Signal for TIM23 complex::GFP expressing vector (BBa_K210010)</SPAN></H3><TABLE class="table"><TBODY><TR><TH>Parts</TH><TH>Origin</TH><TH>Length /bp
</TH></TR><TR><TD>T7 promoter</TD><TD>BBa_I712074</TD><TD>46
</TD></TR><TR><TD>RBS+Signal for TIM23 complex::GFP+terminator</TD><TD>BBa_E0240</TD><TD>929
</TD></TR></TBODY></TABLE><P>†RBS+Signal for TIM23 complex::GFP+terminator was made by two-stage PCR.
</P><P>First-stage PCR
</P><P>Forward primer;
</P><P>TTTAAACCGGCGACCCGTACCCTGTGCTCTTCTCGTTATCTGCTGcgtaaaggagaagaacttttcactggagttg
</P><P>Reverse primer;
</P><P>agtgagctgataccgctcgc
</P><P>Second-stage PCR
</P><P>Forward primer;
</P><P>cggaattcgcggccgcttctagagaaagaggagaaatactagATGCTGAGCCTGCGTCAGTCTATTCGTTTTTTT
AAACCGGCGACCCGTAC
</P><P>Reverse primer;
</P><P>agtgagctgataccgctcgc
</P><H3><SPAN class="mw-headline" id="Signal_for_TIM23_complex::EGFP_expressing_vector_for_HeLa_cells">Signal for TIM23 complex::EGFP expressing vector for HeLa cells</SPAN></H3><P>We made the phosphated primerdimer by below primers. And, ligate it and pEGFP-N3 (GenBank Accession #: U57609) digested by Xho1 and Pst1.
</P><P>Forward primer;
</P><DIV style="width:700px;overflow:auto;"><PRE style="width:800px;">TCGAGgccaccATGggtCTGAGCCTGCGTCAGTCTATTCGTTTTTTTAAACCGGCGACCCGTACCCTGTGCTCTTCTCGTTATCTGCTG</PRE></DIV><P>Reverse primer;
</P><DIV style="width:700px;overflow:auto;"><PRE style="width:800px;">AATTCAGCAGATAACGAGAAGAGCACAGGGTACGGGTCGCCGGTTTAAAAAAACGAATAGACTGACGCAGGCTCAGaccCATggtggcC</PRE></DIV><H3><SPAN class="mw-headline" id="Making_proteoliposome_by_RTS">Making proteoliposome by RTS</SPAN></H3><P>1. Remove the solvent of 50mM DOPC (Di-oleyl phosphatidylcholine, resolute in chloroform: methanol=2:1) 100μl in Ar.
</P><P>2. Desiccate the DOPC in vacuum
</P><P>3. Add 50mM HEPES-KOH 100μl
</P><P>4. Adjust liposome size by mini-extruder of 200nm pore size filter
</P><P>5. Making proteoliposomes in RTS
</P><TABLE class="table"><TBODY><TR><TD>sample name</TD><TD>volume /ul
</TD></TR><TR><TD>E.coli lysate</TD><TD>12
</TD></TR><TR><TD>reaction mix   </TD><TD>10
</TD></TR><TR><TD>Amino acids mix</TD><TD>12
</TD></TR><TR><TD>methionine</TD><TD>1
</TD></TR><TR><TD>DNA</TD><TD>1
</TD></TR><TR><TD>liposome</TD><TD>14
</TD></TR></TBODY></TABLE><P>
6. Ultracentrifuge RTS product in 5, 10, 15, 20, 25% sucrose HEPES-KOH solution.
</P><H3><SPAN class="mw-headline" id="The_isolation_of_mitochindria_from_yeast">The isolation of mitochindria from yeast</SPAN></H3><P>1.Prepare 1L of yeast culture.
　　　　　　
</P><P>2.Cfg.)  at 2000×g for 5 min.
</P><P>3.Weigh the gained yeast pellets.  (Normally, 3-5g in total)
</P><P>4.Resuspend all the pellets together in 50ml of Tris-Buffer and shake at 30℃ for 30 min.
</P><P>5.Cfg.)  at 2000×g for 5 min.
</P><P>6.Resuspend the pellet in 45ml of Sorbitol Buffer.
</P><P>7.Cfg.)  at 2000×g for 5 min.
</P><P>8.Resuspend the pellet in 40ml of Sorbitol Buffer.  Add 5mg of zymolyase 20-T per 1g of yeast pellet.
</P><P>9.Incubate at 30℃ for 15-30 min. (Until the spheroplasts are formed sufficiently. )
</P><P>10.Cfg.)  at 6000×g for 5 min. (at 4℃)
</P><P>11.Resuspend the pellet in 30ml of ice cold Sorbitol Buffer.　　(The following steps should all be carried out on ice, or at 4℃ when centrifuging.)
</P><P>12.Cfg.)  at 3000×g for 5 min. (at 4℃)
</P><P>13.Resuspend the pellet in 30ml of Breaking Buffer.
</P><P>14.Homogenize the suspension in Dounce homogenizer.  With 15 strokes by pestle A (tight fitting).
</P><P>15.Cfg.)  at 1000×g for 5 min.  (at 4℃)
</P><P>16.Transfer the supernatant (~30ml) to a clean FALCON tube and store it as Sup-1 on ice.  
Resuspend the pellet in 30ml of Breaking Buffer and homogenize it.
(Dounce, pestle A, 15 strokes)
</P><P>17.Cfg.)  at 1500×g for 5 min.
</P><P>18.Transfer the supernatant to the Sup-1 tube and mix it together. 
</P><P>19.Cfg.)　at 12000×g for 5 min.
</P><P>20.Discard the supernatant and add 30ml of Breaking Buffer to the pellet, and resuspend it thoroughly by pipetting.
</P><P>21.Cfg.)  at 1500×g for 5 min.
</P><P>22.Transfer the supernatant to a clean FALCON tube.
</P><P>23.Cfg.)  at 12000×g for 10 min.
</P><P>24.Resuspend the pellet in 30ml of SEM Buffer.
</P><P>25.Cfg.)  at 12000×g for 10 min.
</P><P>26.Resuspend the pellet in 1ml of SEM Buffer.  (The isolated mitochondrial fraction is gained.)
</P><H3><SPAN class="mw-headline" id="Observation">Observation</SPAN></H3><H4><SPAN class="mw-headline" id="Subgoal_A">Subgoal A</SPAN></H4><P>In the experiment on subgoal A, we are aimed to confirm that the liposome with HIV-TAT can intrude mammalian cells.We will mix pSB1A2-T7 promoter- HIV-TAT::(LALAAAA)3-ter and liposome with fluorochrome and translated by RTS. When HIV-TAT::(LALAAAA)3 is translated, it sticks the lipid bilayer. As a result, the liposome which has HIV-TAT on their surface is completed. Adding it to HeLa cells. Later, we will observe the fluorescence of liposome in HeLa cells.
</P><P>We chose HeLa cells for this experiment.
HeLa cells were cultured at 37°C in D-MEM/F-12. The media was supplemented with 10% FBS and an antibitic/antimycotic solution for maintaining HeLa cells.
To confirm the function of translocation into cells of TAT-anchored liposomes, HeLa cells were cultured with
prepared TAT-liposome with NBD-PE, which is a fluorescent molecule bind to lipid.
HeLa cells were plated in 3.5 cm glass bottom
dish (6x10<SUP>5</SUP> cells/well), pre-incubated
for 20 hours at 37°C,
and the medium was replaced with Opti-MEM
that containing the NBD-PE-liposome with or without TAT.
The cells were incubated at 37°C for 4 hours and
washed by fresh D-MEM/F-12 supplemented with 10% FBS twice,
and then, obserbed.
The fluorescent images were captured by IX81
microscope system using a 470- to 495-nm excitation
filter and a 510- to 550-nm emission filter.
The conforcal images were captured by Nikon A1Rsi conforcal microscopy
system with 0.6 um step
using a 488 nm semiconductor lazer for excitation and
525/50 nm emission filter.
The result is <A href="#sig-GFP_expression_in_HeLa">here</A>.
</P><H4><SPAN class="mw-headline" id="Subgoal_B">Subgoal B</SPAN></H4><P>In the experiment on subgoal B, we are aimed to confirm that the recombinant of TIM23 complex can work as a protein translocater for its signal peptide. First, we confirm that the protein with the signal peptide for yeast can be taken in yeast’s and HeLa cell’s mitochondria.
</P><P>In the case of yeast’s mitochondria, we isolate mitochondria from yeast, translate signal GFP (BBa_K210010) by RTS and PURE system in which we have added the isolated yeast’s mitochondria. Then we will observe the localization of GFP fluorescence. 
The prepared yeast mitocondria suspension was mixed to the cell-free
protein synthesis with sigGFP plasmid.
We tried the synthesis kit RTS (Roshe, for total amount of synthesized
protein), or Puresystem (postgenome, for the obtained proteins
pulity). The mitocondria suspension was added to the protein synthesis
solution. Except for the mitcondria suspension, the protein synthesis
reactions were done followed the company’s protocols.
After the reaction finished, Mitotracker orange(invitrogen) in DMSO
was added to the liposome/mitocondria sample. The final concentration
was 400nM. After gently shaking for 20min at 37ºC, the sample solution
was spin-washed with 1mL of SEM buffer (12,500xg, 4ºC, 10min., twice).
Washed sample was re-suspended for 20µL by SEM buffer. Obtained
mitocondria suspensions were observed by fluorescent microscopy and
laser scanning microscopy.
</P><P>In the case of HeLa cell’s mitochondria, we transform HeLa cell by Signal for TIM23 complex::EGFP expressing vector for HeLa cells. Later, we will observe the localization of EGFP fluorescence.
This research is aimed to confirm the function of signal sequence using the plasmid coding sig-GFP.
Although the signal sequence is derived from yeast, we determined to use HeLa cell for the confirmation since the sequence was expected to work in other eukaryotic cells and yeast cell might be too small to observe its mitochondria.
HeLa cells were cultured at 37°C in D-MEM/F-12 for this experiment. The media was supplemented
with 10% FBS and an antibitic/antimycotic solution for maintaining HeLa cells.
The incubator contained air that was enriched with 5% CO2.
HeLa cells were plated in a 6 well glass bottom
plate (6x105 cells/well), pre-incubated
for 9 hours at 37°C,
and the medium was replaced with fresh D-MEM/F12 supplemented with 10% FBS.
The cells were transfected with 1.5 ug of pGFP or psigGFP using
3.75 ul of lipofectamine 2000 according to the standard
protocols provided by invtrogen.
After 12 hours or more of the transfection,
the medium was replaced with fresh medium
with or without 100 nM of mitotracker orange in order to visualize the location of mitochondria, and
incubated at 37°C for 15 min.
The stained cells were washed by fresh medium twice, and then
microscopic images were captured.
The microscopic images were captured by a IX81 microscope system
using 470-495 nm or 530-550 nm filters were used for excitation of
GFP or mitotracker orange, and
510-550 nm or 575-625 nm filters were used for emission, respectively.
The conforcal images were captured by an A1Rsi conforcal microscopy system with 0.15 um step
using 488 nm or 567 nm semiconductor laser for excitation
of GFP or mitotracker orange, and
525/50 nm or 595/50 nm filters were used for emission, respectively.
</P><P>Second, we make proteoliposome which has the recombinant of yeast TIM23 complex and mix it in Signal for TIM23 complex::GFP which is translated by RTS. Then we will observe the localization of GFP fluorescence. 
</P><TABLE class="table"><CAPTION>RTS reaction components volume /ul
</CAPTION><TBODY><TR><TH>sample No.</TH><TH>1</TH><TH>2</TH><TH>3</TH><TH>4</TH><TH>5
</TH></TR><TR><TD>E.coli lysate</TD><TD>12</TD><TD>12</TD><TD>12</TD><TD>12</TD><TD>12
</TD></TR><TR><TD>reaction mixture</TD><TD>10</TD><TD>10</TD><TD>10</TD><TD>10</TD><TD>10
</TD></TR><TR><TD>amino acids mix</TD><TD>12</TD><TD>12</TD><TD>12</TD><TD>12</TD><TD>12
</TD></TR><TR><TD>Methionine</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD>1</TD><TD>1
</TD></TR><TR><TD>Reconstitution Buffer</TD><TD>5</TD><TD>5</TD><TD>5</TD><TD>5</TD><TD>5
</TD></TR><TR><TD>liposome</TD><TD>9</TD><TD>9</TD><TD>SEM9</TD><TD>NFW9</TD><TD>NFW9
</TD></TR><TR><TD>plasmid</TD><TD>1</TD><TD>NFW1</TD><TD>1</TD><TD>1</TD><TD>NFW1
</TD></TR></TBODY></TABLE></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2009.igem.org/Team:Kyoto/CiC/Experiment">http://2009.igem.org/Team:Kyoto/CiC/Experiment</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Kyoto/CiC/Experiment" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Kyoto/CiC/Experiment" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Kyoto/CiC/Experiment&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Kyoto/CiC/Experiment&amp;oldid=166959" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2009.igem.org:Privacy_policy" title="2009.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2009.igem.org:General_disclaimer" title="2009.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>