"
Page
Discussion
View source
History
teams
Log in
 
Team:Kyoto/GSDD/Experiment
From 2009.igem.org
Home
GSDD
Modelling
Team
Sponsor
Acknowledgment
GSDD
Abstract
Introduction
Mechanism
Goals
Experiment
Results & Discussion
Parts
Modelling
Notebook
Cells in Cells
Introduction
Goals
Method
Experiment
Results & Discussion
Parts
Notebook
Safety Issue
Sequence
Experiment
Construction
PCR
Part
Origin
Length /bp
Primer sequence
pTEF2
BBa_K165037
403
(Fwd) 5'- CGGAATTCGCGGCCGCTTCTAG -3'
(Rev) 5'- GTTTTTTTGCCGGACTGCAGCG -3'
kozac-LacI
BBa_C0012
1128
(Fwd) 5'- GCTCTAGAGCAAAAAAATGTCTGTGAATGTGAAACCAGTAACGTTATACGATGTC -3'
(Rev) 5'- AGTGAGCTGATACCGCTCGC -3'
SV40-ADH1
BBa_K105013
253
(Fwd)
(Rev)
Part
Origin
Length /bp
Primer sequence
pTEF2
BBa_K165037
403
(Fwd) 5'- CGGAATTCGCGGCCGCTTCTAG -3'
(Rev) 5'- GTTTTTTTGCCGGACTGCAGCG -3'
kozac-eGFP
BBa_E0040
787
(Fwd) 5'- GCTCTAGAGCAAAAAAATGTCTCGTAAAGGAGAAGAACTTTTCACTGGAGTTG -3'
(Rev) 5'- AGTGAGCTGATACCGCTCGC -3'
PEST
pd2eGFP-Basic
123
(Fwd) 5'- CGGAATTCGCGGCCGCTTCTAGAACTGTACAAGAAGCTTAGCCATGGC -3'
(Rev) 5'- CTGCAGCGGCCGCTACTAGTAAGAGCATCTACACATTGATCCTAGCAG -3'
CEN-ARS
pYAC4
1585
(Fwd) 5'- TCGCGGCCGCTTCTAGAGTGGGTTGGAAGGCAAGAGAGC -3'
(Rev) 5'- CTGCAGCGGCCGCTACTAGTATGTCCATGATTCGCCGGGTAAATAC -3'
kozac-URA3
pYAC4
1057
(Fwd) 5'- CGGAATTCGCGGCCGCTTCTAGAGGCAGGAAACGAAGATAAATCATGTCGAAAG -3'
(Rev) 5'- CTGCAGCGGCCGCTACTAGTATCAACGACAGGAGCACGATCATG -3'
SV40-ADH1
BBa_K105013
266
(Fwd)
(Rev)
lacIBS
rpeats
1500
(Fwd)
(Rev)
MPR
We constructed repetitive sequence by MPR(Microgene Polymerization Reaction). The sequence can be divided into some parts according to the length. They were inserted into PSB1A2 which has blunt end. In this process, we dephosphorylate PSB1A2, and phosphorylate repetetitive sequence.
mechanism of MPR
Initially, to design a microgene, a pair of MPR primers was synthesized so that (i) the pair contained complementary bases in their 3' region; (ii) the primers had mismatched bases at their 3'-OH ends. These mismatched nucleotides at the 3'-OH ends of MPR primers are critical for successful polymerization of the microgene. During MPR, the primer pairs are reacted under conditions similar to PCR—i.e. a thermal cycle reaction is repeated in the presence of a thermostable archaeal DNA polymerase and dNTP (but without any template DNA). Without the 3'-OH end mismatch, the primer dimer that corresponds to one unit of the microgene would be amplified; including the 3'-OH end mismatch in the MPR primers and using a DNA polymerase having 3'–5' exonuclease activity enables large DNAs consisting of tandem repeats of the microgene to be synthesized. Moreover, nucleotide insertions and deletions randomly occur at end-joining junctions between microgenes, resulting in synthesis of combinatorial libraries of the repetitive sequences from a single microgene. Based on the above mechanism, we used sets of primer pairs that have 7 complementary base pairs in the 3’ region and have mismatch pairs at their 3’-OH ends.
Sequence
loop Tm
Tm
GC
MPR primer 1 (Fwd)
5'-GAAATGTGAGCGCTCACAACT-CGGCGGGGA-3' (31nt)
64 deg C
86 deg C
58%
MPR primer 2 (Rev)
5'-CACAACCTAGGGTCGACTGTGGGCCCCGCCA-3'
56 deg C
72 deg C
68%
Reaction
Set PCR tube into themal cycler.
After it becomes 94°C, add Vent DNA polymerase 2.6 ul (2 units/ul). (Hot start)
Measurement
transformation of yeast
1.Yeast 10^7/ul 1ml
2.centrifuge 5000rpm 3min
3.remove the top of layer
4.add MilliQ 1ml
5.centrifuge 5000rpm 3min
6.1MLioAc 36ul, 2mg/ml sssDNA 25ul, plusmid DNA 3ul, MilliQ 40ul
7.vortex 8.add 50%PEG#4000 240ul
9.vortex 10.RT 30min, 42℃20min
11.centrifuge 5000rpm 1min
12.remove the top of layer
13.add MilliQ 50ul
Evaluation for the performance of Timer Vector
We evaluate Timer Vector by observing the expression of GFP. Timer Vector is inserted into yeast which is transformed by lacI expression vector. Timer Vector has LacI repetitive sequence in the both ends. It protects from the degradation by exonuclease, so GFP is expressed. The repetitive sequence becomes shorter every time a cell divides. At last, the repetitive sequence is completely lost. Timer Vector is degraded by exonuclease, and stop the expression of GFP.
The expression of GFP is observed by fluorescence microscope. We observe the yeast in liquid culture medium on a glass schale. Liquid culture allows us to choose single yeast to focus on. ConcanavalinA is added into the culture medium. This prevents the yeast from diffusing, and yeast forms colonies in culture medium. We use CSM liquid culture medium without Uracil. The yeast we use can’t synthesize Uracil. Timer Vector has Uracil synthetic gene, so only transformed yeast grow.
In the anticipated result, the colony stops the expression of GFP at the almost same time. The repetitive sequence becomes shorter by about 200 bps per cell division, and it is completely lost after about 7-8 times of cell divisions. This leads the suggestion that observed colonies are composed of over 27 yeast.
Fig The transformed  BY782 by 832, which expresses EGFP.
Retrieved from " http://2009.igem.org/Team:Kyoto/GSDD/Experiment "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
