"
Page
Discussion
View source
History
teams
Log in
 
Team:Kyoto/GSDD/Mechanism
From 2009.igem.org
Home
GSDD
Method
Team
Sponsor
Acknowledgment
GSDD
Abstract
Introduction
Mechanism
Goals
Experiment
Results & Discussion
Parts
Modelling
Notebook
Cells in Cells
Introduction
Goals
Method
Experiment
Results & Discussion
Parts
Notebook
Safety Issue
Sequence
Mechanism
Description of the Mechanism and the Process of GSDD
First, we aimed to design a robust gene expression system during duplication. Even if there were any inner cellular components or chemical reactions, the temporal gene expression system should be robust, just only depending on the number of cell division. Thus, we focused on “the End Replication Problem” found in the duplication of eukaryotic cell’s chromosomes (REF, Figure 1). By repeating the duplication process, both ends of each linear chromosome (telomeric sites) becomes shorter (about 100-200bps in yeast). Based on the system, we considered whether we could apply this mechanism for designing our genetic circuits without using telomerase.
Fig.1
At the same time, we found a reference, by chance, in which a designed linear DNA was stabilized in E.coli by the binding of lactose repressor to the both ends of designed repeated binding sequences (REFERENCE). We also found an online article (REFERENCE) about artificially designed chromosome for yeast, which is also stabilized and enabled to pretend like a natural chromosome in yeast (replicated by DNA polymerase and distributed into each daughter cell in each cell division). This chromosome is called “Yeast Artificial Chromosome” (YAC) (Figure 2) and is used by many biologists in yeast gene cloning. YAC consists of three important parts; intermediate replication origin, both ends of telomeric sites, and centromeric site. From replication origin, chromosome replication starts at the time of chromosome replication. By telomeric sites, even if some end base pairs are cut off by the end replication problem, in YAC the chromosome can keep its ends length same by processing them with DNA repair mechanism. Centromere is essential site for exact chromosomal division. It can let the cell exactly distribute double chromosomes into each daughter cell.
Fig.2
Based on the above knowledge, we designed two functionally different vectors; Timer vector and Bomb vector (Figure 3). Timer vector is linear DNA. We designed this to behave like YAC (this means it would behave in the same way with natural chromosome but has a big difference with YAC), and regulate a killing gene on Bomb vector. Bomb vector is circular DNA, and it consists of a killing gene regulated by a protein expressed from Timer vector. Then we determined to use yeast in our experiment.
Fig.3
Timer vector has, at both ends, repetitive sequences of LacI binding sites (TIMER), and has a replication origin and a centromeric site in its intermediate part. In addition, Timer vector has a constitutive LacI generator (which means constitutively active promoter and a yeast kozac sequence and LacI coding sequence). On the other hand, Bomb vector has a LacI regulated promoter and a killing gene with a yeast kozac sequence. We designed these two vectors to be put into the same cell (Double transformation).
After transforming these two vectors in yeast, we assume that the constitutively expressed LacI binds to the both ends of repeated sequences of Timer vector, and protects them from the degradation by exonuclease in cellular cytoplasm. At the same time, the expressed LacI binds to lac promoter (plac) on Bomb vector, and therefore it prevents the expression of killing gene. As a result, while Timer vector is protected from the attack of exonuclease and keeps the constitutive expression of LacI, it can repress the killing gene on Bomb vector. That is, while Timer exists and keeps LacI generator in its intermediate site, the cell can keep itself alive.  However, as cell division proceeds from the first double transformation, the both ends of Timer vector becomes shorter by the end replication problem, and after several divisions, the repetitive sequences are completely lost. At this point, Timer vector is degraded by exonuclease and the LacI generator is also lost. Finally, the cell expresses a killing gene coded in Bomb vector, and kill itself (Figure 4).
Fig.4
Retrieved from " http://2009.igem.org/Team:Kyoto/GSDD/Mechanism "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
