<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Caltech_Project_Conclusions"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Caltech/Project/Conclusions">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Caltech/Project/Conclusions&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Caltech/Project/Conclusions&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Caltech/Project/Conclusions&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Caltech/Project/Conclusions" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Caltech/Project/Conclusions</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><P><TABLE width="70%" align="center"><TBODY><TR heigth="75px"><TD style="border:1px solid black;" align="center" height="45px" onmouseover="this.bgColor='black'; home.style.color='white'" onmouseout="this.bgColor='white'; home.style.color='black'" bgcolor="white"><A id="home" href="https://2014.igem.org/Team:Caltech" style="color:#000000">Home </A></TD><TD style="border:1px solid black;" align="center" height="45px" onmouseover="this.bgColor='black'; team.style.color='white'" onmouseout="this.bgColor='white'; team.style.color='black'" bgcolor="white"><A id="team" href="https://2014.igem.org/Team:Caltech/Team" style="color:#000000"> Team </A></TD><TD style="border:1px solid black;" align="center" height="45px" onmouseover="this.bgColor='black'; profile.style.color='white'" onmouseout="this.bgColor='white'; profile.style.color='black'" bgcolor="white"><A id="profile" href="https://igem.org/Team.cgi?year=2014&amp;team_name=Caltech" style="color:#000000"> Official Team Profile </A></TD><TD style="border:1px solid black;" align="center" height="45px" onmouseover="this.bgColor='black'; project.style.color='white'" onmouseout="this.bgColor='white'; project.style.color='black'" bgcolor="white"><A id="project" href="https://2014.igem.org/Team:Caltech/Project" style="color:#000000"> Project</A></TD><TD style="border:1px solid black;" align="center" height="45px" onmouseover="this.bgColor='black'; parts.style.color='white'" onmouseout="this.bgColor='white'; parts.style.color='black'" bgcolor="white"><A id="parts" href="https://2014.igem.org/Team:Caltech/Parts" style="color:#000000"> Parts</A></TD><TD style="border:1px solid black;" align="center" height="45px" onmouseover="this.bgColor='black'; modeling.style.color='white'" onmouseout="this.bgColor='white'; modeling.style.color='black'" bgcolor="white"><A id="modeling" href="https://2014.igem.org/Team:Caltech/TXTL" style="color:#000000"> TXTL Promoter Characterization</A></TD><TD style="border:1px solid black;" align="center" height="45px" onmouseover="this.bgColor='black'; notebook.style.color='white'" onmouseout="this.bgColor='white'; notebook.style.color='black'" bgcolor="white"><A id="notebook" href="https://2014.igem.org/Team:Caltech/Notebook" style="color:#000000"> Notebook</A></TD><TD style="border:1px solid black;" align="center" height="45px" onmouseover="this.bgColor='black'; safety.style.color='white'" onmouseout="this.bgColor='white'; safety.style.color='black'" bgcolor="white"><A id="safety" href="https://2014.igem.org/Team:Caltech/Safety" style=" color:#000000"> Safety </A></TD><TD style="border:1px solid black;" align="center" height="45px" onmouseover="this.bgColor='black'; attributions.style.color='white'" onmouseout="this.bgColor='white'; attributions.style.color='black'" bgcolor="white"><A id="attributions" href="https://2014.igem.org/Team:Caltech/Attributions" style="color:black"> Attributions</A></TD></TR></TBODY></TABLE><FONT size="+4"><CENTER> Conclusions </CENTER></FONT><A href="https://2014.igem.org/Team:Caltech/Project">Project Overview</A><A href="https://2014.igem.org/Team:Caltech/Project/Details">Project Details</A><A href="https://2014.igem.org/Team:Caltech/Project/Methods and Methods">Materials and Methods</A><A href="https://2014.igem.org/Team:Caltech/Project/Experiments">The Experiments</A><A href="https://2014.igem.org/Team:Caltech/Project/Results">Our Results</A><B><FONT size="+1">Conclusions</FONT></B><A href="https://2014.igem.org/Team:Caltech/Project/References">References</A>
While we were unable to fully construct the system we had planned to build at the beginning of the summer, we have made significant headway and discovered results that future groups and teams will find useful should they choose to conduct similar projects.
</P><H3>Testing for Signaling Ligand Export</H3><B><I>Conclusions</I></B><P>Based on the results of our experiments conducted over the summer, we concluded that the fsrABC quorum sensing system shows the most promise of being able to be implemented successfully in <I>E. coli</I>. However, it cannot yet be definitively confirmed whether or not fsrABC’s export system fully in E. coli, due to the sub-optimal, crooked Western blot and appearance of very large fragments in the supernatant as well. Additional testing, including LC/MS analysis to attempt to detect presence of GBAP, will need to be conducted to determine whether or not the fsr system really works as we hope.</P><P>We also can conclude that the lamBDCA system does not appear to function properly in E. coli. Negative results both in the Western blots at all levels of aTc induction and in the LC/MS analysis indicate that the lam signaling ligand is simply absent in both the supernatant and within the cell, having not been either synthesized or exported by <I>E. coli.</I></P><P>We were also unable to clone constructs with inserts containing the agrB &amp; agrD ligand export genes for the agrBDCA system. As a result, we were unable to determine whether or not that system would be able to successfully export the agr signaling ligand in <I>E. coli</I>.
<B><I>Future Work</I></B></P><P>In order to confirm the findings presented in this paper, further experiments will need to be conducted to verify the presence of GBAP in the supernatant of cells expressing the fsrABC system and, thus, the synthesis and export of GBAP in E. coli. To do this, future work can focus on conducting LC/MS analysis on the supernatant of liquid cultures of cells expressing the fsrABC system in a manner analogous to our verification of the absence of lam signaling molecule in cells expressing the lamBDCA system. Additionally, the Western blots should be repeated in an attempt to ascertain more clearly the fragment sizes and to run more uniformly. Furthermore, to remove the ambiguity regarding the presence in the supernatant of bands corresponding to supposedly membrane-bound proteins, Western blots should be run on the supernatants of liquid cultures of cells expressing known non-exported proteins tagged with a 3xFLAG (e.g. GFP-3xFLAG). The presence or absence of bands for these negative control samples should give a baseline to determine exactly how much of the fluorescence generated by the bands results from proteins actually exported versus those in the background.</P><H3>Testing for Signaling Ligand Reception</H3><P>Issues with cloning of test constructs preventing us from testing the AgrC/AgrA two-component system via scaffold induction. However, we were able to successfully clone a construct with the AgrC gene with 4 SH3 domains attached, and a construct with the AgrA gene along with the P2 promoter regulating a GFP reporter. This was insufficient for testing via scaffold induction. However, these are sufficient to test the two-component system given successful production and export of auto-inducing peptide (AIP), which is the native signaling molecule in <I>S. aureus</I>.  
</P><H3>A Word of Advice for Future iGEM Teams Attempting Gibson Assembly</H3><P>We made many attempts at the beginning of the summer attempting to clone plasmids containing biobrick prefixes &amp; suffixes at the beginning &amp; end of the insert via Gibson Assembly. However, our colony PCRs were never able to verify properly assembled constructs. Instead, gels with lanes containing colony PCRs of colonies that grew on carbenicillin plates were run and ended up showing bands at roughly 100 base pairs. These bands at ~100bp suggest that the vector does not contain any insert, since a resealed vector containing no insert should yield a fragment of roughly that length when PCRed with our checking primers. On examination of the biobrick prefix and suffix, which were used as the &quot;overlap regions&quot; for Gibson assembly, it was indeed found that there was enough sequence homology between the two to suggest significant binding of the prefix to the suffix without incorporating the insert (resealing of the backbone). In particular, the sequence of similarity (the prefix &amp; suffix) had a very high estimated melting temperature of 46.2°C, owing to its rich GC-content. Since the melting temperature is so close to the 50°C used as our incubation temperature for Gibson assembly, it can be concluded that a significant number of backbone ends were &quot;chewed back&quot; by exonucleases in the Gibson reaction and then annealed to each other, recircularizing the backbone without the desired insert.</P><P>From our analysis of gels run with colony PCR product of the comX-mNeonGreen assemblies, we can conclude that attempting to use biobrick prefixes and suffixes as the overlap regions in Gibson assembly will more likely than not result in failure in assembly. The prefix and suffix sequences share too much sequence homology and anneal at too high a temperature to run efficiently the Gibson assembly. We thus recommend any future iGEM teams that wish to use Gibson assembly in construction of their biobrick parts to assemble their plasmid without the biobrick prefix and suffix. From there, after successful cloning of the construct has been verified, their desired insert can be PCRed back out using primers adding the biobrick prefix and suffix as overhangs. This linear fragment can then finally be &quot;biobricked&quot; via ligation into a standard vector backbone using the standard 3A assembly.</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Caltech/Project/Conclusions">http://2014.igem.org/Team:Caltech/Project/Conclusions</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Caltech/Project/Conclusions" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Caltech/Project/Conclusions" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Caltech/Project/Conclusions&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Caltech/Project/Conclusions&amp;oldid=375219" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>