<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Austin_UTexas_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Austin UTexas/Results</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="menu_wrap"><NAV class="menu"><UL class="clearfix"><LI><A href="https://2016.igem.org/Team:Austin_UTexas">HOME </A></LI><LI><A href="">TEAM <SPAN class="arrow">▼</SPAN></A></LI><LI><A href=" https://2016.igem.org/Team:Austin_UTexas/Team"> Team   </A></LI><LI><A href="https://2016.igem.org/Team:Austin_UTexas/Collaborations">★  Collaborations </A></LI></UL><LI><A href="">PROJECT  <SPAN class="arrow">▼</SPAN></A></LI><LI><A href="https://2016.igem.org/Team:Austin_UTexas/Description"> ★  Description </A></LI><LI><A href="https://2016.igem.org/Team:Austin_UTexas/Demonstrate"> ★ Demonstrate </A></LI><LI><A href="https://2016.igem.org/Team:Austin_UTexas/Results"> Results </A></LI><LI><A href="https://2016.igem.org/Team:Austin_UTexas/Protocols"> Protocols </A></LI><LI><A href="https://2016.igem.org/Team:Austin_UTexas/Parts">PARTS </A></LI><LI><A href="https://2016.igem.org/Team:Austin_UTexas/Safety"> SAFETY </A></LI><LI><A href="https://2016.igem.org/Team:Austin_UTexas/Attributions">★  ATTRIBUTIONS </A></LI><LI><A href=""> HUMAN PRACTICES<SPAN class="arrow">▼</SPAN></A></LI><LI><A href="https://2016.igem.org/Team:Austin_UTexas/Human_Practices"> Human Practices         </A></LI><LI><A href="https://2016.igem.org/Team:Austin_UTexas/HP/Silver">★ Silver </A></LI><LI><A href="https://2016.igem.org/Team:Austin_UTexas/HP/Gold">★ Gold </A></LI><LI><A href="https://2016.igem.org/Team:Austin_UTexas/Integrated_Practices"> ★ Integrated Practices </A></LI><LI><A href="https://2016.igem.org/Team:Austin_UTexas/Interlab Study">INTERLAB STUDY </A></LI></NAV></DIV><DIV class="content_wrapper"><DIV class="column full_size_outer" id="top"><DIV class="column_full_size_inner"><H2>Results</H2><P>
Click on one of the images below to learn more about our results!
</P><DIV class="column sixth_size"><A href="#section1"><P>Kombucha Strains </P></A></DIV><DIV class="column sixth_size"><A href="#section2"><P>Conjugation </P></A></DIV><DIV class="column sixth_size"><A href="#section3"><P>Recapitulation</P></A></DIV><DIV class="column sixth_size"><A href="#section4"><P>Ethanol</P></A></DIV><DIV class="column sixth_size"><A href="#section6"><P>pH Sensors</P></A></DIV><DIV class="column sixth_size"><A href="#section7"><P>Gellan Gum</P></A></DIV><DIV class="naviSection" id="sectionKomResult"><H2>Kombucha Strains </H2><P>The first steps in the characterization of microbes native to kombucha involved the isolation of strains from store-bought kombucha samples. This was accomplished by plating various dilutions of kombucha onto a variety of media including YPD, HS, and R2A.  <CENTER><FIGURE><FIGCAPTION><B>Figure 1:</B> Shows YPD plates spread with various dilutions of GT's brand kombucha samples. Credit: Zach Martinez.</FIGCAPTION></FIGURE></CENTER></P><P> Isolated colonies were selected from each &quot;isolation plate&quot; (<B>Figure 1</B>) and continually grown up and streaked out to ensure that the resulting frozen stock was truly axenic. Each newly isolated microbe was designated with a &quot;KOM #&quot; based on the order in which it was isolated (i.e. KOM 01, KOM 02, etc.) to serve as a placeholder name until the species could be identified. In order to begin this identification process, genomic DNA (gDNA) was first isolated from each individual strain. This DNA was then used as the template for two separate PCR reactions targeting either the 16S rRNA gene in bacteria, or the ITS rRNA gene for fungi. PCR products were then run on a 1% agarose gel to observe which reaction yielded product in gel (<B>Figure 2</B>). 
</P><P>
Once it was determined whether each isolate was a bacterium or a fungus, the PCR products were purified and samples of the gDNA was sequenced using Sanger sequencing. The resulting sequences were then run through the Ribosomal Database Project (RDP) SeqMatch tool in order to identify the exact species of bacteria or yeast that correspond to each tested isolate. The identified microbes are listed below in <B>Table 1</B>. 
</P><CENTER><FIGURE><FIGCAPTION><B>Figure 2:</B> Example of a gel obtained after running PCR products. Lane #1 is a 100 bp ladder; Lanes 2-7 are six different kombucha isolates that underwent PCR reactions selecting for bacteria (targeting 16S rRNA gene); Lanes 8-13 are the same six isolates, in the same order, but which underwent reactions selecting for fungi (targeting ITS rRNA gene). Based on the gel results it can be observed that for any given isolate, a product in gel was only observed for either bacterial or fungal primer reactions. Credit: Matthew Hooper.
</FIGCAPTION></FIGURE></CENTER><CENTER><TABLE class="wikitable" style="width: 80%;"><CAPTION>Table 1:Microbes Isolated and Identified from Various Store Bought Kombucha Samples
</CAPTION><TBODY><TR><TH>Species
</TH><TH>Classification
</TH><TH>Brand of Kombucha Isolated From
</TH></TR><TR style="text-align: center;"><TD>Staphylococcus warneri
</TD><TD>Bacteria
</TD><TD>GT’s Kombucha
</TD></TR><TR style="text-align: center;"><TD>Staphylococcus epidermidis
</TD><TD>Bacteria
</TD><TD>GT's Kombucha
</TD></TR><TR style="text-align: center;"><TD>Gluconobacter oxydans*
</TD><TD>Bacteria
</TD><TD>GT’s Kombucha
</TD></TR><TR style="text-align: center;"><TD>Lachancea fermentati*
</TD><TD>Yeast
</TD><TD>Buddha's Brew
</TD></TR><TR style="text-align: center;"><TD>Propionibacterium acnes
</TD><TD>Bacteria
</TD><TD>Buddha's Brew
</TD></TR><TR style="text-align: center;"><TD>Micrococcus luteus
</TD><TD>Bacteria
</TD><TD>Buddha's Brew
</TD></TR><TR style="text-align: center;"><TD>Bacillus pumilus
</TD><TD>Bacteria
</TD><TD>Buddha's Brew
</TD></TR><TR style="text-align: center;"><TD>Saccharomyces cerevisiae
</TD><TD>Yeast
</TD><TD>LIVE Soda Kombucha
</TD></TR><TR style="text-align: center;"><TD>Schizosaccharomyces pombe*
</TD><TD>Yeast
</TD><TD>LIVE Soda Kombucha
</TD></TR></TBODY></TABLE><P>Credit: Matthew Hooper and Zach Martinez.
(*Indicates a species that is considered vital to the production of kombucha)

</P><P>Back to Top</P></CENTER></DIV><DIV class="naviSection" id="section2"><H2>Conjugation</H2><P>We have attempted to conjugate GFP into both <I>G. oxydans</I> and <I>G. hansenii</I> with a Diaminopimelic Acid (DAP) auxotrophic strain of <I> E. coli</I> (The Barrick Lab). The plasmid contains the vector pMMB67EH, the promoter PA-1, GFP and a spectinomycin resistance gene. </P><P>The first conjugation was done with two of our isolated <I>G. oxydans</I> strains, in case the strains might behave differently. First, a mixture between our recipient strain and the DAP auxotroph strain were plated on an LB+DAP agar plate to allow for conjugation to occur. After 24 hours of incubation, we scraped up the growth and plated each conjugation mixture onto a LB+Spec plate (<B>Figure 1</B>). 24-48 hours later, we viewed the potential transconjugants using a fluorescence microscope. We then picked these glowing colonies and streaked them out onto another LB+Spec plate.  We then followed our protocol for genome DNA isolation and 16S sequencing, as described above, to confirm successful conjugation of <I>G. oxydans</I>. After troubleshooting our 16s procedure, we were finally able to obtain a viable sequencing result. However, all of the glowing colonies were identified as <I>E. coli</I>.</P><DIV class="floatleft"><FIGURE><FIGCAPTION><B>Figure 1:</B> These are our potential transconjugants on a LB+DAP plates. The dark reader was used when taking this picture. The top two are <I>G. oxydans</I> while the bottom two are <I>G. hansenii</I>. Credit: Zach Martinez.</FIGCAPTION></FIGURE></DIV><P>For the next round of conjugation, we used a strain of both <I>G. oxydans</I> and  <I>Gluconacetobacter hansenii</I> from the American Type Culture Collection (ATCC).  Our procedure was identical to the one described in the previously paragraph.</P><P> We then picked isolated colonies and streaked them out onto LB+Spec plates. After using 16s sequencing on the potential transconjugants, we encountered an anomaly. Instead of amplifying the 16s gene, we received the sequence of the L,D Transpeptidase gene of <I> E. coli</I>.</P><P>We then decided to perform a Minimum Inhibitory Concentration (MIC) experiment in order to determine if <I>G. oxydans</I> is able to survive antibiotics above the standard <I>E. coli</I> concentration. We tested the antibiotics kanamycin, spectinomycin and carbenicillin.</P><P>Our results in <B> Figure 2</B> showed that <I>G. oxydans</I> is resistant to at least 4x concentrations of spectinomycin (1x = 60µg/mL) and carbenicillin (1x = 100µg/mL). However, a 1x concentration of kanamycin (1x = 50µg/mL) was sufficient to inhibit growth. With this information, we then performed conjugations with <I>E. coli</I> donors that had a kanamycin resistance. These results are still pending through the wiki freeze.

<CENTER><FIGURE><FIGCAPTION><B>Figure 2:</B> These are a Minimum Inhibitory Concentration experiment that included using kanamycin, spectinomycin and carbenicillin with <I>G. oxydans</I> in order to observe if the strain can resist standard concentrations of antibiotics used with <I>E. coli</I>. The top row of cultures are <I>G. oxydans</I> with concentrations of carbenicillin ranging from 4x to none (1x = 100µg/mL). This set of tubes show that <I>G. oxydans</I> is resistant to carbenicillin, at least up to a 4x concentration. The next row is <I>G. oxydans</I> in concentrations of kanamycin from 4x to none (1x = 50µg/mL). This set of reactions demonstrates how <I>G. oxydans</I> is resistant to kanamycin, up to a 1x concentration. The last row of tubes is <I>G. oxydans</I> in concentrations of spectinomycin ranging from 4x to none (1x = 60µg/mL). This last row shows how <I>G. oxydans</I> is resistant to spectinomycin, at least up to a 4x concentration. Credit: Zach Martinez. </FIGCAPTION></FIGURE></CENTER></P><P>Back to Top</P></DIV><DIV class="naviSection" id="section3"><H2>Recapitulation</H2><P> The process of recapitulations took individual microbes, isolated from kombucha, and recombined these to reform kombucha. This process is essential to create a designer beverage: the microbes that have been taken from kombucha and genetically engineered must be able to be put back together to remake kombucha. The process of recapitulation was repeated multiple times in triplicate. Each trial tested a different growing condition or different combinations of microbes for regenerating kombucha.</P><P> The first trials of recapitulations tested the tea media being used to compare the tea that at-home brewers used to see if this would support the microbial growth of isolates (Christensen, 2015). The next trials performed used strictly the microbes isolated from kombucha, the problem with this approach is that not all microbes that occur in kombucha can be isolated away from the symbiotic they exist in, and further they cannot grow on a solid agar plate (Imperial, 2014). Using the previous research on kombucha found in scientific literature, as well as our findings in lab, the most prominent microbes that help to create kombucha were attained (either by isolation from kombucha or by purchasing from a scientific database) to be used in future recapitulations (Dufresne, 1999 and Marsh, 2013). </P><P>After having the essential kombucha microbes, the next set of recapitulations tested what ratios these microbes needed to be in when placed in tea media so they could properly form a SCOBY and recreate kombucha. This process involved multiple trial and error results, but eventually the results included below were found.

<CENTER><FIGURE><FIGCAPTION><B>Figure 1:</B> Shows samples from a series of completed recapitulation trials. A negative control triplicate set contained only tea media and experienced no microbial growth after 16 days. The positive control was inoculated with 600 uL of home-brewed kombucha and possessed distinct pellicle formation after only 2 days, forming a mature pellicle by Day 16. Both experimental trials pictured yielded successful recapitulations. Row 3 shows a set of trials that incorporated only microbes that had been purchased rather than isolated from kombucha itself. Row 4 shows successful recapitulations that contained two different strains of <I>Lachancea fermentati</I> each isolated from kombucha samples, as well as a strain of <I>Gluconobacter oxydans</I> and <I>Gluconacetobacter hansenii</I>. The cellulose pellicle produced in this set of trials is notably darker than the one observed for the purchased microbe strains as well as the positive controls. Credit: Katelyn Corley and Matthew Hooper.</FIGCAPTION></FIGURE><FIGURE><FIGCAPTION><B>Figure 2:</B> Shows samples from a series of completed recapitulation trials. A negative control triplicate set contained only tea media and experienced no microbial growth after 7 days. The positive control was inoculated with 600 uL of home-brewed kombucha and possessed distinct pellicle formation after 4 days, forming a mature pellicle by Day 7. The experimental trials pictured yielded successful recapitulations. Row 3 shows a set of trials that incorporated microbes that had been purchased and microbes that had been isolated from kombucha itself. The successful recapitulations contained two different strains of <I>Lachancea fermentati</I> each isolated from kombucha samples, as well as a strain of <I>Gluconacetobacter hansenii</I>. The cellulose pellicle produced in this set of trials is thick and has multiple carbon dioxide bubbles. Credit: Katelyn Corley and Matthew Hooper.</FIGCAPTION></FIGURE></CENTER></P><P>Back to Top</P></DIV><DIV class="naviSection" id="section4"><H2>Ethanol Reduction</H2><DIV class="floatleft"><FIGURE><FIGCAPTION><B>Figure 1:</B> YPD plates made with pH indicator bromothymol blue. Colonies are various strains of <I>Lachancea fermentati</I> isolated from kombucha in our lab. Carbon dioxide and ethanol form as products of fermentation. The carbon dioxide reacts with water to form carbonic acid, lowering the pH of the plate and changing the color of the pH indicator. More dramatic color changes should correlate to greater ethanol production, but this assay is limited in that a variety of metabolites unrelated to ethanol production could influence pH. Credit: Matthew Hooper.</FIGCAPTION></FIGURE></DIV><P><U>Approach</U></P><P>Two potential ways to reduce ethanol content over the course of the fermentation process are to reduce the rate at which yeast produce ethanol or increase the rate at which bacteria convert the ethanol to acetic acid. The first of these methods is the most direct approach, and was the first method tested. We considered UV mutagenizing <I>Lachancea fermentati</I>, a yeast we isolated from kombucha, and then screening for ethanol production using a pH indicator, bromothymol blue (<B>Figure 1</B>). Bromothymol blue is blue at basic pH, turns green around pH 7, and turns yellow around pH 6, and has been used previously to screen for fermentation rate among <I>Saccharomyces cerevisiae</I> colonies (Robillard, 2007). During anaerobic respiration, both ethanol and CO2 are produced, and CO2 reacts with water to form carbonic acid, lowering the pH of the plate and changing the color of the indicator. A variety of problems with this approach led us to abandon it. It is likely that <I>L. fermentati</I> produce other acidic metabolic products, so pH would not necessarily correspond to amount of ethanol produced. This assay also relies on distinguishing differences in color in the agar to qualitatively determine amounts of ethanol production, a process that would be somewhat subjective. Additionally, the ethanol produced is necessary for the production of acetic acid, so slowing the rate of ethanol production would likely have slowed the fermentation of the beverage and could have thrown off the flavor. For all these reasons, attempting to decrease the rate of ethanol production by <I>L. fermentati</I> does not seem like a good approach to lowering the ethanol content during the fermentation.</P><P>We next considered increasing the rate at which acetic acid bacteria in kombucha convert ethanol to acetic acid. This would increase the amount of ethanol being used up throughout fermentation, ideally lowering the ethanol content. Two enzymes facilitate steps in this pathway (Mamlouk and Gullo, 2013). An alcohol dehydrogenase (PQQ-ADH) enzyme facilitates the conversion of ethanol to acetaldehyde, and a membrane-bound aldehyde dehydrogenase (ALDH) facilitates the conversion of acetaldehyde to acetic acid. In order to increase the rate at which ethanol is converted into acetic acid, we propose using Golden Gate Assembly to create a genetic construct in which expression of PQQ-ADH and ALDH is governed by a Tac-promoter (pTac), a hybrid promoter which is inhibited except in the presence of allolactose.  The construct would be transformed into electrocompetent <I>Escherichia coli</I> and transferred to <I>Gluconacetobacter hansenii</I> via conjugation.</P><DIV class="floatright"><FIGURE><FIGCAPTION><B>Table 1:</B> Results of BLAST search comparing the amino acid sequence for PQQ-ADH in C. testosteroni against similar amino acid sequences in Komagataeibacter xylinus (identical to Ga. hansenii). Line 3 is a close match, and the accession number matches one of the ADH genes found in K. xylinus. Credit: Stratton Georgoulis.</FIGCAPTION></FIGURE></DIV><P><U>Identifying genes of interest</U></P><P>
In order to design a construct increasing expression of PQQ-ADH and ALDH in <I>Ga. hansenii</I>, it was necessary to find the genome of the ATCC strain and identify the coding sequences for these genes. The whole genome shotgun sequence for our organism, ATCC 53582, is published on NCBI (Abbot, 2015) with annotations regarding the functions of specific sequences. Coding sequences are annotated with proposed gene products. Though there are several aldehyde dehydrogenase genes annotated in the genome, there is only one which is described as membrane-bound, matching the description from Mamlouk and Gullo (Mamlouk and Gullo, 2013). There are additionally multiple alcohol dehydrogenases. A known amino acid sequence for a homologous PQQ-ADH in <I>Comamonas testosteroni</I> was compared against sequences in the <I>Ga. hansenii</I> genome using BLAST (<B>Table 1</B>). One ADH enzyme found in the <I>Ga. hansenii</I> genome sequence matches the <I>C. testosteroni</I> sequence with a query cover value of 94% and an E value of 0 (third line of <B>Table 1</B>).</P><P><U>Creation of Golden Gate parts</U></P><DIV class="floatleft"><FIGURE><FIGCAPTION><B>Figure 2:</B> Linear map of the coding sequence for membrane-bound ALDH with a Golden Gate type 3 prefix and suffix. BsmBI and BsaI sites are indicated. The restriction sites at either end are included in the prefix and suffix, but the internal BsaI site must be removed to create a functioning Golden Gate part. Credit: Stratton Georgoulis.</FIGCAPTION></FIGURE><FIGURE><FIGCAPTION><B>Figure 3:</B> Linear map of the coding sequence for PQQ-ADH with a Golden Gate type 3 prefix and suffix. EcoRI, BsmBI, and BsaI sites are indicated. The centermost BsmBI restriction site is in the coding sequence and must be removed to create a functional Golden Gate part. EcoRI is not used in Golden Gate assembly, so those sites do not necessarily need to be removed. Credit: Stratton Georgoulis.</FIGCAPTION></FIGURE></DIV><P>In order to assemble the construct, the coding sequences for the genes of interest must be amplified from the <I>Ga. hansenii</I> genome and edited such that they have the correct Golden Gate overhangs and no internal BsaI or BsmBI restriction sites. The sequences were uploaded to Benchling for analysis and planning. The coding sequence for the membrane-bound ALDH contains a BsaI restriction site near the middle of the gene (<B>Figure 2</B>), and the PQQ-ADH coding sequence contains a BsmBI restriction site near the end of the gene (<B>Figure 3</B>). To eliminate the BsaI site in ALDH, primers were designed that would introduce a point mutation at the restriction site. One set of primers, igem2016_KOM_EtOH_01 and igem2016_KOM_EtOH_02, amplifies the sequence upstream of the restriction site, adding a type 3 Golden Gate prefix and removing the restriction site. Another set, igem2016_KOM_EtOH_03 and igem2016_KOM_EtOH_04, amplifies the region downstream of the restriction site, introducing a mutation to the site and adding a type 3 Golden Gate suffix to the end of the gene. These two products will be used in an overlap PCR reaction to create a final product with no BsaI restriction sites and the correct prefix and suffix for assembly. To remove the BsmBI site from the PQQ-ADH coding sequence, a set of primers (igem2016_KOM_EtOH_05 and igem2016_KOM_EtOH_06) was designed to amplify the region upstream of the restriction site and add a Golden Gate type 3 prefix to the beginning of the sequence. The reverse primer additionally adds a mutation to existing BsmBI restriction site and creates a new BsmBI restriction site that will be used to join the piece to a double-stranded DNA, igem2016_KOM_EtOH_07, containing the rest of the gene’s coding sequence appended with a Golden Gate type 3 suffix. The assembly of the PQQ-ADH part will therefore take place in two reactions: one reaction in which the upstream piece of DNA is created, and one reaction in which it is ligated to the gBlock. <B>Table 2</B> contains more information about each of these oligonucleotides. All were ordered from IDT.</P><CENTER><FIGURE><FIGCAPTION><B>Table 2:</B> Description of oligonucleotides ordered from IDT and their purposes. All of these are PCR primers except for igem2016_KOM_EtOH_07, which is a gBlock containing the end of the PQQ-ADH with a Golden Gate type 3 suffix appended. Credit: Stratton Georgoulis.</FIGCAPTION></FIGURE></CENTER><CENTER><A href="#top"><P>Back to Top</P></A></CENTER></DIV><DIV class="naviSection" id="section6"><H2>pH Sensors</H2><P>During the kombucha brewing process, the beverage becomes more acidic.  Additionally, it is unclear if or how the microbial community changes within the beverage over time.  Thus, our team decided to find pH sensitive promoters  that could be used to track not only the pH of the maturing beverage, but also the presence of the various microbes within the kombucha over time.  We successfully created a neutral range reporter, attempted to create acidic and basic range reporters, and found three putative acidic range reporters that are endogenous to one of our kombucha bacteria, <I>Gluconobacter oxydans</I></P><P>Though an acidic sensor was what was required for our kombucha analysis, the identification of sensors in other areas of the pH spectrum were explored as well. Three sequences were identified, the CadC operon for the acidic range, CpxA-CpxR complex for the neutral range, and the P-atp2 promoter from the BioBrick Registry (<A href="http://parts.igem.org/Part:BBa_K1675021">BBa_K1675021</A>) for the basic range. Each sequence was paired with a unique corresponding reporter sequence so that if each pH sensitive plasmid were in the same environment, the specific pH of the system could be seen. The reporters used were, <A href="http://parts.igem.org/Part:BBa_E1010">BBa_E1010</A> for the CadC construct, <A href="http://parts.igem.org/Part:BBa_K1033916">BBa_K1033916</A> for the CpxA-CpxR complex, and <A href="http://partsregistry.org/Part:BBa_K592009">BBa_K592009</A> for the P-atp2 promoter.</P><H4>CpxA-CpxR</H4><DIV class="floatright"><FIGURE><FIGCAPTION><B>Figure 1:</B>Testing the CpxR Construct in pH 6-9. From left to right is control pH 6-9 and then experimental pH 6-9. These are showing the gradient change in expression accordingly with the change of pH due to a pH-dependent promotor compared to consistent expression accordingly with a promoter that is always &quot;on&quot;. Credit: Sofia Chinea</FIGCAPTION></FIGURE></DIV><P>CpxA-CpxR is a two-component mechanism that is activated at pH 7.4 and repressed at pH 6.0. CpxA is an intermembrane protein that autophosphorylates at a certain external pH, CpxR (a kinase) then gets phosphorylated by CpxA and acts as a transcription factor. This system originally is a transcription factor for the virF gene, but virF was replaced with a reporter. The original sequence was found in <I>Shigella sonnei</I>, but <I>E. coli</I> has a homolog of these proteins so all that is required on the construct is the appropriate prefix/suffix and CpxR binding site (Nakayama and Watanabe, 1995; Nakayama and Watanabe, 1998).

</P><DIV class="floatleft"><FIGURE><FIGCAPTION><B>Figure 2:</B>amajLime expressed in <I>E. coli</I> in liquid LB. Credit: Sofia Chinea</FIGCAPTION></FIGURE></DIV><DIV class="floatright"><FIGURE><FIGCAPTION><B>Figure 3:</B> Normalized fluorescent values from CpxR construct vs control (YGCP). The fluorescence per cell count stayed generally the same throughout the range of pH while the CpxR has a clear increase in fluorescence per cell. Credit: Sofia Chinea</FIGCAPTION></FIGURE></DIV><P>The order from left to right in <B>figure 1</B> is control pH 6-9 and then Experimental pH 6-9. These are showing the gradient change in expression accordingly with the change of pH due to a pH-dependent promotor compared to consistent expression accordingly with a promoter that is always &quot;on&quot;. The main point is that the control at pH 6 has more expression of the yellow-green chromoprotein than the Experimental at pH 6. The pH-dependent promoter of the experimental group is down-regulated at pH 6 whereas the control is not. Also, there is an increase in YGCP expression between the experiment pH 7 and pH 8 that is not seen in the control between pH 7 and pH 8. The normalized data in <B>figure 3</B> shows the relative expression of YGCP. The CpxA-CpxR construct can be found on the iGEM registry as: <A href="“http://parts.igem.org/Part:Bba_K2097000”">BBa_K2097000</A>, while the construct utilized as a control can be found on the iGEM registry as <A href="http://parts.igem.org/Part:BBa_2097002">BBa_K2097002</A> as well as in <B>figure 2</B>.</P><H4>CadC</H4><P>The CadC operon is a native pathway in <I>E. coli</I>, involved in the cadaverine synthesis pathway. The protein CadC protein on the operon is produced and activates segments downstream of the operon on the CadBA receptors. The CadC protein is pH sensitive to an external pH 5.5 and below, as well as lysine dependent. A point mutation on codon 265, in which argenine is converted to cystine, causes the CadC protein to become lysine independent (Kuper and Jung, 2005).</P><P>Despite several attempts, we have been unable to grow the modified CadC operon on pSB1C3 in <I>E. coli</I> suggesting some form of cell toxicity.  Specifically, all plasmids isolated have contained additional point mutations that appear to inactivate the expression of CadC.  Due to this apparent toxicity, no data regarding the lysine-independent CadC could be collected. Alternative acidic range pH sensor candidates have been identified and are discussed below.</P><H4>P-atp2</H4><P>The P-atp2 promoter, native to the bacterium <I>Corynebacterium glutamicum</I> is reportedly induced at pH 7, to pH 9 (<A href="https://2015.igem.org/Team:BIT-China/Parts">BIT-China-2015</A> and <A href="http://parts.igem.org/Part:BBa_K1675021">BBa_K1675021</A>). Utilizing the blue chromoprotein (<A href="http://partsregistry.org/Part:BBa_K592009">BBa_K592009</A>), a test was designed in which a plasmid containing the P-atp2 promoter with the blue chromoprotein was grown alongside an <I>E. coli</I> line that contained a plasmid with just the blue chromoprotein. We expected to see constant blue chromoprotein production in the control series (those that lacked P-atp2) and a visual increase in blue chromoprotein as the pH was raised from 6 to 9 in the cells that contained the P-atp2 construct. The construct utilized as a control can be found on the iGEM registry <A href="http://parts.igem.org/Part:BBa_2097001">BBa_K2097001</A> as as in <B>figure 4</B>.</P><P>However, as seen in <B>figure 5</B>, no clear change in color expression appears in the experimental trials, suggesting a lack of sensitivity of the P-atp2 promoter.</P></DIV><DIV class="floatleft"><FIGURE><FIGCAPTION><B>Figure 4:</B> amilCP expressed in <I> E. coli </I> and in liquid LB. Credit: Riya Sreenivasan</FIGCAPTION></FIGURE></DIV><H4>GOX Sequences as Putative Promoters</H4><P>Three endogenous upstream regions of loci on the <I>Gluconobacter oxydans</I>  chromosome were reported to show increased mRNA synthesis as pH decreased, were isolated and obtained, as seen in table 1 (Hanke, et al., 2012). Using Golden Gate assembly, these putative promoters have been placed on the Golden Gate entry vector pYTK001 for later use. By utilizing these pH-sensitive promoters with different reporters and transforming them into multiple organisms in kombucha, the visualization of the microbes and their location in kombucha would be possible (Lee, et al., 2015). This will serve as a stepping stone into further understanding how the microbiome of kombucha changes as it brews as well as determining organism concentration specific times during the brewing process.</P><CENTER><FIGURE><FIGCAPTION><B>Figure 5</B>. Spun down P-atp2 constructs compared to controls in pH6-9. There is no clear gradient change in color expression. Credit: Ian Overman and Alex Alario </FIGCAPTION></FIGURE><TABLE class="wikitable" style="width: 80%;"><CAPTION>Table 1:The Three Endogenous GOX Sequences
</CAPTION><TBODY><TR><TH>Locus Tag
</TH><TH>Predicted Functions
</TH><TH>mRNA ratio pH4/pH6
</TH></TR><TR style="text-align: center;"><TD>GOX0647
</TD><TD>Putative exporter protein, ArAE family
</TD><TD>12.91
</TD></TR><TR style="text-align: center;"><TD>GOX0890
</TD><TD>Hypothetical protein GOX0890
</TD><TD>4.93
</TD></TR><TR style="text-align: center;"><TD>GOX1841
</TD><TD>Hypothetical protein GOX1841
</TD><TD>3.36
</TD></TR></TBODY></TABLE><P>Back to Top</P></CENTER></DIV><DIV class="naviSection" id="section7"><H2>Gellan Gum </H2><DIV class="floatleft"><FIGURE><FIGCAPTION><B>Figure 1:</B> Our plate protocol developed from multiple references and trial and error. The bacteria are first inoculated into rich media for cell multiplication over a span 24 hours before being inoculated with a 1 to 10 dilution into Gellan-production minimal media to maximize the concentration of Gellan in the culture. After 48-96 hours, the culture is then autoclaved or microwaved. Immediately after sterilization, concentrated, sterilized media is added and the plates are poured before the Gellan can solidify. After 5-10 minutes, the plates can be streaked with microbes and placed in an incubator. Credit: Jenna McGuffey</FIGCAPTION></FIGURE></DIV><P>
The do-it-yourself (DIY) movement is focused on making science more accessible to the public. Because many consumers brew their own kombucha, we have developed a set of DIY instructions that would allow an average person to analyze their home-brew and identify their kombucha’s species outside of a lab setting. This procedure is possible because of Gellan Gum, produced by the halobacterium Sphingomonas pauci-mobilis.
</P><DIV style="clear:both;"><DIV class="floatright"><FIGURE><FIGCAPTION><B>Figure 2:</B> An LB-Gellan plate streaked with E. coli and incubated for 24 hours at 37°C. Credit: Jenna McGuffey</FIGCAPTION></FIGURE><FIGURE><FIGCAPTION><B>Figure 3:</B>  A YPD-Gellan plate streaked with S. cerevisiae and incubated for 48 hours at 30°C. Credit: Jenna McGuffey</FIGCAPTION></FIGURE></DIV><P>
Gellan Gum is produced through aerobic fermentation (Kang et al. 1982). This exopolysaccharide is a “high-molecular-mass, anionic polysaccharide which consists of a tetrasaccharide structure with 20% glucuronic acid, 60%glucose, and 20% rhamnose” (Wang et. al. 2006). The advantages of using Gellan in place of agar include: requiring half of the quantity, a consistent production, more clarity than agar, a reduced plate preparation time along with a faster setting time, stability at high temperatures, and lack of contamination factors found in agar that are toxic to some organisms (Ioannis et. al. 2007).
</P><P>
This protocol involves creating media from every-day grocery store items, autoclaving or microwaving a culture of S. paucimobilis for sterilization, then pouring them into plates which can be streaked with microbes. <B>Figure 1</B> denotes each of the steps of the protocol. An S. paucimobilis colony is initially grown in conditions that focus the cell’s metabolism on multiplication. This maximizes the number of cells producing Gellan when inoculated into minimal media (Wu et. al. 2014).
</P><P>
We tested this protocol by growing Escherichia coli on LB-Gellan plates and by growing Saccharomyces cerevisiae on YPD-Gellan plates. Both microbes successfully grew on Gellan plates as in <B>Figure 2</B> and <B>Figure 3</B>. We then confirmed the viability of Gellan-tea media plates to be used for kombucha by streaking various strains that we isolated from Kombucha.
</P><P>
In addition to being usable in this DIY procedure, Gellan Gum is environmentally and socially conscious. Currently there is an agar shortage in the world, because of several threats to its origin, red algae (Calloway 2015). 
</P></DIV><DIV style="clear:both;"><CENTER><A href="#top"><P>Back to Top</P></A></CENTER><H3>References</H3><OL type="1"><LI><A href="http://barricklab.org/twiki/bin/view/Lab/ProtocolsConjugation"> The Barrick Lab Conjugation Protocol</A></LI><LI>Abbot, J. Komagataeibacter xylinus isolate ATCC53582 genome assembly, contig: 
ATCC53582_Chromosome, whole genome shotgun sequence. 2015. Accessed from NCBI website.</LI><LI><A href="https://2015.igem.org/Team:BIT-China/Parts">BIT-China-2015</A></LI><LI>Calloway, Ewen. (2015) Lab staple agar hit by seaweed shortage. <I>Nature</I>.</LI><LI>Christensen, Emma. &quot;How To Make Kombucha Tea at Home - Cooking Lessons from The Kitchn.&quot; The Kitchn. AT Media, 05 Apr. 2015. Web. 06 Oct. 201</LI><LI>Dufresne, C., and E. Farnworth. &quot;Tea, Kombucha, and Health: A Review.&quot; Food Research International. Elsevier, 1 Dec. 1999. Web. 14 Sept. 2016</LI><LI>Hanke, T., Richhardt, J., Polen, T., Sahm, H., Bringer, S., and Bott, M. (2012) Influence of oxygen limitation, absence of the cytochrome bc1 complex and low pH on global gene expression in Gluconobacter oxydans 621H using DNA microarray technology. <I>Journal of Biotechnology 157</I>, 359–372.</LI><LI>Imperial. &quot;Project.&quot; Aqualose. Imperial College London, 2014. Web. 06 Oct. 201</LI><LI>Ioannis Giavasis et al. (2000) Gellan Gum <I>Critical Reviews in Biotechnology.</I>, 20.3: 177-211</LI><LI>Kang, Kenneth S. et al. (1982) Agar-Like Polysaccharide Produced by a Pseudomonas Species: Production and Basic Properties. <I>Applied and Environmental Microbiology.</I>, 1086-1091</LI><LI>Kuper, C., and Jung, K. (2005) CadC-mediated activation of the cadBA promoter in Escherichia coli. <I>Journal of Molecular and Microbiological Biotechnology 1</I>, 26–39.</LI><LI>Lee ME, DeLoache, WC A, Cervantes B, Dueber, JE. (2015) A Highly-characterized Yeast Toolkit for Modular, Multi-part Assembly. <I>ACS Synthetic Biology 4</I> 975-986</LI><LI>Mamlouk, Y. and M. Gullo. Acetic Acid Bacteria: Physiology and carbon sources oxidation. 2013. Indian Journal of Microbiology 53 (4): 337-384.</LI><LI>Marsh, Alan, Orla O’Sullivan, R. Ross, and Paul Cotter. &quot;Sequence-based Analysis of the Bacterial and Fungal Compositions Ofmultiple Kombucha (tea Fungus) Samples.&quot; Elsevier (2013): 171-78. Food Microbiology. Web. 15 Sept. 2016.</LI><LI>Nakayama, S.-I., and Watanabe, H. (1998) Identification of cpxR as a Positive Regulator Essential for Expression of the Shigella sonnei virF Gene. <I>Journal of Bacteriology 180</I>, 3522–3528.</LI><LI>Nakayama, S.-I., and Watanabe, H. (1995) Involvement of cpxA, a Sensor of a Two-Component Regulatory System, in the pH-Dependent Regulation of Expression of Shigella sonnei virF Gene. <I>Journal of Bacteriology 177</I>, 5062–5069.</LI><LI>Robillard, R. A microbial breathalyzer: design of a colorimetric assay for the detection and quantification of ethanol production in microbes. 2007. Major qualifying project for a B.S. degree from Worcester Polytechnic Institute.</LI><LI>Wang, Xia, et al. (2006) Modeling for Gellan Gum Production by Sphingomonas paucimobilis ATCC 31461 in a Simplified Medium. <I>Applied and Environmental Microbiology</I>, 3367-3374</LI><LI>Wu et. al. (2014) Yellow pigments generation deficient Sphingomonas strain and application thereof in Gellan Gum. <I>US Patent 8,685,698</I>.</LI><P>Back to Top</P></OL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>