Team:Austin UTexas/Experiments
HOME
TEAM ▼
Team
★  Collaborations
PROJECT ▼
★  Description
★ Demonstrate
Results
Protocols
PARTS
SAFETY
★  ATTRIBUTIONS
HUMAN PRACTICES▼
Human Practices
★ Silver
★ Gold
★ Integrated Practices
INTERLAB STUDY
Austin_UTexas
Ongoing Experiments
Sequencing
By plating store-bought kombuchas onto a variety of media plates, we were able to isolate several microbial colonies. These colonies were harvested and made into frozen stocks for future use. Fresh cultures were innoculated with these isolated strains and allowed to grow until turbid. Using a genomic DNA (gDNA) isolation kit (Invitrogen), DNA was extracted from these unknown microbes. Two PCR reactions were performed for each microbe using the appropriate isolated gDNA as a template and bacterial primers (selects for bacteria and targets the 16s rRNA gene) or fungal primers (selects for fungi and targets ITS rRNA gene). The purpose of these reactions was to determine whether each isolated species was bacterial or fungal. The PCR products were run on a 1% agarose gel and those that yielded products in the gel were PCR purified using BioBasic and sent to UT's ICMB Sequencing Core Facility for sequencing (using Sanger sequencing). The resulting sequences were identified using the Ribosomal Database Project.
Conjugation
Ethanol Screening
Different strains of L. fermentati, a yeast isolated from kombucha, were plated onto YPD plates containing bromothymol blue. Additionally, S. cerevisiae with ADH (ethanol producing gene) knocked out was plated using these plates. Bromothymol blue changes color according to acidity, which was used as a proxy for measuring ethanol production by these different strains. This method of ethanol screening was ultimately abandoned because of its imprecise nature and out inexperience with genetically modifying eukaryotes. We are now exploring the possibility of modifying the ADH and ALDH genes in Ga. hansenii to have a higher rate of conversion of ethanol to acetic acid.
Recapitulation
pH Detection
Moving away from the tested BioBrick promoters, the pH angle has changed to focus on the organism Gluconobacter oxydans and how native sequences to this organism may lead to better integration with a pH detector. As detailed in our results section, three upstream regions of three difference loci in G. oxydans have been identified, made to be Golden Gate compatible, and placed into the entry vector pYTK001. Once these sequences have been confirmed to be in the entry vector, a larger scale Golden Gate reaction will take place where the entry vectors will be ligated with a reporter and other parts necessary for a successful Golden Gate plasmid. This plasmid will then be tested in E. coli while the confirmation of G. oxydans modification is doable.
Looking Forward
Brazzein
We will modify one of our chassis organisms to produce Brazzein, a sweet-tasting protein naturally produced by the Arabadopsis plant, in order to sweeten the taste of our kombucha.
