Team:Austin UTexas/Description
HOME
TEAM ▼
Team
★  Collaborations
PROJECT ▼
★  Description
★ Demonstrate
Results
Protocols
PARTS
SAFETY
★  ATTRIBUTIONS
HUMAN PRACTICES▼
Human Practices
★ Silver
★ Gold
★ Integrated Practices
INTERLAB STUDY
Project Description
Gold Medal Part Characterization
The characterization of the BioBrick P-atp2 from the BIT-China-2015 team was done to see if P-atp2 could be utilized as a basic pH sensor. The results are found here and on the iGEM Registry page under experience, BBa_K1675021
Our Project
Kombucha is a beverage made when a symbiotic community of bacteria and yeast ferments sugared tea. Although kombucha has been consumed for thousands of years in the East, the drink has enjoyed a recent surge in popularity. 1 Several kombucha breweries operate in Austin, Texas, our team’s hometown. The role microbes play in the production of the beverage has led our team to wonder if synthetic biology could allow us to create “designer kombucha” with enhanced properties, such as more appealing flavors or additional nutrients. In order to do so, our team attempted to isolate the strains responsible for the fermentation of kombucha, identify them, genetically modify them, and add the individual strains into tea media to recreate the drink. We additionally considered potential applications of the ability to genetically modify the microbial population of kombucha, such as reducing the ethanol content of the beverage and improving taste with brazzein, a sweet-tasting protein. In consideration of Human Practices, we reached out to the Austin kombucha community to learn more about what kombucha brewers and consumers would want in a customizable kombucha. Through this interaction, we learned that many kombucha consumers and manufacturers value the traditional, all-natural process of producing the beverage, and that many in the industry would be apprehensive of kombucha made with genetically modified organisms. Though we hope increased public awareness of synthetic biology may someday make a genetically modified kombucha marketable, the current attitudes of kombucha consumers have led us to consider methods of creating designer kombucha that rely only on natural genetic variation.
Click the images below to learn more about our project!
Strains
Conjugation
Recapitulation
Ethanol
Brazzein
pH Sensors
Gellan Gum
Kombucha Strains
One of the earliest goals of our project was to identify a specific set of microbes responsible for the production of kombucha. To do this, samples of store-bought kombucha were plated onto a variety of media with various dilutions to isolate microbes. Then, morphologically different colonies were cultured and frozen in glycerol for further use. Once we obtained a collection of microbial isolates, each microbe was sequenced and identified using polymerase chain reaction (PCR) to amplify a particular ribosomal RNA gene. The 16S gene was selected for bacterial strains, and the ITS gene was amplified for the fungal samples.1 After the samples were sequenced, we utilized the Ribosomal Database Project (RDP) SeqMatch Tool to identify our isolated species of bacteria and yeast. By identifying these kombucha strains, we were able to use our own experimentally isolated strains for our future kombucha experiments.
References
Marsh, A. J., O'Sullivan, O., Hill, C., Ross, R. P., and Cotter, P. D. (2014) Sequence-based analysis of the bacterial and fungal compositions of multiple kombucha (tea fungus) samples. Food Microbiology.
Conjugation
In order to demonstrate that we can genetically engineer the bacterial strains that we identified, Gluconobacter oxydans and Gluconacetobacter xylinus, we attempted to conjugate various plasmids encoding fluorescent devices, such as GFP and E2 Crimson, into these bacteria using a DAP (Diaminopimelic Acid) auxotroph strain of E. coli.1 To assist in this process, we also conducted minimal inhibitory concentration studies with each of these bacteria using spectinomycin, kanamycin and carbenicillin.  Ultimately, we determined that G. oxydans is able to survive the standard E. coli antibiotic concentrations we have been using for both spectinomycin and carbenicillin. However, G. oxydans was successfully inhibited by the normal amount of kanamycin. With this data, we can improve our conjugations by either using more concentrated amounts of spectinomyin and carbenicillin, or only using donor strains with kanamycin resistance. The plasmids pBTK518, pBTK519 and pBTK520 (given to us by Jiri Perutka of The Ellington Lab) contain GFP and either a carbenicillin, kanamycin or a spectinomycin resistance gene respectively. Once the bacteria are successfully conjugated, we can introduce other constructs into G. oxydans in order to create a designer beverage.
References
The Barrick Lab Conjugation Protocol
Recapitulation
One of the primary focuses of our project is to study the nature of the symbiotic community of fermenting kombucha. Recapitulation refers to the reformation of kombucha by singly adding isolated microbes to a mixture of black tea, sucrose, and water similar to the mixture used in home-brewing practices. Through this process we determined that the microbial community could be recreated from its constituent bacteria and yeast. We also identified the microbes that appear to be vital for the proper recapitulation of kombucha. However, because we cannot taste our lab-brewed kombucha, these conclusions are solely based on qualitative observations. Successful recapitulations indicate that it is in fact possible to produce kombucha with known microbes rather then simply propagating new kombucha from a previous batch. These results also elucidate the symbiotic relationships that must exist in order for kombucha to form. Future research may allow us to create kombucha with distinct flavor profiles by varying the combination of strains added to the brew.  Additionally, based on our conversations with Greg Goodman of KTonic (see human practices) these non-GMO containing beverages would likely be most appealing to the industry.
Ethanol
During the fermentation process, yeast in kombucha produce ethanol, the type of alcohol present in beer, wine, and other alcoholic beverages. This presents a challenge to kombucha brewers who wish to market their product as a non-alcoholic beverage. If the alcohol content of a manufacturer’s kombucha exceeds 0.5% at any point during production, the manufacturer may not market their beverage as non-alcoholic and must be regulated as a producer of alcoholic beverages.1 Most kombucha on store shelves contains live yeast that continue to produce ethanol before being purchased by the consumer. Through our interactions with the kombucha industry in Austin, we learned that the expiration date of kombucha is often limited to the length of time the kombucha can be stored without exceeding 0.5% ethanol. Therefore, finding a way to slow ethanol production would allow brewers to keep their product on store shelves for longer. One way to tackle this problem with synthetic biology is to ferment with yeast that produce less ethanol. However, this may be impractical because ethanol must be present for the bacteria in the brew to oxidize it to acetic acid, a major component of the beverage’s tart, distinctive flavor.
Another approach is to increase the rate at which the bacteria convert the ethanol to acetic acid. Two enzymes are responsible for this process: an alcohol dehydrogenase and an aldehyde dehydrogenase.2 Using Golden Gate assembly, we plan to assemble a construct containing the coding sequences for these genes and insert the construct into Gluconacetobacter hansenii, an acetic acid-producing bacterium similar to those found in kombucha. Then, we plan to compare the ethanol content of kombucha cultures fermented with either the modified or unmodified bacteria using gas chromatography-mass spectrometry. We also plan to determine whether increasing the acetic acid production will lead to a pH change that could affect the flavor of the beverage by testing the pH and observing the cultures for visible differences. If we are able to create a microbial community that results in a lower ethanol content within the kombucha during fermentation, kombucha brewers could use the modified bacterium to help ensure the ethanol content of their product stays below the legal limit for longer. Even though there exists a climate of apprehension surrounding genetically modified organisms in the kombucha industry, using such a bacterium could give a brewer a distinct commercial advantage.
To learn more about how our interaction with the kombucha industry affected our project, see our Human Practices page.
References
Alcohol and Tobacco Tax and Trade Bureau. Kombucha Information and Resources. 2016. https://www.ttb.gov/kombucha/kombucha-general.shtml
Mamlouk, D., and Gullo, M. (2013) Acetic Acid Bacteria: Physiology and Carbon Sources Oxidation. Indian Journal of Microbiology 53, 377–384.
Brazzein
One of the potential methods to create designer kombucha is to add a brazzein gene into the bacterial strains. Brazzein, a protein found in the pulp of the edible fruit of the African plant Pentadiplandra brazzeana Baill, is an extremely sweet substance1. It is 2,000 times sweeter than sucrose by weight. This makes it a healthy and economical alternative to sugar.  Commercial production of brazzein is limited, however, because it comes from a tropical plant. If it could be more easily harvested, it could be used to improve the flavor of various foods and drinks, including kombucha. By genetically engineering the brazzein gene into the bacteria in kombucha, the drink could be sweetened without adding sugar or excessive calories.  While still being a GMO product, this beverage would be low in sugar and could appeal to a health-conscious consumer.
References
Yan, Sen et al. “Expression of Plant Sweet Protein Brazzein in the Milk of Transgenic Mice.” Ed. Xiao-Jiang Li. PLoS ONE 8.10 (2013): e76769.
Brazzein protein structure acquired from European Bioinformatics Institute
pH Sensors
Many of the microorganisms involved in the fermentation of kombucha produce acidic metabolites that lower the pH of the culture. Using pH-sensitive promoters to control the expression of reporter proteins, such as GFP or a chromoprotein, would allow visualization of the pH change. The promoters Cpx, P-atp2, and Cadc were selected as pH sensors to indicate pH in the neutral, basic, and acidic ranges, respectively.1,3,5,6 These constructs have been or will be transformed into Escherichia coli to confirm pH sensitivity prior to introduction to kombucha and to see if these constructs could be utilized as sensors in mediums besides kombucha.
Modification of Gluconobacter oxydans, a bacterium in kombucha, is also planned to avoid disturbing the kombucha microbiome. Three endogenous upstream regions of loci that were reported to show increased mRNA synthesis as pH decreased were obtained.2 Golden Gate assembly is currently being used to quickly assemble these promoters upstream of Venus (pYTK033).4 Once successful, these pH-sensitive promoters with different reporters will be used to visualize the different members of the kombucha microbiome overtime.
References
BIT-China-2015
Hanke, T., Richhardt, J., Polen, T., Sahm, H., Bringer, S., and Bott, M. (2012) Influence of oxygen limitation, absence of the cytochrome bc1 complex and low pH on global gene expression in Gluconobacter oxydans 621H using DNA microarray technology. Journal of Biotechnology 157, 359–372.
Kuper, C., and Jung, K. (2005) CadC-mediated activation of the cadBA promoter in Escherichia coli. Journal of Molecular and Microbiological Biotechnology 1, 26–39.
Lee ME, DeLoache, WC A, Cervantes B, Dueber, JE. (2015) A Highly-characterized Yeast Toolkit for Modular, Multi-part Assembly. ACS Synthetic Biology 4 975-986
Nakayama, S.-I., and Watanabe, H. (1998) Identification of cpxR as a Positive Regulator Essential for Expression of the Shigella sonnei virF Gene. Journal of Bacteriology 180, 3522–3528.
Nakayama, S.-I., and Watanabe, H. (1995) Involvement of cpxA, a Sensor of a Two-Component Regulatory System, in the pH-Dependent Regulation of Expression of Shigella sonnei virF Gene. Journal of Bacteriology 177, 5062–5069.
Gellan Gum
The do-it-yourself (DIY) movement is focused on making science more accessible to the public. Because many consumers brew their own kombucha, we have developed a set of DIY instructions that would allow an average person to analyze their home-brew and identify their kombucha’s species outside of a lab setting. This procedure is possible because of Gellan Gum, produced by the halobacterium Sphingomonas pauci-mobilis.
