Team:Heidelberg/Description
NAVIGATION
People
Team
Attributions
Sponsoring
For Judges
Giant Jamboree
Project
Overview
Background & Design
PACE
PREDCEL
Results Summary
Software Validation
Organosilicons
Protein Interaction
Cytochrome Engineering
Optogenetics
CRISPR Cas9
InterLab Study
Achievements
Parts
Overview
Basic Parts
Composite Parts
Improved Part
Parts Collection
Part List
RFC
Modeling
Overview
Phage Titer
Mutagenesis Induction
Mutation Rate Estimation
Lagoon Contamination
Medium Consumption
Interactive Web Tools
Software
AiGEM Overview
DeeProtein
GAIA
SafetyNet
MAWS 2.0
Human Practices
Overview
Integrated Human Practices
Expert Interviews
Safety & Security
Public Engagement
Education
Collaborations
Notebook
Overview
Methods
Materials
Notebook Database
Toolbox Guide
iGEM TEAM HEIDELBERG 2017
COLLAPSE NAVIGATION
People
Team
Attributions
Sponsoring
For Judges
Giant Jamboree
Project
Overview
Background & Design
PACE
PREDCEL
Results Summary
Software Validation
Organosilicons
Protein Interaction
Cytochrome Engineering
Optogenetics
CRISPR Cas9
InterLab Study
Achievements
Parts
Overview
Basic Parts
Composite Parts
Improved Part
Parts Collection
Part List
RFC
Modeling
Overview
Phage Titer
Mutagenesis Induction
Mutation Rate Estimation
Lagoon Contamination
Medium Consumption
Interactive Web Tools
Software
AiGEM Overview
DeeProtein
GAIA
SafetyNet
MAWS 2.0
Human Practices
Overview
Integrated Human Practices
Expert Interviews
Safety & Security
Public Engagement
Education
Collaborations
Notebook
Overview
Methods
Materials
Notebook Database
Toolbox Guide
image/svg+xml
1
0
0
0
1
1
0
1
1
1
0
0
0
1
0
1
0
1
0
1
1
1
0
1
0
1
0
0
1
0
1
Project Abstract
Darwinian evolution is an enormously powerful concept that drove biology towards astonishing complexity and beauty. This year, the iGEM team Heidelberg aims at harnessing this power to accelerate the engineering of biomolecules for human benefit.
To this end we build upon the PACE (phage-assisted continuous evolution) method, which couples the survival of quickly mutating phages encoding a gene of interest to directed selection within E. coli host. We present a standardized, comprehensive evolution toolbox that highly simplifies the complex PACE method and widely expands its utility towards various new application areas, including the biological production of organosilicons .
A central component of our toolbox is an innovative workflow for directed in vivo evolution of novel enzymes, which we validated by redirecting the activity of a promiscuous cytochrome towards a naturally unfavored reaction product.
Complementary, to simplify the generation desired functionalities de novo, we created AiGEM , our Artificial Intelligence for Genetic Evolution Mimicking software suite. The heart of our software is DeeProtein , a deep neuronal network trained on ~10 million protein sequences and able to infer sequence-function relationships from raw sequence data with high accuracy. By interfacing DeeProtein with our genetic algorithm GAIA , we – for the first time - established a fully closed, in silico evolution cycle driven by an intelligent network. We used AiGEM to successfully modulate the catalytic efficiency of beta-lactamases , thereby even producing variants which highly outperform the wild type enzyme.
To demonstrate AiGEM’s ability for generating functionality de novo, we created a novel, highly efficient beta-galactosidase purely by in silico evolution of a beta-glucuronidase parental sequence.
Finally, as part of our integrated human practices project, we implemented SafetyNet , a DeeProtein-based web application safeguarding directed evolution experiments.
Taken together, we provide a new foundational advance by introducing an innovative in vivo and in silico evolution interface as novel engineering paradigm to Synthetic Biology!
Quote
“”
Useful Links
Special pages
Main Page
Follow us on
&
Contact us
iGEM-Heidelberg2017
@bioquant.uni-heidelberg.de
Im Neuenheimer Feld 267
69120 Heidelberg
