<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Heidelberg_Protein_Interaction skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Heidelberg/Protein Interaction</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container"><DIV class="navbar-header"><A type="button" data-canvas="body" id="sidebarCollapse" class="navbar-toggle" data-toggle="offcanvas" data-target="#sidebar" style="color: #393939;margin-top: 9px;">NAVIGATION
                    </A></DIV><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav pull-right"><LI class="dropdown"><A href="https://2017.igem.org/Team:Heidelberg/Team">People </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Team">Team</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Attributions">Attributions</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Sponsoring">Sponsoring</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/For_Judges">For Judges</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Jamboree">Giant Jamboree</A></LI></UL><LI class="dropdown"><A href="https://2017.igem.org/Team:Heidelberg/Description">Project </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Description">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Design">Background &amp; Design </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Pace">PACE</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Predcel">PREDCEL</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Results">Results Summary</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Validation">Software Validation</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Organosilicons">Organosilicons</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Protein_Interaction">Protein Interaction</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Cytochrome_Engineering">Cytochrome Engineering</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Optogenetics">Optogenetics</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/CRISPR">CRISPR Cas9</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/InterLab">InterLab Study</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Achievements">Achievements</A></LI><LI class="dropdown"><A href="https://2017.igem.org/Team:Heidelberg/Parts">Parts </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Basic_Part">Basic Parts</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Composite_Part">Composite Parts</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Improved_Part">Improved Part</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Part_Collection">Parts Collection</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Part_List">Part List</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/RFC">RFC</A></LI><LI class="dropdown"><A href="https://2017.igem.org/Team:Heidelberg/Model">Modeling </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Phage_Titer">Phage Titer</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Mutagenesis_Induction">Mutagenesis Induction</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Mutation_Rate_Estimation">Mutation Rate Estimation</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Lagoon_Contamination">Lagoon Contamination</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Medium_Consumption">Medium Consumption</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Tools">Interactive Web Tools</A></LI><LI class="dropdown"><A href="https://2017.igem.org/Team:Heidelberg/Software">Software </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software">AiGEM Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software/DeeProtein">DeeProtein</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software/GAIA">GAIA</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software/SafetyNet">SafetyNet</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software/MAWS">MAWS 2.0</A></LI><LI class="dropdown"><A href="https://2017.igem.org/Team:Heidelberg/Human_Practices">Human Practices </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Human_Practices">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/HP/Gold_Integrated">Integrated Human Practices</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Interviews">Expert Interviews</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Safety">Safety &amp; Security</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Engagement">Public Engagement</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Education">Education</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Collaborations">Collaborations</A></LI><LI class="dropdown"><A href="https://2017.igem.org/Team:Heidelberg/Notebook">Notebook </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Notebook">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Experiments">Methods</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Materials">Materials</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Database">Notebook Database</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Toolbox">Toolbox Guide</A></LI></DIV></DIV><NAV id="sidebar" style="z-index: 10001; background-color: white;border: none;box-shadow:1px 0px 0px #393939;" class="navmenu navmenu-default navmenu-fixed-left offcanvas" role="navigation"><A class="navmenu-brand" href="https://2017.igem.org/Team:Heidelberg"> iGEM TEAM HEIDELBERG 2017</A><DIV id="sidebarHide" data-toggle="offcanvas" data-target="#sidebar" style="padding-left: 20px;"><A style="font-family: 'Josefin Sans' !important; font-size: 16px; line-height: 18px; height: 18px;"> COLLAPSE NAVIGATION </A></DIV><UL class="nav navmenu-nav" id="sidenav"><LI><A data-toggle="collapse" data-target="#People_side">People </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Team">Team</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Attributions">Attributions</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Sponsoring">Sponsoring</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/For_Judges">For Judges</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Jamboree">Giant Jamboree</A></LI></UL><LI><A data-toggle="collapse" data-target="#Project_side">Project </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Description">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Design">Background &amp; Design </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Pace">PACE</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Predcel">PREDCEL</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Results">Results Summary</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Validation">Software Validation</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Organosilicons">Organosilicons</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Protein_Interaction">Protein Interaction</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Cytochrome_Engineering">Cytochrome Engineering</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Optogenetics">Optogenetics</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/CRISPR">CRISPR Cas9</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/InterLab">InterLab Study</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Achievements">Achievements</A></LI><LI><A data-toggle="collapse" data-target="#Parts_side">Parts </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Basic_Part">Basic Parts</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Composite_Part">Composite Parts</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Improved_Part">Improved Part</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Part_Collection">Parts Collection</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Part_List">Part List</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/RFC">RFC</A></LI><LI><A data-toggle="collapse" data-target="#Model_side">Modeling </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Phage_Titer">Phage Titer</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Mutagenesis_Induction">Mutagenesis Induction</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Mutation_Rate_Estimation">Mutation Rate Estimation</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Lagoon_Contamination">Lagoon Contamination</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Medium_Consumption">Medium Consumption</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Tools">Interactive Web Tools</A></LI><LI><A data-toggle="collapse" data-target="#AI_side">Software </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software">AiGEM Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software/DeeProtein">DeeProtein</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software/GAIA">GAIA</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software/SafetyNet">SafetyNet</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software/MAWS">MAWS 2.0</A></LI><LI><A data-toggle="collapse" data-target="#HP_side">Human Practices </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Human_Practices">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/HP/Gold_Integrated">Integrated Human Practices</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Interviews">Expert Interviews</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Safety">Safety &amp; Security</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Engagement">Public Engagement</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Education">Education</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Collaborations">Collaborations</A></LI><LI><A data-toggle="collapse" data-target="#Notebook_side">Notebook </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Notebook">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Experiments">Methods</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Materials">Materials</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Database">Notebook Database</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Toolbox">Toolbox Guide</A></LI></NAV><DIV style="background-color: white !important; padding: 0 !important; margin: 0 !important;"><DIV class="page-title" style="background-color: white !important; border-top: 0px solid #222;"><DIV class="container" style=" padding: 0 !important;"><DIV class="col-lg-12 col-md-12 col-xs-12" style="padding-right: 0 !important; padding-bottom: 0 !important; position: relative; left: -10px;"><DIV style="border-right: 5px solid #393939; padding: 20px 20px 0px 0px"><DIV class="header-title heidelberg-turk"> 
     Protein Interaction </DIV><DIV class="header-subtitle" style="color: #393939 !important;"> 
    Improving Split Protein Auto-Reassembly </DIV></DIV></DIV></DIV></DIV><DIV class="container" style="background-color: white !important; padding: 0 !important;"><DIV class="col-lg-12 col-md-12 col-xs-12" style="padding: 0 !important; margin: 0px important!;"><DIV class="container" style="position: relative; left: -10px; border-left: 5px solid #393939; border-top: 5px solid #393939; padding: 0 !important"><DIV class="col-lg-12 col-md-12 col-xs-12 content" style="text-align: justify; text-color: #393939 !important; padding: 20px 5px 0px 20px !important; font-weight: bold !important; font-family: 'Open Sans' !important">
        
        Proteins rarely fulfill their function alone within a cell or organism. In contrast, they permanently interact with other proteins to orchestrate cellular processes such as signal transduction, metabolism, cell motion or and antigen-antibody recognition. A protein interaction type of particular interest for synthetic biology is the auto-reassembly of engineered split protein domains, which can help achieve efficient protein expression whenever size of the underlying expression cassette is a concern (e.g. in context of viral vectors).  N-terminal and C-terminal fragments of a particular protein are thereby transcribed and translated separately and re-associate in the cell via non-covalent interactions. However, in many cases, the activity of split proteins is compromised as compared to their full-length counterparts. Therefore, we set out to create an in vivo evolution protocol for improving protein-protein interactions using split T7 polymerase as example. After three days of PACE using a mutagenic pT7-geneIII E. coli selection strains, we obtained various split T7 mutants. Remarkably, one coding mutation was thereby located right at the split T7 interface, providing initial evidence for successful evolution towards improved auto-reassembly.
    
    </DIV></DIV><DIV class="container"><DIV class="row"><DIV class="col-lg-12 col-md-12 col-xs-12 content" style="display: inline-block; padding-bottom: 10px; padding-top:10px;"><H1>Introduction</H1><H2>Protein interactions in evolution context</H2>
             The proteome reflects the entirety of proteins that is expressed by a cell or an organism. Most of these proteins do not function alone. Instead, they are dependent on the interactions with a multitude of other proteins to fulfill their functions within the cell. Protein-protein interactions are responsible for almost any kind of cellular processes, such as signal transduction, cell-cell contact, transport, metabolism, cell motion or even antigen-antibody recognition <X-REF>Larance.2015</X-REF>. These interactions are enabled by electrostatic forces due to the different chemical characteristics of the amino acids. Nowadays, many biochemical or microscopy-based methods are available to investigate protein-protein interactions, for instance protein complex immunoprecipitation (Co-IP), Förster resonance energy transfer (FRET) or yeast two-hybrid screenings. As easy it is to detect a protein-protein interaction, as difficult it is to alter the strength of these interactions, although the magnitude of interaction can influence the cellular outcome, for example a stronger signal transduction or a higher expression rate. Phage-assisted continuous evolution (PACE) allows a directed evolution of different kinds of proteins within a few days as it was for example shown for RNA polymerases, proteases or aminoacyl-tRNA synthetases <X-REF>RN37'</X-REF>,<X-REF>Dickinson.2014</X-REF>,<X-REF>Bryson.2017</X-REF>. Technically, a directed evolution towards a tighter protein-protein interaction is possible as well, as it was shown for the toxin Cry1Ac from Bacillus thuringiensis (Bt toxin) that binds a cadherin-like receptor <X-REF>RN120'</X-REF>.

             <H2>Motivation</H2>

             A special type of a protein-protein interaction are split enzymes that allow an auto-reassembly. They usually consist of two enzyme fragments (an N‑terminal and C‑terminal fragment) that are expressed separately, and fuse post translation. This protein fragment complementation is advantageous for any kind of application where the size of an expression cassette is a limiting factor, for instance for the packaging of DNA into virus capsids. For many enzymes auto-reassembly split sites are known, however the efficiency of the joined fragments usually does not reach wildtype efficiency, as it was shown for Cas9 enzymes <X-REF>Zetsche.2015</X-REF>,<X-REF>Kaya.2017</X-REF>. Consequently, a directed evolution of split enzymes that enhance auto-reassembly is of great interest for the synthetic biology community. To demonstrate that an improved protein-protein interaction is not only possible for the Bt toxin, we used a split T7-polymerase as another example <X-REF>Han.2017</X-REF>. We aimed at enhancing the auto-reassembly efficiency of different split sites using two in our lab established methods, PACE and PREDCEL (phage-related discontinuous evolution), which allow a fast and relatively easy directed evolution.
        
    
    </DIV></DIV></DIV><DIV class="col-lg-12 col-md-12 col-xs-12"><DIV class="mdl-shadow--4dp" style="padding:20px; float:left}"><DIV class="img-caption title"><STRONG><B>
      
                Figure 1: Selecting for efficient T7-RNAP auto-reassembly in directed evolution experiment
    </B></STRONG></DIV><DIV class="img-caption subtitle">
      
                As all PREDCEL/PACE experiments are based on phage infection, selection for efficient T7-RNAP auto-reassembly starts with a phage injecting its genome, the selection            plasmid (SP), into a bacterial cell.  The SP carries the two halves of the split T7 RNAP expressed under one geneIII (gIII) promotor. Further it contains required genes for phage reproduction except for gIII.
 If through MP-activation mutation process leads to beneficial mutations in one or both T7 RNAP sequences, split variants assemble efficiently, bind to T7 promotor and thereby activate expression of protein III (pIII). Now phages can be built including pIII. That enables the phage to be released and to infect new cells (left).
If mutations are inefficient, T7 RNAP halves do not reassemble, therefore cannot bind to T7 promotor and pIII is not expressed. Therefore, phages cannot leave cells or infect new cells (right).
    </DIV></DIV></DIV><DIV class="container"><DIV class="row"><DIV class="col-lg-12 col-md-12 col-xs-12 content" style="display: inline-block; padding-bottom: 10px; padding-top:10px;"><H1>Results</H1>
 
 Based on the paper by Tiyun Han <I>et al.</I><X-REF>Han.2017</X-REF>, we created 4 different split T7 polymerase variants expressed from a ploycistronic expression cassette. In order to make sure, that the activity of our split variants is not due to skipping of a stop codon behind the first fragment and corresponding production of full-length T7 polymerase, located the C-terminal split part to the 5’ end of expression cassette (Figure 2). 

 


Applying our phage production toolbox standard, we then produced geneIII-deficient M13 phages encoding the different T7 split variants as well as a full-length T7 as control. Next, we generated a corresponding  E. coli selection strain transformed with an accessory plasmid expressing gene III form a T7 promoter. Hence, only phages encoding function split T7 variants would be able to activate expression of gene III an propagate (Figure 3). 

 <DIV class="container"><DIV class="row"><DIV class="col-lg-12 col-md-12 col-xs-12 content" style="display: inline-block; padding-bottom: 10px; padding-top:10px;"><DIV class="col-lg-6 col-md-12 col-xs-12 pull-left image-box-left"><DIV class="mdl-shadow--4dp" style="padding:20px; float:left"><DIV class="img-caption title"><STRONG><B>
      Figure 2: Split T7 polymerase variants generated in course of this project.
    </B></STRONG></DIV><DIV class="img-caption subtitle">
      Numbers correspond to amino acid position with respect to the full-length T7 polymerase protein.
    </DIV></DIV></DIV><DIV class="col-lg-6 col-md-12 col-xs-12 pull-left image-box-left"><DIV class="mdl-shadow--4dp" style="padding:20px; float:left"><DIV class="img-caption title"><STRONG><B>
      Figure 3: Creating a selection circuit linking (split) T7 polymerase activity to phage propagation. 
    </B></STRONG></DIV><DIV class="img-caption subtitle">
      The E. coli selection strain carries an accessory plasmid encoding gene III driven from a T7 promoter. Therefore, only phages encoding a functional T7 polymerase are able to propagate efficiently on the selection strain. 
    </DIV></DIV></DIV></DIV></DIV></DIV>
 
 To validate, if our selection circuit is working, we infected the aforementioned E. coli selection cells with at identical multiplicity-of-infection (MOI) with the different Split-T7 encoding phage variants as well as the full-length T7 control phage (Figure 4). 

     
<DIV class="col-lg-12 col-md-12 col-xs-12"><DIV class="mdl-shadow--4dp" style="padding:20px; float:left}"><DIV class="img-caption title"><STRONG><B>
      Figure 4: Single-round propagation of M13 phages encoding the indicated split T7 polymerase variant (see Figure 2) or a full-length T7 as control. 
    </B></STRONG></DIV><DIV class="img-caption subtitle">
       The propagation factor corresponds to the fold increase in phage titer ascompared to the used inoculation titer. Data are means +/- standard deviation, 3 replicates. 
    </DIV></DIV></DIV>
 
 As hoped, all phages were able to propagate on the selection strain, albeit to different degrees. Remarkably, the degree of phage propagation almost perfectly represented the efficiency of the underlying split T7 variants as reported by Tiyun Han <I>et al.</I><X-REF>Han.2017</X-REF>. In fact, propagation was most efficient for the phage variant encoding the T7 split at residues 564/565, which is also the most efficient variant described by Tiyun Han <I>et al.</I><X-REF>Han.2017</X-REF>. This data indicates, that our accessory plasmid indeed applies a selection pressure and effectively links the “fitness” of the encoded split T7 variant to the phage replication. 
 
 
 
 Importantly, the full-length T7 polymerase control phage was weaker as compared to all split variants in this particular experiment. A possible explanation for this seemingly contradictory observation is, that the split T7 variants might be simply more quickly and efficiently translated (due to the smaller size of the two split T7 polypeptides as compared to the full-length T7 and the presence of individual RBSs driving translation). While this expression kinetics effect might average out in long-term experiments running over several hours, it certainly has a major impact in context of this PREDCEL experiment, where phage propagation time is limited to 1 h. 
 
  
  
  Due to the highly efficient propagation of our phage encoding the T7 split at residue 565/564, we sought out to take the challenge of using our PACE apparatus and corresponding continuous flow setup for the final evolution experiment to further improve this split T7 variant. Details are described on the <A href="https://2017.igem.org/Team:Heidelberg/Pace">PACE page</A>. In brief, we created a mutagentic selection strain by co-transforming mutagenesis-inducing plasmid MP4 (Badran et al, Nature Communication, 2015) and our pT7-geneIII  accessory plasmid. Phages were propagated  for 3 days with a flow rate of 1 lagoon volume/hour, followed by plague assays. The split T7 fragment of 5 clonal plagues was then amplified by PCR and sanger-sequenced. As hoped, we observed a recurrent, coding mutation (T877P) present in three out of the five sequenced split T7 variants (Figure 5).
  
    
<DIV class="col-lg-12 col-md-12 col-xs-12"><DIV class="mdl-shadow--4dp" style="padding:20px; float:left}"><DIV class="img-caption title"><STRONG><B>
      Figure 5: Mutational pattern of the evolved split T7 variant.
    </B></STRONG></DIV><DIV class="img-caption subtitle">
      Following three days of in vivo evolution with PACE (Link PACE page), a plague assay
was performed and the split T7 insert of five individual phage clones was analyzed by sanger-sequencing. We observed a recurrent mutation (T877P) in three out of the five clones, suggesting an evolutionary advantage (i.e. increased fitness) of the corresponding split T7 mutant as compared its non-mutated counterpart. 
    </DIV></DIV></DIV>
    Interestingly this residue is located very close to the interaction surface of the two split T7 domains, suggesting a possible role in T7 auto-reassembly (Figure 6). 
    
    
<DIV class="col-lg-12 col-md-12 col-xs-12"><DIV class="mdl-shadow--4dp" style="padding:20px; float:left}"><DIV class="img-caption title"><STRONG><B>
      Figure 6: Structure of T7 polymerase.  
    </B></STRONG></DIV><DIV class="img-caption subtitle">
      The color coding corresponds to the T7 split at residues 564/565 in Figure 2. Cyan color indicates the N-terminal split domain, blue the C-terminal split domain. Residue T877 is stained in red.   
    </DIV></DIV></DIV></DIV></DIV></DIV><DIV class="container"><DIV class="row"><DIV class="col-lg-12 col-md-12 col-xs-12 content" style="display: inline-block; padding-bottom: 10px; padding-top:10px;"><H1>Discussion</H1>
 
 Here, we employed our evolution toolbox to engineer split T7 polymerase variants with improved efficiency due to improved auto-reassembly capability. Therefore, we generated split T7 encoding, gene III deficient M13 phages and propagated them on corresponding, mutagenic selection cells carrying encoding a pT7-geneIII accessory plasmid. After three days of continuous evolution in PACE, we observed a recurrent point mutation (T877P) in three out of five analyzed plagues. This striking result suggests a strong evolutionary advantage of the T877P mutant, split T7 encoding phages as compared to their wild type counterparts.
 
Although we can at this point not precisely elucidate the mechanism, by which this mutation improves the T7 function, our structural analysis indicates that the T877P mutation is located right at the interface of the split T7 domains. More precisely, the mutation is located at the C-terminal end of the C-terminal T7 helix. It is particularly interesting to note, that the threonine to proline mutation will likely affect the helical conformation, as proline has the lowest helix propensity of all amino acids. At this point we can only speculate, that the induced change in conformation may improve non-covalent interactions of amino acids located close at the split interface close to the mutated site.

To characterize our wild type split T7 and improved split T7 (T877P mutant) in further detail, we plan to clone these variants onto regular plasmid and compare their ability to activate T7-driven fluorescence or luminescence reporters.

Taken together, our results demonstrate the successful application of our in vivo directed evolution toolbox for improving protein-protein interactions, as demonstrated here for auto-reassembly of split T7 polymerase.

 
        
    
    </DIV></DIV></DIV></DIV></DIV></DIV><DIV style="background-color: white; padding-top: 20px;"><DIV class="container"><DIV class="row"><DIV class="col-lg-12 col-md-12 col-xs-12 content" style="display: inline-block;"><DIV class="row content"><DIV class="col-lg-12 col-md-12 col-xs-12" style="padding-top: 20px; padding-bottom: 50px; display: block"><DIV class="mdl-shadow--4dp" style="padding: 20px"><H3> References </H3></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><SECTION id="footer-sec" style="background-color: #222 !important"><DIV class="container"><DIV class="row" style="padding-top: 30px;"><DIV class="col-md-3"><H4 style="text-align: center">Quote</H4><P><I>“”</I></P></DIV><DIV class="col-md-3"><H4 style="text-align: center"> Useful <STRONG>Links</STRONG></H4><P><A href="https://igem.org/Special:SpecialPages">
			Special pages
		</A></P><P><A href="https://igem.org/Main_Page">
			Main Page
		</A></P></DIV><DIV class="col-md-3"><H4 style="text-align: center"> Follow us on </H4><DIV style="text-align: center;"><SPAN style="color: grey; text-align: center;">&amp;</SPAN></DIV></DIV><DIV class="col-md-3"><H4 style="text-align: center"> Contact us </H4><P>  iGEM-Heidelberg2017@bioquant.uni-heidelberg.de
               
		 Im Neuenheimer Feld 267
		
		69120 Heidelberg
                    </P></DIV></DIV></DIV></SECTION></DIV></DIV></DIV></DIV></BODY></HTML>