<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Heidelberg_CRISPR skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Heidelberg/CRISPR</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container"><DIV class="navbar-header"><A type="button" data-canvas="body" id="sidebarCollapse" class="navbar-toggle" data-toggle="offcanvas" data-target="#sidebar" style="color: #393939;margin-top: 9px;">NAVIGATION
                    </A></DIV><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav pull-right"><LI class="dropdown"><A href="https://2017.igem.org/Team:Heidelberg/Team">People </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Team">Team</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Attributions">Attributions</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Sponsoring">Sponsoring</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/For_Judges">For Judges</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Jamboree">Giant Jamboree</A></LI></UL><LI class="dropdown"><A href="https://2017.igem.org/Team:Heidelberg/Description">Project </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Description">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Design">Background &amp; Design </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Pace">PACE</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Predcel">PREDCEL</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Results">Results Summary</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Validation">Software Validation</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Organosilicons">Organosilicons</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Protein_Interaction">Protein Interaction</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Cytochrome_Engineering">Cytochrome Engineering</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Optogenetics">Optogenetics</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/CRISPR">CRISPR Cas9</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/InterLab">InterLab Study</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Achievements">Achievements</A></LI><LI class="dropdown"><A href="https://2017.igem.org/Team:Heidelberg/Parts">Parts </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Basic_Part">Basic Parts</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Composite_Part">Composite Parts</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Improved_Part">Improved Part</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Part_Collection">Parts Collection</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Part_List">Part List</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/RFC">RFC</A></LI><LI class="dropdown"><A href="https://2017.igem.org/Team:Heidelberg/Model">Modeling </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Phage_Titer">Phage Titer</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Mutagenesis_Induction">Mutagenesis Induction</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Mutation_Rate_Estimation">Mutation Rate Estimation</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Lagoon_Contamination">Lagoon Contamination</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Medium_Consumption">Medium Consumption</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Tools">Interactive Web Tools</A></LI><LI class="dropdown"><A href="https://2017.igem.org/Team:Heidelberg/Software">Software </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software">AiGEM Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software/DeeProtein">DeeProtein</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software/GAIA">GAIA</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software/SafetyNet">SafetyNet</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software/MAWS">MAWS 2.0</A></LI><LI class="dropdown"><A href="https://2017.igem.org/Team:Heidelberg/Human_Practices">Human Practices </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Human_Practices">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/HP/Gold_Integrated">Integrated Human Practices</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Interviews">Expert Interviews</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Safety">Safety &amp; Security</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Engagement">Public Engagement</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Education">Education</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Collaborations">Collaborations</A></LI><LI class="dropdown"><A href="https://2017.igem.org/Team:Heidelberg/Notebook">Notebook </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Notebook">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Experiments">Methods</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Materials">Materials</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Database">Notebook Database</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Toolbox">Toolbox Guide</A></LI></DIV></DIV><NAV id="sidebar" style="z-index: 10001; background-color: white;border: none;box-shadow:1px 0px 0px #393939;" class="navmenu navmenu-default navmenu-fixed-left offcanvas" role="navigation"><A class="navmenu-brand" href="https://2017.igem.org/Team:Heidelberg"> iGEM TEAM HEIDELBERG 2017</A><DIV id="sidebarHide" data-toggle="offcanvas" data-target="#sidebar" style="padding-left: 20px;"><A style="font-family: 'Josefin Sans' !important; font-size: 16px; line-height: 18px; height: 18px;"> COLLAPSE NAVIGATION </A></DIV><UL class="nav navmenu-nav" id="sidenav"><LI><A data-toggle="collapse" data-target="#People_side">People </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Team">Team</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Attributions">Attributions</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Sponsoring">Sponsoring</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/For_Judges">For Judges</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Jamboree">Giant Jamboree</A></LI></UL><LI><A data-toggle="collapse" data-target="#Project_side">Project </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Description">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Design">Background &amp; Design </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Pace">PACE</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Predcel">PREDCEL</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Results">Results Summary</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Validation">Software Validation</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Organosilicons">Organosilicons</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Protein_Interaction">Protein Interaction</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Cytochrome_Engineering">Cytochrome Engineering</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Optogenetics">Optogenetics</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/CRISPR">CRISPR Cas9</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/InterLab">InterLab Study</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Achievements">Achievements</A></LI><LI><A data-toggle="collapse" data-target="#Parts_side">Parts </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Basic_Part">Basic Parts</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Composite_Part">Composite Parts</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Improved_Part">Improved Part</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Part_Collection">Parts Collection</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Part_List">Part List</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/RFC">RFC</A></LI><LI><A data-toggle="collapse" data-target="#Model_side">Modeling </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Phage_Titer">Phage Titer</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Mutagenesis_Induction">Mutagenesis Induction</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Mutation_Rate_Estimation">Mutation Rate Estimation</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Lagoon_Contamination">Lagoon Contamination</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Medium_Consumption">Medium Consumption</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Tools">Interactive Web Tools</A></LI><LI><A data-toggle="collapse" data-target="#AI_side">Software </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software">AiGEM Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software/DeeProtein">DeeProtein</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software/GAIA">GAIA</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software/SafetyNet">SafetyNet</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software/MAWS">MAWS 2.0</A></LI><LI><A data-toggle="collapse" data-target="#HP_side">Human Practices </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Human_Practices">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/HP/Gold_Integrated">Integrated Human Practices</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Interviews">Expert Interviews</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Safety">Safety &amp; Security</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Engagement">Public Engagement</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Education">Education</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Collaborations">Collaborations</A></LI><LI><A data-toggle="collapse" data-target="#Notebook_side">Notebook </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Notebook">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Experiments">Methods</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Materials">Materials</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Database">Notebook Database</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Toolbox">Toolbox Guide</A></LI></NAV><DIV style="background-color: white !important; padding: 0 !important; margin: 0 !important;"><DIV class="page-title" style="background-color: white !important; border-top: 0px solid #222;"><DIV class="container" style=" padding: 0 !important;"><DIV class="col-lg-12 col-md-12 col-xs-12" style="padding-right: 0 !important; padding-bottom: 0 !important; position: relative; left: -10px;"><DIV style="border-right: 5px solid #393939; padding: 20px 20px 0px 0px"><DIV class="header-title heidelberg-orange">                
    CRISPR Cas9 </DIV><DIV class="header-subtitle" style="color: #393939 !important;"> 
    Phage Based Directed Evolution of Endonucleases </DIV></DIV></DIV></DIV></DIV><DIV class="container" style="background-color: white !important; padding: 0 !important;"><DIV class="col-lg-12 col-md-12 col-xs-12" style="padding: 0 !important; margin: 0px important!;"><DIV class="container" style="position: relative; left: -10px; border-left: 5px solid #393939; border-top: 5px solid #393939; padding: 0 !important"><DIV class="col-lg-12 col-md-12 col-xs-12 content" style="text-align: justify; text-color: #393939 !important; padding: 20px 5px 0px 20px !important; font-weight: bold !important; font-family: 'Open Sans' !important">
        

CRISPR (Clustered Regularly Interspaced Short Palindromic Repeats) related endonucleases revolutionized the field of genetic engineering. They provide a simple and robust strategy for sequence specific DNA targeting based on Watson Crick base pairing mediated by an RNA guide. CRISPR/Cas9 is used for various applications, for instance genome editing <X-REF>JM_1</X-REF>, transcription regulation <X-REF>JM_2</X-REF>, and RNA editing<X-REF>JM_3</X-REF>. The development of endonucleases with expanded functionality such as relaxed target sequence requirments (i.e. PAM requirements) would be a benefit for the whole field of genome engineering. We outline an innovative concept for the development of improved Cas9 variants showing a broadened sequence targeting range via PRECEL- or PACE-mediated in vivo directed evolution. We created constructs linking the transcription of the ciritcal M13 phage geneIII to the Cas9 PAM specificity via a dCas9-rpoZ-based transcriptional activator. Our work provides a first steps towards our vision of optimization gene editing technology by means of in vivo directed evolution. 

    
    </DIV></DIV><DIV class="container"><DIV class="row"><DIV class="col-lg-12 col-md-12 col-xs-12 content" style="display: inline-block; padding-bottom: 10px; padding-top:10px;"><H1 id="id">Introduction</H1>

Many of nowadays most threatening diseases are caused by mutations, epi mutations or other changes in the genome. Although medical research was always supplied by innovations in biological research and especially by the field of genetics, which developed rapidly during the last decades, there are still many diseases that cannot be cured or even treated adequately. Recently, the CRISPR/Cas9 technology raised hope of the scientific community to treat genetic disorders. This technique has dramatically simplified the way genomes can be manipulated. However, there are still many challenges to be surpassed. Cas9 and related endonucleases are enzymes, which are able to induce double strand breaks in the genome. Importantly, they only cut specific sequences to which they are guided by a so called guideRNA (gRNA). A gRNA consists of a 3' scaffold, which is obligatory for the binding of the Cas9 enzyme, a protospacer sequence, and 20 nucleotides at the 5'-end that are complementary to the target DNA. Once the Cas9 endonuclease binds to the DNA, it cleaves three nucleotides upstream of the protospacer 3'-end. This system allows to target virtually any position in any genome. However, there is one major restriction in the applicability of this system. Only sequences can be targeted that carry a specific recognition motif directly downstream of the spacer, the protospacer adjacent motif (PAM). In case of Cas9, the consensus PAM is NGG <X-REF>Jinek2012</X-REF>. The past few years, much effort has been put into the development of the CRISPR/Cas9 technique. In order to create even more sophisticated systems, many attempts have been made to modify the CRISPR-associated (Cas) endonucleases, for example the development of a catalytically inactive dCas9 <X-REF>RN141</X-REF> or nickases <X-REF>Mali2013x</X-REF>.
Attempts to change endonuclease activity by directed evolution have already been made <XREF>JM_5</XREF><X-REF>Gao2017</X-REF>. As add on to our PACE toolbox, we decided to show that <I>in vivo</I> directed evolution of endonucleases is possible as well. This approach would overcome the limitations of the evolution based on rationalities and would offer new tools for the genome engineering research area.


<DIV class="col-lg-12 col-md-12 col-xs-12"><DIV class="mdl-shadow--4dp" style="padding:20px; float:left}"><DIV class="img-caption title"><STRONG><B>
      
                Figue 1: Principle of PACE for the Evolution of Cas9
    </B></STRONG></DIV><DIV class="img-caption subtitle">
      
                The production of phages is linked to the fitness of Cas9 via the fusion to rpoZ. When the gRNA targets the fusion protein to a sequence upstream of the promoter, the rpoZ is able to recruit the transcription machinery and therefore activates gene VI expression and subsequently phage production. When the PAM upstream of the spacer is altered, Cas9 cannot bind anymore and the expression stops. In context of PACE the nuclease mutates. Mutants of Cas9 with that can bind to the new PAM, activate protein VI production. This provides a fitness benefit. With the generation of randomized PAM libraries, the overall PAM specificity can be decreased, which is of great interest for many gene editing applications
            
    </DIV></DIV></DIV><H1 id="id">The Idea</H1>
  
To prove our hypothesis, we planned a circuit for the directed evolution of PAM specificity of Cas9 (Fig. 1). The main challenge was to link transcription activation to the binding of Cas9. We chose a system, which contains a dCas9 fused to a RNA polymerase Ω subunit (rpoZ) <X-REF>BIKARDETAL..2013</X-REF>. The rpoZ is able to recruit the transcription machinery to a poromoter and therefore activate gene expression. In our scenario, the nuclease targets a region upstream of a minimal promoter. In case the dCas9 is able to bind the DNA, the fused rpoZ activates gene expressio expreession. The J23117 promoter from the registry was chosen for activation <X-REF>BIKARDETAL..2013</X-REF>. It stands out with its low background activity and can be activated by the factor of 23 when rpoZ binds. Upstream of the promoter sequence, a spacer with a wildtype NGG PAM, that was previously used to target RFP was placed <X-REF>QIETAL..2013</X-REF>. If the Cas9 is  able to bind to the spacer, transcription is activated. By changing the PAM sequence or generating PAM libraries, it is possible to induce a selection pressure on the randomly mutating protein. As a result, proteins with a weaker PAM specificity evolve, which can be used, no matter if the NGG motif is present exactly at the desired position.
Of course, this circuit was designed according to our cloning standard by Gibson assembly.

<H1 id="id">Phage Based <I>in vivo</I> Evolution with gene VI</H1>

One of the major challenges in the context of transcription activation with help of the rpoZ is leaky expression. This is a serious problem for PACE, because if gene III is expressed in absence of phage, the cell may become resistant to phage infection<X-REF>RN44</X-REF>. Fortunately, it was shown that geneVI can be used for phage propagation in the context of directed evolution as well. In contrast to the commonly used gene III, it has the advantage that leaky gene VI expression has no effect on the infectivity of <I>E. coli</I> by phages <X-REF>BRODELETAL..2016</X-REF>. As a consequence, we decided to adapt this approach and towork with geneVI instead of geneIII. 

All parts, which were necessary for the assembly of Accessory Plasmids were generated by PCR with the respective homology regions in the extensions. Subsequently, they were assembled by Gibson assembly (Fig.:2). All APs carry a bicistronic operon for the expression of geneIII and luxAB as luminescent reporter downstream of the promoter, as described above. An expression cassette with the required gRNA under the control of a constitutive promoter is located on the same plasmid. APs varying in the copy number of their origins of replication and the strength of the RBS upstream of geneVI were cloned. To evolve the PAM specificity, we generated PAM libraries with four randomized nucleotides next to the spacer sequence. In order to do so, the whole plasmid was PCR amplified with the four PAM nucleotides as primer extensions. Subsequently, the plasmid was reassembled by Golden Gate assembly. To avoid that nucleotides, that pair with the original PAM are preferred and overrepresented in the library, a BbsI site was inserted next to the PAM. Prior to the PCR, the plasmid was digested with the enzyme, resulting in a linear fragment. The four nucleotides were located in overhangs, in the strand, to which the primer cannot bind. Plasmids that were cloned for the evolution of PAM specificity, the plasmid names, and the functional parts they consist of are shown in (Tab.: 1).

  
<DIV class="col-lg-12 col-md-12 col-xs-12"><DIV class="mdl-shadow--4dp" style="padding:20px; float:left}"><DIV class="img-caption title"><STRONG><B>
      
                Figure 2: Design of the Accessory Plasmids for the Evolution of Cas9
    </B></STRONG></DIV><DIV class="img-caption subtitle">
      
                The AP consists of five subparts that are devided by homology regions for Gibson assembly (numbers). It carries an expression cassette for the transcription of a gRNA (between 1 and 5).  GeneVI (2-4) is under control of a that can be activated by the Cas9-rpoZ in context with the respective gRNA.  The whole plasmid can be produced with different origins of replication (4-5) to modulate the copy number and by exchanging the geneVI part with the RBS.
    </DIV></DIV></DIV><DIV class="col-lg-12 col-md-12 col-xs-12 content" style="padding-top: 20px; padding-bottom: 20px"><P class="content"><B>
                Our Accessory Plasmids for PACE of Endonucleases </B>
    
The different accessory plasmids that were cloned in the context of this project are shown. The constructs differ in their copy number and the strength of their RBSs. 
          
        
  </P><DIV class="table-slot" style="overflow-x: auto"><TABLE><THEAD><TR><TH>Puri-ID</TH><TH style="text-align:center">AP</TH><TH style="text-align:right">Regulatory Sequence</TH><TH>RBS of geneIV</TH><TH>Origin of replication</TH><TH>gRNA cassette</TH><TH>PAM</TH></TR></THEAD><TBODY><TR><TD>821</TD><TD style="text-align:center">AP_Cas9_pSC101_NNNN_SD8_GVI</TD><TD style="text-align:right">minimal promoter downstram of the dCas9 target sequence</TD><TD>SD8</TD><TD>pSC101</TD><TD>gRNA expression cassette</TD><TD>NNNN</TD></TR><TR><TD>822</TD><TD style="text-align:center">AP_Cas9_pSC101_NNNN_sd8_GVI</TD><TD style="text-align:right">minimal promoter downstram of the dCas9 target sequence</TD><TD>sd8</TD><TD>pSC101</TD><TD>gRNA expression cassette</TD><TD>NNNN</TD></TR><TR><TD>823</TD><TD style="text-align:center">AP_Cas9_pSC101_NNNN_sd6_GVI</TD><TD style="text-align:right">minimal promoter downstram of the dCas9 target sequence</TD><TD>sd6</TD><TD>pSC101</TD><TD>gRNA expression cassette</TD><TD>NNNN</TD></TR><TR><TD>824</TD><TD style="text-align:center">AP_Cas9_pSC101_NNNN_SD4_GVI</TD><TD style="text-align:right">minimal promoter downstram of the dCas9 target sequence</TD><TD>sd4</TD><TD>pSC101</TD><TD>gRNA expression cassette</TD><TD>NNNN</TD></TR><TR><TD>825</TD><TD style="text-align:center">AP_Cas9_pSC101_NNNN_sd2_GVI</TD><TD style="text-align:right">minimal promoter downstram of the dCas9 target sequence</TD><TD>sd2</TD><TD>pSC101</TD><TD>gRNA expression cassette</TD><TD>NNNN</TD></TR><TR><TD>826</TD><TD style="text-align:center">AP_Cas9_pSC101_NGAN_SD8_GVI</TD><TD style="text-align:right">minimal promoter downstram of the dCas9 target sequence</TD><TD>SD8</TD><TD>pSC101</TD><TD>gRNA expression cassette</TD><TD>NGAN</TD></TR><TR><TD>829</TD><TD style="text-align:center">AP_Cas9_pSC101_NGAN_SD4_GVI</TD><TD style="text-align:right">minimal promoter downstram of the dCas9 target sequence</TD><TD>sd4</TD><TD>pSC101</TD><TD>gRNA expression cassette</TD><TD>NGAN</TD></TR><TR><TD>830</TD><TD style="text-align:center">AP_Cas9_pSC101_NGAN_sd2_GVI</TD><TD style="text-align:right">minimal promoter downstram of the dCas9 target sequence</TD><TD>sd2</TD><TD>pSC101</TD><TD>gRNA expression cassette</TD><TD>NGAN</TD></TR></TBODY></TABLE></DIV></DIV><H1>Outlook</H1>
The CRISPR/Cas9 technology (Jinek et al., Science, 2012; Mali et al, Science, 2013; Cong et al, Science, 2013) set out to revolutionize our abilities to edit genomes. We envision improving this technology by means of in vivo evolution, to make it as safe, as specific and as broadly applicable as possible. As a humble first, step towards this ambitious goal, we created CRISPR/Cas9 accessory constructs linking the transcription of the critical M13 phage geneIII to the Cas9 PAM specificity via a dCas9-rpoZ-based transcriptional activator. In the future, we plan to further expand our accessory plasmid collection and apply it for PREDCEL-mediated evolution of the CRISPR/Cas9 towards relaxed PAM requirements and improved target sequence specificity.


    </DIV></DIV></DIV></DIV></DIV></DIV><DIV style="background-color: white; padding-top: 20px;"><DIV class="container"><DIV class="row"><DIV class="col-lg-12 col-md-12 col-xs-12 content" style="display: inline-block;"><DIV class="row content"><DIV class="col-lg-12 col-md-12 col-xs-12" style="padding-top: 20px; padding-bottom: 50px; display: block"><DIV class="mdl-shadow--4dp" style="padding: 20px"><H3> References </H3></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><SECTION id="footer-sec" style="background-color: #222 !important"><DIV class="container"><DIV class="row" style="padding-top: 30px;"><DIV class="col-md-3"><H4 style="text-align: center">Quote</H4><P><I>“”</I></P></DIV><DIV class="col-md-3"><H4 style="text-align: center"> Useful <STRONG>Links</STRONG></H4><P><A href="https://igem.org/Special:SpecialPages">
			Special pages
		</A></P><P><A href="https://igem.org/Main_Page">
			Main Page
		</A></P></DIV><DIV class="col-md-3"><H4 style="text-align: center"> Follow us on </H4><DIV style="text-align: center;"><SPAN style="color: grey; text-align: center;">&amp;</SPAN></DIV></DIV><DIV class="col-md-3"><H4 style="text-align: center"> Contact us </H4><P>

               iGEM-Heidelberg2017@bioquant.uni-heidelberg.de
               
		 Im Neuenheimer Feld 267
		
		69120 Heidelberg
                    </P></DIV></DIV></DIV></SECTION></DIV></DIV></DIV></DIV></BODY></HTML>