<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Heidelberg_Optogenetics skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Heidelberg/Optogenetics</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container"><DIV class="navbar-header"><A type="button" data-canvas="body" id="sidebarCollapse" class="navbar-toggle" data-toggle="offcanvas" data-target="#sidebar" style="color: #393939;margin-top: 9px;">NAVIGATION
                    </A></DIV><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav pull-right"><LI class="dropdown"><A href="https://2017.igem.org/Team:Heidelberg/Team">People </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Team">Team</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Attributions">Attributions</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Sponsoring">Sponsoring</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/For_Judges">For Judges</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Jamboree">Giant Jamboree</A></LI></UL><LI class="dropdown"><A href="https://2017.igem.org/Team:Heidelberg/Description">Project </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Description">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Design">Background &amp; Design </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Pace">PACE</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Predcel">PREDCEL</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Results">Results Summary</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Validation">Software Validation</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Organosilicons">Organosilicons</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Protein_Interaction">Protein Interaction</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Cytochrome_Engineering">Cytochrome Engineering</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Optogenetics">Optogenetics</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/CRISPR">CRISPR Cas9</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/InterLab">InterLab Study</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Achievements">Achievements</A></LI><LI class="dropdown"><A href="https://2017.igem.org/Team:Heidelberg/Parts">Parts </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Basic_Part">Basic Parts</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Composite_Part">Composite Parts</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Improved_Part">Improved Part</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Part_Collection">Parts Collection</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Part_List">Part List</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/RFC">RFC</A></LI><LI class="dropdown"><A href="https://2017.igem.org/Team:Heidelberg/Model">Modeling </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Phage_Titer">Phage Titer</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Mutagenesis_Induction">Mutagenesis Induction</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Mutation_Rate_Estimation">Mutation Rate Estimation</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Lagoon_Contamination">Lagoon Contamination</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Medium_Consumption">Medium Consumption</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Tools">Interactive Web Tools</A></LI><LI class="dropdown"><A href="https://2017.igem.org/Team:Heidelberg/Software">Software </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software">AiGEM Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software/DeeProtein">DeeProtein</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software/GAIA">GAIA</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software/SafetyNet">SafetyNet</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software/MAWS">MAWS 2.0</A></LI><LI class="dropdown"><A href="https://2017.igem.org/Team:Heidelberg/Human_Practices">Human Practices </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Human_Practices">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/HP/Gold_Integrated">Integrated Human Practices</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Interviews">Expert Interviews</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Safety">Safety &amp; Security</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Engagement">Public Engagement</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Education">Education</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Collaborations">Collaborations</A></LI><LI class="dropdown"><A href="https://2017.igem.org/Team:Heidelberg/Notebook">Notebook </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Notebook">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Experiments">Methods</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Materials">Materials</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Database">Notebook Database</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Toolbox">Toolbox Guide</A></LI></DIV></DIV><NAV id="sidebar" style="z-index: 10001; background-color: white;border: none;box-shadow:1px 0px 0px #393939;" class="navmenu navmenu-default navmenu-fixed-left offcanvas" role="navigation"><A class="navmenu-brand" href="https://2017.igem.org/Team:Heidelberg"> iGEM TEAM HEIDELBERG 2017</A><DIV id="sidebarHide" data-toggle="offcanvas" data-target="#sidebar" style="padding-left: 20px;"><A style="font-family: 'Josefin Sans' !important; font-size: 16px; line-height: 18px; height: 18px;"> COLLAPSE NAVIGATION </A></DIV><UL class="nav navmenu-nav" id="sidenav"><LI><A data-toggle="collapse" data-target="#People_side">People </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Team">Team</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Attributions">Attributions</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Sponsoring">Sponsoring</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/For_Judges">For Judges</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Jamboree">Giant Jamboree</A></LI></UL><LI><A data-toggle="collapse" data-target="#Project_side">Project </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Description">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Design">Background &amp; Design </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Pace">PACE</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Predcel">PREDCEL</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Results">Results Summary</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Validation">Software Validation</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Organosilicons">Organosilicons</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Protein_Interaction">Protein Interaction</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Cytochrome_Engineering">Cytochrome Engineering</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Optogenetics">Optogenetics</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/CRISPR">CRISPR Cas9</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/InterLab">InterLab Study</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Achievements">Achievements</A></LI><LI><A data-toggle="collapse" data-target="#Parts_side">Parts </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Basic_Part">Basic Parts</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Composite_Part">Composite Parts</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Improved_Part">Improved Part</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Part_Collection">Parts Collection</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Part_List">Part List</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/RFC">RFC</A></LI><LI><A data-toggle="collapse" data-target="#Model_side">Modeling </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Phage_Titer">Phage Titer</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Mutagenesis_Induction">Mutagenesis Induction</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Mutation_Rate_Estimation">Mutation Rate Estimation</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Lagoon_Contamination">Lagoon Contamination</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Medium_Consumption">Medium Consumption</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Tools">Interactive Web Tools</A></LI><LI><A data-toggle="collapse" data-target="#AI_side">Software </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software">AiGEM Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software/DeeProtein">DeeProtein</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software/GAIA">GAIA</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software/SafetyNet">SafetyNet</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software/MAWS">MAWS 2.0</A></LI><LI><A data-toggle="collapse" data-target="#HP_side">Human Practices </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Human_Practices">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/HP/Gold_Integrated">Integrated Human Practices</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Interviews">Expert Interviews</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Safety">Safety &amp; Security</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Engagement">Public Engagement</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Education">Education</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Collaborations">Collaborations</A></LI><LI><A data-toggle="collapse" data-target="#Notebook_side">Notebook </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Notebook">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Experiments">Methods</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Materials">Materials</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Database">Notebook Database</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Toolbox">Toolbox Guide</A></LI></NAV><DIV style="background-color: white !important; padding: 0 !important; margin: 0 !important;"><DIV class="page-title" style="background-color: white !important; border-top: 0px solid #222;"><DIV class="container" style=" padding: 0 !important;"><DIV class="col-lg-12 col-md-12 col-xs-12" style="padding-right: 0 !important; padding-bottom: 0 !important; position: relative; left: -10px;"><DIV style="border-right: 5px solid #393939; padding: 20px 20px 0px 0px"><DIV class="header-title heidelberg-blue">                
    Optogenetics </DIV><DIV class="header-subtitle" style="color: #393939 !important;"> 
    Modulator of selection stringency </DIV></DIV></DIV></DIV></DIV><DIV class="container" style="background-color: white !important; padding: 0 !important;"><DIV class="col-lg-12 col-md-12 col-xs-12" style="padding: 0 !important; margin: 0px important!;"><DIV class="container" style="position: relative; left: -10px; border-left: 5px solid #393939; border-top: 5px solid #393939; padding: 0 !important"><DIV class="col-lg-12 col-md-12 col-xs-12 content" style="text-align: justify; text-color: #393939 !important; padding: 20px 5px 0px 20px !important; font-weight: bold !important; font-family: 'Open Sans' !important">
        
Generating proteins with radically altered activities and functions is a major goal in synthetic biology. While in vivo directed evolution is generally capable of achieving this goal, it requires evolutionary stepping stones that enable a slowly progressing adaptation of the evolving gene pool to the novel, desired function. To implement such selection pressure modulation, a set of fine-tuned accessory plasmids has been required in the past. These then need to be applied in a well-orchestrated fashion to slowly increase selection pressure over time, facilitating genetic drift and eventually emergence of the novel function.  
Here, we present OptoSELECT, an optogenetic selection stringency modulator able to easily tune selection pressure during PREDCEL and PACE experiments using defined doses of blue light. Our tool is based on the previously described, blue light-dependent transcription factor EL222 and a bidirectional promoter system, which either induces or represses the expression of M13 geneIII upon blue light irradiation.  Using OptoSELECT, we demonstrate successful light control of geneIII-dependent phage propagation using an engineered pBLind promoter-geneIII expression cassette. Our work paves the way towards simple and straight-forward optimization of in vivo evolution experiments by optogenetic modulation of the selection pressure.
 

    </DIV></DIV><DIV class="container"><DIV class="row"><DIV class="col-lg-12 col-md-12 col-xs-12 content" style="display: inline-block; padding-bottom: 10px; padding-top:10px;"><H1>Introduction</H1><H2>Modulation of Selection Stringency</H2>
The directed evolution method PACE is an enormously powerful tool to improve and alter activities of different kinds of proteins for industrial, research or therapeutic applications. In some cases, a radically modified or highly specific new activity is demanded, but as a certain basal activity of the unevolved protein is necessary to propagate the selection phage (SP) with the respective accessory plasmid (AP) this task remains highly challenging. Phages encoding the unevolved protein of interest often display no or only low activity for geneIII expression on their AP. If they cannot propagate sufficiently, the phages in the lagoon are washed-out before they were able to gain favorable mutations. To decrease the stringency of an initial selection, intermediate substrates and constructs can be used as evolutionary stepping stones <X-REF>RN16</X-REF>. The difficulty consists in the fact that these evolutionary stepping stones are often not obvious or barely accessible <X-REF>Carlson.2014</X-REF>. 
An easier and generally applicable approach for selection stringency modulation is a carefully regulated provision of proteinIII independent of the favored evolving activity. This allows faint active or inactive variants to propagate in the lagoon and to accumulate mutations through evolutionary drift. Some of these mutations improve the evolving protein, which is coupled to the propagation abilities of the phages and enables phages with beneficial variants to persist higher selection pressure. The general principle of stringency modulation via controlled geneIII expression was proved by a small molecule-controlled selection stringency modulator engineered by Carlson <I>et. al.</I> 2014 using anhydrotetracycline (ATc) as geneIII inducer. They demonstrated that an inactive starting phage library can propagate with the addition of ATc and that a decrease of the ATc concentration leads to a selective enrichment of active mutants <X-REF>Carlson.2014</X-REF>. As exogenous chemical inducers are often limited due to transport process delay, cause toxicity or show a lack of reversibility of gene expression, we designed a novel optogenetic modulator of selection stringency <X-REF>CG01</X-REF><X-REF>CG02</X-REF><X-REF>CG03</X-REF>.




<H2>Optogenetic Tools</H2><DIV class="col-lg-6 col-md-12 col-xs-12 pull-right image-box-right"><DIV class="mdl-shadow--4dp" style="padding:20px; float:right"><DIV class="img-caption title"><STRONG><B>
      
                Figure 1: Crystal Structure of EL222
    </B></STRONG></DIV><DIV class="img-caption subtitle">
      
                The light-inducible transcription factor EL222 is classified into three parts: the LOV domain (blue), the conector helix (grey)  and the HTH domain (red). The flavinmononucleotide is depicted in green.
    </DIV></DIV></DIV>

In the past ten years the prospects of light-regulated systems rapidly expanded and became a powerful application tool in cell biology, neuroscience, and medical research. Optogenetics enable the regulation of biological systems in a non-invasive and reversible manner. In contrast to widely used chemical triggers, light can be applied with high spatial and temporal precision, and does not cause unintended side effects or off-target effects, which are common for most chemical inducers. Furthermore, photosensitive proteins can be easily regulated by light intensity and duration of light illumination <X-REF>CG04</X-REF>.
The naturally occuring light activated transcription factor EL222 from the marine bacterium <I> Erythrobacter litoralis </I> HTCC2594 consists of a N-terminal light-oxygen-voltage (LOV) domain, a linker helix and a C-terminal LuxR-type helix-turn-helix (HTH) motif <X-REF>CG04</X-REF> (Fig: 1). Upon blue light irradiation with a wavelength of 450 nm, an internal flavinmononucleotide-protein adduct is formed, which results in a conformational change of the modular 222 amino acid photosensitive protein. Subsequently, a previously sequestered DNA-binding domain is exposed, which allows the homo-dimerization of EL222, and thus DNA binding. In the dark, the process spontaneously reverses. EL222 is released from the DNA, and deactivates transcription within seconds to minutes<X-REF>CG05</X-REF>.



<H1>Motivation</H1>
The modulation of selection stringency is an essential requirement for PACE and PREDCEL based directed evolution. As we ourselves often struggled with phage wash-out during the initial selection phases, we considered the provision of a non-toxic, rapidly delivered and reversible modulator of the selection stringency as highly important for the scientific community. Our OptoSELECT system enables an easy and prompt adaption of selection pressure to the fitness of the evolving gene pool and minimizes the experimental effort of protein optimization using PACE and PREDCEL.

<H1>Design of OptoSELECT</H1>
The OptoSELECT system composes of the previously described, blue light-dependent transcription factor EL222 and two bidirectional geneIII expression cassettes: the blue light induced pBLind-gIII cassette and the blue light repressed Ppsp-EL222-BR-gIII cassette  (Fig: 2). pBLind is a synthetic, light-inducible promoter based on the luxI promoter. The lux box, a 20-bp inverted repeat from the luxI promoter, is replaced by the 18-bp EL222 binding region <X-REF>CG04</X-REF>. Upon blue light irradiation, the EL222 dimer binds to its binding region on the DNA and activates transcription by recruiting the RNA polymerase (RNAP). Consequently, the protein expression increase up to 5-fold compared to the dark state <X-REF>CG04</X-REF>. To regulate phage propagation and to adapt selection stringency in a light-dependent manner, geneIII was set under control of pBLind. This plasmid containing the pBLind-gIII expression cassette is further referred as AP_light. PACE and PREDCEL experiments with host cells carrying AP_light display a reduced selection stringency in the initial selection phase by illumination with blue light. As soon as the starting phage library acquires enough favorable mutations to persist higher selection pressure, the light can be switched off and the intensified expression of geneIII is aborted. 
To provide a complementary promoter system that allows an increase of selection pressure upon blue light irradiation, we designed the hybrid Psp-EL222-BR promoter. Therefore, the phage-shock-protein promoter (Ppsp), which is induced by infection with filamentous phages<X-REF>CG06</X-REF>, was combined with an EL222 binding region to repress gene expression post infection in the present of blue light irradiation. A similar promoter was previously engineered consisting of a Psp promoter and a tetracycline binding region <X-REF>Carlson.2014</X-REF>. To engineer a novel light-dependent promoter, this Psp-tet promoter was used, but instead of a tetracycline binding region an EL222 binding region (EL222-BR) was inserted adjacent to the 1+ transcription initiation site. The plasmid containing the Ppsp-EL222-BR-gIII expression cassette is referred as AP_dark. <I>E. coli</I>, which are transformed with AP_dark, express geneIII in the dark state only post phage infection. Upon blue light irradiation EL222 binds to the EL222 binding region and inhibits the expression of supplementary geneIII. 

  
<DIV class="col-lg-12 col-md-12 col-xs-12"><DIV class="mdl-shadow--4dp" style="padding:20px; float:left}"><DIV class="img-caption title"><STRONG><B>
      
                Figure 2: Expression Cassetts of OptoSELECT
    </B></STRONG></DIV><DIV class="img-caption subtitle">
      
Selection stringency can be easily modulated using the plasmids AP_light and AP_dark. AP_light contains the blue light induced pBLind-gIII expression cassette, which consists of geneIII under control of a modified luxI promoter with an EL222 binding region. In the dark state EL222 cannot bind to the DNA and the transcription of geneIII is repressed. Upon blue light irradiation EL222 undergoes a conformational change and binds to the EL222 binding region. This interaction recruits the RNA polymerase and activates the transcription of geneIII. AP_dark contains the blue light repressed Psp-EL222-BR-gIII cassette. After phage infection the psp promoter is activated and initiates the expression of geneIII. In the dark state this process can proceed without hindrance. Upon blue light irradiation EL222 binds adjacent to the 1+ transcription initiation site and inhibits the expression of geneIII. 
                
            
    </DIV></DIV></DIV><DIV class="container"><DIV class="row"><DIV class="col-lg-12 col-md-12 col-xs-12 content" style="display: inline-block; padding-bottom: 10px; padding-top:10px;"><H1 id="Results">Characterization</H1><H2>AP_light: Testing of the pBLind-gIII Expression Cassette</H2>

A phage propagation assay was performed to investigate the influence of blue light irradiation on the propagation of geneIII-deficient M13 phages containing EL222 in a culture transformed with the pBLind-gIII cassette of our OptoSELECT system. As we supposed that light might influence the fitness of <I>E. coli</I> and therefore phage propagation, we created a phage carrying the gene for a truncated, non-binding version of EL222. Both phages were used in this propagation assay.
An <I>E. coli</I> culture transformed with AP_light was infected with either SP Opto EL222 containing the gene of a functional EL222 protein or phages with a truncated version of EL222 (10<SUP>7</SUP> PFU/ml). The cultures were split and cultivated in the dark or under blue light illumination pulses (15 s ON, 45 s OFF; 3 W/m<SUP>2</SUP>) for 3 h at 37 °C. Afterwards, one-tenth of the culture volume was used to infect a fresh AP_light culture of OD600 0.6, which was again cultivated for 3 h either in the dark or under blue light irradiation. These steps were repeated three times (cultivation time: 4 x 3 h). Cultures for phage propagation testing under the respective conditions were performed in duplicates. Samples were taken from the final cultures and a plaque assay was performed. The phage titer of the respective cultures was calculated and plotted in the bar chart below (Fig: 3). We detected a more than 3-fold increased phage titer of SP Opto EL222 under blue light irradiation compared to the dark state, whereas the non-binding variant exhibited no significant difference. 




 
<DIV class="col-lg-12 col-md-12 col-xs-12"><DIV class="mdl-shadow--4dp" style="padding:20px; float:left}"><DIV class="img-caption title"><STRONG><B>
      
                Figure 3: Increase of Phage Propagation under Blue Light using pBLind-geneIII Cassette
    </B></STRONG></DIV><DIV class="img-caption subtitle">
      
                Phage titers of SP Opto EL222 and a non-binding variant propagated on AP_light in the dark and under blue light irradiation after four passages were determined by plaque assays. Host cell cultures infected with SP Opto EL222 and cultured under blue light conditions demonstrated a more than 3-fold higher phage titer than the culture cultivated in the dark (left side). The infection with phages containing the non-binding variant of EL222 exhibited no significant difference in phage titer (right side). It was notable that for this variant the phage titer was slightly decreased upon light irradiation. The respective plaque assays are shown below the bar chart.
            
    </DIV></DIV></DIV></DIV></DIV></DIV><DIV class="container"><DIV class="row"><DIV class="col-lg-12 col-md-12 col-xs-12 content" style="display: inline-block; padding-bottom: 10px; padding-top:10px;"><H2>AP_dark: Testing of the Psp-EL222-BR-gIII Expression Cassette</H2>
To determine the influence of light and dark states on the SP Opto EL222 phage propagation with AP_dark, a phage propagation assay similar to AP testing of AP_light was performed. As previous experiments demonstrated a significantly increased phage propagation compared to AP_light, the cultivation time per cycle was reduced to one hour. All other parameters of the propagation assay remained unchanged. Samples were taken, and phage titer were determined by plaque assays. Tests with the SP non-binding EL222 variant displayed much higher phage titer, which were not influenced by light. After one hour of incubation, cultures incubated in the dark showed a SP Opto phage titer nearly twice as high as the respective sample cultivated upon blue light. Two hours and two passages later, there was no significant difference between the phage titer of cultures cultivated in the light or in the dark.
    
 

 
<DIV class="col-lg-12 col-md-12 col-xs-12"><DIV class="mdl-shadow--4dp" style="padding:20px; float:left}"><DIV class="img-caption title"><STRONG><B>
      
                Figure 4: Increase of Phage Propagation in the Dark using Ppsp-EL222-BR-geneIII Cassette
    </B></STRONG></DIV><DIV class="img-caption subtitle">
      
                Phage titers of SP Opto EL222 propagated on AP_dark upon blue light irradiation and in the dark after one and three hours of cultivation (two passages) were determined by plaque assays. After one hour of cultivation host cell cultures infected with SP Opto EL222 and cultured in the dark nearly demonstrated a phage titer twice as high as the culture cultivated upon light irradiation (left side). Two hours later, this result could not be confirmed as the phage titer of cultures cultivated under blue light irradiation and in the dark were similar (right side). 
            
    </DIV></DIV></DIV></DIV></DIV></DIV><DIV class="container"><DIV class="row"><DIV class="col-lg-12 col-md-12 col-xs-12 content" style="display: inline-block; padding-bottom: 10px; padding-top:10px;"><H1>Outlook</H1><H2>Optogenetic Tools as Modulator of Selection Stringency</H2>

In this subproject, we were able to demonstrate that a geneIII-dependent phage propagation can be coupled to an optogenetic system, which in our case led to an increased phage propagation upon blue light irradiation. Thereby, we enrich our evolution toolbox with a powerful tool: the first optogenetic modulator of selection stringency. This tools offers the scientific community to modulate the selection stringency of PACE and PREDCEL experiments during the initial selection phase, and allows a simple and fast adaptation of desired protein functions. 
Our blue light induced pBLind-geneIII expression cassette possesses a more than 3-fold increase in phage propagation upon blue light irradiation, which corresponds to values for pBLind described in literature <X-REF>CG04</X-REF>. For the engineered Ppsp-EL222-BR-gIII cassette only a short-time impact could be observed. Further optimization could be reached by altered light-intensity, different ON/OFF-terms of the illumination pulses, and adapted cultivation times.



<H2>Optimization of Optogenetic Tools with PREDCEL</H2>

The optogenetic toolbox provides numerous applications in the field of synthetic biology. Nevertheless, optogenetic systems for prokaryotes have application issues that could be optimized by PACE/PREDCEL based directed evolution methods. Host cell strains carrying an accessory plasmid, that increases or reduces geneIII expression in a light dependent manner can be used to optimize the corresponding optogenetic system. AP_light and AP_dark of our OptoSELECT system can be used to improve the light-sensitive transcription factor EL222 and optimizes the binding affinity upon blue light irradiation in the dark as shown in Figure 5. By propagating geneIII-deficient phages containing EL222 reiteratively on AP_light cultures under blue light conditions, and AP_dark cultures in the dark, respectively, the EL222 evolves towards more efficient optogenetic system with lower leakiness.

 
<DIV class="col-lg-12 col-md-12 col-xs-12"><DIV class="mdl-shadow--4dp" style="padding:20px; float:left}"><DIV class="img-caption title"><STRONG><B>
      
                Figure 5: Synthetic Gene Circuit for Optimization of EL222 in PREDCEL
    </B></STRONG></DIV><DIV class="img-caption subtitle">
      
                Propagation of SP Opto on AP_light cultures upon blue light irradiation results in an enrichment of phages that accumulated favorable mutations, which increases expression through EL222. To prevent EL222 variants that bind DNA independent of illumination conditions, a negative selection is necessary. Therefore, a further evolved phage library is mixed with an AP_dark culture cultivated in the dark. Phages carrying an EL222 variant that binds despite dark cannot propagate in contrast to non-binding variants. 
            
    </DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV style="background-color: white; padding-top: 20px;"><DIV class="container"><DIV class="row"><DIV class="col-lg-12 col-md-12 col-xs-12 content" style="display: inline-block;"><DIV class="row content"><DIV class="col-lg-12 col-md-12 col-xs-12" style="padding-top: 20px; padding-bottom: 50px; display: block"><DIV class="mdl-shadow--4dp" style="padding: 20px"><H3> References </H3></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><SECTION id="footer-sec" style="background-color: #222 !important"><DIV class="container"><DIV class="row" style="padding-top: 30px;"><DIV class="col-md-3"><H4 style="text-align: center">Quote</H4><P><I>“”</I></P></DIV><DIV class="col-md-3"><H4 style="text-align: center"> Useful <STRONG>Links</STRONG></H4><P><A href="https://igem.org/Special:SpecialPages">
			Special pages
		</A></P><P><A href="https://igem.org/Main_Page">
			Main Page
		</A></P></DIV><DIV class="col-md-3"><H4 style="text-align: center"> Follow us on </H4><DIV style="text-align: center;"><SPAN style="color: grey; text-align: center;">&amp;</SPAN></DIV></DIV><DIV class="col-md-3"><H4 style="text-align: center"> Contact us </H4><P>

               iGEM-Heidelberg2017@bioquant.uni-heidelberg.de
               
		 Im Neuenheimer Feld 267
		
		69120 Heidelberg
                    </P></DIV></DIV></DIV></SECTION></DIV></DIV></DIV></DIV></BODY></HTML>