<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Heidelberg_Validation skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Heidelberg/Validation</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container"><DIV class="navbar-header"><A type="button" data-canvas="body" id="sidebarCollapse" class="navbar-toggle" data-toggle="offcanvas" data-target="#sidebar" style="color: #393939;margin-top: 9px;">NAVIGATION
                    </A></DIV><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav pull-right"><LI class="dropdown"><A href="https://2017.igem.org/Team:Heidelberg/Team">People </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Team">Team</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Attributions">Attributions</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Sponsoring">Sponsoring</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/For_Judges">For Judges</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Jamboree">Giant Jamboree</A></LI></UL><LI class="dropdown"><A href="https://2017.igem.org/Team:Heidelberg/Description">Project </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Description">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Design">Background &amp; Design </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Pace">PACE</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Predcel">PREDCEL</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Results">Results Summary</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Validation">Software Validation</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Organosilicons">Organosilicons</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Protein_Interaction">Protein Interaction</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Cytochrome_Engineering">Cytochrome Engineering</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Optogenetics">Optogenetics</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/CRISPR">CRISPR Cas9</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/InterLab">InterLab Study</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Achievements">Achievements</A></LI><LI class="dropdown"><A href="https://2017.igem.org/Team:Heidelberg/Parts">Parts </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Basic_Part">Basic Parts</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Composite_Part">Composite Parts</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Improved_Part">Improved Part</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Part_Collection">Parts Collection</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Part_List">Part List</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/RFC">RFC</A></LI><LI class="dropdown"><A href="https://2017.igem.org/Team:Heidelberg/Model">Modeling </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Phage_Titer">Phage Titer</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Mutagenesis_Induction">Mutagenesis Induction</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Mutation_Rate_Estimation">Mutation Rate Estimation</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Lagoon_Contamination">Lagoon Contamination</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Medium_Consumption">Medium Consumption</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Tools">Interactive Web Tools</A></LI><LI class="dropdown"><A href="https://2017.igem.org/Team:Heidelberg/Software">Software </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software">AiGEM Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software/DeeProtein">DeeProtein</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software/GAIA">GAIA</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software/SafetyNet">SafetyNet</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software/MAWS">MAWS 2.0</A></LI><LI class="dropdown"><A href="https://2017.igem.org/Team:Heidelberg/Human_Practices">Human Practices </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Human_Practices">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/HP/Gold_Integrated">Integrated Human Practices</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Interviews">Expert Interviews</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Safety">Safety &amp; Security</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Engagement">Public Engagement</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Education">Education</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Collaborations">Collaborations</A></LI><LI class="dropdown"><A href="https://2017.igem.org/Team:Heidelberg/Notebook">Notebook </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Notebook">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Experiments">Methods</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Materials">Materials</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Database">Notebook Database</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Toolbox">Toolbox Guide</A></LI></DIV></DIV><NAV id="sidebar" style="z-index: 10001; background-color: white;border: none;box-shadow:1px 0px 0px #393939;" class="navmenu navmenu-default navmenu-fixed-left offcanvas" role="navigation"><A class="navmenu-brand" href="https://2017.igem.org/Team:Heidelberg"> iGEM TEAM HEIDELBERG 2017</A><DIV id="sidebarHide" data-toggle="offcanvas" data-target="#sidebar" style="padding-left: 20px;"><A style="font-family: 'Josefin Sans' !important; font-size: 16px; line-height: 18px; height: 18px;"> COLLAPSE NAVIGATION </A></DIV><UL class="nav navmenu-nav" id="sidenav"><LI><A data-toggle="collapse" data-target="#People_side">People </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Team">Team</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Attributions">Attributions</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Sponsoring">Sponsoring</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/For_Judges">For Judges</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Jamboree">Giant Jamboree</A></LI></UL><LI><A data-toggle="collapse" data-target="#Project_side">Project </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Description">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Design">Background &amp; Design </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Pace">PACE</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Predcel">PREDCEL</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Results">Results Summary</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Validation">Software Validation</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Organosilicons">Organosilicons</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Protein_Interaction">Protein Interaction</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Cytochrome_Engineering">Cytochrome Engineering</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Optogenetics">Optogenetics</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/CRISPR">CRISPR Cas9</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/InterLab">InterLab Study</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Achievements">Achievements</A></LI><LI><A data-toggle="collapse" data-target="#Parts_side">Parts </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Basic_Part">Basic Parts</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Composite_Part">Composite Parts</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Improved_Part">Improved Part</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Part_Collection">Parts Collection</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Part_List">Part List</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/RFC">RFC</A></LI><LI><A data-toggle="collapse" data-target="#Model_side">Modeling </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Phage_Titer">Phage Titer</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Mutagenesis_Induction">Mutagenesis Induction</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Mutation_Rate_Estimation">Mutation Rate Estimation</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Lagoon_Contamination">Lagoon Contamination</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Medium_Consumption">Medium Consumption</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Tools">Interactive Web Tools</A></LI><LI><A data-toggle="collapse" data-target="#AI_side">Software </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software">AiGEM Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software/DeeProtein">DeeProtein</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software/GAIA">GAIA</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software/SafetyNet">SafetyNet</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software/MAWS">MAWS 2.0</A></LI><LI><A data-toggle="collapse" data-target="#HP_side">Human Practices </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Human_Practices">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/HP/Gold_Integrated">Integrated Human Practices</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Interviews">Expert Interviews</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Safety">Safety &amp; Security</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Engagement">Public Engagement</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Education">Education</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Collaborations">Collaborations</A></LI><LI><A data-toggle="collapse" data-target="#Notebook_side">Notebook </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Notebook">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Experiments">Methods</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Materials">Materials</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Database">Notebook Database</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Toolbox">Toolbox Guide</A></LI></NAV><DIV style="background-color: white !important; padding: 0 !important; margin: 0 !important;"><DIV class="page-title" style="background-color: white !important; border-top: 0px solid #222;"><DIV class="container" style=" padding: 0 !important;"><DIV class="col-lg-12 col-md-12 col-xs-12" style="padding-right: 0 !important; padding-bottom: 0 !important; position: relative; left: -10px;"><DIV style="border-right: 5px solid #393939; padding: 20px 20px 0px 0px"><DIV class="header-title heidelberg-yellow">                
    Software Validation </DIV><DIV class="header-subtitle" style="color: #393939 !important;"> 
    From AiGEM to the bench and back </DIV></DIV></DIV></DIV></DIV><DIV class="container" style="background-color: white !important; padding: 0 !important;"><DIV class="col-lg-12 col-md-12 col-xs-12" style="padding: 0 !important; margin: 0px important!;"><DIV class="container" style="position: relative; left: -10px; border-left: 5px solid #393939; border-top: 5px solid #393939; padding: 0 !important"><DIV class="col-lg-12 col-md-12 col-xs-12 content" style="text-align: justify; text-color: #393939 !important; padding: 20px 5px 0px 20px !important; font-weight: bold !important; font-family: 'Open Sans' !important">
        

In this subproject, we aimed at evaluating the performance of <A href="https://2017.igem.org/Team:Heidelberg/Software" style="color: #000000"><B>AiGEM</B></A>, our Artificial Intelligence for Genetic Evolution Mimicking software suite in the wet lab. In particular we wanted to investigate (i) the sensitivity of AiGEM in discriminating well-functioning from less-functioning protein mutants and (ii) its power in generating functionality <I>de novo</I>. To this end, we designed two simple, but highly informative experiments. Using GAIA, we first generated a set of ~30 \(\beta\)-lactamase single and double coding mutants and computed their corresponding protein label scores. Complementary, we evaluated the efficiency of our \(\beta\)-lactamase variants in the wet lab by determining the minimal inhibitory antibiotics concentration (MIC). Despite the high degree of sequence similarity between the variants (~99 %), we observed a noticeable correlation of the enzymatic activity (MIC) measured in the wet lab and the protein label scores computed by AiGEM. Of note, some of the created, high-scoring \(\beta\)-lactamase variants even highly outperformed wild type \(\beta\)-lactamase regarding the tolerated antibiotics concentration, indicating improved catalytic activity. (ii) To demonstrate AiGEM’s ability in generating functionality <I>de novo</I>, we finally performed <I>in silico</I> evolution of novel \(\beta\)-galactosidases from a \(\beta\)-glucuronidase parent. We cloned and purified seven of the high scoring, <I>in silico</I> evolved \(\beta\)-glucuronidase offsprings, one of which showed \(\beta\)-galactosidase activity similar to the wild type \(\beta\)-galactosidase in a quantitative PNPG calorimetric assay. We can conclude, that the AiGEM software suite bears great potential towards <I>in silico</I> directed evolution and is highly accelerating the directed evolution of proteins towards desired functions. 
    
    </DIV></DIV><DIV class="container"><DIV class="row"><DIV class="col-lg-12 col-md-12 col-xs-12 content" style="display: inline-block; padding-bottom: 10px; padding-top:10px;"><H1 id="lac">Evolution of \(\beta\)-Lactamases</H1><H2>Motivation</H2>
The deep neural network DeeProtein represents the heart of our AiGEM software suite. DeeProtein was trained on about 8 million sequences of the UniProt database to grasp the complex sequence to function relation in proteins. It is able to categorize a sequence multimodal into 886 classes of gene-ontology (GO) terms of the molecular function GO graph. As gene ontology terms are labels for protein functionality, we hypothesize that it is possible to assert protein activity based on the learned representation of the sequence to function relation in DeeProtein. To back this claim we set out for a comprehensive validation of the DeeProtein classification score. We applied GAIA, interfaced with a DeeProtein variant specifically trained on \(\beta\)-lactamases, to predict a set of \(\beta\)-lactamase sequences matching the following criterion: The set should contain a broad range of variants with higher and lower DeeProtein classification scores compared to the wildtype.
As a measure for enzyme activity we state the minimal inhibitory concentration (MIC) of carbenicillin for each candidate in the set. We demonstrate a correlation between the MIC of carbenicillin and the average DeeProtein classification score of the screened candidates. Further we improve the performance of our deep neural network by incorporating the generated wet-lab data into our training process.<H2>Experimental Design - Software</H2><H3>Definition of Mutationsites</H3>
In order to facilitate the cloning process in the wetlab and as a consequence of our limited ressource we defined patches on the \(\beta\)-lactamase sequence to introduce mutations to. The patches were defined after manual assertion of the structure in pyMOL under the premisse of a borad residue spectrum. Thus we selected patches forming the pocket, as well as central structural elements of the \(\beta\)-lactamase enzyme.


<DIV class="col-lg-12 col-md-12 col-xs-12"><DIV class="mdl-shadow--4dp" style="padding:20px; float:left}"><DIV class="img-caption title"><STRONG><B>
      
            	Figure 1: Defined mutation sites on the \(beta\)-Lactamase sequence
    </B></STRONG></DIV><DIV class="img-caption subtitle">
      
Displayed is the wildtype \(\beta\)-lactamase (PDB-code: 1M40), the catalytic residues are colored in yellow. The sequence patches defined for mutagenesis are colored in green: Fragment A, 69-73, light blue: Fragment B, 104-107,  magenta: Fragment C, 129-133, light green: Fragment D: 166-171, purple: Fragement E, 215-220 and in blue: Fragement F, 237-245. All patches are located closely around the active site, where fragments A, C, E and F partly form the pocket.</DIV></DIV></DIV><H3>GAIA preferences and Variant selection</H3>
We ran GAIA separately for each mutative window for up to 1000 generations in a single mutation mode: The rate of mutation was limited to one amino acid substitution per generation with one initial mutation. Additionally, we ran GAIA in double mutation mode over the combined frames for the same number of generations.
For each generation, the top five suggested candidates were saved and added to the pool of suggested sequences. Subsequently, we selected a subset of the proposed sequence pool for wet lab validation under the premise of covering a broad activity spectrum. Thus, we selected variants scoring higher than the wildtype and mutants scoring lower than the wildtype. A table displaying the scores for each mutation is depicted at the <A href="https://2017.igem.org/wiki/index.php?title=Team:Heidelberg/Validation#Appendix">end of the page</A>.<H3>Experimental Design - Wet-lab</H3><H4>Construct Design</H4>
To create a plasmid that is compatible with other biobrick parts, we decided to clone the ß-lactamase cassette into the pSB1C3 backbone. The ampicillin resistance gene was obtained from pSB1A3 and was introduced into the biobrick backbone via Golden Gate cloning. The BsaI restriction site, which is located inside the ß-lactamase coding sequence was previously removed to facilitate easy and fast cloning. The second antibiotic resistance cassette, derived by the pSB1C3 backbone is important, because it enabled us to select for the correct constructs, even when the ß-lactamase gene was not active or activity heavily reduced. The mutations, we cloned were located within six different windows. Cloning of these variants were performed by Golden Gate. The different parts of the plasmid were PCR amplified and the mutations were placed in primer extensions. Subsequently, the mutated plasmid was assembled with Golden Gate cloning. In this way, we generated single, double and triple mutants of the gene.<H4>Determination of the Minimal Inhibitory Concentration</H4>
The activity of an antibiotic resistance can be determined via the minimal inhibitory concentration (MIC). The MIC is the minimum concentration is the concentration, at which the growth of an microorganism, in our case <I>E. coli</I>  can survive. At mic50, for instance, the growth of a culture is slowed down by fifty percent. Typical methods to determine MICs are broth or agar dilutions. The principle is as simple as effective. The microorganism is cultured in liquid medium or on plates with different concentrations of the inhibiting agent <X-REF>JM10291</X-REF>. With this method, concentrations which are tolerated by the organism and those who inhibit growth can be determined. However, such an assay gives no information about the kind of toxicity e.g. whether the chemical is cytotoxic or cytostatic.
In our case, we want to evaluate differences in the activity of a well characterized protein. Consequently, such an dilution method is ideal. It is easy to setup, very robust and gives exactly the information that is needed for the characterization.
To start the assay, colonies were picked into LB medium with 100 µg/ml chloramphenicol, but without carbenicillin and were grown to the stationary phase for 20 h. Then, deep well plates with 500 µl LB broth per well were prepared. The medium was supplemented with 100 µg/ml chloramphenicol and for each mutant, eight wells with different carbenicillin concentrations, ranging from 10-1280 µg/ml were prepared. Each well was inoculated with 2 µl of the proculture. The new cultures were incubated another 8 h. Finally, the OD600 of each well was measured. By this measure we were able to assert the growth ability of each variant.<H2 id="lacResults">Results</H2><H3>Results - Wet-lab</H3>
The MIC of 37 cloned variants, a positive control with the wildtype ß-lactamase and a negative control, without an ampicillin resistance were tested within a carbenicillin concentration range between 10 µg/ml and 1280 µg/ml. When the OD600 was measured, a value of 0.08 was set as cutoff between bacteria that were resistant to a specific concentration and those that were not. The negative control could not grow under all conditions. Thus the lowest tested concentration of 10 µg/ml was already above the MIC of  <I>E. coli</I>. 13 of the 37 tested variants did not show any significant ß-lactamase activity and did not grow at any carbenicillin concentration. As expected, the majority, eleven of thirteen, were double and triple mutants, whereas only two single mutants lead to the destruction of enzyme activity. Interestingly, these two single mutations, D129L and K71W appear in 10 of the 13 catalytically dead mutants, which underlines their meaning for enzyme activity. Furthermore, it is noteworthy, that one double mutant, D129L, L218D has a MIC above 640 µg/ml. This suggests that L218D can somehow compensate for the negative effect of D129L.
Several variants, 12, had their MIC in the range that was tested in our assay. We proved that it is possible to gradually influence enzyme activity with help of our software. Another 11 mutants, as well as the wildtype ß-lactamase could grow under all conditions. Interestingly, only half of the variants with high ß-lactamase activity (6 of 12), were single mutants, whereas the remaining five variants had two or three mutations. Within this set of mutations, M67G and P164G/H are very prominent. These exchanges don’t have drastic effects on the enzyme activity. However these mutations appear also in variants with weak or no activity.
As a result, we decided to perform a second test for these candidates with higher antibiotic concentrations. The three highest carbenicillin concentrations of the first data set were included and the range was extended to a maximum concentration of 19.2 mg/ml. While several candidates turned out to have a weaker activity than the wildtype protein, five of them showed improved properties. The two best candidates could even grow at 19.2 mg/ml. The variants were contained one point mutation each, E102F and M67G. These most benefitial mutations, appear in many of our better candidates as well, which underlines there functionality.<DIV class="col-lg-6 col-md-12 col-xs-12 pull-left image-box-left"><DIV class="mdl-shadow--4dp" style="padding:20px; float:left"><DIV class="img-caption title"><STRONG><B>
      
            	Figure 2: Gowth Behaviour of \(\beta\)-Lactamase Mutants under Different Carbenicillin Concentrations
    </B></STRONG></DIV><DIV class="img-caption subtitle">
      
The plot shows the OD600 of the different \(\beta\)-lactamase mutants at different carbenicillin concentrations. The pool of mutants is very heterogenous. Some proteins are not active at all, some show different grades of activity and 11 mutants, as well as the wildtype enzyme can grow in all conditions.</DIV></DIV></DIV><DIV class="col-lg-6 col-md-12 col-xs-12 pull-left image-box-left"><DIV class="mdl-shadow--4dp" style="padding:20px; float:left"><DIV class="img-caption title"><STRONG><B>
      
            	Figure 3: Gowth Behaviour of \(\beta\)-Lactamase Mutants under Elevated Carbenicillin Concentrations
    </B></STRONG></DIV><DIV class="img-caption subtitle">
      
The plot depicts the \(OD_{600}\) of the \(\beta\)-lactamase mutants that could survive at 1280 \(\frac{µg}{ml}\) in the first assay, under more stringent conditions. The majority of these \(\beta\)-lactamases have a slightly weaker activity than the wildtype, but we could determine five variants with a higher enzymatic activity.

    </DIV></DIV></DIV></DIV></DIV></DIV><DIV class="container"><DIV class="row"><DIV class="col-lg-12 col-md-12 col-xs-12 content" style="display: inline-block; padding-bottom: 10px; padding-top:10px;"><H3>Results - Software</H3>
We calculated a MIC-score from the measured data points for all candidates, by first applying a threshold on the measured relative \(OD_{600}\) at 0.08. As the \(OD_{600}\) was measured in a plate reader it can only be seen as relative measure of growth. Thus, we consider any \(OD_{600}\) below 0.08 as inhibited (0) and higher ODs as growing (1). From the thresholded values, we then calculated the MIC-score as the number of consecutive observed data points until the first carbenicillin concentration where the \(OD_{600}\) fell below 0.08. Next, the DeeProtein scores were averaged among the respective MIC-score intervals and the mean DeeProtein score was plotted against the MIC of carbenicillin for all double mutants. The average DeeProtein score correlates with the MIC with an correlation coefficient of 0.6 as displayed in fig 4.<DIV class="col-lg-12 col-md-12 col-xs-12"><DIV class="mdl-shadow--4dp" style="padding:20px; float:left}"><DIV class="img-caption title"><STRONG><B>
      
      Figure 4: The DeeProtein classification score for screened \(\beta\)-lactamase variants correlates with the MIC of Carbenicillin. 
    </B></STRONG></DIV><DIV class="img-caption subtitle">
      
      he average DeeProtein classification scores assigned to samples in the MIC-score bins are depicted as black dots. The red line is the fitted linear model. Samples assigned with a high classification score tend to sustain higher carbenicillin concentrations, whereas a low classification score is assigned to variants with a low MIC.

    </DIV></DIV></DIV>


To improve the predictive power of DeeProtein, we subsequently incorporated the collected data into our training set. with the aim to better catch the effect of single amino acid substitutions on enzyme activity.</DIV></DIV></DIV><DIV class="container"><DIV class="row"><DIV class="col-lg-12 col-md-12 col-xs-12 content" style="display: inline-block; padding-bottom: 10px; padding-top:10px;"><H2>Discussion</H2>
In course of the \(\beta\)-lactamase subproject we set the DeeProtein classification score in correlation to the MIC as a measure for enzyme activity. As we obtained a positive correlation, we further demonstrate, that by application of the AiGEM software suite it is possible to gradually modulate enzyme activity in \(\beta\)-Lactamases.
Based on the mutations suggested by GAIA for postitive and negative evolution, we generated a set of mutants spanning the whole spectrum of activity. We obtained catalytically dead candidates as well as many proteins with intermediate activity. Most importantly however we also obtained a set of five improved variants, some of them exceeding the wildtype MIC by far. As our experimental results are in correspondence to the averaged DeeProtein classification scores for each MIC-score bin, we state that the scores of the underlying DeeProtein model can be applied as a measure for activity in single and double mutants. However, as all variants were correctly classified as \(\beta\)-lactamases, the assigned scores were all situated in a short intervall. In order to address this issue we incorporated the collected data from the single and double mutations into out training set, forming an integrative cycle of software application, wetlab data collection and software improvement.
The overall set of mutants revealed four different amino acid substitutions that seem to have drastic effects on protein function. D129L and K71W lead to a complete loss of activity. Also in combination with other mutations, no or only weak activity could be detected. M67G and E102F instead, were very prominent in the improved \(beta\)-lactamase subset.  The negative effect of K71W can easily be explained. A lysine in the enzymatic pocket is replaced by a tryptophane, disrupting both pocket shape and polarity. In addition the much larger tryptophane inhibits proper access of the substrates. The D129L substitution in contrast, entails an increase of hydrophobicity in that area, potentially leading to the loss of a hydrogen bond. GAIA scored candidates, containing one of these two mutants with the lower scores, underlining its potential to classify protein function.</DIV></DIV></DIV><DIV class="container"><DIV class="row"><DIV class="col-lg-12 col-md-12 col-xs-12 content" style="display: inline-block; padding-bottom: 10px; padding-top:10px;"><H1 id="gusgal">Reprogrammation of \(\beta\)-Glucuronidase</H1><H3>Motivation</H3>
  The main objective of the AiGEM software suite is the improvement of directed evolution experiments. As the protein space is tremendous in size and impossible to assert with brute force or random walk methods, a directed evolution tool needs to reduce the combinatory complexity of the protein space. Using <A href="https://2017.igem.org/Team:Heidelberg/Software/DeeProtein#Representation">DeeProtein</A>, we learned the complex relation between protein sequence and protein function, thus the properties of the thin manifold of functional protein sequences. To harness this learned representation in a <A href="https://2017.igem.org/Team:Heidelberg/Software/GAIA#Generative">generative approach</A> we developed our directed evolution tool <A href="https://2017.igem.org/Team:Heidelberg/Software/GAIA">GAIA</A> (Genetically Artificially Intelligent Algorithm). GAIA deploys the pre-trained DeeProtein models as scoring function, to assert the class probability of a certain protein function distribution during the evolution process. We hypothesize that by maximization of the class probability of the goal protein function through introduction of amino acid substitutions on the entry sequence its function gets shifted towards the goal term.
To demonstrate the capabilities of GAIA we set out to reprogram the E. Coli \(\beta\)-glucuronidase (GUS) towards \(\beta\)-galactosidase (GAL) activity <I>in silico</I>. Sequences were predicted by GAIA and subsequently the enzyme kinetics were asserted in the wet-lab.<H3>Experimental Setup - Software</H3>
We prepared our experiments by performing <A href="https://2017.igem.org/Team:Heidelberg/Validation#MD">equilibration molecular dynamics</A> simulations on the wildtype and a known variant <X-REF>matsumura2001vitro</X-REF>. Based on equilibration molecular dynamics simulations of the wildtype GUS and the mutant introduced by Matsumura et al.<X-REF>matsumura2001vitro</X-REF>, we determined three mutative windows on the GUS sequence. The limitation of mutations to certain sequence windows was necessary to facilitate the cloning procedure of the mutants in the wet-lab. Subsequently, the GUS with its defined mutative regions was submitted to GAIA with the objective of maximization of the \(\beta\)-galactosidase-activity GO-term (GO:0004565). GAIA was run for 1000 generations and the top five candidates of every generation were added to the candidate library. Thus, we picked five candidates from the library to test them in the wet-lab.<H3 id="MD">Equilibration Molecular Dynamics Simulations</H3>
In order to assert the effects of the introduced mutations on the \(\beta\)-glucuronidase structure, we performed equilibration molecular dynamics (MD) simulations on the wild-type and the mutated variant introduced by matsumura et al. <X-REF>matsumura2001vitro</X-REF>. All simulations were performed in openMM <X-REF>eastman2010openmm</X-REF> with the amber99 forcefield <X-REF>pearlman1995amber</X-REF>. The B-chain of the \(\beta\)-glucuronidase (PDB-code: 3lpg) served as the wildtype and basis for <I>in silico</I> mutagenesis. First all selenomethionines in the sequence were corrected to methionines and the ligand was excluded. Subsequently mutations were introduced in pyMOL <X-REF>delano2002pymol</X-REF> : D508G, T509A, S557P, N566S, K568T to obtain the described GUS variant <X-REF>matsumura2001vitro</X-REF>.
 The apo structure was protonated at pH 6.5 using the modeller class of the openMM library, then the protein was solvated in a cubic box of 10nm with the tip3p water model. The system was then equiibrated for charge with sodium and chloride ions.<DIV class="col-lg-12 col-md-12 col-xs-12"><DIV class="mdl-shadow--4dp" style="padding:20px; float:left}"><DIV class="img-caption title"><STRONG><B>
      
Figure 5: Introduced mutations have little effect on the protein folding in equilibration MD
    </B></STRONG></DIV><DIV class="img-caption subtitle">
      
 Superimposed structures for the wildtype \(\beta\)-glucuronidase (A) (PDB-code: 3lpg) and the mutated version (B) introduced by Matsumura <X-REF>matsumura2001vitro</X-REF>. In both depictions the grey structure is the first frame of the equilibration trajectory and the blue structure is the protein after equilibration MD. The catalytic residues are colored yellow, the symbolic ligand (was not included in MD simulations) in green. Matsumura mutations are colored magenta. The grey and blue structures vary just slightly, thus the introduced mutations did not affect the overall protein folding.</DIV></DIV></DIV>

Subsequently the system was heated from 0 to 300K in 100K intervalls. Each intervall was NPT-simulated for 10000 steps with a stepsize of 2fs. After the heating was completed the system was NPT simulated for another 10000000 steps (20ns) for equilibration. The last frame of the resulting trajectory was then aligned to the input structure (Fig. 5) and a the global root-mean-square deviation of atomic positions (RMSD) was calculated (Tab. 1).<DIV class="col-lg-12 col-md-12 col-xs-12 content" style="padding-top: 20px; padding-bottom: 20px"><P class="content"><B>
Table 1: Calculated RMSD-values after equilibration for the wildtype \(\beta\)-glucuronidase and the Matsumura variant.</B>
    
As the RMSD-values are comparable there is no devastating effect of the mutations suggested by Matsumura <X-REF>matsumura2001vitro</X-REF> on the protein folding. Thus we consider the region where the mutations were introduced as mutable regions.

  </P><DIV class="table-slot" style="overflow-x: auto"><TABLE class="table table-bordered mdl-shadow--4dp" xsscleaned="overflow-x: auto !important"><THEAD style="background-color: #005493 !important;"><TR><TH>Structure</TH><TH>RMSD (Angstroem)</TH></TR></THEAD><TBODY><TR><TD>Wildtype (3lpg)</TD><TD>1.412</TD></TR><TR><TD>Matsumura mutant</TD><TD>1.728</TD></TR></TBODY></TABLE></DIV></DIV></DIV></DIV></DIV><DIV class="container"><DIV class="row"><DIV class="col-lg-12 col-md-12 col-xs-12 content" style="display: inline-block; padding-bottom: 10px; padding-top:10px;"><H3>Definition of Mutationsites</H3>
To facilitate the cloning process in the wetlab and accelerate the production, we limited the mutagnesis on the glucuronidase sequence to three patches (Fig. 6). The defined patches are located around the active site with fragement A and C partly forming the enzymatic pocket. All positions mutated by Matsumura are contained in the defined patches.<DIV class="col-lg-12 col-md-12 col-xs-12"><DIV class="mdl-shadow--4dp" style="padding:20px; float:left}"><DIV class="img-caption title"><STRONG><B>
      
Figure 4: Patches defined for mutagenesis on the \(\beta\)-glucuronidase sequence. 
    </B></STRONG></DIV><DIV class="img-caption subtitle">
      
Displayed is the wildtype (PDB-code: 3lpg) \(\beta\)-glucuronidase, the catalytic residues are colored in yellow. The sequence patches defined for mutagenesis are colored in green: Fragment A, 351-371, blue: Fragment B, 506-512 and magenta: Fragment C, 548-568. All patches are located closely at the active site, where fragments A and C partly form the pocket.</DIV></DIV></DIV></DIV></DIV></DIV><DIV class="container"><DIV class="row"><DIV class="col-lg-12 col-md-12 col-xs-12 content" style="display: inline-block; padding-bottom: 10px; padding-top:10px;"><H3>Variant Selection</H3>
As our resources were limited, it was not feasible for us to synthesize all variants suggested by GAIA. Thus we had to carefully select the variants to be asserted in the wet lab. Matsumura et al. <X-REF>matsumura2001vitro</X-REF> reported a set of single amino acid substitutions and a reprogrammed GUS with 5 amino acid substitutions. In order to compare the predictions made by GAIA with the variants reported by Matsumura, we incorporated single amino acid mutations on positions matching those of the Matsumura variants. For two of these positions, D508 and T509 we additionally assembled the exact variants reported by Matsumura et. al for direct comparison to the predictions made by GAIA.
For the sequence prediction we ran GAIA over 2000 generations and constrained the mutation sites to the determined sequence areas. The number of mutations per sequence was limited to 10 with an initial mutation rate of 10. The mutation rate had a linear decay of 1 mutation every 300 generations, to facilitate conversion and backmutations were allowed throughout the whole run.
Next, the predicted variants were manually curated and investigated for steric conflicts. The single amino acid substitutions were selected with respect to the overall mutation frequencies in these postitions.
The resulting set of variants is listed in tab 2.<DIV class="col-lg-12 col-md-12 col-xs-12 content" style="padding-top: 20px; padding-bottom: 20px"><P class="content"><B>
Table 2: Set of synthesized GUS variants.</B>
    
We synthesized a set of 7 GUS variants. In order to be able to compare the power of <I>in silico</I> evolution by GAIA the set comprises single amino acid mutations on the positions reported by Matsumura et al <X-REF>matsumura2001vitro</X-REF>. These positions are D508, T509, K568 and N556. For two of these positions, D508 and T509 we additionally assembled the exact variants reported by Matsumura et. al for direct comparison to the predictions made by GAIA.

  </P><DIV class="table-slot" style="overflow-x: auto"><TABLE class="table table-bordered mdl-shadow--4dp" xsscleaned="overflow-x: auto !important"><THEAD style="background-color: #005493 !important;"><TR><TH>Variant</TH><TH>Fragment</TH></TR></THEAD><TBODY><TR><TD>D508V</TD><TD>B</TD></TR><TR><TD>D508G</TD><TD>B</TD></TR><TR><TD>T509L</TD><TD>B</TD></TR><TR><TD>T509A</TD><TD>B</TD></TR><TR><TD>K568G</TD><TD>C</TD></TR><TR><TD>N556S</TD><TD>C</TD></TR><TR><TD>V355M, F357D, N358L, G364L, D508L, T509A, F551G, D553F, F554E, G565L</TD><TD>A+B+C</TD></TR></TBODY></TABLE></DIV></DIV><H4>Position 509</H4>
A plot of the calculated mutation frequencies at position 509 is depicted in Fig. 7. For position 509 GAIA suggests the insertion of hydrophobic residues. This is in correspondence to the amino acid exchange in position 509 reported by Matsumura et al <X-REF>matsumura2001vitro</X-REF>, where the threonine was exchanged for an alanine. As isoleucine and leucine are similar in charge, weight and polarity, we selected the most frequent substitution for postition 509: leucine.

<DIV class="col-lg-12 col-md-12 col-xs-12"><DIV class="mdl-shadow--4dp" style="padding:20px; float:left}"><DIV class="img-caption title"><STRONG><B>
      
Figure 7: Substitution rates over 2000 generations of GAIA evolution for position 509. 
    </B></STRONG></DIV><DIV class="img-caption subtitle">
      
GAIA selectively introduced hydrophobic residues to position 509. This is congruent to the mutant T509A reported by Matsumura et al. As isoleucine and leucine are similar in charge, weight and polarity, we selected the most frequently introduced amino acid, leucine.</DIV></DIV></DIV></DIV></DIV></DIV><DIV class="container"><DIV class="row"><DIV class="col-lg-12 col-md-12 col-xs-12 content" style="display: inline-block; padding-bottom: 10px; padding-top:10px;"><H3>Experimental Design - Wet-lab</H3><H4>Cloning of different GUS Variants</H4>
An expression plasmid of the wiltype GUS was obtained from <A href="https://www.addgene.org/61156/">AddGene</A>. In this plasmid, GUS is provided under a lac-promoter and a His-tag is attached to the C-terminus of the protein. First, the wildtype GAL was cloned into the same plasmid. Second, different mutants of the glucuronidase were created. With this method, seven different mutants were generated.<H4>Expression and Purification</H4>
After cloning, one colony of each transformation was inoculated as preculture overnight at 37°C and 220 rpm. The main cultures were inoculated with the respective volume of the overnight cultures to a final density of 0.01 at the starting point. The inducer IPTG was added after cultures reached an OD<SUB>600nm</SUB> of 0.6 - 0.8 and were afterwards incubated for 20 hours at 25°C, 220 rpm. Cells were centrifuged and lysed as described in the <A href="https://2017.igem.org/Team:Heidelberg/Experiments">protocol</A>. Lysates were subjected to the Ni-NTA cartridge and affinity purified throughout several washing steps. Purified proteins were obtained after elution with 500 mM Imidazole solution followed by dialysis. The protein concentration was determind using both, the photospectrometer and a Bradford assay.<H4>Activity Assay</H4>
To determine the activity of the different mutants, an assay with p-nitrophenyl-galactopyranoside (PNPG) as substrate was set up. PNPG is a perfect substrate for GAL. The enzyme hydrolyzes the p-nitophenyl group from the galactopyranosid. The yellow color of the product can be measured at 420 nm. The enzyme was added to different concentrations of the substrate, ranging from 0.047 mM to 3 mM final conentration and the reaction was followed over time for 45 minutes at 21 °C.
With the data, generated from this assay, the kinetic parameters according to Michaelis Menten were calculated via nonlinear regression.<H2 id="gusgalResults">Results</H2><H3>Results - Wet-lab</H3>
When the enzymes were cloned and purified, kinetic Assays were performed to reveal their activity. We tested the mutants, as well as both wildtype enzymes, GUS and GAL as controls. As expected GUS showed no enzyme activity at all (Fig. 8, 11). No significant product formation could be measured. Among the inactive proteins were two mutants, which were previously published by Matsumura et al. <X-REF>matsumura2001vitro</X-REF>. As the published activity increase was only 1.5-2 fold and our enzyme concentration was relatively low, it is no surprise, the we couldn’t determine these activities. For the wildtype GAL we could determine a \(K_{M}\) value of \(0.3466 mM\) and a \(K_{cat}\) value of \(49.38 \frac{1}{s}\). The \(K_{M}\) is higher and \(K_{cat}\) is lower, than similar values in the literature (see Juersetal et al.<X-REF>JUERSETAL..2003</X-REF>), which is mainly due to the reaction conditions, like assay buffer and temperature. The kinetic parameters that were determined can be seen in tab 4.<DIV class="col-lg-12 col-md-12 col-xs-12 content" style="padding-top: 20px; padding-bottom: 20px"><P class="content"><B>
            	Table 4: Kinetic Key Values of wildtype GAL (GAL_wt) and the best GUS mutant T509L </B></P><DIV class="table-slot" style="overflow-x: auto"><TABLE><TABLE class="table table-bordered mdl-shadow--4dp" xsscleaned="overflow-x: auto !important"><THEAD style="background-color: #B9B9B9 !important;"><TR><TH>Protein</TH><TH>\(K_{M} [mM]\)</TH><TH>\(K_{cat}[\frac{1}{s}]\)</TH><TH>\(K_{M}/K_{cat}[\frac{1}/{M \cdot min}]\)</TH></TR></THEAD><TBODY><TR><TD>Gal_wt</TD><TD>0.3466</TD><TD>49.38</TD><TD>8.5E6</TD></TR><TR><TD>GUS_T509L</TD><TD>0.1834</TD><TD>25.02</TD><TD>8.1E6</TD></TR></TBODY></TABLE><DIV class="col-lg-12 col-md-12 col-xs-12"><DIV class="mdl-shadow--4dp" style="padding:20px; float:left}"><DIV class="img-caption title"><STRONG><B>
      
            	Figure 8: Colorimetric Assay for the Determination of GUS Mutant Activity
    </B></STRONG></DIV><DIV class="img-caption subtitle">
      
            	The photo was taken after 45 min reaction time. One can clearly see that the wildtype GUS as well as most of the mutants show no activity. GUS_T409L instead has an intense yellow color.</DIV></DIV></DIV><DIV class="col-lg-12 col-md-12 col-xs-12"><DIV class="mdl-shadow--4dp" style="padding:20px; float:left}"><DIV class="img-caption title"><STRONG><B>
      
            	Figure 9: Time Course Measurement of Galactodidase Activity
    </B></STRONG></DIV><DIV class="img-caption subtitle">
      
            	The product concentration is plotted against the educt concentration for various time points after reaction initiation. The time points are given in seconds.Both Enzymes, the wildtype GAL and GUS_T509L show strong enzymatic activity on PNPG.</DIV></DIV></DIV><DIV class="col-lg-12 col-md-12 col-xs-12"><DIV class="mdl-shadow--4dp" style="padding:20px; float:left}"><DIV class="img-caption title"><STRONG><B>
      
            	Figure 10: Determination of Catalytic Constants
    </B></STRONG></DIV><DIV class="img-caption subtitle">
      
            	For the last time point that was measured, 2640 s, product concentration is plotted versus substrate concentration. \(K_{m}\) and \(K_{cat}\) were determined via nonlinear regression. They prove a high overall activity of GUS_T509L.</DIV></DIV></DIV><DIV class="col-lg-12 col-md-12 col-xs-12"><DIV class="mdl-shadow--4dp" style="padding:20px; float:left}"><DIV class="img-caption title"><STRONG><B>
      
            	Figure 11: Comparison Between the Wildtype Proteins and GUS_T509L
    </B></STRONG></DIV><DIV class="img-caption subtitle">
      
      Our assay demonstrated that the wildtype GUS has no activity on a GAL substrate. The mutant predicted by GAIA however, exhibits extraordinary enzymatic activity on the GAL substrate.</DIV></DIV></DIV><DIV class="container"><DIV class="row"><DIV class="col-lg-12 col-md-12 col-xs-12 content" style="display: inline-block; padding-bottom: 10px; padding-top:10px;"><H2>Discussion</H2>
With help of our AiGEM software suite we successfully reprogrammed a \(\beta\)-glucuronidase (GUS) variant towards \(\beta\)-galactosidase (GAL) activity. We ran equilibration molecular dynamics simulations on the wildtype and the variants reported by Matsumura et al. <X-REF>matsumura2001vitro</X-REF>. The RMSD values of the simulated structures compared to the respective input structue suggests that the amino acid exchanges introduced by Matsumura do not entail structural changes. Thus we liberately defined the mutative windows for GAIA with a constant central amino acid to facilitate the cloning procedure. As the definintion of mutative windows reduces the degrees of freedom for GAIA the outputs are bias by this constraint. Despite this constraint GAIA-scores converged within 2000 generations, however with lower overall scores than for open mutation.
    The sequences suggested by GAIA were subsequently curated manually in order to select single substitution variants. The selection was based on the mutation frequencies over all 2000 generations and an additional assertion of steric conflicts in PyMOL.
The single amino acid substitution T509L was suggested by GAIA with the highest frequency in that position (see Fig. 7) and selected based on the substitution frequency. The mutant T509L displayed a significantly higher activity compared to all other asserted variants, including the variant T509A reported by Matsumura <X-REF>matsumura2001vitro</X-REF>. GAIA correctly identified the potential of a substitution with a hydrophic residue in position 509, suggesting both the Matsumura (T509A) and the improved (T509L) mutants.
We asserted the kinects of the T509L variant in a colorimetric assay and compared it to the wildtype GAL and GUS. The mutant had a \(K_{cat}\) of \(25.02 \frac{1}{s}\) and a \(K_{M}\) of \(0.1834 mM\). This is not only significantly higher than the tested wildtype GUS, which lacked any activity, but also close to the activity of the wildtype GAL. We thus state that with the help of the AiGEM software suite, we were able to reprogram a \(\beta\) glucuronidase towards decent \(\beta\) galactosidase function fully <I>in silico</I>. 
Under our experimental conditions none of the variants reported by Matsumura displayed any significant activity. The reasons for this oberservation could lay in the different experimental conditions, differing in buffer, temperature and substrate concentration. Further Matsumura reported a maximum increase of activity of 2 log-scales compared to the wildtype for all their single amino acid substitutions. However such a slight increase in activity is immeasurable under our experimental conditions.
We demonstrated the capabilities of the AiGEM software suite in real world application in functionality transfer by fully <I>in silico</I> evolution of \(\beta\)-glucuronidases. However this very specific functionality transfer between closely related enzymes is only the first step on the path towards a universal <I>in silico</I> evolution tool. We began taking next steps by incorporating the data collected in the wetlab in the training set for our model. For a comprehensive, universal approach however much more data is needed. With automated directed evolution experiments becoming more easily applicable and more common, we hope this shortage in data will be overcome in close future.
<H1 id="Appendix">Appendix</H1><DIV class="col-lg-12 col-md-12 col-xs-12 content" style="padding-top: 20px; padding-bottom: 20px"><P class="content"><B>
Table 1: Selected single and double mutants proposed by GAIA. </B>
    
Of the pool of proposed \(\beta\)-lactamase sequences we selected 9 single and 16 double mutations for wet-lab validation.

  </P><DIV class="table-slot" style="overflow-x: auto"><TABLE class="table table-bordered mdl-shadow--4dp" xsscleaned="overflow-x: auto !important"><THEAD style="background-color: #B9B9B9 !important;"><TR><TD>Mutation</TD><TD>Fragment</TD><TD>Classification_score</TD></TR></THEAD><TBODY><TR><TD>G236L</TD><TD>F</TD><TD>0.56584185</TD></TR><TR><TD>G216C_P217F</TD><TD>E</TD><TD>0.56683183</TD></TR><TR><TD>L218D</TD><TD>E</TD><TD>0.56554008</TD></TR><TR><TD>L167G_P164L</TD><TD>D</TD><TD>0.56678349</TD></TR><TR><TD>P164G</TD><TD>D</TD><TD>0.56673431</TD></TR><TR><TD>P164H</TD><TD>D</TD><TD>0.56748658</TD></TR><TR><TD>D129L_N130I</TD><TD>C</TD><TD>0.56734717</TD></TR><TR><TD>D129L</TD><TD>C</TD><TD>0.56697899</TD></TR><TR><TD>E102F</TD><TD>B</TD><TD>0.56683093</TD></TR><TR><TD>Y103D</TD><TD>B</TD><TD>0.56666845</TD></TR><TR><TD>K71W</TD><TD>A</TD><TD>0.56605697</TD></TR><TR><TD>M67G</TD><TD>A</TD><TD>0.56762165</TD></TR><TR><TD>M67G_F70G</TD><TD>A</TD><TD>0.56777865</TD></TR><TR><TD>K71W_G236L</TD><TD>A+F</TD><TD>0.56503928</TD></TR><TR><TD>M67G_P164G</TD><TD>A+D</TD><TD>0.5675621</TD></TR><TR><TD>M67G_P164H</TD><TD>A+D</TD><TD>0.56818455</TD></TR><TR><TD>E102F_P164H</TD><TD>B+D</TD><TD>0.56749099</TD></TR><TR><TD>E102F_P164G</TD><TD>B+D</TD><TD>0.56676418</TD></TR><TR><TD>E102F_G236L</TD><TD>B+F</TD><TD>0.5658384</TD></TR><TR><TD>E102F_M67G</TD><TD>B+A</TD><TD>0.56763196</TD></TR><TR><TD>D129L_P164G</TD><TD>C+D</TD><TD>0.56688529</TD></TR><TR><TD>D129L_M67G</TD><TD>C+A</TD><TD>0.56781197</TD></TR><TR><TD>D129L_L218L</TD><TD>C+E</TD><TD>0.56570756</TD></TR><TR><TD>L218D_P164G</TD><TD>E+D</TD><TD>0.56537378</TD></TR><TR><TD>L218D_K71W</TD><TD>E+A</TD><TD>0.56467879</TD></TR><TR><TD>WT</TD><TD>None</TD><TD>0.56685823</TD></TR><TR><TD>WT</TD><TD>None</TD><TD>0.56685823</TD></TR></TBODY></TABLE></DIV></DIV></DIV></DIV></DIV><DIV style="background-color: white; padding-top: 20px;"><DIV class="container"><DIV class="row"><DIV class="col-lg-12 col-md-12 col-xs-12 content" style="display: inline-block;"><DIV class="row content"><DIV class="col-lg-12 col-md-12 col-xs-12" style="padding-top: 20px; padding-bottom: 50px; display: block"><DIV class="mdl-shadow--4dp" style="padding: 20px"><H3> References </H3></DIV></DIV></DIV></DIV></DIV></DIV></DIV><SECTION id="footer-sec" style="background-color: #222 !important"><DIV class="container"><DIV class="row" style="padding-top: 30px;"><DIV class="col-md-3"><H4 style="text-align: center">Quote</H4><P><I>“”</I></P></DIV><DIV class="col-md-3"><H4 style="text-align: center"> Useful <STRONG>Links</STRONG></H4><P><A href="https://igem.org/Special:SpecialPages">
			Special pages
		</A></P><P><A href="https://igem.org/Main_Page">
			Main Page
		</A></P></DIV><DIV class="col-md-3"><H4 style="text-align: center"> Follow us on </H4><DIV style="text-align: center;"><SPAN style="color: grey; text-align: center;">&amp;</SPAN></DIV></DIV><DIV class="col-md-3"><H4 style="text-align: center"> Contact us </H4><P>

               iGEM-Heidelberg2017@bioquant.uni-heidelberg.de
               
		 Im Neuenheimer Feld 267
		
		69120 Heidelberg
                    </P></DIV></DIV></DIV></SECTION></TABLE></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>