<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Heidelberg_Design skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Heidelberg/Design</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container"><DIV class="navbar-header"><A type="button" data-canvas="body" id="sidebarCollapse" class="navbar-toggle" data-toggle="offcanvas" data-target="#sidebar" style="color: #393939;margin-top: 9px;">NAVIGATION
                    </A></DIV><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav pull-right"><LI class="dropdown"><A href="https://2017.igem.org/Team:Heidelberg/Team">People </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Team">Team</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Attributions">Attributions</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Sponsoring">Sponsoring</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/For_Judges">For Judges</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Jamboree">Giant Jamboree</A></LI></UL><LI class="dropdown"><A href="https://2017.igem.org/Team:Heidelberg/Description">Project </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Description">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Design">Background &amp; Design </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Pace">PACE</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Predcel">PREDCEL</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Results">Results Summary</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Validation">Software Validation</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Organosilicons">Organosilicons</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Protein_Interaction">Protein Interaction</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Cytochrome_Engineering">Cytochrome Engineering</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Optogenetics">Optogenetics</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/CRISPR">CRISPR Cas9</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/InterLab">InterLab Study</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Achievements">Achievements</A></LI><LI class="dropdown"><A href="https://2017.igem.org/Team:Heidelberg/Parts">Parts </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Basic_Part">Basic Parts</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Composite_Part">Composite Parts</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Improved_Part">Improved Part</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Part_Collection">Parts Collection</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Part_List">Part List</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/RFC">RFC</A></LI><LI class="dropdown"><A href="https://2017.igem.org/Team:Heidelberg/Model">Modeling </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Phage_Titer">Phage Titer</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Mutagenesis_Induction">Mutagenesis Induction</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Mutation_Rate_Estimation">Mutation Rate Estimation</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Lagoon_Contamination">Lagoon Contamination</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Medium_Consumption">Medium Consumption</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Tools">Interactive Web Tools</A></LI><LI class="dropdown"><A href="https://2017.igem.org/Team:Heidelberg/Software">Software </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software">AiGEM Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software/DeeProtein">DeeProtein</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software/GAIA">GAIA</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software/SafetyNet">SafetyNet</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software/MAWS">MAWS 2.0</A></LI><LI class="dropdown"><A href="https://2017.igem.org/Team:Heidelberg/Human_Practices">Human Practices </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Human_Practices">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/HP/Gold_Integrated">Integrated Human Practices</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Interviews">Expert Interviews</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Safety">Safety &amp; Security</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Engagement">Public Engagement</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Education">Education</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Collaborations">Collaborations</A></LI><LI class="dropdown"><A href="https://2017.igem.org/Team:Heidelberg/Notebook">Notebook </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Notebook">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Experiments">Methods</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Materials">Materials</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Database">Notebook Database</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Toolbox">Toolbox Guide</A></LI></DIV></DIV><NAV id="sidebar" style="z-index: 10001; background-color: white;border: none;box-shadow:1px 0px 0px #393939;" class="navmenu navmenu-default navmenu-fixed-left offcanvas" role="navigation"><A class="navmenu-brand" href="https://2017.igem.org/Team:Heidelberg"> iGEM TEAM HEIDELBERG 2017</A><DIV id="sidebarHide" data-toggle="offcanvas" data-target="#sidebar" style="padding-left: 20px;"><A style="font-family: 'Josefin Sans' !important; font-size: 16px; line-height: 18px; height: 18px;"> COLLAPSE NAVIGATION </A></DIV><UL class="nav navmenu-nav" id="sidenav"><LI><A data-toggle="collapse" data-target="#People_side">People </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Team">Team</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Attributions">Attributions</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Sponsoring">Sponsoring</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/For_Judges">For Judges</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Jamboree">Giant Jamboree</A></LI></UL><LI><A data-toggle="collapse" data-target="#Project_side">Project </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Description">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Design">Background &amp; Design </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Pace">PACE</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Predcel">PREDCEL</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Results">Results Summary</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Validation">Software Validation</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Organosilicons">Organosilicons</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Protein_Interaction">Protein Interaction</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Cytochrome_Engineering">Cytochrome Engineering</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Optogenetics">Optogenetics</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/CRISPR">CRISPR Cas9</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/InterLab">InterLab Study</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Achievements">Achievements</A></LI><LI><A data-toggle="collapse" data-target="#Parts_side">Parts </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Basic_Part">Basic Parts</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Composite_Part">Composite Parts</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Improved_Part">Improved Part</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Part_Collection">Parts Collection</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Part_List">Part List</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/RFC">RFC</A></LI><LI><A data-toggle="collapse" data-target="#Model_side">Modeling </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Phage_Titer">Phage Titer</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Mutagenesis_Induction">Mutagenesis Induction</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Mutation_Rate_Estimation">Mutation Rate Estimation</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Lagoon_Contamination">Lagoon Contamination</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Medium_Consumption">Medium Consumption</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Tools">Interactive Web Tools</A></LI><LI><A data-toggle="collapse" data-target="#AI_side">Software </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software">AiGEM Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software/DeeProtein">DeeProtein</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software/GAIA">GAIA</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software/SafetyNet">SafetyNet</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software/MAWS">MAWS 2.0</A></LI><LI><A data-toggle="collapse" data-target="#HP_side">Human Practices </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Human_Practices">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/HP/Gold_Integrated">Integrated Human Practices</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Interviews">Expert Interviews</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Safety">Safety &amp; Security</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Engagement">Public Engagement</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Education">Education</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Collaborations">Collaborations</A></LI><LI><A data-toggle="collapse" data-target="#Notebook_side">Notebook </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Notebook">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Experiments">Methods</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Materials">Materials</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Database">Notebook Database</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Toolbox">Toolbox Guide</A></LI></NAV><DIV style="background-color: white !important; padding: 0 !important; padding-top: 100px !important; margin: 0 !important;"><DIV class="container" style="background-color: white !important;padding-top: 50px;"><DIV class="col-lg-12 col-md-12 col-xs-12 content"><DIV class="row" style="font-size: 25px !important; font-weight: 400 !important;text-align: left !important"><DIV class="col-lg-12 col-md-12 col-xs-12 content" style="font-size: 70px !important; font-weight: 700 !important; line-height: 70px !important; color: #222 !important">
          Our design</DIV><DIV class="col-lg-1 col-md-1 hidden-sm hidden-xs" style="font-size: 50px !important; font-weight: 700 !important; line-height: 50px !important;padding-top:20px !important; padding-bottom: 50px !important;color: #222 !important">—</DIV><DIV class="col-lg-11 col-md-11 col-xs-12" style="font-size: 50px !important; font-weight: 700 !important; line-height: 50px !important;padding-top:20px !important; padding-bottom: 50px !important;color: #222 !important">
           Interfacing In Vivo and In Silico Directed Evolution As Novel Engineering Paradigm</DIV><DIV class="col-lg-12 col-md-12 col-xs-12" style="font-size: 25px !important; font-weight: 700 !important; line-height: 25px !important;padding-top:10px !important; padding-bottom: 10px !important;color: #222 !important">
           Our Project Design as animated videos</DIV><DIV class="col-lg-6 col-md-6 col-xs-12 content" style="text-align: left !important;font-weight: 600 !important">
            Video 1: Background and general project design
            </DIV><DIV class="col-lg-6 col-md-6 col-xs-12 content" style="text-align: left !important;font-weight: 600 !important">
            Video 2: Details on in vivo directed evolution with PACE and PREDCEL
           </DIV></DIV><DIV class="row"><DIV class="col-lg-12 col-md-12 col-xs-12 content" style="padding-top: 50px;padding-bottom: 50px;"><H2>Background</H2>
„Harnessing evolution for the development of novel proteins and biomolecules for human benefit“ -
This naïve vision stood at the beginning of our this year’s iGEM project. But how should we do that?
While doing our literature research, we came across an exciting method named PACE - phage assisted
continuous evolution. This method was invented by Kevin Esvelt and David Liu at Harvard university in
2011 (Esvelt et al, Nature, 2011). The idea behind this method is rather simple: Mimic natural evolution
in bioreactor, accelerate it and direct it towards a specific goal. More precisely, PACE implements a fully
closed, extremely fast evolution cycle comprising of replication, mutation and selection to enable in vivo
directed evolution of proteins. In brief, the protein to be evolved is encoded by an M13 bacteriophage
and transferred from one E. coli host cell to the next by means of phage replication and propagation.
Importantly, the E. coli hosts express mutator genes (Badran et al, Nature Communictaions, 2015) to
increase the phage mutation rate during replication, thereby creating a highly diverse gene pool. Finally,
synthetic circuits in the E. coli host cell couple phage propagation efficiency to the fitness (i.e. function)
of the protein to be evolved, by controlling the expression of an essential phage gene. Details on the
PACE background can be found on the <A href="https://2017.igem.org/Team:Heidelberg/Pace" class="innerlink">PACE subpage</A>.
By chance, we had the opportunity to meet Kevin Esvelt, the PACE co-inventor, who gave a talk in
Heidelberg on June 1 st 2017, i.e. before we started our wet lab work. Kevin was kind enough to visit our
team and discuss his PACE method in detail. Besides its power for directed
protein evolution and conceptual beauty, we identified two major limitations of the current PACE setup:
<OL class="content"><LI>It requires a complex, custom-made bioreactor (Figure 1) including sophisticated flow control
setup, which is challenging to assemble and difficult to run robustly due to multiple failure
points (potential phage contamination; risk for phage-washout if flow rates are too high; Host
cell biofilm formation proper phage evolution)</LI><LI>PACE is almost entirely limited to improving an already existing protein function. To evolve truly
novel functions, evolutionary stepping stones are required, which can be so far created only by
slowly adapting selection pressure towards the activity to be evolved<X-REF>RN158</X-REF>. In practical terms, this means that evolving novel functions
requires an extremely complex experimental setup comprising multiple, fine-tuned synthetic
selection circuits applied in sequential order over many days. In most cases, it evolving a novel
functionality from scratch remains a practically impossible task to be implemented in PACE.</LI></OL>
As result its complexity and various failure points, only three groups worldwide have successfully
established PACE until today. Of note, two of these groups are headed by the PACE inventors Kevin
Esvelt and David Liu themselves, while the third group is run by Prof. Liu’s former post doc Bryan
Dickinson. In the course of our iGEM project, we were in contact with all three of these groups. We are
extremely grateful for their advice and for them sharing their constructs with us, both of which was
extremely important for us to get going.
<DIV class="content" style="padding-top: 20px; padding-bottom: 20px; font-weight: 700 !important; text-align: center !important">  
Figure 1: Bioreactor and flow setup required to perform PACE
</DIV>

Of note, in iGEM, only a single team has accepted the challenge and tried to set up PACE, namely the <A href="https://2015.igem.org/Team:TU_Dresden" class="innerlink">TU Dresden 2015 team</A>. The team thereby constructed a first PACE bioreactor prototype
and tested the flow controls. However, due to the lack of time, they could not run real PACE

experiments. The team concluded: <A href="https://2015.igem.org/Team:TU_Dresden/Project/Conclusions" class="innerlink">“All in all, the initial experiments helped us to grasp the idea of how
such experiments can look like”</A><H2>Motivation, Initial Design &amp; Engineering Principles</H2>
The goal of our work was to make in vivo directed evolution of proteins faster, easier, more robust and
expand its application scope. The primary, novel application we envisioned was the in vivo directed
evolution of improved and/or novel enzymes, which would be of enormous benefit for diverse
industries (e.g. chemical/pharmaceutical production, biomaterial production etc.), but remains
particularly challenging with conventional directed evolution or rational engineering strategies.
To reach this ambitious goal, we had to leave existing paradigms behind and rethink the concept of
directed evolution. As result, we came up with a novel, highly innovative engineering concept:
interfacing in vivo and in silico directed evolution by coupling PACE to deep learning of protein sequence
–function relationships (Figure 2).
The idea behind this concept is to fast-forward directed evolution using an intelligent algorithm. More
precisely, our algorithm should enable us to skip the otherwise required evolutionary stepping stones
during directed in vivo evolution by pre-evolving proteins with novel, desired target functions in silico.
            
            
 <DIV class="col-lg-12 col-md-12 col-xs-12 content" style="padding-top: 20px; padding-bottom: 20px; font-weight: 700 !important; text-align: center !important"> 
Figure 2: Interfacing in silico and in vivo directed evolution as novel engineering paradigm.
</DIV>           
As side note: As you will see below, we finally replaced the complex PACE-mediated evolution by the
much easier <A href="https://2017.igem.org/Team:Heidelberg/Predcel" class="innerlink">PREDCEL</A> protocol developed by us later on. But first things first.         

The engineering approach our team used to reach our goal was to first divide the in silico and in vivo
evolution problems into their basic components, work them out independently and validate them
thoroughly before finally combining them into a united workflow. We quickly realized that the general
components required for the in vivo and in silico directed evolution systems conceptually identical:

Evolution always consists of a closed cycle running through three major steps: (i) reproduction, (ii)
mutation and subsequent (iii) selection of sequence variants.
The corresponding in vivo evolution counterparts in PACE are well described and hence easy to identify:
<OL class="content"><LI> Reproduction is achieved by propagating M13 bacteriophages on E. coli host cells.</LI><LI> Mutation is achieved during phage replication by expression of mutagenic genes.</LI><LI> Selection is induced using synthetic circuits that couple the expression of geneIII (an
essential M13 gene) to the function (“fitness”) of the phage-encoded transgene to be
evolved.</LI></OL>
All three steps were tested individually by (i) propagating geneIII-deficient M13 phages on E. coli host
cells in a PACE apparatus in the absence of selection pressure and mutagenesis; (ii) overexpressing
mutagenic genes in E. coli and investigate the spontaneous formation of <A href="https://2017.igem.org/Team:Heidelberg/Pace" class="innerlink">resistant colonies</A> due to the mutagenesis-induced genetic drift; and (iii) <A href="https://2017.igem.org/Team:Heidelberg/Protein_Interaction" class="innerlink">comparing the propagation of
phages with known transgene “fitness” on corresponding synthetic circuits </A><DIV class="content" style="padding-top: 20px; padding-bottom: 20px;padding-left: 20px; font-weight: 700 !important; text-align: center !important;width:40%;height:auto;float: right;"> 
Figure 3: Overview of the PREDCEL protocol
</DIV>
Finally, we all three steps were successfully united into a <A href="https://2017.igem.org/Team:Heidelberg/Pace" class="innerlink">completed PACE cycle</A> to evolve improved split
T7 polymerase variants.

The corresponding in silico evolution cycle counterparts were practically not existing and had to be
invented from scratch. To this end, we designed an innovative software suite named <A href="https://2017.igem.org/Team:Heidelberg/Software" class="innerlink">AiGEM</A>, for
Artificial Intelligence for Genetic Evolution Mimicking. AiGEMs core functionalities thereby precisely
correspond to individual steps of a typical directed evolution cycle (see <A href="https://2017.igem.org/Team:Heidelberg/Software" class="innerlink">AiGEM</A> website for details on the
implementation):
<OL class="content"><LI>
We mutate parental sequences using GAIA, the Genetic Artificial intelligence algorithm,
which introducing random errors in a parental protein sequence/sequence pool; note, that
errors can also be limited to selected sequences windows </LI><LI> We select sequences using <A href="https://2017.igem.org/Team:Heidelberg/Software/DeeProtein" class="innerlink">DeeProtein</A>, a <A href="https://2017.igem.org/Team:Heidelberg/Software/DeeProtein#nn101" class="innerlink">deep neural network</A> created and trained on 10
million protein sequences representing more than 800 protein classes. DeeProtein is able to
robustly infer sequence-function relationships from raw sequence data. GAIA, in turn,
converts the DeeProtein analysis into a genetic fitness score and applies a threshold to
select the surviving sequence variants </LI><LI> Reproduction is then simply achieved by starting a new computational cycle with the
evolved sequence variants from the previous in silico evolution round.</LI>
Importantly, as part of <A href="https://2017.igem.org/Team:Heidelberg/Human_Practices" class="innerlink">our integrated human practices</A> activities, we added a fourth step to our in silico
evolution software, which does not have a counterpart in the in vivo directed evolution world:

<LI>We created <A href="https://2017.igem.org/Team:Heidelberg/Software/SafetyNet" class="innerlink">SafetyNet</A>, a DeeProtein based web application to infer “sleeping” hazardous
any parental (and evolved) sequence. Thus, SafetyNet safeguard the directed evolution
process and monitors and avoids the unintentional evolution of hazardous or dangerous
protein sequences</LI></OL>
To validate our in silico evolution software, we <A href="https://2017.igem.org/Team:Heidelberg/Validation" class="innerlink">designed two complementary wet lab experiments</A>, both of which were successfully performed.
<OL class="content"><LI> We aimed at demonstrating (and eventually successfully showed), that DeeProtein can infer the
impact of GAIA-induced mutations on protein function (i.e. the degree of activity). To this end,
we designed <A href="https://2017.igem.org/Team:Heidelberg/Validation" class="innerlink">~30 beta-lactamase</A> mutants using GAIA and correlated
the DeeProtein-based “activity” scores with the in vivo measured minimum inhibitory antibiotics
concentration (MIC).</LI><LI> We aimed at demonstrating (and eventually successfully showed) that we can evolve novel
functions in silico, practically from scratch. To this end, we used GAIA <A href="https://2017.igem.org/Team:Heidelberg/Validation" class="innerlink">transfer beta-
galactosidase activity onto a beta-glucoronidase</A> parental sequence by
in silico evolution.</LI></OL><H2>Improved, finalized Design – Learn, Evolve, PREDCEL!</H2>

While setting up the PACE apparatus and protocols, we ran into a serious of recurrent problems, most
importantly phage washout (i.e. complete phage loss after few hours of continuous evolution) and
phage contamination (due to the complex flow setup, contaminations are difficult to avoid). We also
realized, that the PACE apparatus was very static, strongly limiting its applications scope. For evolution
of proteins inducible by chemical for instance, one would ideally want to instantly alternate between
evolution in presence of the chemical inducer and as well as corresponding selection strains. This is
impossible in a continuous flow setup.
Therefore, we sought out to create a more simple and more flexible PACE alternative, which can be
quickly implemented by any trained biologist without the need for special equipment or knowledge.
Inspired by a recent publication on phage-mediated selection of gene libraries <X-REF>RN160</X-REF>, we created a simple protocol named
PREDCEL (for phage-related discontinuous evolution), which uses simple batch-wise, manual transfer of
the evolving phage gene pool (Figure 3). In essence, PREDCEL reduces the entire complexity of PACE to
simple, standard laboratory procedures, all the while gaining entire flexibility to easily swap conditions
(strains, inducers etc.) between individual rounds of evolution. We also provide an optogenetic tool for
<A href="https://2017.igem.org/Team:Heidelberg/Optogenetics" class="innerlink">simple adaptation of the selection pressure</A> during PREDCEL runs.

           

To validate our PREDCEL method, we performed directed evolution of split T7 polymers towards
improved auto-reassembly (i.e. <A href="https://2017.igem.org/Team:Heidelberg/Protein_Interaction" class="innerlink">protein interaction</A> of the split domains).
Having laid the required, solid foundation, were then able to outline a simple, fully generalizable
workflow for directed evolution enzymes and successfully tested it by re-directing the catalytic activity
of <A href="https://2017.igem.org/Team:Heidelberg/Cytochrome_Engineering" class="innerlink">a promiscuous cytochrome, Cyp1A2, towards a naturally unfavored product</A>. Key to this workflow is MAWS 2.0, a software originally introduced by iGEM Team
Heidelberg 2015, enabling the in silico design of aptamers and corresponding riboswitches (Figure 4).
We demonstrate the functionality of the improved MAWS 2.0 software, by successful design and
validation of a <A href="https://2017.igem.org/Team:Heidelberg/Organosilicons" class="innerlink">riboswitch detecting an organosilicon product</A> synthesized by us in
vitro using an engineered cytochrome C.
These riboswitches are then used for detecting the desired reaction product and mediate selection
pressure during evolution by controlling the expression of M13 geneIII essential for M13 phage
reproduction (Figure 4).
 <DIV class="col-lg-12 col-md-12 col-xs-12 content" style="padding-top: 20px; padding-bottom: 20px; font-weight: 700 !important; text-align: center !important"> 
Figure 4: A fully generalizable Workflow for Engineering of improved and novel enzymes using AiGEM-mediated in silico evolution interfaced with PREDCEL-mediated in vivo evolution.    
</DIV>
In summary, we implemented a unique in vivo – in silico evolution cycle highly accelerating the directed
evolution of protein for human benefit. Our standardized <A href="https://2017.igem.org/Team:Heidelberg/Predcel" class="innerlink">PREDCEL</A> protocol, PREDCEL <A href="https://2017.igem.org/Team:Heidelberg/Parts#Part_Collection" class="innerlink">parts collection</A>,
online <A href="https://2017.igem.org/Team:Heidelberg/Toolbox" class="innerlink">Toolbox guide</A> and accompanying <A href="https://2017.igem.org/Team:Heidelberg/RFC" class="innerlink">RFC</A> smoothly deliver our evolution toolbox to the end user.
Our finalized project design is summarized in Figure 5 (a fully interactive version of this project overview
figure is available under <A href="https://2017.igem.org/Team:Heidelberg/Description" class="innerlink">project overview</A>). Our <A href="https://2017.igem.org/Team:Heidelberg/Results" class="innerlink">Results</A> Page guides you through all subprojects carried
out to establish, validate and apply our evolutionary toolbox. Taken together, we provide a
foundational advance by introducing an innovative in vivo and in silico evolution interface as novel
engineering paradigm to synthetic biology.
</DIV></DIV></DIV></DIV></DIV><DIV style="background-color: white; padding-top: 20px;"><DIV class="container"><DIV class="row"><DIV class="col-lg-12 col-md-12 col-xs-12 content" style="display: inline-block;"><DIV class="row content"><DIV class="col-lg-12 col-md-12 col-xs-12" style="padding-top: 20px; padding-bottom: 50px; display: block"><DIV class="mdl-shadow--4dp" style="padding: 20px"><H3> References </H3></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><SECTION id="footer-sec" style="background-color: #222 !important"><DIV class="container"><DIV class="row" style="padding-top: 30px;"><DIV class="col-md-3"><H4 style="text-align: center">Quote</H4><P><I>“”</I></P></DIV><DIV class="col-md-3"><H4 style="text-align: center"> Useful <STRONG>Links</STRONG></H4><P><A href="https://igem.org/Special:SpecialPages">
			Special pages
		</A></P><P><A href="https://igem.org/Main_Page">
			Main Page
		</A></P></DIV><DIV class="col-md-3"><H4 style="text-align: center"> Follow us on </H4><DIV style="text-align: center;"><SPAN style="color: grey; text-align: center;">&amp;</SPAN></DIV></DIV><DIV class="col-md-3"><H4 style="text-align: center"> Contact us </H4><P>

               iGEM-Heidelberg2017@bioquant.uni-heidelberg.de
               
		 Im Neuenheimer Feld 267
		
		69120 Heidelberg
                    </P></DIV></DIV></DIV></SECTION></DIV></DIV></DIV></DIV></BODY></HTML>