Team:Heidelberg/Pace
NAVIGATION
People
Team
Attributions
Sponsoring
For Judges
Giant Jamboree
Project
Overview
Background & Design
PACE
PREDCEL
Results Summary
Software Validation
Organosilicons
Protein Interaction
Cytochrome Engineering
Optogenetics
CRISPR Cas9
InterLab Study
Achievements
Parts
Overview
Basic Parts
Composite Parts
Improved Part
Parts Collection
Part List
RFC
Modeling
Overview
Phage Titer
Mutagenesis Induction
Mutation Rate Estimation
Lagoon Contamination
Medium Consumption
Interactive Web Tools
Software
AiGEM Overview
DeeProtein
GAIA
SafetyNet
MAWS 2.0
Human Practices
Overview
Integrated Human Practices
Expert Interviews
Safety & Security
Public Engagement
Education
Collaborations
Notebook
Overview
Methods
Materials
Notebook Database
Toolbox Guide
iGEM TEAM HEIDELBERG 2017
COLLAPSE NAVIGATION
People
Team
Attributions
Sponsoring
For Judges
Giant Jamboree
Project
Overview
Background & Design
PACE
PREDCEL
Results Summary
Software Validation
Organosilicons
Protein Interaction
Cytochrome Engineering
Optogenetics
CRISPR Cas9
InterLab Study
Achievements
Parts
Overview
Basic Parts
Composite Parts
Improved Part
Parts Collection
Part List
RFC
Modeling
Overview
Phage Titer
Mutagenesis Induction
Mutation Rate Estimation
Lagoon Contamination
Medium Consumption
Interactive Web Tools
Software
AiGEM Overview
DeeProtein
GAIA
SafetyNet
MAWS 2.0
Human Practices
Overview
Integrated Human Practices
Expert Interviews
Safety & Security
Public Engagement
Education
Collaborations
Notebook
Overview
Methods
Materials
Notebook Database
Toolbox Guide
PACE
Phage-assisted continous evolution
PACE (for Phage-assisted, continuous evolution) enables in vivo directed evolution of proteins. The basic principle of PACE is to link the propagation of quickly mutating phages carrying a gene-of-interest to be evolved to directed selection in E. coli hosts mediated by synthetic circuits. Here, we describe our custom-made PACE setup and exemplify its use for evolving a split T7 polymerase towards improved auto-reassembly. Furthermore, we outline our unexpected journey on setting up PACE from scratch. Finally, we provide detailed information about our PACE bioreactor and the protocols required for running it. Taken together, we created a custom-made PACE bioreactor and demonstrate its successful application for in vivo directed evolution.
Introduction
Phage-assisted continuous evolution (PACE) is a powerful in vivo directed evolution method invented by Kevin Esvelt (now at the MIT media lab) and David Liu (Harvard University)
RN44'
. It can in principle be applied for directed evolution of any gene or protein of interest (GOI and POI, respectively), provided that the function to be evolved can be coupled to gene expression in E. coli
RN44'
. Conceptually, PACE mimics a closed evolution circle composed of (i) variant replication, (ii) mutation and (iii) selection in a bioreactor.  The basic principle of PACE is shown in Figure 1. In brief, PACE employs M13 phages that encode protein of interest (POI) to be evolved. These phages have a knockout for gene III, an essential M13 gene essential for phage replication.
Figure 1: Basic principle of phage-assisted, continuous evolution. (see main text for details).
During PACE, the POI-encoding phages are propagated of E. coli host cells carrying two plasmids. One it the mutagenesis plasmid (MP), which encodes highly mutagenic genes strongly reducing phage replication fidelity and error repair. As these genes are toxic to the E. coli host, they are set under control of an arabinose-inducible pBAD promoter. Different MP variants have been reported, which cause mutation rates up to ~2.3 substitutions per kb during phage replication when fully induced
RN159
, thereby highly accelerating the evolution.
The second plasmid is called accessory plasmid (AP), and is the PACE component inducing the selection pressure directing the evolution towards the desired goal. The AP thereby encodes gene III, the aforementioned essential M13 gene required for phage replication. Importantly, gene III expression from the AP is made dependent on the function of the phage-encoded POI, e.g. via a synthetic circuit. In other words: the AP induces a selection pressure towards optimizing POI for the function required to activate gene III expression. In the simple case of evolving a transcription factor as depicted in Figure 1, gene III would simply be expressed from the promoter (“pTarget”) the transcription factor should be optimized for.
PACE employs a custom bioreactor setup and complex flow controls to achieve continuous in vivo directed evolution of the phage gene pool (Figure 2). The PACE setup comprises four major compartments: medium tank, turbidostat, lagoon and waste tank. The turbidostat thereby which contain “fresh” E. coli host cells (transformed with AP and MP) cultivated at constant cell density and in the absence of arabinose, so that the toxic MP-encoded genes are not expressed. The “lagoon” is the compartment in which the actual evolution takes place, i.e. where the phages infect the host cells and propagate, provided the encoded POI activates gene III on the AP. In addition, the medium in the lagoon is supplemented by arabinose via a corresponding inducer supply line. The lagoon is thereby kept under constant flux (typically ~1 lagoon volume per hour), i.e. fresh host cells are introduced from the turbidostat the identical volume is simultaneously going to the waste line. Thereby, phages which are inefficiently propagating (i.e. due to low fitness of their encoded POI) are washed out, while highly propagating (i.e. fit) variants take over the gene pool and further evolve.
Figure 2: Simplified schematics of in vivo evolution with PACE.
This aforementioned in vivo evolution setup has several advantages as compared to alternative directed evolution methods:
Due to the small phage size, their fast generation time and high mutation rates during replication, large and diverse gene pools can be generated to efficiently sample the space of possible, beneficial mutations
A GOI’s fitness is evaluated in many different host cells due to its fast transfer between cells via the phage intermediate, thereby decoupling evolution form a particular host cell context
These key features enable render PACE a powerful tool for directed evolution in various application contexts. PACE has, for instance, been successfully applied for re-directing T7 polymerase target promoter specificity
RN44'
, for improving transcription activator-like effector nucleases
Liu2015
, evolving T7-based, modular biosensors
RN158
or proteases with novel specifies
packer2017
. In addition, PACE has been employed for studying drug-resistance development based on proteases
Dickinson.2014
. Of note, strategies for applying both, positive and negative selection pressure during evolution in PACE have also been explored
Carlson.2014
. Encouraged by the aforementioned, exciting work by others, we wanted to set up PACE in our lab to harness its power for directed evolution of proteins for human benefit. However, PACE turned out to be more difficult to run than anticipated initially. This was mainly due to the required, complex bioreactor setup. In fact, before we ran our first (and so far only) successful PACE experiment, we had already performed a high number of unsuccessful PACE runs (see the “Troubleshooting PACE” section further below) . The major failure points for us were
Cross-contamination with phages very well propagating on the used AP
Quick phage wash out due to insufficient phage propagation in the lagoon
Problems to reliably control cell density in the turbidostat
In this work, we aimed at evolving a split T7 polymerase toward improved auto-reassembly. As we were still troubleshooting our PACE setup when this project began, we initially planned to employ our PREDCEL protocol to for directed in vivo evolution of the split T7. That is why you will find parts of our results also in the protein interaction subpage . However, we gained confidence in using PACE just when the corresponding split T7 encoding phages and APs were ready. Therefore, we decided to give it a try and – luckily – we were successful (read on).
PACE Methods
Figure 3 shows a detailed, interactive PACE scheme. You can klick on the different components to receive information about the individual components of our PACE setup. Figure 4 and 5 show the different components of our custom-made PACE setup as it is standing in our lab right now. The accompanying video introduced you to our PACE apparatus in further detail.
Figure 3: Interactive schematics of the PACE setup.
Video: PACE buildup
Figure 4: PACE apparatus - Part 1.
This image shows our PACE device, with two turbidostats and two lagoons. All tubings are color coded and equivalent to the tubings shown in the scheme in Fig. 3.
Figure 5: PACE apparatus - Part 2
This picture is showing the second part of the construction of our PACE device. The syringe pumps (green) as well as the valve control and the oxygen supply are shown. All necessary tubings and cables are inserted in the heating cabinet.
Our PACE protocol
Before starting a PACE run, several prerequisites have to be fulfilled. Most of the preparations for the different PACE runs are the same in terms of tests for MP activity, F-Pilus plasmids and contamination. Additionally, several pre-tests are recommended to test the APs activity and the general functionality of the implemented genetic circuit.
Every part of the PACE device including all tubings and connectors have to be autoclaved. All open ends should be wrapped in aluminium foil. It is important to check all ends and tubings to be closed before start the dry autoclavation (Be aware of autoclaving only autoclavable parts of the PACE device).
After autoclavation, the PACE device should be treated with highest carefulness to prevent phage contamination in the turbidostat. To make this possible, the use of 10% H2O2 or incidin as well as the usage of gloves is advised.
Rebuild the PACE apparatus carefully using incidin to desinfect all for the autoclavation wrapped and thereby closed ends. Connect all necessary parts of the tubings.
The medium should be prepared slightly different to the medium used in literature [Esvelt et al., 2011] by mixing 140g dikaliumhydrogenphosphate with 40g kaliumdihydrogenphosphate, 20g ammoniumsulfate and 20 ml tween-80 in 20l dH20. The medium should be autoclaved as well before using it.
the autoclaved medium should be mixed with medium supplements, which should be prepared during autoclavation. 20g glucose, as well as 10g sodium citrate, 0.5g L-leucin, 0.5g and 100g casamino acids or trypton from casein have to be solved in at last 500 ml dH20. If the chemicals cannot be dissolved in this volume, water can be added until it can be solved. The resulting solution have to be sterile filtrated.
the appropiate volume of the prepared supplements can now be added to the autoclaved medium. This should be implemented in as steril conditions as possible, using incidin to sterilize the used pipette. In addition to the supplements, the appropiate antibiotics have to be added into the medium. Final concentrations should be choosen according the stock concentrations proposed by addgene. A blank for the OD600 measurements should be taken before connecting the medium to the tubings.
After connecting the media line of the turbidostat to the medium container, the turbidostat should be filled with medium until a volume of 1.5 l is reached, by starting the media pump.
50 ml bacterial culture resulting from the *MP testing* should be used for inoculation. Therefore draw up the culture into a syringe and inoculate the turbidostat using a cannula through the septa in the turbidostat. Reduce the flow rate to a minimum to ensure an efficient growth of the culture in the turbidostat.
lagoon pump can be started when the turbidostat reaches an OD600 = 0.6 - 1.0. The lagoon volume can be adjusted at a range of 100 - 150 ml lagoon volume.
induce mutagenesis by start adding 10% w/v arabinose to the lagoon. Arabinose should be added at last one hour before infection with bacteriophages to secure the induction of the MPs
When the lagoon is ready, arabinose is added and the cells are on a constant optical density, the lagoon can be infected with bacteriophages. Add 1 ml of 1010 PFU/ml to the lagoon and start the existing PACE run
During the PACE run, samples should be taken every four hours for the first 24 hours and every eight hours from the second day on until the run is finished. Also, phage detection PCRs and plaque assays should be implemented, proving the presence of the phage of interest and a contamination free turbidostat. Positive and negative control always have to be included into the detection PCR as well as the plaque assays.
Results – evolving split T7 polymerase toward improved auto-reassembly
We set out to employ PACE for evolving a split T7 polymerase towards increased auto-reassembly. Details on the how we obtained the particular split variant used in this PACE experiment can be found on our Protein Interaction subpage . In brief, based on the paper by Tiun Han et al.
Han.2017
, we tested M13 phages encoding 4 different split T7 variants for their ability to propagate on an E. coli selection strain transformed with a pT7-geneIII accessory plasmid. The variant encoding the split site at T7 residues 564/565 showed highest propagation and was therefore chosen for further optimization via PACE. We generated a PACE strain, by co-transforming our T7-gene III AP and MP4
RN159
. Then, by using our custom-build setup, we performed 30 hours of in vivo directed evolution via PACE (Figure 6).
Figure 6: Concept for evolution of split T7 towards improved auto-reassembly by PACE
Phages are continuously propagated on a mutagenic E. coli selection strain, carrying an AP comprising of a T7 promoter driving gene III. Thereby, variants should evolve, which show improved split T7 function.
During the PACE run continuously monitored the phage titer as well as optical density of our E. coli hosts (Figure 7). The used flow rate was approximately 1 lagoon volume per hour and adapted according to the optical density measurements.
Figure 7: Monitoring of Phage Titer and Optical Density during the Split T7 PACE run.
(A) Phage titer was estimated by manual sampling every 5 hours followed by plague essay. (B) Samples were taken every two hours and OD600 was determined using a nanophotometer. During the whole PACE run, the optical density is located in the range of 0.5 to 1, which is equivalent to the reaction conditions reported in the literature
RN44'
.
After having finished our PACE run, we performed plague assays and PCR amplified the Split T7 genes from 5 different plagues followed by sanger-sequenced. As hoped, we observed a recurrent, coding mutation (T877P) present in three out of the five sequenced split T7 variants (Figure 8).
Figure 8: Mutational pattern of the evolved split T7 variant.
Following three days of in vivo evolution with PACE, a plague assay was performed and the split T7 insert of five individual phage clones was analyzed by sanger-sequencing. We observed a recurrent mutation (T877P) in three out of the five clones, suggesting an evolutionary advantage (i.e. increased fitness) of the corresponding split T7 mutant as compared its non-mutated counterpart.
Interestingly this residue is located very close to the interaction surface of the two split T7 domains, suggesting a possible role in T7 auto-reassembly (Figure 9).
Figure 9: Structure of T7 polymerase.
The color coding corresponds to the T7 split at residues 564/565 in Figure 2. Cyan color indicates the N-terminal split domain, blue the C-terminal split domain. Residue T877 is stained in red.
Discussion
For discussion about the obtained, recurrent split T7 mutation and its possible role in split domain auto-reassembly, please refer to the protein interaction subpage. In this subproject, we sought out to set up PACE in our lab. Initially, we aimed at reproducing a previously described experiment
RN158
, which turned out to be a non-trivial task, but was – in retrospective view – the ideal model for troubleshooting and optimizing our PACE bioreactor. In fact, it lead to a detailed validation of all of our setup’s functionalities with respect to its ability to maintain host cell at constant OD, enable efficient phage propagation on host cells in the lagoon while avoiding phage washout as well as inducing high mutagenesis over a several, consecutive days. When the principle settings were finally under control, we sought out to apply PACE for directed evolution of a split T7 polymerase to optimize auto-reassembly of the split subunits. After having completed the corresponding run (which was at the same time our final run), we observed a recurrent mutation in a residue closely located to the used T7 split site (T877P), indicating that our experiment was successful.  Importantly, our initial struggle with getting PACE up and running motivated us to create a more simple, alternative protocol – PRECEL (for Phage-related discontinues evolution) -   which can be very quickly set up in standard molecular biology and cell biology lab. Thereby, our journey with PACE finally converged into an idea that became an essential component of our evolution toolbox . Taken together, we demonstrate the first, successful implementation of PACE in context of iGEM.
Troubleshooting
Evolving a split T7 Polymerase -PACE
Based on the high-potential concept of Phage-assisted continous evolution the group of Bryan Dickinson evolved a T7 RNA polymerase in the "Evolution of a split RNA polymerase as a versatile biosensor platform"-paper, publicated in early 2017
RN158
. We choose this example of PACE as our role model for testing of this highly complex method, needing the set-up of a continous culture, as well as a flow setup, demanding costly and highly sensitive hardware for its application. In the paper, Pu et al. grasp at evolving a small N-terminal domain of the T7RNAP for efficiently binding its N-terminal domain by help of two leucine zipper, strongly binding each other. Depending on their proximity, the two domains of the RNAP interact and can activate geneIII transcription. Taking this highly challenging experiment, we wanted build our own PACE device and investigate the general handling of the apparatus. We were first started with building up our PACE apparatus using the instructions of Kevin Esvelt back in 2011
RN44´
. For an accurate description of our PACE device and our procedure before starting PACE, please have a look at the Materials&Methods section. Thankfully, Bryan Dickinson´s lab was very helpful, so that they were sending us their constructs, strains and the phages they used for their evolution. Based on this, we could design our own constructs. Beyond that, we could quickly test our turbidostat, which we used instead of a chemostat, Dickinson and colleagues used for there PACE device. First, we aimed at cultivating our bacterial strains at an nearly constant optical density. This turbidostat-test was also used for evaluating the media consumption of the apparatus during an estimated PACE run suitable for iGEM conditions. Since the original PACE run has to include 29 days of continous evolution as well as ten different strains with different positive and negative selection stringencies, we were targeting at a shorter variant of this PACE experiment. By analyzing most of the data for the PACE run, we detected 3 of 7 essential mutations of the final variant of the N-terminal domain to be present in phages after only seven days of evolution. Outlining a more realistic example of a PACE run during a project implemented in only five wet lab month, we aimed for reproducing some of the mutations detected in the first seven days. In this context, we especially wanted to learn to handle the most difficult aspects of PACE:
phage washout
turbidostat contamination and working without contamination
flow rates
induction of mutagenesis
evaluation of successfully performed PACE
During several short PACE runs we already faced two of these challenging problems. First we had to adjust the flow rates in comparison to the flow rates in Dickinson-PACE based on the fact that we (i) used a turbidostat instead of a chemostat and (ii) investigated a low growth rate of the bacterial culture and thereby a doubling time of more than the expected 30 minutes. Second, we faced problems with a phage contamination in the turbidostat due to an extendable design of our PACE device and the waste tubings. Based on this observations, we slightly adjusted the tubings on our apparatus, yielding the actually used design. In the next steps we further tried to continously cultivate the N-terminal T7 RNAP phages with the first strain, used for PACE in the original paper. Since you cannot detect the phages simultanously during running PACE, we wanted to establish a quick detection method, which reveals a result in a shorter time than the usually used detection and quantification method - plaque assays. Several methods like the dot blot with M13 antibodies or the qPCR did not showed reliable and reproducible results. In contrast to these methods, a simple phage detection PCR using three characteristic PCR products in lengths of 200, 400 and 750 bp. The results of this PCR can be quickly analyzed on a agarose gel and yield an final result for the presence of your phages in the lagoon in only two hours. This detection method enables a more efficient and direct way of controlling the turbidostat.
Figure 10: Agarose gel of phage detection PCR
This image shows the results of an phage detection PCR of samples from the final Dickinson-PACE run. NEB 2-log ladder was loaded on the first lane. All lanes are
Following the phage propagation experiment further, we not only wanted to propagate the phages but trying to start our directed evolution in analogy to the first seven days of the original publicated PACE run. During this seven days, three different strains with different negative APs were used to integrate various selection stringencies. To implement this, the required APs were transformed with MP1 in S1030 cells following the *transformation protocol*. By performing seven days of PACE, it was necessary to change strains for the first time. This presents another challenging step, since a contamination during the build-up procedure pose a risk for contamination again. On this account, it should be avoided if possible. On this basis, first seven days of PACE were implemented. For efficient organisation, the expiry was documented before the run (siehe figure PI-PACE). This enables several team members to carry the experiment forward, without decrease in quality. The PACE run was successfully performed for 137 hours, displaying an acceptable phage titer in comparison to the literature (Esvelt et al., Nature 2011; Pu et al., Nature 2017). Sadly, we were running into another problem, which was highly reported by Kevin Esvelt - phage washout. The results of our plaque assays showed no infective phages in our lagoons after the 23. timepoint, corresponding to 137 hours of PACE. In addition, we could not reveal any mutations in our sequencings in a number of eight plaques which were picked for detection of potential mutations in the N-terminal domain of the RNAP.
Figure 11: Design of the Accessory Plasmids for the Evolution of Cas9
The AP consists of five subparts that are devided by homology regions for Gibson assembly (numbers). It carries an expression cassette for the transcription of a gRNA (between 1 and 5).  GeneVI (2-3) is under control of a that can be activated by the Cas9-rpoZ in context with the respective gRNA. luxAB accounts as a reporter for fluorescent readout of geneIII activation (3-4). The whole plasmid can be produced with different origins of replication (4-5) to modulate the copy number and by exchanging the geneVI part with the RBS.
Facing these problems, the general experimental design of our PACE device, as well as the bacterial strains and the inducer concentration were investigated for potential mistakes. Based on the observation, that we can detect phages even after almost six days, we focused on the induction of mutagenesis during our PACE run. As the mutagenesis is induced by arabinose and inhibted by glucose, we primarily concentrated on the glucose and arabinose concentrations in our PACE apparatus. Therefore, our modeling hypthesized a too high glucose concentration in our medium, as well as a low arabinose concentration, which could be a reason for neither mutations in our picked plaques. Together with this theoretical assumption, we developed an alternative PACE setup for testing of the induction of mutagenesis during PACE.
Random-mutagenesis PACE
Following our first PACE tests based on the the split T7 RNAP paper, we designed a new PACE test, using our modeling to estimate the glucose concentration in the lagoons as well as in the turbidostat. First tested in a pre-test using PREDCEL, we tried to transfer equivalent conditions yielding mutations in PREDCEL on the PACE apparatus. In order to do that, an alternative bacterial strain which contains the pJC175e plasmid, a plasmid usually used for plaque assays, providing geneIII under a psp-promotor for all phages was used. In addition to this propagation plasmid, MP1 and MP4 were transformed into one strain respectively. These strains were used for propagation of phages, used for the Dickinson-PACE testing too. Since the Dickinson-phage propagates in previously performed pre-experiments very well, the effect of the concurrent performed propagation and mutagenesis should be at its maximum. This is further underlined by the fact that in this case, geneIII is provided for free. Thus, there should be no selection pressure except from pressure on the best propagation regarding codon optimisation. Beyond that, a lot of random mutations should be observed when the MPs work reliable. To reach this point, three days of mutations using the MP1 and the MP4 strain simultanously in two turbidostats with one lagoon for each turbidostat were implemented. Sequencings were performed by picking eight plaques of each lagoon from the last sample included into the plaque assay. In this case, phage washout was not observed, displaying a phage titer which were settled between 10^4 and 10^7 in both lagoons.
Figure 12: Design of the Accessory Plasmids for the Evolution of Cas9
The AP consists of five subparts that are devided by homology regions for Gibson assembly (numbers). It carries an expression cassette for the transcription of a gRNA (between 1 and 5).  GeneVI (2-3) is under control of a that can be activated by the Cas9-rpoZ in context with the respective gRNA. luxAB accounts as a reporter for fluorescent readout of geneIII activation (3-4). The whole plasmid can be produced with different origins of replication (4-5) to modulate the copy number and by exchanging the geneVI part with the RBS.
Sanger sequencing results from these plaques showed a high number of different mutations in the 16 picked plaques. Thereby, the number of mutations resulting as a consequence of MP4 usage was slightly higher than the number of mutations in phages cultivated with the MP1 strain. In total, the mutations were widely spread over the N-terminal domain of the RNAP. Certainly, one mutation was recurrent, even the fact that there were no selection pressure. Overall, 15 of 22 mutations acquired during this three day PACE run entailed an amino acid exchange in the domain, while one mutation which occured in four clones resulted in no amino acid exchange.
Figue 13: Random mutations in N-terminal domain of T7 RNAP
The figure is presenting the nucleotide sequence of the N-terminal domain of the T7 RNAP in light grey, a linker sequence in black and the leucine zipper in dark grey
RN158
. At the top of the figure a consensus like sequence is shown, summarizing all mutations of the sequenced plaques, which are displayed as single bars under the summary. All mutations annotated are color coded. Recurrent mutations are divided into mutations leading to an amino acid exchange (red) and the mutations which do not have an amino acid exchange as a consequence (orange). In addition, mutations with amino acid exchanges, which are not recurrent are displayed in yellow, while the mutations without amino acid exchange are shown in blue. In total, only plaques showing mutations are shown. Sequences which did not contain mutations were omitted due to clarity reasons. Overall, sequencing was performed on 24 plaques.
By this, we demonstrated that the we can induce the mutagenesis plasmids successfully and sufficient manner. The evaluated conditions, hypothesized by our modeling, confirmed by our sequencing results from the real experiment should finally describe conditions in which a good induction of the MPs should be ensured.
PI-PACE
Coming to our final experiment we finally had the knowledge which is required to perform our own PACE run. This PACE run is just like Dickinson´s PACE approach based on protein-interaction of a split T7 RNAP. In contrast to the run before, both split sites are located on the selection phage. In principle, the split T7 RNAP is evolved on a better and faster reassembly of both fragments, yielding in a higher transcription of geneIII which is encoded under control of a T7 promotor. For further information on the principle of protein-interaction PACE, please visit our special site (hier Link einfügen). Building up on the equivalent conditions to the random-mutagenesis PACE run, we used the same amount of glucose and arabinose for induction of the mutagenesis plasmids. In this case, we only used a strain with MP4 due to the observation of slightly more mutations in the random mutagenesis experiment. Since this PACE experiment was performed with selection pressure, we estimated for difficulties in the phage propagation during PACE, which is why the flow rate was decreased for enabling better phage propagation. Nevertheless, our plaque assays showed phage washout after only 38 hours. Regardless of these findings, plaque PCRs and sequencings were performed, using plaques from the last available time point. The sequencing results showed one mutation in each of the split sites, of which one mutation could have a functional input on the reassembly of both sites.
Figure 14: Agarose gel of phage detection PCR
This image shows the results of an phage detection PCR of samples from the final Dickinson-PACE run. NEB 2-log ladder was loaded on the first lane. All lanes are
Figure 15: Design of the Accessory Plasmids for the Evolution of Cas9
The AP consists of five subparts that are devided by homology regions for Gibson assembly (numbers). It carries an expression cassette for the transcription of a gRNA (between 1 and 5).  GeneVI (2-3) is under control of a that can be activated by the Cas9-rpoZ in context with the respective gRNA. luxAB accounts as a reporter for fluorescent readout of geneIII activation (3-4). The whole plasmid can be produced with different origins of replication (4-5) to modulate the copy number and by exchanging the geneVI part with the RBS.
Chronologic Documentation of the PACE Experiment
Table 1: Documentation of performed tasks with comments
TIme
Date
Operator
Sample
Action
DONE ToDo:
OD Turbidostat 2
Stock ID
TO DOs:
12:00
24/10/2017
MP
-
Started filling the turbidostat (T2) with medium
199 (pMax69 aka AP-SplitT7 + GenIII+YFP + MP4)
13:30
24/10/2017
MP
-
Inoculation of turbidostat 2 with 50 ml Stock culture
199 (pMax69 aka AP-SplitT7 + GenIII+YFP + MP4)
14:00
24/10/2017
MP
-
Measured OD
"0
18"
199 (pMax69 aka AP-SplitT7 + GenIII+YFP + MP4)
15:00
24/10/2017
199 (pMax69 aka AP-SplitT7 + GenIII+YFP + MP4)
16:00
24/10/2017
MP
-
Measured OD
"0
511"
199 (pMax69 aka AP-SplitT7 + GenIII+YFP + MP4)
17:00
24/10/2017
MP
"Measured OD
started filling the lagoon and the arabinose pump"
"0
548"
199 (pMax69 aka AP-SplitT7 + GenIII+YFP + MP4)
18:00
24/10/2017
MP
1
Inoculation with 1 ml phage supernatant
"0
514"
199 (pMax69 aka AP-SplitT7 + GenIII+YFP + MP4)
19:00
24/10/2017
199 (pMax69 aka AP-SplitT7 + GenIII+YFP + MP4)
20:00
24/10/2017
MS
1
"Measured OD
Took samples L1
T1 - 1"
"0
637"
199 (pMax69 aka AP-SplitT7 + GenIII+YFP + MP4)
21:00
24/10/2017
199 (pMax69 aka AP-SplitT7 + GenIII+YFP + MP4)
22:00
24/10/2017
MP
"Measured OD
Flow rate for lagoon was to high from previous filling ( 7 ml/min in contrast to 1
666 ml/min) - potential washout "
"0
791"
199 (pMax69 aka AP-SplitT7 + GenIII+YFP + MP4)
23:00
24/10/2017
MP
"Lagoon was infected with 0
8 ml 10^8 PFU/ml phages to counteract the potential washout "
"0
818"
199 (pMax69 aka AP-SplitT7 + GenIII+YFP + MP4)
00:00
25/10/2017
MP
2
"Measured OD
Took samples L1
T1 - 2"
"0
833"
199 (pMax69 aka AP-SplitT7 + GenIII+YFP + MP4)
TO DO: Phage-PCR
01:00
25/10/2017
199 (pMax69 aka AP-SplitT7 + GenIII+YFP + MP4)
02:00
25/10/2017
MS
Measured OD
"0
789"
199 (pMax69 aka AP-SplitT7 + GenIII+YFP + MP4)
03:00
25/10/2017
199 (pMax69 aka AP-SplitT7 + GenIII+YFP + MP4)
04:00
25/10/2017
MS
3
"Measured OD
Took samples L1
T1 - 3"
199 (pMax69 aka AP-SplitT7 + GenIII+YFP + MP4)
05:00
25/10/2017
"0
792"
199 (pMax69 aka AP-SplitT7 + GenIII+YFP + MP4)
06:00
25/10/2017
MS
Measured OD
199 (pMax69 aka AP-SplitT7 + GenIII+YFP + MP4)
07:00
25/10/2017
"0
817"
199 (pMax69 aka AP-SplitT7 + GenIII+YFP + MP4)
08:00
25/10/2017
TB
4
"Measured OD
Took samples L1
T1 - 4"
Took samples
199 (pMax69 aka AP-SplitT7 + GenIII+YFP + MP4)
09:00
25/10/2017
199 (pMax69 aka AP-SplitT7 + GenIII+YFP + MP4)
10:00
25/10/2017
TB
Measured OD
"0
816"
199 (pMax69 aka AP-SplitT7 + GenIII+YFP + MP4)
11:00
25/10/2017
199 (pMax69 aka AP-SplitT7 + GenIII+YFP + MP4)
12:00
25/10/2017
TB
Measured OD
Done PCR
"0
81"
199 (pMax69 aka AP-SplitT7 + GenIII+YFP + MP4)
TO DO: Phage-PCR
13:00
25/10/2017
199 (pMax69 aka AP-SplitT7 + GenIII+YFP + MP4)
14:00
25/10/2017
PP
Measured OD
"0
843"
199 (pMax69 aka AP-SplitT7 + GenIII+YFP + MP4)
15:00
25/10/2017
inoculated growth for plaque assay
199 (pMax69 aka AP-SplitT7 + GenIII+YFP + MP4)
16:00
25/10/2017
PP
5
"Measured OD
Took samples L1
T1 - 5"
"0
833"
199 (pMax69 aka AP-SplitT7 + GenIII+YFP + MP4)
17:00
25/10/2017
199 (pMax69 aka AP-SplitT7 + GenIII+YFP + MP4)
18:00
25/10/2017
PP
Measured OD
"0
836"
199 (pMax69 aka AP-SplitT7 + GenIII+YFP + MP4)
19:00
25/10/2017
199 (pMax69 aka AP-SplitT7 + GenIII+YFP + MP4)
20:00
25/10/2017
MP
Measured OD
"0
862"
199 (pMax69 aka AP-SplitT7 + GenIII+YFP + MP4)
TO DO: Plaque Assay
21:00
25/10/2017
199 (pMax69 aka AP-SplitT7 + GenIII+YFP + MP4)
22:00
25/10/2017
MP
Measured OD
"0
81"
199 (pMax69 aka AP-SplitT7 + GenIII+YFP + MP4)
23:00
25/10/2017
199 (pMax69 aka AP-SplitT7 + GenIII+YFP + MP4)
00:00
26/10/2017
LP
6
"Measured OD
Took samples L1
T1 - 6"
Done PCR
"0
809"
199 (pMax69 aka AP-SplitT7 + GenIII+YFP + MP4)
TO DO: Phage-PCR
01:00
26/10/2017
199 (pMax69 aka AP-SplitT7 + GenIII+YFP + MP4)
02:00
26/10/2017
CG
Measured OD
"0
845"
199 (pMax69 aka AP-SplitT7 + GenIII+YFP + MP4)
03:00
26/10/2017
199 (pMax69 aka AP-SplitT7 + GenIII+YFP + MP4)
04:00
26/10/2017
CG
Measured OD
"0
864"
199 (pMax69 aka AP-SplitT7 + GenIII+YFP + MP4)
05:00
26/10/2017
199 (pMax69 aka AP-SplitT7 + GenIII+YFP + MP4)
06:00
26/10/2017
CG
Measured OD
"0
836"
199 (pMax69 aka AP-SplitT7 + GenIII+YFP + MP4)
07:00
26/10/2017
199 (pMax69 aka AP-SplitT7 + GenIII+YFP + MP4)
08:00
26/10/2017
TB
7
"Measured OD
Took samples L1
T1 - 7"
"0
831"
199 (pMax69 aka AP-SplitT7 + GenIII+YFP + MP4)
09:00
26/10/2017
199 (pMax69 aka AP-SplitT7 + GenIII+YFP + MP4)
10:00
26/10/2017
TB
Measured OD
"0
827"
199 (pMax69 aka AP-SplitT7 + GenIII+YFP + MP4)
11:00
26/10/2017
199 (pMax69 aka AP-SplitT7 + GenIII+YFP + MP4)
12:00
26/10/2017
TB
Measured OD
199 (pMax69 aka AP-SplitT7 + GenIII+YFP + MP4)
TO DO: Phage-PCR
13:00
26/10/2017
199 (pMax69 aka AP-SplitT7 + GenIII+YFP + MP4)
14:00
26/10/2017
TB
Measured OD
"0
738"
199 (pMax69 aka AP-SplitT7 + GenIII+YFP + MP4)
15:00
26/10/2017
199 (pMax69 aka AP-SplitT7 + GenIII+YFP + MP4)
16:00
26/10/2017
TB
8
"Measured OD
Took samples L1
T1 - 8"
"Took samples L1
T1"
"0
652"
199 (pMax69 aka AP-SplitT7 + GenIII+YFP + MP4)
17:00
26/10/2017
199 (pMax69 aka AP-SplitT7 + GenIII+YFP + MP4)
18:00
26/10/2017
Measured OD
199 (pMax69 aka AP-SplitT7 + GenIII+YFP + MP4)
19:00
26/10/2017
MS
"0
582"
199 (pMax69 aka AP-SplitT7 + GenIII+YFP + MP4)
20:00
26/10/2017
Measured OD
Plaque assay was performed successfully
199 (pMax69 aka AP-SplitT7 + GenIII+YFP + MP4)
TO DO: Plaque Assay
21:00
26/10/2017
199 (pMax69 aka AP-SplitT7 + GenIII+YFP + MP4)
22:00
26/10/2017
MS
Measured OD
"0
633"
199 (pMax69 aka AP-SplitT7 + GenIII+YFP + MP4)
23:00
26/10/2017
199 (pMax69 aka AP-SplitT7 + GenIII+YFP + MP4)
00:00
27/10/2017
PP
9
"Measured OD
Took samples L1
T1 - 9"
"0
511"
199 (pMax69 aka AP-SplitT7 + GenIII+YFP + MP4)
TO DO: Phage-PCR
01:00
27/10/2017
199 (pMax69 aka AP-SplitT7 + GenIII+YFP + MP4)
02:00
27/10/2017
MS
Measured OD
"0
692"
199 (pMax69 aka AP-SplitT7 + GenIII+YFP + MP4)
03:00
27/10/2017
199 (pMax69 aka AP-SplitT7 + GenIII+YFP + MP4)
04:00
27/10/2017
MS
Measured OD
"0
937"
199 (pMax69 aka AP-SplitT7 + GenIII+YFP + MP4)
05:00
27/10/2017
"0
945"
199 (pMax69 aka AP-SplitT7 + GenIII+YFP + MP4)
06:00
27/10/2017
MP
Measured OD
"could not be measured
as lagoon medium level was too high and therefore pump where set off for a short time "
199 (pMax69 aka AP-SplitT7 + GenIII+YFP + MP4)
07:00
27/10/2017
199 (pMax69 aka AP-SplitT7 + GenIII+YFP + MP4)
08:00
27/10/2017
MS
10
"Measured OD
Took samples L1
T1 - 10"
"Phage detection PCR with samples 8
9 and 10 for Dickinson-Phage and PI-Phage"
"0
912"
199 (pMax69 aka AP-SplitT7 + GenIII+YFP + MP4)
09:00
27/10/2017
199 (pMax69 aka AP-SplitT7 + GenIII+YFP + MP4)
10:00
27/10/2017
PP
Measured OD
"0
903"
199 (pMax69 aka AP-SplitT7 + GenIII+YFP + MP4)
11:00
27/10/2017
199 (pMax69 aka AP-SplitT7 + GenIII+YFP + MP4)
12:00
27/10/2017
LP
Measured OD
see above
"0
924"
199 (pMax69 aka AP-SplitT7 + GenIII+YFP + MP4)
TO DO: Phage-PCR
13:00
27/10/2017
199 (pMax69 aka AP-SplitT7 + GenIII+YFP + MP4)
14:00
27/10/2017
MP
Measured OD
"0
942"
199 (pMax69 aka AP-SplitT7 + GenIII+YFP + MP4)
15:00
27/10/2017
199 (pMax69 aka AP-SplitT7 + GenIII+YFP + MP4)
16:00
27/10/2017
PP
11
"Measured OD
Took samples L1
T1 - 11"
took samples
"0
942"
199 (pMax69 aka AP-SplitT7 + GenIII+YFP + MP4)
17:00
27/10/2017
199 (pMax69 aka AP-SplitT7 + GenIII+YFP + MP4)
18:00
27/10/2017
MP
Measured OD
"0
822"
199 (pMax69 aka AP-SplitT7 + GenIII+YFP + MP4)
19:00
27/10/2017
199 (pMax69 aka AP-SplitT7 + GenIII+YFP + MP4)
20:00
27/10/2017
MK
Measured OD
"0
59"
199 (pMax69 aka AP-SplitT7 + GenIII+YFP + MP4)
TO DO: Plaque Assay
21:00
27/10/2017
199 (pMax69 aka AP-SplitT7 + GenIII+YFP + MP4)
22:00
27/10/2017
CG
Measured OD
"0
61"
199 (pMax69 aka AP-SplitT7 + GenIII+YFP + MP4)
23:00
27/10/2017
199 (pMax69 aka AP-SplitT7 + GenIII+YFP + MP4)
00:00
28/10/2017
MP
12
"Measured OD
Took samples L1
T1 - 12"
Phage detection PCR with sample 11
"0
664"
199 (pMax69 aka AP-SplitT7 + GenIII+YFP + MP4)
TO DO: Phage-PCR
01:00
28/10/2017
199 (pMax69 aka AP-SplitT7 + GenIII+YFP + MP4)
02:00
28/10/2017
CG
Measured OD
"0
809"
199 (pMax69 aka AP-SplitT7 + GenIII+YFP + MP4)
03:00
28/10/2017
199 (pMax69 aka AP-SplitT7 + GenIII+YFP + MP4)
04:00
28/10/2017
MP
Measured OD
PCR: negative - phage washout
"0
820"
199 (pMax69 aka AP-SplitT7 + GenIII+YFP + MP4)
05:00
28/10/2017
"Flow rate was adjusted: 1 ml/min
lagoon was infected by pooled samples 6
7 and 8 "
199 (pMax69 aka AP-SplitT7 + GenIII+YFP + MP4)
06:00
28/10/2017
MP
Measured OD
199 (pMax69 aka AP-SplitT7 + GenIII+YFP + MP4)
07:00
28/10/2017
199 (pMax69 aka AP-SplitT7 + GenIII+YFP + MP4)
08:00
28/10/2017
MP
13
"Measured OD
Took samples L1
T1 - 13"
"0
667"
199 (pMax69 aka AP-SplitT7 + GenIII+YFP + MP4)
09:00
28/10/2017
199 (pMax69 aka AP-SplitT7 + GenIII+YFP + MP4)
10:00
28/10/2017
PP
Measured OD
"0
62"
199 (pMax69 aka AP-SplitT7 + GenIII+YFP + MP4)
11:00
28/10/2017
199 (pMax69 aka AP-SplitT7 + GenIII+YFP + MP4)
12:00
28/10/2017
PP
Measured OD
Phage PCR was performed
"0
768"
199 (pMax69 aka AP-SplitT7 + GenIII+YFP + MP4)
TO DO: Phage-PCR
13:00
28/10/2017
199 (pMax69 aka AP-SplitT7 + GenIII+YFP + MP4)
14:00
28/10/2017
PP
Measured OD
199 (pMax69 aka AP-SplitT7 + GenIII+YFP + MP4)
15:00
28/10/2017
199 (pMax69 aka AP-SplitT7 + GenIII+YFP + MP4)
16:00
28/10/2017
PP
14
"Measured OD
Took samples L1
T1 - 14"
FINAL PROBES
References
Quote
“”
Useful Links
Special pages
Main Page
Follow us on
&
Contact us
iGEM-Heidelberg2017
@bioquant.uni-heidelberg.de
Im Neuenheimer Feld 267
69120 Heidelberg
