<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Heidelberg_Software_SafetyNet skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Heidelberg/Software/SafetyNet</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container"><DIV class="navbar-header"><A type="button" data-canvas="body" id="sidebarCollapse" class="navbar-toggle" data-toggle="offcanvas" data-target="#sidebar" style="color: #393939;margin-top: 9px;">NAVIGATION
                    </A></DIV><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav pull-right"><LI class="dropdown"><A href="https://2017.igem.org/Team:Heidelberg/Team">People </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Team">Team</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Attributions">Attributions</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Sponsoring">Sponsoring</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/For_Judges">For Judges</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Jamboree">Giant Jamboree</A></LI></UL><LI class="dropdown"><A href="https://2017.igem.org/Team:Heidelberg/Description">Project </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Description">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Design">Background &amp; Design </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Pace">PACE</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Predcel">PREDCEL</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Results">Results Summary</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Validation">Software Validation</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Organosilicons">Organosilicons</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Protein_Interaction">Protein Interaction</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Cytochrome_Engineering">Cytochrome Engineering</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Optogenetics">Optogenetics</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/CRISPR">CRISPR Cas9</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/InterLab">InterLab Study</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Achievements">Achievements</A></LI><LI class="dropdown"><A href="https://2017.igem.org/Team:Heidelberg/Parts">Parts </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Basic_Part">Basic Parts</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Composite_Part">Composite Parts</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Improved_Part">Improved Part</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Part_Collection">Parts Collection</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Part_List">Part List</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/RFC">RFC</A></LI><LI class="dropdown"><A href="https://2017.igem.org/Team:Heidelberg/Model">Modeling </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Phage_Titer">Phage Titer</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Mutagenesis_Induction">Mutagenesis Induction</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Mutation_Rate_Estimation">Mutation Rate Estimation</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Lagoon_Contamination">Lagoon Contamination</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Medium_Consumption">Medium Consumption</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Tools">Interactive Web Tools</A></LI><LI class="dropdown"><A href="https://2017.igem.org/Team:Heidelberg/Software">Software </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software">AiGEM Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software/DeeProtein">DeeProtein</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software/GAIA">GAIA</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software/SafetyNet">SafetyNet</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software/MAWS">MAWS 2.0</A></LI><LI class="dropdown"><A href="https://2017.igem.org/Team:Heidelberg/Human_Practices">Human Practices </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Human_Practices">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/HP/Gold_Integrated">Integrated Human Practices</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Interviews">Expert Interviews</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Safety">Safety &amp; Security</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Engagement">Public Engagement</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Education">Education</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Collaborations">Collaborations</A></LI><LI class="dropdown"><A href="https://2017.igem.org/Team:Heidelberg/Notebook">Notebook </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Notebook">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Experiments">Methods</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Materials">Materials</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Database">Notebook Database</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Toolbox">Toolbox Guide</A></LI></DIV></DIV><NAV id="sidebar" style="z-index: 10001; background-color: white;border: none;box-shadow:1px 0px 0px #393939;" class="navmenu navmenu-default navmenu-fixed-left offcanvas" role="navigation"><A class="navmenu-brand" href="https://2017.igem.org/Team:Heidelberg"> iGEM TEAM HEIDELBERG 2017</A><DIV id="sidebarHide" data-toggle="offcanvas" data-target="#sidebar" style="padding-left: 20px;"><A style="font-family: 'Josefin Sans' !important; font-size: 16px; line-height: 18px; height: 18px;"> COLLAPSE NAVIGATION </A></DIV><UL class="nav navmenu-nav" id="sidenav"><LI><A data-toggle="collapse" data-target="#People_side">People </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Team">Team</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Attributions">Attributions</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Sponsoring">Sponsoring</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/For_Judges">For Judges</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Jamboree">Giant Jamboree</A></LI></UL><LI><A data-toggle="collapse" data-target="#Project_side">Project </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Description">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Design">Background &amp; Design </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Pace">PACE</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Predcel">PREDCEL</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Results">Results Summary</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Validation">Software Validation</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Organosilicons">Organosilicons</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Protein_Interaction">Protein Interaction</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Cytochrome_Engineering">Cytochrome Engineering</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Optogenetics">Optogenetics</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/CRISPR">CRISPR Cas9</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/InterLab">InterLab Study</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Achievements">Achievements</A></LI><LI><A data-toggle="collapse" data-target="#Parts_side">Parts </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Basic_Part">Basic Parts</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Composite_Part">Composite Parts</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Improved_Part">Improved Part</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Part_Collection">Parts Collection</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Part_List">Part List</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/RFC">RFC</A></LI><LI><A data-toggle="collapse" data-target="#Model_side">Modeling </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Phage_Titer">Phage Titer</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Mutagenesis_Induction">Mutagenesis Induction</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Mutation_Rate_Estimation">Mutation Rate Estimation</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Lagoon_Contamination">Lagoon Contamination</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Medium_Consumption">Medium Consumption</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Tools">Interactive Web Tools</A></LI><LI><A data-toggle="collapse" data-target="#AI_side">Software </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software">AiGEM Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software/DeeProtein">DeeProtein</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software/GAIA">GAIA</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software/SafetyNet">SafetyNet</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software/MAWS">MAWS 2.0</A></LI><LI><A data-toggle="collapse" data-target="#HP_side">Human Practices </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Human_Practices">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/HP/Gold_Integrated">Integrated Human Practices</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Interviews">Expert Interviews</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Safety">Safety &amp; Security</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Engagement">Public Engagement</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Education">Education</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Collaborations">Collaborations</A></LI><LI><A data-toggle="collapse" data-target="#Notebook_side">Notebook </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Notebook">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Experiments">Methods</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Materials">Materials</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Database">Notebook Database</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Toolbox">Toolbox Guide</A></LI></NAV><DIV style="background-color: white !important; padding: 0 !important; margin: 0 !important;"><DIV class="page-title" style="background-color: white !important; border-top: 0px solid #222;"><DIV class="container" style=" padding: 0 !important;"><DIV class="col-lg-12 col-md-12 col-xs-12" style="padding-right: 0 !important; padding-bottom: 0 !important; position: relative; left: -10px;"><DIV style="border-right: 5px solid #393939; padding: 20px 20px 0px 0px"><DIV class="header-title heidelberg-turk"> 
    SafetyNet </DIV><DIV class="header-subtitle" style="color: #393939 !important;"> 
    Evolution does no harm. </DIV></DIV></DIV></DIV></DIV><DIV class="container" style="background-color: white !important; padding: 0 !important;"><DIV class="col-lg-12 col-md-12 col-xs-12" style="padding: 0 !important; margin: 0px important!;"><DIV class="container" style="position: relative; left: -10px; border-left: 5px solid #393939; border-top: 5px solid #393939; padding: 0 !important"><DIV class="col-lg-12 col-md-12 col-xs-12 content" style="text-align: justify; text-color: #393939 !important; padding: 20px 5px 0px 20px !important; font-weight: bold !important; font-family: 'Open Sans' !important">
        
        When performing large scale, automated directed evolution experiments a manual assertion of every sequence in the library is impossible. However profound background and quality checks on sequences are crucial in the automated context as the experimentator has no direct control of the processes. This especially holds true for <I>in silico</I> evolution, where the immediate effect of a mutation is not assessable.
In order to safeguard our <I>in vivo</I> and <I>in silico</I> directed evolution experiments we developed SafetyNet.
SafetyNet is a web available, neural network based sequence check. It does not only infer the function and species of origin, but does also assert the safety level assigned to the origin species and the potential harm of an input sequence. We applied SafetyNet throughout our directed evolution experiments to ensure safe and flawless sequence improvement all the while preventing the unintended emergence of harmful traits.
    
    </DIV></DIV><DIV class="container"><DIV class="row"><DIV class="col-lg-12 col-md-12 col-xs-12 content" style="display: inline-block; padding-bottom: 10px; padding-top:10px;"><H2>Method</H2>
SafetyNet is based on two algorithmic pillars. The first one is a BLAST search of the input sequence against the swissprot database, performed through the NCBI web API. The request is POSTed to the NCBI server and the result is catched with GET request. Subsequently the result is parsed for the protein IDs of all non redundant matches. Next, the retrieved protein IDs are used to send a GET request to the UniProt database, requesting the entry of the protein in question. The entry is again parsed for key information, this time returning the assigned GO-Terms, the species of origin and the gene of origin. Subsequently the collected information on each entry is combined and a lookup on the SafetyNet internal databases is performed. These comprehensive databases list GO-Terms associated with cytotoxic, viral or pathogenic functions or pathways. Further we included the functional terms for proteases and nucleases, to account for destructive intracellular potential. The biological safety level of the retrieved species of origin is investigated by a database lookup on the biosafety-database of the <A href="">german ministry of consumer and food safety</A> (the german FDA).
The second algorithmic column applies a DeeProtein implementation in the browser. Upon user request the neural network inference can additionally be enabled to support the BLAST search in function classification. This is especially useful as the neural network is able to detect latent or &quot;hidden&quot; potential as it learned the sequence to function relation accross the whole respective functional domain, whereas the BLAST search is limited to direct sequence identity.
The browser integrated neural network is implemented in DeeplearnJS and features GPU support. It is a ResNet30, similar to the Architecture of DeeProtein, asserting the class probability for 886 classes. As the size of the ResNet-weigths is ~100MB we offer a selection mode to guarantee the use of the BLAST-part on mobile connections.
Finally the collected information is concatenated and presented in a easily understandable color coded scheme.

    
    </DIV></DIV></DIV><DIV class="container"><DIV class="row"><DIV class="col-lg-12 col-md-12 col-xs-12 content" style="display: inline-block; padding-bottom: 10px; padding-top:10px;"><H2>SafetyBLAST: a smart inter-database search</H2>
Using the dialogue below, you can test arbitrary protein sequences for potential safety risks, by relying on a homology based database search. A sequence to be tested can be provided either as plain text, by entering it into the text box, or supplied in a single-sequence standard FASTA file, by providing a file handle via the choose file button. You will then be prompted to choose a file on your computer or smart phone, which will then be read by the browser. By pressing the BLAST button, a multi-database query for your sequence is initiated. Sequences you enter will be forwarded to the NCBI's BLAST server via a reverse proxy and resulting BLAST hits will be compared against Uniprot and multiple internal risk databases. No third parties besides NCBI's BLAST and Uniprot will see any of your data, making SafetyBLAST as secure as a series of manual queries.
The results will be displayed in a table under &quot;BLAST results&quot;, with both color-coded and numerical indicators of protein safety and hit correctness. If a protein potentially matches counterparts from different bacterial strains with different safety levels, a worst-case prognosis is displayed. For example, if a protein matches both K12 <I>E. coli</I> (S1 organism) and enterohemorrhagic <I>E. coli</I> (S3 organism), the most pessimistic outcome of the S3 safety level is displayed and relevant cells in the table are marked red.
Keep in mind, that using SafetyNet <B>does not</B> replace common sense and careful deliberation of security risks, merely serving as an aid to screening a large amount of sequences for safety in a short amount of time.

    
    </DIV></DIV></DIV><DIV class="container"><DIV class="row"><DIV class="col-lg-12 col-md-12 col-xs-12 content" style="display: inline-block; padding-bottom: 10px; padding-top:10px;"><DIV style="padding-top: 50px; padding-bottom:50px;"><DIV class="heidel-form container"><DIV class="heidel-form-header col-lg-12" style="text-align: left !important;">Check protein safety using BLAST</DIV><FORM><DIV class="row"><DIV class="col-md-12"><DIV class="form-group"><LABEL class="control-label label-number">
                Protein Sequence
            </LABEL><TEXTAREA class="form-control btn btn-lg btn-block" id="blast_sequence" rows="4" style="text-align: left !important;">Your sequence here ...</TEXTAREA></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-4"><DIV class="form-group"><LABEL class="fileContainer btn form-color"><SPAN>Choose File</SPAN></LABEL></DIV></DIV><DIV class="col-md-4"><DIV class="form-group"><BUTTON type="button" class="btn btn-lg btn-block form-color" onclick="return (function(){})" id="blastit">Use BLAST</BUTTON></DIV></DIV><DIV class="col-md-4"><DIV class="form-group"><BUTTON type="button" class="btn btn-lg btn-block form-color" onclick="return (function(){})" id="blast_status">Idle...</BUTTON></DIV></DIV></DIV></FORM></DIV></DIV><DIV class="col-lg-12 col-md-12 col-xs-12"><DIV style="padding-top: 50px; padding-bottom:50px;"><DIV class="heidel-form container"><FORM><DIV class="img-caption title" style="text-align: justify !important;"><H3>Results of safetyBLAST search.</H3></DIV></FORM></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="container"><DIV class="row"><DIV class="col-lg-12 col-md-12 col-xs-12 content" style="display: inline-block; padding-bottom: 10px; padding-top:10px;"><H2>DeeProtein: a powerful deep protein classifier</H2>
Using the DeeProtein dialogue, you can use our deep neural classifier DeeProtein to infer the function of a protein sequence, based on the full Uniprot training dataset. This allows you to predict the functions of protein sequences impossible to predict using homology-based search. Upload a sequence either by typing into the text box or uploading a standard single-sequence FASTA file using the choose file dialogue. Properties are inferred by pressing the &quot;DeeProtein Infer&quot; button. Using DeeProtein, your sequences never leave your computer, leaving you with the maximum amount of data security possible short of using the software on a secured offline machine, all without having to install any piece of software. The resulting protein functions are arranged under &quot;DeeProtein results&quot;, color coded for safety: green for safe functions, red for potentially dangerous funcitons and gray, in no relevant function was inferred.
 
    </DIV></DIV></DIV><DIV class="container"><DIV class="row"><DIV class="col-lg-12 col-md-12 col-xs-12 content" style="display: inline-block; padding-bottom: 10px; padding-top:10px;"><DIV style="padding-top: 50px; padding-bottom:50px;"><DIV class="heidel-form-color container"><DIV class="heidel-form-color-header col-lg-12" style="text-align: left !important;">IMPORTANT</DIV><FORM><DIV class="col-md-12"><P class="content" style="color: white !important">In order for you to be able use our deep neural network classifier - <B>DeeProtein</B> - the neural network's parameters have to be loaded into memory on your computer. As our classifier is very deep, that is many-layered, the total size of the weights surpasses 100 MiB. To allow you to use <B>DeeProtein</B> without incurring too much of a burden on your internet connection and data plan, we provide the weights as a one-time-only downloadable file, which you can then load into memory on your machine, whenever you plan to use <B>DeeProtein</B>.</P><P class="content" style="color: white !important">If this is your first time using <B>DeeProtein</B>, download the weights using the button on the left. If you have already downloaded the weights, click the button on the right and point <B>DeeProtein</B> to where they are on your computer. Happy testing! </P></DIV><DIV class="row"><DIV class="col-md-4"><DIV class="form-group"><BUTTON type="button" class="btn btn-lg btn-block form-color" onclick="return (function(){})" id="download">Download</BUTTON></DIV></DIV><DIV class="col-md-4"><DIV class="form-group"><LABEL class="fileContainer btn form-color"><SPAN>Choose File</SPAN></LABEL></DIV></DIV></DIV></FORM></DIV></DIV></DIV></DIV></DIV><DIV class="container"><DIV class="row"><DIV class="col-lg-12 col-md-12 col-xs-12 content" style="display: inline-block; padding-bottom: 10px; padding-top:10px;"><DIV style="padding-top: 50px; padding-bottom:50px;"><DIV class="heidel-form container"><DIV class="heidel-form-header col-lg-12" style="text-align: left !important;">Check safety using DeeProtein</DIV><FORM><DIV class="row"><DIV class="col-md-12"><DIV class="form-group"><LABEL class="control-label label-number">
                Protein Sequence
            </LABEL><TEXTAREA class="form-control btn btn-lg btn-block" id="deep_sequence" rows="4" style="text-align: left !important;">Your sequence here ...</TEXTAREA></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-4"><DIV class="form-group"><LABEL class="fileContainer btn form-color"><SPAN>Choose File</SPAN></LABEL></DIV></DIV><DIV class="col-md-4"><DIV class="form-group"><BUTTON type="button" class="btn btn-lg btn-block form-color" onclick="return (function(){})" id="deepit">DeeProtein Infer</BUTTON></DIV></DIV><DIV class="col-md-4"><DIV class="form-group"><BUTTON type="button" class="btn btn-lg btn-block form-color" onclick="return (function(){})" id="deep_status">Idle...</BUTTON></DIV></DIV></DIV></FORM></DIV></DIV><DIV class="col-lg-12 col-md-12 col-xs-12"><DIV style="padding-top: 50px; padding-bottom:50px;"><DIV class="heidel-form container"><FORM><DIV class="img-caption title" style="text-align: justify !important;"><H3>Results of DeeProtein inference.</H3></DIV></FORM></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><SECTION id="footer-sec" style="background-color: #222 !important"><DIV class="container"><DIV class="row" style="padding-top: 30px;"><DIV class="col-md-3"><H4 style="text-align: center">Quote</H4><P><I>“”</I></P></DIV><DIV class="col-md-3"><H4 style="text-align: center"> Useful <STRONG>Links</STRONG></H4><P><A href="https://igem.org/Special:SpecialPages">
			Special pages
		</A></P><P><A href="https://igem.org/Main_Page">
			Main Page
		</A></P></DIV><DIV class="col-md-3"><H4 style="text-align: center"> Follow us on </H4><DIV style="text-align: center;"><SPAN style="color: grey; text-align: center;">&amp;</SPAN></DIV></DIV><DIV class="col-md-3"><H4 style="text-align: center"> Contact us </H4><P>

               iGEM-Heidelberg2017@bioquant.uni-heidelberg.de
               
		 Im Neuenheimer Feld 267
		
		69120 Heidelberg
                    </P></DIV></DIV></DIV></SECTION></DIV></DIV></DIV></DIV></BODY></HTML>