<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Heidelberg_Parts skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Heidelberg/Parts</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container"><DIV class="navbar-header"><A type="button" data-canvas="body" id="sidebarCollapse" class="navbar-toggle" data-toggle="offcanvas" data-target="#sidebar" style="color: #393939;margin-top: 9px;">NAVIGATION
                    </A></DIV><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav pull-right"><LI class="dropdown"><A href="https://2017.igem.org/Team:Heidelberg/Team">People </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Team">Team</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Attributions">Attributions</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Sponsoring">Sponsoring</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/For_Judges">For Judges</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Jamboree">Giant Jamboree</A></LI></UL><LI class="dropdown"><A href="https://2017.igem.org/Team:Heidelberg/Description">Project </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Description">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Design">Background &amp; Design </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Pace">PACE</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Predcel">PREDCEL</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Results">Results Summary</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Validation">Software Validation</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Organosilicons">Organosilicons</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Protein_Interaction">Protein Interaction</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Cytochrome_Engineering">Cytochrome Engineering</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Optogenetics">Optogenetics</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/CRISPR">CRISPR Cas9</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/InterLab">InterLab Study</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Achievements">Achievements</A></LI><LI class="dropdown"><A href="https://2017.igem.org/Team:Heidelberg/Parts">Parts </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Basic_Part">Basic Parts</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Composite_Part">Composite Parts</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Improved_Part">Improved Part</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Part_Collection">Parts Collection</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Part_List">Part List</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/RFC">RFC</A></LI><LI class="dropdown"><A href="https://2017.igem.org/Team:Heidelberg/Model">Modeling </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Phage_Titer">Phage Titer</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Mutagenesis_Induction">Mutagenesis Induction</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Mutation_Rate_Estimation">Mutation Rate Estimation</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Lagoon_Contamination">Lagoon Contamination</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Medium_Consumption">Medium Consumption</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Tools">Interactive Web Tools</A></LI><LI class="dropdown"><A href="https://2017.igem.org/Team:Heidelberg/Software">Software </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software">AiGEM Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software/DeeProtein">DeeProtein</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software/GAIA">GAIA</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software/SafetyNet">SafetyNet</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software/MAWS">MAWS 2.0</A></LI><LI class="dropdown"><A href="https://2017.igem.org/Team:Heidelberg/Human_Practices">Human Practices </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Human_Practices">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/HP/Gold_Integrated">Integrated Human Practices</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Interviews">Expert Interviews</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Safety">Safety &amp; Security</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Engagement">Public Engagement</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Education">Education</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Collaborations">Collaborations</A></LI><LI class="dropdown"><A href="https://2017.igem.org/Team:Heidelberg/Notebook">Notebook </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Notebook">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Experiments">Methods</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Materials">Materials</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Database">Notebook Database</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Toolbox">Toolbox Guide</A></LI></DIV></DIV><NAV id="sidebar" style="z-index: 10001; background-color: white;border: none;box-shadow:1px 0px 0px #393939;" class="navmenu navmenu-default navmenu-fixed-left offcanvas" role="navigation"><A class="navmenu-brand" href="https://2017.igem.org/Team:Heidelberg"> iGEM TEAM HEIDELBERG 2017</A><DIV id="sidebarHide" data-toggle="offcanvas" data-target="#sidebar" style="padding-left: 20px;"><A style="font-family: 'Josefin Sans' !important; font-size: 16px; line-height: 18px; height: 18px;"> COLLAPSE NAVIGATION </A></DIV><UL class="nav navmenu-nav" id="sidenav"><LI><A data-toggle="collapse" data-target="#People_side">People </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Team">Team</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Attributions">Attributions</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Sponsoring">Sponsoring</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/For_Judges">For Judges</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Jamboree">Giant Jamboree</A></LI></UL><LI><A data-toggle="collapse" data-target="#Project_side">Project </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Description">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Design">Background &amp; Design </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Pace">PACE</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Predcel">PREDCEL</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Results">Results Summary</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Validation">Software Validation</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Organosilicons">Organosilicons</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Protein_Interaction">Protein Interaction</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Cytochrome_Engineering">Cytochrome Engineering</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Optogenetics">Optogenetics</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/CRISPR">CRISPR Cas9</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/InterLab">InterLab Study</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Achievements">Achievements</A></LI><LI><A data-toggle="collapse" data-target="#Parts_side">Parts </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Basic_Part">Basic Parts</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Composite_Part">Composite Parts</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Improved_Part">Improved Part</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Part_Collection">Parts Collection</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Part_List">Part List</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/RFC">RFC</A></LI><LI><A data-toggle="collapse" data-target="#Model_side">Modeling </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Phage_Titer">Phage Titer</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Mutagenesis_Induction">Mutagenesis Induction</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Mutation_Rate_Estimation">Mutation Rate Estimation</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Lagoon_Contamination">Lagoon Contamination</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Medium_Consumption">Medium Consumption</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Tools">Interactive Web Tools</A></LI><LI><A data-toggle="collapse" data-target="#AI_side">Software </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software">AiGEM Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software/DeeProtein">DeeProtein</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software/GAIA">GAIA</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software/SafetyNet">SafetyNet</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software/MAWS">MAWS 2.0</A></LI><LI><A data-toggle="collapse" data-target="#HP_side">Human Practices </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Human_Practices">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/HP/Gold_Integrated">Integrated Human Practices</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Interviews">Expert Interviews</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Safety">Safety &amp; Security</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Engagement">Public Engagement</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Education">Education</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Collaborations">Collaborations</A></LI><LI><A data-toggle="collapse" data-target="#Notebook_side">Notebook </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Notebook">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Experiments">Methods</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Materials">Materials</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Database">Notebook Database</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Toolbox">Toolbox Guide</A></LI></NAV><DIV style="background-color: white !important; padding: 0 !important; margin: 0 !important;"><DIV class="page-title" style="background-color: white !important; border-top: 0px solid #222;"><DIV class="container" style=" padding: 0 !important;"><DIV class="col-lg-12 col-md-12 col-xs-12" style="padding-right: 0 !important; padding-bottom: 0 !important; position: relative; left: -10px;"><DIV style="border-right: 5px solid #393939; padding: 20px 20px 0px 0px"><DIV class="header-title heidelberg-red"> 
    Parts </DIV><DIV class="header-subtitle" style="color: #393939 !important;"> 
    Discover our workpieces and standards </DIV></DIV></DIV></DIV></DIV><DIV class="container" style="background-color: white !important; padding: 0 !important;"><DIV class="col-lg-12 col-md-12 col-xs-12" style="padding: 0 !important; margin: 0px important!;"><DIV class="container"><DIV class="row"><DIV class="col-lg-12 col-md-12 col-xs-12 content" style="display: inline-block; padding-bottom: 10px; padding-top:10px;"><DIV class="row equal_height"><DIV class="col-lg-4
                 col-md-4
                 col-sm-12 col-xs-12"><A href="#Basic_Part" class="box hyperbox" style="margin-bottom:15px; display:block;"><H3 style="line-height: 50px !important; margin-top:0px; text-align:left">
                    Our Best Basic Part</H3>
            We present as best basic part the codon-optimized version of the cytochrome c protein that is able to convert silicon educts to organosilicon products. 
        </A></DIV><DIV class="col-lg-6
                 col-md-6
                 col-sm-12 col-xs-12"><A href="#Composite_Part" class="box hyperbox" style="display:block; position:relative; margin-bottom:15px;"><H3 style="line-height: 50px !important; margin-top:0px; text-align: left">
                    Our Best Composite Part</H3>  As best composite part, we present the theophylline riboswitch. In order to select for the cytochrome c that most efficiently converts caffeine to theophylline, evolutionary pressure is applied dependent on geneIII expression and subsequent phage propagation.
          </A></DIV></DIV><DIV class="row equal_height"><DIV class="col-lg-6
                 col-md-6
                 col-sm-12 col-xs-12"><A href="#Improved_Part" class="box hyperbox" style="margin-bottom:15px; display:block;"><H3 style="line-height: 50px !important; margin-top:0px; text-align:left">
                    Our Improved Part</H3>
            Our improved part, beta-lactamase_E102F, was designed to confirm the generative capabilities of our software package. Our algorithm has proven to be successful by engineering the enzyme to exhibit more than twice the activity of the wildtype protein.
        </A></DIV><DIV class="col-lg-6
                 col-md-6
                 col-sm-12 col-xs-12"><A href="#Part_List" class="box hyperbox" style="margin-bottom:15px; display:block;"><H3 style="line-height: 50px !important; margin-top:0px; text-align:left">
                   Our Part List</H3>
            Here, we present the complete list of all parts that compose our evolutionary toolbox. Thereby, we aim to provide the synthetic biology community with all essential parts for their own accelerated evolution. 
        </A></DIV></DIV><DIV class="row equal_height"><DIV class="col-lg-12
                 col-md-12
                 col-sm-12 col-xs-12"><A href="#Part_Collection" class="box hyperbox" style="margin-bottom:15px; display:block;"><H3 style="line-height: 50px !important; margin-top:0px; text-align:left">
                   Our Parts Collection</H3>
            We present the first self-contained in vivo evolution toolbox comprising 26 standardized BioBricks. At the heart of our toolbox is an innovative workflow for enzyme engineering, paving the way towards bio-production of novel compounds. It comprises promiscuous enzyme candidates (CYP1A2 and Cytochrome c) and selection-mediating accessory parts such as our riboswitch for detection of an organosilicon (BBa_K2398555).
        </A></DIV><DIV class="col-lg-12
                 col-md-12
                 col-sm-12 col-xs-12"><A href="https://2017.igem.org/Team:Heidelberg/RFC" class="box hyperbox" style="margin-bottom:15px; display:block;"><H3 style="line-height: 50px !important; margin-top:0px; text-align:left">
                   Our RFC</H3>
             Our BBF RFC combines the methodical aspects of this year´s iGEM Team Heidelberg project, presenting a new golden-gate cloning standard for simple production of geneIII-deficient phages carrying any gene-of-interest and the first accessory plasmid construction kit for selection of the best phage library member on the one hand, while having our easy-to-use and low-cost bacteriophage based in vivo directed method PREDCEL (Phage-RElated DisContinous EvoLution) on the other hand. Together with our ODE-based modeling and computer simulations performed in context of AIGEM, our RFC opens up the principle of Phage-assisted continous evolution (PACE) to the scientific world, enabling every other iGEM team to perform directed evolution on their own without the highly challenging task of setting up a PACE apparatus. 
        </A></DIV></DIV><DIV class="container"><DIV class="row"><DIV class="col-lg-12 col-md-12 col-xs-12 content" style="display: inline-block; padding-bottom: 10px; padding-top:10px;"><H2 id="Basic_Part" style="text-align: center !important">Our Best Basic Part: Cytochrome C</H2><DIV class="row"><DIV class="col-md-4 col-sm-12 col-xs-12 content"><DIV class="well well-sm"><H4><A href="http://parts.igem.org/Part:BBa_K2398000" style="color: #9d1c20 !important">BBa_K2398000</A></H4>
          
            We present as best basic part the codon-optimized version of the cytochrome c protein derived from <I><A href="http://www.rcsb.org/pdb/explore.do?structureId=3cp5" style="color: #9d1c20 !important">Rhodotermus marinus</A></I> that is able to convert silicon educts to organosilicon products.
    </DIV></DIV></DIV>

This cytochrome c variant provides an easy to use tool that is accessible to everyone in the synthetic biology community and allows the user to harness the vast potential of organosilicons.
This basic part exhibits a strong tendency to form silicon-carbon bonds and is a valuable addition to perform controlled organic chemistry in microorganisms.
A triple mutant [Arnold et al., 2016] of this part has already been applied in the successful synthesis of organosilicons as a proof-of-concept.
As a next step, this part can be implemented in the directed evolution approach of phage-assisted continuous evolution (<A href="https://2017.igem.org/Team:Heidelberg/Pace" style="color: #9d1c20 !important">PACE</A>) or in the phage-related discontinuous evolution (<A href="https://2017.igem.org/Team:Heidelberg/Predcel" style="color: #9d1c20 !important">PREDCEL</A>) approach to improve organosilicon synthesis by cytochrome engineering.

For the characterization of this part, we conducted experiments for the production of organosilicons and therefore used a previously described triple mutant, already engineered by [Arnold et al., 2016]. According to the <A href="https://2017.igem.org/Team:Heidelberg/Experiments" style="color: #9d1c20 !important">protocol</A>, we obtained the matured cytochrome c and performed conversion experiments analyzed via GC-MS method. The following figures show the results obtained with the previously engineered enzyme. First, we tested the enzyme on the conversion of the compound (1) with (3) to the final product of (5) (Fig. 1). The gas chromatogram shows one product peak which emerges at 9.2 min retention time. Analysis of the gas chromatogram indicates a conversion rate of &gt;99% on the basis of the non-existence of the respective educt peak. As we performed GC-MS analysis, mass spectrometry data were also available which can be seen in Fig. 3 for the product peak that emerges at 9.2 minutes. It shows the fission of this product and can clearly be identified as our desired product (5). Furthermore, we performed experiments on the compound (1) which differs from compound (2) in its amino functional group. The product is eluted after 11.7 minutes retention time (Fig. 4) and also confirmed as the right product via mass spectrometry in Fig. 5. Worth mentioning, there is still educt (1) present which emerges at 6.9 minutes retention time, indicating incomplete conversion. Calculations showed a conversion rate of 47.5% for the product (4).As these results only show activity for a previously engineered cytochrome c, we are convinced that our part is still functional despite not as effective as the mutant. This part as our best basic part is intended as a platform for future teams to evolve their own cytochrome c for their specific target molecule.

<DIV class="container"><DIV class="row"><DIV class="col-lg-12 col-md-12 col-xs-12 content" style="display: inline-block; padding-bottom: 10px; padding-top:10px;"><DIV class="col-lg-12 col-md-12 col-xs-12"><DIV class="mdl-shadow--4dp" style="padding:20px; float:left}"><DIV class="img-caption title"><STRONG><B>
      Figure 1:
    </B></STRONG></DIV><DIV class="img-caption subtitle">
      Compounds used in the project of organosilicons and its respective products.
    </DIV></DIV></DIV><DIV class="col-lg-12 col-md-12 col-xs-12"><DIV class="mdl-shadow--4dp" style="padding:20px; float:left}"><DIV class="img-caption title"><STRONG><B>
      Figure 2:
    </B></STRONG></DIV><DIV class="img-caption subtitle">
      Gas chromatogram for the reaction of educt (1) and (3) to the product (5). 11.7 minutes retention time indicates product formation. Unconverted educts converge 6.9 and 7.2 (7.4) minutes.
    </DIV></DIV></DIV><DIV class="col-lg-12 col-md-12 col-xs-12"><DIV class="mdl-shadow--4dp" style="padding:20px; float:left}"><DIV class="img-caption title"><STRONG><B>
      Figure 3:
    </B></STRONG></DIV><DIV class="img-caption subtitle">
      Mass chromatogram shows the breakdown of the product (5). The product itself corresponds to a mass of 251 daltons.
    </DIV></DIV></DIV><DIV class="col-lg-12 col-md-12 col-xs-12"><DIV class="mdl-shadow--4dp" style="padding:20px; float:left}"><DIV class="img-caption title"><STRONG><B>
      Figure 4:
    </B></STRONG></DIV><DIV class="img-caption subtitle">
      Gas chromatogram for the reaction of educt (2) and (3) to the product (4). 9.2 minutes retention time indicates product formation.
    </DIV></DIV></DIV><DIV class="col-lg-12 col-md-12 col-xs-12"><DIV class="mdl-shadow--4dp" style="padding:20px; float:left}"><DIV class="img-caption title"><STRONG><B>
      Figure 5:
    </B></STRONG></DIV><DIV class="img-caption subtitle">
      Mass chromatogram shows the breakdown of the product (4). The product itself corresponds to a mass of 236 daltons.
    </DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="container"><DIV class="row"><DIV class="col-lg-12 col-md-12 col-xs-12 content" style="display: inline-block; padding-bottom: 10px; padding-top:10px;"><H2 id="Composite_Part" style="text-align: center !important">Our Best Composite Part</H2><DIV class="container"><DIV class="row"><DIV class="col-lg-12 col-md-12 col-xs-12 content" style="display: inline-block; padding-bottom: 10px; padding-top:10px;"><A href="http://parts.igem.org/Part:BBa_K2398019:Design"><H1>BBa_K2398019:  Theophylline Riboswitch - GeneIII for the Evolution of Enzymes</H1></A>
One of our main goals was to facilitate for the <I>in vivo</I> directed evolution of small molecule binding enzymes. The concept behind this idea is the use of riboswitches for the activation of geneIII. To prove our idea, we made use of a <A href="https://2017.igem.org/Team:Heidelberg/Cytochrome_Engineering">theophylline binding riboswitch</A> in relation with the Enzyme CYP1A2, which catalyzes the reaction of caffeine to theophylline. As we could show that the principle of enzyme evolution with the help of riboswitches works, our best composite part is the theophylline riboswitch, followed by geneIII and flanked ba homology regions, which make it compatible with our <A href="https://2017.igem.org/Team:Heidelberg/RFC">cloning standard</A>. 

<H2>Background</H2></DIV></DIV></DIV><DIV class="container"><DIV class="row"><DIV class="col-lg-12 col-md-12 col-xs-12 content" style="display: inline-block; padding-bottom: 10px; padding-top:10px;"><DIV class="col-lg-12 col-md-12 col-xs-12"><DIV class="mdl-shadow--4dp" style="padding:20px; float:left}"><DIV class="img-caption title"><STRONG><B>
      
Figure 1: Selection process for improved CYP1A2 variants in directed evolution experiments
    </B></STRONG></DIV><DIV class="img-caption subtitle">
      
  The evolutionary circle starts by M13 phages injecting their genome (SP) into bacterial cells, already containing two additional plasmids, AP and MP. The SP encodes CYP1A2 among genes (except geneIII) that are crucial for phage propagation. If through MP activation mutations in the CYP1A2 gene lead to improved CYP1A2 variants the intracellular level of theophylline increases. Theophylline molecules activate the theophylline riboswitch on the AP and thereby enhance geneIII expression. The assembled phages containing the improved CYP1A2 variant can leave the cell and propagate by infecting new cells.
            
    </DIV></DIV></DIV></DIV></DIV></DIV><DIV class="container"><DIV class="row"><DIV class="col-lg-12 col-md-12 col-xs-12 content" style="display: inline-block; padding-bottom: 10px; padding-top:10px;">
      


Our evolution circuit for cytochrome engineering works as follows: Bacteriophages infect bacterial cells by introducing their genome. The genome encodes a Selection Plasmid (SP) that contains the human CYP1A2 variant and all necessary components for virus propagation except geneIII. The Accessory Plasmid (AP) codes for geneIII driven by a Psp-tet promoter and contains a riboswitch, located between the promoter and geneIII, which regulates the expression rate of geneIII. The riboswitch on the AP is only active if theophylline reaches a certain concentration within the bacterium. If CYP1A2 is active, caffeine is converted to theophylline. Gradually the theophylline concentration increases and acts upon the riboswitch, which enables expression of geneIII. Additionally, the AP encodes for the chaperone HDJ-1, which is essential to receive the functional CYP1A2 enzyme (Fig. 1). 


    </DIV></DIV></DIV><DIV class="container"><DIV class="row"><DIV class="col-lg-12 col-md-12 col-xs-12 content" style="display: inline-block; padding-bottom: 10px; padding-top:10px;">
      
        

In a first step, we wanted to validate our AP. Therefore, we added theophylline with a concentration of 100 µM to our inoculated culture and performed two rounds of PREDCEL. Afterwards, we determined the phage titers by plaque assays. Our theophylline treated culture displayed approximately two times higher phage titers than the non-treated control culture.
Using the same experimental conditions, but replacing the theophylline treatment by a 300 µM caffeine treatment, we verified the functionality of CYP1A2 and thus of our SP. If caffeine is added to the culture, CYP1A2 catalyzes the reaction from caffeine to theophylline. The resulting increase of the theophylline concentration further activates the riboswitch on the AP and phage propagation is stimulated (Fig. 2).
For the evolution of proteins via PREDCEL the addition of a Mutagenesis Plasmid (MP) is essential. For our cytochrome engineering approach we have chosen MP4, which induces a medium mutation rate <X-REF>badran2015development</X-REF>. After six iterations of our optimized PREDCEL workflow, we performed plaque assays and sequenced single plaques. The sequenced plaques showed five recurrent mutations demonstrating that we are able to induce mutations with our experimental setup and that we are able to evolve enzymes (Fig. 7).


    </DIV></DIV></DIV><DIV class="container"><DIV class="row"><DIV class="col-lg-12 col-md-12 col-xs-12 content" style="display: inline-block; padding-bottom: 10px; padding-top:10px;"><DIV class="col-lg-12 col-md-12 col-xs-12"><DIV class="mdl-shadow--4dp" style="padding:20px; float:left}"><DIV class="img-caption title"><STRONG><B>
       
Figure 2:
    </B></STRONG></DIV><DIV class="img-caption subtitle">
       Plaque assays performed after two passages of CYP1A2 PREDCEL  
with either adding 100 µM theophylline or 300 µM demonstrating the  
functionality of the Accessory Plasmid and the Selection Plasmid,  
respectively. Adding theophylline increases the geneIII expression  
2-fold. Adding caffeine enhances the conversion of caffeine to  
theophylline and thus increases the geneIII expression as well.
    </DIV></DIV></DIV></DIV></DIV></DIV><DIV class="container"><DIV class="row"><DIV class="col-lg-12 col-md-12 col-xs-12 content" style="display: inline-block; padding-bottom: 10px; padding-top:10px;"><DIV class="col-lg-12 col-md-12 col-xs-12"><DIV class="mdl-shadow--4dp" style="padding:20px; float:left}"><DIV class="img-caption title"><STRONG><B>
       
Figure 3: 
    </B></STRONG></DIV><DIV class="img-caption subtitle">
      Sequencing results of eight plaques after 6 iterations of  
the PREDCEL workflow with MP4 illustrating recurring  
mutations of the CYP1A2 gene. Recurrent mutations with amino acid  
exchange are indicated in red, without amino acid exchange in orange.  
Single mutations with amino acid changes are shown in yellow, and  
without amino acid changes in blue.
    </DIV></DIV></DIV></DIV></DIV></DIV><DIV class="container"><DIV class="row"><DIV class="col-lg-12 col-md-12 col-xs-12 content" style="display: inline-block; padding-bottom: 10px; padding-top:10px;"><DIV class="col-lg-12 col-md-12 col-xs-12"><DIV class="mdl-shadow--4dp" style="padding:20px; float:left}"><DIV class="img-caption title"><STRONG><B>
       
Figure 4:
    </B></STRONG></DIV><DIV class="img-caption subtitle">
      
Cell lysates were used for high-performance liquid chromatography to  
distinguish theophylline (left peak) from caffeine (right peak). This  
assay allows a quantification of the educt and the product of CYP1A2  
and thus a conclusion about the conversion efficiency can be made.
    </DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="container"><DIV class="row"><DIV class="col-lg-12 col-md-12 col-xs-12 content" style="display: inline-block; padding-bottom: 10px; padding-top:10px;"><H2 id="Improved_Part" style="text-align: center !important">Improved Part</H2><A href="http://parts.igem.org/Part:BBa_K2398524"><H1>Engineered highly active Beta-Lactamase mutant E102F</H1></A>
This part is an improvement of <A href="http://parts.igem.org/Part:BBa_K1189009">BBa_K1189009: Beta-Lactamase with His-Tag</A>.The aim of our project was to generate protein functionality, <I>in vivo</I> through PACE and PREDCEL and in <I>in vitro</I> with help of our software suite AiGEM. To prove that our circuit works, we mutated ß-lactamases based on the scores of our software. Many different mutants were cloned and screened for their minimal inhibitory concentration (MIC). Visit our software validation site <A href="https://2017.igem.org/Team:Heidelberg/Validation">here</A>. Typical methods to determine MICs are broth or agar dilutions. The principle is as simple as effective. The microorganism is cultured in liquid medium or on plates with different concentrations of the inhibiting agent <X-REF>JM10291</X-REF>. With this method, concentrations which are tolerated by the organism and those who inhibit growth can be determined. However, such an assay gives no information about the kind of toxicity e.g. whether the chemical is cytotoxic or cytostatic.
In our case, we want to evaluate differences in the activity of a well characterized protein. Consequently, such an dilution method is ideal. It is easy to setup, very robust and gives exactly the information that is needed for the characterization.
To start the assay, colonies were picked into LB medium with 100 µg/ml chloramphenicol, but without carbenicillin and were grown to the stationary phase for 20 h. Then, deep well plates with 500 µl LB broth per well were prepared. The medium was supplemented with 100 µg/ml chloramphenicol and for each mutant, eight wells with different carbenicillin concentrations, ranging from 10-1280 µg/ml were prepared. Each well was inoculated with 2 µl of the proculture. The new cultures were incubated another 8 h. Finally, the OD600 of each well was measured. By this measure we were able to assert the growth ability of each variant. A first screening revealed, that beside catalytically inactive variants, we also had many candidates that even could survive the highest carbenicillin concentrations (1280 µg/ml)(Fig.: 1). 



As a result, we decided to perform a second test for these candidates with higher antibiotic concentrations. The three highest carbenicillin concentrations of the first data set were included and the range was extended to a maximum concentration of 19.2 mg/ml. While several candidates turned out to have a weaker activity than the wildtype protein, five of them showed improved properties. The two best candidates could even grow at 19.2 mg/ml (Fig.: 2). The variants were contained one point mutation each, E102F and M67G. These most benefitial mutations, appear in many of our better candidates as well, which underlines there functionality. 



The outstanding candidate was the E102F mutant, which could <B>even grow at 19.2 mg/ml</B> without affection by the antibiotic. This results in a <B>improvement of at least 100 %</B> in comparison to the wildtype ß-lactamase. This remarkable gain of activity proves, that our software is not only able to recognize protein classes, but also to generate new function.


<DIV class="col-lg-6 col-md-12 col-xs-12 pull-left image-box-left"><DIV class="mdl-shadow--4dp" style="padding:20px; float:left"><DIV class="img-caption title"><STRONG><B>
      
            	Figure 1: Gowth Behaviour of \(\beta\)-Lactamase Mutants under Different Carbenicillin Concentrations
    </B></STRONG></DIV><DIV class="img-caption subtitle">
      
The plot shows the OD600 of the different \(\beta\)-lactamase mutants at different carbenicillin concentrations. The pool of mutants is very heterogenous. Some proteins are not active at all, some show different grades of activity and 11 mutants, as well as the wildtype enzyme can grow in all conditions.</DIV></DIV></DIV><DIV class="col-lg-6 col-md-12 col-xs-12 pull-left image-box-left"><DIV class="mdl-shadow--4dp" style="padding:20px; float:left"><DIV class="img-caption title"><STRONG><B>
      
            	Figure 2: Gowth Behaviour of \(\beta\)-Lactamase Mutants under Elevated Carbenicillin Concentrations
    </B></STRONG></DIV><DIV class="img-caption subtitle">
      
The plot depicts the \(OD_{600}\) of the \(\beta\)-lactamase mutants that could survive at 1280 \(\frac{µg}{ml}\) in the first assay, under more stringent conditions. The majority of these \(\beta\)-lactamases have a slightly weaker activity than the wildtype, but we could determine five variants with a higher enzymatic activity.</DIV></DIV></DIV></DIV></DIV></DIV><DIV class="container"><DIV class="row"><DIV class="col-lg-12 col-md-12 col-xs-12 content" style="display: inline-block; padding-bottom: 10px; padding-top:10px;"><H2 id="Part_Collection" style="text-align: center !important">Parts Collection</H2>
            
We present the first self-contained in vivo evolution toolbox comprising 26 standardized BioBricks. The collection includes several key aspects of our project:




The crucial step of to the cloning of PACE circuits is the generation of the accessory plasmid. These plasmids allow geneIII expression dependent on the evolving protein. The link between the fitness of the protein of interest and the expression of geneIII determines the effectiveness of the directed evolution and the presence of ProteinIII is essential for the production of new phage particles. 
Therefore, we provide an accessory plasmid construction kit for phage library selection and propagation in E. coli based on conditional geneIII complementation. The geneIII-driving promotor, RBS and plasmid ori can thereby be fully customized.
If more than one variant of a circuit should be tested at a time, it is necessary to modify the activation region for geneIII and the additional gene that can be located on the AP with a minimum of effort. This is the another reason for a efficient cloning strategy.
To make AP cloning as simple as possible, we defined a new cloning standard that is specifically suited for the assembly of APs, we wrote a <A href="https://2017.igem.org/Team:Heidelberg/RFC">BBF RFC</A> that describes our concept in detail (Fig. 1).


<DIV class="col-lg-12 col-md-12 col-xs-12"><DIV class="mdl-shadow--4dp" style="padding:20px; float:left}"><DIV class="img-caption title"><STRONG><B>
      
Figure 1: In our cloning standard, compatible building blocks are defined by specific functionalities. They are flanked by defined homology regions,
indicated by numbers, which are necessary for the assembly of the APs with the Gibson method. This
results in a highly customizable plasmid, composed of the desired origin of replication, an antibiotic
resistance (4-5), a bicistronic operon with geneIII (2-3)and the desired reporter (3-4), which can be
activated by any promoter (1-2)and a second expression cassette for additional genes that are necessary
for the respective circuit (1-5).
    </B></STRONG></DIV></DIV></DIV>
In Addition, an <A href="https://2017.igem.org/Team:Heidelberg/Optogenetics">optogenetic selection stringency modulator</A> comprising the light-dependent EL222 transcription factor and an engineered, hybrid Psp-EL222 promoter.

Furthermore, we provide parts for production of geneIII-deficient M13 phages carrying a custom gene-of-interest to be evolved.

On order to validate our system, we constructed the plasmids for our <A href="https://2017.igem.org/Team:Heidelberg/CRISPR">CRISPR/Cas9</A> subproject according to the RFC cloning standard. We could constuct several plasmids with help of this standard.

At the heart of our toolbox is an innovative workflow for <A href="https://2017.igem.org/Team:Heidelberg/Cytochrome_Engineering">enzyme engineering</A>, paving the way towards <A href="https://2017.igem.org/Team:Heidelberg/Organosilicons">bio-production of novel compounds</A>. It comprises promiscuous enzyme candidates (CYP1A2 and Cytochrome c) and selection-mediating accessory parts such as our <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2398019">riboswitch</A> for detection of an organosilicon (BBa_K2398555). Our simple <A href="https://2017.igem.org/Team:Heidelberg/Predcel">PREDCEL protocol</A>  and accompanying <A href="https://2017.igem.org/Team:Heidelberg/Toolbox">online toolbox</A> guide bring in vivo evolution to future iGEM teams and the broader synbio community. 

Finally we created a set of parts, which were used in our different subprojects. These parts were transferred into pSB1C3 in a way that they are still compatible with our standard and submitted them to the registry:<DIV class="accordion" role="tablist" aria-multiselectable="true" style="padding-top: 30px"><DIV class="panel panel-default"><DIV class="panel-heading" role="tab" id="tab1"><H4 class="panel-title"><A role="button" data-toggle="collapse" data-parent="#accordion" href="#parts_collection_table-collapse" aria-expanded="false" aria-controls="parts_collection_table-collapse"><DIV><DIV class="col-md-11">
                        List of all parts of the Collection
                    </DIV></DIV></A></H4></DIV><DIV id="parts_collection_table-collapse" class="panel-collapse collapse" role="tabpanel" aria-labelledby="parts_collection_table"><DIV class="panel-body"><DIV class="col-lg-12 col-md-12 col-xs-12 content" style="padding-top: 20px; padding-bottom: 20px"><DIV class="col-xs-12"><DIV id="partsTable_collection" class="table-slot" style="overflow-x: auto">Loading...</DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="container"><DIV class="row"><DIV class="col-lg-12 col-md-12 col-xs-12 content" style="display: inline-block; padding-bottom: 10px; padding-top:10px;"><H2 id="Part_List" style="text-align: center !important">Parts List</H2><DIV class="col-xs-12"><DIV id="partsTable" class="table-slot" style="overflow-x: auto">Loading...</DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV>

}}
                          
</DIV><SECTION id="footer-sec" style="background-color: #222 !important"><DIV class="container"><DIV class="row" style="padding-top: 30px;"><DIV class="col-md-3"><H4 style="text-align: center">Quote</H4><P><I>“”</I></P></DIV><DIV class="col-md-3"><H4 style="text-align: center"> Useful <STRONG>Links</STRONG></H4><P><A href="https://igem.org/Special:SpecialPages">
			Special pages
		</A></P><P><A href="https://igem.org/Main_Page">
			Main Page
		</A></P></DIV><DIV class="col-md-3"><H4 style="text-align: center"> Follow us on </H4><DIV style="text-align: center;"><SPAN style="color: grey; text-align: center;">&amp;</SPAN></DIV></DIV><DIV class="col-md-3"><H4 style="text-align: center"> Contact us </H4><P>

               iGEM-Heidelberg2017@bioquant.uni-heidelberg.de
               
		 Im Neuenheimer Feld 267
		
		69120 Heidelberg
                    </P></DIV></DIV></DIV></SECTION></DIV></DIV></DIV></DIV></BODY></HTML>