<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Heidelberg_Predcel skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Heidelberg/Predcel</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container"><DIV class="navbar-header"><A type="button" data-canvas="body" id="sidebarCollapse" class="navbar-toggle" data-toggle="offcanvas" data-target="#sidebar" style="color: #393939;margin-top: 9px;">NAVIGATION
                    </A></DIV><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav pull-right"><LI class="dropdown"><A href="https://2017.igem.org/Team:Heidelberg/Team">People </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Team">Team</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Attributions">Attributions</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Sponsoring">Sponsoring</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/For_Judges">For Judges</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Jamboree">Giant Jamboree</A></LI></UL><LI class="dropdown"><A href="https://2017.igem.org/Team:Heidelberg/Description">Project </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Description">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Design">Background &amp; Design </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Pace">PACE</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Predcel">PREDCEL</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Results">Results Summary</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Validation">Software Validation</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Organosilicons">Organosilicons</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Protein_Interaction">Protein Interaction</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Cytochrome_Engineering">Cytochrome Engineering</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Optogenetics">Optogenetics</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/CRISPR">CRISPR Cas9</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/InterLab">InterLab Study</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Achievements">Achievements</A></LI><LI class="dropdown"><A href="https://2017.igem.org/Team:Heidelberg/Parts">Parts </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Basic_Part">Basic Parts</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Composite_Part">Composite Parts</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Improved_Part">Improved Part</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Part_Collection">Parts Collection</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Part_List">Part List</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/RFC">RFC</A></LI><LI class="dropdown"><A href="https://2017.igem.org/Team:Heidelberg/Model">Modeling </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Phage_Titer">Phage Titer</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Mutagenesis_Induction">Mutagenesis Induction</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Mutation_Rate_Estimation">Mutation Rate Estimation</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Lagoon_Contamination">Lagoon Contamination</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Medium_Consumption">Medium Consumption</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Tools">Interactive Web Tools</A></LI><LI class="dropdown"><A href="https://2017.igem.org/Team:Heidelberg/Software">Software </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software">AiGEM Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software/DeeProtein">DeeProtein</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software/GAIA">GAIA</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software/SafetyNet">SafetyNet</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software/MAWS">MAWS 2.0</A></LI><LI class="dropdown"><A href="https://2017.igem.org/Team:Heidelberg/Human_Practices">Human Practices </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Human_Practices">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/HP/Gold_Integrated">Integrated Human Practices</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Interviews">Expert Interviews</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Safety">Safety &amp; Security</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Engagement">Public Engagement</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Education">Education</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Collaborations">Collaborations</A></LI><LI class="dropdown"><A href="https://2017.igem.org/Team:Heidelberg/Notebook">Notebook </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Notebook">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Experiments">Methods</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Materials">Materials</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Database">Notebook Database</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Toolbox">Toolbox Guide</A></LI></DIV></DIV><NAV id="sidebar" style="z-index: 10001; background-color: white;border: none;box-shadow:1px 0px 0px #393939;" class="navmenu navmenu-default navmenu-fixed-left offcanvas" role="navigation"><A class="navmenu-brand" href="https://2017.igem.org/Team:Heidelberg"> iGEM TEAM HEIDELBERG 2017</A><DIV id="sidebarHide" data-toggle="offcanvas" data-target="#sidebar" style="padding-left: 20px;"><A style="font-family: 'Josefin Sans' !important; font-size: 16px; line-height: 18px; height: 18px;"> COLLAPSE NAVIGATION </A></DIV><UL class="nav navmenu-nav" id="sidenav"><LI><A data-toggle="collapse" data-target="#People_side">People </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Team">Team</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Attributions">Attributions</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Sponsoring">Sponsoring</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/For_Judges">For Judges</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Jamboree">Giant Jamboree</A></LI></UL><LI><A data-toggle="collapse" data-target="#Project_side">Project </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Description">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Design">Background &amp; Design </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Pace">PACE</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Predcel">PREDCEL</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Results">Results Summary</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Validation">Software Validation</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Organosilicons">Organosilicons</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Protein_Interaction">Protein Interaction</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Cytochrome_Engineering">Cytochrome Engineering</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Optogenetics">Optogenetics</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/CRISPR">CRISPR Cas9</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/InterLab">InterLab Study</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Achievements">Achievements</A></LI><LI><A data-toggle="collapse" data-target="#Parts_side">Parts </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Basic_Part">Basic Parts</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Composite_Part">Composite Parts</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Improved_Part">Improved Part</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Part_Collection">Parts Collection</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Part_List">Part List</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/RFC">RFC</A></LI><LI><A data-toggle="collapse" data-target="#Model_side">Modeling </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Phage_Titer">Phage Titer</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Mutagenesis_Induction">Mutagenesis Induction</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Mutation_Rate_Estimation">Mutation Rate Estimation</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Lagoon_Contamination">Lagoon Contamination</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Medium_Consumption">Medium Consumption</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Tools">Interactive Web Tools</A></LI><LI><A data-toggle="collapse" data-target="#AI_side">Software </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software">AiGEM Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software/DeeProtein">DeeProtein</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software/GAIA">GAIA</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software/SafetyNet">SafetyNet</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software/MAWS">MAWS 2.0</A></LI><LI><A data-toggle="collapse" data-target="#HP_side">Human Practices </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Human_Practices">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/HP/Gold_Integrated">Integrated Human Practices</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Interviews">Expert Interviews</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Safety">Safety &amp; Security</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Engagement">Public Engagement</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Education">Education</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Collaborations">Collaborations</A></LI><LI><A data-toggle="collapse" data-target="#Notebook_side">Notebook </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Notebook">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Experiments">Methods</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Materials">Materials</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Database">Notebook Database</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Toolbox">Toolbox Guide</A></LI></NAV><DIV style="background-color: white !important; padding: 0 !important; margin: 0 !important;"><DIV class="page-title" style="background-color: white !important; border-top: 0px solid #222;"><DIV class="container" style=" padding: 0 !important;"><DIV class="col-lg-12 col-md-12 col-xs-12" style="padding-right: 0 !important; padding-bottom: 0 !important; position: relative; left: -10px;"><DIV style="border-right: 5px solid #393939; padding: 20px 20px 0px 0px"><DIV class="header-title heidelberg-orange">                
    PREDCEL </DIV><DIV class="header-subtitle" style="color: #393939 !important;"> 
    SIMPLIFY DIRECTED EVOLUTION </DIV></DIV></DIV></DIV></DIV><DIV class="container" style="background-color: white !important; padding: 0 !important;"><DIV class="col-lg-12 col-md-12 col-xs-12" style="padding: 0 !important; margin: 0px important!;"><DIV class="container" style="position: relative; left: -10px; border-left: 5px solid #393939; border-top: 5px solid #393939; padding: 0 !important"><DIV class="col-lg-12 col-md-12 col-xs-12 content" style="text-align: justify; text-color: #393939 !important; padding: 20px 5px 0px 20px !important; font-weight: bold !important; font-family: 'Open Sans' !important">
        
        With our newly created Phage RElated DisContinuous EvoLution (PREDCEL) method adopted from Phage Assisted Continuous Evolution (PACE) we want to enable scientists all over the world to easily improve a whole range of different proteins for innumerable applications. 
    
    </DIV></DIV><DIV class="container"><DIV class="row"><DIV class="col-lg-12 col-md-12 col-xs-12 content" style="display: inline-block; padding-bottom: 10px; padding-top:10px;"><H1>Introduction</H1>

      Trying to set up <A href="https://2017.igem.org/Team:Heidelberg/Pace" class="innerlink">PACE</A> we had to deal with several problems, most importantly phage washout (i.e. complete phage loss after few hours of continuous evolution) and phage contamination (due to the complex flow setup, contaminations are difficult to avoid). Further, we realized, that the PACE apparatus was very static, strongly limiting its applications scope. For evolution of proteins inducible by chemical for instance, one would ideally want to instantly alternate between evolution in presence of the chemical inducer and as well as corresponding selection strains. This is impossible in a continuous flow setup. Therefore, we sought out to create a more simple and more flexible PACE alternative, which can be quickly implemented by any trained biologist without the need for special equipment or knowledge. Inspired by a recent publication on phage-mediated selection of gene libraries <X-REF>BRODELETAL..2016</X-REF><X-REF>RN160</X-REF>, we created a simple protocol named PREDCEL (for phage-related discontinuous evolution), which uses simple batch-wise, manual transfer of the evolving phage gene pool (Figure 1). In essence, PREDCEL reduces the entire complexity of PACE to simple, standard laboratory procedures, all the while gaining entire flexibility to easily swap conditions (strains, inducers etc.) between individual rounds of evolution. We also provide an <A href="https://2017.igem.org/Team:Heidelberg/Optogenetics" class="innerlink">optogenetic tool</A> for simple adaptation of the selection pressure during PREDCEL runs.

       <H1>PREDCEL Procedure</H1>

As it is important to prevent phage contaminations at any time, all the following PREDCEL steps have to be performed with filter tips. Benches should be cleaned with 10% H202 (pay attention with handling) or another suitable solution for phage inactivation. If practicable, use UV light on benches and in incubators to sterilize after and before your experiments.  <DIV class="col-lg-6 col-md-12 col-xs-12 pull-right image-box-right"><DIV class="mdl-shadow--4dp" style="padding:20px; float:right"><DIV class="img-caption title"><STRONG><B>
      
                Figure 1: PREDCEL procedure
    </B></STRONG></DIV><DIV class="img-caption subtitle">
      
                Learn how to perform PREDCEL in only nine steps. Just follow the instructions given right here.
    </DIV></DIV></DIV><B>Step 1: F+ strains</B>
After successfully transforming your AP and MP plasmids into your bacterial strain (see transformation protocols), make sure the used strain carries the F-Pilus plasmid as it is needed for proper M13 phage infection. Therefore, grow a bacterial culture from a picked single colony. This culture should be plated on a tetracycline (tet) plate, since the strain should carry a tetracycline resistance on its F-Pilus plasmid.  <B>Step 2: Prove MP to work</B>
Single colonies showing a positive result in Step 1 should be picked and grown in 2YT medium, supplemented with the appropriate antibiotics and 100 mM glucose. Glucose must be added as it prevents the induction of the pBad promotor on the MP and thereby the mutagenesis. The resulting bacterial growth should grow until an OD600 of 0.6 - 1.0 is reached. Following this, the functionality of the MP should be tested before starting your PREDCEL run. Therefore, follow instructions of the MP testing protocol. The evaluation of your AP should take place before starting your PREDCEL run as well.  <B>Step 3: Glycerol stocks</B>
Based on a positive result in Step 2, glycerol stocks from the prepared bacterial culture should be prepared. In order to do that, a main stock of 2 ml and several aliquots of 100 µl in PCR tubes should be prepared (see respective protocol) and stored at -80°C.  <B>Step 4: Contamination test</B>
Before finally starting the PREDCEL run, the bacterial culture and thereby the glycerol stock should be tested for a phage contamination. Therefore, plaque assays with the supernatant of the culture should be implemented following the plaque assay protocol. If the results are showing no plaques, your PREDCEL run can start.  <B>Step 5: Centrifugation and MP activation</B>
Starting your PREDCEL run, a new culture should be grown, using one of the 100 µl glycerol stock aliquots for inoculation. This culture should also reach an OD600 = 0.6-1.0 and has to be prepared early enough to minimize time between gain of phage supernatant and new infection round. If the culture reaches the respective OD600 too fast, dilute until you can infect with phages or cool on ice if it takes less than 10 minutes until infection. Centrifuge 10 ml or more of your culture for 10 min at 3750 g at room temperature to pellet your bacteria. Afterwards, resolve your cell pellet in a 150 ml flask with a volume of 2YT equal to the volume spun down containing 100 mM arabinose to induce MP.  <B>Optional:</B> Keep 1 ml of culture supernatant after centrifugation and 1ml of medium solution you dissolve with. Store both at 4°C. Additional to PCR they could later be used for contamination check by plaque assay.  <B>Step 6: Phage infection</B>
In the next step, infect the resuspension with a suitable number of phages. As titers of inoculation phage can vary a lot a multiplicity of infection (MOI) of 1 is recommended for PREDCEL, rather than giving a specific inoculation volume. MOI of 1 means that there is one phage per cell in solution. In contrast, while performing PREDCEL, define a certain phage supernatant volume you kept at 4°C to be transferred, for instance 1 ml, as phage titers cannot be determined fast enough by plaque assays to calculate the right MOI. Make sure you have at least 1 ml of phage supernatant left to analyze. Transfer the same number of phages used for infection into a phage buffer solution, especially if titer of inoculation phage is unknown.  

Grow the infected cultures for 1 to 24 hours, respective to your propagation results during the AP testing, at optimal growth conditions (for E. coli choose 37°C and about 220 rpm).  <B>Step 7: Phage supernatant</B>
After incubation time spin down at least 2 ml of cells at 6000 g for 3 min at room temperature to pellet your cells. Phages in the supernatant are now separated from the bacteria in the pellet. Store phage supernatant at 4°C. Discard pellets and the rest of the culture.  

From now on  <B>repeat steps 5 to 7</B>. After one round is completed you can pause the PREDCEL process and restart later with the stored supernatant.  

While PREDCELing, test PCRs with specific products for your SP should be performed as well as test plaque assays should be implemented to detect and prevent phage washout or phage contamination in your used stocks, media and bacterial cell cultures.   
If contamination or wash out is recognized, take the last sample proven to contain phages and which is free of contamination to start a new PREDCEL iteration round with a fresh culture.  <B>Step 8: Plaque assay</B>
Perform plaque assays of your PREDCEL samples including inoculation phage and culture samples from step 6 (see plaque assay protocol) to calculate phage titers and check for washout.  <B>Step 9: Sequencing</B>
To check your evolutionary progress, at last eight plaques should be picked to use them for insert amplification via PCR. Afterwards, sequencing of your PCR product can be implemented with insert specific sequencing primers. Mention that mutations taking place while PREDCEL may lead to inefficient primer annealing.  <B>Troubleshooting:</B>
As phage propagation rate and mutation efficiency usually varies between different gene circuits and phages, but also due to the achieved changes in activity within one PREDCEL run, you might have to adopt several parameters to achieve a lower selection pressure (if phages get lost over time), a higher selection pressure (only random mutations occur) or changes in mutation rate (too little/many mutations). These parameters are: <UL><LI> the amount of culture volume to be infected and the corresponding flask size </LI><LI>  the time of incubation after infection </LI><LI> the amount of phage supernatant transferred </LI></UL>


To check for the right parameters pre-experiments, need to be performed for an optimized propagation efficiency.  

You should also prepare several strains of your bacteria carrying APs with different RBS-strengths and origins of replication and different MPs (MP1/4/6 vary in mutation rate).  

If you have problems with phage propagation due to no or too little starting activity an initial drift phase without selection pressure might help in the beginning.  

Another helpful adaption is the usage of a helper-culture. This culture should be used between every or several PREDCEL AP-iteration rounds and must provide geneIII expression that is only coupled to phage infection. Thereby all phages that are transferred can reproduce to gain a high phage titer that might be needed to prevent phage washout. As previously phages showing higher activity propagated faster than those showing no or low activity they will, as a consequence, achieve higher phage titers while this helper-culture-phase. Thus, helper-culture phase should not redeem previously achieved selection for beneficial mutations.  

Note: Contamination in theory must not a priori lead to end of current evolutionary process as contamination phages should not be able to propagate on your AP-carrying host cells if it lacks geneIII.  Nevertheless, don't go on with PREDCEL when contaminated in order to gather valid data, especially regarding your phage titers. This counts even more if your AP-design might allow other non-wild-type phages to propagate.


    </DIV></DIV></DIV><DIV class="container"><DIV class="row"><DIV class="col-lg-12 col-md-12 col-xs-12 content" style="display: inline-block; padding-bottom: 10px; padding-top:10px;"><H1>Cloning Standard</H1>


The crucial step of to the cloning of PACE circuits is the generation of the accessory plasmid. These plasmids allow geneIII expression dependent on the evolving protein. The link between the fitness of the protein of interest and the expression of geneIII determines the effectiveness of the directed evolution and the presence of ProteinIII is essential for the production of new phage particles.
On the one hand, the initial expression of geneIII needs to be strong enough so that phages with the wildtype protein or negligible mutated protein are able to persist in the lagoon, and that they are not washed out. On the other hand, the expression of geneIII should neither be too high, because reproduction of the phage is only linked to it as long proteinIII is the limiting factor. Consequently, regulation of the geneIII expression influences the selection stringency of the directed evolution process.


For these reasons, it is a major challenge of PACE, to provide the right amount of geneIII. The amount of geneIII that is provided, can easily be regulated by changing the RBS and/or the origin of replication. In addition it is in many cases necessary to use more than one AP to vary selection pressure. Therefore, one must find a quick and easy way to modify APs.

If more than one variant of a circuit should be tested at a time, it is necessary to modify the activation region for geneIII and the additional gene that can be located on the AP with a minimum of effort. This is the another reason for a efficient cloning strategy.

To make AP cloning as simple as possible, we defined a new cloning standard that is specifically suited for the assembly of Aps, we wrote a BBF RFC that describes our concept in detail.

We subdivided the accessory plasmid in five subparts with different functionalities: The promoter for transcription of geneIII with associated regulation sequences (1), geneIII itself with an appropriate RBS (2), a fluorescent or luminescent reporter (3), the plasmid backbone (4) and a second expression cassette for additional genes that are needed for the circuit.
Our aim was it to make it possible that these different fragments can easily be assembled and recombined, to minimize the cloning effort. Therefore we created five standard homology regions that seperate the different subparts and enable for fast and efficient Gibson assembly.<DIV class="col-lg-12 col-md-12 col-xs-12"><DIV class="mdl-shadow--4dp" style="padding:20px; float:left}"><DIV class="img-caption title"><STRONG><B>
      
Figure 2: In our cloning standard, compatible building blocks are defined by specific functionalities. They are flanked by defined homology regions,
indicated by numbers, which are necessary for the assembly of the APs with the Gibson method. This
results in a highly customizable plasmid, composed of the desired origin of replication, an antibiotic
resistance (4-5), a bicistronic operon with geneIII (2-3)and the desired reporter (3-4), which can be
activated by any promoter (1-2)and a second expression cassette for additional genes that are necessary
for the respective circuit (1-5).
    </B></STRONG></DIV></DIV></DIV>


In the following section, our cloning strategy is described in detail.

<H2>Homology Regions</H2>


We defined five different homology regions (HR) that obey the following criteria: The sequences have a length of 32 bp which is long enough for efficient Gibson assembly. They do not contain the ATG start codon. Furthermore the different sequences have low similarity. Last, they exhibit no secondary structures at 50  °C according to Mfold (http://unafold.rna.albany.edu/?q=mfold). A table with the respective  sequences are shown in the table below. Furthermore every homology region starts with an thymine and ends with an adenine. This enables for the compatibility with RFC10 (see below).


<DIV class="col-lg-12 col-md-12 col-xs-12 content" style="padding-top: 20px; padding-bottom: 20px"><P class="content"><B><B>Table 1:</B> Standard homology regions for the use for accessory plasmid construction</B></P><DIV class="table-slot" style="overflow-x: auto"><TABLE class="table table-bordered mdl-shadow--4dp" xsscleaned="overflow-x: auto !important"><THEAD style="background-color: #005493 !important;"><TR><TH style="text-align:right">Homology Region</TH><TH>Sequence</TH></TR></THEAD><TBODY><TR><TD style="text-align:right">HR1</TD><TD>TACGTTTCGTTACAGAGCTCGCCAGTGGATAA</TD></TR><TR><TD style="text-align:right">HR2</TD><TD>TATCAGATCATCGTCGACCTACAGGTGCAGTA</TD></TR><TR><TD style="text-align:right">HR3</TD><TD>TGGTCGGTGCCTATCAACTCGAGTAGTACTAA</TD></TR><TR><TD style="text-align:right">HR4</TD><TD>TTGTCTGGAGCCAAGCCGCATTTGAAGTACCA</TD></TR><TR><TD style="text-align:right">HR5</TD><TD>TCAGTTCTCGTAATGCAGGGCCCAGAATTTCA</TD></TR></TBODY></TABLE></DIV></DIV><B>1. Promoter and Activation Region</B>
The promoter region is flanked by HR1 at the 5'-end and by HR2 at the 3'-end. The promoter is the key part of the PACE circuit. It links the fitness of the evolving protein to the expression of geneIII. Therefore it is important that efficient initiation of transcription is possible by a functional (evolved) protein, whereas nonfunctional proteins exhibit no activity. There are many possibilities to link the activate transcription by a specific protein that can thereby be evolved via PACE. The most obvious example are polymerases, like T7 polymerase that are directly responsible for transcription. Another method is the use of transcription factors that activate gene expression or other DNA binding proteins, which can for example be linked to the RNA polymerase omega subunit (rpoZ), which is able to recruit the transcription machinery. A completely different approach is the activation via riboswitches, which make translation possible after a small molecule has bound. This enable for the evolution of enzymes. For further examples, check out our approaches in the results section.
It is necessary that the activation works robust and even the unevolved protein is able to activate transcription to some extend. At the same time the complete promoter subpart, including both homology regions, should not be shorter than 200 bp to ensure efficient Gibson assembly.
<B>2. GeneIII</B>
GeneIII is the main component of the second subpart. As already decribed above, it is crucial to keep the amount of produced proteinIII in a range where it is proportional to phage production. Beside the copy number of the plasmid Its translation is regulated by a specific RBSs. The RBS and the coding sequence of geneIII are flanked by HR2 and HR3. It is important, that there is no terminator downstream of the gen, only a stop codon, because the reporter should be located on the same mRNA in a bicistronic manner. To make cloning as simple as possible, we submitted geneIII in combination with five different RBS'. The used ribosomal binding sites were published by Ringquist et al. 1992 <X-REF>RN140</X-REF>. These RBS span a range of two orders of magnitude and are therefore perfectly suited for tight regulation of geneIII expression.


<DIV class="col-lg-12 col-md-12 col-xs-12 content" style="padding-top: 20px; padding-bottom: 20px"><P class="content"><B><B>Table 1:</B> RBS’ used in combination with geneIII in the context of our project; The name of the RBS, the sequence, and the strength relative to SD8 are schown</B></P><DIV class="table-slot" style="overflow-x: auto"><TABLE class="table table-bordered mdl-shadow--4dp" xsscleaned="overflow-x: auto !important"><THEAD style="background-color: #005493 !important;"><TR><TH style="text-align:right">Name</TH><TH>Sequence</TH><TH>Relative Strength</TH></TR></THEAD><TBODY><TR><TD style="text-align:right">SD8</TD><TD>AAGGAGGAAAAAAAAA</TD><TD>1.00</TD></TR><TR><TD style="text-align:right">SD4</TD><TD>AAGGAGGAAAAA</TD><TD>0.51</TD></TR><TR><TD style="text-align:right">sd8</TD><TD>AAAGGAAAAAAAAA</TD><TD>0.20</TD></TR><TR><TD style="text-align:right">sd6</TD><TD>AAAAAGGAAAAAAA</TD><TD>0.13</TD></TR><TR><TD style="text-align:right">sd2</TD><TD>AAAAAAAAAGGAAA</TD><TD>0.01</TD></TR></TBODY></TABLE></DIV></DIV><B>3. Reporter</B>

Downstream of geneIII a reporter is placed to enable for monitoring of geneIII expression. We cloned a variety of promoters, ranging from fluorescent proteins, like YFP or RFP over liminescent reporters, like luxAB or nLuc to other enzymes like lacZ. Because some proteins are protected by patent, we could send only some of them to the registry. With different reporters, everyone can choose the reporter that suits his experiment setup best. The reporter subpart must be follewed by a stop-codon and a terminator. It needs is own RBS as well. The whole reporter unit should be flanked by HR3 and HR4 for conformity with our cloning standard.
<B>4. Plasmid Backbone</B>
The plasmid backbone consists of the origin of replication and an antibiotic resistance. It is bounded by HR4 and HR5. The copy number of the plasmid is the second important factor that regulates the copy number of APs. The more replica of a plasmid exist in a cell, the higher is the expression of the respective genes; geneIII in this case. Consequently, varying the copy number of the AP influences the amount of proteinIII and therefore the selection stringency. We used three different origins of replication in our expreiments: pBR322, which is the standard origin of replication of the registry plasmids and has a copy number of 15-20 copies per cell; p15A, which has only 10 copies per call and pSC101, a low copy origin with only 5 replica (see AddGene). In combination with the different RBS', thiy allows for highly regulated expression of the phage protein. It is important to check  the compatibility of the AP origin of replication with all the other plasmids, which need to be used in the experiments. 
Regarding the antibiotic resistance, ampicillin is the most probable variant. As large amounts of antibiotics are necessary for PACE, ampicillin is the modt attractive and commonly used alternative for APs. Nonetheless, different experiment setups may require different resistances, which is why we provide different resistance cassettes in the part, we provide on the registry. The backbone must be flanked by HR4 and HR5.
<B>5. Expression of other Proteins</B>
The majority of circuits needs more proteins, than geneIII and the evolving protein. As a consequence, our cloning standard provides a space for a second expression cassette in addition to geneIII for other circuit related proteins. There are many cases, in which other proteins may be needed for the PACE experiment. This could be virtually any protein that  is needed, for example chaperones for efficient folding of enzymes, proteins that interact with others, gRNA cassettes for CRISPR endonuclease evolution and many more. In this case, it is important to include the full expression cassette between HR1 and HR5, beginning with a promoter, which can be either constitutive or activatable, followed by a RBS and the coding sequence and finally finished by an appropriate terminator. If the termination is not perfect, the second expression cassette should be inserted in the opposite direction than the cassette of geneIII to avoid a secondary expression of geneIII.


<H2>Cloning of APs</H2>

Our plasmids can easily be assembled via Gibson assembly.  In summary, Gibson Aassembly makes use of an exonuclease, which cuts back the 5'-ends of the fragments. Subsequently, the overhangs anneal, gaps are filled up by the phusion polymerase (Thermo Fisher Scientific) and ligated by a taq-ligase. 
A protocol can be found <A href="https://2017.igem.org/Team:Heidelberg/Experiments#table_id-9" class="innerlink">here</A>.


<H2>Assembly of APs from Registry Parts</H2>

Many of the parts we tested in our experiments are available from the registry. Among them most of the standard units, which can be used in any in vivo evolution experiment, like geneIII, different backbones and some reporters. Furthermore, we provide parts, which are necessary for the circuits, we designed during iGEM. Obviously, all these sequences are offered as BioBrick parts in pSB1C3. Accordingly, we made our cloning method compatible with RFC10, the BioBrick standard (Figure 3). All subparts can be cloned into pSB1C3. They only have to be flanked by the respective homology regions and a BglII site, wherat the first or last base of the recognition site is included in the homology region. This is a key criteria, because it facilitates for fast and easy cloning.


<DIV class="col-lg-12 col-md-12 col-xs-12"><DIV class="mdl-shadow--4dp" style="padding:20px; float:left}"><DIV class="img-caption title"><STRONG><B><B>Figure 3:</B> Compatibility of our cloning stadard with the RFC10; Any AP building block can be cloned into RFC[10] standard by inserting BglII sites between the homology regions and the biobrick prefix or suffix, respectively. To use such a part for AP assembly, it has to be digested with BglII. The resulting fragment should be purified and can subsequently used for Gibson assembly with other parts.
    </B></STRONG></DIV><DIV class="img-caption subtitle">
      

If one wants to use such parts for plasmid assembly, they can be cut with BglII. The appropriate band has to be gelexed and can subsequently used for Gibson assembly. BglII creates 5'-overhangs, which are eliminated by the 5'-exonuclease that is used for Gibson assembly. Only 1 bp, a thymine at the 5'-end or an adenine at the 3'-end, remains. Is this base pair is included in the homology region, it does not interfere with Gibson Assembly. As a result, our standard is fully compatible with RFC10 and our parts from the registry can be included without problems.
        
    </DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV style="background-color: white; padding-top: 20px;"><DIV class="container"><DIV class="row"><DIV class="col-lg-12 col-md-12 col-xs-12 content" style="display: inline-block;"><DIV class="row content"><DIV class="col-lg-12 col-md-12 col-xs-12" style="padding-top: 20px; padding-bottom: 50px; display: block"><DIV class="mdl-shadow--4dp" style="padding: 20px"><H3> References </H3></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><SECTION id="footer-sec" style="background-color: #222 !important"><DIV class="container"><DIV class="row" style="padding-top: 30px;"><DIV class="col-md-3"><H4 style="text-align: center">Quote</H4><P><I>“”</I></P></DIV><DIV class="col-md-3"><H4 style="text-align: center"> Useful <STRONG>Links</STRONG></H4><P><A href="https://igem.org/Special:SpecialPages">
			Special pages
		</A></P><P><A href="https://igem.org/Main_Page">
			Main Page
		</A></P></DIV><DIV class="col-md-3"><H4 style="text-align: center"> Follow us on </H4><DIV style="text-align: center;"><SPAN style="color: grey; text-align: center;">&amp;</SPAN></DIV></DIV><DIV class="col-md-3"><H4 style="text-align: center"> Contact us </H4><P>

               iGEM-Heidelberg2017@bioquant.uni-heidelberg.de
               
		 Im Neuenheimer Feld 267
		
		69120 Heidelberg
                    </P></DIV></DIV></DIV></SECTION></DIV></DIV></DIV></DIV></BODY></HTML>