Team:Heidelberg/Toolbox
NAVIGATION
People
Team
Attributions
Sponsoring
For Judges
Giant Jamboree
Project
Overview
Background & Design
PACE
PREDCEL
Results Summary
Software Validation
Organosilicons
Protein Interaction
Cytochrome Engineering
Optogenetics
CRISPR Cas9
InterLab Study
Achievements
Parts
Overview
Basic Parts
Composite Parts
Improved Part
Parts Collection
Part List
RFC
Modeling
Overview
Phage Titer
Mutagenesis Induction
Mutation Rate Estimation
Lagoon Contamination
Medium Consumption
Interactive Web Tools
Software
AiGEM Overview
DeeProtein
GAIA
SafetyNet
MAWS 2.0
Human Practices
Overview
Integrated Human Practices
Expert Interviews
Safety & Security
Public Engagement
Education
Collaborations
Notebook
Overview
Methods
Materials
Notebook Database
Toolbox Guide
iGEM TEAM HEIDELBERG 2017
COLLAPSE NAVIGATION
People
Team
Attributions
Sponsoring
For Judges
Giant Jamboree
Project
Overview
Background & Design
PACE
PREDCEL
Results Summary
Software Validation
Organosilicons
Protein Interaction
Cytochrome Engineering
Optogenetics
CRISPR Cas9
InterLab Study
Achievements
Parts
Overview
Basic Parts
Composite Parts
Improved Part
Parts Collection
Part List
RFC
Modeling
Overview
Phage Titer
Mutagenesis Induction
Mutation Rate Estimation
Lagoon Contamination
Medium Consumption
Interactive Web Tools
Software
AiGEM Overview
DeeProtein
GAIA
SafetyNet
MAWS 2.0
Human Practices
Overview
Integrated Human Practices
Expert Interviews
Safety & Security
Public Engagement
Education
Collaborations
Notebook
Overview
Methods
Materials
Notebook Database
Toolbox Guide
Welcome to our
Toolbox Guide!
To enable rapid design and simple cloning of APs, we created a software tool, so that cloning can easily                     be planned. All parts that can be chosen are available in pSB1C3 backbones from the registry. Just follow                     the instructions and create your own fully customized AP.
Ready to PREDCEL?
See our Questionnaire here
Have you already chosen a protein to be evolved?
Yes
No
You already accomplished the first and most important step. Let’s see whether                                                 you are ready to evolve it!
No problem, there are lots of proteins waiting to be evolved. Go on to find                                                 out what you should take care of when choosing your target protein and preparing your experiments.                                                  Maybe you also want to have a look at our different projects to get an idea of the wide range of proteins you can evolve with PREDCEL .
Which application will your evolved protein be used for?
Medical application
Chemical synthesis
Agriculture
Could be helpful somehow
Other scientific interests
Did you discuss your idea with a medical professional? We are sure this expertise                                                 would be valuable to choose the right experimental settings.
Did you discuss your idea with a chemist? This expertise could be helpful to figure out how to set up the evolution efficiently, as parameters such as passive diffusion of the chemical through the E. coli cell wall, toxicity and many others needs to be carefully considered.
Be aware that a number of countries have strict regulations regarding green biotechnology.
That sounds quite uncertain. Before you start or pursue evolving your protein,                                                 you should know the exact purpose and impact. How about engaging with your                                                 local community to figure out possible needs?
Engage with the community in the field of in vivo evolution – we´d be interested                                                 in discussing your project.
Are you sure your protein is safe to be evolved?
Yes it is safe!
How can I know?
Nevertheless we advise you to check it in our SafetyNet, there could be “sleeping”                                                 hazardous potential.
Please, check your protein sequence for any “sleeping” hazardous potential in our SafetyNet
Do you know which equipment is required to safely work with phages?
Yes
Not sure
No
Did your considerations include that phages are very volatile? You should create                                                 a special “phage space” in your lab to reduce contamination. Furthermore,                                                 you should comply with the safety level of your evolving protein. Check the                                                 safety level of your protein on our SafetyNet
Check our safety page for general advice and get in touch with your safety representative for detailed discussion of the particular work you plan.
Before starting any phage work, please consult our safety page . Also, we advise you to talk to your colleagues next door with experience in phage work as well as your local safety representative.
Did you calculate all required parameters on our Integrated Modeling page?
Yes
Not yet
I don´t intend to
Well done! Now you have the optimal starting conditions for your experiment.
Visit our Integrated Modeling page to figure out all required parameters for a successful PREDCEL run.
You should consider that our Integrated Modeling reduces effort, costs and environmental burden by enabling in silico optimization of experimental conditions.
Did you already engage with your local community to discuss your plans?
Yes
Not yet
I don´t intend to
Fine. For more ideas, how to engage with the public you could watch the expert interview with Dr. Kaufmann
You should reach out and listen to the feedback, hopes and concerns of your local community. Thereby, you can facilitate a positive, beneficial impact of your research project.
Are you sure? We encourage you to do so and be surprised, how enriching it is to discuss your research plans with people inside and outside the research community. Also consider, that your research may very well impact the people around you and hence, they should have a word and vote in what our technologies are used for. Finally, showing that you are a responsible scientists caring about the consequences of your research will surely have a positive impact on your reputation.
Did you consider possible consequences of your planned PREDCEL experiment?
Yes
Not yet
I don´t intend to
You are a responsible scientist. We encourage you to use our PREDCEL toolbox and convey the iGEM spirit of responsible science to your friends and colleagues.
Take your time to reflect on possible consequences to prevent unintended dangers. This is an essential step to be taken before starting directed evolution experiments.
Are you sure? We encourage you to talk to experts and your local community. You will be surprised how much they will appreciate your offer to discuss the consequences of your research BEFORE implementing it in the lab. Also, consider the positive impact for yourself: Showing that you are a responsible scientists caring about the consequences of your research will surely have a positive impact on your reputation.
Now it´s on you!
When you talk about PREDCEL with your colleagues, friends and local communities, you can spread the idea of making directed evolution experiments safe and to use this technology responsibly for addressing urgent human needs.
1. Optimize parental sequence / gene pool
With the AiGEM (Artificial Intelligence for Genetic Evolution Mimicking) software suite we provide powerful software                 tools for in silico evolution. Applying our genetic algorithm, GAIA , we fully in silico evolved a \(\beta\)-glucuronidase towards \(\beta\)-galactosidase function and derived \(\beta\)-lactamase variants exceeding wildtype activity .
We thus strongly recommend the application of GAIA prior to in vivo evolution. Through optimization of the starting sequence and the acompanying reduction of the                 required library size the time until phage pool convergence can be tremendously reduced.
GAIA is written in python and readily available on our github repo .
2. Get phage
Phage-assisted continous evolution is based on the generation of selection phages, which have geneIII replaced by the gene                 of interest. For the generation of your own selection phage, we are offering a golden-gate cloning standard                 being based on the extraordinary potential of the type IIs restriction enzyme BsaI. For insertion of your                 gene of interest into the phage backbone, amplification with primers containing the following overhangs is                 required:
Overhangs
Fwd:
GAGGTCTCGAAAA
Rev:
CAGGTCTCGGTGC
Following the successfully performed amplification with these primers, a golden gate reaction (refering to the golden                 gate assembly protocol) with the BioBricks BBa_K2398021 and BBa_K2398022 should be performed. GoldenGate                 cloning usually delivers good results. In the following of this reaction, the assembled plasmid should be                 transformed into electrocompetent S1059 or S2208 cells. These strains, established by the liu lab, carry                 a plasmid (pJC175e) that provides geneIII under a psp-promoter. This promoter gets activated by phage infection,                 and ensures that geneIII is not present in the uninfected cell. For further information about the principle                 of phage propagation, have a look at our RFC . As an alternative to the assembly with                 two phage backbone fragments, the golden gate assembly reaction can be also performed using only the BBa_K2398023                 BioBrick. In this case, amplification of the gene of interest has to be implemented with other overhangs:
Overhangs
Fwd:
GTGGTCTCAATGG
Rev:
TTGGTCTCAGTTG
If you want to produce your selection phage on your own by using another cloning method, please keep in mind that                 the terminator of geneVIII may be immediately followed by the RBS of the evolving gene. Futhermore, the last                 180 bp of geneIII should not be removed, because they contain the promoter for the following operon of phage                 genes. In that case, we provide the surrunding sequences of the gene of interest below:
Additional                 Sequences
Upstream sequence
AAAGGCTCCTTTTGGAGCCTTTTTTTT
Downstream sequence
CTCCCTCAATCGGTTGAATGTCGCCCTTTTGTCTTT
3. Add selection pressure
For any method of directed evolution to culminate in an improved protein, a means of applying selective pressure is needed.                 Thus, adding components capable of enforcing selection in PACE and PREDCEL is one of the most crucial steps                 in construct design. For directed evolution experiments selective pressure may need to be fine-tuned by using                 different ORIs, promoters, ribosome binding sites and backbones. In this section of the toolbox guide, all                 those components will be set up for you to suit your particular project. To get an idea of how selection                 could be enforced in your projects, take a look at our examples:
MAWS 2.0 - in silico riboswitch design for selective pressure.
A             simple and sure-fire way to couple gene expression to a product of enzyme catalysis is to introduce a riboswitch             for that particular product. This is a short RNA sequence either trapping or releasing the ribosome binding site             in the presence of a small molecule. Back in 2015 the Heidelberg iGEM Team developed a software suite (MAWS -             making aptamers without SELEX) to predict such riboswitches for arbitrary small molecule targets. This year we             use and provide new-and-improved distribution of MAWS. We predict and validate synthetic riboswitches generated             using MAWS 2.0 to couple gene expression to enzyme activity. This establishes riboswitches predicted in             silico as a valuable component of any directed evolution project seeking to evolve novel biocatalytic properties.
MAWS 2.0
OptoSELECT - Light-dependent selective pressure
The generation of proteins with radically altered or highly specific new activities is a major goal in the field             of directed evolution. Modulation of selection stringency is indispensable for the optimization of proteins,             which in their native form exhibit a lack of activity for the required action. To further extend the possibilities             of PACE/PREDCEL based protein optimization and to evolve novel activities without the use of intermediate evolutionary             steps, we provide an optogenetic modulator of selection stringency. The variation of stringency is based on the             blue light-dependent transcription factor EL222 and a bidirectional promoter system that can induce or repress             the expression of geneIII upon blue light irradiation in a non-toxic, rapidly delivered, and reversible manner.
OptoSELECT
Accessory Plasmid Generator
Plasmid
1. First, choose your preferred backbone from the pull down menu. We provide three different origins of replication                     that differ in their copy number and therefore affect the selection stringency, your AP will have. Be                     aware that the origin of replication should be compatible to the origin of replication of all other plasmids                     you plan to use. In most of the cases it is best to combine your origin of replication with an ampicillin                     resistance, but we provide other antibiotic resistances for special applications as well.
2. Second, decide which RBS you need for control of geneIII translation. We offer six different RBS with                     different strengths. For further information take a look at our RFC
3. Next, decide, which reporter you would like to choose. A set of different fluorescent and luminescent                     reporters is available
Resistance & Ori
RBS + geneIII
RBS + Reporter
Activator & Promoter
4. When all standard parts are defined, the promoter and associated regulatory sequences must be set. Decide                     between the given options. Enter either an own sequence (as text or upload the file) or use one of the                     provided standard promoters that were already used by our team. Notice that the length of this part should                     not be below 200 bp. Ensure, that your sequence does not include a RBS.
Overlaps are included
Overlaps are included
Additional sequence
5. Finally, decide which additional gene you would like to express on your AP. Most circuits that are used                     in PACE, need at least one additional protein, like split-proteins, chaperone or interacting factors.                     Again, it is possible to enter the desired sequence or choose one of the provided genes. Make sure, you                     enter a whole expression cassette, with promter, RBS, CDS and terminator.
Overlaps are included
Overlaps are included
Deploy the Plasmid
Deploy your designed Plasmid and remember to PACE or PREDCEL responsibly...
Create Plasmid
Instructions
4. Induce mutagenesis
After the generation of your AP and your SP, we will focus on the third plasmid, playing a highly important role                 in both PACE and PREDCEL. The mutagenesis plasmid is responsible for the library preparation of the different                 selection phage variants. If you have transformed your AP into a bacterial strain, suitable to be used in                 PACE or PREDCEL, one of three possible mutagenesis plasmids MP1 , MP4 and MP6 should be transformed into the strain as well. Those MPs are differing in their mutation rate. For                 further information on the MP´s funcitonality please have a look at the literature
badran2015development
. For the induction of the mutagenesis, arabinose should be added into the culture,                 activating the pBAD promotor and thereby the mutagenesis. For all PREDCEL experiments a concentration of                 \(100\,mM\) arabinose is used, while \(2\,mL/h\) of \(10\%\;w/v\) arabinose solution are added in a \(100\,mL\)                 lagoon in PACE.
5. Optimize conditions
PACE and PREDCEL require well chosen conditions to yield the best results. Use our Interactive Webtools to calculate parameters like Glucose Concentration , Arabinose Concentration and Medium Concumption online, or obtain our phage titer Model and use it to reduce the risk of washout.
When trying new conditions for a PACE experiment check if the setup is vulnerable to contaminations To make monitoring the experiment by sequencing                     less expensive and to save you time, we provide a tool that calculates the number of sequences that needs to be sequenced to find mutations with a                     certain probability.
Quote
“”
Useful Links
Special pages
Main Page
Follow us on
&
Contact us
iGEM-Heidelberg2017
@bioquant.uni-heidelberg.de
Im Neuenheimer Feld 267
69120 Heidelberg
