<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Heidelberg_Collaborations skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Heidelberg/Collaborations</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container"><DIV class="navbar-header"><A type="button" data-canvas="body" id="sidebarCollapse" class="navbar-toggle" data-toggle="offcanvas" data-target="#sidebar" style="color: #393939;margin-top: 9px;">NAVIGATION
                    </A></DIV><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav pull-right"><LI class="dropdown"><A href="https://2017.igem.org/Team:Heidelberg/Team">People </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Team">Team</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Attributions">Attributions</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Sponsoring">Sponsoring</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/For_Judges">For Judges</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Jamboree">Giant Jamboree</A></LI></UL><LI class="dropdown"><A href="https://2017.igem.org/Team:Heidelberg/Description">Project </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Description">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Design">Background &amp; Design </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Pace">PACE</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Predcel">PREDCEL</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Results">Results Summary</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Validation">Software Validation</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Organosilicons">Organosilicons</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Protein_Interaction">Protein Interaction</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Cytochrome_Engineering">Cytochrome Engineering</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Optogenetics">Optogenetics</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/CRISPR">CRISPR Cas9</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/InterLab">InterLab Study</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Achievements">Achievements</A></LI><LI class="dropdown"><A href="https://2017.igem.org/Team:Heidelberg/Parts">Parts </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Basic_Part">Basic Parts</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Composite_Part">Composite Parts</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Improved_Part">Improved Part</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Part_Collection">Parts Collection</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Part_List">Part List</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/RFC">RFC</A></LI><LI class="dropdown"><A href="https://2017.igem.org/Team:Heidelberg/Model">Modeling </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Phage_Titer">Phage Titer</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Mutagenesis_Induction">Mutagenesis Induction</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Mutation_Rate_Estimation">Mutation Rate Estimation</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Lagoon_Contamination">Lagoon Contamination</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Medium_Consumption">Medium Consumption</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Tools">Interactive Web Tools</A></LI><LI class="dropdown"><A href="https://2017.igem.org/Team:Heidelberg/Software">Software </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software">AiGEM Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software/DeeProtein">DeeProtein</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software/GAIA">GAIA</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software/SafetyNet">SafetyNet</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software/MAWS">MAWS 2.0</A></LI><LI class="dropdown"><A href="https://2017.igem.org/Team:Heidelberg/Human_Practices">Human Practices </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Human_Practices">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/HP/Gold_Integrated">Integrated Human Practices</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Interviews">Expert Interviews</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Safety">Safety &amp; Security</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Engagement">Public Engagement</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Education">Education</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Collaborations">Collaborations</A></LI><LI class="dropdown"><A href="https://2017.igem.org/Team:Heidelberg/Notebook">Notebook </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Notebook">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Experiments">Methods</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Materials">Materials</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Database">Notebook Database</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Toolbox">Toolbox Guide</A></LI></DIV></DIV><NAV id="sidebar" style="z-index: 10001; background-color: white;border: none;box-shadow:1px 0px 0px #393939;" class="navmenu navmenu-default navmenu-fixed-left offcanvas" role="navigation"><A class="navmenu-brand" href="https://2017.igem.org/Team:Heidelberg"> iGEM TEAM HEIDELBERG 2017</A><DIV id="sidebarHide" data-toggle="offcanvas" data-target="#sidebar" style="padding-left: 20px;"><A style="font-family: 'Josefin Sans' !important; font-size: 16px; line-height: 18px; height: 18px;"> COLLAPSE NAVIGATION </A></DIV><UL class="nav navmenu-nav" id="sidenav"><LI><A data-toggle="collapse" data-target="#People_side">People </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Team">Team</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Attributions">Attributions</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Sponsoring">Sponsoring</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/For_Judges">For Judges</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Jamboree">Giant Jamboree</A></LI></UL><LI><A data-toggle="collapse" data-target="#Project_side">Project </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Description">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Design">Background &amp; Design </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Pace">PACE</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Predcel">PREDCEL</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Results">Results Summary</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Validation">Software Validation</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Organosilicons">Organosilicons</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Protein_Interaction">Protein Interaction</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Cytochrome_Engineering">Cytochrome Engineering</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Optogenetics">Optogenetics</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/CRISPR">CRISPR Cas9</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/InterLab">InterLab Study</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Achievements">Achievements</A></LI><LI><A data-toggle="collapse" data-target="#Parts_side">Parts </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Basic_Part">Basic Parts</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Composite_Part">Composite Parts</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Improved_Part">Improved Part</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Part_Collection">Parts Collection</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Part_List">Part List</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/RFC">RFC</A></LI><LI><A data-toggle="collapse" data-target="#Model_side">Modeling </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Phage_Titer">Phage Titer</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Mutagenesis_Induction">Mutagenesis Induction</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Mutation_Rate_Estimation">Mutation Rate Estimation</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Lagoon_Contamination">Lagoon Contamination</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Medium_Consumption">Medium Consumption</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Tools">Interactive Web Tools</A></LI><LI><A data-toggle="collapse" data-target="#AI_side">Software </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software">AiGEM Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software/DeeProtein">DeeProtein</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software/GAIA">GAIA</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software/SafetyNet">SafetyNet</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software/MAWS">MAWS 2.0</A></LI><LI><A data-toggle="collapse" data-target="#HP_side">Human Practices </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Human_Practices">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/HP/Gold_Integrated">Integrated Human Practices</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Interviews">Expert Interviews</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Safety">Safety &amp; Security</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Engagement">Public Engagement</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Education">Education</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Collaborations">Collaborations</A></LI><LI><A data-toggle="collapse" data-target="#Notebook_side">Notebook </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Notebook">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Experiments">Methods</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Materials">Materials</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Database">Notebook Database</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Toolbox">Toolbox Guide</A></LI></NAV><DIV class="content"><DIV style="background-color: white !important; padding: 0 !important; margin: 0 !important;"><DIV style="width: 100%; margin: 0; padding: 0; background-color: white !important;"><DIV class="container" style="position: relative; left: 0px; border-left: 5px solid #393939; border-bottom: 5px solid #393939; padding: 0 !important"><DIV style="margin-top: 0 !important; margin-bottom: 1em !important; padding-bottom: 1em !important; align-items: center;"><DIV style="margin-left: auto; margin-right: auto;"><DIV class="container"><DIV class="row" style="display: -webkit-flex; display: flex; -webkit-flex-wrap: wrap; flex-wrap: wrap; align-content: center; justify-content: center;"><DIV class="col-lg-6 col-md-6 col-sm-12 col-xs-12" style="width: 410px; height: 600px;"><DIV style="position: absolute; background-color: white !important;color: white !important; width: 400px; top: 100px; border-radius: 8px; border: none; padding: 15px 15px 15px 15px; display: flex; align-items: space-around; flex-wrap: wrap; display: -webkit-flex; -webkit-flex-wrap: wrap;" id="show"><P style="font-size: 20px; font-weight: bold; font-family: 'Josefin Sans'; flex-basis: 100%;"><A href="https://2017.igem.org/Team:Heidelberg/Collaborations#collaboration1" class="heidelberg-red">&gt; Mutagenesis Plasmid Inter-Lab Study</A></P><P style="font-size: 20px; font-weight: bold; font-family: 'Josefin Sans'; flex-basis: 100%;"><A href="https://2017.igem.org/Team:Heidelberg/Collaborations#collaboration2" class="heidelberg-orange">&gt; Cloning for iGEM Freiburg</A></P><P style="font-size: 20px; font-weight: bold; font-family: 'Josefin Sans'; flex-basis: 100%;"><A href="https://2017.igem.org/Team:Heidelberg/Collaborations#collaboration3" class="heidelberg-blue">&gt; PCR First Aid Service</A></P><P style="font-size: 20px; font-weight: bold; font-family: 'Josefin Sans'; flex-basis: 100%;"><A href="https://2017.igem.org/Team:Heidelberg/Collaborations#collaboration4" class="heidelberg-green">&gt; iGEM goes green</A></P><P style="font-size: 20px; font-weight: bold; font-family: 'Josefin Sans'; flex-basis: 100%;"><A href="https://2017.igem.org/Team:Heidelberg/Collaborations#collaboration5" class="heidelberg-gray">&gt; Mentoring Stuttgart</A></P><P style="font-size: 20px; font-weight: bold; font-family: 'Josefin Sans'; flex-basis: 100%;"><A href="https://2017.igem.org/Team:Heidelberg/Collaborations#collaboration6" class="heidelberg-red">&gt; Translation Center</A></P><P style="font-size: 20px; font-weight: bold; font-family: 'Josefin Sans'; flex-basis: 100%;"><A href="https://2017.igem.org/Team:Heidelberg/Collaborations#collaboration7" class="heidelberg-orange">&gt; Postcards</A></P><P style="font-size: 20px; font-weight: bold; font-family: 'Josefin Sans'; flex-basis: 100%;"><A href="https://2017.igem.org/Team:Heidelberg/Collaborations#collaboration8" class="heidelberg-green">&gt; No Science without Tolerance</A></P><P style="font-size: 20px; font-weight: bold; font-family: 'Josefin Sans'; flex-basis: 100%;"><A href="https://2017.igem.org/Team:Heidelberg/Collaborations#collaboration9" class="heidelberg-blue">&gt; Surveys -
 Development within iGEM</A></P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="page-title" style="background-color: white !important;"><DIV class="container" style=" padding: 0 !important;"><DIV class="col-lg-12 col-md-12 col-xs-12" style="padding-right: 0 !important; padding-bottom: 0 !important; position: relative; left: 0px;"><DIV style="position: relative; top: -10px; border-right: 5px solid #393939; padding: 40px 20px 20px 0px; margin-bottom: -30px;"><DIV style="overflow-x: ellipsis;" class="header-title heidelberg-red"> Collaborations </DIV></DIV></DIV></DIV></DIV><DIV class="container" style="background-color: white !important; padding: 0 !important; margin-top: 2em !important;"><DIV class="col-lg-12 col-md-12 col-xs-12" style="padding: 0 !important; margin: 0px important!;"><DIV class="container"><DIV class="row"><DIV class="col-lg-12 col-md-12 col-xs-12 content" style="display: inline-block; padding-bottom: 10px; padding-top:10px;"><DIV class="col-lg-12 col-md-12 col-xs-12"><DIV class="mdl-shadow--4dp" style="padding:20px; float:left; display: inline-block;"><H2 class="heidelberg-red">Mutagenesis Plasmid Inter-Lab Study</H2>



In our directed evolution approaches PREDCEL and PACE, we select for beneficial mutations in a protein of interest encoded on a M13 phage genome. In order to generate a pool of protein mutants to select from in the first place, the efficient introduction of mutations during phage genome replication is essential. Mutagenesis-inducing plasmids (MPs) that enhance the mutation rate in <I>E. coli</I> by inhibiting DNA repair mechanisms have been described <X-REF>badran2015development</X-REF>. When transformed into <I>E. coli</I> hosts, these plasmids should cause mutation rates several orders of magnitude higher than usually expected under laboratory conditions. 
The exact mutation rate induced by different mutagenesis plasmids could, however, vary between laboratories, as <I>E. coli</I> growth conditions and hence expression strength of the mutagenic proteins (e.g. error-prone polymerase subunits) could differ. However, robust induction of high mutation rates are critical for the success of PREDCEL and PACE and hence of major importance for other teams to reproduce our directed evolution methods. Therefore, we performed a small inter-lab study to evaluate the performance of different mutagenesis inducing plasmids and find the construct setup most robust and hence suitable for future iGEM teams to use for in vivo directed evolution.  To enable an inter-lab comparison, we developed a standardized mutagenesis plasmid test kit and assay and distributed it to iGEM team <A href="https://2017.igem.org/Team:BOKU-Vienna/Collaborations">BOKU Vienna</A>, <A href="https://2017.igem.org/Team:ETH_Zurich/Collaborations">ETH Zurich</A>, <A href="https://2017.igem.org/Team:Freiburg/Collaborations">Freiburg</A> and  <A href="https://2017.igem.org/Team:Stuttgart/Collaborations">Stuttgart</A>. We are very grateful for their kind support and excited to share the <A href="https://2017.igem.org/Team:Heidelberg/Collaborations#InterLab_PP">results</A> from this small inter lab study with all iGEM teams 
<B>Mutagenesis Assay Kit</B>
We shipped 3 mutagenesis plasmids (#1-3). All mutagenesis inducing plasmids contain an arabinose inducible promoter P<SUB>BAD</SUB>. Upstream of P<SUB>BAD</SUB> the araC protein is encoded in the opposite direction and regulates the activity of the P<SUB>BAD</SUB> promoter. The expression cassette downstream P<SUB>BAD</SUB> comprises multiple, different mutagenesis supporting elements, e.g. error-prone polymerase subunits. According to literature <X-REF>RN46</X-REF> the mutagenesis plasmids #1-3 were expected to cause increasingly high mutation rates.
<B>Mutagenesis Assay – Spontaneous Resistance Acquisition</B>
Due to exceptionally high mutagenesis levels of <I>E. coli</I> cells transformed with one of the mutagenesis plasmids, these cells can more quickly adopt to environmental changes. Hence, one way to measure the mutagenesis levels is simply to quantify the level of spontaneous antibiotic resistance acquisition. Therefore, <I>E. coli</I> were transformed with mutagenesis plasmids or remain untransformed (as control) followed by incubation in presence or absence of different antibiotics on agar plates. After 18-21 hours of incubation at 37 °C, colonies are counted. A higher number of colonies (i.e. clones with spontaneous resistance acquired) thereby indicates a higher mutation rate in the corresponding <I>E. coli</I> population due to the presence of the mutagenesis plasmid.
<B>Results from the Study</B>
To decide which mutagenesis plasmid to recommend to future iGEM teams for in vivo directed evolution approaches such as PREDCEL, it was essential to determine the mutagenesis plasmid setup most robustly functioning across different laboratories. The <A href="https://2017.igem.org/Team:Heidelberg/Collaborations#InterLab_PP">results</A> of iGEM team BOKU Vienna, ETH Zürich, Freiburg and Stuttgart indicate that MP #1 induces mutations most reliably .
Hence, we recommend to use mutagenesis plasmid #1 for PREDCEL experiments, which is also noted in our corresponding <A href="https://2017.igem.org/Team:Heidelberg/RFC">RFC</A>.
</DIV></DIV><DIV class="col-lg-12 col-md-12 col-xs-12"><DIV class="mdl-shadow--4dp" style="padding:20px; float:left; display: inline-block;"><H2 class="heidelberg-orange">Cloning a Positive Control for Freiburg</H2>
Collaborating with the <A href="https://2017.igem.org/Team:Freiburg"> iGEM Team Freiburg </A> we cloned a positive control for a subproject of their CARTEL<SUP>™</SUP> project.

Under normoxic conditions HIF1A cannot interact with HIF1B to form the transcription factor HIF1, because its hydroxylated and marked by the E3 ubiquitin ligase for degradation by the proteasome <X-REF>ziello2007hypoxia</X-REF>. 
Therefore the elimination of the two hydroxylated amino acids, that are prolines at the positions 402 and 564, would generate a HIF1A, that is stabilized and interacts with HIF1B under normoxic conditions <X-REF>yasui1988expression</X-REF> to activate the hypoxia response element (HRE).

Because we wanted to collaborate with this year’s <A href="https://2017.igem.org/Team:Freiburg"> iGEM Team Freiburg </A>, we offered to help them with our  Golden Gate cloning expertise to perform a site directed mutagenesis and generate the constitutively active HIF1A. It would be used Freiburg as a positive control for CoCl2 induction in analysis of HRE. 

For the aforementioned two amino acid exchanges alanine to proline we decided to utilize PCR amplification followed by Golden Gate cloning for site directed mutagenesis. To insert the two point mutations three steps were performed. We divided the ~7.9 kb plasmid into three fragments by (i) PCR amplification. The primes were designed with Golden Gate overhangs and BsmBI restriction sites. 


<DIV class="col-lg-12 col-md-12 col-xs-12 content" style="padding-top: 20px; padding-bottom: 20px"><P class="content"><B>
   Table 2: Primer sequences for site directed mutagenesis in HIF1-alpha</B></P><DIV class="table-slot" style="overflow-x: auto"><TABLE class="table table-bordered mdl-shadow--4dp" xsscleaned="overflow-x: scroll !important"><THEAD><TR><TH>Primer position</TH><TH>Position 402</TH><TH>Position 564</TH></TR></THEAD><TBODY><TR><TD>forward</TD><TD>TTCGTCTCAGCAGCCGCTGGAGACACAATCATATCTTTAGATTTTG</TD><TD>TTCGTCTCAGCCTATATCCCAATGGATGATGACTTCCAGTTACGTTC</TD></TR><TR><TD>reverse</TD><TD>TTCGTCTCACTGCGGCCAGCAAAGTTAAAGCATCAGGTTCCTTC</TD><TD>TTCGTCTCAAGGCAGCTAACATCTCCAAGTCTAAATCTGTGTCCTGAGTAG</TD></TR></TBODY></TABLE></DIV></DIV>




Primers to split the backbone into two fragments in the ampicillin resistance ensured that plasmids with re-ligated fragments would not provide ampicillin resistance to the E. coli cells.



<DIV class="col-lg-12 col-md-12 col-xs-12 content" style="padding-top: 20px; padding-bottom: 20px"><P class="content"><B>
   Table 3: Primers for backbone splitting</B></P><DIV class="table-slot" style="overflow-x: auto"><TABLE class="table table-bordered mdl-shadow--4dp" xsscleaned="overflow-x: scroll !important"><THEAD><TR><TH>Primer position</TH><TH>Ampicillin resistance</TH></TR></THEAD><TBODY><TR><TD>forward</TD><TD>TTCGTCTCAATCTTCAGCATCTTTTACTTTCACCAGCGTTTCTG</TD></TR><TR><TD>reverse</TD><TD>TTCGTCTCAAGATCAGTTGGGTGCACGAGTGGGTTAC</TD></TR></TBODY></TABLE></DIV></DIV>



The extension PCR was performed following the standard PCR protocol for Q5 polymerase. In a second step an agarose gel proved that all amplifications worked out well and after gel extraction (ii) Golden Gate assembly was performed following our <A href="https://2017.igem.org/Team:Heidelberg/Experiments">standard Golden Gate protocol</A>. 

The plasmid was (iii) transformed into chemically competent TOP10 strain <I>E.coli</I> cells and plated on LB agar plates supplemented with ampicillin  to select successfully <A href="https://2017.igem.org/Team:Heidelberg/Experiments">transformed E. coli cellsl</A>transformed <I>E. coli</I> cells.
To evaluate the assembly and test whether the point mutations were successfully inserted, the sequence of HIF1-alpha containing the point mutations as well as the regulatory element (CMV) promoter were verified by Sanger Sequencing. The sequencing results were positive and therefore, we are happy to have helped the iGEM Team Freiburg with the cloning of a constitutive HIF1-alpha. 




  </DIV></DIV><DIV class="col-lg-12 col-md-12 col-xs-12"><DIV class="mdl-shadow--4dp" style="padding:20px; float:left; display: inline-block;"><H2 class="heidelberg-blue">PCR First Aid Service</H2>


As we had several PCR problems ourselves in the beginning of our project, we asked our advisors for help and created a PCR troubleshooting protocol, which you can find on our <A href="https://2017.igem.org/Team:Heidelberg/Experiments">protocol section</A>.  Because we considered it to be helpful for other teams as well, we decided to offer help for all other iGEM teams: our <A href="https://2017.igem.org/Community/Collaborations">First Aid Service for PCR Problems</A>. Happily we could help some teams overcoming their PCR problem.
<B>Helping iGEM Hamburg with their PCR problems</B>
Since the beginning of their project the <A href="https://2017.igem.org/Team:Hamburg/Collaborations">iGEM team Hamburg</A> experienced difficulties performing gene amplifications. After tedious trial and error using different gradient-PCRs, experimenting with GC-enhancer solutions and DMSO concentrations we figured out a way to amplify eleven out of their twelve genes and gene fragments. The only one that did not work despite their efforts was the AraC family transcriptional regulator.
They noticed our PCR First Aid Service and after a skype call in which Hamburg described their problems and attempts we gave them valuable tips. Our ideas were a raise of the annealing temperature to 72 °C, adding 0.5 M betaine, try adding DMSO in 3 %, 5 % or even 10 % concentration or a combination of both DMSO and betaine.
Hamburg gave a try to all these suggestions, unfortunately without having a hit.


We discussed the result with the iGEM Team Hamburg becuase we did not want to give up an gave additional tips to try. Our next suggestion was to design new primes, as ours had a very high GC content in the middle part and were therefore prone to the formation of secondary structures. Also, the new primer pairs should be designed to anneal at 72 °C. They offered to design to take over the design for us.
These were the original primers:
Fw: CATTATGAATTCGCGGCCGCTTCTAGA
Rv: GATAATCTGCAGCGGCCGCTACTAGTA

        

<DIV class="col-lg-12 col-md-12 col-xs-12 content" style="padding-top: 20px; padding-bottom: 20px"><P class="content"><B>
   Table 4: Original primers designed by Hamburg</B></P><DIV class="table-slot" style="overflow-x: auto"><TABLE class="table table-bordered mdl-shadow--4dp" xsscleaned="overflow-x: scroll !important"><THEAD><TR><TH>Orientation</TH><TH>Sequence</TH></TR></THEAD><TBODY><TR><TD>forward</TD><TD>CATTATGAATTCGCGGCCGCTTCTAGA</TD></TR><TR><TD>reverse</TD><TD>GATAATCTGCAGCGGCCGCTACTAGTA</TD></TR></TBODY></TABLE></DIV></DIV>
        
        
        
We designed new primers with lower tendency for secondary structure formation:
        
        

<DIV class="col-lg-12 col-md-12 col-xs-12 content" style="padding-top: 20px; padding-bottom: 20px"><P class="content"><B>
   Table 5: Improved primer sequences designed by Heidelberg</B></P><DIV class="table-slot" style="overflow-x: auto"><TABLE class="table table-bordered mdl-shadow--4dp" xsscleaned="overflow-x: scroll !important"><THEAD><TR><TH>Orientation</TH><TH>Sequence</TH></TR></THEAD><TBODY><TR><TD>forward</TD><TD>CATTATGAATTCGCGGCCGCTTCTAGATGACCATCACCATC</TD></TR><TR><TD>reverse</TD><TD>GATAATCTGCAGCGGCCGCTACTAGTATTAG</TD></TR></TBODY></TABLE></DIV></DIV>
        
           
 
Unfortunately, Hamburg did not have enough time at this stage of the project to order and test the new primers designed by us. Therefore, they will only add the primers to the gene to facilitate the work for the next group who wants to work on the gene.
Even though the gene amplification was not successful in the end, we they emphasized that they are very grateful for the help we provided and the effort we raised to getting fixed their PCR problems. It was also a pleasure for us to work with you. 

        


  </DIV></DIV><DIV class="col-lg-12 col-md-12 col-xs-12"><DIV class="mdl-shadow--4dp" style="padding:20px; float:left; display: inline-block;"><H2 class="heidelberg-green">iGEM goes green</H2>      
Inspired by this years' <A href="https://2017.igem.org/Team:TU_Dresden/iGEM-goes-green/Green_Teams">iGEM goes green</A> initiative launched by the <A href="https://2017.igem.org/Team:TU_Dresden/iGEM-goes-green/Green_Teams">iGEM Team Dresden</A>, we designed a  <A href="https://2017.igem.org/Team:Heidelberg/Collaborations/iGEM-goes-green">model</A> that has provided us and will provide the iGEM community with a versatile tool to minimize the amount of medium used in the process of operating Phage-Assisted Continuous Evolution (PACE) to develop enhanced proteins. With the help of this model, it is possible for everyone to calculate the appropriate amount of medium that is needed to operate a successful implementation by tuning all parameters. We also provide information about PACE applications conducted by others in the past to compare the values to. For more detailed information about our iGEM goes green activites and to get more information about our medium consumption model please visit our <A href="https://2017.igem.org/Team:Heidelberg/Collaborations/iGEM-goes-green">iGEM goes green page</A>.
        
    
      


  </DIV></DIV><DIV class="col-lg-12 col-md-12 col-xs-12"><DIV class="mdl-shadow--4dp" style="padding:20px; float:left; display: inline-block;"><H2 class="heidelberg-blue">Mentoring Stuttgart</H2><A href="https://2017.igem.org/Team:Stuttgart/Collaborations">iGEM Stuttgart</A> did not only participate in our mutagenesis plasmid inter-lab study but also we helped them according several issues. As they are participating in iGEM for the first time, we gave them general advice accoring time management and helped them to set priorites.

To reach their goal for &quot;lighting up the pipe&quot; they also needed to include safety aspects and asked us for some inspirations. They asked us to design a killswitch for their bacteria and we suggested to use an optogenetic-killswitch to ensure, that the <I>E. coli</I> are killed after they &quot;lighted up the pipe&quot; by secreting keratinases, lipases and esterases. We are happy that you integrated our ideas and suggestions into your project and that we hereby could contribute to make your project <A href="https://2017.igem.org/Team:Heidelberg/Safety">safer</A> for the environment.



  </DIV></DIV><DIV class="col-lg-12 col-md-12 col-xs-12"><DIV class="mdl-shadow--4dp" style="padding:20px; float:left; display: inline-block;"><H2 class="heidelberg-red">Translation Center</H2> 
Bringing science closer to the public is a major goal in iGEM. As communicating scientific content can be challenging because complex issues need to be unraveled to easy understandable terms, one step could be translating scientific texts into <A href="https://2017.igem.org/Team:Heidelberg/Collaborations/Translation">many languages</A>. Following this idea, the iGEM Team KU Leuven started a translation center and asked all teams to participate. Each team uploaded its <A href="https://2017.igem.org/Team:Heidelberg/Collaborations/Translation">project description</A> and translated other project descriptions into their mother tongue. This is a first step towards simplification and easier communication with the general public because one barrier, namely the language barrier, can be overcome with the translation center. We translated several project descriptions into German and could thereby hopefully contribute to a better communication between scientists and a broad public.
</DIV></DIV><DIV class="col-lg-12 col-md-12 col-xs-12"><DIV class="mdl-shadow--4dp" style="padding:20px; float:left; display: inline-block;"><H2 class="heidelberg-orange">Postcards</H2>


Participating in the Postcard campaign of Duesseldorf-Cologne was a real success. Thanks to the inspiring and eye-catching designs of all postcards we started talks about synthetic biology, GMOs and other issues addressed by over 36 postcards we received.    
</DIV></DIV><DIV class="col-lg-12 col-md-12 col-xs-12"><DIV class="mdl-shadow--4dp" style="padding:20px; float:left; display: inline-block;"><H2 class="heidelberg-green">No Science without Tolerance</H2>



      
        
Respect is an essential iGEM value and tolerance an important aspect of it. We were thus very happy to support the tolerance campaign by iGEM Team Technion (Israel). All our creativity was used to draw this lab-associated tolerance lettering and we can hopefully contribute to draw more attention to the fundamental value of tolerance in the scientific community and in public.
<DIV class="col-lg-12 col-md-12 col-xs-12"><DIV class="img-caption title" style="width: auto;"><STRONG><B>
      
                iGEM Team Heidelberg strongly supporting Tolerance
    </B></STRONG></DIV></DIV></DIV></DIV><DIV class="col-lg-12 col-md-12 col-xs-12"><DIV class="mdl-shadow--4dp" style="padding:20px; float:left; display: inline-block;"><H2 class="heidelberg-blue">Surveys – Supporting Responsible and Efficient Technology Development within iGEM</H2><B>Bostons Microfluidic Survey</B>
The Boston University Hardware team worked with microfluidic chips and wanted to establish an archive with different designs and protocols. Their goal is to fit the archival structure to any needs of its users. To support this project, we filled in their survey and could thereby hopefully contribute to a well applied, sustainable design of their archive.
<B>Beijing Institute of Technology: Survey according health  care and liver cancer</B>
iGEM Team BIT worked on a liver cancer related project and wanted to integrate the opinion of us iGEMers into the design of their project. We answered their survey and are looking forward to seeing their great ideas realized.
<B>Biological Material Transport Survey from State University of Amazonas</B>
Shipment of biological material often faces problems when cooling is required and long distances need to be conquered. This is particularly true for less developed countries and regions with no direct access to an (freight) airport. The Amazonas Brazil team wanted to collect information from all over the world to get a deeper insight into import and export regulation processes of countries around the world. We hope that we supported your collection of information by thoroughly answering your questionnaire.
<B>Air pollution survey from iGEM Pasteur Paris</B>
Environmental pollution and climate change are two of the major challenges we desperately need to solve towards our agreed goal of a sustainable human civilization on our planet. The iGEM Team Pasteur Paris focused on environmental issues and highlighted the problem of air pollution, which is particularly strong problem in quickly industrialized countries with low standards of waste air cleaning in fossil fuel power stations, industrial facilities and transportation vehicles. To receive an impression about the air pollution situation in different regions all over the world they created a comprehensive survey. We hope that we could convey an impression about the situation in Heidelberg, which fortunately is a rather small, bicycle-friendly city closely surrounded by five beautiful nature reserves. iGEM Team Heidelberg is also taking part in the <A href="https://2017.igem.org/Team:Heidelberg/Collaborations">iGEM goes green initiative</A> by team TU Dresden.
<B>Survey about methane production from UNebraska-Lincoln</B>
The iGEM Team UNebraska-Lincoln aims at inhibiting the methane production in cattle and wanted to integrate the general public opinion into the design of their project. All members of iGEM team Heidelberg filled in their survey and we are looking forward to helping you gather information and maybe even contribute to decreasing the alarming speed of climate change.
<B>Survey about cholera from iGEM Toulouse</B>
In order to tackle the cholera disease, the iGEM Team Toulouse asked everyone to answering their questionnaire to receive an impression about the informational situation according to cholera disease. Hopefully we could contribute a little bit to succeeding with your project on eradicating <I>Vibrio cholerae</I>.
<B>Genetic modification to produce medicines - Cardiff iGEM</B>
To investigate the public´s view on genetic modification for pharmaceutical production the Cardiff iGEM Team created a survey. They aim at treating the Grave´s Disease, an autoimmune disease affecting the thyroid, with the help of genetically modified organisms. We filled in their questionnaire and hope that they succeed with realizing their innovative therapeutic approach.
<B>GMO perception survey from Ionis Paris</B>
To gain more information about GMO perception by the public the iGEM Team Ionis Paris conducted a survey we took part in. They aim at creating an interactive map to make the comprehensive information from their interesting questionnaire easily accessible. This map contains regimentations and opinions about GMOs and could provide novel insights into region-specific, public perception on the creation and use of GMOs.
<B>CRISPR along the iGEM: survey from Amazonas Brazil</B>
No doubt that CRISPR is a breakthrough technology that should be easily accessible by any (responsible) researcher around the globe. To establish a simple and standardized methodology for applying CRISPR the iGEM Team Amazonas Brazil created a survey to communicate with other iGEM teams and exchange information about problems when working with the CRISPR technology. We participated in this survey and are eager to see your solutions for an easier handling of this method come into play in iGEM.
<B>Diagnosis device survey from iGEM Team Munich</B>
The iGEM Team Munich collected opinions about diagnostic devices in a survey that was distributed to people with different professional backgrounds. In certain countries access to regular medical care is limited, either due to regional fluctuation in the density of hospitals, pharmacies and doctors or due to country-specific healthcare laws. This issue affects millions of people world-wide, both in developing as well as modern countries. iGEM Team Munich aims at developing diagnostic devices, that could be used at home and provide people with diagnosis and hence medical care that would otherwise have no access to.        
        


  </DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV style="background-color: white; padding-top: 20px;"><DIV class="container"><DIV class="row"><DIV class="col-lg-12 col-md-12 col-xs-12 content" style="display: inline-block;"><DIV class="row content"><DIV class="col-lg-12 col-md-12 col-xs-12" style="padding-top: 20px; padding-bottom: 50px; display: block"><DIV class="mdl-shadow--4dp" style="padding: 20px"><H3> References </H3></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><SECTION id="footer-sec" style="background-color: #222 !important"><DIV class="container"><DIV class="row" style="padding-top: 30px;"><DIV class="col-md-3"><H4 style="text-align: center">Quote</H4><P><I>“”</I></P></DIV><DIV class="col-md-3"><H4 style="text-align: center"> Useful <STRONG>Links</STRONG></H4><P><A href="https://igem.org/Special:SpecialPages">
			Special pages
		</A></P><P><A href="https://igem.org/Main_Page">
			Main Page
		</A></P></DIV><DIV class="col-md-3"><H4 style="text-align: center"> Follow us on </H4><DIV style="text-align: center;"><SPAN style="color: grey; text-align: center;">&amp;</SPAN></DIV></DIV><DIV class="col-md-3"><H4 style="text-align: center"> Contact us </H4><P>

               iGEM-Heidelberg2017@bioquant.uni-heidelberg.de
               
		 Im Neuenheimer Feld 267
		
		69120 Heidelberg
                    </P></DIV></DIV></DIV></SECTION></DIV></DIV></DIV></DIV></BODY></HTML>