<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Heidelberg_Organosilicons skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Heidelberg/Organosilicons</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container"><DIV class="navbar-header"><A type="button" data-canvas="body" id="sidebarCollapse" class="navbar-toggle" data-toggle="offcanvas" data-target="#sidebar" style="color: #393939;margin-top: 9px;">NAVIGATION
                    </A></DIV><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav pull-right"><LI class="dropdown"><A href="https://2017.igem.org/Team:Heidelberg/Team">People </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Team">Team</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Attributions">Attributions</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Sponsoring">Sponsoring</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/For_Judges">For Judges</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Jamboree">Giant Jamboree</A></LI></UL><LI class="dropdown"><A href="https://2017.igem.org/Team:Heidelberg/Description">Project </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Description">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Design">Background &amp; Design </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Pace">PACE</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Predcel">PREDCEL</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Results">Results Summary</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Validation">Software Validation</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Organosilicons">Organosilicons</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Protein_Interaction">Protein Interaction</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Cytochrome_Engineering">Cytochrome Engineering</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Optogenetics">Optogenetics</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/CRISPR">CRISPR Cas9</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/InterLab">InterLab Study</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Achievements">Achievements</A></LI><LI class="dropdown"><A href="https://2017.igem.org/Team:Heidelberg/Parts">Parts </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Basic_Part">Basic Parts</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Composite_Part">Composite Parts</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Improved_Part">Improved Part</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Part_Collection">Parts Collection</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Part_List">Part List</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/RFC">RFC</A></LI><LI class="dropdown"><A href="https://2017.igem.org/Team:Heidelberg/Model">Modeling </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Phage_Titer">Phage Titer</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Mutagenesis_Induction">Mutagenesis Induction</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Mutation_Rate_Estimation">Mutation Rate Estimation</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Lagoon_Contamination">Lagoon Contamination</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Medium_Consumption">Medium Consumption</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Tools">Interactive Web Tools</A></LI><LI class="dropdown"><A href="https://2017.igem.org/Team:Heidelberg/Software">Software </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software">AiGEM Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software/DeeProtein">DeeProtein</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software/GAIA">GAIA</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software/SafetyNet">SafetyNet</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software/MAWS">MAWS 2.0</A></LI><LI class="dropdown"><A href="https://2017.igem.org/Team:Heidelberg/Human_Practices">Human Practices </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Human_Practices">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/HP/Gold_Integrated">Integrated Human Practices</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Interviews">Expert Interviews</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Safety">Safety &amp; Security</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Engagement">Public Engagement</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Education">Education</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Collaborations">Collaborations</A></LI><LI class="dropdown"><A href="https://2017.igem.org/Team:Heidelberg/Notebook">Notebook </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Notebook">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Experiments">Methods</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Materials">Materials</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Database">Notebook Database</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Toolbox">Toolbox Guide</A></LI></DIV></DIV><NAV id="sidebar" style="z-index: 10001; background-color: white;border: none;box-shadow:1px 0px 0px #393939;" class="navmenu navmenu-default navmenu-fixed-left offcanvas" role="navigation"><A class="navmenu-brand" href="https://2017.igem.org/Team:Heidelberg"> iGEM TEAM HEIDELBERG 2017</A><DIV id="sidebarHide" data-toggle="offcanvas" data-target="#sidebar" style="padding-left: 20px;"><A style="font-family: 'Josefin Sans' !important; font-size: 16px; line-height: 18px; height: 18px;"> COLLAPSE NAVIGATION </A></DIV><UL class="nav navmenu-nav" id="sidenav"><LI><A data-toggle="collapse" data-target="#People_side">People </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Team">Team</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Attributions">Attributions</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Sponsoring">Sponsoring</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/For_Judges">For Judges</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Jamboree">Giant Jamboree</A></LI></UL><LI><A data-toggle="collapse" data-target="#Project_side">Project </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Description">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Design">Background &amp; Design </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Pace">PACE</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Predcel">PREDCEL</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Results">Results Summary</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Validation">Software Validation</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Organosilicons">Organosilicons</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Protein_Interaction">Protein Interaction</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Cytochrome_Engineering">Cytochrome Engineering</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Optogenetics">Optogenetics</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/CRISPR">CRISPR Cas9</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/InterLab">InterLab Study</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Achievements">Achievements</A></LI><LI><A data-toggle="collapse" data-target="#Parts_side">Parts </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Basic_Part">Basic Parts</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Composite_Part">Composite Parts</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Improved_Part">Improved Part</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Part_Collection">Parts Collection</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Part_List">Part List</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/RFC">RFC</A></LI><LI><A data-toggle="collapse" data-target="#Model_side">Modeling </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Phage_Titer">Phage Titer</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Mutagenesis_Induction">Mutagenesis Induction</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Mutation_Rate_Estimation">Mutation Rate Estimation</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Lagoon_Contamination">Lagoon Contamination</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Medium_Consumption">Medium Consumption</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Tools">Interactive Web Tools</A></LI><LI><A data-toggle="collapse" data-target="#AI_side">Software </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software">AiGEM Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software/DeeProtein">DeeProtein</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software/GAIA">GAIA</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software/SafetyNet">SafetyNet</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software/MAWS">MAWS 2.0</A></LI><LI><A data-toggle="collapse" data-target="#HP_side">Human Practices </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Human_Practices">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/HP/Gold_Integrated">Integrated Human Practices</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Interviews">Expert Interviews</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Safety">Safety &amp; Security</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Engagement">Public Engagement</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Education">Education</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Collaborations">Collaborations</A></LI><LI><A data-toggle="collapse" data-target="#Notebook_side">Notebook </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Notebook">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Experiments">Methods</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Materials">Materials</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Database">Notebook Database</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Toolbox">Toolbox Guide</A></LI></NAV><DIV style="background-color: white !important; padding: 0 !important; margin: 0 !important;"><DIV class="page-title" style="background-color: white !important; border-top: 0px solid #222;"><DIV class="container" style=" padding: 0 !important;"><DIV class="col-lg-12 col-md-12 col-xs-12" style="padding-right: 0 !important; padding-bottom: 0 !important; position: relative; left: -10px;"><DIV style="border-right: 5px solid #393939; padding: 20px 20px 0px 0px"><DIV class="header-title heidelberg-yellow"> 
    Organosilicons </DIV><DIV class="header-subtitle" style="color: #393939 !important;"> 
    Synthesis of organosilicons and cytochrome engineering </DIV></DIV></DIV></DIV></DIV><DIV class="container" style="background-color: white !important; padding: 0 !important;"><DIV class="col-lg-12 col-md-12 col-xs-12" style="padding: 0 !important; margin: 0px important!;"><DIV class="container"><DIV class="row"><DIV class="col-lg-12 col-md-12 col-xs-12 content" style="display: inline-block; padding-bottom: 10px; padding-top:10px;"><DIV class="container" style="position: relative; left: -10px; border-left: 5px solid #393939; border-top: 5px solid #393939; padding: 0 !important"><DIV class="col-lg-12 col-md-12 col-xs-12 content" style="text-align: justify; text-color: #393939 !important; padding: 20px 5px 0px 20px !important; font-weight: bold !important; font-family: 'Open Sans' !important">
        
            Organosilicons, compounds containing covalently linked silicon and carbon moieties, provide completely new structural scaffolds with altered chemical properties and metabolism. Therefore, they hold great promise for engineering of novel pharmaceuticals and materials. While organosilicons are absent in natural, biological systems, promiscuous enzymes such as cytochromes exist, that could in principle be engineered to efficiently catalyze C-Si products. Towards our vision of evolving novel cytochromes for fully biological production of organosilicons, we employed a recently described, engineered cytochrome c mutant for in vitro synthesis of ethyl 2-((4-aminophenyl) dimethylsilyl) propanoate and ethyl 2-(dimethyl(phenyl)silyl) propanoate, and validated product formation by GC-MS. Then, we demonstrate a successful in silico prediction of what we think is the first organosilicon-dependent riboswitch and validate its activity and specificity in E. coli hosts cells using a luciferase reporter assay. Our work paves the way towards in vivo directed evolution of enzymes for biological organosilicon production.

    </DIV></DIV><DIV class="container"><DIV class="row"><DIV class="col-lg-12 col-md-12 col-xs-12 content" style="display: inline-block; padding-bottom: 10px; padding-top:10px;"><H1>Introduction</H1>


Organosilicons are organometallic compounds that contain carbon-silicon bonds. In comparison to their respective organic analogs, they display different intrinsic properties due to the distinct chemical properties of silicon. The bond formation tendencies of silicon have a significant impact on their bioavailability and their applicability in pharmacy<X-REF>Franz.2013</X-REF>.

Recent publications cluster their unique features into three categories<X-REF>Franz.2013</X-REF><X-REF>Kan.2016</X-REF>:
The first category comprises the chemical properties of silicon bonds. Typically, silicon forms longer bonds with variable angles, which allows diverse ring conformations and thus alterations in reactivity. Furthermore, its preference to form single bonds leads to chemical compounds that have a higher intrinsic stability than their carbon analogs<X-REF>Schwarz.2016</X-REF>.

The second category represents the bioavailability of organosilicons<X-REF>Franz.2013</X-REF>. As they are more lipophilic compared to their respective carbon counterparts, they are able to easily overcome the membrane barrier of cells. The third - and most important - category addresses the potential pharmaceutical application of carbon-silicon compounds<X-REF>Tacke.2016</X-REF>. Due to their aforementioned tendency to form single rather than double or triple bonds, they display a viable source for stable pharmaceuticals, which are inaccessible as carbon-based molecules. Additionally, the more electropositive characteristic of silicon facilitates hydrogen bond formation and conveniently increases the acidity of compounds. Considering these facts, organosilicons offer major opportunities in the synthesis of bioactive pharmaceuticals, the design of pro-drugs, as well as a safe medicine with a genuine biomedical benefit. 

Recently, a cytochrome c variant was described, which is able to catalyze the formation of C-Si bonds<X-REF>Kan.2016</X-REF> and therefore offers the opportunity to further develop C-Si bond catalyzing enzymes.
In contrast to industrially manufactured organosilicons, the energy input that is required for the enzyme-catalyzed bond formation can be significantly decreased, making their production much more feasible<X-REF>Frampton.2009</X-REF>. In fact, the reaction conditions regarding temperature and pressure shift from 300°C and 2-5 bars in the case of the industrial synthesis to 25°C and atmospheric pressure in the case of the enzyme-mediated synthesis<X-REF>Pachaly.2008</X-REF>. This makes C-Si bond catalyzing enzymes and their natural enantioselectivity a beneficial instrument for the future environmentally friendly research in organosilicon synthesis<X-REF>Frampton.2009</X-REF>.


<DIV class="col-lg-12 col-md-12 col-xs-12"><DIV class="mdl-shadow--4dp" style="padding:20px; float:left}"><DIV class="img-caption title"><STRONG><B>
      Possible application of organosilicons are found in medicine<X-REF>Tacke.2016</X-REF> agriculture and electronics<X-REF>Cui.2009</X-REF></B></STRONG></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="container"><DIV class="row"><DIV class="col-lg-12 col-md-12 col-xs-12 content" style="display: inline-block; padding-bottom: 10px; padding-top:10px;"><H1>Our Idea</H1>
According to our idea to evolve proteins by <A href="https://2017.igem.org/Team:Heidelberg/Pace">PACE</A> and <A href="https://2017.igem.org/Team:Heidelberg/Predcel">PREDCEL</A>, we also envisioned to further develop the previously engineered cytochrome c. Therefore, we linked organosilicon-production directly to a reporter expression via a small molecule-sensing riboswitch<X-REF>Henkin.2008</X-REF>. This riboswitch was designed <I>in silico</I> using the <A href="https://2017.igem.org/Team:Heidelberg/Software/MAWS">MAWS</A> 2.0 software, that was provided by the iGEM Team Heidelberg 2015. To obtain the riboswitch sequence, the chemical structure of the desired product was <I>in silico</I> aligned to randomly generated RNA sequences which were scored according to their ability to form hydrogen bonds with the product. The expression was evaluated using the <A href="https://www.promega.de/resources/technologies/nanoluc-luciferase-redefining-reporter-assays/">NanoLuc</A> reporter provided by Promega.

<DIV class="container"><DIV class="row"><DIV class="col-lg-12 col-md-12 col-xs-12 content" style="display: inline-block; padding-bottom: 10px; padding-top:10px;"><DIV class="col-lg-6 col-md-12 col-xs-12 pull-left image-box-left"><DIV class="mdl-shadow--4dp" style="padding:20px; float:left"><DIV class="img-caption title"><STRONG><B>
      Figure 1:
    </B></STRONG></DIV><DIV class="img-caption subtitle">
      Our workflow for organosilicon production and cytochrome engineering.
    </DIV></DIV></DIV><DIV class="col-lg-6 col-md-12 col-xs-12 pull-left image-box-left"><DIV class="mdl-shadow--4dp" style="padding:20px; float:left"><DIV class="img-caption title"><STRONG><B>
      Figure 2:
    </B></STRONG></DIV><DIV class="img-caption subtitle">
      3D structure of the cytochrome c derived from <I><A href="http://www.rcsb.org/pdb/explore.do?structureId=3cp5">Rhodothermus marinus</A></I> that is used as the catalytic unit in the production of organosilicon. Depicted as yellow sticks is the heme prosthetic group of the electron carrier protein. The protein part of the cytochrome c is illustrated as orange ribbon structure.
    </DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="container"><DIV class="row"><DIV class="col-lg-12 col-md-12 col-xs-12 content" style="display: inline-block; padding-bottom: 10px; padding-top:10px;"><H1>Experimental procedures</H1><H2>Design and cloning of the riboswitch and cytochrome c constructs</H2>

            The educts for the organosilicon synthesis were commercially available in the case of <I>dimethyl(phenyl)silane</I> (1) and <I>ethyl 2-diazopropanoate</I> (3) or were custom synthesized by Fabian Ebner (Greb group, ACI Heidelberg, Germany) in the case of <I>4-(dimethylsilyl)aniline</I> (2) (Fig. 4). The corresponding riboswitch was designed accordingly using the <A href="https://2017.igem.org/Team:Heidelberg/Software/MAWS">MAWS</A> 2.0 software developed by the iGEM Team Heidelberg 2015. The most favorable sequence (<A href="http://parts.igem.org/Part:BBa_K2398555">BBa_K2398555</A>) was ordered as oligos, which were annealed in a single-cycle Touch-Down PCR, decreasing the temperature by 0.1°C x sec<SUP>-1</SUP> from 95°C to 10°C. The sequence was ordered as oligos and not as gBlock to ensure overhangs of a specific length at the 5’ and 3’ ends. Vector and reporter were amplified via PCR and purified by gel extraction (Qiagen). The final plasmid was assembled by using equimolar concentrations of vector, reporter, and the riboswitch in a <A href="https://2017.igem.org/Team:Heidelberg/Experiments">golden gate</A> reaction. The plasmid was transformed into DH10beta cells and purified by plasmid purification (Qiagen).
            
To make organosilicon production more accessible for other iGEM Teams, we codon optimized the wild-type cytochrome c derived from <I>Rhodotermus marinus</I> and cloned it into the pSB1C3 vector. We are proud to present you this part as our best basic part (<A href="http://parts.igem.org/Part:BBa_K2398000">BBa_K2398000</A>). For our purpose, we used a triple mutant created by F. Arnold<X-REF>Kan.2016</X-REF>, that showed a higher activity towards our substrates.
            
    <H2>Riboswitch binding assay</H2><DIV class="col-lg-6 col-md-12 col-xs-12 pull-left image-box-left"><DIV class="mdl-shadow--4dp" style="padding:20px; float:left"><DIV class="img-caption title"><STRONG><B>
      Figure 3:
    </B></STRONG></DIV><DIV class="img-caption subtitle">
      The riboswitch binding assay in a 96-well plate immediately after substrate addition. Rows show biological triplicates (n = 3) and riboswitch activators are compound (5) and compound (1).
    </DIV></DIV></DIV>


The purified construct was re-transformed into DH10beta and incubated at 37°C overnight. Three clones were picked for biological triplicated and inoculated separately overnight in 5ml LB including chloramphenicol. The assay was performed in a transparent 96-well plate (Greiner) with a final volume of 250µl per well (Fig. 3). The light absorbance was measured at 460nm. The background absorbance was determined by measuring separately medium only, medium including cells, and medium including cells and the Nano-Glo substrate. The cell concentration was adjusted to a final OD<SUB>600nm</SUB> of 0.6 per well. The riboswitch activity was detected in wells that additionally contained 5mM or 15mM of either the educt (1) as the initial substrate or the intended product (5), respectively. The reaction was incubated at room temperature for 10min without the Nano-Glo substrate. Afterwards, the Nano-Glo substrate was added to each well (125µl) to ensure its maximum reactivity. The light absorbance was measured immediately using the Tecan Infinite M200 Pro plate reader.

    </DIV></DIV></DIV><DIV class="container"><DIV class="row"><DIV class="col-lg-12 col-md-12 col-xs-12 content" style="display: inline-block; padding-bottom: 10px; padding-top:10px;"><H1 id="Results">Results</H1><H2>Synthesis of the organosilicon compounds</H2><DIV class="col-lg-6 col-md-12 col-xs-12 pull-left image-box-left"><DIV class="mdl-shadow--4dp" style="padding:20px; float:left"><DIV class="img-caption title"><STRONG><B>
      Figure 4:
    </B></STRONG></DIV><DIV class="img-caption subtitle">
      Educts (1), (2) and (3) and the products (4) and (5) of the organosilicon synthesis.
    </DIV></DIV></DIV>




          The compounds <I>ethyl 2-((4-aminophenyl)dimethylsilyl)propanoate</I> (4) and <I>ethyl 2-(dimethyl(phenyl)silyl)propanoate</I> (5) were successfully synthesized, validated by gas chromatography-mass spectrometry (GC-MS) and the conversion rate was determined (Fig. 5-8). Fig. 5 shows the <I>ethyl 2-((4-aminophenyl)dimethylsilyl)propanoate</I> product with a retention time corresponding to 11.7 minutes. The silicon educt <I>4-(dimethylsilyl)aniline</I> that emerges at 6.9 minutes implies that there is an incomplete conversion due to enzyme inactivity or an unfavorable reaction. The value of the conversion rate is 47.5%, which differs from the literature value of 70%<X-REF>Kan.2016</X-REF>. Nonetheless, the amount of the product formed was sufficient for all further experiments. The unconverted diazo educt <I>ethyl 2-diazo propanoate</I> corresponds to the retention time peaks at 7.2 and 7.4 minutes. The small peak at 10.8 minutes is likely a side product of the reaction. Fig. 6 shows the respective mass spectrum of the breakdown of the product and its correct mass of 251 daltons.
 
        The GC-MS analysis of the second organosilicon <I>ethyl 2-(dimethyl(phenyl)silyl)propanoate</I> is demonstrated in the Fig. 5 and 6. The educt dimethyl(phenyl)silane was completely converted, and the product emerges at a retention time of 9.2 minutes (Fig. 7). The mass spectrometry analysis verified the product with a mass of 236 daltons (Fig. 8).
The reaction without the enzyme represents our negative control and is depicted in Fig. 9. As expected, only the two educts emerged after their respective retention time.

       In summary, sufficient concentrations of the organosilicon products were synthesized, and in case of the organosilicon (5) also purified to &gt;98% chemical purity.


<DIV class="col-lg-12 col-md-12 col-xs-12"><DIV class="mdl-shadow--4dp" style="padding:20px; float:left}"><DIV class="img-caption title"><STRONG><B>
      Figure 5:
    </B></STRONG></DIV><DIV class="img-caption subtitle">
      Gas chromatogram for the reaction of educt (2) and (3) to the product (4). 11.7 minutes retention time indicates product formation. Unconverted educts converge 6.9 and 7.2 (7.4) minutes.
    </DIV></DIV></DIV><DIV class="col-lg-12 col-md-12 col-xs-12"><DIV class="mdl-shadow--4dp" style="padding:20px; float:left}"><DIV class="img-caption title"><STRONG><B>
      Figure 6:
    </B></STRONG></DIV><DIV class="img-caption subtitle">
      Mass chromatogram shows the breakdown of the product (4) <I>ethyl 2-((4-aminophenyl)dimethylsilyl)propanoate</I>. The product itself corresponds to a mass of 251 daltons.
    </DIV></DIV></DIV><DIV class="col-lg-12 col-md-12 col-xs-12"><DIV class="mdl-shadow--4dp" style="padding:20px; float:left}"><DIV class="img-caption title"><STRONG><B>
      Figure 7:
    </B></STRONG></DIV><DIV class="img-caption subtitle">
      Gas chromatogram for the reaction of educt (1) and (3) to the product (5). 9.2 minutes retention time indicates product formation.
    </DIV></DIV></DIV><DIV class="col-lg-12 col-md-12 col-xs-12"><DIV class="mdl-shadow--4dp" style="padding:20px; float:left}"><DIV class="img-caption title"><STRONG><B>
      Figure 8:
    </B></STRONG></DIV><DIV class="img-caption subtitle">
      Mass chromatogram shows the breakdown of the product (5) <I>ethyl 2-(dimethyl(phenyl)silyl)propanoate</I>. The product itself corresponds to a mass of 236 daltons.
    </DIV></DIV></DIV><DIV class="col-lg-12 col-md-12 col-xs-12"><DIV class="mdl-shadow--4dp" style="padding:20px; float:left}"><DIV class="img-caption title"><STRONG><B>
      Figure 9:
    </B></STRONG></DIV><DIV class="img-caption subtitle">
      Gas chromatogram for the control reaction where the enzyme has been substituted.
    </DIV></DIV></DIV></DIV></DIV></DIV><DIV class="container"><DIV class="row"><DIV class="col-lg-12 col-md-12 col-xs-12 content" style="display: inline-block; padding-bottom: 10px; padding-top:10px;"><H2>Validation of the Riboswitch</H2>

We further tested our synthesized compounds by using the NanoLuc Reporter linked to our riboswitch. As expected, the addition of 15 mM of the compound (5) resulted in a higher emission rate compared to the sample containing only 5 mM. Remarkably, the specifically for the riboswitch designed compound (5) showed significantly higher activity compared to the original precursor compound (1), indicating a higher binding affinity (Fig. 10). The increase in enzyme activity upon addition of the specific riboswitch activator was determined in relation to the enzyme activity in presence of the initial substrate. The result demonstrated in Fig. 11, shows a 1.5-fold enzyme activity when using the newly synthesized substrate.


<DIV class="col-lg-12 col-md-12 col-xs-12"><DIV class="mdl-shadow--4dp" style="padding:20px; float:left}"><DIV class="img-caption title"><STRONG><B>
      Figure 10:
    </B></STRONG></DIV><DIV class="img-caption subtitle">
      Light emission detection of the NanoLuc reaction for different riboswitch activators and concentrations. Addition of compound (5) to the reaction resulted in increased enzyme activity as indicated by the two bars on the left-hand side compared to the compound (1) reaction.
    </DIV></DIV></DIV><DIV class="col-lg-12 col-md-12 col-xs-12"><DIV class="mdl-shadow--4dp" style="padding:20px; float:left}"><DIV class="img-caption title"><STRONG><B>
      Figure 11:
    </B></STRONG></DIV><DIV class="img-caption subtitle">
      Fold enzyme activity upon addition of compound (5) compared to compound (1) at either 5 mM concentration (left bar) or 15 mM concentration (right bar). At both concentrations, compound (5) exceeded the reaction activity of compound (1) by 1.5-fold.
    </DIV></DIV></DIV></DIV></DIV></DIV><DIV class="container"><DIV class="row"><DIV class="col-lg-12 col-md-12 col-xs-12 content" style="display: inline-block; padding-bottom: 10px; padding-top:10px;"><H1>Outlook</H1><DIV class="col-lg-6 col-md-12 col-xs-12 pull-left image-box-left"><DIV class="mdl-shadow--4dp" style="padding:20px; float:left"><DIV class="img-caption title"><STRONG><B>
      Figure 12:
    </B></STRONG></DIV><DIV class="img-caption subtitle">
      The selection of any small molecule synthesis is implemented into the product formation drive <A href="https://2017.igem.org/Team:Heidelberg/Predcel">PREDCEL</A> approach. The selection pressure is applied by the most efficient riboswitch binding and subsequent gene expression activation.
    </DIV></DIV></DIV>

            
        We were able to synthesize two organosilicons and investigated their purity using GC-MS. Employing the <A href="https://2017.igem.org/Team:Heidelberg/Software/MAWS">MAWS</A> 2.0 software, we designed to our knowledge the first organosilicon dependent riboswitch and validated its specificity using a luciferase reporter assay. Together with our improved  directed evolution method (<A href="https://2017.igem.org/Team:Heidelberg/Predcel">PREDCEL</A>) and our <A href="https://2017.igem.org/Team:Heidelberg/Cytochrome_Engineering">Cytochrome Engineering</A> approach, our organosilicon part paves the way towards an <I>in vivo</I> directed evolution of enzymes, and brings us closer to the application of organosilicons in pharmacy.
In addition, we provide a general circuit that allows an easy design for the evolution of in principle any enzyme based on riboswitches mediated by positive and negative selection (Fig. 12). A pool of pre-selected enzyme isoforms are provided by the <A href="https://2017.igem.org/Team:Heidelberg/Software">software tool AiGEM</A>. In the case of a positive selection, the desired product binds to the riboswitch that was specifically designed to be activated by it. The subsequent conformational change then enhances phage propagation via the increased expression of the geneIII. Consequently, the enzyme that most efficiently forms the product of interest is enriched over time in a positive selection process. 
In contrast, to ensure that enzymes that synthesize undesired products are not propagated, negative selection is conducted. Therefore, riboswitches mediate a negative geneIII expression upon product binding. The negative geneIII prevents the further phage propagation and thus, decreases the abundance of all enzymes not specifically forming the desired product<X-REF>Carlson.2014</X-REF>. In any case, directed evolution is driven towards the enzyme of interest.  

    </DIV></DIV></DIV></DIV></DIV></DIV><DIV style="background-color: white; padding-top: 20px;"><DIV class="container"><DIV class="row"><DIV class="col-lg-12 col-md-12 col-xs-12 content" style="display: inline-block;"><DIV class="row content"><DIV class="col-lg-12 col-md-12 col-xs-12" style="padding-top: 20px; padding-bottom: 50px; display: block"><DIV class="mdl-shadow--4dp" style="padding: 20px"><H3> References </H3></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><SECTION id="footer-sec" style="background-color: #222 !important"><DIV class="container"><DIV class="row" style="padding-top: 30px;"><DIV class="col-md-3"><H4 style="text-align: center">Quote</H4><P><I>“”</I></P></DIV><DIV class="col-md-3"><H4 style="text-align: center"> Useful <STRONG>Links</STRONG></H4><P><A href="https://igem.org/Special:SpecialPages">
			Special pages
		</A></P><P><A href="https://igem.org/Main_Page">
			Main Page
		</A></P></DIV><DIV class="col-md-3"><H4 style="text-align: center"> Follow us on </H4><DIV style="text-align: center;"><SPAN style="color: grey; text-align: center;">&amp;</SPAN></DIV></DIV><DIV class="col-md-3"><H4 style="text-align: center"> Contact us </H4><P>

               iGEM-Heidelberg2017@bioquant.uni-heidelberg.de
               
		 Im Neuenheimer Feld 267
		
		69120 Heidelberg
                    </P></DIV></DIV></DIV></SECTION></DIV></DIV></DIV></DIV></BODY></HTML>