<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Heidelberg_Notebook skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Heidelberg/Notebook</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container"><DIV class="navbar-header"><A type="button" data-canvas="body" id="sidebarCollapse" class="navbar-toggle" data-toggle="offcanvas" data-target="#sidebar" style="color: #393939;margin-top: 9px;">NAVIGATION
                    </A></DIV><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav pull-right"><LI class="dropdown"><A href="https://2017.igem.org/Team:Heidelberg/Team">People </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Team">Team</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Attributions">Attributions</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Sponsoring">Sponsoring</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/For_Judges">For Judges</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Jamboree">Giant Jamboree</A></LI></UL><LI class="dropdown"><A href="https://2017.igem.org/Team:Heidelberg/Description">Project </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Description">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Design">Background &amp; Design </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Pace">PACE</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Predcel">PREDCEL</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Results">Results Summary</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Validation">Software Validation</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Organosilicons">Organosilicons</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Protein_Interaction">Protein Interaction</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Cytochrome_Engineering">Cytochrome Engineering</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Optogenetics">Optogenetics</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/CRISPR">CRISPR Cas9</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/InterLab">InterLab Study</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Achievements">Achievements</A></LI><LI class="dropdown"><A href="https://2017.igem.org/Team:Heidelberg/Parts">Parts </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Basic_Part">Basic Parts</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Composite_Part">Composite Parts</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Improved_Part">Improved Part</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Part_Collection">Parts Collection</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Part_List">Part List</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/RFC">RFC</A></LI><LI class="dropdown"><A href="https://2017.igem.org/Team:Heidelberg/Model">Modeling </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Phage_Titer">Phage Titer</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Mutagenesis_Induction">Mutagenesis Induction</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Mutation_Rate_Estimation">Mutation Rate Estimation</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Lagoon_Contamination">Lagoon Contamination</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Medium_Consumption">Medium Consumption</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Tools">Interactive Web Tools</A></LI><LI class="dropdown"><A href="https://2017.igem.org/Team:Heidelberg/Software">Software </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software">AiGEM Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software/DeeProtein">DeeProtein</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software/GAIA">GAIA</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software/SafetyNet">SafetyNet</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software/MAWS">MAWS 2.0</A></LI><LI class="dropdown"><A href="https://2017.igem.org/Team:Heidelberg/Human_Practices">Human Practices </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Human_Practices">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/HP/Gold_Integrated">Integrated Human Practices</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Interviews">Expert Interviews</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Safety">Safety &amp; Security</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Engagement">Public Engagement</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Education">Education</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Collaborations">Collaborations</A></LI><LI class="dropdown"><A href="https://2017.igem.org/Team:Heidelberg/Notebook">Notebook </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Notebook">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Experiments">Methods</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Materials">Materials</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Database">Notebook Database</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Toolbox">Toolbox Guide</A></LI></DIV></DIV><NAV id="sidebar" style="z-index: 10001; background-color: white;border: none;box-shadow:1px 0px 0px #393939;" class="navmenu navmenu-default navmenu-fixed-left offcanvas" role="navigation"><A class="navmenu-brand" href="https://2017.igem.org/Team:Heidelberg"> iGEM TEAM HEIDELBERG 2017</A><DIV id="sidebarHide" data-toggle="offcanvas" data-target="#sidebar" style="padding-left: 20px;"><A style="font-family: 'Josefin Sans' !important; font-size: 16px; line-height: 18px; height: 18px;"> COLLAPSE NAVIGATION </A></DIV><UL class="nav navmenu-nav" id="sidenav"><LI><A data-toggle="collapse" data-target="#People_side">People </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Team">Team</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Attributions">Attributions</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Sponsoring">Sponsoring</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/For_Judges">For Judges</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Jamboree">Giant Jamboree</A></LI></UL><LI><A data-toggle="collapse" data-target="#Project_side">Project </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Description">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Design">Background &amp; Design </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Pace">PACE</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Predcel">PREDCEL</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Results">Results Summary</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Validation">Software Validation</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Organosilicons">Organosilicons</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Protein_Interaction">Protein Interaction</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Cytochrome_Engineering">Cytochrome Engineering</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Optogenetics">Optogenetics</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/CRISPR">CRISPR Cas9</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/InterLab">InterLab Study</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Achievements">Achievements</A></LI><LI><A data-toggle="collapse" data-target="#Parts_side">Parts </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Basic_Part">Basic Parts</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Composite_Part">Composite Parts</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Improved_Part">Improved Part</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Part_Collection">Parts Collection</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Part_List">Part List</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/RFC">RFC</A></LI><LI><A data-toggle="collapse" data-target="#Model_side">Modeling </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Phage_Titer">Phage Titer</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Mutagenesis_Induction">Mutagenesis Induction</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Mutation_Rate_Estimation">Mutation Rate Estimation</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Lagoon_Contamination">Lagoon Contamination</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Medium_Consumption">Medium Consumption</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Tools">Interactive Web Tools</A></LI><LI><A data-toggle="collapse" data-target="#AI_side">Software </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software">AiGEM Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software/DeeProtein">DeeProtein</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software/GAIA">GAIA</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software/SafetyNet">SafetyNet</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software/MAWS">MAWS 2.0</A></LI><LI><A data-toggle="collapse" data-target="#HP_side">Human Practices </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Human_Practices">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/HP/Gold_Integrated">Integrated Human Practices</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Interviews">Expert Interviews</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Safety">Safety &amp; Security</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Engagement">Public Engagement</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Education">Education</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Collaborations">Collaborations</A></LI><LI><A data-toggle="collapse" data-target="#Notebook_side">Notebook </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Notebook">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Experiments">Methods</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Materials">Materials</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Database">Notebook Database</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Toolbox">Toolbox Guide</A></LI></NAV><DIV style="background-color: white !important; padding: 0 !important; margin: 0 !important;"><DIV class="page-title" style="background-color: white !important; border-top: 0px solid #222;"><DIV class="container" style=" padding: 0 !important;"><DIV class="col-lg-12 col-md-12 col-xs-12" style="padding-right: 0 !important; padding-bottom: 0 !important; position: relative; left: -10px;"><DIV style="border-right: 5px solid #393939; padding: 20px 20px 0px 0px"><DIV class="header-title heidelberg-orange"> Notebook </DIV><DIV class="header-subtitle" style="color: #393939 !important;"> Weekly Reports </DIV></DIV></DIV></DIV></DIV><DIV class="container" style="background-color: white !important; padding: 0 !important;"><DIV class="col-lg-12 col-md-12 col-xs-12" style="padding: 0 !important; margin: 0px important!;"><DIV class="container" style="position: relative; left: -10px; border-left: 5px solid #393939; border-top: 5px solid #393939; padding: 0 !important"><DIV class="col-lg-12 col-md-12 col-xs-12 content" style="text-align: justify; text-color: #393939 !important; padding: 20px 5px 0px 20px !important; font-weight: bold !important; font-family: 'Open Sans' !important">
        
In our Notebooks you will find all information on how specifically we developed the different parts of our project. Have a look at our weekly reports below to find descriptions of what we worked on chronologically. Information on <A href="https://2017.igem.org/Team:Heidelberg/Materials">Materials</A> and <A href="https://2017.igem.org/Team:Heidelberg/Experiments">Methods</A> is provided on separate pages. Finally you will find a detailled documentation of all our labwork in the <A href="https://2017.igem.org/Team:Heidelberg/Database">Notebook Database</A>. 

    </DIV></DIV><DIV class="container"><DIV class="row"><DIV class="col-lg-12 col-md-12 col-xs-12 content" style="display: inline-block; padding-bottom: 10px; padding-top:10px;"><DIV class="panel-group accordion" id="accordion"><DIV class="panel panel-default"><A data-toggle="collapse" data-parent="#accordion" href="#collapseweek_id-0"><DIV class="panel-heading">
            Calendar Week
28
        </DIV></A><DIV id="collapseweek_id-0" class="panel-collapse collapse"><DIV class="panel-body"><H2>Split Polymerase</H2><H2 id="transformation-of-electrocompetent-cells-for-reproduction-of-dickinson-experiments">Transformation of electrocompetent cells for reproduction of Dickinson-Experiments</H2>
In order to gain the three strains which are required for the reproduction of the first seven days of the “Evolution of a split RNA polymerase as a versatile biosensor platform” (Pu et al., 2017), transformation of electrocompetent cells with different strains was performed. S1030 electrocompetent cells were thawed and DNA of the appropriate plasmids were added. The amount of DNA of each plasmid was adjusted at equivalent amounts. First, the recovery medium was cooled on ice and thawed. The electroporation cuvettes were also refrigerated on ice. Following this, the 40 µl of S1030 cells were added to the DNA in a separate tube and mixed carefully. Next, the mixture was transferred to the cuvettes and electroporation was performed using 2500 V. Cells were immediately recovered using 1 ml SOC medium and grown at 37 °C for 1 h and 275 rpm. The resulting cell culture was centrifuged at 6000 rpm for 2 min and the cell pellet was resuspended using 200 µl SOC medium. After this, 200 µl of the cell culture were applied on the related LB-Agar plates, displaying the corresponding antibiotic resistance and 10 mM Glucose concentration. At last, the plates were incubated at 37 °C over night.
For further information about the detailed transformation: awesome_sheet/Trafos/ ID 66-69
<H2 id="transformation-of-electrocompetent-cells-for-reproduction-of-dickinson-experiments">Transformation of electrocompetent cells for reproduction of Dickinson-Experiments</H2>
Due to the non-functionality of the first transformations on the day before, the transformations were repeated. The general procedure was implemented following the instructions of <EM>Wednesday 07/05/2017</EM>. Several changes were made in order to complete the transformation successfully. All DNA amounts were adjusted at 100 ng to improve the efficiency. Beyond that, 25 mM Glucose instead of 10 mM Glucose were added to the LB-agar plates, following the instructions of the information supplied by addgene.
For further information about the detailed transformation: awesome_sheet/Trafos/ ID 70 -73
<H2 id="transformation-of-electrocompetent-cells-for-reproduction-of-dickinson-experiments">Transformation of electrocompetent cells for reproduction of Dickinson-Experiments</H2>
Since the second transformations did not work as well, several changes were made for enable a successfull transformation. The general protocol of Wednesday 07/05/2017 was implemented like before. Glucose concentration in LB-agar plates as well as in SOC medium was adjusted at 100 mM Glucose for prevention of the activation of the MP1 plasmid. In comparison, another transformation was implemented, using TOP10 chemo competent cells to investigate the effect of MP1 as a non-transformable plasmid. Furthermore, pSB1C3 plasmid was also used for transformation to test the general competence of the electrocompetent S1030 cells. A last, a transformation using only 1 ng of MP1 DNA was implemented under equivalent conditions with S1030 cells.

<DIV class="col-lg-12 col-md-12 col-xs-12 content" style="padding-top: 20px; padding-bottom: 20px"><DIV class="table-slot" style="overflow-x: auto"><TABLE class="table table-bordered mdl-shadow--4dp" xsscleaned="overflow-x: scroll !important"><THEAD><TR><TH>Reagent</TH><TH>Stock concentration [mg/ml]</TH></TR></THEAD><TBODY><TR><TD>Ampicillin</TD><TD>50</TD></TR><TR><TD>Chloramphenicol</TD><TD>34</TD></TR><TR><TD>Kanamycin</TD><TD>10</TD></TR><TR><TD>Tetracycline</TD><TD>5</TD></TR><TR><TD>Streptomycin</TD><TD>???</TD></TR><TR><TD>Glucose</TD><TD>1.1 M</TD></TR></TBODY></TABLE></DIV></DIV><DIV class="col-lg-12 col-md-12 col-xs-12 content" style="padding-top: 20px; padding-bottom: 20px"><DIV class="table-slot" style="overflow-x: auto"><TABLE class="table table-bordered mdl-shadow--4dp" xsscleaned="overflow-x: scroll !important"><THEAD><TR><TH>ID</TH><TH>Plasmids</TH><TH>Purification concentration [ng/µl]</TH></TR></THEAD><TBODY><TR><TD>1</TD><TD>MP1</TD><TD>83,8</TD></TR><TR><TD>41</TD><TD>pJin 70</TD><TD>N/A</TD></TR><TR><TD>60</TD><TD>pJin 178</TD><TD>28</TD></TR><TR><TD>62</TD><TD>2-22: pJC173b</TD><TD>29</TD></TR><TR><TD>63</TD><TD>pJin 69 (JM)</TD><TD>140</TD></TR><TR><TD>76</TD><TD>pJin 69 (MW)</TD><TD>373</TD></TR><TR><TD>77</TD><TD>pJin 177</TD><TD>167</TD></TR><TR><TD>79</TD><TD>pJin 172</TD><TD>514</TD></TR><TR><TD>80</TD><TD>pJin 173</TD><TD>232</TD></TR><TR><TD>124</TD><TD>pSB1C3</TD><TD>232</TD></TR></TBODY></TABLE></DIV></DIV>

Plasmids with the IDs 41, 60, 62 and 63 were used incorrectly. The correct plasmids for further transformations have the following IDs: 76,77,79 and 80.
<H2 id="preparation-of-davis-rich-medium-for-transformation-of-mp1-in-s1030">Preparation of Davis Rich Medium for transformation of MP1 in S1030</H2>
Davis Rich Medium (DRM) was used in all PACE papers for transformations as well as the PACE device itself. Therefore, using DRM should be of any great advantage against using normal media like LB, 2xYT or simple Esvelt medium. For preparation of DRM, the amounts of different chemicals was mixed according to the methods of the “Negative selection and stringency modulation in phage-assisted continuous evolution” (Carlson et al., 2014). The amounts were adjusted to 1 l medium instead of 20 l. 10 mM of 50 ml Borsäure and Coppersulfat were prepared using the appropriate amounts calculated using the molecular weights.

<DIV class="col-lg-12 col-md-12 col-xs-12 content" style="padding-top: 20px; padding-bottom: 20px"><DIV class="table-slot" style="overflow-x: auto"><TABLE class="table table-bordered mdl-shadow--4dp" xsscleaned="overflow-x: scroll !important"><THEAD><TR><TH>Reagent</TH><TH>Molecular weight [g/mol]</TH></TR></THEAD><TBODY><TR><TD>Guanin</TD><TD>151,13</TD></TR><TR><TD>H3BO3</TD><TD>61,83</TD></TR><TR><TD>CuSO4</TD><TD>159,61</TD></TR></TBODY></TABLE></DIV></DIV><H2 id="gibson-assembly-preparation">Gibson Assembly preparation</H2>
For gibson assembly, reaction mix was prepared according to the conditions of NEB. The overall reaction volume was 20 µl. 10 µl Gibson MM were mixed with 0.02 – 0.5 pmol of the DNA fragments. The reaction mixture was filled up with ddH2O. For calculation of the molecular weight of the fragments, <A href="http://nebiocalculator.neb.com/#!/main" title="NEBioCalculator">NEBioCalculator</A> was used. The respective length of the fragments was extracted from the awesome_sheet. If the insert contains under 200 bp, the respective amount was multiplicated by five. The reaction mix was incubated at 50 °C for 15 min (2-3 fragments) or for 60 min (4-6 fragments) and the resulting sample was stored at -20 °C. For the following transformation, 2 µl of the assembly mix were used.
For further information, please visit <A href="https://www.neb.com/protocols/2012/12/11/gibson-assembly-protocol-e5510" title="Gibson Assembly protocol">Gibson Assembly - NEB</A>.
Preparation of Davis Rich Medium-like medium for transformation of MP1 in S1030
Transformation of electrocompetent cells for reproduction of Dickinson-Experiments
<H2 id="references-">References:</H2>
Carlson, J.C., Badran, A.H., Guggiana-Nilo, D.A., and Liu, D.R. (2014). Negative selection and stringency modulation in phage-assisted continuous evolution. Nat Chem Biol 10, 216-222.Pu, J., Zinkus-Boltz, J., and Dickinson, B.C. (2017). Evolution of a split RNA polymerase as a versatile biosensor platform. Nat Chem Biol 13, 432-438.
</DIV></DIV></DIV><DIV class="panel panel-default"><A data-toggle="collapse" data-parent="#accordion" href="#collapseweek_id-1"><DIV class="panel-heading">
            Calendar Week
29
        </DIV></A><DIV id="collapseweek_id-1" class="panel-collapse collapse"><DIV class="panel-body"><H2>Split Polymerase</H2><H2 id="preparation-of-davis-rich-medium-for-triple-and-double-transformation-of-dickinson-strains">Preparation of Davis Rich medium for triple and double transformation of Dickinson strains</H2>
Equation for calculation of the required mass for preparation of a 10 mM stock concentration.
<EM>m = c x M x V</EM><DIV class="col-lg-12 col-md-12 col-xs-12 content" style="padding-top: 20px; padding-bottom: 20px"><DIV class="table-slot" style="overflow-x: auto"><TABLE class="table table-bordered mdl-shadow--4dp" xsscleaned="overflow-x: scroll !important"><THEAD><TR><TH>Reagent</TH><TH>Molecular weight [g/mol]</TH></TR></THEAD><TBODY><TR><TD>Guanin</TD><TD>151,13</TD></TR><TR><TD>H3BO3</TD><TD>61,83</TD></TR><TR><TD>CuSO4</TD><TD>159,61</TD></TR><TR><TD>CoCl2</TD><TD>129.839</TD></TR><TR><TD>MnCl2</TD><TD>125.844</TD></TR><TR><TD>ZnSO4</TD><TD>161,47</TD></TR><TR><TD>(NH4)6Mo7</TD><TD>1235,86</TD></TR></TBODY></TABLE></DIV></DIV><DIV class="col-lg-12 col-md-12 col-xs-12 content" style="padding-top: 20px; padding-bottom: 20px"><DIV class="table-slot" style="overflow-x: auto"><TABLE class="table table-bordered mdl-shadow--4dp" xsscleaned="overflow-x: scroll !important"><THEAD><TR><TH>Davis Rich Medium</TH><TH>20 liters</TH><TH>1 liter</TH><TH>unit</TH></TR></THEAD><TBODY><TR><TD>K2HPO4</TD><TD>140</TD><TD>7</TD><TD>g</TD></TR><TR><TD>KH2PO4</TD><TD>40</TD><TD>2</TD><TD>g</TD></TR><TR><TD>(NH4)2SO4</TD><TD>20</TD><TD>1</TD><TD>g</TD></TR><TR><TD>NaCl</TD><TD>58</TD><TD>2,9</TD><TD>g</TD></TR><TR><TD>casamino acids</TD><TD>100</TD><TD>5</TD><TD>g</TD></TR><TR><TD>Tween-80</TD><TD>20ml</TD><TD>1ml</TD></TR><TR><TD>l-cysteine</TD><TD>1</TD><TD>0,05</TD><TD>g</TD></TR><TR><TD>l-tryptophan</TD><TD>0,5</TD><TD>0.025</TD><TD>g</TD></TR><TR><TD>adenine</TD><TD>0,5</TD><TD>0.025</TD><TD>g</TD></TR><TR><TD>guanine</TD><TD>0,5</TD><TD>0.025</TD><TD>g</TD></TR><TR><TD>cytosine</TD><TD>0,5</TD><TD>0.025</TD><TD>g</TD></TR><TR><TD>uracil</TD><TD>0,5</TD><TD>0.025</TD><TD>g</TD></TR><TR><TD>CaCl2</TD><TD>0.5 µM</TD></TR><TR><TD>After autoclaving</TD></TR><TR><TD>NaHCO3</TD><TD>16,8</TD><TD>0,84</TD><TD>g</TD></TR><TR><TD>glucose</TD><TD>90</TD><TD>4,5</TD><TD>g</TD></TR><TR><TD>sodium citrate</TD><TD>10</TD><TD>0,5</TD><TD>g</TD></TR><TR><TD>MgSO4</TD><TD>1</TD><TD>0,05</TD><TD>g</TD></TR><TR><TD>FeSO4</TD><TD>56 mg</TD><TD>2,8</TD><TD>mg</TD></TR><TR><TD>thiamine</TD><TD>134 mg</TD><TD>6,7</TD><TD>mg</TD></TR><TR><TD>calcium pantothenate</TD><TD>94 mg</TD><TD>4,7</TD><TD>mg</TD></TR><TR><TD>para-aminobenzoic acid</TD><TD>54 mg</TD><TD>2,7</TD><TD>mg</TD></TR><TR><TD>para-hydroxybenzoic acid</TD><TD>54 mg</TD><TD>2,7</TD><TD>mg</TD></TR><TR><TD>2,3-dihydroxybenzoic acid</TD><TD>62 mg</TD><TD>3,1</TD><TD>mg</TD></TR><TR><TD>(NH4)6Mo7</TD><TD>3 nM</TD></TR><TR><TD>H3BO3</TD><TD>400 nM</TD></TR><TR><TD>CoCl2</TD><TD>30 nM</TD></TR><TR><TD>CuSO4</TD><TD>10 nM</TD></TR><TR><TD>MnCl2</TD><TD>80 nM</TD></TR><TR><TD>ZnSO4</TD><TD>10 nM</TD></TR></TBODY></TABLE></DIV></DIV><H2 id="preparation-of-esvelt-medium-for-turbidostat-study-on-the-influence-of-casamino-acids-on-the-bacterial-growth">Preparation of Esvelt medium for turbidostat study on the influence of casamino acids on the bacterial growth</H2>
Esvelt medium was prepared according to the instructions found in (Esvelt et al., 2011). Different amounts of casamino acids were added, evaluating the effect on the growth by OD measurement at different timepoints (one measurement per hour).Several transformations were performed in order to gain the Dickinson plasmids or rather identify the problematic aspects of the triple transformation implemented in week 27. Therefore, MP1 was transformed in different amounts (1 ng and 100 ng) in electrocompetent S1030, as well as in chemocompetent Top10 cells. As a control, pSB1C3 was also transformed in S1030 for investigation of the general competence of the cells. 100 mM Glucose was added to each of the recovery media
<H2 id="restriction-digest-of-mp1-mp4-and-mp6">Restriction digest of MP1, MP4 and MP6</H2>
For digestion of MP1, as well as MP4 and MP6, 0,5 µl of the appropriate enzymes (MP1 – EcoRI; MP4 – BsmBI; MP6 – BsaI and SpeI) were added into 1 µl of the appropriate buffer (EcoRI – Cutsmart; BsmBI – 3.1; BsaI and SpeI – Cutsmart). 1 µl DNA was added to the mixture. All samples were filled up to 10 µl with ddH20 and digested for 30 min at 37 °C, or rather 55 °C for BsmBI.Growth and preparation of MP1, MP4 and MP6Agar stocks previously ordered from addgene were scratched out on LB-agar plates with 25 mM glucose and chloramphenicol resistance. Cells were grown on these plates overnight and picked for single colony growth in 2xYT-medium, containing 25 mM glucose and chloramphenicol resistance as well. Miniprep of plasmid DNA was performed using Qiagen Plasmid Mini Kit following the instructions of the attached protocol. All samples were eluted in ddH20 and concentration of the product was measured using the Implen spectrometer.
For further information about the growth, please visit the awesome_sheet/growth.
<H2 id="pcr-of-phage_backbone_b-for-pam-pace-fragment1-and-fragment2-of-destructase-pace-and-giii-for-enzyme-pace">PCR of Phage_backbone_b for PAM-PACE, fragment1 and fragment2 of destructase PACE and gIII for enzyme-PACE</H2>
PCR was performed in a total volume of 50 µl containing 25 µl 2x Phusion Flash Master mix, 2.5 µl of each 10 µM primer, as well as 1 µl of the appropriate template. 2,5 µl DMSO and 10 µl 5M Betaine were added to the mixture. PCR was performed under the following conditions: Initial denaturation at 98 °C for 10 secs, followed by 35 cycles of denaturation at 98 °C for 1 s, annealing at the respective temperature for 5 s, and primer extension at 72 °C for the associated time. The last extension step was carried out at 72 °C for 1 min. PCR product was evaluated by agarose gel electrophoresis. The annealing temperature for each primer pair was estimated using the <A href="https://www.thermofisher.com/de/en/home/brands/thermo-scientific/molecular-biology/molecular-biology-learning-center/molecular-biology-resource-library/thermo-scientific-web-tools/tm-calculator.html" title="Phusion Flash Tm Calculator">Phusion Flash Tm calculator</A>.
For further information visit the awesome_sheet/PCRs ID35 - 41.
<H2 id="pcr-of-phage_backbone_b-for-pam-pace-fragment1-and-fragment2-of-destructase-pace-and-giii-for-enzyme-pace">PCR of Phage_backbone_b for PAM-PACE, fragment1 and fragment2 of destructase PACE and gIII for enzyme-PACE</H2>
PCR was performed in a total volume of 50 µl containing 25 µl 2x Phusion Flash Master mix, 2.5 µl of each 10 µM primer, as well as 1 µl of the appropriate template. 2,5 µl DMSO and 10 µl 5M Betaine were added to the mixture. PCR was performed under the following conditions: Initial denaturation at 98 °C for 10 secs, followed by 35 cycles of denaturation at 98 °C for 1 s, annealing at the respective temperature for 5 s, and primer extension at 72 °C for the associated time. The last extension step was carried out at 72 °C for 1 min. PCR product was evaluated by agarose gel electrophoresis. The annealing temperature for each primer pair was estimated using the <A href="https://www.thermofisher.com/de/en/home/brands/thermo-scientific/molecular-biology/molecular-biology-learning-center/molecular-biology-resource-library/thermo-scientific-web-tools/tm-calculator.html" title="Phusion Flash Tm Calculator">Phusion Flash Tm calculator</A>.PCR Products 29 (65.9°C), 34 (65.4°C), 35 (66.2°C) and 36 (71.6°C) – awesome_sheet/PCRprodAgarose gel electrophoresisDNA fragments were separated using agarose gel electrophoresis. The gels were prepared by dissolving 1% agarose in 1 % TAE buffer. For visualization of the DNA, 5 µl Roti Safe gel stain was added. Gel electrophoresis was performed for 20 min at 140 V. For image acquisition, BioRad Gel imaging and documentation software was used. Size of the DNA was determined by comparison with the NEB 2-log ladder.Gel extractionGel extraction has been conducted by using Monarch DNA Gel Extraction Kit executing the attached protocol. All samples were eluted in ddH20 and concentration of the product was measured using a nanodrop.
<H2 id="pcr-of-phage_backbone_b-for-pam-pace-fragment1-and-fragment2-of-destructase-pace-and-giii-for-enzyme-pace-different-gene-iii-variants">PCR of Phage_backbone_b for PAM-PACE, fragment1 and fragment2 of destructase PACE and gIII for enzyme-PACE + different gene III variants</H2>
PCR was performed according to the instructions of the previous day (Wednesday 07/12/2017). Several adjustments concerning the annealing time and the extension time were made, because of the non-functional PCR of the last day. In addition, three other PCRs were performed using the same reaction layout as the ones of the previous day.For further information, please visit the awesome_sheet/PCRs, ID 88 - 108Gibson Assembly of AP_Cas9_pSB1A3For Gibson assembly reaction, 10 µl Gibson Assembly MasterMix is mixed with 0.2 – 1 pmol DNA of the different fragments. The reaction mix is filled up to 20 µl with ddH2O. The substance amount is calculated using <A href="http://nebiocalculator.neb.com/#!/dsdnaamt" title="NEBioCalculator">NEBioCalculator</A> with the length of the fragment and the concentration per µl. For assembly of the AP_Cas9_pSB1A3, five different fragments were mixed in equimolar concentrations. According to this, 15 µl of Gibson assembly MM, 13 µl of equimolar DNA fragments and 2 µl of ddH2O was mixed and incubated at 50 °C for 60 min.
Preparation of media supplements for esvelt medium used in first PACE experiment

<DIV class="col-lg-12 col-md-12 col-xs-12 content" style="padding-top: 20px; padding-bottom: 20px"><DIV class="table-slot" style="overflow-x: auto"><TABLE class="table table-bordered mdl-shadow--4dp" xsscleaned="overflow-x: scroll !important"><THEAD><TR><TH>Substance</TH><TH>Stock concentration [mg/ml]</TH><TH>Volume [ml]</TH><TH>Mass [g]</TH></TR></THEAD><TBODY><TR><TD>Ampicillin</TD><TD>50</TD><TD>250</TD><TD>12,5</TD></TR><TR><TD>Chloramphenicol</TD><TD>34</TD><TD>250</TD><TD>8,5</TD></TR><TR><TD>Kanamycine</TD><TD>10</TD><TD>250</TD><TD>2,5</TD></TR></TBODY></TABLE></DIV></DIV>

All antibiotics were steril filtered using 0.45 µl steril filter and a syringe. The stock concentrations were all diluted 1:1000. Due to that, the respective volumes of the 250 ml stock concentrations were added to the media.
<H2 id="supplements-for-esvelt-medium-">Supplements for esvelt medium:</H2><DIV class="col-lg-12 col-md-12 col-xs-12 content" style="padding-top: 20px; padding-bottom: 20px"><DIV class="table-slot" style="overflow-x: auto"><TABLE class="table table-bordered mdl-shadow--4dp" xsscleaned="overflow-x: scroll !important"><THEAD><TR><TH>Substance</TH><TH>1 liter</TH><TH>60 liters</TH><TH>80 liters</TH><TH>140 liters</TH></TR></THEAD><TBODY><TR><TD>Glucose</TD><TD>4.5 g</TD><TD>270 g</TD><TD>360 g</TD><TD>630 g</TD></TR><TR><TD>Sodium Citrat</TD><TD>0.5 g</TD><TD>30 g</TD><TD>40 g</TD><TD>70 g</TD></TR><TR><TD>L-Leucine</TD><TD>0.025 g</TD><TD>1.5 g</TD><TD>2 g</TD><TD>3.5 g</TD></TR><TR><TD>MgSO4</TD><TD>0.025 g</TD><TD>1.5 g</TD><TD>2 g</TD><TD>3.5 g</TD></TR><TR><TD>Casamino acids</TD><TD>5 g</TD><TD>300 g</TD><TD>400 g</TD><TD>700 g</TD></TR></TBODY></TABLE></DIV></DIV>

The volume indicated in the heading marks the amount of ddH2O used for solvation of the substances.One-liter arabinose solution was prepared in a concentration of 5 w/v (50 g/1 l).
<H2 id="dotblot-for-identification-of-phage-contamination">Dotblot for identification of Phage contamination</H2>
Dotblot for investigation of the PACE run2 µl of samples are spotted on the nitrocellulose membrane. A grid is drawn to indicate different samples. The membrane is drown and all non-specific sites are blocked by using 5 % BSA in TBS-Tween for 30 to 60 min. Afterwards, the membrane is incubated with the 1:200 dilution of M13 coat protein antibody which is directly bound to Alexa 488 for 30 min. The dilution was prepared using 2 ml BSA/TBS-Tween and 10 µl antibody. Following this, the membrane was washed three times with TBS-Tween for 5 min and imaged using a BioRad chemidoc with Image Lab 5.2.1.PCRs for mNeonGreen amplification, Ampicillin resistance from pSB1A3 andIn order to optimize the mentioned PCRs listed above, the reaction mix wasSunday 07/16/2017
<H2 id="reference-">Reference:</H2>
Esvelt, K.M., Carlson, J.C., and Liu, D.R. (2011). A system for the continuous directed evolution of biomolecules. Nature 472, 499-503.
</DIV></DIV></DIV><DIV class="panel panel-default"><A data-toggle="collapse" data-parent="#accordion" href="#collapseweek_id-2"><DIV class="panel-heading">
            Calendar Week
30
        </DIV></A><DIV id="collapseweek_id-2" class="panel-collapse collapse"><DIV class="panel-body"><H2>Modeling</H2>
The theoretical model for PALE is built, but not solved yet. Probably finding a solution symbolically is not possible. For that reason the next step is to write a python script that solves the system numerically.A basic model is running in browser, written in javascript with numjys and plotly.js. Further models will be written the same way, old ones translated.
<H2>Software</H2><H2 id="deeprotein">DeeProtein</H2><UL><LI>We implemented 1d convolutions to better capture the 1-dimensional character of protein sequences.</LI><LI>We generated a new test case to harness the accuracy issue for DeeProtein (the CNN was not able to grasp differences in the classes for the toplevel EC dataset). The new test case looks like this:</LI></UL><DIV class="col-lg-12 col-md-12 col-xs-12 content" style="padding-top: 20px; padding-bottom: 20px"><DIV class="table-slot" style="overflow-x: auto"><TABLE class="table table-bordered mdl-shadow--4dp" xsscleaned="overflow-x: scroll !important"><THEAD><TR><TH>EC-number</TH><TH>Function</TH><TH>n(samples)</TH></TR></THEAD><TBODY><TR><TD>1.1.1.-</TD><TD>With NAD(+) or NADP(+) as acceptor</TD><TD>2558</TD></TR><TR><TD>2.1.1.-</TD><TD>Methyltransferases</TD><TD>3739</TD></TR><TR><TD>2.7.1.-</TD><TD>Phosphotransferases with an alcohol group as acceptor</TD><TD>2137</TD></TR><TR><TD>4.2.1.-</TD><TD>Hydro-lyases</TD><TD>3242</TD></TR><TR><TD>6.1.1.-</TD><TD>Ligases forming aminoacyl-tRNA and related compounds</TD><TD>reduced to 2913 (og: 11652)</TD></TR></TBODY></TABLE></DIV></DIV><UL><LI>For the k-merized sequences encoded in a 20-dimensional word embedding the tests did not finish successfull: <A href="https://drive.google.com/open?id=0B6xCRR43l99dWEpjYk1rblo2N1U">KW30_DeeProtein_ONEHOT</A></LI><LI>Accuracies of &gt;86% were obtained for 1-hot encoded sequences: <A href="https://drive.google.com/open?id=0B6xCRR43l99dc19iQjVEanRJd0E">KW30_DeeProtein_EMBED20</A></LI></UL><H2 id="embedding">Embedding</H2>
A first version of the rewritten generate batch function for a more memory efficient version of the embedding is ready. Testing and further improvements have to follow. This makes training an embedding on the full uniprot dataset possible.
<H2>Split Polymerase</H2><UL><LI><EM>Monday</EM>: &quot;Nachschicht&quot; The psp-genIII AP of S1059 (pJC175e) strain was purified and transformed into electro-competent S2060 cells, in order to gain a better readout using plaque assays as phage detection method. S2060 has a beta-Galactosidase integrated in its genome, due to which blue plaques are resulting from a phage infection on soft agar with IPTG.</LI><LI><EM>Tuesday</EM>: General concept of an addition to the awesome_sheet was prepared and adjusted. Further tests and progress report of Catharina are needed for further development.</LI><LI>In order to quantify a possible phage contamination, PCR using two different primer pairs were implemented. Master mix was prepared using OneTaq MM as well as primers JM_074, JM_082, JM_076 and JM_083. For further details about the detection PCR, please visit the PCR sheet of the awesome_sheet.</LI></UL><LI><EM>Wednesday</EM>: &quot;Nachschicht&quot; For quantification of phage contamination of the turbidostat, qPCR was performed following the standard protocol. Primers JM_074 and JM_082 were used for PCR reaction as well as ABI Sybr Green master mix.</LI><DIV class="col-lg-12 col-md-12 col-xs-12 content" style="padding-top: 20px; padding-bottom: 20px"><DIV class="table-slot" style="overflow-x: auto"><TABLE class="table table-bordered mdl-shadow--4dp" xsscleaned="overflow-x: scroll !important"><THEAD><TR><TH style="text-align:center">1-3</TH><TH style="text-align:center">4-6</TH><TH style="text-align:center">7-9</TH><TH style="text-align:center">10-12</TH></TR></THEAD><TBODY><TR><TD style="text-align:center">1</TD><TD style="text-align:center">H20</TD><TD style="text-align:center">T34</TD><TD style="text-align:center">T35</TD><TD style="text-align:center">T36</TD></TR><TR><TD style="text-align:center">2</TD><TD style="text-align:center">T39</TD><TD style="text-align:center">T41</TD><TD style="text-align:center">L1.34</TD><TD style="text-align:center">L1.35</TD></TR><TR><TD style="text-align:center">3</TD><TD style="text-align:center">L1.36</TD><TD style="text-align:center">L1.39</TD><TD style="text-align:center">L1.50</TD><TD style="text-align:center">L2.34</TD></TR><TR><TD style="text-align:center">4</TD><TD style="text-align:center">L2.35</TD><TD style="text-align:center">L2.36</TD><TD style="text-align:center">L2.39</TD><TD style="text-align:center">L2.41</TD></TR><TR><TD style="text-align:center">5</TD><TD style="text-align:center">L2.50</TD><TD style="text-align:center">Pos 10<SUP>-4</SUP></TD><TD style="text-align:center">Pos 10<SUP>-8</SUP></TD><TD style="text-align:center">Pos 10<SUP>-12</SUP></TD></TR><TR><TD style="text-align:center">6</TD><TD style="text-align:center">Neg unv.</TD><TD style="text-align:center">2xYT</TD><TD style="text-align:center">LB</TD><TD style="text-align:center">Glu</TD></TR></TBODY></TABLE></DIV></DIV>

The qPCR did not yield an result. In the following, qPCR was not referred to as a option for phage detection during the PACE runs.As an alternative quantification method, DotBlots were tested. Due to too much noise during detection, they were not considered as detection method as well. For further information, please have a look at Catharina´s weekly report.
<UL><LI><EM>Thursday</EM>: PACE device was cleaned and autoclaved in order to change strains of split-polymerase PACE.</LI><LI><EM>Friday</EM>: PACE device was build again and installed for usage (have a look at the provided video). Stock was thawn and a culture with S1030 + MP1 + Jin177 + Jin172 was grown. Inoculation of the turbidostat was performed and culture was grown until OD 0,6 - 0,8. Thereby, the last phase of our Dickinson-PACE experiment was started.
</LI><LI><EM>Saturday</EM>: PREDCEL - phage related dis-continous evolution concept was designed together with Tobias Stadelmann. Tobias tested the new experimental design by using a 50 ml S1030 + MP1 + Jin69 culture. Culture was prepared using LB-Medium, Cmp, Amp and 25 mM Glucose. The culture was grown until OD 0,6 - 0,8 and separated into nine 5 ml cultures (in 14 ml PP tubes). Afterwards, they were infected with different amount of phages (10 µl, 100 µl and 1 ml) to evaluate which of the amounts has to be used for following experiments. Beyond that, the infected cultures were centrifuged after different amounts of time (20, 30 or 60 min). The supernatant containing the phages was then transfered into a new 5 ml culture.
</LI></UL><H2 id="general-concept-of-predcel-">General concept of PREDCEL:</H2><EM>Idea:</EM> Using PACE in an experimental design, suitable to be carried out by every iGEM Team.PREDCEL describes a manual implementation of PACE.The incubation of E.coli with phages in a reaction tube (max. 14 ml) using a normal incubator should be performed.First we wanted to test PREDCEL under as equivalent reaction conditions as possible (Dickinson-PREDCEL).
PREDCEL allows us to test more “PACE-runs” manually and in parallel. It also enables to test different reaction conditions (different media, inducer, etc.) while having asimple handling, suitable for every iGEM Team.
<H2 id="pre-experiments-">Pre-experiments:</H2><OL><LI>Test of conditions - amount of phage supernatant to be transferred and time until transfer between each &quot;run&quot;</LI><LI>Results: 1 ml phage supernatant should be transferred</LI><LI>should prevent washout of phages in experimental set-up</LI><LI>time independent - BUT to small reproduction size for a validated statement about the amount of time needed for propagation (20, 30, 60 min)</LI><LI>according to Julius phages need 10 min to infect and reproduce themselves for the first time</LI><LI>analysis was performed using plaque assays</LI><LI>Test of medium which should be used for experiments</LI><LI>LB, 2YT and Esvelt medium</LI><LI>infectivity of the phages should be observed in dependence of the medium</LI><LI>analysis was performed using plaque assays</LI><LI>the three different times of experiment number 1. were adopted (20, 30, 60 min)</LI><LI>six rounds were implemented for 2YT and LB</LI><LI>three rounds were implemented for Esvelt medium</LI></OL><H2 id="experiments-to-go-">Experiments to go:<OL><LI>Dickinson-PACE reproduction using the real phage (unevolved)</LI><LI>using new strains with Jin69_xxxx_with_LuxAB and MP1, Jin177_LuxAB + Jin172_LuxAB + MP1, Jin177_LuxAB + Jin173_LuxAB + MP1</LI><LI>Aim: measure LuxAB activity and sequencing of Dickinson phage for proof of concept</LI><LI>MP4 or MP6 activity should be tested as well</LI><LI>First experiment of PAM-PACE functionality</LI><LI>does the PAM-PACE circuit work like it is supposed to</LI></OL></H2><H2 id="prerequesite-">Prerequesite:</H2><UL><LI>knowledge about time and medium</LI></UL><UL><LI><EM>Sunday</EM>: Qiagen Midi Prep protocol for M13 phages was implemented for purification of our selection phage. Following this further, oxford nanopore standard protocol for testing the MinION sequencing was tested for feasibility. Several kits, supplied by NEB were ordered for preparation of the test library. A cooperation with AG Conrad of the Eils Labs should be started. They work with single cell sequencing and should therefore know a lot about library preparation and NGS.</LI></UL></DIV></DIV></DIV><DIV class="panel panel-default"><A data-toggle="collapse" data-parent="#accordion" href="#collapseweek_id-3"><DIV class="panel-heading">
            Calendar Week
31
        </DIV></A><DIV id="collapseweek_id-3" class="panel-collapse collapse"><DIV class="panel-body"><H2>Modeling</H2>
We have a first set of equations for the PREDCEL <STRONG>Model</STRONG>, but did not solve it yet. Probably finding a solution symbolically is not possible. For that reason the next step is to write a python script that solves the system numerically.<STRONG>Modelling</STRONG> the specialised variants did not start this week, as well as the overview.However we have a basic model running in the browser, written in javascript with numjys and plotly.js. The next set of models should be written the same way, also the old ones have to be translated.
<H2>PACE</H2>
The current PACE setup was documented. Future PACE setups were discussed. Especially a setup that allows for host strains to be switched would be interesting for PamPACE. The idea is to use two turbidostats alternating. The host supply for the lagoons has a Y-connector that potentially connects both Turbidostats to the lagoons. Only one of the two tubings to the lagoons is allowed to be open at a time. When strains are switched, the block is switched. The Turbidostat that is not in use anymore, can be disconnected, the resulting open Luer-connector has to be sprayed with incidin, H2O2 or whatever kills phage and bacteria. Then the unused Turbidostat can be cleaned, autoclaved and reinstalled. It should be possible to switch strains every 12h theoretically. This setup needs either a second bottle of medium or changed lids, that hold two tubings that collect medium. New tubing to the turbidostat waste is needed.This setup makes it possible to switch back and forth between two strains, that can live on different media.The overall principle of a chemostat is that the population is kept constant by adjusting the amount of an essential resource. This helps to save medium but decreases the metabolic activity of the E. coli which may be problematic for PACE. But there are papers (Dickinson <EM>et al.</EM>) that used chemostats. Probably their protocol can be used with a few adjustments.
<H2>Phage_Propagation_and_Quantification</H2><H2 id="phage-quick-test">Phage Quick-Test</H2>
To find an easy and quick detection and quantification method for phages, the activity of beta-galactosidase, which is under control of a psp-promoter can be measured.Therefore, 100 µl of the <EM>E. coli</EM> strain S2208, stock ID: 47 (OD600 ~ 0.5) was pipetted to 8 wells of a 96-well plate and 1 µl X-gal stock solution (40 mg/ml) was added to all wells except of the first. Different volumes of phage suspension were added according to the following table:

<DIV class="col-lg-12 col-md-12 col-xs-12 content" style="padding-top: 20px; padding-bottom: 20px"><DIV class="table-slot" style="overflow-x: auto"><TABLE class="table table-bordered mdl-shadow--4dp" xsscleaned="overflow-x: scroll !important"><THEAD><TR><TH>Number</TH><TH>Phage</TH><TH>Volume [µl]</TH><TH>X-gal</TH></TR></THEAD><TBODY><TR><TD>1</TD><TD>Dickinson_Phage_001</TD><TD>10</TD><TD>no</TD></TR><TR><TD>2</TD><TD>No Phage</TD><TD>-</TD><TD>yes</TD></TR><TR><TD>3</TD><TD>Dickinson_Phage_001</TD><TD>1</TD><TD>yes</TD></TR><TR><TD>4</TD><TD>Dickinson_Phage_001</TD><TD>10</TD><TD>yes</TD></TR><TR><TD>5</TD><TD>Positive Ctrl</TD><TD>1</TD><TD>yes</TD></TR><TR><TD>6</TD><TD>Positive Ctrl</TD><TD>10</TD><TD>yes</TD></TR></TBODY></TABLE></DIV></DIV>

The 96-well plate was placed in the plate reader and the phage detection protocol was started measuring the absorption at 600 nm every 10 min for 2 h.There was no significant difference in absorption between the infected wells and the non-infected control as well as the well without X-gal. The results demonstrated that the conversion of X-gal is not a suitable method for phage quantification by spectroscopy as the product absorbs light at 650 nm, which interferes with the absorbance of cells. This explains the continuous increase of absorption during 2 h of cultivation.
Further tests are planned using the beta-galactosidase substrate ONPG as the product absorbs at 420 nm.
<H2 id="plaque-assay-test">Plaque Assay Test</H2>
To simplify the method, plaque assays were performed using only top agar. Top agar (0.7 %) supplemented with carbenicillin and X-gal stock solution was mixed with <EM>E. coli</EM> (ID:47) and phage suspension, according to the protocol: Blue Plaque Assay, and was poured into an empty well of a six well plate.Plate layout:

<DIV class="col-lg-12 col-md-12 col-xs-12 content" style="padding-top: 20px; padding-bottom: 20px"><DIV class="table-slot" style="overflow-x: auto"><TABLE class="table table-bordered mdl-shadow--4dp" xsscleaned="overflow-x: scroll !important"><THEAD><TR><TH>Number</TH><TH>Phage</TH><TH>Dilution</TH><TH>Comments</TH></TR></THEAD><TBODY><TR><TD>1</TD><TD>Positive Ctrl</TD><TD>10<SUP>-11</SUP></TD><TD>-</TD></TR><TR><TD>2</TD><TD>Positive Ctrl</TD><TD>10<SUP>-12</SUP></TD><TD>sample was frozen</TD></TR><TR><TD>3</TD><TD>-</TD><TD>-</TD><TD>Cells Only</TD></TR><TR><TD>4</TD><TD>-</TD><TD>-</TD><TD>Top Agar Only</TD></TR></TBODY></TABLE></DIV></DIV>

The plate was stored at 37 °C overnight.On the next day top agar with <EM>E. coli</EM> was slightly blue, but only the positive control 10<SUP>-11</SUP>showed 12 clear plaques. This proves the assumption that frozen phage samples are not longer infectious.
<H2>Software</H2><H2 id="deeprotein">DeeProtein</H2><UL><LI>We implemented 1d convolutions to better capture the 1-dimensional character of protein sequences.</LI><LI>We generated a new test case to harness the accuracy issue for DeeProtein (the CNN was not able to grasp differences in the classes for the toplevel EC dataset). The new test case looks like this:</LI></UL><DIV class="col-lg-12 col-md-12 col-xs-12 content" style="padding-top: 20px; padding-bottom: 20px"><DIV class="table-slot" style="overflow-x: auto"><TABLE class="table table-bordered mdl-shadow--4dp" xsscleaned="overflow-x: scroll !important"><THEAD><TR><TH>EC-number</TH><TH>Function</TH><TH>n(samples)</TH></TR></THEAD><TBODY><TR><TD>1.1.1.-</TD><TD>With NAD(+) or NADP(+) as acceptor</TD><TD>2558</TD></TR><TR><TD>2.1.1.-</TD><TD>Methyltransferases</TD><TD>3739</TD></TR><TR><TD>2.7.1.-</TD><TD>Phosphotransferases with an alcohol group as acceptor</TD><TD>2137</TD></TR><TR><TD>4.2.1.-</TD><TD>Hydro-lyases</TD><TD>3242</TD></TR><TR><TD>6.1.1.-</TD><TD>Ligases forming aminoacyl-tRNA and related compounds</TD><TD>reduced to 2913 (og: 11652)</TD></TR></TBODY></TABLE></DIV></DIV><UL><LI>For the k-merized sequences encoded in a 20-dimensional word embedding the tests did not finish successfull: <A href="https://drive.google.com/open?id=0B6xCRR43l99dWEpjYk1rblo2N1U">KW30_DeeProtein_ONEHOT</A></LI><LI>Accuracies of &gt;86% were obtained for 1-hot encoded sequences: <A href="https://drive.google.com/open?id=0B6xCRR43l99dc19iQjVEanRJd0E">KW30_DeeProtein_EMBED20</A></LI></UL><H2>Split Polymerase</H2><H2 id="dickinson-pace-">Dickinson-PACE:</H2>
Several plaque PCRs were performed using plaque assays of Dickinson-PACE samples. The sequence of interest was amplified using PCR reaction and sended to GATC for sequencing analysis. Sequencing revealed that we got an evolved phage from the Dickinson lab. This phage obviously not showed any mutations or differences after the seven days of evolution in our PACE device.
As results we can retain the facts that we could not determine the danger of washout. On top of that, we have learned how to handle the turbidostats, as well as the lagoons. Beyond that, we are now familiar with the workflow the apparature. We know how to propagate phages and moreover avoid turbidostat contamination with phages.
<H2 id="predcel-">PREDCEL:</H2>
First PREDCEL runs and the associated plaque assays showed that we are not running into wash out for all of the three different amounts of transfered supernatant (10 µl, 100 µl and 1000 µl). On top of that, the phages are propagating well, as we can have similar phage titers for the different timepoints (20, 30 and 60 min).
Our second experiment should evaluate, whether 2YT, LB or Esvelt-Medium should be used for further experiments. Since Kevin Esvelt reported in his PhD thesis, that medium influences the amount of F-pili an E.coli expresses. This should be directly correlated to the number of phages which can infect one of the E.colis, since M13 phages are using f-pili for their transport in and out of the cell.
The results of these tests showed a high number of single plaques for 10-2 and 10-4 dilutions of the supernatant from LB medium. In contrast, 2YT and Esvelt medium showed uncountable numbers for these dilutions, with a slightly higher propagation using Esvelt medium. Exceptions have been made, since we had poblemes with the negative control in all cases. Since we want to observe the general feasibility of each medium to propagate phages, this should not be important in this case.
<H2 id="reporter-systems-">Reporter systems:</H2>
Four different reporter systems should be integrated in our assesory plasmids. mNeonGreen is currently available in all our APs, while the Nano Luciferase, provided by Promega, will be cloned in one of Jan´s APs. This reporter should supply a bright signal while having a moderate size. LuxAB was former used in the Bt toxine paper from Badran et al.. All our important strains contain the LuxCDE genes in their genome (S1059, S1030, S2060, S2208). We should consider it as a reporter system. At last, LacZ is considered as another possibility for a simple readout before infecting the laggons. A possible reaction could include the addition of phages to a culture which contains AP with LacZ. After an amount of time, for example ten minutes, phage infections should be observed.
For the next week, cloning of the different reporter systems is planed.
<H2 id="minion-sequencing-">MinION sequencing:<UL><LI>spoke to Jan-Philipp Mallm who uses the MinION in AG Rippe</LI><LI>he is sitting in the fourth floor (R425 or something)</LI><LI>told me that the device is, like we already know, leaky in terms of error probability (around every tenth base is called incorrect)</LI><LI>first he was pessimistic to detect small mutations</LI><LI>BUT one could calculate the number of random mutations by comparing the occured mutations over time</LI><LI>one should find the <STRONG>directed</STRONG> mutations by the number of occurences and the occurence over time</LI></UL><LI>due to that we should not need barcording - is not useful</LI><LI>maybe barcoding for different sample pools from different timepoints - simultanous sequencing of several samples</LI><LI>99.7 % accuracy is possible using a 50x coverage (Oxford nanopore)</LI><OL><LI>DNA Lobind tubes from (n irgendwas) are essential and will be needed</LI><LI>first part of each oxford nanopore protocol is for the generation of cDNA</LI><LI>we should start with generating phage DNA (at least 1 µg) - in a first experiment Max purificated 3 µg of phage ssDNA</LI><LI>Jan told us to use random priming for generation of dsDNA for sequencing</LI><LI>NEB Ligase Kit is needed as well as the TA-tailing Kit, which we have samples for (RNA library prep kit for Ilumina by NEB) in -20 °C</LI><LI>we should simply follow the protocol which is straight forward in this concern</LI></OL></H2><H2 id="to-do-">To do:</H2><UL><LI>order all the needed stuff (with Roland´s permission)</LI><LI>DNA low binding tubes</LI><LI>NEBNext Ultra II End-repair/dA-tailing module</LI><LI>NEB Blunt/TA Ligase Master Mix</LI><LI>ThermoFisher Scientific Dynabeads</LI><LI>Klenow Fragment</LI><LI>Random hexamers</LI></UL><LI>think about a possibility to use</LI><H2>Tetracyclin_Destructase</H2>
Destructase PACE Cloning I:
<H2 id="-weekly-report-23-07-17-30-07-17-pp-"><STRONG>Weekly Report 23.07.17 - 30.07.17 PP</STRONG>
Destructase PACE consists, as all the other PACE experiments, of three plasmids, namely the accessory plasmid (AP), the selection phage (SP), and the mutagenesis plasmid (MP).Design of the AP and SP is described in the following lines.
</H2><H2 id="sp">SP</H2>
The gene of interest in this PACE run is an anhydrotetracycline destructase named Tet(x) from Bacterioides fragilis.We aim at destructing Tet(x) and thereby increasing anhydrotetracycline (aTC) concentration and gIII expression.  The sequence was codon optimized for E. coli and a strong, self-created RBS was calculated (Salis Lab). The selection phage is re-assembled because stop-codon creating mutations within tet(x) should be prevented.Therefore gII which is essential for phage replication by introducing a nick into the circular phage genome <X-REF>lin1972role</X-REF> was linked by an GGSGGS directly behind tet(x).
<H2 id="ap">AP</H2>
For phage and anhydrotetracycline dependent geneIII (gIII) expression a fusion promoter (Carlson et. al) consisting of a phage shock promoter (psp) and anhydrotetracycline induced domain was used (Ppsp-tet).Gene III is followed by mNeonGreen, a reporter fluorophore to measure gIII expression level. The backbone consists of a E. coli origin of replication (ColE1 ori) and an ampicilline resistance. A non-coding sequence has been added between backbone and Ppsp-tet because all fragments are assembled by Gibson assembly following the cloning standard.
<H2 id="destructase-pace-cloning-ii-">Destructase PACE Cloning II:</H2><H2 id="-1-ap-"><STRONG>1. AP</STRONG></H2><UL><LI>The psp-tet promoter sequence is taken from Carlson et. al (Negative Selection) and was ordered as a gblock. psp-tet promoter is flanked by gibson overlap 1 on the 5´end and gibson overlap 2 on the 3´end.
</LI><LI>GeneIII is extracted from a Dickinson plasmid (Jin_177:_p15A_origin_pG). On the 5´end geneIII is concatenated to gibson overlap 2 and on the 3´end to gibson overlap 3.
</LI><LI>mNeonGreen or any other reporter system is taken from the PAM-PACE accessory plasmid. mNeonGreen gblock includes gibson overlaps 3 and 4 at 5´and 3´end.
</LI><LI>The backbone includes a ColE1 ori and AmpR and is extracted from pSB1A3. By using primers with extensions gibson overlap 4 is added to the 5´end and gibson overlap 5 to the 5´end.
</LI><LI>The non-coding fragment between Gibson overlap 5 and 1 is extracted from the iGEM Distribution Kit and contains 256 bp from BBa_K808003, a small subunit B1 of the tripartite tricarboxylate transporter family.With primers PP_005_fwd and PP006_rev a fragment from coding sequence is extracted (bp 78-334). The extracted fragment consists of 265 bp and with the named primers gibson overlap 5 is added to the 5´end and gibson overlap 1 is added to the 3´end.(PCR Protocol PP_001)
</LI></UL>
Primers 10 µM
Template 10ng/µl

<DIV class="col-lg-12 col-md-12 col-xs-12 content" style="padding-top: 20px; padding-bottom: 20px"><DIV class="table-slot" style="overflow-x: auto"><TABLE class="table table-bordered mdl-shadow--4dp" xsscleaned="overflow-x: scroll !important"><THEAD><TR><TH style="text-align:center">Name</TH><TH style="text-align:right">Volume [µl]</TH></TR></THEAD><TBODY><TR><TD>Template</TD><TD style="text-align:center">BBa_K808003</TD><TD style="text-align:right">0.5</TD></TR><TR><TD>Primer forward</TD><TD style="text-align:center">PP_007_fwd</TD><TD style="text-align:right">2.5</TD></TR><TR><TD>Priner reverse</TD><TD style="text-align:center">PP_008_rev</TD><TD style="text-align:right">2.5</TD></TR><TR><TD>Master Mix</TD><TD style="text-align:center">Phusion Flash 2x</TD><TD style="text-align:right">25</TD></TR><TR><TD>ddH2O</TD><TD style="text-align:center">ad 50 µl</TD><TD style="text-align:right">19.5</TD></TR></TBODY></TABLE></DIV></DIV>

The PCR product was purified with die Quiagen PCR Purification Kit.
To assemble the five fragments using the gibson overlaps a CPEC (circular polymerase extension cloning)was performed.
<UL><LI>low ramping temperature 0.5 °C/s</LI><LI>Q5</LI></UL><DIV class="col-lg-12 col-md-12 col-xs-12 content" style="padding-top: 20px; padding-bottom: 20px"><DIV class="table-slot" style="overflow-x: auto"><TABLE class="table table-bordered mdl-shadow--4dp" xsscleaned="overflow-x: scroll !important"><THEAD><TR><TH style="text-align:center">Name</TH><TH style="text-align:right">amount [ng]</TH></TR></THEAD><TBODY><TR><TD>backbone</TD><TD style="text-align:center">pSB1A3</TD><TD style="text-align:right">75</TD></TR><TR><TD>insert 1</TD><TD style="text-align:center">BBa_K808003</TD><TD style="text-align:right">2x equimolar</TD></TR><TR><TD>insert 2</TD><TD style="text-align:center">psp-tet promoter</TD><TD style="text-align:right">2x equimolar</TD></TR><TR><TD>insert 3</TD><TD style="text-align:center">geneIII</TD><TD style="text-align:right">equimolar</TD></TR><TR><TD>insert 4</TD><TD style="text-align:center">mNeonGreen</TD><TD style="text-align:right">equimolar</TD></TR></TBODY></TABLE></DIV></DIV><DIV class="col-lg-12 col-md-12 col-xs-12 content" style="padding-top: 20px; padding-bottom: 20px"><DIV class="table-slot" style="overflow-x: auto"><TABLE class="table table-bordered mdl-shadow--4dp" xsscleaned="overflow-x: scroll !important"><THEAD><TR><TH>step</TH><TH style="text-align:center">temperature [°C]</TH><TH style="text-align:right">time [s]</TH></TR></THEAD><TBODY><TR><TD>initial denaturation</TD><TD style="text-align:center">98</TD><TD style="text-align:right">30</TD></TR><TR><TD>denaturation</TD><TD style="text-align:center">98</TD><TD style="text-align:right">20</TD></TR><TR><TD>annealing</TD><TD style="text-align:center">55</TD><TD style="text-align:right">20</TD></TR><TR><TD>elongation</TD><TD style="text-align:center">72</TD><TD style="text-align:right">150</TD></TR><TR><TD>final extension</TD><TD style="text-align:center">72</TD><TD style="text-align:right">300</TD></TR></TBODY></TABLE></DIV></DIV>

To test whether the CPEC worked out the CPEC product was transformed by chemical transformation (iGEM protocol) into TOP10 cells and plated out on ampicilline agar plates. The plates incubated at 37 °C overnight but after 20 hours of incubation no colonies grew up.Because the CPEC did not work after several attempts the whole fragment between gibson overlap 2 and 5 was amplified from PAM-PACE accessory plasmid (AP_Cas9_pSB1A3_NGG_SD4) following this PCR protocol

<DIV class="col-lg-12 col-md-12 col-xs-12 content" style="padding-top: 20px; padding-bottom: 20px"><DIV class="table-slot" style="overflow-x: auto"><TABLE class="table table-bordered mdl-shadow--4dp" xsscleaned="overflow-x: scroll !important"><THEAD><TR><TH style="text-align:center">Name</TH><TH style="text-align:right">Volume [µl]</TH></TR></THEAD><TBODY><TR><TD>Template</TD><TD style="text-align:center">AP_Cas9_pSB1A3_NGG_SD4</TD><TD style="text-align:right">0.5</TD></TR><TR><TD>Primer forward</TD><TD style="text-align:center">JM_053</TD><TD style="text-align:right">1.25</TD></TR><TR><TD>Priner reverse</TD><TD style="text-align:center">JM_073</TD><TD style="text-align:right">1.25</TD></TR><TR><TD>Master Mix</TD><TD style="text-align:center">Q5 Master Mix 2x</TD><TD style="text-align:right">12.5</TD></TR><TR><TD>ddH2O</TD><TD style="text-align:center">ad 25 µl</TD><TD style="text-align:right">9.5</TD></TR></TBODY></TABLE></DIV></DIV><DIV class="col-lg-12 col-md-12 col-xs-12 content" style="padding-top: 20px; padding-bottom: 20px"><DIV class="table-slot" style="overflow-x: auto"><TABLE class="table table-bordered mdl-shadow--4dp" xsscleaned="overflow-x: scroll !important"><THEAD><TR><TH>step</TH><TH style="text-align:center">temperature [°C]</TH><TH style="text-align:right">time [s]</TH></TR></THEAD><TBODY><TR><TD>initial denaturation</TD><TD style="text-align:center">98</TD><TD style="text-align:right">30</TD></TR><TR><TD>denaturation</TD><TD style="text-align:center">98</TD><TD style="text-align:right">10</TD></TR><TR><TD>annealing</TD><TD style="text-align:center">66</TD><TD style="text-align:right">20</TD></TR><TR><TD>elongation</TD><TD style="text-align:center">72</TD><TD style="text-align:right">150</TD></TR><TR><TD>final extension</TD><TD style="text-align:center">72</TD><TD style="text-align:right">120</TD></TR></TBODY></TABLE></DIV></DIV>

The PCR did not work out and therefore, the same PCR was performed with different concentrations of Betain (0,5 M and 1 M) and DMSO (3 %, 5 % and 10 %)This did not work as well and therefore, new primers are designed to anneal at 72 °C (PP_013 and PP_014). The PCR worked out and a gel extraction was performed and the PCR product AP_destructase_g2-g5_PP013-PP014 is used for further CPEC.
<H2>weekly_report_organosilicones</H2><H2 id="organosilicones">Organosilicones</H2><H2 id="weeks-before">weeks before</H2>
After we decided in our group to pursue the challenge to build Si-C bonds in vivo building on the intriguing paper &gt;„Directed evolution of cytochrome c for carbon–silicon bond formation Bringing silicon to life“ by Francis Arnold (2016) and use PACE to lastly modify the catalytic enzyme, we began forming a strategy. Combining various literature methods, our plan more and more assumed a form. We wanted to use a riboswitch, detecting the Si-C bond containing small molecule <EM>Rma</EM> cytochrome c forms, to logically link the specificity and activity of our evolving protein to the Phage production.To start this, we wanted to use the mutated version of cyt c  to enantio- and chemoselectively produce our target molecule.Next, the so generated molecule should enable the direct evolution of our RNA riboswitch with the method SELEX (Systematic Evolution of Ligands by EXponential Enrichment). Lastly, the respective implementation of the riboswitch and cyt c DNA in PACE should give rise to new mutants with an improved catalysing mechanisms for our desired molecule.
The educt for the reaction requires a molecule with a diazo functional group plus one with a Si-H bond. The ethyl-diazo propionate forms, under nitrogen split-off, a carben intermediate with the iron ion of the heme cofactor of the cytochrome c. This in turn insertes itself specifically in the Si-H bond, giving rise to a product with a chiral carbon which is now covalently bound to the Si atom of the other molecule.
After we consulted Reiner Tacke for a deeper insight of the relevance of Si-C bonds in industry, we decided that especially enantio- and chemselective biocatalyst are of high importance, narrowing the pool of possible substrates for our reaction. Me-EDA, are used in the paper, was commercially available, whereas 4-(dimethylsilyl)-anilin is not. The expert opinion of various professors of the chemical institute of our university implied that the synthesis is to dangerous for our team internal biochemist, who has a bit of organic lab experience. So we lastly found a group willing to cooperate and synthesising our molecule. Later on they also supported us regarding general chemical matters of this subproject.   
For the SELEX part, we got help of Dr. Murat Sunbül from the Jäschke Lab, who is experienced in aptamer development.
<H2 id="week-10-16-07-2107">week 10-16.07.2107</H2>
The plan was to produce the enantioselective molecule we want to SELEX against later on. To start with, we needed the protein. As described in the Supplementary Materials of &gt;&quot;Directed evolution of cytochrome c for carbon–silicon bond formation Bringing silicon to life“, further on referred to as S.M. (p.3 (B)), we expressed and purified the <EM>Rma</EM> cytochrome c TDE (triple mutant. Thereby IPTG initiated the expression of the T7 Polymerase in the bacterial strain BL21 (DE3), in turn triggering the production of cyt c from  the pet22(b) plasmid. Purification was facilitated though the apparent colour of the heme cofactor of the protein, which coloured the sample brown-red (Bild1) and therefore enabled an easy identification of the wanted fractions.
<H2 id="week-17-23-07-1207">week 17-23.07.1207</H2>
The purity of the previously extracted protein was determined via a NuPAGE () and the concentration through the ferrous assay (S.M). The reactions of Me-EDA and 4-(dimethylsilyl)-anilin were performed with purified cyt c as well as whole cell solution. Both samples were treated as described in the &gt;“Preparative-Scale Whole-Cell Biocatalytic Reaction“, whereby anaerobic conditions were mimicked through Nitrogen Gas washing of the reaction vial. Purification of the product through silica column chromatography delivered 25 fractions.
Simultaneously, for the SELEX part, the RNA-library given to us by the Jäschke Lab was amplified. Therefore, a large scale PCR was conducted with primer ordered by IDT, in whole taking a volume of 10 mL (Bild). Next, the DNA was purified through Ethanol precipitation.
<H2 id="week-24-30-07-2017">week 24-30-07.2017</H2>
The from silica column chromatography obtained fractions were analysed first with DC and building on that, the most prosperous fraction was sent to the cooperating  Grebe group for GC-MS. The spectra was analysed with the help of a chemist from the group, Fabian Ebner, revealing that the reaction did not successfully take place as only educt was detected by the GC-MS. After further informing ourselves about the exact reaction mechanisms and the conditions as far as published in the technical literature, we derived at two suspicions for the failure of there reaction. One was that the incativation of the catalytic activity of the protein through carben transfer to a protein side chain as described in &gt;“Identification of mechanism-based inactivation in P450-catalyzedcyclopropanation facilitates engineering of improved enzymes“ (H.Renata, 2016). Here the author states, that &gt;“inactivation of whole-cell catalyst was previously found to be slower than purified enzyme“.The other approach suggested that our protein was in its oxidised state due to lack of sufficient oxygen exclusion and no addition of reduction reagent to the samples, implying that Carben Intermediat formation is only possible with the reduced Heme group.
For this reason, a set of test reactions were performed with the commercially available Dimethylphenylsilan as a replacement of the 4-(dimethylsilyl)-anilin, always requiring custom synthesis. Hereby, we referred to the procedure of &gt;“Lysate-small scale biocatalytic reactions“, mainly differing from the previous experiment by the addition of Dithionite solution as a reducing agent and GC Septum vials as reaction vials. Besides four samples with purified cyt c under various conditions, two samples still contained the whole cell solution.
</DIV></DIV></DIV><DIV class="panel panel-default"><A data-toggle="collapse" data-parent="#accordion" href="#collapseweek_id-4"><DIV class="panel-heading">
            Calendar Week
32
        </DIV></A><DIV id="collapseweek_id-4" class="panel-collapse collapse"><DIV class="panel-body"><H2>Modeling</H2>
Interactive modeling works, the information on hover does not work yet. Interactive modeling is expanded to plotting user data in the models.Numeric modeling of PredCel is implemented but only generates oscillations so far.
<H2>Phage_Propagation_and_Quantification</H2><H2 id="detection-of-destructase-and-cas9-phages">Detection of Destructase and Cas9 Phages</H2>
As we assumed, phage production is very inefficient Blue Plaque Assays were performed using morephage suspension. 100 µl <EM>E. coli</EM> suspension (stock ID:47) and 200 µl phage suspension was mixed with 1500 µl top agar supplemented with carbenicillin and X-gal stock solution and plated on LB/Amp-plates.

<DIV class="col-lg-12 col-md-12 col-xs-12 content" style="padding-top: 20px; padding-bottom: 20px"><DIV class="table-slot" style="overflow-x: auto"><TABLE class="table table-bordered mdl-shadow--4dp" xsscleaned="overflow-x: scroll !important"><THEAD><TR><TH>Plate</TH><TH>Phage</TH></TR></THEAD><TBODY><TR><TD>1</TD><TD>SP_Destructase (GG1)</TD></TR><TR><TD>2</TD><TD>SP_Destructase (GG2)</TD></TR><TR><TD>3</TD><TD>SP_dCas9 (GG1)</TD></TR><TR><TD>4</TD><TD>Positive Ctrl</TD></TR><TR><TD>5</TD><TD>Negative Ctrl</TD></TR></TBODY></TABLE></DIV></DIV>

Positive control exhibited plaques and negative control was clear, but only one plaque was visible for SP_Destructase (GG2).
Blue Plaque Assay was repeated using a 15 cm dish. 500 µl <EM>E. coli</EM> suspension and 1 ml phage suspension was mixed with 5 ml top agar supplemented with carbenicillin and X-gal stock solution and plated on 15 cm LB/Amp-plates.

<DIV class="col-lg-12 col-md-12 col-xs-12 content" style="padding-top: 20px; padding-bottom: 20px"><DIV class="table-slot" style="overflow-x: auto"><TABLE class="table table-bordered mdl-shadow--4dp" xsscleaned="overflow-x: scroll !important"><THEAD><TR><TH>Plate</TH><TH>Phage</TH></TR></THEAD><TBODY><TR><TD>1</TD><TD>SP_Destructase (GG1)</TD></TR><TR><TD>2</TD><TD>SP_Destructase (GG2)</TD></TR><TR><TD>3</TD><TD>SP_dCas9 (GG1)</TD></TR><TR><TD>4</TD><TD>SP_dCas9 (GG2)</TD></TR><TR><TD>5</TD><TD>Positive Ctrl</TD></TR><TR><TD>6</TD><TD>Negative Ctrl</TD></TR></TBODY></TABLE></DIV></DIV>

On the next day, positive control exhibited plaques and negative control had no plaques. SP_dCas9 (GG2) showed around 50 plaques, but no plaques were visible on the other plates.
<H2>Software</H2><H2 id="generation-of-datasets-for-go-term-classification">Generation of datasets for GO-term classification</H2><UL><LI>Filtered the Swissprot file and rewrote it to a .csv containing: name, seq, EC_nr, GO-terms</LI><LI>Then GO-term specific .csv files were generated. The GO-DAG (directed acyclic graph) was completed for each GO annotation in each protein, and the respective protein was then added to all GO-term specific files, that it has GO-annotations for.</LI><LI>Extremely generic GO-labels like: GO:0003674 molecular function, or GO:0005488 binding were excluded.</LI></UL><H2 id="testing-of-the-padded-input-pipeline">Testing of the padded input-pipeline</H2><UL><LI>A lot more noise through the different padding sizes</LI><LI>The system is a lot more complicated as we're feeding directly from the queues and therefore have two different graphs, that share parameters

<DIV class="col-lg-12 col-md-12 col-xs-12 content" style="padding-top: 20px; padding-bottom: 20px"><DIV class="table-slot" style="overflow-x: auto"><TABLE class="table table-bordered mdl-shadow--4dp" xsscleaned="overflow-x: scroll !important"><THEAD><TR><TH style="text-align:center">Name</TH><TH style="text-align:right">Volume [µl]</TH></TR></THEAD><TBODY><TR><TD>Template</TD><TD style="text-align:center">SP_CBXTAL supernatant</TD><TD style="text-align:right">2</TD></TR><TR><TD>Primer forward</TD><TD style="text-align:center">PP_001/PP_003</TD><TD style="text-align:right">1.0</TD></TR><TR><TD>Priner reverse</TD><TD style="text-align:center">PP_002/PP_004</TD><TD style="text-align:right">1.0</TD></TR><TR><TD>Master Mix</TD><TD style="text-align:center">Q5 Master Mix 2x</TD><TD style="text-align:right">10</TD></TR><TR><TD>ddH2O</TD><TD style="text-align:center">ad 25 µl</TD><TD style="text-align:right">6</TD></TR></TBODY></TABLE></DIV></DIV><DIV class="col-lg-12 col-md-12 col-xs-12 content" style="padding-top: 20px; padding-bottom: 20px"><DIV class="table-slot" style="overflow-x: auto"><TABLE class="table table-bordered mdl-shadow--4dp" xsscleaned="overflow-x: scroll !important"><THEAD><TR><TH>step</TH><TH style="text-align:center">temperature [°C]</TH><TH style="text-align:right">time [s]</TH></TR></THEAD><TBODY><TR><TD>initial denaturation</TD><TD style="text-align:center">98</TD><TD style="text-align:right">300</TD></TR><TR><TD>10x</TD></TR><TR><TD>denaturation</TD><TD style="text-align:center">98</TD><TD style="text-align:right">20</TD></TR><TR><TD>annealing</TD><TD style="text-align:center">64</TD><TD style="text-align:right">20</TD></TR><TR><TD>elongation</TD><TD style="text-align:center">72</TD><TD style="text-align:right">120</TD></TR><TR><TD>25x</TD></TR><TR><TD>denaturation</TD><TD style="text-align:center">98</TD><TD style="text-align:right">300</TD></TR><TR><TD>annealing</TD><TD style="text-align:center">72</TD><TD style="text-align:right">20</TD></TR><TR><TD>elongation</TD><TD style="text-align:center">72</TD><TD style="text-align:right">120</TD></TR><TR><TD>final extension</TD><TD style="text-align:center">72</TD><TD style="text-align:right">300</TD></TR></TBODY></TABLE></DIV></DIV>

Both fragments were successfully amplified as to see gel electrophoresis picture XXXX.
To assemble these two fragments and the Tet(x) gblock all primers included extensions with BsaI restriction sites and Golden Gate overhangs.The Golden Gate was performed following the Golden Gate protocol:

<DIV class="col-lg-12 col-md-12 col-xs-12 content" style="padding-top: 20px; padding-bottom: 20px"><DIV class="table-slot" style="overflow-x: auto"><TABLE class="table table-bordered mdl-shadow--4dp" xsscleaned="overflow-x: scroll !important"><THEAD><TR><TH style="text-align:center">Name</TH><TH style="text-align:right">amount [ng]</TH></TR></THEAD><TBODY><TR><TD>backbone</TD><TD style="text-align:center">SP_2</TD><TD style="text-align:right">75</TD></TR><TR><TD>insert 1</TD><TD style="text-align:center">SP_1</TD><TD style="text-align:right">equimolar</TD></TR><TR><TD>insert 2</TD><TD style="text-align:center">Tet(x) promoter</TD><TD style="text-align:right">equimolar</TD></TR></TBODY></TABLE></DIV></DIV><DIV class="col-lg-12 col-md-12 col-xs-12 content" style="padding-top: 20px; padding-bottom: 20px"><DIV class="table-slot" style="overflow-x: auto"><TABLE class="table table-bordered mdl-shadow--4dp" xsscleaned="overflow-x: scroll !important"><THEAD><TR><TH style="text-align:right">temperature [°C]</TH><TH style="text-align:right">time [min]</TH></TR></THEAD><TBODY><TR><TD style="text-align:right">15x</TD></TR><TR><TD style="text-align:right">37</TD><TD style="text-align:right">2</TD></TR><TR><TD style="text-align:right">16</TD><TD style="text-align:right">1</TD></TR><TR><TD style="text-align:right">1x</TD></TR><TR><TD style="text-align:right">37</TD><TD style="text-align:right">30</TD></TR><TR><TD style="text-align:right">55</TD><TD style="text-align:right">5</TD></TR></TBODY></TABLE></DIV></DIV>

The Golden Gate product is purified using the ZymoResearch Kit and then 1 µl iselectroporated into 25 µl of S1059 E. coli. After 4 hours incubation in 3 ml SOC medium the cell culture is used for an plaque assay. A plaque PCR showed that the Tet(X)-insert is not included in the Golden Gate product andtherefore new Golden Gate overhangs are designed.
<DIV class="panel panel-default"><A data-toggle="collapse" data-parent="#accordion" href="#collapseweek_id-5"><DIV class="panel-heading">
            Calendar Week
33
        </DIV></A><DIV id="collapseweek_id-5" class="panel-collapse collapse"><DIV class="panel-body"><H2>Cytochrome_Engineering</H2><H2 id="-1-cloning-of-sp-cytochrome-pace-"><STRONG>1. Cloning of SP Cytochrome PACE:</STRONG>
All required parts for the selection phage were amplified in sufficient concentrations by PCR to be assembled by Golden Gate cloning.<DIV class="col-lg-12 col-md-12 col-xs-12 content" style="padding-top: 20px; padding-bottom: 20px"><DIV class="table-slot" style="overflow-x: auto"><TABLE class="table table-bordered mdl-shadow--4dp" xsscleaned="overflow-x: scroll !important"><THEAD><TR><TH>1</TH><TH>2</TH><TH>3</TH></TR></THEAD><TBODY><TR><TD>Q5 MM (2x)</TD><TD>25 µl</TD><TD>25 µl</TD><TD>12 µl</TD></TR><TR><TD>CG_011 (10 µM)</TD><TD>2.5 µl</TD><TD>2.5 µl</TD><TD>2.5 µl</TD></TR><TR><TD>SP_R (10 µM)</TD><TD>2.5 µl</TD><TD>2.5 µl</TD><TD>2.5 µl</TD></TR><TR><TD>SP_ATMTAL</TD><TD>1 µl</TD><TD>1 µl</TD><TD>1 µl</TD></TR><TR><TD>DMSO</TD><TD>2.5 µl</TD><TD>-</TD><TD>2.5 µl</TD></TR><TR><TD>Betain</TD><TD>-</TD><TD>5 µl</TD><TD>5 µl</TD></TR></TBODY></TABLE></DIV></DIV>

The cycler conditions were as follows:

<DIV class="col-lg-12 col-md-12 col-xs-12 content" style="padding-top: 20px; padding-bottom: 20px"><DIV class="table-slot" style="overflow-x: auto"><TABLE class="table table-bordered mdl-shadow--4dp" xsscleaned="overflow-x: scroll !important"><THEAD><TR><TH>Phase</TH><TH>Temperature [°C]</TH><TH>Time [min]</TH><TH>Cycles</TH></TR></THEAD><TBODY><TR><TD>Initial Denaturation</TD><TD>98</TD><TD>0:30</TD><TD>1</TD></TR><TR><TD>Denaturation</TD><TD>98</TD><TD>0:10</TD><TD>35</TD></TR><TR><TD>Annealing</TD><TD>72</TD><TD>0:25</TD><TD>35</TD></TR><TR><TD>Extension</TD><TD>72</TD><TD>3:00</TD><TD>35</TD></TR><TR><TD>Final Extension</TD><TD>72</TD><TD>2:00</TD><TD>1</TD></TR><TR><TD>Hold</TD><TD>10</TD><TD>∞</TD><TD>1</TD></TR></TBODY></TABLE></DIV></DIV>

The PCR fragments were analyzed by gel electrophoresis. There were no bands on the gel. Maybe the primer concentration, which was 1 µl instead of 10 µl was too small.
PCR was repeated with two different polymerases, Q5 and Phusion Flash and two different conditions in a 25 µl mix.

<DIV class="col-lg-12 col-md-12 col-xs-12 content" style="padding-top: 20px; padding-bottom: 20px"><DIV class="table-slot" style="overflow-x: auto"><TABLE class="table table-bordered mdl-shadow--4dp" xsscleaned="overflow-x: scroll !important"><THEAD><TR><TH>1</TH><TH>2</TH></TR></THEAD><TBODY><TR><TD>Polymerase MM (2x)</TD><TD>25 µl</TD><TD>25 µl</TD></TR><TR><TD>CG_011 (10 µM)</TD><TD>2.5 µl</TD><TD>2.5 µl</TD></TR><TR><TD>SP_R (10 µM)</TD><TD>2.5 µl</TD><TD>2.5 µl</TD></TR><TR><TD>SP_ATMTAL</TD><TD>1 µl</TD><TD>1 µl</TD></TR><TR><TD>Betain</TD><TD>-</TD><TD>5 µl</TD></TR></TBODY></TABLE></DIV></DIV>

The cycler conditions for Q5 were the same as above, the conditions for Phusion Flash were as follows:

<DIV class="col-lg-12 col-md-12 col-xs-12 content" style="padding-top: 20px; padding-bottom: 20px"><DIV class="table-slot" style="overflow-x: auto"><TABLE class="table table-bordered mdl-shadow--4dp" xsscleaned="overflow-x: scroll !important"><THEAD><TR><TH>Phase</TH><TH>Temperature [°C]</TH><TH>Time [min]</TH><TH>Cycles</TH></TR></THEAD><TBODY><TR><TD>Initial Denaturation</TD><TD>98</TD><TD>3:00</TD><TD>1</TD></TR><TR><TD>Denaturation</TD><TD>98</TD><TD>0:01</TD><TD>35</TD></TR><TR><TD>Annealing</TD><TD>72</TD><TD>0:05</TD><TD>35</TD></TR><TR><TD>Extension</TD><TD>72</TD><TD>1:30</TD><TD>35</TD></TR><TR><TD>Final Extension</TD><TD>72</TD><TD>1:00</TD><TD>1</TD></TR><TR><TD>Hold</TD><TD>10</TD><TD>∞</TD><TD>1</TD></TR></TBODY></TABLE></DIV></DIV>

The reaction mix with Q5 did not exhibit any bands on the gel, but with Phusion Flash there were bands of the expected length (6.198 kb) visible on the gel. The reaction mix without betain exhibited a stronger band.  
The PCR product was purified with the QIAquick Gel Extraction Kit and eluted in 30 µl ddH<SUB>2</SUB>0 with 9.6 ng/µl DNA.
</H2><H2 id="aps-opto">APs Opto</H2><H2 id="pcr-amplification-of-ap-fragments-">PCR amplification of AP fragments:</H2>
For the generation of the two Opto APs, AP_light and AP_dark three fragments are necessary:

<DIV class="col-lg-12 col-md-12 col-xs-12 content" style="padding-top: 20px; padding-bottom: 20px"><DIV class="table-slot" style="overflow-x: auto"><TABLE class="table table-bordered mdl-shadow--4dp" xsscleaned="overflow-x: scroll !important"><THEAD><TR><TH>AP_light</TH><TH>AP_dark</TH></TR></THEAD><TBODY><TR><TD>1</TD><TD>gIII_luxAB</TD><TD>gIII_luxAB</TD><TD>PCR product  </TD></TR><TR><TD>2</TD><TD>AP_light_BB</TD><TD>AP_dark BB</TD><TD>PCR product   </TD></TR><TR><TD>3</TD><TD>pBLind</TD><TD>pBLrep</TD><TD>oligo  </TD></TR></TBODY></TABLE></DIV></DIV>

gIII_luxAB:
In order to amplify gIII_luxAB, primers gIII_luxAB_F and gIII_luxAB_R were used together with AP_ATML (plasmid ID:5) as template in a 50 µl Q5 PCR reaction.
The cycler conditions were set as follows to amplify the 3.784 kb gIII_luxAB fragment:

<DIV class="col-lg-12 col-md-12 col-xs-12 content" style="padding-top: 20px; padding-bottom: 20px"><DIV class="table-slot" style="overflow-x: auto"><TABLE class="table table-bordered mdl-shadow--4dp" xsscleaned="overflow-x: scroll !important"><THEAD><TR><TH>Phase</TH><TH>Temperature [°C]</TH><TH>Time [min]</TH><TH>Cycles</TH></TR></THEAD><TBODY><TR><TD>Initial Denaturation</TD><TD>98</TD><TD>0:30</TD><TD>1</TD></TR><TR><TD>Denaturation</TD><TD>98</TD><TD>0:10</TD><TD>35</TD></TR><TR><TD>Annealing</TD><TD>72</TD><TD>0:25</TD><TD>35</TD></TR><TR><TD>Extension</TD><TD>72</TD><TD>2:00</TD><TD>35</TD></TR><TR><TD>Final Extension</TD><TD>72</TD><TD>2:00</TD><TD>1</TD></TR><TR><TD>Hold</TD><TD>10</TD><TD>∞</TD><TD>1</TD></TR></TBODY></TABLE></DIV></DIV>

Analysis by gel electrophoresis demonstrated a band of the expected length. The PCR product was purified with the QIAquick Gel Extraction Kit and eluated in 30 µl ddH<SUB>2</SUB>0 with 52.5 ng/µl DNA.
AP_light_BB and AP_dark_BB:
To amplify AP_light_BB and AP_dark_BB a 50 µl Q5 reaction mix was prepared using AP-ATML as template and the respective primers for the APs:

<DIV class="col-lg-12 col-md-12 col-xs-12 content" style="padding-top: 20px; padding-bottom: 20px"><DIV class="table-slot" style="overflow-x: auto"><TABLE class="table table-bordered mdl-shadow--4dp" xsscleaned="overflow-x: scroll !important"><THEAD><TR><TH>AP</TH><TH>Primer forward</TH><TH>Primer reverse</TH></TR></THEAD><TBODY><TR><TD>AP_light_BB</TD><TD>BB_F</TD><TD>CG_005</TD></TR><TR><TD>AP_dark_BB</TD><TD>BB_F</TD><TD>CG_008</TD></TR></TBODY></TABLE></DIV></DIV>

The cycler conditions were set as follows to amplify the 2,74 kb AP_light_BB and AP_dark_BB fragments:

<DIV class="col-lg-12 col-md-12 col-xs-12 content" style="padding-top: 20px; padding-bottom: 20px"><DIV class="table-slot" style="overflow-x: auto"><TABLE class="table table-bordered mdl-shadow--4dp" xsscleaned="overflow-x: scroll !important"><THEAD><TR><TH>Phase</TH><TH>Temperature [°C]</TH><TH>Time [min]</TH><TH>Cycles</TH></TR></THEAD><TBODY><TR><TD>Initial Denaturation</TD><TD>98</TD><TD>0:30</TD><TD>1</TD></TR><TR><TD>Denaturation</TD><TD>98</TD><TD>0:10</TD><TD>35</TD></TR><TR><TD>Annealing</TD><TD>72</TD><TD>0:25</TD><TD>35</TD></TR><TR><TD>Extension</TD><TD>72</TD><TD>2:00</TD><TD>35</TD></TR><TR><TD>Final Extension</TD><TD>72</TD><TD>2:00</TD><TD>1</TD></TR><TR><TD>Hold</TD><TD>10</TD><TD>∞</TD><TD>1</TD></TR></TBODY></TABLE></DIV></DIV>

After the cycler run, 5 µl of the reaction mix was load on an agarose gel and detected by UV light. AP_light_BB and AP_dark_BB exhibited a strong and clear band at 2,7 kb, therefore PCR product was purified with the QIAquick PCR Purification Kit.

<DIV class="col-lg-12 col-md-12 col-xs-12 content" style="padding-top: 20px; padding-bottom: 20px"><DIV class="table-slot" style="overflow-x: auto"><TABLE class="table table-bordered mdl-shadow--4dp" xsscleaned="overflow-x: scroll !important"><THEAD><TR><TH>AP</TH><TH>concentration [ng/µl]</TH></TR></THEAD><TBODY><TR><TD>AP_light_BB</TD><TD>166.0</TD></TR><TR><TD>AP_dark_BB</TD><TD>146.8</TD></TR></TBODY></TABLE></DIV></DIV><H2>Phage_Propagation_and_Quantification</H2><H2 id="phage-propagation-of-the-unevolved-dickinson-phage">Phage propagation of the <STRONG>unevolved</STRONG> Dickinson phage
Phage supernatant of the unevolved Dickinson phage: target_133_N-term_T7-C was received from Dickinson group. To propagate the phages, 4 ml <EM>E. coli</EM> culture (Stock ID: 47) was cultivated to an OD600 of 0.6 (in LB media + 25 mM glucose + amp) and infected with 4 µl of the phage supernatant. Culture was shaked at 37 °C overnight. On the next morning, culture was centrifuged at 6000 xg for 5 min and supernatant, which contains the phages, was stored at 4 °C.
A Blue Plaque Assay was performed to determine the phage titer of the supernatant. 143 plaques were counted at the 10<SUP>-10</SUP> dilution, which leads to a phage titer of 1.43*10<SUP>15</SUP> PFU/ml.
A plaque of this plate was picked to infect a 4 ml <EM>E. coli</EM> culture (Stock ID: 47) with an OD600 of 0.4. This culture was cultivated for two hours shaking at 37 °C and was subsequently transferred to 100 ml fresh 2xYT medium. After 1 hour carbenicillin (1000x) was add.
On the next day, culture was centrifuged at 3640 xg for 20 min. Supernatant was stored at 4 °C.
A Blue Plaque Assay was performed to determine the phage titer of the supernatant. The monoclonal Dickinson phage target_133_N-term_T7-C exhibited a phage titer of 1.85*10<SUP>9</SUP> PFU/ml.
</H2><H2>Software</H2><H2 id="kw33">KW33</H2><H2 id="generation-of-datasets-for-427-classes-and-299-classes">Generation of datasets for 427-classes and 299 classes</H2>
In order to further simplify the problem of having a high amount of weakly represented classes we generated two additional datasets:427 EC classes with a minimum sample size of 100, and 299 EC classes with a minimum samplesize of 200.
<H2 id="implementation-of-first-generative-model">Implementation of first generative Model</H2>
To reach the lowest hanging fruit in generative modeling, we switched DeeProtein to the generative mode. Generative modes take an input (or noise input) and return a modified version of the inout maximizing the class probability for the target class. In the context of protein sequences, generative modeling could for instance generate new funcitonalities by &quot;shifting&quot; a sequence in the multidimensional featurespace aling the direction of the target class.The easiest way to achieve functionality is by just &quot;training&quot; the input of the network. By freezing all the weights and allowing only the input to be modyfied iteratively, a optimized sequence is obtained.
Results have been disentchanting.
<H2>Split Polymerase</H2><H2 id="dickinson-pace-">Dickinson-PACE:</H2>
For implementation of the LuxAB reporter system in the Dickinson strains, golden gate cloning was implemented. The resulting strains had LuxAB integrated directly behind gene III. This should allow us to investigate the phage infection directly correlated to the LuxAB expression. These slightly different versions were transformed into S2060 cells for use in the PACE apparature.
Another change of the experimental design was implemented using MP6 instead of MP1. MP6 was transformed into the S2060 used for the Dickinson transformations. This change was made in order to generate more mutations than before. On top of that, MP6 allows to implement less genetic drift, as well as no stepping stones for the evolution.
The general design of the PACE experiment itself was held constant, including strain swapping after three and four and a half days. Currently we have no wash out and no phage contamination in the turbidostat according to the phage detection PCR.
<H2 id="reporter-system-">Reporter system:</H2>
The first testing of the S2060 + MP6 + alternative Dickinson-APs showed only &quot;special&quot; results. Generally, we can detect a luminescence signal. However, the signal of the phage supernatant is higher than the signal of the infected cells. Beyond that, the turbidostat displayed an higher luminescence signal in both lagoons. Based on previous results from Jan, we suppose that a phage titer of 10hoch10 is required for a sufficient &quot;real-time&quot; readout. Due to that, &quot;real-time&quot; PACE will not be possible.
Based on other results from different LuxAB testing, where LuxAB was tested under the psp-AR2 promotor, one can suppose that leaky expression of the T7 promotor could be the reason. In contrast to the T7 promotor, the psp-AR2 promotor did not show any background signal. This underlines the previous hypothesis.
<H2 id="toolbox-impressions-">Toolbox impressions:</H2><H2 id="thermofisher-">ThermoFisher:</H2>
ThermoFisher supplies a table containing different parts of which you can choose.

<DIV class="col-lg-12 col-md-12 col-xs-12 content" style="padding-top: 20px; padding-bottom: 20px"><DIV class="table-slot" style="overflow-x: auto"><TABLE class="table table-bordered mdl-shadow--4dp" xsscleaned="overflow-x: scroll !important"><THEAD><TR><TH style="text-align:center">part number/name</TH><TH style="text-align:center">public name (gene name)</TH><TH style="text-align:center">part family/class of the part</TH><TH style="text-align:center">part category</TH></TR></THEAD><TBODY><TR><TD style="text-align:center">Abbrevation (BBa_K747000)</TD><TD style="text-align:center">LacZ gene</TD><TD style="text-align:center">CDS</TD><TD style="text-align:center">reporter system</TD></TR></TBODY></TABLE></DIV></DIV><H2 id="blueheron-">BlueHeron:</H2>
BlueHeron offers a broad amount of different vectors for several purposes (protein purification detection - epitope tags, protein visualization - fluorescence tags, resistances, expression vectors). Each of the higher categories is divided into the purposes. Every subclass is then characterized by the following table:

<DIV class="col-lg-12 col-md-12 col-xs-12 content" style="padding-top: 20px; padding-bottom: 20px"><DIV class="table-slot" style="overflow-x: auto"><TABLE class="table table-bordered mdl-shadow--4dp" xsscleaned="overflow-x: scroll !important"><THEAD><TR><TH style="text-align:center">Entry vector</TH><TH style="text-align:center">SKU</TH><TH style="text-align:center">FP Feature</TH><TH style="text-align:center">N-Tag</TH><TH style="text-align:center">C-Tag</TH><TH>E.coli Selection</TH><TH>Cell selection</TH><TH>Expression</TH></TR></THEAD><TBODY><TR><TD style="text-align:center">name of the plasmid (pLenti-N-tRFP)</TD><TD style="text-align:center">abbrevation (BBa_K747000)</TD><TD style="text-align:center">RFP</TD><TD style="text-align:center">tRFP</TD><TD style="text-align:center">Myc-DDK</TD><TD>resistance</TD><TD>resistance</TD><TD>Bacteria</TD></TR></TBODY></TABLE></DIV></DIV><H2 id="idt-">IDT:</H2>
IDT only offers six different vectors, where your gene of interest will be inserted. Your sequence can be pasted and will be inserted in the vector after complexity scan and codon optimization.

<DIV class="col-lg-12 col-md-12 col-xs-12 content" style="padding-top: 20px; padding-bottom: 20px"><DIV class="table-slot" style="overflow-x: auto"><TABLE class="table table-bordered mdl-shadow--4dp" xsscleaned="overflow-x: scroll !important"><THEAD><TR><TH style="text-align:center">vector name</TH><TH style="text-align:center">Size</TH><TH style="text-align:center">selection marker</TH><TH style="text-align:center">application</TH><TH style="text-align:center">sequence</TH></TR></THEAD><TBODY><TR><TD style="text-align:center">name of the plasmid (pUCIDT (Amp))</TD><TD style="text-align:center">in bp</TD><TD style="text-align:center">ampicillin</TD><TD style="text-align:center">cloning, in vitro transcription, etc.</TD><TD style="text-align:center">pUCIDT.txt/genbank etc.</TD></TR></TBODY></TABLE></DIV></DIV><H2 id="agilent-">Agilent:</H2>
SureVector Tool is the best example for our potential toolbox:
<PRE>&lt;code&gt;* several fragments can be choosen
1. Origin (Kanamycine)
2. Promotor/Prom-tag (p15A)
3. Tag (yARS)
4. Bacterial selection (URA3)
5. XP1 Slot (PgalHis)
6. XP2 Slot (Gene of interest)
7. Upload sequence of interest (as file or by copy paste)
&lt;/code&gt;</PRE><H2 id="several-steps-to-create-the-vector-map-">Several steps to create the vector map:</H2></DIV></DIV></DIV></LI><LI>Select a bacterial origin of replication</LI><LI>Select a promotor or a promotor-tag fusion</LI><LI>Select a tag (skip this step if you selected a promoter.tag fusion)</LI><LI>Select a bacterial selection marker</LI><LI>Select additional options for functionality in yeast or mammalian cells.</LI><LI>Supply your 5´-3´ gene-of-interest in the sense strand direction by uploading a fastA file or pasting the sequence in the box</LI><LI>Click create, review your configuration and order the corresponding SureVector kits.</LI>
Making your gene-of-interest Sure Vector compatible:If amplifying from an existing template:Forward primer and reverse primer:
<UL><LI>two primers are provided for insertion into the Vector</LI></UL><H2 id="to-do-">To Do:</H2>
Protocol every different part we have:
<H2 id="1-aktivator-promotor">1. Aktivator + Promotor</H2>
Each user can insert the required input sequence but can use several ready-to-use promotor systems like the psp-promotor.
<H2 id="2-rbs-geneiii">2. RBS + geneIII</H2>
Different RBS sites in combination with gene III (different stringency)
<H2 id="3-rbs-reporter-system">3. RBS + reporter system</H2>
LuxAB, mNeonGreen, Lacz, as well as one or two other reporter system will be provided (estimation of leaky expression of the used circuit - NOT RT-PACE)
<H2 id="4-rbs-additional-cds-with-terminator-and-rbs">4. RBS + additional CDS with terminator and RBS</H2>
Another input sequence for the user depending on the protein of interest.
<H2 id="5-resistance-origin-of-replication">5. Resistance + origin of replication</H2>
Three different ORIs in combination with all our available resistances except from Chloramphenicol (always on the MPs)
<UL><LI>for every PACE circuit document the different possibilities
</LI><LI>Activator + Promotor:
</LI></UL><UL><LI>PAM-PACE gRNA (binding sequence) with Promotor (<EM>muss noch kloniert werden</EM>)</LI><LI>pBlind(v2) (promotor Blue-light induced) (<EM>muss noch kloniert werden</EM>)</LI><LI>pBLrep(v2) (promotor blue-light repressed) (<EM>muss noch kloniert werden</EM>)</LI><LI>Upstream activating sequence II (UASII) + UASI + integrating host factor (IHF) + psp-tet (alles gesamt psp-tet) (<EM>muss noch kloniert werden</EM>)</LI><LI>T7 Promotor (<EM>muss noch kloniert werden</EM>)</LI></UL>
Amplification with primers containing gibson overlaps 1 and 2 (g1 and g2)
Fwd_primer_g1: TGAAATTCTGGGCCCTGCATTACGAGAACTGARev_primer_g2: TACTGCACCTGTAGGTCGACGATGATCTGATA
<H2 id="2-rbs-geneiii-">2. RBS + geneIII:</H2><UL><LI>SD8 (always with geneIII)</LI><LI>sd8</LI><LI>sd6</LI><LI>SD4</LI><LI>sd2</LI><LI>sd5 (<EM>muss noch kloniert werden</EM>)</LI><LI>Theophylline Riboswitch 12.1 (<EM>muss noch kloniert werden</EM>)</LI></UL>
Amplification with primers containing gibson overlaps 2 and 3 (g2 and g3)
Fwd_primer_g2: TATCAGATCATCGTCGACCTACAGGTGCAGTARev_primer_g3: TTAGTACTACTCGAGTTGATAGGCACCGACCA
<H2 id="3-rbs-reporter-system-">3. RBS + reporter system:</H2><UL><LI>mNeonGreen-RBS + mNeonGreen + BBa B1002 and BBa B1006 Terminator  </LI><LI>LuxAB + BBA B1002 and BBa B1006 Terminator</LI><LI>NanoLuciferase</LI><LI>Beta-Galactosidase</LI></UL>
Amplification with primers containing gibson overlaps 3 and 4 (g3 and g4)
Fwd_primer_g3: TGGTCGGTGCCTATCAACTCGAGTAGTACTAARev_primer_g4: TGGTACTTCAAATGCGGCTTGGCTCCAGACAA
<H2 id="4-rbs-additional-cds-with-terminator-and-rbs-">4. RBS + additional CDS with terminator and RBS:</H2><UL><LI>RBS + sgRNA_expressioncassette_Cas9_RFP_TrrnB_terminator</LI><LI>BBa_J23105 + HDJ_Chaperone + BBa_B0010</LI><LI>Lac UV5 promotor SD8 RBS cI - Ser/Gly Linker-FKBP + SD8 RBS DHFR - Ser/Gly Linker - RPA</LI><LI>BBa_K808003 - (non sense)</LI><LI>Kat Promotor - linker - C-terminal T7-RNAP - Terminator</LI></UL>
Amplification with primers containing gibson overlaps 5 and 1 (g5 and g1)
Fwd_primer_g5: TCAGTTCTCGTAATGCAGGGCCCAGAATTTCARev_primer_g1: TTATCCACTGGCGAGCTCTGTAACGAAACGTA
The additional CDS dependent on the actual circuit which will be used. Due to that, the user has to insert a second sequence, where we can put an RBS in front of and a terminator behind.
<H2 id="5-resistance-origin-of-replication-">5. Resistance + origin of replication:</H2><UL><LI>pSC101 + Ampicillin</LI><LI>pBR322/ColE1 (=pSB1A3) + Ampicillin</LI><LI>p15A + Ampicillin</LI><LI>pSB1A3 + Tetracyclin (<EM>muss noch kloniert werden</EM>)</LI><LI>pSC101 + Tetracyclin (<EM>muss noch kloniert werden</EM>)</LI><LI>p15A + Tetracyclin (<EM>muss noch kloniert werden</EM>)</LI><LI>p15A + streptomycin (<EM>muss noch kloniert werden</EM>)</LI><LI>pBR322 + streptomycin (<EM>muss noch kloniert werden</EM>)</LI><LI>pSC101 + streptomycin (<EM>muss noch kloniert werden</EM>)</LI></UL>
Amplification with primers containing gibson overlaps 4 and 5
Fwd_primer_g4: TTGTCTGGAGCCAAGCCGCATTTGAAGTACCARev_primer_g5: TGAAATTCTGGGCCCTGCATTACGAGAACTGA
<UL><LI>check for typeIIs restriction sites</LI></UL><H2 id="recommendation-">Recommendation:</H2>
Use Ampcillin for the AP, Kanamycine for the SP and Chloramphenicol for MP
<UL><LI>Chloramphenicol cannot be used because of the mutagenesis plasmids which contain a Cmp Resistance</LI><LI>tetracyclin cannot be used in combination with the psp-tet promotor</LI></UL><H2 id="selection-phage-">Selection Phage:</H2><UL><LI>linearized selection phage without inserted gene of interest - fixed overhangs for golden gate cloning</LI><LI>Amplification of your gene of interest with primers A and B for cloning into the selection phage (either paulines primer - destruction or jans primer - for evolution of an gene-of-interest)</LI><LI>check for typeIIs restriction sites</LI><LI>provide only resistances without typeIIs</LI><LI>will be tested (<EM>muss noch gemacht werden</EM>)</LI><LI>cloning of SPs with different resistances (<EM>muss noch gemacht werden</EM>)</LI></UL></UL><LI>Alternative for selection phage if you want to analyze the essential amino acids:
</LI><LI>use paulines primer with backbone</LI><LI>Primers contain BsaI restriction sites</LI><LI>The sequences in brackets are the overhangs for the new primers, the rest of the sequences are specific primers for paulines and Jans PACE (can be ignored for the toolbox)
</LI><H2 id="paulines-primer-overhangs-for-cloning-of-sp-">Paulines Primer overhangs (for cloning of SP):</H2><UL><LI>gene of interest:Fwd: [TTGGTCTCATATC]</LI></UL>
Rev: [TTGGTCTCACAAT]TGAACCTCCAGAGC
<UL><LI>backbone part_1:Fwd [GCTTGGTCTCAATTG]ACATGCTAGTTTTACGATTACC</LI></UL>
Rev: [GCTCGGTCTCATAGA]TTAGAAAAACTCATCGAGCATCAAATG
<UL><LI>backbone part_2:Fwd: [TTGCGGTCTCATCTA]GAAGGAGATTTTCAACATGCT</LI></UL>
Rev: [TTGCGGTCTCAGATA]ACCCCGGTTGATAATAAG
<H2 id="jans-primer-overhangs-for-cloning-of-sp-">Jans Primer overhangs (for cloning of SP):</H2><UL><LI>gene of interest:Fwd:[CACCACAGGTCTCGAAAA]ATGGATAAGAAATACTCAATAGGCTTAGC</LI></UL>
Rev: [CACCACAGGTCTCGGTGC]GTCACCTCCTAGCTGACTC
<UL><LI>backbone part_1:Fwd: [CACAGGTCTCATCAG]GTCAGAAGGGTTCTATCTC</LI></UL>
Rev: [CACCACAGGTCTCGTTTT]TTTTCCTCCTAAGCTGTTAGAAAAACTC
<UL><LI>backbone part_2:Fwd: [CACCACAGGTCTCGGCAC]GCGTAACTGTTCAG</LI></UL>
Rev: [CACAGGTCTCTCTGA]AAGCGTAAGAATACGTGGC
<UL><LI>backbone should be supplied with kanamycin, ampicillin, streptomycin and tetracyclin
</LI><LI>the antibotics cannot be the same in AP and SP
</LI></UL><H2 id="examples-of-tested-genes-of-interest-">Examples of tested genes of interest:</H2><UL><LI>Tet(X)</LI><LI>rpoZ-Cas9</LI><LI>CYP1A2</LI><LI>EL 222</LI><LI>Split-Cas9</LI><LI>T7-RNAP 1-179 (N-terminal) - Dickinson</LI><LI>RNAP Cytochrome C</LI></UL>
For the integration of the toolbox in our project, in silico cloning using Geneious was performed. Primers were ordered to clone the different parts in our cloning standard as well as building up parts for the iGEM registry.
<H2>Tetracyclin_Destructase</H2><H2 id="-ap-"><STRONG>AP</STRONG>
CPEC is performed with PCR product AP_destructase_g2-g5_PP013-PP014 as backbone and psp-tet promoter and and BBa_K808003 as inserts.The CPEC product (AP_destructase) is electroporated S2060 electrocompetent cells an plated on an ampicilline plate. Incubation over night was successful and a colony PCR is performed.

<DIV class="col-lg-12 col-md-12 col-xs-12 content" style="padding-top: 20px; padding-bottom: 20px"><DIV class="table-slot" style="overflow-x: auto"><TABLE class="table table-bordered mdl-shadow--4dp" xsscleaned="overflow-x: scroll !important"><THEAD><TR><TH>step</TH><TH style="text-align:center">temperature [°C]</TH><TH style="text-align:right">time [s]</TH></TR></THEAD><TBODY><TR><TD>initial denaturation</TD><TD style="text-align:center">94</TD><TD style="text-align:right">30</TD></TR><TR><TD>denaturation</TD><TD style="text-align:center">94</TD><TD style="text-align:right">25</TD></TR><TR><TD>annealing</TD><TD style="text-align:center">51</TD><TD style="text-align:right">40</TD></TR><TR><TD>elongation</TD><TD style="text-align:center">68</TD><TD style="text-align:right">50</TD></TR><TR><TD>final extension</TD><TD style="text-align:center">68</TD><TD style="text-align:right">5</TD></TR></TBODY></TABLE></DIV></DIV><DIV class="col-lg-12 col-md-12 col-xs-12 content" style="padding-top: 20px; padding-bottom: 20px"><DIV class="table-slot" style="overflow-x: auto"><TABLE class="table table-bordered mdl-shadow--4dp" xsscleaned="overflow-x: scroll !important"><THEAD><TR><TH style="text-align:center">Name</TH><TH style="text-align:right">Volume [µl]</TH></TR></THEAD><TBODY><TR><TD>Template</TD><TD style="text-align:center">AP_destructase</TD><TD style="text-align:right">pipette tip</TD></TR><TR><TD>Primer forward</TD><TD style="text-align:center">JM_072</TD><TD style="text-align:right">0.5</TD></TR><TR><TD>Priner reverse</TD><TD style="text-align:center">JM_065</TD><TD style="text-align:right">0.5</TD></TR><TR><TD>Master Mix</TD><TD style="text-align:center">OneTaq 2x Master Mix</TD><TD style="text-align:right">12.5</TD></TR><TR><TD>ddH2O</TD><TD style="text-align:center">ad 25 µl</TD><TD style="text-align:right">11.5</TD></TR></TBODY></TABLE></DIV></DIV>

PCR proofed that the insertes are present in AP_destructase and a glycerol stock is created.
<DIV class="panel panel-default"><A data-toggle="collapse" data-parent="#accordion" href="#collapseweek_id-6"><DIV class="panel-heading">
            Calendar Week
34
        </DIV></A></DIV></H2><H2>Cytochrome_Engineering</H2><H2 id="-1-cloning-of-ap-cytochrome-pace-"><STRONG>1. Cloning of AP Cytochrome PACE:</STRONG>
As the assembly of the AP worked, the next aim was to produce AP variants with different ORIs. To achieve this, PCRs of the backbone (all plasmid parts except the ORI and AmpR) were performed. Unfortunately problems occured, which is the reason why series of following PCRs have been performed under different conditions (different polymerases, different annealing temperatures, adapted cycler runs). After all those actions had no promising effect we assumed, that there must have been a mistake in the previous procedure.In order to repeat the CPEC, one part (LuxAB) was still needed. So in the following PCRs were performed to amplify the missing LuxAB. This again beared problems, so different conditions were tested again (different polymerases, different primer concentrations, DMSO, Betain, different cycler runs, touch down PCR, slow ramping after denaturation)After none of this worked, we decided to chose mNeonGreen as a reporter instead of LuxAB, as this one has already been amplified.Now all necessary parts for the CPEC were abundant. The CPEC product was then transformed into chemically competent cells. On the next day colony PCRs have been conducted with 6 clones and simultaneously the corresponding growths were purified.
</H2><H2 id="-2-cloning-of-sp-cytochrome-pace-"><STRONG>2. Cloning of SP Cytochrome PACE:</STRONG>
In parallel to the Plaque Assys, plaque PCRs have been performed to check for the correct SP. These PCRs unfortunately did not show the correct outcome. Also new Primers have been designed and ordered, that amplify a smaller fragment of the SP making the test PCRs more efficient.When the new primers arrived test PCRs of the phage supernatant were performed, as an inefficient PCR could not be clearly excluded as source of error at this point. As this PCR also did not show the correct bands we decided to repeat the SP assembly with new backbone template. Those templates needed to be generated first. So PCRs from SP CBX TAL were conducted. These PCRs did not lead to the desired result and were repeated with different conditions (different polymerase, DMSO, Betain, cycler run). The changed conditions helped optimize the PCRs so that the products could be isolated from the agarose gel.Afterwards the Golden Gate could be repeated and the isolated GG product was tranformed into 2060 electro competent cells. When recovering after the transformation, after 4 hours of incubation at 37�C a centrifugation step followed to gather the supernatant containing SP. Using the supernatant, a Plaque Assay was performed.
</H2><H2>Modeling</H2>
A first numeric model of PredCel works without oscillations. Graphs look reasonable so far. Modelling was performed on three levels: On the lowest level one Step of Predcel monitoring Phage concentration as well as uninfected, infected and phage producing E. coli concentration <A href="170820mod-lvl1.png">Graph of level 1 Predcel model</A>. One level above all the concentrations were tracked over 100 iterations of Predcel <A href="170820mod-lvl2.png">Graph of level 2 Predcel model</A>. And on the third level different sets of values for starting fitness, starting phage concentration and starting E. coli concentration were tested. In this case we only monitored how long the phage titer at the end of each iteration of Predcel stayed above 1 pfu/mL and below 1e8 pfu/mL.<A href="170820mod-lvl3.png">Graph of level 3 Predcel model</A>At least the two higher levels probably only work in python, but maybe an interactive version of what happens during one iteration is possible. A final more comfortable version of the script was started.
<H2>Optogenetics</H2><H2 id="sp-opto">SP Opto</H2><H2 id="pcr-amplification-of-sp-fragments-">PCR amplification of SP fragments:</H2>
The blue-light mediated transcription activator EL222 was amplified from a gblock (EL222_expression_cassette) using CG_012_fwd and CG_013_rev as primers in a 25 µl Phusion Flash reaction mix.The cycler conditions for the 678 bp fragment were as follows:

<DIV class="col-lg-12 col-md-12 col-xs-12 content" style="padding-top: 20px; padding-bottom: 20px"><DIV class="table-slot" style="overflow-x: auto"><TABLE class="table table-bordered mdl-shadow--4dp" xsscleaned="overflow-x: scroll !important"><THEAD><TR><TH>Phase</TH><TH>Temperature [°C]</TH><TH>Time [min]</TH><TH>Cycles</TH></TR></THEAD><TBODY><TR><TD>Initial Denaturation</TD><TD>98</TD><TD>0:10</TD><TD>1</TD></TR><TR><TD>Denaturation</TD><TD>98</TD><TD>0:01</TD><TD>35</TD></TR><TR><TD>Annealing</TD><TD>72</TD><TD>0:05</TD><TD>35</TD></TR><TR><TD>Extension</TD><TD>72</TD><TD>0:11</TD><TD>35</TD></TR><TR><TD>Final Extension</TD><TD>72</TD><TD>1:00</TD><TD>1</TD></TR><TR><TD>Hold</TD><TD>10</TD><TD>∞</TD><TD>1</TD></TR></TBODY></TABLE></DIV></DIV><H2 id="oligo-annealing-and-phosphorylation-of-the-ap-promoters-">Oligo annealing and phosphorylation of the AP promoters:</H2>
pBLind: CG_010_fwd, CG_009_revpBLrep: CG_007_fwd, CG_006_rev
5 µl of each primer (CG_010_fwd, CG_009_rev for pBLind and CG_007_fwd, CG_006_rev for pBLrep) (100 µM) was added to 1.1 µl T4 ligase buffer in a PCR tube. Reaction mix was heated up to 98 °C and slowly cooled down (0.1 °C/s).
The annealed oligos were phosphorylated using the T4 PNK according to the neb protocol &quot;Non-radioactive Phosphorylation with T4 PNK or T4 PNK (3´ phosphatase minus)&quot;.
<H2 id="golden-gate-assembly-of-sp-and-ap-">Golden Gate assembly of SP and AP:</H2>
GG of Opto SP (20 µl Reaction):

<DIV class="col-lg-12 col-md-12 col-xs-12 content" style="padding-top: 20px; padding-bottom: 20px"><DIV class="table-slot" style="overflow-x: auto"><TABLE class="table table-bordered mdl-shadow--4dp" xsscleaned="overflow-x: scroll !important"><THEAD><TR><TH>Reagents</TH><TH>Volume [µl]</TH></TR></THEAD><TBODY><TR><TD>SP_BB (Purification ID: 515)</TD><TD>15</TD></TR><TR><TD>EL_222 (Purification ID: 520)</TD><TD>2 (of a 1:10 dilution)</TD></TR><TR><TD>BsaI</TD><TD>0.5</TD></TR><TR><TD>T4 Ligase</TD><TD>0.5</TD></TR><TR><TD>t4 Ligase Buffer</TD><TD>2</TD></TR></TBODY></TABLE></DIV></DIV>

GG of Opto APs (20 µl Reaction):
AP_light

<DIV class="col-lg-12 col-md-12 col-xs-12 content" style="padding-top: 20px; padding-bottom: 20px"><DIV class="table-slot" style="overflow-x: auto"><TABLE class="table table-bordered mdl-shadow--4dp" xsscleaned="overflow-x: scroll !important"><THEAD><TR><TH>reagents</TH><TH>Volume [µl]</TH></TR></THEAD><TBODY><TR><TD>gIII_luxAB (Purification ID: 514)</TD><TD>2</TD></TR><TR><TD>AP_light_BB (Purification ID: 516)</TD><TD>0.5</TD></TR><TR><TD>pBLind</TD><TD>1 (of a 1:100 dilution)</TD></TR><TR><TD>BsaI</TD><TD>0.5</TD></TR><TR><TD>T4 Ligase</TD><TD>0.5</TD></TR><TR><TD>t4 Ligase Buffer</TD><TD>2</TD></TR></TBODY></TABLE></DIV></DIV>

AP_dark

<DIV class="col-lg-12 col-md-12 col-xs-12 content" style="padding-top: 20px; padding-bottom: 20px"><DIV class="table-slot" style="overflow-x: auto"><TABLE class="table table-bordered mdl-shadow--4dp" xsscleaned="overflow-x: scroll !important"><THEAD><TR><TH>reagents</TH><TH>Volume [µl]</TH></TR></THEAD><TBODY><TR><TD>gIII_luxAB (Purification ID: 514)</TD><TD>2</TD></TR><TR><TD>AP_dark_BB (Purification ID: 517)</TD><TD>0.5</TD></TR><TR><TD>pBLrep</TD><TD>1 (of a 1:100 dilution)</TD></TR><TR><TD>BsaI</TD><TD>0.5</TD></TR><TR><TD>T4 Ligase</TD><TD>0.5</TD></TR><TR><TD>t4 Ligase Buffer</TD><TD>2</TD></TR></TBODY></TABLE></DIV></DIV>

Cycler conditions were as follows:

<DIV class="col-lg-12 col-md-12 col-xs-12 content" style="padding-top: 20px; padding-bottom: 20px"><DIV class="table-slot" style="overflow-x: auto"><TABLE class="table table-bordered mdl-shadow--4dp" xsscleaned="overflow-x: scroll !important"><THEAD><TR><TH>Temperature [°C]</TH><TH>Time [min]</TH><TH>Cycles</TH></TR></THEAD><TBODY><TR><TD>37</TD><TD>3:00</TD><TD>15</TD></TR><TR><TD>16</TD><TD>4:00</TD><TD>15</TD></TR><TR><TD>37</TD><TD>30:00</TD><TD>1</TD></TR><TR><TD>65</TD><TD>0:05</TD><TD>1</TD></TR></TBODY></TABLE></DIV></DIV><H2 id="trafos-of-aps-and-sp-">Trafos of APs and SP:</H2>
APs were transformed into Top 10 cells (chemically competent) and SP was transformed into S1059 (electrocompetent).
AP trafos were plated on LB-plates containing Amp. SP trafos were cultivated in SOC for 4 h and a Blue Plaque Assay was subsequently performed with the phage supernatant. The phage supernatant was also used to perform a PCR of the phage insert. Therefore, a 25 µl Q5 PCR reaction was prepared. To amplify the 1.9 kb fragment containing EL222, the primers JM_068_rev and JM_069_fwd were used together with 2 µl of the phage supernatant as template.
The cycler conditions were as follows:

<DIV class="col-lg-12 col-md-12 col-xs-12 content" style="padding-top: 20px; padding-bottom: 20px"><DIV class="table-slot" style="overflow-x: auto"><TABLE class="table table-bordered mdl-shadow--4dp" xsscleaned="overflow-x: scroll !important"><THEAD><TR><TH>Phase</TH><TH>Temperature [°C]</TH><TH>Time [min]</TH><TH>Cycles</TH></TR></THEAD><TBODY><TR><TD>Initial Denaturation</TD><TD>98</TD><TD>3:00</TD><TD>1</TD></TR><TR><TD>Denaturation</TD><TD>98</TD><TD>0:10</TD><TD>35</TD></TR><TR><TD>Annealing</TD><TD>64</TD><TD>0:25</TD><TD>35</TD></TR><TR><TD>Extension</TD><TD>72</TD><TD>0:50</TD><TD>35</TD></TR><TR><TD>Final Extension</TD><TD>72</TD><TD>2:00</TD><TD>1</TD></TR><TR><TD>Hold</TD><TD>10</TD><TD>∞</TD><TD>1</TD></TR></TBODY></TABLE></DIV></DIV>

Analysis by Gel electrophoresis revealed two bands at 1 kb and 1.9 kb.
<H2>Software</H2><H2 id="word2vec-embeddings-on-proteinsequences">Word2Vec Embeddings on Proteinsequences</H2>
We rewrote a word2vec implementation from tensorflows tutorials that implements Efficient Estimation of Word Representations in Vector Space, ICLR 2013 (Mikolov, et. al.). The model is a skipgram model with negative sample that uses custom ops written in C. The code was adapted to our needs, mainly by changing datatypes in the C kernels and writing a different evaluation function based on predicting the nearest words to the most frequent words instead of using analogies. Two new datasets were generated based on both swissprot and uniprot. Training of 4mer embeddings in 50, 100 and 200 dimensions were started but have not been calculated yet.Visualisation of the first checkpoints is possible via tensorboard <A href="170820ai-vistestemb">Visualisation of an example embedding via tensorboard</A>.
<H2 id="implementation-of-squeezenet-architecture">IMPLEMENTATION OF SQUEEZENET Architecture</H2>
With implamentation of a new architecture based on Sequeeze-net (Forrest N. Iandola, 2017), relying on 1x1 convolutions we were able to grasp the 299 as well as the 637 classes dataset. The new model architecture looks the following:
<UL><LI>InputLayer  model_valid/input_layer_valid: (64, 20, 1000, 1)</LI><LI>PadLayer   model_valid/block1/pad_layer_valid: paddings:[[0, 0], [0, 0], [3, 3], [0, 0]] mode:CONSTANT</LI><LI>Conv2dLayer model_valid/block1/cnn_layer_valid: shape:[20, 7, 1, 128] strides:[1, 5, 1, 1] pad:VALID act:prelu</LI><LI>Conv1dLayer model_valid/block2/cnn_layer_valid: shape:[6, 128, 128] stride:1 pad:SAME act:prelu</LI><LI>Conv1dLayer model_valid/1x1_I/1x1_valid: shape:[1, 128, 64] stride:1 pad:SAME act:prelu</LI><LI>BatchNormLayer model_valid/1x1_I/batchnorm_layer_valid: decay:0.900000 epsilon:0.000010 act:identity is_train:False</LI><LI>Conv1dLayer model_valid/block3/cnn_layer_valid: shape:[5, 64, 256] stride:1 pad:SAME act:prelu</LI><LI>PoolLayer   model_valid/block3/pool_layer_valid: ksize:[2] strides:[2] padding:VALID pool:pool</LI><LI>BatchNormLayer model_valid/block3/batchnorm_layer_valid: decay:0.900000 epsilon:0.000010 act:identity is_train:False</LI><LI>Conv1dLayer model_valid/block4/cnn_layer_valid: shape:[5, 256, 256] stride:1 pad:SAME act:prelu</LI><LI>PoolLayer   model_valid/block4/pool_layer_valid: ksize:[2] strides:[2] padding:VALID pool:pool</LI><LI>BatchNormLayer model_valid/block4/batchnorm_layer_valid: decay:0.900000 epsilon:0.000010 act:identity is_train:False</LI><LI>Conv1dLayer model_valid/1x1_II/1x1_valid: shape:[1, 256, 128] stride:1 pad:SAME act:prelu</LI><LI>BatchNormLayer model_valid/1x1_II/batchnorm_layer_valid: decay:0.900000 epsilon:0.000010 act:identity is_train:False</LI><LI>Conv1dLayer model_valid/block5/cnn_layer_valid: shape:[5, 128, 256] stride:1 pad:SAME act:prelu</LI><LI>PoolLayer   model_valid/block5/pool_layer_valid: ksize:[2] strides:[2] padding:VALID pool:pool</LI><LI>BatchNormLayer model_valid/block5/batchnorm_layer_valid: decay:0.900000 epsilon:0.000010 act:identity is_train:False</LI><LI>Conv1dLayer model_valid/block6/cnn_layer_valid: shape:[5, 256, 512] stride:1 pad:SAME act:prelu</LI><LI>PoolLayer   model_valid/block6/pool_layer_valid: ksize:[2] strides:[2] padding:VALID pool:pool</LI><LI>BatchNormLayer model_valid/block6/batchnorm_layer_valid: decay:0.900000 epsilon:0.000010 act:identity is_train:False</LI><LI>Conv1dLayer model_valid/1x1_III/1x1_valid: shape:[1, 512, 256] stride:1 pad:SAME act:prelu</LI><LI>BatchNormLayer model_valid/1x1_III/batchnorm_layer_valid: decay:0.900000 epsilon:0.000010 act:identity is_train:False</LI><LI>Conv1dLayer model_valid/block7/cnn_layer_valid: shape:[5, 256, 516] stride:1 pad:SAME act:prelu</LI><LI>PoolLayer   model_valid/block7/pool_layer_valid: ksize:[2] strides:[2] padding:VALID pool:pool</LI><LI>BatchNormLayer model_valid/block7/batchnorm_layer_valid: decay:0.900000 epsilon:0.000010 act:identity is_train:False</LI><LI>Conv1dLayer model_valid/block8/cnn_layer_valid: shape:[5, 516, 1024] stride:1 pad:SAME act:prelu</LI><LI>PoolLayer   model_valid/block8/pool_layer_valid: ksize:[2] strides:[2] padding:VALID pool:pool</LI><LI>BatchNormLayer model_valid/block8/batchnorm_layer_valid: decay:0.900000 epsilon:0.000010 act:identity is_train:False</LI><LI>Conv1dLayer model_valid/1x1_IV/cnn_layer_valid: shape:[1, 1024, 512] stride:1 pad:SAME act:prelu</LI><LI>BatchNormLayer model_valid/1x1_IV/batchnorm_layer_valid: decay:0.900000 epsilon:0.000010 act:identity is_train:False</LI><LI>Conv1dLayer model_valid/block9/cnn_layer_valid: shape:[5, 512, 1024] stride:1 pad:SAME act:prelu</LI><LI>PoolLayer   model_valid/block9/pool_layer_valid: ksize:[2] strides:[2] padding:VALID pool:pool</LI><LI>BatchNormLayer model_valid/block9/batchnorm_layer_valid: decay:0.900000 epsilon:0.000010 act:identity is_train:False</LI><LI>Conv1dLayer model_valid/outlayer/cnn_layer_valid: shape:[1, 1024, 637] stride:1 pad:SAME act:prelu</LI><LI>BatchNormLayer model_valid/outlayer/batchnorm_layer_valid: decay:0.900000 epsilon:0.000010 act:identity is_train:False</LI><LI>MeanPool1d global_avg_pool: filter_size:[7] strides:1 padding:valid</LI></UL>
The architecture is fully convolutional, ending in an average pooling layer as outlayer, with the channels dimension corresponding to the number of classes. All inputs were 1-hot encoded and zero padded to a boxsize of 1000 positions.

<DIV class="col-lg-12 col-md-12 col-xs-12 content" style="padding-top: 20px; padding-bottom: 20px"><DIV class="table-slot" style="overflow-x: auto"><TABLE class="table table-bordered mdl-shadow--4dp" xsscleaned="overflow-x: scroll !important"><THEAD><TR><TH>Model</TH><TH>lr</TH><TH>classes</TH><TH>Comment</TH><TH>restored</TH><TH>maxstep</TH><TH>boxsize</TH><TH>ACC</TH></TR></THEAD><TBODY><TR><TD>0.01</TD><TD>299</TD><TD>NO</TD><TD>220000</TD><TD>1000</TD><TD>0.8 (valid)</TD></TR><TR><TD>0.01</TD><TD>637</TD><TD>NO</TD><TD>180000</TD><TD>1000</TD><TD>0.55 (valid)</TD></TR><TR><TD>0.01</TD><TD>637</TD><TD>YES</TD><TD>35000</TD><TD>1000</TD><TD>0.75 (valid)</TD></TR></TBODY></TABLE></DIV></DIV><H2 id="references-">References:</H2><OL><LI>Iandola, F. N., Han, S., Moskewicz, M. W., Ashraf, K., Dally, W. J., &amp; Keutzer, K. (2016). SqueezeNet: AlexNet-level accuracy with 50x fewer parameters and&lt; 0.5 MB model size. arXiv preprint arXiv:1602.07360.</LI><LI>Mikolov, T., Sutskever, I., Chen, K., Corrado, G. S., &amp; Dean, J. (2013). Distributed representations of words and phrases and their compositionality. In Advances in neural information processing systems (pp. 3111-3119).</LI></OL><H2>Tetracyclin_Destructase</H2><H2 id="-ap-"><STRONG>AP</STRONG>
To test under which conditions the selection phage propagates best, 12 variations of the destrucatase accessory plasmid are assembled by gibson assembly.The variations include different RBS and origins of replication. Because time was running out we decided to end this project at that point and concentrated on other PACE and PREDCEL experiments.
<DIV class="panel panel-default"><A data-toggle="collapse" data-parent="#accordion" href="#collapseweek_id-7"><DIV class="panel-heading">
            Calendar Week
35
        </DIV></A></DIV></H2><H2>Cytochrome_Engineering</H2><H2 id="-1-cloning-of-ap-cytochrome-pace-"><STRONG>1. Cloning of AP Cytochrome PACE:</STRONG>
After the purification showed extremely low DNA concentrations, the transformation of the AP was repeated using the previously purified AP product. Afterwards the performed colony PCRs did not show the expected results. So a test digest was conducted which also did not show the desired results. The next day the colony PCR was performed again using a different polymerase. After this did not lead to the expected result, the CPEC was conducted one more time. Loading the CPEC product on an agarose gel sadly showed that this repeated CPEC also did not work.Starting right from the beginning, new PCRs have been performed in order to generate all needed parts to repeat the CPEC once again. After those PCRs finally worked, the products were extracted from the agarose gel to avoid the impact of primer dimers in the following CPEC.The CPEC was conducted under a different strategy this time as it did not work several times before. Instead of using all fragments/parts in one CPEC reaction, the procedure was split in 3 CPECs as a whole. In the first 2 CPECs half of the parts were assembled together respectively. Then the AP parts were assembled toghether in the third CPEC to form the full length AP. This product was then again transformed into chemically competent cells.After the transformation did not work, the 3 separate CPECs have been repeated once again. This time the third CPEC from the previous attempt and from the new attempt have been transformed into two different strains of chemically competent cell respectively to make sure that the cells used for the latest transformation are ok.This time the transformations with the second strain of chemically competent cells worked in both cases, the new and the previous CPEC. Test PCRs followed of several clones of both transformations. Some of those clones showed bands at the correct height. Subsequently, the corresponding growhts containing the AP plasmids were preparated in order to isolate the APs and one of them was sent in to GATC to be sequenced. 
</H2><H2 id="-2-cloning-of-sp-cytochrome-pace-"><STRONG>2. Cloning of SP Cytochrome PACE:</STRONG>
On the next day, one Plaque was picked to be tested by PCR using the new Primers. The result was not what we were hoping for. So 10 more Plaques were picked and tested by PCR. Three of these Plaques showed the correct bands and a PCR purification was performed with those three PCRs. Two of the purifications had suitable concentrations and were sent to GATC for sequencing. The Plaque Assay was repeated twice with the picked plaques that were sent in for sequencing using different dilution steps.The results of these Plaque Assays were promising but did not show single plaques. So in order to determine the phage titer the Plaque Assay was repeated one more time. On the next day plaque PCRs have been conducted with the new Primers again with 5 chosen plaques. 
</H2><H2 id="-3-test-constructs-for-riboswitch-and-cyp1a2-"><STRONG>3. Test constructs for Riboswitch and CYP1A2:</STRONG>
Test constructs were designed in order to seperately test the Riboswitch of the AP and the CYP1A2 on the SP.<EM>1)    Riboswitch test construct:</EM>Taking the backbone of the pSB4K5 Plasmid which contains RFP and cloning the Riboswitch right infront of the RFP replacing the previous RBS. This construct enables an easy readout when adding theophylline to see whether theophylline induces the Riboswitch.<EM>2)    CYP1A2 test construct:</EM>Taking pSB1C3 and replacing the RFP by the CYP1A2 gene. When this construct is transformed into competent cells already containing the Theophylline PACE AP (contains the chaperone) and the substrate (caffein) is added, the activity of the enzyme can be checked.So far the test construct have only been implemented in silico. Also all needed primers were designed and ordered.
</H2><H2>Modeling</H2>
Calculations of medium consumption for iGEM goes green were performed and made interactive. A heatmap visualizes medium consumption, calculation of ideal turbidostat and lagoon sizes are possible. The user can annotate own experiments and compare them to others.Heatmap with default valuesExperimentation with elements of a Predcel model based on distributions instead of scalars were started. The idea is that a population of phages does not have one fitness between 0 and 1 but rather has individuals that have different fitness values. In this more complex model the concentrations have to be calculated for each phage fitness value, depending on the amount of phages that have that fitness value. The fitness distribution is changed by mutation and by selection. The first naive approach for mutation was programmed. It simply substracts a given percentage of the difference between the amount of a fitness value and the mean amount from the amount of a fitness value. Obviously this is oversimplified and will therefore be replaced by a model based on the idea that every sequence that mutates gets better or worse with normally distributed changes.
<H2>Optogenetics</H2><H2 id="sp-opto">SP Opto</H2>
The plaque assay of the SP trafo revealed plaques, but the negative control also. Nevertheless, 5 plaques were picked and a SP PCR with Q5 was performed in the same way as the day before. Analysis by gel electrophoresis exhibited bands of incorrect length. Probably a contamination of the TALEN phage occurred.
<H2 id="ap-opto">AP Opto</H2>
As the AP trafos exhibited no colonies on the Amp-plates, the Golden Gate was repeated using a slightly different cycler protocol:

<DIV class="col-lg-12 col-md-12 col-xs-12 content" style="padding-top: 20px; padding-bottom: 20px"><DIV class="table-slot" style="overflow-x: auto"><TABLE class="table table-bordered mdl-shadow--4dp" xsscleaned="overflow-x: scroll !important"><THEAD><TR><TH>Temperature [°C]</TH><TH>Time [min]</TH><TH>Cycles</TH></TR></THEAD><TBODY><TR><TD>37</TD><TD>3:00</TD><TD>15</TD></TR><TR><TD>16</TD><TD>4:00</TD><TD>15</TD></TR><TR><TD>16</TD><TD>5:00</TD><TD>1</TD></TR><TR><TD>65</TD><TD>2:00</TD><TD>1</TD></TR></TBODY></TABLE></DIV></DIV>

Trafos in chemically competent Top 10 cells were performed and cells were plated on amp-plates.
<H2>Protein_Interaction</H2><H2 id="-week-35-and-previous-"><STRONG>Week 35 and previous:</STRONG>
The idea of building a new genetic gene circuit for evolution of protein interactions in context of phage assisted continuous evolution (PACE) came up. Lots of different proteins were proofed whether they match the requirements of PACE evolution in iGEM context or not. Major focus at the beginning was the evolution of an anti-body-like protein. Several ideas and the respective cloning concepts were discussed. Even thus the imagined gene circuit should have worked several concerns let to further research.
</H2><H2>Software</H2><H2 id="performance-of-the-squeezenet-architecture-singlelabel-599">Performance of the Squeezenet Architecture - singlelabel 599</H2>
The model was run successfully on the old 599 classes dataset.Parameters: lr = E-2, batchsize=64, epsilon=0.1<A href="DeeProtein_TFRECORDS_PURECONV_1x1tuned_750k_restored750kfull_sce_adam_1dconv637_1000_one_hot_padded_64_0.001_0.1.roc_16.svg">ROC</A><A href="DeeProtein_TFRECORDS_PURECONV_1x1tuned_750k_restored750kfull_sce_adam_1dconv637_1000_one_hot_padded_64_0.001_0.1.precision_16.svg">Precision</A><H2 id="performance-of-the-squeezenet-architecture-singlelabel-679">Performance of the Squeezenet Architecture - singlelabel 679</H2>
The model was run successfully on the 679 classes dataset.Parameters: lr = E-5, batchsize=64, epsilon=0.1<A href="DeeProtein_TFRECORDS_PURECONV_1x1tuned_restored679_sce_adam_1dconv_EC_679_1000_one_hot_padded_64_0.001_0.1.roc_9.svg">ROC</A><A href="DeeProtein_TFRECORDS_PURECONV_1x1tuned_restored679_sce_adam_1dconv_EC_679_1000_one_hot_padded_64_0.001_0.1.precision_9.svg">Precision</A><H2 id="performance-of-the-squeezenet-architecture-multilabel-1084">Performance of the Squeezenet Architecture - Multilabel 1084</H2>
The model was run successfully on the 1084 GO-classes dataset.Parameters: lr = E-3, batchsize=64, epsilon=0.1<A href="DeeProtein_TFRECORDS_PURECONV_1x1LARGE_MULTI_restored1084_sce_adam_1dconv_EC_1084_1000_one_hot_padded_64_0.0001_0.1.roc_39.svg">ROC</A><A href="DeeProtein_TFRECORDS_PURECONV_1x1LARGE_MULTI_restored1084_sce_adam_1dconv_EC_1084_1000_one_hot_padded_64_0.0001_0.1.precision_39.svg">Precision</A><H2 id="corrected-datasets-for-missing-classes-reworte-eval-to-enclude-the-whole-validation-set">Corrected datasets for missing classes, reworte <CODE>eval()</CODE> to enclude the whole validation set</H2><UL><LI>Dataset 637, was missing 138 classes due to the min. length requirement in the <CODE>DatasetGenerator</CODE> class. The requirement was lowered to 175AA. Further the <CODE>DatasetGenerator</CODE> class was rewritten, to ensure to contain 5 samples from every class in the validation set.</LI><LI>the <CODE>eval()</CODE> function of <CODE>DeeProtein</CODE> was rewritten to perform the validaion on the <EM>whole</EM> validation set at given steps.</LI></UL>
Performance on 679 classes with minlength 175:lr=0.01, e=0.1, batchsize=64<A href="DeeProtein_TFRECORDS_PURECONV_1x1tuned_restored637750kfull_sce_adam_1dconv679_1000_one_hot_padded_64_0.01_0.1.roc_32.svg">ROC</A><A href="DeeProtein_TFRECORDS_PURECONV_1x1tuned_restored637750kfull_sce_adam_1dconv679_1000_one_hot_padded_64_0.01_0.1.precision_32.svg">Precision</A>
lr=0.001, e=0.1, batchsize=64<A href="DeeProtein_TFRECORDS_PURECONV_1x1tuned_restored637750kfull_sce_adam_1dconv679_1000_one_hot_padded_64_0.01_0.1.roc_32.svg">ROC</A><A href="DeeProtein_TFRECORDS_PURECONV_1x1tuned_restored637750kfull_sce_adam_1dconv679_1000_one_hot_padded_64_0.01_0.1.precision_32.svg">Precision</A>
Reinitialization with pretrained parameters and lower learning rate allowed finetuning of the classifier. Especially as the validation set is uniformally distributed (in contrast to the training set) the classifier can be considered as trained.
<H2 id="roc-acc-auc-metrics">ROC/ACC/AUC-metrics</H2>
ROC and AUC was added to be calculated on the fly (after validation on the whole validation set.).
<H2 id="training-models-on-the-embedded-sequences">Training models on the embedded sequences</H2>
We generated batches from the word embeddings (dim=100, kmer-length=3) for the 679(EC) and the 1084 mulilabel network. However training proceeds much more slowly as the parametersize is 5 times the size of the one-hot network.
<H2 id="multilabel-classification">Multilabel-classification</H2>
In order to be able to perform multilabel classification, we rewrite the input pipeline (<CODE>DatasetGenerator, BatchGenerator, TFrecordsgenerator</CODE>) and generated two datasets with 339 and 1084 classes respectively. The considered labels were chosen solely based on their polulation. As the GO-term hierarchy follows a directed acyclic graph (DAG) we looked up all parent nodes for each leaf nodes and included the total set of annotations for each sequence.
First models were run after extending the network for 2 convolutional and 2 1x1 layers on the 1084 classes dataset. Results were disenchanting.
<H2 id="comparison-of-datasets">Comparison of datasets</H2>
Total seqs after filtering (EC): 220488Total seqs after filtering (GO): 235767

<DIV class="col-lg-12 col-md-12 col-xs-12 content" style="padding-top: 20px; padding-bottom: 20px"><DIV class="table-slot" style="overflow-x: auto"><TABLE class="table table-bordered mdl-shadow--4dp" xsscleaned="overflow-x: scroll !important"><THEAD><TR><TH>Datatype</TH><TH>Dataset</TH><TH>Samples</TH><TH>% of filtered sequences considered</TH><TH>% of total sequences considered</TH></TR></THEAD><TBODY><TR><TD>EC</TD><TD>679</TD><TD>165658</TD><TD>75.13</TD><TD>63.18</TD></TR><TR><TD>GO</TD><TD>1084</TD><TD>233386</TD><TD>98.99</TD><TD>89.00</TD></TR></TBODY></TABLE></DIV></DIV><H2 id="word2vec">Word2Vec</H2>
The new word2vec adaption we implemented last week was optimized by a few minor changes. Metadata of the embeddings can now be used to analyse the embeddings with tensorboard. Search for a single k-mer, finding the nearest neighbours, annotating frequencies or other properties to the points and search for groups of k-mers defined by regular expressions work. Embeddings of 3-mers and 4-mers in 50, 100 and 200 dimensions were calculated on whole swissprot and whole uniprot. Principle Component Analysis (PCA) was performed on a 100-dimensional embedding of 3-mers from swissprot and showed interesting properties. However, reduction of 100 dimensions to two or three may account for at least some of these.Here the 3-mers marked darker are more frequent, those seem to cluster together.For each amino acid all 3-mers containing it are marked in read. Some of those selection cluster together, others do not. This may also be due to the dimension reduction.All 3-mers containing cysteine are marked in red.All 3-mers containing lysine are marked in red.All 3-mers containing proline are marked in red.All 3-mers containing valine are marked in red.Especially cysteine, proline and lysine containing 3-mers cluster together, while the ones containing valine appear to be more equally distributed.Clustering of k-mers based on amino acid content is a first hint, embeddings can be useful for input of neural networks. However their performance can probably only be measured indirectly by comparing the performance of a single architecture on different inputs.
<H2 id="dickinson-pace-2-0-">Dickinson-PACE 2.0:</H2>
Due to problems with the documentation in the awesome_sheet, we were using different strains for our PACE run. The results of two PCR reactions showed the abundance of a positive and a negative AP, which should not be a problem based on the findings of the properties of the MPs one week before. Follwing this further, we performed several days of PACE using this strain.After seven days, plaque assays were performed and a plaque PCR was used for amplification of the N-terminal domain of the T7 RNAP. Even the sequencing of over 10 plaques did not yield any mutations in the essential domain.
Due to this, we should go back to the old experimental design of Dickinson-PACE.
<H2 id="dickinson-pace-">Dickinson-PACE:</H2>
We returned to the old experimental design, using MP1 instead of MP6 as well as the normal Dickinson strains without LuxAB behind gene III.
For the organisation of our runs, we now use a better version of our google docs sheet. We are now documenting any problems and any results of our plaque assays and our phage detection PCRs. Plaque assays are implemented every 24 hours, while the Phage detection PCR is carried out every 12 hours.
We could perform this PACE run for seven days, like planed before. After seven days, plaque assays were performed and single plaques were picked. The N-terminal domain of the T7 RNAP was amplified using PCR reaction. PCR product was send to GATC for sequencing analysis.
This run did also not yield any mutations. On the one hand we therefore used an PCR on the phage supernatant, while on the other hand using single plaques as implemented before. Both methods showed no mutations, which is why we evaluated most of the parts. Especially the mutagenesis plasmid could be or should be the most probable problem. Calculations of the amount of arabinose used for induction in our PACE device and the original Dickinson-PACE leaded to the finding that we are using only 4 - 6,5 mM Arabinose instead of 30-40 mM which is used in Dickinson-PACE. This underlines the hypothesis that we are using not enough arabinose to induce the MP in a sufficient way.
<H2 id="predcel-">PREDCEL:</H2>
First experiment:Test the general induction of MP with different arabinose &amp; glucose concentrationsInoculate 20 different cultures with a bacteria from Stock0, 5, 12.5 and 25 mM Glucose0, 6, 12.5, 20 and 31 mM ArabinoseRead-out: time until the cultures reaches OD 0.6 - 0.8Results:only the cultures without arabinose reached the OD in sufficient timeall other cultures needed more than 30 hours for growing up
Second experiment:Let a culture with Glucose grow until it reaches OD 0.6Let it cool on ice and split into 4 ml cultures with different concentrations of glucose (0 mM, 12,5 mM, 25 mM) and arabinose (0 mM, 6 mM, 25 mM)Read-out: Incubate again and measure OD at different time pointsResults:differences in OD are small (probably due to high glucose concentrations)Higher arabinose concentrations result in lower ODs
Use PREDCEL for induction of mutagenesis and mutationstransformation of different Dickinson strains in bacteria containing MP6even high selection pressure should be surmountablethe probability for directed mutation should be highAfterwards - propagation of the phages having the essential mutations by strains containing no MP  
<EM>Random mutagenesis</EM>Alternative:Transformation of MP1, MP4 and MP6 in S2208 strain (= S2060 - PACE strain - and pJC175e - Plasmid which is used for phage propagation)several implementations of PREDCELpropagation of phages should lead to several mutations - “phage-library”Afterwards - propagation of the phages having the essential mutations by strains containing no MP  
<H2 id="toolbox-">Toolbox:</H2>
Most of the PCRs for amplification of the parts in our gibson-standard are done. Further steps will include the golden gate cloning for some parts and the integration into pSB1C3 for the iGEM standard. The amplifications which could not be implemented, will be carried out again or alternative primers will be ordered.
<H2 id="minion-sequencing-">MinION Sequencing:</H2>
We implemented our first two sequencing runs with the MinION. According to the ONT Community, shearing can be performed using 26 gauge needles, instead of using the Covaris g-tubes. The library preparation was perfomred following the lambda control experiment protocol. Following this, the MinION was loaded according to Oxford Nanopores instructions.  
Each run was implemented for 10 h according to the supplied ONT script. Live basecalling was performed, yielding 2000 reads and 11 million events. The accuracy of both runs was approximately 65 %, which is not good enough for sequencing of our amplicons. Possible reasons for this fact, could be the age of the flow cells as well as deviations from the standard protocol, normally using AMPure beads instead of our DNA Dynabeads from ThermoFisher. For this reason, another experiment using a newer flow cell will be implemented. In this experiment, we will use PCR products from supernatant from different timepoints of Dickinson-PACE, in order to investigate the behavior of the MinION sequencing in this case. Further knowledge about the sequencing should be acquired by using the ONT community.
In analogy to the Bt toxin paper, it should be possible to implement a sequencing workflow, enabling a reliable basecalling accuracy. A scheme of this workflow is provided as a picture.
</DIV></DIV></DIV><DIV class="panel panel-default"><A data-toggle="collapse" data-parent="#accordion" href="#collapseweek_id-8"><DIV class="panel-heading">
            Calendar Week
36
        </DIV></A><DIV id="collapseweek_id-8" class="panel-collapse collapse"><DIV class="panel-body"><H2>Cytochrome_Engineering</H2><H2 id="-1-cloning-of-ap-cytochrome-pace-"><STRONG>1. Cloning of AP Cytochrome PACE:</STRONG>
As the colony PCRs of this clone looked promising further expermients were carried out with it in the mean time. The aim of those experiments was to create AP variants with different ORIs. In order to do so, PCRs were performed to generate PCR products from Gibson overhang 5 to Gibson overhang 4 (3.6 kb fragment lacking the pBR ORI and AmpR). As those PCRs did not work, different conditions have been tested according to Phusion Flash polymerase trouble shooting protocol in order to optimize PCR conditions (different template concentrations, different cycler runs, touch down PCRs, Betain, DMSO, decreased amount of total cycles, different primer concetrations). After none of this worked either, further test PCRs with the template AP were conducted over the encoded chaperone protein, to see, whether the corresponding Gibson overhang was correct. The results indicated, that some of the APs carry the correct chaperone and some of the APs do not.So the AP was send back in to GATC for sequencing with a different primer this time. Also the two other AP purifications were send in for sequencing to check, whether they fit better.Sequencing results showed, that the AP clones from which the PCRs for the exchange of ORIs have been tried, is contaminated. The other two AP purifications showed no contamination as one of them showed clear results over the whole sequencing length, this one was used in all further experiments.Since the aim of generating different AP variants could not be achieved yet, the next experiments should enable this aim. To increase the chance of success of the following PCRs, the 3.6 kb fragment was split in 2 smaller fragments (2.7 kb and 900 bp). Those fragments were meant to be amplified in separate PCR reactions. Only one of these PCR reactions succeeded and could be isolated from the agarose gel. For the other PCR the trouble shooting protocol was consulted again to optimize PCR conditions. Several different conditions (annealing temperature changed, annealing temperatur + primer concentration, cycler run have been tried also in combination but did not lead to the desired outcome.Taking one step back, the PCR of the 900 bp fragment was repeated with a different AP template and finally worked. So the second fragment could be isolated from the agarose gel subsequently. The following Gibson assembly was conducted with both backbone fragments (Gibson overhang 5-3, Gibson overhang 3-4) and the two different ORI variants (p15A and pSC101) respectively. Afterwards the particular Gibon assembly products containing the different ORIs were transformed into chemically competent cells and plated on LB-Carbenicillin plates. To test the grown colonies, colony PCRs were performed which showed promising results so the corresponding growths were purified and sent in to GATC for sequencing. The sequencing results confirmed the desired outcome. So Glycerol stocks have been prepared from all different AP variants with pBR ORI, p15A ORI and pSC101 ORI.
</H2><H2 id="-2-preparation-of-ap-pbr-ori-for-predcel-"><STRONG>2. Preparation of AP pBR ORI for PREDCEL:</STRONG>
In order to test the circuits in PREDCEL, the AP with pBR ORI was transformed into 1030 and 2060 electro competent cells and plated. The colonies of these trafos were picked and corresponding growths were inoculated. In parallel colony PCRs were performed to check the Trafo outcome. All of the picked clones showed excellent results. After the colony PCRs were repeated from the growths and did not show clear results, one of the other sequenced APs was transformed into 1030 and 2060 electro competent cells and into chemically competent cells as well. These Trafos have also been checked by colony PCR the next day. As this showed the desired results, glycerol stocks were prepared from the corresponding growths and the growths were plated on LB-Tet plates to select for the F-Plasmid, which is crucial for phage infection.After the sequence of the other two AP variants (ORI p15A and pSC101) were confirmed, they were transformed into 1030 electro competent cells as well. The three different ORI strenghts (high copy-low copy) allow gradual adaption in the following PREDCEL experiments. 
</H2><H2 id="-3-test-constructs-for-riboswitch-and-cyp1a2-"><STRONG>3. Test constructs for Riboswitch and CYP1A2:</STRONG>
Started to transform the pSB4K5 (iGEM Distribution Kit 2017, Plate 4, well 6H) and pSB1C3 (iGEM Distribution Kit 2017, Plate 4, well 4B). But still waiting for all primers necessary to build test constructs.
</H2><H2>Modeling</H2>
Bugfixes and Discovery of new bugs. A first set of experiments was designed to adjust the model to the real behaviour of the experiment. Focus lies in the development of the phage titer under different conditions.
The numeric PREDCEL model contained a mistake in the binding constant between phage and E. coli, k  and was therefore rerun. Compared to the last results the corrected version has better results for longer time in the lagoons and higher starting concentrations of phage.A first version of the iGEM goes green page with calculations for the medium consumption went online. This also was the first real application of in line LaTex as well as plotly.js on the wiki.The numeric model with a fitness distribution instead of a single fitness value for a lagoon was expaneded by adding a mutation model based on normally distributed changes in fitness that is symmetric to the previous value for each value. The next improvement will probabbly be a parameter that allows for assymetric changes, as most mutations decrease fitness.
<H2>Optogenetics</H2><H2 id="sp-opto">SP Opto</H2>
The SP Golden Gate was repeated using a slightly different protocol:
GG of Opto SP (30 µl Reaction):

<DIV class="col-lg-12 col-md-12 col-xs-12 content" style="padding-top: 20px; padding-bottom: 20px"><DIV class="table-slot" style="overflow-x: auto"><TABLE class="table table-bordered mdl-shadow--4dp" xsscleaned="overflow-x: scroll !important"><THEAD><TR><TH>reagents</TH><TH>Volume [µl]</TH></TR></THEAD><TBODY><TR><TD>SP_BB (Purification ID: 669)</TD><TD>19.3</TD></TR><TR><TD>EL_222 (Purification ID: 520)</TD><TD>0.65 (of a 1:10 dilution)</TD></TR><TR><TD>BsaI</TD><TD>2</TD></TR><TR><TD>T4 Ligase</TD><TD>2</TD></TR><TR><TD>t4 Ligase Buffer</TD><TD>3</TD></TR></TBODY></TABLE></DIV></DIV>

Cycler conditions were as follows:

<DIV class="col-lg-12 col-md-12 col-xs-12 content" style="padding-top: 20px; padding-bottom: 20px"><DIV class="table-slot" style="overflow-x: auto"><TABLE class="table table-bordered mdl-shadow--4dp" xsscleaned="overflow-x: scroll !important"><THEAD><TR><TH>Temperature [°C]</TH><TH>Time [min]</TH><TH>Cycles</TH></TR></THEAD><TBODY><TR><TD>37</TD><TD>3:00</TD><TD>25</TD></TR><TR><TD>16</TD><TD>4:00</TD><TD>25</TD></TR><TR><TD>50</TD><TD>5:00</TD><TD>1</TD></TR><TR><TD>80</TD><TD>5:00</TD><TD>1</TD></TR></TBODY></TABLE></DIV></DIV>

About 5 µl of the GG product was transferred into a fresh 1.5-ml reaction tube, the remaining 25 µl were purified using the QIAquick PRC Purification Kit and eluted in 30 µl ddH<SUB>2</SUB>0.The GG products (unpurified and purified) were transformed into electrocompetent S1030 cells and recovered with SOC. Half of the culture was cultivated for 6 h, the other half was cultivated more than 14 h at 37 °C in a shaker. All four samples were used to perform a Blue Plaque Assay.On the next day each sample revealed plaques. Plaques were picked and a SP detection PCR was performed.The primers JM_068_rev and JM_069_fwd were used and a pipette tip containing phages of a single plaque was plunged into the 25 µl OneTaq reaction mix.
The cycler conditions were as follows:

<DIV class="col-lg-12 col-md-12 col-xs-12 content" style="padding-top: 20px; padding-bottom: 20px"><DIV class="table-slot" style="overflow-x: auto"><TABLE class="table table-bordered mdl-shadow--4dp" xsscleaned="overflow-x: scroll !important"><THEAD><TR><TH>Phase</TH><TH>Temperature [°C]</TH><TH>Time [min]</TH><TH>Cycles</TH></TR></THEAD><TBODY><TR><TD>Initial Denaturation</TD><TD>94</TD><TD>5:00</TD><TD>1</TD></TR><TR><TD>Denaturation</TD><TD>94</TD><TD>0:25</TD><TD>30</TD></TR><TR><TD>Annealing</TD><TD>51</TD><TD>0:40</TD><TD>30</TD></TR><TR><TD>Extension</TD><TD>68</TD><TD>2:00</TD><TD>30</TD></TR><TR><TD>Final Extension</TD><TD>72</TD><TD>5:00</TD><TD>1</TD></TR><TR><TD>Hold</TD><TD>10</TD><TD>∞</TD><TD>1</TD></TR></TBODY></TABLE></DIV></DIV>

Analysis by gel electrophoresis revealed the expected band of 1.9 kb (purified GG product, cultivated for more than 14 h).
Four plaques of this plate were used to infect four 4 ml (2xYT + carbenicillin) <EM>E. coli</EM> culture (stock ID: 47) with an OD600 of 0.4. Cultures were cultivated for around 14 h and centrifuged at 8000 xg for 5 min to receive the phage containing supernatants.
<H2 id="ap-opto">AP Opto</H2>
The trafos of the APs revealed 2-5 colonies on each plate. A growth of two colonies of each AP was prepared in 4 ml 2xYT medium with carbenicillin.
On the next day, a 2 ml glycerol stock (25 %) of each clone was prepared and the remaining cell suspension (3 ml) was purified using the QIAprep spin miniprep kit.The AP plasmid purification was analyzed by sanger sequencing (GATC).
The sequence was not as expected. It turns out, that for amplification of the AP_BB_light and AP_BB_dark the primers CG_005_rev and CG_008_rev were mixed-up. The purification of AP_BB_light is actually AP_BB_dark and the other way around. As the overhangs in the GG did not fit, it is hardly surprising that the transformation efficiency was that low.
<H2 id="golden-gate-assembly-of-aps-">Golden Gate assembly of APs:</H2>
The Golden Gate of the APs was repeated using the right fragments:
GG of Opto APs (15 µl Reaction):
AP_light

<DIV class="col-lg-12 col-md-12 col-xs-12 content" style="padding-top: 20px; padding-bottom: 20px"><DIV class="table-slot" style="overflow-x: auto"><TABLE class="table table-bordered mdl-shadow--4dp" xsscleaned="overflow-x: scroll !important"><THEAD><TR><TH>Reagents</TH><TH>Volume [µl]</TH></TR></THEAD><TBODY><TR><TD>gIII_luxAB (Purification ID: 514)</TD><TD>1.9</TD></TR><TR><TD>AP_light_BB (Purification ID: 517)</TD><TD>0.5</TD></TR><TR><TD>pBLind</TD><TD>1 (of a 1:100 dilution)</TD></TR><TR><TD>BsaI</TD><TD>1</TD></TR><TR><TD>Bsa</TD><TD>1.5</TD></TR><TR><TD>T4 Ligase</TD><TD>1</TD></TR><TR><TD>t4 Ligase Buffer</TD><TD>1.5</TD></TR></TBODY></TABLE></DIV></DIV>

AP_dark

<DIV class="col-lg-12 col-md-12 col-xs-12 content" style="padding-top: 20px; padding-bottom: 20px"><DIV class="table-slot" style="overflow-x: auto"><TABLE class="table table-bordered mdl-shadow--4dp" xsscleaned="overflow-x: scroll !important"><THEAD><TR><TH>Reagents</TH><TH>Volume [µl]</TH></TR></THEAD><TBODY><TR><TD>gIII_luxAB (Purification ID: 514)</TD><TD>1.9</TD></TR><TR><TD>AP_dark_BB (Purification ID: 516)</TD><TD>0.5</TD></TR><TR><TD>pBLrep</TD><TD>1 (of a 1:100 dilution)</TD></TR><TR><TD>BsaI</TD><TD>1</TD></TR><TR><TD>Bsa</TD><TD>1.5</TD></TR><TR><TD>T4 Ligase</TD><TD>1</TD></TR><TR><TD>t4 Ligase Buffer</TD><TD>1.5</TD></TR></TBODY></TABLE></DIV></DIV>

Cycler conditions were as follows:

<DIV class="col-lg-12 col-md-12 col-xs-12 content" style="padding-top: 20px; padding-bottom: 20px"><DIV class="table-slot" style="overflow-x: auto"><TABLE class="table table-bordered mdl-shadow--4dp" xsscleaned="overflow-x: scroll !important"><THEAD><TR><TH>Temperature [°C]</TH><TH>Time [min]</TH><TH>Cycles</TH></TR></THEAD><TBODY><TR><TD>37</TD><TD>3:00</TD><TD>25</TD></TR><TR><TD>16</TD><TD>4:00</TD><TD>25</TD></TR><TR><TD>50</TD><TD>5:00</TD><TD>1</TD></TR><TR><TD>80</TD><TD>5:00</TD><TD>1</TD></TR></TBODY></TABLE></DIV></DIV>

The GG products were transformed into chemically competent Top 10 cells and plated on amp-plates.On the next day each plate revealed many single colonies. Two colonies of each plate were picked and cultivated in 4 ml 2xYT with carbenicillin overnight.
A glycerol stock (25 %) was prepared and the remaining 3 ml of the overnight culture was purified with the QIAprep spin miniprep kit.
<H2>Protein_Interaction</H2><H2 id="-week-36-"><STRONG>Week 36:</STRONG>
After some weeks of doing research to find a fitting protein to evolve in protein-interaction(PI)-context, one decided to evolve split variants of T7RNAP and dCas9 to reassemble more efficiently. Both of them are commonly used proteins in biotechnological context and thereby characterized well. Protein structures of the proteins are accessible and basic literature is available. Nevertheless, split variants have not been evolved to reassemble better, yet. Four matching split sizes for T7 RNAP and one for dCas9 were chosen and in consequence first primers have been designed to produce the respective APs and SPs. Primers were tested for primer-dimerization and secondary structures. Primer products will be fused using GoldenGate cloning based on BsaI and BsmBI restriction sites in primer extensions.
</H2><H2>Software</H2><H2 id="embedding">Embedding</H2>
The embedding script did not really support restoring, that was fixed. The first embedding on uniprot that reached the second epoch was trained. Finding the best hyperparameters, especially the embedding dimension, and the length of the k-mers the embedding is based on, still remains. Checkpoints are saved as picklefiles now, making them accessible for applications without tensorflow.
</DIV></DIV></DIV><DIV class="panel panel-default"><A data-toggle="collapse" data-parent="#accordion" href="#collapseweek_id-9"><DIV class="panel-heading">
            Calendar Week
37
        </DIV></A><DIV id="collapseweek_id-9" class="panel-collapse collapse"><DIV class="panel-body"><H2>Cytochrome_Engineering</H2><H2 id="-1-preparation-of-aps-for-predcel-"><STRONG>1. Preparation of APs for PREDCEL:</STRONG>
After all AP variants have been generated and their sequences were confirmed, electro competent cell strains were produced (according to our protocol for electro competent cells), each containing one ORI variant.To start the first PREDCEL attempt the AP 1030 pBR ORI competent cells were chosen, as this is the strongest ORI and serves as a control to see whether something does happen or not. At the same time the AP 1030 pBR competent cells were transformed with MP6 to also generate a strain that produces mutations for later PREDCEL experiments.
</H2><H2 id="-2-first-predcel-run-"><STRONG>2. First PREDCEL run:</STRONG>
The first PREDCEL run was conducted with AP 1030 competent cells (lacking MP6). The aim of this experiment was to see whether the addition of theophylline or caffeine to the AP strain (containing gene 3 and the chaperone protein for the CYP1A2) lead to an increasing phage titer, as gene 3 is crucial for phage propagation. Based on literature, theophylline and caffeine concentrations have been defined that were used in this experiment. So we chose to add 100 �M theophylline, 500 �M caffeine and 5mM caffeine. Also two different sample time points have been defined, after one hour and after 16 hours. The outcome of this experiment was checked by Plaque Assyas. Not to forget to mention, that besides the Theophylline SP also Dickinsons SP was used as a control.<DIV class="col-lg-12 col-md-12 col-xs-12 content" style="padding-top: 20px; padding-bottom: 20px"><DIV class="table-slot" style="overflow-x: auto"><TABLE class="table table-bordered mdl-shadow--4dp" xsscleaned="overflow-x: scroll !important"><THEAD><TR><TH>Phase</TH><TH>Temperature [°C]</TH><TH>Time [min]</TH><TH>Cycles</TH></TR></THEAD><TBODY><TR><TD>Initial Denaturation</TD><TD>94</TD><TD>5:00</TD><TD>1</TD></TR><TR><TD>Denaturation</TD><TD>94</TD><TD>0:25</TD><TD>30</TD></TR><TR><TD>Annealing</TD><TD>51</TD><TD>0:40</TD><TD>30</TD></TR><TR><TD>Extension</TD><TD>68</TD><TD>2:00</TD><TD>30</TD></TR><TR><TD>Final Extension</TD><TD>72</TD><TD>5:00</TD><TD>1</TD></TR><TR><TD>Hold</TD><TD>10</TD><TD>∞</TD><TD>1</TD></TR></TBODY></TABLE></DIV></DIV>

Both samples exhibited the expected 1.9 kb band on the agarose gel.
The remaining PCR product was purified with the QIAquick PCR Purification Kit and sent to GATC for Sanger sequencing (using primer JM_068_rev).The sequencing results revealed the right sequence! We have a infectious phage containing EL222.
To determine the phage titer of the Opto_SP_Klon-8 supernatant, a Blue Plaque Assay was performed.According to this plaque assay the titer of the Opto_SP_Klon-8 supernatant is 2.6*10<SUP>15</SUP> PFU/ml.
</H2><H2 id="ap-opto">AP Opto</H2>
The purified APs (2x AP_light, 2x AP-dark) were sent to GATC for Sanger sequencing (using primer JM_065_rev). The sequencing results revealed the right sequence! We have the right AP containing the promoters pBLind and pBLrep in front of gene III.
Next, growths of AP_light-1 and AP_dark-1 were prepared in 4 ml 2xYT medium with carbenicillin. Plasmids were purified using QIAprep Spin Miniprep Kit.
Trafos were performed transforming AP_light and AP_dark in S1030 and S1030 containing MP6. After one hour recovery in SOC, cells were plated on the respective plate.Only the plate with AP_dark in S1030 cells contained cultures, which were very close and had to be plated on an other plate.
Trafos were repeated with different strains, recovery media, plates, and purifications without success.
A restriction double digest of the APs with HindIII-HF and SpeI-HF was performed and analyzed by gel electrophoresis. The expected bands were slightly visible on the gel, but also smear.
<H2>Phage_Propagation_and_Quantification</H2><H2 id="simplification-of-blue-plaque-assays">Simplification of Blue Plaque Assays</H2>
A 50 ml <EM>E. coli</EM> culture (ID:47) was cultivated in 2xYT with carbenicillin until an OD600 of 0.8. 3 ml of this culture were used to perform a Blue Plaque Assay. The remaining culture was put on ice for 10 min and was afterwards stored at 4 °C in the fridge. On the following five days, a Blue Plaque Assay was performed in the evening. Every Plaque Assay contained the 10<SUP>-11</SUP> dilution of the Dickinson phage target 133 (1x10<SUP>15</SUP> PFU/ml) in triplicates and a negative control. The plaque assays were performed using the  <EM>E. coli</EM> culture from the fridge.
Plaque assays worked for four days with a similar number of plaques. Therefore, it is confirmed that <EM>E. coli</EM> cultures for plaque assays can be stored in the fridge for at least four days.
<H2>Protein_Interaction</H2><H2 id="-week-37-"><STRONG>Week 37:</STRONG>
After primers had been delivered, DNA was resolved and diluted. First, all planned PCRs (see <EM>Awesomesheet</EM> for detailed conditions and protocol for general procedure for all named methods) were performed to get the primer products (PCR product IDs 357-375), that are needed for Golden Gate Cloning (starts with PCR ID <EM>1321</EM>) using Q5 polymerase. Afterwards, PCR products were analyzed on gel and purified via gel extraction or PCR purification. PCRs that did not work in the beginning were optimized by altering annealing temperatures, temperature ramp rates and DMSO and Betain concentrations. Switching from Q5 to Fusion polymerase worked too in some cases. This week PCR products with IDs 358; 359; 373; 374; 375 could be produced with matching band length on gels.
</H2><H2>Software</H2><H2 id="embedding">Embedding</H2>
Training of the embedding on Uniprot continues. First approaches to optimize the current multilabel classifier by adding parameters to it, either in depth or in width started.The concept of a generator that models natural evolution was designed. There, a starting sequence is exchanged by randomly chosen point mutations, which are then scored by the classifier with a sigmoid output for each label. Sequences that score higher are more likely to be chosen to continue working with. Those will then be changed by point mutations again. In this scenario the selection is modeled by a function that uses the classifier. However it remains unclear, if the classifier supports gradual improvements in a direction that improves the function in reality and in the classifier.
</DIV></DIV></DIV><DIV class="panel panel-default"><A data-toggle="collapse" data-parent="#accordion" href="#collapseweek_id-10"><DIV class="panel-heading">
            Calendar Week
38
        </DIV></A><DIV id="collapseweek_id-10" class="panel-collapse collapse"><DIV class="panel-body"><H2>Cytochrome_Engineering</H2><H2 id="-1-big-predcel-run-"><STRONG>1. Big PREDCEL Run:</STRONG>
After the first PREDCEL run did not show any plaques in the Plaque Assay it was repeated. Both Plaque Assays then showed a positive result with blue plaques. The results have been confirmed by a conducted plaque PCR. A phage titer of 1,2 * 10^9 could be determined.After the first PREDCEL Run was conducted with only one ORI variant (AP 1030 pBR ORI competent cells lacking MP6), the experiment was repeated using all three ORI variants under the same conditions (adding 100 �M theophylline, 500 �M caffeine and 5mM caffeine). Also the sample drawing was performed at the same time, after one hour and after 16 hours, for better comparison with the previous experiment. The outcome of this experiment was checked by Plaque Assyas. Again the Dickinsons SP was used as a control. In all three ORI experiments the PREDCEL run seemed successful, but the most promising results could be achieved with the pSC101 ORI. So for further experiments this ORI variant was chosen.As in this big experiment the dilutions fort he control (adding neither theophylline nor caffeine) have been missed in the plaque Assay it was repeated fort he p15A ORI and the pSC101 ORI for both sample time points.Another PREDCEL run was conducted adding a number of different caffeine concentrations (0 �M, 500 �M, 200 �M, 100 �M, 50 �M, 30 �M, 10 �M, 5 �M) to find the threshold where we can detect how much caffeine is needed to be converted into theophylline to have an effect on the phage propagation (evidence for activity of CyP1A2).
</H2><H2 id="-riboswitch-and-cyp1a2-testing-"><STRONG>Riboswitch and CYP1A2 testing:</STRONG>
Next, we planned to use the HPLC to see how precisely we can detect theophylline and caffeine. So all necessary buffers and samples have been prepared.
</H2><H2>Modeling</H2>
Equations for modeling the glucose concentration in PACE lagoon and in PALE lagoon were developed. The model that remains to be build can help finding the ideal glucose concentration in the medium. That means glucose concentration that are high enough to suppress the MPs before their induction, and low enough to allow for their induction.
<H2>Optogenetics</H2><H2 id="ap_dark_ppsp-el222-br">AP_dark_Ppsp-EL222-BR</H2>
To ensure that gIII expression is only activated after phage infection, a new Promoter: Ppsp-EL222-BR was developed. This is necessary to prevent cells from becoming resistancetowards phage infection as a consequence of the continuous gIII expression. The psp-Promoter and pBLrep were combined based on the paper &quot;Negative selection and stringency modulation in phage-assisted continuous evolution&quot;.The tet-binding region of the psp-tet promoter was exchanged by the EL222 binding region, which should lead to a gIII expression only in the dark and after phage infection.
The psp-promoter was amplyfied using AP_destructase_SD2-luxAb (Purification ID: 536) as template and the primers CG_014 and CG_015 in a 50 µl Q5 reaction mix.The cycler conditions were as follows:

<DIV class="col-lg-12 col-md-12 col-xs-12 content" style="padding-top: 20px; padding-bottom: 20px"><DIV class="table-slot" style="overflow-x: auto"><TABLE class="table table-bordered mdl-shadow--4dp" xsscleaned="overflow-x: scroll !important"><THEAD><TR><TH>Phase</TH><TH>Temperature [°C]</TH><TH>Time [min]</TH><TH>Cycles</TH></TR></THEAD><TBODY><TR><TD>Initial Denaturation</TD><TD>98</TD><TD>0:30</TD><TD>1</TD></TR><TR><TD>Denaturation</TD><TD>98</TD><TD>0:10</TD><TD>30</TD></TR><TR><TD>Annealing</TD><TD>67</TD><TD>0:25</TD><TD>30</TD></TR><TR><TD>Extension</TD><TD>72</TD><TD>0:04</TD><TD>30</TD></TR><TR><TD>Final Extension</TD><TD>72</TD><TD>2:00</TD><TD>1</TD></TR><TR><TD>Hold</TD><TD>10</TD><TD>∞</TD><TD>1</TD></TR></TBODY></TABLE></DIV></DIV>

PCR product was analyzed by gel electrophoresis and the 170 bp fragment was purified with the QIAquick gel extraction kit (Purification ID: 852).
Golden Gate of AP_dark_Ppsp_EL222-BR
To generate the AP with gIII under control of Ppsp-EL222-BR, Golden Gate cloning was performed.
AP_dark_Ppsp_EL222-BR

<DIV class="col-lg-12 col-md-12 col-xs-12 content" style="padding-top: 20px; padding-bottom: 20px"><DIV class="table-slot" style="overflow-x: auto"><TABLE class="table table-bordered mdl-shadow--4dp" xsscleaned="overflow-x: scroll !important"><THEAD><TR><TH>reagents</TH><TH>Volume [µl]</TH></TR></THEAD><TBODY><TR><TD>gIII_luxAB (Purification ID: 514)</TD><TD>1.9</TD></TR><TR><TD>AP_dark_BB (Purification ID: 516)</TD><TD>0.5</TD></TR><TR><TD>Ppsp-EL222 (Purification ID: 857)</TD><TD>1.2</TD></TR><TR><TD>BsaI</TD><TD>1</TD></TR><TR><TD>BSA</TD><TD>1.5</TD></TR><TR><TD>T4 Ligase</TD><TD>1</TD></TR><TR><TD>t4 Ligase Buffer</TD><TD>1.5</TD></TR></TBODY></TABLE></DIV></DIV>

Cycler conditions were as follows:

<DIV class="col-lg-12 col-md-12 col-xs-12 content" style="padding-top: 20px; padding-bottom: 20px"><DIV class="table-slot" style="overflow-x: auto"><TABLE class="table table-bordered mdl-shadow--4dp" xsscleaned="overflow-x: scroll !important"><THEAD><TR><TH>Temperature [°C]</TH><TH>Time [min]</TH><TH>Cycles</TH></TR></THEAD><TBODY><TR><TD>37</TD><TD>3:00</TD><TD>15</TD></TR><TR><TD>16</TD><TD>4:00</TD><TD>15</TD></TR><TR><TD>37</TD><TD>30:00</TD><TD>1</TD></TR><TR><TD>65</TD><TD>0:05</TD><TD>1</TD></TR></TBODY></TABLE></DIV></DIV>

TransformationGolden Gate product was transformed into Top 10 (chemically transformed) and cells were plated on a LB-agar-Amp plate.
<H2 id="ap_light-and-ap_dark">AP_light and AP_dark</H2>
Glycerol stocks of AP_light and AP_dark were prepared from overnight cultures.
<H2>Phage_Propagation_and_Quantification</H2><H2 id="purification">Purification</H2>
Theophylline SP and Dickinson SP were propagated and purified according to the NEB protocol &quot;M13 Amplification&quot;. Phage titer was determined by Blue Plaque Assays. It was possible to reach phage titer of (1x10<SUP>15</SUP> PFU/ml) and increase the number of phages per ml by a factor of 100by step 4 to 6.
<H2 id="quantification">Quantification</H2>
A quantification method by spectrophotometry was tested according to &quot;m13 bacteriophage production for large-scale applications&quot;, but as the phage titer differed more than 10<SUP>3</SUP> PFU/ml from plaque assay results the method was considered as unsuitable.
<H2>Protein_Interaction</H2><H2 id="-week-38-"><STRONG>Week 38:</STRONG>
PCRs for primer products that could not be produced in the previous week (PCR product IDs 357; 360 - 372) were repeated under altered conditions. Thereby, all the PCRs except PCR products IDs 358, 369 and 373 were performed successfully. As PCR products ID 369 (AP of Split T7RNAP backbone with GenIII) is not absolutely necessary because it can be replaced equivalently by PCR product ID 370 (AP of Split T7RNAP backbone with GenIII and YFP) all needed PCR products for T7 RNAP cloning were present to start GoldenGate assembly.In contrast PCR products with IDs 358 and 373 are crucial for the dCas9 experiment and could not be produced in any way. Therefore focus moved on cloning T7 RNAP constructs alone rather than trying to manage both, the T7 RNAP and the dCas9 cloning, to work.
</H2><H2>Software</H2><H2 id="pixelcnn">PixelCNN</H2>
Adapting the pixelCNN from a 2-dimensional problem with three channels to a 1-dimensional one with 20 channels began. The network itself works but the logistic sampling function that generates the sequence in the end and the custom loss function are hard to adapt.
<H2 id="gaia">GAIA</H2>
The concept of a genetical artificial intelligent algorithm (GAIA)  that is based on the classifier was developed. Starting from a given sequence it maximizes given labels and minimizes other given labels. At the same time the garbage score of the sequence is minimized. The model starts with one sequence but always works with B sequences. Therefore the starting sequence is in silico mutated B times to yield B different sequences. Those are then classified by DeeProtein and the outputs are used to calculate a sequence score. A given number of best sequences is selected and again mutated in silico to yield B sequences. This cycle of mutation and selection repeats and optimizes the starting sequence.Additionally it is possible to also use recombination after the mutation to combine good subsequences
</DIV></DIV></DIV><DIV class="panel panel-default"><A data-toggle="collapse" data-parent="#accordion" href="#collapseweek_id-11"><DIV class="panel-heading">
            Calendar Week
39
        </DIV></A><DIV id="collapseweek_id-11" class="panel-collapse collapse"><DIV class="panel-body"><H2>Cytochrome_Engineering</H2><H2 id="-1-predcel-"><STRONG>1. PREDCEL:</STRONG>
The plaque assay with different caffeine concentrations did not work as the control (Dickinson phage) could not be detected. The results for the SP theophylline could not be clearly regarded as reliable then. To check for the SP Dickinson and to validate the results for SP theophylline the plaque assay was once more repeated. But unfortunately the negative control in this plaque assay showed a contamination. In order to get rid of the contamination, we performed several following plaque assays using completely new prepared materials.Also new SP theophylline needed to be reproduced (from phage supernatant + stock 47 in 2YT + Carb)
</H2><H2 id="-2-test-constructs-for-riboswitch-and-cyp1a2-"><STRONG>2. Test constructs for Riboswitch and CYP1A2:</STRONG>
The next step, was a double transformation of the pSB4K5 (riboswitch) and pSB1C3 (CYP1A2) in Top 10 chemically competent cells. The colonies were picked to grow growths that are used for further testing. The growth was diluted 1:100 and caffeine or theophylline in different concentrations were added (5 mM, 500 �M,  50 �M, 5 �M, 0,5 �M) respectively. To ensure induction of the Lac Promotor IPTG was added (1:1000) as well as the required antibiotics (Kanamycin and Chloramphenicol).
</H2><H2 id="-3-hplc-test-"><STRONG>3. HPLC Test:</STRONG>
Several samples have been run over the C18 column of the HPLC (theophylline 1 mg/ml, caffeine 1 mg/ml and a mix of theophylline and caffeine 1:1). The mix showed isolated peaks at about 5.92 min for theophylline and 9.23 min for caffeine. The test runs with the pure samples could not be conducted by implication (probably operating mistakes due to little experience with the HPLC).
</H2><H2>Optogenetics</H2><H2 id="ap-opto">AP Opto</H2>
A colony of Top 10 cells, which was transformed with AP_dark_Ppsp_EL222-BR was picked and grown overnight in 4 ml 2xYT + carbenicillin. Out of this culture and an overnight culture of AP_light in S1030 cells the plasmids were purified according to the QIAGEN mini prep protocol and sent to GATC for sequencing.
The DNA sequence of the AP_dark_Ppsp_EL222-BR out of Top 10 cells was correct, whereas the cells, which were thought to be AP_light (in S1030) exhibited the sequence of AP_dark. There must be a contamination or mix-up of AP_light and AP_dark.  
The purified plasmid AP_dark_Ppsp-EL222-BR was transformed in S1030 by electroporation and plated on Amp/Tet plates. Tet is necessary to make sure that the <EM>E. coli</EM> still carry the F-plasmid for phage infection.On the next day, colonies were picked an grown in 4 ml 2xYT overnight. A glycerol stock of the overnight culture was prepared, and the remaining culture was used for plasmid purification.
Plasmid purifications of AP_light, AP_dark and AP_dark_Ppsp_EL222-BR out of S1030 were send to GATC for sequencing. The sequence of AP_dark_Ppsp_EL222-BR was correct, but AP_dark as well as AP_light exhibited the sequence of AP_dark.
<H2>Protein_Interaction</H2><H2 id="-week-39-"><STRONG>Week 39:</STRONG>
GoldenGate cloning for all T7 RNAP constructs were performed and transformed into <EM>S1030 cells</EM> via electroporation. Transformation for phages worked properly, but AP transformation first did not show any cultures on plate to pick for glycerol stocks (plasmid IDs 305 + 306). Therefore, AP-transformation was repeated with <EM>2060 cells</EM>, which worked.
</H2><H2>Software</H2><H2 id="embedding">Embedding</H2>
Embedding on Uniprot is calculated over 15 epochs, vectors that are close to each other in the high dimensional space seem to be similar on sequence level on first sight.Genetic artificial intelligent algorithm is implemented. Features are mutation of a given maximum of residues, mutation of specific residues. Those are inserted in the model in small letters while the rest of the sequence is written in capital letters. Furthermore weighting arbitrary numbers of goal- and avoid- GO-terms are possible. The variance of the given classes, a measure for the authenticity of a protein, is substracted from the overall score. Recombination of sequences was not implemented as it corrupts maintaining less then the maximum allowed mutations in a sequence.
</DIV></DIV></DIV><DIV class="panel panel-default"><A data-toggle="collapse" data-parent="#accordion" href="#collapseweek_id-12"><DIV class="panel-heading">
            Calendar Week
40
        </DIV></A><DIV id="collapseweek_id-12" class="panel-collapse collapse"><DIV class="panel-body"><H2>Cytochrome_Engineering</H2><H2 id="-1-prdecel-"><STRONG>1. PRDECEL:</STRONG>
After the plaque assay with different caffeine concentrations did not work and also the repeated experiment, it needed to be conducted from the beginnig. Therfore, new phages had to be reproduced to have a sufficient amount of phages to inoculate the new PREDCEL culture. The phage reproduction worked well, a titer of 1*10^10 could be achieved. This could then be used to inoculate the new PREDCEL culture in order to repeat the whole process. The outcome was checked by plaque assay afterwards. The plaque assay worked well this time, but no significant difference between caffeine (in different concentrations) and no caffeine could be detected. So we came to the conclusion, that in order to meassure the CYP1A2 activity another assay has to be planned.Furhtermore, the first PREDCEL run with passaging was conducted (in triplicates, and 100 �M and 1000 �M theophylline were used). As most of the reproduced phages were used up for this experiment, new phages had to be reproduced again. The titer was then checked by plaque assay as well as the outcome of the PRDCEL run. Checking all triplicates from all sample timepoints would have been to laborious for a frist check, so only the first, the second and the last time point were checked by plaque assay. There one could see, that the last timepoint did not show plaques, which indicates that the five passages could not be conducted successfully. Nevertheless, some of the passages might be successful and the samples inbetween the second and the last timepoint were checked by plaque assay afterwards.
</H2><H2 id="-2-transformmation-"><STRONG>2. Transformmation:</STRONG>
Previously electrocompetent cells containing the AP and all three ORI variants have been generated. Those were now transformed with the expression plasmid containing the CYP1A2 gene (which was used in further testings before). Right after recovering the transformed cells, they were plated on LB-chloramphenicol- ampicillin-plates.All three variants could be successfully transformed and colonies were picked respectively in order to produce glycerol stocks. The next day an experiment was started to check for the CYP1A2's correct tertiary structure. To do so, growths have been inoculated from the glycerol stocks and the cells containing the CYP1A2 were then lysed by ultrasonic waves and centrifuged. Then 50 �l Disodiumdithionide (10 mg/ml) were added to 700 �l of the supernatant. The Disodiumdithionide reduces the heme group, which is essential for the catalytic activity of the CYP1A2. The heme group can only be bound, when the proper three dimensional structure of the CYP1A2 is formed. So indirectly, this experiment can indicate, whether our CYP1A2 is in its active conformation or not.
</H2><H2 id="-3-hplc-"><STRONG>3. HPLC:</STRONG>
As the HPLC test with a mixture of caffeine and theophylline (1:1) worked but the pure solutions of theophylline and caffeine did not show any peaks, the experiment was repeated several times. We assumed that operating mistakes or wrong settings are the reason for the negative test results.But as the experiments have been run under the same conditions this week again, there is probably another source of mistakes. Only in a few cases peaks could be detected, but then distinct peaks were detected.Outlook: probably an assay to meassure CYP1A2 activity.
</H2><H2>Modeling</H2>
A simple model for calculating the glucose concentration in a turbidostat from the glucose concentration in the medium and vice versa was made. A tool with a short summary of the math behind it was uploaded to the tools page of the wiki.The same model was adapted for a flask instead of a turbidostat, changing it from a steady state model to one that integrates over time. Bugfixes remain to be done.
<H2>Optogenetics</H2><H2 id="ap-opto">AP Opto</H2>
As the S1030 cells, containing AP_light, were contaminated or mixed-up with AP_dark a new AP_light strainhad to be prepared. The purifications of AP_light and AP_dark, which were already checkedby sequencing, were used to transform new S1030 cells by electroporation and plated on Amp/Tet plates. On the next day acolony was picked and grown in 4 ml 2xYT + carbenicillin overnight. A glycerol stock was prepared, and the remaining culture was used to isolate and purify the plasmid.The plasmid purifications were sent to GATC for sequencing. The sequencing results proved that the cells carry the right promoter in front of gIII.
<H2 id="ap-testing">AP Testing</H2>
To test if the SP_Opto phage can propagate in S1030 containing AP_light and AP_dark,cultures with the respective cells were grown in 20 ml 2xYT + carbenicillin to an OD600 of 0.6 and infected with either SP_Opto of the Dickinson phage target_133and grown in the dark and under blue light After three and six hours samples were taken. A plaque assay wasperformed to check if the Opto phages can propagate, but due to a phage contamination of the S2208 cells the plaque assay failed. Three additional plaque assays failed, as the glycerol stock and pipettes were contaminated.
<H2>Protein_Interaction</H2><H2 id="-week-40-"><STRONG>Week 40:</STRONG>
This week was mainly about checking the phages to have the right SP inserts, which came along with lots of troubleshooting. Test PCRs did not or not only show the expected band lengths. As a band appeared, which length matches phage backbone size without insert and as well PCR product of Dickinson phage, a phage construct used for validation of our PACE device, contamination of samples could not be excluded. Therefore at the beginning PCR was optimized and transformation was repeated. As problem could not be solved, transformation was repeated a second time and new primers were designed showing significant differences in product length. New PCR needed optimization as well, but finally worked well and robustly, so it became standard PCR for contamination detection of PI-constructs.
</H2><H2>Software</H2><H2 id="gaia">GAIA</H2>
GAIA was expanded by allowing multiple mutations before selection takes place, a linear decay of the number of mutations before a round of selection is applied. Results look slightly better compared to one mutation before selection. With this mode epistatic mutations can be found, that means mutations that are beneficial together but decrease the score of a protein when occuring alone.
</DIV></DIV></DIV><DIV class="panel panel-default"><A data-toggle="collapse" data-parent="#accordion" href="#collapseweek_id-13"><DIV class="panel-heading">
            Calendar Week
41
        </DIV></A><DIV id="collapseweek_id-13" class="panel-collapse collapse"><DIV class="panel-body"><H2>Cytochrome_Engineering</H2><H2 id="-1-predcel-"><STRONG>1. PREDCEL</STRONG>
Another plaque assay was conducted with the samples of the firste passaging experiment to find out whether we are able to propagate phages over several passaging steps. The outcome showed that 3 passages have been conducted successfully. Subsequently, all triplicates of the last timepoint were screened in order to see if a significant difference between Theo100 and Theo1000 and no Theo can be detected (-&gt; AP works, more phages when theophylline is added, but storing samples in the refigerator reults in lower phage titers).As only three of five passages have been successfull, a next PREDCEL run was started where more pahges were transferred (1ml instead of 0.5 ml) to see whether this can lead to more successful passaging steps. This time Theo100 was tested along with different caffeine concetrations. The samples have again been checked by plaque assay. Unfortunately, there was a phage contamination in all applied dilutions, so no conclusions could be drawn.So another PREDCEL was started using the same conditions plus adding 5 mmol of 5-Aminolevunilic acid, a precursor of the heme molecule which displays an important cofactor for the CYP1A2 (this was added, as in the mean time CYP1A2 testing showed visible colour change when adding the substance, indicating higher activity of the CYP1A2). The plaque assay of this run confirmed that adding theophylline leads to faster phage propagation and showed that caffeine concentrations between 5-500 �M can also impact phage propagation.
</H2><H2 id="-2-cyp1a2-test-"><STRONG>2. CYP1A2 Test</STRONG>
To find out whether our CYP1A2 is functional or not, we repeated the dithionide experiment as no sufficient peak could be detected the first time. Small changes took place: a change of buffer was performed before ultrasonifying the cells. Also a dithionide solution was directly prapared befor meassuring the optical density at 550 nm. As this did not lead to a colour reaction, it was repeated once again adding 5-Aminolevunilic acid, which is a precursor of the heme molecule which in return displays an important factor for CYP1A2 activity. We added 5-Aminolevunilic acid, as we assumed, that mybe ther would not be sufficient amounts in e.coli available. This time a colour change could already be detected by eye, as the colour of the growths was red/brown. The optical density meassurement at 550 nm then showed a peak, indicating, that our CYP1A2 is functional.
</H2><H2>Modeling</H2>
Logistic growth was implemented for glucose modeling to be able to model larger time scales with  <EM>E. coli</EM> leaving exponential phase. Plotting was improved.
<H2>Optogenetics</H2><H2 id="weekly-summary-09-10-15-10-2017-cg">Weekly summary 09.10-15.10.2017 CG</H2><H2 id="opto-pace">Opto PACE</H2><H2 id="ap-opto">AP Opto</H2>
To prevent phage contamination all instruments and tools were cleaned, all solutions were prepared freshly and filter tips were used more consequently. It turned out that there was a phage contamination in the glycerol stock of AP_dark_Ppsp-EL222 containing S1030 cells. As these cell have a psp-promoter, phages which don't contain EL222 can propagate with and without blue light. A new colony was picked from the plate, but this colony was also already contaminated with phages.
 Therefore, AP_dark_Ppsp_EL222-BR had to be transformed again into S1030 cells. The same AP_dark_Ppsp_EL222-BR purification was used to transform S1030 cells by electroporation and cells were plated on Amp/Tet-plates. On the next day, a colony was picked and grown in 4 ml 2xYT + carbenicillin overnight. A glycerol stock was prepared, and the remaining culture was used to isolate and purify the plasmid. The plasmid purifications were sent to GATC for sequencing. The sequencing results proved that the cells carry the right promoter in front of gIII. AP_dark_Ppsp_EL222-BR is further referred as AP_dark.
<H2 id="-ap-testing"> AP Testing</H2>
 Cultures of S1030 cells containing AP_light and AP_dark were prepared and grown to an OD600 of 0.6 in 2xYT with carbenicillin. Then, this culture was split into 20 ml cultures and infected with 10 µl of either SP_Opto or Dickinson phage target_133 (titer: (~10<SUP>15</SUP> PFU/ml) according to the following table:

<DIV class="col-lg-12 col-md-12 col-xs-12 content" style="padding-top: 20px; padding-bottom: 20px"><DIV class="table-slot" style="overflow-x: auto"><TABLE class="table table-bordered mdl-shadow--4dp" xsscleaned="overflow-x: scroll !important"><THEAD><TR><TH>Strain</TH><TH>Phage</TH><TH>Condition</TH></TR></THEAD><TBODY><TR><TD>AP_light</TD><TD>SP_Opto</TD><TD>blue light</TD></TR><TR><TD>AP_light</TD><TD>SP_target_133</TD><TD>blue light</TD></TR><TR><TD>AP_dark</TD><TD>SP_Opto</TD><TD>blue light</TD></TR><TR><TD>AP_dark</TD><TD>SP_target_133</TD><TD>blue light</TD></TR><TR><TD>AP_light</TD><TD>SP_Opto</TD><TD>dark</TD></TR><TR><TD>AP_light</TD><TD>SP_target_133</TD><TD>dark</TD></TR><TR><TD>AP_dark</TD><TD>SP_Opto</TD><TD>dark</TD></TR><TR><TD>AP_dark</TD><TD>SP_target_133</TD><TD>dark</TD></TR></TBODY></TABLE></DIV></DIV>

After infection the culture was mixed and samples were taken. Cultures were incubated for one hour at 37 °C in the shaker, half of them in the dark, the other half was illuminated with blue light pulse 25 % (15 sec. on, 45 sec. off). Afterwards samples were taken. All sampels were centrifuged at 8000 xg for 3 min in a table centrifuge. The supernatant was transferred into a fresh reaction tube and stored at 4 °C in the fridge.
These samples were used to perform a plaque assay using 10 µl phage supernatant per quarter of the plate. Plaque assays was performed according to the Blue Plaque Assay protocol. On the next day, plaques were counted to determine the phage titer:

<DIV class="col-lg-12 col-md-12 col-xs-12 content" style="padding-top: 20px; padding-bottom: 20px"><DIV class="table-slot" style="overflow-x: auto"><TABLE class="table table-bordered mdl-shadow--4dp" xsscleaned="overflow-x: scroll !important"><THEAD><TR><TH>Strain</TH><TH>Phage</TH><TH>Time [h]</TH><TH>Condition</TH><TH>Phage titer [PFU/ml]</TH></TR></THEAD><TBODY><TR><TD>AP_light</TD><TD>SP_Opto</TD><TD>0</TD><TD>-</TD><TD>0.7x10<SUP>7</SUP> PFU/ml</TD></TR><TR><TD>AP_light</TD><TD>SP_target_133</TD><TD>0</TD><TD>-</TD><TD>0.6x10<SUP>7</SUP> PFU/ml</TD></TR><TR><TD>AP_light</TD><TD>SP_Opto</TD><TD>1</TD><TD>blue light</TD><TD>2.2x10<SUP>7</SUP> PFU/ml</TD></TR><TR><TD>AP_light</TD><TD>SP_target_133</TD><TD>1</TD><TD>blue light</TD><TD>1.4x10<SUP>7</SUP> PFU/ml</TD></TR><TR><TD>AP_light</TD><TD>SP_Opto</TD><TD>1</TD><TD>dark</TD><TD>2,3x10<SUP>7</SUP> PFU/ml</TD></TR><TR><TD>AP_light</TD><TD>SP_target_133</TD><TD>1</TD><TD>dark</TD><TD>1.9x10<SUP>7</SUP> PFU/ml</TD></TR><TR><TD>AP_dark</TD><TD>SP_Opto</TD><TD>0</TD><TD>-</TD><TD>1.1x10<SUP>7</SUP> PFU/ml</TD></TR><TR><TD>AP_dark</TD><TD>SP_target_133</TD><TD>0</TD><TD>-</TD><TD>2.1x10<SUP>7</SUP> PFU/ml</TD></TR><TR><TD>AP_dark</TD><TD>SP_Opto</TD><TD>1</TD><TD>blue light</TD><TD>2.3x10<SUP>7</SUP> PFU/ml</TD></TR><TR><TD>AP_dark</TD><TD>SP_target_133</TD><TD>1</TD><TD>blue light</TD><TD>3.6x10<SUP>9</SUP> PFU/ml</TD></TR><TR><TD>AP_dark</TD><TD>SP_Opto</TD><TD>1</TD><TD>dark</TD><TD>6.2x10<SUP>7</SUP> PFU/ml</TD></TR><TR><TD>AP_dark</TD><TD>SP_target_133</TD><TD>1</TD><TD>dark</TD><TD>5.7x10<SUP>9</SUP> PFU/ml</TD></TR></TBODY></TABLE></DIV></DIV>

There was the right tendency for AP_light and AP_dark after one hour. For further experiments the incubation time should be increased. The Dickinson phage target_133 exhibited higher phage titer than SP_Opto. For AP_dark this is not surprising as this phage don't contain EL222 and therefore activates the psp-promoter. But it should not activate the promoter of AP_light, pBLind. As it was suspected that the Dickinson phage is superior in its propagation cycle due to e.g. packing ability, a phage containing a non-binding EL222 should be created.
<H2>Protein_Interaction</H2><H2 id="-week-41-"><STRONG>Week 41:</STRONG>
Parts of phage genome with insert sequence were amplified, sequenced and thereby approved after two to three sequencings. While SP sequencing worked AP sequencing did not show clear results in three sequencings, which is why GoldenGate on AP related PCR products was performed again. Then, APs were transformed and purified, again. As own APs were not approved by sequencings Jin69 plasmid (plasmid ID 17) was used for phage propagation testing over 24 hours.
</H2><H2>Software</H2><H2 id="gaia">GAIA</H2>
GAIA was used to generate mutations in a glucoronidase sequence, beneficial for galactosidase activity. A subset of the generated sequences was ordered as oligonucleotides for kinetic assays. In order to show that the score from the used classifier correlates with the real-world activity, a set of less active beta-lactamase sequences was predicted by gaia. Those were also bought as oligonucleotides and will be tested for their activity.A randomization function was implemented to examine how the different scores develop when the number of mutations increases. The results look sigmoid, showing almost no change in the score for the first mutations, then rapidly dropping below the threshold for positive classification, after which they only decrease slowly.A combination function to examine all possible combinations of a set of mutations was implemented in order to aid the decisions on which sequences to test in the lab.
</DIV></DIV></DIV><DIV class="panel panel-default"><A data-toggle="collapse" data-parent="#accordion" href="#collapseweek_id-14"><DIV class="panel-heading">
            Calendar Week
42
        </DIV></A><DIV id="collapseweek_id-14" class="panel-collapse collapse"><DIV class="panel-body"><H2>Modeling</H2>
Modeling the phage titer based on a distribution of fitness was improved. Mutation modeled by a skewed normal distribution was enabled.The model was compared to phage titer data from the lab and performed well. However, improvements in the scaling are needed.
All available interactive tools were unified and put into the wiki layout.
<H2>Optogenetics</H2><H2 id="weekly-summary-16-10-22-10-2017-cg">Weekly summary 16.10-22.10.2017 CG</H2><H2 id="opto-pace">Opto PACE</H2><H2 id="sp_opto_el222_non-binding">SP_Opto_EL222_non-binding</H2>
A primer was ordered that binds within the helix-turn-helix motive of EL222 and should therefore amplify a truncated non-binding variant of EL222. EL222_non-binding was amplified using the gblock EL222 expression cassette (PCR Product ID: 242) as template and the primers CG_012 and CG_016 in a 25 µl Phusion Flash reaction mix.The cycler conditions were as follows:

<DIV class="col-lg-12 col-md-12 col-xs-12 content" style="padding-top: 20px; padding-bottom: 20px"><DIV class="table-slot" style="overflow-x: auto"><TABLE class="table table-bordered mdl-shadow--4dp" xsscleaned="overflow-x: scroll !important"><THEAD><TR><TH>Phase</TH><TH>Temperature [°C]</TH><TH>Time [min]</TH><TH>Cycles</TH></TR></THEAD><TBODY><TR><TD>Initial Denaturation</TD><TD>98</TD><TD>0:10</TD><TD>1</TD></TR><TR><TD>Denaturation</TD><TD>98</TD><TD>0:01</TD><TD>25</TD></TR><TR><TD>Annealing</TD><TD>72</TD><TD>0:05</TD><TD>25</TD></TR><TR><TD>Extension</TD><TD>72</TD><TD>0:09</TD><TD>25</TD></TR><TR><TD>Final Extension</TD><TD>72</TD><TD>1:00</TD><TD>1</TD></TR><TR><TD>Hold</TD><TD>10</TD><TD>∞</TD><TD>1</TD></TR></TBODY></TABLE></DIV></DIV>

 PCR product was analyzed by gel electrophoresis and the 548 bp fragment was purified with the QIAquick gel extraction kit (Purification ID: 1006).
 Golden Gate of SP_Opto_EL222_non-binding
 To generate an SP with a non-binding EL222 variant Golden Gate cloning was performed.
 SP_Opto_EL222_non-binding

<DIV class="col-lg-12 col-md-12 col-xs-12 content" style="padding-top: 20px; padding-bottom: 20px"><DIV class="table-slot" style="overflow-x: auto"><TABLE class="table table-bordered mdl-shadow--4dp" xsscleaned="overflow-x: scroll !important"><THEAD><TR><TH>reagents</TH><TH>Volume [µl]</TH></TR></THEAD><TBODY><TR><TD>SP_Opto_EL222_non-binding (Purification ID: 1006)</TD><TD>3.56</TD></TR><TR><TD>SP_BB (Purification ID: 1007)</TD><TD>3.33</TD></TR><TR><TD>BsaI</TD><TD>1</TD></TR><TR><TD>BSA</TD><TD>1.5</TD></TR><TR><TD>T4 Ligase</TD><TD>1</TD></TR><TR><TD>T4 Ligase Buffer</TD><TD>1.5</TD></TR></TBODY></TABLE></DIV></DIV>

 Cycler conditions were as follows:

<DIV class="col-lg-12 col-md-12 col-xs-12 content" style="padding-top: 20px; padding-bottom: 20px"><DIV class="table-slot" style="overflow-x: auto"><TABLE class="table table-bordered mdl-shadow--4dp" xsscleaned="overflow-x: scroll !important"><THEAD><TR><TH>Temperature [°C]</TH><TH>Time [min]</TH><TH>Cycles</TH></TR></THEAD><TBODY><TR><TD>37</TD><TD>3:00</TD><TD>15</TD></TR><TR><TD>16</TD><TD>4:00</TD><TD>15</TD></TR><TR><TD>37</TD><TD>30:00</TD><TD>1</TD></TR><TR><TD>65</TD><TD>0:05</TD><TD>1</TD></TR></TBODY></TABLE></DIV></DIV>

 Golden Gate Product was transformed into S2208 cells by electroporation and were grown for 16 h in SOC after recovery. Culture was centrifuged at 8000 xg for 3 min and phage supernatant was transferred into a fresh tube. This supernatant was then used to perform a Blue Plaque Assay. On the next day a plaque was picked and used for an insert PCR (JM_068, JM_069) as well as for infection of a new culture of S2208 cells (OD600 of 0.6).
The PCR and subsequent gel electrophoresis exhibited a fragment of the expected length.
The infected overnight culture was centrifuged at 8000 xg for 3 min and phage supernatant was transferred into a fresh tube stored at 4 °C.
<H2>Protein_Interaction</H2><H2 id="-week-42-"><STRONG>Week 42:</STRONG></H2>
By plaque assay performance phage propagation on Jin69 was proved to work, so gene circuit can be used in PACE and/or PREDCEL. After several attempts of positive AP sequencing AP were finally proven right. Related stocks were proven not to be contaminated by contamination detection PCR. <EM>S1030 Cells</EM> containing approved APs were made electrocompetent and MP1 and MP4 were transformed into these cells separately by electroporation. For proper and complete sequencing of T7 RNAP to detect mutations new sequencing primers were designed and ordered. Furthermore to exchange SD8 RBS sequence with weaker sd8 RBS sequence GoldenGate primers were bought and used to perform GoldenGate cloning.
<H2>Software</H2><H2 id="gaia">GAIA</H2>
For further insight in properties of a sequence, a function was implemented, that scores every possible single mutant and plots changes in the score. Potentially active site and other crucial sequence regions can be predicted by this.
</DIV></DIV></DIV><DIV class="panel panel-default"><A data-toggle="collapse" data-parent="#accordion" href="#collapseweek_id-15"><DIV class="panel-heading">
            Calendar Week
43
        </DIV></A><DIV id="collapseweek_id-15" class="panel-collapse collapse"><DIV class="panel-body"><H2>Optogenetics</H2><H2 id="sp_opto_el222_non-binding">SP_Opto_EL222_non-binding</H2>
A SP_insert PCR (JM_068, JM_069) was performed and sent to GATC for sequencing.The sequencing results proved that the phage contains a truncated version of EL222.This phage is further used as control for AP_testing.
<H2 id="ap_testing">AP_testing</H2>
 Cultures of AP_light and AP_dark were prepared and grown to an OD600 of 0.6. A sample was taken to prove that the cultures were not already contaminated with phages. Then, the two cultures were split into 10 ml cultures and infected with 10 µl of either SP_Opto or SP_Opto_EL222_non-binding (titer: (~10<SUP>11</SUP> PFU/ml) to reach a MOI of 1. All strain, phage and cultivation conditions were prepared in duplicates. Cultures were cultivated for 3 h either in the dark or under blue light (25 %: 15 sec. on, 45 sec. off). After 3 h 2 ml of each culture was centrifuged at 6000 xg for 3 min and supernatant was transferred into a fresh reaction tube. 1 ml of this supernatant was used to infect a fresh <EM>E. coli</EM> culture (OD600 ~ 0.6)which was again cultivated for 3 h under the same conditions as before. This transfer step was repeated twice and final samples of T=4 were analyzed by a Blue Plaque assay (T=4: after 4 x 3 h of cultivation, 3 transfer cycles). On the next day plaques were counted and phage titers were determined:

<DIV class="col-lg-12 col-md-12 col-xs-12 content" style="padding-top: 20px; padding-bottom: 20px"><DIV class="table-slot" style="overflow-x: auto"><TABLE class="table table-bordered mdl-shadow--4dp" xsscleaned="overflow-x: scroll !important"><THEAD><TR><TH>Strain</TH><TH>Phage</TH><TH>Transfer cycle</TH><TH>Condition</TH><TH>Duplicate</TH><TH>Phage titer [PFU/ml]</TH></TR></THEAD><TBODY><TR><TD>AP_light</TD><TD>SP_Opto</TD><TD>0</TD><TD>-</TD><TD>6x10<SUP>6</SUP> PFU/ml</TD></TR><TR><TD>AP_light</TD><TD>SP_Opto_EL222_non-binding</TD><TD>0</TD><TD>-</TD><TD>I</TD><TD>10x10<SUP>6</SUP> PFU/ml</TD></TR><TR><TD>AP_light</TD><TD>SP_Opto</TD><TD>3</TD><TD>blue light</TD><TD>I</TD><TD>7x10<SUP>4</SUP> PFU/ml</TD></TR><TR><TD>AP_light</TD><TD>SP_Opto</TD><TD>3</TD><TD>blue light</TD><TD>II</TD><TD>2x10<SUP>4</SUP> PFU/ml</TD></TR><TR><TD>AP_light</TD><TD>SP_Opto_EL222_non-binding</TD><TD>3</TD><TD>blue light</TD><TD>I</TD><TD>1x10<SUP>4</SUP> PFU/ml</TD></TR><TR><TD>AP_light</TD><TD>SP_Opto_EL222_non-binding</TD><TD>3</TD><TD>blue light</TD><TD>II</TD><TD>1x10<SUP>4</SUP> PFU/ml</TD></TR><TR><TD>AP_light</TD><TD>SP_Opto</TD><TD>3</TD><TD>dark</TD><TD>I</TD><TD>4x10<SUP>4</SUP> PFU/ml</TD></TR><TR><TD>AP_light</TD><TD>SP_Opto</TD><TD>3</TD><TD>dark</TD><TD>II</TD><TD>1x10<SUP>4</SUP> PFU/ml</TD></TR><TR><TD>AP_light</TD><TD>SP_Opto_EL222_non-binding</TD><TD>3</TD><TD>dark</TD><TD>I</TD><TD>4x10<SUP>4</SUP> PFU/ml</TD></TR><TR><TD>AP_light</TD><TD>SP_Opto_EL222_non-binding</TD><TD>3</TD><TD>dark</TD><TD>II</TD><TD>4x10<SUP>4</SUP> PFU/ml</TD></TR></TBODY></TABLE></DIV></DIV><DIV class="col-lg-12 col-md-12 col-xs-12 content" style="padding-top: 20px; padding-bottom: 20px"><DIV class="table-slot" style="overflow-x: auto"><TABLE class="table table-bordered mdl-shadow--4dp" xsscleaned="overflow-x: scroll !important"><THEAD><TR><TH>Strain</TH><TH>Phage</TH><TH>Transfer cycle</TH><TH>Condition</TH><TH>I</TH><TH>Phage titer [PFU/ml]</TH></TR></THEAD><TBODY><TR><TD>AP_dark</TD><TD>SP_Opto</TD><TD>0</TD><TD>-</TD><TD>I</TD><TD>3x10<SUP>6</SUP> PFU/ml</TD></TR><TR><TD>AP_dark</TD><TD>SP_Opto_EL222_non-binding</TD><TD>0</TD><TD>-</TD><TD>I</TD><TD>8x10<SUP>6</SUP> PFU/ml</TD></TR><TR><TD>AP_dark</TD><TD>SP_Opto</TD><TD>3</TD><TD>blue light</TD><TD>I</TD><TD>more than 10<SUP>14</SUP> PFU/ml</TD></TR><TR><TD>AP_dark</TD><TD>SP_Opto</TD><TD>3</TD><TD>blue light</TD><TD>II</TD><TD>5.3x10<SUP>13</SUP> PFU/ml</TD></TR><TR><TD>AP_dark</TD><TD>SP_Opto_EL222_non-binding</TD><TD>3</TD><TD>blue light</TD><TD>I</TD><TD>more than 10<SUP>14</SUP> PFU/ml</TD></TR><TR><TD>AP_dark</TD><TD>SP_Opto_EL222_non-binding</TD><TD>3</TD><TD>blue light</TD><TD>II</TD><TD>more than 10<SUP>14</SUP> PFU/ml</TD></TR><TR><TD>AP_dark</TD><TD>SP_Opto</TD><TD>3</TD><TD>dark</TD><TD>I</TD><TD>9x10<SUP>12</SUP> PFU/ml</TD></TR><TR><TD>AP_dark</TD><TD>SP_Opto</TD><TD>3</TD><TD>dark</TD><TD>II</TD><TD>more than 10<SUP>14</SUP> PFU/ml</TD></TR><TR><TD>AP_dark</TD><TD>SP_Opto_EL222_non-binding</TD><TD>3</TD><TD>dark</TD><TD>I</TD><TD>more than 10<SUP>14</SUP> PFU/ml</TD></TR><TR><TD>AP_dark</TD><TD>SP_Opto_EL222_non-binding</TD><TD>3</TD><TD>dark</TD><TD>II</TD><TD>more than 10<SUP>14</SUP> PFU/ml</TD></TR></TBODY></TABLE></DIV></DIV><H2>Protein_Interaction</H2><H2 id="-week-43-"><STRONG>Week 43:</STRONG>
In Week 43 strain with AP and MPs were used to perform 2 PACE runs and several PREDCEL rounds according to the respective protocols. Besides that RBS exchange was finished and plasmid transformation was done, again, by electroporation. 
</H2><H2>Software</H2><H2 id="gaia">GAIA</H2>
The usability of the script was improved by adding supports for passing parameters via flags. Plotting heatmaps of scores with the sequence on x-axis and the amino acids to which exchanges were scored on the y-axis was improved.
</DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><SECTION id="footer-sec" style="background-color: #222 !important"><DIV class="container"><DIV class="row" style="padding-top: 30px;"><DIV class="col-md-3"><H4 style="text-align: center">Quote</H4><P><I>“”</I></P></DIV><DIV class="col-md-3"><H4 style="text-align: center"> Useful <STRONG>Links</STRONG></H4><P><A href="https://igem.org/Special:SpecialPages">
			Special pages
		</A></P><P><A href="https://igem.org/Main_Page">
			Main Page
		</A></P></DIV><DIV class="col-md-3"><H4 style="text-align: center"> Follow us on </H4><DIV style="text-align: center;"><SPAN style="color: grey; text-align: center;">&amp;</SPAN></DIV></DIV><DIV class="col-md-3"><H4 style="text-align: center"> Contact us </H4><P>

               iGEM-Heidelberg2017@bioquant.uni-heidelberg.de
               
		 Im Neuenheimer Feld 267
		
		69120 Heidelberg
                    </P></DIV></DIV></DIV></SECTION></DIV></DIV></DIV></DIV></BODY></HTML>