<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Heidelberg_Toolbox skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Heidelberg/Toolbox</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container"><DIV class="navbar-header"><A type="button" data-canvas="body" id="sidebarCollapse" class="navbar-toggle" data-toggle="offcanvas" data-target="#sidebar" style="color: #393939;margin-top: 9px;">NAVIGATION
                    </A></DIV><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav pull-right"><LI class="dropdown"><A href="https://2017.igem.org/Team:Heidelberg/Team">People </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Team">Team</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Attributions">Attributions</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Sponsoring">Sponsoring</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/For_Judges">For Judges</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Jamboree">Giant Jamboree</A></LI></UL><LI class="dropdown"><A href="https://2017.igem.org/Team:Heidelberg/Description">Project </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Description">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Design">Background &amp; Design </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Pace">PACE</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Predcel">PREDCEL</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Results">Results Summary</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Validation">Software Validation</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Organosilicons">Organosilicons</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Protein_Interaction">Protein Interaction</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Cytochrome_Engineering">Cytochrome Engineering</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Optogenetics">Optogenetics</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/CRISPR">CRISPR Cas9</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/InterLab">InterLab Study</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Achievements">Achievements</A></LI><LI class="dropdown"><A href="https://2017.igem.org/Team:Heidelberg/Parts">Parts </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Basic_Part">Basic Parts</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Composite_Part">Composite Parts</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Improved_Part">Improved Part</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Part_Collection">Parts Collection</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Part_List">Part List</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/RFC">RFC</A></LI><LI class="dropdown"><A href="https://2017.igem.org/Team:Heidelberg/Model">Modeling </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Phage_Titer">Phage Titer</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Mutagenesis_Induction">Mutagenesis Induction</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Mutation_Rate_Estimation">Mutation Rate Estimation</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Lagoon_Contamination">Lagoon Contamination</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Medium_Consumption">Medium Consumption</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Tools">Interactive Web Tools</A></LI><LI class="dropdown"><A href="https://2017.igem.org/Team:Heidelberg/Software">Software </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software">AiGEM Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software/DeeProtein">DeeProtein</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software/GAIA">GAIA</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software/SafetyNet">SafetyNet</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software/MAWS">MAWS 2.0</A></LI><LI class="dropdown"><A href="https://2017.igem.org/Team:Heidelberg/Human_Practices">Human Practices </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Human_Practices">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/HP/Gold_Integrated">Integrated Human Practices</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Interviews">Expert Interviews</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Safety">Safety &amp; Security</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Engagement">Public Engagement</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Education">Education</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Collaborations">Collaborations</A></LI><LI class="dropdown"><A href="https://2017.igem.org/Team:Heidelberg/Notebook">Notebook </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Notebook">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Experiments">Methods</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Materials">Materials</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Database">Notebook Database</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Toolbox">Toolbox Guide</A></LI></DIV></DIV><NAV id="sidebar" style="z-index: 10001; background-color: white;border: none;box-shadow:1px 0px 0px #393939;" class="navmenu navmenu-default navmenu-fixed-left offcanvas" role="navigation"><A class="navmenu-brand" href="https://2017.igem.org/Team:Heidelberg"> iGEM TEAM HEIDELBERG 2017</A><DIV id="sidebarHide" data-toggle="offcanvas" data-target="#sidebar" style="padding-left: 20px;"><A style="font-family: 'Josefin Sans' !important; font-size: 16px; line-height: 18px; height: 18px;"> COLLAPSE NAVIGATION </A></DIV><UL class="nav navmenu-nav" id="sidenav"><LI><A data-toggle="collapse" data-target="#People_side">People </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Team">Team</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Attributions">Attributions</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Sponsoring">Sponsoring</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/For_Judges">For Judges</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Jamboree">Giant Jamboree</A></LI></UL><LI><A data-toggle="collapse" data-target="#Project_side">Project </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Description">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Design">Background &amp; Design </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Pace">PACE</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Predcel">PREDCEL</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Results">Results Summary</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Validation">Software Validation</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Organosilicons">Organosilicons</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Protein_Interaction">Protein Interaction</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Cytochrome_Engineering">Cytochrome Engineering</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Optogenetics">Optogenetics</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/CRISPR">CRISPR Cas9</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/InterLab">InterLab Study</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Achievements">Achievements</A></LI><LI><A data-toggle="collapse" data-target="#Parts_side">Parts </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Basic_Part">Basic Parts</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Composite_Part">Composite Parts</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Improved_Part">Improved Part</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Part_Collection">Parts Collection</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Part_List">Part List</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/RFC">RFC</A></LI><LI><A data-toggle="collapse" data-target="#Model_side">Modeling </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Phage_Titer">Phage Titer</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Mutagenesis_Induction">Mutagenesis Induction</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Mutation_Rate_Estimation">Mutation Rate Estimation</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Lagoon_Contamination">Lagoon Contamination</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Medium_Consumption">Medium Consumption</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Tools">Interactive Web Tools</A></LI><LI><A data-toggle="collapse" data-target="#AI_side">Software </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software">AiGEM Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software/DeeProtein">DeeProtein</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software/GAIA">GAIA</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software/SafetyNet">SafetyNet</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software/MAWS">MAWS 2.0</A></LI><LI><A data-toggle="collapse" data-target="#HP_side">Human Practices </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Human_Practices">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/HP/Gold_Integrated">Integrated Human Practices</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Interviews">Expert Interviews</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Safety">Safety &amp; Security</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Engagement">Public Engagement</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Education">Education</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Collaborations">Collaborations</A></LI><LI><A data-toggle="collapse" data-target="#Notebook_side">Notebook </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Notebook">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Experiments">Methods</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Materials">Materials</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Database">Notebook Database</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Toolbox">Toolbox Guide</A></LI></NAV><DIV style="padding-top: 100px; background-color: #222;"><FORM name="Toolbox_form" action="" method="get" id="toolbox_form"><DIV class="t-container"><DIV class="t-col t-col_6" style="margin: 0px !important"><H2 style="text-align: center; color: white !important;">Welcome to our
                    <SPAN style="color: #ffd200; font-weight: 400 !important">Toolbox</SPAN> Guide!</H2><P style="color: white !important; font-family: 'Josefin Sans' !important; font-size: 25px !important; font-weight: 200 !important;line-height: 30px !important">To enable rapid design and simple cloning of APs, we created a software tool, so that cloning can easily
                    be planned. All parts that can be chosen are available in pSB1C3 backbones from the registry. Just follow
                    the instructions and create your own fully customized AP.</P></DIV></DIV><DIV class="t-container input-form"><DIV class="t-col t-col_12 questionnaire"><H1 style="text-align: center !important; font-weight: 400 !important">Ready to PREDCEL?</H1><DIV class="accordion" role="tablist" aria-multiselectable="true" style="padding-top: 30px"><DIV class="panel panel-default"><DIV class="panel-heading" role="tab" id="tab1"><H4 class="panel-title"><A role="button" data-toggle="collapse" data-parent="#accordion" href="#tab1-collapse" aria-expanded="false" aria-controls="tab1-collapse"><DIV><DIV class="col-md-11">
                                            See our Questionnaire here
                                        </DIV></DIV></A></H4></DIV><DIV id="tab1-collapse" class="panel-collapse collapse" role="tabpanel" aria-labelledby="tab1"><DIV class="panel-body"><DIV class="col-lg-12 col-md-12 col-xs-12 content" style="padding-top: 20px; padding-bottom: 20px"><DIV class="col-lg-12 col-md-12 col-xs-12 content" style="padding-top: 20px; padding-bottom: 20px"><DIV class="col-lg-offset-1 col-lg-10 col-md-offset-1 col-md-10 col-xs-12 radio-container"><H2>Have you already chosen a protein to be evolved?</H2><DIV class="col-lg-6 col-md-6 col-xs-12 content"><LABEL for="q_11" class="check"><DIV class="labelname content" style="color:#222 !important; width: 100px;">Yes</DIV></LABEL></DIV><DIV class="col-lg-6 col-md-6 col-xs-12"><LABEL for="q_12" class="check"><DIV class="labelname content" style="color:#222 !important;width: 100px;">No</DIV></LABEL></DIV><DIV style="display: none" class="answer" id="a_11">You already accomplished the first and most important step. Let’s see whether
                                                you are ready to evolve it!</DIV><DIV style="display: none" class="answer" id="a_12">No problem, there are lots of proteins waiting to be evolved. Go on to find
                                                out what you should take care of when choosing your target protein and preparing your experiments. 
                                                Maybe you also want to have a look at our <A href="https://2017.igem.org/Team:Heidelberg/Description">different projects</A>
                                                to get an idea of the wide range of proteins you can evolve with <A href="https://2017.igem.org/Team:Heidelberg/Predcel">PREDCEL</A> .
                                                
                                                
                                            </DIV></DIV><DIV class="col-lg-offset-1 col-lg-10 col-md-offset-1 col-md-10 col-xs-12 radio-container"><H2>Which application will your evolved protein be used for?</H2><DIV class="col-lg-offset-1 col-lg-2 col-md-offset-1 col-md-2 col-xs-12"><LABEL for="q_21" class="check"><DIV class="labelname content" style="color:#222 !important;">Medical application</DIV></LABEL></DIV><DIV class="col-lg-2 col-md-2 col-xs-12"><LABEL for="q_22" class="check"><DIV class="labelname content" style="color:#222 !important">Chemical synthesis</DIV></LABEL></DIV><DIV class="col-lg-2 col-md-2 col-xs-12"><LABEL for="q_23" class="check"><DIV class="labelname content" style="color:#222 !important">Agriculture</DIV></LABEL></DIV><DIV class="col-lg-2 col-md-2 col-xs-12"><LABEL for="q_24" class="check"><DIV class="labelname content" style="color:#222 !important">Could be helpful somehow</DIV></LABEL></DIV><DIV class="col-lg-2 col-md-2 col-xs-12"><LABEL for="q_25" class="check"><DIV class="labelname content" style="color:#222 !important">Other scientific interests</DIV></LABEL></DIV><DIV style="display: none" class="answer" id="a_21">Did you discuss your idea with a medical professional? We are sure this expertise
                                                would be valuable to choose the right experimental settings.</DIV><DIV style="display: none" class="answer" id="a_22">Did you discuss your idea with a chemist? This expertise could be helpful to figure out how to set up the evolution efficiently, as parameters such as passive diffusion of the chemical through the E. coli cell wall, toxicity and many others needs to be carefully considered.</DIV><DIV style="display: none" class="answer" id="a_23">Be aware that a number of countries have strict regulations regarding green biotechnology.
                                            </DIV><DIV style="display: none" class="answer" id="a_24">That sounds quite uncertain. Before you start or pursue evolving your protein,
                                                you should know the exact purpose and impact. How about engaging with your
                                                local community to figure out possible needs?</DIV><DIV style="display: none" class="answer" id="a_25">Engage with the community in the field of in vivo evolution – we´d be interested
                                                in discussing your project.</DIV></DIV><DIV class="col-lg-offset-1 col-lg-10 col-md-offset-1 col-md-10 col-xs-12 radio-container"><H2>Are you sure your protein is safe to be evolved?</H2><DIV class="col-lg-6 col-md-6 col-xs-12 content"><LABEL for="q_31" class="check"><DIV class="labelname content" style="color:#222 !important; width: 100px;">Yes it is safe!</DIV></LABEL></DIV><DIV class="col-lg-6 col-md-6 col-xs-12"><LABEL for="q_32" class="check"><DIV class="labelname content" style="color:#222 !important;width: 100px;">How can I know?</DIV></LABEL></DIV><DIV style="display: none" class="answer" id="a_31">Nevertheless we advise you to check it in our SafetyNet, there could be “sleeping”
                                                hazardous potential.</DIV><DIV style="display: none" class="answer" id="a_32">Please, check your protein sequence for any “sleeping” hazardous potential in our
                                                <A href="https://2017.igem.org/Team:Heidelberg/Software/SafetyNet">SafetyNet</A></DIV></DIV><DIV class="col-lg-offset-1 col-lg-10 col-md-offset-1 col-md-10 col-xs-12 radio-container"><H2>Do you know which equipment is required to safely work with phages?</H2><DIV class="col-lg-4 col-md-4 col-xs-12 content"><LABEL for="q_41" class="check"><DIV class="labelname content" style="color:#222 !important; width: 100px;">Yes</DIV></LABEL></DIV><DIV class="col-lg-4 col-md-4 col-xs-12"><LABEL for="q_42" class="check"><DIV class="labelname content" style="color:#222 !important;width: 100px;">Not sure</DIV></LABEL></DIV><DIV class="col-lg-4 col-md-4 col-xs-12"><LABEL for="q_43" class="check"><DIV class="labelname content" style="color:#222 !important;width: 100px;">No</DIV></LABEL></DIV><DIV style="display: none" class="answer" id="a_41">Did your considerations include that phages are very volatile? You should create
                                                a special “phage space” in your lab to reduce contamination. Furthermore,
                                                you should comply with the safety level of your evolving protein. Check the
                                                safety level of your protein on our <A href="https://2017.igem.org/Team:Heidelberg/Software/SafetyNet">SafetyNet</A></DIV><DIV style="display: none" class="answer" id="a_42">Check our<A href="https://2017.igem.org/Team:Heidelberg/Safety">safety page</A> for general advice and get in touch with your safety representative for detailed discussion of the particular work you plan.</DIV><DIV style="display: none" class="answer" id="a_43">Before starting any phage work, please consult our <A href="https://2017.igem.org/Team:Heidelberg/Safety">safety page</A>. Also, we advise you to talk to your colleagues next door with experience in phage work as well as your local safety representative.
                                                </DIV></DIV><DIV class="col-lg-offset-1 col-lg-10 col-md-offset-1 col-md-10 col-xs-12 radio-container"><H2>Did you calculate all required parameters on our Integrated Modeling page?</H2><DIV class="col-lg-4 col-md-4 col-xs-12 content"><LABEL for="q_51" class="check"><DIV class="labelname content" style="color:#222 !important; width: 100px;">Yes</DIV></LABEL></DIV><DIV class="col-lg-4 col-md-4 col-xs-12"><LABEL for="q_52" class="check"><DIV class="labelname content" style="color:#222 !important;width: 100px;">Not yet</DIV></LABEL></DIV><DIV class="col-lg-4 col-md-4 col-xs-12"><LABEL for="q_53" class="check"><DIV class="labelname content" style="color:#222 !important;width: 100px;">I don´t intend to</DIV></LABEL></DIV><DIV style="display: none" class="answer" id="a_51">Well done! Now you have the optimal starting conditions for your experiment.</DIV><DIV style="display: none" class="answer" id="a_52">Visit our Integrated Modeling page to figure out all required parameters for a successful PREDCEL run.</DIV><DIV style="display: none" class="answer" id="a_53">You should consider that our Integrated Modeling reduces effort, costs and environmental burden by enabling in silico optimization of experimental conditions.</DIV></DIV><DIV class="col-lg-offset-1 col-lg-10 col-md-offset-1 col-md-10 col-xs-12 radio-container"><H2>Did you already engage with your local community to discuss your plans?</H2><DIV class="col-lg-4 col-md-4 col-xs-12 content"><LABEL for="q_61" class="check"><DIV class="labelname content" style="color:#222 !important; width: 100px;">Yes</DIV></LABEL></DIV><DIV class="col-lg-4 col-md-4 col-xs-12"><LABEL for="q_62" class="check"><DIV class="labelname content" style="color:#222 !important;width: 100px;">Not yet</DIV></LABEL></DIV><DIV class="col-lg-4 col-md-4 col-xs-12"><LABEL for="q_63" class="check"><DIV class="labelname content" style="color:#222 !important;width: 100px;">I don´t intend to</DIV></LABEL></DIV><DIV style="display: none" class="answer" id="a_61">Fine. For more ideas, how to engage with the public you could watch the
                                                <A href="https://2017.igem.org/Team:Heidelberg/Interviews#Kaufmann">expert interview with Dr. Kaufmann</A></DIV><DIV style="display: none" class="answer" id="a_62">You should reach out and listen to the feedback, hopes and concerns of your local community. Thereby, you can facilitate a positive, beneficial impact of your research project.</DIV><DIV style="display: none" class="answer" id="a_63">Are you sure? We encourage you to do so and be surprised, how enriching it is to discuss your research plans with people inside and outside the research community. Also consider, that your research may very well impact the people around you and hence, they should have a word and vote in what our technologies are used for. Finally, showing that you are a responsible scientists caring about the consequences of your research will surely have a positive impact on your reputation.</DIV></DIV><DIV class="col-lg-offset-1 col-lg-10 col-md-offset-1 col-md-10 col-xs-12 radio-container"><H2>Did you consider possible consequences of your planned PREDCEL experiment?</H2><DIV class="col-lg-4 col-md-4 col-xs-12 content"><LABEL for="q_71" class="check"><DIV class="labelname content" style="color:#222 !important; width: 100px;">Yes</DIV></LABEL></DIV><DIV class="col-lg-4 col-md-4 col-xs-12"><LABEL for="q_72" class="check"><DIV class="labelname content" style="color:#222 !important;width: 100px;">Not yet</DIV></LABEL></DIV><DIV class="col-lg-4 col-md-4 col-xs-12"><LABEL for="q_73" class="check"><DIV class="labelname content" style="color:#222 !important;width: 100px;">I don´t intend to</DIV></LABEL></DIV><DIV style="display: none" class="answer" id="a_71">You are a responsible scientist. We encourage you to use our PREDCEL toolbox and convey the iGEM spirit of responsible science to your friends and colleagues.</DIV><DIV style="display: none" class="answer" id="a_72">Take your time to reflect on possible consequences to prevent unintended dangers. This is an essential step to be taken before starting directed evolution experiments.</DIV><DIV style="display: none" class="answer" id="a_73">Are you sure? We encourage you to talk to experts and your local community. You will be surprised how much they will appreciate your offer to discuss the consequences of your research BEFORE implementing it in the lab. Also, consider the positive impact for yourself: Showing that you are a responsible scientists caring about the consequences of your research will surely have a positive impact on your reputation.</DIV></DIV><DIV class="col-lg-offset-1 col-lg-10 col-md-offset-1 col-md-10 col-xs-12 radio-container"><H1 style="padding-top: 0px;color: #393939 !important">Now it´s on you!</H1><DIV class="content" style="padding: 0px 20px 0px 20px">When you talk about PREDCEL with your colleagues, friends and local communities, you can spread the idea of making directed evolution experiments safe and to use this technology responsibly for addressing urgent human needs.</DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="t-container input-form content" style="position: relative; top: 20px;"><H1 style="text-align: center !important; color:##393939 !important; font-weight: 400 !important;padding: 0px !important; margin: 0px !important">
                1. Optimize parental sequence / gene pool</H1><DIV class="content col-md-12 col-lg-12" style="padding-top: 10px">With the
                <A href="https://2017.igem.org/wiki/index.php?title=Team:Heidelberg/Software">AiGEM</A> (Artificial Intelligence for Genetic Evolution Mimicking) software suite we provide powerful software
                tools for
                <I>in silico</I> evolution. Applying our genetic algorithm,
                <A href="https://2017.igem.org/wiki/index.php?title=Team:Heidelberg/Software/GAIA">GAIA</A>, we
                <A href="https://2017.igem.org/wiki/index.php?title=Team:Heidelberg/Validation#gusgal">fully in silico evolved a \(\beta\)-glucuronidase towards \(\beta\)-galactosidase function</A> and
                <A href="https://2017.igem.org/wiki/index.php?title=Team:Heidelberg/Validation#lac">derived \(\beta\)-lactamase variants exceeding wildtype activity</A>.
                 We thus strongly recommend the application of GAIA prior to
                <I>in vivo</I> evolution. Through optimization of the starting sequence and the acompanying reduction of the
                required library size the time until phage pool convergence can be tremendously reduced.
                 GAIA is written in python and readily available on our
                <A href="https://github.com/igemsoftware2017/AiGEM_TeamHeidelberg2017/blob/master/GAIA/GAIA_README.md">github repo</A>.</DIV></DIV><DIV class="t-container input-form content" style="position: relative; top: 40px;"><H1 style="text-align: center !important; color:##393939 !important; font-weight: 400 !important;padding: 0px !important; margin: 0px !important">
                2. Get phage</H1><DIV class="col-md-12 col-lg12 col-xs-12">
                Phage-assisted continous evolution is based on the generation of selection phages, which have geneIII replaced by the gene
                of interest. For the generation of your own selection phage, we are offering a golden-gate cloning standard
                being based on the extraordinary potential of the type IIs restriction enzyme BsaI. For insertion of your
                gene of interest into the phage backbone, amplification with primers containing the following overhangs is
                required: 
    <DIV style="padding-top: 50px; padding-bottom:50px;"><DIV class="heidel-form container"><DIV class="heidel-form-header col-lg-12" style="text-align: left !important;">Overhangs</DIV><TABLE class="table"><THEAD><TR class="table-row"><TH>Fwd:</TH><TH>GAGGTCTCGAAAA</TH></TR></THEAD><TBODY><TR><TD>Rev:</TD><TD>CAGGTCTCGGTGC</TD></TR></TBODY></TABLE></DIV></DIV></DIV></DIV></FORM></DIV></DIV>
 Following the successfully performed amplification with these primers, a golden gate reaction (refering to the golden
                gate assembly protocol) with the BioBricks BBa_K2398021 and BBa_K2398022 should be performed. GoldenGate
                cloning usually delivers good results. In the following of this reaction, the assembled plasmid should be
                transformed into electrocompetent S1059 or S2208 cells. These strains, established by the liu lab, carry
                a plasmid (pJC175e) that provides geneIII under a psp-promoter. This promoter gets activated by phage infection,
                and ensures that geneIII is not present in the uninfected cell. For further information about the principle
                of phage propagation, have a look at our <A href="https://2017.igem.org/Team:Heidelberg/RFC">RFC</A>. As an alternative to the assembly with
                two phage backbone fragments, the golden gate assembly reaction can be also performed using only the BBa_K2398023
                BioBrick. In this case, amplification of the gene of interest has to be implemented with other overhangs:
                
    <DIV style="padding-top: 50px; padding-bottom:50px;"><DIV class="heidel-form container"><DIV class="heidel-form-header col-lg-12" style="text-align: left !important;">Overhangs</DIV><FORM><TABLE class="table"><THEAD><TR class="table-row"><TH>Fwd:</TH><TH>GTGGTCTCAATGG</TH></TR></THEAD><TBODY><TR><TD>Rev:</TD><TD>TTGGTCTCAGTTG</TD></TR></TBODY></TABLE></FORM></DIV></DIV>
 If you want to produce your selection phage on your own by using another cloning method, please keep in mind that
                the terminator of geneVIII may be immediately followed by the RBS of the evolving gene. Futhermore, the last
                180 bp of geneIII should not be removed, because they contain the promoter for the following operon of phage
                genes. In that case, we provide the surrunding sequences of the gene of interest below: 
    <DIV style="padding-top: 50px; padding-bottom:50px;"><DIV class="heidel-form container"><DIV class="heidel-form-header col-lg-12" style="text-align: left !important;">Additional
                Sequences</DIV><FORM><TABLE class="table"><THEAD><TR class="table-row"><TH>Upstream sequence</TH><TH>AAAGGCTCCTTTTGGAGCCTTTTTTTT</TH></TR></THEAD><TBODY><TR><TD>Downstream sequence</TD><TD>CTCCCTCAATCGGTTGAATGTCGCCCTTTTGTCTTT</TD></TR></TBODY></TABLE></FORM></DIV></DIV></DIV></DIV><DIV class="t-container input-form content" style="position: relative; top: 60px;"><H1 style="text-align: center !important; color:##393939 !important; font-weight: 400 !important;padding: 0px !important; margin: 0px !important">
                3. Add selection pressure</H1><SECTION class="teaser content equal_height" style="padding-top: 100px;"><DIV class="col-md-12">
                For any method of directed evolution to culminate in an improved protein, a means of applying selective pressure is needed.
                Thus, adding components capable of enforcing selection in PACE and PREDCEL is one of the most crucial steps
                in construct design. For directed evolution experiments selective pressure may need to be fine-tuned by using
                different ORIs, promoters, ribosome binding sites and backbones. In this section of the toolbox guide, all
                those components will be set up for you to suit your particular project. To get an idea of how selection
                could be enforced in your projects, take a look at our examples:
            </DIV><DIV class="col-lg-6 col-md-6 col-xs-12"><DIV class="card" style="border-left: 10px solid;"><DIV class="card-body"><H4 class="card-title">MAWS 2.0 -
            <I>in silico</I> riboswitch design for selective pressure.</H4><P class="card-text content">  A
            simple and sure-fire way to couple gene expression to a product of enzyme catalysis is to introduce a riboswitch
            for that particular product. This is a short RNA sequence either trapping or releasing the ribosome binding site
            in the presence of a small molecule. Back in 2015 the Heidelberg iGEM Team developed a software suite (MAWS -
            making aptamers without SELEX) to predict such riboswitches for arbitrary small molecule targets. This year we
            use and provide new-and-improved distribution of MAWS. We predict and validate synthetic riboswitches generated
            using MAWS 2.0 to couple gene expression to enzyme activity. This establishes riboswitches predicted <I>in
            silico</I> as a valuable component of any directed evolution project seeking to evolve novel biocatalytic properties.
            </P><A href="https://2017.igem.org/Team:Heidelberg/Software/MAWS" class="card-button">MAWS 2.0</A></DIV></DIV></DIV><DIV class="col-lg-6 col-md-6 col-xs-12"><DIV class="card" style="border-left: 10px solid;"><DIV class="card-body"><H4 class="card-title">OptoSELECT - Light-dependent selective pressure</H4><P class="card-text content"> 
            The generation of proteins with radically altered or highly specific new activities is a major goal in the field
            of directed evolution. Modulation of selection stringency is indispensable for the optimization of proteins,
            which in their native form exhibit a lack of activity for the required action. To further extend the possibilities
            of PACE/PREDCEL based protein optimization and to evolve novel activities without the use of intermediate evolutionary
            steps, we provide an optogenetic modulator of selection stringency. The variation of stringency is based on the
            blue light-dependent transcription factor EL222 and a bidirectional promoter system that can induce or repress
            the expression of geneIII upon blue light irradiation in a non-toxic, rapidly delivered, and reversible manner.
            </P><A href="https://2017.igem.org/Team:Heidelberg/Optogenetics" class="card-button">OptoSELECT</A></DIV></DIV></DIV></SECTION><DIV style="padding-top: 50px; padding-bottom:50px;"><DIV class="heidel-form container"><DIV class="heidel-form-header col-lg-12" style="text-align: left !important;">Accessory Plasmid Generator</DIV><FORM><H3 style="text-align: center !important; color:#6698BE !important; font-weight: 400 !important;padding: 0px !important; margin: 0px !important ">Plasmid</H3><DIV class="content" style="padding: 10px 20px 10px 20px !important"><I>1. First, choose your preferred backbone from the pull down menu. We provide three different origins of replication
                    that differ in their copy number and therefore affect the selection stringency, your AP will have. Be
                    aware that the origin of replication should be compatible to the origin of replication of all other plasmids
                    you plan to use. In most of the cases it is best to combine your origin of replication with an ampicillin
                    resistance, but we provide other antibiotic resistances for special applications as well.</I></DIV><DIV class="content" style="padding: 10px 20px 10px 20px !important"><I>2. Second, decide which RBS you need for control of geneIII translation. We offer six different RBS with
                    different strengths. For further information take a look at our RFC</I></DIV><DIV class="content" style="padding: 10px 20px 10px 20px !important"><I>3. Next, decide, which reporter you would like to choose. A set of different fluorescent and luminescent
                    reporters is available</I></DIV><DIV class="row" style="padding: 0px 20px 0px 20px;"><DIV class="col-lg-4 col-md-4 col-xs-12"><H3>Resistance &amp; Ori </H3></DIV><DIV class="col-lg-4 col-md-4 col-xs-12"><H3>RBS + geneIII</H3></DIV><DIV class="col-lg-4 col-md-4 col-xs-12"><H3>RBS + Reporter</H3></DIV></DIV></FORM></DIV></DIV><DIV style="padding-top: 50px; padding-bottom:50px;"><DIV class="heidel-form container"><DIV class="heidel-form-header col-lg-12" style="text-align: left !important;">Activator &amp; Promoter</DIV><FORM><DIV class="content" style="padding: 10px 20px 10px 20px !important"><I>4. When all standard parts are defined, the promoter and associated regulatory sequences must be set. Decide
                    between the given options. Enter either an own sequence (as text or upload the file) or use one of the
                    provided standard promoters that were already used by our team. Notice that the length of this part should
                    not be below 200 bp. Ensure, that your sequence does not include a RBS.</I></DIV><DIV class="row" style="padding: 0px 20px 0px 20px;margin: 0px !important;"><DIV class="col-lg-12 col-md-12 col-xs-12"><DIV id="input_promotor_upload" style="visibility:visible;display:none"><SPAN>Overlaps are included</SPAN></DIV><DIV id="input_promotor_paste" style="visibility:visible;display:none"><SPAN> Overlaps are included</SPAN></DIV></DIV></DIV></FORM></DIV></DIV><DIV style="padding-top: 50px; padding-bottom:50px;"><DIV class="heidel-form container"><DIV class="heidel-form-header col-lg-12" style="text-align: left !important;">Additional sequence</DIV><FORM><DIV class="content" style="padding: 10px 20px 10px 20px !important"><I>5. Finally, decide which additional gene you would like to express on your AP. Most circuits that are used
                    in PACE, need at least one additional protein, like split-proteins, chaperone or interacting factors.
                    Again, it is possible to enter the desired sequence or choose one of the provided genes. Make sure, you
                    enter a whole expression cassette, with promter, RBS, CDS and terminator.</I></DIV><DIV class="row" style="padding: 0px 20px 0px 20px"><DIV class="col-lg-12 col-md-12 col-xs-12"><DIV id="input_additional_upload" style="visibility:visible;display:none;padding-bottom:20px;"><SPAN>Overlaps are included</SPAN></DIV></DIV><DIV class="col-lg-12 col-md-12 col-xs-12"><DIV id="input_additional_paste" style="visibility:visible;display:none;padding-bottom:20px;"><SPAN> Overlaps are included</SPAN></DIV></DIV></DIV></FORM></DIV></DIV><DIV style="padding-top: 50px; padding-bottom:50px;"><DIV class="heidel-form-color container"><DIV class="heidel-form-color-header col-lg-12" style="text-align: left !important;">Deploy the Plasmid</DIV><FORM><DIV class="row"><DIV class="col-lg-12 col-md-12 col-xs-12">          Deploy your designed Plasmid and remember to PACE or PREDCEL responsibly...
                    <DIV class="getPlasmidButton" onclick="submit_p1()">Create Plasmid</DIV><DIV class="row" style="display:none" id="results_area" style="min-height:600px !important"><DIV class="col-lg-6 col-md-6 col-xs-12" style="padding: 20px;"><H4>Instructions</H4></DIV></DIV></DIV></DIV></FORM></DIV></DIV></DIV><DIV class="t-container input-form content" style="position: relative; top: 80px;"><H1 style="text-align: center !important; color:##393939 !important; font-weight: 400 !important;padding: 0px !important; margin: 0px !important">
                4. Induce mutagenesis</H1><DIV class="content col-md-12 col-lg-12 col-xs-12">After the generation of your AP and your SP, we will focus on the third plasmid, playing a highly important role
                in both PACE and PREDCEL. The mutagenesis plasmid is responsible for the library preparation of the different
                selection phage variants. If you have transformed your AP into a bacterial strain, suitable to be used in
                PACE or PREDCEL, one of three possible mutagenesis plasmids
                <A href="https://www.addgene.org/69627/">MP1</A>,
                <A href="https://www.addgene.org/69652/">MP4</A> and
                <A href="https://www.addgene.org/69669/">MP6</A> should be transformed into the strain as well. Those MPs are differing in their mutation rate. For
                further information on the MP´s funcitonality please have a look at the literature<X-REF>badran2015development</X-REF>. For the induction of the mutagenesis, arabinose should be added into the culture,
                activating the pBAD promotor and thereby the mutagenesis. For all PREDCEL experiments a concentration of
                \(100\,mM\) arabinose is used, while \(2\,mL/h\) of \(10\%\;w/v\) arabinose solution are added in a \(100\,mL\)
                lagoon in PACE.
            </DIV></DIV><DIV class="t-container input-form content" style="position: relative; top: 100px;"><H1 style="text-align: center !important; color:##393939 !important; font-weight: 400 !important;padding: 0px !important; margin: 0px !important">
                5. Optimize conditions</H1><DIV class="content" style="padding: 10px 20px 10px 20px !important">
                PACE and PREDCEL require well chosen conditions to yield the best results. Use our
                <A href="https://2017.igem.org/Team:Heidelberg/Model/Tools">Interactive Webtools</A> to calculate parameters like
                <A href="https://2017.igem.org/Team:Heidelberg/Model/Glucose">Glucose Concentration</A>,
                    <A href="https://2017.igem.org/Team:Heidelberg/Model/Arabinose">Arabinose Concentration</A> and
                    <A href="https://2017.igem.org/Team:Heidelberg/Model/Medium">Medium Concumption</A> online, or obtain
                    <A href="https://2017.igem.org/Team:Heidelberg/Model/Phage_Titer">our phage titer Model</A> and use it to reduce the risk of washout.
                     When trying new conditions for a PACE experiment
                    <A href="https://2017.igem.org/Team:Heidelberg/Model/Contamination">check if the setup is vulnerable to contaminations</A> To make monitoring the experiment by sequencing
                    less expensive and to save you time, we provide a
                    <A href="https://2017.igem.org/Team:Heidelberg/Model/Mutation">tool that calculates the number of sequences</A> that needs to be sequenced to find mutations with a
                    certain probability.
                    </DIV></DIV></DIV></DIV><SECTION id="footer-sec" style="background-color: #222 !important"><DIV class="container"><DIV class="row" style="padding-top: 30px;"><DIV class="col-md-3"><H4 style="text-align: center">Quote</H4><P><I>“”</I></P></DIV><DIV class="col-md-3"><H4 style="text-align: center"> Useful <STRONG>Links</STRONG></H4><P><A href="https://igem.org/Special:SpecialPages">
			Special pages
		</A></P><P><A href="https://igem.org/Main_Page">
			Main Page
		</A></P></DIV><DIV class="col-md-3"><H4 style="text-align: center"> Follow us on </H4><DIV style="text-align: center;"><SPAN style="color: grey; text-align: center;">&amp;</SPAN></DIV></DIV><DIV class="col-md-3"><H4 style="text-align: center"> Contact us </H4><P>

               iGEM-Heidelberg2017@bioquant.uni-heidelberg.de
               
		 Im Neuenheimer Feld 267
		
		69120 Heidelberg
                    </P></DIV></DIV></DIV></SECTION></BODY></HTML>