<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Heidelberg_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Heidelberg/Results</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container"><DIV class="navbar-header"><A type="button" data-canvas="body" id="sidebarCollapse" class="navbar-toggle" data-toggle="offcanvas" data-target="#sidebar" style="color: #393939;margin-top: 9px;">NAVIGATION
                    </A></DIV><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav pull-right"><LI class="dropdown"><A href="https://2017.igem.org/Team:Heidelberg/Team">People </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Team">Team</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Attributions">Attributions</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Sponsoring">Sponsoring</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/For_Judges">For Judges</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Jamboree">Giant Jamboree</A></LI></UL><LI class="dropdown"><A href="https://2017.igem.org/Team:Heidelberg/Description">Project </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Description">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Design">Background &amp; Design </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Pace">PACE</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Predcel">PREDCEL</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Results">Results Summary</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Validation">Software Validation</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Organosilicons">Organosilicons</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Protein_Interaction">Protein Interaction</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Cytochrome_Engineering">Cytochrome Engineering</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Optogenetics">Optogenetics</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/CRISPR">CRISPR Cas9</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/InterLab">InterLab Study</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Achievements">Achievements</A></LI><LI class="dropdown"><A href="https://2017.igem.org/Team:Heidelberg/Parts">Parts </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Basic_Part">Basic Parts</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Composite_Part">Composite Parts</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Improved_Part">Improved Part</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Part_Collection">Parts Collection</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Part_List">Part List</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/RFC">RFC</A></LI><LI class="dropdown"><A href="https://2017.igem.org/Team:Heidelberg/Model">Modeling </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Phage_Titer">Phage Titer</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Mutagenesis_Induction">Mutagenesis Induction</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Mutation_Rate_Estimation">Mutation Rate Estimation</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Lagoon_Contamination">Lagoon Contamination</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Medium_Consumption">Medium Consumption</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Tools">Interactive Web Tools</A></LI><LI class="dropdown"><A href="https://2017.igem.org/Team:Heidelberg/Software">Software </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software">AiGEM Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software/DeeProtein">DeeProtein</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software/GAIA">GAIA</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software/SafetyNet">SafetyNet</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software/MAWS">MAWS 2.0</A></LI><LI class="dropdown"><A href="https://2017.igem.org/Team:Heidelberg/Human_Practices">Human Practices </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Human_Practices">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/HP/Gold_Integrated">Integrated Human Practices</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Interviews">Expert Interviews</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Safety">Safety &amp; Security</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Engagement">Public Engagement</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Education">Education</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Collaborations">Collaborations</A></LI><LI class="dropdown"><A href="https://2017.igem.org/Team:Heidelberg/Notebook">Notebook </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Notebook">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Experiments">Methods</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Materials">Materials</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Database">Notebook Database</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Toolbox">Toolbox Guide</A></LI></DIV></DIV><NAV id="sidebar" style="z-index: 10001; background-color: white;border: none;box-shadow:1px 0px 0px #393939;" class="navmenu navmenu-default navmenu-fixed-left offcanvas" role="navigation"><A class="navmenu-brand" href="https://2017.igem.org/Team:Heidelberg"> iGEM TEAM HEIDELBERG 2017</A><DIV id="sidebarHide" data-toggle="offcanvas" data-target="#sidebar" style="padding-left: 20px;"><A style="font-family: 'Josefin Sans' !important; font-size: 16px; line-height: 18px; height: 18px;"> COLLAPSE NAVIGATION </A></DIV><UL class="nav navmenu-nav" id="sidenav"><LI><A data-toggle="collapse" data-target="#People_side">People </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Team">Team</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Attributions">Attributions</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Sponsoring">Sponsoring</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/For_Judges">For Judges</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Jamboree">Giant Jamboree</A></LI></UL><LI><A data-toggle="collapse" data-target="#Project_side">Project </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Description">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Design">Background &amp; Design </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Pace">PACE</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Predcel">PREDCEL</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Results">Results Summary</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Validation">Software Validation</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Organosilicons">Organosilicons</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Protein_Interaction">Protein Interaction</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Cytochrome_Engineering">Cytochrome Engineering</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Optogenetics">Optogenetics</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/CRISPR">CRISPR Cas9</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/InterLab">InterLab Study</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Achievements">Achievements</A></LI><LI><A data-toggle="collapse" data-target="#Parts_side">Parts </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Basic_Part">Basic Parts</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Composite_Part">Composite Parts</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Improved_Part">Improved Part</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Part_Collection">Parts Collection</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Part_List">Part List</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/RFC">RFC</A></LI><LI><A data-toggle="collapse" data-target="#Model_side">Modeling </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Phage_Titer">Phage Titer</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Mutagenesis_Induction">Mutagenesis Induction</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Mutation_Rate_Estimation">Mutation Rate Estimation</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Lagoon_Contamination">Lagoon Contamination</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Medium_Consumption">Medium Consumption</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Tools">Interactive Web Tools</A></LI><LI><A data-toggle="collapse" data-target="#AI_side">Software </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software">AiGEM Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software/DeeProtein">DeeProtein</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software/GAIA">GAIA</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software/SafetyNet">SafetyNet</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software/MAWS">MAWS 2.0</A></LI><LI><A data-toggle="collapse" data-target="#HP_side">Human Practices </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Human_Practices">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/HP/Gold_Integrated">Integrated Human Practices</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Interviews">Expert Interviews</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Safety">Safety &amp; Security</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Engagement">Public Engagement</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Education">Education</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Collaborations">Collaborations</A></LI><LI><A data-toggle="collapse" data-target="#Notebook_side">Notebook </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Notebook">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Experiments">Methods</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Materials">Materials</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Database">Notebook Database</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Toolbox">Toolbox Guide</A></LI></NAV><DIV style="background-color: white !important; padding-bottom: 50px !important; margin: 0 !important;"><DIV class="page-title" style="background-color: white !important; border-top: 0px solid #222;padding-top: 100px"><DIV class="container" style=" padding: 0 !important;"><DIV class="col-lg-12 col-md-12 col-xs-12" style="padding-right: 0 !important; padding-bottom: 0 !important; position: relative; left: -10px;"><DIV style="border-right: 5px solid #393939; padding: 20px 20px 0px 0px"><DIV class="header-title heidelberg-orange"> 
    Results </DIV><DIV class="header-subtitle" style="color: #393939 !important;"> 
    Our project in seven minutes </DIV></DIV></DIV></DIV></DIV><DIV class="container" style="background-color: white !important; padding: 0 !important;"><DIV class="col-lg-12 col-md-12 col-xs-12 content" style="padding: 0 !important; margin: 0px important!;"><A href="https://2017.igem.org/Team:Heidelberg/Design" class="big-link"><H3 style="color: #393939 !important; text-align: left !important">Background</H3></A>
            Darwinian evolution is an enormously powerful concept that drove biology towards astonishing diversity, complexity and beauty. An efficient way to harness this power would highly accelerate the engineering of novel biomolecules for human benefit and fundamentally advance synthetic biology.
Towards this vision, our team developed a comprehensive evolution toolbox comprising <A href="https://2017.igem.org/Team:Heidelberg/Parts" class="innerlink">standardized parts</A><A href="https://2017.igem.org/Team:Heidelberg/Predcel" class="innerlink">, protocols</A><A href="https://2017.igem.org/Team:Heidelberg/Model" class="innerlink">, interactive models</A> and <A href="https://2017.igem.org/Team:Heidelberg/Software" class="innerlink">AI-based software</A>, all assembled into a unique workflow tightly interconnecting in vivo and silico <A href="https://2017.igem.org/Team:Heidelberg/Design" class="innerlink">directed evolution</A>. 
                
              Our project grounds on the <A href="https://2017.igem.org/Team:Heidelberg/Pace" class="innerlink">phage-assisted continuous evolution (PACE) method</A> (<I>Esvelt et al</I>, Nature, 2011), which couples the survival of quickly evolving M13 bacteriophages carrying a gene-of-interest to directed selection within E. coli hosts in a customized bioreactor. Despite its powerful concept, the application of PACE has thus far been hampered by its need of a continuous culture and flow setup demanding costly and highly sensitive hardware strongly limiting its application scope. <A href="https://2017.igem.org/Team:Heidelberg/Toolbox" class="big-link"><H3 style="color: #f8991d !important; text-align: left !important">Our PREDCEL (phage-related discontinuous evolution) toolbox</H3></A>
To reduce complexity and increase flexibility of the <A href="https://2017.igem.org/Team:Heidelberg/Pace" class="innerlink">PACE</A> method, we created <A href="https://2017.igem.org/Team:Heidelberg/Predcel" class="innerlink">PREDCEL</A> (for phage-related discontinuous evolution), a simple, low-cost in vivo evolution protocol completely independent of any specialized equipment. In <A href="https://2017.igem.org/Team:Heidelberg/Predcel" class="innerlink">PREDCEL</A>, the phage gene pool is simply propagated batch-wise on highly mutagenic E. coli selection strains grown in standard flasks.  Our self-contained <A href="https://2017.igem.org/Team:Heidelberg/Predcel" class="innerlink">PREDCEL toolbox</A> comprises: (1) A golden-gate cloning standard for simple production of geneIII-deficient M13 phages carrying any gene-of-interest; (2) An accessory plasmid construction kit for selection of the phage gene pool via conditional geneIII complementation in E. coli; (3) An <A href="https://2017.igem.org/Team:Heidelberg/Optogenetics" class="innerlink">optogenetic selection stringency modulator</A> (Fig 1.)
        
 
        
        
        
  <DIV class="col-lg-12 col-md-12 col-xs-12 content" style="padding-top: 20px; padding-bottom: 20px;text-align: justify !important"><DIV class="mdl-shadow--4dp" style="padding: 20px"><DIV style="font-weight: 700 !important;text-align: left!important">Figure 1: Increase of Phage Propagation under Blue Light using pBLind-geneIII Cassette</DIV>Phage titers of SP Opto EL222 and a non-binding variant propagated on AP_light in the dark and under blue light irradiation after four passages were determined by plaque assays. Host cell cultures infected with SP Opto EL222 and cultured under blue light conditions demonstrated a more than 3-fold higher phage titer than the culture cultivated in the dark (left side). The infection with phages containing the non-binding variant of EL222 exhibited no significant difference in phage titer (right side). It was notable that for this variant the phage titer was slightly decreased upon light irradiation. The respective plaque assays are shown below the bar chart.     
</DIV></DIV>           
enabling light-induced supplementation of geneIII. Thereby, the selection pressure can be easily adapted to the fitness of the evolving phage gene pool during PREDCEL runs if needed; (4) A T7 polymerase-based platform for detailed PREDCEL evaluation as well as directed evolution of <A href="https://2017.igem.org/Team:Heidelberg/Protein_Interaction" class="innerlink">protein-protein interactions</A> and (5) The very heart of our toolbox: an innovative and generalizable workflow for <A href="https://2017.igem.org/Team:Heidelberg/Cytochrome_Engineering" class="innerlink">enzyme engineering</A> based on riboswitch-mediated coupling of product formation to enzyme variant selection. 
An online <A href="https://2017.igem.org/Team:Heidelberg/Toolbox" class="innerlink">toolbox guide</A> and accompanying <A href="https://2017.igem.org/Team:Heidelberg/RFC" class="innerlink">RFC</A> simplifies the utilization of our PREDCEL toolbox for future iGEM teams and the synthetic biology community at large.
        
<A href="https://2017.igem.org/Team:Heidelberg/Model" class="big-link"><H3 style="color: #9d1c20 !important; text-align: left !important">Toolbox Characterization and Integrated Modeling</H3></A>             
To lay a thorough foundation for in vivo evolution with PREDCEL, we decided to apply bottom-up engineering and carefully characterize the individual components of our system in isolation first. Employing T7 polymerase as simple and modular platform, we established the required protocols for (i) cloning and generation of transgene-encoding phages, (ii) their propagation and selection on custom-made E. coli selection strains as well as (iii) accelerated in vivo mutation to expedite the evolutionary process. ODE-based <A href="https://2017.igem.org/Team:Heidelberg/Model" class="innerlink" style="color: #9d1c20 !important;">modeling</A> and corresponding computer simulations were thereby used to quantitatively investigate and optimize the parameters of our experimental system, i.e. <A href="https://2017.igem.org/Team:Heidelberg/Model/Phage_Titer" class="innerlink" style="color: #9d1c20 !important;">phage propagation times,</A> (Fig 2.)
<A href="https://2017.igem.org/Team:Heidelberg/Model/Mutagenesis_Induction" class="innerlink" style="color: #9d1c20 !important;"> mutagenesis-controlling inducer/inhibitor concentrations</A> and <A href="https://2017.igem.org/Team:Heidelberg/Model/Medium_Consumption" class="innerlink" style="color: #9d1c20 !important;">medium consumption</A>.
<DIV class="col-lg-12 col-md-12 col-xs-12 content" style="padding-top: 20px; padding-bottom: 20px;  text-align: justify!important"><DIV class="mdl-shadow--4dp" style="padding: 20px"><DIV style="font-weight: 700 !important;text-align: left!important">Figure 2: Basic logarithmic phage and E. coli titer plot with 100 % wildtype fitness.</DIV>
    
The blue lines correspond to the different <I>E. coli</I> populations. Exponential growth of <I>E. coli</I> and constant fitness of 100% of the wildtype, equal in all phages was assumed. After ten minutes infected <I>E. coli</I> start producing phage, corresponding to a stagnation in infected <I>E. coli</I> and an increase in phage concentration.
</DIV></DIV>
Apart from such simple-to-control parameters, our models also suggested that the fitness of the initial (parental) phage gene pool regarding the function to be evolved strongly impacts the robustness as well as speed of PREDCEL-mediated evolution.
        
       
        
We quickly realized that evolving truly novel functions on a given protein would be very challenging with any in vivo directed evolution method including PREDCEL and PACE, as they always depend on initial activity to build upon. 
<A href="https://2017.igem.org/Team:Heidelberg/Software" class="big-link"><H3 style="color: #ffd700 !important; text-align: left !important">Meet AiGEM, our Artificial Intelligence for Genetic Evolution Mimicking</H3></A>

To address these profound limitations and enable true evolutionary jumps from no/minimal activity towards high activity, our team created <A href="https://2017.igem.org/Team:Heidelberg/Software" class="innerlink" style="color: #ffd700 !important;">AiGEM</A>, the Artificial Intelligence for Genetic Evolution Mimicking software. AiGEM significantly reduces the time and cost required for directed evolution of functional proteins by pre-optimizing the parental gene pool for a selected function in silico. The heart of our AiGEM software is <A href="https://2017.igem.org/Team:Heidelberg/Software/DeeProtein" class="innerlink" style="color: #ffd700 !important;">DeeProtein</A>, a deep neuronal network trained on ~10 million protein sequences and able to infer sequence-function relationships from raw sequence data with high accuracy. To <A href="https://2017.igem.org/Team:Heidelberg/Software/DeeProtein" class="innerlink" style="color: #ffd700 !important;">validate DeeProtein’s predictive power</A>, we generated a set of <A href="https://2017.igem.org/Team:Heidelberg/Validation" class="innerlink" style="color: #ffd700 !important;">~30 beta-lactamase mutants</A> and found that the observed catalytic efficiencies (i.e. maximum antibiotic inhibitory concentrations) were reflected in the corresponding DeeProtein activity scores. (Fig. 2)
        <DIV class="row"><DIV class="col-lg-6 col-md-6 col-xs-12"><DIV class="content" style="padding-top: 20px; padding-bottom: 20px text-align: justify !important;width:100%;height:auto;"><DIV class="mdl-shadow--4dp" style="padding: 20px"><DIV style="font-weight: 700 !important;text-align: left!important">Figure 3: The DeeProtein classification score for screened \(\beta\)-lactamase variants correlates with the MIC of Carbenicillin.</DIV>The average DeeProtein classification scores assigned to samples in the MIC-score bins are depicted as black dots. The red line is the fitted linear model. Samples assigned with a high classification score tend to sustain higher carbenicillin concentrations, whereas a low classification score is assigned to variants with a low MIC.
</DIV></DIV></DIV><DIV class="col-lg-6 col-md-6 col-xs-12"><DIV class="content" style="padding-top: 20px; padding-bottom: 20px;text-align: justify !important;width:100%;height:auto;"><DIV class="mdl-shadow--4dp" style="padding: 20px"><DIV style="font-weight: 700 !important;text-align: left!important">Figure 4: Comparison Between the Wildtype Proteins and GUS_T509L</DIV>Our assay demonstrated that the wildtype GUS has no activity on a GAL substrate. The mutant predicted by GAIA however, exhibits extraordinary enzymatic activity on the GAL substrate.
</DIV></DIV></DIV></DIV>
        
        Remarkably, DeeProtein even suggested a number of mutations, which resulted in beta-lactamase variants several fold more active than the wild type.
Then, by interfacing DeeProtein with our genetic algorithm <A href="https://2017.igem.org/Team:Heidelberg/Software/GAIA" class="innerlink" style="color: #ffd700 !important;">GAIA</A>, we – for the first time - established a fully closed, in silico evolution cycle driven by an intelligent network and able to pre-optimize proteins for a selected activity. In other words, AiGEM can fast-forward directed evolution by means of intelligent computing. To demonstrate its predictive power, we used AiGEM for <A href="https://2017.igem.org/Team:Heidelberg/Validation" class="innerlink" style="color: #ffd700 !important;">in silico evolution of novel beta-galactosidases</A> from human beta-glucuronidase parental sequences. (Fig. 3)
<A href="https://2017.igem.org/Team:Heidelberg/Predcel" class="big-link"><H3 style="color: #fbb74b !important; text-align: left !important">Our Applications</H3></A>

Having laid out and validated our concept of an interfaced in vivo and in silico directed evolution, we finally aimed at more deeply exploring its potential for applications in basic research and biochemical production.
<A href="https://2017.igem.org/Team:Heidelberg/Protein_Interaction" class="innerlink" style="color: #393939 !important;"><H4 style="color: #393939!important; text-align: left !important">Improving selected protein-protein interactions </H4></A>

Engineering selected protein-protein interaction is a major goal in synthetic biology and used, e.g. for the construction of split reporter-bases biosensors or improved antigens in context of vaccine development. Using split T7 polymerase as example, we aimed at studying whether we can improve protein-protein interactions (in this case the auto-reassembly of the two split fragments) in PACE and PREDCEL. Following only 3 days of evolution, we obtained numerous, recurrent split T7 mutants in the gene pool. Remarkably, some mutations we positioned right at the prospective interface of the two T7 fragments, hinting at the successful evolution of improved split T7 variants.
<DIV class="row"><DIV class="col-lg-12 col-md-12 col-xs-12 content" style="position: relative;text-align: justify !important"><DIV class="mdl-shadow--4dp" style="padding: 20px"><DIV style="font-weight: 700 !important;text-align: left!important">Figure 5: Mutational pattern of the evolved split T7 variant. </DIV> Following three days of in vivo evolution with PACE (Link PACE page), a plague assay was performed and the split T7 insert of five individual phage clones was analyzed by sanger-sequencing. We observed a recurrent mutation (T877P) in three out of the five clones, suggesting an evolutionary advantage (i.e. increased fitness) of the corresponding split T7 mutant as compared its non-mutated counterpart.     
</DIV></DIV></DIV><A href="https://2017.igem.org/Team:Heidelberg/Cytochrome_Engineering" class="innerlink" style="color: #393939 !important;"><H4 style="color: #393939 !important; text-align: left !important">Engineering novel enzymes for organosilicon production</H4></A>

Finally, we investigated the potential of our evolution toolbox for the development of novel enzymes for biochemical production. <A href="https://2017.igem.org/Team:Heidelberg/Organosilicons" class="innerlink">Organosilicons</A> are a molecular class of high relevance for industry and with great potential for drug development as we learned during discussions with experts in the field. Although biological systems do not employ carbo-silicon chemistry in nature, promiscuous enzymes such as cytochromes exist, which are in principle capable of catalyzing carbon-silicon bond formations albeit with low efficiency.

We developed a complete <A href="https://2017.igem.org/Team:Heidelberg/Cytochrome_Engineering" class="innerlink">PREDCEL workflow capable of optimizing cytochromes for a specific purpose</A> by redirecting their catalytic activity towards a desired, but naturally unfavored reaction product. The workflow employs riboswitches for detection of the enzymatic reaction product and corresponding variant selection, which we design fully computationally in our <A href="https://2017.igem.org/Team:Heidelberg/Software/MAWS" class="innerlink">MAWS (Making Aptamers Without Selex) 2.0 software</A>, an improved MAWS algorithm originally introduced by iGEM Team Heidelberg 2015. We show that the in silico predicted riboswitches are capable of <A href="https://2017.igem.org/Team:Heidelberg/Organosilicons" class="innerlink">specifically detecting selected, organosilicon compounds</A><DIV class="content" style="padding-top: 20px; padding-bottom: 20px;padding-right: 20px; text-align: justify !important;width:60%;height:auto;float: left;"><DIV class="mdl-shadow--4dp" style="padding: 20px"><DIV style="font-weight: 700 !important;text-align: left!important">Figure 6:</DIV> Light emission detection of the NanoLuc reaction for different riboswitch activators and concentrations. Addition of compound (3) to the reaction resulted in increased enzyme activity as indicated by the two bars on the left-hand side compared to the compound (1) reaction
</DIV></DIV><DIV class="col-lg-12 col-md-12 col-xs-12 content" style="padding-top: 20px; padding-bottom: 20px;  text-align: justify !important"><DIV class="mdl-shadow--4dp" style="padding: 20px"><DIV style="font-weight: 700 !important;text-align: left!important">Figure 7: </DIV> Sequencing results of eight plaques after 6 iterations of the PREDCEL workflow with MP4 illustrating recurrently appearing mutations of the CYP1A2 gene. Recurrent mutations with amino acid exchange are indicated in red, without amino acid exchange in orange. Single mutations with amino acid exchange are shown in yellow, and without amino acid exchange in blue.    
</DIV></DIV>     
produced in vitro using a modified cytochrome C. Finally, employing the caffeine-metabolizing <A href="https://2017.igem.org/Team:Heidelberg/Cytochrome_Engineering" class="innerlink">Cytochrome P450 1A2</A>, we provide evidence for the successful application of PREDCEL for the directed evolution of an enzyme towards increased production of a naturally unfavored product, theophylline (Fig. 6)
<DIV class="row"><DIV class="col-lg-12 col-md-12 col-xs-12"><DIV class="content" style="padding-top: 20px; padding-bottom: 20px text-align: justify !important;width:100%;height:auto;"><DIV class="mdl-shadow--4dp" style="padding: 20px"><DIV style="font-weight: 700 !important;text-align: left!important">Figure 8:</DIV>Gas chromatogram for the reaction of educt (2) and (5) to the product (4). 9.2 minutes retention time indicates product formation. 
</DIV></DIV></DIV><DIV class="col-lg-12 col-md-12 col-xs-12"><DIV class="content" style="padding-top: 20px; padding-bottom: 20px;text-align: justify !important;width:100%;height:auto;"><DIV class="mdl-shadow--4dp" style="padding: 20px"><DIV style="font-weight: 700 !important;text-align: left!important">Figure 9:</DIV>Mass chromatogram shows the breakdown of the product (4) ethyl 2-(dimethyl(phenyl)silyl)propanoate. The product itself corresponds to a mass of 236 dalton
</DIV></DIV></DIV></DIV><A href="https://2017.igem.org/Team:Heidelberg/Human_Practices" class="big-link"><H3 style="color: #6698be !important; text-align: left !important">Integrated Human Practices</H3></A>
The most powerful technology can only have a positive impact on humanity, if it is widely accepted, safe, and applied in a responsible manner with the aim of making our world a better place to live. No single person or expert group is smart enough precisely foresee the impact – positive and negative - of a developing technology. Therefore, we consider integrated human practices of particular importance in context of foundational advance projects like ours, which aim at developing technologies with the potential to shape our future - and the future of our (future) kids.
From the early beginning on, we thus openly discussed the aforementioned concepts and ideas with <A href="https://2017.igem.org/Team:Heidelberg/Interviews" class="innerlink" style="color: #6698be!important;">experts</A> from the different fields and <A href="https://2017.igem.org/Team:Heidelberg/Engagement" class="innerlink" style="color: #6698be!important;">reached out to the broad public</A> to listen to their hopes and concerns. 
The <A href="https://2017.igem.org/Team:Heidelberg/HP/Gold_Integrated" class="innerlink" style="color: #6698be!important;">extensive feedback</A> we received pushed us to address three major, occurring issues: (i) Making our technology safe, (ii) stimulating its responsible use and (iii) apply it to address urgent human needs. (i) To safeguard in vivo evolution experiments, we created <A href="https://2017.igem.org/Team:Heidelberg/Software/SafetyNet" class="innerlink" style="color: #6698be!important;">SafetyNet</A>, which is part of our <A href="https://2017.igem.org/Team:Heidelberg/Software" class="innerlink" style="color: #6698be!important;">AiGEM</A> (Artificial Intelligence for Genetic Evolution Mimicking) software. SafetyNet checks any user supplied input sequence for “sleeping” hazardous potential. Thereby, we can strongly decrease the risk of evolving hazardous proteins unintendedly. (ii) We integrated a questionnaire <A href="https://2017.igem.org/Team:Heidelberg/Toolbox" class="innerlink" style="color: #6698be!important;">(“Ready-to-PREDCEL?”)</A> into our evolution toolbox guide to stimulate the responsible use of our technology. (iii) In the wet lab, we chose projects with the highest ecological/medical potential, and focused in the application of our toolbox on the engineering of enzymes for ecofriendly synthesis of <A href="https://2017.igem.org/Team:Heidelberg/Organosilicons" class="innerlink" style="color: #6698be!important;">organosilicons</A>, e.g. as novel pharmaceuticals.
<A href="https://2017.igem.org/Team:Heidelberg/Achievements" class="big-link"><H3 style="color: #393939!important; text-align: left !important">In Summary</H3></A>
We developed and successfully validated a comprehensive evolution toolbox comprising our standardized PREDCEL parts and protocols, interactive models and accompanying, intelligent software, all integrated into unique workflows for accelerated production of novel proteins and bio-compounds.  We believe that our work provides a new foundational advance by introducing an innovative in silico and in vivo directed evolution interface as novel <A href="https://2017.igem.org/Team:Heidelberg/Design" class="innerlink">engineering paradigm</A> to Synthetic Biology.
           
 </DIV></DIV></DIV>
           
}}
                          
</DIV><SECTION id="footer-sec" style="background-color: #222 !important"><DIV class="container"><DIV class="row" style="padding-top: 30px;"><DIV class="col-md-3"><H4 style="text-align: center">Quote</H4><P><I>“”</I></P></DIV><DIV class="col-md-3"><H4 style="text-align: center"> Useful <STRONG>Links</STRONG></H4><P><A href="https://igem.org/Special:SpecialPages">
			Special pages
		</A></P><P><A href="https://igem.org/Main_Page">
			Main Page
		</A></P></DIV><DIV class="col-md-3"><H4 style="text-align: center"> Follow us on </H4><DIV style="text-align: center;"><SPAN style="color: grey; text-align: center;">&amp;</SPAN></DIV></DIV><DIV class="col-md-3"><H4 style="text-align: center"> Contact us </H4><P>

               iGEM-Heidelberg2017@bioquant.uni-heidelberg.de
               
		 Im Neuenheimer Feld 267
		
		69120 Heidelberg
                    </P></DIV></DIV></DIV></SECTION></DIV></DIV></DIV></DIV></BODY></HTML>