<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Heidelberg_Experiments skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Heidelberg/Experiments</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container"><DIV class="navbar-header"><A type="button" data-canvas="body" id="sidebarCollapse" class="navbar-toggle" data-toggle="offcanvas" data-target="#sidebar" style="color: #393939;margin-top: 9px;">NAVIGATION
                    </A></DIV><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav pull-right"><LI class="dropdown"><A href="https://2017.igem.org/Team:Heidelberg/Team">People </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Team">Team</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Attributions">Attributions</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Sponsoring">Sponsoring</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/For_Judges">For Judges</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Jamboree">Giant Jamboree</A></LI></UL><LI class="dropdown"><A href="https://2017.igem.org/Team:Heidelberg/Description">Project </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Description">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Design">Background &amp; Design </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Pace">PACE</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Predcel">PREDCEL</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Results">Results Summary</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Validation">Software Validation</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Organosilicons">Organosilicons</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Protein_Interaction">Protein Interaction</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Cytochrome_Engineering">Cytochrome Engineering</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Optogenetics">Optogenetics</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/CRISPR">CRISPR Cas9</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/InterLab">InterLab Study</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Achievements">Achievements</A></LI><LI class="dropdown"><A href="https://2017.igem.org/Team:Heidelberg/Parts">Parts </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Basic_Part">Basic Parts</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Composite_Part">Composite Parts</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Improved_Part">Improved Part</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Part_Collection">Parts Collection</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Part_List">Part List</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/RFC">RFC</A></LI><LI class="dropdown"><A href="https://2017.igem.org/Team:Heidelberg/Model">Modeling </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Phage_Titer">Phage Titer</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Mutagenesis_Induction">Mutagenesis Induction</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Mutation_Rate_Estimation">Mutation Rate Estimation</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Lagoon_Contamination">Lagoon Contamination</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Medium_Consumption">Medium Consumption</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Tools">Interactive Web Tools</A></LI><LI class="dropdown"><A href="https://2017.igem.org/Team:Heidelberg/Software">Software </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software">AiGEM Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software/DeeProtein">DeeProtein</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software/GAIA">GAIA</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software/SafetyNet">SafetyNet</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software/MAWS">MAWS 2.0</A></LI><LI class="dropdown"><A href="https://2017.igem.org/Team:Heidelberg/Human_Practices">Human Practices </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Human_Practices">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/HP/Gold_Integrated">Integrated Human Practices</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Interviews">Expert Interviews</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Safety">Safety &amp; Security</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Engagement">Public Engagement</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Education">Education</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Collaborations">Collaborations</A></LI><LI class="dropdown"><A href="https://2017.igem.org/Team:Heidelberg/Notebook">Notebook </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Notebook">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Experiments">Methods</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Materials">Materials</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Database">Notebook Database</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Toolbox">Toolbox Guide</A></LI></DIV></DIV><NAV id="sidebar" style="z-index: 10001; background-color: white;border: none;box-shadow:1px 0px 0px #393939;" class="navmenu navmenu-default navmenu-fixed-left offcanvas" role="navigation"><A class="navmenu-brand" href="https://2017.igem.org/Team:Heidelberg"> iGEM TEAM HEIDELBERG 2017</A><DIV id="sidebarHide" data-toggle="offcanvas" data-target="#sidebar" style="padding-left: 20px;"><A style="font-family: 'Josefin Sans' !important; font-size: 16px; line-height: 18px; height: 18px;"> COLLAPSE NAVIGATION </A></DIV><UL class="nav navmenu-nav" id="sidenav"><LI><A data-toggle="collapse" data-target="#People_side">People </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Team">Team</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Attributions">Attributions</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Sponsoring">Sponsoring</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/For_Judges">For Judges</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Jamboree">Giant Jamboree</A></LI></UL><LI><A data-toggle="collapse" data-target="#Project_side">Project </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Description">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Design">Background &amp; Design </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Pace">PACE</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Predcel">PREDCEL</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Results">Results Summary</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Validation">Software Validation</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Organosilicons">Organosilicons</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Protein_Interaction">Protein Interaction</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Cytochrome_Engineering">Cytochrome Engineering</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Optogenetics">Optogenetics</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/CRISPR">CRISPR Cas9</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/InterLab">InterLab Study</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Achievements">Achievements</A></LI><LI><A data-toggle="collapse" data-target="#Parts_side">Parts </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Basic_Part">Basic Parts</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Composite_Part">Composite Parts</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Improved_Part">Improved Part</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Part_Collection">Parts Collection</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Parts#Part_List">Part List</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/RFC">RFC</A></LI><LI><A data-toggle="collapse" data-target="#Model_side">Modeling </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Phage_Titer">Phage Titer</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Mutagenesis_Induction">Mutagenesis Induction</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Mutation_Rate_Estimation">Mutation Rate Estimation</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Lagoon_Contamination">Lagoon Contamination</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Medium_Consumption">Medium Consumption</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Model/Tools">Interactive Web Tools</A></LI><LI><A data-toggle="collapse" data-target="#AI_side">Software </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software">AiGEM Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software/DeeProtein">DeeProtein</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software/GAIA">GAIA</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software/SafetyNet">SafetyNet</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Software/MAWS">MAWS 2.0</A></LI><LI><A data-toggle="collapse" data-target="#HP_side">Human Practices </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Human_Practices">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/HP/Gold_Integrated">Integrated Human Practices</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Interviews">Expert Interviews</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Safety">Safety &amp; Security</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Engagement">Public Engagement</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Education">Education</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Collaborations">Collaborations</A></LI><LI><A data-toggle="collapse" data-target="#Notebook_side">Notebook </A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Notebook">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Experiments">Methods</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Materials">Materials</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Database">Notebook Database</A></LI><LI><A href="https://2017.igem.org/Team:Heidelberg/Toolbox">Toolbox Guide</A></LI></NAV><DIV style="background-color: white !important; padding: 0 !important; margin: 0 !important;"><DIV class="page-title" style="background-color: white !important; border-top: 0px solid #222;"><DIV class="container" style=" padding: 0 !important;"><DIV class="col-lg-12 col-md-12 col-xs-12" style="padding-right: 0 !important; padding-bottom: 0 !important; position: relative; left: -10px;"><DIV style="border-right: 5px solid #393939; padding: 20px 20px 0px 0px"><DIV class="header-title heidelberg-orange"> Notebook </DIV><DIV class="header-subtitle" style="color: #393939 !important;"> Methods </DIV></DIV></DIV></DIV></DIV><DIV class="container" style="background-color: white !important; padding: 0 !important;"><DIV class="col-lg-12 col-md-12 col-xs-12" style="padding: 0 !important; margin: 0px important!;"><DIV class="container"><DIV class="row"><DIV class="col-lg-12 col-md-12 col-xs-12 content" style="display: inline-block; padding-bottom: 10px; padding-top:10px;"><DIV class="panel-group accordion" id="accordion"><DIV class="panel panel-default"><A data-toggle="collapse" data-parent="#accordion" href="#collapsetable_id-0"><DIV class="panel-heading">
            Agarose-Gelelectrophoresis
        </DIV></A><DIV id="collapsetable_id-0" class="panel-collapse collapse"><DIV class="panel-body"><OL class="content"><LI>Prepare a 1 % agarose-gel: Dissolve the agarose in 1X TAE by boiling and add ethidium bromide (5 µl per 100 ml agarose solution).For short DNA fragments, gels with higher agarose percentage should be used. 
</LI><LI>Pour the gel and let it polymerize until it is solid.
</LI><LI>Add the appropriate amount of 6X purple loading dye (NEB) to the PCR or digestion reaction mix (8 µl for 50 µl PCR reaction).
</LI><LI>Place the gel into the tank and add 1X TAE buffer so that the gel is fully covered.
</LI><LI>Pipet the DNA into the pockets.
</LI><LI>Run the gel at 120 V for 20 min.
</LI><LI>Image the gel using ultraviolet light. If necessary, cut out bands.
</LI></OL></DIV></DIV></DIV><DIV class="panel panel-default"><A data-toggle="collapse" data-parent="#accordion" href="#collapsetable_id-1"><DIV class="panel-heading">
            Beta-Galactosidase Activity Assay
        </DIV></A><DIV id="collapsetable_id-1" class="panel-collapse collapse"><DIV class="panel-body">
To determine the activities of the wildtype ß-galactosidase and the ß-glucuronidase mutants, an assay with p-nitrophenyl-galactopyranoside (PNPG) was setup.
1X assay buffer consisted of the following substances:

<DIV class="col-lg-12 col-md-12 col-xs-12 content" style="padding-top: 20px; padding-bottom: 20px"><DIV class="table-slot" style="overflow-x: auto"><TABLE class="table table-bordered mdl-shadow--4dp" xsscleaned="overflow-x: scroll !important"><THEAD><TR><TH>Chemical</TH><TH>Conentration [mM]</TH></TR></THEAD><TBODY><TR><TD>Disodium phosphate</TD><TD>100</TD></TR><TR><TD>Magnesium chloride</TD><TD>1</TD></TR></TBODY></TABLE></DIV></DIV>

Beta-mercaptoethanol</DIV></DIV></DIV><DIV class="panel panel-default"><A data-toggle="collapse" data-parent="#accordion" href="#collapsetable_id-2"><DIV class="panel-heading">
            Beta-Galactosidase assay on Agar-plates
        </DIV></A><DIV id="collapsetable_id-2" class="panel-collapse collapse"><DIV class="panel-body">ß-Galactosidase assay on plates
To generate a visual output for ß-Galactosidase activity of the ß-glucuronidase (GUS) mutants, a 5-bromo-4-chloro-3-indolyl-β-D-galactopyranoside (X-Gal) assay was set up. Therefore, LB-agar plates with 100 µg/ml kanamycin, 4 mg/ml X-Gal and 0.5 mM IPTG were prepared. Cultures with the different GUS mutants and the wiltype enzyme were grown overnight to the stationary phase. The next day, 10 µl of each culture were plated in eighths on the same plates. When the blue color started to develop, the plates were imaged.
</DIV></DIV></DIV><DIV class="panel panel-default"><A data-toggle="collapse" data-parent="#accordion" href="#collapsetable_id-3"><DIV class="panel-heading">
            Blue Plaque Assay
        </DIV></A><DIV id="collapsetable_id-3" class="panel-collapse collapse"><DIV class="panel-body"><H2 id="1-introduction">1. Introduction</H2>
The Blue Plaque Assay is a method to determine M13 phage titer.
<H2 id="2-material">2. Material</H2><STRONG>LB/Amp-Agar Plates (1.5 % agar):</STRONG>Plates with bottom agar (15 g/L agar in LB medium) supplemented with antibiotic are prepared at least 30 min before plaque assay.
<STRONG>Top Agar (0.7 % agar):</STRONG>Top agar containing 7 g/l agar is autoclaved and stored at ~56 °C in the heating cabinet. 3 ml top agar is sufficient for one agar plate.
<STRONG>X-gal stock solution (40 mg/ml):</STRONG> 0.2 g X-gal is dissolved in 5 ml DMSO and stored at -20 °C in the dark. It is useful to aliquot the stock solution in 1.5-ml reaction tubes.
<STRONG>Transformed S2060 <EM>E. coli</EM> strain (ID:47):</STRONG>S2060 <EM>E. coli</EM> possess the lacZ gene under control of a PSP promoter. For the Blue Plaque Assay a transformed S2060 strain like <EM>E. coli</EM> strain ID:47 containing the AP of the S1059 strain (gene III under a PSP promoter) are used.
(This protocol can be modified to investigate if the evolving phage can resist stronger selection pressure. Therefore S2060 strain has to be transformed with the associated AP and the antibiotics has to be adapted)
<H2 id="3-procedure">3. Procedure</H2><STRONG>Day 1:</STRONG>
S2060 AP_PSP_gIII cells (ID:47) or the appropriate cell type are used to inoculate 4 mL of 2xYT medium containing 25 mM glucose and carbenicillin or the necessary antibiotic for that cell type.The culture is placed in the 37°C shaker in the morning and are grown until an  OD600 of 0.6-0.8.
In the meantime, 10 to 10<SUP>2</SUP>-fold serial dilutions of the phage suspension are prepared with LB medium. 1 ml final volumes are convenient. To prevent cross-contamination filter tips must be used. The phage samples can be stored at 4 °C for a long time and still be able to infect <EM>E. coli</EM> cells.
Shortly before the <EM>E. coli</EM> culture reaches the appropriate OD600 top agar (3 ml per plate) is transferred to a 50-ml falcon and placed in a preheated water bath at 50 °C. Top agar is then supplemented with the necessary antibiotic and the x-gal stock solution (40 mg/ml).
For 10 ml top agar:
<UL class="content"><LI>10 µl antibiotic (1000x)</LI><LI>100 µl X-gal stock solution (100x)</LI></UL>
As soon as the <EM>E. coli</EM> culture reach an OD600 of 0.6-0.8, cell suspension is gently mixed by resuspension with a pipette and 50 µl are transferred to a 1.5-ml reaction tube for each condition. The following steps has to be performed quickly and separate for each condition. Phage suspension is shortly vortexed and 1 µl is pipetted to the <EM>E. coli</EM> culture. 700 µl of the top agar is thorough mixed with the cell and phage suspension. This is immediately poured on the corresponding quarter of the plate and rocked gently to evenly spread it.
After allowing the top agar to solidify for few minutes, plates were sealed with parafilm and incubated overnight at 37°C.
<STRONG>Day 2:</STRONG>
On the next day blue plaques can be counted. For quantification of the phage titer the number of plaques should be between 10 and 100.The amount of phages in the solution is calculated by:
PFU/ml = number of plaques x dilution factor x 1000
e.g. 12 blue plaques are counted on a plate quarter where 1 µl of a 10<SUP>-4</SUP> phage dilution was plated.12 x 10<SUP>4</SUP> x 1000 = 1.2 x 10<SUP>8</SUP> PFU/ml
</DIV></DIV></DIV><DIV class="panel panel-default"><A data-toggle="collapse" data-parent="#accordion" href="#collapsetable_id-4"><DIV class="panel-heading">
            Chemical Competent Cells
        </DIV></A><DIV id="collapsetable_id-4" class="panel-collapse collapse"><DIV class="panel-body">
Preparation of Calcium Chloride Competent E. coli cells:
<OL class="content"><LI>Inoculate 5 ml LB with a colony of the respective strain and let it grow overnight at 37 °C at 220 rpm. If the strain carries a plasmid, do not forget to add the respective antibiotics. 
</LI><LI>The next day, inoculate 250 ml LB medium with 1 ml of the overnight culture and let it grow until it reaches an OD600 of 0.6-0.7. * Work on ice from now on. 
</LI><LI>Cool the cells on ice for 10 min. 
</LI><LI>Centrifuge the cells for 10 min at 6000 x g and discard the supernatant. 
</LI><LI>Resuspend the pellet in 10 ml of pre cooled 0.1 M CaCl2
</LI><LI>Incubate the suspension for 20 min on ice. 
</LI><LI>Centrifuge again for 10 min at 6000 g and discard the supernatant. 
</LI><LI>. Resuspend the pellet in 5 ml of cold 0.1 M CaCl with 15% Glycerol
</LI><LI>Freeze 50 µl aliquots of bacteria suspension in liquid nitrogen and store at -80 °C.
<DIV class="panel panel-default"><A data-toggle="collapse" data-parent="#accordion" href="#collapsetable_id-5"><DIV class="panel-heading">
            Chemical Transformation
        </DIV></A><DIV id="collapsetable_id-5" class="panel-collapse collapse"><DIV class="panel-body">
Transformation into chemical competent Cells:
</DIV></DIV></DIV></LI><LI>Thaw an aliquot of CaCl competent cells on ice.
</LI><LI>Add 5 µl of the ligation reaction and incubate for 30 min on ice.
</LI><LI>Heat shock the cells for 1 min at 42 °C
</LI><LI>Incubate on ice for 5 min.
</LI><LI>Add 1 ml of SOC and incubate for 1 h at 37 °C on a shaker.
</LI><LI>Plate the appropriate amount of cells on agar with the respective antibiotic.
<DIV class="panel panel-default"><A data-toggle="collapse" data-parent="#accordion" href="#collapsetable_id-6"><DIV class="panel-heading">
            Circular Polymerase Extension Cloning
        </DIV></A><DIV id="collapsetable_id-6" class="panel-collapse collapse"><DIV class="panel-body">Circular Polymerase Extension Cloning (CPEC)
</DIV></DIV></DIV></LI><LI>Mix 75 ng of backbone with equimolar amounts of inserts (short fragments can be used in excess) and add 25 µl Phusion Flash Master Mix (Thermo Fisher Scientific). Fill up to 50 µl with dH2O.
</LI><LI>Run the polymerase chain reaction in a thermocycler with the following protocol:
</LI></OL><DIV class="col-lg-12 col-md-12 col-xs-12 content" style="padding-top: 20px; padding-bottom: 20px"><DIV class="table-slot" style="overflow-x: auto"><TABLE class="table table-bordered mdl-shadow--4dp" xsscleaned="overflow-x: scroll !important"><THEAD><TR><TH>Step</TH><TH>Temperature</TH><TH>Time</TH></TR></THEAD><TBODY><TR><TD>1</TD><TD>98 °C</TD><TD>10 s</TD></TR><TR><TD>2</TD><TD>98 °C</TD><TD>1 s</TD></TR><TR><TD>slow ramp</TD><TD>0.1 °C/s</TD></TR><TR><TD>3</TD><TD>55 °C</TD><TD>5 s</TD></TR><TR><TD>4</TD><TD>72 °C</TD><TD>15-30 s/kb</TD></TR><TR><TD>5</TD><TD>72 °C</TD><TD>5 min</TD></TR><TR><TD>repeat step 2-4 25x</TD></TR></TBODY></TABLE></DIV></DIV>Transform a few µl of the reaction into a suitable bacteria strain</DIV></DIV></DIV><DIV class="panel panel-default"><A data-toggle="collapse" data-parent="#accordion" href="#collapsetable_id-7"><DIV class="panel-heading">
            DNA Preparation
        </DIV></A><DIV id="collapsetable_id-7" class="panel-collapse collapse"><DIV class="panel-body">
For DNA preparation (Miniprep, PCR purification and gel extraction), kits provided by Qiagen and Zymo Research were used according to the manufacturer's instructions. DNA concentration was determined using a nanophotometer.
</DIV></DIV></DIV><DIV class="panel panel-default"><A data-toggle="collapse" data-parent="#accordion" href="#collapsetable_id-8"><DIV class="panel-heading">
            Electrocompetent Cells
        </DIV></A><DIV id="collapsetable_id-8" class="panel-collapse collapse"><DIV class="panel-body">
Before beginning, autoclave the following solutions:
<UL class="content"><LI>Low salt media:     4.5 g Tryptone, 2.25 g Yeast Extract, 1.8 g NaCl in 450 ml dH2OdH2O: 1 l
</LI><LI>10 % Glycerol:      400 ml
</LI></UL>
Procedure:
<OL class="content"><LI>Prepare an overnight culture of the desired <EM>E. coli</EM> strain
</LI><LI>The next morning, inoculate the low salt medium with the overnight culture
</LI><LI>Let the culture grow until an OD600 of 0.6-0.7 is reached.
</LI><LI>Chill the flask on ice for 15 min
</LI><LI>Centrifuge the bacteria for 20 min at 3000 g at 4 °C
</LI><LI>Discard the supernatant and resuspend it in 400 ml cold H2O
</LI><LI>Repeat the washing step
</LI><LI>Centrifuge the bacteria for 20 min at 3000 g at 4 °C and discard the remaining water
</LI><LI>Resuspend the cells in 400 ml glycerol
</LI><LI>Centrifuge agein at 3000 g for 20 min (4 °C)
</LI><LI>Discard the supernatant and resuspend the pellet in approximately 1 ml of remaining glycerol.
</LI><LI>Freeze 50 µl aliquots in liquid nitrogen and store them at -80 °C
</LI></OL></DIV></DIV></DIV><DIV class="panel panel-default"><A data-toggle="collapse" data-parent="#accordion" href="#collapsetable_id-9"><DIV class="panel-heading">
            Electroporation
        </DIV></A><DIV id="collapsetable_id-9" class="panel-collapse collapse"><DIV class="panel-body">Transformation of <EM>E. coli</EM> via electroporation
Preparation:
<OL class="content"><LI>Pipet 1 µl of the plasmid solution into a pre-cooled 1.5 ml reaction tube. Ligation products have to be diluted in H2O (1:4 usually works) or purified in advance.
</LI><LI>Heat 1 ml of SOC to 37 °C.
</LI><LI>Pre-cool an 1 mm electroporation cuvette on ice.
</LI><LI>Thaw an aliquot of electrocompetent cells on ice.
</LI></OL>
Electroporation Procedure:
<OL class="content"><LI>Pipet 25 µl of the competent cells into the tube with the plasmid solution.
</LI><LI>Transfer the mixture carefully into the cuvette (avoid air bubbles)
</LI><LI>Electroporate at 1800 kV for 5 ms.
</LI><LI>Immediately add 1 ml SOC, mix by pipetting up and down, and transfer the solution into a 1.5 ml reaction tube
</LI><LI>Incubate for 1 h at 37 °C at 350 rpm
</LI><LI>Plate 100 µl on agar with the respective antibiotics.
Note: If more than one plasmid has to be transformed, the procedure stays the same. Both plasmids have to be mixed in equal amount prior to the addition to the cells. Again, only 1 µl should be used. For the last step, agar plates with 2 antibiotics have to be used.
<DIV class="panel panel-default"><A data-toggle="collapse" data-parent="#accordion" href="#collapsetable_id-10"><DIV class="panel-heading">
            Gibson Assembly
        </DIV></A><DIV id="collapsetable_id-10" class="panel-collapse collapse"><DIV class="panel-body">
Preparation of Gibson Assembly mastermix using the protocol provided by the Miller lab:
Preparation of 5X ISO-buffer:

<DIV class="col-lg-12 col-md-12 col-xs-12 content" style="padding-top: 20px; padding-bottom: 20px"><DIV class="table-slot" style="overflow-x: auto"><TABLE class="table table-bordered mdl-shadow--4dp" xsscleaned="overflow-x: scroll !important"><THEAD><TR><TH style="text-align:right">Component</TH><TH style="text-align:right">Amount</TH></TR></THEAD><TBODY><TR><TD style="text-align:right">1 M Tris-HCl pH 7.5</TD><TD style="text-align:right">3 ml</TD></TR><TR><TD style="text-align:right">2 M MgCl2</TD><TD style="text-align:right">150 µl</TD></TR><TR><TD style="text-align:right">100 nM dNTP mix</TD><TD style="text-align:right">60 µl</TD></TR><TR><TD style="text-align:right">1 M DTT</TD><TD style="text-align:right">300µl</TD></TR><TR><TD style="text-align:right">PEG-8000</TD><TD style="text-align:right">1.5 g</TD></TR><TR><TD style="text-align:right">100 mM NAD</TD><TD style="text-align:right">300 µl</TD></TR><TR><TD style="text-align:right">dH2O</TD><TD style="text-align:right">add to 6 ml</TD></TR></TBODY></TABLE></DIV></DIV>Preparation of Gibson Assembly mastermix:

<DIV class="col-lg-12 col-md-12 col-xs-12 content" style="padding-top: 20px; padding-bottom: 20px"><DIV class="table-slot" style="overflow-x: auto"><TABLE class="table table-bordered mdl-shadow--4dp" xsscleaned="overflow-x: scroll !important"><THEAD><TR><TH style="text-align:right">Component</TH><TH style="text-align:right">Amount</TH></TR></THEAD><TBODY><TR><TD style="text-align:right">5X ISO buffer</TD><TD style="text-align:right">320 µl</TD></TR><TR><TD style="text-align:right">10U/µl T5 exonuclease</TD><TD style="text-align:right">0.64 µl</TD></TR><TR><TD style="text-align:right">2 U/µl Phusion ploymerase</TD><TD style="text-align:right">20 µl</TD></TR><TR><TD style="text-align:right">40 U/µl Taq ligase</TD><TD style="text-align:right">160 µl</TD></TR><TR><TD style="text-align:right">dH2O</TD><TD style="text-align:right">add to 1.2 ml</TD></TR></TBODY></TABLE></DIV></DIV>Gibson Assembly Procedure:
</DIV></DIV></DIV></LI><LI>Mix 10 µl Gibson Assembly mastermix with 0.02–0.5 pmol of backbone and equimolar amounts of insert.
</LI><LI>Incubate at 50 °C for 15-60 min
</LI><LI>Use 5 µl for transformation in CaCl competent cells.
<OL class="content"><DIV class="panel panel-default"><A data-toggle="collapse" data-parent="#accordion" href="#collapsetable_id-11"><DIV class="panel-heading">
            Glycerol-Stocks
        </DIV></A></DIV></OL></LI><LI>Prepare a 50 % glycerol solution diluted in H2O and autoclave it.
</LI><LI>Mix 500 µl of an overnight culture with 500 µl of 50 % glycerol solution, and store it at -80 °C.
</LI></OL></DIV></DIV></DIV><DIV class="panel panel-default"><A data-toggle="collapse" data-parent="#accordion" href="#collapsetable_id-12"><DIV class="panel-heading">
            Golden Gate Assembly
        </DIV></A><DIV id="collapsetable_id-12" class="panel-collapse collapse"><DIV class="panel-body">
Prepare the following reaction mix with a total volume of 15 µl:

<DIV class="col-lg-12 col-md-12 col-xs-12 content" style="padding-top: 20px; padding-bottom: 20px"><DIV class="table-slot" style="overflow-x: auto"><TABLE class="table table-bordered mdl-shadow--4dp" xsscleaned="overflow-x: scroll !important"><THEAD><TR><TH>Component</TH><TH>Volume [µl]</TH></TR></THEAD><TBODY><TR><TD>T4 DNA Ligase (NEB)</TD><TD>1</TD></TR><TR><TD>10x T4 DNA Ligase Reaction Buffer (NEB)</TD><TD>1.5</TD></TR><TR><TD>Type IIS cutter</TD><TD>1</TD></TR><TR><TD>DNA fragment</TD><TD>X</TD></TR><TR><TD>dH2O</TD><TD>adjust to 15 µl</TD></TR></TBODY></TABLE></DIV></DIV>

For our project, we used the restriction enzymes BsaI-HF (NEB), BbsI-HF (NEB), and Esp3I (Thermo Fisher Scientific) for Golden Gate Assembly.
75 ng of backbone DNA is mixed with and equimolar amount of inserts. If an insert is shorter than 500 bp, it was added in 2:1 excess.
For the polymerase chain reaction the following programm was used in a thermocycler:

<DIV class="col-lg-12 col-md-12 col-xs-12 content" style="padding-top: 20px; padding-bottom: 20px"><DIV class="table-slot" style="overflow-x: auto"><TABLE class="table table-bordered mdl-shadow--4dp" xsscleaned="overflow-x: scroll !important"><THEAD><TR><TH>Step</TH><TH>Temperature</TH><TH>Time</TH></TR></THEAD><TBODY><TR><TD>1</TD><TD>37 °C</TD><TD>2 min</TD></TR><TR><TD>2</TD><TD>16 °C</TD><TD>3 min</TD></TR><TR><TD>3</TD><TD>16 °C</TD><TD>5 min</TD></TR><TR><TD>4</TD><TD>55 °C</TD><TD>5 min</TD></TR><TR><TD>Repeat step one and two 20x</TD></TR></TBODY></TABLE></DIV></DIV>Use 5 µl of the reaction mix for a chemical transformation.</DIV></DIV></DIV><DIV class="panel panel-default"><A data-toggle="collapse" data-parent="#accordion" href="#collapsetable_id-13"><DIV class="panel-heading">
            MP testing
        </DIV></A><DIV id="collapsetable_id-13" class="panel-collapse collapse"><DIV class="panel-body">

In order to test whether your bacterial strain contains an active mutagenesis plasmid, pre-testing is recommended before starting PACE using this strain. Therefore, the following steps have to be proceeded:
<OL class="content"><LI> Pick a single colony of the bacterial strain containing the AP and MP. It is important, that all strains carrying a MP are plated on plates containing a minimum of 25 mM Glucose (100 mM recommended) for prevention of the induction of the mutagenesis. Additionally, the plates used for single colony growth should be containing the appropriate antibiotics for AP and MP, as well as tetracycline for confirmation of F-pilus positive plasmids.

</LI><LI>Let the picked single colony grow in 2YT medium with the associated antibiotics for 12 - 18 hours in a 37°C incubator until OD600 0.6 - 0.8 is reached.

</LI><LI>The single colony growth should be serially diluted and plated on plates with the appropriate antibiotics and either 100 mM glucose or 100 mM arabinose. Plates are grown for another 12 - 18 hours.

</LI><LI>In parallel with the plate assay, a 50 ml starter culture for starting the PACE turbidostat should be inoculated using the pre-culture from *step 2*. Bacterial starter culture should grow to log-phase density (OD600 0.5 - 0.8).

</LI><LI>If the plate assay shows a positive result by displaying delayed growth of the bacteria on the arabinose plate, the 50 ml starter culture can be used for inoculation of the turbidostat.
<OL class="content"><DIV class="panel panel-default"><A data-toggle="collapse" data-parent="#accordion" href="#collapsetable_id-14"><DIV class="panel-heading">
            Ligation
        </DIV></A></DIV></OL></LI><LI>Assemble the reaction mix as listed in the table below:
</LI><LI>Incubate at room temperature for 10 min or overnight at 16 °C.
</LI><LI>Heat inactivate the ligase at 65 °C for 10 min
</LI><LI> Use 1-5 µl for transformation into competent cells.
</LI></OL><DIV class="col-lg-12 col-md-12 col-xs-12 content" style="padding-top: 20px; padding-bottom: 20px"><DIV class="table-slot" style="overflow-x: auto"><TABLE class="table table-bordered mdl-shadow--4dp" xsscleaned="overflow-x: scroll !important"><THEAD><TR><TH>Component</TH><TH>Volume</TH></TR></THEAD><TBODY><TR><TD>T4 DNA Ligase (NEB)</TD><TD>1 µl</TD></TR><TR><TD>10x T4 DNA Ligase Reaction Buffer (NEB)</TD><TD>1.5 µl</TD></TR><TR><TD>Vector backbone</TD><TD>50 ng</TD></TR><TR><TD>Insert</TD><TD>add in 3:1 molar ratio</TD></TR><TR><TD>dH2O</TD><TD>adjust to 15 µl</TD></TR></TBODY></TABLE></DIV></DIV></DIV></DIV></DIV><DIV class="panel panel-default"><A data-toggle="collapse" data-parent="#accordion" href="#collapsetable_id-15"><DIV class="panel-heading">
            Overnight-Culture
        </DIV></A><DIV id="collapsetable_id-15" class="panel-collapse collapse"><DIV class="panel-body"><OL class="content"><LI>Prepare LB media by resolving 25 g LB-powder (Roth) in 1 l of dH2O, and autoclave it.
</LI><LI>Transfer 4 ml of LB into a culture tube, and add 4 µl of the respective antibiotic stock solutions
</LI><LI>Inoculate the culture by picking a single colony with a pipet tip and tipping it into the medium or by adding a few microliters from a glycerol-stock.
</LI><LI>Shake the culture at 37 °C at 220 rpm for 12-14 h.
</LI><LI>Isolate plasmids or use the culture for further experiments.
</LI></OL>
Concentrations of antibiotics:

<DIV class="col-lg-12 col-md-12 col-xs-12 content" style="padding-top: 20px; padding-bottom: 20px"><DIV class="table-slot" style="overflow-x: auto"><TABLE class="table table-bordered mdl-shadow--4dp" xsscleaned="overflow-x: scroll !important"><THEAD><TR><TH>Antibiotic</TH><TH>Stock Concentration (1000X)</TH><TH>Final Concentration</TH></TR></THEAD><TBODY><TR><TD>Ampicillin</TD><TD>100 mg/ml</TD><TD>100 µg/ml</TD></TR><TR><TD>Streptomycin</TD><TD>100 mg/ml</TD><TD>100 µg/ml</TD></TR><TR><TD>Kanamycin</TD><TD>50 mg/ml</TD><TD>50 µg/ml</TD></TR><TR><TD>Chloramphenicol</TD><TD>25 mg/ml</TD><TD>25 µg/ml</TD></TR><TR><TD>Spectinomycin</TD><TD>50 mg/ml</TD><TD>50 µg/ml</TD></TR><TR><TD>Carbenicillin</TD><TD>100 mg/ml</TD><TD>100 µg/ml</TD></TR><TR><TD>Tetracyclin</TD><TD>10 mg/ml</TD><TD>10 µg/ml</TD></TR></TBODY></TABLE></DIV></DIV></DIV></DIV></DIV><DIV class="panel panel-default"><A data-toggle="collapse" data-parent="#accordion" href="#collapsetable_id-16"><DIV class="panel-heading">
            Phage detection PCR
        </DIV></A><DIV id="collapsetable_id-16" class="panel-collapse collapse"><DIV class="panel-body">

In order to make sure that phages are present in the turbidostat, a phage detection PCR was performed. This PCR uses three different primer pairs, yielding three characteristic PCR products in lengths of 200, 400 and 750 bp.
By this, it is possible to efficiently detect phages in the lagoons to check for phage washout or for evaluating a possible contamination of the turbidostat.

<DIV class="col-lg-12 col-md-12 col-xs-12 content" style="padding-top: 20px; padding-bottom: 20px"><DIV class="table-slot" style="overflow-x: auto"><TABLE><THEAD><TR><TH>Component </TH><TH>Amount </TH></TR></THEAD><TBODY><TR><TD>2x OneTaq Polymerase MM</TD><TD>10 µl</TD></TR><TR><TD>Primer A [10 µM]</TD><TD>1 µl</TD></TR><TR><TD>Primer B [10 µM]</TD><TD>1 µl</TD></TR><TR><TD>Primer C [10 µM]</TD><TD>1 µl</TD></TR><TR><TD>Primer D [10 µM]</TD><TD>1 µl</TD></TR><TR><TD>Primer E [10 µM]</TD><TD>1 µl</TD></TR><TR><TD>Primer F [10 µM]</TD><TD>1 µl</TD></TR><TR><TD>Phage supernatant</TD><TD>2 µl</TD></TR><TR><TD>ddH2O</TD><TD>to 20 µl</TD></TR></TBODY></TABLE></DIV></DIV><H2 id="id">PCR protocol</H2><DIV class="col-lg-12 col-md-12 col-xs-12 content" style="padding-top: 20px; padding-bottom: 20px"><DIV class="table-slot" style="overflow-x: auto"><TABLE><THEAD><TR><TH>Step</TH><TH>Temperature [°C]</TH><TH>Time [s]</TH></TR></THEAD><TBODY><TR><TD>1</TD><TD>94</TD><TD>300</TD></TR><TR><TD>2</TD><TD>94</TD><TD>30</TD></TR><TR><TD>3</TD><TD>xy</TD><TD>30</TD></TR><TR><TD>4</TD><TD>72</TD><TD>90</TD></TR><TR><TD>5</TD><TD>72</TD><TD>180</TD></TR><TR><TD>repeat step 2 - 4 30x</TD><TD> </TD></TR></TBODY></TABLE></DIV></DIV></DIV></DIV></DIV><DIV class="panel panel-default"><A data-toggle="collapse" data-parent="#accordion" href="#collapsetable_id-1600"><DIV class="panel-heading">
            Plaque sequencing PCR
        </DIV></A><DIV id="collapsetable_id-1600" class="panel-collapse collapse"><DIV class="panel-body">
In order to reveal mutations which we were possibly acquired during a PACE or PREDCEL run, plaque PCRs are highly recommended. Therefore, a plaque assay is performed following our *plaque assay* protocol.
Based on the results of the plaque assays, single plaques, belonging to single phages infecting a bacterium, should be picked and the gene of interest should be amplified using a HF-polymerase and suitable primers. In total, the PCR mastermix should be prepared equivalent to a mastermix for a colony PCR of bacteria.

<DIV class="col-lg-12 col-md-12 col-xs-12 content" style="padding-top: 20px; padding-bottom: 20px"><DIV class="table-slot" style="overflow-x: auto"><TABLE><THEAD><TR><TH>Component </TH><TH>Amount </TH></TR></THEAD><TBODY><TR><TD>2x Q5 Polymerase MM</TD><TD>10 µl</TD></TR><TR><TD>Primer A [10 µM]</TD><TD>1 µl</TD></TR><TR><TD>Primer B [10 µM]</TD><TD>1 µl</TD></TR><TR><TD>ddH2O</TD><TD>to 20 µl</TD></TR></TBODY></TABLE></DIV></DIV><H1 id="id">PCR protocol</H1><DIV class="col-lg-12 col-md-12 col-xs-12 content" style="padding-top: 20px; padding-bottom: 20px"><DIV class="table-slot" style="overflow-x: auto"><TABLE><THEAD><TR><TH>Step</TH><TH>Temperature [°C]</TH><TH>Time [s]</TH></TR></THEAD><TBODY><TR><TD>1</TD><TD>98</TD><TD>240</TD></TR><TR><TD>2</TD><TD>98</TD><TD>10</TD></TR><TR><TD>3</TD><TD>xy</TD><TD>60</TD></TR><TR><TD>4</TD><TD>72</TD><TD>60</TD></TR><TR><TD>5</TD><TD>72</TD><TD>120</TD></TR><TR><TD>repeat step 2 - 4 30x</TD><TD> </TD></TR></TBODY></TABLE></DIV></DIV></DIV></DIV></DIV><DIV class="panel panel-default"><A data-toggle="collapse" data-parent="#accordion" href="#collapsetable_id-16001"><DIV class="panel-heading">
            Plates
        </DIV></A><DIV id="collapsetable_id-16001" class="panel-collapse collapse"><DIV class="panel-body"><OL class="content"><LI>For 1 l of 1.5 % agar resolve 15 g Agar-Agar (Roth) and 25 g LB-powder (Roth) in 1 l dH2O, and autoclave it.
</LI><LI>Wait until the agar cools down to ~ 50 °C, and then add the respective antibiotic (see table below).
</LI><LI>Pour 8.5 cm plates with 15-18 ml of agar.
</LI><LI>Store the plates at 4 °C.
</LI><LI>Streak the bacteria out, and incubate at 37 °C for 12-16 h.
</LI></OL>
Antibiotic-concentrations:

<DIV class="col-lg-12 col-md-12 col-xs-12 content" style="padding-top: 20px; padding-bottom: 20px"><DIV class="table-slot" style="overflow-x: auto"><TABLE class="table table-bordered mdl-shadow--4dp" xsscleaned="overflow-x: scroll !important"><THEAD><TR><TH>Antibiotic</TH><TH>Stock Concentration</TH><TH>Final Concentration</TH></TR></THEAD><TBODY><TR><TD>Ampicillin</TD><TD>100 mg/ml</TD><TD>100 µg/ml</TD></TR><TR><TD>Streptomycin</TD><TD>100 mg/ml</TD><TD>100 µg/ml</TD></TR><TR><TD>Kanamycin</TD><TD>50 mg/ml</TD><TD>50 µg/ml</TD></TR><TR><TD>Chloramphenicol</TD><TD>25 mg/ml</TD><TD>25 µg/ml</TD></TR><TR><TD>Spectinomycin</TD><TD>50 mg/ml</TD><TD>50 µg/ml</TD></TR><TR><TD>Carbenicillin</TD><TD>100 mg/ml</TD><TD>100 µg/ml</TD></TR><TR><TD>Tetracyclin</TD><TD>10 mg/ml</TD><TD>10 µg/ml</TD></TR></TBODY></TABLE></DIV></DIV></DIV></DIV></DIV><DIV class="panel panel-default"><A data-toggle="collapse" data-parent="#accordion" href="#collapsetable_id-17"><DIV class="panel-heading">
            Polymerase Chain Assembly
        </DIV></A><DIV id="collapsetable_id-17" class="panel-collapse collapse"><DIV class="panel-body">
Procedure:
<OL class="content"><LI>Mix the DNA fragments in equimolar ratio. The total amount of DNA should be between 10 ng and 15 ng. Add 10 µl of 2X Phusion Flash Master Mix (Thermo Fisher Scientific), and adjust to 20 µl with dH2O.
</LI><LI>Run the following program:
</LI></OL><DIV class="col-lg-12 col-md-12 col-xs-12 content" style="padding-top: 20px; padding-bottom: 20px"><DIV class="table-slot" style="overflow-x: auto"><TABLE class="table table-bordered mdl-shadow--4dp" xsscleaned="overflow-x: scroll !important"><THEAD><TR><TH>Step</TH><TH>Temperature</TH><TH>Time</TH></TR></THEAD><TBODY><TR><TD>1</TD><TD>98 °C</TD><TD>10 s</TD></TR><TR><TD>2</TD><TD>98 °C</TD><TD>1 s</TD></TR><TR><TD>3</TD><TD>55 °C</TD><TD>5 s</TD></TR><TR><TD>4</TD><TD>72 °C</TD><TD>15-30 s/kb</TD></TR><TR><TD>5</TD><TD>72 °C</TD><TD>5 min</TD></TR><TR><TD>Repeat step 2-4 10x</TD></TR></TBODY></TABLE></DIV></DIV>Add 5 µl of the reaction to 25 µl Phusion Flash Master mix. Add primers to a final concentration of 0.5 µM, and add dH2O to a final volume of 50 µl.
Cycle again using the following conditions:


<DIV class="col-lg-12 col-md-12 col-xs-12 content" style="padding-top: 20px; padding-bottom: 20px"><DIV class="table-slot" style="overflow-x: auto"><TABLE class="table table-bordered mdl-shadow--4dp" xsscleaned="overflow-x: scroll !important"><THEAD><TR><TH>Step</TH><TH>Temperature</TH><TH>Time</TH></TR></THEAD><TBODY><TR><TD>1</TD><TD>98 °C</TD><TD>10 s</TD></TR><TR><TD>2</TD><TD>98 °C</TD><TD>1 s</TD></TR><TR><TD>3</TD><TD>Annealing Tm °C</TD><TD>5 s</TD></TR><TR><TD>4</TD><TD>72 °C</TD><TD>15-30 s/kb</TD></TR><TR><TD>5</TD><TD>72 °C</TD><TD>5 min</TD></TR><TR><TD>Repeat step 2-4 20-30x</TD></TR></TBODY></TABLE></DIV></DIV></DIV></DIV></DIV><DIV class="panel panel-default"><A data-toggle="collapse" data-parent="#accordion" href="#collapsetable_id-18"><DIV class="panel-heading">
            PREDCEL Protocol
        </DIV></A><DIV id="collapsetable_id-18" class="panel-collapse collapse"><DIV class="panel-body"><H2 id="1-introduction">1. Introduction</H2>
Phage Related DisContinuous EvoLution (PREDCEL) is a batch method to perform directed evolution adapted from PACE standing for Phage Assisted Continuous Evolution.
<H2 id="2-material">2. Material</H2><UL class="content"><LI>Respective bacterial strains, usually <EM>E. coli</EM>, with Mutation Plasmids (MP), Accessory Plasmids (AP), and if necessary further plasmids, like Complementary Plasmid (CP)</LI><LI>Phages carrying the Selection Plasmid (SP)</LI><LI>2YT medium</LI><LI>Glucose stock solution (final concentration: 100 mM)</LI><LI>Arabinose stock solution (final concentration: 100 mM)</LI><LI>Phage buffer solution (10 ml 1 M Tris; pH 7.5; 10 ml 1 M MgSO4;4 g NaCl; 980 ml ddH2O)</LI><LI>Flasks, all common sizes</LI><LI>Centrifugation tubes (2 ml and 50 ml)</LI><LI>Table top centrifuge (minimum 6000 g are required)</LI><LI>Centrifuge (minimum 3750 g are required)</LI><LI>Incubator</LI><LI>Freezer, -80 °C</LI><LI>Fridge, +4 °C</LI></UL><H2 id="3-procedure">3. Procedure</H2>
As it is important to prevent phage contaminations at any time, all of the following PREDCEL steps has to be performed with filter tips, and benches must be cleaned with 10% H202 (pay attention with handling!) or another suitable solution for phage inactivation. If practicable, use UV light on benches and in incubators to sterilize after and before your experiments.
<STRONG>Step 1: Prove MP and AP to work</STRONG>After successfully transforming your AP and MP plasmids into your bacterial strain (see transformation protocols), but before starting your PREDCEL run, make sure whether your AP and MP work is essential for the success of the intended directed evolution process or not. If yes, follow the instructions of <EM>AP testing</EM> and <EM>MP testing</EM>.
<STRONG>Step 2: F+ strains</STRONG>Make sure the used strain carries the F-Pilus plasmid as it is required for proper M13 phage infection. Thus, use a bacterial culture showing a positive result in <EM>Step 1</EM>, and plate it on a tetracycline (tet) plate, as your strain should carry a tetracycline resistance on its F-Pilus.
<STRONG>Step 3: Growing culture</STRONG>Pick a clone from <EM>Step 2</EM>, and prepare a bacterial culture in a suitable flask filled with 20 ml or more of 2YT medium with 100 mM glucose. Glucose must be added as it prevents MP activation, and thereby keeps cells intact. Incubate the culture until an OD600 of 0.6-0.8 is reached. At this time bacterial cells are in exponential stage and can be safely infected by phages.
<STRONG>Step 4: Glycerol stocks</STRONG>Take 5 ml from <EM>step 3</EM> to prepare glycerol stocks (see respective protocol; a main stock of 1 ml and several aliquots of 100 µl), and store them at -80 °C.
<STRONG>Step 5: Centrifugation and MP activation</STRONG>Centrifuge 10 ml or more of culture for 10 min at 3750 g to pellet your bacterial cells. The cultures result either from <EM>step 3</EM> or while PREDCEL performance from glycerol stock aliquot culture with an OD600 of 0.6-0.8, that needs to be prepared early enough to minimize time between gain of phage supernatant and new infection round. If culture reaches the respective OD600 too fast, dilute the culture until you can infect with phages or cool on ice if it takes less than 10 minutes until infection. Afterwards, resolve your cell pellet in a 150 ml flask with a volume of 2YT equal to the volume spun down containing 100 mM arabinose to induce MP.
<STRONG>Step 6: Phage infection</STRONG>Infect with a suitable amount of phages. As titers of inoculation phage can vary a lot a MOI of 1 is recommended for PREDCEL rather than giving a specific inoculation volume. MOI of 1 means that there is one phage per cell in solution. In contrast, while performing PREDCEL, define a certain phage supernatant volume you kept at 4°C to be transferred, for instance 1 ml, as phage titer cannot be determined fast enough by plaque assays to calculate the right MOI. Make sure you have at least 1 ml of phage supernatant left to analyze. Transfer the same amount of phages used for infection into a phage buffer solution, especially if titer of inoculation phage is unknown. Further, keep 1ml of each culture before infection. They can be used later to calculate propagation efficiency and check contamination.Grow the cultures for 1 to 24 hours at optimal growth conditions (for <EM>E. coli</EM> choose 37°C and about 220 rpm).
<STRONG>Step 7: Phage supernatant</STRONG>After incubation, spin down at least 2 ml of cells at 6000 g for 3 min to pellet your cells. Phages in the supernatant are now separated from the bacteria in the pellet. Store phage supernatant at 4°C. Discard the rest of the culture.
From now on repeat steps 5 to 7. After one round is completed, you can pause the PREDCEL process and restart later on with the stored supernatant.
While PREDCELing, detect and prevent phage washout or phage contamination in your used stocks, media and bacteria cultures. Therefore, perform test PCRs with specific length for your SP.If contamination or wash out is recognized, take the last sample proven to contain phages and free of contamination to start a new PREDCEL iteration round with a fresh culture.
<STRONG>Step 8: Plaque assay</STRONG>Perform plaque assays of your PREDCEL samples including inoculation phages and culture samples from <EM>step 6</EM> (see <EM>plaque assay</EM> protocol) to calculate phage titers, and to check for washout.
<STRONG>Step 9: Check for mutations</STRONG>Pick several plaques to use them for insert amplification via PCR. Sequence your insert with sequencing primers. Mention that mutations taking place while PREDCEL, it may lead to an inefficient primer annealing. Therefore, order several primers and try to find binding regions in conserved parts of your gene that are unlikely to mutate.
<STRONG>Troubleshooting</STRONG>:As phage propagation rate and mutation efficiency usually varies between different gene circuits and phages, but also due to the achieved changes in activity within a PREDCEL run. Thus, you might have to adapt several parameters to achieve a lower selection pressure (if phages get lost over time), a higher selection pressure (only random mutations occur) or changes in mutation rate (too little/ many mutations). The following parameters can be varied:
<UL class="content"><LI>the amount of culture volume to be infected and the corresponding flask size</LI><LI>the time of incubation post infection</LI><LI>the amount of phage supernatant transferred</LI></UL>
To check for the right parameters, pre-experiments need to be performed for an optimized propagation efficiency.
You should also prepare several strains of your bacteria carrying APs with different RBS-strengths, origins of replication, and different MPs (MP1/4/6 vary the mutation rate).
If you have problems with phage propagation due to no or too little starting activity, an initial drift phase without selection pressure might help.
Another helpful adaption is the usage of a helper-culture. This culture should be used between every or several PREDCEL AP-iteration rounds and must provide geneIII expression, that is only coupled to phage infection. Thereby, all phages that are transferred can reproduce themselves to gain a high phage titer, that might be needed to prevent phage washout. As previous phages showing higher activity propagated faster than those showing no or low activity, they will, as a consequence, achieve higher phage titers while this helper-culture-phase. Thus, helper-culture phase should not extinguish previously achieved selection for beneficial mutations.
</DIV></DIV></DIV><DIV class="panel panel-default"><A data-toggle="collapse" data-parent="#accordion" href="#collapsetable_id-19"><DIV class="panel-heading">
            Recovery Filter Paper
        </DIV></A><DIV id="collapsetable_id-19" class="panel-collapse collapse"><DIV class="panel-body"><OL class="content"><LI>Cut out the spot, where the DNA should be, and transfer it into a 1.5 ml reaction tube.
</LI><LI>Add 30 µl of dH2O, and mix by pipetting. Incubate for 10 minutes.
</LI><LI>Use the DNA solution for fransformation.
</LI></OL></DIV></DIV></DIV><DIV class="panel panel-default"><A data-toggle="collapse" data-parent="#accordion" href="#collapsetable_id-20"><DIV class="panel-heading">
            Restriction Digestion
        </DIV></A><DIV id="collapsetable_id-20" class="panel-collapse collapse"><DIV class="panel-body"><OL class="content"><LI>Mix the reaction components as listed below:
</LI><LI>Incubate for 1 h at the respective temperature (37 °C for the majority of enzymes).
</LI><LI>Heat-inactivate at 85 °C for 20 min if necessary
</LI><LI>Separate the DNA fragments via gelelectrophoresis or use them for subsequent ligation.
</LI></OL><DIV class="col-lg-12 col-md-12 col-xs-12 content" style="padding-top: 20px; padding-bottom: 20px"><DIV class="table-slot" style="overflow-x: auto"><TABLE class="table table-bordered mdl-shadow--4dp" xsscleaned="overflow-x: scroll !important"><THEAD><TR><TH>Component</TH><TH>Volume</TH></TR></THEAD><TBODY><TR><TD>10X reaction buffer (usually CutSmart Buffer (NEB))</TD><TD>5 µl</TD></TR><TR><TD>Restriction enzyme, each (NEB)</TD><TD>1 µl</TD></TR><TR><TD>plasmid DNA</TD><TD>1- 10 µg</TD></TR><TR><TD>dH2O</TD><TD>adjust to 50 µl</TD></TR></TBODY></TABLE></DIV></DIV></DIV></DIV></DIV><DIV class="panel panel-default"><A data-toggle="collapse" data-parent="#accordion" href="#collapsetable_id-100"><DIV class="panel-heading">
            Chemicals and Substrate
        </DIV></A><DIV id="collapsetable_id-100" class="panel-collapse collapse"><DIV class="panel-body">Chemicals used were either purchased from Sigma Aldrich Chemie (Steinheim, Germany), VWR International GmbH (Darmstadt, Germany) or synthesized by Arch Bioscience Company (West Chester, Pennsylvania, USA). Unless otherwise mentioned all chemicals were purchased in the highest quality and purity. The synthesis of the 4-(dimethylsilyl) aniline (1) compound was obtained from the working group of Prof. Greb (Heidelberg, Germany). The following chemicals were used during the synthesis of novel organosilicons. Dimethylphenylsilane 2, Ethyl 2-diazopropanoate 3, Sodium sulfate 4, Hexane 5, Ethyl acetate 6, 5-aminolevulinic acid 7 and Isopropyl β-D-1-thiogalactopyranoside 8.

<DIV class="col-lg-12 col-md-12 col-xs-12 content" style="padding-top: 20px; padding-bottom: 20px"><DIV class="table-slot" style="overflow-x: auto"><TABLE class="table table-bordered mdl-shadow--4dp" xsscleaned="overflow-x: scroll !important"><THEAD><TR><TH>Species</TH><TH>Strain</TH><TH>Genotype</TH><TH>Origin</TH></TR></THEAD><TBODY><TR><TD>E.coli</TD><TD>BL21 (DE3)</TD><TD>F-ompT hsdSB   (rB- mB-) gal dcm (DE3)</TD><TD>Novagen</TD></TR><TR><TD>Vector/Plasmid</TD><TD>Properties</TD><TD>Reistance</TD><TD>Origin</TD></TR><TR><TD>pET22</TD><TD>cloning vector</TD><TD>Ampicillin</TD><TD>S.B.J.Kan,   R.D.Lewis, K.Chen, F.H.Arnold, Science 354, 148 (2016)</TD></TR><TR><TD>pEC86</TD><TD>Cytochrome c   maturation plasmid</TD><TD>Chloramphenicol</TD><TD>Culture   Collection of Switzerland AG</TD></TR></TBODY></TABLE></DIV></DIV><DIV class="col-lg-12 col-md-12 col-xs-12 content" style="padding-top: 20px; padding-bottom: 20px"><DIV class="table-slot" style="overflow-x: auto"><TABLE class="table table-bordered mdl-shadow--4dp" xsscleaned="overflow-x: scroll !important"><THEAD><TR><TH>Medium/Buffer</TH><TH>Ingredients</TH></TR></THEAD><TBODY><TR><TD>LB</TD><TD>5 g x L<SUP>-1</SUP> peptone, 10 g x L<SUP>-1</SUP> NaCl, 10 g x L-1   yeast extract</TD></TR><TR><TD>M9-N</TD><TD>47.7 mM Na2HPO4, 22.0 mM KH2PO4, 8.6 mM NaCl, 2.0 mM MgSO4, and 0.1   mM CaCl2</TD></TR></TBODY></TABLE></DIV></DIV></DIV></DIV></DIV><DIV class="panel panel-default"><A data-toggle="collapse" data-parent="#accordion" href="#collapsetable_id-101"><DIV class="panel-heading">
            Cultivation of E.coli BL21 (DE3) containing pET22 and pEC8
        </DIV></A><DIV id="collapsetable_id-101" class="panel-collapse collapse"><DIV class="panel-body">One colony from streaked out transformation and incubation over night at 37°C was picked and used for inoculation of a preculture. Main culture was inoculated with the preculture by adding 10 ml to 490 ml of LB medium. The culture was shaken at 37 °C and 200 rpm until the OD600 was 0.7 (approximately 3 hours). The culture was placed on ice for 30 minutes, and isopropyl β-D-1-thiogalactopyranoside (IPTG) and 5-aminolevulinic acid (ALA) were added to final concentrations of 20 μM and 200 μM respectively. The incubator temperature was reduced to 20 °C, and the culture was allowed to shake for 20 hours at 200 rpm.
</DIV></DIV></DIV><DIV class="panel panel-default"><A data-toggle="collapse" data-parent="#accordion" href="#collapsetable_id-102"><DIV class="panel-heading">
            Experimental Procedure - Silyl-Aniline Synthesis
        </DIV></A><DIV id="collapsetable_id-102" class="panel-collapse collapse"><DIV class="panel-body">All steps were carried out under an Argon atmosphere, with flame-dried glassware and with dried and degassed solvents. 
p-Bromoaniline (1 eq.), triethylamine (2 eq.) and 4-(dimethylamino)-pyridine (0.1 eq.) were stirred in dichlormethane for 30 min. 1,2-Bis(chlorodimethylsilyl)ethane (1.3 eq) were added within 10 min. The mixture was stirred for 3 h at 40 °C. The solvent was removed under reduced pressure an the residue was extracted three times with ether. The ether extract was filtered over dried Celite and the solvent was changed to tetrahydrofurane. The solution was cooled to -78 °C and tert-butyllithium (1.5 eq.) was added within 15 min. After stirring for 2 h at -78 °C chlorodimethylsilane (1.5 eq.) was added and the reaction mixture was warmed to room temperature over night. Ether was added and the mixture was filtered over a mixture of Celite and Alox (1:1). The solvent was removed under reduced pressure. The crude product was purified via silica column chromatography (EE:PE 1:7) after resting on the silica column for 20 min to deprotect the amino group. The pure product was obtained. 
</DIV></DIV></DIV><DIV class="panel panel-default"><A data-toggle="collapse" data-parent="#accordion" href="#collapsetable_id-103"><DIV class="panel-heading">
            Isolation and purification of synthesized Product
        </DIV></A><DIV id="collapsetable_id-103" class="panel-collapse collapse"><DIV class="panel-body">After shaking for 20 hours, the reaction is split in four 2 ml tubes and mixed with 600 µl of ethyl acetate. Vortexing for 30 seconds and centrifugation at 14000 rpm for 7 min leads to phase separation of aqueous and organic layers. The organic layer is taken off and placed in a new glass vial. Two additional washing and separation steps were conducted and organic layer is always taken off. The combined organic solution is then purified via silica column chromatography and analyzed via gas chromatography – mass spectroscopy.
</DIV></DIV></DIV><DIV class="panel panel-default"><A data-toggle="collapse" data-parent="#accordion" href="#collapsetable_id-104"><DIV class="panel-heading">
            Preparation of whole-cell Biocatalyst
        </DIV></A><DIV id="collapsetable_id-104" class="panel-collapse collapse"><DIV class="panel-body">Main culture was centrifuged at 4000 rpm and pellet was resuspended in M9-N buffer to a final OD600 of 15. Aliquots were prepared and frozen until further use.
</DIV></DIV></DIV><DIV class="panel panel-default"><A data-toggle="collapse" data-parent="#accordion" href="#collapsetable_id-105"><DIV class="panel-heading">
            Transformation of E.coli BL21 (DE3) with-pET22 and pEC8
        </DIV></A><DIV id="collapsetable_id-105" class="panel-collapse collapse"><DIV class="panel-body">Electro competent E.coli BL21 (DE3) were thawed on ice for 30 min and incubated with 1 ng of each plasmid for 30 min. Cells were electroporated with the Gene Pulser Xcell™ Electroporation Systems at 1800 volts for bacterial electroporation. Cells were incubated in 1 ml Soc for recovery at 600 rpm and 1 hour. Recovered cells were plated on LB agar plates containing Ampicillin and Chloramphenicol in a final concentration of 100 mg x ml<SUP>-1</SUP> and 20 mg x ml<SUP>-1</SUP> respectively.
</DIV></DIV></DIV><DIV class="panel panel-default"><A data-toggle="collapse" data-parent="#accordion" href="#collapsetable_id-106"><DIV class="panel-heading">
            Whole cell biocatalysis reactio
        </DIV></A><DIV id="collapsetable_id-106" class="panel-collapse collapse"><DIV class="panel-body">One aliquot of whole cells was placed in a sealed glass vial which is free of air and filled with argon. The solution is perfused with argon for approximately 10 min to be sure that no nitrogen is present. Afterwards, 0.01 mM of compound 3 is added followed by 0.1 mM of compound 1 or 2 respectively. Additionally, glucose is added to a final concentration of 10 mM in M9-N buffer. Reaction is shaken for 4 hours and two additional batches were added. After adding of the batches, the reaction is shaken at 480 rpm for a total of 20 hours.
</DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV>

}}                          
</DIV><SECTION id="footer-sec" style="background-color: #222 !important"><DIV class="container"><DIV class="row" style="padding-top: 30px;"><DIV class="col-md-3"><H4 style="text-align: center">Quote</H4><P><I>“”</I></P></DIV><DIV class="col-md-3"><H4 style="text-align: center"> Useful <STRONG>Links</STRONG></H4><P><A href="https://igem.org/Special:SpecialPages">
			Special pages
		</A></P><P><A href="https://igem.org/Main_Page">
			Main Page
		</A></P></DIV><DIV class="col-md-3"><H4 style="text-align: center"> Follow us on </H4><DIV style="text-align: center;"><SPAN style="color: grey; text-align: center;">&amp;</SPAN></DIV></DIV><DIV class="col-md-3"><H4 style="text-align: center"> Contact us </H4><P>

               iGEM-Heidelberg2017@bioquant.uni-heidelberg.de
               
		 Im Neuenheimer Feld 267
		
		69120 Heidelberg
                    </P></DIV></DIV></DIV></SECTION></DIV></DIV></DIV></DIV></BODY></HTML>