<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_HokkaidoU_Japan_Projects"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:HokkaidoU_Japan/Projects">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:HokkaidoU_Japan/Projects&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:HokkaidoU_Japan/Projects&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:HokkaidoU_Japan/Projects&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:HokkaidoU_Japan/Projects" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:HokkaidoU Japan/Projects</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><DIV id="menu"><DIV class="menu_title"><P>MENU
</P></DIV><DIV class="menu_inner"><UL><LI><A href="/Team:HokkaidoU_Japan" title="Team:HokkaidoU Japan">Home</A></LI><LI><STRONG class="selflink">Projects</STRONG></LI><LI><A href="/Team:HokkaidoU_Japan/Team" title="Team:HokkaidoU Japan/Team">Team</A></LI><LI><A href="/Team:HokkaidoU_Japan/Notebook" title="Team:HokkaidoU Japan/Notebook">Notebook</A></LI><LI><A href="/Team:HokkaidoU_Japan/Protocols" title="Team:HokkaidoU Japan/Protocols">Protocols</A></LI><LI><A href="/Team:HokkaidoU_Japan/Parts" title="Team:HokkaidoU Japan/Parts">Parts</A></LI><LI><A href="/Team:HokkaidoU_Japan/Safety" title="Team:HokkaidoU Japan/Safety">Safety</A></LI></UL></DIV><DIV class="menu_title"><P>Sponsors
</P></DIV><DIV class="menu_title"><P>Visitors
</P></DIV><DIV class="menu_inner"><P>
since 13 Jul 2010
since 1 Aug 2010

</P></DIV></DIV><DIV id="main"><P><STRONG class="selflink">English</STRONG>
/
<A href="/Team:HokkaidoU_Japan/Projects_jp" title="Team:HokkaidoU Japan/Projects jp">日本語</A></P><H1><SPAN class="mw-headline" id="Dr._E._coli:_The_smallest_injector_in_the_world">Dr. <I>E. coli</I>: The smallest injector in the world</SPAN></H1><H2><SPAN class="mw-headline" id="Introduction">Introduction</SPAN></H2><P>   Type III Secretion System (T3SS) is a system of pathogenic gram-negative bacterium such as <I>Salmonella</I>, <I>Yersinia</I> and EPEC(entero pathogenic <I>E. coli</I>). Using this system bacteria can inject whole protein molecules through a syringe like organelle named Type 3 Secretion Apparatus. The target of this system is a eukaryotic cell. Naturally it is used to inject Virulence effector proteins.
</P><H2><SPAN class="mw-headline" id="Structure_of_Type_III_secretion_apparatus">Structure of Type III secretion apparatus</SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Fig.1Needle complex of <I>Salmonella typhimurium</I><SUP><A href="#References">[2]</A></SUP>.</DIV></DIV></DIV><DIV class="narrow"><P>   Type III secretion apparatus have a syringe like structure. It can be visible under electron microscope <SUP><A href="#References">[2]</A></SUP>. Its length is about 80nm and the diameter of its needle channel is about 2nm <SUP><A href="#References">[3]</A></SUP>. The length is about 1/10 and the diameter is about 1/400 of an <I>E. coli</I> cell’s minor axis. Thus this is the smallest injector in the world (Fig.1 a~c).
</P></DIV><H2><SPAN class="mw-headline" id="How_does_it_function.3F">How does it function?</SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Fig.2Model for substrate recognition and delivery of proteins by type III secretion machines <SUP><A href="#References">[2]</A></SUP>.</DIV></DIV></DIV><DIV class="narrow"><P>   When the needle tip attaches to the host cell membrane, a translocator complex that is also secreted by the T3SS is assembled on the host cell membrane and mediates the passage of the effector proteins through the target cell membrane. On the other hand an effector protein, which have a unique T3SS secretion signal domain on its N-terminal, is recognized by the specific chaperone and form an effector-chaperone complex. The secretion machinery, including a T3-secretion-associated ATPase, recognizes the complex. Then, the ATPase stripes the chaperone from the complex, which remains within the bacterial cell, and mediates the unfolding and threading of the effector protein through the central channel of the needle complex. Finally, the translocated effectors re-fold within the host cell to carry out their function (Fig.2).
</P></DIV><H2><SPAN class="mw-headline" id="Motivation">Motivation</SPAN></H2><P>   It is valuable to develop a system that can modulate cell's behavior transiently by injecting a desired &quot;protein&quot; directly into cells using a non-pathogenic strain. This system can be applied for many purposes. For example, 
</P><UL><LI> Inject p53 to terminate cell division of cancer cells selectively
</LI><LI> Inject factors required for iPS cells induction
</LI></UL><P>   It has been reported in 2007 that T3SS encoded in Salmonella Pathogenesity Island 2(SPI2) is functional in vitro on the <I>E. coli</I> (K-12) strain <SUP><A href="#References">[9]</A></SUP>. However, the T3SS encoded in SPI-2 naturally function inside of the phagosome of the target cell <SUP><A href="#References">[8]</A></SUP>. So, there was no report about whether the SPI-2 T3SS, that is cloned on <I>E. coli</I> (K-12) can inject a heterologous protein from outside of the target cell or not. That is why we have decided to put this challenging project into practice.
</P><H2><SPAN class="mw-headline" id="Objectives_of_iGEM_2010">Objectives of iGEM 2010</SPAN></H2><P>   Though it is so exciting to create a cancer killer or an iPS generator <I>E. coli</I>, it is too difficult to aim these practical goals from the very first; because it was reported that some domains are not able to pass the needle of T3SS <SUP><A href="#References">[1]</A><A href="#References">[3]</A></SUP>. So, we have established four objectives of iGEM 2010 as following.
</P><UL><LI> To make T3SS available for <I>E. coli</I></LI><LI> To inject desired proteins into target eukaryotic cells
</LI><LI> To determine whether the injected protein function correctly
</LI><LI> To deliver the injected protein toward desired compartment in the target cell
</LI></UL><H2><SPAN class="mw-headline" id="Detail_information_about_SPI-2_region_and_SPI-2_carrying_E._coli_strain">Detail information about SPI-2 region and SPI-2 carrying <I>E. coli</I> strain</SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">Fig.3<I>S. enterica</I> serovar Typhimurium SPI-2 region. A map of the <I>S. enterica</I> serovar Typhimurium SPI-2 region is depicted <SUP><A href="#References">[9]</A></SUP>.</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">Fig.4To confirm the presence of SPI-2 region in the strain send from SGSC  colony-PCR analysis was performed using a primer set which targets a 481 bp region inside of SPI2. [M: λHindIII marker, C: negative control colony of E.coli(DH5α), S: sample colony of E.coli send from SGSC]</DIV></DIV></DIV><DIV class="narrow"><P>   Here we show the map of SPI-2 region (Fig.3). The length of SPI-2 region about 40kb and T3 secretion apparatus is a complex consisted of more than 20 kinds of proteins. As mentioned before cloned SPI-2 on R995 vector was functional in <I>E. coli</I> (K-12) <SUP><A href="#References">[9]</A></SUP>. And also it was reported that SPI-2 region cloned on pBeloBAC11 (BAC=Bacteria Artificial Chromosome) is functional <SUP><A href="#References">[4]</A></SUP>. So, we decided to request a <I>E. coli</I> (K-12) strain carrying a pBeloBAC11 vector encoding a genome fragment of <I>Salmonella enterica</I> serovar Typhimurium LT2 which covers the SPI-2 region [B_STM07H21  SGSC4024  1464540～1562427] from Salmonella Genetic Stock Center(SGSC) in University of Calgary, Canada (<A href="http://people.ucalgary.ca/~kesander/index.html" class="external free" rel="nofollow">http://people.ucalgary.ca/~kesander/index.html</A>). We confirmed the presence of the cloned SPI-2 region in this strain via PCR analysis(Fig.4) shall refer to it as &quot;pBAC-SPI-2&quot; from now on. Because of the provision of MTA, we cannot register this strain as a Bio Brick, but you can also get the same strain from SGSC.
</P></DIV><H2><SPAN class="mw-headline" id="Construction_of_E._coli_base_GFP_injector">Construction of <I>E. coli</I>base GFP injector</SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">Fig.5Conceptional figure of the project</DIV></DIV></DIV><DIV class="narrow"><P>   We have made a T3SS test construct on the tetracycline resistant plasmid [pSB1T3] and transform it to the <I>E. coli</I> (K-12) carrying pBAC-SPI-2 as shown in (Fig.5). It was reported that N-terminus 191 amino acid residues of SlrP (one of the natural effector protein of SPI-2 encoded T3SS) function as a T3 secretion signal domain in <I>Salmonella</I><SUP><A href="#References">[6]</A></SUP>. And it was also reported that GFP can be secreted through T3SS of <I>Yersinia</I><SUP><A href="#References">[5]</A></SUP>. So, we fused the N-terminus 191 a.a. of SlrP with the N-terminal of GFP. In addition we located triple NLS (Nuclear Localization Signal) repeats between the T3 secretion signal and GFP so that the injected GFP would localize inside of the nucleus in the target cell. This [T3signal-NLS-GFP] fusion gene is under control of inducible arabinose promoter. In contrast RFP reporter gene, which is joined to the down stream of the GFP construct, is under control of a constitutive promoter. So, if arabinose is added to the culture medium, <I>E. coli</I> produces both GFP and RFP so that the bacteria become yellow. However, if arabinose is not added, only RFP is produced so that the bacteria become red. We have registered the T3SS signal domain mentioned above as a new BioBrick part.
</P></DIV><H2><SPAN class="mw-headline" id="Bacteria_and_cell_culture_condition">Bacteria and cell culture condition</SPAN></H2><P>   To perform the injection assay, we used LB medium (1.0% Bacto-Tryptone,  0.5% Bacto-yeast extract, 1.0% NaCl) and magnesium minimal medium (MgM) <SUP><A href="#References">[11]</A></SUP> containing 170 mM MES-NaOH buffer(pH=5.0 or 7.2), 7.5 mM (NH<SUB>4</SUB>)2SO<SUB>4</SUB>, 5 mM KCl, 1 mM KH<SUB>2</SUB>PO<SUB>4</SUB>, 8 uM MgCl<SUB>2</SUB>, 38 mM glycerol and 0.1% casamino acids. We named these medium as MgM5(pH=5.0) and MgM7(pH=7.2). Also we used the acidic cell culture medium RPMI-10% FCS + HCl (pH 5.0) [RPMI5] and normal RPMI-10% FCS [RPMI7]. Bacteria were cultured at 37C with aeration and RK13 cells were cultured at 37C in 5.0% CO<SUB>2</SUB>. Appropriate antibiotics were added according to the resistance marker on each plasmid (25 ug/mL of chloramphenicol and 15 ug/mL of tetracycline). To induce GFP fusion protein L-arabinose was added to the medium at each step (final concentration = 0.4% ).
</P><H2><SPAN class="mw-headline" id="Methods_of_Injection_Assay">Methods of Injection Assay</SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">Fig.6Model for control of effector translocation by the SPI-2 T3SS. (i) Following uptake into host cells, acidification of the vacuole lumen induces assembly of 9 the  secretion apparatus.  (ii) Membrane-associated SsaL/SsaM/SsaB (Fig.3)  regulatory complex  (in purple, black  and blue,  respectively) prevents premature  secretion of  effectors  (in brown). Translocon proteins (in green), connected to the T3SS apparatus, form a pore in the vacuolar membrane.  (iii) The pore enables a component(s) of  the T3SS  to sense  the elevated pH of the  host  cell  cytosol,  and  a  signal  is  transduced  to  the  SsaL/SsaM/SsaB  complex, which dissociates. (iv) Relief of effector secretion suppression enables their translocation <SUP><A href="#References">[10]</A></SUP>.</DIV></DIV></DIV><DIV class="narrow"><P>   As mentioned before the T3SS encoded in SPI-2 naturally function inside of the phagosome of the target cell <SUP><A href="#References">[8]</A></SUP>. So, it requires acidic pH to be assembled functionally <SUP><A href="#References">[7]</A></SUP>. However, if the T3 apparatus is assembled successfully under low pH(pH=5.0) condition, only the translocator proteins are secreted through T3SS but effector proteins are not. And it was reported in 2010 that the translocator complex assembles a pore on the phagosome membrane of the host cell enabling the T3SS to sense the neutral pH condition of the cytosol, and this pH elevation switches the function of the T3SS to start secretion of effectors (Fig.6) <SUP><A href="#References">[10]</A></SUP>. In addition we found that initial growth in MgM7 before the growth in MgM5 improve the production of GFP fusion protein in <I>E. coli</I> (data not shown). So, 10 hrs before exposure we transferred <I>E. coli</I> [SPI-2+GFP-T3signal+RFP] overnight culture from LB + arabinose to MgM7 + arabinose and grow for 4 hrs to charge sufficient amount of GFP fusion protein. 5.5 hrs before exposure, bacteria were transferred to MgM5 + arabinose and grow for 4 hrs to assemble T3 secretion apparatus. 1 hr before exposure, bacteria were washed with RPMI5 three times to remove toxin secreted from <I>E. coli</I>. Then it was resuspended diluted with RPMI5 + arabinmose (final ΔOD = 0.06 at 600 nm). On the other hand RK13 cells were seeded on 6-well plate (2x 10&lt;sup5&lt;/sup&gt; cells/well) in antibiotics free RPMI7 at 20 hrs before exposure to the <I>E. coli</I>. When the preparation is completed cell culture medium was replaced with 1 mL of the <I>E. coli</I> suspension (ΔOD = 0.06 at 600 nm) and incubate at 37C in 5.0% CO<SUB>2</SUB> to mimic the environment inside of the phagosome. At the same time samples of  arabinose(-) <I>E. coli</I>[SPI-2+GFP-T3signal+RFP] and <I>E. coli</I>[SPI-2 only] were prepared for the control condition.
</P></DIV><H2><SPAN class="mw-headline" id="Results_.287.5h_after_infection.29">Results (7.5h after infection)</SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption">Fig.7[A-1]~[A-4] are the images of [S+G+R]. [B-1]~[B-4] are the images of [S+R]. [C-1]~[C-4] are the images of [S]. [X-1] is the phase contrast image. [X-2] is the green fluorescence image excited by blue laser. [X-3] is the red fluorescent image excited by green laser. And [X-4] is the merged image of [X-1]~[X-3].</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption">Fig.8wider angle merged images (A~C) are presented (A: [S+G+R], B: [S+R], C: [S]) and in (A’) only yellow color, which represents the <I>E. coli</I> cells, is deleted from (A).</DIV></DIV></DIV><P>   From now on we shall refer to &quot;[SPI-2+T3signal-GPP+RFP]0.4% arabinose(+)&quot; as [S+G+R],  &quot;[SPI-2+T3signal-GPP+RFP] arabinose(-)&quot; as [S+R], and &quot;[SPI-2 only]&quot; as [S].
We observed the cells by Cofocal Laser Scannning Microscope(OLYMPUS FV-1000D) under blue(473 nm) and green(559 nm) excitation light 7.5 hrs after the first exposure (Fig.7). [S+G+R] <I>E. coli</I> express both GFP and RFP, so in [A-4] <I>E. coli</I> cells appear to be yellow and GFP outside of <I>E. coli</I> appear to be green. Comparing [A-1] and[A-4] you can recognize that GFP is located in the cytosol of RK13. In contrast in [B-4] and [C-4] you cannot see GFP in the cytosol (Fig.8). In (A’) you can find that GFP is located in the shape of the cytosol more clearly. In fact GFP began to diffuse in the cytosol 4 hrs after the exposure (data not shown). However, GFP didn’t localize into the nucleus against our expectation even 7.5 hrs after exposure.
</P><H2><SPAN class="mw-headline" id="Discussion">Discussion</SPAN></H2><P>   According to the results it can be suggested that SPI-2 encoded T3SS can inject a heterologous protein(GFP) from outside of the target cell even if it is cloned onto the heterologous host <I>E. coli</I> (K-12) and that injected GFP was functional in the target cell. In the process of our project, even if the <I>E. coli</I> is prepared under acidic pH condition, GFP was not injected into RK13 when the exposure step is performed under neutral pH condition (data not shown), suggesting that acidic pH is required for exposure step to use this system. By the way GFP didn’t localize into the nucleus against our expectation. So, NLS domain might not be able to function correctly in the GFP fusion protein constructed this time.
</P><H2><SPAN class="mw-headline" id="Possible_Future_Plans">Possible Future Plans</SPAN></H2><UL><LI> Collect quantitative and chronological data.
</LI><LI> Clone the [T3signal-NLS-GFP] construct on a mammalian expressing vector to check the behavior of this fusion protein expressed inside of RK13.
</LI><LI> Join NLS with the C-terminal of GFP.
</LI></UL><H1><SPAN class="mw-headline" id="References">References</SPAN></H1><OL><LI> Chen LM, Briones G, Donis RO, Galán JE. 2006. Optimization of the delivery of heterologous proteins by the Salmonella enterica serovar Typhimurium type III secretion system for vaccine development. Infect Immun. Vol.74:5826-5833. <A href="http://www.ncbi.nlm.nih.gov/pubmed/16988261" class="external text" rel="nofollow">PubMed</A></LI><LI> Galán JE, Wolf-Watz H. 2006. Protein delivery into eukaryotic cells by type III secretion machines. Nature. Vol.444:567-573. Review. <A href="http://www.ncbi.nlm.nih.gov/pubmed/17136086" class="external text" rel="nofollow">PubMed</A></LI><LI> Ghosh P. 2004. Process of protein transport by the type III secretion system. Microbiol Mol Biol Rev. Vol.68:771-795. Review.<A href="http://www.ncbi.nlm.nih.gov/pubmed/15590783" class="external text" rel="nofollow">PubMed</A></LI><LI> Hansen-Wester I, Chakravortty D, Hensel M. 2004. Functional transfer of Salmonella pathogenicity island 2 to Salmonella bongori and Escherichia coli. Infect Immun. Vol.72:2879-2888. <A href="http://www.ncbi.nlm.nih.gov/pubmed/15102800" class="external text" rel="nofollow">PubMed</A></LI><LI> Jacobi CA, Roggenkamp A, Rakin A, Zumbihl R, Leitritz L, Heesemann J. 1998. In vitro and in vivo expression studies of yopE from Yersinia enterocolitica using the gfp reporter gene. Mol Microbiol. Vol.30:865-882. <A href="http://www.ncbi.nlm.nih.gov/pubmed/10094634" class="external text" rel="nofollow">PubMed</A></LI><LI> Miao EA, Miller SI. 2000. A conserved amino acid sequence directing intracellular type III secretion by Salmonella typhimurium. Proc Natl Acad Sci U S A. Vol.97:7539-7544. <A href="http://www.ncbi.nlm.nih.gov/pubmed/10861017" class="external text" rel="nofollow">PubMed</A></LI><LI> Rappl C, Deiwick J, Hensel M. 2003. Acidic pH is required for the functional assembly of the type III secretion system encoded by Salmonella pathogenicity island 2. FEMS Microbiol Lett. Vol.226:363-372. <A href="http://www.ncbi.nlm.nih.gov/pubmed/14553934" class="external text" rel="nofollow">PubMed</A></LI><LI> Waterman SR, Holden DW. 2003. Functions and effectors of the Salmonella pathogenicity island 2 type III secretion system. Cell Microbiol. Vol.5:501-511. Review. <A href="http://www.ncbi.nlm.nih.gov/pubmed/12864810" class="external text" rel="nofollow">PubMed</A></LI><LI> Wilson JW, Coleman C, Nickerson CA. 2007. Cloning and transfer of the Salmonella pathogenicity island 2 type III secretion system for studies of a range of gram-negative genera. Appl Environ Microbiol. Vol.73:5911-5918. <A href="http://www.ncbi.nlm.nih.gov/pubmed/17675443" class="external text" rel="nofollow">PubMed</A></LI><LI> Yu XJ, McGourty K, Liu M, Unsworth KE, Holden DW. 2010. pH sensing by intracellular Salmonella induces effector translocation. Science. Vol.328:1040-1043. <A href="http://www.ncbi.nlm.nih.gov/pubmed/20395475" class="external text" rel="nofollow">PubMed</A></LI><LI> Yu XJ, Liu M, Holden DW. 2004. SsaM and SpiC interact and regulate secretion of Salmonella pathogenicity island 2 type III secretion system effectors and translocators. Mol Microbiol. Vol.54:604-619. <A href="http://www.ncbi.nlm.nih.gov/pubmed/15491354" class="external text" rel="nofollow">PubMed</A></LI></OL><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:HokkaidoU_Japan/Projects">http://2010.igem.org/Team:HokkaidoU_Japan/Projects</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:HokkaidoU_Japan/Projects" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:HokkaidoU_Japan/Projects" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:HokkaidoU_Japan/Projects&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:HokkaidoU_Japan/Projects&amp;oldid=209730" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></BODY></HTML>