<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Tsinghua_Demonstrate skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Tsinghua/Demonstrate</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Neon Coli-Necessary Expression Only</TITLE></P><DIV class="container"><DIV class="collapse navbar-collapse" id="navbarResponsive"><UL id="menu" class="navbar-nav  ml-auto"><LI class="nav-item"><A class="nav-link text-uppercase js-scroll-trigger" href="https://2018.igem.org/Team:Tsinghua">Home</A></LI><LI class="nav-item"><A href="#Team" class="nav-link text-uppercase js-scroll-trigger">
                Team </A></LI><LI class=" nav-item dropdown"><A href="#" class="nav-link text-uppercase dropdown-toggle" data-toggle="dropdown">
                Project </A></LI><LI><A href="https://2018.igem.org/Team:Tsinghua/Description">Description</A></LI><LI><A href="https://2018.igem.org/Team:Tsinghua/Design">Design</A></LI><LI><A href="https://2018.igem.org/Team:Tsinghua/Experiments">Experiments</A></LI><LI><A href="https://2018.igem.org/Team:Tsinghua/Results">Results</A></LI><LI><A href="https://2018.igem.org/Team:Tsinghua/Model">Model</A></LI><LI><A href="https://2018.igem.org/Team:Tsinghua/Notebook">Notebook</A></LI><LI><A href="https://2018.igem.org/Team:Tsinghua/Attributions">Attributions</A></LI></UL><LI class=" nav-item dropdown"><A href="#" class="nav-link text-uppercase dropdown-toggle" data-toggle="dropdown">
                Parts </A></LI><LI><A href="https://2018.igem.org/Team:Tsinghua/Basic_Part">Basic Parts</A></LI><LI><A href="https://2018.igem.org/Team:Tsinghua/Composite_Part">Composite Parts</A></LI><LI><A href="https://2018.igem.org/Team:Tsinghua/Improve">Improve</A></LI><LI class=" nav-item dropdown"><A href="#" class="nav-link text-uppercase dropdown-toggle" data-toggle="dropdown">
                HP </A></LI><LI><A href="https://2018.igem.org/Team:Tsinghua/Human_Practices">Overview</A></LI><LI><A href="https://2018.igem.org/Team:Tsinghua/GHP">General HP</A></LI><LI><A href="https://2018.igem.org/Team:Tsinghua/IHP">Intergrated HP</A></LI><LI><A href="https://2018.igem.org/Team:Tsinghua/Safety">Safety</A></LI><LI class="nav-item"><A href="https://2018.igem.org/Team:Tsinghua/Collaborations" class="nav-link text-uppercase js-scroll-trigger">
                Collaboration</A></LI></DIV></DIV><SECTION id="HP"><DIV class="container"><DIV class="row"><DIV class="col-lg-12 text-center"><H2 class="section-heading text-uppercase">Demonstrate</H2></DIV></DIV><DIV class="row text-left"><DIV class="col-md-12 keith"><H4 class="service-heading">NEON system</H4><SPAN class="name-heading"> NEON system is designed to receive input, amplify through positive feedback then generate output sig-nals, while remaining stringent without stimulation. This is achieved with a double control design that in-volves two interlinked devices, lux positive feedback circuit and CRISPRi. In NEON test system, we built two plasmids, Neon the positive feedback plasmid <A href="http://parts.igem.org/part:BBa_K2558214">BBa_K2558214</A>, and Safety Catch the CRISPRi plasmid <A href="http//parts.igem.org/part:BBa_K2558215">BBa_2558215</A>,<A href="http://parts.igem.org/Part:BBa_K2558216">BBa_K2558216</A> and these two are linked by a Ptac/lacI system. 
	We named the positive feedback plasmid Neon because it is the core element in NEON system, and without Safety Catch inhibition it causes E.coli to produce massive amount of output sfGFP, shining brilliantly like a neon light. The constitutively expressed luxR on neon is able to bind with lux-AHL. LuxR/AHL complex activates lux pR that controls the expression of luxI and sfGFP. LuxI protein is the en-zyme that catalyses the synthesis of lux-AHL, which is going to bind with more luxR protein and further activates lux pR expression, forming a positive feedback loop. However lux pR promoter is not stringent and even the slightest leakage is able to put the positive  feedback loop in action. This is where the safety Catch plasmid comes in. It is designed to prevent undesired leakage expression, like the safety catch on a firearm, keeping Neon in check. Constitutively expressed dCas9 and Ptac controlled gRNA binds just downstream on the lux pR promoters on Neon. The binding interrupts the formation of bacterial initiation complex, silences luxI and sfGFP expression and prevents positive feedback circuit from activation. With lux-AHL stimulation, luxR/AHL complex activates the transcription of lacI on Safety Catch, which binds to Ptac before gRNA and down regulates gRNA transcription. Without gRNA, dCas9 will no longer interfere with lux pR promoters on Neon, and the positive feedback circuit is able to operate without hinder. However, IPTG can relieve lacI inhibition on gRNA by altering lacI conformation, thus silencing the positive feedback circuit on addition.
	The expression range of NEON system is considerably wider than non-positive feedback devices. Compared to other artificial positive feedback devices, NEON is more stable and stringent. By tuning pro-motor and RBS intensities it can be customized to have different stimulation thresholds and reaction inten-sity. NEON could be easily integrated to other projects that share the mutual quorum sensing device. And because lux pR is listed as one of the Top 10 Most Used promoters, we believe NEON could help numerous iGEM projects.<DIV class="text-center"><STRONG>Figure.1. Design of lux pR mutation analysis circuit.</STRONG></DIV></SPAN><H4 class="service-heading">Lux pR Mutation Analysis</H4><SPAN class="name-heading"> NEON as a complex double control system is quite delicate. With our model we have identified several key parameters that when changed are going to significantly alter the circuit’s behavior, one of which is lux pR. As lux pR controls the expression of luxI and sfGFP on Neon, and lacI on Safety Catch, it is the mutual component in both the positive feedback and CRISPRi systems. With simulations we learnt that increased lux pR transcription efficiency would increase NEON’s reaction to AHL, and decreased lux pR leak-age is going to improve NEON’s performance in conditions with lower concentration of AHL stimulation. Therefore we sought to use point mutations to improve the current lux pR and design a new promoter with higher expression intensity and lower leakage. 
	With the hope to find an optimal promoter for our NEON system, we designed 9 mutations on sites  -35, -36, -37 near the luxR binding site and tested another on -10 site that TUST 2017 reported to show decreased leakage. We conducted experiments to evaluate these lux pR mutants’ reaction to AHL stimulation and their leakage level. The test devices we designed include a constantly expressed luxR and a lux pR (or mutant) driven sfGFP (like <A href="http://parts.igem.org/Part:BBa_K2558211">BBa_K2558211</A> with original lux pR promoter, and a <A href="http://parts.igem.org/Part:BBa_K2558212">BBa_K2558212</A> with lux pR-HS promoter)We selected one mutation from the luxR binding site, combined it with the mutations on -10 region and generated a promoteor we believed to be hypersen-sitive. We named it lux pR-HS (<A href="http://parts.igem.org/part:BBa_K2558001">BBa_K2558001</A>).<DIV class="text-center"><STRONG>Figure.2. Design of lacI dosage test circuit.</STRONG></DIV></SPAN><H4 class="service-heading">lacI Dosage &amp; IPTG Induction</H4><SPAN class="name-heading">In NEON system, luxR/AHL complex is able to activate lacI expression on Safety Catch and inhibit gRNA transcription and relief CRISPRi on the positive feedback loop. Therefore the concentration of lacI directly influences the stringency of NEON system. During pilot studies we found that high level of lacI expression severely decreased IPTG induction efficiency. Therefore we used Anderson promoter J23100, J23110 and J23114 to design three constitutive lacI generator of different intensities. The three lacI generators were ligated with Ptac driven reporter sfGFP to make three IPTG induction devices <A href="http://parts.igem.org/part:BBa_K2558203">BBa_K2558203</A>,<A href="http://parts.igem.org/Part:BBa_K2558204">BBa_K2558204</A>,<A href="http://parts.igem.org/Part:BBa_K2558205">BBa_K2558205</A>. By measuring sfGFP fluorescence we tested how these devices react to IPTG.
<DIV class="text-center"><STRONG>Figure.3. Design of IPTG induced CRISPRi test circuit.</STRONG></DIV></SPAN><H4 class="service-heading">IPTG Induced CRISPRi</H4><SPAN class="name-heading">After we clarified the correlation between lacI dosage and its induction efficiency, we furthered the exper-iments by adding CRISPRi to the system. Instead of sfGFP, the three IPTG induction devices with three lev-els of lacI generators are now going to produce a gRNA that targets the promoter of a constitutive sfGFP generator <A href="http://parts.igem.org/part:BBa_K2558206">BBa_K2558206</A>,<A href="http://parts.igem.org/Part:BBa_K2558207">BBa_K2558207</A>,<A href="http://parts.igem.org/Part:BBa_K2558208">BBa_K2558208</A>. By adding IPTG, we can induce the transcription of gRNA. Binding with constantly expressed dCas9, gRNA is going to inhibit the expression of sfGFP. We were able to observe how this process proceeded un-der different levels of lacI expression.
                
</SPAN><SPAN class="text-muted">[1]Ford, A. (2010). &quot;Chapter 9: Information feedback and causal loop diagrams&quot;. Modeling the Environment. Island Press. pp. 99 ff. ISBN 9781610914253.
[2]Brandman, O., Jr, F. J., Li, R., &amp; Meyer, T. (2005). Interlinked fast and slow positive feedback loops drive reliable cell decisions. Science,310(5747), 496-498.
[3]Sato, M., Hata, N., Asagiri, M., Nakaya, T., Taniguchi, T., &amp; Tanaka, N. (1998). Positive feedback regulation of type I IFN genes by the ifn-inducible transcription factor irf-7. Febs Letters, 441(1), 106.
[4]Pomerening, J. R., Kim, S. Y., &amp; Jr, F. J. (2005). Systems-level dissection of the cell-cycle oscillator: bypassing positive feedback produces damped oscillations. Cell, 122(4), 565.
[5]Jr, F. J., &amp; Ha, S. H. (2014). Ultrasensitivity part i: michaelian responses and zero-order ultrasensitivity. Trends in Biochemical Sciences, 39(10), 496-503.
[6]Afroz, T., &amp; Beisel, C. L. (2013). Understanding and exploiting feedback in synthetic biology. Chemical Engineering Science, 103(11), 79-90.
[7]Antunes, L. C., Ferreira, R. B., Lostroh, C. P., &amp; Greenberg, E. P. (2008). A mutational analysis defines vibrio fischeri luxr binding sites. Journal of Bacteriology, 190(13), 4392-4397.
[8]Qi, L., Larson, M., Gilbert, L., Doudna, J., Weissman, J., &amp; Arkin, A., et al. (2013). Repurposing crispr as an rna-guided platform for sequence-specific control of gene expression. Cell, 152(5), 1173.</SPAN></DIV></DIV></DIV></SECTION><SECTION id="HP"><DIV class="container"><DIV class="row"><DIV class="col-lg-12 text-center"><H2 class="section-heading text-uppercase">Results</H2></DIV></DIV><DIV class="row text-left"><DIV class="col-md-12 keith"><H4 class="name-heading">Lux pR Mutation Analysis</H4><H5 class="name-heading">&gt;&gt;&gt;&gt;Design</H5><SPAN class="">With the hope to find an optimal promotor for our NEON system, we designed 9 mutations on sites -35, -36, -37 near the luxR binding site and tested another on -10 site that TUST 2017 reported to show decreased leakage. We conducted experiments to evaluate these lux pR mutants’ reaction to AHL stimulation and their leakage level. The test devices we designed include a constantly expressed luxR and a lux pR (or mutant) driven sfGFP (like <A href="http://parts.igem.org/part:BBa_K2558211">BBa_K2558211</A> with original lux pR promotor, and <A href="http://parts.igem.org/part:BBa_K2558212">BBa_K2558212</A> with lux pR-HS promotor). 
                           </SPAN><H5 class="name-heading">&gt;&gt;&gt;&gt;Results</H5><SPAN class="">We conducted the experiment first with 9 mutations on sites -35, -36, -37 near the luxR binding and another on -10 site that TUST 2017 reported to show decreased leakage. We transferred the plasmids with lux pR driven sfGFP into <I>E. coli</I> DH5α. In theory, AHL can induce gene expression by activating lux pR promotor. We measured the value of Fluorescence/OD without AHL stimulation by microplate reader to show the leakages of different lux pR mutants, as lux pR promotor is not supposed to be activated without AHL. Meanwhile, we also regard the value of Fluorescence/OD with 10<SUP>-10</SUP> M AHL stimulation to observe the sensitivity and expression intensity of different lux pR mutants. Experiment data is shown in the figures below. (Figure.1)
                           </SPAN><DIV class="text-center"><STRONG>Figure.1. Leakage and 10<SUP>-10</SUP> M AHL stimulation of lux pR promotor and its mutations. </STRONG></DIV><SPAN class="text-muted">The fluorescence strengths and OD values are measured by microplate reader respectively at 510 nm and 600 nm wavelength. Plasmids with lux pR promotor and its mutants are transferred into <I>E.coli</I> DH5α. <STRONG>A</STRONG> The bacteria were treated without AHL inducing. High (Fluorescence/OD) ratio indicates strong leakage of promotor. Among all the strains, lux pR promotor has the strongest leakage and the largest error bar as well. <STRONG>B</STRONG> The bacteria were treated with 10<SUP>-10</SUP> M AHL induction. High (Fluorescence/OD) ratio indicates high expression intensity. Mutation 5 has almost the same expression intensity as the wild type lux pR promotor.</SPAN>

It is observed that high expression intensity is accompanied by high level of leakage in lux pR promotors. Wild type lux pR represented with black strand in the figures has the highest level of both leakage and intensity. The next three are Mutant 5 (G-36T), Mutant 6 (G-36C), Mutant 9 (T-35G). Mutant 5 has almost the same intensity as wild type, but the leakage is significantly smaller. Mutant 6 and Mutant 9 have a little bit less leakage and medium intensity. We also did the experiment with 10<SUP>-9</SUP> M and 10<SUP>-8</SUP> M AHL, all of the promotors show high expression levels and therefore unable to compare. (Data not shown)

        <DIV class="text-center"><STRONG>Figure.2. Leakage and AHL stimulation of lux pR Mutant 5 and Mutant 10.  </STRONG></DIV><SPAN class="text-muted">fluorescence strengths and OD values are measured by microplate reader respectively at 510 nm and 600 nm wavelength. <STRONG>A</STRONG> Plasmid with lux pR Mutant 5 is transferred into <I>E.coli</I> DH5α. The bacteria were treated without or with 10<SUP>-10</SUP>, 10<SUP>-9</SUP>, 10<SUP>-8</SUP> M AHL inducing. <STRONG>B</STRONG> Plasmid with lux pR Mutant 10 is transferred into E.coli DH5α. The bacteria were treated without or with 10<SUP>-10</SUP>, 10<SUP>-9</SUP>, 10<SUP>-8</SUP> M AHL inducing.</SPAN><SPAN>We repeated AHL stimulation test on Mutant 5 and Mutant 10 (Figure.2) and decided to combine the two mutations together to form a new promotor with high intensity and small leakage. We named it lux pR-HS. We tested lux pR-HS under the same conditions. We can see that the lux pR-HS has similar leakage to Mutant 10, while it has much higher expression intensity. (Figure.3) The statistics of relative induction and leakage intensity of the 10 Mutants and lux pR-HS are shown in Table.1</SPAN><DIV class="text-center"><STRONG>Figure.3. Leakage and 10<SUP>-10</SUP> M AHL stimulation of lux pR promotor WT, Mutant 5, Mutant 10 and HS.</STRONG></DIV><SPAN class="text-muted">The fluorescence strengths and OD values are measured by microplate reader respectively at 510 nm and 600 nm wavelength. Plasmids with lux pR promotor WT, Mutant 5 and Mutant 10 are transferred into <I>E.coli</I> DH5α. A The bacteria were treated without AHL inducing. High (Fluorescence/OD) ratio indicates strong leakage of promotor. B The bacteria were treated with 10<SUP>-10</SUP> M AHL induction. High (Fluorescence/OD) ratio indicates high expression intensity.</SPAN><DIV class="text-center"><STRONG>Table.1. Results of  lux pR mutation analysis. </STRONG></DIV><SPAN class="">As for why some mutations on -35 to -37 sites would decrease the leakage of lux pR without severely influencing the expression level, we have some hypotheses and explanations. It is reported that -35 to -37 sites are last three bases of lux box which is upstream of the lux promotor and bound specifically by luxR, one of regulatory protein involved in lux expression system.<SUP>[1]</SUP> G-36T mutation has less influence than other mutation on sites -35 to -37, which is same to our result. Besides, G-36T is not involved in the two regions of nucleotides -52 to -50 and -39 to -37 which directly contacted with luxR protein. Therefore, this single substitution of nucleotide would decrease the leakage of lux promotor without influencing the expression level too much.<SUP>[2]</SUP></SPAN><H5 class="name-heading">&gt;&gt;&gt;&gt;Protocol</H5><SPAN class="">1.	Transform the plasmids into <I>E. coli</I> DH5α.
2.	Pick a single colony by a sterile tip from each of the LB plates for all the experimental and control groups. Add the colony into 5ml LB medium with ampicillin at 100 ng/µl. Incubate for 6-8 h at 37℃ in a shaker. 
3.	Measure OD600 of the culture medium with photometer. Dilute the culture medium until OD600 reaches 0.6.
4.	Add 100 µl bacteria culture medium into a sterile 96-well plate. Add AHL to final concentrations of 0, 10<SUP>-10</SUP>, 10<SUP>-9</SUP>, 10<SUP>-8</SUP> M. Fresh LB medium serves as blank control. Positive control is colony constantly expressing sfGFP and negative control is colony without sfGFP expression. Place the 96-well plate into an automatic microplate reader. Incubate at 16℃ overnight and measure the fluorometric value at 510 nm and OD600 of each well every 30 minutes.
5.	Each group should be repeated for at least 3 times.</SPAN><SPAN class="text-muted">[1] Antunes, L. C., et al. &quot;A mutational analysis defines Vibrio fischeri LuxR binding sites.&quot; Journal of Bacteriology 190.13(2008):4392-4397.
[2] Zeng, Weiqian, et al. &quot;Rational design of an ultrasensitive quorum-sensing switch.&quot; Acs Synthetic Biology 6.8(2017).</SPAN><H4 class="name-heading">IPTG Induction and lacI Dosage</H4><H5 class="name-heading">&gt;&gt;&gt;&gt;Design</H5><SPAN class="">In order to investigate how lacI dosage affects IPTG induction, we used Anderson promotor J23100, J23110 and J23114 to design three constitutive lacI generators of different intensities. The three lacI generators were then ligated with Ptac driven reporter sfGFP to make three IPTG induction devices (<A href="http://parts.igem.org/part:BBa_K2558203">BBa_K2558203</A>,<A href="http://parts.igem.org/part:BBa_K2558204">BBa_K2558204</A>,<A href="http://parts.igem.org/part:BBa_K2558205">BBa_K2558205</A>). By measuring sfGFP fluorescence we tested how these devices react to IPTG.</SPAN><H5 class="name-heading">&gt;&gt;&gt;&gt;Results</H5><DIV class="text-center"><STRONG>Figure.4. The effect of varied lacI-LVA (<A href="http://parts.igem.org/part:BBa_C0011">BBa_C0011</A>) promotor strength on IPTG induction of Ptac (<A href="http://parts.igem.org/part:BBa_K2558004">BBa_K2558004</A>).</STRONG></DIV><SPAN class="text-muted">Relative fluorescent intensity is fluorescence per OD600 standardized with fluorescence per OD600 value of each test group at Time=0, IPTG=0. The promotors we used are from the Anderson collection: BBa_J23100 for strong lacI-LVA expression (pink), BBa_J23110 for medium expression (green), BBa_J23114 for weak expression (orange).</SPAN><SPAN class="">With high level of lacI expression (<A href="http://parts.igem.org/part:BBa_K2558203">BBa_K2558203</A>), sfGFP fluorescence had almost no response to IPTG induction. Weak lacI expression (<A href="http://parts.igem.org/part:BBa_K2558205">BBa_K2558205</A>) had the most significant IPTG induced sfGFP expression. With medium lacI expression level (<A href="http://parts.igem.org/part:BBa_K2558204">BBa_K2558204</A>), the induction efficiency lay in between. Therefore, the result proves that high level of lacI expression severely decreases IPTG induction efficiency <SUP>[1]</SUP>. Furthermore, IPTG concentration can affect the regulation part performance. The figure shows that without IPTG the sfGFP florescence intensity remained low. After IPTG addition, fluorescence signal immediately began to climb, forming a peak at five hours after induction, then sfGFP florescence intensity decreased and maintained at a lower level afterwards. IPTG concentration did not significantly affect the height of the peak or the expression level after the peak, but rather the peak width and expression stability of the system. Figures indicate that 5-10 mM IPTG had the most stable induction results.<H5 class="name-heading">&gt;&gt;&gt;&gt;Protocol</H5><SPAN class="">1.	Transform the plasmids into <I>E. coli</I> DH5α.
2.	Pick a single colony by a sterile tip from each of the LB plates for all the experimental and control groups. Add the colony into 5ml M9 medium with ampicillin at 100 ng/µl. Incubate for 6-8 h at 37℃ in a shaker. 
3.	Measure OD600 of the culture medium with photometer. Dilute the culture medium until OD600 reaches 0.6.
4.	Add 100 µl bacteria culture medium into a sterile 96-well plate. Add IPTG to final concentrations of 0, 1, 5, 10, 20 mM. Fresh M9 medium serves as blank control. Positive control is colony constantly expressing sfGFP and negative control is colony without sfGFP expression. Place the 96-well plate into an automatic microplate reader. Incubate at 16℃ overnight and record the fluorometric value at 510 nm and OD600 for each well every 30 minutes.
5.	Each group should be repeated for at least 3 times.</SPAN><SPAN class="text-muted">[1] Szabolcs Semsey, Sandeep Krishna. &quot;The effect of LacI autoregulation on the performance of the lactose utilization system in Escherichia coli&quot; Nucleic Acids Res 2013 Jul; 41(13): 6381–6390</SPAN></SPAN><H4 class="name-heading">IPTG Induced CRIPSRi</H4><H5 class="name-heading">&gt;&gt;&gt;&gt;Design</H5><SPAN class="">After we clarified the correlation between lacI dosage and its induction efficiency, we furthered the experiments by adding CRISPRi to the system. Instead of sfGFP, the IPTG induction devices with different levels of lacI generator is now going to produce a gRNA that targets the promotor of a constitutive sfGFP generator (<A href="http://parts.igem.org/part:BBa_K2558206">BBa_K2558206</A>, <A href="http://parts.igem.org/part:BBa_K2558207">BBa_K2558207</A>, <A href="http://parts.igem.org/part:BBa_K2558208">BBa_K2558208</A>). By adding IPTG, we can induce the transcription of gRNA. Binding with constantly expressed dCas9, gRNA is going to inhibit the expression of sfGFP. We will be able to observe how this process can operate under different levels of lacI expression.</SPAN><H5 class="name-heading">&gt;&gt;&gt;&gt;Results</H5><DIV class="text-center"><STRONG>Figure.5. The effect of varied lacI-LVA (<A href="http://parts.igem.org/part:BBa_C0011">BBa_C0011</A>) promotor strength on IPTG induction of CRISPRi.</STRONG></DIV><SPAN class="text-muted"> The Y axis values are fluorescence per OD600 standardized with fluorescence per OD600 value of each test group at Time=0, IPTG=0 M. The promotors we used are from the Anderson collection: BBa_J23110 for medium expression (orange), <A href="http://parts.igem.org/part:BBa_J23114">BBa_J23114</A> for weak expression (blue).</SPAN><SPAN class="">In this experiment, the expression of sfGFP was inhibited by IPTG induced CRISPRi system. (Figure.5) Without CRISPRi the Fluorescence/OD600 increased steadily. With medium lacI expression, the addition of IPTG caused the fluorescence intensity to decrease at three hours after addition. With different IPTG concentration the final sfGFP fluorescence intensity remained approximately the same, which is consistent with previous IPTG induction results. However, with weak lacI expression, sfGFP expression was inhibited from the beginning. With the direct IPTG induction experiment we performed before, we hypothesize that with the weak Anderson promotor, basal gRNA expression will be above the threshold that is required for suppressing the sfGFP expression.<SUP>[1]</SUP><H5 class="name-heading">&gt;&gt;&gt;&gt;Protocol</H5><SPAN class="">1.	 Transform the plasmids into <I>E. coli</I> DH5α.
            2.	 Pick a single colony by a sterile tip from each of the LB plates for all the experimental and control groups. Add the colony into 5ml M9 medium with ampicillin at 100 ng/µl and chloramphenicol at 34 ng/µl. Incubate for 6-8 h at 37℃ in a shaker.
            3.	Measure OD600 of the culture medium with photometer. Dilute the culture medium until OD600 reaches 0.6.
            4.	Add 100 µl bacteria culture medium into a sterile 96-well plate. Add IPTG to final concentrations of 0, 1, 3, 5, 10 and 20 mM. Fresh M9 medium serves as blank control. Place the 96-well plate into an automatic microplate reader. Incubate at 16℃ overnight and record the fluorometric value at 510 nm and OD600 of each well every 30 minutes.
            5.	Each group should be repeated for at least 3 times.</SPAN><SPAN class="text-muted">[1]Mückl A, Schwarz-Schilling M, Fischer K, Simmel FC. “Filamentation and restoration of normal growth in Escherichia coli using a combined CRISPRi sgRNA/antisense RNA approach.” PLoS One. 2018 Sep 11;13(9):e0198058. doi: 10.1371/journal.pone.0198058. eCollection 2018.</SPAN></SPAN><H4 class="service-heading">Neon System Characterization</H4><H5 class="name-heading">&gt;&gt;&gt;&gt;Design</H5><SPAN class="">Assisted with the experiences we gained form the experiments above, we built and tuned the NEON system. We designed this experiment to characterize how Neon the positive feedback plasmid (<A href="http://parts.igem.org/part:BBa_K2558214">BBa_K2558214</A>), and Safety Catch the CRISPRi plasmid (<A href="http://parts.igem.org/part:BBa_K2558215">BBa_K2558215</A>, <A href="http://parts.igem.org/part:BBa_K2558216">BBa_K2558216</A>) work together.</SPAN><H5 class="name-heading">&gt;&gt;&gt;&gt;Results</H5><SPAN class="">We are still calibrating the NEON system. The results are preliminary, however from Figure.6 we can conclude that the system works to some extent. The positive feedback plasmid Neon (<A href="http://parts.igem.org/part:BBa_K2558214">BBa_K2558214</A>) had the highest expression due to uncontrollable leakage. Original lux pR (<A href="http://parts.igem.org/part:BBa_R0062">BBa_R0062</A>) and the new lux pR-HS (<A href="http://parts.igem.org/part:BBa_K2558001">BBa_K2558001</A>) we designed had lower basal expression. The addition of Safety Catch (<A href="http://parts.igem.org/part:BBa_K2558215">BBa_K2558215</A>) and Safety Catch-HS (<A href="http://parts.igem.org/part:BBa_K2558216">BBa_K2558216</A>)  almost eliminated the leakage of both positive feedback and the non-positive feedback systems. It is foreseeable that with appropriate parameters NEON system can be activated to almost 10<SUP>4</SUP> fold.
 <DIV class="text-center"><STRONG>Figure.6. Basal expression of lux quorum sensing systems.</STRONG></DIV><SPAN class="text-muted"> The fluorescence intensity was measured by flow cytometer at 488 nm. Neon, Safety Catch, Safety Catch-HS and lux pR, lux pR-HS test plasmids were transferred into <I>E.coli</I> DH5α. High fluorescence intensity suggests high level of basal expression in the system. </SPAN><H5 class="name-heading">&gt;&gt;&gt;&gt;Protocol</H5><SPAN class="">1.	Transform the plasmids into <I>E. coli</I> DH5α.
2.	Pick a single colony by a sterile tip from each of the LB plates for all the experimental and control groups. Add the colony into 5ml LB medium with ampicillin at 100 ng/µl and chloramphenicol at 34 ng/µl. Incubate for 6-8 h at 37℃ in a shaker. 
3.	Measure OD600 of the culture medium with photometer. Dilute the culture medium until OD600 reaches 0.6.
4.	Add 100 µl bacteria culture medium into a sterile 96-well plate. Add IPTG to final concentrations of 0 or 10 mM and AHL to final concentrations of 0, 10<SUP>-9</SUP>, 10<SUP>-8</SUP> M. Fresh LB medium serves as blank control. Fix sample with 1.5 mg/ml kanamycin at one hour intervals. Then use flow cytometry to measure the fluorescent intensity at 488 nm of each sample.
5.	Each group should be repeated for at least 3 times.</SPAN><SPAN class="text-muted">[1]Afroz, T., &amp; Beisel, C. L. (2013). Understanding and exploiting feedback in synthetic biology. Chemical Engineering Science, 103(11), 79-90.
[2]Qi, L., Larson, M., Gilbert, L., Doudna, J., Weissman, J., &amp; Arkin, A., et al. (2013). Repurposing crispr as an rna-guided platform for sequence-specific control of gene expression. Cell, 152(5), 1173.</SPAN></SPAN></DIV></DIV></DIV></SECTION><SECTION id="HP"><DIV class="container"><DIV class="row"><DIV class="col-lg-12 text-center"><H2 class="section-heading text-uppercase">Model</H2></DIV></DIV><DIV class="row text-left"><DIV class="col-md-12 keith"><H4 class="service-heading">1.Introduction</H4><SPAN class="name-heading"><STRONG>1.1 Why do we need a model</STRONG>
            Developing a mathematical model of the systems we established (or plan to establish) is a vital step in our project. A model, namely, is a mathematical representation of the system. The model can help us with the following:
            - Confirming that our system has the potential to work as expected. For example, as for the Test System, the model should present us that both the self-activating component and safety-catch component work in theory.
            - Determining optimal conditions for the system to work. e.g. Initial concentration of AHL.
            - Estimating biological parameters.
            - Understanding the mechanism of the system better.
            - Optimizing the design of our final project.<STRONG> 1.2 What is our model about?</STRONG>
            The model is basically established by using ODEs (ordinary differential equations) to describe reactions and events in the system. ODE is a powerful tool when it comes to modelling a dynamic system. It allows us to predict the behavior of the system at given conditions, and to analyze several features of the system such as robustness and stability. For the sake of simplicity, various assumptions are made when developing the model. These assumptions usually tend to simplify our systems, reducing our workload. According to the assumptions, many trivial details are omitted, while the overall properties and behaviors of the model remain unchanged.
            When describing the reactions and events involved, we mainly apply the law of mass action, which assumes the system is in quasi steady state. When modelling reactions that resemble enzymatically catalyzed reactions and events that involve ligand binding, we refer to Hill Equation, which is simple but effective.
            We focused on building a model for the “Test System”. The model depicts the behaviors of the Test System under several different conditions, and highlights some key features and properties of the system. The model is generally successful as it predicted the behavior of the system correctly, according to our experiments. We have performed various analysis regarding the model and pinpointed some intriguing characteristics of the system.
            In addition to this, we also built a model for the Neon System. Using the model, we can simulate how Neon System works. It can serve as a “guideline” for our future work on the Neon System.
            </SPAN><H4 class="service-heading">2. Model Development —— Test System</H4><SPAN class="name-heading"><STRONG>2.1 Overview</STRONG>
            In the following sections, we will explain how we developed our model of the Test System and present our results. Figure 1 offers an intuitive explanation of the way our Test System works. It can be seen that the system would act differently (perform different functions) under different conditions. Our goal is to specifically describe the system given different conditions, using ODE. The concentrations of the chemical species in the system are regarded as variables, which change over time. Thus, depicting how the concentrations of the molecules of interest change is the primary step.<DIV class="text-center"><STRONG>Figure 1  Illustration of How the System Works</STRONG></DIV><STRONG>2.2 Notations of Important Variables and Chemical Species</STRONG>
            In the model, a set of variables representing the concentrations of various chemical species in the system are used. All these variables are functions of time. The notations and corresponding meanings are listed below.
            $H(t)$——concentration of 3OC6HSL in the cell
            $I(t)$——concentration of luxI
            $H_{ex}(t)$——concentration of 3OC6HSL outside the cell (in the medium)
            $C(t)$——concentration of 3OC6HSL-LuxR complex
            $R(t)$——concentration of LuxR monomers
            $G(t)$——concentration of GFP
            $Cas(t)$——concentration of dCas protein in the cell
            $C^*(t)$——concentration of dCas/gRNA complex
            $gRNA(t)$——concentration of unbound gRNA
            $I^*(t)$——concentration of lacI<STRONG>2.3 Details of Model Development</STRONG>
           Next, we will dissect the reactions and events involved in the model. For each reaction involved, we will present the relevant assumptions, molecules involved, and formulae and parameters related to reaction rate or steady state.<STRONG> 1) AHL synthesis inside the cell</STRONG> 
            - Reaction: Precursors $\to$ AHL (3OC6HSL)
            - Reaction rate: $v=k_0 I(t)$
            - Relevant parameters: $k_0 \Rightarrow$ rate of synthesis of 3OC6HSL
             Explanation/Assumptions: In bacteria, AHL is synthesized from its precursors via several enzymatic reactions. The last step of synthesis is catalyzed by protein luxI. Due to the scarcity of luxI protein in our system, it can be seen as the rate-limiting factor in the synthesis reaction. Thus, we assume the rate of synthesis of AHL is directly proportional to the concentration of luxI.<STRONG> 2) Diffusion of AHL out of the cell</STRONG> 
            - Event: AHL (in the cell) $\to$ AHL (outside the cell)
            - Rate:$v=r(H(t)-H_{ex}(t))$
            - Relevant parameter: $r \Rightarrow$ coefficient of diffusion
            Explanation: In reality, diffusion of AHL across the membrane is a relatively complicated process. If we see the diffusion of AHL across cell membrane as simple diffusion, the process can be described using Fick's Law, which assumes that the flux of molecules across a membrane is proportional to the gradient concentration.
            <DIV class="text-center">$$J=-D\frac{dc}{dx}$$</DIV>
             Approximately, this means the change of number of molecules on one side of the membrane is proportional to the difference of concentration across the membrane. In our model, we assume that the diffusion of AHL across the membrane is fast enough so that we can omit the process in which AHL travels within the membrane. Also, it is assumed that the distribution of AHL in the medium and inside cells can come to a steady state in a very short time, which means the system is generally homogeneous when it comes to AHL distribution. Previous iGEM teams have discussed the details of AHL diffusion in the extracellular medium, and we believe these details prove to play insignificant roles in our prospect so they are not included in the model. 
<STRONG> 3) Diffusion of AHL into the cell</STRONG> 
            - Event: AHL (outside the cell) $\to$ AHL (in the cell)
            - Rate:$v=pr(H_{ex}(t)-H(t))$
            - Relevant parameters: $p \Rightarrow$ coefficient of population density.
            Explanation: This process is the reverse version of the previous one. A coefficient of population density is multiplied to the gradient concentration, as the number of bacteria in the system would affect how quickly the external AHL concentration changes.<STRONG> 4) Formation of 3OC6HSL-LuxR Complex</STRONG> 
            - Reaction: 2*3OC6HSL + 2 LuxR $\to$ 3OC6HSL-LuxR Complex
            - Reaction rate: $v=2k_1(H(t))^2(R(t)-2C(t))^2$
            - Parameter: $k_1\Rightarrow $rate constant of complex formation
            Explanation: This is based on the law of mass action. Two molecules of 3OC6HSL binds to one luxR protein to form a complex, which can then activate the transcription of luxpR promoters.<STRONG> 5) Dissociation of 3OC6HSL-LuxR Complex</STRONG> 
           - Reaction:  3OC6HSL-LuxR Complex$\to$2 3OC6HSL + 2 LuxR
- Reaction rate (Re-formation of AHL) : $v=2k_2C(t)$
- Parameter: $k_2\Rightarrow $rate constant of complex dissociation
Explanation: This is based on the law of mass action.<STRONG> 6)	Degradation of AHL inside the cell</STRONG> 
            - Reaction:  3OC6HSL $\to$ NULL
- Reaction rate  : $v=d_1H(t)$
- Parameter: $d_1\Rightarrow $rate constant of AHL degradation inside the cell
Explanation: This is based on the law of mass action. We assume that AHL gradually degrades in the cell, while the rate is proportional to its current concentration.<STRONG> 7)	Degradation of AHL outside the cell</STRONG> 
           - Reaction:  3OC6HSL $\to$ NULL
- Reaction rate : $v=d_2H_{ex}(t)$
- Parameter: $d_2\Rightarrow $rate constant of AHL degradation outside the cell
Explanation: This is based on the law of mass action. Similar to the previous one.<STRONG> 8)	Binding of lacI to the placI promoter</STRONG> 
            - Reaction:  lacI+placI $\to$ lacI bound to placI
- Proportion of bound sites: $c_{lac}=\frac{I_l(t)}{K_d^*(t)+I^l(t)}$
- Parameter: $K_d^*\Rightarrow $ Binding affinity of lacI to placI<STRONG> 9)	Production of guide RNA</STRONG> 
            - Reaction:  NULL $\to$ gRNA
- reaction rate : $v=a_g*(1-c_{lac})$
- Parameter: $a_g\Rightarrow $ Expression rate of gRNA when placI is not repressed
Explanation: Only promoters that are not repressed by lacI can support the transcription of gRNA.<STRONG> 10) Degradation of guide RNA</STRONG> 
            - Reaction:  gRNA $\to$ NULL
- reaction rate : $v=d_g*G(t)$
- Parameter: $d_g\Rightarrow $ Rate constant of degradation of gRNA<STRONG> 11) Degradation of dCas9 protein</STRONG> 
            - Reaction:  dCas9 $\to$ NULL
- reaction rate : $v=d_c*Cas(t)$
- Parameter: $d_c\Rightarrow $ Rate constant of degradation of dCas9<STRONG> 12) Production of dCas9 protein</STRONG> 
            - Reaction:  NULL $\to$ dCas9
- reaction rate : $v=a_0$
- Parameter: $a_0\Rightarrow $ Efficiency of the constant promoter<STRONG> 13) Production of dCas9-gRNA complex</STRONG>     
            - Reaction:  dCas9+gRNA $\to$ dCas9-gRNA complex
- reaction rate : $v=k_3gRNA(t)Cas(t)$
- Parameter: $k_3\Rightarrow $ Rate constant of dCas9 complex formation
Explanation: This is based on the law of mass action.<STRONG> 14)	Dissociation of dCas9-gRNA complex</STRONG> 
            - Reaction:  dCas9-gRNA complex$\to$dCas9+gRNA 
- reaction rate : $v=k_4C^*(t)$
- Parameter: $k_4\Rightarrow $ Rate constant of dCas9 complex dissociation
Explanation: This is based on the law of mass action.<STRONG> 15) Binding of dCas9 complex to the luxPR promoter</STRONG> 
            - Reaction:  dCas9-gRNA complex+luxPR $\to$ inhibited luxPR
- Proportion of bound sites  : $c_{crispr}=\frac{C^*(t)}{K_c^*(t)+C^*(t)}$
- Parameter: $K_c^*\Rightarrow $ Binding affinity of dCas9 complex to the luxPR promoter.<STRONG> 16) Production of luxR protein</STRONG> 
            - Reaction:  NULL $\to$ luxR
- reaction rate : $v=a_0$
- Parameter: $a_0\Rightarrow $ Efficiency of the constant promoter<STRONG> 17) Degradation of luxR protein</STRONG> 
           - Reaction:  luxR $\to$ NULL
- reaction rate : $v=b_2R(t)$
- Parameter: $b_2\Rightarrow $ Rate constant of degradation of LuxR<STRONG> 18) Binding of 3OC6HSL-LuxR Complex to luxPR promoter</STRONG> 
            - Reaction: 3OC6HSL-LuxR Complex+ luxPR promoter$\to$ activated luxPR promoter
- Proportion of bound sites: $x=\frac{a_1(C(t))^{\beta_1}}{2(K_m^{\beta_1}+(C(t))^{\beta_1})}$
- Parameter: $K_m\Rightarrow $ Binding affinity of the complex to luxPR promoter
$beta_1\Rightarrow$Hill coefficient for the binding of complex to luxpR
                Explanation: This is based on Hill equation.<STRONG> 19) Production of luxI protein</STRONG> 
          - Reaction:  NULL $\to$ luxI
- reaction rate : $v=(1-c_{crispri})*(l_1+a_1*x)$
- Parameter: $l_1\Rightarrow $ Leaky expression efficiency of lux pR promoter
$a_1\Rightarrow $ Enhanced expression efficiency of lux pR when bound to the complex. (The increment)
Explanation: We assume that all the sites bound by the dCas9 complex are completely inactivated, thus they are unable to produce any product.<STRONG> 20) Degradation of luxI</STRONG> 
            - Reaction:  luxI$\to$ NULL
- reaction rate : $v=b_1*I(t)$
- Parameter: $b_1\Rightarrow $ Rate constant of degradation of luxI<STRONG> 21) Production of GFP</STRONG> 
            - Reaction:  NULL $\to$ GFP
- reaction rate : $v=(1-c_{crispri})*(l_1+a_1*x)$
- Parameter: $l_1\Rightarrow $ Leaky expression efficiency of lux pR promoter
$a_1\Rightarrow $ Enhanced expression efficiency of lux pR when bound to the complex.(The increment)
Explanation: We assume that all the sites bound by the dCas9 complex are completely inactivated, thus they are unable to produce any product.<STRONG> 22) Degradation of GFP</STRONG> 
            - Reaction:  GFP$\to$ NULL
- reaction rate : $v=b_3*gRNA(t)$
- Parameter: $b_3\Rightarrow $ Rate constant of degradation of GFP<STRONG> 23) Production of lacI</STRONG> 
           - Reaction:  NULL $\to$ lacI
- reaction rate : $v=(l_1+a_1*x)$
- Parameter: $l_1\Rightarrow $ Leaky expression efficiency of lux pR promoter
$a_1\Rightarrow $ Enhanced expression efficiency of lux pR when bound to the complex.(The increment)
Explanation: This lux pR promoter is not inhibited by CRISPRi<STRONG> 24) Degradation of lacI</STRONG> 
            - Reaction:  lacI$\to$ NULL
- reaction rate : $v=d_l*I^*(t)$
- Parameter: $d_l\Rightarrow $ Rate constant of degradation of lacI<STRONG> 25) Binding of IPTG to lacI</STRONG> 
           - Reaction: lacI+IPTG $\to$ inactivated lacI
- Property: Since lacI a homotetramer, IPTG can bind to any of the four subunits t inactivate it.
- Concentration of active lacI (presumably at steady state): $I_l(t)=I^*(t)(\frac{K_i}{K_i+IPTG})^4$
- Parameter: $K_i \Rightarrow$ Dissociation constant of lacI-IPTG

  Based on all the reactions and events listed above, we are then able to explicitly describe how the concentration of each type of molecules of interest change over time. We will integrate the information we have so far and begin to write out the equations.
  The first ODE describes the change of AHL concentration inside a cell. The time scale taken into consideration is relatively short, so we omit the impact of cell division and death. The change of AHL concentrations involves AHL synthesis, degradation, diffusion and binding to LuxR.
$$
\frac{d}{dt}H(t)=k_0I(t)-r(H(t)-H_{ex}(t))-2k_1(H(t))^2(R(t)-2C(t))^2+2k_2C(t)-d_1H(t)
$$
  The following equation describes how the concentration of luxR-AHL complex changes over time.
$$
\frac{d}{dt}C(t)=k_1(H(t))^2(R(t)-2C(t))^2-k_2C(t)
$$
  The following equations define the inhibitory effects of CRISPRi and lacI protein on their target sequences, respectively.
$$
c_{crispri}=\frac{C^*(t)}{K_m^*(t)+C^*(t)}
$$
 
$$
 c_{lac}=\frac{I_l(t)}{K_d^*(t)+I_l(t)}
$$
LuxI protein concentration is supposed to change the way described by the ODE below.
$$
\frac{d}{dt}I(t)=(1-c_{crispri})*[l_1+\frac{a_1(C(t))^{\beta_1}}{(K_m^{\beta_1}+(C(t))^{\beta_1})}]-b_1I(t)
$$

The dynamics of luxR monomer is rather simple, as described below.
$$
\frac{d}{dt}R(t)=a_0-b_2R(t)
$$
The change of concentration of AHL outside the cell mainly depends on dilution effect and cross-membrane diffusion.
$$
\frac{d}{dt}H_{ex}(t)=pr(H(t)-H_{ex}(t))-d_2H_{ex}(t)
$$
The following equation describes how GFP is produced under regulation.
$$
\frac{d}{dt}G(t)=(1-c_{crispri})*[\frac{a_1(C(t))^{\beta_1}}{(K_m^{\beta_1}+(C(t))^{\beta_1})}+l_1]-b_3G(t)
$$
And finally, the equations below describe the dynamics of CRISPRi system.
$$
\frac{d gRNA(t)}{dt}=a_g*(1-c_{lac})-k_3gRNA(t)Cas(t)+k_4C^*(t)-d_ggRNA(t)
$$
$$
\frac{dC^*(t)}{dt}=k_3gRNA(t)Cas(t)-k_4C^*(t)
$$
$$
\frac{dCas(t)}{dt}=a_0-k_3gRNA(t)Cas(t)+k_4C^*(t)-d_cCas(t)
$$

$$
\frac{dI^*(t)}{dt}=l_1+\frac{a_1(C(t))^{\beta_1}}{(K_m^{\beta_1}+(C(t))^{\beta_1})}-d_l*I^*(t)
$$


With these equations, we will soon be able to run a simulation of the system. Before we obtain the result of the simulations, we are required to set the values of biological parameters and set the initial state of the system. That, is another story…


            <STRONG>2.4 Setting the Parameters and Initial States</STRONG>
            How do we acquire the parameters we need to simulate and validate our model? Estimating all the parameters based on our experimental results is not realistic, due to the large number of parameters, the nonlinearity of the system and the potential inaccuracy of our experiments. A straightforward way to obtain the values of parameters is to refer to literatures, results of previous iGEM teams and common sense. We trust that the simulations run based on these values of parameters should provide us with generally satisfying results, which can at least predict the general behavior of the system correctly.
  The values of parameters we utilize are listed below.

            </SPAN><DIV class="text-center"><STRONG>Table 1    Biological Parameters Used in the Model</STRONG></DIV><TABLE class="table"><THEAD><TR><TH>Name</TH><TH>Description</TH><TH>Value</TH><TH>Unit</TH><TH>Source</TH></TR></THEAD><TBODY><TR><TD>k0</TD><TD>rate of synthesis of 3OC6HSL</TD><TD>0.04</TD><TD>min<SUP>-1</SUP></TD><TD>Ref<SUP>[1]</SUP></TD></TR><TR><TD>r</TD><TD>coefficient of diffusion</TD><TD>100</TD><TD>min<SUP>-1</SUP></TD><TD>Ref<SUP>[2]</SUP></TD></TR><TR><TD>k1</TD><TD>rate constant of complex formation</TD><TD>0.1</TD><TD>nM<SUP>-1</SUP>min<SUP>-1</SUP></TD><TD>Ref<SUP>[2]</SUP></TD></TR><TR><TD>k2</TD><TD>rate constant of complex dissociation</TD><TD>10</TD><TD>min<SUP>-1</SUP></TD><TD>Ref<SUP>[4]</SUP></TD></TR><TR><TD>d1</TD><TD>rate constant of AHL defradation inside the cell</TD><TD>0.004</TD><TD>min<SUP>-1</SUP></TD><TD>Estimated by ourselves</TD></TR><TR><TD>d2</TD><TD>rate constant of AHL degradation outside the cell</TD><TD>0.0004</TD><TD>min<SUP>-1</SUP></TD><TD>Estimated by ourselves
</TD></TR><TR><TD>Kd</TD><TD>Binding affinity of lacI to placI</TD><TD>10.49</TD><TD>nM</TD><TD>PAMDB database</TD></TR><TR><TD>ag</TD><TD>Transcription rate of placI (not repressed)</TD><TD>5</TD><TD>nMmin<SUP>-1</SUP></TD><TD>Ref<SUP>[3]</SUP></TD></TR><TR><TD>dg</TD><TD>Rate constant of degradation of gRNA</TD><TD>0.42</TD><TD>nM min<SUP>-1</SUP></TD><TD>Ref<SUP>[3]</SUP></TD></TR><TR><TD>dc</TD><TD>Rate constant of degradation of dCas9</TD><TD>0.14</TD><TD>nMmin<SUP>-1</SUP></TD><TD>Ref<SUP>[3]</SUP></TD></TR><TR><TD>a0</TD><TD>Efficiency of the constant promoter</TD><TD>1</TD><TD>nM min<SUP>-1</SUP></TD><TD>Estimated by ourselves</TD></TR><TR><TD>k3</TD><TD>Rate constant of dCas9 complex formation</TD><TD>0.0108</TD><TD>nM<SUP>-1</SUP>min<SUP>-1</SUP></TD><TD>Ref<SUP>[3]</SUP></TD></TR><TR><TD>k4</TD><TD>Rate constant of dCas9 complex dissociation</TD><TD>0.00001</TD><TD>min<SUP>-1</SUP></TD><TD>Ref<SUP>[3]</SUP></TD></TR><TR><TD>Kc</TD><TD>Binding affinity of dCas9 complex to target sequence</TD><TD>0.025</TD><TD>nM</TD><TD>Ref<SUP>[3]</SUP></TD></TR><TR><TD>b2</TD><TD>Rate constant of degradation of LuxR</TD><TD>0.0231</TD><TD>min<SUP>-1</SUP></TD><TD>Team:ETH_Zurich 2014</TD></TR><TR><TD>Km</TD><TD>Binding affinity of the complex to lux pR promoter</TD><TD>10</TD><TD>nM</TD><TD>Team:ETH_Zurich 2014</TD></TR><TR><TD>l1</TD><TD>Leaky expression efficiency of lux pR promoter</TD><TD>0.02</TD><TD>nM min<SUP>-1</SUP></TD><TD>Estimated by ourselves</TD></TR><TR><TD>a1</TD><TD> Enhanced expression efficiency of lux pR on the Neon plasmid when bound to the complex.(The increment)</TD><TD>5</TD><TD>nM min<SUP>-1</SUP></TD><TD>Team:Tokyo Tech 2012</TD></TR><TR><TD>a2</TD><TD>Enhanced expression efficiency of lux pR on the Safety Catch plasmid when bound to the complex.(The increment)</TD><TD>5</TD><TD>nM min<SUP>-1</SUP></TD><TD>Team:Tokyo Tech 2012</TD></TR><TR><TD>b1</TD><TD>Rate constant of degradation of luxI</TD><TD>0.0002</TD><TD>min<SUP>-1</SUP></TD><TD>Team:ETH_Zurich 2014</TD></TR><TR><TD>b3</TD><TD> Rate constant of degradation of GFP</TD><TD>0.005</TD><TD>min<SUP>-1</SUP></TD><TD>Estimated ourselves</TD></TR><TR><TD>dl</TD><TD>Rate constant of degradation of lacI</TD><TD>0.023</TD><TD>min<SUP>-1</SUP></TD><TD>PAMDB database</TD></TR></TBODY></TABLE>
            The initial concentrations of several species, such as dCas9 and guide RNA, which are fairly difficult to &quot;guess&quot; and nowhere to be found in literatures, can be better estimated by running simplified models only regarding the equilibrium of the relevant molecules. Here, we present an example run by SimBiology App on MATLAB. The diagram of the model is shown below.<DIV class="text-center"><STRONG>Figure 2   Graphical Representation of the SimBiology Model for Determining Initial Concentrations of CRISPRi-Related Species</STRONG></DIV>
          
       
              Running the model with relevant parameters set on SimBiology can provide us with diagrams showing the result of simulation. The result of the simulation provides us with initial concentrations of the relevant molecules which are probably more believable.
                 <DIV class="text-center"><STRONG>Figure 3   Results of SimBiology Model. Simulated Concentrations of CRISPRi-Related Species at Equilibrium are Shown.</STRONG></DIV><H4 class="service-heading">3.Model Simulation—— Test System</H4><SPAN class="name-heading"><STRONG>3.1 Effect of AHL concentration</STRONG>
             According to our expectations, the Test System is inducible upon the addition of external AHL. With proper concentrations of AHL, the cells would undergo a series of changes regardin gene expression. The most obvious change is the increase of GFP concentration in the cell, for us. The “safety catch” component also ensures that the system has minimal leakage, meaning that minor fluctuations in the system when no AHL is added, or very trace amount of AHL should not activate the system. To test whether the Test System does possess these features, we will run a series of simulations of our model.
  The parameter values used in the model are stated in previous parts. As for the intial states, the concentrations of CRISPRi related molecules are set according to the SimBiology simulation results. Concentrations of most molecules that are not supposed to be synthesized without AHL are set to 0. External AHL concentrations are tuned to reveal the effect of AHL concentration on the system.
  To begin with, we set intial external AHL concentration as 10-6 M, which is relatively high. The simulation result is shown below.<DIV class="text-center"><STRONG>Figure 4  Simulation Result——With Very High Initial AHL Concentration</STRONG></DIV>
The result meets our expectation. According to the plot, GFP concentration will begin to rise exponentially following the addition of AHL. The velocity of increment decreases gradually, till approximate saturation is reached. The simulation result implies that the Test System is inducible.
  What happens when lower concentrations of AHL is added? The following plots represent 10-7 and 10-9 M of AHL concentration, respectively.<DIV class="text-center"><STRONG>Figure 5       Results of Varied AHL Concentrations</STRONG></DIV>
As can be deduced from these plots, the decrease of AHL concentration from 10-6 to 10-7 has no substantial influence on the behavior of the system. Through further analysis, we found that the rate of GFP concentration increment is indeed higher when 10-6 M of AHL is added than when 10-7 M of AHL is added, however this difference can merely be detected after 100 seconds. Given these two different initial states, the state of the system will eventually become quite same. For one, this shows that the system is robust, to some extent. In addition to this, this phenomenon makes sense and has biological interpretations. Within a certain range of concentrations, AHL serves as a “trigger” which turns the system on by enabling expression of several proteins, so it has an “all-or-none” effect, to some extent.
When the concentrations of AHL is further decreased, to 10-9 M, for instance, the behavior of the system is altered dramatically. When AHL concentration is this low, the system cannot be activated and no GFP is expressed. This is partial thanks to the “safety catch” design. This result gives us confidence that leakage can be prevented with the presence of minor fluctuations.
  You can interact with the plot down below. Use mouse to drag the slider to tune the initial AHL concentration and observe how this will influence GFP production in the cells!<STRONG>Summary:
-	Increased AHL concentration tends to result in faster GFP production.
    -	A threshold for AHL concentration exists. AHL concentration below this threshold cannot activate the system thanks to the safety catch mechanism.</STRONG><DIV class="one"><UL><LI><A href="javascript:;"><SPAN>Concentration=10<SUP>-5</SUP> M</SPAN></A></LI><LI><A href="javascript:;"><SPAN>Concentration=10<SUP>-6</SUP> M</SPAN></A></LI><LI><A href="javascript:;"><SPAN>Concentration=10<SUP>-7</SUP> M</SPAN></A></LI><LI><A href="javascript:;"><SPAN>Concentration&lt;=10<SUP>-8</SUP> M</SPAN></A></LI></UL></DIV><STRONG>3.2  Effect of lux pR efficiency</STRONG>
       Lux pR, a promoter which is activated by AHL-luxR complex, is a critical part we use in our project. We planned to generate various mutated forms of lux pR, and we seek to modify and improve its efficiency. But before we do any experiments, we need to confirm one thing: does the efficiency of lux pR matter? There are a lot of promoters, activators and repressors in the system, so any one of them likely contributes little to the whole story. How to make sure? By using our mathematical model, naturally!
  We assume the mutations alters the transcription efficiency. Efficiency refers to the relative rate of transcription driven by the promoter when AHL-luxR complex is bound. First of all, we reset the transcription efficiency of activated lux pR on the Neon plasmid to 20%, 50%,100% and 150% compared to the default value we used in previous parts. The initial concentration of AHL is 10-6 M. A series of plots are generated. <DIV class="text-center"><STRONG>Figure 6     Impact of lux pR efficiency (on Neon Plasmid) on the Performance of the System</STRONG></DIV>
  Clearly, the efficiency of lux pR on the Neon plasmid significantly affects the behavior of the system. An efficient/sensitive lux pR promoter results in a faster increase of GFP concentration. This also accords with our further experimental data.
  Tuning the activity of the lux pR on the safety catch promoter also alters the behavior of the system. We set the lux pR on the Neon plasmid as the original one, while remaining AHL concentration as 10-6 M, and then tune the efficiency of lux pR on the safety catch plasmid to produce the plots below.<DIV class="text-center"><STRONG>Figure 7     Impact of lux pR efficiency (on Safety Catch Plasmid) on the Performance of the System</STRONG></DIV>
                        
Obviously, the effect of lux pR promoter on both plasmids can interact can produce a variety of results. A heatmap shows how their interaction alters the productivity of GFP. The color indicates the relative concentration of GFP at “30000 s” time point. <DIV class="text-center"><STRONG>Figure 8     Interaction of Effects of lux pR on Two Plasmids</STRONG></DIV>
                        You can interact with the plot down below. Use mouse to drag the slider to tune the lux pR efficiency and observe how this will influence GFP production in the cells!<STRONG>Summary:
-	lux pR efficiency on both plasmids has great impact on the system.
                            -	lux pR efficiency on both plasmids can interact and determine the performance of the system.</STRONG><DIV class="one"><UL><LI><A href="javascript:;"><SPAN>(Neon lux pR:Safety catch lux pR)20:50</SPAN></A></LI><LI><A href="javascript:;"><SPAN>(Neon lux pR:Safety catch lux pR)20:100</SPAN></A></LI><LI><A href="javascript:;"><SPAN>(Neon lux pR:Safety catch lux pR)20:150</SPAN></A></LI><LI><A href="javascript:;"><SPAN>(Neon lux pR:Safety catch lux pR)50:50</SPAN></A></LI><LI><A href="javascript:;"><SPAN>(Neon lux pR:Safety catch lux pR)50:100</SPAN></A></LI><LI><A href="javascript:;"><SPAN>(Neon lux pR:Safety catch lux pR)50:150</SPAN></A></LI><LI><A href="javascript:;"><SPAN>(Neon lux pR:Safety catch lux pR)100:50</SPAN></A></LI><LI><A href="javascript:;"><SPAN>(Neon lux pR:Safety catch lux pR)100:100</SPAN></A></LI><LI><A href="javascript:;"><SPAN>(Neon lux pR:Safety catch lux pR)100:150</SPAN></A></LI><LI><A href="javascript:;"><SPAN>(Neon lux pR:Safety catch lux pR)150:50</SPAN></A></LI><LI><A href="javascript:;"><SPAN>(Neon lux pR:Safety catch lux pR)150:100</SPAN></A></LI><LI><A href="javascript:;"><SPAN>(Neon lux pR:Safety catch lux pR)150:150</SPAN></A></LI></UL></DIV><STRONG>3.3  Effect of lux pR leakage on Safety Catch plasmid</STRONG>
      The leakage of lux pR refers to the extent of transcription driven by lux pR promoters not bound by AHL-luxR complexes. The leakage of lux pR is anticipated to have impact on the system as well.
 The lux pR on Safety Catch plasmid serves as a vital “switch” that determines how easily the system can be activated. If it has a higher leakage level, the lacI expression driven by it should be stronger, thus inhibiting the expression gRNA, making the system more sensitive to AHL’s activation effect. On the other hand, lux pR with strictly low leakage level might result in a system difficult to induce. An optimal level of leakage of lux pR might be a key factor to building an optimal system. Herein, we use our model simulation to validate these thoughts.
As shown below, the AHL concentration is set to be 10-8M, which should not be suffient to induce the system. If we increase the leakage level of lux pR on safety catch, the system becomes active under this condition and GFP is produced readily. It seems that the more “leaky” the promoter is, the more easily can the system be induced.<DIV class="text-center"><STRONG>Figure 9     Impact of lux pR leakage (on Safety Catch Plasmid) on the Performance of the System</STRONG></DIV>
                       Based on the previous results, both the transcription efficiency and the leakage level of lux pR seems to be crucial to the performance of the system. An optimal combination of transcription efficiency and leakage level is required. Thus, we seek to obtain a better mutant form of lux pR. In the end, we developed lux pR-HS, an improved version of the promoter.<STRONG>Summary:
-	lux pR leakage on Safety Catch plasmid influences the “threshold” of the system.
    -	Both leakage and transcription efficiency when activated should be taken into consideration when building an optimal promoter.</STRONG><STRONG>3.4   Effect of IPTG</STRONG>
        IPTG can be added to the system to eliminate the effect of lacI. Addition of IPTG should result in increased production of guide RNA, making the system more difficult to induce. In other words, lacI helps to modulate the threshold of system activation and adjusts the rate of GFP synthesis. To validate this hypothesis, we run simulations of our model.</SPAN><H4 class="service-heading">4.Sensitivity Test and Robustness of the System</H4><SPAN class="name-heading"><STRONG>4.1 Overview</STRONG>
            What is robustness? Robustness means some properties of the system are able to remain the same under perturbations. In our case, if our model is &quot;robust&quot; to some extent, then perturbations of certain &quot;inputs&quot; of the model should cast little effect on the &quot;outputs&quot;. As for the &quot;inputs&quot;, they can be the initial concentrations of chemical species and biological parameters. As for the outputs, they are the simulated GFP concentrations at given time points.
First of all, we would like to determine whether the model is robust given perturbations in parameter values. theoretically, none of the values of these parameters can be exactly precise, so confirming that minor shifts from ideal values do not cause the result of simulation dramatically is naturally important. Robustness to parameter values is somehow closely related to &quot;sensitivity&quot; and &quot;sensitivity test&quot;.... The sensitivities of an ODE system are defined as the derivatives of the solution with respect to the parameters. We choose to perform Global Sensitivity Analysis, which is meant to be used for exploring the sensitivity over a larger domain without calculating derivatives.
 A scan of parameters as well as sensitivity test can be done using R, with the package ODEsensitivity. We primarily employ the &quot;Morris Method&quot; for global sensitivity analysis. Morris Method is also known as Morris' OAT Method, where OAT stands for &quot;One At a Time&quot;.
 Namely, when using Morris Method, one parameter value undergoes is changed during each run. Parameters change within a confined parameter space, while a sensitivity measure named &quot;the elementary effect&quot; is calculated during each run. <DIV class="text-center"><STRONG>$$[EE_i=\frac{f(x_1,x_2,...,x_i+\delta ,...,x_k)-y}{\delta}$$</STRONG></DIV>

 In the test, $\mu^*$, the mean of the elementary effects, is used to measure the influence of a parameter on the output. The larger it is, the more this parameter would influence the model. $\sigma$, the variance, is a measure of non-linearity and/or interaction effects of the parameter.<STRONG>4.2 Sensitivity Test of the Parameters</STRONG>
            What is robustness? Robustness means some properties of the system are able to remain the same under perturbations. In our case, if our model is &quot;robust&quot;  Here are some plots showing the elementary effects of several parameters. As expected, $\mu^*$ and $\sigma$ tend to increase over time.<DIV class="text-center"><STRONG>Figure 10     Results of Sensitivity Test. Two Parameters Have Especially Large Impacts on the System When Perturbed.</STRONG></DIV>
                These are just two pieces of snapshots of the results of a series of scans. Apparently, two parameters stand out to be much more sensitive than any other parameters. The lines that represent them are specially highlighted. These two parameters stand for “efficiency of lux pR promoter” and “binding affinity of luxR-AHL to lux pR”. Intriguingly, these two parameters involve lux pR, which is a primary interest in our project. In previous parts, we have already analyzed how lux pR efficiency greatly impacts our system. This accords with the conclusion drawn from the sensitivity test as well.
 Following numerous repetitive runs of sensitivity test, we confirmed the only those two parameters addressed above are significantly sensitive to perturbations. 
Based on this, we can say that our system is generally robust, as minor changes to the majority of the parameter values does not significantly affect the results. For now, our simulation results can be trusted to be useful.<STRONG>Summary:
-	The system is generally robust and stable according to our sensitivity analysis.
                  -	The parameters related to properties of lux pR are relatively more sensitive to perturbations. This accords with our previous conclusions.</STRONG></SPAN><SPAN class="text-muted"><STRONG>References</STRONG>
[1] Marc, W., &amp; Javier, B. (2013). Dynamics of the quorum sensing switch: stochastic and non-stationary effects. BMC Systems Biology,7,1(2013-01-16), 7(1), 6-6.
[2] James, S., Nilsson, P., James, G., Kjelleberg, S., &amp; Fagerström, T. (2000). Luminescence control in the marine bacterium vibrio fischeri: an analysis of the dynamics of lux regulation. Journal of Molecular Biology, 296(4), 1127-1137.
[3] Samuel E Clamons, Richard M Murray (2017) Modeling Dynamic Transcriptional Circuits with CRISPRi. BioRxiv
[4] Saeidi, N., Arshath, M., Chang, M. W., &amp; Poh, C. L. (2013). Characterization of a quorum sensing device for synthetic biology design: experimental and modeling validation. Chemical Engineering Science, 103(22), 91-99.
[5] Huynh, L., &amp; Tagkopoulos, I. (2016). A parts database with consensus parameter estimation for synthetic circuit design. Acs Synthetic Biology, 5(12).</SPAN></DIV></DIV></DIV></SECTION><FOOTER class="bg-light"><DIV class="container"><DIV class="row"><DIV class="col-md-4"><SPAN class="copyright">Copyright © Tsinghua iGEM 2018</SPAN></DIV></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>