<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Tsinghua-A_Achievements skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Tsinghua-A/Achievements</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="header-nav" class="header-nav"><DIV class="top-nav"><NAV><UL><LI><A href="https://2018.igem.org/Team:Tsinghua-A/Model#Introduction">Introduction</A></LI><LI><A href="https://2018.igem.org/Team:Tsinghua-A/Model#ReactionInSystem">Reaction in system</A></LI><LI><A href="https://2018.igem.org/Team:Tsinghua-A/Model#SignalPathwayInAYeast">Signal pathway in a yeast</A></LI><LI><A href="https://2018.igem.org/Team:Tsinghua-A/Model#DiagnosisModel">Diagnosis Model</A></LI><LI><A href="https://2018.igem.org/Team:Tsinghua-A/Model#Acknowledgement">Acknowledgement</A></LI></UL><A href="https://2018.igem.org/Team:Tsinghua-A/Model">Model</A><LI><A href="https://2018.igem.org/Team:Tsinghua-A/Background">Background</A></LI><LI><A href="https://2018.igem.org/Team:Tsinghua-A/Design">Design</A></LI><LI><A href="https://2018.igem.org/Team:Tsinghua-A/Experiments">Experiments</A></LI><LI><A href="https://2018.igem.org/Team:Tsinghua-A/Achievements">Achievements</A></LI><LI><A href="https://2018.igem.org/Team:Tsinghua-A/Improve">Improve</A></LI><LI><A href="https://2018.igem.org/Team:Tsinghua-A/Safety">Safety</A></LI><LI><A href="https://2018.igem.org/Team:Tsinghua-A/Protocol">Protocol</A></LI><A href="https://2018.igem.org/Team:Tsinghua-A/Background">Project</A><LI class="dropdown"><UL class="dropdown-menu"><P style=" margin: 0; padding: 3px 20px; text-align: left; line-height: 160%; font-size: 18px; text-transform: uppercase; font-weight: 700; ">Questionnaire</P></UL></LI><LI><A href="https://2018.igem.org/Team:Tsinghua-A/Human_Practices#Introduction">Introduction</A></LI><LI><A href="https://2018.igem.org/Team:Tsinghua-A/Human_Practices#Objective">Objective</A></LI><LI><A href="https://2018.igem.org/Team:Tsinghua-A/Human_Practices#Methodology">Methodology</A></LI><LI><A href="https://2018.igem.org/Team:Tsinghua-A/Human_Practices#Result">Result</A></LI><LI><A href="https://2018.igem.org/Team:Tsinghua-A/Human_Practices#Summary">Summary</A></LI><LI><A href="https://2018.igem.org/Team:Tsinghua-A/Human_Practices#Expert">experts' view</A></LI><LI><A href="https://2018.igem.org/Team:Tsinghua-A/Human_Practices#CCiC">CCiC</A></LI><LI><A href="https://2018.igem.org/Team:Tsinghua-A/Public_Engagement">Public Engagement</A></LI><A href="https://2018.igem.org/Team:Tsinghua-A/Human_Practices">Human Practices</A><LI><A href="https://2018.igem.org/Team:Tsinghua-A/TeamTsinghua">Team Tsinghua</A></LI><LI><A href="https://2018.igem.org/Team:Tsinghua-A/TeamBITCHINA">Team BIT-China</A></LI><A href="https://2018.igem.org/Team:Tsinghua-A/Collaborations">Collaborations</A><LI><A href="https://2018.igem.org/Team:Tsinghua-A/Hardware#Overview">Overview</A></LI><LI><A href="https://2018.igem.org/Team:Tsinghua-A/Hardware#DesignedStepsForUsage">Usage</A></LI><LI><A href="https://2018.igem.org/Team:Tsinghua-A/Hardware#PerformanceTesting">Performance Testing</A></LI><LI><A href="https://2018.igem.org/Team:Tsinghua-A/Hardware#DesignAndComponents">Design</A></LI><LI><A href="https://2018.igem.org/Team:Tsinghua-A/Hardware#Manufacture">Manufacture</A></LI><LI><A href="https://2018.igem.org/Team:Tsinghua-A/Hardware#FuturePlan">Future Plan</A></LI><A href="https://2018.igem.org/Team:Tsinghua-A/Hardware">Hardware</A><LI><A href="https://2018.igem.org/Team:Tsinghua-A/Demonstrate">Demonstrate</A></LI><LI><A href="https://2018.igem.org/Team:Tsinghua-A/Attributions">Attributions</A></LI><LI><A href="https://2018.igem.org/Team:Tsinghua-A/Team">Team</A></LI></NAV></DIV></DIV><DIV class="content"><DIV class="container"><DIV class="page-head"><DIV class="head-title">
                        Achievements
                    </DIV><DIV class="head-subtitle">
                        Results of experiments
                    </DIV></DIV><DIV class="page-content"><H2>Modification of the yeast</H2><H3>Gene knockout</H3><P>Based on previous research, three genes are recommended to be knocked out. </P><P><I>Ste2</I>, the receptor of α factor, in order to eliminate the probability of yeasts getting affected by native ligand-pheromone. </P><P><I>Sst2</I>, the GTPase-activating protein for Gpa1p, as a receptor-desensitizing factor in order to eliminate possible up-stream GPCR signal conductive decline due to incessant stimulation. </P><P><I>Far1</I>, the cyclin-dependent protein serine/threonine kinase inhibitor, in order to eliminate side effect to cells’ life circle and formation due to signal intensification in MAPK pathway of the pheromone pathway. </P><P>Fortunately, we found that BIT-China 2017 used the same pheromone pathway in yeast in their project last year, so we established cooperation with them and brought their yeast-- <I>CEN.PK2-1c ura3::far1 his2::sst2 trp3::ste2 △leu2</I>. It greatly reduced our experiment works. </P><H3>Mutation screening</H3><P>Judging that the auxotroph marker is not enough for subsequent experiment, and resistance markers have a few problems such as plasmids’ easy losing in<I> CENPK2-1C ura3::far1 his2::sst2 trp3::ste2 △leu2</I> Saccharomyces cerevisiae, we used positive 5-fluoro-orotic acid (5-FOA) selection to obtain Ura3 mutant strains.In this method, Wild-type strains of yeast (or ura3 mutant strains containing a plasmid-borne URA3+ gene) are unable to grow on medium containing the pyrimidine analog 5-fluoro-orotic acid, whereas ura3- mutants grow normally [1]. </P><P><I>CEN.PK2-1c ura3::far1 his2::sst2 trp3::ste2 △leu</I> Saccharomyces cerevisiae were plating on the <I>ura+, his-, trp-, leu+</I> solid medium containing 5-FOA, monoclonal was obtained and enlarge cultivated. Then plated the yeast on <I>ura-, his-, trp-, leu+ </I>solid medium, no colony can be observed to survive on it. For further confirm, the colony’s genome was sent for 
sequencing, mutation existed in Ura3 gene. We considered that we had gained <I>CEN.PK2-1c ura3::far1 his2::sst2 trp3::ste2 △leu2 △ura3 </I>Saccharomyces cerevisiae. </P><DIV class="img"><DIV class="imgcard"><P> Fig.1 The screening result. Left: no colony appears on the SD-ura culture medium. Right: colonies could grow on FOA+ culture medium


</P></DIV></DIV><H2>Plasmid construction</H2><P>In our project, these plasmids are required. </P><H3>Receptor plasmid </H3><P>Coding sequencing of Human histamine receptor (HRH4) was got from plasmid pH4R-mCherry-N1. Separately, HRH4 CDS and HRH4-mCherry fusion sequencing were obtained by PCR. We chose pGADT7-AD (Leu2 marker) and use HindIII to remove SV40-NLS sequencing, GAL4 activation domain and HA tag. Then insert HRH4 or HRH4-mCherry between ADH1 promoter and ADH1 terminator. </P><DIV class="img"><DIV class="imgcard"><P> Fig.2 The insertion of HRH4 or HRH4-mCherry into pGADT7-AD plasmid
</P></DIV></DIV><P>pGADT7-HRH4-mCherry was used to express HRH4-mCherry fusion protein to show the expression and localization of HRH4 in yeast. pGADT7-HRH4 was used to express functional HRH4. </P><H3> Reporter </H3><P>We used part: BBa_K775004 as our reporter, because it was not released, so we used PCR to get Fus1 promoter from yeast genome, EGFP and ADH1 terminator from other plasmids, then used Gibson assembly it with EGFP and inserted into plasmid pYSE2.0 digested with SpeI and EcoRI (Ura3 marker). </P><DIV class="img"><DIV class="imgcard"><P> Fig.3 Gibson Assembly fus1 promoter, EGFP and ADH1 terminator into pGADT7-AD plasmid
</P></DIV></DIV><H3> Transformation of the yeast</H3><P>pGADT7-HRH4-mcherry was transformed into<I> CEN.PK2-1c ura3::far1 his2::sst2 trp3::ste2 △leu2 △ura3</I> Saccharomyces cerevisiae to test the expression and localization of HRH4-mcherry.   </P><P>pYES2.0-pfua1-EGFP was transformed into original CEN.PK2-1c △his2 △trp3 △leu2 △ura3   Saccharomyces cerevisiae to test the reporter. </P><P>Both pGADT7-HRH4 and pYES2.0-pfua1-EGFP were transformed into <I>CEN.PK2-1c ura3::far1 his2::sst2 trp3::ste2 △leu2 △ura3 </I>Saccharomyces cerevisiae to consist our complete yeast for histamine detection. For confirming both of these two plasmids were transformed successfully, colony PCR was performing to detect the existence of these two plasmids. </P><DIV class="img"><DIV class="imgcard"><P> Fig.4 The co-transformed yeast grow on SD-His,Trp,Leu,Ura culture medium
</P></DIV></DIV><DIV class="img"><DIV class="imgcard"><P> Fig.5 The colony PCR result of co-transformation

</P></DIV></DIV><H2>Test and Result</H2><P>To insure our system can work as we expected, we did a serious of test. </P><H3> HRH4-mCherry expression and localization</H3><P>After transforming the <I>CEN.PK2-1c ura3::far1 his2::sst2 trp3::ste2 △ura3 </I>Saccharomyces cerevisiae with plasmid pGADT7 contains part BBa_K2583001, 30℃ cultivate on △leu solid medium for one day. Select monoclonal and enlarge culture for several days. We chose to feature the expression of H4 at a relatively stable stage to make the result more referable for the modeling analysis. </P><H4> Location observation by fluorescence microscope</H4><P>We observed the fluorescence under the microscope with 580nm exciting light. The red fluorescence shows the position of HRH4-mCherry fusion protein. </P><DIV class="img"><DIV class="imgcard"><P> Fig.2 the fluorescence microscopy result of HRH4-mCherry

</P></DIV></DIV><P>The location of HRH4 indicates that our pioneering work to try to produce Histamine receptor H4 from Homo sapiens in Saccharomyces cerevisiae was basically a success. Because the function of HRH4 requires at least that the protein should be sent to the membrane.  </P><DIV class="img"><DIV class="imgcard"><P> Fig.3 the fluorescence microscopy result of DiO dyeing (showing the plasma membrane) and HRH4-mCherry


</P></DIV></DIV><H4> Quantification by Flow CytoMetry (FCM)</H4><P>From the result of fluorescence microscopy, the expression of HRH4-mCherry fusion proteins can be observed roughly, for quantifying the expression level of HRH4-mCherry, FCM is performed. From the result, a distinct peak value can be observed. Considering the fluorescence microscopy result that most of HRH4 are correctly located, this data does provide us a reliable probability distribution of total H4 protein expression variance between yeast individuals. Finally, the probability distribution of functional H4 receptor expression variance can refer to the total expression probability distribution we get from experiments. </P><DIV class="img"><DIV class="imgcard"><P> Fig.4 Flow Cytometry result of HRH4-mCherry fusion protein expression


</P></DIV></DIV><P>The test of this composite part set the foundation of our project, it proves that HRH4 can be expressed in high level under the trigger of a constitutive promoter-ADH1 promoter. Besides, the HRH4 can locate in the yeasts’ membrane without adding other sequence. It can be part of the evidence that our HRH4 is functional. Then the Flow CytoMetry result provide us a reliable probability distribution of total H4 protein expression variance between yeast individuals, which is significant for modeling. </P><H3> The induce of the reporter </H3><P>To test the ability of responding to upstream pathway, we try to provide valid upstream signal to reporter gene. So we transform the CEN.PK2-1c △ura3 △leu △his △trp Saccharomyces cerevisiae with plasmid pYES2.0 contains part BBa_ K775004, 30℃ cultivate on SD-ura solid medium for one day. Select monoclonal and enlarge culture for several days. And then, several experimental groups are set to get CEN.PK2-1c stimulated by α factors. 1ml of modified yeast is centrifuged at 5000 g for 2 min as well as 2 ml of MATα yeast, then supernatant of modified yeast is discarded and 2 ml of MATα yeast supernatant is added to culture yeast. </P><H4> EGFP expression, Chemotropic Growth and Mating Differentiation observation by fluorescence microscope every hour since the addition of MATα yeast supernatant.</H4><DIV class="img"><DIV class="imgcard"><P> Fig.5 The fluorescence microscopy result of reporter gene, triggered by α factor in MATα yeast supernatant.



</P></DIV></DIV><P>Compared with the Non-Induction group, the EGFP intensity inside the yeast gets more and more stronger from the 1h group to 5h group. And, more and more yeast cells in sight finish the process of mating: stretching out mating projection or can say “shmoo,” and then contacting and fusing between mating partners. </P><P>But when time goes to 6h, we can see that, basically the EGFP protein particles in sight are all distributed outside the yeast cell. We suppose that those EGFP protein particles are excreted by yeast cells for preparing to enter ascospore period. </P><H4> Time scale of EGFP expression upon induction of α factor</H4><P>Similar to the experiment above, we set different culture group and time to measure the time scale of EGFP expression. The sample order is as follow: </P><DIV class="img"><DIV class="imgcard"><P> Table 1. The set of experiment group</P></DIV></DIV><DIV class="img"><DIV class="imgcard"><P> Fig.6 Fig.6a to Fig.6i demonstrate the FCS result of yeast culturing from 9h to 1h.
</P></DIV></DIV><P>As can be seen from Fig.5 and Fig.6, we can see clear distribution between the induction group and non-induction group. As for the expression time scale, the supernatant induction group reached the peak at 6h while the MATα yeast induction group maintain at a rather high level starting from 3h. </P><DIV class="img"><DIV class="imgcard"><P> Fig.7 The average EGFP intensity of each groups.

</P></DIV></DIV><P>The above result indicate that the reporter plasmid could function as expected by responding to upstream signal in pheromone pathway, thus could be used to modify yeast to target histamine. </P><H3> The induce of modified yeast with histamine</H3><P>After successfully transformed the plasmid ADH1-HRH4-Ter and pFUS1-EGFP into <I>CEN.PK2-1c ura3::far1 his2::sst2 trp3::ste2 △ura3 </I>Saccharomyces cerevisiae, we had already finished the modification process of yeast. Now it’s time to test whether our diagnostic yeast works! Based on our previous design, the modified yeast could respond to different ratio of histamine in environment at corresponding EGFP expression level. At meantime, the protein expression rate is also influence by culturing time after first induction. Due to the modelling, the higher the histamine concentration is, the faster yeast will respond and so is the protein degradation rate. So we made a prediction that the EGFP expression rate would positively correspond to histamine concentration in a certain time period after histamine induction. </P><P>Based on the above assumptions, we chose to induce the modified yeast already cultured to logarithm phase in medium lacking His, Trp, Leu and Ura. Then a histamine concentration gradient was made by adding a diverse range of histamine solution to the culture medium. According to medical research, the histamine concentration in normal human blood is around 30 ng/ml to 60ng/ml, while in allergic people, this concentration would reach 100 ng/ml or even more. We set three group of histamine concentration: 0 ng/ml as control group, 50 ng/ml to simulate the normal concentration and 200 ng/ml to simulate allergic concentration. </P><P>According to the test of reporter plasmid, we estimated the expression peak would be between 5 to 10 hours, so we choose 5h, 8h and 11h as three time points to run flow cytometry. </P><P>One thing worth mentioning is that due to the low concentration of yeast, we could only set 10 group so as to meet the criterion concentration of FCS. </P><P>The group set is as follow: </P><DIV class="img"><DIV class="imgcard"><P> Table 2. The sample loading order


</P></DIV></DIV><P>The concentration of yeast is around 100000 cell/ml (measured by OD600). We pick 10000 events inside the gate for each sample during FCS. </P><P>The FCS result is as follow: </P><DIV class="img"><DIV class="imgcard"><P> Fig.8 The relative fluorescence intensity of histamine induced yeast

</P></DIV></DIV><P>From Fig.8, we can see that at 5 to 8 hours’ culturing, the EGFP expression rate is positively correspond to histamine which certify our theory. When we judge each concentration group separately, it can be assumed that 50 ng/ml and 200 ng/ml group both reach its peak during 5 to 11 hours’ period. The degradation rate of each group is also positive correspond with histamine concentration, since at 11h, the expression level of every induction group has fallen, and the expression rate appears to be rather negative correlated to histamine concentration. </P><P>Considering the fact that the reaction time we had estimated for our allergy test kit is around 4 to 5 hours, it turned out that the modified yeast could indeed distinguish between allergic state blood serum and normal blood serum. </P><H3> Growth curve of yeast in blood serum</H3><P>In order to certify the feasibility of yeast maintaining its previous cell activity while reacting with histamine in blood serum, rather than become incompetent or being precipitated by antigens such as IgG in the environment, we carried out a series of experiment by culturing modified yeast in different ratio of blood serum. The ability of proliferation as well as cell activity are revealed by determination of growth curve in different groups. </P><P>One thing worth mentioning is that the blood serum (BS) we use is derived from mice with no immunogenicity. After extracted from mice, the product simply went through the procedure of precipitation and bottled without heat inactivation. Control group was set by filter sterilized half of the blood serum (FBS) to remove large protein and potential bacteria (. This procedure aims to judge whether the complement protein would influence the growth of yeast. </P><P>The experiment is designed as follows. </P><DIV class="img"><DIV class="imgcard"><P> Table 3. The sample loading form


</P></DIV></DIV><P>Due to the fact that normally yeast would enter logarithmic growth phase in 4 to 8 hours, we choose 1~7 h as the 1st stage measure point and 18~19h as the 2nd stage measure point to judge whether the yeast has entered plateau phase. The density of yeast is measured by OD600. </P><DIV class="img"><DIV class="imgcard"><P> Fig.9 The growth curve of yeast cultured in YPD</P><P>E: 10% BS+YPD  F: 10% FBS+YPD  I: 1% BS+YPD  J: 1% FBS+YPD
</P></DIV></DIV><P>As can be seen from Fig.9, the addition of blood serum didn’t influence the growth of yeast significantly. </P><DIV class="img"><DIV class="imgcard"><P> Fig.10 The growth curve of yeast cultured in water
</P><P> C: 10% BS+ water  D: 10% FBS+ water  G: 1% BS+ water  H: 1% FBS+ water
</P></DIV></DIV><P>On the contrast, the addition of blood serum did enhance the proliferation of yeast when comparing the curve of H and G with C and D. What’s more, comparing the curve of adding the original blood serum or sterilized one, we can see that the original blood help yeast to grow better without any precipitation. </P><DIV class="img"><DIV class="imgcard"><P> Fig.11 The growth curve of yeast cultured in pure blood serum
</P><P> A: 100% BS  B: 100% FBS
</P></DIV></DIV><P>Compared the curve of original blood serum culturing and the sterilized one, it is obvious that the yeast cultured in original product outgrew the sterilized group in the first few hours. Though the two group reach the same level after 19 hours culturing, we could still judge the original blood serum to be better since the reaction time we had estimated for our ATM testing box is around 4 to 5 hours ---- still within the dominant period of original blood serum culturing. </P><DIV class="img"><DIV class="imgcard"><P> Fig.12 The growth curve of yeast

</P><P> A: 100% BS  B: 100% FBS</P><P> C: 10% BS+ water  D: 10% FBS+ water  G: 1% BS+ water  H: 1% FBS+ water</P><P> E: 10% BS+YPD  F: 10% FBS+YPD  I: 1% BS+YPD  J: 1% FBS+YPD
</P></DIV></DIV><P>The overall graphic shows that the growth rate of yeast in YPD is faster than water, while during the estimated reaction time, the whole blood serum group also outgrew the water group. The above results suggest that blood serum won’t disturb the proliferation and cell activity of yeast, and even enhance them in water environment. The dominance of original blood serum leads to convenience that users don’t need to filter or heat inactivate complement protein to avoid precipitation. </P><P>In conclusion, the modified yeast could survive in blood serum environment provided by users which is the testing environment in our design normally, without any need of further processing. </P><H2>Future Work </H2><H3> Gα modification </H3><P>Because human G protein coupled receptors (GPCRs) can‘t improperly couple with α subunit of yeasts’ G protein, five carboxy-terminal amino acid residues of the guanine nucleotide binding protein a-subunit (Ga) in yeast Gpa1 can be replaced by human Gα[2]. Human histamine H4 receptor is coupled to Gαi/o proteins [3], so we got the sequence of last five amino acid residues of Gi1 and added it to the primer. Then we used PCR and Gibson assembly to constitute fragment for homologous recombination. Ura3 marker will be recovered after the whole fragment is integrated into yeast genome [4].</P><DIV class="img"><DIV class="imgcard"><P> Figure. Homologous combination fragment for modification of GPA1

</P></DIV></DIV><P>The part: BBa_K2583003 has been made for homologous recombination, but we haven’t done the transformation. In future, we will continue this trying. </P><H3>Integrating the reporter into the genome</H3><P>We also made an assumption that using EGFP to do homologous recombination with Fus1 behind fus1 promoter in yeast genome, the EGFP would be integrated into the genome and eliminate the differences of the copy number between each yeast. What’s more, the reporter gene will not be lost and can be inherited steadily. Besides, more promoter can be tried, such as fus2, fig1 promoter [5], from them we can compare their intensity and choose an appropriate one.</P><H2>Reference</H2><OL><LI>Boeke JD, LaCroute F, Fink GR. 1984. A positive selection for mutants lacking orotidine-5′-phosphate decarboxylase activity in yeast: 5-fluoro-orotic acid resistance. Molecular &amp; general genetics MGG 197: 345–346.</LI><LI>Ishii J, Oda A, Togawa S, et al. Microbial fluorescence sensing for human neurotensin receptor type 1 using Gα-engineered yeast cells[J]. Analytical Biochemistry, 2014, 446:37-43.</LI><LI> Thangam EB, Jemima EA, Singh H, et al. The Role of Histamine and Histamine Receptors in Mast Cell-Mediated Allergy and Inflammation: The Hunt for New Therapeutic Targets. Frontiers in Immunology. 2018;9:1873. doi:10.3389/fimmu.2018.01873.</LI><LI> Hirashima K, Iwaki T, Takegawa K, Giga-Hama Y, Tohda H. 2006. A simple and effective chromosome modification method for large-scale deletion of genome sequences and identification of essential genes in fission yeast. Nucleic Acids Res 34: e11</LI><LI> Tehseen M, Dumancic M, Briggs L, et al. Functional Coupling of a Nematode Chemoreceptor to the Yeast Pheromone Response Pathway[J]. Plos One, 2014, 9(11): e111429.
e11</LI></OL></DIV></DIV></DIV><DIV class="footer pagefooter"><DIV class="footer-right-col"><P> Contact us </P><UL class="contact-info"><LI>Mentor E-mail: <A href="mailto:au_wang@foxmail.com" id="mailto">au_wang@foxmail.com</A></LI><LI> E-mail: <A href="mailto:igem2018_thua@163.com" id="mailto">igem2018_thua@163.com</A></LI><LI> Phone: +86 010-62796050 </LI><LI> WeChat: yqyntx98 </LI></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>