<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Tsinghua_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Tsinghua/Results</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Neon Coli-Necessary Expression Only</TITLE></P><DIV class="container"><DIV class="collapse navbar-collapse" id="navbarResponsive"><UL id="menu" class="navbar-nav  ml-auto"><LI class="nav-item"><A class="nav-link text-uppercase js-scroll-trigger" href="https://2018.igem.org/Team:Tsinghua">Home</A></LI><LI class="nav-item"><A href="#Team" class="nav-link text-uppercase js-scroll-trigger">
                Team </A></LI><LI class=" nav-item dropdown"><A href="#" class="nav-link text-uppercase dropdown-toggle" data-toggle="dropdown">
                Project </A></LI><LI><A href="https://2018.igem.org/Team:Tsinghua/Description">Description</A></LI><LI><A href="https://2018.igem.org/Team:Tsinghua/Design">Design</A></LI><LI><A href="https://2018.igem.org/Team:Tsinghua/Experiments">Experiments</A></LI><LI><A href="https://2018.igem.org/Team:Tsinghua/Results">Results</A></LI><LI><A href="https://2018.igem.org/Team:Tsinghua/Model">Model</A></LI><LI><A href="https://2018.igem.org/Team:Tsinghua/Notebook">Notebook</A></LI><LI><A href="https://2018.igem.org/Team:Tsinghua/Attributions">Attributions</A></LI></UL><LI class=" nav-item dropdown"><A href="#" class="nav-link text-uppercase dropdown-toggle" data-toggle="dropdown">
                Parts </A></LI><LI><A href="https://2018.igem.org/Team:Tsinghua/Basic_Part">Basic Parts</A></LI><LI><A href="https://2018.igem.org/Team:Tsinghua/Composite_Part">Composite Parts</A></LI><LI><A href="https://2018.igem.org/Team:Tsinghua/Improve">Improve</A></LI><LI class=" nav-item dropdown"><A href="#" class="nav-link text-uppercase dropdown-toggle" data-toggle="dropdown">
                HP </A></LI><LI><A href="https://2018.igem.org/Team:Tsinghua/Human_Practices">Overview</A></LI><LI><A href="https://2018.igem.org/Team:Tsinghua/GHP">General HP</A></LI><LI><A href="https://2018.igem.org/Team:Tsinghua/IHP">Intergrated HP</A></LI><LI><A href="https://2018.igem.org/Team:Tsinghua/Safety">Safety</A></LI><LI class="nav-item"><A href="https://2018.igem.org/Team:Tsinghua/Collaborations" class="nav-link text-uppercase js-scroll-trigger">
                Collaboration</A></LI></DIV></DIV><SECTION id="HP"><DIV class="container"><DIV class="row"><DIV class="col-lg-12 text-center"><H2 class="section-heading text-uppercase">Results</H2></DIV></DIV><DIV class="row text-left"><DIV class="col-md-12 keith"><H4 class="name-heading">Lux pR Mutation Analysis</H4><H5 class="name-heading">&gt;&gt;&gt;&gt;Design</H5><SPAN class="">With the hope to find an optimal promotor for our NEON system, we designed 9 mutations on sites -35, -36, -37 near the luxR binding site and tested another on -10 site that TUST 2017 reported to show decreased leakage. We conducted experiments to evaluate these lux pR mutants’ reaction to AHL stimulation and their leakage level. The test devices we designed include a constantly expressed luxR and a lux pR (or mutant) driven sfGFP (like <A href="http://parts.igem.org/part:BBa_K2558211">BBa_K2558211</A> with original lux pR promotor, and <A href="http://parts.igem.org/part:BBa_K2558212">BBa_K2558212</A> with lux pR-HS promotor). 
                           </SPAN><H5 class="name-heading">&gt;&gt;&gt;&gt;Results</H5><SPAN class="">We conducted the experiment first with 9 mutations on sites -35, -36, -37 near the luxR binding and another on -10 site that TUST 2017 reported to show decreased leakage. We transferred the plasmids with lux pR driven sfGFP into <I>E. coli</I> DH5α. In theory, AHL can induce gene expression by activating lux pR promotor. We measured the value of Fluorescence/OD without AHL stimulation by microplate reader to show the leakages of different lux pR mutants, as lux pR promotor is not supposed to be activated without AHL. Meanwhile, we also regard the value of Fluorescence/OD with 10<SUP>-10</SUP> M AHL stimulation to observe the sensitivity and expression intensity of different lux pR mutants. Experiment data is shown in the figures below. (Figure.1)
                           </SPAN><DIV class="text-center"><STRONG>Figure.1. Leakage and 10<SUP>-10</SUP> M AHL stimulation of lux pR promotor and its mutations. </STRONG></DIV><SPAN class="text-muted">The fluorescence strengths and OD values are measured by microplate reader respectively at 510 nm and 600 nm wavelength. Plasmids with lux pR promotor and its mutants are transferred into <I>E.coli</I> DH5α. <STRONG>A</STRONG> The bacteria were treated without AHL inducing. High (Fluorescence/OD) ratio indicates strong leakage of promotor. Among all the strains, lux pR promotor has the strongest leakage and the largest error bar as well. <STRONG>B</STRONG> The bacteria were treated with 10<SUP>-10</SUP> M AHL induction. High (Fluorescence/OD) ratio indicates high expression intensity. Mutation 5 has almost the same expression intensity as the wild type lux pR promotor.</SPAN>

It is observed that high expression intensity is accompanied by high level of leakage in lux pR promotors. Wild type lux pR represented with black strand in the figures has the highest level of both leakage and intensity. The next three are Mutant 5 (G-36T), Mutant 6 (G-36C), Mutant 9 (T-35G). Mutant 5 has almost the same intensity as wild type, but the leakage is significantly smaller. Mutant 6 and Mutant 9 have a little bit less leakage and medium intensity. We also did the experiment with 10<SUP>-9</SUP> M and 10<SUP>-8</SUP> M AHL, all of the promotors show high expression levels and therefore unable to compare. (Data not shown)

        <DIV class="text-center"><STRONG>Figure.2. Leakage and AHL stimulation of lux pR Mutant 5 and Mutant 10.  </STRONG></DIV><SPAN class="text-muted">fluorescence strengths and OD values are measured by microplate reader respectively at 510 nm and 600 nm wavelength. <STRONG>A</STRONG> Plasmid with lux pR Mutant 5 is transferred into <I>E.coli</I> DH5α. The bacteria were treated without or with 10<SUP>-10</SUP>, 10<SUP>-9</SUP>, 10<SUP>-8</SUP> M AHL inducing. <STRONG>B</STRONG> Plasmid with lux pR Mutant 10 is transferred into E.coli DH5α. The bacteria were treated without or with 10<SUP>-10</SUP>, 10<SUP>-9</SUP>, 10<SUP>-8</SUP> M AHL inducing.</SPAN><SPAN>We repeated AHL stimulation test on Mutant 5 and Mutant 10 (Figure.2) and decided to combine the two mutations together to form a new promotor with high intensity and small leakage. We named it lux pR-HS. We tested lux pR-HS under the same conditions. We can see that the lux pR-HS has similar leakage to Mutant 10, while it has much higher expression intensity. (Figure.3) The statistics of relative induction and leakage intensity of the 10 Mutants and lux pR-HS are shown in Table.1</SPAN><DIV class="text-center"><STRONG>Figure.3. Leakage and 10<SUP>-10</SUP> M AHL stimulation of lux pR promotor WT, Mutant 5, Mutant 10 and HS.</STRONG></DIV><SPAN class="text-muted">The fluorescence strengths and OD values are measured by microplate reader respectively at 510 nm and 600 nm wavelength. Plasmids with lux pR promotor WT, Mutant 5 and Mutant 10 are transferred into <I>E.coli</I> DH5α. A The bacteria were treated without AHL inducing. High (Fluorescence/OD) ratio indicates strong leakage of promotor. B The bacteria were treated with 10<SUP>-10</SUP> M AHL induction. High (Fluorescence/OD) ratio indicates high expression intensity.</SPAN><DIV class="text-center"><STRONG>Table.1. Results of  lux pR mutation analysis. </STRONG></DIV><SPAN class="">As for why some mutations on -35 to -37 sites would decrease the leakage of lux pR without severely influencing the expression level, we have some hypotheses and explanations. It is reported that -35 to -37 sites are last three bases of lux box which is upstream of the lux promotor and bound specifically by luxR, one of regulatory protein involved in lux expression system.<SUP>[1]</SUP> G-36T mutation has less influence than other mutation on sites -35 to -37, which is same to our result. Besides, G-36T is not involved in the two regions of nucleotides -52 to -50 and -39 to -37 which directly contacted with luxR protein. Therefore, this single substitution of nucleotide would decrease the leakage of lux promotor without influencing the expression level too much.<SUP>[2]</SUP></SPAN><H5 class="name-heading">&gt;&gt;&gt;&gt;Protocol</H5><SPAN class="">1.	Transform the plasmids into <I>E. coli</I> DH5α.
2.	Pick a single colony by a sterile tip from each of the LB plates for all the experimental and control groups. Add the colony into 5ml LB medium with ampicillin at 100 ng/µl. Incubate for 6-8 h at 37℃ in a shaker. 
3.	Measure OD600 of the culture medium with photometer. Dilute the culture medium until OD600 reaches 0.6.
4.	Add 100 µl bacteria culture medium into a sterile 96-well plate. Add AHL to final concentrations of 0, 10<SUP>-10</SUP>, 10<SUP>-9</SUP>, 10<SUP>-8</SUP> M. Fresh LB medium serves as blank control. Positive control is colony constantly expressing sfGFP and negative control is colony without sfGFP expression. Place the 96-well plate into an automatic microplate reader. Incubate at 16℃ overnight and measure the fluorometric value at 510 nm and OD600 of each well every 30 minutes.
5.	Each group should be repeated for at least 3 times.</SPAN><SPAN class="text-muted">[1] Antunes, L. C., et al. &quot;A mutational analysis defines Vibrio fischeri LuxR binding sites.&quot; Journal of Bacteriology 190.13(2008):4392-4397.
[2] Zeng, Weiqian, et al. &quot;Rational design of an ultrasensitive quorum-sensing switch.&quot; Acs Synthetic Biology 6.8(2017).</SPAN><H4 class="name-heading">IPTG Induction and lacI Dosage</H4><H5 class="name-heading">&gt;&gt;&gt;&gt;Design</H5><SPAN class="">In order to investigate how lacI dosage affects IPTG induction, we used Anderson promotor J23100, J23110 and J23114 to design three constitutive lacI generators of different intensities. The three lacI generators were then ligated with Ptac driven reporter sfGFP to make three IPTG induction devices (<A href="http://parts.igem.org/part:BBa_K2558203">BBa_K2558203</A>,<A href="http://parts.igem.org/part:BBa_K2558204">BBa_K2558204</A>,<A href="http://parts.igem.org/part:BBa_K2558205">BBa_K2558205</A>). By measuring sfGFP fluorescence we tested how these devices react to IPTG.</SPAN><H5 class="name-heading">&gt;&gt;&gt;&gt;Results</H5><DIV class="text-center"><STRONG>Figure.4. The effect of varied lacI-LVA (<A href="http://parts.igem.org/part:BBa_C0011">BBa_C0011</A>) promotor strength on IPTG induction of Ptac (<A href="http://parts.igem.org/part:BBa_K2558004">BBa_K2558004</A>).</STRONG></DIV><SPAN class="text-muted">Relative fluorescent intensity is fluorescence per OD600 standardized with fluorescence per OD600 value of each test group at Time=0, IPTG=0. The promotors we used are from the Anderson collection: BBa_J23100 for strong lacI-LVA expression (pink), BBa_J23110 for medium expression (green), BBa_J23114 for weak expression (orange).</SPAN><SPAN class="">With high level of lacI expression (<A href="http://parts.igem.org/part:BBa_K2558203">BBa_K2558203</A>), sfGFP fluorescence had almost no response to IPTG induction. Weak lacI expression (<A href="http://parts.igem.org/part:BBa_K2558205">BBa_K2558205</A>) had the most significant IPTG induced sfGFP expression. With medium lacI expression level (<A href="http://parts.igem.org/part:BBa_K2558204">BBa_K2558204</A>), the induction efficiency lay in between. Therefore, the result proves that high level of lacI expression severely decreases IPTG induction efficiency <SUP>[1]</SUP>. Furthermore, IPTG concentration can affect the regulation part performance. The figure shows that without IPTG the sfGFP florescence intensity remained low. After IPTG addition, fluorescence signal immediately began to climb, forming a peak at five hours after induction, then sfGFP florescence intensity decreased and maintained at a lower level afterwards. IPTG concentration did not significantly affect the height of the peak or the expression level after the peak, but rather the peak width and expression stability of the system. Figures indicate that 5-10 mM IPTG had the most stable induction results.<H5 class="name-heading">&gt;&gt;&gt;&gt;Protocol</H5><SPAN class="">1.	Transform the plasmids into <I>E. coli</I> DH5α.
2.	Pick a single colony by a sterile tip from each of the LB plates for all the experimental and control groups. Add the colony into 5ml M9 medium with ampicillin at 100 ng/µl. Incubate for 6-8 h at 37℃ in a shaker. 
3.	Measure OD600 of the culture medium with photometer. Dilute the culture medium until OD600 reaches 0.6.
4.	Add 100 µl bacteria culture medium into a sterile 96-well plate. Add IPTG to final concentrations of 0, 1, 5, 10, 20 mM. Fresh M9 medium serves as blank control. Positive control is colony constantly expressing sfGFP and negative control is colony without sfGFP expression. Place the 96-well plate into an automatic microplate reader. Incubate at 16℃ overnight and record the fluorometric value at 510 nm and OD600 for each well every 30 minutes.
5.	Each group should be repeated for at least 3 times.</SPAN><SPAN class="text-muted">[1] Szabolcs Semsey, Sandeep Krishna. &quot;The effect of LacI autoregulation on the performance of the lactose utilization system in Escherichia coli&quot; Nucleic Acids Res 2013 Jul; 41(13): 6381–6390</SPAN></SPAN><H4 class="name-heading">IPTG Induced CRIPSRi</H4><H5 class="name-heading">&gt;&gt;&gt;&gt;Design</H5><SPAN class="">After we clarified the correlation between lacI dosage and its induction efficiency, we furthered the experiments by adding CRISPRi to the system. Instead of sfGFP, the IPTG induction devices with different levels of lacI generator is now going to produce a gRNA that targets the promotor of a constitutive sfGFP generator (<A href="http://parts.igem.org/part:BBa_K2558206">BBa_K2558206</A>, <A href="http://parts.igem.org/part:BBa_K2558207">BBa_K2558207</A>, <A href="http://parts.igem.org/part:BBa_K2558208">BBa_K2558208</A>). By adding IPTG, we can induce the transcription of gRNA. Binding with constantly expressed dCas9, gRNA is going to inhibit the expression of sfGFP. We will be able to observe how this process can operate under different levels of lacI expression.</SPAN><H5 class="name-heading">&gt;&gt;&gt;&gt;Results</H5><DIV class="text-center"><STRONG>Figure.5. The effect of varied lacI-LVA (<A href="http://parts.igem.org/part:BBa_C0011">BBa_C0011</A>) promotor strength on IPTG induction of CRISPRi.</STRONG></DIV><SPAN class="text-muted"> The Y axis values are fluorescence per OD600 standardized with fluorescence per OD600 value of each test group at Time=0, IPTG=0 M. The promotors we used are from the Anderson collection: BBa_J23110 for medium expression (orange), <A href="http://parts.igem.org/part:BBa_J23114">BBa_J23114</A> for weak expression (blue).</SPAN><SPAN class="">In this experiment, the expression of sfGFP was inhibited by IPTG induced CRISPRi system. (Figure.5) Without CRISPRi the Fluorescence/OD600 increased steadily. With medium lacI expression, the addition of IPTG caused the fluorescence intensity to decrease at three hours after addition. With different IPTG concentration the final sfGFP fluorescence intensity remained approximately the same, which is consistent with previous IPTG induction results. However, with weak lacI expression, sfGFP expression was inhibited from the beginning. With the direct IPTG induction experiment we performed before, we hypothesize that with the weak Anderson promotor, basal gRNA expression will be above the threshold that is required for suppressing the sfGFP expression.<SUP>[1]</SUP><H5 class="name-heading">&gt;&gt;&gt;&gt;Protocol</H5><SPAN class="">1.	 Transform the plasmids into <I>E. coli</I> DH5α.
            2.	 Pick a single colony by a sterile tip from each of the LB plates for all the experimental and control groups. Add the colony into 5ml M9 medium with ampicillin at 100 ng/µl and chloramphenicol at 34 ng/µl. Incubate for 6-8 h at 37℃ in a shaker.
            3.	Measure OD600 of the culture medium with photometer. Dilute the culture medium until OD600 reaches 0.6.
            4.	Add 100 µl bacteria culture medium into a sterile 96-well plate. Add IPTG to final concentrations of 0, 1, 3, 5, 10 and 20 mM. Fresh M9 medium serves as blank control. Place the 96-well plate into an automatic microplate reader. Incubate at 16℃ overnight and record the fluorometric value at 510 nm and OD600 of each well every 30 minutes.
            5.	Each group should be repeated for at least 3 times.</SPAN><SPAN class="text-muted">[1]Mückl A, Schwarz-Schilling M, Fischer K, Simmel FC. “Filamentation and restoration of normal growth in Escherichia coli using a combined CRISPRi sgRNA/antisense RNA approach.” PLoS One. 2018 Sep 11;13(9):e0198058. doi: 10.1371/journal.pone.0198058. eCollection 2018.</SPAN></SPAN><H4 class="service-heading">Neon System Characterization</H4><H5 class="name-heading">&gt;&gt;&gt;&gt;Design</H5><SPAN class="">Assisted with the experiences we gained form the experiments above, we built and tuned the NEON system. We designed this experiment to characterize how Neon the positive feedback plasmid (<A href="http://parts.igem.org/part:BBa_K2558214">BBa_K2558214</A>), and Safety Catch the CRISPRi plasmid (<A href="http://parts.igem.org/part:BBa_K2558215">BBa_K2558215</A>, <A href="http://parts.igem.org/part:BBa_K2558216">BBa_K2558216</A>) work together.</SPAN><H5 class="name-heading">&gt;&gt;&gt;&gt;Results</H5><SPAN class="">We are still calibrating the NEON system. The results are preliminary, however from Figure.6 we can conclude that the system works to some extent. The positive feedback plasmid Neon (<A href="http://parts.igem.org/part:BBa_K2558214">BBa_K2558214</A>) had the highest expression due to uncontrollable leakage. Original lux pR (<A href="http://parts.igem.org/part:BBa_R0062">BBa_R0062</A>) and the new lux pR-HS (<A href="http://parts.igem.org/part:BBa_K2558001">BBa_K2558001</A>) we designed had lower basal expression. The addition of Safety Catch (<A href="http://parts.igem.org/part:BBa_K2558215">BBa_K2558215</A>) and Safety Catch-HS (<A href="http://parts.igem.org/part:BBa_K2558216">BBa_K2558216</A>)  almost eliminated the leakage of both positive feedback and the non-positive feedback systems. It is foreseeable that with appropriate parameters NEON system can be activated to almost 10<SUP>4</SUP> fold.
 <DIV class="text-center"><STRONG>Figure.6. Basal expression of lux quorum sensing systems.</STRONG></DIV><SPAN class="text-muted"> The fluorescence intensity was measured by flow cytometer at 488 nm. Neon, Safety Catch, Safety Catch-HS and lux pR, lux pR-HS test plasmids were transferred into <I>E.coli</I> DH5α. High fluorescence intensity suggests high level of basal expression in the system. </SPAN><H5 class="name-heading">&gt;&gt;&gt;&gt;Protocol</H5><SPAN class="">1.	Transform the plasmids into <I>E. coli</I> DH5α.
2.	Pick a single colony by a sterile tip from each of the LB plates for all the experimental and control groups. Add the colony into 5ml LB medium with ampicillin at 100 ng/µl and chloramphenicol at 34 ng/µl. Incubate for 6-8 h at 37℃ in a shaker. 
3.	Measure OD600 of the culture medium with photometer. Dilute the culture medium until OD600 reaches 0.6.
4.	Add 100 µl bacteria culture medium into a sterile 96-well plate. Add IPTG to final concentrations of 0 or 10 mM and AHL to final concentrations of 0, 10<SUP>-9</SUP>, 10<SUP>-8</SUP> M. Fresh LB medium serves as blank control. Fix sample with 1.5 mg/ml kanamycin at one hour intervals. Then use flow cytometry to measure the fluorescent intensity at 488 nm of each sample.
5.	Each group should be repeated for at least 3 times.</SPAN><SPAN class="text-muted">[1]Afroz, T., &amp; Beisel, C. L. (2013). Understanding and exploiting feedback in synthetic biology. Chemical Engineering Science, 103(11), 79-90.
[2]Qi, L., Larson, M., Gilbert, L., Doudna, J., Weissman, J., &amp; Arkin, A., et al. (2013). Repurposing crispr as an rna-guided platform for sequence-specific control of gene expression. Cell, 152(5), 1173.</SPAN></SPAN></DIV></DIV></DIV></SECTION><FOOTER class="bg-light"><DIV class="container"><DIV class="row"><DIV class="col-md-4"><SPAN class="copyright">Copyright © Tsinghua iGEM 2018</SPAN></DIV></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>