<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_BioBricks_3rd_Party_Check skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:BioBricks/3rd Party Check</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="menu_wrapper"><DIV class="collapsable_menu_control"> MENU ▤ </DIV><UL id="accordion" class="accordion"><LI class="menu_item"><A href="https://2016.igem.org/Team:BioBricks">HOME </A></LI><LI class="menu_item"> TEAM
			</LI><LI><A href=" https://2016.igem.org/Team:BioBricks/Team"> Team   </A></LI></UL><LI class="menu_item"> PROJECT  
			</LI><LI><A href="https://2016.igem.org/Team:BioBricks/Description"> Description </A></LI><LI><A href="https://2016.igem.org/Team:BioBricks/Design"> Design </A></LI><LI><A href="https://2016.igem.org/Team:BioBricks/Proof"> Proof of Concept </A></LI><LI><A href="https://2016.igem.org/Team:BioBricks/Demonstrate"> Demonstrate </A></LI><LI><A href="https://2016.igem.org/Team:BioBricks/Expression_Guide"> Expression Guide </A></LI><LI><A href="https://2016.igem.org/Team:BioBricks/Query_Search"> Query Search </A></LI><LI><A href="https://2016.igem.org/Team:BioBricks/3rd_Party_Check"> 3rd Party Check </A></LI><LI><A href="https://2016.igem.org/Team:BioBricks/Result"> Results </A></LI><LI class="menu_item"> PARTS  
			</LI><LI><A href="https://2016.igem.org/Team:BioBricks/Basic_Part"> Basic Parts </A></LI><LI class="menu_item"><A href="https://2016.igem.org/Team:BioBricks/Attributions"> ATTRIBUTIONS </A></LI><LI class="menu_item"><A href="https://2016.igem.org/Team:BioBricks/Human_Practices"> HUMAN PRACTICES </A></LI></DIV><DIV class="content_wrapper"><DIV class="column full_size"><H2> How to Check 3rd Party Rights for your Nucleotide or Peptide on Lens.org </H2><H2><P><FONT size="2pt" color="black"><I>Disclaimer: Algorithm does not constitute legal advice and will not guarantee the absence of third part rights</I></FONT></P><P><FONT size="2pt" color="black">
	To download this tutorial, click here
</FONT></P><A href="/File:T--BioBricks--3rdParty_Tutorial.xlsx" title="File:T--BioBricks--3rdParty Tutorial.xlsx">File:T--BioBricks--3rdParty Tutorial.xlsx</A><P><FONT size="4pt" color="black">
	Search Potential Patents
</FONT></P><P><FONT size="2pt" color="black"><B>
		1. Open Search Page:
	</B></FONT><FONT size="2pt" color="black">
	Open <A href="https://www.lens.org/lens/bio/patseqfinder">Lens.org</A></FONT></P><P><FONT size="2pt" color="black"><B>
		2. Insert Base Pairs:
	</B></FONT><FONT size="2pt" color="black">
	Insert bases for which you want to check 3rd party rights </FONT></P><DIV style="margin-left: 1em;"><P><FONT size="1pt" color="black">
			- e.g. sGFP (atgcgtaaaggcgaagagctgttcactggtgtcgtccctattctggtggaactggatggtgatgtcaacggtcataagttttccgtgcgtggcgagggtgaaggtgacgcaactaatggtaaactgacgctgaagttcatctgtactactggtaaactgccggtaccttggccgactctggtaacgacgctgacttatggtgttcagtgctttgctcgttatccggaccatatgaagcagcatgacttcttcaagtccgccatgccggaaggctatgtgcaggaacgcacgatttcctttaaggatgacggcacgtacaaaacgcgtgcggaagtgaaatttgaaggcgataccctggtaaaccgcattgagctgaaaggcattgactttaaagaagacggcaatatcctgggccataagctggaatacaattttaacagccacaatgtttacatcaccgccgataaacaaaaaaatggcattaaagcgaattttaaaattcgccacaacgtggaggatggcagcgtgcagctggctgatcactaccagcaaaacactccaatcggtgatggtcctgttctgctgccagacaatcactatctgagcacgcaaagcgttctgtctaaagatccgaacgagaaacgcgatcatatggttctgctggagttcgtaaccgcagcgggcatcacgcatggtatggatgaactgtacaaatgatga)

		</FONT></P></DIV><P><FONT size="2pt" color="black"><B>
		3. Select Query Type and DB:
	</B></FONT><FONT size="2pt" color="black">
	Select your Sequence type (Nucleotide vs Protein) and run search in 2 rounds, first on Nucleotide database, then on Amino Acid db
</FONT></P><P><FONT size="2pt" color="black"><B>
		4. Submit:
	</B></FONT><FONT size="2pt" color="black">
	Submit Search
</FONT></P><P><FONT size="2pt" color="black"><B>
		5. Minimum Coverage
	</B></FONT><FONT size="2pt" color="black">
	Select Minimum Coverage to 90% (or maximum of results) to make sure most of your base pairs are covered
</FONT></P><P><FONT size="2pt" color="black"><B>
		6. Minimum Similarity
	</B></FONT><FONT size="2pt" color="black">
	Select Minimum Similarity 80% (or maximum of results) to select for best matches
</FONT></P><P><FONT size="2pt" color="black"><B>
		7. Maximum Sequence Length:
	</B></FONT><FONT size="2pt" color="black">
	Select Maximum Sequence Length to the length of your query base pairs + a buffer of 15% as an average 90% of a claimed sequence can be protected (or minimum of results) </FONT></P><DIV style="margin-left: 1em;"><P><FONT size="1pt" color="black">
		- e.g. sGFP has 720bp, thus limit to ~830
	</FONT></P></DIV><P><FONT size="2pt" color="black"><B>8. Grants, in Claims</B></FONT><FONT size="2pt" color="black">
	Select Grants, in Claims </FONT></P><DIV style="margin-left: 1em;"><P>
			- Optional: Check for &quot;Applications, in claims&quot; to discover whether any patents may eventually be granted on the base pairs in question
		</P></DIV><P><FONT size="4pt" color="black">
	Analyze Patents
</FONT></P><P><FONT size="2pt" color="black"><B>9. Open Patents</B></FONT><FONT size="2pt" color="black">
	Open patents one by one and not the SEQ ID in claims
</FONT></P><P><FONT size="2pt" color="black"><B>10. Open the Legal Info Tab</B></FONT><FONT size="2pt" color="black">
	Click on Patent Details
</FONT></P><P><FONT size="2pt" color="black"><B>11. Check payment status</B></FONT><FONT size="2pt" color="black">
	Look in the Legal Info tab whether payments are completed as a first hint if the patent is expired or abandoned
</FONT></P><P><FONT size="2pt" color="black"><B>12. Open Full Text</B></FONT><FONT size="2pt" color="black">
	Open the full text of the patent
</FONT></P><P><FONT size="2pt" color="black"><B>13. Analyze Description on Claimed Sequence</B></FONT><FONT size="2pt" color="black">
	Search for &quot;Claims&quot; and analyze the description of the claimed sequence matching your query to evaluate whether your query sequence is infringing the claimed sequence.
		</FONT></P><DIV style="margin-left: 1em;"><P><FONT size="1pt" color="black">
			- “...with a DNA sequence having the nucleotide sequence of nucleotides 304-2037 of SEQ ID NO:1” meaning promoter 11:45 is not claimed 
			- “... operon is one having a DNA sequence of SEQ ID No. 1 in the Sequence Listing which is lacking in the sequence extending from 953rd base to 1160th base” meaning terminator 1053:1141 is not claimed 
			- “... operon is one having a DNA sequence of SEQ ID No. 1 in the Sequence Listing which is lacking in the sequence extending from 953rd base to 1160th base” meaning terminator 1053:1141 is not claimed 
			- “A nucleotide comprising the sequence of SEQ ID NO:1 or a functional homologue thereof having at least 95% identity with SEQ ID NO:1” 
			- “Isolated nucleic acid comprising a sequence that has a least 90% identity to SEQ ID NO: 1 and contains at least one substitution modification relative to SEQ ID NO: 1. 
			- “The isolated nucleic acid of claim 1, further comprising a fluorescent label attached to the nucleic acid.” 
			- Check type of claimed sequence: Since June 13 2013 only artificial sequences can be patented. Lawsuits on non-artificial patented sequences before June 13, 2013 are likely to fail </FONT></P></DIV></H2></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>