Team:ICT-Mumbai/Results
Simply
Judging Form ⇗
Team
Team Members
Collaborations
Software
Human Practices
Integrated Human practices
Education and Engagement
Safety
Parts
Parts overview
Composite Parts
Project
Description
Design
Experiments
Notebook
InterLab
Model
Results
Attributions
Contact us: igemteamictmumbai@gmail.com  | Follow us:
Results
Extraction of Exudates
Our first aim was to identify promoters the would be expressed in presence of root exudates. We sought out to observe differences in gene expression in Bacillus subtilis in the presence of root exudates of four different crops. Tomato, wheat, rice and soybean are the most commonly grown crops in Maharashtra, India. Therefore, exudates of these four crops were extracted using protocol explained in Experiments tab.
To collect the exudates, these crops were grown in sterilized soil for three weeks, and the exudates were extracted using water as solvent. Though most of the literature that we referred to suggested the use of artificial media such as Murashige and Skoog medium (MS medium), we decided to use soil as our medium for plant growth to replicate natural conditions. Presence of other microbial flora would have caused disturbances during the collection of exudates and RNA sequencing, hence the soil was sterilized using autoclave before sowing the seeds.
Seeds were germinated in sterile soil and grown for three weeks. Water-soluble root exudates were collected and concentrated using evaporation under vacuum using a rotary evaporator. Some of the chemical moieties in exudate mixtures were expected to be heat sensitive, hence the application of vacuum. (Please click here for detailed protocol ). Exudates were stored at -80 degC.
B. subtilis 168 was grown in presence of the exudates. RNA sequencing is being carried out to identify promoters that are induced in the presence of the exudates.
Construction of Amplification Circuit
As discussed in Design , as a case study, we worked towards developing a positive feedback circuit which can act as genetic amplification circuit. Various steps towards building the construct as well as the parts used in the process are documented below.
We started building our construct from the last element in the circuit and sequentially added required elements upstream of the sequence.
Expected output (Choice of reporter gene):
The aim was to build a positive feedback amplification circuit for production of phosphodiesterase enzyme. phoD gene was isolated from the genomic DNA of B. subtilis 168 strain using PCR.
We selected the reporter gene, Red Fluorescent Protein (rfp), as output for our amplification circuit because of ease in quantification by measurement of fluorescence. This part was obtained from the iGEM 2018 Distribution Kit ( Part #BBa_J04450 ).
Figure 1: Reporter gene
Transcription activator (TA):
In order to construct the positive feedback amplification circuit, we required a transcription activator which did not interfere with native gene regulation in either B. subtilis or E. coli.
We chose two such transcription activators from the iGEM 2018 distribution kit:
Pag activator from phage PSP3 ( Part #BBa_I746351 ): This is a transcriptional unit containing Pag activator and a ribosome binding site ( Part #BBa_B0034 ) upstream of it.
Ogr activator from phage P2 ( Part #BBa_I746350 ): This is a transcriptional unit containing Ogr activator and a ribosome binding site ( Part #BBa_B0034 ) upstream of it.
These parts were cloned upstream of rfp. Positive clones were screened by performing double digestion with EcoRI and PstI and were confirmed by agarose gel electrophoresis.
Figure 2: Construct after cloning of transcription activator with rfp.
Promoter induced by transcription activator:
Promoter which is inducible by transcription activator is pivotal part in the circuit as it regulates the transcription of its inducer, i.e. transcription activator itself (See Design ). Hence, for these constructs, choice of promoters is those that are induced by transcription activators Pag and/or Ogr activators.   PF ( Part #BBa_I746360 ) and PO ( Part #BBa_I746361 ) are such promoters, both isolated form phage P2, which can be induced by transcription activators, Pag and Ogr. As both of these are not native to B. subtilis, they are not expected to interfere with native gene transcription.
We chose following two composite parts which contain promoters with the transcription activators upstream of them:
Part #BBa_K274380 : Promoter PF ( Part #BBa_I746360 ), with Pag activator ( Part #BBa_I746351 ) upstream of it.
Part #BBa_K274371 : Promoter PO ( Part #BBa_I746361 ), with Ogr activator ( Part #BBa_I746350 ) upstream of it.
  Part #BBa_K274380                               Part #BBa_K274371
These parts were successfully cloned upstream of the constructs in Figure 1.
Figure 3: (a) rfp ( Part #BBa_J04450 ) cloned with Ogr transcription activator ( Part #BBa_I746350 ) and composite part ( Part #BBa_K274371 ) containing PO promoter and Ogr transcription activator upstream of it. This whole composite part is submitted to iGEM registry as Part #BBa_K2802003 . (b) rfp ( Part #BBa_J04450 ) cloned with Pag transcription activator ( Part #BBa_I746351 ) and composite part ( Part #BBa_K274380 ) containing PF promoter and Pag transcription activator upstream of it. This whole composite part is submitted to iGEM registry as Part #BBa_K2802001 .
Figure 4: Part #BBa_K2802001 (lane 3) of length 1554 is double digested with enzymes EcoRI and PstI and screened for correct size using agarose gel electrophoresis.
Figure 5: Part #BBa_K2802003 (lane 3) of length 1555 base pairs is double digested with enzymes EcoRI and PstI and screened for correct size using agarose gel electrophoresis.
Inducible Promoter:
The aim of this case study is to use the promoters which are induced by the root exudates of crops for the activation of genetic amplifier. But while the results of RNA sequencing were awaited, we decided to test and characterize our circuit with other inducible promoters readily available in the iGEM Parts Registry.  Although there are several promoters for B. subtilis reported in iGEM Registry, there are limited number of inducible promoters available. PliaI ( Part #BBa_K823001 ) is bacitracin-inducible promoter. We decided to go ahead with this promoter as the chances of leaky expression for an antibiotic-inducible promoter are very low.  We cloned promoter upstream of the constructs shown in Figure 2,
Figure 6: (a) Construct in figure 3(a) with bacitracin-inducible promoter PliaI ( Part #BBa_K823001 ) cloned upstream of it. This construct is the full positive feedback amplifier circuit. (b) Construct in figure 3(b) with bacitracin-inducible promoter PliaI ( Part #BBa_K823001 ) cloned upstream of it. This construct is the full positive feedback amplification circuit.
These whole constructs were then transformed into B. subtilis, using the protocol mentioned here .
As Rfp is the transcription readout for these constructs, red colonies were expected to be observed on agar + bacitracin plates. But at the concentration of 10 uL/mL, we did not observe any red colonies. We carried out same experiments in various concentrations of bacitracin ranging from 10 to 200 uL/mL. At such high concentrations of antibiotic, though we were able to see colonies, none of them were observed to be red.
To troubleshoot this issue, we tried out this promoter with only reporter gene downstream of it. We did not see any red colonies with this construct as well. We concluded that this promoter is not able to express any gene downstream of it and decided to try out different inducible promoters.
Figure 7: Construct for troubleshooting. rfp ( Part #BBa_J04450 ) cloned downstream of bacitracin-inducible promoter PLiaI ( Part #BBa_K823001 ).
Because of lack of any other inducible promoter from B. subtilis in registry, we decided to isolate them from B. subtilis 168 genomic DNA. We chose the following promoters:
PxylA: Xylose inducible promoter
PmtlA: Mannitol inducible promoter
PmalA: Maltose inducible promoter
As the sizes of these sequences were very low (160 bp), experiments were carried out to construct these promoters from synthetic oligos with overlap region. PCR was carried out using the oligos (each 90 bp long) and Taq polymerase enzyme to construct the promoter sequences. DNA of desired length (160 bp) was not observed on agarose gel.
Experiments to construct promoters using PCR were retried by changing various parameters of the polymerase chain reactions such as annealing temperature, annealing time, number of cycles. Experiment to isolate promoter sequences from the genomic DNA of B. subtilis 168 using same oligos was also carried out. We did not get a part of desired size from any of the trials.
We decided to check the validity of the construct constitutive promoter for B. subtilis available in the iGEM 2018 Distribution Kit (Promoter veg: Part #BBa_K143012 ). But because of lack of time, we were unable to ligate this promoter with our construct and characterize our whole circuit.
