Team:ICT-Mumbai/Notebook
Simply
Judging Form ⇗
Team
Team Members
Collaborations
Software
Human Practices
Integrated Human practices
Education and Engagement
Safety
Parts
Parts overview
Composite Parts
Project
Description
Design
Experiments
Notebook
InterLab
Model
Results
Attributions
Contact us: igemteamictmumbai@gmail.com  | Follow us:
Laboratory Notebook
Click on the Week tab to navigate through the notebook:
April 2018
May 2018
June Week 1
Week 2
Week 3
Week 4
July Week 5
Week 6
Week 7
Week 8
Aug Week 9
Week 10
Week 11
Week 12
Sept Week 13
Week 14
Week 15
Week 16
Seeds procured for growth of the following crops – rice, wheat, tomato and soybean.
1. Circuit design and planning of wet lab work. To see the design of our circuit please click here
2. Materials required to start lab work were procured.
3. Parts from the registry to be used in building the circuit were identified. The following parts were shortlisted to be used:
a. J04450
b. K274380  (construct with transcriptional activator)
c. K274371  (construct with transcriptional activator)
d. K823001  (promoter induced by Bacitracin)
4. Following parts ordered from IDT:
a. Xylose inducible promoter for B. subtilis
b. Maltose inducible promoter for B. subtilis
c. Mannitol inducible promoter for B. subtilis
d. Copper ion inducible promoter for B. subtilis
5. Primers ordered for gene phoD which produces alkaline phosphatase enzyme
6. Test devices for the InterLab study were reconstituted as per the protocol given on the iGEM website and transformed into E. coli strain DH5α (for protocol please click here).
Glycerol stocks made for each test device (for protocol please click here) and stored at -80℃.
7. Sowing of seeds for growth of plants into sterile soil at Vaze college.
Plans to make an amplification circuit was made that would look like: Pin (inducible by plant exudates) – RBS – Ta(transcriptional activator) -  Pta(induced by transcriptional activator) – RBS  – Ta(transcriptional activator) – RBS – GENE – TER.  Plans were made to produce Phytases and salicylic acid from B.subtilis. Materials required to start the project were collected, primers were ordered every requirement was met in this week.
Extraction of exudates from the soil of the crops grown (see exudate extraction protocol here )
The following parts from iGEM 2018 distribution kits were reconstituted and transformed into E. coli strain DH5α:
• Jo4450
• K274380  (construct with transcriptional activators1)
• K274371  (construct with transcriptional activators2)
• K823001  (promoter induced by Bacitracin )
Plasmid backbone PSB1C3 was obtained from J04450  K274380 and K274371 have the following structure:
RBS – TA – Ter – Pta (TA – Transcriptional Activator, Pta – promoter activated by transcriptional activator, Ter - terminator) Out of this  RBS – Ta were PCR amplified and joined at the ends of these parts using 3A assembly to give a construct like:
RBS – Ta – Ter – Pta – RBS – Ta.
From this construct, the RBS – TA section was PCR amplified using appropriate primers, and ligated to these parts using the 3A assembly (protocol used given here ). The above parts were digested using EcoRI and SpeI, while the RBS – TA parts were digested using XbaI and PstI to give a construct like:
RBS – Ta – Ter – Pta – RBS – Ta.
Sizes of the parts were checked by gel electrophoresis on a 1% agarose gel. (protocol available here ). Correct sizes of the bands were observed and the parts were gel purified. (protocol available here ). Below is a picture of the gel taken on a BioRad Molecular Imager® VersaDoc™ MP 4000 system.
Glycerol stocks of E coli DH5α strains containing the plasmids with these parts were made and kept at -80℃ for long term storage.
• The part with K274380 was named 80P (Pag is the transcriptional activator)
• Part with k274371 was named 71O (Ogr is the transcriptional activator)
• Along with this k518012 (RBS – RFP - Ter) was pulled out of iGem distribution kit 2018.
• The gene PhoD was PCR amplified from the genome of B.subtilis 168 with an annealing temperature of 60 deg Celsius .
• The K518012 part was assembled downstream of 80P and 71O separately using the 3A assembly. 80P and 71O were digested using EcoRI and SpeI, while the K518012  was digested using XbaI and PstI, followed by gel purification and ligation of the fragments.
• phoD was assembled downstream 80P and 71O separately using 3A assembly. 80P and 71O were digested using EcoRI and SpeI, while the phoD  was digested using XbaI and PstI, followed by gel purification and ligation of the fragments.
• These new parts were analyzed for correct sizes. Since the size of the new part and the plasmid backbone were identical we had to run longer gels in order to observe correct sizes
• The B. subtilis promoter, K823001 (induced by Bacitracin) was assembled upstream of the amplification circuit. This was done by digesting the parts containing plasmids with EcoRI and XbaI, while the promoter was digested using EcoRI and SpeI. This unusual protocol was followed in order to increase the probability of correct ligation. Since, as per our calculations, due to similar sizes (difference of only 100bp) of the plasmid backbone and the parts, a 3A assembly would have led to a lot of undesirable ligations.
• Promoter was also assembled upstream K518012 and phoD to make constructs having the structure:
Pin – RBS – RFP – Ter and Pin – RBS – PhoD - Ter.
This construct is to be used as a control to measure the amount of amplification by the amplification circuit.
• After all of this work we finally had the following constructs
i. Pin – RBS – Pag –Ter -  Ppag – RBS – Pag – RBS – RFP – Ter.
ii. Pin – RBS – Pag –Ter -  Ppag – RBS – Pag – RBS – PhoD – Ter.
iii. Pin – PBS – Ogr – Ter – Pogr – RBS – Ogr – RBS – RFP – Ter.
iv. Pin – PBS – Ogr – Ter – Pogr – RBS – Ogr – RBS – PhoD – Ter.
v. Pin – RBS – RFP – Ter.
vi. Pin – RBS – PhoD – Ter.
• Preparations for the InterLab study – materials procured (plates, 96 – well plates, amber 50 ml Falcon centrifuge tubes, pipette tips, etc.), plates poured, instrument settings adjusted (for more details visit the Interlab page)
• Calibrations were done following the InterLab Plate Reader protocol
• Inoculation of test devices for InterLab study from glycerol stocks prepared in May into LB media and incubation. Samples prepared for the required measurements as per the given InterLab protocol.
• Measurements taken for InterLab study following the protocol provided on the Measurement page of the iGEM website
• Compilation of this data into the given Excel Sheet
• Submission of InterLab data and filling of forms
• Parts obtained in week 6 were transferred into PSB1C plasmid backbone for B.subtilis and then transformed into B.subtilis using the protocol mentioned here.
• Transformed B subtilis plated in different quantities of inducer Bacitracin.
• No fluorescence was observed in any of the parts, possibly a promoter issue.
9. Week 9: 29th Jul – 4th Aug • Further testing of transformed B subtilis in increasing quantities of Bacitracin. The transformed B subtilis cells were plated into LB + agar plates containing higher quantities of Bacitracin.
• Searching for different inducible promoters that can be used instead of the bacitracin  inducible promoter.
• Ordering of primers for overlap extension PCR of a mannitol inducible promoter and maltose inducible promoter.
• Ordering of a few parts from the iGEM registry not present in the 2018 distribution kit.
• Submission of InterLab data and filling of forms
• Parts obtained in week 6 were transferred into PSB1C plasmid backbone for B.subtilis and then transformed into B.subtilis using the protocol mentioned here.
• Transformed B subtilis plated in different quantities of inducer Bacitracin.
• No fluorescence was observed in any of the parts, possibly a promoter issue.
• Primers arrived and overlap extension PCR carried out. PCR products ligated into PSB1C3 plasmid and transformed into E. coli DH5α cells. Cells were inoculated into LB media and incubated at 37℃.
• Plasmids extracted, digested and run on an agarose gel – sizes obtained incorrect. This implied improper PCR reaction.
• Troubleshooting of overlap extension PCR
• Overlap extension PCR repeated by changing annealing temperature, extension time in cycle, and amounts of primers used. All PCR products transformed into E. coli strain DH5α, cells inoculated, incubated and plasmid sizes checked by agarose gel electrophoresis.
• Continuing troubleshooting of overlap extension PCR and testing at different parameter values. Overlap extension PCR does not seem to be working.
• Parts received from iGEM HQ
• Inoculation of constructs from week 6 from their glycerol stocks and plasmid extraction
• Reconstitution and transformation of following parts:
o I0500 (2018 distribution kit) – arabinose inducible E coli promoter Pbad
o K823015 (part ordered from iGEM registry) – xylose inducible B subtilis promoter Pxyl
o K143012 (2018 distribution kit) – constitutive Pveg promoter for B. subtilis
o K823003 (2018 distribution kit) – constitutive Pveg promoter for B. subtilils
• Ligation of each of the above promoters upstream of constructs i, iii and v from week 6 using the 3A assembly. Transformation of ligation products into E. coli  DH5α, inoculation and incubation.
• Plating of ligation products on plates containing respective inducers of each promoter
• No promoter activity seen
• Sizes of ligation products checked on gel. Part sizes incorrect
• Ligation reactions of week 14 repeated by changing the insert:vector ratio to 5:1
• Sizes of ligation products checked. Ligations were unsuccessful.
• Planned the whole process of part purification, digestion and ligation for next week, with gel electrophoresis after each stage to troubleshoot the process.
