<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_ICT-Mumbai_Model skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:ICT-Mumbai/Model</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Simply</TITLE></P><DIV id="cssmenu" class="hov"><UL><LI><A href="https://igem.org/2018_Judging_Form?team=ICT-Mumbai">Judging Form ⇗</A></LI><LI class="has-sub"><A href="#">Team</A></LI><LI><A href="https://2018.igem.org/Team:ICT-Mumbai/Team">Team Members</A></LI><LI clas="has-sub"><A href="https://2018.igem.org/Team:ICT-Mumbai/Collaborations">Collaborations</A></LI></UL><LI><A href="https://2018.igem.org/Team:ICT-Mumbai/Software">Software</A></LI><LI class="has-sub"><A href="#">Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:ICT-Mumbai/Human_Practices">Integrated Human practices</A></LI><LI><A href="https://2018.igem.org/Team:ICT-Mumbai/Public_Engagement">Education and Engagement</A></LI><LI><A href="https://2018.igem.org/Team:ICT-Mumbai/Safety">Safety</A></LI><LI class="has-sub"><A href="#">Parts</A></LI><LI><A href="https://2018.igem.org/Team:ICT-Mumbai/Parts">Parts overview</A></LI><LI><A href="https://2018.igem.org/Team:ICT-Mumbai/Composite_Part">Composite Parts</A></LI><LI class="has-sub"><A href="#">Project</A></LI><LI><A href="https://2018.igem.org/Team:ICT-Mumbai/Description">Description</A></LI><LI><A href="https://2018.igem.org/Team:ICT-Mumbai/Design">Design</A></LI><LI><A href="https://2018.igem.org/Team:ICT-Mumbai/Experiments">Experiments</A></LI><LI><A href="https://2018.igem.org/Team:ICT-Mumbai/Notebook">Notebook</A></LI><LI><A href="https://2018.igem.org/Team:ICT-Mumbai/InterLab">InterLab</A></LI><LI><A href="https://2018.igem.org/Team:ICT-Mumbai/Model">Model</A></LI><LI><A href="https://2018.igem.org/Team:ICT-Mumbai/Results">Results</A></LI><LI><A href="https://2018.igem.org/Team:ICT-Mumbai/Attributions">Attributions</A></LI></DIV><DIV class="footer"><P style="display: block; float: right; padding-top: 50px;">
Contact us: <I>igemteamictmumbai@gmail.com</I> | Follow us:
</P></DIV>&lt;head&gt;&lt;/head&gt;<DIV class="wrapper"><H1>Modeling</H1><H4>Introduction</H4><P>
The modelling of synthetic biology systems is vital in the design of genetic circuits. To be able to construct sequences of increasing complexity – on the scale of the natural networks already present in the bacteria it is necessary develop satisfactory computational tools to predict and mimic the behaviour of the system. 
To predict the output of our <A href="https://2018.igem.org/Team:ICT-Mumbai/Design"><U>genetic circuit</U></A> we created a single cell model for our circuit. The effect of transcription regulators on gene expression was assumed to follow Hill equation kinetics, with first order kinetics for the degradation of the proteins/enzymes formed. The models were created in MATLAB.</P><H3>Phosphatase Amplification Circuit</H3><P>
The positive feedback amplification circuit shown above was used in our study to obtain an amplified production of our gene of interest in presence of an inducing chemical. For more details on the parts used in the circuit please visit the <A href="https://2018.igem.org/Team:ICT-Mumbai/Design"><U>Design page</U></A>.
</P><H5>Deterministic Model</H5><P>
The deterministic model involves creating a series of differential equations which are solved simultaneously using certain initial conditions. 
The differential equations were obtained by performing a mass balance for each protein being expressed. 
For the transcriptional activator (‘TA’):
<UL><LI>It is formed by gene expressed by promoter P1 which is induced by the an inducer ‘I’

\[
Rate\ of\ formation\ of\ TA\ through\
P1=\frac{V_{{max}_I}*{[I]}^{n_I}}{k_I^{n_I}+{[I]}^{n_I}}
\]

</LI><LI>It is formed by gene expressed by promoter P2 which is induced by itself ‘TA’
\[
Rate\ of\ formation\ of\ TA\ through\ P2=\frac{V_{{max}_{TA}}*\
{[TA]}^{n_{TA}}}{k_{TA}^{n_{TA}}+{[TA]}^{n_{TA}}}
\]

</LI><LI>It is degraded
\[
{Rate\ of\ degradation\ of\ TA=\delta{}}_{TA}[TA]
\]
</LI></UL></P><P>
By a mass balance, the overall rate of change of the concentration of the transcriptional regulator is thus given by:
\[
\frac{d[TA]}{dt}=\frac{V_{{max}_I}*{[I]}^{n_I}}{k_I^{n_I}+{[I]}^{n_I}}+\frac{V_{{max}_{TA}}*\
{[TA]}^{n_{TA}}}{k_{TA}^{n_{TA}}+{[TA]}^{n_{TA}}}-{\delta{}}_{TA}\left[TA\right]+{\alpha{}}_{TA}
\]

Similarly for the reporter protein RFP (‘R’):
</P><UL><LI>It is formed by gene expressed by promoter P2 which is induced by the transcriptional regulator ‘TR’
\[
Rate\ of\ formation\ of\ R\ through\ P2=\frac{V_{{max}_{TA}}*\
{[TA]}^{n_{TA}}}{k_{TA}^{n_{TA}}+{[TA]}^{n_{TA}}}
\]
</LI><LI>It is degraded
\[
{Rate\ of\ degradation\ of\ R=\delta{}}_R[R]
\]
</LI></UL><P>
By a mass balance, the overall rate of change of the concentration of the transcriptional regulator is thus given by:
\[
\frac{d[R]}{dt}=\frac{V_{{max}_{TA}}*\
{[TA]}^{n_{TA}}}{k_{TA}^{n_{TA}}+{[TA]}^{n_{TA}}}-{\delta{}}_R\left[R\right]+{\alpha{}}_R
\]


The resulting trend obtained through the deterministic model is shown below:</P><P>
On solving the above two differential equation simultaneously, with the following initial conditions, we get a curve as shown below:
</P><H5>
Stochastic model</H5><P>
The deterministic model takes into account only the initial condition based on which it gives an output curve with respect to time. A more accurate way of representing a biological system is through a stochastic model which accounts for the “randomness” in the biological systems. The stochastic model was built based on the Gillespie algorithm[2]. 
In this model, each possible step was given a probability and then a particular step was selected at random and changes made to the current state of the system. The different steps taken were:
<OL><LI>TA is produced through promoter P1
</LI><LI>TA is produced through promoter P2
</LI><LI>TA is degraded
</LI><LI>R is produced through promoter P2
</LI><LI>R is degraded
</LI></OL></P><P>
The resulting trend obtained through the stochastic model is shown below:
Both the above models are qualitative in nature at this stage. This is because we do not as yet know which inducible promoter will be used and hence cannot estimate the parameters for it .The amplification circuit is known to produce a higher concentration of the target protein than simple expression of the protein – and the change in concentration of the protein with time is similar to the trend we have obtained.[3] Once we are able to identify the promoters and components of the circuit being used, we will be able to plug in the parameter values to this model and quantify it. 

</P><H4>Qualitative model of potential application of project</H4><P>
The above circuit is based on the standard three node repressilator circuit.[4] It has been modified so that the oscillations only take place in presence of ‘E’, i.e., it is an oscillatory circuit with an external switch to start/stop the oscillations. A, B and C are proteins that successively repress the next promoter. ‘G’ is our required product protein. The circuit is designed so as to ensure that there is no G produced in absence of E, but we get an oscillating output of G in presence of E. Since we expect an oscillatory output, we cannot simply solve the differential equations using initial conditions as this gives a Michaelis-Menten type curve (given the nature of the equations used). So we created a stochastic model for this circuit. The steps considered for the stochastic model are shown below:
<B>For A:</B><OL><LI>A is produced by promoter P1 which is repressed by E
\[
rate\ of\ formation=\ \frac{V_{{max}_E}*\ k_E^{n_E}}{k_E^{n_E}+{[E]}^{n_E}}
\]
(**above equation based on the Hill equation)
</LI><LI>A is produced by promoter P2 which is repressed by C
\[
rate\ of\ formation=\ \frac{V_{{max}_C}*\ k_C^{n_C}}{k_C^{n_C}+{[C]}^{n_C}}
\]
</LI><LI>A gets degraded
\[
rate\ of\ degradation=\ {\delta{}}_A[A]
\]
</LI><LI>A is produced due to its basal transcription level (leakage rate)
\[
rate\ of\ formation=\ \alpha_A
\] 
</LI></OL></P><P><B>For B:</B><OL><LI>B is produced by promoter P1 which is repressed by E
\[
rate\ of\ formation=\ \frac{V_{{max}_E}*\ k_E^{n_E}}{k_E^{n_E}+{[E]}^{n_E}}
\]

</LI><LI>B is produced by promoter P3 which is repressed by A
\[
rate\ of\ formation=\ \frac{V_{{max}_A}*\ k_A^{n_A}}{k_A^{n_A}+{[A]}^{n_A}}
\]

</LI><LI>B gets degraded
\[
rate\ of\ degradation=\ {\delta{}}_B[B]
\]
</LI><LI>B is produced due to its basal transcription level (leakage rate)
\[
rate\ of\ formation=\ \alpha_B
\] 
</LI></OL></P><P><B>For C:</B><OL><LI>C is produced by promoter P1 which is repressed by E
\[
rate\ of\ formation=\ \frac{V_{{max}_E}*\ k_E^{n_E}}{k_E^{n_E}+{[E]}^{n_E}}
\]

</LI><LI>C is produced by promoter P4 which is repressed by B
\[
rate\ of\ formation=\ \frac{V_{{max}_B}*\ k_B^{n_B}}{k_B^{n_B}+{[B]}^{n_B}}
\]

</LI><LI>C gets degraded
\[
rate\ of\ degradation=\ {\delta{}}_C[C]
\]
</LI><LI>C is produced due to its basal transcription level (leakage rate)
\[
rate\ of\ formation=\ \alpha_C
\] 
</LI></OL></P><P><B>For G:</B><OL><LI>G is produced by promoter P4 which is repressed by B
\[
rate\ of\ formation=\ \frac{V_{{max}_B}*\ k_B^{n_B}}{k_B^{n_B}+{[B]}^{n_B}}
\]
</LI><LI>G gets degraded
\[
rate\ of\ degradation=\ {\delta{}}_G[G]
\]
</LI><LI>G is produced due to its basal transcription level (leakage rate)
\[
rate\ of\ formation=\ \alpha_G
\] 
</LI></OL></P><P>
The resulting trend obtained for the given model in response to a step input to the concentration of the external inducer E is shown below:


Like the amplification circuit, this model is also qualitative in nature as the exact promoters being used are not known and hence their parameter values are unknown. This circuit is more sensitive to variations in parameters than the amplification circuit. However, the general nature of the curve obtained shows that oscillations do start when the external inducer is added to the system. 
</P><H5>References</H5><P>
[1] Jennifer A.N. Brophy and Christopher A. Voigt, “Principles of Genetic Circuit Design”, Nat.
methods, May 2014
[2] “Systems Biology” by Uri Alon
[3] Goutam J Nistala, Kang Wu, Christopher V Rao, Kaustubh D Bhalerao, ‘A modular positive
feedback-based gene amplifier’, Journal of Biological Engineering, 2010
[4] Michael B. Elowitz &amp; Stanislas Leibler, ‘A synthetic oscillatory network of transcriptional
regulators’, Nature vol. 403, 2000</P></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>