<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_ICT-Mumbai_Software skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:ICT-Mumbai/Software</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Simply</TITLE></P><DIV id="cssmenu" class="hov"><UL><LI><A href="https://igem.org/2018_Judging_Form?team=ICT-Mumbai">Judging Form ⇗</A></LI><LI class="has-sub"><A href="#">Team</A></LI><LI><A href="https://2018.igem.org/Team:ICT-Mumbai/Team">Team Members</A></LI><LI clas="has-sub"><A href="https://2018.igem.org/Team:ICT-Mumbai/Collaborations">Collaborations</A></LI></UL><LI><A href="https://2018.igem.org/Team:ICT-Mumbai/Software">Software</A></LI><LI class="has-sub"><A href="#">Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:ICT-Mumbai/Human_Practices">Integrated Human practices</A></LI><LI><A href="https://2018.igem.org/Team:ICT-Mumbai/Public_Engagement">Education and Engagement</A></LI><LI><A href="https://2018.igem.org/Team:ICT-Mumbai/Safety">Safety</A></LI><LI class="has-sub"><A href="#">Parts</A></LI><LI><A href="https://2018.igem.org/Team:ICT-Mumbai/Parts">Parts overview</A></LI><LI><A href="https://2018.igem.org/Team:ICT-Mumbai/Composite_Part">Composite Parts</A></LI><LI class="has-sub"><A href="#">Project</A></LI><LI><A href="https://2018.igem.org/Team:ICT-Mumbai/Description">Description</A></LI><LI><A href="https://2018.igem.org/Team:ICT-Mumbai/Design">Design</A></LI><LI><A href="https://2018.igem.org/Team:ICT-Mumbai/Experiments">Experiments</A></LI><LI><A href="https://2018.igem.org/Team:ICT-Mumbai/Notebook">Notebook</A></LI><LI><A href="https://2018.igem.org/Team:ICT-Mumbai/InterLab">InterLab</A></LI><LI><A href="https://2018.igem.org/Team:ICT-Mumbai/Model">Model</A></LI><LI><A href="https://2018.igem.org/Team:ICT-Mumbai/Results">Results</A></LI><LI><A href="https://2018.igem.org/Team:ICT-Mumbai/Attributions">Attributions</A></LI></DIV><DIV class="footer"><P style="display: block; float: right; padding-top: 50px;">
Contact us: <I>igemteamictmumbai@gmail.com</I> | Follow us:
</P></DIV><DIV id="div-fixed"><CENTER><H2>BBrickIt</H2></CENTER><A href="http://github.com/igemsoftware2018/Team_ICT_Mumbai" class="button-1">Download</A></DIV><DIV class="wrapper"><H4>BioBrick parts and standard assembly</H4><DIV><P>
A BioBrick part can be defined as a DNA sequence (encoding a particular biological function) that conforms to a set of <A href="http://parts.igem.org/Help:Standards/Assembly"><U>standard rules</U></A>. The Registry of Standard Biological Parts consists of over 2000 standard parts, each of which can be assembled according to BioBrick assembly standards to create multi-component parts of increasing complexity and functionality <A href="https://jbioleng.biomedcentral.com/articles/10.1186/1754-1611-2-5"><U>(Shetty et al., 2008)</U></A>. 

There are different assembly standards, namely BioBrick Standard Assembly (<A href="http://parts.igem.org/Help:Standards/Assembly/RFC10"><U>RFC 10</U></A>), BioBrick BB-2 Assembly (<A href="http://parts.igem.org/Help:Standards/Assembly/RFC12&gt;&lt;u&gt;RFC 12&lt;/u&gt;&lt;/a&gt;), Chris Anderson Berkeley BamBgl (&lt;a href=" http:=""><U>RFC 21</U></A>), Silver/Fusion Bricks (<A href="http://parts.igem.org/Help:Standards/Assembly/RFC23"><U>RFC 23</U></A>), Freiburg Fusion Protein (<A href="http://parts.igem.org/Help:Standards/Assembly/RFC25"><U>RFC 25</U></A>) and BioBrick MoClo (RFC 1000).  (<U><A href="http://parts.igem.org/Help:Standards/Assembly">Registry of Standard Biological Parts, iGEM</A></U>; <U><A href="http://dspace.mit.edu/handle/1721.1/45138">Knight, 2007</A></U>; <U><A href="http://dspace.mit.edu/handle/1721.1/45139">Knight, 2008</A></U>; <U><A href="http://dspace.mit.edu/handle/1721.1/46747">Anderson et al., 2009</A></U>; <U><A href="http://dspace.mit.edu/handle/1721.1/32535">Philips et al., 2006</A></U>; <U><A href="http://dspace.mit.edu/handle/1721.1/45140">Grünberg et al., 2009</A></U>).
The BioBrick assembly standard RFC 10 is the most widely used assembly standard (<U><A href="http://parts.igem.org/Help:Standards/Assembly">Registry of Standard Biological Parts, iGEM</A></U>). Each year hundreds of iGEM teams all over the world contribute and submit BioBrick compatible parts to the registry. The BioBrick standard has a set of stringent requirements for a part to be RFC 10 compatible:
</P></DIV><DIV id="RFC10"><DIV class="wrapper1"><DIV class="col_third"><DIV class="hover panel"><DIV class="front"><DIV class="box1"><H4>Should not contain Illegal and Avoidable sites</H4></DIV></DIV><DIV class="back"><DIV class="box2"><P> Allowed sequences within BioBrick parts include any DNA sequence
   which does not contain the following sequences:<U>Illegal</U>EcoRI (gaattc)
                    XbaI (tctaga)
                    SpeI (actagt)
                    PstI (ctgcag)
                    <U> Avoidable</U>NotI (gcggccgc)</P></DIV></DIV></DIV></DIV><DIV class="col_third"><DIV class="hover panel"><DIV class="front"><DIV class="box1"><H4>Should have appropriate Prefix and Suffix</H4></DIV></DIV><DIV class="back"><DIV class="box2"><P style="font-size: 12px; line-height: 13px;">
   Each BioBrick part must contain the following sequence as suffix downstream of the 3' end of the part:

   T ACTAGT A GCGGCCG CTGCAG
 The prefix used for coding regions is different from the sequence for non-coding BioBrick parts. This is to allow the construction of ribosomal binding sequences 5' of
   coding regions.

   1. For non-coding Biobrick parts this prefix must be placed 5' upstream of
      the part:
      GAATTC GCGGCCGC T TCTAGA G

  2. For BioBrick parts coding for proteins, the following sequence should be placed 5' upstream of the start codon (ATG) of the coding region:

      GAATTC GCGGCCGC T TCTAG</P></DIV></DIV></DIV></DIV><DIV class="col_third end"><DIV class="hover panel"><DIV class="front"><DIV class="box1"><H4>Should have appropriate terminating sequence</H4></DIV></DIV><DIV class="back"><DIV class="box2"><P> All parts containing start codons other than ATG must be changed to
      possess ATG as the start codon.  Though not mandatory, it is highly recommended, that all coding regions terminate with in-frame TAATAA
      stop codons, replacing other stop codons (TGA, TAG) if needed.</P></DIV></DIV></DIV></DIV></DIV></DIV><DIV><P>
To ensure that the biological part conforms to these standards, researchers have to carry out multiple checks and modifications, some of which include:</P><OL><LI>Checking for presence of illegal and avoidable restriction sites in the part.
</LI><LI>Modification of part sequence to replace illegal and avoidable restriction sites, depending on chassis organism.
</LI><LI>Addition of suitable prefix according to standards described above.
</LI><LI>Addition of suitable suffix according to standards described above.
</LI></OL></DIV><DIV><H4> BBrickIt: The one-stop solution to your BioBricking needs</H4><P>
Based on our own experience and a survey that we conducted with iGEM 2018 teams (results can be found <A href="https://static.igem.org/mediawiki/2018/7/7e/T--ICT-Mumbai--BBrickIt-survey.pdf"><U>here</U></A>) we found that one has to alternate between different programmes to achieve all requirements to construct a BioBrick.
We found ourselves alternating between different programmes such NEB Cutter, various codon optimization tools and the iGEM Parts Registry. Although this process is repetitive and tedious (more so when creating/submitting many parts), it also is very important for the sequence that is generated to be free of errors, the chances of which increases when done manually.  
We sought to create an open source online tool which would enable users to check, modify and validate their parts according to the RFC 10 standard, all in a single programme thereby making the entire process simpler, faster and error-free. We believe that this tool will be particularly helpful to first-time iGEM participants.

</P></DIV></DIV><DIV id="grey-div"><DIV id="wrapper"><H1>BioBrick your part and more!</H1></DIV><P style="text-align: center">
Identify and validate part for presence/absence of illegal restriction sites
</P><P style="text-align: center">
Replace sequence according to chassis (<I>E. coli</I>, <I>B. subtilis</I> and <I>S. cerevisiae</I>)
</P><P style="text-align: center">
Add required Prefix and suffix
</P><P style="text-align: center">
Available in <A style="color: #ffffff;" href="https://github.com/igemsoftware2018/Team_ICT_Mumbai/blob/master/BBrickIt_India"><U>Mac</U></A> and <A style="color: #ffffff;" href="https://github.com/igemsoftware2018/Team_ICT_Mumbai/blob/master/BBrickIt_beta%20v1.3.1.exe"><U>Windows</U></A> versions
</P></DIV><DIV class="wrapper"><H4>Key considerations while designing the software</H4><DIV id="circle-div"><P style="font-size: 25px">Open source </P></DIV><DIV id="circle-div"><P style="font-size: 25px">Ease of use</P></DIV><DIV id="circle-div"><P>Single programme for all analysis and design</P></DIV><DIV id="circle-div"><P style="font-size: 22px">Error-free sequence modification</P></DIV><DIV id="circle-div"><P>Codon-optimization for common chassis organisms</P></DIV></DIV><DIV class="wrapper"><H4>User guide</H4><P>
We have created a user guide that outlines the step-by-step procedure for using this software. The user guide can be accessed <A href="https://static.igem.org/mediawiki/2018/b/be/T--ICT-Mumbai--BBrickIt-User-Manual.pdf"><U>here</U></A>.
</P><H4>BBrickIt Software Beta Testing</H4><P>Several iGEM teams tested our software along the way. 
Many teams asked us to make a comprehensive user manual for this software, which we did and shared with all of them! Also, initially, we uploaded a .exe file on Github, which worked only on the Windows operating system. As MacOS is also widely used,  the MacOS version of BBrickIt is also available in the GitHub repository.
</P><DIV id="team"><H1>Target audience: The iGEM community!</H1></DIV><H4>Feedback from iGEM teams</H4><DIV class="container"><BLOCKQUOTE class="brooks">
    Extremely happy to bypass the hectic job of having to think of prefix and suffix sites, scars, illegal sites etc. before planning to clone anything in the standard vectors
    <FOOTER>Team IISER Kolkata</FOOTER></BLOCKQUOTE><BLOCKQUOTE class="brooks">
    The tool is easy to use and well outlined. This software would have helped us catch that we initially forgot to add the BioBrick prefix and suffix to our gene sequence!
    <FOOTER>Team WashU</FOOTER></BLOCKQUOTE><BLOCKQUOTE class="brooks">
“BBrickIt is a useful tool for iGEM teams in producing complete BioBricked part and is also very efficient.”
   <FOOTER>Team ECUST</FOOTER></BLOCKQUOTE></DIV></DIV><DIV class="wrapper"><H4> Methodology </H4><P>
The initial idea for this software came to us while we were shuttling between various software tools in order to identify the compatibility of our sequence as per the BioBrick RFC 10 standard. We shuffled between various tools for performing tasks like identifying illegal restriction sites, reverse complementing the sequence, and manually adding prefix and suffix sequences. We then decided to create a simple, user-friendly GUI that can perform all these functions and specifically look for sites as per BioBricking standards.  
The programme also suggests a codon-optimized sequence in case the input sequence is not RFC 10-compatible. Currently the programme works for three commonly used chassis organisms, <I>Escherichia coli</I>, <I>Bacillus subtilis</I> and <I>Saccharomyces cerevisiae</I>. 
BBrickIt was coded using the Python language, and the GUI was created using the Tkinter library that is provided along with it. This is an open source programme and its source code can be found <A href="https://github.com/igemsoftware2018/Team_ICT_Mumbai/blob/master/BBrickIt_test.py"><U>here</U></A>.
</P><H5>References</H5><P><I><OL><LI>Idempotent Vector Design for Standard Assembly of Biobricks by Tom Knight
</LI><LI>BBF RFC 9: Idempotent vector design for the standard assembly of Biobricks by Tom Knight, Randall Rettberg, Leon Chan, Drew Endy, Reshma Shetty, Austin Che
</LI><LI>BBF RFC10: Draft standard for Biobrick biological parts by Tom Knight
</LI><LI><A href="http://parts.igem.org/Help:BioBrick_Prefix_and_Suffix">http://parts.igem.org/Help:BioBrick_Prefix_and_Suffix</A></LI><LI>Engineering BioBrick Vectors from BioBrick Parts; Shetty RP et. al.; J Biol Eng. 2008; 2: 5; DOI:
10.1186/1754-1611-2-5
</LI><LI><A href="http://parts.igem.org/Help:Primers/Design">http://parts.igem.org/Help:Primers/Design</A></LI></OL></I></P></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>