"
Page
Discussion
View source
History
teams
Log in
 
Team:Aix-Marseille/Notebook 09
From 2014.igem.org
Team
Official Team Profile
Project
S.E.colization : Green light, GO!
Beyond the Bench
Parts
Parts
Achievements
Modeling
Notebook
July
August
September/October
Safety
Attributions
Sponsors
Notebook: September / October
Week 10 : 09/01/2014 ➡ 09/07/2014
09/03/14
09/04/14
09/05/14
09/07/14
Construction of stability epitopes
The main idea is to use the primers 57-58 and 59 as matrices in order to synthesize and amplify the stability epitope. The PCR uses the iGEM PCR protocol with Q5 polymerase. The temperature of annealing is 55°C, the elongation time is 5", the number of cycle is 25.
Mix:
5µL OiGEM-57 or 58 or 59
2,5µL OiGEM-60
2,5µL OiGEM-44
25µL Q5 2x Mix
15µL H2O
Clean-up of these PCR and elution with 30µL H2O at 70°C.
PCR cheA-kan-cheA using the iGEM PCR protocol with Q5 polymerase. The temperature of annealing is 60°C, the elongation time is 1’30”.
Mix:
1µL pKD4 vector
2,5µL primer 3
2,5µL primer 4
2,5µL dNTP
10µL Q5 buffer
qsp 50µL H2O
Construction of stability epitopes
Digestion ES of PCR products of the stability epitopes and of piGEM-01.13 (pSB1C3-BBa_J04450).
Mix:
10µL K10512-06 or 07 or 08 or 5µL piGEM-01.13
1µL EcoRI
1µL SpeI
5µL Buffer 2.1
qsp 50µL of water
Incubation 60' at 37°C.
Then, for K10512-06 or 07 or 08,  inactivation of enzymes 20' at 80°C.
Clean-up of piGEM-01.13 and elution with 30µL of water. Add 1µL of SAP (Promega) + 2µL of SAP buffer (dephosphorylation). Incubation 30' at 37°C, then Inactivation of enzyme 20' at 65°C.
Ligation K10512-06 or 07 or 08 with piGEM-01.13.
Mix:
2µL piGEM-01.13
4µL K10512-06 or 07 or 08
1µL T4 ligase 10U/µL
2µL T4 ligase buffer
11µL H2O
Transformation of 90µL of E.coli TG1 competent cells with 10µL of ligation. The selective medium is LB-Cm 30µg/mL.
Clean-up of the previous PCR. Elution in 45µL of water.
Electroporation of W3110 ∆sdaBC electrocompetent cells from Aimeric (clone 43)
40µL cells + 10µL PCR
40µL cells + 0,5µL pKT25 (negative control)
==> Spread bacteria on LB-Kan
Many clones have grown on the negative control but no clone has grown for the cheA mutant. So cells are competent but they don’t express the recombinase.
Streak out W3110 ∆sdaBC 43 and 63 from Aimeric on LB and LB-Kan
Week 11 : 09/08/2014 ➡ 09/14/2014
09/08/14
09/11/14
Construction of stability epitopes
Test 5 clones of each transformation (with pSB1C3- K10512-06 or 07 or 08) by PCR. The temperature of annealing is 55°C, the elongation time is 1'
Mix:
2µL OiGEM-60
2µL OiGEM-44
4µL dNTP
1µL DMSO
2µL buffer
0,5µL Taq polymérase
8,5µL H2O
All clones are good except the K10512-08-2.
The pSB1C3-K10512-06-1, pSB1C3-K10512-07-1 and pSB1C3-K10512-08-1 are sent to sequencing. They are OK.
The bacteria are sensitive to kanamicyn.
Repreparation of W3110 ∆sdaBC 43 electrocompetent cells (see Lambda Red mutant construction protocol) and restart from this step (day 1).
The clones obtained were KanR and ApS ==> OK
The clones are pooled 3 by 3 to be tested by PCR with primers 52 and 53.
The pools 1 and 2 seemed to contain mutants (1786bp).
Restart PCR with primers 52 and 53 on the isolated clones from pools 1 and 2.
The clones 1a, 2a and 2b seemed good, they are kept at -80°C. So the W3110 ∆sdaBC ∆cheA  is OK.
Week 13 : 09/22/2014 ➡ 09/28/2014
relA Test
09/25/14
Attribution: Emmanuelle Bouveret
Objective: Test the functionality of our RelA part.
The pSB1K3-relA plasmid constructed by the iGEM-AMU team is expected to be able to rescue the phenotype of a MG1655∆relA mutant grown on SMG plates, a medium known to induce a strong amino acid depletion in the cells (Rudd et al.,1985) that is toxic to a ∆relA mutant.
Day 1:
Isolated clones were restreaked on SMG plates supplemented with 50 uM Kanamycin and 0.5mM IPTG. The plates were incubated at 37°C during 48 hrs.
Day 2:
Electrocompetent MG1655∆relA (EB421 strain, Wahl et al. 2011) cells were electroporated with either plasmid pSB1K3-RFP as a control or pSB1K3-relA.
Day 4:
As expected, we observed that the relA construct was able to complement the MG1655∆relA strain, as seen on the following figure.
SMG plate composition:
M9 salts
1X
MgSO4
1mM
CaCl2
0.1mM
VitB1
0.5µg/ml
Glucose
0.2%
Serine
1mM
Methionine
1mM
Glycine
1mM
Bactoagar
15g/L
References:
Rudd KE, Bochner BR, Cashel M, Roth JR. Mutations in the spoT gene of Salmonella typhimurium: effects on his operon expression. J Bacteriol. ;163(2):534-42, 1985.
Wahl A, My L, Dumoulin R, Sturgis JN, Bouveret E.Antagonistic regulation of dgkA and plsB genes of phospholipid synthesis by multiple stress responses in Escherichia coli. Mol Microbiol. 80(5):1260-75, 2011.
CurR-box characterization : introduction of the CusR-box part ( BBa_K1349002 ) into a pUC18 plasmid expressing the GFP from a pLAC promoter.
The insertion of the BBa_K1349002 part downstream of the Plac promoter is expected to allow the binding of the CusR regulator in response to metal (Cu or Ag) availability in the medium. In our construct, we expect CusR binding to the CusR-boxes to physically blocking RNA polymerase, donc consequently to down regulate the expression of the GFP gene.
CurR-box characterization
Digestion ES and XP of cusR box BBa_K1349002 and digestion EP of piGEM-02.28 (pSB1K3-BBa_J04450).
500ng DNA
1µL EcoRI or XbaI
1µL SpeI or PstI
5µL NEB Buffer 2.1
qsp 50µL of water
Incubation 60' at 37°C.
Then inactivation of the enzymes for 20' at 80°C.
Ligation of the cusR box BBa_K1349002 digested ES with cusR box digested XP in piGEM-02.28 digested EP.
2µL piGEM-02.28
4µL BBa_K1349002 (ES)
4µL BBa_K1349002 (XP)
1µL T4 ligase 10U/µL
2µL T4 ligase buffer
7µL H2O
Transformation of 100µL of E.coli TG1 competent cells with 10µL of ligation. The selective medium is LB-Km 25µg/mL.
Week 15 : 10/06/2014 ➡ 10/12/2014
10/06/14
10/08/14
10/09/14
10/10/14
10/12/14
CurR-box characterization
Digestion ES of cusR box BBa_K1349002 (x1 or x2), digestion XP of RBS-GFP (200(2)) and digestion EP of piGEM-01.13 (pSB1C3-BBa_J04450).
500ng DNA
1µL EcoRI or XbaI
1µL SpeI or PstI
5µL NEB Buffer 2.1
qsp 50µL of water
Incubation 60' at 37°C.
Then inactivation of the enzymes for 20’ at 80°C.
Ligation of the cusR box BBa_K1349002 (x1 or x2) with RBS-GFP (200(2)) in piGEM-01.13.
2µL piGEM-01.13
4µL BBa_K1349002 (x1 or x2)
4µL 200(2)
1µL T4 ligase 10U/µL
2µL T4 ligase buffer
7µL H2O
Digestion ES of cusR box BBa_K1349002 (x2), digestion XP of cusR box BBa_K1349002 (x1) and digestion EP of piGEM-01.13 (pSB1C3-BBa_J04450).
500ng DNA
1µL EcoRI or XbaI
1µL SpeI or PstI
5µL NEB Buffer 2.1
qsp 50µL of water
Incubation 60' at 37°C.
Then inactivation of the enzymes for 20’ at 80°C.
Ligation of the cusR box BBa_K1349002 (x2) with cusR box BBa_K1349002 (x1) in piGEM-01.13.
2µL piGEM-01.13
4µL BBa_K1349002 (x2)
4µL BBa_K1349002 (x1)
1µL T4 ligase 10U/µL
2µL T4 ligase buffer
7µL H2O
Transformation of 100µL of E.coli TG1 competent cells with 10µL of ligation. The selective medium is LB-Cm 30µg/mL.
CurR-box characterization
2 clones of each ligation from the 06-10-14 are tested by digestion
The clone 1 of each construction is used for the next steps.
Digestion EP of cusR box (cusBx1)-GFP, cusBx2-GFP and digestion EP of pUC18.
1µg DNA
1µL EcoRI (Promega)
1µL PstI (Promega)
5µL Buffer H of Promega
5µL BSA
qsp 50µL of water
Incubation 60' at 37°C.
Then inactivation of the enzymes for 20’ at 80°C.
Ligation of the cusBx1-GFP or cusBx2-GFP with pUC18.
1µL pUC18
2µL cusBx1-GFP or cusBx2-GFP
0,5µL T4 ligase 10U/µL
2µL T4 ligase buffer
14,5µL H2O
Incubation ON at 16°C
Digestion ES of cusBx3, digestion XP of RBS-GFP (200(2)) and digestion EP of pUC18.
1µg DNA
1µL EcoRI or XbaI(Promega)
1µL PstI or SpeI (Promega)
5µL Buffer H of Promega
5µL BSA
qsp 50µL of water
Incubation 60' at 37°C.
Then inactivation of the enzymes for 20’ at 80°C.
Ligation of cusBx3 with RBS-GFP (200(2)) in pUC18.
1µL pUC18
2µL cusBx3
2µL RBS-GFP (200(2))
0,5µL T4 ligase 10U/µL
2µL T4 ligase buffer
12,5µL H2O
Incubation ON at 16°C
Transformation of 100µL of E.coli TG1 competent cells with 10µL of ligation. The selective medium is LB-Ap 50µg/mL-IPTG 100µg/mL-X-galactose 75µg/mL.
A green colony for each ligation from the 08-10-14 is streaked out on LB-Ap 50µg/mL-IPTG 100µg/mL-X-galactose 75µg/mL.
Preculture of the different strains of interest in LB medium + Ap100 at 37°C O/N
TG1 pUC18-GFP: expresses GFP from a Plac leaky promoter on the pUC18 plasmid
TG1 pUC18-cusRbox(2x)-GFP: same as TG1 pUC18-GFP, with two cusR-box sequences (part BBa_K1349002) inserted between the promoter and the GFP coding sequence.
TG1 pUC18-cusRbox(3x)-GFP: same as TG1 pUC18-GFP, with 3 cusR-box sequences (part BBa_K1349002) inserted between the promoter and the GFP coding sequence.
Week 16 : 10/13/2014 ➡ 10/19/2014
10/13/14
Each preculture was back-diluted 100x  in modified M9 medium (see protocol section), cultivated at 37°C with continuous shaking until they reached an OD600nm=0,4 to 0,6. Then, a 5 mL aliquot of each culture was treated with a sub-lethal concentration of AgNO3 (5 μM, see reference Gudipaty SA et al., 2012) while a second untreated aliquot of 5 mL was used as a control. After Ag treatment, all the cultures were incubated in the dark at 37°C with continuous shaking. Then, cell growth and GFP fluorescence were determined over time by reading the OD at 600 nm and 530 nm, respectively. To do so, 150 μL aliquots were transferred into wells of a black 96-well plate (Greiner), and the absorbance at 600 nm and fluorescence (excitation, 485 nm; emission, 530 nm) were measured with a Tecan Infinite M200 microplate reader.
Growth was determined overtime by following the culture OD at 600 nm on a TECAN instrument (Result section, Figure A).
GFP synthesis was quantified by measuring the OD at 530 nm on a TECAN instrument. Then, fluorescence was normalized by dividing the results with the OD at 600 nm (Result section, Figure B).
See the result section for details on our interpretation of the data.
Week 10
09/01/14 → 09/07/14
Week 11
09/08/14 → 09/14/14
Week 13
09/22/14 → 09/28/14
Week 15
10/06/14 → 10/12/14
Week 16
10/13/14 → 10/19/14
Contact us
Email : equipe.igem.amu@gmail.com
Follow us
iGEM_AMU iGEM.AMU
Our sponsors
Retrieved from " http://2014.igem.org/Team:Aix-Marseille/Notebook_09 "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
