"
Page
Discussion
View source
History
teams
Log in
 
Team:Aix-Marseille/Notebook 08
From 2014.igem.org
Team
Official Team Profile
Project
S.E.colization : Green light, GO!
Beyond the Bench
Parts
Parts
Achievements
Modeling
Notebook
July
August
September/October
Safety
Attributions
Sponsors
Notebook: August
Week 5 : 07/28/2014 ➡ 08/03/2014
07/28/14
07/29/14
07/30/14
07/31/14
08/01/14
08/03/14
Several DNA fragments were amplified thanks to iGEM PCR protocol (Q5 polymerase).
#
DNA Template
Primers Numbers
Primers Names
Weight of amplified DNA
2
Genomic DNA of W3110
6 - 7
serA_mut_ A849G_up
serA_tronq_SufRFC23_down
171 bp
3
Genomic DNA of W3110
5 - 8
serA_PrefRFC10RBS_up
sera_mut_A849_down
861 bp
6
Genomic DNA of W3110
10 - 13
cheA_SufRFC23_down
cheA_mut_T1290C_up
726 bp
8
PiGEM_02.05
16 - 18
relA_mut_C1366_down
relA_sans_stop_SufRFC23_down
882 bp
9
PiGEM_02.05
16 - 19
relA_mut_C1366_down
relA_2_stop_SufRFC23_down
885 bp
10
PiGEM_02.06
20 - 24
Mesh_1_mut_A255T_up
Mesh_1_RFC10-RBS_up
267 bp
11
PiGEM_02.06
21 - 22
Mesh_1_mut_A255T_down
Mesh_1_mut_A343G_up
120 bp
12
PiGEM_02.06
23 - 25
Mesh_1_mut_A343G_down
Mesh1_sans_stop_SufRFC23_down
192 bp
13
PiGEM_02.06
23 - 26
Mesh_1_mut_A343G_down
Mesh_1_2_stop_SufRFC23_down
192 bp
The efficiency of the amplification was tested thanks to agarose gel electrophoresis. Results are presented below.
All DNA fragments seemed to be correctly amplified.
PCR products were purified thanks to Macherey Nagel PCR Clean up Kit.
The DNA concentration of PCR products was tested thanks to the Nanodrop. Results are presented below.
#
Concentration of DNA (ng/µL)
2
52.28
3
44.86
6
49.93
8
73.38
9
76.77
10
57.00
11
39.87
12
50.88
13
62.42
PCR products number 6, 8 and 9 were stocked at -20°C whereas PCR products number 2, 3, 10, 11, 12 and 13 were used to realize SLIC protocol. This protocol permits to assemble all parts of SerA and Mesh1 genes with their directed mutation. The destination plasmid DNA used is the backbone pSB1A3 linearized plasmid. DH5α supercompetent bacteria were transformed with SLIC products and spread over Amp dish.
Several plasmids were purified thanks to Macherey Nagel MiniPrep Kit :
Brick 200 (3)
Brick 200 (4)
Brick 200 (5)
Brick 200 (6)
The efficiency of the purification was tested thanks to agarose gel electrophoresis. Results are presented following.
Brick 200 (6) well did not present any DNA fragment. Other wells contain one or two fragments of DNA, so Brick 200 (3), 200 (4) and 200 (5) were successfully purified.
Several control PCR were realized to verify if SLIC worked. Taq polymerase was used. Three clones of each SLIC were tested. They were also pricked up on Amp dish.
#
DNA Template
Primers Numbers
Primers Names
Weight of amplified DNA
S1
Clone 1 of SLIC SerA
5 - 6
serA_PrefRFC10RBS_up
serA_tronq_SufRFC23_down
1032 bp
S2
Clone 2 of SLIC SerA
5 - 6
serA_PrefRFC10RBS_up
serA_tronq_SufRFC23_down
1032 bp
S3
Clone 3 of SLIC SerA
5 - 6
serA_PrefRFC10RBS_up
serA_tronq_SufRFC23_down
1032 bp
S4
Clone 4 of SLIC Mesh1 without STOP
24 - 25
Mesh_1_RFC10-RBS_up
relA_sans_stop_SufRFC23_down
579 bp
S5
Clone 5 of SLIC Mesh1 without STOP
24 - 25
Mesh_1_RFC10-RBS_up
relA_sans_stop_SufRFC23_down
579 bp
S6
Clone 6 of SLIC Mesh1 without STOP
24 - 25
Mesh_1_RFC10-RBS_up
relA_sans_stop_SufRFC23_down
579 bp
S7
Clone 7 of SLIC Mesh1 with STOP
24 - 26
Mesh_1_RFC10-RBS_up
Mesh_1_2_stop_SufRFC23_down
582 bp
S8
Clone 8 of SLIC Mesh1 with STOP
24 - 26
Mesh_1_RFC10-RBS_up
Mesh_1_2_stop_SufRFC23_down
582 bp
S9
Clone 9 of SLIC Mesh1 with STOP
24 - 26
Mesh_1_RFC10-RBS_up
Mesh_1_2_stop_SufRFC23_down
582 bp
Several DNA fragments were amplified thanks to iGEM PCR protocol (Q5 polymerase).
#
DNA Template
Primers Numbers
Primers Names
Weight of amplified DNA
1
pKD4 plasmid
1 - 2
sdaCsdaB_mut_pKD4_up
sdaCsdaB_ mut_pKD4_down
1032 bp
4
pKD4 plasmid
3 - 4
cheA_mut_pKD4_up
cheA_mut_pKD4_down
1032 bp
5
Genomic DNA of W3110
9 - 14
cheA_PrefRFC23_up
cheA_mut_T1290C_down
1032 bp
7
Genomic DNA of W3110
15 - 17
relA_mut_C1366G_up
relA_RFC10-RBS_up
579 bp
PCR product 1 and 4 were correctly amplified. Thus, they were purified thanks to Macherey Nagel PCR Clean up Kit. PCR product 5 and 7 seemed to be compound with too many different DNA fragments. That's why, a new amplification by PCR was made overnight (iGEM PCR protocol Q5 polymerase).
Realization of "Day 2" of lambda red pKOBEG protocol. W3110 with active recombinase was electroporated by PCR product 1 (sdaCsdaB) and 4 (cheA). Transformed bacteria were spread over a Kan dish.
The efficiency of the purification of PCR product number S1 to S9 was tested thanks to agarose gel electrophoresis. At the same time, the efficiency of the amplification of the beginning sequences of CheA and RelA (PCR product 5 and 7) was tested thanks to agarose gel electrophoresis. Results are presented below.
PCR products 5 and 7 were good for SLIC. Clone 1 and 2 (serA), clone 4 and 6 (Mesh1 without STOP) and clone 7 and 8 (Mesh1 with STOP) were put in culture in LB with Amp.
The DNA concentration of PCR products 5 and 7 was tested thanks to the Nanodrop. Results are presented below.
#
Concentration of DNA (ng/µL)
5
29.78
7
85.51
PCR products number 5, 6, 7, 8 and 9 were used to realize SLIC protocol. This protocol permits to assemble all parts of CheA (5 and 6) and RelA (7 and 8 without STOP, 7 and 9 with STOP) genes with their directed mutation. The destination plasmid DNA used is the backbone pSB1A3 linearized plasmid. DH5α supercompetent bacteria were transformed with SLIC products and spread over Amp dish.
SLIC assembly of RelA was a success but SLIC assembly of CheA failed. That's why, a new SLIC assembly of CheA with PCR products 5 and 6 was made. The destination plasmid DNA used is the backbone pSB1A3 linearized plasmid. DH5α supercompetent bacteria were transformed with SLIC products and spread over Amp dish.
Realization of "Day 2" of lambda red pKOBEG protocol a second time. PCR product 1 and 4 were purified a second time. The efficiency of the purification was tested thanks to agarose gel electrophoresis. Results are presented below. W3110 with active recombinase was electroporated by PCR product 1 (sdaCsdaB) and 4 (cheA). Transformed bacteria were spread over a Kan dish.
Several control PCR were realized to verify whether RelA SLIC worked. Taq polymerase was used. Three clones of each SLIC were tested. They were also pricked up on Amp dish.
#
DNA Template
Primers Numbers
Primers Names
Weight of amplified DNA
S22
Clone 22 of SLIC RelA without Stop
17 - 18
relA_RFC10-RBS_up
relA_sans_stop_SufRFC23_down
2259 bp
S23
Clone 22 of SLIC RelA without Stop
17 - 18
relA_RFC10-RBS_up
relA_sans_stop_SufRFC23_down
2259 bp
S24
Clone 22 of SLIC RelA without Stop
17 - 18
relA_RFC10-RBS_up
relA_sans_stop_SufRFC23_down
2259 bp
S25
Clone 25 of SLIC RelA with STOP
17 - 19
relA_RFC10-RBS_up
relA_2_stop_SufRFC23_down
2262 bp
S26
Clone 26 of SLIC RelA with STOP
17 - 19
relA_RFC10-RBS_up
relA_2_stop_SufRFC23_down
2262 bp
S27
Clone 27 of SLIC RelA with STOP
17 - 19
relA_RFC10-RBS_up
relA_2_stop_SufRFC23_down
2262 bp
The efficiency of the amplification was tested thanks to agarose gel electrophoresis. Results are presented below.
PCR products 1 and 4 were correctly amplified. Thus, they were purified thanks to Macherey Nagel PCR Clean up Kit. Transplanting of S22 to S27 failed, those clones were gave up.
Several plasmids were purified thanks to Macherey Nagel MiniPrep Kit :
Clone 1 and 2 (SerA) ⇒ PiGEM_02.10 and PiGEM_02.11
Clone 4 and 6 (Mesh1 without STOP) ⇒ PiGEM_02.12 and PiGEM_02.13
Clone 7 and 8 (Mesh1 with STOP) ⇒ PiGEM_02.14 and PiGEM_02.15
SLIC assembly of CheA failed. That's why, a new SLIC assembly of CheA with PCR products 5 and 6 was made. The destination plasmid DNA used is the backbone pSB1A3 linearized plasmid. DH5α supercompetent bacteria were transformed with SLIC products and spread over Amp dish.
Several control PCR were realized to verify whether RelA SLIC worked. Taq polymerase was used. Three clones of each SLIC were tested. They were also pricked up on Amp dish.
#
DNA Template
Primers Numbers
Primers Names
Weight of amplified DNA
S28
Clone 28 of SLIC RelA without Stop
17 - 18
relA_RFC10-RBS_up
relA_sans_stop_SufRFC23_down
2259 bp
S29
Clone 28 of SLIC RelA without Stop
17 - 18
relA_RFC10-RBS_up
relA_sans_stop_SufRFC23_down
2259 bp
S30
Clone 28 of SLIC RelA without Stop
17 - 18
relA_RFC10-RBS_up
relA_sans_stop_SufRFC23_down
2259 bp
S31
Clone 31 of SLIC RelA with STOP
17 - 19
relA_RFC10-RBS_up
relA_2_stop_SufRFC23_down
2262 bp
S32
Clone 31 of SLIC RelA with STOP
17 - 19
relA_RFC10-RBS_up
relA_2_stop_SufRFC23_down
2262 bp
S33
Clone 31 of SLIC RelA with STOP
17 - 19
relA_RFC10-RBS_up
relA_2_stop_SufRFC23_down
2262 bp
The efficiency of the amplification was tested thanks to agarose gel electrophoresis. Results are presented below.
PCR product S28, S29, S31, S32 and S33 were good for SLIC. Clones 28 and 29 (RelA without STOP) and clones 31, 32 and 33 (RelA with STOP) were put in culture in LB with Amp.
Realization of "Day 2" of lambda red pKOBEG protocol for the third time. PCR products 1 and 4 were purified a third time. pKOBEG electrocompetent were prepared a third time. W3110 with active recombinase was electroporated with PCR products 1 (sdaCsdaB) and 4 (cheA). Transformed bacteria were spread over a Kan dish.
The efficiency of the purification of PiGEM_02.10, PiGEM_02.11, PiGEM_02.12, PiGEM_02.13, PiGEM_02.14 and PiGEM_02.15 was tested thanks to agarose gel electrophoresis. Results are presented below.
These plasmids seemed to be in low quantity. The DNA concentration of plasmids was tested thanks to the Nanodrop. Results are presented following
#
Concentration of DNA (ng/µL)
PiGEM_02.10
30.6
PiGEM_02.11
37.8
PiGEM_02.12
12.2
PiGEM_02.13
15.11
PiGEM_02.14
14.11
PiGEM_02.15
19.22
The W3110 with active recombinase was electroporated with PCR products 1 (sdaCsdaB) and 4 (cheA). Transformed bacteria were spread over a Kan dish.
Bacteria transformed with PCR products 1 (sdaCsdaB) and 4 (cheA) put in culture on Kan dish on the 08/01/14 showed colonies.
5 different colonies of SdaCsdaB and CheA mutants were spread over Kan dish and Cm30 dish in order to verify whether the pKOBEG plasmid was eliminated (by the previously heat shock) or not as we wanted to, and whether the transformed bacteria contained the kanamycin resistance. The bacteria we want must show colonies on Kan dish and no colonies on Cm 30 dish at the same time.
Protocols used : ...
Week 6 : 08/04/2014 ➡ 08/10/2014
08/04/14
08/05/14
08/06/14
08/07/14
08/08/14
08/09/14
Several plasmids were purified thanks to Macherey Nagel MiniPrep Kit :
SLIC RelA without STOP PCR product 7/8 Clone 28 and 29 ⇒ PiGEM_02.16 and PiGEM_02.17
SLIC RelA with STOP PCR product 7/8 Clone 31, 32 and 33 ⇒ PiGEM_02.18, PiGEM_02.19 and PiGEM_02.20
All plasmids seemed to be correctly purified. No DNA contamination was noticeable.
Several DNA fragments were amplified thanks to iGEM PCR protocol (Q5 polymerase).
#
DNA Template
Primers Numbers
Primers Names
Weight of amplified DNA
14
Genomic DNA of W3110
33 - 34
CusR_iProm-up
CusR_iProm-down
117 bp
15
PiGEM_01.04
37 - 38
BBa_E1010-up
BBa_E1010_NoStop-down
708 bp
16
PiGEM_02.03
39 - 40
BBa_E0040-up
BBa_E0040_NoStop-down
714 bp
17
PiGEM_01.09
45 - 46
(SH3)4-up
(SH3)4-down
932 bp
18
PiGEM_02.08
47 - 48
cusR_L_LZA-up
cusR_L_LZA_down
857 bp
19
PiGEM_02.07
49 - 50
SH3pep_L_LZA-up
SH3pep_L_LZA-down
192 bp
The efficiency of the amplification was tested thanks to agarose gel electrophoresis. Results are presented below.
Fragments 19, 15, 16 and 18 seemed to be correctly amplified. They were going to be cloned in iGEM plasmids. Unfortunately, fragments 14 and 17 were not amplified. Another PCR had to be made.
Several control PCR were realized to verify whether SLIC, bricks 200 and bricks 202 worked. Taq polymerase was used.
#
DNA Template
Primers Numbers
Primers Names
Weight of amplified DNA
20
Clone 1 of SLIC SerA
43 - 44
Prefix_up
Suffix_down
1032 bp
21
Clone 2 of SLIC SerA
43 - 44
Prefix_up
Suffix_down
1032 bp
22
Clone 28 of SLIC SerA
43 - 44
Prefix_up
Suffix_down
2260 bp
23
Clone 29 of SLIC SerA
43 - 44
Prefix_up
Suffix_down
2260 bp
24
Clone 31 of SLIC SerA
43 - 44
Prefix_up
Suffix_down
2260 bp
25
Clone 32 of SLIC SerA
43 - 44
Prefix_up
Suffix_down
2260 bp
26
Clone 33 of SLIC SerA
43 - 44
Prefix_up
Suffix_down
2260 bp
27
Clone 34 of SLIC CheA
43 - 44
Prefix_up
Suffix_down
2432 bp
28
Clone 35 of SLIC CheA
43 - 44
Prefix_up
Suffix_down
2432 bp
29
Clone 36 of SLIC CheA
43 - 44
Prefix_up
Suffix_down
2432 bp
30
Clone 37 of SLIC CheA
43 - 44
Prefix_up
Suffix_down
2432 bp
31
Clone 38 of SLIC CheA
43 - 44
Prefix_up
Suffix_down
2432 bp
32
Clone 39 of SLIC CheA
43 - 44
Prefix_up
Suffix_down
2432 bp
33
Brick 202 (1)
43 - 44
Prefix_up
Suffix_down
730 bp
34
Brick 202 (2)
43 - 44
Prefix_up
Suffix_down
769 bp
35
Brick 200 (3)
43 - 44
Prefix_up
Suffix_down
769 bp
36
Brick 200 (4)
43 - 44
Prefix_up
Suffix_down
769 bp
37
Brick 200 (5)
43 - 44
Prefix_up
Suffix_down
769 bp
The efficiency of the amplification was tested thanks to agarose gel electrophoresis. Results are presented below.
No fragment seemed to be correctly amplified. SLIC were going to be done again with another protocol. Bricks 200 and 202 were also going to be remade.
PiGEM_01.03 (pSB1C3, BBa_B0033) was digested with XbaI and SpeI to be used for stability label cloning.
The efficiency of the digestion was tested thanks to agarose gel electrophoresis. Results are presented below.
PSB1C3 of PiGEM_01.03  seemed to be correctly digested.
PiGEM_01.03 (pSB1C3, BBa_B0033) was digested with EcoRI and SpeI to be used for PCR products 15 and 16 cloning. PCR products 15 and 16 (respectively RFP without STOP codon and GFP without STOP codon) were digested by EcoRI and SpeI. Then, PCR products 15 and 16 were leagued in pSB1C3 digested. DH5 alpha supercompetent were transformed  with these ligation products and spread on Cm dish.
PiGEM_02.27 (pSB1A2, BBa_J04450) was digested with EcoRI and PstI to be used for PCR products 18 and 19 cloning. PCR products 18 and 19 (respectively cusR-L-LZa and SH3pep-L-LZa) were digested by EcoRI and PstI. Then, PCR products 18 and 19 were leagued in pSB1A2 digested. DH5 alpha supercompetent were transformed  with these ligation products and spread on Amp dish.
Because PCR 17 did not work, (SH3)4  was amplified again thanks to iGEM PCR protocol (Q5 polymerase).
#
DNA Template
Primers Numbers
Primers Names
Weight of amplified DNA
38
PiGEM_02.07
49 - 50
SH3pep_L_LZA-up
SH3pep_L_LZA-down
192 bp
The efficiency of the amplification was tested thanks to agarose gel electrophoresis. Results are presented below.
(SH3)4 amplification did not work again. A specific treatment of this cloning was needed.
PiGEM_01.03 (pSB1C3, BBa_B0033) had be digested with XbaI and SpeI the day before to be used for stability label cloning. A gel extract was made this day to collect only the backbone. The backbone obtained was digested by PstI because XbaI and SpeI are compatible restriction sites. Stability labels were assembled by annealing.
Primers numbers
Primers names
Brick created
27 - 28
BBa_K1051206-up
BBa_K1051206-down
BBa_K1051206
29 -30
BBa_K1051207-up
BBa_K1051207-down
BBa_K1051207
31 -32
BBa_K1051208-up
BBa_K1051208-down
BBa_K1051208
Each annealing oligonucleotides were leagued in pSB1C3 digested XSP. DH5 alpha supercompetent were transformed  with these ligation products and spread on Cm dish.
Several plasmids were purified thanks to Macherey Nagel MiniPrep Kit :
SLIC CheA PCR product 5/6 Clone 34, 35, 36, 37, 38 and 39 ⇒ PiGEM_02.21, PiGEM_02.22, PiGEM_02.23, PiGEM_02.24, PiGEM_02.25, PiGEM_02.26
Several control PCR were realized to verify one more time whether SLIC worked. Besides, several control PCR were realized to verify whether bricks 200 and bricks 202 worked. Taq polymerase was used.
#
DNA Template
Primers Numbers
Primers Names
Weight of amplified DNA
39
PiGEM_02.10
5 - 6
serA_PrefRFC10RBS_up
serA_tronq_SufRFC23_down
1032 bp
40
PiGEM_02.11
5 - 6
serA_PrefRFC10RBS_up
serA_tronq_SufRFC23_down
1032 bp
41
PiGEM_02.16
17 - 18
relA_RFC10-RBS_up
relA_sans_stop_SufRFC23_down
2259 bp
42
PiGEM_02.17
17 - 18
relA_RFC10-RBS_up
relA_sans_stop_SufRFC23_down
2259 bp
43
PiGEM_02.18
17 - 19
relA_RFC10-RBS_up
relA_2_stop_SufRFC23_down
2259 bp
44
PiGEM_02.19
17 - 19
relA_RFC10-RBS_up
relA_2_stop_SufRFC23_down
2259 bp
45
PiGEM_02.20
9 - 10
cheA_PrefRFC23_up
cheA_SufRFC23_down
2432 bp
46
PiGEM_02.21
9 - 10
cheA_PrefRFC23_up
cheA_SufRFC23_down
2432 bp
47
PiGEM_02.22
9 - 10
cheA_PrefRFC23_up
cheA_SufRFC23_down
2432 bp
48
PiGEM_02.23
9 - 10
cheA_PrefRFC23_up
cheA_SufRFC23_down
2432 bp
49
PiGEM_02.24
9 - 10
cheA_PrefRFC23_up
cheA_SufRFC23_down
2432 bp
50
PiGEM_02.25
9 - 10
cheA_PrefRFC23_up
cheA_SufRFC23_down
2432 bp
51
PiGEM_02.26
9 - 10
cheA_PrefRFC23_up
cheA_SufRFC23_down
2432 bp
52
PiGEM_01.04
43 - 44
Prefix_up
Suffix_down
706 bp
53
PiGEM_02.10
43 - 44
Prefix_up
Suffix_down
1032 bp
54
Brick 202 (1)
40 - 51
BBa_E0040_NoStop-down
BBa_B0034-up
730 bp
55
Brick 202 (2)
40 - 51
BBa_E0040_NoStop-down
BBa_B0034-up
730 bp
56
Brick 202 (3)
40 - 51
BBa_E0040_NoStop-down
BBa_B0034-up
730 bp
57
Brick 202 (4)
40 - 51
BBa_E0040_NoStop-down
BBa_B0034-up
730 bp
58
Brick 200 (3)
38 - 51
BBa_E0040_NoStop-down
BBa_B0034-up
711 bp
59
Brick 200 (4)
38 - 51
BBa_E0040_NoStop-down
BBa_B0034-up
730 bp
60
Brick 200 (6)
38 - 51
BBa_E0040_NoStop-down
BBa_B0034-up
711 bp
Because PCR 14 did not work, cusR promoter was amplified again thanks to iGEM PCR protocol (Q5 polymerase).
#
DNA Template
Primers Numbers
Primers Names
Weight of amplified DNA
62
Genomic DNA of W3110
33 - 34
CusR_iProm-up
CusR_iProm-down
117 bp
The efficiency of the amplification was tested thanks to agarose gel electrophoresis. Results are presented below.
SLIC had to be made again. Bricks 200 were correctly amplified but bricks 202 were not. CusR promoter was not amplified.
Because PCR 62 did not work, cusR promoter was amplified again thanks to iGEM PCR protocol (Q5 polymerase).
#
DNA Template
Primers Numbers
Primers Names
Weight of amplified DNA
63
Genomic DNA of W3110
33 - 34
CusR_jProm-up
CusR_jProm-down
117 bp
Several control PCR were realized to verify whether bricks 200 worked. Taq polymerase was used.
#
DNA Template
Primers Numbers
Primers Names
Weight of amplified DNA
64
Brick 202 (5)
40 - 51
BBa_E0040_NoStop-down
BBa_B0034-up
730 bp
65
Brick 202 (6)
40 - 51
BBa_E0040_NoStop-down
BBa_B0034-up
730 bp
646
Brick 202 (7)
40 - 51
BBa_E0040_NoStop-down
BBa_B0034-up
730 bp
67
Brick 202 (8)
40 - 51
BBa_E0040_NoStop-down
BBa_B0034-up
730 bp
68
Brick 202 (9)
40 - 51
BBa_E0040_NoStop-down
BBa_B0034-up
730 bp
69
Brick 202 (10)
40 - 51
BBa_E0040_NoStop-down
BBa_B0034-up
730 bp
70
Brick 202 (11)
40 - 51
BBa_E0040_NoStop-down
BBa_B0034-up
730 bp
A gel extract of PCR product 38 (SH3)4 was made.
Several control PCR were realized to verify whether RFP without STOP codon and GFP without STOP codon worked. Taq polymerase was used.
#
DNA Template
Primers Numbers
Primers Names
Weight of amplified DNA
72
RFP without STOP codon
Clone 40
37 - 38
BBa_E1010-up
BBa_E1010_NoStop-down
708 bp
73
RFP without STOP codon
Clone 41
37 - 38
BBa_E1010-up
BBa_E1010_NoStop-down
708 bp
74
RFP without STOP codon
Clone 42
37 - 38
BBa_E1010-up
BBa_E1010_NoStop-down
708 bp
75
RFP without STOP codon
Clone 43
37 - 38
BBa_E1010-up
BBa_E1010_NoStop-down
708 bp
76
RFP without STOP codon
Clone 44
37 - 38
BBa_E1010-up
BBa_E1010_NoStop-down
708 bp
77
GFP without STOP codon
Clone 45
39 - 40
BBa_E0040-up
BBa_E0040_NoStop-down
714 bp
78
GFP without STOP codon
Clone 46
39 - 40
BBa_E0040-up
BBa_E0040_NoStop-down
714 bp
79
GFP without STOP codon
Clone 47
39 - 40
BBa_E0040-up
BBa_E0040_NoStop-down
714 bp
80
GFP without STOP codon
Clone 48
39 - 40
BBa_E0040-up
BBa_E0040_NoStop-down
714 bp
81
GFP without STOP codon
Clone 49
39 - 40
BBa_E0040-up
BBa_E0040_NoStop-down
714 bp
The efficiency of the purification was tested thanks to agarose gel electrophoresis. Results are presented following.
PCR product 63 was correctly amplified. PiGEM_02.27 (pSB1A2, BBa_J04450) was digested with EcoRI and Pst. PCR product 63 (cusR promoter) was digested by EcoRI and PstI. Then, PCR product 63  was leagued in pSB1A2 digested. DH5 alpha supercompetent were transformed  with this ligation product and spread on Amp dish.
To confirm the cloning of stability label and SLIC, each plasmid containing constructions were digested with specific restriction enzymes.
Plasmid Numbers
Plasmid Names
Digestion
Weight of expected bands if constructions are correct
EcoRI
PstI
SpeI
XbaI
PiGEM_01.10
BBa_K1051207
Clone St1
X
X
2082 pb
80 pb
PiGEM_01.11 BBa_K1051207
Clone St2 X X
2082 pb
80 pb
PiGEM_01.12 BBa_K1051208
Clone St3 X X
2082 pb
80 pb
PiGEM_02.10 SLIC SerA produit PCR 2/3
Clone 1 X
2352 pb
847 pb
PiGEM_02.11 SLIC SerA produit PCR 2/3
Clone 2 X
2352 pb
847 pb
PiGEM_02.12 SLIC Mesh1 sans STOP produit PCR 10/11/12
Clone 4 X X
2422 pb
180 pb
84
PiGEM_02.13 SLIC Mesh1 sans STOP produit PCR 10/11/12
Clone 6 X X
2422 pb
180 pb
84
PiGEM_02.14 SLIC Mesh1 sans STOP produit PCR 10/11/13
Clone 7 X X
2422 pb
180 pb
84
PiGEM_02.15 SLIC Mesh1 sans STOP produit PCR 10/11/13
Clone 8 X X
2422 pb
180 pb
84
PiGEM_02.16 SLIC RelA sans STOP produit PCR 7/8
Clone 28 X
3530 pb
872 pb
PiGEM_02.17 SLIC RelA sans STOP produit PCR 7/8
Clone 29 X
3530 pb
872 pb
PiGEM_02.18 SLIC RelA sans STOP produit PCR 7/8
Clone 31 X
3530 pb
872 pb
PiGEM_02.19 SLIC RelA sans STOP produit PCR 7/8
Clone 32 X
3530 pb
872 pb
PiGEM_02.24 SLIC CheA produit PCR 5/6
Clone 37 X
3457 pb
675 pb
PiGEM_02.25 SLIC CheA produit PCR 5/6
Clone 38 X
3457 pb
675 pb
PiGEM_02.26 SLIC CheA produit PCR 5/6
Clone 39 X
3457 pb
675 pb
The efficiency of the digestion was tested thanks to agarose gel electrophoresis. Results are presented below.
For stability label, linearized plasmid was the only fragment visible. For SLIC, no fragment matched with the weight of expected bands. All SLIC construction had to be made again.
PiGEM_02.27 (pSB1A2, BBa_J04450) was digested with XbaI, SpeI and PstI to be used for stability label cloning. Stability labels were assembled again by annealing.
Primers numbers
Primers names
Brick created
27 - 28
BBa_K1051206-up
BBa_K1051206-down
BBa_K1051206
29 -30
BBa_K1051207-up
BBa_K1051207-down
BBa_K1051207
31 -32
BBa_K1051208-up
BBa_K1051208-down
BBa_K1051208
Each annealing oligonucleotides were leagued in pSB1A2 digested XSP. DH5 alpha supercompetent were transformed  with these ligation products and spread on Amp dish.
PiGEM_02.27 (pSB1A2, BBa_J04450) was digested with XbaI and SpeI. CusR box and Flag were assembled by annealing.
Primers numbers
Primers names
Brick created
35 - 36
CusR_Box-up
CusR_Box-down
CusR Box
41 - 42
Flag-up
Flag-down
Flag
Each annealing oligonucleotides were leagued in pSB1A2 digested XS. DH5 alpha supercompetent were transformed  with these ligation products and spread on Amp dish.
SLIC assembly of CheA, SerA, RelA with and without STOP codon and Mesh 1 with and without STOP codon were restarted. DNA fragments were amplified thanks to iGEM PCR protocol (Q5 polymerase).
#
DNA Template
Primers Numbers
Primers Names
Weight of amplified DNA
82
Genomic DNA of W3110
6 - 7
serA_mut_ A849G_up
serA_tronq_SufRFC23_down
171 bp
83
Genomic DNA of W3110
5 - 8
serA_PrefRFC10RBS_up
serA_mut_A849_down
861 bp
84
PiGEM_02.05
15 - 17
relA_mut_C1366G_up
relA_RFC10-RBS_up
1377 bp
85
PiGEM_02.05
16 - 18
relA_mut_C1366_down
relA_sans_stop_SufRFC23_down
882 bp
86
PiGEM_02.05
16 - 19
relA_mut_C1366_down
relA_2_stop_SufRFC23_down
885 bp
87
PiGEM_02.06
20 - 24
Mesh_1_mut_A255T_up
Mesh_1_RFC10-RBS_up
267 bp
88
PiGEM_02.06
21 - 22
Mesh_1_mut_A255T_down
Mesh_1_mut_A343G_up
120 bp
89
PiGEM_02.06
23 - 25
Mesh_1_mut_A343G_down
Mesh1_sans_stop_SufRFC23_down
192 bp
90
PiGEM_02.06
23 - 26
Mesh_1_mut_A343G_down
Mesh_1_2_stop_SufRFC23_down
192 bp
The efficiency of the amplification was tested thanks to agarose gel electrophoresis. Results are presented below.
All DNA fragments were correctly amplified. Thus, SLIC protocol was realized to assemble each fragment of CheA, SerA, RelA and Mesh1. Plasmid pSB1A2 digested XbaI and SpeI was used.
Several control PCR had be realized the day before to verify whether RFP without STOP codon and GFP without STOP codon worked. The efficiency of the amplification was tested thanks to agarose gel electrophoresis. Results are presented following.
Only clones 41, 42 43 and 44 of RPF without STOP codon and clones 47 and 48 of GFP without STOP codon seemed to be correct.
Several plasmids were purified thanks to Macherey Nagel MiniPrep Kit :
PCR product 15 RFP without STOP Clone 41 ⇒ PiGEM_01.14
PCR product 15 RFP without STOP Clone 43 ⇒ PiGEM_01.15
PCR product 16 GFP without STOP Clone 47 ⇒ PiGEM_01.16
PCR product 16 GFP without STOP Clone 48 ⇒ PiGEM_01.17
Protocols used : ...
Week 7 : 08/11/2014 ➡ 08/17/2014
08/11/14
08/12/14
08/13/14
08/14/14
08/15/14
Several control PCR were realized to verify whether cusR promoter, RFP and GFP without STOP codon, cusR-L-LZa, SH3pep, cusR box and SLIC worked. Taq polymerase was used.
#
DNA Template
Primers Numbers
Primers Names
Weight of amplified DNA
106
cusR promoter
Clone 1
43 - 44
Prefix_up
Suffix_down
117 bp
107
cusR promoter
Clone 2
43 - 44
Prefix_up
Suffix_down
117 bp
108
cusR promoter
Clone 3
43 - 44
Prefix_up
Suffix_down
117 bp
109
RFP without STOP codon
Clone 41
43 - 44
Prefix_up
Suffix_down
708 bp
110
RFP without STOP codon
Clone 42
43 - 44
Prefix_up
Suffix_down
708 bp
111
RFP without STOP codon
Clone 43
43 - 44
Prefix_up
Suffix_down
708 bp
112
RFP without STOP codon
Clone 44
43 - 44
Prefix_up
Suffix_down
708 bp
113
RFP without STOP codon
Clone 47
43 - 44
Prefix_up
Suffix_down
708 bp
114
RFP without STOP codon
Clone 48
43 - 44
Prefix_up
Suffix_down
708 bp
126
SH3pep
Clone 4
43 - 44
Prefix_up
Suffix_down
192 bp
127
SH3pep
Clone 6
43 - 44
Prefix_up
Suffix_down
192 bp
128
SH3pep
Clone 8
43 - 44
Prefix_up
Suffix_down
192 bp
129
SH3pep
Clone 10
43 - 44
Prefix_up
Suffix_down
192 bp
130
K1051207
Clone St1
43 - 44
Prefix_up
Suffix_down
86 bp
131
K1051207
Clone St2
43 - 44
Prefix_up
Suffix_down
86 bp
132
K1051207
Clone St3
43 - 44
Prefix_up
Suffix_down
80 bp
133
cusR box
Clone 1
43 - 44
Prefix_up
Suffix_down
117 bp
134
cusR box
Clone 4
43 - 44
Prefix_up
Suffix_down
117 bp
135
cusR box
Clone 5
43 - 44
Prefix_up
Suffix_down
117 bp
136
cusR box
Clone 7
43 - 44
Prefix_up
Suffix_down
117 bp
137
SLIC bis CheA
CA5
43 - 44
Prefix_up
Suffix_down
2432 bp
138
SLIC bis CheA
CA6
43 - 44
Prefix_up
Suffix_down
2432 bp
139
SLIC bis CheA
CA8
43 - 44
Prefix_up
Suffix_down
2432 bp
140
SLIC bis CheA
CA9
43 - 44
Prefix_up
Suffix_down
2432 bp
141
SLIC bis CheA
CA10
43 - 44
Prefix_up
Suffix_down
2432 bp
142
SLIC bis SerA
SA1
43 - 44
Prefix_up
Suffix_down
1032 bp
143
SLIC bis SerA
SA3
43 - 44
Prefix_up
Suffix_down
1032 bp
144
SLIC bis SerA
SA4
43 - 44
Prefix_up
Suffix_down
1032 bp
145
SLIC bis SerA
SA5
43 - 44
Prefix_up
Suffix_down
1032 bp
146
SLIC bis SerA
SA6
43 - 44
Prefix_up
Suffix_down
1032 bp
147
SLIC bis RelA with STOP codon
RA2
43 - 44
Prefix_up
Suffix_down
2260 bp
148
SLIC bis RelA with STOP codon
RA5
43 - 44
Prefix_up
Suffix_down
2260 bp
149
SLIC bis RelA with STOP codon
RA6
43 - 44
Prefix_up
Suffix_down
2260 bp
150
SLIC bis RelA with STOP codon
RA7
43 - 44
Prefix_up
Suffix_down
2260 bp
151
SLIC bis RelA with STOP codon
RA8
43 - 44
Prefix_up
Suffix_down
2260 bp
152
SLIC bis RelA without STOP codon
RAS2
43 - 44
Prefix_up
Suffix_down
2260 bp
153
SLIC bis RelA without STOP codon
RAS3
43 - 44
Prefix_up
Suffix_down
2260 bp
154
SLIC bis RelA without STOP codon
RAS4
43 - 44
Prefix_up
Suffix_down
2260 bp
155
SLIC bis RelA without STOP codon
RAS5
43 - 44
Prefix_up
Suffix_down
2260 bp
156
SLIC bis RelA without STOP codon
RAS6
43 - 44
Prefix_up
Suffix_down
2260 bp
157
SLIC bis Mesh1 without STOP
MSS1
43 - 44
Prefix_up
Suffix_down
579 bp
158
SLIC bis Mesh1 without STOP
MSS2
43 - 44
Prefix_up
Suffix_down
579 bp
159
SLIC bis Mesh1 without STOP
MSS3
43 - 44
Prefix_up
Suffix_down
579 bp
160
SLIC bis Mesh1 without STOP
MSS4
43 - 44
Prefix_up
Suffix_down
579 bp
161
SLIC bis Mesh1 without STOP
MSS5
43 - 44
Prefix_up
Suffix_down
579 bp
162
SLIC bis Mesh1 with STOP
MS1
43 - 44
Prefix_up
Suffix_down
579 bp
163
SLIC bis Mesh1 with STOP
MS2
43 - 44
Prefix_up
Suffix_down
579 bp
164
SLIC bis Mesh1 with STOP
MS3
43 - 44
Prefix_up
Suffix_down
579 bp
165
SLIC bis Mesh1 with STOP
MS4
43 - 44
Prefix_up
Suffix_down
579 bp
166
SLIC bis Mesh1 with STOP
MS5
43 - 44
Prefix_up
Suffix_down
579 bp
167
PiGEM_01.04
43 - 44
Prefix_up
Suffix_down
708 bp
167
PiGEM_01.04
43 - 44
Prefix_up
Suffix_down
708 bp
The efficiency of the amplification was tested thanks to agarose gel electrophoresis. Results are presented below.
Neither cusR promoter no stability label clone seemed to present the right construction, these cloning had to be remade. Nevertheless, every SH3pep and cusR box clones seemed to be correctly amplified. Clones 1 and 4 of cusR box (CB1 and CB4) and clones 4 and 10 of SH3pep (SP4 and SP10) were chosen to go on brick construction.
Clones 42, 43, 44, 47 and 48 of RFP and GFP without STOP codon seemed to be correctly amplified. Only clones 1,2 and 3 of cusR-L-LZa seemed to present the right construction. Clones 42, 43, 47 and 48 of RFP and GFP without STOP codon (RFP42, RFP43, GFP47 and GFP48) and clones 2 and 3 of cusR-L-LZa (CZ2 and CZ3) were chosen to go on brick construction.
No (SH3)4 clone seemed to present the right construction, this cloning had to be remade. Only clones 3 and 4 of Mesh1 without STOP codon (MSS3 and MSS4) seemed to present the right construction, these clones were chosen to go on brick construction. No clone of Mesh1 with STOP codon was correctly amplified, other clones were going to be tested by PCR.
Only clone 5 of SLIC CheA (CA5), clones 2 and 5 of RelA with STOP codon (RA2 and RA5) and clones 4 and 5 of RelA without STOP codon (RAS4 and RAS5) seemed to present the right construction. Thus, only these clones were chosen to go on brick construction.
Clone 1 was the only clone of SLIC SerA correctly amplified. SA1 was chosen to go on brick construction.
TG1 competent cells were prepared to substitute DH5 alpha that grows up too slowly.
PiGEM_02.27 (pSB1A2, BBa_J04450) was digested with XbaI, SpeI and PstI to be used for stability label cloning. Stability labels were assembled a third time by annealing.
Primers numbers
Primers names
Brick created
27 - 28
BBa_K1051206-up
BBa_K1051206-down
BBa_K1051206
29 -30
BBa_K1051207-up
BBa_K1051207-down
BBa_K1051207
31 -32
BBa_K1051208-up
BBa_K1051208-down
BBa_K1051208
Each annealing oligonucleotides were leagued in pSB1A2 digested XSP. DH5 alpha supercompetent were transformed with these ligation products and spread on Amp dish.
Because PCR 106 to 108 did not work, cusR promoter was amplified again thanks to iGEM PCR protocol (Q5 polymerase).
#
DNA Template
Primers Numbers
Primers Names
Weight of amplified DNA
169
Genomic DNA of W3110
33 - 34
CusR_iProm-up
CusR_iProm-down
117 bp
The efficiency of the amplification was tested thanks to agarose gel electrophoresis. Results are presented below.
Once again, the amplification of cusR promoter did not work.
Several control PCR were realized to verify whether SLIC Mesh1 with STOP codon, (SH3)4 worked. Taq polymerase was used.
#
DNA Template
Primers Numbers
Primers Names
Weight of amplified DNA
170
SLIC bis Mesh1 with STOP
MS6
43 - 44
CusR_iProm-up
CusR_iProm-down
579 bp
171
SLIC bis Mesh1 with STOP
MS7
43 - 44
Prefix_up
Suffix_down
579 bp
172
SLIC bis Mesh1 with STOP
MS8
43 - 44
Prefix_up
Suffix_down
579 bp
173
SLIC bis Mesh1 with STOP
MS9
43 - 44
Prefix_up
Suffix_down
579 bp
174
SLIC bis Mesh1 with STOP
MS10
43 - 44
Prefix_up
Suffix_down
579 bp
175
(SH3)4
Clones 11, 12, 13, 14, 15
43 - 44
Prefix_up
Suffix_down
932 bp
176
(SH3)4
Clones 16, 17, 18, 19, 20
43 - 44
Prefix_up
Suffix_down
932 bp
177
(SH3)4
Clones 21, 22, 23, 24, 25
43 - 44
Prefix_up
Suffix_down
932 bp
178
(SH3)4
Clones 26, 27, 28, 29, 30
43 - 44
Prefix_up
Suffix_down
932 bp
179
(SH3)4
Clones 31, 32, 33, 34, 35
43 - 44
Prefix_up
Suffix_down
932 bp
Clone 8 was the only clone of SLIC Mesh1 with STOP codon correctly amplified. MS8 was chosen to go on brick construction. No clone of (SH3)4 seemed to be correctly amplified.
We have control if the insert of FRT-Kan-FRT is in the gene sdaBC and CheA by PCR.
#
DNA Template
Primers Name
Weight of amplified DNA
180
Genomic DNA of W3110
CheA_up
CheA_down
1700
181
CheA-
CheA_up
CheA_down
1500
182
Genomic DNA of W3110
sdaBC_up
sdaBC_down
3100
183
sdaBC- KanR
sdaBC_up
sdaBC_down
1500
184
sdaBC- KanR + pCP20
sdaBC_up
sdaBC_down
1500
CheA- control is unsuccessful, we see a weight of amplification for #180 and #181 of 1700bp  the gene FRT-Kan-FRT isn't insert in cheA.
>SdaBC- control have a problem we don't have amplification for #182 to #184.
Results not shown.
Several control PCR were realized to verify whether stability label and (SH3)4 cloning worked. Taq polymerase was used.
#
DNA Template
Primers Numbers
Primers Names
Weight of amplified DNA
195
K1051206
Clone St4
43 - 44
Prefix_up
Suffix_down
80 bp
196
K1051206
Clone St5
43 - 44
Prefix_up
Suffix_down
80 bp
197
K1051206
Clone St6
43 - 44
Prefix_up
Suffix_down
80 bp
198
K1051206
Clone St7
43 - 44
Prefix_up
Suffix_down
80 bp
199
K1051206
Clone St8
43 - 44
Prefix_up
Suffix_down
80 bp
200
K1051206
Clone St9
43 - 44
Prefix_up
Suffix_down
80 bp
201
K1051206
Clone St10
43 - 44
Prefix_up
Suffix_down
80 bp
202
K1051206
Clone St11
43 - 44
Prefix_up
Suffix_down
80 bp
203
K1051206
Clone St12
43 - 44
Prefix_up
Suffix_down
80 bp
204
K1051206
Clone St13
43 - 44
Prefix_up
Suffix_down
80 bp
205
K1051207
Clone St14
43 - 44
Prefix_up
Suffix_down
86 bp
206
K1051207
Clone St15
43 - 44
Prefix_up
Suffix_down
86 bp
207
K1051207
Clone St16
43 - 44
Prefix_up
Suffix_down
86 bp
208
K1051207
Clone St17
43 - 44
Prefix_up
Suffix_down
86 bp
209
K1051207
Clone St18
43 - 44
Prefix_up
Suffix_down
86 bp
210
K1051207
Clone St19
43 - 44
Prefix_up
Suffix_down
86 bp
211
K1051207
Clone St20
43 - 44
Prefix_up
Suffix_down
86 bp
212
K1051207
Clone St21
43 - 44
Prefix_up
Suffix_down
86 bp
213
K1051207
Clone St22
43 - 44
Prefix_up
Suffix_down
86 bp
214
K1051207
Clone St23
43 - 44
Prefix_up
Suffix_down
86 bp
215
K1051208
Clone St24
43 - 44
Prefix_up
Suffix_down
80 bp
216
K1051208
Clone St25
43 - 44
Prefix_up
Suffix_down
80 bp
217
K1051208
Clone St26
43 - 44
Prefix_up
Suffix_down
80 bp
218
K1051208
Clone St27
43 - 44
Prefix_up
Suffix_down
80 bp
219
K1051208
Clone St28
43 - 44
Prefix_up
Suffix_down
80 bp
220
K1051208
Clone St29
43 - 44
Prefix_up
Suffix_down
80 bp
221
K1051208
Clone St30
43 - 44
Prefix_up
Suffix_down
80 bp
222
K1051208
Clone St31
43 - 44
Prefix_up
Suffix_down
80 bp
223
K1051208
Clone St32
43 - 44
Prefix_up
Suffix_down
80 bp
224
K1051208
Clone St33
43 - 44
Prefix_up
Suffix_down
80 bp
225
(SH3)4Clones sc1
43 - 44
Prefix_up
Suffix_down
932 bp
226
(SH3)4Clones sc2
43 - 44
Prefix_up
Suffix_down
932 bp
227
(SH3)4Clones sc3
43 - 44
Prefix_up
Suffix_down
932 bp
228
(SH3)4Clones sc4
43 - 44
Prefix_up
Suffix_down
932 bp
229
(SH3)4Clones sc5
43 - 44
Prefix_up
Suffix_down
932 bp
230
(SH3)4Clones sc6
43 - 44
Prefix_up
Suffix_down
932 bp
231
(SH3)4Clones sc7
43 - 44
Prefix_up
Suffix_down
932 bp
232
(SH3)4Clones sc8
43 - 44
Prefix_up
Suffix_down
932 bp
233
(SH3)4Clones sc9
43 - 44
Prefix_up
Suffix_down
932 bp
234
(SH3)4Clones sc10
43 - 44
Prefix_up
Suffix_down
932 bp
Clone St8 was the only clone of K1051206 correctly amplified. St8 was chosen to go on brick construction.
Only clones St16, 17, 19, 20 and 21 of K1051207 seemed to present the right construction. These clones were chosen to go on brick construction.
Clones 24, 26, 27, 28, 29 and 33 of K1051208 seemed to be correctly amplified. These clones were chosen to go on brick construction.
No clone of (SH3)4 seemed to present the right construction.
We made a PCR with new oligo to control mutants.
#
DNA Template
Primers Name
Weight of amplified DNA
185
Genomic DNA of W3110
CheA_check_up
CheA_check_down
1700
186
CheA-
CheA_check_up
CheA_check_down
1500
187
Genomic DNA of W3110
sdaBC_check_up
sdaBC_check_down
3100
188
sdaBC- KanR
sdaBC_check_up
sdaBC_check_down
1500
189
sdaBC- KanR + pCP20
sdaBC_check_up
sdaBC_check_down
1500
190
Genomic DNA of W3110
FRT_up
sdaBC_check_down
0
191
sdaBC- KanR
FRT_up
sdaBC_check_down
1600
192
sdaBC- KanR + pCP20
FRT_up
sdaBC_check_down
1600
Second control of CheA- show us the gene FRT-Kan-FRT isn't in the gene cheA
SdaBC control have an other problem with amplification
Results not shown.
Several plasmids were purified thanks to Macherey Nagel MiniPrep Kit :
PCR product 15 RFP without STOP codon Clones 42 and 43 ⇒ PiGEM_01.18 and PiGEM_01.19
PCR product 16 GFP without STOP codon Clones 47 and 48 ⇒ PiGEM_01.20 and PiGEM_01.21
CusRbox clones 1 and 4 (CB1 and CB4) ⇒ PiGEM_02.33 and PiGEM_02.34
SH3 pep L-LZa clones 4 and 10 (SP4 and SP10) ⇒ PiGEM_02.35 and PiGEM_02.36
CusR-L-LZa clones 2 and 3 (CZ2 and CZ3) ⇒ PiGEM_02.37 and PiGEM_02.38
SLIC CheA Bis Clone CA5 ⇒ PiGEM_02.39
SLIC SerA Bis Clone SA1 ⇒ PiGEM_02.40
SLIC RelA Bis avec STOP Clones RA2 and RA5 ⇒ PiGEM_02.41 and PiGEM_02.42
SLIC RelA Bis sans STOP Clone RAS4 and RAS5 ⇒ PiGEM_02.43 and PiGEM_02.44
SLIC Mesh1 Bis sans STOP Clone MSS3 and MSS4 ⇒ PiGEM_02.45 and PiGEM_02.46
SLIC CheA Clone 38 ⇒ PiGEM_02.47
SLIC Mesh1 avec STOP MS8 ⇒ PiGEM_02.48
CusR promoter  was amplified again thanks to iGEM PCR protocol (Q5 polymerase) by switching matrix.
#
DNA Template
Primers Numbers
Primers Names
Weight of amplified DNA
238
Genomic DNA of W3110
33 - 34
CusR_iProm-up
CusR_iProm-down
117 bp
239
PiGEM_02.08
33 - 34
CusR_iProm-up
CusR_iProm-down
117 bp
The efficiency of the amplification was tested thanks to agarose gel electrophoresis. Results are presented below.
The amplification from the genomique template did not work again but the amplification from the Berkeley plasmid worked very well. CusR promoter cloning could be done again.
Several control PCR were realized to verify whether (SH3)4 worked. Taq polymerase was used.
#
DNA Template
Primers Numbers
Primers Names
Weight of amplified DNA
240
(SH3)4
Clones C4.1
43 - 44
CusR_iProm-up
CusR_iProm-down
932 bp
240
(SH3)4
Clones C4.2
43 - 44
CusR_iProm-up
CusR_iProm-down
932 bp
240
(SH3)4
Clones C4.3
43 - 44
CusR_iProm-up
CusR_iProm-down
932 bp
240
(SH3)4
Clones C4.4
43 - 44
CusR_iProm-up
CusR_iProm-down
932 bp
240
(SH3)4
Clones C4.5
43 - 44
CusR_iProm-up
CusR_iProm-down
932 bp
240
(SH3)4
Clones C5.1
43 - 44
CusR_iProm-up
CusR_iProm-down
932 bp
240
(SH3)4
Clones C5.2
43 - 44
CusR_iProm-up
CusR_iProm-down
932 bp
240
(SH3)4
Clones C5.3
43 - 44
CusR_iProm-up
CusR_iProm-down
932 bp
240
(SH3)4
Clones C5.4
43 - 44
CusR_iProm-up
CusR_iProm-down
932 bp
240
(SH3)4
Clones C5.5
43 - 44
CusR_iProm-up
CusR_iProm-down
932 bp
240
(SH3)4
Clones C6.1
43 - 44
CusR_iProm-up
CusR_iProm-down
932 bp
240
(SH3)4
Clones C6.2
43 - 44
CusR_iProm-up
CusR_iProm-down
932 bp
240
(SH3)4
Clones C6.3
43 - 44
CusR_iProm-up
CusR_iProm-down
932 bp
240
(SH3)4
Clones C6.4
43 - 44
CusR_iProm-up
CusR_iProm-down
932 bp
240
(SH3)4
Clones C6.5
43 - 44
CusR_iProm-up
CusR_iProm-down
932 bp
The efficiency of the amplification was tested thanks to agarose gel electrophoresis. Results are presented below.
No clone of (SH3)4 seemed to present the right construction.
Several control PCR were realized to verify whether stability label K1051206 and flag cloning worked. Taq polymerase was used.
#
DNA Template
Primers Numbers
Primers Names
Weight of amplified DNA
255
K1051206
Clone St34
43 - 44
Prefix_up
Suffix_down
80 bp
256
K1051206
Clone St35
43 - 44
Prefix_up
Suffix_down
80 bp
257
K1051206
Clone St36
43 - 44
Prefix_up
Suffix_down
80 bp
258
K1051206
Clone St37
43 - 44
Prefix_up
Suffix_down
80 bp
259
K1051206
Clone St39
43 - 44
Prefix_up
Suffix_down
80 bp
260
Flag
Clone 1
43 - 44
Prefix_up
Suffix_down
80 bp
261
Flag
Clone 2
43 - 44
Prefix_up
Suffix_down
80 bp
262
Flag
Clone 3
43 - 44
Prefix_up
Suffix_down
80 bp
263
Flag
Clone 4
43 - 44
Prefix_up
Suffix_down
80 bp
264
Flag
Clone 5
43 - 44
Prefix_up
Suffix_down
80 bp
265
Flag
Clone 6
43 - 44
Prefix_up
Suffix_down
86 bp
266
Flag
Clone 7
43 - 44
Prefix_up
Suffix_down
86 bp
267
Flag
Clone 8
43 - 44
Prefix_up
Suffix_down
86 bp
268
Flag
Clone 9
43 - 44
Prefix_up
Suffix_down
86 bp
269
Flag
Clone 10
43 - 44
Prefix_up
Suffix_down
86 bp
The efficiency of the amplification was tested thanks to agarose gel electrophoresis. Results are presented below.
No clone of K1051206 seemed to be correctly amplified. Nevertheless, every Flag clone seemed to present the right construction. Fl1 and Fl3 clones were chosen to go on brick construction.
We made a gradient PCR 55° to 65° with oligo sdaBC_checkdo/FRTup and W3110 sdaBC- strain and W3110 Wild type.
No amplification shown by the electrophoresis for all these PCR.
Results not shown
Several plasmids were purified thanks to Macherey Nagel MiniPrep Kit :
Stability label 27/28 K1051206 clone St8 ⇒ PiGEM_02.49
Stability label 29/30 K1051207 clones St16, 17, 19, 20 and 21 ⇒ PiGEM_02.50, PiGEM_02.51, PiGEM_02.52, PiGEM_02.53 and PiGEM_02.54
Stability label 31/32 K1051208 clones St24, 26, 27, 28, 29 and 33 ⇒ PiGEM_02.55, PiGEM_02.56, PiGEM_02.57, PiGEM_02.58, PiGEM_02.59 and PiGEM_02.60
We made Q5 PCR with oligo 1-2 and 3-4 on pKD4 template for prepare new transformation sdaBC and cheA respectively #293 and #294. After we supress the template pKD4 with
We purified PCR products #293 thanks to Macherey Nagel PCR Clean up Kit.
293
22.65 ng.mL-1
We made 2 new PCR on transformed strains. With specific hybridation temperature.
#
DNA Template
Primers Name
Weight of amplified DNA
Hybridation Temperature
300A
CheA-
CheA_check_up
CheA_check_down
1500
55°C
301A
sdaBC- KanR
FRT_up
sdaBC_check_down
1600
50°C
Week 8 : 08/18/2014 ➡ 08/24/2014
08/18/14
08/19/14
08/20/14
08/21/14
08/22/14
Plasmids containing the stability labels 27/28 K1051206 clone St8, the stability labels 29/30 K1051207 clones St16, 17, 19, 20 and 21 and the stability labels 31/32 K1051208 clones St24, 26, 27, 28, 29 and 33 were digested by XbaI to determine whether they presented the expected construction.
The efficiency of the digestion was tested thanks to agarose gel electrophoresis. Results are presented below.
The weight of digested fragment were too heavy. It was impossible that only one stability label was inserted in one iGEM plasmid. Certainly, several stability labels were associated in a same plasmid. The cloning of stability label must have be remade with new ratios of stability label and plasmid.
PiGEM_02.27 (pSB1A2, BBa_J04450) was digested with EcoRI and PstI to be used for PCR product 239 cloning. PCR product 239 (cusR promoter) was digested by EcoRI and PstI. Then, PCR product 239 was leagued in pSB1A2 digested. Tg1 competent cells were transformed  with these ligation products and spread on Amp dish.
Bricks 200 and 202 were built again by changing destination plasmid. 3A protocol was implemented to assemble the upstream part with downstream part and with the destination plasmid. Tg1 competent cells were transformed  with these ligation products and spread on Kan or Cm dish. The cloning plan is presented below.
Part Number
Description
Upstream Part
Downstream Part
Destination Plasmid
200
Reference RBS with RFP with stop codon
PiGEM_02.29 Amp
PiGEM_01.04 Cm
PiGEM_02.28 Kan
202
Reference RBS with GFP with stop codon
PiGEM_02.29 Amp
PiGEM_02.03 Amp
PiGEM_01.13 Cm
Several control PCR were realized to verify whether cusR promoter cloning worked. Taq polymerase was used.
#
DNA Template
Primers Numbers
Primers Names
Weight of amplified DNA
AS1
cusR promoter
Clone 1
43 - 44
Prefix_up
Suffix_down
120 bp
AS2
cusR promoter
Clone 2
43 - 44
Prefix_up
Suffix_down
120 bp
AS3
cusR promoter
Clone 3
43 - 44
Prefix_up
Suffix_down
120 bp
AS4
cusR promoter
Clone 4
43 - 44
Prefix_up
Suffix_down
120 bp
AS5
cusR promoter
Clone 5
43 - 44
Prefix_up
Suffix_down
120 bp
AS6
cusR promoter
Clone 6
43 - 44
Prefix_up
Suffix_down
120 bp
AS7
cusR promoter
Clone 7
43 - 44
Prefix_up
Suffix_down
120 bp
AS8
cusR promoter
Clone 8
43 - 44
Prefix_up
Suffix_down
120 bp
AS9
cusR promoter
Clone 9
43 - 44
Prefix_up
Suffix_down
120 bp
AS10
cusR promoter
Clone 10
43 - 44
Prefix_up
Suffix_down
120 bp
The efficiency of the amplification was tested thanks to agarose gel electrophoresis. Results are presented below.
The majority of clones seemed to present the right construction. Clones 2 and 8 (PCU2 and PCU8) were chosen to go on brick construction.
Bricks 201, 203 to 207, 211, 301 and 303 were built. 3A protocol was implemented to assemble the upstream part with downstream part and with the destination plasmid. Tg1 competent cells were transformed with these ligation products and spread on Kan or Cm dish. The cloning plan is presented below.
Part Number
Description
Upstream Part
Downstream Part
Destination Plasmid
201
Reference RBS with RFP without stop codon
PiGEM_02.29 Amp
PiGEM_01.18 Cm
PiGEM_01.19 Cm
PiGEM_02.28 Kan
203
Reference RBS with GFP without stop codon
PiGEM_02.29 Amp
PiGEM_01.20 Cm
PiGEM_01.21 Cm
PiGEM_02.28 Kan
204
Reference RBS with RelA with stop codon
PiGEM_02.29 Amp
PiGEM_01.40 Amp
PiGEM_01.41 Amp
PiGEM_02.13 Cm
205
Reference RBS with RelA without stop codon
PiGEM_02.29 Amp
PiGEM_01.43 Amp
PiGEM_01.44 Amp
PiGEM_02.13 Cm
206
Reference RBS with Mesh1 with stop codon
PiGEM_02.29 Amp
PiGEM_01.48 Amp
PiGEM_02.13 Cm
207
Reference RBS with Mesh1 without stop codon
PiGEM_02.29 Amp
PiGEM_01.45 Amp
PiGEM_01.46 Amp
PiGEM_02.13 Cm
211
Stability 3 with double terminator
PiGEM_02.29 Amp
PiGEM_01.33 Amp
PiGEM_01.34 Amp
PiGEM_02.13 Cm
301
SH3pep-LZA with double terminator
PiGEM_02.35 Amp
PiGEM_02.36 Amp
PiGEM_01.07 Cm
PiGEM_02.28 Kan
303
RBS with CusR-LZa protein
PiGEM_02.29 Amp
PiGEM_02.37 Amp
PiGEM_02.38 Amp
PiGEM_01.13 Cm
PiGEM_02.27 (pSB1A2, BBa_J04450) was digested with XbaI, SpeI and PstI to be used for stability label cloning. Stability labels were assembled again by annealing.
Primers numbers
Primers names
Brick created
27 - 28
BBa_K1051206-up
BBa_K1051206-down
BBa_K1051206
29 -30
BBa_K1051207-up
BBa_K1051207-down
BBa_K1051207
31 -32
BBa_K1051208-up
BBa_K1051208-down
BBa_K1051208
Each annealing oligonucleotides were leagued in pSB1A2 digested XSP with ratio 1:1 (0.0047 ng of stability label for 117 ng of plasmid) and 1:2 (0.0094 ng of stability label for 117 ng of plasmid). Tg1 competent cells were transformed  with these ligation products and spread on Amp dish.
We made a PCR with new oligo to control mutants.
#
DNA Template
Primers Name
Weight of amplified DNA
185
Genomic DNA of W3110
CheA_check_up
CheA_check_down
1700
186
CheA-
CheA_check_up
CheA_check_down
1500
187
Genomic DNA of W3110
sdaBC_check_up
sdaBC_check_down
3100
188
sdaBC- KanR
sdaBC_check_up
sdaBC_check_down
1500
189
sdaBC- KanR + pCP20
sdaBC_check_up
sdaBC_check_down
1500
190
Genomic DNA of W3110
FRT_up
sdaBC_check_down
0
191
sdaBC- KanR
FRT_up
sdaBC_check_down
1600
192
sdaBC- KanR + pCP20
FRT_up
sdaBC_check_down
1600
After the electrophoresis, we can confirm The strain sdaBC- KanR is successful. We select 3 strains with electrophoresis control
C.23 and C.33 are contaminated. So we select C.43, C.53 and C.63.
Several control PCR were realized to verify whether Bricks 200 and 202 constructions worked. Taq polymerase was used.
#
DNA Template
Primers Numbers
Primers Names
Weight of amplified DNA
AS11
Brick 200
Clone RFP1
43 - 44
Prefix_up
Suffix_down
758 bp
AS12
Brick 200
Clone RFP2
43 - 44
Prefix_up
Suffix_down
758 bp
AS13
Brick 200
Clone RFP3
43 - 44
Prefix_up
Suffix_down
758 bp
AS14
Brick 200
Clone RFP4
43 - 44
Prefix_up
Suffix_down
758 bp
AS15
Brick 200
Clone RFP5
43 - 44
Prefix_up
Suffix_down
758 bp
AS16
Brick 202
Clone GFP1
43 - 44
Prefix_up
Suffix_down
772 bp
AS17
Brick 202
Clone GFP2
43 - 44
Prefix_up
Suffix_down
772 bp
AS18
Brick 202
Clone GFP3
43 - 44
Prefix_up
Suffix_down
772 bp
AS19
Brick 202
Clone GFP4
43 - 44
Prefix_up
Suffix_down
772 bp
AS20
Brick 202
Clone GFP5
43 - 44
Prefix_up
Suffix_down
772 bp
AS21
Brick 200
Clone RFP1
38 - 51
BBa_E1010_NoStop-down
BBa_B0034-up
718 bp
AS22
Brick 200
Clone RFP2
38 - 51
BBa_E1010_NoStop-down
BBa_B0034-up
718 bp
AS23
Brick 200
Clone RFP3
38 - 51
BBa_E1010_NoStop-down
BBa_B0034-up
718 bp
AS24
Brick 200
Clone RFP4
38 - 51
BBa_E1010_NoStop-down
BBa_B0034-up
718 bp
AS25
Brick 200
Clone RFP5
38 - 51
BBa_E1010_NoStop-down
BBa_B0034-up
718 bp
AS26
Brick 202
Clone GFP1
40 - 51
BBa_E0040_NoStop-down
BBa_B0034-up
732 bp
AS27
Brick 202
Clone GFP2
40 - 51
BBa_E0040_NoStop-down
BBa_B0034-up
732 bp
AS28
Brick 202
Clone GFP3
40 - 51
BBa_E0040_NoStop-down
BBa_B0034-up
732 bp
AS29
Brick 202
Clone GFP4
40 - 51
BBa_E0040_NoStop-down
BBa_B0034-up
732 bp
AS30
Brick 202
Clone GFP5
40 - 51
BBa_E0040_NoStop-down
BBa_B0034-up
732 bp
The efficiency of the amplification was tested thanks to agarose gel electrophoresis. Results are presented below.
All clones seemed to present the right construction. Clones 200(3), 200(5), 202(1) and 202(3) were chosen to go on brick construction.
Several plasmids were purified thanks to Macherey Nagel MiniPrep Kit :
CusR promoter clones 2 and 8 (PCU2 and PCU8) ⇒ PiGEM_02.61 and PiGEM_02.62
We prepare seven starter of W3110 sdaBC- KanR strain with Kan50.
Sequencing results of plasmids PiGEM_02.33, PiGEM_02.34, PiGEM_02.35, PiGEM_02.36, PiGEM_02.37, PiGEM_02.38, PiGEM_02.39, PiGEM_02.40, PiGEM_02.41, PiGEM_02.42, PiGEM_02.43, PiGEM_02.44, PiGEM_02.45, PiGEM_02.46, PiGEM_02.47, PiGEM_02.48 were received. Thus, clones CB1, SP10, CZ3, CA38, RA2, MS8, MSS4, GFP47, RFP43, RAS5 and PCU2 had the right construction. Because no clone of SLIC SerA was correct, several control PCR were realized to verify whether other clones of SLIC SerA had the right construction. Taq polymerase was used.
#
DNA Template
Primers Numbers
Primers Names
Weight of amplified DNA
AS31
SLIC bis SerA
SA9
43 - 44
Prefix_up
Suffix_down
1049 bp
AS32
SLIC bis SerA
SA11
43 - 44
Prefix_up
Suffix_down
1049 bp
AS33
SLIC bis SerA
SA12
43 - 44
Prefix_up
Suffix_down
1049 bp
AS34
SLIC bis SerA
SA13
43 - 44
Prefix_up
Suffix_down
1049 bp
AS35
SLIC bis SerA
SA14
43 - 44
Prefix_up
Suffix_down
1049 bp
AS36
SLIC bis SerA
SA15
43 - 44
Prefix_up
Suffix_down
1049 bp
The efficiency of the amplification was tested thanks to agarose gel electrophoresis. Results are presented below.
There were bands at the right molecular weight but clone SA1 got the same band (AS36). Nevertheless, this clone did not have the right construction (bad sequencing result). Thus, SerA was going to be assembled again by another protocol (Gibson protocol).
Plasmids containing the flag (Fl1 and Fl3) were digested by XbaI to determine whether they presented the expected construction.
The efficiency of the digestion was tested thanks to agarose gel electrophoresis. Results are presented below.
Clone Fl1 and Fl3 seemed to present the expected construction.
We realize day 7 of Lambda Red protocol on W3110 sdaBC- pCP20 strain.
Several plasmids were purified thanks to Macherey Nagel MiniPrep Kit :
Bricks 200 (3) and 200 (5) ⇒ PiGEM_02.65 and PiGEM_02.66
Bricks 202 (1) and 202 (3) ⇒ PiGEM_02.67 and PiGEM_02.68
Unfortunately, colonies careering bricks 200 and 202 were respectively red and green while constructions of these bricks were not supposed to contain any promoter. Thus, PiGEM_02.65, PiGEM_02.66, PiGEM_02.67 and PiGEM_02.68 were sent in order to be sequenced. In the mean time, RFP and GFP were assembled to another RBS to test whether the RBS B0034 was correct. The cloning plan is presented below.
Part Number
Description
Upstream Part
Downstream Part
Destination Plasmid
A
RBS B0030 with RFP with stop codon
PiGEM_01.05 Cm
PiGEM_01.04 Cm
PiGEM_01.13 Cm
B
RBS B0030 with GFP with stop codon
PiGEM_01.05 Cm
PiGEM_02.03 Cm
PiGEM_01.13 Cm
Several control PCR were realized to verify whether stability labels cloning with different ratios worked. Taq polymerase was used.
#
DNA Template
Primers Numbers
Primers Names
Weight of amplified DNA
CB1
K1051206 Ratio 1:1
Clone St40
43 - 44
Prefix_up
Suffix_down
80 bp
CB2
K1051206 Ratio 1:1
Clone St41
43 - 44
Prefix_up
Suffix_down
80 bp
CB3
K1051206 Ratio 1:1
Clone St42
43 - 44
Prefix_up
Suffix_down
80 bp
CB4
K1051206 Ratio 1:1
Clone St43
43 - 44
Prefix_up
Suffix_down
80 bp
CB5
K1051206 Ratio 1:1
Clone St44
43 - 44
Prefix_up
Suffix_down
80 bp
CB6
K1051206 Ratio 1:2
Clone St45
43 - 44
Prefix_up
Suffix_down
80 bp
CB7
K1051206 Ratio 1:2
Clone St46
43 - 44
Prefix_up
Suffix_down
80 bp
C86
K1051206 Ratio 1:2
Clone St47
43 - 44
Prefix_up
Suffix_down
80 bp
CB9
K1051206 Ratio 1:2
Clone St48
43 - 44
Prefix_up
Suffix_down
80 bp
CB10
K1051206 Ratio 1:2
Clone St49
43 - 44
Prefix_up
Suffix_down
80 bp
CB11
K1051207 Ratio 1:1
Clone St50
43 - 44
Prefix_up
Suffix_down
86 bp
CB12
K1051207 Ratio 1:1
Clone St51
43 - 44
Prefix_up
Suffix_down
86 bp
CB13
K1051207 Ratio 1:1
Clone St52
43 - 44
Prefix_up
Suffix_down
86 bp
CB14
K1051207 Ratio 1:1
Clone St53
43 - 44
Prefix_up
Suffix_down
86 bp
CB15
K1051207 Ratio 1:1
Clone St54
43 - 44
Prefix_up
Suffix_down
86 bp
CB16
K1051207 Ratio 1:2
Clone St55
43 - 44
Prefix_up
Suffix_down
86 bp
CB17
K1051207 Ratio 1:2
Clone St56
43 - 44
Prefix_up
Suffix_down
86 bp
CB18
K1051207 Ratio 1:2
Clone St57
43 - 44
Prefix_up
Suffix_down
86 bp
CB19
K1051207 Ratio 1:2
Clone St58
43 - 44
Prefix_up
Suffix_down
86 bp
CB20
K1051207 Ratio 1:2
Clone St59
43 - 44
Prefix_up
Suffix_down
86 bp
CB21
K1051208 Ratio 1:1
Clone St60
43 - 44
Prefix_up
Suffix_down
80 bp
CB22
K1051208 Ratio 1:1
Clone St61
43 - 44
Prefix_up
Suffix_down
80 bp
CB23
K1051208 Ratio 1:1
Clone St62
43 - 44
Prefix_up
Suffix_down
80 bp
CB24
K1051208 Ratio 1:1
Clone St63
43 - 44
Prefix_up
Suffix_down
80 bp
CB25
K1051208 Ratio 1:1
Clone St64
43 - 44
Prefix_up
Suffix_down
80 bp
CB26
K1051208 Ratio 1:2
Clone St65
43 - 44
Prefix_up
Suffix_down
80 bp
CB27
K1051208 Ratio 1:2
Clone St66
43 - 44
Prefix_up
Suffix_down
80 bp
CB28
K1051208 Ratio 1:2
Clone St67
43 - 44
Prefix_up
Suffix_down
80 bp
CB29
K1051208 Ratio 1:2
Clone St68
43 - 44
Prefix_up
Suffix_down
80 bp
CB30
K1051208 Ratio 1:2
Clone St69
43 - 44
Prefix_up
Suffix_down
80 bp
The efficiency of the amplification was tested thanks to agarose gel electrophoresis. Results are presented below.
Clone Fl1 and Fl3 seemed to present the expected construction.
Unfortunately, there was not any expected result. Stability labels constructions did not success once again. New oligonucleotides were going to be ordered to try a new protocol.
We realize day 8 of Lambda Red protocol on W3110 sdaBC- strain.
Week 9 : 08/25/2014 ➡ 08/31/2014
08/25/14
08/26/14
08/27/14
08/28/14
RelA (RA2) and Mesh1 (MS8) were assembled with a Lac promoter and the reference RBS (BBa_B0034) to characterize parts. 3A protocol was implemented to assemble the upstream part with downstream part and with the destination plasmid. Tg1 competent cells were transformed  with these ligation products and spread on Kan glucose dish to prevent RelA and Mesh1 expression that could slow cells growth. The cloning plan is presented below.
Part Number
Description
Upstream Part
Downstream Part
Destination Plasmid
x
Promoter with RBS and RelA with stop codon
PiGEM_01.08 Cm
PiGEM_02.41 Amp
PiGEM_02.28 Kan
y
Promoter with RBS and Mesh1 with stop codon
PiGEM_01.08 Cm
PiGEM_02.48 Amp
PiGEM_02.28 Kan
We realize day 9 of Lambda Red protocol on W3110 sdaBC- strain, and day 2 of Lambda Red protocol on W3110 sdaBC- strain with #294 PCR product
Sequencing results of plasmids PiGEM_02.65, PiGEM_02.66, PiGEM_02.67 and PiGEM_02.68 were received. PiGEM_02.67 and PiGEM_02.68 had the right construction but PiGEM_02.65 and PiGEM_02.66 had not. Thus, RBS BBa_B0034 was correct. There was no reason for the colonies containing bricks 200 and 202 to be green or red without promoter sequence.
Several control PCR were realized to verify whether RelA and Mesh1 cloning with their promoter and RBS worked. (SH3)2 new construction were also tested. Taq polymerase was used.
#
DNA Template
Primers Numbers
Primers Names
Weight of amplified DNA
AS43
BBa_J04500 with RelA
Clone 1
43 - 44
Prefix_up
Suffix_down
2480 bp
AS44
BBa_J04500 with RelA
Clone 2
43 - 44
Prefix_up
Suffix_down
2480 bp
AS45
BBa_J04500 with RelA
Clone 3
43 - 44
Prefix_up
Suffix_down
2480 bp
AS46
BBa_J04500 with RelA
Clone 4
43 - 44
Prefix_up
Suffix_down
2480 bp
AS47
BBa_J04500 with RelA
Clone 5
43 - 44
Prefix_up
Suffix_down
2480 bp
AS48
BBa_J04500 with RelA
Clone 6
43 - 44
Prefix_up
Suffix_down
2480 bp
AS49
BBa_J04500 with RelA
Clone 7
43 - 44
Prefix_up
Suffix_down
2480 bp
AS50
BBa_J04500 with RelA
Clone 8
43 - 44
Prefix_up
Suffix_down
2480 bp
AS51
BBa_J04500 with RelA
Clone 9
43 - 44
Prefix_up
Suffix_down
2480 bp
AS52
BBa_J04500 with RelA
Clone 10
43 - 44
Prefix_up
Suffix_down
2480 bp
AS53
BBa_J04500 with Mesh1
Clone 1
43 - 44
Prefix_up
Suffix_down
779 bp
AS54
BBa_J04500 with Mesh1
Clone 2
43 - 44
Prefix_up
Suffix_down
779 bp
AS55
BBa_J04500 with Mesh1
Clone 3
43 - 44
Prefix_up
Suffix_down
779 bp
AS56
BBa_J04500 with Mesh1
Clone 4
43 - 44
Prefix_up
Suffix_down
779 bp
AS57
BBa_J04500 with Mesh1
Clone 5
43 - 44
Prefix_up
Suffix_down
779 bp
AS58
BBa_J04500 with Mesh1
Clone 6
43 - 44
Prefix_up
Suffix_down
779 bp
AS59
BBa_J04500 with Mesh1
Clone 7
43 - 44
Prefix_up
Suffix_down
779 bp
AS60
BBa_J04500 with Mesh1
Clone 8
43 - 44
Prefix_up
Suffix_down
779 bp
AS61
BBa_J04500 with Mesh1
Clone 9
43 - 44
Prefix_up
Suffix_down
779 bp
AS62
BBa_J04500 with Mesh1
Clone 10
43 - 44
Prefix_up
Suffix_down
779 bp
AS63
(SH3)2
Clone 1
43 - 44
Prefix_up
Suffix_down
500 bp
AS64
(SH3)2
Clone 2
43 - 44
Prefix_up
Suffix_down
500 bp
AS65
(SH3)2
Clone 3
43 - 44
Prefix_up
Suffix_down
500 bp
AS66
(SH3)2
Clone 4
43 - 44
Prefix_up
Suffix_down
500 bp
AS61
(SH3)2
Clone 5
43 - 44
Prefix_up
Suffix_down
500 bp
The efficiency of the amplification was tested thanks to agarose gel electrophoresis. Results are presented below.
Some clones seemed to present the right construction. Clones 1 and 2 of BBa_J04500 with RelA were chosen to continue the part characterization. In the mean time, to go on the part characterization, clones 2, 5, 6 and 8 of BBa_J04500 with Mesh1 were selected. Clones 2 and 5 were going to be used as negative controls (empty plasmids) and clones 6 and 8 were going to be used as functional parts.
We realize day 3 of Lambda Red protocol on W3110 sdaBC- strain CheA- KanR.
Check the construction 200(2) (RBS B0034 + RFP E1010) by digestion.
7µL E1010 (control) (500ng)
36µL H2O
5µ NEB Buffer 2.1
1µL XbaI
1µL EcoRI
2µL 200(2) (500ng)
36µL H2O
5µ NEB Buffer 2.1
1µL XbaI
1µL EcoRI
Incubation 53' at 37°C. Then, inactivation of enzymes 25' at 80°C.
It is difficult to see if 200(2) possesses the RBS. The plasmid 200(2) is sent to sequencing and it's OK.
Clean-up of PCR product number 82 and 83 from 8-8-14 (50µL)
SOE reaction (= overlap PCR):
Mix:
PCR product 82= 3µL
PCR product 83 = 3µL
dNTP 10mM = 2µL
Q5 buffer = 8µL
Q5 polymerase = 0,5µL
H2O qsp 40µL
Program:
98°C 2’
98°C 20”
45°C 25”
72°C 1’
72°C 10’
Repeat the steps 3-4-5, 10 times.
Two new cells strain were selected to characterize RelA and Mesh1 parts. The first one is EB425 (ppGpp°) which does not produce ppGpp any more. The second one is EB544 (ppGpp++) which produces ppGpp with greater stability. These Escherichia coli strains were used to prepare two samples of competent cells. EB425 were going to be complemented by RelA whereas EB544 were going to be complemented by Mesh1.
We realize day 4 of Lambda Red protocol on  W3110 sdaBC- strain CheA- KanR. strain.
PCR2: Second PCR round to amplification of serA from the SOE product (26-08-14) with primers 5 and 6, using the iGEM PCR protocol with Q5 polymerase. The temperature of annealing is 60°C
Result: the amplification of serA is good. Clean-up of serA (PCR2) and elution with 45µL of water.
Digestion of pSB1A3 and pSB1C3 during 1h30 at 37°C. Then, inactivation of enzymes during 20’ at 80°C
Mix:
10µL of plasmid
1µL EcoRI
1µL PstI
5µL Buffer 2.1
33 µL H2O
Digestion of the insert (serA) during 1h at 37°C. Then inactivation of enzymes during 20’ at 80°C
Mix:
20µL of serA
1µL EcoRI
1µL PstI
5µL Buffer 2.1
13 µL H2O
NEB ligation:
2µL of digested plasmid
2µL of digested serA insert
2µL ligase buffer
1µL ligase (NEB)
13 µL H2O
Incubation overnight at 16°C
Several plasmids were purified thanks to Macherey Nagel MiniPrep Kit :
J04500 with RelA (RA2) Clones 1 and 2 ⇒ PiGEM_02.69 and PiGEM_02.70
J04500 Mesh1 (MS8) Clones 2 and 5 ⇒ PiGEM_02.71 and PiGEM_02.72
Empty plasmids Clones 6 and 8 ⇒ PiGEM_02.73 and PiGEM_02.74
To characterize RelA and Mesh1 parts, growth rate would be evaluated. At first, EB425 competent cells (ppGpp°) were transformed with PiGEM_02.05 (pT18 RelA, positive control), with PiGEM_02.74 (empty plasmid, negative control) and with PiGEM_02.69 and PiGEM_02.70 (pLac RelA, built part). Then, EB544 competent cells (ppGpp++) were transformed with PiGEM_02.06 (pBAD Mesh1, positive control), with PiGEM_02.74 (empty plasmid, negative control) and with PiGEM_02.71 and PiGEM_02.72 (pLac Mesh1, built part). Cells were spread over Kan glucose dish to prevent RelA and Mesh1 expression. Some colonies were going to be spread later on selective dish to observe any difference in growth rate.
We realize day 5 of Lambda Red protocol on W3110 sdaBC- CheA- KanR strain.
Transformation of 90µL of DH5α super competent cells with 10µL of ligation (pSB1C3-SerA or pSB1A2-SerA). The selective medium is LB + Ampicillin (Ap) 100µg/mL or LB + Chloramphenicol (Cm) 30µg/mL
pSB1C3-SerA-1 seemed good. It was sent to sequencing. It’s OK
Week 5
07/28/14 → 08/03/14
Week 6
08/04/14 → 08/10/14
Week 7
08/11/14 → 08/17/14
Week 8
08/18/14 → 08/24/14
Week 9
08/25/14 → 08/31/14
Contact us
Email : equipe.igem.amu@gmail.com
Follow us
iGEM_AMU iGEM.AMU
Our sponsors
Retrieved from " http://2014.igem.org/Team:Aix-Marseille/Notebook_08 "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
