<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Aix-Marseille_Project"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Aix-Marseille/Project">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Aix-Marseille/Project&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Aix-Marseille/Project&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Aix-Marseille/Project&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Aix-Marseille/Project" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Aix-Marseille/Project</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><DIV class="navbar navbar-default navbar-fixed-top" id="navbar"><DIV class="container"><DIV class="navbar-collapse collapse" id="navbar-main"><UL class="nav navbar-nav" id="nav"><LI><A href="/Team:Aix-Marseille/Team">Team</A></LI><LI><A href="https://igem.org/Team.cgi?year=2014&amp;team_name=Aix-Marseille" target="_blank">Official Team Profile</A></LI><LI class="dropdown" id="dropdown-notebook"><A href="/Team:Aix-Marseille/Project" class="dropdown-toggle">Project</A></LI><LI><A href="/Team:Aix-Marseille/Project">S.E.colization : Green light, GO!</A></LI><LI><A href="/Team:Aix-Marseille/BeyondTheBench">Beyond the Bench</A></LI></UL><LI class="dropdown" id="dropdown-notebook"><A href="/Team:Aix-Marseille/Parts" class="dropdown-toggle">Parts</A></LI><LI><A href="/Team:Aix-Marseille/Parts">Parts</A></LI><LI><A href="/Team:Aix-Marseille/Achievements">Achievements</A></LI><LI><A href="/Team:Aix-Marseille/Modeling">Modeling</A></LI><LI class="dropdown" id="dropdown-notebook"><A href="/Team:Aix-Marseille/Notebook_07" class="dropdown-toggle">Notebook</A></LI><LI><A href="/Team:Aix-Marseille/Notebook_07">July</A></LI><LI><A href="/Team:Aix-Marseille/Notebook_08">August</A></LI><LI><A href="/Team:Aix-Marseille/Notebook_09">September/October</A></LI><LI><A href="/Team:Aix-Marseille/Safety">Safety</A></LI><LI><A href="/Team:Aix-Marseille/Attributions">Attributions</A></LI><LI><A href="/Team:Aix-Marseille/Sponsors">Sponsors</A></LI></DIV></DIV></DIV><DIV class="container"><H1 class="project-title">S.E.colization : Green light, GO!</H1><DIV class="row" style="position:relative;"><DIV class="col-md-9 project"><DIV class="project-section"><H1>Overall project summary</H1><P>Our project centers around synchronizing a culture of <I>E.coli</I> cells, so that they all divide synchronously and change color from green to red and back as they go through the cell cycle. More specifically the culture stops dividing full of red cells and then at regular intervals the cells turn green and divide before returning to the red quiescent state.</P><P>This project relies on developing several new and original modules and components which we believe will be generally useful to other teams in future projects. The first component is an inducible Serine production system based on a mutant SerA protein that is insensitive to retro-inhibition by the pathway product (serine) allowing the development of bacteria secreting serine. The second component involves re-engineering the chemotaxis system to drive changes in gene expression rather than changes in flagellar rotation direction. This component is particularly innovative and potentially useful as it can form the basis of several different synthetic signaling system allowing regulation of gene expression by a wide range of different signaling molecules. The third component, or set of components, is a series of switches that change the protein expressed in response to a signal, that is rather than a simple induction of expression as observed with most inducible systems, one protein will be expressed while a second is repressed. Again it is hoped that the bricks that make this system will be applicable in multiple new projects. Each of these components is designed so that if they are introduced together they will produce an oscillator that regularly drives the switch modules between their two states. This oscillator will be coupled to color changes, the initiation of the cell cycle, and serine production (to make the feedback loop). A particularity of the oscillator is that by passing through a secreted substrate and sensor system the oscillations should be culture wide, and all the cells should fall into phase. Furthermore by using the chemotaxis system which has an intrinsically differential sensor (rather than an absolute sensor) it is hoped that the oscillations can be driven over many cell cycles.</P><P>This project is relatively fundamental but will we hope provide several generally useable and reusable modules for more applied projects where multiple signaling pathways, culture wide oscillators or switches are required. The project involves both experimental constructions, largely derived from previously published work by other authors, resulting in numerous new Biobricks, and also a strong modeling component to understand the function and constraints of the oscillator that we propose to build and the expected effects on bacterial behavior.</P><P><A href="https://static.igem.org/mediawiki/2014/9/9f/Schema_amu_projet.jpg" target="_blank">Click to enlarge image</A></P><DIV class="panel panel-default"><DIV class="panel-footer"><SMALL><I>The different cellular pathways involved in the E.coli cells synchronization</I></SMALL></DIV></DIV></DIV><DIV class="project-section"><H1>Project Details</H1><DIV class="project-subsection"><H2 class="subtitle">Genetic Background</H2><P>In this project, we use a modified <I>E.coli</I> W3110 strain.</P><UL><LI>The <I>sdaC</I> gene encoding a Serine transporter was deleted,</LI><LI>The <I>sdaB</I> gene encoding for a protein able to degrade Serine into 3-PG was deleted.</LI><LI>The <I>cheA</I> gene was deleted. CheA is the histidine protein kinase sensor of chemotactic response.</LI></UL></DIV><DIV class="project-subsection"><H2 class="subtitle">Serine, a signal molecule</H2><P>In this first section, we put a troncated <I>serA</I> (<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1349000" target="_blank">BBa_K1349000</A>) gene under the control of the <I>cusC</I> promoter which is activated by accumulation of phosphorylated CusR protein. The modification of <I>serA</I> leads to the production of a troncated SerA protein which remains functionnal but which is no longer sensitive to feedback inhibition by serine. Thus, expression of this gene leads to accumulation of high concentrations of serine.</P></DIV><DIV class="project-subsection"><H2 class="subtitle">CheA‐SH3, modified chemotaxis signalling</H2><P>The second version has been designed as follows. A change in the extracellular Serine concentration is detected by Tsr receptor. The Tsr inhibits the constitutive activity of the CheA (Histidine kinase) and reduces auto-phosphorylation. This natural circuit is modified so that CheA transfers phosphate groups not only to CheB and CheY as usual but also to CusR. Consequently, the CusR transcription factor is less phosphorylated and binds less well to its DNA recognition sequence.</P><P>When extracellular Serine concentration is stabilized, Tsr is not stimulated anymore and stops inhibiting CheA. As a consequence, CheA becomes activated again and phosphorylates CusR.</P><P>In order to modify the phosphorylation patern of CheA we have used the synthetic protein interaction modules developped by Whitaker et al. 2012 (PNAS 109: 18090-18095). The CheA (<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1349006" target="_blank">BBa_K1349006</A>) is fused to a (SH3)<SUB>4</SUB> module (<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1349005" target="_blank">BBa_K1349005</A>) and CusR to a LZa leucine zipper (<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1349007" target="_blank">BBa_K1349007</A>), these two proteins will then interact in the presence of the adaptor module an SH3-pep-LZA protein (<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1349008" target="_blank">BBa_K1349008</A>). This interaction will bring CheA and CusR closer thus permting the transfert of phosphate from CheA to CusR.</P><P>The combination of these first two sections defines a culture wide oscilatory control circuit that responds through changing extracellular serine concentration levels.</P></DIV><DIV class="project-subsection"><H2 class="subtitle">Regulation of intracellular ppGpp concentration; green light, go !</H2><P>In this third section we worked on making our oscillator to do something. We have devised two reporter modules that respond to the level of CusR phosphorylation : the first modulating cellular levels of ppGpp the secondary messanger responsible for the stringent response, and the second leading to green or red cells.</P><P>In the first reporter module the phosphorylated CusR transcription factor binds to the <I>cusC</I> promoter (<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1349003" target="_blank">BBa_K1349003</A>) and activates RelA (<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1349001" target="_blank">BBa_K1349001</A>) production. RelA is an enzyme that synthesises ppGpp which is known to block the initiation of S phase in the bacterial cell cycle, leading to the arrest of cell division. At the same time, phosphorylated CusR attaches to the CusR-Box (<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1349002" target="_blank">BBa_K1349002</A>) between a promoter and the <I>mesh1</I> gene (<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1349004" target="_blank">BBa_K1349004</A>), inhibiting synthesis of the Mesh1 enzyme. Mesh1 is an enzyme from Drosophila melanogaster that hydrolyses ppGpp. Thus in the presence of phosphorylated CusR this module ensures the presence of RelA and the absence of Mesh1 resulting in the accumulation of ppGpp. In the absence of phosphorylated CusR, the regulator is unable to bind to either the promoter or the binding site upstream of the <I>mesh1</I> gene. Consequently RelA is not synthesised, but the Mesh1 protein can be synthesised resulting in the hydrolysis of ppGpp. This reporter module constitutes a two way switch synthesising either RelA, in the presence of phosphorylated CusR, or Mesh1, in the presence of unphosphorylated CusR.</P><P>The second reported module follows the same logic but RelA is replaced with RFP and Mesh1 with GFP. Thus changing levels of CusR phosphorylation can drive red/green color changes.</P><H4>Conclusion</H4>
By combining these 3 sections we obtain an oscillator based on serine concentration sensing and two antagonist promoters which control the cell cycle at a population level. To observe this control we coupled the cell cycle changes and RFP and GFP color changes. This allows us to observe the cells switching from red (stop) to green (go) and back again.</DIV><DIV class="project-subsection"><H2 class="subtitle">References</H2><P>Pertinent articles related to our project:</P><H3 class="text-info">Top Papers<UL><LI>Gudipaty SA, Larsen AS, Rensing C, McEvoy MM. Regulation of Cu(I)/Ag(I) efflux genes in <I>E.coli</I> by the sensor kinase CusS. FEMS Microbiol Lett. 2012 May;330(1):30-7. </LI><LI>Rudd KE, Bochner BR, Cashel M, Roth JR. Mutations in the spoT gene of Salmonella typhimurium: effects on his operon expression. J Bacteriol. ;163(2):534-42, 1985.</LI><LI>Whitaker WR1, Davis SA, Arkin AP, Dueber JE. Engineering robust control of two-component system phosphotransfer using modular scaffolds. Proc Natl Acad Sci U S A. 2012 Oct 30;109(44):18090-5. </LI><LI>Ferullo DJ, Cooper DL, Moore HR, Lovett ST. Cell cycle synchronization of Escherichia coli using the stringent response, with fluorescence labeling assays for DNA content and replication. Methods. 2009 May;48(1):8-13.</LI><LI>Peters-Wendisch P1, Stolz M, Etterich H, Kennerknecht N, Sahm H, Eggeling L. Metabolic engineering of Corynebacterium glutamicum for L-serine production. Appl Environ Microbiol. 2005 Nov;71(11):7139-44.</LI></UL></H3><H3 class="text-info">Related Papers<UL><LI>Wahl A, My L, Dumoulin R, Sturgis JN, Bouveret E.Antagonistic regulation of dgkA and plsB genes of phospholipid synthesis by multiple stress responses in <I>Escherichia coli. Mol Microbiol.</I> 80(5):1260-75, 2011.</LI><LI>Collins CH. Cell-cell communication special issue. ACS Synth Biol. 2014 Apr 18;3(4):1978.</LI><LI>Hsiao V, de Los Santos EL, Whitaker WR, Dueber JE, Murray RM. Design and Implementation of a Biomolecular Concentration Tracker. ACS Synth Biol. 2014 May 15.</LI><LI>Park JH, Oh JE, Lee KH, Kim JY, Lee SY. Rational design of Escherichia coli for L-isoleucine production. ACS Synth Biol. 2012 Nov 16;1(11):532-40. </LI><LI>Tan MH1, Kozdon JB, Shen X, Shapiro L, McAdams HH. An essential transcription factor, SciP, enhances robustness of Caulobacter cell cycle regulation. Proc Natl Acad Sci U S A. 2010 Nov 2;107(44):18985-90.</LI><LI>Park H, Saha SK, Inouye M. Two-domain reconstitution of a functional protein histidine kinase. Proc Natl Acad Sci U S A. 1998 Jun 9;95(12):6728-32.</LI><LI>Porter SL, Wadhams GH, Armitage JP. Signal processing in complex chemotaxis pathways.Nat Rev Microbiol. 2011 Mar;9(3):153-65.</LI><LI>Olson EJ, Hartsough LA, Landry BP, Shroff R, Tabor JJ. Characterizing bacterial gene circuit dynamics with optically programmed gene expression signals. Nat Methods. 2014 Apr;11(4):449-55.</LI><LI>Skerker JM1, Perchuk BS, Siryaporn A, Lubin EA, Ashenberg O, Goulian M, Laub MT. Rewiring the specificity of two-component signal transduction systems. Cell. 2008 Jun 13;133(6):1043-54.</LI><LI>Skerker JM1, Prasol MS, Perchuk BS, Biondi EG, Laub MT. Two-component signal transduction pathways regulating growth and cell cycle progression in a bacterium: a system-level analysis. PLoS Biol. 2005 Oct;3(10):e334.</LI><LI>Gonzalez D, Collier J. Effects of (p)ppGpp on the progression of the cell cycle of Caulobacter crescentus. J Bacteriol. 2014 Jul;196(14):2514-25.</LI><LI>Thanbichler M. Synchronization of chromosome dynamics and cell division in bacteria. Cold Spring Harb Perspect Biol. 2010 Jan;2(1):a000331.</LI><LI>Wang X, Vallurupalli P, Vu A, Lee K, Sun S, Bai WJ, Wu C, Zhou H, Shea JE, Kay LE, Dahlquist FW. The linker between the dimerization and catalytic domains of the CheA histidine kinase propagates changes in structure and dynamics that are important for enzymatic activity. Biochemistry. 2014 Feb 11;53(5):855-61.</LI><LI>Eggeling L, Sahm H. New ubiquitous translocators: amino acid export by Corynebacterium glutamicum and Escherichia coli. Arch Microbiol. 2003 Sep;180(3):155-60.</LI><LI>Laub MT, Goulian M. Specificity in two-component signal transduction pathways. Annu Rev Genet. 2007;41:121-45.</LI><LI>Alon U, Camarena L, Surette MG, Aguera y Arcas B, Liu Y, Leibler S, Stock JB. Response regulator output in bacterial chemotaxis. EMBO J. 1998 Aug 3;17(15):4238-48.</LI><LI>Chang YC, Armitage JP, Papachristodoulou A, Wadhams GH. A single phosphatase can convert a robust step response into a graded, tunable or adaptive response. Microbiology. 2013 Jul;159(Pt 7):1276-85.</LI><LI>Ganesh I, Ravikumar S, Lee SH, Park SJ, Hong SH. Engineered fumarate sensing Escherichia coli based on novel chimeric two-component system. J Biotechnol. 2013 Dec;168(4):560-6.</LI><LI>Jonas K. To divide or not to divide: control of the bacterial cell cycle by environmental cues. Curr Opin Microbiol. 2014 Apr;18:54-60.</LI></UL></H3><H1>Protocols</H1><P>You can find below some protocols we used. Some others are available on <A href="https://2014.igem.org/Team:Aix-Marseille/Protocols" target="_blank">this page</A> of the wiki.
          </P><DIV class="project-subsection"><H2 class="subtitle">Strains used in the project</H2><UL><LI><I>Escherichia coli</I> W3110 was used for chassis construction.</LI><LI><I>Escherichia coli</I> DH5alpha and TG1 strains were used for regular cloning.</LI></UL></DIV><DIV class="project-subsection"><H2 class="subtitle">Strain storage</H2><P>For long-term conservation, 750 μL of an exponentially grown strain were mixed with 250 μL of glycerol 80%, and at -80°c in a cryotube. </P></DIV><DIV class="project-subsection"><H2 class="subtitle">Culture medium</H2><UL><LI><P>Regular bacterial growth was performed in LB medium or LB-agar plates (DIFCO) supplemented with the appropriated antibiotics if necessary: ampicillin 100 μM, kanamycin 50 μM, chloramphenicol 30 μM. When required, IPTG (Isopropyl β-D-1-thiogalactopyranoside ) was added to the final concentration of 500 μM to induce gene expression from Plac promoters.</P><P>Growth was conducted at 37°C for regular experiments, and at 30°C when carrying temperature sensitive plasmids.</P></LI><LI><P>The SMG medium used to characterize the <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1349001" target="_blank">BBa_K1349001</A> part was prepared based on the work of (Rudd <I>et al</I>.,1985).</P><P><U>SMG plate composition:</U></P><TABLE class="table table-hover table-bordered notes-table"><TBODY><TR><TD>M9 salts</TD><TD>1X</TD></TR><TR><TD>MgSO4</TD><TD>1mM</TD></TR><TR><TD>CaCl<SUB>2</SUB></TD><TD>0.1mM</TD></TR><TR><TD>VitB1</TD><TD>0.5µg/ml</TD></TR><TR><TD>Glucose</TD><TD>0.2%</TD></TR><TR><TD>Serine</TD><TD>1mM</TD></TR><TR><TD>Methionine</TD><TD>1mM</TD></TR><TR><TD>Glycine</TD><TD>1mM</TD></TR><TR><TD>Bactoagar</TD><TD>15g/L</TD></TR></TBODY></TABLE></LI></UL><P><U>References:</U></P><P>Rudd KE, Bochner BR, Cashel M, Roth JR. Mutations in the spoT gene of Salmonella typhimurium: effects on his operon expression. J Bacteriol. ;163(2):534-42, 1985.</P></DIV><DIV class="project-subsection"><H2 class="subtitle">Plasmid extraction</H2><P>Performed with the Macherey-Nagel Kit, following the manufacturer instructions.</P></DIV><DIV class="project-subsection"><H2 class="subtitle">PCR clean-up</H2><P>Performed with the Promega Kit, following the manufacturer instructions.</P></DIV><DIV class="project-subsection"><H2 class="subtitle">Competent cells</H2><P>Cells were grown in LB medium as a starter O/N. The day after, the culture was back diluted in 100 mL of LB to reach an OD600=0.05.</P><P>When the culture reached and OD600 of 0.4 to 0.6, cells were centrifuged at 5000 rpm during 10 min. The cell pellet was carefully re-suspended in 1/2V of CaCl2 50 mM at 4°C. After an incubation of 10 min on ice, the cells were pelleted by centrifugation. The pellet was re-suspended in 1/20V of CaCl2 50 mM, Glycerol 15%. Aliquots of competent cells were stored at -80°C.</P></DIV><DIV class="project-subsection"><H2 class="subtitle">Transformation</H2><P>Performed following the protocol published on the iGEM website : <A href="http://parts.igem.org/Help:Protocols/Transformation" target="_blank">here</A></P></DIV><DIV class="project-subsection"><H2 class="subtitle">Lambda Red mutant construction</H2><P>This protocol was adapted from the initial protocol published by Datsenko and Wanner (2000).</P><P><U>Reference:</U>
            Datsenko KA1, Wanner BL. One-step inactivation of chromosomal genes in Escherichia coli K-12 using PCR products. Proc Natl Acad Sci U S A. 2000 6;97(12):6640-5.</P><UL><LI><P>Day 1:</P></LI><LI>Transform W3110 competent cells with the pKOBEG plasmid (cmR). Grow at 30°C o/n</LI><LI>PCR-amplify the FRT-kan-FRT cassette from the pKD4 plasmid (AmpR) using the set of primers targeting the gene to be deleted. Check the PCR product on an agarose gel. Then digest the pKD4 matrix plasmid with 1uL DpnI enzyme, and incubate at 37°C during 2 hrs. Clean-up the PCR product with the promega kit.</LI></UL><LI><P>Day 2:</P><P>From one isolated W3110 pKOBEG clone, grow the cells at 30°C. When the culture reach an OD of 2.2, add arabinose to a final concentration of 0.05% and incubate 2 hrs at 30°C or until OD600=1. Switch the temperature to 42°C to get rid off the pKOBEG plasmid. Centrifuge the culture at 4°C and prepare a stock of electrocompetent cells.</P><P>Electroporate 10 ul of purified PCR product into 50 ul of electrocompetent cells. Grow the transformed cells on LB+kan 50uM plates, and incubate at 37°C o/n.</P></LI><LI><P>Day 3:</P><P>Restreak 25 clones on LB+Kan plates and Lb+Cm plates, incubate at 37°C O/N.</P></LI><LI><P>Day 4:</P><P>Select 8 clones resistant to Kan but sensitive to Cm. Check the correct replacement of the targeted gene by the FRT-Kan-FRT cassette with primers exterior to the gene.</P></LI><LI><P>Day 5:</P><P>Cultivate and freeze 2 independent clones.</P></LI><P>To flip the FRT-KAN-FRT cassette</P><LI><P>Day 6:</P><P>Prepare a stock of electrocompetent W3110 mutant.</P><P>Electroporate the pCP20 plasmid (CmR, ApR) encoding the flipase and spread the cells on LB+Ap+Cm. Incubate at 30°C O/N</P></LI><LI><P>Day 7:</P><P>To prepare a culture starter, inoculate 1 clone in LB at 30°c O/N</P></LI><LI><P>Day 8:</P><P>Back-dilute the culture 100x in LB without antibiotics. Grow at 30°c until you reach an OD600=0.3. Switch the temperature to 37°C until OD600= 0.8. Back-dilute the culture 100x in LB for 5 hrs. Spread 10 ul of the culture in LB plate without antibiotics to get isolated colonies.</P></LI><LI><P>Day 9:</P><P>Select 20 isolated clones and re-patch on LB, LB+Kan, LB+AP+CM.</P></LI><LI><P>Day 10:</P><P>Select 8 clones sensitive to Kan, Ap and Cm. Check by colony-PCR that the FRT-KAN-FRT cassette was truly deleted.</P></LI></DIV><DIV class="project-subsection"><H2 class="subtitle">SLIC cloning</H2><P>One-step sequence- and ligation-independent cloning (SLIC) was performed following the protocol published as supplemental data in the original manuscript of Jeong and collaborators. This protocol can be found <A href="http://aem.asm.org/content/early/2012/05/13/AEM.00844-12" target="_blank">here</A></P><P><U>References</U>
            Jeong JY1, Yim HS, Ryu JY, Lee HS, Lee JH, Seen DS, Kang SG. One-step sequence- and ligation-independent cloning as a rapid and versatile cloning method for functional genomics studies. <I>Appl Environ Microbiol.</I> 2012 ; 78(15):5440-3.</P></DIV><DIV class="project-subsection"><H2 class="subtitle">3A cloning</H2><P>Performed following the protocol published on the iGEM website : <A href="http://parts.igem.org/Assembly:3A_Assembly" target="_blank">here</A></P></DIV><DIV class="project-subsection"><H2 class="subtitle">Growth and fluorescence measurements in M9 medium</H2><P>An overnight culture of TG1 strain carrying the plasmid of interest was back diluted 100x in modified M9 medium supplemented with casaminoacids, glycerol 0.4% and 10% of LB.</P><P>Strains were cultivated at 37°C with continuous shaking in 10 mL of M9 medium, until they reached an OD600nm=0,4 to 0,6. Then, a 5 mL aliquot of each culture was treated with a sub-lethal concentration of AgNO3 (5 μM, see reference Gudipaty SA et al., 2012) while a second untreated aliquot of 5 mL was used as a control. After Ag treatment, all the cultures were incubated in the dark at 37°C with continuous shaking. Then, cell growth and GFP fluorescence were determined over time by reading the OD at 600 nm and 530 nm, respectively. To do so, 150 μL aliquots were transferred into wells of a black 96-well plate (Greiner), and the absorbance at 600 nm and fluorescence (excitation, 485 nm; emission, 530 nm) were measured with a Tecan Infinite M200 microplate reader.</P></DIV></DIV><DIV class="project-section"><H1>Results and Parts validation</H1><DIV class="project-subsection"><H2 class="subtitle">Result 1: validation of the RelA part: <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1349001" target="_blank">BBa_K1349001</A></H2><P>The relA gene encodes a ppGpp synthetase. In bacteria, ppGpp (guanosine 3'-diphosphate 5-' diphosphate) acts as a signaling molecule that regulates a variety of cellular metabolisms in response to changes in the nutritional state of the cells.</P><P>This part was designed to allow the rapid synthesis of the alarmone ppGpp, responsible for the stringent response in E. coli. In the context of our project ppGpp is accumulation used to stop initiation of cell division.</P><P><U>How we tested the <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1349001" target="_blank">BBa_K1349001</A> part:</U></P><P>Expression of the <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1349001" target="_blank">BBa_K1349001</A> part is expected to cause accumulation of ppGpp and so reduce growth rate of a wild-type cell. At the opposite, expression of this part in a MG1655∆relA mutant unable to make ppGpp is expected to complement the mutant. </P><P>As seen on the figure below, our part is functional as it is able to fully complement the MG1655∆relA mutant.
See lab-book for details on the experimental procedure.</P><DIV class="project-details"><P><I>Attribution of the figure: collaboration with Dr. E. Bouveret.</I></P></DIV></DIV><DIV class="project-subsection"><H2 class="subtitle">Result 2: validation of the CusR box: <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1349002" target="_blank">BBa_K1349002</A></H2><P>In <I>E. coli</I>, the CusR/CusS two-component system induces expression of genes involved in metal efflux under condition of elevated Cu or Ag concentrations. Phosphorylated CusR specifically bind to CusR-dependent promoters. The CusR box is the recognition sequence for the protein CusR.</P><P>We designed a Part corresponding to the CusR box to be inserted downstream of a promoter to allow transcriptional regulation of the downstream cloned gene. In common with many response regulators we expect over-expression of CusR to result in binding and so regulation of an upstream promoter. These effects are expected through bound CusR physically interacting with RNA polymerase. </P><P><U>How we tested the <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1349002" target="_blank">BBa_K1349002</A> part:</U></P><P>Insertion of the <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1349002" target="_blank">BBa_K1349002</A> part downstream of a promoter is expected to allow the binding of the CusR regulator in response to metal (Cu or Ag) availability in the medium. While different in their biological role and cellular targets, these metals share very similar chemical and ligand- binding properties. Consequently, the presence of multiple CusR-boxes would be able to bind multiple CurR regulators as so to titrate the intracellular pool of CusR.</P><P>           To test this part, we inserted the CusR box (<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1349002" target="_blank">BBa_K1349002</A>) in a pLac vector (AmpR) downstream of the Lac dependent promoter, followed by the GFP coding sequence. The Lac promoter is leaky enough to allow GFP synthesis without the addition of inducer to the culture medium, attesting that the construction allow gene expression. As our final objective a fine-tuning of our modified CurR regulatory cascade, we also tested the effect of multiple CusR-boxes insertion in our assay (1 box, 2 boxes and 3 boxes). As a control, the Plac promoter was fused to the GFP coding sequence without any CusR box. Strains were cultivated at 37°C in M9 medium until they reached an OD600nm=0,4 to 0,6. Then, an aliquot of each culture was treated with a sub-lethal concentration of AgNO3 (5 μM, see reference Gudipaty SA et al., 2012) while a second untreated aliquot was used as a control. The two series of experiments presented here were performed twice and showed similar results.
	              <UL><LI>Growth was determined overtime by following the culture OD at 600 nm on a TECAN instrument (Figure A). </LI><LI>GFP synthesis was quantified by measuring the OD at 530 nm on a TECAN instrument. Then, fluorescence was normalized by dividing the results with the OD at 600 nm (Figure B).</LI></UL></P><DIV class="project-details"><P><U>Figure A: The presence of cusR boxes on a pLac vector affects cell growth in the presence of a sub-lethal dose of AgNO3.</U></P><P><I>Attribution of the figure: Aix-Marseille iGEM team.</I></P></DIV><P>As seen on figure A, the introduction of a unique CusR box in our test vector (P-cusB1-GFP) has no effect on growth compared to the control vector (P-GFP), with or without Ag supplementation to the medium. At the opposite the presence of 2 CusR boxes (P-cusBx2-GFP) or 3 CusR boxes (P-cusBx3-GFP) in our test vector has a very strong effect on growth when AgNO3 is added to the medium. </P><BLOCKQUOTE><P style="font-size: 14px;"><B><U>Conclusion on figure A</U>:</B>
               As seen on figure A, the CusR-box part that we designed (<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1349002" target="_blank">BBa_K1349002</A>) works as expected. The insertion of multiple CusR-boxes on the pUC18 vector (2 or 3 boxes) has almost no effect on untreated cells growth, but it leads to a culture growth defect in the presence of Ag. This suggest that the BBa_K1349002 part is able to titrate the endogenous CusR regulator in the cells, preventing a correct stress response when E. coli is exposed to sub-lethal concentrations of Ag.</P></BLOCKQUOTE><DIV class="project-details"><P><U>Figure B: The insertion of 2 or 3 CusR boxes downstream of the Plac promoter allows regulation of gene expression.</U></P><P><I>Attribution of the figure: Aix-Marseille iGEM team.</I></P></DIV><P>- As seen on figure B for the untreated samples (“Untreated”), the expression of GFP increases when CusR-boxes (<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1349002" target="_blank">BBa_K1349002</A>) are inserted into the pUC18 expression plasmid. The effect is particularly strong with the presence of 2 boxes and 3 boxes. 
- Addition of Ag to the medium (“+AgNO3”) seems to have a negative effect on gene expression compared to the untreated samples. Because in this experiment, treated and untreated samples are not in the same growth phase, we are planning to reproduce this experiment with a lower dose of Ag to avoid Ag toxicity.
</P><BLOCKQUOTE><P style="font-size: 14px;"><B><U>Conclusion on figure B</U>:</B>
                Our results show that the CusR-box part that we designed (<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1349002" target="_blank">BBa_K1349002</A>) is functional, as it allows the transcriptional control of the downstream gene. Interestingly, our data suggest that the insertion of 2 or 3 CusR-boxes is optimal to allow the binding of the CusR regulator.  </P></BLOCKQUOTE><P><U>Reference</U>
            Gudipaty SA, Larsen AS, Rensing C, McEvoy MM. Regulation of Cu(I)/Ag(I) efflux genes in Escherichia coli by the sensor kinase CusS. FEMS Microbiol Lett. 2012 May;330(1):30-7.</P></DIV></DIV><DIV class="project-section"><H1>Conclusions</H1><P>This ambitious project aimed to synchronise cell division in a culture of E.coli cells. The project as a whole contained several ususual features, controlling cell division through modulating internal levels of the alarmone ppGpp, cross culture signalling using the chemotaxis system with its inherently differential response, transferring a synthetic protein-protein interaction system to the iGEM standard, and developping an on-off promoter pair to respond in opposite ways. Not surprisingly we were unable to realize the entire system in the available time. However we have been able to provide a number of components that we hope will be of use to future igemmers and characterize some of them. Including an improved characterization of a RelA brick (<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1349001" target="_blank">BBa_K1349001</A>) and a preliminary characterization of a CusR box (<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1349002" target="_blank">BBa_K1349002</A>) used as a negative regulator of gene expression.</P><P>The synthetic protein-protein interaction system was described by Whitaker et al. 2012 and has three parts: an SH3 domain than can be fused to one partner (<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1349005" target="_blank">BBa_K1349005</A>); a leucine zipper peptide that can be fused to the second partner, this is part of our response regulator brick (<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1349007" target="_blank">BBa_K1349007</A>); and an adapter molecule containing an SH3 recognition peptide and a complementary leucine zipper (<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1349008" target="_blank">BBa_K1349008</A>). Unfortunately we did not get round to testing these bricks and characterizing them.</P><P>The regulatory system that we built is based on the CusR box which binds CusR and can be used to positively regulate gene expression as in the case of the natural CusP promoter (<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1349003" target="_blank">BBa_K1349003</A>), or can be used to negatively regulate an upstream promoter ans the CusR box (<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1349002" target="_blank">BBa_K1349002</A>). We were able to characterize the CusR box and show that AgNO3 in the media, which is known to lead to CusR phosphorylation and hence DNA binding, reduced GFP synthesis in an appropriate context, and also multiple CusR boxes were able to increase silver toxicity, presumably by titration CusR. This demonstration of activity makes this brick of potential interest in several different contexts.</P><P>The ppGpp control was based on two bricks, RelA (<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1349001" target="_blank">BBa_K1349001</A>) which synthesises ppGpp and Mesh1 (<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1349004" target="_blank">BBa_K1349004</A>) which degrades ppGpp. We were able to show that the Rel A brick (<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1349001" target="_blank">BBa_K1349001</A>) is able to complement a relA defficient strain and thus confirm that the brick is active.</P><P>Our RelA brick (<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1349001" target="_blank">BBa_K1349001</A>) represents a significant improvement on the previously available RelA brick made by the 2011 Trondheim team (<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K639001" target="_blank">BBa_K639001</A>). In the first place our brick has no amino acid mutations and no upstream unwanted insert. Secondly our brick lasks the PstI site and thus is compatible with the assembly standards RFC 10, 12, 21, 23, 25 and 1000. We have shown that out brick is active when expressed in E. coli and able to complement a relA mutant.</P><P>Beyond these successes we also had a great summer learning synthetic biology and are now ready for Boston.</P></DIV></DIV><DIV class="col-md-3" id="sidebar-project-div"><DIV id="sidebar-project"><UL class="nav"><LI class="active"><A data-scroll="" href="#overall">Overall project summary</A></LI><LI><A data-scroll="" href="#details">Project Details</A></LI><LI><A data-scroll="" href="#Genetic_Background">Genetic Background</A></LI><LI><A data-scroll="" href="#serine_part">Serine, a signal molecule</A></LI><LI><A data-scroll="" href="#CheA-SH3_part">CheA‐SH3, modified chemotaxis signalling</A></LI><LI><A data-scroll="" href="#ppGpp_part">Regulation of intracellular ppGpp concentration</A></LI><LI><A data-scroll="" href="#refs">References</A></LI></UL><LI><A data-scroll="" href="#protocols">Protocols</A></LI><LI><A data-scroll="" href="#results">Results and Parts validation</A></LI><LI><A data-scroll="" href="#conclusions">Conclusions</A></LI></DIV></DIV></DIV></DIV><DIV class="footer"><DIV class="container"><DIV class="row"><DIV class="col-md-4"><H4 class="text-info">Contact us</H4><DIV class="text-warning"><P><B>Email : </B><A href="mailto:equipe.igem.amu@gmail.com">equipe.igem.amu@gmail.com</A></P></DIV></DIV><DIV class="col-md-4"><H4 class="text-info">Follow us</H4><DIV class="text-warning"><P><A href="https://twitter.com/iGEM_AMU"> iGEM_AMU</A><A href="https://www.facebook.com/iGEM.AMU">iGEM.AMU</A></P></DIV></DIV><DIV class="col-md-4"><H4 class="text-info">Our sponsors</H4></DIV></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Aix-Marseille/Project">http://2014.igem.org/Team:Aix-Marseille/Project</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Aix-Marseille/Project" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Aix-Marseille/Project" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Aix-Marseille/Project&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Aix-Marseille/Project&amp;oldid=331406" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></BODY></HTML>