<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Aix-Marseille_Notebook_08"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Aix-Marseille/Notebook_08">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Aix-Marseille/Notebook_08&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Aix-Marseille/Notebook_08&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Aix-Marseille/Notebook_08&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Aix-Marseille/Notebook_08" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Aix-Marseille/Notebook 08</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><DIV class="navbar navbar-default navbar-fixed-top" id="navbar"><DIV class="container"><DIV class="navbar-collapse collapse" id="navbar-main"><UL class="nav navbar-nav" id="nav"><LI><A href="/Team:Aix-Marseille/Team">Team</A></LI><LI><A href="https://igem.org/Team.cgi?year=2014&amp;team_name=Aix-Marseille" target="_blank">Official Team Profile</A></LI><LI class="dropdown" id="dropdown-notebook"><A href="/Team:Aix-Marseille/Project" class="dropdown-toggle">Project</A></LI><LI><A href="/Team:Aix-Marseille/Project">S.E.colization : Green light, GO!</A></LI><LI><A href="/Team:Aix-Marseille/BeyondTheBench">Beyond the Bench</A></LI></UL><LI class="dropdown" id="dropdown-notebook"><A href="/Team:Aix-Marseille/Parts" class="dropdown-toggle">Parts</A></LI><LI><A href="/Team:Aix-Marseille/Parts">Parts</A></LI><LI><A href="/Team:Aix-Marseille/Achievements">Achievements</A></LI><LI><A href="/Team:Aix-Marseille/Modeling">Modeling</A></LI><LI class="dropdown" id="dropdown-notebook"><A href="/Team:Aix-Marseille/Notebook_07" class="dropdown-toggle">Notebook</A></LI><LI><A href="/Team:Aix-Marseille/Notebook_07">July</A></LI><LI><A href="/Team:Aix-Marseille/Notebook_08">August</A></LI><LI><A href="/Team:Aix-Marseille/Notebook_09">September/October</A></LI><LI><A href="/Team:Aix-Marseille/Safety">Safety</A></LI><LI><A href="/Team:Aix-Marseille/Attributions">Attributions</A></LI><LI><A href="/Team:Aix-Marseille/Sponsors">Sponsors</A></LI></DIV></DIV></DIV><DIV class="container"><H1 class="project-title">Notebook: August</H1><DIV class="row" style="position:relative;" id="notebook-row"><DIV class="col-md-9" id="notes"><DIV class="panel panel-info notes-panel"><DIV class="panel-heading"><H3 class="panel-title">Week 5 : 07/28/2014 ➡ 08/03/2014 </H3></DIV><DIV class="panel-body"><UL class="nav nav-tabs" role="tablist"><LI class="active"><A href="#0728" role="tab" data-toggle="tab" class="date-notebook">07/28/14</A></LI><LI><A href="#0729" role="tab" data-toggle="tab" class="date-notebook">07/29/14</A></LI><LI><A href="#0730" role="tab" data-toggle="tab" class="date-notebook">07/30/14</A></LI><LI><A href="#0731" role="tab" data-toggle="tab" class="date-notebook">07/31/14</A></LI><LI><A href="#0801" role="tab" data-toggle="tab" class="date-notebook">08/01/14</A></LI><LI><A href="#0803" role="tab" data-toggle="tab" class="date-notebook">08/03/14</A></LI></UL><DIV class="tab-content"><DIV class="tab-pane fade in active" id="0728"><UL class="list-notebook"><LI><P>Several DNA fragments were amplified thanks to iGEM PCR protocol (Q5 polymerase).</P><TABLE class="table table-hover table-bordered notes-table"><THEAD><TR><TH>#</TH><TH>DNA Template</TH><TH>Primers Numbers</TH><TH>Primers Names</TH><TH>Weight of amplified DNA</TH></TR></THEAD><TBODY><TR><TD>2</TD><TD>Genomic DNA of W3110</TD><TD>6 - 7</TD><TD>serA_mut_ A849G_upserA_tronq_SufRFC23_down</TD><TD>171 bp</TD></TR><TR><TD>3</TD><TD>Genomic DNA of W3110</TD><TD>5 - 8</TD><TD>serA_PrefRFC10RBS_upsera_mut_A849_down</TD><TD>861 bp</TD></TR><TR><TD>6</TD><TD>Genomic DNA of W3110</TD><TD>10 - 13</TD><TD>cheA_SufRFC23_downcheA_mut_T1290C_up</TD><TD>726 bp</TD></TR><TR><TD>8</TD><TD>PiGEM_02.05</TD><TD>16 - 18</TD><TD>relA_mut_C1366_downrelA_sans_stop_SufRFC23_down</TD><TD>882 bp</TD></TR><TR><TD>9</TD><TD>PiGEM_02.05</TD><TD>16 - 19</TD><TD>relA_mut_C1366_downrelA_2_stop_SufRFC23_down</TD><TD>885 bp</TD></TR><TR><TD>10</TD><TD>PiGEM_02.06</TD><TD>20 - 24</TD><TD>Mesh_1_mut_A255T_upMesh_1_RFC10-RBS_up</TD><TD>267 bp</TD></TR><TR><TD>11</TD><TD>PiGEM_02.06</TD><TD>21 - 22</TD><TD>Mesh_1_mut_A255T_downMesh_1_mut_A343G_up</TD><TD>120 bp</TD></TR><TR><TD>12</TD><TD>PiGEM_02.06</TD><TD>23 - 25</TD><TD>Mesh_1_mut_A343G_downMesh1_sans_stop_SufRFC23_down</TD><TD>192 bp</TD></TR><TR><TD>13</TD><TD>PiGEM_02.06</TD><TD>23 - 26</TD><TD>Mesh_1_mut_A343G_downMesh_1_2_stop_SufRFC23_down</TD><TD>192 bp</TD></TR></TBODY></TABLE><P>The efficiency of the amplification was tested thanks to agarose gel electrophoresis. Results are presented below.</P><P>All DNA fragments seemed to be correctly amplified.</P></LI><LI>PCR products were purified thanks to Macherey Nagel PCR Clean up Kit.</LI><LI><P>The DNA concentration of PCR products was tested thanks to the Nanodrop. Results are presented below.</P><TABLE class="table table-hover table-bordered notes-table"><THEAD><TR><TH>#</TH><TH>Concentration of DNA (ng/µL)</TH></TR></THEAD><TBODY><TR><TD>2</TD><TD>52.28</TD></TR><TR><TD>3</TD><TD>44.86</TD></TR><TR><TD>6</TD><TD>49.93</TD></TR><TR><TD>8</TD><TD>73.38</TD></TR><TR><TD>9</TD><TD>76.77</TD></TR><TR><TD>10</TD><TD>57.00</TD></TR><TR><TD>11</TD><TD>39.87</TD></TR><TR><TD>12</TD><TD>50.88</TD></TR><TR><TD>13</TD><TD>62.42</TD></TR></TBODY></TABLE></LI><LI>PCR products number 6, 8 and 9 were stocked at -20°C whereas PCR products number 2, 3, 10, 11, 12 and 13 were used to realize SLIC protocol. This protocol permits to assemble all parts of SerA and Mesh1 genes with their directed mutation. The destination plasmid DNA used is the backbone pSB1A3 linearized plasmid. DH5α supercompetent bacteria were transformed with SLIC products and spread over Amp dish.</LI><LI>
                    Several plasmids were purified thanks to Macherey Nagel MiniPrep Kit :
                    </LI><LI>Brick 200 (3)</LI><LI>Brick 200 (4)</LI><LI>Brick 200 (5)</LI><LI>Brick 200 (6)</LI></UL></DIV><DIV class="tab-pane fade in" id="0729"><UL class="list-notebook"><LI>
                    The efficiency of the purification was tested thanks to agarose gel electrophoresis. Results are presented following.
                    <DIV class="media notes-media"><DIV class="media-body">
                        Brick 200 (6) well did not present any DNA fragment. Other wells contain one or two fragments of DNA, so Brick 200 (3), 200 (4) and 200 (5) were successfully purified.
                      </DIV></DIV></LI><LI><P>Several control PCR were realized to verify if SLIC worked. Taq polymerase was used. Three clones of each SLIC were tested. They were also pricked up on Amp dish.</P><TABLE class="table table-hover table-bordered notes-table"><THEAD><TR><TH>#</TH><TH>DNA Template</TH><TH>Primers Numbers</TH><TH>Primers Names</TH><TH>Weight of amplified DNA</TH></TR></THEAD><TBODY><TR><TD>S1</TD><TD>Clone 1 of SLIC SerA</TD><TD>5 - 6</TD><TD>serA_PrefRFC10RBS_upserA_tronq_SufRFC23_down</TD><TD>1032 bp</TD></TR><TR><TD>S2</TD><TD>Clone 2 of SLIC SerA</TD><TD>5 - 6</TD><TD>serA_PrefRFC10RBS_upserA_tronq_SufRFC23_down</TD><TD>1032 bp</TD></TR><TR><TD>S3</TD><TD>Clone 3 of SLIC SerA</TD><TD>5 - 6</TD><TD>serA_PrefRFC10RBS_upserA_tronq_SufRFC23_down</TD><TD>1032 bp</TD></TR><TR><TD>S4</TD><TD>Clone 4 of SLIC Mesh1 without STOP</TD><TD>24 - 25</TD><TD>Mesh_1_RFC10-RBS_uprelA_sans_stop_SufRFC23_down</TD><TD>579 bp</TD></TR><TR><TD>S5</TD><TD>Clone 5 of SLIC Mesh1 without STOP</TD><TD>24 - 25</TD><TD>Mesh_1_RFC10-RBS_uprelA_sans_stop_SufRFC23_down</TD><TD>579 bp</TD></TR><TR><TD>S6</TD><TD>Clone 6 of SLIC Mesh1 without STOP</TD><TD>24 - 25</TD><TD>Mesh_1_RFC10-RBS_uprelA_sans_stop_SufRFC23_down</TD><TD>579 bp</TD></TR><TR><TD>S7</TD><TD>Clone 7 of SLIC Mesh1 with STOP</TD><TD>24 - 26</TD><TD>Mesh_1_RFC10-RBS_upMesh_1_2_stop_SufRFC23_down</TD><TD>582 bp</TD></TR><TR><TD>S8</TD><TD>Clone 8 of SLIC Mesh1 with STOP</TD><TD>24 - 26</TD><TD>Mesh_1_RFC10-RBS_upMesh_1_2_stop_SufRFC23_down</TD><TD>582 bp</TD></TR><TR><TD>S9</TD><TD>Clone 9 of SLIC Mesh1 with STOP</TD><TD>24 - 26</TD><TD>Mesh_1_RFC10-RBS_upMesh_1_2_stop_SufRFC23_down</TD><TD>582 bp</TD></TR></TBODY></TABLE></LI><LI><P>Several DNA fragments were amplified thanks to iGEM PCR protocol (Q5 polymerase).</P><TABLE class="table table-hover table-bordered notes-table"><THEAD><TR><TH>#</TH><TH>DNA Template</TH><TH>Primers Numbers</TH><TH>Primers Names</TH><TH>Weight of amplified DNA</TH></TR></THEAD><TBODY><TR><TD>1</TD><TD>pKD4 plasmid</TD><TD>1 - 2</TD><TD>sdaCsdaB_mut_pKD4_upsdaCsdaB_ mut_pKD4_down</TD><TD>1032 bp</TD></TR><TR><TD>4</TD><TD>pKD4 plasmid</TD><TD>3 - 4</TD><TD>cheA_mut_pKD4_upcheA_mut_pKD4_down</TD><TD>1032 bp</TD></TR><TR><TD>5</TD><TD>Genomic DNA of W3110</TD><TD>9 - 14</TD><TD>cheA_PrefRFC23_upcheA_mut_T1290C_down</TD><TD>1032 bp</TD></TR><TR><TD>7</TD><TD>Genomic DNA of W3110</TD><TD>15 - 17</TD><TD>relA_mut_C1366G_uprelA_RFC10-RBS_up</TD><TD>579 bp</TD></TR></TBODY></TABLE><DIV class="media notes-media"><DIV class="media-body">
                        PCR product 1 and 4 were correctly amplified. Thus, they were purified thanks to Macherey Nagel PCR Clean up Kit. PCR product 5 and 7 seemed to be compound with too many different DNA fragments. That's why, a new amplification by PCR was made overnight (iGEM PCR protocol Q5 polymerase).
                      </DIV></DIV></LI></UL></DIV><DIV class="tab-pane fade in" id="0730"><UL class="list-notebook"><LI>
                    Realization of &quot;Day 2&quot; of lambda red pKOBEG protocol. W3110 with active recombinase was electroporated by PCR product 1 (sdaCsdaB) and 4 (cheA). Transformed bacteria were spread over a Kan dish.
                  </LI><LI><P>The efficiency of the purification of PCR product number S1 to S9 was tested thanks to agarose gel electrophoresis. At the same time, the efficiency of the amplification of the beginning sequences of CheA and RelA (PCR product 5 and 7) was tested thanks to agarose gel electrophoresis. Results are presented below.</P><DIV class="media notes-media"><DIV class="media-body">
                        PCR products 5 and 7 were good for SLIC. Clone 1 and 2 (serA), clone 4 and 6 (Mesh1 without STOP) and clone 7 and 8 (Mesh1 with STOP) were put in culture in LB with Amp.
                      </DIV></DIV></LI><LI>
                    The DNA concentration of PCR products 5 and 7 was tested thanks to the Nanodrop. Results are presented below.
                    
                    <TABLE class="table table-hover table-bordered notes-table"><THEAD><TR><TH>#</TH><TH>Concentration of DNA (ng/µL)</TH></TR></THEAD><TBODY><TR><TD>5</TD><TD>29.78</TD></TR><TR><TD>7</TD><TD>85.51</TD></TR></TBODY></TABLE></LI><LI>
                    PCR products number 5, 6, 7, 8 and 9 were used to realize SLIC protocol. This protocol permits to assemble all parts of CheA (5 and 6) and RelA (7 and 8 without STOP, 7 and 9 with STOP) genes with their directed mutation. The destination plasmid DNA used is the backbone pSB1A3 linearized plasmid. DH5α supercompetent bacteria were transformed with SLIC products and spread over Amp dish.
                  </LI></UL></DIV><DIV class="tab-pane fade in" id="0731"><UL class="list-notebook"><LI>
                    SLIC assembly of RelA was a success but SLIC assembly of CheA failed. That's why, a new SLIC assembly of CheA with PCR products 5 and 6 was made. The destination plasmid DNA used is the backbone pSB1A3 linearized plasmid. DH5α supercompetent bacteria were transformed with SLIC products and spread over Amp dish.
                  </LI><LI>
                    Realization of &quot;Day 2&quot; of lambda red pKOBEG protocol a second time. PCR product 1 and 4 were purified a second time. The efficiency of the purification was tested thanks to agarose gel electrophoresis. Results are presented below. W3110 with active recombinase was electroporated by PCR product 1 (sdaCsdaB) and 4 (cheA). Transformed bacteria were spread over a Kan dish.
                  </LI><LI><P>Several control PCR were realized to verify whether RelA SLIC worked. Taq polymerase was used. Three clones of each SLIC were tested. They were also pricked up on Amp dish.</P><TABLE class="table table-hover table-bordered notes-table"><THEAD><TR><TH>#</TH><TH>DNA Template</TH><TH>Primers Numbers</TH><TH>Primers Names</TH><TH>Weight of amplified DNA</TH></TR></THEAD><TBODY><TR><TD>S22</TD><TD>Clone 22 of SLIC RelA without Stop</TD><TD>17 - 18</TD><TD>relA_RFC10-RBS_uprelA_sans_stop_SufRFC23_down</TD><TD>2259 bp</TD></TR><TR><TD>S23</TD><TD>Clone 22 of SLIC RelA without Stop</TD><TD>17 - 18</TD><TD>relA_RFC10-RBS_uprelA_sans_stop_SufRFC23_down</TD><TD>2259 bp</TD></TR><TR><TD>S24</TD><TD>Clone 22 of SLIC RelA without Stop</TD><TD>17 - 18</TD><TD>relA_RFC10-RBS_uprelA_sans_stop_SufRFC23_down</TD><TD>2259 bp</TD></TR><TR><TD>S25</TD><TD>Clone 25 of SLIC RelA with STOP</TD><TD>17 - 19</TD><TD>relA_RFC10-RBS_uprelA_2_stop_SufRFC23_down</TD><TD>2262 bp</TD></TR><TR><TD>S26</TD><TD>Clone 26 of SLIC RelA with STOP</TD><TD>17 - 19</TD><TD>relA_RFC10-RBS_uprelA_2_stop_SufRFC23_down</TD><TD>2262 bp</TD></TR><TR><TD>S27</TD><TD>Clone 27 of SLIC RelA with STOP</TD><TD>17 - 19</TD><TD>relA_RFC10-RBS_uprelA_2_stop_SufRFC23_down</TD><TD>2262 bp</TD></TR></TBODY></TABLE><P>The efficiency of the amplification was tested thanks to agarose gel electrophoresis. Results are presented below.</P><DIV class="media notes-media"><DIV class="media-body">
                        PCR products 1 and 4 were correctly amplified. Thus, they were purified thanks to Macherey Nagel PCR Clean up Kit. Transplanting of S22 to S27 failed, those clones were gave up.
                      </DIV></DIV></LI><LI>
                    Several plasmids were purified thanks to Macherey Nagel MiniPrep Kit :
                    </LI><LI>Clone 1 and 2 (SerA) ⇒ PiGEM_02.10 and PiGEM_02.11</LI><LI>Clone 4 and 6 (Mesh1 without STOP) ⇒ PiGEM_02.12 and PiGEM_02.13</LI><LI>Clone 7 and 8 (Mesh1 with STOP) ⇒ PiGEM_02.14 and PiGEM_02.15</LI></UL></DIV><DIV class="tab-pane fade in" id="0801"><UL class="list-notebook"><LI>
                    SLIC assembly of CheA failed. That's why, a new SLIC assembly of CheA with PCR products 5 and 6 was made. The destination plasmid DNA used is the backbone pSB1A3 linearized plasmid. DH5α supercompetent bacteria were transformed with SLIC products and spread over Amp dish.
                  </LI><LI><P>Several control PCR were realized to verify whether RelA SLIC worked. Taq polymerase was used. Three clones of each SLIC were tested. They were also pricked up on Amp dish.</P><TABLE class="table table-hover table-bordered notes-table"><THEAD><TR><TH>#</TH><TH>DNA Template</TH><TH>Primers Numbers</TH><TH>Primers Names</TH><TH>Weight of amplified DNA</TH></TR></THEAD><TBODY><TR><TD>S28</TD><TD>Clone 28 of SLIC RelA without Stop</TD><TD>17 - 18</TD><TD>relA_RFC10-RBS_uprelA_sans_stop_SufRFC23_down</TD><TD>2259 bp</TD></TR><TR><TD>S29</TD><TD>Clone 28 of SLIC RelA without Stop</TD><TD>17 - 18</TD><TD>relA_RFC10-RBS_uprelA_sans_stop_SufRFC23_down</TD><TD>2259 bp</TD></TR><TR><TD>S30</TD><TD>Clone 28 of SLIC RelA without Stop</TD><TD>17 - 18</TD><TD>relA_RFC10-RBS_uprelA_sans_stop_SufRFC23_down</TD><TD>2259 bp</TD></TR><TR><TD>S31</TD><TD>Clone 31 of SLIC RelA with STOP</TD><TD>17 - 19</TD><TD>relA_RFC10-RBS_uprelA_2_stop_SufRFC23_down</TD><TD>2262 bp</TD></TR><TR><TD>S32</TD><TD>Clone 31 of SLIC RelA with STOP</TD><TD>17 - 19</TD><TD>relA_RFC10-RBS_uprelA_2_stop_SufRFC23_down</TD><TD>2262 bp</TD></TR><TR><TD>S33</TD><TD>Clone 31 of SLIC RelA with STOP</TD><TD>17 - 19</TD><TD>relA_RFC10-RBS_uprelA_2_stop_SufRFC23_down</TD><TD>2262 bp</TD></TR></TBODY></TABLE><P>The efficiency of the amplification was tested thanks to agarose gel electrophoresis. Results are presented below.</P><DIV class="media notes-media"><DIV class="media-body">
                        PCR product S28, S29, S31, S32 and S33 were good for SLIC. Clones 28 and 29 (RelA without STOP) and clones 31, 32 and 33 (RelA with STOP) were put in culture in LB with Amp.
                      </DIV></DIV></LI><LI>
                    Realization of &quot;Day 2&quot; of lambda red pKOBEG protocol for the third time. PCR products 1 and 4 were purified a third time. pKOBEG electrocompetent were prepared a third time. W3110 with active recombinase was electroporated with PCR products 1 (sdaCsdaB) and 4 (cheA). Transformed bacteria were spread over a Kan dish.
                  </LI><LI>
                    The efficiency of the purification of PiGEM_02.10, PiGEM_02.11, PiGEM_02.12, PiGEM_02.13, PiGEM_02.14 and PiGEM_02.15 was tested thanks to agarose gel electrophoresis. Results are presented below.
                    <DIV class="media notes-media"><DIV class="media-body">
                        These plasmids seemed to be in low quantity. The DNA concentration of plasmids was tested thanks to the Nanodrop. Results are presented following
                      </DIV></DIV><TABLE class="table table-hover table-bordered notes-table"><THEAD><TR><TH>#</TH><TH>Concentration of DNA (ng/µL)</TH></TR></THEAD><TBODY><TR><TD>PiGEM_02.10</TD><TD>30.6</TD></TR><TR><TD>PiGEM_02.11</TD><TD>37.8</TD></TR><TR><TD>PiGEM_02.12</TD><TD>12.2</TD></TR><TR><TD>PiGEM_02.13</TD><TD>15.11</TD></TR><TR><TD>PiGEM_02.14</TD><TD>14.11</TD></TR><TR><TD>PiGEM_02.15</TD><TD>19.22</TD></TR></TBODY></TABLE></LI></UL></DIV><DIV class="tab-pane fade in" id="0803"><UL class="list-notebook"><LI>
                    The W3110 with active recombinase was electroporated with PCR products 1 (sdaCsdaB) and 4 (cheA). Transformed bacteria were spread over a Kan dish.
                  </LI><LI>
                    Bacteria transformed with PCR products 1 (sdaCsdaB) and 4 (cheA) put in culture on Kan dish on the 08/01/14 showed colonies.                
                  </LI><LI>
                    5 different colonies of SdaCsdaB and CheA mutants were spread over Kan dish and Cm30 dish in order to verify whether the pKOBEG plasmid was eliminated (by the previously heat shock) or not as we wanted to, and whether the transformed bacteria contained the kanamycin resistance. The bacteria we want must show colonies on Kan dish and no colonies on Cm 30 dish at the same time.
                  </LI></UL></DIV></DIV></DIV><DIV class="panel-footer">
            Protocols used : ...
          </DIV></DIV><DIV class="panel panel-info notes-panel"><DIV class="panel-heading"><H3 class="panel-title">Week 6 : 08/04/2014 ➡ 08/10/2014 </H3></DIV><DIV class="panel-body"><UL class="nav nav-tabs" role="tablist"><LI class="active"><A href="#0804" role="tab" data-toggle="tab" class="date-notebook">08/04/14</A></LI><LI><A href="#0805" role="tab" data-toggle="tab" class="date-notebook">08/05/14</A></LI><LI><A href="#0806" role="tab" data-toggle="tab" class="date-notebook">08/06/14</A></LI><LI><A href="#0807" role="tab" data-toggle="tab" class="date-notebook">08/07/14</A></LI><LI><A href="#0808" role="tab" data-toggle="tab" class="date-notebook">08/08/14</A></LI><LI><A href="#0809" role="tab" data-toggle="tab" class="date-notebook">08/09/14</A></LI></UL><DIV class="tab-content"><DIV class="tab-pane fade in active" id="0804"><UL class="list-notebook"><LI>
                    Several plasmids were purified thanks to Macherey Nagel MiniPrep Kit :
                    </LI><LI>SLIC RelA without STOP PCR product 7/8 Clone 28 and 29 ⇒ PiGEM_02.16 and PiGEM_02.17</LI><LI>SLIC RelA with STOP PCR product 7/8 Clone 31, 32 and 33 ⇒ PiGEM_02.18, PiGEM_02.19 and PiGEM_02.20</LI></UL><DIV class="media notes-media"><DIV class="media-body">
                        All plasmids seemed to be correctly purified. No DNA contamination was noticeable.
                      </DIV></DIV><LI><P>Several DNA fragments were amplified thanks to iGEM PCR protocol (Q5 polymerase).</P><TABLE class="table table-hover table-bordered notes-table"><THEAD><TR><TH>#</TH><TH>DNA Template</TH><TH>Primers Numbers</TH><TH>Primers Names</TH><TH>Weight of amplified DNA</TH></TR></THEAD><TBODY><TR><TD>14</TD><TD>Genomic DNA of W3110</TD><TD>33 - 34</TD><TD>CusR_iProm-upCusR_iProm-down</TD><TD>117 bp</TD></TR><TR><TD>15</TD><TD>PiGEM_01.04</TD><TD>37 - 38</TD><TD>BBa_E1010-upBBa_E1010_NoStop-down</TD><TD>708 bp</TD></TR><TR><TD>16</TD><TD>PiGEM_02.03</TD><TD>39 - 40</TD><TD>BBa_E0040-upBBa_E0040_NoStop-down</TD><TD>714 bp</TD></TR><TR><TD>17</TD><TD>PiGEM_01.09</TD><TD>45 - 46</TD><TD>(SH3)4-up(SH3)4-down</TD><TD>932 bp</TD></TR><TR><TD>18</TD><TD>PiGEM_02.08</TD><TD>47 - 48</TD><TD>cusR_L_LZA-upcusR_L_LZA_down</TD><TD>857 bp</TD></TR><TR><TD>19</TD><TD>PiGEM_02.07</TD><TD>49 - 50</TD><TD>SH3pep_L_LZA-upSH3pep_L_LZA-down</TD><TD>192 bp</TD></TR></TBODY></TABLE><P>The efficiency of the amplification was tested thanks to agarose gel electrophoresis. Results are presented below.</P><P>Fragments 19, 15, 16 and 18 seemed to be correctly amplified. They were going to be cloned in iGEM plasmids. Unfortunately, fragments 14 and 17 were not amplified. Another PCR had to be made.</P></LI><LI><P>Several control PCR were realized to verify whether SLIC, bricks 200 and bricks 202 worked. Taq polymerase was used.</P><TABLE class="table table-hover table-bordered notes-table"><THEAD><TR><TH>#</TH><TH>DNA Template</TH><TH>Primers Numbers</TH><TH>Primers Names</TH><TH>Weight of amplified DNA</TH></TR></THEAD><TBODY><TR><TD>20</TD><TD>Clone 1 of SLIC SerA</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>1032 bp</TD></TR><TR><TD>21</TD><TD>Clone 2 of SLIC SerA</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>1032 bp</TD></TR><TR><TD>22</TD><TD>Clone 28 of SLIC SerA</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>2260 bp</TD></TR><TR><TD>23</TD><TD>Clone 29 of SLIC SerA</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>2260 bp</TD></TR><TR><TD>24</TD><TD>Clone 31 of SLIC SerA</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>2260 bp</TD></TR><TR><TD>25</TD><TD>Clone 32 of SLIC SerA</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>2260 bp</TD></TR><TR><TD>26</TD><TD>Clone 33 of SLIC SerA</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>2260 bp</TD></TR><TR><TD>27</TD><TD>Clone 34 of SLIC CheA</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>2432 bp</TD></TR><TR><TD>28</TD><TD>Clone 35 of SLIC CheA</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>2432 bp</TD></TR><TR><TD>29</TD><TD>Clone 36 of SLIC CheA</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>2432 bp</TD></TR><TR><TD>30</TD><TD>Clone 37 of SLIC CheA</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>2432 bp</TD></TR><TR><TD>31</TD><TD>Clone 38 of SLIC CheA</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>2432 bp</TD></TR><TR><TD>32</TD><TD>Clone 39 of SLIC CheA</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>2432 bp</TD></TR><TR><TD>33</TD><TD>Brick 202 (1)</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>730 bp</TD></TR><TR><TD>34</TD><TD>Brick 202 (2)</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>769 bp</TD></TR><TR><TD>35</TD><TD>Brick 200 (3)</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>769 bp</TD></TR><TR><TD>36</TD><TD>Brick 200 (4)</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>769 bp</TD></TR><TR><TD>37</TD><TD>Brick 200 (5)</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>769 bp</TD></TR></TBODY></TABLE><P>The efficiency of the amplification was tested thanks to agarose gel electrophoresis. Results are presented below.</P><P>No fragment seemed to be correctly amplified. SLIC were going to be done again with another protocol. Bricks 200 and 202 were also going to be remade.</P></LI><LI><P>PiGEM_01.03 (pSB1C3, BBa_B0033) was digested with XbaI and SpeI to be used for stability label cloning.</P><P>The efficiency of the digestion was tested thanks to agarose gel electrophoresis. Results are presented below.</P><DIV class="media notes-media"><DIV class="media-body">
                        PSB1C3 of PiGEM_01.03  seemed to be correctly digested.
                      </DIV></DIV></LI></DIV><DIV class="tab-pane fade in" id="0805"><UL class="list-notebook"><LI><P>PiGEM_01.03 (pSB1C3, BBa_B0033) was digested with EcoRI and SpeI to be used for PCR products 15 and 16 cloning. PCR products 15 and 16 (respectively RFP without STOP codon and GFP without STOP codon) were digested by EcoRI and SpeI. Then, PCR products 15 and 16 were leagued in pSB1C3 digested. DH5 alpha supercompetent were transformed  with these ligation products and spread on Cm dish.</P></LI><LI><P>PiGEM_02.27 (pSB1A2, BBa_J04450) was digested with EcoRI and PstI to be used for PCR products 18 and 19 cloning. PCR products 18 and 19 (respectively cusR-L-LZa and SH3pep-L-LZa) were digested by EcoRI and PstI. Then, PCR products 18 and 19 were leagued in pSB1A2 digested. DH5 alpha supercompetent were transformed  with these ligation products and spread on Amp dish.</P></LI><LI><P>Because PCR 17 did not work, (SH3)4  was amplified again thanks to iGEM PCR protocol (Q5 polymerase).</P><TABLE class="table table-hover table-bordered notes-table"><THEAD><TR><TH>#</TH><TH>DNA Template</TH><TH>Primers Numbers</TH><TH>Primers Names</TH><TH>Weight of amplified DNA</TH></TR></THEAD><TBODY><TR><TD>38</TD><TD>PiGEM_02.07</TD><TD>49 - 50</TD><TD>SH3pep_L_LZA-upSH3pep_L_LZA-down</TD><TD>192 bp</TD></TR></TBODY></TABLE><P>The efficiency of the amplification was tested thanks to agarose gel electrophoresis. Results are presented below.</P><DIV class="media notes-media"><DIV class="media-body">
                        (SH3)<SUB>4</SUB> amplification did not work again. A specific treatment of this cloning was needed.
                      </DIV></DIV></LI><LI><P>PiGEM_01.03 (pSB1C3, BBa_B0033) had be digested with XbaI and SpeI the day before to be used for stability label cloning. A gel extract was made this day to collect only the backbone. The backbone obtained was digested by PstI because XbaI and SpeI are compatible restriction sites. Stability labels were assembled by annealing.</P><TABLE class="table table-hover table-bordered notes-table"><THEAD><TR><TH>Primers numbers</TH><TH>Primers names</TH><TH>Brick created</TH></TR></THEAD><TBODY><TR><TD>27 - 28</TD><TD>BBa_K1051206-upBBa_K1051206-down</TD><TD>BBa_K1051206</TD></TR><TR><TD>29 -30</TD><TD>BBa_K1051207-upBBa_K1051207-down</TD><TD>BBa_K1051207</TD></TR><TR><TD>31 -32</TD><TD>BBa_K1051208-upBBa_K1051208-down</TD><TD>BBa_K1051208</TD></TR></TBODY></TABLE><P>Each annealing oligonucleotides were leagued in pSB1C3 digested XSP. DH5 alpha supercompetent were transformed  with these ligation products and spread on Cm dish.</P></LI><LI><P>Several plasmids were purified thanks to Macherey Nagel MiniPrep Kit :</P><P>
                      SLIC CheA PCR product 5/6 Clone 34, 35, 36, 37, 38 and 39 ⇒ PiGEM_02.21, PiGEM_02.22, PiGEM_02.23, PiGEM_02.24, PiGEM_02.25, PiGEM_02.26
                    </P></LI><LI><P>Several control PCR were realized to verify one more time whether SLIC worked. Besides, several control PCR were realized to verify whether bricks 200 and bricks 202 worked. Taq polymerase was used. </P><TABLE class="table table-hover table-bordered notes-table"><THEAD><TR><TH>#</TH><TH>DNA Template</TH><TH>Primers Numbers</TH><TH>Primers Names</TH><TH>Weight of amplified DNA</TH></TR></THEAD><TBODY><TR><TD>39</TD><TD>PiGEM_02.10</TD><TD>5 - 6</TD><TD>serA_PrefRFC10RBS_upserA_tronq_SufRFC23_down</TD><TD>1032 bp</TD></TR><TR><TD>40</TD><TD>PiGEM_02.11</TD><TD>5 - 6</TD><TD>serA_PrefRFC10RBS_upserA_tronq_SufRFC23_down</TD><TD>1032 bp</TD></TR><TR><TD>41</TD><TD>PiGEM_02.16</TD><TD>17 - 18</TD><TD>relA_RFC10-RBS_uprelA_sans_stop_SufRFC23_down</TD><TD>2259 bp</TD></TR><TR><TD>42</TD><TD>PiGEM_02.17</TD><TD>17 - 18</TD><TD>relA_RFC10-RBS_uprelA_sans_stop_SufRFC23_down</TD><TD>2259 bp</TD></TR><TR><TD>43</TD><TD>PiGEM_02.18</TD><TD>17 - 19</TD><TD>relA_RFC10-RBS_uprelA_2_stop_SufRFC23_down</TD><TD>2259 bp</TD></TR><TR><TD>44</TD><TD>PiGEM_02.19</TD><TD>17 - 19</TD><TD>relA_RFC10-RBS_uprelA_2_stop_SufRFC23_down</TD><TD>2259 bp</TD></TR><TR><TD>45</TD><TD>PiGEM_02.20</TD><TD>9 - 10</TD><TD>cheA_PrefRFC23_upcheA_SufRFC23_down</TD><TD>2432 bp</TD></TR><TR><TD>46</TD><TD>PiGEM_02.21</TD><TD>9 - 10</TD><TD>cheA_PrefRFC23_upcheA_SufRFC23_down</TD><TD>2432 bp</TD></TR><TR><TD>47</TD><TD>PiGEM_02.22</TD><TD>9 - 10</TD><TD>cheA_PrefRFC23_upcheA_SufRFC23_down</TD><TD>2432 bp</TD></TR><TR><TD>48</TD><TD>PiGEM_02.23</TD><TD>9 - 10</TD><TD>cheA_PrefRFC23_upcheA_SufRFC23_down</TD><TD>2432 bp</TD></TR><TR><TD>49</TD><TD>PiGEM_02.24</TD><TD>9 - 10</TD><TD>cheA_PrefRFC23_upcheA_SufRFC23_down</TD><TD>2432 bp</TD></TR><TR><TD>50</TD><TD>PiGEM_02.25</TD><TD>9 - 10</TD><TD>cheA_PrefRFC23_upcheA_SufRFC23_down</TD><TD>2432 bp</TD></TR><TR><TD>51</TD><TD>PiGEM_02.26</TD><TD>9 - 10</TD><TD>cheA_PrefRFC23_upcheA_SufRFC23_down</TD><TD>2432 bp</TD></TR><TR><TD>52</TD><TD>PiGEM_01.04</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>706 bp</TD></TR><TR><TD>53</TD><TD>PiGEM_02.10</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>1032 bp</TD></TR><TR><TD>54</TD><TD>Brick 202 (1)</TD><TD>40 - 51</TD><TD>BBa_E0040_NoStop-downBBa_B0034-up</TD><TD>730 bp</TD></TR><TR><TD>55</TD><TD>Brick 202 (2)</TD><TD>40 - 51</TD><TD>BBa_E0040_NoStop-downBBa_B0034-up</TD><TD>730 bp</TD></TR><TR><TD>56</TD><TD>Brick 202 (3)</TD><TD>40 - 51</TD><TD>BBa_E0040_NoStop-downBBa_B0034-up</TD><TD>730 bp</TD></TR><TR><TD>57</TD><TD>Brick 202 (4)</TD><TD>40 - 51</TD><TD>BBa_E0040_NoStop-downBBa_B0034-up</TD><TD>730 bp</TD></TR><TR><TD>58</TD><TD>Brick 200 (3)</TD><TD>38 - 51</TD><TD>BBa_E0040_NoStop-downBBa_B0034-up</TD><TD>711 bp</TD></TR><TR><TD>59</TD><TD>Brick 200 (4)</TD><TD>38 - 51</TD><TD>BBa_E0040_NoStop-downBBa_B0034-up</TD><TD>730 bp</TD></TR><TR><TD>60</TD><TD>Brick 200 (6)</TD><TD>38 - 51</TD><TD>BBa_E0040_NoStop-downBBa_B0034-up</TD><TD>711 bp</TD></TR></TBODY></TABLE></LI><LI><P>Because PCR 14 did not work, cusR promoter was amplified again thanks to iGEM PCR protocol (Q5 polymerase).</P><TABLE class="table table-hover table-bordered notes-table"><THEAD><TR><TH>#</TH><TH>DNA Template</TH><TH>Primers Numbers</TH><TH>Primers Names</TH><TH>Weight of amplified DNA</TH></TR></THEAD><TBODY><TR><TD>62</TD><TD>Genomic DNA of W3110</TD><TD>33 - 34</TD><TD>CusR_iProm-upCusR_iProm-down</TD><TD>117 bp</TD></TR></TBODY></TABLE></LI></UL></DIV><DIV class="tab-pane fade in" id="0806"><UL class="list-notebook"><LI><P>The efficiency of the amplification was tested thanks to agarose gel electrophoresis. Results are presented below.</P><P>SLIC had to be made again. Bricks 200 were correctly amplified but bricks 202 were not. CusR promoter was not amplified.</P></LI><LI><P>Because PCR 62 did not work, cusR promoter was amplified again thanks to iGEM PCR protocol (Q5 polymerase).</P><TABLE class="table table-hover table-bordered notes-table"><THEAD><TR><TH>#</TH><TH>DNA Template</TH><TH>Primers Numbers</TH><TH>Primers Names</TH><TH>Weight of amplified DNA</TH></TR></THEAD><TBODY><TR><TD>63</TD><TD>Genomic DNA of W3110</TD><TD>33 - 34</TD><TD>CusR_jProm-upCusR_jProm-down</TD><TD>117 bp</TD></TR></TBODY></TABLE></LI><LI><P>Several control PCR were realized to verify whether bricks 200 worked. Taq polymerase was used. </P><TABLE class="table table-hover table-bordered notes-table"><THEAD><TR><TH>#</TH><TH>DNA Template</TH><TH>Primers Numbers</TH><TH>Primers Names</TH><TH>Weight of amplified DNA</TH></TR></THEAD><TBODY><TR><TD>64</TD><TD>Brick 202 (5)</TD><TD>40 - 51</TD><TD>BBa_E0040_NoStop-downBBa_B0034-up</TD><TD>730 bp</TD></TR><TR><TD>65</TD><TD>Brick 202 (6)</TD><TD>40 - 51</TD><TD>BBa_E0040_NoStop-downBBa_B0034-up</TD><TD>730 bp</TD></TR><TR><TD>646</TD><TD>Brick 202 (7)</TD><TD>40 - 51</TD><TD>BBa_E0040_NoStop-downBBa_B0034-up</TD><TD>730 bp</TD></TR><TR><TD>67</TD><TD>Brick 202 (8)</TD><TD>40 - 51</TD><TD>BBa_E0040_NoStop-downBBa_B0034-up</TD><TD>730 bp</TD></TR><TR><TD>68</TD><TD>Brick 202 (9)</TD><TD>40 - 51</TD><TD>BBa_E0040_NoStop-downBBa_B0034-up</TD><TD>730 bp</TD></TR><TR><TD>69</TD><TD>Brick 202 (10)</TD><TD>40 - 51</TD><TD>BBa_E0040_NoStop-downBBa_B0034-up</TD><TD>730 bp</TD></TR><TR><TD>70</TD><TD>Brick 202 (11)</TD><TD>40 - 51</TD><TD>BBa_E0040_NoStop-downBBa_B0034-up</TD><TD>730 bp</TD></TR></TBODY></TABLE></LI><LI><P>A gel extract of PCR product 38 (SH3)<SUB>4</SUB> was made.</P></LI></UL></DIV><DIV class="tab-pane fade in" id="0807"><UL class="list-notebook"><LI><P>Several control PCR were realized to verify whether RFP without STOP codon and GFP without STOP codon worked. Taq polymerase was used.</P><TABLE class="table table-hover table-bordered notes-table"><THEAD><TR><TH>#</TH><TH>DNA Template</TH><TH>Primers Numbers</TH><TH>Primers Names</TH><TH>Weight of amplified DNA</TH></TR></THEAD><TBODY><TR><TD>72</TD><TD>RFP without STOP codonClone 40</TD><TD>37 - 38</TD><TD>BBa_E1010-upBBa_E1010_NoStop-down</TD><TD>708 bp</TD></TR><TR><TD>73</TD><TD>RFP without STOP codonClone 41</TD><TD>37 - 38</TD><TD>BBa_E1010-upBBa_E1010_NoStop-down</TD><TD>708 bp</TD></TR><TR><TD>74</TD><TD>RFP without STOP codonClone 42</TD><TD>37 - 38</TD><TD>BBa_E1010-upBBa_E1010_NoStop-down</TD><TD>708 bp</TD></TR><TR><TD>75</TD><TD>RFP without STOP codonClone 43</TD><TD>37 - 38</TD><TD>BBa_E1010-upBBa_E1010_NoStop-down</TD><TD>708 bp</TD></TR><TR><TD>76</TD><TD>RFP without STOP codonClone 44</TD><TD>37 - 38</TD><TD>BBa_E1010-upBBa_E1010_NoStop-down</TD><TD>708 bp</TD></TR><TR><TD>77</TD><TD>GFP without STOP codonClone 45</TD><TD>39 - 40</TD><TD>BBa_E0040-upBBa_E0040_NoStop-down</TD><TD>714 bp</TD></TR><TR><TD>78</TD><TD>GFP without STOP codonClone 46</TD><TD>39 - 40</TD><TD>BBa_E0040-upBBa_E0040_NoStop-down</TD><TD>714 bp</TD></TR><TR><TD>79</TD><TD>GFP without STOP codonClone 47</TD><TD>39 - 40</TD><TD>BBa_E0040-upBBa_E0040_NoStop-down</TD><TD>714 bp</TD></TR><TR><TD>80</TD><TD>GFP without STOP codonClone 48</TD><TD>39 - 40</TD><TD>BBa_E0040-upBBa_E0040_NoStop-down</TD><TD>714 bp</TD></TR><TR><TD>81</TD><TD>GFP without STOP codonClone 49</TD><TD>39 - 40</TD><TD>BBa_E0040-upBBa_E0040_NoStop-down</TD><TD>714 bp</TD></TR></TBODY></TABLE></LI><LI><P>The efficiency of the purification was tested thanks to agarose gel electrophoresis. Results are presented following.</P><DIV class="media notes-media"><DIV class="media-body"><P>PCR product 63 was correctly amplified. PiGEM_02.27 (pSB1A2, BBa_J04450) was digested with EcoRI and Pst. PCR product 63 (cusR promoter) was digested by EcoRI and PstI. Then, PCR product 63  was leagued in pSB1A2 digested. DH5 alpha supercompetent were transformed  with this ligation product and spread on Amp dish.</P></DIV></DIV></LI><LI><P>To confirm the cloning of stability label and SLIC, each plasmid containing constructions were digested with specific restriction enzymes.</P><TABLE class="table table-hover table-bordered notes-table"><THEAD><TR><TH rowspan="2">Plasmid Numbers</TH><TH rowspan="2">Plasmid Names</TH><TH colspan="4">Digestion</TH><TH rowspan="2">Weight of expected bands if constructions are correct</TH></TR><TR><TH>EcoRI</TH><TH>PstI</TH><TH>SpeI</TH><TH>XbaI</TH></TR></THEAD><TBODY><TR><TD>PiGEM_01.10</TD><TD>BBa_K1051207Clone St1</TD><TD>X</TD><TD>X</TD></TR></TBODY></TABLE></LI><LI>2082 pb</LI><LI>80 pb</LI></UL>PiGEM_01.11BBa_K1051207Clone St2XX<UL><LI>2082 pb</LI><LI>80 pb</LI></UL>PiGEM_01.12BBa_K1051208Clone St3XX<UL><LI>2082 pb</LI><LI>80 pb</LI></UL>PiGEM_02.10SLIC SerA produit PCR 2/3Clone 1X<UL><LI>2352 pb</LI><LI>847 pb</LI></UL>PiGEM_02.11SLIC SerA produit PCR 2/3Clone 2X<UL><LI>2352 pb</LI><LI>847 pb</LI></UL>PiGEM_02.12SLIC Mesh1 sans STOP produit PCR 10/11/12Clone 4XX<UL><LI>2422 pb</LI><LI>180 pb</LI><LI>84</LI></UL>PiGEM_02.13SLIC Mesh1 sans STOP produit PCR 10/11/12Clone 6XX<UL><LI>2422 pb</LI><LI>180 pb</LI><LI>84</LI></UL>PiGEM_02.14SLIC Mesh1 sans STOP produit PCR 10/11/13Clone 7XX<UL><LI>2422 pb</LI><LI>180 pb</LI><LI>84</LI></UL>PiGEM_02.15SLIC Mesh1 sans STOP produit PCR 10/11/13Clone 8XX<UL><LI>2422 pb</LI><LI>180 pb</LI><LI>84</LI></UL>PiGEM_02.16SLIC RelA sans STOP produit PCR 7/8Clone 28X<UL><LI>3530 pb</LI><LI>872 pb</LI></UL>PiGEM_02.17SLIC RelA sans STOP produit PCR 7/8Clone 29X<UL><LI>3530 pb</LI><LI>872 pb</LI></UL>PiGEM_02.18SLIC RelA sans STOP produit PCR 7/8Clone 31X<UL><LI>3530 pb</LI><LI>872 pb</LI></UL>PiGEM_02.19SLIC RelA sans STOP produit PCR 7/8Clone 32X<UL><LI>3530 pb</LI><LI>872 pb</LI></UL>PiGEM_02.24SLIC CheA produit PCR 5/6Clone 37X<UL><LI>3457 pb</LI><LI>675 pb</LI></UL>PiGEM_02.25SLIC CheA produit PCR 5/6Clone 38X<UL><LI>3457 pb</LI><LI>675 pb</LI></UL>PiGEM_02.26SLIC CheA produit PCR 5/6Clone 39X<UL><LI>3457 pb</LI><LI>675 pb</LI></UL><P>The efficiency of the digestion was tested thanks to agarose gel electrophoresis. Results are presented below.</P><P>For stability label, linearized plasmid was the only fragment visible. For SLIC, no fragment matched with the weight of expected bands. All SLIC construction had to be made again.</P><LI><P>PiGEM_02.27 (pSB1A2, BBa_J04450) was digested with XbaI, SpeI and PstI to be used for stability label cloning. Stability labels were assembled again by annealing.</P><TABLE class="table table-hover table-bordered notes-table"><THEAD><TR><TH>Primers numbers</TH><TH>Primers names</TH><TH>Brick created</TH></TR></THEAD><TBODY><TR><TD>27 - 28</TD><TD>BBa_K1051206-upBBa_K1051206-down</TD><TD>BBa_K1051206</TD></TR><TR><TD>29 -30</TD><TD>BBa_K1051207-upBBa_K1051207-down</TD><TD>BBa_K1051207</TD></TR><TR><TD>31 -32</TD><TD>BBa_K1051208-upBBa_K1051208-down</TD><TD>BBa_K1051208</TD></TR></TBODY></TABLE><P>Each annealing oligonucleotides were leagued in pSB1A2 digested XSP. DH5 alpha supercompetent were transformed  with these ligation products and spread on Amp dish.</P></LI><LI><P>PiGEM_02.27 (pSB1A2, BBa_J04450) was digested with XbaI and SpeI. CusR box and Flag were assembled by annealing.</P><TABLE class="table table-hover table-bordered notes-table"><THEAD><TR><TH>Primers numbers</TH><TH>Primers names</TH><TH>Brick created</TH></TR></THEAD><TBODY><TR><TD>35 - 36</TD><TD>CusR_Box-upCusR_Box-down</TD><TD>CusR Box</TD></TR><TR><TD>41 - 42</TD><TD>Flag-upFlag-down</TD><TD>Flag</TD></TR></TBODY></TABLE><P>Each annealing oligonucleotides were leagued in pSB1A2 digested XS. DH5 alpha supercompetent were transformed  with these ligation products and spread on Amp dish.</P></LI></DIV><DIV class="tab-pane fade in" id="0808"><UL class="list-notebook"><LI><P>SLIC assembly of CheA, SerA, RelA with and without STOP codon and Mesh 1 with and without STOP codon were restarted. DNA fragments were amplified thanks to iGEM PCR protocol (Q5 polymerase). </P><TABLE class="table table-hover table-bordered notes-table"><THEAD><TR><TH>#</TH><TH>DNA Template</TH><TH>Primers Numbers</TH><TH>Primers Names</TH><TH>Weight of amplified DNA</TH></TR></THEAD><TBODY><TR><TD>82</TD><TD>Genomic DNA of W3110</TD><TD>6 - 7</TD><TD>serA_mut_ A849G_upserA_tronq_SufRFC23_down</TD><TD>171 bp</TD></TR><TR><TD>83</TD><TD>Genomic DNA of W3110</TD><TD>5 - 8</TD><TD>serA_PrefRFC10RBS_upserA_mut_A849_down</TD><TD>861 bp</TD></TR><TR><TD>84</TD><TD>PiGEM_02.05</TD><TD>15 - 17</TD><TD>relA_mut_C1366G_uprelA_RFC10-RBS_up</TD><TD>1377 bp</TD></TR><TR><TD>85</TD><TD>PiGEM_02.05</TD><TD>16 - 18</TD><TD>relA_mut_C1366_downrelA_sans_stop_SufRFC23_down</TD><TD>882 bp</TD></TR><TR><TD>86</TD><TD>PiGEM_02.05</TD><TD>16 - 19</TD><TD>relA_mut_C1366_downrelA_2_stop_SufRFC23_down</TD><TD>885 bp</TD></TR><TR><TD>87</TD><TD>PiGEM_02.06</TD><TD>20 - 24</TD><TD>Mesh_1_mut_A255T_upMesh_1_RFC10-RBS_up</TD><TD>267 bp</TD></TR><TR><TD>88</TD><TD>PiGEM_02.06</TD><TD>21 - 22</TD><TD>Mesh_1_mut_A255T_downMesh_1_mut_A343G_up</TD><TD>120 bp</TD></TR><TR><TD>89</TD><TD>PiGEM_02.06</TD><TD>23 - 25</TD><TD>Mesh_1_mut_A343G_downMesh1_sans_stop_SufRFC23_down</TD><TD>192 bp</TD></TR><TR><TD>90</TD><TD>PiGEM_02.06</TD><TD>23 - 26</TD><TD>Mesh_1_mut_A343G_downMesh_1_2_stop_SufRFC23_down</TD><TD>192 bp</TD></TR></TBODY></TABLE><P>The efficiency of the amplification was tested thanks to agarose gel electrophoresis. Results are presented below.</P><P>All DNA fragments were correctly amplified. Thus, SLIC protocol was realized to assemble each fragment of CheA, SerA, RelA and Mesh1. Plasmid pSB1A2 digested XbaI and SpeI was used.</P></LI><LI><P>Several control PCR had be realized the day before to verify whether RFP without STOP codon and GFP without STOP codon worked. The efficiency of the amplification was tested thanks to agarose gel electrophoresis. Results are presented following.</P><DIV class="media notes-media"><DIV class="media-body"><P>Only clones 41, 42 43 and 44 of RPF without STOP codon and clones 47 and 48 of GFP without STOP codon seemed to be correct.</P></DIV></DIV></LI></UL></DIV><DIV class="tab-pane fade in" id="0809"><UL class="list-notebook"><LI><P>Several plasmids were purified thanks to Macherey Nagel MiniPrep Kit :</P></LI><LI>PCR product 15 RFP without STOP Clone 41 ⇒ PiGEM_01.14</LI><LI>PCR product 15 RFP without STOP Clone 43 ⇒ PiGEM_01.15</LI><LI>PCR product 16 GFP without STOP Clone 47 ⇒ PiGEM_01.16</LI><LI>PCR product 16 GFP without STOP Clone 48 ⇒ PiGEM_01.17</LI></UL></DIV></DIV></DIV><DIV class="panel-footer">
            Protocols used : ...
          </DIV></DIV><DIV class="panel panel-info notes-panel"><DIV class="panel-heading"><H3 class="panel-title">Week 7 : 08/11/2014 ➡ 08/17/2014 </H3></DIV><DIV class="panel-body"><UL class="nav nav-tabs" role="tablist"><LI class="active"><A href="#0811" role="tab" data-toggle="tab" class="date-notebook">08/11/14</A></LI><LI><A href="#0812" role="tab" data-toggle="tab" class="date-notebook">08/12/14</A></LI><LI><A href="#0813" role="tab" data-toggle="tab" class="date-notebook">08/13/14</A></LI><LI><A href="#0814" role="tab" data-toggle="tab" class="date-notebook">08/14/14</A></LI><LI><A href="#0815" role="tab" data-toggle="tab" class="date-notebook">08/15/14</A></LI></UL><DIV class="tab-content"><DIV class="tab-pane fade in active" id="0811"><UL class="list-notebook"><LI><P>Several control PCR were realized to verify whether cusR promoter, RFP and GFP without STOP codon, cusR-L-LZa, SH3pep, cusR box and SLIC worked. Taq polymerase was used.</P><TABLE class="table table-hover table-bordered notes-table"><THEAD><TR><TH>#</TH><TH>DNA Template</TH><TH>Primers Numbers</TH><TH>Primers Names</TH><TH>Weight of amplified DNA</TH></TR></THEAD><TBODY><TR><TD>106</TD><TD>cusR promoterClone 1</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>117 bp</TD></TR><TR><TD>107</TD><TD>cusR promoterClone 2</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>117 bp</TD></TR><TR><TD>108</TD><TD>cusR promoterClone 3</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>117 bp</TD></TR><TR><TD>109</TD><TD>RFP without STOP codonClone 41</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>708 bp</TD></TR><TR><TD>110</TD><TD>RFP without STOP codonClone 42</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>708 bp</TD></TR><TR><TD>111</TD><TD>RFP without STOP codonClone 43</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>708 bp</TD></TR><TR><TD>112</TD><TD>RFP without STOP codonClone 44</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>708 bp</TD></TR><TR><TD>113</TD><TD>RFP without STOP codonClone 47</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>708 bp</TD></TR><TR><TD>114</TD><TD>RFP without STOP codonClone 48</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>708 bp</TD></TR><TR><TD>126</TD><TD>SH3pepClone 4</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>192 bp</TD></TR><TR><TD>127</TD><TD>SH3pepClone 6</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>192 bp</TD></TR><TR><TD>128</TD><TD>SH3pepClone 8</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>192 bp</TD></TR><TR><TD>129</TD><TD>SH3pepClone 10</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>192 bp</TD></TR><TR><TD>130</TD><TD>K1051207Clone St1</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>86 bp</TD></TR><TR><TD>131</TD><TD>K1051207Clone St2</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>86 bp</TD></TR><TR><TD>132</TD><TD>K1051207Clone St3</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>80 bp</TD></TR><TR><TD>133</TD><TD>cusR boxClone 1</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>117 bp</TD></TR><TR><TD>134</TD><TD>cusR boxClone 4</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>117 bp</TD></TR><TR><TD>135</TD><TD>cusR boxClone 5</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>117 bp</TD></TR><TR><TD>136</TD><TD>cusR boxClone 7</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>117 bp</TD></TR><TR><TD>137</TD><TD>SLIC bis CheACA5</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>2432 bp</TD></TR><TR><TD>138</TD><TD>SLIC bis CheACA6</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>2432 bp</TD></TR><TR><TD>139</TD><TD>SLIC bis CheACA8</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>2432 bp</TD></TR><TR><TD>140</TD><TD>SLIC bis CheACA9</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>2432 bp</TD></TR><TR><TD>141</TD><TD>SLIC bis CheACA10</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>2432 bp</TD></TR><TR><TD>142</TD><TD>SLIC bis SerASA1</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>1032 bp</TD></TR><TR><TD>143</TD><TD>SLIC bis SerASA3</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>1032 bp</TD></TR><TR><TD>144</TD><TD>SLIC bis SerASA4</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>1032 bp</TD></TR><TR><TD>145</TD><TD>SLIC bis SerASA5</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>1032 bp</TD></TR><TR><TD>146</TD><TD>SLIC bis SerASA6</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>1032 bp</TD></TR><TR><TD>147</TD><TD>SLIC bis RelA with STOP codonRA2</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>2260 bp</TD></TR><TR><TD>148</TD><TD>SLIC bis RelA with STOP codonRA5</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>2260 bp</TD></TR><TR><TD>149</TD><TD>SLIC bis RelA with STOP codonRA6</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>2260 bp</TD></TR><TR><TD>150</TD><TD>SLIC bis RelA with STOP codonRA7</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>2260 bp</TD></TR><TR><TD>151</TD><TD>SLIC bis RelA with STOP codonRA8</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>2260 bp</TD></TR><TR><TD>152</TD><TD>SLIC bis RelA without STOP codonRAS2</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>2260 bp</TD></TR><TR><TD>153</TD><TD>SLIC bis RelA without STOP codonRAS3</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>2260 bp</TD></TR><TR><TD>154</TD><TD>SLIC bis RelA without STOP codonRAS4</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>2260 bp</TD></TR><TR><TD>155</TD><TD>SLIC bis RelA without STOP codonRAS5</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>2260 bp</TD></TR><TR><TD>156</TD><TD>SLIC bis RelA without STOP codonRAS6</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>2260 bp</TD></TR><TR><TD>157</TD><TD>SLIC bis Mesh1 without STOPMSS1</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>579 bp</TD></TR><TR><TD>158</TD><TD>SLIC bis Mesh1 without STOPMSS2</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>579 bp</TD></TR><TR><TD>159</TD><TD>SLIC bis Mesh1 without STOPMSS3</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>579 bp</TD></TR><TR><TD>160</TD><TD>SLIC bis Mesh1 without STOPMSS4</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>579 bp</TD></TR><TR><TD>161</TD><TD>SLIC bis Mesh1 without STOPMSS5</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>579 bp</TD></TR><TR><TD>162</TD><TD>SLIC bis Mesh1 with STOPMS1</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>579 bp</TD></TR><TR><TD>163</TD><TD>SLIC bis Mesh1 with STOPMS2</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>579 bp</TD></TR><TR><TD>164</TD><TD>SLIC bis Mesh1 with STOPMS3</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>579 bp</TD></TR><TR><TD>165</TD><TD>SLIC bis Mesh1 with STOPMS4</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>579 bp</TD></TR><TR><TD>166</TD><TD>SLIC bis Mesh1 with STOPMS5</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>579 bp</TD></TR><TR><TD>167</TD><TD>PiGEM_01.04</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>708 bp</TD></TR><TR><TD>167</TD><TD>PiGEM_01.04</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>708 bp</TD></TR></TBODY></TABLE><P>The efficiency of the amplification was tested thanks to agarose gel electrophoresis. Results are presented below.</P><DIV class="media notes-media"><DIV class="media-body"><P>Neither cusR promoter no stability label clone seemed to present the right construction, these cloning had to be remade. Nevertheless, every SH3pep and cusR box clones seemed to be correctly amplified. Clones 1 and 4 of cusR box (CB1 and CB4) and clones 4 and 10 of SH3pep (SP4 and SP10) were chosen to go on brick construction.</P></DIV></DIV><DIV class="media notes-media"><DIV class="media-body"><P>Clones 42, 43, 44, 47 and 48 of RFP and GFP without STOP codon seemed to be correctly amplified. Only clones 1,2 and 3 of cusR-L-LZa seemed to present the right construction. Clones 42, 43, 47 and 48 of RFP and GFP without STOP codon (RFP42, RFP43, GFP47 and GFP48) and clones 2 and 3 of cusR-L-LZa (CZ2 and CZ3) were chosen to go on brick construction.</P></DIV></DIV><DIV class="media notes-media"><DIV class="media-body"><P>No (SH3)4 clone seemed to present the right construction, this cloning had to be remade. Only clones 3 and 4 of Mesh1 without STOP codon (MSS3 and MSS4) seemed to present the right construction, these clones were chosen to go on brick construction. No clone of Mesh1 with STOP codon was correctly amplified, other clones were going to be tested by PCR.</P></DIV></DIV><DIV class="media notes-media"><DIV class="media-body"><P>Only clone 5 of SLIC CheA (CA5), clones 2 and 5 of RelA with STOP codon (RA2 and RA5) and clones 4 and 5 of RelA without STOP codon (RAS4 and RAS5) seemed to present the right construction. Thus, only these clones were chosen to go on brick construction.</P></DIV></DIV><DIV class="media notes-media"><DIV class="media-body"><P>Clone 1 was the only clone of SLIC SerA correctly amplified. SA1 was chosen to go on brick construction.</P></DIV></DIV></LI><LI><P>TG1 competent cells were prepared to substitute DH5 alpha that grows up too slowly.</P></LI><LI><P>PiGEM_02.27 (pSB1A2, BBa_J04450) was digested with XbaI, SpeI and PstI to be used for stability label cloning. Stability labels were assembled a third time by annealing.</P><TABLE class="table table-hover table-bordered notes-table"><THEAD><TR><TH>Primers numbers</TH><TH>Primers names</TH><TH>Brick created</TH></TR></THEAD><TBODY><TR><TD>27 - 28</TD><TD>BBa_K1051206-upBBa_K1051206-down</TD><TD>BBa_K1051206</TD></TR><TR><TD>29 -30</TD><TD>BBa_K1051207-upBBa_K1051207-down</TD><TD>BBa_K1051207</TD></TR><TR><TD>31 -32</TD><TD>BBa_K1051208-upBBa_K1051208-down</TD><TD>BBa_K1051208</TD></TR></TBODY></TABLE><P>Each annealing oligonucleotides were leagued in pSB1A2 digested XSP. DH5 alpha supercompetent were transformed with these ligation products and spread on Amp dish.</P></LI></UL></DIV><DIV class="tab-pane fade in" id="0812"><UL class="list-notebook"><LI><P>Because PCR 106 to 108 did not work, cusR promoter was amplified again thanks to iGEM PCR protocol (Q5 polymerase).</P><TABLE class="table table-hover table-bordered notes-table"><THEAD><TR><TH>#</TH><TH>DNA Template</TH><TH>Primers Numbers</TH><TH>Primers Names</TH><TH>Weight of amplified DNA</TH></TR></THEAD><TBODY><TR><TD>169</TD><TD>Genomic DNA of W3110</TD><TD>33 - 34</TD><TD>CusR_iProm-upCusR_iProm-down</TD><TD>117 bp</TD></TR></TBODY></TABLE><P>The efficiency of the amplification was tested thanks to agarose gel electrophoresis. Results are presented below.</P><DIV class="media notes-media"><DIV class="media-body"><P>Once again, the amplification of cusR promoter did not work.</P></DIV></DIV></LI><LI><P>Several control PCR were realized to verify whether SLIC Mesh1 with STOP codon, (SH3)4 worked. Taq polymerase was used.</P><TABLE class="table table-hover table-bordered notes-table"><THEAD><TR><TH>#</TH><TH>DNA Template</TH><TH>Primers Numbers</TH><TH>Primers Names</TH><TH>Weight of amplified DNA</TH></TR></THEAD><TBODY><TR><TD>170</TD><TD>SLIC bis Mesh1 with STOPMS6</TD><TD>43 - 44</TD><TD>CusR_iProm-upCusR_iProm-down</TD><TD>579 bp</TD></TR><TR><TD>171</TD><TD>SLIC bis Mesh1 with STOPMS7</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>579 bp</TD></TR><TR><TD>172</TD><TD>SLIC bis Mesh1 with STOPMS8</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>579 bp</TD></TR><TR><TD>173</TD><TD>SLIC bis Mesh1 with STOPMS9</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>579 bp</TD></TR><TR><TD>174</TD><TD>SLIC bis Mesh1 with STOPMS10</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>579 bp</TD></TR><TR><TD>175</TD><TD>(SH3)<SUB>4</SUB>Clones 11, 12, 13, 14, 15</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>932 bp</TD></TR><TR><TD>176</TD><TD>(SH3)<SUB>4</SUB>Clones 16, 17, 18, 19, 20</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>932 bp</TD></TR><TR><TD>177</TD><TD>(SH3)<SUB>4</SUB>Clones 21, 22, 23, 24, 25</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>932 bp</TD></TR><TR><TD>178</TD><TD>(SH3)<SUB>4</SUB>Clones 26, 27, 28, 29, 30</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>932 bp</TD></TR><TR><TD>179</TD><TD>(SH3)<SUB>4</SUB>Clones 31, 32, 33, 34, 35</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>932 bp</TD></TR></TBODY></TABLE><DIV class="media notes-media"><DIV class="media-body"><P>Clone 8 was the only clone of SLIC Mesh1 with STOP codon correctly amplified. MS8 was chosen to go on brick construction. No clone of (SH3)<SUB>4</SUB> seemed to be correctly amplified.</P></DIV></DIV></LI><LI><P>We have control if the insert of FRT-Kan-FRT is in the gene sdaBC and CheA by PCR.</P><TABLE class="table table-hover table-bordered notes-table"><THEAD><TR><TH>#</TH><TH>DNA Template</TH><TH>Primers Name</TH><TH>Weight of amplified DNA</TH></TR></THEAD><TBODY><TR><TD>180</TD><TD>Genomic DNA of W3110</TD><TD>CheA_upCheA_down</TD><TD>1700</TD></TR><TR><TD>181</TD><TD>CheA-</TD><TD>CheA_upCheA_down</TD><TD>1500</TD></TR><TR><TD>182</TD><TD>Genomic DNA of W3110</TD><TD>sdaBC_upsdaBC_down</TD><TD>3100</TD></TR><TR><TD>183</TD><TD>sdaBC- KanR</TD><TD>sdaBC_upsdaBC_down</TD><TD>1500</TD></TR><TR><TD>184</TD><TD>sdaBC- KanR + pCP20</TD><TD>sdaBC_upsdaBC_down</TD><TD>1500</TD></TR></TBODY></TABLE><P>CheA- control is unsuccessful, we see a weight of amplification for #180 and #181 of 1700bp  the gene FRT-Kan-FRT isn't insert in cheA.</P><P>&gt;SdaBC- control have a problem we don't have amplification for #182 to #184.</P><P>Results not shown.</P></LI></UL></DIV><DIV class="tab-pane fade in" id="0813"><UL class="list-notebook"><LI><P>Several control PCR were realized to verify whether stability label and (SH3)4 cloning worked. Taq polymerase was used.</P><TABLE class="table table-hover table-bordered notes-table"><THEAD><TR><TH>#</TH><TH>DNA Template</TH><TH>Primers Numbers</TH><TH>Primers Names</TH><TH>Weight of amplified DNA</TH></TR></THEAD><TBODY><TR><TD>195</TD><TD>K1051206Clone St4</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>80 bp</TD></TR><TR><TD>196</TD><TD>K1051206Clone St5</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>80 bp</TD></TR><TR><TD>197</TD><TD>K1051206Clone St6</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>80 bp</TD></TR><TR><TD>198</TD><TD>K1051206Clone St7</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>80 bp</TD></TR><TR><TD>199</TD><TD>K1051206Clone St8</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>80 bp</TD></TR><TR><TD>200</TD><TD>K1051206Clone St9</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>80 bp</TD></TR><TR><TD>201</TD><TD>K1051206Clone St10</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>80 bp</TD></TR><TR><TD>202</TD><TD>K1051206Clone St11</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>80 bp</TD></TR><TR><TD>203</TD><TD>K1051206Clone St12</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>80 bp</TD></TR><TR><TD>204</TD><TD>K1051206Clone St13</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>80 bp</TD></TR><TR><TD>205</TD><TD>K1051207Clone St14</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>86 bp</TD></TR><TR><TD>206</TD><TD>K1051207Clone St15</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>86 bp</TD></TR><TR><TD>207</TD><TD>K1051207Clone St16</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>86 bp</TD></TR><TR><TD>208</TD><TD>K1051207Clone St17</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>86 bp</TD></TR><TR><TD>209</TD><TD>K1051207Clone St18</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>86 bp</TD></TR><TR><TD>210</TD><TD>K1051207Clone St19</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>86 bp</TD></TR><TR><TD>211</TD><TD>K1051207Clone St20</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>86 bp</TD></TR><TR><TD>212</TD><TD>K1051207Clone St21</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>86 bp</TD></TR><TR><TD>213</TD><TD>K1051207Clone St22</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>86 bp</TD></TR><TR><TD>214</TD><TD>K1051207Clone St23</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>86 bp</TD></TR><TR><TD>215</TD><TD>K1051208Clone St24</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>80 bp</TD></TR><TR><TD>216</TD><TD>K1051208Clone St25</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>80 bp</TD></TR><TR><TD>217</TD><TD>K1051208Clone St26</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>80 bp</TD></TR><TR><TD>218</TD><TD>K1051208Clone St27</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>80 bp</TD></TR><TR><TD>219</TD><TD>K1051208Clone St28</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>80 bp</TD></TR><TR><TD>220</TD><TD>K1051208Clone St29</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>80 bp</TD></TR><TR><TD>221</TD><TD>K1051208Clone St30</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>80 bp</TD></TR><TR><TD>222</TD><TD>K1051208Clone St31</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>80 bp</TD></TR><TR><TD>223</TD><TD>K1051208Clone St32</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>80 bp</TD></TR><TR><TD>224</TD><TD>K1051208Clone St33</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>80 bp</TD></TR><TR><TD>225</TD><TD>(SH3)<SUB>4</SUB>Clones sc1 </TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>932 bp</TD></TR><TR><TD>226</TD><TD>(SH3)<SUB>4</SUB>Clones sc2</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>932 bp</TD></TR><TR><TD>227</TD><TD>(SH3)<SUB>4</SUB>Clones sc3</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>932 bp</TD></TR><TR><TD>228</TD><TD>(SH3)<SUB>4</SUB>Clones sc4</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>932 bp</TD></TR><TR><TD>229</TD><TD>(SH3)<SUB>4</SUB>Clones sc5</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>932 bp</TD></TR><TR><TD>230</TD><TD>(SH3)<SUB>4</SUB>Clones sc6</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>932 bp</TD></TR><TR><TD>231</TD><TD>(SH3)<SUB>4</SUB>Clones sc7</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>932 bp</TD></TR><TR><TD>232</TD><TD>(SH3)<SUB>4</SUB>Clones sc8</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>932 bp</TD></TR><TR><TD>233</TD><TD>(SH3)<SUB>4</SUB>Clones sc9</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>932 bp</TD></TR><TR><TD>234</TD><TD>(SH3)<SUB>4</SUB>Clones sc10</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>932 bp</TD></TR></TBODY></TABLE><DIV class="media notes-media"><DIV class="media-body"><P>Clone St8 was the only clone of K1051206 correctly amplified. St8 was chosen to go on brick construction.</P></DIV></DIV><DIV class="media notes-media"><DIV class="media-body"><P>Only clones St16, 17, 19, 20 and 21 of K1051207 seemed to present the right construction. These clones were chosen to go on brick construction.</P></DIV></DIV><DIV class="media notes-media"><DIV class="media-body"><P>Clones 24, 26, 27, 28, 29 and 33 of K1051208 seemed to be correctly amplified. These clones were chosen to go on brick construction.</P></DIV></DIV><DIV class="media notes-media"><DIV class="media-body"><P>No clone of (SH3)<SUB>4</SUB> seemed to present the right construction.</P></DIV></DIV></LI><LI><P>We made a PCR with new oligo to control mutants.</P><TABLE class="table table-hover table-bordered notes-table"><THEAD><TR><TH>#</TH><TH>DNA Template</TH><TH>Primers Name</TH><TH>Weight of amplified DNA</TH></TR></THEAD><TBODY><TR><TD>185</TD><TD>Genomic DNA of W3110</TD><TD>CheA_check_upCheA_check_down</TD><TD>1700</TD></TR><TR><TD>186</TD><TD>CheA-</TD><TD>CheA_check_upCheA_check_down</TD><TD>1500</TD></TR><TR><TD>187</TD><TD>Genomic DNA of W3110</TD><TD>sdaBC_check_upsdaBC_check_down</TD><TD>3100</TD></TR><TR><TD>188</TD><TD>sdaBC- KanR</TD><TD>sdaBC_check_upsdaBC_check_down</TD><TD>1500</TD></TR><TR><TD>189</TD><TD>sdaBC- KanR + pCP20</TD><TD>sdaBC_check_upsdaBC_check_down</TD><TD>1500</TD></TR><TR><TD>190</TD><TD>Genomic DNA of W3110</TD><TD>FRT_upsdaBC_check_down</TD><TD>0</TD></TR><TR><TD>191</TD><TD>sdaBC- KanR</TD><TD>FRT_upsdaBC_check_down</TD><TD>1600</TD></TR><TR><TD>192</TD><TD>sdaBC- KanR + pCP20</TD><TD>FRT_upsdaBC_check_down</TD><TD>1600</TD></TR></TBODY></TABLE><P>Second control of CheA- show us the gene FRT-Kan-FRT isn't in the gene cheA</P><P>SdaBC control have an other problem with amplification</P><P>Results not shown.</P></LI></UL></DIV><DIV class="tab-pane fade in" id="0814"><UL class="list-notebook"><LI><P>Several plasmids were purified thanks to Macherey Nagel MiniPrep Kit :</P></LI><LI>PCR product 15 RFP without STOP codon Clones 42 and 43 ⇒ PiGEM_01.18 and PiGEM_01.19</LI><LI>PCR product 16 GFP without STOP codon Clones 47 and 48 ⇒ PiGEM_01.20 and PiGEM_01.21</LI><LI>CusRbox clones 1 and 4 (CB1 and CB4) ⇒ PiGEM_02.33 and PiGEM_02.34</LI><LI>SH3 pep L-LZa clones 4 and 10 (SP4 and SP10) ⇒ PiGEM_02.35 and PiGEM_02.36</LI><LI>CusR-L-LZa clones 2 and 3 (CZ2 and CZ3) ⇒ PiGEM_02.37 and PiGEM_02.38</LI><LI>SLIC CheA Bis Clone CA5 ⇒ PiGEM_02.39</LI><LI>SLIC SerA Bis Clone SA1 ⇒ PiGEM_02.40</LI><LI>SLIC RelA Bis avec STOP Clones RA2 and RA5 ⇒ PiGEM_02.41 and PiGEM_02.42</LI><LI>SLIC RelA Bis sans STOP Clone RAS4 and RAS5 ⇒ PiGEM_02.43 and PiGEM_02.44</LI><LI>SLIC Mesh1 Bis sans STOP Clone MSS3 and MSS4 ⇒ PiGEM_02.45 and PiGEM_02.46</LI><LI>SLIC CheA Clone 38 ⇒ PiGEM_02.47</LI><LI>SLIC Mesh1 avec STOP MS8 ⇒ PiGEM_02.48</LI></UL><LI><P>CusR promoter  was amplified again thanks to iGEM PCR protocol (Q5 polymerase) by switching matrix.</P><TABLE class="table table-hover table-bordered notes-table"><THEAD><TR><TH>#</TH><TH>DNA Template</TH><TH>Primers Numbers</TH><TH>Primers Names</TH><TH>Weight of amplified DNA</TH></TR></THEAD><TBODY><TR><TD>238</TD><TD>Genomic DNA of W3110</TD><TD>33 - 34</TD><TD>CusR_iProm-upCusR_iProm-down</TD><TD>117 bp</TD></TR><TR><TD>239</TD><TD>PiGEM_02.08</TD><TD>33 - 34</TD><TD>CusR_iProm-upCusR_iProm-down</TD><TD>117 bp</TD></TR></TBODY></TABLE><P>The efficiency of the amplification was tested thanks to agarose gel electrophoresis. Results are presented below.</P><DIV class="media notes-media"><DIV class="media-body"><P>The amplification from the genomique template did not work again but the amplification from the Berkeley plasmid worked very well. CusR promoter cloning could be done again.</P></DIV></DIV></LI><LI><P>Several control PCR were realized to verify whether (SH3)<SUB>4</SUB> worked. Taq polymerase was used.</P><TABLE class="table table-hover table-bordered notes-table"><THEAD><TR><TH>#</TH><TH>DNA Template</TH><TH>Primers Numbers</TH><TH>Primers Names</TH><TH>Weight of amplified DNA</TH></TR></THEAD><TBODY><TR><TD>240</TD><TD>(SH3)<SUB>4</SUB>Clones C4.1</TD><TD>43 - 44</TD><TD>CusR_iProm-upCusR_iProm-down</TD><TD>932 bp</TD></TR><TR><TD>240</TD><TD>(SH3)<SUB>4</SUB>Clones C4.2</TD><TD>43 - 44</TD><TD>CusR_iProm-upCusR_iProm-down</TD><TD>932 bp</TD></TR><TR><TD>240</TD><TD>(SH3)<SUB>4</SUB>Clones C4.3</TD><TD>43 - 44</TD><TD>CusR_iProm-upCusR_iProm-down</TD><TD>932 bp</TD></TR><TR><TD>240</TD><TD>(SH3)<SUB>4</SUB>Clones C4.4</TD><TD>43 - 44</TD><TD>CusR_iProm-upCusR_iProm-down</TD><TD>932 bp</TD></TR><TR><TD>240</TD><TD>(SH3)<SUB>4</SUB>Clones C4.5</TD><TD>43 - 44</TD><TD>CusR_iProm-upCusR_iProm-down</TD><TD>932 bp</TD></TR><TR><TD>240</TD><TD>(SH3)<SUB>4</SUB>Clones C5.1</TD><TD>43 - 44</TD><TD>CusR_iProm-upCusR_iProm-down</TD><TD>932 bp</TD></TR><TR><TD>240</TD><TD>(SH3)<SUB>4</SUB>Clones C5.2</TD><TD>43 - 44</TD><TD>CusR_iProm-upCusR_iProm-down</TD><TD>932 bp</TD></TR><TR><TD>240</TD><TD>(SH3)<SUB>4</SUB>Clones C5.3</TD><TD>43 - 44</TD><TD>CusR_iProm-upCusR_iProm-down</TD><TD>932 bp</TD></TR><TR><TD>240</TD><TD>(SH3)<SUB>4</SUB>Clones C5.4</TD><TD>43 - 44</TD><TD>CusR_iProm-upCusR_iProm-down</TD><TD>932 bp</TD></TR><TR><TD>240</TD><TD>(SH3)<SUB>4</SUB>Clones C5.5</TD><TD>43 - 44</TD><TD>CusR_iProm-upCusR_iProm-down</TD><TD>932 bp</TD></TR><TR><TD>240</TD><TD>(SH3)<SUB>4</SUB>Clones C6.1</TD><TD>43 - 44</TD><TD>CusR_iProm-upCusR_iProm-down</TD><TD>932 bp</TD></TR><TR><TD>240</TD><TD>(SH3)<SUB>4</SUB>Clones C6.2</TD><TD>43 - 44</TD><TD>CusR_iProm-upCusR_iProm-down</TD><TD>932 bp</TD></TR><TR><TD>240</TD><TD>(SH3)<SUB>4</SUB>Clones C6.3</TD><TD>43 - 44</TD><TD>CusR_iProm-upCusR_iProm-down</TD><TD>932 bp</TD></TR><TR><TD>240</TD><TD>(SH3)<SUB>4</SUB>Clones C6.4</TD><TD>43 - 44</TD><TD>CusR_iProm-upCusR_iProm-down</TD><TD>932 bp</TD></TR><TR><TD>240</TD><TD>(SH3)<SUB>4</SUB>Clones C6.5</TD><TD>43 - 44</TD><TD>CusR_iProm-upCusR_iProm-down</TD><TD>932 bp</TD></TR></TBODY></TABLE><P>The efficiency of the amplification was tested thanks to agarose gel electrophoresis. Results are presented below.</P><DIV class="media notes-media"><DIV class="media-body"><P>No clone of (SH3)4 seemed to present the right construction.</P></DIV></DIV></LI><LI><P>Several control PCR were realized to verify whether stability label K1051206 and flag cloning worked. Taq polymerase was used.</P><TABLE class="table table-hover table-bordered notes-table"><THEAD><TR><TH>#</TH><TH>DNA Template</TH><TH>Primers Numbers</TH><TH>Primers Names</TH><TH>Weight of amplified DNA</TH></TR></THEAD><TBODY><TR><TD>255</TD><TD>K1051206Clone St34</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>80 bp</TD></TR><TR><TD>256</TD><TD>K1051206Clone St35</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>80 bp</TD></TR><TR><TD>257</TD><TD>K1051206Clone St36</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>80 bp</TD></TR><TR><TD>258</TD><TD>K1051206Clone St37</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>80 bp</TD></TR><TR><TD>259</TD><TD>K1051206Clone St39</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>80 bp</TD></TR><TR><TD>260</TD><TD>FlagClone 1</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>80 bp</TD></TR><TR><TD>261</TD><TD>FlagClone 2</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>80 bp</TD></TR><TR><TD>262</TD><TD>FlagClone 3</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>80 bp</TD></TR><TR><TD>263</TD><TD>FlagClone 4</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>80 bp</TD></TR><TR><TD>264</TD><TD>FlagClone 5</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>80 bp</TD></TR><TR><TD>265</TD><TD>FlagClone 6</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>86 bp</TD></TR><TR><TD>266</TD><TD>FlagClone 7</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>86 bp</TD></TR><TR><TD>267</TD><TD>FlagClone 8</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>86 bp</TD></TR><TR><TD>268</TD><TD>FlagClone 9</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>86 bp</TD></TR><TR><TD>269</TD><TD>FlagClone 10</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>86 bp</TD></TR></TBODY></TABLE><P>The efficiency of the amplification was tested thanks to agarose gel electrophoresis. Results are presented below.</P><DIV class="media notes-media"><DIV class="media-body"><P>No clone of K1051206 seemed to be correctly amplified. Nevertheless, every Flag clone seemed to present the right construction. Fl1 and Fl3 clones were chosen to go on brick construction.</P></DIV></DIV></LI><LI><P>We made a gradient PCR 55° to 65° with oligo sdaBC_checkdo/FRTup and W3110 sdaBC- strain and W3110 Wild type.</P><P>No amplification shown by the electrophoresis for all these PCR.</P><P>Results not shown</P></LI></DIV><DIV class="tab-pane fade in" id="0815"><UL class="list-notebook"><LI><P>Several plasmids were purified thanks to Macherey Nagel MiniPrep Kit :</P></LI><LI>Stability label 27/28 K1051206 clone St8 ⇒ PiGEM_02.49</LI><LI>Stability label 29/30 K1051207 clones St16, 17, 19, 20 and 21 ⇒ PiGEM_02.50, PiGEM_02.51, PiGEM_02.52, PiGEM_02.53 and PiGEM_02.54</LI><LI>Stability label 31/32 K1051208 clones St24, 26, 27, 28, 29 and 33 ⇒ PiGEM_02.55, PiGEM_02.56, PiGEM_02.57, PiGEM_02.58, PiGEM_02.59 and PiGEM_02.60</LI></UL><LI><P>We made Q5 PCR with oligo 1-2 and 3-4 on pKD4 template for prepare new transformation sdaBC and cheA respectively #293 and #294. After we supress the template pKD4 with </P></LI><LI><P>We purified PCR products #293 thanks to Macherey Nagel PCR Clean up Kit. </P><TABLE class="table table-hover table-bordered notes-table"><TBODY><TR><TD>293</TD><TD>22.65 ng.mL<SUP>-1</SUP></TD></TR></TBODY></TABLE></LI><LI><P>We made 2 new PCR on transformed strains. With specific hybridation temperature.</P><TABLE class="table table-hover table-bordered notes-table"><THEAD><TR><TH>#</TH><TH>DNA Template</TH><TH>Primers Name</TH><TH>Weight of amplified DNA</TH><TH>Hybridation Temperature</TH></TR></THEAD><TBODY><TR><TD>300A</TD><TD>CheA-</TD><TD>CheA_check_upCheA_check_down</TD><TD>1500</TD><TD>55°C</TD></TR><TR><TD>301A</TD><TD>sdaBC- KanR</TD><TD>FRT_upsdaBC_check_down</TD><TD>1600</TD><TD>50°C</TD></TR></TBODY></TABLE></LI></DIV></DIV></DIV></DIV><DIV class="panel panel-info notes-panel"><DIV class="panel-heading"><H3 class="panel-title">Week 8 : 08/18/2014 ➡ 08/24/2014 </H3></DIV><DIV class="panel-body"><UL class="nav nav-tabs" role="tablist"><LI class="active"><A href="#0818" role="tab" data-toggle="tab" class="date-notebook">08/18/14</A></LI><LI><A href="#0819" role="tab" data-toggle="tab" class="date-notebook">08/19/14</A></LI><LI><A href="#0820" role="tab" data-toggle="tab" class="date-notebook">08/20/14</A></LI><LI><A href="#0821" role="tab" data-toggle="tab" class="date-notebook">08/21/14</A></LI><LI><A href="#0822" role="tab" data-toggle="tab" class="date-notebook">08/22/14</A></LI></UL><DIV class="tab-content"><DIV class="tab-pane fade in active" id="0818"><UL class="list-notebook"><LI><P>Plasmids containing the stability labels 27/28 K1051206 clone St8, the stability labels 29/30 K1051207 clones St16, 17, 19, 20 and 21 and the stability labels 31/32 K1051208 clones St24, 26, 27, 28, 29 and 33 were digested by XbaI to determine whether they presented the expected construction.</P><P>The efficiency of the digestion was tested thanks to agarose gel electrophoresis. Results are presented below.</P><DIV class="media notes-media"><DIV class="media-body"><P>The weight of digested fragment were too heavy. It was impossible that only one stability label was inserted in one iGEM plasmid. Certainly, several stability labels were associated in a same plasmid. The cloning of stability label must have be remade with new ratios of stability label and plasmid.</P></DIV></DIV></LI><LI><P>PiGEM_02.27 (pSB1A2, BBa_J04450) was digested with EcoRI and PstI to be used for PCR product 239 cloning. PCR product 239 (cusR promoter) was digested by EcoRI and PstI. Then, PCR product 239 was leagued in pSB1A2 digested. Tg1 competent cells were transformed  with these ligation products and spread on Amp dish.</P></LI><LI><P>Bricks 200 and 202 were built again by changing destination plasmid. 3A protocol was implemented to assemble the upstream part with downstream part and with the destination plasmid. Tg1 competent cells were transformed  with these ligation products and spread on Kan or Cm dish. The cloning plan is presented below.</P><TABLE class="table table-hover table-bordered notes-table"><THEAD><TR><TH>Part Number</TH><TH>Description</TH><TH>Upstream Part</TH><TH>Downstream Part</TH><TH>Destination Plasmid</TH></TR></THEAD><TBODY><TR><TD>200</TD><TD>Reference RBS with RFP with stop codon</TD><TD>PiGEM_02.29 Amp</TD><TD>PiGEM_01.04 Cm</TD><TD>PiGEM_02.28 Kan</TD></TR><TR><TD>202</TD><TD>Reference RBS with GFP with stop codon</TD><TD>PiGEM_02.29 Amp</TD><TD>PiGEM_02.03 Amp</TD><TD>PiGEM_01.13 Cm</TD></TR></TBODY></TABLE></LI></UL></DIV><DIV class="tab-pane fade in" id="0819"><UL class="list-notebook"><LI><P>Several control PCR were realized to verify whether cusR promoter cloning worked. Taq polymerase was used.</P><TABLE class="table table-hover table-bordered notes-table"><THEAD><TR><TH>#</TH><TH>DNA Template</TH><TH>Primers Numbers</TH><TH>Primers Names</TH><TH>Weight of amplified DNA</TH></TR></THEAD><TBODY><TR><TD>AS1</TD><TD>cusR promoterClone 1</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>120 bp</TD></TR><TR><TD>AS2</TD><TD>cusR promoterClone 2</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>120 bp</TD></TR><TR><TD>AS3</TD><TD>cusR promoterClone 3</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>120 bp</TD></TR><TR><TD>AS4</TD><TD>cusR promoterClone 4</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>120 bp</TD></TR><TR><TD>AS5</TD><TD>cusR promoterClone 5</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>120 bp</TD></TR><TR><TD>AS6</TD><TD>cusR promoterClone 6</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>120 bp</TD></TR><TR><TD>AS7</TD><TD>cusR promoterClone 7</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>120 bp</TD></TR><TR><TD>AS8</TD><TD>cusR promoterClone 8</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>120 bp</TD></TR><TR><TD>AS9</TD><TD>cusR promoterClone 9</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>120 bp</TD></TR><TR><TD>AS10</TD><TD>cusR promoterClone 10</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>120 bp</TD></TR></TBODY></TABLE><P>The efficiency of the amplification was tested thanks to agarose gel electrophoresis. Results are presented below.</P><DIV class="media notes-media"><DIV class="media-body"><P>The majority of clones seemed to present the right construction. Clones 2 and 8 (PCU2 and PCU8) were chosen to go on brick construction.</P></DIV></DIV></LI><LI><P>Bricks 201, 203 to 207, 211, 301 and 303 were built. 3A protocol was implemented to assemble the upstream part with downstream part and with the destination plasmid. Tg1 competent cells were transformed with these ligation products and spread on Kan or Cm dish. The cloning plan is presented below.</P><TABLE class="table table-hover table-bordered notes-table"><THEAD><TR><TH>Part Number</TH><TH>Description</TH><TH>Upstream Part</TH><TH>Downstream Part</TH><TH>Destination Plasmid</TH></TR></THEAD><TBODY><TR><TD>201</TD><TD>Reference RBS with RFP without stop codon</TD><TD>PiGEM_02.29 Amp</TD><TD>PiGEM_01.18 CmPiGEM_01.19 Cm</TD><TD>PiGEM_02.28 Kan</TD></TR><TR><TD>203</TD><TD>Reference RBS with GFP without stop codon</TD><TD>PiGEM_02.29 Amp</TD><TD>PiGEM_01.20 CmPiGEM_01.21 Cm</TD><TD>PiGEM_02.28 Kan</TD></TR><TR><TD>204</TD><TD>Reference RBS with RelA with stop codon</TD><TD>PiGEM_02.29 Amp</TD><TD>PiGEM_01.40 AmpPiGEM_01.41 Amp</TD><TD>PiGEM_02.13 Cm</TD></TR><TR><TD>205</TD><TD>Reference RBS with RelA without stop codon</TD><TD>PiGEM_02.29 Amp</TD><TD>PiGEM_01.43 AmpPiGEM_01.44 Amp</TD><TD>PiGEM_02.13 Cm</TD></TR><TR><TD>206</TD><TD>Reference RBS with Mesh1 with stop codon</TD><TD>PiGEM_02.29 Amp</TD><TD>PiGEM_01.48 Amp</TD><TD>PiGEM_02.13 Cm</TD></TR><TR><TD>207</TD><TD>Reference RBS with Mesh1 without stop codon</TD><TD>PiGEM_02.29 Amp</TD><TD>PiGEM_01.45 AmpPiGEM_01.46 Amp</TD><TD>PiGEM_02.13 Cm</TD></TR><TR><TD>211</TD><TD>Stability 3 with double terminator</TD><TD>PiGEM_02.29 Amp</TD><TD>PiGEM_01.33 AmpPiGEM_01.34 Amp</TD><TD>PiGEM_02.13 Cm</TD></TR><TR><TD>301</TD><TD>SH3pep-LZA with double terminator</TD><TD>PiGEM_02.35 AmpPiGEM_02.36 Amp</TD><TD>PiGEM_01.07 Cm</TD><TD>PiGEM_02.28 Kan</TD></TR><TR><TD>303</TD><TD>RBS with CusR-LZa protein</TD><TD>PiGEM_02.29 Amp</TD><TD>PiGEM_02.37 AmpPiGEM_02.38 Amp</TD><TD>PiGEM_01.13 Cm</TD></TR></TBODY></TABLE></LI><LI><P>PiGEM_02.27 (pSB1A2, BBa_J04450) was digested with XbaI, SpeI and PstI to be used for stability label cloning. Stability labels were assembled again by annealing.</P><TABLE class="table table-hover table-bordered notes-table"><THEAD><TR><TH>Primers numbers</TH><TH>Primers names</TH><TH>Brick created</TH></TR></THEAD><TBODY><TR><TD>27 - 28</TD><TD>BBa_K1051206-upBBa_K1051206-down</TD><TD>BBa_K1051206</TD></TR><TR><TD>29 -30</TD><TD>BBa_K1051207-upBBa_K1051207-down</TD><TD>BBa_K1051207</TD></TR><TR><TD>31 -32</TD><TD>BBa_K1051208-upBBa_K1051208-down</TD><TD>BBa_K1051208</TD></TR></TBODY></TABLE><P>Each annealing oligonucleotides were leagued in pSB1A2 digested XSP with ratio 1:1 (0.0047 ng of stability label for 117 ng of plasmid) and 1:2 (0.0094 ng of stability label for 117 ng of plasmid). Tg1 competent cells were transformed  with these ligation products and spread on Amp dish.</P></LI><LI><P>We made a PCR with new oligo to control mutants.</P><TABLE class="table table-hover table-bordered notes-table"><THEAD><TR><TH>#</TH><TH>DNA Template</TH><TH>Primers Name</TH><TH>Weight of amplified DNA</TH></TR></THEAD><TBODY><TR><TD>185</TD><TD>Genomic DNA of W3110</TD><TD>CheA_check_upCheA_check_down</TD><TD>1700</TD></TR><TR><TD>186</TD><TD>CheA-</TD><TD>CheA_check_upCheA_check_down</TD><TD>1500</TD></TR><TR><TD>187</TD><TD>Genomic DNA of W3110</TD><TD>sdaBC_check_upsdaBC_check_down</TD><TD>3100</TD></TR><TR><TD>188</TD><TD>sdaBC- KanR</TD><TD>sdaBC_check_upsdaBC_check_down</TD><TD>1500</TD></TR><TR><TD>189</TD><TD>sdaBC- KanR + pCP20</TD><TD>sdaBC_check_upsdaBC_check_down</TD><TD>1500</TD></TR><TR><TD>190</TD><TD>Genomic DNA of W3110</TD><TD>FRT_upsdaBC_check_down</TD><TD>0</TD></TR><TR><TD>191</TD><TD>sdaBC- KanR</TD><TD>FRT_upsdaBC_check_down</TD><TD>1600</TD></TR><TR><TD>192</TD><TD>sdaBC- KanR + pCP20</TD><TD>FRT_upsdaBC_check_down</TD><TD>1600</TD></TR></TBODY></TABLE><P>After the electrophoresis, we can confirm The strain sdaBC- KanR is successful. We select 3 strains with electrophoresis control</P><P>C.23 and C.33 are contaminated. So we select C.43, C.53 and C.63.</P></LI></UL></DIV><DIV class="tab-pane fade in" id="0820"><UL class="list-notebook"><LI><P>Several control PCR were realized to verify whether Bricks 200 and 202 constructions worked. Taq polymerase was used.</P><TABLE class="table table-hover table-bordered notes-table"><THEAD><TR><TH>#</TH><TH>DNA Template</TH><TH>Primers Numbers</TH><TH>Primers Names</TH><TH>Weight of amplified DNA</TH></TR></THEAD><TBODY><TR><TD>AS11</TD><TD>Brick 200Clone RFP1</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>758 bp</TD></TR><TR><TD>AS12</TD><TD>Brick 200Clone RFP2</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>758 bp</TD></TR><TR><TD>AS13</TD><TD>Brick 200Clone RFP3</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>758 bp</TD></TR><TR><TD>AS14</TD><TD>Brick 200Clone RFP4</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>758 bp</TD></TR><TR><TD>AS15</TD><TD>Brick 200Clone RFP5</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>758 bp</TD></TR><TR><TD>AS16</TD><TD>Brick 202Clone GFP1</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>772 bp</TD></TR><TR><TD>AS17</TD><TD>Brick 202Clone GFP2</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>772 bp</TD></TR><TR><TD>AS18</TD><TD>Brick 202Clone GFP3</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>772 bp</TD></TR><TR><TD>AS19</TD><TD>Brick 202Clone GFP4</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>772 bp</TD></TR><TR><TD>AS20</TD><TD>Brick 202Clone GFP5</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>772 bp</TD></TR><TR><TD>AS21</TD><TD>Brick 200Clone RFP1</TD><TD>38 - 51</TD><TD>BBa_E1010_NoStop-downBBa_B0034-up</TD><TD>718 bp</TD></TR><TR><TD>AS22</TD><TD>Brick 200Clone RFP2</TD><TD>38 - 51</TD><TD>BBa_E1010_NoStop-downBBa_B0034-up</TD><TD>718 bp</TD></TR><TR><TD>AS23</TD><TD>Brick 200Clone RFP3</TD><TD>38 - 51</TD><TD>BBa_E1010_NoStop-downBBa_B0034-up</TD><TD>718 bp</TD></TR><TR><TD>AS24</TD><TD>Brick 200Clone RFP4</TD><TD>38 - 51</TD><TD>BBa_E1010_NoStop-downBBa_B0034-up</TD><TD>718 bp</TD></TR><TR><TD>AS25</TD><TD>Brick 200Clone RFP5</TD><TD>38 - 51</TD><TD>BBa_E1010_NoStop-downBBa_B0034-up</TD><TD>718 bp</TD></TR><TR><TD>AS26</TD><TD>Brick 202Clone GFP1</TD><TD>40 - 51</TD><TD>BBa_E0040_NoStop-downBBa_B0034-up</TD><TD>732 bp</TD></TR><TR><TD>AS27</TD><TD>Brick 202Clone GFP2</TD><TD>40 - 51</TD><TD>BBa_E0040_NoStop-downBBa_B0034-up</TD><TD>732 bp</TD></TR><TR><TD>AS28</TD><TD>Brick 202Clone GFP3</TD><TD>40 - 51</TD><TD>BBa_E0040_NoStop-downBBa_B0034-up</TD><TD>732 bp</TD></TR><TR><TD>AS29</TD><TD>Brick 202Clone GFP4</TD><TD>40 - 51</TD><TD>BBa_E0040_NoStop-downBBa_B0034-up</TD><TD>732 bp</TD></TR><TR><TD>AS30</TD><TD>Brick 202Clone GFP5</TD><TD>40 - 51</TD><TD>BBa_E0040_NoStop-downBBa_B0034-up</TD><TD>732 bp</TD></TR></TBODY></TABLE><P>The efficiency of the amplification was tested thanks to agarose gel electrophoresis. Results are presented below.</P><P>All clones seemed to present the right construction. Clones 200(3), 200(5), 202(1) and 202(3) were chosen to go on brick construction.</P></LI><LI><P>Several plasmids were purified thanks to Macherey Nagel MiniPrep Kit :</P></LI><LI>CusR promoter clones 2 and 8 (PCU2 and PCU8) ⇒ PiGEM_02.61 and PiGEM_02.62</LI></UL><LI><P>We prepare seven starter of W3110 sdaBC- KanR strain with Kan50.</P></LI></DIV><DIV class="tab-pane fade in" id="0821"><UL class="list-notebook"><LI><P>Sequencing results of plasmids PiGEM_02.33, PiGEM_02.34, PiGEM_02.35, PiGEM_02.36, PiGEM_02.37, PiGEM_02.38, PiGEM_02.39, PiGEM_02.40, PiGEM_02.41, PiGEM_02.42, PiGEM_02.43, PiGEM_02.44, PiGEM_02.45, PiGEM_02.46, PiGEM_02.47, PiGEM_02.48 were received. Thus, clones CB1, SP10, CZ3, CA38, RA2, MS8, MSS4, GFP47, RFP43, RAS5 and PCU2 had the right construction. Because no clone of SLIC SerA was correct, several control PCR were realized to verify whether other clones of SLIC SerA had the right construction. Taq polymerase was used. </P><TABLE class="table table-hover table-bordered notes-table"><THEAD><TR><TH>#</TH><TH>DNA Template</TH><TH>Primers Numbers</TH><TH>Primers Names</TH><TH>Weight of amplified DNA</TH></TR></THEAD><TBODY><TR><TD>AS31</TD><TD>SLIC bis SerASA9</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>1049 bp</TD></TR><TR><TD>AS32</TD><TD>SLIC bis SerASA11</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>1049 bp</TD></TR><TR><TD>AS33</TD><TD>SLIC bis SerASA12</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>1049 bp</TD></TR><TR><TD>AS34</TD><TD>SLIC bis SerASA13</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>1049 bp</TD></TR><TR><TD>AS35</TD><TD>SLIC bis SerASA14</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>1049 bp</TD></TR><TR><TD>AS36</TD><TD>SLIC bis SerASA15</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>1049 bp</TD></TR></TBODY></TABLE><P>The efficiency of the amplification was tested thanks to agarose gel electrophoresis. Results are presented below.</P><DIV class="media notes-media"><DIV class="media-body"><P>There were bands at the right molecular weight but clone SA1 got the same band (AS36). Nevertheless, this clone did not have the right construction (bad sequencing result). Thus, SerA was going to be assembled again by another protocol (Gibson protocol).</P></DIV></DIV></LI><LI><P>Plasmids containing the flag (Fl1 and Fl3) were digested by XbaI to determine whether they presented the expected construction.</P><P>The efficiency of the digestion was tested thanks to agarose gel electrophoresis. Results are presented below.</P><DIV class="media notes-media"><DIV class="media-body"><P>Clone Fl1 and Fl3 seemed to present the expected construction.</P></DIV></DIV></LI><LI><P>We realize day 7 of Lambda Red protocol on W3110 sdaBC- pCP20 strain.</P></LI></UL></DIV><DIV class="tab-pane fade in" id="0822"><UL class="list-notebook"><LI><P>Several plasmids were purified thanks to Macherey Nagel MiniPrep Kit :</P></LI><LI>Bricks 200 (3) and 200 (5) ⇒ PiGEM_02.65 and PiGEM_02.66</LI><LI>Bricks 202 (1) and 202 (3) ⇒ PiGEM_02.67 and PiGEM_02.68</LI></UL><LI><P>Unfortunately, colonies careering bricks 200 and 202 were respectively red and green while constructions of these bricks were not supposed to contain any promoter. Thus, PiGEM_02.65, PiGEM_02.66, PiGEM_02.67 and PiGEM_02.68 were sent in order to be sequenced. In the mean time, RFP and GFP were assembled to another RBS to test whether the RBS B0034 was correct. The cloning plan is presented below.</P><TABLE class="table table-hover table-bordered notes-table"><THEAD><TR><TH>Part Number</TH><TH>Description</TH><TH>Upstream Part</TH><TH>Downstream Part</TH><TH>Destination Plasmid</TH></TR></THEAD><TBODY><TR><TD>A</TD><TD>RBS B0030 with RFP with stop codon</TD><TD>PiGEM_01.05 Cm</TD><TD>PiGEM_01.04 Cm</TD><TD>PiGEM_01.13 Cm</TD></TR><TR><TD>B</TD><TD>RBS B0030 with GFP with stop codon</TD><TD>PiGEM_01.05 Cm</TD><TD>PiGEM_02.03 Cm</TD><TD>PiGEM_01.13 Cm</TD></TR></TBODY></TABLE></LI><LI><P>Several control PCR were realized to verify whether stability labels cloning with different ratios worked. Taq polymerase was used.</P><TABLE class="table table-hover table-bordered notes-table"><THEAD><TR><TH>#</TH><TH>DNA Template</TH><TH>Primers Numbers</TH><TH>Primers Names</TH><TH>Weight of amplified DNA</TH></TR></THEAD><TBODY><TR><TD>CB1</TD><TD>K1051206 Ratio 1:1Clone St40</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>80 bp</TD></TR><TR><TD>CB2</TD><TD>K1051206 Ratio 1:1Clone St41</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>80 bp</TD></TR><TR><TD>CB3</TD><TD>K1051206 Ratio 1:1Clone St42</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>80 bp</TD></TR><TR><TD>CB4</TD><TD>K1051206 Ratio 1:1Clone St43</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>80 bp</TD></TR><TR><TD>CB5</TD><TD>K1051206 Ratio 1:1Clone St44</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>80 bp</TD></TR><TR><TD>CB6</TD><TD>K1051206 Ratio 1:2Clone St45</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>80 bp</TD></TR><TR><TD>CB7</TD><TD>K1051206 Ratio 1:2Clone St46</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>80 bp</TD></TR><TR><TD>C86</TD><TD>K1051206 Ratio 1:2Clone St47</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>80 bp</TD></TR><TR><TD>CB9</TD><TD>K1051206 Ratio 1:2Clone St48</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>80 bp</TD></TR><TR><TD>CB10</TD><TD>K1051206 Ratio 1:2Clone St49</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>80 bp</TD></TR><TR><TD>CB11</TD><TD>K1051207 Ratio 1:1Clone St50</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>86 bp</TD></TR><TR><TD>CB12</TD><TD>K1051207 Ratio 1:1Clone St51</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>86 bp</TD></TR><TR><TD>CB13</TD><TD>K1051207 Ratio 1:1Clone St52</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>86 bp</TD></TR><TR><TD>CB14</TD><TD>K1051207 Ratio 1:1Clone St53</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>86 bp</TD></TR><TR><TD>CB15</TD><TD>K1051207 Ratio 1:1Clone St54</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>86 bp</TD></TR><TR><TD>CB16</TD><TD>K1051207 Ratio 1:2Clone St55</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>86 bp</TD></TR><TR><TD>CB17</TD><TD>K1051207 Ratio 1:2Clone St56</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>86 bp</TD></TR><TR><TD>CB18</TD><TD>K1051207 Ratio 1:2Clone St57</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>86 bp</TD></TR><TR><TD>CB19</TD><TD>K1051207 Ratio 1:2Clone St58</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>86 bp</TD></TR><TR><TD>CB20</TD><TD>K1051207 Ratio 1:2Clone St59</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>86 bp</TD></TR><TR><TD>CB21</TD><TD>K1051208 Ratio 1:1Clone St60</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>80 bp</TD></TR><TR><TD>CB22</TD><TD>K1051208 Ratio 1:1Clone St61</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>80 bp</TD></TR><TR><TD>CB23</TD><TD>K1051208 Ratio 1:1Clone St62</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>80 bp</TD></TR><TR><TD>CB24</TD><TD>K1051208 Ratio 1:1Clone St63</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>80 bp</TD></TR><TR><TD>CB25</TD><TD>K1051208 Ratio 1:1Clone St64</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>80 bp</TD></TR><TR><TD>CB26</TD><TD>K1051208 Ratio 1:2Clone St65</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>80 bp</TD></TR><TR><TD>CB27</TD><TD>K1051208 Ratio 1:2Clone St66</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>80 bp</TD></TR><TR><TD>CB28</TD><TD>K1051208 Ratio 1:2Clone St67</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>80 bp</TD></TR><TR><TD>CB29</TD><TD>K1051208 Ratio 1:2Clone St68</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>80 bp</TD></TR><TR><TD>CB30</TD><TD>K1051208 Ratio 1:2Clone St69</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>80 bp</TD></TR></TBODY></TABLE><P>The efficiency of the amplification was tested thanks to agarose gel electrophoresis. Results are presented below.</P><DIV class="media notes-media"><DIV class="media-body"><P>Clone Fl1 and Fl3 seemed to present the expected construction.</P></DIV></DIV><P>Unfortunately, there was not any expected result. Stability labels constructions did not success once again. New oligonucleotides were going to be ordered to try a new protocol.</P></LI><LI><P>We realize day 8 of Lambda Red protocol on W3110 sdaBC- strain.</P></LI></DIV></DIV></DIV></DIV><DIV class="panel panel-info notes-panel last-note"><DIV class="panel-heading"><H3 class="panel-title">Week 9 : 08/25/2014 ➡ 08/31/2014 </H3></DIV><DIV class="panel-body"><UL class="nav nav-tabs" role="tablist"><LI class="active"><A href="#0825" role="tab" data-toggle="tab" class="date-notebook">08/25/14</A></LI><LI><A href="#0826" role="tab" data-toggle="tab" class="date-notebook">08/26/14</A></LI><LI><A href="#0827" role="tab" data-toggle="tab" class="date-notebook">08/27/14</A></LI><LI><A href="#0828" role="tab" data-toggle="tab" class="date-notebook">08/28/14</A></LI></UL><DIV class="tab-content"><DIV class="tab-pane fade in active" id="0825"><UL class="list-notebook"><LI><P>RelA (RA2) and Mesh1 (MS8) were assembled with a Lac promoter and the reference RBS (BBa_B0034) to characterize parts. 3A protocol was implemented to assemble the upstream part with downstream part and with the destination plasmid. Tg1 competent cells were transformed  with these ligation products and spread on Kan glucose dish to prevent RelA and Mesh1 expression that could slow cells growth. The cloning plan is presented below.</P><TABLE class="table table-hover table-bordered notes-table"><THEAD><TR><TH>Part Number</TH><TH>Description</TH><TH>Upstream Part</TH><TH>Downstream Part</TH><TH>Destination Plasmid</TH></TR></THEAD><TBODY><TR><TD>x</TD><TD>Promoter with RBS and RelA with stop codon</TD><TD>PiGEM_01.08 Cm</TD><TD>PiGEM_02.41 Amp</TD><TD>PiGEM_02.28 Kan</TD></TR><TR><TD>y</TD><TD>Promoter with RBS and Mesh1 with stop codon</TD><TD>PiGEM_01.08 Cm</TD><TD>PiGEM_02.48 Amp</TD><TD>PiGEM_02.28 Kan</TD></TR></TBODY></TABLE></LI><LI><P>We realize day 9 of Lambda Red protocol on W3110 sdaBC- strain, and day 2 of Lambda Red protocol on W3110 sdaBC- strain with #294 PCR product</P></LI></UL></DIV><DIV class="tab-pane fade in" id="0826"><UL class="list-notebook"><LI><P>Sequencing results of plasmids PiGEM_02.65, PiGEM_02.66, PiGEM_02.67 and PiGEM_02.68 were received. PiGEM_02.67 and PiGEM_02.68 had the right construction but PiGEM_02.65 and PiGEM_02.66 had not. Thus, RBS BBa_B0034 was correct. There was no reason for the colonies containing bricks 200 and 202 to be green or red without promoter sequence.</P></LI><LI><P>Several control PCR were realized to verify whether RelA and Mesh1 cloning with their promoter and RBS worked. (SH3)2 new construction were also tested. Taq polymerase was used.</P><TABLE class="table table-hover table-bordered notes-table"><THEAD><TR><TH>#</TH><TH>DNA Template</TH><TH>Primers Numbers</TH><TH>Primers Names</TH><TH>Weight of amplified DNA</TH></TR></THEAD><TBODY><TR><TD>AS43</TD><TD>BBa_J04500 with RelAClone 1</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>2480 bp</TD></TR><TR><TD>AS44</TD><TD>BBa_J04500 with RelAClone 2</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>2480 bp</TD></TR><TR><TD>AS45</TD><TD>BBa_J04500 with RelAClone 3</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>2480 bp</TD></TR><TR><TD>AS46</TD><TD>BBa_J04500 with RelAClone 4</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>2480 bp</TD></TR><TR><TD>AS47</TD><TD>BBa_J04500 with RelAClone 5</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>2480 bp</TD></TR><TR><TD>AS48</TD><TD>BBa_J04500 with RelAClone 6</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>2480 bp</TD></TR><TR><TD>AS49</TD><TD>BBa_J04500 with RelAClone 7</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>2480 bp</TD></TR><TR><TD>AS50</TD><TD>BBa_J04500 with RelAClone 8</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>2480 bp</TD></TR><TR><TD>AS51</TD><TD>BBa_J04500 with RelAClone 9</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>2480 bp</TD></TR><TR><TD>AS52</TD><TD>BBa_J04500 with RelAClone 10</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>2480 bp</TD></TR><TR><TD>AS53</TD><TD>BBa_J04500 with Mesh1Clone 1</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>779 bp</TD></TR><TR><TD>AS54</TD><TD>BBa_J04500 with Mesh1Clone 2</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>779 bp</TD></TR><TR><TD>AS55</TD><TD>BBa_J04500 with Mesh1Clone 3</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>779 bp</TD></TR><TR><TD>AS56</TD><TD>BBa_J04500 with Mesh1Clone 4</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>779 bp</TD></TR><TR><TD>AS57</TD><TD>BBa_J04500 with Mesh1Clone 5</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>779 bp</TD></TR><TR><TD>AS58</TD><TD>BBa_J04500 with Mesh1Clone 6</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>779 bp</TD></TR><TR><TD>AS59</TD><TD>BBa_J04500 with Mesh1Clone 7</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>779 bp</TD></TR><TR><TD>AS60</TD><TD>BBa_J04500 with Mesh1Clone 8</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>779 bp</TD></TR><TR><TD>AS61</TD><TD>BBa_J04500 with Mesh1Clone 9</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>779 bp</TD></TR><TR><TD>AS62</TD><TD>BBa_J04500 with Mesh1Clone 10</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>779 bp</TD></TR><TR><TD>AS63</TD><TD>(SH3)<SUB>2</SUB>Clone 1</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>500 bp</TD></TR><TR><TD>AS64</TD><TD>(SH3)<SUB>2</SUB>Clone 2</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>500 bp</TD></TR><TR><TD>AS65</TD><TD>(SH3)<SUB>2</SUB>Clone 3</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>500 bp</TD></TR><TR><TD>AS66</TD><TD>(SH3)<SUB>2</SUB>Clone 4</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>500 bp</TD></TR><TR><TD>AS61</TD><TD>(SH3)<SUB>2</SUB>Clone 5</TD><TD>43 - 44</TD><TD>Prefix_upSuffix_down</TD><TD>500 bp</TD></TR></TBODY></TABLE><P>The efficiency of the amplification was tested thanks to agarose gel electrophoresis. Results are presented below.</P><P>Some clones seemed to present the right construction. Clones 1 and 2 of BBa_J04500 with RelA were chosen to continue the part characterization. In the mean time, to go on the part characterization, clones 2, 5, 6 and 8 of BBa_J04500 with Mesh1 were selected. Clones 2 and 5 were going to be used as negative controls (empty plasmids) and clones 6 and 8 were going to be used as functional parts.</P></LI><LI><P>We realize day 3 of Lambda Red protocol on W3110 sdaBC- strain CheA- KanR.</P></LI><LI><P>Check the construction 200(2) (RBS B0034 + RFP E1010) by digestion.</P><P>7µL E1010 (control) (500ng)36µL H2O5µ NEB Buffer 2.11µL XbaI1µL EcoRI</P><P>2µL 200(2) (500ng)36µL H2O5µ NEB Buffer 2.11µL XbaI1µL EcoRI</P><P>Incubation 53' at 37°C. Then, inactivation of enzymes 25' at 80°C.</P><DIV class="media notes-media"><DIV class="media-body" style="padding-left: 10px;"><P>It is difficult to see if 200(2) possesses the RBS. The plasmid 200(2) is sent to sequencing and it's OK.</P></DIV></DIV></LI><LI><P>Clean-up of PCR product number 82 and 83 from 8-8-14 (50µL)</P></LI><LI><P>SOE reaction (= overlap PCR):</P><P>
                      Mix:
                      </P></LI><LI>PCR product 82= 3µL</LI><LI>PCR product 83 = 3µL</LI><LI>dNTP 10mM = 2µL</LI><LI>Q5 buffer = 8µL</LI><LI>Q5 polymerase = 0,5µL</LI><LI>H2O qsp 40µL</LI></UL><P>
                      Program:
                      <OL><LI>98°C 2’</LI><LI>98°C 20”</LI><LI>45°C 25”</LI><LI>72°C 1’</LI><LI>72°C 10’</LI></OL>
                      Repeat the steps 3-4-5, 10 times.
                    </P></DIV><DIV class="tab-pane fade in" id="0827"><UL class="list-notebook"><LI><P>Two new cells strain were selected to characterize RelA and Mesh1 parts. The first one is EB425 (ppGpp°) which does not produce ppGpp any more. The second one is EB544 (ppGpp<SUP>++</SUP>) which produces ppGpp with greater stability. These Escherichia coli strains were used to prepare two samples of competent cells. EB425 were going to be complemented by RelA whereas EB544 were going to be complemented by Mesh1.</P></LI><LI><P>We realize day 4 of Lambda Red protocol on  W3110 sdaBC- strain CheA- KanR. strain.</P></LI><LI><P>PCR2: Second PCR round to amplification of serA from the SOE product (26-08-14) with primers 5 and 6, using the iGEM PCR protocol with Q5 polymerase. The temperature of annealing is 60°C</P><DIV class="media notes-media"><DIV class="media-body" style="padding-left: 10px;"><P>Result: the amplification of serA is good. Clean-up of serA (PCR2) and elution with 45µL of water.</P></DIV></DIV></LI><LI><P>Digestion of pSB1A3 and pSB1C3 during 1h30 at 37°C. Then, inactivation of enzymes during 20’ at 80°C</P><P>
                      Mix:
                      </P></LI><LI>10µL of plasmid</LI><LI>1µL EcoRI</LI><LI>1µL PstI</LI><LI>5µL Buffer 2.1</LI><LI>33 µL H2O</LI></UL><LI><P>Digestion of the insert (serA) during 1h at 37°C. Then inactivation of enzymes during 20’ at 80°C</P><P>
                      Mix:
                      </P></LI><LI>20µL of serA</LI><LI>1µL EcoRI</LI><LI>1µL PstI</LI><LI>5µL Buffer 2.1</LI><LI>13 µL H2O</LI><LI><P>NEB ligation:</P></LI><LI>2µL of digested plasmid </LI><LI>2µL of digested serA insert</LI><LI>2µL ligase buffer</LI><LI>1µL ligase (NEB)</LI><LI>13 µL H2O</LI><P>Incubation overnight at 16°C</P></DIV><DIV class="tab-pane fade in" id="0828"><UL class="list-notebook"><LI><P>Several plasmids were purified thanks to Macherey Nagel MiniPrep Kit :</P></LI><LI>J04500 with RelA (RA2) Clones 1 and 2 ⇒ PiGEM_02.69 and PiGEM_02.70</LI><LI>J04500 Mesh1 (MS8) Clones 2 and 5 ⇒ PiGEM_02.71 and PiGEM_02.72</LI><LI>Empty plasmids Clones 6 and 8 ⇒ PiGEM_02.73 and PiGEM_02.74</LI></UL><LI><P>To characterize RelA and Mesh1 parts, growth rate would be evaluated. At first, EB425 competent cells (ppGpp°) were transformed with PiGEM_02.05 (pT18 RelA, positive control), with PiGEM_02.74 (empty plasmid, negative control) and with PiGEM_02.69 and PiGEM_02.70 (pLac RelA, built part). Then, EB544 competent cells (ppGpp<SUP>++</SUP>) were transformed with PiGEM_02.06 (pBAD Mesh1, positive control), with PiGEM_02.74 (empty plasmid, negative control) and with PiGEM_02.71 and PiGEM_02.72 (pLac Mesh1, built part). Cells were spread over Kan glucose dish to prevent RelA and Mesh1 expression. Some colonies were going to be spread later on selective dish to observe any difference in growth rate.</P></LI><LI><P>We realize day 5 of Lambda Red protocol on W3110 sdaBC- CheA- KanR strain. </P></LI><LI><P>Transformation of 90µL of DH5α super competent cells with 10µL of ligation (pSB1C3-SerA or pSB1A2-SerA). The selective medium is LB + Ampicillin (Ap) 100µg/mL or LB + Chloramphenicol (Cm) 30µg/mL</P><DIV class="media notes-media"><DIV class="media-body" style="padding-left: 10px;"><P>pSB1C3-SerA-1 seemed good. It was sent to sequencing. It’s OK</P></DIV></DIV></LI></DIV></DIV></DIV></DIV></DIV><DIV class="col-md-3" id="sidebar-notebook-div"><DIV class="list-group" id="sidebar-notebook"><A data-scroll="" href="#week5" class="list-group-item"><H4 class="list-group-item-heading">Week 5
              <SMALL style="padding-left:20px">07/28/14 → 08/03/14</SMALL></H4></A><A data-scroll="" href="#week6" class="list-group-item"><H4 class="list-group-item-heading">Week 6
              <SMALL style="padding-left:20px">08/04/14 → 08/10/14</SMALL></H4></A><A data-scroll="" href="#week7" class="list-group-item"><H4 class="list-group-item-heading">Week 7
              <SMALL style="padding-left:20px">08/11/14 → 08/17/14</SMALL></H4></A><A data-scroll="" href="#week8" class="list-group-item"><H4 class="list-group-item-heading">Week 8
              <SMALL style="padding-left:20px">08/18/14 → 08/24/14</SMALL></H4></A><A data-scroll="" href="#week9" class="list-group-item"><H4 class="list-group-item-heading">Week 9
              <SMALL style="padding-left:20px">08/25/14 → 08/31/14</SMALL></H4></A></DIV></DIV></DIV></DIV><DIV class="footer"><DIV class="container"><DIV class="row"><DIV class="col-md-4"><H4 class="text-info">Contact us</H4><DIV class="text-warning"><P><B>Email : </B><A href="mailto:equipe.igem.amu@gmail.com">equipe.igem.amu@gmail.com</A></P></DIV></DIV><DIV class="col-md-4"><H4 class="text-info">Follow us</H4><DIV class="text-warning"><P><A href="https://twitter.com/iGEM_AMU"> iGEM_AMU</A><A href="https://www.facebook.com/iGEM.AMU">iGEM.AMU</A></P></DIV></DIV><DIV class="col-md-4"><H4 class="text-info">Our sponsors</H4></DIV></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Aix-Marseille/Notebook_08">http://2014.igem.org/Team:Aix-Marseille/Notebook_08</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Aix-Marseille/Notebook_08" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Aix-Marseille/Notebook_08" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Aix-Marseille/Notebook_08&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Aix-Marseille/Notebook_08&amp;oldid=219237" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>