<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_NEU_China_A_Demonstrate skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:NEU China A/Demonstrate</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="igem_2018_team_menu displaying_menu"><A href="https://2018.igem.org/Team:NEU_China_A"><DIV class="menu_item direct_link">
				HOME
			</DIV></A><DIV class="menu_item">
			TEAM
		</DIV><DIV class="submenu"><A href="https://2018.igem.org/Team:NEU_China_A/Team"><DIV class="submenu_item">
					Team Members
				</DIV></A><A href="https://2018.igem.org/Team:NEU_China_A/Collaborations"><DIV class="submenu_item">
					Collaborations
				</DIV></A></DIV><DIV class="menu_item">
			PROJECT
		</DIV><DIV class="submenu"><A href="https://2018.igem.org/Team:NEU_China_A/Description"><DIV class="submenu_item">
					Description
				</DIV></A><A href="https://2018.igem.org/Team:NEU_China_A/Design"><DIV class="submenu_item">
					Design
				</DIV></A><A href="https://2018.igem.org/Team:NEU_China_A/Experiments"><DIV class="submenu_item">
					Experiments
				</DIV></A><A href="https://2018.igem.org/Team:NEU_China_A/Notebook"><DIV class="submenu_item">
					Notebook
				</DIV></A><A href="https://2018.igem.org/Team:NEU_China_A/InterLab"><DIV class="submenu_item">
					InterLab
				</DIV></A><A href="https://2018.igem.org/Team:NEU_China_A/Model"><DIV class="submenu_item">
					Model
				</DIV></A><A href="https://2018.igem.org/Team:NEU_China_A/Results"><DIV class="submenu_item">
					Results
				</DIV></A><A href="https://2018.igem.org/Team:NEU_China_A/Demonstrate"><DIV class="submenu_item">
					Demonstrate
				</DIV></A><A href="https://2018.igem.org/Team:NEU_China_A/Improve"><DIV class="submenu_item">
					Improve
				</DIV></A><A href="https://2018.igem.org/Team:NEU_China_A/Attributions"><DIV class="submenu_item">
					Attributions 
				</DIV></A></DIV><DIV class="menu_item">
			PARTS
		</DIV><DIV class="submenu"><A href="https://2018.igem.org/Team:NEU_China_A/Parts"><DIV class="submenu_item">
					Parts Overview
				</DIV></A><A href="https://2018.igem.org/Team:NEU_China_A/Basic_Part"><DIV class="submenu_item">
					Basic Parts
				</DIV></A><A href="https://2018.igem.org/Team:NEU_China_A/Composite_Part"><DIV class="submenu_item">
					Composite Parts
				</DIV></A><A href="https://2018.igem.org/Team:NEU_China_A/Part_Collection"><DIV class="submenu_item">
					Part Collection
				</DIV></A></DIV><A href="https://2018.igem.org/Team:NEU_China_A/Safety"><DIV class="menu_item direct_link">
				SAFETY
			</DIV></A><DIV class="menu_item"> 
			HUMAN PRACTICES 
		</DIV><DIV class="submenu"><A href="https://2018.igem.org/Team:NEU_China_A/Human_Practices"><DIV class="submenu_item">
					Human Practices
				</DIV></A><A href="https://2018.igem.org/Team:NEU_China_A/Public_Engagement"><DIV class="submenu_item">
					Education &amp; Engagement
				</DIV></A></DIV><DIV class="menu_item"> 
			AWARDS
		</DIV><DIV class="submenu"><A href="https://2018.igem.org/Team:NEU_China_A/Applied_Design"><DIV class="submenu_item">
					Applied Design
				</DIV></A><A href="https://2018.igem.org/Team:NEU_China_A/Entrepreneurship"><DIV class="submenu_item">
					Entrepreneurship
				</DIV></A><A href="https://2018.igem.org/Team:NEU_China_A/Hardware"><DIV class="submenu_item">
					Hardware
				</DIV></A><A href="https://2018.igem.org/Team:NEU_China_A/Measurement"><DIV class="submenu_item">
					Measurement
				</DIV></A><A href="https://2018.igem.org/Team:NEU_China_A/Model"><DIV class="submenu_item">
					Model
				</DIV></A><A href="https://2018.igem.org/Team:NEU_China_A/Plant"><DIV class="submenu_item">
					Plant
				</DIV></A><A href="https://2018.igem.org/Team:NEU_China_A/Software"><DIV class="submenu_item">
					Software
				</DIV></A></DIV><A href="https://igem.org/2018_Judging_Form?team=NEU_China_A"><DIV class="menu_item direct_link">
				JUDGING FORM ⇗ 
			</DIV></A></DIV><DIV class="igem_2018_team_content"><DIV class="igem_2018_team_column_wrapper"><HEADER><NAV class="blu"><DIV class="nav-wrapper"><A href="https://2018.igem.org/Team:NEU_China_A" class="brand-logo"> Home</A><A href="#" data-target="mobile-demo" class="sidenav-trigger"><I class="material-icons">menu</I></A><UL class="right hide-on-med-and-down"><LI><A data-target="project" class="dropdown-trigger" href="#!">
                        Project<I style="margin-left: 0" class="material-icons right">arrow_drop_down</I></A></LI><LI><A data-target="lab" class="dropdown-trigger" href="#!">
                        Lab<I style="margin-left: 0" class="material-icons right">arrow_drop_down</I></A></LI><LI><A data-target="parts" class="dropdown-trigger" href="#!">
                        Parts<I style="margin-left: 0" class="material-icons right">arrow_drop_down</I></A></LI><LI><A data-target="hp" class="dropdown-trigger" href="#!">
                        HP<I style="margin-left: 0" class="material-icons right">arrow_drop_down</I></A></LI><LI><A data-target="team" class="dropdown-trigger" href="#!">
                        Team<I style="margin-left: 0" class="material-icons right">arrow_drop_down</I></A></LI></UL></DIV></NAV><UL class="sidenav" id="mobile-demo"><UL class="collapsible" style="margin-top: 16px"><LI><DIV class="collapsible-header"><I style="color: #03a9f4" class="material-icons">folder_open</I>Project</DIV><DIV class="collapsible-body"><DIV class="collection"><A href="https://2018.igem.org/Team:NEU_China_A/Introduction" class="collection-item">Introduction</A><A href="https://2018.igem.org/Team:NEU_China_A/Design" class="collection-item">Design</A><A href="https://2018.igem.org/Team:NEU_China_A/Model" class="collection-item">Model</A><A href="https://2018.igem.org/Team:NEU_China_A/Demonstrate" class="collection-item">Results &amp; Demonstrate</A><A href="https://2018.igem.org/Team:NEU_China_A/InterLab" class="collection-item">InterLab</A></DIV></DIV></LI><LI><DIV class="collapsible-header"><I style="color: #03a9f4" class="material-icons">place</I>Lab</DIV><DIV class="collapsible-body"><DIV class="collection"><A href="https://2018.igem.org/Team:NEU_China_A/Notebook" class="collection-item">Notebook</A><A href="https://2018.igem.org/Team:NEU_China_A/Protocols" class="collection-item">Protocols</A><A href="https://2018.igem.org/Team:NEU_China_A/Safety" class="collection-item">Safety</A></DIV></DIV></LI><LI><DIV class="collapsible-header"><I style="color: #03a9f4" class="material-icons">details</I>Parts</DIV><DIV class="collapsible-body"><DIV class="collection"><A href="https://2018.igem.org/Team:NEU_China_A/BasicAndCompositeParts" class="collection-item">Basic
                            And Composite Parts</A><A href="https://2018.igem.org/Team:NEU_China_A/Improve" class="collection-item">Improved
                            Biobricks</A></DIV></DIV></LI><LI><DIV class="collapsible-header"><I style="color: #03a9f4" class="material-icons">lightbulb_outline</I>HP</DIV><DIV class="collapsible-body"><DIV class="collection"><A href="https://2018.igem.org/Team:NEU_China_A/Human_Practices" class="collection-item">Human
                            Practices</A><A href="https://2018.igem.org/Team:NEU_China_A/Collaborations" class="collection-item">Collaborations</A><A href="https://2018.igem.org/Team:NEU_China_A/Public_Engagement" class="collection-item">Engagement</A></DIV></DIV></LI><LI><DIV class="collapsible-header"><I style="color: #03a9f4" class="material-icons">people</I>Team</DIV><DIV class="collapsible-body"><DIV class="collection"><A href="https://2018.igem.org/Team:NEU_China_A/TeamMember" class="collection-item">Team Member</A><A href="https://2018.igem.org/Team:NEU_China_A/Attributions" class="collection-item">Attributions</A><A href="https://2018.igem.org/Team:NEU_China_A/Gallery" class="collection-item">Gallery</A></DIV></DIV></LI></UL></UL><UL id="project" class="dropdown-content"><LI><A href="https://2018.igem.org/Team:NEU_China_A/Introduction">Introduction</A></LI><LI><A href="https://2018.igem.org/Team:NEU_China_A/Design">Design</A></LI><LI><A href="https://2018.igem.org/Team:NEU_China_A/Model">Model</A></LI><LI><A href="https://2018.igem.org/Team:NEU_China_A/Demonstrate">Results &amp; Demonstrate</A></LI><LI><A href="https://2018.igem.org/Team:NEU_China_A/InterLab">Interlab</A></LI></UL><UL id="lab" class="dropdown-content"><LI><A href="https://2018.igem.org/Team:NEU_China_A/Notebook">Notebook</A></LI><LI><A href="https://2018.igem.org/Team:NEU_China_A/Protocols">Protocols</A></LI><LI><A href="https://2018.igem.org/Team:NEU_China_A/Safety">Safety</A></LI></UL><UL id="parts" class="dropdown-content"><LI><A href="https://2018.igem.org/Team:NEU_China_A/BasicAndCompositeParts">Basic And Composite Parts</A></LI><LI><A href="https://2018.igem.org/Team:NEU_China_A/Improve">Improved Biobricks</A></LI></UL><UL id="hp" class="dropdown-content"><LI><A href="https://2018.igem.org/Team:NEU_China_A/Human_Practices">Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:NEU_China_A/Collaborations">Collaborations</A></LI><LI><A href="https://2018.igem.org/Team:NEU_China_A/Public_Engagement">Engagement</A></LI></UL><UL id="team" class="dropdown-content"><LI><A href="https://2018.igem.org/Team:NEU_China_A/TeamMember">Team member</A></LI><LI><A href="https://2018.igem.org/Team:NEU_China_A/Attributions">Attributions</A></LI><LI><A href="https://2018.igem.org/Team:NEU_China_A/Gallery">Gallery</A></LI></UL></HEADER><DIV style="position:absolute;top: 20;width:100%"><DIV class="row "><H2 style="padding-left:200px; padding-top:calc(100vh - 250px)">Result&amp;Demonstrate</H2></DIV></DIV><DIV class="container"><H3 class="header"><SPAN class="light-blue-text">Overview</SPAN></H3><P class="card-panel hoverable">
The goal of NEU_China_A is to design a biological system to alleviate intestinal inflammatory diseases and prevent potential colorectal cancer. We chose the human probiotic <I>E. coli Nissle 1917</I>, as our chassis for constructing engineered probiotic. On the one hand, when the engineered bacteria senses an inflammatory signal in the intestine, it would release an anti-inflammatory compound (interleukin-10, IL-10) to reduce the intestinal inflammatory responses. On the other hand, engineered <I>E. coli</I> could secrete myrosinase to convert the cruciferous vegetables contained glucosinolate into sulforaphane that act as similar as IL-10 and prevent colorectal cancer induced by chronic inflammation.
	</P><P><STRONG>For details about our project background and introduction, please click to</STRONG><A class="link-color" href="https://2018.igem.org/Team:NEU_China_A/Description">Description</A></P><P><STRONG>For details of whole project design, please click to</STRONG><A class="link-color" href="https://2018.igem.org/Team:NEU_China_A/Design">Design</A></P><P>As a therapeutic project, we knew the difficulties for verifying the entire system in less than one year. We split the entire system into three core parts and verify them separately: the nitric oxide sensor (nitric oxide, NO, is a biomarker of intestinal inflammation), protein transport and production and the “kill switch”. In addition, we tested them in <I>DH5α</I> and <I>BL21 E.coli</I> competent cells. We believed that the success of these core parts will ultimately ensure the entire system functioning in IBD patients.</P><DIV class="row"><DIV class="s10 offset-s2"><SPAN class="flow-text light-blue-text">
                1.Nitric oxide sensor
            </SPAN></DIV><DIV class="s10 offset-s2"><P class="borderleft">
               In order to detect the NO specifically and accurately, we constructed pCDFDuet-1 plasmid as our expression vector, which contains the NO binding protein for expressing reporter gene. (Figure 1). Briefly, we integrated NO-binding protein, NorR, into this plasmid for activating the amilCP reporter gene expression via binding to PnorV promoter. To testify the validity of NO sensor, we used T7 promoter and lacO protein for boosting the NorR expression under the IPTG induction.
            </P><P class="tooltip center-align"><STRONG>Figure 1.  Diagram for NO sensor system in pCDFDuet-1 plasmid.</STRONG> T7 promoter, the gene downstream of this promoter will be transcribed when there is T7 RNA polymerase. lacO, the sequence represses the nearby promoter when there is no inducer (e.g. IPTG). RBS, ribosome binding site. NorR, NO binding protein. PnorV, a promoter which is sensitive to NO. amilCP, blue chromoprotein. T7 terminator, a terminator from T7 phage.
            </P><P class="borderleft">
             According to the previous work, ShanghaiTechChina_B 2016 team demonstrated 100μM Sodium Nitroprusside Dihydrate (SNP) aqueous solution could continuously release NO and its final concentration is stable at about 5.5μM, which as same as the NO amount in IBD patients and 100 times higher than healthy individuals [1]. According to their work, we could simulate the IBD patients’ NO signal <I>in vitro</I>. In the subsequent test of the NO sensor, the final concentration of the SNP aqueous solution was maintained at 100 μM. 
            </P><P class="borderleft">
                We transformed the constructed plasmid with NO sensor into <I>DH5α</I>, cultured at 37 ℃ overnight, and then diluted to OD600 = 0.2. Culturing bacteria at 37 ℃ for 1.5 hours, the appropriate concentration of inducer IPTG and SNP aqueous solution were added. After 6 hours of culturing, 1 mL of the bacterial solution was centrifuged at 8000 r.p.m for 1 min (Figure 2). It can be seen that the NO released by the SNP aqueous solution can effectively activate the expression of the reporter gene. Surprisingly, the blue chromoprotein in group 1mM IPTG without SNP was also be activated with undetermined mechanisms. We speculated that it may be caused by leakage expression of promoter PnorV when the NorR is overexpressed. Due to this reason, we would use a weak promoter to express NorR in further engineered bacteria.
            </P><P class="tooltip center-align"><STRONG>Figure 2. Pellets of bacteria transformed with constructed NO sensor plasmid after 6hr induction at 37 ℃.</STRONG> From left to right: control, 0.5mM IPTG without SNP, 1mM IPTG without SNP, 0.5mM IPTG with 100μM SNP, 1mM IPTG with 100μM SNP.
            </P><P class="borderleft">Besides, during our participation in the 5th CCiC (Conference of China iGEMer Community), we communicated with the iGEM team of SJTU-BioX-Shanghai and learnd that they detected colorectal cancer through the nitric oxide inducible promoter PyeaR, they used NO as their input signal. And at that time which happened to be difficult for us to detect the expression of the reporter gene GFP after using nitric oxide to drive PnorV. So, we also tested the effect of PyeaR (BBa_K381001) in the Distribution Kit (Figure 3).
			</P><P class="borderleft">
We transformed the plasmid containing BBa_K381001 into <I>DH5α</I>, and cultured at 37 ℃ overnight and then diluted to OD600 = 0.4. Then we took half of the bacteria as control and the rest was added SNP aqueous solution, and induced at 37 ℃ for 6 h. Then the fluorescence intensity of cells was observed under microplate reader and fluorescence inverted microscope (Figure 4). The histogram of GFP fluorescent density and microscope images indicated that PyeaR could be effectively activated by NO and there was almost no leakage expression. In the future, we will use PyeaR promoter based plasmid as the NO sensor.
            </P><P class="tooltip center-align"><STRONG>Figure 4. The response to NO of BBa_K381001. A</STRONG>, Histogram of GFP fluorescence: LB control, without SNP, with 100μM SNP. <STRONG>B</STRONG>, GFP fluorescence images from top to bottom: without SNP, with 100μM SNP. 
            </P><P class="borderleft">In conclusion, we demonstrated that the validity of PyeaR based NO sensor in <I>E. coli</I> of responding to chemically produced nitric oxide <I>in vitro</I>. Since the PyeaR promoter is presence in wild type <I>E. coli.</I> strain, it should be functioning in our engineered probiotic <I>E. coli</I> strain. Moreover, this engineered probiotic might detect the inflammatory signal, NO, in the host intestinal tract. 
			</P><P><STRONG>Parts for this section: </STRONG><A class="link-color" href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2817000">BBa_K2817000</A><A class="link-color" href="http://parts.igem.org/Part:BBa_K381001">BBa_K381001</A></P></DIV><DIV class="s10 offset-s2"><SPAN class="flow-text light-blue-text">
               2 Transport and Production
            </SPAN></DIV><DIV class="s10 offset-s2"><P class="borderleft">
At the beginning of the experiment, we directly constructed the plasmids of YebF-IL10 and YebF-myrosinase, but the effect of extracellular detection was not good. During the participation in CCiC, iGEMer suggested that we should first try to test the performance of the secretory tag YebF, so we constructed the plasmid YebF-GFP shown in Figure 5A. To express IL10 and myrosinase, we constructed plasmids IL10-Flag and myrosinase-His as shown in Figure 5B and Figure 5C.
            </P><P class="tooltip center-align"><STRONG>Figure 5. Production module construction. A</STRONG>, the construction of our YebF-GFP using strong promoter. J23110, a strong constitutive promoter. YebF-GFP, GFP with secretion tag. <STRONG>B</STRONG>, the construction of IL10 production module. IL10-Flag, human interleukin 10 with Flag tag. <STRONG>C</STRONG>, the construction of myrosinase production module. Myrosinase-his, our transferase with His tag.
            </P><P class="borderleft">
According to previous works, the secretory tag YebF has been demonstrated as the functioning part in <I>E.coli</I> [2]，thus, we chose this tag for further experiment. 
            </P><P class="borderleft">
When testing the validity of the secretory tag YebF, we transformed the constructed YebF-GFP plasmid into <I>DH5α</I> and cultured overnight at 37 ℃. The bacterial suspension was centrifuged at 3,000 r.p.m for 5 min and the supernatant was taken for fluorescence detection. The result is shown in Figure 6.  From our data, the YebF based secretory part worked efficiently.  
            </P><P class="tooltip center-align"><STRONG>Figure 6. The fluorescence density of overnight bacterial suspension </STRONG></P><P class="borderleft">
Next, we transformed the human IL10-Flag plasmid (Fig 5B) into <I>BL21</I>, incubated at 37 ℃ overnight and then diluted to OD600 = 0.2. After 2 h of growth at 37 ℃, IPTG was added to induce IL10 expression at 30 ℃ for 16 h. Then, the bacterial solution was lysed and the expressed IL10 was detected by Western Blot (Figure 7). From this result, IL10 was successfully expressed in our constructed plasmid. As a widely validated and used anti-inflammatory interleukin, IL10 has been identified with critical functions in immunosuppression in human [3].
            </P><P class="tooltip center-align"><STRONG>Figure 7. Western blot analyses using a flag antibody on bacterial lysate to detect IL10</STRONG>. Lane 1: Negative control (cell lysate without IPTG induction); Lane 2: 0.5mM IPTG.
            </P><P class="borderleft">
               Next, we transformed the plasmid of myrosinase-His (Fig 5C) into <I>BL21</I>, and cultured at 37 ℃ overnight and diluted to OD = 0.2. After growth for 2 h at 37 ℃, different concentrations of IPTG were added to induce at 16 ℃ for 16 h. Subsequently, bacterial cell lysate was obtained and the expression of myrosinase was detected by SDS-PAGE. (Figure 8). The sequence of the myrosinase enzyme was optimized based on the <I>K12</I> strain and the position of the strip is correct, so myrosinase was successfully expressed. The myrosinase can convert cruciferous vegetables contained glucosinolates into sulforaphane which has well-known anti-cancer activity. A previous work demonstrated the activity of myrosinase expressed by <I>E. coli Nissle 1917 in vivo</I> and in vitro [4], the expressed myrosinase in our constructed plasmid should with broad applications <I>in vivo</I>.For more details about this transferase related information, please see our <A class="link-color" href="https://2018.igem.org/Team:NEU_China_A/Design">project design</A>.
            </P><P class="tooltip center-align"><STRONG>Figure 8. SDS-PAGE analyses of bacterial lysate to detect myrosinase.</STRONG> Lane 1: Negative control (cell lysate without IPTG induction); Lane 2: 0.25mM IPTG; Lane3: 0.5mM IPTG, Lane4: 0.75mM IPTG induction for 16h at 16℃.
            </P><P><STRONG>Parts for this section: </STRONG><A class="link-color" href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2817013">BBa_K2817013</A><A class="link-color" href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2817002">BBa_K2817002</A><A class="link-color" href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2817004">BBa_K2817004</A></P></DIV><DIV class="s10 offset-s2"><SPAN class="flow-text light-blue-text">         3.Kill Switch
            </SPAN></DIV><DIV class="s10 offset-s2"><P class="borderleft">
               When we use engineered bacteria to alleviate IBD <I>in vivo</I>, the growth of bacteria must be tightly regulated. Therefore, we designed a cold shock kill switch that based on the toxin-antitoxin system mazEF. The rationale of this kill switch is relying on the mechanism that when <I>E. coli</I> is under stress, the maz-E will degrade faster than maz-F, so the relatively more maz-F will exert its toxicity. The cold shock promoter PcspA was linked with maz-F, so when the temperature is low, the maz-F will express while the maz-E stop expressing. Thus, we could kill our engineered <I>E. coli</I> when it escapes from human bodies. We inserted the reporter gene amilCP into the pColdI plasmid to characterize the performance of the cold shock promoter PcspA (Figure 9A). Then we inserted maz-F into the pColdI plasmid to build our kill switch (Figure 9B).
            </P><P class="tooltip center-align"><STRONG>Figure 9. Kill switch module construction. A</STRONG>, the construction of PcspA-amilCP plasmid. PcspA, the cold shock promoter. <STRONG>B</STRONG>, the construction of PcspA-mazF plasmid. mazF, the toxin gene from <I>E. coli.</I></P><P class="borderleft">
               We transformed the PcspA-amilCP plasmid into <I>DH5α</I> and cultured overnight at 37 ℃. The overnight culture was diluted to 
<MATH xmlns="http://www.w3.org/1998/Math/MathML"><MROW><MSUB><MI>OD</MI><MN>600</MN></MSUB></MROW></MATH>
 = 0.2 and then grow for 2 h at 37 ℃. It was then induced for 6 hours at 16 ° C or 37 ℃ under different concentrations of IPTG conditions (Figure 10). From Figure 10, the reporter gene was efficiently expressed at low temperature, which indicated that the effective expression of the toxin gene mazF and the blocking expression of the anti-toxin gene mazE at low temperature can kill our engineered bacteria in time. Although the cold shock promoter PcspA can prevent the most of the leakage expression at 37 ℃, the weak leakage expression of mazF may still cause the functioning engineered probiotics being killed inside human bodies. Therefore, the introduction of the anti-toxin protein mazE is necessary to effectively antagonize the potential leakage expression of the cold shock promoter PcspA. 
            </P><P class="tooltip center-align"><STRONG>Figure 10. Pellets of bacteria transformed with constructed PcspA-amilCP plasmid after induction of 6h.</STRONG> From left to right: 37℃ without IPTG, 37℃ with 0.5mM IPTG, 37℃ with 1mM IPTG, 16℃ without IPTG, 16℃ with 0.5mM IPTG, 16℃ with 1mM IPTG.
            </P><P class="borderleft">
 Moreover, in order to test the efficiency of mazF. We transformed the constructed PcspA-mazF plasmid into <I>BL21</I>, added 1 mM IPTG to the plate, and cultured at 16℃ for 16 h (Figure 11A). We then cultured <I>BL21</I> transformed with the PcspA-mazF plasmid overnight at 37℃. After diluting to OD600=0.02 on the next day, the cells were cultured at 16℃(Figure 11B) and 37 ℃(Figure 11C), and the OD value was measured every hour for 9 hours. From Figure 11, the kill switch worked efficiently at low temperatures and indicated that mazF enable to cause cell death even at a low expression level. 
            </P><P class="tooltip center-align"><STRONG>Figure 11. The effect of our killer gene under different conditions.. A</STRONG>, the plate of BL21 with and without killer gene under induction. B, C The effect of mazF and Lysis on the growth of Escherichia coli at different temperature (B,16℃; C,37℃) or in different plasmids.
            </P><P><STRONG>Parts for this section: </STRONG><A class="link-color" href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2817012">BBa_K2817012</A><A class="link-color" href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2817006">BBa_K2817006</A></P><P class="card-panel hoverable">Finally, we are convinced the integration of these successful module will guarantee the effective work of our engineered system in <I>E. coli</I> under realistic conditions.</P><P><STRONG>Reference</STRONG></P><P class="link-color">[1] Ljung T, Herulf M, Beijer E, et al. Rectal nitric oxide assessment in children with Crohn disease and ulcerative colitis. Indicator of ileocaecal and colorectal affection[J]. Scandinavian journal of gastroenterology, 2001, 36(10): 1073-1076.</P><P class="link-color">[2] Zhang G, Brokx S, Weiner JH (2006) Extracellular accumulation of recombinant proteins fused to the carrier protein YebF in Escherichia coli. Nat Biotechnol 24:100–104</P><P class="link-color">[3] Moore, K.W., de Waal Malefyt, R., Coffman, R.L., O'Garra, A., 2001. Interleukin-10 and the interleukin-10 receptor. Annu. Rev. Immunol. 19, 683–765.</P><P class="link-color">[4] Ho CL, Tian HQ.et al. 2018 Engineered commensal microbes for diet-mediated colorectal-cancer chemoprevention. Nature Biomedical Engineering. 2, pages27–37</P></DIV></DIV></DIV><FOOTER class="page-footer blu"><DIV class="row container"><DIV class="col s6"><DIV class="col s6">Contact us</DIV></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>