<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_NEU_China_A_Denmonstrate skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:NEU China A/Denmonstrate</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="igem_2018_team_menu displaying_menu"><A href="https://2018.igem.org/Team:NEU_China_A"><DIV class="menu_item direct_link">
				HOME
			</DIV></A><DIV class="menu_item">
			TEAM
		</DIV><DIV class="submenu"><A href="https://2018.igem.org/Team:NEU_China_A/Team"><DIV class="submenu_item">
					Team Members
				</DIV></A><A href="https://2018.igem.org/Team:NEU_China_A/Collaborations"><DIV class="submenu_item">
					Collaborations
				</DIV></A></DIV><DIV class="menu_item">
			PROJECT
		</DIV><DIV class="submenu"><A href="https://2018.igem.org/Team:NEU_China_A/Description"><DIV class="submenu_item">
					Description
				</DIV></A><A href="https://2018.igem.org/Team:NEU_China_A/Design"><DIV class="submenu_item">
					Design
				</DIV></A><A href="https://2018.igem.org/Team:NEU_China_A/Experiments"><DIV class="submenu_item">
					Experiments
				</DIV></A><A href="https://2018.igem.org/Team:NEU_China_A/Notebook"><DIV class="submenu_item">
					Notebook
				</DIV></A><A href="https://2018.igem.org/Team:NEU_China_A/InterLab"><DIV class="submenu_item">
					InterLab
				</DIV></A><A href="https://2018.igem.org/Team:NEU_China_A/Model"><DIV class="submenu_item">
					Model
				</DIV></A><A href="https://2018.igem.org/Team:NEU_China_A/Results"><DIV class="submenu_item">
					Results
				</DIV></A><A href="https://2018.igem.org/Team:NEU_China_A/Demonstrate"><DIV class="submenu_item">
					Demonstrate
				</DIV></A><A href="https://2018.igem.org/Team:NEU_China_A/Improve"><DIV class="submenu_item">
					Improve
				</DIV></A><A href="https://2018.igem.org/Team:NEU_China_A/Attributions"><DIV class="submenu_item">
					Attributions 
				</DIV></A></DIV><DIV class="menu_item">
			PARTS
		</DIV><DIV class="submenu"><A href="https://2018.igem.org/Team:NEU_China_A/Parts"><DIV class="submenu_item">
					Parts Overview
				</DIV></A><A href="https://2018.igem.org/Team:NEU_China_A/Basic_Part"><DIV class="submenu_item">
					Basic Parts
				</DIV></A><A href="https://2018.igem.org/Team:NEU_China_A/Composite_Part"><DIV class="submenu_item">
					Composite Parts
				</DIV></A><A href="https://2018.igem.org/Team:NEU_China_A/Part_Collection"><DIV class="submenu_item">
					Part Collection
				</DIV></A></DIV><A href="https://2018.igem.org/Team:NEU_China_A/Safety"><DIV class="menu_item direct_link">
				SAFETY
			</DIV></A><DIV class="menu_item"> 
			HUMAN PRACTICES 
		</DIV><DIV class="submenu"><A href="https://2018.igem.org/Team:NEU_China_A/Human_Practices"><DIV class="submenu_item">
					Human Practices
				</DIV></A><A href="https://2018.igem.org/Team:NEU_China_A/Public_Engagement"><DIV class="submenu_item">
					Education &amp; Engagement
				</DIV></A></DIV><DIV class="menu_item"> 
			AWARDS
		</DIV><DIV class="submenu"><A href="https://2018.igem.org/Team:NEU_China_A/Applied_Design"><DIV class="submenu_item">
					Applied Design
				</DIV></A><A href="https://2018.igem.org/Team:NEU_China_A/Entrepreneurship"><DIV class="submenu_item">
					Entrepreneurship
				</DIV></A><A href="https://2018.igem.org/Team:NEU_China_A/Hardware"><DIV class="submenu_item">
					Hardware
				</DIV></A><A href="https://2018.igem.org/Team:NEU_China_A/Measurement"><DIV class="submenu_item">
					Measurement
				</DIV></A><A href="https://2018.igem.org/Team:NEU_China_A/Model"><DIV class="submenu_item">
					Model
				</DIV></A><A href="https://2018.igem.org/Team:NEU_China_A/Plant"><DIV class="submenu_item">
					Plant
				</DIV></A><A href="https://2018.igem.org/Team:NEU_China_A/Software"><DIV class="submenu_item">
					Software
				</DIV></A></DIV><A href="https://igem.org/2018_Judging_Form?team=NEU_China_A"><DIV class="menu_item direct_link">
				JUDGING FORM ⇗ 
			</DIV></A></DIV><DIV class="igem_2018_team_content"><DIV class="igem_2018_team_column_wrapper"><HEADER><NAV class="blu"><DIV class="nav-wrapper"><A href="https://2018.igem.org/Team:NEU_China_A" class="brand-logo"> Home</A><A href="#" data-target="mobile-demo" class="sidenav-trigger"><I class="material-icons">menu</I></A><UL class="right hide-on-med-and-down"><LI><A data-target="project" class="dropdown-trigger" href="#!">
                        Project<I style="margin-left: 0" class="material-icons right">arrow_drop_down</I></A></LI><LI><A data-target="lab" class="dropdown-trigger" href="#!">
                        Lab<I style="margin-left: 0" class="material-icons right">arrow_drop_down</I></A></LI><LI><A data-target="parts" class="dropdown-trigger" href="#!">
                        Parts<I style="margin-left: 0" class="material-icons right">arrow_drop_down</I></A></LI><LI><A data-target="hp" class="dropdown-trigger" href="#!">
                        HP<I style="margin-left: 0" class="material-icons right">arrow_drop_down</I></A></LI><LI><A data-target="team" class="dropdown-trigger" href="#!">
                        Team<I style="margin-left: 0" class="material-icons right">arrow_drop_down</I></A></LI></UL></DIV></NAV><UL class="sidenav" id="mobile-demo"><UL class="collapsible" style="margin-top: 16px"><LI><DIV class="collapsible-header"><I style="color: #03a9f4" class="material-icons">folder_open</I>Project</DIV><DIV class="collapsible-body"><DIV class="collection"><A href="https://2018.igem.org/Team:NEU_China_A/Introduction" class="collection-item">Introduction</A><A href="https://2018.igem.org/Team:NEU_China_A/Design" class="collection-item">Design</A><A href="https://2018.igem.org/Team:NEU_China_A/Model" class="collection-item">Model</A><A href="https://2018.igem.org/Team:NEU_China_A/Demonstrate" class="collection-item">Results &amp; Demonstrate</A><A href="https://2018.igem.org/Team:NEU_China_A/InterLab" class="collection-item">InterLab</A></DIV></DIV></LI><LI><DIV class="collapsible-header"><I style="color: #03a9f4" class="material-icons">place</I>Lab</DIV><DIV class="collapsible-body"><DIV class="collection"><A href="https://2018.igem.org/Team:NEU_China_A/Notebook" class="collection-item">Notebook</A><A href="https://2018.igem.org/Team:NEU_China_A/Protocols" class="collection-item">Protocols</A><A href="https://2018.igem.org/Team:NEU_China_A/Safety" class="collection-item">Safety</A></DIV></DIV></LI><LI><DIV class="collapsible-header"><I style="color: #03a9f4" class="material-icons">details</I>Parts</DIV><DIV class="collapsible-body"><DIV class="collection"><A href="https://2018.igem.org/Team:NEU_China_A/BasicAndCompositeParts" class="collection-item">Basic
                            And Composite Parts</A><A href="https://2018.igem.org/Team:NEU_China_A/Improve" class="collection-item">Improved
                            Biobricks</A></DIV></DIV></LI><LI><DIV class="collapsible-header"><I style="color: #03a9f4" class="material-icons">lightbulb_outline</I>HP</DIV><DIV class="collapsible-body"><DIV class="collection"><A href="https://2018.igem.org/Team:NEU_China_A/Human_Practices" class="collection-item">Human
                            Practices</A><A href="https://2018.igem.org/Team:NEU_China_A/Collaborations" class="collection-item">Collaborations</A><A href="https://2018.igem.org/Team:NEU_China_A/Public_Engagement" class="collection-item">Engagement</A></DIV></DIV></LI><LI><DIV class="collapsible-header"><I style="color: #03a9f4" class="material-icons">people</I>Team</DIV><DIV class="collapsible-body"><DIV class="collection"><A href="https://2018.igem.org/Team:NEU_China_A/TeamMember" class="collection-item">Team Member</A><A href="https://2018.igem.org/Team:NEU_China_A/Attributions" class="collection-item">Attributions</A><A href="https://2018.igem.org/Team:NEU_China_A/Gallery" class="collection-item">Gallery</A></DIV></DIV></LI></UL></UL><UL id="project" class="dropdown-content"><LI><A href="https://2018.igem.org/Team:NEU_China_A/Introduction">Introduction</A></LI><LI><A href="https://2018.igem.org/Team:NEU_China_A/Design">Design</A></LI><LI><A href="https://2018.igem.org/Team:NEU_China_A/Model">Model</A></LI><LI><A href="https://2018.igem.org/Team:NEU_China_A/Demonstrate">Results &amp; Demonstrate</A></LI><LI><A href="https://2018.igem.org/Team:NEU_China_A/InterLab">Interlab</A></LI></UL><UL id="lab" class="dropdown-content"><LI><A href="https://2018.igem.org/Team:NEU_China_A/Notebook">Notebook</A></LI><LI><A href="https://2018.igem.org/Team:NEU_China_A/Protocols">Protocols</A></LI><LI><A href="https://2018.igem.org/Team:NEU_China_A/Safety">Safety</A></LI></UL><UL id="parts" class="dropdown-content"><LI><A href="https://2018.igem.org/Team:NEU_China_A/BasicAndCompositeParts">Basic And Composite Parts</A></LI><LI><A href="https://2018.igem.org/Team:NEU_China_A/Improve">Improved Biobricks</A></LI></UL><UL id="hp" class="dropdown-content"><LI><A href="https://2018.igem.org/Team:NEU_China_A/Human_Practices">Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:NEU_China_A/Collaborations">Collaborations</A></LI><LI><A href="https://2018.igem.org/Team:NEU_China_A/Public_Engagement">Engagement</A></LI></UL><UL id="team" class="dropdown-content"><LI><A href="https://2018.igem.org/Team:NEU_China_A/TeamMember">Team member</A></LI><LI><A href="https://2018.igem.org/Team:NEU_China_A/Attributions">Attributions</A></LI><LI><A href="https://2018.igem.org/Team:NEU_China_A/Gallery">Gallery</A></LI></UL></HEADER><DIV class="container"><H3 class="header"><SPAN class="light-blue-text">Overview</SPAN></H3><P class="card-panel hoverable">
	The goal of NEU_China_A this year is to design a biological system aiming to alleviate intestinal inflammatory diseases and prevent potential colorectal cancer. We chose E. coli Nissle 1917 as our chassis, a probiotic that is safe for humans. On the one hand, when it senses an inflammatory signal in the intestine, it releases an anti-inflammatory compound (interleukin-10) to put out the fire in the intestines. On the other hand, it can release myrosinase to convert the glucosinolate contained in cruciferous vegetables into sulforaphane. The sulforaphane can both alleviate inflammation in the intestine and prevent colorectal cancer induced by chronic inflammation.
	</P><P><STRONG>For details about our project background and introduction, please click to</STRONG><A class="link-color" href="https://2018.igem.org/Team:NEU_China_A/Description">Description</A></P><P><STRONG>For details of whole project design, please click to</STRONG><A class="link-color" href="https://2018.igem.org/Team:NEU_China_A/Design">Design</A></P><P>As a therapeutic project, we know that it is very difficult to verify the entire system in less than a year. We split the entire system into three core parts to verify separately: the nitric oxide sensor (nitric oxide is a marker of inflammation), protein transport and production, and the kill switch. In addition, we test them in DH5α and BL21. We believe that the success of these core parts will ultimately make the system work in the real world.</P><DIV class="row"><DIV class="s10 offset-s2"><SPAN class="flow-text light-blue-text">
                1.Nitric oxide sensor
            </SPAN></DIV><DIV class="s10 offset-s2"><P class="borderleft">
                We constructed our NO sensor in the pCDFDuet-1 plasmid (Figure 1). Although the E. coli has native NorR expression, increasing its expression facilitates the elimination of stoichiometric imbalances between NorR in the genome and PnorV on the foreign plasmid, avoiding interference with the host's detoxification of nitric oxide.
            </P><P class="borderleft">
                According to the results of the <A class="link-color" href="https://2016.igem.org/Team:ShanghaiTechChina_B/Project#NO_Sensor">ShanghaiTechChina_B 2016 team</A>, 100μM SNP aqueous solution can continuously release NO, and the NO concentration is stable at about 5.5μM. This concentration is the concentration of NO in patients with IBD and it is nearly 100 times higher than normal people. So, we can simulate the patient's inflammatory signal in vitro. In the subsequent test of the NO sensor, the final concentration of the SNP aqueous solution we added was 100 μM. 
            </P><P class="borderleft">
                We transformed the constructed plasmid of NO sensor into DH5α, and cultured at 37 ℃ overnight to dilute to OD = 0.2. After 1.5 h of growth at 37 ℃, the inducer IPTG and SNP aqueous solution were added. After 6 h at 37 ℃, 1 mL of the bacterial solution was centrifuged at 8000 rpm for 1 min (Figure 2). It can be seen that the NO released by the SNP aqueous solution can effectively activate the expression of the reporter gene. But to our surprise, the blue chromoprotein in group 1mM IPTG without SNP was also be activated. We have not yet figured out the reasons behind this phenomenon. But we speculate that it may be caused by leaky expression of promoter PnorV when the NorR overexpress. Due to this, we will use a weak promoter to express NorR in future engineered bacteria to avoid leakage.
            </P><P class="tooltip center-align">
                From left to right: control, 0.5mM IPTG without SNP, 1mM IPTG without SNP, 0.5mM IPTG with 100μM SNP, 1mM IPTG with 100μM SNP.
            </P><P class="borderleft">Besides, during our participation in the 5th CCiC (Conference of China iGEMer Community), we communicated with the iGEM team SJTU-BioX-Shanghai. We learned that they detected colorectal cancer through the nitric oxide inducible promoter PyeaR, and at that time we were having trouble using nitric oxide to drive PnorV to transcribe our reporter. So, we also tested the effect of PyeaR (BBa_K381001) in the Distribution Kit (Figure 3).
			</P><P class="borderleft">
                We also tested the response of BBa_K381001 to NO in the Distribution Kit (Figure 3).
            </P><P class="borderleft">
                We transformed the plasmid containing BBa_K381001 into DH5α, and cultured at 37 ℃ overnight to dilute to OD = 0.4. Then we took half as control and the other half added SNP aqueous solution and induced at 37 ℃ for 6 h. Then we detected the fluorescence using a microplate reader and a fluorescence microscope (Figure 4). We can see that PyeaR can also be effectively activated by NO with almost no leakage. In the future, we will compare the performance of PyeaR and PnorV in depth.
            </P><P class="tooltip center-align">
               Histogram of GFP fluorescence: LB control, without SNP, with 100μM SNP. B, GFP fluorescence image from top to bottom: without SNP, with 100μM SNP.
            </P><P class="borderleft">In conclusion, we have confirmed that the NO sensor in E. coli we built capable of responding to chemically produced nitric oxide in vitro. And we believe that as a naturally NO response mechanism in E. coli, the NO sensor we construct will also work in vivo.
			</P><P><STRONG>Parts for this section: </STRONG><A class="link-color" href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2817000">BBa_K2817000</A><A class="link-color" href="http://parts.igem.org/Part:BBa_K381001">BBa_K381001</A></P></DIV><DIV class="s10 offset-s2"><SPAN class="flow-text light-blue-text">
               2 Transport and Production
            </SPAN></DIV><DIV class="s10 offset-s2"><P class="borderleft">
                At the beginning of the experiment, we directly constructed the plasmids of YebF-IL10 and YebF-myrosinase, but the effect of extracellular detection was not satisfactory. During the participation in CCiC, iGEMer suggested that we first start from testing the performance of the secretory tag YebF, so we constructed the plasmid YebF-GFP shown in Figure 5A. To express IL10 and myrosinase, we constructed plasmids IL10-flag and myrosinase-his as shown in Figure 5B and Figure 5C.
            </P><P class="tooltip center-align"><STRONG>A</STRONG>, the construction of our YebF-GFP using strong promoter.
                <STRONG>B</STRONG>, the construction of IL10 production module.
                <STRONG>C</STRONG>, the construction of myrosinase production module.
            </P><P class="borderleft">
                When testing the validity of the secretory tag YebF, we transformed the constructed YebF-GFP plasmid into DH5α and cultured overnight at 37 ℃. The supernatant was centrifuged at 3000 rpm for 5 min, and the supernatant was taken for fluorescence detection. The results are shown in the Figure 6. But we forgot to set up a positive control group. Due to time constraints, we have no time to repeat this experiment. But as a secretory tag used in both the literature and the iGEM team, we believe it can actually work.
            </P><P class="borderleft">
                We transformed the IL10-flag plasmid into BL21, and incubated at 37 ℃ overnight to dilute to OD = 0.2. After 2 h of growth at 37 ℃, IPTG was added and induced at 30 ℃ for 16 h. Then, the bacterial solution was lysed and the expression of IL10 was detected by western blot (Figure 7). We used the human IL10 sequence optimized by E. coli from the previous iGEM team (the original part is <A class="link-color" href="http://parts.igem.org/wiki/index.php/Part:BBa_K554004">BBa_K554004</A>). So we have reason to believe that we have successfully expressed IL10. As a widely validated and used cytokine, we believe it can play an immunosuppressive role in vivo.
            </P><P class="tooltip center-align">
                Lane 1: Negative control (cell lysate without IPTG induction); Lane 2: 0.5mM IPTG, Lane3: 1mM IPTG
                induction for 16h at 30℃.
            </P><P class="borderleft">
               We transformed the plasmid of myrosinase-his into BL21, and cultured at 37 ℃ overnight and diluted to OD = 0.2. After growth for 2 h at 37 ℃, different concentrations of IPTG were added and induced at 16 ℃ for 16 h. The bacterial cell lysis was then performed to detect the expression of myrosinase by SDS-PAGE (Figure 8). The sequence of the myrosinase enzyme was optimized based on the K12 strain and the position of the strip is almost correct, so we believe we have successfully expressed it. The myrosinase can convert glucosinolates contained in cruciferous vegetables into sulforaphane which has well-known anti-cancer activity. A literature earlier this year <A class="link-color" href="https://doi.org/10.1038/s41551-017-0181-y">(https://doi.org/10.1038/s41551-017-0181-y)</A> has validated the activity of myrosinase expressed by E. coli Nissle 1917 in vivo and in vitro. So we believe that the myrosinase we expressed can work. For more details about this transferase related information, please see our <A class="link-color" href="https://2018.igem.org/Team:NEU_China_A/Design">project design</A>.
            </P><P class="tooltip center-align">
                Lane 1: Negative control (cell lysate without IPTG induction); Lane 2: 0.25mM IPTG; Lane3: 0.5mM IPTG, Lane4: 0.75mM IPTG induction for 16h at 16℃.
            </P><P><STRONG>Parts for this section: </STRONG><A class="link-color" href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2817013">BBa_K2817013</A><A class="link-color" href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2817002">BBa_K2817002</A><A class="link-color" href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2817004">BBa_K2817004</A></P></DIV><DIV class="s10 offset-s2"><SPAN class="flow-text light-blue-text">
                3.Kill Switch
            </SPAN></DIV><DIV class="s10 offset-s2"><P class="borderleft">
                When we use engineered bacteria to treat IBD, we need to do biocontainment work to avoid the release of engineered bacteria into the environment. Therefore, in our project, we designed a cold shock kill switch based on the toxin-antitoxin system mazEF, so we can kill our engineered E. coli when it escape from human body. We inserted the reporter gene amilCP into the pColdI plasmid to characterize the performance of the cold shock promoter PcspA (Figure 9A). Then we inserted maz-F into the pColdI plasmid to build our kill switch (Figure 9B).
            </P><P class="tooltip center-align"><STRONG>A</STRONG>, the construction of PcspA-amilCP plasmid.
                <STRONG>B</STRONG>, the construction of PcspA-mazF plasmid.
            </P><P class="borderleft">
                We transformed the PcspA-amilCP plasmid into DH5α and cultured overnight at 37℃. The overnight culture was diluted to OD = 0.2 and allowed to grow for 2 h at 37℃. It was then divided into different concentrations of IPTG at 16℃ and 37℃ for 6 h (Figure 10). It can be seen from the figure 11A that the reporter gene is efficiently expressed at low temperature, which indicates that the effective expression of the toxin gene mazF and the closed expression of the anti-toxin gene mazE at low temperature can kill our engineered bacteria in time. Although the cold shock promoter PcspA has a certain leakage at body temperature, the toxin is neutralized by the anti-toxin expressed at body temperature, so the effect is not significant. However, it is best to add a lacO site in the future to suppress leakage.
            </P><P class="tooltip center-align">
                From left to right: 37℃ without IPTG, 37℃ with 0.5mM IPTG, 37℃ with 1mM IPTG, 16℃ without IPTG, 16℃
                with 0.5mM IPTG, 16℃ with 1mM IPTG.
            </P><P class="borderleft">
                    Next, we test the effect of mazF. We transformed the constructed PcspA-mazF plasmid into BL21, added 1 mM IPTG to the plate, and cultured at 16℃ for 16 h (Figure 11A). We then cultured BL21 transformed with the PcspA-mazF plasmid overnight at 16℃. After diluting to OD=0.2 on the next day, the cells were cultured at 16℃, and the OD value was measured every hour for 9 hours (Figure 11B). From Figure 11 below, you can see that our kill switch can work at low temperatures. Although some engineered bacteria still survive, we believe they will be effectively killed in the absence of nutritional support.
            </P><P class="tooltip center-align"><STRONG>A</STRONG>, the plate of BL21 with and without killer gene under induction. B, the growth curve of BL21 with and without killer gene under induction.
            </P><P><STRONG>Parts for this section: </STRONG><A class="link-color" href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2817012">BBa_K2817012</A><A class="link-color" href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2817006">BBa_K2817006</A></P><P class="card-panel hoverable">Finally, we are convinced the integration of these successful module will guarantee the effective work of our engineered system in E. coli under realistic conditions.</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>