<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Hong_Kong_HKU_Description skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Hong Kong HKU/Description</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container" align="center"><DIV class="Logo"><SVG xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" viewbox="-2 -2 504 504" id="PlasmidNav" style="transform-origin: 50% 50% 0px; transform: translate3d(0px, 0px, 0px); touch-action: none; -webkit-user-select: none;"><G id="symbolsContainer"><SYMBOL class="icon icon-" id="icon-1" viewbox="0 0 35 35"><TEXT fill="#FFFFFF" x="50%" y="50%" dy="-1em" text-anchor="middle" font-size="1em" font-family="sans-serif">Project</TEXT></SYMBOL><SYMBOL class="icon icon-" id="icon-2" viewbox="0 0 35 35"><TEXT fill="#FFFFFF" x="50%" y="50%" dy="-1em" text-anchor="middle" font-size="1em" font-family="sans-serif">Notebook</TEXT></SYMBOL><SYMBOL class="icon icon-" id="icon-3" viewbox="0 0 35 35"><TEXT fill="#FFFFFF" x="50%" y="50%" dy="-1em" text-anchor="middle" font-size="1em" font-family="sans-serif">Links</TEXT></SYMBOL><SYMBOL class="icon icon-" id="icon-4" viewbox="0 0 35 35"><TEXT fill="#FFFFFF" x="50%" y="50%" dy="-1em" text-anchor="middle" font-size="1em" font-family="sans-serif">Attributions</TEXT></SYMBOL><SYMBOL class="icon icon-" id="icon-5" viewbox="0 0 35 35"><TEXT fill="#FFFFFF" x="50%" y="50%" dy="-1em" text-anchor="middle" font-size="1em" font-family="sans-serif">Practices</TEXT></SYMBOL><SYMBOL class="icon icon-" id="icon-6" viewbox="0 0 35 35"><TEXT fill="#FFFFFF" x="50%" y="50%" dy="-1em" text-anchor="middle" font-size="1em" font-family="sans-serif">Achievements</TEXT></SYMBOL><SYMBOL class="icon icon-" id="icon-7" viewbox="0 0 35 35"><TEXT fill="#FFFFFF" x="50%" y="50%" dy="-1em" text-anchor="middle" font-size="1em" font-family="sans-serif">HOMEPAGE</TEXT></SYMBOL></G></SVG></DIV></DIV><DIV class="container" align="center"><H2>Project</H2><UL class="nav nav-pills"><LI class="active dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">Description</A></LI><LI><A data-toggle="collapse" data-parent="#DescriptionContent" href="#Inspiration">Inspiration</A></LI><LI><A data-toggle="collapse" data-parent="#DescriptionContent" href="#Objectives">Objectives</A></LI><LI><A data-toggle="collapse" data-parent="#DescriptionContent" href="#Progress">Progress</A></LI><LI><A data-toggle="collapse" data-parent="#DescriptionContent" href="#Sigificances">Sigificances</A></LI></UL><LI><A href="https://2016.igem.org/Team:Hong_Kong_HKU/Design">Design</A></LI><DIV class="tab-content"><H3>Description</H3><P class="text-justify" align="left"><FONT size="3">
        This summer, we focuses on the intriguing world of diagnostic probes. 
        We spent approximately 2 weeks to finalize our designs and we finally came up with an elegant tetrahedral design. 
        Our ultimate goal is to enable our functional DNA nanostructure to be synthesized and self-assembled in E. coli. 
        They will detect specific biomarkers expressed in the diseased cells.
        We are presenting our brainstorming of ideas and linking them to the key elements of the iGEM competition.
        Click to explore and enjoy.
      	</FONT></P><DIV id="Description" class="tab-pane fade in active"><DIV class="panel-group" id="DescriptionContent" role="tablist" aria-multiselectable="true"><DIV class="panel panel-transparent"><DIV class="panel-heading" role="tab"><H3>Inspiration</H3></DIV><DIV id="Inspiration" class="panel-collapse collapse in"><DIV class="panel-body"><H4>Early diagnosis of cancer</H4><P class="text-justify" align="left"><FONT size="3">
               Cancer has always been a devastating disease. In 2012, there were 14.1 million new cancer cases worldwide.<SUP>[1]</SUP> 
               Early diagnosis of cancer may help to reduce the mortality rate and extend the life expectancy of patients. 
               For instance, in the U. K., nearly 90% of patients diagnosed with stage I lung cancer lived for more than a year while only 19% of patients diagnosed at stage IV do so.<SUP>[2]</SUP> 
               Early diagnosis of cancer is also believed to be vital for successful treatment and recovery.
               Significant gene mutations might indicate the possibility of development of cancers. Although recent research has diagnosed cancers by analyzing individual genetic mutation profiles<SUP>[3],[4]</SUP>, 
               such diagnostic method takes up considerable amount of time to obtain accurate results. As conventional diagnostic methods involve complicated procedures, 
               DNA nanostructures have been introduced to detect cancer biomarkers to facilitate simple diagnosis.</FONT></P><H4>DNA nanostructures and miRNAs as biomarkers</H4><P class="text-justify" align="left"><FONT size="3">DNA has emerged as a promising material that allows researchers to construct novel designs as its structure could be predicted easily and accurately.<SUP>[5]</SUP> 
               Examples of DNA nanostructures include nano-tweezers to detect norovirus and a DNA ‘Nano-Claw’ to detect membrane markers of cancer cells.<SUP>[6],[7]</SUP>
               DNA Boolean logic gates have been constructed to produce signals in the presence of multiple targets, such as OR-gate and AND-gate DNA tetrahedra that generate fluorescence resonance energy transfer (FRET) signal when multiple inputs hybridize with the probe.<SUP>[8]</SUP>
               As for the targets to be detected, different microRNAs (miRNAs) have been identified to be associated with cancers. For example, miR-15b-5p, miR-338-5p, and miR-764 found in plasma are potential biomarkers for detecting hepatocellular carcinoma cancer (HCC), 
               a common type of liver cancer.<SUP>[9]</SUP> It has already been reported that it is promising to use these biomarkers - miRNAs to detect cancers.<SUP>[10]</SUP></FONT><SPAN class="label label-info" align="left">References</SPAN><FONT size="2">
               1. American Cancer Society. (2015). Global Cancer Facts &amp; Figures. Retrieved from http://www.cancer.org/acs/groups/content/@research/documents/document/acspc-044738.pdf
               2.Public Health England. (2014). National Cancer Intelligence Network Cancer survival in England by stage. Retrieved from http://www.cancerresearchuk.org/health-professional/cancer-statistics/statistics-by-cancer-type/lung-cancer/survival#ref-3
               3.Pereira, B., Chin, S., Rueda, O. M., Vollan, H. M., Provenzano, E., Bardwell, H. A., Pugh, M., et al. (2016). The somatic mutation profiles of 2500 primary breast cancers refine their genomic landscapes. Nature Communications
               4.Pereira, B., Chin, S. F., Rueda, O. M., Vollan, H. K. M., Provenzano, E., Bardwell, H. A., ... &amp; Tsui, D. W. (2016). The somatic mutation profiles of 2,433 breast cancers refines their genomic and transcriptomic landscapes. Nature communications, 7.
               5.Chen, Y. J., Groves, B., Muscat, R. A., &amp; Seelig, G. (2015). DNA nanotechnology from the test tube to the cell. Nature nanotechnology, 10(9), 748-760.
               6.Nakatsuka, K., Shigeto, H., Kuroda, A., &amp; Funabashi, H. (2015). A split G-quadruplex-based DNA nano-tweezers structure as a signal-transducing molecule for the homogeneous detection of specific nucleic acids. Biosensors and Bioelectronics, 74, 222-226.
               7.You, M., Peng, L., Shao, N., Zhang, L., Qiu, L., Cui, C., &amp; Tan, W. (2014). DNA “nano-claw”: logic-based autonomous cancer targeting and therapy. Journal of the American Chemical Society, 136(4), 1256-1259.
               8.Pei, H., Liang, L., Yao, G., Li, J., Huang, Q., &amp; Fan, C. (2012). Reconfigurable Three‐Dimensional DNA Nanostructures for the Construction of Intracellular Logic Sensors. Angewandte Chemie, 124(36), 9154-9158.
               9.Chen, Y., Chen, J., Liu, Y., Li, S., &amp; Huang, P. (2015). Plasma miR-15b-5p, miR-338-5p, and miR-764 as Biomarkers for Hepatocellular Carcinoma. Medical science monitor: international medical journal of experimental and clinical research, 21, 1864.
               10.Montani, F., &amp; Bianchi, F. (2016). Circulating Cancer Biomarkers: The Macro-revolution of the Micro-RNA. EBioMedicine, 5, 4-6.</FONT></P><BUTTON type="button" class="btn btn-info center-block" align="center"><A href="https://2016.igem.org/Team:Hong_Kong_HKU/Demonstrate" target="_blank">Check out our demonstration here</A></BUTTON></DIV></DIV></DIV><DIV class="panel panel-transparent"><DIV class="panel-heading"><H3>Objectives</H3></DIV><DIV id="Objectives" class="panel-collapse collapse"><DIV class="panel-body"><H4><I>In vivo</I> synthesis of functional DNA nanostructure</H4><P class="text-justify" align="left"><FONT size="3">Our aim is to design a novel DNA nanostructure that can detect multiple miRNA targets simultaneously. 
                We hope that our design can discriminate a single base mutation of the target miRNAs. Hence, it can be highly specific to our targets and avoid false positives. 
                Our goal is clear - we aim to design a tool which can possibly detect a combination of biomarkers and enhance the sensitivity of detecting a particular type of cancer.
                Recently, <I>in vitro</I> applications of DNA nanostructure have already achieved point-of-care (POC) diagnosis<SUP>[11]</SUP>. 
                Therefore, we hope to move from <I>in vitro</I> to <I>in vivo</I> by developing a self-assembled DNA nanostructure that can potentially target miRNAs <I>in vivo</I>. 
                Detecting serum miRNA can be challenging because of the low serum miRNA level, so methods such as quantitative polymerase chain reaction are used to amplify the target miRNAs before detecting them.<SUP>[12]</SUP> 
                We hope that our DNA nanostructure, which is synthesized and assembled <I>in vivo</I>, can potentially eliminate the need of target amplification. 
                In addition, our design has an advantage over the current designs of molecular beacon. Molecular beacon makes use of fluorophores and quenchers<SUP>[13]</SUP>, which cannot be synthesized <I>in vivo</I>. 
                Our design does not require the use of fluorophore and quencher and thus can work well inside cells. In addition, our DNA nanostructure can be produced at a lower cost as fluorophore and quencher are not used.
                </FONT><SPAN class="label label-info" align="left">References</SPAN><FONT size="2">
                11. Hartman, Mark R., et al.(2013) . &quot;Point-of-care nucleic acid detection using nanotechnology.&quot; Nanoscale 5.21 (2013): 10141-10154.
                12. Wang. W.T., Chen.Y.Q. (2014). &quot;Circulating miRNAs in cancer: from detection to therapy.&quot; Journal of Hematology &amp; Oncology. (2014): Vol.7. 86.
                13. TSOURKAS, Andrew, et al. (2003). Hybridization kinetics and thermodynamics of molecular beacons. Nucleic acids research, 2003, 31.4: 1319-1330.
                </FONT></P><BUTTON type="button" class="btn btn-info center-block" align="center"><A href="https://2016.igem.org/Team:Hong_Kong_HKU/Design" target="_blank">Check out our design here</A></BUTTON></DIV></DIV></DIV><DIV class="panel panel-transparent"><DIV class="panel-heading"><H3>Progress</H3></DIV><DIV id="Progress" class="panel-collapse collapse"><DIV class="panel-body"><P class="text-justify" align="left"><FONT size="3">Our design is a 3-dimensional structure that can be self-assembled from oligonucleotides. 
                Our aim is to construct a nanostructure that is able to detect multiple miRNA biomarkers such that it can reach a higher accuracy for diagnosis. 
                For the selection of biomarkers, we are looking for a combination of miRNAs that are specific to a certain type of disease including cancers.
                At current stage, we are testing different designs <I>in vitro</I> to see if they can produce desired signals. After proving our designs can work <I>in vitro</I>, 
                we will attempt to test them <I>in vivo</I>. Finally, we will design a mechanism such that E. coli can synthesize the required oligonucleotides to form the specified nanostructure.
                </FONT></P><BUTTON type="button" class="btn btn-info center-block" align="center"><A href="https://2016.igem.org/Team:Hong_Kong_HKU/Notebook" target="_blank">Check out our notebook here</A></BUTTON></DIV></DIV></DIV><DIV class="panel panel-transparent"><DIV class="panel-heading"><H3>Sigificances</H3></DIV><DIV id="Sigificances" class="panel-collapse collapse"><DIV class="panel-body"><H4>A leap forward - <I>in vivo</I> synthesis of 3D functional DNA nanostructures</H4><P class="text-justify" align="left"><FONT size="3">In the past decade, functional DNA nanostructures have been used in similar <I>in vitro</I> approaches to detect various cancer biomarkers.<SUP>[14],[15]</SUP> 
                It is noted that most of those designs were applied <I>in vitro</I>. Recently, 1D and 2D DNA structures were successfully expressed and assembled <I>in vivo</I><SUP>[16]</SUP>, 
                while several novel 3D DNA structures were synthesized to produce signals <I>in vivo</I><SUP>[17],[18]</SUP>. 
                Given these advancements, our ultimate goal is to enable our functional DNA nanostructure to be synthesized and self-assembled in E. coli, that can function inside the disease cells. 
                This, if successful and with further refinement, could be a great replacement to colour coded surgery in the surgical field.<SUP>[19]</SUP>
                Last but not least, the cost and quality of production, efficiency and accuracy of our intracellularly-synthesized 3D structure will be compared to current diagnostic methods.
                </FONT><SPAN class="label label-info">References</SPAN><FONT size="2">
                14. Miao, P., Wang, B., Chen, X., Li, X., &amp; Tang, Y. (2015). Tetrahedral DNA nanostructure-based microRNA biosensor coupled with catalytic recycling of the analyte. ACS applied materials &amp; interfaces, 7(11), 6238-6243.
                15. Li W. et. al. (2015). Highly selective and sensitive detection of miRNA based on toehold-mediated strand displacement reaction and DNA tetrahedron substrate. Biosensors and Bioelectronics. 71, 401-406.
                16. Elbaz, J., Yin, P., &amp; Voigt, C. A. (2016). Genetic encoding of DNA nanostructures and their self-assembly in living bacteria. Nature communications, 7.
                17. Kim K. et. al. (2013). Drug delivery by self-assembled DNA tetrahedron for overcoming drug resistance in breast cancer cells. Chem. Commun. 49, 2010-2012.
                18. Kim K. et. al. (2013). Sentinel lymph node imaging by a fluorescently labeled DNA tetrahedron. Biomaterials. 34, 5226-5235.
                19. Nguyen, Q. T., &amp; Tsien, R. Y. (2013). Fluorescence-guided surgery with live molecular navigation [mdash] a new cutting edge. Nature reviews cancer, 13(9), 653-662.
                </FONT></P><BUTTON type="button" class="btn btn-info center-block" align="center"><A href="https://2016.igem.org/Team:Hong_Kong_HKU/Proof" target="_blank">Check out proof of concept here</A></BUTTON></DIV></DIV></DIV></DIV></DIV></DIV></DIV><FOOTER class="text-center"><DIV class="container" align="center"><H5>Copyright © 2016 HKU iGEM. All rights reserved.</H5></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>