<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Waterloo_HP_Silver skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Waterloo/HP/Silver</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><NAV class="navbar navbar-toggleable-md navbar-inverse bg-primary fixed-top"><A class="navbar-brand" href="https://2017.igem.org/Team:Waterloo">Waterloo</A></NAV></P><DIV class="collapse navbar-collapse" id="navbarCollapse"><UL class="navbar-nav mr-auto"><LI class="nav-item"><A class="nav-link" href="https://2017.igem.org/Team:Waterloo"><SPAN>Home</SPAN></A></LI><LI class="dropdown nav-item"><A href="#" class="nav-link dropdown-toggle nav-link" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Team</A></LI><LI class=""><A class="dropdown-item" href="https://2017.igem.org/Team:Waterloo/Team"><SPAN>Team</SPAN></A></LI><LI class=""><A class="dropdown-item" href="https://2017.igem.org/Team:Waterloo/Collaborations"><SPAN>Collaborations</SPAN></A></LI></UL><LI class="dropdown nav-item"><A href="#" class="nav-link dropdown-toggle nav-link" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Project</A></LI><LI class=""><A class="dropdown-item" href="https://2017.igem.org/Team:Waterloo/Contribution"><SPAN>Contribution</SPAN></A></LI><LI class=""><A class="dropdown-item" href="https://2017.igem.org/Team:Waterloo/Description"><SPAN>Description</SPAN></A></LI><LI class=""><A class="dropdown-item" href="https://2017.igem.org/Team:Waterloo/Design"><SPAN>Design</SPAN></A></LI><LI class=""><A class="dropdown-item" href="https://2017.igem.org/Team:Waterloo/Experiments"><SPAN>Experiments</SPAN></A></LI><LI class=""><A class="dropdown-item" href="https://2017.igem.org/Team:Waterloo/Proof"><SPAN>Proof</SPAN></A></LI><LI class=""><A class="dropdown-item" href="https://2017.igem.org/Team:Waterloo/Demonstrate"><SPAN>Demonstrate</SPAN></A></LI><LI class=""><A class="dropdown-item" href="https://2017.igem.org/Team:Waterloo/Results"><SPAN>Results</SPAN></A></LI><LI class=""><A class="dropdown-item" href="https://2017.igem.org/Team:Waterloo/Notebook"><SPAN>Notebook</SPAN></A></LI><LI class="nav-item"><A class="nav-link" href="https://2017.igem.org/Team:Waterloo/Parts"><SPAN>Parts</SPAN></A></LI><LI class="nav-item"><A class="nav-link" href="https://2017.igem.org/Team:Waterloo/Safety"><SPAN>Safety</SPAN></A></LI><LI class="nav-item"><A class="nav-link" href="https://2017.igem.org/Team:Waterloo/Attributions"><SPAN>Attributions</SPAN></A></LI><LI class="active dropdown nav-item"><A href="#" class="nav-link dropdown-toggle nav-link" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Human Practices</A></LI><LI class=""><A class="dropdown-item" href="https://2017.igem.org/Team:Waterloo/Human_Practices"><SPAN>Human Practices</SPAN></A></LI><LI class="active"><A class="dropdown-item" href="https://2017.igem.org/Team:Waterloo/HP/Silver"><SPAN>Silver</SPAN></A></LI><LI class=""><A class="dropdown-item" href="https://2017.igem.org/Team:Waterloo/HP/Gold_Integrated"><SPAN>Gold</SPAN></A></LI><LI class=""><A class="dropdown-item" href="https://2017.igem.org/Team:Waterloo/Integrated_Practices"><SPAN>Integrated Practices</SPAN></A></LI><LI class=""><A class="dropdown-item" href="https://2017.igem.org/Team:Waterloo/Engagement"><SPAN>Engagement</SPAN></A></LI><LI class=""><A class="dropdown-item" href="https://2017.igem.org/Team:Waterloo/Snake35"><SPAN>Snake35</SPAN></A></LI><LI class="dropdown nav-item"><A href="#" class="nav-link dropdown-toggle nav-link" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Awards</A></LI><LI class=""><A class="dropdown-item" href="https://2017.igem.org/Team:Waterloo/Software"><SPAN>Software</SPAN></A></LI><LI class=""><A class="dropdown-item" href="https://2017.igem.org/Team:Waterloo/Model"><SPAN>Model</SPAN></A></LI></DIV><DIV class="content"><DIV class="titleBox row" style="background: url(undefined)"><DIV class="layer shade"><DIV class="titleCaption col-xs-4"><H1>
          HP Silver
        </H1></DIV></DIV></DIV><DIV class="row"><DIV class="col"><DIV class="content-main"><H3 id="-project-determination-"><STRONG>Project Determination</STRONG></H3><P>At the beginning our team season, every member gets to present possible project ideas. Follow this initial meeting, we spend the next couple of weeks flushing out the ideas in groups in more detail, looking into primary literature, designing possible experiments, and sourcing required materials. We then present these ideas again before selecting the project. To select our team project this year, we wanted to conduct  quantitative and qualitative analysis, so that we could come to a consensus in an efficient and fair manner.</P><P>To achieve this, we created a template for project selection that we used to evaluate all of our potential projects. The assessments were based on the criteria described in the iGEM judging guide, project assessment documentation from previous iGEM teams’ wikis, and anything else we felt is was import to evaluate.</P><P>Each team member was asked to score different qualities of the project (novelty, feasibility, interest, applications, and usefulness) and back up their quantitative evaluation with written comments. With the diverse disciplines amongst our team, we received input from a science, mathematics, engineering and business perspective. This analysis was then presented to our advisors to ensure that we had not overlooked any relevant aspect in our consideration.</P><P>Below we have included our general guide template with the justification for our choices in criteria, as well as our team’s evaluation of all the projects we pitched. This evaluation resulting in our team’s selection of this year’s project, the functional prion project now called: Prions be Lit - Functional Amyloid as a Biological Tool.</P><H3 id="-project-selection-guide-"><STRONG>Project Selection Guide</STRONG></H3><H3 id="-primary-criteria-evaluates-the-viability-of-a-project-"><STRONG>Primary Criteria: evaluates the viability of a project</STRONG></H3><H4 id="-novelty-"><STRONG>Novelty</STRONG></H4><P>If the proposed project was likely to fall under the Foundational Advance track, we added an additional primary criteria. The basis of this track is to develop novel solutions to technical problems, thus the novelty of the project was one of the priorities.</P><H4 id="-feasibility-"><STRONG>Feasibility</STRONG></H4><P>When developing project ideas, teams must also consider the amount of resources available to them. It is important to be able to achieve their goals with the resources, time, and technical abilities they have to ensure the project is completed and finished on time.</P><H3 id="-secondary-criteria-evaluates-benefits-of-selecting-a-particular-idea-as-an-igem-project-"><STRONG>Secondary Criteria: evaluates benefits of selecting a particular idea as an iGEM project.</STRONG></H3><H4 id="-interest-"><STRONG>Interest</STRONG></H4><P>The level of interest amongst the members in a project is valuable as a teams with a greater interest in their project are like more likely to excel and be successful.</P><H4 id="-applicability-of-the-team-"><STRONG>Applicability of the team</STRONG></H4><P>It is important to select a project that will enable all subteams to contribute to the  project and fulfill medal criteria.</P><H4 id="-usefulness-"><STRONG>Usefulness</STRONG></H4><P>How useful the project is and how it may impact society (the degree of stakeholder benefit and how many stakeholders would gain this benefit).</P><P>Following the team analysis, the project proposals were presented to advisors from the University of Waterloo’s Faculties of Biology and Math. Advisors from the University of Waterloo Faculty of Biology: Brian Ingalls, Trevor Charles, Barbara Moffatt, Andrew Doxey, and Forbes Burkowski. Their feedback and expertise was acknowledged and factored into our project decision as well.</P><H3 id="-criteria-weighting-out-of-5-"><STRONG>Criteria Weighting (out of 5)</STRONG></H3><P>Interest - /2
Application to Subteam - /3
Novelty - /5 <EM>*</EM>For Foundational Advancements only
Usefulness - /2
Impact - /2
Feasibility - / 5</P><TABLE><THEAD><TR><TH>Criteria</TH><TH>Functional Prion Project</TH></TR></THEAD><TBODY><TR><TD><B> Description of Project </B><UL><LI> What kind of problem are we solving? </LI><LI>How are we doing it? </LI></UL></TD><TD><UL><LI> It is observed that fusing proteins together typically reduces function, although it is possible to engineer the prions such that the function increases </LI><LI> Reconstruct DNA such that it creates a prion domain in a new protein, allowing it to aggregate with other prion proteins </LI></UL></TD></TR><TR><TD><B>Feasibility </B><UL><LI> Do we have the resources? </LI><LI> Do we have the experience? </LI></UL></TD><TD><UL><LI> 90% feasibility 7/10 or higher </LI><LI> Potential roadblock: not enough experience with yeast? </LI><LI> Profs will be around to help us learn </LI><LI> Last year’s project worked with prions, so we have experience with prions </LI><LI> High chance of working </LI></UL></TD></TR><TR><TD><B>Interest/Applicability to Subteam </B><UL><LI> Personal opinions </LI></UL></TD><TD><UL><LI> 63.4% interested 7/10 or more </LI><LI> 76.6% applicable to subteam 7/10 or more </LI></UL></TD></TR><TR><TD><B> Novelty </B><UL><LI> Did we build on other ideas? </LI></UL></TD><TD><UL><LI> The idea to specifically apply prion domain to other protein is something new in our understanding </LI><LI> We have previously worked with prions in 2016, but not exclusively on the prion domains </LI></UL></TD></TR><TR><TD><B> Impact/Usefulness/Why are we doing it? </B><UL><LI> Economic </LI><LI> Societal </LI><LI> Environmental </LI><LI> Ethical </LI><LI> Material </LI><LI> Emotional </LI></UL> Consider both positive and negative impacts</TD><TD><UL><LI> Understanding the aggregation mechanisms of prion could contribute in the discovery of new treatment against it </LI><LI> The aggregation of enzyme with prion domain could help increasing the speed and efficiency of biochemical reactions </LI><LI> The aggregate structure could be used to construct new biomaterial with additional durability, etc. </LI></UL></TD></TR><TR><TD><B> Alternative </B><UL><LI> Is there another more efficient/better way to go about solving the problem? </LI></UL></TD><TD><UL><LI> No viable alternatives discussed </LI></UL></TD></TR></TBODY></TABLE><TABLE><THEAD><TR><TH>Criteria</TH><TH>Oscillatory Fluorescence for Measurement</TH></TR></THEAD><TBODY><TR><TD><B> Description of Project </B><UL><LI> What kind of problem are we solving? </LI><LI>How are we doing it? </LI></UL></TD><TD><UL><LI> Fusing Green Fluorescent Protein (GFP) to a protein of interest is a common way to detect the expression of that protein </LI><LI> The idea is that when you shine a certain frequency of light on your sample, if you see that it’s glowing green, you know that the GFP is being expressed and so the protein it’s fused to is also being expressed </LI><LI> Problem: The problem is that if you want to measure how much a given protein is being expressed, you have to measure how much it is fluorescing, which is appreciably harder to do </LI><LI> Solution: Fluorescence is hard to measure whereas time is easy to measure. </LI><LI> If we can create a system where we see fluctuations in time rather than fluorescence, we can make a much easier technique for continuously collecting data on the level of expression of a protein over time </LI></UL></TD></TR><TR><TD><B>Feasibility </B><UL><LI> Do we have the resources? </LI><LI> Do we have the experience? </LI></UL></TD><TD><UL><LI> 86.6% feasibility 7/10 or higher </LI><LI> System involves many parts, chance that lab will have troubles with manipulating concentrations of components </LI><LI> Only working with E.Coli, no other organisms </LI><LI> Flow cytometry is widely used </LI><LI> General agreement that it’s feasible </LI></UL></TD></TR><TR><TD><B>Interest/Applicability to Subteam </B><UL><LI> Personal opinions </LI></UL></TD><TD><UL><LI> 56.7% interest 7/10 or more </LI><LI> 66.6% applicable to subteam 7/10 or more </LI></UL></TD></TR><TR><TD><B> Novelty </B><UL><LI> Did we build on other ideas? </LI></UL></TD><TD><UL><LI>  This project was inspired by Wageningen university 2013 iGEM team </LI></UL></TD></TR><TR><TD><B> Impact/Usefulness/Why are we doing it? </B><UL><LI> Economic </LI><LI> Societal </LI><LI> Environmental </LI><LI> Ethical </LI><LI> Material </LI><LI> Emotional </LI></UL> Consider both positive and negative impacts</TD><TD><UL><LI> Having to take samples and manually run tests for gene expression is a huge pain </LI><LI> Last year, one of the biggest problems we had was our limited data due to this restraint </LI><LI> When you don’t have enough data, it’s hard to confidently predict what is happening in your system </LI><LI> This project would make it significantly easier to collect large amounts of continuous data on gene expression which would be <EM>extremely</EM> useful for a huge number of synbio experiments </LI></UL></TD></TR><TR><TD><B> Alternative </B><UL><LI> Is there another more efficient/better way to go about solving the problem? </LI></UL></TD><TD><UL><LI> Measurement of LuxI - how does one know what the protein of interest is doing? (either fuse our protein of interest to LuxI or get the same promoter to control LuxI expression and the expression of protein of interest, but in the latter case we don’t know about protein degradation of our protein of interest, and the former is harder to do) </LI></UL></TD></TR></TBODY></TABLE><TABLE><THEAD><TR><TH>Criteria</TH><TH>Magnetic Bacteria</TH></TR></THEAD><TBODY><TR><TD><B> Description of Project </B><UL><LI> What kind of problem are we solving? </LI><LI>How are we doing it? </LI></UL></TD><TD><UL><LI> Ability for the bacteria to move throughout the body (flagella) </LI><LI> Ability to externally control the movement of bacteria </LI><LI> Specific apoptosis targeting of cancerous cells only </LI><LI> Ability to detect the bacteria </LI><LI> How to do this? Use mgFRN as a transcription factor to express a gene of interest </LI><LI> Cytotic gene only turned on during low oxygen levels </LI><LI> As tumor shrinks, levels of O2 increase turning off our system </LI></UL></TD></TR><TR><TD><B>Feasibility </B><UL><LI> Do we have the resources? </LI><LI> Do we have the experience? </LI></UL></TD><TD><UL><LI> 13.3% feasibility 7/10 or higher </LI><LI> Lack of time to complete this, slow growth rate of bacteria </LI><LI> Lack of resources, no access to mice </LI><LI> Lack of experience, would need more time to learn how to perform this in lab </LI><LI> Low feasibility in performing this with non-model organism </LI><LI> General agreement of low feasibility </LI></UL></TD></TR><TR><TD><B>Interest/Applicability to Subteam </B><UL><LI> Personal opinions </LI></UL></TD><TD><UL><LI> 80% interested 7/10 or more </LI><LI> 76.7% applicable to subteam 7/10 or more </LI></UL></TD></TR><TR><TD><B> Novelty </B><UL><LI> Did we build on other ideas? </LI></UL></TD><TD><UL><LI> We have rarely observed any project that have focused on magnetic bacteria </LI><LI> We believe that the idea of use magnetism as a guiding mechanism is relatively new and minimally developed </LI></UL></TD></TR><TR><TD><B> Impact/Usefulness/Why are we doing it? </B><UL><LI> Economic </LI><LI> Societal </LI><LI> Environmental </LI><LI> Ethical </LI><LI> Material </LI><LI> Emotional </LI></UL> Consider both positive and negative impacts</TD><TD><UL><LI> The magnetic properties could be utilized to improve targeted delivery of drugs </LI><LI> This will be promising in the treatment of on-site diseases such as cancer, infection </LI></UL></TD></TR><TR><TD><B> Alternative </B><UL><LI> Is there another more efficient/better way to go about solving the problem? </LI></UL></TD><TD><UL><LI> No viable alternatives discussed </LI></UL></TD></TR></TBODY></TABLE><TABLE><THEAD><TR><TH>Criteria</TH><TH>Zika</TH></TR></THEAD><TBODY><TR><TD><B> Description of Project </B><UL><LI> What kind of problem are we solving? </LI><LI>How are we doing it? </LI></UL></TD><TD><UL><LI> Our proposal aims to provide a novel method for controlling the prevalence of Zika virus (ZIKV) in Aedes mosquito populations through the use of genetically engineered Pichia, a yeast endosymbiont of Aedes </LI><LI> This measure utilizes two components: the expression of AXL, a tyrosine-protein kinase receptor thought to be required for ZIKV internalization; and a CRISPR/FnCas9 system, equipped with a guide RNA (gRNA) to target the RNA-dependent RNA polymerase (RdRP) encoding region of the ZIKV ssRNA genome </LI><LI> These constructs will be integrated into the Pichia genome, allowing the yeast to sequester ZIKV intracellularly and destroy the viral particles, effectively decreasing the risk of transmission from Aedes to humans </LI><LI> Since Pichia are naturally part of the Aedes gut microbiome, this process should not change the competitiveness of Aedes, resulting in minimal ecological consequences </LI></UL></TD></TR><TR><TD><B>Feasibility </B><UL><LI> Do we have the resources? </LI><LI> Do we have the experience? </LI></UL></TD><TD><UL><LI> 49.9% feasibility 7/10 or higher </LI><LI> No access to Zika or mosquitoes, our lab team would be performing very different experiments than what we actually want </LI><LI> Working with non-model organisms </LI><LI> The resources needed to develop this include different strains of yeast, dry pellets, CRISPR, zika like strands of RNA to implement as a proof of concept </LI><LI> Most certainly not enough experiment to implement this far reaching proof of concept design not yet tested </LI></UL></TD></TR><TR><TD><B>Interest/Applicability to Subteam </B><UL><LI> Personal opinions </LI></UL></TD><TD><UL><LI> 83.4% interest 7/10 or more </LI><LI> 73.4% applicable to subteam 7/10 or more </LI></UL></TD></TR><TR><TD><B> Novelty </B><UL><LI> Did we build on other ideas? </LI></UL></TD><TD><UL><LI> The idea of using a yeast species employed with CRISPR is an interesting and novel idea, albeit it is inspired by research papers </LI><LI> However, we would be really excited with the opportunity to work with these </LI></UL></TD></TR><TR><TD><B> Impact/Usefulness/Why are we doing it? </B><UL><LI> Economic </LI><LI> Societal </LI><LI> Environmental </LI><LI> Ethical </LI><LI> Material </LI><LI> Emotional </LI></UL> Consider both positive and negative impacts</TD><TD><UL><LI> Protect people in Latin American countries from being infected with zika virus without harming the mosquito population and surrounding ecosystem </LI><LI> Deployment is intuitive and fast to implement with closed off bodies of water and pellets of yeast thrown into </LI></UL></TD></TR><TR><TD><B> Alternative </B><UL><LI> Is there another more efficient/better way to go about solving the problem? </LI></UL></TD><TD><UL><LI> Wolbachia bacteria have been used similar studies, but have not implemented CRISPR, nor would this experiment work to use a human AXL receptor inside a prokaryote cell </LI></UL></TD></TR></TBODY></TABLE><H3 id="-conclusion-"><STRONG>Conclusion:</STRONG></H3><P><STRONG>In conclusion, the Waterloo iGEM 2017 team has decided to select the engineered prion project for this year.</STRONG></P></DIV></DIV></DIV></DIV><DIV class="footer"><H2>Waterloo iGEM 2017</H2><P>© 2017 Waterloo iGEM</P></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>