<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Waterloo_Integrated_Practices skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Waterloo/Integrated Practices</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><NAV class="navbar navbar-toggleable-md navbar-inverse bg-primary fixed-top"><A class="navbar-brand" href="https://2017.igem.org/Team:Waterloo">Waterloo</A></NAV></P><DIV class="collapse navbar-collapse" id="navbarCollapse"><UL class="navbar-nav mr-auto"><LI class="nav-item"><A class="nav-link" href="https://2017.igem.org/Team:Waterloo"><SPAN>Home</SPAN></A></LI><LI class="dropdown nav-item"><A href="#" class="nav-link dropdown-toggle nav-link" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Team</A></LI><LI class=""><A class="dropdown-item" href="https://2017.igem.org/Team:Waterloo/Team"><SPAN>Team</SPAN></A></LI><LI class=""><A class="dropdown-item" href="https://2017.igem.org/Team:Waterloo/Collaborations"><SPAN>Collaborations</SPAN></A></LI></UL><LI class="dropdown nav-item"><A href="#" class="nav-link dropdown-toggle nav-link" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Project</A></LI><LI class=""><A class="dropdown-item" href="https://2017.igem.org/Team:Waterloo/Contribution"><SPAN>Contribution</SPAN></A></LI><LI class=""><A class="dropdown-item" href="https://2017.igem.org/Team:Waterloo/Description"><SPAN>Description</SPAN></A></LI><LI class=""><A class="dropdown-item" href="https://2017.igem.org/Team:Waterloo/Design"><SPAN>Design</SPAN></A></LI><LI class=""><A class="dropdown-item" href="https://2017.igem.org/Team:Waterloo/Experiments"><SPAN>Experiments</SPAN></A></LI><LI class=""><A class="dropdown-item" href="https://2017.igem.org/Team:Waterloo/Proof"><SPAN>Proof</SPAN></A></LI><LI class=""><A class="dropdown-item" href="https://2017.igem.org/Team:Waterloo/Demonstrate"><SPAN>Demonstrate</SPAN></A></LI><LI class=""><A class="dropdown-item" href="https://2017.igem.org/Team:Waterloo/Results"><SPAN>Results</SPAN></A></LI><LI class=""><A class="dropdown-item" href="https://2017.igem.org/Team:Waterloo/Notebook"><SPAN>Notebook</SPAN></A></LI><LI class="nav-item"><A class="nav-link" href="https://2017.igem.org/Team:Waterloo/Parts"><SPAN>Parts</SPAN></A></LI><LI class="nav-item"><A class="nav-link" href="https://2017.igem.org/Team:Waterloo/Safety"><SPAN>Safety</SPAN></A></LI><LI class="nav-item"><A class="nav-link" href="https://2017.igem.org/Team:Waterloo/Attributions"><SPAN>Attributions</SPAN></A></LI><LI class="active dropdown nav-item"><A href="#" class="nav-link dropdown-toggle nav-link" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Human Practices</A></LI><LI class=""><A class="dropdown-item" href="https://2017.igem.org/Team:Waterloo/Human_Practices"><SPAN>Human Practices</SPAN></A></LI><LI class=""><A class="dropdown-item" href="https://2017.igem.org/Team:Waterloo/HP/Silver"><SPAN>Silver</SPAN></A></LI><LI class=""><A class="dropdown-item" href="https://2017.igem.org/Team:Waterloo/HP/Gold_Integrated"><SPAN>Gold</SPAN></A></LI><LI class="active"><A class="dropdown-item" href="https://2017.igem.org/Team:Waterloo/Integrated_Practices"><SPAN>Integrated Practices</SPAN></A></LI><LI class=""><A class="dropdown-item" href="https://2017.igem.org/Team:Waterloo/Engagement"><SPAN>Engagement</SPAN></A></LI><LI class=""><A class="dropdown-item" href="https://2017.igem.org/Team:Waterloo/Snake35"><SPAN>Snake35</SPAN></A></LI><LI class="dropdown nav-item"><A href="#" class="nav-link dropdown-toggle nav-link" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Awards</A></LI><LI class=""><A class="dropdown-item" href="https://2017.igem.org/Team:Waterloo/Software"><SPAN>Software</SPAN></A></LI><LI class=""><A class="dropdown-item" href="https://2017.igem.org/Team:Waterloo/Model"><SPAN>Model</SPAN></A></LI></DIV><DIV class="content"><DIV class="integratedPractices"><DIV class="row"><DIV class="col"><DIV class="content-main"><H1 id="-policy-practices-"><STRONG>Policy &amp; Practices</STRONG></H1><P>Human practices work was central in so many aspects of our project. The work we completed in human practices was integral to the direction of the wet lab portion of the project Our work was also  constantly influenced by the data and information we collected when interacting with stakeholders, the public, and experts.</P><P>To make our project accessible to our community, we designed an implicit association test or “IAT” to evaluate potential unconscious bias towards technical terms related to our project which guided the way we communicated our work.</P><P>Waterloo iGEM integrated novel and creative ideas to help bridge knowledge barriers when explaining foundational advance projects to relevant stakeholders by creating the game <A href="https://2017.igem.org/Team:Waterloo/Snake35">Snake 35</A> as a fun and interactive experience to explain our project in a safe and novel way.</P><P>We also uncovered problems common to several local iGEM teams at the Ontario iGEM(oGEM) meetup. We realized that many teams struggle communicating the learning curve required in joining an iGEM Human Practices team. In response we created a policy &amp; practices handbook that broke down many aspects of what it means to be  an iGEM policy team member and how to successfully integrate human practices into an iGEM project. We hope this guide will be helpful to many future iGEM teams.</P><P>We also alleviated financial costs for our team by creating 3D printing files of some essential lab equipment which is available for all iGEM teams to access.</P><P>As well, we documented and conducted interviews throughout the year as cited in the <A href="https://2017.igem.org/Team:Waterloo/HP-Gold">Gold Medal</A> page. These interviews with experts helped influence the design of the project, additional applications, and the direction of social media campaign.</P></DIV></DIV></DIV><DIV id="inserts"><CENTER><DIV class="ip-group-wrapper"><DIV class="btn-group btn-group-lg" role="group" aria-label="Integrated Practices"><SPAN class="btn-group btn-group-lg" v-for="item in ip.items"><BUTTON class="btn btn-primary" v-on:click="ip.showing=item.title"> ^^item.title$$  </BUTTON></SPAN></DIV></DIV></CENTER><DIV class="row" v-if="ip.showing=='Guidebook'"><DIV class="row"><DIV class="col"><DIV class="content-main"><P>Waterloo iGEM created a handbook about Human Practices and our Policy and Practices (PNP) subteam in hopes to help future iGEM teams understand the role of Human Practices in science. In this guide, we discuss the iGEM Competition, what makes a great project, PNP medal criteria, and other side projects we work on. Come the beginning of the competition year, this guide will be given to our future team members as a tool to help them get a better understanding of our their roles as part of the team. Continue reading and check out what we’ve written.</P><H1 id="-center-introduction-to-policy-and-practices-pnp-center-"><CENTER> Introduction to Policy and Practices(PNP) </CENTER></H1><H1 id="-center-igem-2017-center-"><CENTER> iGEM 2017 </CENTER></H1><H4 id="-center-university-of-waterloo-igem-2017-center-"><CENTER>  University of Waterloo iGEM,  2017 </CENTER></H4><P><EM>Welcome to Waterloo’s iGEM Policy and Practices team! This guide is designed to help members understand what our team is about and the logistics behind our roles for the project. In the next few pages, this guide will focus on the i    GEM competition, responsibilities of PNP, how we work to fulfil those responsibilities, and the selection process of our annual project.</EM></P><H4 id="what-is-the-igem-competition-">What is the iGEM Competition?</H4><P>The International Genetically Engineered Machines (iGEM) Foundation is a non-profit organization dedicated to the education and advancement of synthetic biology, as well as promoting an open science community and collaboration.</P><P>One of their three main programs is the annual iGEM Competition, also known as the Jamboree, which is hosted in Boston every year. Student teams around the world build genetically engineered systems using standard biological parts called BioBricks and showcase their parts at the Jamboree, competing for various awards and prizes.</P><P>iGEM projects are organized into specific subject areas. The standard ten parts-based tracks are as follows: Diagnostics,  Energy, Environment, Food &amp; Nutrition, Foundational Advance, High School, Information Processing, Manufacturing, New Applications, and Therapeutics. Additionally there are four Special Tracks, which are showcased in a separate space at the Jamboree and although are not eligible for the Grand Prize.</P><P>At the Jamboree, iGEM teams will be competing for various awards, including Medals, Track Awards, and Special Awards. Any number of teams can win one of three levels of medals: Bronze, Silver and Gold. Each level of award has its own specific medal criteria, and teams must prove to judges that they have fulfilled them all to earn the medal.  </P><P>iGEM teams can also win other special awards. While any number of teams can be awarded medal standards, only one team can win each of the following awards:</P><UL><LI><STRONG><EM>The Grand Prize:</EM></STRONG> awarded to one team based on the overall
   excellence of their entire project</LI><LI><STRONG><EM>Standard Track:</EM></STRONG> awarded to the team with the best project in
   each Standard Track</LI><LI><STRONG><EM>Special Track Awards:</EM></STRONG> awarded to the team with the best
   project in each Special Track</LI><LI><STRONG><EM>Special Prizes:</EM></STRONG> awarded to honour unique and innovative
   contributions to iGEM including: Best Poster, Best   Presentation, Best Wiki, Best New Basic Part, etc...</LI></UL><H5 id="what-makes-igem-projects-great-">What Makes iGEM Projects GREAT?</H5><P>The first eight aspects are the key iGEM values that apply to all teams, irrespective of
track. The final two aspects are distinct for standard(parts-based) tracks and special
(non-parts-based) tracks.</P><OL><LI>How impressive is this project?</LI><LI>How creative is the team’s project?</LI><LI>Did the project work?</LI><LI>How much did the team accomplish?</LI><LI>Is the project likely to have an impact?</LI><LI>How well are engineering principles used?</LI><LI>How thoughtful and thorough was the team’s consideration of human practices?</LI><LI>How much of the work did the team do themselves and how much was done by others?</LI><LI>Did the team design a project based on synthetic biology and standard parts?</LI><LI>Are the functions and behaviors of the parts well-documented in the Registry?</LI></OL><H4 id="what-is-pnp-">What is PNP?</H4><P>While the Lab and Math Teams are in charge of conducting experiments in the development of the project, the role of PNP is to support them by assessing the non-technical aspects that are just as important as purely getting results. In addition, we are responsible for connecting with the public to generate more interest and awareness in the field of synthetic biology.</P><H5 id="why-is-pnp-important-">Why is PNP important?</H5><P>PNP is responsible for bridging the world of science to the rest of the global community. Science does not exist in isolation, rather it is a player within the larger local, national, and global communities. Within the science community, we are responsible for informing stakeholders (those who are directly affected by our project) about the work being done and evaluate the logistical executions of how to implement the project commercially. Together, this is what it means to go “beyond the bench” (this phrase has been used to describe PNP in the past, however without a further explanation it can be an unhelpful and confusing description).</P><P>PNP and the Lab Team work hand-in-hand. In every step of the project, teams should be considering the policy and practice side of things. This includes:</P><UL><LI>Before beginning an experiment to evaluate any potential work</LI><LI>Every time you go to work on your project throughout the year, especially as the project develops throughout the build season</LI><LI>Following the completion of the project as you release it into the world</LI><LI>When you come back and refine your project to evaluate the reaction and reception from the public</LI></UL><H4 id="pnp-medal-criteria">PNP Medal Criteria</H4><P>Each subteam has their own aspect of the medal criteria that they focus on throughout the year. PNP focuses on (but is not strictly limited to) the following criteria:</P><P><STRONG>*Collaborations(silver):</STRONG> Convince the judges you have significantly worked with any other registered iGEM team in a meaningful way. For example, mentor a team, characterize a part, troubleshoot a project, model/simulate a system or validate a software/hardware solution to a SynBio problem, or be the recipient of any of these activities.*</P><P><STRONG>*Human Practices (silver):</STRONG> Convince the judges you have thought carefully and creatively about whether your work is safe, responsible, and good for the world. You could accomplish this through engaging with your local, national and/or international communities or other approaches. (Exceptional outreach and education work is recognized by the Education &amp; Public Engagement Special Award)*</P><P><STRONG>*Integrated Human Practices (gold):</STRONG> Expand on your silver medal activity by demonstrating how you have integrated the investigated issues into the design and/or execution of your project. (Exceptional Human Practices is recognized by the Best Integrated Human Practices award)*</P><P><EM>The current iGEM rubric contains four aspects for evaluating Integrated Human Practices:</EM></P><OL><LI><EM>Was their work integrated into their project?</EM></LI><LI><EM>Does it serve as an inspiring example to others?</EM></LI><LI><EM>Is it documented in a way that others can build upon?</EM></LI><LI><EM>Was it thoughtfully implemented (i.e., did they explain the context, rationale, prior work)?</EM></LI></OL><H4 id="what-projects-need-to-be-accomplished-in-pnp-">What Projects Need to be Accomplished in PNP?</H4><P><STRONG>Project Assessment:</STRONG></P><P><EM>Is your work safe, responsible, and good for the world?</EM></P><P>It is critical to evaluate how science can affect the people who will be implementing it and the people outside of the science community. According to the 2016 judging handbook, teams should “pursue questions relating to regulatory, economic, ethical, social, legal, philosophical, ecological, security, or other societal questions relating to synthetic biology”. This is where PNP comes into play. Referring to the Integrated Human Practices medal criteria, projects should be developed with local and/or global considerations. A way to achieve this is consult various stakeholders, experts, and the team throughout the design, execution, and presentation of their project.</P><P>In addition, PNP teams up with Lab to come up with potential applications for the main project idea using engineering principles, where the motivation for potential applications of the project must clearly defined. For example, we could take the overall project and explore how it interacts in different environments, how it can be implemented in different situations, how we can alter it slightly to gain new/different functions, how it can be combined with other project, etc... Project Assessment is particularly important as it fulfils both silver and gold criteria.</P><P>Example Projects:</P><UL><LI>The 2017 team created a catalogue of 3D printable lab equipment to increase customizability and access to printable lab wear and decrease cost to teams.</LI></UL><UL><LI><P>Waterloo iGEM 2017 also wanted to indirectly gauge the public’s opinion on the project through the use of an Implicit Association Task (IAT). This task can show if people have an implicit, or subconscious, bias against or towards certain words. By utilizing words that relate to our lab project in the IAT, we can determine which words are most likely to provoke a subconscious reaction in the general public and use this information to tailor the presentation of our project to decrease public bias.</P></LI><LI><P>Waterloo iGEM’s 2017 project were engineered prions that had enhanced function when in the amyloid state, and the team investigated the use of our engineered proin joining aptamers (oligonucleotides) to the prion domain amplify the detection rate of viruses.</P></LI></UL><P><STRONG>Open Science:</STRONG></P><P>Another important aspect in the design of a project is open science is to documented in a way that others can build upon: improving the flow of information, minimizing restrictions on the use of intellectual resources, and increasing transparency of research practice. With a scientific field as new and novel as synthetic biology, it is imperative that all accomplishments and research is open for all teams, research groups and industries to access in order to advance and progress the field forward as efficiently as possible.</P><P><STRONG>Public Engagement:</STRONG></P><P>Education &amp; Public Engagement allows for outreach and education work regarding synthetic biology. Great public engagement projects will focus on establishing a dialogue or sparking new scientific interest in the synthetic biology community in general (and not your specific project). This criteria is not limited to just members of the policy team - all members of iGEM should get involved.</P><P>The current iGEM rubric contains four aspects for evaluating Education &amp; Public Engagement:</P><OL><LI>Did their work establish a dialogue?</LI><LI>Does it serve as an inspiring example to others?</LI><LI>Is it documented in a way that others can build upon?</LI><LI>Was it thoughtfully implemented (i.e., did they explain the context, rationale, prior work)?</LI></OL><P>Example Projects:</P><UL><LI>Waterloo iGEM 2015 gave weekly sessions to children at ESQ (Engineering Science Quest), a children’s camp on the Waterloo campus.</LI><LI>Waterloo iGEM runs a synthetic biology workshop for top high school students attending SHAD Valley in Waterloo. In this workshop, we introduced concepts within synthetic biology, conducted lab experiments, and discussed social aspects of science in general. Our curriculum is available in the 2017 PNP Google Drive and (hopefully) on the wiki too</LI><LI>Waterloo iGEM maintains several social media platforms including Facebook, Instagram, Twitter, Youtube, and a team webpage</LI></UL><H4 id="collaborations">Collaborations</H4><P>In iGEM, the idea of significance plays a vital role in determining the validity of contributions for medal criteria. As outlined by iGEM HQ, we believe that a collaboration or contribution is significant if it betters a project or a team, ultimately contributing towards its advancement. That being said, significance is also largely subjective because what one judge may see as beneficial, another may believe it to be too passive of an effort to qualify.</P><P>Collaboration is not specific to PNP. All subteams should collaborate with other teams to improve different aspects of projects. Examples of how Waterloo iGEM has helped out new teams entering iGEM providing web construction technical support, providing a guide to accelerate new development, and assisting with the development of their wiki pages. In addition, Waterloo participates in an annual oGEM conference for iGEM teams in Ontario. At the conference, teams all share their ongoing projects, critique the projects of others, and allow them to network with each other and develop ideas for collaborations.</P><H4 id="project-determination">Project Determination</H4><P>The 2016 Imperial College team inspired us to create a structured method to select the annual project for the team. The typical selection process for the project begins with some of our members coming up with ideas and presenting them to the whole team. From there, members across the entire team conduct individual assessments, which get compiled to produce a quantitative analysis of which project the team believed was the best to pursue. The assessments were based on the set of criteria described as follows:</P><H5 id="-primary-criteria-"><EM>Primary Criteria:</EM></H5><UL><LI>Novelty: As the focus of the competition is the advancement of synthetic biology, teams must either build upon a previous project or come up with an original idea</LI><LI>Feasibility: It is important to consider the amount of resources available to teams such that they can produce a complete project</LI></UL><H5 id="-secondary-criteria-"><EM>Secondary Criteria:</EM></H5><UL><LI>Interest: Teams with a greater interest in their project are like more likely to excel above and beyond</LI><LI>Applicability to Subteams: It is important to select a project that will enable all subteams to work on their respective aspects of the project to fulfil medal criteria</LI><LI>Impact/Usefulness: The degree to which how useful the project is and how it can impact society (how stakeholders benefit and how many stakeholders are benefitted).</LI></UL><P>Each of the criteria are assigned a weight for each track, in an attempt to reflect how different tracts emphasize different criteria to varying degrees. For example, if we’re in the Foundational Advance and New Application tracks, the novelty of the project is much more important).</P><P>The general criteria weighting is described as follows:</P><P>Feasibility - /5Novelty - /4Application to Subteam - /3Impact - /2Usefulness - /2
Interest - /2</P><P>The projects with highest scores are taken to our iGEM advisors from the Faculty of Biology. The advisors offer technical advice, as well as suggestions based on their experience advising previous iGEM teams. The final decision is made after consideration of all of these evaluations.</P><H5 id="team-wiki">Team Wiki</H5><P>As part of the iGEM Competition Requirements, each team is responsible for constructing a wiki site to demonstrate their projects. In the later half of the year, our team puts together a Wiki Taskforce, who are members across the whole team who are responsible for creating the layout, content and the code for the wiki page. Each subteam focuses on writing about their own aspects of the project. For PNP, this means we create the pages for Human Practices, Public Engagement, Project Assessment, and other notable projects that we work on throughout the year. We also contribute to pages that are general to the whole project including project design, online notebook, and safety assessment; wherever we can contribute.</P><P>The main tasks that come with Wiki building are the design of the page’s physical appearance, the writing of content to fill the pages with, and the coding of the page. The team tries to keep a consistent theme throughout the wiki through the use of the same colour scheme, the same designs, similar use of images, and other visual aspects. Recently, the team has begun using a program called sketch to design graphics, and in our most recent wiki - 2017 - we have incorporated designs into many of our wiki pages. While the development of content is specific to each subteam and the work they did, there is some content that can be contributed to by any team member. This includes outlines of collaboration, descriptions of the team and individual bios. Finally, the actual code behind the page is headed by the math and modelling team, but any team member is welcome to contribute according to their ability.</P><P>Wiki development is a stressful process with a very strict time constraint as iGEM sets a wiki freeze day that all teams are subject to. We hope that you are up to the challenge, and look forward to seeing all that you contribute.</P><H5 id="conclusion-what-s-next-">Conclusion - What’s Next?</H5><P>So what’s next? Well, for the remainder of the Winter Term, Waterloo iGEM will be coming up with this year’s project idea. Anyone can make a project proposal, so if you have an idea that you want to pitch forward, go for it! On the PNP side of things to do, we’ll start thinking about how we want to tackle our medal criteria, whether it’s building off of tasks from last year or coming up with new ideas altogether. The Spring Term is when all subteams really start working on their respective tasks until the Jamboree in October/November.</P><P>So there you have it! Hopefully after reading this guide you have a better understanding of the iGEM Competition and what our responsibilities are as the Policy and Practices subteam. We strive to maintain an open environment to promote discussion and collaboration amongst ourselves, but most importantly, we want everyone to have a positive experience as part of Waterloo iGEM. In addition, it is imperative that you have a great summer - it is bronze medal criteria. We look forward to hearing your ideas and input this year, and don’t forget; feel free to ASK anything if you need clarification.</P><P><EM>Finally, add to this guide, distribute it to other teams, and make it your own so that teams in the future can benefit from all you learn.</EM></P><P>Welcome to the team, and we look forward to working with you!</P></DIV></DIV></DIV></DIV><DIV class="row" v-if="ip.showing=='Applications'"><DIV class="row"><DIV class="col"><DIV class="content-main"><H2 id="applications-of-prion-project">Applications of Prion Project</H2><P>The Waterloo iGEM 2017 project is a proof of concept for a system that has an application in increasing the speed of enzymatic reactions in bulk industrial reactors. This highlighted application is in the spotlight as the most practical, useful and feasible application possible based on this project.</P><P>In enzymatic pathways, the product of one enzyme must be passed onto the next enzyme in the pathway. In cells, related enzymes are often close to each other and so this is not a concern. However, if performing such a reaction in vitro the proteins would on average be much further apart due to their paths following random walks. This slows the speed of the reaction.</P><P>Furthermore, it has been demonstrated that decreasing the distance between enzymes is sufficient to increase their interaction and decrease the time taken for intermediates to move between them. Knowing this, a system that brings enzymes close together should be able to speed up a reaction in vitro.</P><P>By attaching Sup35’s prion domain to different associated proteins and then inducing aggregation, this goal can be accomplished. It has been established that it is possible to attach prion domains to proteins while retaining the functions of those proteins in aggregates, so the addition of the tag should not be a problem. Additionally, production of these tagged proteins can be performed E. coli, and the subsequent extraction of the proteins and induction of aggregation should be relatively simple.</P><P>There are a few existing techniques that one could use to bring proteins close together for this purpose, but this method has several advantages over them.</P><P>One group has proposed bringing enzymes together by anchoring them to the surface of a fat micelle. However, such a micelle could not be gathered from a solution easily and reused; the Sup35 domain aggregates can be centrifuged out of solution, which is a simple process that should allow them to be reused.</P><P>Another potential solution is using an engineered scaffold protein to latch onto the different enzymes in a system. One downside of this solution are that it requires that one make the enzymes in the system compatible with the scaffold by tagging them. While this is required in our solution as well, the tags are all the same and so the process of adding them is less complex. Additionally, one does not need to make a separate protein to serve as a scaffold. Another downside is, as with the last solution, that reusing the enzymes on a scaffold protein may be difficult. A third downside is that one is limited in how many enzymes can be added to the scaffold. With the Sup35 tag based system, any number of proteins can be tagged and added to the aggregate.</P><P>In summary, our solution is feasible, and due to its versatility and the reusability is better than existing solutions. One specific example of a way this system could be used would be in conjunction with Guelph iGEM’s 2017 project; the project is based on a pair of enzymes that can be used to remove calcium oxalate from brewing containers. Because the enzymes work together, bringing them onto the same aggregate may help them work together and speed up the removal process.</P><P>There are more secondary applications that have been brainstormed based on consultations and interviews from experts we have contacted as well as other research papers read thoroughly. These have been formated showing each application’s general concept, how the application is used, the benefits and restrictions respectively.</P><H2 id="vaccine-preparation">Vaccine Preparation</H2><H4 id="general-concept">General Concept</H4><P>This foundational advance also has the opportunity to be introduced in vaccine preparation. Using prion aggregates or prion-like aggregated proteins, we could make a more effective vaccine to dissolve at a slower rate once introduced into the body. This works much to the same therapeutic effect as the lipid molecule squalane or other liposome encapsulated nanoparticles (Fox, 2009). The aggregated proteins are developed and used in vitro via biotechnology scale up methodologies.</P><H4 id="application">Application</H4><P>This can allow for a slow release of the vaccine being degraded at a slower and steady rate. Due to the variability of disease causing agents used in different vaccines, different prion-like proteins could be designed to adapt to each individual vaccine for optimal sustained degradation.</P><H4 id="benefits">Benefits</H4><P>Such a therapeutic could then be licensed off to larger biopharmaceutical firms with enough capital to sustain and develop a large enough commercial market to make a global difference in the innovative studies of vaccine preparation. But why is this therapeutic important? Sustained degradation of vaccines furthermore leads to fewer vaccine administration dosages. This means less cost for pharmaceutical firms to develop, and also gives a lower cost to the patient as well consuming fewer dosages.</P><H4 id="restrictions">Restrictions</H4><P>Vaccine preparation and administration is already quite established within the biopharmaceutical market with minimal doses already given for many vaccines. Such innovation will be only given in niche vaccine drug deliveries based on the the restrictive properties of each pathogen coated with its respective prion-like proteins.</P><H4 id="references">References</H4><P>Fox, C. B. (2009). Squalene emulsions for parenteral vaccine and drug delivery. Molecules, 14(9), 3286-3312.</P><H2 id="increased-efficiency-of-antimicrobial-peptides">Increased Efficiency of Antimicrobial Peptides</H2><H4 id="general-concept">General Concept</H4><P>With the recent rise of antibiotic superbugs, alternative approaches to eliminate harmful bacteria are necessary. In 2015, researchers from Leuven, Belgium conducted a study to investigate the use of protein aggregates as a measure to kill bacteria. They proposed that the introduction of engineered antimicrobial peptides coding aggregate-prone sequences would induce bactericidal. Antimicrobial peptides (AMP) are known peptides that disrupt bacterial cell membrane and induce cell lysis. The aggregation of AMP is expected to not only amplify its efficiency but also corrupt proteolysis in bacteria, completely rendering their vital processes. Additionally, AMP has a multi-target mode of action, which makes it difficult for bacteria to develop resistance against them. More importantly, AMP only binds to bacterial transporter protein not found in mammalian cells, which makes them harmless to animals.</P><H4 id="application-and-benefit">Application and Benefit</H4><P>With Waterloo 2017 iGEM team’s project, the fusion of prion domain with AMP can be achieved to effectively increase its aggregation in vivo. As a result, the efficiency of AMP activity will be magnified, which would improve its function as a bactericidal agent.  </P><H4 id="restrictions">Restrictions</H4><P>The mechanism of AMP remains a relative mystery to us, which would hinder our progress in designing this method. Additionally, S. aureus is seemingly resistance against AMP due to possession negatively charged lipid in its membrane.</P><H4 id="references">References</H4><P>Protein aggregation as an antibiotic design strategy. (2015). Molecular Microbiology, 99(5), 849–865. Retrieved from <A href="http://onlinelibrary.wiley.com/doi/10.1111/mmi.13269/full">http://onlinelibrary.wiley.com/doi/10.1111/mmi.13269/full</A></P><H2 id="nanosensors-application">Nanosensors Application</H2><H4 id="general-concept">General Concept</H4><P>We could take our prion aggregate and use it as a nanoparticle. By equipping the aggregate with antibodies for a specific protein, we can use it to identify the presence of the target protein to various concentrations as well (a separate test done afterwards).</P><H4 id="application">Application</H4><P>The whole concept of nanosensors is fascinating itself because it reduces the amount of time needed to identify the presence of bacteria such as E. Coli. This is also perfect for using in labs to enhance identification speed.</P><H4 id="benefits">Benefits</H4><P>The price of around 20 nm nanogold is approximately $500 per 1mL of solution. Our application would be more cost effective than synthesizing gold nanoparticles. We could hence make large structures of proteins.</P><H4 id="restrictions">Restrictions</H4><P>Storing conditions may affect the feasibility of using it in the field.</P><H4 id="references">References</H4><P>Gold nanoparticles. (n.d.). Retrieved October 28, 2017, from <A href="http://www.sigmaaldrich.com/catalog/substance/goldnanoparticles1969798765?lang=en®ion=CA">http://www.sigmaaldrich.com/catalog/substance/goldnanoparticles1969798765?lang=en®ion=CA</A></P><P>Gold nanoparticle-based immunochromatographic test for identification of Staphylococcus aureus from clinical specimens. (2006, May 23). Retrieved from <A href="http://www.sciencedirect.com/science/article/pii/S0009898106003305">http://www.sciencedirect.com/science/article/pii/S0009898106003305</A></P><H2 id="increased-efficiency-of-production-and-purification-of-peptides">Increased Efficiency of Production and Purification of Peptides</H2><H4 id="general-concept">General Concept</H4><P>Our project considers the use of prion domains to increase the function of another protein via fusion due to the aggregating abilities of the prion domain. Research done by a team at Tsinghua University in Beijing used ELK16, which forms “highly active enzyme aggregates”, and inteins, also known as protein introns, to reduce the cost and time associated with the production and purification of medium to large sized peptides [1].
The C terminus of the protein you want to purify is fused to the N terminus of the intein, and the C terminus of the intein is fused to the N terminus of ELK16[1]. The ELK16 regions form insoluble aggregates, which are easily isolated via centrifugation. A reagent induces cleavage at the N-termini of the intein, causing the isolated insoluble unit to shed the target protein, which are then isolated via reverse phase-high performance liquid chromatography.</P><H4 id="benefits">Benefits</H4><P>The potential use of prion domains instead of ELK16 could further reduce the costs associated with this process by allowing for the reuse of the intein-aggregation domain.</P><H4 id="application">Application</H4><P>This could be achieved by inducing the prion domains into their normal state, and using expressed protein ligation to attach new target proteins to the N-termini of the inteins.</P><H4 id="restrictions">Restrictions</H4><P>The question that must be asked is whether the reduction of costs gained in the reuse of the intein-aggregation domain will offset the increase in cost associated with protein ligation. It essentially boils down to what would be more expensive, the act of producing new fusion proteins, or the act of de-aggregating and reusing your existing fusion proteins.</P><H4 id="references">References</H4><P>[1] Zhao, Q., Xu, W., Xing, L., &amp; Lin, Z. (2016). Recombinant production of medium- to large-sized peptides in Escherichia coli using a cleavable self-aggregating tag. Microbial Cell Factories, 15, 136. <A href="http://doi.org/10.1186/s12934-016-0534-3">http://doi.org/10.1186/s12934-016-0534-3</A></P><H2 id="targeted-apoptosis-application">Targeted Apoptosis Application</H2><H4 id="general-concept">General Concept</H4><P>The misfolding of proteins leads to the formation of prions, aggregates of misfolded proteins that crowd within a cell, up to the point where the cell undergoes apoptosis (cell death) due to overcrowding. While prions do cause diseases in humans and other animals, and are rightly considered dangerous, there are some circumstances for which overcrowding is a desired feature of a treatment system.</P><P>The general idea is to take advantage of the “natural apoptotic process of prokaryotes” which was coined as “proapoptosis” by researcher Alaya Hochman in 1997. In summary, the paper claims that the execution of proapoptosis requires the activation of a developmental program that results in massive structural and morphological changes. These changes may include cell shrinkage, chromatin condensation, and proteolysis and DNA degradation, followed by cell fragmentation into apoptotic bodies that are phagocytized, just like eukaryotes in the standard apoptotic process. Bacteria currently exhibit a range of natural developmental programs for survival purposes. For example, <EM>Caulobacter cereus</EM> forms swarmer cells and differentiation in order to adapt to its stressful environment. If we are interested in activating a developmental program instead of using a targeted payload system as described above, we would need to formulate a system that is possible to be activated by the bacterium itself, and this would possibly involve similar chemical pathways. It would be of interest at this point to investigate the possible chemical pathways that our prion-like proteins can take, and this would be a good first step in the design process.</P><H4 id="application-and-benefits">Application and Benefits</H4><P>There was a study done by Eric Kandel and Kausik Si which determined that cytoplasmic polyadenylation element-binding protein (CPEB), an RNA binding protein, had a role to play in memory storage. CPEB has two conformations, the native one and the non-native (prion-like) conformation. This non-native conformation is capable of acting like a prion, such that it changes conformation to become multimeric and self-perpetuates this by inducing other CPEB to change conformation, as shown in yeast and the sea slug Aplysia. It was shown, in Aplysia, that this prion-like form was what marked active synapses to store long-term memories. A similarly functioning protein, Orb2, was found in drosophila, so it is possible this may have a human homologue. Knowing this, we may be able to use our own knowledge of prions to help treat diseases concerning long-term memory loss, such as Alzheimer’s or amnesia.</P><H4 id="restrictions">Restrictions</H4><P>As previously mentioned, this system requires activation by the strain of bacteria that is being targeted. If there is no natural pathway for the strain to activate the system on its own, a supplementary system would have to be created to support this action. Another restriction with this system is finding compatible chemical pathways between our prions and the strain of interest. If there is little compatibility, even with the prion with the best match, we would be forced to catalyze the process with an additional set of reactions.</P><H4 id="references">References</H4><P>Bratkovik, Hafner. (2017). Prions, prionoid complexes and amyloids: the bad, the good and the something in between. Swiss Medical Weekly, 147 doi: 10.4414/smw.2017.14424</P><H2 id="ethanol-fermentation-pathway-using-aggregation">Ethanol Fermentation Pathway using Aggregation</H2><H4 id="general-concept">General Concept</H4><P>One application of our project is that it could help speed up fermentation processes in the production of ethanol. To create ethanol, we start off with complex sugars and yeast; the yeast then breaks the complex sugars down into glucose, and eventually into ethanol and carbon dioxide (Lin et. al, 2005).</P><P>With the help of our project, we could use aggregation to speed up the fermentation pathway. To do this, however, we use monomeric enzymes, as it has not yet been proven that we can form homodimers in the prion state. Most of the enzymes in the fermentation pathway for glucose are not monomeric. Instead of using glucose as a starting sugar, we can use mannose. The first two enzymes in mannose’s fermentation pathway are monomeric. We could therefore test whether we can apply our project idea to mannose’s fermentation pathway to help speed up its process.</P><H4 id="benefits">Benefits</H4><P>If we can find a way to speed up the fermentation process to create ethanol, it will have an enormous impact on the alcohol industry. Beer and wine, for example, will be created at a much faster pace; this will have a dramatic impact on many stakeholders, such as beer and wine producers, alcoholic beverage vendors, and consumers of alcohol.</P><H4 id="application">Application</H4><P>This would be used to speed up the production of ethanol, and in turn, of alcoholic beverages.</P><H4 id="restrictions">Restrictions</H4><P>Mannose’s fermentation process is already slower than glucose’s; will our project speed it up to be even faster than glucose’s on its own. For example, glucose can be 70-80% fermented in 20-30 days, while mannose is only 30-50% percent fermented in the same amount of time (Peterson et. al, 1920).</P><H4 id="references">References</H4><P>Lin, Y., Tanaka, S. (December, 2005). Ethanol fermentation from biomass resources: current state and prospects. Applied Microbiology and Biotechnology. (2006) 69: 627. <A href="https://doi.org/10.1007/s00253-005-0229-x">https://doi.org/10.1007/s00253-005-0229-x</A></P><P>Peterson, W.H., Fred, E.B., Anderson, J.A. (April, 1920) The Fermentation of Glucose, Galactose, and Mannose by Lacrobacillus pentoaceticus, N. SP.*. J. Biol. Chem. 1920, 42:273-287. No doi. Retrieved from <A href="http://www.jbc.org/content/42/2/273.full.pdf">http://www.jbc.org/content/42/2/273.full.pdf</A></P></DIV></DIV></DIV></DIV><DIV class="row" v-if="ip.showing=='IAT'"><DIV class="row"><DIV class="col"><DIV class="content-main"><CENTER><H1><B> Implicit Association Test </B></H1></CENTER><H3 id="-what-is-an-iat-"><STRONG>What is an IAT?</STRONG></H3><P>The implicit association test (IAT) is a psychological tool for determining implicit (subconscious) biases towards or against opposing categories (Gray and Bjorklund, 2014). It runs on the theory that the more closely two things are already neuropathologically linked in someone’s mind, the faster they are going to respond when those two things share a response key on the keyboard (Gray and Bjorklund, 2014). For example, often people will respond more quickly when words concerning insects share a response key with negative words, pointing towards a possible bias against insects.</P><H3 id="-what-are-we-doing-with-it-"><STRONG>What are we doing with it?</STRONG></H3><P>Waterloo’s iGEM is hoping to utilize this tool in an effort to better understand the public’s biases and reactions towards the field of synthetic biology. We would like to see what words, if any, people have an implicit bias against or towards. Once we know the words will most likely elicit a negative reaction, we can work on finding and using alternate words to better introduce the public to synthetic biology ideas and projects.</P><H3 id="-how-does-it-work-"><STRONG>How does it work?</STRONG></H3><P>The task itself is fairly simple; five “blocks” or sets of 30 words that the participants have to sort into their categories using the “E” or “I” keys on the keyboard. In our study, participants will be asked to sort synthetic biology words such as yeast and amyloid, while sorting traditional technology words such as computer and tablet into their own category. They will also be asked to sort negative and positive words, such as anger or laughter. The most interesting portions of the task are when two categories share a response key. For instance, when negative and synthetic biology share a key, and traditional technology and positive share a key, faster responses in this block could indicate an implicit bias against synthetic biology. With the data generated, we can tell which synthetic biology words people were slowest and fastest to associate with negative, and in another block, faster or slower to associate with positive.</P><H3 id="-how-did-we-set-it-up-"><STRONG>How did we set it up?</STRONG></H3><P>We set up the IAT using code graciously offered by PsyToolKit, an online bank of psychological tests and tools. The next step was submitting it to the University of Waterloo’s Office of Research Ethics. We wanted to ensure that we followed governing rules for research with human participants. As well, we wanted to make sure study participants were protected and that our research was conducted in a way that serves the needs of participants and society. In order to achieve this we made sure we made an official research review application to our university’s research ethics board. Once our survey and task got Ethics Approval, we shared the links to our task and consent surveys to our Facebook page. We will run the task until the end of term.</P><H3 id="-methods-"><STRONG>Methods</STRONG></H3><P>15 voluntary participants were recruited from Waterloo iGEM’s Facebook page to participate in this study. We assumed that Waterloo iGEM’s diverse range of followers is a reasonable representation of the public. Participants were given an online consent form to read and agree to before participating in the study.</P><P>After consenting to the study, participants were given instructions to download the IAT study and complete it. Participants sorted words into categories as they appeared on the screen using the “E” and “I&quot; keys on the keyboard. The first round was a practice and control round so participants could learn which words are in the biotechnology category and which are in the traditional technology category. Whether the participants had the biotechnology category on the “E” or the “I” key for the first block was random, so as to minimize introducing bias into the experiment.</P><P>After pressing a key (“E” for the category on the left, “I” for the category on the right) to sort a word, a smiley face would appear if the word was sorted correctly; a frowny face would appear if the word was sorted incorrectly or not sorted at all within 3000 milliseconds. The second block was entirely the same, with the exception that there was instead be a pleasant word category and an unpleasant word category. The third block was a combination of the first two (two categories will share the same key). The fourth block was the same as the first, with the categories switched to the other key they were not previously on. The fifth block was similar to the third.</P><P>After the task was completed participants were directed to a short survey to answer some questions, give the researchers any feedback, and entered for a chance to win a Starbucks gift card.</P><P>All claims are tested with alpha = 0.05, with Microsoft Excel 2016 and IBM SPSS Statistics 24.</P><H3 id="-hypotheses-tests-"><STRONG>Hypotheses Tests</STRONG></H3><P>Ho: μ control blocks = μ mix compatible block
H1: μ control blocks ≠ μ mix compatible blockWhere μ = the population (members of the public) mean response time (ms).</P><P>Ho: μ control blocks = μ mix incompatible block
H1: μ control blocks ≠ μ mix incompatible blockWhere μ = the population (members of the public) mean response time (ms).</P><H3 id="-results-"><STRONG>Results</STRONG></H3><P>There is insufficient evidence to reject the null hypothesis at alpha = 0.05, the mean response time differs significantly differs at alpha = 0.05 between the control blocks and the mix compatible block (paired-samples t-test; p &gt;0.05, d.f = 14). Even though there was a violation of the normality assumption, the departure from normality is of little concern.</P><P>There is sufficient evidence to reject the null hypothesis at alpha = 0.05, the mean response time differs significantly at alpha = 0.05 between the control blocks and the mix incompatible block (paired-samples t-test; p &lt; 0.03, d.f = 14). Even though there was a violation of the normality assumption, the departure from normality is of little concern.</P><P>
Boxplots comparing the distribution of mean response time (ms) each presented word of participants (n = 15) for the the mixed compatible and mixed incompatible blocks. Sampled in 2017.</P><H3 id="-discussion-"><STRONG>Discussion</STRONG></H3><P>We noticed some very interesting trends.  Overall, we saw that the mean response times for the mix incompatible block have been significantly slower than the mean response times in the control blocks two and four. Surprisingly, this indicates that participants were generally slower at responding when the “biotechnology” and “unpleasant” categories shared a key (p &lt; 0.03). This could mean that the average participant may have had a slight bias towards synthetic biology. Conversely,  participants showed no statistical difference, indicating no bias (positive or negative), between the mix compatible block and the control blocks. Their mean response times were not statistically different when “biotechnology” and “pleasant” shared a key.</P><P>When we looked at specific words from our project, we also got some interesting results. The word “aggregate” was slightly faster in the mix incompatible than in the mix compatible, suggesting a possible bias against the word aggregate (i.e. associated faster with “unpleasant” than with “pleasant”). However, the difference in the response times could also suggest that there is some confusion around the word itself. It is possible participants do not have a preconceived bias towards or against the word aggregate, thus allowing for some inconsistencies. We should be very careful to clearly explain this term in all future communication. As well, the word yeast was slightly slower in the mix compatible block. This could suggest a slight bias against yeast, but it could also suggest some confusion around yeast’s involvement in synthetic biology. Yeast is commonly associated with things such as beer and bread, and we think there is a possibility that participants were discovering yeast is used in synthetic biology. We should continue to educate people about the use of yeast as a model organism in synthetic biology and lessen potential confusion.</P><P>Interestingly, the word “prion”, one of the most integral terms related to our project, was actually <EM>faster</EM> in the mix compatible block, and was associated more quickly with pleasant words. This is exciting, because it suggests we have been doing a good job with engaging and educating the public about our project. Other words related to our project, including amyloid, FRET, and enzymes, had no statistical difference in response times between the mix compatible and mix incompatible blocks. This suggests no bias towards or against these words, and it provides a good place to continue our communication about the benefits of our project, as we may be able to elicit a positive initial feeling towards these terms and our project.</P><P>Of course, we noticed some trends with words associated with the field of synthetic biology as a whole, as opposed to our project specific terms. The word bacteria was sorted particularly slowly in both mix blocks, and was even slow enough to be an outlier in the mix compatible block. This suggests that participants have an implicit bias against bacteria, as they associate bacteria with pleasant words very slowly. Bacteria are such an important tool in synthetic biology, and, although bacteria can be harmful, we and others in the field of synthetic biology must really stress the benefits and safety of this system to elicit more positive associations with this term.</P><P>BioBricks was sorted slightly faster in the mix compatible, and we think this could be due to the fact that it had “bio” in its name, and this was easier to sort with the category that shared part of its name.</P><P>Overall, we’ve learned a fair amount of information already, but there is lots more to be done. We can use different words, and we can make improvements on the structure of the test. Soon enough, we’ll know for sure which words we should use to better relate our project and synthetic biology to the general public!</P><H3 id="-next-steps-"><STRONG>Next Steps</STRONG></H3><P>A next step for this project would include extending the sample size to make it as representative as possible. This year, a large proportion of the participants were students associated with the University of Waterloo. In the future, we would like to expand this to not only other universities and colleges nationally and internationally, but also to workplaces and schools outside of the university spheres. With more diverse participants, we can better recognize the general public’s reactions to synthetic biology, as well as how different places may perceive the field. The more we know about people’s thoughts and biases towards synthetic biology, the better equipped we’ll be when relating to the public and introducing new scientific concepts.</P><P>As well, we’ve found some of the words used in the control categories “pleasant” and “unpleasant” may not be the same for everybody. Words such as family, accident, or friends could sometimes be sorted much slower than average in the pleasant versus unpleasant block, and this could be because these words are not universally positively or negatively associated for everybody. Any of these words need to be identified to avoid making the test more difficult for participants as well as preventing a less robust control baseline if they do not associate a categorically pleasant word with positive bias. As well, some words and acronyms are not well known. For instance, HVAC was sorted incorrectly more than any other word, suggesting that most participants did not know that HVAC stands for Heating, Ventilation, and Air Conditioning. Words or acronyms like this should be avoided to ensure the test is as rigorous as possible.</P><P>We would also like to extend the project and our results such that other iGEM teams can use it. Once we collect more data we intend to create a thesaurus of synthetic biology terms. This way, iGEM teams could use the more “public friendly” version of the concepts used in their projects, which could foster a more positive response to their projects and the field of synthetic biology.</P><P>Overall, there is still a lot of work to be done, but this could be a great tool for educating and introducing the public to synthetic biology. As support grows, so does research, and there are many breakthroughs on that horizon.</P><P>Please try out our IAT below!
<A href="http://igem.uwaterloo.ca/IAT/">http://igem.uwaterloo.ca/IAT/</A></P><P>References</P><P> Gray, P., &amp; Bjorklund D. (2014). Implicit Association Task. Psychology, 525-526.</P></DIV></DIV></DIV></DIV><DIV class="row" v-if="ip.showing=='Snake35'"><DIV class="row"><DIV class="col"><DIV class="content-main"><H3 id="what-is-snake35-">What is Snake35?</H3><P>This year, iGEM Waterloo has chosen a foundational advance project. These types of projects focus on the technology and processes involved in synthetic biology, which are not very well-known to the general public. In contrast to other types of projects, foundational advance projects do not usually focus on a single application, such as curing a disease. Furthermore, they tend to have less popular buzzwords for the general public. As such, it can be difficult to explain the idea and purpose of our foundational advance project to the general public.After consulting iGEM Imperial 2016’s <A href="https://2016.igem.org/Team:Imperial_College/Integrated_Practices"><EM>Visualization: A Guide for Synthetic Biologists</EM></A>, we decided a great way to convey the information from our project would be to create a visual, interactive game! We’ve called it Snake35.</P><H3 id="instructions">Instructions</H3><P>What makes Snake35 different than classic Snake? Well for starters, your objective is no longer just to make the biggest, baddest snake of all time. Our point-based version requires you to match a CFP block next to a YFP block in order to excite it. For every YFP in the excited, fluorescing state, you receive a point. However, if you have a YFP that isn’t beside CFP, it stays dark (deactivated) and you don’t receive a point. Get the highest score and give yourself a pat on the back!In our game, you must assemble a blue block next to a dim yellow block in order to score points. The result of successfully doing so causes the yellow block to fluoresce which is analogous to our FRET experiment. This game demonstrates how you must organize the proteins in a specific arrangement to observe fluorescence of YFP. However, if you have a Sup35 in between a YFP and a CFP, the YFP will not fluoresce as the arrangement is not correct.</P><H3 id="testing-snake35">Testing Snake35</H3><P>During the Waterloo Faculty of Science Open House, our booth had a table where we displayed and presented Snake35. Kids got the chance to play the game and learn about the foundational concepts of FRET. Our game received lots of positive feedback from kids and parents alike which proves that our demonstration tool was very effective in conveying the information we wanted!</P></DIV></DIV></DIV></DIV><DIV class="row" v-if="ip.showing=='3D Printing'"><DIV class="row"><DIV class="col"><DIV class="content-main"><H1 id="3d-printing">3D Printing</H1><P>At some point, every lab faces limitations when it comes to resources for experiments. For some, this limit is much lower than others, to the extent where even basic experiments become difficult to complete. To address this problem, iGEM Waterloo has begun creating a catalogue of 3D-printable lab equipment for teams to download and print for their own use.</P><H2 id="assessing-the-need">Assessing the Need</H2><P>From communicating with teams at last year’s Jamboree, we became cognizant about financial barriers many teams faced over the year. In some cases, these barriers significantly limited the experiments teams are able to perform.  </P><P>We hypothesized that there are teams in the iGEM community who would benefit from an inexpensive, yet performance grade alternative to obtain simple lab equipment. To confirm this, we sent out surveys to all iGEM teams asking about their opinions on 3D printing lab equipment, past uses and accessibility to 3D Printers, how much they spent on resources, and the importance of having sufficient lab equipment.</P><P>Based on survey responses, the general consensus was that the expense of lab equipment could be reduced by using 3D printed lab equipment as long as it could meet specific performance criteria. This provided validity to our hypothesis, and we continued with the development of this project.</P><H2 id="3d-printing">3D Printing</H2><P>To start, we needed ideas on what we could 3D print. Ideally what we made would have to be useful and have a geometry that is feasible to make using 3D printers of different qualities. We consulted our lab team about what sorts of equipment they were lacking in the lab or what types of contraptions would help with their experiments. In addition, we asked the iGEM teams we surveyed for examples of equipment we could print as well. Throughout the design process, our lab team tested out the prototypes and provided feedback, which helped us improve designs and increase effectiveness and quality of the printed equipment.</P><P>Above is the schematic for a test tube rack we designed and printed. We designed many different combs for different sized gel rigs and with different numbers of teeth to create gels with different numbers of well.</P><P>The main limitation with using 3D printing plastics, such as acrylonitrile butadiene styrene (ABS) and polyactic acid (PLA), include sterility, porousness, and strength. The plastics must be able to withstand the heat from an autoclave in order to be sterilized, they cannot be used to hold liquids, and they must be able to withstand certain amounts of force. With these guidelines, we wanted to start by creating equipment that would not be subject to these risks. Before attempting to design and print everything on our list, we started off by designing a gel comb. Using PLA plastic, we created our first piece of lab equipment for trial. After successful results, we continued to pursue other designs such as different sized gel combs, test tube racks, and microcentrifuge tube holders.</P><P>One of our team members with 3D modelling experience ran a quick workshop on how to use CAD software for other team members to attend. As the Student version of AutoCAD was free to download, it fits really well with our goal of accessibility.  Learning how to use AutoCAD under the tutelage of a peer was a fun activity and helped the team grow together, while also learning a new skill. While working in a spatial simulated environment poses its challenges, it becomes relatively easy to use after some practice. Even without an in-house expert, there are many free tutorials available online.</P><P>After a couple hours of work together, the ability to perform basic operations and the knowledge of some useful tricks were gained. Further hours of individual practice allowed for the creation of quality schematics. This knowledge was used by the team members to produce gel combs of varying sizes and dimensions, as well as experimental test tube rack designs, and assist with other designs used in the collection.</P><P>To determine quality, we ran performance testing on the plastics. We tested the gel combs under the criteria of: well quality, well consistency, and comb deterioration. In comparison to commercial grade gel combs, our 3D printed versions performed equally as well, and have been used over a dozen times without and signs of wear and tear. In terms of strength, PLA plastic has a tensile strength of ~65 MPa and ABS has a tensile strength of ~38 MPa. <STRONG>Both values are subject to printer settings such as infill, shell thickness, layer height, and print speed.</STRONG> It was determined that the type of plastic we were using could not withstand the heat from an autoclave or from a bunsen burner.</P><P>Please visit our official Waterloo iGEM page to view our catalogue seen <A href="hyperlink goes between these brackets and you will click on the word here">here</A> to print your own labwear.</P></DIV></DIV></DIV></DIV><DIV class="row" v-if="ip.showing=='Project Talks'"><DIV class="row"><DIV class="col"><DIV class="content-main"><H3 id="talks">Talks</H3><H4 id="biotech-bootcamp-">Biotech Bootcamp:</H4><P>One of our policy team members, along with two of our advisors, attended the Biotech Bootcamp at University of Guelph this past May.</P><P>At this event, presentations and panel discussions focused on science policy and communication. There was many valuable discussions about the use of genetically modified (GM) technology, science advocacy, laws and regulations, biotechnology in the developing world, and environmental sustainability. As well, one of our advisors Dr. Trevor Charles spoke about iGEM highlighting the emphasis that the competition puts on human practices as a fundamental aspect of scientific work along with showing <A href="https://www.youtube.com/watch?v=SnZx5P7sn9A&amp;feature=youtu.be&amp;a=">Our Video</A> we made during our recruitment process. There also was a public event including the Canadian premier of documentary <A href="https://www.foodevolutionmovie.com/">Food Evolution</A> which was followed by a public discussion on GMOs.</P><P>It was really an eye opening experience because the overwhelming message about the importance science communication and how important it is to tell the story of your work before someone else spins it in an undesirable or misconstrued way. This experience drove us to prioritize social media and outreach as one of our main human practices goals for our project.</P><H4 id="csm-and-western-conferences-">CSM and Western Conferences:</H4><P>This year, the University of Waterloo hosted the 67th annual Canadian Society of Microbiologists (CSM) Conference. Dr. Trevor Charles, one of our advisors, was mediating the synthetic biology section of the Conference and invited us to give a presentation during this session. In the presentation, we touched on iGEM, synthetic biology from an undergraduate perspective, our 2016 and 2017 iGEM projects, and the many benefits students receive through undergraduate research experiences.</P><P>Our talk was actually so well received that we got an invitation to also speak a the Synthetic Biology Symposium at the Western University later that summer. We were able to bring several members of our team, and we all had a great time hearing about academic and industrial research in synthetic biology, talking to experts in the field, and getting advice on some of the technical issues we were facing in our project. It was also exciting presenting our talk again, and being able to demonstrate the progress we had made since our previous talk at CSM.</P><P><EM>A Gem in the Rough</EM></P><P>Although we spend a lot of time showing others our final project, we also thought that it might be interesting to show what we do in and out of the lab on a daily basis during our build season: the elusive life of an iGEMer. One of our lab leads Max made a documentary of one 24 hour day during the summer interspersed with interviews from members of all our sub-teams. Check out <A href="https://www.youtube.com/watch?v=MX8XX0fGqEY">A Gem in the Rough</A>.</P></DIV></DIV></DIV></DIV></DIV><CENTER><DIV class="ip-group-wrapper"><DIV class="btn-group btn-group-lg" role="group" aria-label="Integrated Practices"><SPAN class="btn-group btn-group-lg" v-for="item in ip.items"><BUTTON class="btn btn-primary" v-on:click="ip.showing=item.title"> ^^item.title$$  </BUTTON></SPAN></DIV></DIV></CENTER></DIV></DIV><A href="#" class="go-top">Go Top</A><DIV class="footer"><H2>Waterloo iGEM 2017</H2><P>© 2017 Waterloo iGEM</P></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>