"
Page
Discussion
View source
History
teams
Log in
 
Team:uOttawa/Project
From 2011.igem.org
Home
Team
Project
Project Overview
BrickMason Assembly
Results
Parts Submitted
Human Practices
Essay
Gears of Evolution
Safety
Attributions
Sponsors
Data Page
Quote of the Day
"I don't always clone (that's actually not true) but when I do, I prefer BrickMason!"  Dan Jedrysiak
Project Overview
Introduction
2010 was a great year for the uOttawa team, we successfully streamlined protocols and methods for manipulating the budding yeast S. cerevisiae. We submitted a number of important BioBricks™ to the registry. Among the submissions were the two drug selection cassettes NatMX and KanMX6, a novel cloning-vector that allows for rapid integration of BioBricks™ into the Ade4 locus of S. cerevisiae, as well as a range of promoters and repressors that function in yeast. Building off of last year’s successes, the uOttawa team focused primarily on three objectives.
Objective 1: Quantitative Characterization
We feel that in order for synthetic biology to truly adopt an engineering ethos and set itself apart from traditional molecular biology, the quantitative characterization of genetic elements needs to be addressed. To this end we set out to design a reference strain of S. cerevisiae that would allow for the characterization of individual parts in whatever experimental conditions necessary. The initial inspiration for how such a characterization scheme would work came from Kelly et al. (2009). In this paper, a method for measuring the activity of BioBrick™ promoters is put forward. Promoter activity was measured against an internal reference promoter under defined experimental conditions. The method proposed controls for such factors as plasmid copy number and selection marker derived effects.
We found the Kelly paper to be instructive; however, because of differences in our model organism we adopted several modifications in our method. Whereas, Kelly et al. use E. coli transformed with low copy number plasmids, we integrate all of our constructs directly into the yeast genome. By integrating into the genome we are assured that there is only one copy of each construct, obviating the need to control for copy-number. Secondly, we wanted to simultaneously measure transcription factor (TF) expression and activity of its cognate promoter. Finally, the inherent difficulties associated with manipulating eukaryotic organisms forced us to commit to always integrate into the same genomic loc.
The above considerations, and our extensive experience with yeast led us to the following design for a reference strain. The reference strain will have a full length Act1 promoter driving the expression of yBFP (yeast-codon-optimized blue fluorescent protein) integrated into the Ade2 locus, likewise a second copy of the Act1 promoter driving yEGFP (yeast-codon-optimized green fluorescent protein) expression will be integrated into the Ade4 locus (fig. 1).
Figure 1: The reference strain of S. cerevisiae. The BFP expressing construct is integrated into the Ade2 locus using 200 bp regions of homology and uses KanMX6 selection. The GFP expressing construct is integrated into the Ade4 locus using 200 bp regions of homology and uses NatMX selection.
Several design considerations went into this reference strain. Firstly, both Ade2 and Ade4 are genes in the adenine synthesis pathway, and are considered neutral deletions. That is, if the adenine synthesis pathway is disrupted in yeast, the yeast can be grown in media supplemented with adenine. This media supplies the yeast with the necessary adenine for normal growth. Secondly, by targeting the adenine synthesis pathway, we are more easily able to select the correctly transformed yeast on the basis of color phenotype.
So, since our starting yeast strain (BY4742) has a fully intact adenine synthesis pathway, they appear creamy-white in color. The first step to generating the reference strain is the transformation of our wild-type (WT) BY4742 with Ade2-KanMX6-pACT1-BFP-Ade2. Once the yeast is transformed the DNA-repair mechanisms homologous recombinates our construct into the Ade2 locus, and in so doing removes the native ADE2 gene. Our yeast is then plated on YPD-agar plates supplemented with the antibiotic G418 and lacking adenine. The KanMX6 selection cassette confers resistance to G418 and ensures that correctly transformed yeasts are viable. In addition to drug selection, yeast with either an Ade2 deletion or mutation exhibit a color phenotype when grown with adenine, resulting in red colonies (fig.2).
Figure 2. This image shows the transformation product of BY4742 with a construct targeted to the Ade2 locus. Red colonies indicate the successful deletion of the Ade2 gene. Yeast were plated on YPD-agar with G418. The media is not supplemented with adenine.
After a colony with the correct integrant is identified, the yeast strain is transformed with the second construct (Ade4-NatMX-pAct1-GFP-Ade4). These transformed yeast are then plated on YPD-agar supplemented with Natamycin. The NatMX cassette confers resistance to the antibiotic Natamycin and serves as a selection marker for identifying correctly transformed yeast. In addition to the drug selection marker targeting the removal of the Ade4 gene causes the yeast to return to a creamy-white color.
The final mutant BY4742 yeast strain will be both Natamycin and G418 resistant, require adenine supplementation to grow properly, and will be a creamy white color. In addition to the above phenotypes, this strain will constitutively express both BFP and yEGFP.
These constructs would be integrated directly into the yeast genome maintaining tight control over copy number. BioBrick™ parts tagged with either yeast-optimized blue fluorescent protein (BFP) or GFP and networks built out of these parts can then have their own expression normalized against this reference strain acting as a standard for BioBrick™ characterization in yeast.
With the completion of the reference strain, we can set about the quantitative characterization of our TFs and the effect that they have on their cognate promoters. In order to do this all yBFP-labeled TF-containing constructs will be transformed into the Ade2 locus using the KanMX6 selection marker. Likewise, the corresponding cognate promoters are transformed into the Ade4 locus as described above with their activity monitored via yEGFP expression (fig. 3).
Figure 3: Two variants of a dual color inducible gene network. (A) Repressor tagged with BFP binds to its operators in the cognate promoter, repressing expression. Expression of the repressor is quantified via normalization against the reference strain grown under the same experimental conditions. Activity of the cognate promoter is quantified via GFP fluorescence normalized against the same references strain tagged trans-regulated promoter blocking transcription. (B) Trans-activators generated by placing the VP16 activation domain on the carboxy terminal of the BFP tagged repressor. Quantification of expression is determined as in ‘A’.
With both a reference strain and an experimental strain, the expression of the fluorescent proteins yBFP and yGFP are in the experimental strain are normalized against the reference strain. In this way the quantitative characterization of TFs and their cognate promoters can be accomplished.
Objective 2: Make Everyone’s Lives Easier and Happier
One of the major stumbling blocks of synthetic biology is the synthesis and cloning of DNA. Through our own experience and personal correspondence with other teams we came to realize that an inordinate amount of time is spent cloning and synthesizing BioBricks™. This means less time performing experiments, and less time getting results. As such, we set out to develop a rapid, reliable, and cost effective method to clone complex sequences. We’re proud to say that we achieved all of these objectives, and have dubbed the resultant technology BrickMason™ assembly.
In developing this technology we started by identifying what were the shortcoming of current cloning methods. Our conclusion was that cloning into E. coli to replicate plasmids was a major bottleneck in terms of time and energy. To answer this shortcoming we settled on an in vitro method; i.e., polymerase chain reaction (PCR). Currently, our method allows us to easily assemble up to 8 BioBricks™ in one day. For a more detailed description of the BrickMason™ method, please see our BrickMason™ Assembly page on this Wiki.
Objective 3: Keep on Building Bricks
In order to fulfill our first objective it was necessary for us to design and construct a set of novel TFs and promoters. To this end we have submitted a set of fluorescently labeled trans-activators and trans-repressors for use in yeast.
Reference:
Kelly J.R. et al.(2009), Measuring the activity of BioBrick promoters using an in vitro reference standard. Journal of Biological Engineering. 3:4
Retrieved from " http://2011.igem.org/Team:uOttawa/Project "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
