"
Page
Discussion
View source
History
teams
Log in
 
Team:uOttawa/ResultsTemp
From 2011.igem.org
Contents
1 BrickMason Results
1.1 The construct
1.2 The assembly
1.3 Results
1.4 Benchmarking
1.5 Summary
BrickMason Results
The construct
In order to test the BrickMason assembly method, we decided to build the following construct:
Figure 1. Construct created by BrickMason assembly. It is composed of 6 parts, the parts will be referred to by the red numbers shown above them.
Table 1. The number, name, and size of each part used in the construction. The top half of the table lists information for monomers, and the bottom half lists information for heterodimers.
When transformed into yeast this construct should knock out the ADE2 gene, causing the cell to turn red. Cells that have successfully taken up the construct should be resistant to the G418 drug (conferred by KanMx) and should express the Ura3-GFP fusion protein. They should therefore also grow on Ura- plates, and express GFP.
The assembly
Constructs 1-6 were PCRed using biobrick primers and then cut out of a gel, This is shown below in figure 2. The ladder used in all of our gels is the NEB 2-Log DNA Ladder which can be found at: http://www.neb.com/nebecomm/products/productN3200.asp
Figure 2. The NEB 2-Log DNA Ladder used in all of our gel pictures.
Figure 3. Parts 1-6 PCRed and cut out of a gel. Lane 1 contains the ladder, and the other lanes contain 100 ng of parts 1,2,3,4,5,6  loaded in that order.
These pieces were then used to create overlapping heterodimers as explained in the BrickMason animation and description. Part A of the heterodimer was digested with SpeI, and part B was digested with XbaI, part B was then dephosphorylated.  Part A and B were ligated together,  and  were then digested with XbaI and SpeI to get rid of homodimers. This reaction mixture was used as a template in the a PCR reaction using the forward BioBrick primr for part A and the reverse BioBrick primer for part B. The heterodimers used in this assembly (12,23,34,45,56) are shown in figure 4.
Figure 4. Heterodimers after the PCR reaction, the brightest band in each lane was cut out of the gel, lane 1 contains the ladder. The other lanes contained heterodimers 12, 23, 34, 45, and 56 run in that order.
Each heterodimer was cut out of the gel in figure 4, gel purified, and digested with XbaI and SpeI. These heterdimers were then combined in one final PCR reaction to obtain the final construct shown in figure 5.
Figure 5. The heterodimers were combined in a final PCR reaction using the forward primer for part 1 and the reverse primer for part 6 to create the final assembled construct. The expected size is 4216 bp, and a band of the correct size is clearly visible. This band was cut out of the gel and purified.
The ~4 KB band shown in figure 5 was cut out of the gel and transformed into yeast using standard transformation protocols. Figure 6 shows a picture of the transformation plate, red colonies indicate positive transform ants.
Results
Figure 6. Yeast cells growing in the presence of G418 after transformation with the construct. Red colonies indicate that the construct integrated at the correct genomic locus. There are 31 red colonies and 24 white colonies.
16 red colonies form the plate shown in figure 6 were restreaked onto SM – Ura plates, and all of them grew, the wild type BY4742 strain did not grow.
Figure 7. Colonies from the transformation plate shown in figure 6 streaked onto SM –Ura plates. A. Colonies 1-8 are shown growing B. Colonies 9-16 are shown growing C. The wild type strain BY4742 did not grow on this media.
These colonies were then inoculated into liquid culture and grown overnight and measure on the flow cytometer. Figure 8 shows that 6 of the 16 colonies expressed GFP.
Figure 8. Single colonies were innoculated into 2 mL of YPD + 2% glucose + 2% adenine and grown overnight. Overnight cultures were reinnoculated in 400 uL of SM + 2% glucose + 2% adenine to a cell density of 0.153 x 10^7 cells/mL and grown for 3 hours. Cultures were then analyzed on the flow cytometer using a 488 nm laser for excitation and a FITC emission filter. 3 technical replicates of each sample were measured and an autogate was used to keep 75% of the collected data. The mean is graphed and error bars show the standard deviation.
Benchmarking
Our construct was amplified out of each of the 16 colonies and was sequenced. Out of the 40 000 bp that were sequenced, there were only 7 definitive mutations identified. Therefore we have benchmarked out method as having an error rate of  0.0175 errors per nucleotide. Unfortunately many of these errors occurred at the junction of the Ura3-GFP fusion protein causing the GFP to be out of frame. This error occurred early on in the procedure during heterodimer formation and propagated throughout the assembly. That is why all the colonies grew in –Ura but only 6 expressed GFP. Most likely this problem is related to the ligase rather than multiple rounds of PCR.
Summary
We developed a new DNA assembly protocol called BrickMason assembly. We assembled 6 pieces in one day's work and showed that the assembled constructs functioned as expected in yeast. The yeast cells were extensively sequenced and we found that the method had an error rate of 0.0175 errors per nucleotide.  The same cloning protocol would have taken 9 days to complete using traditional cloning methods. The whole procedure can be completed with standard lab equipment and reagents. Users who will not be using E. coli as the final destination for their constructs do not need to use  E. coli at all, they can instead transform the final construct directly into the organism of interest. We hope that BrickMason makes you enjoy cloning as much as we do!
Retrieved from " http://2011.igem.org/Team:uOttawa/ResultsTemp "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
