<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Minnesota_Protocols"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Minnesota/Protocols">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Minnesota/Protocols&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Minnesota/Protocols&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Minnesota/Protocols&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Minnesota/Protocols" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Minnesota/Protocols</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><TABLE style="color:gold;background-color:#800000;" cellpadding="3" cellspacing="1" border="1" width="90%" align="center"><TBODY><TR><TH align="center"><A href="/Team:Minnesota" title="Team:Minnesota"><FONT color="gold">Home</FONT></A></TH><TH align="center"><A href="/Team:Minnesota/Team" title="Team:Minnesota/Team"><FONT color="gold">Team</FONT></A></TH><TH align="center"><A href="/Team:Minnesota/Project" title="Team:Minnesota/Project"><FONT color="gold">Project</FONT></A></TH><TH align="center"><STRONG class="selflink"><FONT color="gold">Protocols</FONT></STRONG></TH><TH align="center"><A href="/Team:Minnesota/Notebook" title="Team:Minnesota/Notebook"><FONT color="gold">Notebook</FONT></A></TH><TH align="center"><A href="/Team:Minnesota/Judging" title="Team:Minnesota/Judging"><FONT color="gold">Judging Criteria</FONT></A></TH><TH align="center"><A href="/Team:Minnesota/Attributions" title="Team:Minnesota/Attributions"><FONT color="gold">Attributions</FONT></A></TH><TH align="center"><A href="/Team:Minnesota/Safety" title="Team:Minnesota/Safety"><FONT color="gold">Safety</FONT></A></TH></TR></TBODY></TABLE><TABLE align="justify"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Agarose_gels"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Agarose gels</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Antibiotics"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Antibiotics</SPAN></A></LI><LI class="toclevel-1 tocsection-3"><A href="#Bicinchoninic_acid_.28BCA.29_protein_assay"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Bicinchoninic acid (BCA) protein assay</SPAN></A></LI><LI class="toclevel-1 tocsection-4"><A href="#Dephosphorylation_.E2.80.93_CIP_treatment"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Dephosphorylation – CIP treatment</SPAN></A></LI><LI class="toclevel-1 tocsection-5"><A href="#DNA-Sequencing"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">DNA-Sequencing</SPAN></A></LI><LI class="toclevel-1 tocsection-6"><A href="#Extraction_of_DNA_from_agarose_gels"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">Extraction of DNA from agarose gels</SPAN></A></LI><LI class="toclevel-1 tocsection-7"><A href="#Flow_Cytometry"><SPAN class="tocnumber">7</SPAN><SPAN class="toctext">Flow Cytometry</SPAN></A></LI><LI class="toclevel-1 tocsection-8"><A href="#Glycerol_cultures"><SPAN class="tocnumber">8</SPAN><SPAN class="toctext">Glycerol cultures</SPAN></A></LI><LI class="toclevel-1 tocsection-9"><A href="#Ligase_Free_Cloning"><SPAN class="tocnumber">9</SPAN><SPAN class="toctext">Ligase Free Cloning</SPAN></A></LI><LI class="toclevel-1 tocsection-10"><A href="#Ligation"><SPAN class="tocnumber">10</SPAN><SPAN class="toctext">Ligation</SPAN></A></LI><LI class="toclevel-1 tocsection-11"><A href="#Loading_buffer_for_agarose_gels_.28Stop-mix.29"><SPAN class="tocnumber">11</SPAN><SPAN class="toctext">Loading buffer for agarose gels (Stop-mix)</SPAN></A></LI><LI class="toclevel-1 tocsection-12"><A href="#Media"><SPAN class="tocnumber">12</SPAN><SPAN class="toctext">Media</SPAN></A></LI><LI class="toclevel-1 tocsection-13"><A href="#Microscopy"><SPAN class="tocnumber">13</SPAN><SPAN class="toctext">Microscopy</SPAN></A></LI><LI class="toclevel-1 tocsection-14"><A href="#PCR_reaction"><SPAN class="tocnumber">14</SPAN><SPAN class="toctext">PCR reaction</SPAN></A></LI><LI class="toclevel-1 tocsection-15"><A href="#Plasmid_preparation"><SPAN class="tocnumber">15</SPAN><SPAN class="toctext">Plasmid preparation</SPAN></A></LI><LI class="toclevel-1 tocsection-16"><A href="#Purification_of_PCR_products_and_DNA"><SPAN class="tocnumber">16</SPAN><SPAN class="toctext">Purification of PCR products and DNA</SPAN></A></LI><LI class="toclevel-1 tocsection-17"><A href="#Restriction_enzyme_digestion"><SPAN class="tocnumber">17</SPAN><SPAN class="toctext">Restriction enzyme digestion</SPAN></A></LI><LI class="toclevel-1 tocsection-18"><A href="#SDS_gel_electrophoresis"><SPAN class="tocnumber">18</SPAN><SPAN class="toctext">SDS gel electrophoresis</SPAN></A></LI><LI class="toclevel-1 tocsection-19"><A href="#Transformation_of_E._coli_.E2.80.93_TSS_method"><SPAN class="tocnumber">19</SPAN><SPAN class="toctext">Transformation of E. coli – TSS method</SPAN></A></LI></UL></TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="Agarose_gels">Agarose gels</SPAN></H1><P>50 x TAE running buffer
					1 l
	Tris-base			242 g
	Glacial acetic acid		57.1 ml
	0.5 M EDTA			100 ml
</P><P>Note: To prepare 0.5 M EDTA, dissolve 73 g EDTA in small volume of 1M NaOH, then ad H2O to 500 ml and adjust pH t0 8.0.
</P><P>
Usually make 1 % agarose – 4 g agarose (note: agarose is very expensive) in 400 ml 1x TAE buffer.
</P><P>Heat in microwave to dissolve agarose (do not boil oer). Store at 60 C for continuous use.
</P><P>To 30 ml agarose solution (30-40 ml full mini-gel and 15 – 20 ml half mini-gel) ad 1 ul ethidum bromide solution (1 % stock solution stored at 4C, small aliquot covered in aluminum foil stored on bench).
</P><P>Pour gel and check for bubbles – let solidify for approx. 20 min.
</P><P>Add approx. 2 ul Stop-mix to 10 ul sample and load 8 ul 1-kb ladder (400 ng --- each single band corresponds to approx. 20 ng DNA) as marker.
</P><P>Run gel in 1 x TAE buffer at 120 V for 20 min (if current goes over 100 mA, the buffer is old or you have a different buffer in you gel than in the reservoir).
</P><P>Note: You can pour several gels at once and store them in 1 x TAE buffer plus a drop of ethidium bromide at 4C (for approx. 1 week). For example, you get two gels out of a full mini tray by using two combs and cutting the gel in two pieces.	
</P><P>Attention !!!: Ethidium bromide is a carcinogen – wear gloves and dispose used gels in the corresponding waste.
</P><P>Attention !!!: The UV-lamp of the photo documentation system is pretty strong – use the protective face shield if you don’t want to end up with a bad sunburn.
</P><P>Turn off monitor of photo documentation system after use.
</P><H1><SPAN class="mw-headline" id="Antibiotics">Antibiotics</SPAN></H1><P>Stock-solutions (store at –20C)
</P><P>Ampicillin			100 mg/ml,
Chloramphenicol		50 mg/ml (in ethanol),
Tetracycline HCl		30 mg/ml,
Carbenicillin			100 mg/ml,
Kanamycin			30 mg/ml,
Spectinomycin			50 mg/ml.
</P><P>
Add 1 ul of antibiotic  stock solution to for each 1 ml of medium. As antibiotics degrade quickly, add them to the liquid medium just before doing a cultivation. Allow media (plate) to cool before adding antibiotics (touch bottle without feeling burned). Store plates containing antibiotics at 4C.
</P><H1><SPAN class="mw-headline" id="Bicinchoninic_acid_.28BCA.29_protein_assay">Bicinchoninic acid (BCA) protein assay</SPAN></H1><P>(Pierce instructions: <A href="http://www.piercenet.com/Technical/default.cfm?tmpl=/Lib/ViewDoc.cfm&amp;num=0731" class="external free" rel="nofollow">http://www.piercenet.com/Technical/default.cfm?tmpl=/Lib/ViewDoc.cfm&amp;num=0731</A>)
</P><P>Sensitivity:	5 – 200 ug/ml protein concentration in the sample (not in reaction mix !!!)
</P><P>Prepare working reagent:	Mix 50 parts reagent A and 1 part reagent B
</P><P>Reaction:	50 ul sample plus 1 ml working reagent
	Mix and incubate 30 min at 60C, stop by placing on ice
	(3 reactions per sample !)
</P><P>Standard protein:	BSA (prepare standard concentrations 10, 20, 30, 40, 50, 60, 70, 80, 100 ug/l)
</P><P>Calibration curve:	50 ul BSA standard solution (10-100 ug/l, samples each concentration) plus 1 ml working reagent
	Mix and incubate 30 min at 60C, stop by placing on ice
</P><P>
Biological buffer systems – pH ranges
</P><P>pH range
</P><P>1.1 – 3.3		glycine/HCl,
1.1 – 4.9		Na-citrate/HCl,
3.6 – 5.6  		acetic acid/NaOH (acetate buffer),
4.9 – 8.0		KH2PO4/Na2HPO4 (Soerensen buffer),
5.0 – 6.6 		Na-citrate/NaOH,
5.8 – 8.0		KH2PO4/NaOH,
5.8 – 8.0		Na2HPO4/NaH2PO4 (isotonic),
7.2 – 9.0		Tris/HCl,
8.55 – 12.9		glycine/NaOH.
</P><H1><SPAN class="mw-headline" id="Dephosphorylation_.E2.80.93_CIP_treatment">Dephosphorylation – CIP treatment</SPAN></H1><P>•	To avoid religation of a vector backbone it is necessary to treat the cut plasmid with calf intestinal alkaline phoshpatase (CIP) to remove 5’-phosphate groups. As only the vector backbone lacks the 5’-phosphate groups, the 5’-phosphate groups provided by the insert are sufficient for the ligase to carry out the ligation of the two DNA strands (the remaining two nicks are later repaired in E. coli).
</P><P>•	Note: Never treat both insert and vector backbone with CIP.
</P><P>•	CIP treatment becomes necessary, if you want to ligate an insert into a single restriction site. For example, an EcoRI digested insert into a likewise EcoRI digested plasmid. Although you could theoretically discriminate on an agarose gel between cut and non-cut vector and isolate the band corresponding to the cut vector from the gel, in most cases you get some cross-contamination from uncut plasmid (which can run in different supercoiled forms and apparent sizes) in your gel-extract. Since the transformation efficiency of uncut plasmid is much higher than your ligation products, you will get a huge background of plasmids without inserts.
</P><P>•	For CIP treatment:
</P><P>	Add 1-2 ul CIP to the digestion mixture and incubate together with the restriction enzymes at 37C.
</P><P>See NEB catalog and web-page:	<A href="http://www.neb.com/neb/frame_cat.html" class="external free" rel="nofollow">http://www.neb.com/neb/frame_cat.html</A><A href="http://www.neb.com/neb/frame_tech.html" class="external free" rel="nofollow">http://www.neb.com/neb/frame_tech.html</A></P><H1><SPAN class="mw-headline" id="DNA-Sequencing">DNA-Sequencing</SPAN></H1><P>ABI dye terminator cycle sequencing – preparation of sequencing reaction for BMGC DNA-Sequencing Facility (U of M):
</P><P>	3-4 ul mini-prep plasmid DNA  (1-2 ug DNA required) plus
	4 ul sequencing primer (1 pmol/ul) (3.2 pmol required).
	Add ddH2O to total volume of 12 ul
</P><H1><SPAN class="mw-headline" id="Extraction_of_DNA_from_agarose_gels">Extraction of DNA from agarose gels</SPAN></H1><P>A preparative agarose gel to separate DNA fragements (e.g. digestion products, PCR products) from other DNA fragments is often required. Load entire DNA containing solution (containing 20% Stop-mix) on a preparative of a gel (use preparative combs) and run gel. Carefully cut out the desired DNA band (do not contaminate with other DNA bands) using a clean scalpel and transfer into a 2 ml tube.
</P><P>For extraction use one of the following Qiagen Kits:
</P><P>Qiagen Qiaquick Gel Extraction Kit:	(recommended for most applications)
</P><P>	Note: Fragments &lt; 100 bps can not be isolated.
	Note: Use Qiagen EB buffer pH 8.5 for elution.
</P><P>&gt;&gt;&gt;&gt; Refer to the Qiaquick Spin Handbook for instructions.
</P><P>Qiagen Qiaex II Gel Extraction Kit (use for DNA-shuffling)
</P><P>Note: Fragments &lt;100 bps can be isolated – this kit must be used for the extraction of 50 bps during DNA-shuffling.
Note: Use Qiagen EB buffer pH 8.5 for elution.
</P><P>Additional information can be found in the “Qiagen DNA cleanup and gel extraction handbook” at <A href="http://www.qiagen.com/literature/cleanlit.asp#qiaexii" class="external free" rel="nofollow">http://www.qiagen.com/literature/cleanlit.asp#qiaexii</A>)
</P><H1><SPAN class="mw-headline" id="Flow_Cytometry">Flow Cytometry</SPAN></H1><P>•	Inoculate single colony of freshly transformed DH5αPro or TOP10 cells in 4 ml LB medium containing 50μg/ml chloramphenicol (or appropriate antibiotic). 
</P><P>•	Grow the culture overnight at 37 C with shaking (250 rpm). 
</P><P>•	Next day re-inoculate the cultures into 4 ml fresh LB medium having antibiotics and varying inducer concentrations. Inducer concentrations can be varied from 0-1mM of IPTG or 0-200 ng/ml aTc.
</P><P>•	Collect the samples at different time intervals of 3, 6 and 9 hours.
</P><P>•	Monitor the growth rate by measuring optical density at 600 nm. 
</P><P>•	Measure the fluorescence in a Becton Dickinson FACS Calibur flow cytometer equipped with a 488 nm argon laser and a 515-545 nm emission filter (FL-1) and a 585-610 nm emission filter (FL-2). 
</P><P>•	Make sure that machine has settings for E. coli.
</P><P>•	To measure the fluorescence, add 3-5 μl of the growing culture in ~1 ml PBS (phosphate buffer saline, pH-7.5). Measurement should be done at low flow rate (~1000 events/second).
</P><P>•	For each sample, collect 50,000 events.  
</P><P>•	Analyze the fluorescence in both FL-1 and FL-2 channel using FlowJo software (BD Biosciences). 
</P><P>•	Determine the background fluorescence by using controls (cells having empty plasmid vector).
</P><H1><SPAN class="mw-headline" id="Glycerol_cultures">Glycerol cultures</SPAN></H1><P>Add to 2 ml of a mid-log or 1 ml of a fresh stationary phase culture 1 ml sterile glycerol solution ( 65 % (v/v) glycerol, 0.1 M MgSO4, 25 mM Tris/HCl pH 8.0). Mix and store at –80C.
To inoculate from a glycerol culture – scrape some ice from the top of the culture – never allow the culture to thaw (freezing and thawing destroys cells).
</P><H1><SPAN class="mw-headline" id="Ligase_Free_Cloning">Ligase Free Cloning</SPAN></H1><P>1.	Plasmid preparation: 
Cut the plasmid/vector with appropriate enzymes (for cloning into pUCBB-GFP, digest it with BglII and NotI). Gel purify the vector fragment. Measure the vector concentration by nano-drop (you need 100ng vector for 1 cloning reaction)
</P><P>2. Insert preparation:
Each primer should have the following:
•	15-20 bp overlap with the gene of interest
•	15-20 bp overlap with the linearized plasmid including restriction site
•	Primers for CSD lab biobrick vectors should add the following to the gene specific sequence:
-	BB-BglII-F	5’-AGAAGGAGGAGATCT---Start
-	BB-NotI-R	5’-GATGCTCGAGGCGGCCGC---End (reverse complement) i.e., …TAAGCGGCCGCCTCGAGCATC-3’ is added to the end of the gene sequence,. PCR (2 or 4 x 50µl) for the insert using Vent following the usual protocol. Gel purify the insert. Measure the insert concentration by nano-drop (you need 100ng insert for 1 cloning reaction).
</P><P>3.	Transformation of competent E. coli
If necessary, use speed-vac to concentrate your vector/insert, so that you can have 100ng vector + 100ng DNA in about 5-25 µl total volume. 1 tube of chemically competent E.coli + 100ng vector + 100ng insert. Incubate on ice, 20 minutes. Heat shock 35-40s. Add 800µl SOC, incubate at 37°C for 1 hour with shaking (recovery). Spin down the cells: 4000rpm, 5 minutes. Remove approximately 900µl SOC, so the volume of the cell pellet + remaining SOC is less than 120µl. Resuspend the cells in the remaining SOC, plate the cells on an appropriate antibiotic plate. Invert the plate &amp; incubate at 37°C overnight.
</P><H1><SPAN class="mw-headline" id="Ligation">Ligation</SPAN></H1><P>(with NEB-ligase)
Typical reaction volume either 10 or 20 ul:
</P><P>For example:
	20 – 50 ng cut vector (also called backbone) preparation
	50 – 200 ng cut insert preparation
	1 ul 	10 x ligase buffer (note: Gibco buffer is 5 x fold)
	0.5 ul ligase
	ad. H2O to 10 ul total volume
</P><P>Incubate 1 h at RT (or overnight at 4 C, although there will rarely better results than at RT) and transform complete mixture.
</P><P>Note: Thaw ligation buffer completely – no precipitate visible – and mix thoroughly before using. Ligase buffer must be aliquoted in 10-20 ul portions and these frozen aliquots are then used for ligation. Repeatedly thawing and freezing of ligation buffer kills the ATP in the buffer.
</P><P>Pipette ligase carefully – no excess drops on the tip to ensure accurate volume.
Never vortex reactions containing DNA modifying enzymes – always mix by aspirating with the pippetor.
</P><P>Sticky-end ligation: Use a 1:2-3 ratio of vector:insert.
Blunt-end ligation: Use a 1:1 ratio of vector:insert.
</P><P>Always check DNA-concentrations of vector on insert on the same gel prior to ligation.
(You can compare the intensity of your DNA bands with that of the bands of the 1-kb ladder: if you load 8 ul kb-ladder (400 ng) then each band corresponds to approx. 20 ng)
</P><P>Too high vector and insert concentrations do not result in good ligation efficiencies – it is better to try different concentrations of vector and insert.
</P><P>Be sure that both vector and insert preparation are pure – e.g. by purifying with the Promega Wizzard Kit or Qiagen gel extraction kit.
</P><P>Additional tips and protocols are found on NEB’s web-page:
<A href="http://www.neb.com/neb/frame_cat.html" class="external free" rel="nofollow">http://www.neb.com/neb/frame_cat.html</A><A href="http://www.neb.com/neb/frame_tech.html" class="external free" rel="nofollow">http://www.neb.com/neb/frame_tech.html</A></P><H1><SPAN class="mw-headline" id="Loading_buffer_for_agarose_gels_.28Stop-mix.29">Loading buffer for agarose gels (Stop-mix)</SPAN></H1><P>For 100 ml loading buffer:
</P><P>4 M Urea 24.02 g, 10 mM EDTA 372 mg, 50% glycerol 50 ml, 0.1 % bromphenol blue 0.1 g. Add H2O to 100ml
</P><P>Note:	For separation of fragments &lt; 100 bp use Stop-Mix without dye.
</P><H1><SPAN class="mw-headline" id="Media">Media</SPAN></H1><P>LB 	(low-salt Luria Bertani)
(standard E. coli medium)
</P><P>tryptone		10 g/l,
yeast extract		5 g/l,
NaCl 			5 g/l (low-salt medium).
</P><P>
Pouring plates
</P><P>For 1L LB agar, add 15g agar to the above. Let the media cool until you can touch the bottle (flask) without feeling burned. Don’t cool too much or the agar will solidify. Add the appropriate antibiotic. Pour plates so that bottom of the plate is covered (25-35ml). Flame media briefly to get rid of bubbles, but not too long otherwise antibiotic gets killed and plastic melts. Let the plates cool before refrigerating to avoid condensation inside the plate. Store plates upside down so that no liquid drops on the media – write any information on the bottom of the plate so that losing the top of the plate will not mess up your experiment.
</P><H1><SPAN class="mw-headline" id="Microscopy">Microscopy</SPAN></H1><P>Preparations were viewed using a Nikon Eclipse E800 photomicroscope equipped with brightfield, DIC, phase and fluorescence optics including a 120 W X-Cite epi-fluorescence illuminator with UV (excitation filter 330-380, barrier 420 nm), blue (excitation filter 470-490 nm, barrier 520-580 nm) and green (excitation filter 510- 560 nm, barrier 570- 620 nm) filter sets. The samples were viewed using either a 4X, 0.13 n.a. plan fluor; 10X, 0.30 plan fluor; 20X, 0.50 n.a. plan fluor; 40X, 0.75 n.a. plan fluor; 60X, 1.40 n.a. plan apo; or100X, 1.3 n.a. plan apo objectives.16-bit digital images were collected using a Roper CoolSnap HQ monochrome camera and captured to a Pentium IV 2.6 GHz personal computer using Image Pro Plus AMS version 6.3 software (Media Cybernetics, Silver Springs, MD 20910).X, Y and Z-motor movements, filters and shutters were managed using a Ludl MAC 3000 controller (Ludl Electronic Products Ltd., Hawthorne, NY 10532) interfaced to the ImagePro software.The instrument is housed and maintained by the University of Minnesota - College of Biological Sciences, Imaging Center, <A href="http://www.cbs.umn.edu/ic/" class="external free" rel="nofollow">http://www.cbs.umn.edu/ic/</A>.
</P><H1><SPAN class="mw-headline" id="PCR_reaction">PCR reaction</SPAN></H1><P>1. Typical reaction mixture (100ul) :
</P><P>	10 x PCR buffer (w/o added MgCl2)		10 ul,
	25 mM MgCl2 a)				8 ul 	(final concentration 2 mM),
dNTP Mix (2.5 mM each dNTP)		8 ul 	(final concentration 200uM of each dNTP),
	Primer 1 (100 pmol/ul)			1 ul 	(100 pmol),
	Primer 2 (100 pmol/ul)			1 ul 	(100 pmol),
	Template DNA b)				1-10 ul (10 –200 ng),.
	Taq						0.5 ul 	(2.5 U)
</P><P>2. Sometimes different MgCl2 concentration (1 – 5 mM) can increase PCR yield. I recommend, however, to optimize other parameters first, e.g. annealing temperature, as suboptimal MgCl2 concentrations increase the error-rate of Taq polymerase.
3. Don´t use too little template, as early errors introduced by the Taq polymerase quickly amplify to significant error-rates.
</P><P>	Cycle conditions:
</P><P>	95 C	2 min (initial denaturation step)			1x
Followed by 25-32 cycles of
</P><P>	95 C	1 min (denaturation),
	55 C    1 min (annealing),					
	72 C	1 min per 1.5 kb template (extension).
</P><P>	72 C	7 min (final extension step)			
</P><P>Depending on the PCR reaction, denaturation and annealing temperature and times are different. Denaturation times can be shorter in case of low-melting temperature, but maybe longer for PCR with genomic DNA templates.
The annealing temperature depends on the melting temperature of the primer (note: calculate melting temperature only from the annealing part of the primer, not extensions that provide e.g. restriction enzyme recognition sequences).
Usually, a annealing temperature 5C below the calculated melting temperature works well.
</P><P>
Note: 	Use 200 ul thin-wall tubes, place the tubes into the corresponding smaller
spaces of the heating block of the Eppendorf Mastercycler.
</P><P>PCR reaction can also be carried out in smaller volumes (e.g. 50 ul), this can sometimes be beneficial for the reaction performance.
</P><P>Refer to the instruction manual of the Eppendorf Mastercycler for setting up a gradient, cycle extension or modified ramp times.
</P><P>
For difficult PCR reactions, addition of 2 – 20 % (v/v) DMSO (molecular biology grade) to the reaction mixture can help to obtain better PCR yields (try 2, 5 and 10% first). If this does not help, performing the PCR reactions with the Q-solution of the Qiagen PCR Kit might help (or using all components of the kit – although addition of Q-solution to a standard reaction works just as well).
</P><P>	dNTP-Mix:			2.5 mM each nucleotide
</P><P>					10 ul dATP (100 mM stock solution)
					10 ul dCTP (100 mM stock solution)
					10 ul dGTP (100 mM stock solution)
					10 ul dTTP (100 mM stock solution)
					360 ul H2O
</P><P>					aliquot 100 ul portions and store at –20C
</P><P>For use: thaw carefully and keep  on ice, freeze immediately after use.
</P><P>Oligonucleotides
</P><P>Always a 100 pmol/ul stock solution is prepared by dissolving the freeze dried pellet in H2O. Oligo-concentrations are found on the manufacture reference data sheet for each oligo.
</P><P>Remember:	dNTP’s and Taq polymerase are very expensive – use them with consideration.
</P><P>
A comprehensive list of tips and hints for PCR are found in the Qiagen handbook “Critical factors for successful PCR” available at <A href="http://www.qiagen.com/literature/brochures/pcr/index.html" class="external free" rel="nofollow">http://www.qiagen.com/literature/brochures/pcr/index.html</A>.
</P><P>Primer design:
</P><P>-	15 bps minimum length of annealing primer part – Primer needs to be 5-8 bps longer if annealing part is very AT rich in order to increase melting temperature
-	try to design primer with a 50% GC content
-	primer should contain at least one G or C at its 3’prime end to ensure optimal annealing where the polymerase starts DNA-extension.
</P><H1><SPAN class="mw-headline" id="Plasmid_preparation">Plasmid preparation</SPAN></H1><P>Purified plasmid preparations are prepared from overnight cultures of E. coli cells harboring the plasmid of interest (do not forget to add the appropriate antibiotic) using Qiagen Kits.
</P><P>Small-scale plasmid preparations (up to 20 ug of high-copy plasmid DNA) are done with the Qiaprep Miniprep Kit from 1-5 ml overnight cultures.
•	This scale is sufficient for most applications.
</P><P>&gt;&gt;&gt;&gt; Refer to the Qiaprep Miniprep Handbook for instructions.
</P><P>Medium-scale plasmid preparations (up to 100 ug of high-copy plasmid DNA) are done with the Qiaprep Miniprep Kit from 50 ml overnight cultures.
•	Note, that the obtained plasmid DNA may contain more salt impurities than the Minipreparation and my need additional purification with the Promega Wizard Kit for efficient digestion with some enzymes.
</P><P>&gt;&gt;&gt;&gt; Refer to the Qiagen Plasmid Purification Handbook for instructions.
</P><P>Qiagen handbook plasmid purification and other information can be downloaded at: <A href="http://www.qiagen.com/literature/plklit.asp" class="external free" rel="nofollow">http://www.qiagen.com/literature/plklit.asp</A></P><H1><SPAN class="mw-headline" id="Purification_of_PCR_products_and_DNA">Purification of PCR products and DNA</SPAN></H1><P>DNA (plasmids, digestion reaction mixtures, PCR reactions etc.) can be purified and concentrated with the:
</P><P>Promega Wizard PCR and DNA prep purification Kit:
</P><P>Note: Use 3 ccc luer lock syringes to apply resin and wash buffer to the columns.
</P><P>Important: Use 10 mM Tris/HCl buffer pH 7.5 for elution (do not use the Qiagen EB buffer which has a higher pH of 8.5)
</P><P>Note:	PCR products need to be purified with this kit prior to DNase digestion.
</P><P>&gt;&gt;&gt;&gt; Refer to the manual for instructions. Additional information can be found at:
<A href="http://www.promega.com/tbs/tb118/tb118.html" class="external free" rel="nofollow">http://www.promega.com/tbs/tb118/tb118.html</A></P><H1><SPAN class="mw-headline" id="Restriction_enzyme_digestion">Restriction enzyme digestion</SPAN></H1><P>Most of the enzymes used in the lab are purchased from NEB. The NEB-catalog and the NEB homepage provide you with reaction buffer charts and lots of other information on restriction enzymes, that will give you some ideas as to why your digestion might not work--- see  <A href="http://www.neb.com/neb/frame_tech.html" class="external free" rel="nofollow">http://www.neb.com/neb/frame_tech.html</A> and <A href="http://www.neb.com/neb/frame_cat.html" class="external free" rel="nofollow">http://www.neb.com/neb/frame_cat.html</A>.
</P><P>
Single enzyme digestion:
</P><P>	For example:
</P><P>	10 ul plasmid DNA (mini-prep, typically 500 ng/ul plasmid DNA)
	3 ul 10 x reaction buffer (check for enzyme specific buffer)
	1 ul  restriction enzyme (10-20 Units/ul)
	ad. H2O to 30 ul
</P><P>incubate for 1-2 hrs at 37C (!!!! some enzymes require different reaction temperatures, e.g. SmaI 25C etc.) and check result on agarose gel.
</P><P>
Double enzyme digestion:
</P><P>	For example:
</P><P>	10 ul plasmid DNA (mini-prep, typically 500 ng/ul plasmid DNA)
	3 ul 10 x reaction buffer *
	1 ul  restriction enzyme I (10-20 Units/ul)
	1 ul restriction enzyme  II (10-20 Units/ul)
	ad. H2O to 30 ul
</P><P>incubate for 1-2 hrs at 37C (!!!! some enzymes require different reaction temperatures, e.g. SmaI 25C etc.) and check result on agarose gel.
</P><P>•	Chose a 10 x reaction buffer in which both enzymes work – simultaneous digestions may not be possible for all enzyme combinations (Then you have to purify reactions (e.g. using Promega Wizzard) before submitting to a second digestion)
</P><P>•	Tip: Some enzymes do not cut well if less than 10 bps extend beyond their recognition sequence. This can happen when simultaneously digesting with a second enzyme (especially, if this enzyme is more active) cutting near that recognitions sequence (for example in a multiple cloning site).  You can look up the cleavage reactivity of enzymes close to the end of DNA-fragments in the NEB-catalog or at (<A href="http://www.neb.com/neb/frame_tech.html" class="external free" rel="nofollow">http://www.neb.com/neb/frame_tech.html</A>).
</P><P>o	To increase your digestion efficiency you should first add the less efficient enzyme to the digestion mixture and incubate for 1 hr and check a small aliquot of the digestion mixture on a agarose gel. If the plasmid is efficiently cut, add the less critical enzyme to the mixture and incubate for another hr. ---- By doing so (if cutting a plasmid), you can at least be sure that the less efficient enzyme has efficiently cut  (you won’t be able to see whether or not the second enzyme did cut).
</P><P>When designing oligo nucleotides for cloning experiments keep in mind that many enzymes need their recognition sequence at least 4 bps from the DNA-end for efficient cutting (see appendix of NEB catalog or <A href="http://www.neb.com/neb/frame_tech.html" class="external free" rel="nofollow">http://www.neb.com/neb/frame_tech.html</A>)
</P><H1><SPAN class="mw-headline" id="SDS_gel_electrophoresis">SDS gel electrophoresis</SPAN></H1><P>•	Refer to Mini-Protean III Cell Instruction Manual for assembly of electrophoresis cell and running SDS-PAGE using the Laemmli Buffer System (preferably 12% gel).
</P><P>•	Stock solutions for running gel buffer, stacking gel buffer, 30%  acrylamide/bis, TEMED solution are kept at 4C, 10% APS and 10% SDS solutions are kept at –20C.
</P><P>•	SDS-Sample Loading buffer:
</P><P>10 ml glycerol
5 ml mercaptoethanol
3 g SDS
2.5 ml 0.05 % bromphenol solution (filtered)
12.5 ml stacking gel buffer
</P><P>(store in aliquots at –20C)
</P><P>Sample preparation:
</P><P>For example,
	10 ul protein sample + 90 ul sample buffer
	denature 5 min at 95C and centrifuge
	load 5-20ul on gel
</P><P>For protein expression rate analysis of complete cells,
•	centrifuge 1 ml E. coli culture and wash cells with 50 mM Tris/HCl buffer pH 7.5
•	dissolve cells in 200 ul sample buffer and denature 10 min 95C
•	centrifuge cells and load 5-20 ul cleared supernatant onto gel
•	if high DNA concentrations make sample to viscose, dissolve cells of 1 ml culture in 100 ul 50 mM Tris/HCl buffer pH 7.5 and disrupt cells by repeatedly thawing and freezing, centrifuge and add 2 ul (1:10 diluted DNAse in5 0 mM Tris/HCl buffer pH 7.5) to cleared supernatant and incubate 20 min 37C to degrade DNA, add 200 ul sample buffer and proceed as described above.
</P><P>Staining:
Use ready to use Coomassie Stain according to instructions – destain in water.
</P><P>Dry gels according to BioRad’s instructions using the gel drying frame and drying solution.
</P><H1><SPAN class="mw-headline" id="Transformation_of_E._coli_.E2.80.93_TSS_method">Transformation of E. coli – TSS method</SPAN></H1><P>TSS solution:
</P><P>					100 ml
PEG 4000				15 g
DMSO					5 ml (add after autoclaving)
1 M MgCl2-solution			5 ml
LB					add to 95 ml				
</P><P>Adjust pH to 6.5 prior to autoclaving.
</P><P>After addition of DMSO aliquot TSS solution in 10 – 15 ml portions and store at –20C (TSS can get contaminated very quick).
</P><P>
Preparation of competent E. coli cells:
</P><P>•	Cultivate overnight E. coli culture* (*LB, add appropriate antibiotics if competent cells containing a plasmid for co-transformation are required) to inoculate main culture* 1:100 with overnight culture.
		Note:		50 ml culture will give 10 aliquots of competent cells
Use larger culture volumes (e.g. 100 ml) to prepare more aliquots.
</P><P>•	Grow main culture at 37C and 260 rpm to ensure rapid growth to OD 0.4 – 0.6 (typically 2 – 3 hrs, fast growing cells to OD 0.4 reach highest transformation efficiencies)
•	Centrifuge cells for 10 min at 4000 rpm (4C)
•	Carefully resuspend cell pellet in cold (4C) TSS solution (2 ml TSS for each 50 ml culture volume).
•	Incubate resuspended cells for 5 min on ice and aliquot 200 ul competent cells in 15 ml sterile tubes.
o	Note: Handle cells carefully and keep them always on ice as they get very fragile during the TSS treatment.
•	Shock-freeze aliquoted cells in liquid nitrogen and store cells at –80C.
</P><P>Transformation of TSS competent cells:
</P><P>•	Quickly thaw frozen competent cells at RT and place on ice.
•	Add 1 ul – 20 ul DNA solution (0.5 ul plasmid prep, 10 or 20 ul ligation mixture) to a 200 ul aliquote competent cells and mix carefully.
•	Incubate 20 min on ice.
•	Heat shock cells for 35 s in a water bath at 45C – Immediately add 800 ul LB (without antibiotics) and place on ice.
•	Incubate transformation mixture for 1 h at 37C to allow recovery.
•	Plate 10 ul (plasmid transformation) of mixture on LB plate. In case of ligation mixture, briefly spin down cells and decant most of the supernatant. Carefully resuspend cells in remaining liquid and plate on 4-6 plates.
</P><P>Test-transformation:
</P><P>•	For checking transformation efficiency of competent cells:
</P><P>Transform 10 ng of plasmid DNA (1 ul of pUC19 reference plasmid prepared by diluting 1:50 500 ug/ml pUC19 from Pharmacia) and plate 50 ul of 1 ml transformation mixture on LB plate:
</P><P>		Obtained colony number x 2 x 104	=  transformation efficiency/ug DNA
</P><P>•	Transformation efficiency for E. coli cells should be &gt; 106 to ensure good cloning results.
</P></TABLE><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:Minnesota/Protocols">http://2010.igem.org/Team:Minnesota/Protocols</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Minnesota/Protocols" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Minnesota/Protocols" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Minnesota/Protocols&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Minnesota/Protocols&amp;oldid=192466" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>