"
Page
Discussion
View source
History
teams
Log in
 
Team:Minnesota/Judging
From 2010.igem.org
Home
Team
Project
Protocols
Notebook
Judging Criteria
Attributions
Safety
Contents
1 Welcome Judges!
2 Parts Submitted to Registry
3 Characterization of mutated Lac Promoter
4 Characterization of Registry Parts submitted by other teams
5 References
Welcome Judges!
This section will include all the work we have done to satisfy competition requirements beyond our project . We have included a link to the Judging Form for your convenience.
Parts Submitted to Registry
Below is our list of parts that have been submitted to the Parts Registry.
Judges, we would like to draw your attention to the part EutSMNLK (Figure 1). In this composite part, we have stacked all the 5 Ethanolamine Utilization (Eut) bacterial microcompartment genes from Salmonella LT2. As demonstrated in our Results section, the recombinant Eut proteins from this part are expressed well in E.coli. We have also shown that co-expression of EutSMNLK with a signal sequence-tagged reporter leads to localization of the reporter to a distinct point in the cell, suggesting that this part is able to form a functional shell which encloses the reporter protein.
Figure 1. EutSMNLK insert
EutSMNLK BBa K311004
EutSN BBa K311003
Plac GFP BBa K311002
Strong TET promoter BBa K311001
Weak TET promoter BBa K311000
Characterization of mutated Lac Promoter
Figure 3. pSB1C3 Plac GFP
Figure 2. Results of flow cytometry of E. coli transformants expressing EGFP. The fluorescence is on the horizontal axis while the percentage of events or  percentage of cells counted is on the vertical axis. Red, blue and green peaks represent florescence intensities of GFP in DH5α pro cells grown with 0 mM, 0.5 mM and 1 mM IPTG, repectively. The orange and cyan peaks represent the florescence of GFP in JM109 cells grown with 0 mM and 1 mM IPTG. As expected, the peaks for the JM109 transformants had similar peaks because this strain does not express LacI which is the Lac Operon repressor protein that is inactivated by IPTG. Conversely, DH5α pro cells expressed the LacI. The conformity among the peaks for the wild-type cells suggest that LacI does not bind to the promoter sequence for GFP and it is hence constitutive. Notice the peaks associated with either cell type share similar numbers of events while the peaks from the different cells do not. The difference in events is likely due to the differential GFP expression between the two E. coli strains.
Cloning of the ethanolamine utilization microcompartment proteins relied upon the use of a vector created in the lab of our faculty mentor Dr. Claudia Schmidt-Dannert (Johnson et al, manuscript in prep). The plasmid contains a constitutively active mutant version of the E. coli Lac promoter: it has the RNA polymerase binding region, but lacks the repressor protein binding site (Schmidt-Dannert, 2000). The modified lac promoter has strong constitutive activity, and is a good candidate for consideration by other registry users. To make our modified lac promoter more attractive (and useful) to other teams, we have characterized it using the flow cytometry technique known as Fluorescence Activated Cell Sorting (FACS). To assay our promoter, we used 2 different E.coli strains - JM109 and DH5 alpha-pro- the latter constitutively expresses the Lac repressor protein (LacI). FACS was used to assay promoter activity in these 2 E. coli strains under varied levels of Isopropyl β-D-1-thiogalactopyranoside (IPTG), a compound known to inactivate the LacI protein.  Our rationale was that as our modified lac promoter is constitutively active, neither the presence of LacI nor addition of IPTG should have any effect on the promoter output (GFP expression). The FACS data, presented in Figure 2, demonstrates that activity of the mutant lac promoter is not affected by LacI or varying levels of IPTG, and our promoter is indeed constitutively active.
Our modified lac promoter, along with downstream EGFP, has been cloned into the submission vector pSB1C3 and submitted to the Registry of Standard Biological Parts as BBa_K311002 (Figure 3). Before submission, DNA sequencing was performed to confirm the sequence of the mutated lac promoter.
Characterization of Registry Parts submitted by other teams
For fulfilling the judging criteria, apart from our main project we decided to characterize other parts that already exist in the registry. We have picked up few parts from the registry, described in the table below. We have submitted our characterized constructs to the registry. All our constructs are in the biobrick plasmid pSB1C3, and were sequenced prior to submission. Our experimental results are described here in detail.
We looked for the parts in the registry that were not characterized previously. We decided to look mainly for regulatory parts, as we wanted to compare strength of different promoters by reporter assays. We looked for a range of regulatory parts like metal sensitive promoters or yeast promoters, primarily to have a sense of variety of regulations in different systems and also to study a part which could be useful to our lab advisors (see Table below). However, despite our best efforts, we were unable to clone and confirm most of the promoter parts.
We finally zeroed down on a family of constitutive promoters that were isolated from a combinatorial library by the iGEM Berkeley team in 2006. Further search and sequence alignment revealed that these are mutant tet promoters. The labs of our faculty advisors use inducible tet promoters regularly for different experiments. We were tempted to compare the tet promoters from this library to our tet promoter. We decided to study one weak and one strong tet promoter, to get a practical idea of the difference between promoter strengths. We picked the plasmid DNA for the part Bba_J23118 and part Bba_J23105 from the 384 well plates, which was supplied to us by iGEM. We first resuspended the DNA from the plate A, well #20 (J23105) and well #22 (J23118). We transformed the resuspended DNA into TOP 10 cells. Both the plasmids are Ampicillin resistant. We got ~100 transformants the next day. We inoculated a single colony from these freshly transformed plates into 50 ml LB media with ampicillin. Next day, we isolated both the plasmids to proceed with further cloning. We digested both the plasmid with EcoR I and Pst I enzymes to get the Promoter+RFP part or PoPS generator part. We ligated each promoter+RFP part into the pSB1C3 plasmid backbone (cut by the same restriction enzymes). The transformation of this ligase mix was done in Top10 cells and clones were confirmed by isolating the plasmids from the recombinant cells. These plasmids were subjected to restriction digestion by EcoR I and Pst I enzymes to look for the correct insert size. Biobrick specific primers (VF and VR primers) were used for PCR amplification of the insert, we borrowed the primer sequences from the iGEM registry. We also sequenced some of the plasmids to make sure that there are no errors in nucleotide sequence and we have the correct promoter. In all, our sequencing results indicated that we had successfully cloned the two tet promoters, so we decided to proceed with promoter activity assays for those two.
Name
Part
Experience
Type
Group
(TA)repeat
BBa_K137085
None
Regulatory, Constitutive
iGEM08_Caltech
Pspv2
BBa_K112706
None
From Salmonella, regulatory, constitutive
iGEM08_UC_Berkeley
overlapping T7
BBa_K113010
None
From T7 bacteriophage, regulatory, constitutive
iGEM08_HKUSTers
pCyc
BBa_I766555
None
For yeast, regulatory, constitutive
iGEM07_UCSF
Yeast GPD (TDH3)
BBa_K124002
None
From yeast, regulatory, constitutive
iGEM08_Brown
Cu-sensitive
BBa_I760005
None
Regulatory, metal sensitive
iGEM07_Saint_Petersburg
Tet strong constitutive
BBa_J23118
None
constitutive
iGEM2006_Berkeley
Tet weak constitutive
BBa_J23105
None
constitutive
iGEM2006_Berkeley
We transformed our new plasmids into DH5αPro cells, which constitutively express TetR and LacR (the cell has genome integrated copies of tetR and lacR) and in TOP10 cells (tetR and lacR negative cells). Single colony from each plate was inoculated and culture was allowed to grow overnight in LB medium with chloramphenicol.  Next day, cells were re-inoculated in fresh LB medium having varying inducer (anhydrotetracyline, aTc) concentration. The inducer concentration was 0, 100 and 200 ng/ml aTc.
Figure 4. Fluorescence intensities of RFP with weak promoter
Figure 5. Fluorescence intensities of RFP with strong promoter
Figure 6. Comparison of strong and weak promoters
In vivo RFP fluorescence was measured using a Becton Dickinson FACS Calibur flow cytometer equipped with a 488 nm argon laser and a 585-610 nm emission filter (FL2) at low flow rate. Fluorescence for all the samples was recorded 6 hours post induction. Flow cytometry allowed us to pick the population of living cells and determine the amount of RFP that each cell was producing. For each sample, 50,000 events were collected and analyzed using FlowJo software (BD Biosciences).
As shown in figures 4, 5 and 6, the inducer concentration has no effect on the fluorescence intensity. For all the different inducer concentrations, the fluorescence remains more or less same for the individual promoters. This indicates that these 2 tet promoters (submitted by the Berkeley team) are constitutive. We calculated the geometric mean for the red fluorescence by FlowJo software, it is interesting to see that J23105 on an average shows a mean fluorescence value of ~21.0 which is less than half of the mean fluorescence values of J23118 promoter (, ~51.0, Figure. 6). The difference in both the promoters was very visible by naked eye. It could be seen on plates and growing liquid cultures, we could not resist clicking photographs of this clean result (Figures 7 & 8). Therefore we conclude that the promoters from the combinatorial library have different characteristics, some of them are weak ( Bba_J23105 ) and some of them are strong ( Bba_J23118 ). It would be worth trying these two promoters in a system, which requires relative expression of certain genes in the same system. We hope that our characterization of these 2 promoters is useful to future iGEM teams as well as other users of the Registry of Standard Biological Parts.
Figure 7. Liquid cultures of DH5αPro transformed with RFP plasmid with weak and strong promoters
Figure 8. LB-agar plates of DH5αPro transformed with RFP plasmid with weak and strong promoters
References
Schmidt-Dannert, C., D. Umeno and F. Arnold. "Molecular breeding of carotenoid biosynthetic pathways." Nature biotechnology 18.7 (2000):750-753.
Retrieved from " http://2010.igem.org/Team:Minnesota/Judging "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
