Team:SJTU-software/Design
SJTU-software
Metlab: a metabolic network research tool box
HOME
TEAM
TEAM
VIDEO
COLLABRATIONS
PROJECT
DESCRIPTION
DOCUMENT
DESIGN
DEMONSTRATE
IMPROVE
ATTRIBUTIONS
HUMAN PRACTICE
INTERCRATED AND GOLD
PUBLIC ENGAGEMENT
SAFETY
Project —— Design & Contribution
Design
Since the birth of synthetic biology in 2000, metabolic engineering has made increasingly huge progress on high value-added natural products. The synthesis and construction of metabolic pathways play a fundamental role in microbiology, biochemistry, and many other relevant fields. In 2003, Jay Keasling’s team reconstructed the biological synthesis pathway of arteannuinic acid in E. coli, providing functional confirmation for this metabolic process. In recent years, researchers have engineered yeast to produce a variety of plant-based natural products. These pathways are likely to involve the introduction of many heterologous genes and numerous genetic modifications to increase productivity. In the biosynthesis of opioids in yeast, 17 enzymes from different plants, animals, and yeasts were encoded through synthetic biological way. Although there are several tools for scientists to analyze and select the potential genes of the designed pathway, these tools all focus on the one or two specific gene instead of the whole metabolic system. Therefore, what our team’s trying to do is making comparisons between the designed pathway we want to synthesize and the natural-existing pathways to make sure that the researchers can get the whole picture. In this way, scientists can learn more about molecular functions and biological process from an innovating aspect.
During the project, we had the pleasure to interview a PhD student in the university of Toronto. Based on his experiences working on biological metabolism, changing from several different software to visualize the network, do simulations or other studies, is quite inconvenient. Therefore, he suggested us to make our tool more handful for researches by integrating a few frequently-used functions together.
Besides, according to our survey among students in the biology department in our school, an integrated synthetic biology tool box was in urgent need. So we selected some commonly-needed functions to combine into the toolkit, containing SBML Drawer, SBML Differ, SMILES Drawer and DNA Editor. We found these existing open-source projects, did some modification and improvement, and integrated them into our Metlab.  
Now, we are still conducting more user surveys and looking for more practical needs. For these user demands, we will seek existing projects or develop more softwares, making our tool box more integrated.
Contribution
The aim of our project has always been designing and developing useful programs for Synthetic Biology and iGEM community. Comparative analysis of biological network is very useful and indicative in synthetic biology and other related studies. Specially, it enables people to handle unknown organisms starting from known ones with alike networks. However, most current tools are only for a specific problem or application, and the existing algorithm is inefficient. In order to solve these problems, we launch METLAB, a general purpose alignment software.
Metlab is a suite of synthetic biology aided tools and mainly functions as BLAST for networks. Besides network alignment, users may visualize networks, compare and draw metabolites and design their wet-lab experiments.
Combining various commonly-used functions into the toolkit, our project will definitely come in handy to people studying synthetic biology and related fields.
Design & Contribution
Design
Contribution
Address
NO. 800 Dongchuan Road, Minhang District, Shanghai, China
Contact Us
rockywei@sjtu.edu.cn
Metlab
Welcome to use our software!
Metlab: a metabolic network research tool box
Related Links
School of Life Sciences and Biotechnology, SJTU
Department of Bioinformatics and Biostatistics, SJTU
School of Electronic Information and Electrical Engineering, SJTU
SJTU-software
