<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_SJTU-software_Description skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:SJTU-software/Description</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="wrapper"><HEADER class="cf"><DIV id="title">SJTU-software</DIV><DIV id="subtitle">Metlab: a metabolic network research tool box</DIV></HEADER><NAV class="cf"><UL id="nav" class="sf-menu"><LI><A href="https://2018.igem.org/Team:SJTU-software">HOME</A></LI><LI><A href="">TEAM</A></LI><LI><A href="https://2018.igem.org/Team:SJTU-software/Team">TEAM</A></LI><LI><A href="https://2018.igem.org/Team:SJTU-software/Video">VIDEO</A></LI><LI><A href="https://2018.igem.org/Team:SJTU-software/Collaborations">COLLABRATIONS</A></LI></UL><LI><A href="">PROJECT</A></LI><LI><A href="https://2018.igem.org/Team:SJTU-software/Description">DESCRIPTION</A></LI><LI><A href="https://2018.igem.org/Team:SJTU-software/Document">DOCUMENT</A></LI><LI><A href="https://2018.igem.org/Team:SJTU-software/Design">DESIGN</A></LI><LI><A href="https://2018.igem.org/Team:SJTU-software/Demonstrate">DEMONSTRATE</A></LI><LI><A href="https://2018.igem.org/Team:SJTU-software/Improve">IMPROVE</A></LI><LI><A href="https://2018.igem.org/Team:SJTU-software/Attributions">ATTRIBUTIONS</A></LI><LI><A href="">HUMAN PRACTICE</A></LI><LI><A href="https://2018.igem.org/Team:SJTU-software/Human_Practices">INTERCRATED AND GOLD</A></LI><LI><A href="https://2018.igem.org/Team:SJTU-software/Public_Engagement">PUBLIC ENGAGEMENT</A></LI><LI><A href="https://2018.igem.org/Team:SJTU-software/Safety">SAFETY</A></LI></NAV></DIV><DIV class="wrapper"><DIV id="main" class="cf"><DIV class="headline">
                    Project —— Description      
                </DIV><DIV id="posts-list" class="cf"><ARTICLE class="cf" id="Background"><DIV class="entry-title"><DIV class="post-heading">BACKGROUND</DIV></DIV><DIV class="excerpt">
                               With the development of experimental techniques such as yeast two-hybrid, mass spectrometry, chromosome immunoprecipitation, tandem affinity purification, protein chip, phage display and literature mining, a large number of molecular interaction data, also known as biological network data, such as protein interaction network, metabolic network, Gene expression network, gene regulatory network and signal transduction network have been generated. And these data showed an exponential growth trend.
                            </DIV><DIV class="excerpt">
                             Nowadays, a great deal of research work on biological network data has been carried out. Among them, one of important researches is the comparative analysis of biological network data, the alignment of biological networks. Through the alignment, we can understand and study organisms, find the correlation between their structure and function, study the evolution and evolution of organisms based on the comparison results of biological network data, and transfer knowledge between different networks. With unknown organisms, we study un-known organisms.
                            </DIV><DIV class="excerpt">
                               At present, most of the research work is only for a specific problem or application, the time complexity of the algorithm is high, and the algorithm is inefficient. The aim of the research on alignment models and algorithms of biological networks is to develop a general-purpose alignment software, which can efficiently align multiple biological networks with multiple application patterns, similar to the sequence alignment software BLAST.
                            </DIV><DIV class="excerpt">
                              At the same time, the Systems Biology Markup Language (SBML), a representation and standard format representing many different classes of biological phenomena, including metabolic networks, cell signaling pathways, is frequently used and visualized. There are currently three LEVELs of SBML defined. SBML is defined in LEVELs. However, each LEVEL can have multiple VERSIONs within it, and new VERSIONs of a LEVEL dosupersede old VERSIONs of that same LEVEL. Therefore, it is necessary to be compared in different versions.
                            </DIV><DIV class="excerpt">
                               Using Metlab, researchers are likely to find some interesting genes through network comparisons to establish their own metabolic pathway. Therefore, we present them with the gene editing section. The gene of interest can be easily analyzed in different aspects. Certainly, the analysis of a chromosome or a plasmid is also available. Different from the traditional NCBI database, this section is nicely visualized and functionally improved.
                            </DIV></ARTICLE><ARTICLE class="cf" id="What"><DIV class="entry-title"><DIV class="post-heading">WHAT IS METLAB ?</DIV></DIV><DIV class="excerpt">
                              METLAB is an online integrated Toolbox, focusing on more efficient network alignment and more convenient model visualization. Due to requirement we found in actual study, several bioinformatics tools are integrated into METLAB, to help users research more effectively. The interface is terse, friendly and can be used conveniently with steady and safe in motion. Simply choose one function, upload the .met file and run, users can get what they need.
                            </DIV><DIV class="excerpt">
                              One of the primary functions in Metlab is biological network alignment, presented in a vivid way. It makes comparisons between local pathway and online pathways accessible, in order to get the whole picture. Its metabolic networks model is constructed on the basis of BiGG. With just one click, users can obtain an analysis report for their pathway, which is defined by metabolite reaction. Metlab can also be used to visualize SBML model and compare different versions. Other functions include SMILES’s visualization and DNA editing.
                            </DIV></ARTICLE><ARTICLE class="cf" id="How"><DIV class="entry-title"><DIV class="post-heading">HOW TO USE IT?</DIV></DIV><DIV class="excerpt">
                                Visit <A href="http://metlab-sjtu.com" target="_blank">http://metlab-sjtu.com </A></DIV><DIV class="excerpt">
                                Select the function in need on the page.
                            </DIV><DIV class="excerpt">
                                Upload the .met file.
                            </DIV><DIV class="excerpt">
                                Have a try!
                            </DIV></ARTICLE></DIV><ASIDE id="sidebar"><UL><LI class="block"><H4 class="heading" align="center">Description</H4></LI><LI class="cat-item"><A href="#Background">BACKGROUND</A></LI><LI class="cat-item"><A href="#What">WHAT IS METLAB?</A></LI><LI class="cat-item"><A href="#How">HOW TO USE IT?</A></LI></UL></ASIDE></DIV><DIV class="wrapper"><DIV class="widget-cols cf"><UL class="first-col"><DIV class="widget-block"><H4>Address</H4><P style="text-align:left;">NO. 800 Dongchuan Road, Minhang District, Shanghai, China</P></DIV></UL><UL class="second-col"><DIV class="widget-block"><H4>Contact Us</H4><P>rockywei@sjtu.edu.cn</P></DIV></UL><UL class="third-col"><DIV class="widget-block"><H4>Metlab</H4><P>Welcome to use our software!</P><P style="text-align:left;"><A target="_blank" href="http://weibohot.org">Metlab: a metabolic network research tool box</A></P></DIV></UL><UL class="fourth-col"><DIV class="widget-block"><H4>Related Links</H4><A target="_blank" href="http://life.sjtu.edu.cn/">School of Life Sciences and Biotechnology, SJTU</A><A target="_blank" href="http://cbb.sjtu.edu.cn/">Department of Bioinformatics and Biostatistics, SJTU</A><A target="_blank" href="http://www.seiee.sjtu.edu.cn/">School of Electronic Information and Electrical Engineering, SJTU</A></DIV></UL></DIV><FOOTER class="cf">
                    SJTU-software
                
            </FOOTER></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>