<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_SJTU-software_Human_Practices skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:SJTU-software/Human Practices</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="wrapper"><HEADER class="cf"><DIV id="title">SJTU-software</DIV><DIV id="subtitle">Metlab: a metabolic network research tool box</DIV></HEADER><NAV class="cf"><UL id="nav" class="sf-menu"><LI><A href="https://2018.igem.org/Team:SJTU-software">HOME</A></LI><LI><A href="">TEAM</A></LI><LI><A href="https://2018.igem.org/Team:SJTU-software/Team">TEAM</A></LI><LI><A href="https://2018.igem.org/Team:SJTU-software/Video">VIDEO</A></LI><LI><A href="https://2018.igem.org/Team:SJTU-software/Collaborations">COLLABRATIONS</A></LI></UL><LI><A href="">PROJECT</A></LI><LI><A href="https://2018.igem.org/Team:SJTU-software/Description">DESCRIPTION</A></LI><LI><A href="https://2018.igem.org/Team:SJTU-software/Document">DOCUMENT</A></LI><LI><A href="https://2018.igem.org/Team:SJTU-software/Design">DESIGN</A></LI><LI><A href="https://2018.igem.org/Team:SJTU-software/Demonstrate">DEMONSTRATE</A></LI><LI><A href="https://2018.igem.org/Team:SJTU-software/Improve">IMPROVE</A></LI><LI><A href="https://2018.igem.org/Team:SJTU-software/Attributions">ATTRIBUTIONS</A></LI><LI><A href="">HUMAN PRACTICE</A></LI><LI><A href="https://2018.igem.org/Team:SJTU-software/Human_Practices">INTERCRATED AND GOLD</A></LI><LI><A href="https://2018.igem.org/Team:SJTU-software/Public_Engagement">PUBLIC ENGAGEMENT</A></LI><LI><A href="https://2018.igem.org/Team:SJTU-software/Safety">SAFETY</A></LI></NAV></DIV><DIV class="wrapper"><DIV id="main" class="cf"><DIV class="headline">
            HUMAN PRACTICE —— INTERCRATED AND GOLD        
        </DIV><DIV id="posts-list" class="cf"><ARTICLE class="cf"><DIV class="excerpt">
                      Throughout the whole preparation process of Metlab, we paid attention to every suggestion from the professions and the peers. 
                   </DIV><DIV class="excerpt"><STRONG>April</STRONG>, we came up to build a metabolic-network-related tool, and soon consulted Prof. Tao to make sure that our idea is worth developing. At the same time, he raised the challenges. 
                       <STRONG>May</STRONG>, when we were digging the background, the communication with Prof. Wang pushed us to enlarge our database for network alignment.
                       <STRONG>Summertime</STRONG>, we sought suggestions aboard. It is highly effective that we made our tool far more comprehensive and user-friendly by adding other related functions and achieving visualization.
                       <STRONG>In the meantime</STRONG>, we kept active communication with organizations and teams all over the world. Every presentation we made to others helped our Metlab become better.

                   </DIV><DIV class="excerpt">
                     Each step, we keep improving our project based on researchers'needs and doubts. 
                     That’s what we truly care.
                      The details are as follows.
                    </DIV><DIV class="entry-title"><DIV class="post-heading" id="ExpertInterview">Expert Interview</DIV></DIV><DIV class="expert_name" id="FT">Prof. Fei Tao</DIV><DIV class="subtitle">School of Life Sciences &amp; Biotechnology</DIV><DIV class="subtitle">Shanghai Jiao Tong University</DIV><DIV id="FeiTao"><DIV class="member_info"><BLOCKQUOTE>
                                Dr. Tao is now an associate professor of SJTU. He has been engaged in the research of microbial synthetic biology since he got the doctoral degree. He mainly focuses on developing efficient cell factories and advanced biocatalytic technology based on the concept of synthetic biology.
                            </BLOCKQUOTE></DIV></DIV><DIV class="excerpt">
                        Our interview concentrated on the topic “Application of metabolic network alignment tool in the laboratory”. Dr. Tao told us design of metabolic networks in synthetic biology focus on these aspects: 1.Yield; 2. Efficiency(in A -&gt; B, concentration of B/A); 3. Period. Then we briefly introduced the idea of our software (a metabolic network alignment tool). Dr. Tao was greatly interest in the tool. He also analyzed a number of factors that lead to the difficulty of metabolic network alignment. How to evaluate similarity of two metabolic networks? How to grade the result of alignment? How to evaluate similarity of different compound…… His opinion made us have a deeper understanding of how make our software more practical and functional in the laboratory.
                    </DIV><DIV class="expert_name" id="ZW">Prof. Zhuo Wang</DIV><DIV class="subtitle">Department of Bioinformatics and Biostatistics</DIV><DIV class="subtitle">Shanghai Jiao Tong University</DIV><DIV id="ZhuoWang"><DIV class="member_info"><BLOCKQUOTE>
                                        Prof. Zhuo Wang is an expert in computational system biology. Her store of knowledge in the field of system biology helped us a lot in the project. At the first time of our interview, she expressed her praise towards our idea of building a network alignment tool. Meanwhile, she listed a few methods already existed for the network alignment for us to study and improve.
                                    </BLOCKQUOTE></DIV></DIV><DIV class="excerpt">
                                Later, Prof.Wang reminded us that we should not only focus on the alignment within E.coli, but also need to expand it to other bacteria. This is because in reality, there are far more engineering strains that could benefit from our software. Moreover, a larger number of bacteria involved makes our database richer and larger.
                            </DIV><DIV class="expert_name" id="DF">Prof. Daniel Figeys</DIV><DIV class="subtitle">Department of Biochemistry, Microbiology and Immunology</DIV><DIV class="subtitle">University of	Ottawa</DIV><DIV id="DanielFigeys"><DIV class="member_info"><BLOCKQUOTE>
                                Daniel Figeys is a Professor and the Chair of the Department of Biochemistry, Microbiology and Immunology (BMI) and cross-appointed in the Department of Chemistry and Biomolecular Sciences at the University of Ottawa.  He is also a Canada Research Chair in Proteomics and Systems Biology.  His expertise is in the development of proteomics and systems biology and more recently in metaproteomics for microbial community analysis.
                            </BLOCKQUOTE></DIV></DIV><DIV class="excerpt">
                        In Prof. Daniel Figeys’s Lab, they have developed a advanced bioinformatics software “iMetaLab”. Their project is originally designed for human/mouse gut metaproteomics protein identification and quantification, rooted from the MetaProIQ workflow. Upon the rising request, they wrap up the workflow into a desktop standalone version, with taxon analysis module built in. In the meantime, they have been working on some meta-function annotation and enrichment analysis, as well as some data visualization modules. Then they realize that it would be good to wrap up all related functions and share them to the community. They aim to make iMetaLab a free and one-stop toolset for gut metaproteomics, with increasing amount of tools under development. This interview inspired us how to make our software “Metlab: a metabolic network alignment tool” more user-friendly and complete. The process of their software development enlightened us to make “Metlab” a functional toolset with many tools related metabolic network.
                    </DIV><DIV class="expert_name" id="JM">Dr. Juntao Mai</DIV><DIV class="subtitle">PhD in Molecular biology</DIV><DIV class="subtitle">University of Toronto</DIV><DIV class="excerpt">
                                Juntao Mai works at the department of molecular genetics, University of Toronto. During one of our team members’ exchange there, he gave many valuable advice on the improvement of the project. For the most important part, he recommended that we should add a function of metabolites comparison. Based on his research experiences, he was always troubled by changing from one software to another to finding interested molecules from network. And there are always problems of format conversion between different tools. We took his advice, and took a step further. At the end, we completed a toolkit including network visualization, network alignment, SMILES comparison and DNA editing. 
                       </DIV></ARTICLE><ARTICLE class="cf" id="BUDS"><DIV class="entry-title"><DIV class="meta"> May 18th  to  May 20th , 2018</DIV><DIV class="post-heading">Biology Undergraduates Development Symposium（BUDS）</DIV></DIV><DIV class="excerpt">
                        Participants: Sun Yat-Sen University, Shanghai Jiao Tong University, Fudan University, Nanjing University, Central South University.
                    </DIV><DIV class="excerpt">
                        From May 18th 2018 to May 20th 2018, our team members attended Biology Undergraduates Development Symposium（BUDS）,which was held by School of Life Science, Sun Yat-Sen University.
                    </DIV><DIV class="excerpt">
                        During poster presentation, we present a poster on our work to foster communication and possible future collaboration between different iGEM teams. In the communication conference, we introduced the idea of our projects, gave our suggestions and shared fresh ideas. Through the communication we learned from each other, which was a great help to our 2018 iGEM projects.
                    </DIV><DIV class="entry-title" id="TECHNOMISE"><DIV class="meta"> July 7th to July 13th , 2018</DIV><DIV class="post-heading">TECHNOMISE Summer Boot Camp</DIV></DIV><DIV class="excerpt">
                            From July 7th 2018 to July 13th 2018, our team member took part in Science Summer Boot Camp on “Microbes and Biotechnology” at the University of Ottawa. The charming of the summer boot camp lies in its interdisciplinary integration, we learned many bioinformatics analysis methods which promoted our projects “Metlab: a metabolic network alignment tool” implementation.
                    </DIV><DIV class="excerpt">
                            The one-week summer boot camp closes with a scientific symposium with invited speakers, student presentations and posters. We presented our project and got a lot of advice from professors and students. The interesting and pleasant summer boot camp served as valuable inspiration to advance our project of 2018 iGEM with fresh motivation.
                    </DIV><DIV class="entry-title" id="APC"><DIV class="meta"> July 30th to August 3rd , 2018</DIV><DIV class="post-heading">6th Asia-Pacific iGEM Conference</DIV></DIV><DIV class="excerpt">
                                This year, we continued to be an active participant in the iGEM Asia Pacific Conference, since we consider this conference as a great platform for us to report our interim achievements. In the poster session, our project intrigued many teams, especially those that also focus their projects on synthetic metabolic pathways. They believed that our comparison between the designed metabolic pathway and the natural biochemical network would help them accomplish their project better. 
                             </DIV><DIV class="excerpt">
                               During the presentation, we demonstrated the background and design thought to the conventioneers and the professors, and they gave us some sincerer and professional suggestions. We also discovered our own shortcomings through this conference. We couldn’t present our algorism lucidly to our audience, so that we will pay special attention to this problem for the Grant Jamboree presentation.
                            </DIV></ARTICLE></DIV><ASIDE id="sidebar"><UL><LI class="block"><H4 class="heading" align="center">Expert Interview</H4></LI><LI class="cat-item"><A href="#FT">Prof. Fei Tao</A></LI><LI class="cat-item"><A href="#ZW">Prof. Zhuo Wang</A></LI><LI class="cat-item"><A href="#DF">Prof. Daniel Figeys</A></LI><LI class="cat-item"><A href="#JM">Dr. Juntao Mai</A></LI></UL><LI class="block"><H4 class="heading" align="center">Communication</H4></LI><LI class="cat-item"><A href="#BUDS">BUDS</A></LI><LI class="cat-item"><A href="#TECHNOMISE">TECHNOMISE</A></LI><LI class="cat-item"><A href="#APC">Asia-Pacific iGEM Conference</A></LI></ASIDE></DIV><DIV class="wrapper"><DIV class="widget-cols cf"><UL class="first-col"><DIV class="widget-block"><H4>Address</H4><P style="text-align:left;">NO. 800 Dongchuan Road, Minhang District, Shanghai, China</P></DIV></UL><UL class="second-col"><DIV class="widget-block"><H4>Contact Us</H4><P>rockywei@sjtu.edu.cn</P></DIV></UL><UL class="third-col"><DIV class="widget-block"><H4>Metlab</H4><P>Welcome to use our software!</P><P style="text-align:left;"><A target="_blank" href="http://weibohot.org">Metlab: a metabolic network research tool box</A></P></DIV></UL><UL class="fourth-col"><DIV class="widget-block"><H4>Related Links</H4><A target="_blank" href="http://life.sjtu.edu.cn/">School of Life Sciences and Biotechnology, SJTU</A><A target="_blank" href="http://cbb.sjtu.edu.cn/">Department of Bioinformatics and Biostatistics, SJTU</A><A target="_blank" href="http://www.seiee.sjtu.edu.cn/">School of Electronic Information and Electrical Engineering, SJTU</A></DIV></UL></DIV><FOOTER class="cf">
                    SJTU-software
                
            </FOOTER></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>