Team:Uppsala/Transcriptomics/Bioinformatics
Home
Project
Worm Culturing
Transcriptomics
Phage Display
Reporter System
Modeling
Demonstrate
InterLab
Improve
Parts
Parts
Composite Part
Team
Meet the Team
Collaboration
Attribution
Achievements
Portfolio
Notebook
Protocol
Human Practice
Human Practices
Public Engagement
Safety
Results
Resultat from the Lab
Achievement
Bioinformatics
Experiment
Results
References
Bioinformatics
After a succesfull sequencing has been performed and you’re left with raw data containing millions and millions (and millions) of lines of base sequences, all of this needs to be processed and interpreted. This is where the interdisciplinary field of bioinformatics comes in. A vast range of software tools are available, tailored to different kinds of analysis as well as being unique to the different sequencing methods being used.
Most of the tools we used were available through the free website Usegalaxy.org which as well let us do the processing on their servers. Because we also made use of nanopore sequencing, tailored tools used for the MinION data were available from their community hub which could be run from a terminal window.
Experiment
We decided to create our bioinformatics pipeline from scratch. This was not an easy task however as nanopore technology is novel and many of the available pipelines are tailored to illumina sequencing. Generally though, a basic transcriptomics pipeline looks like the following: Alignment to a reference genome, gene counting and differential gene expression [1]. However a couple of data processing steps were needed for the nanopore data beforehand such as demultiplexing and adapter trimming.
Demultiplexing and Adapter Trimming
Because the sequencing itself runs pooled samples containing both the barcoded cultured- and control-group samples, the data produced needs to be demultiplexed i.e separated into files containing the reads from respective groups. Because the barcodes used to fingerprint each group is made up of its own base sequence, this also had to be removed or ”trimmed” from the data, leaving us with the pure mRNA sequences. This was achieved using a free nanopore community tool called porechop.
Genome Alignment
The base sequences needs to be aligned to the reference genome of the sequenced species in question for the downstream data analysis. This is important because we want to know where each sequence actually lies in the genome and which genes they correspond to. Genome alignment was done using another community tool called minimap2.
Figure 1. Running demultiplexing and barcode trimming from the terminal. The programme first separates the reads according to barcode and then searches for available possible barcodes to be trimmed off.
Gene Counting
Gene counting basically means that you count how many times each mRNA sequence (aligned over a gene from the previous step) occurs. This in turn directly correlates to the amount of up- or down-regulation of that particular gene. A lot of different tools were available for gene counting but ”featureCounts” was chosen through galaxy.
After the differential gene expression analysis is done the data was filtered twice, one time for the best adjusted P-value and subsequently for the highest (meaning the most significant) fold changes. Left were a couple of candidate genes which could be easily identified by their gene ID through various databases such as NCBI.
Figure 2. Results of a differential gene expression analysis using Deseq2 on test files. The genes (shown with their gene ID) as well as their mean base length and several statistical results can be seen.
Result
Validating our Transcriptomics Pipeline
The transcriptomics pipeline was tried out and validated using read files available from the internet. The files consisted of two datasets of E. coli (triplicates) cultured in regular LB and a sugar solution respectively.
Figure 3. Results of the differential gene expression analysis using Deseq2 on test files. The genes (shown with their gene ID) as well as their mean base length and several statistical results can be seen.
Figure 4. Results of the differential gene expression after filtering for statistical significance and fold change.
The results after searching for the genes in the NCBI database showed that the most expressed gene from the sugar-cultured E. coli was shown to be involved in a type of sugar system, proving that the pipeline was indeed working.
Figure 5. Highly expressed gene produced from the pipeline matching a glucose specific gene.
Figure 6. Results of the differential gene expression done on our own data.
Analyzing Our Own Sequencing Data
Table 1. The first few genes as a result of the differential gene expression analysis seen in Figure 6                 together with their promotor sequence and function in the organism.
Gene ID
Gene name
Promotor sequence
Function
Fold change
ER3413_45
ER3413_70
ER3413_87
ER3413_126
ER3413_173
apaG
leuA
murG
panD
frr
ggcaccatgcagggtcactacgaaatgatcgatgaaa
ttgacatccgtttttgtatccagtaactctaaaagc
-
tagacactaaacaaaaatcgggcaatactgcgtga
ttacccgtaatatgtttaatcagggctatacttagcac
protein associated with Co2+ and Mg2+ efflux
2-isopropylmalate synthase
N-acetylglucosaminyl transferase
putative inner membrane protein
inner membrane protein, UPF0118 family
0.40
0.40
0.40
0.40
0.40
The resuts from our runs unfortunately did not produce as good results as seen above. Due to the major issues with sequencing and actually generating enough data, it can be seen in figure 6 what kind of effects it had. Judging by the adjusted p-values it is clear that even though the genes can indeed be identified as seen in Table 1 the statistical significance is extremely uncertain (the minimal accepted threshold is an adjusted p-value of <= 0.05). Any up-or down regulation of fold-change of interest was not able to be identified either. Looking at these errors it can be assumed that no major change in fold-change as well as low significancy is due to simply not enough data being generated from the prior sequencing step. Because of these facts no gene could be identified as a possible candidate for our reporter system.
References
[1] Galaxyproject, 2018. Reference-based RNA-Seq data analysis Galaxyproject Date of visit 2018-10-15
