<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Uppsala_phagedisplay_experiment skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Uppsala/phagedisplay/experiment</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="wrapper-for-mobile"><NAV id="nav-mobile" class="nav-mobile" role="navigation" aria-labelledby="nav-mobile-toggleLabel" aria-hidden="true"><UL id="menu-mobile" class="menu-nav-mobile" role="menu-mobile" aria-orientation="vertical"><LI class="mobile-list-item-home" role=""><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala" role="menuitem" tabindex="-1"> Home </A></LI><LI class="mobile-list-item" role=""><A class="mobile-list-link" href="#" role="menuitem" tabindex="-1"> Project </A></LI><UL class="mobile-nav-ul"><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Worm_Culturing" role="menuitem" tabindex="-1"> Worm Culturing  </A></LI><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Transcriptomics" role="menuitem" tabindex="-1"> Transcriptomics</A></LI><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Phage_Display" role="menuitem" tabindex="-1"> Phage Display </A></LI><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Reporter_System" role="menuitem" tabindex="-1"> Reporter System  </A></LI><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Model" role="menuitem" tabindex="-1"> Modeling  </A></LI><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Demonstrate" role="menuitem" tabindex="-1"> Demonstrate  </A></LI><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/InterLab" role="menuitem" tabindex="-1"> InterLab  </A></LI><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Improve" role="menuitem" tabindex="-1"> Improve  </A></LI></UL><LI class="mobile-list-item" role=""><A class="mobile-list-link" href="#" role="menuitem" tabindex="-1"> Parts </A></LI><UL class="mobile-nav-ul"><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Parts" role="menuitem" tabindex="-1"> Parts  </A></LI><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Composite_Part" role="menuitem" tabindex="-1"> Composite Part  </A></LI></UL><LI class="mobile-list-item" role=""><A class="mobile-list-link" href="#" role="menuitem" tabindex="-1"> Team </A></LI><UL class="mobile-nav-ul"><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Team" role="menuitem" tabindex="-1"> Meet the Team  </A></LI><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Collaborations" role="menuitem" tabindex="-1"> Collaboration  </A></LI><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Attributions" role="menuitem" tabindex="-1"> Attribution  </A></LI><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Achievements" role="menuitem" tabindex="-1"> Achievements  </A></LI></UL><LI class="mobile-list-item" role=""><A class="mobile-list-link" href="#" role="menuitem" tabindex="-1"> Portfolio </A></LI><UL class="mobile-nav-ul"><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Notebook" role="menuitem" tabindex="-1"> Notebook  </A></LI><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Protocol" role="menuitem" tabindex="-1"> Protocol </A></LI></UL><LI class="mobile-list-item" role=""><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/  " role="menuitem" tabindex="-1"> Human Practice </A></LI><UL class="mobile-nav-ul"><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Human Practices" role="menuitem" tabindex="-1"> Human Practices  </A></LI><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Public_Engagement" role="menuitem" tabindex="-1"> Public Engagement  </A></LI><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Safety" role="menuitem" tabindex="-1"> Safety </A></LI></UL><LI class="mobile-list-item" role=""><A class="mobile-list-link" href="#" role="menuitem" tabindex="-1"> Results </A></LI><UL class="mobile-nav-ul" style="visibility: hidden;"><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Result" role="menuitem" tabindex="-1"> Resultat from the Lab </A></LI><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Achivements" role="menuitem" tabindex="-1">  Achievement </A></LI></UL></UL></NAV></DIV><DIV class="body"><DIV class="content blur-box" style="font-size:16px;"><DIV class="content-text" id="scrolldown"><DIV><H1>Experiment</H1></DIV><DIV><H2>New Application</H2><H3>Unknown genome/proteome </H3><P>Due to our target strongyle having an unknown genome/proteome, selection of a single target protein is not feasible. Efforts could have been made to separate surface proteins from the organism, but such an approach does not provide any guarantees of the extracted proteins displaying correct conformations. To preserve each of the possible protein targets in a physiological state, the entire organism was used for the experiment. </P><H3>Accessible high throughput screening </H3><P>Having decided to use a whole organism as the target, the next challenge is finding a way to deal with the plethora of exposed surface proteins. Attempting to identify possible ligands one at a time would have been prohibitively laborious, and a method had to be selected that could screen binding compounds in a massively parallel manner. Several such methods exist, but most would require specialized equipment and/or comprehensive precursor libraries. Phage display is a simple yet robust approach to the same problem. </P><H3>Immobilization hard for whole worms</H3><P>To be able to wash away unbound phages, while retaining the bound phages, existing protocols employ various methods for immobilizing the target. Since the target of our experiment is the surface of an entire organism rather than a single protein, the common methods of fixing the target were ineffective, and a new method had to be developed. 
The implemented solution relies on filter inserts for eppendorf tubes. The filters simultaneously act as a substitute for well plates and an immobilization method. 
</P><H3>Experiment procedure</H3><P>
With the experiment necessitating the use of phage display on an entire organism, no existing protocol was entirely applicable. This meant that we had to adapt a protocol insofar as it was possible, and introduce completely new solutions to problems unique to our application. 
</P><P>
For a detailed description of our protocol, click the link. Our whole organism phage display experiments started with preparation of the containers that the whole procedure were performed in, eppendorf tubes tubes, with filter inserts. The preparation consisted of blocking the tubes with blocking buffer, to prevent non-specific interactions.

To avoid selecting for phage with affinity for the plastic and not the organism, an affinity screening for the tubes was performed before introducing the target. After collecting the phage elute that doesnâ€™t bind to the tubes, we performed our first affinity screening against the small strongyle. This was done by introducing the 12-mer peptides expressing phages to the strongyle placed in the filter tubes. Unbound phages were then washed away followed by elution and collection of the bound phages with an general buffer. All washing and elution steps were performed in the filter tubes, where liquid during centrifugation could pass through the filters, leaving the strongyle still on the top of the filter.
</P><P>
In the next step of the experiment, phage titering was carried out, to visualise the amount of infectious phages present in the elute. The elute phage with affinity to the strongyle were then amplified in the e.coli ER2738 host strain, by adding the phage to the strain in early log-phase. 

Phage and bacterias were then separated with the help of centrifugation to discard e. coli and the phage were extracted by precipitation.

The success of the amplification was examined by a new round of phage titering. Once aware of the phage titer, the next round of this previously described procedure consisting of affinity screening to the strongyle, washing and elution, phage titering, amplification of phage and a new round of phage titering was performed.
</P><P>
After the third round of affinity screening, single plaques, (consisting of single phage clones) were picked from the titering plates and amplified separate. The phages DNA from the amplified plaques were then extracted and purified.
</P></DIV><DIV><H2>Phage Titering</H2><P>
Phage titering is done after every affinity screening, to assess the amount of phages that bind to the target. By following the titering protocol consisting of plating phages together with mid-log phase bacteria, visible blue plaques are formed on Xgal/IPTGl plates. The goal with plating is to achieve plates with around 100 plaques, which is fulfilled by doing several dilution series of the infected bacteria. The reason for this is that plaques will only increase linearly with added phage, when the multiplicity of infection (MOI) is much less than 1. Also low MOI result in one DNA sequence per plaque. To asses the titre, plaque forming unit (pfu) can be calculate by multiply number of plaques with the bacteria dilution. 
</P><P>
Phage titering is also carried out after every phage amplification, now to assess the successfulness of the amplification and to make sure you have a sufficient amount of phage for the subsequente affinity screening.
</P><P>
We did three rounds of panning to select for phages having affinity peptides for the target. The last phage titering could then be used to select single clones to be prepared for sequencing. 
</P></DIV><DIV><H2>Verify affinity and analysis of DNA</H2><P>
So how do we prove that the selected phages clones really binds to our target, the strongyle? With the help of ELISA (enzyme-linked immunosorbent assay) single phage clones affinity to the target could be tested, by screening against plastic binders an assessment regarding which samples are viable can be made. The ELISA was performed in the centrifugal filter-tubes as to expose the phages to the same environment as the panning stage of the process. 
</P><P>
After amplification of each chosen plaque, the phage DNA was extracted an analysed before sequencing. We used nanodrop to determine the DNA concentration and gel electrophoresis to determine purity and the size of the DNA.
</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>