<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Uppsala_Transcriptomics_cDNA_Conversion skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Uppsala/Transcriptomics/cDNA Conversion</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="wrapper-for-mobile"><NAV id="nav-mobile" class="nav-mobile" role="navigation" aria-labelledby="nav-mobile-toggleLabel" aria-hidden="true"><UL id="menu-mobile" class="menu-nav-mobile" role="menu-mobile" aria-orientation="vertical"><LI class="mobile-list-item-home" role=""><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala" role="menuitem" tabindex="-1"> Home </A></LI><LI class="mobile-list-item" role=""><A class="mobile-list-link" href="#" role="menuitem" tabindex="-1"> Project </A></LI><UL class="mobile-nav-ul"><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Worm_Culturing" role="menuitem" tabindex="-1"> Worm Culturing  </A></LI><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Transcriptomics" role="menuitem" tabindex="-1"> Transcriptomics </A></LI><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Phage_Display" role="menuitem" tabindex="-1"> Phage Display </A></LI><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Reporter_System" role="menuitem" tabindex="-1"> Reporter System  </A></LI><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Model" role="menuitem" tabindex="-1"> Modeling  </A></LI><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Demonstrate" role="menuitem" tabindex="-1"> Demonstrate  </A></LI><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/InterLab" role="menuitem" tabindex="-1"> InterLab  </A></LI><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Improve" role="menuitem" tabindex="-1"> Improve  </A></LI></UL><LI class="mobile-list-item" role=""><A class="mobile-list-link" href="#" role="menuitem" tabindex="-1"> Parts </A></LI><UL class="mobile-nav-ul"><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Parts" role="menuitem" tabindex="-1"> Parts  </A></LI><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Composite_Part" role="menuitem" tabindex="-1"> Composite Part  </A></LI></UL><LI class="mobile-list-item" role=""><A class="mobile-list-link" href="#" role="menuitem" tabindex="-1"> Team </A></LI><UL class="mobile-nav-ul"><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Team" role="menuitem" tabindex="-1"> Meet the Team  </A></LI><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Collaborations" role="menuitem" tabindex="-1"> Collaboration  </A></LI><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Attributions" role="menuitem" tabindex="-1"> Attribution  </A></LI><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Achievements" role="menuitem" tabindex="-1"> Achievements  </A></LI></UL><LI class="mobile-list-item" role=""><A class="mobile-list-link" href="#" role="menuitem" tabindex="-1"> Portfolio </A></LI><UL class="mobile-nav-ul"><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Notebook" role="menuitem" tabindex="-1"> Notebook  </A></LI><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Protocol" role="menuitem" tabindex="-1"> Protocol </A></LI></UL><LI class="mobile-list-item" role=""><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/  " role="menuitem" tabindex="-1"> Human Practice </A></LI><UL class="mobile-nav-ul"><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Human Practices" role="menuitem" tabindex="-1"> Human Practices  </A></LI><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Public_Engagement" role="menuitem" tabindex="-1"> Public Engagement  </A></LI><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Safety" role="menuitem" tabindex="-1"> Safety </A></LI></UL><LI class="mobile-list-item" role=""><A class="mobile-list-link" href="#" role="menuitem" tabindex="-1"> Results </A></LI><UL class="mobile-nav-ul" style="visibility: hidden;"><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Result" role="menuitem" tabindex="-1"> Resultat from the Lab </A></LI><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Achivements" role="menuitem" tabindex="-1">  Achievement </A></LI></UL></UL></NAV></DIV><DIV class="body"><DIV class="content blur-box" style="font-size:16px;"><DIV id="toc" class="toc"><UL><LI class="toclevel tocsection"><A href="#Project_Description" class="scroll"><SPAN id="whereYouAre"> Transcriptomics</SPAN></A></LI><LI class="toclevel nav-item active"><A href="#top" class="nav-link scroll"> cDNA Conversion</A></LI><LI class="toclevel nav-item"><A href="#Exp" class="nav-link scroll">  Experiment</A></LI><LI class="toclevel nav-item"><A href="#Res" class="nav-link scroll">  Results</A></LI><LI class="toclevel nav-item"><A href="#Tro" class="nav-link scroll"> Troubleshooting</A></LI><LI class="toclevel nav-item"><A href="#Disc" class="nav-link scroll"> Discussion</A></LI><LI class="toclevel nav-item"><A href="#Conc" class="nav-link scroll"> Conclusion</A></LI><LI class="toclevel nav-item"><A href="#Ref" class="nav-link scroll"> References</A></LI></UL></DIV><DIV class="content-text" id="scrolldown"><DIV class="card-holder"><H1 id="top">cDNA Conversion</H1><P> cDNA stands for complementary DNA, and can be described as a double stranded DNA that is made with RNA as a template. Our ultimate goal is to sequence the RNA that we have extracted - and with our sequencing method of choice, we need to turn the RNA back into DNA for it to be read properly. To do this, we use a technique known as reverse transcription to read and copy the RNA contents onto a newly made DNA strand, with no genetic information lost!
</P><H2 id="Exp">Experiment</H2><P>The aim of this experiment was the conversion of RNA to cDNA through reverse transcription primed by oligo-dT. The process consists of 3 major steps - complementary DNA strand synthesis, RNA digestion and synthesis of second strand [1]. The steps we conducted are described below:</P><H3>Synthesis of Complementary DNA Strand</H3><P>Due to previous polyA addition to 3'OH, all RNA molecules have similar sequence at 3' end which only differs in number of added adenine bases. This allows us to use polyT primers (Oxford Nanopore) to anneal to RNA template and reverse transcriptase (SuperScript IV, ThermoFisher) can initiate the transcription. 

A second, so-called strand switching primer is added to the reaction. This compensates for under-representations of 5´ends in cDNA by introducing an additional template and therefore protecting the terminal base pairs. Terminal transferase activity of the RT adds a number of deoxycytidine bases. The SSP primer is complementary to these bases and acts as an extended template for the RT, not only protecting the terminal bases, but also allowing to introduce sequence of choice into the newly synthesized first strand.</P><H3>RNA Template Digestion</H3><P>RNA template needs to be removed before the second DNA strand can be synthesized. This is done by adding ribonucleases (RNAse Cocktail Enzyme Mix, ThermoFischer) into the reaction and incubating. The enzyme mix consists of RNase A and T1.</P><H3>Second Strand Synthesis</H3><P>The second DNA strand is synthesized using LongAmp Taq Polymerase (NEB) incubated for one round. Primers used in the reaction are complementary to the sequences introduced by SSP and VNP primers. </P><H2 id="Res"> Results </H2><P>Initially, very low concentrations of cDNA were achieved. It has been decided to amplify the product using the same primers, which are used for 2nd strand synthesis in order to see if any cDNA was synthesized what so ever. After visulizing the amplified PCR product with cDNA as a template as seen in Figure 1, we could conclude that cDNA was in fact successfully synthesized. It has a similiar size distribution as bacterial mRNA, alhought strong band at 270 bp was of unclear origin. </P><P>The remaining task was to ensure that we synthesized enough cDNA. Eventually, we have managed to achieve high yields of cDNA after excessive trouble shooting, which revealed that the reason for low yield was inefficient polyadenylation as described <A href="https://2018.igem.org/Team:Uppsala/Transcriptomics/PolyA_Tailing"> here</A>.</P><DIV class="card-holder"><DIV class="content-card pic-next-to-text"><DIV class="side-text"><H3> cDNA Yield</H3><P>The amount of cDNA as measured by Qubit (Thermo Fisher). Typically, the cDNA amount would be equal to roughly twice the mass of the input RNA. For 125 ng of mRNA input we have synthesized (in an usual experiment) about 300 ng of cDNA. This amount was sufficient for further experiments.</P><H3> RNA Contamination</H3><P>cDNA which was synthesized during this experiment was used to prepare the sequencing library. Due to the suboptimal sequencing performance as described <A href="https://2018.igem.org/Team:Uppsala/Transcriptomics/Sequencing"> here</A>, we began to investigate among other factors the quality of the input cDNA. After extensive troubleshooting it was determined that RNA was still present in the cDNA samples despite digestion and cleaning steps. In several samples very high amount of RNA has been found, often corresponding to the input quantity. RNA content was measured using Qubit HS RNA Kit specific to RNA.  Unfortunately, an efficient way of removing RNA has not been found during the course of the project, which is the main reason for poor sequencing results.</P><P>The following section describes the various troubleshooting approaches to investigating the RNA contamination.</P></DIV><DIV class="side-img" style="background-color:darkolivegreen;"><P><B>Figure 1.</B> Amplified cDNA using primers provided in the kit shows that cDNA has been indeed synthesized. The sizes correspond approximately to size distribution of bacterial distribution. A strong band (270 bp) is seen which suggests preferential amplification of certain small fragments. </P></DIV></DIV></DIV><DIV class="card-holder"><H2 id="Tro">Troubleshooting</H2><H3>Does Mix of RNA/DNA Interfere with Qubit Measurement?</H3><H4>Hypothesis:</H4><P>The measured RNA in the sample could be due to lack of specificity of Qubit dye eg. RNA dye actually has affinity to DNA and therefore shows RNA in our samples.</P><H4>Experiment:</H4><P>Samples containing only RNA or DNA in concentration of 10 ng/µl were prepared in triplicate and each sample was measured with two different Qubit kits (RNA HS Kit, DNA HS Kit, Thermo Fisher). </P><H4>Results:</H4><P>  Measuring RNA (or DNA) with corresponding Kit resulted in expected results eg. around 10 ng of RNA and very low amount of DNA as shown in Table 1. Similar results were seen for DNA sample. When DNA was added into RNA, the measured RNA amount decreased, which was the only measuring bias we have observed. </P><P><B>Table 1.</B> The average of measured values for each of the samples. Measuring RNA (or DNA) with corresponding Kit resulted in expected results eg. around 10 ng of RNA and very low amount of DNA. </P><TABLE class="pgrouptable tablesorter our-table" style="width: 100%;" cellspacing="0" cellpadding="0"><THEAD><TR><TH style="“width:" auto="">Sample</TH><TH style="“width:" auto="">RNA [ng/µL]</TH><TH style="“width:" auto="">DNA [ng/µL]</TH><TH style="“width:" auto="">After adding DNA</TH><TH style="“width:" auto="">After adding RNA</TH></TR></THEAD><TBODY><TR><TD>RNA sample 10 ng/µl</TD><TD>
9.84
</TD><TD>
0.64
</TD><TD>6.86</TD><TD>-</TD></TR><TR><TD>DNA sample 10 ng/µl</TD><TD>
0.33
</TD><TD>
11.3
</TD><TD>-</TD><TD>
11.7
</TD></TR></TBODY></TABLE><H4>Conclusion</H4><P>The experiment only investigated interactions between dsDNA and ssRNA, which are not supposed to introduce any bias into the measurement as per manufacturer's information [2]. The remaining question is how would a RNA:DNA hybrid be treated by the dye. It is possible that RNA was not properly digested and therefore remains in the hybrid form. A hybrid could hypothetically be detected by both RNA and DNA specific kit. Unfortunately no proven hybrid sample was available and therefore this hypothesis could not be tested. 

It was concluded that having DNA - RNA mixture does not influence the measurement in a significant way. RNA amount is moderately decreased after addition of DNA into the sample, which would not explain the presence of RNA in cDNA samples. What remains to be investigated is how a RNA:DNA hybrid would influence the measurement. We therefore conclude that the RNA measurement in our samples is accurate and there is RNA present, either as a ssRNA or RNA:DNA hybrid. </P><H3>Is RNA Template Properly Digested? Is It Carried Over with the AMPure XP Beads?</H3><H4>Hypothesis:</H4><P>RNA is found in the final cDNA product as it is not digested properly. The RNase Cocktail contains RNAse A and T1, which are only supposed to introduce nicks into the RNA. Provided that LongAmp Taq polymerase does not degrade these fragment and that the purification beads (AMPure XP, Agencourt) do have affinity to both RNA and DNA, the RNA would be found in the sample. </P><H4>Experiment:</H4><P>This troubleshooting experiment had three major parts: i) optimizing the protocol for the use of RNase Cocktail using longer incubation time and/or larger quantity, ii) use of RNAse H for digestion and iii) clean-up step with purification beads to determine whether the beads have affinity for RNA.</P><P><B>1.</B> Total of 8 samples with 125 ng of RNA were prepared (from previous cDNA samples that shown RNA contamination, eg. there was DNA present in the samples as well). The experiment was optimized for time and amount of RNase Cocktail.</P><P><B>2.</B> RNA ladder was digested using both RNase H, RNAse Cocktail or both. Also each of the tests was prepared in either RNase-free water or buffer solution used during the cDNA synthesis. Digested DNA was visualised on a gel. </P><P><B>3.</B> 1000 ng of RNA ladder were purified using AMX beads and shown on a gel.</P><H4>Results:</H4><P>Overall, it was seen that subsequent digestion of RNA contaminated samples was successful regardless of used concentration as shown below. Digestion of RNA ladder using RNase H and/or RNase Cocktail shows complete digestion for RNase Cocktail while RNase H seems to not be working. The beads do carry over most of the RNA including rather small fragments (200 bp). The detailed results of each experiments are shown below:</P><P><B>1.</B></P><P><B>Table 2.</B> The table shows treatment for 9 samples of RNA and DNA mixture, each containing 125 ng of RNA. *Corresponds to experimental set-up used during the actual cDNA synthesis. </P><P><TABLE class="pgrouptable tablesorter our-table" style="width: 100%;" cellspacing="15" cellpadding="0"><THEAD><TR><TH style="“width:" auto="">Sample No.</TH><TH style="“width:" auto="">Incubation Time [min]</TH><TH style="“width:" auto="">RNase Cocktail [µl]</TH><TH style="“width:" auto="">RNA after treatment</TH></TR></THEAD><TBODY><TR><TD>
1*
</TD><TD>
10
</TD><TD>1</TD><TD>Not Detectable</TD></TR><TR><TD>2</TD><TD>
10
</TD><TD>
2
</TD><TD>Not Detectable</TD></TR><TR><TD>3</TD><TD>
10
</TD><TD>
0,5
</TD><TD>Not Detectable</TD></TR><TR><TD>4</TD><TD>
30
</TD><TD>
1
</TD><TD>Not Detectable</TD></TR><TR><TD>5</TD><TD>
30
</TD><TD>
2
</TD><TD>Not Detectable</TD></TR><TR><TD>6</TD><TD>
30
</TD><TD>
0,5
</TD><TD>Not Detectable</TD></TR><TR><TD>7</TD><TD>
45
</TD><TD>
1
</TD><TD>Not Detectable</TD></TR><TR><TD>8</TD><TD>
45
</TD><TD>
2
</TD><TD>Not Detectable</TD></TR><TR><TD>9</TD><TD>
45
</TD><TD>
0,5
</TD><TD>Not Detectable</TD></TR></TBODY></TABLE></P><P>According to measurement with Qubit RNA HS, all samples regardless of treatment show all RNA being degraded including sample, where treatment is identical to the one used in the actual experiment. </P><P><B>2.</B> The gel below in Figure 2 shows results of the digestion of RNA ladder with the available RNases, RNase H (line 5 and 6) and RNase Cocktail (line 7 and 8) or both (line 8 and 9).

From this experiment we can conclude that the buffer has no effect on digestion since sample in water and in reaction buffer appear the same. It can further be said that RNase Cocktail efficiently degrades the ladder, which is composed of ssRNA. RNase H seems to not digest the ladder at all. 

This corresponds with described ability of RNase H to preferentially digest RNA:DNA hybrids.</P></DIV><DIV class="card-holder"><P><B>Figure 2.</B> Results after digestion with RNases.</P><P><B>3.</B> The gel showed very significant carry over of RNA during the purification with AMPure beads (image not shown), which corresponds to manufacturer's information. If RNA was not properly digested, it would therefore be carried together with the cDNA during the entire library prep process.</P><H4>Conclusion</H4><P>Overall, the results of this troubleshooting procedure present results conflicting with result of the actual experiment. Here the RNases are shown to work very efficiently. Most likely, RNA used in troubleshooting has different properties that the RNA template in the cDNA synthesis (could be in a form of hybrid or some other unusual form) which leads to different digestion results.</P><H2 id="Disc">Discussion</H2><P>During the cDNA synthesis it was possible to achieve sufficiently high yields, usually exceeding double of the input mRNA amount. This cDNA was used to prepare sequencing libraries. As was shown later, it unfortunately contained undigested RNA, which significantly decrease the quality of sequencing results. We have therefore spend significant amount of time searching for the source of RNA contamination as reported in this section. 

The experiments above show very contradicting results. In section i) it can be seen that treatment of cDNA with RNase Cocktail after synthesis resulted in complete clearance of RNA from the sample. RNase Cocktail has also been shown to digest RNA ladder as visualized on the gel in Figure 1.

The same enzyme has always been used during the cDNA synthesis procedure and it remains unclear why does it efficiently digest RNA after synthesis or RNA ladder and would not work during the actual synthesis. 

One hypothesis that we considered was the presence of RNA:DNA hybrids, which would decrease efficiency of RNase Cocktail, which is more efficient in digesting ssDNA. RNase H was therefore added to address this issue but clearance of RNA from samples did not significantly increase. Even more confusingly, digestion did work in some of the samples despite the content/treatment being identical. </P><H2 id="Conc">Conclusion</H2><P>We have managed to successfully synthesize complementary DNA to our mRNA samples, which unfortunately did contain undigested RNA. A protocol needs to be developed that assures all of the RNA has been removed from the sample prior to the preparation of the library. Moreover, additional troubleshooting needs to be performed to determine why the digestion is not efficient. </P></DIV><DIV class="card-holder"><H2 id="Ref">References</H2><P><B>[1]</B> IDT, Use of template switching oligos (TS oligos, TSOs) for efficient cDNA library construction, [online], 2018 
 <A href="https://eu.idtdna.com/pages/education/decoded/article/use-of-template-switching-oligos-(ts-oligos-tsos)-for-efficient-cdna-library-construction">https://eu.idtdna.com/pages/education/decoded/article/use-of-template-switching-oligos-(ts-oligos-tsos)-for-efficient-cdna-library-construction</A></P><P><B>[2]</B> Thermo Fisher Scientific Inc, User Guide: Qubit RNA HS Assay Kits, [online], 2015 
 <A href="https://www.thermofisher.com/order/catalog/product/Q32852">https://www.thermofisher.com/order/catalog/product/Q32852</A></P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>