<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Uppsala_NotUnaG skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Uppsala/NotUnaG</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="wrapper-for-mobile"><NAV id="nav-mobile" class="nav-mobile" role="navigation" aria-labelledby="nav-mobile-toggleLabel" aria-hidden="true"><UL id="menu-mobile" class="menu-nav-mobile" role="menu-mobile" aria-orientation="vertical"><LI class="mobile-list-item-home" role=""><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala" role="menuitem" tabindex="-1"> Home </A></LI><LI class="mobile-list-item" role=""><A class="mobile-list-link" href="#" role="menuitem" tabindex="-1"> Project </A></LI><UL class="mobile-nav-ul"><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Worm_Culturing" role="menuitem" tabindex="-1"> Worm Culturing  </A></LI><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Transcriptomics" role="menuitem" tabindex="-1"> Transcriptomics </A></LI><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Phage_Display" role="menuitem" tabindex="-1"> Phage Display </A></LI><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Reporter_System" role="menuitem" tabindex="-1"> Reporter System  </A></LI><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Model" role="menuitem" tabindex="-1"> Modeling  </A></LI><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Demonstrate" role="menuitem" tabindex="-1"> Demonstrate  </A></LI><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/InterLab" role="menuitem" tabindex="-1"> InterLab  </A></LI><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Improve" role="menuitem" tabindex="-1"> Improve  </A></LI></UL><LI class="mobile-list-item" role=""><A class="mobile-list-link" href="#" role="menuitem" tabindex="-1"> Parts </A></LI><UL class="mobile-nav-ul"><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Parts" role="menuitem" tabindex="-1"> Parts  </A></LI><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Composite_Part" role="menuitem" tabindex="-1"> Composite Part  </A></LI></UL><LI class="mobile-list-item" role=""><A class="mobile-list-link" href="#" role="menuitem" tabindex="-1"> Team </A></LI><UL class="mobile-nav-ul"><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Team" role="menuitem" tabindex="-1"> Meet the Team  </A></LI><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Collaborations" role="menuitem" tabindex="-1"> Collaboration  </A></LI><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Attributions" role="menuitem" tabindex="-1"> Attribution  </A></LI><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Achievements" role="menuitem" tabindex="-1"> Achievements  </A></LI></UL><LI class="mobile-list-item" role=""><A class="mobile-list-link" href="#" role="menuitem" tabindex="-1"> Portfolio </A></LI><UL class="mobile-nav-ul"><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Notebook" role="menuitem" tabindex="-1"> Notebook  </A></LI><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Protocol" role="menuitem" tabindex="-1"> Protocol </A></LI></UL><LI class="mobile-list-item" role=""><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/  " role="menuitem" tabindex="-1"> Human Practice </A></LI><UL class="mobile-nav-ul"><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Human Practices" role="menuitem" tabindex="-1"> Human Practices  </A></LI><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Public_Engagement" role="menuitem" tabindex="-1"> Public Engagement  </A></LI><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Safety" role="menuitem" tabindex="-1"> Safety </A></LI></UL><LI class="mobile-list-item" role=""><A class="mobile-list-link" href="#" role="menuitem" tabindex="-1"> Results </A></LI><UL class="mobile-nav-ul" style="visibility: hidden;"><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Result" role="menuitem" tabindex="-1"> Resultat from the Lab </A></LI><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Achivements" role="menuitem" tabindex="-1">  Achievement </A></LI></UL></UL></NAV></DIV><DIV class="content blur-box" style="font-size:16px;"><DIV class="content-text"><H1 id="first-title">UnaG Troubleshooting</H1><P>One of the biobrick parts submitted by the 2016 Uppsala team was <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2003011"><B>UnaG combined with a histidine tag and a flexible linker</B></A> for extraction in affinity chromatography.  We decided to use this part as our reporter system when we read about it.  Mammalian intestines naturally have small amounts of bilirubin in them and also have a limited amount of oxygen present [3] (which is necessary for chromoprotein maturation [4]). we thought this could be a useful reporter.  It also came with a flexible linker, which could be used to potentially connect this reporter system with another output molecule that might be usable to act as a secondary reporter to help detect our target nematodes.  

While observing this part's sequence however, we found that there was an error and no histidine tag would be expressed due to the start codon being placed after the histidine tag.  In addition, this part would also express less or no UnaG at all due to the RBS now having a significant amount of space between it and the start codon.  We decided to incorporate this biobrick part into a custom composite part by moving the start codon to its proper location and then proving that the histidine tag works by extracting and purifying the protein via affinity chromatography.  In addition, we conducted a fluorescent bilirubin test and used a plate reader to determine if our new part expressed more UnaG than the 2016 part.  </P><P>However extraction of this protein poses some difficulty.  UnaG, many other chromoproteins, is a membrane protein [1] and therefore needs special conditions to purify.  The following report will show how we successfully extracted and purified UnaG from BL21 E. coli cells expressing our custom made plasmid.  
</P><DIV><DIV><DIV><H1>Design</H1><DIV class="inner-card-text"><P><FONT size="3"><B>Figure 1:</B> Our annotated modified UnaG sequence with an N-terminal his tag.  The terminator, RBS, and promoter sequences were all obtained from the iGEM website.  The UnaG gene was taken from the iGEM website and <B>only the start codon was moved</B> so that the gene would properly express with a histidine tag.  The start codon was previously immediately <B>after</B> the histidine tag.  Note that two plasmids were designed, one using the original UnaG part from the iGEM 2016 Uppsala team and one modified one as shown above.  The only modification between the two plasmids is the repositioned start codon. Our <A href="http://parts.igem.org/Part:BBa_K2669000"><B>composite part</B></A> and <A href="http://parts.igem.org/Part:BBa_K2669001"><B>basic part</B></A> can both be found on the iGEM registry site. </FONT></P><P> Note that we expressed both our modified composite part and the part from 2016 in a pUCIDT (Amp) backbone, which is a low copy plasmid backbone with ampicillin resistance. </P></DIV></DIV></DIV></DIV><DIV class="card-holder"><DIV><DIV><H1>Results</H1><P> Cell lysis and affinity chromotography were used to extract UnaG from our cells.  Bilirubin tests (addition of a small amount of bilirubin to samples) allowed us to see if the UnaG was present in our samples, since as mentioned earlier UnaG fluoresces in the presence of bilirubin.   </P><P><B>Figure 2:</B> Bilirubin test before/after affinity chromatography.  Going from right to left the samples are:</P><UL><LI> Lysed sample of the “bad” part before AC</LI><LI> Lysed sample of the “good” part before AC</LI><LI> &quot;Bad&quot; part after AC</LI><LI> &quot;Good&quot; part after AC</LI></UL><P>UnaG can be observed in all tubes except the third one, which should not have a histidine tag since we used the 2016 part that was on the iGEM registry and therefore it should not bind in the IMAC column. This supports our claim that our new part functions and provides a histidine tag to the protein, and the old part did not.</P><P><B>Figure 3: Comparison of blank tube to successful extraction/previous iGEM part. The tubes reading from left to right are as followed:</B></P><UL><LI> Blank tube with AC elution buffer/bilirubin</LI><LI> Tube with bilirubin + original iGEM UnaG part</LI><LI> Our extracted modified UnaG with a moved start codon, as can be seen in <B>Figure 1</B></LI></UL><P>A good degree of fluorescence can be seen in the last tube compared to the other two, which clearly contain none of our protein of interest. </P><P><B>Figure 4:</B> SDS-PAGE gel after affinity chromatography</P><P>UnaG is approximately 15.6 kDa, showing that it is indeed in the extracted sample.  Other proteins are shown, and this is likely because we used no imidazole in the initial running buffer, leading to unspecific binding.  We did this to ensure that we obtained as much UnaG as possible in our sample so that we could conduct fluorescence tests visible by the naked eye. </P><P><B>Figure 5:</B> Fluorescence measurement of unlysed cells. From left to right: Bacterial strain BL21 transformed with a plasmid containing <A href="http://parts.igem.org/Part:BBa_K2669000">Part:BBa_K2669000</A> from 2018, Bl21 transfected with <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2003011">Part:BBa_K2003011</A> from 2016 and normal BL21 cells, all at the same OD600 value. </P><P><B>Figure 6:</B> The supernantant of lysed cells before and after the &quot;His Gravitrap&quot; affinity chromotography.  Because of our lysis method UnaG was suspended in the supernatant of the cell cultures.  The left samples are supernantant containing the UnaG-protein from 2016 and the right samples are the supernantant containing our UnaG-protein (2018). </P><H1>Results Conclusion</H1><P>With the above experiments, we have shown that we successfully modified the 2016 UnaG part to maintain proper functionality while adding in a constiative promoter + RBS + double terminator.   </P><P>We have also shown that we have improved the “Inducible Green Fluorescent Protein UnaG+6xHis-tag+Flexible linker” protein part from the iGEM website by making it properly express a histidine tag that allows it to be extracted in affinity chromatography.  This is shown in <B>figures 3 and 4</B>.  In addition, we have also shown that we have an increased yield for UnaG than the previous iGEM part as can be seen in <B>figure 5</B>. <B>Figure 6</B> shows that even using the plate reader there is little to no UnaG present in the 2016 sample after IMAC, which suggests a histidine tag was not expressing.  The combination of fluorescence after IMAC purification and the correct sized band on the gel proves our biobrick part functions as intended.  In addition, we developed a simple protocol to extract membrane proteins, which are traditionally notoriously difficult to extract.  
                            
                            The usage of the Triton X-100 incubation step theoretically created micelles in the solution, allowing membrane fragments to float around and protect UnaG since it is a beta barrel integral membrane protein that is quite hydrophobic in nature (Kumagai A, 2013).  It was also found in the literature that using 5-10% glycerol [2] in all solutions involved in the extraction of integral membrane proteins is advised and theoretically helps keep them stabilized.  
                            
                            It may have been beneficial to express UnaG in a low copy plasmid, which might have lessened the risk that these heterologous proteins would conglomerate.  We also chose to express UnaG in a constitutive manner, since we previously had no trouble expressing RFP or GFP (proteins with similar structures and environments) constitutively.  This saved us time and also provided one less “moving part” that could go wrong in our experiment, such as an induction system not working properly.   
                            </P><H1>Protocol</H1><H2>Transforming the Plasmid:</H2><P>When the plasmids were received from IDT they were transformed into BL21 E. coli cells graciously provided to us by the esteemed Forster Laboratory.  Same-day-made competent cells using the protocol from the “Synthetic Biology Handbook”  were used to provide maximum transformation efficiency.  </P><H2>Extraction of UnaG:</H2><P>The protocol for the extraction of our integral membrane protein from the transformed BL21 cells proceeded as follows: Note that this was done for both iGEM 2016 cells transformed with the previous part (nicknamed “bad”) and our repositioned start codon (graced with the moniker “good”).  </P><H3>Materials/Procedure</H3><UL><LI><B>Lysis Buffer:</B> PBS solution with 1mM EDTA, 5% glycerol, and 20mM Tris, pH7.4</LI><LI><B>Elution Buffer:</B> 20 mM sodium phosphate, 0.5 M NaCl, 0.5 M imidazole, pH 7.4, 5% glycerol
                                        PBS, 1mM EDTA, 5% glycerol, 20mM Tris, pH7.4</LI><LI><B>Binding/Washing Buffer:</B>0.5 M NaCl, 2 EDTA-free tablets, 10 % glycerol, 20mM sodium phosphate, 1% Triton x100, pH 7.4 (400 mL total)</LI><LI><B>Binding/washing buffer</B> with 1% triton x-100 by weight</LI></UL><P>Cells were centrifuged at 4000 g 25 minutes at 4 degrees Celsius and then resuspended in Lysis buffer.  Cells were lysed using cell disruption with a french press.  The now lysed cells were then centrifuged again at  at 4000 g 25 minutes at 4 degrees Celsius.  The pellet was resuspended in 20mL binding/washing buffer with 1% triton x-100.  The solution was incubated on ice for one hour before another round of centrifugation at the same temperature and speed.  After centrifugation the supernatant should contain the protein of interest.  Bilirubin tests were conducted on both solutions of the pellet and supernatant to observe any fluorescence under a UV light.  </P><P>Affinity chromatography was then performed on both “good” and “bad” solutions using prepacked “His-Gravitrap” Columns from GE Healthcare.  The protocol for use was performed according to GE healthcare’s specifications, with modified binding/washing/elution buffers.  After affinity chromatography, the resulting elutants were tested for fluorescence with a bilirubin test.  </P><H1> References </H1><P><B>[1] </B>Kumagai A, Ando R, Miyatake H, Greimel P, Kobayashi T, Hirabayashi Y, 
Shimogori T, Miyawaki A. 2013. A Bilirubin-Inducible Fluorescent Protein from Eel Muscle. Cell 153: 1602–1611.<B>[2] </B>Patel H, Tsamaloukas A, Heerklotz H. The Effect Of Glycerol On Membrane Solubilization By Nonionic 
Surfactants. Biophysics 96: 163A-164A </P><P><B>[3] </B> Bowen R. Microbial Life in the Digestive Tract. online: http://www.vivo.colostate.edu/hbooks/pathphys/digestion/basics/gi_bugs.html. Accessed October 12, 2018.</P><P><B>[4] </B> Engineering a palette of eukaryotic chromoproteins for bacterial synthetic biology | Journal of Biological Engineering | Full Text. online: https://jbioleng.biomedcentral.com/articles/10.1186/s13036-018-0100-0. Accessed October 12, 2018. </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>