<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Uppsala_Transcriptomics skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Uppsala/Transcriptomics</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="wrapper-for-mobile"><NAV id="nav-mobile" class="nav-mobile" role="navigation" aria-labelledby="nav-mobile-toggleLabel" aria-hidden="true"><UL id="menu-mobile" class="menu-nav-mobile" role="menu-mobile" aria-orientation="vertical"><LI class="mobile-list-item-home" role=""><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala" role="menuitem" tabindex="-1"> Home </A></LI><LI class="mobile-list-item" role=""><A class="mobile-list-link" href="#" role="menuitem" tabindex="-1"> Project </A></LI><UL class="mobile-nav-ul"><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Worm_Culturing" role="menuitem" tabindex="-1"> Worm Culturing  </A></LI><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Transcriptomics" role="menuitem" tabindex="-1"> Transcriptomics </A></LI><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Phage_Display" role="menuitem" tabindex="-1"> Phage Display </A></LI><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Reporter_System" role="menuitem" tabindex="-1"> Reporter System  </A></LI><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Model" role="menuitem" tabindex="-1"> Modeling  </A></LI><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Demonstrate" role="menuitem" tabindex="-1"> Demonstrate  </A></LI><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/InterLab" role="menuitem" tabindex="-1"> InterLab  </A></LI><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Improve" role="menuitem" tabindex="-1"> Improve  </A></LI></UL><LI class="mobile-list-item" role=""><A class="mobile-list-link" href="#" role="menuitem" tabindex="-1"> Parts </A></LI><UL class="mobile-nav-ul"><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Parts" role="menuitem" tabindex="-1"> Parts  </A></LI><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Composite_Part" role="menuitem" tabindex="-1"> Composite Part  </A></LI></UL><LI class="mobile-list-item" role=""><A class="mobile-list-link" href="#" role="menuitem" tabindex="-1"> Team </A></LI><UL class="mobile-nav-ul"><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Team" role="menuitem" tabindex="-1"> Meet the Team  </A></LI><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Collaborations" role="menuitem" tabindex="-1"> Collaboration  </A></LI><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Attributions" role="menuitem" tabindex="-1"> Attribution  </A></LI><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Achievements" role="menuitem" tabindex="-1"> Achievements  </A></LI></UL><LI class="mobile-list-item" role=""><A class="mobile-list-link" href="#" role="menuitem" tabindex="-1"> Portfolio </A></LI><UL class="mobile-nav-ul"><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Notebook" role="menuitem" tabindex="-1"> Notebook  </A></LI><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Protocol" role="menuitem" tabindex="-1"> Protocol </A></LI></UL><LI class="mobile-list-item" role=""><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/  " role="menuitem" tabindex="-1"> Human Practice </A></LI><UL class="mobile-nav-ul"><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Human Practices" role="menuitem" tabindex="-1"> Human Practices  </A></LI><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Public_Engagement" role="menuitem" tabindex="-1"> Public Engagement  </A></LI><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Safety" role="menuitem" tabindex="-1"> Safety </A></LI></UL><LI class="mobile-list-item" role=""><A class="mobile-list-link" href="#" role="menuitem" tabindex="-1"> Results </A></LI><UL class="mobile-nav-ul" style="visibility: hidden;"><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Result" role="menuitem" tabindex="-1"> Resultat from the Lab </A></LI><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Achivements" role="menuitem" tabindex="-1">  Achievement </A></LI></UL></UL></NAV></DIV><DIV class="body"><DIV class="content blur-box" style="font-size:16px;"><DIV id="toc" class="toc"><UL><LI class="toclevel tocsection"><A href="#Project_Description" class="scroll"><SPAN id="whereYouAre"> Transcriptomics  </SPAN></A></LI><LI class="toclevel nav-item active"><A href="#top" class="nav-link scroll"> Introduction </A></LI><LI class="toclevel nav-item"><A href="#Lysis" class="nav-link scroll">  Worm Culturing and Lysis of Bacteria  </A></LI><LI class="toclevel nav-item"><A href="#Total" class="nav-link scroll">  Total RNA Purification</A></LI><LI class="toclevel nav-item"><A href="#Dep" class="nav-link scroll"> rRNA Depletion</A></LI><LI class="toclevel nav-item"><A href="#Poly" class="nav-link scroll"> Poly(A)-Tailing</A></LI><LI class="toclevel nav-item"><A href="#Conv" class="nav-link scroll"> cDNA Conversion</A></LI><LI class="toclevel nav-item"><A href="#Bar" class="nav-link scroll"> Barcoding + Library Preparation</A></LI><LI class="toclevel nav-item"><A href="#Seq" class="nav-link scroll"> Sequencing</A></LI><LI class="toclevel nav-item"><A href="#Info" class="nav-link scroll"> Bioinformatics</A></LI></UL></DIV><DIV class="content-text" id="scrolldown"><DIV class="card-holder"><H1>Introduction</H1><P id="top">The idea behind the work of the transcriptomics group is to try and detect any changes in gene expression when <I>E. coli</I> is grown alongside strongyles as opposed to just being grown in a normal lab environment. The aim for this undertaking is to find any and all genes which might be expressed exclusively in proximity to the worms - giving us a prime opportunity to work some biotech magic and find a way to make a diagnostics tool that would allow the bacteria to detect strongyle parasites by using this bacteria gene.

The way we decided to approach this was by running a full-scale transcriptome sequencing on the entire <I>E. coli</I> genome, both in bacteria grown normally and grown alongside the worms. By doing this, we would be able to quantitatively determine the differences in gene expression and pick out the expressions that are different.

For this, we created an eight-step pipeline starting from lysing the bacteria to extracting their RNA contents, to refining, and finally sequencing the genetic material using Oxford Nanopore’s MinION device - which you plug in to your laptop. That same laptop will then be used to understand what exactly is going on in the bacteria.
 </P></DIV><DIV class="card-holder"><P align="center"><STRONG>Figure 1:</STRONG> Flowchart with all steps of the transcriptomics outline.</P></DIV><DIV class="card-holder"><H1 id="Lysis"><STRONG>1.</STRONG> Worm Culturing and Lysis of Bacteria</H1></DIV><DIV class="card-holder"><DIV class="content-card pic-next-to-text"><DIV class="side-text"><P><I>E. coli</I> cells are cultured in minimal media (M9), exposed to the nematodes and lysed. A special reagent (RNAProtect, QIAGEN) is used to assure RNA is preserved during the process. It is important that the cells are harvested during their log phase, which ensures the maximum yield of information since genetic expression is at its highest. </P><DIV class="card-holder"><DIV class="buttons"><DIV class="container-button"><A href="https://2018.igem.org/Team:Uppsala/Worm_Culturing" class="btn"> Find Out More! </A></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="card-holder"><H1 id="Total"><STRONG>2.</STRONG> Total RNA Purification</H1></DIV><DIV class="card-holder"><DIV class="content-card pic-next-to-text"><DIV class="side-text"><P>The RNA is extracted using QIAGEN’s RNeasy Kit. Two cultures are purified in sufficient amounts to allow duplicates - one worm group and one control group. The RNA is quality checked and then stored. </P><DIV class="card-holder"><DIV class="buttons"><DIV class="container-button"><A href="https://2018.igem.org/wiki/index.php?title=Team:Uppsala/Transcriptomics/Total_RNA_Purification" class="btn"> Find Out More! </A></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="card-holder"><H1 id="Dep"><STRONG>3.</STRONG> rRNA Depletion</H1></DIV><DIV><DIV class="content-card pic-next-to-text"><DIV class="side-text"><P>The ribosomal RNA - rRNA, does not hold any relevant genetic information for us and makes up about 90% of the total RNA content of the cell. Using Thermo Fishers MICROBExpress Kit, the rRNA is removed to get rid of genetic static and to make future steps easier in terms of total material that needs to be processed.</P><DIV class="card-holder"><DIV class="buttons"><DIV class="container-button"><A href="https://2018.igem.org/wiki/index.php?title=Team:Uppsala/Transcriptomics/rRNA_Depletion" class="btn"> Find Out More! </A></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="card-holder"><H1 id="Poly"><STRONG>4.</STRONG> Poly(A)-Tailing</H1></DIV><DIV class="card-holder"><DIV class="content-card pic-next-to-text"><DIV class="side-text"><P>Poly(A)-tails consists of adenosine nucleotides that are added onto the 3’ end of the nucleotide strand in mostly eukaryotes for translation purposes. Since these polyA tails are not present in bacteria, they need to be added onto the ends of our RNA to work as a primer target in the next step of the process.</P><DIV class="card-holder"><DIV class="buttons"><DIV class="container-button"><A href="https://2018.igem.org/Team:Uppsala/Transcriptomics/PolyA_Tailing" class="btn"> Find Out More! </A></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="card-holder"><H1 id="Conv"><STRONG>5.</STRONG> cDNA Conversion</H1></DIV><DIV class="card-holder"><DIV class="content-card pic-next-to-text"><DIV class="side-text"><P>Using our newly added Poly(A)-tails as a primer target, we subject our RNA to reverse transcription to create complementary DNA copies - or cDNA. The RNA needs to be converted into DNA to be sequenced with the MinION device. The original RNA template is then degraded and a second strand is synthesized onto the cDNA.</P><DIV class="card-holder"><DIV class="buttons"><DIV class="container-button"><A href="https://2018.igem.org/Team:Uppsala/Transcriptomics/cDNA_Conversion" class="btn"> Find Out More! </A></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="card-holder"><H1 id="Bar"><STRONG>6.</STRONG> Barcoding + Library Preparation</H1></DIV><DIV class="card-holder"><DIV class="content-card pic-next-to-text"><DIV class="side-text"><P>A necessary step to be able to make sense of the sequenced data is to attach so-called barcodes to the end of each cDNA strand. Barcodes are small sequences which are known beforehand and allows us to differentiate between different samples. Because two transcriptome samples will be sequenced at the same time, this will let us know which sequence came from which sample. </P><P>One final step before sequencing can begin is to attach the adaptor sequences onto the ends of the barcodes. These specialized adaptors allows the pores of the sequencing device to recognize the cDNA-strand and move it into the sequencing machinery.</P><DIV class="card-holder"><DIV class="buttons"><DIV class="container-button"><A href="https://2018.igem.org/Team:Uppsala/Transcriptomics/Barcoding-Library_Preparation" class="btn"> Find Out More! </A></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="card-holder"><H1 id="Seq"><STRONG>7.</STRONG> Sequencing</H1></DIV><DIV class="card-holder"><DIV class="content-card pic-next-to-text"><DIV class="side-text"><P>The MinION device, produced by Oxford Nanopore Technologies, works by applying an electrical current and pulling the DNA strand through small biological pores. This movement causes changes in the electrical current also passing through the pore - and the charge of the current varies depending on which nucleotide of the DNA strand is passing through at the moment. This is what enables the sequencing - the current changes are translated into the nucleotide order of the DNA strand.</P><DIV class="card-holder"><DIV class="buttons"><DIV class="container-button"><A href="https://2018.igem.org/Team:Uppsala/Transcriptomics/Sequencing" class="btn"> Find Out More! </A></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="card-holder"><H1 id="Info"><STRONG>8.</STRONG> Bioinformatics</H1></DIV><DIV class="card-holder"><DIV class="content-card pic-next-to-text"><DIV class="side-text"><P>With the genetic information of the samples finally deciphered, it is time to match it to known genes in the <I>E.coli</I> genome and quantify the amount of genes that was expressed way back before step one of this pipeline. This allows us to see the possible difference in gene expression between the <I>E.coli</I> worm samples and the <I>E.coli</I> control samples.</P><DIV class="card-holder"><DIV class="buttons"><DIV class="container-button"><A href="https://2018.igem.org/Team:Uppsala/Transcriptomics/Bioinformatics" class="btn"> Find Out More! </A></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="card-holder"><DIV class="buttons"><DIV class="container-button"><A href="https://2018.igem.org/Team:Uppsala/Notebook" class="btn"> Notebook </A><A href="https://2018.igem.org/Team:Uppsala/Protocol" class="btn"> Protocol </A></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>