<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Uppsala_Transcriptomics_Total_RNA_Purification skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Uppsala/Transcriptomics/Total RNA Purification</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="wrapper-for-mobile"><NAV id="nav-mobile" class="nav-mobile" role="navigation" aria-labelledby="nav-mobile-toggleLabel" aria-hidden="true"><UL id="menu-mobile" class="menu-nav-mobile" role="menu-mobile" aria-orientation="vertical"><LI class="mobile-list-item-home" role=""><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala" role="menuitem" tabindex="-1"> Home </A></LI><LI class="mobile-list-item" role=""><A class="mobile-list-link" href="#" role="menuitem" tabindex="-1"> Project </A></LI><UL class="mobile-nav-ul"><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Worm_Culturing" role="menuitem" tabindex="-1"> Worm Culturing  </A></LI><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Transcriptomics" role="menuitem" tabindex="-1"> Transcriptomics </A></LI><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Phage_Display" role="menuitem" tabindex="-1"> Phage Display </A></LI><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Reporter_System" role="menuitem" tabindex="-1"> Reporter System  </A></LI><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Model" role="menuitem" tabindex="-1"> Modeling  </A></LI><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Demonstrate" role="menuitem" tabindex="-1"> Demonstrate  </A></LI><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/InterLab" role="menuitem" tabindex="-1"> InterLab  </A></LI><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Improve" role="menuitem" tabindex="-1"> Improve  </A></LI></UL><LI class="mobile-list-item" role=""><A class="mobile-list-link" href="#" role="menuitem" tabindex="-1"> Parts </A></LI><UL class="mobile-nav-ul"><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Parts" role="menuitem" tabindex="-1"> Parts  </A></LI><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Composite_Part" role="menuitem" tabindex="-1"> Composite Part  </A></LI></UL><LI class="mobile-list-item" role=""><A class="mobile-list-link" href="#" role="menuitem" tabindex="-1"> Team </A></LI><UL class="mobile-nav-ul"><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Team" role="menuitem" tabindex="-1"> Meet the Team  </A></LI><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Collaborations" role="menuitem" tabindex="-1"> Collaboration  </A></LI><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Attributions" role="menuitem" tabindex="-1"> Attribution  </A></LI><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Achievements" role="menuitem" tabindex="-1"> Achievements  </A></LI></UL><LI class="mobile-list-item" role=""><A class="mobile-list-link" href="#" role="menuitem" tabindex="-1"> Portfolio </A></LI><UL class="mobile-nav-ul"><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Notebook" role="menuitem" tabindex="-1"> Notebook  </A></LI><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Protocol" role="menuitem" tabindex="-1"> Protocol </A></LI></UL><LI class="mobile-list-item" role=""><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/  " role="menuitem" tabindex="-1"> Human Practice </A></LI><UL class="mobile-nav-ul"><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Human Practices" role="menuitem" tabindex="-1"> Human Practices  </A></LI><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Public_Engagement" role="menuitem" tabindex="-1"> Public Engagement  </A></LI><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Safety" role="menuitem" tabindex="-1"> Safety </A></LI></UL><LI class="mobile-list-item" role=""><A class="mobile-list-link" href="#" role="menuitem" tabindex="-1"> Results </A></LI><UL class="mobile-nav-ul" style="visibility: hidden;"><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Result" role="menuitem" tabindex="-1"> Resultat from the Lab </A></LI><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Achivements" role="menuitem" tabindex="-1">  Achievement </A></LI></UL></UL></NAV></DIV><DIV class="body"><DIV class="content blur-box" style="font-size:16px;"><DIV id="toc" class="toc"><UL><LI class="toclevel tocsection"><A href="#Project_Description" class="scroll"><SPAN id="whereYouAre"> Transcriptomics </SPAN></A></LI><LI class="toclevel nav-item active"><A href="#Puri" class="nav-link scroll"> Total RNA Purification</A></LI><LI class="toclevel nav-item"><A href="#Exp" class="nav-link scroll">  Experiment</A></LI><LI class="toclevel nav-item"><A href="#Res" class="nav-link scroll">  Results</A></LI><LI class="toclevel nav-item"><A href="#Conc" class="nav-link scroll">  Conclusions</A></LI><LI class="toclevel nav-item"><A href="#References" class="nav-link scroll"> References </A></LI></UL></DIV><DIV class="content-text" id="scrolldown"><DIV class="card-holder"><H1 id="Puri">Total RNA Purification</H1></DIV><DIV class="card-holder"><P>Separating the total RNA content from the bacterial culture is the first step in determining how the organisms’ genes are being expressed. Because each mRNA strand is a product of an activated gene, they can be used as a powerful indicator to see what happens, and what doesn’t happen, in the bacterial cell! In the coming steps of the pipeline, the RNA will be gradually processed into a form which can be interpreted and sequenced, allowing us to figure out what the bacteria are up to.

                        Total RNA purification can be divided into two steps - cell lysis and RNA extraction. Both of these steps are covered by using Qiagen’s RNeasy Mini Kit - which lets us rupture the cell membranes of the bacterial cells, freeing the RNA - and then filter out the RNA from the lysate by spinning it through a specialized silica membrane [1].

                        To ensure the RNA is in good shape, we have employed several quality controls. The purified RNA samples are measured and checked for any impurities and contaminants using Thermo Fishers NanoDrop [2]. In addition, all samples are analyzed by gel electrophoresis in order to find any damage to the RNA itself. In addition, we have used Agilent's BioAnalyzer device which measures the total RNA sample's nucleotide sequence sizes as a final quality control [3].

                        Lastly, working with RNA requires a very sterile environment. Enzymes called RNases are present everywhere: on tables, lab instruments, and especially on the skin and clothes of unsuspecting lab scientists. They are hardy and do not degrade easily, and inadvertently getting them into a sample means that the RNA content will be severely damaged [4]. Because of this, we have used nuclease-free reagents, done regular lab cleanings with DEPC, and employed a general gloves-only rule [5]. </P><H2 id="Exp">Experiment</H2><H3>Qiagen RNeasy kit:</H3><P>Qiagens RNeasy kit purifies bacterial RNA from harvested cell culture. The process can be divided into two steps: cell lysis and RNA extraction.

                        The cell lysis is done by suspension in TE-buffer containing lysozyme and proteinase K, which permeates the cell wall and denatures the cell protein content, respectively. This frees the nucleic acid contents of the cell, while the remaining contents of the TE-buffer ensures a stable pH-environment and deactivates any present DNase and RNase enzymes, which may cause nucleic acid degradation. RLT buffer with added 2-mercaptoethanol is then added to the sample. The RLT buffer includes guanidine salts which enhances the silica membranes’ affinity for RNA. Meanwhile, the 2-mercaptoethanol serves as a powerful reducing agent which denatures any remaining protein - including RNases - present in the sample [1].</P><P>The sample is then washed with ethanol, and is subjected to centrifugation through the RNeasy silica membrane several times along with several washing buffers (RW1, RPE), which are designed to remove any debris, salts and contaminants as the lysate moves through the membrane. This includes the active agents added previously throughout the procedure. Before the washing is finished, DNase is also added to the sample. DNase degrades the DNA content of the cell and ensures that only RNA is collected through the membrane. This enzyme is also removed at the end of the washing procedure. The collected RNA content of the membrane can then be collected by washing it with pure water, yielding a volume of pure RNA [1].</P><H3>Gel Electrophoresis:</H3><P>Good quality RNA which has been cast on a gel is characterized by the two strong ribosomal bands, 16S and 23S. They are present at the 1,5kb mark and 2,9kb, respectively. This ribosomal RNA (rRNA) is about 90% of the total RNA content of the cell, which makes it a good indicator for purified RNA quality. A degraded RNA sample will show weak ribosomal bands and usually a large smear present over the gel lane.</P><H3>Nanodrop:</H3><P>Thermo Fishers Nanodrop uses a droplet of the RNA sample to check the absorbance of the contents within by photospectrometry. High absorbance values at certain wavelengths indicate how much RNA, or unwanted contaminants, are present in the sample. Important values, apart from the RNA content of the sample, are the ratios of 260nm/280nm as well as 260nm/230nm. 260/280 and 260/230 values above 2,0 indicate an RNA sample free of any contaminants such as DNA, salts or chemicals [2].</P><H3>Bioanalyzer:</H3><P>Agilent's Bioanalyzer runs an elecrophoresis measurement on the RNA sample and uses it to determine the amount of nucleotide sequences in the sample, as well as the sizes of them. It also awards a RIN-score to each sample, where 1 indicates a severely degraded sample and 10 indicates a very well preserved sample[3].</P><H3>RNases and Sterile Work Technique:</H3><P>RNases are nuclease proteins whose function is to break down RNA in its surrounding environment. They are employed in both bacterial and eukaryotic cells where they naturally break down RNA strands that are no longer needed by the cell. In addition, RNases are secreted through the skin, and are thus everywhere on the body and quickly spreads through the environment, through the air or by contact with objects. RNases are very stable proteins, able to resist degradation by ethanol cleanings and remains active in temperatures above 80 degrees celsius. Because of this, the best way to avoid them is to work as sterile as possible, preferably in a lab not shared by other people, and with gloves on at all time. Certain powerful denaturing agents such as 2-mercaptoethanol are used in the total RNA purification as an extra measure to avoid RNA degradation by any RNases [1].</P><H3>Nuclease-Free Reagents and DEPC:</H3><P>Any reagents risk being contaminated by nucleases over time as you work with them. To minimize the risk of having results damaged by nuclease presence, we have only used nuclease-free reagents and used caution while in the lab to avoid contamination. In addition, we’ve cleaned spaces and lab tools with DEPC, which is a chemical capable of inactivating RNase [3, 4].</P><H2 id="Res">Result</H2><P>Our RNA purification yielded twelve RNA samples which can be divided into two groups - the initial eluate as well as the second eluate which was collected from the same experiment. The second eluate contains less nucleic acid and is mainly used for gel electrophoresis quality control. 

                        Out of these two groups, half are control groups which has had no contact with the strongyles. The second group is the worm group which was grown alongside with the strongyles. Lastly, the RNA samples were made with either 1,6ml, 1,7ml or 1,8ml of cell culture input. </P><P><B>Table 1.</B> This table shows the nucleic acid concentrations, 260/280, and 260/230 ratios of our purified samples. Looking at the samples, “6c” stands for control group, while “6w” stands for worm group. “16”, “17” or “18” stands for the initial cell input into the purification - either 1,6ml, 1,7ml or 1,8ml. “elute2” stands for the second eluate of the purification, which is used for gel electrophoresis and not used further. The nucleic acid concentrations of the samples vary strongly - from 130ng/μl to 2100ng/μl. There is also a large variance in the 260/230 and 260/280 ratios which measure sample purity - values between 1.9-2.0 can be seen as acceptable. Lastly, the last column shows the volume (in μl) needed to reach 10ug of total nucleic acid content, which is the base input for rRNA depletion - the next step of the transcriptomics project. </P><TABLE class="pgrouptable tablesorter our-table" style="width: 100%;" cellspacing="15" cellpadding="0"><THEAD><TR><TH style="“width:" auto="">Sample</TH><TH style="“width:" auto="">Nucleic Acid Conc. [ng/uL]</TH><TH style="“width:" auto="">a260/280</TH><TH style="“width:" auto="">260/230</TH><TH style="“width:" auto="">Input for rRNA depletion (for 10 ug)</TH></TR></THEAD><TBODY><TR><TD>6c16</TD><TD>276,6</TD><TD>2,1</TD><TD>1,3</TD><TD>36,3</TD></TR><TR><TD>6w16</TD><TD>1202,0</TD><TD>2,1</TD><TD>2,1</TD><TD>8,2</TD></TR><TR><TD>6c17</TD><TD>1833,0</TD><TD>2,1</TD><TD>1,9</TD><TD>5,5</TD></TR><TR><TD>6w17</TD><TD>2148,1</TD><TD>2,1</TD><TD>2,0</TD><TD>4,65</TD></TR><TR><TD>6c18</TD><TD>335,2</TD><TD>2,1</TD><TD>1,7</TD><TD>29,8</TD></TR><TR><TD>6w18</TD><TD>133,6</TD><TD>2,1</TD><TD>1,1</TD><TD>74,8</TD></TR><TR><TD>6c16 elute 2</TD><TD>179,7</TD><TD>2,1</TD><TD>2,0</TD><TD>-</TD></TR><TR><TD>6w16 elute 2</TD><TD>1587,0</TD><TD>2,1</TD><TD>2,2</TD><TD>-</TD></TR><TR><TD>6c17 elute 2</TD><TD>721,0</TD><TD>2,1</TD><TD>2,2</TD><TD>-</TD></TR><TR><TD>6w17 elute 2</TD><TD>359,9</TD><TD>2,1</TD><TD>2,0</TD><TD>-</TD></TR><TR><TD>6c18 elute 2</TD><TD>244,4</TD><TD>2,1</TD><TD>1,8</TD><TD>-</TD></TR><TR><TD>6w18 elute 2</TD><TD>162.8</TD><TD>2,2</TD><TD>1,4</TD><TD>-</TD></TR></TBODY></TABLE><P>The second eluates of the purification were subjected to gel electrophoresis to determine the integrity of the RNA:</P></DIV><DIV class="card-holder"><DIV class="content-card pic-next-to-text"><DIV class="side-text"><P align="left">We deemed all samples of the purification to have good integrity, judging by the strong ribosomal bands seen on the gel electrophoresis. As can be seen on sample W18 (see Figure 1), there is a faint strong band above the rest of the highway. We suspected this to be possible DNA contamination.

                                As can be seen on <B>Table 1</B>, there was significant variance in nucleic acid concentration and purity between the samples. Samples 6c17 and 6w17 had good nucleic acid concentrations and purity ratios according to Nanodrop - therefore we decided to run a Bioanalyzer measurement on them to confirm their quality.</P><P>
                                While all samples have high 260/280 ratios which imply little/no DNA contamination, some samples with low nucleic acid concentration also has low 260/230 ratios. A low 260/230 ratio can imply guanidine salt contamination - a compound used in the purification by binding to RNA and reducing the RNAs affinity to water. Ethanol is later added to disassociate the guanidine salt bindings [6]. We believe this to have failed in some of the samples, leaving the guanidine salts bound to RNA which might make them unable to be detected by using NanoDrop.</P></DIV><DIV class="side-img" style="background-color:darkolivegreen;"><P><B>Figure 1.</B> This figure depicts the successful RNA purification of all six samples. The cell input increases from left to right - from 1,6ml (16) to 1,7ml (17) and finally 1,8ml (18). “C” stands for control (culture not grown alongside strongyles) while “W” stands for worm sample (co-cultured with strongyles). All samples display clear ribosomal bands at the 2,9kb and 1,5kb marks, which implies RNA integrity. A faint band can be seen above the main bands on the W18 sample. Note that due to the varying nucleic acid concentrations of the samples, all inputs onto the gel were adjusted to 1μg.</P></DIV></DIV></DIV><DIV class="section"><DIV class="card-holder"><P><B>Figure. 2</B> These graphs depict the Bioanalyzer results from our two samples 6c17 (to the left, labeled &quot;tc&quot;) and 6w17 (to the right, labeled &quot;ts&quot;). Both samples show strong peaks at the 1,5kb and 2,9kb marks, indicating the ribosomal RNA. </P><P> The bioanalyzer graphs (see figure 2) show that both RNA samples seem to be of good quality, with sharp peaks at the sites where the rRNA is expected to be. The rest of the curve increases indicated on the graphs represent the remaining RNA. The RIN scores for the samples are 9,80 and 9,90, which indicate a very low level of degradation. Judging by this, the NanoDrop measurements and the gel electrophoresis, we deemed the samples to be of good enough quality to be used in the next step. </P></DIV><DIV class="card-holder"><H2 id="Conc">Conclusion</H2><P>In this particular total RNA purification, we experienced quite a lot of variation between the purified samples. The general baseline results we are looking for in a total RNA is at least 10000ng of total nucleic acid material (in general, a concentration of 1000ng/ul is sufficient.) and 260/280 and 260/230 ratios of at least 1,9-2,0. While some samples were clearly contaminated and had low RNA concentrations, two samples (6c17 and 6w17) showed good quality in all measurement methods we employed. Because of this, we opted to move forward to the next step of the pipeline with them. </P></DIV><DIV class="card-holder"><H2 id="References">References</H2><P><B>[1]</B> QIAGEN, 2018. RNAprotect Bacteria Reagent Handbook. <A href="https://www.qiagen.com/us/resources/resourcedetail?id=95346297-ae79-41a8-9259-e48f702d4f36&amp;lang=en">RNAprotect Bacteria Reagent Handbook</A> Date of Visit 2018-10-15</P><P><B>[2]</B> Thermo Fisher, 2018. NanoDrop: How it works. <A href="https://www.thermofisher.com/se/en/home/industrial/spectroscopy-elemental-isotope-analysis/molecular-spectroscopy/ultraviolet-visible-visible-spectrophotometry-uv-vis-vis/uv-vis-vis-instruments/nanodrop-microvolume-spectrophotometers/nanodrop-products-guide/nanodrop-how-it-works.html">Thermo Fisher, 2018. NanoDrop: How it works</A> Date of Visit 2018-10-15</P><P><B>[3]</B> Agilent, 2018. Agilent 2100 Bioanalyzer system 2100 Expert Software User's Guide. <A href="https://www.agilent.com/cs/library/usermanuals/public/2100_Bioanalyzer_Expert_USR.pdf">Agilent 2100 Bioanalyzer system 2100 Expert Software User's Guide</A> Date of Visit 2018-10-15</P><P><B>[4]</B> Thermo Fisher, 2018. Living with RNase. <A href="https://www.thermofisher.com/se/en/home/references/ambion-tech-support/nuclease-enzymes/general-articles/ten-sources-of-rnase-contamination.html">Thermo Fisher, 2018. Living with RNase</A> Date of Visit 2018-10-15</P><P><B>[5]</B>Thermo Fisher, 2018. Avoiding RNase Contamination <A href="https://assets.fishersci.com/TFS-Assets/LSG/manuals/MAN0011925_Avoiding_RNase_Contamination_UG.pdf">Thermo Fisher, 2018. Avoiding RNase Contamination</A> Date of Visit 2018-10-15</P><P><B>[6]</B> Melzak KA, Sherwood CS, Turner RFB, Haynes CA. 1996. Driving Forces for DNA adsorption to Silica in Perchlorate Solutions. Journal of Colloid and Interface Science. 181(2):635-644</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>