<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Uppsala_Phage_Display skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Uppsala/Phage Display</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="wrapper-for-mobile"><NAV id="nav-mobile" class="nav-mobile" role="navigation" aria-labelledby="nav-mobile-toggleLabel" aria-hidden="true"><UL id="menu-mobile" class="menu-nav-mobile" role="menu-mobile" aria-orientation="vertical"><LI class="mobile-list-item-home" role=""><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala" role="menuitem" tabindex="-1"> Home </A></LI><LI class="mobile-list-item" role=""><A class="mobile-list-link" href="#" role="menuitem" tabindex="-1"> Project </A></LI><UL class="mobile-nav-ul"><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Worm_Culturing" role="menuitem" tabindex="-1"> Worm Culturing  </A></LI><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Transcriptomics" role="menuitem" tabindex="-1"> Transcriptomics </A></LI><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Phage_Display" role="menuitem" tabindex="-1"> Phage Display </A></LI><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Reporter_System" role="menuitem" tabindex="-1"> Reporter System  </A></LI><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Model" role="menuitem" tabindex="-1"> Modeling  </A></LI><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Demonstrate" role="menuitem" tabindex="-1"> Demonstrate  </A></LI><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/InterLab" role="menuitem" tabindex="-1"> InterLab  </A></LI><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Improve" role="menuitem" tabindex="-1"> Improve  </A></LI></UL><LI class="mobile-list-item" role=""><A class="mobile-list-link" href="#" role="menuitem" tabindex="-1"> Parts </A></LI><UL class="mobile-nav-ul"><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Parts" role="menuitem" tabindex="-1"> Parts  </A></LI><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Composite_Part" role="menuitem" tabindex="-1"> Composite Part  </A></LI></UL><LI class="mobile-list-item" role=""><A class="mobile-list-link" href="#" role="menuitem" tabindex="-1"> Team </A></LI><UL class="mobile-nav-ul"><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Team" role="menuitem" tabindex="-1"> Meet the Team  </A></LI><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Collaborations" role="menuitem" tabindex="-1"> Collaboration  </A></LI><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Attributions" role="menuitem" tabindex="-1"> Attribution  </A></LI><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Achievements" role="menuitem" tabindex="-1"> Achievements  </A></LI></UL><LI class="mobile-list-item" role=""><A class="mobile-list-link" href="#" role="menuitem" tabindex="-1"> Portfolio </A></LI><UL class="mobile-nav-ul"><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Notebook" role="menuitem" tabindex="-1"> Notebook  </A></LI><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Protocol" role="menuitem" tabindex="-1"> Protocol </A></LI></UL><LI class="mobile-list-item" role=""><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/  " role="menuitem" tabindex="-1"> Human Practice </A></LI><UL class="mobile-nav-ul"><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Human Practices" role="menuitem" tabindex="-1"> Human Practices  </A></LI><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Public_Engagement" role="menuitem" tabindex="-1"> Public Engagement  </A></LI><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Safety" role="menuitem" tabindex="-1"> Safety </A></LI></UL><LI class="mobile-list-item" role=""><A class="mobile-list-link" href="#" role="menuitem" tabindex="-1"> Results </A></LI><UL class="mobile-nav-ul" style="visibility: hidden;"><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Result" role="menuitem" tabindex="-1"> Resultat from the Lab </A></LI><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Achivements" role="menuitem" tabindex="-1">  Achievement </A></LI></UL></UL></NAV></DIV><DIV class="body"><DIV class="content blur-box" style="font-size:16px;"><DIV id="toc" class="toc"><UL><LI class="toclevel tocsection"><A href="#Project_Description" class="scroll"><SPAN id="whereYouAre"> Phage Display  </SPAN></A></LI><LI class="toclevel nav-item active"><A href="#Intro" class="nav-link scroll">  Introduction  </A></LI><LI class="toclevel nav-item"><A href="#Gen_con" class="nav-link scroll"> General Concept </A></LI><LI class="toclevel nav-item"><A href="#Exp" class="nav-link scroll">  Experiment  </A></LI><LI class="toclevel nav-item"><A href="#Res" class="nav-link scroll">  Result </A></LI><LI class="toclevel nav-item"><A href="#ConcPhage" class="nav-link scroll"> Conclusion</A></LI><LI class="toclevel nav-item"><A href="#Ref" class="nav-link scroll"> References</A></LI></UL></DIV><DIV class="content-text" id="scrolldown"><H1 id="Intro">Introduction</H1><DIV class="card-holder"><P> Phage Display is a powerful method for finding interesting peptide interactions through affinity screening of a large random peptide library. 
                        
                        Bacterial viruses (phages) have been genetically modified to express a variety of peptides which then are allowed to interact with a target of interest. One of the advantages of using phages is that they can be amplified in bacteria. This allows us to repeat the experiment with narrower pools of peptides and thus finding more specific bindings.
                        Phage genomes are small and known which allows easy determination of the final specific peptide by DNA sequencing of the viruses. 
                        
                        Since the method of phage display was developed in 1985 it has been applied for various purposes such as studying antibodies, observing immune system responses, and evolving antibodies for therapy purposes [1]. It has also been used for whole cell recognition (often used for targeting cancer) and <I>in vivo</I> screening of individual tissues for endothelial cell markers [2].
                        More commonly, phage display is performed with a protein as the target, in order to find a ligand that would affect the function of the protein. 
                    </P></DIV><DIV class="card-holder"><DIV><DIV><H2 id="Gen_con">General Concept</H2><P>In order to determine a peptide with affinity to a molecule of interest an iterative process of affinity binding and washing, called panning, is needed to yield an end result with a high concentration of high affinity peptides.
                            </P><P><B>Figure 1.</B> Flowchart over the workflow of a typical phage display screening.</P></DIV></DIV></DIV><DIV class="card-holder"><P>For the purpose of our project we need a way to detect the parasites. A peptide binding specifically to the exterior of the strongyle would fulfill this purpose. Since we are working with an under-researched organism, starting with a random peptide library seemed reasonable. The only similar organisms that have been used in this method were <I>C. elegens</I>, but as it is a free living nematode whereas strongyles are endoparasites. Due to this fact we had reason to believe that the exterior of the membranes would differ greatly due to the different environments the different species inhabit.
                        Using a random phage library streamlined our work. Phages ensure a physical link between the DNA sequence and its encoded peptides. Furthermore the indigenous behavior of lysogenic phages allowed visualizing and following the procedure as well as enabling amplification of the peptides upon infecting bacteria. Thus, in spite the lack of a premade protocol we decided to try and apply phage display for characterizing our nematodes. 
                    </P></DIV><DIV class="card-holder"><H1 id="Exp">Experiment</H1><H2>New Application</H2><H3>Unknown Genome/Proteome </H3><P>Due to our target strongyle having an unknown genome/proteome, selection of a single target protein is not feasible. Efforts could have been made to separate surface proteins from the organism, but such an approach does not provide any guarantees of the extracted proteins displaying correct conformations. To preserve each of the possible protein targets in a physiological state, the entire organism was used for the experiment. </P><H3>Accessible High Throughput Screening </H3><P>Having decided to use a whole organism as the target, the next challenge is finding a way to deal with the plethora of exposed surface proteins. Attempting to identify possible ligands one at a time would have been prohibitively laborious, and a method had to be selected that could screen binding compounds in a massively parallel manner. Several such methods exist, but most would require specialized equipment and/or comprehensive precursor libraries. Phage display is a simple yet robust approach to the same problem. </P><H3>Immobilization Hard for Whole Worms</H3><P>To be able to wash away unbound phages, while retaining the bound phages, existing protocols employ various methods for immobilizing the target. Since the target of our experiment is the surface of an entire organism rather than a single protein, the common methods of fixing the target were ineffective, and a new method had to be developed. 
                        The implemented solution relies on filter inserts for eppendorf tubes. The filters simultaneously act as a substitute for well plates and an immobilization method. 
                    </P><H3>Experiment Procedure</H3><P>
                        With the experiment necessitating the use of phage display on an entire organism, no existing protocol was entirely applicable. This meant that we had to adapt a protocol as much as it was possible and introduce completely new solutions to problems unique to our application. We based our adapted protocol on the product manual included in the Ph.D.™-12 Phage Display Peptide Library Kit ordered from New Englands BioLabs. 
                    </P><P>
                        For a detailed description of our protocol click <A href="https://static.igem.org/mediawiki/2018/2/22/T--Uppsala--PhageDisplay_Panning_titering_amplification.pdf">here</A>. Our whole organism phage display experiments started with preparation of the containers that the whole procedure were performed in: microcentrifuge tubes with filter inserts. The preparation consisted of blocking the tubes with blocking buffer to prevent non-specific interactions.  
                        
                        To avoid selecting a phage with affinity for the plastic and not the organism, an affinity screening for the tubes was performed before introducing the target. After collecting the phage elute that doesn’t bind to the tubes, we performed our first affinity screening against the small strongyle. This was done by introducing the 12-mer peptides expressing phages to the strongyle placed in the filter tubes. Unbound phages were then washed away followed by elution and collection of the bound phages with an acidic buffer. All washing and elution steps were performed in the filter tubes, where liquid during centrifugation could pass through the filters, leaving the strongyle still on the top of the filter.
                    </P></DIV><DIV class="card-holder"><DIV class="content-card content-card-3"><DIV class="inner-card left-card"><DIV class="inner-card-text"><P><STRONG>Figure 2.</STRONG> Filter tube blocked with Blocking Buffer.</P></DIV></DIV><DIV class="inner-card middle-card"><DIV class="inner-card-text"><P><STRONG>Figure 3.</STRONG> The strongyles are added, excess liquid spun down and discarded.</P></DIV></DIV><DIV class="inner-card right-card"><DIV class="inner-card-text"><P><STRONG>Figure 4.</STRONG> Phages are added, left to incubate in solution with the worms. The unbound phages are spun down and discarded. The bound phages are subsequently eluted with a acidic buffer.</P></DIV></DIV></DIV></DIV><P>
                    In the next step of the experiment, phage titering was carried out, to visualise the amount of infectious phages present in the elute. The elute phage with affinity to the strongyle were then amplified in the <I>E.coli ER2738</I> host strain, by adding the phage to the strain in early log-phase. 
                    
                    Phage and bacteria were then separated with the help of centrifugation to discard <I>E. coli</I> and the phage were extracted by precipitation.
                    
                    The success of the amplification was examined by a new round of phage titering. Once aware of the phage titer, the next round of this previously described procedure consisting of affinity screening to the strongyle, washing and elution, phage titering, amplification of phage and a new round of phage titering was performed.
                </P><P>
                    After the third round of affinity screening, single plaques, (consisting of single phage clones) were picked from the titering plates and amplified separate. The phages DNA from the amplified plaques were then extracted and purified.
                </P><DIV><H2>Phage Titering</H2><P>
                        Phage titering is done after every affinity screening to assess the amount of phages that bind to the target. By following the titering protocol consisting of plating phages together with mid-log phase bacteria visible blue plaques are formed on Xgal/IPTG plates. Xgal is a substrate for the enzyme β-galactosidase, which hydrolyzes the compound into a form that spontaneously dimerizes into an insoluble blue pigment. Our cell strain can only express the enzyme after phage infection and induction by IPTG. This makes it simple to distinguish infected from non infected colonies since only the infected ones will appear blue. 
                        
                        The goal with plating is to achieve plates with around 100 plaques, which is fulfilled by doing several dilution series of the infected bacteria. The reason for this is that plaques will only increase linearly with added phages when the multiplicity of infection (MOI) is much less than 1. Also, low MOI result in one DNA sequence per plaque. To asses the titre, plaque forming unit (pfu) can be calculated by multiple numbers of plaques with the bacteria dilution. 
                    </P><P>
                        Phage titering is also carried out after every phage amplification in order to assess the success of the amplification and to make sure a sufficient amount of phages for the subsequente affinity screening are present.
                    </P><P>
                        We did three rounds of panning to select for phages having affinity peptides for the target. The last phage titering could then be used to select single clones to be prepared for sequencing. 
                    </P><H2>Verifying the Affinity and Analysis of DNA</H2><P>
                        So how do we prove that the selected phages clones really bind to our target? With the help of ELISA (enzyme-linked immunosorbent assay) single phage clones, affinity to the target could be tested by screening against plastic binders. This allowed us to identify all the viable samples. The ELISA was performed in the centrifugal filter-tubes to expose the phages to the same environment as the panning stage of the process. 
                    </P><P>
                        After amplification of each chosen plaque, the phage DNA was extracted an analyzed before sequencing. We used a nanodrop to determine the DNA concentration and gel electrophoresis to determine the purity and the size of the DNA.
                    </P></DIV><DIV class="card-holder"><H1 id="Res">Result</H1><P> Three series of panning were performed on small strongyles. The results of each round of panning and amplification were titered and plaque-forming phages were counted to ensure a high enough representation of the library in subsequent pannings.</P></DIV><DIV class="card-holder"><H2>Sample Titers from Series 1</H2><DIV class="content-card content-card-3"><DIV class="inner-card left-card"><DIV id="figuretext" class="inner-card-text"><P><STRONG>Figure 5.</STRONG> Titered Eluate: Panning 1.</P></DIV></DIV><DIV class="inner-card middle-card"><DIV id="figuretext" class="inner-card-text"><P><STRONG>Figure 6.</STRONG> Titered Eluate: Panning 2</P></DIV></DIV><DIV class="inner-card right-card"><DIV id="figuretext" class="inner-card-text"><P><STRONG>Figure 7.</STRONG> Titered Eluate: Panning 3. </P></DIV></DIV></DIV></DIV><DIV class="card-holder"><H2>Titering Table</H2><P><STRONG>Table 1:</STRONG> Titering results. *Negative panning for interactions with the tube were performed in conjugate with regular panning.</P><TABLE class="pgrouptable tablesorter our-table" style="width: 100%;" cellspacing="0" cellpadding="0"><THEAD><TR><TH style="width: auto">pfu/ml</TH><TH style="width: auto">Panning 1</TH><TH style="width: auto">Amplification 1</TH><TH style="width: auto">Panning 2</TH><TH style="width: auto">Amplification 2</TH><TH style="width: auto">Panning 3</TH><TH style="width: auto">Panning 3 replate</TH></TR></THEAD><TBODY><TR><TD>Series 1
                                </TD><TD>\(2.3 \cdot 10^{7*}\)
                                </TD><TD>\(&gt;10^{15}\)
                                </TD><TD>\(3.2 \cdot 10^{10}\)
                                </TD><TD>\(4 \cdot 10^{12}\)
                                </TD><TD>\(1 \cdot 10^{14}\)
                                </TD><TD>\(6 \cdot 10^{12}\)
                                </TD></TR><TR><TD>Series 2
                                </TD><TD>Faulty XGal
                                </TD><TD>\(1.8 \cdot 10^{10}\)
                                </TD><TD>\(1.6 \cdot 10^{9}\)
                                </TD><TD>No Amp.
                                </TD><TD>\(7 \cdot 10^{5*}\)
                                </TD><TD>-
                                </TD></TR><TR><TD>Series 3
                                </TD><TD>\(2.9 \cdot 10^{7*}\)
                                </TD><TD>\(6.2 \cdot 10^{12}\)
                                </TD><TD>-
                                </TD><TD>-
                                </TD><TD>-
                                </TD><TD>-
                                </TD></TR></TBODY></TABLE><H2>Phage ELISA</H2></DIV><DIV class="card-holder"><DIV class="content-card pic-next-to-text"><DIV class="side-text"><P>Unfortunately the ELISA results were inconclusive due to noticable chromogenic signaling in the filter tubes without any contact to either phages nor strongyles. Since the antibodies used were M13 monoclonal the possibility of the them interacting with the filter or plastic is highly unlikely. One factor that may have contributed to the false ELISA signal could be the prolonged exposure to the blocking buffer in the filter-tubes. Due to its alkaline nature and the stability of the filter being between pH  4-8, the prolonged exposure may have caused filter degradation. The degradation may have also prevented the antibodies to be successfully washed away thus exhibiting the false signal. </P></DIV><DIV class="side-img"><P><STRONG>Figure 8.</STRONG> Negative control of phage ELISA, containing <B>only</B> monoclonal M13 antibodies and <B>no</B> phage.</P></DIV></DIV></DIV><DIV class="card-holder"><H2>Sequenced Samples and Computational Analysis</H2><P><B>Table 2:</B> Samples sent to sequencing. *TUP = Target-Unrelated Peptide.</P><TABLE class="pgrouptable tablesorter our-table" style="width: 100%;" cellspacing="0" cellpadding="0"><THEAD><TR><TH style="width: auto">Sample</TH><TH style="width: 180px">Aligned Sequence</TH><TH style="width: auto">Propagation Advantage</TH><TH style="width: auto">Predicted Polysterene Binder</TH><TH style="width: auto">Known TUP*-Motif</TH></TR></THEAD><TBODY><TR><TD>Series 1
                                </TD><TD>EF01122224
                                </TD><TD>TP<SPAN class="blue-letter">I</SPAN>FL<SPAN class="red-letter">P</SPAN>TPA<SPAN class="blue-letter">Q</SPAN>EH---
                                </TD><TD> Yes
                                </TD><TD> No
                                </TD><TD> No
                                </TD></TR><TR><TD>Series 3
                                </TD><TD>EF01122218
                                </TD><TD>---FS<SPAN class="red-letter">P</SPAN>TQA<SPAN class="blue-letter">N</SPAN>T<SPAN class="blue-letter">IH</SPAN>RW
                                </TD><TD>No
                                </TD><TD>No
                                </TD><TD>No
                                </TD></TR><TR><TD>EF01122220
                                </TD><TD>--<SPAN class="blue-letter">V</SPAN>GGTVQSESH<SPAN class="blue-letter">R</SPAN>R-
                                </TD><TD>No
                                </TD><TD>No
                                </TD><TD>No
                                </TD></TR><TR><TD>EF01122222
                                </TD><TD>-S<SPAN class="blue-letter">M</SPAN>GRTDYV<SPAN class="blue-letter">Q</SPAN>Q<SPAN class="blue-letter">LR</SPAN>--
                                </TD><TD>No
                                </TD><TD>No
                                </TD><TD>No
                                </TD></TR><TR><TD>EF01122217
                                </TD><TD>--RVQ<SPAN class="red-letter">P</SPAN>AHF<SPAN class="blue-letter">N</SPAN>V<SPAN class="blue-letter">M</SPAN>GQ
                                </TD><TD>No
                                </TD><TD>Yes
                                </TD><TD>No
                                </TD></TR></TBODY></TABLE><P>
                        Five samples contained pure enough samples of sufficient concentration to satisfy the standard for third party sequencing. Series 1 produced a single sample of high enough quality, Series 2 were all deemed too low for accurate sequencing and Series 3 yieded four samples. The aligment used ClustalW with penalties 25 for gap-creation and 25 for gap-elongation to ensure strict alignments. No clear consensus motifs were distinguishable.

                        Predictive analysis was performed with <A href="http://immunet.cn/sarotup/cgi-bin/TUPScan.pl">SAROTUP: Target-Unrelated Peptides Scanners</A>[3, 4, 5].
                    </P></DIV><DIV class="card-holder"><H2>Peptides</H2><P>Click on the images below:</P></DIV><DIV class="container"><SPAN onclick="this.parentElement.style.display='none'" class="closebtn">×</SPAN></DIV><DIV class="card-holder"><H1 id="ConcPhage">Conclusion</H1><P>
                        Due to the fact that our ELISA-test was inconclusive we can not be sure that our peptides are specific binders. However, only one was predicted as a Polysterene binder and none contain any known TUP-sequences. A blastp on Biopanning Databank showed that EF01122224 showed some similarities with an antibody for a pork tapeworm epithelial protein and EF01122218 for an antibody for rabbit epithelia. The E-values were high, 211  and 10 respectively, but those two may be promising candidates for future studies. 
                    </P></DIV><DIV class="card-holder"><DIV class="buttons"><DIV class="container-button"><A href="https://2018.igem.org/Team:Uppsala/Notebook" class="btn"> Notebook </A><A href="https://2018.igem.org/Team:Uppsala/Protocol" class="btn"> Protocol </A></DIV></DIV></DIV><DIV class="card-holder"><H2 id="Ref">References</H2><P><B>[1]</B> Smith GP. Filamentous fusion phage: novel expression vectors that display cloned antigens on the virion surface. Science. 1985;228:1315–1317.
                        <B>[2]</B> Arap, Marco Antonio. (2005). Phage display technology: applications and innovations.<A href="https://dx.doi.org/10.1590/S1415-47572005000100001"><I>Genetics and Molecular Biology, 28(1)</I>, 1-9.</A><B>[3]</B>  Qiang, Xu, Keyong Sun, Lijun Xing, Yifeng Xu, Hong Wang, Zhengpin Zhou, Juan Zhang, m.fl. ”Discovery of a Polystyrene Binding Peptide Isolated from Phage Display Library and Its Application in Peptide Immobilization”. Scientific Reports 7, nr 1 (december 2017). <A href="https://doi.org/10.1038/s41598-017-02891-x">https://doi.org/10.1038/s41598-017-02891-x</A>.
                        <B>[4] </B> Huang J, Ru B, Li S, Lin H, Guo FB. SAROTUP: scanner and reporter of target-unrelated peptides. Journal of Biomedicine and Biotechnology 2010; 2010: 101932. [PMID: <A href="https://www.ncbi.nlm.nih.gov/pubmed/20339521">20339521</A>
                        ; Download: <A href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2842971/pdf/JBB2010-101932.pdf"> fulltext pdf</A> ]
                        <B>[5] </B> Huang J, Ru B, Zhu P, Nie F, Yang J, Wang X, Dai P, Lin H, Guo FB, Rao N. MimoDB 2.0: a mimotope database and beyond. Nucleic Acids Research 2012; 40(Database issue): D271-D277. [PMID: <A href="https://www.ncbi.nlm.nih.gov/pubmed/22053087">22053087</A>; Download: <A href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2842971/pdf/JBB2010-101932.pdf">fulltext pdf</A> ]
                    </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>