<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Uppsala_Transcriptomics_Sequencing skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Uppsala/Transcriptomics/Sequencing</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="wrapper-for-mobile"><NAV id="nav-mobile" class="nav-mobile" role="navigation" aria-labelledby="nav-mobile-toggleLabel" aria-hidden="true"><UL id="menu-mobile" class="menu-nav-mobile" role="menu-mobile" aria-orientation="vertical"><LI class="mobile-list-item-home" role=""><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala" role="menuitem" tabindex="-1"> Home </A></LI><LI class="mobile-list-item" role=""><A class="mobile-list-link" href="#" role="menuitem" tabindex="-1"> Project </A></LI><UL class="mobile-nav-ul"><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Worm_Culturing" role="menuitem" tabindex="-1"> Worm Culturing  </A></LI><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Transcriptomics" role="menuitem" tabindex="-1"> Transcriptomics </A></LI><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Phage_Display" role="menuitem" tabindex="-1"> Phage Display </A></LI><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Reporter_System" role="menuitem" tabindex="-1"> Reporter System  </A></LI><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Model" role="menuitem" tabindex="-1"> Modeling  </A></LI><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Demonstrate" role="menuitem" tabindex="-1"> Demonstrate  </A></LI><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/InterLab" role="menuitem" tabindex="-1"> InterLab  </A></LI><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Improve" role="menuitem" tabindex="-1"> Improve  </A></LI></UL><LI class="mobile-list-item" role=""><A class="mobile-list-link" href="#" role="menuitem" tabindex="-1"> Parts </A></LI><UL class="mobile-nav-ul"><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Parts" role="menuitem" tabindex="-1"> Parts  </A></LI><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Composite_Part" role="menuitem" tabindex="-1"> Composite Part  </A></LI></UL><LI class="mobile-list-item" role=""><A class="mobile-list-link" href="#" role="menuitem" tabindex="-1"> Team </A></LI><UL class="mobile-nav-ul"><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Team" role="menuitem" tabindex="-1"> Meet the Team  </A></LI><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Collaborations" role="menuitem" tabindex="-1"> Collaboration  </A></LI><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Attributions" role="menuitem" tabindex="-1"> Attribution  </A></LI><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Achievements" role="menuitem" tabindex="-1"> Achievements  </A></LI></UL><LI class="mobile-list-item" role=""><A class="mobile-list-link" href="#" role="menuitem" tabindex="-1"> Portfolio </A></LI><UL class="mobile-nav-ul"><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Notebook" role="menuitem" tabindex="-1"> Notebook  </A></LI><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Protocol" role="menuitem" tabindex="-1"> Protocol </A></LI></UL><LI class="mobile-list-item" role=""><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/  " role="menuitem" tabindex="-1"> Human Practice </A></LI><UL class="mobile-nav-ul"><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Human Practices" role="menuitem" tabindex="-1"> Human Practices  </A></LI><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Public_Engagement" role="menuitem" tabindex="-1"> Public Engagement  </A></LI><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Safety" role="menuitem" tabindex="-1"> Safety </A></LI></UL><LI class="mobile-list-item" role=""><A class="mobile-list-link" href="#" role="menuitem" tabindex="-1"> Results </A></LI><UL class="mobile-nav-ul" style="visibility: hidden;"><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Result" role="menuitem" tabindex="-1"> Resultat from the Lab </A></LI><LI><A class="mobile-list-link" href="https://2018.igem.org/Team:Uppsala/Achivements" role="menuitem" tabindex="-1">  Achievement </A></LI></UL></UL></NAV></DIV><DIV class="body"><DIV class="content blur-box" style="font-size:16px;"><DIV id="toc" class="toc"><UL><LI class="toclevel tocsection"><A href="#Project_Description" class="scroll"><SPAN id="whereYouAre"> Transciptomics</SPAN></A></LI><LI class="toclevel nav-item active"><A href="#Seq" class="nav-link scroll"> Sequencing</A></LI><LI class="toclevel nav-item"><A href="#Exp" class="nav-link scroll">  Expermient</A></LI><LI class="toclevel nav-item"><A href="#Res" class="nav-link scroll">  Result</A></LI><LI class="toclevel nav-item"><A href="#References" class="nav-link scroll"> References </A></LI></UL></DIV><DIV class="content-text" id="scrolldown"><DIV class="card-holder"><H1 id="Seq">Sequencing</H1></DIV><DIV class="card-holder"><DIV class="content-card pic-next-to-text"><DIV class="side-text"><P>Sequencing is the final step of the journey and generally makes up a lot of different and varying methods with the common purpose of determining the primary atomic-level structure of what you’re interested in, resulting in sequences of bases making it all up. Most often this includes proteins, polymers or as in our case the keys to life: genetic material; enabling us to read it like a book of sorts.
 
                                Oxford Nanopore’s MinION third generation sequencing device make use of nano-sized holes called nanopores with an applied voltage across them. In essence: when your material of choice passes through one of these pores, each base fluctuates this current by a given amount which acts as a fingerprint to identify the given base [1].</P></DIV><DIV class="side-img" style="background-color:darkolivegreen;"><P><B>Figure 1.</B> The MinION device.</P></DIV></DIV></DIV><DIV class="card-holder"><H2 id="Exp">Experiment</H2><P> Sequencing the final library-prepped RNA includes three primary steps: Priming, loading and sequencing.</P><H3>Priming the Flowcell</H3><P>The flow cell is essentially where the magic happens. Before the sequencing can take place, a type of storing liquid which protects the flow cell has to be emptied and replaced with a mixture that primes the flow cell, essentially making it ready to sequence. A scan is also performed during this step which tells you how many pores are actually working.</P><H3>Loading the Flowcell</H3><P>Loading the flow cell is a bit tricky, as the wrong action can completely destroy the whole cell. By sucking up the primer mixture carefully with a pipette through a disposal vent, a kind of suction is created over the actual pore-membrane area which lets you (with some quick manouvering skills) drop by drop load your sample into that chamber with the help of a pipette which sucks it all down.
 
Being to slow in this process can result in bubbles forming across the membrane which effectively kills all of the pores.
</P><H3>Sequencing</H3><P>At this step the sequencing can be started, as the software essentially takes care of everything!</P><H2 id="Res">Result</H2><P>Down below are some example pictures from one of our sequencing runs, giving a clue as of what kind of information that can be gathered.</P></DIV><DIV class="card-holder"><DIV class="content-card content-card-2"><DIV class="inner-card left-card"><DIV class="inner-card-text"><P><B>Figure 2.</B> State of the sequencing pores after a full sequencing. Notice the amount of sequencing pores (light green) dropping off as issues like saturation (black, orange) start to increase.</P></DIV></DIV><DIV class="inner-card right-card"><DIV class="inner-card-text"><P><B>Figure 3.</B> Passed versus failed reads.</P></DIV></DIV></DIV></DIV><DIV class="card-holder"><DIV class="content-card content-card-2"><DIV class="inner-card left-card"><DIV class="inner-card-text"><P><B>Figure 4.</B> Distribution of read lengths.</P></DIV></DIV><DIV class="inner-card right-card"><DIV class="inner-card-text"><P><B>Figure 5.</B> Quality score distribution for the different read lengths, were a quality score of at least 7 is acceptable.</P></DIV></DIV></DIV></DIV><DIV class="card-holder"><P>Down below are the total of all of our sequencing runs summarised with each run showing runtime, reads, read bases as well as an attached quality score distribution.</P><H3>Run 1</H3></DIV><DIV class="card-holder"><DIV class="content-card pic-next-to-text"><DIV class="side-text"><TABLE class="pgrouptable tablesorter our-table" style="width: 100%;" cellspacing="15" cellpadding="0"><THEAD><TR><TH style="“width:" auto="">Runtime (D:HH:MM)</TH><TH style="“width:" auto="">Reads (k) [ng]</TH><TH style="“width:" auto="">Bases (MB)</TH></TR></THEAD><TBODY><TR><TD>
0:01:37
</TD><TD>
20.21
</TD><TD>9.26
</TD></TR></TBODY></TABLE></DIV></DIV></DIV><DIV class="card-holder"><H3>Run 2</H3></DIV><DIV class="card-holder"><DIV class="content-card pic-next-to-text"><DIV class="side-text"><TABLE class="pgrouptable tablesorter our-table" style="width: 100%;" cellspacing="15" cellpadding="0"><THEAD><TR><TH style="“width:" auto="">Runtime (D:HH:MM)</TH><TH style="“width:" auto="">Reads (k) [ng]</TH><TH style="“width:" auto="">Bases (MB)</TH></TR></THEAD><TBODY><TR><TD>
0:00:22
</TD><TD>
10.36
</TD><TD>3.92
</TD></TR></TBODY></TABLE></DIV></DIV></DIV><DIV class="card-holder"><H3>Run 3</H3></DIV><DIV class="card-holder"><DIV class="content-card pic-next-to-text"><DIV class="side-text"><TABLE class="pgrouptable tablesorter our-table" style="width: 100%;" cellspacing="15" cellpadding="0"><THEAD><TR><TH style="“width:" auto="">Runtime (D:HH:MM)</TH><TH style="“width:" auto="">Reads (k) [ng]</TH><TH style="“width:" auto="">Bases (MB)</TH></TR></THEAD><TBODY><TR><TD>
0:01:05
</TD><TD>
52.9
</TD><TD>8.2
</TD></TR></TBODY></TABLE></DIV></DIV><H3>Run 4</H3></DIV><DIV class="card-holder"><DIV class="content-card pic-next-to-text"><DIV class="side-text"><TABLE class="pgrouptable tablesorter our-table" style="width: 100%;" cellspacing="15" cellpadding="0"><THEAD><TR><TH style="“width:" auto="">Runtime (D:HH:MM)</TH><TH style="“width:" auto="">Reads (k) [ng]</TH><TH style="“width:" auto="">Bases (MB)</TH></TR></THEAD><TBODY><TR><TD>
0:00:33
</TD><TD>
9.57
</TD><TD>2.4
</TD></TR></TBODY></TABLE></DIV></DIV><H3>Run 5</H3></DIV><DIV class="card-holder"><DIV class="content-card pic-next-to-text"><DIV class="side-text"><TABLE class="pgrouptable tablesorter our-table" style="width: 100%;" cellspacing="15" cellpadding="0"><THEAD><TR><TH style="“width:" auto="">Runtime (D:HH:MM)</TH><TH style="“width:" auto="">Reads (k) [ng]</TH><TH style="“width:" auto="">Bases (MB)</TH></TR></THEAD><TBODY><TR><TD>
0:01:35
</TD><TD>
33.98
</TD><TD>6.29
</TD></TR></TBODY></TABLE></DIV></DIV><H2>Conclusion</H2><P>An ideal sequencing run would mean that a lot of the material (depending on run time) has been read and converted into large amounts of ”passed reads” data files containing lines and lines of base sequences. However during our multiple sequencing runs a couple of re-emergin issues consistently showed up, therefore a lot of time was dedicated to troubleshooting this. It was later hypothesized that the source of poor sequencing was because of mRNA left in the samples due to inadequate reverse transcription. See cDNA synthesis </P><H3>Low Throughput</H3><P>Because it is impossible to have a 100% purely adaptor ligated mRNA sample we initially decided on that it would be acceptable to run the sequencing for a longer time with a below average throughput. However it was noted during repeated sequencing runs that the throughput was actually sub-par, showing that something in the sample(s) was consistently clogging the pores. This meant that only a fraction of our total material was actually being sequenced, eventually leading to near-zero throughputs.</P><H3>Failed Reads</H3><P>Another reoccuring theme was the issues with ”failed” reads. A failed read consitutes a failed characterization of the base passing through the pores, effectively discarding it. What this means is that whatever came through the pore was actually not cDNA but something foreign.</P><P>Even though some progress was made creating read-data that passed, the amount of data as well as the issue with consistency in form of duplicate or triplicate sequencing runs made it deemed as not significant enough to carry over to the bioinformatics.</P></DIV><DIV class="card-holder"><H2 id="References">References</H2><P><B>[1]</B> Oxford Nanopore, 2018. How does Nanopore DNA/RNA sequencing work? <A href="https://nanoporetech.com/how-it-works">https://nanoporetech.com/how-it-works</A> Date of visit 2018-10-15</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>