Team:Uppsala/Improve
Home
Project
Worm Culturing
Transcriptomics
Phage Display
Reporter System
Modeling
Demonstrate
InterLab
Improve
Parts
Parts
Composite Part
Team
Meet the Team
Collaboration
Attribution
Achievements
Portfolio
Notebook
Protocol
Human Practice
Human Practices
Public Engagement
Safety
Results
Resultat from the Lab
Achievement
Improve
Introduction
Experiments
Conclusion
Refrences
Introduction
In 2011 the iGEM Uppsala team managed to create an amilCP biobrick ( BBa_K592009 ), a great addition to the iGEM repository of chromoproteins. When the part, which originates from a coral named Acropora millepora, gets expressed it produces a strong natural blue color that is very much visible to the naked eye [1].
In 2017, the University of Texas at Austin (UT Austin) iGEM Team decided to improve the part hoping for stronger expression resulting from codon optimization. By performing codon optimization they hypothesized that the seemingly unstable amilCP from 2011 would become more stable resulting in less white cultures on the plate, hence higher protein yield. The output part after optimization can be found in the registry as BBa_K2253002 [2].
Figure 2 shows one single successful transformed colony of the UT Austin codon-optimized amilCP. In addition we can never know for sure if Austin Texas 2017 iGEM Team's optimization actually is better than the original part since a direct comparison in a picture is missing on both the wiki and the part registry.
Figure 1. The image above shows various streak colonies from the original blue chromoprotein, BBa_K592009 . Both phenotypically blue and clear colonies grew, supporting the hypothesis that the chromoprotein gene is unstable during cell growth, i.e. there is strong selection for cells carrying the resistance plasmid with a mutation that prevents expression of the chromoprotein due to its fitness cost. Plates were streaked by Microbe Hackers 2016 Chromoprotein Team [2].
Original AmilCP sequence ( BBa_K592009 ) was optimized by us for expression in E.coli K12 strain using the IDT tool for codon optimization. The sequence was then put in front of a strong constitutive promoter and RBS along with a double terminator. It can be found in the registry as BBa_K2669003 . The sequence, and the original native AmilCP sequence with the same elements was then ordered as customized genes from IDT inserted in pUCIDT, a cloning vector containing ampicillin resistance.
Unfortunately IDT wasnâ€™t able to synthesize the codon optimized Austin Texas sequence even though multiple attempts were made. Each attempt resulted in the switch of a base, causing an early stop codon to form. Hence we never were able to compare our codon optimized sequence with the sequence from the Austin Texas team. We continued with our experiment and decided to make a stability assay where we compared the expression between our two Biobrick parts containing optimized AmilCP sequences and the original AmilCP sequence from the iGEM registry.
Figure 2. The image above taken by UT Austin shows the transformation plate of the electroporation of the codon-optimized blue chromoprotein. The blue colony shown was used to make overnight cultures to miniprep and sequence [1].
Experiments
Design and Transformation
The original native AmilCP sequence, Part:BBa_K592009 was codon optimized for E.coli K12 with a codon optimization tool provided by Integrated DNA Technologies (IDT). In order to conduct a stability assay through growth in liquid culture, the codon optimized basic part and the original native AmilCP part was attached to a constitutive promotor, RBS and a double terminator separately. The two Biobrick parts were ordered as customized gene fragments from IDT inserted in pUCIDT (Amp).
Both vectors were transformed into DH5-aplha E.coli cells through single tube transformation and yielded phenotypically blue colonies for cells containing plasmid with original part and plasmid containing codon optimized part.
Figure 3. Transformation plate of colonies with the incorporated plasmid containing the codon optimized AmilCP sequence in the pUCIDT backbone.
Figure 4. Transformation plate of colonies with the incorporated plasmid containing the codon optimized AmilCP sequence in the pUCIDT backbone. Colonies retrieved for the assay are circled.
Figure 5. Transformation plate of colonies with the incorporated plasmid containing the Original native AmilCP sequence in the pUCIDT backbone.
Figure 6. Transformation plate of colonies with the incorporated plasmid containing the Original native AmilCP sequence in the pUCIDT backbone. Colonies retrieved for the assay are circled.
Stability Assay
Stability assay through liquid experiment was performed with both inserts in the pUCIDT (Amp) vector. 1 mL of LB with ampicillin (25 ug/ml) was inoculated with one single colony in eppendorf tubes in 10 replicates for each part. In order to allow 10 generations of growth, a 1:1000 dilution was made of the overnight culture and incubated for 24 h. The colour comparison was done through visualization of centrifuged cultures and by analyzing the color intensity we could deduce the stability of the chromoprotein-encoding plasmid.
Figure 7. Results after 10 Generations of growth. Culture pellets marked as IDT have the plasmid with the codon optmized AmilCP sequence and culture pellets market as ORI have the plasmid containing the original native AmilCP sequence.
Submission Preparation
The plasmid containing our biobrick part with the codon optimized AmilCP sequence was extracted from an O.N. culture with JeneGET plasmid miniprep kit from Thermo Fischer Scientific.
A 2A assembly was then performed to move our Biobrick part from the pUCIDT (Amp) cloning vector into pSB1C3 backbone.  This experiment was done according to the iGEM protocol for 3A assembly. The size of digested DNA was checked by gel elecrophoresis. After ligation into pSB1C3, the concentration of plasmid DNA was measured with Qubit before the pSB1C3 plasmid containing our Biobrick part was sent for submission.
Notebook Protocol
Conclusion
Color Intensity Comparison
Already after 10 generations of growth it was clearly visible that the color intensity was better distributed in the cells containing the plasmid with the codon optimized AmilCP sequence. The result confirms that the plasmid containing codon optimized part generates better maintenance of protein expression than the plasmid containing the original native part. The improved stability of expression makes this part, BBa_K2669003 , a good candidate as a reliable reporter or biosensor.
Table 1: Table of the interpreted result of color intensity (see Figure 7). C. P. = Culture Pellets, Optimized = Biobrick part with codon optimized AmilCP sequence in pUCIDT. Original = Biobrick part with original native AmilCP sequence in pUCIDT.
Plasmid
C.P. 1
C.P. 2
C.P. 3
C.P. 4
C.P. 5
C.P. 6
C.P. 7
C.P. 8
C.P. 9
C.P. 10
Optimized
++
++
++
++
++
++
++
++
++
++
Original
++
+
++
++
+
+
++
++
++
+
By improving the stability of the blue chromoprotein-encoding part, iGEM Uppsala 2018 has contributed to iGEM's collection of standard biological parts. We hope that  BBa_K2669003 will be of use for future iGEM teams.
References
[1] Part:BBa K592009 - parts.igem.org. WWW-dokument: http://parts.igem.org/Part:BBa_K592009 . Retrieved 2018-08-02.
[2] Team:Austin UTexas/Improve - 2017.igem.org. WWW-dokument: https://2017.igem.org/Team:Austin_UTexas/Improve . Retrieved 2018-08-02.
