Team:Uppsala/Phage Display/Results
Home
Project
Worm Culturing
Transcriptomics
Phage Display
Reporter System
Modeling
Demonstrate
InterLab
Improve
Parts
Parts
Composite Part
Team
Meet the Team
Collaboration
Attribution
Achievements
Portfolio
Notebook
Protocol
Human Practice
Human Practices
Public Engagement
Safety
Results
Resultat from the Lab
Achievement
Project Description
Overview
Problem
Solution
References
Results
Three series of panning were performed on Small Strongyles. Each round of panning and amplification were titered and plaque-forming phages were counted to ensure a high enough representation of the library in subsequent pannings.
Sample titers from Series 1
Figure 1: Titered Eluate: Panning 1
Figure 2: Titered Eluate: Panning 2
Figure 3: Titered Eluate: Panning 3
Titering tables
Table 1: Titering results. *Negative panning for interactions with the tube were performed in conjugate with regular panning.
Phage Elisa
The ELISA results even though a chromogenic signal was established in the presence of phages and strongyles the results were inconclusive due to noticing chromogenic signaling in the filter tubes without any contact to either phages nor strongyles. Since the antibodies used were M13 monoclonal the possibility of the them interacting with the filter or plastic is highly unlikely. One factor that may have contributed to the false ELISA signal could be the prolonged exposure to the blocking buffer in the filter-tubes. Due to its alkaline nature and the stability of the filter being between pH  4-8, the prolonged exposure may have caused filter degradation. The degradation may have prevented the antibodies to successfully be washed away thus exhibiting the false signal.
Figure 4:Negative control of phage ELISA, containing ONLY monoclonal M13 antibodies and NO phage.
Sequenced Samples and Computational Analysis
Table 2: Samples sent to sequencing. *TUP = Target-Unrelated Peptide.
Sample
Aligned Sequence
Propagation Advantage
Predicted Polysterene Binder
Known TUP*-Motif
Series 1:
EF01122224: TPIFLPTPAQEH
TPIFLPTPAQEH---
Yes
No
No
Series 3:
EF01122218: FSPTQANTIHRW
---FSPTQANTIHRW
No
No
No
EF01122220: VGGTVQSESHRR
--VGGTVQSESHRR-
No
No
No
EF01122222: SMGRTDYVQQLR
-SMGRTDYVQQLR--
No
No
No
EF01122217: RVQPAHFNVMGQ
--RVQPAHFNVMGQ
No
Yes
No
Five samples contain pure enough samples of sufficient concentration to satisfy the standard for third party sequencing. Series 1 produced a single samples of high enough quality, Series 2 were all deemed too low for accurate sequencing and Series 3 yieded four samples. The aligment used ClustalW with penalties 25 for gap-creation and 25 for gap-elongation to ensure strict alignments. No clear consensus motifs are distinguishable except slighty hydrophilic residues towards the end of the sequence.  Predictive analysis was performed with SAROTUP: Target-Unrelated Peptides Scanners [1].
Conclusions
Due to the fact that our ELISA-test were inconclusive we can not be sure that our peptides are specific binders. Three of the peptides; EF01122218, EF01122220 and EF01122222 shows no predicted or comparative causes for unspecific binding and would be good candidates for future studies.
References
[1].  Qiang, Xu, Keyong Sun, Lijun Xing, Yifeng Xu, Hong Wang, Zhengpin Zhou, Juan Zhang, m.fl. ”Discovery of a Polystyrene Binding Peptide Isolated from Phage Display Library and Its Application in Peptide Immobilization”. Scientific Reports 7, nr 1 (december 2017). https://doi.org/10.1038/s41598-017-02891-x . .
