Team:Uppsala/Phage Display/Design
Home
Project
Worm Culturing
Transcriptomics
Phage Display
Reporter System
Modeling
Demonstrate
InterLab
Improve
Parts
Parts
Composite Part
Team
Meet the Team
Collaboration
Attribution
Achievements
Portfolio
Notebook
Protocol
Human Practice
Human Practices
Public Engagement
Safety
Results
Resultat from the Lab
Achievement
Design
What is phage display?
TPhage Display is a powerful method for finding interesting peptide interactions through affinity screening of a large random peptide library.
Bacterial viruses (phages) have been genetically modified to express variety of peptides which then are allowed to interact with a target of interest. One of the advantages of using phages is that  they can be amplified in bacteria. This allows repeating the experiment with more narrow pool of peptides thus finding more specific bindings. The genome of phages is small and known which allows easy determination of the final specific peptide by DNA sequencing of the viruses.
Since the method of phage display was developed at 1985 [1] it has been applied for various purposes e.g. studying antibodies - response of immune system and evolving antibodies for therapy purposes; whole cell recognition (often used for targeting cancer);  in-vivo screening of individual tissues for endothelial cell markers [2] Most commonly phage display is performed with a protein as a target to find a ligand that would affect the function of the protein.
General Concept
In order to determine a peptide with affinity to a molecule of interest an iterative process of affinity binding and washing, called panning, will yield an end result with a high concentration of high affinity peptides.
Figure 1: Flowchart over the workflow of a typical phage display screening
For the purpose of our project - developing a diagnostic system, a distinct detection of the parasites is a precondition. A peptide binding specifically to the exterior of the strongyle would fulfill this purpose. As we were setting out to detect a under-researched organism, starting with a random peptide library seemed reasonable. Any remote parallels we could have drawn regarding possible surface proteins were with the model organism c.elegans but as it is a free living nematode whereas strongyles are endoparasites we had a reason to believe that the exterior of the membranes would differ greatly due to the different environments the different species are inhabiting.
Approach with viruses serving as vessels for the random peptides streamlined our work. Phages ensured a link between the DNA sequence and the physical peptides. Furthermore the indigenous behavior of lysogenic phages allowed visualizing and following the procedure as well as enabling amplification of the peptides upon infecting bacteria.  Thus, in spite of lack of a premade protocol we decided to try and apply phage display for characterizing our nematodes.
References
[1]Smith GP. Filamentous fusion phage: novel expression vectors that display cloned antigens on the virion surface. Science. 1985;228:1315â€“1317.
[2] Arap, Marco Antonio. (2005). Phage display technology: applications and innovations. Genetics and Molecular Biology, 28(1), 1-9.
