Team:Uppsala/Transcriptomics test
Home
Project
Worm Culturing
Transcriptomics
Phage Display
Reporter System
Modeling
Demonstrate
InterLab
Improve
Parts
Parts
Composite Part
Team
Meet the Team
Collaboration
Attribution
Achievements
Portfolio
Notebook
Protocol
Human Practice
Human Practices
Public Engagement
Safety
Results
Resultat from the Lab
Achievement
The idea behind the work of the transcriptomics group is to try and detect any changes in the gene expression when E. coli is grown alongside strongyles as opposed to just being grown in a normal lab environment. The aim for this undertaking is to find any and all genes which might be expressed exclusively in the proximity to the worms - giving us a prime opportunity to work some biotech magic and find a way to make a diagnostics tool for the strongyle parasites out of that very bacteria gene.
The way we decided to approach this was by running a full-scale transcriptome sequencing on the entire E. coli genome, both in bacteria grown normally and grown alongside the worms. By doing this, we would be able to quantitatively determine the differences in gene expression.
For this, we created an eight-step pipeline starting from lysing the bacteria to extracting their RNA contents, to refining and finally sequencing the genetic material using Oxford Nanopore’s MinION device - which you plug in to your laptop. That same laptop will then be used to understand what exactly is going on in the bacteria.
1. Cell Lysis and Total RNA Purification
The cultured E. coli cells are lysed and the RNA is extracted using QIAGEN’s RNeasy Kit. Two cultures are purified - one worm group and one control group. The cells are harvested at their log phase when genetic expression is at the highest to ensure the maximum genetic variety. The RNA is quality checked and then stored.
2. rRNA Depletion
The ribosomal RNA - rRNA, does not hold any relevant genetic information for us and consists about 90% of the total RNA content of the cell. Using Thermo Fishers MICROBExpress Kit, the rRNA is removed to get rid of genetic static and to make future steps easier in terms of total material that needs to be processed.
3. Poly(A)-Tailing
Poly(A)-tails consists of adenosine nucleotides that are added onto the 3’ end of the nucleotide strand in mostly eukaryotes for translation purposes. They are added onto the ends of our RNA to work as a primer target in the next step of the process.
4. cDNA Conversion
Using our newly added Poly(A)-tails as a primer target, we subject our RNA to reverse transcription to create complementary DNA copies - or cDNA. The RNA needs to be converted into DNA to be sequenced with the MinION device. The original RNA template is then degraded and a second strand is synthesized onto the cDNA.
5. Barcoding
A necessary step to be able to make sense of the sequenced data is to attach so-called barcodes to the end of each cDNA strand. Barcodes are small sequences which are known beforehand and allows us to differentiate between different samples. Because two transcriptome samples will be sequenced at the same time, this will let us know which sequence came from which sample.
6. Library Preparation
One final step before sequencing can begin is to attach adaptor sequences onto the end of the barcodes. These specialized adaptors allows the sequencing device to recognize the cDNA-strand and move it into the sequencing machinery.
7. Sequencing
The MinION device works by applying an electrical current and pulling the DNA strand through small biological pores. This movement causes changes in the electrical current also passing through the pore - and the charge of the current varies depending on which nucleotide of the DNA strand is passing through at the moment. This is what enables the sequencing - the current changes are translated into the nucleotide order of the DNA strand.
8. Bioinformatics
With the genetic information of the samples finally deciphered, it is time to match it to known genes in the E.coli genome and quantify the amount of genes that was expressed way back before step one of this pipeline. This allows us to see the possible difference in gene expression between the E.coli worm samples and the E.coli control samples.
