<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Freiburg_Surface_Display skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Freiburg/Surface Display</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="menuSmall"><H5><A href=" https://2016.igem.org/Team:Freiburg/sitemap">  Menu  </A></H5></DIV><DIV class="menu-wrap"><NAV class="menu"><UL class="clearfix"><LI class="menu_item"><A href=" https://2016.igem.org/Team:Freiburg/Team"> Team </A></LI><LI><A href="https://2016.igem.org/Team:Freiburg/Attributions">  Attributions </A></LI><LI><A href="https://2016.igem.org/Team:Freiburg/Achievements">  Achievements </A></LI><LI><A href="https://2016.igem.org/Team:Freiburg/Pictures">  Pictures </A></LI><LI><A href="https://2016.igem.org/Team:Freiburg/GiantJamboree"> Giant Jamboree </A></LI></UL><LI class="menu_item"><A href="https://2016.igem.org/Team:Freiburg/Overview"> Project </A></LI><LI><A href="https://2016.igem.org/Team:Freiburg/Motivation"> Motivation </A></LI><LI><A href="https://2016.igem.org/Team:Freiburg/Goals_Approach"> Approach </A></LI><LI><A href="https://2016.igem.org/Team:Freiburg/B_subtilis"> B. subtilis </A></LI><LI><A href="https://2016.igem.org/Team:Freiburg/Surface_Display"> Microbial surface display
 </A></LI><LI><A href="https://2016.igem.org/Team:Freiburg/Targeted Drug Delivery"> Targeted Drug Delivery</A></LI><LI class="menu_item"><A href="https://2016.igem.org/Team:Freiburg/Lab"> Lab </A></LI><LI><A href="https://2016.igem.org/Team:Freiburg/Experiments"> Methods </A></LI><LI><A href="https://2016.igem.org/Team:Freiburg/Notebook"> Journal </A></LI><LI class="menu_item"><A href="https://2016.igem.org/Team:Freiburg/Results"> Results</A></LI><LI><A href="https://2016.igem.org/Team:Freiburg/Proof"> Proof of Concept </A></LI><LI><A href="https://2016.igem.org/Team:Freiburg/Binding"> Binding </A></LI><LI><A href="https://2016.igem.org/Team:Freiburg/Delivery"> Delivery </A></LI><LI><A href="https://2016.igem.org/Team:Freiburg/Production_Nanocillus"> Nanocillus </A></LI><LI><A href="https://2016.igem.org/Team:Freiburg/Biobricks"> Biobricks </A></LI><LI><A href="https://2016.igem.org/Team:Freiburg/Shampoo"> Extra </A></LI><LI class="menu_item"><A href="https://2016.igem.org/Team:Freiburg/Safety"> Safety </A></LI><LI><A href="https://2016.igem.org/Team:Freiburg/Knockouts"> Knock Outs </A></LI><LI><A href="https://2016.igem.org/Team:Freiburg/Killswitch"> Kill Switch </A></LI><LI><A href="https://2016.igem.org/Team:Freiburg/PR"> Public relations </A></LI><LI><A href="https://2016.igem.org/Team:Freiburg/Human_Practices"> Human Practice </A></LI><LI><A href="https://2016.igem.org/Team:Freiburg/Integrated_Practices"> Integrated </A></LI><LI><A href="https://2016.igem.org/Team:Freiburg/Collaborations"> Collaborations </A></LI><LI><A href="https://2016.igem.org/Team:Freiburg/Poster"> Poster </A></LI></NAV></DIV><P id="back-top"><A href="#top">Back to Top</A></P><DIV class="column full_size"><DIV class="color6"><H5 style="text-align: center"> Microbial surface display </H5><DIV class="para_center_20"><FONT color="#aea8d3"> 
The display of recombinant proteins on the surface of microbial cells or bacteriophages has had a significant impact on biotechnological and medical research and represents a valuable tool with far-reaching applications. Since its introduction in 1985 by George Smith<SUP>1</SUP>, cell surface display techniques were continuously extended and are applicable in various fields including the administration of vaccines, screening of protein libraries, protein engineering or even biocatalysis. The principle of display approaches is based on the fusion of a heterologous passenger protein to a naturally occurring anchor protein on the surface of the host cell. Depending on the anchoring motif, the heterologous protein is transported by endogenous pathways across the cell membrane or wall and displayed on the surface where it is accessible by its surroundings.

Selecting a suitable chassis for surface display of proteins considerably depends on the experimental settings and involves a wide range of platforms to choose from. Bacteriophages represent the first expression system commonly used for the selection of antibody fragment libraries exhibiting the highest affinity towards their respective target<SUP>2</SUP> (Figure 1).
</FONT></DIV><DIV class="color8"><DIV class="para_center_figure"><B>Figure 1: Phage display screening of binding proteins.</B>
A library containing multiple variants of genes for a binding protein are expressed and displayed on the surface of bacteriophages resulting in a phage library. This library is screening for variants with the highest binding properties towards a surface attached target. By washing procedures unspecific and weak interacting phages can be removed while phages with high affinity binding proteins are maintained. After elution the desirable phages are amplified by infection of their respective bacterial host. Once the phages are purified they can be analyzed or re-entered into the screening process. 
</DIV></DIV><DIV class="para_center_20">
Further development of phage libraries displaying a variety of exogenous proteins and peptides increased their applicability in other scientific fields including drug discovery design, vaccine development or analysis of protein-protein interactions<SUP>3</SUP>. Though the phagemid vectors enable a high flexibility and an enormous diversity, a major drawback still represents the restriction of the displayed protein size<SUP>4</SUP>. Cell based display techniques offer the advantage of a comparatively easy display of functional enzymes on the surface providing the possibility to generate biocatalysts<SUP>5</SUP>. Gram-negative bacteria such as <I> E. coli</I> represent a frequently used platform, mostly due to established and easily available tools for genetic modifications, high transformation efficiency and extensive characterization of the host<SUP>6</SUP>. The anchoring of functional enzymes on the bacterial membrane resulted in versatile applications including the adsorption and removal of heavy metals<SUP>7,8</SUP>, vaccination<SUP>9</SUP> or whole-cell factories for the synthesis of biomolecules<SUP>10</SUP>. For the display of proteins requiring posttranslational modifications, which are not accomplished by prokaryotic platforms, yeast surface display represents a more suitable approach harnessing the eukaryotic expression system<SUP>11</SUP>. However, conventional prokaryotic surface display systems require passenger proteins to cross a cytoplasmic membrane and cell wall, which greatly hampers efficiency of display<SUP>12</SUP>. Novel bacterial display systems, utilizing microbial endospores overcome those limitations. Bacteria, such as <I> Bacillus subtilis</I> are able to sporulate upon unfavorable conditions resulting in an extremely stable and resistant survival form of the bacteria, withstanding UV radiations, toxic chemicals, lytic enzymes or extreme temperatures<SUP>13</SUP>. Due to the mechanism of sporulation displayed proteins are synthesized by the mother cell and then assembled on the spore surface. They do not need to be transported across a membrane, thus circumventing bacterial secretion and transport machineries for the display of heterologous proteins (See Figure 2). Previous approaches demonstrated the feasibility of <I> B. subtilis</I> endospores for the expression and display of multimeric proteins<SUP>14,15</SUP>. The synthesis of recombinant proteins during late stages of the life cycle enables <I>B. subtilis</I> to tolerate the display of growth‑disturbing or even lethal proteins<SUP>15</SUP>.
</DIV></DIV><DIV class="color8"><DIV class="para_center_figure"><B>Figure 2: Sporulation process and surface display.</B><B>(A)</B> Upon stress <I>B. subtilis</I> performs a polar cell division. The smaller forespore is engulfed by the mother cell.<B>(B)</B> The mother cell synthesizes and assembles the multiple layers of spore coat proteins, including the passenger proteins which are anchored on the naturally occurring surface proteins of the spore. This mechanism eliminates the need of a transport across membranes or cell walls.<B>(C)</B> The mother cell is lysed in order to release the highly resistant spores displaying heterologous proteins on their surface.
</DIV></DIV><DIV class="color6"><DIV class="para_center_20"><FONT color="#aea8d3">
Considering the high resistance of the spores towards exterior stress in combination with their <A target="_blank" href="https://2016.igem.org/Team:Freiburg/B_subtilis" style="text-align: center;font-size:20px;color:#e8a126">probiotic</A> attributes and their bio-compatibility in the human body making them an ideal chassis for the display of functional enzymes in medical and environmental applications. <A target="_blank" href="https://2016.igem.org/Team:Freiburg/Goals_Approach" style="text-align: center;font-size:20px;color:#e8a126">Our project</A> involves the display of functional glutathione S-transferase (GST) and an epitope-specific nanobody. GST, as the activity moiety provides the functionality to convert the prodrug azathioprine to its active form while the nanobody ensures the specific adhesion to disease affected areas. Combining both moieties in <I>B. subtilis</I> provides the utilization of the spores for <A target="_blank" href="https://2016.igem.org/Team:Freiburg/Targeted_Drug_Delivery" style="text-align: center;font-size:20px;color:#e8a126">targeted drug delivery</A> .
</FONT></DIV></DIV><DIV class="color8"><DIV class="para_center_Quellen"><FONT color="#504646">


References:


1.     Smith, G. P. Filamentous fusion phage: novel expression vectors that display cloned antigens on the virion surface. Science 228, 1315–1317 (1985).

2.     Schmitz, U., Versmold, A., Kaufmann, P. &amp; Frank, H. G. Phage display: A molecular tool for the generation of antibodies - A review. Placenta 21, 106–112 (2000).

3.     Pande, J., Szewczyk, M. M. &amp; Grover, A. K. Phage display: Concept, innovations, applications and future. Biotechnol. Adv. 28, 849–858 (2010).

4.     Li, M. Applications of display technology in protein analysis. Nat. Biotechnol. 18, 1251–1256 (2000).

5.     Richins, R. D., Kaneva, I., Mulchandani, A. &amp; Chen, W. Biodegradation of organophosphorus pesticides by surface-expressed organophosphorus hydrolase. Nat. Biotechnol. 15, 984–7 (1997).

6.     van Bloois, E., Winter, R. T., Kolmar, H. &amp; Fraaije, M. W. Decorating microbes: Surface display of proteins on Escherichia coli. Trends Biotechnol. 29, 79–86 (2011).

7.     Sousa, C. et al. Metalloadsorption by Escherichia coli Cells Displaying Yeast and Mammalian Metallothioneins Anchored to the Outer Membrane Protein LamB Metalloadsorption by Escherichia coli Cells Displaying Yeast and Mammalian Metallothioneins Anchored to the Outer Membr. J. Bacteriol. 180, 2280–2284 (1998).

8.     Bae, W., Chen, W., Mulchandani, A. &amp; Mehra, R. K. Enhanced bioaccumulation of heavy metals by bacterial cells displaying synthetic phytochelatins. Biotechnol. Bioeng. 70, 518–524 (2000).

9.     Lång, H., Mäki, M., Rantakari, A. &amp; Korhonen, T. K. Characterization of adhesive epitopes with the OmpS display system. Eur. J. Biochem. 267, 163–170 (2000).

10.     Jose, J., Bernhardt, R. &amp; Hannemann, F. Cellular surface display of dimeric Adx and whole cell P450-mediated steroid synthesis on E. coli. J. Biotechnol. 95, 257–268 (2002).

11.     Lauren R. Pepper, Yong Ku Cho, Eric T. Boder, E. V. S. A decade of yeast surface display technology: Where are we now? Comb Chem High Throughput Screen 11, 127–134 (2008).

12.     Lee, S. Y., Choi, J. H. &amp; Xu, Z. Microbial cell-surface display. Trends Biotechnol. 21, 45–52 (2003).

13.     Henriques, A. O. &amp; Moran, C. P. Structure, assembly, and function of the spore surface layers. Annu. Rev. Microbiol. 61, 555–588 (2007).

14.     Hinc, K. et al. Expression and display of UreA of Helicobacter acinonychis on the surface of Bacillus subtilis spores. Microb. Cell Fact. 9, 2 (2010).

15.     Kim, J. H., Lee, C. S. &amp; Kim, B. G. Spore-displayed streptavidin: A live diagnostic tool in biotechnology. Biochem. Biophys. Res. Commun. 331, 210–214 (2005).</FONT></DIV></DIV></DIV><FOOTER><DIV class="center"><DIV style="display:inline-block"><P>Posted by: iGEM Freiburg </P></DIV><DIV style="display:inline-block"><P> Nanocillus - 'cause spore is more! </P></DIV><DIV style="display:inline-block"><P><A style="color:#e8a126" href="https://2016.igem.org/Team:Freiburg/Kontakt"> Contact </A></P></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>