<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Freiburg_NotebookCloning skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Freiburg/NotebookCloning</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="menuSmall"><H5><A href=" https://2016.igem.org/Team:Freiburg/sitemap">  Menu  </A></H5></DIV><DIV class="menu-wrap"><NAV class="menu"><UL class="clearfix"><LI class="menu_item"><A href=" https://2016.igem.org/Team:Freiburg/Team"> Team </A></LI><LI><A href="https://2016.igem.org/Team:Freiburg/Attributions">  Attributions </A></LI><LI><A href="https://2016.igem.org/Team:Freiburg/Achievements">  Achievements </A></LI><LI><A href="https://2016.igem.org/Team:Freiburg/Pictures">  Pictures </A></LI><LI><A href="https://2016.igem.org/Team:Freiburg/GiantJamboree"> Giant Jamboree </A></LI></UL><LI class="menu_item"><A href="https://2016.igem.org/Team:Freiburg/Overview"> Project </A></LI><LI><A href="https://2016.igem.org/Team:Freiburg/Motivation"> Motivation </A></LI><LI><A href="https://2016.igem.org/Team:Freiburg/Goals_Approach"> Approach </A></LI><LI><A href="https://2016.igem.org/Team:Freiburg/B_subtilis"> B. subtilis </A></LI><LI><A href="https://2016.igem.org/Team:Freiburg/Surface_Display"> Microbial surface display
 </A></LI><LI><A href="https://2016.igem.org/Team:Freiburg/Targeted Drug Delivery"> Targeted Drug Delivery</A></LI><LI class="menu_item"><A href="https://2016.igem.org/Team:Freiburg/Lab"> Lab </A></LI><LI><A href="https://2016.igem.org/Team:Freiburg/Experiments"> Methods </A></LI><LI><A href="https://2016.igem.org/Team:Freiburg/Notebook"> Journal </A></LI><LI class="menu_item"><A href="https://2016.igem.org/Team:Freiburg/Results"> Results</A></LI><LI><A href="https://2016.igem.org/Team:Freiburg/Proof"> Proof of Concept </A></LI><LI><A href="https://2016.igem.org/Team:Freiburg/Binding"> Binding </A></LI><LI><A href="https://2016.igem.org/Team:Freiburg/Delivery"> Delivery </A></LI><LI><A href="https://2016.igem.org/Team:Freiburg/Production_Nanocillus"> Nanocillus </A></LI><LI><A href="https://2016.igem.org/Team:Freiburg/Biobricks"> Biobricks </A></LI><LI><A href="https://2016.igem.org/Team:Freiburg/Shampoo"> Extra </A></LI><LI class="menu_item"><A href="https://2016.igem.org/Team:Freiburg/Safety"> Safety </A></LI><LI><A href="https://2016.igem.org/Team:Freiburg/Knockouts"> Knock Outs </A></LI><LI><A href="https://2016.igem.org/Team:Freiburg/Killswitch"> Kill Switch </A></LI><LI><A href="https://2016.igem.org/Team:Freiburg/PR"> Public relations </A></LI><LI><A href="https://2016.igem.org/Team:Freiburg/Human_Practices"> Human Practice </A></LI><LI><A href="https://2016.igem.org/Team:Freiburg/Integrated_Practices"> Integrated </A></LI><LI><A href="https://2016.igem.org/Team:Freiburg/Collaborations"> Collaborations </A></LI><LI><A href="https://2016.igem.org/Team:Freiburg/Poster"> Poster </A></LI></NAV></DIV><P id="back-top"><A href="#top">Back to Top</A></P><DIV class="color1"><DIV class="img_menu"><H5 style="text-align: center">Lab Journal </H5></DIV></DIV><DIV class="color8"><DIV class="para_center_20"><DIV class="level1"><H5 class="sectionedit1"> Group 1 - Cloning</H5><DIV class="tags"> To keep an overview of the cloned constructs every plasmid was assigned to an ID: pIG16_000.
                     All used oligos were assigned to an ID as well: oIG16_000.
                     The complete list of the resulting bacterial strains and oligos can be found in the attached Excel files. The spore coat proteins cotZ, cotG, cotB and cgeA were amplified from the genome of B. subtilis 168. The anti-GFP nanobody and the GST were amplified from plasmids provided by Dr. Nicole Gensch and Dr. Maximilian Ulbrich. For the cloning strategy see Project - <A target="_blank" href="https://2016.igem.org/Team:Freiburg/Goals_Approach">Approach</A>. 
<A href="https://static.igem.org/mediawiki/2016/7/7f/T--Freiburg--OligoList.xlsx" target="blank" style="color:#e3c5c5">List of Oligos</A><A href="https://static.igem.org/mediawiki/2016/4/4f/T--Freiburg--StrainList.xlsx" target="blank" style="color:#e3c5c5">List of Strains</A></DIV></DIV></DIV></DIV><DIV class="color2"><DIV class="para_20"><H3 class="sectionedit2"><A name="section070716" id="section070716">07.07.16</A></H3><DIV class="level3"><P><STRONG>1) Transformation</STRONG> Julia Bartels from iGEM 2012 sent us integration vectors for B.subtilis. Sebastian from AG Weber (BIOSS) shared plasmids containing mCherry and GFP. </P><P><STRONG>Vectors:</STRONG> No.1. pBS0K-Pspac-[RFP]
                     No.2. pBS1C3-[RFP]
                     No.3. pBS2E-[RFP]
                     No.4. pBS3Clux-[RFP]
                     No.5. pBS4S-[RFP]
                     No.6. pSBBs4S-Sporovector
                     7. mCherry
                     8. GFP
                     Transformation of the Vectors from Julia Bartels and Sebastian (from AG Weber) into chemically competent E.coli K12 DH5alpha was performed according to protocol. The E.coli were incubated for 1 hour at 37 °C and 250 rpm and spread on LB-Agar plates supplemented with ampicilin. Incubation for o/n at 37 °C.
                    <STRONG>2) Preparation of competent E.coli</STRONG> For the preparation for chemically competent E.coli one colony from an agar plate was picked in inoculated in 5 mL LB-medium. Incubation o/n at 37 °C and 250 rpm.
                    </P></DIV><H3 class="sectionedit3"><A name="section080716" id="section080716">08.07.16</A></H3><DIV class="level3"><P><STRONG>1) Production of competent E.coli</STRONG> Preparation of chemically competent cells according to the protocol of the manufacturer (Zymoresearch, Z competent Mix&amp;Go kit)
                     The resulting cell suspension was aliquoted (100 µL) in tubes and stored at -80 °C.
                    </P><P><STRONG>2) Inoculation </STRONG> From each transformation plate(No. 1-6) 5 colonies were picked and inoculated in 5 mL LB medium supplemented with Ampicilin.
                     Incubation o/n at 37°C and 250 rpm.
                     The B.subtilis W168 strain was inoculated in 5 mL of LB medium and incubated o/n at 37 °C and 250 rpm. In parallel, a sample of the W168 strain was spread on a LB agar plate and incubated o/n at 37 °C.
                    </P></DIV><H3 class="sectionedit4"><A name="section090716" id="section090716">09.07.16</A></H3><DIV class="level3"><P><STRONG>1) MiniPrep</STRONG> The the plasmids of the inoculated colonies (No.1-6) were prepared using the QiaGen MiniPrep kit according the instructions of the manufacturer and eluted in 20 µL of ultra pure water. The concentration of the DNA was spectroscopically determined by a Nanodrop.
                    <STRONG>2) Colony PCR</STRONG> Colony PCR for the amplication of the cotZ and cgeA gene was performed using the following oligos:
                     cotZ: oIG16_1+2 Annealing temperature: 57 °C
                     cgeA: oIG16_43+34 Annealing temperature: 65 °C
                    <STRONG>Reaction conditions</STRONG></P><DIV class="table sectionedit5"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0">component</TH><TH class="col1">concentration</TH><TH class="col2">volume [µL]</TH></TR><TR class="row1"><TD class="col0">1 B.subtilis colony</TD><TD class="col1">-</TD><TD class="col2">-</TD></TR><TR class="row2"><TD class="col0">Primer fw</TD><TD class="col1">10 µM</TD><TD class="col2">1.0</TD></TR><TR class="row3"><TD class="col0">Primer rv</TD><TD class="col1">10 µM</TD><TD class="col2">1.0</TD></TR><TR class="row4"><TD class="col0">dNTPs</TD><TD class="col1">40 µM</TD><TD class="col2">0.4</TD></TR><TR class="row5"><TD class="col0">HF Buffer</TD><TD class="col1">5X</TD><TD class="col2">4.0</TD></TR><TR class="row6"><TD class="col0">Phusion Pol</TD><TD class="col1">2U/µL</TD><TD class="col2">0.2</TD></TR><TR class="row7"><TD class="col0">ddH2O</TD><TD class="col1">-</TD><TD class="col2">ad20 µL </TD></TR></TBODY></TABLE></DIV><P><STRONG>Cycler Program</STRONG></P><DIV class="table sectionedit6"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0">Step</TH><TH class="col1">Temperature [°C]</TH><TH class="col2">Duration</TH><TH class="col3">Repeats</TH></TR><TR class="row1"><TD class="col0">Initial denaturation</TD><TD class="col1">98 </TD><TD class="col2"> 1 min</TD><TD class="col3">1</TD></TR><TR class="row2"><TD class="col0">Denaturation</TD><TD class="col1">98 </TD><TD class="col2">10 s</TD><TD class="col3"> 30</TD></TR><TR class="row3"><TD class="col0">Annealing</TD><TD class="col1">58 or 65</TD><TD class="col2">10 s</TD><TD class="col3"> 30</TD></TR><TR class="row4"><TD class="col0">Elongation</TD><TD class="col1">72 </TD><TD class="col2">30 s</TD><TD class="col3"> 30</TD></TR><TR class="row5"><TD class="col0">Final elongation</TD><TD class="col1">72 </TD><TD class="col2">5 min</TD><TD class="col3">1</TD></TR><TR class="row6"><TD class="col0">Storage</TD><TD class="col1">8</TD><TD class="col2"> - </TD></TR></TBODY></TABLE></DIV></DIV></DIV></DIV><DIV class="color3"><DIV class="para_20"><H3 class="sectionedit7"><A name="section100716" id="section100716">10.07.16</A></H3><DIV class="level3"><P><STRONG>1) Gel electrophoresis</STRONG> 20 µL of the colony PCR samples were supplemented with the appropriate amount of 10X Orange G loading buffer, loaded on a 1 % agarose TAE gel and subjected to electrophoresis for 40 min at 120 V. The DNA was stained with Midori Green. (No bands were observable at all. The amount of Midori Green was not sufficient)
                    </P></DIV><H3 class="sectionedit8"><A name="section110716" id="section110716">11.07.16</A></H3><DIV class="level3"><P><STRONG>1)Colony PCR</STRONG> from 09.07.2016 was repeated at a total volume of 50 µL. After electrophoresis the sample were supplemented with the appropriate amount of 10X orange G loading buffer and loaded on an 1 % TAE agarose gel. The gel was subjected to electrophoresis for 40 min at 120 V. The DNA was stained with Midori Green.
                     Only the amplified cotZ was visible at UV Light.
                     The cotZ band was excised and the DNA was extracted using the QiaQuick Gel extraction kit according the the instructions of the manufacturer. The DNA was eluted in 20 µL of ultra pure H2O and the concentration was photometrically determined by a NanoDrop.
                     CotZ gelextraction concentration: 18 ng/µL </P></DIV><H3 class="sectionedit9"><A name="section120716" id="section120716">12.07.16</A></H3><DIV class="level3"><P><STRONG>1) Colony PCR</STRONG> Colony-PCR for the amplification of cgeA and cotZ was repeated
                     Primer:
                     cgeA: oIG16_034 + 043
                     cotZ: oIG16_001 + 002
                    </P><P><STRONG>Reaction conditions</STRONG></P><DIV class="table sectionedit10"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0">component</TH><TH class="col1">concentration</TH><TH class="col2">volume [µL]</TH></TR><TR class="row1"><TD class="col0">1 B.subtilis colony W168</TD><TD class="col1">-</TD><TD class="col2">-</TD></TR><TR class="row2"><TD class="col0">Primer fw</TD><TD class="col1">10 µM</TD><TD class="col2">2.5</TD></TR><TR class="row3"><TD class="col0">Primer rv</TD><TD class="col1">10 µM</TD><TD class="col2">2.5</TD></TR><TR class="row4"><TD class="col0">dNTPs</TD><TD class="col1">40 µM</TD><TD class="col2">0.4</TD></TR><TR class="row5"><TD class="col0">HF Buffer</TD><TD class="col1">5X</TD><TD class="col2">10</TD></TR><TR class="row6"><TD class="col0">Phusion Pol</TD><TD class="col1">2U/µL</TD><TD class="col2">0.5</TD></TR><TR class="row7"><TD class="col0">ddH2O</TD><TD class="col1">-</TD><TD class="col2">ad50 µL </TD></TR></TBODY></TABLE></DIV><P><STRONG>Cycler Program</STRONG></P><DIV class="table sectionedit11"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0">Step</TH><TH class="col1">Temperature [°C]</TH><TH class="col2">Duration</TH><TH class="col3">Repeats</TH></TR><TR class="row1"><TD class="col0">Initial denaturation</TD><TD class="col1">98 </TD><TD class="col2"> 1 min</TD></TR><TR class="row2"><TD class="col0">Denaturation</TD><TD class="col1">98 </TD><TD class="col2">10 s</TD><TD class="col3"> 30X</TD></TR><TR class="row3"><TD class="col0">Annealing</TD><TD class="col1">58 or 65</TD><TD class="col2">10 s</TD><TD class="col3"> 30X</TD></TR><TR class="row4"><TD class="col0">Elongation</TD><TD class="col1">72 </TD><TD class="col2">30 s</TD><TD class="col3">30X </TD></TR><TR class="row5"><TD class="col0">Final elongation</TD><TD class="col1">72 </TD><TD class="col2">5 min</TD></TR><TR class="row6"><TD class="col0">Storage</TD><TD class="col1">8</TD><TD class="col2"> - </TD></TR></TBODY></TABLE></DIV><P> The samples were supplemented with the appropriate amount of 10 X Orange G loading dye and loaded on a 1 % TAE agarose gel. As molecular marker a 2-log marker was used.
                     No bands were observed at UV light. </P></DIV></DIV></DIV><DIV class="color4"><DIV class="para_20"><H3 class="sectionedit12"><A name="section130716" id="section130716">13.07.16</A></H3><DIV class="level3"><P><STRONG>1) Gradient colony PCR</STRONG> To amplify cgeA 7 samples (20 µL each) were amplified by gradient colony PCR applying annealing temperatures from 58 - 68 °C.
                     An additional 50µl sample was made to amplify CotZ.
                    </P><P> The samples were supplemented with 10 X Orange G loading buffer and analyzed by gel electrophoresis. No bands were observable.
                    </P><H3 class="sectionedit13"><A name="section140716" id="section140716">14.07.16</A></H3><DIV class="level3"><P><STRONG>1) Repeat of gradient colony PCR</STRONG> To amplify cgeA 7 samples (20 µL each) were amplified by gradient colony PCR applying annealing temperatures from 58 - 68 °C. The duration of initial denaturation was elongated to 5 min.
                         The samples were supplemented with 10 X Orange G loading buffer and analyzed by gel electrophoresis. No bands were observable.
                        </P></DIV><H3 class="sectionedit14"><A name="section150716" id="section150716">15.07.16</A></H3><DIV class="level3"><P><STRONG>1) Lysis of B. subtilis for colony PCR</STRONG> 4 different approaches were tried to lyse the bacteria prior to colony PCR.
                         1. 
                         3. Mini-Prep lysis buffer.
                         4. Laemmlie sample buffer 100°C 5 Min.
                        </P><P> 1 µl of each sample was used for amplification of cotZ using the oligos oIG16_1+2. The extracted cotZ gene from 11.07.16 was amplified as control alongside the lysed samples
                        </P><P><STRONG>Reaction conditions</STRONG></P><DIV class="table sectionedit15"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0">component</TH><TH class="col1">concentration</TH><TH class="col2">volume [µL]</TH></TR><TR class="row1"><TD class="col0">lysed B.subtilis W168</TD><TD class="col1">-</TD><TD class="col2">1 µL</TD></TR><TR class="row2"><TD class="col0">Primer fw</TD><TD class="col1">10 µM</TD><TD class="col2">2.5</TD></TR><TR class="row3"><TD class="col0">Primer rv</TD><TD class="col1">10 µM</TD><TD class="col2">2.5</TD></TR><TR class="row4"><TD class="col0">dNTPs</TD><TD class="col1">40 µM</TD><TD class="col2">0.4</TD></TR><TR class="row5"><TD class="col0">HF Buffer</TD><TD class="col1">5X</TD><TD class="col2">10</TD></TR><TR class="row6"><TD class="col0">Phusion Pol</TD><TD class="col1">2U/µL</TD><TD class="col2">0.5</TD></TR><TR class="row7"><TD class="col0">ddH2O</TD><TD class="col1">-</TD><TD class="col2">ad50 µL </TD></TR></TBODY></TABLE></DIV><P><STRONG>Cycler Program</STRONG></P><DIV class="table sectionedit16"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0">Step</TH><TH class="col1">Temperature [°C]</TH><TH class="col2">Duration</TH><TH class="col3">Repeats</TH></TR><TR class="row1"><TD class="col0">Initial denaturation</TD><TD class="col1">98 </TD><TD class="col2"> 5 min</TD></TR><TR class="row2"><TD class="col0">Denaturation</TD><TD class="col1">98 </TD><TD class="col2">10 s</TD><TD class="col3"> 30X</TD></TR><TR class="row3"><TD class="col0">Annealing</TD><TD class="col1">65</TD><TD class="col2">10 s</TD><TD class="col3"> 30X</TD></TR><TR class="row4"><TD class="col0">Elongation</TD><TD class="col1">72 </TD><TD class="col2">30 s</TD><TD class="col3"> 30X</TD></TR><TR class="row5"><TD class="col0">Final elongation</TD><TD class="col1">72 </TD><TD class="col2">5 min</TD></TR><TR class="row6"><TD class="col0">Storage</TD><TD class="col1">8</TD><TD class="col2"> - </TD></TR></TBODY></TABLE></DIV><P><STRONG> 2) Testdigestion</STRONG> PvuII test digestion for verification of the plasmids sent by Julia Bartels.
                         Reaction conditions:
                        </P><DIV class="table sectionedit17"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0">component</TH><TH class="col1">concentration</TH><TH class="col2">volume</TH></TR><TR class="row1"><TD class="col0">MiniPrep DNA</TD><TD class="col1">200-600 ng/µL</TD><TD class="col2">2 µL</TD></TR><TR class="row2"><TD class="col0">CutSmart Buffer (NEB)</TD><TD class="col1">10X</TD><TD class="col2">2 µL</TD></TR><TR class="row3"><TD class="col0">PvuII-HF</TD><TD class="col1">20 U/ µL</TD><TD class="col2">0.1 µL</TD></TR><TR class="row4"><TD class="col0">ddH2O</TD><TD class="col1">-</TD><TD class="col2">ad20 µL</TD></TR></TBODY></TABLE></DIV><P> Incubation at 37 °C for 1 hour. The samples were analyzed by gel electrophoresis on a 1 % TAE agarose gel.
                        </P><P><STRONG>3) Subcloning</STRONG> Subcloning of the amplified <EM>cotZ</EM> gene into the linearized pJET1.2 vector was performed using the pJET subcloning kit according to the instructions of the manufacturer. 5 µL of the ligation mixture was used for transformation of chemically competent E.coli DH5alpha (Mix &amp; Go). The transformed bacteria were spread on a LB-agar plate supplemented with amplicilin and incubated o/n at 37 °C. A control ligation was prepared using the DNA fragments supplied by the manufacturer.
                        </P></DIV></DIV></DIV></DIV><DIV class="color5"><DIV class="para_20"><H3 class="sectionedit18"><A name="section160716" id="section160716">16.07.16</A></H3><DIV class="level3"><P><STRONG>1) Colony PCR </STRONG> for the amplication of the cgeA was performed using the following oligos:
                     cgeA: oIG16_43+34 Annealing temperature: 65 °C
                     To lyse the bacteria prior to the colony PCR.
                    </P><P><STRONG>Reaction conditions</STRONG></P><DIV class="table sectionedit19"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0">component</TH><TH class="col1">concentration</TH><TH class="col2">volume [µL]</TH></TR><TR class="row1"><TD class="col0">lysed B.subtilis W168</TD><TD class="col1">-</TD><TD class="col2">1 µL</TD></TR><TR class="row2"><TD class="col0">Primer fw</TD><TD class="col1">10 µM</TD><TD class="col2">2.5</TD></TR><TR class="row3"><TD class="col0">Primer rv</TD><TD class="col1">10 µM</TD><TD class="col2">2.5</TD></TR><TR class="row4"><TD class="col0">dNTPs</TD><TD class="col1">40 µM</TD><TD class="col2">0.25</TD></TR><TR class="row5"><TD class="col0">HF Buffer</TD><TD class="col1">5X</TD><TD class="col2">10</TD></TR><TR class="row6"><TD class="col0">Phusion Pol</TD><TD class="col1">2U/µL</TD><TD class="col2">0.5</TD></TR><TR class="row7"><TD class="col0">ddH2O</TD><TD class="col1">-</TD><TD class="col2">ad50 µL </TD></TR></TBODY></TABLE></DIV><P><STRONG>Cycler Program</STRONG></P><DIV class="table sectionedit20"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0">Step</TH><TH class="col1">Temperature [°C]</TH><TH class="col2">Duration</TH><TH class="col3">Repeats</TH></TR><TR class="row1"><TD class="col0">Initial denaturation</TD><TD class="col1">98 </TD><TD class="col2"> 5 min</TD></TR><TR class="row2"><TD class="col0">Denaturation</TD><TD class="col1">98 </TD><TD class="col2">10 s</TD><TD class="col3">30X</TD></TR><TR class="row3"><TD class="col0">Annealing</TD><TD class="col1">65</TD><TD class="col2">10 s</TD><TD class="col3">30X</TD></TR><TR class="row4"><TD class="col0">Elongation</TD><TD class="col1">72 </TD><TD class="col2">30 s</TD><TD class="col3">30X</TD></TR><TR class="row5"><TD class="col0">Final elongation</TD><TD class="col1">72 </TD><TD class="col2">5 min</TD></TR><TR class="row6"><TD class="col0">Storage</TD><TD class="col1">8</TD><TD class="col2"> - </TD></TR></TBODY></TABLE></DIV><P><STRONG>2) Inoculation</STRONG> Subcloning and transformation of pJET1.2-cotZ resulted only in a few colonies. Three of them were picked and inoculated in 5 mL LB-medium w/ ampicilin and incubated o/n at 37°C and 250 rpm. The transformation with the control plate yielded &gt;100 colonies. </P></DIV><H3 class="sectionedit21"><A name="section170716" id="section170716">17.07.16</A></H3><DIV class="level3"><P><STRONG>1) Colony PCR</STRONG> 5 µL of vegetative B.subtilis from a liquid culture were diluted 1:10 in Qiagen EB buffer and incubated at 100°C for 5min to achieve cell lysis
                     1 µL of the dilution was used for amplification of cgeA by gradient PCR and 8 samples were prepared.
                    </P><P><STRONG> Reaction conditions</STRONG></P><DIV class="table sectionedit22"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0">component</TH><TH class="col1">concentration</TH><TH class="col2">volume [µL]</TH></TR><TR class="row1"><TD class="col0">lysed B.subtilis W168</TD><TD class="col1">-</TD><TD class="col2">1 µL</TD></TR><TR class="row2"><TD class="col0">Primer fw</TD><TD class="col1">10 µM</TD><TD class="col2">1</TD></TR><TR class="row3"><TD class="col0">Primer rv</TD><TD class="col1">10 µM</TD><TD class="col2">1</TD></TR><TR class="row4"><TD class="col0">dNTPs</TD><TD class="col1">40 µM</TD><TD class="col2">0.1</TD></TR><TR class="row5"><TD class="col0">HF Buffer</TD><TD class="col1">5X</TD><TD class="col2">5</TD></TR><TR class="row6"><TD class="col0">Phusion Pol</TD><TD class="col1">2U/µL</TD><TD class="col2">0.2</TD></TR><TR class="row7"><TD class="col0">ddH2O</TD><TD class="col1">-</TD><TD class="col2">ad20 µL </TD></TR></TBODY></TABLE></DIV><P><STRONG>Cycler Program</STRONG></P><DIV class="table sectionedit23"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0">Step</TH><TH class="col1">Temperature [°C]</TH><TH class="col2">Duration</TH><TH class="col3">Repeats</TH></TR><TR class="row1"><TD class="col0">Initial denaturation</TD><TD class="col1">98 </TD><TD class="col2"> 5 min</TD></TR><TR class="row2"><TD class="col0">Denaturation</TD><TD class="col1">98 </TD><TD class="col2">10 s</TD><TD class="col3">30X</TD></TR><TR class="row3"><TD class="col0">Annealing</TD><TD class="col1">gradient 58-68</TD><TD class="col2">20 s</TD><TD class="col3">30X</TD></TR><TR class="row4"><TD class="col0">Elongation</TD><TD class="col1">72 </TD><TD class="col2">30 s</TD><TD class="col3">30X</TD></TR><TR class="row5"><TD class="col0">Final elongation</TD><TD class="col1">72 </TD><TD class="col2">5 min</TD></TR><TR class="row6"><TD class="col0">Storage</TD><TD class="col1">10</TD><TD class="col2"> - </TD></TR></TBODY></TABLE></DIV><P> After thermal cycling the samples were supplemented with the appropriate amount of 10X Orange G loading dye and analyed by agarose gel electrophoresis. </P><P><STRONG>2)  MiniPrep</STRONG> The plasmids pJET1.2-cotZ were prepared using the QIAprep Spin Miniprep Kit according the instructions of the manufacturer and eluted in 50µL of EB Buffer. The concentration of the DNA was spectroscopically determined by a nanodrop.
                    </P><DIV class="table sectionedit24"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0">DNA</TH><TH class="col1">concentration[ng/µL]</TH></TR><TR class="row1"><TD class="col0">pJet_cotZ_#1</TD><TD class="col1">120</TD></TR><TR class="row2"><TD class="col0">pJet_cotZ_#2</TD><TD class="col1">89</TD></TR><TR class="row3"><TD class="col0">pJet_cotZ_#3</TD><TD class="col1">68</TD></TR></TBODY></TABLE></DIV><P><STRONG>3) Testdigestion</STRONG><STRONG> Reaction conditions </STRONG></P><DIV class="table sectionedit25"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0">component</TH><TH class="col1">concentration</TH><TH class="col2">volume</TH></TR><TR class="row1"><TD class="col0">DNA</TD><TD class="col1">68 - 120 ng/µL</TD><TD class="col2">7 µL</TD></TR><TR class="row2"><TD class="col0">XbaI</TD><TD class="col1">20U/µL</TD><TD class="col2">0.2</TD></TR><TR class="row3"><TD class="col0">XhoI</TD><TD class="col1">20U/µL</TD><TD class="col2">0.2</TD></TR><TR class="row4"><TD class="col0">CutSmart</TD><TD class="col1">10X</TD><TD class="col2">2 µL</TD></TR><TR class="row5"><TD class="col0">dH2O</TD><TD class="col1">-</TD><TD class="col2">11 µL</TD></TR></TBODY></TABLE></DIV><P> Incubation for 1 h at 37 °C. Analysis of digestion by agarose gel electrophoresis. </P><P> 2-log DNA ladder is bad in the last couple of gels –&gt; discarded.
                     Not the expected band pattern. Subcloning should be repeated.
                    </P><P><STRONG>3) Colony PCR</STRONG> 5 µL of vegetative B.subtilis from a liquid culture were diluted 1:10 in Qiagen EB buffer and incubated at 100°C for 5min to achieve cell lysis
                     1 µL of the dilution was used for amplification of cgeA and cotZ by PCR.
                     oIG16_001+002: CotZ, Tm = 58 °C oIG16_034+043: cgeA, Tm = 63 °C </P><DIV class="table sectionedit26"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0">component</TH><TH class="col1">concentration</TH><TH class="col2">volume [µL]</TH></TR><TR class="row1"><TD class="col0">lysed B.subtilis W168</TD><TD class="col1">-</TD><TD class="col2">1 µL</TD></TR><TR class="row2"><TD class="col0">Primer fw</TD><TD class="col1">10 µM</TD><TD class="col2">2.5</TD></TR><TR class="row3"><TD class="col0">Primer rv</TD><TD class="col1">10 µM</TD><TD class="col2">2.5</TD></TR><TR class="row4"><TD class="col0">dNTPs</TD><TD class="col1">40 µM</TD><TD class="col2">0.25</TD></TR><TR class="row5"><TD class="col0">HF Buffer</TD><TD class="col1">5X</TD><TD class="col2">10</TD></TR><TR class="row6"><TD class="col0">Phusion Pol</TD><TD class="col1">2U/µL</TD><TD class="col2">0.5</TD></TR><TR class="row7"><TD class="col0">ddH2O</TD><TD class="col1">-</TD><TD class="col2">ad50 µL </TD></TR></TBODY></TABLE></DIV><P><STRONG>Cycler Program</STRONG></P><DIV class="table sectionedit27"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0">Step</TH><TH class="col1">Temperature [°C]</TH><TH class="col2">Duration</TH><TH class="col3">Repeats</TH></TR><TR class="row1"><TD class="col0">Initial denaturation</TD><TD class="col1">98 </TD><TD class="col2"> 5 min</TD></TR><TR class="row2"><TD class="col0">Denaturation</TD><TD class="col1">98 </TD><TD class="col2">10 s</TD><TD class="col3">30X</TD></TR><TR class="row3"><TD class="col0">Annealing</TD><TD class="col1">58 or 63</TD><TD class="col2">10 s</TD><TD class="col3">30X</TD></TR><TR class="row4"><TD class="col0">Elongation</TD><TD class="col1">72 </TD><TD class="col2">30 s</TD><TD class="col3">30X</TD></TR><TR class="row5"><TD class="col0">Final elongation</TD><TD class="col1">72 </TD><TD class="col2">5 min</TD></TR><TR class="row6"><TD class="col0">Storage</TD><TD class="col1">8</TD><TD class="col2"> - </TD></TR></TBODY></TABLE></DIV><P> The bands were excised from the gel and stored at -20 °C o/n </P></DIV><H3 class="sectionedit28"><A name="section180716" id="section180716">18.07.16</A></H3><DIV class="level3"><P><STRONG>1) Gelextraction &amp; Subcloning</STRONG> The DNA was extracted using the QiaGen gel extraction kit according to the protocol of the manufacturer and eluted in 30 µL of ultra pure water. The concentration was determined spectroscopically by a nanodrop.
                    </P><DIV class="table sectionedit29"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0">gene</TH><TH class="col1">concentration</TH></TR><TR class="row1"><TD class="col0">cgeA</TD><TD class="col1">6 ng/µL</TD></TR><TR class="row2"><TD class="col0">cotZ#1</TD><TD class="col1">8 ng/µL</TD></TR><TR class="row3"><TD class="col0">cotZ#2</TD><TD class="col1">11 ng/µL</TD></TR></TBODY></TABLE></DIV><P> The extracted DNA was subcloned into a linearized pJET1.2 vector using the CloneJET PCR Cloning Kit according to the protocol of the manufacturer. 5 µL of the ligation mixture were transformed into chemically competent E.coli DH5alpha (Mix&amp;Go) and spread on a LB-agar plate supplemented with ampicilin and incubated at 37 °C o/n. </P></DIV></DIV></DIV><DIV class="color6"><DIV class="para_20"><H3 class="sectionedit30"><A name="section190716" id="section190716">19.07.16</A></H3><DIV class="level3"><P><STRONG>1) Inoculation</STRONG> 4 colonies per plate were picked and incubated in 5 mL LB-medium (amp) o/n at 37 °C and 250 rpm. </P></DIV><H3 class="sectionedit31"><A name="section200716" id="section200716">20.07.16</A></H3><DIV class="level3"><P><STRONG>1) MiniPrep</STRONG> The plasmids pJET1.2-cotZ-I.#1-4;pJET1.2-cotZ-II.#1-4 and pJET1.2-cgeA#1-4 were prepared using the QIAprep Spin Miniprep Kit according the instructions of the manufacturer and eluted in 20µL of ultra pure water (for better results put tubes for 2 minutes on 50°C thermo; centrifuge for 1min at 13000rpm and repeat with 10µL; increases the amount of deluted DNA). The concentration of the DNA was spectroscopically determined by a nanodrop.
                    </P><DIV class="table sectionedit32"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0">DNA</TH><TH class="col1">concentration[ng/µL]</TH></TR><TR class="row1"><TD class="col0">pJet1.2_cgeA_#1</TD><TD class="col1">127</TD></TR><TR class="row2"><TD class="col0">pJet1.2_cgeA_#2</TD><TD class="col1">214 –&gt; Seq</TD></TR><TR class="row3"><TD class="col0">pJet1.2_cgeA_#3</TD><TD class="col1">279</TD></TR><TR class="row4"><TD class="col0">pJet1.2_cgeA_#4</TD><TD class="col1">141</TD></TR><TR class="row5"><TD class="col0">pJet1.2_cotZ-I_#1</TD><TD class="col1">260</TD></TR><TR class="row6"><TD class="col0">pJet1.2_cotZ-I_#2</TD><TD class="col1">101</TD></TR><TR class="row7"><TD class="col0">pJet1.2_cotZ-I_#3</TD><TD class="col1">409</TD></TR><TR class="row8"><TD class="col0">pJet1.2_cotZ-I_#4</TD><TD class="col1">435</TD></TR><TR class="row9"><TD class="col0">pJet1.2_cotZ-II_#1</TD><TD class="col1">475</TD></TR><TR class="row10"><TD class="col0">pJet1.2_cotZ-II_#2</TD><TD class="col1">350 –&gt; Seq</TD></TR><TR class="row11"><TD class="col0">pJet1.2_cotZ-II_#3</TD><TD class="col1">353</TD></TR><TR class="row12"><TD class="col0">pJet1.2_cotZ-II_#4</TD><TD class="col1">342</TD></TR></TBODY></TABLE></DIV><P><STRONG>2) Test digestion</STRONG> Due to the concentraion difference of the DNA the samples where divided into two groups </P><P><STRONG> Reaction conditions </STRONG> DNA concentration &lt; 300ng/µL
                    </P><DIV class="table sectionedit33"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0">component</TH><TH class="col1">concentration</TH><TH class="col2">volume</TH></TR><TR class="row1"><TD class="col0">DNA</TD><TD class="col1">101 - 279 ng/µL</TD><TD class="col2">5 µL</TD></TR><TR class="row2"><TD class="col0">XbaI</TD><TD class="col1">20U/µL</TD><TD class="col2">0.2</TD></TR><TR class="row3"><TD class="col0">XhoI</TD><TD class="col1">20U/µL</TD><TD class="col2">0.2</TD></TR><TR class="row4"><TD class="col0">CutSmart</TD><TD class="col1">10X</TD><TD class="col2">2 µL</TD></TR><TR class="row5"><TD class="col0">dH2O</TD><TD class="col1">-</TD><TD class="col2">14.1 µL</TD></TR></TBODY></TABLE></DIV><P> DNA concentration &gt; 300ng/µL
                    </P><DIV class="table sectionedit34"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0">component</TH><TH class="col1">concentration</TH><TH class="col2">volume</TH></TR><TR class="row1"><TD class="col0">DNA</TD><TD class="col1">342-475 ng/µL</TD><TD class="col2">1.5 µL</TD></TR><TR class="row2"><TD class="col0">XbaI</TD><TD class="col1">20U/µL</TD><TD class="col2">0.2</TD></TR><TR class="row3"><TD class="col0">XhoI</TD><TD class="col1">20U/µL</TD><TD class="col2">0.2</TD></TR><TR class="row4"><TD class="col0">CutSmart</TD><TD class="col1">10X</TD><TD class="col2">2 µL</TD></TR><TR class="row5"><TD class="col0">dH2O</TD><TD class="col1">-</TD><TD class="col2">12.6 µL</TD></TR></TBODY></TABLE></DIV><P> Control group &lt; 300ng/µL
                    </P><DIV class="table sectionedit35"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0">component</TH><TH class="col1">concentration</TH><TH class="col2">volume</TH></TR><TR class="row1"><TD class="col0">DNA</TD><TD class="col1">101 - 279 ng/µL</TD><TD class="col2">5 µL</TD></TR><TR class="row2"><TD class="col0">dH2O</TD><TD class="col1">-</TD><TD class="col2">5 µL</TD></TR></TBODY></TABLE></DIV><P> Control group &gt; 300ng/µL
                    </P><DIV class="table sectionedit36"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0">component</TH><TH class="col1">concentration</TH><TH class="col2">volume</TH></TR><TR class="row1"><TD class="col0">DNA</TD><TD class="col1">342-475 ng/µL</TD><TD class="col2">1.5 µL</TD></TR><TR class="row2"><TD class="col0">dH2O</TD><TD class="col1">-</TD><TD class="col2">8,5 µL</TD></TR></TBODY></TABLE></DIV><P> Incubation for 1 h at 37 °C. Analysis of digestion by electrophoresis on a 1% TAE agarose gel.
                    </P><P><STRONG>3) Sequencing</STRONG> Sequencing of pJET1.2-cgeA #2 (IC0225) and pJET1.2-cotZII #2 (IC0224) by GATC biotech. </P></DIV><H3 class="sectionedit37"><A name="section210716" id="section210716">21.07.16</A></H3><DIV class="level3"><P><STRONG>1) Inoculation</STRONG> Confirmation of the proper sequences for pJET1.2_cgeA#2 and pJET1.2_cotZII#2. The colonies were re-inoculated in 5 mL LB-medium (w/ amp) and incubated at 37 °C, 250 rpm o/n. </P></DIV></DIV></DIV><DIV class="color7"><DIV class="para_20"><H3 class="sectionedit38"><A name="section220716" id="section220716">22.07.16</A></H3><DIV class="level3"><P><STRONG>1) Transformation &amp; Inoculation</STRONG> Nicole provided us with a LB-plate containing E.coli harboring pGEX6P1 and pRP261 (Both plasmids contain glutathion S transferase). Max provided us with a sample of pET303_aGFPnano_TEV_10His (plasmid containing the anti-GFP nanobody). The pET303 plasmid was transformed into chemically competent E.coli DH5alpha and spread on a LB-agar plate supplemented with ampicilin and incubated o/n at 37 °C. </P></DIV><H3 class="sectionedit39"><A name="section230716" id="section230716">23.07.16</A></H3><DIV class="level3"><P><STRONG>1) Inoculation</STRONG> Inoculation of the E.coli DH5alpha containing pGEX6P1, pRP261 and pET303_aGFPnano in 5 mL LB medium supplemented with ampicilin. Incubation o/n at 37 °C, 250 rpm.
                     Glycerol stocks of pJET1.2-cgeA and pJET1.2-cotZ were prepared (15% [v/v] Glycerol).
                     Inoculation of pBS1C3-[RFP] culture #2 for test-transformation of B.subtilis. </P><P><STRONG>2) Transformation</STRONG> The pSB1C3-[RFP] vector from the iGEM distribution kit (plate 1, position 23-O) was solubilized with 10 µL of ultra pure water. 1 µL was used for transformation of chemically competent E.coli DH5alpha and spread on a LB-agar plate supplemented with chloramphenicol and incubated o/n at 37 °C. </P><P><STRONG>3) Extension PCR </STRONG> of cgeA (from pJET1.2_cgeA) and anti GFP-Nanobody (pET303) *Reaction conditions*
                    </P><DIV class="table sectionedit40"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0">component</TH><TH class="col1">concentration</TH><TH class="col2">volume</TH></TR><TR class="row1"><TD class="col0">DNA</TD><TD class="col1">~10 ng/µL</TD><TD class="col2">1 µL</TD></TR><TR class="row2"><TD class="col0">Primer fw</TD><TD class="col1">10 µM</TD><TD class="col2">2.5</TD></TR><TR class="row3"><TD class="col0">Primer rv</TD><TD class="col1">10 µM</TD><TD class="col2">2.5</TD></TR><TR class="row4"><TD class="col0">Q5 High-Fidelity Master Mix (NEB)</TD><TD class="col1">2x</TD><TD class="col2">25 µL</TD></TR><TR class="row5"><TD class="col0">ultra pure H2O</TD><TD class="col1">-</TD><TD class="col2">ad 50 µL</TD></TR></TBODY></TABLE></DIV><DIV class="wrap_column plugin_wrap" style="width:47%;"><P> Touchdown-PCR Template: pJET1.2_cgeA
                         Primer:olG16_44fw + olG16_45rw </P><P><STRONG>Cycler Program GFP-Nanobody</STRONG></P><DIV class="table sectionedit41"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0">Step</TH><TH class="col1">Temperature [°C]</TH><TH class="col2">Duration</TH><TH class="col3">Repeats</TH></TR><TR class="row1"><TD class="col0">Initial denaturation</TD><TD class="col1">98 </TD><TD class="col2"> 5 min</TD></TR><TR class="row2"><TD class="col0">Denaturation</TD><TD class="col1">98 </TD><TD class="col2">10 s</TD><TD class="col3">10X</TD></TR><TR class="row3"><TD class="col0">Annealing</TD><TD class="col1">72* (-1°C per cycle)</TD><TD class="col2">10 s</TD><TD class="col3">10X</TD></TR><TR class="row4"><TD class="col0">Elongation</TD><TD class="col1">72 </TD><TD class="col2">30 s</TD><TD class="col3">10X</TD></TR><TR class="row5"><TD class="col0">Denaturation</TD><TD class="col1">98 </TD><TD class="col2">10 s</TD><TD class="col3">20X</TD></TR><TR class="row6"><TD class="col0">Annealing</TD><TD class="col1">63</TD><TD class="col2">10 s</TD><TD class="col3">20X</TD></TR><TR class="row7"><TD class="col0">Elongation</TD><TD class="col1">72 </TD><TD class="col2">30 s</TD><TD class="col3">20X</TD></TR><TR class="row8"><TD class="col0">Final elongation</TD><TD class="col1">72 </TD><TD class="col2">5 min</TD></TR><TR class="row9"><TD class="col0">Storage</TD><TD class="col1">8</TD><TD class="col2"> - </TD></TR></TBODY></TABLE></DIV><P> Template: pET303_aGFPnano
                         Primer:olG30_fw/olG31_rw
                        </P><P><STRONG>Cycler Program CgeA</STRONG></P><DIV class="table sectionedit42"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0">Step</TH><TH class="col1">Temperature [°C]</TH><TH class="col2">Duration</TH><TH class="col3">Repeats</TH></TR><TR class="row1"><TD class="col0">Initial denaturation</TD><TD class="col1">98 </TD><TD class="col2"> 5 min</TD></TR><TR class="row2"><TD class="col0">Denaturation</TD><TD class="col1">98 </TD><TD class="col2">10 s</TD><TD class="col3">10X</TD></TR><TR class="row3"><TD class="col0">Annealing</TD><TD class="col1">72* (-1°C per cycle) </TD><TD class="col2">10 s</TD><TD class="col3">10X</TD></TR><TR class="row4"><TD class="col0">Elongation</TD><TD class="col1">72</TD><TD class="col2">30 s</TD><TD class="col3">10X</TD></TR><TR class="row5"><TD class="col0">Denaturation</TD><TD class="col1">98 </TD><TD class="col2">10 s</TD><TD class="col3">20X</TD></TR><TR class="row6"><TD class="col0">Annealing</TD><TD class="col1">62</TD><TD class="col2">10 s</TD><TD class="col3">20X</TD></TR><TR class="row7"><TD class="col0">Elongation</TD><TD class="col1">72 </TD><TD class="col2">30 s</TD><TD class="col3">20X</TD></TR><TR class="row8"><TD class="col0">Final elongation</TD><TD class="col1">72 </TD><TD class="col2">5 min</TD></TR><TR class="row9"><TD class="col0">Storage</TD><TD class="col1">8</TD><TD class="col2"> - </TD></TR></TBODY></TABLE></DIV></DIV><DIV class="wrap_column plugin_wrap" style="width:47%;"><P> After amplification the samples were analyzed by agarose gel electrophoresis. </P><P><STRONG>Gel GFP-Nanobody/CgeA</STRONG></P></DIV></DIV></DIV></DIV><DIV class="color8"><DIV class="para_20"><H3 class="sectionedit43"><A name="section240716" id="section240716">24.07.16</A></H3><DIV class="level3"><P><STRONG>1) Gelextraction</STRONG> Gel Extraction of pET303-aGFPnanobody oIG16_30_fw/oIG16_031_rw PCR product.
                     37.7ng/µl (18µl) stored at -20 °C
                    </P><P><STRONG>2) MiniPrep</STRONG> Plasmid preparation of inoculated E.coli DH5alpha transformed with Mini prep of pBS1C3-RFP culture#2 using the QiaQuick MiniPrep kit according to the instructions of the manufacturer. Elution with 30 µL of ultra pure water.
                     148.6ng/µl (28µl)
                     placed in freezer -20 </P><P> Plasmid preparation of inoculated E.coli DH5alpha transformed with pGEX6P1, pRP261, pET303-aGFPnano_TEV_10His using the QiaQuick miniprep kit according to the instructions of the manufacturer. The plasmids were eluted in 30 µL of ultra pure water. The DNA concentration was determined by NanoDrop:
                    </P><DIV class="table sectionedit44"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0">Sample</TH><TH class="col1">concentration[ng/µL]</TH></TR><TR class="row1"><TD class="col0">pRP261 #1</TD><TD class="col1">248.9</TD></TR><TR class="row2"><TD class="col0">pRP261 #2</TD><TD class="col1">175.0</TD></TR><TR class="row3"><TD class="col0">pRP261 #3</TD><TD class="col1">235.9</TD></TR><TR class="row4"><TD class="col0">pRP261 #4</TD><TD class="col1">221.0</TD></TR><TR class="row5"><TD class="col0">pGEX-6P1 #1</TD><TD class="col1">198.7</TD></TR><TR class="row6"><TD class="col0">pGEX-6P1 #2</TD><TD class="col1">177.3</TD></TR><TR class="row7"><TD class="col0">pGEX-6P1 #3</TD><TD class="col1">149.1</TD></TR><TR class="row8"><TD class="col0">pGEX-6P1 #4</TD><TD class="col1">131.0</TD></TR><TR class="row9"><TD class="col0">pET303-aGFPnano_TEV_10His #1</TD><TD class="col1">167.4</TD></TR><TR class="row10"><TD class="col0">pET303-aGFPnano_TEV_10His #2</TD><TD class="col1">151.1</TD></TR><TR class="row11"><TD class="col0">pET303-aGFPnano_TEV_10His #3</TD><TD class="col1">161.7</TD></TR><TR class="row12"><TD class="col0">pET303-aGFPnano_TEV_10His #4</TD><TD class="col1">165.0</TD></TR></TBODY></TABLE></DIV><P><STRONG>2) Testdigestion</STRONG> Verification of the plasmid was performed by test digestion with EcoRI and PstI.
                     Reaction conditions:
                    </P><DIV class="table sectionedit45"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0">Component</TH><TH class="col1">concentration</TH><TH class="col2">volume [µL]</TH></TR><TR class="row1"><TD class="col0">DNA</TD><TD class="col1">131-248ng/µL</TD><TD class="col2">3 </TD></TR><TR class="row2"><TD class="col0">PstI</TD><TD class="col1">20u/µL</TD><TD class="col2">0.1</TD></TR><TR class="row3"><TD class="col0">EcoRI</TD><TD class="col1">20u/µL</TD><TD class="col2">0.1</TD></TR><TR class="row4"><TD class="col0">NEBuffer 2.1</TD><TD class="col1">10X</TD><TD class="col2">2</TD></TR><TR class="row5"><TD class="col0">ultra pure water</TD><TD class="col1">-</TD><TD class="col2">ad 20 µL</TD></TR></TBODY></TABLE></DIV><P> The samples were incubated at 37 °C for 1h and analyzed by gel electrophoresis. </P></DIV><H3 class="sectionedit46"><A name="section250716" id="section250716">25.07.16</A></H3><DIV class="level3"><P><STRONG>1) MiniPrep</STRONG> Plasmids of pSB1C3 were prepared. DNA concentration was determined by Nanodrop:
                     Culture #1 127.5ng/µl
                     Culture #2 88.4ng/µl
                     Culture #3 116.8ng/µl
                     Culture #4 126.1ng/µl
                    </P><P><STRONG>2) Testdigestion</STRONG> pSB1C3 was treated with </P><P><STRONG>Digestion mixture:</STRONG></P><DIV class="table sectionedit47"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0">Component</TH><TH class="col1">concentration</TH><TH class="col2">volume [µL]</TH></TR><TR class="row1"><TD class="col0">DNA</TD><TD class="col1">500ng/µL</TD><TD class="col2">5-6</TD></TR><TR class="row2"><TD class="col0">PstI</TD><TD class="col1">20u/µL</TD><TD class="col2">0.1</TD></TR><TR class="row3"><TD class="col0">EcoRI-HF</TD><TD class="col1">20u/µL</TD><TD class="col2">0.1</TD></TR><TR class="row4"><TD class="col0">NEBuffer 2.1</TD><TD class="col1">10X</TD><TD class="col2">2</TD></TR><TR class="row5"><TD class="col0">ultra pure water</TD><TD class="col1">-</TD><TD class="col2">ad 20 µL</TD></TR></TBODY></TABLE></DIV><P><STRONG>Agarose Gel of pSB1C3 digestion:</STRONG></P><P><STRONG>3) Inoculation</STRONG> Culture #3 was re-inoculated in chloramphenicol-medium. </P></DIV><H3 class="sectionedit48"><A name="section260716" id="section260716">26.07.16</A></H3><DIV class="level3"><P><STRONG>1) MiniPrep</STRONG></P><P> With culture #3 from the day before a standard mini prep was performed.
                     cultur #3 169,4ng/µl (28µl)
                     → placed in the -20 freezer </P><P><STRONG>2) Colony PCR</STRONG> Colony PCR of CotG and CotB from B. subtilis 168 genome. PCR for the amplication of the cotG and cgeB gene was performed using the following oligos:
                     cotG: oIG16_009+010 Annealing temperature: 62 °C
                     cotB: oIG16_017+018 Annealing temperature: 57 °C
                     Template DNA preparation: 1 µL of a o/n culture with B.subtilis W168 cells were diluted with EB-Buffer (Qiagen, 1:10) and incubated at 100°C for 5 min for lysis. 1 µL of the lysed cells was used as template for amplificaiton of the genes.
                    </P><P><STRONG>Reaction conditions</STRONG></P><DIV class="table sectionedit49"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0">component</TH><TH class="col1">concentration</TH><TH class="col2">volume [µL]</TH></TR><TR class="row1"><TD class="col0">Q5 HiFi MasterMix</TD><TD class="col1">2x</TD><TD class="col2">25</TD></TR><TR class="row2"><TD class="col0">Primer fw</TD><TD class="col1">10 µM</TD><TD class="col2">2.5</TD></TR><TR class="row3"><TD class="col0">Primer rv</TD><TD class="col1">10 µM</TD><TD class="col2">2.5</TD></TR><TR class="row4"><TD class="col0">lysed cells</TD><TD class="col1">-</TD><TD class="col2">1</TD></TR><TR class="row5"><TD class="col0">ultra pure water</TD><TD class="col1">-</TD><TD class="col2">ad 50 µL</TD></TR></TBODY></TABLE></DIV><P> Appropriate controls w/o template DNA were included. </P><P> The PCR was analyzed by agarose gel electrophoresis.
                     The bands corresponding to the size of the cotG and cotB were excised and the DNA was extracted using the QiaQuick Gel extraction kit according to the instructions of the manufacturer. The DNA was eluted in 30 µL of ultra pure water.
                    </P><P><STRONG>2) Gel extraction</STRONG> The concentration of the extracted DNA was determined by a Nanodrop. </P><DIV class="table sectionedit50"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0">sample</TH><TH class="col1">concentration</TH></TR><TR class="row1"><TD class="col0">CotB</TD><TD class="col1">50.2 ng/µL</TD></TR><TR class="row2"><TD class="col0">CotG</TD><TD class="col1">42.1 ng/µL</TD></TR></TBODY></TABLE></DIV><P><STRONG>3) Subcloning</STRONG> The amplified genes were subcloned into the pJET1.2 vector according to the protocol of the manufacturer, transformed into chemically competent E.coli DH5alpha, spread on a LB-agar plate supplemented with ampicilin and incubated o/n at 37 °C.
                    </P><P><STRONG>4) Inoculation</STRONG> The E.coli liquid cultures containing the plasmids pRP261, pGEX6P1 and pET303_aGFPnano_TEV_10His were reinoculated in 5 mL LB medium supplemented with ampicilin and incubated o/n at 37 °C and 250 rpm. </P></DIV></DIV></DIV><DIV class="color1"><DIV class="para_20"><H3 class="sectionedit51"><A name="section270716" id="section270716">27.07.16</A></H3><DIV class="level3"><P><STRONG>1) Inoculation</STRONG> 4 colonies of plated E.coli DH5alpha with the pJET1.2-cotB and pJET1.2-cotG plasmids were picked and inoculated in 5 mL of LB medium supplemented with ampicilin and incbated o/n at 37 °C and 250 rpm. <STRONG>2) Sequencing</STRONG> Sequencing of plasmids sent by Julia Bartels: </P><DIV class="table sectionedit52"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0">Plasmid</TH><TH class="col1">colony#</TH><TH class="col2">Primer:oIG16_</TH></TR><TR class="row1"><TD class="col0">pBS1C-[RFP]</TD><TD class="col1">#2</TD><TD class="col2">025</TD></TR><TR class="row2"><TD class="col2">026</TD></TR><TR class="row3"><TD class="col2">028</TD></TR><TR class="row4"><TD class="col2">029</TD></TR><TR class="row5"><TD class="col2">032</TD></TR><TR class="row6"><TD class="col0">pBS2E-[RFP]</TD><TD class="col1">#5</TD><TD class="col2">025</TD></TR><TR class="row7"><TD class="col0">pBS4S-[RFP]</TD><TD class="col1">#5</TD><TD class="col2">025</TD></TR><TR class="row8"><TD class="col2">26</TD></TR><TR class="row9"><TD class="col2">27</TD></TR><TR class="row10"><TD class="col0">SporoVector-[RFP]</TD><TD class="col1">#1</TD><TD class="col2">025</TD></TR><TR class="row11"><TD class="col2">026</TD></TR><TR class="row12"><TD class="col2">027</TD></TR></TBODY></TABLE></DIV></DIV><H3 class="sectionedit53"><A name="section280716" id="section280716">28.07.16</A></H3><DIV class="level3"><P><STRONG>1) Mini-prep:</STRONG></P><P> A standard Qiagen Mini-Prep was performed with the following cultures:
                     pJET1.2-cotB cultures #2/3/4, pJET1.2-cotG cultures #1/2/3/4, pGEX6P1 culture #1 (GST 2), pRP261 culture #1 (GST 1), pET303_aGFPnano_TEV_10His culture #1. </P><P> The DNA concentrations were measured with Nanodrop:
                     pJET1.2-cotB cultures #2=450,9(ng/µl)
                     pJET1.2-cotB cultures #3=820,3(ng/µl)
                     pJET1.2-cotB cultures #4=621,2(ng/µl)
                     pJET1.2-cotG cultures #1=696,5(ng/µl)
                     pJET1.2-cotG cultures #2=149,6(ng/µl)
                     pJET1.2-cotG cultures #3=614,8(ng/µl)
                     pJET1.2-cotG cultures #4=145,2(ng/µl)
                     pGEX6P1 culture #1=243,1(ng/µl)
                     pRP261 culture #1=308,2(ng/µl)
                     pET303_aGFPnano_TEV_10His culture #1=91,1(ng/µl)
                    </P><P> µl taken for Test Digestion:
                     pJET1.2-cotB cultures #2=1(µl)
                     pJET1.2-cotB cultures #3=1(µl)
                     pJET1.2-cotB cultures #4=1(µl)
                     pJET1.2-cotG cultures #1=1(µl)
                     pJET1.2-cotG cultures #2=4(µl)
                     pJET1.2-cotG cultures #3=1(µl)
                     pJET1.2-cotG cultures #4=4(µl)
                     pGEX6P1 culture #2=(µl)
                     pRP261 culture #2=(µl)
                     pET303_aGFPnano_TEV_10His culture #1=6(µl)
                    </P><P><STRONG>2) Test-digestion </STRONG></P><P><STRONG>Digestion mixture 1:</STRONG></P><DIV class="table sectionedit54"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0">Component</TH><TH class="col1">concentration</TH><TH class="col2">volume [µL]</TH></TR><TR class="row1"><TD class="col0">DNA</TD><TD class="col1">500ng/µL</TD><TD class="col2">5-6</TD></TR><TR class="row2"><TD class="col0">XbaI</TD><TD class="col1">20u/µL</TD><TD class="col2">0.1</TD></TR><TR class="row3"><TD class="col0">XhoI</TD><TD class="col1">20u/µL</TD><TD class="col2">0.1</TD></TR><TR class="row4"><TD class="col0">NEBuffer 2.1</TD><TD class="col1">10X</TD><TD class="col2">2</TD></TR><TR class="row5"><TD class="col0">ultra pure water</TD><TD class="col1">-</TD><TD class="col2">ad 20 µL</TD></TR></TBODY></TABLE></DIV><P><STRONG>Digestion mixture 2:</STRONG></P><DIV class="table sectionedit55"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0">Component</TH><TH class="col1">concentration</TH><TH class="col2">volume [µL]</TH></TR><TR class="row1"><TD class="col0">DNA</TD><TD class="col1">500ng/µL</TD><TD class="col2">5-6</TD></TR><TR class="row2"><TD class="col0">PstI</TD><TD class="col1">20u/µL</TD><TD class="col2">0.1</TD></TR><TR class="row3"><TD class="col0">EcoRI-HF</TD><TD class="col1">20u/µL</TD><TD class="col2">0.1</TD></TR><TR class="row4"><TD class="col0">NEBuffer 2.1</TD><TD class="col1">10X</TD><TD class="col2">2</TD></TR><TR class="row5"><TD class="col0">ultra pure water</TD><TD class="col1">-</TD><TD class="col2">ad 20 µL</TD></TR></TBODY></TABLE></DIV><P><STRONG>1% Agarose Gel of Test Digestion</STRONG></P></DIV><H3 class="sectionedit56"><A name="section290716" id="section290716">29.07.16</A></H3><DIV class="level3"><P><STRONG>1) Mini-prep:</STRONG></P><P> A standard Qiagen Mini-Prep was performed with the following cultures:
                     pJET1.2-cotB cultures #2/3/4, pJET1.2-cotG cultures #1/2/3/4, pET303_aGFPnano_TEV_10His culture #1. </P><P> The DNA concentrations were measured with Nanodrop:
                     pJET1.2-cotB cultures #2=225,5(ng/µl)
                     pJET1.2-cotB cultures #3=289,5(ng/µl)
                     pJET1.2-cotB cultures #4=229,1(ng/µl)
                     pJET1.2-cotG cultures #1=104,7(ng/µl)
                     pJET1.2-cotG cultures #2=63,2(ng/µl)
                     pJET1.2-cotG cultures #3=97,6(ng/µl)
                     pJET1.2-cotG cultures #4=192,1(ng/µl)
                     pET303_aGFPnano_TEV_10His culture #1=135,1(ng/µl)
                    </P><P> µl taken for Test Digestion:
                     pJET1.2-cotB cultures #2=2(µl)
                     pJET1.2-cotB cultures #3=2(µl)
                     pJET1.2-cotB cultures #4=2(µl)
                     pJET1.2-cotG cultures #1=5(µl)
                     pJET1.2-cotG cultures #2=6(µl)
                     pJET1.2-cotG cultures #3=5(µl)
                     pJET1.2-cotG cultures #4=3(µl)
                     pET303_aGFPnano_TEV_10His culture #1=4(µl)
                    </P><P><STRONG>2) Test-digestion</STRONG></P><P><STRONG>Digestion mixture:</STRONG></P><DIV class="table sectionedit57"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0">Component</TH><TH class="col1">concentration</TH><TH class="col2">volume [µL]</TH></TR><TR class="row1"><TD class="col0">DNA</TD><TD class="col1">500ng/µL</TD><TD class="col2">5-6</TD></TR><TR class="row2"><TD class="col0">PstI</TD><TD class="col1">20u/µL</TD><TD class="col2">0.1</TD></TR><TR class="row3"><TD class="col0">EcoRI-HF</TD><TD class="col1">20u/µL</TD><TD class="col2">0.1</TD></TR><TR class="row4"><TD class="col0">NEBuffer Smart-Cut</TD><TD class="col1">10X</TD><TD class="col2">2</TD></TR><TR class="row5"><TD class="col0">ultra pure water</TD><TD class="col1">-</TD><TD class="col2">ad 20 µL</TD></TR></TBODY></TABLE></DIV><P><STRONG>1% Agerose Gel of Test Digestion</STRONG></P><P> After testdigestion pJET1.2-cotB#3 and pJET1.2-cotG#1 were sent to sequencing. </P></DIV></DIV></DIV><DIV class="color2"><DIV class="para_20"><H3 class="sectionedit58"><A name="section300716" id="section300716">30.07.16</A></H3><DIV class="level3"><P><STRONG>1) Extension PCR</STRONG> Extension PCR of of cgeA and aGFP-Nanobody to generate overhangs for Gibson cloning.
                    </P><DIV class="table sectionedit59"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0">Template</TH><TH class="col1">Primer</TH><TH class="col2">Annealing Temp. [°C]</TH></TR><TR class="row1"><TD class="col0">pJET1.2_cgeA #2</TD><TD class="col1">oIG16_044 + 045</TD><TD class="col2">62</TD></TR><TR class="row2"><TD class="col1">oIG16_050 + 035</TD><TD class="col2">62</TD></TR><TR class="row3"><TD class="col0">pET303_aGFPnano_TEV_10His</TD><TD class="col1">oIG16_030 + 031</TD><TD class="col2">67</TD></TR><TR class="row4"><TD class="col1">oIG16_051 + 042</TD><TD class="col2">67</TD></TR></TBODY></TABLE></DIV><P> *Reaction conditions* </P><DIV class="table sectionedit60"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0">Component</TH><TH class="col1">concentration</TH><TH class="col2">Volume[µL]</TH></TR><TR class="row1"><TD class="col0">Primer fw</TD><TD class="col1">10 µM</TD><TD class="col2">2.5</TD></TR><TR class="row2"><TD class="col0">Primer rv</TD><TD class="col1">10 µM</TD><TD class="col2">2.5</TD></TR><TR class="row3"><TD class="col0">Q5 HiFi MasterMix</TD><TD class="col1">2x</TD><TD class="col2">25</TD></TR><TR class="row4"><TD class="col0">ultra pure H2O</TD><TD class="col1">-</TD><TD class="col2">19</TD></TR></TBODY></TABLE></DIV><P> *PCR program*
                     Touchdown PCR </P><DIV class="table sectionedit61"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0">Step</TH><TH class="col1">Temperature [°C]</TH><TH class="col2">Duration</TH><TH class="col3">Repeats</TH></TR><TR class="row1"><TD class="col0">Initial denaturation</TD><TD class="col1">98 </TD><TD class="col2"> 5 min</TD></TR><TR class="row2"><TD class="col0">Denaturation</TD><TD class="col1">98 </TD><TD class="col2">10 s</TD><TD class="col3">10X</TD></TR><TR class="row3"><TD class="col0">Annealing</TD><TD class="col1">Annealing Temp + 10°C (-1°C per cycle) </TD><TD class="col2">10 s</TD><TD class="col3">10X</TD></TR><TR class="row4"><TD class="col0">Elongation</TD><TD class="col1">72</TD><TD class="col2">30 s</TD><TD class="col3">10X</TD></TR><TR class="row5"><TD class="col0">Denaturation</TD><TD class="col1">98 </TD><TD class="col2">10 s</TD><TD class="col3">20X</TD></TR><TR class="row6"><TD class="col0">Annealing</TD><TD class="col1">Annealing temp.</TD><TD class="col2">10 s</TD><TD class="col3">20X</TD></TR><TR class="row7"><TD class="col0">Elongation</TD><TD class="col1">72 </TD><TD class="col2">30 s</TD><TD class="col3">20X</TD></TR><TR class="row8"><TD class="col0">Final elongation</TD><TD class="col1">72 </TD><TD class="col2">5 min</TD></TR><TR class="row9"><TD class="col0">Storage</TD><TD class="col1">8</TD><TD class="col2"> - </TD></TR></TBODY></TABLE></DIV><P> After PCR the samples were analyzed by agarose gel electrophoresis (1 % agarose TAE gel). </P><P><STRONG>2) Gel extraction</STRONG> The bands corresponding to the appropriate sizes were extracted using the QiaQuick gel extraction kit and the DNA was eluted in 30 µL of ultra pure water. The concentration was photometrically determined by a Nanodrop. </P><DIV class="table sectionedit62"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0">Sample</TH><TH class="col1">Concentration [ng/µL]</TH></TR><TR class="row1"><TD class="col0">cgeA_oIG16_044+045</TD><TD class="col1">85.2</TD></TR><TR class="row2"><TD class="col0">cgeA_oIG16_50+35</TD><TD class="col1">62.4</TD></TR><TR class="row3"><TD class="col0">aGFPnano_oIG16_31+30</TD><TD class="col1">93.7</TD></TR><TR class="row4"><TD class="col0">aGFPnano_oIG16_51+42</TD><TD class="col1">104</TD></TR></TBODY></TABLE></DIV></DIV><H3 class="sectionedit63"><A name="section310716" id="section310716">31.07.16</A></H3><DIV class="level3"><P><STRONG>1) Restriction digestion </STRONG></P><P> Linearization of pSB1C3 for Gibson Cloning: </P><P><STRONG>Digestion mixture</STRONG></P><DIV class="table sectionedit64"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0">Component</TH><TH class="col1">concentration</TH><TH class="col2">volume [µL]</TH></TR><TR class="row1"><TD class="col0">DNA</TD><TD class="col1">100ng/µL</TD><TD class="col2">5-6</TD></TR><TR class="row2"><TD class="col0">XbaI</TD><TD class="col1">20u/µL</TD><TD class="col2">0.5</TD></TR><TR class="row3"><TD class="col0">XhoI</TD><TD class="col1">20u/µL</TD><TD class="col2">0.5</TD></TR><TR class="row4"><TD class="col0">NEBuffer 2.1</TD><TD class="col1">10X</TD><TD class="col2">10</TD></TR><TR class="row5"><TD class="col0">ultra pure water</TD><TD class="col1">-</TD><TD class="col2">ad 50 µL</TD></TR></TBODY></TABLE></DIV><P><STRONG>2) Agarose Gel of pSB1c3</STRONG></P><P> → followed by gel extraction for gibson assembly. </P></DIV><H3 class="sectionedit65"><A name="section010816" id="section010816">01.08.16</A></H3><DIV class="level3"><P><STRONG>1) NEBuilder® HiFi DNA Assembly Cloning</STRONG>: </P><P> Assembly of extensionPCR products into pSB1C3. </P><DIV class="table sectionedit66"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0">No.</TH><TH class="col1">amplified fragment1</TH><TH class="col2"> amplified fragment 2</TH><TH class="col3">digested Backbone</TH><TH class="col4" colspan="2">Resulting Plasmid</TH></TR><TR class="row1"><TD class="col0">1</TD><TD class="col1"> cgeA oIG16_50+35</TD><TD class="col2">aGFPnano_oIG16_51+42 </TD><TD class="col3"> XbaI - pSB1C3 - SpeI</TD><TD class="col4"> pSB1C3_cgeA_aHelix_HA_aGFPnano pIG16_038</TD></TR><TR class="row2"><TD class="col0">2</TD><TD class="col1"> aGFPnano oIG16_30+47 </TD><TD class="col2 leftalign">cgeA oIG16_46+45 </TD><TD class="col3">EcoRI - pSB1C3 - SpeI</TD><TD class="col4"> pSB1C3_aGFPnano_HA_aHelix_cgeA pIG16_039</TD></TR></TBODY></TABLE></DIV><P><STRONG>Reaction conditions</STRONG> DNA: 1.5 µL
                     Fragment1: 25 ng
                     Fragment2: 15 ng
                     linearized pSB1C3: 25 ng
                     2X HiFi MasterMix: 5 µL
                     ultra pure water: 3.5 µL
                    </P><P> The reaction was incubated at 50 °C for 1 hour. 2 µL of the reaction mixture was transformed into chemically competent E.coli DH5alpha provided by the NEBuilder HiFi kit according to their protocol, plated on LB-agar plates containing chloramphenicol and incubated o/n at 37 °C.
                     The control reaction provided by the kit was performed according to the instructions of the manufacturer. </P><P><STRONG>2) Digestion of pBS1C</STRONG><STRONG>Digestion mixture:</STRONG></P><DIV class="table sectionedit67"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0">Component</TH><TH class="col1">concentration</TH><TH class="col2">volume [µL]</TH></TR><TR class="row1"><TD class="col0">DNA</TD><TD class="col1">148ng/µL</TD><TD class="col2">16</TD></TR><TR class="row2"><TD class="col0">XhoI</TD><TD class="col1">20u/µL</TD><TD class="col2">0.5</TD></TR><TR class="row3"><TD class="col0">NEBuffer CutSmart</TD><TD class="col1">10X</TD><TD class="col2">10</TD></TR><TR class="row4"><TD class="col0">ultra pure water</TD><TD class="col1">-</TD><TD class="col2">ad 50 µL</TD></TR></TBODY></TABLE></DIV><P><STRONG>Agarose gel of pBS1c:</STRONG></P><P> → followed by a Gel extraction for Transformation of competent B.subtilis W168. </P></DIV></DIV></DIV><DIV class="color3"><DIV class="para_20"><H3 class="sectionedit68"><A name="section020816" id="section020816">02.08.16</A></H3><DIV class="level3"><P><STRONG>1) Inoculations</STRONG> Inoculation of 2 colonies from each plate with transformed E.coli containing pSB1C3_cgeA_aHelix_HA-Tag_aGFPnano and pSB1C3_aGFPnano_aHelix_HA-Tag_cgeA. Incubation o/n at 37 °C and 250 rpm.
                     Remaining E.coli from the transformation (see previous day) were plated on LB-agar plates supplemented with cml and incubated o/n at 37 °C. </P><P><STRONG>2) Digestion of pBS1C (For transformation of B.subtilis)</STRONG></P><P><STRONG>Digestion mixture:</STRONG></P><DIV class="table sectionedit69"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0">Component</TH><TH class="col1">concentration</TH><TH class="col2">volume [µL]</TH></TR><TR class="row1"><TD class="col0">DNA</TD><TD class="col1">700ng/µL</TD><TD class="col2">10</TD></TR><TR class="row2"><TD class="col0">XhoI</TD><TD class="col1">20u/µL</TD><TD class="col2">1.5</TD></TR><TR class="row3"><TD class="col0">NEBuffer CutSmart</TD><TD class="col1">10X</TD><TD class="col2">10</TD></TR><TR class="row4"><TD class="col0">ultra pure water</TD><TD class="col1">-</TD><TD class="col2">ad 50 µL</TD></TR></TBODY></TABLE></DIV><P> → PCR purification was peformed </P></DIV><H3 class="sectionedit70"><A name="section030816" id="section030816">03.08.16</A></H3><DIV class="level3"><P><STRONG>1) MiniPrep</STRONG> Plasmid preparation of the inoculated E.coli from the previous day using the QiaQuick Plasmid preparation kit according to the protocol of the manufacturer and eluted in 30 µL of ultra pure water. The DNA concentration was photometrically determined by a NanoDrop: </P><DIV class="table sectionedit71"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0">sample</TH><TH class="col1">concentration [ng/µL]</TH></TR><TR class="row1"><TD class="col0">pSB1C3_CgeA_alphaHelix_HA-Tag_aGFPnano #1</TD><TD class="col1">69.1</TD></TR><TR class="row2"><TD class="col0">pSB1C3_CgeA_alphaHelix_HA-Tag_aGFPnano #2</TD><TD class="col1">126.7</TD></TR><TR class="row3"><TD class="col0">pSB1C3_aGFPnano_alphaHelix_HA-Tag_cgeA #1</TD><TD class="col1">120.4</TD></TR><TR class="row4"><TD class="col0">pSB1C3_aGFPnano_alphaHelix_HA-Tag_cgeA #2</TD><TD class="col1">182.1</TD></TR></TBODY></TABLE></DIV><P><STRONG>2) Testdigestion</STRONG> 500 ng of DNA was treated with 5 unit of XbaI and PstI in 1x NEBuffer 2.1 in a total reaction volume of 20 µL. Incubation for 1hour at 37 °C. </P><P><STRONG>2) Inoculation</STRONG><STRONG>I</STRONG> Inoculation of E.coli DH5alpha containing an mCherry and GFP plasmid (provided by AG Weber) in 5 mL LB containig (Three colonies per plate were picked)
                    <STRONG>II</STRONG> E.coli DH5alpha containing plasmids sent from Poland (by Dr. Krystof Hinc) for the construction of fusion proteins for spore surface display arrived:
                     1)pCotG-N
                     2)pCotG-C
                     3)pCotB-N
                     4)pCotB-C
                     5)pCgeA-C
        6)pCotZ-C
                     The E.coli were spread on LB-Agar plates containing ampicilin and incubated o/n at 37 °C.
                    <STRONG>III</STRONG>* 6 colonies per plate of E.coli DH5alpha containing pSB1C3_CgeA_alphaHelix_HA-Tag_aGFPnano or pSB1C3_aGFPnano_alphaHelix_HA-Tag_cgeA were inoculated in 5 mL LB-medium supplemented with chloramphenicol and incubated at 37 °C and 250rpm. </P></DIV><H3 class="sectionedit72"><A name="section040816" id="section040816">04.08.16</A></H3><DIV class="level3"><P><STRONG>1) MiniPrep pIG16_038 + pIG16_039, GFP + mCherry</STRONG> Plasmid preparation of inoculated E.coli DH5alpha transformed with plasmids containing GFP and mCherry (from AG Weber) using the QiaQuick MiniPrep kit according to the instructions of the manufacturer. Elution with 30 µL of ultra pure water. The concentration was determined by Nanodrop. </P><DIV class="table sectionedit73"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0">No</TH><TH class="col1">Sample ID</TH><TH class="col2">Nucleic Acid Conc.</TH></TR><TR class="row1"><TD class="col0">1</TD><TD class="col1">h2o</TD><TD class="col2">0,9</TD><TD class="col3">ng/µl</TD></TR><TR class="row2"><TD class="col0">2</TD><TD class="col1">pIG16_038 #1</TD><TD class="col2">106,9ng/µl</TD></TR><TR class="row3"><TD class="col0">3</TD><TD class="col1">pIG16_038 #2</TD><TD class="col2">92,4ng/µl</TD></TR><TR class="row4"><TD class="col0">4</TD><TD class="col1">pIG16_038 #3</TD><TD class="col2">77,3ng/µl</TD></TR><TR class="row5"><TD class="col0">5</TD><TD class="col1">pIG16_038 #4</TD><TD class="col2">64,4ng/µl</TD></TR><TR class="row6"><TD class="col0">6</TD><TD class="col1">pIG16_038 #5</TD><TD class="col2">63,2ng/µl</TD></TR><TR class="row7"><TD class="col0">7</TD><TD class="col1">pIG16_038 #6</TD><TD class="col2">55,4ng/µl</TD></TR><TR class="row8"><TD class="col0">8</TD><TD class="col1">pIG16_039 #1</TD><TD class="col2">105,2ng/µl</TD></TR><TR class="row9"><TD class="col0">9</TD><TD class="col1">pIG16_039 #2</TD><TD class="col2">101,3ng/µl</TD></TR><TR class="row10"><TD class="col0">10</TD><TD class="col1">pIG16_039 #3</TD><TD class="col2">73,2ng/µl</TD></TR><TR class="row11"><TD class="col0">11</TD><TD class="col1">pIG16_039 #4</TD><TD class="col2">85,2ng/µl</TD></TR><TR class="row12"><TD class="col0">12</TD><TD class="col1">pIG16_039 #5</TD><TD class="col2">80,4ng/µl</TD></TR><TR class="row13"><TD class="col0">13</TD><TD class="col1">pIG16_039 #6</TD><TD class="col2">86,7ng/µl</TD></TR><TR class="row14"><TD class="col0">14</TD><TD class="col1">pIG16_024 GFP #1</TD><TD class="col2">93,2ng/µl</TD></TR><TR class="row15"><TD class="col0">15</TD><TD class="col1">pIG16_024 GFP #2</TD><TD class="col2">76,3ng/µl</TD></TR><TR class="row16"><TD class="col0">16</TD><TD class="col1">pIG16_024 GFP #3</TD><TD class="col2">100,9ng/µl</TD></TR><TR class="row17"><TD class="col0">17</TD><TD class="col1">pIG16_025mCherry#3</TD><TD class="col2">143,2ng/µl</TD></TR><TR class="row18"><TD class="col0">18</TD><TD class="col1">pIG16_025mCherry#2</TD><TD class="col2">175,8ng/µl</TD></TR><TR class="row19"><TD class="col0">19</TD><TD class="col1">pIG16_025mCherry#3</TD><TD class="col2">132,8ng/µl</TD></TR></TBODY></TABLE></DIV><P><STRONG>2)Inoculation</STRONG> Inoculation of the E.coli containing the plasmids sent from Poland. 4 Colonies from each plate were picked and inoculated in 5 mL of LB medium supplemented with Amp.
                    </P><P><STRONG>3)ExtensionPCRs Q5 –&gt; Gel+extraction</STRONG></P><DIV class="table sectionedit74"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0"> No. </TH><TH class="col1"> Template </TH><TH class="col2"> Primer: oIG16_# </TH><TH class="col3"> Annealing Temp[°C] </TH><TH class="col4"> Resulting construct </TH></TR><TR class="row1"><TD class="col0">1</TD><TD class="col1"> pET303_aGFPnano</TD><TD class="col2"> 030 + 047</TD><TD class="col3"> 67</TD><TD class="col4">Nano_HA_G4S</TD></TR><TR class="row2"><TD class="col0">2</TD><TD class="col1"> pET303_aGFPnano</TD><TD class="col2"> 053 + 042</TD><TD class="col3"> 67</TD><TD class="col4">G4S_HA_Nano</TD></TR><TR class="row3"><TD class="col0">3</TD><TD class="col1"> pJET_cotG</TD><TD class="col2"> 015 + 016</TD><TD class="col3"> 62</TD><TD class="col4">HA_aHelix_CotG</TD></TR><TR class="row4"><TD class="col0">4</TD><TD class="col1"> pJET_cotG</TD><TD class="col2"> 011 + 013</TD><TD class="col3"> 62</TD><TD class="col4">CotG_aHelix_HA</TD></TR><TR class="row5"><TD class="col0">5</TD><TD class="col1"> pSBBs4S_Sporovector</TD><TD class="col2"> 048 + 049</TD><TD class="col3"> 55</TD><TD class="col4">Bb-Prefix_PCotYZ_Bb-Suffix</TD></TR><TR class="row6"><TD class="col0">6</TD><TD class="col1"> pJET_cgeA</TD><TD class="col2"> 046 + 045</TD><TD class="col3"> 62</TD><TD class="col4">HA_G4S_CgeA</TD></TR><TR class="row7"><TD class="col0">7</TD><TD class="col1"> pJET_cgeA</TD><TD class="col2"> 050 + 052</TD><TD class="col3"> 62</TD><TD class="col4">CgeA_G4S_HA</TD></TR><TR class="row8"><TD class="col0">8</TD><TD class="col1"> dH2O</TD><TD class="col2"> 046 + 045</TD><TD class="col3"> 62</TD><TD class="col4">negative control</TD></TR><TR class="row9"><TD class="col0">9</TD><TD class="col1"> dH2O</TD><TD class="col2"> 015 + 016</TD><TD class="col3"> 62</TD><TD class="col4"> negative control</TD></TR><TR class="row10"><TD class="col0">10</TD><TD class="col1"> dH2O</TD><TD class="col2"> 030 + 047</TD><TD class="col3"> 67</TD><TD class="col4"> negative control</TD></TR></TBODY></TABLE></DIV><P><STRONG>Reaction conditions</STRONG></P><DIV class="table sectionedit75"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0">Component</TH><TH class="col1">Volume µL</TH></TR><TR class="row1"><TD class="col0">2XHiFi MasterMix</TD><TD class="col1">25</TD></TR><TR class="row2"><TD class="col0">Primer1</TD><TD class="col1">2.5</TD></TR><TR class="row3"><TD class="col0">Primer2</TD><TD class="col1">2.5</TD></TR><TR class="row4"><TD class="col0">template DNA</TD><TD class="col1">0.1</TD></TR><TR class="row5"><TD class="col0">ultra pure water</TD><TD class="col1">19.9</TD></TR></TBODY></TABLE></DIV><P><STRONG>PCR Program</STRONG> Touchdown PCRs corresponding to the annealing temperatures.
                    </P><P> After amplification the samples were loaded on a 1% Agarose TAE gel. </P><P> The bands corresponding to the expected fragment sizes were extracted from the gel, purified and eluted with 30 µL of ultra pure water. </P></DIV></DIV></DIV><DIV class="color4"><DIV class="para_20"><H3 class="sectionedit76"><A name="section050816" id="section050816">05.08.16</A></H3><DIV class="level3"><P><STRONG>1) Testdigestion</STRONG> pIG16_038+039 were verified by testdigestion. 500 ng of DNA was treated with XbaI and PstI and analyzed by gel electrophoresis.
                    </P><P><STRONG>2) MiniPrep</STRONG> The inoculated cultures containing the plasmids sent from poland were preparated using the QiaQuick MiniPrep kit according to the protocol. The DNA was eluted with 30 µL of ultra pure water. The concentration was determined by Nanodrop.
                    </P><P><STRONG>3) GFP purification</STRONG> Sebastian from AG Weber gave a purified GFP (1.35 mg/mL) and mCherry (2 mg/mL).
                     Stored at 4 °C, protected from light. </P></DIV><H3 class="sectionedit77"><A name="section060816" id="section060816">06.08.16</A></H3><DIV class="level3"><P><STRONG>1) ExtensionPCR</STRONG></P><DIV class="table sectionedit78"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0">template</TH><TH class="col1">Primer oIG16_</TH><TH class="col2">Annealing Temp. [°C]</TH><TH class="col3">Resulting construct</TH></TR><TR class="row1"><TD class="col0">pJet1.2_cotG</TD><TD class="col1">011 + 013</TD><TD class="col2"> 62 </TD><TD class="col3">CotG_aHelix_HA</TD></TR></TBODY></TABLE></DIV><P> After amplification the sample was loaded on a 1% Agarose TAE gel. The band corresponding to the expected size was extracted and purified.
                    </P><P><STRONG>2) 3A assembly</STRONG></P><P><STRONG>Digestion mixture PCotYZ:</STRONG> The B.subtilis spore coat promoter PCotYZ was digested from the Sporovector pIG16_017 provided by Julia Bartels from the Mascher Lab at the TU Dresden. </P><DIV class="table sectionedit79"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0">Component</TH><TH class="col1">concentration</TH><TH class="col2">volume [µL]</TH></TR><TR class="row1"><TD class="col0">DNA</TD><TD class="col1">500ng/µL</TD><TD class="col2">13.5</TD></TR><TR class="row2"><TD class="col0">EcoRI</TD><TD class="col1">20u/µL</TD><TD class="col2">0.1</TD></TR><TR class="row3"><TD class="col0">SpeI</TD><TD class="col1">20u/µL</TD><TD class="col2">0.1</TD></TR><TR class="row4"><TD class="col0">NEBuffer Cut Smart</TD><TD class="col1">10X</TD><TD class="col2">2</TD></TR><TR class="row5"><TD class="col0">ultra pure water</TD><TD class="col1">-</TD><TD class="col2">ad 20 µL</TD></TR></TBODY></TABLE></DIV><P><STRONG>Digestion mixture for pIG16_038+039:</STRONG></P><DIV class="table sectionedit80"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0">Component</TH><TH class="col1">concentration</TH><TH class="col2">volume [µL]</TH></TR><TR class="row1"><TD class="col0">DNA</TD><TD class="col1">500ng/µL</TD><TD class="col2">variable</TD></TR><TR class="row2"><TD class="col0">XbaI</TD><TD class="col1">20u/µL</TD><TD class="col2">0.1</TD></TR><TR class="row3"><TD class="col0">PstI</TD><TD class="col1">20u/µL</TD><TD class="col2">0.1</TD></TR><TR class="row4"><TD class="col0">NEBuffer 3.1</TD><TD class="col1">10X</TD><TD class="col2">2</TD></TR><TR class="row5"><TD class="col0">ultra pure water</TD><TD class="col1">-</TD><TD class="col2">ad 20 µL</TD></TR></TBODY></TABLE></DIV><P><STRONG>Digestion mixture for pBS1C:</STRONG></P><DIV class="table sectionedit81"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0">Component</TH><TH class="col1">concentration</TH><TH class="col2">volume [µL]</TH></TR><TR class="row1"><TD class="col0">DNA</TD><TD class="col1">500ng/µL</TD><TD class="col2"> variable</TD></TR><TR class="row2"><TD class="col0">EcoRI</TD><TD class="col1">20u/µL</TD><TD class="col2">0.2</TD></TR><TR class="row3"><TD class="col0">PstI</TD><TD class="col1">20u/µL</TD><TD class="col2">0.2</TD></TR><TR class="row4"><TD class="col0">NEBuffer 2.1</TD><TD class="col1">10X</TD><TD class="col2">2</TD></TR><TR class="row5"><TD class="col0">ultra pure water</TD><TD class="col1">-</TD><TD class="col2">ad 20 µL</TD></TR></TBODY></TABLE></DIV><P><STRONG>T4 Ligation</STRONG></P><P><STRONG>Ligation mixture of plG16_38/39 #1:</STRONG></P><DIV class="table sectionedit82"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0">Component</TH><TH class="col1">concentration</TH><TH class="col2">volume [µL]</TH></TR><TR class="row1"><TD class="col0">pBS1C</TD><TD class="col1">2000ng/µL</TD><TD class="col2">1</TD></TR><TR class="row2"><TD class="col0">plG_38</TD><TD class="col1">105ng/µL</TD><TD class="col2">4</TD></TR><TR class="row3"><TD class="col0">plG_39</TD><TD class="col1">104ng/µL</TD><TD class="col2">4</TD></TR><TR class="row4"><TD class="col0">PcotYZ</TD><TD class="col1">24ng/µL</TD><TD class="col2">0.2</TD></TR><TR class="row5"><TD class="col0" colspan="2">T-4 Ligase</TD><TD class="col2">1µl</TD></TR><TR class="row6"><TD class="col0">NEBuffer T-4 Ligase</TD><TD class="col1">10X</TD><TD class="col2">2</TD></TR><TR class="row7"><TD class="col0">ultra pure water</TD><TD class="col1">-</TD><TD class="col2">ad 20 µL</TD></TR></TBODY></TABLE></DIV><P> The reaction was incubated for 30min at RT and transformed into chemically competent mix and go E. coli DH5alpha
                    </P><P><STRONG>3) Gibson assembly</STRONG> Assembly of extensionPCR products into pSB1C3. </P><DIV class="table sectionedit83"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0">No.</TH><TH class="col1">amplified fragment1</TH><TH class="col2"> amplified fragment 2</TH><TH class="col3">digested Backbone</TH><TH class="col4" colspan="2">Resulting Plasmid</TH></TR><TR class="row1"><TD class="col0">1</TD><TD class="col1"> cgeA oIG16_050+052</TD><TD class="col2">aGFPnano_oIG16_53+42 </TD><TD class="col3"> XbaI - pSB1C3 - SpeI</TD><TD class="col4"> pSB1C3_cgeA_G4S_HA_aGFPnano pIG16_040</TD></TR><TR class="row2"><TD class="col0">2</TD><TD class="col1">aGFPnano oIG16_30+47</TD><TD class="col2">cotG oIG16_46+45</TD><TD class="col3">EcoRI - pSB1C3 - SpeI</TD><TD class="col4"> pSB1C3_aGFPnano_HA_G4S_cgeA pIG16_041</TD></TR><TR class="row3"><TD class="col0">3</TD><TD class="col1">aGFPnano oIG16_030+031</TD><TD class="col2">cotG oIG16_015+016</TD><TD class="col3">EcoRI - pSB1C3 - SpeI</TD><TD class="col4"> pSB1C3_aGFPnano_HA_aHelix_cotG pIG16_042</TD></TR><TR class="row4"><TD class="col0">4</TD><TD class="col1">cotG oIG16_011+013</TD><TD class="col2">aGFPnano oIG16_051+042</TD><TD class="col3">EcoRI - pSB1C3 - SpeI</TD><TD class="col4">pSB1C3_cotG_aHelix_HA_aGFPnano pIG16_043</TD></TR></TBODY></TABLE></DIV><P><STRONG>Reaction conditions</STRONG> DNA: 1.5 µL
                     Fragment1: 25 ng
                     Fragment2: 15 ng
                     linearized pSB1C3: 25 ng
                     2X HiFi MasterMix: 5 µL
                     ultra pure water: 3.5 µL
                     The reaction was incubated at 50 °C for 1 hour. 2µL of the reaction mix were transformed into chemically competent E.coli DH5alpha and spread on agarose LB plates supplemented with chloramphenicol. Incubation o/n at 37 °C.
                    </P><P><STRONG>4) Testdigestion of GFP </STRONG> For verification 500 ng of the plasmid was treated with XbaI, XhoI, BamHI in 1X NEBuffer 3.1 for 90 min at 37°C. The fragments were analyzed by agarose gel electrophoresis.
                     amp;media=labor:2016_08_06_gfp_testdigestion_invert_label.png&quot; class=&quot;mediacenter&quot; alt=&quot;&quot; width=&quot;300&quot; /&gt; </P><P><STRONG>5) Test Digestion Plasmids provided by Krystof Hinc</STRONG> From university Gdansk. A set of vectors for surface display in B.subtilis. </P><P><STRONG>Digestion mixture cgeA-C/cotZ-C:</STRONG></P><DIV class="table sectionedit84"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0">Component</TH><TH class="col1">concentration</TH><TH class="col2">volume [µL]</TH></TR><TR class="row1"><TD class="col0">DNA</TD><TD class="col1">500ng/µL</TD><TD class="col2">cgeA: #1/2/4=5 #3=8 cotZ: #1/2/3/4=7</TD></TR><TR class="row2"><TD class="col0">XhoI</TD><TD class="col1">20u/µL</TD><TD class="col2">0.1</TD></TR><TR class="row3"><TD class="col0">PstI</TD><TD class="col1">20u/µL</TD><TD class="col2">0.1</TD></TR><TR class="row4"><TD class="col0">NEBuffer 3.1</TD><TD class="col1">10X</TD><TD class="col2">2</TD></TR><TR class="row5"><TD class="col0">ultra pure water</TD><TD class="col1">-</TD><TD class="col2">ad 20 µL</TD></TR></TBODY></TABLE></DIV><P><STRONG>Digestion mixture cotB-C/N:</STRONG></P><DIV class="table sectionedit85"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0">Component</TH><TH class="col1">concentration</TH><TH class="col2">volume [µL]</TH></TR><TR class="row1"><TD class="col0">DNA</TD><TD class="col1">500ng/µL</TD><TD class="col2">cotB-C: #1/2=5 #3/4=6 cotB-N #1=5 #2=6 #3/4=7</TD></TR><TR class="row2"><TD class="col0">PstI</TD><TD class="col1">20u/µL</TD><TD class="col2">0.1</TD></TR><TR class="row3"><TD class="col0">NEBuffer 3.1</TD><TD class="col1">10X</TD><TD class="col2">2</TD></TR><TR class="row4"><TD class="col0">ultra pure water</TD><TD class="col1">-</TD><TD class="col2">ad 20 µL</TD></TR></TBODY></TABLE></DIV><P><STRONG>Digestion mixture cotG-C:</STRONG></P><DIV class="table sectionedit86"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0">Component</TH><TH class="col1">concentration</TH><TH class="col2">volume [µL]</TH></TR><TR class="row1"><TD class="col0">DNA</TD><TD class="col1">500ng/µL</TD><TD class="col2">#1/2/3/4=2</TD></TR><TR class="row2"><TD class="col0">BamHI</TD><TD class="col1">20u/µL</TD><TD class="col2">0.1</TD></TR><TR class="row3"><TD class="col0">NEBuffer 3.1</TD><TD class="col1">10X</TD><TD class="col2">2</TD></TR><TR class="row4"><TD class="col0">ultra pure water</TD><TD class="col1">-</TD><TD class="col2">ad 20 µL</TD></TR></TBODY></TABLE></DIV><P><STRONG>Digestion mixture cotG-N:</STRONG></P><DIV class="table sectionedit87"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0">Component</TH><TH class="col1">concentration</TH><TH class="col2">volume [µL]</TH></TR><TR class="row1"><TD class="col0">DNA</TD><TD class="col1">500ng/µL</TD><TD class="col2">#1/2/3=2 #4=6</TD></TR><TR class="row2"><TD class="col0">XbhI</TD><TD class="col1">20u/µL</TD><TD class="col2">0.1</TD></TR><TR class="row3"><TD class="col0">XhoI</TD><TD class="col1">20u/µL</TD><TD class="col2">0.1</TD></TR><TR class="row4"><TD class="col0">NEBuffer CutSmart</TD><TD class="col1">10X</TD><TD class="col2">2</TD></TR><TR class="row5"><TD class="col0">ultra pure water</TD><TD class="col1">-</TD><TD class="col2">ad 20 µL</TD></TR></TBODY></TABLE></DIV><P><STRONG>cgeA-C cotB-C/N</STRONG></P><P><STRONG>cotB-C/N</STRONG></P><P><STRONG>cot-Z</STRONG></P></DIV><H3 class="sectionedit88"><A name="section070816" id="section070816">07.08.16</A></H3><DIV class="level3"><P><STRONG>1) Repeat of 3A Assembly</STRONG></P><P><STRONG>Ligation mixture of plG_38/39 #1:</STRONG></P><DIV class="table sectionedit89"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0">Component</TH><TH class="col1">concentration</TH><TH class="col2">volume [µL]</TH></TR><TR class="row1"><TD class="col0">pBS1C</TD><TD class="col1">2000ng/µL</TD><TD class="col2">1</TD></TR><TR class="row2"><TD class="col0">plG16_038</TD><TD class="col1">105ng/µL</TD><TD class="col2">7</TD></TR><TR class="row3"><TD class="col0">plG16_039</TD><TD class="col1">104ng/µL</TD><TD class="col2">7</TD></TR><TR class="row4"><TD class="col0">PcotYZ</TD><TD class="col1">24ng/µL</TD><TD class="col2">0.4</TD></TR><TR class="row5"><TD class="col0" colspan="2">T4 Ligase</TD><TD class="col2">1µl</TD></TR><TR class="row6"><TD class="col0">T4 Ligase reaction buffer</TD><TD class="col1">10X</TD><TD class="col2">2</TD></TR><TR class="row7"><TD class="col0">ultra pure water</TD><TD class="col1">-</TD><TD class="col2">ad 20 µL</TD></TR></TBODY></TABLE></DIV><P><STRONG>2) Inoculation</STRONG> Inoculation of Culture #1/2 with construct plG16_038 and Culture #1/2 with construct plG16_039. </P></DIV></DIV></DIV><DIV class="color5"><DIV class="para_20"><H3 class="sectionedit90"><A name="section080816" id="section080816">08.08.16</A></H3><DIV class="level3"><P><STRONG>1)Miniprep</STRONG></P><P> Standard Qiagen Miniprep of Culture #1/2 with construct plG16_38 and Culture #1/2 with construct plG16_39. </P><P><STRONG>2) Digestion</STRONG></P><P><STRONG>Digestion mixture poG16_38/39:</STRONG></P><DIV class="table sectionedit91"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0">Component</TH><TH class="col1">concentration</TH><TH class="col2">volume [µL]</TH></TR><TR class="row1"><TD class="col0">DNA</TD><TD class="col1" colspan="2">500ng/µL</TD></TR><TR class="row2"><TD class="col0">XbaI</TD><TD class="col1">20u/µL</TD><TD class="col2">0.1</TD></TR><TR class="row3"><TD class="col0">PstI</TD><TD class="col1">20u/µL</TD><TD class="col2">0.1</TD></TR><TR class="row4"><TD class="col0">NEBuffer 3.1</TD><TD class="col1">10X</TD><TD class="col2">2</TD></TR><TR class="row5"><TD class="col0">ultra pure water</TD><TD class="col1">-</TD><TD class="col2">ad 20 µL</TD></TR></TBODY></TABLE></DIV><P><STRONG>Digestion mixture pBS1C:</STRONG></P><DIV class="table sectionedit92"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0">Component</TH><TH class="col1">concentration</TH><TH class="col2">volume [µL]</TH></TR><TR class="row1"><TD class="col0">DNA</TD><TD class="col1" colspan="2">500ng/µL</TD></TR><TR class="row2"><TD class="col0">EcoRI</TD><TD class="col1">20u/µL</TD><TD class="col2">0.2</TD></TR><TR class="row3"><TD class="col0">PstI</TD><TD class="col1">20u/µL</TD><TD class="col2">0.2</TD></TR><TR class="row4"><TD class="col0">NEBuffer 2.1</TD><TD class="col1">10X</TD><TD class="col2">2</TD></TR><TR class="row5"><TD class="col0">ultra pure water</TD><TD class="col1">-</TD><TD class="col2">ad 20 µL</TD></TR></TBODY></TABLE></DIV><P><STRONG>3) Repeat of 3A Assembly</STRONG></P><P><STRONG>Ligation mixture of plG_38/39 #1:</STRONG></P><DIV class="table sectionedit93"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0">Component</TH><TH class="col1">concentration</TH><TH class="col2">volume [µL]</TH></TR><TR class="row1"><TD class="col0">pBS1C</TD><TD class="col1">2000ng/µL</TD><TD class="col2">0.5</TD></TR><TR class="row2"><TD class="col0">plG_38</TD><TD class="col1">500ng/µL</TD><TD class="col2">1.5</TD></TR><TR class="row3"><TD class="col0">plG_39</TD><TD class="col1">500ng/µL</TD><TD class="col2">1.5</TD></TR><TR class="row4"><TD class="col0">PcotYZ</TD><TD class="col1">24ng/µL</TD><TD class="col2">0.5</TD></TR><TR class="row5"><TD class="col0" colspan="2">T-4 Ligase</TD><TD class="col2">1µl</TD></TR><TR class="row6"><TD class="col0">NEBuffer T-4 Ligase</TD><TD class="col1">10X</TD><TD class="col2">2</TD></TR><TR class="row7"><TD class="col0">ultra pure water</TD><TD class="col1">-</TD><TD class="col2">ad 20 µL</TD></TR></TBODY></TABLE></DIV><P><STRONG>4) Inoculation</STRONG> Transformed E.coli containing the plasmids pIG16_040 - 043 were inoculated. 6 colonies per plate were picked and inoculated in 5 mL LB medium supplemented with chloramphenicol. Incubation o/n at 37 °C and 250 rpm. </P><P><STRONG>5) Subcloning of PCotYZ</STRONG> PCotYZ PCR product was subcloned into pJET2.1 vector according to protocol delivered with the kit. </P></DIV><H3 class="sectionedit94"><A name="section090816" id="section090816">09.08.16</A></H3><DIV class="level3"><P><STRONG>1) MiniPrep</STRONG> The inoculated E.coli containing the plasmids pIG16_040 - 043 were preparated using the QiaQuick MiniPrep kit. The DNA was eluted with 30 µL of ultra pure water. Verification of the plasmids was performed by testdigestion. 500 ng of DNA was treated with XbaI and PstI in 1X NEBuffer 3.1 for 90 min at 37 °C. </P><P><STRONG>2) Sequencing</STRONG> pIG16_038#1 and pIG16_039#1 were sent to sequencing by GATCbiotech. </P><P><STRONG>3) ExtensionPCRs</STRONG></P><DIV class="table sectionedit95"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0">No.</TH><TH class="col1">Template</TH><TH class="col2">Primer oIG16_</TH><TH class="col3">Annealing temp [°C]</TH><TH class="col4">Resulting construct </TH></TR><TR class="row1"><TD class="col0">1</TD><TD class="col1">pGEX6P1</TD><TD class="col2">036 + 054</TD><TD class="col3">64</TD><TD class="col4">GST_HA_G4S</TD></TR><TR class="row2"><TD class="col0">2</TD><TD class="col1">pGEX6P1</TD><TD class="col2">041 + 055</TD><TD class="col3">64</TD><TD class="col4">G4S_HA_GST</TD></TR><TR class="row3"><TD class="col0">3</TD><TD class="col1">pJET_cotZ</TD><TD class="col2">006 + 008</TD><TD class="col3">58</TD><TD class="col4">HA_G4S_CotZ</TD></TR><TR class="row4"><TD class="col0">4</TD><TD class="col1">pJET_cotZ</TD><TD class="col2">003 + 004</TD><TD class="col3">58</TD><TD class="col4">CotZ_G4S_HA</TD></TR><TR class="row5"><TD class="col0">5</TD><TD class="col1">pJET_cotG</TD><TD class="col2">014 + 016</TD><TD class="col3">62</TD><TD class="col4">HA_G4S_CotG</TD></TR><TR class="row6"><TD class="col0">6</TD><TD class="col1">pJET_cotG</TD><TD class="col2">011 + 012</TD><TD class="col3">62</TD><TD class="col4">CotG_G4S_HA</TD></TR><TR class="row7"><TD class="col0">7</TD><TD class="col1">pJET_cotB</TD><TD class="col2">022 + 024</TD><TD class="col3">56</TD><TD class="col4">HA_G4S_CotB</TD></TR><TR class="row8"><TD class="col0">8</TD><TD class="col1">pJET_cotB</TD><TD class="col2">019 + 020</TD><TD class="col3">56</TD><TD class="col4">CotB_G4S_HA</TD></TR></TBODY></TABLE></DIV><P><STRONG>Reaction conditions</STRONG></P><DIV class="table sectionedit96"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0">Component</TH><TH class="col1">volume</TH></TR><TR class="row1"><TD class="col0">2X Q5 MasterMix</TD><TD class="col1">25 µL</TD></TR><TR class="row2"><TD class="col0">Primer1</TD><TD class="col1">2.5 µL</TD></TR><TR class="row3"><TD class="col0">Primer2</TD><TD class="col1">2.5 µL</TD></TR><TR class="row4"><TD class="col0">Template</TD><TD class="col1">0.1 µL</TD></TR><TR class="row5"><TD class="col0">water</TD><TD class="col1">19.9 µL</TD></TR></TBODY></TABLE></DIV><P><STRONG>PCR Program</STRONG> Touchdown PCRs corresponding to the respective annealing temperatures. </P><P><STRONG>3) Inoculation of pJET-PcotYZ colonies</STRONG></P><P> Tree colonies from the the plate 1 (Wlad) and Tree colonies from the the plate 2 (iGEM) were picked and inoculated. </P></DIV><H3 class="sectionedit97"><A name="section100816" id="section100816">10.08.16</A></H3><DIV class="level3"><P><STRONG>1) Gelextraction of ExtensionPCRs</STRONG> Was performed according to the protocol of the manufacturer. The samples were extracted from the gel from the previous day. </P><P><STRONG>2) Sequence analysis</STRONG> pIG16_038 + 039 #1
                     Sequencing showed that the samples were switched at some point. pIG16_039 contained an additional insert of ~9 bp between the alpha helical linker and the HA-tag. New samples have to be prepared for analysis. </P><P><STRONG>3) Reinoculation </STRONG> New colonies containing pIG16_038#2 039#2 were inoculated in 5 mL of LB medium supplemented with chloramphenicol. </P><P><STRONG>4) Miniprep of pJET-PCotYZ</STRONG></P><P> Standard Qiagen miniprep was performed with 3 cultures from the the plate 1 and 3 cultures from the the plate 2 . </P><P><STRONG>5) Test digestion pJET-PcotYZ</STRONG></P><P><STRONG>Digestion pJET-PcotYZ:</STRONG></P><DIV class="table sectionedit98"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0">Component</TH><TH class="col1">concentration</TH><TH class="col2">volume [µL]</TH></TR><TR class="row1"><TD class="col0">DNA</TD><TD class="col1">500ng/µL</TD><TD class="col2">1-3</TD></TR><TR class="row2"><TD class="col0">EcoRI</TD><TD class="col1">20u/µL</TD><TD class="col2">0.1</TD></TR><TR class="row3"><TD class="col0">SpeI</TD><TD class="col1">20u/µL</TD><TD class="col2">0.1</TD></TR><TR class="row4"><TD class="col0">NEBuffer Cut Smart</TD><TD class="col1">10X</TD><TD class="col2">2</TD></TR><TR class="row5"><TD class="col0">ultra pure water</TD><TD class="col1">-</TD><TD class="col2">ad 20 µL</TD></TR></TBODY></TABLE></DIV><P> Culture #1 was sent to sequencing. </P></DIV><H3 class="sectionedit99"><A name="section110816" id="section110816">11.08.16</A></H3><DIV class="level3"><P><STRONG>1) MiniPrep</STRONG> Plasmid preparation of pIG16_038#2 and pIG16_039#2 using the QiaQuick MiniPrep kit. The DNA was eluted with 30 µL of ultra pure water.
                    </P><P><STRONG>1) Sequencing</STRONG> The plasmids pIG16_038#2, pIG16_039#2, pIG16_040#2, pIG16_041#3, pIG16_IG16_042#3 and pJET1.2-PCotYZ#1 were sent to sequencing by GATCbiotech. </P></DIV><H3 class="sectionedit100"><A name="section120816" id="section120816">12.08.16</A></H3><DIV class="level3"><P><STRONG>1) Sequencing Results</STRONG> pIG16_040 - pIG16_042 could be confirmed by sequencing. Glycerol stock was prepared.
                     The samples from pIG16_038 and pIG16_039 were switched in previous steps. –&gt; switched back. pIG16_039 was confirmed by sequencing.
                     pIG16_038 had a 1bp deletion. Further colonies pIG16_038#3 &amp; #4 were sent to sequencing by GATC-Biotech.
                    </P><P><STRONG>2)MiniPrep of pIG16_039-042 and PcotYZ</STRONG> Standard Quiagen Miniprep was performed. </P><P><STRONG>3) Digestion of pIG16_039-042, PcotYZ</STRONG></P><P><STRONG>Digestion mixture pIG16_39-42:</STRONG></P><DIV class="table sectionedit101"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0">Component</TH><TH class="col1">concentration</TH><TH class="col2">volume [µL]</TH></TR><TR class="row1"><TD class="col0">DNA</TD><TD class="col1">2500ng</TD><TD class="col2">5µl</TD></TR><TR class="row2"><TD class="col0">XbaI</TD><TD class="col1">20u/µL</TD><TD class="col2">0.5</TD></TR><TR class="row3"><TD class="col0">PstI</TD><TD class="col1">20u/µL</TD><TD class="col2">0.5</TD></TR><TR class="row4"><TD class="col0">NEBuffer 3.1</TD><TD class="col1">10X</TD><TD class="col2">2</TD></TR><TR class="row5"><TD class="col0">ultra pure water</TD><TD class="col1">-</TD><TD class="col2">ad 20 µL</TD></TR></TBODY></TABLE></DIV><P><STRONG>Digestion mixture PcotYZ:</STRONG></P><DIV class="table sectionedit102"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0">Component</TH><TH class="col1">concentration</TH><TH class="col2">volume [µL]</TH></TR><TR class="row1"><TD class="col0">DNA</TD><TD class="col1">5000ng</TD><TD class="col2">10</TD></TR><TR class="row2"><TD class="col0">EcoRI</TD><TD class="col1">20u/µL</TD><TD class="col2">1</TD></TR><TR class="row3"><TD class="col0">SpeI</TD><TD class="col1">20u/µL</TD><TD class="col2">1</TD></TR><TR class="row4"><TD class="col0">NEBuffer Cut Smart</TD><TD class="col1">10X</TD><TD class="col2">5</TD></TR><TR class="row5"><TD class="col0">ultra pure water</TD><TD class="col1">-</TD><TD class="col2">ad 50 µL</TD></TR></TBODY></TABLE></DIV><P><STRONG>4) 3A Assembly</STRONG></P><P><STRONG>Ligation mixture of plG_38/39 #1:</STRONG></P><DIV class="table sectionedit103"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0">Component</TH><TH class="col1">concentration</TH><TH class="col2">volume [µL]</TH></TR><TR class="row1"><TD class="col0">pBS1C</TD><TD class="col1">50ng</TD><TD class="col2">1</TD></TR><TR class="row2"><TD class="col0">plG_39-41</TD><TD class="col1">20ng</TD><TD class="col2">1</TD></TR><TR class="row3"><TD class="col0">plG_42</TD><TD class="col1">25ng/µL</TD><TD class="col2">11</TD></TR><TR class="row4"><TD class="col0">PcotYZ</TD><TD class="col1">5.7ng</TD><TD class="col2">0.5</TD></TR><TR class="row5"><TD class="col0">T-4 Ligase</TD><TD class="col1">20u/µl</TD><TD class="col2">1µl</TD></TR><TR class="row6"><TD class="col0">NEBuffer T-4 Ligase</TD><TD class="col1">10X</TD><TD class="col2">2</TD></TR><TR class="row7"><TD class="col0">ultra pure water</TD><TD class="col1">-</TD><TD class="col2">ad 20 µL</TD></TR></TBODY></TABLE></DIV><P><STRONG>5) Gibson Cloning</STRONG><STRONG>1.33x MasterMix preparation.</STRONG> 5X Iso buffer: 50 µL
                     T5 Exonuclease (NEB): 1 µL (Diluted 1:10 in ultra pure water)
                     Taq Ligase (NEB): 25 µL
                     Phusion Polymerase (NEB): 3.125 µL
                     Nuclease free water: 108.4 µL
                     Aliquots: 7.5 µL, stored at -20 °C </P><P><STRONG>Assemblies</STRONG> All samples were adjusted to 50 ng/µL </P><DIV class="table sectionedit104"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0">No.</TH><TH class="col1">Fragment 1</TH><TH class="col2">Fragment 2</TH><TH class="col3">digested backbone</TH><TH class="col4">resulting construct</TH></TR><TR class="row1"><TD class="col0">1</TD><TD class="col1">GST oIG16_036+054</TD><TD class="col2">CotG oIG16_014+016</TD><TD class="col3">pSB1C3</TD><TD class="col4">pIG16_046</TD></TR><TR class="row2"><TD class="col0">2</TD><TD class="col1">GST oIG16_036+054</TD><TD class="col2">CgeA oIG16_046+045</TD><TD class="col3">pSB1C3</TD><TD class="col4">pIG16_047</TD></TR><TR class="row3"><TD class="col0">3</TD><TD class="col1">GST oIG16_036+054</TD><TD class="col2">CotZ 006+008</TD><TD class="col3">pSB1C3</TD><TD class="col4">pIG16_048</TD></TR><TR class="row4"><TD class="col0">4</TD><TD class="col1">GST oIG16_036+054</TD><TD class="col2">CotB 022+024</TD><TD class="col3">pSB1C3</TD><TD class="col4">pIG16_049</TD></TR><TR class="row5"><TD class="col0">5</TD><TD class="col1">GST oIG16_041+055</TD><TD class="col2">CgeA 050+052</TD><TD class="col3">pSB1C3</TD><TD class="col4">pIG16_050</TD></TR><TR class="row6"><TD class="col0">6</TD><TD class="col1">GST oIG16_041+055</TD><TD class="col2">CotZ 003+004</TD><TD class="col3">pSB1C3</TD><TD class="col4">pIG16_051</TD></TR><TR class="row7"><TD class="col0">7</TD><TD class="col1">GST oIG16_041+055</TD><TD class="col2">CotB 019+020</TD><TD class="col3">pSB1C3</TD><TD class="col4">pIG16_052</TD></TR><TR class="row8"><TD class="col0">8</TD><TD class="col1">aGFPnano oIG16_030+047</TD><TD class="col2">CotG 014+016</TD><TD class="col3">pSB1C3</TD><TD class="col4">pIG16_053</TD></TR><TR class="row9"><TD class="col0">9</TD><TD class="col1">aGFPnano oIG16_030+047</TD><TD class="col2">CotZ 006+008</TD><TD class="col3">pSB1C3</TD><TD class="col4">pIG16_054</TD></TR><TR class="row10"><TD class="col0">10</TD><TD class="col1">aGFPnano oIG16_030+047</TD><TD class="col2">CotB 022+024</TD><TD class="col3">pSB1C3</TD><TD class="col4">pIG16_055</TD></TR><TR class="row11"><TD class="col0">11</TD><TD class="col1">positive control</TD><TD class="col2">positive control</TD><TD class="col3">postive control</TD><TD class="col4">positive control from NEB HiFi Assembly kit</TD></TR></TBODY></TABLE></DIV><P> 0.5 µL of each fragment and 0.5 µL of the digested backbone were mixed with 1µL of ultra pure water and added to the 1.33x Gibson Mix and incubated for 60 min at 50 °C. 5µL of the reaction mix were used to transform chemically competent E.coli DH5alpha. Subsequently, 300 µL of LB medium were added. The Bacteria were incubated at 37 °C and 250 rpm and spread on Agar-LB plates supplemented with chloramphenicol. </P></DIV></DIV></DIV><DIV class="color6"><DIV class="para_20"><H3 class="sectionedit105"><A name="section130816" id="section130816">13.08.16</A></H3><DIV class="level3"><P><STRONG>1) Extension PCR</STRONG></P><DIV class="table sectionedit106"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0">template</TH><TH class="col1">Primer</TH><TH class="col2">Resulting construct</TH></TR><TR class="row1"><TD class="col0">pJET1.2_CotZ</TD><TD class="col1">oIG16_007+008</TD><TD class="col2">HA_alphaHelix_CotZ_pSB1C3-OH</TD></TR><TR class="row2"><TD class="col0">pJET1.2_CotZ</TD><TD class="col1">oIG16_003+005</TD><TD class="col2">pSB1C3-OH_CotZ_alphaHelix_HA</TD></TR></TBODY></TABLE></DIV><P><STRONG>Reaction conditions</STRONG></P><DIV class="table sectionedit107"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0">component</TH><TH class="col1">volume</TH></TR><TR class="row1"><TD class="col0">2XMM</TD><TD class="col1">25 µL</TD></TR><TR class="row2"><TD class="col0">Primer1</TD><TD class="col1">2.5 µL</TD></TR><TR class="row3"><TD class="col0">Primer2</TD><TD class="col1">2.5 µL</TD></TR><TR class="row4"><TD class="col0">template</TD><TD class="col1">0.1 µL</TD></TR><TR class="row5"><TD class="col0">dH2O</TD><TD class="col1">19.9 µL</TD></TR></TBODY></TABLE></DIV><P><STRONG>Cycler Program</STRONG> Touchdown58 </P><P> The samples analyzed by agarose gel electrophoresis and the bands corresponding to the expected sizes were extracted and purified using the Qiagen gel extraction kit. </P><P><STRONG>2) Inoculation</STRONG> The transformed E.coli from the Gibson assembly (previous day) were inoculated in 5 mL LB-medium supplemented with chloramphenicol and incubated o/n at 37 °C and 250 rpm. </P><P><STRONG>3) Gibson assembly</STRONG></P><DIV class="table sectionedit108"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0">No.</TH><TH class="col1">Fragment1</TH><TH class="col2">Fragment2</TH><TH class="col3">Digested Backbone</TH><TH class="col4">resulting construct</TH></TR><TR class="row1"><TD class="col0">1</TD><TD class="col1">pJet1.2_cotZ oIG16_007+008</TD><TD class="col2">aGFPnano oIG16_30+31</TD><TD class="col3">pSB1C3</TD><TD class="col4">pIG16_056</TD></TR><TR class="row2"><TD class="col0">2</TD><TD class="col1">pJet1.2_cotZ oIG16_006+008</TD><TD class="col2">aGFPnano oIG16_30+47</TD><TD class="col3">pSB1C3</TD><TD class="col4">pIG16_057</TD></TR><TR class="row3"><TD class="col0">3</TD><TD class="col1">pJet1.2_cotZ oIG16_003+005</TD><TD class="col2">aGFPnano oIG16_51+42</TD><TD class="col3">pSB1C3</TD><TD class="col4">pIG16_058</TD></TR><TR class="row4"><TD class="col0">4</TD><TD class="col1">pJet1.2_cotZ oIG16_003+004</TD><TD class="col2">aGFPnano oIG16_53+42</TD><TD class="col3">pSB1C3</TD><TD class="col4">pIG16_059</TD></TR><TR class="row5"><TD class="col0">5</TD><TD class="col1">positive control</TD><TD class="col2">positive control</TD><TD class="col3">positive control</TD><TD class="col4">C+ from NEB</TD></TR></TBODY></TABLE></DIV><P> 0.5 µL from each fragment were mixed with 1 µL of ultra pure water, added to 1.33x Gibson Master Mix and incubated at 50 °C for 1 h. 2µL of the reaction mix were transformed into chemically competent E.coli DH5alpha. </P></DIV><H3 class="sectionedit109"><A name="section140816" id="section140816">14.08.16</A></H3><DIV class="level3"><P><STRONG>1) MiniPrep</STRONG> Plasmid preparation of pIG16_046 - 055 using the QiaQuick MiniPrep kit. The DNA was eluted in 30 µL of ultra pure water. Verification of the plasmids by Testdigestion using. 500 ng of DNA was treated with 2 unit of XbaI and PstI in 1X NEBuffer 3.1 for 1 hour. . Unexpected band patterns. The DNA was not completly digested. </P><P><STRONG>2) Inoculation</STRONG> The transformed E.coli with the Gibson assemblies from the previous day were inoculated in 5 mL LB-medium supplemented with chloramphenicol </P></DIV><H3 class="sectionedit110"><A name="section150816" id="section150816">15.08.16</A></H3><DIV class="level3"><P><STRONG>1) Testdigestion</STRONG> The digestion of pIG16_046 - 055 from the previous day was repeated. 500 ng of DNA was treated with 4 unit of XbaI and PstI in 1X NEBuffer 2.1 for 90 min at 37 °C. </P><P> Positive samples were sent to sequencing:
                     1) pIG16_047#3
                     2) pIG16_048#1
                     3) pIG16_050#5
                     4) pIG16_051#1
                     5) pIG16_054#1
                     6) pIG16_055#2
                    </P></DIV><H3 class="sectionedit111"><A name="section160816" id="section160816">16.08.16</A></H3><DIV class="level3"><P><STRONG>1) Production of competent DH5-alpha E. coli</STRONG> E. colis were made competent according to zymo research Mix and go protocol </P></DIV><H3 class="sectionedit112"><A name="section170816" id="section170816">17.08.16</A></H3><DIV class="level3"><P><STRONG>1) Inoculation</STRONG> Since pIG16_046, 049, 052 were all negative 5 additional colonies were picked and inoculated in 5 mL of LB medium supplemented with chloramphenicol.
                     5 colonies of the E.coli transformed with the 3A assembly reaction (pIG16_062 - 065) were pickend and inoculated in 5 mL of LB medium supplemented with ampicilin.
                     5 colonies of the E.coli transformed with the Gibson assembly reaction (pIG16_057 - 059).
                    </P><P><STRONG>2) Glycerol stocks</STRONG> Preparation of Glycerol stocks of pIG16_038, 039, 040, 041, 042 </P></DIV><H3 class="sectionedit113"><A name="section180816" id="section180816">18.08.16</A></H3><DIV class="level3"><P><STRONG>1) Plasmid preparation </STRONG> Preparation of plasmids from transformed E.coli containing the constructs pIG16_46 + 049 + 052+ 058+ 059+ 062+ 063+ 064+ 065 </P><P><STRONG>2) Testdigestion</STRONG> 3A assembly: NotI + XhoI
                     Gibson assembly: XbaI + PstI
                    </P><P> Digestions partially incomplete.
                     Samples sent to sequencing: </P><P><STRONG>3) Sequencing</STRONG></P><DIV class="table sectionedit114"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0">Sample</TH><TH class="col1">Sequencing Primer</TH></TR><TR class="row1"><TD class="col0">pIG16_046#7</TD><TD class="col1">oIG16_039</TD></TR><TR class="row2"><TD class="col0">pIG16_046#7</TD><TD class="col1">oIG16_040</TD></TR><TR class="row3"><TD class="col0">pIG16_049#6</TD><TD class="col1">oIG16_039</TD></TR><TR class="row4"><TD class="col0">pIG16_049#6</TD><TD class="col1">oIG16_040</TD></TR><TR class="row5"><TD class="col0">pIG16_052#10</TD><TD class="col1">oIG16_039</TD></TR><TR class="row6"><TD class="col0">pIG16_052#10</TD><TD class="col1">oIG16_040</TD></TR><TR class="row7"><TD class="col0">pIG16_058#2</TD><TD class="col1">oIG16_039</TD></TR><TR class="row8"><TD class="col0">pIG16_058#2</TD><TD class="col1">oIG16_040</TD></TR><TR class="row9"><TD class="col0">pIG16_059#5</TD><TD class="col1">oIG16_039</TD></TR><TR class="row10"><TD class="col0">pIG16_059#5</TD><TD class="col1">oIG16_040</TD></TR><TR class="row11"><TD class="col0">pIG16_062#3</TD><TD class="col1">oIG16_028</TD></TR><TR class="row12"><TD class="col0">pIG16_062#3</TD><TD class="col1">oIG16_029</TD></TR><TR class="row13"><TD class="col0">pIG16_063#2</TD><TD class="col1">oIG16_028</TD></TR><TR class="row14"><TD class="col0">pIG16_063#2</TD><TD class="col1">oIG16_029</TD></TR><TR class="row15"><TD class="col0">pIG16_064#3</TD><TD class="col1">oIG16_028</TD></TR><TR class="row16"><TD class="col0">pIG16_064#3</TD><TD class="col1">oIG16_029</TD></TR><TR class="row17"><TD class="col0">pIG16_065#5</TD><TD class="col1">oIG16_028</TD></TR><TR class="row18"><TD class="col0">pIG16_065#5</TD><TD class="col1">oIG16_029</TD></TR></TBODY></TABLE></DIV></DIV></DIV></DIV><DIV class="color7"><DIV class="para_20"><H3 class="sectionedit115"><A name="section190816" id="section190816">19.08.16</A></H3><DIV class="level3"><P><STRONG>1) Glycerol stock </STRONG> Preparation of a 15 % Glycerol stock of pIG16_039 and storage at - 80 °C. </P><P><STRONG>2) Reinoculation </STRONG> New 6 mL LB cultures were prepared for pIG16_047#3, 050#5, 048#4, 051#1 054#4 and incubated o/n at 37 °C and 250 rpm. For subsequent 3A assembly. </P><P><STRONG>3) Preparation of competent E.coli</STRONG></P><P> The Ecoli strain DH5-alpha was prepered and made competent according to Zymoreserch MIX and GO kit. </P></DIV><H3 class="sectionedit116"><A name="section200816" id="section200816">20.08.16</A></H3><DIV class="level3"><P><STRONG> 1) Extension PCR</STRONG></P><DIV class="table sectionedit117"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0">Template</TH><TH class="col1">Primer</TH><TH class="col2">Resulting construct</TH></TR><TR class="row1"><TD class="col0">pJet1.2_CotG</TD><TD class="col1">oIG056+012</TD><TD class="col2">pSB1C3-OH_CotG_G4S_HA</TD></TR><TR class="row2"><TD class="col0">pJet1.2_CotG</TD><TD class="col1">oIG16_056+013</TD><TD class="col2">pSB1C3-OH_CotG_aHelix_HA</TD></TR></TBODY></TABLE></DIV><P><STRONG>Reaction conditions</STRONG></P><DIV class="table sectionedit118"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0">component</TH><TH class="col1">concentration</TH><TH class="col2">volume[µL]</TH></TR><TR class="row1"><TD class="col0">Q5 MasterMix</TD><TD class="col1">2X</TD><TD class="col2">25</TD></TR><TR class="row2"><TD class="col0">Primer1</TD><TD class="col1">10 µM</TD><TD class="col2">2.5</TD></TR><TR class="row3"><TD class="col0">Primer2</TD><TD class="col1">10 µM</TD><TD class="col2">2.5</TD></TR><TR class="row4"><TD class="col0">ultrapure water</TD><TD class="col1">-</TD><TD class="col2">20.0</TD></TR><TR class="row5"><TD class="col0">template DNA</TD><TD class="col2">0.1</TD></TR></TBODY></TABLE></DIV><P><STRONG>Thermal Cycling</STRONG> Touchdown 62
                    </P><P><STRONG>2) Gibson Assembly </STRONG></P><DIV class="table sectionedit119"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0">Number</TH><TH class="col1">Fragment 1</TH><TH class="col2">Fragment 2</TH><TH class="col3">Linearized backbone</TH><TH class="col4">Resulting construct</TH></TR><TR class="row1"><TD class="col0">1</TD><TD class="col1">CotG oIG16_056+013</TD><TD class="col2">aGFPnano oIG16_051+042</TD><TD class="col3">pSB1C3 XbaI+SpeI</TD><TD class="col4">pIG16_043</TD></TR><TR class="row2"><TD class="col0">2</TD><TD class="col1">CotG oIG16_056+012</TD><TD class="col2">aGFPnano oIG16_053+042</TD><TD class="col3">pSB1C3 XbaI+SpeI</TD><TD class="col4">pIG16_065</TD></TR><TR class="row3"><TD class="col0">3</TD><TD class="col1">CotG oIG16_056+013</TD><TD class="col2">GST oIG16_041+055</TD><TD class="col3">pSB1C3 XbaI+SpeI</TD><TD class="col4">pIG16_066</TD></TR><TR class="row4"><TD class="col0">4</TD><TD class="col1">CotG oIG16_014+016</TD><TD class="col2">aGFPnano oIG16_030+047</TD><TD class="col3">pSB1C3 XbaI+SpeI</TD><TD class="col4">pIG16_053</TD></TR></TBODY></TABLE></DIV><P> The concentration of all fragments was adjusted to 50 ng/µL. 0.5 µL of each fragment were mixed alongside with 0.5 µL of the linearized vector and added to the 1.33x Gibson MasterMix and incubated for 60 min at 50 °C.
                     5µL of the Gibson mix were transformed into chemically competent E.coli DH5alpha and spread on LB agar plate supplemented with chloramphenicol.
                    </P><P><STRONG> 3) Digestion for 3 A Assembly </STRONG></P><P><STRONG>Digestion of PcotYZ</STRONG></P><DIV class="table sectionedit120"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0">component</TH><TH class="col1">concentration</TH><TH class="col2">volume[µL]</TH></TR><TR class="row1"><TD class="col0">PcotYZ</TD><TD class="col1">6000ng</TD><TD class="col2">20µl</TD></TR><TR class="row2"><TD class="col0">CS Buffer</TD><TD class="col1">10x</TD><TD class="col2">5µl</TD></TR><TR class="row3"><TD class="col0">EcoRI-HF</TD><TD class="col1">20u/µl</TD><TD class="col2">1.2µl</TD></TR><TR class="row4"><TD class="col0">SpeI-HF</TD><TD class="col1">20u/µl</TD><TD class="col2">1.2µl</TD></TR><TR class="row5"><TD class="col0">ultrapure water</TD><TD class="col1">-</TD><TD class="col2">50.0</TD></TR></TBODY></TABLE></DIV><P><STRONG>Digestion of pBS1C and pBS4S</STRONG></P><DIV class="table sectionedit121"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0">component</TH><TH class="col1">concentration</TH><TH class="col2">volume[µL]</TH></TR><TR class="row1"><TD class="col0">pBS1C and pBS4S</TD><TD class="col1">5000ng</TD><TD class="col2">2,5µl</TD></TR><TR class="row2"><TD class="col0">2.1 Buffer</TD><TD class="col1">10x</TD><TD class="col2">5µl</TD></TR><TR class="row3"><TD class="col0">EcoRI-HF</TD><TD class="col1">20u/µl</TD><TD class="col2">1.2µl</TD></TR><TR class="row4"><TD class="col0">PstI</TD><TD class="col1">20u/µl</TD><TD class="col2">1.2µl</TD></TR><TR class="row5"><TD class="col0">ultrapure water</TD><TD class="col1">-</TD><TD class="col2">50.0</TD></TR></TBODY></TABLE></DIV><P><STRONG>Digestion of pIG16_38 47 48 50 51 54</STRONG></P><DIV class="table sectionedit122"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0">component</TH><TH class="col1">concentration</TH><TH class="col2">volume[µL]</TH></TR><TR class="row1"><TD class="col0">pIG16_#</TD><TD class="col1">3000ng</TD><TD class="col2">20µl</TD></TR><TR class="row2"><TD class="col0">3.1 Buffer</TD><TD class="col1">10x</TD><TD class="col2">5µl</TD></TR><TR class="row3"><TD class="col0">XbaI</TD><TD class="col1">20u/µl</TD><TD class="col2">0.6µl</TD></TR><TR class="row4"><TD class="col0">PstI</TD><TD class="col1">20u/µl</TD><TD class="col2">0.6µl</TD></TR><TR class="row5"><TD class="col0">ultrapure water</TD><TD class="col1">-</TD><TD class="col2">50.0</TD></TR></TBODY></TABLE></DIV></DIV><P><STRONG> 4) 3 A Assembly </STRONG></P><P> The 3 A Assembly was carried out according to the neb protocol. </P><P><STRONG>3 A Assembly with pBS1C</STRONG></P><DIV class="table sectionedit123"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0">component</TH><TH class="col1">concentration</TH><TH class="col2">volume[µL]</TH></TR><TR class="row1"><TD class="col0">pBS1C</TD><TD class="col1">102ng/µl</TD><TD class="col2">50ng</TD></TR><TR class="row2"><TD class="col0">T4 ligase Buffer</TD><TD class="col1">10x</TD><TD class="col2">2µl</TD></TR><TR class="row3"><TD class="col0">T4 ligase</TD><TD class="col1">20u/µl</TD><TD class="col2">1µl</TD></TR><TR class="row4"><TD class="col0">PcotYZ</TD><TD class="col1">20ng/µl</TD><TD class="col2">5ng</TD></TR><TR class="row5"><TD class="col0">pIG16_38/47/48/50/51/54</TD><TD class="col1">differentng/µl</TD><TD class="col2">25ng/µl</TD></TR><TR class="row6"><TD class="col0">ultrapure water</TD><TD class="col1">-</TD><TD class="col2">20.0</TD></TR></TBODY></TABLE></DIV><P><STRONG>3 A Assembly with pBS4S</STRONG></P><DIV class="table sectionedit124"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0">component</TH><TH class="col1">concentration</TH><TH class="col2">volume[µL]</TH></TR><TR class="row1"><TD class="col0">pBS4S</TD><TD class="col1">102ng/µl</TD><TD class="col2">50ng</TD></TR><TR class="row2"><TD class="col0">T4 ligase Buffer</TD><TD class="col1">10x</TD><TD class="col2">2µl</TD></TR><TR class="row3"><TD class="col0">T4 ligase</TD><TD class="col1">20u/µl</TD><TD class="col2">1µl</TD></TR><TR class="row4"><TD class="col0">PcotYZ</TD><TD class="col1">20ng/µl</TD><TD class="col2">5ng</TD></TR><TR class="row5"><TD class="col0">pIG16_38/47/48/50/51/54</TD><TD class="col1">differentng/µl</TD><TD class="col2">25ng/µl</TD></TR><TR class="row6"><TD class="col0">ultrapure water</TD><TD class="col1">-</TD><TD class="col2">20.0</TD></TR></TBODY></TABLE></DIV></DIV></DIV><DIV class="color8"><DIV class="para_20"><H3 class="sectionedit125"><A name="section210816" id="section210816">21.08.16</A></H3><DIV class="level3"><P><STRONG>1) Inoculation </STRONG> 5 colonies per plate were inoculated in 5 mL of LB medium supplemented with chloramphenicol from the Gibson assembly from the previous day. Incubation o/n at 37 °C and 250 rpm.
                    </P><P> Reinoculation of pIG16_046#7, 049#6, 052#10, 058#2, 059#5, 062#3
                    </P><P><STRONG>2) Glycerol stocks</STRONG></P><P> Preparation of 15 % glycerol stocks for:
                     pIG16_045#3, 062#2, 063#3, 064#5 </P><P><STRONG>3) Linearization #1 </STRONG></P><P> 3 µg of the following plasmids were linearized using the appropriate enzyme in a reaction volume of 50 µL. The digestion was loaded on a TAE agarose gel and subjected to electrophoresis. </P><P> The linearized plasmids were extracted from the gel using the Qiagen gel extraction kit and subsequently transformed into competent Bacillus subtilis. </P><P><STRONG>4) Linearization #2 </STRONG></P><P><STRONG>Digestion of pIG16_77/81/82/100/101</STRONG></P><DIV class="table sectionedit126"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0">component</TH><TH class="col1">concentration</TH><TH class="col2">volume[µL]</TH></TR><TR class="row1"><TD class="col0">pIG16_77/81/82/100/101</TD><TD class="col1">2000ng</TD><TD class="col2">4,5µl</TD></TR><TR class="row2"><TD class="col0">Cutsmart</TD><TD class="col1">10x</TD><TD class="col2">5µl</TD></TR><TR class="row3"><TD class="col0">XhoI</TD><TD class="col1">20u/µl</TD><TD class="col2">0.5µl</TD></TR><TR class="row4"><TD class="col0">ultra pure water</TD><TD class="col1">-</TD><TD class="col2">ad50.0</TD></TR></TBODY></TABLE></DIV><P><STRONG>Digestion of pIG16_104/105/117</STRONG></P><DIV class="table sectionedit127"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0">component</TH><TH class="col1">concentration</TH><TH class="col2">volume[µL]</TH></TR><TR class="row1"><TD class="col0">pIG16_104/105/117</TD><TD class="col1">2000ng</TD><TD class="col2">4,5µl</TD></TR><TR class="row2"><TD class="col0">CS Buffer</TD><TD class="col1">10x</TD><TD class="col2">5µl</TD></TR><TR class="row3"><TD class="col0">EcoRV-HF</TD><TD class="col1">20u/µl</TD><TD class="col2">0.5µl</TD></TR><TR class="row4"><TD class="col0">ultrapure water</TD><TD class="col1">-</TD><TD class="col2">ad50.0</TD></TR></TBODY></TABLE></DIV><P> →Linearization was followed by gel extraction for transformation in Bacillus subtilis. </P><P> 4) Inoculation of 3 A Assemblies colonies. </P></DIV><H3 class="sectionedit128"><A name="section220816" id="section220816">22.08.16</A></H3><DIV class="level3"><P><STRONG>1) Plasmid preparation</STRONG> The plasmids of the following strains were extracted using the QiaQuick MiniPrep kit:
                     pIG16_043 053 065 066
                     500 ng of DNA was digested with 5 units of XbaI and PstI and analyzed by gel electrophoresis.
                    </P><P><STRONG>2) Glycerol stocks</STRONG> Glycerol stocks of pIG16_045 062 063 064 46 were prepared and stored at -80 °C.
                    </P><P><STRONG>3) Sequencing </STRONG> pIG16_058#3 and pIG16_058#4 were sent to sequencing. </P><P><STRONG>4) Inoculation </STRONG>new colonies picked of pIG16_049 #11 - 16 and pIG16_052 #11-16 were picked
                    </P><P><STRONG>5) Transformation</STRONG> The Biobrick BBa_J18932 from the Registry distribution kit was dissolved in 10 µL of ultrapure water. 2 µL were used for transformation of chemically competent E.coli DH5alpha and spread on a LB agar plate supplemented with chloramphenicol. </P><P><STRONG>6) Testdigestion of 3 A assembly from 20.9</STRONG></P><P><STRONG>Digestion 3 A assembly</STRONG></P><DIV class="table sectionedit129"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0">component</TH><TH class="col1">concentration</TH><TH class="col2">volume[µL]</TH></TR><TR class="row1"><TD class="col0">pIG16_</TD><TD class="col1">500ng</TD><TD class="col2">4µl</TD></TR><TR class="row2"><TD class="col0">2.1 Buffer</TD><TD class="col1">10x</TD><TD class="col2">5µl</TD></TR><TR class="row3"><TD class="col0">EcoRI-HF</TD><TD class="col1">20u/µl</TD><TD class="col2">0.1µl</TD></TR><TR class="row4"><TD class="col0">PstI</TD><TD class="col1">20u/µl</TD><TD class="col2">0.1µl</TD></TR><TR class="row5"><TD class="col0">ultrapure water</TD><TD class="col1">-</TD><TD class="col2">ad20.0</TD></TR></TBODY></TABLE></DIV><P> →Colonies were sent to sequencing. </P></DIV><H3 class="sectionedit130"><A name="section230816" id="section230816">23.08.16</A></H3><DIV class="level3"><P><STRONG>1) Plasmid preparation</STRONG> Miniprep of pIG16_049#11-16 and pIG16_052#11-16. Testdigestion: 500 ng of DNA was treated with 5 unit of XbaI and PstI and analyzed by gel electrophoresis. </P><P><STRONG>2) Preparation of Plasmids for transformation of B.subtilis</STRONG> MiniPrep of pIG16_045 + 062\ Linearization of 2µg of DNA with a 10 unit of ScaI. The DNA was purified with the QiaGen PCR purification kit and used for transformation of competent B. subtilis. </P></DIV></DIV></DIV><DIV class="color1"><DIV class="para_20"><H3 class="sectionedit131"><A name="section240816" id="section240816">24.08.16</A></H3><DIV class="level3"><P><STRONG>1) Sequencing</STRONG> The following samples were sent to sequencing:
                     pIG16-044, 059#4, 058#3, 077, 082 </P><P><STRONG>2) Plasmid preparation</STRONG> The DNA of pIG16_043#6-11 was extracted using the QiaQuick Miniprep kit. </P><P><STRONG>3) Test digestion</STRONG> 500ng of pIG16_043, pIG16_120(BBa_J18932) were treated with 5 unit of XbaI+PstI and analyzed by gel electrophoresis. </P><P><STRONG>4) Sequencing</STRONG> The following plasmids were sent to sequencing:
                     pSB1C3_mCherry#2 (BBa_J18932), pIG16_044#1, 049#12, 058#3, 059#4, 077#1, 082#1, 101#1, 104#1, 105#1, 117#5 </P></DIV><H3 class="sectionedit132"><A name="section250816" id="section250816">25.08.16</A></H3><DIV class="level3"><P><STRONG>1) Inoculations</STRONG> Inoculations of the following samples for subsequent plasmid preparations and glycerol stocks: pIG16_045, 062, 063, 064, 046#7, 049#12, 059#4, 056#1, 058#3, 101#1, 044#1, 082#2, 104#1, 077#1, 081#1,105#2, 100#1, 117#4 </P></DIV><H3 class="sectionedit133"><A name="section260816" id="section260816">26.08.16</A></H3><DIV class="level3"><P><STRONG>1) Plasmid preparation</STRONG> The plasmids from the previous inoculation were extracted using the Qiagen Plasmid MiniPrep kit. The DNA was eluted in 30 µL of ultra pure water and the DNA concentration was determined by NanoDrop. </P><P><STRONG>2) Sequencing</STRONG> The following plasmids were sent to sequencing: pIG16_081#1, 100#1, 117#4 </P><P><STRONG>3) Preparation of Plasmids for Transformation of B.subtilis</STRONG> 2 µg of the following plasmids were linearized by treatment with ScaI-HF in 1X cutsmart buffer at a total volume of 50 µL.
                     pIG16_045, 062, 063, 064, 117, 104, 044, 077, 081, 082, 100, 101, 105.
                     The linearized plasmids were loaded on a 1% TAE agarose gel and extracted after electrophoresis and used for transformation of competent B. subtilis. </P></DIV></DIV></DIV><DIV class="color2"><DIV class="para_20"><H3 class="sectionedit134"><A name="section270816" id="section270816">27.08.16</A></H3><DIV class="level3"><P><STRONG>1) Digestion for 3A assembly </STRONG> Digestion 1 µg of DNA of pIG16_046, 049, 053, 059, 065, 066, 053 with 10 units of XbaI and PstI in 1XCutSmart buffer in a reaction volume of 50 µL. The digested samples were loaded on an 1 % agarose TAE gel and subjected to electrophoresis. </P><P><STRONG>Digestion 3 A assembly</STRONG></P><DIV class="table sectionedit135"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0">component</TH><TH class="col1">concentration</TH><TH class="col2">volume[µL]</TH></TR><TR class="row1"><TD class="col0">pIG16_60</TD><TD class="col1">6000ng</TD><TD class="col2">10µl</TD></TR><TR class="row2"><TD class="col0">2.1 Buffer</TD><TD class="col1">10x</TD><TD class="col2">2µl</TD></TR><TR class="row3"><TD class="col0">EcoRI-HF</TD><TD class="col1">20u/µl</TD><TD class="col2">1.2µl</TD></TR><TR class="row4"><TD class="col0">SpeI</TD><TD class="col1">20u/µl</TD><TD class="col2">1.2µl</TD></TR><TR class="row5"><TD class="col0">ultrapure water</TD><TD class="col1">-</TD><TD class="col2">ad20.0</TD></TR></TBODY></TABLE></DIV><P><STRONG>Digestion 3 A assembly</STRONG></P><DIV class="table sectionedit136"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0">component</TH><TH class="col1">concentration</TH><TH class="col2">volume[µL]</TH></TR><TR class="row1"><TD class="col0">pBS1C/4S/E2</TD><TD class="col1">2000ng</TD><TD class="col2">4µl</TD></TR><TR class="row2"><TD class="col0">CS Buffer</TD><TD class="col1">10x</TD><TD class="col2">2µl</TD></TR><TR class="row3"><TD class="col0">EcoRI-HF</TD><TD class="col1">20u/µl</TD><TD class="col2">0.4µl</TD></TR><TR class="row4"><TD class="col0">PstI</TD><TD class="col1">20u/µl</TD><TD class="col2">0.4µl</TD></TR><TR class="row5"><TD class="col0">ultrapure water</TD><TD class="col1">-</TD><TD class="col2">ad20.0</TD></TR></TBODY></TABLE></DIV><P><STRONG>Digestion 3 A assembly</STRONG></P><DIV class="table sectionedit137"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0">component</TH><TH class="col1">concentration</TH><TH class="col2">volume[µL]</TH></TR><TR class="row1"><TD class="col0">pIG16_46/47/49/56/65/mCherry</TD><TD class="col1">1000ng</TD><TD class="col2">4µl</TD></TR><TR class="row2"><TD class="col0">3.1 Buffer</TD><TD class="col1">10x</TD><TD class="col2">2µl</TD></TR><TR class="row3"><TD class="col0">XbaI</TD><TD class="col1">20u/µl</TD><TD class="col2">0.5µl</TD></TR><TR class="row4"><TD class="col0">PstI</TD><TD class="col1">20u/µl</TD><TD class="col2">0.5µl</TD></TR><TR class="row5"><TD class="col0">ultrapure water</TD><TD class="col1">-</TD><TD class="col2">ad20.0</TD></TR></TBODY></TABLE></DIV><P> After separation the appropriate fragments were extracted from the gel using the QiaGen gel extraction kit. </P><P><STRONG>2) Extension PCR</STRONG></P><DIV class="table sectionedit138"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0">No.</TH><TH class="col1">Template</TH><TH class="col2">Oligos oIG16_</TH><TH class="col3">Resulting construct</TH></TR><TR class="row1"><TD class="col0">1</TD><TD class="col1">pIG16_22(GST)</TD><TD class="col2">37+55</TD><TD class="col3">aHelix_HA_GST_pSB1C3-OH</TD></TR><TR class="row2"><TD class="col0">2</TD><TD class="col1">pIG16_22(GST)</TD><TD class="col2">36+38</TD><TD class="col3">pSB1C3-OH_GST_HA_aHelix</TD></TR><TR class="row3"><TD class="col0">3</TD><TD class="col1">pIG16_030(CotB)</TD><TD class="col2">23+24</TD><TD class="col3">HA_aHelix_CotB_pSB1C3-OH</TD></TR><TR class="row4"><TD class="col0">4</TD><TD class="col1">pIG16_030(CotB)</TD><TD class="col2">19+21</TD><TD class="col3">pSB1C3-OH_CotB_aHelix_HA</TD></TR></TBODY></TABLE></DIV><P><STRONG>Reaction conditions</STRONG></P><DIV class="table sectionedit139"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0">component</TH><TH class="col1">volume[µL]</TH></TR><TR class="row1"><TD class="col0">2X Q5 MasterMix</TD><TD class="col1">25</TD></TR><TR class="row2"><TD class="col0">Primer1 10 M</TD><TD class="col1">2.5</TD></TR><TR class="row3"><TD class="col0">Primer2 10 M</TD><TD class="col1">2.5</TD></TR><TR class="row4"><TD class="col0">template</TD><TD class="col1">&lt;0.1 </TD></TR><TR class="row5"><TD class="col0">dH2O</TD><TD class="col1">20</TD></TR></TBODY></TABLE></DIV><P> The samples were loaded on an 1 % Agarose TAE gel and subjected to electrophoresis.
                    </P><P> After separation the appropriate fragments were extracted using the QiaGen gel extraction kit. </P><P><STRONG>3) Gibson Master Mix</STRONG> Preparation of 1.33x Gibson Master Mix. </P><P><STRONG>4) Inoculation</STRONG></P><P> 4 colonies of each plate were picked from the previous transformation: pIG16_079, 085, 088, 096, 108, 112, 121, 122, 123 </P><P><STRONG>5) 3 A assembly</STRONG></P><P><STRONG>3 A Assembly with pBS1C</STRONG></P><DIV class="table sectionedit140"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0">component</TH><TH class="col1">concentration</TH><TH class="col2">volume[µL]</TH></TR><TR class="row1"><TD class="col0">pBS1C</TD><TD class="col1">102ng/µl</TD><TD class="col2">50ng</TD></TR><TR class="row2"><TD class="col0">T4 ligase Buffer</TD><TD class="col1">10x</TD><TD class="col2">2µl</TD></TR><TR class="row3"><TD class="col0">T4 ligase</TD><TD class="col1">20u/µl</TD><TD class="col2">1µl</TD></TR><TR class="row4"><TD class="col0">PcotYZ</TD><TD class="col1">20ng/µl</TD><TD class="col2">9.5ng</TD></TR><TR class="row5"><TD class="col0">pIG16_46/49/53/56/65/mCherry</TD><TD class="col1">different ng/µl</TD><TD class="col2">41ng/µl</TD></TR></TBODY></TABLE></DIV><P><STRONG>3 A Assembly with pBS2E</STRONG></P><DIV class="table sectionedit141"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0">component</TH><TH class="col1">concentration</TH><TH class="col2">volume[µL]</TH></TR><TR class="row1"><TD class="col0">pBS2E</TD><TD class="col1">80ng/µl</TD><TD class="col2">50ng</TD></TR><TR class="row2"><TD class="col0">T4 ligase Buffer</TD><TD class="col1">10x</TD><TD class="col2">2µl</TD></TR><TR class="row3"><TD class="col0">T4 ligase</TD><TD class="col1">20u/µl</TD><TD class="col2">1µl</TD></TR><TR class="row4"><TD class="col0">PcotYZ</TD><TD class="col1">20ng/µl</TD><TD class="col2">9.5ng</TD></TR><TR class="row5"><TD class="col0">pIG16_mCherry</TD><TD class="col1">different ng/µl</TD><TD class="col2">28ng/µl</TD></TR></TBODY></TABLE></DIV><P><STRONG>3 A Assembly with pBS4S</STRONG></P><DIV class="table sectionedit142"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0">component</TH><TH class="col1">concentration</TH><TH class="col2">volume[µL]</TH></TR><TR class="row1"><TD class="col0">pBS4S</TD><TD class="col1">75ng/µl</TD><TD class="col2">50ng</TD></TR><TR class="row2"><TD class="col0">T4 ligase Buffer</TD><TD class="col1">10x</TD><TD class="col2">2µl</TD></TR><TR class="row3"><TD class="col0">T4 ligase</TD><TD class="col1">20u/µl</TD><TD class="col2">1µl</TD></TR><TR class="row4"><TD class="col0">PcotYZ</TD><TD class="col1">20ng/µl</TD><TD class="col2">12.5ng</TD></TR><TR class="row5"><TD class="col0">pIG16_46/49/mCherry</TD><TD class="col1">different ng/µl</TD><TD class="col2">55ng/µl</TD></TR></TBODY></TABLE></DIV></DIV><H3 class="sectionedit143"><A name="section280816" id="section280816">28.08.16</A></H3><DIV class="level3"><P> 1) Inoculation </P><P> Colonies from the 3 A assembly (27.08) were picked. </P></DIV><H3 class="sectionedit144"><A name="section290816" id="section290816">29.08.16</A></H3><DIV class="level3"><P><STRONG>1)Sequencing</STRONG> The following plasmids were sent to sequencing: pIG16_032 033 034 035. (Integration vectors sent from Dr. Hinc, University of Gdansk) </P><P><STRONG>2) 3A assembly</STRONG></P><P><STRONG>3 A Assembly with pBS1C</STRONG></P><DIV class="table sectionedit145"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0">component</TH><TH class="col1">concentration</TH><TH class="col2">volume[µL]</TH></TR><TR class="row1"><TD class="col0">pBS1C</TD><TD class="col1">102ng/µl</TD><TD class="col2">50ng</TD></TR><TR class="row2"><TD class="col0">T4 ligase Buffer</TD><TD class="col1">10x</TD><TD class="col2">2µl</TD></TR><TR class="row3"><TD class="col0">T4 ligase</TD><TD class="col1">20u/µl</TD><TD class="col2">1µl</TD></TR><TR class="row4"><TD class="col0">PcotYZ</TD><TD class="col1">20ng/µl</TD><TD class="col2">9.5ng</TD></TR><TR class="row5"><TD class="col0">pIG16_58/59/66</TD><TD class="col1">different ng/µl</TD><TD class="col2">55ng/µl</TD></TR></TBODY></TABLE></DIV><P><STRONG>3 A Assembly with pBS4S</STRONG></P><DIV class="table sectionedit146"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0">component</TH><TH class="col1">concentration</TH><TH class="col2">volume[µL]</TH></TR><TR class="row1"><TD class="col0">pBS4S</TD><TD class="col1">75ng/µl</TD><TD class="col2">50ng</TD></TR><TR class="row2"><TD class="col0">T4 ligase Buffer</TD><TD class="col1">10x</TD><TD class="col2">2µl</TD></TR><TR class="row3"><TD class="col0">T4 ligase</TD><TD class="col1">20u/µl</TD><TD class="col2">1µl</TD></TR><TR class="row4"><TD class="col0">PcotYZ</TD><TD class="col1">20ng/µl</TD><TD class="col2">12.5ng</TD></TR><TR class="row5"><TD class="col0">pIG16_66</TD><TD class="col1">different ng/µl</TD><TD class="col2">55ng/µl</TD></TR></TBODY></TABLE></DIV></DIV><H3 class="sectionedit147"><A name="section300816" id="section300816">30.08.16</A></H3><DIV class="level3"><P><STRONG>1) Sequencing</STRONG> The following plasmids were sent to sequencing: pIG16_112#3, 121#3, 122#2, 079#1, 085#3, 088#4, 096#1 </P><P><STRONG>2) Inoculation</STRONG> E.coli DH5alpha containing the following plasmids were inoculated in 5 mL LB medium supplemented with the appropriate antibiotics. Incubation o/n at 37 °C and 200 rpm. pIG16_100, 117, 123, 081, 104, 077, 082, 122, 121, 101, 045, 062, 063, 064, 032, 033, 034, 035 </P><P><STRONG>3) Gibson assembly</STRONG></P><P> The Fragments for Gibson assembly were adjusted to a concentration of 50 ng/µL. </P><DIV class="table sectionedit148"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0">Number</TH><TH class="col1">Fragment1 amplified with oIG16_</TH><TH class="col2">Fragment2 amplified with oIG16_</TH><TH class="col3">Linearized backbone</TH><TH class="col4">resulting construct</TH></TR><TR class="row1"><TD class="col0">1</TD><TD class="col1">CotZ oIG16_007 + 008</TD><TD class="col2">GST 036 + 038</TD><TD class="col3">pSB1C3 </TD><TD class="col4">pIG16_067</TD></TR><TR class="row2"><TD class="col0">2</TD><TD class="col1">CotZ 003 + 005</TD><TD class="col2">GST 037 + 055</TD><TD class="col3">pSB1C3</TD><TD class="col4">pIG16_068</TD></TR><TR class="row3"><TD class="col0">3</TD><TD class="col1">CotG 056 + 013</TD><TD class="col2">aGFPnano 051 + 042</TD><TD class="col3">pSB1C3</TD><TD class="col4">pIG16_043</TD></TR><TR class="row4"><TD class="col0">4</TD><TD class="col1">CotG 015 + 015</TD><TD class="col2">GST 036 + 038</TD><TD class="col3">pSB1C3</TD><TD class="col4">pIG16_069</TD></TR><TR class="row5"><TD class="col0">5</TD><TD class="col1">CotG 056 + 013</TD><TD class="col2">GST 037 + 055</TD><TD class="col3">pSB1C3</TD><TD class="col4">pIG16_070</TD></TR><TR class="row6"><TD class="col0">6</TD><TD class="col1">CgeA 044 + 045</TD><TD class="col2">GST 036 + 038</TD><TD class="col3">pSB1C3</TD><TD class="col4">pIG16_075</TD></TR><TR class="row7"><TD class="col0">7</TD><TD class="col1">CgeA 050 + 035</TD><TD class="col2">GST 037 + 055</TD><TD class="col3">pSB1C3</TD><TD class="col4">pIG16_076</TD></TR><TR class="row8"><TD class="col0">8</TD><TD class="col1">CotB 019 + 020</TD><TD class="col2">GST 037 + 055</TD><TD class="col3">pSB1C3</TD><TD class="col4">pIG16_052</TD></TR><TR class="row9"><TD class="col0">9</TD><TD class="col1">CotB 022 + 024</TD><TD class="col2">aGFPnano 030 + 047</TD><TD class="col3">pSB1C3</TD><TD class="col4">pIG16_055</TD></TR><TR class="row10"><TD class="col0">10</TD><TD class="col1">CotB 023 024</TD><TD class="col2">GST 036 + 038</TD><TD class="col3">pSB1C3</TD><TD class="col4">pIG16_073</TD></TR><TR class="row11"><TD class="col0">11</TD><TD class="col1">CotB 019 + 021</TD><TD class="col2">GST 037 + 055</TD><TD class="col3">pSB1C3</TD><TD class="col4">pIG16_074</TD></TR><TR class="row12"><TD class="col0">12</TD><TD class="col1">positive control</TD><TD class="col2">positive control</TD><TD class="col3">positive control</TD><TD class="col4">-</TD></TR></TBODY></TABLE></DIV><P><STRONG>4) Linearization</STRONG> Linearization of integration vectors for transformation of B.subtilis: 2 µg of the DNA was treated with 20 units of an appropriate enzyme in 1X cutsmart buffer at a total reaction volume of 50 µL for 2 hours at 37 °C. </P><DIV class="table sectionedit149"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0">Vector pIG16_</TH><TH class="col1">Containing coat gene</TH><TH class="col2">Insert locus</TH><TH class="col3">Enzyme for linearization</TH></TR><TR class="row1"><TD class="col0">077</TD><TD class="col1">CotZ</TD><TD class="col2">AmyE</TD><TD class="col3">XhoI</TD></TR><TR class="row2"><TD class="col0">081</TD><TD class="col1">CotZ</TD><TD class="col2">AmyE</TD><TD class="col3">XhoI</TD></TR><TR class="row3"><TD class="col0">082</TD><TD class="col1">CotZ</TD><TD class="col2">AmyE</TD><TD class="col3">XhoI</TD></TR><TR class="row4"><TD class="col0">100</TD><TD class="col1">CgeA</TD><TD class="col2">AmyE</TD><TD class="col3">XhoI</TD></TR><TR class="row5"><TD class="col0">045</TD><TD class="col1">CgeA</TD><TD class="col2">AmyE</TD><TD class="col3">XhoI</TD></TR><TR class="row6"><TD class="col0">062</TD><TD class="col1">CgeA</TD><TD class="col2">AmyE</TD><TD class="col3">XhoI</TD></TR><TR class="row7"><TD class="col0">063</TD><TD class="col1">CgeA</TD><TD class="col2">AmyE</TD><TD class="col3">XhoI</TD></TR><TR class="row8"><TD class="col0">064</TD><TD class="col1">CotG</TD><TD class="col2">AmyE</TD><TD class="col3">XhoI</TD></TR><TR class="row9"><TD class="col0">108</TD><TD class="col1">CotG</TD><TD class="col2">thrC</TD><TD class="col3">NcoI</TD></TR><TR class="row10"><TD class="col0">123</TD><TD class="col1">mCherry</TD><TD class="col2">thrC</TD><TD class="col3">ScaI</TD></TR><TR class="row11"><TD class="col0">104</TD><TD class="col1">CotZ</TD><TD class="col2">thrC</TD><TD class="col3">NcoI</TD></TR><TR class="row12"><TD class="col0">105</TD><TD class="col1">CotZ</TD><TD class="col2">thrC</TD><TD class="col3">NcoI</TD></TR><TR class="row13"><TD class="col0">117</TD><TD class="col1">CgeA</TD><TD class="col2">thrC</TD><TD class="col3">NcoI</TD></TR><TR class="row14"><TD class="col0">101</TD><TD class="col1">CgeA</TD><TD class="col2">AmyE</TD><TD class="col3">XhoI</TD></TR></TBODY></TABLE></DIV><P> After linearization the DNA was purified using the QiaGen PCR purification kit. The DNA was used for transformation of B. subtilis. </P></DIV></DIV></DIV><DIV class="color3"><DIV class="para_20"><H3 class="sectionedit150"><A name="section310816" id="section310816">31.08.16</A></H3><DIV class="level3"><P><STRONG>1) Glycerolstocks </STRONG> 15 % Glycerol stocks of pIG16_032 033 034 035 were prepared and stored at -80 °C. </P><P><STRONG>2) Sequencing</STRONG> The following samples were sent to sequencing by GATC labtech pIG16_044#4, 078#4 080#3 086#2 089#4 109#4
                    </P><P><STRONG>3) Plasmid preparation</STRONG> The following plasmids were prepared using the QiaQuick Miniprep kit: pIG16_032 33 34 35 45 62 63 64 123 117 81 82 100 117 123 121 122 101
                    </P><P><STRONG>4) Gibson assembly </STRONG> The fragments were adjusted to a concentration of 50 ng/µL </P><DIV class="table sectionedit151"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0">No</TH><TH class="col1">amplified Fragment 1</TH><TH class="col2">amplified Fragment 2</TH><TH class="col3">Backbone</TH><TH class="col4">Resulting plasmids pIG16_</TH></TR><TR class="row1"><TD class="col0">1</TD><TD class="col1">CotG oIG16_056+013</TD><TD class="col2">aGFPnano 51 + 42</TD><TD class="col3">pSB1C3</TD><TD class="col4">043</TD></TR><TR class="row2"><TD class="col0">2</TD><TD class="col1">CotZ oIG16_7+8</TD><TD class="col2">GST_37+55 </TD><TD class="col3">pSB1C3</TD><TD class="col4">067</TD></TR><TR class="row3"><TD class="col0">3</TD><TD class="col1">CotG 056+013 </TD><TD class="col2">aGFPnano 051 + 042</TD><TD class="col3">pSB1C3</TD><TD class="col4">052</TD></TR><TR class="row4"><TD class="col0">4</TD><TD class="col1">CotG 015 + 016</TD><TD class="col2">GST 036 + 038</TD><TD class="col3">pSB1C3</TD><TD class="col4">055</TD></TR><TR class="row5"><TD class="col0">5</TD><TD class="col1">cotG 056 + 013</TD><TD class="col2">GST 037 + 055</TD><TD class="col3">pSB1C3</TD><TD class="col4">070</TD></TR><TR class="row6"><TD class="col0">6</TD><TD class="col1">cgeA 044 + 045</TD><TD class="col2">GST 036 + 038</TD><TD class="col3">pSB1C3</TD><TD class="col4">068</TD></TR><TR class="row7"><TD class="col0">7</TD><TD class="col1">cgeA 050 + 035</TD><TD class="col2">GST 037 + 055</TD><TD class="col3">pSB1C3</TD><TD class="col4">069</TD></TR><TR class="row8"><TD class="col0">8</TD><TD class="col1">cotB 019 + 020</TD><TD class="col2">GST 037 + 055</TD><TD class="col3">pSB1C3</TD><TD class="col4">073</TD></TR><TR class="row9"><TD class="col0">9</TD><TD class="col1">cotB 002 + 024</TD><TD class="col2">aGFPnano 030 + 047</TD><TD class="col3">pSB1C3</TD><TD class="col4">074</TD></TR><TR class="row10"><TD class="col0">10</TD><TD class="col1">cotB 023 + 024</TD><TD class="col2">GST 036 + 038</TD><TD class="col3">pSB1C3</TD><TD class="col4">075</TD></TR><TR class="row11"><TD class="col0">11</TD><TD class="col1">cotB 019 + 021</TD><TD class="col2">GST 037 + 055</TD><TD class="col3">pSB1C3</TD><TD class="col4">076</TD></TR></TBODY></TABLE></DIV><P> 0.5 µL of each fragment were mixed alongside with 0.5 µL of the backbone. The volume was adjusted to 2.5 µL and added to 7.5 µL of 1.33X Gibson Master Mix. The Reaction was incubated for 1h at 50 min and transformed into competent E. coli DH5alpha. The transformed cells were plated on LB agar plates supplemented with chloramphenicol. </P></DIV><H3 class="sectionedit152"><A name="section010916" id="section010916">01.09.16</A></H3><DIV class="level3"><P><STRONG>1) Inoculation </STRONG> Inoculation of the colonies from the transformation of the Gibson assembly: pIG16_043 067 052 055 070 068 069 073 074 075 076. 5 colonies were picked and inoculated in 5 mL of LB medium with chloramphenicol. Incubation o/n at 37°C and 200 rpm.
                    </P><P> Inoculation of pIG16_043 070 074 067 068 075 073 055 076 052 069 in 5 mL LB medium supplemented with ampicilin. </P></DIV><H3 class="sectionedit153"><A name="section020916" id="section020916">02.09.16</A></H3><DIV class="level3"><P><STRONG>1) Plasmid Preparation</STRONG> The following samples were prepared from E.coli DH5alpha by a MiniPrep:
                    <STRONG>I)</STRONG> From Gibson assembly (Previous day): pIG16_043 067 052 055 070 068 069 073 074 075 076
                    </P><P><STRONG>II)</STRONG> For Transformation of B.subtilis: pIG16_077 81 82 100 45 62 63 64 108 123 104 105 117 101 122 </P><P><STRONG>2) Linearization of plasmids for transformation of B.subtilis</STRONG> Linearization of integration vectors for transformation of B.subtilis: 2 µg of the DNA was treated with 20 units of an appropriate enzyme in 1X cutsmart buffer at a total reaction volume of 50 µL for 2 hours at 37 °C. </P><DIV class="table sectionedit154"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0">Vector pIG16_</TH><TH class="col1">Containing coat gene</TH><TH class="col2">Insert locus</TH><TH class="col3">Enzyme for linearization</TH></TR><TR class="row1"><TD class="col0">077</TD><TD class="col1">CotZ</TD><TD class="col2">AmyE</TD><TD class="col3">XhoI</TD></TR><TR class="row2"><TD class="col0">081</TD><TD class="col1">CotZ</TD><TD class="col2">AmyE</TD><TD class="col3">XhoI</TD></TR><TR class="row3"><TD class="col0">082</TD><TD class="col1">CotZ</TD><TD class="col2">AmyE</TD><TD class="col3">XhoI</TD></TR><TR class="row4"><TD class="col0">100</TD><TD class="col1">CgeA</TD><TD class="col2">AmyE</TD><TD class="col3">XhoI</TD></TR><TR class="row5"><TD class="col0">045</TD><TD class="col1">CgeA</TD><TD class="col2">AmyE</TD><TD class="col3">XhoI</TD></TR><TR class="row6"><TD class="col0">062</TD><TD class="col1">CgeA</TD><TD class="col2">AmyE</TD><TD class="col3">XhoI</TD></TR><TR class="row7"><TD class="col0">063</TD><TD class="col1">CgeA</TD><TD class="col2">AmyE</TD><TD class="col3">XhoI</TD></TR><TR class="row8"><TD class="col0">064</TD><TD class="col1">CotG</TD><TD class="col2">AmyE</TD><TD class="col3">XhoI</TD></TR><TR class="row9"><TD class="col0">108</TD><TD class="col1">CotG</TD><TD class="col2">thrC</TD><TD class="col3">NcoI</TD></TR><TR class="row10"><TD class="col0">123</TD><TD class="col1">mCherry</TD><TD class="col2">thrC</TD><TD class="col3">ScaI</TD></TR><TR class="row11"><TD class="col0">104</TD><TD class="col1">CotZ</TD><TD class="col2">thrC</TD><TD class="col3">NcoI</TD></TR><TR class="row12"><TD class="col0">105</TD><TD class="col1">CotZ</TD><TD class="col2">thrC</TD><TD class="col3">NcoI</TD></TR><TR class="row13"><TD class="col0">117</TD><TD class="col1">CgeA</TD><TD class="col2">thrC</TD><TD class="col3">NcoI</TD></TR><TR class="row14"><TD class="col0">101</TD><TD class="col1">CgeA</TD><TD class="col2">AmyE</TD><TD class="col3">XhoI</TD></TR><TR class="row15"><TD class="col0">122</TD><TD class="col1">mCherry</TD><TD class="col2">LacA</TD><TD class="col3">NgoMIV</TD></TR></TBODY></TABLE></DIV><P> After linearization the DNA was purified using the QiaGen PCR purification kit. The DNA was used for transformation of B. subtilis. </P></DIV></DIV></DIV><DIV class="color4"><DIV class="para_20"><H3 class="sectionedit155"><A name="section030916" id="section030916">03.09.16</A></H3><DIV class="level3"><P><STRONG>1) Test digestion</STRONG> For the verification of the proper sizes 500 ng the following plasmids were treated with 5 U of XbaI and PstI and analyzed by gel electrophoresis on a 1 % Agarose TAE gel:
                     pIG16_043 067 052 055 070 068 069 073 074 075 076
                     The expected bands were not visible.
                    </P></DIV><H3 class="sectionedit156"><A name="section040916" id="section040916">04.09.16</A></H3><DIV class="level3"><P><STRONG>1) Annealed oligo cloning</STRONG> In order to insert an alpha helical linker into the Sporovector from the Munich iGEM team 2012 the oligos oIG16_079 and oIG16_080 were annealed at a molar ratio of 1:1, heated at 100 °C for 30 min and slowly cooled down. The annealed oligos contained NgoMIV corresponding sticky ends and were stored at 4 °C. 2 µg of the vector pIG16_017 (Sporovector) was linearized with NgoMIV and purified. The linearized backbone and the annealed oligos were ligated at molar ratios of 1:1, 1:2 and 1:3 using 1 µL of T4 ligase (from NEB) in 1X T4 buffer in a total reaction volume of 20 µL. The reaction was incubated for 30 min at RT and transformed into chemically competent E.coli. The cells were plated on LB agar plates supplemented with ampicilin. </P></DIV><H3 class="sectionedit157"><A name="section050916" id="section050916">05.09.16</A></H3><DIV class="level3"><P><STRONG>1)Inoculation</STRONG> Inoculation of E.coli containing integration vectors from glycerol stocks: pIG16_077 81 82 100 45 62 63 64 108 123 104 105 117 101 122. Incubation in 5 mL LB medium supplemented with ampicilin. Incubation o/n at 37 °C and 200 rpm. </P><P> Inoculation of colonies from the annealed oligo cloning. 5 colonies per plate were inoculated in 5 mL of LB medium with ampicilin. Incubation o/n at 37 °C and 200 rpm. </P></DIV></DIV></DIV><DIV class="color5"><DIV class="para_20"><H3 class="sectionedit158"><A name="section060916" id="section060916">06.09.16</A></H3><DIV class="level3"><P><STRONG>1) Plasmid preparation</STRONG> MiniPrep of integration vectors: pIG16_077 81 82 100 45 62 63 64 108 123 104 105 117 101 122 </P><P> pIG16_124 (at molar ratios 1:1, 1:2, 1:3,each #1-5). </P><DIV class="notewarning">Gel image: NIKLAS??? </DIV><P><STRONG>2) Sequencing</STRONG> pIG16_124 1:1 #2
                     pIG16_124 1:2 #3
                     pIG16_124 1:3 #1
                    </P></DIV><H3 class="sectionedit159"><A name="section070916" id="section070916">07.09.16</A></H3><DIV class="level3"><P><STRONG>1) Plasmid Preparation</STRONG> MiniPrep of further colonies of pIG16_043 067 052 055 070 068 069 073 074 075 076 </P><P><STRONG>2) Testdigestion</STRONG> 500 ng of DNA was treated with 5 units of XbaI + PstI in 1X NEBuffer 2.1 in a total reaction volume of 20 µL. Incubation for 1 h at 37 °C. The reaction was analyzed by gel electrophoresis. </P></DIV><H3 class="sectionedit160"><A name="section080916" id="section080916">08.09.16</A></H3><DIV class="level3"><P><STRONG>1) Sequencing</STRONG> Gibson assembly reactions were sent to sequencing:
                     pIG16_043 067 052 055 070 068 069 073 074 075 076 </P><P><STRONG>2) Digestion</STRONG><STRONG>I</STRONG> pIG16_017: 2 µg of plasmid DNA was treated with 10 unit of XbaI and NgoMIV in 1X Cutsmart buffer in a total reaction volume of 50 µL for 2 hours at 37 °C. For further assembly by Freiburg Standard RFC25.
                    <STRONG>II</STRONG> pIG16_020: 2 µg of plasmid DNA was treated with 10 unit of XbaI and SpeI-HF in 1X Cutsmart buffer in a total reaction volume of 50 µL for 2 hours at 37 °C for further gibson assembliy reactions.
                    </P><P><STRONG>3) PCR</STRONG> The biobrick BBa_J18932 (pIG16_120) containing mCherry was amplified in include additional restriction site for Freiburg standard assembly RFC25.
                    <STRONG>Reaction conditions</STRONG></P><DIV class="table sectionedit161"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0">Component</TH><TH class="col1">concentration</TH><TH class="col2">volume in µL</TH></TR><TR class="row1"><TD class="col0">Q5 Master Mix</TD><TD class="col1">2X</TD><TD class="col2">37.5</TD></TR><TR class="row2"><TD class="col0">DNA template</TD><TD class="col1">-</TD><TD class="col2">1 µL</TD></TR><TR class="row3"><TD class="col0">oIG16_075</TD><TD class="col1">10 µM</TD><TD class="col2">3.75</TD></TR><TR class="row4"><TD class="col0">oIG16_076</TD><TD class="col1">10 µM</TD><TD class="col2">3.75</TD></TR><TR class="row5"><TD class="col0">dH2O</TD><TD class="col1">-</TD><TD class="col2">29</TD></TR></TBODY></TABLE></DIV><P> Amplified by Touchdown-PCR. The annealing temperature was determined by the NEB online tool Tm-calculator. </P></DIV></DIV></DIV><DIV class="color6"><DIV class="para_20"><H3 class="sectionedit162"><A name="section090916" id="section090916">09.09.16</A></H3><DIV class="level3"><P><STRONG>1) Gel extraction</STRONG> The digestions and PCR reaction from the previous day were extracted from an agarose gel accorind the the protocol of the manufacturer. </P><P><STRONG>2) Digestion</STRONG> 1 µg of the amplified mCherry (pIG16_120 + oIG16_075+076) was treated with AgeI and XbaI in 1X cutsmart buffer in a total reaction volume of 50 µL for 90 min at 37 °C. The DNA was directly purified using the Qiagen PCR purification kit according to the instructions of the manufacturer. </P><P><STRONG>3) T4 Ligation</STRONG> Ligation of digested mCherry (Insert) into the digested pIG16_017 (Sporovector) at a molar ratio of 1:3 (Vector : Insert) in 1X T4-ligation buffer at a total reaction volume of 20 µL. The ligation was incubated for 30 min at RT and transformed into chemically competent E.coli. </P><P><STRONG>4) Gibson Assembly</STRONG></P><DIV class="table sectionedit163"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0">No</TH><TH class="col1">amplified Fragment 1</TH><TH class="col2">amplified Fragment 2</TH><TH class="col3">Backbone</TH><TH class="col4">Resulting plasmids pIG16_</TH></TR><TR class="row1"><TD class="col0">1</TD><TD class="col1">CotZ oIG16_7+8</TD><TD class="col2">GST_37+55 </TD><TD class="col3">pSB1C3</TD><TD class="col4">067</TD></TR><TR class="row2"><TD class="col0">2</TD><TD class="col1">cgeA 044 + 045</TD><TD class="col2">GST 036 + 038</TD><TD class="col3">pSB1C3</TD><TD class="col4">068</TD></TR><TR class="row3"><TD class="col0">3</TD><TD class="col1">cotB 019 + 020</TD><TD class="col2">GST 037 + 055</TD><TD class="col3">pSB1C3</TD><TD class="col4">073</TD></TR><TR class="row4"><TD class="col0">4</TD><TD class="col1">cotB 002 + 024</TD><TD class="col2">aGFPnano 030 + 047</TD><TD class="col3">pSB1C3</TD><TD class="col4">074</TD></TR><TR class="row5"><TD class="col0">5</TD><TD class="col1">cotB 023 + 024</TD><TD class="col2">GST 036 + 038</TD><TD class="col3">pSB1C3</TD><TD class="col4">075</TD></TR><TR class="row6"><TD class="col0">6</TD><TD class="col1">cotB 019 + 021</TD><TD class="col2">GST 037 + 055</TD><TD class="col3">pSB1C3</TD><TD class="col4">076</TD></TR><TR class="row7"><TD class="col0">7</TD><TD class="col1">pos. control</TD><TD class="col2">positive control</TD><TD class="col3">positive control</TD><TD class="col4">-</TD></TR></TBODY></TABLE></DIV><P> The reaction was incubated for 1 h at 50 °C and tranformed into chemically competent E.coli DH5 alpha. </P></DIV><H3 class="sectionedit164"><A name="section100916" id="section100916">10.09.16</A></H3><DIV class="level3"><P><STRONG>1) Inoculation</STRONG><STRONG>I</STRONG> From each plate with E.coli transformed with the gibson assemblies 5 colonies were picked and inoculated in 5 mL of LB medium supplemented with chloramphenicol.
                    <STRONG>II</STRONG> Further colonies of pIG16_043, 052, 126 were picked and inoculated. </P></DIV><H3 class="sectionedit165"><A name="section110916" id="section110916">11.09.16</A></H3><DIV class="level3"><P><STRONG>1) Linearization</STRONG> 3.5 µg of the following plasmids were linearized with the listed restriction enzyme in the appropriate buffer for transformation into B. subtilis: </P><DIV class="table sectionedit166"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0">Plasmid pIG16_</TH><TH class="col1">coat gene</TH><TH class="col2">Insert Locus/Selection marker</TH><TH class="col3">Enzyme</TH></TR><TR class="row1"><TD class="col0">044</TD><TD class="col1">cgeA</TD><TD class="col2">1C</TD><TD class="col3">XhoI</TD></TR><TR class="row2"><TD class="col0">045</TD><TD class="col1">cgeA</TD><TD class="col2">1C</TD><TD class="col3">XhoI</TD></TR><TR class="row3"><TD class="col0">062</TD><TD class="col1">cgeA</TD><TD class="col2">1C</TD><TD class="col3">XhoI</TD></TR><TR class="row4"><TD class="col0">063</TD><TD class="col1">cgeA</TD><TD class="col2">1C</TD><TD class="col3">XhoI</TD></TR><TR class="row5"><TD class="col0">064</TD><TD class="col1">cotG</TD><TD class="col2">1C</TD><TD class="col3">XhoI</TD></TR><TR class="row6"><TD class="col0">077</TD><TD class="col1">cotZ</TD><TD class="col2">1C</TD><TD class="col3">XhoI</TD></TR><TR class="row7"><TD class="col0">078</TD><TD class="col1">cotZ</TD><TD class="col2">1C</TD><TD class="col3">XhoI</TD></TR><TR class="row8"><TD class="col0">079</TD><TD class="col1">cotZ</TD><TD class="col2">1C</TD><TD class="col3">XhoI</TD></TR><TR class="row9"><TD class="col0">080</TD><TD class="col1">cotZ</TD><TD class="col2">1C</TD><TD class="col3">XhoI</TD></TR><TR class="row10"><TD class="col0">081</TD><TD class="col1">cotZ</TD><TD class="col2">1C</TD><TD class="col3">XhoI</TD></TR><TR class="row11"><TD class="col0">082</TD><TD class="col1">cotZ</TD><TD class="col2">1C</TD><TD class="col3">XhoI</TD></TR><TR class="row12"><TD class="col0">085</TD><TD class="col1">cotG</TD><TD class="col2">1C</TD><TD class="col3">XhoI</TD></TR><TR class="row13"><TD class="col0">086</TD><TD class="col1">cotG</TD><TD class="col2">1C</TD><TD class="col3">XhoI</TD></TR><TR class="row14"><TD class="col0">088</TD><TD class="col1">cotG</TD><TD class="col2">1C</TD><TD class="col3">XhoI</TD></TR><TR class="row15"><TD class="col0">089</TD><TD class="col1">cotG</TD><TD class="col2">1C</TD><TD class="col3">XhoI</TD></TR><TR class="row16"><TD class="col0">096</TD><TD class="col1">cotB</TD><TD class="col2">1C</TD><TD class="col3">XhoI</TD></TR><TR class="row17"><TD class="col0">100</TD><TD class="col1">cgeA</TD><TD class="col2">1C</TD><TD class="col3">XhoI</TD></TR><TR class="row18"><TD class="col0">101</TD><TD class="col1">cgeA</TD><TD class="col2">1C</TD><TD class="col3">XhoI</TD></TR><TR class="row19"><TD class="col0">104</TD><TD class="col1">cotZ</TD><TD class="col2">4S</TD><TD class="col3">NcoI</TD></TR><TR class="row20"><TD class="col0">105</TD><TD class="col1">cotZ</TD><TD class="col2">4S</TD><TD class="col3">NcoI</TD></TR><TR class="row21"><TD class="col0">108</TD><TD class="col1">cotG</TD><TD class="col2">4S</TD><TD class="col3">NcoI</TD></TR><TR class="row22"><TD class="col0">109</TD><TD class="col1">cotG</TD><TD class="col2">4S</TD><TD class="col3">NcoI</TD></TR><TR class="row23"><TD class="col0">117</TD><TD class="col1">cgeA</TD><TD class="col2">4S</TD><TD class="col3">NcoI</TD></TR><TR class="row24"><TD class="col0">122</TD><TD class="col1">mCherry</TD><TD class="col2">2E</TD><TD class="col3">NgoMIV</TD></TR><TR class="row25"><TD class="col0">123</TD><TD class="col1">mCherry</TD><TD class="col2">4S</TD><TD class="col3">ScaI</TD></TR></TBODY></TABLE></DIV><P> After linearization 5 µL of the reaction was analyzed by agarose gel electrophoresis. The remaining volume was purified using the QiaGen PCR purification kit. The eluted DNA was used for transformation of B. subtilis. </P><P><STRONG>2) MiniPrep</STRONG> The plasmids from the following inoculations were prepared using the ZymoResearch Zyppy Prep kit:
             pIG16_043 052 067 068 070 073 074 075 076 126 (colonies #5-9 each). </P><P><STRONG>3) Testdigestion</STRONG> 500 ng of the prepared plasmids were treated with 5 units of XbaI and PstI in 1X NEBuffer 2.1 in a total reaction volume of 20 µL for 2 hours at 37 °C. The Digestion was analyzed by agarose gel electrophoresis.
                    </P></DIV></DIV></DIV><DIV class="color7"><DIV class="para_20"><H3 class="sectionedit167"><A name="section120916" id="section120916">12.09.16</A></H3><H3 class="sectionedit168"><A name="section130916" id="section130916">13.09.16</A></H3><DIV class="level3"><P><STRONG>1) PCR</STRONG> Amplification of the Sporovector pSBBS4S-Sporo (pIG16_017) in order to introduce an alpha helical linker to the MCS by Gibson assembly. </P><DIV class="table sectionedit169"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0">template</TH><TH class="col1">Oligos oIG16_</TH></TR><TR class="row1"><TD class="col0">pIG16_017</TD><TD class="col1">079+082</TD></TR><TR class="row2"><TD class="col0">pIG16_017</TD><TD class="col1">081+080</TD></TR></TBODY></TABLE></DIV><P><STRONG>Reaction conditions</STRONG></P><DIV class="table sectionedit170"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0">component</TH><TH class="col1">conc.</TH><TH class="col2">volume</TH></TR><TR class="row1"><TD class="col0">Q5 MasterMix</TD><TD class="col1">2x</TD><TD class="col2">25µL</TD></TR><TR class="row2"><TD class="col0">Primer fw</TD><TD class="col1">10 µM</TD><TD class="col2">2.5</TD></TR><TR class="row3"><TD class="col0">Primer rv</TD><TD class="col1">10 µM</TD><TD class="col2">2.5</TD></TR><TR class="row4"><TD class="col0">template</TD><TD class="col1">-</TD><TD class="col2">0.1</TD></TR><TR class="row5"><TD class="col0">dH20</TD><TD class="col1">-</TD><TD class="col2">20µL</TD></TR></TBODY></TABLE></DIV><P><STRONG>Cycling</STRONG> Touchdown PCR 60 and 67 </P><P><STRONG>2) Gibson Assembly</STRONG> The two amplified fragments were mixed at a ratio of 1:1 (50 ng each) and incuabted in NEB HiFi assembly mix at 50 °C for 1 hour. Transformation of 2µL into chemically competent E.coli and spreaded on LB agar plates supplemented with ampicilin. </P></DIV><H3 class="sectionedit171"><A name="section140916" id="section140916">14.09.16</A></H3><DIV class="level3"><P><STRONG>1) Linearization</STRONG> 3.5 µg of the following plasmids were linearized with the listed restriction enzyme in the appropriate buffer for transformation into B. subtilis: </P><DIV class="table sectionedit172"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0">Plasmid pIG16_</TH><TH class="col1">coat gene</TH><TH class="col2">Insert Locus/Selection marker</TH><TH class="col3">Enzyme</TH></TR><TR class="row1"><TD class="col0">044</TD><TD class="col1">cgeA</TD><TD class="col2">1C</TD><TD class="col3">XhoI</TD></TR><TR class="row2"><TD class="col0">045</TD><TD class="col1">cgeA</TD><TD class="col2">1C</TD><TD class="col3">XhoI</TD></TR><TR class="row3"><TD class="col0">062</TD><TD class="col1">cgeA</TD><TD class="col2">1C</TD><TD class="col3">XhoI</TD></TR><TR class="row4"><TD class="col0">063</TD><TD class="col1">cgeA</TD><TD class="col2">1C</TD><TD class="col3">XhoI</TD></TR><TR class="row5"><TD class="col0">064</TD><TD class="col1">cotG</TD><TD class="col2">1C</TD><TD class="col3">XhoI</TD></TR><TR class="row6"><TD class="col0">077</TD><TD class="col1">cotZ</TD><TD class="col2">1C</TD><TD class="col3">XhoI</TD></TR><TR class="row7"><TD class="col0">078</TD><TD class="col1">cotZ</TD><TD class="col2">1C</TD><TD class="col3">XhoI</TD></TR><TR class="row8"><TD class="col0">079</TD><TD class="col1">cotZ</TD><TD class="col2">1C</TD><TD class="col3">XhoI</TD></TR><TR class="row9"><TD class="col0">080</TD><TD class="col1">cotZ</TD><TD class="col2">1C</TD><TD class="col3">XhoI</TD></TR><TR class="row10"><TD class="col0">081</TD><TD class="col1">cotZ</TD><TD class="col2">1C</TD><TD class="col3">XhoI</TD></TR><TR class="row11"><TD class="col0">082</TD><TD class="col1">cotZ</TD><TD class="col2">1C</TD><TD class="col3">XhoI</TD></TR><TR class="row12"><TD class="col0">085</TD><TD class="col1">cotG</TD><TD class="col2">1C</TD><TD class="col3">XhoI</TD></TR><TR class="row13"><TD class="col0">086</TD><TD class="col1">cotG</TD><TD class="col2">1C</TD><TD class="col3">XhoI</TD></TR><TR class="row14"><TD class="col0">088</TD><TD class="col1">cotG</TD><TD class="col2">1C</TD><TD class="col3">XhoI</TD></TR><TR class="row15"><TD class="col0">089</TD><TD class="col1">cotG</TD><TD class="col2">1C</TD><TD class="col3">XhoI</TD></TR><TR class="row16"><TD class="col0">096</TD><TD class="col1">cotB</TD><TD class="col2">1C</TD><TD class="col3">XhoI</TD></TR><TR class="row17"><TD class="col0">100</TD><TD class="col1">cgeA</TD><TD class="col2">1C</TD><TD class="col3">XhoI</TD></TR><TR class="row18"><TD class="col0">101</TD><TD class="col1">cgeA</TD><TD class="col2">1C</TD><TD class="col3">XhoI</TD></TR><TR class="row19"><TD class="col0">104</TD><TD class="col1">cotZ</TD><TD class="col2">4S</TD><TD class="col3">NcoI</TD></TR><TR class="row20"><TD class="col0">105</TD><TD class="col1">cotZ</TD><TD class="col2">4S</TD><TD class="col3">NcoI</TD></TR><TR class="row21"><TD class="col0">108</TD><TD class="col1">cotG</TD><TD class="col2">4S</TD><TD class="col3">NcoI</TD></TR><TR class="row22"><TD class="col0">109</TD><TD class="col1">cotG</TD><TD class="col2">4S</TD><TD class="col3">NcoI</TD></TR><TR class="row23"><TD class="col0">117</TD><TD class="col1">cgeA</TD><TD class="col2">4S</TD><TD class="col3">NcoI</TD></TR><TR class="row24"><TD class="col0">122</TD><TD class="col1">mCherry</TD><TD class="col2">2E</TD><TD class="col3">NgoMIV</TD></TR><TR class="row25"><TD class="col0">123</TD><TD class="col1">mCherry</TD><TD class="col2">4S</TD><TD class="col3">ScaI</TD></TR></TBODY></TABLE></DIV><P> After linearization 5 µL of the reaction was analyzed by agarose gel electrophoresis. The remaining volume was purified using the QiaGen PCR purification kit. The eluted DNA was used for transformation of B. subtilis. </P><DIV class="notewarning">update: Gel image </DIV></DIV></DIV></DIV><DIV class="color8"><DIV class="para_20"><H3 class="sectionedit173"><A name="section150916" id="section150916">15.09.16</A></H3><DIV class="level3"><P><STRONG>1) PCR</STRONG> Amplification of anti-GFP nanobody and mCherry to introduce additional overhangs containing restriction site. </P><DIV class="table sectionedit174"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0">template</TH><TH class="col1">Oligos</TH><TH class="col2">Restriction site</TH></TR><TR class="row1"><TD class="col0">1</TD><TD class="col1">pIG16_023</TD><TD class="col2">oIG16_083 + 084</TD><TD class="col3">EcoRI + XbaI</TD></TR><TR class="row2"><TD class="col0">2</TD><TD class="col1">pIG16_023</TD><TD class="col2">oIG16_085 + 086</TD><TD class="col3">NcoI + NheI</TD></TR><TR class="row3"><TD class="col0">3</TD><TD class="col1">pIG16_020 (mCherry)</TD><TD class="col2">oIG16_087 + 088</TD><TD class="col3">XhoI + XbaI</TD></TR></TBODY></TABLE></DIV><P><STRONG>Reaction conditions</STRONG></P><DIV class="table sectionedit175"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0">component</TH><TH class="col1">conc</TH><TH class="col2">volume [µL]</TH></TR><TR class="row1"><TD class="col0">Q5 MasterMix</TD><TD class="col1">2x</TD><TD class="col2">25</TD></TR><TR class="row2"><TD class="col0">Primer 1</TD><TD class="col1">10 µM</TD><TD class="col2">2.5</TD></TR><TR class="row3"><TD class="col0">Primer 2</TD><TD class="col1">10 µM</TD><TD class="col2">2.5</TD></TR><TR class="row4"><TD class="col0">template</TD><TD class="col1">-</TD><TD class="col2">0.1</TD></TR><TR class="row5"><TD class="col0">dH2O</TD><TD class="col1">-</TD><TD class="col2">20</TD></TR></TBODY></TABLE></DIV><P><STRONG>Cycling</STRONG> pIG16_023: Touchdown68
                     pIG16_020: Touchdown66
                    </P><P> After amplification the reaction was analyzed by agarose gel electrophoresis. </P><P><STRONG>2) Transformation</STRONG> BBa_K180000 (Lac regulated taq promoter) from the iGEM distribution kit was dissolved in 10 µL of ultra pure water. 2 µL were used for the transformation of chemically competent E.coli DH5alpha and spread on an LB agar plate supplemented with chloramphenicol. </P><P><STRONG>3) Oligo design</STRONG> The PCotYZ-Promoter (BBa_K823030) did not contain a ribosome binding site. An additional RBS is introduced by primers containing an additional insert. </P></DIV><H3 class="sectionedit176"><A name="section160916" id="section160916">16.09.16</A></H3><DIV class="level3"><P><STRONG>1) Gel extraction</STRONG> The amplified samples from the previous day were extracted from the gel and purified using the QiaGen gel extraction kit. </P><P><STRONG>2) Digestion</STRONG> 1µg of the amplified nanobody/mCherry were digested with the 10 unit of the appropriate enzymes in 1X Reaction buffer in a total reaction volume of 30 µL in order to clone them into their respective backbone. </P><DIV class="table sectionedit177"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0">DNA</TH><TH class="col1">oligos</TH><TH class="col2">Restriction enzymes</TH></TR><TR class="row1"><TD class="col0">aGFPnano</TD><TD class="col1">83 + 84</TD><TD class="col2">EcoRI + XbaI</TD></TR><TR class="row2"><TD class="col0">aGFPnano</TD><TD class="col1">85 + 86</TD><TD class="col2">NcoI-HF + NheI-HF</TD></TR><TR class="row3"><TD class="col0">mCherry</TD><TD class="col1">87 + </TD><TD class="col2">XhoI + XbaI</TD></TR><TR class="row4"><TD class="col0">pIG16_033</TD><TD class="col1">-</TD><TD class="col2">NcoI-HF + NheI-HF</TD></TR><TR class="row5"><TD class="col0">pIG16_034</TD><TD class="col1">-</TD><TD class="col2">XhoI + XbaI-HF</TD></TR><TR class="row6"><TD class="col0">pIG16_035</TD><TD class="col1">-</TD><TD class="col2">EcoRI-HF + XbaI-HF</TD></TR></TBODY></TABLE></DIV><P> The amplified inserts were directly purified using the Qiagen PCR purification kit. The digested backbones (pIG16_033, 034, 035) were loaded on an agarose gel. After electrophoresis the bands corresponding to the expected sizes were extracted. </P><P><STRONG>3) T4-Ligation</STRONG> The digested inserts and backbones were ligated at a molar ration of 1:3 (Backbone:Insert) using T4 ligase in 1X T4 ligation buffer in a total reaction volume of 20 µL. The reaction was incubated at RT for 20 min. </P><DIV class="table sectionedit178"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0">No.</TH><TH class="col1">Insert</TH><TH class="col2">Backbone</TH><TH class="col3" colspan="2">Resulting construct</TH></TR><TR class="row1"><TD class="col0">1</TD><TD class="col1">EcoRI-aGFPnano-XbaI</TD><TD class="col2">EcoRI-pIG16_035-XbaI</TD><TD class="col3">pIG16_128</TD></TR><TR class="row2"><TD class="col0">2</TD><TD class="col1">NcoI-aGFPnano-NheI</TD><TD class="col2">NcoI-pIG16_033-NheI</TD><TD class="col3">pIG16_127</TD></TR><TR class="row3"><TD class="col0">3</TD><TD class="col1">XhoI-mCherry-XbaI</TD><TD class="col2">XhoI-pIG16_034-XbaI</TD><TD class="col3">pIG16_129</TD></TR></TBODY></TABLE></DIV><P> 5 µL of the reaction mix was transformed into chemically competent E.coli DH5alpha and spread on LB agar plates supplemented with ampicilin. Incubation over night at 37 °C. </P><P><STRONG>4) Inoculation</STRONG> 5 colonies of pIG16_130 were inoculated in 5 mL of LB medium supplemented with chloramphenicol. </P></DIV><H3 class="sectionedit179"><A name="section170916" id="section170916">17.09.16</A></H3><DIV class="level3"><P><STRONG>1) MiniPrep</STRONG> The plasmids from the transformed E.coli containing pIG16_130#1-5 were prepared using the QiaGen MiniPrep kit. The DNA was eluted in 30 µL of ultra pure water.
                    </P><P><STRONG>2) Testdigestion</STRONG> 500 ng of pIG16_130 #1-5 was treated with 4 units of EcoRI-HF and SpeI-HF in 1X cutsmart buffer in a total reaction volume of 20 µL for 2 hours at 37 °C. </P><DIV class="notewarning">Gel image: NIKLAS??? </DIV><P><STRONG>3) Digestion</STRONG> 5 µg of pIG16_130 #2 was treated with 20 units of EcoRI-HF and SpeI-HF in 1X cutsmart buffer in a total reaction volume of 50 µL for 2 hours at 37 °C in order to release the biobrick part (Lac inducible promoter). The digestion was loaded on an agarose gel and subjected to electrophoresis. The band corresponding to the size of the promoter was extracted using the QiaGen gel extraction kit. </P></DIV></DIV></DIV><DIV class="color1"><DIV class="para_20"><H3 class="sectionedit180"><A name="section180916" id="section180916">18.09.16</A></H3><DIV class="level3"><P><STRONG>1) Restriction digestion</STRONG> 1.5 µg of pIG16_033 and 035 were treated with 5 unit of NcoI-HF+NheI-HF and EcoRI-HF+XbaI, respectively. The reaction was performed in 1X cutsmart buffer in a total reaction volume of 50 µL for 2 hours at 37 °C. After digestion the linearized plasmids were purified using the QiaGen PCR purification kit. </P><P><STRONG>2) T4-Ligation</STRONG></P><P> The digested inserts and backbones were ligated at a molar ration of 1:3 (Backbone:Insert) using T4 ligase in 1X T4 ligation buffer in a total reaction volume of 20 µL. The reaction was incubated at RT for 20 min. </P><DIV class="table sectionedit181"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0">Backbone</TH><TH class="col1">Insert</TH><TH class="col2">Resulting construct</TH></TR><TR class="row1"><TD class="col0">NcoI-pIG16_033-NheI</TD><TD class="col1">NcoI-aGFPnano-NheI</TD><TD class="col2">pIG16_127</TD></TR><TR class="row2"><TD class="col0">EcoRI-pIG16_035-XbaI</TD><TD class="col1">EcoRI-aGFPnano-XbaI</TD><TD class="col2">pIG16_128</TD></TR></TBODY></TABLE></DIV><P> 2 µL of the ligation mix was used for transformation of chemically competent E.coli DH5alpha. </P></DIV><H3 class="sectionedit182"><A name="section190916" id="section190916">19.09.16</A></H3><DIV class="level3"><P><STRONG>1) Inoculation</STRONG> 5 colonies of each plate containing the ligations from the previous day were picked and inoculated in 5 mL of LB medium supplemented with ampicilin. Incubation o/n at 37 °C and 200 rpm. </P></DIV><H3 class="sectionedit183"><A name="section200916" id="section200916">20.09.16</A></H3><DIV class="level3"><P><STRONG>1) MiniPrep</STRONG> Plasmid preparation of colonies pIG16_127#1-5 and pIG16_128#1-5 using the QiaGen MiniPrep kit. The DNA was eluted in 30 µL of ultra pure water.
                    </P><P><STRONG>2) Testdigestion</STRONG><STRONG>I</STRONG> 500 ng of pIG16_127 was treated with 4 units of NcoI-HF and NheI-HF in 1X Cutsmart buffer for 90 min at 37 °C. <STRONG>II</STRONG> 500 ng of pIG16_128 was treated with 4 units of EcoRI-HF and XbaI-HF. The digested DNA was analyzed by agarose gel electrophoresis. </P><P><STRONG>3) PCR</STRONG> Amplification of additional promoters for the expression of fusion consructs. </P><DIV class="table sectionedit184"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0">No.</TH><TH class="col1">Template</TH><TH class="col2">Oligos</TH><TH class="col3">comment</TH><TH class="col4">Annealing temperature [°C]</TH></TR><TR class="row1"><TD class="col0">1</TD><TD class="col1">pSB1C3 (pIG16_020)</TD><TD class="col2">oIG16_093+094</TD><TD class="col3">pTrpC-promoter from RFP-cassette</TD><TD class="col4">62</TD></TR><TR class="row2"><TD class="col0">2</TD><TD class="col1">pIG16_017</TD><TD class="col2">oIG16_091+092</TD><TD class="col3">PCotYZ-RBS</TD><TD class="col4">54</TD></TR><TR class="row3"><TD class="col0">3</TD><TD class="col1">pIG16_022</TD><TD class="col2">oIG16_095+096</TD><TD class="col3">BamHI-GST-XbaI</TD><TD class="col4">64</TD></TR></TBODY></TABLE></DIV><P><STRONG>Reaction conditions</STRONG></P><DIV class="table sectionedit185"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0">component</TH><TH class="col1">conc</TH><TH class="col2">volume [µL]</TH></TR><TR class="row1"><TD class="col0">Q5 MasterMix</TD><TD class="col1">2X</TD><TD class="col2">25</TD></TR><TR class="row2"><TD class="col0">template</TD><TD class="col1">-</TD><TD class="col2">0.1</TD></TR><TR class="row3"><TD class="col0">Primer 1</TD><TD class="col1">10 µM</TD><TD class="col2">2.5</TD></TR><TR class="row4"><TD class="col0">Primer 2</TD><TD class="col1">10 µM</TD><TD class="col2">2.5</TD></TR><TR class="row5"><TD class="col0">dH2O</TD><TD class="col1">-</TD><TD class="col2">20 µL</TD></TR></TBODY></TABLE></DIV><P> After amplification the samples were loaded on an agarose gel and extracted from the gel using the Qiagen gel extraction kit. </P><P><STRONG>4) Digestions </STRONG><STRONG>I</STRONG> 5 µg of integration vector DNA was treated with 20 units of BamHI-HF and XbaI in 1X cutsmart buffer in a total reaction volume of 20 µL for 2 hours at 37 °C. </P><DIV class="table sectionedit186"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0">No.</TH><TH class="col1">Vector</TH><TH class="col2">Enzymes</TH></TR><TR class="row1"><TD class="col0">1</TD><TD class="col1">pCgeA-C(pIG16_034)</TD><TD class="col2">BamHI-HF + XbaI</TD></TR><TR class="row2"><TD class="col0">2</TD><TD class="col1">pCotZ-C(pIG16_032)</TD><TD class="col2">BamHI-HF + XbaI</TD></TR><TR class="row3"><TD class="col0">3</TD><TD class="col1">GST(amplified from pIG16_022)</TD><TD class="col2">BamHI-HF + XbaI</TD></TR></TBODY></TABLE></DIV><P><STRONG>II</STRONG> For digestion of the amplified promoters 3 µg of extracted DNA was treated with the appropriate restriction enzymes in 1X reaction buffer in a total volume of 50 µL for 1 h at 37 °C.
                    </P><DIV class="table sectionedit187"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0">Template</TH><TH class="col1">Promoter</TH><TH class="col2">Restriction enzymes</TH></TR><TR class="row1"><TD class="col0">pIG16_017</TD><TD class="col1">PCotYZ-RBS</TD><TD class="col2">EcoRI-HF + SpeI-HF</TD></TR><TR class="row2"><TD class="col0">pIG16_020</TD><TD class="col1">PTrpC</TD><TD class="col2">EcoRI-HF + SpeI-HF</TD></TR></TBODY></TABLE></DIV><P> The digested backbones were loaded on an agarose gel and the bands corresponding to the expected size were extracted using the QiaGen gel extraction kit.
                   </P><P><STRONG>5) 3A Assembly</STRONG> The constructs for surface display were assembled using various promoters for expression in E.coli and B.subtilis. 3A assembly of a promoter and a fusion construct into an integration vector. All inserts were treated with the appropriate restriction enzymes from previous assemblies and stored at -20 °C. The ligation was performed at a molar ratio of 1:3 (vector:inserts) using 50 ng of vector DNA. The reaction was incubated at RT for 20 min and subsequently transformed into chemically competent E. coli DH5alpha and spread on LB-agar plates supplemented with the appropriate antibiotic.
                    <STRONG>I</STRONG> pLac promoter (inducible, BBa_K180000) </P><DIV class="table sectionedit188"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0">No.</TH><TH class="col1">Fusion construct</TH><TH class="col2">Result</TH></TR><TR class="row1"><TD class="col0">1</TD><TD class="col1">pIG16_047</TD><TD class="col2">pIG16_134</TD></TR><TR class="row2"><TD class="col0">2</TD><TD class="col1">pIG16_048</TD><TD class="col2">pIG16_135</TD></TR><TR class="row3"><TD class="col0">3</TD><TD class="col1">pIG16_049</TD><TD class="col2">pIG16_136</TD></TR><TR class="row4"><TD class="col0">4</TD><TD class="col1">pIG16_066</TD><TD class="col2">pIG16_138</TD></TR></TBODY></TABLE></DIV><P><STRONG>II</STRONG>pTrpC promoter (consitutive promoter, amplified from RFP-cassette) </P><DIV class="table sectionedit189"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0">No.</TH><TH class="col1">Fusion construct</TH><TH class="col2">Result</TH></TR><TR class="row1"><TD class="col0">1</TD><TD class="col1">pIG16_042</TD><TD class="col2">pIG16_143</TD></TR><TR class="row2"><TD class="col0">2</TD><TD class="col1">pIG16_047</TD><TD class="col2">pIG16_144</TD></TR><TR class="row3"><TD class="col0">3</TD><TD class="col1">pIG16_048</TD><TD class="col2">pIG16_145</TD></TR><TR class="row4"><TD class="col0">4</TD><TD class="col1">pIG16_049</TD><TD class="col2">pIG16_146</TD></TR><TR class="row5"><TD class="col0">5</TD><TD class="col1">pIG16_059</TD><TD class="col2">pIG16_147</TD></TR><TR class="row6"><TD class="col0">6</TD><TD class="col1">pIG16_066</TD><TD class="col2">pIG16_148</TD></TR><TR class="row7"><TD class="col0">7</TD><TD class="col1">pIG16_039</TD><TD class="col2">pIG16_149</TD></TR></TBODY></TABLE></DIV><P><STRONG>III</STRONG>pCotYZ-RBS (B. subtilis CotYZ-promoter with ribosome binding site) </P><DIV class="table sectionedit190"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0">No.</TH><TH class="col1">Fusion construct</TH><TH class="col2">Result</TH></TR><TR class="row1"><TD class="col0">1</TD><TD class="col1">pIG16_039</TD><TD class="col2">pIG16_150</TD></TR><TR class="row2"><TD class="col0">2</TD><TD class="col1">pIG16_042</TD><TD class="col2">pIG16_151</TD></TR><TR class="row3"><TD class="col0">3</TD><TD class="col1">pIG16_047</TD><TD class="col2">pIG16_152</TD></TR><TR class="row4"><TD class="col0">4</TD><TD class="col1">pIG16_048</TD><TD class="col2">pIG16_153</TD></TR><TR class="row5"><TD class="col0">5</TD><TD class="col1">pIG16_049</TD><TD class="col2">pIG16_154</TD></TR><TR class="row6"><TD class="col0">6</TD><TD class="col1">pIG16_059</TD><TD class="col2">pIG16_155</TD></TR><TR class="row7"><TD class="col0">7</TD><TD class="col1">pIG16_066</TD><TD class="col2">pIG16_156</TD></TR><TR class="row8"><TD class="col0">8</TD><TD class="col1">pIG16_056</TD><TD class="col2">pIG16_157</TD></TR><TR class="row9"><TD class="col0">9</TD><TD class="col1">pIG16_058</TD><TD class="col2">pIG16_158</TD></TR><TR class="row10"><TD class="col0">10</TD><TD class="col1">pIG16_054</TD><TD class="col2">pIG16_159</TD></TR><TR class="row11"><TD class="col0">11</TD><TD class="col1">pIG16_051</TD><TD class="col2">pIG16_160</TD></TR></TBODY></TABLE></DIV><P><STRONG>IV</STRONG> Further ligations (Ligation of GST into surface display vectors) </P><DIV class="table sectionedit191"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0">No.</TH><TH class="col1">Insert</TH><TH class="col2">Backbone</TH><TH class="col3">Result</TH></TR><TR class="row1"><TD class="col0">1</TD><TD class="col1">BamHI-pIG16_032-XbaI</TD><TD class="col2">BamHI-GST-XbaI</TD><TD class="col3">pIG16_131</TD></TR><TR class="row2"><TD class="col0">2</TD><TD class="col1">BamHI-pIG16_034-XbaI</TD><TD class="col2">BamHI-GST-XbaI</TD><TD class="col3">pIG16_142</TD></TR></TBODY></TABLE></DIV></DIV></DIV></DIV><DIV class="color2"><DIV class="para_20"><H3 class="sectionedit192"><A name="section210916" id="section210916">21.09.16</A></H3><DIV class="level3"><P><STRONG>1) Transformation</STRONG> The transformations of the T4-ligations of the previous day were repeated.
                    </P><P><STRONG>2) Inoculation</STRONG> Inoculation of the following colonies transformed on the previous day:
                    <STRONG>I</STRONG> PTrpC: pIG16_143#1 148#1 149#1-2
                    <STRONG>II</STRONG> PLac: pIG16_134#1-2 135#5 138#1-5
                    <STRONG>III</STRONG> PCotYZ: pIG16_150#1 159#1-4 157#1-4 155#1
                    <STRONG>IV</STRONG> pIG16_131 #1-2
                     Inoculation in 5 mL of LB medium supplemented with the appropriate antibiotic. </P></DIV><H3 class="sectionedit193"><A name="section220916" id="section220916">22.09.16</A></H3><DIV class="level3"><P><STRONG>1) MiniPrep</STRONG> Plasmid preparation of the inoculated colonies from the previous day using the zymo research zyppy kit. <STRONG>I</STRONG> PTrpC: pIG16_143#1 148#1 149#1-2
                    <STRONG>II</STRONG> PLac: pIG16_134#1-2 135#5 138#1-5
                    <STRONG>III</STRONG> PCotYZ-RBS: pIG16_150#1 159#1-4 157#1-4 155#1
                    <STRONG>IV</STRONG> pIG16_131 #1-2
                    </P><P><STRONG>2) Testdigestion</STRONG> 500 ng of the prepared plasmids were treated with 4 units of XbaI and PstI in 1X NEBuffer 2.1 for 1 hour at 37 °C. The digested DNA was analyzed by agarose gel electrophoresis. </P><P><STRONG>3) Sequencing</STRONG> the Following plasmids were sent so sequencing: pIG16_131#1 138#3 149#2 143#1 150#1 159#2+3 155#11 158#4+5 </P><P><STRONG>4) Inoculation</STRONG> Inoculation of further constructs with the new promoters:
                    <STRONG>I</STRONG> PTrpC: pIG16_147#1-2 148#2 149#2-3 142#2 145#1-3
                    <STRONG>II</STRONG> PLac: pIG16_134#1-5 136#1-3
                    <STRONG>III</STRONG> PCotYZ-RBS: pIG16_160#1 157#1-5 155#2 150#2-4 159#1 151#1-2
                    <STRONG>IV</STRONG> pIG16_131 #3-5 142#1
                    </P></DIV><H3 class="sectionedit194"><A name="section230916" id="section230916">23.09.16</A></H3><DIV class="level3"><P><STRONG>1) MiniPrep</STRONG> The the plasmids of inoculated colonies from the previous day were prepared using the Zyppy Miniprep kit:
                    </P><P><STRONG>2) Testdigestion</STRONG> 500 ng of prepared DNA was treated with 4 units of XbaI and PstI for 1 hour at 37 °C. The digestion was analyzed by agarose gel electrophoresis.
                    </P><P><STRONG>3) Sequencing</STRONG> Positive colonies were sent to sequencing. </P><P><STRONG>4) Inoculation</STRONG> Inoculation of colonies for preparation of glycerol stocks:
                    <STRONG>I</STRONG> PLac: 132#1 137#1 138#3 133#2
                    <STRONG>II</STRONG> PTrpC: 143#1 149#2
                    <STRONG>III</STRONG> PCotYZ-RBS: 150#1 158#5 159#3 151#1 160#1
                    <STRONG>IV</STRONG> 129#1 142#1 131#1 126#1 127#3
                    </P></DIV></DIV></DIV><DIV class="color3"><DIV class="para_20"><H3 class="sectionedit195"><A name="section240916" id="section240916">24.09.16</A></H3><DIV class="level3"><P><STRONG>1) MiniPrep &amp; Glycerl stocks</STRONG> The plasmids from the inoculated colonies of the previous day were prepared using the Zyppy MiniPrep kit.
                     1 mL of the inoculated colonies was used for the preparation of 15 % glycerol stocks. </P><P><STRONG>2) Linearization</STRONG> 5 µg of the following plasmids was linearized using 25 units of the appropriate enzyme in 1X reaction buffer for 2 hours at 37 °C: </P><DIV class="table sectionedit196"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0">No</TH><TH class="col1">Plasmid pIG16_</TH><TH class="col2">Enzyme</TH></TR><TR class="row1"><TD class="col0">1</TD><TD class="col1">150</TD><TD class="col2">XhoI</TD></TR><TR class="row2"><TD class="col0">2</TD><TD class="col1">151</TD><TD class="col2">XhoI</TD></TR><TR class="row3"><TD class="col0">3</TD><TD class="col1">158</TD><TD class="col2">XhoI</TD></TR><TR class="row4"><TD class="col0">4</TD><TD class="col1">159</TD><TD class="col2">XhoI</TD></TR><TR class="row5"><TD class="col0">5</TD><TD class="col1">160</TD><TD class="col2">XhoI</TD></TR><TR class="row6"><TD class="col0">6</TD><TD class="col1">126</TD><TD class="col2">ScaI</TD></TR><TR class="row7"><TD class="col0">7</TD><TD class="col1">128</TD><TD class="col2">ScaI</TD></TR><TR class="row8"><TD class="col0">8</TD><TD class="col1">129</TD><TD class="col2">ScaI</TD></TR><TR class="row9"><TD class="col0">9</TD><TD class="col1">141</TD><TD class="col2">PstI</TD></TR><TR class="row10"><TD class="col0">10</TD><TD class="col1">131</TD><TD class="col2">PstI</TD></TR></TBODY></TABLE></DIV><P> After linearization the DNA was purified using the QiaGen PCR purification kit. </P><P><STRONG>3) PCR</STRONG> The anti-GFP nanobody was amplified in order to introduce BamHI and XbaI restrictions sites:
                    </P><DIV class="table sectionedit197"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0">Component</TH><TH class="col1">Conc.</TH><TH class="col2">Vol [µL]</TH></TR><TR class="row1"><TD class="col0">DNA template</TD><TD class="col1">.</TD><TD class="col2">0.1</TD></TR><TR class="row2"><TD class="col0">oIG16_97</TD><TD class="col1">10 µM</TD><TD class="col2">2.5</TD></TR><TR class="row3"><TD class="col0">oIG16_98</TD><TD class="col1">10 µM</TD><TD class="col2">2.5</TD></TR><TR class="row4"><TD class="col0">Q5 MasterMix</TD><TD class="col1">2X</TD><TD class="col2">25</TD></TR><TR class="row5"><TD class="col0">dH2O</TD><TD class="col1">.</TD><TD class="col2">20</TD></TR></TBODY></TABLE></DIV><P> Amplification using the Touchdown67 program. The PCR product was loaded on a 1% TAE agarose gel and subjected to electrophoresis. The band corresponding to the expected size was extracted and purified from the gel using the QiaGen Gelextraction kit.
                    </P><P><STRONG>4) Digestion</STRONG> 2 µg of the extracted PCR product was treated with 10 units of BamHI and XbaI for 1 hour at 37 °C. The DNA was purified using the QiaGen PCR purification kit. </P><P><STRONG>5) T4 ligations</STRONG></P><DIV class="table sectionedit198"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0">No.</TH><TH class="col1">Insert</TH><TH class="col2">Backbone</TH><TH class="col3">Resulting plasmid</TH></TR><TR class="row1"><TD class="col0">1</TD><TD class="col1">BamHI - aGFPnano - XbaI</TD><TD class="col2">BamHI - pCgeA-C - XbaI</TD><TD class="col3">pIG16_161</TD></TR><TR class="row2"><TD class="col0">2</TD><TD class="col1">BamHI - aGFPnano - XbaI</TD><TD class="col2">BamHI - pCotZ-C - XbaI</TD><TD class="col3">pIG16_141</TD></TR></TBODY></TABLE></DIV><P> The DNA was ligated using T4 ligase at a molar ratio of 1:3 (Vector : Insert) in 1X T4 ligation buffer in a total volume of 20 µL for 30 min at RT. 5 µL of the ligation mix were transformed into chemically competent E. coli DH5alpha and plated on LB agar plates supplemented with ampicilin. Incubation o/n at 37 °C. </P></DIV><H3 class="sectionedit199"><A name="section250916" id="section250916">25.09.16</A></H3><DIV class="level3"><P><STRONG>1) Glycerol stocks</STRONG> Preparation of 15 % glycerol stocks with the following strains:
                     pIG16_134 143 149 150 151 157 159 160 138 137 134 133 132 126 127 129 131 141 </P></DIV><H3 class="sectionedit200"><A name="section260916" id="section260916">26.09.16</A></H3><DIV class="level3"><P><STRONG>1) MiniPrep</STRONG> The Plasmids from the following strains were prepared using the Zyppy MiniPrep kit:
                     pIG16_142#1,2 136#1-3 131#1 141#1 126#1 </P><P><STRONG>2) Testdigestion</STRONG> 500 ng of the prepared DNA (pIG16_142 and 136) was treated with 5 units of the appropriate enzyme in 1X reaction buffer in a total volume of 20 µL for 1 h at 37 °C. </P><DIV class="table sectionedit201"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0">Plasmid</TH><TH class="col1">Restriction enzymes</TH></TR><TR class="row1"><TD class="col0">pIG16_142</TD><TD class="col1">XbaI + NcoI-HF</TD></TR><TR class="row2"><TD class="col0">pIG16_136</TD><TD class="col1">XbaI + PstI</TD></TR></TBODY></TABLE></DIV><P> Analysis by agarose gel electrophoresis.
                    </P><P><STRONG>3) Linearization</STRONG> 3.5 µg of the following plasmids were linearized with 20 units of the appropriate enzyme in 1X reaction buffer in a total reaction volume of 50 µL for 1 hour at 37 °C. </P><DIV class="table sectionedit202"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0">Plasmid</TH><TH class="col1">enzyme</TH></TR><TR class="row1"><TD class="col0">131</TD><TD class="col1">PstI</TD></TR><TR class="row2"><TD class="col0">141</TD><TD class="col1">PstI</TD></TR><TR class="row3"><TD class="col0">126</TD><TD class="col1">ScaI</TD></TR><TR class="row4"><TD class="col0">142</TD><TD class="col1">PstI</TD></TR><TR class="row5"><TD class="col0">034</TD><TD class="col1">ScaI</TD></TR></TBODY></TABLE></DIV><P> 5 µL of the digestion was analyzed by gel electrophoresis. The remaining sample was purified using the QiaGen PCR purification kit.
                    </P><P><STRONG>4) Inoculation</STRONG> Inoculation of further colonies for test digestion: pIG16_161#1-5 </P></DIV></DIV></DIV><DIV class="color4"><DIV class="para_20"><H3 class="sectionedit203"><A name="section270916" id="section270916">27.09.16</A></H3><DIV class="level3"><P><STRONG>1) MiniPrep</STRONG> The plasmid DNA of the inoculated colonies of pIG16_161#1-5 were prepared using the QiaGen MiniPrep kit. </P><P><STRONG>2) Testdigestion</STRONG> 500 ng of plasmid DNA was treated with 4 units of SpeI-HF and XbaI in 1XCutsmart buffer in a total reaction volume of 20 µL for 1 hour at 37 °C. The samples were analyzed by gel electrophoresis. Colony #1 was sent to sequencing. </P></DIV><H3 class="sectionedit204"><A name="section280916" id="section280916">28.09.16</A></H3><H3 class="sectionedit205"><A name="section290916" id="section290916">29.09.16</A></H3><H3 class="sectionedit206"><A name="section300916" id="section300916">30.09.16</A></H3><H3 class="sectionedit207"><A name="section011016" id="section011016">01.10.16</A></H3><H3 class="sectionedit208"><A name="section021016" id="section021016">02.10.16</A></H3><H3 class="sectionedit209"><A name="section031016" id="section031016">03.10.16</A></H3><H3 class="sectionedit210"><A name="section041016" id="section041016">04.10.16</A></H3><DIV class="level3"><P><STRONG>1.) Digestion for 3 A Assembly</STRONG></P></DIV><H3 class="sectionedit211"><A name="section051016" id="section051016">05.10.16</A></H3><H3 class="sectionedit212"><A name="section061016" id="section061016">06.10.16</A></H3><DIV class="level3"><P><STRONG>1.) Testdigestion</STRONG></P><P><STRONG>Digestion mixture:</STRONG></P><DIV class="table sectionedit213"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0">Component</TH><TH class="col1">concentration</TH><TH class="col2">volume [µL]</TH></TR><TR class="row1"><TD class="col0">DNA</TD><TD class="col1">500ng/µL</TD><TD class="col2">4-6µl</TD></TR><TR class="row2"><TD class="col0">PstI-HF</TD><TD class="col1">20u/µL</TD><TD class="col2">0.1</TD></TR><TR class="row3"><TD class="col0">XbaI</TD><TD class="col1">20u/µL</TD><TD class="col2">0.1</TD></TR><TR class="row4"><TD class="col0">NEBuffer CS</TD><TD class="col1">10X</TD><TD class="col2">2</TD></TR><TR class="row5"><TD class="col0">ultra pure water</TD><TD class="col1">-</TD><TD class="col2">ad 20 µL</TD></TR></TBODY></TABLE></DIV></DIV></DIV></DIV><DIV class="color5"><DIV class="para_20"><H3 class="sectionedit214"><A name="section071016" id="section071016">07.10.16</A></H3><DIV class="level3"><DIV class="tags"><SPAN><A href="/igem2016/doku.php?id=tag:lab_book" class="wikilink1" title="tag:lab_book" rel="tag">lab book</A></SPAN></DIV></DIV></DIV></DIV><FOOTER><DIV class="center"><DIV style="display:inline-block"><P>Posted by: iGEM Freiburg </P></DIV><DIV style="display:inline-block"><P> Nanocillus - 'cause spore is more! </P></DIV><DIV style="display:inline-block"><P><A style="color:#e8a126" href="https://2016.igem.org/Team:Freiburg/Kontakt"> Contact </A></P></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>