Team:Freiburg/NotebookCloning
Menu
Team
Attributions
Achievements
Pictures
Giant Jamboree
Project
Motivation
Approach
B. subtilis
Microbial surface display
Targeted Drug Delivery
Lab
Methods
Journal
Results
Proof of Concept
Binding
Delivery
Nanocillus
Biobricks
Extra
Safety
Knock Outs
Kill Switch
Public relations
Human Practice
Integrated
Collaborations
Poster
Back to Top
Lab Journal
Group 1 - Cloning
To keep an overview of the cloned constructs every plasmid was assigned to an ID: pIG16_000.
All used oligos were assigned to an ID as well: oIG16_000.
The complete list of the resulting bacterial strains and oligos can be found in the attached Excel files. The spore coat proteins cotZ, cotG, cotB and cgeA were amplified from the genome of B. subtilis 168. The anti-GFP nanobody and the GST were amplified from plasmids provided by Dr. Nicole Gensch and Dr. Maximilian Ulbrich. For the cloning strategy see Project - Approach .
List of Oligos
List of Strains
07.07.16
1) Transformation
Julia Bartels from iGEM 2012 sent us integration vectors for B.subtilis. Sebastian from AG Weber (BIOSS) shared plasmids containing mCherry and GFP.
Vectors:
No.1. pBS0K-Pspac-[RFP]
No.2. pBS1C3-[RFP]
No.3. pBS2E-[RFP]
No.4. pBS3Clux-[RFP]
No.5. pBS4S-[RFP]
No.6. pSBBs4S-Sporovector
7. mCherry
8. GFP
Transformation of the Vectors from Julia Bartels and Sebastian (from AG Weber) into chemically competent E.coli K12 DH5alpha was performed according to protocol. The E.coli were incubated for 1 hour at 37 °C and 250 rpm and spread on LB-Agar plates supplemented with ampicilin. Incubation for o/n at 37 °C.
2) Preparation of competent E.coli
For the preparation for chemically competent E.coli one colony from an agar plate was picked in inoculated in 5 mL LB-medium. Incubation o/n at 37 °C and 250 rpm.
08.07.16
1) Production of competent E.coli
Preparation of chemically competent cells according to the protocol of the manufacturer (Zymoresearch, Z competent Mix&Go kit)
The resulting cell suspension was aliquoted (100 µL) in tubes and stored at -80 °C.
2) Inoculation
From each transformation plate(No. 1-6) 5 colonies were picked and inoculated in 5 mL LB medium supplemented with Ampicilin.
Incubation o/n at 37°C and 250 rpm.
The B.subtilis W168 strain was inoculated in 5 mL of LB medium and incubated o/n at 37 °C and 250 rpm. In parallel, a sample of the W168 strain was spread on a LB agar plate and incubated o/n at 37 °C.
09.07.16
1) MiniPrep
The the plasmids of the inoculated colonies (No.1-6) were prepared using the QiaGen MiniPrep kit according the instructions of the manufacturer and eluted in 20 µL of ultra pure water. The concentration of the DNA was spectroscopically determined by a Nanodrop.
2) Colony PCR Colony PCR for the amplication of the cotZ and cgeA gene was performed using the following oligos:
cotZ: oIG16_1+2 Annealing temperature: 57 °C
cgeA: oIG16_43+34 Annealing temperature: 65 °C
Reaction conditions
component
concentration
volume [µL]
1 B.subtilis colony
-
-
Primer fw
10 µM
1.0
Primer rv
10 µM
1.0
dNTPs
40 µM
0.4
HF Buffer
5X
4.0
Phusion Pol
2U/µL
0.2
ddH2O
-
ad20 µL
Cycler Program
Step
Temperature [°C]
Duration
Repeats
Initial denaturation
98
1 min
1
Denaturation
98
10 s
30
Annealing
58 or 65
10 s
30
Elongation
72
30 s
30
Final elongation
72
5 min
1
Storage
8
-
10.07.16
1) Gel electrophoresis
20 µL of the colony PCR samples were supplemented with the appropriate amount of 10X Orange G loading buffer, loaded on a 1 % agarose TAE gel and subjected to electrophoresis for 40 min at 120 V. The DNA was stained with Midori Green. (No bands were observable at all. The amount of Midori Green was not sufficient)
11.07.16
1)Colony PCR
from 09.07.2016 was repeated at a total volume of 50 µL. After electrophoresis the sample were supplemented with the appropriate amount of 10X orange G loading buffer and loaded on an 1 % TAE agarose gel. The gel was subjected to electrophoresis for 40 min at 120 V. The DNA was stained with Midori Green.
Only the amplified cotZ was visible at UV Light.
The cotZ band was excised and the DNA was extracted using the QiaQuick Gel extraction kit according the the instructions of the manufacturer. The DNA was eluted in 20 µL of ultra pure H2O and the concentration was photometrically determined by a NanoDrop.
CotZ gelextraction concentration: 18 ng/µL
12.07.16
1) Colony PCR
Colony-PCR for the amplification of cgeA and cotZ was repeated
Primer:
cgeA: oIG16_034 + 043
cotZ: oIG16_001 + 002
Reaction conditions
component
concentration
volume [µL]
1 B.subtilis colony W168
-
-
Primer fw
10 µM
2.5
Primer rv
10 µM
2.5
dNTPs
40 µM
0.4
HF Buffer
5X
10
Phusion Pol
2U/µL
0.5
ddH2O
-
ad50 µL
Cycler Program
Step
Temperature [°C]
Duration
Repeats
Initial denaturation
98
1 min
Denaturation
98
10 s
30X
Annealing
58 or 65
10 s
30X
Elongation
72
30 s
30X
Final elongation
72
5 min
Storage
8
-
The samples were supplemented with the appropriate amount of 10 X Orange G loading dye and loaded on a 1 % TAE agarose gel. As molecular marker a 2-log marker was used.
No bands were observed at UV light.
13.07.16
1) Gradient colony PCR
To amplify cgeA 7 samples (20 µL each) were amplified by gradient colony PCR applying annealing temperatures from 58 - 68 °C.
An additional 50µl sample was made to amplify CotZ.
The samples were supplemented with 10 X Orange G loading buffer and analyzed by gel electrophoresis. No bands were observable.
14.07.16
1) Repeat of gradient colony PCR
To amplify cgeA 7 samples (20 µL each) were amplified by gradient colony PCR applying annealing temperatures from 58 - 68 °C. The duration of initial denaturation was elongated to 5 min.
The samples were supplemented with 10 X Orange G loading buffer and analyzed by gel electrophoresis. No bands were observable.
15.07.16
1) Lysis of B. subtilis for colony PCR
4 different approaches were tried to lyse the bacteria prior to colony PCR.
1.
3. Mini-Prep lysis buffer.
4. Laemmlie sample buffer 100°C 5 Min.
1 µl of each sample was used for amplification of cotZ using the oligos oIG16_1+2. The extracted cotZ gene from 11.07.16 was amplified as control alongside the lysed samples
Reaction conditions
component
concentration
volume [µL]
lysed B.subtilis W168
-
1 µL
Primer fw
10 µM
2.5
Primer rv
10 µM
2.5
dNTPs
40 µM
0.4
HF Buffer
5X
10
Phusion Pol
2U/µL
0.5
ddH2O
-
ad50 µL
Cycler Program
Step
Temperature [°C]
Duration
Repeats
Initial denaturation
98
5 min
Denaturation
98
10 s
30X
Annealing
65
10 s
30X
Elongation
72
30 s
30X
Final elongation
72
5 min
Storage
8
-
2) Testdigestion
PvuII test digestion for verification of the plasmids sent by Julia Bartels.
Reaction conditions:
component
concentration
volume
MiniPrep DNA
200-600 ng/µL
2 µL
CutSmart Buffer (NEB)
10X
2 µL
PvuII-HF
20 U/ µL
0.1 µL
ddH2O
-
ad20 µL
Incubation at 37 °C for 1 hour. The samples were analyzed by gel electrophoresis on a 1 % TAE agarose gel.
3) Subcloning
Subcloning of the amplified cotZ gene into the linearized pJET1.2 vector was performed using the pJET subcloning kit according to the instructions of the manufacturer. 5 µL of the ligation mixture was used for transformation of chemically competent E.coli DH5alpha (Mix & Go). The transformed bacteria were spread on a LB-agar plate supplemented with amplicilin and incubated o/n at 37 °C. A control ligation was prepared using the DNA fragments supplied by the manufacturer.
16.07.16
1) Colony PCR
for the amplication of the cgeA was performed using the following oligos:
cgeA: oIG16_43+34 Annealing temperature: 65 °C
To lyse the bacteria prior to the colony PCR.
Reaction conditions
component
concentration
volume [µL]
lysed B.subtilis W168
-
1 µL
Primer fw
10 µM
2.5
Primer rv
10 µM
2.5
dNTPs
40 µM
0.25
HF Buffer
5X
10
Phusion Pol
2U/µL
0.5
ddH2O
-
ad50 µL
Cycler Program
Step
Temperature [°C]
Duration
Repeats
Initial denaturation
98
5 min
Denaturation
98
10 s
30X
Annealing
65
10 s
30X
Elongation
72
30 s
30X
Final elongation
72
5 min
Storage
8
-
2) Inoculation
Subcloning and transformation of pJET1.2-cotZ resulted only in a few colonies. Three of them were picked and inoculated in 5 mL LB-medium w/ ampicilin and incubated o/n at 37°C and 250 rpm. The transformation with the control plate yielded >100 colonies.
17.07.16
1) Colony PCR
5 µL of vegetative B.subtilis from a liquid culture were diluted 1:10 in Qiagen EB buffer and incubated at 100°C for 5min to achieve cell lysis
1 µL of the dilution was used for amplification of cgeA by gradient PCR and 8 samples were prepared.
Reaction conditions
component
concentration
volume [µL]
lysed B.subtilis W168
-
1 µL
Primer fw
10 µM
1
Primer rv
10 µM
1
dNTPs
40 µM
0.1
HF Buffer
5X
5
Phusion Pol
2U/µL
0.2
ddH2O
-
ad20 µL
Cycler Program
Step
Temperature [°C]
Duration
Repeats
Initial denaturation
98
5 min
Denaturation
98
10 s
30X
Annealing
gradient 58-68
20 s
30X
Elongation
72
30 s
30X
Final elongation
72
5 min
Storage
10
-
After thermal cycling the samples were supplemented with the appropriate amount of 10X Orange G loading dye and analyed by agarose gel electrophoresis.
2)  MiniPrep
The plasmids pJET1.2-cotZ were prepared using the QIAprep Spin Miniprep Kit according the instructions of the manufacturer and eluted in 50µL of EB Buffer. The concentration of the DNA was spectroscopically determined by a nanodrop.
DNA
concentration[ng/µL]
pJet_cotZ_#1
120
pJet_cotZ_#2
89
pJet_cotZ_#3
68
3) Testdigestion
Reaction conditions
component
concentration
volume
DNA
68 - 120 ng/µL
7 µL
XbaI
20U/µL
0.2
XhoI
20U/µL
0.2
CutSmart
10X
2 µL
dH2O
-
11 µL
Incubation for 1 h at 37 °C. Analysis of digestion by agarose gel electrophoresis.
2-log DNA ladder is bad in the last couple of gels –> discarded.
Not the expected band pattern. Subcloning should be repeated.
3) Colony PCR
5 µL of vegetative B.subtilis from a liquid culture were diluted 1:10 in Qiagen EB buffer and incubated at 100°C for 5min to achieve cell lysis
1 µL of the dilution was used for amplification of cgeA and cotZ by PCR.
oIG16_001+002: CotZ, Tm = 58 °C oIG16_034+043: cgeA, Tm = 63 °C
component
concentration
volume [µL]
lysed B.subtilis W168
-
1 µL
Primer fw
10 µM
2.5
Primer rv
10 µM
2.5
dNTPs
40 µM
0.25
HF Buffer
5X
10
Phusion Pol
2U/µL
0.5
ddH2O
-
ad50 µL
Cycler Program
Step
Temperature [°C]
Duration
Repeats
Initial denaturation
98
5 min
Denaturation
98
10 s
30X
Annealing
58 or 63
10 s
30X
Elongation
72
30 s
30X
Final elongation
72
5 min
Storage
8
-
The bands were excised from the gel and stored at -20 °C o/n
18.07.16
1) Gelextraction & Subcloning
The DNA was extracted using the QiaGen gel extraction kit according to the protocol of the manufacturer and eluted in 30 µL of ultra pure water. The concentration was determined spectroscopically by a nanodrop.
gene
concentration
cgeA
6 ng/µL
cotZ#1
8 ng/µL
cotZ#2
11 ng/µL
The extracted DNA was subcloned into a linearized pJET1.2 vector using the CloneJET PCR Cloning Kit according to the protocol of the manufacturer. 5 µL of the ligation mixture were transformed into chemically competent E.coli DH5alpha (Mix&Go) and spread on a LB-agar plate supplemented with ampicilin and incubated at 37 °C o/n.
19.07.16
1) Inoculation
4 colonies per plate were picked and incubated in 5 mL LB-medium (amp) o/n at 37 °C and 250 rpm.
20.07.16
1) MiniPrep
The plasmids pJET1.2-cotZ-I.#1-4;pJET1.2-cotZ-II.#1-4 and pJET1.2-cgeA#1-4 were prepared using the QIAprep Spin Miniprep Kit according the instructions of the manufacturer and eluted in 20µL of ultra pure water (for better results put tubes for 2 minutes on 50°C thermo; centrifuge for 1min at 13000rpm and repeat with 10µL; increases the amount of deluted DNA). The concentration of the DNA was spectroscopically determined by a nanodrop.
DNA
concentration[ng/µL]
pJet1.2_cgeA_#1
127
pJet1.2_cgeA_#2
214 –> Seq
pJet1.2_cgeA_#3
279
pJet1.2_cgeA_#4
141
pJet1.2_cotZ-I_#1
260
pJet1.2_cotZ-I_#2
101
pJet1.2_cotZ-I_#3
409
pJet1.2_cotZ-I_#4
435
pJet1.2_cotZ-II_#1
475
pJet1.2_cotZ-II_#2
350 –> Seq
pJet1.2_cotZ-II_#3
353
pJet1.2_cotZ-II_#4
342
2) Test digestion
Due to the concentraion difference of the DNA the samples where divided into two groups
Reaction conditions
DNA concentration < 300ng/µL
component
concentration
volume
DNA
101 - 279 ng/µL
5 µL
XbaI
20U/µL
0.2
XhoI
20U/µL
0.2
CutSmart
10X
2 µL
dH2O
-
14.1 µL
DNA concentration > 300ng/µL
component
concentration
volume
DNA
342-475 ng/µL
1.5 µL
XbaI
20U/µL
0.2
XhoI
20U/µL
0.2
CutSmart
10X
2 µL
dH2O
-
12.6 µL
Control group < 300ng/µL
component
concentration
volume
DNA
101 - 279 ng/µL
5 µL
dH2O
-
5 µL
Control group > 300ng/µL
component
concentration
volume
DNA
342-475 ng/µL
1.5 µL
dH2O
-
8,5 µL
Incubation for 1 h at 37 °C. Analysis of digestion by electrophoresis on a 1% TAE agarose gel.
3) Sequencing
Sequencing of pJET1.2-cgeA #2 (IC0225) and pJET1.2-cotZII #2 (IC0224) by GATC biotech.
21.07.16
1) Inoculation
Confirmation of the proper sequences for pJET1.2_cgeA#2 and pJET1.2_cotZII#2. The colonies were re-inoculated in 5 mL LB-medium (w/ amp) and incubated at 37 °C, 250 rpm o/n.
22.07.16
1) Transformation & Inoculation
Nicole provided us with a LB-plate containing E.coli harboring pGEX6P1 and pRP261 (Both plasmids contain glutathion S transferase). Max provided us with a sample of pET303_aGFPnano_TEV_10His (plasmid containing the anti-GFP nanobody). The pET303 plasmid was transformed into chemically competent E.coli DH5alpha and spread on a LB-agar plate supplemented with ampicilin and incubated o/n at 37 °C.
23.07.16
1) Inoculation
Inoculation of the E.coli DH5alpha containing pGEX6P1, pRP261 and pET303_aGFPnano in 5 mL LB medium supplemented with ampicilin. Incubation o/n at 37 °C, 250 rpm.
Glycerol stocks of pJET1.2-cgeA and pJET1.2-cotZ were prepared (15% [v/v] Glycerol).
Inoculation of pBS1C3-[RFP] culture #2 for test-transformation of B.subtilis.
2) Transformation
The pSB1C3-[RFP] vector from the iGEM distribution kit (plate 1, position 23-O) was solubilized with 10 µL of ultra pure water. 1 µL was used for transformation of chemically competent E.coli DH5alpha and spread on a LB-agar plate supplemented with chloramphenicol and incubated o/n at 37 °C.
3) Extension PCR
of cgeA (from pJET1.2_cgeA) and anti GFP-Nanobody (pET303) *Reaction conditions*
component
concentration
volume
DNA
~10 ng/µL
1 µL
Primer fw
10 µM
2.5
Primer rv
10 µM
2.5
Q5 High-Fidelity Master Mix (NEB)
2x
25 µL
ultra pure H2O
-
ad 50 µL
Touchdown-PCR Template: pJET1.2_cgeA
Primer:olG16_44fw + olG16_45rw
Cycler Program GFP-Nanobody
Step
Temperature [°C]
Duration
Repeats
Initial denaturation
98
5 min
Denaturation
98
10 s
10X
Annealing
72* (-1°C per cycle)
10 s
10X
Elongation
72
30 s
10X
Denaturation
98
10 s
20X
Annealing
63
10 s
20X
Elongation
72
30 s
20X
Final elongation
72
5 min
Storage
8
-
Template: pET303_aGFPnano
Primer:olG30_fw/olG31_rw
Cycler Program CgeA
Step
Temperature [°C]
Duration
Repeats
Initial denaturation
98
5 min
Denaturation
98
10 s
10X
Annealing
72* (-1°C per cycle)
10 s
10X
Elongation
72
30 s
10X
Denaturation
98
10 s
20X
Annealing
62
10 s
20X
Elongation
72
30 s
20X
Final elongation
72
5 min
Storage
8
-
After amplification the samples were analyzed by agarose gel electrophoresis.
Gel GFP-Nanobody/CgeA
24.07.16
1) Gelextraction
Gel Extraction of pET303-aGFPnanobody oIG16_30_fw/oIG16_031_rw PCR product.
37.7ng/µl (18µl) stored at -20 °C
2) MiniPrep
Plasmid preparation of inoculated E.coli DH5alpha transformed with Mini prep of pBS1C3-RFP culture#2 using the QiaQuick MiniPrep kit according to the instructions of the manufacturer. Elution with 30 µL of ultra pure water.
148.6ng/µl (28µl)
placed in freezer -20
Plasmid preparation of inoculated E.coli DH5alpha transformed with pGEX6P1, pRP261, pET303-aGFPnano_TEV_10His using the QiaQuick miniprep kit according to the instructions of the manufacturer. The plasmids were eluted in 30 µL of ultra pure water. The DNA concentration was determined by NanoDrop:
Sample
concentration[ng/µL]
pRP261 #1
248.9
pRP261 #2
175.0
pRP261 #3
235.9
pRP261 #4
221.0
pGEX-6P1 #1
198.7
pGEX-6P1 #2
177.3
pGEX-6P1 #3
149.1
pGEX-6P1 #4
131.0
pET303-aGFPnano_TEV_10His #1
167.4
pET303-aGFPnano_TEV_10His #2
151.1
pET303-aGFPnano_TEV_10His #3
161.7
pET303-aGFPnano_TEV_10His #4
165.0
2) Testdigestion
Verification of the plasmid was performed by test digestion with EcoRI and PstI.
Reaction conditions:
Component
concentration
volume [µL]
DNA
131-248ng/µL
3
PstI
20u/µL
0.1
EcoRI
20u/µL
0.1
NEBuffer 2.1
10X
2
ultra pure water
-
ad 20 µL
The samples were incubated at 37 °C for 1h and analyzed by gel electrophoresis.
25.07.16
1) MiniPrep
Plasmids of pSB1C3 were prepared. DNA concentration was determined by Nanodrop:
Culture #1 127.5ng/µl
Culture #2 88.4ng/µl
Culture #3 116.8ng/µl
Culture #4 126.1ng/µl
2) Testdigestion pSB1C3 was treated with
Digestion mixture:
Component
concentration
volume [µL]
DNA
500ng/µL
5-6
PstI
20u/µL
0.1
EcoRI-HF
20u/µL
0.1
NEBuffer 2.1
10X
2
ultra pure water
-
ad 20 µL
Agarose Gel of pSB1C3 digestion:
3) Inoculation
Culture #3 was re-inoculated in chloramphenicol-medium.
26.07.16
1) MiniPrep
With culture #3 from the day before a standard mini prep was performed.
cultur #3 169,4ng/µl (28µl)
→ placed in the -20 freezer
2) Colony PCR
Colony PCR of CotG and CotB from B. subtilis 168 genome. PCR for the amplication of the cotG and cgeB gene was performed using the following oligos:
cotG: oIG16_009+010 Annealing temperature: 62 °C
cotB: oIG16_017+018 Annealing temperature: 57 °C
Template DNA preparation: 1 µL of a o/n culture with B.subtilis W168 cells were diluted with EB-Buffer (Qiagen, 1:10) and incubated at 100°C for 5 min for lysis. 1 µL of the lysed cells was used as template for amplificaiton of the genes.
Reaction conditions
component
concentration
volume [µL]
Q5 HiFi MasterMix
2x
25
Primer fw
10 µM
2.5
Primer rv
10 µM
2.5
lysed cells
-
1
ultra pure water
-
ad 50 µL
Appropriate controls w/o template DNA were included.
The PCR was analyzed by agarose gel electrophoresis.
The bands corresponding to the size of the cotG and cotB were excised and the DNA was extracted using the QiaQuick Gel extraction kit according to the instructions of the manufacturer. The DNA was eluted in 30 µL of ultra pure water.
2) Gel extraction
The concentration of the extracted DNA was determined by a Nanodrop.
sample
concentration
CotB
50.2 ng/µL
CotG
42.1 ng/µL
3) Subcloning
The amplified genes were subcloned into the pJET1.2 vector according to the protocol of the manufacturer, transformed into chemically competent E.coli DH5alpha, spread on a LB-agar plate supplemented with ampicilin and incubated o/n at 37 °C.
4) Inoculation
The E.coli liquid cultures containing the plasmids pRP261, pGEX6P1 and pET303_aGFPnano_TEV_10His were reinoculated in 5 mL LB medium supplemented with ampicilin and incubated o/n at 37 °C and 250 rpm.
27.07.16
1) Inoculation
4 colonies of plated E.coli DH5alpha with the pJET1.2-cotB and pJET1.2-cotG plasmids were picked and inoculated in 5 mL of LB medium supplemented with ampicilin and incbated o/n at 37 °C and 250 rpm. 2) Sequencing
Sequencing of plasmids sent by Julia Bartels:
Plasmid
colony#
Primer:oIG16_
pBS1C-[RFP]
#2
025
026
028
029
032
pBS2E-[RFP]
#5
025
pBS4S-[RFP]
#5
025
26
27
SporoVector-[RFP]
#1
025
026
027
28.07.16
1) Mini-prep:
A standard Qiagen Mini-Prep was performed with the following cultures:
pJET1.2-cotB cultures #2/3/4, pJET1.2-cotG cultures #1/2/3/4, pGEX6P1 culture #1 (GST 2), pRP261 culture #1 (GST 1), pET303_aGFPnano_TEV_10His culture #1.
The DNA concentrations were measured with Nanodrop:
pJET1.2-cotB cultures #2=450,9(ng/µl)
pJET1.2-cotB cultures #3=820,3(ng/µl)
pJET1.2-cotB cultures #4=621,2(ng/µl)
pJET1.2-cotG cultures #1=696,5(ng/µl)
pJET1.2-cotG cultures #2=149,6(ng/µl)
pJET1.2-cotG cultures #3=614,8(ng/µl)
pJET1.2-cotG cultures #4=145,2(ng/µl)
pGEX6P1 culture #1=243,1(ng/µl)
pRP261 culture #1=308,2(ng/µl)
pET303_aGFPnano_TEV_10His culture #1=91,1(ng/µl)
µl taken for Test Digestion:
pJET1.2-cotB cultures #2=1(µl)
pJET1.2-cotB cultures #3=1(µl)
pJET1.2-cotB cultures #4=1(µl)
pJET1.2-cotG cultures #1=1(µl)
pJET1.2-cotG cultures #2=4(µl)
pJET1.2-cotG cultures #3=1(µl)
pJET1.2-cotG cultures #4=4(µl)
pGEX6P1 culture #2=(µl)
pRP261 culture #2=(µl)
pET303_aGFPnano_TEV_10His culture #1=6(µl)
2) Test-digestion
Digestion mixture 1:
Component
concentration
volume [µL]
DNA
500ng/µL
5-6
XbaI
20u/µL
0.1
XhoI
20u/µL
0.1
NEBuffer 2.1
10X
2
ultra pure water
-
ad 20 µL
Digestion mixture 2:
Component
concentration
volume [µL]
DNA
500ng/µL
5-6
PstI
20u/µL
0.1
EcoRI-HF
20u/µL
0.1
NEBuffer 2.1
10X
2
ultra pure water
-
ad 20 µL
1% Agarose Gel of Test Digestion
29.07.16
1) Mini-prep:
A standard Qiagen Mini-Prep was performed with the following cultures:
pJET1.2-cotB cultures #2/3/4, pJET1.2-cotG cultures #1/2/3/4, pET303_aGFPnano_TEV_10His culture #1.
The DNA concentrations were measured with Nanodrop:
pJET1.2-cotB cultures #2=225,5(ng/µl)
pJET1.2-cotB cultures #3=289,5(ng/µl)
pJET1.2-cotB cultures #4=229,1(ng/µl)
pJET1.2-cotG cultures #1=104,7(ng/µl)
pJET1.2-cotG cultures #2=63,2(ng/µl)
pJET1.2-cotG cultures #3=97,6(ng/µl)
pJET1.2-cotG cultures #4=192,1(ng/µl)
pET303_aGFPnano_TEV_10His culture #1=135,1(ng/µl)
µl taken for Test Digestion:
pJET1.2-cotB cultures #2=2(µl)
pJET1.2-cotB cultures #3=2(µl)
pJET1.2-cotB cultures #4=2(µl)
pJET1.2-cotG cultures #1=5(µl)
pJET1.2-cotG cultures #2=6(µl)
pJET1.2-cotG cultures #3=5(µl)
pJET1.2-cotG cultures #4=3(µl)
pET303_aGFPnano_TEV_10His culture #1=4(µl)
2) Test-digestion
Digestion mixture:
Component
concentration
volume [µL]
DNA
500ng/µL
5-6
PstI
20u/µL
0.1
EcoRI-HF
20u/µL
0.1
NEBuffer Smart-Cut
10X
2
ultra pure water
-
ad 20 µL
1% Agerose Gel of Test Digestion
After testdigestion pJET1.2-cotB#3 and pJET1.2-cotG#1 were sent to sequencing.
30.07.16
1) Extension PCR
Extension PCR of of cgeA and aGFP-Nanobody to generate overhangs for Gibson cloning.
Template
Primer
Annealing Temp. [°C]
pJET1.2_cgeA #2
oIG16_044 + 045
62
oIG16_050 + 035
62
pET303_aGFPnano_TEV_10His
oIG16_030 + 031
67
oIG16_051 + 042
67
*Reaction conditions*
Component
concentration
Volume[µL]
Primer fw
10 µM
2.5
Primer rv
10 µM
2.5
Q5 HiFi MasterMix
2x
25
ultra pure H2O
-
19
*PCR program*
Touchdown PCR
Step
Temperature [°C]
Duration
Repeats
Initial denaturation
98
5 min
Denaturation
98
10 s
10X
Annealing
Annealing Temp + 10°C (-1°C per cycle)
10 s
10X
Elongation
72
30 s
10X
Denaturation
98
10 s
20X
Annealing
Annealing temp.
10 s
20X
Elongation
72
30 s
20X
Final elongation
72
5 min
Storage
8
-
After PCR the samples were analyzed by agarose gel electrophoresis (1 % agarose TAE gel).
2) Gel extraction
The bands corresponding to the appropriate sizes were extracted using the QiaQuick gel extraction kit and the DNA was eluted in 30 µL of ultra pure water. The concentration was photometrically determined by a Nanodrop.
Sample
Concentration [ng/µL]
cgeA_oIG16_044+045
85.2
cgeA_oIG16_50+35
62.4
aGFPnano_oIG16_31+30
93.7
aGFPnano_oIG16_51+42
104
31.07.16
1) Restriction digestion
Linearization of pSB1C3 for Gibson Cloning:
Digestion mixture
Component
concentration
volume [µL]
DNA
100ng/µL
5-6
XbaI
20u/µL
0.5
XhoI
20u/µL
0.5
NEBuffer 2.1
10X
10
ultra pure water
-
ad 50 µL
2) Agarose Gel of pSB1c3
→ followed by gel extraction for gibson assembly.
01.08.16
1) NEBuilder® HiFi DNA Assembly Cloning:
Assembly of extensionPCR products into pSB1C3.
No.
amplified fragment1
amplified fragment 2
digested Backbone
Resulting Plasmid
1
cgeA oIG16_50+35
aGFPnano_oIG16_51+42
XbaI - pSB1C3 - SpeI
pSB1C3_cgeA_aHelix_HA_aGFPnano pIG16_038
2
aGFPnano oIG16_30+47
cgeA oIG16_46+45
EcoRI - pSB1C3 - SpeI
pSB1C3_aGFPnano_HA_aHelix_cgeA pIG16_039
Reaction conditions
DNA: 1.5 µL
Fragment1: 25 ng
Fragment2: 15 ng
linearized pSB1C3: 25 ng
2X HiFi MasterMix: 5 µL
ultra pure water: 3.5 µL
The reaction was incubated at 50 °C for 1 hour. 2 µL of the reaction mixture was transformed into chemically competent E.coli DH5alpha provided by the NEBuilder HiFi kit according to their protocol, plated on LB-agar plates containing chloramphenicol and incubated o/n at 37 °C.
The control reaction provided by the kit was performed according to the instructions of the manufacturer.
2) Digestion of pBS1C
Digestion mixture:
Component
concentration
volume [µL]
DNA
148ng/µL
16
XhoI
20u/µL
0.5
NEBuffer CutSmart
10X
10
ultra pure water
-
ad 50 µL
Agarose gel of pBS1c:
→ followed by a Gel extraction for Transformation of competent B.subtilis W168.
02.08.16
1) Inoculations
Inoculation of 2 colonies from each plate with transformed E.coli containing pSB1C3_cgeA_aHelix_HA-Tag_aGFPnano and pSB1C3_aGFPnano_aHelix_HA-Tag_cgeA. Incubation o/n at 37 °C and 250 rpm.
Remaining E.coli from the transformation (see previous day) were plated on LB-agar plates supplemented with cml and incubated o/n at 37 °C.
2) Digestion of pBS1C (For transformation of B.subtilis)
Digestion mixture:
Component
concentration
volume [µL]
DNA
700ng/µL
10
XhoI
20u/µL
1.5
NEBuffer CutSmart
10X
10
ultra pure water
-
ad 50 µL
→ PCR purification was peformed
03.08.16
1) MiniPrep
Plasmid preparation of the inoculated E.coli from the previous day using the QiaQuick Plasmid preparation kit according to the protocol of the manufacturer and eluted in 30 µL of ultra pure water. The DNA concentration was photometrically determined by a NanoDrop:
sample
concentration [ng/µL]
pSB1C3_CgeA_alphaHelix_HA-Tag_aGFPnano #1
69.1
pSB1C3_CgeA_alphaHelix_HA-Tag_aGFPnano #2
126.7
pSB1C3_aGFPnano_alphaHelix_HA-Tag_cgeA #1
120.4
pSB1C3_aGFPnano_alphaHelix_HA-Tag_cgeA #2
182.1
2) Testdigestion
500 ng of DNA was treated with 5 unit of XbaI and PstI in 1x NEBuffer 2.1 in a total reaction volume of 20 µL. Incubation for 1hour at 37 °C.
2) Inoculation
I Inoculation of E.coli DH5alpha containing an mCherry and GFP plasmid (provided by AG Weber) in 5 mL LB containig (Three colonies per plate were picked)
II E.coli DH5alpha containing plasmids sent from Poland (by Dr. Krystof Hinc) for the construction of fusion proteins for spore surface display arrived:
1)pCotG-N
2)pCotG-C
3)pCotB-N
4)pCotB-C
5)pCgeA-C
6)pCotZ-C
The E.coli were spread on LB-Agar plates containing ampicilin and incubated o/n at 37 °C.
III* 6 colonies per plate of E.coli DH5alpha containing pSB1C3_CgeA_alphaHelix_HA-Tag_aGFPnano or pSB1C3_aGFPnano_alphaHelix_HA-Tag_cgeA were inoculated in 5 mL LB-medium supplemented with chloramphenicol and incubated at 37 °C and 250rpm.
04.08.16
1) MiniPrep pIG16_038 + pIG16_039, GFP + mCherry
Plasmid preparation of inoculated E.coli DH5alpha transformed with plasmids containing GFP and mCherry (from AG Weber) using the QiaQuick MiniPrep kit according to the instructions of the manufacturer. Elution with 30 µL of ultra pure water. The concentration was determined by Nanodrop.
No
Sample ID
Nucleic Acid Conc.
1
h2o
0,9
ng/µl
2
pIG16_038 #1
106,9ng/µl
3
pIG16_038 #2
92,4ng/µl
4
pIG16_038 #3
77,3ng/µl
5
pIG16_038 #4
64,4ng/µl
6
pIG16_038 #5
63,2ng/µl
7
pIG16_038 #6
55,4ng/µl
8
pIG16_039 #1
105,2ng/µl
9
pIG16_039 #2
101,3ng/µl
10
pIG16_039 #3
73,2ng/µl
11
pIG16_039 #4
85,2ng/µl
12
pIG16_039 #5
80,4ng/µl
13
pIG16_039 #6
86,7ng/µl
14
pIG16_024 GFP #1
93,2ng/µl
15
pIG16_024 GFP #2
76,3ng/µl
16
pIG16_024 GFP #3
100,9ng/µl
17
pIG16_025mCherry#3
143,2ng/µl
18
pIG16_025mCherry#2
175,8ng/µl
19
pIG16_025mCherry#3
132,8ng/µl
2)Inoculation
Inoculation of the E.coli containing the plasmids sent from Poland. 4 Colonies from each plate were picked and inoculated in 5 mL of LB medium supplemented with Amp.
3)ExtensionPCRs Q5 –> Gel+extraction
No.
Template
Primer: oIG16_#
Annealing Temp[°C]
Resulting construct
1
pET303_aGFPnano
030 + 047
67
Nano_HA_G4S
2
pET303_aGFPnano
053 + 042
67
G4S_HA_Nano
3
pJET_cotG
015 + 016
62
HA_aHelix_CotG
4
pJET_cotG
011 + 013
62
CotG_aHelix_HA
5
pSBBs4S_Sporovector
048 + 049
55
Bb-Prefix_PCotYZ_Bb-Suffix
6
pJET_cgeA
046 + 045
62
HA_G4S_CgeA
7
pJET_cgeA
050 + 052
62
CgeA_G4S_HA
8
dH2O
046 + 045
62
negative control
9
dH2O
015 + 016
62
negative control
10
dH2O
030 + 047
67
negative control
Reaction conditions
Component
Volume µL
2XHiFi MasterMix
25
Primer1
2.5
Primer2
2.5
template DNA
0.1
ultra pure water
19.9
PCR Program
Touchdown PCRs corresponding to the annealing temperatures.
After amplification the samples were loaded on a 1% Agarose TAE gel.
The bands corresponding to the expected fragment sizes were extracted from the gel, purified and eluted with 30 µL of ultra pure water.
05.08.16
1) Testdigestion
pIG16_038+039 were verified by testdigestion. 500 ng of DNA was treated with XbaI and PstI and analyzed by gel electrophoresis.
2) MiniPrep
The inoculated cultures containing the plasmids sent from poland were preparated using the QiaQuick MiniPrep kit according to the protocol. The DNA was eluted with 30 µL of ultra pure water. The concentration was determined by Nanodrop.
3) GFP purification
Sebastian from AG Weber gave a purified GFP (1.35 mg/mL) and mCherry (2 mg/mL).
Stored at 4 °C, protected from light.
06.08.16
1) ExtensionPCR
template
Primer oIG16_
Annealing Temp. [°C]
Resulting construct
pJet1.2_cotG
011 + 013
62
CotG_aHelix_HA
After amplification the sample was loaded on a 1% Agarose TAE gel. The band corresponding to the expected size was extracted and purified.
2) 3A assembly
Digestion mixture PCotYZ:
The B.subtilis spore coat promoter PCotYZ was digested from the Sporovector pIG16_017 provided by Julia Bartels from the Mascher Lab at the TU Dresden.
Component
concentration
volume [µL]
DNA
500ng/µL
13.5
EcoRI
20u/µL
0.1
SpeI
20u/µL
0.1
NEBuffer Cut Smart
10X
2
ultra pure water
-
ad 20 µL
Digestion mixture for pIG16_038+039:
Component
concentration
volume [µL]
DNA
500ng/µL
variable
XbaI
20u/µL
0.1
PstI
20u/µL
0.1
NEBuffer 3.1
10X
2
ultra pure water
-
ad 20 µL
Digestion mixture for pBS1C:
Component
concentration
volume [µL]
DNA
500ng/µL
variable
EcoRI
20u/µL
0.2
PstI
20u/µL
0.2
NEBuffer 2.1
10X
2
ultra pure water
-
ad 20 µL
T4 Ligation
Ligation mixture of plG16_38/39 #1:
Component
concentration
volume [µL]
pBS1C
2000ng/µL
1
plG_38
105ng/µL
4
plG_39
104ng/µL
4
PcotYZ
24ng/µL
0.2
T-4 Ligase
1µl
NEBuffer T-4 Ligase
10X
2
ultra pure water
-
ad 20 µL
The reaction was incubated for 30min at RT and transformed into chemically competent mix and go E. coli DH5alpha
3) Gibson assembly
Assembly of extensionPCR products into pSB1C3.
No.
amplified fragment1
amplified fragment 2
digested Backbone
Resulting Plasmid
1
cgeA oIG16_050+052
aGFPnano_oIG16_53+42
XbaI - pSB1C3 - SpeI
pSB1C3_cgeA_G4S_HA_aGFPnano pIG16_040
2
aGFPnano oIG16_30+47
cotG oIG16_46+45
EcoRI - pSB1C3 - SpeI
pSB1C3_aGFPnano_HA_G4S_cgeA pIG16_041
3
aGFPnano oIG16_030+031
cotG oIG16_015+016
EcoRI - pSB1C3 - SpeI
pSB1C3_aGFPnano_HA_aHelix_cotG pIG16_042
4
cotG oIG16_011+013
aGFPnano oIG16_051+042
EcoRI - pSB1C3 - SpeI
pSB1C3_cotG_aHelix_HA_aGFPnano pIG16_043
Reaction conditions
DNA: 1.5 µL
Fragment1: 25 ng
Fragment2: 15 ng
linearized pSB1C3: 25 ng
2X HiFi MasterMix: 5 µL
ultra pure water: 3.5 µL
The reaction was incubated at 50 °C for 1 hour. 2µL of the reaction mix were transformed into chemically competent E.coli DH5alpha and spread on agarose LB plates supplemented with chloramphenicol. Incubation o/n at 37 °C.
4) Testdigestion of GFP
For verification 500 ng of the plasmid was treated with XbaI, XhoI, BamHI in 1X NEBuffer 3.1 for 90 min at 37°C. The fragments were analyzed by agarose gel electrophoresis.
amp;media=labor:2016_08_06_gfp_testdigestion_invert_label.png" class="mediacenter" alt="" width="300" />
5) Test Digestion Plasmids provided by Krystof Hinc
From university Gdansk. A set of vectors for surface display in B.subtilis.
Digestion mixture cgeA-C/cotZ-C:
Component
concentration
volume [µL]
DNA
500ng/µL
cgeA: #1/2/4=5 #3=8 cotZ: #1/2/3/4=7
XhoI
20u/µL
0.1
PstI
20u/µL
0.1
NEBuffer 3.1
10X
2
ultra pure water
-
ad 20 µL
Digestion mixture cotB-C/N:
Component
concentration
volume [µL]
DNA
500ng/µL
cotB-C: #1/2=5 #3/4=6 cotB-N #1=5 #2=6 #3/4=7
PstI
20u/µL
0.1
NEBuffer 3.1
10X
2
ultra pure water
-
ad 20 µL
Digestion mixture cotG-C:
Component
concentration
volume [µL]
DNA
500ng/µL
#1/2/3/4=2
BamHI
20u/µL
0.1
NEBuffer 3.1
10X
2
ultra pure water
-
ad 20 µL
Digestion mixture cotG-N:
Component
concentration
volume [µL]
DNA
500ng/µL
#1/2/3=2 #4=6
XbhI
20u/µL
0.1
XhoI
20u/µL
0.1
NEBuffer CutSmart
10X
2
ultra pure water
-
ad 20 µL
cgeA-C cotB-C/N
cotB-C/N
cot-Z
07.08.16
1) Repeat of 3A Assembly
Ligation mixture of plG_38/39 #1:
Component
concentration
volume [µL]
pBS1C
2000ng/µL
1
plG16_038
105ng/µL
7
plG16_039
104ng/µL
7
PcotYZ
24ng/µL
0.4
T4 Ligase
1µl
T4 Ligase reaction buffer
10X
2
ultra pure water
-
ad 20 µL
2) Inoculation
Inoculation of Culture #1/2 with construct plG16_038 and Culture #1/2 with construct plG16_039.
08.08.16
1)Miniprep
Standard Qiagen Miniprep of Culture #1/2 with construct plG16_38 and Culture #1/2 with construct plG16_39.
2) Digestion
Digestion mixture poG16_38/39:
Component
concentration
volume [µL]
DNA
500ng/µL
XbaI
20u/µL
0.1
PstI
20u/µL
0.1
NEBuffer 3.1
10X
2
ultra pure water
-
ad 20 µL
Digestion mixture pBS1C:
Component
concentration
volume [µL]
DNA
500ng/µL
EcoRI
20u/µL
0.2
PstI
20u/µL
0.2
NEBuffer 2.1
10X
2
ultra pure water
-
ad 20 µL
3) Repeat of 3A Assembly
Ligation mixture of plG_38/39 #1:
Component
concentration
volume [µL]
pBS1C
2000ng/µL
0.5
plG_38
500ng/µL
1.5
plG_39
500ng/µL
1.5
PcotYZ
24ng/µL
0.5
T-4 Ligase
1µl
NEBuffer T-4 Ligase
10X
2
ultra pure water
-
ad 20 µL
4) Inoculation
Transformed E.coli containing the plasmids pIG16_040 - 043 were inoculated. 6 colonies per plate were picked and inoculated in 5 mL LB medium supplemented with chloramphenicol. Incubation o/n at 37 °C and 250 rpm.
5) Subcloning of PCotYZ
PCotYZ PCR product was subcloned into pJET2.1 vector according to protocol delivered with the kit.
09.08.16
1) MiniPrep
The inoculated E.coli containing the plasmids pIG16_040 - 043 were preparated using the QiaQuick MiniPrep kit. The DNA was eluted with 30 µL of ultra pure water. Verification of the plasmids was performed by testdigestion. 500 ng of DNA was treated with XbaI and PstI in 1X NEBuffer 3.1 for 90 min at 37 °C.
2) Sequencing
pIG16_038#1 and pIG16_039#1 were sent to sequencing by GATCbiotech.
3) ExtensionPCRs
No.
Template
Primer oIG16_
Annealing temp [°C]
Resulting construct
1
pGEX6P1
036 + 054
64
GST_HA_G4S
2
pGEX6P1
041 + 055
64
G4S_HA_GST
3
pJET_cotZ
006 + 008
58
HA_G4S_CotZ
4
pJET_cotZ
003 + 004
58
CotZ_G4S_HA
5
pJET_cotG
014 + 016
62
HA_G4S_CotG
6
pJET_cotG
011 + 012
62
CotG_G4S_HA
7
pJET_cotB
022 + 024
56
HA_G4S_CotB
8
pJET_cotB
019 + 020
56
CotB_G4S_HA
Reaction conditions
Component
volume
2X Q5 MasterMix
25 µL
Primer1
2.5 µL
Primer2
2.5 µL
Template
0.1 µL
water
19.9 µL
PCR Program
Touchdown PCRs corresponding to the respective annealing temperatures.
3) Inoculation of pJET-PcotYZ colonies
Tree colonies from the the plate 1 (Wlad) and Tree colonies from the the plate 2 (iGEM) were picked and inoculated.
10.08.16
1) Gelextraction of ExtensionPCRs Was performed according to the protocol of the manufacturer. The samples were extracted from the gel from the previous day.
2) Sequence analysis
pIG16_038 + 039 #1
Sequencing showed that the samples were switched at some point. pIG16_039 contained an additional insert of ~9 bp between the alpha helical linker and the HA-tag. New samples have to be prepared for analysis.
3) Reinoculation
New colonies containing pIG16_038#2 039#2 were inoculated in 5 mL of LB medium supplemented with chloramphenicol.
4) Miniprep of pJET-PCotYZ
Standard Qiagen miniprep was performed with 3 cultures from the the plate 1 and 3 cultures from the the plate 2 .
5) Test digestion pJET-PcotYZ
Digestion pJET-PcotYZ:
Component
concentration
volume [µL]
DNA
500ng/µL
1-3
EcoRI
20u/µL
0.1
SpeI
20u/µL
0.1
NEBuffer Cut Smart
10X
2
ultra pure water
-
ad 20 µL
Culture #1 was sent to sequencing.
11.08.16
1) MiniPrep
Plasmid preparation of pIG16_038#2 and pIG16_039#2 using the QiaQuick MiniPrep kit. The DNA was eluted with 30 µL of ultra pure water.
1) Sequencing The plasmids pIG16_038#2, pIG16_039#2, pIG16_040#2, pIG16_041#3, pIG16_IG16_042#3 and pJET1.2-PCotYZ#1 were sent to sequencing by GATCbiotech.
12.08.16
1) Sequencing Results
pIG16_040 - pIG16_042 could be confirmed by sequencing. Glycerol stock was prepared.
The samples from pIG16_038 and pIG16_039 were switched in previous steps. –> switched back. pIG16_039 was confirmed by sequencing.
pIG16_038 had a 1bp deletion. Further colonies pIG16_038#3 & #4 were sent to sequencing by GATC-Biotech.
2)MiniPrep of pIG16_039-042 and PcotYZ
Standard Quiagen Miniprep was performed.
3) Digestion of pIG16_039-042, PcotYZ
Digestion mixture pIG16_39-42:
Component
concentration
volume [µL]
DNA
2500ng
5µl
XbaI
20u/µL
0.5
PstI
20u/µL
0.5
NEBuffer 3.1
10X
2
ultra pure water
-
ad 20 µL
Digestion mixture PcotYZ:
Component
concentration
volume [µL]
DNA
5000ng
10
EcoRI
20u/µL
1
SpeI
20u/µL
1
NEBuffer Cut Smart
10X
5
ultra pure water
-
ad 50 µL
4) 3A Assembly
Ligation mixture of plG_38/39 #1:
Component
concentration
volume [µL]
pBS1C
50ng
1
plG_39-41
20ng
1
plG_42
25ng/µL
11
PcotYZ
5.7ng
0.5
T-4 Ligase
20u/µl
1µl
NEBuffer T-4 Ligase
10X
2
ultra pure water
-
ad 20 µL
5) Gibson Cloning
1.33x MasterMix preparation.
5X Iso buffer: 50 µL
T5 Exonuclease (NEB): 1 µL (Diluted 1:10 in ultra pure water)
Taq Ligase (NEB): 25 µL
Phusion Polymerase (NEB): 3.125 µL
Nuclease free water: 108.4 µL
Aliquots: 7.5 µL, stored at -20 °C
Assemblies
All samples were adjusted to 50 ng/µL
No.
Fragment 1
Fragment 2
digested backbone
resulting construct
1
GST oIG16_036+054
CotG oIG16_014+016
pSB1C3
pIG16_046
2
GST oIG16_036+054
CgeA oIG16_046+045
pSB1C3
pIG16_047
3
GST oIG16_036+054
CotZ 006+008
pSB1C3
pIG16_048
4
GST oIG16_036+054
CotB 022+024
pSB1C3
pIG16_049
5
GST oIG16_041+055
CgeA 050+052
pSB1C3
pIG16_050
6
GST oIG16_041+055
CotZ 003+004
pSB1C3
pIG16_051
7
GST oIG16_041+055
CotB 019+020
pSB1C3
pIG16_052
8
aGFPnano oIG16_030+047
CotG 014+016
pSB1C3
pIG16_053
9
aGFPnano oIG16_030+047
CotZ 006+008
pSB1C3
pIG16_054
10
aGFPnano oIG16_030+047
CotB 022+024
pSB1C3
pIG16_055
11
positive control
positive control
postive control
positive control from NEB HiFi Assembly kit
0.5 µL of each fragment and 0.5 µL of the digested backbone were mixed with 1µL of ultra pure water and added to the 1.33x Gibson Mix and incubated for 60 min at 50 °C. 5µL of the reaction mix were used to transform chemically competent E.coli DH5alpha. Subsequently, 300 µL of LB medium were added. The Bacteria were incubated at 37 °C and 250 rpm and spread on Agar-LB plates supplemented with chloramphenicol.
13.08.16
1) Extension PCR
template
Primer
Resulting construct
pJET1.2_CotZ
oIG16_007+008
HA_alphaHelix_CotZ_pSB1C3-OH
pJET1.2_CotZ
oIG16_003+005
pSB1C3-OH_CotZ_alphaHelix_HA
Reaction conditions
component
volume
2XMM
25 µL
Primer1
2.5 µL
Primer2
2.5 µL
template
0.1 µL
dH2O
19.9 µL
Cycler Program Touchdown58
The samples analyzed by agarose gel electrophoresis and the bands corresponding to the expected sizes were extracted and purified using the Qiagen gel extraction kit.
2) Inoculation The transformed E.coli from the Gibson assembly (previous day) were inoculated in 5 mL LB-medium supplemented with chloramphenicol and incubated o/n at 37 °C and 250 rpm.
3) Gibson assembly
No.
Fragment1
Fragment2
Digested Backbone
resulting construct
1
pJet1.2_cotZ oIG16_007+008
aGFPnano oIG16_30+31
pSB1C3
pIG16_056
2
pJet1.2_cotZ oIG16_006+008
aGFPnano oIG16_30+47
pSB1C3
pIG16_057
3
pJet1.2_cotZ oIG16_003+005
aGFPnano oIG16_51+42
pSB1C3
pIG16_058
4
pJet1.2_cotZ oIG16_003+004
aGFPnano oIG16_53+42
pSB1C3
pIG16_059
5
positive control
positive control
positive control
C+ from NEB
0.5 µL from each fragment were mixed with 1 µL of ultra pure water, added to 1.33x Gibson Master Mix and incubated at 50 °C for 1 h. 2µL of the reaction mix were transformed into chemically competent E.coli DH5alpha.
14.08.16
1) MiniPrep
Plasmid preparation of pIG16_046 - 055 using the QiaQuick MiniPrep kit. The DNA was eluted in 30 µL of ultra pure water. Verification of the plasmids by Testdigestion using. 500 ng of DNA was treated with 2 unit of XbaI and PstI in 1X NEBuffer 3.1 for 1 hour. . Unexpected band patterns. The DNA was not completly digested.
2) Inoculation
The transformed E.coli with the Gibson assemblies from the previous day were inoculated in 5 mL LB-medium supplemented with chloramphenicol
15.08.16
1) Testdigestion
The digestion of pIG16_046 - 055 from the previous day was repeated. 500 ng of DNA was treated with 4 unit of XbaI and PstI in 1X NEBuffer 2.1 for 90 min at 37 °C.
Positive samples were sent to sequencing:
1) pIG16_047#3
2) pIG16_048#1
3) pIG16_050#5
4) pIG16_051#1
5) pIG16_054#1
6) pIG16_055#2
16.08.16
1) Production of competent DH5-alpha E. coli
E. colis were made competent according to zymo research Mix and go protocol
17.08.16
1) Inoculation
Since pIG16_046, 049, 052 were all negative 5 additional colonies were picked and inoculated in 5 mL of LB medium supplemented with chloramphenicol.
5 colonies of the E.coli transformed with the 3A assembly reaction (pIG16_062 - 065) were pickend and inoculated in 5 mL of LB medium supplemented with ampicilin.
5 colonies of the E.coli transformed with the Gibson assembly reaction (pIG16_057 - 059).
2) Glycerol stocks
Preparation of Glycerol stocks of pIG16_038, 039, 040, 041, 042
18.08.16
1) Plasmid preparation
Preparation of plasmids from transformed E.coli containing the constructs pIG16_46 + 049 + 052+ 058+ 059+ 062+ 063+ 064+ 065
2) Testdigestion
3A assembly: NotI + XhoI
Gibson assembly: XbaI + PstI
Digestions partially incomplete.
Samples sent to sequencing:
3) Sequencing
Sample
Sequencing Primer
pIG16_046#7
oIG16_039
pIG16_046#7
oIG16_040
pIG16_049#6
oIG16_039
pIG16_049#6
oIG16_040
pIG16_052#10
oIG16_039
pIG16_052#10
oIG16_040
pIG16_058#2
oIG16_039
pIG16_058#2
oIG16_040
pIG16_059#5
oIG16_039
pIG16_059#5
oIG16_040
pIG16_062#3
oIG16_028
pIG16_062#3
oIG16_029
pIG16_063#2
oIG16_028
pIG16_063#2
oIG16_029
pIG16_064#3
oIG16_028
pIG16_064#3
oIG16_029
pIG16_065#5
oIG16_028
pIG16_065#5
oIG16_029
19.08.16
1) Glycerol stock
Preparation of a 15 % Glycerol stock of pIG16_039 and storage at - 80 °C.
2) Reinoculation
New 6 mL LB cultures were prepared for pIG16_047#3, 050#5, 048#4, 051#1 054#4 and incubated o/n at 37 °C and 250 rpm. For subsequent 3A assembly.
3) Preparation of competent E.coli
The Ecoli strain DH5-alpha was prepered and made competent according to Zymoreserch MIX and GO kit.
20.08.16
1) Extension PCR
Template
Primer
Resulting construct
pJet1.2_CotG
oIG056+012
pSB1C3-OH_CotG_G4S_HA
pJet1.2_CotG
oIG16_056+013
pSB1C3-OH_CotG_aHelix_HA
Reaction conditions
component
concentration
volume[µL]
Q5 MasterMix
2X
25
Primer1
10 µM
2.5
Primer2
10 µM
2.5
ultrapure water
-
20.0
template DNA
0.1
Thermal Cycling
Touchdown 62
2) Gibson Assembly
Number
Fragment 1
Fragment 2
Linearized backbone
Resulting construct
1
CotG oIG16_056+013
aGFPnano oIG16_051+042
pSB1C3 XbaI+SpeI
pIG16_043
2
CotG oIG16_056+012
aGFPnano oIG16_053+042
pSB1C3 XbaI+SpeI
pIG16_065
3
CotG oIG16_056+013
GST oIG16_041+055
pSB1C3 XbaI+SpeI
pIG16_066
4
CotG oIG16_014+016
aGFPnano oIG16_030+047
pSB1C3 XbaI+SpeI
pIG16_053
The concentration of all fragments was adjusted to 50 ng/µL. 0.5 µL of each fragment were mixed alongside with 0.5 µL of the linearized vector and added to the 1.33x Gibson MasterMix and incubated for 60 min at 50 °C.
5µL of the Gibson mix were transformed into chemically competent E.coli DH5alpha and spread on LB agar plate supplemented with chloramphenicol.
3) Digestion for 3 A Assembly
Digestion of PcotYZ
component
concentration
volume[µL]
PcotYZ
6000ng
20µl
CS Buffer
10x
5µl
EcoRI-HF
20u/µl
1.2µl
SpeI-HF
20u/µl
1.2µl
ultrapure water
-
50.0
Digestion of pBS1C and pBS4S
component
concentration
volume[µL]
pBS1C and pBS4S
5000ng
2,5µl
2.1 Buffer
10x
5µl
EcoRI-HF
20u/µl
1.2µl
PstI
20u/µl
1.2µl
ultrapure water
-
50.0
Digestion of pIG16_38 47 48 50 51 54
component
concentration
volume[µL]
pIG16_#
3000ng
20µl
3.1 Buffer
10x
5µl
XbaI
20u/µl
0.6µl
PstI
20u/µl
0.6µl
ultrapure water
-
50.0
4) 3 A Assembly
The 3 A Assembly was carried out according to the neb protocol.
3 A Assembly with pBS1C
component
concentration
volume[µL]
pBS1C
102ng/µl
50ng
T4 ligase Buffer
10x
2µl
T4 ligase
20u/µl
1µl
PcotYZ
20ng/µl
5ng
pIG16_38/47/48/50/51/54
differentng/µl
25ng/µl
ultrapure water
-
20.0
3 A Assembly with pBS4S
component
concentration
volume[µL]
pBS4S
102ng/µl
50ng
T4 ligase Buffer
10x
2µl
T4 ligase
20u/µl
1µl
PcotYZ
20ng/µl
5ng
pIG16_38/47/48/50/51/54
differentng/µl
25ng/µl
ultrapure water
-
20.0
21.08.16
1) Inoculation
5 colonies per plate were inoculated in 5 mL of LB medium supplemented with chloramphenicol from the Gibson assembly from the previous day. Incubation o/n at 37 °C and 250 rpm.
Reinoculation of pIG16_046#7, 049#6, 052#10, 058#2, 059#5, 062#3
2) Glycerol stocks
Preparation of 15 % glycerol stocks for:
pIG16_045#3, 062#2, 063#3, 064#5
3) Linearization #1
3 µg of the following plasmids were linearized using the appropriate enzyme in a reaction volume of 50 µL. The digestion was loaded on a TAE agarose gel and subjected to electrophoresis.
The linearized plasmids were extracted from the gel using the Qiagen gel extraction kit and subsequently transformed into competent Bacillus subtilis.
4) Linearization #2
Digestion of pIG16_77/81/82/100/101
component
concentration
volume[µL]
pIG16_77/81/82/100/101
2000ng
4,5µl
Cutsmart
10x
5µl
XhoI
20u/µl
0.5µl
ultra pure water
-
ad50.0
Digestion of pIG16_104/105/117
component
concentration
volume[µL]
pIG16_104/105/117
2000ng
4,5µl
CS Buffer
10x
5µl
EcoRV-HF
20u/µl
0.5µl
ultrapure water
-
ad50.0
→Linearization was followed by gel extraction for transformation in Bacillus subtilis.
4) Inoculation of 3 A Assemblies colonies.
22.08.16
1) Plasmid preparation
The plasmids of the following strains were extracted using the QiaQuick MiniPrep kit:
pIG16_043 053 065 066
500 ng of DNA was digested with 5 units of XbaI and PstI and analyzed by gel electrophoresis.
2) Glycerol stocks Glycerol stocks of pIG16_045 062 063 064 46 were prepared and stored at -80 °C.
3) Sequencing pIG16_058#3 and pIG16_058#4 were sent to sequencing.
4) Inoculation new colonies picked of pIG16_049 #11 - 16 and pIG16_052 #11-16 were picked
5) Transformation
The Biobrick BBa_J18932 from the Registry distribution kit was dissolved in 10 µL of ultrapure water. 2 µL were used for transformation of chemically competent E.coli DH5alpha and spread on a LB agar plate supplemented with chloramphenicol.
6) Testdigestion of 3 A assembly from 20.9
Digestion 3 A assembly
component
concentration
volume[µL]
pIG16_
500ng
4µl
2.1 Buffer
10x
5µl
EcoRI-HF
20u/µl
0.1µl
PstI
20u/µl
0.1µl
ultrapure water
-
ad20.0
→Colonies were sent to sequencing.
23.08.16
1) Plasmid preparation
Miniprep of pIG16_049#11-16 and pIG16_052#11-16. Testdigestion: 500 ng of DNA was treated with 5 unit of XbaI and PstI and analyzed by gel electrophoresis.
2) Preparation of Plasmids for transformation of B.subtilis
MiniPrep of pIG16_045 + 062\ Linearization of 2µg of DNA with a 10 unit of ScaI. The DNA was purified with the QiaGen PCR purification kit and used for transformation of competent B. subtilis.
24.08.16
1) Sequencing
The following samples were sent to sequencing:
pIG16-044, 059#4, 058#3, 077, 082
2) Plasmid preparation
The DNA of pIG16_043#6-11 was extracted using the QiaQuick Miniprep kit.
3) Test digestion
500ng of pIG16_043, pIG16_120(BBa_J18932) were treated with 5 unit of XbaI+PstI and analyzed by gel electrophoresis.
4) Sequencing
The following plasmids were sent to sequencing:
pSB1C3_mCherry#2 (BBa_J18932), pIG16_044#1, 049#12, 058#3, 059#4, 077#1, 082#1, 101#1, 104#1, 105#1, 117#5
25.08.16
1) Inoculations
Inoculations of the following samples for subsequent plasmid preparations and glycerol stocks: pIG16_045, 062, 063, 064, 046#7, 049#12, 059#4, 056#1, 058#3, 101#1, 044#1, 082#2, 104#1, 077#1, 081#1,105#2, 100#1, 117#4
26.08.16
1) Plasmid preparation
The plasmids from the previous inoculation were extracted using the Qiagen Plasmid MiniPrep kit. The DNA was eluted in 30 µL of ultra pure water and the DNA concentration was determined by NanoDrop.
2) Sequencing
The following plasmids were sent to sequencing: pIG16_081#1, 100#1, 117#4
3) Preparation of Plasmids for Transformation of B.subtilis
2 µg of the following plasmids were linearized by treatment with ScaI-HF in 1X cutsmart buffer at a total volume of 50 µL.
pIG16_045, 062, 063, 064, 117, 104, 044, 077, 081, 082, 100, 101, 105.
The linearized plasmids were loaded on a 1% TAE agarose gel and extracted after electrophoresis and used for transformation of competent B. subtilis.
27.08.16
1) Digestion for 3A assembly
Digestion 1 µg of DNA of pIG16_046, 049, 053, 059, 065, 066, 053 with 10 units of XbaI and PstI in 1XCutSmart buffer in a reaction volume of 50 µL. The digested samples were loaded on an 1 % agarose TAE gel and subjected to electrophoresis.
Digestion 3 A assembly
component
concentration
volume[µL]
pIG16_60
6000ng
10µl
2.1 Buffer
10x
2µl
EcoRI-HF
20u/µl
1.2µl
SpeI
20u/µl
1.2µl
ultrapure water
-
ad20.0
Digestion 3 A assembly
component
concentration
volume[µL]
pBS1C/4S/E2
2000ng
4µl
CS Buffer
10x
2µl
EcoRI-HF
20u/µl
0.4µl
PstI
20u/µl
0.4µl
ultrapure water
-
ad20.0
Digestion 3 A assembly
component
concentration
volume[µL]
pIG16_46/47/49/56/65/mCherry
1000ng
4µl
3.1 Buffer
10x
2µl
XbaI
20u/µl
0.5µl
PstI
20u/µl
0.5µl
ultrapure water
-
ad20.0
After separation the appropriate fragments were extracted from the gel using the QiaGen gel extraction kit.
2) Extension PCR
No.
Template
Oligos oIG16_
Resulting construct
1
pIG16_22(GST)
37+55
aHelix_HA_GST_pSB1C3-OH
2
pIG16_22(GST)
36+38
pSB1C3-OH_GST_HA_aHelix
3
pIG16_030(CotB)
23+24
HA_aHelix_CotB_pSB1C3-OH
4
pIG16_030(CotB)
19+21
pSB1C3-OH_CotB_aHelix_HA
Reaction conditions
component
volume[µL]
2X Q5 MasterMix
25
Primer1 10 M
2.5
Primer2 10 M
2.5
template
<0.1
dH2O
20
The samples were loaded on an 1 % Agarose TAE gel and subjected to electrophoresis.
After separation the appropriate fragments were extracted using the QiaGen gel extraction kit.
3) Gibson Master Mix
Preparation of 1.33x Gibson Master Mix.
4) Inoculation
4 colonies of each plate were picked from the previous transformation: pIG16_079, 085, 088, 096, 108, 112, 121, 122, 123
5) 3 A assembly
3 A Assembly with pBS1C
component
concentration
volume[µL]
pBS1C
102ng/µl
50ng
T4 ligase Buffer
10x
2µl
T4 ligase
20u/µl
1µl
PcotYZ
20ng/µl
9.5ng
pIG16_46/49/53/56/65/mCherry
different ng/µl
41ng/µl
3 A Assembly with pBS2E
component
concentration
volume[µL]
pBS2E
80ng/µl
50ng
T4 ligase Buffer
10x
2µl
T4 ligase
20u/µl
1µl
PcotYZ
20ng/µl
9.5ng
pIG16_mCherry
different ng/µl
28ng/µl
3 A Assembly with pBS4S
component
concentration
volume[µL]
pBS4S
75ng/µl
50ng
T4 ligase Buffer
10x
2µl
T4 ligase
20u/µl
1µl
PcotYZ
20ng/µl
12.5ng
pIG16_46/49/mCherry
different ng/µl
55ng/µl
28.08.16
1) Inoculation
Colonies from the 3 A assembly (27.08) were picked.
29.08.16
1)Sequencing The following plasmids were sent to sequencing: pIG16_032 033 034 035. (Integration vectors sent from Dr. Hinc, University of Gdansk)
2) 3A assembly
3 A Assembly with pBS1C
component
concentration
volume[µL]
pBS1C
102ng/µl
50ng
T4 ligase Buffer
10x
2µl
T4 ligase
20u/µl
1µl
PcotYZ
20ng/µl
9.5ng
pIG16_58/59/66
different ng/µl
55ng/µl
3 A Assembly with pBS4S
component
concentration
volume[µL]
pBS4S
75ng/µl
50ng
T4 ligase Buffer
10x
2µl
T4 ligase
20u/µl
1µl
PcotYZ
20ng/µl
12.5ng
pIG16_66
different ng/µl
55ng/µl
30.08.16
1) Sequencing
The following plasmids were sent to sequencing: pIG16_112#3, 121#3, 122#2, 079#1, 085#3, 088#4, 096#1
2) Inoculation
E.coli DH5alpha containing the following plasmids were inoculated in 5 mL LB medium supplemented with the appropriate antibiotics. Incubation o/n at 37 °C and 200 rpm. pIG16_100, 117, 123, 081, 104, 077, 082, 122, 121, 101, 045, 062, 063, 064, 032, 033, 034, 035
3) Gibson assembly
The Fragments for Gibson assembly were adjusted to a concentration of 50 ng/µL.
Number
Fragment1 amplified with oIG16_
Fragment2 amplified with oIG16_
Linearized backbone
resulting construct
1
CotZ oIG16_007 + 008
GST 036 + 038
pSB1C3
pIG16_067
2
CotZ 003 + 005
GST 037 + 055
pSB1C3
pIG16_068
3
CotG 056 + 013
aGFPnano 051 + 042
pSB1C3
pIG16_043
4
CotG 015 + 015
GST 036 + 038
pSB1C3
pIG16_069
5
CotG 056 + 013
GST 037 + 055
pSB1C3
pIG16_070
6
CgeA 044 + 045
GST 036 + 038
pSB1C3
pIG16_075
7
CgeA 050 + 035
GST 037 + 055
pSB1C3
pIG16_076
8
CotB 019 + 020
GST 037 + 055
pSB1C3
pIG16_052
9
CotB 022 + 024
aGFPnano 030 + 047
pSB1C3
pIG16_055
10
CotB 023 024
GST 036 + 038
pSB1C3
pIG16_073
11
CotB 019 + 021
GST 037 + 055
pSB1C3
pIG16_074
12
positive control
positive control
positive control
-
4) Linearization
Linearization of integration vectors for transformation of B.subtilis: 2 µg of the DNA was treated with 20 units of an appropriate enzyme in 1X cutsmart buffer at a total reaction volume of 50 µL for 2 hours at 37 °C.
Vector pIG16_
Containing coat gene
Insert locus
Enzyme for linearization
077
CotZ
AmyE
XhoI
081
CotZ
AmyE
XhoI
082
CotZ
AmyE
XhoI
100
CgeA
AmyE
XhoI
045
CgeA
AmyE
XhoI
062
CgeA
AmyE
XhoI
063
CgeA
AmyE
XhoI
064
CotG
AmyE
XhoI
108
CotG
thrC
NcoI
123
mCherry
thrC
ScaI
104
CotZ
thrC
NcoI
105
CotZ
thrC
NcoI
117
CgeA
thrC
NcoI
101
CgeA
AmyE
XhoI
After linearization the DNA was purified using the QiaGen PCR purification kit. The DNA was used for transformation of B. subtilis.
31.08.16
1) Glycerolstocks
15 % Glycerol stocks of pIG16_032 033 034 035 were prepared and stored at -80 °C.
2) Sequencing
The following samples were sent to sequencing by GATC labtech pIG16_044#4, 078#4 080#3 086#2 089#4 109#4
3) Plasmid preparation
The following plasmids were prepared using the QiaQuick Miniprep kit: pIG16_032 33 34 35 45 62 63 64 123 117 81 82 100 117 123 121 122 101
4) Gibson assembly
The fragments were adjusted to a concentration of 50 ng/µL
No
amplified Fragment 1
amplified Fragment 2
Backbone
Resulting plasmids pIG16_
1
CotG oIG16_056+013
aGFPnano 51 + 42
pSB1C3
043
2
CotZ oIG16_7+8
GST_37+55
pSB1C3
067
3
CotG 056+013
aGFPnano 051 + 042
pSB1C3
052
4
CotG 015 + 016
GST 036 + 038
pSB1C3
055
5
cotG 056 + 013
GST 037 + 055
pSB1C3
070
6
cgeA 044 + 045
GST 036 + 038
pSB1C3
068
7
cgeA 050 + 035
GST 037 + 055
pSB1C3
069
8
cotB 019 + 020
GST 037 + 055
pSB1C3
073
9
cotB 002 + 024
aGFPnano 030 + 047
pSB1C3
074
10
cotB 023 + 024
GST 036 + 038
pSB1C3
075
11
cotB 019 + 021
GST 037 + 055
pSB1C3
076
0.5 µL of each fragment were mixed alongside with 0.5 µL of the backbone. The volume was adjusted to 2.5 µL and added to 7.5 µL of 1.33X Gibson Master Mix. The Reaction was incubated for 1h at 50 min and transformed into competent E. coli DH5alpha. The transformed cells were plated on LB agar plates supplemented with chloramphenicol.
01.09.16
1) Inoculation
Inoculation of the colonies from the transformation of the Gibson assembly: pIG16_043 067 052 055 070 068 069 073 074 075 076. 5 colonies were picked and inoculated in 5 mL of LB medium with chloramphenicol. Incubation o/n at 37°C and 200 rpm.
Inoculation of pIG16_043 070 074 067 068 075 073 055 076 052 069 in 5 mL LB medium supplemented with ampicilin.
02.09.16
1) Plasmid Preparation
The following samples were prepared from E.coli DH5alpha by a MiniPrep:
I) From Gibson assembly (Previous day): pIG16_043 067 052 055 070 068 069 073 074 075 076
II) For Transformation of B.subtilis: pIG16_077 81 82 100 45 62 63 64 108 123 104 105 117 101 122
2) Linearization of plasmids for transformation of B.subtilis
Linearization of integration vectors for transformation of B.subtilis: 2 µg of the DNA was treated with 20 units of an appropriate enzyme in 1X cutsmart buffer at a total reaction volume of 50 µL for 2 hours at 37 °C.
Vector pIG16_
Containing coat gene
Insert locus
Enzyme for linearization
077
CotZ
AmyE
XhoI
081
CotZ
AmyE
XhoI
082
CotZ
AmyE
XhoI
100
CgeA
AmyE
XhoI
045
CgeA
AmyE
XhoI
062
CgeA
AmyE
XhoI
063
CgeA
AmyE
XhoI
064
CotG
AmyE
XhoI
108
CotG
thrC
NcoI
123
mCherry
thrC
ScaI
104
CotZ
thrC
NcoI
105
CotZ
thrC
NcoI
117
CgeA
thrC
NcoI
101
CgeA
AmyE
XhoI
122
mCherry
LacA
NgoMIV
After linearization the DNA was purified using the QiaGen PCR purification kit. The DNA was used for transformation of B. subtilis.
03.09.16
1) Test digestion
For the verification of the proper sizes 500 ng the following plasmids were treated with 5 U of XbaI and PstI and analyzed by gel electrophoresis on a 1 % Agarose TAE gel:
pIG16_043 067 052 055 070 068 069 073 074 075 076
The expected bands were not visible.
04.09.16
1) Annealed oligo cloning
In order to insert an alpha helical linker into the Sporovector from the Munich iGEM team 2012 the oligos oIG16_079 and oIG16_080 were annealed at a molar ratio of 1:1, heated at 100 °C for 30 min and slowly cooled down. The annealed oligos contained NgoMIV corresponding sticky ends and were stored at 4 °C. 2 µg of the vector pIG16_017 (Sporovector) was linearized with NgoMIV and purified. The linearized backbone and the annealed oligos were ligated at molar ratios of 1:1, 1:2 and 1:3 using 1 µL of T4 ligase (from NEB) in 1X T4 buffer in a total reaction volume of 20 µL. The reaction was incubated for 30 min at RT and transformed into chemically competent E.coli. The cells were plated on LB agar plates supplemented with ampicilin.
05.09.16
1)Inoculation
Inoculation of E.coli containing integration vectors from glycerol stocks: pIG16_077 81 82 100 45 62 63 64 108 123 104 105 117 101 122. Incubation in 5 mL LB medium supplemented with ampicilin. Incubation o/n at 37 °C and 200 rpm.
Inoculation of colonies from the annealed oligo cloning. 5 colonies per plate were inoculated in 5 mL of LB medium with ampicilin. Incubation o/n at 37 °C and 200 rpm.
06.09.16
1) Plasmid preparation
MiniPrep of integration vectors: pIG16_077 81 82 100 45 62 63 64 108 123 104 105 117 101 122
pIG16_124 (at molar ratios 1:1, 1:2, 1:3,each #1-5).
Gel image: NIKLAS???
2) Sequencing
pIG16_124 1:1 #2
pIG16_124 1:2 #3
pIG16_124 1:3 #1
07.09.16
1) Plasmid Preparation
MiniPrep of further colonies of pIG16_043 067 052 055 070 068 069 073 074 075 076
2) Testdigestion 500 ng of DNA was treated with 5 units of XbaI + PstI in 1X NEBuffer 2.1 in a total reaction volume of 20 µL. Incubation for 1 h at 37 °C. The reaction was analyzed by gel electrophoresis.
08.09.16
1) Sequencing
Gibson assembly reactions were sent to sequencing:
pIG16_043 067 052 055 070 068 069 073 074 075 076
2) Digestion
I pIG16_017: 2 µg of plasmid DNA was treated with 10 unit of XbaI and NgoMIV in 1X Cutsmart buffer in a total reaction volume of 50 µL for 2 hours at 37 °C. For further assembly by Freiburg Standard RFC25.
II pIG16_020: 2 µg of plasmid DNA was treated with 10 unit of XbaI and SpeI-HF in 1X Cutsmart buffer in a total reaction volume of 50 µL for 2 hours at 37 °C for further gibson assembliy reactions.
3) PCR
The biobrick BBa_J18932 (pIG16_120) containing mCherry was amplified in include additional restriction site for Freiburg standard assembly RFC25.
Reaction conditions
Component
concentration
volume in µL
Q5 Master Mix
2X
37.5
DNA template
-
1 µL
oIG16_075
10 µM
3.75
oIG16_076
10 µM
3.75
dH2O
-
29
Amplified by Touchdown-PCR. The annealing temperature was determined by the NEB online tool Tm-calculator.
09.09.16
1) Gel extraction
The digestions and PCR reaction from the previous day were extracted from an agarose gel accorind the the protocol of the manufacturer.
2) Digestion
1 µg of the amplified mCherry (pIG16_120 + oIG16_075+076) was treated with AgeI and XbaI in 1X cutsmart buffer in a total reaction volume of 50 µL for 90 min at 37 °C. The DNA was directly purified using the Qiagen PCR purification kit according to the instructions of the manufacturer.
3) T4 Ligation
Ligation of digested mCherry (Insert) into the digested pIG16_017 (Sporovector) at a molar ratio of 1:3 (Vector : Insert) in 1X T4-ligation buffer at a total reaction volume of 20 µL. The ligation was incubated for 30 min at RT and transformed into chemically competent E.coli.
4) Gibson Assembly
No
amplified Fragment 1
amplified Fragment 2
Backbone
Resulting plasmids pIG16_
1
CotZ oIG16_7+8
GST_37+55
pSB1C3
067
2
cgeA 044 + 045
GST 036 + 038
pSB1C3
068
3
cotB 019 + 020
GST 037 + 055
pSB1C3
073
4
cotB 002 + 024
aGFPnano 030 + 047
pSB1C3
074
5
cotB 023 + 024
GST 036 + 038
pSB1C3
075
6
cotB 019 + 021
GST 037 + 055
pSB1C3
076
7
pos. control
positive control
positive control
-
The reaction was incubated for 1 h at 50 °C and tranformed into chemically competent E.coli DH5 alpha.
10.09.16
1) Inoculation
I From each plate with E.coli transformed with the gibson assemblies 5 colonies were picked and inoculated in 5 mL of LB medium supplemented with chloramphenicol.
II Further colonies of pIG16_043, 052, 126 were picked and inoculated.
11.09.16
1) Linearization
3.5 µg of the following plasmids were linearized with the listed restriction enzyme in the appropriate buffer for transformation into B. subtilis:
Plasmid pIG16_
coat gene
Insert Locus/Selection marker
Enzyme
044
cgeA
1C
XhoI
045
cgeA
1C
XhoI
062
cgeA
1C
XhoI
063
cgeA
1C
XhoI
064
cotG
1C
XhoI
077
cotZ
1C
XhoI
078
cotZ
1C
XhoI
079
cotZ
1C
XhoI
080
cotZ
1C
XhoI
081
cotZ
1C
XhoI
082
cotZ
1C
XhoI
085
cotG
1C
XhoI
086
cotG
1C
XhoI
088
cotG
1C
XhoI
089
cotG
1C
XhoI
096
cotB
1C
XhoI
100
cgeA
1C
XhoI
101
cgeA
1C
XhoI
104
cotZ
4S
NcoI
105
cotZ
4S
NcoI
108
cotG
4S
NcoI
109
cotG
4S
NcoI
117
cgeA
4S
NcoI
122
mCherry
2E
NgoMIV
123
mCherry
4S
ScaI
After linearization 5 µL of the reaction was analyzed by agarose gel electrophoresis. The remaining volume was purified using the QiaGen PCR purification kit. The eluted DNA was used for transformation of B. subtilis.
2) MiniPrep
The plasmids from the following inoculations were prepared using the ZymoResearch Zyppy Prep kit:
pIG16_043 052 067 068 070 073 074 075 076 126 (colonies #5-9 each).
3) Testdigestion
500 ng of the prepared plasmids were treated with 5 units of XbaI and PstI in 1X NEBuffer 2.1 in a total reaction volume of 20 µL for 2 hours at 37 °C. The Digestion was analyzed by agarose gel electrophoresis.
12.09.16
13.09.16
1) PCR
Amplification of the Sporovector pSBBS4S-Sporo (pIG16_017) in order to introduce an alpha helical linker to the MCS by Gibson assembly.
template
Oligos oIG16_
pIG16_017
079+082
pIG16_017
081+080
Reaction conditions
component
conc.
volume
Q5 MasterMix
2x
25µL
Primer fw
10 µM
2.5
Primer rv
10 µM
2.5
template
-
0.1
dH20
-
20µL
Cycling
Touchdown PCR 60 and 67
2) Gibson Assembly
The two amplified fragments were mixed at a ratio of 1:1 (50 ng each) and incuabted in NEB HiFi assembly mix at 50 °C for 1 hour. Transformation of 2µL into chemically competent E.coli and spreaded on LB agar plates supplemented with ampicilin.
14.09.16
1) Linearization
3.5 µg of the following plasmids were linearized with the listed restriction enzyme in the appropriate buffer for transformation into B. subtilis:
Plasmid pIG16_
coat gene
Insert Locus/Selection marker
Enzyme
044
cgeA
1C
XhoI
045
cgeA
1C
XhoI
062
cgeA
1C
XhoI
063
cgeA
1C
XhoI
064
cotG
1C
XhoI
077
cotZ
1C
XhoI
078
cotZ
1C
XhoI
079
cotZ
1C
XhoI
080
cotZ
1C
XhoI
081
cotZ
1C
XhoI
082
cotZ
1C
XhoI
085
cotG
1C
XhoI
086
cotG
1C
XhoI
088
cotG
1C
XhoI
089
cotG
1C
XhoI
096
cotB
1C
XhoI
100
cgeA
1C
XhoI
101
cgeA
1C
XhoI
104
cotZ
4S
NcoI
105
cotZ
4S
NcoI
108
cotG
4S
NcoI
109
cotG
4S
NcoI
117
cgeA
4S
NcoI
122
mCherry
2E
NgoMIV
123
mCherry
4S
ScaI
After linearization 5 µL of the reaction was analyzed by agarose gel electrophoresis. The remaining volume was purified using the QiaGen PCR purification kit. The eluted DNA was used for transformation of B. subtilis.
update: Gel image
15.09.16
1) PCR
Amplification of anti-GFP nanobody and mCherry to introduce additional overhangs containing restriction site.
template
Oligos
Restriction site
1
pIG16_023
oIG16_083 + 084
EcoRI + XbaI
2
pIG16_023
oIG16_085 + 086
NcoI + NheI
3
pIG16_020 (mCherry)
oIG16_087 + 088
XhoI + XbaI
Reaction conditions
component
conc
volume [µL]
Q5 MasterMix
2x
25
Primer 1
10 µM
2.5
Primer 2
10 µM
2.5
template
-
0.1
dH2O
-
20
Cycling
pIG16_023: Touchdown68
pIG16_020: Touchdown66
After amplification the reaction was analyzed by agarose gel electrophoresis.
2) Transformation
BBa_K180000 (Lac regulated taq promoter) from the iGEM distribution kit was dissolved in 10 µL of ultra pure water. 2 µL were used for the transformation of chemically competent E.coli DH5alpha and spread on an LB agar plate supplemented with chloramphenicol.
3) Oligo design
The PCotYZ-Promoter (BBa_K823030) did not contain a ribosome binding site. An additional RBS is introduced by primers containing an additional insert.
16.09.16
1) Gel extraction
The amplified samples from the previous day were extracted from the gel and purified using the QiaGen gel extraction kit.
2) Digestion
1µg of the amplified nanobody/mCherry were digested with the 10 unit of the appropriate enzymes in 1X Reaction buffer in a total reaction volume of 30 µL in order to clone them into their respective backbone.
DNA
oligos
Restriction enzymes
aGFPnano
83 + 84
EcoRI + XbaI
aGFPnano
85 + 86
NcoI-HF + NheI-HF
mCherry
87 +
XhoI + XbaI
pIG16_033
-
NcoI-HF + NheI-HF
pIG16_034
-
XhoI + XbaI-HF
pIG16_035
-
EcoRI-HF + XbaI-HF
The amplified inserts were directly purified using the Qiagen PCR purification kit. The digested backbones (pIG16_033, 034, 035) were loaded on an agarose gel. After electrophoresis the bands corresponding to the expected sizes were extracted.
3) T4-Ligation
The digested inserts and backbones were ligated at a molar ration of 1:3 (Backbone:Insert) using T4 ligase in 1X T4 ligation buffer in a total reaction volume of 20 µL. The reaction was incubated at RT for 20 min.
No.
Insert
Backbone
Resulting construct
1
EcoRI-aGFPnano-XbaI
EcoRI-pIG16_035-XbaI
pIG16_128
2
NcoI-aGFPnano-NheI
NcoI-pIG16_033-NheI
pIG16_127
3
XhoI-mCherry-XbaI
XhoI-pIG16_034-XbaI
pIG16_129
5 µL of the reaction mix was transformed into chemically competent E.coli DH5alpha and spread on LB agar plates supplemented with ampicilin. Incubation over night at 37 °C.
4) Inoculation
5 colonies of pIG16_130 were inoculated in 5 mL of LB medium supplemented with chloramphenicol.
17.09.16
1) MiniPrep
The plasmids from the transformed E.coli containing pIG16_130#1-5 were prepared using the QiaGen MiniPrep kit. The DNA was eluted in 30 µL of ultra pure water.
2) Testdigestion
500 ng of pIG16_130 #1-5 was treated with 4 units of EcoRI-HF and SpeI-HF in 1X cutsmart buffer in a total reaction volume of 20 µL for 2 hours at 37 °C.
Gel image: NIKLAS???
3) Digestion
5 µg of pIG16_130 #2 was treated with 20 units of EcoRI-HF and SpeI-HF in 1X cutsmart buffer in a total reaction volume of 50 µL for 2 hours at 37 °C in order to release the biobrick part (Lac inducible promoter). The digestion was loaded on an agarose gel and subjected to electrophoresis. The band corresponding to the size of the promoter was extracted using the QiaGen gel extraction kit.
18.09.16
1) Restriction digestion
1.5 µg of pIG16_033 and 035 were treated with 5 unit of NcoI-HF+NheI-HF and EcoRI-HF+XbaI, respectively. The reaction was performed in 1X cutsmart buffer in a total reaction volume of 50 µL for 2 hours at 37 °C. After digestion the linearized plasmids were purified using the QiaGen PCR purification kit.
2) T4-Ligation
The digested inserts and backbones were ligated at a molar ration of 1:3 (Backbone:Insert) using T4 ligase in 1X T4 ligation buffer in a total reaction volume of 20 µL. The reaction was incubated at RT for 20 min.
Backbone
Insert
Resulting construct
NcoI-pIG16_033-NheI
NcoI-aGFPnano-NheI
pIG16_127
EcoRI-pIG16_035-XbaI
EcoRI-aGFPnano-XbaI
pIG16_128
2 µL of the ligation mix was used for transformation of chemically competent E.coli DH5alpha.
19.09.16
1) Inoculation
5 colonies of each plate containing the ligations from the previous day were picked and inoculated in 5 mL of LB medium supplemented with ampicilin. Incubation o/n at 37 °C and 200 rpm.
20.09.16
1) MiniPrep
Plasmid preparation of colonies pIG16_127#1-5 and pIG16_128#1-5 using the QiaGen MiniPrep kit. The DNA was eluted in 30 µL of ultra pure water.
2) Testdigestion
I 500 ng of pIG16_127 was treated with 4 units of NcoI-HF and NheI-HF in 1X Cutsmart buffer for 90 min at 37 °C. II 500 ng of pIG16_128 was treated with 4 units of EcoRI-HF and XbaI-HF. The digested DNA was analyzed by agarose gel electrophoresis.
3) PCR
Amplification of additional promoters for the expression of fusion consructs.
No.
Template
Oligos
comment
Annealing temperature [°C]
1
pSB1C3 (pIG16_020)
oIG16_093+094
pTrpC-promoter from RFP-cassette
62
2
pIG16_017
oIG16_091+092
PCotYZ-RBS
54
3
pIG16_022
oIG16_095+096
BamHI-GST-XbaI
64
Reaction conditions
component
conc
volume [µL]
Q5 MasterMix
2X
25
template
-
0.1
Primer 1
10 µM
2.5
Primer 2
10 µM
2.5
dH2O
-
20 µL
After amplification the samples were loaded on an agarose gel and extracted from the gel using the Qiagen gel extraction kit.
4) Digestions
I 5 µg of integration vector DNA was treated with 20 units of BamHI-HF and XbaI in 1X cutsmart buffer in a total reaction volume of 20 µL for 2 hours at 37 °C.
No.
Vector
Enzymes
1
pCgeA-C(pIG16_034)
BamHI-HF + XbaI
2
pCotZ-C(pIG16_032)
BamHI-HF + XbaI
3
GST(amplified from pIG16_022)
BamHI-HF + XbaI
II For digestion of the amplified promoters 3 µg of extracted DNA was treated with the appropriate restriction enzymes in 1X reaction buffer in a total volume of 50 µL for 1 h at 37 °C.
Template
Promoter
Restriction enzymes
pIG16_017
PCotYZ-RBS
EcoRI-HF + SpeI-HF
pIG16_020
PTrpC
EcoRI-HF + SpeI-HF
The digested backbones were loaded on an agarose gel and the bands corresponding to the expected size were extracted using the QiaGen gel extraction kit.
5) 3A Assembly
The constructs for surface display were assembled using various promoters for expression in E.coli and B.subtilis. 3A assembly of a promoter and a fusion construct into an integration vector. All inserts were treated with the appropriate restriction enzymes from previous assemblies and stored at -20 °C. The ligation was performed at a molar ratio of 1:3 (vector:inserts) using 50 ng of vector DNA. The reaction was incubated at RT for 20 min and subsequently transformed into chemically competent E. coli DH5alpha and spread on LB-agar plates supplemented with the appropriate antibiotic.
I pLac promoter (inducible, BBa_K180000)
No.
Fusion construct
Result
1
pIG16_047
pIG16_134
2
pIG16_048
pIG16_135
3
pIG16_049
pIG16_136
4
pIG16_066
pIG16_138
IIpTrpC promoter (consitutive promoter, amplified from RFP-cassette)
No.
Fusion construct
Result
1
pIG16_042
pIG16_143
2
pIG16_047
pIG16_144
3
pIG16_048
pIG16_145
4
pIG16_049
pIG16_146
5
pIG16_059
pIG16_147
6
pIG16_066
pIG16_148
7
pIG16_039
pIG16_149
IIIpCotYZ-RBS (B. subtilis CotYZ-promoter with ribosome binding site)
No.
Fusion construct
Result
1
pIG16_039
pIG16_150
2
pIG16_042
pIG16_151
3
pIG16_047
pIG16_152
4
pIG16_048
pIG16_153
5
pIG16_049
pIG16_154
6
pIG16_059
pIG16_155
7
pIG16_066
pIG16_156
8
pIG16_056
pIG16_157
9
pIG16_058
pIG16_158
10
pIG16_054
pIG16_159
11
pIG16_051
pIG16_160
IV Further ligations (Ligation of GST into surface display vectors)
No.
Insert
Backbone
Result
1
BamHI-pIG16_032-XbaI
BamHI-GST-XbaI
pIG16_131
2
BamHI-pIG16_034-XbaI
BamHI-GST-XbaI
pIG16_142
21.09.16
1) Transformation
The transformations of the T4-ligations of the previous day were repeated.
2) Inoculation
Inoculation of the following colonies transformed on the previous day:
I PTrpC: pIG16_143#1 148#1 149#1-2
II PLac: pIG16_134#1-2 135#5 138#1-5
III PCotYZ: pIG16_150#1 159#1-4 157#1-4 155#1
IV pIG16_131 #1-2
Inoculation in 5 mL of LB medium supplemented with the appropriate antibiotic.
22.09.16
1) MiniPrep
Plasmid preparation of the inoculated colonies from the previous day using the zymo research zyppy kit. I PTrpC: pIG16_143#1 148#1 149#1-2
II PLac: pIG16_134#1-2 135#5 138#1-5
III PCotYZ-RBS: pIG16_150#1 159#1-4 157#1-4 155#1
IV pIG16_131 #1-2
2) Testdigestion
500 ng of the prepared plasmids were treated with 4 units of XbaI and PstI in 1X NEBuffer 2.1 for 1 hour at 37 °C. The digested DNA was analyzed by agarose gel electrophoresis.
3) Sequencing
the Following plasmids were sent so sequencing: pIG16_131#1 138#3 149#2 143#1 150#1 159#2+3 155#11 158#4+5
4) Inoculation
Inoculation of further constructs with the new promoters:
I PTrpC: pIG16_147#1-2 148#2 149#2-3 142#2 145#1-3
II PLac: pIG16_134#1-5 136#1-3
III PCotYZ-RBS: pIG16_160#1 157#1-5 155#2 150#2-4 159#1 151#1-2
IV pIG16_131 #3-5 142#1
23.09.16
1) MiniPrep
The the plasmids of inoculated colonies from the previous day were prepared using the Zyppy Miniprep kit:
2) Testdigestion
500 ng of prepared DNA was treated with 4 units of XbaI and PstI for 1 hour at 37 °C. The digestion was analyzed by agarose gel electrophoresis.
3) Sequencing
Positive colonies were sent to sequencing.
4) Inoculation
Inoculation of colonies for preparation of glycerol stocks:
I PLac: 132#1 137#1 138#3 133#2
II PTrpC: 143#1 149#2
III PCotYZ-RBS: 150#1 158#5 159#3 151#1 160#1
IV 129#1 142#1 131#1 126#1 127#3
24.09.16
1) MiniPrep & Glycerl stocks The plasmids from the inoculated colonies of the previous day were prepared using the Zyppy MiniPrep kit.
1 mL of the inoculated colonies was used for the preparation of 15 % glycerol stocks.
2) Linearization
5 µg of the following plasmids was linearized using 25 units of the appropriate enzyme in 1X reaction buffer for 2 hours at 37 °C:
No
Plasmid pIG16_
Enzyme
1
150
XhoI
2
151
XhoI
3
158
XhoI
4
159
XhoI
5
160
XhoI
6
126
ScaI
7
128
ScaI
8
129
ScaI
9
141
PstI
10
131
PstI
After linearization the DNA was purified using the QiaGen PCR purification kit.
3) PCR
The anti-GFP nanobody was amplified in order to introduce BamHI and XbaI restrictions sites:
Component
Conc.
Vol [µL]
DNA template
.
0.1
oIG16_97
10 µM
2.5
oIG16_98
10 µM
2.5
Q5 MasterMix
2X
25
dH2O
.
20
Amplification using the Touchdown67 program. The PCR product was loaded on a 1% TAE agarose gel and subjected to electrophoresis. The band corresponding to the expected size was extracted and purified from the gel using the QiaGen Gelextraction kit.
4) Digestion
2 µg of the extracted PCR product was treated with 10 units of BamHI and XbaI for 1 hour at 37 °C. The DNA was purified using the QiaGen PCR purification kit.
5) T4 ligations
No.
Insert
Backbone
Resulting plasmid
1
BamHI - aGFPnano - XbaI
BamHI - pCgeA-C - XbaI
pIG16_161
2
BamHI - aGFPnano - XbaI
BamHI - pCotZ-C - XbaI
pIG16_141
The DNA was ligated using T4 ligase at a molar ratio of 1:3 (Vector : Insert) in 1X T4 ligation buffer in a total volume of 20 µL for 30 min at RT. 5 µL of the ligation mix were transformed into chemically competent E. coli DH5alpha and plated on LB agar plates supplemented with ampicilin. Incubation o/n at 37 °C.
25.09.16
1) Glycerol stocks
Preparation of 15 % glycerol stocks with the following strains:
pIG16_134 143 149 150 151 157 159 160 138 137 134 133 132 126 127 129 131 141
26.09.16
1) MiniPrep
The Plasmids from the following strains were prepared using the Zyppy MiniPrep kit:
pIG16_142#1,2 136#1-3 131#1 141#1 126#1
2) Testdigestion
500 ng of the prepared DNA (pIG16_142 and 136) was treated with 5 units of the appropriate enzyme in 1X reaction buffer in a total volume of 20 µL for 1 h at 37 °C.
Plasmid
Restriction enzymes
pIG16_142
XbaI + NcoI-HF
pIG16_136
XbaI + PstI
Analysis by agarose gel electrophoresis.
3) Linearization
3.5 µg of the following plasmids were linearized with 20 units of the appropriate enzyme in 1X reaction buffer in a total reaction volume of 50 µL for 1 hour at 37 °C.
Plasmid
enzyme
131
PstI
141
PstI
126
ScaI
142
PstI
034
ScaI
5 µL of the digestion was analyzed by gel electrophoresis. The remaining sample was purified using the QiaGen PCR purification kit.
4) Inoculation Inoculation of further colonies for test digestion: pIG16_161#1-5
27.09.16
1) MiniPrep
The plasmid DNA of the inoculated colonies of pIG16_161#1-5 were prepared using the QiaGen MiniPrep kit.
2) Testdigestion
500 ng of plasmid DNA was treated with 4 units of SpeI-HF and XbaI in 1XCutsmart buffer in a total reaction volume of 20 µL for 1 hour at 37 °C. The samples were analyzed by gel electrophoresis. Colony #1 was sent to sequencing.
28.09.16
29.09.16
30.09.16
01.10.16
02.10.16
03.10.16
04.10.16
1.) Digestion for 3 A Assembly
05.10.16
06.10.16
1.) Testdigestion
Digestion mixture:
Component
concentration
volume [µL]
DNA
500ng/µL
4-6µl
PstI-HF
20u/µL
0.1
XbaI
20u/µL
0.1
NEBuffer CS
10X
2
ultra pure water
-
ad 20 µL
07.10.16
lab book
Posted by: iGEM Freiburg
Nanocillus - 'cause spore is more!
Contact
