<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Freiburg_Experiments skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Freiburg/Experiments</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="menuSmall"><H5><A href=" https://2016.igem.org/Team:Freiburg/sitemap">  Menu  </A></H5></DIV><DIV class="menu-wrap"><NAV class="menu"><UL class="clearfix"><LI class="menu_item"><A href=" https://2016.igem.org/Team:Freiburg/Team"> Team </A></LI><LI><A href="https://2016.igem.org/Team:Freiburg/Attributions">  Attributions </A></LI><LI><A href="https://2016.igem.org/Team:Freiburg/Achievements">  Achievements </A></LI><LI><A href="https://2016.igem.org/Team:Freiburg/Pictures">  Pictures </A></LI><LI><A href="https://2016.igem.org/Team:Freiburg/GiantJamboree"> Giant Jamboree </A></LI></UL><LI class="menu_item"><A href="https://2016.igem.org/Team:Freiburg/Overview"> Project </A></LI><LI><A href="https://2016.igem.org/Team:Freiburg/Motivation"> Motivation </A></LI><LI><A href="https://2016.igem.org/Team:Freiburg/Goals_Approach"> Approach </A></LI><LI><A href="https://2016.igem.org/Team:Freiburg/B_subtilis"> B. subtilis </A></LI><LI><A href="https://2016.igem.org/Team:Freiburg/Surface_Display"> Microbial surface display
 </A></LI><LI><A href="https://2016.igem.org/Team:Freiburg/Targeted Drug Delivery"> Targeted Drug Delivery</A></LI><LI class="menu_item"><A href="https://2016.igem.org/Team:Freiburg/Lab"> Lab </A></LI><LI><A href="https://2016.igem.org/Team:Freiburg/Experiments"> Methods </A></LI><LI><A href="https://2016.igem.org/Team:Freiburg/Notebook"> Journal </A></LI><LI class="menu_item"><A href="https://2016.igem.org/Team:Freiburg/Results"> Results</A></LI><LI><A href="https://2016.igem.org/Team:Freiburg/Proof"> Proof of Concept </A></LI><LI><A href="https://2016.igem.org/Team:Freiburg/Binding"> Binding </A></LI><LI><A href="https://2016.igem.org/Team:Freiburg/Delivery"> Delivery </A></LI><LI><A href="https://2016.igem.org/Team:Freiburg/Production_Nanocillus"> Nanocillus </A></LI><LI><A href="https://2016.igem.org/Team:Freiburg/Biobricks"> Biobricks </A></LI><LI><A href="https://2016.igem.org/Team:Freiburg/Shampoo"> Extra </A></LI><LI class="menu_item"><A href="https://2016.igem.org/Team:Freiburg/Safety"> Safety </A></LI><LI><A href="https://2016.igem.org/Team:Freiburg/Knockouts"> Knock Outs </A></LI><LI><A href="https://2016.igem.org/Team:Freiburg/Killswitch"> Kill Switch </A></LI><LI><A href="https://2016.igem.org/Team:Freiburg/PR"> Public relations </A></LI><LI><A href="https://2016.igem.org/Team:Freiburg/Human_Practices"> Human Practice </A></LI><LI><A href="https://2016.igem.org/Team:Freiburg/Integrated_Practices"> Integrated </A></LI><LI><A href="https://2016.igem.org/Team:Freiburg/Collaborations"> Collaborations </A></LI><LI><A href="https://2016.igem.org/Team:Freiburg/Poster"> Poster </A></LI></NAV></DIV><P id="back-top"><A href="#top">Back to Top</A></P><DIV class="color1"><DIV class="img_menu"><H5 style="text-align: center"> Our Methods </H5></DIV></DIV><DIV class="color2"><DIV id="cloning" class="para_center_20"><H1>Cloning</H1><H2>I. Culture conditions</H2><DIV>The <I>E. coli</I> strains were maintained in lysogeny broth medium. Liquid cultures were incubated at 37°C and 250 g. Solid cultures were incubated on 1.5% Agar LB plates. Selection of transformed <I>E. coli</I> was performed by supplementation of the LB medium with the antibiotics ampicillin at a final concentration of 100 µg/mL and chloramphenicol at 25 µg/mL.
            </DIV><H2>II. Preparation of chemically competent <I>E. coli</I></H2><DIV>
                Competent <I>E. coli</I> were prepared using the Zymo Research Mix &amp; Go <I>E. coli</I> Transformation Kit according to the protocol of the manufacturer. The <I>E. coli</I> were inoculated and incubated in 50 mL of ZymoBroth™ medium until an OD600 of 0.6 was reached. The cells were harvested by centrifugation and washed with 1x Wash Buffer. After centrifugation the supernatant was removed and the cells were resuspended in pre-cooled 1x Competent Buffer. The competent <I>E. coli</I> were aliquoted into sterile tubes and stored at -80°C until further use. Using the chemically competent <I>E. coli</I> prepared by the Mix &amp; Go Transformation Kit no heat shock or electroporation was necessary.
            </DIV><H2>III. Transformation of chemically competent<I> E. coli</I></H2><DIV>
                The competent <I>E. coli</I> were transformed by addition of 2 – 5 µL of a reaction mix containing the appropriate plasmids. <I>E. coli</I> transformed with plasmids harboring an ampicillin resistance were directly spread on agar plates supplemented with ampicilin. For antibiotic resistances the transformed cells were incubated in 300 µL of LB medium at 37°C and 250 rpm prior to spreading on a LB agar plate. The spread cells were incubated over night at 37°C. Subsequently, 5 colonies per plate were picked by a pipet tip, inoculated in 5 mL of LB medium supplemented the appropriate antibiotics and incubated over night at 37°C and 250 rpm.
            </DIV><H2>IV. Bacterial glycerol stocks</H2><DIV>For storage of bacterial strains 500 µL of the overnight inoculation were added to 500 µL of 30% glycerol and gently mixed. The stocks were stored at – 80°C for later use.
            </DIV><H2>V. Plasmid preparation</H2><DIV>The preparation of plasmids from transformed <I>E. coli</I> was performed using the kits from Qiagen, PeqLab or Zymo Research according to the protocols of the manufacturers. Prior to the preparation the transformed <I>E. coli</I> were incubated in 5 mL of LB medium supplemented with the appropriate amount of antibiotics. The cells were harvested by centrifugation and lysed at alkaline conditions. The lysate containing the plasmids was loaded on spin columns and eluted in 30 µL of nuclease free water. The concentration of the DNA was determined by a NanoDrop spectrophotometer.
            </DIV><H2>VI. Restriction digestion of plasmids</H2><DIV><H3>a) Digestion of plasmids for cloning</H3><DIV>To release inserts from their backbone for further applications 2µg of plasmid DNA was treated with 10 units of the appropriate enzyme in 1X buffer at a total reaction volume of 50 µL and incubated for 1 hour at the appropriate temperature. If two restriction enzymes were used simultaneously the right conditions were determined by the “NEB Double Digest Finder” online tool. The reaction was stopped by addition of 10x orange G loading dye and analyzed by agarose gel electrophoresis.
                </DIV><H3>b) Test digestion</H3><DIV>Verification of transformed plasmids was performed by treatment of 500 ng of the prepared DNA with 5 units of appropriate restriction enzymes in its respective 1x reaction buffer and analyzed by agarose gel electrophoresis. The sizes of the observed bands were compared to the sizes of the expected fragments. Samples that exhibited the expected fragment sizes were further analyzed by sequencing
                </DIV><H3>c) Linearization of plasmids for transformation of <I>B. subtilis</I></H3><DIV>The transformation of integration vectors into <I>B. subtilis</I> and to insert the fusion constructs into the genome 3 µg of the plasmid was linearized by a single cutter enzyme in the appropriate 1x buffer at a total reaction volume of 50 µL. The reaction was stopped by addition of 10x orange G loading dye, analyzed by agarose gel electrophoresis and subsequently extracted and purified from the gel.
                </DIV></DIV><H2>VII. Agarose gel electrophoresis</H2><DIV>Amplified of digested DNA samples were supplemented with the appropriate amount of 10x Orange G loading dye and loaded on a 1% (w/v) agarose TAE gel. Staining of the DNA was performed by the PeqGreen DNA dye at the recommended concentration. The gel was subjected to electrophoresis at 110 V for approximately 30 min or until a sufficient separation of the bands was observable. Visualization of the separated DNA fragments in the gel was performed by an ultraviolet light imager. A 2-log DNA ladder and a 1kb DNA ladder were included as molecular size marker.
            </DIV><H2>VIII. Gel extraction</H2><DIV>Extraction of amplified genes or digested fragments from agarose gels was performed using the QIAquick gel extraction kit from Qiagen according to the protocol of the manufacturer. The bands corresponding to the expected fragment sizes were excised from the gel, lysed with QG buffer at 50°C and loaded on a DNA spin column. The DNA was eluted with 30 µL of nuclease free water and stored at –20°C until further use.
            </DIV><H2>IX. Gibson assembly</H2><DIV>In order to generate the fusion constructs the coat genes, the anti-GFP nanobody and the glutathione S-transferase were amplified by extension PCR to introduce overlapping sequences of at least 25 bp required for Gibson assembly. The iGEM standard backbone pBS1C3 was linearized by treatment with XbaI and SpeI. The assembly was performed with the linearized backbone, an amplified coat gene, and the anti-GFP nanobody or the glutathione S-transferase at molar ratios of 3:1 (insert : vector). The volume of the fragments was adjusted to 2.5 µL with nuclease free water and added to 7.5 µL of prepared 1.33X Gibson master mix. The reaction was incubated for 60 min at 50°C and 2 µL were used to transform chemically competent <I>E. coli</I> DH5α.
            </DIV><H2>X. Polymerase chain reactions</H2><DIV><H3>a) Colony PCR</H3><DIV>For the amplification of the genes from <I>B.subtilis</I> a single colony was dissolved in water and lysed by heat shock at 100°C for 10 min. 1 µL of the lysate was used as template for the amplification. The PCR was performed using the Q5 High-Fidelity 2x Master Mix from NEB, alongside with the appropriate forward and reverse primer at a final concentration of 0.5 µM at a total volume of 50 µL. The reaction was subjected to thermal cycling, subsequently stopped by addition of 10x Orange G loading dye and analyzed by gel electrophoresis.
                </DIV><H3>b) Extension PCR </H3><DIV>In order to introduce additional elements to the used genes, such as linker regions or epitope tags, the PCR was performed using primers with the appropriate overhangs. Approximately, 10 ng of plasmid DNA was used for amplification in Q5 High-Fidelity 2x Master Mix from NEB, alongside with the forward and reverse primer at a final concentration of 0.5 µM in a total volume of 50 µL. The reaction was subjected to thermal cycling, subsequently stopped by addition of 10x Orange G loading dye and analyzed by gel electrophoresis.
                </DIV><H3>c) Thermal cycling </H3><DIV>The thermal cycling was performed according to the recommendations of the manufacturer. Initial denaturation was carried out at 98°C for 5 min, followed by 30 cycling steps including denaturation at 98°C for 15 s, primer annealing at the appropriate temperature for 15 s and elongation at 72°C for 30s/ kb of the amplicon. The cycling was followed by a final elongation at 72°C for 5 min and the reaction was stored at 8°C. The annealing temperatures for the used primers was determined by the NEB TM calculator online tool. The thermal cycling was performed as touchdown-PCR. The initial annealing temperature was increased by 10°C and after each cycle step reduced by 1°C until the final annealing temperature was reached. backbone
                </DIV></DIV><H2>XI. T4-Ligation and subcloning</H2><DIV>
                Ligation of amplified <I>B. subtilis</I> coat genes or digested DNA fragments with fusion constructs was performed using the T4 ligase from NEB. The insert, alongside with 50 ng of the linearized backbone were ligated at a molar ratio of 3 : 1 (vector : insert) using 5 units of T4 ligase in 1x T4 reaction buffer in a total reaction volume of 20 µL. The reaction was incubated at room temperature for 30 minutes and 5 µL of the reaction mixture were transformed into chemically competent <I>E. coli</I> DH5α.
            </DIV><H2>XII. Annealed oligo cloning</H2><DIV>For the insertion of small elements into a vector, two oligos containing the desired sequence were mixed at a molar ratio of 1 : 1 and incubated for 15 minutes at 100°C, subsequently the temperature was gradually decreased to room temperature over the course of 3 hours. The annealed oligos were ligated into a plasmid with the appropriate overhangs using the T4 ligase from NEB.
            </DIV><H2> XIII. Sequencing</H2><DIV>
                Plasmids that were verified by restriction digestion were sent to sequencing by GATC biotech. The concentration of the DNA samples was adjusted to 30 – 50 ng/µL at a total volume 20 µL. The appropriate primer required for the sequencing reaction were sent in an additional microtube at a concentration of 10 µM. The sequencing results were analyzed using the Geneious software.
            </DIV></DIV></DIV><DIV class="color3"><DIV id="spores" class="para_center_20"><H1><I>Bacillus subtilis</I></H1><H2>I. Cultivation and germination of Spores</H2><DIV>The <I>Bacillus subtilis</I> strains were maintained on 1.5% LB agar. Liquid cultures were grown in LB medium at 37°C and 200 rpm. To germinate <I>Bacillus subtilis</I> spores, the samples were also inoculated in LB medium and incubated overnight.  

            </DIV><H2>II. Sporulation</H2><DIV>
                The cultures were inoculated in LB medium at 37°C and 200 rpm until they reach their exponential growth phase and then resuspended in Difco Sporulation Medium (DSM). After 24 hours of incubation, the cells are sporulated. To purify the samples, they were treated with lysozyme.<A target="_blank" href="https://static.igem.org/mediawiki/2016/7/71/T--Freiburg--SporulationProtocol.pdf"> Protocol</A></DIV><H2>III. Competent cells + Transformation</H2><DIV>To make <I>Bacillus subtilis</I> competent, the samples were inoculated in minimal medium and grow until they reached an OD600 of 1.0-1.3/ml. 600 ng of DNA was added to 400 µl of the competent cells, incubated time temperature shaking and streaked on selective agar.<A target="_blank" href="https://static.igem.org/mediawiki/2016/7/77/T--Freiburg--ProtocolCompetentCellsandTransformation.pdf">Protocol</A></DIV><H2> IV. Selection </H2><DIV> 
To find out if the positive colonies have the DNA inserted in the right locus after transformation the colonies are picked. For the locus amyE the colonies are plated on starch plates and incubated. By adding lugol’s iodite it is possible to differ the colonies. The wt and control is able to reduce the starch and should decolorize the area around it , whilst the transformed colonies should stay purple. 
        </DIV><H2> V. The concentration of antibiotics used for <I>Bacillus subtilis</I></H2><DIV>
The table shows the minimal concentrations of different antibiotics used for the selection of antibiotic resistant <I>Bacillus subtilis</I> cells.<DIV class="table sectionedit3"><TABLE class="inline"><TBODY><TR class="row0"><TH class="col0">Antibiotic</TH><TH class="col1">Stock solution</TH><TH class="col2">Dilution <I>Bacillus subtilis</I></TH></TR><TR class="row1"><TD class="col0">Spectinomycin (Spec)</TD><TD class="col1">100 mg/ML</TD><TD class="col2">1 to 1,000</TD></TR><TR class="row2"><TD class="col0">Chloramphenical (Cml)</TD><TD class="col1">25 mg/ml</TD><TD class="col2">1 to 5,000</TD></TR><TR class="row3"><TD class="col0">Erythromycin(Erm)</TD><TD class="col1">10 mg/ml</TD><TD class="col2">1 to 1,000</TD></TR></TBODY></TABLE></DIV></DIV></DIV></DIV><DIV class="color4"><DIV id="proteins" class="para_center_20"><H1> Expression analysis  </H1><H2>1. Preparation of spore coat lysates </H2><DIV>
Approximately 700 Mio spores of <I>Bacillus subtilis</I> were resuspended in lysis buffer containing 50 mM Tris base, 1% SDS, 8 M Urea and 50 mM DTT. The samples were incubated o/n at 60°C while shaking.1 The samples were centrifuged and the SN was taken for further analysis.
</DIV><H2>2. SDS-PAGE and Western Blot </H2><DIV>
40 µl of the spore coat lysates were incubated with 6X Loading dye and boiled for 10 minutes.
The prepared spore coat lysates were loaded onto a 10% polyacrylamide gel for SDS-PAGE. The gel was run according to the manufacturer (Bio-Rad).
The gel was either stained with Bio-SafeTM Coomassie Stain from Bio-Rad or transferred onto a PVDF membrane for Western Blot analysis. As all constructs contain a HA tag, Anti-HA.11 Epitope Tag Antibody from Biolegend® were used. HRP Goat anti-mouse IgG (minimal x-reactivity) Antibodies from Biolegend® were used for detection. For imaging the Fusion Fx (Vilber Lourmat) was used.</DIV><H2>3. Flow cytometry </H2><DIV>
Flow cytometry analysis was performed using 5Mio spores of <I>Bacillus subtilis</I>. HA-Tag (6E2) Mouse mAb (Alexa Fluor® 647 Conjugate) from Cell Signaling Technology® were used for detection of the HA tag. The spores were incubated for 40mins on ice using a dilution of 1:50 in a total volume of 50 µl.  
Nanobody expressing spores were: 
1. incubated with 100 µg/ml GFP in a total volume of 50µl for 40 mins on ice. 
2. blocked in 5% BSA for 1h on ice, washed three times with 1x PBS and stained with 100 µg/ml GFP in a total volume of 50µl for 40 mins on ice.
3. blocked in 5% BSA for 1h on ice and subsequently stained with 100 µg/ml GFP in a total volume of 50µl for 40 mins on ice. 
All samples were washed 5x with 0.05% TBS-T and resuspended in 1x PBS for FACS analysis. 
For imaging the GALLIOS flow cytometer (Beckman-Coulter) was used. The data was analyzed using the FlowJo software.
</DIV><H2>4. Protein purification </H2><DIV>
Plasmid was transformed into Rosetta pLysS competent cells and grown in LB medium supplemented with ampicillin (100 µg/ml) and chloramphenicol (25 µg/ml). Cells were induced at OD600=0.8 w. 1 mM IPTG (end conc.) and incubated overnight at 25°C.
 
Cells were pelleted at 4000 g 20 min. Resuspended in buffer A, sonicated and purified on HisTrap column.
 
Questions
No. 1: Sample
Answer: aGFP nanobody
 
No. 2: Column
Answer: HisTrap FF 5 ml
 
No. 3: Buffer A
Answer: 50 mM Tris 500 mM NaCl 20 mM Imidazole 10% glycerol pH 8.0
 
No. 4: Buffer B
Answer: 50 mM Tris 150 mM NaCl 400 mM Imidazole 10% glycerol pH 7.6
The elution buffer was exchanged to PBS using HiPrep 26/10 Desalting column. His tag was cleaved using TEV protease overnight at 4°C. Free his-tag, TEV protease and uncleaved aGFP nanobody were removed from cleaved aGFP nanobody by second pass through HisTrap FF 5 ml column.
</DIV><H2>5. Preparation of <I>E. coli</I> whole cell lysates</H2><DIV>
The <I>E. coli</I> samples were incubated in 1xPBS and sonified for 10 minutes at 20%.
</DIV><H2>6. SDS-PAGE and Western Blot</H2><DIV>
The protein concentration was determined via a Lowry assay.
20 µl of the <I>E. coli</I> whole cell lysates with wanted protein amount were incubated with 4 µl of 6X Loading dye and boiled for 10 minutes.
The prepared lysates were loaded onto a 10% polyacrylamide gel for SDS-PAGE. The gel was run according to the manufacturer (Bio-Rad).
The gel was either stained with Bio-SafeTM Coomassie Stain from Bio-Rad or transferred onto a PVDF membrane for Western Blot analysis. As all constructs contain a HA tag, Anti-HA.11 Epitope Tag Antibody from Biolegend® were used. HRP Goat anti-mouse IgG (minimal x-reactivity) Antibodies from Biolegend® were used for detection. For imaging the Fusion Fx (Vilber Lourmat) was used.
</DIV><DIV>
References
1.   Riesenman, P. J. &amp; Nicholson, W. L. Role of the Spore Coat Layers in <I>Bacillus subtilis</I> Spore Resistance to Hydrogen Peroxide, Artificial UV-C, UV-B, and Solar UV Radiation // Role of the spore coat layers in <I>Bacillus subtilis</I> spore resistance to hydrogen peroxide, artificial UV-C, UV-B, and solar UV radiation. Applied and environmental microbiology 66, 620–626 (2000).</DIV></DIV></DIV><DIV class="color5"><DIV id="targeting" class="para_center_20"><H1>Targeting</H1><H2>1. (3-Glycidyloxypropyl)trimethoxysilane (GOPTS) - Coating of microscope slides</H2><DIV>
                Glass slides were washed with EtOH (70%) and ddH2O and activated by oxygen plasma (300 s, 50% Power, Plasma Cleaner ZEPTO, Electronic Diener). The activated surface was incubated with GOPTS (50 µl, 2 h, darkness). The slides were then washed with acetone (99,9%) to remove excess GOPTS.
            </DIV><H2>2. Spotting</H2><DIV>
                A spotting mask (silicone, CultureWell™) was used to evenly create sample spots on slides. The spots were incubated with protein solution (GFP/anti-GFP nanobody) (2µl, 15 min, 4°C, darkness) of different concentrations. Blocking was done with a BSA (4%)- or milk protein (5%)-solution.
            </DIV><H2>3. Spore binding assay</H2><DIV>
                A solution of spores (of different concentrations) was pipetted on a nanobody spot and incubated for 20 min. Excess spore were washed off using PBS(1x) (Phosphate Buffered Saline) (10 min, 10 rpm).
            </DIV><H2>4. Sample preparation</H2><DIV>
                After washing the slides were dried with compressed air. A Mowiol/DABCO (2.5%) mounting solution was used to avoid withering of the proteins (amount depending on size of the cover slip). The samples have been incubated overnight. The coverslips were sealed with nail polish.
            </DIV><H2>5. Fluorescence microscopy</H2><DIV>Sample analysis was done by fluorescence microscopy (Nikon C2+ confocal).
            </DIV></DIV></DIV><DIV class="color6"><DIV id="delivery" class="para_center_20"><H1>V) Enzymatic activity</H1><H2>1.   Preparation of reagents</H2>
Purified glutathione S-transferase (25 U/mg) was aliquoted at a concentration 0.2 mg/ml in PBS buffer supplemented with EDTA (100 mM) and stored at -20°C. The enzyme substrates CDNB and reduced L‑glutathione (GSH) were both aliquoted at a concentration of 100 mM in ethanol and stored at -20°C.

 
 
<H2>2.    GST assay</H2>
In order to determine the GST activity in a sample, the reaction was freshly set up with 75µl CDNB and 75µl GSH in 1X PBS for a volume of 1500µl, sufficient for about 16 wells (à 90µl). This equals 0,05% of each substrate in the master mix. The purified glutathione S-transferase was used as positive reaction control. As negative controls empty wells with PBS instead of enzyme solution were used. The reaction was monitored for a time-course of 2 hours with orbital averaging by means of absorbance at 340 nm in the plate reader (CLARIOstar, BMG Labtech). Before each measurement the plate was shook at 300 rpm for 1 s. To calculate the activity of GST a formula based on the Beer-Lambert law was used:

Where:
 = slope of the graph
ε<SUB>mM</SUB>  = extinction coefficient of CDNB at 340 nm (adjusted to the path length of the wells by taking the path length [cm] times 9.6)
V<SUB>ges</SUB> = reaction volume in the well
V<SUB>enz</SUB> = the volume of the sample tested<H1>Shampoo:  </H1><H2>1.     Coating of microplates with GFP</H2>
 
Bicarbonate/carbonate coating buffer (100 mM) was prepared by dissolving 3.03 g Na2CO3, 6.0 g NaHCO3 in 1000 ml of distilled water. The pH value was adjusted to 9.6.
Purified GFP was diluted in prepared coating buffer at a concentration of 20 µg/ml and 100 µl were added to each well. The plate was incubated for 2 h at RT and subsequently washed twice with 200 µl of PBS. The remaining protein-binding sites were blocked with 200 µl blocking buffer containing 5%(w/v) non-fat dry milk in PBS. After the incubation the blocking solution was removed and the wells were washed twice with 200 µl of PBS. The coating of the plates with GFP was verified by determination of the fluorescence emission at 496 nm with 50 flashes per well, at a focal height of 3.6 nm and a gain value of 1740.


<H2>2.     Washing with probiotic shampoo</H2>
The GFP-coated plates were were washed with detergent solutions containing SDS or a commercially available shampoo (Syoss OLEO21) and with detergent solutions containing purified anti-GFP nanobody (4.8 ng per well) or anti-GFP nanobody displaying spores (20 million spores per well). After an incubation time of 5 minutes with the detergent solutions the plates were washed twice with 200 µl of PBS. The remaining GFP fluorescence was determined at an emission wavelength of 496 nm with 50 flashes per well, focal height of 3.6 nm and a gain value of 1740. To calculate the washing efficiency of the detergent solutions the ratio of GFP fluorescence before and after the washing procedure was determined.




        </DIV></DIV><FOOTER><DIV class="center"><DIV style="display:inline-block"><P>Posted by: iGEM Freiburg </P></DIV><DIV style="display:inline-block"><P> Nanocillus - 'cause spore is more! </P></DIV><DIV style="display:inline-block"><P><A style="color:#e8a126" href="https://2016.igem.org/Team:Freiburg/Kontakt"> Contact </A></P></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>