<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Freiburg_Description skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Freiburg/Description</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="menuSmall"><H5><A href=" https://2016.igem.org/Team:Freiburg/sitemap">  Menu  </A></H5></DIV><DIV class="menu-wrap"><NAV class="menu"><UL class="clearfix"><LI class="menu_item"><A href=" https://2016.igem.org/Team:Freiburg/Team"> Team </A></LI><LI><A href="https://2016.igem.org/Team:Freiburg/Attributions">  Attributions </A></LI><LI><A href="https://2016.igem.org/Team:Freiburg/Achievements">  Achievements </A></LI><LI><A href="https://2016.igem.org/Team:Freiburg/Pictures">  Pictures </A></LI><LI><A href="https://2016.igem.org/Team:Freiburg/GiantJamboree"> Giant Jamboree </A></LI></UL><LI class="menu_item"><A href="https://2016.igem.org/Team:Freiburg/Overview"> Project </A></LI><LI><A href="https://2016.igem.org/Team:Freiburg/Motivation"> Motivation </A></LI><LI><A href="https://2016.igem.org/Team:Freiburg/Goals_Approach"> Approach </A></LI><LI><A href="https://2016.igem.org/Team:Freiburg/B_subtilis"> B. subtilis </A></LI><LI><A href="https://2016.igem.org/Team:Freiburg/Surface_Display"> Microbial surface display
 </A></LI><LI><A href="https://2016.igem.org/Team:Freiburg/Targeted Drug Delivery"> Targeted Drug Delivery</A></LI><LI class="menu_item"><A href="https://2016.igem.org/Team:Freiburg/Lab"> Lab </A></LI><LI><A href="https://2016.igem.org/Team:Freiburg/Experiments"> Methods </A></LI><LI><A href="https://2016.igem.org/Team:Freiburg/Notebook"> Journal </A></LI><LI class="menu_item"><A href="https://2016.igem.org/Team:Freiburg/Results"> Results</A></LI><LI><A href="https://2016.igem.org/Team:Freiburg/Proof"> Proof of Concept </A></LI><LI><A href="https://2016.igem.org/Team:Freiburg/Binding"> Binding </A></LI><LI><A href="https://2016.igem.org/Team:Freiburg/Delivery"> Delivery </A></LI><LI><A href="https://2016.igem.org/Team:Freiburg/Production_Nanocillus"> Nanocillus </A></LI><LI><A href="https://2016.igem.org/Team:Freiburg/Biobricks"> Biobricks </A></LI><LI><A href="https://2016.igem.org/Team:Freiburg/Shampoo"> Extra </A></LI><LI class="menu_item"><A href="https://2016.igem.org/Team:Freiburg/Safety"> Safety </A></LI><LI><A href="https://2016.igem.org/Team:Freiburg/Knockouts"> Knock Outs </A></LI><LI><A href="https://2016.igem.org/Team:Freiburg/Killswitch"> Kill Switch </A></LI><LI><A href="https://2016.igem.org/Team:Freiburg/PR"> Public relations </A></LI><LI><A href="https://2016.igem.org/Team:Freiburg/Human_Practices"> Human Practice </A></LI><LI><A href="https://2016.igem.org/Team:Freiburg/Integrated_Practices"> Integrated </A></LI><LI><A href="https://2016.igem.org/Team:Freiburg/Collaborations"> Collaborations </A></LI><LI><A href="https://2016.igem.org/Team:Freiburg/Poster"> Poster </A></LI></NAV></DIV><DIV class="column full_size"><DIV class="color10"><DIV class="para_center_20"><H5 style="text-align: center"> Improved Biobricks </H5></DIV></DIV><DIV class="color3"><DIV class="para_center_20"><H5 id="pelB"><FONT color="red">pelB_anti-GFPnano body</FONT></H5>

The discovery of camelid heavy-chain antibodies and their subsequent modification to single-domain antibodies also called nanobodies<SUP>1,2</SUP> provide researchers with a wide range of tools including labeling methods for imaging, receptor modulation or therapeutic agents. They exhibit a small size of only 15 kDa and their easy structure enables a high-efficiency production in bacterial strains such as <I>E. coli</I>. The anti-GFP nanobody represents an established and well characterized variant of those proteins. In order to directly utilize their functionality, the purification of the over-expressed nanobody provides the advantage of acquiring a highly concentrated protein solution. The BioBrick in the iGEM registry BBa_K929104 contains the full sequence of the anti-GFP nanobody. However, the purification of the nanobody from a bacterial lysate requires its export into the periplasmatic space in order to avoid inclusion bodies and provide an oxidative environment which promotes the formation of the characteristic disulfide bond. The anti-GFP nanobody was provided to us by our supervisor Dr. Maximilian Ulbrich. To avoid the formation of inclusion bodies and increase the yield of purified nanobodies the pelB leader sequence was included at the N-terminal section of the nanobody. This sequence facilitates the export of the protein into the periplasmatic space. The nanobody can be expressed in standard expression vectors such as pGEX or pET in an appropriate bacterial strain. 
</DIV></DIV><DIV class="color10"><DIV class="para_center_figure">
Figure 1: Cloning strategy of the nanobody improvement. The anti-GFP nanobody was amplified with primers carrying extensions for the pelB leader sequence. The resulting fragments were assembled by Gibson assembly.
</DIV></DIV><DIV class="color3"><DIV class="para_center_20"><H3>Characterization</H3><H4>I)	Expression analysis</H4>

The anti-GFP nanobody was be expressed in pET303 including a C-terminal 10xHis-tag in the <I>E. coli</I> strain BL21 after induction with IPTG for 3 hours at 37 °C. The lysate was purified using nickel columns and analysed by SDS-PAGE and coomassie staining to verifiy the expected size of 17 kDa (figure 2 A). In comparison, the bacterial over-expression of the anti-GFP nanobody without pelB leader sequence results in low yields since a considerable amount of expressed proteins remains in inclusion bodies in the bacterial pellet and therefore it cannot be purified from the bacterial lysate (figure 2 B).
</DIV></DIV><DIV class="color10"><DIV class="para_center_figure">
Figure 2: Expression analysis of the anti-GFP nanobody. (A)  Purified pelB anti-GFP nanobody. The pelB leader sequence facilitates an easy and reliable purification at high concentrations. (B) Lysate and pellet of E. coli BL21 after over-expression of anti-GFP nanobody without pelB leader sequence. Overexpression of anti-GFP nanobody without the pelB leader sequence results in the formation of inclusion bodies that remain in the bacterial pellet and therefore decreases the purification efficiency.
</DIV></DIV><DIV class="color3"><DIV class="para_center_20"><H4>II)	Evaluation of binding kinetic</H4>
The evaluation of the binding properties of the anti-GFP nanobody was performed with the Octet RED96 system based on biolayer interferometry. The biosensors were loaded with His-tagged GFP at a concentration of 1.25 µg/mL. After baseline adjustment the association rate ka and the dissociation rate kd of the anti-GFP nanobody at a concentration of 4.8 µg/mL were measured. The resulting sensogram contained intereference patterns over time and was analyzed by the supplemented software. The dissociation constant KD was determined at 1.57 nM.
</DIV></DIV><DIV class="color0"><DIV class="para_center_figure">
Figure 3. Binding kinetics of the anti-GFP nanobody. The interaction of the anti-GFP nanobody to His-tagged GFP was determined by biolayer interferometry. The resulting sensogram represents the complex formation and is devided into 4 sections. The section depicts the binding of the His-tagged GFP to the biosensor followed by the adjustment of the baseline in section 2. As shown in section 3 the incubation with the purified anti-GFP nanobody resulted in its association to GFP. Subsequently, the dissociation of the complex was determined in binding buffer without purified nanobody.
</DIV></DIV><DIV class="color3"><DIV class="para_center_20"><H5 id="Pcot"><FONT color="red">P<SUB>CotYZ</SUB>-RBS</FONT></H5>
The P<SUB>CotYZ</SUB> promoter from <I>Bacillus subtilis</I> plays an important role in the sporulation of <I>B. subtilis</I>. The Promoter is located in the cotVWXYZ gene cluster and drives the expression of the late-stage spore crust proteins CotY and CotZ<SUP>3</SUP>. We included the naturally occurring ribosome binding site for cotZ to the promoter while maintaining the BioBrick compatibility. The improved promoter can be used for the 3A assembly with a coding region and cloned into a suitable integration vector for <I>B. subtilis</I>. The resulting device enables the transformation of <I>B. subtilis</I> for integration and expression of desirable genes<SUP>4</SUP>.
</DIV></DIV><DIV class="color10"><DIV class="para_center_figure">
Figure 4: Cloning strategy of the promoter improvement. P<SUB>CotYZ</SUB> was amplified with primers containing extensions in order to introduce the ribosome binding site. The resulting fragment was digested with XbaI and PstI and ligated into the linearized pSB1C3 vector with the corresponding overhangs.
</DIV></DIV><DIV class="color3"><DIV class="para_center_20"><H3>Characterization</H3>
This promoter was used for the expression of fusion constructs in the spores of <I>B. subtilis</I>. The respective construct was assembled into an integration vector alongside with the P<SUB>CotYZ</SUB>-RBS promoter by 3A assembly – See assembly of fusion genes. 
</DIV></DIV><DIV class="color3"><DIV class="para_center_20"><H4>I)	Expression analysis of fusion constructs driven by P<SUB>CotYZ</SUB>-RBS</H4>
The promoter P<SUB>CotYZ</SUB>-RBS was cloned alongside with BBa_K214001 into the integration vector pBS1C<SUP>4</SUP> by 3A assembly.
After transformation the cells were selected by chloramphenicol resistance and screened for the disruption of the amyE gene on starch agar plates. Subsequently the positive clones were further cultivated and sporulation was induced by nutrient starvation. The resulting spores were purified from vegetative cells with lysozyme and analyzed by SDS-PAGE and Western blotting as described in the methods sectio n. The immunostaining with anti-HA antibodies resulted in the visualization of the expected band at approximately 33 kDa. Additional bands at higher molecular weight were hypothesized to be results from the high cross-linking of spore coat proteins<SUP>5</SUP>.
</DIV></DIV><DIV class="color10"><DIV class="para_center_figure">
Figure 5: Expression analysis of genes regulated by P<SUB>CotYZ</SUB>-RBS. The expression of BBa_K214001 driven by the P<SUB>CotYZ</SUB>-RBS promoter resulted in the expected band at 33 kDa. Not transformed spores of <I>B. subtilis</I> were used as controls. As loading control the lysate was analyzed after SDS PAGE and coomassie staining in order to verify the presence of extracted spore coat proteins.
</DIV></DIV><DIV class="color3"><DIV class="para_center_20"><H4>II)	Localization analysis of displayed proteins</H4>
The gene expression driven by the P<SUB>CotYZ</SUB>-RBS promoter for the display of fusion proteins on the surface of B. subtilis spores was analyzed by flow cytometry. After transformation and induction of sporulation the resulting spores were purified and stained with an anti HA antibody conjugated to AlexaFluor® 647 (Cell Signaling Technology®). The antibody could only access the surface-localized epitopes of the expressed fusion genes and could confirm the successful display of heterologous proteins on the surface of <I>B. subtilis</I> spores.
</DIV></DIV><DIV class="color0"><DIV class="para_center_figure">
Figure 6. Flow cytometry analysis of displayed proteins. The expression of the part BBa_K2114002 was regulated by the P<SUB>CotYZ</SUB>-RBS promoter resulting in the surface display on spores <I>B. subtilis</I>. The containing HA epitope tag was was stained with anti-HA antibodies conjugated to AlexaFluor 647. Flow cytometry analysis could confirm the display of the fusion proteins.
</DIV></DIV><DIV class="color3"><DIV class="para_center_Quellen">
References:
1. Hamers-Casterman, C. et al. Naturally occurring antibodies devoid of light chains. Nature 363, 446–8 (1993).
2. Muyldermans, S. &amp; Lauwereys, M. Unique single-domain antigen binding fragments derived from naturally occurring camel heavy-chain antibodies. J. Mol. Recognit. 12, 131–140 (1999).
3. Imamura, D., Kuwana, R., Takamatsu, H. &amp; Watabe, K. Proteins involved in formation of the outermost layer of Bacillus subtilis spores. J. Bacteriol. 193, 4075–4080 (2011).
4. Radeck, J. et al. The Bacillus BioBrick Box: generation and evaluation of essential genetic building blocks for standardized work with Bacillus subtilis. J. Biol. Eng. 7, 29 (2013).
5. Driks, A. Bacillus subtilis Spore Coat. Microbiol. Mol. Biol. Rev. 63, 1–20 (1999).</DIV></DIV></DIV><FOOTER><DIV class="center"><DIV style="display:inline-block"><P>Posted by: iGEM Freiburg </P></DIV><DIV style="display:inline-block"><P> Nanocillus - 'cause spore is more! </P></DIV><DIV style="display:inline-block"><P><A style="color:#e8a126" href="https://2016.igem.org/Team:Freiburg/Kontakt"> Contact </A></P></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>