<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Freiburg_Goals_Approach skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Freiburg/Goals Approach</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="menuSmall"><H5><A href=" https://2016.igem.org/Team:Freiburg/sitemap">  Menu  </A></H5></DIV><DIV class="menu-wrap"><NAV class="menu"><UL class="clearfix"><LI class="menu_item"><A href=" https://2016.igem.org/Team:Freiburg/Team"> Team </A></LI><LI><A href="https://2016.igem.org/Team:Freiburg/Attributions">  Attributions </A></LI><LI><A href="https://2016.igem.org/Team:Freiburg/Achievements">  Achievements </A></LI><LI><A href="https://2016.igem.org/Team:Freiburg/Pictures">  Pictures </A></LI><LI><A href="https://2016.igem.org/Team:Freiburg/GiantJamboree"> Giant Jamboree </A></LI></UL><LI class="menu_item"><A href="https://2016.igem.org/Team:Freiburg/Overview"> Project </A></LI><LI><A href="https://2016.igem.org/Team:Freiburg/Motivation"> Motivation </A></LI><LI><A href="https://2016.igem.org/Team:Freiburg/Goals_Approach"> Approach </A></LI><LI><A href="https://2016.igem.org/Team:Freiburg/B_subtilis"> B. subtilis </A></LI><LI><A href="https://2016.igem.org/Team:Freiburg/Surface_Display"> Microbial surface display
 </A></LI><LI><A href="https://2016.igem.org/Team:Freiburg/Targeted Drug Delivery"> Targeted Drug Delivery</A></LI><LI class="menu_item"><A href="https://2016.igem.org/Team:Freiburg/Lab"> Lab </A></LI><LI><A href="https://2016.igem.org/Team:Freiburg/Experiments"> Methods </A></LI><LI><A href="https://2016.igem.org/Team:Freiburg/Notebook"> Journal </A></LI><LI class="menu_item"><A href="https://2016.igem.org/Team:Freiburg/Results"> Results</A></LI><LI><A href="https://2016.igem.org/Team:Freiburg/Proof"> Proof of Concept </A></LI><LI><A href="https://2016.igem.org/Team:Freiburg/Binding"> Binding </A></LI><LI><A href="https://2016.igem.org/Team:Freiburg/Delivery"> Delivery </A></LI><LI><A href="https://2016.igem.org/Team:Freiburg/Production_Nanocillus"> Nanocillus </A></LI><LI><A href="https://2016.igem.org/Team:Freiburg/Biobricks"> Biobricks </A></LI><LI><A href="https://2016.igem.org/Team:Freiburg/Shampoo"> Extra </A></LI><LI class="menu_item"><A href="https://2016.igem.org/Team:Freiburg/Safety"> Safety </A></LI><LI><A href="https://2016.igem.org/Team:Freiburg/Knockouts"> Knock Outs </A></LI><LI><A href="https://2016.igem.org/Team:Freiburg/Killswitch"> Kill Switch </A></LI><LI><A href="https://2016.igem.org/Team:Freiburg/PR"> Public relations </A></LI><LI><A href="https://2016.igem.org/Team:Freiburg/Human_Practices"> Human Practice </A></LI><LI><A href="https://2016.igem.org/Team:Freiburg/Integrated_Practices"> Integrated </A></LI><LI><A href="https://2016.igem.org/Team:Freiburg/Collaborations"> Collaborations </A></LI><LI><A href="https://2016.igem.org/Team:Freiburg/Poster"> Poster </A></LI></NAV></DIV><P id="back-top"><A href="#top">Back to Top</A></P><DIV class="column full_size"><DIV class="color15"><H5 style="text-align: center"> Our Approach </H5><DIV class="para_center_20"><FONT color="#e7d9d9">
“All Disease Begins in the Gut!” - Hippocrates


Some people are tough. And then there are some people, who have to fight against their own immune system- how tough are they? 


In the case of ulcerative colitis patients, a mistakes made by the body’s immune system lead to cyclic inflammations and ulcers in rectum and colon. To treat this overreacted immune response, patients rely on therapeutic options, which are distributed systemically. These options lead to horrible side effects, such as anemia, hysterical depression<SUP>1</SUP> and sadly, the risk for colon cancer increases enormously. 

It is our goal to develop a new system of targeted drug delivery, where the drug only acts on  the affected tissue and is not getting dispersed in the whole body. This approach will not harm the human body and has no negative side effects on it.</FONT></DIV></DIV><DIV class="color0"><DIV class="para_center_20"><FONT color="#7a7f7a"><CENTER><B>Figure 1: Drug dispersal through the body</B><B>Left</B>: Current Approacheswhich lead to negative side effects<B> Right</B>: Our new system of targeted drug delivery
</CENTER></FONT></DIV></DIV><DIV class="color15"><DIV class="para_center_20"><FONT color="#e7d9d9"> 
First, it is important to differ the terms prodrug and active drug in this system. A prodrug, precursor of a drug, must undergo chemical conversion by metabolic processes before becoming an active pharmacological agent, the active drug.
Standard drug delivery systems are based on oral administration. Patients take the pills and most of the  prodrug will be activated by enzymes in the liver and distributed throughout the entire body. Thus, the activated drug will interact with healthy tissues and organs. With the standard approach, vulnerable organs like bone marrow or brain will show symptoms like lower production of blood cells or nausea, headache etc.

The attempt to bypass the liver to avoid systemic side effects, can be achieved by a specific activation of drugs at a specific area, i.e. the inflamed cells in the colon. In our approach we used nanobodies (NB), glutathione S-transferase (GST) and endospores of the bacterium <I>B. subtilis</I> to specifically target the tissue of interest. 
Through cloning and transformation of the <I>Bacillus subtilis</I>, we were able to customize the spores to our approach. We designed fusion proteins of the outer spore coat proteins CotZ, CotB, CotG and CgeA, containing a linker peptide with HA-Tag and a nanobody, which is able to bind to a specific marker. To enable the conversion  of the prodrug to its active form, we also fused coat proteins with GST (glutathione S-transferase). Our end product is a spore combining these two functions, which is able to function as a complete new designed targeted drug delivery system, causing no negative side effects in the human body.
Our project's main target is the disease ulcerative colitis, an inflammatory bowel disease characterized by enhanced expression of CEA (carcinoembryonic antigen) on affected tissue<SUP>2</SUP>. Therefore, we considered CEA to be the perfect marker. An engineered spore presents the nanobody against CEA, for targeting of inflamed intestinal cells and the GST to locally activate the prodrug azathioprine. 
The GST on the spore surface converts azathioprine into 6-mercaptopurine. This has a toxic effect on cells in state of mitosis, because it replaces adenine and guanine in the DNA. It leads to a loss of function and therefore apoptosis of the diseased cell and enables the colon to regenerate.  </FONT></DIV></DIV><DIV class="color0"><DIV class="para_center_20"><CENTER><FONT color="#7a7f7a"><B> Figure 2: Summary of our approach </B>
The spore is binding via nanobody to the marker CEA of the inflammed tissue. The GST on the surface of the spore is able to convert azathioprine to 6-mercaptopurine.
</FONT></CENTER></DIV></DIV><DIV class="color15"><DIV class="para_center_20"><H5>Assembly of fusion genes</H5><FONT color="#e7d9d9"><B>1.       Anchor proteins</B>
The display of heterologous proteins on the surface of the spores requires their anchoring on naturally occurring structures. The spore coat of <I>B. subtilis</I> includes more than 50 components in multiple layers<SUP>3</SUP>, allowing us to test different approaches. In order to facilitate a reliable and efficient display, the anchor proteins should exhibit the following characteristics:<B>(1)</B>    They should be located at the outer layers of the coat to ensure the accessibility by the surrounding environment.<B>(2)</B>    The anchor proteins should be stable and fusion of heterologous proteins should not interfere with their folding.<B>(3)</B>    Fusion to an anchor protein should not disturb the formation and stability of the spores.<B>(4)</B>    A high abundance of the anchor is required for a high display efficiency.

 
Based on those criteria, we performed an extensive literature research evaluating anchor proteins for fusion constructs. The spore coat proteins CotB, CotG, CotZ and CgeA revealed to be the most promising candidates, due to their high abundance and localization in the outer coat and crust. Previous studies provided evidence of their feasibility for surface display applications<SUP>4–7</SUP>. <B>2.       Passenger proteins</B>
Utilizing bacterial spores for targeted drug-delivery requires the display of a moiety for specific binding of disease-associated structures and an enzymatic moiety for the activation of prodrugs. Nanobodies that are derived from camelid heavy-chain antibodies represent the ideal targeting unit, due to their high stability and specificity towards antigens. Their plain structure enables a high expression in prokaryotes and an efficient surface display on bacterial spores. The conversion of the prodrug azathioprine to its active form 6-mercaptopurine is catalyzed by glutathione S-transferase (GST) in the liver. Displaying GST on the spores permits the local activation of the prodrug avoiding systemic side effects.<B>3.       Cloning strategy</B>

For the display of the passenger proteins on the spore they should be only expressed during late stages of sporulation to avoid interference with the spore coat assembly. In addition, to prevent uneven or excessive display, the genes should be stably integrated into the genome of <I>B. subtilis</I> and the expression should be driven by an endogenous promoter.
Considering those points, we adapted our cloning strategy to achieve highly efficient display without impairing the spores themselves. The coat genes cotB, cotG, cotZ and cgeA were amplified directly from the genome of <I>B. subtilis</I> 168 and subcloned into the pJET1.2 vector for further cloning procedures to avoid unspecific amplifications and contaminations with genomic DNA. The subcloned genes were amplified again with specific primers containing extensions for the introduction of a hemagglutinin epitope (HA) tag for subsequent detection and linker regions to facilitate flexibility between the anchor and the passenger protein. The enzyme glutathione S-transferase or the anti-GFP nanobody were directly amplified from the plasmids provided by our institute. By means of the introduced overlapping regions both the coat gene and the GST or nanobody were cloned by Gibson assembly into the iGEM standard backbone pSB1C3. The Gibson assembly cloning provided the advantage of generating fusion genes without undesirable cloning scars that might interfere with the folding of the displayed proteins. The resulting fusion constructs were used for 3A assembly alongside with the endogenous <I>B. subtilis</I> promoter PCotYZ and assembled into integration vectors pBS1C, pBS2E and pBS4S. Those vectors were introduced by the iGEM Team LMU in 2012 and enable a stable integration of genes into various genomic loci of <I>B. subtilis</I><SUP>8</SUP>. Figure 1 provides an overview of the cloning strategy.
</FONT></DIV></DIV><DIV class="color0"><DIV class="para_center_20"><FONT color="#7a7f7a"><CENTER><FONT color="#7a7f7a"><B>Figure 3: Summary of the cloning strategy</B><B>(A)</B> The fusion constructs were generated by Gibson assembly and cloned into the iGEM vector pSB1C3.<B>(B)</B> Using 3A assembly the fusion constructs were inserted into an integration vector alongside with the CotYZ promoter.
</FONT></CENTER></FONT></DIV></DIV><DIV class="color15"><DIV class="para_center_20"><H5>Nanocillus</H5><FONT color="#e7d9d9"> 

Generating functionalized spores of <I>Bacillus subtilis</I> for targeted drug delivery requires the production and purification of the spores and the subsequent evaluation of protein expression. This involves transformation of competent <I>B. subtilis</I> cells with appropriate DNA encoding for proteins that can be displayed on the surface of the spores.<I>Bacillus subtilis</I> is a microorganism that is able to stably integrate DNA strands into their genome. This ability can be used to specifically choose a region in the genome where the DNA is to be inserted. <I>B. subtilis</I> is a bacterium that is able to synthesize the enzyme amylase which can degrade starch. Using the gene coding for amylase enables the easy verification of stable integration of the chosen DNA with the use of  a simple starch degradation test. 
After the selection of successfully transformed clones, sporulation can be induced through nutrient starvation of the cells in minimal media. Our protocols can be found in our <A target="_blank" href="https://2016.igem.org/Team:Freiburg/Experiments#spores" style="text-align: center;font-size:20px;color:#e8a126">method</A> section. 

Applying genetically modified organisms to the human body involves additional safety precautions to avoid undesired and uncontrolled proliferation of introduced cells causing disturbance of the own microbiome. An unbalanced microbiome, especially in the gut, can lead to serious conditions like inflammations of the bowel tissue or colorectal cancer.
Considering this important safety aspect, we included the analysis of <A target="_blank" href="https://2016.igem.org/Team:Freiburg/Knockouts" style="text-align: center;font-size:20px;color:#e8a126">germination deficient</A> strains of <I>B. subtilis</I> in our project. To avoid the administration of vegetative cells, we established a purification protocol using lysozyme to hydrolyse bacterial cells by affecting the vital component peptidoglycan, which isn’t accessible on the endospores. Since not all the vegetative cells sporulate, the purification is a very important step as the vegetative cells would distort the results in further experiments.
A quality control of the generated Nanocillus was achieved by the verification of expression and proper surface localization of ectopic proteins. Chemical decoating of endospores enabled the analysis of extracted surface proteins by including a hemagglutinin epitope tag to the introduced proteins. The detection can be verified via Western Blot. 
Verification that the introduced protein is located on the surface of the endospores and is accessible, can be tested by conjugated anti-HA antibodies. Staining of the spores with said antibodies can be used for flow cytometry analyses to demonstrate proper surface localization.
After this verification of our Nanocillus and the actual production of the spores at high scales, the capsules, serving as a carrier to the colon, need to be filled. To safely and easily manage this step, the spores were lyophilized.
</FONT></DIV></DIV><DIV class="color15"><DIV class="para_center_20"><H5>Binding of target</H5><FONT color="#e7d9d9"> 


Targeted drug delivery requires specific binding to a target structure. Many diseases are characterized by marker proteins. Some are secreted as soluble proteins and present in the blood while others are expressed on the surface of the affected cells. The latter can be used as the target structures. We modified <I>B. subtilis</I> as a carrying tool for nanobodies to bind the target structures.
Nanobodies are single domain antibodies engineered from heavy chain antibodies found in camelids<SUP>9</SUP>.</FONT></DIV></DIV><DIV class="color0"><DIV class="para_center_20"><CENTER><FONT color="#7a7f7a"><B>Figure 4: Difference between antibody and nanobody</B></FONT></CENTER></DIV></DIV><DIV class="color15"><DIV class="para_center_20"><FONT color="#e7d9d9"> 

The advantage of nanobodies includes their high specificity, small size and producibility in bacteria<SUP>9</SUP>. Therefore they are affordable and due to their size, can be variously loaded on our <I>B. subtilis</I> spores, which benefits in enormous multivalency. 

With the help of nanobodies we aim to directly target the tissues of interest. Hence, we want to minimize the amount of drugs, which would normally be distributed throughout the entire body.

During flares of ulcerative colitis the amount of carcinoembryonic antigen (CEA) increases in the intestinal epithelium<SUP>2</SUP> and is consequently accessible for nanobodies for targeting. 
CEA is a glycoprotein functioning as an adherence molecule located on the surface of mucosal cells, which we can use as a marker for ulcerative colitis<SUP>10</SUP>.</FONT></DIV></DIV><DIV class="color0"><DIV class="para_center_20"><FONT color="#7a7f7a"><CENTER><B>Figure 5: Inflamed tissue </B></CENTER></FONT></DIV></DIV><DIV class="color15"><DIV class="para_center_20"><FONT color="#e7d9d9"> 
Ideally, an anti-CEA nanobody would bind to the diseased tissue, expressing high amounts of CEA.
To verify if a nanobody-spore-complex can actually bind to a specific structure, we test our approach in the lab by using a different set of components. For our approach we used GFP, a fluorescent protein, and modified spores to incorporate anti-GFP nanobodies into their crust, resembling the binding of the anti-CEA nanobody to the CEA expressing tissue.


Due to the advantages of GFP being easily detectable, different strategies of verification could be used.
Its binding to our nanobody spore complex can be quantified via fluorescence activated cell sorting (FACS) and fluorescence microscopy. 
To further visualize the binding, glass slides were coated with GFP. Thus functioning as a substitute for the diseased tissue, to test whether our nanobody-spore-complex can directly bind to it. These samples were analysed by fluorescence microscopy.
For better illustration of the GFP signal, we’ve created 3-dimensional images using the program Mathematica.</FONT></DIV></DIV><DIV class="color15"><DIV class="para_center_20"><H5>Enzymatic activity</H5><FONT color="#e7d9d9"> 

Common medication of ulcerative colitis is based on the prodrug azathioprine. This prodrug gets converted to the active drug by glutathione S-transferase (GST) in the liver and is then distributed in the whole body through the bloodstream. 

GST is an enzyme, which plays an important role in the detoxification of the body and occurs in high amounts in the liver. The active drug, 6-mercaptopurine, replaces adenine and guanine in the DNA, which leads to errors in the DNA replication and thus apoptosis of the cell. 

Since azathioprine is administered systemically and distributed by the bloodstream, it affects the cells throughout the whole body. Our approach avoids these side effects, by only converting the prodrug at the inflamed site. Hence, our Nanocillus, will activate the drug azathioprine locally through exclusively binding to the site of inflammation. This activation is made possible due to the expression of GST on the spore surface.

To test the enzymatic function of GST on the spores a different substrate than azathioprine was used to generate more reliable results. 1-chloro-2,4-dinitrobenzene (CDNB), which represents azathioprine, and reduced glutathione (GSH) are conjugated by GST to 1-<I>S</I>-glutathionyl-2,4-dinitrobenzene (G-SDNB)<SUP>11</SUP>. The GST-catalyzed reaction produces a dinitrophenyl thioether which can be detected by spectrophotometer at 340 nm, which leads to an increase of absorbance. This gain of absorbance is proportional to the activity of the GST. 

First, a GST assay was established to determine the best conditions to characterize the spores later on. Next, different amounts of GST-expressing spores (Nanocillus) and wild type spores were compared in their enzymatic activity. Since the increase of absorbance stands in relation to the reaction rate, the conversion rate mediated by Nanocillus can then be calculated<SUP>12</SUP>.
   </FONT></DIV></DIV><DIV class="color5"><DIV class="para_center_Quellen">
Reference: 

1. George, L. , Engel, M.D., Studies of Ulcerative Colitis, The American Journal of Medicine 6 April 2004
2.  Pavelic ZP, Pavelic L, Expression of carcinoembryonic antigen in ulcerative colitis, tubular adenomas and hyperplastic polyps: correlations with the degree of dysplasia, 1991 Sep-Oct;11(5):1671-5

3.           Henriques, A. O. &amp; Moran, C. P. Structure, assembly, and function of the spore surface layers. Annu. Rev. Microbiol. 61, 555–588 (2007).
4.           Hinc, K. et al. Expression and display of UreA of Helicobacter acinonychis on the surface of Bacillus subtilis spores. Microb. Cell Fact. 9, 2 (2010).
5.           Isticato, R. et al. Surface display of recombinant proteins on Bacillus subtilis spores. J. Bacteriol. 183, 6294–301 (2001).
6.           Iwanicki, A. et al. A system of vectors for Bacillus subtilis spore surface display. Microb. Cell Fact. 13, 30 (2014).
7.           Kim, J. H., Lee, C. S. &amp; Kim, B. G. Spore-displayed streptavidin: A live diagnostic tool in biotechnology. Biochem. Biophys. Res. Commun. 331, 210–214 (2005).
8.           Radeck, J. et al. The Bacillus BioBrick Box: generation and evaluation of essential genetic building blocks for standardized work with Bacillus subtilis. J. Biol. Eng. 7, 29 (2013).


9. S. Muyldermans, T.N. Baral, 2008, Camelid immunoglobulins and nanobody technology, Volume 128, Issues 1–3, pp 178–183

10. Gardner et al, Serial carcinoembryonic antigen (CEA) blood levels in patients with ulcerative colitis., The American Journal of Digestive Diseases, Volume 23, Issue 2, pp 129–133 (1978)

11. Awasthi, Y. C., Garg, H. S., Dao, D. D., Partridge, C. A. &amp; Srivastava, S. K. Enzymatic conjugation of erythrocyte glutathione with 1-chloro-2,4-dinitrobenzene: the fate of glutathione conjugate in erythrocytes and the effect of glutathione depletion on hemoglobin. Blood 58, pp 733–738 (1981).

12. Corp, S.-A. Glutathione S-Transferase (GST) Assay Kit (CS0410)- Technical Bulletin.</DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>