<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Georgia_State_PartsDesign skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Georgia State/PartsDesign</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV class="igem_2017_content_wrapper"><DIV class="container"><DIV class="site-header"><DIV class="logo"><H1>Design</H1></DIV><DIV class="menu"><DIV class="navbar"><DIV class="nav-collapse collapse"><UL class="nav"><LI><A href="https://2017.igem.org/Team:Georgia_State" class="active">Home</A></LI><LI class="dropdown-submenu"><A class="dropdown-toggle" data-toggle="dropdown">Team
                </A></LI><LI><A href="https://2017.igem.org/Team:Georgia_State/Team">Team</A></LI><LI><A href="https://2017.igem.org/Team:Georgia_State/Attributions">Attributions</A></LI></UL><LI class="dropdown-submenu"><A class="dropdown-toggle" data-toggle="dropdown">Project
                </A></LI><LI><A href="https://2017.igem.org/Team:Georgia_State/Description">Description</A></LI><LI><A href="https://2017.igem.org/Team:Georgia_State/Design">Design</A></LI><LI><A href="https://2017.igem.org/Team:Georgia_State/Results">Results</A></LI><LI><A href="https://2017.igem.org/Team:Georgia_State/Collaborations">Collaborations</A></LI><LI class="dropdown-submenu"><A class="dropdown-toggle" data-toggle="dropdown">Parts
                </A></LI><LI><A href="https://2017.igem.org/Team:Georgia_State/Parts">Parts</A></LI><LI><A href="https://2017.igem.org/Team:Georgia_State/PartsDesign">Design</A></LI><LI><A href="https://2017.igem.org/Team:Georgia_State/Contribution">Contribution</A></LI><LI><A href="https://2017.igem.org/Team:Georgia_State/Improve">Improvement</A></LI><LI><A href="https://2017.igem.org/Team:Georgia_State/Safety">Safety</A></LI><LI class="dropdown-submenu"><A class="dropdown-toggle" data-toggle="dropdown">Notebook
                </A></LI><LI><A href="https://2017.igem.org/Team:Georgia_State/Notebook">Notebook</A></LI><LI><A href="https://2017.igem.org/Team:Georgia_State/Experiments">Experiments</A></LI>

        , <LI class="dropdown-submenu"><A class="dropdown-toggle" data-toggle="dropdown">Human Practices
                </A></LI><LI><A href="https://2017.igem.org/Team:Georgia_State/Engagement">Public Engagement</A></LI><LI><A href="https://2017.igem.org/Team:Georgia_State/HP/Silver">Silver</A></LI><LI><A href="https://2017.igem.org/Team:Georgia_State/HP/Gold_Integrated">Integrated and Gold</A><UL><A href="https://2017.igem.org/Team:Georgia_State/HP/ASL">ASL Gallery</A></UL></LI><LI><A href="https://igem.org/2017_Judging_Form?team=Georgia_State">Judging</A></LI></DIV></DIV></DIV></DIV></DIV><DIV class="container"><DIV class="featured-heading"><H1><I><FONT style="text-transform: none;">&quot;As soon as your brain starts telling you that you can't have a tree that is blue then you stop being able to paint trees.&quot;</FONT></I></H1><H2>Steven Johnson</H2><H1 style="color:#ffffff; background-color:#1F618D;; -moz-border-radius: 15px; -webkit-border-radius: 15px; padding:15px; text-align: center; font-family: Trebuchet MS">Limulus Factor C cDNA</H1><B>Limulus Factor C </B>is a relatively large protein (1020 amino acids), so the synthesis of its coding sequence presented a challenge. We determined that instead of trying to have the entire construct synthesized as one piece it would be more cost effective to build it in pieces. The coding sequence for Limulus Clotting Factor C (LFC) was designed as four contiguous blocks to be joined together via overlap extension PCR. Three of the four blocks were 800 bp long; the fourth block was 660 bp long. We designed primers for overlap extension PCR to join the g-blocks together into the final Factor C cDNA sequence. Primers were designed in both forward and reverse complement to the top and bottom strands of the blocks. For example, the forward primer of Block 2 was created with a few base pairs from Block 1 5’ top strand end. These base pairs were added to the beginning of the 5’ top strand to start amplification of Block 2’s top strand. The reverse complement primer for Block 1 was created by taking a few base pairs from the end of the 3’ bottom strand of Block 1 and adding it to the bottom strand of Block 2. The goal of creating these primers is to create overhangs from the previous block to bind to the next block. During amplification, the blocks would then be joined and extend the 5’ top strand and 3’ bottom strand of each block. Once all four blocks and a total of 8 primers were created, we began PCR amplification of Blocks 1 and 2 and then amplified Blocks 3 and 4. The fusion of the coupled blocks would then be fused together to make the final product, LFC Blocks 1-4 via overlap extension PCR. 
<B> PCR Overlap Extension</B> 
due to the expense of purchasing the entire Factor C sequence as a new part and the opportunity to attempt a method that has never been done in the Georgia State University iGEM lab Factor C is planned to be synthesized via overlap extension PCR. A modified protocol written by Ichiro Matsumura was used. This method uses PCR to recombine DNA sequences instead of using restriction sites. By using a 3'-end primer that matches each template block and a 5'-end primer that matches a part of the block sequence and a part of the new block sequence. The two blocks can be recombined using DNA polymerase. The basic mechanism for overlap begins with a PCR to generate the two fragments (AB and CD)  that have ends modified by mispriming so that they have homologous regions. Through mixture, denaturation, and reannealing the AB 3'-end will anneal onto the 3'-end of the bottom strand of the CD. Extension of the overlap product is used to form the recombinant product (the overlapping ends of the fragments are created by primers b and c while a and d match the individual fragments). The method can be repeated to add together more than two DNA fragments. 
The image above and to the left is a figure to illustrate the mechanism described above. 

<H1 style="color:#ffffff; background-color:#1F618D;; -moz-border-radius: 15px; -webkit-border-radius: 15px; padding:15px; text-align: center; font-family: Trebuchet MS">Human Chorionic Gonadotropin</H1>
When designing this part we used the recombinant cDNA sequence. While the same amino acids are present, the cDNA sequence is altered. To detect the activation of Factor C autocatalysis, a hCG-beta subunit sequence component was a necessary addition to our final Factor C cDNA sequence. Similar to the primers designed for Factor C, we designed a forward and a reverse complement primer with iGEM prefix and suffix to join Block 4 of Factor C and Block 5 of hCG-beta subunit together. For our idea of a detection system, we plan to use a pregnancy test to detect the recombinant hCG-beta subunit once Factor C is activated.
<H1 style="color:#ffffff; background-color:#1F618D;; -moz-border-radius: 15px; -webkit-border-radius: 15px; padding:15px; text-align: center; font-family: Trebuchet MS">Limulus Factor C-hCG Fusion </H1>
The basic mechanism for overlap begins with a PCR to generate two fragments (Block 1 and Block 2) that have ends modified by mispriming so that they have homologous regions. Through mixture, denaturation, and annealing of Block 1 5'-end of the top strand will anneal onto the 3'-end of the bottom strand of Block 2. Extension of the overlap product is used to form the recombinant product (the overlapping ends of the fragments are created by the forward primers and reverse complement primers to match the individual fragments). The method can be repeated to add together more than two DNA fragments (Block 3 and Block 4 + hCG-beta subunit). Together, all 5 g-blocks and a total of 10 primers would be joined via overlap extension PCR. If this joining of LFC blocks 1-4 and hCG-beta subunit block 5 is successful, the final product of <B>Limulus Clotting Factor C + hCG-beta subunit sequence</B> would be used as our complete detection system. 
<A href="https://2017.igem.org/Team:Georgia_State"><H1 style="color:#ffffff; background-color:#A9CCE3; -moz-border-radius: 15px; -webkit-border-radius: 15px; padding:15px; text-align: center; font-family: Trebuchet MS">Back to Home</H1></A><B>References</B>



- - -Horton, R. M., Cai, Z., Ho, S. N., Pease, L. R., &amp; Mayo Clinic. (2013). <I>Gene Splicing by Overlap Extension: Tailor-Made Genes Using the Polymerase Chain Reaction. BioTechniques, 54(3), 129-130 . doi:10.2144/000114017</I><A https:="">https://www.biotechniques.com/multimedia/archive/00190/BTN_A_000114017_O_190204a.pdf </A>


- - - <I>CGB protein [Homo sapiens] - Protein - NCBI. (2003, October 7). Retrieved July 7 2017, from  </I><A href="https://www.ncbi.nlm.nih.gov/protein/26996824?report=genbank&amp;log%24=protalign&amp;blast_rank=1&amp;RID=PU2MZS8Z014"> https://www.ncbi.nlm.nih.gov/protein/26996824report=genbank&amp;log%24=protalign&amp;blast_rank=1&amp;RID=PU2MZS8Z014</A>

- - - <I>P28175 (LFC_TACTR) Tachypleus tridentatus (Japanese horseshoe crab). (n.d.). Retrieved June 21, 2017, from 
 </I><A href=" https://swissmodel.expasy.org/repository/uniprot/P28175?csm=5BC2864C6715289B"> https://swissmodel.expasy.org/repository/uniprot/P28175?csm=5BC2864C6715289B</A></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>