"
Page
Discussion
View source
History
teams
Log in
 
Team:Michigan/Project
From 2011.igem.org
Home | Project | Team | BioBricks | Human Practices | Notebooks | Safety | Attributions
Cell Patterning
This year’s project explores developing a cell patterning platform based on oligonucleotide-directed cell binding to substrate surfaces. Our approach entails engineering cells to selectively bind to certain nucleotide sequences (via surface display of DNA binding proteins, such as zinc fingers), allowing for guided assembly of defined cell patterns on surfaces patterned with oligonucleotides.
Abstract
The ability of zinc finger domains to selectively bind specific double stranded DNA sequences have largely been applied intracellularly, such as in engineered zinc finger nucleases for genomic manipulations. Proteins containing zinc finger domains can also be used extracellularly to precisely adhere objects to surfaces containing bound oligonucleotides. This project aims to utilize the specificity of zinc finger protein to direct binding of Escherichia coli to oligonucleotides bound on surfaces. The fusion protein engineered to contain a fragment of the OmpA membrane domain and a zinc finger domain allows the protein to be expressed on the outside of the cell while remaining bound to the host cell. Possible applications of this project include creating patterns with fluorescently labeled cells or studying cell-cell interactions.
Research Modules
Surface Display
The goal of the SD team is to identify, construct, and test various surface display systems to support the team's ultimate aim of displaying a Zn-Finger protein on the surface of E. coli. Specifically, we have chosen to focus on systems utilizing OmpA, INP, and AIDA-1, allowing for the use of both established and unestablished BioBrick parts. Currently we are trying to build plasmids to test display by combining promoters, RBS’s, carrier’s, and test display proteins (e.g. GFP).  Combining these parts we will be able to build biobricks and test our display systems so that we are ready to attempt surface display for a zinc finger.
DNA-Binding Proteins
The objective of the DNA binding team is to find and characterize DNA binding proteins in order to identify one with 1) the strongest affinity to DNA and 2) a structure that is predicted to tolerate the linker attachment in order to surface display it on Escherichia coli. Our preliminary approach is to use zinc finger motifs as it has been well-known for its strong ability to bind to DNA and there exists a plethora of literature documenting its mechanism and combinations for optimizing binding specificity and affinity. We currently have three zinc finger protein candidates, two freely available from the iGEM Parts Registry and the third is a designer zinc finger described in Jantz et al. 2010. Binding of fluorescently labeled E. coli expressing the fusion protein will be confirmed under fluorescence microscope, and binding affinity assayed with fluorescence anisotropy.
DNA-Printing (Microarrays)
The purpose of the DNA printing team is to determine the best way to print a specific pattern of oligonucleotides onto a glass slide. After looking at several methods, we concluded that there is a huge barrier to entry for synthesizing our own glass slides and that the best way to proceed would be to order the slides pre-synthesized. This still leaves several aspects to the design of the slide that we need to work on. These include oligonucleotide density, sequences and linker sizes. We are currently conducting assays to determine the optimal density. The sequences we work with are determined by the zinc fingers that we will use. They consist of repeats of the zinc finger recognition sites, sometimes with filler DNA in between.
Results
Assaying Binding Affinity with Fluorimeter
To test whether the fusion proteins are present on the cell surface in active form, we assayed the binding affinity of the cells to the fluorescence-labeled oligonucleotides (oligos) by measuring the intensity of fluorescence after polarized excitation through an emission polarizer. We labeled our recognition oligos with 6-FAM and CY3: 6-FAM is bound to the recognition sequence corresponding to Zif268, while CY3 is bound to the recongnition sequence of Gli-1. CY3's excitation wavelength is 550 nm, and emission is 564 nm; 6-FAM excitation is 495 nm and emission is 520 nm. This method is a derivative of fluorescence anisotropy, an assay that measures the tumbling frequency of any molecule tagged with a fluorophore. It requires a polarized excitation of the fluorophore to produce a partially oriented population, and to measure emission with a polarizer parallel or perpendicular to the orientation of the excitation polarization. Factors that affect the tumbling frequency depends on the mass and shape of the tagged molecule as while as the viscosity of the solvent. When another, significantly larger, untagged molecule is tightly bound to the tagged molecule, the tumbling frequency decreases due to the change in inertia and shape of the complex.
We carried out an experiment that measured the binding affinity of cells expressing the zinc finger DNA binding domain on the outer membrane to the fluorescently-labelled oligos. We had two sets of negative controls: one with uninduced BL21 and the other is BL21 expressing the other zinc finger domain. The negative control with the uninduced BL21 is to test nonspecific binding of oligo to the outer membrane of ''E. coli'', and the second control of BL21 expressing a zinc finger domain with a different recognition sequence (e.g. Cy3 mixing with Zif268) is to test for nonspecific binding of zinc finger domain to oligo. We carried out our experiment in FluoroMax-2 machine with autopolarizer accessory.
Discussion
Based on the data from our experiment, we conclude that there was not a significant difference in the tumbling frequency of the oligos whether it was in the presence of uninduced BL21 or zinc finger domain expressing cells. Unfortunately, the software controlling the fluorimeter was not fully compatible with the anisotropy accessory, and we were unable to alter emission polarization orientation. As a result, we are only able to compare between negative control and sample values, but no meaningful anisotropy or polarization measurements can be extracted from the data.
Binding Assay with UV/Vis Spectrophotometer
A second attempt to assay the binding affinity of the expressed cells with the fluorescence oligos was carried out with a UV/Vis Spectrophotometer.  A binding assay was performed with fluorescently-labeled oligos. Three types of cells were grown: BL21 with no fusion construct, BL21 carrying an uninduced fusion construct (LOG or LOZ), and BL21 carrying an induced fusion product. The latter two were then incubated with double-stranded, fluorescently-labeled oligos corresponding to the Gli-1 or Zif268 recognition sequence. After incubation, cells were pelleted, washed, and resupended in 20μM ZnSO4, 20mM Tris, pH 8.0 buffer. Emission data were recorded at wavelengths corresponding to the fluorophores used.
Left: Samples with Gli-1 expression under an excitation frequency of 550 and emission freq of 564 (bottom plate read). Right: Samples with Znf268 expression under excitation of 495 and emission of 520 (bottom plate read). Dup1, 2 and Rep1 and 2 represent the two different colonies selected from the same plate. Samples on both graphs from left to right: lac-ompA-Gli (induced), lac-ompA-Gli (uninduced), lac-ompA-Znf268 (induced), LOZ (uninduced), buffer alone, buffer and CY3, buffer and 6FAM.
Discussion
Unfortunately, none of the samples showed fluorescence higher than BL21 alone, which is consistent with the fluorescent polarization assay. We cannot say whether this is due to failure to establish proper conditions for binding, inefficient transport to the membrane, improper folding or instability of the fusion protein.
Retrieved from " http://2011.igem.org/Team:Michigan/Project "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
