"
Page
Discussion
View source
History
teams
Log in
 
Team:Paris Bettencourt/tRNA diffusion/Random walker
From 2011.igem.org
Team IGEM Paris 2011
Home
Presentation
Overview
Potential applications
Collaborations
The team
Acknowledgements →
Host laboratory
Sponsors
Designs
Designs overview & concepts
Designs list
Modeling
General overview
Modeling results for our designs →
The basics
Our assumptions
T7 polymerase diffusion
tRNA diffusion
ComK/ComS system
Parameters
Results of our passive diffusion model
Toward assisted diffusion
Data
Results
Achievements
Methodologies →
Microscopy
Micro-fluidic
Integration in B.subtilis
Kinetics
Parts list
Notebook
Human practice & Safety
Human practice
Safety
Brainstorming
Bonus track
Effective translation of mRNA amber and tRNA amber diffusion
We already justified the assumption of neglecting the time it takes for a molecule to reach any given point in a cell (see Modeling assumptions ). However in this design we want to have two tRNA amber reaching the amber codons on the same mRNA, exactly when the ribosome is on the first and second amber codon. This requires a little more analysis before validating the possibility of the translation of the mRNA amber into a fully functional T7 RNA polymerase within a reasonable timeframe.
Random walker model
This model is very similar to the one used in [1] . We consider that the tRNA amber diffusing in the cell is a random walker.
We want to see how long it takes for a tRNA to diffuse to any point of a cell. We use the following parameters:
V volume of the cell (10-18 m3)
characteristic size of the particle (1.5x10-8m [1] )
D diffusion coefficient of the particle (2.57x10-11m2.s-1 [1] )
We divide the cytoplasm volume V into
for the walker. The characteristic time
associated with the transition from one site to another is:
[2]
If we have R walkers of this type, the probability that a molecule arrives at a given occupation site during the time interval
is:
.
Until this point, the model does not differ from what we used to do. But let's dive into the additions we had to make to explore the possibility of functional T7 RNA polymerase production.
The specificities of our design
What really interest us is the probability that two amber codons are successfully translated with the help of tRNA amber by the same ribosome. We have to monitor all the amber codons placed on our mRNA amber present in a cell. We therefore have two occupation sites of interest for each mRNA amber in the cell.
There are three important time scales in this model:
The time scale for tRNA diffusion. The tRNA amber population of an occupation site is changed every 1.46x10-5s. This means that you obtain a new tRNA population in an occupation site every 1.46x10-5s with a
probability of picking a tRNA amber.
The time scale of ribosome movement on mRNA. The speed of ribosome is roughly 15 codon.s-1 [3] so the average time spent by a ribosome on a codon is:
The time scale of ribosome movement between two amber codons. Since there are 48 nucleotides between two amber codons in our design and given the fact that the speed of ribosome is roughly 15 codon.s-1 [3] , we assumed there was one ribosome per second passing on each codon. This is also confirmed by the fact that there are usually between 40 and 80 nucleotides between two ribosomes on the same mRNA strand [4] .
Our model checks every second the tRNA amber population of each of our amber codon (corresponding each to an occupation site). When the ribosome is on the codon, the population of the occupation site is renewed
times (approximately 4500 times). This means that the population site is renewed 4500 times with each time a probability of
of picking a tRNA amber.
Our simulation records the tRNA amber population of all the occupation sites. This means we assume that at each second there is a ribosome on the amber codon (which is justified by our third time scale). We then check the pairs of occupation sites (corresponding to the pairs of amber codons on mRNA amber). If there is a tRNA amber at time t in the first codon of the pair and a tRNA amber at time t+1 on the second codon of the pair, then a functional T7 RNA polymerase has been produced.
Results
We ran our simulation in Matlab using the following parameters:
There are 40 mRNA amber in the cell (a reasonnable assumption).
There are 20 tRNA amber in the cell
Results can be found in the following figure:
Matlab simulation of the T7 RNA polymerase production through mRNA amber
As you can see, production of T7 RNA polymerase is swift and since we have another independant auto-amplification loop, the receiver for the tRNA diffusion design should work perfectly.
Limits
Our model does not take any kind of translation error into account. We assume that if a tRNA amber is accessible to the ribosome at the right time, it will be used, that a ribosome always reads the entire mRNA strand without detaching itself and that the protein is always functionnal when the translation was correct. We also placed ourself in a situation with a "steady state" for the number of tRNA amber. Since our model was used only to prove that mRNA amber correct translation with very few tRNA amber was possible (not necessarily efficient) and since our results are more than satisfactory, we find these limits not to be a real obstacle to the validity of the model.
References
Ribosome kinetics and aa-tRNA competition determine rate and fidelity of peptide synthesis. Fluitt A, Pienaar E, Viljoen H. Comput Biol Chem. 2007 Oct;31(5-6):335-46. Epub 2007 Aug 15.
Diffusion-based Channel Characterization in Molecular Nanonetworks. Llatser, I., Alarcón, E. and Pierobon, M., to appear in Proc. of the 1st IEEE International Workshop on Molecular and Nano Scale Communication (MoNaCom), held in conjunction with IEEE INFOCOM, Shanghai (China), April 2011
BioNumbers
BioNumbers
Recent changes
What links here
Related changes
Upload file
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
Retrieved from " http://2011.igem.org/Team:Paris_Bettencourt/tRNA_diffusion/Random_walker "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
