"
Page
Discussion
View source
History
teams
Log in
 
Team:Paris Bettencourt/MultiHost
From 2011.igem.org
Team IGEM Paris 2011
Home
Presentation
Overview
Potential applications
Collaborations
The team
Acknowledgements →
Host laboratory
Sponsors
Designs
Designs overview & concepts
Designs list
Modeling
General overview
Modeling results for our designs →
The basics
Our assumptions
T7 polymerase diffusion
tRNA diffusion
ComK/ComS system
Parameters
Results of our passive diffusion model
Toward assisted diffusion
Data
Results
Achievements
Methodologies →
Microscopy
Micro-fluidic
Integration in B.subtilis
Kinetics
Parts list
Notebook
Human practice & Safety
Human practice
Safety
Brainstorming
Bonus track
Contents
1 Description
1.1 A new vector
1.2 Biobricking
1.3 Sequencing
2 Caracterization
2.1 Miniprep preparation
2.2 Sequence analysis
2.3 Cloning in K606003
2.4 Electroporation protocol for B. subtilis
2.4.1 Reagents and Equipment needed
2.4.2 Day 1: preparation
2.4.3 Day 2: electro-poration
Description
A new vector
There were one multi host vector in the registry, but they were not able to send it to us. The one we received finally was not able to growth in culture. So we had to build our new plasmid for our experiments.
This plasmid was obtained thanks to Harald Putzer's laboratory at IBPC, and in the corresponding publication, hold the name of pHM3. This is the fusion of two commercial plasmids via their EcoRI site. The pBC16-1 plasmid is a commercial low copy plasmid for Bacillus Subtilis. SynPTZ19R is a medium copy plamid for E. Coli.
Biobricking
As there were two EcoRI site, we had to remove one of them. This was done by menaged digestion and completion using Taq poylmerase. The blunt ends were then religated. In order to make this plasmid biobricked and in order to have a negative digestion control in the frame of the new 2011 cloning method, we inserted the biobrick J0445 inside . By doing that we recovered the EcoRI, XbaI, SpeI and PstI site of the biobrick format. The plasmid map is now the following:
Sequencing
This vector has been sequenced de novo and we testify the sequence in the registry is correct.
We sequenced several time with sucess the biobrick inserted using these primers:
K606003-Fwd: GTATATAAACATTCTCAAAGG
K606003-Rev: GGATAACAATTTCACACAGG
They also can be used for PCR Colony
Caracterization
Miniprep preparation
An overnight miniprep of this plasmid in E. Coli ofter gives yields around 200ng/µL which is lower than the standards plasmid of the database that gives around 500-800 ng/µL in the same conditions. This confirms the plasmid is medium copy in E. Coli.
Sequence analysis
The plasmid is found out to have a pT7 promoter in the reverse direction compared to the biobrick. It may cause a problem wih construction holding a T7 polymerase
Cloning in K606003
K606003 is not a synthetic biology plasmid. It is not silent, and it is a bit more difficult to clone inside than a regular registry plasmid. We have troubledshooted this plasmid a lot and we can now gives good advices for a good method to clone in this vector.
The vector is ampicilyn resistant. The resistance casette is a bit stronger than a standard resistance casette. We advise you to put a bit more antibiotic than usual. Ampicilyn is an antibiotic that is destroyed outside the cells. If the bacteria are too concentrated, the ampicilyn disapears quickly, and the bacteria loose it's plasmid (that is already big). That's why, one have to care not to grow the bacteria for a too long time when minipreping, and to take well insulated colonies when picking on the plate. Beware, satellites colonies appears fast due to the strengh of the ampicilyn casette. They do not contain the plasmid.
Picture of the plasmid in E. Coli when the cells are plated with a too high density
Here is a protocol we advise you to follow. Once your ligation is done and plated, pick up 10 colonies, that you grow in LB and make PCR colony on in the mean time. Use the primers whose sequence is given above. From the result of the PCR, select the goods tubes that indeed contains your insert, replicate them for glycerols, miniprep them and sequence. Your cloning will be sucessful each time!
Electroporation protocol for B. subtilis
Electroporation protocol adapted from Cao et al. article (2011) that has given us some stable results over time. Because of the resistance of Bacillus subtilis to electro-poration higher voltages are needed to the point of destroying the population, all the different sugars used here are meant to stabilize the membrane of the cells.
Reagents and Equipment needed
Mannitol, Sorbitol, Trehalose, LB, glycerol (99,5%)
DNA (50 ng/μL), B. subtilis strain for transformation (no need to be competent)
Cuvette (2mm), Gene Pulser (Bio-rad) set on 200 ohms and 25 μF (≈ 5 ms pulses) and between 2 and 2.5 kV
Centrifuge set at 3000g and 10 minutes
Micropipettes: P2, P200, P1000
Pipettes: 25 mL, 10 mL, 5 mL
Polyethylene round bottom tubes
Day 1: preparation
Growth medium:
LB + 0.5 mol.L-1 sorbitol → expected final volume: 52 mL for 1 cell culture (including 1 mL for taring the absorbance machine)
msorbitol ≈ 4,736 g
Electro-poration medium:
de-ionized water + 0.5 mol.L-1 sorbitol + 0.5 mol.L-1 mannitol + 0.5 mol.L-1 trehalose + 10% glycerol (v/v) → expected final volume ≈ 40 mL for 1 round of poration
msorbitol = mmannitol ≈  3,643 g
mtrehalose ≈ 7,566 g
V99,5 % glycerol ≈  4 mL
Recovery medium:
LB + 0.5 mol.L-1 sorbitol + 0.38 mol.L-1 mannitol → expected final volume: ≈ 1.1 ml per poly... tube (Approximately 10 tubes ≈  11 mL total)
msorbitol ≈ 1,002 g (for 10 tubes)
mmannitol ≈  0,761 g (for 10 tubes)
Sterilise the solution: Filtration or autoclave.
Inoculate a falcon containing 10 ml of LB with your subtilis strain and let it grow overnight (37°C with shaking).
Day 2: electro-poration
Monitor the OD600 of your overnight culture.
In a 500 ml erlenmeyer: dilute your culture into 50mL of Growth Medium so that the OD 600 is 0.01.
Let the culture grow (37°C with shaking) until OD600 is between 0.85 and 1.
Cool the cells on ice for 5 minutes.
NOTE: KEEP ALL YOUR MATERIAL ON ICE AND ALWAYS MANIPULATE ON ICE FROM NOW ON, KEEP AS STERILE AS POSSIBLE.
Distribute evenly the culture into two falcons and centrifuge at 3000g for 10 minutes.
Get rid of supernatant, tap the falcon upside down on a piece of paper to get rid of as much solution possible. Detach the pellet.
Add 20 mL of ice-cold electro-poration medium to one falcon, suspend the cells and transfer the content to the other falcon. Re-suspend.
Centrifuge 3000g for 10 minutes.
Remove supernatant, detach pellet, add 10 mL of ice-cold electro-poration medium. Centrifuge.
Repeat step 10 with 5 mL, 2.5 mL and finally add 0.625 mL (1/80 of initial volume).
During the centrifugation time, prepare the polyethylene tubes (label them) with recovery medium in them and put the cuvettes on ice: 1 of each at least has to be a control of cells without DNA, then 1 for each transformant you wish to make.
Transfer in a cuvette: 60 μL of cells + 1 μL of DNA (50ng/μL; none if control).
Pulse the cuvette.
Transfer immediately the content into the poly... tube (STERILE CONDITIONS).
Repeat 13, 14 and 15 for the number of prepared cuvettes.
Incubate the polyethylene tubes at 37°C for 3 to 6 hours.
Prepare plates with antibiotics (none for the control).
Note: ≈ 25 ml of LBA per petri dish, make sure the antibiotic is well diluted, labeling should be obvious.
Plate max 100 μL of transformed cells per petri dish and let grow overnight. More is going to create a biofilm if the transformation worked and therefore make it impossible to select a single colony.
Recent changes
What links here
Related changes
Upload file
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
Retrieved from " http://2011.igem.org/Team:Paris_Bettencourt/MultiHost "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
