<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Paris_Bettencourt_MultiHost"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Paris_Bettencourt/MultiHost">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Paris_Bettencourt/MultiHost&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Paris_Bettencourt/MultiHost&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Paris_Bettencourt/MultiHost&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Paris_Bettencourt/MultiHost" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Paris Bettencourt/MultiHost</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><P><TITLE>Team IGEM Paris 2011</TITLE></P><DIV id="main_paris2011"><DIV id="menu_paris2011"><UL id="menu_list_paris2011"><LI class="menu_list_item_paris2011"><A href="https://2011.igem.org/Team:Paris_Bettencourt">Home</A></LI><LI class="menu_list_item_paris2011"><A href="https://2011.igem.org/Team:Paris_Bettencourt/Project">Presentation</A></LI><LI class="menu_list_item_paris2011"><A href="https://2011.igem.org/Team:Paris_Bettencourt/Project">Overview</A></LI><LI class="menu_list_item_paris2011"><A href="https://2011.igem.org/Team:Paris_Bettencourt/Potential_Application">Potential applications</A></LI><LI class="menu_list_item_paris2011"><A href="https://2011.igem.org/Team:Paris_Bettencourt/collaborations">Collaborations</A></LI><LI class="menu_list_item_paris2011"><A href="https://2011.igem.org/Team:Paris_Bettencourt/Team">The team</A></LI><LI class="menu_list_item_paris2011"><A href="https://2011.igem.org/Team:Paris_Bettencourt/Sponsors">Acknowledgements  →</A></LI><LI class="menu_list_item_paris2011"><A href="https://2011.igem.org/Team:Paris_Bettencourt/Inserm">Host laboratory</A></LI><LI class="menu_list_item_paris2011"><A href="https://2011.igem.org/Team:Paris_Bettencourt/Sponsors">Sponsors</A></LI></UL><LI class="menu_list_item_paris2011"><A href="https://2011.igem.org/Team:Paris_Bettencourt/Designs">Designs</A></LI><LI class="menu_list_item_paris2011"><A href="https://2011.igem.org/Team:Paris_Bettencourt/Designs">Designs overview &amp; concepts</A></LI><LI class="menu_list_item_paris2011"><A href="https://2011.igem.org/Team:Paris_Bettencourt/Designs/List">Designs list</A></LI><LI class="menu_list_item_paris2011"><A href="https://2011.igem.org/Team:Paris_Bettencourt/Modeling">Modeling</A></LI><LI class="menu_list_item_paris2011"><A href="https://2011.igem.org/Team:Paris_Bettencourt/Modeling">General overview</A></LI><LI class="menu_list_item_paris2011"><A href="https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Designs">Modeling results for our designs →</A></LI><LI class="menu_list_item_paris2011"><A href="https://2011.igem.org/Team:Paris_Bettencourt/what_is_modeling">The basics</A></LI><LI class="menu_list_item_paris2011"><A href="https://2011.igem.org/Team:Paris_Bettencourt/Hypothesis">Our assumptions</A></LI><LI class="menu_list_item_paris2011"><A href="https://2011.igem.org/Team:Paris_Bettencourt/Modeling/T7_diffusion">T7 polymerase diffusion</A></LI><LI class="menu_list_item_paris2011"><A href="https://2011.igem.org/Team:Paris_Bettencourt/Modeling/tRNA_diffusion">tRNA diffusion</A></LI><LI class="menu_list_item_paris2011"><A href="https://2011.igem.org/Team:Paris_Bettencourt/Modeling/ComS_diffusion">ComK/ComS system</A></LI><LI class="menu_list_item_paris2011"><A href="https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Parameters">Parameters</A></LI><LI class="menu_list_item_paris2011"><A href="https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Diffusion">Results of our passive diffusion model</A></LI><LI class="menu_list_item_paris2011"><A href="https://2011.igem.org/Team:Paris_Bettencourt/Modeling/Assisted_diffusion">Toward assisted diffusion</A></LI><LI class="menu_list_item_paris2011"><A href="https://2011.igem.org/Team:Paris_Bettencourt/Data">Data</A></LI><LI class="menu_list_item_paris2011"><A href="https://2011.igem.org/Team:Paris_Bettencourt/Experiments/List">Results</A></LI><LI class="menu_list_item_paris2011"><A href="https://2011.igem.org/Team:Paris_Bettencourt/Experiments/List">Achievements</A></LI><LI class="menu_list_item_paris2011"><A href="https://2011.igem.org/Team:Paris_Bettencourt/Experiments/Methodologies">Methodologies →</A></LI><LI class="menu_list_item_paris2011"><A href="https://2011.igem.org/Team:Paris_Bettencourt/Experiments/Microscopy">Microscopy</A></LI><LI class="menu_list_item_paris2011"><A href="https://2011.igem.org/Team:Paris_Bettencourt/Experiments/Methodologies/Microchemostat_HastyJ">Micro-fluidic</A></LI><LI class="menu_list_item_paris2011"><A href="https://2011.igem.org/Team:Paris_Bettencourt/Experiments/Methodologies/Integration">Integration in <I>B.subtilis</I></A></LI><LI class="menu_list_item_paris2011"><A href="https://2011.igem.org/Team:Paris_Bettencourt/Experiments/Methodologies/Tecan">Kinetics</A></LI><LI class="menu_list_item_paris2011"><A href="https://2011.igem.org/Team:Paris_Bettencourt/Parts">Parts list</A></LI><LI class="menu_list_item_paris2011"><A href="https://2011.igem.org/Team:Paris_Bettencourt/Notebook">Notebook</A></LI><LI class="menu_list_item_paris2011"><A href="https://2011.igem.org/Team:Paris_Bettencourt/HumanPractice" style="width:150px">Human practice &amp; Safety</A></LI><LI class="menu_list_item_paris2011"><A href="https://2011.igem.org/Team:Paris_Bettencourt/HumanPractice">Human practice</A></LI><LI class="menu_list_item_paris2011"><A href="https://2011.igem.org/Team:Paris_Bettencourt/Safety">Safety</A></LI><LI class="menu_list_item_paris2011"><A href="https://2011.igem.org/Team:Paris_Bettencourt/Brainstorming">Brainstorming</A></LI><LI class="menu_list_item_paris2011"><A href="https://2011.igem.org/Team:Paris_Bettencourt/Protocol_the_song">Bonus track</A></LI></DIV><DIV id="maincontainer_paris2011"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Description"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Description</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#A_new_vector"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">A new vector</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Biobricking"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Biobricking</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Sequencing"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Sequencing</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-5"><A href="#Caracterization"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Caracterization</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Miniprep_preparation"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Miniprep preparation</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#Sequence_analysis"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">Sequence analysis</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#Cloning_in_K606003"><SPAN class="tocnumber">2.3</SPAN><SPAN class="toctext">Cloning in K606003</SPAN></A></LI><LI class="toclevel-2 tocsection-9"><A href="#Electroporation_protocol_for_B._subtilis"><SPAN class="tocnumber">2.4</SPAN><SPAN class="toctext">Electroporation protocol for B. subtilis</SPAN></A></LI><LI class="toclevel-3 tocsection-10"><A href="#Reagents_and_Equipment_needed"><SPAN class="tocnumber">2.4.1</SPAN><SPAN class="toctext">Reagents and Equipment needed</SPAN></A></LI><LI class="toclevel-3 tocsection-11"><A href="#Day_1:_preparation"><SPAN class="tocnumber">2.4.2</SPAN><SPAN class="toctext">Day 1: preparation</SPAN></A></LI><LI class="toclevel-3 tocsection-12"><A href="#Day_2:_electro-poration"><SPAN class="tocnumber">2.4.3</SPAN><SPAN class="toctext">Day 2: electro-poration</SPAN></A></LI></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Description"> Description </SPAN></H2><H3><SPAN class="mw-headline" id="A_new_vector"> A new vector </SPAN></H3><P>There were one multi host vector in the registry, but they were not able to send it to us. The one we received finally was not able to growth in culture. So we had to build our new plasmid for our experiments.
</P><P>This plasmid was obtained thanks to Harald Putzer's laboratory at IBPC, and in the corresponding publication, hold the name of pHM3. This is the fusion of two commercial plasmids via their EcoRI site. The pBC16-1 plasmid is a commercial low copy plasmid for Bacillus Subtilis. SynPTZ19R is a medium copy plamid for E. Coli.
</P><H3><SPAN class="mw-headline" id="Biobricking"> Biobricking </SPAN></H3><P>As there were two EcoRI site, we had to remove one of them. This was done by menaged digestion and completion using Taq poylmerase. The blunt ends were then religated. In order to make this plasmid biobricked and in order to have a negative digestion control in the frame of the new 2011 cloning method, we inserted the biobrick J0445 inside . By doing that we recovered the EcoRI, XbaI, SpeI and PstI site of the biobrick format. The plasmid map is now the following:
</P><H3><SPAN class="mw-headline" id="Sequencing"> Sequencing </SPAN></H3><P>This vector has been sequenced de novo and we testify the sequence in the registry is correct.
</P><P>We sequenced several time with sucess the biobrick inserted using these primers:
</P><P><U>K606003-Fwd:</U>
GTATATAAACATTCTCAAAGG
</P><P><U>K606003-Rev:</U>
GGATAACAATTTCACACAGG
</P><P>They also can be used for PCR Colony
</P><H2><SPAN class="mw-headline" id="Caracterization"> Caracterization </SPAN></H2><H3><SPAN class="mw-headline" id="Miniprep_preparation"> Miniprep preparation </SPAN></H3><P>An overnight miniprep of this plasmid in E. Coli ofter gives yields around 200ng/µL which is lower than the standards plasmid of the database that gives around 500-800 ng/µL in the same conditions. This confirms the plasmid is medium copy in E. Coli.
</P><H3><SPAN class="mw-headline" id="Sequence_analysis"> Sequence analysis </SPAN></H3><P>The plasmid is found out to have a pT7 promoter in the reverse direction compared to the biobrick. It may cause a problem wih construction holding a T7 polymerase
</P><H3><SPAN class="mw-headline" id="Cloning_in_K606003"> Cloning in K606003 </SPAN></H3><P>K606003 is not a synthetic biology plasmid. It is not silent, and it is a bit more difficult to clone inside than a regular registry plasmid. We have troubledshooted this plasmid a lot and we can now gives good advices for a good method to clone in this vector.
</P><P>The vector is ampicilyn resistant. The resistance casette is a bit stronger than a standard resistance casette. We advise you to put a bit more antibiotic than usual. Ampicilyn is an antibiotic that is destroyed outside the cells. If the bacteria are too concentrated, the ampicilyn disapears quickly, and the bacteria loose it's plasmid (that is already big). That's why, one have to care not to grow the bacteria for a too long time when minipreping, and to take well insulated colonies when picking on the plate. Beware, satellites colonies appears fast due to the strengh of the ampicilyn casette. They do not contain the plasmid.
</P><TABLE><TBODY><TR><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Picture of the plasmid in E. Coli when the cells are plated with a too high density</DIV></DIV></DIV></DIV></TR></TBODY></TABLE><P>Here is a protocol we advise you to follow. Once your ligation is done and plated, pick up 10 colonies, that you grow in LB and make PCR colony on in the mean time. Use the primers whose sequence is given above. From the result of the PCR, select the goods tubes that indeed contains your insert, replicate them for glycerols, miniprep them and sequence. Your cloning will be sucessful each time!
</P><H3><SPAN class="mw-headline" id="Electroporation_protocol_for_B._subtilis"> Electroporation protocol for <I>B. subtilis</I></SPAN></H3><P>Electroporation protocol adapted from Cao et al. article (2011) that has given us some stable results over time. Because of the resistance of <I>Bacillus subtilis</I> to electro-poration higher voltages are needed to the point of destroying the population, all the different sugars used here are meant to stabilize the membrane of the cells.
</P><H4><SPAN class="mw-headline" id="Reagents_and_Equipment_needed"> Reagents and Equipment needed </SPAN></H4><P>Mannitol, Sorbitol, Trehalose, LB, glycerol (99,5%) 
DNA (50 ng/μL), <I>B. subtilis</I> strain for transformation (no need to be competent) 
Cuvette (2mm), Gene Pulser (Bio-rad) set on 200 ohms and 25 μF (≈ 5 ms pulses) and between 2 and 2.5 kV 
Centrifuge set at 3000g and 10 minutes 
Micropipettes: P2, P200, P1000
Pipettes: 25 mL, 10 mL, 5 mL
Polyethylene round bottom tubes </P><H4><SPAN class="mw-headline" id="Day_1:_preparation"> Day 1: preparation </SPAN></H4><UL><LI>Growth medium: 
</LI><LI>LB + 0.5 mol.L<SUP>-1</SUP> sorbitol → expected final volume: 52 mL for 1 cell culture (including 1 mL for taring the absorbance machine)
</LI><LI>msorbitol ≈ 4,736 g 
</LI></UL><LI>Electro-poration medium: 
</LI><LI>de-ionized water + 0.5 mol.L<SUP>-1</SUP> sorbitol + 0.5 mol.L<SUP>-1</SUP> mannitol + 0.5 mol.L<SUP>-1</SUP> trehalose + 10% glycerol (v/v) → expected final volume ≈ 40 mL for 1 round of poration
</LI><LI>m<SUB>sorbitol</SUB> = m<SUB>mannitol</SUB> ≈  3,643 g
</LI><LI>m<SUB>trehalose</SUB> ≈ 7,566 g 
</LI><LI>V<SUB>99,5 % glycerol</SUB> ≈  4 mL
</LI><LI>Recovery medium: 
</LI><LI>LB + 0.5 mol.L<SUP>-1</SUP> sorbitol + 0.38 mol.L<SUP>-1</SUP> mannitol → expected final volume: ≈ 1.1 ml per poly... tube (Approximately 10 tubes ≈  11 mL total)
</LI><LI>m<SUB>sorbitol</SUB> ≈ 1,002 g (for 10 tubes)
</LI><LI>m<SUB>mannitol</SUB> ≈  0,761 g (for 10 tubes)
</LI><LI>Sterilise the solution: Filtration or autoclave.
</LI><LI>Inoculate a falcon containing 10 ml of LB with your <I>subtilis</I> strain and let it grow overnight (37°C with shaking).
</LI><H4><SPAN class="mw-headline" id="Day_2:_electro-poration"> Day 2: electro-poration </SPAN></H4><OL><LI> Monitor the OD600 of your overnight culture.
</LI><LI> In a 500 ml erlenmeyer: dilute your culture into 50mL of Growth Medium so that the OD 600 is 0.01.
</LI><LI> Let the culture grow (37°C with shaking) until OD600 is between 0.85 and 1.
</LI><LI> Cool the cells on ice for 5 minutes. 
</LI><LI> NOTE: KEEP ALL YOUR MATERIAL ON ICE AND ALWAYS MANIPULATE ON ICE FROM NOW ON, KEEP AS STERILE AS POSSIBLE.
</LI><LI> Distribute evenly the culture into two falcons and centrifuge at 3000g for 10 minutes.
</LI><LI> Get rid of supernatant, tap the falcon upside down on a piece of paper to get rid of as much solution possible. Detach the pellet.
</LI><LI> Add 20 mL of ice-cold electro-poration medium to one falcon, suspend the cells and transfer the content to the other falcon. Re-suspend.
</LI><LI> Centrifuge 3000g for 10 minutes.
</LI><LI> Remove supernatant, detach pellet, add 10 mL of ice-cold electro-poration medium. Centrifuge.
</LI><LI>Repeat step 10 with 5 mL, 2.5 mL and finally add 0.625 mL (1/80 of initial volume).
</LI><LI> During the centrifugation time, prepare the polyethylene tubes (label them) with recovery medium in them and put the cuvettes on ice: 1 of each at least has to be a control of cells without DNA, then 1 for each transformant you wish to make.
</LI><LI> Transfer in a cuvette: 60 μL of cells + 1 μL of DNA (50ng/μL; none if control).
</LI><LI> Pulse the cuvette.
</LI><LI> Transfer immediately the content into the poly... tube (STERILE CONDITIONS).
</LI><LI> Repeat 13, 14 and 15 for the number of prepared cuvettes.
</LI><LI> Incubate the polyethylene tubes at 37°C for 3 to 6 hours.
</LI><LI> Prepare plates with antibiotics (none for the control).
</LI><LI> Note: ≈ 25 ml of LBA per petri dish, make sure the antibiotic is well diluted, labeling should be obvious.
</LI><LI> Plate max 100 μL of transformed cells per petri dish and let grow overnight. More is going to create a biofilm if the transformation worked and therefore make it impossible to select a single colony.
</LI></OL><DIV id="footer-wrapper"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Paris_Bettencourt/Modeling" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Paris_Bettencourt/Modeling" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-upload"><A href="/Special:Upload" title="Upload files [u]" accesskey="u">Upload file</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Paris_Bettencourt/Modeling&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Paris_Bettencourt/Modeling&amp;oldid=86565" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:Paris_Bettencourt/MultiHost">http://2011.igem.org/Team:Paris_Bettencourt/MultiHost</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Paris_Bettencourt/MultiHost" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Paris_Bettencourt/MultiHost" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Paris_Bettencourt/MultiHost&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Paris_Bettencourt/MultiHost&amp;oldid=111536" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>