"
Page
Discussion
View source
History
teams
Log in
 
Team:Yale/Project Overview
From 2013.igem.org
Return to iGEM main page
Home
Team
Official Team Profile
Project
Parts Submitted to the Registry
Modeling
Notebook
Outreach
Safety
Attributions
Project Overview
Validate PLA Synthesis
Develop Bioassay
Apply Mage
Introduce Export System
Make a Bioplastic
Collaboration
Project Overview
We aim to develop a more efficient method for the biosynthesis of poly(lactic acid) from E. coli through the use of multiplex automated genome engineering. We hope this method will lower both the economic and environmental costs of synthesizing PLA in such a way as to revolutionize the biomaterials industry.
Background: Poly(lactic acid) and its biosynthesis
Poly(lactic acid) (PLA) is a plastic that has become very attractive as of late due to various properties that make it an excellent biomaterial. It is biodegradable, having a typical lifetime of about 6 months to 2 years until microorganisms break it down into water and carbon dioxide. It is biocompatible, degrading throughout the entire plastic instead of starting with the outermost layer, allowing the body’s immune response to break down the pieces before it has the time to overreact and damage surrounding tissues. It is bioabsorbable, allowing it to be resorbed into the body for applications such as spinal implants, slowly transferring the load to the body and allowing the bone to heal in a physically supportive environment. Lastly, it is thermoplastic, allowing it to be extruded in filament form and reshaped, for use in a three-dimensional (3D) printer.
Due to these properties, PLA is a popular material used in everything from making compostable cups to scaffolding for tissue engineering to 3D printing parts for a remote-control helicopter. Unfortunately, conventional methods of synthesizing PLA chemically are quite expensive: one gram of pure PLA costs around $90. Moreover, even though it is mostly manufactured from corn, the processing and purifying steps use many chemicals that are environmentally unfriendly. Even ignoring those steps, it takes about 2.65 kilograms of corn to make 1 kilogram of PLA; all that corn occupies a lot of land.
These issues have led some groups to pursue biological methods of synthesizing PLA. One group that was successful in their efforts was a research group in Korea under Sang Yup Lee that published their findings in the Journal of Biotechnology and Bioengineering in 2009 (Yang et al.). They were able to synthesize PLA from a one-step fermentation process using a glycolic intermediate. They inserted two genes into the E. coli genome: a Clostridum propionicum propionate CoA transferase (denoted PctCp) and a Pseudomonas resinovorans polyhydroxyalkanoate synthase (denoted PhaClPre). Acetyl CoA normally enters the citric acid cycle of glycolysis, but they redirected some of it to produce PLA. The first gene, PctCp, allows the E. coli to produce (D)-lactyl-CoA from acetyl CoA. The second gene, PhaClPre, takes the monomer (D)-lactyl-CoA and creates the PLA homopolymer (see Figure 1). The research group added a plasmid containing both of these genes to the E. coli genome. They performed site-directed mutagenesis on the PhaClPre gene in order to alter the enzyme so that it would accept lactyl-CoA as a substrate. Also, they performed random mutagenesis by error-prone PCR in order to increase the activity of the PctCp gene. Both of these changes allowed them to increase yields of PLA (Yang et al.).
A second paper published at the same time, in the same lab, discusses metabolic engineering of this genetically-engineered strain in order to increase yields of PLA. In the paper, the research group attempted to funnel resources in order to increase yields of PLA. The process of synthesizing the PLA begins with acetyl-CoA, an intermediate in the glycolysis, as mentioned above. They attempted to knock out any competing reactions, such as converting acetyl-CoA to ethanol, as well as preventing reactions that progressed in the opposite direction such as acetate to acetyl-CoA. Furthermore, they attempted to increase transcription of two essential genes in the pathway by replacing the wild type promoter with a more efficient promoter. In a third paper, they sought to increase the chain length of their product. After their efforts, they were able to produce the PLA homopolymer with an efficiency of 7.3 wt% (dry cell weight) (Yang et al. 2011). We believe that we could engineer a strain with a higher efficiency by utilizing multiplex automated genome engineering.
Background: Multiplex automated genome engineering (MAGE)
Multiplex automated genome engineering (MAGE) is a newly-developed technology in small-scale genomic engineering. Genomic engineering allows for parallel, site-specific, effective modification of the DNA, as opposed to traditional genetic engineering techniques, which are slow and inefficient.
MAGE utilizes a specific strain of E. coli called EcNR2. This strain is special because it has the λ Red recombination proteins Exo, Beta, and Gam (Wang HH). The λ Red recombination proteins come from the lambda phage, a virus that infects E. coli. The lambda phage naturally uses these proteins to integrate its DNA into the host E. coli genome. The λ Red beta protein is crucial because it mediates homologous recombination of exogenous single-stranded pieces of DNA. When using MAGE, the DNA strands are delivered into the cell by electroporation, and then the λ Red beta protein promotes annealing of the DNA fragment to the lagging strand of DNA of the replication fork.
As opposed to the traditional technique for recombination, which requires much more homology in DNA fragments, the λ Red system allows short linear DNA with less homology to be incorporated into the DNA strand (Ellis et al.). It has been found that a 30-bp region of homology at either end of a sequence is sufficient to drive recombination of the entire strand.
In addition to having the λ Red recombination proteins, which incorporate the mismatch, this strain is mismatch-repair-gene deficient. This means that when mutations are introduced, the E. coli does not repair it, and thus the mutations are passed on to the daughter cells. This leads to a high efficiency – up to 30% – of mismatches being incorporated in the genome and passed on to offspring (Wang HH).  The mutations come from short single-stranded DNA oligonucleotides that are added to the E. coli cells. The λ Red beta protein helps appeal the oligonucleotide to the lagging strand of the DNA during the replication fork. The oligonucleotides can introduce a mismatch, insertion, or deletion mutation into the genome.
This process can be cycled in order to create an enormous amount of genetic diversity in a short amount of time. Using MAGE, one can simultaneously target many genetic locations to quickly optimize a pathway.  The first pathway MAGE was used to optimize was the 1-deocy-D-xyulose-t-phosphate pathway to create ioprenoid lycopene (Wang HH). Lycopene is an industrially important anti-cancer compound found in tomatoes (Bhuvaneswari and Nagini). In order to optimize the pathway, 24 genes were targeted for mutation. Twenty of the genes were reported to increase lycopene yield, and thus they were targeted to increase efficiency. Four were genes in competing pathways, which were knocked out by introducing nonsense mutations. There were 35 MAGE cycles run, which created as many as 15 billion genetic variants. Since lycopene is a red substance, the cells were screened for their red color and the reddest cells were selected. These cells had their genome sequenced and their biological fitness tested. In just three days, a five-fold increase in lycopene production was observed and the production of lycopene did not negatively affect the fitness of the E. coli (Wang HH).
Retrieved from " http://2013.igem.org/Team:Yale/Project_Overview "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
