"
Page
Discussion
View source
History
teams
Log in
Â 
Team:Yale/Project MAGE
From 2013.igem.org
Return to iGEM main page
Home
Team
Official Team Profile
Project
Parts Submitted to the Registry
Modeling
Notebook
Outreach
Safety
Attributions
Project Overview
Validate PLA Synthesis
Develop Bioassay
Apply Mage
Introduce Export System
Make a Bioplastic
Collaboration
Contents
1 MAGE Targets
1.1 Enzyme Targets
1.1.1 Propionate CoA-transferase
1.1.2 P. resinovorans PHA synthases
2 Pathway Engineering
2.1 Enzyme KOs
2.2 RBS Tuning
3 Results
3.1 Next steps
4 List of Papers
MAGE Targets
The first step in applying MAGE is finding MAGE targets.  This involved reading numerous scientific papers to learn as much as possible about the heterologous enzymes, and the pathway that was being used to create the PLA
Enzyme Targets
Sadly there was no crystal structure of either enzyme we could use to locate the sites to introduce mutations
However, we used the literature available to locate spots where we would want to introduce mutations
Propionate CoA-transferase
According to Selmer et al. 2002, there is a catalytic glutamate found in all CoA-transferases
Using alignment tools along with the crystal structure of YdiF CoA transferase (That shares 45% sequence identity with the Propionate CoA-transferase) we were able to locate the catalytic glutamate in our enzyme
Also, we were able to locate the oxyanion hole of the enzyme
We also targeted the regions mentioned in the  Lee papers
This gave a toal of 4 MAGE oligos ( Sequences Here )
P. resinovorans PHA synthases
From Yang et al. 2011 we knew there were 4 targets for mutation that affected the efficiency of this enzyme (E130, S325, S477, and Q481)
After some more reading we found three amino acids that were putative catalytic residues (C296, D451, and H479)
This gave the impression of one hotspot area (amino acids 477-481) so an oligo was designed to target the enzymes in between (AA478 and AA480)
This gave a total of 10 MAGE oligos ( Sequences Here )
Pathway Engineering
We wanted to divert resources toward our desired pathway
This mainly consisted of increasing the production of lactate
In order to better understand the pathway we were tampering with we created this metabolic engineering graphic (using the sources listed at the bottom of this page)
Enzyme KOs
This led us to find 8 targets for Knockouts (enzymes which would divert resources from our desired pathway)
These include adhE, ackA, pfl(A,B), frd(A,B,C,D, ppc, atoB, pta, eutD
Oligos were designed to introduce two nonsence mutation near the begining of these ezymes, so they would not be expressed
Using Flux Balance Analysis ( instructions here ) we were able to determine that none of these enzymes would cause a fitness hit except ppc
RBS Tuning
Using the pathway diagram shown above we found 20 enzymes that could be potential targets for mutation
Using the same approach taken in MAGE paper (Wang et al. 2009), we designed oligos that target the RBS of each enzyme, and made the oligos degenerate with the following sequence DDRRRRRDDDD (-4 through -14 positions from the start codon)
This would alter the RBS to adjust transcription and thus allow some cells to have higher expression and some cells have lower expression
Using FACS we would be able to sort through the cells and determine which cells have the highest PLA production
The figure shows how to design a oligo depending on which strand and which replichore the gene is located.
Here is a link to the 20 Enzymes targeted for RBS tuning
Here is a link to the sequences of the 28 Oligos for RBS tuning
Results
The first time we used FACS to sort the cells, we saw roughly a two fold increase in fluorescence when we tested on the plate reader after only 1 round of MAGE with the KO oligos
We used this 3 cultures of this FACS sorted strain and ran 5 more MAGE cycles each culture (one culture was MAGEd with the RBS oligos, one with KO, and one with all Oligos)
Here are the results of the FACS sorting of these strains
The first sample is the baseline.  This is EcNR2 with our plasmid containing both the PCT and PHA gene.  This strain was grown overnight with the cells induced and in the presence of Nile red to strain the PLA (this is the same procedure for all later strains as well).  The gate (labeled P2) was chosen to select those with the highest levels of fluorescence.  Due to the abnormally high levels of fluorescence in some cells in the baseline, the gate was chosen to include some baseline cells, but only those with very high levels of fluorescence.
The second sample is the KO oligos.  This means it had a pool of 11 oligos to knock out 11 enzymes.  The enzymes targeted for Knockout were ackA, frdB, frdD pflA, pflB, adhE, frdA, frdC, ATOB, PTA, EUTD.  The chart shows that 26 cells within the P2 gate compared to the 15 in the baseline.  That means number of cells with high levels of fluorescence nearly doubled.
The third sample is the RBS tuning olgios.  These are oligos have degenerate RBS site with the following sequence DDRRRRRDDDD (-4 through -14 positions from the start codon).  The enzymes targeted for RBS tuning are DHA, ACS, ATOB, EUTD, PTA, MAEA, MAEB, PYKF, ENO, PGM, PGK, GAPA, TPIA, FBAA, FBAB, PFKA, PFKB, GLK, PGL, RPOS.  The number of cells within the P2 gate is 65, which is greater than a fourfold increase compared to the baseline.
The fourth sample is the all oligs.  This is a combination of both the RBS oligos and the KO oligos.  This sample has 98 cells within the gate, which is a six and a half fold increase.
Due to the fact the last sample with all the oligos had so many cells within the gate we decided to make a stringent gate.  This gate was moved to only select the top 0.1% of the cells.  This corresponded to 7/10,000.  These cells were collected as well as the cells from the three previous graphs
Next steps
The next step is to grow up these cultures (we sorted out 1,000 cells from each culture to start a culture).
These cells will be tested on the plate reader compared to the wild type EcNR2, as well as the wild type EcNR2 with our plasmid.  The  MAGEd and FACS sorted cells should show an increased levels of fluorescence, due to increased PLA production.  With only 1 MAGE cycles we were able generate a twofold increase, so in theory with an additional 5 MAGE cycles this increase should be even more drastic.
List of Papers
Jacob et al. 1997
Matsuzaki et al. 1998
Sawers et al. 1998
Park et al. 2002
Selmer et al. 2002
Takase et al. 2002
Fong et al. 2005
Matsumoto et al. 2005
Rangarajan ES et al. 2005
Matsumoto et al. 2006
Jung et al. 2009
Matsumoto et al. 2009
Juang et al. 2010
Orth et al. 2010
Yang et al. 2011
Kandasamy et al. 2012
Yang et al. 2013
Retrieved from " http://2013.igem.org/Team:Yale/Project_MAGE "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
