<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Heidelberg_Notebook_Killing_I_Notebook_week6"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:Heidelberg/Notebook/Killing_I/Notebook/week6">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Heidelberg/Notebook/Killing_I/Notebook/week6&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/Notebook/Killing_I/Notebook/week6&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/Notebook/Killing_I/Notebook/week6&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Heidelberg/Notebook/Killing_I/Notebook/week6" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Heidelberg/Notebook/Killing I/Notebook/week6</H1><DIV id="bodyContent"><H3 id="siteSub">From 2008.igem.org</H3><DIV class="MenuBar" id="navi"><UL><LI><A href="https://2008.igem.org/Team:Heidelberg" style="color: white">Home
			</A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Team" style="color: white">Team</A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Team"><SPAN><SPAN>Overview</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Team_Members#Advisors"><SPAN><SPAN>Advisors</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Team_Members#Undergraduates"><SPAN><SPAN>Undergraduates</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Team#There_are_many_Universities_-_and_there_is_Heidelberg_..."><SPAN><SPAN>University</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Team#DKFZ"><SPAN><SPAN>DKFZ</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Team#Bioquant"><SPAN><SPAN>BioQuant</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/BioRegioRN"><SPAN><SPAN>BioRegion Rhein-Neckar</SPAN></SPAN></A></LI></UL><LI><A href="https://2008.igem.org/Team:Heidelberg/Project" style="color: white">Project</A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Project"><SPAN><SPAN>Overall Project</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Notebook/material"><SPAN><SPAN>Material &amp; Methods</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Project/Sensing"><SPAN><SPAN>Sensing</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Project/Killing_I"><SPAN><SPAN>Killing I  - Phages</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Project/Killing_II"><SPAN><SPAN>Killing II - Colicin</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Project/Visualization"><SPAN><SPAN>Visualization</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Parts" style="color: white">Parts</A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Parts"><SPAN><SPAN>Submitted Parts</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Parts/Characterization"><SPAN><SPAN>Characterization</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Modeling" style="color: white">Modeling</A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Modeling"><SPAN><SPAN>Overview</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Modeling/Chemotaxis"><SPAN><SPAN>Chemotaxis-Colicin</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Modeling/Phage"><SPAN><SPAN>Phage Dynamics model</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Notebook/Overview" style="color: white">Notebook</A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Notebook/Sensing_Group/Notebook"><SPAN><SPAN>Sensing &gt;</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Notebook/Sensing_Group/Cloning"><SPAN>Cloning strategy</SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Notebook/Sensing_Group/Notebook"><SPAN>Notebook</SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Notebook/Killing_I/Notebook"><SPAN><SPAN>Killing I - Phages &gt;</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Notebook/Killing_I/Cloning"><SPAN>Cloning strategy</SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Notebook/Killing_I/Notebook"><SPAN>Notebook</SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Notebook/Killing_II/Notebook"><SPAN><SPAN>Killing II - Colicin &gt;</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Notebook/Killing_II/Cloning"><SPAN>Cloning strategy</SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Notebook/Killing_II/Notebook"><SPAN>Notebook</SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Notebook/visualization"><SPAN><SPAN>Visualization</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Notebook/material"><SPAN><SPAN>Material &amp; Methods</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Notebook/team_meetings"><SPAN><SPAN>Team Meetings</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Notebook/seminar"><SPAN><SPAN>Seminar on Synthetic Biology</SPAN></SPAN></A></LI><LI style="width: 160px"><A href="https://2008.igem.org/Team:Heidelberg/Human_Practice/Project_Overview" style="color: white">Human Practice</A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Human_Practice/Project_Overview"><SPAN><SPAN>Project Overview</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Human_Practice/Phips_the_Phage"><SPAN><SPAN>Phips the Phage</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Human_Practice/Essay"><SPAN><SPAN>Essay</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Human_Practice/Surveys"><SPAN><SPAN>Surveys</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Human_Practice/Open_Day"><SPAN><SPAN>Open Day</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Human_Practice/Nobel_Prize"><SPAN><SPAN>Nobel Prize</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Sponsors" style="color: white">Sponsors</A></LI></DIV><TABLE class="wikitable"><TBODY><TR bgcolor="white"><TH height="20px," width="250px"><A href="/Team:Heidelberg/Notebook/Killing_I/Notebook/week5" title="Team:Heidelberg/Notebook/Killing I/Notebook/week5">&lt;&lt; Week 5</A></TH><TH width="500px"><A href="/Team:Heidelberg/Notebook/Killing_I/Notebook" title="Team:Heidelberg/Notebook/Killing I/Notebook">Overview</A></TH><TH width="250px"><A href="/Team:Heidelberg/Notebook/Killing_I/Notebook/week7" title="Team:Heidelberg/Notebook/Killing I/Notebook/week7"> Week 7 &gt;&gt;</A></TH></TR></TBODY></TABLE><P><B>Week 6</B></P><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Monday.2C_09.2F08.2F08"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Monday, 09/08/08</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Phage_cloning_strategy_one"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Phage cloning strategy one</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-3"><A href="#Tuesday.2C_09.2F09.2F08"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Tuesday, 09/09/08</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Phage_cloning_strategy_one_2"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Phage cloning strategy one</SPAN></A></LI><LI class="toclevel-1 tocsection-5"><A href="#Wednesday.2C_09.2F10.2F08"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Wednesday, 09/10/08</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#New_strategy_because_of_sequencing_results"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">New strategy because of sequencing results</SPAN></A></LI><LI class="toclevel-1 tocsection-7"><A href="#Thursday.2C_09.2F11.2F08"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Thursday, 09/11/08</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#Phage_cloning_strategy_one_3"><SPAN class="tocnumber">4.1</SPAN><SPAN class="toctext">Phage cloning strategy one</SPAN></A></LI><LI class="toclevel-1 tocsection-9"><A href="#Friday.2C_09.2F12.2F08"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Friday, 09/12/08</SPAN></A></LI><LI class="toclevel-2 tocsection-10"><A href="#Phage_cloning_strategy_one_4"><SPAN class="tocnumber">5.1</SPAN><SPAN class="toctext">Phage cloning strategy one</SPAN></A></LI></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Monday.2C_09.2F08.2F08">Monday, 09/08/08</SPAN></H2><H3><SPAN class="mw-headline" id="Phage_cloning_strategy_one">Phage cloning strategy one</SPAN></H3><UL><LI> we got colonies on transformation of the 4 fragments in pBluescript
</LI></UL><P>--&gt; only from the ligation at 14°C
</P><UL><LI> inoculation of liquid cultures
</LI><LI> miniprep
</LI><LI> digestion of the minipreps and pBlue with SpeI over night --&gt; it will otherwise get too late to run the gel
</LI></UL><H2><SPAN class="mw-headline" id="Tuesday.2C_09.2F09.2F08">Tuesday, 09/09/08</SPAN></H2><H3><SPAN class="mw-headline" id="Phage_cloning_strategy_one_2">Phage cloning strategy one</SPAN></H3><UL><LI> run digested samples and undigested pBlue on gel
</LI><LI> transform pZeo for another prep --&gt; we first need Zeocin plates since this is the only resistance
</LI><LI> cotransformation of pBlue+insert with pUB307 to generate new donor bacteria for conjugation tests 
</LI></UL><UL><LI>gel:
</LI></UL><UL><LI> lane 0: DNA ladder mix
</LI><LI> lane 1-10: digested samples with SpeI (if correct: 4583/2068)
</LI><LI> lane 11: pBluescript diegested with SpeI (2961bp)
</LI><LI> lane 12: pBluescript undigested
</LI></UL><UL><LI> PCR of all 10 Minipreps of pBlue::lambda-Insert
</LI><LI> PCR without Phusion master mix:
</LI></UL><PRE>Phusion: 0.5µl
5x HF buffer: 10µl
dNTP: 1µl
Primer1 (oriT fw): 2µl
Primer2 (GAM rv): 2µl
water: up to 50 µl (33.5µl)
DNA template: 1µl (~20ng)
</PRE><PRE>PCR protocol
98°C 1min
98°C 30s      |
55°C 30s      | 25x
72°C 2.5min   |
72°C 5min
</PRE><UL><LI> --&gt; elongation time changed up to 2.5 min (whole fragment has 3.75kb)
</LI></UL><UL><LI>Gel
</LI></UL><UL><LI> lane0: 1kb plus DNA ladder
</LI><LI> lane1-10: PCR of insert 
</LI><LI> lane11: positive control: amplification of GAM (1,7kb)--&gt; matches
</LI></UL><UL><LI>result of gel: 
</LI><LI> lower band of samples has nearly 3kb 
</LI><LI> upper band of samples has nearly 7kb
</LI><LI> PCR worked since we got the right product from GAM
</LI></UL><UL><LI> we send sample 4 for sequencing
</LI><LI> primer: M13 FP, M13 RP (both primers from GATC that bind outside the MCS of the pBluescript)
</LI><LI> primer: GFP_fw, GFP_rv : two primers of us that should bind in the middle of the insert
</LI></UL><UL><LI> contransformation of pBlue::lambda-Insert and pUB307
</LI><LI> inoculate pBlue::lambda-Insert culture for transformation test
</LI><LI> each 2µl in 100 µl top10
</LI><LI> two aliquots from Miniprep 4 and 6
</LI></UL><LI> transformation of Zeomycin
</LI><LI>1µl in 100µl Top10; plate: 1:2000 zeocin
</LI><LI> transformation of 4x-Ligation from Miniprep#6
</LI><LI> 2µl in 100 µl Top10
</LI><LI> inoculate culture as aceptor for transformation test
</LI><UL><LI>gel
</LI></UL><UL><LI> lane 1: 1kb plus DNA ladder
</LI><LI> lane 2 - 7:  lambda DNA digested with XbaI - XhoI
</LI><LI> lane 8: 4xLigation: Miniprep 6
</LI><LI> the big lambda fragments have been cut out and wait in the freezer for their gel extraction
</LI><LI> result of ligation: there is only one fragment but it seems to have about 9kb (same size as small fragment of lambda digestion) --&gt;this is a lot too big, so it could be that concatemers formed
</LI></UL><UL><LI>new PCRs: all with Miniprep4 as template
</LI><LI> primer: oriT_fw / GFP_rv
</LI><LI> primer: GFP_fw / GAM_rv
</LI><LI> primer: oriT_fw / GAM_rv
</LI><LI> annealing temperature and time stayed the same in all runs
</LI><LI> elongation time was 2.5 minutes for the first two samples and 4 minutes for the last sample
</LI></UL><UL><LI>Gel
</LI></UL><UL><LI>lane0: DNA ladder mix(the upper bright band is 3kb, the one above 4kb, the two lower are 2kb and 2.5kb)
</LI><LI>lane1: oirT_fw/GFP_rv (2180bp)
</LI><LI>lane2: GFP_fw/GAM_rv (3350bp)
</LI><LI>lane3: oriT_fw/GAM_rv (3850bp)
</LI></UL><UL><LI>Gel
</LI></UL><UL><LI>lane0: DNA ladder mix
</LI><LI>lane1: pBlue + insert digested with AgeI
</LI><LI>lane2: pBlue + insert digested with XbaI
</LI><LI>lane3: pBlue + insert digested with XbaI/XhoI
</LI><LI>lane4: gel extraction of lambda digestion
</LI></UL><UL><LI>results of gel:
</LI><LI> digestion with AgeI leads to a linearised vector with the right size (6.75 kb)
</LI><LI> digestion with XbaI/XhoI leads to the correct pattern (2.9; 2.6; 0.73; 0.5 kb and 60 bp (not visible))
</LI><LI> digestion with XbaI alone leads to a wrong digestion pattern
</LI></UL><H2><SPAN class="mw-headline" id="Wednesday.2C_09.2F10.2F08">Wednesday, 09/10/08</SPAN></H2><UL><LI> I changed the vector according to our sequencing results: we use pBluescript SK instead of pBluescript KS!
</LI></UL><UL><LI> we got the sequencing results from the GFP primer from gatc and now have fully sequenced the fragment
</LI></UL><H3><SPAN class="mw-headline" id="New_strategy_because_of_sequencing_results">New strategy because of sequencing results</SPAN></H3><P>The sequence we got from our lambda insert we ligated shows a repeating sequence in the GFP/Cm-Gene. This sequence contains a XbaI and a BamHI restriction site and is similar to the sequence of the primer we used to amplify this part via PCR.
Therefore we got two additional XbaI restriction sites at bp 1895 and 1954. Together with the XbaI site between OriT and GFP they make it impossible to digest our fragment with XbaI and XhoI and ligate it with the lambda vector. Furthermore, the CmR-protein, which is produced is longer than the natural protein, but it seems to be functional, because we selected the bacteria on Cm.
So we set up two strategies to cope with this problem:
</P><OL><LI> We will digest the pBluescript with the lambda insert with NotI and XhoI, extract the whole insert from the gel and then digest with KpnI to get two fragments: One containing the OriT and the other the GFP/CmR/GAM. The GFP/CmR/GAM fragment has already the proper sticky ends for ligation. the OriT fragment will further be digested with XbaI. Then the two fragments will be ligated with the lambda vector. 
</LI><LI> we will get rid of the three additional XbaI restriction sites via PCR mutagenesis: 
</LI><LI> first we will mutate the XbaI sequence between OriT and GFP (primer sequence: ctgaggggacggtaccTCTACAtttacagctagctcag)
</LI><LI> then we will cut the plasmid with BamHI to delete another XbaI restriction site. By doing this, we shift the reading frame of the CmR protein. By another mutagenesis we mutate the remaining XbaI site to a stop codon since through the digestion with BamHI we lost the stop codon.(primer sequence: ggcaggcggggcgtaaTCTATAggatccggctaataaagg)
</LI><LI> we finally will have a CmR that is 2 aminoacids longer than the natural one (aminoacid sequence of the last aa of the natural one: Gln Gly Gly Ala STOP; our mutated one: Gln Ala Gly Arg Asn Leu STOP). so the last 3 aa are changed into 5 different ones. All 216 aa before are the same. What we have now (a CmR that is functional) has the same mutated aa sequence as we will get through this mutagenesis and only 17 further aa at the end. So if we cut off these 17 additional aa we hope to get a still functional CmR.
</LI></OL><UL><LI> CmR (pDEST15):
</LI></UL><P>Met E K K I T G Y T T V D I S Q W H R K E H F E A F Q S V A Q C T Y N Q T V Q L D I T A F L K T V K K N K H K F Y P A F I H I L A R L Met N A H P E F R Met A Met K D G E L V I W D S V H P C Y T V F H E Q T E T F S S L W S E Y H D D F R Q F L H I Y S Q D V A C Y G E N L A Y F P K G F I E N Met F F V S A N P W V S F T S F D L N V A N Met D N F F A P V F T Met G K Y Y T Q G D K V L Met P L A I Q V H H A V C D G F H V G R Met L N E L Q Q Y C D E W Q G G A Stop
</P><UL><LI> CmR (what we now have):
</LI></UL><P>Met E K K I T G Y T T V D I S Q W H R K E H F E A F Q S V A Q C T Y N Q T V Q L D I T A F L K T V K K N K H K F Y P A F I H I L A R L Met N A H P E F R Met A Met K D G E L V I W D S V H P C Y T V F H E Q T E T F S S L W S E Y H D D F R Q F L H I Y S Q D V A C Y G E N L A Y F P K G F I E N Met F F V S A N P W V S F T S F D L N V A N Met D N F F A P V F T Met G K Y Y T Q G D K V L Met P L A I Q V H H A V C D G F H V G R Met L N E L Q Q Y C D E W Q A G R N L E D P A N K G E P T A Met S G R R G V I Stop
</P><UL><LI> CmR (what we will get with our strategy):
</LI></UL><P>Met E K K I T G Y T T V D I S Q W H R K E H F E A F Q S V A Q C T Y N Q T V Q L D I T A F L K T V K K N K H K F Y P A F I H I L A R L Met N A H P E F R Met A Met K D G E L V I W D S V H P C Y T V F H E Q T E T F S S L W S E Y H D D F R Q F L H I Y S Q D V A C Y G E N L A Y F P K G F I E N Met F F V S A N P W V S F T S F D L N V A N Met D N F F A P V F T Met G K Y Y T Q G D K V L Met P L A I Q V H H A V C D G F H V G R Met L N E L Q Q Y C D E W Q A G R N L Stop
</P><H2><SPAN class="mw-headline" id="Thursday.2C_09.2F11.2F08">Thursday, 09/11/08</SPAN></H2><H3><SPAN class="mw-headline" id="Phage_cloning_strategy_one_3">Phage cloning strategy one</SPAN></H3><UL><LI> lane 0: DNA ladder mix
</LI><LI> lane 1: oriT/GAM (HF buffer)(3.85 kb)
</LI><LI> lane 2: oriT/GAM (GC buffer)
</LI><LI> lane 3: GFP/GAM (HF buffer) (3.3 kb)
</LI><LI> lane 4: GFP/GAM (GC buffer)
</LI></UL><LI>results: so for whatever reason our amplified fragments seem to be too little. 
</LI><UL><LI> explanation: after digestion of GAM with XhoI and AgeI and ligation of this part with the other parts into the vector we have lost the primer binding sites for the GAM primer. So actually in the PCRs we have made where the sizes seemed to be too small we had only one binding primer!!!!
</LI></UL><P>nanodrop results for gelextraction of big lambda fragment
</P><OL><LI> extraction with kit for up to 10 kb: 15,4 ng/µl, 260/280: 1,97
</LI><LI> extraction with kit for over 10 kb: 100 ng/µl, 260/280: 1,57
</LI></OL><UL><LI>cotransformation of PUB307 and pBlue with the lambda insert again did not work. we tried different ratios of pUB and pBlue and also TOP10, MG1655 and DH5a.
</LI></UL><H2><SPAN class="mw-headline" id="Friday.2C_09.2F12.2F08">Friday, 09/12/08</SPAN></H2><H3><SPAN class="mw-headline" id="Phage_cloning_strategy_one_4">Phage cloning strategy one</SPAN></H3><UL><LI> Maxiprep of lambda insert (Mini 4), nanodrop result: 2593 ng/µl, 260/280: 1,93
</LI></UL><UL><LI> digestion of pBlue with lambda insert:
</LI><LI> triple digestion with NotI, XhoI and KpnI in Neb2 with BSA
</LI><LI> double digestion with NotI and XhoI in Neb3 with BSA
</LI></UL><UL><LI>gel for cutting out our fragments
</LI></UL><UL><LI>lane0: DNA ladder mix
</LI><LI>lane1: NotI/XhoI
</LI><LI>lane2: NotI/XhoI
</LI><LI>lane3: NotI/XhoI/KpnI
</LI><LI>lane4: NotI/XhoI/KpnI
</LI><LI>lane5: undigested
</LI></UL><UL><LI>digestion of the oriT (500bp fragment from lane3 and 4) with XbaI
</LI><LI>digestion of the insert (3.85kb fragment from lane1 and 2) with KpnI
</LI><LI>ligation of lambda + oriT + insert (3.3kb from lane 3 and 4)
</LI><LI>transformation --&gt; incubation at 28°C (to make sure that lambda does not get lytic)
</LI></UL><TABLE class="wikitable"><TBODY><TR bgcolor="white"><TH height="20px," width="250px"><A href="/Team:Heidelberg/Notebook/Killing_I/Notebook/week5" title="Team:Heidelberg/Notebook/Killing I/Notebook/week5">&lt;&lt; Week 5</A></TH><TH width="500px"><A href="/Team:Heidelberg/Notebook/Killing_I/Notebook" title="Team:Heidelberg/Notebook/Killing I/Notebook">Overview</A></TH><TH width="250px"><A href="/Team:Heidelberg/Notebook/Killing_I/Notebook/week7" title="Team:Heidelberg/Notebook/Killing I/Notebook/week7"> Week 7 &gt;&gt;</A></TH></TR></TBODY></TABLE><DIV class="printfooter">
Retrieved from &quot;<A href="http://2008.igem.org/Team:Heidelberg/Notebook/Killing_I/Notebook/week6">http://2008.igem.org/Team:Heidelberg/Notebook/Killing_I/Notebook/week6</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Heidelberg/Notebook/Killing_I/Notebook/week6" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Heidelberg/Notebook/Killing_I/Notebook/week6" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Heidelberg/Notebook/Killing_I/Notebook/week6&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Heidelberg/Notebook/Killing_I/Notebook/week6&amp;oldid=88308" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2008.igem.org:Privacy_policy" title="2008.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2008.igem.org:General_disclaimer" title="2008.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>