<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Heidelberg_Notebook_Killing_II_10thweek"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:Heidelberg/Notebook/Killing_II/10thweek">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Heidelberg/Notebook/Killing_II/10thweek&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/Notebook/Killing_II/10thweek&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/Notebook/Killing_II/10thweek&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Heidelberg/Notebook/Killing_II/10thweek" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Heidelberg/Notebook/Killing II/10thweek</H1><DIV id="bodyContent"><H3 id="siteSub">From 2008.igem.org</H3><DIV class="MenuBar" id="navi"><UL><LI><A href="https://2008.igem.org/Team:Heidelberg" style="color: white">Home
			</A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Team" style="color: white">Team</A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Team"><SPAN><SPAN>Overview</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Team_Members#Advisors"><SPAN><SPAN>Advisors</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Team_Members#Undergraduates"><SPAN><SPAN>Undergraduates</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Team#There_are_many_Universities_-_and_there_is_Heidelberg_..."><SPAN><SPAN>University</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Team#DKFZ"><SPAN><SPAN>DKFZ</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Team#Bioquant"><SPAN><SPAN>BioQuant</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/BioRegioRN"><SPAN><SPAN>BioRegion Rhein-Neckar</SPAN></SPAN></A></LI></UL><LI><A href="https://2008.igem.org/Team:Heidelberg/Project" style="color: white">Project</A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Project"><SPAN><SPAN>Overall Project</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Notebook/material"><SPAN><SPAN>Material &amp; Methods</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Project/Sensing"><SPAN><SPAN>Sensing</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Project/Killing_I"><SPAN><SPAN>Killing I  - Phages</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Project/Killing_II"><SPAN><SPAN>Killing II - Colicin</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Project/Visualization"><SPAN><SPAN>Visualization</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Parts" style="color: white">Parts</A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Parts"><SPAN><SPAN>Submitted Parts</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Parts/Characterization"><SPAN><SPAN>Characterization</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Modeling" style="color: white">Modeling</A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Modeling"><SPAN><SPAN>Overview</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Modeling/Chemotaxis"><SPAN><SPAN>Chemotaxis-Colicin</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Modeling/Phage"><SPAN><SPAN>Phage Dynamics model</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Notebook/Overview" style="color: white">Notebook</A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Notebook/Sensing_Group/Notebook"><SPAN><SPAN>Sensing &gt;</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Notebook/Sensing_Group/Cloning"><SPAN>Cloning strategy</SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Notebook/Sensing_Group/Notebook"><SPAN>Notebook</SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Notebook/Killing_I/Notebook"><SPAN><SPAN>Killing I - Phages &gt;</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Notebook/Killing_I/Cloning"><SPAN>Cloning strategy</SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Notebook/Killing_I/Notebook"><SPAN>Notebook</SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Notebook/Killing_II/Notebook"><SPAN><SPAN>Killing II - Colicin &gt;</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Notebook/Killing_II/Cloning"><SPAN>Cloning strategy</SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Notebook/Killing_II/Notebook"><SPAN>Notebook</SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Notebook/visualization"><SPAN><SPAN>Visualization</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Notebook/material"><SPAN><SPAN>Material &amp; Methods</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Notebook/team_meetings"><SPAN><SPAN>Team Meetings</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Notebook/seminar"><SPAN><SPAN>Seminar on Synthetic Biology</SPAN></SPAN></A></LI><LI style="width: 160px"><A href="https://2008.igem.org/Team:Heidelberg/Human_Practice/Project_Overview" style="color: white">Human Practice</A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Human_Practice/Project_Overview"><SPAN><SPAN>Project Overview</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Human_Practice/Phips_the_Phage"><SPAN><SPAN>Phips the Phage</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Human_Practice/Essay"><SPAN><SPAN>Essay</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Human_Practice/Surveys"><SPAN><SPAN>Surveys</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Human_Practice/Open_Day"><SPAN><SPAN>Open Day</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Human_Practice/Nobel_Prize"><SPAN><SPAN>Nobel Prize</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Sponsors" style="color: white">Sponsors</A></LI></DIV><P><B>10<SUP>th</SUP> week</B></P><P><A href="/Team:Heidelberg/Notebook/Killing_II/9thweek" title="Team:Heidelberg/Notebook/Killing II/9thweek"> go back to 9<SUP>th</SUP> week</A></P><P><A href="/Team:Heidelberg/Notebook/Killing_II/Notebook" title="Team:Heidelberg/Notebook/Killing II/Notebook">go back to the overview</A></P><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Monday_10.2F06.2F2008"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Monday 10/06/2008</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#pSB1A2-Receiver-Colicin_cloning"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">pSB1A2-Receiver-Colicin cloning</SPAN></A></LI><LI class="toclevel-3 tocsection-3"><A href="#Colicin_Activity_Test"><SPAN class="tocnumber">1.1.1</SPAN><SPAN class="toctext">Colicin Activity Test</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-4"><A href="#Tuesday_10.2F07.2F2008"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Tuesday 10/07/2008</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#pSB1A2-Receiver-Colicin_cloning_2"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">pSB1A2-Receiver-Colicin cloning</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Sender_cloning:_constitutive_promotor-sender"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">Sender cloning: constitutive promotor-sender</SPAN></A></LI><LI class="toclevel-3 tocsection-7"><A href="#Sender-activity-test"><SPAN class="tocnumber">2.2.1</SPAN><SPAN class="toctext">Sender-activity-test</SPAN></A></LI><LI class="toclevel-1 tocsection-8"><A href="#Wednesday_10.2F08.2F2008"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Wednesday 10/08/2008</SPAN></A></LI><LI class="toclevel-2 tocsection-9"><A href="#pSB1A2-Receiver-Colicin_cloning_3"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">pSB1A2-Receiver-Colicin cloning</SPAN></A></LI><LI class="toclevel-3 tocsection-10"><A href="#Standardization_of_pSB1A2-colE9lys-Receiver"><SPAN class="tocnumber">3.1.1</SPAN><SPAN class="toctext">Standardization of pSB1A2-colE9lys-Receiver</SPAN></A></LI><LI class="toclevel-3 tocsection-11"><A href="#Colicin_Acitivity_test"><SPAN class="tocnumber">3.1.2</SPAN><SPAN class="toctext">Colicin Acitivity test</SPAN></A></LI><LI class="toclevel-2 tocsection-12"><A href="#Sender_cloning:_constitutive_promotor-sender_2"><SPAN class="tocnumber">3.2</SPAN><SPAN class="toctext">Sender cloning: constitutive promotor-sender</SPAN></A></LI><LI class="toclevel-3 tocsection-13"><A href="#Sender-activity-test_2"><SPAN class="tocnumber">3.2.1</SPAN><SPAN class="toctext">Sender-activity-test</SPAN></A></LI><LI class="toclevel-1 tocsection-14"><A href="#Thursday_10.2F09.2F2008"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Thursday 10/09/2008</SPAN></A></LI><LI class="toclevel-2 tocsection-15"><A href="#pSB1A2-Receiver-Colicin_cloning_4"><SPAN class="tocnumber">4.1</SPAN><SPAN class="toctext">pSB1A2-Receiver-Colicin cloning</SPAN></A></LI><LI class="toclevel-3 tocsection-16"><A href="#Mutagenesis_of_pSB1A2-Receiver-Colicin_E1.2FE9"><SPAN class="tocnumber">4.1.1</SPAN><SPAN class="toctext">Mutagenesis of pSB1A2-Receiver-Colicin E1/E9</SPAN></A></LI><LI class="toclevel-3 tocsection-17"><A href="#Colicin_Activity_tests"><SPAN class="tocnumber">4.1.2</SPAN><SPAN class="toctext">Colicin Activity tests</SPAN></A></LI><LI class="toclevel-2 tocsection-18"><A href="#Sender_cloning:_constitutive_promotor-sender_3"><SPAN class="tocnumber">4.2</SPAN><SPAN class="toctext">Sender cloning: constitutive promotor-sender</SPAN></A></LI><LI class="toclevel-3 tocsection-19"><A href="#Sender-activity-test_3"><SPAN class="tocnumber">4.2.1</SPAN><SPAN class="toctext">Sender-activity-test</SPAN></A></LI><LI class="toclevel-1 tocsection-20"><A href="#Friday_10.2F10.2F2008"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Friday 10/10/2008</SPAN></A></LI><LI class="toclevel-2 tocsection-21"><A href="#pSB1A2-Receiver-Colicin_cloning_5"><SPAN class="tocnumber">5.1</SPAN><SPAN class="toctext">pSB1A2-Receiver-Colicin cloning</SPAN></A></LI><LI class="toclevel-3 tocsection-22"><A href="#Mutagenesis_of_pSB1A2-Receiver-Colicin_E1.2FE9_2"><SPAN class="tocnumber">5.1.1</SPAN><SPAN class="toctext">Mutagenesis of pSB1A2-Receiver-Colicin E1/E9</SPAN></A></LI><LI class="toclevel-2 tocsection-23"><A href="#Sender_Cloning:_constitutive_promotor_-_sender"><SPAN class="tocnumber">5.2</SPAN><SPAN class="toctext">Sender Cloning: constitutive promotor - sender</SPAN></A></LI><LI class="toclevel-1 tocsection-24"><A href="#Saturday_10.2F11.2F2008"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">Saturday 10/11/2008</SPAN></A></LI><LI class="toclevel-2 tocsection-25"><A href="#Construction_of_reporterplasmid_with_kanamycin_resistance"><SPAN class="tocnumber">6.1</SPAN><SPAN class="toctext">Construction of reporterplasmid with kanamycin resistance</SPAN></A></LI><LI class="toclevel-3 tocsection-26"><A href="#mCherry"><SPAN class="tocnumber">6.1.1</SPAN><SPAN class="toctext">mCherry</SPAN></A></LI><LI class="toclevel-3 tocsection-27"><A href="#RFP"><SPAN class="tocnumber">6.1.2</SPAN><SPAN class="toctext">RFP</SPAN></A></LI><LI class="toclevel-2 tocsection-28"><A href="#pSB1A2-Receiver-Colicin_cloning_6"><SPAN class="tocnumber">6.2</SPAN><SPAN class="toctext">pSB1A2-Receiver-Colicin cloning</SPAN></A></LI><LI class="toclevel-3 tocsection-29"><A href="#Mutagenesis_of_pSB1A2-Receiver-Colicin_E1.2FE9_3"><SPAN class="tocnumber">6.2.1</SPAN><SPAN class="toctext">Mutagenesis of pSB1A2-Receiver-Colicin E1/E9</SPAN></A></LI><LI class="toclevel-3 tocsection-30"><A href="#Colicin_E9_operon_-_parts:_Colicin_E9"><SPAN class="tocnumber">6.2.2</SPAN><SPAN class="toctext">Colicin E9 operon - parts: Colicin E9</SPAN></A></LI><LI class="toclevel-2 tocsection-31"><A href="#Sender_Cloning:_constitutive_promotor_-_sender_2"><SPAN class="tocnumber">6.3</SPAN><SPAN class="toctext">Sender Cloning: constitutive promotor - sender</SPAN></A></LI><LI class="toclevel-3 tocsection-32"><A href="#Sender_activity_test"><SPAN class="tocnumber">6.3.1</SPAN><SPAN class="toctext">Sender activity test</SPAN></A></LI><LI class="toclevel-1 tocsection-33"><A href="#Sunday_10.2F12.2F2008"><SPAN class="tocnumber">7</SPAN><SPAN class="toctext">Sunday 10/12/2008</SPAN></A></LI><LI class="toclevel-2 tocsection-34"><A href="#Construction_of_reporterplasmid_with_kanamycin_resistance_2"><SPAN class="tocnumber">7.1</SPAN><SPAN class="toctext">Construction of reporterplasmid with kanamycin resistance</SPAN></A></LI><LI class="toclevel-3 tocsection-35"><A href="#mCherry_2"><SPAN class="tocnumber">7.1.1</SPAN><SPAN class="toctext">mCherry</SPAN></A></LI><LI class="toclevel-3 tocsection-36"><A href="#RFP_2"><SPAN class="tocnumber">7.1.2</SPAN><SPAN class="toctext">RFP</SPAN></A></LI><LI class="toclevel-2 tocsection-37"><A href="#pSB1A2-Receiver-Colicin_cloning_7"><SPAN class="tocnumber">7.2</SPAN><SPAN class="toctext">pSB1A2-Receiver-Colicin cloning</SPAN></A></LI><LI class="toclevel-3 tocsection-38"><A href="#Mutagenesis_of_pSB1A2-Receiver-Colicin_E1"><SPAN class="tocnumber">7.2.1</SPAN><SPAN class="toctext">Mutagenesis of pSB1A2-Receiver-Colicin E1</SPAN></A></LI><LI class="toclevel-3 tocsection-39"><A href="#Standardization_of_pSB1A2-colE9plasmid-Receiver"><SPAN class="tocnumber">7.2.2</SPAN><SPAN class="toctext">Standardization of pSB1A2-colE9plasmid-Receiver</SPAN></A></LI></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Monday_10.2F06.2F2008">Monday 10/06/2008</SPAN></H2><H3><SPAN class="mw-headline" id="pSB1A2-Receiver-Colicin_cloning">pSB1A2-Receiver-Colicin cloning</SPAN></H3><UL><LI>Gel of  E9 lys PCR-Screen: 1% Agarose, 135 V, 35 min </LI><LI>Gel-Results of E9-PCR-Screen: 
</LI><LI>Estimated Bands: ~2150bp
</LI><LI>Estimated Bands visible in many tubes -&gt; Screening of single colonies.
</LI></UL><LI>PCR-Screen of single colonies of pSB1A2-Receiver-Col E9 lys
</LI><PRE>25.0 µl Taq Master Mix (Fermentas)
 2.5 µl T9002_LuxpR_Not_Eco_Xba_G_fw (T<SUB>m</SUB>=80,96°C)
 2.5 µl ColE9_lyProt_rv_A_SpeI (T<SUB>m</SUB>=64,29°C)
20.0 µl H<SUB>2</SUB>O
   1    colony
-------
50.0 µl
</PRE><PRE>program:
95 °C    3 min
95 °C    1 min          |
64 °C    1 min          | 30 cycles
72 °C    1 min 30 sec   |
72 °C    10 min
 4 °C   constant
</PRE><UL><LI>Gel of  E9 lys PCR-Screen: 1% Agarose, 135 V, 35 min </LI><LI>Gel-Results of E9-PCR-Screen: Colonies 1, 2, 3, 4, 7 &amp; 8 positive; Inoculation of 5 ml LB-Amp media with this colonies for miniprep, 37 °C -&gt; ON
</LI></UL><H4><SPAN class="mw-headline" id="Colicin_Activity_Test">Colicin Activity Test</SPAN></H4><UL><LI>Plan for activity tests of Colicin-Receivers:
</LI><LI>Overnight measurement (Monday - Tuesday) in Tecan plate reader with reference promotor cells (BBa_I20260) in combination with pSB1A2-Rec-ColE9 lys lysed supernatant.
</LI><LI>Plate scheme (click for detailed image): </LI></UL><P>[<A href="https://2008.igem.org/Team:Heidelberg/Notebook/Killing_II/10thweek" class="external text" rel="nofollow">back</A>]
</P><H2><SPAN class="mw-headline" id="Tuesday_10.2F07.2F2008">Tuesday 10/07/2008</SPAN></H2><H3><SPAN class="mw-headline" id="pSB1A2-Receiver-Colicin_cloning_2">pSB1A2-Receiver-Colicin cloning</SPAN></H3><UL><LI>Colicin-Activity-Test: Cultures were not grown -&gt; new inoculation
</LI><LI>Miniprep of pSB1A2-Receiver-ColE9lys (BioBrick Standard): colonies 1, 2, 3, 4, 7 &amp; 8, eluted in 35 µl H<SUB>2</SUB>O
</LI><LI>Digestion of pSB1A2-Receiver-ColE9lys (BioBrick Standard): colonies 1, 2, 3, 4, 7 &amp; 8
</LI></UL><PRE>colonies 1 &amp; 8: 
 5.0 µl DNA (~65-80 ng/µl)
25.5 µl H<SUB>2</SUB>O
 4.0 µl BSA 10x (NEB)
 4.0 µl NEBuffer 2 (NEB)
 0.5 µl SpeI (NEB)
 1.0 µl XbaI (NEB)
-------
40.0 µl
</PRE><PRE>colonies 2, 3 &amp; 4: 
 2.0 µl DNA (~245-275 ng/µl)
28.5 µl H<SUB>2</SUB>O
 4.0 µl BSA 10x (NEB)
 4.0 µl NEBuffer 2 (NEB)
 0.5 µl SpeI (NEB)
 1.0 µl XbaI (NEB)
-------
40.0 µl
</PRE><PRE>colony 7: 
 2.0 µl DNA (~125 ng/µl)
28.5 µl H<SUB>2</SUB>O
 4.0 µl BSA 10x (NEB)
 4.0 µl NEBuffer 2 (NEB)
 0.5 µl SpeI (NEB)
 1.0 µl XbaI (NEB)
-------
40.0 µl
</PRE><UL><LI>Gel of Digestion: 1% Agarose, 135 V, 30 min </LI><LI>Digestion results:
</LI><LI>Expected fragments:
</LI><LI>2 x ~2000 (insert &amp; vector), if XbaI site is recovered
</LI><LI>1 x ~4000, if XbaI site is not recovered
</LI></UL><LI>In each colony there is only one fragment. Probably we have no positive clones with the BioBrick standard. To check the sequence we send the probes for sequencing.
</LI><LI>Send probes 1, 2, 3, 4, 7, &amp; 8 to GATC for sequencing: Primers VF2 &amp; VR
</LI><LI>Growth test of each Rec-Colicin glycerol stock: Inoculation in 2 ml LB-Amp and 2 ml TB-Amp. 37 °C -&gt; ON
</LI><H3><SPAN class="mw-headline" id="Sender_cloning:_constitutive_promotor-sender">Sender cloning: constitutive promotor-sender</SPAN></H3><H4><SPAN class="mw-headline" id="Sender-activity-test">Sender-activity-test</SPAN></H4><UL><LI>Inoculation of 15 ml TB-Amp media with sender (J23107 + F1610) from glycerol stock
</LI><LI>Inoculation of 15 ml TB-Amp media with GFP-receiver (<A href="http://partsregistry.org/Part:BBa_T9002" class="external text" rel="nofollow">BBa_T9002</A>)
</LI><LI>Inoculation of 15 ml LB-Amp media with sender (J23107 + F1610) from glycerol stock
</LI><LI>Inoculation of 15 ml LB-Amp media with GFP-receiver (<A href="http://partsregistry.org/Part:BBa_T9002" class="external text" rel="nofollow">BBa_T9002</A>)
</LI></UL><UL><LI>Plan for Wednesdaynesday:
</LI><LI>Inoculation of 7 ml TB media with 150 µl from sender ONC
</LI><LI>Inoculation of 7 ml TB media with 7 ml from GFP-receiver ONC
</LI><LI>Count amount of cells from ONC dilution t = 0 h (counting chamber)
</LI><LI>Measurement of optical density (OD) of ONC dilution t = 0 h
</LI><LI>Measurement of OD of one probe half-hourly for 8 - 10 h
</LI><LI>Creation of supernatant the measured probe by centrifugation and filtration half-hourly for 8 - 10 h
</LI><LI>Storing supernatant at 4 °C
</LI><LI>Count amount of cells from last probe (counting chamber)
</LI><LI>Start overnight measurement in Tecan plate reader
</LI><LI>plate scheme (click for detailed image): </LI></UL><P>[<A href="https://2008.igem.org/Team:Heidelberg/Notebook/Killing_II/10thweek" class="external text" rel="nofollow">back</A>]
</P><H2><SPAN class="mw-headline" id="Wednesday_10.2F08.2F2008">Wednesday 10/08/2008</SPAN></H2><H3><SPAN class="mw-headline" id="pSB1A2-Receiver-Colicin_cloning_3">pSB1A2-Receiver-Colicin cloning</SPAN></H3><H4><SPAN class="mw-headline" id="Standardization_of_pSB1A2-colE9lys-Receiver">Standardization of pSB1A2-colE9lys-Receiver</SPAN></H4><UL><LI>Sequencing results: Colonies 1, 2, 3, 4 &amp; 8 have the right sequence. Our first colicin Receiver fullfills BioBrick standard and is ready for activity tests. Colony 7 has mistakes inside the receiver and is thrown away.
</LI><LI>Digestion of pSB1A2-Receiver-ColE9lys colony 2 with XbaI/SpeI, XbaI, SpeI, EcoRI/PstI:
</LI></UL><PRE>'XbaI/SpeI:'
 5.0 µl DNA (~240 ng/µl)
 5.0 µl NEBuffer 2 (NEB)
 5.0 µl BSA 10x (NEB)
 0.5 µl XbaI (20 000 Units/ml, NEB)
 1.0 µl SpeI (10 000 Units/ml, NEB)
33.5 µl H<SUB>2</SUB>O
-------
50.0 µl

'XbaI:'
 5.0 µl DNA (~240 ng/µl)
 5.0 µl NEBuffer 2 (NEB)
 5.0 µl BSA 10x (NEB)
 0.5 µl XbaI (20 000 Units/ml, NEB)
34.5 µl H<SUB>2</SUB>O
-------
50.0 µl
</PRE><PRE>'SpeI:'
 5.0 µl DNA (~240 ng/µl)
 5.0 µl NEBuffer 2 (NEB)
 5.0 µl BSA 10x (NEB)
 1.0 µl SpeI (10 000 Units/ml, NEB)
34.0 µl H<SUB>2</SUB>O
-------
50.0 µl
</PRE><PRE>'EcoRI/PstI:'
 5.0 µl DNA (~240 ng/µl)
 5.0 µl NEBuffer 2 (NEB)
 5.0 µl BSA 10x (NEB)
 0.5 µl EcoRI (20 000 Units/ml, NEB)
 0.5 µl PstI (20 000 Units/ml, NEB)
34.0 µl H<SUB>2</SUB>O
-------
50.0 µl
</PRE><H4><SPAN class="mw-headline" id="Colicin_Acitivity_test">Colicin Acitivity test</SPAN></H4><UL><LI>Inoculation of ONC for colicin activity test on Thursday, 2008-10-09:
</LI><LI>10 ml TB-Kana with reference promoter cells from glycerolstocks (<A href="http://partsregistry.org/Part:BBa_I20260" class="external text" rel="nofollow">BBa_I20260</A>)
</LI><LI>10 ml TB-Amp with pSB1A2-Receiver, pSB1A2-Receiver-ColE9lys, pSB1A2-Receiver-ColE1 from Glycerolstocks
</LI></UL><H3><SPAN class="mw-headline" id="Sender_cloning:_constitutive_promotor-sender_2">Sender cloning: constitutive promotor-sender</SPAN></H3><H4><SPAN class="mw-headline" id="Sender-activity-test_2">Sender-activity-test</SPAN></H4><UL><LI>Inoculation of 7 ml TB media with 150 µl from sender ONC
</LI><LI>Inoculation of 7 ml TB media with 7 ml from GFP-receiver ONC
</LI><LI>Measurement of optical density (OD) of ONC dilution t = 0 h
</LI><LI>Measurement of OD of one probe half-hourly for 7 h
</LI><LI>Creation of supernatant the measured probe by centrifugation and filtration half-hourly for 8 - 10 h
</LI><LI>Storing supernatant at 4 °C
</LI><LI>Start overnight measurement in Tecan plate reader
</LI><LI>plate scheme (click for detailed image): [[Image: 081007-pipetting_scheme_sender_test.jpg | 800 px | center ]
</LI></UL><P>[<A href="https://2008.igem.org/Team:Heidelberg/Notebook/Killing_II/10thweek" class="external text" rel="nofollow">back</A>]
</P><H2><SPAN class="mw-headline" id="Thursday_10.2F09.2F2008">Thursday 10/09/2008</SPAN></H2><H3><SPAN class="mw-headline" id="pSB1A2-Receiver-Colicin_cloning_4">pSB1A2-Receiver-Colicin cloning</SPAN></H3><H4><SPAN class="mw-headline" id="Mutagenesis_of_pSB1A2-Receiver-Colicin_E1.2FE9">Mutagenesis of pSB1A2-Receiver-Colicin E1/E9</SPAN></H4><UL><LI>Mutagenesis PCR:
</LI></UL><PRE>5.0  µl <I>pfu</I> buffer
1.0  µl forward primer (10 µM)
1.0  µl reverse primer (10 µM)
1.0  µl dNTPs (12.5 mM, 50x)
0.5  µl template DNA - 5 to 50 ng plasmid DNA
40.5 µl milliQ water
1.0  µl cloned <I>pfu</I> polymerase (Stratagene)
</PRE><PRE>95 °C   30 sec
95 °C   30 sec    |
55 °C   60 sec    | 16 cycles
68 °C   11 min    |
 4 °C   constant
</PRE><UL><LI>Digestion with DpnI (NEB) to cut parental plasmid: Added 1 µl of DpnI to the finished Mutagenesis PCR. DpnI cuts at methylated GATC sites, so that only parental plasmids without the mutation are cutted. 2 h -&gt; 37 °C
</LI><LI>PCR Purification Kit (Qiagen) to purify the plasmids.
</LI><LI>Transformation of prior mutagenesis: 5 µl per 100 µl <I>E. coli</I> TOP 10 competent cells
</LI></UL><PRE>- start thawing 100 µl competent <I>E. coli</I> TOP 10 cells on crushed ice
- add 5 µl ligation
- incubate on ice for 20 minutes
- heat shock at 42 °C for 45 sec
- incubate 2 min on ice
- add 400 µl preheated LB media
- incubate at 37 °C for 1 h, 500 rpm
- Plate 200 µl on preheated LB-Amp plate
- incubate overnight
</PRE><UL><LI>Controlgel of Mutagenesis: 1 % Agarose, 135 V, 30 min; DNA is visible -&gt; probably the mutagenesis worked. </LI></UL><H4><SPAN class="mw-headline" id="Colicin_Activity_tests">Colicin Activity tests</SPAN></H4><UL><LI>10 am: Inocullation of TB media from ONC
</LI><LI>50 ml TB-Kana with 1.4 ml ONC of reference promoter cells (<A href="http://partsregistry.org/Part:BBa_I20260" class="external text" rel="nofollow">BBa_I20260</A>)
</LI><LI>3 x 50 ml TB-Amp with 1.4 ml ONC of pSB1A2-Receiver, pSB1A2-Receiver-ColE9lys, pSB1A2-Receiver-ColE1
</LI></UL><LI>12.30 am: OD -&gt; 0.320. Separation of the 3 Receiver cultures into 8 tubes with 3 ml. Activation with 8 different Autoinducer(<A href="http://www.sigmaaldrich.com/catalog/search/ProductDetail/SIGMA/K3007" class="external text" rel="nofollow">detailed information, Sigma</A>) concentrations: 0 nM, 1 nM, 5 nM, 10 nM, 25 nM, 50 nM, 75 nM &amp; 100 nM.
</LI><LI>8.00 pm: Preparing the test
</LI><LI>plate scheme: </LI><LI>Preparation of probes: </LI><LI>10 pm start of measurement
</LI><H3><SPAN class="mw-headline" id="Sender_cloning:_constitutive_promotor-sender_3">Sender cloning: constitutive promotor-sender</SPAN></H3><H4><SPAN class="mw-headline" id="Sender-activity-test_3">Sender-activity-test</SPAN></H4><UL><LI>Results of ON measurements: The graph shows the GFP intensities over time for the different supernatants and one AI-1 concentration (c = 1 nM). The GFP induction is much lower for the supernatants in comparison to the AI-1. Other tests with lower AI-1 concentration have to be performed to determine the AI-1 production rate. </LI></UL><P>[<A href="https://2008.igem.org/Team:Heidelberg/Notebook/Killing_II/10thweek" class="external text" rel="nofollow">back</A>]
</P><H2><SPAN class="mw-headline" id="Friday_10.2F10.2F2008">Friday 10/10/2008</SPAN></H2><H3><SPAN class="mw-headline" id="pSB1A2-Receiver-Colicin_cloning_5">pSB1A2-Receiver-Colicin cloning</SPAN></H3><H4><SPAN class="mw-headline" id="Mutagenesis_of_pSB1A2-Receiver-Colicin_E1.2FE9_2">Mutagenesis of pSB1A2-Receiver-Colicin E1/E9</SPAN></H4><UL><LI>Inoculation of liquid cultures from transformation of the first mutagenesis.
</LI><LI>Miniprep of liquidcultures: eluted in 35 µl H<SUB>2</SUB>O (Qiagen Miniprepkit)
</LI><LI>2nd mutagenesis PCR of Colicin E1 - Receiver (mutation of first PstI site):
</LI><LI>Mutagenesis PCR:
</LI></UL><PRE>5.0  µl <I>pfu</I> buffer
1.0  µl forward primer (10 µM)
1.0  µl reverse primer (10 µM)
1.0  µl dNTPs (12.5 mM, 50x)
0.5  µl template DNA - 5 to 50 ng plasmid DNA
40.5 µl milliQ water
1.0  µl Turbo <I>pfu</I> polymerase (Stratagene)
</PRE><PRE>95 °C   30 sec
95 °C   30 sec    |
55 °C   60 sec    | 16 cycles
68 °C   11 min    |
 4 °C   constant
</PRE><H3><SPAN class="mw-headline" id="Sender_Cloning:_constitutive_promotor_-_sender">Sender Cloning: constitutive promotor - sender</SPAN></H3><P>[<A href="https://2008.igem.org/Team:Heidelberg/Notebook/Killing_II/10thweek" class="external text" rel="nofollow">back</A>]
</P><H2><SPAN class="mw-headline" id="Saturday_10.2F11.2F2008">Saturday 10/11/2008</SPAN></H2><H3><SPAN class="mw-headline" id="Construction_of_reporterplasmid_with_kanamycin_resistance">Construction of reporterplasmid with kanamycin resistance</SPAN></H3><H4><SPAN class="mw-headline" id="mCherry">mCherry</SPAN></H4><UL><LI>For the visualization of our killer cells we need a RFP or mCherry reporter plasmid with kanamycin resistance. We decided to clone a cheY-mCherry fusion protein into pBAD18 vector.
</LI><LI>Digestion of pBAD18 plasmid &amp; pES16-CheY-mCherry with SacI/XbaI: 2h 37 °C, 20 min 65 °C
</LI></UL><PRE>20.0 µl DNA
 0.5 µl SacI (20 000 U/ml, 100 % Activity in NEBuffer 4, NEB)
 0.7 µl XbaI (20 000 U/ml, 75 % Activity in NEBuffer 4, NEB)
 5.0 µl BSA 10x
 5.0 µl NEBuffer 4 (NEB)
18.8 µl H<SUB>2</SUB>O
-------
50.0 µl
</PRE><UL><LI>Gel of Digestion: 0.7% Agarose, 135 V, 30 min </LI><LI>Gelresults:
</LI><LI>Expected Fragments:
</LI><LI>pBAD -&gt; ~4600 bp
</LI><LI>CheY-mCherry -&gt; ~1100 bp
</LI></UL><LI>We were able to cut both fragments.
</LI><LI>Gelextraction with Qiagen Kit: eluted in 35 µl H<SUB>2</SUB>O
</LI><LI>Ligation of pBAD &amp; mCherry: 11 h -&gt; 16 °C, 20 min -&gt; 65 °C
</LI><PRE>upper band 
 2.0 µl T4 DNA Ligase Buffer (Fermentas)
 2.0 µl T4 DNA Ligase (Fermentas)
 0.5 µl pBAD (11.4 ng/µl) 
15.5 µl CheY-mCherry (8.3 ng/µl) 
-------
20.0 µl
</PRE><UL><LI>EDIT 10/12/202008: calculated DNA ratio wrong
</LI></UL><H4><SPAN class="mw-headline" id="RFP">RFP</SPAN></H4><UL><LI>In addition we try to clone RFP gene cassette from J23102 into pSB3K3.
</LI><LI>Digestion of BBa_J23102 &amp; BBa_I15030 with EcoRI/PstI: 2 h -&gt; 37 °C, 20 min -&gt; 65 °C
</LI></UL><PRE>20.0 µl DNA
 1.0 µl EcoRI (20 000 U/ml, NEB)
 1.0 µl PstI (10 000 U/ml, NEB)
 5.0 µl BSA 10x
 5.0 µl EcoRI Buffer(NEB)
18.5 µl H<SUB>2</SUB>O
-------
50.0 µl
</PRE><UL><LI>Gel of Digestion: 0.7% Agarose, 135 V, 30 min </LI><LI>Gelresults:
</LI><LI>Expected Fragments:
</LI><LI>J23102 -&gt; ~945 bp
</LI><LI>I15030 -&gt; ~1909 bp
</LI><LI>pSB3K3 -&gt; ~2750 bp
</LI></UL><LI>J23102 was cutted out of the gel. For pSB3K3 we were not sure which was the right fragment so we go on with both fragments.
</LI><LI>Gelextraction with Qiagen Kit: eluted in 35 µl H<SUB>2</SUB>O
</LI><LI>Ligation of J23102 and pSB3K3 (5:1):
</LI><PRE>upper band 
 2.0 µl T4 DNA Ligase Buffer (Fermentas)
 2.0 µl T4 DNA Ligase (Fermentas)
 8.5 µl pSB3K3 (upper Band) 
 7.5 µl BBa_J23102
-------
20.0 µl
</PRE><PRE>lower band 
 2.0 µl T4 DNA Ligase Buffer (Fermentas)
 2.0 µl T4 DNA Ligase (Fermentas)
11.6 µl pSB3K3 (lower band) 
 4.4 µl BBa_J23102
-------
20.0 µl
</PRE><UL><LI>EDIT 10/12/202008: calculated DNA ratio wrong
</LI></UL><H3><SPAN class="mw-headline" id="pSB1A2-Receiver-Colicin_cloning_6">pSB1A2-Receiver-Colicin cloning</SPAN></H3><H4><SPAN class="mw-headline" id="Mutagenesis_of_pSB1A2-Receiver-Colicin_E1.2FE9_3">Mutagenesis of pSB1A2-Receiver-Colicin E1/E9</SPAN></H4><UL><LI>Digestion of overnight PCR probes with DpnI for 2 h -&gt; 37 °C.
</LI><LI>PCR purification of probes: eluted in 35 µl H<SUB>2</SUB>O (Qiagen Kit)
</LI><LI>Transformation of prior mutagenesis: 5 µl per 100 µl <I>E. coli</I> TOP 10 competent cells
</LI></UL><PRE>- start thawing 100 µl competent <I>E. coli</I> TOP 10 cells on crushed ice
- add 5 µl ligation
- incubate on ice for 20 minutes
- heat shock at 42 °C for 45 sec
- incubate 2 min on ice
- add 400 µl preheated LB media
- incubate at 37 °C for 1 h, 500 rpm
- Plate 200 µl on preheated LB-Amp plate
- incubate overnight
</PRE><UL><LI>Controlgel of Mutagenesis: 1 % Agarose, 135 V, 30 min; DNA is visible -&gt; probably the mutagenesis worked. </LI><LI>Controlldigestion of first mutagenesis of Colicin E1 &amp; E9 - Receiver with EcoRI: 2h -&gt; 37 °C
</LI></UL><PRE>10.0 µl DNA
 2.0 µl BSA 10x (NEB)
 0.5 µl EcoRI (NEB)
 2.0 µl Buffer EcoRI 
 5.5 µl H<SUB>2</SUB>O
-------
20.0 µl
</PRE><UL><LI>Gel of Digestion: 1% Agarose, 135 V, 30 min </LI><LI>Gelresults: Mutated Colicin E1 and E9 Receivers were cutted only once by EcoRI. -&gt; Mutagenesis sucessful.
</LI></UL><H4><SPAN class="mw-headline" id="Colicin_E9_operon_-_parts:_Colicin_E9">Colicin E9 operon - parts: Colicin E9</SPAN></H4><UL><LI>Digestion of Colicin E9 PCR product &amp; pSB3K3 (from BBa_I15030)
</LI></UL><PRE>20.0 µl DNA
 1.0 µl EcoRI (20 000 U/ml, NEB)
 1.0 µl PstI (10 000 U/ml, NEB)
 5.0 µl BSA 10x
 5.0 µl EcoRI Buffer(NEB)
18.5 µl H<SUB>2</SUB>O
-------
50.0 µl
</PRE><UL><LI>Gel of Digestion: 0.7% Agarose, 135 V, 30 min </LI><LI>Gelresults:
</LI><LI>Expected Fragments:
</LI><LI>Colicin E9 -&gt; ~620 bp
</LI></UL><LI>Colicin E9 was cutted out of the gel.
</LI><LI>Gelextraction with Qiagen Kit: eluted in 35 µl H<SUB>2</SUB>O
</LI><LI>Ligation of colicin E9 and pSB3K3 (5:1):
</LI><PRE>upper band 
 2.0 µl T4 DNA Ligase Buffer (Fermentas)
 2.0 µl T4 DNA Ligase (Fermentas)
 6.1 µl pSB3K3 (upper Band) 
 9.9 µl Colicin E9
-------
20.0 µl
</PRE><PRE>lower band 
 2.0 µl T4 DNA Ligase Buffer (Fermentas)
 2.0 µl T4 DNA Ligase (Fermentas)
 9.5 µl pSB3K3 (lower band) 
 8.5 µl Colicin E9
-------
20.0 µl
</PRE><P>EDIT 10/12/202008: calculated DNA ratio wrong
</P><H3><SPAN class="mw-headline" id="Sender_Cloning:_constitutive_promotor_-_sender_2">Sender Cloning: constitutive promotor - sender</SPAN></H3><H4><SPAN class="mw-headline" id="Sender_activity_test">Sender activity test</SPAN></H4><UL><LI>Constitutive sender and amplifier (<A href="http://partsregistry.org/Part:BBa_I15030" class="external text" rel="nofollow">BBa_I15030</A>) activity test
</LI><LI>7x Inoculation of 7 ml TB media with 140 µl from sender ONC
</LI><LI>7x Inoculation of 7 ml TB media with 140 µl from amplifier ONC
</LI><LI>Inoculation of 7 ml TB media with 7 ml from GFP-receiver ONC
</LI><LI>Measurement of optical density (OD) of ONC dilutions t = 0 h
</LI><LI>Measurement of OD of one probe hourly for 6 h
</LI><LI>Creation of supernatant of the measured probe by centrifugation and filtration hourly for 6 h
</LI><LI>Storing supernatant at 4 °C
</LI><LI>Count amount of cells from last probe (counting chamber)
</LI><LI>plate scheme: </LI><LI>Unfortunately we were not able to measure ON because the Tecan plate reader was not recognized by the PC.
</LI></UL><P>[<A href="https://2008.igem.org/Team:Heidelberg/Notebook/Killing_II/10thweek" class="external text" rel="nofollow">back</A>]
</P><H2><SPAN class="mw-headline" id="Sunday_10.2F12.2F2008">Sunday 10/12/2008</SPAN></H2><H3><SPAN class="mw-headline" id="Construction_of_reporterplasmid_with_kanamycin_resistance_2">Construction of reporterplasmid with kanamycin resistance</SPAN></H3><H4><SPAN class="mw-headline" id="mCherry_2">mCherry</SPAN></H4><UL><LI>Transformation of pBAD-mCherry Ligation of saturday, 10/11/2008 (wrong vector-insert ratio): 5 µl per 100 µl <I>E. coli</I> TOP 10 competent cells
</LI></UL><PRE>- start thawing 100 µl competent <I>E. coli</I> TOP 10 cells on crushed ice
- add 5 µl ligation
- incubate on ice for 20 minutes
- heat shock at 42 °C for 45 sec
- incubate 2 min on ice
- add 400 µl preheated LB media
- incubate at 37 °C for 1 h, 500 rpm
- Plate 500 µl on preheated LB-Amp plate
- incubate overnight at 37°C
</PRE><UL><LI>Ligation of pBAD and mCherry (1:5): 1 h -&gt; 22 °C, 20 min -&gt; 65 °C
</LI><LI>Reason: Transformation on Saturdayurday was carried out with wrong ratio of vector and insert (calculation error).
</LI></UL><PRE> 2.0 µl T4 DNA Ligase Buffer (Fermentas)
 2.0 µl T4 DNA Ligase (Fermentas)
6.05 µl pBAD (11.4 ng/µl) 
9.95 µl CheY-mCherry (8.3 ng/µl) 
-------
20.0 µl
</PRE><UL><LI>Transformation of pBAD-mCherry Ligation: 5 µl per 100 µl <I>E. coli</I> TOP 10 competent cells
</LI></UL><PRE>- start thawing 100 µl competent <I>E. coli</I> TOP 10 cells on crushed ice
- add 5 µl ligation
- incubate on ice for 20 minutes
- heat shock at 42 °C for 45 sec
- incubate 2 min on ice
- add 400 µl preheated LB media
- incubate at 37 °C for 1 h, 500 rpm
- Plate 500 µl on preheated LB-Amp plate
- incubate overnight at 37°C
</PRE><H4><SPAN class="mw-headline" id="RFP_2">RFP</SPAN></H4><UL><LI>Transformation of J23102 and pSB3K3 Ligation of saturday, 10/11/2008 (wrong vector-insert ratio): 5 µl per 100 µl <I>E. coli</I> TOP 10 competent cells
</LI></UL><PRE>- start thawing 100 µl competent <I>E. coli</I> TOP 10 cells on crushed ice
- add 5 µl ligation
- incubate on ice for 20 minutes
- heat shock at 42 °C for 45 sec
- incubate 2 min on ice
- add 400 µl preheated LB media
- incubate at 37 °C for 1 h, 500 rpm
- Plate 500 µl on preheated LB-Amp plate
- incubate overnight at 37°C
</PRE><H3><SPAN class="mw-headline" id="pSB1A2-Receiver-Colicin_cloning_7">pSB1A2-Receiver-Colicin cloning</SPAN></H3><H4><SPAN class="mw-headline" id="Mutagenesis_of_pSB1A2-Receiver-Colicin_E1">Mutagenesis of pSB1A2-Receiver-Colicin E1</SPAN></H4><UL><LI>Inoculation of Col E1 second mutagenesis step in 10 ml LB-amp; 5 h -&gt; 37 °C
</LI><LI>Colonies 1-8 of Col E1 (5)
</LI><LI>Colonies 10-17 of Col E1 (1)
</LI></UL><LI>Miniprep of liquidcultures (Col E1 (1.14;1.16;5.1;5.7): eluted in 35 µl H2O (Qiagen Miniprepkit)
</LI><LI>Controlldigestion of second mutagenesis of Colicin E1 - Receiver with PstI: 2 h -&gt; 37 °C
</LI><PRE>10.0 µl DNA (Miniprep)
 2.0 µl BSA 10x (NEB)
 0.5 µl PstI(NEB)
 2.0 µl Buffer EcoRI 
 5.5 µl H<SUB>2</SUB>O
-------
20.0 µl
</PRE><UL><LI>Gel of Digestion: 1% Agarose, 135 V, 30 min </LI><LI>Gelresults: It can not be concluded weather the mutation worked. The fragments that are expected with and without the mutated PstI-Site are very similar.
</LI><LI>Third Mutagenesis PCR:
</LI></UL><PRE> 5.0  µl <I>pfu</I> buffer
 1.0  µl forward primer (10 µM Col E1_mut_Pst_2_fw)
 1.0  µl reverse primer (10 µM Col E1_mut_Pst_2_rv)
 1.0  µl dNTPs (12.5 mM, 50x)
 0.5  µl template DNA - 5 to 50 ng plasmid DNA
40.5 µl milliQ water
 1.0  µl turbo <I>pfu</I> polymerase (Stratagene)
 ----
 50.0 µl
</PRE><PRE>95 °C   30 sec
95 °C   30 sec    |
55 °C   60 sec    | 16 cycles
68 °C   11 min    |
 4 °C   constant
</PRE><UL><LI>Digestion with DpnI (NEB) of the third mutagenesis to cut parental plasmid: Added 1 µl of DpnI to the finished Mutagenesis PCR. DpnI cuts at methylated GATC sites, so that only parental plasmids without the mutation are cutted. 2 h 35 min -&gt; 37 °C
</LI><LI>Controlgel of third Mutagenesis (PstI_2): 1 % Agarose, 135 V, 30 min; DNA only visible in products from the colonie 1 (1.14 and 1.16) -&gt; probably the mutagenesis worked on this plasmid. </LI><LI>PCR Purification Kit (Qiagen) to purify the plasmids. Eluated in 35 µl H<SUB>2</SUB>O 
</LI><LI>Transformation of prior mutagenesis: 5 µl per 100 µl <I>E. coli</I> TOP 10 competent cells
</LI></UL><PRE>- start thawing 100 µl competent <I>E. coli</I> TOP 10 cells on crushed ice
- add 5 µl ligation
- incubate on ice for 20 minutes
- heat shock at 42 °C for 45 sec
- incubate 2 min on ice
- add 400 µl preheated LB media
- incubate at 37 °C for 1 h, 500 rpm
- Plate 500 µl on preheated LB-Amp plate
- incubate overnight
</PRE><H4><SPAN class="mw-headline" id="Standardization_of_pSB1A2-colE9plasmid-Receiver">Standardization of pSB1A2-colE9plasmid-Receiver</SPAN></H4><UL><LI>PCR to add right BioBrick prefix and suffix to mutated ColicinE9-Receiver (without EcoRI site, colonies 2 &amp; 4)
</LI></UL><PRE> 2.5 µl Primer fw T9002 (10 µM)
 2.5 µl Primer rv (10 µM)
18.0 µl H<SUB>2</SUB>O
25.0 µl Phusion MasterMix (Finnzymes, NEB)
 2.0 µl Template (ColE9-Receiver mutated)
-------
50.0 µl
</PRE><PRE>98 °C    30 sec
98 °C    10 sec          |
56 °C    30 sec          | 25 cycles
72 °C    1 min 30 sec    |
72 °C    10 min
 4 °C    constant
</PRE><UL><LI>Gel to purify PCR products: 0.7%, 135 V, 30 min </LI><LI>Gelextraction: eluted in 35 µl H<SUB>2</SUB>O (Qiagen, Gelextraction Kit)
</LI><LI>Digestion of ColE9-Receiver PCR product with EcoRI and SpeI: 2h -&gt; 37 °C
</LI></UL><PRE> 6.5 µl H<SUB>2</SUB>O
 5.0 µl BSA 10x (NEB)
 5.0 µl EcoRI Buffer (NEB)
 0.5 µl EcoRI (NEB)
 1.0 µl SpeI (NEB)
32.0 µl DNA
-------
50.0 µl
</PRE><UL><LI>Ligation of ColicinE9-Receiver (BioBrick) with pSB1A2 (3:1): 1h -&gt; 22 °C, 20 min 65 °C
</LI></UL><PRE>ColicinE9-Receiver (4)
 1.1 µl pSB1A2
14.9 µl ColicinE9-Receiver
 2.0 µl T4 DNA Ligase
 2.0 µl T4 DNA Ligase Buffer
-------
20.0 µl
</PRE><PRE>ColicinE9-Receiver (2)
 1.8 µl pSB1A2
14.2 µl ColicinE9-Receiver
 2.0 µl T4 DNA Ligase
 2.0 µl T4 DNA Ligase Buffer
-------
20.0 µl
</PRE><UL><LI>Transformation of prior ligation: 5 µl per 100 µl <I>E. coli</I> TOP 10 competent cells
</LI></UL><PRE>- start thawing 100 µl competent <I>E. coli</I> TOP 10 cells on crushed ice
- add 5 µl ligation
- incubate on ice for 20 minutes
- heat shock at 42 °C for 45 sec
- incubate 2 min on ice
- add 400 µl preheated LB media
- incubate at 37 °C for 1 h, 500 rpm
- Plate 500 µl on preheated LB-Amp plate
- incubate overnight
</PRE><P>[<A href="https://2008.igem.org/Team:Heidelberg/Notebook/Killing_II/10thweek" class="external text" rel="nofollow">back</A>]
</P><P><A href="/Team:Heidelberg/Notebook/Killing_II/11thweek" title="Team:Heidelberg/Notebook/Killing II/11thweek"> go to 11<SUP>th</SUP> week</A></P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2008.igem.org/Team:Heidelberg/Notebook/Killing_II/10thweek">http://2008.igem.org/Team:Heidelberg/Notebook/Killing_II/10thweek</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Heidelberg/Notebook/Killing_II/10thweek" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Heidelberg/Notebook/Killing_II/10thweek" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Heidelberg/Notebook/Killing_II/10thweek&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Heidelberg/Notebook/Killing_II/10thweek&amp;oldid=82466" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2008.igem.org:Privacy_policy" title="2008.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2008.igem.org:General_disclaimer" title="2008.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>