<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Heidelberg_Project_Killing_II"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:Heidelberg/Project/Killing_II">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Heidelberg/Project/Killing_II&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/Project/Killing_II&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/Project/Killing_II&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Heidelberg/Project/Killing_II" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Heidelberg/Project/Killing II</H1><DIV id="bodyContent"><H3 id="siteSub">From 2008.igem.org</H3><DIV class="MenuBar" id="navi"><UL><LI><A href="https://2008.igem.org/Team:Heidelberg" style="color: white">Home
			</A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Team" style="color: white">Team</A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Team"><SPAN><SPAN>Overview</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Team_Members#Advisors"><SPAN><SPAN>Advisors</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Team_Members#Undergraduates"><SPAN><SPAN>Undergraduates</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Team#There_are_many_Universities_-_and_there_is_Heidelberg_..."><SPAN><SPAN>University</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Team#DKFZ"><SPAN><SPAN>DKFZ</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Team#Bioquant"><SPAN><SPAN>BioQuant</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/BioRegioRN"><SPAN><SPAN>BioRegion Rhein-Neckar</SPAN></SPAN></A></LI></UL><LI><A href="https://2008.igem.org/Team:Heidelberg/Project" style="color: white">Project</A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Project"><SPAN><SPAN>Overall Project</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Notebook/material"><SPAN><SPAN>Material &amp; Methods</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Project/Sensing"><SPAN><SPAN>Sensing</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Project/Killing_I"><SPAN><SPAN>Killing I  - Phages</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Project/Killing_II"><SPAN><SPAN>Killing II - Colicin</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Project/Visualization"><SPAN><SPAN>Visualization</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Parts" style="color: white">Parts</A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Parts"><SPAN><SPAN>Submitted Parts</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Parts/Characterization"><SPAN><SPAN>Characterization</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Modeling" style="color: white">Modeling</A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Modeling"><SPAN><SPAN>Overview</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Modeling/Chemotaxis"><SPAN><SPAN>Chemotaxis-Colicin</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Modeling/Phage"><SPAN><SPAN>Phage Dynamics model</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Notebook/Overview" style="color: white">Notebook</A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Notebook/Sensing_Group/Notebook"><SPAN><SPAN>Sensing &gt;</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Notebook/Sensing_Group/Cloning"><SPAN>Cloning strategy</SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Notebook/Sensing_Group/Notebook"><SPAN>Notebook</SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Notebook/Killing_I/Notebook"><SPAN><SPAN>Killing I - Phages &gt;</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Notebook/Killing_I/Cloning"><SPAN>Cloning strategy</SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Notebook/Killing_I/Notebook"><SPAN>Notebook</SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Notebook/Killing_II/Notebook"><SPAN><SPAN>Killing II - Colicin &gt;</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Notebook/Killing_II/Cloning"><SPAN>Cloning strategy</SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Notebook/Killing_II/Notebook"><SPAN>Notebook</SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Notebook/visualization"><SPAN><SPAN>Visualization</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Notebook/material"><SPAN><SPAN>Material &amp; Methods</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Notebook/team_meetings"><SPAN><SPAN>Team Meetings</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Notebook/seminar"><SPAN><SPAN>Seminar on Synthetic Biology</SPAN></SPAN></A></LI><LI style="width: 160px"><A href="https://2008.igem.org/Team:Heidelberg/Human_Practice/Project_Overview" style="color: white">Human Practice</A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Human_Practice/Project_Overview"><SPAN><SPAN>Project Overview</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Human_Practice/Phips_the_Phage"><SPAN><SPAN>Phips the Phage</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Human_Practice/Essay"><SPAN><SPAN>Essay</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Human_Practice/Surveys"><SPAN><SPAN>Surveys</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Human_Practice/Open_Day"><SPAN><SPAN>Open Day</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Human_Practice/Nobel_Prize"><SPAN><SPAN>Nobel Prize</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Sponsors" style="color: white">Sponsors</A></LI></DIV><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Colicins_-_Project_Description"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Colicins - Project Description</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Introduction"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Introduction</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Theoretical_Background"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Theoretical Background</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Part_Design"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Part Design</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Results"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">Results</SPAN></A></LI><LI class="toclevel-3 tocsection-6"><A href="#Sender"><SPAN class="tocnumber">1.4.1</SPAN><SPAN class="toctext">Sender</SPAN></A></LI><LI class="toclevel-3 tocsection-7"><A href="#Colicin-Receiver"><SPAN class="tocnumber">1.4.2</SPAN><SPAN class="toctext">Colicin-Receiver</SPAN></A></LI><LI class="toclevel-4 tocsection-8"><A href="#Colicin_E9"><SPAN class="tocnumber">1.4.2.1</SPAN><SPAN class="toctext">Colicin E9</SPAN></A></LI><LI class="toclevel-4 tocsection-9"><A href="#Colicin_E1"><SPAN class="tocnumber">1.4.2.2</SPAN><SPAN class="toctext">Colicin E1</SPAN></A></LI><LI class="toclevel-4 tocsection-10"><A href="#Final_Prey-Killer_System"><SPAN class="tocnumber">1.4.2.3</SPAN><SPAN class="toctext">Final Prey-Killer System</SPAN></A></LI><LI class="toclevel-4 tocsection-11"><A href="#Eukaryotic_killing_Assay_.28Preliminary_Results.29"><SPAN class="tocnumber">1.4.2.4</SPAN><SPAN class="toctext">Eukaryotic killing Assay (Preliminary Results)</SPAN></A></LI><LI class="toclevel-5 tocsection-12"><A href="#Data_Analysis"><SPAN class="tocnumber">1.4.2.4.1</SPAN><SPAN class="toctext">Data Analysis</SPAN></A></LI><LI class="toclevel-5 tocsection-13"><A href="#Preliminary_Results"><SPAN class="tocnumber">1.4.2.4.2</SPAN><SPAN class="toctext">Preliminary Results</SPAN></A></LI></UL><LI class="toclevel-2 tocsection-14"><A href="#References"><SPAN class="tocnumber">1.5</SPAN><SPAN class="toctext">References</SPAN></A></LI></TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="Colicins_-_Project_Description">Colicins - Project Description</SPAN></H1><H2><SPAN class="mw-headline" id="Introduction">Introduction</SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:322px;"><DIV class="thumbcaption"><B> Figure 1 Scheme of Colicin System. </B>  The prey contains genes for LuxI which produces AHL. The killer cells are able to sense AHL by LuxR which activates the colicin production and release.</DIV></DIV></DIV><P>
Pathogenous bacteria or even cancer cells are often hard do kill or even to sense. Therefore the iGEM team Heidelberg decided this year to work on a production of a bacterial machine that will be able to sense, swim towards and kill pathogens or even cancer cells. 
To proof in principle the feasibility of this system two <I>E. coli</I> strains were created, one prey strain and one killer strain. The prey strain emblematizing the pathogen produces certain, for this pathogen typical, molecules. For testing the system molecules of the quorum sensing system of Vibrio fischeri, Autoinducer-1 (AI-1 or AHL) and Autoinducer-2 (AI-2), were used. The killer strain is able to sense the secreted molecules and act as follows: by sensing the Autoinducer-2 the killer strain swims toward the pathogen. The AI-1 is responsible for the activation of the killing mechanism of the prey strain by inducing the production of a bacterial toxin, termed colicin.
Colicins are toxins produced by <I>Escherichia coli</I> bacteria, acting against related strains or spices. The toxin is encoded on plasmids, termed pCol, and contains in general three genes which are related to their function. The colicin gene encodes for the toxin itself. Colicins have different mode of actions. Most colicins are either pore-forming colicins (e.g. colicin E1) or have enzymatic activity, as nucleases (e.g. colicin E9). Their toxicity can be inhibited by a protein-protein interaction with the immunity protein, produced by the immunity gene in order to protect the host cell against cell death. The lysis protein, produced by the lysis gene is finally responsible for the release of colicins. 
For the experimental approach the nuclease colicin E9 and the pore-forming colicin E1 were used. The functionality was only shown for colicin E1 which is also known to inhibit the growth of bacteria and eukaryotic cell lines.
</P><P>[<A href="https://2008.igem.org/Team:Heidelberg/Project/Killing_II" class="external text" rel="nofollow">back</A>]
</P><H2><SPAN class="mw-headline" id="Theoretical_Background">Theoretical Background</SPAN></H2><P>The main idea of the project was to create a system to kill pathogens or even harmful eukaryotic cells such as tumour cells. Therefore, secreted molecules of these pathogens are sensed by a generated bacterial killing strain and will be killed with the help of a secreted toxin.
Bacteriocins are toxins, more precisely ribosomally synthesized antimicrobial peptides <A href="/Team:Heidelberg/Project/Killing_II#References" title="Team:Heidelberg/Project/Killing II">[1]</A>, produced by certain bacteria. These toxins are active against other bacteria of the same species or against bacteria of other species. <A href="/Team:Heidelberg/Project/Killing_II#References" title="Team:Heidelberg/Project/Killing II">[1]</A> The host bacterium is immune to the toxin due to production of immunity proteins. <A href="/Team:Heidelberg/Project/Killing_II#References" title="Team:Heidelberg/Project/Killing II">[1]</A> 
To establish the project it was decided to test the system with <I>Escherichia coli</I> bacteria and their bacteriocins, so called colicins. Colicins are proteins produced by certain strains of <I>E. coli</I> that are lethal for related <I>E. coli</I> strains. Colicin was first identified by Gratia in 1925 who also established the name âcolicinâ, coming from its producing bacteria (<I>E. coli</I>). <A href="/Team:Heidelberg/Project/Killing_II#References" title="Team:Heidelberg/Project/Killing II">[2]</A>
Most of the colicins act as nucleases or form pores in cell membranes. All colicins kill due to the presence of a receptor in the outer membrane of the target cell where the colicin binds to. <A href="/Team:Heidelberg/Project/Killing_II#References" title="Team:Heidelberg/Project/Killing II">[3]</A> To enter the cell colicins use either the <I>tol</I> or the <I>tonB</I> system, which allows to classify colicins into two groups: group A and B <A href="/Team:Heidelberg/Project/Killing_II#References" title="Team:Heidelberg/Project/Killing II">[4]</A>, <A href="/Team:Heidelberg/Project/Killing_II#References" title="Team:Heidelberg/Project/Killing II">[5]</A>.
Group A colicins are translocated by the <I>tol</I> system such as Colicin A, E1-E9, K, L, N, S4, U and Y. For colicin E1, only TolA and TolQ are required whereas TolABQR and OmpF are essential for colicin E9. <A href="/Team:Heidelberg/Project/Killing_II#References" title="Team:Heidelberg/Project/Killing II">[6]</A> Group B colicins comprise colicins that are translocated by the <I>tonB</I> system as Colicin B, D, H, Ia,Ib, M, 5 and 10. Another difference is that colicins of group B usually are not secreted, whereas colicins of group A are released into the media.<A href="/Team:Heidelberg/Project/Killing_II#References" title="Team:Heidelberg/Project/Killing II">[4]</A>, <A href="/Team:Heidelberg/Project/Killing_II#References" title="Team:Heidelberg/Project/Killing II">[5]</A> 
Both, the Tol- and TonB- dependant colicins are encoded on colicinogenic plasmids, termed pCol. Colicins are organized in three domains: a reception-, a translocation- and a killing domain. These three domains are related to their steps of action, which will be described below. 
In nature colicinogenic strains are widespread and can be found for example in the guts of animals. However there are two classes of pCols. The type I pCol plasmids are small multicopy plasmids (6-10 kb, ~20 copies/cell) and are usually encoded by the Group A colicins. The type II pCol plasmids are large monocopy plasmids of approximately 40 kb which encode mainly for colicins of group B. 
</P><P>For the project the decision was to work with colicins of group A only. Their genetic organization was described by Riley et al. in 1993. <A href="/Team:Heidelberg/Project/Killing_II#References" title="Team:Heidelberg/Project/Killing II">[7]</A>, <A href="/Team:Heidelberg/Project/Killing_II#References" title="Team:Heidelberg/Project/Killing II">[8]</A>
In general plasmids of group A colicins contain three genes: a colicin gene encoding for the toxin, an immunity gene and a lysis gene (see figure 1). The colicin gene is under the control of a SOS promoter (P<SUB>SOS</SUB>). The promoter responds to a variety of agents such as UV-light, physical agents and stress. <A href="/Team:Heidelberg/Project/Killing_II#References" title="Team:Heidelberg/Project/Killing II">[9]</A> The immunity gene is under the regulation of two promoters, the P<SUB>SOS</SUB> of the colicin operon and its own constitutive promoter that allows a constitutive production of the immunity protein. This ensures that there is no free colicin inside the cytoplasm, which would kill the host cell. The separated constitutive promoter is located within the structural gene of nuclease colicins. There is no immunity gene in operons encoding pore-forming (ionophoric) colicins: It is located on the opposite DNA strand of the intergenetic space between the colicin and the lysis structural gene and is transcribed from its own promoter under constitutive regulation. The last gene of the colicin operon encodes for the lysis protein, which is controlled by the same promoter as the colicin production. Due to the terminator localized downstream of the colicin gene, the lysis protein is expressed in lower amounts than the colicins. After reaching a certain threshold of lysis protein the host cell is lysed which leads to the release of colicins into the medium. <A href="/Team:Heidelberg/Project/Killing_II#References" title="Team:Heidelberg/Project/Killing II">[3]</A></P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:802px;"><DIV class="thumbcaption"><B> Figure 2: Organization of colicin genes. </B> Genes on the leading strand are represented by black arrows, gene on the opposite strand by blue arrow. SOS promoter (P<SUB>SOS</SUB>), immunity promoter (P<SUB>im</SUB>) and terminators (T) are indicated by arrows. The first gene in each colicin operon is the colicin encoding gene (cxa â colicin X activity gene, X specifies a certain colicin gene as for example colicin E1 activity gene), followed by a gene encoding the immunity protein (cxi â colicin X immunity gene). The immunity gene is located in the colicin operon of nuclease colicins but at the opposite DNA strand for pore-forming colicins. The lyses protein, produced by the colicin X lyses gene (cxl) is responsible for the colicin release in the medium and the death of the host cell. <A href="/Team:Heidelberg/Project/Killing_II#References" title="Team:Heidelberg/Project/Killing II">[3]</A></DIV></DIV></DIV></DIV><P>
Lysis proteins are small lipoproteins with a size of 27 to 35 amino acids. As the sequences of the lyses proteins from different colicins are very similar, it is supposed that they have got a common origin. The main function of the lysis protein is to trigger the release of colicin. The amount of lyses protein that is produced varies among the same environmental factors as the colicin production. The mechanism of the colicin release due to the lysis protein is not elucidated so far. Clear is that most of the colicin is released in the media, but only in a late state of release small amounts of lysis proteins were detected. This indicates an adjusted release of the colicins rather than a lysis of the whole cell. 
Due to the release of the colicin into the media group A colicins are able to diffuse towards other cells. After binding to an adequate receptor on the target cell, colicin translocation starts. BtuB is the receptor for all nine colicins E and was the first that was purified.  The binding domain of the colicin is located in the center of the protein. <A href="/Team:Heidelberg/Project/Killing_II#References" title="Team:Heidelberg/Project/Killing II">[11]</A>, <A href="/Team:Heidelberg/Project/Killing_II#References" title="Team:Heidelberg/Project/Killing II">[12]</A>, <A href="/Team:Heidelberg/Project/Killing_II#References" title="Team:Heidelberg/Project/Killing II">[13]</A> Colicin binding to its adequate receptor is the first step of the colicin action. <A href="/Team:Heidelberg/Project/Killing_II#References" title="Team:Heidelberg/Project/Killing II">[12]</A> 
The essential step for the translocation is to pass the outer membrane. Therefore, a second outer membrane protein is required (mainly OmpF, but TolC for the colicin E1) which enables a correct initial binding of the colicin at the cell surface. 
The toxic activity of the colicins is given due to its C-terminal domain. It either carries a pore-forming (as colicin E1) or nuclease (like colicin E9) activity. <A href="/Team:Heidelberg/Project/Killing_II#References" title="Team:Heidelberg/Project/Killing II">[13]</A>, <A href="/Team:Heidelberg/Project/Killing_II#References" title="Team:Heidelberg/Project/Killing II">[14]</A>, <A href="/Team:Heidelberg/Project/Killing_II#References" title="Team:Heidelberg/Project/Killing II">[15]</A>To protect the bacteria against their own toxic product, an immunity protein is synthesized. This immunity protein binds to the C-terminal domain of the enzymatic colicins in order to prevent the death of the host cell. The immunity protein of pore-forming colicins is located in the cytoplasmic membrane. The protein-protein interaction between this immunity protein and the colicin proteins inhibits the formation of channels during the process of colicin association of the membrane. <A href="/Team:Heidelberg/Project/Killing_II#References" title="Team:Heidelberg/Project/Killing II">[16]</A></P><P>For realization of the project two different colicins were used. Colicin E9 (ColE9-J plasmid â pCK67), a colicin with nuclease activity, was received from the Kleanthous Laboratory at the University of York and the colicin E1 (pJC411), a pore forming colicin, from the German Collection of Microorganisms and Cell Cultures (DSMZ). 
</P><P>Colicin E1 is a 522 residue containing protein and belongs to the group A colicins. It kills sensitive cells, which harbour no ColE1 plasmid but contain the appropriated receptor BtuB, by forming pores, or more precisely ion channels into the inner membrane. <A href="/Team:Heidelberg/Project/Killing_II#References" title="Team:Heidelberg/Project/Killing II">[17]</A> Affected cells can be <I>E. colis</I> or related strains but also some eukaryotic cells have been affected by the toxic activity of colicin. <A href="/Team:Heidelberg/Project/Killing_II#References" title="Team:Heidelberg/Project/Killing II">[18]</A> 
The first step of the lethal action of colicin E1 is binding to the vitamin B<SUB>12</SUB> receptor (BtuB). Colicins are able to kill bacteria by a âone hitâ mechanism. This works due to destruction of the target cells cellular energy. The voltage dependant channel in the cytoplasmatic membrane leads to a depolarization of the target cell. The conductance of the channel has been shown to be ~10<SUP>7</SUP> ions/channel-sec in 1 M NaCl. <A href="/Team:Heidelberg/Project/Killing_II#References" title="Team:Heidelberg/Project/Killing II">[19]</A> 
Its ability to form pores is given by the C-terminal domain. This domain consists of 10 tightly packed Î±âhelices (see Figure 1). <A href="/Team:Heidelberg/Project/Killing_II#References" title="Team:Heidelberg/Project/Killing II">[20]</A>, <A href="/Team:Heidelberg/Project/Killing_II#References" title="Team:Heidelberg/Project/Killing II">[5]</A> The colicin protein is a water soluble molecule containing mostly amphipathic helices except from two hydrophobic ones. Due to these two hydrophobic helices the protein is able to target the cell membrane and act as a membrane protein.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:802px;"><DIV class="thumbcaption"><B> Figure 3 (a)</B> Ribbon diagram of the Colicin E1 pore-forming domain by X ray crystal structure. The channel forming domain contains 10 helices (I-X). The two hydrophobic helices (VIII and IX) are labeled yellow. <B>(b)</B> Colicin E1 in a membrane bound state in two perspectives. Left panel shows the surface bound helices with an area of ~4200 Ã<SUP>2</SUP>. Right panel shows colicin E1 bound to a lipid bilayer. The intra-membrane sections of the two helices VIII and IX are shown in yellow. <A href="/Team:Heidelberg/Project/Killing_II#References" title="Team:Heidelberg/Project/Killing II">[21]</A></DIV></DIV></DIV></DIV><P>
To proof that the system is functional, one prey strain, representing the pathogen, and one killer strain were generated. The quorum sensing system, which was already described in several papers and reviews <A href="/Team:Heidelberg/Project/Killing_II#References" title="Team:Heidelberg/Project/Killing II">[22]</A>, <A href="/Team:Heidelberg/Project/Killing_II#References" title="Team:Heidelberg/Project/Killing II">[23]</A>, <A href="/Team:Heidelberg/Project/Killing_II#References" title="Team:Heidelberg/Project/Killing II">[24]</A>, was used as cell-cell communication system. The molecules, secreted by the pathogens, were represented by acyl-homoserine lactone (AHL), originally produced by the LuxI from <I>Vibrio fischeri</I>. <A href="/Team:Heidelberg/Project/Killing_II#References" title="Team:Heidelberg/Project/Killing II">[22]</A> The prey cell therefore contains a plasmid encoding the LuxI protein which produces AHL molecules. These molecules are able to diffuse through the cell wall and medium where they form a gradient. The killer strain, which is attracted by another molecule, swims toward the prey cell and senses the AHL molecules. The killer bacteria produce the LuxR protein. AHL molecules bind to LuxR <A href="/Team:Heidelberg/Project/Killing_II#References" title="Team:Heidelberg/Project/Killing II">[25]</A> and act as transcriptional factor for the regulation of the colicin promoter P<SUB>LuxR</SUB>. This promoter regulates the colicin production. Another interesting aspect of colicins is their ability to harm and kill tumor cells. <A href="/Team:Heidelberg/Project/Killing_II#References" title="Team:Heidelberg/Project/Killing II">[26]</A></P><P>[<A href="https://2008.igem.org/Team:Heidelberg/Project/Killing_II" class="external text" rel="nofollow">back</A>]
</P><H2><SPAN class="mw-headline" id="Part_Design">Part Design</SPAN></H2><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:452px;"><DIV class="thumbcaption"><B> Figure 4 Cloning strategy of the sender part (<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K150000" class="external text" rel="nofollow">BBa_K150000</A>) </B> Parts <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_J23107" class="external text" rel="nofollow">BBa_J23107</A> and <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_F1610" class="external text" rel="nofollow">BBa_F1610</A> were combined according to the BioBrick<SUP>TM</SUP> assembly. Therefore <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_J23107" class="external text" rel="nofollow">BBa_J23107</A> on <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_J61002" class="external text" rel="nofollow">BBa_J61002</A> was digested with SpeI and PstI and <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_F1610" class="external text" rel="nofollow">BBa_F1610</A> was digested with XbaI and PstI. Cloning was finished with the final ligation.</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:902px;"><DIV class="thumbcaption"><B> Figure 5 Cloning strategy of the LuxR-colicin-receiver parts (E1: <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K150009" class="external text" rel="nofollow">BBa_K150009</A>, E9 long: <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K150011" class="external text" rel="nofollow">BBa_K150011</A>, E9 short: <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K150013" class="external text" rel="nofollow">BBa_K150013</A>) </B> First of all the LuxR receiver part (<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_T9002" class="external text" rel="nofollow">BBa_T9002</A>) was amplified by PCR. The product, containing a P<SUB>tet</SUB> promoter, the luxR gene followed by the P<SUB>LuxR</SUB> promoter and a ribosome binding site was cloned into pSB1A2 vector. This vector was extracted from <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_T9002" class="external text" rel="nofollow">BBa_T9002</A>. In a second cloning step the different colicin operons, amplified from pColE1 or pColE9-J were cloned behind the P<SUB>LuxR</SUB> promoter.</DIV></DIV></DIV></DIV><P>[<A href="https://2008.igem.org/Team:Heidelberg/Project/Killing_II" class="external text" rel="nofollow">back</A>]
</P><H2><SPAN class="mw-headline" id="Results">Results</SPAN></H2><H3><SPAN class="mw-headline" id="Sender">Sender</SPAN></H3><P>The sender, also termed prey, is composed of a plasmid containing the <I>luxI</I> gene downstream of a constitutive promoter (<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_J23107" class="external text" rel="nofollow">BBa_J23107</A>) in <I>Escherichia coli</I> TOP10 cells. The prey cell is responsible for continuous production of AHL through the LuxI protein expressed by the <I>luxI</I> gene. To generate this device, the sender part (<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_F1610" class="external text" rel="nofollow">BBa_F1610</A>) was obtained directly from the plasmid by digestion. The vector (<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_J61002" class="external text" rel="nofollow">BBa_J61002</A>) which already contains the constitutive promoter, was also digested. Both parts were ligated by the BioBrick<SUP>TM</SUP> standard assembly. To verify the right gene product, palsmid DNA of a positive clone was digested and sequenced (see Figure 6). 
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><B> Figure 6: Control digestion of the prey cell cloning (<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_J23107" class="external text" rel="nofollow">BBa_J23107</A>+<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_F1610" class="external text" rel="nofollow">BBa_F1610</A>). </B> Bright bands of the DNA ladder from bottom to top: 3000 bp, 1000 bp and 500 bp. Expected bands for the digestion: ~850 bp for the Insert (<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_J23107" class="external text" rel="nofollow">BBa_J23107</A>+<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_F1610" class="external text" rel="nofollow">BBa_F1610</A>) and ~2070 bp for the vector (<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_J61002" class="external text" rel="nofollow">BBa_J61002</A>) at the left hand side of the ladder. Undigested plasmid was loaded on the right hand side as a negative control.</DIV></DIV></DIV></DIV><P>
To quantify the rate of AHL production, sender activity tests were performed. Therefore sterile supernatants of liquid <I>E. coli</I> TOP 10 overnight cultures containing the sender part (<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K150000" class="external text" rel="nofollow">BBa_K150000</A>) were produced hourly. In addition, the optical density at these time points was measured to estimate the amount of producing cells (see Figure 7, top left). Afterwards the AHL concentrations of the supernatants were determined by the GFP production of <I>E. coli</I> containing a LuxR-GFP producing part (<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_T9002" class="external text" rel="nofollow">BBa_T9002</A>). Therefore the supernatants were diluted with TB media and inoculated with a constant amount of the GFP producing culture. 
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:902px;"><DIV class="thumbcaption"><B> Figure 7: Results of sender activity test. </B> Top left: Growth curve of sender culture. Top right: GFP/OD values of sender supernatants at t = 10 h. Bottom left: GFP/OD â AHL calibration curve. Dotted line visualized regression of an exponential growth curve. Bottom right: Produced AHL concentrations of the sender cells.</DIV></DIV></DIV></DIV><P>
As calibration the same measurements were performed with defined AHL concentrations. Figure 8 shows the change in GFP/OD ratio over time depending on the sender supernatants (left panel) and the preassigned AHL concentrations (right panel). The values increase during the first six hours and reach a maximum. Thereafter they decrease slightly.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:902px;"><DIV class="thumbcaption"><B> Figure 8: GFP/OD results of sender activity measurements. </B> On The left panel the GFP/OD values for induction with sender supernatants are shown and on the right panel the GFP/OD values for predefined AHL inductions can be observed. During the first sic hours the values increase and reach a maximum until the decrease slightly. The values at t = 10 h were used for calculations of AHL concentrations in supernatants.</DIV></DIV></DIV></DIV><P>
The GFP/OD values at t = 10 hours were used to calculate the AHL concentrations of the supernatants. For calibration, the GFP/OD values at t = 10 h were plotted against the preassigned AHL concentrations (Figure 7, bottom left). An exponential growth curve was fitted to this data (blue dotted line; y = 4325 * e<SUP>0.00346*x</SUP>). The AHL concentrations of the sender supernatants were determined using the GFP/OD values at t = 10 h and the exponential regression. In the course of the time the AHL concentration increases until it reaches a certain maximum before a slight decrease (see Figure 7, bottom right, Table 1). This maximum is reached at the end of the logarithmic phase. Several replications of this test have shown that it occurs between 1100 and 1300 pM.
</P><TABLE class="wikitable"><TBODY><TR bgcolor="grey"><TH height="20px," width="220px"> time [h] </TH><TH width="220px"> GFP/OD </TH><TH width="220px"> GFP/OD STD</TH><TH width="220px"> c(AHL) [pM] </TH><TH width="220px"> c(AHL) STD [pM]
</TH></TR><TR align="center"><TD style="font-weight:bold;">0</TD><TD> 5427,847 </TD><TD>  48,01405</TD><TD> 135,7623515 </TD><TD> 5,310956899
</TD></TR><TR align="center"><TD style="font-weight:bold;">1</TD><TD>8713,655 </TD><TD> 235,8338</TD><TD> 418,6964705 </TD><TD> 16,40039009 </TD></TR><TR align="center"><TD style="font-weight:bold;">2</TD><TD> 11849,3 </TD><TD> 296,7075</TD><TD> 602,4247781 </TD><TD> 15,1577546 </TD></TR><TR align="center"><TD style="font-weight:bold;">3</TD><TD> 15982,94</TD><TD> 1123,325</TD><TD> 781,2969295 </TD><TD> 43,55933826 </TD></TR><TR align="center"><TD style="font-weight:bold;">4</TD><TD> 25196,72 </TD><TD> 720,799</TD><TD> 1053,378175 </TD><TD> 17,34848139 </TD></TR><TR align="center"><TD style="font-weight:bold;">5</TD><TD> 36745,75</TD><TD> 652,7182</TD><TD> 1278,906415 </TD><TD> 10,71294138 </TD></TR><TR align="center"><TD style="font-weight:bold;">6</TD><TD>28636,28 </TD><TD> 2980,17</TD><TD> 1129,8639 </TD><TD> 65,68601047 </TD></TR><TR align="center"><TD style="font-weight:bold;">7</TD><TD>33271,21 </TD><TD> 2281,635</TD><TD> 1219,533962 </TD><TD> 42,46359884 </TD></TR></TBODY></TABLE><P>[<A href="https://2008.igem.org/Team:Heidelberg/Project/Killing_II" class="external text" rel="nofollow">back</A>]
</P><H3><SPAN class="mw-headline" id="Colicin-Receiver">Colicin-Receiver</SPAN></H3><P>The killing module was created using a LuxR Receiver module (part of <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_T9002" class="external text" rel="nofollow">BBa_T9002</A>) and different colicin operons: colicin E1 operon, colicin E9 short operon and colicin E9 long operon. All three parts were successfully assembled as described in the <A href="/Team:Heidelberg/Notebook/Killing_II/Cloning" title="Team:Heidelberg/Notebook/Killing II/Cloning"> cloning strategy</A> and fit the BioBrick<SUP>TM</SUP> standard. The following two paragraphs describe the results of the colicin E1 and E9 receivers separately.
[<A href="https://2008.igem.org/Team:Heidelberg/Project/Killing_II" class="external text" rel="nofollow">back</A>]
</P><H4><SPAN class="mw-headline" id="Colicin_E9">Colicin E9</SPAN></H4><P>Colicin E9 is an enzymatic toxin with nuclease activity. Before the cloning of the colicin-receiver the natural toxicity of colicin E9 was tested. Therefore supernatant of lysed <I>E. coli</I> MG1655 cells harbouring the pColE9-J plasmid was diluted with media. The growth ability of <I>E. coli</I> TOP 10 cells containing <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_I20260" class="external text" rel="nofollow">BBa_I20260</A> was measured in this dilution. As result an efficient killing of these bacteria was observed (see Figure 1). A reference experiment with supernatant of MG1655 cells showed normal growth behaviour.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:852px;"><DIV class="thumbcaption"><B> Figure 9 Colicin toxicity test of lysed supernatant on <I>E. coli</I> TOP 10 cells harbouring <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_I20260" class="external text" rel="nofollow">BBa_I20260</A>: </B> The GFP intensity is related to the growth of the <I>E. coli</I> TOP10 cells. These could not grew while adding supernatant of the killing strain, but grew only with supernatant of non toxic MG1655 cells (red line).</DIV></DIV></DIV></DIV><P>
After verifying the activity of the colicin E9 protein the plasmid was extracted. Two different length of parts were amplified by PCR (see figure XX) and extracted from the gel. 
In parallel the receiver part (<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_T9002" class="external text" rel="nofollow">BBa_T9002</A>) was amplified by PCR. The product, containing a P<SUB>tet</SUB> promoter, the luxR gene followed by the P<SUB>LuxR</SUB> promoter and a ribosome binding site was also extracted from the gel (see figure XX). The pSB1A2 plasmid containing the receiver insert (<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_T9002" class="external text" rel="nofollow">BBa_T9002</A>) was used to construct the vector (see figure XX). 
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:852px;"><DIV class="thumbcaption"><B> Figure 10:</B> PCR-amplification results of the Colicin E9 short (~1050 bp) and long (~2014 bp) insert, the receiver part insert (T9002 without GFP) ~1088 bp and the vector pSB1A2.</DIV></DIV></DIV></DIV><P>
The vector and the amplified receiver part were digested and ligated. Using colony PCR screening positive clones were selected. Plasmids of these colonies were digested and send for sequencing to confirm the results. To receive a functional colicin producing cell the short and long fragment of colicin E9 and the produced pSB1A2-receiver were digested and ligated. The final product, containing the receiver part and the colicin E9 genes was confirmed by sequencing. 
</P><P>After the successful combination of the LuxR-receiver with the colicin E9 operons the short colicin E9 receiver was finished. For the long fragment an EcoRI site was mutated (silent mutation) to fit BioBrick<SUP>TM</SUP> standard. To test the production and release of the toxin, several tests were performed but showed no proper activity so far. Further test will be necessary to study the killing ability of both colicin E9 receivers.
</P><P>[<A href="https://2008.igem.org/Team:Heidelberg/Project/Killing_II" class="external text" rel="nofollow">back</A>]
</P><H4><SPAN class="mw-headline" id="Colicin_E1">Colicin E1</SPAN></H4><P>Colicin E1 is, in contrast to the enzymatic colicin E9, a pore-forming toxin. Its natural toxicity was tested as described for colicin E9. The prey cells were not able to grow in colicin E1 supernatants which prove their toxicity (see Figure 9).
</P><P>After the activity tests of the colicin E1 protein the plasmid was extracted and the colicin E1 operon amplified by PCR (see Figure 11). The gel extraction of the colicin E1 was digested and stored for the second cloning step. 
In addition a part of the receiver (<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_T9002" class="external text" rel="nofollow">BBa_T9002</A>), allocated by the registry, was partly amplified by PCR. The product, containing a P<SUB>tet</SUB> promoter, the luxR gene followed by the P<SUB>LuxR</SUB> promoter and a ribosome binding site was also extracted from the gel (see Figure 11). The pSB1A2 plasmid containing the receiver insert (<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_T9002" class="external text" rel="nofollow">BBa_T9002</A>) was used to gain the vector (see Figure 11). 
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:752px;"><DIV class="thumbcaption"><B> Figure 11:</B> PCR amplification of colicin E1 operon, the receiver part insert (T9002 without GFP) ~1088 bp and the vector pSB1A2.</DIV></DIV></DIV></DIV><P>
The vector and the amplified receiver part were digested and ligated. Via colony PCR screening positive clones were selected. The plasmids of these clones were digested and send for sequencing to confirm the results. To receive a functional colicin producing cell the colicin E1 operon and the produced pSB1A2-receiver were digested and ligated. The final product, containing the receiver part and the colicin E1 genes, was confirmed by sequencing and digestion (see Figure 12).
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption"><B>Figure 12: Control digestion of the final pSB1A2-receiver-colicinE1 plasmid (LuxR-colicinE1-receiver <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K150009" class="external text" rel="nofollow">BBa_K150009</A>).</B> Bright bands of ladder from bottom to top: 3000 bp, 1000 bp and 500 bp. First two bands contain approximately 5200 bp, which is the whole plasmid, cutted once by XbaI (X) or SpeI (S). Third row confirms right size of plasmid (~2000 bp) and Insert (receiver-colicinE1 ~3200 bp) and therefore confirms successful cloning. Last row shows the undigested plasmid as negative control.</DIV></DIV></DIV></DIV><P>
To fit BioBrick<SUP>TM</SUP> standard, four restriction sites (one EcoRI, three PstI) of the colicin E1 â LuxP<SUB>R</SUB>were mutated via mutagenesis PCR.
</P><P>The functionality of this receiver was first tested with supernatant assays. After the confirmation of its functionality, further studies were performed. Therefore cells (<I>E. coli</I> TOP 10 containing the <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_I20260" class="external text" rel="nofollow">BBa_I20260</A>; plasmid for GFP-labelling) working as prey cells were directly added to killer cells which were induced using AHL. Several experiments with different âpreyâ-killer ratios as well as certain AHL concentrations were performed to describe the characteristics and limitations of the killing part. 
Figure 13 shows results of tests with a prey-killer ratio of 100:1. It is visible that the GFP intensity correlates with the optical density. For this reason the GFP intensity was used as marker for the living prey cells in the following experiments.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:902px;"><DIV class="thumbcaption"><B>Figure 13: Results of colicin E1 toxicity tests for different AHL concentrations with a prey-killer ratio of 100:1.</B> Top left panel: GFP intensities of prey cells treated with killer cells. Top right panel:  Optical densities of prey cells treated with killer cells. Bottom left panel: GFP intensities of prey cells treated with reference cells. Top right panel:  Optical densities of prey cells treated with reference cells. The graphs proof the toxicity of colicin E1as well as the functionality of the part <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K150009" class="external text" rel="nofollow">BBa_K150009</A>. Each killer cell is able to kill up to 100 prey cell if there is a sufficient amount of AHL present.</DIV></DIV></DIV></DIV><P>
In experiments where killer cells were used and the amount of AHL in the medium was sufficient, all prey cells were killed. In reference experiments (see Figure 13, bottom) using <I>E. coli</I> TOP 10 cells harbouring a LuxR receiver without colicin operon (comparable to part BBa_T9002 without GFP), the prey cells were able to grow for all AHL concentrations. These results suggest the lethal action of the killing part. 
In addition we observed that the killing efficiency is related to the AHL concentration in the medium. Figure 14 shows a dose-response curve of GFP intensity after 12 hours dependent on different AHL concentrations. The P<SUB>LuxR</SUB> promoter has to be activated with an AHL concentration above 500 pM. Thus enough colicin E1 is produced and released to kill the prey cells (Figure 14).
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:755px;"><DIV class="thumbcaption"><B> Figure 14 Dose-response curve of AHL concentration and killing efficiency of the killer strain for a prey-killer ratio of 100:1. </B> The GFP intensities at t = 12 h are plotted against AHL concentrations (0 M â 1 nM). For AHL concentrations higher than 600 nM all preys were killed.</DIV></DIV></DIV></DIV><P>Regarding the prey survival, dependent on the prey killer ratio, several effects can be observed (see Figure 15). Having a prey-killer ratio of 1:1, or even a higher killer fraction, all prey cells were already killed when no AHL is present. This effect is caused by the leakiness of the P<SUB>LuxR</SUB> promoter. For prey-killer ratios between 5:1 and 100:1 the killing efficiency can be regulated by the AHL concentration. In experiments having prey-killer ratios higher than 100:1 the growth of the prey strain was not influenced. 
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:902px;"><DIV class="thumbcaption"><B>Figure 15 Toxicity of LuxR-colicin E1-receiver part for different prey-killer ratios.</B> The graphs show GFP intensities of prey cells at t = 0 h and t = 12 h. For prey-killer ratios of 1:1 all preys were killed after 12 hours even if there was no AHL present. For prey-killer ratios between 5:1 (not shown) and 100:1 the killing efficiency depends on the AHL concentration in the media. If there is a greater predominace of prey cells (500:1) the growth of the prey population is not influenced</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:902px;"><DIV class="thumbcaption"><B>Figure 16 Lysis test with LuxR-colicin E1-Receiver dependent on AHL concentrations.</B> On the left panel the growth of a killer population induced with different concentrations of AHL is shown. At the right panel the data of reference experiments are plotted. For AHL concentrations between 0 M and 1 nM no influence on the growth of the killer population is observed. For concentrations between 5 nM and 25 nM the population grew in the first hour. Afterwards a lysis effect can be observed.</DIV></DIV></DIV></DIV><P>Besides the killing efficiency the lysis of the killer strain was analyzed. Therefore growth curves of killer cells (<I>E. coli</I> TOP10) induced with different AHL concentrations were measured (see Figure 16). No effect can be observed during the first hour. Afterwards cell lysis dependent on AHL concentrations can be observed. For AHL concentrations between 0 M and 1 nM only few cells were lysed. Thus growth of the population is hardly influenced. Concentrations from 5 nM up to 25 nM show a strong effect. For 5 nM and 10 nM the growth curves flattens out. After reaching a maximum during the third and fourth hour the population size decreases and converges to a constant value. For higher AHL concentrations, e.g. 25 nM, the same effect can be observed, but it is much stronger. The population only shows a weak growing tendency but then converges directly to a constant size. In addition to these growth measurements lysis of the killer cells was observed under the microscope. Figure 17 shows a killer cell which is induced by AHL (left panel, t = 0 min). After 30 minutes the cell is lysed (right panel).
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:902px;"><DIV class="thumbcaption"><B>Figure 17 Visualization of the lysis effect of the killer cells. </B> The prey (GFP) and the killer (mCherry) strain are mixed on a 0.5Â % agarose pad. Killer cells were induced with AHL before adding to the medium. The killer cell on the left picture (white arrow) is lysed after 30 minutes. This effect is caused by the AHL indtion. Colicin production and at same time in lower amounts the lyses protein production are induced leading to a lysis of the host cell and colicin release.</DIV></DIV></DIV></DIV><P>Further tests on other bacteria strains as well as comparisons to effects caused by purified colicins are planed to elucidate possible applications.
</P><P>[<A href="https://2008.igem.org/Team:Heidelberg/Project/Killing_II" class="external text" rel="nofollow">back</A>]
</P><H4><SPAN class="mw-headline" id="Final_Prey-Killer_System">Final Prey-Killer System</SPAN></H4><P>To proof the functionality of the system a prey strain containing a GFP producing part (<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_I20260" class="external text" rel="nofollow">BBa_I20260</A>) as well as the <I>luxI</I> sender part (<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K150000" class="external text" rel="nofollow">BBa_K150000</A>) and a killer strain containing the LuxR-colicinE1-receiver (<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K150009" class="external text" rel="nofollow">BBa_K150009</A>) were designed (both <I>E. coli</I> TOP10). Therefore the prey cells were able to produce and release AHL which activates the P<SUB>LuxR</SUB> promoter and subsequently colicin production in killer cells. The system was tested for different prey-killer ratios with GFP measurements over 12 hours. Figure 18 shows that the prey cells were killed for prey-killer ratios from 1:1 up to 25:1. Therefore it is proven that our system works as expected. 
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:702px;"><DIV class="thumbcaption"><B>Figure 18 Prey-Killer system test.</B> GFP intensity of prey strains was measured over 12 h at 37 Â°C. Prey cells were killed for prey-killer ratios from 1:1 up to 25:1 (blue, green, red, light blue) due to the colicin production of the killer strain. This production was activated by the AHL secreted by the prey strain. In a reference experiment prey-cells were able to grow (violet line).</DIV></DIV></DIV></DIV><P>[<A href="https://2008.igem.org/Team:Heidelberg/Project/Killing_II" class="external text" rel="nofollow">back</A>]
</P><H4><SPAN class="mw-headline" id="Eukaryotic_killing_Assay_.28Preliminary_Results.29">Eukaryotic killing Assay (Preliminary Results)</SPAN></H4><P>The toxicity of colicin E1 on eukaryotic cells was tested by using breast cancer cells (MCF-7) and inoculating them in vitro with killer and reference cells for several hours.
[<A href="https://2008.igem.org/Team:Heidelberg/Project/Killing_II" class="external text" rel="nofollow">back</A>]
</P><H5><SPAN class="mw-headline" id="Data_Analysis">Data Analysis</SPAN></H5><P>MCF-7 cell death, measured by Flow Cytometry, was evaluated byÂ % of stained cells with PI over the time, using colicin producing E. coli or strains which do not produce colicins as a control (Figure 20). E. coli was also quantified (data not shown). The PI measurements (filter FL3-620nm) and the GFP signal intensities (FL1-488nm) were gated only to MCF-7 cell population (see Scatterplot figure 19). [<A href="https://2008.igem.org/Team:Heidelberg/Project/Killing_II" class="external text" rel="nofollow">back</A>]
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:702px;"><DIV class="thumbcaption"><B> Figure 19: Scatterplot of Flow Cytometry measurement.</B> Two cell populations can be differentiated and for the following fluorescent measurement gated: In the upper right quadrant the big eukaryotic MCF-7 cells and in the lower right quadrant the small E.  coli bacterial cells.</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:632px;"><DIV class="thumbcaption"><B> Figure 20 Positive PI MCF-7 cells (%) over time. </B><I> E coli</I> producing colicin E1 protein cause cell death at the earliest time point (2 h) and keep it higher then the other condition. After 3 h, the reference strain also starts to present PI positive cells that increase over time.</DIV></DIV></DIV></DIV><P>[<A href="https://2008.igem.org/Team:Heidelberg/Project/Killing_II" class="external text" rel="nofollow">back</A>]
</P><H5><SPAN class="mw-headline" id="Preliminary_Results">Preliminary Results</SPAN></H5><P>The initial results show that killing by E. coli producing colicin E1 is effective on eukaryotic cancer cells (MCF-7). Propidium iodide (PI) is an intercalating agent and a fluorescent molecule with a molecular mass of 668.4 Da that can be used to stain DNA. It can be used to differentiate, apoptotic and normal cells, since it is a membrane impermeant and generally excluded from viable cells. The high percentage of positive PI cells refers to cells that are permeable to it, thus, going into apoptosis. In this assay, after 2 h of incubation with the eukaryotic cells the death rate was close to 40Â % of the whole eukaryotic cell population. In comparison with the control conditions (MCF-7 alone + PI), where cells (maximum of 5Â %) due to the trypsin treatment die, the effect is clearly visible. Furthermore, looking at the eukaryotic cell death after incubation with the reference strain (LuxR-receiver TOP 10), it can be suggested that the simple incubation of these mammalian cells with bacteria is causing cell death over time (maximum of 20Â % after 5 h). This effect is highly enhanced and faster when the bacteria produce colicin E1 (maximum of 37Â % at 2 h). After 5 h, both bacterial strains produce similar results in killing the mammalian cancer cells. After reaching a peak, the killing efficiency drops with a significant higher peak for E. coli bacteria producing colicin E1 than for the reference strain.
On the other hand, using a stable cell line of MCF-7 expressing GFP, the effects of bacteria were not visible for the initial timepoints. The aim of this assay was to see the GFP signal droping down as E. coli progressed to kill the eukaryotic cells. Since GFP is a very robust protein we can suggest that colicin E1 producing bacteria start apoptotic events at early time (2 h, see Figure 20), but GFP loss can only be observed after a certain time delay (see figure 21). For that reason, it would be necessary to observe also later timepoints to characterize the killing effect by GFP loss.
[<A href="https://2008.igem.org/Team:Heidelberg/Project/Killing_II" class="external text" rel="nofollow">back</A>]
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:633px;"><DIV class="thumbcaption"><B> Figure 21: GFP expressing MCF-7 cells (%) over time.</B> E coli bacteria producing colicin E1 protein cause GFP loss after 6 h, to an extend of 60% loss in comparison with the both control conditions that remained around 99Â % GFP detection.</DIV></DIV></DIV></DIV><P>
Summerized, the preliminary results suggest that colicin E1 producing E. coli bacteria might be able to kill cancer cells at a high rate. However, further studies need to provide more insight into these events, as it might be by other pro-apoptotic markers or by repeating this experiment using different conditions (timepoints, autoinducer concentrations, another eukaryotic cell type). Future studies should yield to information about the colicin E1 target in eukaryotic cells and perhaps answers how it can kill cancer cells and spare healthy ones.
[<A href="https://2008.igem.org/Team:Heidelberg/Project/Killing_II" class="external text" rel="nofollow">back</A>]
</P><H2><SPAN class="mw-headline" id="References">References</SPAN></H2><P>[1]  Cotter P. et al.(2005) Bacteriocins: developing innate immunity for food. Nature Rev. Microbiol. 3: 777â788 
[2]  Gratia A. (1925) Sur un remarquable example d'antagonisme entre deux souches de colibacille. Compt. Rend. Soc. Biol. 93: 1040â2.
[3]  Cascales et al. (2007) Colicin Biology. Microbio. and Mol. Bio. Rev. 71(1): 158-229
[4]  Davies J. et al. (1975) Genetics of resistance to colicins in Escherichia coli K12: cross-resistance among resistance of group B. J. Bacteriol. 123:96â101
[5]  Davies, J. et al. (1975) Genetics of resistance to colicins in Escherichia coli K12: cross-resistance among resistance of group A. J. Bacteriol. 123:102â117
[6]  Lazzaroni, J. (2002) The Tol proteins of Escherichia coli and their involvement in the translocation of group A colicins. Biochimie. 84: 391â397.
[7]  Riley M. (1993). Molecular mechanisms of colicin evolution. Mol. Biol. Evol. 10: 1380â1395.
[8]  Riley M. (1993). Positive selection for colicin diversity in bacteria. Mol. Biol. Evol. 10: 1048â1059.
[9]  Lwoff A. (1952) Induction of bacteriophage production and of a colicin by peroxydes, ethyleneimines and halogenated alkylamines. C. R. Acad. Sci. 234: 2308â2310
[10]  Sabet S. et al. (1973) Purification and properties of the colicin E3 receptor of Escherichia coli. J. Biol. Chem. 248: 1797â1806
[11]  Brunden, K. (1984). Purification of a small receptor-binding peptide from the central region of the colicin E1 molecule. J. Biol. Chem. 259: 190â196.
[12]  Ohno-Iwashita Y. (1980). Assignment of the functional loci in colicin E2 and E3 molecules by characterization of their proteolytic fragments. Biochemistry 19:652â659.
[13]  Ohno-Iwashita Y. (1982). Assignment of the Functional Loci in the Colicin El Molecule by Characterization of Its Proteolytic Fragments J. Biol. Chem. 257: 6446-6451
[14]  Bullock J. (1983) Comparison of the macroscopic and single channel conductance properties of colicin E1 and its COOH-terminal tryptic peptide. J. Biol. Chem. 258: 9908â9912.
[15]  Pommer A. (1998) Enzymological characterization of the nuclease domain from the bacterial toxin colicin E9 from Escherichia coli Biochem. J. 334: 387-392
[16]  Lindeberg M. (2001) Identification of Specific Residues in Colicin E1 Involved in Immunity Protein Recognition. J Bacteriol. 183: 2132â2136
[17] Cramer W. et al. (1990) Structure and dynamics of the colicin E1 channel. Molecular Microbiology 4: 519-526.
[18] 	 Smarda J. et al. (2001) Cytotoxic effects of colicins E1 and E3 on v-myb-transformed chicken monoblasts. Folia Microbiol. (Praha) 47: 11â13.
[19]	 Bullock J. et al. (1983) Comparison of the macroscopic and single channel conductance properties of colicin E1 and its COOH-terminal tryptic peptide. J. Biol. Chem. 258: 9908â9912.
[20] 	 Elkins P. et al. (1997) A mechanism for toxin insertion into membranes is suggested by the crystal structure of the channel-forming domains of Colicin E1. Structure. 5: 443-458.
[21] 	 Lindeberg M. et al. (2000) Unfolding Pathway of the Colicin E1 Channel Protein on a Membrane Surface. J. Mol. Biol. 295: 679-692
[22]  Waters C. et al. (2005) Quorum sensing: Cell-to-Cell Communication in Bacteria. Annu. Rev. Cell Dev. Biol. 21: 319â46
[23]  Joost C et al. (2008) Small Molecules for Interference with Cell-Cell-Communication Systems in Gram-Negative Bacteria, 13: 2144-2156
[24]  Lazdunski A. et al. (2004) Regulatory Circuits And Comminication in Gram Negative Bacteria. Natur reviews Microbiology. 2:| 581-592
[25]  Engebrecht J. et al. (1984) Identification of genes and gene products necessary for bacterial bioluminescence. Proc. Natl. Acad. Sci. USA 81: 4154â58
[26]  Smarda J. et al. (2001) Cytotoxic effects of colicins E1 and E3 on v-myb-transformed chicken monoblasts. Folia Microbiol. (Praha) 47: 11â13</P><P>[<A href="https://2008.igem.org/Team:Heidelberg/Project/Killing_II" class="external text" rel="nofollow">back</A>]
</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2008.igem.org/Team:Heidelberg/Project/Killing_II">http://2008.igem.org/Team:Heidelberg/Project/Killing_II</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Heidelberg/Project/Killing_II" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Heidelberg/Project/Killing_II" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Heidelberg/Project/Killing_II&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Heidelberg/Project/Killing_II&amp;oldid=92109" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2008.igem.org:Privacy_policy" title="2008.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2008.igem.org:General_disclaimer" title="2008.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>