<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Heidelberg_Notebook_material"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:Heidelberg/Notebook/material">Page               </A></LI><LI><A href="/Talk:Team:Heidelberg/Notebook/material">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/Notebook/material&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/Notebook/material&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Heidelberg/Notebook/material" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Heidelberg/Notebook/material</H1><DIV id="bodyContent"><H3 id="siteSub">From 2008.igem.org</H3><DIV class="MenuBar" id="navi"><UL><LI><A href="https://2008.igem.org/Team:Heidelberg" style="color: white">Home
			</A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Team" style="color: white">Team</A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Team"><SPAN><SPAN>Overview</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Team_Members#Advisors"><SPAN><SPAN>Advisors</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Team_Members#Undergraduates"><SPAN><SPAN>Undergraduates</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Team#There_are_many_Universities_-_and_there_is_Heidelberg_..."><SPAN><SPAN>University</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Team#DKFZ"><SPAN><SPAN>DKFZ</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Team#Bioquant"><SPAN><SPAN>BioQuant</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/BioRegioRN"><SPAN><SPAN>BioRegion Rhein-Neckar</SPAN></SPAN></A></LI></UL><LI><A href="https://2008.igem.org/Team:Heidelberg/Project" style="color: white">Project</A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Project"><SPAN><SPAN>Overall Project</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Notebook/material"><SPAN><SPAN>Material &amp; Methods</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Project/Sensing"><SPAN><SPAN>Sensing</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Project/Killing_I"><SPAN><SPAN>Killing I  - Phages</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Project/Killing_II"><SPAN><SPAN>Killing II - Colicin</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Project/Visualization"><SPAN><SPAN>Visualization</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Parts" style="color: white">Parts</A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Parts"><SPAN><SPAN>Submitted Parts</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Parts/Characterization"><SPAN><SPAN>Characterization</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Modeling" style="color: white">Modeling</A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Modeling"><SPAN><SPAN>Overview</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Modeling/Chemotaxis"><SPAN><SPAN>Chemotaxis-Colicin</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Modeling/Phage"><SPAN><SPAN>Phage Dynamics model</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Notebook/Overview" style="color: white">Notebook</A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Notebook/Sensing_Group/Notebook"><SPAN><SPAN>Sensing &gt;</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Notebook/Sensing_Group/Cloning"><SPAN>Cloning strategy</SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Notebook/Sensing_Group/Notebook"><SPAN>Notebook</SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Notebook/Killing_I/Notebook"><SPAN><SPAN>Killing I - Phages &gt;</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Notebook/Killing_I/Cloning"><SPAN>Cloning strategy</SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Notebook/Killing_I/Notebook"><SPAN>Notebook</SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Notebook/Killing_II/Notebook"><SPAN><SPAN>Killing II - Colicin &gt;</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Notebook/Killing_II/Cloning"><SPAN>Cloning strategy</SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Notebook/Killing_II/Notebook"><SPAN>Notebook</SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Notebook/visualization"><SPAN><SPAN>Visualization</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Notebook/material"><SPAN><SPAN>Material &amp; Methods</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Notebook/team_meetings"><SPAN><SPAN>Team Meetings</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Notebook/seminar"><SPAN><SPAN>Seminar on Synthetic Biology</SPAN></SPAN></A></LI><LI style="width: 160px"><A href="https://2008.igem.org/Team:Heidelberg/Human_Practice/Project_Overview" style="color: white">Human Practice</A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Human_Practice/Project_Overview"><SPAN><SPAN>Project Overview</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Human_Practice/Phips_the_Phage"><SPAN><SPAN>Phips the Phage</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Human_Practice/Essay"><SPAN><SPAN>Essay</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Human_Practice/Surveys"><SPAN><SPAN>Surveys</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Human_Practice/Open_Day"><SPAN><SPAN>Open Day</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Human_Practice/Nobel_Prize"><SPAN><SPAN>Nobel Prize</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Sponsors" style="color: white">Sponsors</A></LI></DIV><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Material"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Material</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Buffers_.26_Solution"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Buffers &amp; Solution</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Kits"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Kits</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Marker"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Marker</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Enzymes"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">Enzymes</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Plasmidvectors"><SPAN class="tocnumber">1.5</SPAN><SPAN class="toctext">Plasmidvectors</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#Synthetic_oligonucleotides"><SPAN class="tocnumber">1.6</SPAN><SPAN class="toctext">Synthetic oligonucleotides</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#Phages"><SPAN class="tocnumber">1.7</SPAN><SPAN class="toctext">Phages</SPAN></A></LI><LI class="toclevel-2 tocsection-9"><A href="#Bacteria"><SPAN class="tocnumber">1.8</SPAN><SPAN class="toctext">Bacteria</SPAN></A></LI><LI class="toclevel-2 tocsection-10"><A href="#Bacteria_Growth_Media"><SPAN class="tocnumber">1.9</SPAN><SPAN class="toctext">Bacteria Growth Media</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-11"><A href="#Methods"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Methods</SPAN></A></LI><LI class="toclevel-2 tocsection-12"><A href="#Preparing_chemically_competent_cells"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Preparing chemically competent cells</SPAN></A></LI><LI class="toclevel-2 tocsection-13"><A href="#Transformation_of_bacteria"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">Transformation of bacteria</SPAN></A></LI><LI class="toclevel-2 tocsection-14"><A href="#Isolation_of_plasmid_DNA_by_alkaline_lysis_.28mini_and_maxiprep.29"><SPAN class="tocnumber">2.3</SPAN><SPAN class="toctext">Isolation of plasmid DNA by alkaline lysis (mini and maxiprep)</SPAN></A></LI><LI class="toclevel-2 tocsection-15"><A href="#DNA_amplification_using_polymerase_chain_reaction_.28PCR.29"><SPAN class="tocnumber">2.4</SPAN><SPAN class="toctext">DNA amplification using polymerase chain reaction (PCR)</SPAN></A></LI><LI class="toclevel-2 tocsection-16"><A href="#Purification_of_DNA_from_PCR_reactions"><SPAN class="tocnumber">2.5</SPAN><SPAN class="toctext">Purification of DNA from PCR reactions</SPAN></A></LI><LI class="toclevel-2 tocsection-17"><A href="#Enzymatic_hydrolysis_of_DNA_by_restriction_enzymes"><SPAN class="tocnumber">2.6</SPAN><SPAN class="toctext">Enzymatic hydrolysis of DNA by restriction enzymes</SPAN></A></LI><LI class="toclevel-2 tocsection-18"><A href="#Agarose_gel_electrophoresis_for_separating_DNA"><SPAN class="tocnumber">2.7</SPAN><SPAN class="toctext">Agarose gel electrophoresis for separating DNA</SPAN></A></LI><LI class="toclevel-2 tocsection-19"><A href="#Isolation_of_DNA_fragments_from_an_agarose_gel"><SPAN class="tocnumber">2.8</SPAN><SPAN class="toctext">Isolation of DNA fragments from an agarose gel</SPAN></A></LI><LI class="toclevel-2 tocsection-20"><A href="#Ligation_of_dsDNA_fragments"><SPAN class="tocnumber">2.9</SPAN><SPAN class="toctext">Ligation of dsDNA fragments</SPAN></A></LI><LI class="toclevel-2 tocsection-21"><A href="#Lysing_of_bacteria_by_ultrasonication"><SPAN class="tocnumber">2.10</SPAN><SPAN class="toctext">Lysing of bacteria by ultrasonication</SPAN></A></LI><LI class="toclevel-2 tocsection-22"><A href="#Glycerol_stock"><SPAN class="tocnumber">2.11</SPAN><SPAN class="toctext">Glycerol stock</SPAN></A></LI><LI class="toclevel-2 tocsection-23"><A href="#Preparation_of_plating_bacteria_for_bacteriophage_experiments"><SPAN class="tocnumber">2.12</SPAN><SPAN class="toctext">Preparation of plating bacteria for bacteriophage experiments</SPAN></A></LI><LI class="toclevel-2 tocsection-24"><A href="#Bacteriophage_.CE.BB_plaque_assay"><SPAN class="tocnumber">2.13</SPAN><SPAN class="toctext">Bacteriophage λ plaque assay</SPAN></A></LI><LI class="toclevel-2 tocsection-25"><A href="#Preparing_Stocks_of_Bacteriophage_.CE.BB"><SPAN class="tocnumber">2.14</SPAN><SPAN class="toctext">Preparing Stocks of Bacteriophage λ</SPAN></A></LI><LI class="toclevel-2 tocsection-26"><A href="#Preparing_of_.CE.BB_DNA"><SPAN class="tocnumber">2.15</SPAN><SPAN class="toctext">Preparing of λ DNA</SPAN></A></LI><LI class="toclevel-2 tocsection-27"><A href="#In_vitro_packaging_of_.CE.BB_DNA"><SPAN class="tocnumber">2.16</SPAN><SPAN class="toctext">In vitro packaging of λ DNA</SPAN></A></LI><LI class="toclevel-2 tocsection-28"><A href="#Colicin_Activity_Test"><SPAN class="tocnumber">2.17</SPAN><SPAN class="toctext">Colicin Activity Test</SPAN></A></LI><LI class="toclevel-2 tocsection-29"><A href="#Lysis_test"><SPAN class="tocnumber">2.18</SPAN><SPAN class="toctext">Lysis test</SPAN></A></LI><LI class="toclevel-2 tocsection-30"><A href="#Sender_Activity_Test"><SPAN class="tocnumber">2.19</SPAN><SPAN class="toctext">Sender Activity Test</SPAN></A></LI><LI class="toclevel-2 tocsection-31"><A href="#LuxS_Activity_Test"><SPAN class="tocnumber">2.20</SPAN><SPAN class="toctext">LuxS Activity Test</SPAN></A></LI><LI class="toclevel-2 tocsection-32"><A href="#Conjugation_test"><SPAN class="tocnumber">2.21</SPAN><SPAN class="toctext">Conjugation test</SPAN></A></LI><LI class="toclevel-2 tocsection-33"><A href="#Receptor_Polarization_Test"><SPAN class="tocnumber">2.22</SPAN><SPAN class="toctext">Receptor Polarization Test</SPAN></A></LI><LI class="toclevel-2 tocsection-34"><A href="#Swarm_Assays"><SPAN class="tocnumber">2.23</SPAN><SPAN class="toctext">Swarm Assays</SPAN></A></LI><LI class="toclevel-2 tocsection-35"><A href="#Microscopy_.E2.80.93_Chemotaxis"><SPAN class="tocnumber">2.24</SPAN><SPAN class="toctext">Microscopy – Chemotaxis</SPAN></A></LI><LI class="toclevel-2 tocsection-36"><A href="#Microscopy_.E2.80.93_Colicin_Toxicity"><SPAN class="tocnumber">2.25</SPAN><SPAN class="toctext">Microscopy – Colicin Toxicity</SPAN></A></LI><LI class="toclevel-2 tocsection-37"><A href="#Microscopy_.E2.80.93_Cell_lysis"><SPAN class="tocnumber">2.26</SPAN><SPAN class="toctext">Microscopy – Cell lysis</SPAN></A></LI><LI class="toclevel-2 tocsection-38"><A href="#Cell_Culture"><SPAN class="tocnumber">2.27</SPAN><SPAN class="toctext">Cell Culture</SPAN></A></LI><LI class="toclevel-2 tocsection-39"><A href="#Eucaryotic_Killing_Assay_with_Colicins"><SPAN class="tocnumber">2.28</SPAN><SPAN class="toctext">Eucaryotic Killing Assay with Colicins</SPAN></A></LI><LI class="toclevel-2 tocsection-40"><A href="#Flow_Cytometer_Acquisition"><SPAN class="tocnumber">2.29</SPAN><SPAN class="toctext">Flow Cytometer Acquisition</SPAN></A></LI></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Material">Material</SPAN></H2><H3><SPAN class="mw-headline" id="Buffers_.26_Solution">Buffers &amp; Solution</SPAN></H3><TABLE class="wikitable"><TBODY><TR align="center"><TD style="font-weight:bold;">SM </TD><TD> 50 mM </TD><TD> Tris, pH 7.5
</TD></TR><TR align="center"><TD> 20 mM </TD><TD> MgSO4
</TD></TR><TR align="center"><TD> 50 mM </TD><TD> NaCl
</TD></TR><TR align="center"><TD> 0.01 % </TD><TD> gelatine
</TD></TR><TR align="center"><TD style="font-weight:bold;">Tethering buffer, pH 7.0 </TD><TD> 10 mM  </TD><TD> potassium phosphate
</TD></TR><TR align="center"><TD> (K2HPO4 : KH2PO4 = 1 :1 )
</TD></TR><TR align="center"><TD> 100 µM </TD><TD> EDTA
</TD></TR><TR align="center"><TD> 1 µM </TD><TD> L-Methionine
</TD></TR><TR align="center"><TD> 10 mM </TD><TD> lactic acid
</TD></TR><TR align="center"><TD style="font-weight:bold;">M63 salt </TD><TD> 3 g/l </TD><TD> KH2PO4
</TD></TR><TR align="center"><TD> 9 g/l </TD><TD> K2HPO4
</TD></TR><TR align="center"><TD> 4 g/l </TD><TD> (NH4)2SO4
</TD></TR><TR align="center"><TD> 0.5 g/l </TD><TD> FeSO4
</TD></TR><TR align="center"><TD style="font-weight:bold;">Amino acid mix </TD><TD> 5 g/l </TD><TD> L-threonine
</TD></TR><TR align="center"><TD> 5 g/l </TD><TD> L-histidin
</TD></TR><TR align="center"><TD> 5 g/l </TD><TD> L-leucine
</TD></TR><TR align="center"><TD> 5 g/l </TD><TD> L-methionine
</TD></TR><TR align="center"><TD style="font-weight:bold;">M9 salts </TD><TD> 64 g/l </TD><TD> Na2HPO4 x 7H2O
</TD></TR><TR align="center"><TD> 15 g/l </TD><TD> KH2PO4
</TD></TR><TR align="center"><TD> 2.5 g/l </TD><TD> NaCl
</TD></TR><TR align="center"><TD> 5 g/l </TD><TD> NH4Cl
</TD></TR></TBODY></TABLE><P>[<A href="https://2008.igem.org/Team:Heidelberg/Notebook/material" class="external text" rel="nofollow">back</A>]
</P><H3><SPAN class="mw-headline" id="Kits">Kits</SPAN></H3><TABLE class="wikitable"><TBODY><TR bgcolor="grey"><TH height="20px," width="350px"> Kit </TH><TH width="350px"> supplier
</TH></TR><TR align="center"><TD style="font-weight:bold;">CompactPrep Plasmid Maxi Kit </TD><TD> Qiagen
</TD></TR><TR align="center"><TD style="font-weight:bold;">HISpeed Plasmid Maxi Kit </TD><TD> Qiagen
</TD></TR><TR align="center"><TD style="font-weight:bold;">MaxPlax™ Lambda Packaging Extracts </TD><TD> EPICENTRE Biotechnologies
</TD></TR><TR align="center"><TD style="font-weight:bold;">QIAEX II Gel Extraction Kit </TD><TD> Qiagen
</TD></TR><TR align="center"><TD style="font-weight:bold;">QIAGEN Lambda Mini Kit </TD><TD> Qiagen
</TD></TR><TR align="center"><TD style="font-weight:bold;">QIAprep Spin Mini Kit  </TD><TD> Qiagen
</TD></TR><TR align="center"><TD style="font-weight:bold;">QIAquick Gel Extraction Kit </TD><TD> Qiagen
</TD></TR><TR align="center"><TD style="font-weight:bold;">QIAquick PCR Purification Kit </TD><TD> Qiagen
</TD></TR></TBODY></TABLE><P>[<A href="https://2008.igem.org/Team:Heidelberg/Notebook/material" class="external text" rel="nofollow">back</A>]
</P><H3><SPAN class="mw-headline" id="Marker">Marker</SPAN></H3><TABLE class="wikitable"><TBODY><TR bgcolor="grey"><TH height="20px," width="350px"> Marker </TH><TH width="350px"> supplier
</TH></TR><TR align="center"><TD style="font-weight:bold;">GeneRuler™ High Range DNA Ladder </TD><TD> MBI Fermentas
</TD></TR><TR align="center"><TD style="font-weight:bold;">GeneRuler™ 1kb DNA Ladder Mix </TD><TD> MBI Fermentas
</TD></TR><TR align="center"><TD style="font-weight:bold;">GeneRuler™ 1kb Plus DNA Ladder Mix </TD><TD> MBI Fermentas
</TD></TR></TBODY></TABLE><P>[<A href="https://2008.igem.org/Team:Heidelberg/Notebook/material" class="external text" rel="nofollow">back</A>]
</P><H3><SPAN class="mw-headline" id="Enzymes">Enzymes</SPAN></H3><TABLE class="wikitable"><TBODY><TR bgcolor="grey"><TH height="20px," width="350px"> Enzym </TH><TH width="350px"> supplier
</TH></TR><TR align="center"><TD style="font-weight:bold;">Pfu DNA polymerase </TD><TD> Stratagene
</TD></TR><TR align="center"><TD style="font-weight:bold;">Pfu turbo DNA polymerase </TD><TD> Stratagene
</TD></TR><TR align="center"><TD style="font-weight:bold;">Phusion DNA polymerase </TD><TD> Finnzymes
</TD></TR><TR align="center"><TD style="font-weight:bold;">Taq DNA polymerase </TD><TD> MBI Fermentas
</TD></TR><TR align="center"><TD style="font-weight:bold;">T4 DNA ligase </TD><TD> MBI Fermentas / New England Biolabs
</TD></TR><TR align="center"><TD style="font-weight:bold;">AgeI </TD><TD> New England Biolabs
</TD></TR><TR align="center"><TD style="font-weight:bold;">BamHI </TD><TD> New England Biolabs
</TD></TR><TR align="center"><TD style="font-weight:bold;">BglI </TD><TD> MBI Fermentas
</TD></TR><TR align="center"><TD style="font-weight:bold;">BseJI </TD><TD> MBI Fermentas
</TD></TR><TR align="center"><TD style="font-weight:bold;">BspEI </TD><TD> New England Biolabs
</TD></TR><TR align="center"><TD style="font-weight:bold;">DpnI </TD><TD> Roche Diagnostics / New England Biolabs
</TD></TR><TR align="center"><TD style="font-weight:bold;">DraI </TD><TD> New England Biolabs
</TD></TR><TR align="center"><TD style="font-weight:bold;">EcoRI </TD><TD> New England Biolabs
</TD></TR><TR align="center"><TD style="font-weight:bold;">EcoRV </TD><TD> New England Biolabs
</TD></TR><TR align="center"><TD style="font-weight:bold;">HindIII </TD><TD> New England Biolabs
</TD></TR><TR align="center"><TD style="font-weight:bold;">KpnI </TD><TD> New England Biolabs
</TD></TR><TR align="center"><TD style="font-weight:bold;">NcoI </TD><TD> New England Biolabs
</TD></TR><TR align="center"><TD style="font-weight:bold;">NdeI </TD><TD> New England Biolabs
</TD></TR><TR align="center"><TD style="font-weight:bold;">NotI </TD><TD> New England Biolabs
</TD></TR><TR align="center"><TD style="font-weight:bold;">PstI </TD><TD> New England Biolabs
</TD></TR><TR align="center"><TD style="font-weight:bold;">SacI </TD><TD> New England Biolabs
</TD></TR><TR align="center"><TD style="font-weight:bold;">SalI </TD><TD> MBI Fermentas
</TD></TR><TR align="center"><TD style="font-weight:bold;">ScaI </TD><TD> New England Biolabs
</TD></TR><TR align="center"><TD style="font-weight:bold;">SfcI </TD><TD> New England Biolabs
</TD></TR><TR align="center"><TD style="font-weight:bold;">SmiI </TD><TD> MBI Fermentas
</TD></TR><TR align="center"><TD style="font-weight:bold;">SpeI </TD><TD> New England Biolabs
</TD></TR><TR align="center"><TD style="font-weight:bold;">SpeI </TD><TD> New England Biolabs
</TD></TR><TR align="center"><TD style="font-weight:bold;">XbaI </TD><TD> New England Biolabs
</TD></TR><TR align="center"><TD style="font-weight:bold;">XhoI </TD><TD> MBI Fermentas
</TD></TR><TR align="center"><TD style="font-weight:bold;">XmaI </TD><TD> New England Biolabs
</TD></TR></TBODY></TABLE><P>[<A href="https://2008.igem.org/Team:Heidelberg/Notebook/material" class="external text" rel="nofollow">back</A>]
</P><H3><SPAN class="mw-headline" id="Plasmidvectors">Plasmidvectors</SPAN></H3><TABLE class="wikitable"><TBODY><TR bgcolor="grey"><TH height="20px," width="200px"> Name </TH><TH width="250px"> application </TH><TH width="250px"> reference
</TH></TR><TR align="center"><TD style="font-weight:bold;"><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_B0015" class="external text" rel="nofollow">BBa_B0015</A></TD><TD> terminator </TD><TD><A href="http://partsregistry.org" class="external free" rel="nofollow">http://partsregistry.org</A></TD></TR><TR align="center"><TD style="font-weight:bold;"><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_F1610" class="external text" rel="nofollow">BBa_F1610</A></TD><TD> LuxI </TD><TD><A href="http://partsregistry.org" class="external free" rel="nofollow">http://partsregistry.org</A></TD></TR><TR align="center"><TD style="font-weight:bold;"><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_I15030" class="external text" rel="nofollow">BBa_I15030</A></TD><TD> AI-1 amplifier </TD><TD><A href="http://partsregistry.org" class="external free" rel="nofollow">http://partsregistry.org</A></TD></TR><TR align="center"><TD style="font-weight:bold;"><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_I20260" class="external text" rel="nofollow">BBa_I20260</A></TD><TD> GFP </TD><TD><A href="http://partsregistry.org" class="external free" rel="nofollow">http://partsregistry.org</A></TD></TR><TR align="center"><TD style="font-weight:bold;"><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_J01003" class="external text" rel="nofollow">BBa_J01003</A></TD><TD> oriT </TD><TD><A href="http://partsregistry.org" class="external free" rel="nofollow">http://partsregistry.org</A></TD></TR><TR align="center"><TD style="font-weight:bold;"><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_J16002" class="external text" rel="nofollow">BBa_J16002</A></TD><TD> cloning vector </TD><TD><A href="http://partsregistry.org" class="external free" rel="nofollow">http://partsregistry.org</A></TD></TR><TR align="center"><TD style="font-weight:bold;"><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_J23107" class="external text" rel="nofollow">BBa_J23107</A></TD><TD> constitutive promotor </TD><TD><A href="http://partsregistry.org" class="external free" rel="nofollow">http://partsregistry.org</A></TD></TR><TR align="center"><TD style="font-weight:bold;"><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_T9002" class="external text" rel="nofollow">BBa_T9002</A></TD><TD> AI-1 GFP receiver </TD><TD><A href="http://partsregistry.org" class="external free" rel="nofollow">http://partsregistry.org</A></TD></TR><TR align="center"><TD style="font-weight:bold;">CheY-mCherry </TD><TD> cloning vector </TD><TD> V. Sourjik, ZMBH
</TD></TR><TR align="center"><TD style="font-weight:bold;">ColE1 </TD><TD> colicin E1 </TD><TD> DSMZ
</TD></TR><TR align="center"><TD style="font-weight:bold;">ColE9-J </TD><TD> colicin E9 </TD><TD> C. Kleanthous, University of York
</TD></TR><TR align="center"><TD style="font-weight:bold;">pBad18 </TD><TD> cloning vector </TD><TD> V. Sourjik, ZMBH
</TD></TR><TR align="center"><TD style="font-weight:bold;">pBad33 </TD><TD> cloning vector </TD><TD> V. Sourjik, ZMBH
</TD></TR><TR align="center"><TD style="font-weight:bold;">pBluescript II SK (+) </TD><TD> cloning vector </TD><TD> V. Sourjik, ZMBH
</TD></TR><TR align="center"><TD style="font-weight:bold;">pDest15  </TD><TD> cloning vector </TD><TD> DKFZ Library
</TD></TR><TR align="center"><TD style="font-weight:bold;">pDK4 </TD><TD> visualization </TD><TD> V. Sourjik, ZMBH
</TD></TR><TR align="center"><TD style="font-weight:bold;">pDK48 </TD><TD> cloning vector </TD><TD> V. Sourjik, ZMBH
</TD></TR><TR align="center"><TD style="font-weight:bold;">pDK58 </TD><TD> cloning vector </TD><TD> V. Sourjik, ZMBH
</TD></TR><TR align="center"><TD style="font-weight:bold;">pDK6 </TD><TD> visualization </TD><TD> V. Sourjik, ZMBH
</TD></TR><TR align="center"><TD style="font-weight:bold;">pED374 </TD><TD> oriT </TD><TD> K. Derbyshire, Wadsworth
</TD></TR><TR align="center"><TD style="font-weight:bold;">pES16 </TD><TD> cloning vector </TD><TD> V. Sourjik, ZMBH
</TD></TR><TR align="center"><TD style="font-weight:bold;">pMMB863 </TD><TD> oriT </TD><TD> M. Bagdasarian, MSU
</TD></TR><TR align="center"><TD style="font-weight:bold;">pQE-30 </TD><TD> cloning vector </TD><TD> Invitrogen
</TD></TR><TR align="center"><TD style="font-weight:bold;">pSB1A2 </TD><TD> cloning vector </TD><TD><A href="http://partsregistry.org" class="external free" rel="nofollow">http://partsregistry.org</A></TD></TR><TR align="center"><TD style="font-weight:bold;">pSB2K3 </TD><TD> cloning vector </TD><TD><A href="http://partsregistry.org" class="external free" rel="nofollow">http://partsregistry.org</A></TD></TR><TR align="center"><TD style="font-weight:bold;">pTrc99a </TD><TD> cloning vector </TD><TD> V. Sourjik, ZMBH
</TD></TR><TR align="center"><TD style="font-weight:bold;">pUB307 </TD><TD> helper plasmid </TD><TD> E. Lanka, BfR
</TD></TR><TR align="center"><TD style="font-weight:bold;">RP4 </TD><TD> helper plasmid </TD><TD> M. Bagdasarian, MSU
</TD></TR></TBODY></TABLE><P>[<A href="https://2008.igem.org/Team:Heidelberg/Notebook/material" class="external text" rel="nofollow">back</A>]
</P><H3><SPAN class="mw-headline" id="Synthetic_oligonucleotides">Synthetic oligonucleotides</SPAN></H3><P>All oligonucleotides were purchased from Invitrogen (Karlsruhe) and adjusted to a standard concentration of 100 <SUP>pmol</SUP>/<SUB>µl</SUB>.
</P><TABLE class="wikitable"><TBODY><TR bgcolor="grey"><TH height="20px," width="250px"> Name </TH><TH width="450px"> SEQUENCE (5´-&gt;3´)
</TH></TR><TR align="center"><TD style="font-weight:bold;">Bam_fw </TD><TD> GACAAGTGTTGGCCATGGAACAGG
</TD></TR><TR align="center"><TD style="font-weight:bold;">Bam_rv </TD><TD> GCCGTCTGTGATGGCTTCCATG
</TD></TR><TR align="center"><TD style="font-weight:bold;">cI_mut_fw </TD><TD> GCGTCTGGGTGGTGATGAGTTCACCTTCAAAAAACTG
</TD></TR><TR align="center"><TD style="font-weight:bold;">cI_mut_rv </TD><TD> CAGTTTTTTGAAGGTGAACTCATCACCACCCAGACGC
</TD></TR><TR align="center"><TD style="font-weight:bold;">CmR_EcoRI_mut_fw </TD><TD> GAATGCTCATCCGGAGTTCCGTATGGCAATG
</TD></TR><TR align="center"><TD style="font-weight:bold;">CmR_EcoRI_mut_rev </TD><TD> CATTGCCATACGGAACTCCGGATGAGCATTC
</TD></TR><TR align="center"><TD style="font-weight:bold;">CmR_fw </TD><TD> GCTAAAATGGAGAAAAAAATCACTGG
</TD></TR><TR align="center"><TD style="font-weight:bold;">CmR_new_fw </TD><TD> TACGAGGTACCTTTACAGCTAGCTCAGTCCTAGGTATTATGC
</TD></TR><TR align="center"><TD style="font-weight:bold;">CmR_new_rev </TD><TD> TATATAAGCTTTTACGCCCCGCCCTGCCACTCATCGCAGTACTGTTG
</TD></TR><TR align="center"><TD style="font-weight:bold;">CmR_Prefix_fw </TD><TD> GAATTCGCGGCCGCTTCTAGAGTTTACAGCTAGCTCAGTCCTAGG
</TD></TR><TR align="center"><TD style="font-weight:bold;">CmR_rv </TD><TD> AGGTTCTCCTTTATTAGCCGGATCCTCTAGATTACGCC
</TD></TR><TR align="center"><TD style="font-weight:bold;">CmR_Suffix_rv </TD><TD> CTGCAGCGGCCGCTACTAGTATATAAACGCAGAAAGGCCCACCC
</TD></TR><TR align="center"><TD style="font-weight:bold;">colE1_kil_prot_rv_A_SpeI </TD><TD> TATATACTAGTACTACTGAACCGCGATCCCCG
</TD></TR><TR align="center"><TD style="font-weight:bold;">colE1_mut_EcoRI_fw </TD><TD> GGTATTGCTATTGTTACAGGTATTCTATGCTCCTATATTGATAAG
</TD></TR><TR align="center"><TD style="font-weight:bold;">colE1_mut_EcoRI_rv </TD><TD> CTTATCAATATAGGAGCATAGAATACCTGTAACAATAGCAATACC
</TD></TR><TR align="center"><TD style="font-weight:bold;">colE1_mut_PstI_1_fw </TD><TD> GCAGTAAAAGTGAAAGTTCAGCAGCTATTCATGCAACTGC
</TD></TR><TR align="center"><TD style="font-weight:bold;">colE1_mut_PstI_1_rv </TD><TD> GCAGTTGCATGAATAGCTGCTGAACTTTCACTTTTACTGC
</TD></TR><TR align="center"><TD style="font-weight:bold;">colE1_mut_PstI_2_fw </TD><TD> GCTGCCCGGGCAAAAGCAGCAGCGGAAGCACAGG
</TD></TR><TR align="center"><TD style="font-weight:bold;">colE1_mut_PstI_2_rv </TD><TD> CCTGTGCTTCCGCTGCTGCTTTTGCCCGGGCAGC
</TD></TR><TR align="center"><TD style="font-weight:bold;">colE1_mut_PstI_3_fw </TD><TD> CATTAGAGAAGAAAGCAGCAGATGCAGGGGTGAG
</TD></TR><TR align="center"><TD style="font-weight:bold;">colE1_mut_PstI_3_rv </TD><TD> CTCACCCCTGCATCTGCTGCTTTCTTCTCTAATG
</TD></TR><TR align="center"><TD style="font-weight:bold;">colE1_prot_fw_BamH1 </TD><TD> TACGAGGATCCATGGAAACCGCGGTAGCG
</TD></TR><TR align="center"><TD style="font-weight:bold;">colE1_prot_rv_HindIII </TD><TD> TATATAAGCTTTTAAATCCCTAACACCTC
</TD></TR><TR align="center"><TD style="font-weight:bold;">colE9_lysProt_rv_A_SpeI </TD><TD> TATATACTAGTACTAGGTTTTCGGCTTAGAACCCC
</TD></TR><TR align="center"><TD style="font-weight:bold;">colE9_mut_EcoRI_fw </TD><TD> GTTGGGTGGACGATTCGAGAGTTCAATGGGGAAATAAAAATG
</TD></TR><TR align="center"><TD style="font-weight:bold;">colE9_mut_EcoRI_rv </TD><TD> CATTTTTATTTCCCCATTGAACTCTCGAATCGTCCACCCAAC
</TD></TR><TR align="center"><TD style="font-weight:bold;">colE9_plasmid_rv_A_SpeI </TD><TD> TATATACTAGTACACATGGAACTTTTGTGAC
</TD></TR><TR align="center"><TD style="font-weight:bold;">colE9_prot_fw_BamH1 </TD><TD> TACGAGGATCCATCGATTTGCCCATGACCC
</TD></TR><TR align="center"><TD style="font-weight:bold;">colE9_prot_rv_XmaI </TD><TD> TATATCCCGGGTTACTTACCTCGGTGAATATCG
</TD></TR><TR align="center"><TD style="font-weight:bold;">DK13 </TD><TD> TCACCCGCACGCGC</TD></TR><TR align="center"><TD style="font-weight:bold;">DK167 </TD><TD> AGGATACTAGTAGGCCATTACTTT
</TD></TR><TR align="center"><TD style="font-weight:bold;">DK9b </TD><TD> CGATGCGGCCGCTCAAAATGTTTCCCAGTTTGG
</TD></TR><TR align="center"><TD style="font-weight:bold;">F1610_fw_XbaI </TD><TD> TACGATCTAGAAAAGAGGAGAAATACTAG
</TD></TR><TR align="center"><TD style="font-weight:bold;">F1610_rv_HindIII </TD><TD> TATATAAGCTTTATAAACGCAGAAAGGCCC
</TD></TR><TR align="center"><TD style="font-weight:bold;">GAM_fw </TD><TD> AGTGCTTTAGCGTTAACTTCCG
</TD></TR><TR align="center"><TD style="font-weight:bold;">GAM_rv </TD><TD> GGTTTTACCGCATACCAATAACG
</TD></TR><TR align="center"><TD style="font-weight:bold;">GFP_CmR_fw </TD><TD> CTCGTTGGTACCTCTAGATTTACAGCTAGCTCAGTCCTAGG
</TD></TR><TR align="center"><TD style="font-weight:bold;">GFP_CmR_rv </TD><TD> TATTCGACCGGTACTAGTTATAAACGCAGAAAGGCCCACC
</TD></TR><TR align="center"><TD style="font-weight:bold;">GFP_new_fw </TD><TD> TACGAGAGCTCTTTACAGCTAGCTCAGTCCTAGG
</TD></TR><TR align="center"><TD style="font-weight:bold;">GFP_new_rv </TD><TD> TATATACCGGTACTAGTTATAAACGCAGAAAGGCCCACC
</TD></TR><TR align="center"><TD style="font-weight:bold;">Lambda_insert_fw </TD><TD> TTGTAAAAACAGCCCTCCTC
</TD></TR><TR align="center"><TD style="font-weight:bold;">Lambda_insert_rv </TD><TD> GATATGACTATCAAGGCCGC
</TD></TR><TR align="center"><TD style="font-weight:bold;">LuxP_mut_F </TD><TD> CGTGAATTAGCAACAGAGTTCGGAAAGTTCTTCCC
</TD></TR><TR align="center"><TD style="font-weight:bold;">LuxP_mut_R </TD><TD> GGGAAGAACTTTCCGAACTCTGTTGCTAATTCACG
</TD></TR><TR align="center"><TD style="font-weight:bold;">LuxP_prefix_F </TD><TD> GAGGGAGAATTCGCGGCCGCTTCTAGATGAAGAAAGCGTTACTATTTTC
</TD></TR><TR align="center"><TD style="font-weight:bold;">LuxP_sufffix_R </TD><TD> GGAGAGCTGCAGCGGCCGCTACTAGTAATTATCTGAATATCTAAATGCG
</TD></TR><TR align="center"><TD style="font-weight:bold;">LuxPc </TD><TD> ATTACGCGGCCGCAGGAAACAGACCATGAAGAAAGCGTTACTATTTTCCC
</TD></TR><TR align="center"><TD style="font-weight:bold;">LuxPd </TD><TD> GTAATGTCGACTCAATTATCTGAATATC
</TD></TR><TR align="center"><TD style="font-weight:bold;">LuxP-seq-FW </TD><TD> CCCGTCCTGCCAGTGAGC
</TD></TR><TR align="center"><TD style="font-weight:bold;">LuxQa </TD><TD> ATCGACCATGGGCAATAAATTTCGCTTAGC
</TD></TR><TR align="center"><TD style="font-weight:bold;">LuxQc </TD><TD> GTAATGGATCCTTAGTGGAGGCTTGAGCC
</TD></TR><TR align="center"><TD style="font-weight:bold;">LuxQTar_1a </TD><TD> CACAAATCATTGCCAATGAACGTATGTTGCTTACTCCGCTGG
</TD></TR><TR align="center"><TD style="font-weight:bold;">LuxQTar_1b </TD><TD> CCAGCGGAGTAAGCAACATACGTTCATTGGCAATGATTTGTG
</TD></TR><TR align="center"><TD style="font-weight:bold;">LuxQTar_2a </TD><TD> CTTAGCGACCATGAGCCATGAGTTTGCCCAGTGGCAACTGGC
</TD></TR><TR align="center"><TD style="font-weight:bold;">LuxQTar_2b </TD><TD> GCCAGTTGCCACTGGGCAAACTCATGGCTCATGGTCGCTAAG
</TD></TR><TR align="center"><TD style="font-weight:bold;">LuxS mutBbaIR </TD><TD> CCACACCCACTTCTAGGATGTTCTTCGCGATTTGC
</TD></TR><TR align="center"><TD style="font-weight:bold;">LuxS mutXbaIF </TD><TD> GCAAATCGCGAAGAACATCCTAGAAGTGGGTGTGG
</TD></TR><TR align="center"><TD style="font-weight:bold;">LuxS_mut_fw </TD><TD> CATCCTTTCTGAGAAAGGCATTCATACATTAGAGC
</TD></TR><TR align="center"><TD style="font-weight:bold;">LuxS_mut_rev </TD><TD> GCTCTAATGTATGAATGCCTTTCTCAGAAAGGATG
</TD></TR><TR align="center"><TD style="font-weight:bold;">LuxS_prefix_F </TD><TD> GGAGAGGAATTCGCGGCCGCTTCTAGATGGGCAATGCACCAGCGGTTCG
</TD></TR><TR align="center"><TD style="font-weight:bold;">luxS_suffix_R </TD><TD> GAGGGACTGCAGCGGCCGCTACTAGTAGTCGATGCGTAGCTCTCTCAGC
</TD></TR><TR align="center"><TD style="font-weight:bold;">LuxSa </TD><TD> ATCAGTCCATGGGCAATGCACCAGCGGTTCG
</TD></TR><TR align="center"><TD style="font-weight:bold;">LuxSb </TD><TD> GTAATGGATCCTTAGTCGATGCGTAGC
</TD></TR><TR align="center"><TD style="font-weight:bold;">mut_insert_fw </TD><TD> CTGAGGGGACGGTACCTCTACATTTACAGCTAGCTCAG
</TD></TR><TR align="center"><TD style="font-weight:bold;">mut_insert_rv </TD><TD> CTGAGCTAGCTGTAAATGTAGAGGTACCGTCCCCTCAG
</TD></TR><TR align="center"><TD style="font-weight:bold;">mut_insert2_fw </TD><TD> GGCAGGCGGGGCGTAATCTATAGGATCCGGCTAATAAAGG
</TD></TR><TR align="center"><TD style="font-weight:bold;">mut_insert2_rv </TD><TD> CCTTTATTAGCCGGATCCTATAGATTACGCCCCGCCTGCC
</TD></TR><TR align="center"><TD style="font-weight:bold;">mut_kpn1_pBlue </TD><TD> CGAGGGGGGGCCCGGTTCCCAATTCGCCCTATAG
</TD></TR><TR align="center"><TD style="font-weight:bold;">mut_kpn1_pBlue </TD><TD> CTATAGGGCGAATTGGGAACCGGGCCCCCCCTCG
</TD></TR><TR align="center"><TD style="font-weight:bold;">oriT_pre </TD><TD> GAATTCGCGGCCGCTTCTAGAGGACAGGCTCATGCCGGCCGC
</TD></TR><TR align="center"><TD style="font-weight:bold;">oriT_RP4_fw </TD><TD> CTCGTTTCTAGAACTAGTGACAGGCTCATGCCGGCCGC
</TD></TR><TR align="center"><TD style="font-weight:bold;">oriT_RP4_rv </TD><TD> TATTCGGGTACCGTCCCCTCAGTTCAGTAATTTCCTGC
</TD></TR><TR align="center"><TD style="font-weight:bold;">oriT_suf </TD><TD> CTGCAGCGGCCGCTACTAGTAGTCCCCTCAGTTCAGTAATTTCCTGC
</TD></TR><TR align="center"><TD style="font-weight:bold;">T9002_Lux_pR_rv_SpeI_BamHI_RBS </TD><TD> TATATACTAGTGGATCCGGTTCTGTTTCCTCTCTAGTATTTATTCGAC
</TD></TR><TR align="center"><TD style="font-weight:bold;">T9002_LuxpR_Not_Eco_Xba_G_fw </TD><TD> TACGAGAATTCGCGGCCGCTTCTAGAGTCCCTATCAGTGATAGAGATTG
</TD></TR><TR align="center"><TD style="font-weight:bold;">Term_new_fw </TD><TD> TACGAAAGCTTCCAGGCATCAAATAAAACGAAAGG
</TD></TR><TR align="center"><TD style="font-weight:bold;">Term_new_rv </TD><TD> TATATGAGCTCTATAAACGCAGAAAGGCCCACCC
</TD></TR><TR align="center"><TD style="font-weight:bold;">VF2 </TD><TD> TGCCACCTGACGTCTAAGAA
</TD></TR><TR align="center"><TD style="font-weight:bold;">VIC121 </TD><TD> TTTATCGCAACTCTCTACTG
</TD></TR><TR align="center"><TD style="font-weight:bold;">VIC122 </TD><TD> CTGATTTAATCTGTATCAGG
</TD></TR><TR align="center"><TD style="font-weight:bold;">VIC131 </TD><TD> ATGTGTGGAATTGTGAGCGG
</TD></TR><TR align="center"><TD style="font-weight:bold;">VIC132 </TD><TD> CTGATTTAATCTGTATCAGG
</TD></TR><TR align="center"><TD style="font-weight:bold;">VR </TD><TD> ATTACCGCCTTTGAGTGAGC
</TD></TR></TBODY></TABLE><P>[<A href="https://2008.igem.org/Team:Heidelberg/Notebook/material" class="external text" rel="nofollow">back</A>]
</P><H3><SPAN class="mw-headline" id="Phages">Phages</SPAN></H3><TABLE class="wikitable"><TBODY><TR bgcolor="grey"><TH height="20px," width="200px"> Name </TH><TH width="250px"> application </TH><TH width="250px"> reference
</TH></TR><TR align="center"><TD style="font-weight:bold;">Lambda cI mut </TD><TD> cI deleted </TD><TD> W. Reiser, ZMBH
</TD></TR><TR align="center"><TD style="font-weight:bold;">Lambda cI857 </TD><TD> heat inducible </TD><TD> MBI Fermentas
</TD></TR></TBODY></TABLE><P>[<A href="https://2008.igem.org/Team:Heidelberg/Notebook/material" class="external text" rel="nofollow">back</A>]
</P><H3><SPAN class="mw-headline" id="Bacteria">Bacteria</SPAN></H3><TABLE class="wikitable"><TBODY><TR bgcolor="grey"><TH height="20px," width="200px"> E.coli strain </TH><TH width="250px"> usage </TH><TH width="250px"> reference
</TH></TR><TR align="center"><TD style="font-weight:bold;">DH5a </TD><TD> amplification of plasmids </TD><TD> Invitrogen
</TD></TR><TR align="center"><TD style="font-weight:bold;">HCB33 </TD><TD> swarm assays </TD><TD> V. Sourjik, ZMBH
</TD></TR><TR align="center"><TD style="font-weight:bold;">MG1655 </TD><TD> swarm assays </TD><TD> V. Sourjik, ZMBH
</TD></TR><TR align="center"><TD style="font-weight:bold;">TOP10 </TD><TD> amplification of plasmids </TD><TD> Invitrogen
</TD></TR><TR align="center"><TD style="font-weight:bold;">UU1250 </TD><TD> chemotaxis receptor knock out </TD><TD> V. Sourjik, ZMBH
</TD></TR></TBODY></TABLE><P>[<A href="https://2008.igem.org/Team:Heidelberg/Notebook/material" class="external text" rel="nofollow">back</A>]
</P><H3><SPAN class="mw-headline" id="Bacteria_Growth_Media">Bacteria Growth Media</SPAN></H3><TABLE class="wikitable"><TBODY><TR align="center"><TD style="font-weight:bold;">Luria Broth (LB) </TD><TD> 10 <SUP>g</SUP>/<SUB>l</SUB></TD><TD> tryptone
</TD></TR><TR align="center"><TD> 5 <SUP>g</SUP>/<SUB>l</SUB></TD><TD> yeast extract
</TD></TR><TR align="center"><TD> 10 <SUP>g</SUP>/<SUB>l</SUB></TD><TD> NaCl
</TD></TR><TR align="center"><TD style="font-weight:bold;">Luria Broth (LB) plus </TD><TD> 10 <SUP>g</SUP>/<SUB>l</SUB></TD><TD> tryptone
</TD></TR><TR align="center"><TD> 5 <SUP>g</SUP>/<SUB>l</SUB></TD><TD> yeast extract
</TD></TR><TR align="center"><TD> 20 <SUP>g</SUP>/<SUB>l</SUB></TD><TD> NaCl
</TD></TR><TR align="center"><TD style="font-weight:bold;">TB </TD><TD> 10 <SUP>g</SUP>/<SUB>l</SUB></TD><TD> bacto tryptone
</TD></TR><TR align="center"><TD> 5 <SUP>g</SUP>/<SUB>l</SUB></TD><TD> NaCl
</TD></TR><TR align="center"><TD style="font-weight:bold;">Standard I </TD><TD> 15.6 <SUP>g</SUP>/<SUB>l</SUB></TD><TD> peptone P
</TD></TR><TR align="center"><TD> 2.8 <SUP>g</SUP>/<SUB>l</SUB></TD><TD> yeast extract
</TD></TR><TR align="center"><TD> 5.6 <SUP>g</SUP>/<SUB>l</SUB></TD><TD> NaCl
</TD></TR><TR align="center"><TD> 1 <SUP>g</SUP>/<SUB>l</SUB></TD><TD> D-glucose
</TD></TR><TR align="center"><TD style="font-weight:bold;">Minimal medium </TD><TD> 9.8 % </TD><TD> M63 salts
</TD></TR><TR align="center"><TD> 0.2 %</TD><TD> glycerol
</TD></TR><TR align="center"><TD> 0.1 <SUP>g</SUP>/<SUB>l</SUB></TD><TD> Thiamine
</TD></TR><TR align="center"><TD> 1 mM </TD><TD> MgSO<SUB>4</SUB></TD></TR><TR align="center"><TD> 0.8 % </TD><TD> amino acid mix
</TD></TR><TR align="center"><TD style="font-weight:bold;">M9 </TD><TD> 20 % </TD><TD> M9 salts
</TD></TR><TR align="center"><TD> 2 mM </TD><TD> MgSO<SUB>4</SUB></TD></TR><TR align="center"><TD> 0.4 %</TD><TD> glucose
</TD></TR><TR align="center"><TD> 0.1 mM </TD><TD> CaCl<SUB>2</SUB></TD></TR></TBODY></TABLE><P>
For cultivation of phages the media were supplied with 0.2 % maltose and 10 mM MgSO4. For preparation of agar plates 15 g/l agar, for preparation of top agar 7 g/l agar was added prior the autoclaving. For selection of resistant bacteria following antibiotics were added during cooling of at agar at about 50 °C: 
</P><P>[<A href="https://2008.igem.org/Team:Heidelberg/Notebook/material" class="external text" rel="nofollow">back</A>]
</P><TABLE class="wikitable"><TBODY><TR bgcolor="grey"><TH height="20px," width="200px"> Antibiotic </TH><TH width="250px"> concentration
</TH></TR><TR align="center"><TD style="font-weight:bold;">Ampicillin </TD><TD> 100 µg/ml
</TD></TR><TR align="center"><TD style="font-weight:bold;">Chloramphenicol </TD><TD> 35 µg/ml
</TD></TR><TR align="center"><TD style="font-weight:bold;">Kanamycin </TD><TD> 50 µg/ml
</TD></TR></TBODY></TABLE><P>The same concentrations of antibiotics were used for selection of resistant in media.
</P><P>[<A href="https://2008.igem.org/Team:Heidelberg/Notebook/material" class="external text" rel="nofollow">back</A>]
</P><H2><SPAN class="mw-headline" id="Methods">Methods</SPAN></H2><H3><SPAN class="mw-headline" id="Preparing_chemically_competent_cells">Preparing chemically competent cells</SPAN></H3><P>First, a 20 ml over night culture was inoculated in antibiotic free LB medium from a fresh single colony and transferred into 400 ml antibiotic free LB medium the next day. This culture was incubated at 37 °C while shacking until an OD600 of 0.5 – 0.6 was achieved. The culture was than cooled down on ice, centrifuged (8 min, 4 °C, 3500 rpm), the supernatant discarded and the pellet resuspended in 10 ml 100 mM CaCl<SUB>2</SUB>. After addition of further 190 ml 100 mM CaCl<SUB>2</SUB> the suspension was incubated on ice for 30 min. The suspension was than again centrifuged (8 min, 4 °C, 3500 rpm), the supernatant discarded, the pellet resuspended in 20 ml 82.5 mM CaCl2 with 17.5 % glycerol and aliquoted. The aliquots were flash frozen in liquid nitrogen and than stored at -80 °C until usage.
[<A href="https://2008.igem.org/Team:Heidelberg/Notebook/material" class="external text" rel="nofollow">back</A>]
</P><H3><SPAN class="mw-headline" id="Transformation_of_bacteria">Transformation of bacteria</SPAN></H3><P>For enrichment of vectors, <I>E .coli</I> TOP10 and DH5α were used. For the transformation 100 µl of the competent cells were thawed on ice and 50 – 200 ng DNA solution added (depending on the concentration of the DNA solution). After a 20 minute incubation on ice, cells were made permeable for the DNA by heat shocking for 45 seconds at 42 °C and a further 2 minute incubation on ice. The samples were than rescued by adding 250µl preheated antibiotic free LB-medium and incubated for one hour at 37 °C while shacking for induction of the antibiotic resistance. The selection for plasmid containing and therefore antibiotic resistant bacteria was conducted by plating them on antibiotic containing LB-agar plates.
[<A href="https://2008.igem.org/Team:Heidelberg/Notebook/material" class="external text" rel="nofollow">back</A>]
</P><H3><SPAN class="mw-headline" id="Isolation_of_plasmid_DNA_by_alkaline_lysis_.28mini_and_maxiprep.29">Isolation of plasmid DNA by alkaline lysis (mini and maxiprep)</SPAN></H3><P>For analysis of ligations and transformations QIAprep Spin Kits (Qiagen, Hilden) were used following the manufacturer instructions.
For miniprep a single colony was picked from a LB-agar plate or glycerol stock was used to inoculate 5 ml LB-medium with appropriate antibiotic for selection (100 µg/µl ampicillin, 50 µg/µl kanamycin, 35 µg/µl chloramphenicol). Bacteria were grown over night at 37 °C while shaking (200 rpm). By using 4 ml over night culture with this kit the yield was around 6-10 µg. 
For maxipreps the Qiagen CompactPrep Plasmid Maxi Kit was used following the instructions given by the instruction manual. In this case 250 ml LB-medium were inoculated and used for preparation of plasmid DNA. The routinely yield was 300-400 µg plasmid DNA. Purity and amount of DNA was analysed using a NanoDrop.
[<A href="https://2008.igem.org/Team:Heidelberg/Notebook/material" class="external text" rel="nofollow">back</A>]
</P><H3><SPAN class="mw-headline" id="DNA_amplification_using_polymerase_chain_reaction_.28PCR.29">DNA amplification using polymerase chain reaction (PCR)</SPAN></H3><P>By using PCR smallest amount of DNA can be detected and amplified. The principle of PCR is the selective amplification of any region of the DNA. Nevertheless, the sequence at both ends must be known for the binding of two complementary primers. The DNA region of interest is than amplified exponentially while the reproduction of the DNA takes place in three temperature stpes: denaturation of parental DNA at high temperature (95-98 °C), hybridisation of primers (58-65 °C) and DNA elongation (72 °C). 
For the amplification a heat-stable DNA polymerase I with a 3’-5’ exonuclease activity (proof reading) is used such as Pfu (from Pyrococcus furiosus) and Phusion (a Pyrocuccus-like enzyme) or the non proof reading enzyme Taq (from Thermus aquaticus).
Phsuion polymerase was used in a 2x master mix adding only primers (20pmol each) and DNA template (~10ng) or alternatively colonies from agar plates (1-5 colonies) in a final volume of 50 µl. For taq and Pfu polymerase following reaction batch was commonly used:
</P><TABLE class="wikitable"><TBODY><TR align="center"><TD style="font-weight:bold;"> 1 µl   template (10 ng) / colonies
</TD></TR><TR align="center"><TD style="font-weight:bold;"> 2 µl   primer 1 (10 pmol/µl)
</TD></TR><TR align="center"><TD style="font-weight:bold;"> 2 µl   primer 2 (10 pmol/µl)
</TD></TR><TR align="center"><TD style="font-weight:bold;"> 1 µl   polymerase (2,5 Units)
</TD></TR><TR align="center"><TD style="font-weight:bold;"> 1 µl   dNTP mix (10 mM)
</TD></TR><TR align="center"><TD style="font-weight:bold;"> 5 µl   10x buffer
</TD></TR><TR align="center"><TD style="font-weight:bold;">38 µl   dH2O
</TD></TR></TBODY></TABLE><P>The PCR procedure was as follows:
</P><TABLE class="wikitable"><TBODY><TR align="center"><TD style="font-weight:bold;">Initiale denaturation </TD><TD> 95- 98 °C, 3-5 min </TD><TD> 1 cycle
</TD></TR><TR align="center"><TD style="font-weight:bold;">denaturation </TD><TD> 95-98 °C, 15 sec - 1 min
</TD></TR><TR align="center"><TD style="font-weight:bold;">annealing </TD><TD> 58-65 °C, 15 sec - 1 min </TD><TD> 25-28 cycles
</TD></TR><TR align="center"><TD style="font-weight:bold;">elongation </TD><TD> 72 °C, 15 sec - 3 min
</TD></TR><TR align="center"><TD style="font-weight:bold;">termination </TD><TD> 72 °C, 5 – 10 min </TD><TD> 1 cycle
</TD></TR><TR align="center"><TD> 4 °C </TD><TD> forever
</TD></TR></TBODY></TABLE><P>For site directed mutagenesis PCR Pfu turbo polymerase (Stratagene) was used in the same reaction batch described above. The temperature program was as follows:
</P><TABLE class="wikitable"><TBODY><TR align="center"><TD style="font-weight:bold;">initiale denaturation </TD><TD> 95 °C, 30 sec </TD><TD> 1 cycle
</TD></TR><TR align="center"><TD style="font-weight:bold;">denaturation </TD><TD> 95 °C, 30 sec
</TD></TR><TR align="center"><TD style="font-weight:bold;">annealing </TD><TD> 55 °C, 1 min </TD><TD> 16 cycles
</TD></TR><TR align="center"><TD style="font-weight:bold;">elongation </TD><TD> 68 °C, 2 min per kb of plasmid
</TD></TR><TR align="center"><TD style="font-weight:bold;">termination </TD><TD> 68 °C, 10 min </TD><TD> 1 cycle
</TD></TR><TR align="center"><TD> 4 °C </TD><TD> forever
</TD></TR></TBODY></TABLE><P>For point mutagenesis primers with around 33 bases were used having the base to be altered in the middle. The melting temperature of these primers was always over 78 °C. If this melting temperature could not be achieved using 15 unchanged nucleotides at both sites the flanking arms were enlarged properly. After the PCR reaction 10 units of DpnI was added directly to the PCR tube, mixed and incubated for 1-5 h at 37 °C. DpnI digests the parental methylated plasmid DNA leaving only the mutated one. After purification using QIAquick PCR Purification Kit the plasmid was transformed as described above. 
[<A href="https://2008.igem.org/Team:Heidelberg/Notebook/material" class="external text" rel="nofollow">back</A>]
</P><H3><SPAN class="mw-headline" id="Purification_of_DNA_from_PCR_reactions">Purification of DNA from PCR reactions</SPAN></H3><P>PCR products were purified by the QIAquick PCR Purification Kit from Qiagen following the instructions of the Qiagen Handbook. To check the purity and amount of extracted DNA an aliquot was analysed using a NanoDrop.
[<A href="https://2008.igem.org/Team:Heidelberg/Notebook/material" class="external text" rel="nofollow">back</A>]
</P><H3><SPAN class="mw-headline" id="Enzymatic_hydrolysis_of_DNA_by_restriction_enzymes">Enzymatic hydrolysis of DNA by restriction enzymes</SPAN></H3><P>The restriction digest of DNA was used for analysis of purified DNA form mini or maxiprep or for isolation of specific DNA fragments for further cloning. Analytical digestions were routinely conducted in 20 µl volume. In all digestions a minimum of 2 Units restriction enzyme(s) was used per microgram DNA. Optimised buffer conditions were secured by using NEB buffer system. The final reaction volume was achieved by adding H<SUB>2</SUB>O dest. and the sample was incubated at optimal temperature for the restriction enzyme(s) (normally 37 °C). Preparative digestions were conducted in a volume of 50 µl. For analysis and preparation DNA loading dye was added to the samples and they were loaded on a agarose gel or alternatively purified by QIAquick PCR Purification Kit from Qiagen.
[<A href="https://2008.igem.org/Team:Heidelberg/Notebook/material" class="external text" rel="nofollow">back</A>]
</P><H3><SPAN class="mw-headline" id="Agarose_gel_electrophoresis_for_separating_DNA">Agarose gel electrophoresis for separating DNA</SPAN></H3><P>In the agarose gel electrophoresis a mixture of DNA fragments with different sizes are separated in an electrical field by their size. This is achieved by moving the negatively charged DNA through an agarose matrix while shorter fragments will run faster. The size of the pores can be controlled by agarose concentration. The higher the agarose concentration the smaller the pores are and the smaller fragments can be separated. Agarose concentrations between 0.7 and 1.5 % agarose in 0.5x TE buffer were used. 
The agarose was dissolved completely by heating up and 0.1 µg/ml ethidium bromide was added. 
The DNA fragments were separated using a constant voltage between 80 and 130 V. Under UV light (λ = 254 nm) DNA is visible through the unspecific intercalated ethidium bromide and can be documented or cut out and extracted from the gel.
[<A href="https://2008.igem.org/Team:Heidelberg/Notebook/material" class="external text" rel="nofollow">back</A>]
</P><H3><SPAN class="mw-headline" id="Isolation_of_DNA_fragments_from_an_agarose_gel">Isolation of DNA fragments from an agarose gel</SPAN></H3><P>Plasmid DNA and DNA fragments were extracted using the Gel Extraction Kit from Qiagen following the manufacture instructions. To check the purity and amount of extracted DNA an aliquot was analysed using a NanoDrop.
[<A href="https://2008.igem.org/Team:Heidelberg/Notebook/material" class="external text" rel="nofollow">back</A>]
</P><H3><SPAN class="mw-headline" id="Ligation_of_dsDNA_fragments">Ligation of dsDNA fragments</SPAN></H3><P>T4 DNA ligase was used to form covalent phosphodiesterbonds between doublestranded DNA fragments having blunt or compatible sticky ends. For the ligation the restricted vector -DNA and the insert were mixed 1:3 or 1:5 in a total ligationvolume of 20 µl. This mixture was incubated in ligation buffer using 5 Weiss units T4 DNA ligase over night at 16 °C or at room temperature for 20 minutes and afterwards used to transform chemical competent cells. 
[<A href="https://2008.igem.org/Team:Heidelberg/Notebook/material" class="external text" rel="nofollow">back</A>]
</P><H3><SPAN class="mw-headline" id="Lysing_of_bacteria_by_ultrasonication">Lysing of bacteria by ultrasonication</SPAN></H3><P>For the lysis of bacteria an ultrasonic tip was used. The 10-15 ml of bacteria over night cultures were sonicated with an ultrasonic tip three time for 15 seconds. After sonication cell extract was observed under microscope.
[<A href="https://2008.igem.org/Team:Heidelberg/Notebook/material" class="external text" rel="nofollow">back</A>]
</P><H3><SPAN class="mw-headline" id="Glycerol_stock">Glycerol stock</SPAN></H3><P>To store bacteria for long term glycerol stocks were used. Therefore 1 ml of an over night culture were added to 150 µl of 80 % Glycerol into a cryo tube, vortexed and incubated at room temperature for 30 min. Afterwards the glycerole stock was stored at -80 °C.
[<A href="https://2008.igem.org/Team:Heidelberg/Notebook/material" class="external text" rel="nofollow">back</A>]
</P><H3><SPAN class="mw-headline" id="Preparation_of_plating_bacteria_for_bacteriophage_experiments">Preparation of plating bacteria for bacteriophage experiments</SPAN></H3><P>A overnight culture of the appropriate <I>E. coli</I> strain was grown in LB medium containing 10 mM MgSO<SUB>4</SUB> and 0.2 % maltose at 30 °C to reduce the amount of cell debris in the medium. The added maltose leads to a substantial induction of the maltose operon including the lamb gene, which encodes the cell surface receptor to which bacteriophage λ binds. After harvesting the cells they were resuspended in 10 mM MgSO<SUB>4</SUB> and diluted to a final concentration of 2.0 OD<SUB>600</SUB>. The suspension of plating bacteria was stored at 4 °C for up to 1 week.
[<A href="https://2008.igem.org/Team:Heidelberg/Notebook/material" class="external text" rel="nofollow">back</A>]
</P><H3><SPAN class="mw-headline" id="Bacteriophage_.CE.BB_plaque_assay">Bacteriophage λ plaque assay</SPAN></H3><P>Tenfold serial dilutions of the bacteriophage stocks were prepared, 100 µl of it mixed with the same amount of plating bacteria, incubated for 20 minutes at 37 °C to allow the bacteriophage particles to adsorb to the bacteria, this mixture added to 3 ml molten top agar which was kept liquid at 48 °C and the entire contend poured onto a agar plate. After harden of the top agar the inverted plates were incubated at 37 °C over night. On the next day plaques could be counted.
[<A href="https://2008.igem.org/Team:Heidelberg/Notebook/material" class="external text" rel="nofollow">back</A>]
</P><H3><SPAN class="mw-headline" id="Preparing_Stocks_of_Bacteriophage_.CE.BB">Preparing Stocks of Bacteriophage λ</SPAN></H3><P>Completely lysed agarose plates were prepared by mixing 0.1 ml plating bacteria with ~10<SUP>5</SUP> pfu, incubation for 20 minutes at 37 °C and that bringing out the mixture in 3 ml molten top agarose. Agarose is preferred in this protocol in order to minimize contaminations with polyanionic inhibitors that can interfere with subsequent enzymatic analysis of the bacteriophage DNA. The plates were incubated 12-16 hours without inversion at 37 °C. Plates were not inverted during incubation to encourage sweating of fluid onto the surface of the dish, which allows the bacteriophage to spread more easily. The plates were than overlayed with 5 ml SM and incubated while shaking at 4 °C over night. The SM was that transferred to a tube and the plate washed again with 1 ml SM. 2 % v/v chloroform was added and bacterial debris removed by centrifugation at 10,000 g for 10 minutes at 4 °C. The concentration of infectious particles in the stock was measured by plaque assays as described above.
[<A href="https://2008.igem.org/Team:Heidelberg/Notebook/material" class="external text" rel="nofollow">back</A>]
</P><H3><SPAN class="mw-headline" id="Preparing_of_.CE.BB_DNA">Preparing of λ DNA</SPAN></H3><P>Lambda DNA was prepared using the QIAGEN® Lambda mini kit following the manufacture instructions.
[<A href="https://2008.igem.org/Team:Heidelberg/Notebook/material" class="external text" rel="nofollow">back</A>]
</P><H3><SPAN class="mw-headline" id="In_vitro_packaging_of_.CE.BB_DNA">In vitro packaging of λ DNA</SPAN></H3><P>In vitro packaging was conducted using the MaxPlax lambda packaging extracts from Epicentre Biotechnologies following the manufacture instructions. 
[<A href="https://2008.igem.org/Team:Heidelberg/Notebook/material" class="external text" rel="nofollow">back</A>]
</P><H3><SPAN class="mw-headline" id="Colicin_Activity_Test">Colicin Activity Test</SPAN></H3><P>To measure the killing efficiency and which amount of cells or colicins are needed to reach any killing activity a colicin activity test was carried out. Therefore bacteria containing the colicin plasmid (<I>E. coli</I> TOP10 or <I>E. coli</I> MG1655) and GFP producing cells (reference promoter <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_I20260" class="external text" rel="nofollow">BBa_I20260</A>, <I>E. coli</I> TOP10) were inoculated in TB-media with appropriated antibiotics at 37 °C for 4 to 6 hours and the optical densities of the two strains were adapted. The colicin cells themselves, their produced supernatant or the supernatant of the lysed colicin cells were added in different ratios to a constant amount of GFP producing cells. The total volume was kept constant and the missing amount added with TB-media without antibiotics. The colicin production was induced by several concentrations (0 M-100 nM) of N-Acyl-Homoserin-Lactone (AHL). The OD and GFP intensities were measured at 37 °C in the Tecan Microplate Reader every half an hour for about 12 hours.
As a negative control the similar test was carried out with cells, containing the same plasmid without the colicin operon on it. 
[<A href="https://2008.igem.org/Team:Heidelberg/Notebook/material" class="external text" rel="nofollow">back</A>]
</P><H3><SPAN class="mw-headline" id="Lysis_test">Lysis test</SPAN></H3><P>Lysis of killercells dependent on the AHL concentration was measured due to a lysis test. Therefore <I>E. coli</I> TOP 10 containing the luxR-colicinE1 part (<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K150009" class="external text" rel="nofollow">BBa_K150009</A>) and a GFP producing part (<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_I20260" class="external text" rel="nofollow">BBa_I20260</A>) were inoculated in TB-media with appropriate antibiotics at 37 °C for 4 to 6 hours. As reference <I>E. coli</I> TOP 10 cells containing a luxR part (comparable to <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_F2620" class="external text" rel="nofollow">BBa_F2620</A>) instead of the luxR-colicinE1 part (<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K150009" class="external text" rel="nofollow">BBa_K150009</A>). For the lysis measurement 250 µl of the liquid cultures were diluted with 250 µl TB-media. The Lux pR promoter was activated with AHL concentrations from 0 M up to 50 nM. The optical densities and GFP intensities were measured in Tecan Microplate Reader every half-hourly for about 12 hours.
[<A href="https://2008.igem.org/Team:Heidelberg/Notebook/material" class="external text" rel="nofollow">back</A>]
</P><H3><SPAN class="mw-headline" id="Sender_Activity_Test">Sender Activity Test</SPAN></H3><P>The sender activity test was performed to quantify the AHL production of the cells containing the sender part (<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K150000" class="external text" rel="nofollow">BBa_K150000</A>). Therefore TB-media with the appropriated antibiotic was inoculated with 2 µl/ml of an overnight culture. For 7 hours every hour the OD of the medium was measured and 5 ml of the culture were spin down to produce supernatant. The sterile filtered supernatant, still containing the AHL, was stored at 4 °C. The supernatant of every hour was then added in different ratios to a constant amount of AHL inducible cells which produce GFP after induction. The OD and GFP intensities were measured at 37 °C in the Tecan Microplate Reader every half an hour for about 12 - 24 hours. The same test was carried out with amplifier cells (<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_I15030" class="external text" rel="nofollow">BBa_I15030</A>). To calculate the produced amount of AHL by sender or amplifier cells as reference the test were performed also with different AHL concentration (0 M-100 nM) instead of AHL producing cells.
[<A href="https://2008.igem.org/Team:Heidelberg/Notebook/material" class="external text" rel="nofollow">back</A>]
</P><H3><SPAN class="mw-headline" id="LuxS_Activity_Test">LuxS Activity Test</SPAN></H3><P>This protocol for the functional check AI-2 production is mainly adapted from a paper of Vanderleyden et al. [1] , for further questions please refer to it. 10ml ONC DH5α strain with LuxS (S) and 10ml DH5α strain with ptrc99α as blank (B) were incubated in LB to a volume of 50mL each at 37°C up to an OD of 0.35 in 100ml erlenmeyer flasks. Then S and B each were induced with 500 µM IPTG for 3h, following centrifugation at maximum speed for 20min. The supernatants were then transferred in two new falcons by pipetting or decanting. Optional the supernatant was sterile filtrated and/or 10 g/L NaCl was added to a final concentration of ~20g/L. 10 mL ONC of <I>V. harveyi</I> reporter (BB120 in LB plus at 30 °C) was incubated in LB plus to a final volume of 50 mL at 37°C up to an OD of 0.35 in a 100 mL erlenmeyer flask. Supernatant S and B were both mixed with <I>Vibrio</I> cells at ratios 1:3, 1:5 and 1:10 each to a final volume of 5 mL in 50 mL falcon tubes and incubated for 30min at 30°C. Subsequently the cells were incubated again for 3h, OD and Luminescence were measured with a microplate reader. [<A href="https://2008.igem.org/Team:Heidelberg/Notebook/material" class="external text" rel="nofollow">back</A>]
</P><P>[1] Sigrid C. J. DeKeersmaecker and Jos Vanderleyden ‘Constraints on detection of autoinducer-2 (AI-2) signalling molecules using Vibrio harveyi as a reporter’, Microbiology 2003 149: 1953-1956.
</P><H3><SPAN class="mw-headline" id="Conjugation_test">Conjugation test</SPAN></H3><P>Equal amounts of donor (cells harbouring the oriT containing plasmid and pUB307) and acceptor cells from overnight cultures were centrifuged (2 min, 13000 rpm), the pellet washed twice with 1.5 ml LB medium without antibiotics, the pellets combined and this mixture resuspended in 100 µl LB medium without antibiotics. This cell mix was transferred to a filter membrane on a LB agar plate and incubated here for a time range from 0 to 42 minutes in 6 minutes steps at 37 °C. The membrane was then transferred in 1 ml LB medium, vortexed, the medium diluted and plated out on LB agar plates with proper antibiotics for plasmid harbouring the oriT and antibiotic resistance of acceptor bacteria. On the next day the number of transconjugants was counted and therewith the conjugation efficiency calculated as ratio of cell density of transconjugants to cell density of donor cells depending on the reaction time.
[<A href="https://2008.igem.org/Team:Heidelberg/Notebook/material" class="external text" rel="nofollow">back</A>]
</P><H3><SPAN class="mw-headline" id="Receptor_Polarization_Test"> Receptor Polarization Test </SPAN></H3><P>To check the functionality of the fusion receptor constructs each construct, on pDK48, was transformed into strain UU1250 together with pDK36 (YFP-CheA). The expression of the receptor was induced with 0.01% arabinose, of YFP-CheA with 30µM IPTG. Polarization was tested by putting 10 µL of the cells on a 0.5% agarose pad and making snapshots using a Zeiss Axio Imager.Z1 fluorescent microscope.
</P><H3><SPAN class="mw-headline" id="Swarm_Assays">Swarm Assays</SPAN></H3><P>The sensitivity of the LuxP-tar fusion constructs to AI-2 was tested in swarm assays. For this two different strains were used: <I>E. coli</I> UU1250 and <I>E. coli</I> HCB33. The assays were performed in TB medium for UU1250 and TB or minimal medium for HCB33 (0.3% agar). As chemo attractant either 3 µl of an AI-2 producing ONC, induced with 100 nM IPTG, or 10 µl supernatant from those cells was spotted 13 times on the plate to create a center line. Gradient building was allowed for 12-16 hours at 4 °C. Afterwards the fusion receptor expressing cells (0.01% arabinose induced overnight) as well as non-chemotactic reference strains were spotted at a distance of about 3-3.5 cm to the attractant line. Swarm plates were incubated at 30 °C and pictures taken after 24, 48 and 72 hours.
[<A href="https://2008.igem.org/Team:Heidelberg/Notebook/material" class="external text" rel="nofollow">back</A>]
</P><H3><SPAN class="mw-headline" id="Microscopy_.E2.80.93_Chemotaxis">Microscopy – Chemotaxis</SPAN></H3><P>An Applied Precision - DeltaVision wide field microscope was used to visualize chemotactic behaviour of the LuxP-tar fusion. <I>E. coli</I> UU1250 and <I>E. coli</I> HCB33 were used for these measurements. The monitoring was performed with Labtek<SUP>TM</SUP> chambered coverglass systems (Nunc) which were filled with 2 ml of TB media, Tethering Buffer and minimal media containing 0.2 % agar. As chemo attractant either 3 µl of an AI-2 producing ONC, induced with 100 nM IPTG, or 10 µl supernatant from those cells was spotted at the one side of the chamber. Gradient building was allowed for 12-16 hours at 4 °C.
Afterwards either fusion receptor expressing cells (0.01% arabinose induced overnight) or non-chemotactic reference strains were spotted at the center of the chamber. The chamber was then incubated for several hours at 30 °C. After 3 hours frame pictures were taken, using confocal fluorescence imaging and 8x6 frames. (8 pictures in length, 6 in height)
Images were processed and auto arranged with housemade ImageJ plugins. Matlab was used for the quantitative analysis.
[<A href="https://2008.igem.org/Team:Heidelberg/Notebook/material" class="external text" rel="nofollow">back</A>]
</P><H3><SPAN class="mw-headline" id="Microscopy_.E2.80.93_Colicin_Toxicity">Microscopy – Colicin Toxicity</SPAN></H3><P>The killing efficiency of colicin producing cells was monitored using an Applied Precision - DeltaVision wide field microscope. Therefore a prey and a killer strain were designed in <I>E. coli</I> TOP10. The prey strain contained a GFP producing part (<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_I20260" class="external text" rel="nofollow">BBa_I20260</A>) as well as the luxI sender part (<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K150000" class="external text" rel="nofollow">BBa_K150000</A>) and the killer strain contained as marker a mCherry producing part as well as the LuxR-colicinE1-receiver (<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K150009" class="external text" rel="nofollow">BBa_K150009</A>).
Measurements were prepared with 1 ml of a prey ONC and 1 ml of a killer ONC in 3 ml TB media containing 0,01% arabinose and 10 nM AHL for induction of colicin production. A second probe in which cells without colicin genes were used instead of the killer cells was prepared in parallel. Both samples were incubated for 6-7 hours at 37 °C.
Monitoring was performed on the Deltavision microscope with 8-well Labtek<SUP>TM</SUP> chambered coverglass systems (Nunc). For that 6 µl of each sample were distributed equaly in 200 µl TB media. Frame pictures have been taken using 8x8 frames evenly distributed over the whole chamber.
Images were processed and auto arranged with housemade ImageJ plugins.
[<A href="https://2008.igem.org/Team:Heidelberg/Notebook/material" class="external text" rel="nofollow">back</A>]
</P><H3><SPAN class="mw-headline" id="Microscopy_.E2.80.93_Cell_lysis">Microscopy – Cell lysis</SPAN></H3><P>To visualize the lysis of prey and killer cells cultures were observed on Zeiss Axio Imager.Z1 microscope. The killer strain (<I>E. coli</I> TOP 10) contained the luxR-colicinE1 part (<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K150009" class="external text" rel="nofollow">BBa_K150009</A>) and a mCherry producing part and the prey strain contained the senderpart (<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K150000" class="external text" rel="nofollow">BBa_K150000</A>) and a GFP producing part (<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_I20260" class="external text" rel="nofollow">BBa_I20260</A>). Both strains were inoculated in TB-media with appropriate antibiotics at 37 °C for 4 to 6 hours. Thereafter OD was adjusted and the killer strain was induced by 25 nM AHL. After 2 hour 200 µl of the induced killer culture and 200 µl of the prey culture were combined and centrifuged down. For microscopy pelleted cells were diluted with 10 µl and plated on a 0.5% agarosepad. Fixed positions were observed half hourly in the mCherry and GFP channel with a 100x oil immersion objective.
[<A href="https://2008.igem.org/Team:Heidelberg/Notebook/material" class="external text" rel="nofollow">back</A>]
</P><H3><SPAN class="mw-headline" id="Cell_Culture">Cell Culture</SPAN></H3><P>MCF-7 cells (breast cancer cell line) wt or stably expressing cytoplasmic GFP, were plated in 24 well plates (150 thousand cells per well) with 10 % FCS (fetal calf serum) DMEM (Dulbecco's Modified Eagle Medium) and incubated overnight at 37 °C, 5 % CO2.
E. coli TOP 10 strains containing LuxR-colicin E1-receiver (BBa_K150009) and strains containing LuxR-receiver (no colicin) were grown overnight in TB media with appropriate antibiotics at 37 °C to an OD of 1.0 and diluted in the morning to 0.1. Once bacterial cells reached an OD of 0.35 they were harvested and inoculated on the MCF-7 wt and on MCF-7-GFP cells.
[<A href="https://2008.igem.org/Team:Heidelberg/Notebook/material" class="external text" rel="nofollow">back</A>]
</P><H3><SPAN class="mw-headline" id="Eucaryotic_Killing_Assay_with_Colicins">Eucaryotic Killing Assay with Colicins</SPAN></H3><P>Once MCF-7 cells were confluent (12h after passage), E. coli TOP 10 bacteria were spinned down (1 ml of OD = 0.35) and resuspended with DMEM medium from the original wells containing the MCF-7 wt/MCF-7-GFP cells. A 24-well plate was used containing MCF-7  cells stained with Propidium Iodide (PI) alone, with LuxR-receiver cells not containing colicin and LuxR-receiver-colicin E1 cells. The second 24-well plate with MCF-7GFP was processed as described for wt MCF-7, but without any PI staining. AHL was added to every well to a final concentration of 5 nM. Well content was collected at different time-points (0 – 6 h) and when necessary, PI was added to the suspension 15 minutes before Flow Cytometer analysis. 
[<A href="https://2008.igem.org/Team:Heidelberg/Notebook/material" class="external text" rel="nofollow">back</A>]
</P><H3><SPAN class="mw-headline" id="Flow_Cytometer_Acquisition">Flow Cytometer Acquisition</SPAN></H3><P>At the referred timepoints, supernatants were collected and spinned down. Eukaryotic cells were detached by trypsinization (200 μl) for 10 minutes at 37 °C and then resuspended in 800 μl of KH + 1 % BSA (Krebs Henseleit medium with 1 % Bovine Serum Albumin). After resuspending, well content was added to the precipitated supernatant to account for floating dead cells. PI was added to each sample (1:1000) and incubated at RT for 15 minutes before acquisition. PI positive cells and GFP expressing cells (%) were measured for 3000 cells for each condition. Histograms and fluorescent measurements were obtained using a Cytomics FC 500 MPL from Beckman&amp;Coulter.
[<A href="https://2008.igem.org/Team:Heidelberg/Notebook/material" class="external text" rel="nofollow">back</A>]
</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2008.igem.org/Team:Heidelberg/Notebook/material">http://2008.igem.org/Team:Heidelberg/Notebook/material</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Heidelberg/Notebook/material" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Heidelberg/Notebook/material" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Heidelberg/Notebook/material&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Heidelberg/Notebook/material&amp;oldid=106500" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2008.igem.org:Privacy_policy" title="2008.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2008.igem.org:General_disclaimer" title="2008.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>