<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Heidelberg_Project_Sensing"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:Heidelberg/Project/Sensing">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Heidelberg/Project/Sensing&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/Project/Sensing&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/Project/Sensing&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Heidelberg/Project/Sensing" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Heidelberg/Project/Sensing</H1><DIV id="bodyContent"><H3 id="siteSub">From 2008.igem.org</H3><DIV class="MenuBar" id="navi"><UL><LI><A href="https://2008.igem.org/Team:Heidelberg" style="color: white">Home
			</A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Team" style="color: white">Team</A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Team"><SPAN><SPAN>Overview</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Team_Members#Advisors"><SPAN><SPAN>Advisors</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Team_Members#Undergraduates"><SPAN><SPAN>Undergraduates</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Team#There_are_many_Universities_-_and_there_is_Heidelberg_..."><SPAN><SPAN>University</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Team#DKFZ"><SPAN><SPAN>DKFZ</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Team#Bioquant"><SPAN><SPAN>BioQuant</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/BioRegioRN"><SPAN><SPAN>BioRegion Rhein-Neckar</SPAN></SPAN></A></LI></UL><LI><A href="https://2008.igem.org/Team:Heidelberg/Project" style="color: white">Project</A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Project"><SPAN><SPAN>Overall Project</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Notebook/material"><SPAN><SPAN>Material &amp; Methods</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Project/Sensing"><SPAN><SPAN>Sensing</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Project/Killing_I"><SPAN><SPAN>Killing I  - Phages</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Project/Killing_II"><SPAN><SPAN>Killing II - Colicin</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Project/Visualization"><SPAN><SPAN>Visualization</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Parts" style="color: white">Parts</A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Parts"><SPAN><SPAN>Submitted Parts</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Parts/Characterization"><SPAN><SPAN>Characterization</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Modeling" style="color: white">Modeling</A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Modeling"><SPAN><SPAN>Overview</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Modeling/Chemotaxis"><SPAN><SPAN>Chemotaxis-Colicin</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Modeling/Phage"><SPAN><SPAN>Phage Dynamics model</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Notebook/Overview" style="color: white">Notebook</A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Notebook/Sensing_Group/Notebook"><SPAN><SPAN>Sensing &gt;</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Notebook/Sensing_Group/Cloning"><SPAN>Cloning strategy</SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Notebook/Sensing_Group/Notebook"><SPAN>Notebook</SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Notebook/Killing_I/Notebook"><SPAN><SPAN>Killing I - Phages &gt;</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Notebook/Killing_I/Cloning"><SPAN>Cloning strategy</SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Notebook/Killing_I/Notebook"><SPAN>Notebook</SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Notebook/Killing_II/Notebook"><SPAN><SPAN>Killing II - Colicin &gt;</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Notebook/Killing_II/Cloning"><SPAN>Cloning strategy</SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Notebook/Killing_II/Notebook"><SPAN>Notebook</SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Notebook/visualization"><SPAN><SPAN>Visualization</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Notebook/material"><SPAN><SPAN>Material &amp; Methods</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Notebook/team_meetings"><SPAN><SPAN>Team Meetings</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Notebook/seminar"><SPAN><SPAN>Seminar on Synthetic Biology</SPAN></SPAN></A></LI><LI style="width: 160px"><A href="https://2008.igem.org/Team:Heidelberg/Human_Practice/Project_Overview" style="color: white">Human Practice</A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Human_Practice/Project_Overview"><SPAN><SPAN>Project Overview</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Human_Practice/Phips_the_Phage"><SPAN><SPAN>Phips the Phage</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Human_Practice/Essay"><SPAN><SPAN>Essay</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Human_Practice/Surveys"><SPAN><SPAN>Surveys</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Human_Practice/Open_Day"><SPAN><SPAN>Open Day</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Human_Practice/Nobel_Prize"><SPAN><SPAN>Nobel Prize</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Sponsors" style="color: white">Sponsors</A></LI></DIV><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Project_-_Sensing"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Project - Sensing</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Abstract"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Abstract</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Introduction"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Introduction</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Results"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Results</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Discussion"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">Discussion</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#References"><SPAN class="tocnumber">1.5</SPAN><SPAN class="toctext">References</SPAN></A></LI></UL></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Project_-_Sensing">Project - Sensing</SPAN></H2><H3><SPAN class="mw-headline" id="Abstract"> Abstract </SPAN></H3><DIV class="thumb tright"><DIV class="thumbinner" style="width:322px;"><DIV class="thumbcaption"><B>Fig. 1</B>: Schema of Sensing Project. &quot;Prey&quot; cells produce Autoinducer synthase LuxS and secrete Autoinducer AI-2. The AI-2 diffuses through the medium and can bind to the LuxQ-Tar chimeric receptor of the &quot;killer&quot; cells. Binding of AI-2 triggers chemotaxis toward the prey cell.</DIV></DIV></DIV><P>Bacteria have the ability to sense gradients of chemoeffectors in their environment by specific receptor molecules which are present in the bacteria membrane. Our aim was to take advantage of bacterial chemotaxis in order to generate a &quot;killer&quot; strain which is able to detect and finally kill a &quot;prey&quot; strain by chemotaxis induced movement towards it. For this purpose we cloned the LuxS gene of Vibrio harveyi in an expression plasmid. LuxS encodes the Autoinducer-2 synthase, an enzyme that is essential for the production of the chemoattractant Autoinducer-2 (AI-2). We wanted to transform the &quot;prey&quot; cells with this expression plasmid in order to establish a concentration gradient of AI-2, which can be sensed by the &quot;killer&quot; strain. To make the &quot;killer&quot; strain sensitive towards AI-2, we generated two different chimeric receptors consisting each of the periplasmic ligand-binding domain of LuxQ, the natural receptor for AI-2, and the cytoplasmic part of the chemotaxis receptor (Tar). To allow proper folding and signal transduction we included either the second transmembrane domain of the LuxQ receptor or the Tar receptor in the chimeric receptor sequence. In addition to the chimeric receptor sequence we wanted to co-express from the same plasmid also LuxP, a cofactor which is essential for AI-2 ligand binding of the LuxQ receptor. We were successful in the cloning of all constructs mentioned above. The chimeric receptors were expressed in transformed bacteria as controlled by receptor fusions to the yellow fluorescent protein YFP. To our surprise none of the chimeric receptor constructs showed a polarized localization as expected for chemotaxis receptors. We were also not able to detect chemotactic activity in the transformed bacteria. Future experiments with sequence optimized receptor fusions will have to show if AI-2 is able to induce Tar-based chemotaxis.
[<A href="https://2008.igem.org/Team:Heidelberg/Project/Sensing" class="external text" rel="nofollow">back</A>]
</P><H3><SPAN class="mw-headline" id="Introduction"> Introduction </SPAN></H3><P>Chemotaxis enables bacteria to swim towards or away from a concentration gradient of chemoeffector molecules, which can be either chemoattractants or repellants. The best understood and characterized model for the investigation of chemotaxis is that of E.coli. It is known that the presence of an attractant can cause E.coli to swim (run) towards the source by the flagellar rotation in a counterclockwise manner. This swimming is interupted by short phases of tumbling caused by the clockwise rotation of the flagellar motor. The swimming direction is chosen randomly, but swimming towards the attractanct source leads to the shortening of the tumbling periods thereby resulting in a positive movement towards the attractant. The Tar receptor which is also used in our studies represents one of the most abundant chemotaxis receptor molecules of E. coli. It usually can be activated by aspartate binding. The receptor activation leads to the induction of signaling events that involve as in all signal transduction systems the phosphorylation and dephosphorylation of regulatory target proteins. In case of the Tar receptor, upon ligand binding the adaptor protein CheW and CheA, a histidine kinase, are recruited to the cytoplasmic domain of the receptor. While attractants inhibit the autophosphorylation of the effector protein CheA, the presence of repellants promote this process and cause the rapid transfer of the phosphoryl group to the protein CheY. Phosphorylated CheY then initiates after diffusion to the flagellar motor the change in flagellar motion orientation leading to bacteria tumbling due to a clockwise flagellar motion. Reajustment of the system is efficiently controlled by the phosphatase CheZ which can quickly remove the phosphoryl group from CheY. Adaptation of the system is mediated by two proteins called CheR and CheB (Sourjik, 2004). For a schematic presentation of the chemotaxis signaling in E.coli see figure 1. 
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:552px;"><DIV class="thumbcaption"><B>Fig. 2</B>: Chemotaxis signalling. Chemoattractants or repellents are sensed by transmembrane receptors. The signal is further transduced by CheA (A) via phosphorylation of CheY (Y). Phosphorylated CheY (CheY-P) diffuses to the flagellar motor and mediates clockwise rotation, resulting in cell tumbling. CheY-P is rapidly dephosphorylated by CheZ. [Sourjik, 2004].</DIV></DIV></DIV></DIV><P>[<A href="https://2008.igem.org/Team:Heidelberg/Project/Sensing" class="external text" rel="nofollow">back</A>]
</P><P>Many bacteria use a mechanism called quorum sensing for the coordination of their gene expression in a population according to cell density. This allows bacteria to synchronize their behavior and lets them act to a certain extend similar to multicellular organisms [Waters &amp; Bassler, 2005; Vendeville, 2005].
</P>
A major role in this process play small diffusible molecules, called Autoinducers (AI), which are secreted by the cells and can be detected by other bacteria to sense cell density. There are two major signaling molecules involved in quorum-sensing, AI-1 and AI-2. AI-1 is a acyl-homoserine lactone and AI- 2 is a furanosyl-borate-diester [Sun, 2004]. For the chemical structure of these molecules see figure 3 and 4.<DIV class="thumb tright"><DIV class="thumbinner" style="width:152px;"><DIV class="thumbcaption"><B>Fig.3</B>: Core structure of AI-1 [Waters &amp; Bassler, 2005]</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:152px;"><DIV class="thumbcaption"><B>Fig. 4</B>: Structure of AI-2 from V. harveyi. [Waters &amp; Bassler, 2005]</DIV></DIV></DIV> While AI-1 was found to be responsible for the intraspecies communication, i.e. within one species, AI-2 which is shared by both gram-negative and gram-positive bacteria, allows cross-talk among different species (interspecies communication)[DeKeersmaecker, 2006]. 
<P>
The main objective of our project is to engineer a so called killer strain of E.coli  which would be able to swim by chemotaxis towards a “prey” strain which secretes a specific chemoattractant. We want to link quorum sensing features of receptor molecules to chemotactic activity of bacteria.  In more detail, we want to generate chimeric receptors composed of the ligand binding domain of LuxQ, a receptor involved in quorum sensing via Autoinducer-2 (AI-2) in Vibrio bacteria fused to the cytoplasmic domain of the chemotaxis receptor Tar. Both receptors share structural similarities: a very small cytoplasmic domain is followed by the first transmembrane domain, TM1, and the periplasmic domain, which is a less conserved region responsible for ligand binding. This domain is followed by a second transmembrane domain, TM2 and finally the cytoplasmic domain to which interacting proteins are recruited. 
</P><P>To ensure the proper folding and chemotactic activity of our chimeric receptors we will engineer two different fusion sequences: 
The first one (Fusion 1) will contain the complete N-terminal part of the LuxQ receptor until the end of its second transmembrane domain fused to the cytoplasmic domain (a.a. 213-553) of the Tar receptor. The second (Fusion 2) will consist of the first transmembrane domain and the periplasmic ligand binding domain of LuxQ followed by the second transmembrane (a.a.189-212) and the cytoplasmic domain of the Tar receptor (for illustration of chimeric receptors see <A href="/Team:Heidelberg/Notebook/Sensing_Group/Cloning" title="Team:Heidelberg/Notebook/Sensing Group/Cloning"> cloning strategy</A>).
</P><P>
These chimeric receptors will then be expressed in our “killer” cells together with LuxP, a protein which is needed for the proper binding of AI-2 to the receptor. The LuxP gene will be cloned from the Vibrio harveji genome. The presence of the chimeric receptors will allow the killer cells to sense concentration gradients of AI-2 established by the “prey” cells.  In this context the “prey” cells will be transformed with an expression plasmid encoding LuxS, an the Autoinducer-2 synthase essential for the synthesis of AI-2. We will analyse the correct receptor localization by the expression of fusions of the two different chimeric receptors to a fluorescent protein. In this context we will also analyse the receptor functionality. In parallel we want to test the functionality of our &quot;prey-killer&quot; system by swarm assays. Once we have established this system, we will perform 4D microscopy to quantitatively characterize our &quot;prey-killer&quot; system in a time resolved manner.
</P><P>[<A href="https://2008.igem.org/Team:Heidelberg/Project/Sensing" class="external text" rel="nofollow">back</A>]
</P><H3><SPAN class="mw-headline" id="Results"> Results </SPAN></H3><DIV class="thumb tright"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption"><B>Fig. 5</B>: Fusion-PCR. Gelelectrophoresis showed correct length of the DNA constructs.</DIV></DIV></DIV><P>We successfully cloned both chimeric constructs, F1 and F1, into our expression plasmid pDK48 in HCB33 or pBAD33 in UU1250 strain. In addition to our chimeric receptor sequence this plasmid encoded also LuxP, the protein which is essential for AI-2 binding of the LuxQ receptor. In order to verify the expression of the chimeric receptor proteins and to control the correct localization of the receptor molecules in our prey cells, we transfomed DH5α cells with the chimera constructs fused to the yellow fluorescent protein YFP. Expression of this construct was verified by a detectable fluorecence signal in the transformed cells. However, when tested in swarm assays the transformed bacteria showed no chemotactic activity. This was observed for both the UU1250 strain and the HCB33 strain. That the chimeric receptors did not show any chemotactic activity was further confirmed by polarisation tests. Therefore we coexpressed the unlabeled receptors in UU1250 cells together with YFP-CheA. The CheA fusion was thereby expected to localize to the cytoplasmic part of the receptors and subsequently fluorescent signals should be visible at the cell membrane [Bourret, 1993; Morrison &amp; Parkinson]. Nevertheless, no membrane localization of fluorescence could be detected. The reason remains unclear, but we assume that either the cytoplasmic domain of the receptor was not able to bind YFP-CheA properly or the receptor did not localize to the membrane at all. We have further performed kinase-activity tests, but these also did not show any positive results. [<A href="https://2008.igem.org/Team:Heidelberg/Project/Sensing" class="external text" rel="nofollow">back</A>]
</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption"><B>Fig. 6</B>: Fluorescence Image of Fusion1-YFP</DIV></DIV></DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption"><B>Fig. 7</B>: Fluorescence Image of Fusion2-YFP</DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Discussion"> Discussion </SPAN></H3><P>Our aim was to generate a new chemotactic receptor chimera consisting of parts of the LuxQ receptor and the well studied Tar receptor. This kind of receptor would allow bacteria to sense gradients of the autoinducer molecule AI-2 and swim towards the gradient due to the Tar receptor part. In order to allow the correct folding and activation of the tar receptor in the cytoplasm we included either the second transmembrane domain of the LuxQ receptor or the Tar receptor. We could conclude from our expression tests in different bacteria that both chimeric receptors were expressed from the transformed plasmids. Nevertheless none of the chimeras showed a polarization at one pole of the bacteria cells. This finding was confirmed by our swarm assays where the transformed bacteria did not show any chemotactic movement towards the source of AI-2 while control bacteria did. We can only speculate that our chimeric receptor is not correctly inserted and located within the bacteria membrane due to the presence of the periplasmic domain of LuxQ. We plan to mutate the linker sequences between the LuxQ sensing domain and the Tar receptor transmembrane part, which would allow a proper folding and functioning of the receptor molecules. If we are thereby able to ensure the proper localization of the chimeric receptor molecules, the next step would be to analyze the receptors with respect to chemotactaxis of our killer cells towards our prey cells by swarm plate assays and 4D microscopy. [<A href="https://2008.igem.org/Team:Heidelberg/Project/Sensing" class="external text" rel="nofollow">back</A>]
</P><H3><SPAN class="mw-headline" id="References"> References </SPAN></H3><P>[Waters &amp; Bassler, 2005] Christopher M. Waters, and Bonnie L. Bassler: <B>Quorum Sensing: Cell-to-Cell Communication in Bacteria</B>, Annu. Rev. Cell Dev. Biol. 21, 319–346, 2005
</P><P>[Sourjik, 2004] Victor Sourjik: <B>Receptor clustering and signal processing in E. coli chemotaxis</B>, Trends in Microbiology 12(12), 569–576, December 2004
</P><P>[Vendeville, 2005] Agnes Vendeville, Klaus Winzer, Karin Heurlier, Christoph M. Tang, and Kim R. Hardie: <B>Making sense of metabolism: Autoinducer-2, LuxS and pathogenic bacteria</B>, Nature Reviews 3, 383–396, 2005
</P><P>[Wadhams, 2004] George H. Wadhams, and Judith P. Armitage: <B>Making Sense of it all: bacterial chemotaxis</B>, Molecular Cell Biology 5, 1024–1037, December 2004
</P><P>[Mowbray, 1998] Sherry L. Mowbray, and Mats O. J. Sandgren: <B>Chemotaxis Receptors: A progress report on structure and function</B>, Journal of Structural Biology 124, 257–275, 1998
</P><P>[DeKeersmaecker, 2006] Sigrid C. J. De Keersmaecker, Kathleen Sonck, and Jos Vanderleyden: <B>Let LuxS speak up in AI-2 signaling</B>, Trends in Microbiology 14(3), 114–119, 2006
</P><P>[Sun, 2004] Jibin Sun, Rolf Daniel, Irene Wagner-Döbler, and An-Ping Zeng: <B>Is autoinducer-2 a universal signal for interspecies communication: a comparative genomic and phylogenetic analysis of the synthesis and signal transduction pathways</B>, BMC Evolutionary Biology 4(36), 2004
</P><P>[Neiditch, 2005] Matthew B. Neiditch, Michael J. Federle, Stephan T. Miller, Bonnie L. Bassler, and Frederick M. Hughson: <B>Regulation of LuxPQ Receptor Activity by the Quorum-Sensing Signal Autoinducer-2</B>, Molecular Cell 18, 507–518, May 2005
</P><P>[Bourret, 1993] R B Bourret, J Davagnino, and M I Simon: <B>The carboxy-terminal portion of the CheA kinase mediates regulation of autophosphorylation by transducer and CheW</B>, Journal of Bacteriology 175, 2097–2101, 1993
</P><P>[Morrison &amp; Parkinson] TB Morrison, and JS Parkinson: <B>A fragment liberated from the Escherichia coli CheA kinase that blocks stimulatory, but not inhibitory, chemoreceptor signaling</B>, Journal of Bacteriology 179, 5543–5550, 1997
[<A href="https://2008.igem.org/Team:Heidelberg/Project/Sensing" class="external text" rel="nofollow">back</A>]
</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2008.igem.org/Team:Heidelberg/Project/Sensing">http://2008.igem.org/Team:Heidelberg/Project/Sensing</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Heidelberg/Project/Sensing" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Heidelberg/Project/Sensing" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Heidelberg/Project/Sensing&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Heidelberg/Project/Sensing&amp;oldid=97288" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2008.igem.org:Privacy_policy" title="2008.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2008.igem.org:General_disclaimer" title="2008.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>