<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Heidelberg_Parts_Characterization"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:Heidelberg/Parts/Characterization">Page               </A></LI><LI><A href="/Talk:Team:Heidelberg/Parts/Characterization">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/Parts/Characterization&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/Parts/Characterization&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Heidelberg/Parts/Characterization" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Heidelberg/Parts/Characterization</H1><DIV id="bodyContent"><H3 id="siteSub">From 2008.igem.org</H3><DIV class="MenuBar" id="navi"><UL><LI><A href="https://2008.igem.org/Team:Heidelberg" style="color: white">Home
			</A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Team" style="color: white">Team</A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Team"><SPAN><SPAN>Overview</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Team_Members#Advisors"><SPAN><SPAN>Advisors</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Team_Members#Undergraduates"><SPAN><SPAN>Undergraduates</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Team#There_are_many_Universities_-_and_there_is_Heidelberg_..."><SPAN><SPAN>University</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Team#DKFZ"><SPAN><SPAN>DKFZ</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Team#Bioquant"><SPAN><SPAN>BioQuant</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/BioRegioRN"><SPAN><SPAN>BioRegion Rhein-Neckar</SPAN></SPAN></A></LI></UL><LI><A href="https://2008.igem.org/Team:Heidelberg/Project" style="color: white">Project</A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Project"><SPAN><SPAN>Overall Project</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Notebook/material"><SPAN><SPAN>Material &amp; Methods</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Project/Sensing"><SPAN><SPAN>Sensing</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Project/Killing_I"><SPAN><SPAN>Killing I  - Phages</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Project/Killing_II"><SPAN><SPAN>Killing II - Colicin</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Project/Visualization"><SPAN><SPAN>Visualization</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Parts" style="color: white">Parts</A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Parts"><SPAN><SPAN>Submitted Parts</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Parts/Characterization"><SPAN><SPAN>Characterization</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Modeling" style="color: white">Modeling</A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Modeling"><SPAN><SPAN>Overview</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Modeling/Chemotaxis"><SPAN><SPAN>Chemotaxis-Colicin</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Modeling/Phage"><SPAN><SPAN>Phage Dynamics model</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Notebook/Overview" style="color: white">Notebook</A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Notebook/Sensing_Group/Notebook"><SPAN><SPAN>Sensing &gt;</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Notebook/Sensing_Group/Cloning"><SPAN>Cloning strategy</SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Notebook/Sensing_Group/Notebook"><SPAN>Notebook</SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Notebook/Killing_I/Notebook"><SPAN><SPAN>Killing I - Phages &gt;</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Notebook/Killing_I/Cloning"><SPAN>Cloning strategy</SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Notebook/Killing_I/Notebook"><SPAN>Notebook</SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Notebook/Killing_II/Notebook"><SPAN><SPAN>Killing II - Colicin &gt;</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Notebook/Killing_II/Cloning"><SPAN>Cloning strategy</SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Notebook/Killing_II/Notebook"><SPAN>Notebook</SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Notebook/visualization"><SPAN><SPAN>Visualization</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Notebook/material"><SPAN><SPAN>Material &amp; Methods</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Notebook/team_meetings"><SPAN><SPAN>Team Meetings</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Notebook/seminar"><SPAN><SPAN>Seminar on Synthetic Biology</SPAN></SPAN></A></LI><LI style="width: 160px"><A href="https://2008.igem.org/Team:Heidelberg/Human_Practice/Project_Overview" style="color: white">Human Practice</A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Human_Practice/Project_Overview"><SPAN><SPAN>Project Overview</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Human_Practice/Phips_the_Phage"><SPAN><SPAN>Phips the Phage</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Human_Practice/Essay"><SPAN><SPAN>Essay</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Human_Practice/Surveys"><SPAN><SPAN>Surveys</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Human_Practice/Open_Day"><SPAN><SPAN>Open Day</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Human_Practice/Nobel_Prize"><SPAN><SPAN>Nobel Prize</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Sponsors" style="color: white">Sponsors</A></LI></DIV><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Part_Characterization"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Part Characterization</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Bacteriophage_lambda_cI_.28BBa_K150004.29"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Bacteriophage lambda cI (BBa_K150004)</SPAN></A></LI><LI class="toclevel-3 tocsection-3"><A href="#cI_protein_-_an_introduction"><SPAN class="tocnumber">1.1.1</SPAN><SPAN class="toctext">cI protein - an introduction</SPAN></A></LI><LI class="toclevel-3 tocsection-4"><A href="#Function_of_cI_protein_in_bacteriophage_.CE.BB"><SPAN class="tocnumber">1.1.2</SPAN><SPAN class="toctext">Function of cI protein in bacteriophage λ</SPAN></A></LI><LI class="toclevel-3 tocsection-5"><A href="#Characterization_of_cI_.28BBa_K150004.29"><SPAN class="tocnumber">1.1.3</SPAN><SPAN class="toctext">Characterization of cI (BBa_K150004)</SPAN></A></LI><LI class="toclevel-3 tocsection-6"><A href="#References"><SPAN class="tocnumber">1.1.4</SPAN><SPAN class="toctext">References</SPAN></A></LI></UL><LI class="toclevel-2 tocsection-7"><A href="#oriT_R_.28BBa_J01003.29"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">oriT R (BBa_J01003)</SPAN></A></LI><LI class="toclevel-3 tocsection-8"><A href="#Test_1_Conjugation_kinetics_of_oriT_BBa_J01003_between_E._coli_Top10"><SPAN class="tocnumber">1.2.1</SPAN><SPAN class="toctext">Test 1 Conjugation kinetics of oriT BBa_J01003 between E. coli Top10</SPAN></A></LI><LI class="toclevel-3 tocsection-9"><A href="#Test_2_Conjugation_kinetics_of_oriT_BBa_J01003_between_different_E.coli_strains"><SPAN class="tocnumber">1.2.2</SPAN><SPAN class="toctext">Test 2 Conjugation kinetics of oriT BBa_J01003 between different E.coli strains</SPAN></A></LI><LI class="toclevel-3 tocsection-10"><A href="#Test_3_Conjugative_competence_of_oriT_BBa_J01003_at_different_donor.2Frecipient_ratios"><SPAN class="tocnumber">1.2.3</SPAN><SPAN class="toctext">Test 3 Conjugative competence of oriT BBa_J01003 at different donor/recipient ratios</SPAN></A></LI><LI class="toclevel-3 tocsection-11"><A href="#Test_4_Conjugative_competence_of_oriT_BBa_J01003_at_different_temperatures"><SPAN class="tocnumber">1.2.4</SPAN><SPAN class="toctext">Test 4 Conjugative competence of oriT BBa_J01003 at different temperatures</SPAN></A></LI><LI class="toclevel-3 tocsection-12"><A href="#Summary"><SPAN class="tocnumber">1.2.5</SPAN><SPAN class="toctext">Summary</SPAN></A></LI><LI class="toclevel-2 tocsection-13"><A href="#Chloramphenicol_resistance_cassette_.28BBa_K150003.29"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Chloramphenicol resistance cassette (BBa_K150003)</SPAN></A></LI><LI class="toclevel-2 tocsection-14"><A href="#LuxR-Colicin_E1-Receiver_.28BBa_K150009.29"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">LuxR-Colicin E1-Receiver (BBa_K150009)</SPAN></A></LI><LI class="toclevel-3 tocsection-15"><A href="#ColicinE1_Producer_Controlled_by_3OC6HSL_Receiver_Device_-_Introduction"><SPAN class="tocnumber">1.4.1</SPAN><SPAN class="toctext">ColicinE1 Producer Controlled by 3OC6HSL Receiver Device - Introduction</SPAN></A></LI><LI class="toclevel-3 tocsection-16"><A href="#Colicin_E1_-_Mode_of_Action"><SPAN class="tocnumber">1.4.2</SPAN><SPAN class="toctext">Colicin E1 - Mode of Action</SPAN></A></LI><LI class="toclevel-3 tocsection-17"><A href="#Killing_efficiency_on_bacteria"><SPAN class="tocnumber">1.4.3</SPAN><SPAN class="toctext">Killing efficiency on bacteria</SPAN></A></LI><LI class="toclevel-3 tocsection-18"><A href="#Killer-prey_system"><SPAN class="tocnumber">1.4.4</SPAN><SPAN class="toctext">Killer-prey system</SPAN></A></LI><LI class="toclevel-3 tocsection-19"><A href="#Lysis_of_killing_strain"><SPAN class="tocnumber">1.4.5</SPAN><SPAN class="toctext">Lysis of killing strain</SPAN></A></LI><LI class="toclevel-3 tocsection-20"><A href="#Eukaryotic_killing_Assay_.28Preliminary_Results.29"><SPAN class="tocnumber">1.4.6</SPAN><SPAN class="toctext">Eukaryotic killing Assay (Preliminary Results)</SPAN></A></LI><LI class="toclevel-3 tocsection-21"><A href="#References_2"><SPAN class="tocnumber">1.4.7</SPAN><SPAN class="toctext">References</SPAN></A></LI><LI class="toclevel-2 tocsection-22"><A href="#LuxI_3OC6HSL_sender_device_.28BBa_K150000.29"><SPAN class="tocnumber">1.5</SPAN><SPAN class="toctext">LuxI 3OC6HSL sender device (BBa_K150000)</SPAN></A></LI><LI class="toclevel-3 tocsection-23"><A href="#General_Description"><SPAN class="tocnumber">1.5.1</SPAN><SPAN class="toctext">General Description</SPAN></A></LI><LI class="toclevel-3 tocsection-24"><A href="#Quantification_of_AHL_production"><SPAN class="tocnumber">1.5.2</SPAN><SPAN class="toctext">Quantification of AHL production</SPAN></A></LI><LI class="toclevel-2 tocsection-25"><A href="#LuxS_from_Vibrio_harveyi_.28partly_characterized.29"><SPAN class="tocnumber">1.6</SPAN><SPAN class="toctext">LuxS from Vibrio harveyi (partly characterized)</SPAN></A></LI><LI class="toclevel-3 tocsection-26"><A href="#Functional_check_of_LuxS.2FAI-2_production"><SPAN class="tocnumber">1.6.1</SPAN><SPAN class="toctext">Functional check of LuxS/AI-2 production</SPAN></A></LI><LI class="toclevel-3 tocsection-27"><A href="#Results"><SPAN class="tocnumber">1.6.2</SPAN><SPAN class="toctext">Results</SPAN></A></LI><LI class="toclevel-3 tocsection-28"><A href="#Reference"><SPAN class="tocnumber">1.6.3</SPAN><SPAN class="toctext">Reference</SPAN></A></LI></TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="Part_Characterization">Part Characterization</SPAN></H1><H2><SPAN class="mw-headline" id="Bacteriophage_lambda_cI_.28BBa_K150004.29">Bacteriophage lambda cI (<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K150004" class="external text" rel="nofollow">BBa_K150004</A>)</SPAN></H2><H3><SPAN class="mw-headline" id="cI_protein_-_an_introduction">cI protein - an introduction</SPAN></H3><P>Bacteriophage λ cI exists as an inactive monomer at very low concentrations (&lt;10<SUP>-9</SUP> M) but forms functional homodimers at physiological concentrations that remarkably lack a global symmetry [1]. It consists of 236 amino acids, although 237 amino acids are translated since the initiator methionine is removed by the host. Although λ cI is commonly called a repressor because of its negative regulatory functions at o<SUB>L</SUB> and o<SUB>R</SUB>, cI protein acts also as a positive regulator of gene transcription and can activate transcription of its own gene through the p<SUB>RM</SUB> promoter in bacteriophage λ. 
In genetic and biochemical studies as well as through the crystal structure it has been shown, that the carboxy-terminal domain of the cI protein contains the major sites for dimerization and oligomerization [2, 3]. The cI protein has a two-domain structure with an N-terminal portion involved in DNA binding, and a C-terminal domain that mediates dimer formation, dimer-dimer interaction, and self-cleavage. The self-cleavage reaction is triggert when the lysogenic cell suffers DNA damage and depends upon an activated form of the bacterial RecA protein [2, 4-6]. The ‘hinge region’ connecting both domains contains a conserved site that can undergo this RecA-mediated autodigestion resulting in inactivation of the repressor by separating the N-terminal from the C-terminal domain of the repressor [4, 7]. The cI protein binds symmetrically to DNA, so that each amino-terminal domain contacts a similar set of bases. The N-terminal DNA-binding domain is made up of 5 α-helices of which helix 2 and helix 3 (the helix-turn-helix motif) are involved in nucleotide sequence specific DNA recognition and binding to the major groove of DNA [8-11].
[<A href="https://2008.igem.org/Team:Heidelberg/Parts/Characterization" class="external text" rel="nofollow">back</A>]
</P><H3><SPAN class="mw-headline" id="Function_of_cI_protein_in_bacteriophage_.CE.BB">Function of cI protein in bacteriophage λ</SPAN></H3><P>The cI repressor in bacterophage λ is the key component of a ‘genetic switch’ that enables the phage to transition from lysogenic growth to lytic development. The cI protein binds the o<SUB>L</SUB> and o<SUB>R</SUB> operator which overlap with the p<SUB>L</SUB> and p<SUB>R</SUB> promoter (the lytic promoters). This binding allows the maintenance of the lysogenic state to be governed by cI alone. As soon as cI is inactivated, e.g., by the SOS response after UV damage or other agents that cause DNA damage, the lytic development follows [12]. 
The two operators o<SUB>L</SUB> and o<SUB>R</SUB> both contain three binding sites for cI protein. In each case, site 1 (i.e. o<SUB>L1</SUB> and o<SUB>R1</SUB>) has a ~10-fold greater affinity than the other sites for cI protein. The repressor, therefore, always binds to o<SUB>L1</SUB> and o<SUB>R1</SUB> first and than to the other sites in the operator in a cooperative manner [13, 14]. The carboxy-terminal domains of the repressor dimers mediate this cooperativity which improves the specificity and strength of the cI DNA binding, enabling strong repression of the lytic promoters [15].
Furthermore, another cooperative interaction between these two sets of tetramers bound to o<SUB>L</SUB> and o<SUB>R</SUB>, 2.4 kb apart, leads to the formation of a DNA loop held by a cI octamer, i.e. two interacting tetramers, that enhances repression of the early promoters [16-19]. In this DNA-multiprotein complex, the cI dimer bound at o<SUB>R2</SUB> represses p<SUB>R</SUB> and at the same time also stimulates p<SUB>RM</SUB> transcription, thus activating cI synthesis in a repressed prophage by a positive autoregulatory loop [20-22]. As the cI concentration increases because of p<SUB>RM</SUB> activation, two additional cI dimers are recruited to bind o<SUB>L3</SUB> and o<SUB>R3</SUB> to further stabilize the o<SUB>L</SUB> – o<SUB>R</SUB> loop. In this context, cI overexpression is prevented by the binding of cI to o<SUB>R3</SUB> which represses p<SUB>RM</SUB> [17, 19].
This positive and negative autoregulation at p<SUB>RM</SUB> by cI ensures a narrow range of cI repressor level to be maintained, which is optimum for stable lysogeny but is at the same time adjusted low enough for efficient induction of the lysogen. By repressing transcription from the p<SUB>R</SUB> promoter not only expression of genes in that operon is inhibited but also phage DNA replication by preventing transcriptional activation of λ ori, the site where phage DNA replication is initiated [23]. Even if O and P functions are present this inhibition occurs and appears to be critical for establishing a lysogen [24, 25].
The stable lysogen produces sufficient repressor not only to block prophage lytic development but also to block lytic development of any extraneous infection phage, thus imparting immunity to the lysogen against lytic superinfection [26].
[<A href="https://2008.igem.org/Team:Heidelberg/Parts/Characterization" class="external text" rel="nofollow">back</A>]
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:449px;"><DIV class="thumbcaption"><B>Figure 1: λ cI repressor dimer bound to an operator fragment.</B> A ribbon representation of the three-dimensional structure of the dimer complex crystallized by Stayrook et al. [1]. The N-terminal domain interacts with the operator DNA, the C-terminal domain self-associates and forms the dimer contacts. The two domains are connected by a linker that tethers the N-terminal domain to the C-terminal domain.</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><B>Figure 2: Structure of the λ cI C-terminal domain octamer.</B> A ribbon representation of the three-dimensional structure of the octameric complex crystallized by Bell and Lewis [27]. Each of the four carboxy-terminal domain dimers that comprise the octamer are colored differently. The amino-terminal residues of each subunit are indicated. Because the amino-terminal residue of each subunit in the octamer points to the outside of the structure, the DNA-binding domains would fan around the structure. Since the structure closes up to itself, no higher-ordered oligomers can be formed.</DIV></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Characterization_of_cI_.28BBa_K150004.29">Characterization of cI (<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K150004" class="external text" rel="nofollow">BBa_K150004</A>)</SPAN></H3><P>For the characterization of λ cI protein its ability to repress lytic development of bacteriophage λ was measured over a range of temperature from 28 °C to 42 °C in a phage burst experiment. If the used cI is functional no plaques should appear in a lawn of cells harboring this cI when infected with a phage that lacks its natural cI. In control cells in contrast every of this virulent particles should lead to a plaque.
</P><P><B>Protocol</B></P><P><B>Preparation of plating bacteria</B></P><P>A overnight culture of the appropriate <I>E. coli</I> strain was grown in LB medium containing 10 mM MgSO<SUB>4</SUB> and 0.2 % maltose at 30 °C to reduce the amount of cell debris in the medium. For this characterization <I>E. coli</I> TOP 10 and <I>E. coli</I> TOP 10 with transformed cI were used. The added maltose leads to a substantial induction of the maltose operon including the lamb gene, which encodes the cell surface receptor to which bacteriophage λ binds. After harvesting the cells at 3000g for 10 minutes they were resuspended in 10 mM MgSO<SUB>4</SUB> and diluted to a final concentration of 2.0 OD<SUB>600</SUB>. The suspension of plating bacteria was stored at 4 °C for up to 1 week. 
</P><P><B>Bacteriophage λ plaque assay</B></P><P>Tenfold serial dilutions of the bacteriophage λ stocks were prepared. In this case a bacteriophage mutant was used lacking a functional cI which would therefore always lead to plaques. From these dilutions 100 µl were mixed with the same amount of plating bacteria, incubated for 20 minutes at 37 °C to allow the bacteriophage particles to adsorb to the bacteria, this mixture than added to 3 ml molten top agar which was kept liquid at 48 °C and the entire contend poured onto a agar plate. After harden of the top agar the inverted plates were incubated at indicated temperatures over night. On the next day plaques could be counted.
</P><P><B>Results</B></P><P>As it can be seen in figure 3 – 5 the cells harboring the part <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K150004" class="external text" rel="nofollow">BBa_K150004</A> are not lysed by the used phage. This means that the constructed cI protein generator successfully expresses high levels of cI protein which is functional. Through the use of different temperatures it could be shown that the ability of the used cI protein to repress lytic development is not dependent of the temperature. 
</P><P>[<A href="https://2008.igem.org/Team:Heidelberg/Parts/Characterization" class="external text" rel="nofollow">back</A>]
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><B>Figure 3: phage burst experiment at 37 °C.</B> Shown is the number of plaque forming units (pfu) over the dilution from our bacteriophage λ stock. <I>E. coli</I> Top10 that express cI protein (green curve) as well as untransformed <I>E. coli</I> (red curve) as control were infected with a bacteriophage λ lacking its own cI. After growth at 37 °C over night the amount of plaques was counted the next day. Note, that the used cI protein generator successfully blocks the phage burst.</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><B>Figure 4: phage burst experiment at 42 °C.</B> Shown is the number of plaque forming units (pfu) over the dilution from our bacteriophage λ stock. <I>E. coli</I> Top10 that express cI protein (green curve) as well as untransformed <I>E. coli</I> (red curve) as control were infected with a bacteriophage λ lacking its own cI. After growth at 42 °C over night the amount of plaques was counted the next day. Note, that the used cI protein generator successfully blocks the phage burst.</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><B>Figure 5: phage burst experiment at 28 °C.</B> Shown is the number of plaque forming units (pfu) over the dilution from our bacteriophage λ stock. <I>E. coli</I> Top10 that express cI protein (green curve) as well as untransformed E. coli (red curve) as control were infected with a bacteriophage λ lacking its own cI. After growth at 28 °C over night the amount of plaques was counted the next day. Note, that the used cI protein generator successfully blocks the phage burst.</DIV></DIV></DIV></DIV><P>[<A href="https://2008.igem.org/Team:Heidelberg/Parts/Characterization" class="external text" rel="nofollow">back</A>]
</P><H3><SPAN class="mw-headline" id="References">References</SPAN></H3><P>[1]	S. Stayrook, P. Jaru-Ampornpan, J. Ni, A. Hochschild, M. Lewis, Crystal structure of the lambda repressor and a model for pairwise cooperative operator binding, Nature 452 (2008) 1022-1025.
</P><P>[2]	C.O. Pabo, R.T. Sauer, J.M. Sturtevant, M. Ptashne, The lambda repressor contains two domains, Proc Natl Acad Sci U S A 76 (1979) 1608-1612.
</P><P>[3]	C.E. Bell, P. Frescura, A. Hochschild, M. Lewis, Crystal structure of the lambda repressor C-terminal domain provides a model for cooperative operator binding, Cell 101 (2000) 801-811.
</P><P>[4]	J.W. Little, Autodigestion of lexA and phage lambda repressors, Proc Natl Acad Sci U S A 81 (1984) 1375-1379.
</P><P>[5]	R.T. Sauer, M.J. Ross, M. Ptashne, Cleavage of the lambda and P22 repressors by recA protein, J Biol Chem 257 (1982) 4458-4462.
</P><P>[6]	R.T. Sauer, C.O. Pabo, B.J. Meyer, M. Ptashne, K.C. Backman, Regulatory functions of the lambda repressor reside in the amino-terminal domain, Nature 279 (1979) 396-400.
</P><P>[7]	J.W. Little, LexA cleavage and other self-processing reactions, J Bacteriol 175 (1993) 4943-4950.
</P><P>[8]	C.O. Pabo, M. Lewis, The operator-binding domain of lambda repressor: structure and DNA recognition, Nature 298 (1982) 443-447.
</P><P>[9]	L.J. Beamer, C.O. Pabo, Refined 1.8 A crystal structure of the lambda repressor-operator complex, J Mol Biol 227 (1992) 177-196.
</P><P>[10]	A. Hochschild, Transcriptional activation. How lambda repressor talks to RNA polymerase, Curr Biol 4 (1994) 440-442.
</P><P>[11]	C.O. Pabo, R.T. Sauer, Transcription factors: structural families and principles of DNA recognition, Annu Rev Biochem 61 (1992) 1053-1095.
</P><P>[12]	A. Hochschild, The lambda switch: cI closes the gap in autoregulation, Curr Biol 12 (2002) R87-89.
</P><P>[13]	A.D. Johnson, B.J. Meyer, M. Ptashne, Interactions between DNA-bound repressors govern regulation by the lambda phage repressor, Proc Natl Acad Sci U S A 76 (1979) 5061-5065.
</P><P>[14]	M. Ptashne, Genetic switch: phage lambda revisited, 2nd ed., Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY, 2004.
</P><P>[15]	I.B. Dodd, K.E. Shearwin, J.B. Egan, Revisited gene regulation in bacteriophage lambda, Curr Opin Genet Dev 15 (2005) 145-152.
</P><P>[16]	I.B. Dodd, A.J. Perkins, D. Tsemitsidis, J.B. Egan, Octamerization of lambda CI repressor is needed for effective repression of P(RM) and efficient switching from lysogeny, Genes Dev 15 (2001) 3013-3022.
</P><P>[17]	I.B. Dodd, K.E. Shearwin, A.J. Perkins, T. Burr, A. Hochschild, J.B. Egan, Cooperativity in long-range gene regulation by the lambda CI repressor, Genes Dev 18 (2004) 344-354.
</P><P>[18]	B. Revet, B. von Wilcken-Bergmann, H. Bessert, A. Barker, B. Muller-Hill, Four dimers of lambda repressor bound to two suitably spaced pairs of lambda operators form octamers and DNA loops over large distances, Curr Biol 9 (1999) 151-154.
</P><P>[19]	S.L. Svenningsen, N. Costantino, D.L. Court, S. Adhya, On the role of Cro in lambda prophage induction, Proc Natl Acad Sci U S A 102 (2005) 4465-4469.
</P><P>[20]	B.J. Meyer, M. Ptashne, Gene regulation at the right operator (OR) of bacteriophage lambda. III. lambda repressor directly activates gene transcription, J Mol Biol 139 (1980) 195-205.
</P><P>[21]	D. Jain, B.E. Nickels, L. Sun, A. Hochschild, S.A. Darst, Structure of a ternary transcription activation complex, Mol Cell 13 (2004) 45-53.
</P><P>[22]	B.E. Nickels, S.L. Dove, K.S. Murakami, S.A. Darst, A. Hochschild, Protein-protein and protein-DNA interactions of sigma70 region 4 involved in transcription activation by lambdacI, J Mol Biol 324 (2002) 17-34.
</P><P>[23]	M.E. Furth, W.F. Dove, B.J. Meyer, Specificity determinants for bacteriophage lambda DNA replication. III. Activation of replication in lambda ric mutants by transcription outside of ori, J Mol Biol 154 (1982) 65-83.
</P><P>[24]	K. Mensa-Wilmot, K. Carroll, R. McMacken, Transcriptional activation of bacteriophage lambda DNA replication in vitro: regulatory role of histone-like protein HU of Escherichia coli, EMBO J 8 (1989) 2393-2402.
</P><P>[25]	M.S. Wold, J.B. Mallory, J.D. Roberts, J.H. LeBowitz, R. McMacken, Initiation of bacteriophage lambda DNA replication in vitro with purified lambda replication proteins, Proc Natl Acad Sci U S A 79 (1982) 6176-6180.
</P><P>[26]	D.L. Court, A.B. Oppenheim, S.L. Adhya, A new look at bacteriophage lambda genetic networks, J Bacteriol 189 (2007) 298-304.
</P><P>[27]	C.E. Bell, M. Lewis, Crystal structure of the lambda repressor C-terminal domain octamer, J Mol Biol 314 (2001) 1127-1136.
</P><P>[<A href="https://2008.igem.org/Team:Heidelberg/Parts/Characterization" class="external text" rel="nofollow">back</A>]
</P><H2><SPAN class="mw-headline" id="oriT_R_.28BBa_J01003.29"> oriT R (<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_J01003" class="external text" rel="nofollow">BBa_J01003</A>)</SPAN></H2><P>We have characterized the oriT <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_J01003" class="external text" rel="nofollow">BBa_J01003</A> in a conjugation system in which pUB307 is used as helper plasmid and <I>E. coli</I> as donor and recipient cells. We measured quantitatively the conjugative competence of oriT <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_J01003" class="external text" rel="nofollow">BBa_J01003</A> by conjugation between different strains, at different temperatures and with different donor/recipient ratios. We also compared the conjugation kinetics of <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_J01003" class="external text" rel="nofollow">BBa_J01003</A> and pUB307 quantitatively. 
</P><P>The part oriT R <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_J01003" class="external text" rel="nofollow">BBa_J01003</A> was cotransformed together with pUB307 (helper plasmid) into <I>E. coli</I> strain Top10. These cells therefore became conjugation donor and have a kanamycin (from pUB307) and ampicillin (from <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_J01003" class="external text" rel="nofollow">BBa_J01003</A>) resistance. The recipient cells have the plasmid pBAD33 with a chloramphenicol resistance. After mixing both cell types the conjugation starts and <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_J01003" class="external text" rel="nofollow">BBa_J01003</A> as well as pUB307 can be transported. The helper plasmid pUB307 can still be transported since it still contains its oriT region. The transconjugants which have received <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_J01003" class="external text" rel="nofollow">BBa_J01003</A> through the conjugation have ampicillin and chloramphenicol resistance and can therefore be easily selected on agar plates containing these two antibiotics. 
[<A href="https://2008.igem.org/Team:Heidelberg/Parts/Characterization" class="external text" rel="nofollow">back</A>]
</P><H3><SPAN class="mw-headline" id="Test_1_Conjugation_kinetics_of_oriT_BBa_J01003_between_E._coli_Top10">Test 1 Conjugation kinetics of oriT <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_J01003" class="external text" rel="nofollow">BBa_J01003</A> between <I>E. coli</I> Top10</SPAN></H3><P>On the first day we inoculated LB + antibiotics medium with a single colony or from a glycerol stock. For this test we used donor and the recipient cells that were both <I>E. coli</I> Top10. On the second day we washed the cells of the overnight culture twice with LB medium, transferred the donor cell suspension to the pellet of the recipient cells and resuspended the recipient cells in this medium. After another centrifugation step we resuspended the pellet which now is a mixture of donor and recipient cells in 100 µl LB medium without antibiotics. We than transferred the cell suspension on a membrane filter, which was placed on LB plates before and incubated these plates at 37 °C for 0, 6, 12, 18, 24, 30, 36, 42 minutes. Afterwards the membrane was put into 1 ml LB medium and the cells were resuspended by vortexing for 30 seconds. The cell suspension was diluted 10<SUP>-5</SUP> and plated on the LB / ampicillin + chloramphenicol (CA) plates. The plates were incubated at 37 °C overnight and on the third day the colonies could be counted. 
We have done this test thrice independently and calculated the conjugation efficient which is defined as [transconjugants per ml] / [donor per ml] from this assay. 
To count the donor and recipient cell density small aliquots of the used suspensions were diluted 10<SUP>-7</SUP> and plated on agar plates containing kanamycin + ampicillin for donor cells and chloramphenicol for the recipients. 
[<A href="https://2008.igem.org/Team:Heidelberg/Parts/Characterization" class="external text" rel="nofollow">back</A>]
</P><P><B>Table 1:</B> Conjugation kinetics of oriT BBa_J01003 between <I>E. coli</I> Top10.
</P><TABLE class="wikitable"><TBODY><TR align="center" bgcolor="grey"><TD style="font-weight:bold;">Group</TD><TD colspan="2" align="center">Test A</TD><TD colspan="2" align="center">Test B </TD><TD colspan="2" align="center"> Test C
</TD></TR><TR align="center" bgcolor="grey"><TD style="font-weight:bold;">Cell</TD><TD> Donor </TD><TD>Recipient</TD><TD> Donor </TD><TD> Recipient </TD><TD> Donor </TD><TD> Recipient
</TD></TR><TR align="center" bgcolor="grey"><TD style="font-weight:bold;">Strain</TD><TD> Top10</TD><TD> Top10</TD><TD> Top10 </TD><TD> Top10 </TD><TD> Top10 </TD><TD> Top10
</TD></TR><TR align="center" bgcolor="grey"><TD style="font-weight:bold;">Cell number</TD><TD> 9325 </TD><TD> 8100</TD><TD> 3465 </TD><TD> 8330 </TD><TD> 7140 </TD><TD> 7470
</TD></TR><TR align="center" bgcolor="grey"><TD style="font-weight:bold;">Time [min]</TD><TD> Transconjugants </TD><TD>Efficiency</TD><TD> Transconjugants </TD><TD> Efficiency </TD><TD> Transconjugants </TD><TD> Efficiency
</TD></TR><TR align="center"><TD style="font-weight:bold;">0</TD><TD> 150 </TD><TD>0.016</TD><TD> 9 </TD><TD> 0,003 </TD><TD> 37 </TD><TD> 0,005
</TD></TR><TR align="center"><TD style="font-weight:bold;">6</TD><TD> 500 </TD><TD> 0.054</TD><TD> 20 </TD><TD> 0,006 </TD><TD> 390 </TD><TD> 0,055
</TD></TR><TR align="center"><TD style="font-weight:bold;">12</TD><TD> 780 </TD><TD>0.084</TD><TD> 94 </TD><TD> 0,027 </TD><TD> 1390 </TD><TD> 0,195
</TD></TR><TR align="center"><TD style="font-weight:bold;">18</TD><TD> 1160</TD><TD>0.124</TD><TD> 172 </TD><TD> 0,050 </TD><TD> 1720 </TD><TD> 0,241
</TD></TR><TR align="center"><TD style="font-weight:bold;">24</TD><TD> 1400</TD><TD>0.150</TD><TD> 262 </TD><TD> 0,076 </TD><TD> 2880 </TD><TD> 0,403
</TD></TR><TR align="center"><TD style="font-weight:bold;">30</TD><TD> 3360</TD><TD>0.360</TD><TD> 1450 </TD><TD> 0,418 </TD><TD> 3500 </TD><TD> 0,490
</TD></TR><TR align="center"><TD style="font-weight:bold;">36</TD><TD> 1790 </TD><TD> 0.517 </TD><TD> 3750 </TD><TD> 0,525
</TD></TR><TR align="center"><TD style="font-weight:bold;">42</TD><TD> 2170 </TD><TD> 0.626 </TD></TR></TBODY></TABLE><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><B>Figure 1: conjugation kinetic of <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_J01003" class="external text" rel="nofollow">BBa_J01003</A> between <I>E. coli</I> Top10 strains.</B> Note that the conjugation efficiency increases with the time allowed for the conjugation to take place.</DIV></DIV></DIV></DIV><P>Figure 1 shows that the kinetic of the conjugative plasmid <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_J01003" class="external text" rel="nofollow">BBa_J01003</A> has three phases. In the first (0 min to 24 min) and in the third phase (30 min to 42 min) the curve has a smaller slope, whereas the second phase (24 min to 30 min) it has a very big slope. An explanation for this result could be that the cell growth in the first 24 min can be neglected due to the average generation time of <I>E. coli</I> in the LB medium which lies between 20 and 30 minutes whereas in the second phase the growth of <I>E. coli</I> dominates the conjugation process and leads to this deviation from linearity. In this test we used very concentrated bacteria suspension in a small conjugation volume which leads to the observation of conjugation in an approximately linear process in the primary phase.
We have done a linear regression analysis for the first phase (0 min to 24 min) and we calculated the slope for this phase as 9.00 • 10<SUP>-3</SUP> [transconjugants • donor<SUP>-1</SUP> • minute <SUP>-1</SUP>]. We calculated the conjugation rate for <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_J01003" class="external text" rel="nofollow">BBa_J01003</A> which is defined as transconjugants • donor<SUP>-1</SUP> • recipient<SUP>-1</SUP> • minute <SUP>-1</SUP>. The conjugation rate of <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_J01003" class="external text" rel="nofollow">BBa_J01003</A> is 1.05 • 10<SUP>-12</SUP> [ml • min<SUP>-1</SUP> • cell<SUP>-1</SUP>]. This rate was used by the modeling group as a parameter value. 
Figure 1 also shows us that the transport of the plasmid pSB1A2 containing the part <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_J01003" class="external text" rel="nofollow">BBa_J01003</A> by conjugation is a very fast process since we got transconjugants in the first minutes of the experiment. Through this experiment we can conclude that the transport of the plasmid pSB1A2 containing <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_J01003" class="external text" rel="nofollow">BBa_J01003</A> by conjugation needs only about 2 minutes. 
Since we have two conjugative plasmids in our donor cells, we wanted to see which plasmid is preferred to be transported by conjugation. We therefore plated the cells in the test C (see table 2) also on agar plates containing chloramphenicol + kanamycin (CK) and chloramphenicol + kanamycin + ampicillin (CKA) to select for transconjugants getting pUB307 and both plasmids. 
[<A href="https://2008.igem.org/Team:Heidelberg/Parts/Characterization" class="external text" rel="nofollow">back</A>]
</P><P><B>Table 2:</B> Test C conjugation kinetics of oriT BBa_J01003 and pUB307 between <I>E. coli</I> Top10.
</P><TABLE class="wikitable"><TBODY><TR align="center" bgcolor="grey"><TD style="font-weight:bold;">Select</TD><TD colspan="2" align="center">Chlorophencol+Kanamycin (J01103) </TD><TD colspan="2" align="center">Chlorophencol+Kanamycin (pUB307) </TD><TD colspan="2" align="center"> Chlorophenicol+Ampicilin+Kanamycin
</TD></TR><TR align="center" bgcolor="grey"><TD style="font-weight:bold;">Cell</TD><TD> Donor </TD><TD> Recipient</TD><TD> Donor </TD><TD> Recipient </TD><TD> Donor </TD><TD> Recipient
</TD></TR><TR align="center" bgcolor="grey"><TD style="font-weight:bold;">Strain</TD><TD> Top10</TD><TD> Top10</TD><TD>Top10 </TD><TD> Top10 </TD><TD> Top10 </TD><TD> Top10
</TD></TR><TR align="center" bgcolor="grey"><TD style="font-weight:bold;">Cell number</TD><TD> 7140</TD><TD>7470</TD><TD> 7140 </TD><TD> 7470 </TD><TD> 7140 </TD><TD> 7470
</TD></TR><TR align="center" bgcolor="grey"><TD style="font-weight:bold;">Time [min]</TD><TD> Transconjugants </TD><TD>Efficiency</TD><TD> Transconjugants </TD><TD> Efficiency </TD><TD> Transconjugants </TD><TD> Efficiency
</TD></TR><TR align="center"><TD style="font-weight:bold;">0</TD><TD> 37 </TD><TD>0.005</TD><TD> 2 </TD><TD> 0,000 </TD><TD> 1 </TD><TD> 0,000
</TD></TR><TR align="center"><TD style="font-weight:bold;">6</TD><TD> 390 </TD><TD> 0.055</TD><TD> 17 </TD><TD> 0,002 </TD><TD> 4 </TD><TD> 0,001
</TD></TR><TR align="center"><TD style="font-weight:bold;">12</TD><TD> 1390 </TD><TD> 0.195</TD><TD> 194 </TD><TD> 0,027 </TD><TD> 90 </TD><TD> 0,013
</TD></TR><TR align="center"><TD style="font-weight:bold;">18</TD><TD> 1720 </TD><TD>0.241</TD><TD> 230 </TD><TD> 0,032 </TD><TD> 109 </TD><TD> 0,015
</TD></TR><TR align="center"><TD style="font-weight:bold;">24</TD><TD> 2880 </TD><TD> 0.403</TD><TD> 1130 </TD><TD> 0,158 </TD><TD> 800 </TD><TD> 0,112
</TD></TR><TR align="center"><TD style="font-weight:bold;">30</TD><TD> 3500</TD><TD> 0.490</TD><TD> 2500 </TD><TD> 0,350 </TD><TD> 1540 </TD><TD> 0,216
</TD></TR><TR align="center"><TD style="font-weight:bold;">36</TD><TD> 3750 </TD><TD>0.525</TD><TD> 2600 </TD><TD> 0,364 </TD><TD> 1800 </TD><TD> 0,252
</TD></TR></TBODY></TABLE><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><B>Figure 2: Conjugation efficiency of different plasmids.</B> Transconjugants with <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_J01003" class="external text" rel="nofollow">BBa_J01003</A> were selected by plating on LB agar containing chloramphenicol and ampicillin, transconjugants with pUB307 by plates containing chloramphenicol and kanamycin, and transconjugants with both plasmids by plates containing chloramphenicol and kannamycin and ampicillin.</DIV></DIV></DIV></DIV><P>
Figure 2 clearly shows that the plasmid pSB1A2 containing only <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_J01003" class="external text" rel="nofollow">BBa_J01003</A> is preferably transported during the first 20 minutes in comparison to pUB307. Since pUB307 has the same oriT as <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_J01003" class="external text" rel="nofollow">BBa_J01003</A> is, we assume the size of the plasmid to be the reason for this since <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_J01003" class="external text" rel="nofollow">BBa_J01003</A> in pSB1A2 is approx. 2.5 kb in size whereas pUB307 has 60 kb. A comparison of the red and the green curve shows us that the difference in the duration of the transport of plasmids with different size is not very big. Looking at the blue curve you can see that the first transconjugants with both plasmids can be detected after approx. 18 minutes which is longer than the sum of both transport times.
[<A href="https://2008.igem.org/Team:Heidelberg/Parts/Characterization" class="external text" rel="nofollow">back</A>]
</P><H3><SPAN class="mw-headline" id="Test_2_Conjugation_kinetics_of_oriT_BBa_J01003_between_different_E.coli_strains">Test 2 Conjugation kinetics of oriT <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_J01003" class="external text" rel="nofollow">BBa_J01003</A> between different <I>E.coli</I> strains</SPAN></H3><P>For this experiment we used the same technique as described in test 1. In this case we used different <I>E. coli</I> strains as recipients, namely <I>E. coli</I> Top10, MG1655 and DH5α. In this experiment the cells on the membranes were incubated for 0, 6, 12, 18, 24, 30, 36 and 42 minutes at 37 °C.
This test was conducted twice independently and the conjugation efficiency calculated as described in test 1. 
[<A href="https://2008.igem.org/Team:Heidelberg/Parts/Characterization" class="external text" rel="nofollow">back</A>]
</P><P><B>Table 3:</B> Conjugation kinetics of oriT BBa_J01004 between different <I>E. coli</I> strains.
</P><TABLE class="wikitable"><TBODY><TR align="center" bgcolor="grey"><TD style="font-weight:bold;">Group</TD><TD colspan="3" align="center"> Test D (Donor: Top10; Cell number: 5440)</TD><TD colspan="3" align="center">Test E (Donor: Top10; Cell number: 7140)
</TD></TR><TR align="center" bgcolor="grey"><TD style="font-weight:bold;">Cell</TD><TD> Recipient</TD><TD>Recipient</TD><TD> Recipient </TD><TD> Recipient </TD><TD> Recipient </TD><TD> Recipient
</TD></TR><TR align="center" bgcolor="grey"><TD style="font-weight:bold;">Strain</TD><TD>Top10 </TD><TD> MG1655</TD><TD> DH5alpha </TD><TD> Top10 </TD><TD> MG1655 </TD><TD> DH5alpha
</TD></TR><TR align="center" bgcolor="grey"><TD style="font-weight:bold;">Cell number</TD><TD> 6086 </TD><TD> 5566</TD><TD> 5096 </TD><TD> 7470 </TD><TD> 7275 </TD><TD> 6900
</TD></TR><TR align="center" bgcolor="grey"><TD style="font-weight:bold;">Time [min]</TD><TD> Efficiency</TD><TD>Efficiency</TD><TD> Efficiency </TD><TD> Efficiency </TD><TD> Efficiency </TD><TD> Efficiency
</TD></TR><TR align="center"><TD style="font-weight:bold;">0</TD><TD> 0.002 </TD><TD> 0,000</TD><TD> 0,000 </TD><TD> 0,005 </TD><TD> 0,000 </TD><TD> 0,001
</TD></TR><TR align="center"><TD style="font-weight:bold;">6</TD><TD> 0.002 </TD><TD>0,000</TD><TD> 0,000 </TD><TD> 0,055 </TD><TD> 0,000 </TD><TD> 0,003
</TD></TR><TR align="center"><TD style="font-weight:bold;">12</TD><TD> 0.003 </TD><TD>0,000</TD><TD> 0,000 </TD><TD> 0,195 </TD><TD> 0,001 </TD><TD> 0,067
</TD></TR><TR align="center"><TD style="font-weight:bold;">18</TD><TD> 0.005 </TD><TD>0,000</TD><TD> 0,001 </TD><TD> 0,241 </TD><TD> 0,007 </TD><TD> 0,165
</TD></TR><TR align="center"><TD style="font-weight:bold;">24</TD><TD> 0.021 </TD><TD>0,000</TD><TD> 0,018 </TD><TD> 0,403 </TD><TD> 0,019 </TD><TD> 0,269
</TD></TR><TR align="center"><TD style="font-weight:bold;">30</TD><TD> 0.071 </TD><TD>0,001</TD><TD> 0,075 </TD><TD> 0,490 </TD><TD> 0,036 </TD><TD> 0,415
</TD></TR><TR align="center"><TD style="font-weight:bold;">36</TD><TD> 0.136 </TD><TD>0,013</TD><TD> 0,246 </TD><TD> 0,525 </TD><TD> 0,129 </TD><TD> 0,482
</TD></TR><TR align="center"><TD style="font-weight:bold;">42</TD><TD>0.041</TD><TD> 0,301 </TD><TD> 0,134 </TD><TD> 0,566
</TD></TR></TBODY></TABLE><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><B>Figure 3: Conjugation of <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_J01003" class="external text" rel="nofollow">BBa_J01003</A> in pSB1A2 between different strains of <I>E. coli</I>.</B> The transconjugants were selected through growing on plates containing chloramphenicol and ampicillin. Note that conjugation from Top10 to Top10 and DH5α is comparable whereas conjugation in MG1655 takes place a lot slower.</DIV></DIV></DIV></DIV><P>
As you can see in figure 3 there is a difference in the conjugation kinetic between Top10 as donor and different strains as recipient. With <I>E. coli</I> Top10 as donor <I>E. coli</I> DH5α is a comparably good recipient as Top10 whereas MG1655 showed up as a bad recipient under these conditions. Comparing the red with the blue curve one can see that DH5α transconjugants appear later (6 minutes) that Top10, but the conjugation efficiency and also the conjugation rate (slope between 6 and 24 minutes) are in the same range with Top10 and DH5α. We assume that more time is needed for stabilizing the pillus contact between Top10 and DH5α than between two Top10 cells, but the conjugation between Top10 and DH5α can be done as good as between two Top10 cells. Comparing the red and the green curve one can see that the first MG1655 transconjugants appear much later (20 minutes) and the conjugation efficiency and rate (slope between 20 and 32 minutes) are much smaller. Therefore we conclude that conjugation between different strains is possible using this oriT but the efficiency depends on the used strains and their compatibility.
[<A href="https://2008.igem.org/Team:Heidelberg/Parts/Characterization" class="external text" rel="nofollow">back</A>]
</P><H3><SPAN class="mw-headline" id="Test_3_Conjugative_competence_of_oriT_BBa_J01003_at_different_donor.2Frecipient_ratios">Test 3 Conjugative competence of oriT <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_J01003" class="external text" rel="nofollow">BBa_J01003</A> at different donor/recipient ratios</SPAN></H3><P>In this experiment we again used the same protocol as described above in test 1 but used different donor / recipient rations, namely 50:1, 25:1, 10:1, 8:1, 5:1, 3:1, 2:1, 1:1, 1:2, 1:3, 1:5, 1:10, 1:25, and 1:50. This test was conducted twice independently and the conjugation efficiency calculated as described above. The donor and recipient cell density was measured by plating 10<SUP>-7</SUP> dilutions on agar plates containing kanamycin + ampicillin for the donor and chloramphenicol for recipient cells.
[<A href="https://2008.igem.org/Team:Heidelberg/Parts/Characterization" class="external text" rel="nofollow">back</A>]
</P><P><B>Table 4:</B> Conjugative competence of otiT BBa_J01003 at different donor / recipient ratios.
</P><TABLE class="wikitable"><TBODY><TR align="center" bgcolor="grey"><TD style="font-weight:bold;">Group</TD><TD colspan="4" align="center"> Test F  </TD><TD colspan="4" align="center"> Test G
</TD></TR><TR align="center" bgcolor="grey"><TD style="font-weight:bold;">Anticipant ratio</TD><TD> Donor </TD><TD> Recipient</TD><TD> Actual ratio </TD><TD> Transconjugants </TD><TD> Donor </TD><TD> Recipient </TD><TD> Actual ratio </TD><TD> Transconjugants
</TD></TR><TR align="center"><TD style="font-weight:bold;">50:1</TD><TD> 1003 </TD><TD> 23</TD><TD> 43:1 </TD><TD> 12 </TD><TD> 832 </TD><TD> 21 </TD><TD> 40:1 </TD><TD> 16
</TD></TR><TR align="center"><TD style="font-weight:bold;">25:1</TD><TD> 982 </TD><TD> 46</TD><TD> 21.5:1 </TD><TD> 28 </TD><TD> 815 </TD><TD> 41 </TD><TD> 20:1 </TD><TD> 31
</TD></TR><TR align="center"><TD style="font-weight:bold;">10:1</TD><TD> 930 </TD><TD> 110</TD><TD> 8.5:1 </TD><TD> 62 </TD><TD> 729 </TD><TD> 97 </TD><TD> 7.5:1 </TD><TD> 63
</TD></TR><TR align="center"><TD style="font-weight:bold;">8:1</TD><TD> 909 </TD><TD> 133</TD><TD> 7:1 </TD><TD> 81 </TD><TD> 711 </TD><TD> 117 </TD><TD> 6:1 </TD><TD> 75
</TD></TR><TR align="center"><TD style="font-weight:bold;">5:1</TD><TD> 854 </TD><TD> 202</TD><TD> 4:1 </TD><TD> 115 </TD><TD> 665 </TD><TD> 153 </TD><TD> 4.5:1 </TD><TD> 120
</TD></TR><TR align="center"><TD style="font-weight:bold;">3:1</TD><TD> 767</TD><TD>302</TD><TD> 2.5:1 </TD><TD> 161 </TD><TD> 599 </TD><TD> 266 </TD><TD> 2.3:1 </TD><TD> 129
</TD></TR><TR align="center"><TD style="font-weight:bold;">2:1</TD><TD> 680</TD><TD>403</TD><TD> 1.7:1 </TD><TD> 213 </TD><TD> 533 </TD><TD> 355 </TD><TD> 1.5:1 </TD><TD> 212
</TD></TR><TR align="center"><TD style="font-weight:bold;">1:1</TD><TD> 510</TD><TD>605</TD><TD> 1:1.2 </TD><TD> 302 </TD><TD> 400 </TD><TD> 532 </TD><TD> 1:1.3 </TD><TD> 303
</TD></TR><TR align="center"><TD style="font-weight:bold;">1:2</TD><TD> 340</TD><TD>802</TD><TD> 1:2.3 </TD><TD> 351 </TD><TD> 268 </TD><TD> 705 </TD><TD> 1:2.6 </TD><TD> 298
</TD></TR><TR align="center"><TD style="font-weight:bold;">1:3</TD><TD> 257</TD><TD>902</TD><TD> 1:3.5 </TD><TD> 294 </TD><TD> 199 </TD><TD> 794 </TD><TD> 1:4 </TD><TD> 302
</TD></TR><TR align="center"><TD style="font-weight:bold;">1:5</TD><TD> 170</TD><TD>1003</TD><TD> 1:5.9 </TD><TD> 289 </TD><TD> 133 </TD><TD> 883 </TD><TD> 1:6.7 </TD><TD> 194
</TD></TR><TR align="center"><TD style="font-weight:bold;">1:8</TD><TD> 122</TD><TD>1072</TD><TD> 1:8.8 </TD><TD> 241 </TD><TD> 89 </TD><TD> 943 </TD><TD> 1:10.5 </TD><TD> 190
</TD></TR><TR align="center"><TD style="font-weight:bold;">1:10</TD><TD> 94</TD><TD>1095</TD><TD> 1:11.7 </TD><TD> 158 </TD><TD> 72 </TD><TD> 963 </TD><TD> 1:13 </TD><TD> 134
</TD></TR><TR align="center"><TD style="font-weight:bold;">1:25</TD><TD> 40</TD><TD>1159</TD><TD> 1:29 </TD><TD> 86 </TD><TD> 31 </TD><TD> 1020 </TD><TD> 1:33 </TD><TD> 82
</TD></TR><TR align="center"><TD style="font-weight:bold;">1:50</TD><TD> 20</TD><TD>1182</TD><TD> 1:59 </TD><TD> 31 </TD><TD> 16 </TD><TD> 1040 </TD><TD> 1:67 </TD><TD> 30
</TD></TR></TBODY></TABLE><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><B>Figure 4: Conjugative competence of oriT <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_J01003" class="external text" rel="nofollow">BBa_J01003</A> at different donor / recipient ratios.</B> For all of the samples a similar cell density was used. For the conjugation a time of 20 minutes was set. The transconjugants were selected by plating on plates containing chloramphenicol and ampicillin.</DIV></DIV></DIV></DIV><P>
Figure 4 shows very clearly that at same cell density, if donor : recipient ratio is 1 : 2 the highest amount of transconjugants can be achieved. A big difference in the amount of donor : recipient leads to a low number of transconjugants. At low donor : recipient ratios the conjugation efficiency was quiet high but lead to a low number of transconjugants since the density of donor cells was too low. Only at donor : recipient rations between 1 : 1 and 1 : 3, a high conjugation efficiency and a high donor density was achieved leading to a high number of transconjugants. A maximum in the number of transconjugants could be achieved at a ratio of 1 : 2.
[<A href="https://2008.igem.org/Team:Heidelberg/Parts/Characterization" class="external text" rel="nofollow">back</A>]
</P><H3><SPAN class="mw-headline" id="Test_4_Conjugative_competence_of_oriT_BBa_J01003_at_different_temperatures">Test 4 Conjugative competence of oriT <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_J01003" class="external text" rel="nofollow">BBa_J01003</A> at different temperatures</SPAN></H3><P>In this experiment the same method is used as described above in test 1. Here, donor and recipient both were <I>E. coli</I> Top10. The incubation during conjugation took place at different temperatures, namely 4 °C, 22 °C, 30 °C, 37 °C and 42 °C for 20, 40 and 60 minutes. For counting of transconjugants the cell suspension was diluted 10<SUP>-6</SUP> and plated on agar containing ampicillin and chloramphenicol (CA). This experiment was conducted twice independently. The donor and recipient cell density was again calculated by plating 10<SUP>-7</SUP> dilutions on plates containing kanamycin and ampicillin or chloramphenicol, respectively. 
[<A href="https://2008.igem.org/Team:Heidelberg/Parts/Characterization" class="external text" rel="nofollow">back</A>]
</P><P><B>Table 5:</B> Conjugative competence of otiT BBa_J01003 at different temperatures.
</P><TABLE class="wikitable"><TBODY><TR align="center" bgcolor="grey"><TD style="font-weight:bold;">Group</TD><TD colspan="3" align="center"> Test H</TD><TD colspan="3" align="center">Test J
</TD></TR><TR align="center" bgcolor="grey"><TD style="font-weight:bold;">Cell</TD><TD> Donor </TD><TD> Recipient</TD><TD> donor : recipient </TD><TD> Donor </TD><TD> Recipient </TD><TD> donor/recipient
</TD></TR><TR align="center" bgcolor="grey"><TD style="font-weight:bold;">Strain</TD><TD> Top10</TD><TD>Top10</TD><TD> 1 : 2.02 </TD><TD> Top10 </TD><TD> Top10 </TD><TD> 1:1.77
</TD></TR><TR align="center" bgcolor="grey"><TD style="font-weight:bold;">Cell number</TD><TD> 361</TD><TD>731</TD><TD> sum:1092 </TD><TD> 330 </TD><TD> 584 </TD><TD> sum:914
</TD></TR><TR align="center" bgcolor="grey"><TD style="font-weight:bold;">Sample</TD><TD colspan="3" align="center"> Transconjugants </TD><TD colspan="3" align="center"> Transconjugants
</TD></TR><TR align="center" bgcolor="grey"><TD style="font-weight:bold;">Temperature</TD><TD> 10min</TD><TD>40min</TD><TD> 60min </TD><TD> 20min </TD><TD> 40min </TD><TD> 60min
</TD></TR><TR align="center"><TD style="font-weight:bold;">4°C</TD><TD> 5</TD><TD> 5</TD><TD> 1 </TD><TD> 2 </TD><TD> 1 </TD><TD> 0
</TD></TR><TR align="center"><TD style="font-weight:bold;">22°C</TD><TD> 2</TD><TD>78</TD><TD> 117 </TD><TD> 29 </TD><TD> 61 </TD><TD> 79
</TD></TR><TR align="center"><TD style="font-weight:bold;">30°C</TD><TD> 71 </TD><TD>178</TD><TD> 241 </TD><TD> 43 </TD><TD> 117 </TD><TD> 227
</TD></TR><TR align="center"><TD style="font-weight:bold;">37°C</TD><TD> 154 </TD><TD>285</TD><TD> 550 </TD><TD> 118 </TD><TD> 290 </TD><TD> 332
</TD></TR><TR align="center"><TD style="font-weight:bold;">42°C</TD><TD> 167</TD><TD> 273</TD><TD> 402 </TD><TD> 122 </TD><TD> 271 </TD><TD> 234
</TD></TR></TBODY></TABLE><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><B>Figure 5: Conjugative competence of oriT <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_J01003" class="external text" rel="nofollow">BBa_J01003</A> at different temperatures.</B> All of the samples have the similar cell density and similar donor : recipient ratio. The conjugation time was set to 20, 40 and 60 minutes. Transconjugants are selected by plating on the agar containing chloramphenicol and ampicillin. Note, that the conjugation efficiency after 20 and 40 minutes is the same at 37 °C and 42 °C but differs after 60 minutes.</DIV></DIV></DIV></DIV><P>
As you can see in figure 5, conjugation is most efficient at 37 °C. From 0 to 20 minutes during the primary conjugation phase the cell growth can be neglected. From 20 to 40 minutes, the cell growth is also presented in the conjugation curves. At 4 °C, the conjugation curve has a very small slope (near zero) and no significant transconjugants production could be observed. At 22, 30, 37 and 42 °C, a significant number of transconjugants could be achieved, but the conjugation efficiency and the conjugation rate (slope between 0 and 20 minutes) differ strongly. From 22 °C to 37°C, the conjugation efficiency and conjugation rate increases with rising temperature. Comparing these three curves between 20 and 40 minutes one can see that the cell growth gets more important with rising temperature. Dynamically, the cells have more mobility at higher temperature, and have a bigger possibility to touch another cell, which is necessary for conjugation. When we compared the conjugation curve at 42 °C with the one at 37 °C, one can see, that the conjugation efficient and the conjugation rate at 42 °C is as good as at 37 °C. After the primary conjugation phase, the production of transconjugants at 42 °C is clearly worse than the production at 37 °C what can be seen at comparing the curves between 40 and 60 minutes. A possible reason for this could be a lower growth and higher death rate at 42 °C compared to 37 °C. So, one can conclude, that for best conjugation efficiency one needs a warm environment for the conjugation to take place but also temperatures which favor cell growth.
[<A href="https://2008.igem.org/Team:Heidelberg/Parts/Characterization" class="external text" rel="nofollow">back</A>]
</P><H3><SPAN class="mw-headline" id="Summary">Summary</SPAN></H3><P>The characterization of <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_J01003" class="external text" rel="nofollow">BBa_J01003</A> in pSB1A2 has given the following results:
</P><UL><LI> The conjugation has an average conjugation rate of 1.05 • 10<SUP>-12</SUP> [ml • min<SUP>-1</SUP> • cell<SUP>-1</SUP>]
</LI><LI> The conjugation has an approximate transport time of 2 minutes
</LI><LI> The oriT <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_J01003" class="external text" rel="nofollow">BBa_J01003</A> will be transported first in attendance of pUB307 in the donor cells
</LI><LI> Smaller plasmids favor the conjugation and are conjugated first
</LI><LI> The oriT <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_J01003" class="external text" rel="nofollow">BBa_J01003</A> can be conjugated between Top10 cells and from Top10 to DH5α cells with similar efficiency  but from Top10 to MG1655 with a strongly lower efficiency
</LI><LI> The conjugation favors a donor : recipient ratio of 1 : 2
</LI><LI> The conjugation has the best production of transconjugants, the best efficiency and the best conjugation rate at 37 °C
</LI></UL><P>[<A href="https://2008.igem.org/Team:Heidelberg/Parts/Characterization" class="external text" rel="nofollow">back</A>]
</P><H2><SPAN class="mw-headline" id="Chloramphenicol_resistance_cassette_.28BBa_K150003.29">Chloramphenicol resistance cassette (<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K150003" class="external text" rel="nofollow">BBa_K150003</A>)</SPAN></H2><P>To characterize the ability of this protein generator to mediate resistance against chloramphenicol we used <I>E. coli</I> Top10 transformed with the part BBa_K150003 and measured their ability to grow in LB medium containing different amounts of chloramphenicol (see figure 2). As control we used untransformed <I>E. coli</I> Top10 (see figure 1). To exclude the possibility that high concentrations of chloramphenicol led to the death of the bacteria only through the solvent ethanol we also measured the same ethanol concentrations as mock treated (see figure 3).
It was shown that this part leads to acceptable resistance up to 100 µg/ml chloramphenicol, even in 500 µg/ml unmistakably cell growth can be detected. The effect of the solvent ethanol can even be at high concentrations neglected.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:702px;"><DIV class="thumbcaption"><B>Figure 1</B>: Growth of untransformed <I>E. coli</I> Top 10 in LB medium containing the indicated amounts of chloramphenicol. Note that even 5 µg/ml chloramphenicol lead to hindert cell growth and cell death.</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:702px;"><DIV class="thumbcaption"><B>Figure 2</B>: Growth of <I>E. coli</I> Top 10 transformed with BBa_K150003 in the indicated amounts of chloramphenicol. Note that this part leads to acceptable resistance up to 100 µg/ml. Even in 500 µg/ml chloramphenicol unmistakably cell growth can be detected.</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:664px;"><DIV class="thumbcaption"><B>Figure 3</B>: Growth curve of mock treated <I>E. coli</I> Top 10 to examinate the effect of the solvent on the dying cells in figure 2. It can be seen that the effect of the solvent ethanol can be neglected.</DIV></DIV></DIV></DIV><H2><SPAN class="mw-headline" id="LuxR-Colicin_E1-Receiver_.28BBa_K150009.29">LuxR-Colicin E1-Receiver (<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K150009" class="external text" rel="nofollow">BBa_K150009</A>) </SPAN></H2><H3><SPAN class="mw-headline" id="ColicinE1_Producer_Controlled_by_3OC6HSL_Receiver_Device_-_Introduction">ColicinE1 Producer Controlled by 3OC6HSL Receiver Device - Introduction</SPAN></H3><P>The constitutive expressed LuxR protein is able to build a complex with an AHL molecule. [7] This complex functions as transcription factor for the lux pR promoter and activates therefore colicinE1 production. The activation of the lux pR promoter leads additionally to expression of the kil protein, also called lysis protein by readthrough of a terminator located in the colicinE1 operon. Production of the lysis protein leads to lysis of the host cell and colicin release. The colicinE1 immunity protein (imm) located on the opposite strand is expressed under a constitutive promoter and protects the host cell from lysis up to a certain threshold of the lysis protein. 
[<A href="https://2008.igem.org/Team:Heidelberg/Parts/Characterization" class="external text" rel="nofollow">back</A>]
</P><H3><SPAN class="mw-headline" id="Colicin_E1_-_Mode_of_Action">Colicin E1 - Mode of Action</SPAN></H3><P>Colicin E1 is a 522 residue containing protein and belongs to the group A colicins. It kills sensitive cells, which harbor no ColE1 plasmid but contain the appropriated receptor BtuB, by forming pores, or more precisely ion channels into the inner membrane. [1] Affected cells can be <I>E. colis</I> or related strains but also some eukaryotic cells have been affected by the toxic activity of colicin. [2]
The first step of the lethal action of colicin E1 is binding to the vitamin B12 receptor (BtuB). Colicins are able to kill bacteria by a “one hit” mechanism. This works due to destruction of the target cells cellular energy. The voltage dependant channel in the cytoplasmatic membrane leads to a depolarization of the target cell. The conductance of the channel has been shown to be ~10<SUP>7</SUP> ions/channel-sec in 1 M NaCl. [3] 
Its ability to form pores is given by the C-terminal domain. This domain consists of 10 tightly packed α–helices (see Figure 1). [4], [6] The colicin protein is a water soluble molecule containing mostly amphipathic helices except from two hydrophobic ones. Due to these two hydrophobic helices the protein is able to target the cell membrane and act as a membrane protein.
[<A href="https://2008.igem.org/Team:Heidelberg/Parts/Characterization" class="external text" rel="nofollow">back</A>]
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:802px;"><DIV class="thumbcaption"><B> Figure 1 (a) </B> Ribbon diagram of the Colicin E1 pore-forming domain by X ray crystal structure. The channel forming domain contains 10 helices (I-X). The two hydrophobic helices (VIII and IX) are labeled yellow. <B>  (b) </B> Colicin E1 in a membrane bound state in two perspectives. Left panel shows the surface bound helices with an area of ~4200 Å2. Right panel shows colicin E1 bound to a lipid bilayer. The intra-membrane sections of the two helices VIII and IX are shown in yellow.</DIV></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Killing_efficiency_on_bacteria">Killing efficiency on bacteria</SPAN></H3><P>To measure the killing efficiency and which amount of cells or colicins are needed to reach any killing activity a colicin activity test was carried out. Therefore bacteria containing the colicin part (<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K150009" class="external text" rel="nofollow">BBa_K150009</A> in TOP10 or MG1655) and GFP producing cells (reference promoter, TOP10) were inoculated in TB-media with appropriate antibiotics at 37 °C for 4 to 6 hours and the optical density of the two strains was adjusted. The colicin cells were added in different ratios to a constant amount of GFP producing cells. The total volume was kept constant by adding TB-media (without antibiotics). The colicin production was induced by several concentrations (0 M-100 nM) of N-Acyl-Homoserin-Lactone (AHL). The OD and GFP intensities were measured at 37 °C in the Tecan Microplate Reader every 30 minutes for about 12 hours.
As a negative control the similar test was carried out with cells containing the same plasmid without the colicin gene on it. 
Figure 2 shows results of tests with a prey-killer ratio of 100:1. Due to the correlation of the GFP intensity and the optical density, GFP intensity was used as marker for the living prey cells. In all experiments using killer cells and high AHL concentrations in the medium, the prey cells were killed completely. In reference experiments (see Figure 2, bottom) using <I>E. coli</I> TOP 10 cells harboring a LuxR-receiver without colicin operon (comparable to part <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_T9002" class="external text" rel="nofollow">BBa_T9002</A> without GFP), the prey cells were able to grow for each AHL concentration. Consequently the lethal action of the killing part could be shown. 
[<A href="https://2008.igem.org/Team:Heidelberg/Parts/Characterization" class="external text" rel="nofollow">back</A>]
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:902px;"><DIV class="thumbcaption"><B> Figure 2: Results of colicin E1 toxicity tests for different AHL concentrations with a prey-killer ratio of 100:1.</B> Top left panel: GFP intensities of prey cells treated with killer cells over time. Top right panel: Optical densities of prey cells treated with killer cells plotted against time. Bottom left panel: GFP intensities of prey cells treated with reference cells versus time. Top right panel: Optical densities of prey cells treated with reference cells over time. The graphs proof the toxicity of colicin E1 as well as the functionality of the part <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K150009" class="external text" rel="nofollow">BBa_K150009</A>. Each killer cell is able to kill up to 100 prey cell if there is a sufficient amount of AHL present.</DIV></DIV></DIV></DIV><P>
Additionally it can be seen that the killing efficiency is related to the AHL concentration in the medium. Figure 3 shows a dose-response curve of GFP intensity after 12 hours dependent on the AHL concentration. The P<SUB>LuxR</SUB> promoter has to be activated with an AHL concentration above 500 pM. Thus enough colicin E1 is produced and released to kill the prey cells (see Figure 3).
[<A href="https://2008.igem.org/Team:Heidelberg/Parts/Characterization" class="external text" rel="nofollow">back</A>]
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:755px;"><DIV class="thumbcaption"><B> Figure 3: Dose-response curve of AHL concentration and killing efficiency of the killer strain for a prey-killer ratio of 100:1. </B> The GFP intensities at t = 12 h are plotted against AHL concentrations (0 M – 1 nM). For AHL concentrations above 600 nM all preys were killed.</DIV></DIV></DIV></DIV><P>
Regarding the prey survival, dependent on the prey-killer ratio, several effects can be observed (see Figure 4). Having a prey-killer ratio of 1:1, or even a higher killer fraction, all prey cells were already killed when no AHL is present. This effect is caused by the leakiness of the P<SUB>LuxR</SUB> promoter. For prey-killer ratios between 5:1 and 100:1 the killing efficiency can be regulated by the AHL concentration. In experiments with prey-killer ratios higher than 100:1 the growth of the prey strain was not influenced. 
[<A href="https://2008.igem.org/Team:Heidelberg/Parts/Characterization" class="external text" rel="nofollow">back</A>]
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:902px;"><DIV class="thumbcaption"><B> Figure 4: Toxicity of LuxR-colicin E1-receiver part for different prey-killer ratios.</B> The graphs show GFP intensities of prey cells at t = 0 h and t = 12 h. For prey-killer ratios of 1:1 all prey cells were killed after 12 hours even if there was no AHL present. For prey-killer ratios between 5:1 (not shown) and 100:1 the killing efficiency depends on the AHL concentration in the media. If there is a greater predominace of prey cells (500:1) the growth of the prey population is not influenced.</DIV></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Killer-prey_system">Killer-prey system</SPAN></H3><P>To measure the functionality the killer-prey system were tested. Therefore bacteria containing the colicin E1 part (<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K150009" class="external text" rel="nofollow">BBa_K150009</A> in TOP10 or MG1655) and prey cells containing an AHL-producing part (<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K150000" class="external text" rel="nofollow">BBa_K150000</A>) were inoculated in TB-media with appropriate antibiotics at 37 °C for 4 to 6 hours. The optical density of the two strains was adjusted. The colicin E1 producing cells were added in different ratios to a constant amount of prey cells. The total volume was kept constant by adding TB-media (without antibiotics). The colicin production was induced by several concentrations (0 M-100 nM) of N-Acyl-Homoserin-Lactone (AHL). The OD and GFP intensities were measured at 37 °C in the Tecan Microplate Reader every 30 minutes for about 12 hours.
As a negative control the similar test was carried out with cells, containing the same plasmid without the colicin E1 gene on it. 
Figure 5 shows that the prey cells were killed for prey-killer ratios from 1:1 up to 25:1. Therefore it is proven that our system works as expected. 
[<A href="https://2008.igem.org/Team:Heidelberg/Parts/Characterization" class="external text" rel="nofollow">back</A>]
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:799px;"><DIV class="thumbcaption"><B> Figure 5: Killer-prey system test. </B> GFP intensity of prey strains was measured over 12 h at 37 °C. Prey cells were killed for prey-killer ratios from 1:1 up to 25:1 (blue, green, red, light blue) due to the colicin E1 production of the killer strain. This production was activated by the AHL secretion of the prey strain. In a control experiment prey cells were able to grow (violet line).</DIV></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Lysis_of_killing_strain">Lysis of killing strain</SPAN></H3><P>Besides the killing efficiency the time course of the killer strain lysis was analyzed. Therefore growth curves of killer cells (<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K150009" class="external text" rel="nofollow">BBa_K150009</A> in <I>E. coli</I> TOP10 cells) induced with different AHL concentrations were measured (see Figure 6). 
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:902px;"><DIV class="thumbcaption"><B> Figure 6: Lysis test with LuxR-colicin E1-receiver dependent on AHL concentrations. </B>  On the left panel the growth of a killer population induced with different concentrations of AHL is shown. On the right panel the data of control experiments are plotted. For AHL concentrations between 0 M and 1 nM no influence on the growth of the killer population is visible. For concentrations between 5 nM and 25 nM the population grew for one hour, afterwards a lysis effect can be observed.</DIV></DIV></DIV></DIV><P>
During the first hour of the measurement no effect can be observed. Afterwards cell lysis dependent on AHL concentrations is visible. For AHL concentrations between 0 M and 1 nM only few cells lysed. Thus growth of the population is hardly influenced. Concentrations from 5 nM up to 25 nM show a strong effect on the amount of lysed killer cells. For 5 nM and 10 nM the growth curves flatten out. After reaching a maximum during the third and fourth hour the population size decreases and converges to a constant value. For higher AHL concentrations, e.g. 25 nM, a similar effect can be observed, but it is much stronger. The population shows only a weak growing tendency but then converges directly to a constant value. In addition to these growth measurements lysis of the killer cells was observed under the microscope. Figure 7 shows killer cells which were induced by AHL (left panel, t = 0 min). After 30 minutes one cell is lysed (right panel).
[<A href="https://2008.igem.org/Team:Heidelberg/Parts/Characterization" class="external text" rel="nofollow">back</A>]
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:802px;"><DIV class="thumbcaption"><B> Figure 7: Visualization of the lysis effect of the killer cells.</B> The prey (GFP) and the killer (mCherry) strains are mixed on a 0.5 % agarosepad. Killer cells were induced with AHL before adding to the medium. The killer cell on the left picture (white arrow) is lysed after 30 minutes. This effect is caused by the AHL induction. Colicin production and, at the same time in lower amounts, the lysis protein production leads to lysis of the host cell and colicin release.</DIV></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Eukaryotic_killing_Assay_.28Preliminary_Results.29">Eukaryotic killing Assay (Preliminary Results)</SPAN></H3><P>The toxicity of colicin E1 on eukaryotic cells was tested by using breast cancer cells (MCF 7). They were inoculated with killer cells (BBa_K150009 in TOP 10) and reference cells (BBa_150009-like, no colicin E1 operon) for several hours.
</P><P>Therefore MCF-7 cells (breast cancer cell line) were plated in 24 well plates (150 thousand cells per well) with 10 % FCS (fetal calf serum) DMEM (Dulbecco's Modified Eagle Medium) and incubated overnight at 37 °C, 5 % CO2.
E. coli TOP 10 strains containing LuxR-colicin E1-receiver (BBa_K150009) and strains containing LuxR-receiver (no colicin) were grown overnight in TB media with appropriate antibiotics at 37 °C to an OD of 1.0 and diluted in the morning to an OD of 0.1. Once bacterial cells reached an OD of 0.35 they were harvested and inoculated on the MCF-7 wt and on MCF-7 GFP producing cells. [<A href="https://2008.igem.org/Team:Heidelberg/Parts/Characterization" class="external text" rel="nofollow">back</A>]
</P><P>Once MCF-7 cells were confluent (12 h after passage), E. coli TOP 10 bacteria were spinned down (1 ml of OD = 0.35) and resuspended with DMEM medium from the original wells containing the MCF-7 wt/MCF-7 GFP cells. A 24-well plate was used containing MCF-7 cells stained with Propidium Iodide (PI) alone, with LuxR-receiver cells not containing colicin and LuxR-receiver-colicin E1 cells. AHL was added to every well to a final concentration of 5 nM. Well content was collected at different time-points (0 – 6 h) and when necessary PI was added to the suspension 15 minutes before Flow Cytometer analysis.[<A href="https://2008.igem.org/Team:Heidelberg/Parts/Characterization" class="external text" rel="nofollow">back</A>]
</P><P>At the referred timepoints, supernatants were collected and spinned down. Eukaryotic cells were detached by trypsinization (200 μl) for 10 minutes at 37 °C and then resuspended in 800 μl of KH + 1 % BSA (Krebs Henseleit medium with 1 % Bovine Serum Albumin). After resuspending, well content was added to the precipitated supernatant to account for floating dead cells. PI was added to each sample (1:1000) and incubated at RT for 15 minutes before acquisition. PI positive cells and GFP expressing cells (%) were measured for 3000 cells for each condition. Histograms and fluorescent measurements were obtained using a Cytomics FC 500 MPL from Beckman&amp;Coulter. [<A href="https://2008.igem.org/Team:Heidelberg/Parts/Characterization" class="external text" rel="nofollow">back</A>]
</P><P>MCF-7 cell death, measured by Flow Cytometry, was evaluated by % of stained cells with PI over the time, using colicin producing E. coli or strains which do not produce colicins as a control (Figure 1). E. coli was also quantified (data not shown). The PI measurements (filter FL3-620nm) were gated only to Mcf-7 cell population. [<A href="https://2008.igem.org/Team:Heidelberg/Parts/Characterization" class="external text" rel="nofollow">back</A>]
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:632px;"><DIV class="thumbcaption"><B>Figure 8 Positive PI MCF-7 cells (%) over time.</B> E coli bacteria producing colicin E1 protein cause cell death at the earliest time point (2 h) and keep it at higher levels compared to the other conditions. After 3 h, the reference strain also starts to present PI positive cells that increase over time. Nevertheless a lethal effect of the colicin producing killer cells on the eukaryotic cancer sells can be assumed.</DIV></DIV></DIV></DIV><P>[<A href="https://2008.igem.org/Team:Heidelberg/Parts/Characterization" class="external text" rel="nofollow">back</A>]
</P><P>The initial results show that killing by E. coli producing colicin E1 is effective on eukaryotic cancer cells (MCF-7). Propidium iodide (or PI) is an intercalating agent and a fluorescent molecule with a molecular mass of 668.4 Da that can be used to stain DNA. It can be used to differentiate, apoptotic and normal cells, since it is membrane impermeant and generally excluded from viable cells. The high percentage of positive PI cells refers to cells that are permeable to it, thus, going into apoptosis. In this assay, after 2 h of incubation with the eukaryotic cells, the death rate was close to 40 % of the whole eukaryotic cell population. In comparison with the control conditions (MCF-7 alone + PI), where cells (maximum of 5%) due to trypsin treatment die, the effect is clearly visible. Furthermore, looking at the eukaryotic cell death after incubation with the reference strain (LuxR-receiver TOP 10), it can be suggested that the simple incubation of these mammalian cells with bacteria is causing cell death over time (maximum of 20% after 5 h). This effect is highly enhanced and faster when the bacteria produce colicin E1 (maximum of 37 % at 2 h). After 5 h, both bacterial strains produce similar results in killing the mammalian cancer cells. After reaching a peak, the killing efficiency drops, with a significant higher peak for E. coli bacteria producing colicin E1 than for the reference strain.
On the other hand, using a stable cell line of MCF-7 expressing GFP, the effects of bacteria were not visible for the initial timepoints. The aim of this assay was to see the GFP signal droping down as E. coli progressed to kill the eukaryotic cells. Since GFP is a very robust protein we can suggest that colicin E1 producing bacteria start apoptotic events at early time (2 h, see Figure 8), but GFP loss can only be observed after a certain time delay (see Figure 9).
For that reason, it would be necessary to observe also later timepoints to characterize the killing effect by GFP loss. [<A href="https://2008.igem.org/Team:Heidelberg/Parts/Characterization" class="external text" rel="nofollow">back</A>]
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:633px;"><DIV class="thumbcaption"><B>Figure 9: GFP expressing MCF-7 cells (%) over time.</B> E coli bacteria producing colicin E1 protein cause GFP loss after 6 h, to an extend of 60% loss in comparison with the both control conditions that remained around 99 % GFP detection.</DIV></DIV></DIV></DIV><P>
Summerized, the preliminary results suggest that colicin E1 producing E. coli bacteria might be able to kill cancer cells at a high rate. However, further studies need to provide more insight into these events, as it might be by other pro-apoptotic markers or by repeating this experiment using different conditions (timepoints, autoinducer concentrations, another eukaryotic cell type). Future studies should yield to information about the colicin E1 target in eukaryotic cells and perhaps answers how it can kill cancer cells and spare healthy ones. [<A href="https://2008.igem.org/Team:Heidelberg/Parts/Characterization" class="external text" rel="nofollow">back</A>]
</P><H3><SPAN class="mw-headline" id="References_2">References</SPAN></H3><P>[1] Cramer W. et al. (1990) Structure and dynamics of the colicin E1 channel. Molecular Microbiology 4: 519-526. 
[2] Smarda J. et al. (2001) Cytotoxic effects of colicins E1 and E3 on v-myb-transformed chicken monoblasts. Folia Microbiol. (Praha) 47: 11–13. 
[3] Bullock J. et al. (1983) Comparison of the macroscopic and single channel conductance properties of colicin E1 and its COOH-terminal tryptic peptide. J. Biol. Chem. 258: 9908–9912. 
[4] Elkins P. et al. (1997) A mechanism for toxin insertion into membranes is suggested by the crystal structure of the channel-forming domains of Colicin E1. Structure. 5: 443-458. 
[5] Lindeberg M. et al. (2000) Unfolding Pathway of the Colicin E1 Channel Protein on a Membrane Surface. J. Mol. Biol. 295: 679-692 
[6] Davies, J. et al. (1975) Genetics of resistance to colicins in Escherichia coli K12: cross-resistance among resistance of group A. J. Bacteriol. 123:102–117
[7] Engebrecht J. et al. (1984) Identification of genes and gene products necessary for bacterial bioluminescence. Proc. Natl. Acad. Sci. USA 81: 4154–58
</P><P>[<A href="https://2008.igem.org/Team:Heidelberg/Parts/Characterization" class="external text" rel="nofollow">back</A>]
</P><H2><SPAN class="mw-headline" id="LuxI_3OC6HSL_sender_device_.28BBa_K150000.29">LuxI 3OC6HSL sender device (<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K150000" class="external text" rel="nofollow">BBa_K150000</A>)</SPAN></H2><H3><SPAN class="mw-headline" id="General_Description">General Description</SPAN></H3><P>AHL, a cell-cell signaling molecule is produced by the LuxI protein. The LuxI protein is expressed due to the <I>luxI</I> gene which is under the control of a constitutive promoter (<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_J23107" class="external text" rel="nofollow">BBa_J23107</A>). For this reason no input is needed for induction of gene expression. As output AHL can be measured. 
[<A href="https://2008.igem.org/Team:Heidelberg/Parts/Characterization" class="external text" rel="nofollow">back</A>]
</P><H3><SPAN class="mw-headline" id="Quantification_of_AHL_production">Quantification of AHL production</SPAN></H3><P>For AHL quantification the amount of AHL in supernatant of producing cells (<I>E. coli</I> TOP 10 containing <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K150000" class="external text" rel="nofollow">BBa_K150000</A>) was determined. Therefore eight samples containing TB-media were inoculated with 2 µl/ml of an AHL producing cell overnight culture at 37 °C. Sterile filtered supernatant of these samples was produced hourly for 7 hours and stored at 4 °C. To estimate the amount of AHL producing cells the optical density was measured before. (Figure 1, top left). 
[<A href="https://2008.igem.org/Team:Heidelberg/Parts/Characterization" class="external text" rel="nofollow">back</A>]
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:902px;"><DIV class="thumbcaption"><B>Figure 1: Results of sender activity test.</B> Top left: Growth curve of sender culture. Top right: GFP/OD values of sender supernatants at t = 10 h. Bottom left: GFP/OD – AHL calibration curve. Dotted line visualized regression of an exponential growth curve. Bottom right: Produced AHL concentrations of the sender cells.</DIV></DIV></DIV></DIV><P>
For the AHL determination 1 ml supernatant was diluted with 1 ml TB-media and inoculated with a constant amount of AHL inducible cells which produce GFP after induction (<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_T9002" class="external text" rel="nofollow">BBa_T9002</A> in <I>E. coli</I> TOP 10). The OD and GFP intensities were measured at 37 °C half-hourly for about 28 hours. For calibration the same test were performed with distinct AHL concentrations from 0M to 50 nM (SIGMA). Figure 2 shows the GFP/OD development over the time depending on the sender supernatants (left panel) or the preassigned AHL concentrations (right panel). The values increase and reach a maximum after 6 hours. Then they decrease slightly.
[<A href="https://2008.igem.org/Team:Heidelberg/Parts/Characterization" class="external text" rel="nofollow">back</A>]
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:852px;"><DIV class="thumbcaption"><B>Figure 2: GFP/OD results of sender activity measurements.</B>  Left panel shows the GFP/OD values for induction with sender supernatants. Right panel shows the GFP/OD values for predefined AHL inductions. During the first six hours the values increase and reach a maximum until the decrease slightly. The values at t = 10 h were used for calculations of AHL concentrations in supernatants.</DIV></DIV></DIV></DIV><P>
A GFP/OD – AHL calibration curve (Figure 1 bottom left) was calculated using the GFP/OD values at t = 10 h. An exponential growth curve was fitted to these data (blue dotted line; y = 4325 * e<SUP>0.00346*x</SUP>). The AHL concentrations of the sender supernatants were determined using the GFP/OD values at t = 10 h and the exponential regression. In the course of the time the AHL concentration increase until it reaches a certain maximum before a slight decrease (see Figure 1, bottom right, Table 1). The maximum is reached at the end of the logarithmic phase. Several tests have shown that the maximal concentration that can be produced by part <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K150000" class="external text" rel="nofollow">BBa_K150000</A>, is approximately 1100 and 1300 pM.
</P><P>
[<A href="https://2008.igem.org/Team:Heidelberg/Parts/Characterization" class="external text" rel="nofollow">back</A>]
</P><H2><SPAN class="mw-headline" id="LuxS_from_Vibrio_harveyi_.28partly_characterized.29">LuxS from <I>Vibrio harveyi</I> (partly characterized)</SPAN></H2><P>LuxS encodes for the protein (LuxS autoinducer synthase) that is responsible for the synthesis of autoinducer 2 (AI-2) in many species of Gram-negative and Gram-positive bacteria. This LuxS construct is cloned from <I>Vibrio harveyi</I>, a Gram-negative bioluminescent marine bacterium, which regulates light production in response to two distinct autoinducers (AI-1 and AI-2). [1]	
[<A href="https://2008.igem.org/Team:Heidelberg/Parts/Characterization" class="external text" rel="nofollow">back</A>]
</P><H3><SPAN class="mw-headline" id="Functional_check_of_LuxS.2FAI-2_production">Functional check of LuxS/AI-2 production</SPAN></H3><P>The functionality of the new LuxS part that was cloned by PCR from <I>V. harveyi</I> (BB120) in a ptrc99α plasmid was proven by measurement of luciferase activity in Vibrio as a reporter strain. DH5α cells with the new construct pTrc99α_LuxS were grown up and the supernatant was harvested by using centrifugation and sterile filtration. The gained supernatant was added to the reporter cells in different dilutions and incubated for three hours. Luminescence and OD were measured after 0.5h and 3.5h. 
[<A href="https://2008.igem.org/Team:Heidelberg/Parts/Characterization" class="external text" rel="nofollow">back</A>]
</P><H3><SPAN class="mw-headline" id="Results">Results</SPAN></H3><P>Cell growth rate of all samples LuxS and Blank was shown to be similar in OD measurements (data not shown). Luminescence of Vibrio reporter strain (data shown in figure 1) seems to increase with the concentration of supernatant in the cell delusion. Blanks with supernatant of DH5α cells transformed with pTrc99α plasmid show a significantly smaller Luminescence and there is no increase of Luminescence for increasing supernatant concentration in the cell delusion. 
[<A href="https://2008.igem.org/Team:Heidelberg/Parts/Characterization" class="external text" rel="nofollow">back</A>]
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:610px;"><DIV class="thumbcaption"><B>Figure 4</B>: <I>Vibrio</I> cell delusions with different AI-2 supernatants 1:3, 1:5 and 1:10 were measured for luciferase activity after 30min and 3.5h incubation at 30°C using the Infinite® 200 microplate reader by Tecan, Switzerland; Supernatant sterile filtrated, no NaCl added (Protocol)</DIV></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Reference">Reference</SPAN></H3><P>[1] The LuxS family of bacterial autoinducers: biosynthesis of a novel quorum-sensing signal molecule. Mol Microbiol. 2001 Jul; 41(2):463-76.
</P><P>[<A href="https://2008.igem.org/Team:Heidelberg/Parts/Characterization" class="external text" rel="nofollow">back</A>]
</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2008.igem.org/Team:Heidelberg/Parts/Characterization">http://2008.igem.org/Team:Heidelberg/Parts/Characterization</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Heidelberg/Parts/Characterization" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Heidelberg/Parts/Characterization" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Heidelberg/Parts/Characterization&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Heidelberg/Parts/Characterization&amp;oldid=91924" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2008.igem.org:Privacy_policy" title="2008.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2008.igem.org:General_disclaimer" title="2008.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>