<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Heidelberg_7_October_2008"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:Heidelberg/7_October_2008">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Heidelberg/7_October_2008&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/7_October_2008&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/7_October_2008&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Heidelberg/7_October_2008" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Heidelberg/7 October 2008</H1><DIV id="bodyContent"><H3 id="siteSub">From 2008.igem.org</H3><P><B>Project description lambda phage</B></P><P>Antibiotics are a very powerful way to cure bacterial infections till today, but their power withers. More and more bacterial strains become resistant against different antibiotics, due to the even greater powers of evolution: chromosomal mutations do not only make them resistant against natural antibiotics, but also against newly developed synthetic ones.  Bacteria also have a very effective way to share those resistances with each other, not limited by species boundaries: the transfer of plasmids by conjugation (which will be investigated in more detail in the second part of the project). Multiresistant bacterial strains like Pseudomonas aeruginosa, Staphylococcus aureus and Enterococcus faecalis are a serious problem for the usage of antibiotics aginst infectious deseases, because they can give their plasmids to other pathogens and cannot be killed by ordinary antibiotics. 
</P><P>Nevertheless, there are some alternatives to antibiotics, which are currently researched: the usage of bacterial toxins, the usage of probiotic bacteria and the usage of bacteriophages as antibiotics. In our project we focus on two approaches: To engineer a probiotic bacteria which uses toxins as a killing device and one which uses bacteriophages, respectively.
</P><P>Our part of the team is concerned with the phage strategy. Bacteriophages are a group of viruses, for which bacteria are the natural hosts, among them human pathogens. There are two types of phages: virulent and temperent ones. The virulent ones infect a host cell and multiply by using the host resources and lyse the host to be set free. The temperent ones are able to either integrate into the host chromosome (lysogenic) or follow a lytic strategy and multiply and thereby lyse the cell. Virulent, non-lysogenic phages are used for the treatment of infectious diseases caused by bacteria since the 1930s. Phage therapy was intensely used since the first antibiotics where discovered in the 1940s. Later, this technique was extensively investigated and used especially in the Soviet Union.   Since there are more and more multiresistant bacterial strains, phage therapy becomes interesting again. 
</P><P>The special quality of our approach is that we use bacteria as vectors for the phages. This will avoid the consequences of a systemic application of bacteriophages, because they are only released at the site of infection. Smaller amounts of phages are needed, because they are not spreading through the whole body. 
</P><P>The bacteriophages we use are lambda phages, because this project should be a proof of principle for our technique, and therefore we rely on this very well investigated type of phage. The lambda phages are a class of temperent bacteriophages. They have a 48,502 bp genome consisting of linear double stranded DNA. 
Like all viruses they carry out distinct steps of phage infection. The first step is adsorption. Lambda phages have receptors at their tail fibres, which bind specifically to mannose transporters of E.coli. These receptors are not able to bind stably to structures on other bacteria strains and are therefore responsible for the host specificity of the lambda phage. After adsorption, the lambda phage injects its genome into the host cell. In the environment of the host, the linear DNA circularizes at the so called cos sites: these are 12bp complementary overhangs at the ends of the genome. 
As temperent phages the lambda phages are able to undergo either a lytic or a lysogenic cycle. Undergoing a lysogenic cycle, the infecting phage integrates its genome into the host genome and stays there as a stable prophage. It will not lyse the cell and multiply without an extern trigger – for example if the host cell is stressed. 
If carrying out a lytic cycle, the lambda phage also injects its genome into the host cell. This will then be replicated, viral proteins will be produced, first for the formation of new phage particles, which are formed via self assembly, and second for the lysis of the host cell, upon which the newly produced phages are released. 
The lambda phage has a very complex regulatory network that controls which cycle to carry out. 
The key role in this process is taken by a protein called cI. This binds to regulatory sequences that stabilize the transcription of the genes which are essential for the lysogenic cycle. The lysogenic proteins include an integrase (int), which integrates the phage genome into the host genome. If this is not present, the phage genome stays in the cell in a plasmid like state. For the induction of the lytic cycle from a lysogenic one, a protein called excisase (xis) is essential. This one cuts the phage genome out of the host genome and thereby allows the replication and transcription of it. During the lysogenic cycle, cI is expressed continuously. If another phage injects its genome into the host cell of a prophage, cI binds to regulatory sequences on this second genome and prevents any transcription, thereby making the host immune to secondary infections.
Our work focussed on designing a non-lysogenic phage, because we want to produce an efficient killing module for the probiotic bacterium. In preventing the phage from undergoing a lysogenic cycle, we assure that it will kill the target bacteria as fast as possible and without extern triggers. For this task we aimed to discard the int gene from the phage genome, so that it would be incapable of integrating into the host genome and being lytically inactive      .  		         
</P><P>Cloning with the lambda phage genome is tricky, because it contains few single cutter restriction sites. Therefore we had to cut out one fragment containing the int, the xis and another gene, gam. Xis and int can and shall be discarded, but the gam gene codes for a protein that prevents the overhanging single strains at the cos sites from being degraded by host enzymes. So we would need the gam gene in our engineered phage. This is why we designed an insert for the phage to put in the remaining genome after cutting out the fragment containing xis, int and gam. This insert contains gam, and an antibiotic selection marker to be able to select those bacterial clones that contain the lambda genome. 
</P><P>
We also added a GFP gene for visualizing and measurement purposes and an OriT (see protocol 2). To be able to later modify this phage insert again, we added special restriction sites to only cut out the non-phage genes. 
</P><P>To avoid that our probiotic bacteria, that would contain and transfer the phage to the target pathogenic bacteria, would also be killed by the phage, we used the natural way to make them immune. Therefore we took the lambda cI gene and put it behind a constitutive promoter. The cI protein will then prevent the expression of the lambda genome in our bacteria and prevent them from being lysed. 
</P><P><B>Results</B></P><P>Cutting the lambda phage
</P><P>As restriction enzymes we took XhoI and XbaI to cut out the xis and int – fragment. The resulting parts are the 9 kb fragment and the rest genome of 40 kb. (picture and exact data). Because the lambda DNA comes linear, there are different fragments for the rest genome if put on a gel: the two ‘arms’ of the vector (kb) and some hybridized complete rest (40 kb). 
</P><P>The new insert (take, if it gets finished, the second strategy, maybe even with the test results of the first one – just mention it)
</P><P>The new insert consisting of the gam, a GFP, a chloramphenicol gene and an OriT was designed by PCR (each part on its own) and after this cloned together in pBluescript. (cloning strategy)
</P><P>The parts in pBluescript are functional (for conjugation results see protocol 2).
</P><P>Ligation into the lambda vector
</P><P>The insert was ligated into the purified lambda vector. Then the ligation was transformed and cultivated for 2 to 3 days at 28°C. This was done because the lambda DNA we were using had a temperature sensitive cI. This is stable at temperatures under 37°C and thereby preventing lysis of the cells, and at more than 37°C it changes its conformation to an inactive state and the lytic cycle of the phage will be induced. Therefore we could amplify the ligated lambda genome at 28°C to later extract it via miniprep.
</P><P>Infection tests
</P><P>Infection tests with the lambda phage had to be carried out at 42°C, so cI would be inactive and could not prevent the lytic cycle.
</P><P><B>Outlook</B></P><P>Up to now, our system is a proof of principle that the usage of phages as a killer module for probiotic bacteria works. For a later application, either the host range of the lambda phage has to be altered, to broaden the range of target pathogens. Another way would be to use other pahges with other host ranges within the same system. The problem with this approach is, that they have different regulatory mechanisms and therefore the engineering work would have to start new with each phage.
</P><P>The most elegant way would be to broaden the host range of the lambda phage. Since phages identify their host via specific receptors on their tail fibers that bind to structures on the surface of their host one would have to modify or exchange these receptors. One approach could be to make chimeric receptors out of the lambda regulatory part and the identifying part of the receptor of another phage.
</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2008.igem.org/Team:Heidelberg/7_October_2008">http://2008.igem.org/Team:Heidelberg/7_October_2008</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Heidelberg/7_October_2008" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Heidelberg/7_October_2008" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Heidelberg/7_October_2008&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Heidelberg/7_October_2008&amp;oldid=52569" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2008.igem.org:Privacy_policy" title="2008.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2008.igem.org:General_disclaimer" title="2008.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>