<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Heidelberg_Notebook_Killing_II_9thweek"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:Heidelberg/Notebook/Killing_II/9thweek">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Heidelberg/Notebook/Killing_II/9thweek&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/Notebook/Killing_II/9thweek&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Heidelberg/Notebook/Killing_II/9thweek&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Heidelberg/Notebook/Killing_II/9thweek" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Heidelberg/Notebook/Killing II/9thweek</H1><DIV id="bodyContent"><H3 id="siteSub">From 2008.igem.org</H3><DIV class="MenuBar" id="navi"><UL><LI><A href="https://2008.igem.org/Team:Heidelberg" style="color: white">Home
			</A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Team" style="color: white">Team</A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Team"><SPAN><SPAN>Overview</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Team_Members#Advisors"><SPAN><SPAN>Advisors</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Team_Members#Undergraduates"><SPAN><SPAN>Undergraduates</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Team#There_are_many_Universities_-_and_there_is_Heidelberg_..."><SPAN><SPAN>University</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Team#DKFZ"><SPAN><SPAN>DKFZ</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Team#Bioquant"><SPAN><SPAN>BioQuant</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/BioRegioRN"><SPAN><SPAN>BioRegion Rhein-Neckar</SPAN></SPAN></A></LI></UL><LI><A href="https://2008.igem.org/Team:Heidelberg/Project" style="color: white">Project</A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Project"><SPAN><SPAN>Overall Project</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Notebook/material"><SPAN><SPAN>Material &amp; Methods</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Project/Sensing"><SPAN><SPAN>Sensing</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Project/Killing_I"><SPAN><SPAN>Killing I  - Phages</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Project/Killing_II"><SPAN><SPAN>Killing II - Colicin</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Project/Visualization"><SPAN><SPAN>Visualization</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Parts" style="color: white">Parts</A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Parts"><SPAN><SPAN>Submitted Parts</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Parts/Characterization"><SPAN><SPAN>Characterization</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Modeling" style="color: white">Modeling</A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Modeling"><SPAN><SPAN>Overview</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Modeling/Chemotaxis"><SPAN><SPAN>Chemotaxis-Colicin</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Modeling/Phage"><SPAN><SPAN>Phage Dynamics model</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Notebook/Overview" style="color: white">Notebook</A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Notebook/Sensing_Group/Notebook"><SPAN><SPAN>Sensing &gt;</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Notebook/Sensing_Group/Cloning"><SPAN>Cloning strategy</SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Notebook/Sensing_Group/Notebook"><SPAN>Notebook</SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Notebook/Killing_I/Notebook"><SPAN><SPAN>Killing I - Phages &gt;</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Notebook/Killing_I/Cloning"><SPAN>Cloning strategy</SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Notebook/Killing_I/Notebook"><SPAN>Notebook</SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Notebook/Killing_II/Notebook"><SPAN><SPAN>Killing II - Colicin &gt;</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Notebook/Killing_II/Cloning"><SPAN>Cloning strategy</SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Notebook/Killing_II/Notebook"><SPAN>Notebook</SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Notebook/visualization"><SPAN><SPAN>Visualization</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Notebook/material"><SPAN><SPAN>Material &amp; Methods</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Notebook/team_meetings"><SPAN><SPAN>Team Meetings</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Notebook/seminar"><SPAN><SPAN>Seminar on Synthetic Biology</SPAN></SPAN></A></LI><LI style="width: 160px"><A href="https://2008.igem.org/Team:Heidelberg/Human_Practice/Project_Overview" style="color: white">Human Practice</A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Human_Practice/Project_Overview"><SPAN><SPAN>Project Overview</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Human_Practice/Phips_the_Phage"><SPAN><SPAN>Phips the Phage</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Human_Practice/Essay"><SPAN><SPAN>Essay</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Human_Practice/Surveys"><SPAN><SPAN>Surveys</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Human_Practice/Open_Day"><SPAN><SPAN>Open Day</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Human_Practice/Nobel_Prize"><SPAN><SPAN>Nobel Prize</SPAN></SPAN></A></LI><LI><A href="https://2008.igem.org/Team:Heidelberg/Sponsors" style="color: white">Sponsors</A></LI></DIV><P><B>9<SUP>th</SUP> week</B></P><P><A href="/Team:Heidelberg/Notebook/Killing_II/8thweek" title="Team:Heidelberg/Notebook/Killing II/8thweek"> go back to 8<SUP>th</SUP> week</A></P><P><A href="/Team:Heidelberg/Notebook/Killing_II/Notebook" title="Team:Heidelberg/Notebook/Killing II/Notebook">go back to the overview</A></P><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Monday_09.2F29.2F2008"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Monday 09/29/2008</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#pSB1A3-Receiver-Colicin_cloning"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">pSB1A3-Receiver-Colicin cloning</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#HisTag_cloning_of_Colicins_for_purification"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">HisTag cloning of Colicins for purification</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Sender_cloning:_pBAD-sender"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Sender cloning: pBAD-sender</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Sender_cloning:_constitutive_promotor-sender"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">Sender cloning: constitutive promotor-sender</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-6"><A href="#Tuesday_09.2F30.2F2008"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Tuesday 09/30/2008</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#pSB1A3-Receiver-Colicin_cloning_2"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">pSB1A3-Receiver-Colicin cloning</SPAN></A></LI><LI class="toclevel-3 tocsection-8"><A href="#Receiver-Colicin_E9.2FE9_lys"><SPAN class="tocnumber">2.1.1</SPAN><SPAN class="toctext">Receiver-Colicin E9/E9_lys</SPAN></A></LI><LI class="toclevel-3 tocsection-9"><A href="#Receiver-Colicin_E1"><SPAN class="tocnumber">2.1.2</SPAN><SPAN class="toctext">Receiver-Colicin E1</SPAN></A></LI><LI class="toclevel-2 tocsection-10"><A href="#HisTag_cloning_of_Colicins_for_purification_2"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">HisTag cloning of Colicins for purification</SPAN></A></LI><LI class="toclevel-2 tocsection-11"><A href="#Sender_cloning:_pBAD-sender_2"><SPAN class="tocnumber">2.3</SPAN><SPAN class="toctext">Sender cloning: pBAD-sender</SPAN></A></LI><LI class="toclevel-2 tocsection-12"><A href="#Sender_cloning:_constitutive_promotor-sender_2"><SPAN class="tocnumber">2.4</SPAN><SPAN class="toctext">Sender cloning: constitutive promotor-sender</SPAN></A></LI><LI class="toclevel-1 tocsection-13"><A href="#Wednesday_10.2F01.2F2008"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Wednesday 10/01/2008</SPAN></A></LI><LI class="toclevel-2 tocsection-14"><A href="#pSB1A3-Receiver-Colicin_cloning_3"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">pSB1A3-Receiver-Colicin cloning</SPAN></A></LI><LI class="toclevel-1 tocsection-15"><A href="#Thursday_10.2F02.2F2008"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Thursday 10/02/2008</SPAN></A></LI><LI class="toclevel-2 tocsection-16"><A href="#pSB1A2-Receiver-Colicin_cloning"><SPAN class="tocnumber">4.1</SPAN><SPAN class="toctext">pSB1A2-Receiver-Colicin cloning</SPAN></A></LI><LI class="toclevel-1 tocsection-17"><A href="#Friday_10.2F03.2F2008"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Friday 10/03/2008</SPAN></A></LI><LI class="toclevel-2 tocsection-18"><A href="#pSB1A2-Receiver-Colicin_cloning_2"><SPAN class="tocnumber">5.1</SPAN><SPAN class="toctext">pSB1A2-Receiver-Colicin cloning</SPAN></A></LI><LI class="toclevel-2 tocsection-19"><A href="#Colicin_activity_test"><SPAN class="tocnumber">5.2</SPAN><SPAN class="toctext">Colicin activity test</SPAN></A></LI><LI class="toclevel-1 tocsection-20"><A href="#Saturday_10.2F04.2F2008"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">Saturday 10/04/2008</SPAN></A></LI><LI class="toclevel-2 tocsection-21"><A href="#pSB1A3-Receiver-Colicin_cloning_4"><SPAN class="tocnumber">6.1</SPAN><SPAN class="toctext">pSB1A3-Receiver-Colicin cloning</SPAN></A></LI><LI class="toclevel-2 tocsection-22"><A href="#HisTag_cloning_of_Colicins_for_purification_3"><SPAN class="tocnumber">6.2</SPAN><SPAN class="toctext">HisTag cloning of Colicins for purification</SPAN></A></LI><LI class="toclevel-2 tocsection-23"><A href="#Sender_Cloning:_pBAD_-_sender"><SPAN class="tocnumber">6.3</SPAN><SPAN class="toctext">Sender Cloning: pBAD - sender</SPAN></A></LI><LI class="toclevel-2 tocsection-24"><A href="#Sender_Cloning:_constitutive_promotor_-_sender"><SPAN class="tocnumber">6.4</SPAN><SPAN class="toctext">Sender Cloning: constitutive promotor - sender</SPAN></A></LI><LI class="toclevel-1 tocsection-25"><A href="#Sunday10.2F05.2F2008"><SPAN class="tocnumber">7</SPAN><SPAN class="toctext">Sunday10/05/2008</SPAN></A></LI><LI class="toclevel-2 tocsection-26"><A href="#pSB1A3-Receiver-Colicin_cloning_5"><SPAN class="tocnumber">7.1</SPAN><SPAN class="toctext">pSB1A3-Receiver-Colicin cloning</SPAN></A></LI><LI class="toclevel-3 tocsection-27"><A href="#Colicin_E9_lys"><SPAN class="tocnumber">7.1.1</SPAN><SPAN class="toctext">Colicin E9 lys</SPAN></A></LI><LI class="toclevel-3 tocsection-28"><A href="#Colicin_E1"><SPAN class="tocnumber">7.1.2</SPAN><SPAN class="toctext">Colicin E1</SPAN></A></LI></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Monday_09.2F29.2F2008">Monday 09/29/2008</SPAN></H2><H3><SPAN class="mw-headline" id="pSB1A3-Receiver-Colicin_cloning">pSB1A3-Receiver-Colicin cloning</SPAN></H3><UL><LI>Digestion of pSB1A3-Receiver-ColicinE9/E9lys: E9 -&gt; 61, 63; E9lys -&gt; 34, 35, 47
</LI></UL><PRE>10 µl DNA
 5 µl BSA 10x (NEB) 
 5 µl EcoRI Buffer (NEB)
 1 µl EcoRI
 1 µl PstI
28 µl H<SUB>2</SUB>O
-----
50 µl
</PRE><UL><LI>Gel of Digestion: 1% Agarose, 135 V, 30 min 
</LI><LI>Results of Digestion:
</LI></UL><H3><SPAN class="mw-headline" id="HisTag_cloning_of_Colicins_for_purification">HisTag cloning of Colicins for purification</SPAN></H3><UL><LI>PCR-Screen of pQE-30-Col E9 His
</LI></UL><PRE>25.0 µl Taq Master Mix (Fermentas)
 2.5 µl ColE9_prot_fw_BamHI
 2.5 µl ColE9_prot_rv_XmaI
20.0 µl H<SUB>2</SUB>O
   1    colony
-------
50.0 µl
</PRE><PRE>program:
95 °C    3 min
95 °C    1 min          |
54 °C    1 min          | 30 cycles
72 °C    1 min          |
72 °C    10 min
 4 °C   constant
</PRE><UL><LI>Gel of  E9 PCR-Screen: 1% Agarose, 135 V, 30 min </LI><LI>Gel-Results of E9-PCR-Screen: Negative controll shows right band size -&gt; its maybe a supercoiled form of the plasmid. No positive clones.
</LI></UL><H3><SPAN class="mw-headline" id="Sender_cloning:_pBAD-sender">Sender cloning: pBAD-sender</SPAN></H3><UL><LI>Digestion of Sender pBAD: 1h 40min at 37°C
</LI></UL><PRE>10 µl DNA (1)
 5 µl NEBuffer 2 (NEB)
 5 µl BSA 10x
 1 µl XbaI
 1 µl SpeI
28 µl H<SUB>2</SUB>O
-----
50 µl
</PRE><UL><LI>Gel of Digestion: 1% Agarose, 135 V, 30 min </LI><LI>Results of Digestion: 
</LI></UL><UL><LI>I0500+F1610: 
</LI><LI>Expected Fragments:
</LI><LI>~ 2030bp (Insert: I0500+F1610)
</LI><LI>~ 4425bp (Backbone: pSB2K3)
</LI></UL><P>No bands visible. Probably no DNA or low concentrated DNA.
Nanodrop measurement ~80ng/µl. Despite of neg. digestion we sended one probe to GATC for sequencing.
</P><UL><LI>J23107+F1610:
</LI><LI>Expected Fragments:
</LI><LI>~ 850bp (Insert: J23107+F1610)
</LI><LI>~ 2070bp (Backbone: J61002)
</LI></UL><P>Expected bands are visible on the gel. Probes send for sequencing (GATC. 
</P><H3><SPAN class="mw-headline" id="Sender_cloning:_constitutive_promotor-sender">Sender cloning: constitutive promotor-sender</SPAN></H3><UL><LI>Digestion of Sender with constitutive promotor: 1h 40min at 37°C
</LI></UL><PRE>10 µl DNA (1)
 5 µl NEBuffer 2 (NEB)
 5 µl BSA 10x
 1 µl XbaI
 1 µl SpeI
28 µl H<SUB>2</SUB>O
-----
50 µl
</PRE><UL><LI>Gel of Digestion: 1% Agarose, 135 V, 30 min </LI><LI>Results of Digestion:
</LI></UL><P>[<A href="https://2008.igem.org/Team:Heidelberg/Notebook/Killing_II/9thweek" class="external text" rel="nofollow">back</A>]
</P><H2><SPAN class="mw-headline" id="Tuesday_09.2F30.2F2008">Tuesday 09/30/2008</SPAN></H2><H3><SPAN class="mw-headline" id="pSB1A3-Receiver-Colicin_cloning_2">pSB1A3-Receiver-Colicin cloning</SPAN></H3><H4><SPAN class="mw-headline" id="Receiver-Colicin_E9.2FE9_lys">Receiver-Colicin E9/E9_lys</SPAN></H4><UL><LI>Sequencing results: The colonies 34, 35 and 47 contains the right insert. But the sequences behind the BioBrick Suffix do not fit to <A href="http://partsregistry.org/Part:pSB1A3" class="external text" rel="nofollow">pSB1A3</A> vector. The sequence which we received from GATC has 78 bp less than the one from partsregistry.org. To fix this problem we aligned our sequencing results against the pSB1A2 vector and they fit perfect. Because of that we assume that our BBa_T9002 is inside the pSB1A2 vector and not as expected in the pSB1A3 vector. </LI><LI>Sequencing results: The colonies 61 and 63 contains the right insert, but like ColE9lys in <A href="http://partsregistry.org/Part:pSB1A2" class="external text" rel="nofollow">pSB1A2</A> vector. </LI><LI>PCR for amplification of Receiver-Colicin E9/E9_lys for recloning with the right prefix and suffix
</LI></UL><PRE>25.0 µl Phusion Master Mix (Finnzymes, NEB)
 2.5 µl T9002_LuxpR_Not_Eco_Xba_G_fw (T<SUB>m</SUB>=80,96°C)
 2.5 µl ColE9_plasmid_rv_A_SpeI(T<SUB>m</SUB>=57,48°C)/ColE9_lysProt_rv_A_SpeI (T<SUB>m</SUB>=64,29°C)
20.0 µl H<SUB>2</SUB>O
 2.0 µl Miniprep (E9: Colony 61, E9lys: Colony 34)
-------
50.0 µl
</PRE><PRE>program:
95 °C    30 sec
95 °C    10 sec          |
54 °C    30 sec          | 25 cycles
72 °C    1 min 30 sec    |
72 °C    10 min
 4 °C   constant
</PRE><UL><LI>Gel of E9 and E9 lys PCR-amplification: 1% Agarose, 135 V, 30 min 
</LI><LI>Gel-Results of E9-PCR-amplification: 
</LI><LI>Gel-Results of E9lys-PCR-amplification: 
</LI><LI>Plan for activity tests of Colicin-Receivers:
</LI><LI>Overnight measurement (Wednesday - Thursday) in Tecan plate reader with reference promotor cells (BBa_I20260) in combination with pSB1A3-Rec-ColE9/ColE9lys cells, supernatant and lysed supernatant.
</LI><LI>Pipetting scheme: </LI><LI>Legend_1: </LI><LI>Legend_2: </LI><LI>Inocculation of Overnightcultures:
</LI><LI>10 ml M9 + Kanamycin media + Reference promoter cells (BBa_I20260)
</LI><LI>6 x 3 ml M9 Media + pSB1A3-Receiver without GFP
</LI><LI>6 x 3 ml M9 Media + pSB1A3-Receiver-ColE9
</LI><LI>6 x 3 ml M9 Media + pSB1A3-Receiver-ColE9lys
</LI></UL><H4><SPAN class="mw-headline" id="Receiver-Colicin_E1">Receiver-Colicin E1</SPAN></H4><UL><LI>PCR-Screen of pSB1A3-Receiver-Colicin E1 Cloning 
</LI></UL><PRE>25.0 µl Taq Master Mix (Fermentas)
 2.5 µl ColE1_prot_fw_BamHI
 2.5 µl ColE1_prot_rv_HindIII
20.0 µl H<SUB>2</SUB>O
   5    colonies
-------
50.0 µl
</PRE><PRE>program:
95 °C    3 min
95 °C    1 min          |
54 °C    1 min          | 30 cycles
72 °C    1 min          |
72 °C    10 min
 4 °C   constant
</PRE><UL><LI>Gel of Col E1 PCR-Screen: 1% Agarose, 135 V, 30 min </LI></UL><UL><LI>Gel-Results of E1-PCR-Screen:
</LI><LI>Expected Fragmentsize: ~1580 bp
</LI><LI>Colonies 6-10 and 36-40 show the right fragment size. -&gt; New colony-PCR-Screen.
</LI></UL><H3><SPAN class="mw-headline" id="HisTag_cloning_of_Colicins_for_purification_2">HisTag cloning of Colicins for purification</SPAN></H3><H3><SPAN class="mw-headline" id="Sender_cloning:_pBAD-sender_2">Sender cloning: pBAD-sender</SPAN></H3><UL><LI>Sequencing results: No sequence -&gt; Inocculation of new ONC
</LI></UL><H3><SPAN class="mw-headline" id="Sender_cloning:_constitutive_promotor-sender_2">Sender cloning: constitutive promotor-sender</SPAN></H3><UL><LI>Sequencing results: Both colonies contain the right fragment. So we inocculate new overnight cultures for Minipreps, Glycerolstocks and activity tests. </LI></UL><P>[<A href="https://2008.igem.org/Team:Heidelberg/Notebook/Killing_II/9thweek" class="external text" rel="nofollow">back</A>]
</P><H2><SPAN class="mw-headline" id="Wednesday_10.2F01.2F2008">Wednesday 10/01/2008</SPAN></H2><H3><SPAN class="mw-headline" id="pSB1A3-Receiver-Colicin_cloning_3">pSB1A3-Receiver-Colicin cloning</SPAN></H3><UL><LI>Gel of pSB1A3-Receiver-Col E1 PCR-Singlescreen: 1% Agarose, 135 V, 30 min 
</LI><LI>Gel result:
</LI></UL><P>Primer Col E1 (above):
- T9002_LuxpR_Eco_Xba_G_fw
- ColE1_prot_rv_A_SpeI
</P><P>Expected Fragment:
~ 3200 bp
</P><P>Primer Col E1 His (below):
- T9002_LuxpR_Eco_ Xba_G_fw
- ColE1_kil_rv_HindIII
</P><P>Expected Fragment:
~ 2150 bp
</P><P>Comment:
Possible explanation for the high amount of bands.
Maybe annealing temperature was to low and primers annealed anywhere.
-&gt; Sequencing for colony 6, 38 &amp; 39
</P><UL><LI>Digestion of PCR-product Colicin E9, E9lys with right prefix and sufix: 2h 40 min at 37 °C
</LI></UL><PRE>38,5 µl DNA (PCR-amplification)
   1 µl SpeI (NEB)
 0,5 µl EcoRI (NEB)
   5 µl NEBuffer EcoRI (NEB)
   5 µl BSA 10x (NEB)
   -----
   50 µl
</PRE><UL><LI>Gel of pSB1A3-Receiver-Col E9/E9 lys digestion: 1% Agarose, 135 V, 30 min 
</LI></UL><LI>Gel result:
</LI><P>Primer Col E9:
- T9002_LuxpR_Eco_ Xba_G_fw
- ColE9_plasmid_rv_A_SpeI
</P><P>Expacted Fragment:
~ 3100 bp
</P><P>Primer Col E9 lys:
- T9002_LuxpR_Eco_ Xba_G_fw
- ColE9_lysProt_rv_A_SpeI
</P><P>Expacted Fragment:
~ 2150 bp
</P><P>Comment:
Possible explanation for the high amount of bands.
May be annealing temperature was to low and primers annealed anywhere.
</P><UL><LI>Digestion of : 2h 40 min at 37 °C
</LI></UL><PRE>   9 µl DNA (Miniprep of ColE9 colonies 61 and 63)
   1 µl SpeI (NEB)
 0,5 µl EcoRI (NEB)
   5 µl NEBuffer EcoRI (NEB)
   5 µl BSA 10x (NEB)
28,5 µl H<SUB>2</SUB>O
   -----
   50 µl
</PRE><UL><LI>Gelextraction (?)
</LI></UL><P>[<A href="https://2008.igem.org/Team:Heidelberg/Notebook/Killing_II/9thweek" class="external text" rel="nofollow">back</A>]
</P><H2><SPAN class="mw-headline" id="Thursday_10.2F02.2F2008">Thursday 10/02/2008</SPAN></H2><H3><SPAN class="mw-headline" id="pSB1A2-Receiver-Colicin_cloning">pSB1A2-Receiver-Colicin cloning</SPAN></H3><UL><LI> By sequencing we figured out, that the plasmid-backbone we used (and are still using) is not the pSB1A3 as it should be but the pSB1A2. Sequencing of the registery part BBa_T9002, where we cut out the plasmid-backbone we are using, affirmed that this part is not on the pSB1A3 plasmid as mentioned but on the pSB1A2. Because we have no working registry part on a pSB1A3 plasmid available in lab, we decided to continue with the pSB1A2 backbone and will also do our standardization with this plasmid.
</LI></UL><P><B>Sequencing results of T9002/GFP_ColE1 cloning</B></P><UL><LI> two of the sequenced colonies gave positive results: Colony 6 and Colony 39
</LI><LI> --&gt; two positive colE1 cloning constructs: <B>T9002/GFP_ColE1.6</B> and <B>T9002/GFP_ColE1.39</B></LI></UL><P><B>Continuance of T9002/GFP_colE9 and _colE9lys cloning for standardization (with correct pre-/suffixes)</B></P><UL><LI> gel (0,7 %, 135 V, 60 min) of the restriction digest done yesterday (EcoRI + PstI) with the samples for the backbone (61, 63, 63)
</LI><LI> pSB1A2 at ~2000 bp where cut out
</LI><LI> gel extraction with these bands (61, 63, 63) and PCR purification with the restriction digests done yesterday (EcoRI + PstI) with the samples ColE9 and ColE9lys. An additional made analytical gel of these two samples reminded us, that the colE9 fragment contains an EcoRI site inside the fragment. For this reason, we could not use the digested colE9 fragment and continued the cloning only with the colE9lys fragment.
</LI><LI> overnight ligation of pSB1A2 (9,1 ng/µl) and colE9lys (14,4 ng/µl)
</LI></UL><PRE> 2 µl T4 ligase buffer 10x (Fermentas)
 2 µl T4 ligase (Fermentas) 
13 µl ColE9lys
 3 µl pSB1A2
-----
20 µl
</PRE><PRE>14 h 16°C --&gt; 20 min 65°C
</PRE><P>[<A href="https://2008.igem.org/Team:Heidelberg/Notebook/Killing_II/9thweek" class="external text" rel="nofollow">back</A>]
</P><H2><SPAN class="mw-headline" id="Friday_10.2F03.2F2008">Friday 10/03/2008</SPAN></H2><H3><SPAN class="mw-headline" id="pSB1A2-Receiver-Colicin_cloning_2">pSB1A2-Receiver-Colicin cloning</SPAN></H3><P>Transformation of the ON ligations of yesterday.
</P><H3><SPAN class="mw-headline" id="Colicin_activity_test">Colicin activity test</SPAN></H3><UL><LI>Different colicin producing cells were sonified:
</LI><LI>naturally colicin producing cells (colE1 and colE9) stressed with mitomycin C and unstressed
</LI><LI>cells containing different colicin-receiver cloning constructs (colE1, colE9 and colE9lys)
</LI><LI>reference cells (T9002 without GFP)
</LI></UL><LI>this supernatants were mixed with GFP producing prey cells in different ratios
</LI><LI>measurement of OD and GFP over night
</LI><P>[<A href="https://2008.igem.org/Team:Heidelberg/Notebook/Killing_II/9thweek" class="external text" rel="nofollow">back</A>]
</P><H2><SPAN class="mw-headline" id="Saturday_10.2F04.2F2008">Saturday 10/04/2008</SPAN></H2><H3><SPAN class="mw-headline" id="pSB1A3-Receiver-Colicin_cloning_4">pSB1A3-Receiver-Colicin cloning</SPAN></H3><UL><LI>Inocculation of Overnightcultures:
</LI><LI>25 ml TB + Kanamycin media + Reference promoter cells (BBa_I20260)
</LI><LI>25 ml TB + Ampicillin media + pSB1A3-Receiver without GFP
</LI><LI>25 ml TB + Ampicillin media + pSB1A3-Receiver-ColE1
</LI></UL><H3><SPAN class="mw-headline" id="HisTag_cloning_of_Colicins_for_purification_3">HisTag cloning of Colicins for purification</SPAN></H3><H3><SPAN class="mw-headline" id="Sender_Cloning:_pBAD_-_sender">Sender Cloning: pBAD - sender</SPAN></H3><H3><SPAN class="mw-headline" id="Sender_Cloning:_constitutive_promotor_-_sender">Sender Cloning: constitutive promotor - sender</SPAN></H3><P>[<A href="https://2008.igem.org/Team:Heidelberg/Notebook/Killing_II/9thweek" class="external text" rel="nofollow">back</A>]
</P><H2><SPAN class="mw-headline" id="Sunday10.2F05.2F2008">Sunday10/05/2008</SPAN></H2><H3><SPAN class="mw-headline" id="pSB1A3-Receiver-Colicin_cloning_5">pSB1A3-Receiver-Colicin cloning</SPAN></H3><H4><SPAN class="mw-headline" id="Colicin_E9_lys">Colicin E9 lys</SPAN></H4><UL><LI>PCR-Screen of Colonies (Transformation from Fr,10/03/2008) of pSB1A3-Receiver-Col E9 lys
</LI></UL><PRE>25.0 µl Taq Master Mix (Fermentas)
 2.5 µl T9002_LuxpR_Not_Eco_Xba_G_fw (T<SUB>m</SUB>=80,96°C)
 2.5 µl ColE9_lyProt_rv_A_SpeI (T<SUB>m</SUB>=64,29°C)
20.0 µl H<SUB>2</SUB>O
   1    colony
-------
50.0 µl
</PRE><PRE>program:
95 °C    3 min
95 °C    1 min          |
64 °C    1 min          | 30 cycles
72 °C    1 min 30 sec   |
72 °C    10 min
 4 °C   constant
</PRE><UL><LI>Inocculation of Overnightcultures:
</LI><LI>12 ml TB + Kanamycin media + Reference promoter cells (BBa_I20260)
</LI><LI>23 x 2 ml TB + Ampicillin media + pSB1A3-Receiver without GFP
</LI><LI> each 2 ml induced with the following AI-1 concentration:
</LI></UL><PRE>  0 nM    200 nM
 60 nM    250 nM
 70 nM    300 nM
 80 nM    350 nM
 90 nM    400 nM
100 nM    500 nM
110 nM    600 nM
120 nM    700 nM
130 nM    800 nM
140 nM    900 nM
150 nM      1 µM
160 nM
</PRE><UL><LI>23 x 2 ml TB + Ampicillin media + pSB1A3-Receiver-ColE9 lys
</LI><LI> each 2 ml induced with the following AI-1 concentration:
</LI></UL><PRE>  0 nM    200 nM
 60 nM    250 nM
 70 nM    300 nM
 80 nM    350 nM
 90 nM    400 nM
100 nM    500 nM
110 nM    600 nM
120 nM    700 nM
130 nM    800 nM
140 nM    900 nM
150 nM      1 µM
160 nM
</PRE><H4><SPAN class="mw-headline" id="Colicin_E1">Colicin E1</SPAN></H4><UL><LI>Plan for activity tests of Colicin-Receivers:
</LI><LI>Overnight measurement (Sunday - Monday) in Tecan plate reader with reference promotor cells (BBa_I20260) in combination with pSB1A3-Rec-ColE1 cells.
</LI><LI>Plate scheme (click for detailed picture): </LI></UL><P>[<A href="https://2008.igem.org/Team:Heidelberg/Notebook/Killing_II/9thweek" class="external text" rel="nofollow">back</A>]
</P><P><A href="/Team:Heidelberg/Notebook/Killing_II/10thweek" title="Team:Heidelberg/Notebook/Killing II/10thweek"> go to 10<SUP>th</SUP> week</A></P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2008.igem.org/Team:Heidelberg/Notebook/Killing_II/9thweek">http://2008.igem.org/Team:Heidelberg/Notebook/Killing_II/9thweek</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Heidelberg/Notebook/Killing_II/9thweek" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Heidelberg/Notebook/Killing_II/9thweek" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Heidelberg/Notebook/Killing_II/9thweek&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Heidelberg/Notebook/Killing_II/9thweek&amp;oldid=82441" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2008.igem.org:Privacy_policy" title="2008.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2008.igem.org:General_disclaimer" title="2008.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>