"
Page
Discussion
View source
History
teams
Log in
 
Team:Heidelberg/Notebook/Killing I/Notebook/week6
From 2008.igem.org
Home
Team
Overview
Advisors
Undergraduates
University
DKFZ
BioQuant
BioRegion Rhein-Neckar
Project
Overall Project
Material & Methods
Sensing
Killing I  - Phages
Killing II - Colicin
Visualization
Parts
Submitted Parts
Characterization
Modeling
Overview
Chemotaxis-Colicin
Phage Dynamics model
Notebook
Sensing >
Cloning strategy
Notebook
Killing I - Phages >
Cloning strategy
Notebook
Killing II - Colicin >
Cloning strategy
Notebook
Visualization
Material & Methods
Team Meetings
Seminar on Synthetic Biology
Human Practice
Project Overview
Phips the Phage
Essay
Surveys
Open Day
Nobel Prize
Sponsors
<< Week 5
Overview
Week 7 >>
Week 6
Contents
1Monday, 09/08/08
1.1Phage cloning strategy one
2Tuesday, 09/09/08
2.1Phage cloning strategy one
3Wednesday, 09/10/08
3.1New strategy because of sequencing results
4Thursday, 09/11/08
4.1Phage cloning strategy one
5Friday, 09/12/08
5.1Phage cloning strategy one
Monday, 09/08/08
Phage cloning strategy one
we got colonies on transformation of the 4 fragments in pBluescript
--> only from the ligation at 14°C
inoculation of liquid cultures
miniprep
digestion of the minipreps and pBlue with SpeI over night --> it will otherwise get too late to run the gel
Tuesday, 09/09/08
Phage cloning strategy one
run digested samples and undigested pBlue on gel
transform pZeo for another prep --> we first need Zeocin plates since this is the only resistance
cotransformation of pBlue+insert with pUB307 to generate new donor bacteria for conjugation tests
gel:
lane 0: DNA ladder mix
lane 1-10: digested samples with SpeI (if correct: 4583/2068)
lane 11: pBluescript diegested with SpeI (2961bp)
lane 12: pBluescript undigested
PCR of all 10 Minipreps of pBlue::lambda-Insert
PCR without Phusion master mix:
Phusion: 0.5µl 5x HF buffer: 10µl dNTP: 1µl Primer1 (oriT fw): 2µl Primer2 (GAM rv): 2µl water: up to 50 µl (33.5µl) DNA template: 1µl (~20ng)
PCR protocol 98°C 1min 98°C 30s      | 55°C 30s      | 25x 72°C 2.5min   | 72°C 5min
--> elongation time changed up to 2.5 min (whole fragment has 3.75kb)
Gel
lane0: 1kb plus DNA ladder
lane1-10: PCR of insert
lane11: positive control: amplification of GAM (1,7kb)--> matches
result of gel:
lower band of samples has nearly 3kb
upper band of samples has nearly 7kb
PCR worked since we got the right product from GAM
we send sample 4 for sequencing
primer: M13 FP, M13 RP (both primers from GATC that bind outside the MCS of the pBluescript)
primer: GFP_fw, GFP_rv : two primers of us that should bind in the middle of the insert
contransformation of pBlue::lambda-Insert and pUB307
inoculate pBlue::lambda-Insert culture for transformation test
each 2µl in 100 µl top10
two aliquots from Miniprep 4 and 6
transformation of Zeomycin
1µl in 100µl Top10; plate: 1:2000 zeocin
transformation of 4x-Ligation from Miniprep#6
2µl in 100 µl Top10
inoculate culture as aceptor for transformation test
gel
lane 1: 1kb plus DNA ladder
lane 2 - 7:  lambda DNA digested with XbaI - XhoI
lane 8: 4xLigation: Miniprep 6
the big lambda fragments have been cut out and wait in the freezer for their gel extraction
result of ligation: there is only one fragment but it seems to have about 9kb (same size as small fragment of lambda digestion) -->this is a lot too big, so it could be that concatemers formed
new PCRs: all with Miniprep4 as template
primer: oriT_fw / GFP_rv
primer: GFP_fw / GAM_rv
primer: oriT_fw / GAM_rv
annealing temperature and time stayed the same in all runs
elongation time was 2.5 minutes for the first two samples and 4 minutes for the last sample
Gel
lane0: DNA ladder mix(the upper bright band is 3kb, the one above 4kb, the two lower are 2kb and 2.5kb)
lane1: oirT_fw/GFP_rv (2180bp)
lane2: GFP_fw/GAM_rv (3350bp)
lane3: oriT_fw/GAM_rv (3850bp)
Gel
lane0: DNA ladder mix
lane1: pBlue + insert digested with AgeI
lane2: pBlue + insert digested with XbaI
lane3: pBlue + insert digested with XbaI/XhoI
lane4: gel extraction of lambda digestion
results of gel:
digestion with AgeI leads to a linearised vector with the right size (6.75 kb)
digestion with XbaI/XhoI leads to the correct pattern (2.9; 2.6; 0.73; 0.5 kb and 60 bp (not visible))
digestion with XbaI alone leads to a wrong digestion pattern
Wednesday, 09/10/08
I changed the vector according to our sequencing results: we use pBluescript SK instead of pBluescript KS!
we got the sequencing results from the GFP primer from gatc and now have fully sequenced the fragment
New strategy because of sequencing results
The sequence we got from our lambda insert we ligated shows a repeating sequence in the GFP/Cm-Gene. This sequence contains a XbaI and a BamHI restriction site and is similar to the sequence of the primer we used to amplify this part via PCR. Therefore we got two additional XbaI restriction sites at bp 1895 and 1954. Together with the XbaI site between OriT and GFP they make it impossible to digest our fragment with XbaI and XhoI and ligate it with the lambda vector. Furthermore, the CmR-protein, which is produced is longer than the natural protein, but it seems to be functional, because we selected the bacteria on Cm. So we set up two strategies to cope with this problem:
We will digest the pBluescript with the lambda insert with NotI and XhoI, extract the whole insert from the gel and then digest with KpnI to get two fragments: One containing the OriT and the other the GFP/CmR/GAM. The GFP/CmR/GAM fragment has already the proper sticky ends for ligation. the OriT fragment will further be digested with XbaI. Then the two fragments will be ligated with the lambda vector.
we will get rid of the three additional XbaI restriction sites via PCR mutagenesis:
first we will mutate the XbaI sequence between OriT and GFP (primer sequence: ctgaggggacggtaccTCTACAtttacagctagctcag)
then we will cut the plasmid with BamHI to delete another XbaI restriction site. By doing this, we shift the reading frame of the CmR protein. By another mutagenesis we mutate the remaining XbaI site to a stop codon since through the digestion with BamHI we lost the stop codon.(primer sequence: ggcaggcggggcgtaaTCTATAggatccggctaataaagg)
we finally will have a CmR that is 2 aminoacids longer than the natural one (aminoacid sequence of the last aa of the natural one: Gln Gly Gly Ala STOP; our mutated one: Gln Ala Gly Arg Asn Leu STOP). so the last 3 aa are changed into 5 different ones. All 216 aa before are the same. What we have now (a CmR that is functional) has the same mutated aa sequence as we will get through this mutagenesis and only 17 further aa at the end. So if we cut off these 17 additional aa we hope to get a still functional CmR.
CmR (pDEST15):
Met E K K I T G Y T T V D I S Q W H R K E H F E A F Q S V A Q C T Y N Q T V Q L D I T A F L K T V K K N K H K F Y P A F I H I L A R L Met N A H P E F R Met A Met K D G E L V I W D S V H P C Y T V F H E Q T E T F S S L W S E Y H D D F R Q F L H I Y S Q D V A C Y G E N L A Y F P K G F I E N Met F F V S A N P W V S F T S F D L N V A N Met D N F F A P V F T Met G K Y Y T Q G D K V L Met P L A I Q V H H A V C D G F H V G R Met L N E L Q Q Y C D E W Q G G A Stop
CmR (what we now have):
Met E K K I T G Y T T V D I S Q W H R K E H F E A F Q S V A Q C T Y N Q T V Q L D I T A F L K T V K K N K H K F Y P A F I H I L A R L Met N A H P E F R Met A Met K D G E L V I W D S V H P C Y T V F H E Q T E T F S S L W S E Y H D D F R Q F L H I Y S Q D V A C Y G E N L A Y F P K G F I E N Met F F V S A N P W V S F T S F D L N V A N Met D N F F A P V F T Met G K Y Y T Q G D K V L Met P L A I Q V H H A V C D G F H V G R Met L N E L Q Q Y C D E W Q A G R N L E D P A N K G E P T A Met S G R R G V I Stop
CmR (what we will get with our strategy):
Met E K K I T G Y T T V D I S Q W H R K E H F E A F Q S V A Q C T Y N Q T V Q L D I T A F L K T V K K N K H K F Y P A F I H I L A R L Met N A H P E F R Met A Met K D G E L V I W D S V H P C Y T V F H E Q T E T F S S L W S E Y H D D F R Q F L H I Y S Q D V A C Y G E N L A Y F P K G F I E N Met F F V S A N P W V S F T S F D L N V A N Met D N F F A P V F T Met G K Y Y T Q G D K V L Met P L A I Q V H H A V C D G F H V G R Met L N E L Q Q Y C D E W Q A G R N L Stop
Thursday, 09/11/08
Phage cloning strategy one
lane 0: DNA ladder mix
lane 1: oriT/GAM (HF buffer)(3.85 kb)
lane 2: oriT/GAM (GC buffer)
lane 3: GFP/GAM (HF buffer) (3.3 kb)
lane 4: GFP/GAM (GC buffer)
results: so for whatever reason our amplified fragments seem to be too little.
explanation: after digestion of GAM with XhoI and AgeI and ligation of this part with the other parts into the vector we have lost the primer binding sites for the GAM primer. So actually in the PCRs we have made where the sizes seemed to be too small we had only one binding primer!!!!
nanodrop results for gelextraction of big lambda fragment
extraction with kit for up to 10 kb: 15,4 ng/µl, 260/280: 1,97
extraction with kit for over 10 kb: 100 ng/µl, 260/280: 1,57
cotransformation of PUB307 and pBlue with the lambda insert again did not work. we tried different ratios of pUB and pBlue and also TOP10, MG1655 and DH5a.
Friday, 09/12/08
Phage cloning strategy one
Maxiprep of lambda insert (Mini 4), nanodrop result: 2593 ng/µl, 260/280: 1,93
digestion of pBlue with lambda insert:
triple digestion with NotI, XhoI and KpnI in Neb2 with BSA
double digestion with NotI and XhoI in Neb3 with BSA
gel for cutting out our fragments
lane0: DNA ladder mix
lane1: NotI/XhoI
lane2: NotI/XhoI
lane3: NotI/XhoI/KpnI
lane4: NotI/XhoI/KpnI
lane5: undigested
digestion of the oriT (500bp fragment from lane3 and 4) with XbaI
digestion of the insert (3.85kb fragment from lane1 and 2) with KpnI
ligation of lambda + oriT + insert (3.3kb from lane 3 and 4)
transformation --> incubation at 28°C (to make sure that lambda does not get lytic)
<< Week 5
Overview
Week 7 >>
Retrieved from " http://2008.igem.org/Team:Heidelberg/Notebook/Killing_I/Notebook/week6 "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
