"
Page
Discussion
View source
History
teams
Log in
 
Team:Heidelberg/Notebook/material
From 2008.igem.org
Home
Team
Overview
Advisors
Undergraduates
University
DKFZ
BioQuant
BioRegion Rhein-Neckar
Project
Overall Project
Material & Methods
Sensing
Killing I  - Phages
Killing II - Colicin
Visualization
Parts
Submitted Parts
Characterization
Modeling
Overview
Chemotaxis-Colicin
Phage Dynamics model
Notebook
Sensing >
Cloning strategy
Notebook
Killing I - Phages >
Cloning strategy
Notebook
Killing II - Colicin >
Cloning strategy
Notebook
Visualization
Material & Methods
Team Meetings
Seminar on Synthetic Biology
Human Practice
Project Overview
Phips the Phage
Essay
Surveys
Open Day
Nobel Prize
Sponsors
Contents
1Material
1.1Buffers & Solution
1.2Kits
1.3Marker
1.4Enzymes
1.5Plasmidvectors
1.6Synthetic oligonucleotides
1.7Phages
1.8Bacteria
1.9Bacteria Growth Media
2Methods
2.1Preparing chemically competent cells
2.2Transformation of bacteria
2.3Isolation of plasmid DNA by alkaline lysis (mini and maxiprep)
2.4DNA amplification using polymerase chain reaction (PCR)
2.5Purification of DNA from PCR reactions
2.6Enzymatic hydrolysis of DNA by restriction enzymes
2.7Agarose gel electrophoresis for separating DNA
2.8Isolation of DNA fragments from an agarose gel
2.9Ligation of dsDNA fragments
2.10Lysing of bacteria by ultrasonication
2.11Glycerol stock
2.12Preparation of plating bacteria for bacteriophage experiments
2.13Bacteriophage λ plaque assay
2.14Preparing Stocks of Bacteriophage λ
2.15Preparing of λ DNA
2.16In vitro packaging of λ DNA
2.17Colicin Activity Test
2.18Lysis test
2.19Sender Activity Test
2.20LuxS Activity Test
2.21Conjugation test
2.22Receptor Polarization Test
2.23Swarm Assays
2.24Microscopy – Chemotaxis
2.25Microscopy – Colicin Toxicity
2.26Microscopy – Cell lysis
2.27Cell Culture
2.28Eucaryotic Killing Assay with Colicins
2.29Flow Cytometer Acquisition
Material
Buffers & Solution
SM
50 mM
Tris, pH 7.5
20 mM
MgSO4
50 mM
NaCl
0.01 %
gelatine
Tethering buffer, pH 7.0
10 mM
potassium phosphate
(K2HPO4 : KH2PO4 = 1 :1 )
100 µM
EDTA
1 µM
L-Methionine
10 mM
lactic acid
M63 salt
3 g/l
KH2PO4
9 g/l
K2HPO4
4 g/l
(NH4)2SO4
0.5 g/l
FeSO4
Amino acid mix
5 g/l
L-threonine
5 g/l
L-histidin
5 g/l
L-leucine
5 g/l
L-methionine
M9 salts
64 g/l
Na2HPO4 x 7H2O
15 g/l
KH2PO4
2.5 g/l
NaCl
5 g/l
NH4Cl
[ back ]
Kits
Kit
supplier
CompactPrep Plasmid Maxi Kit
Qiagen
HISpeed Plasmid Maxi Kit
Qiagen
MaxPlax™ Lambda Packaging Extracts
EPICENTRE Biotechnologies
QIAEX II Gel Extraction Kit
Qiagen
QIAGEN Lambda Mini Kit
Qiagen
QIAprep Spin Mini Kit
Qiagen
QIAquick Gel Extraction Kit
Qiagen
QIAquick PCR Purification Kit
Qiagen
[ back ]
Marker
Marker
supplier
GeneRuler™ High Range DNA Ladder
MBI Fermentas
GeneRuler™ 1kb DNA Ladder Mix
MBI Fermentas
GeneRuler™ 1kb Plus DNA Ladder Mix
MBI Fermentas
[ back ]
Enzymes
Enzym
supplier
Pfu DNA polymerase
Stratagene
Pfu turbo DNA polymerase
Stratagene
Phusion DNA polymerase
Finnzymes
Taq DNA polymerase
MBI Fermentas
T4 DNA ligase
MBI Fermentas / New England Biolabs
AgeI
New England Biolabs
BamHI
New England Biolabs
BglI
MBI Fermentas
BseJI
MBI Fermentas
BspEI
New England Biolabs
DpnI
Roche Diagnostics / New England Biolabs
DraI
New England Biolabs
EcoRI
New England Biolabs
EcoRV
New England Biolabs
HindIII
New England Biolabs
KpnI
New England Biolabs
NcoI
New England Biolabs
NdeI
New England Biolabs
NotI
New England Biolabs
PstI
New England Biolabs
SacI
New England Biolabs
SalI
MBI Fermentas
ScaI
New England Biolabs
SfcI
New England Biolabs
SmiI
MBI Fermentas
SpeI
New England Biolabs
SpeI
New England Biolabs
XbaI
New England Biolabs
XhoI
MBI Fermentas
XmaI
New England Biolabs
[ back ]
Plasmidvectors
Name
application
reference
BBa_B0015
terminator
http://partsregistry.org
BBa_F1610
LuxI
http://partsregistry.org
BBa_I15030
AI-1 amplifier
http://partsregistry.org
BBa_I20260
GFP
http://partsregistry.org
BBa_J01003
oriT
http://partsregistry.org
BBa_J16002
cloning vector
http://partsregistry.org
BBa_J23107
constitutive promotor
http://partsregistry.org
BBa_T9002
AI-1 GFP receiver
http://partsregistry.org
CheY-mCherry
cloning vector
V. Sourjik, ZMBH
ColE1
colicin E1
DSMZ
ColE9-J
colicin E9
C. Kleanthous, University of York
pBad18
cloning vector
V. Sourjik, ZMBH
pBad33
cloning vector
V. Sourjik, ZMBH
pBluescript II SK (+)
cloning vector
V. Sourjik, ZMBH
pDest15
cloning vector
DKFZ Library
pDK4
visualization
V. Sourjik, ZMBH
pDK48
cloning vector
V. Sourjik, ZMBH
pDK58
cloning vector
V. Sourjik, ZMBH
pDK6
visualization
V. Sourjik, ZMBH
pED374
oriT
K. Derbyshire, Wadsworth
pES16
cloning vector
V. Sourjik, ZMBH
pMMB863
oriT
M. Bagdasarian, MSU
pQE-30
cloning vector
Invitrogen
pSB1A2
cloning vector
http://partsregistry.org
pSB2K3
cloning vector
http://partsregistry.org
pTrc99a
cloning vector
V. Sourjik, ZMBH
pUB307
helper plasmid
E. Lanka, BfR
RP4
helper plasmid
M. Bagdasarian, MSU
[ back ]
Synthetic oligonucleotides
All oligonucleotides were purchased from Invitrogen (Karlsruhe) and adjusted to a standard concentration of 100 pmol/µl.
Name
SEQUENCE (5´->3´)
Bam_fw
GACAAGTGTTGGCCATGGAACAGG
Bam_rv
GCCGTCTGTGATGGCTTCCATG
cI_mut_fw
GCGTCTGGGTGGTGATGAGTTCACCTTCAAAAAACTG
cI_mut_rv
CAGTTTTTTGAAGGTGAACTCATCACCACCCAGACGC
CmR_EcoRI_mut_fw
GAATGCTCATCCGGAGTTCCGTATGGCAATG
CmR_EcoRI_mut_rev
CATTGCCATACGGAACTCCGGATGAGCATTC
CmR_fw
GCTAAAATGGAGAAAAAAATCACTGG
CmR_new_fw
TACGAGGTACCTTTACAGCTAGCTCAGTCCTAGGTATTATGC
CmR_new_rev
TATATAAGCTTTTACGCCCCGCCCTGCCACTCATCGCAGTACTGTTG
CmR_Prefix_fw
GAATTCGCGGCCGCTTCTAGAGTTTACAGCTAGCTCAGTCCTAGG
CmR_rv
AGGTTCTCCTTTATTAGCCGGATCCTCTAGATTACGCC
CmR_Suffix_rv
CTGCAGCGGCCGCTACTAGTATATAAACGCAGAAAGGCCCACCC
colE1_kil_prot_rv_A_SpeI
TATATACTAGTACTACTGAACCGCGATCCCCG
colE1_mut_EcoRI_fw
GGTATTGCTATTGTTACAGGTATTCTATGCTCCTATATTGATAAG
colE1_mut_EcoRI_rv
CTTATCAATATAGGAGCATAGAATACCTGTAACAATAGCAATACC
colE1_mut_PstI_1_fw
GCAGTAAAAGTGAAAGTTCAGCAGCTATTCATGCAACTGC
colE1_mut_PstI_1_rv
GCAGTTGCATGAATAGCTGCTGAACTTTCACTTTTACTGC
colE1_mut_PstI_2_fw
GCTGCCCGGGCAAAAGCAGCAGCGGAAGCACAGG
colE1_mut_PstI_2_rv
CCTGTGCTTCCGCTGCTGCTTTTGCCCGGGCAGC
colE1_mut_PstI_3_fw
CATTAGAGAAGAAAGCAGCAGATGCAGGGGTGAG
colE1_mut_PstI_3_rv
CTCACCCCTGCATCTGCTGCTTTCTTCTCTAATG
colE1_prot_fw_BamH1
TACGAGGATCCATGGAAACCGCGGTAGCG
colE1_prot_rv_HindIII
TATATAAGCTTTTAAATCCCTAACACCTC
colE9_lysProt_rv_A_SpeI
TATATACTAGTACTAGGTTTTCGGCTTAGAACCCC
colE9_mut_EcoRI_fw
GTTGGGTGGACGATTCGAGAGTTCAATGGGGAAATAAAAATG
colE9_mut_EcoRI_rv
CATTTTTATTTCCCCATTGAACTCTCGAATCGTCCACCCAAC
colE9_plasmid_rv_A_SpeI
TATATACTAGTACACATGGAACTTTTGTGAC
colE9_prot_fw_BamH1
TACGAGGATCCATCGATTTGCCCATGACCC
colE9_prot_rv_XmaI
TATATCCCGGGTTACTTACCTCGGTGAATATCG
DK13
TCACCCGCACGCGC
DK167
AGGATACTAGTAGGCCATTACTTT
DK9b
CGATGCGGCCGCTCAAAATGTTTCCCAGTTTGG
F1610_fw_XbaI
TACGATCTAGAAAAGAGGAGAAATACTAG
F1610_rv_HindIII
TATATAAGCTTTATAAACGCAGAAAGGCCC
GAM_fw
AGTGCTTTAGCGTTAACTTCCG
GAM_rv
GGTTTTACCGCATACCAATAACG
GFP_CmR_fw
CTCGTTGGTACCTCTAGATTTACAGCTAGCTCAGTCCTAGG
GFP_CmR_rv
TATTCGACCGGTACTAGTTATAAACGCAGAAAGGCCCACC
GFP_new_fw
TACGAGAGCTCTTTACAGCTAGCTCAGTCCTAGG
GFP_new_rv
TATATACCGGTACTAGTTATAAACGCAGAAAGGCCCACC
Lambda_insert_fw
TTGTAAAAACAGCCCTCCTC
Lambda_insert_rv
GATATGACTATCAAGGCCGC
LuxP_mut_F
CGTGAATTAGCAACAGAGTTCGGAAAGTTCTTCCC
LuxP_mut_R
GGGAAGAACTTTCCGAACTCTGTTGCTAATTCACG
LuxP_prefix_F
GAGGGAGAATTCGCGGCCGCTTCTAGATGAAGAAAGCGTTACTATTTTC
LuxP_sufffix_R
GGAGAGCTGCAGCGGCCGCTACTAGTAATTATCTGAATATCTAAATGCG
LuxPc
ATTACGCGGCCGCAGGAAACAGACCATGAAGAAAGCGTTACTATTTTCCC
LuxPd
GTAATGTCGACTCAATTATCTGAATATC
LuxP-seq-FW
CCCGTCCTGCCAGTGAGC
LuxQa
ATCGACCATGGGCAATAAATTTCGCTTAGC
LuxQc
GTAATGGATCCTTAGTGGAGGCTTGAGCC
LuxQTar_1a
CACAAATCATTGCCAATGAACGTATGTTGCTTACTCCGCTGG
LuxQTar_1b
CCAGCGGAGTAAGCAACATACGTTCATTGGCAATGATTTGTG
LuxQTar_2a
CTTAGCGACCATGAGCCATGAGTTTGCCCAGTGGCAACTGGC
LuxQTar_2b
GCCAGTTGCCACTGGGCAAACTCATGGCTCATGGTCGCTAAG
LuxS mutBbaIR
CCACACCCACTTCTAGGATGTTCTTCGCGATTTGC
LuxS mutXbaIF
GCAAATCGCGAAGAACATCCTAGAAGTGGGTGTGG
LuxS_mut_fw
CATCCTTTCTGAGAAAGGCATTCATACATTAGAGC
LuxS_mut_rev
GCTCTAATGTATGAATGCCTTTCTCAGAAAGGATG
LuxS_prefix_F
GGAGAGGAATTCGCGGCCGCTTCTAGATGGGCAATGCACCAGCGGTTCG
luxS_suffix_R
GAGGGACTGCAGCGGCCGCTACTAGTAGTCGATGCGTAGCTCTCTCAGC
LuxSa
ATCAGTCCATGGGCAATGCACCAGCGGTTCG
LuxSb
GTAATGGATCCTTAGTCGATGCGTAGC
mut_insert_fw
CTGAGGGGACGGTACCTCTACATTTACAGCTAGCTCAG
mut_insert_rv
CTGAGCTAGCTGTAAATGTAGAGGTACCGTCCCCTCAG
mut_insert2_fw
GGCAGGCGGGGCGTAATCTATAGGATCCGGCTAATAAAGG
mut_insert2_rv
CCTTTATTAGCCGGATCCTATAGATTACGCCCCGCCTGCC
mut_kpn1_pBlue
CGAGGGGGGGCCCGGTTCCCAATTCGCCCTATAG
mut_kpn1_pBlue
CTATAGGGCGAATTGGGAACCGGGCCCCCCCTCG
oriT_pre
GAATTCGCGGCCGCTTCTAGAGGACAGGCTCATGCCGGCCGC
oriT_RP4_fw
CTCGTTTCTAGAACTAGTGACAGGCTCATGCCGGCCGC
oriT_RP4_rv
TATTCGGGTACCGTCCCCTCAGTTCAGTAATTTCCTGC
oriT_suf
CTGCAGCGGCCGCTACTAGTAGTCCCCTCAGTTCAGTAATTTCCTGC
T9002_Lux_pR_rv_SpeI_BamHI_RBS
TATATACTAGTGGATCCGGTTCTGTTTCCTCTCTAGTATTTATTCGAC
T9002_LuxpR_Not_Eco_Xba_G_fw
TACGAGAATTCGCGGCCGCTTCTAGAGTCCCTATCAGTGATAGAGATTG
Term_new_fw
TACGAAAGCTTCCAGGCATCAAATAAAACGAAAGG
Term_new_rv
TATATGAGCTCTATAAACGCAGAAAGGCCCACCC
VF2
TGCCACCTGACGTCTAAGAA
VIC121
TTTATCGCAACTCTCTACTG
VIC122
CTGATTTAATCTGTATCAGG
VIC131
ATGTGTGGAATTGTGAGCGG
VIC132
CTGATTTAATCTGTATCAGG
VR
ATTACCGCCTTTGAGTGAGC
[ back ]
Phages
Name
application
reference
Lambda cI mut
cI deleted
W. Reiser, ZMBH
Lambda cI857
heat inducible
MBI Fermentas
[ back ]
Bacteria
E.coli strain
usage
reference
DH5a
amplification of plasmids
Invitrogen
HCB33
swarm assays
V. Sourjik, ZMBH
MG1655
swarm assays
V. Sourjik, ZMBH
TOP10
amplification of plasmids
Invitrogen
UU1250
chemotaxis receptor knock out
V. Sourjik, ZMBH
[ back ]
Bacteria Growth Media
Luria Broth (LB)
10 g/l
tryptone
5 g/l
yeast extract
10 g/l
NaCl
Luria Broth (LB) plus
10 g/l
tryptone
5 g/l
yeast extract
20 g/l
NaCl
TB
10 g/l
bacto tryptone
5 g/l
NaCl
Standard I
15.6 g/l
peptone P
2.8 g/l
yeast extract
5.6 g/l
NaCl
1 g/l
D-glucose
Minimal medium
9.8 %
M63 salts
0.2 %
glycerol
0.1 g/l
Thiamine
1 mM
MgSO4
0.8 %
amino acid mix
M9
20 %
M9 salts
2 mM
MgSO4
0.4 %
glucose
0.1 mM
CaCl2
For cultivation of phages the media were supplied with 0.2 % maltose and 10 mM MgSO4. For preparation of agar plates 15 g/l agar, for preparation of top agar 7 g/l agar was added prior the autoclaving. For selection of resistant bacteria following antibiotics were added during cooling of at agar at about 50 °C:
[ back ]
Antibiotic
concentration
Ampicillin
100 µg/ml
Chloramphenicol
35 µg/ml
Kanamycin
50 µg/ml
The same concentrations of antibiotics were used for selection of resistant in media.
[ back ]
Methods
Preparing chemically competent cells
First, a 20 ml over night culture was inoculated in antibiotic free LB medium from a fresh single colony and transferred into 400 ml antibiotic free LB medium the next day. This culture was incubated at 37 °C while shacking until an OD600 of 0.5 – 0.6 was achieved. The culture was than cooled down on ice, centrifuged (8 min, 4 °C, 3500 rpm), the supernatant discarded and the pellet resuspended in 10 ml 100 mM CaCl2. After addition of further 190 ml 100 mM CaCl2 the suspension was incubated on ice for 30 min. The suspension was than again centrifuged (8 min, 4 °C, 3500 rpm), the supernatant discarded, the pellet resuspended in 20 ml 82.5 mM CaCl2 with 17.5 % glycerol and aliquoted. The aliquots were flash frozen in liquid nitrogen and than stored at -80 °C until usage. [ back ]
Transformation of bacteria
For enrichment of vectors, E .coli TOP10 and DH5α were used. For the transformation 100 µl of the competent cells were thawed on ice and 50 – 200 ng DNA solution added (depending on the concentration of the DNA solution). After a 20 minute incubation on ice, cells were made permeable for the DNA by heat shocking for 45 seconds at 42 °C and a further 2 minute incubation on ice. The samples were than rescued by adding 250µl preheated antibiotic free LB-medium and incubated for one hour at 37 °C while shacking for induction of the antibiotic resistance. The selection for plasmid containing and therefore antibiotic resistant bacteria was conducted by plating them on antibiotic containing LB-agar plates. [ back ]
Isolation of plasmid DNA by alkaline lysis (mini and maxiprep)
For analysis of ligations and transformations QIAprep Spin Kits (Qiagen, Hilden) were used following the manufacturer instructions. For miniprep a single colony was picked from a LB-agar plate or glycerol stock was used to inoculate 5 ml LB-medium with appropriate antibiotic for selection (100 µg/µl ampicillin, 50 µg/µl kanamycin, 35 µg/µl chloramphenicol). Bacteria were grown over night at 37 °C while shaking (200 rpm). By using 4 ml over night culture with this kit the yield was around 6-10 µg.  For maxipreps the Qiagen CompactPrep Plasmid Maxi Kit was used following the instructions given by the instruction manual. In this case 250 ml LB-medium were inoculated and used for preparation of plasmid DNA. The routinely yield was 300-400 µg plasmid DNA. Purity and amount of DNA was analysed using a NanoDrop. [ back ]
DNA amplification using polymerase chain reaction (PCR)
By using PCR smallest amount of DNA can be detected and amplified. The principle of PCR is the selective amplification of any region of the DNA. Nevertheless, the sequence at both ends must be known for the binding of two complementary primers. The DNA region of interest is than amplified exponentially while the reproduction of the DNA takes place in three temperature stpes: denaturation of parental DNA at high temperature (95-98 °C), hybridisation of primers (58-65 °C) and DNA elongation (72 °C).  For the amplification a heat-stable DNA polymerase I with a 3’-5’ exonuclease activity (proof reading) is used such as Pfu (from Pyrococcus furiosus) and Phusion (a Pyrocuccus-like enzyme) or the non proof reading enzyme Taq (from Thermus aquaticus). Phsuion polymerase was used in a 2x master mix adding only primers (20pmol each) and DNA template (~10ng) or alternatively colonies from agar plates (1-5 colonies) in a final volume of 50 µl. For taq and Pfu polymerase following reaction batch was commonly used:
1 µl   template (10 ng) / colonies
2 µl   primer 1 (10 pmol/µl)
2 µl   primer 2 (10 pmol/µl)
1 µl   polymerase (2,5 Units)
1 µl   dNTP mix (10 mM)
5 µl   10x buffer
38 µl   dH2O
The PCR procedure was as follows:
Initiale denaturation
95- 98 °C, 3-5 min
1 cycle
denaturation
95-98 °C, 15 sec - 1 min
annealing
58-65 °C, 15 sec - 1 min
25-28 cycles
elongation
72 °C, 15 sec - 3 min
termination
72 °C, 5 – 10 min
1 cycle
4 °C
forever
For site directed mutagenesis PCR Pfu turbo polymerase (Stratagene) was used in the same reaction batch described above. The temperature program was as follows:
initiale denaturation
95 °C, 30 sec
1 cycle
denaturation
95 °C, 30 sec
annealing
55 °C, 1 min
16 cycles
elongation
68 °C, 2 min per kb of plasmid
termination
68 °C, 10 min
1 cycle
4 °C
forever
For point mutagenesis primers with around 33 bases were used having the base to be altered in the middle. The melting temperature of these primers was always over 78 °C. If this melting temperature could not be achieved using 15 unchanged nucleotides at both sites the flanking arms were enlarged properly. After the PCR reaction 10 units of DpnI was added directly to the PCR tube, mixed and incubated for 1-5 h at 37 °C. DpnI digests the parental methylated plasmid DNA leaving only the mutated one. After purification using QIAquick PCR Purification Kit the plasmid was transformed as described above.  [ back ]
Purification of DNA from PCR reactions
PCR products were purified by the QIAquick PCR Purification Kit from Qiagen following the instructions of the Qiagen Handbook. To check the purity and amount of extracted DNA an aliquot was analysed using a NanoDrop. [ back ]
Enzymatic hydrolysis of DNA by restriction enzymes
The restriction digest of DNA was used for analysis of purified DNA form mini or maxiprep or for isolation of specific DNA fragments for further cloning. Analytical digestions were routinely conducted in 20 µl volume. In all digestions a minimum of 2 Units restriction enzyme(s) was used per microgram DNA. Optimised buffer conditions were secured by using NEB buffer system. The final reaction volume was achieved by adding H2O dest. and the sample was incubated at optimal temperature for the restriction enzyme(s) (normally 37 °C). Preparative digestions were conducted in a volume of 50 µl. For analysis and preparation DNA loading dye was added to the samples and they were loaded on a agarose gel or alternatively purified by QIAquick PCR Purification Kit from Qiagen. [ back ]
Agarose gel electrophoresis for separating DNA
In the agarose gel electrophoresis a mixture of DNA fragments with different sizes are separated in an electrical field by their size. This is achieved by moving the negatively charged DNA through an agarose matrix while shorter fragments will run faster. The size of the pores can be controlled by agarose concentration. The higher the agarose concentration the smaller the pores are and the smaller fragments can be separated. Agarose concentrations between 0.7 and 1.5 % agarose in 0.5x TE buffer were used.  The agarose was dissolved completely by heating up and 0.1 µg/ml ethidium bromide was added.  The DNA fragments were separated using a constant voltage between 80 and 130 V. Under UV light (λ = 254 nm) DNA is visible through the unspecific intercalated ethidium bromide and can be documented or cut out and extracted from the gel. [ back ]
Isolation of DNA fragments from an agarose gel
Plasmid DNA and DNA fragments were extracted using the Gel Extraction Kit from Qiagen following the manufacture instructions. To check the purity and amount of extracted DNA an aliquot was analysed using a NanoDrop. [ back ]
Ligation of dsDNA fragments
T4 DNA ligase was used to form covalent phosphodiesterbonds between doublestranded DNA fragments having blunt or compatible sticky ends. For the ligation the restricted vector -DNA and the insert were mixed 1:3 or 1:5 in a total ligationvolume of 20 µl. This mixture was incubated in ligation buffer using 5 Weiss units T4 DNA ligase over night at 16 °C or at room temperature for 20 minutes and afterwards used to transform chemical competent cells.  [ back ]
Lysing of bacteria by ultrasonication
For the lysis of bacteria an ultrasonic tip was used. The 10-15 ml of bacteria over night cultures were sonicated with an ultrasonic tip three time for 15 seconds. After sonication cell extract was observed under microscope. [ back ]
Glycerol stock
To store bacteria for long term glycerol stocks were used. Therefore 1 ml of an over night culture were added to 150 µl of 80 % Glycerol into a cryo tube, vortexed and incubated at room temperature for 30 min. Afterwards the glycerole stock was stored at -80 °C. [ back ]
Preparation of plating bacteria for bacteriophage experiments
A overnight culture of the appropriate E. coli strain was grown in LB medium containing 10 mM MgSO4 and 0.2 % maltose at 30 °C to reduce the amount of cell debris in the medium. The added maltose leads to a substantial induction of the maltose operon including the lamb gene, which encodes the cell surface receptor to which bacteriophage λ binds. After harvesting the cells they were resuspended in 10 mM MgSO4 and diluted to a final concentration of 2.0 OD600. The suspension of plating bacteria was stored at 4 °C for up to 1 week. [ back ]
Bacteriophage λ plaque assay
Tenfold serial dilutions of the bacteriophage stocks were prepared, 100 µl of it mixed with the same amount of plating bacteria, incubated for 20 minutes at 37 °C to allow the bacteriophage particles to adsorb to the bacteria, this mixture added to 3 ml molten top agar which was kept liquid at 48 °C and the entire contend poured onto a agar plate. After harden of the top agar the inverted plates were incubated at 37 °C over night. On the next day plaques could be counted. [ back ]
Preparing Stocks of Bacteriophage λ
Completely lysed agarose plates were prepared by mixing 0.1 ml plating bacteria with ~105 pfu, incubation for 20 minutes at 37 °C and that bringing out the mixture in 3 ml molten top agarose. Agarose is preferred in this protocol in order to minimize contaminations with polyanionic inhibitors that can interfere with subsequent enzymatic analysis of the bacteriophage DNA. The plates were incubated 12-16 hours without inversion at 37 °C. Plates were not inverted during incubation to encourage sweating of fluid onto the surface of the dish, which allows the bacteriophage to spread more easily. The plates were than overlayed with 5 ml SM and incubated while shaking at 4 °C over night. The SM was that transferred to a tube and the plate washed again with 1 ml SM. 2 % v/v chloroform was added and bacterial debris removed by centrifugation at 10,000 g for 10 minutes at 4 °C. The concentration of infectious particles in the stock was measured by plaque assays as described above. [ back ]
Preparing of λ DNA
Lambda DNA was prepared using the QIAGEN® Lambda mini kit following the manufacture instructions. [ back ]
In vitro packaging of λ DNA
In vitro packaging was conducted using the MaxPlax lambda packaging extracts from Epicentre Biotechnologies following the manufacture instructions.  [ back ]
Colicin Activity Test
To measure the killing efficiency and which amount of cells or colicins are needed to reach any killing activity a colicin activity test was carried out. Therefore bacteria containing the colicin plasmid (E. coli TOP10 or E. coli MG1655) and GFP producing cells (reference promoter BBa_I20260 , E. coli TOP10) were inoculated in TB-media with appropriated antibiotics at 37 °C for 4 to 6 hours and the optical densities of the two strains were adapted. The colicin cells themselves, their produced supernatant or the supernatant of the lysed colicin cells were added in different ratios to a constant amount of GFP producing cells. The total volume was kept constant and the missing amount added with TB-media without antibiotics. The colicin production was induced by several concentrations (0 M-100 nM) of N-Acyl-Homoserin-Lactone (AHL). The OD and GFP intensities were measured at 37 °C in the Tecan Microplate Reader every half an hour for about 12 hours. As a negative control the similar test was carried out with cells, containing the same plasmid without the colicin operon on it.  [ back ]
Lysis test
Lysis of killercells dependent on the AHL concentration was measured due to a lysis test. Therefore E. coli TOP 10 containing the luxR-colicinE1 part ( BBa_K150009 ) and a GFP producing part ( BBa_I20260 ) were inoculated in TB-media with appropriate antibiotics at 37 °C for 4 to 6 hours. As reference E. coli TOP 10 cells containing a luxR part (comparable to BBa_F2620 ) instead of the luxR-colicinE1 part ( BBa_K150009 ). For the lysis measurement 250 µl of the liquid cultures were diluted with 250 µl TB-media. The Lux pR promoter was activated with AHL concentrations from 0 M up to 50 nM. The optical densities and GFP intensities were measured in Tecan Microplate Reader every half-hourly for about 12 hours. [ back ]
Sender Activity Test
The sender activity test was performed to quantify the AHL production of the cells containing the sender part ( BBa_K150000 ). Therefore TB-media with the appropriated antibiotic was inoculated with 2 µl/ml of an overnight culture. For 7 hours every hour the OD of the medium was measured and 5 ml of the culture were spin down to produce supernatant. The sterile filtered supernatant, still containing the AHL, was stored at 4 °C. The supernatant of every hour was then added in different ratios to a constant amount of AHL inducible cells which produce GFP after induction. The OD and GFP intensities were measured at 37 °C in the Tecan Microplate Reader every half an hour for about 12 - 24 hours. The same test was carried out with amplifier cells ( BBa_I15030 ). To calculate the produced amount of AHL by sender or amplifier cells as reference the test were performed also with different AHL concentration (0 M-100 nM) instead of AHL producing cells. [ back ]
LuxS Activity Test
This protocol for the functional check AI-2 production is mainly adapted from a paper of Vanderleyden et al. [1] , for further questions please refer to it. 10ml ONC DH5α strain with LuxS (S) and 10ml DH5α strain with ptrc99α as blank (B) were incubated in LB to a volume of 50mL each at 37°C up to an OD of 0.35 in 100ml erlenmeyer flasks. Then S and B each were induced with 500 µM IPTG for 3h, following centrifugation at maximum speed for 20min. The supernatants were then transferred in two new falcons by pipetting or decanting. Optional the supernatant was sterile filtrated and/or 10 g/L NaCl was added to a final concentration of ~20g/L. 10 mL ONC of V. harveyi reporter (BB120 in LB plus at 30 °C) was incubated in LB plus to a final volume of 50 mL at 37°C up to an OD of 0.35 in a 100 mL erlenmeyer flask. Supernatant S and B were both mixed with Vibrio cells at ratios 1:3, 1:5 and 1:10 each to a final volume of 5 mL in 50 mL falcon tubes and incubated for 30min at 30°C. Subsequently the cells were incubated again for 3h, OD and Luminescence were measured with a microplate reader. [ back ]
[1] Sigrid C. J. DeKeersmaecker and Jos Vanderleyden ‘Constraints on detection of autoinducer-2 (AI-2) signalling molecules using Vibrio harveyi as a reporter’, Microbiology 2003 149: 1953-1956.
Conjugation test
Equal amounts of donor (cells harbouring the oriT containing plasmid and pUB307) and acceptor cells from overnight cultures were centrifuged (2 min, 13000 rpm), the pellet washed twice with 1.5 ml LB medium without antibiotics, the pellets combined and this mixture resuspended in 100 µl LB medium without antibiotics. This cell mix was transferred to a filter membrane on a LB agar plate and incubated here for a time range from 0 to 42 minutes in 6 minutes steps at 37 °C. The membrane was then transferred in 1 ml LB medium, vortexed, the medium diluted and plated out on LB agar plates with proper antibiotics for plasmid harbouring the oriT and antibiotic resistance of acceptor bacteria. On the next day the number of transconjugants was counted and therewith the conjugation efficiency calculated as ratio of cell density of transconjugants to cell density of donor cells depending on the reaction time. [ back ]
Receptor Polarization Test
To check the functionality of the fusion receptor constructs each construct, on pDK48, was transformed into strain UU1250 together with pDK36 (YFP-CheA). The expression of the receptor was induced with 0.01% arabinose, of YFP-CheA with 30µM IPTG. Polarization was tested by putting 10 µL of the cells on a 0.5% agarose pad and making snapshots using a Zeiss Axio Imager.Z1 fluorescent microscope.
Swarm Assays
The sensitivity of the LuxP-tar fusion constructs to AI-2 was tested in swarm assays. For this two different strains were used: E. coli UU1250 and E. coli HCB33. The assays were performed in TB medium for UU1250 and TB or minimal medium for HCB33 (0.3% agar). As chemo attractant either 3 µl of an AI-2 producing ONC, induced with 100 nM IPTG, or 10 µl supernatant from those cells was spotted 13 times on the plate to create a center line. Gradient building was allowed for 12-16 hours at 4 °C. Afterwards the fusion receptor expressing cells (0.01% arabinose induced overnight) as well as non-chemotactic reference strains were spotted at a distance of about 3-3.5 cm to the attractant line. Swarm plates were incubated at 30 °C and pictures taken after 24, 48 and 72 hours. [ back ]
Microscopy – Chemotaxis
An Applied Precision - DeltaVision wide field microscope was used to visualize chemotactic behaviour of the LuxP-tar fusion. E. coli UU1250 and E. coli HCB33 were used for these measurements. The monitoring was performed with LabtekTM chambered coverglass systems (Nunc) which were filled with 2 ml of TB media, Tethering Buffer and minimal media containing 0.2 % agar. As chemo attractant either 3 µl of an AI-2 producing ONC, induced with 100 nM IPTG, or 10 µl supernatant from those cells was spotted at the one side of the chamber. Gradient building was allowed for 12-16 hours at 4 °C. Afterwards either fusion receptor expressing cells (0.01% arabinose induced overnight) or non-chemotactic reference strains were spotted at the center of the chamber. The chamber was then incubated for several hours at 30 °C. After 3 hours frame pictures were taken, using confocal fluorescence imaging and 8x6 frames. (8 pictures in length, 6 in height) Images were processed and auto arranged with housemade ImageJ plugins. Matlab was used for the quantitative analysis. [ back ]
Microscopy – Colicin Toxicity
The killing efficiency of colicin producing cells was monitored using an Applied Precision - DeltaVision wide field microscope. Therefore a prey and a killer strain were designed in E. coli TOP10. The prey strain contained a GFP producing part ( BBa_I20260 ) as well as the luxI sender part ( BBa_K150000 ) and the killer strain contained as marker a mCherry producing part as well as the LuxR-colicinE1-receiver ( BBa_K150009 ). Measurements were prepared with 1 ml of a prey ONC and 1 ml of a killer ONC in 3 ml TB media containing 0,01% arabinose and 10 nM AHL for induction of colicin production. A second probe in which cells without colicin genes were used instead of the killer cells was prepared in parallel. Both samples were incubated for 6-7 hours at 37 °C. Monitoring was performed on the Deltavision microscope with 8-well LabtekTM chambered coverglass systems (Nunc). For that 6 µl of each sample were distributed equaly in 200 µl TB media. Frame pictures have been taken using 8x8 frames evenly distributed over the whole chamber. Images were processed and auto arranged with housemade ImageJ plugins. [ back ]
Microscopy – Cell lysis
To visualize the lysis of prey and killer cells cultures were observed on Zeiss Axio Imager.Z1 microscope. The killer strain (E. coli TOP 10) contained the luxR-colicinE1 part ( BBa_K150009 ) and a mCherry producing part and the prey strain contained the senderpart ( BBa_K150000 ) and a GFP producing part ( BBa_I20260 ). Both strains were inoculated in TB-media with appropriate antibiotics at 37 °C for 4 to 6 hours. Thereafter OD was adjusted and the killer strain was induced by 25 nM AHL. After 2 hour 200 µl of the induced killer culture and 200 µl of the prey culture were combined and centrifuged down. For microscopy pelleted cells were diluted with 10 µl and plated on a 0.5% agarosepad. Fixed positions were observed half hourly in the mCherry and GFP channel with a 100x oil immersion objective. [ back ]
Cell Culture
MCF-7 cells (breast cancer cell line) wt or stably expressing cytoplasmic GFP, were plated in 24 well plates (150 thousand cells per well) with 10 % FCS (fetal calf serum) DMEM (Dulbecco's Modified Eagle Medium) and incubated overnight at 37 °C, 5 % CO2. E. coli TOP 10 strains containing LuxR-colicin E1-receiver (BBa_K150009) and strains containing LuxR-receiver (no colicin) were grown overnight in TB media with appropriate antibiotics at 37 °C to an OD of 1.0 and diluted in the morning to 0.1. Once bacterial cells reached an OD of 0.35 they were harvested and inoculated on the MCF-7 wt and on MCF-7-GFP cells. [ back ]
Eucaryotic Killing Assay with Colicins
Once MCF-7 cells were confluent (12h after passage), E. coli TOP 10 bacteria were spinned down (1 ml of OD = 0.35) and resuspended with DMEM medium from the original wells containing the MCF-7 wt/MCF-7-GFP cells. A 24-well plate was used containing MCF-7  cells stained with Propidium Iodide (PI) alone, with LuxR-receiver cells not containing colicin and LuxR-receiver-colicin E1 cells. The second 24-well plate with MCF-7GFP was processed as described for wt MCF-7, but without any PI staining. AHL was added to every well to a final concentration of 5 nM. Well content was collected at different time-points (0 – 6 h) and when necessary, PI was added to the suspension 15 minutes before Flow Cytometer analysis.  [ back ]
Flow Cytometer Acquisition
At the referred timepoints, supernatants were collected and spinned down. Eukaryotic cells were detached by trypsinization (200 μl) for 10 minutes at 37 °C and then resuspended in 800 μl of KH + 1 % BSA (Krebs Henseleit medium with 1 % Bovine Serum Albumin). After resuspending, well content was added to the precipitated supernatant to account for floating dead cells. PI was added to each sample (1:1000) and incubated at RT for 15 minutes before acquisition. PI positive cells and GFP expressing cells (%) were measured for 3000 cells for each condition. Histograms and fluorescent measurements were obtained using a Cytomics FC 500 MPL from Beckman&Coulter. [ back ]
Retrieved from " http://2008.igem.org/Team:Heidelberg/Notebook/material "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
