"
Page
Discussion
View source
History
teams
Log in
 
Team:Lethbridge/Project
From 2009.igem.org
Home
Project Overview
Our Sponsors
Team
Project
Project Details
Experiments
Results
Ethics and Safety
Ethics and Synbio
The Survey
Safety
Notebook
Calendar
May
June
July
August
September
October
Meetings
April
May
June
July
August
September
October
Events
Modeling
Parts
Favourites
Individual Parts
Constructs
Intermediates
Collaboration
Judging
Extra Information on SynBio
 
Project Details
Many of the metabolic processes in eukaryotic cells are localized to various subcellular compartments. For example, transcription of mRNA from DNA is localized in the nucleus; post-translational modification and folding of proteins occurs in the endoplasmic reticulum (Voet et al., 2004).
This essentially segregates and regulates these processes. However, bacteria lack distinct organelles, and as such, many metabolic processes intermingle. Engineering an artificial organelle, capable of containing metabolic proteins, in bacteria would thus represent a fundamental advance in biotechnology. Towards this end, the University of Lethbridge iGEM team is working towards producing microcompartments from the lumazine synthase gene, which forms 60 subunit icosahedral capsids (Seebeck et al., 2006).
By generating a highly negative interior to the microcompartment, we hope to be able to target fluorescent proteins tagged with positively charged termini to the interior of the microcompartment (Figure 1). Utilizing fluorescence resonance energy transfer (FRET) between cyan and yellow fluorescent proteins, we will demonstrate the co-localization of these two proteins into the microcompartment.
For the future application of this technology we are working towards the targeting of proteins from the photosynthetic pathway into the microcompartment to optimize a biological photosynthetic fuel cell.  The segregation of proteins and metabolites increases the efficiency of cellular processes and following this theme, we are working towards creating uniform nanoparticles Through the segregation of Mms6 protein within Escherichia coli (Figure 2). (Prozorov et al., 2007).
We have also created a novel tool, the N and C terminus phusion vectors (BBa_K249004 and BBa_K249005) which will allow the targeting of any protein into the lumazine microcompartment.  These vectors are an important highlight of the team's project, as they create a straightforward method for introducing the positively charged arginine tail onto any protein using standard biobrick assembly methods.
References
Prozorov, T., Mallapragada, S. K., Narasimhan, B., Wang, L., Palo, P., Nilsen-Hamilton, M., Williams, T. J., Bazylinski, D. A., Prozorov, R., and Canfield, P. C. (2007). Protein-mediated synthesis of uniform superparamagnetic magnetite nanocrystals. Adv. Funct. Mater. Advanced Functional Materials. 17, 951-957
Seebeck, F. P., Woycechowsky, K. J., Zhuang, W., Rabe, J. P., and Hilvert, D., (2006). A simple tagging system for protein encapulation. J. Am. Chem. Soc. 128, 4516-4517
Voet, D., and Voet, J.G., (2004). Biochemistry 3rd Edition. John Wiley & Sons, Inc.
Home
Top of Page
 
The Experiments
Gene expression
The expression of a particular gene can be detected by measuring the amount of the protein product at different time points. The gene must be controlled in some sort of regulatory manner for expression to be calculable. The gene must be transformed into an expression cell strain, typically BL21 for E. coli. The cells are grown to an OD of 0.6. At this point the regulatory chemical is added to induce or repress the protein’s expression. Samples of the culture are taken at different time points. The cells are opened up and the contents are examined via some visualization technique, typically SDS-PAGE. We intend to view the expression of the following constructs in this manner.
pLacI:sRBS:mms6:dT (induced with IPTG)
pLacI:sRBS:Lumazine Synthase:dT (induced with IPTG)
pStrong:Riboswitch:cheZ:dT (induced with Theophylline)
pStrong:Riboswitch:GFP:dT (induced with Theophylline)
pBAD:RBS:TetR:dT:pTetR:mRBS:N-EYFP:dT (repressed with Arabinose)
pBAD:RBS:TetR:dT:pTetR:mRBS:C-EYFP:dT (repressed with Arabinose)
pBAD:RBS:TetR:dT:pTetR:mRBS:N-ECFP:dT (repressed with Arabinose)
pBAD:RBS:TetR:dT:pTetR:mRBS:C-ECFP:dT (repressed with Arabinose)
For the repressible constructs, as the repression of a protein is far more difficult to detect than the overexpression of a protein we intend to use fluorescence as our measurable, since in each case the protein which is repressed is a fluorescent protein. This is done in the same manner, as the expression, with samples of the culture being taken at different time points. Since we are unsure whether the intrinsic fluorescence of the cell will interfere with our measurements, we will measure the fluorescence of samples where the cells have been opened, as well as samples where they have not been opened (taken from the same culture, at the same time point). By measuring the change in fluorescence of the cells (or cell lysate) we can measure the effect of repression on the gene.
Motility Assay
The pStrong:Riboswitch:cheZ:dT construct is designed as a regulated motility construct. This gene has been designed to introduce regulated motility, to our motility knockout strain of E. coli, RP1616. In order to determine the mobility of our bacteria, we will qualitatively measure the apparent mobility of our bug in the absence and the presence of different concentrations of theophylline. We intend to do this in several different ways.
The first of which is simply a motility test. We will stab some of our bacteria into a motility media, in order to determine whether or not our bacteria are motile. A control of regular DH5a cells will be used as well.
The second is to plate the RP1616 cells with the motility construct on Plates containing different concentrations of theophylline. The cells which are plated where no theophylline is present are not expected to migrate from their original placement; Cells plated on relatively low concentrations (0.1 µM theophylline) are expected to migrate only slightly; and cells plated at moderate concentrations of theophylline (1 µM) are expected to migrate a fair distance; cells which are plated at concentration above the threshold for survival (1.5 – 2.0 µM) are not expected to survive.
The third method for determining relative motility is to view the movement of the cells via a microscope. Control cells (DH5a) are expected to have superb motility; RP1616 cells transformed with the motility construct are expected to behave differently at different concentrations of theophylline. Cells where no theophylline is present are expected to experience on a tumbling-like motion. Cells where low concentrations of theophylline are present are expected to mostly tumble, but some running is expected. Cells which are present under moderate concentrations of theophylline are expected to exhibit mostly a running motion. Cells which are present at theophylline concentrations above the threshold for survival are not expected to exhibit any motility whatsoever.
Electron microscopy
The Magnetite nanoparticle producing construct, pLacI:sRBS:mms6:dT, is expected to produce uniform nanoparticles. In order to visualize the size and shape of the nanoparticles we intend to use electron microscopy. With the aid of the electron microscope it is possible to visualize particles on the nanoscale. For this we will need to overexpress the protein (see gene expression) with IPTG. The magnetite, Fe3O4, nanoparticles form in the presence of FeCl2. Varying concentrations of FeCl2 will be added to the cells. After 24 hours, the cells will be opened up using chemical, mechanical and chemical/mechanical techniques, in order to determine if this poses any significant difference on the nanoparticles themselves. The nanoparticles are insoluble, and therefore can be filtered off from the cell lysate and washed. There are different types of filters which can be used to collect nanoparticles, we will use thin polycarbonate membrane as our filter of choice since the nanoparticles which we expect to obtain are on the smaller scale (6-20 nm in diameter). We will then place the sample under the electron microscope to visualize our magnetite nanoparticles.
Home
Top of Page
 
Fluorescence
Fluorescence will be used to determine the localization of a single fluorescent protein within our microcompartments. In order to do this, we will have to overexpress our Lumazine microcompartments using IPTG. Since we are not sure if the microcompartments from around our proteins, or if the proteins are targeted into the microcompartments we will perform the experiment in two different ways.
The lumazine synthase will be induced and the microcompartments will be allowed to form while the expression of the fluorescent protein (either YFP or CFP) is repressed using the Tetracycline inverter, which we have chosen to induce with the pBAD promoter (controlled by the presence of Arabinose). An arabinose pulse is added to repress the expression of the fluorescent protein, after some time, the arabinose will be used up and natural expression of the fluorescent protein will occur.
The lumazine synthase will be induced and the microcompartments will be allowed to form while expression of the fluorescent protein is allowed to take place. No arabinose is added.
The cells will then be visualized in the fluorimeter. In order to determine if the fluorescent proteins are localized with the microcompartments, it may be useful to visualize the cells with the aid of a confocal microscope. This will allow us to determine if the fluorescence is localized to specific points with the cells.
FRET
Fluorescence Resonance Energy Transfer, FRET will be used to determine the colocalization of two fluorescent proteins within the microcompartment. Yellow Fluorescent Protein (YFP) and Cyan Fluorescent Protein (CFP) are FRET pairs and are thus perfect for this application. The way that FRET works is that a donor molecule, in this case CFP, is excited by the application of an initial wavelength. The excited CFP emits fluorescence energy at a wavelength similar to that at which the acceptor molecule, in this case YFP, is excited. The fluorescence energy is thus transferred from molecule to another at that resonance wavelength. The acceptor molecule then emits fluorescence energy of its own at a wavelength which is distinguishable from the rest of the excitation and emission wavelengths. The energy transfer is distance dependent. If the two molecules are close in proximity (1 – 10 nm) the energy is transferred efficiently. The further away the two molecules are, the less efficient the energy transfer, and finally if the two molecules are too far away (more than 10 nm apart) the energy will not be transferred at all. CFP is excited at 436 nm, and emits fluorescence energy at 480 nm. YFP is excited at 514 nm and emits fluorescence energy at 538 nm. If the two fluorescent proteins are colocalized within the microcompartment, then one would expect that efficient fluorescence energy transfer would occur. If the two proteins are not colocalized, then one would expect not to see any fluorescent energy transfer.  In order to show fluorescence energy transfer, the intrinsic fluorescence of the cell will be used as a background, and the fluorescence of CFP alone in the cell will be compared to the fluorescence of CFP alone inside the microcompartment.
Home
Top of Page
 
RiPS measurements
In a similar way, fluorescence can be used to assess the theophylline riboswitch and rpsA TIR efficiencies, by generating values for the Ribosome initiation per second (or RiPS).
pStrong:Riboswitch:GFP:dT
pStrong:rpsA TIR:GFP:dT
Both the riboswitch and TIR act as alternative Ribosomal Binding Sites (RBS), by using GFP as the expression portion of these constructs it is possible to provide a calculation for the RiPS of these two parts. The parts are inserted in a RiPS generating plasmid (GFP producing plasmid). The fluorescence of GFP can be quantitatively measured using a fluorimeter. The fluorescence of GFP is measured over time and the Rise time, and expression time of the fluorescence is measured. These values are imported into a modelling program and the value for RiPS is generated for each of these parts. In order to measure the RiPS for the theophylline riboswitch, GFP’s fluorescence must be measured under varying concentrations of theophylline (0.1 µM, 0.25 µM, 0.5 µM, 0.75 µM, 1.0 µM, 1.25 µM, 1.5 µM and 2.0 µM). In order to measure the RiPS for the rspA TIR, GFP’s fluorescence must be measure under various stress conditions (i.e. Minimal media, heat, cold, various toxins, etc).
Results
Fluorescence
Two constructs (EYFP with either N- or C-terminal 10Arg (10R) tags) were characterized by the following procedure, either in the presence or absence of 0.4% w/v arabinose.
Briefly, 5 mL cultures (LB + 10 mg/mL ampicillin, with or without 0.4 % w/v arabinose) were inoculated with 1 mL of previously prepared culture of E. coli DH5α containing the EYFP constructs. After 1 hour growth at 37 °C (with shaking), 1 mL of the culture was harvested. Cells were centrifuged (13500 g, 5 min) and resuspended in 1 mL of either 8 M urea or buffer TAKM7 (50 mM Tris-Cl pH 7.5 at room temperature, 30 mM NH4Cl, 70 mM KCl, 7 mM MgCl2) containing 1 mg/mL lysozyme. Following a 10 min incubation at room temperature, cell debris was removed by centrifugation (13500 g, 5 min). The fluorescence of the resulting clear cell debris was determined using a Varian Cary Eclipse Fluorescence Spectrophotometer in quartz cuvettes, referenced with respect to the background fluorescence of either 8 M urea or buffer TAKM7. The fluorescence of EYFP was excited at 514 nm; the fluorescence emission was measured through 5 nm slits from 520 to 600 nm. The results are summarized below:
These results indicate that cells express either N- or C-terminal 10R tagged YFP, as shown by an increase in the fluorescence of cleared cell lysates. This fluorescence can be removed by treating cell lysates with 8 M urea, which would denature the protein, reducing the fluorescence of the lysate. This is demonstrated by a change of 10 fluorescence units between total cellular proteins under native conditions compared to denaturing conditions.
Home
Top of Page
 
Sequencing
Lumazine (K249002)
Score =  809 bits (1860),  Expect = 0.0
Identities = 465/465 (100%), Gaps = 0/465 (0%)
Strand=Plus/Plus
Query  112  ATGCAGATTTATGAAGGCAAACTGACCGCGGAAGGCCTGCGCTTTGGCATTGTGGCGAGC  171
Sbjct  1    ATGCAGATTTATGAAGGCAAACTGACCGCGGAAGGCCTGCGCTTTGGCATTGTGGCGAGC  60
Query  172  CGCTTTAACCATGCGCTGGTGGATCGCCTGGTGGAAGGCGCGATTGATTGCATTGTGCGC  231
Sbjct  61   CGCTTTAACCATGCGCTGGTGGATCGCCTGGTGGAAGGCGCGATTGATTGCATTGTGCGC  120
Query  232  CATGGTGGTCGCGAAGAAGATATTACCCTGGTGCGCGTGCCGGGCAGCTGGGAAATTCCG  291
Sbjct  121  CATGGTGGTCGCGAAGAAGATATTACCCTGGTGCGCGTGCCGGGCAGCTGGGAAATTCCG  180
Query  292  GTGGCGGCGGGCGAACTGGCGCGCAAAGAAGATATTGATGCGGTGATTGCGATTGGCGTG  351
Sbjct  181  GTGGCGGCGGGCGAACTGGCGCGCAAAGAAGATATTGATGCGGTGATTGCGATTGGCGTG  240
Query  352  CTGATTGAAGGCGCGGAACCGCATTTTGATTATATTGCGAGCGAAGTGAGCAAAGGCCTG  411
Sbjct  241  CTGATTGAAGGCGCGGAACCGCATTTTGATTATATTGCGAGCGAAGTGAGCAAAGGCCTG  300
Query  412  GCGAACCTGAGCCTGGAACTGCGCAAACCGATTACCTTTGGCGTGATTACCGCGGATGAA  471
Sbjct  301  GCGAACCTGAGCCTGGAACTGCGCAAACCGATTACCTTTGGCGTGATTACCGCGGATGAA  360
Query  472  CTGGAAGAAGCGATTGAACGCGCGGGCACCAAACATGGCAACAAAGGCTGGGAAGCGGCG  531
Sbjct  361  CTGGAAGAAGCGATTGAACGCGCGGGCACCAAACATGGCAACAAAGGCTGGGAAGCGGCG  420
Query  532  CTGAGCGCGATTGAAATGGCGAACCTGTTTAAAAGCCTGCGCTAG  576
Sbjct  421  CTGAGCGCGATTGAAATGGCGAACCTGTTTAAAAGCCTGCGCTAG  465
Home
Top of Page
 
Lumazine-dT (S042....)
>lcl|33723
Length=465
Score =  809 bits (1860),  Expect = 0.0
Identities = 465/465 (100%), Gaps = 0/465 (0%)
Strand=Plus/Plus
Query  108  ATGCAGATTTATGAAGGCAAACTGACCGCGGAAGGCCTGCGCTTTGGCATTGTGGCGAGC  167
Sbjct  1    ATGCAGATTTATGAAGGCAAACTGACCGCGGAAGGCCTGCGCTTTGGCATTGTGGCGAGC  60
Query  168  CGCTTTAACCATGCGCTGGTGGATCGCCTGGTGGAAGGCGCGATTGATTGCATTGTGCGC  227
Sbjct  61   CGCTTTAACCATGCGCTGGTGGATCGCCTGGTGGAAGGCGCGATTGATTGCATTGTGCGC  120
Query  228  CATGGTGGTCGCGAAGAAGATATTACCCTGGTGCGCGTGCCGGGCAGCTGGGAAATTCCG  287
Sbjct  121  CATGGTGGTCGCGAAGAAGATATTACCCTGGTGCGCGTGCCGGGCAGCTGGGAAATTCCG  180
Query  288  GTGGCGGCGGGCGAACTGGCGCGCAAAGAAGATATTGATGCGGTGATTGCGATTGGCGTG  347
Sbjct  181  GTGGCGGCGGGCGAACTGGCGCGCAAAGAAGATATTGATGCGGTGATTGCGATTGGCGTG  240
Query  348  CTGATTGAAGGCGCGGAACCGCATTTTGATTATATTGCGAGCGAAGTGAGCAAAGGCCTG  407
Sbjct  241  CTGATTGAAGGCGCGGAACCGCATTTTGATTATATTGCGAGCGAAGTGAGCAAAGGCCTG  300
Query  408  GCGAACCTGAGCCTGGAACTGCGCAAACCGATTACCTTTGGCGTGATTACCGCGGATGAA  467
Sbjct  301  GCGAACCTGAGCCTGGAACTGCGCAAACCGATTACCTTTGGCGTGATTACCGCGGATGAA  360
Query  468  CTGGAAGAAGCGATTGAACGCGCGGGCACCAAACATGGCAACAAAGGCTGGGAAGCGGCG  527
Sbjct  361  CTGGAAGAAGCGATTGAACGCGCGGGCACCAAACATGGCAACAAAGGCTGGGAAGCGGCG  420
Query  528  CTGAGCGCGATTGAAATGGCGAACCTGTTTAAAAGCCTGCGCTAG  572
Sbjct  421  CTGAGCGCGATTGAAATGGCGAACCTGTTTAAAAGCCTGCGCTAG  465
pBAD-TetR inverter
Score = 17.3 bits (36),  Expect = 6.6
Identities = 509/509 (100%), Gaps = 0/509 (0%)
Strand=Plus/Minus
Home
Top of Page
 
pTetT-EYFP
Score = 1247 bits (2868),  Expect = 0.0
Identities = 717/717 (100%), Gaps = 0/717 (0%)
Strand=Plus/Plus
pSB-N-terminus
Score = 60.0 bits (30),  Expect = 1e-15
Identities = 30/30 (100%), Gaps = 0/30 (0%)
Strand=Plus/Plus
Query  10
ATGCGCCGCCGCCGCCGCCGCCGCCGCCGC  39
Sbjct  1
ATGCGCCGCCGCCGCCGCCGCCGCCGCCGC  30
Score = 54.0 bits (27),  Expect = 6e-14
Identities = 27/27 (100%), Gaps = 0/27 (0%)
Strand=Plus/Plus
Query  16
CGCCGCCGCCGCCGCCGCCGCCGCCGC  42
Sbjct  4
CGCCGCCGCCGCCGCCGCCGCCGCCGC  30
pSB - C-terminus
>lcl|20071
Length=30
Score = 60.0 bits (30),  Expect = 1e-15
Identities = 30/30 (100%), Gaps = 0/30 (0%)
Strand=Plus/Plus
Query  12
CGCCGCCGCCGCCGCCGCCGCCGCCGCTAA  41
Sbjct  1
CGCCGCCGCCGCCGCCGCCGCCGCCGCTAA  30
Home
Top of Page
 
pBAD-TetR-CEYFP-dT
Score =  275 bits (143),  Expect = 1e-77
Identities = 143/143 (100%), Gaps = 0/143 (0%)
Strand=Plus/Plus
Query  139  TACTAGAGCCAGGCATCAAATAAAACGAAAGGCTCAGTCGAAAGACTGGGCCTTTCGTTT  198
Sbjct  704  TACTAGAGCCAGGCATCAAATAAAACGAAAGGCTCAGTCGAAAGACTGGGCCTTTCGTTT  763
Query  199  TATCTGTTGTTTGTCGGTGAACGCTCTCTACTAGAGTCACACTGGCTCACCTTCGGGTGG  258
Sbjct  764  TATCTGTTGTTTGTCGGTGAACGCTCTCTACTAGAGTCACACTGGCTCACCTTCGGGTGG  823
Query  259  GCCTTTCTGCGTTTATATACTAG  281
Sbjct  824  GCCTTTCTGCGTTTATATACTAG  846
pBAD-TetR-NEYFP-dT
Score = 41.1 bits (21),  Expect = 5e-07
Identities = 21/21 (100%), Gaps = 0/21 (0%)
Strand=Plus/Minus
Query  197  CATCTAGTATTTCTCCTCTTT  217
Sbjct  21   CATCTAGTATTTCTCCTCTTT  1
C-terminal ECFP-dT
Score = 41.1 bits (21),  Expect = 4e-07
Identities = 21/21 (100%), Gaps = 0/21 (0%)
Strand=Plus/Plus
Query  116  GGCATGGACGAGCTGTACAAG  136
Sbjct  697  GGCATGGACGAGCTGTACAAG  717
Score = 29.5 bits (15),  Expect = 0.001
Identities = 15/15 (100%), Gaps = 0/15 (0%)
Strand=Plus/Plus
Query  147  GACGAGCTGTACAAG  161
Sbjct  703  GACGAGCTGTACAAG  717
Home
Top of Page
 
C-terminal EYFP-dT
Score = 19.9 bits (10),  Expect = 0.80
Identities = 10/10 (100%), Gaps = 0/10 (0%)
Strand=Plus/Plus
Query  659  GAAGTTCATC  668
Sbjct  135  GAAGTTCATC  144
Score = 18.0 bits (9),  Expect = 3.0
Identities = 9/9 (100%), Gaps = 0/9 (0%)
Strand=Plus/Plus
Query  714  ACAGCCACA  722
Sbjct  440  ACAGCCACA  448
mRBS-N-terminal ECFP
Score =  335 bits (174),  Expect = 1e-95
Identities = 178/180 (98%), Gaps = 0/180 (0%)
Strand=Plus/Minus
Query  135  GGTGAGCAAGGGCACGGGCAGCTTGCCGGTGGTGCAGATGAACTTCAGGGTCAGCTTGCC  194
Sbjct  180  GGTGGGCCAGGGCACGGGCAGCTTGCCGGTGGTGCAGATGAACTTCAGGGTCAGCTTGCC  121
Query  195  GTAGGTGGCATCGCCCTCGCCCTCGCCGGACACGCTGAACTTGTGGCCGTTCACGTCGCC  254
Sbjct  120  GTAGGTGGCATCGCCCTCGCCCTCGCCGGACACGCTGAACTTGTGGCCGTTCACGTCGCC  61
Query  255  GTCCAGCTCGACCAGGATGGGCACCACCCCGGTGAACAGCTCCTCGCCCTTGCTCACCAT  314
Sbjct  60   GTCCAGCTCGACCAGGATGGGCACCACCCCGGTGAACAGCTCCTCGCCCTTGCTCACCAT  1
mRBS-N-terminal EYFP
Score = 1379 bits (717),  Expect = 0.0
Identities = 717/717 (100%), Gaps = 0/717 (0%)
Strand=Plus/Plus
Query  147  ATGGTGAGCAAGGGCGAGGAGCTGTTCACCGGGGTGGTGCCCATCCTGGTCGAGCTGGAC  206
Sbjct  1    ATGGTGAGCAAGGGCGAGGAGCTGTTCACCGGGGTGGTGCCCATCCTGGTCGAGCTGGAC  60
Query  207  GGCGACGTAAACGGCCACAAGTTCAGCGTGTCCGGCGAGGGCGAGGGCGATGCCACCTAC  266
Sbjct  61   GGCGACGTAAACGGCCACAAGTTCAGCGTGTCCGGCGAGGGCGAGGGCGATGCCACCTAC  120
Query  267  GGCAAGCTGACCCTGAAGTTCATCTGCACCACCGGCAAGCTGCCCGTGCCCTGGCCCACC  326
Sbjct  121  GGCAAGCTGACCCTGAAGTTCATCTGCACCACCGGCAAGCTGCCCGTGCCCTGGCCCACC  180
Query  327  CTCGTGACCACCTTCGGCTACGGCCTGCAATGCTTCGCCCGCTACCCCGACCACATGAAG  386
Sbjct  181  CTCGTGACCACCTTCGGCTACGGCCTGCAATGCTTCGCCCGCTACCCCGACCACATGAAG  240
Query  387  CTGCACGACTTCTTCAAGTCCGCCATGCCCGAAGGCTACGTCCAGGAGCGCACCATCTTC  446
Sbjct  241  CTGCACGACTTCTTCAAGTCCGCCATGCCCGAAGGCTACGTCCAGGAGCGCACCATCTTC  300
Query  447  TTCAAGGACGACGGCAACTACAAGACCCGCGCCGAGGTGAAGTTCGAGGGCGACACCCTG  506
Sbjct  301  TTCAAGGACGACGGCAACTACAAGACCCGCGCCGAGGTGAAGTTCGAGGGCGACACCCTG  360
Query  507  GTGAACCGCATCGAGCTGAAGGGCATCGACTTCAAGGAGGACGGCAACATCCTGGGGCAC  566
Sbjct  361  GTGAACCGCATCGAGCTGAAGGGCATCGACTTCAAGGAGGACGGCAACATCCTGGGGCAC  420
Query  567  AAGCTGGAGTACAACTACAACAGCCACAACGTCTATATCATGGCCGACAAGCAGAAGAAC  626
Sbjct  421  AAGCTGGAGTACAACTACAACAGCCACAACGTCTATATCATGGCCGACAAGCAGAAGAAC  480
Query  627  GGCATCAAGGTGAACTTCAAGATCCGCCACAACATCGAGGACGGCAGCGTGCAGCTCGCC  686
Sbjct  481  GGCATCAAGGTGAACTTCAAGATCCGCCACAACATCGAGGACGGCAGCGTGCAGCTCGCC  540
Query  687  GACCACTACCAGCAGAACACCCCCATCGGCGACGGCCCCGTGCTGCTGCCCGACAACCAC  746
Sbjct  541  GACCACTACCAGCAGAACACCCCCATCGGCGACGGCCCCGTGCTGCTGCCCGACAACCAC  600
Query  747  TACCTGAGCTACCAGTCCGCCCTGAGCAAAGACCCCAACGAGAAGCGCGATCACATGGTC  806
Sbjct  601  TACCTGAGCTACCAGTCCGCCCTGAGCAAAGACCCCAACGAGAAGCGCGATCACATGGTC  660
Query  807  CTGCTGGAGTTCGTGACCGCCGCCGGGATCACTCTCGGCATGGACGAGCTGTACAAG  863
Sbjct  661  CTGCTGGAGTTCGTGACCGCCGCCGGGATCACTCTCGGCATGGACGAGCTGTACAAG  717
Home
Top of Page
 
Overexpression
Figure 1:A SDS-PAGE gel of the overexpression of cheZ protein (50kDa). DH5 cells containing cheZ gene were inoculated in a 250mL LB media at 37oC, and the culture was induced with 1mM theothylline after the absorbance reached 0.6 a.u. at 600nm. The samples were collected every one hour after induction. The protein sample appears to be a monomer.
Figure 2:A growth curve for the overexpression of cheZ protein in DH5 cells. Arrow indicates the time when the culture was induced with 1mM theothylline.
Home
Top of Page
 
Retrieved from " http://2009.igem.org/Team:Lethbridge/Project "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
