<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Columbia-Cooper_Project"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Columbia-Cooper/Project">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Columbia-Cooper/Project&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Columbia-Cooper/Project&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Columbia-Cooper/Project&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Columbia-Cooper/Project" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Columbia-Cooper/Project</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><DIV id="circlemenu"><DIV id="circlemenuwrapper"><DIV class="circlebox" style="left:0px"><A class="circlelink home" href="https://2011.igem.org/Team:Columbia-Cooper">Home</A></DIV><DIV class="circlebox" style="left:120px"><A class="circlelink project" href="https://2011.igem.org/Team:Columbia-Cooper/Project">Project</A></DIV><DIV class="circlebox" style="left:240px"><A class="circlelink qd" href="https://2011.igem.org/Team:Columbia-Cooper/Quantum Dots">Quantum Dots</A></DIV><DIV class="circlebox" style="left:360px"><A class="circlelink notebook" href="https://2011.igem.org/Team:Columbia-Cooper/Notebook">Notebook</A></DIV></DIV></DIV></DIV><H1>Our Project/Data</H1><DIV style="text-align:left"><H2>Abstract</H2><P>We are using synthetic biology to build a bacteria-based system to biologically manufacture quantum dots (QDs). While QDs can be made through chemical processes, these processes are toxic, energy intensive, and yield dots that are challenging to use for promising biological applications. Furthermore, QDs created biologically are also thought to be more compatible with biological systems, require less energy to produce (Mi et. al.), and are more uniform in quality and color.
The addition of QD manufacturing to the toolbox of synthetic biology can expand the horizons of existing isolated systems; for example, motility control and light responsiveness ( might couple with dot production to generate self assembling circuits.</P><P>In order to achieve this, our team’s primary goal was to engineer E. coli bacteria to express several different peptides which bind to and nucleate salts of heavy metals, thereby crystallizing them into QDs.</P><P>In addition, we have designed a novel sensor/feedback device in order to enable the production of QDs with more uniform emission wavelengths. Since the size of QDs is directly related to their light emission spectrum, the goal is to have E. coli produced QDs, while growing in the presence of long wave UV light,  activate a light-sensitive promoter that is sensitive to the emission spectrum of the required QD size. This promoter is coupled to the expression of an antibiotic resistance cassette. As an initial proof-of-principal, our device uses a gene encoding for chloramphenicol antibiotic resistance, placed under the control of a blue light sensitive promoter, which had been <A href="https://2009.igem.org/Team:KULeuven/Design/Blue_Light_Receptor">previously characterized</A> by the 2009 iGEM team of K.U. Leuven. Thus, blue QD producing E. coli would stimulate the blue light promoter resulting in antibiotic resistance, allowing the survival of only the cells producing the desired wavelength of light.</P><H2>Goals and Strategies</H2><P>Our initial goal was to clone the nucleotide sequences corresponding to three small peptides, A7 (N-SLTPLTTSHLRS-C), Z8 (N-VISNHAESSRRL-C), and J140 ((N-TGCAACAACCCGATGCACCAGAACTGC-C) metal binding proteins, which have been previously shown by Mao, et. al. to nucleate zinc sulfide (A7 and Z8 peptides) and cadmium sulfide (J140 peptide) to form  quantum dot containing nano-wires using phage display.</P><P>Since all of the QD nucleating peptides were small sequences of 70 base pairs or less, we opted to generate the inserts using an oligonucleotide annealing procedure (see protocols) using designed oligos ordered from IDT. These were to be then cloned into the BioBrick vector PSB13C. Also, the sequence of the small peptide CDS7 (N-GDVHHHGRHGAEHADI-C), which previously demonstrated by Mi, et. al. to nucleate the formation of cadmium sulfide containing QDs, was synthesized by Invitrogen in a pANY vector and then amplified from the construct using primers containing either Biobrick ends conforming to RFC23 Silver lab standard or BamHI and NcoI restriction sites for cloning into the pET28 expression vector.</P><P>In addition to those four peptides, we identified an existing BioBrick part (Bba_K231000; Metal Binding Peptide) which we hypothesize to have the ability to nucleate Quantum Dots. We intend to further modify this part by adding additional restriction sites, BamHI and NcoI, internal to the BioBrick standard restriction sites in order that the part may be subcloned into the commercially available IPTG-inducible expression vector, pET28 and test it for this new application.</P><P>Third, we set out to create a device that would allow us to refine the biological QD manufacturing process to favor the production of uniform crystals of specific emission wavelengths. The device would consist of the Blue light promoter combined with a chloramphenicol resistance cassette.</P><P>Fourth, we would like to test the ability of the QD nucleating peptides to bind a wider range of less toxic metals such as zinc and selenium in order to expand their biocompatibility and lessen their environmental impact.</P><H2>Procedures</H2><H3>Oligo design for Quantum Dot nucleating peptides A7, Z8 and J140</H3><P>We utilized the “Gene Synthesis Optimization Program”, <A href="http://gcat.davidson.edu/IGEM06/oligo.html">originally developed</A> by the 2006 Davidson College iGEM team, to design a series of overlapping single-stranded oligos for subsequent annealing reactions. For each sequence, four overlapping oligos were annealed and ligated into a PSB1C3 vector digested with EcoR1 and Spe1 and gel purified.</P><P>The oligos used for the annealing reactions were as follows:</P><P>Oligos for peptide J140 (for Cd2S quantum dots)</P><UL><P>24-mer 5'-AATTCGCGGCCGCTTCTAGAGCCA-3’</P><P>52-mer 5'-TGGGCGTTATCTCTAACCACGCGGAATCTTCTCGTCGTCTGTAAGGATCCTA-3’</P><P>43-mer 5'-CGCGTGGTTAGAGATAACGCCCATGGCTCTAGAAGCGGCCGCG-3’</P><P>33-mer 5'-CTAGTAGGATCCTTACAGACGACGAGAAGATTC-3’</P></UL><P>Oligos for peptide A7 (for ZnS quantum dots)</P><UL><P>24-mer 5'-AATTCGCGGCCGCTTCTAGAGCCA-3’</P><P>43-mer 5'-TGGGCTGCAACAACCCGATGCACCAGAACTGCTAAGGATCCTA-3’</P><P>40-mer 5'-CATCGGGTTGTTGCAGCCCATGGCTCTAGAAGCGGCCGCG-3’</P><P>27-mer 5'-CTAGTAGGATCCTTAGCAGTTCTGGTG-3’</P></UL><P>Oligos for peptide Z8 (for ZnS quantum dots)</P><UL><P>24-mer 5'-AATTCGCGGCCGCTTCTAGAGCCA-3’</P><P>52-mer 5'-TGGGCGTTATCTCTAACCACGCGGAATCTTCTCGTCGTCTGTAAGGATCCTA-3’</P><P>43-mer 5'-CGCGTGGTTAGAGATAACGCCCATGGCTCTAGAAGCGGCCGCG-3’</P><P>33-mer 5'-CTAGTAGGATCCTTACAGACGACGAGAAGATTC-3’</P></UL><H3>Oligo annealing reactions</H3><P>The oligos for the quantum dot nucleation peptide sequences were annealed using the <A href="http://openwetware.org/wiki/Silver:_Oligonucleotide_Inserts">Silver lab protocol</A>.</P><H3>Synthesis of quantum dot nucleating peptide sequence CDS7</H3><P>The CDS7 insert for ligation was synthesized by Invitrogen/Mr. Gene and cloned into the pANY vector. We used PCR to amplify the CDS7 insert from the pANY vector using the following primers:</P><P>Forward primer: 5’-CGATCGAGAATTCGCGGCCGCTTCTAGAGCCATCATCATCATCATCAC-3’</P><P>Reverse primer:
5’-GCTATGCACTGCAGCGGCCGCTACTAGTTAAATATCCGCATGTTCCGC-3’</P><P>The PCR product was digested with EcoRI and PstI and ligated into the backbone plasmid PSB1C3. 
For cloning the CDS7 insert into the pET28 expression vector, both pET28 vector and PSB1C3 containing CDS7 were digested with BamHI.The digested pET28 vector was treated with antarctic phosphatase and  then ligated to the PSB1C3 vector containing the CDS7 insert. Ligated constructs generated were approximately about 7Kb in length. This new construct was then digested with NcoI and the larger fragment, the pET28 backbone containing the CDS7 insert, was gel purified and then self-ligated.</P><P>The sequence of the synthesized CDS7 insert, containing RFC23 Silver lab standard BioBrick ends and NcoI and BamHI restriction sites is:</P><P>5’-GAATTCGCGGCCGCTTCTAGAGCCATGGGCCATCATCATCATCATCACGGCGATGTGCATCATCATGGCCGCCACGGCGCGGAACATGCGGATATTT... 
AAGGATCCTACTAGTAGCGGCCGCTGCAG-3’</P><H3>Ligations</H3><P>Ligations of QD binding peptide sequences into PSB1C3 and pET28 were performed using the protocol listed in the <A href="http://partsregistry.org/Help:Protocols/Linearized_Plasmid_Backbones">Registry of Standard Biological Parts</A>. In some case different enzymatic digestions were used for the appropriate vector, i.e BamHI and NcoI for the pET28 IPTG inducible expression vector. Ligated plasmids were sent out to GeneWiz for sequence confirmation.</P><H3>Transformations</H3><P>Transformations were performed using either NEB Turbo Competent E. coli cells and following the high efficiency transformation protocol recommended by the manufacturer or using fresh cultures of JM109 E. coli cells that had been made competent using the Fermentas TransformAid bacterial transformation kit and following the manufacturer’s recommended protocol.</P><H3>Quantum Dot Production in E. coli (Modified from Mi, et. al.)</H3><OL><LI>Inoculate single colonies transformed with pET28-CDS7 into 1m of LB-Kanamycin media and incubate for 8 hours in a shaking incubator, set at 250 rpm, at 37 degrees Celsius until an O.D. 600 of ~1.0 was reached.</LI><LI>From the culture, re-inoculate 5 ml of LB-Kanamycin media to a starting optical density of 0.1 at 600 nm. </LI><LI>Incubate until mid-log phase is reached, O.D. 600 ~0.5, ~2 hours.</LI><LI>Add IPTG (formula weight.=238.3; add 0.0024 grams per 20ml LB-Kan) to a final concentration of 0.5mM and cadmium chloride (formula weight.= 183.3; add 0.0037 grams per 20ml LB-Kan ) to a final concentration of 1mM. </LI><LI>Incubate in the shaker for an additional 3 hours.</LI><LI>Slowly add a freshly prepared solution of sodium sulfide (anhydrous formula weight=78, nonahydrate formula weight=240.2; we made a 100mM stock solution, 0.024 gms per ml for nonahydrate) into LB-Kan to a final concentration of 1mM.</LI><LI>Incubate at room temperature with slow “end-over-end” rotation for 1.5 hours.</LI><LI>Centrifuge and wash samples 3 times with distilled water and characterize with fluorescence spectrometry. (350nm excitation, 450nm emission for 1mM reagents, 510nm emission for 10mM reagents)</LI></OL><H2>Blue light stimulated antibiotic resistance device</H2><P>Our new device consists of a blue light inducible promoter (part BBa_K28013) that had been previously characterized, driving the expression of a previously submitted chloramphenicol resistance gene (part BBa_P1004).  When blue light is present, the device activates chloramphenicol resistance.  This device was intended as a system for using antibiotic selection to generate quantum dots with a narrow range of wavelengths. Our submitted part uses the pSB1A3 backbone as opposed to the psB1C3 backbone, since the device produces chloramphenicol resistance in the cells.</P><P>The blue light promoter is inhibited by native repressor ycgF.  Without dimerizing with ycgE, ycgF will remain bound to the DNA and prevent transcription.  When blue light is present,  ycgF changes conformation and dimerizes with ycgE.  Dimerized ycgF releases from the promoter region and no longer represses gene transcription of the chloramphenicol resistance gene.</P><P>The incubation protocol is as follows:</P><OL><LI>Transformed E. coli samples containing the Blue light stimulated antibiotic resistance device and controls were incubated at 37 degrees Celsius, in a shaking incubator, irradiated in blue light. </LI><LI>The light induced samples were incubated in a foil lined container with 73 blue LEDs that were positioned approximately 2 cm above the sample tubes containing 1mL bacteria, 3.5ml LB broth and 5uL ampicillin.</LI><LI>The OD 600 reading was recorded for each sample every hour. The dilution was 40uL sample and 160uL sterile water. 40uL Lb broth and 160uL water was used to zero the spectrophotometer.</LI><LI>After the first hour of growth 5uL of chloramphenicol was added to each
sample. Another 5uL was added to each sample after the 3 hour mark to
ensure that there was a sufficiently high concentration of chlorophenicol within all samples.</LI></OL><H2>Results</H2><H3>QD nucleation experiments performed with CDS7 peptide in pET28b</H3><P>Bacterial cells from two colonies resulting from a transformation of JM109 with pET28-CDS7 were compared to pET28 alone for their ability to nucleate QDs.</P><P>5 mL cultures were inoculated to OD600 0.1 and grown at 37C w/shaking at 225 rpm until OD600 reached 0.5 at which time the synthesis of the peptide was induced with IPTG 0.5mM. In three separate experiments, cells were either immediately treated with 1mM or 10mM CdCl or were allowed to incubate with the IPTG for 2h prior to addition of CdCl.  Following CdCl addition the cells were incubated for 3h. They were then removed from the 37C shaker and sodium sulfide added slowly to 1mM. The tubes were placed on a rotator at room temperature for 1.5 h, after which the cells were washed 4x with 10mL volumes of distilled water and analyzed via fluorescence microscopy with UV excitation.  The sole variable determining fluorescence intensity appeared to be the CdCl treatment. Cells containing vector with no CDS7 insert appeared as bright to the eye as those engineered to produce CDS7. In addition, the presence or absence of IPTG did not appear to affect the intensity. We intend to pursue further experiments since it cannot yet be determined if there was a flaw in our QD manufacturing procedure, an inadequately sensitive measuring system, or whether the report of CDS7’s capabilities in the literature were erroneous.</P><H3>Blue promoter verified</H3><H3>400x UV Fluorescence Images of Bacteria</H3><H3>Video of Engineered Bacteria</H3><H2>References</H2><P>Biosynthesis and characterization of CdS quantum dots in genetically engineered
Escherichia coli. Congcong Mi, Yanyan Wang, Jingpu Zhang, Huaiqing Huang, Linru Xu, Shuo 
Wang, Xuexun Fang, Jin Fang, Chuanbin Mao, Shukun Xu. Journal of Biotechnology. 153 (2011) 125-132.</P><P>Viral assembly of oriented quantum dot nanowires. Chuanbin Mao, Christine E. Flynn, Andrew Hayhurst, Rozamond Sweeney, Jifa Qi, George Georgiou,
Brent Iverson, and Angela M. Belcher. PNAS. 100:12 (2003) 6946-6951.</P></DIV><H3><SPAN class="mw-headline" id="Parts_submitted_by_our_team.21">Parts submitted by our team!</SPAN></H3><P>&lt;groupparts&gt;iGEM011 Columbia-Cooper&lt;/groupparts&gt;</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:Columbia-Cooper/Project">http://2011.igem.org/Team:Columbia-Cooper/Project</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Columbia-Cooper/Project" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Columbia-Cooper/Project" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Columbia-Cooper/Project&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Columbia-Cooper/Project&amp;oldid=258294" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>