<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Columbia-Cooper_CUNotebook"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Columbia-Cooper/CUNotebook">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Columbia-Cooper/CUNotebook&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Columbia-Cooper/CUNotebook&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Columbia-Cooper/CUNotebook&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Columbia-Cooper/CUNotebook" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Columbia-Cooper/CUNotebook</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><DIV id="circlemenu"><DIV id="circlemenuwrapper"><DIV class="circlebox" style="left:0px"><A class="circlelink home" href="https://2011.igem.org/Team:Columbia-Cooper">Home</A></DIV><DIV class="circlebox" style="left:120px"><A class="circlelink project" href="https://2011.igem.org/Team:Columbia-Cooper/Project">Project</A></DIV><DIV class="circlebox" style="left:240px"><A class="circlelink qd" href="https://2011.igem.org/Team:Columbia-Cooper/Quantum Dots">Quantum Dots</A></DIV><DIV class="circlebox" style="left:360px"><A class="circlelink notebook" href="https://2011.igem.org/Team:Columbia-Cooper/Notebook">Notebook</A></DIV></DIV></DIV></DIV><H1>Cooper-Union Lab Notebook</H1><DIV style="text-align:left"><H2>October 25</H2><UL><LI>Restriction Digest + Ligation</LI><LI>Double Digest with EcoRI + Pst</LI><LI>a) Spe+Pst* BLP buffer 2 bufer b</LI><LI>b) GFP (kan j54103(\</LI><LI>c) chl)
</LI><LI>Double Digest with Ncol &amp; BamHI</LI><LI>d) pET28</LI><LI>e) annealed A7</LI><LI>f) annealed Z8</LI><LI>Reaction</LI><LI>10x RXn buffer 2uL</LI><LI>10x BSA 2uL</LI><LI>ddH2O 10uL</LI><LI>template 5uL</LI><LI>enzyme mix 1uL</LI><LI>total 20uL</LI><LI>incubated at 37 degrees celsius for one hour. heat denatured at 65 degrees</LI><LI>celsius for 20minutes</LI></UL><UL><LI>Ligation 10x bufffer 1uL</LI><LI>ddH2O 4.5uL</LI><LI>vector DNA 1uL</LI><LI>Insert DNA</LI><LI>T4 DNA Ligase 0.5uL</LI></UL><H2>October 20</H2><P>Kinase Reaction: use up to 300pmol of 5' termini in 50uL Rxn.</P><P>Kinase Rxn:</P><UL><LI>10x Kinase Buffer 1uL</LI><LI>T4 PNK 1uL</LI><LI>Template(A7-[43,40,24,27] Z8-[33,52,24,43]) 1uL</LI><LI>dd H2O 7uL</LI></UL><P>incubated at 37degrees celsius for 30minutes</P><P>heat denature 20min at 65degrees celsius.  store at -20</P><P>thin walled PCR tubes</P><P>oligo annealing RXn:</P><UL><LI>10x kinase buffer 2uL</LI><LI>Primer1 2uL</LI><LI>Primer 2 2uL</LI><LI>Primer 3 2uL</LI><LI>Primer 4 2uL</LI><LI>dd H2O 10uL</LI></UL><P>Turn on thermocyclin-program 95degrees for 3-5min turn off thermocyclin.</P><P>allow heat block to come to Resting Temperature (1-2)hours.</P><P>store -20</P><H2>October 14</H2><P>The blue light test was run again today. Only cultures with BLP + Chlor were grown; two in spray painted tubes with tinfoil and two exposed to blue light. Again the results proved uncharacteristic. A PCR is being run in order to determine whether the genes present in the cultures are correct. If the BLP + Chlor is not the correct length then it will have to be remade using the original cassettes and a ligation procedure.</P><H2>October 13</H2><P>Today a blue light test was run. Cultures with BLP + Chlor, grown both exposed to blue light and in the dark, were sampled in triplicate every hour. Chloramphenicol was added after the first hour passed. The results were unexpected. Growth continued in every sample, indicating that growth was not inhibited by the absence of blue light. Cultures of BLP + Kan, kanamycin resistance, were run simultaneously in both dark and blue light, as controls for growth. Chloramphenicol did not inhibit the growth of these cultures. This result was not expected.</P><P>Two errors in the experiment are being considered. Light leakage and/or incorrect gene may cause the error observed today. Tomorrow the blue light test will be run again, to determine similar results occur. The sample tubes will be spray painted black and covered in tin foil in order to further prevent light leakage. If tomorrow’s results are also uncharacteristic then PCR analysis will be used to test the length of the genes present in the BLP + Chlor samples.</P><H2>October 12</H2><P>Cultures of pET28 + CDS7 and BLP + Chlor were grown up last night. Today, the cultures of pET28 + CDS7 were also to produce quantum dots (1). Also, a co-transformation was performed, in which pET28 + Chlor and BLP + Chlor were transformed into E. Coli, strain JM109 (2).</P><P>(1) The pET + CDS7 culture was processed using the following procedure to make quantum dots.</P><P> Procedure: </P><OL><LI>Transfer 18 ml LB broth to 50 ml conical tubes. Add 20 microliters of Kanamycin.</LI><LI>Inoculate with 2ml of pET28 + CDS7 culture grown overnight.</LI><LI>Incubate at 27 degrees Celsius for 2 hours or until an OD600 is reached.</LI><LI>Add 10 microliters of 1M IPTG and 200 microliters of 100mM CdCl2.
a. IPTG allows for the transcription of the lac operon and is advantageous because it can not be metabolized by E. Coli </LI><LI>allowing its concentration to remain constant. IPTG activates the promoter for the pET + CDS7.</LI><LI>Incubate for 3 hours at 37 degrees Celsius.</LI><LI>Add 0.0048 grams of NaS(9H20) to 100 microliters of double distilled H20. Add solution drop wise.</LI><LI>Incubate at 25 degrees Celsius for 1.5 hours.</LI></OL><P>The culture solutions made to produce quantum dots were centrifuged in order to isolate the quantum dots. The pellets isolated today will be tested tomorrow in the fluorescent spectrophotometer to determine the wavelength of light radiated. This will indicate whether quantum dots were produced.</P><P>(2) Strain JM109 of E. Coli was transformed with both BLP + Chlor and pET28 + CDS7. The bacteria were cultured on plates containing both kanamycin and ampicillin. Tomorrow they will be observed for growth.
The backbone of the BLP + Chlor  has the origin site pMBI which differs from the origin of the pET28  + CDS7, ori CoIEI, by only one basepair. The difference by one basepair is a site-specific mutation and will be recognized as such by the cell. The BLP + Chlor is about 3000 basepairs and the pET28 - CDS7 is about 5500 basepairs. Because the BLP + Chlor is less energy expensive it will be transcribed preferentially over pET28 + CDS7. This means a deficit of pET28 + CDS7 will exist.</P><P>A solution to this problem is changing the backbones of each of the plasmids. We must wait to see if the colonies grow on plates containing both kanamycin and ampicillin. If no colonies grow, then the backbones of the plasmids will need to be replaced.</P><P>The BLP + Chlor backbone contains ampicillin resistance while the pET28 + CDS7 has resistance to kanamycin. If the colonies survive on the plates then both kanamycin and ampicillin have been expressed, meaning both plasmids have been transcribed. This indicates the risk of “losing” the CDS7 plasmid was avoided. </P><H2>September 26</H2><P>Samples C-4 K2, K8, G5,G8 were grown under blue light in tubes containing 1mL
bacteria, 3.5ml LB broth and 5uL ampicilin.</P><P>Every hour the OD 600 reading was taken for each sample with the
spectrophotometer. the dilution was 40uL bacteria, 160uL autoclaved
water. A 40uL Lb broth 160uL water was used to  eliminate background noise.</P><P>After the first hour of growth 5uL of Chlorophenicol was added to each
sample.Another 5uL was added to each sample after the 3 hour mark to
ensure enough chlorophenicol concentration within all tubes.</P><H2>September 22</H2><P>QIAquick Gel Extraction Kit Protocol
with chloramphenicol 0.175g gel +0.645uL QG buffer
kanamicin 0.2024g gel + 1.404uL QG buffer
GFP 0.058g gel  +0.3510QG buffer</P><OL><LI>Excise the DNA fragment from the agarose gel with a clean, sharp scalpel.</LI><LI>Weigh the gel slice in a colorless tube. Add 3 volumes Buffer QG to 1
volume gel (100 mg ~ 100 &amp;).</LI><LI>Incubate at 50°C for 10 min (or until the gel slice has completely
dissolved).Vortex the tube every 2–3 min to help dissolve gel.</LI><LI>After the gel slice has dissolved completely, check that the color of
the mixture is yellow (similar to Buffer QG without dissolved agarose). If
the color of the
mixture is orange or violet, add 10 ;l 3 M sodium acetate, pH 5.0,
and mix. The
color of the mixture will turn yellow.</LI><LI>Add 1 gel volume of isopropanol to the sample and mix.
October 2010
Quick-Start Protocol
Sample &amp; Assay Technologies</LI><LI>Place a QIAquick spin column in a provided 2 ml collection
tube or into a vacuum manifold.</LI><LI>To bind DNA, apply the sample to the QIAquick column and centrifuge for
1 min or apply vacuum to the manifold until all the samples
have passed
through the column. Discard flow-through and place the QIAquick
column back into the same tube.</LI><LI>If the DNA will subsequently be used for sequencing, in vitro
transcription, or
microinjection, add 0.5 ml Buffer QG to the QIAquick column and centrifuge
for 1 min or apply vacuum.  Discard flow-through and
place the QIAquick
column back into the same tube.</LI><LI>To wash, add 0.75 ml Buffer PE to QIAquick column and centrifuge for 1
min back into the same tube.</LI><LI>Centrifuge the QIAquick column once more in the provided 2 ml
collection tube
for 1 min at 17,900 x g (13,000 rpm) to remove residual wash buffer.</LI><LI>Place QIAquick column into a clean 1.5 ml microcentrifuge tube.</LI><LI>To elute DNA, add 50 Buffer EB (10 mM Tris·Cl, pH 8.5) or
water to the center
of the QIAquick membrane and centrifuge the column for 1 min. For increased
DNA concentration, add 30 &amp;Buffer EB to the center of the QIAquick
membrane, let the column stand for 1 min, and then centrifuge for 1 min.</LI></OL><H2>September 13</H2><P>Transformation-again with more ampicilin to isolate colonies of bacterial growth.</P><P>For each transformation, pre-warm 37 C 155uL LB broth, 5uL Sterile glucose</P><P>Add 1uL ligation to 40uL bacteria incubate on ice for 10-15 min</P><P>Heat shock. 50 seconds at 42C incubate immediately on ice for 2-3 min</P><P>Transfer bacteria + DNA to prewarmed LB. Incubate 1 hour at 37C</P><P>Add 10 uL ampicilin to bacteria 6) transfer 25 uL to plate 1</P><P>Transfer rest to plate 2</P><P>Grow overnight at 37C</P><H2>September 8</H2><P>Qia spin rep for GFP to isolate DNA</P><P>Following DNA isolation both GFP and BLP were digested.</P><P>BLP was digested with Spe &amp; Pst enzymes</P><P>the following were added to a 1.5ml micro centrifuge for a total of a 20uL
reaction:</P><UL><LI>2uL 10x Buffer B</LI><LI>2uL 10x BSA</LI><LI>13uL Plasmid</LI><LI>* 2uL Pst I</LI><LI>1uL Spe1</LI></UL><P>* twice the normal amount of Pst I restriction enzyme was used to balance
out the imbalances in Buffer B*</P><P>The GFP was digested with xba &amp; Pst enzymes</P><P>the following were added to a 1.5ml micro centrifuge for a total of a 20uL
reaction</P><UL><LI>2ul 10x Buffer d3</LI><LI>2uL 10x BSA</LI><LI>14uL Plasmid</LI><LI>1uL xba</LI><LI>1ul Pst</LI></UL><P>Both tubes were than incubated at 37 degrees celsius for 60 min
and then heat inactivated at 65 degrees celsius for 15min.</P><H2>September 6</H2><P>The part assumed as BLP + GFP was processed using electrophoresis and found too short to be the part claimed by the registry. Therefore, our next goal is to supplement our own BLP + GFP part. This will be accomplished by ligating BLP to GFP ourselves. Today we began this process by reconstituting promoter-less GFP using the following procedure:</P><P>Plate 2, part J54103 (promoter-less GFP):</P><OL><LI> Reconstitute biobrick part  in 10 microliters double distilled water.</LI><LI> Add 2 microliters re-hydrated DNA to 40 microliters of competent cells.</LI><LI> Incubate on ice for 10 - 15 minutes.</LI><LI> Heat shock at 42 degrees Celsius to allow plasmids bound to calcium to pass through cell membranes.</LI><LI> Chill on ice for 2-3 minutes to seal pores in cells.</LI><LI> Incubate in 160 microliters of LB and glucose broth for 1 hour at 37 degrees Celsius.</LI><LI> Plate the solution on a Kanamycin resistance plate and grow overnight at 37 degrees Celsius.</LI></OL><H2>August 24</H2><P>Yesterday the BLP+GFP plasmid was double digested with EcoRI and PstI to cut
out the insert; the backbone should be 4022bp and the insert is 970bp. The
gel results were inconclusive - the uncut plasmid was running smaller than
the cut plasmid (expected), but the cut plasmid appeared to be 6000bp long.
No 1000bp fragment was observed. Unfortunately, the stain was either too old
or the band was too faint to be detected by my camera.</P><P>The remedy for this was to rerun the digest (half of the reaction is
available), but added ethidium bromide to both the gel and the buffer while
the instrument is on. This should remove the stain time required after
electrophoresis and improve resolution, however caution must be used since
EtBr is a mutagen. Also, we single digested the plasmid too to verify the
wholeplasmid size. And made a differ DNA marker which may be easier to
compare our target fragment sizes to our samples.</P><P>Re run digest:</P><UL><LI>Use Lambda/PstI Marker</LI><LI>Run undigested plasmid</LI><LI>Run single digested plasmid</LI><LI>Run double digested plasmid</LI><LI>Use EtBr in the gel and buffer</LI></UL><H2>August 22</H2><P>We did the following
 a) linearize the plasmid b) cut the insert out c) run this
on a gel to size. set up overnight cultures to miniprep tomorrow
(Tuesday).</P><P>In preparation for the BLP test in is *absolute* dark. We grew up competent
strains and made competent cells. Then transformed with the BLP+GFP and pGLO.</P><H2>August 16</H2><P>Experiment ran and data posted on google docs (8.16.11 blp data).
ideally). Samples were inoculated for testing.</P><H2>August 15</H2><P>Inoculated mini-cultures in preparation for blue light sensor experiment
to be run for a prolonged time (6+ hours).</P><H2>August 10</H2><P>Today the attached procedure outlined for synthesizing quantum dots biologically was followed for each of the bacteria cultures made on August 8. However, when samples were prepared using this protocol, certain variables were left out. This decision made these samples into controls. As such, they were used to determine whether quantum dots had actually been synthesized last week.</P><P>Notice, in the list of samples prepared below certain samples were made excluding one of the following variables: cadmium chloride solution, sodium sulfide or bacteria.</P><OL><LI>DH5alpha, Cadmium Chloride and Sodium Sulfide</LI><LI>CDS7, Cadmium Chloride and Sodium Sulfide</LI><LI>127, Cadmium Chloride and Sodium Sulfide</LI><LI>CDS7 and Sodium Sulfide</LI><LI>CDS7 and Cadmium Chloride</LI><LI>Cadmium Chloride and Sodium Sulfide</LI></OL><P>It was unclear whether quantum dots had been synthesized because each supernatant of each controls fluoresced. It was found that LB broth is fluorescent. Because LB has this quality and is present in all of our samples, the presence of quantum dots can not be confirmed using fluorescent spectrometry at this time. The quantum dots supposedly produced last week may not be present and the fluorescence of the sample attributable to the LB broth alone.</P><H2>August 8</H2><P>Today was used in preparation for experiments planned for the following days.</P><P>In order to verify the production of quantum dots accomplished last week, bacteria containing the following parts were cultured separately:</P><P>Ran Chemical dot synthesis with olive oil as oleic aicd substitute as
oleic acid had not arrived in time.</P><OL><LI>CDS7</LI><LI>Rel Light Promoter and GFP (labeled as 127, the last three digits of its part identification number)</LI><LI>DH5alpha</LI></OL><P>These bacteria were cultured in LB prepared today. 720 ml of LB was made all total.</P><H2>August 2</H2><P>The procedure for August 1 was repeated, but less reliable results were obtained.</P><H2>August 1</H2><P>Task 1: 
Today we developed a protocol to be completed tomorrow outlining the procedure to synthesize quantum dots. The protocol was adopted from the journal article &quot;Biosynthesis and Characterization of CdS Quantum Dots in Genetically Engineered E. Coli.&quot;</P><P>Note, instead of using the pET - 28b vector, which requires IPTG as a promoter, our protocol calls for pMA vector, requiring no promoter.</P><P>The CdCl<SUB>2</SUB> and Na2S[9H<SUB>2</SUB>O] solutions necessary for our protocol were prepared today.</P><P>Task 2:
The procedure completed last Thursday, July 28, will be repeated tomorrow, Aug 2. Today, the bacteria containing the following parts were inoculated.</P><OL><LI>pUC19</LI><LI>pUC19</LI><LI>GFP and arabinose</LI><LI>GFP alone</LI><LI>BL alone</LI><LI>BL with GFP</LI><LI>BL with GFP</LI><LI>CD7</LI></OL><H2>July 28</H2><P>Minicultures of cultures made the evening before were prepared with the following components:</P><UL><LI> 3.5 ml of LB broth</LI><LI>4 microliters of Ampicillin</LI><LI>4 microliters of Arabinose when required</LI><LI>0.5 ml of appropriate culture</LI></UL><P>Using the components listed the following minicultures were prepared. Their labels refer to how the cultures would be incubated, either in the presence of blue light from the LED box or in the absence of light.</P><OL><LI> 470 nm Blue Light Promoter (BLP) and Green Fluorescent Protein (GFP)</LI><LI> unused sample</LI><LI> 470 nm pUC19</LI><LI> Dark BLP and GFP</LI><LI> unused sample</LI><LI> Dark pGLO without Arabinose</LI><LI> Dark pGLO with Arabinose</LI><LI> Dark pUC19</LI></OL><P>Cultures 1 and 3 were set to incubate in the LED box constructed on July 27. The remaining cultures were set to incubate in a hot water bath in the dark. Phosphorescent readings were taken at timestamps 30min, 60, 120 and 180 for each culture. Each culture was sampled in triplicate on a 96 well plate using LB broth as the control.</P><H2>July 27</H2><P>A goal of our project is to use LED lights to signal bacteria into synthesizing quantum dots. Today, the LED box was constructed. Two bread boards were lined with about 50 LEDs (Light Emitting Diodes), resistors (330 Ohms), and a 9 Volt battery. This plate was fastened to the ceiling of a foil lined box. A switch was also soldered into the circuit. The box is large enough to hold multiple minicultures, preps in which cells will be incubated. The blue light from the LEDs used will signal the blue light sensors transformed into the bacteria which will in turn initiate the synthesis of quantum dots.</P><H2>July 25</H2><P>Gels of agarose concentrations 1.5%, 2% and 3% were loaded with prepped digest samples from July 21. The gels were loaded as follows. Single and double refer to the type of digest. (Two 1.5% gels were used.) The gels were analyzed during the next lab session.</P><TABLE><TBODY><TR><TD><B>Gel 1</B></TD><TD>1.5% A</TD></TR><TR><TD>Well</TD><TD>1</TD><TD>2</TD><TD>3</TD><TD>4</TD><TD>5</TD><TD>6</TD><TD>7</TD></TR><TR><TD>Lambda/HinIII</TD><TD>pUC19 single</TD><TD>pUC19 double</TD><TD>K238015 single</TD><TD>K238013 single</TD></TR><TR><TD><B>Gel 2</B></TD><TD>1.5% B</TD></TR><TR><TD>Well</TD><TD>1</TD><TD>2</TD><TD>3</TD><TD>4</TD><TD>5</TD><TD>6</TD><TD>7</TD></TR><TR><TD>Lambda/HinIII</TD><TD>pUC19 single</TD><TD>pUC19 double</TD></TR><TR><TD><B>Gel 3</B></TD><TD>2%</TD></TR><TR><TD>Well</TD><TD>1</TD><TD>2</TD><TD>3</TD><TD>4</TD><TD>5</TD><TD>6</TD><TD>7</TD></TR><TR><TD>Lambda/HinIII</TD><TD>K238015 double</TD><TD>K238015 no digest</TD></TR><TR><TD><B>Gel 4</B></TD><TD>3%</TD></TR><TR><TD>Well</TD><TD>1</TD><TD>2</TD><TD>3</TD><TD>4</TD><TD>5</TD><TD>6</TD><TD>7</TD></TR><TR><TD>Lambda/HinIII</TD><TD>K238013 single</TD><TD>K238013 double</TD></TR></TBODY></TABLE><H2>July 21</H2><H2><P>Made minipreps of pUC19, K238015, and K238013. The following were calculated from the samples: Plasmid concentration in each sample; Volume of sample required for 1 microgram of DNA. This information determined how the plasmids would be prepared for the single and double digests.</P><TABLE><TBODY><TR><TD>Sample</TD><TD>Concentration (microgram/milliliter)</TD><TD>Volume Required for 1 microgram of DNA</TD></TR><TR><TD>K238015 (4)</TD><TD>0.2</TD><TD>5.0</TD></TR><TR><TD>K238013 (8)</TD><TD>1.7</TD><TD>.59</TD></TR><TR><TD>pUC19 (15)</TD><TD>3.6</TD><TD>.28</TD></TR></TBODY></TABLE><P>Unfortunately the concentration of DNA in each of the samples was found to be low. The digest solutions prepared contained buffer, enzyme and DNA but no water, in order to compensate. These samples were properly incubated, following the digest preparation procedure and then stored.</P><P>Electrophoresis gels were prepared for use in analyzing the digest preps the following day.</P></H2><H2>July 20</H2><P>The same procedure was followed for July 20 and 21. Unfortunately, the yield for July 20 was found to be too low and the data discarded. July 21 accurately outlines the procedure followed and reports accepted data.</P><H2>July 14</H2><P>Quantum dots were resynthesized chemically.</P><H2>July 13</H2><P>The following buffers were prepared today and used for the QIAprep Spin Miniprep Kit Protocol: P1 Buffer containing HCl, EDTA and RNaseA; P2 buffer containing NaOH and SDS; PE Buffer containing HCl and Ethanol.</P><P>To isolate the plasmids from the recultured cells of samples 1 through 12 from July 11, the QIAprep Miniprep protocol was followed.</P><P>The broths not used as part of the QIAprep of samples 1 through 12 were preserved in glycerol stock solution.</P><H2>July 12</H2><P>The chemically synthesized quantum dots prepared last week were tested for light intensity today. A fluorescent spectrometer was first blanked using ligroin solution. The readings for the samples tested are listed in the charts below. The first sample to be tested was Top10. Repeatedly it was tested while adjusting variables, including slit size and scan control, in attempt to find a better intensity peak in the data produced by the spectrometer. However, it was found after testing many samples that the readings were not precise and the expected intensity was not observed. It was concluded the dots were useless and would have to be resynthesized.</P><P>8 of the 12 samples of cultured cells from July 11 lost their labels in the incubation bath. These samples had to be recultured today.</P><H2>July 11</H2><P>The following cells were reconstituted: cells containing parts pSB1C3 and pSB1AK3. These cells along with the cells containing the CD7 protein were incubated in a solution of LB broth and glucose prepared in the lab today.</P><H2>June 14</H2><P>The procedure of June 13 was repeated. The values found again proved useless.</P><H2>June 13</H2><P>The first day of wetlab was used to isolate DNA from bacteria. Heavy Metal Binding Protein and Phytochelatin Synthase were parts present in bacteria newly cultured. The purpose of this lab was to verify the presence of the parts within the cultured bacteria as well as to practice and become more comfortable with lab procedure.</P><P>Plasmid minipreps were prepared using the alkaline lysis procedure. To determine the purity of the DNA isolated, UV spectrophotometeric measurements were taken. Unfortunately the spectrophotometeric measurements which resulted were useless. We had failed in the experiment and planned to repeat it on Day 2.</P></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:Columbia-Cooper/CUNotebook">http://2011.igem.org/Team:Columbia-Cooper/CUNotebook</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Columbia-Cooper/CUNotebook" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Columbia-Cooper/CUNotebook" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Columbia-Cooper/CUNotebook&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Columbia-Cooper/CUNotebook&amp;oldid=252219" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>