<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Utah_State_Modeling"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:Utah_State/Modeling">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Utah_State/Modeling&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Utah_State/Modeling&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Utah_State/Modeling&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Utah_State/Modeling" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Utah State/Modeling</H1><DIV id="bodyContent"><H3 id="siteSub">From 2008.igem.org</H3><TABLE cellpadding="3" cellspacing="5" border="2" width="960px"><TBODY><TR><TH style="text-align:center; background-color:#212223; border-width:0px; padding:2px;"><A href="/Team:Utah_State" title="Team:Utah State"><FONT color="#ffffff">Home</FONT></A></TH><TH style="text-align:center; background-color:#212223; border-width:0px; padding:2px;"><A href="/Team:Utah_State/Team" title="Team:Utah State/Team"><FONT color="#ffffff">The Team</FONT></A></TH><TH style="text-align:center; background-color:#212223; border-width:0px; padding:2px;"><A href="/Team:Utah_State/Project" title="Team:Utah State/Project"><FONT color="#ffffff">The Project</FONT></A></TH><TH style="text-align:center; background-color:#212223; border-width:0px; padding:2px;"><A href="/Team:Utah_State/Parts" title="Team:Utah State/Parts"><FONT color="#ffffff">Parts</FONT></A></TH><TH style="text-align:center; background-color:#212223; border-width:0px; padding:2px;"><A href="/Team:Utah_State/Notebook" title="Team:Utah State/Notebook"><FONT color="#ffffff">Notebook</FONT></A></TH><TH style="text-align:center; background-color:#212223; border-width:0px; padding:2px;"><A href="/Team:Utah_State/Protocols" title="Team:Utah State/Protocols"><FONT color="#99CCFF">Protocols</FONT></A></TH><TH style="text-align:center; background-color:#212223; border-width:0px; padding:2px;"><A href="/Team:Utah_State/Links" title="Team:Utah State/Links"><FONT color="#ffffff">Links</FONT></A></TH></TR></TBODY></TABLE><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Bacterial_Transformation"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Bacterial Transformation</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Streak_Plates_and_Liquid_Cultures_from_Transformed_Colonies"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Streak Plates and Liquid Cultures from Transformed Colonies</SPAN></A></LI><LI class="toclevel-1 tocsection-3"><A href="#Preparation_for_DNA_Separation"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Preparation for DNA Separation</SPAN></A></LI><LI class="toclevel-1 tocsection-4"><A href="#Restriction_Enzyme_Digestion_and_Electrophoresis"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Restriction Enzyme Digestion and Electrophoresis</SPAN></A></LI><LI class="toclevel-1 tocsection-5"><A href="#Media_Preparation"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Media Preparation</SPAN></A></LI><LI class="toclevel-1 tocsection-6"><A href="#1X_TAE_Preparation"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">1X TAE Preparation</SPAN></A></LI><LI class="toclevel-1 tocsection-7"><A href="#Polymerase_Chain_Reaction_.28PCR.29"><SPAN class="tocnumber">7</SPAN><SPAN class="toctext">Polymerase Chain Reaction (PCR)</SPAN></A></LI><LI class="toclevel-1 tocsection-8"><A href="#Ligation"><SPAN class="tocnumber">8</SPAN><SPAN class="toctext">Ligation</SPAN></A></LI></UL></TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Bacterial_Transformation"> Bacterial Transformation </SPAN></H3><P>Once the target DNA has been successfully ligated into the plasmid vector, the plasmid must be transferred into the host cell for replication and cloning. In order to do this, the bacterial cells must first be made “competent.” The term “competent” is to describe a cell state in which there exist gaps or openings in the cell wall which will allow the plasmid containing the target genes to enter into the cell. Several methods to make bacterial cells competent exist, such as the calcium chloride method and electroporation. Competent cells may also be purchased commercially. The team at USU has purchased competent cells for all experiments. The following is the method used by the USU team to insert the plasmids containing the PHB biobricks into the cells. 
<I>Trent Mortensen is shown in the right image inspecting agar plates containing transforming cells</I></P><P><B>Method</B>
1. Ensure the necessary antibiotic agar plates have been prepared or begin their preparation now.  Four plates per transformation will be necessary (two today, then two tomorrow for streaking).  Also ensure that 10 ml liquid media is made up per transformation (also for tomorrow).
2. Clean punchout tool by dipping in 10% chlorox, deionized water, deionized water again, then 80%aq ethanol.  Let dry for 2 minutes and repeat cleaning procedure between punchouts. 
3. Punch out gene of choice with a twisting motion, allowing the metal to cut the paper.  Use the center part of the punchout tool to dislodge the paper into a 2.5 ml microcentrifuge tube.  
4. Add 5 μl TE buffer, place in a 42˚C water bath, and allow plasmids in the paper to elute for 20 minutes.
5. Centrifuge tube(s) at 15K RPM for 3 minutes.  Remove SOC media from the -20˚C freezer and leave out to thaw.
6. Take competent cells from the -80˚C freezer and place on an ice bath.
7. Pipette contents of tube up and down a few times then take 2 μl of the DNA solution and add to the competent cells.  Ensure the pipetting is done directly into the cell solution.  Let cells incubate in the ice bath for 30 minutes.  Heat water bath to 42˚C.
8. Heat shock cells in the 42˚C water bath for 30 seconds.  Remove and place back in the ice bath for 2 minutes.
9. In the hood, add 250 μl SOC media to each tube, bringing the total cell solution to 300 μl.  Incubate at 37˚C for 1 hour.
10. Get out the antibiotic agar plates.  In the hood, add 200 μl of each transformed cell solution to the appropriate antibiotic plate.  Use the Bunsen burner to create a “hockey stick” out of a glass pipette tip by holding over the flame until it bends.  Allow to cool.  Spread cell solution uniformly over the agar plate using the “hockey stick,” then before discarding, spread residual solution on the “stick” over a second plate to get more a more sparse colony distribution.  
11. Parafilm all plates and place in 37˚C incubator 12-14 hours, or overnight if that is not possible.  </P><H3><SPAN class="mw-headline" id="Streak_Plates_and_Liquid_Cultures_from_Transformed_Colonies"> Streak Plates and Liquid Cultures from Transformed Colonies   </SPAN></H3><P>After bacterial cells have been transformed, successfully transformed cells must be selected for. Because 100% of the cells do not receive the desired plasmid and target gene, it is essential to select for cells that do have the target genes. Several methods are used to accomplish this, such as incorporation of antibiotic resistance and also the lac operon. The USU team has used antibiotic resistance to select for successful transformations. To do this, an antibiotic resistance gene is also added to the plasmid vector that contains the target genes. By doing so, it is possible to know that a cell was successfully transformed based on its ability to grow on an agar plate with antibiotics added. Because the cell is able to grow, the antibiotic resistance gene must be present as well as the target gene. From the agar plates containing the antibiotics, a colony is picked off and transferred into a liquid culture for further analysis and cellular production. The following is the method used by USU to clone the DNA and select for the successful transformation of the PHB biobrick into the cells.
</P><P><B>Method</B>
1.	Prepare two 12 ml tubes per transformation, each with 5 ml media containing the appropriate antibiotic (if felt necessary).  Get out antibiotic agar plates.  Inspect plates from yesterday for colonies.  At least two colonies are preferred, but one will do.  Select two colonies and label them.  
2.	Use a pipette tip to extract half of each colony and inoculate one agar plate per colony.  Using a pipette with a tip, extract the other half of each colony and inoculate one liquid media tube per colony.  Label all tubes and plates and place in the 37˚C incubator until tomorrow morning.     </P><H3><SPAN class="mw-headline" id="Preparation_for_DNA_Separation"> Preparation for DNA Separation  </SPAN></H3><P>Following successful bacterial cloning and isolation, it is important to verify that the target gene is in the cell and that the plasmid is functional. To do this, it is a common practice to sequence the DNA plasmid. To obtain enough DNA for sequencing, the bacterial clones are grown in a liquid culture to allow for plasmid replication within the cell and also large amounts of biomass containing these plasmids. The cells are harvested by centrifugation and then prepared for DNA plasmid extraction. DNA plasmid extraction can be done several ways, and the overall purpose is to lyse the cells and separate the plasmid DNA from all other cellular proteins, DNA, and debris. The following is the method used by the USU team to isolate the plasmid DNA containing the PHB biobricks. <I>An Eppendorf Microcentrifuge was used in these experiments and is shown on the right.</I></P><P><B>Method</B>
1.	Prepare two water baths, one boiling and the other 68˚C.    
2.	Centrifuge the 12 ml tubes containing the 5 ml cultures in the large centrifuge at 8K RPM for 20 min.  Discard supernatant.  
3.	Resuspend cells in 200 μl of STET buffer.  
4.	Add 10 μl (if older preparation) Lysozyme (50 mg/ml) and incubate at room temperature for 5 min.   
5.	Boil for 45 sec and centrifuge for 20 min at 13K RPM (or until pellet gets tight). 
6.	Use a pipette tip to remove the pellet. 
7.	Add 5 μl RNase A (10 mg/ml) and incubate at 68˚C for 10 minutes. 
8.	Add 10 μl of 5% CTAB and incubate at room temperature for 3 min. 
9.	Centrifuge for 5 min at 13K RPM, discard supernatant, and resuspend in 300 μl of 1.2 M NaCl using a vortex mixer. 
10.	Add 750 μl of ethanol and centrifuge for 5 min at 13K RPM. 
11.	Discard supernatant, rinse pellet (cannot see) in 80% ethanol, and let tubes dry upside down with caps open.</P><H3><SPAN class="mw-headline" id="Restriction_Enzyme_Digestion_and_Electrophoresis"> Restriction Enzyme Digestion and Electrophoresis </SPAN></H3><P>Restriction enzyme digestion is the process by which a target DNA sequence is separated from the rest of the DNA molecule. Specific knowledge of the target DNA gene is needed to determine which enzyme and conditions to use during the digestion reaction. Once the target DNA sequence is known and the correct enzymes have been selected, the DNA may be digested. Listed below is the procedure used by USU to digest the cellular DNA and obtain the target DNA of PHB synthesis and gene promotion. After enzyme digestion, electrophoresis is used to separate the plasmid DNA from the target DNA genes. A gel is prepared and the respective reaction mixes are loaded into the gel. Using a DNA ladder, and knowing the size of the target DNA gene, the corresponding band can be seen and cut out of the gel. The target DNA may then be removed and isolated from the gel, thus yielding the desired target DNA genes. The DNA from this may then be used in PCR reactions, sequencing, ligations for further experimentation, and many more. Listed below are the protocols used by the USU team to run the electrophoresis reaction.</P><P><B>Method</B>
1. Resuspend DNA in 40 μl water, vortex, and do a brief centrifuge to get solution to the bottom of the tube. 
2. Add components to the digestion solution in the following order:  DNA (23 μl), 10X Tango buffer (3 μl), Xba1 (2 μl), and Pst1 (2 μl).  The volume and restriction enzymes can be varied, but it should be ensured that the total volume is 10X the amount of Tango buffer added.  Tap tubes periodically and allow to digest while preparing electrophoresis gel. 
3. Prepare electrophoresis gel by adding 2 g agarose to 200 ml TAE (1% solution).  This is best done in an Erlenmeyer flask of adequate volume as swirling will need to be done.  Place in the microwave and microwave on high for 20 seconds at a time, pulling it out and swirling until solution is homogeneous again, then repeating (BE CAREFUL to watch the solution closely when swirling – it superheats and can boil over and cause severe burns).  Continue until solution is seen boiling in the microwave then gently swirl again. 
4. Add 20 μl ethidium bromide to solution and swirl until dissolved evenly. 
5. Add 6 μl of 6X loading dye to each tube of digested DNA solution. 
6. Prepare the electrophoresis unit by orienting the basin sideways with rubber gaskets firmly against the side.  Place desired well template in the basin.   
7. When the agarose solution is cool enough to comfortably touch the flask, pour into the basin until the solution is about ¾ of the way to the top of the well template. 
8. When the gel is solidified (should look somewhat cloudy), remove the well template and change basin orientation to have the wells closest to the negative pole (as the DNA will flow towards the positive pole).  Pour 1X TAE buffer into both sides of the electrophoresis unit until it just covers the gel and fills the wells.
9. By inserting the pipette tip below the TAE liquid and into the well, add 10 μl of DNA ladder solution to first (and last if desired) well, skip one well, then begin adding the digested DNA solutions to the wells by adding about 2 μl less than the total volume in the tubes to prevent air bubbles in the wells.
10. Place the cover on the electrophoresis unit, plug into the power source, and turn on voltage to 70 V (this can be as high as 100 V if time is an issue), and press the start button.  Separation should take two to three hours.  The yellow dye shows the location of the smaller nucleotide lengths and the blue dye shows the location of the larger nucleotide lengths.  DNA separation can be observed as time goes on by turning off the power supply then gently removing the basin from the electrophoresis unit (be careful not to let the gel slip out of the basin) and placing on the UV transilluminator to see DNA bands.  The basin can then be placed back in the electrophoresis unit for further separation if desired.  Take care to not have the power supply on without the lid to the unit in place.  
11. When the desired level of separation is attained, the basin can be placed on the transilluminator for picture taking.  Place the cone-shaped cover over the transilluminator and place the digital camera in the top hole for pictures.</P><P><I>The electrophoresis units and UV transilluminator used in this project are shown in the left and center images below. An electrophoresis image is given on the right.</I></P><H3><SPAN class="mw-headline" id="Media_Preparation"> Media Preparation </SPAN></H3><P>For all experimentation involving the need for bacterial biomass and experimentation, proper media is needed to grow the cells. The following is the media composition and methods used by USU to prepare the media.
</P><P>1. Add 5 g yeast extract, 10 g NaCl, 10 g Bacto Tryptone, and 15 g agar (if desired) to a 2 L Erlenmeyer flask and bring the volume up to 1 L with ddH20.  Mix by swirling.  Cover top with foil.
2. Autoclave for 45 minutes (liquid setting, 0 minutes drying time).  It will take an additional half hour for the autoclave to finish cooling then an additional 20 to 30 minutes until the media is cool enough to pour.</P><H3><SPAN class="mw-headline" id="1X_TAE_Preparation"> 1X TAE Preparation </SPAN></H3><P>1. Add 40 ml 50X TAE solution to a 2 L flask and bring level up to 2 L with ddH20.  </P><H3><SPAN class="mw-headline" id="Polymerase_Chain_Reaction_.28PCR.29"> Polymerase Chain Reaction (PCR) </SPAN></H3><P>When large amounts of DNA are needed it is necessary to do a PCR reaction. When a PCR reaction is set up and operated properly it will amplify a single target sequence of DNA. The reaction is first set up by designing primers to use that will bind only to the desired regions of the DNA sequence to be replicates and choosing a polymerase to copy the DNA. Once the primer and polymerase has been chosen, the reaction conditions of time and temperature must be optimized. Thus, when the reaction works properly only the target DNA will be amplified into large quantities that may then be isolated and used for further experimentation. The following is the procedure used by USU for PCR reactions to amplify the PHB synthesis and promotion genes.<I>An Eppendorf Thermocycle was used in these experiments, as shown in the image on the right.</I></P><P><B>Method</B>
1. Obtain the following reagents from the freezer:  DNA template (cells or DNA), 10X Taq buffer (+KCl, -Mg/Cl<SMALL>2</SMALL>), MgCl<SMALL>2</SMALL>, 10 mM dNTP Mix, Taq polymerase (take out of freezer only immediately when needed and put back), and sterile distilled H<SMALL>2</SMALL>O.  Place all reagents on ice.  Also obtain PCR (either 0.2 or 0.5 ml) tubes.
2.	Add the following reagents to a tube (50 μl reaction) in the following volumes and order:
•	32 μl sterile H<SMALL>2</SMALL>O,
•	5 μl 10X buffer, 
•	2 μl dNTP Mix, 
•	6 μl MgCl<SMALL>2</SMALL>
•	3 μl cells/DNA, 
•	0.25 μl Taq Polymerase
•	1 μl primer 1
•	1 μl primer 2
MgCl<SMALL>2</SMALL> volume can be varied (lower to increase specificity – just ensure total volume is 50 ul with H<SMALL>2</SMALL>O).  If many reactions are to be constructed, a master mix can be made up to cut down on time and pipette tip usage (if this is done, ensure primers are added to the appropriate reaction, i.e. perhaps not to the master mix).  Tap or vortex tubes and take to the thermocyler.  Place all reagents back in the -20˚C freezer.  
3. Choose thermocycler temperatures.  The Eppendorf Mastercycler will cycle between three temperatures:  typical temperatures are 94˚C for denaturing, 50-60˚C for primer annealing, and 72˚C for polymerase extending.  Lowering the annealing temperature decreases DNA specificity; 55˚C is a good temperature to begin if no trials have been made with the sample.  
4. Turn on thermocycler with the switch in the back of the unit and open the lid.  The placement of the tubes depends on the size of the tube (0.2 or 0.5 ml) and whether or not a temperature gradient is to be used. </P><UL><LI> If no gradient will be used, tubes can be placed anywhere on the unit in the appropriately-sized hole. Select “Files” and press enter.  Select “Load” and then “Standard.”  If cells will be used in the reaction, include a 1-minute lysing step at the beginning (step 1); this will be followed by a 1-minute DNA denaturing step (step 2).  If purified DNA will be used, set step 1 to 1 second.   Set an annealing temperature for step 3.  Ensure the lid temperature is 105˚C and the extending temperature is 72˚C.  Press exit.  If prompted to save, save by pressing enter three times.  Press exit to return to the main menu.  Choose “Start” on the main menu and select “Standard.”  The program should begin.  </LI></UL><UL><LI> If a gradient is to be used, temperature will vary according to column.  A 20˚C range is the maximum range that can be used (+/- 10˚C).  The range is made by setting a temperature for the middle column and then setting a +/- range.  To see what the temperatures will be if a gradient is used, select “OPTIONS” on the main menu, then select “Gradient.”  Select the size tube that is being used by pressing “Sel,” then press enter.  Choose a temperature for the center column, press enter, then select a +/- range and press enter.  The column number along with the corresponding temperature is shown.  Decide tube placement based on this information.  Press exit twice to return on the main menu.  Select “Files” then “Load,” then “Gradient.”  If cells are being used, set the cell lysing step (step 1) to 1 minute (1:00); if purified DNA is being used, set this time to 1 second (0:01).  Step 2 should be 94C, Step 4 should be 72˚C, and the lid temperature should be 105˚C.  Go to step 3 and set an annealing temperature for the center column.  Leave the next two lines as they are, and change the gradient setting (“G”) to the +/- the center temperature amount.  Press exit.  If prompted to save, press enter three times; if not prompted to save, press enter once.  Press exit to get back to the main menu.  To begin cycle, select “Start,” then select “Gradient.”  The program should begin.  </LI></UL><P>5. The thermocycler is set to store the completed reaction tubes at 4˚C when finished. </P><H3><SPAN class="mw-headline" id="Ligation"> Ligation </SPAN></H3><P>Ligation is the process by which the insert (target DNA gene) is inserted into a plasmid. Both the plasmid and insert have been digested and have the proper “sticky” or blunt ends which are compatible for joining the two DNA pieces together into one molecule. These two DNA pieces are placed in a reaction tube and the proper DNA ligase, buffer, and cofactors are added for the reaction to take place. When done properly, the ligation will result in a successful combination of the insert and plasmid into one plasmid. This newly formed plasmid may then be isolated using gel electrophoresis and then used for bacterial transformation or other experimentation. The following is the procedure used by USU to ligate PHB genes into the biobrick plasmids. <B>Method</B>
1. Obtain the following reagents, some of which are in the -20˚C freezer:  DNA vector, DNA insert, 10X ligation buffer, T4 DNA ligase (take out only when needed, then return immediately to freezer), and sterile distilled water.
2. Ideally, it is desirable to have the concentration of insert ends (or moles of insert) be two to three times the concentration of vector ends (or moles of vector), with a total DNA concentration of 50-400 ng/μl in the reaction.  If determining the DNA concentration is not possible, place two to three times the volume of vector as the volume of insert in the reaction.  As this is often the case, place the following reagents in a thin-walled PCR tube in the following volumes:</P><P>•	10 μl insert DNA
•	3 μl vector DNA
•	2 μl 10X ligation buffer
•	4 μl H2O
•	1 μl T4 DNA ligase
= <I>20 μl total</I></P><P>This could also be done in different volumes depending on DNA concentration/total volume desired.
3. Gently mix the tube, and place the tube in the PCR thermocyler, turn on the machine, select “Start,” from the main menu, select “22” and press “Start.”  The thermocycler will keep the reaction at 22˚C.
4. Incubate for 60 minutes.  Heat-inactivate by placing tubes in 68C water bath for 10 minutes.  Place in the freezer if storing for later use.  </P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2008.igem.org/Team:Utah_State/Modeling">http://2008.igem.org/Team:Utah_State/Modeling</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Utah_State/Modeling" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Utah_State/Modeling" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Utah_State/Modeling&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Utah_State/Modeling&amp;oldid=91678" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2008.igem.org:Privacy_policy" title="2008.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2008.igem.org:General_disclaimer" title="2008.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>