<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_William_and_Mary_Binding_Array skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:William and Mary/Binding Array</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container"><DIV class="navbar-header"><BUTTON id="menu-toggle" type="button" class="navbar-toggle" data-toggle="collapse" data-target="#example"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON><A href="https://2016.igem.org/Team:William_and_Mary" class="navbar-brand">
				W&amp;M
				</A></DIV><DIV class="collapse navbar-collapse"><UL class="nav navbar-nav navbar-right navbar-uppercase"><LI><A href="https://2016.igem.org/Team:William_and_Mary"> 
						Overview
						</A></LI><LI><A href="https://2016.igem.org/Team:William_and_Mary/Description"> 
						Description
						</A></LI><LI><A href="https://2016.igem.org/Team:William_and_Mary/Parts">
						Parts
						</A></LI><LI><A href="https://2016.igem.org/Team:William_and_Mary/Results"> 
						Results
						</A></LI><LI><A href="https://2016.igem.org/Team:William_and_Mary/Model"> 
						Modeling
						</A></LI><LI><A href="https://2016.igem.org/Team:William_and_Mary/Collaborations"> 
						Collaboration
						</A></LI><LI><A href="https://2016.igem.org/Team:William_and_Mary/Engagement"> 
						Outreach
						</A></LI><LI><A href="https://2016.igem.org/Team:William_and_Mary/Team"> 
						Team
						</A></LI></UL></DIV></DIV><DIV class="wrapper"><DIV class="section section-numbers-2" id="numbers2" style="padding-top: 0px !important; border-top: 0px !important; margin-top: 0px !important;"><DIV class="container"><DIV class="col-md-8 col-md-offset-2 text-center"><DIV class="description"><DIV class="title"><H2 style="padding-top: 40px; font-family: &quot;Verlag-Book&quot;; font-size: 50px;">
								Binding Array
								</H2></DIV></DIV></DIV></DIV></DIV><DIV class="section section-we-are-2"><DIV class="text-area"><DIV class="container"><DIV class="row"><DIV class="description"><P class="large">
								Past iGEM teams have managed to create a dizzying number of unique transfer functions from their genetic circuits, and while altering the maximal expression level through ribosome binding sites (RBS) HYPERLINK is common among the iGEM community, there is oftentimes a need to change the threshold of a transfer function. That is, to move a transfer function horizontally. For the Circuit Control Toolbox to reach the stated goal of having orthogonal control over transfer functions we needed to find a way to orthogonally shift the threshold of a given arbitrary genetic circuit. In the end we managed to accomplish this by using molecular titration. 
								</P><P class="large">
								Molecular titration as the name implies is the process of titrating out molecules of transcription factor. That means, for some amount of transcription factor, a constant amount is taken away, such that for any given amount of transcription factor concentration, we are actually working with functionally less of said transcription factor (Figure 1).
								</P><P style="text-align: center">Figure 1: Diagram showing the interactions between an activator transcription factor and decoy binding array, which as a molecular titrator. Note that the number of decoy binding sites impacts the equilibrium of free transcription factor, which in turn impacts the equilibrium of the amount bound to the promoter. Diagram adapted from Lee et al. 2012 (“A regulatory role for repeated decoy transcription factor binding sites in target gene expression”) </P><P class="large">To accomplish this shift in E. coli we used decoy binding arrays, which are plasmids containing many repeated sequences of transcription factor binding sites, these repeated binding sites cause a large number of transcription factors to be bound to sites which produce no product, thus titrating them out, see Brewster et al. 2014 (“The Transcription Factor Titration Effect Dictates Levels of Gene Expression”). This causes a rightward shift in graph of transcription factor vs gene product. If we then graph a that same gene product versus a small molecule inducer for said transcription factor, then depending on the type of transcription factor we will either get a shift to the right (activator) or the the left (repressor). (See in depth example)</P><DIV class="spoiler"><DIV><P class="large" style="padding-bottom: 50px;">
	Say you have a transcription factor TetR, which binds to some promoter, say pTet, which effects the production of some gene, GFP then the concentration of that repressor is related to the production of that gene product. That is to say, for our circuit that you can go from [TetR] -&gt; [GFP] or in a more measurable sense [TetR] -&gt; Fluorescence <B>(Figure 1.5)</B></P><P style="text-align: center">Figure 1.5: Example circuit. pTet GFP is repressed by TetR, and this relationship is repressed by the introduction of the small molecule inducer aTC. Diagram made with pidgeon cad.</P><P class="large">This makes sense, because the chance of the transcription factor being bound to a given promoter is based on the number of transcription factors as well as the number of unbound promoters. Since the number of promoters is more or less held constant by plasmid number, you can model production of the gene under the control of the promoter off of [Transcription Factor] alone. So you can use molecular titration to shift your transfer function by making any given amount of transcription factor be equivalent to a smaller amount.</P><P class="large">Coming back to our example, since TetR is a repressor, as we increase [TetR] we get a corresponding decrease in fluorescence. If we then introduce a decoy binding array encoding a repeated number of tetO (TetR binding sites), say 85 of them, then for some [TetR], we actually have a working [TetR] of some amount less (Figure 2) That means at a given [TetR] in our circuit with the decoy binding array we have the fluorescence of a different lesser [TetR] of our original circuit. This is a shift along the x axis to the right. </P><P style="text-align: center">Figure 2: Schematic illustration of the effect of molecular titration on a transfer function. You can see in the original circuit that once TetR reaches a high enough concentration, the fluorescence (GFP production) goes down, until it reaches a point of maximal repression. In the circuit with the binding array, each [TetR] actually corresponds to a lower working [TetR], shifting the transfer function to the right. </P><P class="large">We can also think about the molecular titration from an induction level. Suppose we use the same reporter circuit pTet GFP and instead of having a variable amount of TetR we instead constitutively express it such that we have a constant amount. Since we only ever have one concentration of TetR, the amount of fluorescence will always be the same. However, when we induce with aTC (which causes TetR to be unable to bind), then there will be functionally less TetR, and thus a higher amount of fluorescence (Figure 3). If we add a decoy binding array, each amount of working TetR remaining, will be functionally equivalent to less due to molecular titration.</P><P style="text-align: center">Figure 3: Schematic illustration of an induction curve of a circuit repressed by TetR. Note that until a certain threshold of aTC is reached, there is so much TetR available to bind that very little GFP is produced. When the binding array is added, that shifts to the left, as each [TetR] is functionally equivalent to less.</P></DIV></DIV><P class="large" style="padding-top:30px !important;">The ability to tune the threshold of transfer functions embodies the ethos of the Circuit Control Toolbox. Oftentimes a practical iGEM project ends up infeasible as a real world solution due to inability to tune the threshold of sensitivity. For example, a cancer detector whose reporter function is either too sensitive or not sensitive enough. While the binding array portion of the Circuit Control Toolbox is designed to be able to orthogonally slot onto any existing circuit, binding arrays can be used in any circuit regardless of the use of our toolbox. Since a binding array requires only that the circuit it modifies contain a DNA binding transcription factor, binding arrays can be constructed (using the method detailed below) for almost all circuits.</P><DIV class="spoiler"><DIV><P class="large">To test our ability to shift an arbitrary circuit, we first attempted to shift our previous example circuit, a pTet GFP and constitutive expressed TetR. We obtained a plasmid containing 85 TetO repeats off of Addgene from Finney-Manchester et al. (2013) (“Harnessing mutagenic homologous recombination for targeted mutagenesis in vivo by TaGTEAM). We moved the segment containing the repeats to the Biobrick Backbone, and then transformed a reporter circuit containing pTet GFP and TetR (Bba_K2066053) on the high copy plasmid 1A3 either with or without the repeat array on the high copy 1C3 backbone. We induced both circuits with varying concentrations of aTC and then measured fluorescence using flow cytometry, which allowed us to get single cell level resolution (Figure 5)</P><P style="text-align: center">Figure 5: Population level FACs data comparing the relative fluorescence of a pTet GFP and TetR reporter with and without a tetO binding array. While the data is noisy, it is clear that the inflection point of the circuit with the binding array has shifted to the left as expected. Additionally, both circuits experienced a decrease in fluorescence at higher aTC concentrations, which was thought to be a result of ATC causing toxicity at high levels. However, during the course of our modeling we found that in fact this is the expected result of a repressor system.</P></DIV></DIV><P class="large" style="padding-top:30px !important;">While the 85x array will undoubtedly be useful to many teams in the future, one of the strengths of decoy binding arrays is their modularity. Since the magnitude of the shift is determined by the number of binding sites, it is possible to vary the magnitude of the shift by using the same binding array on different copy number plasmids. While this allows for more possible numbers of total binding sites, it is inherently limited by the fact that there are only a limited number of plasmid origins. To get around this problem we decided to submit to the registry a suite of Iterative Capped Assembly (ICA) parts that can be used to assemble a TetO or LacO binding array of any size. These parts were designed based upon the ICA method of assembling repeat sequences from Briggs, et al. 2012 (“Iterative capped assembly: rapid and scalable synthesis of repeat-module DNA such as TAL effectors from individual monomers”). </P><P class="large">Additionally, for the tetO ICA parts we submitted 3 different versions, with either 8,16 or 64 base pair spacers between the tetO monomers, this is because it has been suggested by Amit et al. 2012 (“Building Enhancers from the Ground Up: A Synthetic Biology Approach” that anticooperativity plays a role in the effectivity of DNA protein binding. Anticoopritivity means that if two repressor binding sites are very close to each other, a repressor binding to one can spatially hinder, or even completely prevent a repressor from binding to the other. We thought that enabling another level of tuning to allow for even finer levels of shift magnitude by altering the level of anticooperativity would be a useful tool to include within the binding array section of the toolbox. Finally, to help future teams get started, we created a mathematical model to determine the strength of titration needed to shift their transfer function, as well as the number of repeats required to achieve that titration for a given tetO or LacO array.</P></DIV><P class="large" style="padding-left:70px; padding-bottom: 30px;"><B>1.</B> Brewster, R. C., Weinert, F. M., Garcia, H. G., Song, D., Rydenfelt, M., &amp; Phillips, R. (2014). The transcription factor titration effect dictates level of gene expression. Cell, 156(6), 1312. doi:10.1016/j.cell.2014.02.022 </P><P class="large" style="padding-left:70px; padding-bottom: 30px;"><B>2</B> Briggs, A. W., Rios, X., Chari, R., Yang, L., Zhang, F., Mali, P., &amp; Church, G. M. (2012). Iterative capped assembly: rapid and scalable synthesis of repeat-module DNA such as TAL effectors from individual monomers. Nucleic Acids Research, 40(15), e117. http://doi.org/10.1093/nar/gks624
								</P></DIV></DIV></DIV></DIV></DIV><DIV class="container"><NAV class="pull-left"><UL><LI><A href="" data-scroll="true" data-id="#description">
						Top
					</A></LI><LI><A href="https://2016.igem.org/Team:William_and_Mary/contact">
						Contact
					</A></LI><LI><A href="http://learnsynbio.org">
						learnsynbio
					</A></LI></UL></NAV><DIV class="copyright">
			2016 William &amp; Mary iGEM
		</DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>