<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_William_and_Mary_Part_Collection skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:William and Mary/Part Collection</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container"><DIV class="navbar-header"><BUTTON id="menu-toggle" type="button" class="navbar-toggle" data-toggle="collapse" data-target="#example"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON><A href="https://2016.igem.org/Team:William_and_Mary" class="navbar-brand">
				W&amp;M
				</A></DIV><DIV class="collapse navbar-collapse"><UL class="nav navbar-nav navbar-right navbar-uppercase"><LI><A href="https://2016.igem.org/Team:William_and_Mary"> 
						Overview
						</A></LI><LI><A href="https://2016.igem.org/Team:William_and_Mary/Description"> 
						Description
						</A></LI><LI><A href="https://2016.igem.org/Team:William_and_Mary/Parts">
						Parts
						</A></LI><LI><A href="https://2016.igem.org/Team:William_and_Mary/Results"> 
						Results
						</A></LI><LI><A href="https://2016.igem.org/Team:William_and_Mary/Model"> 
						Modeling
						</A></LI><LI><A href="https://2016.igem.org/Team:William_and_Mary/Collaborations"> 
						Collaboration
						</A></LI><LI><A href="https://2016.igem.org/Team:William_and_Mary/Engagement"> 
						Outreach
						</A></LI><LI><A href="https://2016.igem.org/Team:William_and_Mary/Team"> 
						Team
						</A></LI></UL></DIV></DIV><DIV class="wrapper"><DIV class="section section-numbers-2" id="numbers2" style="padding-top: 0px !important; border-top: 0px !important; margin-top: 0px !important;"><DIV class="container"><DIV class="col-md-8 col-md-offset-2 text-center"><DIV class="description"><DIV class="title"><H2 style="padding-top: 40px; font-family: &quot;Verlag-Book&quot;; font-size: 50px;">
								Part Collection
								</H2></DIV></DIV></DIV></DIV></DIV><DIV class="section section-we-are-2"><DIV class="text-area"><DIV class="container"><DIV class="row"><DIV class="col-md-4"><DIV class="title add-animation-stopped"><P class="h2WM" style="padding-left:70px; padding-top: 0px;">
									The Circuit Control Toolbox
									</P></DIV></DIV><DIV class="col-md-7 col-md-offset-1" style="padding-top: 0px;"><DIV class="description add-animation-stopped animation-1"><P class="large" style="color:#A9A9A9;">
									  	The Circuit Control Toolbox consists of a diverse collection of biological parts that provide precise control over the behavior of any arbitrary genetic circuit. It encompasses a series of components able to achieve a range of modifications; additionally, we provide tools that may be used to measure and characterize the parts within the toolbox itself. The Toolbox contains a total of 118 unique Biobrick parts, many of which have been sequence-confirmed and characterized on multiple BioBrick backbones.
									</P></DIV></DIV><DIV class="description"><P class="large">
										Every part submitted in our Toolbox is flanked by standard Unique Nucleotide Sequence (UNS) segments. These segments, generated by Pam Silver and colleagues at Harvard University, were designed to facilitate insulated assembly of complex and repeat-heavy components without biological interference. They consist of randomized sequences of nucleotides which have been optimized for low hairpin and heterodimer formation frequency; they were further checked to ensure minimal overlap with promoter-like sequences, start codons, and the E. coli genome sequence. By flanking part inserts with distinct UNS sequences, one can achieve secure and efficient synthesis of multiple-gene networks [1].
									</P><P class="large">
										We chose to standardize each part in our Toolbox with an upstream UNS2 and downstream UNS3 sequence. These sequences were chosen based on their compatibility with standard BioBrick enzymes, as well as the BsmBI enzyme used for Iterative Capped Assembly; UNS1 was thus eliminated from consideration, as it contained the recognition site for BsmBI. By surrounding each part by these two UNS sequences, we were able to make all of our parts far more compatible with Gibson Assembly than the standard Biobrick Prefix/Suffix regions. <A href="parts.igem.org/wiki/images/b/bf/T--William_and_Mary--UNSGibson.pdf">UNS-guided Gibson Assembly</A>, using UNS regions directly interior to the Biobrick Prefix and Suffix, provides the advantages of significantly decreased homodimer affinity compared to Gibson Assembly on the Prefix/Suffix regions, while maintaining the 3A assembly capability of the Biobrick standard. 
									</P><P class="large">
										Many of the parts in the Circuit Control Toolbox can be categorized into one of five subsections based on their function. These include the following:
									</P><P class="large" style="padding-left:50px;"><B>RBS Characterization Constructs</B></P><P class="large" style="padding-left:20px; text-indent:20px;"> 	These parts are IPTG-inducible constructs which contain self-cleaving ribozyme RiboJ immediately upstream of the RBS sequence, which insulates the contribution of the RBS to gene expression from contributions from the 5â€™ UTR region. The inclusion of this element provides an unprecedented avenue for standardizing RBS measurements regardless of associated coding sequences. </P></DIV><P class="large" style="padding-left:50px;"><B>Promoter Characterization Constructs</B></P><P class="large" style="padding-left:20px; text-indent:20px;"> 		This category consists of the Anderson Library of promoters driving our B0034 RBS Characterization construct; each promoter construct was further characterized with and without RiboJ insertion immediately upstream of the RBS. The purpose of these constructs is to investigate and characterize the impact of RiboJ on absolute gene expression level.   
 </P></DIV><P class="large" style="padding-left:50px;"><B>Decoy Binding Array Construction Kit</B></P><P class="large" style="padding-left:20px; text-indent:20px;"> 			Here we provide an 85-repeat array of tetO binding sites on the BioBrick backbone. Furthermore, we offer a suite of parts which can be used to construct arrays of arbitrary length using Iterative Capped Assembly. These parts are doubly-flanked by UNS regions (UNS 2, 4, 5, and 3) to facilitate UNS-guided Iterative Capped Assembly [link to protocol page], which can be used to construct arrays which are compatible with the UNS standard.
 
 </P></DIV><P class="large" style="padding-left:70px !important;"><B>Synthetic Enhancer Suite</B></P><P class="large" style="padding-left:20px; text-indent:20px;"> 			This is a collection of UNS-flanked BioBrick standard parts associated with the Synthetic Enhancer project. In addition to our well-characterized 3x tetO Synthetic Enhancer part (BBa_K2066114), we provide additional variants in which we express sfGFP rather than mCherry for stronger readout; furthermore, we include a constitutive tetR expression cassette on the same part so that the synthetic enhancer component can be characterized completely using a single plasmid. In order to facilitate the construction of additional variants of the Synthetic Enhancer, we include intermediate construction parts, such as the functional NRII coding region, on UNS-flanked BioBrick backbone.

 </P></DIV><P class="large" style="padding-left:50px;"><B>UNS-Standardized Repressor/Reporter Constructs
</B></P><P class="large" style="padding-left:20px; text-indent:20px;"> 			This collection of parts provides a useful avenue for testing other parts within the Circuit Control Toolbox; this is accomplished by simulating arbitrary circuit input and being able to visualize the repressor concentration. The collection includes tetR-mCherry and lacI-mCherry fusions under the pBad promoter taken from Daniel et al [2].
  
 </P></DIV><DIV align="center"><H3 style="padding-top: 50px; padding-bottom: 50px;">References</H3></DIV><P class="large" style="padding-left:70px; padding-bottom: 30px;"><B>1.</B> Torella, J. P., Boehm, C. R., Lienert, F., Chen, J., Way, J. C., &amp; Silver, P. A. (2014). Rapid construction of insulated genetic circuits via synthetic sequence-guided isothermal assembly. Nucleic Acids Research,42(1), 681-689.doi:<A>10.1093/nar/gkt860</A></P><P class="large" style="padding-left:70px; padding-bottom: 30px;"><B>2.</B> Daniel, R., Rubens, J. R., Sarpeshkar, R., &amp; Lu, T. K. (2013). Synthetic analog computation in living cells.Nature, 497, 619-623. 
							doi:10.1038/nature12148
							</P></DIV></DIV></DIV></DIV><DIV class="container"><NAV class="pull-left"><UL><LI><A href="" data-scroll="true" data-id="#description">
						Top
					</A></LI><LI><A href="https://2016.igem.org/Team:William_and_Mary/contact">
						Contact
					</A></LI><LI><A href="http://learnsynbio.org">
						learnsynbio
					</A></LI></UL></NAV><DIV class="copyright">
			2016 William &amp; Mary iGEM
		</DIV></DIV></DIV></DIV></BODY></HTML>