<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_William_and_Mary_RiboJ skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:William and Mary/RiboJ</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container"><DIV class="navbar-header"><BUTTON id="menu-toggle" type="button" class="navbar-toggle" data-toggle="collapse" data-target="#example"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON><A href="https://2016.igem.org/Team:William_and_Mary" class="navbar-brand">
				W&amp;M
				</A></DIV><DIV class="collapse navbar-collapse"><UL class="nav navbar-nav navbar-right navbar-uppercase"><LI><A href="https://2016.igem.org/Team:William_and_Mary"> 
						Overview
						</A></LI><LI><A href="https://2016.igem.org/Team:William_and_Mary/Description"> 
						Description
						</A></LI><LI><A href="https://2016.igem.org/Team:William_and_Mary/Parts">
						Parts
						</A></LI><LI><A href="https://2016.igem.org/Team:William_and_Mary/Results"> 
						Results
						</A></LI><LI><A href="https://2016.igem.org/Team:William_and_Mary/Model"> 
						Modeling
						</A></LI><LI><A href="https://2016.igem.org/Team:William_and_Mary/Collaborations"> 
						Collaboration
						</A></LI><LI><A href="https://2016.igem.org/Team:William_and_Mary/Engagement"> 
						Outreach
						</A></LI><LI><A href="https://2016.igem.org/Team:William_and_Mary/Team"> 
						Team
						</A></LI></UL></DIV></DIV><DIV class="wrapper"><DIV class="section section-numbers-2" id="numbers2" style="padding-top: 0px !important; border-top: 0px !important; margin-top: 0px !important;"><DIV class="container"><DIV class="col-md-8 col-md-offset-2 text-center"><DIV class="description"><DIV class="title"><H2 style="padding-top: 40px; font-family: &quot;Verlag-Book&quot;; font-size: 50px;">
								RiboJ
								</H2></DIV></DIV></DIV></DIV></DIV><DIV class="section section-we-are-2"><DIV class="text-area"><DIV class="container"><DIV class="row"><DIV class="description"><P class="large" style="padding-left:40px; text-indent: 50px;">
								In the past many teams have rigorously characterized parts both new and old, so what separates the Circuit Control Toolbox from being just another characterization project? The answer to that is multifold, but mainly consists of the fact that the Circuit Control Toolbox not only rigorously characterizes novel or useful tools to alter transfer functions, but also allows these tools to be generalizable to any circuit, not just to the specific circuit in which they were characterized. 
								</P><P class="large" style="padding-left:40px; text-indent: 50px;">
							A key problem of generalizing the results of characterizations is that the dynamics of gene expression are influenced by the protein coded for. For our toolbox to be able to to be used orthogonally at the end of an arbitrary genetic circuit, we must be able to use the characterization done in fluorescent proteins on any other protein. That is, we have to be able to insulate a circuit from its genetic context.
								</P><P class="large" style="padding-left:40px; text-indent: 50px;">
								Fortunately, Lou et al. from the Voigt lab at MIT discovered a ribozyme which does just that. RiboJ is ribozyme derived from a hammerhead ribozyme, which self cleaves, removing the upstream region, most importantly the 5’ untranslated region <B>(Figure 1)</B>.
 When Lou et al. 2012 (“Ribozyme-based insulator parts buffer synthetic circuits from genetic context”) measured two different variants of GFP (sfGFP and cl-sfGFP fusion), they found that the relative expression for the induction curves was not always the same, likely due to upstream effects. However, when they added RiboJ before the coding region, the relative expression levels for the induction curves collapsed, indicating that the relative expression had been homogenized for a given promoter <B>(Figure 2)</B>. This means that the gene expression dynamics are controlled only by the promoter, and not by the coded for protein, which means as long as the promoter is kept the same and RiboJ is added before all coding regions, our characterizations can be used for different proteins.

								</P><P class="large" style="padding-left:40px; text-indent: 50px; color:#A9A9A9; font-size: .75em !important;"><B>Figure 1:</B> Structure of RiboJ, the loop in uppercase is derived from the sTRSV ribozyme, while the additional loop is in lowercase. The catalytic core of the active ribozyme is circled in red and the smaller loop serves to promote translation by exposing the RBS. Figure adapted from Lou et al. 
								</P><P class="large" style="padding-left:40px; text-indent: 50px; color:#A9A9A9; font-size: .75em !important;"><B>Figure 2:</B> Figure from Lou et al. showing that the relative relationship is preserved when RiboJ is used as an insulator. 
								</P><P class="large" style="padding-left:40px; text-indent: 50px;">
First, we wanted to make sure that RiboJ worked the way Lou et al. had outlined, so we characterized RiboJ in the Biobrick Backbone. We reconstructed some of the characterization parts from Lou et al. in the Biobrick Backbone and then proceeded to characterize the impact of RiboJ <B>(Figure 3)</B></P><P class="large" style="padding-left:40px; text-indent: 50px; color:#A9A9A9; font-size: .75em !important;"><B>Figure 3A</B></P><P class="large" style="padding-left:40px; text-indent: 50px; color:#A9A9A9; font-size: .75em !important;"><B>Figure 3B</B></P><P class="large" style="padding-left:40px; text-indent: 50px; color:#A9A9A9; font-size: .75em !important;"><B>Figure 3A and B:</B> Measurements of plLacO-1 sfGFP and cI-GFP with (B) and without (A) RiboJ. Our findings were consistent with Lou et al. Notice how the relative expression curves collapse with RiboJ compared to the ones without RiboJ which do not. Measurements were done in triplicate using FACS. MEFL was calculated using Spherotech beads. The reporters (Bba_K2066014, Bba_K2066015) were located on a high copy 1C3 plasmid while the repressor LacI (Bba_K2066016) was constitutively expressed on a low copy 3K3 plasmid.

</P><P class="large" style="padding-left:40px; text-indent: 50px;">
While we confirmed the findings of Lou et al. that the relative expression of different proteins under one promoter converges, we also noticed that absolute expression of the same protein is altered by the inclusion of RiboJ. Furthermore, we noticed that the direction of this shift changes based on the promoter used <B>(Fig. 4)</B>.
</P><P class="large" style="padding-left:40px; text-indent: 50px;">
These phenomena led us to initiate a thorough characterization of the impact of RiboJ on the absolute gene expression levels of widely used promoters, including the entire Anderson library of constitutive promoters. We designed a library of promoter characterization constructs with and without RiboJ (Bba_K2066057-K2066108), which consist of the Anderson library as well as additional inducible promoters on a standard superfolder GFP expression cassette. To our knowledge, this will be the first such targeted characterization effort on these promoters in the literature.
</P><P class="large" style="padding-left:40px; text-indent: 50px; color:#A9A9A9; font-size: .75em !important;"><B>Figure 4A</B></P><P class="large" style="padding-left:40px; text-indent: 50px; color:#A9A9A9; font-size: .75em !important;"><B>Figure 4B</B></P><P class="large" style="padding-left:40px; text-indent: 50px; color:#A9A9A9; font-size: .75em !important;"><B>Figure 4A and B:</B> Mean levels of sfGFP expression with and without RiboJ using the IPTG inducible promoters pLacO-1 (A) and pTac (B). Plasmid set up is the same as in Figure 3. For the plLacO-1 reporter, RiboJ increases the absolute expression level, while for the pTac reporter, RiboJ has little effect on the absolute expression level.

</P><P class="large" style="padding-left:40px; text-indent: 50px;">
After validating Lou et al.’s results in the BioBrick standard, we carried out our subsequent <A href="https://2016.igem.org/Team:William_and_Mary/Results"> RBS </A> characterizations with RiboJ insulation to ensure that our results are applicable and useful to other scientists and iGEM teams in general.
</P><DIV align="center"><H3 style="padding-top: 50px; padding-bottom: 50px;">References</H3></DIV><P class="large" style="padding-left:70px; padding-bottom: 30px;"><B>1.</B> C. Lou, B. Stanton, Y.-J. Chen, B. Munsky, C. A. Voigt, Ribozyme-based insulator parts buffer synthetic circuits from
								genetic context. Nat. Biotechnol. 30, 1137 (2012). doi:10.1038/nbt.2401 pmid:23034349
								</P></DIV></DIV></DIV></DIV></DIV><DIV class="container"><NAV class="pull-left"><UL><LI><A href="" data-scroll="true" data-id="#description">
						Top
					</A></LI><LI><A href="https://2016.igem.org/Team:William_and_Mary/contact">
						Contact
					</A></LI><LI><A href="http://learnsynbio.org">
						learnsynbio
					</A></LI></UL></NAV><DIV class="copyright">
			2016 William &amp; Mary iGEM
		</DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>