<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_William_and_Mary_Synthetic_Enhancer skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:William and Mary/Synthetic Enhancer</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container"><DIV class="navbar-header"><BUTTON id="menu-toggle" type="button" class="navbar-toggle" data-toggle="collapse" data-target="#example"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON><A href="https://2016.igem.org/Team:William_and_Mary" class="navbar-brand">
				W&amp;M
				</A></DIV><DIV class="collapse navbar-collapse"><UL class="nav navbar-nav navbar-right navbar-uppercase"><LI><A href="https://2016.igem.org/Team:William_and_Mary"> 
						Overview
						</A></LI><LI><A href="https://2016.igem.org/Team:William_and_Mary/Description"> 
						Description
						</A></LI><LI><A href="https://2016.igem.org/Team:William_and_Mary/Parts">
						Parts
						</A></LI><LI><A href="https://2016.igem.org/Team:William_and_Mary/Results"> 
						Results
						</A></LI><LI><A href="https://2016.igem.org/Team:William_and_Mary/Model"> 
						Modeling
						</A></LI><LI><A href="https://2016.igem.org/Team:William_and_Mary/Collaborations"> 
						Collaboration
						</A></LI><LI><A href="https://2016.igem.org/Team:William_and_Mary/Engagement"> 
						Outreach
						</A></LI><LI><A href="https://2016.igem.org/Team:William_and_Mary/Team"> 
						Team
						</A></LI></UL></DIV></DIV><DIV class="wrapper"><DIV class="section section-numbers-2" id="numbers2" style="padding-top: 0px !important; border-top: 0px !important; margin-top: 0px !important;"><DIV class="container"><DIV class="col-md-8 col-md-offset-2 text-center"><DIV class="description"><DIV class="title"><H2 style="padding-top: 40px; font-family: &quot;Verlag-Book&quot;; font-size: 50px;">
								Synthetic Enhancer
								</H2></DIV></DIV></DIV></DIV></DIV><DIV class="section section-we-are-2"><DIV class="text-area"><DIV class="container"><DIV class="row"><DIV class="description"><DIV align="center"><H3 style="padding-top: 50px; padding-bottom: 50px;">Background</H3></DIV><P class="large" style="padding-top:30px !important;">One of the fundamental constraints of an electrical engineering style way of thinking about genetic circuitry is that electrical circuits often include digital, often binary, states of expression.  This kind of input-output relationship showing two discrete “on” and “off” states is represented by the illustrative gene expression curve in <B>Figure 1</B>. As one can see, although the binary interpretation is appropriate at the low and high extremes of inducer concentration, there is an intermediate region where the continuous, or analog nature of gene expression becomes important.  </P><P class="large" style="text-align: center !important">Figure 1. Illustration of induction curve for generic gene expression</P><P class="large" style="padding-top:30px !important;">A natural way to expand on the advantages of the digital-circuit paradigm while incorporating the non-binary characteristics of gene expression is to develop genes and circuits which can exhibit discrete, multi-step response profiles. These would allow circuits to exhibit not just “off” and “on” states, but a number of intermediate, discrete states to better facilitate both precision of circuit behavior and readout as well as the expansion of the possibilities in the computation that can be performed with genetic circuitry.</P><P class="large" style="padding-top:30px !important;">Often people will use the low and high expression regions to represent the analogous expression levels to binary 0 and 1, respectively.  However, Amit et. al. developed a method by which intermediate levels of gene expression can be obtained in a stable, non-transient manner. To do so, they created a synthetic enhancer suites, schematically shown in <B>Figure 2</B>, consisting of enhancer binding sites, an assembled σ54 promoter, and small cassettes between the enhancer and the promoter, which can house DNA binding proteins in different combinations.  It is the looping of the DNA from the enhancer to the promoter that causes an interaction that allows for transcription of the output.  Binding of repressor proteins, such as TetR, to the addition cassettes between the enhancer and promoter can affect the flexibility of DNA looping and thus make it thermodynamically more difficult for the interaction to take place and thus suppresses transcriptional initiation. This synthetic enhancer suite can not only increase the complexity and sensitivity of a circuit, but also allow for a multimodal response with addition of discrete sets of defined enhancer binding protein binding sites, TetR, that interact independently from the enhancer and promoter.</P><DIV align="center"><H3 style="padding-top: 50px; padding-bottom: 50px;">Mechanism</H3></DIV><P class="large" style="padding-top:30px !important;">The physical interaction between the assembled σ54 promoter complex (glnAp2) and enhancer sites causes the activation of transcription.  Amit et. al. generated a genetic circuit regulated by the availability of NRI binding protein as well as the NRII2302 helper protein, which can phosphorylate the NRI protein and activate it as to allow it to bind to the enhancer region.  Once this happens, the looping (shown in <B>Figure 2</B>) the DNA from the enhancer to the promoter allows for transcription of more NRI (positive feedback) as well as the fluorescent reporter.  When repressor binding sites are placed in the spacer region between the enhancer and promoter, this allows for the regulation of output.  The TetR binding makes the DNA looping harder and more rigid, resulting in less transcription of the fluorescent reporter.  Multiple repressor binding sites can give rise to discrete number of TetR binding to the region at a given time, thus modulating the ability for the DNA to loop, ultimately giving rise to different states of expression depending on the availability of functional apo-repressor protein.</P><P class="large" style="padding-top:30px !important;">NRII endogenously has phosphatase and kinase activity, and thus to control the phosphatase activity, the 3.300LG E.coli strain, which consists of a knocked down version of NRII2302, was used to decouple the circuit from a nitrogen dependent PII signal transduction pathway. </P><P class="large" style="text-align: center !important">Figure 2. Adapted from Amit et. al. a. Cartoon representation of how the synthetic enhancer suite works.  The DNA binding NRI-P hexamer has to bind to the enhancer binding sites and this complex can loop (shown on the right) and kinetically bind with the poised promoter to allow for the transcription of NRI and mCherry.  The NRI production allows for positive feedback so as long as it gets phosphorylated by NRII2302 (expressed by pACT Tet helper plasmid), it can bind to the enhancer sequences and continue expression.  Adding TetR repressor binding sites between the promoter and enhancer regions can weaker the probability of the looping event and reduce the expression of NRI and mCherry at a given bound state.</P><DIV align="center"><H3 style="padding-top: 50px; padding-bottom: 50px;">Concept</H3></DIV><P class="large" style="padding-top:30px !important;">Unlike the traditional model of repression, this circuit does not use the repressor to block or compete for the RNA polymerase binding site, but rather decreases the likelihood for the enhancer to loop and bind to the promoter region.</P><P class="large" style="padding-top:30px !important;">To affect the looping probability, Amit et. al. cloned in TetR repressor binding sites, TetO sites, within the Spacer region (region between the NRI binding sites and enhancer which promotes the looping and activation of transcription).  The binding of TetR causes DNA to become less flexible as to weaken the looping process and decrease the quantity of output depending on how many repressor proteins are bound to the TetO sites at a given time.</P><P class="large" style="padding-top:30px !important;">Amit et. al. used small molecule induction with anhydrous-tetracycline (aTc) to modulate the conversion of a variable induced concentration input into discrete or step like output expressions via discrete number of TetO binding sites between the promoter and enhancer, affecting looping propensity. For example, in Figure 3, Amit et. al. generated a synthetic enhancer suite that has three TetO sites between the promoter and enhancer. Conceptually, these three sites, should allow for four different states of expression: 1. Completely repressed state when there is almost no aTc in the cell to block the TetR from binding and repressing the looping, 2. An intermediate step where there is enough aTc to bind to TetR as to allow for an average of 2 TetRs to be available to bind to the TetO sites and partially repress the expression, 3. A second intermediate step where there is more aTc in the cell than the former situation as to allow for only very few TetR being active and available to bind and repress the circuit and finally 4. Enough aTc to reach the saturated, unrepressed state of the circuit where there most of the aTc is bound to a TetR, making it functionally inactive to bind to TetO and repress the circuit.</P><P class="large" style="text-align: center !important">Figure 3. Graph adapted from Amit et. al., showing the three discrete steps of the transfer function once three TetO binding sites were placed between the promoter and enhancer, affecting the looping and transcriptional activation propensity. There are 4 discrete steps, consecutively shown on the graph as the aTc concentration increases logarithmically as a completely repressed state, two intermediate states, and an unrepressed state.  The data was plotted on the y-axis as the ratio of the fluorescence level measured in the presence of a given aTc concentration divided by the maximum fluorescence level. </P><DIV align="center"><H3 style="padding-top: 50px; padding-bottom: 50px;">Results</H3></DIV><P class="large" style="padding-top:30px !important;">In order to apply the concept of modulating a transfer function by having the ability to create a multimodal dose-response regulatory output function, we first wanted to reproduce the results from the Amit et al. paper.   We managed to get plasmids with different number of TetO binding sites within the looping region and transformed them into the special 3.300LG strain (gift from Orna Atar).  Our main plasmid had three TetR binding sites and was transformed with a pACT Tet helper plasmid, which allows the constitutive expression of LacI, TetR, and NRII2302.  After inducing our system with different concentrations of aTc, we were able to replicate Amit’s step-like graph with four discrete states of expression as shown in <B>Figure 4</B>.</P><P class="large" style="text-align: center !important">Figure 4. Reproduction of Amit et. al. work by WM iGEM 2016.  The 3.300LG strain was transformed with the helper plasmid as well as the synthetic enhancer construct (52S) with three TetO binding sites affecting the looping propensity between the enhancer and promoter region, allowing for the modulation of discrete states of output. The data was plotted on the y-axis as the ratio of the fluorescence level measured in the presence of a given aTc concentration divided by the maximum fluorescence level. The shading represents the standard error of the mean.</P><P class="large" style="padding-top:30px !important;">However, Amit et. al. included a lacO site in front of their reporter, allowing them to turn “on” the synthetic enhancer suite with IPTG.  In order the make the synthetic enhancer more useful for integration into a genetic circuit, we wanted it to be reliant on as few inducers as possible.  Thus, we removed the LacI expression dependent feature from the synthetic enhancer by extracting only the functional coding region and promoter for NRII from the pACT Tet helper plasmid from Amit et al.  Because NRII was part of an operon and thus only had the translational stop codon, but not a transcriptional terminator, we cloned in a B0015 double terminator at the end of the sequence.  We then used Gibson Assembly to put the NRII insert into the UNS flanked Biobrick standard backbone and submitted it into the registry as Bba_K2066112.   Subsequently, we cloned the functional NRII unit onto the same plasmid as our 52S construct (which has the three TetR binding sites) in an attempt to reduce number of plasmids required for transformations and thus keep the metabolic strain on the cell low.</P><P class="large" style="padding-top:30px !important;">In summary, the three modifications WM iGEM 2016 did to the Synthetic Enhancer constructs gifted by Amit et. al. were:
1.We moved the 52S construct with three TetR binding sites onto the BioBrick backbone flanked by the UNS regions for ease of cloning (BBa_ K2066120).
2. We then removed the synthetic enhancers system’s reliance on LacI/IPTG induction to make it more compatible with use in genetic circuits. 
3. Finally, we put the helper NRII expression and the synthetic enhancer suite onto the same plasmid backbone to make it more compatible with the existing genetic circuitry in the cell.</P><P class="large" style="padding-top:30px !important;">After doing these modifications, we cotransformed 3.300LG strain with the novel 52S and NRII plasmid along with a plasmid that could constitutively express TetR (BBa_I739001).  We induced the plasmid with varying concentrations of aTc and characterized the updated synthetic enhancer circuit. We were able to confirm that the multi-step response is preserved though our modifications.</P><P class="large" style="text-align: center !important">Figure 5. 3.300LG cells were transformed with our updated synthetic enhancer construct BBa_K2066114 and a constitutive tetR expression cassette on pSB1A3. Data shows four discrete steps, representing the four different binding states of TetR modulated by the inducer concentration, which affects the looping propensity and thus, the overall quantitative expression of the output.  The data was plotted on the y-axis as the ratio of the fluorescence level measured in the presence of a given aTc concentration divided by the maximum fluorescence level.  The shading represents the standard error of the mean.</P><P class="large" style="padding-top:30px !important;">In addition to validating the results in Amit et al. for the 3x tetO Synthetic Enhancer construct, we also redesigned it for better compatibility for use in genetic circuitry. This provides the first construction and characterization of a synthetic enhancer construct on the BioBrick standard,  which will open a wide range of possibilities for iGEM teams and scientists interested in taking advantage of the rich theory and literature on the design and behavior of multi-step digital circuitry.</P><DIV align="center"><H3 style="padding-top: 50px; padding-bottom: 50px;">Source</H3></DIV><P class="large" style="padding-top:30px !important;">Synthetic enhancer sequences and concept derived from:

Amit, R., Garcia, H. G., Phillips, R. &amp; Fraser, S. E. Building enhancers from the ground up: a synthetic biology approach. <I>Cell</I><B>146</B>, 105–118 (2011).</P></DIV></DIV></DIV></DIV></DIV><DIV class="container"><NAV class="pull-left"><UL><LI><A href="" data-scroll="true" data-id="#description">
						Top
					</A></LI><LI><A href="https://2016.igem.org/Team:William_and_Mary/contact">
						Contact
					</A></LI><LI><A href="http://learnsynbio.org">
						learnsynbio
					</A></LI></UL></NAV><DIV class="copyright">
			2016 William &amp; Mary iGEM
		</DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>