<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_William_and_Mary_RBS skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:William and Mary/RBS</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container"><DIV class="navbar-header"><BUTTON id="menu-toggle" type="button" class="navbar-toggle" data-toggle="collapse" data-target="#example"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON><A href="https://2016.igem.org/Team:William_and_Mary" class="navbar-brand">
				W&amp;M
				</A></DIV><DIV class="collapse navbar-collapse"><UL class="nav navbar-nav navbar-right navbar-uppercase"><LI><A href="https://2016.igem.org/Team:William_and_Mary"> 
						Overview
						</A></LI><LI><A href="https://2016.igem.org/Team:William_and_Mary/Description"> 
						Description
						</A></LI><LI><A href="https://2016.igem.org/Team:William_and_Mary/Parts">
						Parts
						</A></LI><LI><A href="https://2016.igem.org/Team:William_and_Mary/Results"> 
						Results
						</A></LI><LI><A href="https://2016.igem.org/Team:William_and_Mary/Model"> 
						Modeling
						</A></LI><LI><A href="https://2016.igem.org/Team:William_and_Mary/Collaborations"> 
						Collaboration
						</A></LI><LI><A href="https://2016.igem.org/Team:William_and_Mary/Engagement"> 
						Outreach
						</A></LI><LI><A href="https://2016.igem.org/Team:William_and_Mary/Team"> 
						Team
						</A></LI></UL></DIV></DIV><DIV class="wrapper"><DIV class="section section-numbers-2" id="numbers2" style="padding-top: 0px !important; border-top: 0px !important; margin-top: 0px !important;"><DIV class="container"><DIV class="col-md-8 col-md-offset-2 text-center"><DIV class="description"><DIV class="title"><H2 style="padding-top: 40px; font-family: &quot;Verlag-Book&quot;; font-size: 50px;">
								RBS Tuning
								</H2></DIV></DIV></DIV></DIV></DIV><DIV class="section section-we-are-2"><DIV class="text-area"><DIV class="container"><DIV class="row"><DIV class="description"><P class="large" style="padding-left:40px; text-indent: 50px;">
								One of the more familiar ways of tuning a biological circuit is to change the strength of a Ribosome binding site (RBS). 
								Changing the strength of an RBS allows for the alteration of the amplitude of a given transfer function. While this is 
								a familiar concept to most iGEM teams, it takes on a special importance in the context of the Circuit Control Toolbox. 
								This is because while transfer functions are preserved when shifted, the amplitude of those transfer functions is 
								oftentimes reduced due to increased metabolic load. So we characterized a library of variable strength RBSs for use 
								in amplitude tuning in the Circuit Control Toolbox.
								</P><P class="large" style="padding-left:40px; text-indent: 50px;">
								While it would have been possible to construct brand new RBSs for use in the Circuit Control Toolbox, we thought that 
								it would be more useful to characterize existing commonly used RBSs on the registry. So we chose to characterize the 
								Community Collection, which is made up of 8 variable strength RBSs, and is commonly used by most iGEM teams. 
								</P><P class="large" style="padding-left:40px; text-indent: 50px;">
								 For our characterization we chose to emulate the methods of Lou et al. 2012 (“Ribozyme-based insulator parts buffer 
								 synthetic circuits from genetic context”) and used a super folder Green Fluorescent Protein (sfGFP) with RiboJ under 
								 the control of pLacO1 (Bba_K2066014). The use of the IPTG inducible promoter pLacO1 allowed us to characterize the 
								 functions over an induction curve, and the use of RiboJ, a self cleaving ribozyme from Lou et al. 2012 prevented the 5’ 
								 untranslated region of the promoter from effecting the translational efficiency of the mRNA transcript (Figure 1). 
								 That means that unlike past iGEM characterization, our results are generalizable beyond the promoter pLacO1. <B>(See the 
								 RiboJ section for more details)</B></P><P class="large" style="padding-left:40px; text-indent: 50px; color:#A9A9A9; font-size: .75em !important;"><B>Figure 1:</B> RiboJ acts to homogenize translational efficiency between promoters by cleaving the 5’ region upstream from 
								RiboJ. -35 and -10 boxes are shown in bold, and transcription factor binding sites are underlined. Note how there is 
								an operator sequence located within the transcribed region of the pTac promoter (BBa_K864400), and that pSal and pBad 
								include transcribed regions. These untranslated regions can change translational efficiency of the downstream protein, 
								leading to inconsistent expression between different combinations of promoters and coding sequences. Figure modified 
								from figure S1 of Lou et al. 2012. 
								</P><P class="large" style="padding-left:40px; text-indent: 50px;">
								Characterization was done on 07/26/16 using flow cytometry, which means that our data shows single cell resolution of the 
								impact of various RBSs. Characterization was done in BL21 E. coli, which is a standard protein expression strain. A high 
								copy plasmid (1C3) containing one of the pLacO1 RiboJ RBS variants (ex. Bba_K206636) was co-transformed with a low copy 
								plasmid (3K3) which contained a constitutively expressed LacI (Bba_K206616) to repress the promoter. Individual colonies 
								were picked from antibiotic selection plates, and colony PCRed to ensure the correct plasmids were contained. For each RBS 
								variant 3 colonies containing both plasmids were inoculated in M9 and incubated till midlog growth was reached. At that 
								point the cells were induced with various concentrations of IPTG until steady state was reached. Cells were then FACSed 
								and their fluorescence was reported in MEFL calculated using Spherotech calibration beads. <B>(Figure 2)</B>. 
								</P><P class="large" style="padding-left:40px; text-indent: 50px; color:#A9A9A9; font-size: .75em !important;"><B>Figure 2:</B> Average (3 biological replicates) population level fluorescence of RBS library over different induction conditions. 
								Note the wide range of RBS strengths in the library, and that the strongest RBS B0035, is not actually the strongest until 
								induced at a concentration of 1000µM IPTG. This illustrates the importance of evaluating over a transfer function, not just 
								at an arbitrary steady state. 
								</P><P class="large" style="padding-left: 40px; text-indent: 50px; padding-bottom: 50px;">
								All of the individual RBS plots are available on our <A href="https://2016.igem.org/Teams:William_and_Mary/Measurement">measurement</A>
								page, but here are a few examples of individual RBS measurements. 
								</P><P class="large" style="padding-left:40px; text-indent: 50px; color:#A9A9A9; font-size: .75em !important;"><B>Figure 3A</B></P><P class="large" style="padding-left:40px; text-indent: 50px; color:#A9A9A9; font-size: .75em !important;"><B>Figure 3B</B></P><P class="large" style="padding-left:40px; text-indent: 50px; color:#A9A9A9; font-size: .75em !important;"><B>Figure 3A and 3B:</B> Fluorescence measurements of replicate 1 of B0031 (BBa_K2066036) un-induced (A) and induced at [IPTG] 
								100µM (B). Note the relative crispness of peaks, indicating a nearly normal distribution of florescence. The majority 
								of all replicates are similarly unimodel in their distribution. 
								</P><P class="large" style="padding-left:40px; text-indent: 50px; color:#A9A9A9; font-size: .75em !important;"><B>Figure 4A</B></P><P class="large" style="padding-left:40px; text-indent: 50px; color:#A9A9A9; font-size: .75em !important;"><B>Figure 4B</B></P><P class="large" style="padding-left:40px; text-indent: 50px; color:#A9A9A9; font-size: .75em !important;"><B>Figure 4A and 4B:</B> Fluorescence measurements of replicate 3 of B0031 (BBa_K2066036) un-induced (A) and induced [IPTG] 100µM(B). 
								Note the bimodal distribution compared to figure. In this case induction occurs, but a subset of the population is maximally 
								induced at all induction conditions. All induction conditions of number 3 contained this distribution, while other replicates 
								of this biological circuit had a more normal fluorescence distribution, indicating that something unusual, likely the loss of 
								the repressor is going on in this replicate. Of note, this bimodal distribution would not be noticeable without the use of FACs 
								to observe at a single cell level. 
								</P><P class="large" style="padding-left:40px; text-indent: 50px;">
								While teams have measured RBSs in the past <B>(Warsaw 2010)</B>, to our knowledge no team has ever measured the entire community 
								library over a variety of induction conditions. As illustrated by B0035’s variable relative strength at different induction 
								conditions, it is vital to perform characterization at different induction conditions rather than just constitutively. 
								However, our findings have been consistent with the measurements on the registry, and much like Warsaw 2010 we found that 
								B0030 was stronger than reported in previous measurements on the registry. Additionally, measurements on the registry by Kelly 
								and Rubin in 2007 indicate that B0035 is stronger than B0034. Given that their data was recorded from a strong constitutively 
								expressed promoter, their measurement makes sense in the context of our findings that B0035 is a stronger RBS only when expressed 
								at a high level.
								</P><P class="large" style="padding-left:40px; text-indent: 50px;">
								While the community RBSs are designed for and most commonly used for expression in E. Coli, we thought that it would be 
								useful to characterize them in a variety of conditions. So as a collaboration we sent a blinded version of the library to 
								University of Pittsburgh’s iGEM team, who then proceeded to characterize the RBSs in the cell free system S30 <B>(Figure 5)</B>. 
								We also sent a blinded library to the high school team Alverno, who characterized them using Richard Murray’s TX-TL cell free system 
								<B>(Figure 6)</B>. In both cases, the majority of RBSs had relative expressions similar to our characterization. However, Team Alverno 
								characterized B0064 as having a significantly higher relative expression than was found in existing registry characterization as 
								well as our characterization. We talked to UPitt iGEM, and they noted that due to technical concerns they were not confident about 
								the validity of their first set of measurements <B>(Figure 7)</B>. When we looked at the unblinded data from the first set of measurements, 
								we noted that they had a similarly high level of B0064 expression. Future experiments are needed to determine whether that is an 
								artifact of a common procedural error in cell free systems, or is representative of a novel result in a cell free system. 
								</P><P class="large" style="padding-left:40px; text-indent: 50px; color:#A9A9A9; font-size: .75em !important;"><B>Figure 5:</B> Unblinded graph of UPitt iGEM’s measurements of our RBS library in the S30 cell free system. Their data 
								is in line with both existing measurements, as well as our measurements.  
								</P><P class="large" style="padding-left:40px; text-indent: 50px; color:#A9A9A9; font-size: .75em !important;"><B>Figure 6A:</B> Unblinded graph of Team Alverno’s characterization of our RBS library in the TX-TL cell free system.  
								</P><P class="large" style="padding-left:40px; text-indent: 50px; color:#A9A9A9; font-size: .75em !important;"><B>Figure 6B:</B> Time course measurements of Team Alverno’s characterization of our RBS library in the TX-TL cell free system.  
								</P><P class="large" style="padding-left:40px; text-indent: 50px;">
								Our RBS characterization highlights the importance of rigorous characterization, especially in the context of our 
								Circuit Control Toolbox. To be able to use RBSs to tune the amplitude of any given transfer function, you need to be 
								sure that you can apply your characterization to a given situation. For example, using our RBS characterization it 
								becomes clear that it is important to consider both the system you are working in as well as the number of mRNA 
								transcripts created. Someone trying to use B0035 for high expression of a protein under the control of a weak 
								constitutive promoter needs to be aware that B0035 is only the strongest relative RBS when there are a large number 
								of copies of mRNA. Keeping this in mind, it is now possible to use our characterization to tune the amplitude and 
								relative max of various transfer functions. Using RiboJ to remove 5’ untranslated regions, and keeping in mind the 
								various limitations of any set of characterizations, our characterization of this will be of use in the Circuit Control 
								Toolbox, and for teams at large.
								</P><P class="large" style="padding-left:40px; text-indent: 50px; color:#A9A9A9; font-size: .75em !important;"><B>Figure 7:</B> Team Pitt’s first series of measurements  
								</P><DIV align="center"><H3 style="padding-top: 50px; padding-bottom: 50px;">References</H3></DIV><P class="large" style="padding-left:70px; padding-bottom: 30px;"><B>1.</B> C. Lou, B. Stanton, Y.-J. Chen, B. Munsky, C. A. Voigt, Ribozyme-based insulator parts buffer synthetic circuits from
								genetic context. Nat. Biotechnol. 30, 1137 (2012). doi:10.1038/nbt.2401 pmid:23034349
								</P></DIV></DIV></DIV></DIV></DIV><DIV class="container"><NAV class="pull-left"><UL><LI><A href="" data-scroll="true" data-id="#description">
						Top
					</A></LI><LI><A href="https://2016.igem.org/Team:William_and_Mary/contact">
						Contact
					</A></LI><LI><A href="http://learnsynbio.org">
						learnsynbio
					</A></LI></UL></NAV><DIV class="copyright">
			2016 William &amp; Mary iGEM
		</DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>