"
Page
Discussion
View source
History
teams
Log in
 
Team:UTP-Software/SoftwareTool
From 2012.igem.org
Home
Team & Attributions
Project
S2MT
Tutorial
Biosinergia
Notebook
Human Practices
Safety
Sponsors
Project Details
What is the S2MT tool?
The SynBio Sequence Mutation Tool (S2MT) it's a MATLAB implementation software that helps teams to start their projects once they have selected some possible BioBricks or DNA sequences to work with.
This tool was developed in MATLAB basically for the reason that, what new iGEM teams require is a flexible environment capable of not only running the algorithm that we develop, but also a place where they can continue working their ideas, examine more in detail their sequences and even continue the development of our tool.  So we thing that our system will also knock down the barriers that exist between computation and biology and give students the tools required to do better simulations, and save a lot of valuable time!
Behind the code there are three simple steps in which the program consists:
First, the user introduces the DNA sequence or link to the BioBrick in the part registry. And the program scans the desired sequence to look for the restriction sites within the assembly standards that are used in the iGEM competition.
Then the user is prompt to specify in case of incompatibility with the standards, which RFC he would like to fix. Once selected, the program designs the primers required for the Site directed mutagenesis procedure that will allow the sequence to be compatible with the standard.
And last, if the program successfully created the required primer, it will print the sequence, giving the user the start position where that primer should be applied.
In some cases, it’s impossible to fix some sequences, because based on the protocol, the output primer may not comply with the steps that are listed in the “Protocol Followed to design the primers” section. Meaning that another standard should be selected.
Flow diagram for the S2MT - Primer Design Section
Protocol Followed to design the primers: [1]
Both the mutagenic primers must contain the desired mutation and anneal to the same sequence on opposite strands of the plasmid.
Primers should be between 25 and 45 bases in length, and the melting temperature (Tm) of the primers should be greater than or equal to 78°C. The following formula is commonly used for estimating the Tm of primers:
Tm = 81.5 + 0.41(%GC) − 675 / N - where N is the primer length in base pairs.
The desired mutation (deletion or insertion) should be in the middle of the primer with ~10–15 bases of correct sequence on both sides.
The primers optimally should have a minimum GC content of 40% and should terminate in one or more C or G bases.
Primers need not be 5´ phosphorylated but must be purified either by fast polynucleotide liquid chromatography (FPLC) or by polyacrylamide gel electrophoresis (PAGE). Failure to purify the primers results in a significant decrease in mutation efficiency.
It is important to keep primer concentration in excess. Stratagene suggests varying the amount of template while keeping the concentration of the primer constantly in excess.
Next S2MT Version
After finishing the first version, we had in mind several things to implement to the next:
Our next version will for sure incorporate a "ab1" chromatogram file reader, because right now we are relying on phred files, which eventhoug phred scores are pretty accurate, not every sequencing company provide those. Some bugs are still to be fixed, and some performance improvement can also be a part of the next version.
Our Second idea (still under development) is about adding a whole new tool to our software, and thats what we call BioSinergia (spanish for "BioSinergy"), which will be like a smart database of metabolic rutes that is going to help teams and researchers to work and study the production of bioenergy through synthetic biology. This project will make use of our S2MT tool to make it easy for new iGEM teams to work with biofuels in their projects.
References
[1]. Stratagene. QuikChange™ Site-Directed Mutagenesis Kit. Catalog #200518. Revision #108005h. Available in: http://web.physics.ucsb.edu/~deborah/pro/pro_pdf/Stratagene%20QuikChange.pdf .
[2]. Prather:Gibson CBA. Available in: http://openwetware.org/wiki/Prather:Gibson_CBA
Retrieved from " http://2012.igem.org/Team:UTP-Software/SoftwareTool "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
