<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_FAU_Erlangen_Tour22 skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:FAU Erlangen/Tour22</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="BackgroundLayout"><DIV class="top bannerShadow"><DIV class="NavbarContainer"><DIV class="HeaderTitelAlt">SCATTER</DIV><DIV class="NavbarContainerInner"><DIV class="MenuButton" id="btnWelcome"><A class="MenuButtonLink" href="https://2015.igem.org/Team:FAU_Erlangen">Welcome</A></DIV><DIV class="MenuButton" id="btnTour"><A class="MenuButtonLink" href="https://2015.igem.org/Team:FAU_Erlangen/Tour10">Interactive wiki-tour</A></DIV><DIV class="MenuButton" id="btnMenu"><A class="MenuButtonLink" href="#" id="btnMenu2">Menu</A><DIV class="menuPopup"><DIV class="menuPopupContent"><DIV class="menuPopupContentInner bannerShadow"><DIV class="menuIntro"><DIV class="menuItems"><UL><LI class="header"><A href="https://2015.igem.org/Team:FAU_Erlangen/Tour10">Introduction</A></LI><LI><A href="https://2015.igem.org/Team:FAU_Erlangen/Tour11">The team</A></LI><LI><A href="https://2015.igem.org/Team:FAU_Erlangen/Tour12">Gallery</A></LI><LI><A href="https://2015.igem.org/Team:FAU_Erlangen/Tour13">Attributions</A></LI><LI><A href="https://2015.igem.org/Team:FAU_Erlangen/Tour14">Collaborations</A></LI></UL><UL><LI class="header"><A href="https://2015.igem.org/Team:FAU_Erlangen/Tour20">Planning</A></LI><LI><A href="https://2015.igem.org/Team:FAU_Erlangen/Tour21">Spy Tag and Spy Catcher</A></LI><LI><A href="https://2015.igem.org/Team:FAU_Erlangen/Tour22">Deacetylase</A></LI><LI><A href="https://2015.igem.org/Team:FAU_Erlangen/Tour23">TALE</A></LI><LI><A href="https://2015.igem.org/Team:FAU_Erlangen/Tour24">Sequences</A></LI><LI><A href="https://2015.igem.org/Team:FAU_Erlangen/Tour25">Matching Program</A></LI><LI><A href="https://2015.igem.org/Team:FAU_Erlangen/Tour26">Vectors</A></LI><LI><A href="https://2015.igem.org/Team:FAU_Erlangen/Tour27">Yeast strains</A></LI><LI><A href="https://2015.igem.org/Team:FAU_Erlangen/Tour28">Methodology</A></LI></UL><UL><LI class="header"><A href="https://2015.igem.org/Team:FAU_Erlangen/Tour30">Execution</A></LI><LI><A href="https://2015.igem.org/Team:FAU_Erlangen/Tour31">Timetable</A></LI><LI><A href="https://2015.igem.org/Team:FAU_Erlangen/Tour32">Lab book</A></LI><LI><A href="https://2015.igem.org/Team:FAU_Erlangen/Tour33">Protocols</A></LI><LI><A href="https://2015.igem.org/Team:FAU_Erlangen/Tour34">Setbacks</A></LI></UL><UL><LI class="header"><A href="https://2015.igem.org/Team:FAU_Erlangen/Tour40">Results</A></LI><LI><A href="https://2015.igem.org/Team:FAU_Erlangen/Tour41">Submitted Parts</A></LI><LI><A href="https://2015.igem.org/Team:FAU_Erlangen/Tour42">YFP Detection</A></LI></UL><UL><LI class="header"><A href="https://2015.igem.org/Team:FAU_Erlangen/Tour50">Future</A></LI><LI><A href="https://2015.igem.org/Team:FAU_Erlangen/Tour51">The end</A></LI></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="tourBoundingBox"><DIV class="tourBoundingBoxInner bannerShadow"><DIV class="parentTourItem "><A class="tourItemLabel" title="Introduction to this years iGEM project">
                Introduction
            </A></DIV><DIV class="expandedTour" style="display:none"><DIV class="expandedTourInner"><DIV class="parentTourItem"><A class="tourItemLabel" title="Get to know the team and contributors">
                        The team
                    </A></DIV><DIV class="parentTourItem"><A class="tourItemLabel" title="Pictures from in and outside the Lab">
                        Gallery
                    </A></DIV><DIV class="parentTourItem"><A class="tourItemLabel" title="Without these, there would have been no team">
                        Attributions
                    </A></DIV><DIV class="parentTourItem"><A class="tourItemLabel" title="Working with other people is always fun">
                        Collaborations
                    </A></DIV></DIV></DIV><DIV class="parentTourItem"><A class="tourItemLabel" title="What we thought we will do">
                Planning
            </A></DIV><DIV class="expandedTour" style="display:none"><DIV class="expandedTourInner"><DIV class="parentTourItem"><A class="tourItemLabel" title="SpyTag and Catcher">
                        Spy-System
       </A></DIV><DIV class="parentTourItem"><A class="tourItemLabel" title="How to silence a gene">
                        Deacetylase
                    </A></DIV><DIV class="parentTourItem"><A class="tourItemLabel" title="The dumb physicist still does not entirely understand what a TALE is">
                        TALE
                    </A></DIV><DIV class="parentTourItem"><A class="tourItemLabel" title="Generating sequences">
                    Sequences
                    </A></DIV><DIV class="parentTourItem"><A class="tourItemLabel" title="Finding out, which sequences can be used at all">
                    Matching program
                    </A></DIV><DIV class="parentTourItem"><A class="tourItemLabel" title="How to get stuff into other stuff">
                    Vectors
                    </A></DIV><DIV class="parentTourItem"><A class="tourItemLabel" title="What stuff we will use">
                    Yeast strains
                    </A></DIV><DIV class="parentTourItem"><A class="tourItemLabel" title="The result - our masterplan">
                    Methodology
                    </A></DIV></DIV></DIV><DIV class="parentTourItem"><A class="tourItemLabel" title="What we actually did">
                Execution
            </A></DIV><DIV class="expandedTour" style="display:none"><DIV class="expandedTourInner"><DIV class="parentTourItem"><A class="tourItemLabel" title="It never works the way you plan it">
                        Timetable
                    </A></DIV><DIV class="parentTourItem"><A class="tourItemLabel" title="All our little secrets in one tiny book">
                        Lab Book
                    </A></DIV><DIV class="parentTourItem"><A class="tourItemLabel" title="A little bit of this and a little bit of that, then just wait and you will see">
                        Protocols
                    </A></DIV><DIV class="parentTourItem"><A class="tourItemLabel" title="Things that did not work">
                        Setbacks
                    </A></DIV></DIV></DIV><DIV class="parentTourItem"><A class="tourItemLabel" title="What we found out">
                Results
            </A></DIV><DIV class="expandedTour" style="display:none"><DIV class="expandedTourInner"><DIV class="parentTourItem"><A class="tourItemLabel" title="We hope it arrived in one piece">
                        Submitted Parts
                    </A></DIV><DIV class="parentTourItem"><A class="tourItemLabel" title="Who doesn't like glowing stuff?">
                        YFP-Detection
                    </A></DIV></DIV></DIV><DIV class="parentTourItem"><A class="tourItemLabel" title="What will have to be done">
                Future
            </A></DIV><DIV class="expandedTour" style="display:none"><DIV class="expandedTourInner"><DIV class="parentTourItem"><A class="tourItemLabel" title="What do you mean, tomorrow is the deadline?">
                        The end
                    </A></DIV></DIV></DIV></DIV></DIV><DIV class="MainLayout"><DIV class="content"><H2>Epigenetics</H2><P align="justify">
Since a Cold Spring Harbor meeting in 2008, epigenetics is defined as a stably heritable phenotype resulting from changes in a chromosome without alterations in the DNA sequence. 
DNA methylations and post-translational histone modifications which alter the expression of genes but not the underlying DNA sequence were introduced. These alterations could be modulated by other modifications, caused by transcriptional or environmental factors and enzymes. 
The executing histone modifying enzymes could be identified as phosphorylases, methyltransferases, acetylases and deacetylases. These enzymes catalyze covalent modifications to the N-terminal histon tails which on their part are connected with transcriptional regulation in yeast and other eukaryotes. 
One example of the influence of these epigenetic alterations is the process of cellular differentiation. During eukaryotic development, stem cells have to become fully differentiated cells, such as neurons, muscle cells and epithelium. This is achieved by activating different sets of genes while inhibiting others. Therefore a variation of different histone modifications and DNA methylations are introduced during the cell cycle. These create different expression patterns and thus cellular differentiation.
However, this system is quite fragile. Minor changes in the epigenetic code induced by mutated histon modifying enzymes, for example, can partially reverse epigenetically repression of genes. Some of these regulate oncogene expression and are therefore known as cancer triggers. Thus, epigenetics has nowadays become a key role in cancer research.
</P><H2>(De-)Acetylation regulates transcription</H2><P align="justify">
As mentioned, methylations, phosphorylations and acetylations can regulate transcription. In the parts ahead we will have a closer look on the acetylation of N-terminal histone tails and their ability of regulating the transcription by altering the DNA structure. </P><P align="justify"> At the moment there are three different models to explain regulation via N-terminal histon tail acetylation. </P><UL><LI><P align="justify"> Acetylation induces a conformational change in the core histones
A perfect correlation between acetylation and transcriptional activity is not existing. But acetylation of charged lysine residues alters the core histone structure.  That possibly results in a facilitated transcription while it is not sufficient to induce transcription of inactive genes.  </P></LI><LI><P align="justify">Acetylation is a transducing signal
Particular acetylation of lysine residues may manipulate signals which are recognized by other factors. Thus the transcription efficiency could be improved or decreased. </P></LI><LI><P align="justify">Acetylation affects the activity of nonhistone protein
not just histones, rather nonhistone proteins are acetylated. An important example of this kind of transcriptional regulation are the high mobility group (HMG) proteins. They are also acetylated at lysine residues. The deacetylation of these proteins could be inhibited by sodium butyrate, which is a common deacetylase inhibitor. Therefore deacetylated HMG proteins could contribute to the effects that are seen with Rpd3 and related proteins. (reviewed by Pazin and Kadonaga, 1997) </P></LI></UL><H2>RPD3, chosing an histone deacetylase</H2><P align="justify">
At the beginning of the project it we found out that the idea to manipulate gene expression via epigenetic modification was not as novel as thought. A literature research aided by our instructors quickly showed that the research group around Kevin Struhl already successfully conducted experiments with the yeast endogene histone deacetylase rpd3. This is why we chose this particular deacetylase for our project.

Rpd3 encodes a histone deacetylase, which deacetylates and acetylases histones. This leads to epigenetic changes thus regulating gene expression on the transcriptional level. Furthermore, it shares 60% sequence identity with the human deacetylase. (Taunton et al 1996) In yeast rpd3 repression is usually achieved by the recruitment of two complexes: Rpd3L and  Rpd3S. However both of them contain as a core element Sin3 and Ume6. (Keogh et al 2005)

Ume 6 binds a a GC-rich sequence upstream of rpd3 controlled promoters. This region is called URS1. In an experiment the Struhl group showed that lacZ expression could be repressed by adding an URS1 up stream of a CYC1 promoter region. This artificial repression was reduced in rpd3, ume6 and sin3 deletion mutants (Kadosh and Struhl 1997). 

This effect could be reproduced with his3 as a reporter gene as well as with different activator regions. However, the effect was less strong with strong activators (Deckert and Struhl 2002).

Additionally it was shown that the deacetylating effect of rpd3 was very local. Deacetylation occurred only 150-250 bp from the target sequence, which is about the distance covered by one histone (Kadosch and Struhl 1998). So target regions were only picked within this range.
</P><CENTER><DL><DD><B>Deckert and Struhl (2002):</B> could show that if an URS segment is positioned upstream of a his3 gene (A), the mRNA-concentration sinks. This effect could not be seen in rpd3, sin3 and ume6. The reduction also was less drastical with stronger activators (B). </DD></DL><H4>References:</H4><UI><LI>Michael J Pazin, James T Kadonaga (1997). What's Up and Down with Histone Deacetylation and Transcription?, Cell, Volume 89 (3), Pages 325-328

</LI><LI>Michael-Christopher Keogh, Siavash K. Kurdistani, Stephanie A. Morris, Seong Hoon Ahn, Vladimir Podolny, Sean R. Collins, Maya Schuldiner, Kayu Chin, Thanuja Punna, Natalie J. Thompson, Charles Boone, Andrew Emili, Jonathan S. Weissman, Timothy R. Hughes, Brian D. Strahl, Michael Grunstein, Jack F. Greenblatt, Stephen Buratowski, Nevan J. Krogan, (2005) Cotranscriptional Set2 Methylation of Histone H3 Lysine 36 Recruits a Repressive Rpd3 Complex, Cell, 124(4),  Pages 593-605

</LI><LI>Taunton, J., Hassig, C.A., and Schreiber, S.L. (1996). A mammalian histone deacetylase related to the yeast transcriptional regulator Rpd3p. Science 272, 408-411

</LI><LI>Kadosch, D. and Struhl, K. (1997) Repression by Ume6 Involves Recruitment of a Complex Conataining Sin3 Corepressor and Rpd3 Histone Deacetylase to Target Promoters, Cell 89, 365-371

</LI><LI>Kadosh, D., &amp; Struhl, K. (1998). Targeted Recruitment of the Sin3-Rpd3 Histone Deacetylase Complex Generates a Highly Localized Domain  of Repressed Chromatin In Vivo. Molecular and Cellular Biology, 18(9), 5121–5127. 

</LI><LI>Deckert, J. and Struhl, K. (2002) Targeted Recruitment of Rpd3 Histone Deacetylase Represses Transcription by Inhibiting Recruitment of Swi/Snf, SAGA and TATA Binding Protein. Molecular and Cellular Biology 22(18), 6458-6470 
</LI></UI></CENTER></DIV><DIV class="footer "><DIV class="footerContainer"><A class="footerBtn footerShadow" id="btnFastForward"><P class="footerBtnPre">Next chapter

</P></A><A class="footerBtn footerShadow" id="btnNextChapter"><P class="footerBtnPre">Next chapter

</P></A><A class="footerBtn footerShadow" id="btnDigDeeper"><P class="footerBtnPre">Dig deeper

</P></A><A class="footerBtn footerShadow" id="btnTheEnd" href="http://www.igem.org" target="_blank"><P class="footerBtnPre">End of tour

</P><P class="footerBtnHeader" style="text-transform: none !important;">Want more? Go to iGEM.org!</P><P class="footerBtnText">

If you want to learn more about Synthetic Biology you should visit iGEM's homepage, which contains loads 

on the subject. You can even visit other teams' wikis and take a look at what they accomplished. Or maybe 

join a team yourself?</P></A><A class="footerBtn footerShadow" id="btnStartTour" href="https://2015.igem.org/Team:FAU_Erlangen/Tour10"><P class="footerBtnPre">A tour for everyone

</P><P class="footerBtnHeader">Click here to begin the interactive wiki-tour</P><P class="footerBtnText">Take a quick glance at our project or dig deeper to get a complete walkthrough of 

the wiki. Or do something in-between, the choice is yours.</P></A></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>