Team:FAU Erlangen/Tour24
SCATTER
Welcome
Interactive wiki-tour
Menu
Introduction
The team
Gallery
Attributions
Collaborations
Planning
Spy Tag and Spy Catcher
Deacetylase
TALE
Sequences
Matching Program
Vectors
Yeast strains
Methodology
Execution
Timetable
Lab book
Protocols
Setbacks
Results
Submitted Parts
YFP Detection
Future
The end
Introduction
The team
Gallery
Attributions
Collaborations
Planning
Spy-System
Deacetylase
TALE
Sequences
Matching program
Vectors
Yeast strains
Methodology
Execution
Timetable
Lab Book
Protocols
Setbacks
Results
Submitted Parts
YFP-Detection
Future
The end
Sequences
We had three core constructs that we used in our project. All of them are meant for yeast expression. As reporter gene we chose eYFP, which we amplified from the 2015 DNA Distribution (BBa_E2030).
We wanted to target the promoter in our reporter and down-regulate the expression. However, we wanted to avoid off-target activity, as we had no way of predicting the effect it would have on our yeast cells. Therefore, we chose a promoter that is not endogenous in yeast. The CMV (Cytomegalovirus) promoter had been used in yeast before and proven to lead to moderate gene expression (Belli et al., 1998). We used a modified version of BBa_K747096. According to the sequence map provided by snapgene , the actual promoter only makes up the last ~200bp, the first 300 being the enhancer. Thus we used only the 200 bp designated as promoter. Upstream we tagged the promoter with a short sequence not present in yeast. For optimal transcription efficiency we added a yeast RBS (BBa_K792001). Our construct was designed in such a way, that the RBS could be exchanged via BamHI and XhoI restriction sites. As terminator we used the yeast ADH1 terminator from the parts registry (BBa_K392003). A promoter::spacer::terminator construct was synthesized by IDT. Our construct is flanked by sites homologous to the His3 locus for insertion in the yeast genome by homologous recombination. However, we soon noticed that we had no way of selecting for genome insertion. Therefore we cloned the entire construct into a yeast insertion plasmid and inserted it in the Trp1 locus.
For our deacetylase we used Rpd3, the sequence was taken from the Saccharomyces Genome Database and modified to suit our needs by fusing it with a SpyCatcher (BBa_K1159200), separated by a short Glycine-Serine linker. Due to the length of our construct it was split in two parts which were synthesized by IDT and subsequently fused by overlap-PCR.
TALE sequences were designed with the TAL Plasmids Sequence Assembly Tool , slightly modified to fit our project (removed activation domains, added NLS), and synthesized by IDT. Due to the Repeats we could not synthesize the constructs in one run and therefore adapted the standard TALE GoldenGate Assembly (Cermak et al., 2011) to our needs. We created a construct containing both constant domains separated by a short spacer with Esp3I restriction sites. The repeat domain was split in half and optimized for synthesis with DNAWorks . Esp3I sites were added for GoldenGate Assembly.  SpyTag (BBa_K1159201) was fused to the construct. We created three constructs targeting different positions in the promoter of our reporter construct.
Both Rpd3::SpyCatcher and TALE::SpyTag constructs were cloned into an expression cassette with an Rpd3 promoter and Adh1 terminator that was synthesized by IDT. The Rpd3 Promoter sequence was found in the Yeast Promoter Atlas .
Restriction sites were removed where necessary by point mutations to ensure BioBrick compatibility.
To express our constructs in yeast we used yeast shuttle vectors YCplac22, YIplac204, YEplac181, and pTUM12.
In silico Sequence assembly was done using Genome Compiler. Sequencing results were analyzed using Genome Compiler as well as the sequence alignment tools on the EMBL-EBI website .
References
Bellí G, Garí E, Piedrafita L, Aldea M and Herrero E. “An Activator/repressor Dual System Allows Tight Tetracycline-Regulated Gene Expression in Budding Yeast.” Nucleic Acids Research 26, no. 4 (February 15, 1998): 942–47.
Cermak, T, Doyle E L, Christian M, Wang L, Zhang Y, Schmidt C, Baller J A, Somia N V, Bogdanove A J, and Voytas D F. “Efficient Design and Assembly of Custom TALEN and Other TAL Effector-Based Constructs for DNA Targeting.” Nucleic Acids Research 39, no. 12 (July 1, 2011): e82–e82. doi:10.1093/nar/gkr218..
Next chapter
Next chapter
Dig deeper
End of tour
Want more? Go to iGEM.org!
If you want to learn more about Synthetic Biology you should visit iGEM's homepage, which contains loads   on the subject. You can even visit other teams' wikis and take a look at what they accomplished. Or maybe   join a team yourself?
A tour for everyone
Click here to begin the interactive wiki-tour
Take a quick glance at our project or dig deeper to get a complete walkthrough of   the wiki. Or do something in-between, the choice is yours.
