<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_FAU_Erlangen_Tour32 skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:FAU Erlangen/Tour32</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="BackgroundLayout"><DIV class="top bannerShadow"><DIV class="NavbarContainer"><DIV class="HeaderTitelAlt">SCATTER</DIV><DIV class="NavbarContainerInner"><DIV class="MenuButton" id="btnWelcome"><A class="MenuButtonLink" href="https://2015.igem.org/Team:FAU_Erlangen">Welcome</A></DIV><DIV class="MenuButton" id="btnTour"><A class="MenuButtonLink" href="https://2015.igem.org/Team:FAU_Erlangen/Tour10">Interactive wiki-tour</A></DIV><DIV class="MenuButton" id="btnMenu"><A class="MenuButtonLink" href="#" id="btnMenu2">Menu</A><DIV class="menuPopup"><DIV class="menuPopupContent"><DIV class="menuPopupContentInner bannerShadow"><DIV class="menuIntro"><DIV class="menuItems"><UL><LI class="header"><A href="https://2015.igem.org/Team:FAU_Erlangen/Tour10">Introduction</A></LI><LI><A href="https://2015.igem.org/Team:FAU_Erlangen/Tour11">The team</A></LI><LI><A href="https://2015.igem.org/Team:FAU_Erlangen/Tour12">Gallery</A></LI><LI><A href="https://2015.igem.org/Team:FAU_Erlangen/Tour13">Attributions</A></LI><LI><A href="https://2015.igem.org/Team:FAU_Erlangen/Tour14">Collaborations</A></LI></UL><UL><LI class="header"><A href="https://2015.igem.org/Team:FAU_Erlangen/Tour20">Planning</A></LI><LI><A href="https://2015.igem.org/Team:FAU_Erlangen/Tour21">Spy Tag and Spy Catcher</A></LI><LI><A href="https://2015.igem.org/Team:FAU_Erlangen/Tour22">Deacetylase</A></LI><LI><A href="https://2015.igem.org/Team:FAU_Erlangen/Tour23">TALE</A></LI><LI><A href="https://2015.igem.org/Team:FAU_Erlangen/Tour24">Sequences</A></LI><LI><A href="https://2015.igem.org/Team:FAU_Erlangen/Tour25">Matching Program</A></LI><LI><A href="https://2015.igem.org/Team:FAU_Erlangen/Tour26">Vectors</A></LI><LI><A href="https://2015.igem.org/Team:FAU_Erlangen/Tour27">Yeast strains</A></LI><LI><A href="https://2015.igem.org/Team:FAU_Erlangen/Tour28">Methodology</A></LI></UL><UL><LI class="header"><A href="https://2015.igem.org/Team:FAU_Erlangen/Tour30">Execution</A></LI><LI><A href="https://2015.igem.org/Team:FAU_Erlangen/Tour31">Timetable</A></LI><LI><A href="https://2015.igem.org/Team:FAU_Erlangen/Tour32">Lab book</A></LI><LI><A href="https://2015.igem.org/Team:FAU_Erlangen/Tour33">Protocols</A></LI><LI><A href="https://2015.igem.org/Team:FAU_Erlangen/Tour34">Setbacks</A></LI></UL><UL><LI class="header"><A href="https://2015.igem.org/Team:FAU_Erlangen/Tour40">Results</A></LI><LI><A href="https://2015.igem.org/Team:FAU_Erlangen/Tour41">Submitted Parts</A></LI><LI><A href="https://2015.igem.org/Team:FAU_Erlangen/Tour42">YFP Detection</A></LI></UL><UL><LI class="header"><A href="https://2015.igem.org/Team:FAU_Erlangen/Tour50">Future</A></LI><LI><A href="https://2015.igem.org/Team:FAU_Erlangen/Tour51">The end</A></LI></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="tourBoundingBox"><DIV class="tourBoundingBoxInner bannerShadow"><DIV class="parentTourItem "><A class="tourItemLabel" title="Introduction to this years iGEM project">
                Introduction
            </A></DIV><DIV class="expandedTour" style="display:none"><DIV class="expandedTourInner"><DIV class="parentTourItem"><A class="tourItemLabel" title="Get to know the team and contributors">
                        The team
                    </A></DIV><DIV class="parentTourItem"><A class="tourItemLabel" title="Pictures from in and outside the Lab">
                        Gallery
                    </A></DIV><DIV class="parentTourItem"><A class="tourItemLabel" title="Without these, there would have been no team">
                        Attributions
                    </A></DIV><DIV class="parentTourItem"><A class="tourItemLabel" title="Working with other people is always fun">
                        Collaborations
                    </A></DIV></DIV></DIV><DIV class="parentTourItem"><A class="tourItemLabel" title="What we thought we will do">
                Planning
            </A></DIV><DIV class="expandedTour" style="display:none"><DIV class="expandedTourInner"><DIV class="parentTourItem"><A class="tourItemLabel" title="SpyTag and Catcher">
                        Spy-System
       </A></DIV><DIV class="parentTourItem"><A class="tourItemLabel" title="How to silence a gene">
                        Deacetylase
                    </A></DIV><DIV class="parentTourItem"><A class="tourItemLabel" title="The dumb physicist still does not entirely understand what a TALE is">
                        TALE
                    </A></DIV><DIV class="parentTourItem"><A class="tourItemLabel" title="Generating sequences">
                    Sequences
                    </A></DIV><DIV class="parentTourItem"><A class="tourItemLabel" title="Finding out, which sequences can be used at all">
                    Matching program
                    </A></DIV><DIV class="parentTourItem"><A class="tourItemLabel" title="How to get stuff into other stuff">
                    Vectors
                    </A></DIV><DIV class="parentTourItem"><A class="tourItemLabel" title="What stuff we will use">
                    Yeast strains
                    </A></DIV><DIV class="parentTourItem"><A class="tourItemLabel" title="The result - our masterplan">
                    Methodology
                    </A></DIV></DIV></DIV><DIV class="parentTourItem"><A class="tourItemLabel" title="What we actually did">
                Execution
            </A></DIV><DIV class="expandedTour" style="display:none"><DIV class="expandedTourInner"><DIV class="parentTourItem"><A class="tourItemLabel" title="It never works the way you plan it">
                        Timetable
                    </A></DIV><DIV class="parentTourItem"><A class="tourItemLabel" title="All our little secrets in one tiny book">
                        Lab Book
                    </A></DIV><DIV class="parentTourItem"><A class="tourItemLabel" title="A little bit of this and a little bit of that, then just wait and you will see">
                        Protocols
                    </A></DIV><DIV class="parentTourItem"><A class="tourItemLabel" title="Things that did not work">
                        Setbacks
                    </A></DIV></DIV></DIV><DIV class="parentTourItem"><A class="tourItemLabel" title="What we found out">
                Results
            </A></DIV><DIV class="expandedTour" style="display:none"><DIV class="expandedTourInner"><DIV class="parentTourItem"><A class="tourItemLabel" title="We hope it arrived in one piece">
                        Submitted Parts
                    </A></DIV><DIV class="parentTourItem"><A class="tourItemLabel" title="Who doesn't like glowing stuff?">
                        YFP-Detection
                    </A></DIV></DIV></DIV><DIV class="parentTourItem"><A class="tourItemLabel" title="What will have to be done">
                Future
            </A></DIV><DIV class="expandedTour" style="display:none"><DIV class="expandedTourInner"><DIV class="parentTourItem"><A class="tourItemLabel" title="What do you mean, tomorrow is the deadline?">
                        The end
                    </A></DIV></DIV></DIV></DIV></DIV><DIV class="MainLayout"><DIV class="content"><DIV class="accordion" style="width:650px;"><DIV class="accordionTitel"> Yeast transformation with YFP </DIV><DIV class="pane" style="height:15750px;"><H4>Day 1, 15/06/25:</H4><B>
 1. Cultivating S. cerevisiae strain K699 and BY4742 derivative 4196 out of hostlab's strain collection 
</B><UL><LI> Strains taken from -80°C freezer were  spread out on YPED-Medium plates
</LI><LI> Incubation for 3 days at 26°C
</LI></UL><B>
2. Transformation of Vectors YCplac22, YEplac 181 and YIplac204 in DH5α E. coli 
</B><UL><LI> 1 µl DNA Stocksolution was given onto 50µl bacteria suspension
	</LI><LI> Incubation on ice for 30 minutes
	</LI><LI> Heatshock at 42°C for 90 seconds
	</LI><LI> Incubation on ice for 2 minutes
	</LI><LI> Addition of 500µl SOC-Medium
	</LI><LI> Incubation at 37°C for 60 minutes
	</LI><LI> 100µl of suspension wase plated on agarplates containing ampicilin
	</LI><LI> Incubation at 37° C over night
</LI></UL><H4>
Day 2, 15/06/26:
</H4><B>1. Picking colonies for overnight cultures (ONC)</B><UL><LI> Picking a colony for each vector and additionally from the agarstamp ordered from the registry (K801000)
	</LI><LI> Afterwards incubation overnight at 37°C

</LI></UL><H4> Day 3, 15/06/29:</H4><B> 1. DNA-preparation via alkaline Lysis </B><UL><LI> Experimental procedure according to &quot; Protocol 1: Alkaline Lysis &quot;</LI><LI> Changes to protocol:
		 </LI><LI> No centrifugation of ONC 
		 </LI><LI> Two 2ml reaction tubes filled with ONC instead
		 </LI></UL><B> 2. Picking of yeast colonies </B><UL><LI> Two yeast colonies picked out of YPED-Medium plates from Day 1 (see day 1/1.)
	 </LI><LI> Clone 1 named A
	 </LI><LI> Clone 2 named B
	 </LI></UL><H4>
Day 4, 15/06/30:
</H4><B>
1. Restriction digest of the obtained DNA-Solutions
</B><UL><LI> Mastermix for a control digest of the obtained DNA-Solutions of plasmids YCplac22, YEplac181, YIplac204 produced according to following table
</LI></UL><TABLE><TBODY><TR><TH align="left">Reagent</TH><TH align="left">Volume for one sample</TH><TH align="left">Mastermix for four samples </TH></TR><TR><TD>EcoRI</TD><TD>0.5 µl</TD><TD>2 µl</TD></TR><TR><TD>EcoRV</TD><TD>0.5 µl</TD><TD>2 µl</TD></TR><TR><TD>Tango Buffer</TD><TD>4 µl</TD><TD>16 µl</TD></TR><TR><TD>Add 20µl ddH2O</TD><TD>14 µl</TD><TD>56 µl</TD></TR><TR><TH align="left">Σ</TH><TD>19 µl</TD><TD>76 µl</TD></TR></TBODY></TABLE><UL><LI> 19 µl out of the mastermix were transferred to three 1.5 ml reaction tubes
</LI><LI> 1 µl YCplac22/YEplac181/YIplac204 solution obtained in day 3/1 were given in one mastermix tube each
</LI><LI> 1µl K80100 solution obtained in 3/1 was added to following reagents:
</LI></UL><TABLE><TBODY><TR><TH align="left">Reagent</TH><TH align="left">Volume for one sample</TH></TR><TR><TD> PstI</TD><TD>0.5 µl</TD></TR><TR><TD> Buffer O </TD><TD> 2 µl </TD></TR><TR><TD> Add 20µl ddH2O </TD><TD> 16.5 µl </TD></TR><TR><TH align="left">Σ</TH><TD> 19 µl </TD></TR></TBODY></TABLE><UL><LI> Digests were incubated at 37°C for 3 h

</LI></UL><B> 2. Gelelectrophoresis of digested DNA </B><UL><LI> Gelectrophoresis was executed according to &quot;protocol 2: Gelelectrophoresis &quot;</LI><LI> 2 µl  6x staining solution were given unto 10 µl digest
</LI><LI> Samples were loaded onto the gel according to following scheme
⇒ 1kb-DNA-Marker (6µl)\\YCplac22 undigested\\ YCplac22 digested\\YIplac204 undigested\\YIplac204 digested\\ YEplac181 undigested\\ empty (pocket damaged)\\ YEplac181 digested\\K801000 undigested\\K801000digested\\empty\\1kb DNA-marker (6µl)

</LI><LI> Photos were taken 1 hour, 1 hour 40 minutes and 2 hours 20 minutes
</LI></UL><H4> Day 5, 15/07/01:</H4><B> 1. Repetition of restriction digest for Ycplac 204 and K801000 digested </B><UL><LI>Mastermix created for BBa_K801000 and YIplac204
<TABLE><TBODY><TR><TH align="left">Reagent</TH><TH align="left">Volume for one sample</TH><TH align="left">Mastermix for three samples</TH></TR><TR><TD>Eco RI</TD><TD> 0.5 µl</TD><TD> 1.5 µl</TD></TR><TR><TD> EcoRV </TD><TD> 0.5 µl</TD><TD> 1.5 µl</TD></TR><TR><TD>Tango Buffer</TD><TD>4 µl</TD><TD>12 µl</TD></TR><TR><TD> Add 20µl ddH2O </TD><TD> 13 µl </TD><TD> 39 µl </TD></TR><TR><TH align="left">Σ</TH><TD> 18 µl </TD><TD> 54 µl </TD></TR></TBODY></TABLE></LI><LI> 2µl of obtained DNA-Solution were transfered to 18 µl of mastermix
</LI><LI> Incubation at 37°C for 3 hours
</LI><LI>  Gelelectrophoresis was conducted according to protocol 2
</LI><LI> 2 µl of 6x staining solution were added onto 10 µl of digest
</LI><LI> Samples were loaded in pockets according to following scheme ⇒ 6µl 1kb-DNA-marker// YIplac204 digested (15/07/01) // K801000 digested(15/07/01)//YCplac22 (15/06/30)// K801000(15/06/30)// YIplac204 (15/06/30)//empty//YEplac181 (15/06/30)// empty // 6µl 1kb-DNA-marker
</LI></UL><B> 2. Preparation of S. Cerevisiae tryptophan negative plates </B><UL><LI> Added following substances in two 1 liter flasks:

	 </LI><LI> 3.35g Difco yeast nitrogen base 2/o aminoacid
	 </LI><LI> 5.5 g CAA vitamin assay
	 </LI><LI> 10 g Glucose
	 </LI><LI> 83.0 mg Tyrosin-Uracil-Adenin-mix
	 </LI><LI> 50.5 mg Leucin
	 </LI><LI> 22g Agar
	 </LI></UL><B> 3. Preparation of S. Cerevisiae full media plates </B><UL><LI>Added following substances in two 1 liter flasks

	</LI><LI>     5.3g yeast extract</LI><LI>	 11g Bacopepton</LI><LI>	 10g Glucose</LI><LI>	 22.7 mg Adenin</LI><LI>	 11g Agar</LI></UL><B> 4. Overnightculture of YIplac204 positive bacteria </B><UL><LI> Two times 5 ml ampicilin medium (80µg/ml) inoculated with 2 ampicilline ressistant bacteria colonies picked of plate from day 1/2.
</LI><LI> Incubation at 37°C over night
</LI></UL><H4> Day 6, 15/07/02: </H4><B> 1. Moulding of Agar-plates </B><UL><LI> Added 500ml destilled Water to each Flask of day 5/2. and day 5/3.
</LI><LI> Short mixing with stirring bar
</LI><LI> Flask autoclaved
</LI><LI> Stirring with stirring bar for 20 minutes at room temperature
</LI><LI> Moulding of plates underneath a laminar flow hood
</LI></UL><B> 2. DNA-Preperation of overnight culture (see day 5/4.)</B><UL><LI> Conducted analogous to day 3/1. according to protocol 1
</LI></UL><B> 3. Digest of obtained DNA </B><UL><LI> Mastermix created to digest 2 µl DNA solution <TABLE><TBODY><TR><TH align="left"> Reagent </TH><TH align="left"> Volume for one sample</TH><TH align="left"> Mastermix for three samples</TH></TR><TR><TD> EcoRV </TD><TD> 0.5 µl</TD><TD> 1.5µl </TD></TR><TR><TD> Buffer R</TD><TD> 2.0 µl</TD><TD> 6 µl </TD></TR><TR><TD> Add 20µl H2O </TD><TD> 15.5µl </TD><TD> 46.5µl </TD></TR><TR><TH align="left">Σ</TH><TD> 18µl </TD><TD> 18µl </TD></TR></TBODY></TABLE></LI><LI> 2 µl DNA-preparation of each dublicat of YIp204 were added onto 18µl mastermix
</LI><LI> Incubated for 3 hours at 37°C
</LI><LI> Electrophoresis was conducted according to protocol 2
</LI><LI> Added 4µl 6x staining buffer to each digest after incubation time
</LI><LI> Added 4µl 6x staining buffer to 20µl undigest DNA</LI><LI> Preperation
</LI><LI> Loaded gel according to following scheme
⇒ 6µl 1kb-DNA-marker\\ Prep A digested\\ Prep B digested \\ Prep A undigested \\ Prep B undigested

</LI><LI> DNA-fragments of unknown origin were found
</LI><LI> Repitition of experiment needed
</LI></UL><B> 4. Overnight culture of strains 4196 and K699 for transformation </B><UL><LI>  3 ml YEPD Medium was given to each of 4 reaction tubes
</LI><LI>  Two  yeast colonies of 4196 and K669 were picked
</LI><LI>  Incubation at 30°C over night
</LI></UL><B> 5. Restriction digest of Vector DNA for  the transformation </B><UL><LI> Mix to digest 10 µl YIplac204 DNA obtained in preparation day 3/1. for transformation created
 <TABLE><TBODY><TR><TH align="left"> Reagent </TH><TH align="left"> Volume for one sample </TH></TR><TR><TD> Eco RV </TD><TD>1 µl </TD></TR><TR><TD> Buffer R </TD><TD> 5 µl </TD></TR><TR><TD> Add 20µl ddH2O </TD><TD> 34 µl </TD></TR><TR><TH align="left">Σ</TH><TD> 40 µl </TD></TR></TBODY></TABLE></LI><LI> Incubation over night at 37°C
</LI></UL><B> 6. Overnight culture of YIplac204 </B><UL><LI> 5 ml of ampiciline-medium (80µg/ml) were inoculated with one colony obtained of experiment  day 1/2. 
</LI><LI> Incubation at 37°C overnight
</LI></UL><H4> Day 7, 15/07/03 </H4><B> 1. Transformation of S. cerevisiae strains K699 and 4196 with YCplac22 and Ylplac204 </B><UL><LI> over night culture diluted 1:100 with MQ (10 µl suspension and 990 µl MQ)
</LI><LI> measurement 0D600:
 </LI><LI> 699 A = 0.203
	 </LI><LI> 699 B = 0.143 → used
	 </LI><LI> 7196 A = 0.12
	 </LI><LI> 4196 B = 0.139 → used
	 </LI></UL><LI> two flasks filled with YEPG next to Bunsenburner
	</LI><LI> A: 50 ml
	</LI><LI> B: 25 ml → Media (wrong estimate)
	</LI><LI> in flask A 2 ml suspension 699 oD600 = 0.30
</LI><LI> in flask B 1 ml suspension 4196 oD600 = 0.293
</LI><LI> 3 h at 30 °C and incubated while shaking
<B> 2. DNA-Preparation with YIplac204 overnight culture (see day 6/6.)</B></LI><LI>  Experiment conducted according to protocol 1
<B> 3. Gelelectrophoresis of overnight digestion and DNA Preparation </B></LI><LI> Experiment conducted according to protocol 2
</LI><LI> Changes to protocol:
  → 1.2 % agarose gel
</LI><LI> 5 µl of overnight digest added to 1 µl 6x staining buffer
</LI><LI> loaded onto gel according to following scheme
⇒ 6 µl 1kb-DNA-marker \\ overnight digest

</LI><LI> After the photo was taken DNA-Preparation from day 7/2. loaded on same gel
</LI><LI> Scheme : 6 µl 1kb-DNA-marker //overnight digest //empty// marker // Miniprep

<B> 4. Precipitation of plasmid-DNA of digested YIplac204 </B></LI><LI> 3.5 µl 3M NaAC added to DNA solution
</LI><LI> 1.5 µl Glycogen (Thermo Scientific) added
</LI><LI> 100µl 95% Ethanol added
</LI><LI> Incubation for 5 minutes at room temperature
</LI><LI> Centrifugation (fullspeed, 5 min, room temperature)
</LI><LI> DNA appears as a small pellet
</LI><LI> Supernatant removed
</LI><LI> Pellet washed in two volumes (240 µl) EtOH 70 %
</LI><LI> Dried at room temperature for 30 min
</LI><LI> DNA solved in 5 µl TE
<B> 5. Transformation of the yeast K699 </B></LI><LI> 25 ml yeast culture from the flask given in 50 ml falcon
</LI><LI> oD determined 
	 	</LI><LI> 699 = 0.76
		</LI><LI> 4196 = 0.75
		</LI><LI> Centrifugation (5 min, 3500 rpm, room temperature)
</LI><LI> Supernatant discarded
</LI><LI> Pellet resuspended in 25 ml ddH2O 
</LI><LI> Pelletised at 3500 rpm and room temperature
</LI><LI> Supernatant discarded
</LI><LI> Pellet in 1 ml of 100 mM LiAc resuspended and given in reaction tubes
</LI><LI> Centrifugation (10 s, 3500 rpm, room temperature)
</LI><LI> Supernatant discarded
</LI><LI> Pellet resuspended  in 500 µl 100 mM LiAc 
</LI><LI> For each transformation 50 µl cell suspension (4) transferred in 1.5ml ml reaction tubes 
</LI><LI> Centrifugated for 10 s and supernatant discarded
</LI><LI> All samples of 4196 discarded (too few DNA)
</LI><LI> Mix for transformation added (adhered to order as written below)</LI><LI> 240 µl 50 % PEG 3350</LI><LI> 36 µl 1 M LiAc</LI><LI> 5 µl carrier DNA (10mg/ml)</LI><LI> 4 µl YEP122 (positive control) / 5 µl YIP204/ 3µl YCplac22/ 5µl ddH2O (negative control)</LI>→ 65 µl of ddH2O to YEplac122 → 360 µl → 64 µl of ddH2O to YIplac204 → 360 µl → 66 µl ddH2O to YCplac22 → 360 µl → 64 µl ddH2O to negative control → 360 µl
<LI> Sample vortexed until pellet was resuspended
</LI><LI> Incubation at room temperature (30 °C)
</LI><LI> heat shock for 20 min at 42 °C
</LI><LI> centrifugated for 10 s, pellet resuspended in 400 µl H2O
	</LI><LI> YEplac122 200 µl plated</LI><LI> YIplac204 400 µl plated</LI><LI> YCplac22 200 µl plated</LI><LI> negative control 200 µl plated</LI><LI> incubated (72 h, 30 °C)
<H4> Day 8, 15/07/06: </H4><B> 1. Examination of 7/4.</B></LI><LI> Colonies grown!
</LI><LI> Transformation works
<B> 2. Resuspension of IDT geneBricks expression cassette (EC) and target cassette (TC) (TC) </B></LI><LI> 500 ng in 50 µl TE given (c = 10 ng/ml)
</LI><LI> incubated (50 °C, 20 min)
</LI><LI> vortexed and centrifugated (10 s at fullspeed)
<B> 3. Restriction digest for ligation of expression cassette (EC) and target cassette (TC) (TC) </B></LI><LI> calculations for needed amount of DNA via following formula: 

m(plasmid) * lengt (insert)/length(Vector)*5 → factor 5 is based on experience


Thus following values were calculated:
<TABLE><TBODY><TR><TD> Insert TC for cloning in YIplac204</TD><TD> =200 ng (plasmid) * 700 bp/3500 bp *5 = 200ng </TD></TR><TR><TD> Insert TC for cloning in YCplac22</TD><TD> =200 ng (plasmid) * 700 bp/4850 bp *5 = 144 ng </TD></TR><TR><TD> Insert EC for cloning in YEplac181</TD><TD> =200 ng (plasmid) * 500 bp/5740 bp *5 = 86ng  </TD></TR><TR><TD> Insert EC for cloning in K801000</TD><TD>=200 ng (plasmid) * 500 bp/5500 bp *5 = 90ng  </TD></TR></TBODY></TABLE></LI><LI> following restriction digests were constructed:
</LI><LI>insert EC/ insert TC</LI> DNA  40µl  MM for 3 samples EcoRI2µl 6µl PstI  2µl  6µl  Buffer 0  20µl  60µl  add 200µl ddH2O  136 µl 408 µl<UL><LI> Vector YEplac181/ YCplac22/ YEplac204 (7/3)</LI></UL><TABLE><TBODY><TR><TH align="left"> DNA </TH><TH align="left"> 5 µl</TH><TH align="left"> MM for 4 samples</TH></TR><TR><TD> RNAse </TD><TD> 1 µl</TD><TD> 4 µl</TD></TR><TR><TD> EcoRI	 </TD><TD> 1 µl</TD><TD> 4 µl</TD></TR><TR><TD> PstI	 </TD><TD> 1 µl</TD><TD> 4 µl</TD></TR><TR><TD> Buffer O </TD><TD> 5 µl</TD><TD> 20 µl</TD></TR><TR><TD> add 50µl ddH2O	 </TD><TD> 37 µl</TD><TD> 148 µl</TD></TR></TBODY></TABLE><UL><LI> Vector K801000
	</LI></UL><TABLE><TBODY><TR><TH align="left"> DNA </TH><TH align="left"> 40 µl</TH></TR><TR><TD> RNAse </TD><TD> 1 µl</TD></TR><TR><TD> EcoRI	 </TD><TD> 2 µl</TD></TR><TR><TD> PstI	 </TD><TD> 2 µl</TD></TR><TR><TD> Buffer O </TD><TD> 10 µl</TD></TR><TR><TD> add 100µl ddH2O	 </TD><TD> 45 µl</TD></TR></TBODY></TABLE><LI> large volumes were chosen because of the high EDTA-concentration
</LI><LI> over night incubation at 37 °C
<B> 4. CloneJet (Thermo Scientific) PCR-cloning with target cassette (TC) and EC</B></LI><LI> as a backup the following plasmids were generated using the CloneJET PCR Cloning Kit
<TABLE><TBODY><TR><TD>2 µl </TD><TD>10 x ligase buffer</TD></TR><TR><TD> 2.5 µl </TD><TD> DNA solution </TD></TR><TR><TD>1 µl</TD><TD>pjet-vector</TD></TR><TR><TD>add 19 µl ddH2O </TD><TD> 13.5µl</TD></TR><TR><TD>1 µl </TD><TD> T4Ligase </TD></TR></TBODY></TABLE></LI><LI>	Incubation (ca. 10 min)

<B> 5. Transformation</B></LI><LI> DH5alpha <I>E. coli</I> unfreezed
</LI><LI> Each 5 µl from Day8 4. given on top of 50 µl competent cells
</LI><LI> As a positive control  1µl PBSC</LI><LI>Bluescript was given on top of 50 µl  E.coli
</LI><LI> As a negative control no changes were applied to 50 µl E. coli
</LI><LI> incubate for about 15 min on ice
</LI><LI> heat shock for 90 s
</LI><LI> 2 min on ice
</LI><LI> 500 µl SOC medium added
</LI><LI> incubated (45 min, 37 °C)
</LI><LI> centrifugation (2 min, 7000 rpm)
</LI><LI> remove 450 µl supernatant
</LI><LI> E.coli in remained 100 µl resuspended
</LI><LI> 100 µl plated on ampiciline plates

<H4> Day 9, 15/07/07 </H4><B> 1. Analysis of over night digest from Day 8/3. </B></LI><LI> Electrophoresis was conducted according to protocol 2 
</LI><LI> The gel was loaded with 10 % digest volume
</LI><LI> 6x staining buffer was added according to volume (given in Brackets)
<DIV style=" text-align: left; text-indent: 0px; padding: 0px 0px 0px 0px; margin: 0px 0px 0px 0px;"><TABLE width="100%" border="0" cellpadding="2" cellspacing="2" style="background-color: #ffffff;"><TBODY><TR valign="top"><TD width="261" style="border-width : 0px;"><P style=" text-align: left; text-indent: 0px; padding: 0px 0px 0px 0px; margin: 0px 0px 0px 0px;"><SPAN style=" font-size: 10pt; font-family: 'Arial', 'Helvetica', sans-serif; font-style: normal; font-weight: normal; color: #000000; background-color: transparent; text-decoration: none;">YEplac181/ YCplac22/YIplac204</SPAN></P></TD><TD width="68" style="border-width : 0px;"><P style=" text-align: left; text-indent: 0px; padding: 0px 0px 0px 0px; margin: 0px 0px 0px 0px;"><SPAN style=" font-size: 10pt; font-family: 'Arial', 'Helvetica', sans-serif; font-style: normal; font-weight: normal; color: #000000; background-color: transparent; text-decoration: none;">5 µl</SPAN></P></TD><TD width="53" style="border-width : 0px;"><P style=" text-align: left; text-indent: 0px; padding: 0px 0px 0px 0px; margin: 0px 0px 0px 0px;"><SPAN style=" font-size: 10pt; font-family: 'Arial', 'Helvetica', sans-serif; font-style: normal; font-weight: normal; color: #000000; background-color: transparent; text-decoration: none;">(1 µl)</SPAN></P></TD></TR><TR valign="top"><TD width="261" style="border-width : 0px;"><P style=" text-align: left; text-indent: 0px; padding: 0px 0px 0px 0px; margin: 0px 0px 0px 0px;"><SPAN style=" font-size: 10pt; font-family: 'Arial', 'Helvetica', sans-serif; font-style: normal; font-weight: normal; color: #000000; background-color: transparent; text-decoration: none;">Bba_K801000</SPAN></P></TD><TD width="68" style="border-width : 0px;"><P style=" text-align: left; text-indent: 0px; padding: 0px 0px 0px 0px; margin: 0px 0px 0px 0px;"><SPAN style=" font-size: 10pt; font-family: 'Arial', 'Helvetica', sans-serif; font-style: normal; font-weight: normal; color: #000000; background-color: transparent; text-decoration: none;">10 µl</SPAN></P></TD><TD width="53" style="border-width : 0px;"><P style=" text-align: left; text-indent: 0px; padding: 0px 0px 0px 0px; margin: 0px 0px 0px 0px;"><SPAN style=" font-size: 10pt; font-family: 'Arial', 'Helvetica', sans-serif; font-style: normal; font-weight: normal; color: #000000; background-color: transparent; text-decoration: none;">(2 µl)</SPAN></P></TD></TR><TR valign="top"><TD width="261" style="border-width : 0px;"><P style=" text-align: left; text-indent: 0px; padding: 0px 0px 0px 0px; margin: 0px 0px 0px 0px;"><SPAN style=" font-size: 10pt; font-family: 'Arial', 'Helvetica', sans-serif; font-style: normal; font-weight: normal; color: #000000; background-color: transparent; text-decoration: none;">Inserts (target cassette (TC)/ EC</SPAN></P></TD><TD width="68" style="border-width : 0px;"><P style=" text-align: left; text-indent: 0px; padding: 0px 0px 0px 0px; margin: 0px 0px 0px 0px;"><SPAN style=" font-size: 10pt; font-family: 'Arial', 'Helvetica', sans-serif; font-style: normal; font-weight: normal; color: #000000; background-color: transparent; text-decoration: none;">20 µl</SPAN></P></TD><TD width="53" style="border-width : 0px;"><P style=" text-align: left; text-indent: 0px; padding: 0px 0px 0px 0px; margin: 0px 0px 0px 0px;"><SPAN style=" font-size: 10pt; font-family: 'Arial', 'Helvetica', sans-serif; font-style: normal; font-weight: normal; color: #000000; background-color: transparent; text-decoration: none;">(4 µl)</SPAN></P></TD></TR></TBODY></TABLE></DIV></LI><LI> Pockets were loaded according to following scheme
⇒ 1 kb DNA-marker //YEplac181//YCplac22//YIplac204//K801000//EC//target cassette (TC)
<B> 2. Restriction digestion 204 </B></LI><LI> 40 µl plasmid solution 204 obtained out of Day 7 2. digested
<TABLE border="0" width="75%"><TBODY><TR><TD>DNA	</TD><TD>40 µl</TD></TR><TR><TD>EcoRI</TD><TD>2 µl</TD></TR><TR><TD>PstI</TD><TD>2 µl</TD></TR><TR><TD>Buffer O</TD><TD>20 µl</TD></TR><TR><TD>H2O</TD><TD>136 µl</TD></TR><TR><TD>Σ</TD><TD>200 µl</TD></TR></TBODY></TABLE></LI><LI> incubation (2h, 37 °C)
<B> 3. Gelelectrophoresis for extraction </B></LI><LI> Gelelectrophoresis conducted according to protocol 2
</LI><LI> Middlepocket of gel loaded with 45 µl YCplac22 Day 8 3.according to following scheme: ⇒ 6µl 1 kb DNA</LI><LI>marker // empty // YCplac22 // empty
    </LI><LI> Changes to protocol:
	</LI><LI> large comb used
	</LI><LI> run for 2h at 50V
	<B> 4. Gelextraction via Quiagen kit</B></LI><LI> Gel fragment weighted: 470 mg
</LI><LI> 3 times the volume QC-buffer added → 1410 µl QC-buffer added
</LI><LI> 10 min at 50 °C gel dissolved
</LI><LI> After 3 min and 7 min for 5-7 s vortexed
</LI><LI> 450 µl isopropanol added
</LI><LI> Sample inverted and shortly vortexed
</LI><LI> 800 µl given on speedcolumn with wastetube
</LI><LI> Centrifugation (13300 rpm, 1 min) → flowthrough discarded
</LI><LI> Step 3 times repeated
</LI><LI> 0,75 ml PE buffer added on column for cleaning
</LI><LI> Centrifugation (13300 rpm, 1 min)
</LI><LI> Flowthrough discarded
</LI><LI> Centrifugation (13300 rpm, 1 min)
</LI><LI> Flowthrough discarded
</LI><LI> Column transferred on 1.5 ml tube
</LI><LI> 30 µl Elutionbuffer added on column
</LI><LI> Centrifugation (13300 rpm, 1 min)
</LI><LI> Eluate used for further experiments
<B> 5.  Further cleaning with Quiagen PCR-purification-kit </B></LI><LI> 180 µl sample given to 900 µl PB buffer given
</LI><LI> 800 µl Sample given on column
</LI><LI> column centrifugated (13300 rpm, 1 min)
</LI><LI> remaining 280 µl given on column
</LI><LI> column centrifugated at 13300 rpm
</LI><LI> both times flowthrough was discarded
</LI><LI> column loaded with 750 µl PE
</LI><LI> centrifugation (13300 rpm, 1 min)
</LI><LI> flowthrough discarded
</LI><LI> centrifugation (13300 rpm, 1 min)
</LI><LI> flowthrough discarded
</LI><LI> column transferred on 1.5 ml Eppi
</LI><LI> 30 µl EB (elution buffer) given
</LI><LI> centrifugation (13300 rpm, 1 min)
</LI><LI> eluate used for further experiments
<B> 6. Controlgelelectrophoresis of eluate and digestion of YIplac204 and YCplac22 (day 9/2.) </B></LI><LI>  Gelelectrophoresis according to protocol 2: Electrophoresis
</LI><LI>  Gel loaded with samples according to following scheme:
⇒	Standard (6µl) // YCplac22 (3µl) //YIplac204 digested (20µl) // empty // Standard (6µl) // Insert his3_reporter_klein (2.8µl)
</LI><LI>Changes to protocol:
	</LI><LI> Gelelectrophoresis for 45 min
	</LI><LI>note: insert was added after 20 min 
	<B> 7. Overnight culture of Pjet-transformed E. coli </B></LI><LI> 2 colonies on plate 204, Pjet_His_spacer, Pjet_reporter_klein picked
</LI><LI> Each were given into 5 ml 80 µg/µl ampiciline medium given
</LI><LI> over night incubation at 37 °C
<H4> Day 10, 15/07/08: </H4><B> 1. Ligation of the linearised vector YCplac22 and the target cassette (TC) </B></LI><LI> 3 samples for ligation generated
 </LI><LI> ligation with insert
	</LI><LI> 3 µl vector (YCPlac22) linearized
	</LI><LI> 14 µl insert (target cassette (TC)/ EC) linearized
	</LI><LI> 2 µl ligase buffer
	</LI><LI> 1 µl T4 Ligase
	</LI><LI> Religation control without insert
	</LI><LI> 3 µl vector (YCPlac22) linearized
	</LI><LI>14 µl ddH2O
	</LI><LI> 2 µl ligase buffer
	</LI><LI> 1 µl T4 Ligase
	</LI><LI> control for complete digestion
	</LI><LI> 3 µl vector (YCPlac22) linearized
	</LI><LI> 15 µl H2O
	</LI><LI> 2 µl ligase buffer
	</LI><LI> ligation incubated for 3 h at room temperature
	<B> 2. Miniprep from overnight culture day 9/7.</B></LI><LI> 2 ml over night culture transferred to 2 ml-reaction tubes
</LI><LI> centrifugation (7000 rpm, 5 min)
</LI><LI> supernatant discarded
</LI><LI> 2 ml from over night culture transferred in the same reaction tubes as their precursors to increase the amount of bacteria
</LI><LI> centrifugation (7000 rpm, 5 min)
</LI><LI> supernatant discarded
</LI><LI> DNA</LI><LI>Preparation conducted as given by Quia-Gen Miniprep instruction
</LI><LI> Elution in 50 µl elution buffer
<B> 3. Digestion of Miniprep </B></LI><LI>Digest prepared
<TABLE><TABLE border="0" width="70%"><TBODY><TR><TH align="left">1 µl DNA from the miniprep(see above)</TH><TH align="left">MM for 7 samples à 19 µl</TH></TR><TR><TD>EcoRI </TD><TD>0.5 µl</TD><TD>3.5 µl</TD></TR><TR><TD>PstI</TD><TD>0.5 µl </TD><TD>3.5 µl</TD></TR><TR><TD>Buffer O</TD><TD>2 µl</TD><TD>14 µl	</TD></TR><TR><TD>ddH2O</TD><TD>16 µl </TD><TD>112 µl 	</TD></TR></TBODY></TABLE></TABLE></LI><LI> digestion for 3 h at 37 °C
<B> 4. Moulding of Chloramphenicol agar plates </B></LI><LI> TY-Medium with agar heated
</LI><LI> Cooling while mixing
</LI><LI> Addition of 50 µl Chloramphenicol (100 mg/ml)
</LI><LI> c_end= 10µg/ml
</LI><LI> Moulding of plates
<B> 5 Preparation of X-gal plates </B></LI><LI> 40µl 2% x-gal spread on 6 ampicilin enriched plates
</LI><LI> 40µl IPTG spread on the dried plates
</LI><LI> 40µl dH2O spread on the dried plates
<B> 6 Transformation of Ligation and EYFP (BBa_E2030)</B></LI><LI> Each of folowing DNA</LI><LI>samples added to 50 µl DH5α on ice
	</LI><LI>5µl ligation from 10.1
	</LI><LI>5µl ligase + vectorligation from 10.1
	</LI><LI>5µl vector + ligase buffer solution from 10.1
	</LI><LI>1µl pBluescript
	</LI><LI>5µl H2O
	</LI><LI>2.5µl EYFP  (BBa_E2030)
	</LI><LI>Transformation analogous to 8.5
</LI><LI>Transformed <I>E. coli</I> samples spread on ampiciline plates as followed
	</LI><LI> 100 µl ligation
	</LI><LI> 200 µl ligation
	</LI><LI> 200 µl ligase + vector
	</LI><LI> 200 µl pBluescript
	</LI><LI> 200 µl H2O
	</LI><LI>Remaining transformed bacteria spread on chloramphenicole agar plates
	</LI><LI> 200 µl EYFP  (BBa_E2030)
	</LI><LI> 200 µl H2O
	<B> 7. Gelelectrophoresis of the Digestion from 3. (see above)</B></LI><LI> Gelelectrophoresis conducted according to protocol 2
</LI><LI> 20 µl taken from earlier digest (see above) and added to 4µl Staining Solution
</LI><LI> Gel loaded according to following scheme:
⇒ target cassette (TC) A // target cassette (TC) B // EC A // EC B// YIplac204 A //YIplac 204 B// 1kb DNA-marker

<H4>Day 11, 15/07/09</H4><B>1. Cultures picked for over night culture </B></LI><LI> 12 colonies taken from 100 µl plate  (compare Day 10 6.) 
</LI><LI> inoculation of 2 ml 80 µg/ml ampiciline medium
</LI><LI> 2 colonies picked from YFP</LI><LI>plate
</LI><LI> 5 ml 25 µg/ml Canamycin medium inoculated
</LI><LI> over night incubation at 37 °C
<H4>Day 12, 15/07/10</H4><B> 1. Miniprep of the over night culture from day 11/1. (see above)</B></LI><LI> conducted according to protocol 1
</LI><LI> eluted in 30µl 1x TE
<B> 2. Restriction digestion </B></LI><LI> Restriction for all DNA-preparations (14 samples)
<TABLE border="0" width="100%"><TBODY><TR><TD>DNA Solution</TD><TD>3 µl 	</TD><TD>MM for 15</TD></TR><TR><TD>PstI</TD><TD>0.5 µl</TD><TD>7.5 µl </TD></TR><TR><TD>EcoRI</TD><TD>0.5 µl</TD><TD>7.5 µl</TD></TR><TR><TD>Buffer O</TD><TD>2 µl</TD><TD>30 µl</TD></TR><TR><TD>ddH2O</TD><TD>14 µl</TD><TD>210 µl</TD></TR></TBODY></TABLE></LI><LI> incubation (1.5 h, 37 °C)
<B> 3. Gelelectrophoresis</B></LI><LI> Gelelectrophoresis performed according to protocol 2
</LI><LI> Changes to protocol:
	</LI><LI> Volume for two 1% agarosegels produced 170 ml (100 ml+70 ml)
	</LI><LI> 13,6 µl Roth Ethidiumbromide added
	</LI><LI> Samples  from Day 12 2. (see above) were added to 4 µl staining solution
</LI><LI> Gels were loaded as followed:
<TABLE border="0" width="57.3%"><TBODY><TR><TD>Gel I:</TD><TD>1kb DNA-marker// YCPlac22 + insert</TD><TD>	1 //</TD></TR><TR><TD>2 //</TD></TR><TR><TD>3 //</TD></TR><TR><TD>4 //</TD></TR><TR><TD>5 //</TD></TR><TR><TD>6 //</TD></TR><TR><TD>7 //</TD></TR><TR><TD>8 //</TD></TR></TBODY></TABLE><TABLE border="0" width="70%"><TBODY><TR><TD>Gel II:</TD><TD>1kb DNA-marker // YCPlac22 + insert </TD><TD>9 //</TD></TR><TR><TD>10 //</TD></TR><TR><TD>11 //</TD></TR><TR><TD>12 //</TD></TR><TR><TD>empty//</TD></TR><TR><TD>YFP clone 1 	//</TD></TR><TR><TD>YFP clone 2//</TD></TR><TR><TD>stamdard//</TD></TR></TBODY></TABLE></LI><LI>  Results not expected ratio insert vector seems tilted
</LI><LI>  Over night culture of all samples analogous to Day 11/1. 
<H4> Day 13, 15/07/11 </H4><B> 1. Miniprep via Quiagen column </B></LI><LI> 5 ml over night culture from day 12/3.  centrifuged (7000 rpm, 5min) in 2 ml Eppis 
</LI><LI> Quiagen Miniprep analogous to 10/2. performed
<B> 2. Restriction digestion </B></LI><LI> DNA-Solutions obtained in Day 13/1. (see above) EcoRI PstI digested
</LI><LI> Mastermix created
<TABLE border="0" width="100%"><TBODY><TR><TH align="left">single sample</TH><TH align="left">Mastermix for 13 Samples</TH></TR><TR><TD> 1 µl DNA	</TD><TD>-</TD></TR><TR><TD>0.5 µl EcoRI</TD><TD>6.5 µl EcoRI</TD></TR><TR><TD>0.5 µl PstI	</TD><TD>6.5 µl PstI</TD></TR><TR><TD>2 µl Buffer0</TD><TD>26 µl Buffer0</TD></TR><TR><TD> 16 µl ddH2O</TD><TD>208 µl  ddH2O</TD></TR></TBODY></TABLE></LI><LI> incubation (2 h, 37 °C)
<B> 3. Gelelectrophoresis </B></LI><LI> Gelelectrophoresis conducted according to protocol 2
</LI><LI> 4 µl staining solution added to each of the 12 samples from experiment Day 13/2. (see above) 
</LI><LI> 24 µl loaded on gel according to following scheme
	</LI><LI> 1kb DNA-marker //
	</LI><LI> YCPlac 22 + insert 1 //
	</LI><LI> YCPlac 22 + insert 2 //
	</LI><LI> YCPlac 22 + insert 3 //
	</LI><LI> YCPlac 22 + insert 4 //
	</LI><LI> ...
	</LI><LI> YCPlac 22 + insert 12 //
	<H4> Day 14, 15/07/14</H4><B> 1. Restriction digestion XhoI, Sal I from YCPlac22 + insert10 (day 13/13)</B></LI><LI> restriction digestion conducted
<TABLE border="0" width="70%"><TBODY><TR><TD>DNA YCPlac22 + insert10 from day 13</TD><TD>20 µl</TD></TR><TR><TD> Sal I</TD><TD>2 µl</TD></TR><TR><TD>XhoI</TD><TD>4 µl</TD></TR><TR><TD>Buffer 0</TD><TD>10 µl</TD></TR><TR><TD>ddH2O</TD><TD>64 µl</TD></TR></TBODY></TABLE></LI><LI> incubation (2 h, 37 °C)
<B> 2. Addition of SalI and XhoI restriction sites via PCR </B><TABLE border="0" width="70%"><TBODY><TR><TH align="left">Sample	</TH><TH align="left">water control</TH></TR><TR><TD>H2O 	</TD><TD>71 µl </TD><TD>73 µl</TD></TR><TR><TD>5 x buffer 	</TD><TD>20 µl 	</TD><TD>20 µl</TD></TR><TR><TD>template </TD><TD>2 µl</TD><TD>-</TD></TR><TR><TD>Forward Primer </TD><TD>2 µl</TD><TD>2 µl</TD></TR><TR><TD>Reverse Primer </TD><TD>2 µl 		</TD><TD>2 µl</TD></TR><TR><TD>Fusion Polymerase 	</TD><TD>1 µl	</TD><TD>1 µl</TD></TR><TR><TH align="left">Σ</TH><TD> 100 µl </TD><TD> 100 µl </TD></TR></TBODY></TABLE></LI><LI> program:
	 	</LI><LI> 1. 	98.0 °C		15 s
		</LI><LI> 2. 	98.0 °C		10 s  
		</LI><LI> 3. 	72.0 °C		15 s
		</LI><LI> 4. 	72.0 °C		3 s
		</LI><LI> each step was repeated 30 times 
<B> 3. Gelelectrophoresis of the restriction digestion and  PCR </B></LI><LI> Gelelectrophoresis performed according to protocol 2
</LI><LI> Changes to protocol
	 </LI><LI>  70 ml 1 % agarose gel moulded
	 </LI><LI> 1 µl staining solution added to 5 µl water control
</LI><LI> 1 µl staining solution added to 5 µl PCR-sample
</LI><LI> 2 µl staining solution added to 10 µl of digest 
</LI><LI> Gel loaded according to following scheme
	 1kb DNA-marker // digestion // water control // PCR
<B> 4. Purification of the PCR fragment </B></LI><LI> Analogous to Day 9/5.  
</LI><LI> pellet was solved in 30 µl EB
<B> 5. Restriction digestion of the YFP-PCR fragment </B></LI><LI> purified insert digested with XhoI Sal I
<TABLE border="0" width="100%"><TBODY><TR><TD>DNA</TD><TD>30 µl</TD></TR><TR><TD>Sal I</TD><TD>2 µl</TD></TR><TR><TD>XhoI</TD><TD>4 µl</TD></TR><TR><TD> Buffer O</TD><TD>10 µl</TD></TR><TR><TD>ddH2O</TD><TD>54 µl</TD></TR></TBODY></TABLE></LI><LI> incubation (3 h, 37 °C))
<B> 6. Gelextraction of the digested vector </B></LI><LI> gelelectrophoresis (1.5 h) in 1 % agarose gel analogous to Day 9 4.
</LI><LI> band with scalpel removed and weighted = 390 mg
</LI><LI> three fold amount of QC added (1160 µl)
</LI><LI> analogue to Day 9 4.  continued
<H4> Day 15, 15/07/15 </H4><B>1. Purification of the digestion from day 14/5.</B></LI><LI> Analogue to  Day 9/5. 
</LI><LI> Changes to Day 9/5.
	 </LI><LI>  380 µl PB used
	</LI><LI>  eluted in 30 µl  Elution Buffer 
	<B> 2. Dephosphorylation of the vector </B></LI><LI> alkaline phosphatase mix created
<TABLE border="0" width="70%"><TBODY><TR><TD>TCPlac22 + insert XhoI Sal I digested</TD><TD>20 µl	</TD></TR><TR><TD>FAP buffer</TD><TD>3 µl</TD></TR><TR><TD>FAP</TD><TD>1 µl</TD></TR><TR><TD>H2O</TD><TD>6 µl</TD></TR></TBODY></TABLE></LI><LI> incubation (20 min, 37 °C)
</LI><LI> incubation (5 min, 75 °C)
<B> 3. Ligation YCPlac22 + insert target cassette (TC) and YFP </B></LI><LI> 3 ligation samples prepared
<TABLE border="0" width=" 70%"><TBODY><TR><TD>A)</TD><TD>4 µl vector, dephosphorylated (YCPlac22)</TD></TR><TR><TD>5 µl insert (YFP)</TD></TR><TR><TD>2 µl ligase buffer</TD></TR><TR><TD>1 µl T4 ligase</TD></TR><TR><TD>8 µl ddH2O</TD></TR></TBODY></TABLE><TABLE border="0" width="53%"><TBODY><TR><TD>B)</TD><TD>4 µl vector, dephosphorylated</TD></TR><TR><TD>2 µl ligase buffer</TD></TR><TR><TD>1 µl T4 ligase</TD></TR><TR><TD>12 µl ddH2O</TD></TR></TBODY></TABLE><TABLE border="0" width="53%"><TBODY><TR><TD>C)</TD><TD>4 µl vector, dephosphorylated</TD></TR><TR><TD>2 µl ligase buffer</TD></TR><TR><TD>14 µl ddH2O</TD></TR></TBODY></TABLE></LI><LI> incubation (3 h, room temperature)
<B> 4. Transformation of the ligation</B></LI><LI> performed analogue to Day 8 5. 
</LI><LI> transformation plated as followed:
	 </LI><LI> 200 µl, 100 µl, 50 µl of ligation on ampiciline plated
	</LI><LI> 200 µl ligase + vector on ampiciline plated
	</LI><LI> 200 µl vector on ampiciline plated
	</LI><LI> 200 µl PBSC-Bluescript and H2O on ampiciline plated
	<H4>Day 16, 15/07/16</H4><B> 1. Picking of colonies </B></LI><LI> 20 colonies of the ligation plate picked
</LI><LI> Each given in 5 ml 100 µg/ml ampiciline medium
</LI><LI> Incubation over night at 37 °C
<B> 2. Retransformation of the ligation</B></LI><LI> Repition of Day 15/3. (see above)
<H4> Day 17, 15/07/17 </H4><B> 1. Miniprep of the over night culture </B></LI><LI> Performed according to protocol 1
<B> 2. Restriction digestion from the Miniprep </B></LI><LI> Restriction digest created
<TABLE border="0" width="100%"><TBODY><TR><TH align="left"> Miniprep</TH><TH align="left">1 µl </TH><TH align="left">Mastermix for 21 samples</TH></TR><TR><TD>Sal I</TD><TD>0.5 µl</TD><TD>10.5 µl</TD></TR><TR><TD>XhoI</TD><TD>1 µl</TD><TD>21 µl</TD></TR><TR><TD>Buffer 0</TD><TD>2 µl </TD><TD>42 µl</TD></TR><TR><TD>ddH2O	</TD><TD>15.5 µl</TD><TD>325.5 µl</TD></TR></TBODY></TABLE></LI><LI> digestion (2 h, 37 °C)
<B> 3. Gelelectrophoresis </B></LI><LI> Gelelectrophoresis performed according to protocol 2
</LI><LI> 4 µl  staining solution was added to each digest
</LI><LI> 6 µl 1kb DNA-marker loaded
</LI><LI> scheme:
<TABLE><TBODY><TR><TD> 1kb DNA-marker // Miniprep 1 - 10 = Gel I </TD><TD> 1kb DNA-marker // Miniprep 11 - 20 = Gel II </TD></TR></TBODY></TABLE></LI><LI> Restriction sites XhoI and SalI are compatible, but after XhoI-SalI-ligation the restriction site is removed. Thus &quot;homo&quot;-ligation creates an insert after XhoI-SalI digestion whereas &quot;hetero&quot;-ligation does not.
</LI><LI> Restriction sites XhoI and SalI are compatible, but after XhoI-SalI-ligation the restriction site is removed. Thus &quot;homo&quot;-ligation creates an insert after XhoI-SalI digestion whereas &quot;hetero&quot;-ligation does not.
</LI><LI> Clones 1, 8, 9, 12, 14, 18 had an insert, which were similar to the expected size
<H4> Day 18, 15/07/20</H4><B> 1. Overnight culture of S. cerevisiae K699 and 4196 </B></LI><LI> Two colonies picked of each strain
</LI><LI> Inoculation of 5 ml YEPD-Medium each
</LI><LI> Incubation overnight at 28°C
<H4>Day 19, 15/07/21</H4><B> 1. Transformation of S. cerevisiae strains K699 and 4196 with YCplac22-YFP</B></LI><LI> over night culture dilitued 1:100 with MQ (10 µl suspension and 990 µl MQ)
</LI><LI> measurement 0D600:
<TABLE border="0" width="19%"><TBODY><TR><TD>699 A =</TD><TD>0.019</TD><TD>→ used</TD></TR><TR><TD>699 B =</TD><TD>0.014 </TD></TR><TR><TD>4196 A =</TD><TD>0.108</TD><TD>→ used</TD></TR><TR><TD>4196 B=</TD><TD>0.056</TD></TR></TBODY></TABLE></LI><LI> two flasks filled with YEPG next to Bunsenburner
<TABLE><TBODY><TR><TD>A: 25 ml</TD></TR><TR><TD>B: 25 ml</TD></TR></TBODY></TABLE></LI><LI> 4,46 ml added to flask A suspension 699 oD600 = 0,208
</LI><LI> 0.58ml added to flask B suspension 4196 oD600 = 0,193
</LI><LI> 3 h at 30 °C and incubated while shaking
</LI><LI> 25 ml yeast culture from the flask given in 50 ml falcon
</LI><LI> oD600 determined 
<TABLE><TBODY><TR><TD>699 = 0.543</TD></TR><TR><TD>4196 = 0.503</TD></TR></TBODY></TABLE></LI><LI> Centrifugation (5 min, 3500 rpm, room temperature)
</LI><LI> Supernatant discarded
</LI><LI> Contamination in 4196 suspension detected =&gt; Sample discarded
</LI><LI> Pellet resuspended in 25 ml ddH2O 
</LI><LI> Pelletised at 3500 rpm and room temperature
</LI><LI> Supernatant discarded
</LI><LI> Pellet in 1 ml of 100 mM LiAc resuspended and given in 1.5 reaction tubes
</LI><LI> Centrifugation (10 s, 3500 rpm, room temperature)
</LI><LI> Supernatant discarded
</LI><LI> Pellet resuspended  in 500 µl 100 mM LiAc 
</LI><LI> For each transformation 50 µl cell suspension (6) transferred in 1.5ml ml reaction tubes 
</LI><LI> Centrifugated for 10 s and supernatant discarded
</LI><LI> Mix for transformation added (adhered to order as written below)
	 </LI><LI>&gt; 240 µl 50 % PEG 3350
	</LI><LI>&gt; 36 µl 1 M LiAc
	</LI><LI>&gt; 5 µl carrier DNA (10mg/ml)
	</LI><LI>&gt; 5µl ddH2O (negative control)/ YCplac22-YFP  1, 8, 9, 12, 14 and 18
	</LI><LI>&gt; 64 µl of ddH2O 
	</LI><LI> Sample vortexed until pellet was resuspended
</LI><LI> Incubation at room temperature (30 °C)
</LI><LI> Heat shock for 20 min at 42 °C
</LI><LI> Centrifugated for 10 s,
</LI><LI> Pellet resuspended in 400 µl H2O
</LI><LI> 200 µl and 100µl of transformation plated on mediaplates without tryptophan
</LI><LI> Incubated (72 h, 30 °C)
<H4> Day 20, 15/07/24 </H4><B>1. Confocal Microskopy</B></LI><LI> One colony per construct picked  and diluted in Water
</LI><LI> Microscopy → see result section
</LI></DIV><DIV class="accordionTitel"> Homologous Recombination </DIV><DIV class="pane" style="height:4750px;"><H4> 2015-07-27 </H4><B> gBlock Can1_rep_klein </B><UL><LI> was resuspended in 50µl TE Buffer according to manufacturer's instructions (10ng/µl) 
</LI></UL><B> Ligation </B><UL><LI> For amplification and Storage, the gBlock was cloned into pJET 1.2 (linearized, blunt ends). 
</LI><LI> 0.15 pmol of DNA ends (Insert) are recommended to optimize ligation efficiency. --&gt; 32ng DNA (calculated using NEBioCalculator) + 3.2µl gBlock 
</LI><LI> reaction: 3.2µl gBlock // 2µl 10x Buffer (Protocol: 10µl 2x Buffer) // 1µl pJET 1.2 // 12.8µl ddH2O // 1µl T4 DNA Ligase
</LI><LI> Incubated ~20'at RT
</LI></UL><B> Transformation </B><UL><LL> Transfomed 3x50µl chemically competent <I> E.coli </I> DH5a 
<LI> positive control: 1ng/µl (-&gt; 1µl) pBKS
</LI><LI> negative control: 1µl ddH2O  
</LI><LI> Ligation 5µl
</LI><LI> DNA (or water) was pipetted onto the cells and stirred gently. Incubation 30' on ice.
</LI><LI> Heat shock: 90&quot; at 42°C, 2' on ice,  added 450µl SOC.
</LI><LI> Incubation 30' at 37°C
</LI><LI> Cells were plated on TY-Amp plates (100µg Amp/l)
</LI><LI> controls: 100µl
</LI><LI> ligation: 100µl and 400µl
</LI><LI> Incubation at 37°C over night.
</LI></LL></UL><H4> 2015-07-30 </H4><UL><LI> Miniprepped overnight cultures of (YC22-YFP 1.1, 1.2, 4.1, 4.2, pJET-Can1_rep 1, 2) using the Promega PureYield Plasmid Miniprep System following manufacturer’s instructions.
</LI><LI> Test digest of Minipreps as well as YIplac 204 (A. Rieck) with EcoRI &amp; PstI
</LI><LI> 7x Mastermix: 14µl OBuffer // 7µl EcoRI // 7µl PstI // 105µl dH2O
</LI><LI> 19µl Mastermix + 1µl Miniprep were used for each digest. Incubation 1h at 37°C.
</LI><LI> Expected fragment lengths (approximate): YC: 1.4kbp + 4.8 kbp // pJET: 3kbp + 700bp // YIplac204: 3.5kbp
</LI><LI> Gel: 100ml 1% agarose in TAE, 5µl EtBr loaded 20µl sample + 4µl 6x Loading Dye and 6µl GeneRuler 1kb DNA ladder
</LI><LI> Plasmids were not completely digested, resulting in additional bands on the gel. Expected bands are present
</LI></UL><H4> 2015-07-31 </H4><B> Digested YC1.1 and YIplac204 with EcoRI and PstI </B><UL><LI> 50µl reactions: 5µl 10x O Buffer // 2µl EcoRI // 2µl PstI // 30µl YIplac204 Miniprep // 11µl dH2O or 10µl YC1.1 Miniprep // 31µl dH2O 
</LI><LI> Digested 2h at 37°C, then loaded 2µl on test gel (100ml 1.2% agarose in TAE, 5µl EtBr)
</LI><LI> Bands on gel as expected: YIplac: 3.5kbp, YC1.1: 4.8kbp/1.4kbp
</LI><LI> Loaded 48µl sample on fresh gel (as above) for extraction. Extracted 3.5kb YIplac Band and 1.4kb YC1.1 Band.
</LI><LI> QIAquick Gel extraction Kit was used for gel extraction. Followed manufacturer’s instructions.
</LI></UL><H4> 2015-08-04 </H4><UL><LI> Test gel of extracted DNA: Backbone concentration too low
</LI><LI> Digested YIplac204 Miniprep from 2015-07-08 with EcoRI and EcoRV to estimate concentration 
</LI><LI> Complete digest of miniprep (~50µl) with EcoRI and PstI over night: added 5µl EcoRI Buffer and 2µl EcoRI.
</LI></UL><H4> 2015-08-05 </H4><UL><LI> Added 0.5µl EcoRI to digest. Incubated 0.5h at 37°C, then added 2µl PstI. Incubated 2h at 37°C.
</LI><LI> Purification with QIAquick PCR Purification Kit. Elution in 30µl EB.
</LI><LI> 2µl eluate were loaded on gel (80ml 1% agarose in TAE, 4µl EtBr). Standard: 12µl GeneRuler 1kb.
</LI><LI> Estimation: approx. 40ng/ul DNA
</LI><LI> Dephosphorylation: 25.5µl Backbone // 3µl FastAP buffer // 1.5µl FastAP // Incubation 20 min at 37°C. 
</LI><LI> Inactivation: 5min at 75°C 
</LI><LI> Ligation: 	2µl T4 DNA Ligase Buffer // 1µl T4 DNA Ligase // 2µl Backbone // 2µl Insert // 5µl H2O		
</LI><LI> Incubation ON at 4°C
</LI><LI> Digested YFP 4.1 and YIplac204A with EcoRI &amp; PstI 
</LI><LI> 20µl Miniprep: 2.5µl 10x O Buffer // 1.5µl EcoRI // 1.5µl PstI
</LI><LI> Stored at 4°C
</LI></UL><H4> 2015-08-06 </H4><UL><LI> Incubated digest 1h at 37°C, then added 0.5µl EcoRI and 0.5µl PstI, incubated 1h at 37°C and subsequently stored at -20°C.
</LI><LI> Ligation: 	DH5a E. coli were transformed with 1µl ligation reaction. Controls: -ligase, -insert; positive control: pBKs; negative control: H2O; retransformation of YIplac204A
</LI><LI> Plated on TY-Amp plates, ligation and retransformation were plated on TY-Amp with X-Gal.
</LI><LI> Prepared 2x 1% agarose gel for the next day.
</LI></UL><H4> 2015-08-07 </H4><B> Transformation </B><UL><LI> On the plate with the ligation reaction both blue and white colonies have grown. Blue: uncut/religated, white: ligated. Both ligation controls yielded several colonies as well.
Water control: no colonies; pBKs control: several colonies; YIplac204: a lot of colonies 
</LI><LI> Tested restriction digest: loaded 1µl on gel (80ml 1% agarose in TAE). Standard: 6µl GeneRuler 1kb
</LI><LI> Gel extraction: Loaded rest of the sample on 1.2% (100ml) agarose gel.
</LI><LI> The smaller band from YC4.1 as well as the backbone band were cut out from the gel and stored at -20°C.
</LI><LI> 8 clones were picked from the ligation transformation plate and 1 clone from the retransformation plate (2015-08-06) and inoculated in 4ml TY-Amp, stored at 4°C for two days, and then incubated over night at 37°C.
</LI><LI> Test digest of YIplac204 Miniprep (executed by V. Vyshnevskyi) with EcoRI and EcoRV; Expected bands: ~ 880 + 2660. The result matches the expectations.
</LI></UL><H4> 2015-08-10 </H4><UL><LI> Miniprepped ONCs using the Promega PureYield Plasmid Miniprep System. Followed manufacturer’s instructions.
</LI><LI> Test digest: YIplac+YFP constructs with EcoRI/PstI -&gt; 9x Master mix: 4.5µl EcoRI // 4.5µl PstI // 18µl O Buffer // 144µl dH2O // +1µl Miniprep per reaction. The reactions were incubated for 1h at 37°C. 
</LI><LI> Gel electrophoresis: 80ml 1% agarose in TAE, 4µl EtBr. Standard: 6µl GeneRuler 1kb; Expected fragment sizes: approx. 1.4kbp/3.5kbp. Both expected bands present
Partially digested DNA is responsible for the additional band (~4.9kbp) in lanes 3, 6, and 7.
</LI><LI> New nomenclature: YIplac204 constructs with YFP cassette are now “YIYFP” 1-8
</LI></UL><H4> 2015-08-11 </H4><UL><LI> Second test digest of YIYFP minipreps: EcoRV/XhoI -&gt; 9x Master mix: 4.5µl EcoRV // 4.5µl XhoI // 18µl O Buffer // 144µl dH2O +1µl Miniprep per reaction. Incubated 2h at 37°C.
</LI><LI> Gel electrophoresis: 80ml 1% agarose in TAE, 4µl EtBr. Standard: 6µl GeneRuler 1kb. Expected fragment sizes: approx. 1.2kbp/3.7kbp. Both expected bands are present
</LI></UL><H4> 2015-08-12 </H4><UL><LI> Inoculated yeast (<I> S. cerevisiae </I>) overnight culture: One colony from 699 strain in 3ml YEPD was incubated over night at 28°C and 180RPM
</LI><LI> Made TY-Agar mix for Chloramphenicol plates: 16g Tryptone, 10g Yeast extract, 5g NaCl, 15g Agar. Did not add water yet.
</LI></UL><H4> 2015-08-13 </H4><UL><LI> Measured OD600 of yeast ONC: OD600 = 2.165
</LI><LI> According to the protocol, OD600 should be at 0.25-0.5 for the final culture. Culture was diluted in YEPD until the OD reached 0.423 to a final volume of approximately 40 ml. Incubation: 3h at 28°C and 180RPM
</LI><LI> Digested YIYFP plasmids with EcoRV (for insertion in Trp1 locus)
</LI><LI> 10µl each of minipreps YIYFP 1, YIYFP 2 and YIYFP 8 were digested. 4x Master mix: 6µl EcoRV // 8µl R Buffer // 26µl dH2O +10µl Miniprep per reaction
</LI><LI> 2µl of each digest were loaded on a gel (80ml 1% agarose in TAE, 4µl EtBr)
</LI><LI> Attempted to purify linearized plasmid (18µl) using the Promega PureYield Miniprep Kit.
</LI><LI> 2µl of each sample were loaded on a gel (80ml 1% agarose in TAE, 4µl EtBr). Expected fragment size: 4.8kbp. Bands on gel as expected. # Purification of linearized plasmid with Promega miniprep kit seems to work.
</LI></UL><B> Yeast transformation </B><UL><LI> 1 transformation with purified YIYFP1 (linearized with EcoRV) DNA. 
</LI><LI> 1 positive control with YC22-YFP construct (clone 1). 
</LI><LI> 1 negative control without transforming DNA 
</LI><LI> Resuspended pellet in 100µl TE and plated the cells.
</LI></UL><H4> 2015-08-17 </H4><B> Yeast plates </B><UL><LI> negative control – 0 colonies
</LI><LI> positive control – almost continuous cell layer
</LI><LI> YIYFP transformation – approx. 100 colonies
</LI></UL><H4> 2015-08-18 </H4><UL><LI> Test digest of „Vector 181 lin.“ too see if this is actually YEplac181 linearized with EcoRI and PstI.
</LI><LI> Digest with EcoRV should then yield two fragments of approx. 3.4kb and 2.3kb. --&gt; Digest mix: 2µl DNA // 2µl R Buffer // 1µl EcoRV // 15µl dH2O
</LI><LI> Bands fit expectations.
</LI></UL><H4> 2015-08-19 </H4><UL><LI> Miniprepped EC clone 5 for V. Vyshnevskyi using the Promega PureYield Plasmid Miniprep System.
</LI><LI> Inoculated Yeast overnight cultures for CLSM. 5 clones of the YIYFP1 transformation and 699YFP1A + 699YFP 4B (Yeast K699 containing YCplac22 with YFP construct. Clones previously proved to express YFP)
</LI></UL><H4> 2015-08-20 </H4><UL><LI> Note: negative control (K699) was not inoculated. Picked colony from plate and incubated for 4 hours at 28°C and 180RPM.
</LI><LI> PCR of YIYFP1 plasmid with YFP primers to test the primer efficiency.
</LI><LI> Dilution series: 1:100, 1:1000, 1:10000
</LI><LI> 50µl PCR reactions - 5x Master mix: 2.5µl fw primer // 2.5µl rv primer // 25µl 10x Y-Buffer // 5µl dNTPs // 2.5µl Taq-Pol // 207.5µl dH2O +1µl DNA per reaction
</LI><LI> PCR program: (1) 4 min 94°C; (2) 30 sec 94°C; (3) 30 sec 60°C;  (4) 1 min 72°C; Repeat 2-4 30x; (5) 10 min 72°C; (6) -- 4°C
</LI><LI> Expected 700bp bands are present on gel. However, the product is too large for qPCR. Need new primers.
</LI><LI> CLSM analysis of 5 YIYFP transformants. Clones 1-3 exhibit strong fluorescence. Clones 4 and 5: moderate fluorescence. Stored ONCs at 4°C.
</LI></UL><H4> 2015-08-21 </H4><UL><LI> Viewed ONCs with fluorescence microscope. Fluorescence is weak, but visible (GFP longpass).
</LI></UL><H4> 2015-08-24 </H4><UL><LI> Plated clones 1, 2, and 3 on Trp deficience plate.
</LI></UL><H4> 2015-08-26 </H4><UL><LI> Placed yeast plates in cold room.
</LI></UL><H4> 2015-08-27 </H4><UL><LI> Miniprepped plasmids with TALE constructs for V. Vyshnevskyi. 
</LI><LI> Prepped clones 5-8 of each plasmid. (12 Minis). Used the Promega Miniprep System. 
</LI></UL><H4> 2015-09-01 </H4><UL><LI> Designed qPCR primers for YFP and Tub1 with IDT PrimerQuest. Ordered the Primers from IDT.
</LI></UL><H4> 2015-09-02 </H4><UL><LI> Prepared YEPD liquid medium and -Trp -Ura -Leu SC-Agar for 0.5l each. Did not add water yet. 
</LI><LI> No Drop-out mix was added to SC mix because there was no glycin.
</LI></UL><H4> 2015-09-03 </H4><UL><LI> Added Glycin (2g) to Drop-out mix), mixed thoroughly.
</LI><LI> Added 1g Drop-out mix to SC mix
</LI><LI> Filled to 500ml with MilliQ H2O. Portioned YEPD (3 bottles, approx. 180ml each).
</LI><LI> Autoclaved. # <I> Note: Do not add glucose to YEPD before autoclaving. It will caramelize. Fortunately, Yeast grows fine in caramelized YEPD, too </I>.
SC-agar: Poured 18 plates.
</LI><LI> Inoculated 5 overnight cultures of YIYFP1 yeast in YEPD. Incubated at 28°C and 180RPM.
</LI></UL><H4> 2015-09-04 </H4><UL><LI> Measured OD600 of one ONC: OD600 = 1.429 
</LI><LI> Transferred two ONCs into 40ml YEPD. OD600 = 0.394. Added YEPD until volume was approx. 50ml. OD600 = 0.311
</LI><LI> Yeast transformation: followed small scale yeast transformation protocol. Transformed the following constructs: Rpd3 + TALE1; Rpd3 + TALE2; Rpd3 + TALE3; Rpd3 + pTUM; Negative control
</LI><LI> Plated 40µl of each transformation and additionally 200µl of Rpd+TALE1 on SC (-Ura -Leu –Trp) plates.
</LI><LI> Also plated on –Trp plate to test if Trp1 is still there.
</LI><LI> Mixed ingredients SC –Ura –Leu –Trp medium. Did not add water yet.
</LI></UL><H4> 2015-09-07 </H4><UL><LI> Yeast transformation. Colonies on all plates except negative control.
</LI><LI> SC medium: added 225ml MQ and mixed. Medium was autoclaved. 25ml 20% Glucose were added after autoclaving.
</LI><LI> Inoculated ONCs: 	2x YIYFP1 yeast in YEPD (3ml); 3 clones each of Rpd3+TALE1, Rpd3+TALE2, Rpd3+TALE3, In 3ml SC Medium (-Ura –Leu – Trp)
</LI><LI> Sent in minipreps for sequencing (TALE: clones 1-8, 2-5, 3-7 in pTUM12 and Rpd3 in YEplac181)
</LI></UL><H4> 2015-09-08 </H4><UL><LI> Viewed ONCs with fluorescence microscope. Transformed strains glow, positive control doesn’t.
</LI><LI> Mixed –Trp SC medium (added 0.095g Leu and 0.019g Ura to triple deficient SC)
</LI><LI> Did not add water.
</LI><LI> Inoculated 2x YIYFP1 in YEPD
</LI><LI> Also plated all ONCs on –Trp plates (test: does YIYFP1 survive?)
</LI></UL><H4> 2015-09-09 </H4><UL><LI> Yeast transformation: controls with only one plasmid (should not survive triple deficient SC plates) and TALE1/YEplac181 (should survive)
</LI><LI> Overnight cultures: 2 clones each of Rpd3/TALE1, Rpd3/TALE2, Rpd3/TALE3  in triple deficient SC; YIYFP1: 2 ONCs in –Trp SC; 699 in SC full medium; Negative control transformation (no plasmids) in –Trp SC
</LI></UL></DIV><DIV class="accordionTitel"> Generation of TALE and beginning of generating RPD3 constructs  </DIV><DIV class="pane" style="height:6500px;"><H4>15/06/30</H4><B>Primer/rpd3 sequence fragments:</B><UL><LI>centrifugation (5'; 13,000rpm): RPD3 sequences and primer
</LI><LI>adjust forward and reverse primer to 100µM (ddH2O) ? working concentration: 5µM (10µl primer + 190µl ddH2O)
</LI><LI>adjust RPD3 fragments to 10ng/µl (TE buffer) ? mixing ? incubation (50°C, 20') ? mixing
</LI></UL><B>PCR</B><UL><LI>ddH2O: 28.5 µl		
</LI><LI>rdp3 fragment 2.1: 1 µl		  
</LI><LI>rdp3 fragment 2.2: 1 µl		 
</LI><LI>dNTP: 4 µl		
</LI><LI>Phusion-Polymerase: 0.5 µl		
</LI><LI>Phusion-Buffer (5x): 10 µl		
</LI><LI>Primer rpd3_fw (5µM): 2.5 µl		
</LI><LI>Primer rpd3_rv (5µM): 2.5 µl		
</LI></UL><P>negative control:</P><UL><LI>ddH2O: 30.5 µl		
</LI><LI>dNTP: 4 µl		
</LI><LI>Phusion-Polymerase: 0.5 µl		
</LI><LI>Phusion-Buffer (5x): 10 µl		
</LI><LI>Primer rpd3_fw (5µM): 2.5 µl		
</LI><LI>Primer rpd3_rv (5µM): 2.5 µl		
</LI><LI>PCR Program 1 + 2
</LI><LI>10µl negativ control-sample + 2µl loading dye
</LI><LI>whole PCR tube sample + 8µl loading dye
</LI><LI>130 V,  45 min
</LI><LI>cut out both 1718bp fragments
</LI><LI>weigh fragments:  fragment PCR programm 1: 0.2g	   fragment PCR programm 2: 0.38g
</LI></UL><B>DNA fragment extraction (protocol Qiagen gel extraction kit)</B><UL><LI>eluate extracted DNA in 30µl ddH2O
</LI><LI>control 1%-Agarosegel: 3µl elution + 2µl loading dye
</LI></UL><B>Ligation</B><UL><LI>of rpd3 fragment (PCR program 2) into PCR Blunt/StuI
</LI></UL><H4>15/07/01</H4><B>Transformation</B><UL><LI>Transformation protocol 1
</LI></UL><B>Golden Gate</B><UL><LI>adjust TALE fragments to 10ng/µl (TE buffer) ? mixing ? incubation (50°C, 20') ? mixing
</LI><LI>each TAL two samples (? 6 samples)
</LI><LI>TALE
</LI><LI>1. CMV127_1	+	CMV127_2	+	 TALE-CD		(= 2416 bp)
</LI><LI>2. CMV191_1	+	CMV191_2	+	 TALE-CD		(= 2416 bp)
</LI><LI>3. GGGT_1	+	GGGT_2	        +	 TALE-CD		(= 2416 bp)
</LI><LI>long protocol in ligase buffer
</LI><LI>sample:
</LI><LI>T4 ligase buffer (10x): 2 µl			
</LI><LI>BSA: 2 µl		
</LI><LI>T4 Ligase: 1 µl		
</LI><LI>BsmBI (Esp III): 0.5 µl		
</LI><LI>3 TAL fragments	each: 5.2 µl		
</LI><LI>ddH2O: ad 20 µl
</LI><LI>PCR Program 3
</LI></UL><B>Vector pUC19</B><UL><LI>concentration measurment (NanoDrop): ~ 3µg/µl
</LI><LI>digestion
</LI></UL><B>Golden Gate part 2: control gel</B><UL><LI>10µl TALE sample (1, 2 or 3) + 2µl loading dye
</LI><LI>1% Agarose-TAE-Gel
</LI></UL><H4>15/07/02</H4><P>no white colonies</P><B>Transformation</B><UL><LI>Transformation protocol 1 (Repetition rpd3) 
</LI></UL><B>Repetition pf T4 Ligation</B><UL><LI>rpd3 fragment PCR programm 1
</LI></UL><B>Golden Gate</B><UL><LI>combine equal samples (6 samples ? 3 samples)
</LI><LI>add 0.5µl BsmBI to each sample, incubate 1h, 55°C
</LI><LI>add 0.5µl Ligase  to each sample
</LI><LI>PCR Program 4
</LI></UL><B>1% Agarose Gel</B><H4>15/07/06</H4><B>Gel extraction</B><UL><LI>TALEs and pUC19 (protocol Qiagen gel extraction kit)
</LI></UL><B>Concentration measurement (NonoDrop)</B><UL><LI>pUC19: 0.9 ng/µl 
</LI><LI>pUC19: 5.4 ng/µl
</LI><LI>TAL1: 2.5 ngµl
</LI><LI>TAL2: 2.0 ng/µl
</LI><LI>TAL3: 1.4 ng/µl
</LI></UL><B>Ligation</B><UL><LI>of TALE in pUC19
</LI></UL><B>Fluid culture</B><UL><LI>12 test tubes: á 2ml LB medium with Kanamycin
</LI><LI>add one white colonie to each tube
</LI><LI>incubation: 300rpm, over night, 37°C
</LI></UL><H4>15/07/07</H4><B>Transformation</B><UL><LI>Transformation protocol 2 (3x TALE)
</LI></UL><H4>15/07/08</H4><B>Digestion</B><UL><LI>of rpd3
</LI></UL><B>Fluid culture</B><UL><LI>of TALE
</LI><LI>6 test tubes: á 2ml LB medium with Kanamycin
</LI><LI>add one white colony to each tube (plate1: 2 colonies; plate2: 3colonies; plate3: 1 colony) 
</LI><LI>incubation: 300rpm, over night, 37°C
</LI></UL><B>Gel-electrophorese</B><UL><LI>15µl digestion probe + 3µl loading dye
</LI></UL><B>Sequencing</B><UL><LI>10µl Klon7 + 20µl ddH2O		(M13-RP, GATC IJ1672)
</LI></UL><H4>15/07/09</H4><B>Golden Gate (2)</B><UL><LI>T4 ligase buffer (10x): 2 µl			
</LI><LI>BSA: 2 µl		
</LI><LI>T4 Ligase: 1 µl		
</LI><LI>BsmBI (Esp III): 0.5 µl		
</LI><LI>TAL fragments: each 5.2 µl		
</LI><LI>ddH2O ad 20 µl
</LI><LI>PCR Program 5
</LI></UL><B>Miniprep (1)</B><UL><LI>culture sample ? protocol „Miniscreens“
</LI></UL><B>Digestion</B><UL><LI>of the mini-prep
</LI><LI>with BamHI and SacI
</LI></UL><B>Agarosegel (2)</B><UL><LI>of the Golden Gate
</LI></UL><B>Insert EC culture</B><UL><LI>3 clones, 2ml LB Ampr
</LI></UL><B>Digestion (2)</B><UL><LI>of the Golden Gate
</LI></UL><H4>15/07/10</H4><B>Agarosegel (test) (2)</B><UL><LI>over night digestion golden gate insert 2 
</LI></UL><B>Ligation: TALE in Vector pUC19_2</B><UL><LI>3.5x Mastermix
</LI></UL><B>Plurification</B><UL><LI>of vector 181 (linearazed) Ampr-Leu
</LI><LI>DNA plurification kit
</LI></UL><B>ReTrafo</B><UL><LI>of K801000 iGEM vector Ampr-Ura
</LI></UL><B>Miniprep EC insert</B><UL><LI>Miniscreen protocol, resuspend DNA in 40µl ddH2O
</LI></UL><B>Digestion of EC</B><UL><LI>22.5 µl DNA
</LI><LI>3 µl Tango Buffer			
</LI><LI>0.5 µl EcoRI
</LI><LI>4.5µl µl ddH2O
</LI><LI>1h at 37°C	
</LI><LI>inactiviation: 20min at 65°C
</LI></UL><B>Sequencing</B><UL><LI>10µl clone2 + 20µl ddH2O			—&gt; mutations
</LI><LI>10µl clone5 + 20µl ddH2O			—&gt; mutations
</LI></UL><B>Transformation</B><UL><LI>of Golden Gate (2)
</LI></UL><B>Fluid culture</B><UL><LI>prepare 12 tubes with 2.5ml LB + Amp (4 for each insert) for the weekend
</LI></UL><H4>15/07/13</H4><B>Digestion</B><UL><LI>of EC Plasmid-DNA + iGEM vector + 181 vector
</LI></UL><B>Miniprep Golden Gate (2)</B><UL><LI>4 samples of each insert
</LI><LI>protocoll „Miniscreens“
</LI><LI>solve in 40µl ddH2O 
</LI></UL><B>Digestion</B><UL><LI>of Golden Gate (2)
</LI></UL><B>Agarose gel</B><UL><LI>of EC + 181 + iGEM vector
</LI><LI>+ 5µl Loading Dye
</LI><LI>complete digestion (30µl)
</LI></UL><B>Plurification</B><UL><LI>EC1 + 2 —&gt; plurification kit
</LI></UL><B>Ligation</B><UL><LI>EC2 ligation into 181 and iGEM
</LI></UL><B>Agarose gel</B><UL><LI>2µl Golden Gate digestion sample + 0.5µl Loading Dye
</LI></UL><B>Sequencing</B><UL><LI>Insert 2_2: 3µl + 17µl ddH2O 
</LI></UL><B>Golden Gate (3)</B><UL><LI>T4 ligase buffer (10x): 2 µl				
</LI><LI>BSA: 2 µl			
</LI><LI>T4 Ligase: 1 µl		
</LI><LI>BsmBI (Esp III): 0.5 µl			
</LI><LI>TAL fragments: each 5.2 µl		
</LI><LI>ddH2O ad 20 µl
</LI><LI>PCR Program 5
</LI></UL><H4>15/07/14</H4><B>Addition of ligase (3)</B><UL><LI>because golden gate was not optimal

</LI><LI>T4 ligase buffer (10x; new): 2 µl						
</LI><LI>T4 Ligase: 1 µl		
</LI><LI>BsmBI (Esp III): 0.5 µl			
</LI><LI>Golden Gate reaction: each 10 µl		
</LI><LI>ddH2O: 8.5 µl
</LI><LI>40' at room temperature
</LI></UL><B>Digestion (3)</B><UL><LI>of Golden Gate-reaction
</LI><LI>with BamHI and SacI
</LI><LI>Inactivation of the enzymes for 20 min at 80°C
</LI></UL><B>Transformation</B><UL><LI>of the ligation
</LI></UL><B>Agarose gel</B><UL><LI>of the digested Golden Gate reaction
</LI><LI>1/5 of the digestion with 1.5 µl Loading Dye  
</LI></UL><B>Sequencing</B><UL><LI>Clone 10 + Clone 11 of rpd3
</LI></UL><B>Fluid culture</B><UL><LI>one white colony of the ReTrafo of the iGEM vector in 20ml LB-medium
</LI></UL><B>Ligation (3)</B><UL><LI>of the Golden Gate reaction into pUC19
</LI></UL><H4>15/07/15</H4><B>Transformation</B><UL><LI>of the Golden Gate ligation
</LI></UL><B>Mixi-Prep</B><UL><LI>of the iGEM vector
</LI><LI>protocol „mixi-prep“
</LI></UL><B>Ligation</B><UL><LI>of EC2-fragment into 181-vector
</LI><LI>because there were no white colonies last time, we do a new ligation
</LI></UL><B>Fluid culture</B><UL><LI>of 6 clones of EC2 + iGEM-vector
</LI></UL><B>Transformation</B><UL><LI>of the rpd3-ligation
</LI></UL><H4>15/07/16</H4><B>Mini-Prep</B><UL><LI>of EC + iGEM-vector
</LI><LI>protocol „miniscreens“
</LI></UL><B>Transformation</B><UL><LI>of EC2 + 181-vector ligation (of 15/07/07)
</LI><LI>and Golden Gate ligation (of 15/07/14)
</LI></UL><B>Digestion</B><UL><LI>of the mini-prep
</LI></UL><B>Plating</B><UL><LI>of the transformation onto LB-plates containing ampicillin, XGal and IPTG
</LI></UL><B>Agarose gel</B><UL><LI>of the digestion 
</LI><LI>add 4µl of Loading Dye
</LI></UL><H4>15/07/17</H4><B>Digestion</B><UL><LI>of the Golden Gate reaction (of 15/07/13), because no colonies grew
</LI><LI>first incubate 1h at 37°C with only SacI
</LI><LI>then add BamHI and incubate for another hour
</LI></UL><B>Digestion</B><UL><LI>of the 181-vector, because there were only blue colonies on the plates
</LI></UL><B>Fluid culture</B><UL><LI>of rpd3 —&gt; 12 clones
</LI><LI>and of 2 blue clones of EC + 181 to test them
</LI></UL><B>Ligation</B><UL><LI>of Golden Gate into pUC19
</LI><LI>of EC into 181-vector
</LI></UL><H4>15/07/20</H4><B>Mini-prep</B><UL><LI>of the fluid cultures from 15/07/17
</LI><LI>protocol „miniscreens“
</LI></UL><B>Transformation</B><UL><LI>of the ligations from 15/07/17
</LI></UL><B>Digestion</B><UL><LI>of the mini-preps
</LI></UL><B>Agarose gel</B><UL><LI>of the digestion
</LI><LI>rpd3: 15µl with 3µl loading dye
</LI><LI>EC/181: 25µl with 5µl loading dye
</LI></UL><B>Sequencing</B><UL><LI>of 2 rpd3 clones
</LI></UL><H4>15/07/22</H4><B>Fluid culture</B><UL><LI>of 4 clones for Golden Gate
</LI><LI>and 4 clones of EC + 181
</LI></UL><H4>15/07/23</H4><B>Mini-prep</B><UL><LI>of the fluid cultures
</LI><LI>protocol „miniscreens“
</LI></UL><B>Digestion</B><UL><LI>of the mini-preps
</LI></UL><B>Agarose Gel</B><UL><LI>of the digestion
</LI><LI>with 3µl loading dye
</LI></UL><B>Overlap-PCR</B><UL><LI>of rpd3
</LI><LI>PCR program of 15/06/30
</LI></UL><B>Sequencing</B><UL><LI>of Golden Gate fragment 1 and 3 
</LI></UL><H4>15/07/24</H4><B>Agarose gel</B><UL><LI>of the overlap-PCR
</LI></UL><B>Gel extraction</B><UL><LI>with Qiagen Gel Extraction Kit
</LI></UL><B>Agarose gel</B><UL><LI>to control the gel extraction
</LI></UL><B> Ligation </B><UL><LI>of rpd3 into PCR/Blunt
</LI><LI>protocol of 15/06/30
</LI></UL></DIV><DIV class="accordionTitel"> Generation of RPD3 constructs </DIV><DIV class="pane" style="height:11000px;"><H4>2015/07/27</H4><UL><LI>Transformation of plasmid pCR-Blunt and insert rpd3		</LI><LI>TP1</LI></UL><H4>2015/07/28</H4><UL><LI>Inoculation of plasmid pCR-Blunt and insert rpd3</LI><LI>IP1		</LI></UL><H4>2015/07/29</H4><UL><LI>Twelve plasmid preparations of plasmid pCR-Blunt and insert rpd3</LI><LI>MP1</LI><LI>Twelve digestions of plasmid pCR-Blunt and insert rpd3</LI><LI>DP1</LI><LI>Gel electrophoresis of the digestion of plasmid pCR-Blunt and insert rpd3</LI><LI>GP1</LI></UL><H4>2015/07/30</H4><UL><LI>Sequencing of samples 2 and 5 of the plasmid preparation of plasmid pCR-Blunt and insert rpd3 of 2015/07/29</LI><LI>SP1</LI><LI>Ligation of plasmid YEplac181, digested on 2015/07/17, and insert EC1, purified on 2015/07/13</LI><LI>LP1</LI></UL><H4>2015/07/31</H4><UL><LI>Transformation of plasmid YEplac181 and insert EC		</LI><LI>TP2</LI></UL><H4>2015/08/01</H4><UL><LI>Incubation of transformation of plasmid YEplac181 and insert EC over night at 37°C</LI></UL><H4>2015/08/02</H4><UL><LI>Inoculation of plasmid YEplac181 and insert EC</LI><LI>IP2 </LI></UL><H4>2015/08/03</H4><UL><LI>Twelve plasmid preparations of plasmid YEplac181 and insert EC</LI><LI>MP1</LI><LI>Twelve digestions of plasmid YEplac181 and insert EC</LI><LI>DP2</LI><LI>Gel electrophoresis of the digestion of plasmid YEplac181 and insert EC</LI><LI>GP1</LI><LI>Sequencing of sample 3 of the plasmid preparation of plasmid YEplac181 and insert EC</LI><LI>SP2</LI></UL><H4>2015/08/05</H4><UL><LI>Sequencing of sample 8 of the plasmid preparation of plasmid YEplac181 and insert EC</LI><LI>SP1</LI></UL><H4>2015/08/07</H4><UL><LI>Two digestions of the plasmid preparation of plasmid pUC19 and inserts TALE1 and TALE3 of 2015/07/23</LI><LI>DP3</LI><LI>Gel electrophoresis of the digestion of plasmid pUC19 and inserts TALE1, TALE2, and TALE3 of 2015/07/23 and today</LI><LI>GP2</LI><LI>Three digestions of the digestion of plasmid YEplac181 of 2015/07/17 and the samples 2 and 6 of the plasmid preparation of plasmid YEplac181 and insert EC of 2015/08/03</LI><LI>DP4</LI><LI>Gel electrophoresis of the digestion of the digestion of plasmid YEplac181 and the samples 2 and 6 of the plasmid preparation of plasmid YEplac181 and insert EC</LI><LI>GP1</LI></UL><H4> 2015/08/10</H4><UL><LI>Four digestions of sample 6 of the plasmid preparation of plasmid BBa_K801000 and insert EC of 2015/07/16 and samples 3, 5, and 8 of the plasmid preparation of plasmid YEplac181 and insert EC of 2015/08/03</LI><LI>DP5</LI><LI>Gel extraction of the gel slices of 2015/08/07</LI><LI>EP1</LI><LI>Achievement: obtain all TALE inserts</LI><LI>Sequencing of sample 3 of the plasmid preparation of plasmid YEplac181 and insert EC of 2015/08/03</LI><LI>SP3 </LI></UL><H4>2015/08/11</H4><UL><LI>Gel electrophoresis of the digestion of plasmid BBa_K801000 with insert EC and plasmid YEplac181 with insert EC of 2015/08/10</LI><LI>GP1</LI><LI>Four digestions of sample 6 of the plasmid preparation of plasmid BBa_K801000 and insert EC of 2015/07/16 and samples 3, 5, and 8 of the plasmid preparation of plasmid YEplac181 and insert EC of 2015/08/03</LI><LI>DP6</LI><LI>Gel electrophoresis of the digestion of plasmid BBa_K801000 with insert EC and plasmid YEplac181 with insert EC of today</LI><LI>GP1</LI></UL><H4>2015/08/12</H4><UL><LI>Mass digestion of sample 6 of the plasmid preparation of plasmid BBa_K801000 and insert EC of 2015/07/16 and samples 3, 5, and 8 of the plasmid preparation of plasmid YEplac181 and insert EC of 2015/08/03</LI><LI>DP7</LI><LI>Gel electrophoresis of the mass digestion of plasmid BBa_K801000 with insert EC and plasmid YEplac181 with insert EC</LI><LI>GP1</LI><LI>Retransformation of sample 6 of the plasmid preparation of plasmid BBa_K801000 and insert EC of 2015/07/16</LI><LI>TP3</LI></UL><H4> 2015/08/13</H4><UL><LI>Digestion of sample 2 of the plasmid preparation of plasmid YEplac181 and insert EC of 2015/08/03</LI><LI>DP8</LI><LI>Gel electrophoresis of the digestion of plasmid YEplac181 with insert EC</LI><LI>GP1</LI><LI>Inoculation of the retransformation of plasmid BBa_K801000 and insert EC of 2015/08/12</LI><LI>IP3		</LI></UL><H4>2015/08/14</H4><UL><LI>Twelve plasmid preparations of plasmid BBa_K801000 and insert EC</LI><LI>MP1</LI><LI>Four digestions of samples 1, 2, 7, and 8 of plasmid BBa_K801000 and insert EC</LI><LI>DP2</LI><LI>Gel electrophoresis of the digestion of plasmid BBa_K801000 with insert EC</LI><LI>GP1</LI><LI>Sequencing of sample 2 of the plasmid preparation of plasmid YEplac181 and insert EC of 2015/08/03</LI><LI>SP1</LI><LI>Sequencing of sample 5 of the plasmid preparation of plasmid BBa_K801000 and insert EC of 2015/07/16</LI><LI>SP6</LI></UL><H4>2015/08/18</H4><UL><LI>Sequence of sample 5 of the plasmid preparation of plasmid BBa_K801000 and insert EC of 2015/07/16 is correct</LI></UL><B>Achievement: obtain the insert EC in the plasmid BBa_K801000 with a confirmed sequence</B><UL><LI>Digestion of sample 5 of the plasmid preparation of plasmid BBa_K801000 and insert EC of 2015/07/16</LI><LI>DP9</LI><LI>Five digestions of samples 1, 2, 4, 5, and 6 of the plasmid preparation of plasmid BBa_K801000 and insert EC of 2015/07/16</LI><LI>DP10</LI><LI>Gel electrophoresis of both digestions of plasmid BBa_K801000 with insert EC</LI><LI>GP3</LI><LI>Gel extraction</LI><LI>EP1</LI></UL><B>Achievement: obtain the insert EC with a confirmed sequence</B><UL><LI>Reinoculation of sample 5 of bacteria culture with plasmid BBa_K801000 and insert EC of 2015/07/15</LI><LI>IP4</LI></UL><H4>2015/08/19</H4><UL><LI>Gel electrophoresis of gel extraction of 2015/08/18 and several samples of the plasmid YEplac181</LI><LI>GP1</LI><LI>Ligation of plasmid YEplac181, purified on 2015/07/10, and insert EC, purified on 2015/08/18</LI><LI>LP2</LI><LI>Three plasmid preparations of plasmid BBa_K801000 and insert EC of bacteria culture of 2015/08/18</LI><LI>MP2</LI><LI>Three digestions of plasmid preparations of plasmid BBa_K801000 and insert EC</LI><LI>DP11</LI><LI>Gel electrophoresis of the digestion of three plasmid preparations of plasmid BBa_K801000 and insert EC</LI><LI>GP1</LI></UL><H4>2015/08/20</H4><B>Thanks to Marlene P., we have obtained a construct of plasmid pCR-Blunt and insert rpd3 with a confirmed sequence</B><B>Achievement: get someone else to do your work</B><UL><LI>Digestion of plasmid pCR-Blunt with insert rpd3 and sample 1 of  the plasmid preparation of plasmid BBa_K801000 with insert EC of 2015/08/19</LI><LI>DP12</LI><LI>Gel electrophoresis of the digestion of plasmid pCR-Blunt with insert rpd3 and sample 1 of  the plasmid preparation of plasmid BBa_K801000 with insert EC and the three TALE inserts purified on 2015/08/10</LI><LI>GP4</LI><LI>Gel extraction</LI><LI>EP1</LI></UL><B>Achievement: obtain the insert rpd3 with a confirmed sequence</B><UL><LI>Transformation of the ligation of plasmid YEplac181 and insert EC of 2015/08/19	</LI><LI>TP4</LI></UL><H4>2015-08-21</H4><UL><LI>Digestion of the plasmid preparation of plasmid BBa_K801000 and insert EC performed by Filip C. on 2015/08/19</LI><LI>DP13 </LI><LI>Gel electrophoresis of the digestion of plasmid BBa_K801000 and insert EC, the gel extraction of insert rpd3 and linearized plasmid BBa_K801000 with insert EC purified on 2015/08/20, and insert TALE1 purified on 2015/08/10</LI><LI>GP5</LI><LI>Gel extraction</LI><LI>EP1</LI><LI>Ligation of plasmid BBa_K801000 with insert EC, purified on 2015/08/20, and insert rpd3, purified on 2015/08/20</LI><LI>LP3</LI><LI>Inoculation of plasmid YEplac181 and insert EC</LI><LI>IP5</LI></UL><H4>2015/08/23</H4><UL><LI>Incubation of the inoculation of plasmid YEplac181 and insert EC of 2015/08/21 over night (37°C, 45rpm)</LI></UL><H4>2015/08/24</H4><UL><LI>Six plasmid preparations of plasmid YEplac181 and insert EC of 2015/08/21 using the PureYield™ Plasmid Miniprep System</LI><LI>MP3</LI><LI>Six digestions of the plasmid preparation of plasmid YEplac181 and insert EC</LI><LI>DP14</LI><LI>Gel electrophoresis of the digestion of plasmid YEplac181 with insert EC, the linearized plasmid BBa_K801000 with insert EC, purified on 2015/08/21, and the insert TALE1, purified on 2015/08/21</LI><LI>GP1</LI><LI>Transformation of plasmid BBa_K801000 with insert EC and insert rpd3		</LI><LI>TP5 </LI><LI>Ligation of plasmid BBa_K801000 with insert EC, purified on 2015/08/20, and insert TALE1, purified on 2015/08/21, and inserts TALE2 and TALE3, both purified on 2015/08/10</LI><LI>LP3</LI></UL><H4>2015/08/25</H4><UL><LI>Sequencing of sample 1 of the plasmid preparation of plasmid YEplac181 and insert EC</LI><LI>SP1</LI><LI>Inoculation of plasmid BBa_K801000 with insert EC and insert rpd3</LI><LI>IP6</LI><LI>Transformation of plasmid BBa_K801000 with insert EC and inserts TALE1, TALE2, and TALE3	</LI><LI>TP6</LI></UL><H4>2015/08/26</H4><UL><LI>Sequence of sample 1 of the plasmid preparation of plasmid YEplac181 and insert EC of 2015/08/24 is correct</LI></UL><B>Achievement: obtain the insert EC in the plasmid YEplac181 with a confirmed sequence</B><UL><LI>Inoculation of plasmid BBa_K801000 with insert EC and inserts TALE1, TALE2, and TALE3 of 2015/08/25</LI><LI>IP7</LI><LI>Six plasmid preparations of plasmid BBa_K801000 with insert EC and insert rpd3 of 2015/08/25 using the PureYield™ Plasmid Miniprep System</LI><LI>MP3</LI><LI>Seven digestions of the plasmid preparation of plasmid BBa_K801000 with insert EC and insert rpd3 and of sample 3 of the plasmid preparation of plasmid YEplac181 and insert EC of 2015/08/24</LI><LI>DP15</LI><LI>Gel electrophoresis of the digestion of the plasmid preparation of plasmid BBa_K801000 with insert EC and insert rpd3 and of the digestion of sample 3 of the plasmid preparation of plasmid YEplac181 and insert EC</LI><LI>GP1</LI></UL><H4>2015/08/27</H4><UL><LI>Twenty-four plasmid preparations of plasmid BBa_K801000 with insert EC and inserts TALE1, TALE2, and TALE3 of 2015-08-26 using the PureYield™ Plasmid Miniprep System</LI><LI>MP4</LI><LI>Twenty-four digestions of the plasmid preparation of plasmid BBa_K801000 with insert EC and inserts TALE1, TALE2, and TALE3</LI><LI>DP16</LI><LI>Gel electrophoresis of the digestion of the plasmid preparation of plasmid BBa_K801000 with insert EC and inserts TALE1, TALE2, and TALE3</LI><LI>GP1</LI></UL><B>Achievement: obtain the finished constructs of plasmid BBa_K801000 with insert EC and inserts TALE1, TALE2, and TALE3
</B><H4>2015/08/28</H4><UL><LI>Digestion of sample 1 of the plasmid preparation of plasmid YEplac181 and insert EC of 2015/08/24</LI><LI>DP17</LI><LI>Gel electrophoresis of the digestion of plasmid YEplac181 and insert EC</LI><LI>GP6</LI><LI>Gel extraction</LI><LI>EP2</LI><LI>Ligation of plasmid YEplac181 with insert EC and insert rpd3, purified on 2015/08/20</LI><LI>LP3</LI></UL><H4>2015/08/31</H4><UL><LI>Transformation of plasmid YEplac181 with insert EC and insert rpd3 of 2015/08/28</LI><LI>TP5</LI></UL><H4>2015/09/01</H4><UL><LI>Inoculation of plasmid YEplac181 with insert EC and insert rpd3</LI><LI>IP8</LI></UL><H4>2015/09/02</H4><UL><LI>Eight plasmid preparations of plasmid YEplac181 with insert EC and insert rpd3 of 2015/09/01 using the PureYield™ Plasmid Miniprep System</LI><LI>MP3</LI><LI>Eight digestions of the plasmid preparation of plasmid YEplac181 with insert EC and insert rpd3</LI><LI>DP18</LI><LI>Gel electrophoresis of the digestion of the plasmid preparation of plasmid YEplac181 with insert EC and insert rpd3</LI><LI>GP1</LI></UL><B>Achievement: obtain the finished construct of plasmid YEplac181 with insert EC and insert rpd3
</B><H4>2015/09/03</H4><UL><LI>Digestion of the linearized TALE constant domain (CD), sample 6 of the plasmid preparation of plasmid YEplac181 with insert EC and insert rpd3 of 2015/09/02, and samples 18, 25, and 37 of the plasmid preparations of plasmid BBa_K801000 with insert EC and inserts TALE1, TALE2, and TALE3 of 2015/08/27</LI><LI>DP19</LI><LI>Gel electrophoresis of the digestion of sample 6 of the plasmid preparation of plasmid YEplac181 with insert EC and insert rpd3 and samples 18, 25, and 37 of the plasmid preparations of plasmid BBa_K801000 with insert EC and inserts TALE1, TALE2, and TALE3</LI><LI>GP1</LI><LI>Purification of the digestion of CD</LI><LI>PP1</LI><LI>Ligation of plasmid BBa_K801000 with insert EC, purified on 2015/08/20, and insert CD</LI><LI>LP3</LI></UL><H4>2015/09/04</H4><UL><LI>Transformation of plasmid BBa_K801000 with insert EC and insert CD of 2015/08/28</LI><LI>TP7</LI><LI>Reinoculation of sample 6 of the plasmid preparation of plasmid YEplac181 with insert EC and insert rpd3 of 2015/09/02, and samples 18, 25, and 37 of the plasmid preparations of plasmid BBa_K801000 with insert EC and inserts TALE1, TALE2, and TALE3 of 2015/08/27</LI><LI>IP9</LI></UL><H4>2015/09/06</H4><UL><LI>Incubate the transformation of plasmid BBa_K801000 with insert EC and insert CD and the reinoculation of sample 6 of the plasmid preparation of plasmid YEplac181 with insert EC and insert rpd3, and samples 18, 25, and 37 of the plasmid preparations of plasmid BBa_K801000 with insert EC and inserts TALE1, TALE2, and TALE3, all four samples of 2015/08/27, at 37°C over night</LI></UL><H4>2015/09/07</H4><UL><LI>Four plasmid preparations of plasmid YEplac181 with insert EC and insert rpd3 and plasmid BBa_K801000 with insert EC and inserts TALE1, TALE2, and TALE3 of 2015-08-21 using the PureYield™ Plasmid Miniprep System</LI><LI>MP3</LI><LI>Four digestions of the plasmid preparation of plasmid YEplac181 with insert EC and insert rpd3 and of the plasmid preparation of plasmid BBa_K801000 with insert EC and inserts TALE1, TALE2, and TALE3</LI><LI>DP20</LI><LI>Gel electrophoresis of the digestion of the plasmid preparation of plasmid YEplac181 with insert EC and insert rpd3 and samples 18, 25, and 37 of the plasmid preparations of plasmid BBa_K801000 with insert EC and inserts TALE1, TALE2, and TALE3</LI><LI>GP1</LI><LI>Inoculation of plasmid BBa_K801000 with insert EC and insert CD using the 1:100 dilution plate</LI><LI>IP6</LI><LI>Sequencing of the plasmid preparation of plasmid YEplac181 with insert EC and insert rpd3 and of the plasmid preparation of plasmid BBa_K801000 with insert EC and inserts TALE1, TALE2, and TALE3</LI><LI>SP4</LI></UL><H4>2015/09/08</H4><UL><LI>Six plasmid preparations of plasmid BBa_K801000 with insert EC and insert CD using the PureYield™ Plasmid Miniprep System</LI><LI>MP3</LI><LI>Six digestions of the plasmid preparation of plasmid BBa_K801000 with insert EC and insert CD</LI><LI>DP20</LI><LI>Gel electrophoresis of the digestion of the plasmid preparation of plasmid BBa_K801000 with insert EC and insert CD</LI><LI>GP1</LI><LI>Sequencing of the plasmid preparation of plasmid YEplac181 with insert EC and insert rpd3 and of the plasmid preparation of plasmid BBa_K801000 with insert EC and inserts TALE1, TALE2, and TALE3, all samples of 2015/09/07</LI><LI>SP5</LI></UL><H4>2015/09/09</H4><UL><LI>Eight digestions of sample 1 of the plasmid preparation of plasmid BBa_K801000 and insert EC of 2015/08/19, 
	 sample 6 of the plasmid preparation of  plasmid YEplac181 with insert EC and insert rpd3 of 2015/09/02,
	 samples 18, 25, and 37 of the plasmid preparations of plasmid BBa_K801000 with insert EC and inserts TALE1, TALE2, and TALE3, respectively, of 2015/08/27,
	 sample 6 of the plasmid preparation of plasmid BBa_K801000 with insert EC and insert CD of 2015/09/08, 
	  a construct with plasmid YIplac204 and insert YFP received from Filip C., and the plasmid pSB1C3 from the Submission Kit</LI><LI>DP21</LI><LI>Gel electrophoresis of the digestion of sample 1 of the plasmid preparation of plasmid BBa_K801000 and insert EC of 2015/08/19, 
	sample 6 of the plasmid preparation of  plasmid YEplac181 with insert EC and insert rpd3 of 2015/09/02, 
	samples 18, 25, and 37 of the plasmid preparations of plasmid BBa_K801000 with insert EC and inserts TALE1, TALE2, and TALE3, respectively, of 2015/08/27,
	 sample 6 of the plasmid preparation of plasmid BBa_K801000 with insert EC and insert CD of 2015/09/08,
	 a construct with plasmid YIplac204 and insert YFP received from Filip C., and the plasmid pSB1C3 from the Submission Kit</LI><LI>GP1</LI><LI>Purification of the eight digestions</LI><LI>PP2</LI><LI>Ligation of purified plasmid pSB1C3 and inserts rpd3, CD, TALE1, TALE2, and TALE3, all in insert EC, as well as inserts EC and YFP</LI><LI>LP4</LI></UL><H4>2015/09/10</H4><UL><LI>Seven transformation of plasmid pSB1C3 with inserts YFP, EC, and inserts rpd3, CD, TALE1, TALE2, and TALE3 in insert EC		</LI><LI>TP8 </LI></UL><H4>2015/09/11</H4><UL><LI>Inoculation of plasmid pSB1C3 with inserts YFP, EC, and inserts rpd3, CD, TALE1, TALE2, and TALE3 in insert EC</LI><LI>IP10</LI></UL><H4>2015/09/13</H4><UL><LI>Incubate 42 inoculations of 2015/09/11 over night (37°C, 45rpm)</LI></UL><H4>2015/09/14</H4><UL><LI>42 plasmid preparations of plasmid pSB1C3 with inserts EC, YFP, and inserts rpd3, CD, TALE1, TALE2, and TALE3, all in insert EC, using the PureYield™ Plasmid Miniprep System</LI><LI>MP3</LI><LI>42 digestions of the plasmid preparation of plasmid pSB1C3 with inserts EC, YFP, and inserts rpd3, CD, TALE1, TALE2, and TALE3, all in insert EC</LI><LI>DP22</LI><LI>Gel electrophoresis of the digestion of the plasmid preparation of plasmid pSB1C3 with inserts EC, YFP, and inserts rpd3, CD, TALE1, TALE2, and TALE3, all in insert EC</LI><LI>GP1</LI></UL><B>Achievement: obtain a sample of TALE2 in EC good enough to submit</B><B>Achievement: obtain a sample of YFP in EC good enough to submit</B><UL><LI>Inoculation of plasmid pSB1C3 with insert  EC and inserts rpd3, CD, TALE1, and TALE3 in insert EC of 2015/09/10</LI><LI>IP11</LI><LI>7 digestions of the most promising samples of each insert: sample 2 of TALE2, sample 1 of YFP, sample 1 of CD, sample 6 of rpd3, sample 6 of EC, sample 3 of TALE1 and sample 2 of TALE3</LI><LI>DP23</LI></UL><H4>2015/09/15</H4><UL><LI>20 plasmid preparations of plasmid pSB1C3 with insert  EC and inserts rpd3, CD, TALE1, and TALE3 in insert EC of 2015/09/14</LI><LI>MP5</LI><LI>Purification of the plasmid preparations of plasmid pSB1C3 with insert  EC and inserts rpd3, CD, TALE1, and TALE3 in insert EC</LI><LI>PP3</LI><LI>20 digestions of the purification of plasmid pSB1C3 with insert  EC and inserts rpd3, CD, TALE1, and TALE3 in insert EC</LI><LI>DP24</LI><LI>7 digestions of sample 2 of TALE2, sample 1 of YFP, sample 1 of CD, sample 6 of rpd3, sample 6 of EC, sample 3 of TALE1 and sample 2 of TALE3</LI><LI>DP25</LI><LI>Gel electrophoresis of the digestion of sample 2 of TALE2, sample 1 of YFP, sample 1 of CD, sample 6 of rpd3, sample 6 of EC, sample 3 of TALE1 and sample 2 of TALE3 of 2015/09/14, as well as all samples of both digestions of today</LI><LI>GP1</LI></UL><B>Achievement: obtain a sample of TALE3 in EC good enough to submit</B><UL><LI>Seven transformation of plasmid pSB1C3 with inserts YFP, EC, and inserts rpd3, CD, TALE1, TALE2, and TALE3 in insert EC, all digested with HindIII	</LI><LI>TP9</LI></UL><H4>2015/09/16</H4><UL><LI>40 inoculations of sample 1 of CD, sample 6 of rpd3, sample 6 of EC, sample 3 of TALE1 and sample 2 of TALE3 of 2015/09/14, as well as the transformation of plasmid pSB1C3 with inserts YFP, EC, and inserts rpd3, CD, TALE1, TALE2, and TALE3 in insert EC, all digested with HindIII, of 2015/09/15</LI><LI>IP12</LI></UL><H4> 2015/09/17</H4><UL><LI>40 plasmid preparations of the reinoculation of plasmid pSB1C3 with insert EC, and inserts rpd3, CD, TALE1, and TALE3, all in insert EC, and of the inoculation of the HindIII-digested plasmid pSB1C3 with insert EC, and inserts rpd3, CD, TALE1, and TALE3, all in insert EC</LI><LI>MP5</LI><LI>40 digestions of the plasmid preparation of plasmid pSB1C3 with insert EC, and inserts rpd3, CD, TALE1, and TALE3, all in insert EC, and of the HindIII-digested plasmid pSB1C3 with insert EC, and inserts rpd3, CD, TALE1, and TALE3, all in insert EC</LI><LI>DP26</LI><LI>Gel electrophoresis of all 40 digestions</LI><LI>GP1</LI></UL><B>Achievement: obtain a sample of TALE1 good enough to submit</B><B>Achievement: obtain a sample of CD good enough to submit</B><B>Achievement: fail to obtain a satisfactory sample of EC and rpd3 in EC before the deadline, but submit them anyway</B></DIV><DIV class="accordionTitel"> Results of the project </DIV><DIV class="pane" style="height:1000px;"><H4> 2015-09-10 </H4><UL><LI> Viewed ONCs with fluorescence microscope: 699 (negative control) shows strong fluorescence. Probably contaminated with YFP yeast. Plated on –Trp plate to test the assumption. 
</LI><LI> Picked 699 clone from fresh plate (M. Dahl) – clone shows no fluorescence. 
</LI><LI> Transferred ONCs to 50ml of the appropriate deficient SC medium. Also inoculated one clone from 699 plate (M. Dahl) in 50ml SC full medium. Additionally inoculated 699 strain in –Trp SC and –Ura –Leu –Trp SC to see if it survives. Incubation over night at 28°C and 180rpm
</LI></UL><H4> 2015-09-11 </H4><UL><LI> RNA extraction from yeast overnight cultures using the RiboPure Yeast RNA isolation Kit. Handling according to manufacturer’s instructions. Executed by F. Winter and N. Übelmesser. 
</LI><LI> A sample was taken from 699 and YIYFP1 isolates before DNase treatment to test via PCR for YFP DNA.
</LI><LI> All samples were tested with YFP PCR for residual gDNA. PCR over night.
</LI></UL><H4> 2015-09-14 </H4><UL><LI> PCR samples were loaded on a gel. Bands appeared in all samples except the 699 sample.
</LI><LI> Yeast plates from 09-09: Colonies on TALE1/YEplac181 plate, otherwise none.
</LI><LI> Mixed RT reaction according to protocol.
</LI><LI> 50µl reactions for +RT and 25µl reactions for –RT control.
</LI></UL><H4> 2015-09-15 </H4><UL><LI> Inoculated one colony of YIYFP1, 3 Rpd3/TALE constructs and 699 in SC medium at 30°C for 4 hours. Then the cultures were inspected via CLSM. All cultures show strong fluorescence. Most likely the SC medium is the cause.
</LI><LI> Tested qPCR primers: eYFP primer efficiency: 95%, tub Primer efficiency: 85%
</LI></UL><H4> 2015-09-16 </H4><UL><LI> Mixed qPCR Master mix (Promega goTaq qPCR Master Mix) for tomorrow. Test PCR of all RT reactions for YFP DNA. Bands in every sample except 699.
</LI></UL><H4> 2015-09-17 </H4><UL><LI> Pipetted all qPCR reactions on 96 well plate. qPCR reaction was successful. 
</LI><LI> Inoculated yeast ONCs for CLSM tomorrow.
</LI></UL><H4> 2015-09-18 </H4><UL><LI> Data analysis of qPCR with J. Schmidpeter. Rpd3/TALE transformed yeast strains exhibit strongly lowered levels of YFP mRNA.
</LI><LI> One more qPCR to confirm primer efficiency. The result does not differ much from the last time.
</LI><LI> CLSM: Transformed YIYFP strains appear to show less fluorescence.
</LI></UL></DIV></DIV></DIV><DIV class="footer "><DIV class="footerContainer"><A class="footerBtn footerShadow" id="btnFastForward"><P class="footerBtnPre">Next chapter

</P></A><A class="footerBtn footerShadow" id="btnNextChapter"><P class="footerBtnPre">Next chapter

</P></A><A class="footerBtn footerShadow" id="btnDigDeeper"><P class="footerBtnPre">Dig deeper

</P></A><A class="footerBtn footerShadow" id="btnTheEnd" href="http://www.igem.org" target="_blank"><P class="footerBtnPre">End of tour

</P><P class="footerBtnHeader" style="text-transform: none !important;">Want more? Go to iGEM.org!</P><P class="footerBtnText">

If you want to learn more about Synthetic Biology you should visit iGEM's homepage, which contains loads 

on the subject. You can even visit other teams' wikis and take a look at what they accomplished. Or maybe 

join a team yourself?</P></A><A class="footerBtn footerShadow" id="btnStartTour" href="https://2015.igem.org/Team:FAU_Erlangen/Tour10"><P class="footerBtnPre">A tour for everyone

</P><P class="footerBtnHeader">Click here to begin the interactive wiki-tour</P><P class="footerBtnText">Take a quick glance at our project or dig deeper to get a complete walkthrough of 

the wiki. Or do something in-between, the choice is yours.</P></A></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>