Team:FAU Erlangen/Tour22
SCATTER
Welcome
Interactive wiki-tour
Menu
Introduction
The team
Gallery
Attributions
Collaborations
Planning
Spy Tag and Spy Catcher
Deacetylase
TALE
Sequences
Matching Program
Vectors
Yeast strains
Methodology
Execution
Timetable
Lab book
Protocols
Setbacks
Results
Submitted Parts
YFP Detection
Future
The end
Introduction
The team
Gallery
Attributions
Collaborations
Planning
Spy-System
Deacetylase
TALE
Sequences
Matching program
Vectors
Yeast strains
Methodology
Execution
Timetable
Lab Book
Protocols
Setbacks
Results
Submitted Parts
YFP-Detection
Future
The end
Epigenetics
Since a Cold Spring Harbor meeting in 2008, epigenetics is defined as a stably heritable phenotype resulting from changes in a chromosome without alterations in the DNA sequence.  DNA methylations and post-translational histone modifications which alter the expression of genes but not the underlying DNA sequence were introduced. These alterations could be modulated by other modifications, caused by transcriptional or environmental factors and enzymes.  The executing histone modifying enzymes could be identified as phosphorylases, methyltransferases, acetylases and deacetylases. These enzymes catalyze covalent modifications to the N-terminal histon tails which on their part are connected with transcriptional regulation in yeast and other eukaryotes.  One example of the influence of these epigenetic alterations is the process of cellular differentiation. During eukaryotic development, stem cells have to become fully differentiated cells, such as neurons, muscle cells and epithelium. This is achieved by activating different sets of genes while inhibiting others. Therefore a variation of different histone modifications and DNA methylations are introduced during the cell cycle. These create different expression patterns and thus cellular differentiation. However, this system is quite fragile. Minor changes in the epigenetic code induced by mutated histon modifying enzymes, for example, can partially reverse epigenetically repression of genes. Some of these regulate oncogene expression and are therefore known as cancer triggers. Thus, epigenetics has nowadays become a key role in cancer research.
(De-)Acetylation regulates transcription
As mentioned, methylations, phosphorylations and acetylations can regulate transcription. In the parts ahead we will have a closer look on the acetylation of N-terminal histone tails and their ability of regulating the transcription by altering the DNA structure.
At the moment there are three different models to explain regulation via N-terminal histon tail acetylation.
Acetylation induces a conformational change in the core histones A perfect correlation between acetylation and transcriptional activity is not existing. But acetylation of charged lysine residues alters the core histone structure.  That possibly results in a facilitated transcription while it is not sufficient to induce transcription of inactive genes.
Acetylation is a transducing signal Particular acetylation of lysine residues may manipulate signals which are recognized by other factors. Thus the transcription efficiency could be improved or decreased.
Acetylation affects the activity of nonhistone protein not just histones, rather nonhistone proteins are acetylated. An important example of this kind of transcriptional regulation are the high mobility group (HMG) proteins. They are also acetylated at lysine residues. The deacetylation of these proteins could be inhibited by sodium butyrate, which is a common deacetylase inhibitor. Therefore deacetylated HMG proteins could contribute to the effects that are seen with Rpd3 and related proteins. (reviewed by Pazin and Kadonaga, 1997)
RPD3, chosing an histone deacetylase
At the beginning of the project it we found out that the idea to manipulate gene expression via epigenetic modification was not as novel as thought. A literature research aided by our instructors quickly showed that the research group around Kevin Struhl already successfully conducted experiments with the yeast endogene histone deacetylase rpd3. This is why we chose this particular deacetylase for our project.  Rpd3 encodes a histone deacetylase, which deacetylates and acetylases histones. This leads to epigenetic changes thus regulating gene expression on the transcriptional level. Furthermore, it shares 60% sequence identity with the human deacetylase. (Taunton et al 1996) In yeast rpd3 repression is usually achieved by the recruitment of two complexes: Rpd3L and  Rpd3S. However both of them contain as a core element Sin3 and Ume6. (Keogh et al 2005)  Ume 6 binds a a GC-rich sequence upstream of rpd3 controlled promoters. This region is called URS1. In an experiment the Struhl group showed that lacZ expression could be repressed by adding an URS1 up stream of a CYC1 promoter region. This artificial repression was reduced in rpd3, ume6 and sin3 deletion mutants (Kadosh and Struhl 1997).   This effect could be reproduced with his3 as a reporter gene as well as with different activator regions. However, the effect was less strong with strong activators (Deckert and Struhl 2002).  Additionally it was shown that the deacetylating effect of rpd3 was very local. Deacetylation occurred only 150-250 bp from the target sequence, which is about the distance covered by one histone (Kadosch and Struhl 1998). So target regions were only picked within this range.
Deckert and Struhl (2002): could show that if an URS segment is positioned upstream of a his3 gene (A), the mRNA-concentration sinks. This effect could not be seen in rpd3, sin3 and ume6. The reduction also was less drastical with stronger activators (B).
References:
Michael J Pazin, James T Kadonaga (1997). What's Up and Down with Histone Deacetylation and Transcription?, Cell, Volume 89 (3), Pages 325-328
Michael-Christopher Keogh, Siavash K. Kurdistani, Stephanie A. Morris, Seong Hoon Ahn, Vladimir Podolny, Sean R. Collins, Maya Schuldiner, Kayu Chin, Thanuja Punna, Natalie J. Thompson, Charles Boone, Andrew Emili, Jonathan S. Weissman, Timothy R. Hughes, Brian D. Strahl, Michael Grunstein, Jack F. Greenblatt, Stephen Buratowski, Nevan J. Krogan, (2005) Cotranscriptional Set2 Methylation of Histone H3 Lysine 36 Recruits a Repressive Rpd3 Complex, Cell, 124(4),  Pages 593-605
Taunton, J., Hassig, C.A., and Schreiber, S.L. (1996). A mammalian histone deacetylase related to the yeast transcriptional regulator Rpd3p. Science 272, 408-411
Kadosch, D. and Struhl, K. (1997) Repression by Ume6 Involves Recruitment of a Complex Conataining Sin3 Corepressor and Rpd3 Histone Deacetylase to Target Promoters, Cell 89, 365-371
Kadosh, D., & Struhl, K. (1998). Targeted Recruitment of the Sin3-Rpd3 Histone Deacetylase Complex Generates a Highly Localized Domain  of Repressed Chromatin In Vivo. Molecular and Cellular Biology, 18(9), 5121–5127.
Deckert, J. and Struhl, K. (2002) Targeted Recruitment of Rpd3 Histone Deacetylase Represses Transcription by Inhibiting Recruitment of Swi/Snf, SAGA and TATA Binding Protein. Molecular and Cellular Biology 22(18), 6458-6470
Next chapter
Next chapter
Dig deeper
End of tour
Want more? Go to iGEM.org!
If you want to learn more about Synthetic Biology you should visit iGEM's homepage, which contains loads   on the subject. You can even visit other teams' wikis and take a look at what they accomplished. Or maybe   join a team yourself?
A tour for everyone
Click here to begin the interactive wiki-tour
Take a quick glance at our project or dig deeper to get a complete walkthrough of   the wiki. Or do something in-between, the choice is yours.
