<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Chalmers-Gothenburg_Model skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Chalmers-Gothenburg/Model</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Chalmers Gothenburg iGEM 2017</TITLE></P><DIV class="project"><DIV id="header_navbar_wrapper"><DIV id="header-nav-anchor"><NAV><UL class="header_navbar_ul"><LI id="header_navbar_home"><A href="https://2017.igem.org/Team:Chalmers-Gothenburg" id="home_link"> Home </A></LI><LI class="dropdown"><A id="team_link">Team</A><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:Chalmers-Gothenburg/Team/students">Students</A><A href="https://2017.igem.org/Team:Chalmers-Gothenburg/Team/supervisors">Supervisors</A><A href="https://2017.igem.org/Team:Chalmers-Gothenburg/Attributions">Attributions</A><A href="https://2017.igem.org/Team:Chalmers-Gothenburg/Collaborations">Collaborations</A><A href="https://2017.igem.org/Team:Chalmers-Gothenburg/Team/sponsors">Sponsors</A></DIV></LI><LI class="dropdown"><A id="project_link">Project</A><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:Chalmers-Gothenburg/Description">Description</A><A href="https://2017.igem.org/Team:Chalmers-Gothenburg/Project/constructs">Constructs</A><A href="https://2017.igem.org/Team:Chalmers-Gothenburg/Model">Mathematical modeling</A></DIV></LI><LI class="dropdown"><A id="HP_link">Human Practices</A><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:Chalmers-Gothenburg/HP/Silver">Human practices</A><A href="https://2017.igem.org/Team:Chalmers-Gothenburg/HP/Gold_Integrated">Integrated human practices</A></DIV></LI><LI class="dropdown"><A id="achievements_link">Achievements</A><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:Chalmers-Gothenburg/Results">Project results</A><A href="https://2017.igem.org/Team:Chalmers-Gothenburg/Parts">Biobricks</A><A href="https://2017.igem.org/Team:Chalmers-Gothenburg/InterLab">Interlab study</A><A href="https://2017.igem.org/Team:Chalmers-Gothenburg/Achievements/medal">Accomplishments</A></DIV></LI><LI class="dropdown"><A id="notebook_link">Notebook</A><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:Chalmers-Gothenburg/Notebook">Lab journal</A><A href="https://2017.igem.org/Team:Chalmers-Gothenburg/Notebook/protocols">Protocols</A><A href="https://2017.igem.org/Team:Chalmers-Gothenburg/Safety">Safety</A></DIV></LI></UL></NAV></DIV></DIV><DIV class="wrap"><DIV class="header_test"><DIV id="banner"><H3>Mathematical modeling</H3></DIV></DIV><ASIDE id="side-nav"><DIV id="nav-anchor"><NAV><A href="#introduction" class="active">Introduction</A><A href="#results">Modeling and results</A><A href="#code">MATLAB code</A><A href="#references">References</A></NAV></DIV></ASIDE><DIV class="wrap-content"><DIV class="target" id="introduction"><DIV class="dashed_line_left_first"><H4 class="subtitle">Introduction</H4><P class="text">
The modeling part of our project was to simulate our biosensor <I>in silico</I> to gain insight in how the output might look, and what may be changed or modified in it to produce a better result.
        </P><P class="text">
One of the main parts of our biosensor uses the <B>yeast pheromone pathway</B> with custom GPCRs that will make the cells sense volatile organic compounds (VOCs) that are lung cancer biomarkers. These receptors, when activated by the VOCs in the breath of patients will trigger a proteolytic cascade, <B>the MAPK cascade</B>, in the pheromone pathway which ultimately leads to increased levels <B>of a transcription factor called Ste12</B> [1]. Ste12 acts as a “first checkpoint” in our biosensor as its values will correlate with the downstream processes. This transcription factor will then promote expression of <B>the protein Fus1</B>, whose function involves coordinating cell polarization and fusion in order to make the yeast mate with the opposing mating type [3]. Therefore, the promoter region (P<SUB><I>FUS1</I></SUB>) was used as a way to signal when the pheromone pathway is activated.
        </P><P class="text">
In parallel to the normal pheromone pathway, a construct containing <B>Cre recombinase</B> with P<SUB><I>FUS1</I></SUB> in front of it was added, which causes it to be co-expressed by pheromone pathway activation. Cre in turn flips the constitutive promoter P<SUB><I>TEF</I></SUB>  by recombining two <B>loxP-sites</B> surrounding the promoter. This leads to expression of <B>Cas9 in one mating type</B> and <B>a gRNA in the other</B>. When the cells of opposing mating types then mate to form a diploid, the Cas9 and gRNA will bind to each other. This will in turn cause the Cas9-gRNA complex to cut the <I>ADE2</I> gene in yeast and turn the cells red.
        </P></DIV></DIV><DIV class="target" id="results"><DIV class="dashed_line_right"><H4 class="sidetitle">Modeling and results</H4><P class="text">
To model this system, we decided to use a set of <B>coupled ordinary differential equations (ODEs)</B> that correspond to the different species involved in the <B>pheromone pathway</B>. The model <B>relies upon the work of Kofahl and Klipp</B> [1], whose model of this pathway has been used extensively by many iGEM teams throughout the years and has produced good results [4,5,6]. The model is very detailed and involves 36 different species with almost 50 reactions between them [1]. The modeling part of the project uses all of the ODEs and parameter from their original paper, along with other ODEs stated in Table 3.
        </P><FIGURE><FIGCAPTION><B>Figure 1.</B>The reaction network used in the model by Kofahl and Klipp [1].</FIGCAPTION></FIGURE></DIV><DIV class="dashed_line_left"><P class="text">
As seen in the bottom right corner of Figure 1, the cascade ends with an activation of Ste12 which makes it possible for it to bind into DNA and begin transcriptional activation. This was therefore used as a first step into validating the function of the biosensor. The concentration of Ste12 should be higher in lung cancer patients compared to healthy people, due to the fact that they produce a higher amount of VOCs that can activate the pheromone pathway [7]. To check this, <B>data on different VOC levels was used</B>, both in in lung cancer patients and compared to a healthy control group and a group consisting of smokers [7]. The data of the median level of each group is presented in Table 1 and consists only of n-octanal levels. The choice of only studying the n-octanal levels is due to the fact that no data (kinetic rates <I>v<SUB>1</SUB></I> to <I>v<SUB>5</SUB></I> in Kofahl and Klipp’s model, see Figure 1) is available for the two receptors used in this project, thus the 2-butanone levels were omitted. The original yeast parameters of these reactions were used together with the n-octanal concentrations from the three patient groups.
        </P><FIGURE><FIGCAPTION class="table-caption"><B>Table 1.</B> Median n-octanal level values for different patient groups [7]. </FIGCAPTION><TABLE><TBODY><TR><TH><B>Patient group (n=12) </B></TH><TH><B>Controls</B></TH><TH><B>Smokers</B></TH><TH><B>Lung cancer patients</B></TH></TR><TR><TD>Median n-octanal values (nmol/L)</TD><TD>0,011</TD><TD>0,006</TD><TD>0,052</TD></TR></TBODY></TABLE></FIGURE></DIV><DIV class="dashed_line_right"><P class="text">
Given these assumptions, the simulation for the Ste12 concentration can be run on the three different patient groups, which gives the result presented in Figure 2a. Even though the n-octanal level is about <B>five times higher in lung cancer patients</B> compared to the controls, this only leads to a <B>50% increase in peak Ste12 concentration</B>. To study whether this affected expression of Cre recombinase another ODE was added into the system, see Table 2, that models binding and gene expression of Ste12 onto P<SUB><I>FUS1</I></SUB>.
        </P><FIGURE><FIGCAPTION><B>Figure 2.</B> Time course plots for the different species in our model.</FIGCAPTION></FIGURE><FIGURE><FIGCAPTION class="table-caption"><B>Table 2.</B> ODEs describing the temporal behavior of different species in the model. The parameter values are either randomly chosen or from literature.</FIGCAPTION><TABLE><TBODY><TR><TH><B>ODEs added to Kofahl &amp; Klipp's model </B></TH><TH><B>Parameter values</B></TH></TR><TR><TD> Cre/Fus1 production: </TD><TD>[2,8]</TD></TR><TR><TD> Cas9/gRNA formation: </TD><TD>Arbitrary value</TD></TR><TR><TD> Diploid formation: </TD><TD>Arbitrary value</TD></TR><TR><TD> Haploid mating: </TD><TD>Arbitrary value</TD></TR><TR><TD> Cas9:gRNA complex formation: </TD><TD>Arbitrary value</TD></TR></TBODY></TABLE></FIGURE></DIV><DIV class="dashed_line_left"><P class="text">
The binding kinetics are assumed to be of <B>Hill type</B> with values found in the literature [2,8]. The initial concentrations of Fus1 and Cre recombinase are assumed to be zero. The time courses for Cre/Fus1 are shown in Figure 2b. Notice that we now have about double the Fus1/Cre recombinase amounts for the lung cancer patients compared to the controls. The Cre recombinase will flip a constitutive promoter P<SUB><I>TEF1</I></SUB> flanked between two LoxP sites. This will in turn activate transcription of the gRNA for <I>ADE2</I> disruption in one mating type and transcription of the protein Cas9 in the other. Since the production rate of P<SUB><I>TEF1</I></SUB> is unknown, we set it to an arbitrary value of 1mmol/min/L. Setting an arbitrary production rate does not allow us to make any quantitative predictions about the gRNA/Cas9 levels. However, due to the fact that the original yeast parameters are used for the receptors, quantitative information is lost anyway. Although we can still estimate relative concentrations of these species between our patient groups.
        </P><P class="text">
The mating rate in this model is assumed to be proportional to the amount of haploid cells in the culture and the Fus1 concentration. Given an initial 1000 haploids, we produce the corresponding amount of diploids for each patient group, shown in Figure 2d.
For the concentration of activated Cas9-gRNA complex per diploid cell we assumed it to be proportional to the amount of haploids, the gRNA concentration in each cell and the Cas9 concentration in each cell. The result from this simulation is shown in Figure 2e. After 24 hours there is about a <B>50% increase in Cas9:gRNA dimers</B> compared with the controls. This increase will in turn promote deletion of the <I>ADE2</I> gene, causing the cell to turn red. Although this can be modeled, since there were not any parameters at all for the <I>ADE2</I> pathway in the literature, we decided to only model up until the Cas9:gRNA dimer formation.
        </P><P class="text">
From this we can conclude that it might be possible to differentiate between lung cancer patients and controls since we get a 50% higher level of Cas9:dimers after 24 hours.
        </P></DIV></DIV><DIV class="target" id="code"><DIV class="dashed_line_right"><H4 class="subtitle">MATLAB code</H4><P class="text">
The MATLAB code used in this project can be found here:
        </P><H2>pheromonepathway.m</H2><P class="text">
This is the main .m-file that should be run. Available <A href="https://static.igem.org/mediawiki/2017/9/9d/T--Chalmers-Gothenburg--pheromonepathway.m.txt" target="blank">here.</A></P><H2>f.m</H2><P class="text">
This function is called by the ODE solver and contains all of the ODEs. Available <A href="https://static.igem.org/mediawiki/2017/8/8d/T--Chalmers-Gothenburg--f.txt" target="blank">here.</A></P></DIV></DIV><DIV class="target" id="references"><DIV class="dashed_line_left_last"><H4 class="sidetitle">References</H4><DIV class="reference_list"><UL><LI>[1]   Kofahl B,Klipp E Modelling the dynamics of the yeast pheromone pathway. Yeast 2004; 21: 831–850.</LI><LI>[2]   Dyjack N, Azeredo-Tseng C, Yildirim N Mathematical modeling reveals differential regulation of MAPK activity by phosphatase proteins in the yeast pheromone response pathway. Mol. BioSyst. Mol. BioSyst 2017; 13: 1323-1335.</LI><LI>[3]   Yeastgenome.org. (2017). FUS1 | SGD. [online] Available at: https://www.yeastgenome.org/locus/S000000532 [Accessed 17 Sep. 2017].</LI><LI>[4]   2012.igem.org. (2017). Team:TU-Delft/Modeling/SingleCellModel - 2012.igem.org. [online] Available at: https://2012.igem.org/Team:TU-Delft/Modeling/SingleCellModel [Accessed 17 Sep. 2017].</LI><LI>[5]   2015.igem.org. (2017). Team:CGU Taiwan/Modeling - 2015.igem.org. [online] Available at: https://2015.igem.org/Team:CGU_Taiwan/Modeling [Accessed 17 Sep. 2017].</LI><LI>[6]  2015.igem.org. (2017). Team:Chalmers-Gothenburg/Detection - 2015.igem.org. [online] Available at: https://2015.igem.org/Team:Chalmers-Gothenburg/Detection [Accessed 17 Sep. 2017]. </LI><LI>[7]   Fuchs P, Loeseken C, Schubert J. K. , Miekisch W Breath gas aldehydes as biomarkers of lung cancer. International Journal of Cancer 2010.</LI><LI>[8]   Groves B, Khakhar A, Nadel C. M. , Gardner R. G. ,Seelig G, Rewiring MAP kinases in Saccharomyces cerevisiae to regulate novel targets through ubiquitination. eLife 2016. </LI></UL></DIV></DIV></DIV></DIV><FOOTER id="footer-top"><DIV id="logorow"><DIV id="footer-contact-row"><A href="https://www.facebook.com/chalmersigem/?fref=ts" target="_blank">facebook</A><A href="https://twitter.com/igemchalmers" target="_blank">twitter</A><A href="mailto:igemgothenburg@gmail.com">e-mail</A><A href="https://www.instagram.com/chalmersigem/" target="_blank">instagram</A></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>