<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Chalmers-Gothenburg_Description skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Chalmers-Gothenburg/Description</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Chalmers Gothenburg iGEM 2017</TITLE></P><DIV class="project"><DIV id="header_navbar_wrapper"><DIV id="header-nav-anchor"><NAV><UL class="header_navbar_ul"><LI id="header_navbar_home"><A href="https://2017.igem.org/Team:Chalmers-Gothenburg" id="home_link"> Home </A></LI><LI class="dropdown"><A id="team_link">Team</A><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:Chalmers-Gothenburg/Team/students">Students</A><A href="https://2017.igem.org/Team:Chalmers-Gothenburg/Team/supervisors">Supervisors</A><A href="https://2017.igem.org/Team:Chalmers-Gothenburg/Attributions">Attributions</A><A href="https://2017.igem.org/Team:Chalmers-Gothenburg/Collaborations">Collaborations</A><A href="https://2017.igem.org/Team:Chalmers-Gothenburg/Team/sponsors">Sponsors</A></DIV></LI><LI class="dropdown"><A id="project_link">Project</A><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:Chalmers-Gothenburg/Description">Description</A><A href="https://2017.igem.org/Team:Chalmers-Gothenburg/Project/constructs">Constructs</A><A href="https://2017.igem.org/Team:Chalmers-Gothenburg/Model">Mathematical modeling</A></DIV></LI><LI class="dropdown"><A id="HP_link">Human Practices</A><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:Chalmers-Gothenburg/HP/Silver">Human practices</A><A href="https://2017.igem.org/Team:Chalmers-Gothenburg/HP/Gold_Integrated">Integrated human practices</A></DIV></LI><LI class="dropdown"><A id="achievements_link">Achievements</A><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:Chalmers-Gothenburg/Results">Project results</A><A href="https://2017.igem.org/Team:Chalmers-Gothenburg/Parts">Biobricks</A><A href="https://2017.igem.org/Team:Chalmers-Gothenburg/InterLab">Interlab study</A><A href="https://2017.igem.org/Team:Chalmers-Gothenburg/Achievements/medal">Accomplishments</A></DIV></LI><LI class="dropdown"><A id="notebook_link">Notebook</A><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:Chalmers-Gothenburg/Notebook">Lab journal</A><A href="https://2017.igem.org/Team:Chalmers-Gothenburg/Notebook/protocols">Protocols</A><A href="https://2017.igem.org/Team:Chalmers-Gothenburg/Safety">Safety</A></DIV></LI></UL></NAV></DIV></DIV><DIV class="wrap"><DIV class="header_test"><DIV id="banner"><H3>Description</H3></DIV></DIV><ASIDE id="side-nav"><DIV id="nav-anchor"><NAV><A href="#intro" class="active">Introduction</A><A href="#biomarkers">Lung cancer and biomarkers</A><A href="#breathtaking">BREATHtaking</A><A class="sub-level" href="#detection">Detection</A><A class="sub-level" href="#switch">Switch</A><A class="sub-level" href="#output">Output</A><A href="#references">References</A></NAV></DIV></ASIDE><DIV class="wrap-content"><DIV class="target" id="intro"><DIV class="dashed_line_left_first"><H4 class="subtitle">Introduction</H4><P class="text"><B>Lung cancer</B> is one of the most common cancer forms today and due to the lack of good detection methods, it is one of the deadliest. A suggested approach for detection of the disease have been to use specific <B>volatile organic compounds</B> (VOCs) found in elevated levels in the breath of lung cancer patients.
         </P><P class="text">
BREATHtaking is a biosensor for <B>detection of lung cancer</B> through the analysis of exhaled air from patients. The biosensor is designed as an AND-gate and consist of <B>olfactory receptors</B> from mouse and rat incorporated into two different strains of <I>Saccharomyces cerevisiae</I>. When these receptors detect specific VOCs, a CRISPR/Cas9-system is initiated  and mating is induced between the two strains. The CRISPR/Cas9-system is designed to partially delete the <I>ADE2</I> gene in the fused, diploid cells and cause a <B>colony colour change</B>. The colony colour change in the biosensor will indicate presence of VOCs in the patient’s exhaled breath.
         </P></DIV></DIV><DIV class="target" id="biomarkers"><DIV class="dashed_line_right"><H4 class="sidetitle">Lung cancer &amp; VOCs as biomarkers</H4><P class="text">
Lung cancer is the cancer form with the highest death toll, causing <B>1.69 million deaths</B> globally in 2015 [1]. One reason for the high death toll is the late detection of the cancer, due to symptoms appearing first in the later stage of the disease.
         </P><P class="text">
Three different stages are used to characterize lung cancer; I, II and III. The magnitude of the disease is based on size of the tumor and the extent of spreading to lymph nodes and other organs in the body [2]. A majority of the lung cancer cases today are diagnosed in stage III with a <B>5-year survival rate of 5-14%</B>. An earlier detection at stage I or II would increase the figure to 45-49% which clearly show that early detection is a key to increase the chances of surviving lung cancer [3].
           </P><P class="text">
At the appearance of the first symptoms, a <B>chest X-ray</B> is used today to scan for tumors. The scan can indicate the position and size of the tumors. Despite being a non-invasive procedure, it is accompanied by <B>limitations such as cost, time and expertise</B>. Due to these limitations X-ray is not suitable for mass screening of lung cancer [4]. Therefore, there is a demand for new diagnostic tools with mass screening potential.
         </P></DIV><DIV class="dashed_line_left"><P class="text"> 
One of the most promising approaches for early diagnosis is the use of volatile organic compounds (VOCs) as <B>biomarkers</B>. Some specific <B>VOCs</B> can be detected in higher concentrations in the breath of lung cancer patients compared to healthy individuals, since the cancer cells have an increased oxidative rate that results in oxidative stress products. The <B>low concentration</B> of VOCs, pmol/l,  however has provided a challenge in using this approach for detection and no efficient screening method is yet available [5].

         </P><P class="text">
Studies have shown that there is not only one compound that can be used as a biomarker in the breath of cancer patients [5]. An optimal diagnostic tool would therefore be able to <B>detect multiple VOCs</B> in order to decrease the number of false positive diagnoses. One example of a diagnostic tool is a biosensor, which is a biological-based system that can detect the presence of a target analyte and convert it into a signal. A biosensor can be developed with many different species, antibodies or enzymes. In this case, a <B>yeast-based biosensor</B> is a suitable approach where some of the advantages are the ease of genetic manipulation, as well as the <B>cheap and easy cultivation</B> [6]. 
         </P><P class="text">
To create this novel, non-invasive diagnostic tool, the budding yeast <B><I>Saccharomyces cerevisiae</I></B> will be used as a biosensor. The yeast cells will have two xenogeneic olfactory receptors incorporated. These can bind the biomarkers that are present in the breath of lung cancer patients, opening up for the possibility to detect the cancer before any symptoms appear.
         </P></DIV></DIV><DIV class="target" id="breathtaking"><DIV class="dashed_line_right"><H4 class="sidetitle">BREATHtaking</H4><P class="text">
The biosensor BREATHtaking is designed to detect lung cancer in the breath of patients and is built as an <B>AND-gate</B> in <I>S. cerevisiae</I>, dependent on mating between the two mating types α and a. The biosensor is divided into three parts; detection, switch and output, see Figure 1. Recombinant <B>G-protein coupled receptors</B> (GPCRs) detect <B>butanone and n-octanal</B>, two of the VOCs most commonly found in elevated levels in exhaled air from lung cancer patients [7,8]. Detection of n-octanal and butanone is the first part of the biosensor and leads to both expression of mating genes in the cells and initiation of the switch, which is the second part of the biosensor. The switch consists of a <B>CRISPR/Cas9 system</B> where a gRNA is expressed in mating type α and the Cas9 protein in mating type a. In the initiation of the switch, Cre recombinase is expressed which is needed to commence the transcription of the gRNAs and the Cas9 protein in the cells.
         </P><P class="text">
The third, and final, part of the biosensor is the output of the AND-gate. Detection of the VOCs leads to <B>mating</B> between the two mating types resulting in both the gRNA and the Cas9 being in the same, fused cell. As the gRNA corresponds to a sequences in <I>ADE2</I>, the CRISPR/Cas9 system disrupts the gene. <B>Disruption of <I>ADE2</I></B> gives the colony a colour change that is the final output of the biosensor.
         </P><FIGURE><FIGCAPTION><B>Figure 1.</B> The biosensor BREATHtaking is an AND-gate. Recombinant receptors sense butanone and n-octanal in mating type α and a respectively, and initiates a CRISPR/Cas9 system. The system disrupts <I>ADE2</I> and gives a colony colour change as output from the AND-gate.</FIGCAPTION></FIGURE></DIV></DIV><DIV class="target" id="detection"><DIV class="dashed_line_left"><H2 class="h2style">Detection</H2><P class="text">
Native mating in <I>S. cerevisiae</I> starts with the two mating types (a and α) producing <B>pheromones</B> called a- and α-factor [9]. These factors are recognized by GPCRs on the cell membrane of the opposite mating type. <B>Ste2</B> is found on the membrane of an a-cell and recognizes α-factor, while <B>Ste3</B> is found on α-cells and recognizes a-factor. When sensing the corresponding pheromone, Ste2 and Ste3 activates the Gαβγ G-protein, which then activates the <B>mitogen-activated protein kinase (MAPK) cascade</B>. The MAPK cascade promotes expression of mating-specific genes and cell cycle arrest. During this process, the cell starts to direct itself towards the partner cell and create a shmoo where the cells then can fuse together. During the fusion, the cells become diploid and will thereby have the genetic properties of both the original haploid cells [9].
           </P></DIV><DIV class="dashed_line_right"><P class="text">
In BREATHtaking, the native GPCRs are replaced by the mammalian GPCRs <B>Ri7</B> and <B>Olfr1258</B>. Ri7 originates from <I>Rattus norvegicus</I> and <B>recognizes n-octanal</B> [7], while Olfr1258 originates from <I>Mus musculus</I> and <B>recognizes butanone</B> [8]. Previous studies have shown that Ri7 can successfully be incorporated into <I>S. cerevisiae</I> [7], while Olfr1258 has yet only been expressed in mammalian cells [8]. Olfr1258 is incorporated into mating type α and Ri7 into a, see Figure 2. Like the native GPCRs, Olfr1258 and Ri7 will <B>activate the MAPK cascade</B> and <B>initiate mating</B> between the cells. Because the cells now lack Ste3 and Ste2, solely the presence of butanone and n-octanal will lead to mating as the recombinant GPCRs will not recognize a- and α-factors. 
           </P><FIGURE><FIGCAPTION><B>Figure 2.</B> The receptor Olfr1258 in α cells recognizes butanone and activates the MAPK cascade. Ri7 has a similar task in a cells, but recognizes n-octanal instead of butanone.</FIGCAPTION></FIGURE></DIV></DIV><DIV class="target" id="switch"><DIV class="dashed_line_left"><H2 class="h2style">Switch</H2><P class="text">
When the cells detect n-octanal and butanone, the MAPK cascade is activated and mating promoted. The second part of the biosensor, the switch, is simultaneously turned on. The basic design of the switch is the same in both α and a cells; <B>Cre recombinase</B> is expressed by <B>P<SUB><I>FUS1</I></SUB></B> and two <B>mutated loxP-sites</B> in opposite directions surrounds a backward directed promoter for expression of part of the CRISPR/Cas9 system.  <I>FUS1</I> is a mating specific gene and the promoter P<SUB><I>FUS1</I></SUB> is <B>activated by the MAPK cascade</B>. Thereby, detection of the VOCs leads to expression of Cre recombinase. The mutations in the loxP-sites prevent the sites from further recombining (and returning to their original location) after the first recombination has occurred [10]. Once this irreversible system is turned on, it <B>amplifies the internal signal</B> of the biosensor by continuing to express the CRISPR/Cas9 system and help creating a more significant output of the AND-gate.
         </P></DIV><DIV class="dashed_line_right"><P class="text">
The CRISPR/Cas9-system consist of <B>two parts</B>; the <B>Cas9 protein</B> and a <B>gRNA</B>. Both parts are needed for the system to function and this is exploited in the AND-gate. Two different constructs are designed on plasmids and cloned into either mating type α and a, making α cells express the gRNA and a cells Cas9. In mating type a, the strong constitutive promoter <B>P<SUB><I>TEF1</I></SUB></B> expresses the Cas9 protein after recombination of the loxP-sites, see Figure 3. Transcription and translation solely occurs after recombination of the loxP-sites, as the original direction of the promoter is backwards. 
         </P><FIGURE><FIGCAPTION><B>Figure 3.</B> Switch in mating type a. Cre recombinase changes the direction of P<SUB><I>TEF1</I></SUB> by recombining the surrounding loxP-sites, which leads to transcription of Cas9.</FIGCAPTION></FIGURE></DIV><DIV class="dashed_line_left"><P class="text">
In mating type α, the loxP site, the <B>ribozyme Hammerhead (HH)</B> and the gRNA will be expressed by <B>P<SUB><I>SNR52</I></SUB></B> after recombination, see Figure 4. The SNR52 promoter recruits <B>polymerase III</B> and prevents the gRNAs from translating into proteins [11]. After transcription, the ribozyme HH starts a <B>self-cleavage reaction</B> and leave the gRNA separated from the loxP site [12].
         </P><FIGURE><FIGCAPTION><B>Figure 4.</B> Switch in mating type α. Cre recombinase changes the direction of P<SUB><I>SNR52</I></SUB> by recombining the surrounding loxP-sites, which leads to transcription of the loxP site, the ribozyme HH and the gRNA. A self-cleavage reaction of HH after transcription separates the gRNA from the loxP site.</FIGCAPTION></FIGURE></DIV><DIV class="target" id="output"><DIV class="dashed_line_right"><H2 class="h2style">Output</H2><P class="text">
In the diploid cells created during the mating, <B>the whole CRISPR/Cas9 system</B> is present, as α cells contribute with the gRNA and a cells with Cas9. The gRNA corresponds to a site in <I>ADE2</I>, which is a native gene expressed during adenine synthesis. Guided by the gRNA, Cas9 makes <B>doublestranded breaks</B> in the gene and <B>disrupts <I>ADE2</I></B>. Mutations in <I>ADE2</I> cause accumulation of a <B>red pigment</B> during biosynthesis of adenine [13]. The red pigment creates a colour shift from white to red in the diploid cells and gives the final output of the biosensor, see Figure 5. The colony colour change only occurs when both butanone and n-octanal are present, which lowers the risk for false positives when testing for lung cancer. The colony colour change does not depend on fluorescence and will be visible in regular light. No fluorescence microscope is needed leading to <B>simple detection</B>. 
         </P><FIGURE><FIGCAPTION><B>Figure 5.</B> The gRNA from α cells and Cas9 from a cells are necessary to disrupt <I>ADE2</I> and create the colony colour change that is the final output of the biosensor.</FIGCAPTION></FIGURE></DIV></DIV><DIV class="target" id="references"><DIV class="dashed_line_left_last"><H4 class="sidetitle">References</H4><DIV class="reference_list"><UL><LI>[1]   Cancer [Internet]. World Health Organization; 2017 [Cited 2017-06-13]. Available at: http://www.who.int/mediacentre/factsheets/fs297/en/</LI><LI>[2]   Non-Small Cell Lung Cancer Stages [Internet]. American Cancer Society; 2016 [Updated 2017-03-02; Cited 2017-06-26]. Available at:
          https://www.cancer.org/cancer/non-small-cell-lung-cancer/detection-diagnosis-staging/staging.html</LI><LI>[3]   Non-Small Lung Cancer Survival Rates, by Stage [Internet]. American Cancer Society; 2016 [Updated 2016-05-16; Cited 2017-06-13]. Available at:
          https://www.cancer.org/cancer/non-small-cell-lung-cancer/detection-diagnosis-staging/survival-rates.htm </LI><LI>[4]   Lungcancer [internet]. Cancerfonden; 2015 [Cited 2017-06-07] Availiable at: https://www.cancerfonden.se/om-cancer/lungcancer</LI><LI>[5]   D’Amico A, Pennazza G, Santonico M, Martinelli E, Roscioni C, Galluccio G, Paolesse R, and Di Natale C. An investigation on electronic nose 
           diagnosis of lung cancer. Lung Cancer, 68(2):170–176, 2010</LI><LI>[6]   French C, de Mora K, Joshi N, Elfick A, Haseloff J, and Ajioka J. Synthetic biology and the art of biosensor design. The Science and Applications 
           of Synthetic and Systems Biology:Workshop Summary, 2011 </LI><LI>[7]   Minic J, Persuy M-A, Godel E, Aioun J, Connerton I, Salesse R, and Pajot-Augy E.
          Functional expression of olfactory receptors in yeast and development of a bioassay for odorant screening. FEBS Journal, 272(2):524-537, 2005</LI><LI>[8]   Suzuki Y and Shimono K. Deciphering the receptor repertoire encoding specific odorants by time-lapse single-cell array cytometry. Scientic Reports, 6(19934):1-9, 2015</LI><LI>[9]   Merlini L, Dudin O and Martin S. G. Mate and fuse: how yeast cells. Open Biology, 3(3):1-13, 2013. </LI><LI>[10]   Carter Z and Delneri D. New generation of loxp-mutated deletion cassettes for the genetic manipulation of yeast natural isolates. Yeast, 27(9):765-775, 2010</LI><LI>[11]   DiCarlo JE, Norville JE, Mali P, Rios X, Aach J, and Church GM. Genome engineering in saccharomyces cerevisiae using crispr-cas systems. 
           Nucleic Acids Research, 41(7):4336-4343, 2013</LI><LI>[12]   Fujita T and Fujii H. Applications of engineered dna-binding molecules such as tal proteins and the crispr/cas system in biology research.
           International Journal of Molecular Sciences, 16(10):23143-23164, 2015</LI><LI>[13]   Ugolini S and Bruschi CV. The red/white colony color assay in the yeast saccharomyces cerevisiae: epistatic growth advantage of white ade8-18, 
           ade2 cells over red ade2 cells. Current Genetics, 30(6):485-492, 1996</LI></UL></DIV></DIV></DIV></DIV></DIV><FOOTER id="footer-top"><DIV id="logorow"><DIV id="footer-contact-row"><A href="https://www.facebook.com/chalmersigem/?fref=ts" target="_blank">facebook</A><A href="https://twitter.com/igemchalmers" target="_blank">twitter</A><A href="mailto:igemgothenburg@gmail.com">e-mail</A><A href="https://www.instagram.com/chalmersigem/" target="_blank">instagram</A></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>