<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Chalmers-Gothenburg_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Chalmers-Gothenburg/Results</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Chalmers Gothenburg iGEM 2017</TITLE></P><DIV class="achievements"><DIV id="header_navbar_wrapper"><DIV id="header-nav-anchor"><NAV><UL class="header_navbar_ul"><LI id="header_navbar_home"><A href="https://2017.igem.org/Team:Chalmers-Gothenburg" id="home_link"> Home </A></LI><LI class="dropdown"><A id="team_link">Team</A><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:Chalmers-Gothenburg/Team/students">Students</A><A href="https://2017.igem.org/Team:Chalmers-Gothenburg/Team/supervisors">Supervisors</A><A href="https://2017.igem.org/Team:Chalmers-Gothenburg/Attributions">Attributions</A><A href="https://2017.igem.org/Team:Chalmers-Gothenburg/Collaborations">Collaborations</A><A href="https://2017.igem.org/Team:Chalmers-Gothenburg/Team/sponsors">Sponsors</A></DIV></LI><LI class="dropdown"><A id="project_link">Project</A><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:Chalmers-Gothenburg/Description">Description</A><A href="https://2017.igem.org/Team:Chalmers-Gothenburg/Project/constructs">Constructs</A><A href="https://2017.igem.org/Team:Chalmers-Gothenburg/Model">Mathematical modeling</A></DIV></LI><LI class="dropdown"><A id="HP_link">Human Practices</A><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:Chalmers-Gothenburg/HP/Silver">Human practices</A><A href="https://2017.igem.org/Team:Chalmers-Gothenburg/HP/Gold_Integrated">Integrated human practices</A></DIV></LI><LI class="dropdown"><A id="achievements_link">Achievements</A><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:Chalmers-Gothenburg/Results">Project results</A><A href="https://2017.igem.org/Team:Chalmers-Gothenburg/Parts">Biobricks</A><A href="https://2017.igem.org/Team:Chalmers-Gothenburg/InterLab">Interlab study</A><A href="https://2017.igem.org/Team:Chalmers-Gothenburg/Achievements/medal">Accomplishments</A></DIV></LI><LI class="dropdown"><A id="notebook_link">Notebook</A><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:Chalmers-Gothenburg/Notebook">Lab journal</A><A href="https://2017.igem.org/Team:Chalmers-Gothenburg/Notebook/protocols">Protocols</A><A href="https://2017.igem.org/Team:Chalmers-Gothenburg/Safety">Safety</A></DIV></LI></UL></NAV></DIV></DIV><DIV class="wrap"><DIV class="header_test"><DIV id="banner"><H3>Project results</H3></DIV></DIV><ASIDE id="side-nav"><DIV id="nav-anchor"><NAV><A href="#overview" class="active">Overview</A><A href="#receptors">Receptors</A><A href="#pathway">Pathway</A><A href="#future">Future directions</A><A href="#references">References</A></NAV></DIV></ASIDE><DIV class="wrap-content"><DIV class="target" id="overview"><DIV class="dashed_line_left_first"><H4 class="subtitle">Overview</H4><P class="text">
The lung cancer biosensor was divided into two major parts: <B>receptor integration</B> and <B>pathway</B>. The native membrane localized GPCRs STE2 and STE3 were substituted with the two heterologous GPCRs Olf1258 and Ri7 in two different yeast mating types. These two heterologous GPCRs were fused with GFP to test membrane localization. Finally, the functionality of the GPCRs was tested using the plasmid pGFP, were P<SUB><I>FUS1</I></SUB> is coupled to GFP. P<SUB><I>FUS1</I></SUB> is a mating gene promoter induced after GPCR induction. For testing the functional pathway, the loxP-P<SUB><I>TEF</I></SUB>-loxP switch was investigated. Before the P<SUB><I>TEF</I></SUB> promoter is turned, GFP is expressed. Therefore, the decrease in GFP expression was tracked after the turning of the promoter, indicating that the promoter switch was successful. 
         </P></DIV></DIV><DIV class="target" id="receptors"><DIV class="dashed_line_right"><H4 class="sidetitle">Receptors</H4><H2 class="h2style">Integration of Olfr1258 and Ri7</H2><P class="text">
For the biosensor to work, modifications of the two yeast strains, a and α, needed to be performed. One major modification was the substitution of the native GPCRs, STE2 and STE3, with the two heterogenous olfactory receptors, Ri7 and Olfr1258. This was executed using a CRISPR system with Cas9 and gRNA specific to STE2 and STE3. For more information see <A href="https://2017.igem.org/Team:Chalmers-Gothenburg/Project/constructs" target="blank">Project, constructs</A>.        
         </P><P class="text">
By using colony PCR, the substitution/integration of the GPCRs could be validated, after transformation of sequences for GPCR and CRISPR system. The colony PCRs showed successful results for the incorporation of both of the GPCRs into the genome of the respective yeast strain. Olfr1258 has been incorporated into the strain CENPK_111-61A and Ri7 into CENPK_11-11C.        
         </P></DIV><DIV class="dashed_line_left"><H2 class="h2style">Membrane localization</H2><P class="text">
To enable visualization of the localization of the receptors in the yeast cells, the yeast was transformed with a plasmid containing the GPCRs fused together with GFP. The green fluorescence could then easily enable detection of the GPCRs. For the yeast strain with Olfr1258, the membrane localization of the GPCR could be verified. In the leftmost picture in Figure 1, it can be seen that Olfr1258 is clearly located in the outer membrane of the cell. In the rightmost picture, it can be seen that these cells fluoresce in the nuclear membrane which is clearly shown when the cells in this image is dividing. This image thereby suggest that the GPCR is localized in all membranes.
         </P><FIGURE><FIGCAPTION><B>Figure 1.</B>The yeast expressing Olfr1258 fused together with GFP. The image visualizes the membrane-localization of Olfr1258 due to the green fluorescence expressed at the edges of the yeast cells. </FIGCAPTION></FIGURE><P class="text"> 
For the yeast cells expressing Ri7 fused together with GFP it could be visualized that this GPCR is membrane-localized as well, see Figure 2.
         </P><FIGURE><FIGCAPTION><B>Figure 2.</B>The yeast expressing Ri7 fused together with GFP. The image visualizes that Ri7 is membrane-localized due to the green fluorescence expressed at the edges of the yeast cell. </FIGCAPTION></FIGURE></DIV><DIV class="dashed_line_right"><H2 class="h2style">Lethality of VOCSs in unmodified strain</H2><P class="text">
A lethality test of the VOCs in unmodified yeast strain was performed to be able to examine what range of concentrations the cells could handle and survive in. This test was crucial for the project since the concentration that could be used in the other test of the VOCs together with the modified strain needed to be in a concentration that was known not to affect the cell growth.
         </P><FIGURE><FIGCAPTION><B>Figure 3.</B> The lethality test for butanone showing the OD of the yeast cells over approximately 5 hours after exposure to different concentrations of butanone and DMSO.</FIGCAPTION></FIGURE><P class="text">
This experiment was designed to expose the cells to different concentrations of respective VOC as well as different amounts of DMSO. The OD was measured after 2, 4 and 5,5 hours.
         </P><P class="text">
From the lethality test for butanone it could be seen that this range of concentrations, the butanone itself did not kill the cells. The cells that were exposed to 1 μM (1% DMSO), 10 μM (1% DMSO),1 mM (0.1% DMSO) and 10 mM (1% DMSO) did grow without being affected. However, the cells that were exposed to a higher concentration of DMSO had a clear decreased growth rate.
         </P><FIGURE><FIGCAPTION><B>Figure 4.</B> The lethality test for n-octanal showing the OD over approximately 5 hours after exposure to different concentrations of octanal and DMSO.</FIGCAPTION></FIGURE><P class="text">
n-Octanal showed to have more negative impact on the lethality of the cells. The cells that were exposed to 10 μM (1% DMSO) and 1 μM (1% DMSO) had the highest survival. The cells that were exposed to 100 μM (10% DMSO), 10 mM (1% DMSO) and the DMSO control (10% DMSO) had a clear decreased growth rate. However, the 1 mM (0.1% DMSO) showed a weak growth and it was clearly visualized that these cells were affected.
         </P><P class="text">
From these two lethality test, it can be concluded that the concentration of DMSO is crucial for the survival of the cells and the concentration should be below 1%. The concentration range tested for butanone did not affect the cells. However, the concentration range tested for n-octanal had an effect on the cells and the concentration of octanal should be kept between 1 μM and 10 μM for the cells to stay unaffected.
         </P></DIV><DIV class="dashed_line_left"><H2 class="h2style">BioLector</H2><P class="text">
The GPCRs’ ability to recognize and bind the VOCs, initiate the MAPK pathway and thus activate the P<SUB><I>FUS1</I></SUB> could be evaluated with a BioLector. Yeast with Ri7 and Olfr1258 incorporated was induced with n-octanal and butanone, respectively. A plasmid with GFP, pGFP, under the control of P<SUB><I>FUS1</I></SUB> was incorporated and thus, GFP would be expressed as a positive output.  
         </P><P class="text">
The result for Olfr1258 shows a slightly higher GFP expression when the yeast is induced with butanone, see Figure 5. For Ri7 there is no clear result. The higher GFP expression for Olfr1258 indicates that the receptor is able to sense the butanone and initiate the MAPK pathway which in turn activates the P<SUB><I>FUS1</I></SUB> promoter. However, one need to consider the possibility that such differences might occur due to varying growth rates compared to the cells’ autofluorescence and further tests would be needed. If the further tests strengthen the results, it shows that the Olfr1258 works as it should. 
         </P><FIGURE><FIGCAPTION><B>Figure 5.</B>The results from the BioLector of Olfr1258. The different results for respective concentrations are plotted with GFP emission per OD at the y-axis against time on the x-axis. </FIGCAPTION></FIGURE></DIV></DIV><DIV class="target" id="pathway"><DIV class="dashed_line_right"><H4 class="sidetitle">Pathway</H4><H2 class="h2style">Cre-Cas9</H2><P class="text">
The Cre-Cas9 construct was successfully assembled together into a plasmid. To make the P<SUB><I>FUS1</I></SUB> promoter activate, pheromones are needed since P<SUB><I>FUS1</I></SUB> activation is accomplished through the yeast pheromone pathway. Pheromones are produced through the mating mechanism of the yeast. The activation of P<SUB><I>FUS1</I></SUB> promoter will lead to expression of Cre-recombinase so the LoxP-sites would switch. The switch-system was designed so that it would be non-reversible and by flipping the promoter P<SUB><I>TEF1</I></SUB>, expression of the GFP would stop and Cas9 would be expressed instead.
         </P><P class="text">
The two different mating strains successfully mated which was done using selection and microscopic imaging. Further fluorescence microscopy images and colony-PCR runs were done to verify that the switch worked. The results came back negative. Sequencing of the Cre-Cas9 construct showed mutations in the Cre-recombinase part and might be the reason to why these experiments were unsuccessful.
         </P><P class="text">
To truly make sure that the flip worked and that the problem was not due to the deviation in the sequence of P<SUB><I>CRE</I></SUB> or that the P<SUB><I>FUS1</I></SUB> promoter is too weak to induce expression, another test was done by using an inducible Cre-recombinase plasmid. The activation of Cre-recombinase in the plasmid is accomplished by the presence of galactose. The cell culture was grown in a Delft medium, containing galactose, and observed under a fluorescence microscope during three days. Three replicates were used and all gave a similar result. Figure 6 shows the fluorescence measurements during these three days for one colony. The GFP expression is clearly decreased with the passage of time. 
         </P><FIGURE><FIGCAPTION><B>Figure 6.</B> A schematic view over the three days the cells were grown in galactose. The left figure show a microscopic image of the cells to give a sense of the amount of cells and the right figure show the fluorescence of these cells.</FIGCAPTION></FIGURE></DIV><DIV class="dashed_line_left"><P class="text">
A colony-PCR was also performed for this test, in order to confirm that the flip worked at sequence level and to make sure that the decrease in expression comes from the switch and not something else. The gel electrophoresis, Figure 2, shows that the flip appear to have succeeded in one of the replicates tested. The clear band at 1300 bp indicates that the flip worked. The upper run shows a gel electrophoresis with only two primers. Only the cells who succeeded to perform the flip will show a band. The lower run shows a gel electrophoresis with three primers. If the flip did not succeed, a band at 500 bp will appear as opposed to 1300 bp, if the flip worked. Two colony-PCRs were run, due to the the fact that the flip might not have a 100 % efficiency, and since the PCR would favour the shorter construct. In the lower gel, there are two band shown for the second replicate, which indicate that the efficiency was not 100 %. Notable is that the other two replicates showed no bands at all in either test, which means that the PCR did not work for them. There should at least be a band in the lower run regardless of the direction of the promoter.
         </P><FIGURE><FIGCAPTION><B>Figure 7.</B>The gel electrophoresis result for the colony-PCR. In the first well the control is loaded, and the other three show the three replicates. Same goes for the lower run. A band at 1300 bp indicate a success in the flipping of the promoter, and a band at 500 bp show a failure. The ladder used is GeneRuler 1kb.</FIGCAPTION></FIGURE><H2 class="h2style">Cre-gRNA</H2><P class="text">
No further tests was done on this pathway. Please check out <A href="https://2017.igem.org/Team:Chalmers-Gothenburg/Description" target="blank">Project, description</A> or check out the reference for more information [1]. 
         </P></DIV></DIV><DIV class="target" id="future"><DIV class="dashed_line_right"><H4 class="sidetitle">Future directions</H4><P class="text">For future directions of this project, there are some aspects to consider; 
firstly, to make all parts of the system work and get a complete functional system, from the sensing of the VOCs to the output of <I>ADE2</I>. 
         </P><P class="text">
Once a functioning system is finalized, several steps are required to allow it to be taken into use. To get the biosensor approved for usage in a screening program, extensive testing of the method is required both on healthy people and lung cancer patients. This to be able to show that the system can distinguish between healthy and sick patients. 
         </P></DIV></DIV><DIV class="target" id="references"><DIV class="dashed_line_left_last"><H4 class="sidetitle">References</H4><DIV class="reference_list"><UL><LI>[1]   DiCarlo J, Chavez A, Dietz S, Esvelt K, Church G. Safeguarding CRISPR-Cas9 gene drives in yeast. Nature Biotechnology [Internet]. 2015 [cited 10 October 2017];33(12):1250-1255. Available from: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4675690/</LI></UL></DIV></DIV></DIV></DIV><FOOTER id="footer-top"><DIV id="logorow"><DIV id="footer-contact-row"><A href="https://www.facebook.com/chalmersigem/?fref=ts" target="_blank">facebook</A><A href="https://twitter.com/igemchalmers" target="_blank">twitter</A><A href="mailto:igemgothenburg@gmail.com">e-mail</A><A href="https://www.instagram.com/chalmersigem/" target="_blank">instagram</A></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>