<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Chalmers-Gothenburg_Project_constructs skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Chalmers-Gothenburg/Project/constructs</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Chalmers Gothenburg iGEM 2017</TITLE></P><DIV class="project"><DIV id="header_navbar_wrapper"><DIV id="header-nav-anchor"><NAV><UL class="header_navbar_ul"><LI id="header_navbar_home"><A href="https://2017.igem.org/Team:Chalmers-Gothenburg" id="home_link"> Home </A></LI><LI class="dropdown"><A id="team_link">Team</A><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:Chalmers-Gothenburg/Team/students">Students</A><A href="https://2017.igem.org/Team:Chalmers-Gothenburg/Team/supervisors">Supervisors</A><A href="https://2017.igem.org/Team:Chalmers-Gothenburg/Attributions">Attributions</A><A href="https://2017.igem.org/Team:Chalmers-Gothenburg/Collaborations">Collaborations</A><A href="https://2017.igem.org/Team:Chalmers-Gothenburg/Team/sponsors">Sponsors</A></DIV></LI><LI class="dropdown"><A id="project_link">Project</A><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:Chalmers-Gothenburg/Description">Description</A><A href="https://2017.igem.org/Team:Chalmers-Gothenburg/Project/constructs">Constructs</A><A href="https://2017.igem.org/Team:Chalmers-Gothenburg/Model">Mathematical modeling</A></DIV></LI><LI class="dropdown"><A id="HP_link">Human Practices</A><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:Chalmers-Gothenburg/HP/Silver">Human practices</A><A href="https://2017.igem.org/Team:Chalmers-Gothenburg/HP/Gold_Integrated">Integrated human practices</A></DIV></LI><LI class="dropdown"><A id="achievements_link">Achievements</A><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:Chalmers-Gothenburg/Results">Project results</A><A href="https://2017.igem.org/Team:Chalmers-Gothenburg/Parts">Biobricks</A><A href="https://2017.igem.org/Team:Chalmers-Gothenburg/InterLab">Interlab study</A><A href="https://2017.igem.org/Team:Chalmers-Gothenburg/Achievements/medal">Accomplishments</A></DIV></LI><LI class="dropdown"><A id="notebook_link">Notebook</A><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:Chalmers-Gothenburg/Notebook">Lab journal</A><A href="https://2017.igem.org/Team:Chalmers-Gothenburg/Notebook/protocols">Protocols</A><A href="https://2017.igem.org/Team:Chalmers-Gothenburg/Safety">Safety</A></DIV></LI></UL></NAV></DIV></DIV><DIV class="wrap"><DIV class="header_test"><DIV id="banner"><H3>Constructs</H3></DIV></DIV><ASIDE id="side-nav"><DIV id="nav-anchor"><NAV><A href="#intro" class="active">Introduction</A><A href="#incorporation">Incorporation of GPCRs </A><A href="#GPCR">GPCR constructs</A><A href="#switch">Switch constructs</A><A href="#test">Testing-constructs</A></NAV></DIV></ASIDE><DIV class="wrap-content"><DIV class="target" id="intro"><DIV class="dashed_line_left_first"><H4 class="subtitle">Introduction</H4><P class="text">
In this section all constructs used in the project are summarized. Assembly of fragments was accomplished with Gibson assembly, PCR overlaps with primers containing overhangs and homologous recombination in yeast, which often worked best. The DNA fragments were designed in Snapgene and Benchling. Primer design included use of software from NEBuilders, such as the Tm-Calculator. The DNA fragments where ordered from IDT, ATUM and Genscript. In addition primers were ordered from Eurofins. 
         </P></DIV></DIV><DIV class="target" id="incorporation"><DIV class="dashed_line_right"><H4 class="subtitle">Incorporation of GPCRs in the genome</H4><H2 class="h2style">The Cas9 vector</H2><P class="text">
The vector contains the gene for Cas9 and markers for ampicillin and kanamycin, as shown in Figure 1.
            </P><FIGURE><FIGCAPTION><B>Figure 1.</B> Cas9-vector which contains the gene for Cas9 and the markers ampicillin and kanamycin.
</FIGCAPTION></FIGURE></DIV><DIV class="dashed_line_left"><H2 class="h2style">gRNA constructs</H2><P class="text">
The constructs contain gRNA able to cut <I>STE2</I> and <I>STE3</I> respectively. In both constructs, a P<SUB><I>SNR52</I></SUB> promoter, a gRNA scaffold and a T<SUB><I>SUP4</I></SUB> terminator are included. The construct used for cutting <I>STE2</I> is shown in Figure 2, and the construct used for cutting <I>STE3</I> in Figure 3. 
            </P><FIGURE><FIGCAPTION><B>Figure 2.</B> gRNA construct for deletion of <I>STE2</I>.
</FIGCAPTION></FIGURE><FIGURE><FIGCAPTION><B>Figure 3.</B> gRNA construct for deletion of <I>STE3</I>.
</FIGCAPTION></FIGURE></DIV></DIV><DIV class="target" id="GPCR"><DIV class="dashed_line_right"><H4 class="subtitle">GPCR constructs</H4><H2 class="h2style">Ri7</H2><P class="text">
The Ri7 construct consists of the gene for Ri7, a P<SUB><I>TEF1</I></SUB> promoter, a T<SUB><I>CYC1</I></SUB> terminator and two sequences overlapping with the upstream and downstream regions of <I>STE2</I> which makes it possible to use homologous recombination for integration. See Figure 4. 
         </P><FIGURE><FIGCAPTION><B>Figure 4.</B> Ri7 construct with the P<SUB><I>TEF1</I></SUB> promotor and the Ri7 gene, a terminator and overlaps with the sequences surrounding <I>STE2</I>. 
</FIGCAPTION></FIGURE><H2 class="h2style">Olfr1258</H2><P class="text">
The Olfr1258 construct is similar to the Ri7 construct and consists of the gene for Olfr1258, a P<SUB><I>TEF1</I></SUB> promoter, a T<SUB><I>CYC1</I></SUB> terminator and two sequences overlapping with the upstream and downstream regions of <I>STE3</I> which makes it possible to use homologous recombination for integration. See Figure 5. 
         </P><FIGURE><FIGCAPTION><B>Figure 5.</B> Olfr1258 construct with the P<SUB><I>TEF1</I></SUB> promotor and the Olfr1258 gene, a terminator and overlaps with the sequences surrounding <I>STE3</I>. 
</FIGCAPTION></FIGURE></DIV></DIV><DIV class="target" id="switch"><DIV class="dashed_line_left"><H4 class="sidetitle">Switch constructs</H4><P class="text">
The switch consists of a promoter flanked by two mutated loxP regions. After activation of the pheromone pathway by VOCs, the P<SUB><I>FUS1</I></SUB> promoter transcribes the Cre-recombinase which switches the orientation of the promoter. The promoter switch causes the transcription of either the gRNA or the Cas9 depending on the gene of interest.
         </P><H2 class="h2style">Cre-Cas9 construct</H2><P class="text">
The construct contains the genes for Cre-recombinase, GFP and Cas9. The Cre-recombinase is activated by the pheromone pathway, transcribed under the P<SUB><I>FUS1</I></SUB> promoter and terminated by the T<SUB><I>CYC1</I></SUB> terminator,  see Figure 6. 
         </P><FIGURE><FIGCAPTION><B>Figure 6.</B> Map of the whole Cre-Cas9 construct. 
</FIGCAPTION></FIGURE><P class="text">
To ensure expression of Cas9 only when the VOC is present, Cre-recombinase and loxP-sites are used. The P<SUB><I>FUS1</I></SUB> promoter is only activated when the pheromone pathway is activated by VOC binding to the GPCR. This means that Cre-recombinase is only active and can invert the P<SUB><I>TEF1</I></SUB> promoter when VOC’s are present. The loxP-sites are mutated to ensure that once the switch is flipped, it can not go back to its original orientation and Cas9 is constantly expressed. If the P<SUB><I>TEF1</I></SUB> promoter is not inverted due to the lack of Cre-recombinase GFP will be transcribed instead. In that way one can determine whether the flip of promoter worked or not. The GFP and Cas9 are followed by the terminators T<SUB><I>ADH1</I></SUB> and T<SUB><I>FBA1</I></SUB> respectively. 
         </P><P class="text">
The gblocks where ordered from IDT technologies and therefore the constructs where order in sizes in 1000 bp with overlaps. At the ends of the construct there is overlaps with the pRS416 that was used for transformation into yeast. pRS416 contains a <I>URA3</I> marker, see Figure 7. 
         </P><FIGURE><FIGCAPTION><B>Figure 7.</B> pRS416 containing a <I>URA3</I> marker. 
</FIGCAPTION></FIGURE></DIV><DIV class="dashed_line_right"><H2 class="h2style">Cre-gRNA construct</H2><P class="text">
The construct contains the genes for Cre recombinase, mCherry and gRNA for disruption of <I>ADE2</I>. The Cre recombinase is transcribed under the P<SUB><I>FUS1</I></SUB> promoter, activated by the pheromone pathway and terminated by the T<SUB><I>CYC1</I></SUB> terminator, see Figure 8. 
    </P><FIGURE><FIGCAPTION><B>Figure 8.</B> The Cre-gRNA construct. 
</FIGCAPTION></FIGURE><P class="text">
In order to guarantee expression of gRNA only when the VOC is present, Cre recombinase and loxP-sites are used in this construct as well. Cre-recombinase is only active and can invert the P<SUB><I>SNR52</I></SUB> promoter when VOC’s are present. P<SUB><I>SNR52</I></SUB> enables transcription of,  but not translation of RNA and is therefore used for creating gRNAs. The loxP-sites in this construct are mutated, loxPLE and loxPRE, in order to ensure that once the switch is flipped, it can not go back to its original direction and the gRNA is constantly expressed. A ribozyme is incorporated upstream of the gRNA in order to remove the loxP from the finished gRNA.  
         </P><P class="text">
Like the Cre-Cas9 construct, the team wanted to insert a fluorescence marker in this construct. Due to the fact that P<SUB><I>SNR52</I></SUB> can not transcribe mRNA and therefore in order to ensure the expression of mCherry, a second promoter, P<SUB><I>TEF1</I></SUB> is added between the loxP-site together with P<SUB><I>SNR52</I></SUB>, as can be seen in Figure 8. Only when the loxP-site is inverted, both gRNA and the mCherry will be transcribed. When the promoters are in the initial direction no fluorescence can be seen. However, the inversion of the promoter enables the expression of mCherry and a red fluorescence marker can be seen.
         </P><P class="text">
At the ends of the construct there are overlaps with pRS413. The plasmid pRS413 contains a <I>HIS3</I> marker, the plasmid is illustrated in Figure 9. This construct was ordered from IDT in gblocks of 1000 bp with overlap of the parts. 
         </P><FIGURE><FIGCAPTION><B>Figure 9.</B> The plasmid pRS413 containing a <I>HIS3</I> marker. 
</FIGCAPTION></FIGURE></DIV></DIV><DIV class="target" id="test"><DIV class="dashed_line_left_last"><H4 class="sidetitle">Testing-constructs</H4><P class="text">
Several constructs were created in order to use for verification of functionality of the GPCRs. Fusion-GPCRs were designed in order to investigate the localization of the GPCR’s.  Fusion GPCR’s were designed for both Ri7 and Olfr1258.  In addition, a plasmid containing GFP under the P<SUB><I>FUS1</I></SUB> promoter was designed. This was done in order to investigate the ability of the GPCRs to bind the VOCs, initiate the MAPK pathway and activate the P<SUB><I>FUS1</I></SUB> promoter. 
           </P><H2 class="h2style">Fusion-GPCRs</H2><P class="text">
In order to determine where the GPCRs were localized, fusion proteins were created. The genes for the GPCRs were connected with the gene for GFP using a small bridge of four amino acids, three glycine and one serine on the N-terminal, see Figure 10 and 11. The fusion GPCRs, Olfr1258 and Ri7, were put together with primers and the templates used were the existing construct of GPCRs and Cas9.  
           </P><FIGURE><FIGCAPTION><B>Figure 10.</B> Fusion protein of Ri7 coupled to GFP.
</FIGCAPTION></FIGURE><FIGURE><FIGCAPTION><B>Figure 11.</B> Fusion protein of Olfr1258 coupled to GFP.
</FIGCAPTION></FIGURE><H2 class="h2style">pGFP</H2><P class="text">
This construct was created to see if the binding of VOCs to the GPCRs correctly enables the P<SUB><I>FUS1</I></SUB> promoter to express a gene. The construct encodes for a GFP gene under the control of the P<SUB><I>FUS1</I></SUB> promoter, see Figure 12. If a VOC binds to the corresponding GPCR the pheromone pathway should be initiated which should induce the P<SUB><I>FUS1</I></SUB> promoter. The construct can be transformed into a strain with either GPCR.
           </P><FIGURE><FIGCAPTION><B>Figure 12.</B> pGFP containing GFP and the markers <I>HIS3</I> and ampicillin. 
</FIGCAPTION></FIGURE></DIV></DIV></DIV><FOOTER id="footer-top"><DIV id="logorow"><DIV id="footer-contact-row"><A href="https://www.facebook.com/chalmersigem/?fref=ts" target="_blank">facebook</A><A href="https://twitter.com/igemchalmers" target="_blank">twitter</A><A href="mailto:igemgothenburg@gmail.com">e-mail</A><A href="https://www.instagram.com/chalmersigem/" target="_blank">instagram</A></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>