<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Waterloo_Project"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:Waterloo/Project">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Waterloo/Project&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Waterloo/Project&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Waterloo/Project&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Waterloo/Project" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Waterloo/Project</H1><DIV id="bodyContent"><H3 id="siteSub">From 2008.igem.org</H3><TABLE style="color:#1b2c8a;background-color:#FBCC30;" cellpadding="3" cellspacing="1" border="1" width="62%" align="center"><TBODY><TR><TH align="center"><A href="/Team:Waterloo" title="Team:Waterloo">Home</A></TH><TH align="center"><A href="/Team:Waterloo/Team" title="Team:Waterloo/Team">The Team</A></TH><TH align="center"><STRONG class="selflink">The Project</STRONG></TH><TH align="center"><A href="/Team:Waterloo/Parts" title="Team:Waterloo/Parts">Parts Submitted to the Registry</A></TH><TH align="center"><A href="/Team:Waterloo/Modeling" title="Team:Waterloo/Modeling">Modeling</A></TH><TH align="center"><A href="/Team:Waterloo/Notebook" title="Team:Waterloo/Notebook">Notebook</A></TH><TH align="center"><A href="/Team:Waterloo/Sponsors" title="Team:Waterloo/Sponsors">Sponsors</A></TH></TR></TBODY></TABLE><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Project_Aim_and_Rationale"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Project Aim and Rationale</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Project_Design"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Project Design</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Production_and_Regulation_of_Nuclease_Genes"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Production and Regulation of Nuclease Genes</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-4"><A href="#Construction_Strategy"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Construction Strategy</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Biobrick_Part_Number_Legend"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">Biobrick Part Number Legend</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Steps_in_Assembly"><SPAN class="tocnumber">3.2</SPAN><SPAN class="toctext">Steps in Assembly</SPAN></A></LI><LI class="toclevel-1 tocsection-7"><A href="#Implementation"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Implementation</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#Preliminary_Assay_of_Genomic_Degradation"><SPAN class="tocnumber">4.1</SPAN><SPAN class="toctext">Preliminary Assay of Genomic Degradation</SPAN></A></LI><LI class="toclevel-2 tocsection-9"><A href="#Integration_of_Nuclease_Genes"><SPAN class="tocnumber">4.2</SPAN><SPAN class="toctext">Integration of Nuclease Genes</SPAN></A></LI><LI class="toclevel-2 tocsection-10"><A href="#Safe_Target_Plasmid"><SPAN class="tocnumber">4.3</SPAN><SPAN class="toctext">Safe Target Plasmid</SPAN></A></LI><LI class="toclevel-1 tocsection-11"><A href="#Preliminary_Testing_of_Promoter_Reporter_Constructs"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Preliminary Testing of Promoter Reporter Constructs</SPAN></A></LI><LI class="toclevel-1 tocsection-12"><A href="#Future_Directions_and_Applications"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">Future Directions and Applications</SPAN></A></LI><LI class="toclevel-1 tocsection-13"><A href="#References"><SPAN class="tocnumber">7</SPAN><SPAN class="toctext">References</SPAN></A></LI></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Project_Aim_and_Rationale"><B>Project Aim and Rationale</B></SPAN></H2><P>There are many useful bioproducts that can be efficiently expressed <I>in vivo</I> but are either cost-prohibitive or impossible to synthesiize and purify <I>in vitro</I>. To address this need for a means of manufacturing such bioproducts, a great deal of work had been done to study <A href="http://openwetware.org/wiki/Minicells" class="external text" rel="nofollow">minicells</A>, genome-free sacks of cytoplasm that bud off from certain mutant bacterial strains, and from which plasmid-driven expression has previously shown to be possible. Minicells showed substantial promise for possible therapeutic applications since they could generate useful products without replicating themselves. However, for <I>in situ</I> delivery of their products, they must be filtered from the live bacteria from which they were released. 
</P><P>Transient plasmid-driven expression in minicells has been shown to last for several hours, and though this &quot;lifetime&quot; can be improved by use of a nutrient bath, it is limited by cytosolic resources. Once these resources are exhausted, the minicell becomes inactive and degrades.
</P><P>Our project improves on the two issues described above. A bacterial cell already containing a plasmid encoding bioproduct synthesis genes, will self-destruct by degrading its own genome and transiently produce the bioproduct until cell resources have been exhausted. Having the original bacteria serve as the factory means that not only is there no live parental budding strain to purify from the culture, but also more cell resources are available to express larger quantities of bioproduct.
</P><P>Genome degradation is achieved using the combined activity of a restriction endonuclease to fragment the genome and an exonuclease to hasten degradation. The gene for the protein of interest will be located on a plasmid lacking recognition sites for the endonuclease, allowing it to remain intact after genomic degradation. The plasmid genes will be expressed using the remaining cell resources until they expire. The primary application of this design would be an in situ compound production and delivery system for agricultural, industrial or therapeutic use.
</P><H2><SPAN class="mw-headline" id="Project_Design"> Project Design </SPAN></H2><H3><SPAN class="mw-headline" id="Production_and_Regulation_of_Nuclease_Genes"> Production and Regulation of Nuclease Genes </SPAN></H3><UL><LI>Endo- and exonucleases will be cloned between the lambda pR promoter and the Plac promoter. The lambda promoter will drive the production of the forward transcript for translation of the genes (which are toxic), while pLac will drive production of antisense transcripts to maximize repression of the two nucleases during uninduced conditions. Cultures will initially be grown on media containing IPTG. Expression of the unstable cI repressor (C0051) will be driven by another instance of a lacI repressable promoter. Since nuclease production is usually repressed, endogenous lacI production should be sufficient. Any antisense transcripts from the leaky Plac will not be able to compete with the strong lambda promoter during expression of the nucleases.
</LI></UL><H2><SPAN class="mw-headline" id="Construction_Strategy">Construction Strategy</SPAN></H2><H3><SPAN class="mw-headline" id="Biobrick_Part_Number_Legend">Biobrick Part Number Legend</SPAN></H3><UL><LI>[TT] BBa_B0015 
</LI><LI>[Pr] BBa_R0051 
</LI><LI>[Plac] BBa_R0011 
</LI><LI>[CI] BBa_C0051 
</LI><LI>[T7gene6] BBa_K093001
</LI><LI>[PmeI] BBa_K093002
</LI></UL><H3><SPAN class="mw-headline" id="Steps_in_Assembly">Steps in Assembly</SPAN></H3><UL><LI>T7 gene 6 exonuclease will be amplified using PCR with primer-adapters for the BioBrick Prefix and Suffix. 
</LI><LI><I>Pseudomonas mendocina</I> PmeI endonuclease will be synthesized with its ORF flanked by BioBrick Prefix and Suffix.
</LI></UL><UL><LI>A <B>nuclease construct</B> will be constructed using crossover PCR of the nuclease gene. 
</LI></UL><UL><LI>A <B> convergent promoter system</B> designed to flank the nuclease genes are denoted below by Construct 1 and Construct 2. Construct 2 will be added downstream of the nuclease constructs such that no sense transcript and protein will be made at this stage. Construct 1 will be added upstream in the next round of assembly. This will ensure repression of the system in the presence of CI repressor and IPTG. 
</LI></UL><P> Construct 1 Construct 2
</P><UL><LI>The <B>addition of Construct 2</B> is a non toxic intermediate step.
</LI></UL><UL><LI>The <B>addition of Construct 1</B> ensures for proper repression in the presence of IPTG and the CI repressor.
</LI></UL><UL><LI>The already assembled CI repressor under the control of Plac must be <B>cotransformed</B> and <B>plated on IPTG</B></LI></UL><UL><LI>The <B>final independent construct</B>:
</LI></UL><UL><LI>These tests will determine:
</LI><LI>The length of time needed for the destruction of the genome by nuclease expression
</LI><LI>The optimal integration site based on the ideal length of time for nuclease gene production.
</LI><LI>Where the REN sites are on the genome.
</LI><LI>The rate of degradation by the exonuclease.
</LI></UL><P>Note: 
</P><UL><LI>Ligation scars are not shown. 
</LI><LI>Ribosome binding sites are present for open reading frames (T7gene6, PmeI, CI) upstream in the forward orientation.  
</LI><LI>[forward arrow] denotes forward orientation
</LI><LI>[reverse arrow] denotes reverse position.
</LI></UL><H2><SPAN class="mw-headline" id="Implementation">Implementation</SPAN></H2><H3><SPAN class="mw-headline" id="Preliminary_Assay_of_Genomic_Degradation">Preliminary Assay of Genomic Degradation</SPAN></H3><UL><LI>Chassis will be transformed with the preliminary construct. 
</LI><LI>Uninduced cell growth will be assayed to determine if the repression is strong enough to allow for the uninhibited growth of uninduced cells. 
</LI><LI>Cells will be induced for different periods of time. The induction will be terminated by washing to eliminate IPTG.  Assay for sterility of culture growth test with positive and negative controls (uninduced transformants and no inoculation, respectively). Genome destruction and plasmid safeness will be checked by genomic and plasmid preps of the induced culture. [(looking for smear and nice band?)]
</LI></UL><H3><SPAN class="mw-headline" id="Integration_of_Nuclease_Genes">Integration of Nuclease Genes</SPAN></H3><UL><LI>The genome degradation construct will be integrated into the genome via a proposed standard integration vector. 
</LI><LI>The vector will contain a BioBrick cloning site and unique double digest sites flanking the BioBrick cloning site and suitable elements for selection (Sac B and antibiotic resistance). The unique double digest sites will allow the user to customize the regions of homology for a double crossover homologous recombination. 
</LI></UL><UL><LI>The integration vector will be a vector with the oriR6k origin of replication that only replicates in <I>E. coli</I> lambda pir lysogen. For integration, the plasmid will be transformed into a strain of <I>E. coli</I> that is not a lambda pir lysogen by selecting for recombination using the antibiotic respective to the vector. The vector will be a suicide vector and therefore in any non lambda pir strain, a single recombination will be selected. The cells will then be removed from selective pressure to allow for the next recombination to occur. The double recombinants will be selected using sucrose, since the vector will also contain a <I>sacB</I> gene, which confers sucrose sensitivity. The putative double recombinants must then be screened for the respective antibiotic sensitivity.
</LI></UL><H3><SPAN class="mw-headline" id="Safe_Target_Plasmid">Safe Target Plasmid</SPAN></H3><UL><LI>In order to ensure expression of a desired gene post degradation of the genome, the plasmid must be free of recognition sites for the restriction endonuclease PmeI and therefore be unable to be influenced by the exonuclease. Red fluorescent protein is used as a reporter gene.
</LI></UL><UL><LI>The reporter will be put under the control of the LexA repressible promoter. The genome degradation method produces single stranded DNA due to the presence of the 5â-3â [(check gene 6  exo activity direction to confirm)] exonuclease. When there is single stranded DNA in the cell, RecA is upregulated as part of the SOS response, and RecA cleaves lexA. Therefore when the genome degradation is induced, the expression of the gene of interest will be subsequently induced. For this reason, the final construct must be integrated into a RecA + strain (TG1 is Rec+ and lacIq, and will work completely since lacI is required for the induction of the genome degradation operon).
</LI></UL><UL><LI>The plasmid safeness will be assayed when exposed to the nucleases. From that, reporter production can be evaluated, and the induction time relative to the time of degradation can be optimized.
</LI></UL><H2><SPAN class="mw-headline" id="Preliminary_Testing_of_Promoter_Reporter_Constructs">Preliminary Testing of Promoter Reporter Constructs</SPAN></H2><P>A preliminary experiment was conducted to test the expression of the pRecA promoter with LexA binding sites (K093000), the constitutive promoter pconst (J23118) and PocI, the lambda Pr promoter containing the operator for lambda cI repressor (K093010). Visual observations of liquid cultures show that expression of RFP under the control of the PocI promoter in absence of the repressor is high and similar to expression under control of the constitutive promoter. These cultures were all extremely pink, and also form pink colonies on solid media. The uninduced pRecA culture shows very low levels of expression and light pink colonies. 
</P><P>Cultures were inoculated in triplicate. Saturated overnight cultures were inoculated from a single colony. A second set of cultures were inoculated and grown to an OD600 of approximately 0.8. For pRecA, 3 conditions were set and conducted at both cell concentrations; uninduced culture, culture exposed to 125V of UV culture exposed to 250V of UV. To provide consistency for these exposures, sample was taken from the same culture at each level of exposure. 
</P><P>Readings were taken using a microplate reader, each sample plated in triplicate. The excitation peak and emission peaks of RFP (E1010) at 584 nm and 607 nm respectively were used for the readings. Saturated culture and culture at an OD600 of approximately 0.8 with no RFP expression were used as negative controls. 
</P><P>These preliminary tests resulted in very high background readings and results that were not statistically significant as a result of this signal to noise ratio (data not shown). Repetition and protocol modification is required to continue these tests. UV seemed to have little to no effect on the induction of the pRecA promoter. Other crosslinking agents will be tested, including mitomycin C, in addtion to testing other levels of UV exposure. PocI will also be tested in the presence of the cI repressor. 
</P><H2><SPAN class="mw-headline" id="Future_Directions_and_Applications">Future Directions and Applications</SPAN></H2><UL><LI>A post induction of genome degradation freeze drying system can be utilized to enable storage and transport for subsequent thawing and induction of the desired product. 
</LI><LI>Quorum sensing control T7PoI can be expressed. The nuclease operon will be put under T7 control instead of TetR
</LI></UL><H2><SPAN class="mw-headline" id="References">References</SPAN></H2><P>O'Connor,C.D., &amp; Timmins, K.N. (1987). Highly repressible expression system for cloning genes that specify potentialy toxic proteins. Journal of Bacteriology. 169, 4457-4462. 
</P><P>GUZMAN, L, BELIN, D, CARSON, M. J., &amp; BECKWITH, J (1995). Tight Regulation, Modulation, and High-Level Expression by  J. of Bacteriology, 177, 4121-4130.
</P><P>Macdiarmid, et al. Cancer Cell 11, 431-445, May 2007
</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2008.igem.org/Team:Waterloo/Project">http://2008.igem.org/Team:Waterloo/Project</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Waterloo/Project" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Waterloo/Project" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Waterloo/Project&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Waterloo/Project&amp;oldid=104276" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2008.igem.org:Privacy_policy" title="2008.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2008.igem.org:General_disclaimer" title="2008.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>