<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_CAU_China_Protocal"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:CAU_China/Protocal">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:CAU_China/Protocal&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:CAU_China/Protocal&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:CAU_China/Protocal&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:CAU_China/Protocal" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:CAU China/Protocal</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><TABLE id="maintable"><TBODY><TR><TD id="maincontent"><H1><SPAN class="mw-headline" id="Experimental_procotol"> Experimental procotol </SPAN></H1><H2><SPAN class="mw-headline" id="Material.2C_reagent_and_apparatus"> Material, reagent and apparatus </SPAN></H2><H4><SPAN class="mw-headline" id="1._Gene_and_plasmid"> 1. Gene and plasmid </SPAN></H4><P>(1) adh1 from Neurospora crassa (Nadh1, cDNA, gift from He lab);
</P><P>(2) adh2 from Saccharomyces cerevisiae (Sadh2, cDNA, gift from Lou lab);
</P><P>(3) ta0841 from Thermoplasma acidophilum (ta0841, commercially synthesized CDS, BGI Crop.);
</P><P>(4) expression vector pET-28a(+) (Novagen)
</P><H4><SPAN class="mw-headline" id="2._Bacteria_strain"> 2. Bacteria strain </SPAN></H4><P>E.coli DH5α, JM109, BL21(DE3); all are gifts from Chen lab;
</P><H4><SPAN class="mw-headline" id="3._Kit_and_reagent"> 3. Kit and reagent </SPAN></H4><P>(1) Plasmid Mini-preparation Kit (BioTeke);
</P><P>(2) PCR kit (Takara);
</P><P>(3) Gel purification Kit (BioTeKe);
</P><P>(4) Commonly used endonucleases(Takara and NEB);
</P><P>(5) T4 DNA ligase (Takara);
</P><P>(6) LB medium,
</P><P>Antibiotics:
Agarose, Bromophenol blue, Ethidium bromide (EB);
</P><H4><SPAN class="mw-headline" id="4._Apparatus"> 4. Apparatus </SPAN></H4><P>(1) tubes, Petri dishes, spreader, knife, micropipette, tips; Bechtop;
</P><P>(2) DNA gel electrophoresis apparatus; UV transilluminator;
</P><P>(3) Incubator, Shaker; PCR thermocycler;
</P><H1><SPAN class="mw-headline" id="Procedure"> Procedure </SPAN></H1><H2><SPAN class="mw-headline" id="Part_1._Expression_vector_construction"> Part 1. Expression vector construction </SPAN></H2><H4><SPAN class="mw-headline" id="1.1_Plasmid_extraction"> 1.1 Plasmid extraction </SPAN></H4><P>We use pET-28a(+)（Novagen） to construct expression vector and this plasmid was prepared using Plasmid Mini-preparation Kit (BioTeke). Here was the protocol:
</P><P>(1) cullture DH5α which contains pET-28a(+) overnight in LB (plus Kan) at 37℃, 220 rpm;
</P><P>(2) Pipette 1ml of culture into a 1.5 ml Eppendorf tube, and pellet at 9000 rpm for 30s;
</P><P>(3) Repeat step (2) three times, 3 ml of culture in a 1.5 ml Eppendorf as a resullt;
</P><P>(4) Pipette 250 μl of P1 solution, and vortex to suspend pellet;
</P><P>(5) Pipette 250μl of P2 solution, and invert the Eppendorf tube slightly 6 to 10 times;
</P><P>(6) Pipette 400μl of P3 solution, and immediately invert the Eppendorf tube slightly 6 to 10 times;
</P><P>(7) Leave the Eppendorf tube at room temperature for 5 min, and centrifuge at 12000 rpm for 10min;
</P><P>(8) Transfer the supernatant (about 750μl) into a Mini-Spin column, and spin at 12000 rpm for 30 s;
</P><P>(9) Pipette 500μl of WB solution (containing ethanol) into the column, spin at 12000 rpm for 30 s and discard the liquid in the collecting tube; 
</P><P>(10) Repeat step (9), and spin at 12000 rpm for 2 min; 
</P><P>(11) Transfer the column onto a new collecting tube; 
</P><P>(12) Add 50μl of sterilized H2O (65℃) into the column, and spin at 12000 rpm for 1min; the liquid in the collecting tube is the plasmid sample; 
</P><P>(13) Mold a 1% agarose gel, load 5μl of our plasmid sample (plus 1ul of 6×loading buffer) and run the gel at constant 120V for 30 min; 
</P><P>(14) Rinse the gel in EB solution for 5 min, and visualize at UV transilluminator to verify our plasmid sample; 
</P><H4><SPAN class="mw-headline" id="1.2_PCR_to_amplify_gene"> 1.2 PCR to amplify gene </SPAN></H4><P>Three genes coding for alcohol dehydrogenase from different organism were amplified by PCR in our project. They are adh1 from Neurospora crassa (Nadh1), adh2 from Saccharomyces cerevisiae (Sadh2), and ta0841 from Thermoplasma acidophilum (ta0841). Template for Nadh1 and Sadh2 are the corresponding cDNA, gifts from He lab and Lou lab respectively, while that for ta0841 was a commercially synthesized coding sequence purchased from BGI corporation.PCR kit was bought from Takara corporation. Table 1 lists the reaction mixture (50 μl) and condition:
</P><P><I>Table 1: PCR Mixture in 50μl of Reaction System</I></P></TD></TR><TR><TD width="171" nowrap="nowrap" valign="bottom"><P><STRONG>Reagent</STRONG></P></TD><TD width="89" nowrap="nowrap" valign="bottom"><P><STRONG>Volume</STRONG></P></TD></TR><TR><TD width="171" nowrap="nowrap" valign="bottom"><P>Sterilized    H2O</P></TD><TD width="89" nowrap="nowrap" valign="bottom"><P align="center">36.25</P></TD></TR><TR><TD width="171" nowrap="nowrap" valign="bottom"><P>10 × PCR    buffer</P></TD><TD width="89" nowrap="nowrap" valign="bottom"><P align="center">5</P></TD></TR><TR><TD width="171" nowrap="nowrap" valign="bottom"><P>dNTP mix    (2.5 mM each)</P></TD><TD width="89" nowrap="nowrap" valign="bottom"><P align="center">4</P></TD></TR><TR><TD width="171" nowrap="nowrap" valign="bottom"><P>25 mM    MgCl2</P></TD><TD width="89" nowrap="nowrap" valign="bottom"><P align="center">3</P></TD></TR><TR><TD width="171" nowrap="nowrap" valign="bottom"><P>Template    (20 ng/ul)</P></TD><TD width="89" nowrap="nowrap" valign="bottom"><P align="center">0.5</P></TD></TR><TR><TD width="171" nowrap="nowrap" valign="bottom"><P>Primer    Forward (1μM)</P></TD><TD width="89" nowrap="nowrap" valign="bottom"><P align="center">0.5</P></TD></TR><TR><TD width="171" nowrap="nowrap" valign="bottom"><P>Primer    Reverse (1μM)</P></TD><TD width="89" nowrap="nowrap" valign="bottom"><P align="center">0.5</P></TD></TR><TR><TD width="171" nowrap="nowrap" valign="bottom"><P>Taq polymerase    (5 U/μl)</P></TD><TD width="89" nowrap="nowrap" valign="bottom"><P align="center">0.25</P></TD></TR><TR><TD width="260" nowrap="nowrap" colspan="2" valign="bottom"><P>95℃，5min; 95℃, 10s, 55℃, 30s, 72℃, 
      1min 30s,    30 cycles; 72℃, 10min</P></TD></TR></TBODY></TABLE><B>Here were primers for each gene:</B><B>Nadh1:</B><P>Primer Forward (1μM, BamH I) 5’- CGGGATCCATGCCTCAGTTCGAGATTCCAG -3’
</P><P>Primer Reverse (1μM, Not I) 5’-ATAAGAATGCGGCCGCCTATTTGCTGGTATCGACGACATATC-3’
</P><P><B>Sadh2: </B></P><P>Primer Forward (1μM, BamH I) 5’-ATGCGGATCCATGTCTATTCCAGAAACTCAAAAAGCCATT-3’
</P><P>Primer Reverse (1μM, Sal I) 5’-ATGCGTCGACTTATTTAGAAGTGTCAACAACGTATCTACCAGC-3’
</P><P><B>ta0841:</B> 
</P><P>Primer Forward (1μM, BamH I) 5'- ATGCGGATCCATGAAGGCAGCCCTACTAGAA-3'
</P><P>Primer Reverse (1μM, Sal I) 5'- ATGCGTCGACTCA-ACTAAATTTAATCAGAACACG-3'
</P><H4><SPAN class="mw-headline" id="1.3_Gel_purification_of_PCR_product"> 1.3 Gel purification of PCR product </SPAN></H4><P>Gel purification Kit (BioTeKe) was used to purify our product for downstream endonuclease cleavage. Next is the modified protocol:
</P><P>(1) Mold a 1% agarose gel, load 50μl of our PCR product (plus 5ul of 11×loading buffer) and run the gel at constant 120V for 30 min; 
</P><P>(2) Rinse the gel in EB solution for 5 min, and visualize at UV transilluminator and cut the corresponding band;
</P><P>(3) Dissolve the gel in 600μl of DB solution at 65 ℃, transfer into a Mini-Spin column, and chill on ice for 2 min;
</P><P>(4) Centrifuge the column at 12000 rpm for 30s, discard the liquid in the collecting tube, and add 500μl of WB solution into the column;
</P><P>(5) Repeat step (4), spin the column at 12000 rpm for 2 min, transfer the column onto a new collecting tube;
</P><P>(6) Pipette 30μl of sterilized H2O (65℃), and spin the column at 12000 rpm for 1min; the liquid in the new collecting tube contains our purified PCR product;
</P><H4><SPAN class="mw-headline" id="1.4_Endonuclease_cleavage"> 1.4 Endonuclease cleavage </SPAN></H4><P><I>Table 2: Endonuclease reaction mixture (20μl)</I>
Purified PCR product was further cleaved by relative endonuclease introduced in the primer. The endonucleases are bought from Takara. Table 2 is the our reaction system (20μl) and condition: 

</P><P align="center"><STRONG>Reagent</STRONG></P><P align="center"><STRONG>Volume/μl</STRONG></P><P align="center">Endonuclease(15 U/μl)</P><P align="center">1</P><P align="center">10×buffer</P><P align="center">2</P><P align="center">substrate DNA</P><P align="center">1ug</P><P align="center"> </P><DIV><P>Note:        substrate DNA can be either PCR product or plamsid sample; All reaction        were set up at 37℃ for 1h</P></DIV> <P>sterilized H2O</P><P align="center">up to 20</P><H4><SPAN class="mw-headline" id="1.5_Purification_of_cleaved_PCR_product_and_plasmid"> 1.5 Purification of cleaved PCR product and plasmid </SPAN></H4><P>After endonuclease cleavage, PCR product and plasmid were purified using Gel-purification Kit (BioTeke) for downstream ligation. Plasmid was purified the same step as that described in 1.3, while PCR product purified as the following:
</P><P>(1) Blend PCR product with 500μl of DB solution, transfer into a Mini-Spin column, and chill on ice for 2 min;
</P><P>(4) Centrifuge the column at 12000 rpm for 30s, discard the liquid in the collecting tube, and add 500μl of WB solution into the column;
</P><P>(5) Repeat step (4), spin the column at 12000 rpm for 2 min, transfer the column onto a new collecting tube;
</P><P>(6) Pipette 30μl of sterilized H2O (65℃), and spin the column at 12000 rpm for 1min; the liquid in the new collecting tube contains our purified PCR product;
</P><H4><SPAN class="mw-headline" id="1.6_Ligation"> 1.6 Ligation </SPAN></H4><P>We used T4 DNA ligase (Takara) to construct expression vector, and table 3 describes the reaction system (20ul) and condition:
</P><P><I>Table 3: T4 DNA Ligse reaction mixture (20μl)</I></P><P><STRONG>Reagent</STRONG></P><P align="center"><STRONG>Volume/μl</STRONG></P><P align="center">T4 DNA ligase(350 U/μl)</P><P align="center">1</P><P align="center">10×buffer</P><P align="center">2</P><P align="center">Vector DNA</P><P align="center">50 ng (0.025 pmol)</P><P align="center">Insert DNA</P><P align="center">50 ng (0.075 pmol)</P><P align="center">sterilized H2O</P><P align="center">up to 20</P>

Note: reaction was set up at 4℃ overnight
<H4><SPAN class="mw-headline" id="1.7_Tansformation"> 1.7 Tansformation </SPAN></H4><P>(1) Transfer half of ligase reaction mixture (20μl) into 100μl competent cell on ice (competent cell is either DH5α or JM109), and chill on ice for 40 min;
</P><P>(2) Heat shock at 42℃ for 90s;
</P><P>(3) Chill on ice for 5 min;
</P><P>(4) Add 600 μl of LB, and recover at 37℃, 200 rpm for 45 min;
</P><P>(5) Pellet at 9000 rpm for 30s, and resuspend with 20μl of sterilized H2O;
</P><P>(6) Plate the culture at solid LB plus Kan, and incubate at 37℃ overnight;
</P><H4><SPAN class="mw-headline" id="1.8_Endo-cleavage_verification_of_positive_clone"> 1.8 Endo-cleavage verification of positive clone </SPAN></H4><P>Colony grows on the solid LB plus antibiotic Kan was designated as positive clone. To tell whether the positive clone was true transformant, we set up a endo-cleavage test where plasmid was extracted from each positive clone and double cleaved to see whether our insert DNA exist after gel electrophoresis. Here is the procedure:
</P><P>(1) Prepare the plasmid from each clone the same step as that described in 1.1;
</P><P>(2) Set up double endonuclease (NEB) reaction system (table 4), and react at 37℃ for at least 1h; 
</P><P><I>Table 4: Double endonuclease reaction mixture (20μl)</I></P><P align="center"><STRONG>Reagent</STRONG></P><P align="center"><STRONG>Volume/μl</STRONG></P><P align="center">Endonuclease 1 (10U/μl)</P><P align="center">1</P><P align="center">Endonuclease 2 (10U/μl)</P><P align="center">1</P><P align="center">10×NEB buffer</P><P align="center">5</P><P align="center">Substrate DNA</P><P align="center">1μg</P><P align="center">Sterilized H2O</P><P align="center">up to 50</P><I>Note: type of NEB buffer is enzyme dependent.</I><P>(3) Mold a 1% agarose gel, load 10μl of reaction mixture (plus 1ul of 11×loading buffer) and run the gel at constant 120V for 30 min; 
</P><P>(4) Rinse the gel in EB solution for 5 min, and visualize at UV transilluminator to detect corresponding band; 1.9 DNA sequencing
Our constructs were finally confirmed by DNA sequencing in BGI corporation after the endo-cleavage verification. 
</P><H2><SPAN class="mw-headline" id="Part_2.Library_construction"> Part 2.Library construction </SPAN></H2><P>After inserting wild-type alcohol dehydrogenase (ADH) gene into our expression vector pET-28a(+), we introduced mutations into our gene to construct a mutant library. The mutated genes were inserted into expression vector pET-28a(+) the same step as described in Part 1. This mutant constructs were then transformed into E.coli BL21(DE3) ,expressed and screened for acid resistance. Two methods were used to introduce mutations: site-directed mutagenesis and error-prone PCR, both using wild-type gene as template.Table5 is the error-prone PCR reaction system using Error-prone Kit (TIANDZ corporation): 
</P><P><I>Table 5: Error-prone reaction mixture (30μl)</I></P><P><STRONG>Reagent</STRONG></P><P align="center"><STRONG>Volume/μl</STRONG></P><P align="center">10×buffer</P><P align="center">3</P><P align="center">10×dNTP</P><P align="center">3</P><P align="center">MnCl2, 5mM</P><P align="center">3</P><P align="center">Template(10 ng/μl)</P><P align="center">1</P><P align="center">Primer Forward (1μM)</P><P align="center">0.3</P><P align="center">Primer Reverse (1μM)</P><P align="center">0.3</P><P align="center">Taq DNA polymerase (5U/μl)</P><P align="center">0.5</P><P align="center">Sterilized H2O</P><P align="center">up to 30</P><P align="center">94℃, 3min;    94℃, 1min, 45℃, 1min,    72℃ 1min, 20 cycles; 72℃, 10 min;</P><H2><SPAN class="mw-headline" id="Part_3._Protein_expression_and_enzyme_activity_assay"> Part 3. Protein expression and enzyme activity assay </SPAN></H2><P>Both wild type gene and mutant library were transformed into E.coli BL21(DE3) the same step as described in part 1. We use IPTG to induce exogenous protein expression, lyse the bacteria and assay the enzyme activity of crude enzyme extract in the acid buffer. Here is the step:
</P><P>(1) Culture 3ml of BL21(DE3), either containing wild-type or mutated genes,at 37℃ overnight in LB media plus antibiotic Kan;
</P><P>(2) Pipette 1ml of overnight culture into 200ml of LB plus Kan, and cluture at 37℃, 220 rpm for 3h; 
</P><P>(3) Pipette 100μl of IPTG (final conc. 0.6mM) into 100ml culture, induce at 30℃, 220 rpm for 3h;
</P><P>(4)Pellet at 4℃, 4000g for 30min;
</P><P>(5) Suspend the pellet in 20 ml lytic buffer;
</P><P>(6) Sonicate for 16min;
</P><P>(7)Centrifuge at 4℃，14000g for 30min, and the supernatant is crude enzyme extract;
</P><P>(8) Set up the enzyme activity assay mixture (table 6), react 5min and measure OD340;
</P><P><I>Table 6: Enzyme activity mixture (6ml)</I></P><P><STRONG>Reagent</STRONG></P><P><STRONG>Volume/μl</STRONG></P><P>Assay    buffer</P><P>　1300</P><P>15mM NAD+</P><P>　200</P><P>100% Ethanol</P><P>　35</P><P>Crude    enzyme extract</P><P>　500</P><P><B>Note: Assay buffer is sodium phosphate buffer, either 0..5 or 0.2 mM;Buffer pH of 6.9, 6.0, 5.0, 4.0, 3.0, and 2.0 are used when needed;</B></P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:CAU_China/Protocal">http://2013.igem.org/Team:CAU_China/Protocal</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:CAU_China/Protocal" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:CAU_China/Protocal" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:CAU_China/Protocal&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:CAU_China/Protocal&amp;oldid=198702" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>