"
Page
Discussion
View source
History
teams
Log in
 
Team:CAU China/Protocal
From 2013.igem.org
Experimental procotol
Material, reagent and apparatus
1. Gene and plasmid
(1) adh1 from Neurospora crassa (Nadh1, cDNA, gift from He lab);
(2) adh2 from Saccharomyces cerevisiae (Sadh2, cDNA, gift from Lou lab);
(3) ta0841 from Thermoplasma acidophilum (ta0841, commercially synthesized CDS, BGI Crop.);
(4) expression vector pET-28a(+) (Novagen)
2. Bacteria strain
E.coli DH5α, JM109, BL21(DE3); all are gifts from Chen lab;
3. Kit and reagent
(1) Plasmid Mini-preparation Kit (BioTeke);
(2) PCR kit (Takara);
(3) Gel purification Kit (BioTeKe);
(4) Commonly used endonucleases(Takara and NEB);
(5) T4 DNA ligase (Takara);
(6) LB medium,
Antibiotics: Agarose, Bromophenol blue, Ethidium bromide (EB);
4. Apparatus
(1) tubes, Petri dishes, spreader, knife, micropipette, tips; Bechtop;
(2) DNA gel electrophoresis apparatus; UV transilluminator;
(3) Incubator, Shaker; PCR thermocycler;
Procedure
Part 1. Expression vector construction
1.1 Plasmid extraction
We use pET-28a(+)（Novagen） to construct expression vector and this plasmid was prepared using Plasmid Mini-preparation Kit (BioTeke). Here was the protocol:
(1) cullture DH5α which contains pET-28a(+) overnight in LB (plus Kan) at 37℃, 220 rpm;
(2) Pipette 1ml of culture into a 1.5 ml Eppendorf tube, and pellet at 9000 rpm for 30s;
(3) Repeat step (2) three times, 3 ml of culture in a 1.5 ml Eppendorf as a resullt;
(4) Pipette 250 μl of P1 solution, and vortex to suspend pellet;
(5) Pipette 250μl of P2 solution, and invert the Eppendorf tube slightly 6 to 10 times;
(6) Pipette 400μl of P3 solution, and immediately invert the Eppendorf tube slightly 6 to 10 times;
(7) Leave the Eppendorf tube at room temperature for 5 min, and centrifuge at 12000 rpm for 10min;
(8) Transfer the supernatant (about 750μl) into a Mini-Spin column, and spin at 12000 rpm for 30 s;
(9) Pipette 500μl of WB solution (containing ethanol) into the column, spin at 12000 rpm for 30 s and discard the liquid in the collecting tube;
(10) Repeat step (9), and spin at 12000 rpm for 2 min;
(11) Transfer the column onto a new collecting tube;
(12) Add 50μl of sterilized H2O (65℃) into the column, and spin at 12000 rpm for 1min; the liquid in the collecting tube is the plasmid sample;
(13) Mold a 1% agarose gel, load 5μl of our plasmid sample (plus 1ul of 6×loading buffer) and run the gel at constant 120V for 30 min; 
(14) Rinse the gel in EB solution for 5 min, and visualize at UV transilluminator to verify our plasmid sample;
1.2 PCR to amplify gene
Three genes coding for alcohol dehydrogenase from different organism were amplified by PCR in our project. They are adh1 from Neurospora crassa (Nadh1), adh2 from Saccharomyces cerevisiae (Sadh2), and ta0841 from Thermoplasma acidophilum (ta0841). Template for Nadh1 and Sadh2 are the corresponding cDNA, gifts from He lab and Lou lab respectively, while that for ta0841 was a commercially synthesized coding sequence purchased from BGI corporation.PCR kit was bought from Takara corporation. Table 1 lists the reaction mixture (50 μl) and condition:
Table 1: PCR Mixture in 50μl of Reaction System
Reagent
Volume
Sterilized    H2O
36.25
10 × PCR    buffer
5
dNTP mix    (2.5 mM each)
4
25 mM    MgCl2
3
Template    (20 ng/ul)
0.5
Primer    Forward (1μM)
0.5
Primer    Reverse (1μM)
0.5
Taq polymerase    (5 U/μl)
0.25
95℃，5min; 95℃, 10s, 55℃, 30s, 72℃,
1min 30s,    30 cycles; 72℃, 10min
Here were primers for each gene:Nadh1:
Primer Forward (1μM, BamH I) 5’- CGGGATCCATGCCTCAGTTCGAGATTCCAG -3’
Primer Reverse (1μM, Not I) 5’-ATAAGAATGCGGCCGCCTATTTGCTGGTATCGACGACATATC-3’
Sadh2: 
Primer Forward (1μM, BamH I) 5’-ATGCGGATCCATGTCTATTCCAGAAACTCAAAAAGCCATT-3’
Primer Reverse (1μM, Sal I) 5’-ATGCGTCGACTTATTTAGAAGTGTCAACAACGTATCTACCAGC-3’
ta0841: 
Primer Forward (1μM, BamH I) 5'- ATGCGGATCCATGAAGGCAGCCCTACTAGAA-3'
Primer Reverse (1μM, Sal I) 5'- ATGCGTCGACTCA-ACTAAATTTAATCAGAACACG-3'
1.3 Gel purification of PCR product
Gel purification Kit (BioTeKe) was used to purify our product for downstream endonuclease cleavage. Next is the modified protocol:
(1) Mold a 1% agarose gel, load 50μl of our PCR product (plus 5ul of 11×loading buffer) and run the gel at constant 120V for 30 min; 
(2) Rinse the gel in EB solution for 5 min, and visualize at UV transilluminator and cut the corresponding band;
(3) Dissolve the gel in 600μl of DB solution at 65 ℃, transfer into a Mini-Spin column, and chill on ice for 2 min;
(4) Centrifuge the column at 12000 rpm for 30s, discard the liquid in the collecting tube, and add 500μl of WB solution into the column;
(5) Repeat step (4), spin the column at 12000 rpm for 2 min, transfer the column onto a new collecting tube;
(6) Pipette 30μl of sterilized H2O (65℃), and spin the column at 12000 rpm for 1min; the liquid in the new collecting tube contains our purified PCR product;
1.4 Endonuclease cleavage
Table 2: Endonuclease reaction mixture (20μl) Purified PCR product was further cleaved by relative endonuclease introduced in the primer. The endonucleases are bought from Takara. Table 2 is the our reaction system (20μl) and condition:
Reagent
Volume/μl
Endonuclease(15 U/μl)
1
10×buffer
2
substrate DNA
1ug
 
Note:        substrate DNA can be either PCR product or plamsid sample; All reaction        were set up at 37℃ for 1h
 
sterilized H2O
up to 20
1.5 Purification of cleaved PCR product and plasmid
After endonuclease cleavage, PCR product and plasmid were purified using Gel-purification Kit (BioTeke) for downstream ligation. Plasmid was purified the same step as that described in 1.3, while PCR product purified as the following:
(1) Blend PCR product with 500μl of DB solution, transfer into a Mini-Spin column, and chill on ice for 2 min;
(4) Centrifuge the column at 12000 rpm for 30s, discard the liquid in the collecting tube, and add 500μl of WB solution into the column;
(5) Repeat step (4), spin the column at 12000 rpm for 2 min, transfer the column onto a new collecting tube;
(6) Pipette 30μl of sterilized H2O (65℃), and spin the column at 12000 rpm for 1min; the liquid in the new collecting tube contains our purified PCR product;
1.6 Ligation
We used T4 DNA ligase (Takara) to construct expression vector, and table 3 describes the reaction system (20ul) and condition:
Table 3: T4 DNA Ligse reaction mixture (20μl)
Reagent
Volume/μl
T4 DNA ligase(350 U/μl)
1
10×buffer
2
Vector DNA
50 ng (0.025 pmol)
Insert DNA
50 ng (0.075 pmol)
sterilized H2O
up to 20
Note: reaction was set up at 4℃ overnight
1.7 Tansformation
(1) Transfer half of ligase reaction mixture (20μl) into 100μl competent cell on ice (competent cell is either DH5α or JM109), and chill on ice for 40 min;
(2) Heat shock at 42℃ for 90s;
(3) Chill on ice for 5 min;
(4) Add 600 μl of LB, and recover at 37℃, 200 rpm for 45 min;
(5) Pellet at 9000 rpm for 30s, and resuspend with 20μl of sterilized H2O;
(6) Plate the culture at solid LB plus Kan, and incubate at 37℃ overnight;
1.8 Endo-cleavage verification of positive clone
Colony grows on the solid LB plus antibiotic Kan was designated as positive clone. To tell whether the positive clone was true transformant, we set up a endo-cleavage test where plasmid was extracted from each positive clone and double cleaved to see whether our insert DNA exist after gel electrophoresis. Here is the procedure:
(1) Prepare the plasmid from each clone the same step as that described in 1.1;
(2) Set up double endonuclease (NEB) reaction system (table 4), and react at 37℃ for at least 1h;
Table 4: Double endonuclease reaction mixture (20μl)
Reagent
Volume/μl
Endonuclease 1 (10U/μl)
1
Endonuclease 2 (10U/μl)
1
10×NEB buffer
5
Substrate DNA
1μg
Sterilized H2O
up to 50
Note: type of NEB buffer is enzyme dependent.
(3) Mold a 1% agarose gel, load 10μl of reaction mixture (plus 1ul of 11×loading buffer) and run the gel at constant 120V for 30 min; 
(4) Rinse the gel in EB solution for 5 min, and visualize at UV transilluminator to detect corresponding band; 1.9 DNA sequencing Our constructs were finally confirmed by DNA sequencing in BGI corporation after the endo-cleavage verification.
Part 2.Library construction
After inserting wild-type alcohol dehydrogenase (ADH) gene into our expression vector pET-28a(+), we introduced mutations into our gene to construct a mutant library. The mutated genes were inserted into expression vector pET-28a(+) the same step as described in Part 1. This mutant constructs were then transformed into E.coli BL21(DE3) ,expressed and screened for acid resistance. Two methods were used to introduce mutations: site-directed mutagenesis and error-prone PCR, both using wild-type gene as template.Table5 is the error-prone PCR reaction system using Error-prone Kit (TIANDZ corporation):
Table 5: Error-prone reaction mixture (30μl)
Reagent
Volume/μl
10×buffer
3
10×dNTP
3
MnCl2, 5mM
3
Template(10 ng/μl)
1
Primer Forward (1μM)
0.3
Primer Reverse (1μM)
0.3
Taq DNA polymerase (5U/μl)
0.5
Sterilized H2O
up to 30
94℃, 3min;    94℃, 1min, 45℃, 1min,    72℃ 1min, 20 cycles; 72℃, 10 min;
Part 3. Protein expression and enzyme activity assay
Both wild type gene and mutant library were transformed into E.coli BL21(DE3) the same step as described in part 1. We use IPTG to induce exogenous protein expression, lyse the bacteria and assay the enzyme activity of crude enzyme extract in the acid buffer. Here is the step:
(1) Culture 3ml of BL21(DE3), either containing wild-type or mutated genes,at 37℃ overnight in LB media plus antibiotic Kan;
(2) Pipette 1ml of overnight culture into 200ml of LB plus Kan, and cluture at 37℃, 220 rpm for 3h;
(3) Pipette 100μl of IPTG (final conc. 0.6mM) into 100ml culture, induce at 30℃, 220 rpm for 3h;
(4)Pellet at 4℃, 4000g for 30min;
(5) Suspend the pellet in 20 ml lytic buffer;
(6) Sonicate for 16min;
(7)Centrifuge at 4℃，14000g for 30min, and the supernatant is crude enzyme extract;
(8) Set up the enzyme activity assay mixture (table 6), react 5min and measure OD340;
Table 6: Enzyme activity mixture (6ml)
Reagent
Volume/μl
Assay    buffer
1300
15mM NAD+
200
100% Ethanol
35
Crude    enzyme extract
500
Note: Assay buffer is sodium phosphate buffer, either 0..5 or 0.2 mM;Buffer pH of 6.9, 6.0, 5.0, 4.0, 3.0, and 2.0 are used when needed;
Retrieved from " http://2013.igem.org/Team:CAU_China/Protocal "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
