<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Technion_Israel_Tar_improvements skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Technion Israel/Tar improvements</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>S.tar, by iGEM Technion 2016</TITLE></P><DIV class="navbar navbar-inverse navbar-fixed-top" style="margin-top: 10px; padding-top: 10px; padding-bottom: 10px;"><DIV class="container"><DIV class="collapse navbar-collapse"><UL class="nav navbar-nav"><LI><A href="https://2016.igem.org/Team:Technion_Israel/Description">Project</A></LI><LI class="dropdown"><A href="https://2016.igem.org/Team:Technion_Israel/S.Tar_intro" class="dropdown-toggle" data-toggle="dropdown">S.Tar</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/S.Tar_intro">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Tar_improvements">Tar Improvements and Characterization</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Modifications/Rosetta">Computational Design</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Proof">PctA-Tar Chimera</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Modifications/narx">NarX-Tar Chimera</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Modifications/EstoTar">hERα-Tar Chimera</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Modifications/Intein">Intein</A></LI></UL><LI class="dropdown"><A href="https://2016.igem.org/Team:Technion_Israel/Design" class="dropdown-toggle" data-toggle="dropdown">FlashLab</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Design">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Model">Modeling</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Hardware">Hardware</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Color">Chromoprotein</A></LI><LI class="dropdown"><A href="" class="dropdown-toggle" data-toggle="dropdown">Wet Lab</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Results">Highlight Results</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Experiments">Experiments</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Protocols">Protocols</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Notebook">Notebook</A></LI><LI class="dropdown"><A href="" class="dropdown-toggle" data-toggle="dropdown">Judging</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Proof">Proof of concept</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Parts">Biobricks</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Software">Software</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Measurement">Measurments</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Safety">Safety</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Medals">Medals</A></LI><LI class="dropdown"><A href="" class="dropdown-toggle" data-toggle="dropdown">People</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Human_Practices">Human Practice</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Team">Team Members</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Attributions">Attributions</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Collaborations">Collaborations</A></LI></DIV></DIV></DIV><TITLE>S.tar, by iGEM Technion 2016</TITLE><DIV class="desk_wrapper"><DIV class="row"><DIV class="col-sm-8 col-sm-offset-2"><DIV class="row"><DIV class="col-md-12"><UL class="nav nav-tabs" role="tablist"><LI role="presentation" class="col-sm-3 col-xs-6"><A href="#111" aria-controls="111" role="tab" data-toggle="tab"><H4 class="text-center">Introduction</H4></A></LI><LI role="presentation" class="col-sm-3 col-xs-6"><A href="#222" aria-controls="222" role="tab" data-toggle="tab"><H4 class="text-center">Expression</H4></A></LI><LI role="presentation" class="col-sm-3 col-xs-6"><A href="#333" aria-controls="333" role="tab" data-toggle="tab"><H4 class="text-center">Location</H4></A></LI><LI role="presentation" class="col-sm-3 col-xs-6"><A href="#444" aria-controls="444" role="tab" data-toggle="tab"><H4 class="text-center">Movement</H4></A></LI></UL></DIV></DIV><DIV class="tab-content"><DIV role="tabpanel" class="tab-pane fade in active" id="111"><DIV class="cont_box"><DIV class="row"><DIV class="col-md-12 col-sm-12"><H2>Introduction</H2></DIV></DIV><DIV class="row"><DIV class="col-md-12 col-sm-12"><P class="text-justify">
									We chose Tar (Taxis Aspartate Receptor) of <I>E. coli</I> as the template chemoreceptor to engineer the chimeric chemoreceptor. To do that, we first need to characterize Tar in 
									terms of location <I> in vivo </I> and its function, mainly the response and movement of bacteria. In order to do so, the plasmid expressing 
									Tar was cloned to an <I>E.coli</I> strain lacking chemoreceptors, 
									<A data-toggle="popover" data-trigger="click" data-original-title="Info:" data-html="true" data-content="an &lt;i&gt; E.coli &lt;/i&gt; derivative, which lacks chemoreceptors genes, means this strain does not obtain chemotaxis ability">
									UU1250</A>. 
									A proper characterization of the bacteria will serve as a good reference for indicating that our newly 
									designed chemoreceptors are functional.

									</P></DIV></DIV><DIV class="row"><DIV class="col-sm-10 col-sm-offset-1"><DIV class="row"><DIV class="col-md-6 col-sm-12"><P class="text-justify"><B>a.</B></P></DIV><DIV class="col-md-6 col-sm-12"><P class="text-justify"><B>b.</B></P></DIV></DIV><P class="text-center"><B>Fig.1:</B><B>a.</B> Scheme of Tar chemoreceptor. <B>b.</B><A href="http://parts.igem.org/Part:BBa_K1992004" target="_blank">K1992004</A> - High expression biological circuit ; 
												<A href="http://parts.igem.org/Part:BBa_J23100" target="_blank">J23100 </A> promoter, 
												<A href="http://parts.igem.org/Part:BBa_B0034" target="_blank">B0034  </A> RBS, 
												<A href="http://parts.igem.org/Part:BBa_K777000" target="_blank">K777000   </A> Tar chemoreceptor and									
												<A href="http://parts.igem.org/Part:BBa_B0015" target="_blank">terminator</A>.
										</P></DIV></DIV></DIV></DIV><DIV role="tabpanel" class="tab-pane fade" id="222"><DIV class="cont_box"><DIV class="row"><DIV class="col-md-12 col-sm-12"><H2>Expression</H2></DIV></DIV><DIV class="row"><DIV class="col-md-6 col-sm-12 vcenter"><P class="text-justify">
									Studies have shown that overexpression of a single chemoreceptor increases the sensitivity of the bacteria to the chemoreceptor's ligands <B>(1)</B>. Due to this property we constructed a high expression 
									system of Tar chemoreceptor, based on the <A href="http://parts.igem.org/Part:BBa_K777000" target="_blank">K777000</A> BioBrick. This expression system includes the strongest Anderson promoter (<A href="http://parts.igem.org/Part:BBa_J23100" target="_blank">J23100</A>),
									the strongest RBS (<A href="http://parts.igem.org/Part:BBa_B0034" target="_blank">B0034</A>), 
									 according to <A href="https://2010.igem.org/Team:Warsaw/Stage1/RBSMeas" target="_blank">Warsaw 2010's measurement</A>, the Tar encoding sequence (<A href="http://parts.igem.org/Part:BBa_K777000" target="_blank">K777000</A>) and a double terminator (<A href="http://parts.igem.org/Part:BBa_B0015" target="_blank">B0015</A>).</P></DIV><DIV class="col-md-6 col-sm-12 vcenter"><P class="text-center"><B>Fig. 1:</B><A href="http://parts.igem.org/Part:BBa_K1992004" target="_blank">K1992004</A> - High expression biological circuit ; 
										<A href="http://parts.igem.org/Part:BBa_J23100" target="_blank">J23100 </A> promoter, 
										<A href="http://parts.igem.org/Part:BBa_B0034" target="_blank">B0034  </A> RBS, 
										<A href="http://parts.igem.org/Part:BBa_K777000" target="_blank">K777000   </A> Tar chemoreceptor and									
										<A href="http://parts.igem.org/Part:BBa_B0015" target="_blank">terminator</A>.
									</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 col-sm-12"><P class="text-justify">
									This plasmid, <A href="http://parts.igem.org/Part:BBa_K1992004" target="_blank">K1992004</A>, 
									was then transformed into <A data-toggle="popover" data-trigger="click" data-original-title="Info:" data-html="true" data-content="An E.coli derivative, which lacks chemoreceptors genes, means this strain does not obtain chemotaxis ability."> UU1250</A>  strain to generate UST strain. This new strain is assumed to have a high expression of the single chemoreceptor, due to the strong promoter and strong RBS (Fig. 1).
									</P></DIV></DIV><DIV class="row"><DIV class="col-md-6 col-sm-12 vcenter"><P class="text-justify">
									In order to explore the Tar chemoreceptor as it is in nature, we also decided to examine the effect of a native RBS, as found in <I>E. coli</I>, on the chemotaxis system. The strong RBS (referred as
                                                                        RBS) was replaced by the native RBS (referred as nRBS) 
									of Tar as found in the <A href="http://ecocyc.org/" target="_blank"><I>E.coli genome</I></A> (Fig. 2). The new expression 
									system <A href="http://parts.igem.org/Part:BBa_K1992005" target="_blank">K1992005</A> differs 
									only by the RBS, allowing the comparison between the  two RBSs. </P></DIV><DIV class="col-md-6 col-sm-12 vcenter"><P class="text-center"><B>Fig. 2:</B><A href="http://parts.igem.org/Part:BBa_K1992005" target="_blank">K1992005</A> - High expression circuit using the Tar native RBS.
									</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 col-sm-12"><P class="text-justify">
									The two strains, which differ in the RBS (RBS vs. nRBS) were examined with a <A href="https://2016.igem.org/Team:Technion_Israel/Experiments)">swarming assay</A>, in which the chemotaxis ability is tested in a swarm plate. In this assay, the bacteria metabolize the attractant found in the medium and move outwards from a center point, creating a halo. Results can be seen in the movement section, Fig. 3.
The results were quite surprising. The strain containing a native RBS had a better chemotaxis ability than the strain containing a strong RBS. This finding suggests a higher expression of Tar, meaning the native RBS is more effective than the strong RBS, according to <A href="https://2010.igem.org/Team:Warsaw/Stage1/RBSMeas" target="_blank">Warsaw 2010's measurement</A>. 
									</P></DIV></DIV></DIV></DIV><DIV role="tabpanel" class="tab-pane fade" id="333"><DIV class="cont_box"><DIV class="row"><DIV class="col-md-12 col-sm-12"><H2>Location</H2></DIV></DIV><DIV class="row"><DIV class="col-md-12 col-sm-12"><P class="text-justify"><I> E.coli </I> native chemoreceptors cluster in the cell poles. This property is critical for signal 
									amplification and adaptation of the cell, since it is crucial for additional proteins, such as kinases and adaptors to interact with the receptor, once it is situated in its proper location in the membrane (for profound information on the chemotaxis system, <A href="https://2016.igem.org/Team:Technion_Israel/Chemotaxis">click here</A>). Although little is known about the mechanism of 
									localization, it is important to retain this property with our newly designed receptors - to ensure a functional 
									and sensitive chemotactic response <B>(2)</B>.
									GFP labeling is a very common technique to examine the migration and localization of proteins <I> in vivo </I>. 
									Fusion of GFP (<A href="http://parts.igem.org/Part:BBa_E0040" target="_blank">E0040</A>) to the Tar 
									chemoreceptor enabled us to track the migration and localization of the Tar-GFP fusion protein to the cell poles. The fusion was conducted using a flexible linker (<A href="http://parts.igem.org/Part:BBa_J18921" target="_blank">J18921</A>), 
									in order to preserve the three-dimensional structures of the receptor-proteins. The Tar-GFP (<A href="http://parts.igem.org/Part:BBa_K1992003" target="_blank">K1992003</A>) 
									was expressed using the two expression systems (<A href="http://parts.igem.org/Part:BBa_K1992008" target="_blank">K1992008</A> and 
									<A href="http://parts.igem.org/Part:BBa_K1992009" target="_blank">K1992009</A>), and localization was examined using an inverted fluorescence microscope (Nikon Eclipse Ti, magnification X100, 490nm wavelength) 
									(Fig. 1 and Fig. 2). In both cases, high concentration of fluorescence can be detected in the cell poles, indicating a proper 
									migration and localization of the Tar receptor. Comparison between the two expression systems (strong RBS and native RBS) 
									did not show any significant difference. This finding is consistent with the results of the chemotaxis assays, which indicate that both strains (with RBS &amp; nRBS) contain an active Tar. 
									</P></DIV></DIV><DIV class="row"><DIV class="row"><DIV class="col-md-12 col-sm-12"><P class="text-center"><B>Fig. 1:</B> Tar-GFP fusion results: <B>A.</B> Positive control - <I>E.Coli</I> strain expressing GFP protein. <B>B.</B> Negative control - <A data-toggle="popover" data-trigger="click" data-original-title="Info:" data-html="true" data-content="An E.coli derivative, which lacks chemoreceptors genes, means this strain does not obtain chemotaxis ability."> UU1250</A> strain expressing Tar chemoreceptor. <B>C.</B> Tar-GFP expressed using B0034 RBS. <B>D.</B> Tar-GFP expressed using the native RBS (488nm wavelength).
								    </P></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-12 col-sm-12"><P class="text-center"><B>Fig. 2:</B> Alignment of Tar-GFP white light and 488nm wavelength results: <B>A.</B> Positive control- <I>E.Coli</I> strain expressing GFP protein, <B>B.</B> Negative control- UU1250 strain expressing Tar chemoreceptor, <B>C.</B> Tar-GFP expressed using B0034 RBS, <B>D.</B> Tar-GFP expressed using the native RBS.
								</P></DIV></DIV></DIV></DIV><DIV role="tabpanel" class="tab-pane fade" id="444"><DIV class="cont_box"><DIV class="row"><DIV class="col-md-12 col-sm-12"><H2>Response and movement</H2></DIV></DIV><DIV class="row"><DIV class="col-md-12 col-sm-12"><P class="text-justify">
									Tar exhibits attractant response toward aspartate and a repellent response away from 
									Ni<SUP>+2</SUP> and Co<SUP>+2</SUP> concentrations <B>(3)</B>. Various <A href="https://2016.igem.org/Team:Technion_Israel/Experiments">chemotaxis assays </A> were performed, 
									using those substances, to show the bacteria's response and movement. In turn, these results 
									were used as a reference, in order to test the bacterial behavior with our designed chemoreceptors.<A href="https://2016.igem.org/Team:Technion_Israel/Experiments">Swarming assay</A> was conducted 
									with strains that express the Tar chemoreceptor with both RBS and native RBS, respectively (Fig. 1 and Fig. 2). Both exhibited chemotactic response and motility, compared to the 
									negative and positive control. Moreover, these results show a difference in radius size between the strong RBS, according to <A href="https://2010.igem.org/Team:Warsaw/Stage1/RBSMeas" target="_blank">Warsaw 2010's measurement</A>, 
									and the native RBS (Fig. 8). The larger radius of the native RBS suggests a higher expression of Tar, that might raise the sensitivity of the chemotaxis system <B>(1)</B>.
									</P></DIV></DIV><DIV class="row"><P class="text-center"><B>Fig. 1:</B><B>a.</B> Tar expression using <A href="http://parts.igem.org/Part:BBa_B0034" target="_blank">(B0034)</A> RBS in <A data-toggle="popover" data-trigger="click" data-original-title="Info:" data-html="true" data-content="An E.coli derivative, which lacks chemoreceptors genes, means this strain does not obtain chemotaxis ability."> UU1250</A> strain, resulting in a halo indicating a functional chemotaxis response. 
									<B>b.</B> Negative control - <A data-toggle="popover" data-trigger="click" data-original-title="Info:" data-html="true" data-content="An E.coli derivative, which lacks chemoreceptors genes, means this strain does not obtain chemotaxis ability."> UU1250</A> strain without the Tar expression plasmid. <B>c.</B> Positive control - ΔZ strain 
									expressing all chemoreceptors.								
								</P></DIV><DIV class="row"><P class="text-center"><B>Fig. 2:</B><B>a.</B> Tar expression using the native RBS in <A data-toggle="popover" data-trigger="click" data-original-title="Info:" data-html="true" data-content="An E.coli derivative, which lacks chemoreceptors genes, means this strain does not obtain chemotaxis ability."> UU1250</A> strain, resulting in a halo indicating a functional 
								chemotaxis response. <B>b.</B> Negative control - <A data-toggle="popover" data-trigger="click" data-original-title="Info:" data-html="true" data-content="An E.coli derivative, which lacks chemoreceptors genes, means this strain does not obtain chemotaxis ability."> UU1250</A> strain w/o the Tar expression plasmid. <B>c.</B> 
								Positive control - ΔZ strain expressing all chemoreceptors.
								</P></DIV><DIV class="row"><P class="text-center"><B>Fig. 3:</B> Comparison between our two new strains <A data-toggle="popover" data-trigger="click" data-original-title="Info:" data-html="true" data-content="A UU1250 strain cloned with Tar under strong RBS."> UST</A> and <A data-toggle="popover" data-trigger="click" data-original-title="Info:" data-html="true" data-content="A UU1250 strain cloned with Tar under native RBS."> UNT</A>: 
								<B>a.</B> UST strain - with the plasmid <A href=" http://parts.igem.org/Part:BBa_K1992004" target="_blank">K1992004</A>. <B>b.</B> UNT strain - with the plasmid <A href=" http://parts.igem.org/Part:BBa_K1992005" target="_blank">K1992005</A>.
								</P></DIV><DIV class="row"><DIV class="col-md-12 col-sm-12"><P class="text-justify">
									Attractant response of the Tar receptor was tested using the 
									<A href="https://2016.igem.org/Team:Technion_Israel/Experiments">chip microscope assay</A>. In this assay, a microfluidic chip was filled with a suspension of bacteria in motility buffer (does not trigger response), and placed under the microscope. The chip was then filled with an attractant, to track the bacterial movement. 
									The strain expressing Tar, moves towards a high concentration of Aspartate. As indicated (Fig. 4), after 
									15 minutes, the number of the bacteria in the frame increased from 49 at the beginning to 120 after an attractant was added – more than twofold, compared to the control - The same strain, which is treated with motility buffer (does not trigger chemotaxis)  
									and remained approximately unchanged (55 bacteria at time zero, and 66 after adding the attractant) (Fig. 10).
									</P></DIV></DIV><DIV class="row"><DIV class="col-md-6 col-xs-6"><P class="text-center"><B>a.</B></P></DIV><DIV class="col-md-6 col-xs-6"><P class="text-center"><B>b.</B></P></DIV><P class="text-center"><B>Fig. 4:</B><B>a.</B> Cells expressing Tar treated with aspartate t=0 <B>b.</B> Cells 
															expressing Tar treated with aspartate t=15 min.
								</P></DIV><DIV class="row"><DIV class="col-md-6 col-xs-6"><P class="text-center"><B>a.</B></P></DIV><DIV class="col-md-6 col-xs-6"><P class="text-center"><B>b.</B></P></DIV><P class="text-center"><B>Fig. 5:</B><B>a.</B> Cells expressing Tar treated with motility buffer t=0. 
															<B>b.</B> Cells expressing Tar treated with motility buffer t=15 min.

								</P></DIV><DIV class="row"><DIV class="col-md-12 col-sm-12"><P class="text-justify">
									Repellent response of Tar receptor was tested using a 
									<A href="https://2016.igem.org/Team:Technion_Israel/Experiments">chip color assay</A>. In this assay, a suspension of colored bacteria is spread evenly in the chip. After which, an attractant or repellent is added to the chip, and the bacterial movement is monitored.
									The bacteria expressing Tar, are fleeing from the high concentration of Co<SUP>+2</SUP>. As can 
									be seen in Fig. 6 a. After 15 minutes, the colored bacteria formed a cluster, which is visible to 
									the naked eye, whereas the control bacteria seen in Fig. 6 b did not react.
									</P></DIV></DIV><DIV class="row"><DIV class="col-md-6 col-xs-6"><P class="text-center"><B>a.</B></P></DIV><DIV class="col-md-6 col-xs-6"><P class="text-center"><B>b.</B></P></DIV></DIV><DIV class="row"><P class="text-center"><B>Fig. 6:</B> Chemotaxis test on the Chip for a<A data-toggle="popover" data-trigger="click" data-original-title="Info:" data-html="true" data-content="An E.coli derivative, which lacks chemoreceptors genes, means this strain does not obtain chemotaxis ability."> UU1250</A> strain expressing both the Tar chemoreceptor and a<A href="http://parts.igem.org/Part:BBa_K1357009" target="_blank"> blue-chromoprotein </A>. Images were taken as a time lapse. The red arrow represents the entrance where the substance was added <B>a.</B> Repellent was added to the chip's lower entrance. Through time, the bacteria move away from the repellent, creating a cluster, which results in a darker shade of color several inches from the repellent. <B>b.</B> Control - motility buffer was added to the chip's lower entrance. There is no difference in the shade ofcolor through time.

								</P></DIV></DIV></DIV><DIV class="row"><P class="references">
References:
1. SOURJIK, Victor; BERG, Howard C. Functional interactions between receptors in bacterial chemotaxis. Nature, 2004, 428.6981: 437-441.‏
2. SHIOMI, Daisuke, et al. Helical distribution of the bacterial chemoreceptor via colocalization with the Sec protein translocation machinery. Molecular microbiology, 2006, 60.4: 894-906.‏
3. BI, Shuangyu; LAI, Luhua. Bacterial chemoreceptors and chemoeffectors.Cellular and Molecular Life Sciences, 2015, 72.4: 691-708.‏ </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="modal fade" id="imagemodal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel" aria-hidden="true"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><BUTTON type="button" class="close" data-dismiss="modal"><SPAN aria-hidden="true">×</SPAN><SPAN class="sr-only">Close</SPAN></BUTTON></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>