Team:Technion Israel/Modifications/narx
S.tar, by iGEM Technion 2016
Project
S.Tar
Overview
Tar Improvements and Characterization
Computational Design
PctA-Tar Chimera
NarX-Tar Chimera
hERα-Tar Chimera
Intein
FlashLab
Overview
Modeling
Hardware
Chromoprotein
Wet Lab
Highlight Results
Experiments
Protocols
Notebook
Judging
Proof of concept
Biobricks
Software
Measurments
Safety
Medals
People
Human Practice
Team Members
Attributions
Collaborations
S.Tar, by iGEM Technion 2016
Introduction
Design
Results
Conclusion
Introduction
As a proof of concept of our platform, the Tar chemoreceptor LBD was replaced with the NarX LBD.               The NarX sensor protein of E. coli binds nitrite and nitrate and induces the expression               of proteins involved in anaerobic respiration (1).  We followed a protocol which was               previously shown to be successful (1).               This work resulted in a NarX-Tar chimera, comprised by the NarX LBD and Tar’s cytoplasmic region.               According to the article the chimera was supposed to function as a repellent chemoreceptor to               nitrite and nitrate.
Fig. 1: Scheme of Tar chemoreceptor on the left, NarX-Tar chimera on the right.
Design and Implementation
To construct this chimera, the protein sequence of both the LBD and the linker region of NarX were obtained the literature (1).  Using the DNA sequences of both frgamnets and Tar’s cytoplasmic region, we built a Biobrick device (part not submitted) (2). As in the PctA, first a 3D model was generated using the Phyre2 Protein Fold Recognition server, to ensure the correct folding of all parts of the chimera (Figure 1, results).
The next step was transforming to bacteria that lacks chemoreceptors - UU1250 , and to test the chemotactic ability of the cloned strain the following chemotaxis assays were performed
Since the NarX-Tar chimera is supposed to serve as a repellent chemoreceptor, the conventional assay is a chemical in plug assay, in which the repellent is added to a motility buffer and the movement of the bacteria is detected (See chemical in plug assay ). Here sodium nitrate was used as a repellent for the NarX-Tar strain.
Following, a drop assay was conducted. A suspension of bacteria in motility buffer was placed on a microscope slide and a drop of repellent was added in the concentrations of 10-2M and 10-6M. The slide was monitored throughout the entire procedure in order to examine the response of bacteria.
In addition, GFP was fused to the NarX-Tar chimera's C-terminus, in order to validate the location of the expressed chemoreceptor on both poles of the bacterial membrane. Flow cytometry analysis was conducted in order to verify the expression of the NarX-Tar chimera. We generated the NarX-Tar-GFP fusion gene that we planned to use for the localization assay. To verify the successful cloning of GFP to the NarX-Tar chimera, we screened 20 clones with flow cytometry.
Fig. 1: Biobrick device of the NarX-Tar chimera.
Results
The results of the 3D structure were not promising, as the folding was not               achieved in the correct manner, as can be seen in figure 1a. The 3D structure               of the native Tar is presented in figure 1b.
a.
b.
Fig. 1: (a)NarX-Tar chimera 3D structure. The Tar signaling regions is in gray, the NarX LBD is in red. (b)Native Tar 3D structure.
When tested on the chemical in plug assay no movement was detected, indicating that             the NarX-Tar strain is not responsive to sodium nitrate. (figure 2)
Fig. 2: Chemical in plug assay results: on the left,              positive control- ΔZ in the presence of the attractant aspartic acid. On              the rightNarX-Tar strain with repellent sodium nitrate.
The expected results of the drop assay were a rapid tumbling response of the bacteria.              Nevertheless, the strain showed no response to different concentrations of sodium              nitrate and unfortunately a sufficient documentation hasn't been obtained.
Lastly, the testing of the clone carrying the GFP gene fused to the chimera with flow              cytometry showed no indication to fluorescence. In figure 3, we plotted each clone as              a single histogram (light green histograms) and compared them with the histograms of a              positive control (Pct-Tar-GFP; dark green) and a negative control (NarX-Tar only, black).              As comparison between the single histograms is tedious, we plotted all histograms in              one figure for easier comparison. As can be seen in figure 4, the fluorescence of              NarX-Tar-GFP is comparable with the fluorescence of the  negative strain indicating that              all colonies could not been cloned successfully. Note that the negative control showed              two peaks, one peak which was defined as negative, and the second, albeit smaller peak              overlapped with the fluorescence of PctA-Tar (positive control). This is probably due              to a very high concentration of bacteria (event rate: 21000 events/sec). As a result,              the samples were diluted in the ratio of 1:4 in order to avoid false positive results.
Fig. 3: Flow cytometry results.
Fig. 4: A comparison between NarX colonies (light green), positive control (dark green) and negative (black) control.
Conclusion
As the results showed, NarX-Tar strain failed to show any response to sodium               nitrate. Moreover, as the reporter gene (GFP) showed no indication of expression,               this subproject was put aside.
It is unfortunate that the NarX-Tar clone was not successful, as it could have               served as an additional proof of concept the S.Tar platform, in addition to the               other strains we have cloned.
×Close
References:
1. R Ward, S.M., Delgado, A., Gunsalus, R.P., and Manson, M.D. (2002). A NarX-Tar chimera mediates repellent chemotaxis to nitrate and nitrite. Mol. Microbiol. 44, 709–719.
2. The complete E. coli genome sequence.
S.tar, by iGEM Technion 2016
