Team:Technion Israel/Proof
S.tar, by iGEM Technion 2016
Project
S.Tar
Overview
Tar Improvements and Characterization
Computational Design
PctA-Tar Chimera
NarX-Tar Chimera
hERα-Tar Chimera
Intein
FlashLab
Overview
Modeling
Hardware
Chromoprotein
Wet Lab
Highlight Results
Experiments
Protocols
Notebook
Judging
Proof of concept
Biobricks
Software
Measurments
Safety
Medals
People
Human Practice
Team Members
Attributions
Collaborations
S.Tar, by iGEM Technion 2016
PctA-Tar
Histamine-Tar
PctA-Tar chimera Introduction
One of S.Tars sub-projects focused on altering the LBD of Tar chemoreceptor in order to design new hybrid chimeras. That was accomplished by replacing the LBD of the original Tar chemoreceptor with a new one, from a different source, without modifying Tar signaling region. As a proof of concept for the newly designed Tar chimeras and the S.Tar project, we focused on testing the PctA-Tar hybrid.
Fig. 1: Scheme of native Tar chemoreceptor, native PctA receptor and PctA-Tar chimera. Adapted from (1)
PctA is a chemoreceptor found in the Pseudomonas Aeruginosa .It mediates             chemotaxis towards amino acids and away from organic compounds. It can sense all             amino acids except for Aspartate (1).
To construct this chimera, the LBD sequence of the PctA was obtained from the Pseudomonas genome database ,              while the signaling region of Tar was obtained from the iGEM parts catalog (K777000) . Using these two sequences, we built a Biobrick part (K1992007) which was tested by transforming the device to bacteria that lacks chemoreceptors - UU1250 . It is important             to note that this chimera has been constructed before in the literature (1).
Fig. 2: Biobrick device of the PctA-Tar chimera.
Test and results
As an initial step, we generated a 3D model of the PctA-Tar chimera, figure 3, using the            Phyre2 Protein Fold Recognition server to assure the correct folding of both the LBD and the signaling regions.
Fig. 3: PctA-Tar chimera 3D structure. The Tar signaling regions is in gray, the PctA LBD is in red.
Following transformation, a swarming plate assay was performed in order to            confirm the functionality of the hybrid receptor. A scheme of the assay is            presented below (figure 4). It is important to mention that this assay was            performed on BA medium as the original assay on TB medium failed. From the            results seen in figure 5, it is          clear that the chimera functions and controls the chemotactic ability of            the bacteria leading to swarming response.  This is compared to the control, UU1250 strain, that did not show any chemotactic ability as expected and no chemotactic rings were formed.
Fig. 4: a scheme for the swarming plate assay. In brief: Using a low percentage agar media plate,            add a drop of bacteria to the middle of the plate. Incubate at 30 degrees for several hours. A            halo or “chemotactic ring” should be formed in the agar plate.
a.
b.
c.
Fig. 5: Swarming assay for attractant response of the PctA-Tar chimera. a. PctA chimera, b. Negative control- UU1250 strain w/o the Tar expression plasmid, c. positive control - ΔZras strain expressing all chemoreceptors.
Next, to prove that the chimera is localized in the membrane, GFP was fused            to its C-terminus with a short linker sequence (K1992010) ,            figure 6. The results seen in figure 7, show that indeed the chimera is localized to the membrane (poles).
Fig. 6: Biobrick device of the PctA-Tar chimera fused to GFP.
Fig. 7: Results of GFP fusion. (A) Positive control- E.Coli strain expressing GFP protein,(B) Negative control- UU1250 strain expressing Tar chemoreceptor, (C) UU1250 strain expressing Tar-GFP chemoreceptor, (D) UU1250 strain expressing PctA-Tar-GFP Chimera, fluorescence (490nm excitation).
Finally, in the video below is a working concept of the FlashLab project - a chip that serves           as a detection tool based on the chemotaxis system of E. coli bacteria. In the video, a commercial ibidi            microfluidic chip filled with a suspension of bacteria expressing the chimera and chromoprotein           ( J23100 + K1357009 ) can been seen.            A solution of repellent (10-3M Tetrachloroethylene) was added to the chip           and the displacement of the bacteria was monitored and recorded.
Fig. 8: A steps scheme of the FlashLab concept: Add bacteria expressing both the chemoreceptor of your choice and a chromoprotein to a fluidic chip. Add the tested sample to the chip. If the chemoreceptor detecets the substance in the sample, a displacement of the bacteria will become visible.  .
In video 1, the displacement of the bacteria can be clearly seen in test chip compared to the control chip.
Video 1: from left to right:           (1) PctA-Tar chimera with Tetrachloroethylene repellent added.          (2) PctA-Tar chimera with Motility buffer added (control).
With the supporting evidence of the results presented above, it can be concluded           that both concepts: LBD altering and the FlashLab platform, have been proven and work under real life conditions and seem promising for detection of various substances in the near future.
Histamine-Tar Introduction
The heart of the S.Tar project is the Tar chemoreceptor, one of four E. coli receptors.           Our goal is to create an engineered bacteria which has chemotaxis receptors sensitive           to materials other than its native ligands. We show that E.            coli can be engineered to respond to completely new materials. By changing Tar’s ligand binding domain            (LBD) to other LBDs from various sources or by mutating it.
Fig. 1: Sequencing Results. "Query" describes the native Tar LBD            sequence and "Sbjct" describes the design mutations sequence. The mutation region is marked with color (blue or red).
The bacterial world offers a relatively small selection of chemoreceptors in comparison to          the vast number of possible ligands. These receptors evolved specifically to recognize substances           which benefit or harm the organism. On top of that the fact that the majority of known           receptors today are not well characterized meant that we had very few options of creating chimeric           receptors as we initially planned.
In light of the above we had to turn to a new path – redesigning the Tar chemoreceptor to bind a           different ligand using computational biology - The Rosetta software .          Out of the Rosetta’s 870 suggested mutations only 11 variants were eventually cloned into the native Tar ligand-binding domain (LBD).           See Computational Design page for more information regarding the design process.          Out of all the tested variants, only one was discovered to be attracted to Histamine. Sequencing results           showed that the only mutations to occur in this variant were those planned by the Rosetta’s design. The desired sequences can be seen in figure 1.
Fig. 2: Histamine-Tar filtering process scheme
Test and results
We observed the bacteria’s response to the attractant,            Histamine, by using a microscope. It is evident in figure 3b that roughly 20 minutes after the addition of the            Histamine, the bacteria concentration in the vicinity of the Histamine is much greater            than in the the beginning of the experiment (figure 3a).
a.
b.
c.
d.
Fig. 3: Microscope results of chemotaxis activity for variant His_9 with 10mM Histamine. (a) Tar-Histamine: before adding Histamine. (b) Tar-Histamine: 20 minutes after adding Histamine. (c) Tar-Histamine: before adding the Motility buffer (control solution). (d)  Tar-Histamine: 20 minutes after adding the Motility buffer.
To prove the correct localization of the Tar-Histamine, GFP was fused to its C-terminus with a short linker sequence (E0040) . The results seen in figure 4 show that indeed the chimera is localized to the membrane (poles).
Fig. 4: Results of GFP fusion. (A) Positive control- E.Coli strain expressing GFP protein, (B) Negative control- UU1250 strain expressing Tar chemoreceptor, (C) UU1250 strain expressing Tar-GFP chemoreceptor, (D) UU1250 strain expressing Histamine-Tar-GFP Chimera, fluorescence (490nm excitation).
Finally, in the video below, a working concept of the FlashLab project -            a chip that serves as a detection tool based on the chemotaxis system of E.            coli bacteria is presented. In the video, a commercial ibidi microfluidic chip filled with a suspension of            bacteria expressing both the chemoreceptor and chromoprotein           ( K1992011 + K1357008 ) can been seen.           A solution of attractant (10-3M Histamine)           was added to the chip and bacteria displacement was monitored and recorded
Video 1: from left to right:           (1) Histamine-Tar with Histamine attractant added.          (2) Histamine-Tar with Motility buffer added (control).
The results presented above, mainly proves the concept of the ability to alter the LBD of a chemoreceptor using software.          Moreover, these results might help with breakthroughs and lead to newly designed reporters for novel detection.
References:
1. Reyes-Darias, J.A., Yang, Y., Sourjik, V., and Krell, T. (2015). Correlation between signal input and output in PctA and PctB amino acid chemoreceptor of Pseudomonas aeruginosa. Mol. Microbiol. 96, 513–525.
2. Moretti, R., Bender, B.J., Allison, B. and Meiler, J., 2016. Rosetta and the Design      of Ligand Binding Sites. Computational Design of Ligand Binding Proteins, pp.47-62.
×Close
S.tar, by iGEM Technion 2016
