<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Technion_Israel_Modifications_narx skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Technion Israel/Modifications/narx</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>S.tar, by iGEM Technion 2016</TITLE></P><DIV class="navbar navbar-inverse navbar-fixed-top" style="margin-top: 10px; padding-top: 10px; padding-bottom: 10px;"><DIV class="container"><DIV class="collapse navbar-collapse"><UL class="nav navbar-nav"><LI><A href="https://2016.igem.org/Team:Technion_Israel/Description">Project</A></LI><LI class="dropdown"><A href="https://2016.igem.org/Team:Technion_Israel/S.Tar_intro" class="dropdown-toggle" data-toggle="dropdown">S.Tar</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/S.Tar_intro">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Tar_improvements">Tar Improvements and Characterization</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Modifications/Rosetta">Computational Design</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Proof">PctA-Tar Chimera</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Modifications/narx">NarX-Tar Chimera</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Modifications/EstoTar">hERα-Tar Chimera</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Modifications/Intein">Intein</A></LI></UL><LI class="dropdown"><A href="https://2016.igem.org/Team:Technion_Israel/Design" class="dropdown-toggle" data-toggle="dropdown">FlashLab</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Design">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Model">Modeling</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Hardware">Hardware</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Color">Chromoprotein</A></LI><LI class="dropdown"><A href="" class="dropdown-toggle" data-toggle="dropdown">Wet Lab</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Results">Highlight Results</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Experiments">Experiments</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Protocols">Protocols</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Notebook">Notebook</A></LI><LI class="dropdown"><A href="" class="dropdown-toggle" data-toggle="dropdown">Judging</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Proof">Proof of concept</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Parts">Biobricks</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Software">Software</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Measurement">Measurments</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Safety">Safety</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Medals">Medals</A></LI><LI class="dropdown"><A href="" class="dropdown-toggle" data-toggle="dropdown">People</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Human_Practices">Human Practice</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Team">Team Members</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Attributions">Attributions</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Collaborations">Collaborations</A></LI></DIV></DIV></DIV><TITLE>S.Tar, by iGEM Technion 2016</TITLE><DIV class="pcta_wrapper"><DIV class="row"><DIV class="col-sm-8 col-sm-offset-2"><UL class="nav nav-tabs" role="tablist"><LI role="presentation" class="col-sm-3 col-xs-6"><A href="#Peshawar" aria-controls="Peshawar" role="tab" data-toggle="tab"><H4 class="text-center">Introduction</H4></A></LI><LI role="presentation" class="col-sm-3 col-xs-6"><A href="#Aachen" aria-controls="Aachen" role="tab" data-toggle="tab"><H4 class="text-center">Design</H4></A></LI><LI role="presentation" class="col-sm-3 col-xs-6"><A href="#Results" aria-controls="Results" role="tab" data-toggle="tab"><H4 class="text-center">Results</H4></A></LI><LI role="presentation" class="col-sm-3 col-xs-6"><A href="#Conclusion" aria-controls="Conclusion" role="tab" data-toggle="tab"><H4 class="text-center">Conclusion</H4></A></LI></UL></DIV></DIV><DIV class="tab-content"><DIV role="tabpanel" class="tab-pane fade in active" id="Peshawar"><DIV class="row"><DIV class="col-sm-8 col-sm-offset-2"><DIV class="cont_box"><DIV class="row"><DIV class="col-sm-12"><H2>Introduction</H2></DIV></DIV><DIV class="row"><DIV class="col-md-6 col-sm-12 vcenter"><P class="text-justify">				As a proof of concept of our platform, the Tar chemoreceptor LBD was replaced with the NarX LBD. 
													The NarX sensor protein of <I>E. coli</I> binds nitrite and nitrate and induces the expression 
													of proteins involved in anaerobic respiration <B>(1)</B>.  We followed a protocol which was 
													previously shown to be successful <B>(1)</B>. 
													This work resulted in a NarX-Tar chimera, comprised by the NarX LBD and Tar’s cytoplasmic region. 
													According to the article the chimera was supposed to function as a repellent chemoreceptor to 
													nitrite and nitrate.
												</P></DIV><DIV class="col-md-6 col-sm-12 vcenter"><P class="text-center"><B>Fig. 1:</B> Scheme of Tar chemoreceptor on the left, NarX-Tar chimera on the right.</P></DIV></DIV></DIV></DIV></DIV></DIV><DIV role="tabpanel" class="tab-pane fade" id="Aachen"><DIV class="col-sm-8 col-sm-offset-2"><DIV class="cont_box"><DIV class="row"><DIV class="col-sm-12"><DIV class="col-md-12 col-sm-12"><H2 class="">Design and Implementation</H2><P class="text-justify">

To construct this chimera, the protein sequence of both the LBD and the linker region of NarX were obtained the literature <B>(1)</B>. 
Using the DNA sequences of both frgamnets and Tar’s cytoplasmic region, we built a Biobrick device (part not submitted) <B>(2)</B>. As in the PctA, first a 3D model was generated using the Phyre2 Protein Fold Recognition server, to ensure the correct folding of all parts of the chimera (Figure 1, results).
The next step was transforming to bacteria that lacks chemoreceptors - <A data-toggle="popover" data-trigger="click" data-original-title="Info:" data-html="true" data-content="An E.coli derivative, which lacks chemoreceptors genes, means this strain does not obtain chemotaxis ability (Parkinson J S, University of Utah).">
													UU1250</A>,
and to test the chemotactic ability of the cloned strain the following chemotaxis assays were performedSince the NarX-Tar chimera is supposed to serve as a repellent chemoreceptor, the conventional assay is a chemical in plug assay, in which the repellent is added to a motility buffer and the movement of the bacteria is detected (See <A href="https://2016.igem.org/Team:Technion_Israel/Experiments">chemical in plug assay</A>). Here sodium nitrate was used as a repellent for the NarX-Tar strain.Following, a  <A href="https://static.igem.org/mediawiki/2016/f/f3/T--Technion_Israel--Chemotaxis_assays.pdf" target="_blank">drop assay</A> was conducted. A suspension of bacteria in motility buffer was placed on a microscope slide and a drop of repellent was added in the concentrations of 10<SUP>-2</SUP>M and 10<SUP>-6</SUP>M. The slide was monitored throughout the entire procedure in order to examine the response of bacteria.
In addition, GFP was fused to the NarX-Tar chimera's C-terminus, in order to validate the location of the expressed chemoreceptor on both poles of the bacterial membrane. Flow cytometry analysis was conducted in order to verify the expression of the NarX-Tar chimera. We generated the NarX-Tar-GFP fusion gene that we planned to use for the localization assay. To verify the successful cloning of GFP to the NarX-Tar chimera, we screened 20 clones with flow cytometry.  
 



												</P><DIV class="col-md-12 col-sm-12"><P class="text-center"><B>Fig. 1:</B> Biobrick device of the NarX-Tar chimera.</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV role="tabpanel" class="tab-pane fade" id="Results"><DIV class="row"><DIV class="col-sm-8 col-sm-offset-2"><DIV class="cont_box"><DIV class="row"><DIV class="col-md-12 col-sm-12"><H2 class="">Results</H2></DIV></DIV><DIV class="row"><DIV class="col-md-12 col-sm-12"><P class="text-justify">
													The results of the 3D structure were not promising, as the folding was not 
													achieved in the correct manner, as can be seen in figure 1a. The 3D structure 
													of the native Tar is presented in figure 1b.
												</P></DIV></DIV><DIV class="row"><DIV class="col-sm-12"><DIV class="col-md-6 col-sm-12"><P class="text-justify">a.</P></DIV><DIV class="col-md-6 col-sm-12"><P class="text-justify">b.</P></DIV><DIV class="col-md-12 col-sm-12"><P class="text-center"><B>Fig. 1:</B><B>(a)</B>NarX-Tar chimera 3D structure. The Tar signaling regions is in gray, the NarX LBD is in red. <B>(b)</B>Native Tar 3D structure.</P></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-12 col-sm-12"><P class="text-justify">
												When tested on the chemical in plug assay no movement was detected, indicating that
												the NarX-Tar strain is not responsive to sodium nitrate. (figure 2)
											</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 col-sm-12"><P class="text-center"><B>Fig. 2:</B> Chemical in plug assay results: on the left, 
												positive control- ΔZ in the presence of the attractant aspartic acid. On 
												the rightNarX-Tar strain with repellent sodium nitrate.</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 col-sm-12"><P class="text-justify">
												The expected results of the drop assay were a rapid tumbling response of the bacteria. 
												Nevertheless, the strain showed no response to different concentrations of sodium 
												nitrate and unfortunately a sufficient documentation hasn't been obtained. 
												Lastly, the testing of the clone carrying the GFP gene fused to the chimera with flow 
												cytometry showed no indication to fluorescence. In figure 3, we plotted each clone as 
												a single histogram (light green histograms) and compared them with the histograms of a 
												positive control (Pct-Tar-GFP; dark green) and a negative control (NarX-Tar only, black). 
												As comparison between the single histograms is tedious, we plotted all histograms in 
												one figure for easier comparison. As can be seen in figure 4, the fluorescence of 
												NarX-Tar-GFP is comparable with the fluorescence of the  negative strain indicating that 
												all colonies could not been cloned successfully. Note that the negative control showed 
												two peaks, one peak which was defined as negative, and the second, albeit smaller peak 
												overlapped with the fluorescence of PctA-Tar (positive control). This is probably due 
												to a very high concentration of bacteria (event rate: 21000 events/sec). As a result, 
												the samples were diluted in the ratio of 1:4 in order to avoid false positive results.
											</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 col-sm-12"><P class="text-center"><B>Fig. 3:</B> Flow cytometry results.</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 col-sm-12"><P class="text-center"><B>Fig. 4:</B> A comparison between NarX colonies (light green), positive control (dark green) and negative (black) control.</P></DIV></DIV></DIV></DIV></DIV></DIV><DIV role="tabpanel" class="tab-pane fade" id="Conclusion"><DIV class="row"><DIV class="col-sm-8 col-sm-offset-2"><DIV class="cont_box"><DIV class="row"><DIV class="col-sm-12"><H2 class="">Conclusion</H2></DIV></DIV><DIV class="row"><DIV class="col-sm-12"><P class="text-justify">
													As the results showed, NarX-Tar strain failed to show any response to sodium 
													nitrate. Moreover, as the reporter gene (GFP) showed no indication of expression, 
													this subproject was put aside.
													It is unfortunate that the NarX-Tar clone was not successful, as it could have 
													served as an additional proof of concept the S.Tar platform, in addition to the 
													other strains we have cloned. 
 												</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="modal fade" id="imagemodal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel" aria-hidden="true"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><BUTTON type="button" class="close" data-dismiss="modal"><SPAN aria-hidden="true">×</SPAN><SPAN class="sr-only">Close</SPAN></BUTTON></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-sm-8 col-sm-offset-2"><P class="references">
							References:
							1. R Ward, S.M., Delgado, A., Gunsalus, R.P., and Manson, M.D. (2002). A NarX-Tar chimera mediates repellent chemotaxis to nitrate and nitrite. Mol. Microbiol. 44, 709–719.
							2. <A href="http://www.ecogene.org/sources/1" target="_blank">The complete <I>E. coli</I> genome sequence.</A></P></DIV></DIV></DIV><TITLE>S.tar, by iGEM Technion 2016</TITLE></DIV></DIV></DIV></DIV></DIV></BODY></HTML>