Team:Technion Israel/Results
S.tar, by iGEM Technion 2016
Project
S.Tar
Overview
Tar Improvements and Characterization
Computational Design
PctA-Tar Chimera
NarX-Tar Chimera
hERα-Tar Chimera
Intein
FlashLab
Overview
Modeling
Hardware
Chromoprotein
Wet Lab
Highlight Results
Experiments
Protocols
Notebook
Judging
Proof of concept
Biobricks
Software
Measurments
Safety
Medals
People
Human Practice
Team Members
Attributions
Collaborations
S.tar, by iGEM Technion 2016
Histamine-Tar novel chemoreceptor
We successfully redesigned, cloned and tested a novel Histamine-Tar            chemoreceptor . The design was done with the Rosetta bioinformatics tool,            by following the protocol published by Moretti, R. et al. (1). The goal was to design a binding pocket for a selected small ligand.  The Rosetta’s design process for the new ligand, Histamine, produced 870 results, out of which 11 variants remained after filtering. The 11 variants were cloned into the native Tar ligand binding domain (LBD), out of these only 6 exhibited the expected sequences in sequencing and thus were subjected to chemotaxis tests. Out of all 6 tested variants only one showed chemotactic response, the ability to be attracted to Histamine. Furthermore, the sequencing results proved that the only mutations that were present in this variant were those planned by the Rosetta’s design.
Fig. 1: Histamine-Tar filtering process scheme.
We observed the bacteria’s response to the attractant, Histamine, by using a microscope assay. It is evident in figure 3b that roughly 20 minutes after the addition of the Histamine, the bacterial concentration in the vicinity of the Histamine is much greater than in the beginning of the experiment (figure 3a).
Fig. 2: Sequencing Results. "Query" describes the native Tar LBD            sequence and "Sbjct" describes the design mutations sequence. The mutation region is marked with color (blue or red).
a.
b.
c.
d.
Fig. 3: Microscope results of chemotaxis activity for variant His_9 with 10mM Histamine. a. Tar-Histamine: before adding Histamine. b. Tar-Histamine:            20 minutes after adding Histamine. c. Tar-Histamine: before adding the            Motility buffer (control solution). d. Tar-Histamine: 20 minutes after            adding the Motility buffer.
Histamine-Tar fused with GFP marker:
To prove the correct localization of the Tar-Histamine,           GFP was fused to its C-terminus with a short linker sequence (E0040) .           The results of these tests as seen in figure 4, prove our assumption           of correct localizations.
Fig. 4: Results of GFP fusion. a. Positive control- E. Coli strain expressing GFP protein, b. Negative control- UU1250 strain expressing Tar chemoreceptor, c. UU1250 strain expressing Tar-GFP chemoreceptor, d. UU1250 strain expressing Histamine-Tar-GFP Chimera,           fluorescence (490nm excitation).
Finally, in the following video, a working concept of the FlashLab project -            a chip that serves as a detection tool based on the chemotaxis system of E. coli bacteria. In the video, a commercial ibidi microfluidic chip filled with a suspension of bacteria expressing the chemoreceptor and chromoprotein            ( K1992011 + K1357008 ) can been observed.           A solution of attractant (10-3M Histamine)           was added to the chip and bacteria displacement was monitored and recorded
Video 1: from left to right:           (1) Histamine-Tar with Histamine attractant added.          (2) Histamine-Tar with Motility buffer added (control).
As we demonstrated, Rosetta provides us with the means to           redesign a chemoreceptor to bind new ligands. In the           future this ability can be used in the same manner to           design dozens of new receptors. The critical step of the           design process remains the lab work required to clone and          test the variants, this step can be optimized by using a high           throughput chemotaxis assay. Aside from this, any receptor designed           can be further improved by introducing a directed evolution step to           improve its specificity towards the new ligand.
In addition, we wrote a “How to use” Rosetta guide in cooperation with the Rosetta developers and          iGEM TU Eindhoven team, and built a framework software tool to allow fellow researchers the ability to redesign a binding site easily.
FlashLab - microfluidic chip
We successfully demonstrated a working concept of the FlashLab project - a chip (commercial ibidi chip) that serves as a detection tool based on the chemotaxis system of E. coli bacteria.
This chip is filled with a suspension of bacteria expressing a chromoprotein, enabling easy visualization of the bacteria, and a chemoreceptor engineered to respond to a specific ligand of your choice. Once a sample containing the ligand is inserted into the chip, the bacteria will respond accordingly leading a chemotactic movement towards or away from the sample. This movement causes the bacteria to form a visible cluster within the chip, due to the chromoprotein, which is seen as a darker shade of color.
This concept has been successfully proven for both PctA-Tar chimera and Histamine-Tar variant of the Rosetta software.
In video 2, the displacement of the bacteria can be clearly seen in the test chip (left chip, PctA-Tar with repellent), compared to the control chip (right chip, PctA-Tar with buffer).
Video 2: from left to right:           (1) PctA-Tar chimera with Tetrachloroethylene repellent added.          (2) PctA-Tar chimera with Motility buffer added (control).
In conclusion, we proved the concept of the FlashLab project, by demonstrating the color gradient being formed due to chemotaxis.
PctA-Tar chimera
We have successfully confirmed the functionality of the hybrid           PctA-Tar receptor using a swarming assay. From the results seen           below, figure 5, and compared to the negative control. it is clear that the chimera functions and controls the chemotactic ability of the bacteria and mediates to a swarming response. This is compared to the control where no swarming can be seen.
a.
b.
c.
Fig. 5: Swarming assay for attractant response of the PctA-Tar chimera. a. PctA chimera, b. Negative control- UU1250 strain w/o the Tar expression plasmid, c. positive control - ΔZras strain expressing all chemoreceptors.
PctA-Tar fused with GFP marker:
In addition, the correct localization of the chimera on both poles of the bacterial membrane, was demonstrated by fusing GFP to the C- terminus of the chimera with a short linker sequence (K1992010) . The results of these tests as seen in figure 6,           indeed show the expected localizations.
Fig. 6: Results of GFP fusion. a. Positive control- E.Coli strain expressing GFP protein,b. Negative control- UU1250 strain expressing           Tar chemoreceptor, c. UU1250 strain expressing Tar-GFP chemoreceptor, d. UU1250 strain expressing PctA-Tar-GFP Chimera, Flourcense (490nm excitation).
Trap and Track - Novel assay for digital chemotaxis detection
Although there is an abundant number of chemotaxis assays available            today, most of them were designed 40 to 50 years ago and almost            none provide a real time measurement without the use of fluorescence labeling.
The use of Porous Si (PSi) and oxidized PSi (PSiO2) matrices for biological            sensing is on the rise. So far various analytes such as DNA, proteins and            bacteria have been proven to be detectable on such matrices. The common            method to monitor the interaction of said analytes within the porous films            is reflective interferometric Fourier transform spectroscopy (RIFTS), as it            allows a real time measurement and output for the user.
We present the results of an early experiment for the detection of            chemotactic activity on the porous silicon films initially developed for bacterial detection.
This assay provides us with the means to digitally quantify chemotaxis -            meaning we have a clear distinction between an attractant response            (negative trend), repellent response (positive trend) and no response (zero trend).
The following graph presents only the chemotactic responses taken from 4            different experiments, The top 2 plots, in blue and in grey, represent a            repellent response. The yellow plot represents a negative control experiment            and the bottom red graph represents an attractant response.
For more information see our Measurement page .
Fig. 7: Digital chemotaxis measurements.
References:
1. Moretti, R., Bender, B.J., Allison, B. and Meiler, J., 2016. Rosetta and the Design of Ligand Binding Sites. Computational Design of Ligand Binding Proteins, pp.47-62
×Close
S.tar, by iGEM Technion 2016
