<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Technion_Israel_Modifications_Intein skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Technion Israel/Modifications/Intein</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>S.tar, by iGEM Technion 2016</TITLE></P><DIV class="navbar navbar-inverse navbar-fixed-top" style="margin-top: 10px; padding-top: 10px; padding-bottom: 10px;"><DIV class="container"><DIV class="collapse navbar-collapse"><UL class="nav navbar-nav"><LI><A href="https://2016.igem.org/Team:Technion_Israel/Description">Project</A></LI><LI class="dropdown"><A href="https://2016.igem.org/Team:Technion_Israel/S.Tar_intro" class="dropdown-toggle" data-toggle="dropdown">S.Tar</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/S.Tar_intro">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Tar_improvements">Tar Improvements and Characterization</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Modifications/Rosetta">Computational Design</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Proof">PctA-Tar Chimera</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Modifications/narx">NarX-Tar Chimera</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Modifications/EstoTar">hERα-Tar Chimera</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Modifications/Intein">Intein</A></LI></UL><LI class="dropdown"><A href="https://2016.igem.org/Team:Technion_Israel/Design" class="dropdown-toggle" data-toggle="dropdown">FlashLab</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Design">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Model">Modeling</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Hardware">Hardware</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Color">Chromoprotein</A></LI><LI class="dropdown"><A href="" class="dropdown-toggle" data-toggle="dropdown">Wet Lab</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Results">Highlight Results</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Experiments">Experiments</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Protocols">Protocols</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Notebook">Notebook</A></LI><LI class="dropdown"><A href="" class="dropdown-toggle" data-toggle="dropdown">Judging</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Proof">Proof of concept</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Parts">Biobricks</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Software">Software</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Measurement">Measurments</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Safety">Safety</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Medals">Medals</A></LI><LI class="dropdown"><A href="" class="dropdown-toggle" data-toggle="dropdown">People</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Human_Practices">Human Practice</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Team">Team Members</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Attributions">Attributions</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Collaborations">Collaborations</A></LI></DIV></DIV></DIV><TITLE>S.tar, by iGEM Technion 2016</TITLE><DIV class="intein_wrapper"><DIV class="row"><DIV class="col-sm-8 col-sm-offset-2"><UL class="nav nav-tabs" role="tablist"><LI role="presentation" class="col-sm-3 col-xs-6"><A href="#Peshawar" aria-controls="Peshawar" role="tab" data-toggle="tab"><H4 class="text-center">Introduction</H4></A></LI><LI role="presentation" class="col-sm-3 col-xs-6"><A href="#Aachen" aria-controls="Aachen" role="tab" data-toggle="tab"><H4 class="text-center">Design</H4></A></LI><LI role="presentation" class="col-sm-3 col-xs-6"><A href="#Results" aria-controls="Results" role="tab" data-toggle="tab"><H4 class="text-center">Results</H4></A></LI><LI role="presentation" class="col-sm-3 col-xs-6"><A href="#outlook" aria-controls="outlook" role="tab" data-toggle="tab"><H4 class="text-center">Outlook</H4></A></LI></UL></DIV></DIV><DIV class="tab-content"><DIV role="tabpanel" class="tab-pane fade in active" id="Peshawar"><DIV class="row"><DIV class="col-sm-8 col-sm-offset-2"><DIV class="cont_box"><DIV class="row"><DIV class="col-sm-12"><DIV class="col-md-12 col-sm-12"><H2 class="">Introduction</H2><P class="text-justify">
The bacterial world offers a relatively small selection of chemoreceptors, in comparison to the vast number of possible ligands. These receptors evolved specifically to recognize substances, which benefit or harm the bacteria in some way. On top of that, the fact that the majority of known receptors today are not well characterized, meant that we had very few options of creating chimeric receptors like we initially planned.
													In our attempts to expand our system and design new chemoreceptors we decided to direct our search to new organisms. This required us to attempt a new approach that was proven in literature <B>(3)</B>. According to it, it would be possible to take receptors from other organisms, specifically humans, and generate a switching mechanism by using an intein protein. As a proof of concept we tried the human Estrogen receptor - hERɑ that binds estrogenic compounds.
													</P></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-sm-8 col-sm-offset-2"><DIV class="cont_box"><DIV class="row"><DIV class="col-sm-12"><DIV class="col-md-4 col-sm-12"><H2 class="">Inteins</H2><P class="text-justify">
													An intein is a protein element found as an in-frame insertion within the sequence of a particular host gene. 
													It possesses the ability to excise itself post-translationally from its protein host, and ligate the flanking peptide sequences 
													with a native peptide bond via a very efficient self-catalyzed reaction, called protein splicing. 
													</P></DIV><DIV class="col-md-8 col-sm-12 new-row no-title-col"><P class="text-center"><B>Fig. 1:</B> Intein splicing scheme, adapted from ELLEUCHE, Skander and PÖGGELER, Stefanie <B>(2)</B>.</P></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-sm-8 col-sm-offset-2"><DIV class="cont_box"><DIV class="row"><DIV class="col-sm-12"><DIV class="col-md-12 col-sm-12"><H2 class="">Conditional protein splicing</H2><P class="text-justify">
													Splicing is the name of the process in which an intein excises itself out of the flanking protein segments. 
													This process can be conditioned to be activated by an external trigger, such as a small molecule, light or even temperature <B>(7)</B>. 
													We attempted to engineer a molecule controlled intein. Such a construct was successfully designed in the past, by fusing a ligand binding domain into the intein <B>(7)</B>. 
													This domain undergoes a conformational change upon ligand binding and induces splicing by doing so. Using this method, we can remain compatible with S.Tar's overall approach of changing ligand binding domains. By changing the 
														<A data-toggle="popover" data-trigger="click" data-original-title="Info:" data-html="true" data-content="Ligand Binding Domain">
														LBD</A>, 
													which is fused into the intein, it would become sensitive to different substances.
													
													</P></DIV><DIV class="col-md-12 col-sm-12 new-row no-title-col"><P class="text-center"><B>Fig.2:</B> Recent advances in <I>in vivo</I> applications of intein-mediated protein splicing <B>(7)</B>.</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV role="tabpanel" class="tab-pane fade" id="Aachen"><DIV class="row"><DIV class="col-sm-8 col-sm-offset-2"><DIV class="cont_box"><DIV class="row"><DIV class="col-sm-12"><DIV class="row"><DIV class="col-md-12 col-sm-12"><H2 class="">Detection using chemotaxis and inteins</H2></DIV></DIV><DIV class="col-md-5 col-sm-12"><P class="text-justify">
															One of the LBDs used with inteins is that of hERɑ - a human Estrogen receptor, which was inserted into an intein and fused to lacZ, as a proof of concept in the literature <B>(2,6)</B>.
															We tried the same construct, the only difference being that we fused the intein into the Tar receptor in critical locations, which will render Tar inactive - <B>“off mode”</B>. 
															Upon contact with an estrogenic compound - the hERɑ-LBD will bind to it, and undergo a conformational change, which will induce intein splicing. 
															During the process the intein will ligate the Tar back to it’s native form and enable chemotaxis - <B>“On mode”</B>.
														</P></DIV><DIV class="row"><P class="text-justify"><B>Fig. 1:</B><B>a.</B> Plasmid construct containing the intain gBlock in the Tar chemoreceptor.<B>b.</B> General form of Tar chemoreceptor with and without the intein.<B>c.</B> The overall switching strategy.</P></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-sm-8 col-sm-offset-2"><DIV class="cont_box"><DIV class="row"><DIV class="col-sm-12"><DIV class="col-md-12 col-sm-12"><H2 class="">Our intein switch</H2><P class="text-justify">
													The <I>Saccharomyces cerevisiae</I> VMA intein has been shown to retain its activities after being transferred into non native genes or cell hosts.
													The VMA intein contains two regions: 
													1.  Splicing region.
													2. Endonuclease region.
													The latter one is not essential for protein splicing.
													Initially, we found a Biobrick of an intein with hERɑ-LBD designed by <A href="http://parts.igem.org/Part:BBa_K1491024" target="_blank">Carnegie Mellon iGEM2014 team</A>. 
													However, this design was not successful in their experiments, thus we designed an original gBlock using the VMA intein in two steps:
													1. The endonuclease region of the VMA intein (which is not essential for splicing) was replaced with a cDNA coding for the LBD of the human estrogen receptor <B>(8)</B>.
													2. Four restriction sites (EcoRI, XbaI, SptI, PstI) were deleted by silent mutations (These enzymes exist in iGEM plasmid backbones pSB1C3).</P></DIV><DIV class="col-md-12 col-sm-12 new-row no-title-col"><P class="text-center"><B>Fig. 2:</B> Basic structure of our designed gBlock with the hERɑ ligand binding domain, instead of the original endonuclease region of the intein.</P></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-sm-8 col-sm-offset-2"><DIV class="cont_box"><DIV class="row"><DIV class="col-sm-12"><DIV class="col-md-12 col-sm-12"><H2 class="">Procedure</H2><P class="text-justify">
													Two separate strains were generated by inserting the intein-gBlock in two different locations in the Tar receptor, 
													following the requirements for efficient protein splicing: 
													- The first amino acid of the C-terminal extein should be a cysteine (Cys), serine (Ser), or threonine (Thr).
													- The last amino acid of the N-terminal extein should be alanine (Ala) <B>(1,3)</B>.
													</P></DIV><DIV class="col-md-12 col-sm-12 new-row no-title-col"><P class="text-center"><B>Fig. 3:</B> Demonstration of insertion of intein gBlock into Tar chemoreceptor, with the requirement for efficient protein splicing.</P><P class="text-justify">
													The aim was to insert the intein into the LBD region, rather than the cytoplasmic region, due to several reasons:<B>1.</B> Prevent interruption of the signal peptide (the sequence responsible for signaling the cell to migrate the protein to the membrane), that is assumed to be in the C-terminus of the protein.<B>2.</B> Avoid interruption of the transmembrane region.
													Two regions in Tar-LBD have been found to answer the requirements, and for each we modeled the structure of Tar-intein complex, using Phyre2 Fold Recognition server <B>(4)</B>:
													</P></DIV><DIV class="col-md-12 col-sm-12 new-row no-title-col"><P class="text-center"><B>Fig. 4:</B><B>a.</B> Location 1 - between Ala101 and Cys102. <B>b.</B> Location 2 - between Ala118 and Cys119. 
												<B>c.</B> Native Tar receptor.</P><DIV class="col-md-12 col-sm-12 new-row no-title-col"><H2>Challenges</H2><P class="text-justify">
														As far as we know, this approach has never been done in any research, so we were heading into unknown territory. 
														There were a few major questions that needed to be answered:<B>1.</B> Will Tar-intein chimera reach its proper location in the cell membrane? <B>2.</B> What happens if the conformational change is happening, while the chemoreceptor is in the membrane? <B>3.</B> Will the intein splice out when it binds estrogen? <B>4.</B> Will Tar fold correctly after intein splicing? </P><H2>Designed experiments</H2><P class="text-justify">
														Several experiments are required to answer said questions.<B>1.</B> In order to verify that the Tar-intein chimera migrated to the proper location in the cell membrane, 
														a GFP will fused to the C-terminus of Tar. This will enable tracking of the protein in the cell by using fluorescence microscopy. 
														<B>2.</B> To validate our switching mechanism - the new <I>E.coli</I> strains will be incubated with 17-beta-estradiol for 2 hours to trigger intein splicing. 
														Afterwards, a chemotaxis assay will be performed with Aspartate (the native Tar ligand) as an attractant, to check whether 
														the strain has an active Tar, and is able to move.
													</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV role="tabpanel" class="tab-pane fade" id="Results"><DIV class="row"><DIV class="col-sm-8 col-sm-offset-2"><DIV class="cont_box"><DIV class="row"><DIV class="col-sm-12"><H2>Results</H2><P class="text-justify">
												We were unable to insert the intein-gBlock in the Tar receptor via Gibson assembly. Only a short part of the intein was succefully assembled. 
												We hypothesized that the intein might be too long (1575 bp), which is why new intein-gBlocks were generated, by splitting the original-intein-gBlock into two parts. 
												An attempt to insert these parts into Tar by Gibson assembly has failed as well. 
												In our third and final attempt, we used a different cloning method - blunt ligation. In this attempt, the original-intein-gBlock 
												(one part) was used. This method unfortunatly did not work as well.
												</P></DIV></DIV></DIV></DIV></DIV></DIV><DIV role="tabpanel" class="tab-pane fade" id="outlook"><DIV class="row"><DIV class="col-sm-8 col-sm-offset-2"><DIV class="cont_box"><DIV class="row"><DIV class="col-sm-12"><H2>Outlook</H2><P class="text-justify">
Long fragment cloning is often challenging, due to the fact that cloning efficiency is reduced with fragment size. 
Using optimized cloning products might be essential.
As cloning methods did not work well, the next step is to assemble the two-parts intein gBlocks via Gibson assembly, 
in order to test whether it can be assembled to form a single intein segment. Following that, the segment will be cloned
into Tar using blunt ligation strategy.
We believe this novel approach has the potential to produce new <I>E.coli</I> strains which contain a Tar-switch.
Except for material detection, this approach can also be used as a biological AND gate -  
where we could verify if two specific materials are found in a sample. 
												</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="modal fade" id="imagemodal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel" aria-hidden="true"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><BUTTON type="button" class="close" data-dismiss="modal"><SPAN aria-hidden="true">×</SPAN><SPAN class="sr-only">Close</SPAN></BUTTON></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-sm-8 col-sm-offset-2"><P class="references">
References:
1. CHONG, Shaorong; XU, Ming-Qun. Protein splicing of the Saccharomyces cerevisiae VMA intein without the endonuclease motifs. Journal of Biological Chemistry, 1997, 272.25: 15587-15590.‏ 
2. ELLEUCHE, Skander; PÖGGELER, Stefanie. Inteins, valuable genetic elements in molecular biology and biotechnology. Applied Microbiology and Biotechnology, 2010, 87.2: 479-489.‏ 
3. LIANG, Rubing; ZHOU, Jing; LIU, Jianhua. Construction of a bacterial assay for estrogen detection based on an estrogen-sensitive intein. Applied and environmental microbiology, 2011, 77.7: 2488-2495.‏ 
4. Phyre2 modeling server <A href="http://www.sbg.bio.ic.ac.uk/phyre2/html/page.cgi?id=index" target="_blank">online</A>. 
5. SHIOMI, Daisuke, et al. Helical distribution of the bacterial chemoreceptor via colocalization with the Sec protein translocation machinery. Molecular microbiology, 2006, 60.4: 894-906.‏ 
6. SKRETAS, Georgios; WOOD, David W. Regulation of protein activity with small‐molecule‐controlled inteins. Protein Science, 2005, 14.2: 523-532.‏ 
7. TOPILINA, Natalya I.; MILLS, Kenneth V. Recent advances in in vivo applications of intein-mediated protein splicing. Mobile Dna, 2014, 5.1: 1.‏ 
8. <A href="http://www.uniprot.org/uniprot/P03372" target="_blank">Uniprot‏</A></P></DIV></DIV></DIV><TITLE>S.tar, by iGEM Technion 2016</TITLE></DIV></DIV></DIV></DIV></DIV></BODY></HTML>