<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Technion_Israel_Modifications_Rosetta skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Technion Israel/Modifications/Rosetta</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>S.tar, by iGEM Technion 2016</TITLE></P><DIV class="navbar navbar-inverse navbar-fixed-top" style="margin-top: 10px; padding-top: 10px; padding-bottom: 10px;"><DIV class="container"><DIV class="collapse navbar-collapse"><UL class="nav navbar-nav"><LI><A href="https://2016.igem.org/Team:Technion_Israel/Description">Project</A></LI><LI class="dropdown"><A href="https://2016.igem.org/Team:Technion_Israel/S.Tar_intro" class="dropdown-toggle" data-toggle="dropdown">S.Tar</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/S.Tar_intro">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Tar_improvements">Tar Improvements and Characterization</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Modifications/Rosetta">Computational Design</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Proof">PctA-Tar Chimera</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Modifications/narx">NarX-Tar Chimera</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Modifications/EstoTar">hERα-Tar Chimera</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Modifications/Intein">Intein</A></LI></UL><LI class="dropdown"><A href="https://2016.igem.org/Team:Technion_Israel/Design" class="dropdown-toggle" data-toggle="dropdown">FlashLab</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Design">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Model">Modeling</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Hardware">Hardware</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Color">Chromoprotein</A></LI><LI class="dropdown"><A href="" class="dropdown-toggle" data-toggle="dropdown">Wet Lab</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Results">Highlight Results</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Experiments">Experiments</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Protocols">Protocols</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Notebook">Notebook</A></LI><LI class="dropdown"><A href="" class="dropdown-toggle" data-toggle="dropdown">Judging</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Proof">Proof of concept</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Parts">Biobricks</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Software">Software</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Measurement">Measurments</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Safety">Safety</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Medals">Medals</A></LI><LI class="dropdown"><A href="" class="dropdown-toggle" data-toggle="dropdown">People</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Human_Practices">Human Practice</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Team">Team Members</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Attributions">Attributions</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Collaborations">Collaborations</A></LI></DIV></DIV></DIV><TITLE>S.tar, by iGEM Technion 2016</TITLE><DIV class="desk_wrapper"><DIV class="row"><DIV class="col-sm-8 col-sm-offset-2"><DIV class="row"><DIV class="col-md-12"><UL class="nav nav-tabs" role="tablist"><LI role="presentation" class="col-sm-2 col-xs-4"><A href="#111" aria-controls="111" role="tab" data-toggle="tab"><H4 class="text-center">Intro</H4></A></LI><LI role="presentation" class="col-sm-2 col-xs-4"><A href="#222" aria-controls="222" role="tab" data-toggle="tab"><H4 class="text-center">Design</H4></A></LI><LI role="presentation" class="col-sm-2 col-xs-4"><A href="#333" aria-controls="333" role="tab" data-toggle="tab"><H4 class="text-center">Auto- mation</H4></A></LI><LI role="presentation" class="col-sm-2 col-xs-4"><A href="#444" aria-controls="444" role="tab" data-toggle="tab"><H4 class="text-center">Newligands</H4></A></LI><LI role="presentation" class="col-sm-2 col-xs-4"><A href="#555" aria-controls="555" role="tab" data-toggle="tab"><H4 class="text-center">Results</H4></A></LI><LI role="presentation" class="col-sm-2 col-xs-4"><A href="#666" aria-controls="666" role="tab" data-toggle="tab"><H4 class="text-center">Guide</H4></A></LI></UL></DIV></DIV><DIV class="tab-content"><DIV role="tabpanel" class="tab-pane fade in active" id="111"><DIV class="cont_box"><DIV class="row"><DIV class="col-md-12 col-sm-12"><H1>Computational Design of Ligand Binding Sites</H1></DIV></DIV><DIV class="row"><DIV class="col-md-12 col-sm-12"><P class="text-justify">
									The bacterial world offers a relatively small selection of chemoreceptors, in comparison 
									to the vast number of possible ligands. These receptors evolved specifically to recognize 
									substances, which benefit or harm the organism in some way. On top of that, the fact that 
									the majority of known receptors today are not well characterized, meant that we had a very few 
									options of designing chimeric receptors like we initially planned.
									In light of the above we had to turn to a new path – redesigning the Tar chemoreceptor 
									to bind a different ligand using computational biology with the Rosetta software.</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 col-sm-12"><H2>Rosetta</H2></DIV></DIV><DIV class="row"><DIV class="col-md-6 col-sm-12 vcenter"><P class="text-left">
									Rosetta is a bioinformatics software for macromolecular modeling and design, created 
									by the <A href="https://www.rosettacommons.org/" target="_blank">Rosetta Commons</A> organization - a collaboration between several universities and 
									research groups from around the world.</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 col-sm-12"><P class="text-justify">
									Rosetta development began in the laboratory of Dr. David Baker at the University of 
									Washington as a structure prediction tool but since then has been expanded to solve 
									many different computational macromolecular problems.
									As of 2016, Rosetta algorithms have been used to predict, design and analyze almost 
									every set of biomolecular systems: proteins, RNA, DNA, peptides, small molecules and 
									non-canonical amino acids.
									</P></DIV></DIV><DIV class="row"><DIV class="col-md-6 col-sm-12 vcenter"><H2>Local installation of Rosetta</H2><P class="text-justify">
									 	For complex problems such as redesigning a protein, 
										Rosetta requires more computational power than a regular PC 
										has to offer. While searching for possible computing resources, 
										we came across the local Technion grid of WLCG.</P></DIV><DIV class="col-md-6 col-sm-12 vcenter"><P class="text-center"><B>Fig. 1:</B> WLCG computing grid.</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 col-sm-12"><P class="text-justify"><A href="http://wlcg-public.web.cern.ch/" target="_blank">The Worldwide LHC Computing Grid (WLCG)</A>  is a 
									global collaboration of more than 170 computing centers in 42 countries, linking up national 
									and international grid infrastructures. It was launched in 2002 to provide a resource to 
									store, distribute and analyze the 15 petabytes (15 million gigabytes) of data generated every 
									year by the world’s largest and most powerful particle accelerator - The Large Hadron Collider (LHC).
									In Israel, there are three computing centers connected to the grid, located in the Technion Institute, 
									Tel Aviv University and Weizmann Institute.
									WLCG supports not only the particle accelerator, but also allows unregister users to benefit from 
									this amazing project. We contacted the local Technion grid administrator and received a temporary user on the Atlas server (one of four particle accelerator components). This granted us access to vast computational power, much more than was necessary. With the help of David Cohen, grid computing specialist from the Technion physics department, we successfully installed Rosetta and all required programs.
									</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 col-sm-12"><P class="text-center"><B>Fig. 2:</B> WLCG distribution. Yellow dots represent different computing centers.</P></DIV></DIV></DIV></DIV><DIV role="tabpanel" class="tab-pane fade" id="222"><DIV class="cont_box"><DIV class="row"><DIV class="col-md-12 col-sm-12"><H1>Designing a binding site</H1></DIV></DIV><DIV class="row"><DIV class="col-md-4 col-sm-12 vcenter"><P class="text-left">
									To redesign the Tar chemoreceptor we followed the protocol presented in &quot;Rosetta and the 
									Design of Ligand Binding Sites&quot;, <B>(1)</B>.  The purpose of the protocol is to design a binding 
									site around a selected small molecule ligand. The general steps of the protocol can be 
									seen in the flowchart to the right.
									Using this protocol we managed to generate a library of mutated Tar receptors that theoretically 
									bind a substance in a novel way and activate the chemotaxis pathway in response to it. For each 
									design we ran 3-5 iterations of the protocol to assure optimal results.
									</P></DIV><DIV class="col-md-8 col-sm-12 vcenter"><P class="text-center"><B>Fig. 1:</B> Flowchart of the ligand binding domain design protocol.</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 col-sm-12"><H2>Filtering Process</H2></DIV></DIV><DIV class="row"><DIV class="col-md-12 col-sm-12"><P class="text-justify">
									The output of the protocol is a library of variants, ranging from dozens to even thousands 
									of protein PDB files, depending on the parameters of the design run. This fact means that 
									filtering the results is an extremely crucial part of the process.
									Rosetta is able to predict which protein designs are likely to have improved protein 
									activity. This is done by measuring every aspect of the protein complex such as binding 
									energies, interactions between amino acids, backbone angles, hydrogen bonds and more. After 
									the calculation process the user can decide which parameters are relevant and to discard 
									the lowest results obtained for the mentioned parameters. The specific filters we used in our designs 
									can be seen in this <A href="https://static.igem.org/mediawiki/2016/3/3f/Rosetta_scores.pdf" target="_blank">attachment</A>.</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 col-sm-12"><H2>First run - Benchmark Test</H2></DIV></DIV><DIV class="row"><DIV class="col-md-12 col-sm-12 "><P class="text-justify">
									As a test phase, before advancing to more complex designs, 
									we ran the protocol with Aspartic acid which is Tar receptor's native ligand. This was done in order to make sure that Rosetta can &quot;handle&quot; 
									the Tar protein, meaning it does not create unnecessary or drastic changes in the protein.
									From this design process we received four output structures (after filtering) with 3-5 
									mutations each, all of which in the binding pocket. These results proved that Rosetta 
									can recognize and work with the Tar ligand-binding domain (LBD).
									</P></DIV><DIV class="col-md-12 col-sm-12 "><P class="text-center"><B>Fig. 2:</B> Design results after filtration for native Tar ligand- Aspartic acid.</P></DIV></DIV></DIV></DIV><DIV role="tabpanel" class="tab-pane fade" id="333"><DIV class="cont_box"><DIV class="row"><DIV class="col-md-12 col-sm-12"><H1>Protocol automation</H1></DIV></DIV><DIV class="row"><DIV class="col-md-6 col-sm-12 vcenter"><P class="text-justify">			For the purpose of our work, we automated the different steps of the protocol, 
									including the filtering process, turning it into a single main script file complete 
									with well-documented instructions. This script also enables easy modification 
									of the filtering parameters to suit the specific ligand being used in the design. For more information 
									see our <A href="https://2016.igem.org/Team:Technion_Israel/Software">software tool</A>.</P></DIV></DIV></DIV></DIV><DIV role="tabpanel" class="tab-pane fade" id="444"><DIV class="cont_box"><DIV class="row"><DIV class="col-md-12 col-sm-12"><H1>Redesigning for new ligands</H1></DIV></DIV><DIV class="row"><DIV class="col-md-12 col-sm-12"><P class="text-justify">

									Using the Rosetta software's designs, we attempted to construct several new ligand binding sites for different materials:

- <A href="#Histamine">Histamine</A>
- <A href="#Lactose">Lactose and glucose</A>
- <A href="#Rohypnol">Rohypnol</A>
- <A href="#Ampicillin">Ampicillin</A></P></DIV></DIV><DIV class="row"><DIV class="col-md-12 col-sm-12"><H2>Histamine</H2></DIV></DIV><DIV class="row"><DIV class="col-md-6 col-sm-12 vcenter"><P class="text-justify">
									Histamine is known to be found in decaying food, especially rotten fish. This was our motivation for redesigning the Tar LBD to bind Histamine, as a proof of concept for this computational design technique. This ligand is a 
									derivative of Histidine, which is also an amino acid as Aspartic acid, the native Tar ligand. This increases the chances of a successful result. 
									Figure 1 and video 1 demonstrate the library for Histamine ligand we obtained after running several cycles and filtration.
									</P></DIV><DIV class="col-md-6 col-sm-12 vcenter"><P class="text-center"><B>Fig. 1:</B> Alignment of Tar ligand-binding domain (LBD). The alignment presents 
										the 11 variants in the library with the native Tar (wild type).
									</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 col-sm-12"><P class="text-center"><B>Video 1:</B> 3D imaging of the 11 variants in the library with 
											the native Tar (wild type), each color represents a different variant. As expected, the mutations 
											can be seen near the binding pocket.
										</P></DIV></DIV><DIV class="row"><DIV class="col-md-8 col-sm-12 vcenter"><P class="text-justify">
									Analysis of the results shows two main regions of mutations, one around amino acid 
									number 34 in the LBD sequence and the second around the 115<SUP>th</SUP> amino acid. These 
									results led us to design and perform a two-step cloning assay, 
									in each step we insert the mutations with single PCR reaction.
									</P></DIV><DIV class="col-md-4 col-sm-12 vcenter"><P class="text-center"><B>Fig. 2:</B> General Histamine-Tar fragment design. Blue lines represent the mutated regions.</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 col-sm-12"><H2>Lactose and Glucose</H2></DIV></DIV><DIV class="row"><DIV class="col-md-6 col-sm-12 vcenter"><P class="text-justify">
Awareness among people regarding lactose intolerance is rapidly increasing. Thus, we want to offer a detection solution using the Rosetta's computational design applications. Therefore, we redesigned the Tar LBD to bind Lactose.
									Figure 3 and figure 4 demonstrate the library we obtained for Lactose after running several cycles and filtration.
									</P></DIV><DIV class="col-md-6 col-sm-12 vcenter"><P class="text-center"><B>Fig. 3:</B> Alignment of Tar ligand-binding domain (LBD). The alignment presents 
										the 7 variants in the library with the native Tar (wild type).
									</P></DIV></DIV><DIV class="row"><DIV class="col-md-6 col-sm-12 vcenter"><P class="text-center"><B>Fig. 4:</B> 3D imaging of the 7 variants in the library with the 
									native Tar (wild type), each color represents a different variant. 
									</P></DIV></DIV><DIV class="row"><DIV class="col-md-6 col-sm-12 vcenter"><P class="text-justify">
									As this is a novel design of a Tar ligand-binding domain to bind a sugar molecule, we decided to 
									have a proof of concept with a smaller and suitable molecule to bind, a Lactose component - Glucose. 
									Glucose is well known monosaccharide and is the main compound used in the production of 
									energy in living organisms. For this reason we can find existing chemoreceptors for Glucose 
									<B>(2)</B>, however redesigning the Tar LBD to bind Glucose was performed as a preliminary step before redesigning 
									it to bind Lactose. 
									Figure 5 and video 2 demonstrate the library we obtained for Glucose after running several cycles
									and filtration.
 									</P></DIV><DIV class="col-md-6 col-sm-12 vcenter"><P class="text-center"><B>Fig. 5:</B> Alignment of the ligand binding domain (LBD). The alignment 
									presents the 8 variants in the library with the native Tar (wild type). 
									</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 col-sm-12"><P class="text-center"><B>Video 2:</B> 3D imaging of the 8 variants in the library with the 
										native Tar (wild type), each color represents a different variant.
										</P></DIV></DIV><DIV class="row"><DIV class="col-md-8 col-sm-12 vcenter"><P class="text-justify">
									Analysis of the results shows one main region of mutations, around the 75<SUP>th</SUP> 
									amino acid the LBD sequence. These results led us to design and 
									perform a single PCR reaction to insert the mutations.
																		</P></DIV><DIV class="col-md-4 col-sm-12 vcenter"><P class="text-center"><B>Fig. 6:</B> 
									General Glucose-Tar fragment design. 
									Blue line represent the mutated region.</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 col-sm-12"><H2>Rohypnol</H2></DIV></DIV><DIV class="row"><DIV class="col-md-6 col-sm-12 vcenter"><P class="text-justify">
									Rohypnol, also known as Flunitrazepam, is used in some countries to treat insomnia. However, it is better known 
									as a 'date rape drug'.
									As one application of our project, we would like to offer a simple drug test based on 
									our system to help men and women identify danger when going out. Redesigning the Tar 
									LBD to bind Rohypnol takes us one step closer to achieving this goal.
									Figure 7 and figure 8 demonstrate the library we obtained for Rohypnol after running several cycles 
									and filtration.
									</P></DIV><DIV class="col-md-6 col-sm-12 vcenter"><P class="text-center"><B>Fig. 7:</B> Alignment of the ligand binding domain (LBD). 
									The alignment presents the 4 variants in the library with the native Tar (wild type). 
									</P></DIV></DIV><DIV class="row"><DIV class="col-md-6 col-sm-12 vcenter"><P class="text-center"><B>Fig. 8:</B> 3D imaging of the 4 variants in the library 
									with the native Tar (wild type), each color represents a different variant
									</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 col-sm-12"><H2>Ampicillin</H2></DIV></DIV><DIV class="row"><DIV class="col-md-6 col-sm-12 vcenter"><P class="text-justify">
									Targeting bacteria towards antibiotics may seem redundant as the bacteria will simply die, 
									however it can be also used as an effective kill switch - a small dose of antibiotics can kill 
									more bacteria if those are attracted to it. To expand our novel approach we redesigned 
									the Tar LBD to bind Ampicillin antibiotics.
									Figure 9 and figure 10 demonstrate the library we obtained for Ampicillin after running several cycles and filtration.
									</P></DIV><DIV class="col-md-6 col-sm-12 vcenter"><P class="text-center"><B>Fig. 9:</B> Alignment of the ligand binding domain (LBD). The 
										alignment presents the 13 variants in the library with the native Tar (wild type). 
									</P></DIV></DIV><DIV class="row"><DIV class="col-md-6 col-sm-12 vcenter"><P class="text-center"><B>Fig. 10:</B> 3D imaging of the 13 variants in the library 
									with the native Tar (wild type), each color represents a different variant.
									</P></DIV></DIV></DIV></DIV><DIV role="tabpanel" class="tab-pane fade" id="555"><DIV class="cont_box"><DIV class="row"><DIV class="col-md-12 col-sm-12"><H1>Histamine results</H1></DIV></DIV><DIV class="row"><DIV class="col-md-6 col-sm-12 vcenter"><P class="text-justify">
									The Rosetta’s design process for Histamine ligand produced 870 results, out of which 11 variants remained after filtering. 
									Induction of point mutations was performed on the native Tar LBD in order to build those 11 variants,
									and out of them only 6 exhibited the expected sequences in sequencing and were subjected to <A href="https://static.igem.org/mediawiki/2016/1/11/T--Technion_Israel--Protocol_-_Chemotaxis_on_chip_test.pdf" target="_blank">chemotaxis tests</A>.
									The tests consisted of placing the cloned bacterial solution in a commercial ibidi chip. The chip was then placed under a microscope and the attractant Histamine was added to start the experiment. The interaction of the bacteria with the attractant was observed over time and one frame was taken every 30 seconds for 20 minutes.
									The control consisted of the same conditions, with motility buffer being placed instead of the attractant, Histamine. Out of 6 tested variants only one (variant number 9) was discovered to be attracted to Histamine.
									The results of the chemotaxis test for variant number 9 are presented in figure 2.
									</P></DIV><DIV class="col-md-6 col-sm-12 vcenter"><P class="text-center"><B>Fig. 1:</B> Histamine-Tar filtering process scheme.</P></DIV></DIV><DIV class="row"><DIV class="col-sm-8 col-sm-offset-2"><DIV class="col-md-6 col-sm-12"><P class="text-justify">a.</P></DIV><DIV class="col-md-6 col-sm-12"><P class="text-justify">b.</P></DIV><DIV class="col-md-6 col-sm-12"><P class="text-justify">c.</P></DIV><DIV class="col-md-6 col-sm-12"><P class="text-justify">d.</P></DIV><DIV class="col-md-12 col-sm-12"><P class="text-center"><B>Fig. 2:</B> 
									Microscope results of chemotaxis activity for variant Histamine_9 with 10mM of Histamine.<B> a.</B> Histamine-Tar after 0 minutes (when the Histamine added).<B> b.</B> Histamine-Tar after 20 minutes.<B> c.</B> Control after 0 minutes (when motility buffer added).<B> d.</B> Control after 20 minutes.</P></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-12 col-sm-12"><P class="text-justify">
									To prove the correct localization of the Histamine-Tar receptor on both poles of the bacteria, GFP was fused to its C-terminus with a short linker sequence <A href="http://parts.igem.org/Part:BBa_E0040" target="_blank">(E0040)</A> . The results of these experiments as seen in figure 3, prove our assumption of correct localizations.


									</P></DIV></DIV><DIV class="row"><DIV class="col-sm-8 col-sm-offset-2"><DIV class="col-md-12 col-sm-12"><P class="text-center"><B>Fig. 3:</B> Results of GFP fusion.<B> a.</B> White light image of Histamine-Tar-GFP <B>b.</B> Florescent image (490nm excitation) of Histamine-Tar-GFP <B>c.</B> White light image of native Tar <B>d.</B> Florescent image (490nm excitation) of native Tar
									</P></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-6 col-sm-12 vcenter"><P class="text-justify">
									Finally, showed in video 1 a working prototype of the FlashLab design - 
										a chip that serves as a detection tool based on the chemotaxis system of <I>E. 
										coli</I> bacteria - by using a commercial ibidi chip filled with a suspension of 
										bacteria expressing the novel chemoreceptor and chromoprotein
										<A href="http://parts.igem.org/Part:BBa_K1357008" target="_blank">(K1357008)</A>.
										A solution of Histamine in concentration of 10<SUP>-3</SUP>M (the attractant)
										was added to the chip and the movement of the bacteria was monitored 
										and recorded.	
									</P></DIV><DIV class="col-md-6 col-sm-12 vcenter"><P class="text-center"><B>Video 1:</B> from left to right: 
									(1) Histamine-Tar with Histamine atrractent added.
									(2) Histamine-Tar with motility buffer added (control).</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 col-sm-12"><H1>Glucose results</H1></DIV></DIV><DIV class="row"><DIV class="col-md-12 col-sm-12"><P class="text-justify">

A swarming plate assay was performed in order to confirm the functionality of the 
<A data-toggle="popover" data-trigger="click" data-original-title="Info:" data-html="true" data-content="Tar chemoreceptor with mutations at the LBD.">
												Tar-Glucose</A> receptor (see  <A href="https://static.igem.org/mediawiki/2016/f/f3/T--Technion_Israel--Chemotaxis_assays.pdf" target="_blank">swarming assay protocol</A>).
Two glucose concentrations were tested- 1mM and 10mM. A control was performed with no glucose added.
The results of the swarming assay did not indicate any chemotaxis activity of the Tar-Glucose.
</P></DIV></DIV><DIV class="row"><DIV class="col-sm-8 col-sm-offset-2"><P class="text-center"><B>Fig. 4:</B> Swarming assay results for Glucose variants. 
									Circled in red are the halos (distinguished swarming).<B>Row (a)</B> Negative and positive controls with small concentrations 
									of amino acids. <B>Row (b)</B> Tar with small concentrations 
									of amino acids and different concentrations of Glucose.<B>Row (c)</B> Glucose-Tar with small concentrations 
									of amino acids and different concentrations of Glucose.
									</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 col-sm-12"><H2>Future development</H2></DIV></DIV><DIV class="row"><DIV class="col-md-12 col-sm-12"><P class="text-justify">
									As demonstrated, Rosetta provides us with the means to 
									redesign a chemoreceptor to bind new ligands. In the 
									future, this ability can be used in the same manner to 
									design dozens of new receptors. The critical step of the 
									design process remains the lab work required to clone and
									test the variants, this step can be optimized by using a high 
									throughput chemotaxis assay. Aside from this, any receptor designed 
									can be further improved by introducing a directed evolution step, using the Rosetta software, to 
									improve its specificity for the new ligand.
									</P></DIV></DIV></DIV></DIV><DIV role="tabpanel" class="tab-pane fade" id="666"><DIV class="cont_box"><DIV class="row"><DIV class="col-md-12 col-sm-12"><H1>Rosetta Guide for the iGEM beginner</H1></DIV></DIV><DIV class="row"><DIV class="col-md-12 col-sm-12"><P class="text-justify">
									During our work with Rosetta we encountered quite a few challenges that required us to browse 
									through the official documentation and the Rosetta support forums and also consult with experts 
									in computational design. These problems made us realize how difficult Rosetta can be to completely 
									new users, especially undergraduates lacking the necessary knowledge.
									To make Rosetta more accessible to the iGEM community we decided to team up with iGEM TU Eindhoven 
									and compile a quick start guide complete with important links, protocols and information we gathered 
									from our experience with Rosetta.
									We hope that this guide will help future iGEM teams and novice Rosetta users in general.<A href="https://static.igem.org/mediawiki/2016/5/59/Rosetta_Guide_for_the_iGEM_Beginner.pdf" target="_blank">Click here</A> to see the full guide.
									</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-sm-8 col-sm-offset-2"><P class="references">
References:
1. Moretti, R., Bender, B.J., Allison, B. and Meiler, J., 2016. Rosetta and the Design 
    of Ligand Binding Sites. Computational Design of Ligand Binding Proteins, pp.47-62.
2. Adler, J., Hazelbauer, G.L. and Dahl, M.M., 1973. Chemotaxis toward sugars in Escherichia coli. Journal of bacteriology, 115(3), pp.824-847.</P></DIV></DIV></DIV><DIV class="modal fade" id="imagemodal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel" aria-hidden="true"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><BUTTON type="button" class="close" data-dismiss="modal"><SPAN aria-hidden="true">×</SPAN><SPAN class="sr-only">Close</SPAN></BUTTON></DIV></DIV></DIV></DIV><TITLE>S.tar, by iGEM Technion 2016</TITLE></DIV></DIV></DIV></DIV></DIV></BODY></HTML>