<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Technion_Israel_Design skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Technion Israel/Design</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>S.tar, by iGEM Technion 2016</TITLE></P><DIV class="navbar navbar-inverse navbar-fixed-top" style="margin-top: 10px; padding-top: 10px; padding-bottom: 10px;"><DIV class="container"><DIV class="collapse navbar-collapse"><UL class="nav navbar-nav"><LI><A href="https://2016.igem.org/Team:Technion_Israel/Description">Project</A></LI><LI class="dropdown"><A href="https://2016.igem.org/Team:Technion_Israel/S.Tar_intro" class="dropdown-toggle" data-toggle="dropdown">S.Tar</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/S.Tar_intro">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Tar_improvements">Tar Improvements and Characterization</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Modifications/Rosetta">Computational Design</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Proof">PctA-Tar Chimera</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Modifications/narx">NarX-Tar Chimera</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Modifications/EstoTar">hERα-Tar Chimera</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Modifications/Intein">Intein</A></LI></UL><LI class="dropdown"><A href="https://2016.igem.org/Team:Technion_Israel/Design" class="dropdown-toggle" data-toggle="dropdown">FlashLab</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Design">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Model">Modeling</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Hardware">Hardware</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Color">Chromoprotein</A></LI><LI class="dropdown"><A href="" class="dropdown-toggle" data-toggle="dropdown">Wet Lab</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Results">Highlight Results</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Experiments">Experiments</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Protocols">Protocols</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Notebook">Notebook</A></LI><LI class="dropdown"><A href="" class="dropdown-toggle" data-toggle="dropdown">Judging</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Proof">Proof of concept</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Parts">Biobricks</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Software">Software</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Measurement">Measurments</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Safety">Safety</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Medals">Medals</A></LI><LI class="dropdown"><A href="" class="dropdown-toggle" data-toggle="dropdown">People</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Human_Practices">Human Practice</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Team">Team Members</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Attributions">Attributions</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Collaborations">Collaborations</A></LI></DIV></DIV></DIV><TITLE>S.tar, by iGEM Technion 2016</TITLE><DIV class="model_wrapper"><DIV class="row"><DIV class="col-sm-8 col-sm-offset-2"><UL class="nav nav-tabs" role="tablist"><LI role="presentation" class="col-sm-3 col-xs-6"><A href="#Peshawar" aria-controls="Peshawar" role="tab" data-toggle="tab"><H4 class="text-center">Introduction</H4></A></LI><LI role="presentation" class="col-sm-3 col-xs-6"><A href="#Aachen" aria-controls="Aachen" role="tab" data-toggle="tab"><H4 class="text-center">Design</H4></A></LI><LI role="presentation" class="col-sm-3 col-xs-6"><A href="#Results" aria-controls="Results" role="tab" data-toggle="tab"><H4 class="text-center">Results</H4></A></LI><LI role="presentation" class="col-sm-3 col-xs-6"><A href="#outlook" aria-controls="outlook" role="tab" data-toggle="tab"><H4 class="text-center">Conclusion</H4></A></LI></UL></DIV></DIV><DIV class="tab-content"><DIV role="tabpanel" class="tab-pane fade in active" id="Peshawar"><DIV class="row"><DIV class="col-sm-8 col-sm-offset-2"><DIV class="cont_box"><DIV class="row"><DIV class="col-sm-12"><H2 class="">Introduction</H2><P class="text-justify">
FlashLab is a novel detection tool based on the <A data-toggle="popover" data-trigger="click" data-original-title="Info:" data-html="true" data-content="biological system that controls the bacterial motion and induces movement away from a chemo-repellent or towards a chemo-attractant">chemotaxis system</A> of <I>E. coli</I> 								bacteria. It utilizes chemotaxis to concentrate bacteria expressing a chromo-protein, this in turn, creates a visible gradient in color – detection of a target material. 
FlashLab is an application of the S.Tar project. S.Tar is a platform for programmable chemotaxis that allows the user to select the material that will induce a bacterial chemotactic response. For more information please visit <A href="https://2016.igem.org/Team:Technion_Israel/S.Tar_intro">S.Tar page</A>.
Using S.Tar technology, FlashLab can detect a variety of materials: hormones, amino acids, organic compounds etc.

					</P></DIV><DIV class="col-md-12"><P class="text-center"><B>Fig. 1:</B> A scheme of the FlashLab concept, add bacteria expressing the chemoreceptor of your choice and a chromo protein, to a fluidic chip. Add the sample in question to the chip. If the sample contains the substance that is recognized by the chemoreceptor, a displacement of the bacteria will become visible. If not, then no displacement will be seen.</P></DIV></DIV></DIV></DIV></DIV></DIV><DIV role="tabpanel" class="tab-pane fade" id="Aachen"><DIV class="row"><DIV class="col-sm-8 col-sm-offset-2"><DIV class="cont_box"><DIV class="row"><DIV class="col-sm-12"><H2 class="">Design</H2><H3>FlashLab parts</H3><P class="text-justify">
						The device is composed of a <A href="http://ibidi.com/xtproducts/en/ibidi-Labware/sticky-Slides/sticky-Slide-I-Luer" target="_blank">commercial fluidic chip</A>:
					</P></DIV><DIV class="col-md-12"><P class="text-center"><B>Fig. 1:</B>The geometry of a commercial fluidic chip.</P></DIV><DIV class="col-sm-12"><P class="text-justify">
						The chip is open on the bottom part and closed with a standard microscope cover glass (0.3 [mm] thick).
					</P></DIV><DIV class="col-sm-12"><H3>FlashLab setup</H3><P class="text-justify">
						The setup of the device is composed of two parts, as shown below (Figure 2):      <B>a. </B>The channel is filled with colored <I>E. coli</I>  bacteria suspended in motility buffer.      <B>b. </B>The sample is loaded into one of the entry slots.

					</P></DIV><DIV class="col-md-12"><P class="text-center"><B>Fig. 2: </B>The chip setup.</P></DIV><DIV class="col-sm-12"><H3>FlashLab assay</H3><P class="text-justify">
					Once the sample is loaded, it diffuses into the channel. If the sample contains a repellent, the bacteria 
					will react and flee from it as shown below:
					</P></DIV><DIV class="col-md-12"><P class="text-center"><B>Fig. 3: </B>Chemotaxis reaction in the chip.</P></DIV><DIV class="col-sm-12"><P class="text-justify">
					The chemotactic response will result in visible changes in the bacterial concentration throughout the chip: Very low concentration in the immediate area of the slot in which the sample was loaded, adjacent to a higher concentration of bacteria created due to the fleeing bacterial population. These changes will be visible to the naked eye, as the higher concentration of colored bacteria results in darker color (blue gradient, figures 2 and 3).
If the sample does not contain the target material, the bacteria will not react and no gradient will be formed. 

					For more information see <A href="https://2016.igem.org/Team:Technion_Israel/Model">mathematical model</A>.
					</P></DIV></DIV></DIV></DIV></DIV></DIV><DIV role="tabpanel" class="tab-pane fade" id="Results"><DIV class="row"><DIV class="col-sm-8 col-sm-offset-2"><DIV class="cont_box"><DIV class="row"><DIV class="col-sm-12"><H2 class="">Results</H2><H3>FlashLab parts</H3><P class="text-justify">
					The setup of the device is as shown in the &quot;Design&quot; Tab. The bacteria, UU1250  <I>E.coli</I> strain with a cloned Tar-PctA receptor , was picked from a petri dish and suspended in motility buffer. The Chemo-repellent used was TCE (trichloroethylene) in concentration of 0.02 [M] while the control was motility buffer. The images were taken in a different times. 
	
					</P></DIV></DIV><DIV class="row"><DIV class="col-sm-8 col-sm-offset-2"><DIV class="col-md-6 col-sm-12"><P class="text-justify">a.</P></DIV><DIV class="col-md-6 col-sm-12"><P class="text-justify">b.</P></DIV><DIV class="col-md-12 col-sm-12"><P class="text-center"><B>Fig. 1:</B><B>a.</B> Chemotaxis of <I>E. coli</I> with a S.Tar PctA receptor due to exposure to TCE (enhanced picture).
									<B>b.</B><I>E. coli</I> with a S.Tar PctA receptor exposed to motility buffer (control).
									</P></DIV></DIV></DIV><DIV class="row"><DIV class="col-sm-12"><P class="text-justify">
After 15[min] , a noticeable color gradient is formed in the channel. In 120 [min] a relatively light shade can be seen near the entry point. Adjacent to it, an area with a darker tone and from there up to the end of the channel, the color did not change at all. This fits the theory perfectly, as the lighter shade is caused by colored bacteria moving away from the repellent. The darker shade is the clustering of bacteria in the chemo-repellent diffusion limit. All other bacteria in the channel, were not exposed to the repellent and did not react accordingly.
						

						For more information see chip <A href="https://static.igem.org/mediawiki/2016/1/11/T--Technion_Israel--Protocol_-_Chemotaxis_on_chip_test.pdf" target="_blank">experiment protocol</A>.
					</P></DIV></DIV></DIV></DIV></DIV></DIV><DIV role="tabpanel" class="tab-pane fade" id="outlook"><DIV class="row"><DIV class="col-sm-8 col-sm-offset-2"><DIV class="cont_box"><DIV class="row"><DIV class="col-sm-12"><H2 class="">Conclusion</H2><P class="text-justify">

We developed FlashLab, the hardware of the S.Tar platform. S.Tar has the potential to offer a wide library of “expert” strains for the detection of materials that do not naturally induce chemotactic movement in the native <I>E. coli</I>.  This was obtained using synthetic biology tools, including computational design which allowed us to create new mutations in the Tar chemoreceptor. 

FlashLab is a tool which combines both biological and mechanical aspects. As we mentioned before, computational tools were required as well in order to develop the S.Tar platform. This multidisciplinary innovation was achieved thanks to the different backgrounds of our team members. Our team is comprised of students from the faculty of Biotechnology that oversaw the Tar chemoreceptor modification in the wet lab, students from the faculties of Mechanical engineering and Chemical engineering that developed the mathematical model for the flow and chemotaxis profiles inside the microchannel and students from the faculties of Electrical engineering and Computer Science that were in charge of the computational design aspects of the project. The unique composition of our team was crucial to the construction of a complex but elegant system.
</P><DIV class="row"><DIV class="col-sm-12"><H2 class="">Advantages</H2><P class="text-justify">
There are several methods for detection of small molecules. The following table summarizes the most common methods in the field:
</P></DIV></DIV><TABLE class="table table-list-search"><THEAD><TR><TH>Method</TH><TH>Description</TH><TH>Disadvantages</TH></TR></THEAD><TBODY><TR><TD>Reporter genes <B>(2)</B></TD><TD>Fusion of a reporter gene to a biological biobrick which is regulated by a material of interest. </TD><TD>Requires gene expression and translation.The material of interest must penetrate the bacterial membrane. </TD></TR><TR><TD>Chromogenic assay <B>(3)</B></TD><TD>A chemical reaction which results in the change of absorbance of a certain molecule when it interacts with the analyte.</TD><TD>Specific for very limited range of molecules.Expensive.</TD></TR><TR><TD>ELISA- Immunofluorescence assay <B>(4)</B></TD><TD>A specific interaction between an antibody and its antigen is coupled to an enzymatic fluorescent reaction.</TD><TD>Specific antibodies are required.Expensive.Requires dedicated equipment.</TD></TR><TR><TD>HPLC <B>(5)</B></TD><TD>Chromatographic separation based on polarity.</TD><TD>Requires dedicated equipment and expertise.Expensive</TD></TR></TBODY></TABLE><P class="text-justify">
FlashLab advantages as a detection tool: 
- <B>Cost effective</B>: The only major cost is the fluidic chip which costs about 15$ and can be reused multiple times. 
- <B>Fast</B>: as shown in the experiments, the detection takes about 30 minutes. This is faster than most other bacterial based detection that depends on transcription and translation.
- <B>Versatile</B>: The S.Tar system enables this device to detect a variety of materials: hormones, amino acids, organic compounds etc.
- <B>User friendly</B>: The setup of the system is an easy two part process.
FlashLab, offers a fast, cheap, easy to use and versatile detection.</P></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="modal fade" id="imagemodal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel" aria-hidden="true"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><BUTTON type="button" class="close" data-dismiss="modal"><SPAN aria-hidden="true">×</SPAN><SPAN class="sr-only">Close</SPAN></BUTTON></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-sm-8 col-sm-offset-2"><P class="references">
References:
1. MAZZAG, B. C.; ZHULIN, I. B.; MOGILNER, Alexander. Model of bacterial band formation in aerotaxis. Biophysical journal, 2003, 85.6: 3558-3574.
2. Naylor, Louise H. &quot;Reporter gene technology: the future looks bright.&quot;Biochemical pharmacology 58.5 (1999): 749-757.
3. Santos-Figueroa, Luis E., et al. &quot;Chromogenic and fluorogenic chemosensors and reagents for anions. A comprehensive review of the years 2010–2011.&quot; Chemical Society Reviews 42.8 (2013): 3489-3613.‏
4. Blake, Christopher, and Barry J. Gould. &quot;Use of enzymes in immunoassay techniques. A review.&quot; Analyst 109.5 (1984): 533-547.‏
5. Kucharska, Marta, and Jan Grabka. &quot;A review of chromatographic methods for determination of synthetic food dyes.&quot; Talanta 80.3 (2010): 1045-1051.‏‏
</P></DIV></DIV><TITLE>S.tar, by iGEM Technion 2016</TITLE></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>