<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Technion_Israel_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Technion Israel/Results</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>S.tar, by iGEM Technion 2016</TITLE></P><DIV class="navbar navbar-inverse navbar-fixed-top" style="margin-top: 10px; padding-top: 10px; padding-bottom: 10px;"><DIV class="container"><DIV class="collapse navbar-collapse"><UL class="nav navbar-nav"><LI><A href="https://2016.igem.org/Team:Technion_Israel/Description">Project</A></LI><LI class="dropdown"><A href="https://2016.igem.org/Team:Technion_Israel/S.Tar_intro" class="dropdown-toggle" data-toggle="dropdown">S.Tar</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/S.Tar_intro">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Tar_improvements">Tar Improvements and Characterization</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Modifications/Rosetta">Computational Design</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Proof">PctA-Tar Chimera</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Modifications/narx">NarX-Tar Chimera</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Modifications/EstoTar">hERα-Tar Chimera</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Modifications/Intein">Intein</A></LI></UL><LI class="dropdown"><A href="https://2016.igem.org/Team:Technion_Israel/Design" class="dropdown-toggle" data-toggle="dropdown">FlashLab</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Design">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Model">Modeling</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Hardware">Hardware</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Color">Chromoprotein</A></LI><LI class="dropdown"><A href="" class="dropdown-toggle" data-toggle="dropdown">Wet Lab</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Results">Highlight Results</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Experiments">Experiments</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Protocols">Protocols</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Notebook">Notebook</A></LI><LI class="dropdown"><A href="" class="dropdown-toggle" data-toggle="dropdown">Judging</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Proof">Proof of concept</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Parts">Biobricks</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Software">Software</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Measurement">Measurments</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Safety">Safety</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Medals">Medals</A></LI><LI class="dropdown"><A href="" class="dropdown-toggle" data-toggle="dropdown">People</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Human_Practices">Human Practice</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Team">Team Members</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Attributions">Attributions</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Collaborations">Collaborations</A></LI></DIV></DIV></DIV><TITLE>S.tar, by iGEM Technion 2016</TITLE><DIV class="desk_wrapper"><DIV class="row"><DIV class="col-sm-8 col-sm-offset-2"><DIV class="cont_box"><DIV class="row"><DIV class="row"><DIV class="col-md-12 col-sm-12"><H2>Histamine-Tar novel chemoreceptor</H2></DIV></DIV><DIV class="row"><DIV class="col-md-6 col-sm-12 vcenter"><P class="text-justify">
										We successfully redesigned, cloned and tested a novel <A href="https://2016.igem.org/Team:Technion_Israel/Modifications/Rosetta">Histamine-Tar 
										chemoreceptor</A>. The design was done with the Rosetta bioinformatics tool, 
										by following the protocol published by <I>Moretti, R. et al.</I><B>(1)</B>. The goal was to design a binding pocket for a selected small ligand.  The Rosetta’s design process for the new ligand, Histamine, produced 870 results, out of which 11 variants remained after filtering. The 11 variants were cloned into the native Tar ligand binding domain (LBD), out of these only 6 exhibited the expected sequences in sequencing and thus were subjected to chemotaxis tests. Out of all 6 tested variants only one showed chemotactic response, the ability to be attracted to Histamine. Furthermore, the sequencing results proved that the only mutations that were present in this variant were those planned by the Rosetta’s design.
									</P></DIV><DIV class="col-md-6 col-sm-12 vcenter"><P class="text-center"><B>Fig. 1:</B> Histamine-Tar filtering process scheme.</P></DIV></DIV><DIV class="row"><DIV class="col-md-6 col-sm-12 vcenter"><P class="text-justify">
										We observed the bacteria’s response to the attractant, Histamine, by using a microscope assay. It is evident in figure 3b that roughly 20 minutes after the addition of the Histamine, the bacterial concentration in the vicinity of the Histamine is much greater than in the beginning of the experiment (figure 3a).
									</P></DIV><DIV class="col-md-6 col-sm-12 vcenter"><P class="text-center"><B>Fig. 2:</B> Sequencing Results. &quot;Query&quot; describes the native Tar LBD 
										sequence and &quot;Sbjct&quot; describes the design mutations sequence. The mutation region is marked with color (blue or red).</P></DIV></DIV><DIV class="row"><DIV class="col-sm-8 col-sm-offset-2"><DIV class="row"><DIV class="col-md-6 col-xs-12"><P class="text-justify">a.</P></DIV><DIV class="col-md-6 col-xs-12"><P class="text-justify">b.</P></DIV><DIV class="col-md-6 col-xs-12"><P class="text-justify">c.</P></DIV><DIV class="col-md-6 col-xs-12"><P class="text-justify">d.</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 col-sm-12"><P class="text-center"><B>Fig. 3:</B>  
										Microscope results of chemotaxis activity for variant His_9 with 10mM Histamine.
										<B>a.</B> Tar-Histamine: before adding Histamine. <B>b.</B> Tar-Histamine: 
										20 minutes after adding Histamine. <B>c.</B> Tar-Histamine: before adding the 
										Motility buffer (control solution). <B>d.</B>  Tar-Histamine: 20 minutes after 
										adding the Motility buffer.
										</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-12 col-sm-12"><H4>Histamine-Tar fused with GFP marker:</H4><P class="text-justify">
									To prove the correct localization of the Tar-Histamine, 
									GFP was fused to its C-terminus with a short linker sequence <A href="http://parts.igem.org/Part:BBa_E0040" target="_blank">(E0040)</A>. 
									The results of these tests as seen in figure 4, prove our assumption 
									of correct localizations.
									</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 col-sm-12"><P class="text-center"><B>Fig. 4:</B> 
									Results of GFP fusion. <B>a.</B> Positive control- <I>E. Coli</I> strain expressing GFP protein, 
									<B>b.</B> Negative control- UU1250 strain expressing Tar chemoreceptor, 
									<B>c.</B> UU1250 strain expressing Tar-GFP chemoreceptor, 
									<B>d.</B> UU1250 strain expressing Histamine-Tar-GFP Chimera, 
									fluorescence (490nm excitation).
									</P></DIV></DIV><DIV class="row"><DIV class="col-md-6 col-sm-12 vcenter"><P class="text-justify">
									Finally, in the following video, a working concept of the <A href="https://2016.igem.org/Team:Technion_Israel/Design">FlashLab project</A> - 
										a chip that serves as a detection tool based on the chemotaxis system of <I>E. coli</I> bacteria. In the video, a commercial ibidi microfluidic chip filled with a suspension of bacteria expressing the chemoreceptor and chromoprotein 
										(<A href="http://parts.igem.org/Part:BBa_K1992011" target="_blank">K1992011</A> + <A href="http://parts.igem.org/Part:BBa_K1357008" target="_blank">K1357008</A>) can been observed.
										A solution of attractant (10<SUP>-3</SUP>M Histamine)
										was added to the chip and bacteria displacement was monitored and recorded	
									</P></DIV><DIV class="col-md-6 col-sm-12 vcenter"><P class="text-center"><B>Video 1:</B> from left to right: 
									(1) Histamine-Tar with Histamine attractant added.
									(2) Histamine-Tar with Motility buffer added (control).</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 col-sm-12"><P class="text-justify">
									
									As we demonstrated, Rosetta provides us with the means to 
									redesign a chemoreceptor to bind new ligands. In the 
									future this ability can be used in the same manner to 
									design dozens of new receptors. The critical step of the 
									design process remains the lab work required to clone and
									test the variants, this step can be optimized by using a high 
									throughput chemotaxis assay. Aside from this, any receptor designed 
									can be further improved by introducing a directed evolution step to 
									improve its specificity towards the new ligand.
									
									In addition, we wrote a <A href="https://static.igem.org/mediawiki/2016/5/59/Rosetta_Guide_for_the_iGEM_Beginner.pdf" target="_blank">“How to use”</A> Rosetta guide in cooperation with the Rosetta developers and
									iGEM TU Eindhoven team, and built a <A href="https://2016.igem.org/Team:Technion_Israel/Software">framework software tool</A> to allow fellow researchers the ability to redesign a binding site easily.
									</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 col-sm-12"><H2>FlashLab - microfluidic chip</H2></DIV></DIV><DIV class="row"><DIV class="col-md-12 col-sm-12"><P class="text-justify">
									We successfully demonstrated a working concept of the <A href="https://2016.igem.org/Team:Technion_Israel/Design">FlashLab</A> project - a chip (commercial ibidi chip) that serves as a detection tool based on the chemotaxis system of<I> E. coli</I> bacteria. 
This chip is filled with a suspension of bacteria expressing a chromoprotein, enabling easy visualization of the bacteria, and a chemoreceptor engineered to respond to a specific ligand of your choice. Once a sample containing the ligand is inserted into the chip, the bacteria will respond accordingly leading a chemotactic movement towards or away from the sample. This movement causes the bacteria to form a visible cluster within the chip, due to the chromoprotein, which is seen as a darker shade of color. 
This concept has been successfully proven for both  <A href="https://2016.igem.org/Team:Technion_Israel/Proof">PctA-Tar</A> chimera and <A href="https://2016.igem.org/Team:Technion_Israel/Proof">Histamine-Tar</A> variant of the Rosetta software.
									</P></DIV></DIV><DIV class="row"><DIV class="col-md-6 col-sm-12 vcenter"><P class="text-justify">
									 In video 2, the displacement of the bacteria can be clearly seen in the test chip (left chip, PctA-Tar with repellent), compared to the control chip (right chip, PctA-Tar with buffer).
									</P></DIV><DIV class="col-md-6 col-sm-12 vcenter"><P class="text-center"><B>Video 2:</B> from left to right: 
									(1) PctA-Tar chimera with Tetrachloroethylene repellent added.
									(2) PctA-Tar chimera with Motility buffer added (control).
									</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 col-sm-12"><P class="text-justify">
									In conclusion, we proved the concept of the FlashLab project, by demonstrating the color gradient being formed due to chemotaxis.
									</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 col-sm-12"><H2>PctA-Tar chimera</H2></DIV></DIV><DIV class="row"><DIV class="col-md-12 col-sm-12"><P class="text-justify">
									We have successfully confirmed the functionality of the hybrid 
									PctA-Tar receptor using a swarming assay. From the results seen 
									below, figure 5, and compared to the negative control. it is clear that the chimera functions and controls the chemotactic ability of the bacteria and mediates to a swarming response. This is compared to the control where no swarming can be seen.
									</P></DIV></DIV><DIV class="row"><DIV class="col-md-4 col-sm-12"><P class="text-justify">a.</P></DIV><DIV class="col-md-4 col-sm-12"><P class="text-justify">b.</P></DIV><DIV class="col-md-4 col-sm-12"><P class="text-justify">c.</P></DIV><DIV class="col-md-12 col-sm-12"><P class="text-justify"><B>Fig. 5:</B> Swarming assay for attractant response of the PctA-Tar chimera. 
									<B>a.</B> PctA chimera, <B>b.</B> Negative control- UU1250 strain w/o the Tar expression plasmid, <B>c. </B> positive control - ΔZras strain expressing all chemoreceptors.
									</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 col-sm-12"><H4>PctA-Tar fused with GFP marker:</H4><P class="text-justify">
									In addition, the correct localization of the chimera on both poles of the bacterial membrane, was demonstrated by fusing GFP to the C- terminus of the chimera with a short linker sequence <A href="http://parts.igem.org/Part:BBa_K1992010" target="_blank">(K1992010)</A>. The results of these tests as seen in figure 6, 
									indeed show the expected localizations.
									</P></DIV></DIV><DIV class="row"><DIV class="col-sm-8 col-sm-offset-2"><P><B>Fig. 6:</B> Results of GFP fusion. <B>a.</B> Positive control- <I>E.Coli</I> 
									strain expressing GFP protein,<B>b.</B> Negative control- UU1250 strain expressing 
									Tar chemoreceptor, <B>c.</B>  UU1250 strain expressing Tar-GFP chemoreceptor, 
									<B>d.</B> UU1250 strain expressing PctA-Tar-GFP Chimera, Flourcense (490nm excitation).
									</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 col-sm-12"><H2>Trap and Track - Novel assay for digital chemotaxis detection</H2></DIV></DIV><DIV class="row"><DIV class="col-md-12 col-sm-12"><P class="text-justify">
										Although there is an abundant number of chemotaxis assays available 
										today, most of them were designed 40 to 50 years ago and almost 
										none provide a real time measurement without the use of fluorescence labeling.
										The use of Porous Si (PSi) and oxidized PSi (PSiO2) matrices for biological 
										sensing is on the rise. So far various analytes such as DNA, proteins and 
										bacteria have been proven to be detectable on such matrices. The common 
										method to monitor the interaction of said analytes within the porous films 
										is reflective interferometric Fourier transform spectroscopy (RIFTS), as it 
										allows a real time measurement and output for the user.
										We present the results of an early experiment for the detection of 
										chemotactic activity on the porous silicon films initially developed for bacterial detection.
										This assay provides us with the means to digitally quantify chemotaxis - 
										meaning we have a clear distinction between an attractant response 
										(negative trend), repellent response (positive trend) and no response (zero trend).
										The following graph presents only the chemotactic responses taken from 4 
										different experiments, The top 2 plots, in blue and in grey, represent a 
										repellent response. The yellow plot represents a negative control experiment 
										and the bottom red graph represents an attractant response.
										For more information see our <A href="https://2016.igem.org/Team:Technion_Israel/Measurement">Measurement page</A>.
									</P></DIV></DIV><DIV class="row"><DIV class="col-sm-8 col-sm-offset-2"><P class="text-center"><B>Fig. 7:</B> Digital chemotaxis measurements.</P></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-sm-8 col-sm-offset-2"><P class="referances">
References:
1. Moretti, R., Bender, B.J., Allison, B. and Meiler, J., 2016. Rosetta and the Design of Ligand Binding Sites. Computational Design of Ligand Binding Proteins, pp.47-62</P></DIV></DIV></DIV><DIV class="modal fade" id="imagemodal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel" aria-hidden="true"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><BUTTON type="button" class="close" data-dismiss="modal"><SPAN aria-hidden="true">×</SPAN><SPAN class="sr-only">Close</SPAN></BUTTON></DIV></DIV></DIV></DIV><TITLE>S.tar, by iGEM Technion 2016</TITLE></DIV></DIV></DIV></DIV></DIV></BODY></HTML>