<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Technion_Israel_Proof skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Technion Israel/Proof</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>S.tar, by iGEM Technion 2016</TITLE></P><DIV class="navbar navbar-inverse navbar-fixed-top" style="margin-top: 10px; padding-top: 10px; padding-bottom: 10px;"><DIV class="container"><DIV class="collapse navbar-collapse"><UL class="nav navbar-nav"><LI><A href="https://2016.igem.org/Team:Technion_Israel/Description">Project</A></LI><LI class="dropdown"><A href="https://2016.igem.org/Team:Technion_Israel/S.Tar_intro" class="dropdown-toggle" data-toggle="dropdown">S.Tar</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/S.Tar_intro">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Tar_improvements">Tar Improvements and Characterization</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Modifications/Rosetta">Computational Design</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Proof">PctA-Tar Chimera</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Modifications/narx">NarX-Tar Chimera</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Modifications/EstoTar">hERα-Tar Chimera</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Modifications/Intein">Intein</A></LI></UL><LI class="dropdown"><A href="https://2016.igem.org/Team:Technion_Israel/Design" class="dropdown-toggle" data-toggle="dropdown">FlashLab</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Design">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Model">Modeling</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Hardware">Hardware</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Color">Chromoprotein</A></LI><LI class="dropdown"><A href="" class="dropdown-toggle" data-toggle="dropdown">Wet Lab</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Results">Highlight Results</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Experiments">Experiments</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Protocols">Protocols</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Notebook">Notebook</A></LI><LI class="dropdown"><A href="" class="dropdown-toggle" data-toggle="dropdown">Judging</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Proof">Proof of concept</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Parts">Biobricks</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Software">Software</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Measurement">Measurments</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Safety">Safety</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Medals">Medals</A></LI><LI class="dropdown"><A href="" class="dropdown-toggle" data-toggle="dropdown">People</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Human_Practices">Human Practice</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Team">Team Members</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Attributions">Attributions</A></LI><LI><A href="https://2016.igem.org/Team:Technion_Israel/Collaborations">Collaborations</A></LI></DIV></DIV></DIV><TITLE>S.Tar, by iGEM Technion 2016</TITLE><DIV class="desk_wrapper"><DIV class="row"><DIV class="col-sm-8 col-sm-offset-2"><DIV class="row"><DIV class="col-md-12"><UL class="nav nav-tabs" role="tablist"><LI role="presentation" class="col-sm-6 col-xs-6"><A href="#111" aria-controls="111" role="tab" data-toggle="tab"><H4 class="text-center">PctA-Tar</H4></A></LI><LI role="presentation" class="col-sm-6 col-xs-6"><A href="#222" aria-controls="222" role="tab" data-toggle="tab"><H4 class="text-center">Histamine-Tar</H4></A></LI></UL></DIV></DIV><DIV class="tab-content"><DIV role="tabpanel" class="tab-pane fade in active" id="111"><DIV class="cont_box"><DIV class="row"><DIV class="col-md-12 col-sm-12"><H2>PctA-Tar chimera Introduction</H2></DIV></DIV><DIV class="row"><DIV class="col-md-6 col-sm-12 vcenter"><P class="text-justify">
										One of S.Tars sub-projects focused on altering the LBD of Tar chemoreceptor in order to design new hybrid chimeras. That was accomplished by replacing the LBD of the original Tar chemoreceptor with a new one, from a different source, without modifying Tar signaling region. As a proof of concept for the newly designed Tar chimeras and the S.Tar project, we focused on testing the PctA-Tar hybrid.

									</P></DIV><DIV class="col-md-6 col-sm-12 vcenter"><P class="text-center"><B>Fig. 1:</B> Scheme of native Tar chemoreceptor, native PctA receptor and PctA-Tar chimera. Adapted from <B> (1) </B></P></DIV></DIV><DIV class="row"><DIV class="col-md-12 col-sm-12"><DIV class="col-md-12 col-sm-12"><P class="text-justify">
											PctA is a chemoreceptor found in the <I> Pseudomonas Aeruginosa </I>.It mediates 
											chemotaxis towards amino acids and away from organic compounds. It can sense all 
											amino acids except for Aspartate <B>(1)</B>.
											To construct this chimera, the LBD sequence of the PctA was obtained from the <A href="http://www.pseudomonas.com/" target="_blank"><I> Pseudomonas </I> genome database</A>, 
												while the signaling region of Tar was obtained from the iGEM parts catalog 
											<A href="http://parts.igem.org/Part:BBa_K777000" target="_blank">(K777000)</A>. Using these two sequences, we built a Biobrick part <A href="http://parts.igem.org/Part:BBa_K1992007" target="_blank">(K1992007)</A> 
											which was tested by transforming the device to bacteria that lacks chemoreceptors - <A data-toggle="popover" data-trigger="click" data-original-title="Info:" data-html="true" data-content="An E.coli derivative, which lacks chemoreceptors genes, means this strain does not obtain chemotaxis ability (Parkinson J S, University of Utah).">
													UU1250</A>. It is important 
											to note that this chimera has been constructed before in the literature <B>(1)</B>.
										</P></DIV><DIV class="col-sm-6 col-sm-offset-3"><P><B>Fig. 2:</B> Biobrick device of the PctA-Tar chimera.</P></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-12 col-sm-12"><H2>Test and results</H2></DIV></DIV><DIV class="row"><DIV class="col-md-12 col-sm-12"><P class="text-justify">
										As an initial step, we generated a 3D model of the PctA-Tar chimera, figure 3, using the 
										Phyre2 Protein Fold Recognition server to assure the correct folding of both the LBD and the signaling regions. 
									</P></DIV></DIV><DIV class="row"><DIV class="col-sm-8 col-sm-offset-2"><P><B>Fig. 3: </B> PctA-Tar chimera 3D structure. The Tar signaling regions is in gray, the PctA LBD is in red.</P></DIV></DIV><DIV class="row"><DIV class="col-md-6 col-sm-12 vcenter"><P class="text-justify">
										Following transformation, a swarming plate assay was performed in order to 
										confirm the functionality of the hybrid receptor. A scheme of the assay is 
										presented below (figure 4). It is important to mention that this assay was 
										performed on BA medium as the original assay on TB medium failed. From the 
										results seen in figure 5, it is 
								clear that the chimera functions and controls the chemotactic ability of 
										the bacteria leading to swarming response. 
This is compared to the control, UU1250 strain, that did not show any chemotactic ability as expected and no chemotactic rings were formed. 

									</P></DIV><DIV class="col-md-6 col-sm-12 vcenter"><P class="text-center"><B>Fig. 4:</B>  a scheme for the swarming plate assay. In brief: Using a low percentage agar media plate, 
										add a drop of bacteria to the middle of the plate. Incubate at 30 degrees for several hours. A 
										halo or “chemotactic ring” should be formed in the agar plate.
									</P></DIV></DIV><DIV class="row"><DIV class="col-md-4 col-sm-12"><P class="text-justify">a.</P></DIV><DIV class="col-md-4 col-sm-12"><P class="text-justify">b.</P></DIV><DIV class="col-md-4 col-sm-12"><P class="text-justify">c.</P></DIV><DIV class="col-md-12 col-sm-12"><P class="text-justify"><B>Fig. 5:</B> Swarming assay for attractant response of the PctA-Tar chimera. 
									<B>a.</B> PctA chimera, <B>b.</B> Negative control- UU1250 strain w/o the Tar expression plasmid, <B>c. </B> positive control - ΔZras strain expressing all chemoreceptors.
									</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 col-sm-12 vcenter"><P class="text-justify">
										Next, to prove that the chimera is localized in the membrane, GFP was fused 
										to its C-terminus with a short linker sequence <A href="http://parts.igem.org/Part:BBa_K1992010" target="_blank">(K1992010)</A>, 
										figure 6. The results seen in figure 7, show that indeed the chimera is localized to the membrane (poles). 
									</P></DIV><DIV class="col-md-6 col-md-offset-3 col-sm-12 vcenter"><P><B>Fig. 6:</B> Biobrick device of the PctA-Tar chimera fused to GFP.</P></DIV></DIV><DIV class="row"><DIV class="col-sm-8 col-sm-offset-2"><P><B>Fig. 7:</B> Results of GFP fusion. <B>(A)</B> Positive control- <I>E.Coli</I> strain expressing GFP protein,<B>(B)</B> 
									Negative control- UU1250 strain expressing Tar chemoreceptor, <B>(C)</B>  UU1250 strain expressing Tar-GFP chemoreceptor, <B>(D)</B> 
									UU1250 strain expressing PctA-Tar-GFP Chimera, fluorescence (490nm excitation).
									</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 col-sm-12"><P class="text-justify">
									Finally, in the video below is a working concept of the <A href="https://2016.igem.org/Team:Technion_Israel/Design">FlashLab project</A> - a chip that serves 
									as a detection tool based on the chemotaxis system of <I>E. coli</I> bacteria. In the video, a commercial ibidi  
									microfluidic chip filled with a suspension of bacteria expressing the chimera and chromoprotein 
									(<A href="http://parts.igem.org/Part:BBa_J23100" target="_blank">J23100</A> + <A href="http://parts.igem.org/Part:BBa_K1357009" target="_blank">K1357009</A>) can been seen.  
									A solution of repellent (10<SUP>-3</SUP>M Tetrachloroethylene) was added to the chip 
									and the displacement of the bacteria was monitored and recorded. 
									</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 col-sm-12"><P class="text-center"><B>Fig. 8:</B> A steps scheme of the <A href="https://2016.igem.org/Team:Technion_Israel/Design">FlashLab</A> concept: Add bacteria expressing both the chemoreceptor of your choice and a chromoprotein to a fluidic chip. Add the tested sample to the chip. If the chemoreceptor detecets the substance in the sample, a displacement of the bacteria will become visible. 
.</P></DIV></DIV><DIV class="row"><DIV class="col-md-6 col-sm-12 vcenter"><P class="text-justify">
									 In video 1, the displacement of the bacteria can be clearly seen in test chip compared to the control chip.
									</P></DIV><DIV class="col-md-6 col-sm-12 vcenter"><P class="text-center"><B>Video 1:</B> from left to right: 
									(1) PctA-Tar chimera with Tetrachloroethylene repellent added.
									(2) PctA-Tar chimera with Motility buffer added (control).
									</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 col-sm-12"><P class="text-justify">
									With the supporting evidence of the results presented above, it can be concluded 
									that both concepts: LBD altering and the <A href="https://2016.igem.org/Team:Technion_Israel/Design">FlashLab</A> platform, have been proven and work under real life conditions and seem promising for detection of various substances in the near future.

									</P></DIV></DIV></DIV></DIV><DIV role="tabpanel" class="tab-pane fade" id="222"><DIV class="cont_box"><DIV class="row"><DIV class="col-md-12 col-sm-12"><H2>Histamine-Tar Introduction</H2></DIV></DIV><DIV class="row"><DIV class="col-md-6 col-sm-12 vcenter"><P class="text-justify">
										The heart of the S.Tar project is the Tar chemoreceptor, one of four <I>E. coli</I> receptors.
										Our goal is to create an engineered bacteria which has chemotaxis receptors sensitive
										to materials other than its native ligands. We show that <I>E. 
										coli</I> can be engineered to respond to completely new materials. By changing Tar’s ligand binding domain 
										(LBD) to other LBDs from various sources or by mutating it.
									</P></DIV><DIV class="col-md-6 col-sm-12 vcenter"><P class="text-center"><B>Fig. 1:</B> Sequencing Results. &quot;Query&quot; describes the native Tar LBD 
										sequence and &quot;Sbjct&quot; describes the design mutations sequence. The mutation region is marked with color (blue or red).</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 col-sm-12"><P class="text-justify">
									The bacterial world offers a relatively small selection of chemoreceptors in comparison to
									the vast number of possible ligands. These receptors evolved specifically to recognize substances 
									which benefit or harm the organism. On top of that the fact that the majority of known 
									receptors today are not well characterized meant that we had very few options of creating chimeric 
									receptors as we initially planned.
									In light of the above we had to turn to a new path – redesigning the Tar chemoreceptor to bind a 
									different ligand using computational biology - The Rosetta <A href="https://2016.igem.org/Team:Technion_Israel/Software">software</A>.
									Out of the Rosetta’s 870 suggested mutations only 11 variants were eventually cloned into the native Tar ligand-binding domain (LBD). 
									See <A href="https://2016.igem.org/Team:Technion_Israel/Modifications/Rosetta">Computational Design</A> page for more information regarding the design process.
									Out of all the tested variants, only one was discovered to be attracted to Histamine. Sequencing results 
									showed that the only mutations to occur in this variant were those planned by the Rosetta’s design. The desired sequences can be seen in figure 1. 
									</P></DIV></DIV><DIV class="row"><DIV class="col-sm-6 col-sm-offset-3"><P><B>Fig. 2:</B> Histamine-Tar filtering process scheme</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 col-sm-12"><H2>Test and results</H2></DIV><DIV class="row"><DIV class="col-md-12 col-sm-12"><P class="text-justify">
										We observed the bacteria’s response to the attractant, 
										Histamine, by using a microscope. It is evident in figure 3b that roughly 20 minutes after the addition of the 
										Histamine, the bacteria concentration in the vicinity of the Histamine is much greater 
										than in the the beginning of the experiment (figure 3a). 

									</P></DIV><DIV class="row"><DIV class="col-sm-8 col-sm-offset-2"><DIV class="row"><DIV class="col-md-6 col-xs-12"><P class="text-justify">a.</P></DIV><DIV class="col-md-6 col-xs-12"><P class="text-justify">b.</P></DIV><DIV class="col-md-6 col-xs-12"><P class="text-justify">c.</P></DIV><DIV class="col-md-6 col-xs-12"><P class="text-justify">d.</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 col-sm-12"><P class="text-center"><B>Fig. 3:</B>  Microscope results of chemotaxis activity for variant His_9 with 10mM Histamine. (a) Tar-Histamine: before adding Histamine. (b) Tar-Histamine: 20 minutes after adding Histamine. (c) Tar-Histamine: before adding the Motility buffer (control solution). (d)  Tar-Histamine: 20 minutes after adding the Motility buffer. 
										</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-12 col-sm-12"><P class="text-justify">
									To prove the correct localization of the Tar-Histamine, GFP was fused to its C-terminus with a short linker sequence <A href="http://parts.igem.org/Part:BBa_E0040" target="_blank">(E0040)</A> . The results seen in figure 4 show that indeed the chimera is localized to the membrane (poles).


									</P></DIV></DIV><DIV class="row"><DIV class="col-sm-8 col-sm-offset-2"><P class="text-center"><B>Fig. 4:</B>  Results of GFP fusion. (A) Positive control- E.Coli strain expressing GFP protein, (B) Negative control- UU1250 strain expressing Tar chemoreceptor, (C) UU1250 strain expressing Tar-GFP chemoreceptor, (D) UU1250 strain expressing Histamine-Tar-GFP Chimera, fluorescence (490nm excitation).


									</P></DIV></DIV><DIV class="row"><DIV class="col-md-6 col-sm-12 vcenter"><P class="text-justify">
									Finally, in the video below, a working concept of the <A href="https://2016.igem.org/Team:Technion_Israel/Design">FlashLab project</A> - 
										a chip that serves as a detection tool based on the chemotaxis system of <I>E. 
										coli</I> bacteria is presented. In the video, a commercial ibidi microfluidic chip filled with a suspension of 
										bacteria expressing both the chemoreceptor and chromoprotein
										(<A href="http://parts.igem.org/Part:BBa_K1992011" target="_blank">K1992011</A> + <A href="http://parts.igem.org/Part:BBa_K1357008" target="_blank">K1357008</A>) can been seen.
										A solution of attractant (10<SUP>-3</SUP>M Histamine)
										was added to the chip and bacteria displacement was monitored and recorded	
									</P></DIV><DIV class="col-md-6 col-sm-12 vcenter"><P class="text-center"><B>Video 1:</B> from left to right: 
									(1) Histamine-Tar with Histamine attractant added.
									(2) Histamine-Tar with Motility buffer added (control).</P></DIV></DIV><DIV class="row"><DIV class="col-md-12 col-sm-12"><P class="text-justify">
								The results presented above, mainly proves the concept of the ability to alter the LBD of a chemoreceptor using software. 
								Moreover, these results might help with breakthroughs and lead to newly designed reporters for novel detection.
									</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-sm-8 col-sm-offset-2"><P class="referances">
References:
1. Reyes-Darias, J.A., Yang, Y., Sourjik, V., and Krell, T. (2015). Correlation between signal input and output in PctA and PctB amino acid chemoreceptor of Pseudomonas aeruginosa. Mol. Microbiol. 96, 513–525.
2. Moretti, R., Bender, B.J., Allison, B. and Meiler, J., 2016. Rosetta and the Design 
    of Ligand Binding Sites. Computational Design of Ligand Binding Proteins, pp.47-62.</P></DIV></DIV></DIV><DIV class="modal fade" id="imagemodal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel" aria-hidden="true"><DIV class="modal-dialog"><DIV class="modal-content"><DIV class="modal-body"><BUTTON type="button" class="close" data-dismiss="modal"><SPAN aria-hidden="true">×</SPAN><SPAN class="sr-only">Close</SPAN></BUTTON></DIV></DIV></DIV></DIV><TITLE>S.tar, by iGEM Technion 2016</TITLE></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>