Team:Technion Israel/Modifications/Rosetta
S.tar, by iGEM Technion 2016
Project
S.Tar
Overview
Tar Improvements and Characterization
Computational Design
PctA-Tar Chimera
NarX-Tar Chimera
hERα-Tar Chimera
Intein
FlashLab
Overview
Modeling
Hardware
Chromoprotein
Wet Lab
Highlight Results
Experiments
Protocols
Notebook
Judging
Proof of concept
Biobricks
Software
Measurments
Safety
Medals
People
Human Practice
Team Members
Attributions
Collaborations
S.tar, by iGEM Technion 2016
Intro
Design
Auto-
mation
New
ligands
Results
Guide
Computational Design of Ligand Binding Sites
The bacterial world offers a relatively small selection of chemoreceptors, in comparison           to the vast number of possible ligands. These receptors evolved specifically to recognize           substances, which benefit or harm the organism in some way. On top of that, the fact that           the majority of known receptors today are not well characterized, meant that we had a very few           options of designing chimeric receptors like we initially planned.
In light of the above we had to turn to a new path – redesigning the Tar chemoreceptor           to bind a different ligand using computational biology with the Rosetta software.
Rosetta
Rosetta is a bioinformatics software for macromolecular modeling and design, created           by the Rosetta Commons organization - a collaboration between several universities and           research groups from around the world.
Rosetta development began in the laboratory of Dr. David Baker at the University of           Washington as a structure prediction tool but since then has been expanded to solve           many different computational macromolecular problems.
As of 2016, Rosetta algorithms have been used to predict, design and analyze almost           every set of biomolecular systems: proteins, RNA, DNA, peptides, small molecules and           non-canonical amino acids.
Local installation of Rosetta
For complex problems such as redesigning a protein,            Rosetta requires more computational power than a regular PC            has to offer. While searching for possible computing resources,            we came across the local Technion grid of WLCG.
Fig. 1: WLCG computing grid.
The Worldwide LHC Computing Grid (WLCG) is a           global collaboration of more than 170 computing centers in 42 countries, linking up national           and international grid infrastructures. It was launched in 2002 to provide a resource to           store, distribute and analyze the 15 petabytes (15 million gigabytes) of data generated every           year by the world’s largest and most powerful particle accelerator - The Large Hadron Collider (LHC).
In Israel, there are three computing centers connected to the grid, located in the Technion Institute,           Tel Aviv University and Weizmann Institute.
WLCG supports not only the particle accelerator, but also allows unregister users to benefit from           this amazing project. We contacted the local Technion grid administrator and received a temporary user on the Atlas server (one of four particle accelerator components). This granted us access to vast computational power, much more than was necessary. With the help of David Cohen, grid computing specialist from the Technion physics department, we successfully installed Rosetta and all required programs.
Fig. 2: WLCG distribution. Yellow dots represent different computing centers.
Designing a binding site
To redesign the Tar chemoreceptor we followed the protocol presented in "Rosetta and the           Design of Ligand Binding Sites", (1).  The purpose of the protocol is to design a binding           site around a selected small molecule ligand. The general steps of the protocol can be           seen in the flowchart to the right.
Using this protocol we managed to generate a library of mutated Tar receptors that theoretically           bind a substance in a novel way and activate the chemotaxis pathway in response to it. For each           design we ran 3-5 iterations of the protocol to assure optimal results.
Fig. 1: Flowchart of the ligand binding domain design protocol.
Filtering Process
The output of the protocol is a library of variants, ranging from dozens to even thousands           of protein PDB files, depending on the parameters of the design run. This fact means that           filtering the results is an extremely crucial part of the process.
Rosetta is able to predict which protein designs are likely to have improved protein           activity. This is done by measuring every aspect of the protein complex such as binding           energies, interactions between amino acids, backbone angles, hydrogen bonds and more. After           the calculation process the user can decide which parameters are relevant and to discard           the lowest results obtained for the mentioned parameters. The specific filters we used in our designs           can be seen in this attachment .
First run - Benchmark Test
As a test phase, before advancing to more complex designs,           we ran the protocol with Aspartic acid which is Tar receptor's native ligand. This was done in order to make sure that Rosetta can "handle"           the Tar protein, meaning it does not create unnecessary or drastic changes in the protein.
From this design process we received four output structures (after filtering) with 3-5           mutations each, all of which in the binding pocket. These results proved that Rosetta           can recognize and work with the Tar ligand-binding domain (LBD).
Fig. 2: Design results after filtration for native Tar ligand- Aspartic acid.
Protocol automation
For the purpose of our work, we automated the different steps of the protocol,           including the filtering process, turning it into a single main script file complete           with well-documented instructions. This script also enables easy modification           of the filtering parameters to suit the specific ligand being used in the design. For more information           see our software tool .
Redesigning for new ligands
Using the Rosetta software's designs, we attempted to construct several new ligand binding sites for different materials:
- Histamine
- Lactose and glucose
- Rohypnol
- Ampicillin
Histamine
Histamine is known to be found in decaying food, especially rotten fish. This was our motivation for redesigning the Tar LBD to bind Histamine, as a proof of concept for this computational design technique. This ligand is a           derivative of Histidine, which is also an amino acid as Aspartic acid, the native Tar ligand. This increases the chances of a successful result.
Figure 1 and video 1 demonstrate the library for Histamine ligand we obtained after running several cycles and filtration.
Fig. 1: Alignment of Tar ligand-binding domain (LBD). The alignment presents            the 11 variants in the library with the native Tar (wild type).
Video 1: 3D imaging of the 11 variants in the library with             the native Tar (wild type), each color represents a different variant. As expected, the mutations             can be seen near the binding pocket.
Analysis of the results shows two main regions of mutations, one around amino acid           number 34 in the LBD sequence and the second around the 115th amino acid. These           results led us to design and perform a two-step cloning assay,           in each step we insert the mutations with single PCR reaction.
Fig. 2: General Histamine-Tar fragment design. Blue lines represent the mutated regions.
Lactose and Glucose
Awareness among people regarding lactose intolerance is rapidly increasing. Thus, we want to offer a detection solution using the Rosetta's computational design applications. Therefore, we redesigned the Tar LBD to bind Lactose.
Figure 3 and figure 4 demonstrate the library we obtained for Lactose after running several cycles and filtration.
Fig. 3: Alignment of Tar ligand-binding domain (LBD). The alignment presents            the 7 variants in the library with the native Tar (wild type).
Fig. 4: 3D imaging of the 7 variants in the library with the           native Tar (wild type), each color represents a different variant.
As this is a novel design of a Tar ligand-binding domain to bind a sugar molecule, we decided to           have a proof of concept with a smaller and suitable molecule to bind, a Lactose component - Glucose.           Glucose is well known monosaccharide and is the main compound used in the production of           energy in living organisms. For this reason we can find existing chemoreceptors for Glucose (2), however redesigning the Tar LBD to bind Glucose was performed as a preliminary step before redesigning           it to bind Lactose.
Figure 5 and video 2 demonstrate the library we obtained for Glucose after running several cycles          and filtration.
Fig. 5: Alignment of the ligand binding domain (LBD). The alignment           presents the 8 variants in the library with the native Tar (wild type).
Video 2: 3D imaging of the 8 variants in the library with the            native Tar (wild type), each color represents a different variant.
Analysis of the results shows one main region of mutations, around the 75th amino acid the LBD sequence. These results led us to design and           perform a single PCR reaction to insert the mutations.
Fig. 6: General Glucose-Tar fragment design.           Blue line represent the mutated region.
Rohypnol
Rohypnol, also known as Flunitrazepam, is used in some countries to treat insomnia. However, it is better known           as a 'date rape drug'.
As one application of our project, we would like to offer a simple drug test based on           our system to help men and women identify danger when going out. Redesigning the Tar           LBD to bind Rohypnol takes us one step closer to achieving this goal.
Figure 7 and figure 8 demonstrate the library we obtained for Rohypnol after running several cycles           and filtration.
Fig. 7: Alignment of the ligand binding domain (LBD).           The alignment presents the 4 variants in the library with the native Tar (wild type).
Fig. 8: 3D imaging of the 4 variants in the library           with the native Tar (wild type), each color represents a different variant
Ampicillin
Targeting bacteria towards antibiotics may seem redundant as the bacteria will simply die,           however it can be also used as an effective kill switch - a small dose of antibiotics can kill           more bacteria if those are attracted to it. To expand our novel approach we redesigned           the Tar LBD to bind Ampicillin antibiotics.
Figure 9 and figure 10 demonstrate the library we obtained for Ampicillin after running several cycles and filtration.
Fig. 9: Alignment of the ligand binding domain (LBD). The            alignment presents the 13 variants in the library with the native Tar (wild type).
Fig. 10: 3D imaging of the 13 variants in the library           with the native Tar (wild type), each color represents a different variant.
Histamine results
The Rosetta’s design process for Histamine ligand produced 870 results, out of which 11 variants remained after filtering.           Induction of point mutations was performed on the native Tar LBD in order to build those 11 variants,          and out of them only 6 exhibited the expected sequences in sequencing and were subjected to chemotaxis tests .          The tests consisted of placing the cloned bacterial solution in a commercial ibidi chip. The chip was then placed under a microscope and the attractant Histamine was added to start the experiment. The interaction of the bacteria with the attractant was observed over time and one frame was taken every 30 seconds for 20 minutes.          The control consisted of the same conditions, with motility buffer being placed instead of the attractant, Histamine. Out of 6 tested variants only one (variant number 9) was discovered to be attracted to Histamine.          The results of the chemotaxis test for variant number 9 are presented in figure 2.
Fig. 1: Histamine-Tar filtering process scheme.
a.
b.
c.
d.
Fig. 2: Microscope results of chemotaxis activity for variant Histamine_9 with 10mM of Histamine. a. Histamine-Tar after 0 minutes (when the Histamine added). b. Histamine-Tar after 20 minutes. c. Control after 0 minutes (when motility buffer added). d. Control after 20 minutes.
To prove the correct localization of the Histamine-Tar receptor on both poles of the bacteria, GFP was fused to its C-terminus with a short linker sequence (E0040) . The results of these experiments as seen in figure 3, prove our assumption of correct localizations.
Fig. 3: Results of GFP fusion. a. White light image of Histamine-Tar-GFP b. Florescent image (490nm excitation) of Histamine-Tar-GFP c. White light image of native Tar d. Florescent image (490nm excitation) of native Tar
Finally, showed in video 1 a working prototype of the FlashLab design -            a chip that serves as a detection tool based on the chemotaxis system of E.            coli bacteria - by using a commercial ibidi chip filled with a suspension of            bacteria expressing the novel chemoreceptor and chromoprotein (K1357008) .           A solution of Histamine in concentration of 10-3M (the attractant)           was added to the chip and the movement of the bacteria was monitored            and recorded.
Video 1: from left to right:           (1) Histamine-Tar with Histamine atrractent added.          (2) Histamine-Tar with motility buffer added (control).
Glucose results
A swarming plate assay was performed in order to confirm the functionality of the Tar-Glucose receptor (see swarming assay protocol ). Two glucose concentrations were tested- 1mM and 10mM. A control was performed with no glucose added. The results of the swarming assay did not indicate any chemotaxis activity of the Tar-Glucose.
Fig. 4: Swarming assay results for Glucose variants.           Circled in red are the halos (distinguished swarming).
Row (a) Negative and positive controls with small concentrations           of amino acids. Row (b) Tar with small concentrations           of amino acids and different concentrations of Glucose.Row (c) Glucose-Tar with small concentrations           of amino acids and different concentrations of Glucose.
Future development
As demonstrated, Rosetta provides us with the means to           redesign a chemoreceptor to bind new ligands. In the           future, this ability can be used in the same manner to           design dozens of new receptors. The critical step of the           design process remains the lab work required to clone and          test the variants, this step can be optimized by using a high           throughput chemotaxis assay. Aside from this, any receptor designed           can be further improved by introducing a directed evolution step, using the Rosetta software, to           improve its specificity for the new ligand.
Rosetta Guide for the iGEM beginner
During our work with Rosetta we encountered quite a few challenges that required us to browse           through the official documentation and the Rosetta support forums and also consult with experts           in computational design. These problems made us realize how difficult Rosetta can be to completely           new users, especially undergraduates lacking the necessary knowledge.
To make Rosetta more accessible to the iGEM community we decided to team up with iGEM TU Eindhoven           and compile a quick start guide complete with important links, protocols and information we gathered           from our experience with Rosetta.
We hope that this guide will help future iGEM teams and novice Rosetta users in general.
Click here to see the full guide.
References:
1. Moretti, R., Bender, B.J., Allison, B. and Meiler, J., 2016. Rosetta and the Design      of Ligand Binding Sites. Computational Design of Ligand Binding Proteins, pp.47-62.
2. Adler, J., Hazelbauer, G.L. and Dahl, M.M., 1973. Chemotaxis toward sugars in Escherichia coli. Journal of bacteriology, 115(3), pp.824-847.
×Close
S.tar, by iGEM Technion 2016
