Team:CGU Taiwan/Parts
iGem CGU_Taiwan 2017 - Parts
PROJECT
DESCRIPTION RESULTS INTERLAB PROOF AND DEMONSTRATION MODEL PARTS FUTURE WORK
|
NOTEBOOK
LAB NOTE PROTOCOLS SAFETY
|
TEAM
ATTRIBUTION MEMBER ACKNOWLEDGEMENT COLLABORATION
|
HUMAN PRACTICE
SILVER HP INTERGRATED AND GOLD PUBLIC ENGAGEMENT 2017 iGEM TAIWAN CONFERENCE
|
AWARD
ENTREPRENEURSHIP HARDWARE MEASUREMENT MODEL INTERLAB INTERGRATED AND GOLD PUBLIC ENGAGEMENT
   
Parts
We had designed eight new biobricks in our project, three of them are composite parts and the rest are single biobricks. Our parts were designed to be use as a light inducible deinking enzyme secreting system, the codons were optimized and designed for expression in Saccharomyces cerevisiae.
BBa_K2376000
endo-1,4-beta-xylanase A
endo-1,4-beta-xylanase A ORF sequence took from Neocallimastix patriciarum (http://www.uniprot.org/uniprot/P29127) , a rumen fungi normally found in goat rumen. This enzyme digests xylan (hemicellulose) into oligosaccharides (xylobiose and xylose) in the working condition of 30-35-degree Celcius and pH4.5-5.5. The sequence had been optimized to a Saccharomyces cerevisiae codon and had replaced the original signal peptide (1-18 amino acids) into Saccharomyces cerevisiae SUC2 secreting signal. This enzyme was used for digesting paper fibers in our project, it could be used in paper pulping, deinking, cleaning processes or other fiber related digestion.
BBa_K2376001
endo-beta-1,4-glucanase B
endo-beta-1,4-glucanase B ORF sequence took from Aspergillus oryzae (http://www.uniprot.org/uniprot/Q2UPQ4) . This enzyme digests cellulose into oligosaccharides in the working condition of 30-35-degree Celcius and pH4.5-5.5. The sequence had been optimized to a Saccharomyces cerevisiae codon and had replaced the original signal peptide (1-17 amino acids) into Saccharomyces cerevisiae SUC2 secreting signal. This enzyme was used for digesting paper fibers in our project, it could be used in paper pulping, deinking, cleaning processes or other fiber related digestion.
BBa_K2376002
Lipase
Lipase ORF sequence took from Rhizopus niveus (http://www.uniprot.org/uniprot/P61871) . This enzyme hydrolyzes ester bonds of triglycerides in the working condition of 30-35-degree Celcius and pH4.5-5.5. The sequence had been optimized to a Saccharomyces cerevisiae codon and had replaced the original signal peptide (1-26 amino acids) into Saccharomyces cerevisiae SUC2 secreting signal. This enzyme was used for digesting ink in our project (such as soil bean ink), it could deinking process or lipid digesting.
BBa_K2376003
ste12 promoter
Upstream 500bp of a pheromone related pathway transcription factor of Saccharomyces cerevisiae (http://onlinelibrary.wiley.com/doi/10.1111/j.1742-4658.2010.07728.x/full) . This promoter contains four PREs (pheromone-response element) that allows STE12 protein dimer binds on it. This is a very complex pathway induced by alpha or a factors, note that several inhibiting and antagonist pathway involves in the ste12 expression. However, the positive up-regulation feedback of the ste12 itself gives the potential of acting as an amplifier in various response pathway. We had designed this promoter for improving the expression of protein level after light inducing promoter systems. (https://www.ncbi.nlm.nih.gov/pubmed/2193847)
BBa_K2376004
ste12 protein
STE12 is a pheromone related pathway transcription factor of Saccharomyces cerevisiae (http://onlinelibrary.wiley.com/doi/10.1111/j.1742-4658.2010.07728.x/full) , it acts as the final controller of several pheromone induced protein expression. This protein would form a dimer and can bind to PREs (pheromone-response element) sequence. This is a very complex pathway induced by alpha or a factors in haploid budding yeast, note that several inhibiting and antagonist pathway involves in the ste12 expression. However, the positive up-regulation feedback of the ste12 itself gives the potential of acting as an amplifier in various response pathway. We had designed this promoter for improving the expression of protein level after light inducing promoter systems. (https://www.ncbi.nlm.nih.gov/pubmed/2193847)
BBa_K2376005
Glucanase and Xylanase with ste12 promoter
This is a composite part combining endo-1,4-beta-xylanase A and Probable endo-beta-1,4-glucanase B from Neocallimastix patriciarum and Aspergillus oryzae, respectively. The protein codon had been optimized to Saccharomyces cerevisiae codon and had added a SUC2 secrete signal. Xylanase (BBa_K2376000) and glucanase (BBa_K2376001) were both promote by a ste12 promoter (BBa_K2376003) and followed by a TEF terminator (BBa_K801011). Kozak sequence was added in front of the start codon of both proteins. This composite part is used to induce the expression of deinking enzymes, it can be either activated by the induction of pheromone or any pathway that overexpress ste12 protein (BBa_K2376004). The ste12 transcription factor could bind to the ste12 promoter which has PREs, there is also a ste12 sequence in the yeast genome, which could cause a positive feedback of ste12 for binding and promote itself. The sequence of expression two proteins was reversed and formed a head-to-head sequence, wishing to lower the stereo obstacle the entry of RNA polymerase II while transcription. This composite biobrick remains central to our project. The two enzymes are the main enzymes that deink the recycled paper.
BBa_K2376006
In this biobrick, STE12 protein, a transcription factor, can be driven by the system we made or the pheromone response system originally in yeast. Once the STE12 protein has been made, it initiates the positive feedback in yeast genome. The mass production of STE12 protein helps increase STE12 promoter to lift the expression of downstream gene, Lipase, Xylanase and Glucanase. Besides, Lipase is also constructed in this biobrick.
BBa_K2376007
improved GAL4 based yeast light-switchable promoter system
This biobricks was made from BBa_K801042. Which was originally designed for a red light inducible protein expression system. In order to improve the funtionality and the efficiency, we tried to reverse half of the sequence, forming a head-to-head stricture, this mAy lower the stereo effect of RNA polymerase II.
