<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_CGU_Taiwan_Parts skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:CGU Taiwan/Parts</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>iGem CGU_Taiwan 2017 - Parts</TITLE></P><NAV><UL><LI class="dropdown"><A href="https://2017.igem.org/Team:CGU_Taiwan/Description">PROJECT</A><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:CGU_Taiwan/Description">DESCRIPTION</A><A href="https://2017.igem.org/Team:CGU_Taiwan/Results">RESULTS</A><A href="https://2017.igem.org/Team:CGU_Taiwan/InterLab">INTERLAB</A><A href="https://2017.igem.org/Team:CGU_Taiwan/Demonstrate">PROOF AND DEMONSTRATION</A><A href="https://2017.igem.org/Team:CGU_Taiwan/Model">MODEL</A><A href="https://2017.igem.org/Team:CGU_Taiwan/Parts">PARTS</A><A href="https://2017.igem.org/Team:CGU_Taiwan/Future">FUTURE WORK</A></DIV></LI>
  |
  <LI class="dropdown"><A href="https://2017.igem.org/Team:CGU_Taiwan/Notebook">NOTEBOOK</A><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:CGU_Taiwan/Notebook">LAB NOTE</A><A href="https://2017.igem.org/Team:CGU_Taiwan/Protocols">PROTOCOLS</A><A href="https://2017.igem.org/Team:CGU_Taiwan/Safety">SAFETY</A></DIV></LI>
  |
  <LI class="dropdown"><A href="https://2017.igem.org/Team:CGU_Taiwan/Team">TEAM</A><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:CGU_Taiwan/Attributions">ATTRIBUTION</A><A href="https://2017.igem.org/Team:CGU_Taiwan/Team">MEMBER</A><A href="https://2017.igem.org/Team:CGU_Taiwan/Acknowledgement">ACKNOWLEDGEMENT</A><A href="https://2017.igem.org/Team:CGU_Taiwan/Collaborations">COLLABORATION</A></DIV></LI>
  |
  <LI class="dropdown"><A href="">HUMAN PRACTICE</A><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:CGU_Taiwan/HP/Silver">SILVER HP</A><A href="https://2017.igem.org/Team:CGU_Taiwan/HP/Gold_Integrated">INTERGRATED AND GOLD</A><A href="https://2017.igem.org/Team:CGU_Taiwan/Engagement">PUBLIC ENGAGEMENT</A><A href="https://2017.igem.org/Team:CGU_Taiwan/TWConference">2017 iGEM TAIWAN CONFERENCE</A></DIV></LI>
  |
<LI class="dropdown2"><A href="">AWARD</A><DIV class="dropdown2-content"><A href="https://2017.igem.org/Team:CGU_Taiwan/Entrepreneurship">ENTREPRENEURSHIP</A><A href="https://2017.igem.org/Team:CGU_Taiwan/Hardware">HARDWARE</A><A href="https://2017.igem.org/Team:CGU_Taiwan/Measurement">MEASUREMENT</A><A href="https://2017.igem.org/Team:CGU_Taiwan/Model">MODEL</A><A href="https://2017.igem.org/Team:CGU_Taiwan/InterLab">INTERLAB</A><A href="https://2017.igem.org/Team:CGU_Taiwan/HP/Gold_Integrated">INTERGRATED AND GOLD</A><A href="https://2017.igem.org/Team:CGU_Taiwan/Engagement">PUBLIC ENGAGEMENT</A></DIV></LI><LI>   </LI></UL></NAV><DIV class="description" style="text-align:center"><H1>Parts</H1><P style="font-size:150%"> We had designed eight new biobricks in our project, three of them are composite
parts and the rest are single biobricks. Our parts were designed to be use as a light
inducible deinking enzyme secreting system, the codons were optimized and
designed for expression in Saccharomyces cerevisiae.</P><H2>BBa_K2376000</H2><H3>endo-1,4-beta-xylanase A</H3><P style="font-size:150%"> endo-1,4-beta-xylanase A ORF sequence took from Neocallimastix patriciarum
<A href="http://www.uniprot.org/uniprot/P29127">(http://www.uniprot.org/uniprot/P29127)</A>
, a rumen fungi normally found in goat rumen. This enzyme digests xylan (hemicellulose) into oligosaccharides (xylobiose and xylose) in the working condition of 30-35-degree Celcius and pH4.5-5.5. The sequence had been optimized to a Saccharomyces cerevisiae codon and had replaced the original signal peptide (1-18 amino acids) into Saccharomyces cerevisiae SUC2 secreting signal. This enzyme was used for digesting paper fibers in our project, it could be used in paper pulping, deinking, cleaning processes or other fiber related digestion.</P><H2>BBa_K2376001
</H2><H3>endo-beta-1,4-glucanase B
</H3><P style="font-size:150%"> endo-beta-1,4-glucanase B ORF sequence took from Aspergillus oryzae 
<A href="http://www.uniprot.org/uniprot/Q2UPQ4">(http://www.uniprot.org/uniprot/Q2UPQ4)</A>
. This enzyme digests cellulose into oligosaccharides in the working condition of 30-35-degree Celcius and pH4.5-5.5. The sequence had been optimized to a Saccharomyces cerevisiae codon and had replaced the original signal peptide (1-17 amino acids) into Saccharomyces cerevisiae SUC2 secreting signal. This enzyme was used for digesting paper fibers in our project, it could be used in paper pulping, deinking, cleaning processes or other fiber related digestion.
</P><H2>BBa_K2376002
</H2><H3>Lipase
</H3><P style="font-size:150%">Lipase ORF sequence took from Rhizopus niveus 
<A href="http://www.uniprot.org/uniprot/P61871">(http://www.uniprot.org/uniprot/P61871)</A>
. This enzyme hydrolyzes ester bonds of triglycerides in the working condition of 30-35-degree Celcius and pH4.5-5.5. The sequence had been optimized to a Saccharomyces cerevisiae codon and had replaced the original signal peptide (1-26 amino acids) into Saccharomyces cerevisiae SUC2 secreting signal. This enzyme was used for digesting ink in our project (such as soil bean ink), it could deinking process or lipid digesting.
 </P><H2>BBa_K2376003
</H2><H3>ste12 promoter
</H3><P style="font-size:150%"> Upstream 500bp of a pheromone related pathway transcription factor of Saccharomyces cerevisiae
<A href="http://onlinelibrary.wiley.com/doi/10.1111/j.1742-4658.2010.07728.x/full">(http://onlinelibrary.wiley.com/doi/10.1111/j.1742-4658.2010.07728.x/full)</A> 
. This promoter contains four PREs (pheromone-response element) that allows STE12 protein dimer binds on it. This is a very complex pathway induced by alpha or a factors, note that several inhibiting and antagonist pathway involves in the ste12 expression. However, the positive up-regulation feedback of the ste12 itself gives the potential of acting as an amplifier in various response pathway. We had designed this promoter for improving the expression of protein level after light inducing promoter systems. 
<A href="https://www.ncbi.nlm.nih.gov/pubmed/2193847">(https://www.ncbi.nlm.nih.gov/pubmed/2193847)</A></P><H2>BBa_K2376004
</H2><H3>ste12 protein
</H3><P style="font-size:150%">STE12 is a pheromone related pathway transcription factor of Saccharomyces cerevisiae 
<A href="http://onlinelibrary.wiley.com/doi/10.1111/j.1742-4658.2010.07728.x/full">(http://onlinelibrary.wiley.com/doi/10.1111/j.1742-4658.2010.07728.x/full)</A>
, it acts as the final controller of several pheromone induced protein expression. This protein would form a dimer and can bind to PREs (pheromone-response element) sequence. This is a very complex pathway induced by alpha or a factors in haploid budding yeast, note that several inhibiting and antagonist pathway involves in the ste12 expression. However, the positive up-regulation feedback of the ste12 itself gives the potential of acting as an amplifier in various response pathway. We had designed this promoter for improving the expression of protein level after light inducing promoter systems. <A href="https://www.ncbi.nlm.nih.gov/pubmed/2193847">(https://www.ncbi.nlm.nih.gov/pubmed/2193847)</A></P><H2>BBa_K2376005
</H2><H3>Glucanase and Xylanase with ste12 promoter
</H3><P style="font-size:150%"> This is a composite part combining endo-1,4-beta-xylanase A and Probable endo-beta-1,4-glucanase B from Neocallimastix patriciarum and Aspergillus oryzae, respectively. The protein codon had been optimized to Saccharomyces cerevisiae codon and had added a SUC2 secrete signal. Xylanase (BBa_K2376000) and glucanase (BBa_K2376001) were both promote by a ste12 promoter (BBa_K2376003) and followed by a TEF terminator (BBa_K801011). Kozak sequence was added in front of the start codon of both proteins. This composite part is used to induce the expression of deinking enzymes, it can be either activated by the induction of pheromone or any pathway that overexpress ste12 protein (BBa_K2376004). The ste12 transcription factor could bind to the ste12 promoter which has PREs, there is also a ste12 sequence in the yeast genome, which could cause a positive feedback of ste12 for binding and promote itself. The sequence of expression two proteins was reversed and formed a head-to-head sequence, wishing to lower the stereo obstacle the entry of RNA polymerase II while transcription. This composite biobrick remains central to our project. The two enzymes are the main enzymes that deink the recycled paper.
</P><H2>BBa_K2376006
</H2><P style="font-size:150%">In this biobrick, STE12 protein, a transcription factor, can be driven by the system we made or the pheromone response system originally in yeast. Once the STE12 protein has been made, it initiates the positive feedback in yeast genome. The mass production of STE12 protein helps increase STE12 promoter to lift the expression of downstream gene, Lipase, Xylanase and Glucanase. Besides, Lipase is also constructed in this biobrick.
 </P><H2>BBa_K2376007
</H2><H3>improved GAL4 based yeast light-switchable promoter system</H3><P style="font-size:150%">This biobricks was made from BBa_K801042. Which was originally designed for a red light inducible protein expression system. In order to improve the funtionality and the efficiency, we tried to reverse half of the sequence, forming a head-to-head stricture, this mAy lower the stereo effect of RNA polymerase II.
 </P></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>