"
Page
Discussion
View source
History
teams
Log in
 
Team:BostonU/FusionProteins
From 2014.igem.org
TEAM
PROJECT
Overview
Priority Encoder
MoClo Assembly Method
Low Copy Backbones
Tandem Promoters
Fusion Proteins
Repressor Proteins
Multiplexing
BioDesign Automation Tools
Future Work
ACHIEVEMENTS
Data Collected
Parts Submitted
Chimera Workflow
Chimera Example
Medal Fulfillment
NOTEBOOK
Training
Protocols
Backbones
Tandem Promoters and Repressors
Fusion Proteins
CONSIDERATIONS
Collaborations
Measurement Track
Safety
Interlab Study
NEGEM
Policy and Practices
ACKNOWLEDGEMENTS
Fusion Proteins
Why Fusion Proteins?
As described here , our main goal this summer was to come up with an efficient workflow to design, build and test large genetic devices. To demonstrate the efficacy of our workflow , we started designed and started building a priority encoder . To facilitate measurement of each component, we added the fluorescent fusion proteins to the priority encoder design. Since we lacked a lot of the internal repressor genes that we needed, we decided to test well known proteins available in our lab (tetR, lacI, and araC) with a GFP fusion protein before using this linker sequence for all fusions.
Priority encoder with fusion GFP shown linked to repressor proteins (CDS1-5)
In order to test the individual transcriptional units and the regulatory arcs associated with them, we needed the output for the individual parts of the device to be fluorescent. One way to do this is by using the bicistronic design shown below that mimics operons seen in natural bacterial systems.
Bicistronic design that could be used as an alternative to fusion proteins
A peculiar effect for these types of designs has been reported in the literature. Specifically, when ribosomes bind to the genes sequentially, the ribosome that binds at the 5' end of gene 1 can interrupt the attachment of a ribosome at the 5' end of gene 2, thus blocking the next ribosome from binding to the mRNA and translating the gene. So, the genes further down a transcript are expressed less than the genes upstream [1].
The image above is Figure 4 from Lim et al., 2011 [1]. Excerpt from Lim N H et al. (2011): "We found that CFP and YFP expression at the first position in the operons was higher than at the third position for each time point. The protein concentration at each time point was normalized by the final concentration to determine their relative expression (R). R is independent of protein production and depends only on the protein degradation rate and any delay in the induction (SI Materials and Methods). When R was calculated, it was the same for positions 1 and 3 at each time point (Fig. 4 D and E). Therefore, protein degradation and any delay after induction are the same at both positions, indicating the differences in expression must be due to greater protein production in the more proximal position of the operons."
In summary, when genes are placed in this operon strategy in synthetic systems, the first gene always expresses higher amounts of protein than the second or third genes [1]. Since these constructs are designed to measure protein degradation rate directly, these types of designs are inefficient. This problem can be solved by fusing the regulator protein with the fluorescent protein, as only one ribosome will then be required to translate the entire sequence, eliminating this problem.
Another way to measure function by fluorescence, using consecutive transcriptional units
Because it takes a minimum of 3 days to build these types of devices using MoClo these types of designs result in the expenditure of more time and resources as compared to building single transcriptional units with fusion proteins. Any design involving just one transcriptional unit instead of the above construct will making cloning faster and more cost-effective.
Fusion Proteins are fused to regulator proteins to allow us to measure degradation rate directly. They reduce order effect seen with the bicistronic design and are cheaper to build, compared to cloning multiple TUs.
Design and Assembly
Design strategy for building the fusion proteins. Two PCR products (top lines) with C and I or I and D fusion sites flanking the 5' and 3' ends are cloned into a destination vector to form a cohesive fusion protein containing the repressor protein fused with a green fluorescent protein.
To make fusion proteins, we used the Modular Cloning method .
First, we added a new MoClo fusion site (I - TCTA) to the genes (at the end of repressors and before the reporter proteins). The designed primer sequences can be accessed on the Data Collected page.
Then, we used Phusion Polymerase Chain reaction on the basic MoClo Level 0 parts using primers designed based on the fusion sites. We ended up with repressors with TCTAGA sequence at their ends and reporter proteins with the same sequence at the start. These can be treated as standard Level 0 parts which can then be assembled and tested using MoClo.
As discussed above, the end goal of building fusion proteins is to use them in the Priority Encoders with the new repressors proteins we designed. However, we wanted to test the effect of fusing proteins on regulatory proteins using this amino acid linker. Thus, we proceed to design primers for the regulators tetR, araC and for the reporters, GFP and YFP.
Testing
The following Level 2 constructs were assembled using MoClo -
* All transcriptional units have BCD2 as the 5' UTR and B0015 as the terminator
It is important that the fusion proteins made aren’t drastically inferior to the individual function of the repressor or the fusion proteins. All constructs made were to compare with those above. Simply put, all 6 Level 2s below were used a controls.
Experimental Results can be found in the Data Collected page. Detailed progress on the construction of fusion proteins can be found in the Fusion Proteins notebook .
References
[1] Lim N H, Lee Y, Hussein R (2011) "Fundamental relationship between operon organization and gene expression" ,  PNAS Vol. 108 No. 26, doi: 10.1073/pnas.1105692108
Our Sponsors
Retrieved from " http://2014.igem.org/Team:BostonU/FusionProteins "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
