<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_BostonU_FusionProteins"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:BostonU/FusionProteins">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:BostonU/FusionProteins&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:BostonU/FusionProteins&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:BostonU/FusionProteins&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:BostonU/FusionProteins" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:BostonU/FusionProteins</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><UL id="nav"><LI><A href="https://2014.igem.org/Team:BostonU/Team">TEAM</A></LI><LI><A href="">PROJECT</A></LI><LI><A href="https://2014.igem.org/Team:BostonU/Chimera">Overview</A></LI><LI><A href="https://2014.igem.org/Team:BostonU/Encoder">Priority Encoder</A></LI><LI><A href="https://2014.igem.org/Team:BostonU/MoClo">MoClo Assembly Method</A></LI><LI><A href="https://2014.igem.org/Team:BostonU/Backbones">Low Copy Backbones</A></LI><LI><A href="https://2014.igem.org/Team:BostonU/ProjectTandemPromoters">Tandem Promoters</A></LI><LI><A href="https://2014.igem.org/Team:BostonU/FusionProteins">Fusion Proteins</A></LI><LI><A href="https://2014.igem.org/Team:BostonU/Repressors">Repressor Proteins</A></LI><LI><A href="https://2014.igem.org/Team:BostonU/Multiplexing">Multiplexing</A></LI><LI><A href="https://2014.igem.org/Team:BostonU/Software">BioDesign Automation Tools</A></LI><LI><A href="https://2014.igem.org/Team:BostonU/Future">Future Work</A></LI></UL><LI><A href="">ACHIEVEMENTS</A></LI><LI><A href="https://2014.igem.org/Team:BostonU/Data">Data Collected</A></LI><LI><A href="https://2014.igem.org/Team:BostonU/Parts">Parts Submitted</A></LI><LI><A href="https://2014.igem.org/Team:BostonU/Workflow">Chimera Workflow</A></LI><LI><A href="https://2014.igem.org/Team:BostonU/ChimeraExample">Chimera Example</A></LI><LI><A href="https://2014.igem.org/Team:BostonU/Medals">Medal Fulfillment</A></LI><LI><A href="">NOTEBOOK</A></LI><LI><A href="https://2014.igem.org/Team:BostonU/Training">Training</A></LI><LI><A href="https://2014.igem.org/Team:BostonU/Protocols">Protocols</A></LI><LI><A href="https://2014.igem.org/Team:BostonU/BackbonesNotebook">Backbones</A></LI><LI><A href="https://2014.igem.org/Team:BostonU/TandemPromoters">Tandem Promoters and Repressors</A></LI><LI><A href="https://2014.igem.org/Team:BostonU/FusionProteinsNotebook">Fusion Proteins</A></LI><LI><A href="">CONSIDERATIONS</A></LI><LI><A href="https://2014.igem.org/Team:BostonU/Collaborations">Collaborations</A></LI><LI><A href="https://2014.igem.org/Team:BostonU/Measurement">Measurement Track</A></LI><LI><A href="https://2014.igem.org/Team:BostonU/Safety">Safety</A></LI><LI><A href="https://2014.igem.org/Team:BostonU/Interlab">Interlab Study</A></LI><LI><A href="https://2014.igem.org/Team:BostonU/NEGEM">NEGEM</A></LI><LI><A href="https://2014.igem.org/Team:BostonU/HumanPractices">Policy and Practices</A></LI><LI><A href="https://2014.igem.org/Team:BostonU/Acknowledgements">ACKNOWLEDGEMENTS</A></LI><DIV class="pagescontent"><PAGEHEADER>Fusion Proteins</PAGEHEADER><MAINCONTENT><TABLE width="100%" border="0" cellspacing="15" cellpadding="0"><TBODY><TR><TD colspan="2" scope="col"><H3>Why Fusion Proteins? </H3>
As described <A href="https://2014.igem.org/Team:BostonU/Chimera"> here</A>, our main goal this summer was to come up with an efficient workflow to design, build and test large genetic devices. To demonstrate the efficacy of our <A href="https://2014.igem.org/Team:BostonU/Workflow"> workflow</A>, we started designed and started building a <A href="https://2014.igem.org/Team:BostonU/Encoder"> priority encoder</A>. To facilitate measurement of each component, we added the fluorescent fusion proteins to the priority encoder design. Since we lacked a lot of the internal repressor genes that we needed, we decided to test well known proteins available in our lab (<I>tetR</I>, <I>lacI</I>, and <I>araC</I>) with a GFP fusion protein before using this linker sequence for all fusions.<CAPT><CENTER>Priority encoder with fusion GFP shown linked to repressor proteins (CDS1-5)</CENTER></CAPT>In order to test the individual transcriptional units and the regulatory arcs associated with them, we needed the output for the individual parts of the device to be fluorescent. One way to do this is by using the bicistronic design shown below that mimics operons seen in natural bacterial systems. 
<CAPT><CENTER>Bicistronic design that could be used as an alternative to fusion proteins</CENTER></CAPT>
A peculiar effect for these types of designs has been reported in the literature. Specifically, when ribosomes bind to the genes sequentially, the ribosome that binds at the 5' end of gene 1 can interrupt the attachment of a ribosome at the 5' end of gene 2, thus blocking the next ribosome from binding to the mRNA and translating the gene. So, the genes further down a transcript are expressed less than the genes upstream [1]. <CENTER><CAPT><CENTER>The image above is Figure 4 from Lim et al., 2011 [1]. Excerpt from Lim N H et al. (2011): &quot;We found that CFP and YFP expression at the first position in the operons was higher than at the third position for each time point. The protein concentration at each time point was normalized by the final concentration to determine their relative expression (R). R is independent of protein production and depends only on the protein degradation rate and any delay in the induction (SI Materials and Methods). When R was calculated, it was the same for positions 1 and 3 at each time point (Fig. 4 D and E). Therefore, protein degradation and any delay after induction are the same at both positions, indicating the differences in expression must be due to greater protein production in the more proximal position of the operons.&quot;</CENTER></CAPT></CENTER>In summary, when genes are placed in this operon strategy in synthetic systems, the first gene always expresses higher amounts of protein than the second or third genes [1]. Since these constructs are designed to measure protein degradation rate directly, these types of designs are inefficient. This problem can be solved by fusing the regulator protein with the fluorescent protein, as only one ribosome will then be required to translate the entire sequence, eliminating this problem. 


Another way to measure function by fluorescence, using consecutive transcriptional units


Because it takes a minimum of 3 days to build these types of devices using MoClo these types of designs result in the expenditure of more time and resources as compared to building single transcriptional units with fusion proteins. Any design involving just one transcriptional unit instead of the above construct will making cloning faster and more cost-effective. 

Fusion Proteins are fused to regulator proteins to allow us to measure degradation rate directly. They reduce order effect seen with the bicistronic design and are cheaper to build, compared to cloning multiple TUs.<H3>Design and Assembly</H3><CAPT><CENTER>Design strategy for building the fusion proteins. Two PCR products (top lines) with C and I or I and D fusion sites flanking the 5' and 3' ends are cloned into a destination vector to form a cohesive fusion protein containing the repressor protein fused with a green fluorescent protein.</CENTER></CAPT>
To make fusion proteins, we used the <A href="https://2014.igem.org/Team:BostonU/MoClo"> Modular Cloning </A>method . 

First, we added a new MoClo fusion site (I - TCTA) to the genes (at the end of repressors and before the reporter proteins). The designed primer sequences can be accessed on the<A href="https://2014.igem.org/Team:BostonU/Data"> Data Collected</A> page.

Then, we used Phusion Polymerase Chain reaction on the basic MoClo Level 0 parts using primers designed based on the fusion sites. We ended up with repressors with TCTAGA sequence at their ends and reporter proteins with the same sequence at the start. These can be treated as standard Level 0 parts which can then be assembled and tested using MoClo.

As discussed above, the end goal of building fusion proteins is to use them in the Priority Encoders with the new <A href="https://2014.igem.org/Team:BostonU/Repressors"> repressors proteins</A> we designed. However, we wanted to test the effect of fusing proteins on regulatory proteins using this amino acid linker. Thus, we proceed to design primers for the regulators <I> tetR, araC </I> and for the reporters, GFP and YFP.
<H3>Testing </H3>

The following Level 2 constructs were assembled using MoClo -
* All transcriptional units have BCD2 as the 5' UTR and B0015 as the terminator
It is important that the fusion proteins made aren’t drastically inferior to the individual function of the repressor or the fusion proteins. All constructs made were to compare with those above. Simply put, all 6 Level 2s below were used a controls.


Experimental Results can be found in the <A href="https://2014.igem.org/Team:BostonU/Data"> Data Collected </A>page.
Detailed progress on the construction of fusion proteins can be found in the <A href="https://2014.igem.org/Team:BostonU/FusionProteinsNotebook">Fusion Proteins notebook</A>.<H3>References</H3>
[1] Lim N H, Lee Y, Hussein R (2011) &quot;Fundamental relationship between operon organization and gene expression&quot; ,  PNAS Vol. 108 No. 26, doi: 10.1073/pnas.1105692108

</TD></TR></TBODY></TABLE></MAINCONTENT></DIV><DIV class="sponsors" align="center"><HEADER1>Our Sponsors</HEADER1></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:BostonU/FusionProteins">http://2014.igem.org/Team:BostonU/FusionProteins</A>&quot;</DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:BostonU/FusionProteins" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:BostonU/FusionProteins" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:BostonU/FusionProteins&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:BostonU/FusionProteins&amp;oldid=358966" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>