"
Page
Discussion
View source
History
teams
Log in
 
Team:BostonU/Backbones
From 2014.igem.org
TEAM
PROJECT
Overview
Priority Encoder
MoClo Assembly Method
Low Copy Backbones
Tandem Promoters
Fusion Proteins
Repressor Proteins
Multiplexing
BioDesign Automation Tools
Future Work
ACHIEVEMENTS
Data Collected
Parts Submitted
Chimera Workflow
Chimera Example
Medal Fulfillment
NOTEBOOK
Training
Protocols
Backbones
Tandem Promoters and Repressors
Fusion Proteins
CONSIDERATIONS
Collaborations
Measurement Track
Safety
Interlab Study
NEGEM
Policy and Practices
ACKNOWLEDGEMENTS
Lower Copy Backbones
Why Lower Copy Count Origins of Replication?
The work carried out by previous BU iGEM teams used destination vectors, or plasmid backbones, with high-copy origins of replication. Having only these high-copy plasmids as an option in our library would not allow for optimal performance of our multiplexed transcriptional units and larger constructs. Additionally, works upon which we are basing our complex circuit assembly do not use high copy origins for all plasmids in their devices. Circuit behavior can be desynchronized with the use of a high copy origins for all plasmids, as it causes over-expression of the plasmid in a cell, leading to a high amount of transcription and protein expression, which can sometimes be toxic or problematic for complex device function. For more complex circuits, this over-expression pushes the limit of the amount of ribosomes that can be sequestered for translation, in addition to straining the cell's protein degradation mechanisms.
Published complex genetic circuits, for example the NOR gates [1] and the Collins counter [2], are often in cloned in medium or low copy vectors, using the ColE1 or p15A origin of replication, respectively. The plasmid maps shown below are from those papers and show the origins of replication used for their work.
The image above shows two plasmid maps from the NOR paper [1] (right and middle) and one of the counters [2] (left). These were obtained from the Supplemental Information for both publications.
Detailed progress on new vector backbone creation can be found in the backbones notebook .
Design and Assembly
The origins we selected were sourced from plasmids employed in Prof. Chris Voigt's work on layered logic gates [3]. We selected the ColE1, p15A, and pSC101 origins because they are commonly used origins of replication in the literature for high, medium, and low copy number respectively. The cloning strategy involved using PCR to amplify out the desired backbone from our library without its high copy origin, in addition to the desired origin from its parent construct. The PCR primer sequences added restriction endonuclease sites on each end of the amplicons, one for the enzyme MfeI and the other for AscI These were selected because neither the backbone nor any of the origins contained restriction sites for these endonucleases. The amplicons were digested with both enzymes, and were ligated and transformed.
Schematic of origin cloning - The PCR primer design added restriction sites for the MfeI and AscI restriction enzymes, which would give the ends of each of the amplified fragments compatible 4bp overhangs suitable for ligation. (Detailed primer design available here ).
Testing
Flow cytometry testing of new low copy backbones has been carried out for the p15A origin against the pMB1 origin, using the flow cytometry controls. MoClo was used to replace the LacZ fragment of the new backbone with a transcriptional unit constitutively expressing fluorescence - all the combinations of fluorescent proteins and origins of replications are found below. More information on the flow cytometry experiments for the backbones can be found in the backbones notebook .
The combinations of plasmid origins of replication and fluorescent reporters used for testing the function of the p15A origin of replication against the high-copy pMB1 origin.
References
[1] A. Tamsir, J. Tabor, C. Voigt (2011). “Robust multicellular computing using genetically encoded NOR gates and chemical ‘wires’.” Nature 469: 212-215
[2] A. Friedland, T. Lu, X. Wang, D. Shi, G. Church, J. Collins (2009). "Synthetic Gene Networks That Count." Science 324:1199-1202 DOI: 10.1126/science.1172005
[3]  T. S. Moon, C. Lou, A. Tamsir, B. C. Stanton, C. A. Voigt (2012). "Genetic programs constructed from layered logic gates in single cells" Nature 491, 249.
Our Sponsors
Retrieved from " http://2014.igem.org/Team:BostonU/Backbones "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
