"
Page
Discussion
View source
History
teams
Log in
 
Team:BostonU/BackbonesNotebook
From 2014.igem.org
TEAM
PROJECT
Overview
Priority Encoder
MoClo Assembly Method
Low Copy Backbones
Tandem Promoters
Fusion Proteins
Repressor Proteins
Multiplexing
BioDesign Automation Tools
Future Work
ACHIEVEMENTS
Data Collected
Parts Submitted
Chimera Workflow
Chimera Example
Medal Fulfillment
NOTEBOOK
Training
Protocols
Backbones
Tandem Promoters and Repressors
Fusion Proteins
CONSIDERATIONS
Collaborations
Measurement Track
Safety
Interlab Study
NEGEM
Policy and Practices
ACKNOWLEDGEMENTS
Notebook: Lower Copy Backbones
This notebook details the process undertaken to replace the high copy pMB1 origin in our existing MoClo Level 1 and Level 2 destination vectors (named DVL1 and DVL2, respectively) with lower copy origins. Namely, the ColE1 (~50 plasmids/cell), p15A (~10 plasmids/cell), and pSC101 (~5 plasmids/cell) origins were selected to replace the high copy origin in DVL1 and DVL2.
All protocols used in this notebook are found in our protocols section.
Plasmid map of a MoClo Level 1 destination vector with original pMB1 origin of replication, LacZ fragment, and designed primers for backbone extraction.
June
Week of June 23
The backbones that would have their origin replaced were selected and new origins were selected. DVL1 with "A" and "E" MoClo fusion sites and DVL2 with "A" and "F" fusion sites were initially chosen, as they are the most commonly used MoClo level 1 and 2 destination vectors, respectively (See MoClo for more information on our assembly method). The general plan to replace the backbones was formulated, which comprised of:
1. Using PCR to extract the backbones without their high-copy origins from their full destination vectors, and to extract the lower-copy origins from their respective plasmids. (Detailed primer design available here )
2.Performing a restriction digest on the backbone and origin fragments to have compatible sticky ends.
3. Ligating the origins to the destination vectors.
4. Transforming into E. Coli, purifying the plasmid DNA, and sequencing for confirmation.
• Struck out devices with low copy origins for PCR on plates with appropriate antibiotic.
• Prepared liquid cultures, incubated, and miniprepped.
• Received primers, diluted.
Week of June 30
• Carried out PCR of DVL1_AE, DVL2_AF, ColE1, p15A, and pSC101.
• Ran gel to confirm primer functionality.
Gel verifying the successful PCR extraction of the desired backbones (~2kb each) and origins (~750-950kb).
Lanes from left to right are: 2-log ladder, DVL1_AE, DVL2_AF, ColE1, p15A, pSC101, 2-log ladder.
• Since gel verification was successful, performed PCR in triplicate of backbones and origins with same reaction conditions.
• Performed PCR cleanup and quantified using Nanodrop spectrophotometer. DVL1_AE had a very low DNA concentration compared to negative control → repeat PCR.
• Carried out 50µL MfeI restriction digest and cleanup.
• Performed ligation of each backbone with each new origin. and transformed onto Kanamycin (for DVL1 backbones) and Ampicillin (for DVL2 backbones) plates with X-Gal and IPTG. Because of the intact LacZ fragment in the plasmid, the presence of IPTG will allow for activation of β-galactosidase, which will react with X-Gal to produce blue-pigmented colonies.
• Successful blue colonies grew for DVL1_ColE1, DVL1_p15A, DVL1_pSC101, and DVL2_ColE1. No growth observed for DVL2_p15A or DVL2_pSC101. → Repeat ligation step from purified RD.
• Re-transformed DVL2_p15A and DVL2_pSC101 ligations and performed blue-white screening after overnight incubation. Blue colonies grew for DVL2_pSC101, but blue and white colonies grew for DVL2_p15A. This should not be the case, as any plasmid without the LacZ fragment should not have an origin, and therefore should not have replicated.
July
Week of July 7
• Miniprepped liquid cultures of blue colonies of DVL1_ColE1, DVL1_p15A, DVL1_pSC101, and DVL2_ColE1. Sent samples for sequencing, including "VF" and "VR" primers to verify presence of LacZ fragment, and appropriate origin primers to verify new origin presence.
• Sequence verification was successful for DVL1_ColE1 and DVL2_ColE1 origins. LacZ presence was confirmed in all sequenced samples, but desired origins were not in remaining samples. → perform colony PCR on DVL1_ColE1, DVL1_p15A, DVL1_pSC101, DVL2_ColE1, DVL2_p15A, and DVL2_pSC101 with origin primers to verify problem.
• Gel from colony PCR verified the presence of the origin on DVL1_ColE1 and DVL2_ColE1, but the PCR failed on all others. The likely cause is that the original templates of DVL1_AE, DVL2_AF (with their intact high-copy pMB1 origins), and the complete devices containing the lower copy plasmids made it past the PCR and all purifications. The pXcpi device (containing the p15 ori) had Ampicillin resistance but no LacZ fragment, which explains the white colonies on the DVL2_p15A plate.
Portion of Colony PCR gel showing no primer binding on p15A and pSC101 clones.
• To relieve this problem, the purified restriction digests were separated by gel electrophoresis, and a Qiagen gel extraction was performed to remove the bands of desired length. The desired origins and backbones were easy to pick out, since the origins vary from 750bp-950bp, distinguishing them in size from their parent constructs.
• The gel extraction was purified and a ligation was carried out to make new DVL1_p15A, DVL1_pSC101, DVL2_p15A, and DVL2_pSC101 plasmids.
• The ligations were transformed and large numbers of blue cultures grew for all ligations. The liquid cultures were miniprepped and sent for sequencing.
Week of July 14
• Ran temperature-gradient PCR from 48˚C to 58˚C to identify optimal annealing temperature for p15A and pSC101 primers.
• 54˚C identified as optimal annealing temperature.
PCR showing temperature gradient and optimal temperature for p15A amplification at 54˚C.
• Ran restriction digest and ligated overnight (12 hrs) at 16˚C.
• Sequencing for p15A and pSC101 still failing
Week of July 21
• A colony PCR on the p15A and pSC101 colonies from last week's transformations was run with their primers to determine if the correct colonies are present.
• The colony PCR failed with all p15A and pSC101 samples.
• The purified restriction digest products of the origins were sent for sequencing and returned correct results, so the incorrect final product is likely a ligation issue.
• A 2:1 insert:backbone ratio ligation was carried out, transformed, prepped and sequenced. The sequencing reactions for p15A and pSC101 failed.
Week of July 28
• A GFP single expression control was struck out and miniprepped as preparation for a FACS experiment to test the functioning of the ColE1 clones vs the controls' pMB1 origins.
• A new PCR with new reagents was carried out for the p15A and pSC101 origins, in addition to DVL1_AE and DVL2_AF backbones, all the results of which were successfully verified in a 1% agarose gel.
• A MoClo reaction was carried out to clone a constitutively-expressed GFP transcriptional unit into the DVL1_AE_ColE1 plasmids.
• FACS was successful for the control and the ColE1 clone, although their fluorescence measures were the same.
August
Week of August 4
• A BLAST analysis of the pSB1K3 plasmid on which DVL1_AE and DVL2_AF are based showed that it contains a 683-bp sequence 99% identical to our ColE1 origin already surrounding the pMB1 origin.
• New primers were designed for the backbone to be amplified outside of the "ColE1-like" region, while adding an additional 4 base pairs on the 3' and 5' ends of the MfeI restriction sites.
• Participated in NEGEM 2.0 workshop.
Week of August 18
• A Phusion PCR with the newly purchased primers of the origins and backbones was carried out, at a new Tm of 57˚C.
PCR gel showing re-amplification of desired origins and backbones Note the bright bands of remaining DVL1 and 2 template. The brighter amplicons at approximately 2000 bp were extracted.
• All the PCR products were confirmed on a gel, and were extracted. The restriction digest was carried out and an overnight ligation was carried out at insert:backbone ratios of 1:1 and 3:1.
• All ligations produced only blue colonies. One blue colony was picked per plate, prepped and sequenced. Sequencing results failed for p15A and pSC101, but were successful for ColE1.
Week of August 25
• Performed a colony PCR with 45 cycles on the failed p15A and pSC101 sequenced colonies and confirmed on gel. Longer PCR times were successful in identifying p15A, but not pSC101.
• Ran a negative-control ligation experiment by self-ligating the restriction digests of solely DVL1_AE and DVL2_AF. These self-ligations did produce blue colonies, which, after miniprepping and sequencing, revealed that they were the original backbone template.
• Since the results are likely contaminated with template, ran a Phusion PCR with 45 cycles on the final minipreps of the p15A and pSC101 origin ligations with DVL1 and DVL2. Each sample underwent 2 reactions, one with its primers and another with the ColE1 origin primers. Since the original plasmid backbones are also specific to ColE1 primers, this reaction would reveal if there is any template still present.
• 2 p15A reactions were successful for the p15A primers, but also for the ColE1 primers, indicating original backbone template contamination. The reaction failed for the pSC101 origins.
September
Week of September 1
• A new cloning strategy was devised to attempt elimination of template contamination and re-ligation. The enzyme BseYi, which has only one cut site within the pMB1 origin, would be used to linearize the DVL1 and DVL2 backbones. This digest would be dephosphorylated with Calf Intestinal Phosphatase (CIP) to prevent re-ligation of template. PCR amplification of the backbones would proceed as before with the MfeI primers, and these amplicons would also be digested with CIP after gel extraction. Following this second dephosphorylation, the MfeI digest, ligation, and transformation would proceed as before.
• The BseYI restriction endonuclease was ordered for this new strategy.
Week of September 15
• The backbones were digested and dephosphorylated according to the most recent cloning strategy, and transformed.
• Due to the amount of PCR/digest spin cleanups performed with this new strategy (4 spin cleanups and 1 gel extraction vs. 1 spin cleanup and 1 gel extraction in previous protocol), the yield of ligated DNA was very low. This resulted in no colonies growing for any of the experiments.
• The strategy was again altered, this time to include two different restriction sites on each end of the origin and backbone amplicons. The restriction endonuclease AscI was selected, as the original backbones and origins did not have any sites specific to this enzyme. Reverse PCR primers were altered to add AscI cut sites on one end of amplicons.  (Detailed primer design available here ) This strategy was aimed at returning to previous levels of cleanups to maximize yield, while eliminating background and self-ligation of template.
Summary of new cloning strategy with two restriction endonucleases.
Week of September 22
• The new primers were used to add an AscI site on one end of the DVL1, DVL2, ColE1, p15A, and pSC101 amplicons in the initial PCR step. The old forward primers were also used to add an MfeI site to the other end. This PCR was performed in triplicate to avoid loss of yield after the gel extraction. The gel showed that the PCR was successful, and the appropriate bands were extracted and purified. A restriction digest with AscI and MfeI was carried out, followed by a ligation and transformation.
The gel from the Phusion PCR on 9-23 showing the amplicons of the p15A (in duplicate), pSC101, and ColE1 (both in triplicate) origins at their correct sizes.
• Blue colonies were only observed for the ColE1 and p15A clones. Two colonies were picked per plate, and the DNA was purified and sequenced. Sequencing reactions were carried out with the MfeI primers corresponding to the samples, and the ligation of the p15A origin in the DVL1_AE and DVL2_AF backbones was confirmed.
• The ColE1 sequencing reaction was of low quality, so a colony PCR was carried out to determine the presence of ColE1. The bands from this reaction were very dim, so the original PCR was redone for the ColE1 and pSC101 origins.
Week of September 29
• A new round of cloning for the ColE1 and pSC101 origins was undertaken with addition of the MfeI and AscI restriction sites. A colony PCR was carried out before miniprepping the transformed colonies, which again had very faint bands for the ColE1 and pSC101 clones.
October
Week of October 6
• A flow cytometry experiment was prepared for testing of the gene expression with the p15A and original pMB1 origins. The DVL1_AE backbone with p15A was placed in a MoClo reaction to add one transcriptional unit with constitutive promoter J23104, ribosomal binding site BCD2, GFP E0040 or RFP E1010 genes, and terminator B0015. MoClo was used to add two of the above transcriptional units (expressing GFP then RFP, and RFP then GFP) into the DVL2_AF backbone with p15A. Our flow cytometry workflow was used to carry out the reaction, and the data were interpreted using the TASBE Tools. These results are found in our data page and the plate setup with all the samples prepared are shown below.
The plate setup used to run the single and coexpression fluorescence controls with the pMB1 origin, in addition to the samples with p15A.
October
Week of October 13
• A new round of cloning for the ColE1 and pSC101 origins was undertaken with MfeI and AscI restriction sites. After the initial PCR, the reaction was digested with the enzyme DpnI, which targets methylated DNA. Since only template DNA would be methylated, DpnI was used to linearize it and ensure that no template was incorporated at transformation. This digest was performed as an alternative to a gel extraction, which saw very low purification yields.
• After transformation, a colony PCR was carried out, which confirmed the presence of the ColE1 and pSC101 origins in DVL1_AE and DVL2_AF.
The gel confirming the ColE1 and pSC101 origins in DVL1_AE and DVL2_AF backbones.
• Sequencing results confirmed the origin ColE1 in DVL1 and DVL2, and the origin pSC101 in DVL1. A second sequencing sample with more DNA was sent for pSC101_DVL2 for verification.
• Flow cytometry experiments will be carried out for the new backbones and the results will be presented at the Jamboree.
Our Sponsors
Retrieved from " http://2014.igem.org/Team:BostonU/BackbonesNotebook "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
