<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_ETH_Zurich_project_tools"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:ETH_Zurich/project/tools">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:ETH_Zurich/project/tools&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:ETH_Zurich/project/tools&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:ETH_Zurich/project/tools&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:ETH_Zurich/project/tools" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:ETH Zurich/project/tools</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><TITLE>iGEM ETH Zurich 2014</TITLE><DIV id="header"><NAV id="nav"><UL><LI><A href="/Team:ETH_Zurich" title="Team:ETH Zurich">Home</A></LI><LI><A name="Project" style="cursor: default" href="javascript:void(0)">Project</A></LI><LI><A href="/Team:ETH_Zurich/project/overview" title="Team:ETH Zurich/project/overview">Overview</A></LI><LI><A href="/Team:ETH_Zurich/project/infopro" title="Team:ETH Zurich/project/infopro">Information Processing</A></LI><LI><A href="/Team:ETH_Zurich/project/background" title="Team:ETH Zurich/project/background">Background</A></LI><LI><A href="/Team:ETH_Zurich/project/goals" title="Team:ETH Zurich/project/goals">Goals</A></LI><LI><A href="/Team:ETH_Zurich/project/applications" title="Team:ETH Zurich/project/applications">Applications</A></LI><LI><A href="/Team:ETH_Zurich/project/references" title="Team:ETH Zurich/project/references">References</A></LI><LI><A href="/Team:ETH_Zurich/blog" title="Team:ETH Zurich/blog">Blog</A></LI></UL><LI><A name="Achievements" style="cursor: default" href="javascript:void(0)">Achievements</A></LI><LI><A href="/Team:ETH_Zurich/achievements" title="Team:ETH Zurich/achievements">Achievements</A></LI><LI><A href="/Team:ETH_Zurich/criteria" title="Team:ETH Zurich/criteria">Medal Criteria</A></LI><LI><A href="/Team:ETH_Zurich/expresults" title="Team:ETH Zurich/expresults">Results</A></LI><LI><A name="Data page" style="cursor: default" href="javascript:void(0)">Data page</A></LI><LI><A href="/Team:ETH_Zurich/data#Gene_Circuit" title="Team:ETH Zurich/data">Gene Circuit and Parts</A></LI><LI><A href="/Team:ETH_Zurich/data#Used_and_Characterized_Pre-Existing_Parts" title="Team:ETH Zurich/data">Used and Characterized Pre-Existing Parts</A></LI><LI><A href="/Team:ETH_Zurich/data#Our_Favorite_New_Characterized_Parts" title="Team:ETH Zurich/data">Our Favorite New Characterized Parts</A></LI><LI><A href="/Team:ETH_Zurich/data/raw" title="Team:ETH Zurich/data/raw">Raw Data</A></LI><LI><A name="Modeling" style="cursor: default" href="javascript:void(0)">Modeling</A></LI><LI><A href="/Team:ETH_Zurich/modeling/overview" title="Team:ETH Zurich/modeling/overview">Overview</A></LI><LI><A href="/Team:ETH_Zurich/modeling/qs" title="Team:ETH Zurich/modeling/qs">Quorum Sensing</A></LI><LI><A href="/Team:ETH_Zurich/modeling/int" title="Team:ETH Zurich/modeling/int">Integrases</A></LI><LI><A href="/Team:ETH_Zurich/modeling/xor" title="Team:ETH Zurich/modeling/xor">XOR Gate</A></LI><LI><A href="/Team:ETH_Zurich/modeling/whole" title="Team:ETH Zurich/modeling/whole">Whole cell model</A></LI><LI><A href="/Team:ETH_Zurich/modeling/diffmodel" title="Team:ETH Zurich/modeling/diffmodel">Diffusion</A></LI><LI><A href="/Team:ETH_Zurich/modeling/parameters" title="Team:ETH Zurich/modeling/parameters">Parameters and Tools</A></LI><LI><A name="Lab" style="cursor: default" href="javascript:void(0)">Lab</A></LI><LI><A href="/Team:ETH_Zurich/lab/overview" title="Team:ETH Zurich/lab/overview">Overview</A></LI><LI><A href="/Team:ETH_Zurich/lab/protocols" title="Team:ETH Zurich/lab/protocols">Materials &amp; Methods</A></LI><LI><A href="/Team:ETH_Zurich/lab/sequences" title="Team:ETH Zurich/lab/sequences">Sequences</A></LI><LI><A href="/Team:ETH_Zurich/lab/bead" title="Team:ETH Zurich/lab/bead">Alginate Beads</A></LI><LI><A href="/Team:ETH_Zurich/lab/chip" title="Team:ETH Zurich/lab/chip">Millifluidic Chip</A></LI><LI><A href="/Team:ETH_Zurich/lab/safety" title="Team:ETH Zurich/lab/safety">Safety</A></LI><LI><A name="Human Practice" style="cursor: default" href="javascript:void(0)">Human practice</A></LI><LI><A href="/Team:ETH_Zurich/human/overview" title="Team:ETH Zurich/human/overview">Overview</A></LI><LI><A href="/Team:ETH_Zurich/human/survey" title="Team:ETH Zurich/human/survey">Survey</A></LI><LI><A href="/Team:ETH_Zurich/human/interviews" title="Team:ETH Zurich/human/interviews">Interviews</A></LI><LI><A href="/Team:ETH_Zurich/human/outreach" title="Team:ETH Zurich/human/outreach">Outreach</A></LI><LI><A href="/Team:ETH_Zurich/human/essay" title="Team:ETH Zurich/human/essay">Our insights</A></LI><LI><A name="Team" style="cursor: default" href="javascript:void(0)">Team</A></LI><LI><A href="https://igem.org/Team.cgi?year=2014&amp;team_name=ETH_Zurich" class="external text" rel="nofollow">Official Team Profile</A></LI><LI><A href="/Team:ETH_Zurich/team" title="Team:ETH Zurich/team">Our team</A></LI><LI><A href="/Team:ETH_Zurich/adandin" title="Team:ETH Zurich/adandin">Advisors and Instructors</A></LI><LI><A href="/Team:ETH_Zurich/collaborations" title="Team:ETH Zurich/collaborations">Collaborations</A></LI><LI><A href="/Team:ETH_Zurich/acknowledgements" title="Team:ETH Zurich/acknowledgements">Acknowledgements</A></LI><LI><A href="/Team:ETH_Zurich/contact" title="Team:ETH Zurich/contact">Contacting us</A></LI></NAV><DIV class="inner1"><HEADER><H1>  Biological tools  </H1></HEADER><DIV style="text-align:center;"><SVG xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" version="1.1" id="Layer_1" x="50%" y="0px" width="200" height="200" viewbox="200 0 1000 1000" enable-background="new 200 0 1000 1000" xml:space="preserve"><A href="#quorum" id="bred" class="circled scrolly"><TEXT class="all" x="290" y="200" style="cursor: pointer" fill="#FFFFFF" font-size="100"><TSPAN x="290">Quorum</TSPAN><TSPAN x="290" dy="150">sensing</TSPAN></TEXT></A><A href="#integrases" id="bblue" class="circled scrolly"><TEXT class="lab" x="740" y="280" style="cursor: pointer" fill="#FFFFFF" font-size="100">Integrases</TEXT></A></SVG><H4><SPAN class="mw-headline" id="Quorum_Sensing"> Quorum Sensing </SPAN></H4><P>Bacteria, usually found in multi-species neighborhoods with billions of cells, where thought to live and divide isolated from its surroundings. However, bacteria do not try to out-compete the neighbors by mindless exponential growth but can exchange information via cell-to-cell communications in order to coordinate gene-expression and as a result the overall behavior of the population. This  communicative endeavor is commonly described as quorum sensing and based on the synthesis, secretion and diffusion of small molecules or 'autoinducers'.
</P><P>One of the best studied quorum sensing models was found in the Gram-negative bacterium <I>V. fischeri</I> in the early 1980s<SUP><A href="/Team:ETH_Zurich/project/references#refEberhard" title="Team:ETH Zurich/project/references">[4]</A></SUP>. Depending on the cell density and the corresponding autoinducer concentration, communication between the bacteria induces bioluminescence in the whole <I>V. fischeri</I> population. The small molecule responsible for this behavior was identified as <I>N</I>-(3-oxohexanoyl)-L-homoserine lactone, which is part of the <I>N</I>-acyl homoserine lactone (AHL) family. This AHL is synthesized by an enzyme called LuxI, diffuses through cell membranes and can then reach the cytosol of neighboring cells. There it binds to and activates LuxR, a protein positively regulating gene transcription. Subsequently to the investigation of <I>V. fischeri</I>, several other AHLs (with specific acyl side chains) and corresponding LuxR/LuxI like systems were found in various bacterial species, coordinating diverse responses including plasmid transfer, virulence, and antibiotic resistance<SUP><A href="/Team:ETH_Zurich/project/references#refWilliams" title="Team:ETH Zurich/project/references">[5]</A></SUP>.
</P><P>Here we focus in particular on the quorum sensing circuitry of <I>V. fischeri</I> and <I>Pseudomonas aeruginosa</I>, another Gram-negative bacteria. The additional systems are called LasR/LasI and RhlR/RhlI, also employing AHL molecules as autoinducers and are similar to the prototypical LuxR/LuxI system. Even though a key feature of quorum sensing molecules and its receptors is high specificity designed for intraspecies communication, the structural similarity of the different AHL molecules can cause unwanted gene expression. This cross-talk has to be taken into account when re-engineering LuxR/LuxI like systems for synthetic modules and circuits<SUP><A href="/Team:ETH_Zurich/project/references#refJayaraman" title="Team:ETH Zurich/project/references">[6]</A></SUP>.
</P><H4><SPAN class="mw-headline" id="Integrases"> Integrases </SPAN></H4><P>Site specific serine recombinases, or integrases, are enzymes recognizing pairs of short and non-identical DNA sequences. Within those sites they irreversibly catalyze the excision or unidirectional inversion of the DNA bases, depending on the orientation of the recognition sequence (often referred to as attB or attP). Many recombinases originate from phages integrating their genes into bacterial host genomes, hence they often do not require specific co-factors and are fully functional when heterologously expressed in bacteria. As a result, they can be employed as molecular tools in biotechnology<SUP><A href="/Team:ETH_Zurich/project/references#refKobi" title="Team:ETH Zurich/project/references">[7]</A></SUP>.
</P><P>Due to the site-specificity, recombinases like Bxb1 and ΦC31 can be employed orthogonally in synthetic systems. In addition, the combination of multiple recombinases -and their corresponding specific recognition sites- with gene regulatory sequences like promoters and terminators, allowed the engineering of Boolean logic gates<SUP><A href="/Team:ETH_Zurich/project/references#refSiuti" title="Team:ETH Zurich/project/references">[8]</A></SUP>. For example, when an asymmetric terminator sequence is placed within a pair of recombination sites, the RNA polymerase can only proceed along this sequence if exactly one recombinase is available. Otherwise the terminator is blocking gene transription by default, or due to the repeated inversion, is again fully function as soon as both recombinases are present<SUP><A href="/Team:ETH_Zurich/project/references#refBonnet" title="Team:ETH Zurich/project/references">[9]</A></SUP>. Such recombinase-based logic gates are at the core of our genetic circuits.
&lt;/article&gt;&lt;/html&gt;</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:ETH_Zurich/project/tools">http://2014.igem.org/Team:ETH_Zurich/project/tools</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:ETH_Zurich/project/tools" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:ETH_Zurich/project/tools" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:ETH_Zurich/project/tools&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:ETH_Zurich/project/tools&amp;oldid=101824" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>