<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_ETH_Zurich_lab_protocols"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:ETH_Zurich/lab/protocols">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:ETH_Zurich/lab/protocols&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:ETH_Zurich/lab/protocols&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:ETH_Zurich/lab/protocols&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:ETH_Zurich/lab/protocols" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:ETH Zurich/lab/protocols</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><TITLE>iGEM ETH Zurich 2014</TITLE><DIV id="header"><NAV id="nav"><UL><LI><A href="/Team:ETH_Zurich" title="Team:ETH Zurich">Home</A></LI><LI><A name="Project" style="cursor: default" href="javascript:void(0)">Project</A></LI><LI><A href="/Team:ETH_Zurich/project/overview" title="Team:ETH Zurich/project/overview">Overview</A></LI><LI><A href="/Team:ETH_Zurich/project/infopro" title="Team:ETH Zurich/project/infopro">Information Processing</A></LI><LI><A href="/Team:ETH_Zurich/project/background" title="Team:ETH Zurich/project/background">Background</A></LI><LI><A href="/Team:ETH_Zurich/project/goals" title="Team:ETH Zurich/project/goals">Goals</A></LI><LI><A href="/Team:ETH_Zurich/project/applications" title="Team:ETH Zurich/project/applications">Applications</A></LI><LI><A href="/Team:ETH_Zurich/project/references" title="Team:ETH Zurich/project/references">References</A></LI><LI><A href="/Team:ETH_Zurich/blog" title="Team:ETH Zurich/blog">Blog</A></LI></UL><LI><A name="Achievements" style="cursor: default" href="javascript:void(0)">Achievements</A></LI><LI><A href="/Team:ETH_Zurich/achievements" title="Team:ETH Zurich/achievements">Achievements</A></LI><LI><A href="/Team:ETH_Zurich/criteria" title="Team:ETH Zurich/criteria">Medal Criteria</A></LI><LI><A href="/Team:ETH_Zurich/expresults" title="Team:ETH Zurich/expresults">Results</A></LI><LI><A name="Data page" style="cursor: default" href="javascript:void(0)">Data page</A></LI><LI><A href="/Team:ETH_Zurich/data#Gene_Circuit" title="Team:ETH Zurich/data">Gene Circuit and Parts</A></LI><LI><A href="/Team:ETH_Zurich/data#Used_and_Characterized_Pre-Existing_Parts" title="Team:ETH Zurich/data">Used and Characterized Pre-Existing Parts</A></LI><LI><A href="/Team:ETH_Zurich/data#Our_Favorite_New_Characterized_Parts" title="Team:ETH Zurich/data">Our Favorite New Characterized Parts</A></LI><LI><A href="/Team:ETH_Zurich/data/raw" title="Team:ETH Zurich/data/raw">Raw Data</A></LI><LI><A name="Modeling" style="cursor: default" href="javascript:void(0)">Modeling</A></LI><LI><A href="/Team:ETH_Zurich/modeling/overview" title="Team:ETH Zurich/modeling/overview">Overview</A></LI><LI><A href="/Team:ETH_Zurich/modeling/qs" title="Team:ETH Zurich/modeling/qs">Quorum Sensing</A></LI><LI><A href="/Team:ETH_Zurich/modeling/int" title="Team:ETH Zurich/modeling/int">Integrases</A></LI><LI><A href="/Team:ETH_Zurich/modeling/xor" title="Team:ETH Zurich/modeling/xor">XOR Gate</A></LI><LI><A href="/Team:ETH_Zurich/modeling/whole" title="Team:ETH Zurich/modeling/whole">Whole cell model</A></LI><LI><A href="/Team:ETH_Zurich/modeling/diffmodel" title="Team:ETH Zurich/modeling/diffmodel">Diffusion</A></LI><LI><A href="/Team:ETH_Zurich/modeling/parameters" title="Team:ETH Zurich/modeling/parameters">Parameters and Tools</A></LI><LI><A name="Lab" style="cursor: default" href="javascript:void(0)">Lab</A></LI><LI><A href="/Team:ETH_Zurich/lab/overview" title="Team:ETH Zurich/lab/overview">Overview</A></LI><LI><STRONG class="selflink">Materials &amp; Methods</STRONG></LI><LI><A href="/Team:ETH_Zurich/lab/sequences" title="Team:ETH Zurich/lab/sequences">Sequences</A></LI><LI><A href="/Team:ETH_Zurich/lab/bead" title="Team:ETH Zurich/lab/bead">Alginate Beads</A></LI><LI><A href="/Team:ETH_Zurich/lab/chip" title="Team:ETH Zurich/lab/chip">Millifluidic Chip</A></LI><LI><A href="/Team:ETH_Zurich/lab/safety" title="Team:ETH Zurich/lab/safety">Safety</A></LI><LI><A name="Human Practice" style="cursor: default" href="javascript:void(0)">Human practice</A></LI><LI><A href="/Team:ETH_Zurich/human/overview" title="Team:ETH Zurich/human/overview">Overview</A></LI><LI><A href="/Team:ETH_Zurich/human/survey" title="Team:ETH Zurich/human/survey">Survey</A></LI><LI><A href="/Team:ETH_Zurich/human/interviews" title="Team:ETH Zurich/human/interviews">Interviews</A></LI><LI><A href="/Team:ETH_Zurich/human/outreach" title="Team:ETH Zurich/human/outreach">Outreach</A></LI><LI><A href="/Team:ETH_Zurich/human/essay" title="Team:ETH Zurich/human/essay">Our insights</A></LI><LI><A name="Team" style="cursor: default" href="javascript:void(0)">Team</A></LI><LI><A href="https://igem.org/Team.cgi?year=2014&amp;team_name=ETH_Zurich" class="external text" rel="nofollow">Official Team Profile</A></LI><LI><A href="/Team:ETH_Zurich/team" title="Team:ETH Zurich/team">Our team</A></LI><LI><A href="/Team:ETH_Zurich/adandin" title="Team:ETH Zurich/adandin">Advisors and Instructors</A></LI><LI><A href="/Team:ETH_Zurich/collaborations" title="Team:ETH Zurich/collaborations">Collaborations</A></LI><LI><A href="/Team:ETH_Zurich/acknowledgements" title="Team:ETH Zurich/acknowledgements">Acknowledgements</A></LI><LI><A href="/Team:ETH_Zurich/contact" title="Team:ETH Zurich/contact">Contacting us</A></LI></NAV><DIV class="inner1"><HEADER><H1>  Protocols  </H1></HEADER><CENTER><P><SVG xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" version="1.1" id="Layer_1" x="50%" y="0px" width="150" height="150" viewbox="0 0 100 100" enable-background="0 0 1000 1000" xml:space="preserve"><A href="#Materials" id="red" class="circled scrolly" style="text-decoration:none;"><TEXT x="50" y="53" style="cursor: pointer" fill="#FFFFFF" font-size="15" text-anchor="middle">Materials</TEXT></A></SVG><SVG xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" version="1.1" id="Layer_1" x="50%" y="0px" width="150" height="150" viewbox="0 0 100 100" enable-background="0 0 1000 1000" xml:space="preserve"><A href="#Methods" id="green" class="circled scrolly" style="text-decoration:none;"><TEXT x="50" y="53" style="cursor: pointer" fill="#FFFFFF" font-size="15" text-anchor="middle">Methods</TEXT></A></SVG></P></CENTER><H2><SPAN class="mw-headline" id="Materials">Materials</SPAN></H2><H3><SPAN class="mw-headline" id="NanoDrop_2000.2C_UV-Vis_Spectrophotometer">NanoDrop 2000, UV-Vis Spectrophotometer</SPAN></H3><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption"><B>Figure 1</B> Our NanoDrop spectrophotometer</DIV></DIV></DIV><P>A Thermo Scientific NanoDrop® 2000 spectrophotometer was used to determine the concentrations of our plasmids after miniprep. 
</P><P>This microvolume spectrophotometer can be used to measure the concentration and purity of DNA, RNA or protein samples. Having a wide spectral range (190-840nm) allows for measuring a variety of samples types such as peptides, DNA, RNA, purified protein and even gold nanoparticles.
Even for highly concentrated samples no dilutions are required. Results are obtained in less than 15 seconds from sample pipetting to wiping the pedestal clean. For the measurement only 0.5 – 2uL of the samples are needed.
The software with an intuitive user interface allows for classical NanoDrop applications such as nucleic acids, protein A280, colorimetric assays and new user defined custom methods. 
</P><P>Please visit the <A href="http://www.nanodrop.com/Productnd2000overview.aspx" class="external text" rel="nofollow">NanoDrop webpage</A> for more information. 
</P><H3><SPAN class="mw-headline" id="Tecan_Infinite_M200_Pro.E2.84.A2">Tecan Infinite M200 Pro™</SPAN></H3><P>For performing fluorescence measurements we used a Tecan Infinite M200 ProTM equipped with the Tecan iControl TM software. 
The Infinite 200 PRO is a user-friendly and affordable multimode reader, designed to cater for the needs of today’s applications. Based on the highly successful Infinite 200 series of microplate reader, the Infinite 200 PRO can provide a full range of leading detection methods in one easy-to-use modular instrument, with either Quad4 monochromator™ or filter-based technologies. Users can select the desired modules to create the perfect reader for their needs, with the option to upgrade as requirements change. The Infinite 200 PRO offers excellent sensitivity, multiplexing capabilities and high format flexibility, including 6- to 384-well microplates, PCR plates, cuvettes and Tecan’s patentedNanoQuant Plate™ for low sample volumes.
</P><UL><LI>For uncompromised performance in all detection modes, the Infinite 200 PRO uses advanced optics and high-performance detectors, optimized for the requirements of fluorescence, luminescence and absorbance reading. The Infinite 200 PRO offers a wide range of detection modes, including:
</LI><LI>Fluorescence intensity (top and bottom reading) (UV – NIR &amp; wavelength scanning)
</LI><LI>Fluorescence resonance energy transfer (FRET)
</LI><LI>Time resolved fluorescence energy transfer (TR-FRET) like HTRF®
</LI><LI>Time resolved fluorescence (TRF)
</LI><LI>Fluorescence polarization (FP)
</LI><LI>Flash luminescence
</LI><LI>Glow luminescence
</LI><LI>Dual-color luminescence (including BRET 1 &amp; BRET 2 applications)
</LI><LI>Absorbance  (UV – NIR &amp; wavelength scanning)
</LI><LI>AlphaScreen® and AlphaLISA® technology
</LI><LI>Injectors system for up to two reagents
</LI><LI>Temperature control
</LI><LI>Small sample measurement in NanoQuant Plate
</LI></UL><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><B>Figure 2</B>Tecan Infinite M200 Pro™</DIV></DIV></DIV></DIV><P>Please visit the <A href="http://www.tecan.com/platform/apps/product/index.asp?MenuID=1812&amp;ID=1916&amp;Menu=1&amp;Item=21.2.10.1" class="external text" rel="nofollow">Tecan webpage</A> for more information.
</P><H3><SPAN class="mw-headline" id="Tecan_iControl.E2.84.A2_software">Tecan iControl™ software</SPAN></H3><P>Our Tecan is equipped with i-control, the easy-to-use microplate reader software. It allows the user to define the workflow for each application. Each workflow is easily created by dragging and dropping the processing steps into a sequence according to the assay protocol. The application workflow is then visible to the user and can be saved for future use. Data are easily managed and exported to Windows® compatible formats (Excel®).
Intuitive
Easy set up of workflows via drag &amp; drop
Workflows are visible to the user
Workflows may be saved as measurement scripts
On-line data presentation in Excel®
Graphical definition of measurement range
Plate definition editor
Enhanced data management
Export of data into user-defined Excel® templates
Flexible
User sets up the workflow to suit his own application
Multilabel measurements for multiplexed assays
Different kinds of kinetic measurements
Supports ratio mode also for well kinetic measurements
</P><P>Please visit the <A href="http://www.tecan.com/platform/apps/product/index.asp?MenuID=1817&amp;ID=1924&amp;Menu=1&amp;Item=21.7.8" class="external text" rel="nofollow">Tecan software webpage</A> for more information.
</P><H3><SPAN class="mw-headline" id="Novaspec_Plus.E2.84.A2_Visible_Spectrophotometer">Novaspec Plus™ Visible Spectrophotometer</SPAN></H3><P>Novaspec™ Plus Visible Spectrophotometer is ideal for general biotech laboratory use. There are stored methods for the standard colorimetric methods of Bradford, BCA, Biuret and Lowry in addition to the basic modes of absorbance, transmittance,OD600 and concentration measurements. The use of diode array technology permits rapid wavelength scans to be made as well and, as there are no moving parts, make for a highly reliable and low maintenance product (Fig 1). 
The large backlit display enables the graphical results for wavelength scans, kinetics assays (including slope calculation for rate/activity studies) and standard curves to be viewed (Fig 2).Up to 99 methods for such experiments can be stored for instant recall and use by a laboratory assistant. Novaspec Plusis delivered with Graﬁco, a PC utility software package, and the requisite serial lead, providing the user with the means to capture, print and store data from the instrument (with a time/date stamp) onto a PC so that a results log can be built up or data exported to Microsoft™ Excel.
</P><P>The cell holder supplied with Novaspec Plus accepts standard 10-mm pathlength glass or plastic cells (adapters are availableto convert it to accept 10-, 12- and 16-mm diameter test tubes).It can be removed for cleaning, or ﬂushed through with water in situ, if spillages occur. 
For users who require thermostatting, particularly for kineticsstudies and use with test kits, a special version of Novaspec Plus is available with a factory ﬁtted heated cell holder (37 °C only);note that this cell holder cannot be added retrospectively.
</P><P>Features:
</P><UL><LI>“Flash Scan” diode array 
</LI><LI>Stored protein methods
</LI><LI>Bacterial cell culture measurement at OD600
</LI><LI>Kinetics for activity studies
</LI><LI>Stored methods
</LI></UL><P>
Please visit the <A href="http://www.heraco.se/images/user/PDF/Novaspec%20Plus.pdf" class="external text" rel="nofollow">Novaspec™ Plus Visible Spectrophotometer webpage</A> for more information.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption"><B>Figure 3</B> Novaspec Plus™ Visible Spectrophotometer</DIV></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Biostep_Dark-Hood_DH-50.E2.84.A2_and_the_Argus-X1.E2.84.A2_software">Biostep Dark-Hood DH-50™  and the Argus-X1™ software</SPAN></H3><P>
Pictures of our gels, long term imaging of our chips with beads containing GFP-expressing cells and pictures of our plates with bacterial colonies were recorded with the  DH-50™ and the Argus-X1™ software. 
</P><P>Dark Hood DH-50 with control panel, display, sliding table, UV protection, for CCD camera
stationary dark hood with large door
sliding table to put on a transilluminator incl. UV protection shield – no additional protection necessary
dimmable white top-light by LEDs
automatic UV cut-off when opening the UV protection shield
preparative function enables observation of gels under UV light
control panel with keypad and LCD display
</P><P>Functionality: 
transilluminator: on/off
white top-light: on/off, dim function
preparative function
optionally available: control of zoom, aperture and focus by motor zoom objective
optionally available: storage of application methods repeatable
</P><UL><LI> transilluminators with dimensions (W x D x H): 32.5x32.2x10.5cm; corresponds to the series of biostep UST, USDT, UXFT, BST, GST, YST, RST, WST 
</LI></UL><P>optionally upgradeable with UV protection shield, filter wheel, white transmission light, UV or epi light in the visible range
</P><P>
Please visit the <A href="http://www.biostep.de/i18n_en/products/Bio_Imaging_components_1776/Tripod__dark_hoods_and_accessories_1863/Dark_hoods_1865/Dark_Hood_DH_50_6/index.html" class="external text" rel="nofollow">Biostep Dark-Hood DH-50™ webpage</A> for more information.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption"><B>Figure 4</B> Biostep Dark-Hood DH-50™</DIV></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Biostep_argusX1.E2.84.A2_basic_licence">Biostep argusX1™ basic licence</SPAN></H3><P>PC-software for controlling the camera basic modules Felix 1000/2000, 5000/6000/7000 and the transmission scanners of the series ViewPix 900/1300 
</P><P>Attention: Imperatively necessary: module for controlling the camera or scanner!!!
</P><P>General software functions: 
integrated database for administration of the images
copying and cropping of single images or group images within a database or even between database to database if the additional module Project administration exists
deletion of single images or group images
functions for image processing incl. Undo
rotating in arbitrary angles and around 90°, 180°
reflecting in horizontal and vertical direction
inverting
cropping
various filter functions
marking of the images with text, rectangle and line
automated generation and numeration of sample names after the acquisition
memo function: input of a note for each image and printout of the same on a protocol
history for documentation of the acquisition parameters for each image
overexposure control
data export into other Windows applications
data import of images in different formats
creation of individual print reports
quick print
selection of the export format, export path, print report for quick print
selection of the user language: German, English, Spanish, French or Italian
control of the functions a of biostep transilluminators with PC interface
</P><P>Please visit the <A href="http://www.biostep.de/products/Bio_Imaging_software_1843/Acquisition_and_control_software_1844/argus_X1___for_cameras_and_scanners_1845/biostep_argusX1__basic_licence_1/index.html" class="external text" rel="nofollow">Biostep argusX1™ basic licence webpage</A> for more information.
</P><H3><SPAN class="mw-headline" id="Eppendorf_Centrifuge_5810_R">Eppendorf Centrifuge 5810 R</SPAN></H3><P>Centrifuge 5810/5810 R is a workhorse for medium to high-throughput laboratories. It combines extraordinary versatility and capacity for both tubes and plates with an extraordinary compact footprint. With the new rotor generation the capacity of Centrifuge 5810/ 5810 R increases to max. 4 x 750 mL (or max. 28 x 50 mL / 56 x 15 mL). Considering its small footprint, Centrifuge 5810/5810 R is the most compact 3 liter centrifuge currently available in the market. 
</P><P>Features: 
Swing-bucket rotors and adapters accommodate tubes and bottles from 0.2 mL to 750 mL
Plate rotors for centrifugation of all types of MTP, PCR, cell culture, or deepwell plates
Fixed-angle rotors for high-speed molecular biology applications in tubes from 0.2 mL to 85 mL
High centrifugation speed of up to 20,913 x g (14,000 rpm)
Aerosol-tight Eppendorf QuickLock® caps and lids for easy, one-hand operation
Centrifuge lid with soft-touch lid closure
Low access height of 29 cm for easy loading and unloading of samples
Quiet operation to improve your work environment
Compact footprint saves valuable bench space
Automatic rotor recognition and imbalance detection for maximum operational safety
Additional features of refrigerated Centrifuge 5810 R:
Temperature range from -9 °C to 40 °C
FastTemp function for fast pre-cooling
Standby cooling function holds temperature when centrifuge is not in use
ECO shut-off engages after 8 hours of non-use to reduce energy consumption and to extend compressor life
Dynamic compressor control (DCC) technology for optimized cooling performance
</P><P>Please visit the <A href="http://www.eppendorf.com/int/index.php?pb=5ffa644d89305ae7&amp;action=products&amp;contentid=1&amp;catalognode=10019&amp;productpage=1" class="external text" rel="nofollow">Eppendorf Centrifuge 5810 R webpage</A> for more information.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption"><B>Figure 5</B> Eppendorf Centrifuge 5810 R</DIV></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Eppendorf_Centrifuge_5424_R">Eppendorf Centrifuge 5424 R</SPAN></H3><P>
This highly advanced 24-place model is unbelievable quiet and equipped with a number of innovative features. The Centrifuge 5424 and Centrifuge 5424 R set new standards for silence, speed and simplicity – for others to follow! The micro centrifuge standard in your lab – refrigerated. It features all the benefits already combined in the Centrifuge 5424 and adds the patented cooling feature for temperature sensitive samples.
</P><P>Product features of Centrifuge 5424 and 5424 R
Very compact size
Very silent operation, even without the rotor lid
Low access height – for easy loading and unloading of the centrifuge
Can be equipped with four different rotors:
- Standard 24-place rotor, aerosol-tight
- 24-place rotor, aerosol-tight, PTFE coated for increased chemical resistance
- 18-place Kit rotor for processing spin-columns with open tube lids
- 4 × 8 PCR strip rotor
Two different operation panel
Rotary knobs for fast and easy setting of parameters
Foil version for easier cleaning of the front panel
Automatic lid opening following centrifuge run – prevents sample warming (can be optionally deactivated (only 5424))
Separate Short-spin button – for fast and short spins
SOFT-brake to protect delicate samples
</P><P>
Additional cooling features of refrigerated Centrifuge 5424 R
Patented compressor technology minimizes vibrations – protects your precious sample!
Temperature range: –9 °C to +40 °C
Small footprint and low access height of 26 cm, as refrigeration unit is at the rear of the unit
Maintains constant 4 °C at max. speed
Fast Temp for rapid pre-cooling of the centrifuge within 8 minutes from ambient temperature 23 °C
Continuous cooling maintains temperature when centrifugation process is not active
</P><P>
Please visit the <A href="http://www.eppendorf.com/int/index.php?sitemap=2.1&amp;pb=92b8536165b27ad1&amp;action=products&amp;contentid=1&amp;catalognode=22420&amp;productpage=1" class="external text" rel="nofollow">Eppendorf Centrifuge 5424 R webpage</A> for more information.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption"><B>Figure 6</B> Eppendorf Centrifuge 5424 R</DIV></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Harvard_Pump_11_Plus">Harvard Pump 11 Plus</SPAN></H3><P>Our team used the syringe pump for fabricating the alginate beads containing our bacteria. 
</P><P>Harvard Apparatus’ Syringe Pump 11 Plus combines smoother flow and updated features to create a high performance syringe pump with a basic syringe pump price! 
</P><P>Bright Display and Easy-To-Use Interface 
A new, two-line 16 character vacuum fluorescent display and six membrane keys make this a powerful, easy-to-use syringe pump. Only two entries are required to start pumping; Syringe ID and flow rate. The flow rate can be changed while the pump is running. 
</P><P>Two Modes of Operation, 
Constant Flow Rate &amp; Volume Dispense 
The new Pump 11 Plus will operate continuously in RATE mode or accurately dispense a specific amount of fluid in VOLUME mode. 
</P><P>Versatile 
Select from two different models. The standard Pump 11 Plus model has the following features: 
• Infusion Only 
• End of Travel Limit Stop 
• Anti-Siphon Bracket 
</P><P>Non-Volatile Memory 
New micro-stepping pump profiles deliver very smooth and consistent flow over the entire flow rate range. These pumps now feature non-volatile memory. The pump remembers its last syringe size, flow rate used and configuration settings. An advanced universal power supply means there is no need to change AC line switches, fuses or wires. 
</P><P>
Please visit the <A href="https://www.harvardapparatus.com/webapp/wcs/stores/servlet/product_11051_10001_44010_-1_HAI_ProductDetail___" class="external text" rel="nofollow">Harvard Pump 11 Plus webpage</A> for more information.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption"><B>Figure 7</B> Harvard Apparatus syringe Pump 11 Plus</DIV></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="BD_LSRFortessa.E2.84.A2_Flow_Cytometer_System">BD LSRFortessa™ Flow Cytometer System</SPAN></H3><P>
In order to determine the exact sensivity shift and expression level of GFP with the mutated promoters the BD LSRFortessa™ Flow Cytometer System was used for the measurement,FACSDiva™ for the recording of the data and the FlowJo™ software for the data analysis.
</P><P>BD LSRFortessa™ Flow Cytometer System
</P><P>The BD LSRFortessa™ cell analyzer offers the ultimate in choice for flow cytometry, providing power, performance, and consistency. 
Designed to be affordable and expandable, the BD LSRFortessa has the flexibility to support the expanding needs of multicolor flow cytometry assays.The BD LSRFortessa analyzer can be configured with up to 4 lasers— blue, red, violet, and UV—which enables the detection of up to 18 colors simultaneously. The BD LSRFortessa may be upgraded subsequently with additional or new lasers from BD, as future requirements dictate.Innovative Designs for Optical Efficiency Built on a proven platform, BD LSR analyzers feature innovative designs for both the excitation optics and collection optics that reduce excitation losses and improve light collection efficiency. The result is optical efficiency that delivers maximum sensitivity and resolution for multicolor applications by reducing excitation laser light loss, and improved fluorescent signal collection.Special Order Research ProgramFor research needs at the leading edge of biomedical discovery, the BD LSRFortessa special order research program offers the latest innovations in laser technology to researchers to enable future upgrades with new excitation, wavelength, and power choices. The program lets customers fully customize configurations to deliver added flexibility and power to support advanced research.
</P><P>
Please visit the <A href="http://www.bdbiosciences.com/documents/bd_lsrfortessa_brochure.pdf" class="external text" rel="nofollow">BD LSRFortessa™ Flow Cytometer System webpage</A> for more information.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption"><B>Figure 8</B> BD LSRFortessa™ Flow Cytometer System</DIV></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="FACSDiva.E2.84.A2_software">FACSDiva™ software</SPAN></H3><P>BD FACSDiva™ software is a collection of rich tools for flow cytometer and application setup, data acquisition, and data analysis that help streamline flow cytometry workflows for today’s busy laboratory.
</P><P>BD FACSDiva software provides new features to help users integrate flow systems into new application areas, including index sorting for stem cell and single-cell applications, as well as automation protocols for high-throughput and robotic laboratories.
</P><P>Please visit the <A href="http://www.bdbiosciences.com/instruments/software/facsdiva/" class="external text" rel="nofollow">FACSDiva™ software webpage</A> for more information.
</P><H3><SPAN class="mw-headline" id="MakerBot_3D-printer">MakerBot 3D-printer</SPAN></H3><P>The mold designs were printed with a commercial 3D-printer (2nd generation MakerBot with MakerWare software; MakerBotIndustries, Brooklyn, US; 5th generation US$2'899) with acrylonitrile butadiene styrene (ABS, about US$160 per kg). The maximum object size printable is [mm]: 225 x 145 x 150. The precision and minimum feature size are given as [mm]: 0.011 (XY-axis), 0.0025 (Z-axis); and 0.4 (XY-axis), 0.2 (Z-axis) respectively.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption"><B>Figure 9</B> 2nd generation MakerBot</DIV></DIV></DIV></DIV><H2><SPAN class="mw-headline" id="Methods">Methods</SPAN></H2><H3><SPAN class="mw-headline" id="LB_medium_from_dehydrated_product">LB medium from dehydrated product</SPAN></H3><P>Suspend/dissolve the compounds in 1 L of purified water:
</P><TABLE border="0"><TBODY><TR><TD align="left" width="300"> Difco™ LB Agar, Miller </TD><TD align="left"> 40 g
</TD></TR><TR><TD align="left"> Difco™ LB Broth, Miller </TD><TD align="left"> 25 g
</TD></TR><TR><TD align="left"><I>Mix thoroughly</I></TD></TR><TR><TD align="left"><I>Boil the agar medium with agitation for 1 min to dissolve the powder</I></TD></TR><TR><TD align="left"><I>Autoclave at 121°C for 15 min</I></TD></TR></TBODY></TABLE><P>According to <A href="https://www.bd.com/europe/regulatory/Assets/IFU/Difco_BBL/244510.pdf" class="external text" rel="nofollow">LB Agar, Miller; LB Broth, Miller</A></P><H3><SPAN class="mw-headline" id="Preparation_of_antibiotics_stock_.281000x.29">Preparation of antibiotics stock (1000x)</SPAN></H3><TABLE border="0"><TBODY><TR><TD align="left"> Ampicillin (200 mg/mL) in H<SUB>2</SUB>O, sterile filtered
</TD></TR><TR><TD align="left"> Kanamycin (50 mg/mL) in H<SUB>2</SUB>O, sterile filtered
</TD></TR><TR><TD align="left"> Chloramphenicol (34 mg/mL) in ethanol
</TD></TR><TR><TD align="left"> Tetracycline (10 mg/mL) in ethanol
</TD></TR><TR><TD align="left"> Streptamycin (25 mg/mL) in H<SUB>2</SUB>O, sterile filtered
</TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Defined_M9_medium_with_0.4_.25_glycerol">Defined M9 medium with 0.4 % glycerol</SPAN></H3><P>Suspend/dissolve the sterile compounds in purified water to give 1 L final volume:
</P><TABLE border="0"><TBODY><TR><TD align="left" width="300"><A href="http://openwetware.org/wiki/M9_salts" class="external text" rel="nofollow">M9 salts, 5x</A></TD><TD align="left"> 200 mL
</TD></TR><TR><TD align="left"> 30% (v/v) Glycerol </TD><TD align="left"> 13.33 mL
</TD></TR><TR><TD align="left"> 10% (w/v) CAA </TD><TD align="left"> 100 mL
</TD></TR><TR><TD align="left"> Thiamin (10 g/L) </TD><TD align="left">  2 mL
</TD></TR><TR><TD align="left"> TE-US solution<SUP><A href="/Team:ETH_Zurich/project/references#refPanke" title="Team:ETH Zurich/project/references">[33]</A></SUP></TD><TD align="left"> 1 mL
</TD></TR><TR><TD align="left"> 1M CaCl<SUB>2</SUB></TD><TD align="left"> 0.5 mL
</TD></TR><TR><TD align="left"> 1M MgSO<SUB>4</SUB></TD><TD align="left"> 2 mL
</TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="SOC">SOC</SPAN></H3><P>Suspend/dissolve the compounds in 1 L of purified water:
</P><TABLE border="0"><TBODY><TR><TD align="left" width="300"> Tryptone </TD><TD align="left"> 20 g
</TD></TR><TR><TD align="left"> Yeast extract </TD><TD align="left"> 5 g
</TD></TR><TR><TD align="left"> NaCl </TD><TD align="left"> 0.5 g
</TD></TR><TR><TD align="left"><I>Dissolve, then add</I></TD></TR><TR><TD align="left"> KCl (250 mM) </TD><TD align="left"> 10 mL
</TD></TR><TR><TD align="left"> MgCl<SUB>2</SUB></TD><TD align="left"> 5 mL
</TD></TR><TR><TD align="left"><I>Autoclave, then add</I></TD></TR><TR><TD align="left"> Sterile glucose (1 M) </TD><TD align="left"> 20 mL
</TD></TR></TBODY></TABLE><P>According to <A href="http://www.qiagen.com/resources/molecular-biology-methods/dna/" class="external text" rel="nofollow">QIAGEN <I>DNA protocols and applications</I></A></P><H3><SPAN class="mw-headline" id="Transformation_buffer_1_.28TFB1.29">Transformation buffer 1 (TFB1)</SPAN></H3><P>Suspend/dissolve the compounds in 1 L of purified water:
</P><TABLE border="0"><TBODY><TR><TD align="left" width="300"> RbCl (100 mM)  </TD><TD align="left"> 12.1 g
</TD></TR><TR><TD align="left"> MnCl<SUB>2</SUB>·4H<SUB>2</SUB>O (50 mM) </TD><TD align="left"> 9.9 g
</TD></TR><TR><TD align="left"> Potassium acetate (30 mM) </TD><TD align="left"> 2.9 g
</TD></TR><TR><TD align="left"> CaCl<SUB>2</SUB>·2H<SUB>2</SUB>O (10 mM) </TD><TD align="left"> 1.4 g
</TD></TR><TR><TD align="left"> Glycerol (15%) </TD><TD align="left"> 150 mL
</TD></TR><TR><TD align="left"><I>Adjust pH to 5.8  with KOH</I></TD></TR><TR><TD align="left"><I>Sterilize by filtration</I></TD></TR></TBODY></TABLE><P>According to <A href="http://www.qiagen.com/resources/molecular-biology-methods/dna/" class="external text" rel="nofollow">QIAGEN <I>DNA protocols and applications</I></A></P><H3><SPAN class="mw-headline" id="Transformation_buffer_2_.28TFB2.29">Transformation buffer 2 (TFB2)</SPAN></H3><P>Suspend/dissolve the compounds in 1 L of purified water:
</P><TABLE border="0"><TBODY><TR><TD align="left" width="300"> RbCl (10 mM)  </TD><TD align="left"> 1.2 g
</TD></TR><TR><TD align="left"> MOPS (10 mM) </TD><TD align="left"> 2.1 g
</TD></TR><TR><TD align="left"> CaCl<SUB>2</SUB>·2H<SUB>2</SUB>O (75 mM) </TD><TD align="left"> 11.0 g
</TD></TR><TR><TD align="left"> Glycerol (15%) </TD><TD align="left"> 150 mL
</TD></TR><TR><TD align="left"><I>Adjust pH to 6.8  with KOH</I></TD></TR><TR><TD align="left"><I>Sterilize by filtration</I></TD></TR></TBODY></TABLE><P>According to <A href="http://www.qiagen.com/resources/molecular-biology-methods/dna/" class="external text" rel="nofollow">QIAGEN <I>DNA protocols and applications</I></A></P><H3><SPAN class="mw-headline" id="Gibson_Assembly_reaction_mixture">Gibson Assembly reaction mixture</SPAN></H3><H3><SPAN class="mw-headline" id="5x_isothermal_reaction_buffer">5x isothermal reaction buffer</SPAN></H3><TABLE border="0"><TBODY><TR><TD align="left" width="300"> Tris-HCl pH 7.5 (1 M)  </TD><TD align="left"> 3 mL
</TD></TR><TR><TD align="left"> MgCl<SUB>2</SUB> (2 M) </TD><TD align="left"> 150 μL
</TD></TR><TR><TD align="left"> dGTP (100 mM) </TD><TD align="left"> 60 μL
</TD></TR><TR><TD align="left"> dATP (100 mM) </TD><TD align="left"> 60 μL
</TD></TR><TR><TD align="left"> dTTP (100 mM) </TD><TD align="left"> 60 μL
</TD></TR><TR><TD align="left"> dCTP (100 mM) </TD><TD align="left"> 60 μL
</TD></TR><TR><TD align="left"> DTT (1 M) </TD><TD align="left"> 300 mL
</TD></TR><TR><TD align="left"> PEG-8000 </TD><TD align="left"> 1.5 g
</TD></TR><TR><TD align="left"> NAD (100 mM) </TD><TD align="left"> 300 μL
</TD></TR></TBODY></TABLE><P>Aliquots can be stored at -20 °C
</P><H3><SPAN class="mw-headline" id="Assembly_master_mixture">Assembly master mixture</SPAN></H3><TABLE border="0"><TBODY><TR><TD align="left" width="400"> Isothermal reactio buffer (5x)  </TD><TD align="left"> 320 μL
</TD></TR><TR><TD align="left"> T5 exonuclease (10 U/μL) </TD><TD align="left"> 0.64 μL
</TD></TR><TR><TD align="left"> Phusion DNA polymerase (2 U/μL) </TD><TD align="left"> 20 μL
</TD></TR><TR><TD align="left"> Taq DNA ligase (40 U/μL) </TD><TD align="left"> 160 μL
</TD></TR><TR><TD align="left"> H<SUB>2</SUB>O </TD><TD align="left"> 1.2 mL
</TD></TR></TBODY></TABLE><P>Aliquots of 15 μL can be stored at -20 °C
</P><H3><SPAN class="mw-headline" id="Complex_bead_medium_.28CB_medium.29">Complex bead medium (CB medium)</SPAN></H3><TABLE border="0"><TBODY><TR><TD align="left" width="300"> Yeast extract  </TD><TD align="left"> 4 g
</TD></TR><TR><TD align="left"> Typtone </TD><TD align="left"> 1 g
</TD></TR><TR><TD align="left"> Glycerol </TD><TD align="left"> 1 g
</TD></TR><TR><TD align="left"> Tris(hydroxymethyl)aminomethane </TD><TD align="left"> 1.21 g
</TD></TR><TR><TD align="left"><I>Fill up to 980 ml with H<SUB>2</SUB>O</I></TD></TR><TR><TD align="left"><I>Adjust pH to 7  with HCl</I></TD></TR><TR><TD align="left"> CaCl<SUB>2</SUB>⋅H<SUB>2</SUB>O (100 mM) </TD><TD align="left"> 20 mL
</TD></TR><TR><TD align="left"><I>Sterilize by filtration</I></TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Preparation_of_chemically_competent_E._coli">Preparation of chemically competent <I>E. coli</I></SPAN></H3><P>Day before: Preparation of a Top10 cells preculture  in LB-medium containing streptomycin (25 μg/mL)
</P><OL><LI> Addition of 1 mL overnight preculture to 100 mL LB-medium + streptomycin (25 μg/mL)
</LI><LI> Cultivate culture at 37 °C, 220 rpm until it reaches an OD600 of 0.5
</LI><LI> Cool culture for 5 min on ice and centrifuge it for 5 min at 4 °C, 4000 g
</LI><LI> Discard the supernatant and resuspend the cells in cold TFB1 buffer (30 mL, 4 °C)
</LI><LI> Keep the suspension on ice for 90 min
</LI><LI> Centrifuge the suspension for 5 min at 4 °C, 4000 g and discard the supernatant
</LI><LI> Resuspend the cells in 4 mL cold TFB2 buffer
</LI><LI> Make aliquots of 100 μL and freeze the aliquots in dry ice in ethanol
</LI><LI> Store aliquots at -80 °C
</LI></OL><P>According to <A href="http://www.qiagen.com/resources/molecular-biology-methods/dna/" class="external text" rel="nofollow">QIAGEN <I>DNA protocols and applications</I></A></P><H3><SPAN class="mw-headline" id="Preparation_of_electrocompetent_E._coli">Preparation of electrocompetent <I>E. coli</I></SPAN></H3><OL><LI> Grow a 5mL suspension of your cells of interest to OD=0.6
</LI><LI> Pre-cool your centrifuge to 4°C
</LI><LI> Pre-cool autoclaved water on ice
</LI><LI> Cool the cells on ice for 5-10 minutes
</LI><LI> Centrifuge for 6 minutes
</LI><LI> Discard supernatant
</LI><LI> Resuspend pellet with 1mL of chilled water
</LI><LI> Centrifuge for 6 min
</LI><LI> Repeat washing step followed by centrifugation 3 times
</LI><LI> After last centrifugation step, discard supernatant and resuspend pellet in 50uL chilled water.
</LI><LI> Add plasmid you want to transform 
</LI><LI> Electroporate
</LI><LI> Add SOC medium and let cells recover at 37°C for 1 hour
</LI><LI> Streak on LB-Agar plates with the corresponding antibiotic
</LI></OL><H3><SPAN class="mw-headline" id="Preparation_of_DNA_from_iGEM_kit">Preparation of DNA from iGEM kit</SPAN></H3><OL><LI>Add 10 μL H<SUB>2</SUB>O to appropriate well, wait for 5 min, transfer into sterile tube
</LI><LI>Use 2 μL to transform competent cells
</LI></OL><H3><SPAN class="mw-headline" id="Biological_parts_used_from_iGEM_kit:">Biological parts used from iGEM kit:</SPAN></H3><TABLE border="0"><TBODY><TR><TD align="left" width="300"><B> Part-ID</B></TD><TD align="left"><B>Function</B></TD></TR><TR><TD align="left"> BBa_F2620 </TD><TD align="left"> tetR-<I>luxR</I>-pLuxR
</TD></TR><TR><TD align="left"> BBa_K084007 </TD><TD align="left"> pLacI-<I>lasI</I></TD></TR><TR><TD align="left"> BBa_C0070 </TD><TD align="left"><I>rhlI</I>-LVA
</TD></TR><TR><TD align="left"> BBa_C0171 </TD><TD align="left"><I>rhlR</I></TD></TR><TR><TD align="left"> BBa_J23100 </TD><TD align="left"> constitutive promoter
</TD></TR><TR><TD align="left">BBa_K553003</TD><TD align="left"> promoter-RBS-<I>lasR</I>-Term
</TD></TR><TR><TD align="left"> BBa_C0161 </TD><TD align="left"><I>luxI</I></TD></TR></TBODY></TABLE><P>For confirmation all plasmids used from the iGEM kit were sequenced
</P><H3><SPAN class="mw-headline" id="Transformation_of_competent_E._coli">Transformation of competent <I>E. coli</I></SPAN></H3><OL><LI>Thaw the competent cells on ice
</LI><LI>Add 1 μL DNA (0.2 - 200 ng) to 50-100 μL competent cells
</LI><LI>Leave sample on ice for approximately 20 min
</LI><LI>Heat shock the cells for 90 s at 42 °C
</LI><LI>Add 500 μL of SOC to the sample
</LI><LI>Let the cells recover for 60 min at 37 °C, 220 rpm
</LI><LI> Plate appropriate amount of cell suspension (50 - 200 μL) on LB-agar-plates containing the appropriate antibiotic
</LI><LI>Let bacteria grow overnight at 37 °C
</LI></OL><P>According to <A href="http://www.qiagen.com/resources/molecular-biology-methods/dna/" class="external text" rel="nofollow">QIAGEN <I>DNA protocols and applications</I></A></P><H3><SPAN class="mw-headline" id="Plasmid_preparation">Plasmid preparation</SPAN></H3><P>Day before: Preparation of a preculture in LB-medium containing the appropriate antibiotics. The amount of cell culture required for plasmid preparation depends on the copy number of the plasmid (between 5 and 20 mL)
</P><OL><LI>Centrifuge the preculture in appropriate tubes for 10 min at 4000 g
</LI><LI>Carefully remove the supernatant
</LI><LI>Resuspend pellet in 200 μL resuspension solution
</LI><LI>Add 200 μL lysis solution and invert the tube gently 1 to 2 times
</LI><LI>Lysis should not exceed 5 min
</LI><LI>Add 350 μL neutralization solution and invert the tube 4 to 6 times 
</LI><LI>Centrifuge the suspension for 10 min at 12 000 rcf
</LI><LI>Prepare the columns by adding 500 μL of column preparation solution and centrifuging it for 1 min at 12 000 rcf. Discard the flow-through
</LI><LI>Transfer the supernatant of the centrifuged samples onto columns 
</LI><LI>Spin for 1 min at 12 000 rcf and subsequently discard the flow-through
</LI><LI>Add 750 μL wash solution and spin the loaded column for 1 min at 12 000 rcf. Discard the flow-through
</LI><LI>Dry the columns by centrifuging them for 1 min at 12 000 rcf
</LI><LI>Place columns in new collection tubes
</LI><LI>Elute the plasmid DNA with 50 μL ddH<SUB>2</SUB>O to increase the plasmid concentration
</LI></OL><P>The <A href="http://www.sigmaaldrich.com/life-science/molecular-biology/dna-and-rna-purification/plasmid-miniprep-kit.html" class="external text" rel="nofollow">Sigma-Aldrich GenElute™ Plasmid Miniprep Kit</A> was used
</P><H3><SPAN class="mw-headline" id="Preparation_of_samples_for_sequencing_at_Microsynth">Preparation of samples for sequencing at Microsynth</SPAN></H3><P>Add 12 μL DNA (60-100 ng/μL) to 3 μL of the corresponding primer (10 μM).
</P><H3><SPAN class="mw-headline" id="PCR_protocol_for_phusion_DNA_polymerase">PCR protocol for phusion DNA polymerase</SPAN></H3><TABLE class="wikitable"><TBODY><TR><TH> Components </TH><TH> 20 μL total reaction volume </TH><TH> 50 μL total reaction volume
</TH></TR><TR><TD> 5x Phusion HF buffer </TD><TD> 4 μL	 </TD><TD> 10 μL
</TD></TR><TR><TD> 10 mM dNTPs </TD><TD> 2 μL </TD><TD> 5 μL
</TD></TR><TR><TD> Forward Primer (10 μM) </TD><TD> 1 μL </TD><TD> 2.5 μL
</TD></TR><TR><TD> Reverse Primer (10 μM) </TD><TD> 1 μL </TD><TD> 2.5 μL
</TD></TR><TR><TD> DMSO  </TD><TD> 0.6 μL </TD><TD> 1.5 μL
</TD></TR><TR><TD> Phusion DNA polymerase </TD><TD> 0.2 μL </TD><TD>  0.5 μL
</TD></TR><TR><TD> DNA </TD><TD> 40-200 ng </TD><TD>  40-200 ng
</TD></TR><TR><TD> H<SUB>2</SUB>O </TD><TD> add to reach a total volume of 20 μL </TD><TD>  add to reach a total volume of 50 μL
</TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Restriction_Endonuclease_Reaction_.28double_digestion.29">Restriction Endonuclease Reaction (double digestion)</SPAN></H3><TABLE border="0"><TBODY><TR><TD align="left"> 1-2.5 μL restriction endonuclease 1
</TD></TR><TR><TD align="left"> 1-2.5 μL restriction endonuclease 2
</TD></TR><TR><TD align="left"> 5 μL Cut Smart Buffer
</TD></TR><TR><TD align="left"> 1-3 μg template DNA
</TD></TR><TR><TD align="left"> add H<SUB>2</SUB>O to reach a total volume of 50 μL
</TD></TR></TBODY></TABLE><P>Enzymes, buffers and protocol are from New England BioLabs
</P><H3><SPAN class="mw-headline" id="Dephosphorylation_of_5.E2.80.99-ends_of_DNA_using_CIP">Dephosphorylation of 5’-ends of DNA using CIP</SPAN></H3><P>Add 1 unit of CIP for every 1 pmol of DNA ends (about 1 μg of a 3 kb plasmid) and incubate at 37 °C for 30–60 min.
</P><P>Enzymes, buffers and protocol are from New England BioLabs
</P><H3><SPAN class="mw-headline" id="DNA_Purification_by_Centrifugation">DNA Purification by Centrifugation</SPAN></H3><H3><SPAN class="mw-headline" id="A._Dissolving_the_Gel_Slice">A. Dissolving the Gel Slice</SPAN></H3><P>Following electrophoresis, excise DNA band from gel and place gel slice in a 1.5 ml microcentrifuge tube.
Add 10 µl Membrane Binding Solution per 10 mg of gel slice. Vortex and incubate at 50–65 °C until gel slice is completely dissolved.
</P><P>or
</P><H3><SPAN class="mw-headline" id="B._Processing_PCR_Amplifications">B. Processing PCR Amplifications</SPAN></H3><P>Add an equal volume of Membrane Binding Solution to the PCR mix.
</P><OL><LI>Insert SV Minicolumn into Collection Tube.
</LI><LI>Transfer dissolved gel mixture or prepared PCR product to the Minicolumn assembly. Incubate at room temperature for 1 min.
</LI><LI>Centrifuge at 16 000 rcf for 1 min. Discard flowthrough and reinsert Minicolumn into Collection Tube.
</LI><LI>Add 700 µL Membrane Wash Solution (ethanol added). Centrifuge at 16 000 rcf for 1 min. Discard flowthrough and reinsert Minicolumn into Collection Tube.
</LI><LI>Repeat the step before with 500 µL Membrane Wash Solution. Centrifuge at 16 000 rcf for 5 min.
</LI><LI>Empty the Collection Tube and recentrifuge the column assembly for 1 min with the microcentrifuge lid off to allow evaporation of any residual ethanol.
</LI><LI>Carefully transfer Minicolumn to a clean 1.5 mL microcentrifuge tube.
</LI><LI>Add 50 µL of Nuclease-Free Water to the Minicolumn. Incubate at room temperature for 1 min. Centrifuge at 16 000 rcf for 1 min.
</LI><LI>Discard Minicolumn and store DNA at 4 °C or –20 °C.
</LI></OL><P>According to Promega Wizard SV Gel and PCR Clean-Up System, Quick protocol
</P><H3><SPAN class="mw-headline" id="Agarose_gel_electrophoresis">Agarose gel electrophoresis</SPAN></H3><OL><LI>For a 5 cm x 6 cm gel add 0.25 g Agarose to 25 mL TAE (0.5x) and heat up in microwave to dissolve it
</LI><LI>Let the solution cool down to approximately 50 °C
</LI><LI>Add nucleic acid dye (e.g. 2.5 μL peqGREEN) and mix
</LI><LI>Pour the solution in a tray using an appropriate comb
</LI><LI>Add loading dye (e.g. NEB purple loading dye) to the samples
</LI><LI>Fill the samples and ladder in the wells
</LI><LI>Run the gel at 135 V
</LI></OL><H3><SPAN class="mw-headline" id="Preparation_of_cryostocks">Preparation of cryostocks</SPAN></H3><OL><LI>Cultivate bacteria (37 °C) in medium with antibiotic (e.g. 5 mL LB, 5 μL amp + 500 μL preculture) until they are in log phase (OD=0.8-1.2) 
</LI><LI>Add 750 μL sterile glycerol (30%) to 750 μL bacteria culture in a screw top tube
</LI><LI>Freeze the glycerol stock tube at -80 °C
</LI></OL><H3><SPAN class="mw-headline" id="Site-directed_mutagenesis">Site-directed mutagenesis</SPAN></H3><TABLE><TBODY><TR><TD align="left"> 14 μL H<SUB>2</SUB>O
</TD></TR><TR><TD align="left"> 2 μL HF buffer
</TD></TR><TR><TD align="left"> 1.6 μL dNTPs
</TD></TR><TR><TD align="left"> 0.5 μL of primer 1
</TD></TR><TR><TD align="left"> 0.5 μL of primer 2
</TD></TR><TR><TD align="left"> 0.4 μL Phusion polymerase
</TD></TR><TR><TD align="left"> 1 μL template DNA (2-20 ng)
</TD></TR></TBODY></TABLE><OL><LI>Run PCR
</LI><LI>Digestion of template DNA: Addition of 1 μL DpnI, 1h at 37 °C 
</LI><LI>Heat inactivation of DpnI: 20 min at 80 °C
</LI></OL><P>Protocol based on <A href="https://2014.igem.org/wiki/index.php?title=Team:ETH_Zurich/lab/protocols&amp;action=edit&amp;section=19" class="external text" rel="nofollow">QuikChange Site-Directed Mutagenesis</A></P><H3><SPAN class="mw-headline" id="Gibson_Assembly">Gibson Assembly</SPAN></H3><P>One-step isothermal DNA assembly protocol: the exonuclease amount is ideal for the assembly of DNA molecules with 20–150 bp overlaps
</P><OL><LI> Mix the backbone and PCR fragments in 5 µL total volume in equimolar amounts
</LI><LI> Thaw the Gibson assembly reaction mixture on ice
</LI><LI> Add DNA mixture (5 µL) to the reaction mixture (15 µL)
</LI><LI> Run the reaction for 30-60 min at 50 °C
</LI><LI> Subsequently the reaction mixture (5 uL are enough) can be used directly to transform competent cells (75 uL)
</LI></OL><H3><SPAN class="mw-headline" id="Multi_site_directed_mutagenesis">Multi site directed mutagenesis</SPAN></H3><P>Single Primer Method
</P><P>This protocol only uses one oligo per mutation site (cuts the oligo cost in half) and can mutate more than one site at a time. While this protocol is less reliable than the previous protocol (only ~25% of colonies will contain the desired mutation(s)), if you're doing multiple mutations on the same piece of DNA than this can save you many days of work by just sequencing a few more colonies. In this protocol there is an initial Kinase (phosphorylation) reaction to allow your primers to be available for ligation. The polymerase will extend from one primer to another and then the Taq ligase will seal the nick. This allows multiple mutations to be done at the same time.
</P><P>Method
</P><OL><LI>Design mutagenesis primer(s). The targeted mutation should be in the middle of the primer. Design your primers (including the mutations) to have a Tm &gt;=78°C.
</LI><LI>Purify template plasmid from a dam+ <I>E. coli</I> strain via miniprep.
</LI><LI>Set up mutagenesis PCR mix:
</LI></OL><P>
16 μL Water

1.5 μL DMSO(100%)

0.5 μL MgSO4(50 µM)

10 μL Phusion HF Buffer

5 μL T4 Ligase Buffer

5 μL 10x Taq Ligase Buffer

0.5 μL Each Mutagenesis Primer (at 40 µM)

2 μL Reverse Biobrick Primer

2 μL dNTPs (100mM)

2 μL Template

2 μL PNK

2 μL Taq Ligase

1 μL Phusion Polymerase (hotstart)
</P><OL><LI>Run PCR
</LI></OL><UL><LI>37°C for 30 minutes (this is the kinase step)
</LI><LI>95°C for 3 minutes
</LI><LI>Run the following for 20 cycles:
</LI><LI>95°C for 1 minute
</LI><LI>55°C for 1 minute
</LI><LI>65°C for 30 sec/kb of plasmid length minimum
</LI><LI>4°C infinite
</LI></UL><OL><LI>Add 1μL DpnI restriction enzyme to the PCR tube directly. (Purification is not necessary)
</LI><LI>Incubate 4-6 hours at 37°C.
</LI><LI>Purify PCR product and elute into 30μL.
</LI><LI>Transform 3μL purified DNA into highly competent cells.
</LI><LI>Screen the transformants for the desired mutation using restriction digest or sequencing.
</LI></OL><P>
Notes
Because the inital kinase step could lead to non-specific binding, it would be best to use a hot-start polymerase for this procedure.
Any combination of DMSO and MgSO4 has shown significant improvments in product.
</P><P>According to <A href="http://openwetware.org/wiki/Richard_Lab:Site_Directed_Mutagenesis" class="external text" rel="nofollow">Multi Site Directed Mutagenesis, Single Primer Method</A></P><H3><SPAN class="mw-headline" id="Preparation_of_alginate_beads">Preparation of alginate beads</SPAN></H3><OL><LI>Harvest bacteria in exponential phase (OD between 1.5 and 2.0) 
</LI><LI>Centrifuge the culture for 10 min at 4000 rcf
</LI><LI>Resuspend and dilute the pellet in sterile NaCl solution (0.9%)
</LI><LI>Add bacteria-NaCl-suspension to sterile alginate (2.5%) so as to reach an alginate concentration of 2%
</LI><LI>Fill the suspension into a sterile syringe 
</LI><LI>Use a suitable device for uniform droplet formation
</LI><LI>Collect the droplets in a beaker containing CaCl<SUB>2</SUB> (100mM) to allow electrostatic cross-linking within the alginate bead
</LI><LI>Store the beads in CaCl<SUB>2</SUB> (10mM)
</LI></OL><P>
1 bead corresponds to approximatly 14.5 µl alginate-NaCl
</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:ETH_Zurich/lab/protocols">http://2014.igem.org/Team:ETH_Zurich/lab/protocols</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:ETH_Zurich/lab/protocols" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:ETH_Zurich/lab/protocols" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:ETH_Zurich/lab/protocols&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:ETH_Zurich/lab/protocols&amp;oldid=370215" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></DIV></BODY></HTML>