<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_ETH_Zurich_expresults_integrases"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:ETH_Zurich/expresults/integrases">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:ETH_Zurich/expresults/integrases&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:ETH_Zurich/expresults/integrases&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:ETH_Zurich/expresults/integrases&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:ETH_Zurich/expresults/integrases" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:ETH Zurich/expresults/integrases</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><H2><SPAN class="mw-headline" id="Integrases"> Integrases </SPAN></H2><P>The design of our <A href="https://2014.igem.org/Team:ETH_Zurich/modeling/xor" class="external text" rel="nofollow">XOR gates</A> was based on <A href="https://2014.igem.org/Team:ETH_Zurich/modeling/xor#Biological_Principles" class="external text" rel="nofollow">integrase logic</A><SUP><A href="/Team:ETH_Zurich/project/references#refBonnet" title="Team:ETH Zurich/project/references">[9]</A></SUP>. This means, depending on the input molecules, integrases can be expressed, subsequently switch a terminator sequence previously blocking gene expression, and then the output gene can be transcribed. This approach is explained <A href="https://2014.igem.org/Team:ETH_Zurich/modeling/xor#Biological_Principles" class="external text" rel="nofollow">here</A>.
</P><P>
In order to characterize the integrase system described above, we first combined the <A href="http://parts.igem.org/Part:BBa_R0062" class="external text" rel="nofollow">pLux promoter (BBa_R0062)</A> with one of our <A href="https://2014.igem.org/Team:ETH_Zurich/modeling/int" class="external text" rel="nofollow">integrase</A> genes <I>bxb1</I>, followed directly by a red fluorescent protein (RFP, mCherry) to make the expression accessible. Also, this system includes an XOR buffer gate per default blocking transcription of sfGFP. Upon BXB1 activity and switching the gate into ON-state, the terminator should have been removed and sfGFP should have been expressed. We intially designed three different constructs for characterization of the recombinases and their cross-activity. However, the measurement of fluorescent proteins, with both a <A href="https://2014.igem.org/Team:ETH_Zurich/lab/protocols#Tecan_Infinite_M200_Pro.E2.84.A2" class="external text" rel="nofollow">plate reader</A> and a <A href="https://2014.igem.org/Team:ETH_Zurich/lab/protocols#BD_LSRFortessa.E2.84.A2_Flow_Cytometer_System" class="external text" rel="nofollow">flow cytometer</A>, did not indicate sfGFP expression due to recombinase activity. Nevertheless, RFP was clearly detectable upon induction in plate reader experiments suggesting that the induction itself worked (see figure 5 and 6).
</P><TABLE class="wikitable resized" style="background-color: white; border: 0px+important;"><TBODY><TR><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><B>Figure 5</B><B>Expression of red fluorescent protein (RFP) and green fluorescent protein (GFP) 250 min after induction with various concentrations of 3OC6-HSL (10<SUP>-13</SUP> M to 10<SUP>-4</SUP> M).</B> RFP is under the control of the <A href="http://parts.igem.org/Part:BBa_R0062" class="external text" rel="nofollow">pLux promoter (BBa_R0062)</A> together with the integrase BXB1. Upon expression of BXB1 a buffer gate should have been swtiched to ON-state producing GFP. Data points are mean values of triplicate measurements in 96-well microtiter plates ± standard deviation.  For the full data set and kinetics please <A href="https://2014.igem.org/Team:ETH_Zurich/contact" class="external text" rel="nofollow">contact</A> us or visit the <A href="https://2014.igem.org/Team:ETH_Zurich/data/raw" class="external text" rel="nofollow">raw data</A> page.</DIV></DIV></DIV></DIV></TD><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><B>Figure 6</B><B>Expression of red fluorescent protein (RFP) and green fluorescent protein (GFP) 610 min after induction with various concentrations of 3OC6-HSL (10<SUP>-13</SUP> M to 10<SUP>-4</SUP> M).</B> RFP is under the control of the <A href="http://parts.igem.org/Part:BBa_R0062" class="external text" rel="nofollow">pLux promoter (BBa_R0062)</A> together with the integrase BXB1. Upon expression of BXB1 a buffer gate should have been swtiched to ON-state producing GFP. Data points are mean values of triplicate measurements in 96-well microtiter plates ± standard deviation. For the full data set and kinetics please <A href="https://2014.igem.org/Team:ETH_Zurich/contact" class="external text" rel="nofollow">contact</A> us or visit the <A href="https://2014.igem.org/Team:ETH_Zurich/data/raw" class="external text" rel="nofollow">raw data</A> page.</DIV></DIV></DIV></DIV></TD></TR></TBODY></TABLE><P>
As our constructs did not show the expected functionality, we decided to directly use the plasmids described by Bonnet <I>et al</I>.<SUP><A href="/Team:ETH_Zurich/project/references#refBonnet" title="Team:ETH Zurich/project/references">[9]</A></SUP> which where obtained from addgene (<A href="http://www.addgene.org/44456" class="external text" rel="nofollow">Dual-recombinase-controller</A>, <A href="http://www.addgene.org/44453/" class="external text" rel="nofollow">XOR gate-V2.0</A>). The data available from the original experiment by Bonnet<SUP><A href="/Team:ETH_Zurich/project/references#refBonnet" title="Team:ETH Zurich/project/references">[9]</A></SUP> was used in our model to retrieve the <A href="https://2014.igem.org/Team:ETH_Zurich/modeling/int" class="external text" rel="nofollow">missing parameters of integrases</A>. However, we were using a <A href="http://www.openwetware.org/wiki/E._coli_genotypes#TOP10_.28Invitrogen.29" class="external text" rel="nofollow">TOP10</A> strain not expressing TetR by default (as compared to <A href="http://www.expressys.com/main_strains.html" class="external text" rel="nofollow">DH5alphaZ1</A>) and as a result our strain had to be co-transformed with an additional plasmid encoding TetR. Also, we used defined <A href="https://2014.igem.org/Team:ETH_Zurich/lab/protocols#Defined_M9_medium_with_0.4_.25_glycerol" class="external text" rel="nofollow">M9 medium</A> with 0.4% glycerol and 1% CAA instead of proprietary defined medium (<A href="http://teknova-blog.com/hi-def-azure-media" class="external text" rel="nofollow">Teknova Hi-Def Azure medium</A>). As of today, this set-up did not allow us to get the integrase XOR gate running. The fluorescence readout for ON-states (exactly one input, either 0.1% L-arabinose or 200 ng/mL anhydrous tetracycline) and the OFF-states was not as expected (see figure 7). The OFF-state should not show increased fluorescence over time, while the ON-states should increase significantly after 4 h and continue to increase over the whole time span<SUP><A href="/Team:ETH_Zurich/project/references#refBonnet" title="Team:ETH Zurich/project/references">[9]</A></SUP>. We are not giving up on this and are proceeding with debugging our construct further and hope to find a solution until the Giant Jamboree in Boston.
</P><P><CENTER><SVG xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" version="1.1" id="Layer_1" x="50%" y="0px" width="80" height="80" viewbox="0 0 100 100" enable-background="0 0 1000 1000" xml:space="preserve"><A href="#top" id="green" class="circled scrolly" style="text-decoration:none;"><TEXT x="50" y="50" style="cursor: pointer" fill="#FFFFFF" font-size="15" text-anchor="middle">top</TEXT></A></SVG></CENTER></P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><B>Figure 7 Expression of green fluorescent protein (GFP) after induction of <A href="https://2014.igem.org/Team:ETH_Zurich/modeling/xor#Biological_Principles" class="external text" rel="nofollow">XOR integrase logic</A>.</B> Upon induction of integrase BXB1 (t=0 min) a <A href="https://2014.igem.org/Team:ETH_Zurich/lab/sequences#BUFFER_Gate_Construct" class="external text" rel="nofollow">buffer gate</A> should have been switched to ON-state producing GFP only if either L-arabinose (L-ara) or anhydrous tetracycline (aTc) is present. Data points are mean values of triplicate measurements in 96-well microtiter plates ± standard deviation. For the full data set and kinetics please <A href="https://2014.igem.org/Team:ETH_Zurich/contact" class="external text" rel="nofollow">contact</A> us or visit the <A href="https://2014.igem.org/Team:ETH_Zurich/data/raw" class="external text" rel="nofollow">raw data</A> page.</DIV></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:ETH_Zurich/expresults/integrases">http://2014.igem.org/Team:ETH_Zurich/expresults/integrases</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:ETH_Zurich/expresults/integrases" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:ETH_Zurich/expresults/integrases" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:ETH_Zurich/expresults/integrases&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:ETH_Zurich/expresults/integrases&amp;oldid=381349" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>