<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_ETH_Zurich_expresults"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:ETH_Zurich/expresults">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:ETH_Zurich/expresults&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:ETH_Zurich/expresults&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:ETH_Zurich/expresults&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:ETH_Zurich/expresults" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:ETH Zurich/expresults</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><TITLE>iGEM ETH Zurich 2014</TITLE><DIV id="header"><NAV id="nav"><UL><LI><A href="/Team:ETH_Zurich" title="Team:ETH Zurich">Home</A></LI><LI><A name="Project" style="cursor: default" href="javascript:void(0)">Project</A></LI><LI><A href="/Team:ETH_Zurich/project/overview" title="Team:ETH Zurich/project/overview">Overview</A></LI><LI><A href="/Team:ETH_Zurich/project/infopro" title="Team:ETH Zurich/project/infopro">Information Processing</A></LI><LI><A href="/Team:ETH_Zurich/project/background" title="Team:ETH Zurich/project/background">Background</A></LI><LI><A href="/Team:ETH_Zurich/project/goals" title="Team:ETH Zurich/project/goals">Goals</A></LI><LI><A href="/Team:ETH_Zurich/project/applications" title="Team:ETH Zurich/project/applications">Applications</A></LI><LI><A href="/Team:ETH_Zurich/project/references" title="Team:ETH Zurich/project/references">References</A></LI><LI><A href="/Team:ETH_Zurich/blog" title="Team:ETH Zurich/blog">Blog</A></LI></UL><LI><A name="Achievements" style="cursor: default" href="javascript:void(0)">Achievements</A></LI><LI><A href="/Team:ETH_Zurich/achievements" title="Team:ETH Zurich/achievements">Achievements</A></LI><LI><A href="/Team:ETH_Zurich/criteria" title="Team:ETH Zurich/criteria">Medal Criteria</A></LI><LI><STRONG class="selflink">Results</STRONG></LI><LI><A name="Data page" style="cursor: default" href="javascript:void(0)">Data page</A></LI><LI><A href="/Team:ETH_Zurich/data#Gene_Circuit" title="Team:ETH Zurich/data">Gene Circuit and Parts</A></LI><LI><A href="/Team:ETH_Zurich/data#Used_and_Characterized_Pre-Existing_Parts" title="Team:ETH Zurich/data">Used and Characterized Pre-Existing Parts</A></LI><LI><A href="/Team:ETH_Zurich/data#Our_Favorite_New_Characterized_Parts" title="Team:ETH Zurich/data">Our Favorite New Characterized Parts</A></LI><LI><A href="/Team:ETH_Zurich/data/raw" title="Team:ETH Zurich/data/raw">Raw Data</A></LI><LI><A name="Modeling" style="cursor: default" href="javascript:void(0)">Modeling</A></LI><LI><A href="/Team:ETH_Zurich/modeling/overview" title="Team:ETH Zurich/modeling/overview">Overview</A></LI><LI><A href="/Team:ETH_Zurich/modeling/qs" title="Team:ETH Zurich/modeling/qs">Quorum Sensing</A></LI><LI><A href="/Team:ETH_Zurich/modeling/int" title="Team:ETH Zurich/modeling/int">Integrases</A></LI><LI><A href="/Team:ETH_Zurich/modeling/xor" title="Team:ETH Zurich/modeling/xor">XOR Gate</A></LI><LI><A href="/Team:ETH_Zurich/modeling/whole" title="Team:ETH Zurich/modeling/whole">Whole cell model</A></LI><LI><A href="/Team:ETH_Zurich/modeling/diffmodel" title="Team:ETH Zurich/modeling/diffmodel">Diffusion</A></LI><LI><A href="/Team:ETH_Zurich/modeling/parameters" title="Team:ETH Zurich/modeling/parameters">Parameters and Tools</A></LI><LI><A name="Lab" style="cursor: default" href="javascript:void(0)">Lab</A></LI><LI><A href="/Team:ETH_Zurich/lab/overview" title="Team:ETH Zurich/lab/overview">Overview</A></LI><LI><A href="/Team:ETH_Zurich/lab/protocols" title="Team:ETH Zurich/lab/protocols">Materials &amp; Methods</A></LI><LI><A href="/Team:ETH_Zurich/lab/sequences" title="Team:ETH Zurich/lab/sequences">Sequences</A></LI><LI><A href="/Team:ETH_Zurich/lab/bead" title="Team:ETH Zurich/lab/bead">Alginate Beads</A></LI><LI><A href="/Team:ETH_Zurich/lab/chip" title="Team:ETH Zurich/lab/chip">Millifluidic Chip</A></LI><LI><A href="/Team:ETH_Zurich/lab/safety" title="Team:ETH Zurich/lab/safety">Safety</A></LI><LI><A name="Human Practice" style="cursor: default" href="javascript:void(0)">Human practice</A></LI><LI><A href="/Team:ETH_Zurich/human/overview" title="Team:ETH Zurich/human/overview">Overview</A></LI><LI><A href="/Team:ETH_Zurich/human/survey" title="Team:ETH Zurich/human/survey">Survey</A></LI><LI><A href="/Team:ETH_Zurich/human/interviews" title="Team:ETH Zurich/human/interviews">Interviews</A></LI><LI><A href="/Team:ETH_Zurich/human/outreach" title="Team:ETH Zurich/human/outreach">Outreach</A></LI><LI><A href="/Team:ETH_Zurich/human/essay" title="Team:ETH Zurich/human/essay">Our insights</A></LI><LI><A name="Team" style="cursor: default" href="javascript:void(0)">Team</A></LI><LI><A href="https://igem.org/Team.cgi?year=2014&amp;team_name=ETH_Zurich" class="external text" rel="nofollow">Official Team Profile</A></LI><LI><A href="/Team:ETH_Zurich/team" title="Team:ETH Zurich/team">Our team</A></LI><LI><A href="/Team:ETH_Zurich/adandin" title="Team:ETH Zurich/adandin">Advisors and Instructors</A></LI><LI><A href="/Team:ETH_Zurich/collaborations" title="Team:ETH Zurich/collaborations">Collaborations</A></LI><LI><A href="/Team:ETH_Zurich/acknowledgements" title="Team:ETH Zurich/acknowledgements">Acknowledgements</A></LI><LI><A href="/Team:ETH_Zurich/contact" title="Team:ETH Zurich/contact">Contacting us</A></LI></NAV><DIV class="inner1"><HEADER><H1>  Results  </H1></HEADER><CENTER><P><SVG xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" version="1.1" id="Layer_1" x="50%" y="0px" width="150" height="150" viewbox="0 0 100 100" enable-background="0 0 1000 1000" xml:space="preserve"><A href="#Diffusion" id="green" class="circled scrolly" style="text-decoration:none;"><TEXT x="56" y="53" style="cursor: pointer" fill="#FFFFFF" font-size="15" text-anchor="middle"><TSPAN y="43" x="50">Diffusion</TSPAN><TSPAN x="50" dy="20">on Chip</TSPAN></TEXT></A></SVG><SVG xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" version="1.1" id="Layer_1" x="50%" y="0px" width="150" height="150" viewbox="0 0 100 100" enable-background="0 0 1000 1000" xml:space="preserve"><A href="#Quorum" id="blue" class="circled scrolly" style="text-decoration:none;"><TEXT x="56" y="53" style="cursor: pointer" fill="#FFFFFF" font-size="15" text-anchor="middle"><TSPAN y="43" x="50">Quorum</TSPAN><TSPAN x="50" dy="20">Sensing</TSPAN></TEXT></A></SVG><SVG xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" version="1.1" id="Layer_1" x="50%" y="0px" width="150" height="150" viewbox="0 0 100 100" enable-background="0 0 1000 1000" xml:space="preserve"><A href="#Ribo-" id="red" class="circled scrolly" style="text-decoration:none;"><TEXT x="56" y="53" style="cursor: pointer" fill="#FFFFFF" font-size="15" text-anchor="middle"><TSPAN y="43" x="50">Ribo-</TSPAN><TSPAN x="50" dy="20">regulators</TSPAN></TEXT></A></SVG><SVG xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" version="1.1" id="Layer_1" x="50%" y="0px" width="150" height="150" viewbox="0 0 100 100" enable-background="0 0 1000 1000" xml:space="preserve"><A href="#Integrases" id="green" class="circled scrolly" style="text-decoration:none;"><TEXT x="50" y="53" style="cursor: pointer" fill="#FFFFFF" font-size="15" text-anchor="middle">Integrases</TEXT></A></SVG></P></CENTER><H2><SPAN class="mw-headline" id="Diffusion_On_Chip"> Diffusion On Chip </SPAN></H2><P>Our project aims for the biological implementation of <A href="https://2014.igem.org/Team:ETH_Zurich/project/background/modeling#Cellular_Automata" class="external text" rel="nofollow">cellular automata</A> with <A href="https://2014.igem.org/Team:ETH_Zurich/modeling/xor#XOR_Logic_Gate" class="external text" rel="nofollow">XOR</A> logic gates. In order to achieve this, we found a way to create a regular grid of cells with a defined, optimal neighborhood. This means channel length, well size, and medium were optimized and the properties were modelled with Matlab and Comsol whenever feasible. With these <A href="https://2014.igem.org/Team:ETH_Zurich/modeling/diffmodel" class="external text" rel="nofollow"><I>in silico</I> results</A> in mind we used CAD software to design our custom made molds, which where then 3D-printed and used for the production of <A href="https://2014.igem.org/Team:ETH_Zurich/lab/chip#PDMS_Chip_Preparation" class="external text" rel="nofollow">PDMS chips</A>. The cells containing one of our genetic circuits were encapsulated in <A href="https://2014.igem.org/Team:ETH_Zurich/lab/bead" class="external text" rel="nofollow">alginate beads</A> and loaded on the <A href="https://2014.igem.org/Team:ETH_Zurich/lab/chip" class="external text" rel="nofollow">millifluidic chip</A> arranged in a Sierpinski triangle. All other wells were filled with beads containing cells not able to produce GFP as a background reference. This approach allowed us to establish a method for measuring diffusion and cell-to-cell communication. In particular, a step towards the emergence of complex patterns by cell-to-cell communication was made. Also the <A href="https://2014.igem.org/Team:ETH_Zurich/modeling/diffmodel#Pattern_developing" class="external text" rel="nofollow">Comsol model</A> regarding pattern formation was confirmed experimentally with our rapid-prototyping approach. The final time-lapse video of the  <A href="https://2014.igem.org/Team:ETH_Zurich/project/background/biotools#Quorum_Sensing" class="external text" rel="nofollow">cell-to-cell communication</A> experiment is shown below in video 1.
</P><TABLE class="wikitable" style="background-color: white; text-align:center; width:auto; margin: auto; font-size:10pt;"><TBODY><TR><TD colspan="2"><B>Video 1</B><B>Row wise, self-propagating <A href="https://2014.igem.org/Team:ETH_Zurich/project/background#Biotools" class="external text" rel="nofollow">cell-to-cell communication</A> of <I>E. coli</I> cells confined in <A href="https://2014.igem.org/Team:ETH_Zurich/lab/bead" class="external text" rel="nofollow">alginate beads</A> (d=3 mm, initially 10<SUP>7</SUP> cells/bead) on a <A href="https://2014.igem.org/Team:ETH_Zurich/lab/chip" class="external text" rel="nofollow">custom-made millifluidic PDMS chip</A>.</B></TD></TR><TR><TD style="width:50%"> All cells contained <A href="https://2014.igem.org/Team:ETH_Zurich/expresults/rr#Riboregulators" class="external text" rel="nofollow">riboregulated</A> sfGFP followed by <A href="http://parts.igem.org/Part:BBa_C0161" class="external text" rel="nofollow">LuxI (BBa_C0161)</A> together under the control of the <A href="http://parts.igem.org/Part:BBa_R0062" class="external text" rel="nofollow">pLux promoter (BBa_R0062)</A>, and <A href="http://parts.igem.org/Part:BBa_J23100" class="external text" rel="nofollow">constitutively (BBa_J23100)</A> expressed <A href="http://parts.igem.org/Part:BBa_C0062" class="external text" rel="nofollow">LuxR (BBa_C0062)</A>. LuxI catalyzes the production of the autoinducer 3OC6-HSL, which is then diffusing from cell to cell. For initialization, the cells in one bead of the top row were induced with 3OC6-HSL before encapsulation. Imaging was implemented with a <A href="https://2014.igem.org/Team:ETH_Zurich/lab/protocols#Biostep_Dark-Hood_DH-50.E2.84.A2__and_the_Argus-X1.E2.84.A2_software" class="external text" rel="nofollow">Biostep Dark-Hood DH-50 (Argus X1 software)</A> fitted with a Canon EOS 500D DSLR camera and a fluorescence filter (545 nm filter). Pictures were usually taken every 2 min at an excitation wavelength of 470 nm with the standard Canon EOS Utility software. Time-lapse movies were created with Adobe After Effects CC software. 1950x faster than real-time, the video shown starts 10 h after the initiation of the experiment (however the time scale shown corresponds to minutes after loading of the chip)
</TD><TD>Simulation of the propagation of the pattern in the millifluidic chip. <A href="http://www.comsol.com/comsol-multiphysics" class="external text" rel="nofollow">Comsol Multiphysics Simulation software</A> was used in order to simulate a detailed diffusion model including quorum sensing steps in colonies and cell growth. Overall GFP concentration in beads has been scaled in order to account for the high background of the experimental setup. Green Fluorescence Protein is produced earlier in the wells, but can be seen only above a certain threshold.Accurate prediction of experimental data by the model has been achieved, with parameters from our own fittings or from the literature. Experimental observation combined with simulation enables to show that a pattern is able to develop in the millifluidic chip in a reasonable time scale. For precise equations and other dynamic results, check the <A href="https://2014.igem.org/Team:ETH_Zurich/modeling/diffmodel" class="external text" rel="nofollow">Diffusion model</A> page.
</TD></TR></TBODY></TABLE><P><CENTER><SVG xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" version="1.1" id="Layer_1" x="50%" y="0px" width="80" height="80" viewbox="0 0 100 100" enable-background="0 0 1000 1000" xml:space="preserve"><A href="#top" id="green" class="circled scrolly" style="text-decoration:none;"><TEXT x="50" y="50" style="cursor: pointer" fill="#FFFFFF" font-size="15" text-anchor="middle">top</TEXT></A></SVG></CENTER></P><H2><SPAN class="mw-headline" id="Quorum_Sensing"> Quorum Sensing </SPAN></H2><P>For our Mosia<I>coli</I> project, we were looking for molecular systems that allow orthogonal <A href="https://2014.igem.org/Team:ETH_Zurich/project/background#Quorum_Sensing" class="external text" rel="nofollow">cell-to-cell communication</A> in order to implement connected <A href="https://2014.igem.org/Team:ETH_Zurich/modeling/xor#XOR_Logic_Gate" class="external text" rel="nofollow">XOR logic gates</A>. We decided to exploit the quorum sensing systems <A href="https://2014.igem.org/Team:ETH_Zurich/data#Gene_Circuit" class="external text" rel="nofollow">LuxI/LuxR, LasI/LasR, and RhlI/RhlR</A> in order to achieve the required orthogonal cell-to-cell communication. We developed a <A href="https://2014.igem.org/Team:ETH_Zurich/modeling/qs" class="external text" rel="nofollow">model</A> for these cellular information processing. Even though the corresponding inducer molecules are commercially available and the systems often used, in particular in iGEM projects (e.g. <A href="http://parts.igem.org/Part:BBa_R0062" class="external text" rel="nofollow">pLux (BBa_R0062)</A>, '<A href="http://parts.igem.org/Frequently_Used_Parts" class="external text" rel="nofollow">Top 10 Most used promoters</A>' with 246 uses), potential crosstalk activity between the different systems may be a severe problem (e. g. <A href="https://2013.igem.org/Team:Tokyo_Tech/Project/Ninja_State_Switching" class="external text" rel="nofollow">Tokyo_Tech 2013</A>, <A href="https://2011.igem.org/Team:Peking_S/project/wire/matrix" class="external text" rel="nofollow">Peking University 2011</A>).
</P><P>
In order to address this challenge, we measured a) a given promoter with its corresponding regulator and a different inducer molecule, b) a given promoter with an unspecific regulator and a particular inducer, c) a given promoter with both regulator and inducer being unspecific, and always included the correct combination of inducer molecule, regulator and promoter as a positive control. This gives in total 27 possible combinations. The output was assessed via sfGFP and measured in terms of fluorescence on microtiter-plate scale.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:735px;"><DIV class="thumbcaption"><B>Figure 1</B> Each quorum sensing system is based on three components: a signaling molecule, a regulatory protein and a promoter. These elements are here ordered into three layers. Cross-talk evaluation can be done by comparing all combinations of those three elements. After collecting the <A href="https://2014.igem.org/Team:ETH_Zurich/expresults" class="external text" rel="nofollow">experimental data</A> of all possible pathways, we modeled their influence.</DIV></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Summary_of_experimental_results_regarding_quorum_sensing">Summary of experimental results regarding quorum sensing</SPAN></H3><P>The following matrices serve as an overview summarizing the most significant results of our experiments to characterize crosstalk on different levels. 
On the horizontal top row we see the three different inducer molecules (3OC12-HSL, 3OC6-HSL, C4-HSL). In the top left corner we see the quorum sensing promoter used for all the experiments summarized in this matrix. On the vertical axis we see the three regulators ( <A href="https://2014.igem.org/Team:ETH_Zurich/data#Used_and_Characterized_Pre-Existing_Parts" class="external text" rel="nofollow">LuxR, LasR, RhlR</A>). 
These matrices are giving an overview of the experimental results conducted in relation with <A href="https://2014.igem.org/Team:ETH_Zurich/project/background/biotools#Quorum_Sensing" class="external text" rel="nofollow">quorum sensing</A> and crosstalk. The graph shown in each matrix on the very top left describes the situation where the correct autoinducer molecule has bound the corresponding regulator and this complex has then induced the correct promoter. 
The solid lines in the graphs show the <A href="https://2014.igem.org/Team:ETH_Zurich/modeling/qs" class="external text" rel="nofollow">model data</A>, whereas the data points indicated with standard deviation show experimental data in triplicates (mean values of triplicate micro titerplate measurements). 
</P><TABLE class="wikitable" style="background-color: white; text-align:center; width:auto; margin: auto;"><CAPTION><B>Table 1</B> Crosstalk matrix for the promoter pLux (<A href="http://parts.igem.org/Part:BBa_R0062:Experience" class="external text" rel="nofollow">BBa_R0062</A>)
</CAPTION><TBODY><TR><TD colspan="4"><P>In all the measurements conducted to create this matrix the <A href="http://parts.igem.org/Part:BBa_R0062" class="external text" rel="nofollow">promoter pLux</A> was the basis and was induced in six different variations shown. 
The dark blue points in the graph top left show the activation of gene expression when <A href="http://parts.igem.org/Part:BBa_R0062" class="external text" rel="nofollow">pLux</A> is induced by 3OC6-HSL (Lux-AHL) binding to the corresponding <A href="http://parts.igem.org/Part:BBa_C0062" class="external text" rel="nofollow">LuxR regulator</A>. The observed transition occurs at a concentration of approximately 1 nM of 3OC6-HSL. The light-blue curve plotted shows modeling data of <A href="http://parts.igem.org/Part:BBa_R0062" class="external text" rel="nofollow">pLux</A> induced by 3OC6-HSL (Lux-AHL) binding to the corresponding <A href="http://parts.igem.org/Part:BBa_C0062" class="external text" rel="nofollow">LuxR regulator</A>. This curve from the model and the dark blue data points obtained from experiments were plotted as a reference in all the other graphs describing <A href="http://parts.igem.org/Part:BBa_R0062" class="external text" rel="nofollow">pLux</A>. Crosstalk can be observed for the cases where the 3OC12-HSL (Las-AHL) binds the <A href="http://parts.igem.org/Part:BBa_C0062" class="external text" rel="nofollow">LuxR regulator</A>. 
Additionally for 3OC12-HSL binding to its corresponding <A href="http://parts.igem.org/Part:BBa_C0179" class="external text" rel="nofollow">regulator LasR</A> and then binding to the <A href="http://parts.igem.org/Part:BBa_R0062" class="external text" rel="nofollow">pLux</A> as seen in the middle of the top row and center of the matrix. 
For the case of Las-AHL binding the <A href="http://parts.igem.org/Part:BBa_C0179" class="external text" rel="nofollow">regulator LasR</A> and subsequently the <A href="http://parts.igem.org/Part:BBa_R0062" class="external text" rel="nofollow">promoter pLux</A>, the transition occurs at 1 nM and reaches 0.5 fold the fluorescence as <A href="http://parts.igem.org/Part:BBa_R0062" class="external text" rel="nofollow">pLux</A> induced by 3OC6-HSL binding <A href="http://parts.igem.org/Part:BBa_C0062" class="external text" rel="nofollow">LuxR</A>. In the case of 3OC12-HSL binding <A href="http://parts.igem.org/Part:BBa_C0062" class="external text" rel="nofollow">LuxR</A> and inducing the promoter <A href="http://parts.igem.org/Part:BBa_R0062" class="external text" rel="nofollow">pLux</A>, the transition is observed at approximately 100 nM and severe crosstalk is observed, meaning that the ON-OFF-ratio is not significantly different from the reference curve. 
</P><P>Observation of C4-HSL has shown, that there is no significant crosstalk with the <A href="http://parts.igem.org/Part:BBa_C0062" class="external text" rel="nofollow">LuxR regulator</A> and <A href="http://parts.igem.org/Part:BBa_C0179" class="external text" rel="nofollow">LasR regulator</A> binding C4-HSL and subsequently to <A href="http://parts.igem.org/Part:BBa_R0062" class="external text" rel="nofollow">pLux</A>. This is indicated on top right and middle right graphs. However, <A href="http://parts.igem.org/Part:BBa_C0171" class="external text" rel="nofollow">RhlR</A> induced with its corresponding inducer (C4-HSL) binds to <A href="http://parts.igem.org/Part:BBa_R0062" class="external text" rel="nofollow">pLux</A> and activates expression of GFP at about 100 nM. 
</P></TD></TR></TBODY></TABLE><TABLE class="wikitable" style="background-color: white; text-align:center; width:auto; margin: auto;"><CAPTION><B>Table 2</B> Crosstalk matrix for the promoter plas (<A href="http://parts.igem.org/Part:BBa_R0079:Experience" class="external text" rel="nofollow">BBa_R0079</A>)

</CAPTION><TBODY><TR><TD colspan="4"><P>The promoter of interest in this matrix is <A href="http://parts.igem.org/Part:BBa_R0079" class="external text" rel="nofollow">pLas</A>. The graph on top left corner shows the induction of <A href="http://parts.igem.org/Part:BBa_R0079" class="external text" rel="nofollow">pLas</A> by its corresponding inducer (3OC12-HSL) binding the corresponding <A href="http://parts.igem.org/Part:BBa_C0179" class="external text" rel="nofollow">LasR</A>. The red line shows the model whereas the datapoints shown in red represent the experimental results. The transition can be observed at a concentration of Las-AHL of about 2 nM. 3OC6-HSL binding <A href="http://parts.igem.org/Part:BBa_C0171" class="external text" rel="nofollow">RhlR</A> does not induce the <A href="http://parts.igem.org/Part:BBa_R0079" class="external text" rel="nofollow">pLas</A>. For the binding of 3OC12-HSL to <A href="http://parts.igem.org/Part:BBa_C0171" class="external text" rel="nofollow">RhlR</A> a minor increase of fluorescence can be observed. The same can be observed for 3OC12-HSL binding to the <A href="http://parts.igem.org/Part:BBa_C0062" class="external text" rel="nofollow">LuxR</A> as this combination is to a small degree inducing <A href="http://parts.igem.org/Part:BBa_R0079" class="external text" rel="nofollow">pLas</A>. The most significant case of crosstalk when observing <A href="http://parts.igem.org/Part:BBa_R0079" class="external text" rel="nofollow">pLas</A> is shown in the graph in the center of the matrix. It is clearly shown that 3OC6-HSL (Lux-AHL) binding to the corresponding <A href="http://parts.igem.org/Part:BBa_C0062" class="external text" rel="nofollow">LuxR</A> regulator is able to induce <A href="http://parts.igem.org/Part:BBa_R0079" class="external text" rel="nofollow">pLas</A>, resulting in fluorescence values of about 250 a.u.. This is the most severe case of crosstalk observed as the induction of <A href="http://parts.igem.org/Part:BBa_R0079" class="external text" rel="nofollow">pLas</A> by the corresponding inducer and regulator molecule is not significantly different measured by fluorescence as induction by Lux-AHL binding the <A href="http://parts.igem.org/Part:BBa_C0062" class="external text" rel="nofollow">LuxR</A> and subsequently <A href="http://parts.igem.org/Part:BBa_R0079" class="external text" rel="nofollow">pLas</A>. 
For C4-HSL binding the three regulators <A href="http://parts.igem.org/Part:BBa_C0179" class="external text" rel="nofollow">LasR</A>, <A href="http://parts.igem.org/Part:BBa_C0062" class="external text" rel="nofollow">LuxR</A> and <A href="http://parts.igem.org/Part:BBa_C0171" class="external text" rel="nofollow">RhlR</A> and then the <A href="http://parts.igem.org/Part:BBa_R0079" class="external text" rel="nofollow">pLas</A> no crosstalk can be observed.
</P></TD></TR></TBODY></TABLE><TABLE class="wikitable" style="background-color: white; text-align:center; width:auto; margin: auto;"><CAPTION><B>Table 3</B> Crosstalk matrix for the promoter prhl (<A href="http://parts.igem.org/Part:BBa_I14017:Experience" class="external text" rel="nofollow">BBa_I14017</A>)

</CAPTION><TBODY><TR><TD colspan="4"><P>In this set of experiments the promoter <A href="http://parts.igem.org/Part:BBa_R0071" class="external text" rel="nofollow">pRhl</A> was tested for potential crosstalk. In the top left position we observe the induction of <A href="http://parts.igem.org/Part:BBa_R0071" class="external text" rel="nofollow">pRhl</A> by C4-HSL bound to the <A href="http://parts.igem.org/Part:BBa_C0171" class="external text" rel="nofollow">regulator RhlR</A>. The switching behaviour was observed at a C4-HSL concentration of 1 μM.  
In the case of 3OC12-HSL binding the <A href="http://parts.igem.org/Part:BBa_C0171" class="external text" rel="nofollow">RhlR regulator</A> and subsequently the <A href="http://parts.igem.org/Part:BBa_R0071" class="external text" rel="nofollow">promoter pRhl</A>insignificant crosstalk has been observed.
Severe crosstalk was observed in the case of 3OC6-HSL binding the <A href="http://parts.igem.org/Part:BBa_C0171" class="external text" rel="nofollow">RhlR regulator</A> followed by induction of <A href="http://parts.igem.org/Part:BBa_R0071" class="external text" rel="nofollow">pRhl</A>. The transition occurred at a concentration of the inducer molecule of 1 μM but compared to the reference curve a lower value of fluorescence per OD was observed (1000 a.u.). Another case of crosstalk with the <A href="http://parts.igem.org/Part:BBa_R0071" class="external text" rel="nofollow">pRhl</A> was detected with 3OC12-HSL binding to the corresponding <A href="http://parts.igem.org/Part:BBa_C0179" class="external text" rel="nofollow">LasR regulator</A> followed by inducing the promoter <A href="http://parts.igem.org/Part:BBa_R0071" class="external text" rel="nofollow">pRhl</A>. Here switching occurred at a concentration 1 nM of 3OC12-HSL and reached fluorescence per OD of 750 a.u.. This is approximately 0.5 fold the value of the fluorescence per OD shown by the reference curve indicated in green.
</P></TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Conclusion_of_crosstalk_experiments">Conclusion of crosstalk experiments</SPAN></H3><P>As shown in the graphs in the matrices above, we found and quantitatively characterized all three levels of crosstalk. The three levels were the following:
</P><UL><LI>A given promoter with its corresponding regulator and a different inducer molecule
</LI><LI>A given promoter with an unspecific regulator and a particular inducer
</LI><LI>A given promoter with both regulator and inducer being unspecific
</LI></UL><P>Unspecific inducers binding to the regulators as well as unspecific binding of the regulator to another promoter species was observed in almost all possible combinations. 
To conclude, we were not able to find an orthogonal quorum sensing pair out of the three systems investigated (<A href="https://2014.igem.org/Team:ETH_Zurich/data#Used_and_Characterized_Pre-Existing_Parts" class="external text" rel="nofollow">LuxI/LuxR, LasI/LasR, or RhlI/RhlR</A>). While we see a significant effect when implementing the influence of these crosstalks (on an inducer-, regulator- and promoter-level) in our <A href="https://2014.igem.org/Team:ETH_Zurich/modeling/whole" class="external text" rel="nofollow">whole cell model</A>, the logic gate still continues to function for a range of inputs at physiological concentrations.
</P><P><CENTER><SVG xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" version="1.1" id="Layer_1" x="50%" y="0px" width="80" height="80" viewbox="0 0 100 100" enable-background="0 0 1000 1000" xml:space="preserve"><A href="#top" id="red" class="circled scrolly" style="text-decoration:none;"><TEXT x="50" y="50" style="cursor: pointer" fill="#FFFFFF" font-size="15" text-anchor="middle">top</TEXT></A></SVG></CENTER></P><H2><SPAN class="mw-headline" id="Riboregulators">Riboregulators</SPAN></H2><P>High basal expression from inactive inducible promoter (leakiness) is a common challenge in the implementation of robust biological control system. The problem is especially severe for systems that approximate digital Boolean logic or drive an amplified downstream response. In the first case an OFF state can be interpreted as ON resulting in incorrect computations. The second scenario leads to high levels of undesired final response. Many former iGEM teams have encountered unwanted basal expression of their genes of interest (<I>goi</I>) due to promoter leakiness (<A href="https://2013.igem.org/Team:ETH_Zurich/Optimization" class="external text" rel="nofollow">ETH2013</A>, <A href="https://2012.igem.org/Team:Groningen/pigmentproduction" class="external text" rel="nofollow">Groningen2012</A>, amongst others), resulting often in narrow optimal operation conditions for their biological devices. Since our system is based on Boolean Logic for its decisions and relies on downstream amplification to regenerate and propagate the signal as a first step we investigated strategies to improve tight gene control. Looking into the available literature we found riboregulators as a promising, highly generalizable approach to address the issue of leakiness<SUP><A href="/Team:ETH_Zurich/project/references#refIsaacs" title="Team:ETH Zurich/project/references">[32]</A></SUP>.  As a convenient proof of concept we coupled the system with <A href="https://2014.igem.org/Team:ETH_Zurich/project/background#Quorum_Sensing" class="external text" rel="nofollow">quorum sensing modules</A> characterized the response and made the resulting parts available for the iGEM community in the <A href="http://parts.igem.org/Main_Page" class="external text" rel="nofollow">Registry of Standard Biological Parts</A>. We suggest that this approach can be generalized to improve many of the inducible system contained in the registry
</P><P>The riboregulator systems include two parts: 1) a <I>cis</I>-repressed RBS in front of the <I>goi</I>, and 2) a co-expressed <I>trans</I>-activating RNA. 
</P><P>
To characterize this system for reliable cell-to-cell communication, we combined the <A href="https://2014.igem.org/Team:ETH_Zurich/data#Our_Favorite_New_Characterized_Parts" class="external text" rel="nofollow">riboregulator part</A> with promoters of the quorum-input-sensing systems used by our team (<A href="https://2014.igem.org/Team:ETH_Zurich/data#Used_and_Characterized_Pre-Existing_Parts" class="external text" rel="nofollow">LuxI/LuxR, LasI/LasR, and RhlI/RhlR</A>). In our final <A href="https://2014.igem.org/Team:ETH_Zurich/data#Gene_Circuit_and_Parts" class="external text" rel="nofollow">gene circuit</A>, the riboregulators were intended for tight control of <A href="https://2014.igem.org/Team:ETH_Zurich/project/background#Integrases" class="external text" rel="nofollow">integrases</A>. However, to have an easily quantifiable and proportional output, we initially used GFP as our riboregulated <I>goi</I> and measured the output fluorescence with a <A href="https://2014.igem.org/Team:ETH_Zurich/lab/protocols#Tecan_Infinite_M200_Pro.E2.84.A2" class="external text" rel="nofollow">Tecan plate reader</A>. This approach is described below in the figure 1.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:802px;"><DIV class="thumbcaption"><B>Figure 2</B> Characterization of the riboregulator system. First, a defined amount of the incoming signal molecule 3OC6-HSL binds to LuxR. After dimerization, the complex binds to its corresponding promoter pLux. Both, the <I>trans</I>-activator(taR12y) and the <I>cis</I>-repressor (crR12y) together with superfolder green fluorecent protein (sfGFP), are under control of pLux. Upon induction with LuxR/3OC6-HSL, both are transcribed simultaneously and the taR12y (key symbol) opens up the crR12y sequence (lock) in front of the sfGFP mRNA. This enables translation and sfGFP production (left-hand side). Low transcription (leakiness) gives only small concentrations of mRNA, as a result 'lock' (<I>cis</I>-repressor) and 'key' (<I>trans</I>-activator) structures do not encounter each other and the translation is not enabled (right-hand side).</DIV></DIV></DIV></DIV><P>The first set of experiments was conducted with cells transformed with two different pPaB plasmids. These plasmids contained either the pBR322 origin (pMB1) or the p15A origin, yielding  a stable two-plasmid system with about 15-20 copies per plasmid and cell (medium-copy).  Furthermore the published riboregulator sequence includes two forbidden restriction sites (EcoRI and XbaI), since the sites removal   could affect the folding or functionality of the part, the original sequences were used as a control.  After characterizing the original part the restriction sites were removed by two different approaches: a) multiple site-directed mutagenesis, b) blunting and ligation (Klenow and T4 DNA polymerase). The new Biobrick compatible riboregulator was tested to confirm that no severe loss-of-function occurred due to the site-removal.
In a third step, the construct was transferred into the <A href="http://parts.igem.org/Part:pSB1C3" class="external text" rel="nofollow">pSB1C3</A> backbone (pMB1 origin, high copy number of 100-300) to be in line with the <A href="http://parts.igem.org/Help:Standards/Assembly" class="external text" rel="nofollow">BioBrick Standard</A>. Below, you can find the corresponding graphs in figure 2 and figure3, respectively. Future characterization of these constructs should allow us to explore the leakiness dependence on plasmid amount and if the tight expression is robust for different copy numbers. 
</P><P>We have characterized promoters response of the three quorum-sensing systems. These are <A href="https://2014.igem.org/Team:ETH_Zurich/data#Used_and_Characterized_Pre-Existing_Parts" class="external text" rel="nofollow">pLux, pLas, and pRhl</A>. We first measured the transfer functions with a non-regulated RBS in front of sfGFP, i. e. the amount of fluorescence in dependence of the inducer concentration present and compared the response with the corresponding riboregulated constructs with and without the forbidden restriction sites. All experiments were carried out in 96-well <A href="https://2014.igem.org/Team:ETH_Zurich/lab/protocols#Tecan_Infinite_M200_Pro.E2.84.A2" class="external text" rel="nofollow">microtiter plate format</A> for 10 h. This kinetic data-sets were also used for parameter-fitting in our <A href="https://2014.igem.org/Team:ETH_Zurich/modeling/qs#Retrieving_degradation_rates" class="external text" rel="nofollow">modeling approach</A>.
</P><P>We found an about 60-fold reduced basal GFP expression and an increased signal-to-noise ratio of about 6-fold when using our riboregulator construct, as compared to a non-regulated <A href="http://parts.igem.org/Part:BBa_B0034" class="external text" rel="nofollow">reference RBS (B0034)</A>. These results are summarized in figure 2. The BioBrick conform module was confirmed to show no loss-of-funtion due to EcoRI and XbaI site removal. Interestingly the sensitivity towards the inducer was reduced (see figure 3). Is interesting to speculate if small changes in the sequences flanking the riboregulator could be used as a way to switch the response curve.
</P><TABLE class="wikitable" style="background-color: white; text-align:center; width:auto; margin: auto;"><TBODY><TR><TD><B>Figure 3</B> Improved signal-to-noise ratio and decreased basal GFP expression (leakiness) due to the use of a riboregulator in combination with a quorum-sensing module. The fluorescence per OD<SUB>600</SUB> is shown for the LuxR-system with a complete riboregulator over an inducer-range of 10<SUP>-13</SUP> M to 10<SUP>-5</SUP> M (dashed, light blue). An incomplete riboregulator without the <I>trans</I>-activator shows the expected reduced sensitivity towards the inducer (dark blue). As a reference, a system with a <A href="http://parts.igem.org/Part:BBa_B0034" class="external text" rel="nofollow">non-regulated RBS (BBa_B0034)</A> is shown (light blue). Data points are mean values of triplicate measurements in 96-well microtiter plates 200 min after induction ± standard deviation. For the full data set and kinetics please <A href="https://2014.igem.org/Team:ETH_Zurich/contact" class="external text" rel="nofollow">contact</A> us or visit the <A href="https://2014.igem.org/Team:ETH_Zurich/data/raw" class="external text" rel="nofollow">raw data</A> page.
</TD><TD><B>Figure 4</B> Confirmation of the improved signal-to-noise ratio and decreased basal GFP expression (leakiness) due to the use of a riboregulator without (w/o) EcoRI and XbaI restriction sites  in combination with a quorum-sensing module. The fluorescence per OD<SUB>600</SUB> is shown for the LuxR-system with an unchanged riboregulator (dashed, light blue) and a regulator with a changed sequence due to EcoRI and XbaI restriction site removal (dashed, dark blue). The inducer range covers 10<SUP>-13</SUP> M to 10<SUP>-5</SUP> M. As a reference, a system with a <A href="http://parts.igem.org/Part:BBa_B0034" class="external text" rel="nofollow">non-regulated RBS (BBa_B0034)</A> is shown (light blue). Data points are mean values of triplicate measurements in 96-well microtiter plates 200 min after induction ± standard deviation. For the full data set and kinetics please <A href="https://2014.igem.org/Team:ETH_Zurich/contact" class="external text" rel="nofollow">contact</A> us or visit the <A href="https://2014.igem.org/Team:ETH_Zurich/data/raw" class="external text" rel="nofollow">raw data</A> page.
</TD></TR></TBODY></TABLE><P><CENTER><SVG xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" version="1.1" id="Layer_1" x="50%" y="0px" width="80" height="80" viewbox="0 0 100 100" enable-background="0 0 1000 1000" xml:space="preserve"><A href="#top" id="blue" class="circled scrolly" style="text-decoration:none;"><TEXT x="50" y="50" style="cursor: pointer" fill="#FFFFFF" font-size="15" text-anchor="middle">top</TEXT></A></SVG></CENTER></P><H2><SPAN class="mw-headline" id="Integrases"> Integrases </SPAN></H2><P>The design of our <A href="https://2014.igem.org/Team:ETH_Zurich/modeling/xor" class="external text" rel="nofollow">XOR gates</A> was based on <A href="https://2014.igem.org/Team:ETH_Zurich/modeling/xor#Biological_Principles" class="external text" rel="nofollow">integrase logic</A><SUP><A href="/Team:ETH_Zurich/project/references#refBonnet" title="Team:ETH Zurich/project/references">[9]</A></SUP>. This means, depending on the input molecules, integrases can be expressed, subsequently switch a terminator sequence previously blocking gene expression, and then the output gene can be transcribed. This approach is explained <A href="https://2014.igem.org/Team:ETH_Zurich/modeling/xor#Biological_Principles" class="external text" rel="nofollow">here</A>.
</P><P>
In order to characterize the integrase system described above, we first combined the <A href="http://parts.igem.org/Part:BBa_R0062" class="external text" rel="nofollow">pLux promoter (BBa_R0062)</A> with one of our <A href="https://2014.igem.org/Team:ETH_Zurich/modeling/int" class="external text" rel="nofollow">integrase</A> genes <I>bxb1</I>, followed directly by a red fluorescent protein (RFP, mCherry) to make the expression accessible. Also, this system includes an XOR buffer gate per default blocking transcription of sfGFP. Upon BXB1 activity and switching the gate into ON-state, the terminator should have been removed and sfGFP should have been expressed. We intially designed three different constructs for characterization of the recombinases and their cross-activity. However, the measurement of fluorescent proteins, with both a <A href="https://2014.igem.org/Team:ETH_Zurich/lab/protocols#Tecan_Infinite_M200_Pro.E2.84.A2" class="external text" rel="nofollow">plate reader</A> and a <A href="https://2014.igem.org/Team:ETH_Zurich/lab/protocols#BD_LSRFortessa.E2.84.A2_Flow_Cytometer_System" class="external text" rel="nofollow">flow cytometer</A>, did not indicate sfGFP expression due to recombinase activity. Nevertheless, RFP was clearly detectable upon induction in plate reader experiments suggesting that the induction itself worked (see figure 5 and 6).
</P><TABLE class="wikitable resized" style="background-color: white; border: 0px+important;"><TBODY><TR><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><B>Figure 5</B><B>Expression of red fluorescent protein (RFP) and green fluorescent protein (GFP) 250 min after induction with various concentrations of 3OC6-HSL (10<SUP>-13</SUP> M to 10<SUP>-4</SUP> M).</B> RFP is under the control of the <A href="http://parts.igem.org/Part:BBa_R0062" class="external text" rel="nofollow">pLux promoter (BBa_R0062)</A> together with the integrase BXB1. Upon expression of BXB1 a buffer gate should have been swtiched to ON-state producing GFP. Data points are mean values of triplicate measurements in 96-well microtiter plates ± standard deviation.  For the full data set and kinetics please <A href="https://2014.igem.org/Team:ETH_Zurich/contact" class="external text" rel="nofollow">contact</A> us or visit the <A href="https://2014.igem.org/Team:ETH_Zurich/data/raw" class="external text" rel="nofollow">raw data</A> page.</DIV></DIV></DIV></DIV></TD><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><B>Figure 6</B><B>Expression of red fluorescent protein (RFP) and green fluorescent protein (GFP) 610 min after induction with various concentrations of 3OC6-HSL (10<SUP>-13</SUP> M to 10<SUP>-4</SUP> M).</B> RFP is under the control of the <A href="http://parts.igem.org/Part:BBa_R0062" class="external text" rel="nofollow">pLux promoter (BBa_R0062)</A> together with the integrase BXB1. Upon expression of BXB1 a buffer gate should have been swtiched to ON-state producing GFP. Data points are mean values of triplicate measurements in 96-well microtiter plates ± standard deviation. For the full data set and kinetics please <A href="https://2014.igem.org/Team:ETH_Zurich/contact" class="external text" rel="nofollow">contact</A> us or visit the <A href="https://2014.igem.org/Team:ETH_Zurich/data/raw" class="external text" rel="nofollow">raw data</A> page.</DIV></DIV></DIV></DIV></TD></TR></TBODY></TABLE><P>
As our constructs did not show the expected functionality, we decided to directly use the plasmids described by Bonnet <I>et al</I>.<SUP><A href="/Team:ETH_Zurich/project/references#refBonnet" title="Team:ETH Zurich/project/references">[9]</A></SUP> which where obtained from addgene (<A href="http://www.addgene.org/44456" class="external text" rel="nofollow">Dual-recombinase-controller</A>, <A href="http://www.addgene.org/44453/" class="external text" rel="nofollow">XOR gate-V2.0</A>). The data available from the original experiment by Bonnet<SUP><A href="/Team:ETH_Zurich/project/references#refBonnet" title="Team:ETH Zurich/project/references">[9]</A></SUP> was used in our model to retrieve the <A href="https://2014.igem.org/Team:ETH_Zurich/modeling/int" class="external text" rel="nofollow">missing parameters of integrases</A>. However, we were using a <A href="http://www.openwetware.org/wiki/E._coli_genotypes#TOP10_.28Invitrogen.29" class="external text" rel="nofollow">TOP10</A> strain not expressing TetR by default (as compared to <A href="http://www.expressys.com/main_strains.html" class="external text" rel="nofollow">DH5alphaZ1</A>) and as a result our strain had to be co-transformed with an additional plasmid encoding TetR. Also, we used defined <A href="https://2014.igem.org/Team:ETH_Zurich/lab/protocols#Defined_M9_medium_with_0.4_.25_glycerol" class="external text" rel="nofollow">M9 medium</A> with 0.4% glycerol and 1% CAA instead of proprietary defined medium (<A href="http://teknova-blog.com/hi-def-azure-media" class="external text" rel="nofollow">Teknova Hi-Def Azure medium</A>). As of today, this set-up did not allow us to get the integrase XOR gate running. The fluorescence readout for ON-states (exactly one input, either 0.1% L-arabinose or 200 ng/mL anhydrous tetracycline) and the OFF-states was not as expected (see figure 7). The OFF-state should not show increased fluorescence over time, while the ON-states should increase significantly after 4 h and continue to increase over the whole time span<SUP><A href="/Team:ETH_Zurich/project/references#refBonnet" title="Team:ETH Zurich/project/references">[9]</A></SUP>. We are not giving up on this and are proceeding with debugging our construct further and hope to find a solution until the Giant Jamboree in Boston.
</P><P><CENTER><SVG xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" version="1.1" id="Layer_1" x="50%" y="0px" width="80" height="80" viewbox="0 0 100 100" enable-background="0 0 1000 1000" xml:space="preserve"><A href="#top" id="green" class="circled scrolly" style="text-decoration:none;"><TEXT x="50" y="50" style="cursor: pointer" fill="#FFFFFF" font-size="15" text-anchor="middle">top</TEXT></A></SVG></CENTER></P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><B>Figure 7 Expression of green fluorescent protein (GFP) after induction of <A href="https://2014.igem.org/Team:ETH_Zurich/modeling/xor#Biological_Principles" class="external text" rel="nofollow">XOR integrase logic</A>.</B> Upon induction of integrase BXB1 (t=0 min) a <A href="https://2014.igem.org/Team:ETH_Zurich/lab/sequences#BUFFER_Gate_Construct" class="external text" rel="nofollow">buffer gate</A> should have been switched to ON-state producing GFP only if either L-arabinose (L-ara) or anhydrous tetracycline (aTc) is present. Data points are mean values of triplicate measurements in 96-well microtiter plates ± standard deviation. For the full data set and kinetics please <A href="https://2014.igem.org/Team:ETH_Zurich/contact" class="external text" rel="nofollow">contact</A> us or visit the <A href="https://2014.igem.org/Team:ETH_Zurich/data/raw" class="external text" rel="nofollow">raw data</A> page.</DIV></DIV></DIV></DIV></DIV></DIV><DIV id="footer"><HEADER><H5>We thank our sponsors:</H5></HEADER></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:ETH_Zurich/expresults">http://2014.igem.org/Team:ETH_Zurich/expresults</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:ETH_Zurich/expresults" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:ETH_Zurich/expresults" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:ETH_Zurich/expresults&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:ETH_Zurich/expresults&amp;oldid=377713" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>