<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_ETH_Zurich_project_infopro"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:ETH_Zurich/project/infopro">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:ETH_Zurich/project/infopro&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:ETH_Zurich/project/infopro&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:ETH_Zurich/project/infopro&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:ETH_Zurich/project/infopro" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:ETH Zurich/project/infopro</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><TITLE>iGEM ETH Zurich 2014</TITLE><DIV id="header"><NAV id="nav"><UL><LI><A href="/Team:ETH_Zurich" title="Team:ETH Zurich">Home</A></LI><LI><A name="Project" style="cursor: default" href="javascript:void(0)">Project</A></LI><LI><A href="/Team:ETH_Zurich/project/overview" title="Team:ETH Zurich/project/overview">Overview</A></LI><LI><STRONG class="selflink">Information Processing</STRONG></LI><LI><A href="/Team:ETH_Zurich/project/background" title="Team:ETH Zurich/project/background">Background</A></LI><LI><A href="/Team:ETH_Zurich/project/goals" title="Team:ETH Zurich/project/goals">Goals</A></LI><LI><A href="/Team:ETH_Zurich/project/applications" title="Team:ETH Zurich/project/applications">Applications</A></LI><LI><A href="/Team:ETH_Zurich/project/references" title="Team:ETH Zurich/project/references">References</A></LI><LI><A href="/Team:ETH_Zurich/blog" title="Team:ETH Zurich/blog">Blog</A></LI></UL><LI><A name="Achievements" style="cursor: default" href="javascript:void(0)">Achievements</A></LI><LI><A href="/Team:ETH_Zurich/achievements" title="Team:ETH Zurich/achievements">Achievements</A></LI><LI><A href="/Team:ETH_Zurich/criteria" title="Team:ETH Zurich/criteria">Medal Criteria</A></LI><LI><A href="/Team:ETH_Zurich/expresults" title="Team:ETH Zurich/expresults">Results</A></LI><LI><A name="Data page" style="cursor: default" href="javascript:void(0)">Data page</A></LI><LI><A href="/Team:ETH_Zurich/data#Gene_Circuit" title="Team:ETH Zurich/data">Gene Circuit and Parts</A></LI><LI><A href="/Team:ETH_Zurich/data#Used_and_Characterized_Pre-Existing_Parts" title="Team:ETH Zurich/data">Used and Characterized Pre-Existing Parts</A></LI><LI><A href="/Team:ETH_Zurich/data#Our_Favorite_New_Characterized_Parts" title="Team:ETH Zurich/data">Our Favorite New Characterized Parts</A></LI><LI><A href="/Team:ETH_Zurich/data/raw" title="Team:ETH Zurich/data/raw">Raw Data</A></LI><LI><A name="Modeling" style="cursor: default" href="javascript:void(0)">Modeling</A></LI><LI><A href="/Team:ETH_Zurich/modeling/overview" title="Team:ETH Zurich/modeling/overview">Overview</A></LI><LI><A href="/Team:ETH_Zurich/modeling/qs" title="Team:ETH Zurich/modeling/qs">Quorum Sensing</A></LI><LI><A href="/Team:ETH_Zurich/modeling/int" title="Team:ETH Zurich/modeling/int">Integrases</A></LI><LI><A href="/Team:ETH_Zurich/modeling/xor" title="Team:ETH Zurich/modeling/xor">XOR Gate</A></LI><LI><A href="/Team:ETH_Zurich/modeling/whole" title="Team:ETH Zurich/modeling/whole">Whole cell model</A></LI><LI><A href="/Team:ETH_Zurich/modeling/diffmodel" title="Team:ETH Zurich/modeling/diffmodel">Diffusion</A></LI><LI><A href="/Team:ETH_Zurich/modeling/parameters" title="Team:ETH Zurich/modeling/parameters">Parameters and Tools</A></LI><LI><A name="Lab" style="cursor: default" href="javascript:void(0)">Lab</A></LI><LI><A href="/Team:ETH_Zurich/lab/overview" title="Team:ETH Zurich/lab/overview">Overview</A></LI><LI><A href="/Team:ETH_Zurich/lab/protocols" title="Team:ETH Zurich/lab/protocols">Materials &amp; Methods</A></LI><LI><A href="/Team:ETH_Zurich/lab/sequences" title="Team:ETH Zurich/lab/sequences">Sequences</A></LI><LI><A href="/Team:ETH_Zurich/lab/bead" title="Team:ETH Zurich/lab/bead">Alginate Beads</A></LI><LI><A href="/Team:ETH_Zurich/lab/chip" title="Team:ETH Zurich/lab/chip">Millifluidic Chip</A></LI><LI><A href="/Team:ETH_Zurich/lab/safety" title="Team:ETH Zurich/lab/safety">Safety</A></LI><LI><A name="Human Practice" style="cursor: default" href="javascript:void(0)">Human practice</A></LI><LI><A href="/Team:ETH_Zurich/human/overview" title="Team:ETH Zurich/human/overview">Overview</A></LI><LI><A href="/Team:ETH_Zurich/human/survey" title="Team:ETH Zurich/human/survey">Survey</A></LI><LI><A href="/Team:ETH_Zurich/human/interviews" title="Team:ETH Zurich/human/interviews">Interviews</A></LI><LI><A href="/Team:ETH_Zurich/human/outreach" title="Team:ETH Zurich/human/outreach">Outreach</A></LI><LI><A href="/Team:ETH_Zurich/human/essay" title="Team:ETH Zurich/human/essay">Our insights</A></LI><LI><A name="Team" style="cursor: default" href="javascript:void(0)">Team</A></LI><LI><A href="https://igem.org/Team.cgi?year=2014&amp;team_name=ETH_Zurich" class="external text" rel="nofollow">Official Team Profile</A></LI><LI><A href="/Team:ETH_Zurich/team" title="Team:ETH Zurich/team">Our team</A></LI><LI><A href="/Team:ETH_Zurich/adandin" title="Team:ETH Zurich/adandin">Advisors and Instructors</A></LI><LI><A href="/Team:ETH_Zurich/collaborations" title="Team:ETH Zurich/collaborations">Collaborations</A></LI><LI><A href="/Team:ETH_Zurich/acknowledgements" title="Team:ETH Zurich/acknowledgements">Acknowledgements</A></LI><LI><A href="/Team:ETH_Zurich/contact" title="Team:ETH Zurich/contact">Contacting us</A></LI></NAV><DIV class="inner1"><HEADER><H1>  Information Processing  </H1></HEADER><H2><SPAN class="mw-headline" id="Information_Processing"> Information Processing </SPAN></H2><H3><SPAN class="mw-headline" id="About_the_track"> About the track </SPAN></H3><P><A href="https://2014.igem.org/Tracks/Information_Processing" class="external text" rel="nofollow"><I>«Information Processing in iGEM covers a diverse range of projects. IP teams are not trying to solve a real world problem with practical applications, but to tackle an interesting problem that might otherwise not attract attention. Teams enter this track if they are attempting projects such as building elements of a biological computer, creating a game using biology or working on a signal processing challenges. Engineering ways to make biological systems perform computation is one of the core goals of synthetic biology. We generally work at the DNA level, engineering systems to function using BioBricks. In most biological systems, protein-protein interactions are where the majority of processing takes place. Being able to design proteins to accomplish computation would allow for systems to function on a much faster timescale than the current transcription-translation paradigm»</I> (iGEM mainpage)</A></P><H3><SPAN class="mw-headline" id="Why_we_chose_this_track"> Why we chose this track </SPAN></H3><P>With our project, Mosai<I>coli</I>, we investigate the emergence of complex patterns from simple mathematical rules. Such rules can be reduced to Boolean logic gates, in our case XOR gates. The computations were implemented with integrases, proteins modifying DNA between specific sites. These modifications in turn influence the expression of other proteins which can then indicate the previous change on the genetic level. Additionally chemical wiring of the information to the next cells happens, this again on the protein or small-molecule level. This wiring of information and repeated information processing allows the construction of cellular automata and eventually biocomputers using the analogy of electrical circuits.
</P><H3><SPAN class="mw-headline" id="The_goal:_Emergence_of_patterns_via_information_processing"> The goal: Emergence of patterns via information processing</SPAN></H3><P>We implement a cellular automaton with bacterial cells containing our logic circuitry. Each bacterial colony serves as a core, computing an XOR gate. First, a sensor device detects the input, <I>N</I>-acyl homoserine lactones (AHL). Check the <A href="https://2014.igem.org/Team:ETH_Zurich/project/background/biotools#Biological_Tools" class="external text" rel="nofollow">quorum sensing and integrases article</A> for more information. Then, the cell integrates this signal through a logic gate, performed by serine integrases by sensing on the protein level and acting on the DNA level. A necessary post-processing step allows then the production of a new AHL variant due to activated gene expression through the integrase. Meanwhile, green fluorescent protein (GFP) indicates the state of the colony and serves as a long-lasting visual read out. The produced AHL output-signal then propagates in a directed fashion through a <A href="https://2014.igem.org/Team:ETH_Zurich/lab/chip" class="external text" rel="nofollow">millifluidic</A> grid to the next bacterial colony. This iterative process faces the challenges of <A href="https://2014.igem.org/Team:ETH_Zurich/expresults/rr#Riboregulators" class="external text" rel="nofollow">leakiness</A>, crosstalk, protein-level computation and exact <A href="https://2014.igem.org/Team:ETH_Zurich/modeling/diffmodel" class="external text" rel="nofollow">diffusion</A> steps. This information pathway is shown in figure 1.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:802px;"><DIV class="thumbcaption"><B>Figure 1</B> The information pathway in the project Mosai<I>coli</I> on a colony level. Each cell colony is part of a greater memory unit (left-hand side) and gets its input information from the neighboring colonies. The input is given in the form of two different AHLs, here named A and B. This information is then step-wise processed in each of the colonies: sensing - computing - producing - sending. The output molecules serve then as the input for the next colony (middle and left-hand side). This iterative process allows the information processing from the top row of the memory unit to the bottom by chemical wiring via diffusing signals and bio-computations inside of the colonies.</DIV></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="The_steps_involved:_From_sensing_to_sending"> The steps involved: From sensing to sending</SPAN></H3><H3><SPAN class="mw-headline" id="Sensing"> Sensing </SPAN></H3><P>Diffused AHL molecules are identified via the <A href="https://2014.igem.org/Team:ETH_Zurich/expresults#Quorum_Sensing" class="external text" rel="nofollow">quorum sensing machinery</A>. The <A href="https://2014.igem.org/Team:ETH_Zurich/modeling/qs" class="external text" rel="nofollow">transfer function</A> between the sensed AHL concentration and the promoter activation deviates usually from the ideal one due to leakiness and crosstalk between different types of AHL molecules, regulatory proteins and promoters (we <A href="https://2014.igem.org/Team:ETH_Zurich/expresults/qs#Quorum_Sensing" class="external text" rel="nofollow">characterized</A> it for <A href="http://parts.igem.org/Part:BBa_R0062" class="external text" rel="nofollow">Lux promoter (BBa_R0062)</A>, <A href="http://parts.igem.org/Part:BBa_R0079" class="external text" rel="nofollow">Las promoter (BBa_R0079)</A> and <A href="http://parts.igem.org/Part:BBa_R0071" class="external text" rel="nofollow">Rhl promoter (BBa_R0071)</A>). To limit the risk of error propagation, we implemented a <A href="http://parts.igem.org/Part:BBa_K1541000" class="external text" rel="nofollow">riboregulated system</A>.
</P><H3><SPAN class="mw-headline" id="Computing"> Computing </SPAN></H3><P><A href="https://2014.igem.org/Team:ETH_Zurich/expresults#Integrases" class="external text" rel="nofollow">Integrases</A> compute an <A href="https://2014.igem.org/Team:ETH_Zurich/modeling/xor" class="external text" rel="nofollow">XOR logic gate</A> via single switching or double switching of a terminator. Their behavior depends on a set of <A href="https://2014.igem.org/Team:ETH_Zurich/modeling/int" class="external text" rel="nofollow">parameters</A>.
</P><H3><SPAN class="mw-headline" id="Producing"> Producing </SPAN></H3><P>To propagate the computational results, the output has to be translated again into a quorum sensing molecule. Therefore a synthase is expressed to catalyze the production of a new AHL signal.
</P><H3><SPAN class="mw-headline" id="Memorising"> Memorising </SPAN></H3><P>Alongside the AHL synthase, GFP is produced according to the XOR truth table. The fluorescent protein acts both as long term memory for the colonies in ON-state and as visual signal, storing and showing the propagating pattern formation.
</P><H3><SPAN class="mw-headline" id="Sending"> Sending </SPAN></H3><P>The produced AHL diffuses through channels on the memory unit to the rows below, becoming the input of the following information processing layer. The speed of this 'wiring' by directed diffusion determines the time needed for the pattern to emerge (as long as the channels are  relatively long and diffusion is the time-limting step as compared to the actual computation itself). You can find out more information about diffusion and chip design on our <A href="https://2014.igem.org/Team:ETH_Zurich/modeling/diffmodel" class="external text" rel="nofollow">modeling page</A>, our <A href="https://2014.igem.org/Team:ETH_Zurich/expresults#Diffusion_On_Chip" class="external text" rel="nofollow">experimental results page</A> and our <A href="https://2014.igem.org/Team:ETH_Zurich/lab/chip" class="external text" rel="nofollow">chip page</A>.
</P><P>
The overall process is summarized for the chip, or memory unit, level in figure 2.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><B>Figure 2</B> The information pathway in the project Mosai<I>coli</I> on the chip level. After initializing the signal in the top row, it propagates through the wells chemically wired by directed diffusion. In each well, a bacterial colony has to be able to proceed in the information pathway: sensing, integrating (computing), producing and sending. These successive iterations leads to possible error propagation and robustness is one major issue of our system.</DIV></DIV></DIV></DIV></DIV></DIV><DIV id="footer"><HEADER><H5>We thank our sponsors:</H5></HEADER></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:ETH_Zurich/project/infopro">http://2014.igem.org/Team:ETH_Zurich/project/infopro</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:ETH_Zurich/project/infopro" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:ETH_Zurich/project/infopro" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:ETH_Zurich/project/infopro&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:ETH_Zurich/project/infopro&amp;oldid=398855" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>