<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_ETH_Zurich_blog"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:ETH_Zurich/blog">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:ETH_Zurich/blog&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:ETH_Zurich/blog&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:ETH_Zurich/blog&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:ETH_Zurich/blog" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:ETH Zurich/blog</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><DIV id="contentSub">(Redirected from <A href="/wiki/index.php?title=Team:ETH_Zurich/Blog&amp;redirect=no" title="Team:ETH Zurich/Blog">Team:ETH Zurich/Blog</A>)</DIV><TITLE>iGEM ETH Zurich 2014</TITLE><DIV id="header"><NAV id="nav"><UL><LI><A href="/Team:ETH_Zurich" title="Team:ETH Zurich">Home</A></LI><LI><A name="Project" style="cursor: default" href="javascript:void(0)">Project</A></LI><LI><A href="/Team:ETH_Zurich/project/overview" title="Team:ETH Zurich/project/overview">Overview</A></LI><LI><A href="/Team:ETH_Zurich/project/infopro" title="Team:ETH Zurich/project/infopro">Information Processing</A></LI><LI><A href="/Team:ETH_Zurich/project/background" title="Team:ETH Zurich/project/background">Background</A></LI><LI><A href="/Team:ETH_Zurich/project/goals" title="Team:ETH Zurich/project/goals">Goals</A></LI><LI><A href="/Team:ETH_Zurich/project/applications" title="Team:ETH Zurich/project/applications">Applications</A></LI><LI><A href="/Team:ETH_Zurich/project/references" title="Team:ETH Zurich/project/references">References</A></LI><LI><STRONG class="selflink">Blog</STRONG></LI></UL><LI><A name="Achievements" style="cursor: default" href="javascript:void(0)">Achievements</A></LI><LI><A href="/Team:ETH_Zurich/achievements" title="Team:ETH Zurich/achievements">Achievements</A></LI><LI><A href="/Team:ETH_Zurich/criteria" title="Team:ETH Zurich/criteria">Medal Criteria</A></LI><LI><A href="/Team:ETH_Zurich/expresults" title="Team:ETH Zurich/expresults">Results</A></LI><LI><A name="Data page" style="cursor: default" href="javascript:void(0)">Data page</A></LI><LI><A href="/Team:ETH_Zurich/data#Gene_Circuit" title="Team:ETH Zurich/data">Gene Circuit and Parts</A></LI><LI><A href="/Team:ETH_Zurich/data#Used_and_Characterized_Pre-Existing_Parts" title="Team:ETH Zurich/data">Used and Characterized Pre-Existing Parts</A></LI><LI><A href="/Team:ETH_Zurich/data#Our_Favorite_New_Characterized_Parts" title="Team:ETH Zurich/data">Our Favorite New Characterized Parts</A></LI><LI><A href="/Team:ETH_Zurich/data/raw" title="Team:ETH Zurich/data/raw">Raw Data</A></LI><LI><A name="Modeling" style="cursor: default" href="javascript:void(0)">Modeling</A></LI><LI><A href="/Team:ETH_Zurich/modeling/overview" title="Team:ETH Zurich/modeling/overview">Overview</A></LI><LI><A href="/Team:ETH_Zurich/modeling/qs" title="Team:ETH Zurich/modeling/qs">Quorum Sensing</A></LI><LI><A href="/Team:ETH_Zurich/modeling/int" title="Team:ETH Zurich/modeling/int">Integrases</A></LI><LI><A href="/Team:ETH_Zurich/modeling/xor" title="Team:ETH Zurich/modeling/xor">XOR Gate</A></LI><LI><A href="/Team:ETH_Zurich/modeling/whole" title="Team:ETH Zurich/modeling/whole">Whole cell model</A></LI><LI><A href="/Team:ETH_Zurich/modeling/diffmodel" title="Team:ETH Zurich/modeling/diffmodel">Diffusion</A></LI><LI><A href="/Team:ETH_Zurich/modeling/parameters" title="Team:ETH Zurich/modeling/parameters">Parameters and Tools</A></LI><LI><A name="Lab" style="cursor: default" href="javascript:void(0)">Lab</A></LI><LI><A href="/Team:ETH_Zurich/lab/overview" title="Team:ETH Zurich/lab/overview">Overview</A></LI><LI><A href="/Team:ETH_Zurich/lab/protocols" title="Team:ETH Zurich/lab/protocols">Materials &amp; Methods</A></LI><LI><A href="/Team:ETH_Zurich/lab/sequences" title="Team:ETH Zurich/lab/sequences">Sequences</A></LI><LI><A href="/Team:ETH_Zurich/lab/bead" title="Team:ETH Zurich/lab/bead">Alginate Beads</A></LI><LI><A href="/Team:ETH_Zurich/lab/chip" title="Team:ETH Zurich/lab/chip">Millifluidic Chip</A></LI><LI><A href="/Team:ETH_Zurich/lab/safety" title="Team:ETH Zurich/lab/safety">Safety</A></LI><LI><A name="Human Practice" style="cursor: default" href="javascript:void(0)">Human practice</A></LI><LI><A href="/Team:ETH_Zurich/human/overview" title="Team:ETH Zurich/human/overview">Overview</A></LI><LI><A href="/Team:ETH_Zurich/human/survey" title="Team:ETH Zurich/human/survey">Survey</A></LI><LI><A href="/Team:ETH_Zurich/human/interviews" title="Team:ETH Zurich/human/interviews">Interviews</A></LI><LI><A href="/Team:ETH_Zurich/human/outreach" title="Team:ETH Zurich/human/outreach">Outreach</A></LI><LI><A href="/Team:ETH_Zurich/human/essay" title="Team:ETH Zurich/human/essay">Our insights</A></LI><LI><A name="Team" style="cursor: default" href="javascript:void(0)">Team</A></LI><LI><A href="https://igem.org/Team.cgi?year=2014&amp;team_name=ETH_Zurich" class="external text" rel="nofollow">Official Team Profile</A></LI><LI><A href="/Team:ETH_Zurich/team" title="Team:ETH Zurich/team">Our team</A></LI><LI><A href="/Team:ETH_Zurich/adandin" title="Team:ETH Zurich/adandin">Advisors and Instructors</A></LI><LI><A href="/Team:ETH_Zurich/collaborations" title="Team:ETH Zurich/collaborations">Collaborations</A></LI><LI><A href="/Team:ETH_Zurich/acknowledgements" title="Team:ETH Zurich/acknowledgements">Acknowledgements</A></LI><LI><A href="/Team:ETH_Zurich/contact" title="Team:ETH Zurich/contact">Contacting us</A></LI></NAV><DIV class="inner1"><HEADER><H1>  Blog  </H1></HEADER><H4 style="color:#FFFFFF; text-align:center;"> We have uploaded our chronological progress in several articles which are classified by subject. Click on the triangles below to filter the category of articles you want to read. If you want an even more precise filtering in that category, you can choose subjects on the mosaic around.</H4><DIV id="Container"><ARTICLE class="mix human school carousel" id="schoolVisit" date="20141014"><H2><SPAN class="mw-headline" id="High_School_Students"> High School Students </SPAN></H2><H4><SPAN class="mw-headline" id="Tuesday.2C_October_14th"> Tuesday, October 14th </SPAN></H4><P>On Oct 14 2014 Stefanie and Nadine visited a high school in Bern as a part of our human practice project. The girls had the chance to teach students at the age of 16-17 years. Through these lectures, we attempted to impart the basic principles of synthetic biology and encouraged the students to participate in a discussion of the concepts presented. In these discussions we tried to focus on risks to humans, health, the environment and the public. The structure of the lessons was really open for personal inputs from the side of the students. The topics discussed ranged from whether you should or should not express GFP in other organisms (the question of freewill came up), animal trials in research, GMO's in general to the production of artificial meat in bioreactors.
</P><P>Furthermore former iGEM projects were quickly outlined and their possibilities and potential discussed. Thus, we increased the level of awareness of synthetic biology and simultaneously teach the students how to approach complex problems. Another aspect of our visit was to promote the studies of science in general. The students were encouraged to ask all questions that came to their minds. 
This school visit belongs to the outreach aspect of our human practice project as it aims to share knowledge with the public.
</P><P><A class="read-more" href="#{{{2}}}" id="red"><SVG xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" version="1.1" id="Layer_1" x="50%" y="0px" width="100" height="100" viewbox="0 0 100 100" enable-background="0 0 1000 1000" xml:space="preserve"><TEXT x="50" y="53" style="cursor: pointer" fill="#FFFFFF" font-size="15" text-anchor="middle">Read More</TEXT></SVG></A></P><CENTER><SVG xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" version="1.1" id="Layer_1" x="50%" y="0px" width="80" height="80" viewbox="0 0 100 100" enable-background="0 0 1000 1000" xml:space="preserve"><A href="#top" id="red" class="circled scrolly" style="text-decoration:none;"><TEXT x="50" y="50" style="cursor: pointer" fill="#FFFFFF" font-size="15" text-anchor="middle">top</TEXT></A></SVG></CENTER></ARTICLE><ARTICLE class="mix carousel slams human" date="20141010hum"><H2><SPAN class="mw-headline" id="Science_Slam_Basel"> Science Slam Basel </SPAN></H2><H4><SPAN class="mw-headline" id="Friday.2C_October_10th"> Friday, October 10th </SPAN></H4><P>Today, Elise went on stage: her science slam was based on the audience participation. With green and white papers, the goal of her talk was to build a human cellular automata. It was a fun night and we learned a lot from the other crazy presentations.
</P><P><CENTER><SVG xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" version="1.1" id="Layer_1" x="50%" y="0px" width="80" height="80" viewbox="0 0 100 100" enable-background="0 0 1000 1000" xml:space="preserve"><A href="#top" id="red" class="circled scrolly" style="text-decoration:none;"><TEXT x="50" y="50" style="cursor: pointer" fill="#FFFFFF" font-size="15" text-anchor="middle">top</TEXT></A></SVG></CENTER></P><P><A class="read-more" href="#week13" id="red"><SVG xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" version="1.1" id="Layer_1" x="50%" y="0px" width="100" height="100" viewbox="0 0 100 100" enable-background="0 0 1000 1000" xml:space="preserve"><TEXT x="50" y="53" style="cursor: pointer" fill="#FFFFFF" font-size="15" text-anchor="middle">Read More</TEXT></SVG></A></P></ARTICLE><ARTICLE class="mix carousel modeling stochastic" id="stocha" date="20141004"><H2><SPAN class="mw-headline" id="Stochastic_model_giving_insight_on_parameters"> Stochastic model giving insight on parameters</SPAN></H2><H4><SPAN class="mw-headline" id="Tuesday.2C_October_4th"> Tuesday, October 4th </SPAN></H4><P>We modeled the XOR module with a stochastic model, as the sites are discrete and not continuous and thus binding events might be sensitive to stochasticity. In this process, we figured out that the concentration of GFP was rapidly increasing due to fast flipping rates. It is very unlikely in biological systems. Thus, this model helped us debug an estimated paramter
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">GFP and LuxI produced when the system is ON.</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">Both inputs are present. There is an immediate GFP response, followed by degradation.</DIV></DIV></DIV></DIV><P><A class="read-more" href="#{{{2}}}" id="blue"><SVG xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" version="1.1" id="Layer_1" x="50%" y="0px" width="100" height="100" viewbox="0 0 100 100" enable-background="0 0 1000 1000" xml:space="preserve"><TEXT x="50" y="53" style="cursor: pointer" fill="#FFFFFF" font-size="15" text-anchor="middle">Read More</TEXT></SVG></A></P><CENTER><SVG xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" version="1.1" id="Layer_1" x="50%" y="0px" width="80" height="80" viewbox="0 0 100 100" enable-background="0 0 1000 1000" xml:space="preserve"><A href="#top" id="blue" class="circled scrolly" style="text-decoration:none;"><TEXT x="50" y="50" style="cursor: pointer" fill="#FFFFFF" font-size="15" text-anchor="middle">top</TEXT></A></SVG></CENTER></ARTICLE><ARTICLE class="mix human carousel talks" date="20140930"><H2><SPAN class="mw-headline" id="Talk_for_the_public"> Talk for the public</SPAN></H2><H4><SPAN class="mw-headline" id="Tuesday.2C_September_30th"> Tuesday, September 30th </SPAN></H4><H3><SPAN class="mw-headline" id="Sharing_iGEM_with_a_heterogeneous_public">Sharing iGEM with a heterogeneous public</SPAN></H3><P>Today Max gave a presentation of our project to an open public. Everyone showed great interest and we could start thinking about how to present such a complex project with a single poster or presentation.
</P><P><CENTER><SVG xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" version="1.1" id="Layer_1" x="50%" y="0px" width="80" height="80" viewbox="0 0 100 100" enable-background="0 0 1000 1000" xml:space="preserve"><A href="#top" id="red" class="circled scrolly" style="text-decoration:none;"><TEXT x="50" y="50" style="cursor: pointer" fill="#FFFFFF" font-size="15" text-anchor="middle">top</TEXT></A></SVG></CENTER></P><P><A class="read-more" href="#{{{2}}}" id="red"><SVG xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" version="1.1" id="Layer_1" x="50%" y="0px" width="100" height="100" viewbox="0 0 100 100" enable-background="0 0 1000 1000" xml:space="preserve"><TEXT x="50" y="53" style="cursor: pointer" fill="#FFFFFF" font-size="15" text-anchor="middle">Read More</TEXT></SVG></A></P></ARTICLE><ARTICLE class="mix meetings integrase carousel biobricks" id="week19" date="20140922"><H2><SPAN class="mw-headline" id="Week_19_:_Biobricks.2C_where_are_you.3F"> Week 19Â : Biobricks, where are you? </SPAN></H2><H4><SPAN class="mw-headline" id="Monday.2C_September_29th"> Monday, September 29th </SPAN></H4><P>After investigation, it is more difficult than expected to put our bricks in the biobrick format.
</P><P>To debug the integrase's experiment, we tried to have another insight by using FACS data.

</P><P><A class="read-more" href="#{{{2}}}" id="silver"><SVG xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" version="1.1" id="Layer_1" x="50%" y="0px" width="100" height="100" viewbox="0 0 100 100" enable-background="0 0 1000 1000" xml:space="preserve"><TEXT x="50" y="53" style="cursor: pointer" fill="#FFFFFF" font-size="15" text-anchor="middle">Read More</TEXT></SVG></A></P><CENTER><SVG xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" version="1.1" id="Layer_1" x="50%" y="0px" width="80" height="80" viewbox="0 0 100 100" enable-background="0 0 1000 1000" xml:space="preserve"><A href="#top" id="silver" class="circled scrolly" style="text-decoration:none;"><TEXT x="50" y="50" style="cursor: pointer" fill="#FFFFFF" font-size="15" text-anchor="middle">top</TEXT></A></SVG></CENTER></ARTICLE><ARTICLE class="mix meetings whole carousel biobricks" id="week18" date="20140922"><H2><SPAN class="mw-headline" id="Week_18_:_Decisive_meeting"> Week 18Â : Decisive meeting </SPAN></H2><H4><SPAN class="mw-headline" id="Monday.2C_September_22th"> Monday, September 22th </SPAN></H4><P>This meeting was a turning point of our iGEM adventure. We reviewed all the experiments that had been done so far. Most of them concerns quorum sensing and its non-orthogonality. We summed up the results in a three-layer graphs.
</P><P>The dry lab part presented for the first time a whole-cell model. With the parameters estimated, the model actually implemented an XOR gate. However, leakiness of promoters and crosstalk seemed to be affecting the robustness of our design.
</P><P>As all our experiments were done in our plasmids, we have to standardize them into the Biobrick format. This year, the deadline corresponds to the delivery in Boston of the biobricks. Thus, it is becoming slowly urgent...

</P><P><A class="read-more" href="#{{{2}}}" id="silver"><SVG xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" version="1.1" id="Layer_1" x="50%" y="0px" width="100" height="100" viewbox="0 0 100 100" enable-background="0 0 1000 1000" xml:space="preserve"><TEXT x="50" y="53" style="cursor: pointer" fill="#FFFFFF" font-size="15" text-anchor="middle">Read More</TEXT></SVG></A></P><CENTER><SVG xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" version="1.1" id="Layer_1" x="50%" y="0px" width="80" height="80" viewbox="0 0 100 100" enable-background="0 0 1000 1000" xml:space="preserve"><A href="#top" id="silver" class="circled scrolly" style="text-decoration:none;"><TEXT x="50" y="50" style="cursor: pointer" fill="#FFFFFF" font-size="15" text-anchor="middle">top</TEXT></A></SVG></CENTER></ARTICLE><ARTICLE class="mix carousel modeling edinburgh" id="edinburgh" date="20140615"><H2><SPAN class="mw-headline" id="Let_a_new_collaboration_begin.21"> Let a new collaboration begin! </SPAN></H2><H4><SPAN class="mw-headline" id="Sunday.2C_September_21th"> Sunday, September 21th </SPAN></H4><P>After discovering the awesome video of the Edinburgh team on Twitter, we wanted to know more about their project. As it is clear that the three quorum sensing systems we consider are crosstalking, finding a team focused on the development on communication ways between bacteria was really interesting for us. Their topic is metabolic wiring. 

We had a Skype meeting with them this week to discuss about possible opportunities of collaboration. Given the stages of our experiments, we decided that a theoretical collaboration would be more plausible. Next step will be to talk about the models and design a set-up that could fit our two projects.

</P><P><A class="read-more" href="#{{{2}}}" id="blue"><SVG xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" version="1.1" id="Layer_1" x="50%" y="0px" width="100" height="100" viewbox="0 0 100 100" enable-background="0 0 1000 1000" xml:space="preserve"><TEXT x="50" y="53" style="cursor: pointer" fill="#FFFFFF" font-size="15" text-anchor="middle">Read More</TEXT></SVG></A></P><CENTER><SVG xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" version="1.1" id="Layer_1" x="50%" y="0px" width="80" height="80" viewbox="0 0 100 100" enable-background="0 0 1000 1000" xml:space="preserve"><A href="#top" id="blue" class="circled scrolly" style="text-decoration:none;"><TEXT x="50" y="50" style="cursor: pointer" fill="#FFFFFF" font-size="15" text-anchor="middle">top</TEXT></A></SVG></CENTER></ARTICLE><ARTICLE class="mix carousel survey human" date="20140920hum"><H2><SPAN class="mw-headline" id="500_survey_responses_reached_.21"> 500 survey responses reachedÂ !</SPAN></H2><H4><SPAN class="mw-headline" id="Saturday.2C_September_20th"> Saturday, September 20th </SPAN></H4><P>We reached 500 responses for our <A href="https://2014.igem.org/Team:ETH_Zurich/human/survey" class="external text" rel="nofollow">survey </A>Â ! If you still want to participate, here are the links in<A href="http://docs.google.com/forms/d/1pvFPVzfH1aiNdy3MdAaaFFzeWysCZyDDybcqgt7LyzY/viewform?usp=send_form" class="external text" rel="nofollow"><B> English</B></A>, <A href="http://docs.google.com/forms/d/1nLa1qMEAo9QjmIsOJgaN5pEVURDlVDqYKXuOvFnpjW4/viewform?usp=send_form" class="external text" rel="nofollow"><B> German</B></A> and <A href="http://docs.google.com/forms/d/1fQ0ZjKvnr-Ssek9fihlDRhmQsVaG8rwxCrNGHDC2JDU/viewform?usp=send_form" class="external text" rel="nofollow"><B> French</B></A>.
</P><P><CENTER><SVG xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" version="1.1" id="Layer_1" x="50%" y="0px" width="80" height="80" viewbox="0 0 100 100" enable-background="0 0 1000 1000" xml:space="preserve"><A href="#top" id="red" class="circled scrolly" style="text-decoration:none;"><TEXT x="50" y="50" style="cursor: pointer" fill="#FFFFFF" font-size="15" text-anchor="middle">top</TEXT></A></SVG></CENTER></P><P><A class="read-more" href="#week13" id="red"><SVG xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" version="1.1" id="Layer_1" x="50%" y="0px" width="100" height="100" viewbox="0 0 100 100" enable-background="0 0 1000 1000" xml:space="preserve"><TEXT x="50" y="53" style="cursor: pointer" fill="#FFFFFF" font-size="15" text-anchor="middle">Read More</TEXT></SVG></A></P></ARTICLE><ARTICLE class="mix meetings carousel integrase" id="week17" date="20140528"><H2><SPAN class="mw-headline" id="Week_17_:_Integrase.2C_again_and_again"> Week 17Â : Integrase, again and again </SPAN></H2><H4><SPAN class="mw-headline" id="Monday.2C_September_15th"> Monday, September 15th </SPAN></H4><P>After characterizing exhaustively crosstalk, it was more than time to begin with the integrases experiments. However, all trials in different experimental set-ups were negative. The construct used was built by our wetlab team. After trying to determine what the difference was between the published plasmid by Bonnet<SUP><A href="/Team:ETH_Zurich/project/references" title="Team:ETH Zurich/project/references">[9]</A></SUP> and ours. No switching was observed after 20 hours with 1% arabinose.
</P><P>A diffusion assay with LuxI producers and LuxAHL receivers was done.
</P><P>In the model, the description of the quorum sensing module went on.
</P><P><A class="read-more" href="#{{{2}}}" id="silver"><SVG xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" version="1.1" id="Layer_1" x="50%" y="0px" width="100" height="100" viewbox="0 0 100 100" enable-background="0 0 1000 1000" xml:space="preserve"><TEXT x="50" y="53" style="cursor: pointer" fill="#FFFFFF" font-size="15" text-anchor="middle">Read More</TEXT></SVG></A></P><CENTER><SVG xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" version="1.1" id="Layer_1" x="50%" y="0px" width="80" height="80" viewbox="0 0 100 100" enable-background="0 0 1000 1000" xml:space="preserve"><A href="#top" id="silver" class="circled scrolly" style="text-decoration:none;"><TEXT x="50" y="50" style="cursor: pointer" fill="#FFFFFF" font-size="15" text-anchor="middle">top</TEXT></A></SVG></CENTER></ARTICLE><ARTICLE class="mix human carousel hpcollaboration" date="20140915"><H2><SPAN class="mw-headline" id="Human_Practice_Collaboration"> Human Practice Collaboration </SPAN></H2><H4><SPAN class="mw-headline" id="Saturday.2C_September_15th"> Saturday, September 15th </SPAN></H4><P>We are participating to a project of weekly newsletter launched by Paris Bettencourt. We are giving updates on our project every week and other articles such as description of one member of the team. The idea is also to ask questions to other teams on their projects in order to get feedback.
</P><P><CENTER><SVG xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" version="1.1" id="Layer_1" x="50%" y="0px" width="80" height="80" viewbox="0 0 100 100" enable-background="0 0 1000 1000" xml:space="preserve"><A href="#top" id="red" class="circled scrolly" style="text-decoration:none;"><TEXT x="50" y="50" style="cursor: pointer" fill="#FFFFFF" font-size="15" text-anchor="middle">top</TEXT></A></SVG></CENTER></P><P><A class="read-more" href="#interview1" id="red"><SVG xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" version="1.1" id="Layer_1" x="50%" y="0px" width="100" height="100" viewbox="0 0 100 100" enable-background="0 0 1000 1000" xml:space="preserve"><TEXT x="50" y="53" style="cursor: pointer" fill="#FFFFFF" font-size="15" text-anchor="middle">Read More</TEXT></SVG></A></P></ARTICLE><ARTICLE class="mix human carousel interviews" id="interview1" date="20140510"><H2><SPAN class="mw-headline" id="Discussions_with_experts"> Discussions with experts </SPAN></H2><H4><SPAN class="mw-headline" id="Tuesday.2C_September_9th"> Tuesday, September 9th </SPAN></H4><H3><SPAN class="mw-headline" id="Complexity_in_town_planning_and_complexity_in_religion">Complexity in town planning and complexity in religion</SPAN></H3><P>In the last days Stefanie met an architect and a priest to discuss complexity in their occupational field. Both conversations were highly interesting and at the same very different. It was fascinating and inspiring to once change the point of view and to approach complexity from new positions.
For the priest, Josef Fuisz, the struggle of men with complexity might be represented by the following psalm (42.5):
</P><P>
Why, my soul, are you downcast?â¨    
</P><P>Why so disturbed within me?
â¨
</P><P>Put your hope in God,â¨    
</P><P>for I will yet praise him,â¨    
</P><P>my Savior and my God.
</P><P>Other interesting interviews were done in various fields. For example a philosophy teacher, a physicist, a nano-technologist and a theoretical computer scientist were partners for talks. From these interviews we expect to gain insight into the diversity of complexity. It became even more clear than before that complexity can be found in many different fields. 
</P><P>Visit <A href="https://2014.igem.org/Team:ETH_Zurich/human/interviews" class="external text" rel="nofollow">our interview page</A> for the details of each conversation. 

<CENTER><SVG xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" version="1.1" id="Layer_1" x="50%" y="0px" width="80" height="80" viewbox="0 0 100 100" enable-background="0 0 1000 1000" xml:space="preserve"><A href="#top" id="red" class="circled scrolly" style="text-decoration:none;"><TEXT x="50" y="50" style="cursor: pointer" fill="#FFFFFF" font-size="15" text-anchor="middle">top</TEXT></A></SVG></CENTER></P><P><A class="read-more" href="#interview1" id="red"><SVG xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" version="1.1" id="Layer_1" x="50%" y="0px" width="100" height="100" viewbox="0 0 100 100" enable-background="0 0 1000 1000" xml:space="preserve"><TEXT x="50" y="53" style="cursor: pointer" fill="#FFFFFF" font-size="15" text-anchor="middle">Read More</TEXT></SVG></A></P></ARTICLE><ARTICLE class="mix carousel meetings" date="20140908" id="week16"><H2><SPAN class="mw-headline" id="Week_16_:_choosing_an_AHL_pair.2C_improving_diffusion"> Week 16Â : choosing an AHL pair, improving diffusion </SPAN></H2><H4><SPAN class="mw-headline" id="Monday_8th_September"> Monday 8th September </SPAN></H4><UL><LI> We have enough crosstalk results to decide which pair of quorum sensing systems we will useÂ : Las and Lux. See  <A id="insidelink" targetid="ctresults" href="https://2014.igem.org/Team:ETH_Zurich/blog#ctresults"> this article </A>  for more details.
</LI></UL><UL><LI> Diffusion experiment (trying to have a correlation between channel length and time of fluorescence, having cells at the end of a channel and adding pure HSL at the other end of the channel, see  <A id="insidelink" targetid="diffusionexp" href="https://2014.igem.org/Team:ETH_Zurich/blog#diffusionexp"> this article </A>  for more details on the setup)Â : We tried lower concentrations of pure HSL and the wells start shining at different moments, but we still don't have any correlation between channel length and starting time of fluorescence, because our chip seems to be leaky and some wells dried completely. However, this experiment enables us to check that fluorescence appears after a reasonable time and that channel length probably doesn't influence this time too much. 
</LI></UL><UL><LI> On the modeling side, we are not able to retrieve diffusion coefficient of HSL from this diffusion experiment but we can use a parameter from literature.
</LI></UL><P><CENTER><SVG xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" version="1.1" id="Layer_1" x="50%" y="0px" width="80" height="80" viewbox="0 0 100 100" enable-background="0 0 1000 1000" xml:space="preserve"><A href="#top" id="silver" class="circled scrolly" style="text-decoration:none;"><TEXT x="50" y="50" style="cursor: pointer" fill="#FFFFFF" font-size="15" text-anchor="middle">top</TEXT></A></SVG></CENTER></P><P><A class="read-more" href="#week16" id="silver"><SVG xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" version="1.1" id="Layer_1" x="50%" y="0px" width="100" height="100" viewbox="0 0 100 100" enable-background="0 0 1000 1000" xml:space="preserve"><TEXT x="50" y="53" style="cursor: pointer" fill="#FFFFFF" font-size="15" text-anchor="middle">Read More</TEXT></SVG></A></P></ARTICLE><ARTICLE class="mix lab quorum carousel" id="ctresults" date="20140825"><H2><SPAN class="mw-headline" id="Crosstalk_experiments_continued"> Crosstalk experiments continued </SPAN></H2><H4><SPAN class="mw-headline" id="Monday.2C_6th_September"> Monday, 6th September </SPAN></H4><P>Until now we thought Lux was crosstalking too much with las and tried to improve the pair Las-Rhl. However we realized today that Rhl is difficult to improve and that our induction curves of Las and Lux are good enough, since Las is never responding to lux-HSL and we only have to be in a range of las-HSL concentrations that is below 10 nM to avoid Lux responding to las-HSL.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:802px;"><DIV class="thumbcaption">Summary of results from cross-talk experiments</DIV></DIV></DIV></DIV><P>Cross-talk experiments setup is described in  <A targetid="cross-talk" id="insidelink" href="https://2014.igem.org/Team:ETH_Zurich/blog#cross-talk"> this article </A> . 
</P><P><CENTER><SVG xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" version="1.1" id="Layer_1" x="50%" y="0px" width="80" height="80" viewbox="0 0 100 100" enable-background="0 0 1000 1000" xml:space="preserve"><A href="#top" id="green" class="circled scrolly" style="text-decoration:none;"><TEXT x="50" y="50" style="cursor: pointer" fill="#FFFFFF" font-size="15" text-anchor="middle">top</TEXT></A></SVG></CENTER></P><P><A class="read-more" href="#diffusionexp" id="green"><SVG xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" version="1.1" id="Layer_1" x="50%" y="0px" width="100" height="100" viewbox="0 0 100 100" enable-background="0 0 1000 1000" xml:space="preserve"><TEXT x="50" y="53" style="cursor: pointer" fill="#FFFFFF" font-size="15" text-anchor="middle">Read More</TEXT></SVG></A></P></ARTICLE><ARTICLE class="mix carousel meetings" date="20140901" id="week15"><H2><SPAN class="mw-headline" id="Week_15_:_Integrase_parameters_retrieved_from_Bonnet.27s_paper.2C_problems_with_integrase_experiments"> Week 15Â : Integrase parameters retrieved from Bonnet's paper, problems with integrase experiments</SPAN></H2><H4><SPAN class="mw-headline" id="Monday_1st_September"> Monday 1st September </SPAN></H4><UL><LI> We finally managed to retrieve integrase parameters from Bonnet et al.'s <SUP><A href="/Team:ETH_Zurich/project/references#refEmergence" title="Team:ETH Zurich/project/references">[9]</A></SUP> paper steady state behavior curve. You can check   <A targetid="intparam"> the corresponding article </A>   for more details. We can get Hill function parameters this way, but still need dynamic data from our own experiments to retrieve dynamic parameters (i. e. degradation rates).
</LI></UL><UL><LI> Integrases don't seem to switch the terminator at the moment, maybe because of a mutation that we found in their sequence.
</LI></UL><P><CENTER><SVG xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" version="1.1" id="Layer_1" x="50%" y="0px" width="80" height="80" viewbox="0 0 100 100" enable-background="0 0 1000 1000" xml:space="preserve"><A href="#top" id="silver" class="circled scrolly" style="text-decoration:none;"><TEXT x="50" y="50" style="cursor: pointer" fill="#FFFFFF" font-size="15" text-anchor="middle">top</TEXT></A></SVG></CENTER></P><P><A class="read-more" href="#week15" id="silver"><SVG xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" version="1.1" id="Layer_1" x="50%" y="0px" width="100" height="100" viewbox="0 0 100 100" enable-background="0 0 1000 1000" xml:space="preserve"><TEXT x="50" y="53" style="cursor: pointer" fill="#FFFFFF" font-size="15" text-anchor="middle">Read More</TEXT></SVG></A></P></ARTICLE><ARTICLE class="mix modeling intmodule intmodule carousel parameter" id="intparam" date="20140829"><H2><SPAN class="mw-headline" id="Integrase_parameters"> Integrase parameters </SPAN></H2><H4><SPAN class="mw-headline" id="Friday.2C_August_29th"> Friday, August 29th </SPAN></H4><P>In order to predict accurately the experiments we designed, we defined two different strategies to retrieve parameters for the integrase system. 
</P><P>
As we did not find these parameters on the web, we used the data given by Bonnet et al.'s paper &quot;Amplifying genetic logic gates&quot; <SUP><A href="/Team:ETH_Zurich/project/references#refEmergence" title="Team:ETH Zurich/project/references">[9]</A></SUP> . The figure S4 is particularly interesting, even if it does not give any information on the dynamics of the system. We modeled and wrote the equations corresponding to Bonnet's experiment. 
</P><P>
At first, we try to fit the dynamic parameters of the system with only the steady states values.  To fit the parameters with the data, we first try to use a minimum search for the difference between the data and the parametric function. Then, using these first approximations, we tried to do a Markov Chain Monte Carlo approach to find a more suited curve. As these two methods are strongly prior dependent, we planned another step of screening in order to have meaningful results. However, our knowledge on the parameters, the number of data points compared to the high number of parameters did not give any meaningful results.
</P><P>
We then decided to simplify our systemÂ :
</P><UL><LI> We consider that Bxb1a and Bxb1b have the same behavior. Therefore, they don't appear as differentiated in our system anymore.
</LI><LI> The paper uses the tet system to induce their system. This was modeled explicitly before. We approximated this part to a Hill function, which takes leakiness into consideration. The parameters for the Hill function are the result from a study of an other iGEM team UCSF from last year. Please visit  <A class="all" href="https://2013.igem.org/Team:UCSF/Modeling"> their page </A> for more information. The same biobrick is indeed used in their work and in Bonnet's paper.
</LI><LI> We consider that the dimer of the integrase and the binding to DNA were at quasi steady-state.
</LI></UL><P>
As we tried to fit the steady state, we obtained two different models, depending on which assumption we made. As these two models fit the Bonnet's curve well, we used them both to derive more precisely the dissociation constant of the dimerization reaction and the one of the DNA binding reaction. Each of them appear in one of the lumped parameters we fitted.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:702px;"><DIV class="thumbcaption">Fitting of the figure S4 from Bonnet's paper with the model 1. The assumption is a simplification of the expression of [Bxb1] after the derivation of the steady state.</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:702px;"><DIV class="thumbcaption">Fitting of the figure S4 from Bonnet's paper with the model 1. The assumption is the fact that the degradation of the dimer of Bxb1 is neglected with respect to the dimerization reaction rate.</DIV></DIV></DIV></DIV><P>By considering reasonable order of magnitude for other parameters present in the lumped parameters, we find that the value of the dissociation constants are both of the order of 10<SUP>-4</SUP>. This order of magnitude is not surprising at all, as we know that the binding of integrases to DNA is very specific (which is underlined by the low value of the dissociation constant of this reaction) and that integrases are more stable as dimer than as monomer.
</P><P><CENTER><SVG xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" version="1.1" id="Layer_1" x="50%" y="0px" width="80" height="80" viewbox="0 0 100 100" enable-background="0 0 1000 1000" xml:space="preserve"><A href="#top" id="blue" class="circled scrolly" style="text-decoration:none;"><TEXT x="50" y="50" style="cursor: pointer" fill="#FFFFFF" font-size="15" text-anchor="middle">top</TEXT></A></SVG></CENTER></P><P><A class="read-more" href="#intparam" id="blue"><SVG xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" version="1.1" id="Layer_1" x="50%" y="0px" width="100" height="100" viewbox="0 0 100 100" enable-background="0 0 1000 1000" xml:space="preserve"><TEXT x="50" y="53" style="cursor: pointer" fill="#FFFFFF" font-size="15" text-anchor="middle">Read More</TEXT></SVG></A></P></ARTICLE><ARTICLE class="mix lab milli carousel" id="diffusionexp" date="20140825"><H2><SPAN class="mw-headline" id="Diffusion_Experiments"> Diffusion Experiments </SPAN></H2><H4><SPAN class="mw-headline" id="Monday.2C_25th_August"> Monday, 25th August </SPAN></H4><P>Now that we tested the different constructs for leakiness, dose response and cross talk, we are investigating in the diffusion rate of AHL. Therefore we designed a chip with chambers connected by channels of different length (see figure). In one chamber we added a strain containing a regulator and a sensor plasmid, to the second chamber we added AHL or an AHL producer strain. As soon as AHL is diffused to the former chamber it induces sfGFP production on the sensor plasmid. The time it takes until we can measure the sfGFP signal and the length of the channel gives us information about the velocity of AHL diffusion.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">PDMS millifluidic chip used for diffusion experiments</DIV></DIV></DIV></DIV><P>Three main experiment designs were tested and modified: the agar-design, the liquid-design and the beads-design. 
</P><UL><LI> For the agar-design we filled the chamber and channels with LB-agar or in a second attempt with agarose only. Subsequently, we punched a cylindric hole in each chamber. The capacities were filled with regulator/sensor strain culture or AHL/AHL producer strain culture, respectively. After 3 to 4 h at 37 Â°C we could observe an increase in the sfGFP signal. Since we used comparably high concentration of AHL in our first attempts, we could not detect a delayed sfGFP signal for chambers connected by a longer channel. The experiment will be repeated using lower concentrations. However, conducting the experiment using the agar-design, we encountered a problem: after some hours the LB-agar/agarose started to shrink and dried out. Thus, the agar-design cannot be used without modifications for experiments of longer duration. 
</LI><LI> In an attempt we only left a small agar fragment in each channel and filled the rest with liquid medium, so as to decrease chance of it drying out, while avoiding at the same time that bacteria from one chamber reach the other chamber. 
</LI><LI> The bead-design consists of an alginate bead for each chamber and liquid medium in between. The beads either contained regulator/sensor-bacteria or AHL.
</LI></UL><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Diffusion experiment with agar. Click <A href="https://2014.igem.org/Team:ETH_Zurich/Video1" class="external text" rel="nofollow">here</A> to watch the video.</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Diffusion experiment with liquid. Click <A href="https://2014.igem.org/Team:ETH_Zurich/Video2" class="external text" rel="nofollow">here</A> to watch the video.</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Diffusion experiment with beads. Click <A href="https://2014.igem.org/Team:ETH_Zurich/Video3" class="external text" rel="nofollow">here</A> to watch the video.</DIV></DIV></DIV></DIV><P><CENTER><SVG xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" version="1.1" id="Layer_1" x="50%" y="0px" width="80" height="80" viewbox="0 0 100 100" enable-background="0 0 1000 1000" xml:space="preserve"><A href="#top" id="green" class="circled scrolly" style="text-decoration:none;"><TEXT x="50" y="50" style="cursor: pointer" fill="#FFFFFF" font-size="15" text-anchor="middle">top</TEXT></A></SVG></CENTER></P><P><A class="read-more" href="#diffusionexp" id="green"><SVG xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" version="1.1" id="Layer_1" x="50%" y="0px" width="100" height="100" viewbox="0 0 100 100" enable-background="0 0 1000 1000" xml:space="preserve"><TEXT x="50" y="53" style="cursor: pointer" fill="#FFFFFF" font-size="15" text-anchor="middle">Read More</TEXT></SVG></A></P></ARTICLE><ARTICLE class="mix carousel meetings" id="week14" date="20140825"><H2><SPAN class="mw-headline" id="Week_14_:_Diffusion_experiments_started.2C_simulated_with_Comsol.2C_and_fitting_of_quorum_sensing_dynamics"> Week 14Â : Diffusion experiments started, simulated with Comsol, and fitting of quorum sensing dynamics </SPAN></H2><H4><SPAN class="mw-headline" id="Monday.2C_August_25th"> Monday, August 25th </SPAN></H4><UL><LI> We carried diffusion experiment with different designsÂ : punched PDMS, liquid, beads. For every design we added high concentrations of pure AHL in the sender well, so cells are glowing a lot. However because of this high concentration, diffusion is very high and at a given AHL concentration, receiving cells are glowing at the same time for all 4 channel lengths. Check   <A id="insidelink" targetid="diffusionexp" href="https://2014.igem.org/Team:ETH_Zurich/blog#diffusionexp"> the corresponding article </A>   for more details.
</LI></UL><UL><LI>We have a Comsol simulation for these experiments that explains why at this concentration of AHL (1mM), all 4 wells glow at the same time irrespectively of channel length. Check  <A id="insidelink" targetid="diffcomsol" href="https://2014.igem.org/Team:ETH_Zurich/blog#diffcomsol"> this article </A>   for more details.
</LI></UL><UL><LI> We could fit degradation rates of external, internal AHL and GFP thanks to quorum sensing dynamic curves.
</LI></UL><P>Check  <A id="insidelink" targetid="qsdynamics" href="https://2014.igem.org/Team:ETH_Zurich/blog#qsdynamics"> the corresponding article </A>  for more details.
</P><P><CENTER><SVG xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" version="1.1" id="Layer_1" x="50%" y="0px" width="80" height="80" viewbox="0 0 100 100" enable-background="0 0 1000 1000" xml:space="preserve"><A href="#top" id="silver" class="circled scrolly" style="text-decoration:none;"><TEXT x="50" y="50" style="cursor: pointer" fill="#FFFFFF" font-size="15" text-anchor="middle">top</TEXT></A></SVG></CENTER></P><P><A class="read-more" href="#{{{2}}}" id="silver"><SVG xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" version="1.1" id="Layer_1" x="50%" y="0px" width="100" height="100" viewbox="0 0 100 100" enable-background="0 0 1000 1000" xml:space="preserve"><TEXT x="50" y="53" style="cursor: pointer" fill="#FFFFFF" font-size="15" text-anchor="middle">Read More</TEXT></SVG></A></P></ARTICLE><ARTICLE class="mix carousel modeling parameter qsmodule" id="qsdynamics" date="20140824"><H2><SPAN class="mw-headline" id="Fitting_quorum_sensing_dynamic_curves"> Fitting quorum sensing dynamic curves </SPAN></H2><H4><SPAN class="mw-headline" id="Monday.2C_August_24th"> Monday, August 24th </SPAN></H4><P>We could use quorum sensing dynamic curves to fit degradation rates of external, internal AHL and GFP. We fitted the experimental results for lux with riboregulator and thus didn't consider leakiness (we showed leakiness was low enough in that case when fitting <A targetid="qsparameters" id="insidelink" href="https://2014.igem.org/Team:ETH_Zurich/blog#qsparameters"> quorum sensing Hill function parameters </A> ). By writing GFP production rate and replacing AHL in this equation by $$[AHL]=[AHL_i]-[AHL_{bound}]$$ we find 
$$\frac{d[GFP]}{dt}=\frac{k_{45}[P_{tot}]\Big(-1+\sqrt{1+8\frac{AHL_i}{K_{d3}}}\Big)^2}{16\frac{K_{d4}}{K_{d3}}+\Big(-1+\sqrt{1+8\frac{AHL_i}{K_{d3}}}\Big)^2}-d_{GFP}[GFP]$$.
For very high initial concentrations AHLi, we have 
$$\frac{d[GFP]}{dt}=k_{45}[P_{tot}]-d_{GFP}[GFP]$$
</P><PRE>so by taking 
</PRE><P>$$t_{1/2}=\frac{ln(2)}{d_{GFP}}$$
</P><P>from experimental curves, we are able to retrieve 
</P><P>$$d_{GFP} = 4.9 . 10^{-3} min^{-1}$$.
For very low initial concentrations of initial AHL and considering degradation, we have 
</P><P>$$ \frac{d[GFP]}{dt}=\frac{k_{45}[P_{tot}][AHLi]^2e^{-2d_{AHL}t}}{K_{d4}K_{d3}}-d_{GFP}[GFP].$$
</P><PRE>We find 
</PRE><P>$$ [GFP]=\frac{k_{45}[P_{tot}][AHLi]^2}{K_{d4}K_{d3}(d_{GFP}-2d_{AHL})}(e^{-2d_{AHL}t}-e^{-d_{GFP}t}).$$ 
</P><PRE>This curve has a maximum at 
</PRE><P>$$ t_{max}=\frac{1}{d_{GFP}-2d_{AHL}}ln\big(\frac{d_{GFP}}{2d_{AHL}}\big).$$
</P><P>This way we can find from experimental curves $$d_{AHL}=4,0.10^{-3} min^{-1}$$
</P><P>This AHL degradation rate is alumped parameter between internal and external degradation rates, equivalent to
</P><P>$$d_{AHLext}+\alpha d_{AHLint}$$ 
</P><P>where Î± is the fraction of the volume occupied by cells in the whole culture. From that on we could find reasonable values for internal and external AHL degradation rates. 
</P><P>Finally the curves simulated by our model with these parameters fit the experiments quite well:
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:802px;"><DIV class="thumbcaption">Fitting of quorum sensing dynamic parameters.</DIV></DIV></DIV></DIV><P><CENTER><SVG xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" version="1.1" id="Layer_1" x="50%" y="0px" width="80" height="80" viewbox="0 0 100 100" enable-background="0 0 1000 1000" xml:space="preserve"><A href="#top" id="blue" class="circled scrolly" style="text-decoration:none;"><TEXT x="50" y="50" style="cursor: pointer" fill="#FFFFFF" font-size="15" text-anchor="middle">top</TEXT></A></SVG></CENTER></P><P><A class="read-more" href="#{{{2}}}" id="blue"><SVG xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" version="1.1" id="Layer_1" x="50%" y="0px" width="100" height="100" viewbox="0 0 100 100" enable-background="0 0 1000 1000" xml:space="preserve"><TEXT x="50" y="53" style="cursor: pointer" fill="#FFFFFF" font-size="15" text-anchor="middle">Read More</TEXT></SVG></A></P></ARTICLE><ARTICLE class="mix carousel modeling diffusion" id="diffcomsol" date="20140822"><H2><SPAN class="mw-headline" id="Comsol_simulation_for_diffusion_experiments"> Comsol simulation for diffusion experiments </SPAN></H2><H4><SPAN class="mw-headline" id="Monday.2C_August_22nd"> Monday, August 22nd </SPAN></H4><P>We have a Comsol simulation for diffusion experiments that explains why at 1mM AHL concentration in sender wells, all 4 wells glow at the same time irrespectively of channel length. On the other hand if we use Î¼ , the wells should start glowing at different times. The channels are 1 to 4mm long. We used a Hill function to model activation of GFP in the receiver wells.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:902px;"><DIV class="thumbcaption">Comsol simulation of diffusion experiments for 1mM and for 1 Î¼M AHL initial concentration in sender wells. It explains why with 1mM all wells start glowing at the same time.</DIV></DIV></DIV></DIV><P><CENTER><SVG xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" version="1.1" id="Layer_1" x="50%" y="0px" width="80" height="80" viewbox="0 0 100 100" enable-background="0 0 1000 1000" xml:space="preserve"><A href="#top" id="blue" class="circled scrolly" style="text-decoration:none;"><TEXT x="50" y="50" style="cursor: pointer" fill="#FFFFFF" font-size="15" text-anchor="middle">top</TEXT></A></SVG></CENTER></P><P><A class="read-more" href="#{{{2}}}" id="blue"><SVG xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" version="1.1" id="Layer_1" x="50%" y="0px" width="100" height="100" viewbox="0 0 100 100" enable-background="0 0 1000 1000" xml:space="preserve"><TEXT x="50" y="53" style="cursor: pointer" fill="#FFFFFF" font-size="15" text-anchor="middle">Read More</TEXT></SVG></A></P></ARTICLE><ARTICLE class="mix meetings carousel survey" id="week13" date="20140820"><H2><SPAN class="mw-headline" id="Week_13_:_Steady_state_quorum_sensing_fitted_and_human_practice_survey_launched_.21"> Week 13Â : Steady state quorum sensing fitted and human practice survey launchedÂ !</SPAN></H2><H4><SPAN class="mw-headline" id="Wednesday.2C_August_20th"> Wednesday, August 20th </SPAN></H4><UL><LI> Our first diffusion try with highlighter inkÂ !
</LI></UL><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Camera picture of our punched agar chip glowing highlighter ink</DIV></DIV></DIV></DIV><P>The design of punched agar doesn't seem to be the best but at least we can see something. Diffusion experiments with different designs are running now.
</P><UL><LI> We have been fitting quorum sensing experiments at steady state to a Hill function and found some quorum sensing parameters.
</LI></UL><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:702px;"><DIV class="thumbcaption">Fitting quorum sensing Hill function parameters thanks to steady-state data from experiments. Check  <A targetid="qsparameters" id="insidelink" href="https://2014.igem.org/Team:ETH_Zurich/blog#qsparameters"> the corresponding article </A>  for more details.</DIV></DIV></DIV></DIV><UL><LI> We wrote a <A href="https://docs.google.com/forms/d/1pvFPVzfH1aiNdy3MdAaaFFzeWysCZyDDybcqgt7LyzY/viewform" class="external text" rel="nofollow">survey in English</A>, <A href="https://docs.google.com/forms/d/1fQ0ZjKvnr-Ssek9fihlDRhmQsVaG8rwxCrNGHDC2JDU/viewform" class="external text" rel="nofollow">in French</A> and <A href="https://docs.google.com/forms/d/1nLa1qMEAo9QjmIsOJgaN5pEVURDlVDqYKXuOvFnpjW4/viewform" class="external text" rel="nofollow">in German</A> on complexity for our human practice project. We expect 500 responses.
</LI></UL><P><CENTER><SVG xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" version="1.1" id="Layer_1" x="50%" y="0px" width="80" height="80" viewbox="0 0 100 100" enable-background="0 0 1000 1000" xml:space="preserve"><A href="#top" id="silver" class="circled scrolly" style="text-decoration:none;"><TEXT x="50" y="50" style="cursor: pointer" fill="#FFFFFF" font-size="15" text-anchor="middle">top</TEXT></A></SVG></CENTER></P><P><A class="read-more" href="#week13" id="silver"><SVG xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" version="1.1" id="Layer_1" x="50%" y="0px" width="100" height="100" viewbox="0 0 100 100" enable-background="0 0 1000 1000" xml:space="preserve"><TEXT x="50" y="53" style="cursor: pointer" fill="#FFFFFF" font-size="15" text-anchor="middle">Read More</TEXT></SVG></A></P></ARTICLE><ARTICLE class="mix modeling parameter qsmodule carousel" id="qsparameters" date="20140818"><H2><SPAN class="mw-headline" id="Fitting_quorum_sensing_Hill_functions_parameters"> Fitting quorum sensing Hill functions parameters </SPAN></H2><H4><SPAN class="mw-headline" id="Wednesday.2C_August_18th"> Wednesday, August 18th </SPAN></H4><P>The plasmids for quorum sensing experiments concerning the Lux system are made and operational to be used for experiments. We have three different plasmids:
</P><UL><LI> Lux promoter folowed by GFP gene
</LI><LI> Lux promoter followed by cisRNA and then by GFP
</LI><LI> Lux promoter controlling the expression of taRNA and cisRNA. The cisRNA coding gene is followed by GFP.
</LI></UL><P>The third plasmid corresponds to a riboregulating system. The second plasmid allowed us to validate the crucial role of taRNA in this riboregulating system.
</P><P> 
On the modeling point of view, we only tried to fit the data coming from experiments done with plasmid one (with no riboregulating system) and plasmid three (with an effective riboregulating system). They were the only one to show an answer to a change of input that was not on the noise level.
</P><P>
The experimental setting was to inject different concentration of the signaling molecule AHL at the same time point into different bacterial colonies. The factor that we want to observe is the strength of the activation of the Lux promoter. The read-out is made by the fluorescence. The data was gathered over several time points in order to be able to observe the dynamic behavior. More precisely, the read-out of the promoter activation is made by the relative fluorescence, which corresponds to the ratio of the absolute fluorescence over the Optical Density (OD). We consider that the concentration in GFP per cell is proportional to the relative fluorescence. As we try to avoid fitting the saturation events, we chose to normalize with respect to the highest value before a certain amount of time, which corresponds for each experiment to the apparition of the saturation.
</P><P>
As a first approach, we only try to fit the steady state value. As the signaling molecule is inducing the production of GFP, we chose to use an Hill function to fit the function [GFP] = f([AHL]) at steady state. 
</P><P>
If Hill functions fit these two curves good, the parameters used are surprising. For our system, we suppose that AHL binds to the regulator protein LuxR. This complex called Rlux dimerizes before binding to the promoter to induce its expression. Thus, the dimerization event makes cooperativity emerge from AHL molecules. It is translated for a Hill function by n &gt; 1. In this particular case, one could expect to find n=2 (as it is a dimerization event). Nevertheless, the n found are on the order of magnitude of 1 (0.89 for the non-riboregulated promoter plasmid and 1.22 for the ribo-regulated promoter plasmid). The data points were collected in triplicates and this allow us to have a look at the variability (namely standard deviation) around each value. Where the switch happens, the variability is higher (it is expected because the output GFP is more sensitive to variation of the input [AHL] around the switching point). Therefore, the inflexion points could need some improvment, by taking more points around this identified sensitive region. Moreover, this critical concentration of AHL was the one that was expected given the litterature and our system. We observe that it needs a higher [AHL] for the riboregulated system to be activated than for the non-riboregulated system. 
</P><P>
We modeled leakiness as an offset in the Hill functionÂ : [GFP] = g([AHL]) = f([AHL]) + aLeakiness. We obtain the expected information that the signal over noise ratio is greater by one order of magnitude for the riboregulated system than for the non-riboregulated system. A drawback from the riboregulated system is that the amplitude of the signal corresponds to the amplitude of the noise in the non-riboregulated system.
</P><P>
The experimental design also allowed us to detect eventual cross-talk with other signaling molecules, the one corresponding to the las system and the one corresponding to the rhl system. No cross-talk was observed with the signaling molecule of the rhl system. However, the las system signaling molecule activated the Lux promoter. We chose to model this cross-talk, which corresponds to the induction of a promoter, by an Hill function. This fitting allowed us to verify that the Hill function was adapted to describe a cross-talk phenomena. The derived parameters indicated that the concentration in the las signaling molecule had to be high in order to have an effect on our system (It was two orders of magnitude away from the needed concentration of AHL to activate the system). Nevertheless, once activated, the effects of these crosstalk are not to be neglected. Check  <A targetid="cross-talk" id="insidelink" href="https://2014.igem.org/Team:ETH_Zurich/blog#cross-talk"> the following article </A>  for more information on the experimental setting.
</P><P> 
These results are promising and we look forward to fit the dynamic data of this system.
</P><P><CENTER><SVG xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" version="1.1" id="Layer_1" x="50%" y="0px" width="80" height="80" viewbox="0 0 100 100" enable-background="0 0 1000 1000" xml:space="preserve"><A href="#top" id="blue" class="circled scrolly" style="text-decoration:none;"><TEXT x="50" y="50" style="cursor: pointer" fill="#FFFFFF" font-size="15" text-anchor="middle">top</TEXT></A></SVG></CENTER></P><P><A class="read-more" href="#qsparameters" id="blue"><SVG xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" version="1.1" id="Layer_1" x="50%" y="0px" width="100" height="100" viewbox="0 0 100 100" enable-background="0 0 1000 1000" xml:space="preserve"><TEXT x="50" y="53" style="cursor: pointer" fill="#FFFFFF" font-size="15" text-anchor="middle">Read More</TEXT></SVG></A></P></ARTICLE><ARTICLE class="mix meetings carousel" id="week12" date="20140813"><H2><SPAN class="mw-headline" id="Week_12_:_Nice_cross-talk_results.2C_problems_with_producer_strains.2C_and_with_synthesis_of_our_Las_riboregulated_sensor"> Week 12Â : Nice cross-talk results, problems with producer strains, and with synthesis of our Las riboregulated sensor </SPAN></H2><H4><SPAN class="mw-headline" id="Wednesday.2C_August_13th"> Wednesday, August 13th </SPAN></H4><UL><LI> We have a lot of results for cross-talk experiments. Check  <A targetid="cross-talk" id="insidelink" href="https://2014.igem.org/Team:ETH_Zurich/blog#cross-talk"> the corresponding article </A>  for more details. For the moment lux promoter doesn't respon to other AHLs. This is a good step towards orthogonality.
</LI></UL><UL><LI> Current challengesÂ : 
</LI><LI> AHL producer strains are not producing enough. We have to improve the RBS.
</LI><LI> GenScript has problems to produce our Las sensor with riboregulator RR42.
</LI></UL><UL><LI> We have a project descriptionÂ !
</LI></UL><P><CENTER><SVG xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" version="1.1" id="Layer_1" x="50%" y="0px" width="80" height="80" viewbox="0 0 100 100" enable-background="0 0 1000 1000" xml:space="preserve"><A href="#top" id="silver" class="circled scrolly" style="text-decoration:none;"><TEXT x="50" y="50" style="cursor: pointer" fill="#FFFFFF" font-size="15" text-anchor="middle">top</TEXT></A></SVG></CENTER></P><P><A class="read-more" href="#week12" id="silver"><SVG xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" version="1.1" id="Layer_1" x="50%" y="0px" width="100" height="100" viewbox="0 0 100 100" enable-background="0 0 1000 1000" xml:space="preserve"><TEXT x="50" y="53" style="cursor: pointer" fill="#FFFFFF" font-size="15" text-anchor="middle">Read More</TEXT></SVG></A></P></ARTICLE><ARTICLE class="mix lab quorum carousel" date="20140811" id="cross-talk"><H2><SPAN class="mw-headline" id="First_cross_talk_experiments"> First cross talk experiments </SPAN></H2><H4><SPAN class="mw-headline" id="Friday.2C_11th_August"> Friday, 11th August </SPAN></H4><P>After having constructed the necessary regulator and sensor plasmids we were able to start first cross talk experiments. Therefore we transformed competent cells with different combinations of regulator and sensor plasmids. One example would be the strain siG0024 which contains the regulator plasmid piG0041 producing LuxR under a constitutive promoter and the sensor plasmid piG0051 that encodes for sfGFP. The latter is under control of a pluxR promoter, hence it will be transcribed in the presence of LuxR and Lux.
To assess crosstalk and leakiness we conducted multiple experiments: after adding one of the three AHLs to the cultures we analyzed the raise in sfGFP production using a microtiterplate reader. In a first step we measured the dose response of constructs with different ribosomal binding sites (RBSs) and with or without riboregulator. We could observe differences not only in sensitivity, but also in leakiness; some strains expressed GFP even without AHL induction.
To give an example: we evaluated the sensitivity of the siG0024 strain to different concentrations of 3OC<SUB>6</SUB>-HSL (LuxI product) and compared its characteristics to  those of other constructs. Additionally, we measured the sensitivity of siG0024 to 3OC<SUB>12</SUB>-HSL (LasI product) and C<SUB>4</SUB>-HSL (RhlI product). Indeed, we could observe the induction of sfGFP transcription by comparably low levels of 3OC<SUB>12</SUB>-HSL. Crosstalk can be observed between the Las and the Lux system. To simulate a situation most similar to the final experiment, we added the diluted supernatant of AHL producer cells to the test cultures. However, we could not detect any GFP production as reaction to the supernatant. We assume that the AHL concentration is not high enough to induce transcription. By constructing stronger RBSs on the producer plasmids we aim at increasing the AHL production.
All these informations we will use to design a system tailored to our plans. 
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:792px;"><DIV class="thumbcaption">Summary of cross-talk experiments results</DIV></DIV></DIV></DIV><P>The figure shows the dose response of xxx to 3OC<SUB>12</SUB>-HSL, 3OC<SUB>6</SUB>-HSL and C<SUB>4</SUB>-HSL. Both 3OC<SUB>12</SUB>-HSL and 3OC<SUB>6</SUB>-HSL induce production of GFP, although at different concentrations. This is an example for cross talk between the lux and the las system.
</P><P><CENTER><SVG xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" version="1.1" id="Layer_1" x="50%" y="0px" width="80" height="80" viewbox="0 0 100 100" enable-background="0 0 1000 1000" xml:space="preserve"><A href="#top" id="green" class="circled scrolly" style="text-decoration:none;"><TEXT x="50" y="50" style="cursor: pointer" fill="#FFFFFF" font-size="15" text-anchor="middle">top</TEXT></A></SVG></CENTER></P><P><A class="read-more" href="#cross-talk" id="green"><SVG xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" version="1.1" id="Layer_1" x="50%" y="0px" width="100" height="100" viewbox="0 0 100 100" enable-background="0 0 1000 1000" xml:space="preserve"><TEXT x="50" y="53" style="cursor: pointer" fill="#FFFFFF" font-size="15" text-anchor="middle">Read More</TEXT></SVG></A></P></ARTICLE><ARTICLE class="mix carousel meetings" id="week11" date="20140806"><H2><SPAN class="mw-headline" id="Week_11_:_PDMS_chip_built.2C_First_quorum_sensing_experiment"> Week 11Â : PDMS chip built, First quorum sensing experiment </SPAN></H2><H4><SPAN class="mw-headline" id="Wednesday.2C_August_6th"> Wednesday, August 6th </SPAN></H4><UL><LI> We have a first draft of our abstract. 
</LI><LI> We have results for our  <A id="insidelink" targetid="firstquorum" href="https://2014.igem.org/Team:ETH_Zurich/blog#firstquorum"> first quorum sensing experiment </A>.
</LI><LI> We have now a proper <A id="insidelink" targetid="firstPDMS" href="https://2014.igem.org/Team:ETH_Zurich/blog#firstPDMS"> PDMS chip </A> for diffusion experiments.
</LI><LI>We have a <A id="insidelink" targetid="GFPqss" href="https://2014.igem.org/Team:ETH_Zurich/blog#GFPqss"> simple GFP production model</A> for simulation of these quorum sensing experiments.
</LI><LI> In the next weeks we will fit quorum sensing parameters thanks to the first quorum sensing experiments and our simple GFP production model, refine this Comsol diffusion model with the right quorum sensing parameters and then fit this diffusion model to diffusion experiments to find the diffusion constant.
</LI></UL><P><CENTER><SVG xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" version="1.1" id="Layer_1" x="50%" y="0px" width="80" height="80" viewbox="0 0 100 100" enable-background="0 0 1000 1000" xml:space="preserve"><A href="#top" id="silver" class="circled scrolly" style="text-decoration:none;"><TEXT x="50" y="50" style="cursor: pointer" fill="#FFFFFF" font-size="15" text-anchor="middle">top</TEXT></A></SVG></CENTER></P><P><A class="read-more" href="#week11" id="silver"><SVG xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" version="1.1" id="Layer_1" x="50%" y="0px" width="100" height="100" viewbox="0 0 100 100" enable-background="0 0 1000 1000" xml:space="preserve"><TEXT x="50" y="53" style="cursor: pointer" fill="#FFFFFF" font-size="15" text-anchor="middle">Read More</TEXT></SVG></A></P></ARTICLE><ARTICLE class="mix lab quorum carousel" date="20140804" id="firstquorum"><H2><SPAN class="mw-headline" id="First_quorum_sensing_experiment"> First quorum sensing experiment </SPAN></H2><H4><SPAN class="mw-headline" id="Monday.2C_August_4th"> Monday, August 4th </SPAN></H4><P>We performed double transformations with different combinations of regulator + sensor plasmids for cross-talk experiments. We have results for our first quorum sensing experimentÂ : fluorescence under control of wildtype lux sensor (s24) over time for different concentrations of AHL. s1 is a negative control.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption">Fluorescence under control of wildtype lux sensor (s24) over time for different concentrations of AHL.</DIV></DIV></DIV></DIV><P><CENTER><SVG xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" version="1.1" id="Layer_1" x="50%" y="0px" width="80" height="80" viewbox="0 0 100 100" enable-background="0 0 1000 1000" xml:space="preserve"><A href="#top" id="green" class="circled scrolly" style="text-decoration:none;"><TEXT x="50" y="50" style="cursor: pointer" fill="#FFFFFF" font-size="15" text-anchor="middle">top</TEXT></A></SVG></CENTER></P><P><A class="read-more" href="#cross-talk" id="green"><SVG xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" version="1.1" id="Layer_1" x="50%" y="0px" width="100" height="100" viewbox="0 0 100 100" enable-background="0 0 1000 1000" xml:space="preserve"><TEXT x="50" y="53" style="cursor: pointer" fill="#FFFFFF" font-size="15" text-anchor="middle">Read More</TEXT></SVG></A></P></ARTICLE><ARTICLE class="mix modeling diffusion carousel" date="20140803" id="GFPqss"><H2><SPAN class="mw-headline" id="GFP_production_model_for_diffusion_simulation"> GFP production model for diffusion simulation </SPAN></H2><H4><SPAN class="mw-headline" id="Sunday.2C_August_3rd"> Sunday, August 3rd </SPAN></H4><P>We need an efficient way to simulate diffusion experiments. A receiver cell at the end of a channel receives AHL produced by the well at the other end of the channel, and starts to produce GFP.
By using quasi steady state aproximations (considering that binding is much faster than transcription and diffusion), we have a simple equation for GFP production as a function of AHLÂ :
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption">Equations and approximations for the model of GFP production by sensor cells used in simulation of diffusion and cross-talk experiments</DIV></DIV></DIV></DIV><P>By using producer cells on one side of the channel and receiver cells in the other side, and putting cells in a punched cylinder in agar, we should see fluorescence appear at different times depending on the channel length (1 to 4mm)Â :
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption">Simulation of diffusion experimentsÂ : fluorescence over time for the punched cylinder design and for 4 different channel lengths</DIV></DIV></DIV></DIV><P><CENTER><SVG xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" version="1.1" id="Layer_1" x="50%" y="0px" width="80" height="80" viewbox="0 0 100 100" enable-background="0 0 1000 1000" xml:space="preserve"><A href="#top" id="blue" class="circled scrolly" style="text-decoration:none;"><TEXT x="50" y="50" style="cursor: pointer" fill="#FFFFFF" font-size="15" text-anchor="middle">top</TEXT></A></SVG></CENTER></P><P><A class="read-more" href="#{{{2}}}" id="blue"><SVG xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" version="1.1" id="Layer_1" x="50%" y="0px" width="100" height="100" viewbox="0 0 100 100" enable-background="0 0 1000 1000" xml:space="preserve"><TEXT x="50" y="53" style="cursor: pointer" fill="#FFFFFF" font-size="15" text-anchor="middle">Read More</TEXT></SVG></A></P></ARTICLE><ARTICLE class="mix lab milli carousel" date="20140804" id="firstPDMS"><H2><SPAN class="mw-headline" id="First_millifluidic_chip"> First millifluidic chip </SPAN></H2><H4><SPAN class="mw-headline" id="Thursday.2C_August_2nd"> Thursday, August 2nd </SPAN></H4><P>Here is our first millifluidic chip for diffusion experimentsÂ ! This time we used a layer of PDMS thin enough so that we could peel it without breaking it. In some wells will be added producer cells or pure AHL, and in the wells on the other side of the channels, receiver cells. There are several lengths of channels, this should enable us to retrieve diffusion parameters by recording when receiver cells start glowing.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Our millifluidic chip for diffusion experiments</DIV></DIV></DIV></DIV><P><CENTER><SVG xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" version="1.1" id="Layer_1" x="50%" y="0px" width="80" height="80" viewbox="0 0 100 100" enable-background="0 0 1000 1000" xml:space="preserve"><A href="#top" id="green" class="circled scrolly" style="text-decoration:none;"><TEXT x="50" y="50" style="cursor: pointer" fill="#FFFFFF" font-size="15" text-anchor="middle">top</TEXT></A></SVG></CENTER></P><P><A class="read-more" href="#cross-talk" id="green"><SVG xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" version="1.1" id="Layer_1" x="50%" y="0px" width="100" height="100" viewbox="0 0 100 100" enable-background="0 0 1000 1000" xml:space="preserve"><TEXT x="50" y="53" style="cursor: pointer" fill="#FFFFFF" font-size="15" text-anchor="middle">Read More</TEXT></SVG></A></P></ARTICLE><ARTICLE class="mix meetings carousel" id="week10" date="20140730"><H2><SPAN class="mw-headline" id="Week_10:_Chip_design.2C_Plasmid_assembly.2C_Bead_production"> Week 10: Chip design, Plasmid assembly, Bead production </SPAN></H2><H4><SPAN class="mw-headline" id="Wednesday.2C_July_30th"> Wednesday, July 30th </SPAN></H4><UL><LI> We designed two possibilities for the chip: either cells laying on agar or punched cylinders in the agar. 
</LI></UL><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:747px;"><DIV class="thumbcaption">Comsol simulations of a part of the future chip, in two different designsÂ : droplets of cells laying on the agar or introduced in a punched cylinder in the agar. Cells are producing AHL which diffuses through the channels.</DIV></DIV></DIV></DIV><UL><LI> 8 out of 21 plasmids were positively sequenced. 7 further plasmids are sent for sequencing.
</LI><LI> We will start cross-talk experiments this week. It should help us choose an orthogonal AHL pair among three different quorum sensing systems.
</LI><LI> We have started to learn bead production.
</LI></UL><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">One of our first alginate beads</DIV></DIV></DIV></DIV><P><CENTER><SVG xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" version="1.1" id="Layer_1" x="50%" y="0px" width="80" height="80" viewbox="0 0 100 100" enable-background="0 0 1000 1000" xml:space="preserve"><A href="#top" id="silver" class="circled scrolly" style="text-decoration:none;"><TEXT x="50" y="50" style="cursor: pointer" fill="#FFFFFF" font-size="15" text-anchor="middle">top</TEXT></A></SVG></CENTER></P><P><A class="read-more" href="#week10" id="silver"><SVG xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" version="1.1" id="Layer_1" x="50%" y="0px" width="100" height="100" viewbox="0 0 100 100" enable-background="0 0 1000 1000" xml:space="preserve"><TEXT x="50" y="53" style="cursor: pointer" fill="#FFFFFF" font-size="15" text-anchor="middle">Read More</TEXT></SVG></A></P></ARTICLE><ARTICLE class="mix carousel meetings" id="week9" date="20140723"><H2><SPAN class="mw-headline" id="Week_9:_First_simulations.2C_Skype_with_Paris_Bettencourt.2C_Chip_fail"> Week 9: First simulations, Skype with Paris Bettencourt, Chip fail  </SPAN></H2><H4><SPAN class="mw-headline" id="Wednesday.2C_July_23rd"> Wednesday, July 23rd </SPAN></H4><UL><LI> We started to simulate diffusion experiments on a Comsol model. We consider cells will lay on agar for the moment.
</LI></UL><P>We will have a chip with these four different channels 1 to 4 mm long.
The model still has to be refined because at the moment it doesn't take into account internal/external concentrations of AHL.
</P><UL><LI> Our first mold didn't enable us to peal the PDMS correctly. We have to make it less deep.
</LI></UL><UL><LI> 5 of 8 regulator plasmids are done. We have to repeat construction of producer plasmids.
</LI><LI> Parameter estimation for integrases is not working at the moment.
</LI><LI> We skyped with Paris Bettencourt for human practice collaboration and are participating to their newsletter project.
</LI></UL><P><CENTER><SVG xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" version="1.1" id="Layer_1" x="50%" y="0px" width="80" height="80" viewbox="0 0 100 100" enable-background="0 0 1000 1000" xml:space="preserve"><A href="#top" id="silver" class="circled scrolly" style="text-decoration:none;"><TEXT x="50" y="50" style="cursor: pointer" fill="#FFFFFF" font-size="15" text-anchor="middle">top</TEXT></A></SVG></CENTER></P><P><A class="read-more" href="#week9" id="silver"><SVG xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" version="1.1" id="Layer_1" x="50%" y="0px" width="100" height="100" viewbox="0 0 100 100" enable-background="0 0 1000 1000" xml:space="preserve"><TEXT x="50" y="53" style="cursor: pointer" fill="#FFFFFF" font-size="15" text-anchor="middle">Read More</TEXT></SVG></A></P></ARTICLE><ARTICLE class="mix meetings carousel" id="week8" date="20140716"><H2><SPAN class="mw-headline" id="Week_8_:_Plasmdis_assembly.2C_First_mold_3D_printed"> Week 8Â : Plasmdis assembly, First mold 3D printed </SPAN></H2><H4><SPAN class="mw-headline" id="Wednesday.2C_July_16th"> Wednesday, July 16th </SPAN></H4><UL><LI> Our first Gibson assemblies worked well. Regulator plasmids piG0040, piG0041 and piG0042 have been constructedÂ :
</LI></UL><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:702px;"><DIV class="thumbcaption">Regulator plasmids construction plan</DIV></DIV></DIV></DIV><UL><LI> We printed our first mold for millifluidic chipÂ :
</LI></UL><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Our first mold for millifluidic chip</DIV></DIV></DIV></DIV><UL><LI> We simulated the whole model with and without riboswitch in Matlab with random parameters for integrases. We are still trying to estimate integrase parameters from the literature but it is very difficult.
</LI></UL><P><CENTER><SVG xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" version="1.1" id="Layer_1" x="50%" y="0px" width="80" height="80" viewbox="0 0 100 100" enable-background="0 0 1000 1000" xml:space="preserve"><A href="#top" id="silver" class="circled scrolly" style="text-decoration:none;"><TEXT x="50" y="50" style="cursor: pointer" fill="#FFFFFF" font-size="15" text-anchor="middle">top</TEXT></A></SVG></CENTER></P><P><A class="read-more" href="#week8" id="silver"><SVG xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" version="1.1" id="Layer_1" x="50%" y="0px" width="100" height="100" viewbox="0 0 100 100" enable-background="0 0 1000 1000" xml:space="preserve"><TEXT x="50" y="53" style="cursor: pointer" fill="#FFFFFF" font-size="15" text-anchor="middle">Read More</TEXT></SVG></A></P></ARTICLE><ARTICLE class="mix meetings carousel human" id="week7" date="20140709"><H2><SPAN class="mw-headline" id="Week_7:_Human_practice_planning.2C_Plasmid_assembly_running.2C_Logo_design"> Week 7: Human practice planning, Plasmid assembly running, Logo design  </SPAN></H2><H4><SPAN class="mw-headline" id="Wednesday.2C_July_9th"> Wednesday, July 9th </SPAN></H4><UL><LI> We have a logoÂ !
</LI></UL><UL><LI>In the lab, we did someÂ :
</LI><LI>Plasmid preparation
</LI><LI>Sequencing
</LI><LI>Digests
</LI><LI>Purification of backbone fragments needed for GA
</LI></UL><UL><LI>On the modeling side, we tried to estimate integrase parameters from the paper from Bonnet et al. <SUP><A href="/Team:ETH_Zurich/project/references#refEmergence" title="Team:ETH Zurich/project/references">[9]</A></SUP>, more particularly with the figure S4Â :
</LI></UL><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Data we are using from the Bonnet et al. <A href="/Team:ETH_Zurich/project/references#refEmergence" title="Team:ETH Zurich/project/references"><SUP>[9]</SUP></A> paper</DIV></DIV></DIV></DIV><P>We are trying several strategiesÂ : minimization of the error function and Markov Chain Monte Carlo.
</P><UL><LI> We have a precise plan for our human practice project. We would like to study the emergence of complexity in many different fields and investigate how people deal with it. We will do this with interviews of experts in different fields, and with a survey for a wider outreach. We will also reach younger people by organizing talks in schools. We will finally give our own advice on the question by writing an essay on the subject and linking it to our experience with Mosaicoli. 
</LI></UL><P><CENTER><SVG xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" version="1.1" id="Layer_1" x="50%" y="0px" width="80" height="80" viewbox="0 0 100 100" enable-background="0 0 1000 1000" xml:space="preserve"><A href="#top" id="silver" class="circled scrolly" style="text-decoration:none;"><TEXT x="50" y="50" style="cursor: pointer" fill="#FFFFFF" font-size="15" text-anchor="middle">top</TEXT></A></SVG></CENTER></P><P><A class="read-more" href="#week7" id="silver"><SVG xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" version="1.1" id="Layer_1" x="50%" y="0px" width="100" height="100" viewbox="0 0 100 100" enable-background="0 0 1000 1000" xml:space="preserve"><TEXT x="50" y="53" style="cursor: pointer" fill="#FFFFFF" font-size="15" text-anchor="middle">Read More</TEXT></SVG></A></P></ARTICLE><ARTICLE class="mix lab plasmids carousel" id="constructionreg" date="20140705"><H2><SPAN class="mw-headline" id="Construction_of_the_regulator_plasmids"> Construction of the regulator plasmids </SPAN></H2><H4><SPAN class="mw-headline" id="Thursday.2C_July_5th"> Thursday, July 5th </SPAN></H4><H3><SPAN class="mw-headline" id="Construction_of_the_lasR_regulator_plasmid_.28piG0040.29"> Construction of the <I>lasR</I> regulator plasmid (piG0040) </SPAN></H3><P>Competent cells were transformed with piG0028 (K553003, <I>lasR</I>) and selected on chloramphenicol-LB-plates. Plasmid DNA was extracted by performing a miniprep. The relevant plasmid sequence was sequenced by Microsynth using the primers oiG0001 and oiG0002. Amplification of <I>lasR</I> by PCR using oiG0003 and oiG0004 resulted in the fragment fiG0004 (1.0 kb). The vector piG0034 (pSEVA 181) was digested with the restriction enzymes HindIII and PacI to fiG0001 (3.0 kb). The two fragments, fiG0001 and fiG0004, were assembled using Gibbson assembly. Thus we used the plasmid backbone of piG0034 and <I>lasR</I> of piG0028 to construct the <I>lasR</I> regulator plasmid piG0040. Competent cells were transformed with piG0040 and selected on ampicillin-LB-plates. Colony PCR using the primers oiG0035 and oiG0036 was conducted to check the size of the inserted fragment (expected bands at 1.0 and 1.2 kb). The sequence was verified by Microsynth using the same primers. 
</P><H3><SPAN class="mw-headline" id="Construction_of_the_luxR_regulator_plasmid_.28piG0041.29">Construction of the <I>luxR</I> regulator plasmid (piG0041)</SPAN></H3><P>Competent cells were transformed with piG0008 (F2620, <I>luxR</I>) and selected on chloramphenicol-LB-plates. Plasmid DNA was extracted by performing a miniprep. The relevant plasmid sequence was sequenced by Microsynth using the primers oiG0001 and oiG0002. Amplification of <I>luxR</I> by PCR using oiG0005 and oiG0006 resulted in the fragment fiG0005 (0.8 kb). The backbone of the vector piG0040 (<I>lasR</I> in pSEVA 181) was amplified by PCR using oiG0007 and oiG0008. This fragment, fiG0011 (3.2 kb), and fiG0005 were assembled using Gibbson assembly. Thus we formally replaced <I>lasR</I> by <I>luxR</I> to construct the <I>luxR</I> regulator plasmid piG0041. Competent cells were transformed with piG0041 and selected on ampicillin-LB-plates. Colony PCR using the primers oiG0035 and oiG0036 was conducted to check the size of the inserted fragment (expected bands at 1.1 and 1.2 kb). The sequence was verified by Microsynth using the same primers.
</P><H3><SPAN class="mw-headline" id="Construction_of_the_rhlR_regulator_plasmid_.28piG0042.29">Construction of the <I>rhlR</I> regulator plasmid (piG0042)</SPAN></H3><P>Competent cells were transformed with piG0023 (C0171, <I>rhlR</I>) and selected on chloramphenicol-LB-plates. Plasmid DNA was extracted by performing a miniprep. The relevant plasmid sequence was sequenced by Microsynth using the primers oiG0001 and oiG0002. Amplification of <I>rhlR</I> by PCR using oiG0009 and oiG0010 resulted in the fragment fiG0006 (0.8 kb). The backbone of the vector piG0040 (<I>lasR</I> in pSEVA 181) was amplified by PCR using oiG0011 and oiG0012. This fragment, fiG0012 (3.2 kb), and fiG0006 were assembled using Gibbson assembly. Thus we formally replaced <I>lasR</I> by <I>rhlR</I> to construct the <I>rhlR</I> regulator plasmid piG0042. Competent cells were transformed with piG0042 and selected on ampicillin-LB-plates. Colony PCR using the primers oiG0035 and oiG0036 was conducted to check the size of the inserted fragment (expected bands at 1.1 and 1.2 kb). The sequence was verified by Microsynth using the same primers.
</P><H3><SPAN class="mw-headline" id="Construction_of_the_luxR_regulator_plasmids_with_alternative_constitutive_promoters_.28piG0046_and_piG0047.29">Construction of the <I>luxR</I> regulator plasmids with alternative constitutive promoters (piG0046 and piG0047)</SPAN></H3><P>The promoter site of the <I>luxR</I> regulator plasmid piG0041 was mutated by QuikChange site-specific mutagenesis to produce promoters of different strength. The primers oiG0031 and oiG0032 were used to establish piG0047, a plasmid with a promoter of intermediate strength (J23111). Analoguos the primers oiG0033 and oiG0034 were used to construct piG0046,  a plasmid with a weak promoter (J23109). Competent cells were transformed with piG0046 and piG0047 and selected on ampicillin-LB-plates. The sequence was verified by Microsynth using the primers oig0035 and oiG0036.
</P><P><CENTER><SVG xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" version="1.1" id="Layer_1" x="50%" y="0px" width="80" height="80" viewbox="0 0 100 100" enable-background="0 0 1000 1000" xml:space="preserve"><A href="#top" id="green" class="circled scrolly" style="text-decoration:none;"><TEXT x="50" y="50" style="cursor: pointer" fill="#FFFFFF" font-size="15" text-anchor="middle">top</TEXT></A></SVG></CENTER></P><P><A class="read-more" href="#constructionreg" id="green"><SVG xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" version="1.1" id="Layer_1" x="50%" y="0px" width="100" height="100" viewbox="0 0 100 100" enable-background="0 0 1000 1000" xml:space="preserve"><TEXT x="50" y="53" style="cursor: pointer" fill="#FFFFFF" font-size="15" text-anchor="middle">Read More</TEXT></SVG></A></P></ARTICLE><ARTICLE class="mix meetings carousel" id="week6" date="20140702"><H2><SPAN class="mw-headline" id="Week_6:_Project_planning.2C_Gibson_assemblies_planning.2C_First_Matlab_simulations"> Week 6: Project planning, Gibson assemblies planning, First Matlab simulations  </SPAN></H2><H4><SPAN class="mw-headline" id="Wednesday.2C_July_2nd"> Wednesday, July 2nd </SPAN></H4><UL><LI> We have a clear organization of the different experiments we would like to perform.
</LI></UL><P>We want to optimize quorum sensing to have non-leaky non-cross-talking constructs. We want to prevent cross-talk between integrases and check their dynamics with different quorum sensing promoters. Then we will test the XOR gate without production of LuxI, to check how it works without the loop, and we will finally test our final construct with the loop, hoping that the delay between GFP production and the possible second switching due to AHL production is long enough to have enough fluorescence.
</P><P>The red line is our critical timeline. Numbers of days are very optimistic,therefore we multiply the whole timeline by 2. The stars stand for modeling inputs.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:802px;"><DIV class="thumbcaption">Tasks organization with critical timeline in red.</DIV></DIV></DIV></DIV><UL><LI>We have also clear plans for the Gibson assemblies to perform. As an example, here is the plan for regulator plasmidsÂ :
</LI></UL><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:802px;"><DIV class="thumbcaption">Exemple of Gibson assembly plan for regulator constructs</DIV></DIV></DIV></DIV><UL><LI> We have a <A href="https://www.facebook.com/iGEM.ETH.Zurich" class="external text" rel="nofollow">facebook page</A>Â ! 
</LI></UL><UL><LI> We simulated quorum sensing without leakiness on Matlab, with parameters from the literature.
</LI></UL><P><CENTER><SVG xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" version="1.1" id="Layer_1" x="50%" y="0px" width="80" height="80" viewbox="0 0 100 100" enable-background="0 0 1000 1000" xml:space="preserve"><A href="#top" id="silver" class="circled scrolly" style="text-decoration:none;"><TEXT x="50" y="50" style="cursor: pointer" fill="#FFFFFF" font-size="15" text-anchor="middle">top</TEXT></A></SVG></CENTER></P><P><A class="read-more" href="#week6" id="silver"><SVG xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" version="1.1" id="Layer_1" x="50%" y="0px" width="100" height="100" viewbox="0 0 100 100" enable-background="0 0 1000 1000" xml:space="preserve"><TEXT x="50" y="53" style="cursor: pointer" fill="#FFFFFF" font-size="15" text-anchor="middle">Read More</TEXT></SVG></A></P></ARTICLE><ARTICLE class="mix meetings carousel" date="20140625" id="week5"><H2><SPAN class="mw-headline" id="Week_5:_Assembly_strategy_and_steady_states_derivation"> Week 5: Assembly strategy and steady states derivation  </SPAN></H2><H4><SPAN class="mw-headline" id="Wednesday.2C_June_25th"> Wednesday, June 25th </SPAN></H4><UL><LI> We will use Gibson assembly to assemble our constructs, and have now a construct assembly strategy. We know the list of experiments we want to carry (cross-talk between quorum sensing molecules, quorum sensing leakiness with or without riboswitch, cross-talk between integrases, integrase activity and logic gate behaviour) and the exact sequences of base units we will assemble (lux, rhl, las promoter and repressor units, reporter units, logic units with 3 different integrases, constitutive AHL production units for diffusion test, backbone units). 
</LI></UL><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:802px;"><DIV class="thumbcaption">Exemple of sequences of base units.</DIV></DIV></DIV></DIV><UL><LI>From the modeling side, we calculated analytically the steady state of the integrase and terminator modules, and it  appears that the deterministic model gives a binary steady state corresponding to an XOR gate. This is what we expected, it confirms that deterministic model in our case is interesting for the study of dynamics of the whole system, and not for steady state calculations. So we are now moving to dynamic studies.
</LI></UL><P><CENTER><SVG xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" version="1.1" id="Layer_1" x="50%" y="0px" width="80" height="80" viewbox="0 0 100 100" enable-background="0 0 1000 1000" xml:space="preserve"><A href="#top" id="silver" class="circled scrolly" style="text-decoration:none;"><TEXT x="50" y="50" style="cursor: pointer" fill="#FFFFFF" font-size="15" text-anchor="middle">top</TEXT></A></SVG></CENTER></P><P><A class="read-more" href="#week5" id="silver"><SVG xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" version="1.1" id="Layer_1" x="50%" y="0px" width="100" height="100" viewbox="0 0 100 100" enable-background="0 0 1000 1000" xml:space="preserve"><TEXT x="50" y="53" style="cursor: pointer" fill="#FFFFFF" font-size="15" text-anchor="middle">Read More</TEXT></SVG></A></P></ARTICLE><ARTICLE class="mix meetings carousel" id="week4" date="20140618"><H2><SPAN class="mw-headline" id="Week_4:_Improving_plasmid_design.2C_Modularization_of_the_whole_model"> Week 4: Improving plasmid design, Modularization of the whole model  </SPAN></H2><H4><SPAN class="mw-headline" id="Wednesday.2C_June_18th"> Wednesday, June 18th </SPAN></H4><UL><LI> We improved a little bit the plasmid design, determined which parts will be taken from the iGEM registry, and which parts we have to synthesize.
</LI></UL><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption">Second draft of plasmids. The repressors production genes have been moved to the sensor plasmid in order to distribute more evenly the number of kb.</DIV></DIV></DIV></DIV><P>Sensor plasmids will be approx. 9.3kb long, and logic plasmids will be approx. 5,2kb long.
</P><UL><LI> From the modeling side, the whole model has been <A id="insidelink" targetid="modularization" href="https://2014.igem.org/Team:ETH_Zurich/blog#modularization"> modularized </A> and we have calculated and simulated steady state for the quorum sensing module.
</LI><LI> GeneScript, Microsynth, EraSynbio are willing to support usÂ !
</LI></UL><P><CENTER><SVG xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" version="1.1" id="Layer_1" x="50%" y="0px" width="80" height="80" viewbox="0 0 100 100" enable-background="0 0 1000 1000" xml:space="preserve"><A href="#top" id="silver" class="circled scrolly" style="text-decoration:none;"><TEXT x="50" y="50" style="cursor: pointer" fill="#FFFFFF" font-size="15" text-anchor="middle">top</TEXT></A></SVG></CENTER></P><P><A class="read-more" href="#week4" id="silver"><SVG xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" version="1.1" id="Layer_1" x="50%" y="0px" width="100" height="100" viewbox="0 0 100 100" enable-background="0 0 1000 1000" xml:space="preserve"><TEXT x="50" y="53" style="cursor: pointer" fill="#FFFFFF" font-size="15" text-anchor="middle">Read More</TEXT></SVG></A></P></ARTICLE><ARTICLE class="mix modeling carousel" id="modularization" date="20140615"><H2><SPAN class="mw-headline" id="Modularization_of_the_system"> Modularization of the system </SPAN></H2><H4><SPAN class="mw-headline" id="Sunday.2C_June_15th"> Sunday, June 15th </SPAN></H4><P>The idea of our system is to associate a quorum sensing part, which should be able to react to a certain concentration of input, with a integrase-base logic part, which should be able to derive a computation given the transformed inputs. 
</P><P>
From a modeling point of view, it seemed obvious that we had to consider these two entities separately. That is why we chose to have a modularized approach, where we regrouped, on one hand, the equations corresponding to quorum sensing and, on the other hand, the ones corresponding to the integrase logic phenomena. 
</P><P>
As the integrase module is not that well known, we further decomposed it into two successive parts. The first one only models the path integrases go through in order to bind to DNA. The second one models the switch behavior of the integrases (after they are bound to DNA) and describe the XOR logic of our system.
</P><P>
As we see, these modules are successive. After the logic module, we added a production module, which is responsible for the production of our outputs (GFP and an Homoserine Lactone that depends on the type of cell modeled). As one of our output is a messenger molecule, it has an influence on the quorum sensing module, which detects those molecules. Therefore, we have a feedback loop. This feedback loop comes from our choice to put only two different strains in our grid. 
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:702px;"><DIV class="thumbcaption">Decomposition of our system in different modules and submodules. The inputs are AHL and PAI2 and our output is AHL and GFP going out from the Production module.</DIV></DIV></DIV></DIV><P><CENTER><SVG xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" version="1.1" id="Layer_1" x="50%" y="0px" width="80" height="80" viewbox="0 0 100 100" enable-background="0 0 1000 1000" xml:space="preserve"><A href="#top" id="blue" class="circled scrolly" style="text-decoration:none;"><TEXT x="50" y="50" style="cursor: pointer" fill="#FFFFFF" font-size="15" text-anchor="middle">top</TEXT></A></SVG></CENTER></P><P><A class="read-more" href="#modularization" id="blue"><SVG xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" version="1.1" id="Layer_1" x="50%" y="0px" width="100" height="100" viewbox="0 0 100 100" enable-background="0 0 1000 1000" xml:space="preserve"><TEXT x="50" y="53" style="cursor: pointer" fill="#FFFFFF" font-size="15" text-anchor="middle">Read More</TEXT></SVG></A></P></ARTICLE><ARTICLE class="mix meetings carousel" id="week3" date="20140604"><H2><SPAN class="mw-headline" id="Week_3:_Plasmid_design_started.2C_Modeling_started"> Week 3: Plasmid design started, Modeling started  </SPAN></H2><H4><SPAN class="mw-headline" id="Wednesday.2C_June_11th"> Wednesday, June 11th </SPAN></H4><UL><LI> We started plasmid designÂ :
</LI></UL><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption">First draft of plasmids. The sensor plasmid is the same in every cell. Different logic + repressors plasmids are present in cells p and q. The logic + repressors plasmid from cells p is producing the quorum sensing molecule p, and another version is present in cells q which produces QSq. Colonies of p and q cells will be arranged in an alternate way on the millifluidic chip. Here as an example we use the lux system for p cells and the rhl system for q cells.</DIV></DIV></DIV></DIV><P>Î¦C31 and Bxb1 are integrases. LuxR an RhlR are quorum sensing repressors. A riboswitch construct is placed around quorum sensing constructs to prevent leakiness of lux and rhl promoters. Type p colonies produce AHL by expressing the enzyme LuxI. Type q colonies produce Rhl by expressing the enzyme RhlI. In fact, we don't know yet which quorum sensing systems we will use. We will have to perform cross-talk experimetns in order to choose the ones that are the most orthogonal.
</P><UL><LI> We tried to print our first agar millifluidic chipÂ : we printed it too small, and the printer had resolution problems.
</LI><LI> We wrote all reactions and found parameters from the literature for our model.
</LI></UL><P><CENTER><SVG xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" version="1.1" id="Layer_1" x="50%" y="0px" width="80" height="80" viewbox="0 0 100 100" enable-background="0 0 1000 1000" xml:space="preserve"><A href="#top" id="silver" class="circled scrolly" style="text-decoration:none;"><TEXT x="50" y="50" style="cursor: pointer" fill="#FFFFFF" font-size="15" text-anchor="middle">top</TEXT></A></SVG></CENTER></P><P><A class="read-more" href="#week3" id="silver"><SVG xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" version="1.1" id="Layer_1" x="50%" y="0px" width="100" height="100" viewbox="0 0 100 100" enable-background="0 0 1000 1000" xml:space="preserve"><TEXT x="50" y="53" style="cursor: pointer" fill="#FFFFFF" font-size="15" text-anchor="middle">Read More</TEXT></SVG></A></P></ARTICLE><ARTICLE class="mix meetings carousel" id="week2" date="20140604"><H2><SPAN class="mw-headline" id="Week_2:_Investigating_microfluidics.2C_Writing_a_modeling_pipeline"> Week 2: Investigating microfluidics, Writing a modeling pipeline  </SPAN></H2><H4><SPAN class="mw-headline" id="Wednesday.2C_June_4th"> Wednesday, June 4th </SPAN></H4><P>This week, we have talked with people from the microfluidics group and it came out that we should be able to use beads to encapsulate our cells and use a microfluidic chip. Therefore we will try to develop this possibility in parallel with the 3D-printed agar chip.

We have set up a modeling pipeline. We divided the modeling project into 3 partsÂ : 
</P><UL><LI>diffusion, 
</LI><LI>parameter fitting, 
</LI><LI>modeling of the genetic circuit itself, which comprises 
</LI><LI>a deterministic model
</LI><LI>a stochastic model. 
</LI></UL><P>We know that the modeling challenges will be to model how integrases work, and to model the delay between reception of a quorum sensing signal and production of a quorum sensing signal by the receiver cell. Indeed, a colony could switch itself OFF when it should be ON, if it receives QS1 and produces QS2, for example. If a delay is present between reception of QS1 and production of QS2, the colony will produce GFP before it is switched OFF, and as GFP is stable enough, it will stay visible during a long time and the self-switching OFF won't be observed. This delay should be modeled.
</P><P><CENTER><SVG xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" version="1.1" id="Layer_1" x="50%" y="0px" width="80" height="80" viewbox="0 0 100 100" enable-background="0 0 1000 1000" xml:space="preserve"><A href="#top" id="silver" class="circled scrolly" style="text-decoration:none;"><TEXT x="50" y="50" style="cursor: pointer" fill="#FFFFFF" font-size="15" text-anchor="middle">top</TEXT></A></SVG></CENTER></P><P><A class="read-more" href="#week2" id="silver"><SVG xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" version="1.1" id="Layer_1" x="50%" y="0px" width="100" height="100" viewbox="0 0 100 100" enable-background="0 0 1000 1000" xml:space="preserve"><TEXT x="50" y="53" style="cursor: pointer" fill="#FFFFFF" font-size="15" text-anchor="middle">Read More</TEXT></SVG></A></P></ARTICLE><ARTICLE class="mix meetings carousel" id="week1" date="20140528"><H2><SPAN class="mw-headline" id="Week_1_:_Project_selected"> Week 1Â : Project selected </SPAN></H2><H4><SPAN class="mw-headline" id="Wednesday.2C_May_28th"> Wednesday, May 28th </SPAN></H4><P>After more than one month of endless meetings and passionate debates, we finally chose the project that will keep us occupied in the next five months. From the beautiful pattern made by the Sierpinski triangles, we will focus on cellular automata and try to implement one. 
</P><P>Sierpinski triangles appear when the rule 90 is followed by every cell on the gridÂ :
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Pattern given by rule 90 if only one cell is switched on in the first row at the beginning. The pattern appears row by row.</DIV></DIV></DIV></DIV><P>Ideally we will use a microfluidic chip. We could also use a 3D printed agar plate like this one to load the colonies. On this grid we can implement the rule 6, which is the simplification of rule 90 considered as a rule with 2 inputsÂ : each cell computes a simple XOR gate of its two parents.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption">Arrangement of p and q cells on the chip</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Pattern given by rule 6 if only one cell is switched on in the first row at the beginning. The pattern appears row by row.</DIV></DIV></DIV></DIV><P>
The logic part will be built with integrases and the colony-to-colony communication will use quorum sensing.
</P><P>Every colony will receive two quorum sensing signals (QSp and QSq) from the two cells above it. These two signals trigger the production of two different integrases r and s in the colony. Integrases enable to build biological XOR logic gates by switching twice a terminator. Indeed, every integrase can switch the terminator only once. Thus if the colony produces only r or only s, the terminator is switched only once, so the terminator is OFF, and GFP and QS1 or QS2 are produced (depending on the colony). If the colony produces r and s, the terminator is switched twice, so it is ON and it blocks expression of GFP and of the quorum sensing molecule. 
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:802px;"><DIV class="thumbcaption">Rule that will be computed by all cellsÂ : an XOR gate between the signals from the 2 cells above. Biologically, integrases p and q are able to bind sites around a terminator and switch this terminator, thereby turning off the effect of the terminator and enabling gfp transcription. If p and q integrases are both present, the terminator is switched twice and comes back to its initial state, gfp is not transcribed.</DIV></DIV></DIV></DIV><P>We need to Â :
</P><UL><LI>find orthogonal quorum sensing molecules and orthogonal integrases
</LI><LI>discuss with microfluidics experts to check if using microfluidics is possible and presents advantages in our case
</LI><LI>find possible parts in the registry for integrases, and design plasmids
</LI></UL><P><CENTER><SVG xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" version="1.1" id="Layer_1" x="50%" y="0px" width="80" height="80" viewbox="0 0 100 100" enable-background="0 0 1000 1000" xml:space="preserve"><A href="#top" id="silver" class="circled scrolly" style="text-decoration:none;"><TEXT x="50" y="50" style="cursor: pointer" fill="#FFFFFF" font-size="15" text-anchor="middle">top</TEXT></A></SVG></CENTER></P><P><A class="read-more" href="#week1" id="silver"><SVG xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" version="1.1" id="Layer_1" x="50%" y="0px" width="100" height="100" viewbox="0 0 100 100" enable-background="0 0 1000 1000" xml:space="preserve"><TEXT x="50" y="53" style="cursor: pointer" fill="#FFFFFF" font-size="15" text-anchor="middle">Read More</TEXT></SVG></A></P></ARTICLE><ARTICLE class="mix human carousel talks" id="openhouse" date="20140510"><H2><SPAN class="mw-headline" id="BSSE_Openhouse_Day"> BSSE Openhouse Day</SPAN></H2><H4><SPAN class="mw-headline" id="Saturday.2C_May_10th"> Saturday, May 10th </SPAN></H4><H3><SPAN class="mw-headline" id="Sharing_our_iGEM_and_synthetic_biology_interest_with_the_public">Sharing our iGEM and synthetic biology interest with the public</SPAN></H3><P>On May 10th the public in Basel had the unique chance to get an insight into many different scientific laboratories and the work done there. It was the joint open house day of D-BSSE of ETH (Department of Biosystems Science and Engineering) and the Biozentrum of the University of Basel. The many different labs opened their doors to the public and many scientists were present to give interested people some details about their daily work. So did the ETH iGEM team 2014. The team was present with a poster showing the history of iGEM, the previous ETH iGEM teams with their projects and general information about synthetic biology. Additionally there was a slideshow giving a best-of photo collection of last years jamboree. The goal of this day was to inform the public about synthetic biology in general and specifically about the spirit and the many different projects of iGEM. Many people showed strong interest in truly student driven projects and are curious to follow our team wiki for the next months. 
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:802px;"><DIV class="thumbcaption">Max and Nadine participating to BSSE Openhouse event and sharing iGEM and their synthetic biology interest to the public</DIV></DIV></DIV></DIV><P><CENTER><SVG xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" version="1.1" id="Layer_1" x="50%" y="0px" width="80" height="80" viewbox="0 0 100 100" enable-background="0 0 1000 1000" xml:space="preserve"><A href="#top" id="red" class="circled scrolly" style="text-decoration:none;"><TEXT x="50" y="50" style="cursor: pointer" fill="#FFFFFF" font-size="15" text-anchor="middle">top</TEXT></A></SVG></CENTER></P><P><A class="read-more" href="#openhouse" id="red"><SVG xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" version="1.1" id="Layer_1" x="50%" y="0px" width="100" height="100" viewbox="0 0 100 100" enable-background="0 0 1000 1000" xml:space="preserve"><TEXT x="50" y="53" style="cursor: pointer" fill="#FFFFFF" font-size="15" text-anchor="middle">Read More</TEXT></SVG></A></P></ARTICLE></DIV></DIV></DIV><DIV id="footer"><HEADER><H5>We thank our sponsors:</H5></HEADER></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:ETH_Zurich/blog">http://2014.igem.org/Team:ETH_Zurich/blog</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:ETH_Zurich/blog" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:ETH_Zurich/blog" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:ETH_Zurich/blog&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:ETH_Zurich/blog&amp;oldid=397894" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>