<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_ETH_Zurich_project_overview3"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:ETH_Zurich/project/overview3">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:ETH_Zurich/project/overview3&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:ETH_Zurich/project/overview3&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:ETH_Zurich/project/overview3&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:ETH_Zurich/project/overview3" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:ETH Zurich/project/overview3</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><TITLE>iGEM ETH Zurich 2014</TITLE><DIV id="header"><NAV id="nav"><UL><LI><A href="/Team:ETH_Zurich" title="Team:ETH Zurich">Home</A></LI><LI><A name="Project" style="cursor: default" href="javascript:void(0)">Project</A></LI><LI><A href="/Team:ETH_Zurich/project/overview" title="Team:ETH Zurich/project/overview">Overview</A></LI><LI><A href="/Team:ETH_Zurich/project/infopro" title="Team:ETH Zurich/project/infopro">Information Processing</A></LI><LI><A href="/Team:ETH_Zurich/project/background" title="Team:ETH Zurich/project/background">Background</A></LI><LI><A href="/Team:ETH_Zurich/project/goals" title="Team:ETH Zurich/project/goals">Goals</A></LI><LI><A href="/Team:ETH_Zurich/project/applications" title="Team:ETH Zurich/project/applications">Applications</A></LI><LI><A href="/Team:ETH_Zurich/project/references" title="Team:ETH Zurich/project/references">References</A></LI><LI><A href="/Team:ETH_Zurich/blog" title="Team:ETH Zurich/blog">Blog</A></LI></UL><LI><A name="Achievements" style="cursor: default" href="javascript:void(0)">Achievements</A></LI><LI><A href="/Team:ETH_Zurich/achievements" title="Team:ETH Zurich/achievements">Achievements</A></LI><LI><A href="/Team:ETH_Zurich/criteria" title="Team:ETH Zurich/criteria">Medal Criteria</A></LI><LI><A href="/Team:ETH_Zurich/expresults" title="Team:ETH Zurich/expresults">Results</A></LI><LI><A name="Data page" style="cursor: default" href="javascript:void(0)">Data page</A></LI><LI><A href="/Team:ETH_Zurich/data#Gene_Circuit" title="Team:ETH Zurich/data">Gene Circuit and Parts</A></LI><LI><A href="/Team:ETH_Zurich/data#Used_and_Characterized_Pre-Existing_Parts" title="Team:ETH Zurich/data">Used and Characterized Pre-Existing Parts</A></LI><LI><A href="/Team:ETH_Zurich/data#Our_Favorite_New_Characterized_Parts" title="Team:ETH Zurich/data">Our Favorite New Characterized Parts</A></LI><LI><A href="/Team:ETH_Zurich/data/raw" title="Team:ETH Zurich/data/raw">Raw Data</A></LI><LI><A name="Modeling" style="cursor: default" href="javascript:void(0)">Modeling</A></LI><LI><A href="/Team:ETH_Zurich/modeling/overview" title="Team:ETH Zurich/modeling/overview">Overview</A></LI><LI><A href="/Team:ETH_Zurich/modeling/qs" title="Team:ETH Zurich/modeling/qs">Quorum Sensing</A></LI><LI><A href="/Team:ETH_Zurich/modeling/int" title="Team:ETH Zurich/modeling/int">Integrases</A></LI><LI><A href="/Team:ETH_Zurich/modeling/xor" title="Team:ETH Zurich/modeling/xor">XOR Gate</A></LI><LI><A href="/Team:ETH_Zurich/modeling/whole" title="Team:ETH Zurich/modeling/whole">Whole cell model</A></LI><LI><A href="/Team:ETH_Zurich/modeling/diffmodel" title="Team:ETH Zurich/modeling/diffmodel">Diffusion</A></LI><LI><A href="/Team:ETH_Zurich/modeling/parameters" title="Team:ETH Zurich/modeling/parameters">Parameters and Tools</A></LI><LI><A name="Lab" style="cursor: default" href="javascript:void(0)">Lab</A></LI><LI><A href="/Team:ETH_Zurich/lab/overview" title="Team:ETH Zurich/lab/overview">Overview</A></LI><LI><A href="/Team:ETH_Zurich/lab/protocols" title="Team:ETH Zurich/lab/protocols">Materials &amp; Methods</A></LI><LI><A href="/Team:ETH_Zurich/lab/sequences" title="Team:ETH Zurich/lab/sequences">Sequences</A></LI><LI><A href="/Team:ETH_Zurich/lab/bead" title="Team:ETH Zurich/lab/bead">Alginate Beads</A></LI><LI><A href="/Team:ETH_Zurich/lab/chip" title="Team:ETH Zurich/lab/chip">Millifluidic Chip</A></LI><LI><A href="/Team:ETH_Zurich/lab/safety" title="Team:ETH Zurich/lab/safety">Safety</A></LI><LI><A name="Human Practice" style="cursor: default" href="javascript:void(0)">Human practice</A></LI><LI><A href="/Team:ETH_Zurich/human/overview" title="Team:ETH Zurich/human/overview">Overview</A></LI><LI><A href="/Team:ETH_Zurich/human/survey" title="Team:ETH Zurich/human/survey">Survey</A></LI><LI><A href="/Team:ETH_Zurich/human/interviews" title="Team:ETH Zurich/human/interviews">Interviews</A></LI><LI><A href="/Team:ETH_Zurich/human/outreach" title="Team:ETH Zurich/human/outreach">Outreach</A></LI><LI><A href="/Team:ETH_Zurich/human/essay" title="Team:ETH Zurich/human/essay">Our insights</A></LI><LI><A name="Team" style="cursor: default" href="javascript:void(0)">Team</A></LI><LI><A href="https://igem.org/Team.cgi?year=2014&amp;team_name=ETH_Zurich" class="external text" rel="nofollow">Official Team Profile</A></LI><LI><A href="/Team:ETH_Zurich/team" title="Team:ETH Zurich/team">Our team</A></LI><LI><A href="/Team:ETH_Zurich/adandin" title="Team:ETH Zurich/adandin">Advisors and Instructors</A></LI><LI><A href="/Team:ETH_Zurich/collaborations" title="Team:ETH Zurich/collaborations">Collaborations</A></LI><LI><A href="/Team:ETH_Zurich/acknowledgements" title="Team:ETH Zurich/acknowledgements">Acknowledgements</A></LI><LI><A href="/Team:ETH_Zurich/contact" title="Team:ETH Zurich/contact">Contacting us</A></LI></NAV><DIV class="inner1"><HEADER><H1>  {{{1}}}  </H1></HEADER><CENTER><H2><SPAN class="mw-headline" id="Mosaicoli_:_from_simplicity_to_complexity_with_biologic_gates"> Mosai<I>coli</I> : from simplicity to complexity with bio<I>logic</I> gates </SPAN></H2></CENTER><H3 style="font-weight:800; text-align:center"> Abstract</H3><P>Emergence of complex patterns in nature is a fascinating and widely spread phenomenon, which is not fully understood yet. Mosai<I>coli</I> aims to investigate emergence of complex patterns from a simple rule by engineering a cellular automaton into <I>E. coli</I> bacteria. This automaton comprises a grid of colonies on a <A href="/Team:ETH_Zurich/lab/chip" title="Team:ETH Zurich/lab/chip">3D-printed millifluidic chip</A>. Each colony is either in an ON or OFF state and updates its state by integrating signals from its neighbors according to a genetically <A href="/Team:ETH_Zurich/project/background#Pattern" title="Team:ETH Zurich/project/background">pre-programmed logic rule</A>. Complex patterns such as Sierpinski triangles are visualized by fluorescence after <A href="/Team:ETH_Zurich/expresults#Diffusion_On_Chip" title="Team:ETH Zurich/expresults">several steps of row-wise propagation</A>. Sequential logic computation based on quorum sensing is challenged by <A href="/Team:ETH_Zurich/expresults#Quorum_Sensing" title="Team:ETH Zurich/expresults">leakiness and crosstalk</A> present in biological systems. Mosai<I>coli</I> overcomes these issues by exploiting multichannel orthogonal communication, <A href="/Team:ETH_Zurich/expresults#Riboregulators" title="Team:ETH Zurich/expresults">riboregulators</A> and integrase-based XOR logic gates. Engineering such a reliable system not only enables a better understanding of emergent patterns, but also provides novel building blocks for biological computers. 


</P><H2> Background </H2><ARTICLE class="carousel"><H3><SPAN class="mw-headline" id=".22The_whole_is_more_than_the_sum_of_its_parts.22_Aristotle"> &quot;The whole is more than the sum of its parts&quot; <I>Aristotle</I></SPAN></H3><P>Emergence describes how novel properties can arise from lower-complexity parts, where these properties are not observed.<SUP><A href="/Team:ETH_Zurich/project/references#refEmergence" title="Team:ETH Zurich/project/references">[12]</A></SUP> This phenomenon is wide-spread in nature. One straight-forward example is water: from the molecules of H<SUB>2</SUB>O, which are here considered as simple subparts, we are able to consider the liquid &quot;water&quot; at a higher level of complexity, with properties such as viscosity, pressure, transparency...
</P><P>One particular pattern formation retained our attention: some sea snail present a complex pattern on their shells. The &quot;Cloth of Gold&quot; observed on textile cones results from an inner computation of the neural cells of the sea snail. This pattern appears row by row (computation after computation) and the pigments of the next row are determined by the pigments of the row before.  
</P><P>
This pattern formation was formalized by Neuman in the concept of cellular automata. Following a simple pre-programmed logic rule, the state of a spot on the shell, corresponding to the color (either white or brown), is determined by the states of three parent spots (from the previous computation round). Wolfram <SUP><A href="/Team:ETH_Zurich/project/references#refWolfram" title="Team:ETH Zurich/project/references">[11]</A></SUP> elaborated a whole theory on these cellular automata.
</P><P>According to cellular automata theory, emergent patterns offer a large panel of properties: striking examples are the rule 30 which gives an apparently random pattern and the rule 110 which has been proven to be Turing complete. With cellular automata, you cannot predict how the final pattern will look like even if you know the rule that governs its property. Thus the intricated computations of steps poses the problem of complexity. 
</P><P class="read-more"><A href="#" class="button twolines">Read More</A></P></ARTICLE><H2> Our Project: Mosai<EM>coli</EM></H2><H3> Principle and goals </H3><ARTICLE class="carousel"><P>The aim of our project is to investigate the emergence of complexity and how we can deal with complexity in general. Our project addresses this goal in two ways.

First, we follow a biomimetic approach. This approach corresponds to the motto &quot;What I cannot create, I cannot understand&quot; (Richard Feynman). We are inspired by Sierpinski triangle patterns present on sea snail shells, and engineer the same kind of emergent patterns on grids of bacterial colonies. These patterns are emergent because they arise directly from one logic gate implemented in all bacteria. This project that combines modeling and wet-lab work will enable us to answer some questions such as how complexity can emerge from simple rules, whether it can be predicted from simple rules, how we can deal with crosstalk and leakiness of biological systems to enable a good predictability.

Second, we widen the scope of our investigation to other projects and disciplines, from scientific fields to philosophy, sociology or art. We address the issue of how to deal with complexity, by interviewing experts in several fields and conducting a larger scale study with a survey. Do these people consider that parts are strictly ordered, and try to reduce complexity to simple parts strictly following a set of deterministic rules, or do they accept that complexity comprises a mix of order and disorder, that a part of uncertainty cannot be neglected and that complex systems should be studied as a whole? Both approaches have their advantages and their drawbacks, which one should we choose to deal with the increasing complexity of our world?

Here is a more detailed list of subgoals:
</P><UL><LI> Make a Sierpinski triangle pattern appear on a grid of bacteria
</LI><LI> Conjugate quorum sensing and logic gates in bacterial colonies
</LI><LI> Implement an XOR gate in an <I>E. coli</I></LI><LI> Characterize integrases (retrieve missing parameters)
</LI><LI> Lower the leakiness in quorum sensing systems
</LI><LI> Study quorum sensing crosstalk in order to implement orthogonal communication
</LI><LI> Be able to predict accurately the system’s behavior
</LI><LI> Conduct a survey about how to deal with complexity
</LI><LI> Gather interviews of experts from different fields
</LI><LI> Find our own answer to this question thanks to our Mosai<I>coli</I></LI></UL><P class="read-more"><A href="#" class="button twolines">Read More</A></P></ARTICLE><H3> Implementation in <EM>E. Coli</EM></H3><DIV class="carousel"><H3 style="font-weight:800; text-align:center"> More details</H3><P>Mosai<I>coli</I> involves three main plasmid constructs per cell, one for quorum sensing, one for production of integrases, and the last with the integrase-based XOR logic gate to perform computation. Each cell can receive two orthogonal <I>N</I>-acyl homoserine latones (AHLs) - the LuxI product <I>N</I>-(3-oxohexanoyl)-L-homoserine (3OC<SUB>6</SUB>-HSL) and the LasI product <I>N</I>-(3-oxododecanoyl)-L-homoserine (3OC<SUB>12</SUB>-HSL).  The 3OC<SUB>6</SUB>-HSL or the 3OC<SUB>12</SUB>-HSL received by the cell bind to their corresponding receptor proteins  LuxR and LasR, thus activating them. The LuxR-3OC<SUB>6</SUB>-HSL or LasR-3OC<SUB>12</SUB>-HSL complexes bind to the corresponding promoters Plux and Plas on the second plasmid and induce the expression of two integrases Bxb1 or ΦC31, respectively. Additionally, we use riboregulators to reduce leakiness in the expression of the integrase genes from the promoters P<SUB>lux</SUB> or P<SUB>las</SUB><SUP><A href="/Team:ETH_Zurich/project/references#refWilliams" title="Team:ETH Zurich/project/references">[10]</A></SUP>.

The integrases act on an asymmetric transcription terminator present  present on the third plasmid, that contains the XOR gate. The terminator is located between a constitutive promoter and the start of two consecutive  downstream genes in one operon, <I>luxI</I> and <I>gfp</I>. If the terminator is in its &quot;functional” orientation, expression from these genes is not possible, because it is prematurely terminated. Further, the terminator is flanked by two pairs of opposing recombination sites recognised by ΦC31 and Bxb1, respectively. In the absence of both integrases, the terminator is in its functional orientation and thus blocks transcription. Expression of either integrase alone inverts the DNA encoding the terminator. It is no longer in its functional orientation and transcription of <I>luxI</I> (or <I>lasI</I>) and <I>gfp</I> is enabled. Presence of both integrases inverts the terminator twice bringing it back to its original functional orientation. Thus, transcription is blocked again<SUP><A href="/Team:ETH_Zurich/project/references#refBonnet" title="Team:ETH Zurich/project/references">[9]</A></SUP>. After the integrases acted on their corresponding recombination sites, the sites recombine in a way leaving the integrases unable to invert the region a second time. 

Colonies of such cells are placed in a grid in a <A href="https://2014.igem.org/Team:ETH_Zurich/lab/chip" class="external text" rel="nofollow">3D-printed millifluidic chip</A>. In each colony, all cells can exist in one of two states - ON or OFF. The cells are OFF if they do not produce any GFP and LuxI (or LasI) and ON when they produce GFP and LuxI (or LasI). The produced LuxI or LasI enzymes catalyse the production of the corresponding AHL molecules which diffuse out and are propagated to the colonies in the next row. Each colony updates its state by integrating signals from its neighbours (colonies in the previous rows). We expect to see complex fluorescent patterns, such as the Sierpinski triangles, after several rows of colonies on the grid have updated their states. 
</P><P class="read-more"><A href="#" class="button twolines">Read More</A></P></DIV><H3> Biological tools: quorum sensing and integrases </H3><DIV class="carousel"><H4><SPAN class="mw-headline" id="Quorum_Sensing"> Quorum Sensing </SPAN></H4><P>Bacteria, usually found in multi-species neighborhoods with billions of cells, where thought to live and divide isolated from its surroundings. However, bacteria do not try to out-compete the neighbors by mindless exponential growth but can exchange information via cell-to-cell communications in order to coordinate gene-expression and as a result the overall behavior of the population. This  communicative endeavor is commonly described as quorum sensing and based on the synthesis, secretion and diffusion of small molecules or 'autoinducers'.
</P><P>One of the best studied quorum sensing models was found in the Gram-negative bacterium <I>V. fischeri</I> in the early 1980s<SUP><A href="/Team:ETH_Zurich/project/references#refEberhard" title="Team:ETH Zurich/project/references">[4]</A></SUP>. Depending on the cell density and the corresponding autoinducer concentration, communication between the bacteria induces bioluminescence in the whole <I>V. fischeri</I> population. The small molecule responsible for this behavior was identified as <I>N</I>-(3-oxohexanoyl)-L-homoserine lactone, which is part of the <I>N</I>-acyl homoserine lactone (AHL) family. This AHL is synthesized by an enzyme called LuxI, diffuses through cell membranes and can then reach the cytosol of neighboring cells. There it binds to and activates LuxR, a protein positively regulating gene transcription. Subsequently to the investigation of <I>V. fischeri</I>, several other AHLs (with specific acyl side chains) and corresponding LuxR/LuxI like systems were found in various bacterial species, coordinating diverse responses including plasmid transfer, virulence, and antibiotic resistance<SUP><A href="/Team:ETH_Zurich/project/references#refWilliams" title="Team:ETH Zurich/project/references">[5]</A></SUP>.
</P><P>Here we focus in particular on the quorum sensing circuitry of <I>V. fischeri</I> and <I>Pseudomonas aeruginosa</I>, another Gram-negative bacteria. The additional systems are called LasR/LasI and RhlR/RhlI, also employing AHL molecules as autoinducers and are similar to the prototypical LuxR/LuxI system. Even though a key feature of quorum sensing molecules and its receptors is high specificity designed for intraspecies communication, the structural similarity of the different AHL molecules can cause unwanted gene expression. This cross-talk has to be taken into account when re-engineering LuxR/LuxI like systems for synthetic modules and circuits<SUP><A href="/Team:ETH_Zurich/project/references#refJayaraman" title="Team:ETH Zurich/project/references">[6]</A></SUP>.
</P><H4><SPAN class="mw-headline" id="Integrases"> Integrases </SPAN></H4><P>Site specific serine recombinases, or integrases, are enzymes recognizing pairs of short and non-identical DNA sequences. Within those sites they irreversibly catalyze the excision or unidirectional inversion of the DNA bases, depending on the orientation of the recognition sequence (often referred to as attB or attP). Many recombinases originate from phages integrating their genes into bacterial host genomes, hence they often do not require specific co-factors and are fully functional when heterologously expressed in bacteria. As a result, they can be employed as molecular tools in biotechnology<SUP><A href="/Team:ETH_Zurich/project/references#refKobi" title="Team:ETH Zurich/project/references">[7]</A></SUP>.
</P><P>Due to the site-specificity, recombinases like Bxb1 and ΦC31 can be employed orthogonally in synthetic systems. In addition, the combination of multiple recombinases -and their corresponding specific recognition sites- with gene regulatory sequences like promoters and terminators, allowed the engineering of Boolean logic gates<SUP><A href="/Team:ETH_Zurich/project/references#refSiuti" title="Team:ETH Zurich/project/references">[8]</A></SUP>. For example, when an asymmetric terminator sequence is placed within a pair of recombination sites, the RNA polymerase can only proceed along this sequence if exactly one recombinase is available. Otherwise the terminator is blocking gene transription by default, or due to the repeated inversion, is again fully function as soon as both recombinases are present<SUP><A href="/Team:ETH_Zurich/project/references#refBonnet" title="Team:ETH Zurich/project/references">[9]</A></SUP>. Such recombinase-based logic gates are at the core of our genetic circuits. 
</P><P class="read-more"><A href="#" class="button twolines">Read More</A></P></DIV><H2> Applications and outlook </H2><DIV class="carousel"><P>Future development of synthetic biological systems will require the implementation of reliable synthetic circuits. These gene circuits will be designed to program new biological behaviour, dynamics, and logic control<SUP><A href="/Team:ETH_Zurich/project/references#refWeiss" title="Team:ETH Zurich/project/references">[1]</A></SUP>. A crucial part to coordinate and control the biological computation needed to perform their tasks can be seen in multichannel orthogonal communication. In order to achieve the goals of our project we need to thoroughly characterize available components, improve their performance, and develop novel constructs that get closer to actual multichannel orthogonal communication and subsequent logic processing. With our findings we will try to make a contribution to the field of biological computation by enabling further development of complex systems that rely on the interaction between its subparts.
</P><P>These biological computers in turn also need to find a way to be implemented in patients, if they are developed to perform therapeutic tasks. A promising application of such synthetic biological systems is cell based therapy by alginate-microencapsulated implants<SUP><A href="/Team:ETH_Zurich/project/references#refAuslander" title="Team:ETH Zurich/project/references">[2]</A><A href="/Team:ETH_Zurich/project/references#refBugaj" title="Team:ETH Zurich/project/references">[3]</A></SUP>. We will try to achieve communication and input integration in a grid of alginate beads and thus producing a more complex behavior than a single bead could show. This can be seen as a step towards the development of artificial organs composed of multiple alginate-microencapsulated implants, that work together to tackle tasks a single implant, which is not communicating with its neighbors, could not solve. 
</P><P class="read-more"><A href="#" class="button twolines">Read More</A></P></DIV></DIV></DIV><DIV id="footer"><HEADER><H5>We thank our sponsors:</H5></HEADER></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:ETH_Zurich/project/overview3">http://2014.igem.org/Team:ETH_Zurich/project/overview3</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:ETH_Zurich/project/overview3" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:ETH_Zurich/project/overview3" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:ETH_Zurich/project/overview3&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:ETH_Zurich/project/overview3&amp;oldid=172333" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>