<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_HUST-China_wetlab_protocols skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:HUST-China/wetlab/protocols</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Interlab</TITLE></P><NAV class="navbar navbar-default"><DIV class="container"><SPAN class="menu">Menu</SPAN><DIV class="banner-top"><UL class="nav banner-nav"><LI><A href="https://2016.igem.org/Team:HUST-China">HOME</A></LI><LI class="dropdown1"><A class="down-scroll" href="#">PROJECT</A></LI><LI><A href="https://2016.igem.org/Team:HUST-China/project/background">Background</A></LI><LI><A href="https://2016.igem.org/Team:HUST-China/Description">Description</A></LI><LI><A href="https://2016.igem.org/Team:HUST-China/Proof">Proof of concept</A></LI><LI><A href="https://2016.igem.org/Team:HUST-China/Demonstrate">Application</A></LI><LI><A href="https://2016.igem.org/Team:HUST-China/project/inprovement">Previous improvement</A></LI><LI><A href="https://2016.igem.org/Team:HUST-China/Safety">Safety</A></LI></UL><LI class="dropdown1"><A href="#" class="down-scroll active">WETLAB</A></LI><LI><A href="https://2016.igem.org/Team:HUST-China/Experiments">Experiments</A></LI><LI><A href="https://2016.igem.org/Team:HUST-China/Results">Results</A></LI><LI><A href="https://2016.igem.org/Team:HUST-China/InterLab">Interlab</A></LI><LI><A href="https://2016.igem.org/Team:HUST-China/Notebook">Notebook</A></LI><LI><A href="https://2016.igem.org/Team:HUST-China/wetlab/protocols">Protocols</A></LI><LI class="dropdown1"><A class="down-scroll" href="#">MODELING</A></LI><LI><A href="https://2016.igem.org/Team:HUST-China/Model">Overview</A></LI><LI><A href="https://2016.igem.org/Team:HUST-China/Model/model-pro">Prokaryotic circuit</A></LI><LI><A href="https://2016.igem.org/Team:HUST-China/Model/model-euk">Eukaryotic circuit</A></LI><LI><A href="https://2016.igem.org/Team:HUST-China/Model/model-app">Application circuit</A></LI><LI class="dropdown1"><A class="down-scroll" href="#">PARTS</A></LI><LI><A href="https://2016.igem.org/Team:HUST-China/Parts">Summary</A></LI><LI><A href="https://2016.igem.org/Team:HUST-China/Basic_Part">Basic part</A></LI><LI><A href="https://2016.igem.org/Team:HUST-China/Composite_Part">Composite part</A></LI><LI><A href="https://2016.igem.org/Team:HUST-China/Part_Collection">Part Collection</A></LI><LI class="dropdown1"><A class="down-scroll" href="#">HUMAN PRACTICES</A></LI><LI><A href="https://2016.igem.org/Team:HUST-China/Integrated_Practices">Integrated HP</A></LI><LI><A href="https://2016.igem.org/Team:HUST-China/Collaborations">Collaborations</A></LI><LI class="dropdown1"><A class="down-scroll" href="#">TEAM</A></LI><LI><A href="https://2016.igem.org/Team:HUST-China/Team">Team Roster</A></LI><LI><A href="https://2016.igem.org/Team:HUST-China/Attributions">Attributions</A></LI></DIV></DIV></NAV><DIV class="container-fluid member-head"><DIV class="container"><H2 class="text-center">Protocols</H2></DIV></DIV><DIV class="container-fluid"><DIV class="text-content container" id="text-content"><ARTICLE><H2>Cloning methods</H2><H3>The PCR Reaction System</H3><TABLE border="1"><TBODY><TR><TH> Components (50uL)</TH><TH> Volume(uL)</TH></TR><TR><TD> PrimerStar Buffer </TD><TD>10 </TD></TR><TR><TD> dNTPs(2.5mM) </TD><TD>4</TD></TR><TR><TD> Primer-F(10uM) </TD><TD>1 </TD></TR><TR><TD> Primer-R(10uM)</TD><TD> 1</TD></TR><TR><TD> Template </TD><TD>1 </TD></TR><TR><TD> PrimerStar </TD><TD>0.5 </TD></TR><TR><TD> ddH2O </TD><TD>32.5</TD></TR></TBODY></TABLE><P>
                    To find the optimal temperature for our parts composite amplification, we set a gradient of annealing temperature. The result shows that 58℃ is suitable to the PCR.(Specific temperature according to different primers of components)
                </P><H3>The double enzyme digestion system (Q.cut)</H3><TABLE border="1"><TBODY><TR><TH> Components (50uL)</TH><TH> Volume(uL)</TH></TR><TR><TD>10 x Q.cut buffer </TD><TD>5</TD></TR><TR><TD> EcoRI </TD><TD>1.5</TD></TR><TR><TD> PstI </TD><TD>1.5</TD></TR><TR><TD> ddH2O </TD><TD>1</TD></TR><TR><TD> Conditions </TD><TD>37℃ 30~40min </TD></TR></TBODY></TABLE><H3>Ligation system</H3><TABLE border="1"><TBODY><TR><TH> Components (10uL)</TH><TH> Volume(uL)</TH></TR><TR><TD> T4 ligase </TD><TD>1</TD></TR><TR><TD> T4 ligase buffer </TD><TD>4</TD></TR><TR><TD> Linearized Vector </TD><TD>0.5</TD></TR><TR><TD> Insert Gene </TD><TD>4.5</TD></TR></TBODY></TABLE><H3>Infusion protocols</H3><P><SPAN style="font-weight:bold">Overlap DNA segments preparation：</SPAN></P><P>*primer design:</P><P>Every In-Fusion primer must have two characteristics: The 5’ end of the primer must contain 15 bases that are homologous to 15 bases at one end of the DNA fragment to which it will be joined:</P><P class="text-center">Fig1:Different pairing modes according to different DNA ends.</P><P><SPAN style="font-weight:bold">Materials：Clontech In-fusion HD Cloning Kit</SPAN></P><TABLE border="1"><TBODY><TR><TD> 5X In-Fusion HD Enzyme Premix </TD><TD>2 μl </TD></TR><TR><TD> Linearized Vector *</TD><TD>50~200ng</TD></TR><TR><TD> Purified PCR segments**</TD><TD>10~200ng </TD></TR><TR><TD> dH2O(deionized water)</TD><TD> Total 10 μl </TD></TR></TBODY></TABLE><P>*
                    &lt;10 kb: 50~100 ng,&gt;10 kb: 50~200 ng </P><P>**
                    &lt;0.5Kb: 10~50ng, 0.5Kb~10Kb:50ng~100ng,&gt;10Kb:50ng~200ng</P><P>Incubate the reaction for 15 min at 50°C, then place on ice.</P><H3>Transformation into E. coli DH5a</H3><P>The ligation product was transformed into E. coli DH5a Strain. The strains were cultivated in LB plate medium containing 25ug/mL chloramphenicol at 37℃.</P><H3>Reaction System of Colony PCR</H3><TABLE border="1"><TBODY><TR><TH> Components (10uL)</TH><TH> Volume(uL)</TH></TR><TR><TD> Es Taq Mix(2×) </TD><TD>5 </TD></TR><TR><TD> Primer-F(10uM) </TD><TD>0.3 </TD></TR><TR><TD> Primer-R(10uM) </TD><TD>0.3 </TD></TR><TR><TD> ddH2O </TD><TD>4.4 </TD></TR></TBODY></TABLE><H2>Fluorescence measurement assay</H2><P><SPAN style="font-weight:bold">Materials:</SPAN> E.coli DH5α,M9 medium</P><P><SPAN style="font-weight:bold">Procedures:</SPAN></P><P>1.Inoculate 0.2% volume of bacetria containing constructed pathways into medium, meanwhile, set bacteria without the pathways as a blank control. Culture them overnight in shake incubator.</P><P>2.Inoculate 10% volume of the bacteria liquid into fresh medium and culture them to an A600 of 0.6. Then dilute the liquid with corresponding medium to an A600 of 0.1 in deep-well 96-wells plate.</P><P>3.Add inducer into the liquid under the light proof condition.The volumes are as follows:</P><TABLE border="1"><TBODY><TR><TD> Bacteria </TD><TD> Inducer (for IAA 50μg/mL,for IPTG 1mg/mL)</TD></TR><TR><TD> constructed </TD><TD> +</TD></TR><TR><TD> constructed </TD><TD> - </TD></TR><TR><TD> unconstructed </TD><TD> +</TD></TR><TR><TD> unconstructed </TD><TD> - </TD></TR></TBODY></TABLE><P>Note:Each group should set three parallels.</P><P>4. After adding the inducer,take 100μl of liquid from each well to measure fluoresence and OD600.The green fluoresence should be measured using excitation and emission wavelengths of 485nm and 518nm, respectively. </P><P>5. Cover the plate with tin foil and put it into the shake incubator.Take samples 30 minutes a time,then measure the fuorescence and OD following the method above.</P><P>For instrument’s setting:</P><TABLE border="1"><TBODY><TR><TH colspan="2"> FlexStation 3</TH></TR><TR><TH>type</TH><TH>endpoint</TH></TR><TR><TD> Read Mode</TD><TD> RFUs, Top read </TD></TR><TR><TD rowspan="3"> Wavelength </TD><TD> Emission: 518nm</TD></TR><TR><TD> Excitation:485nm </TD></TR><TR><TD> Cutoff: Auto </TD></TR><TR><TD> Sensitivity </TD><TD>6 flashes per well </TD></TR><TR><TD> PMT settings </TD><TD> Auto </TD></TR><TR><TD> Autocalibrate </TD><TD> On </TD></TR></TBODY></TABLE><P>6. Analyze the data：Divide fluorescence by OD and use these results to draw curves.</P><H2>Pichia expression assay</H2><UL><LI><H3>Electroporation of Pichia</H3><P><SPAN style="font-weight:bold">Preparation of Cells:</SPAN></P><P>1. Grow 5 ml of Pichia pastoris in YPD in a 50 ml conical flask at 30°C overnight.</P><P>2. Inoculate 500 ml of fresh medium in a 2 liter flask with 0.1~0.5 ml of the overnight culture. Grow overnight again to an OD600 between 1.3~1.5.</P><P>3. Centrifuge the cells at 1500 xg for 5 minutes at +4°C. Resuspend the pellet with 500 ml of ice-cold, sterile water.</P><P>4. Centrifuge the cells as in Step 3, then resuspend the pellet with 250 ml of ice-cold, sterile water.</P><P>5. Centrifuge the cells as in Step 3, then resuspend the pellet in 20 ml of ice-cold 1 M sorbitol.</P><P>6. Centrifuge the cells as in Step 3, then resuspend the pellet in 1 ml of ice-cold 1 M sorbitol for a final volume of approximately 1.5 ml.</P><P><SPAN style="font-weight:bold">Transformation:</SPAN></P><P>1. Mix 80 ul of the cells from Step 6 (above) with 5-20 ug of linearized DNA (in 5-10 ul TE Buffer) and transfer them to an ice-cold 0.2 cm electroporation cuvette.
                        </P><P>2. Incubate the cuvette with the cells on ice for 5 minutes.</P><P>3. Pulse the cells according to the parameters for yeast (Saccharomyces cerevisiae) suggested by the manufacturer of the specific electroporation device being used.</P><P>4. Immediately add 1 ml of ice-cold 1 M sorbitol to the cuvette. Transfer the cuvette contents to a sterile microcentrifuge tube.</P><P>5. Spread 200-600 ul aliquots on MD or RDB plates.</P><P>6. Incubate the plates at 28°C until colonies appear.</P></LI><LI><H3>Total DNA Isolation from Pichia</H3><P><SPAN style="font-weight:bold">Solutions:</SPAN></P><P>Prepare the following solutions. There is not enough of some of these reagents in the kit
                           to perform this experiment.</P><P>Minimal Medium (MD, MGY)</P><P><SPAN style="font-weight:bold">Sterile water：</SPAN></P><P>SCED (1 M sorbitol, 10 mM sodium citrate, pH 7.5, 10 mM EDTA, 10 mM DTT) Zymolyase, 3 mg/ml stock solution in water                    (Seikagaku America, Inc.)</P><P>1% SDS in water</P><P>5 M potassium acetate, pH 8.9</P><P>TE buffer, pH 7.4 (10 mM Tris-HCl, pH 7.4, 1 mM EDTA, pH 8.0)</P><P>7.5 M ammonium acetate, pH 7.5</P><P>Phenol:chloroform (1:1 v/v)</P><P><SPAN style="font-weight:bold">Preparation:</SPAN></P><P>1. Grow at 30°C the recombinant strain and the parent strain to an OD600 of 5-10 in 10 ml of minimal media.
                        </P><P>2. Collect the cells by centrifugation at 1500 x g for 5-10 minutes at room temperature.
                        </P><P>3. Wash the cells with 10 ml sterile water by centrifugation as in Step 2.</P><P><SPAN style="font-weight:bold">Spheroplasting and Lysis:</SPAN></P><P>1. Resuspend the cells in 2 ml of SCED buffer, pH 7.5. Make this solution fresh.
                        </P><P>2. Add 0.1-0.3 mg of Zymolyase (mix well before adding to the cells). Incubate at 37°C for 50 minutes to achieve
                            &lt; 80% spheroplasting. </P><P>3.Add 2 ml of 1% SDS, mix gently and set on ice (0 to +4°C) for 5 minutes.
                        </P><P>4. Add 1.5 ml of 5 M potassium acetate, pH 8.9, and mix gently.
                        </P><P>5. Centrifuge at 10,000 x g for 5-10 minutes at +4°C and save the supernatant.</P><P><SPAN style="font-weight:bold">DNA Precipitation:</SPAN></P><P>1. Transfer the supernatant from Step 5 above and add 2 volumes of ethanol. Incubate at room temperature for 15 minutes.
                        </P><P>2. Centrifuge at 10,000 x g for 20 minutes at +4°C.
                        </P><P>3. Resuspend the pellet gently in 0.7 ml of TE buffer, pH 7.4 and transfer to a microcentrifuge tube.
                        </P><P>4. Gently extract with an equal volume of phenol:chloroform (1:1 v/v) followed by an equal volume of chloroform:isoamyl alcohol (24:1). Split the aqueous layer into two microcentrifuge tubes.
        </P><P>5. Add 1/2 volume of 7.5 M ammonium acetate, pH 7.5, and 2 volumes of ethanol to each tube. Place on dry ice for 10 minutes or at -20°C for 60 minutes.
                        </P><P>6. Centrifuge at 10,000 x g for 20 minutes at +4°C and wash the pellets once with 1 ml of 70% ethanol. Resuspend each pellet in 50 µl of TE buffer, pH 7.5. Determine the concentration of the DNA sample. The two samples can be stored separately or combined and stored at -20°C until ready for use.</P></LI><LI><H3>PCR Analysis of Pichia Integrants</H3><P><SPAN style="font-weight:bold">Analysis by PCR:</SPAN></P><P>1. Set up PCR reactions as follows:</P><P>  10X PCR Buffer 5 µl</P><P>  Genomic DNA (~1 µg) 5 µl</P><P>  100 mM dNTPs (25 mM each) 1 µl</P><P>  5´ AOX1 Primer (0.1 µg/µl) 5 µl</P><P>   3´ AOX1 Primer (0.1 µg/µl) 5 µl</P><P>   Sterile water to 50 µl</P><P>   Taq Polymerase (5 U/µl) 0.25 µl</P><P>For use, resuspend the lyophilized primer in 20 µl sterile water. The amount of primer may be decreased if desired. For ~20 pmoles primer, use 2 µl of each primer.For amplification controls, use 100 ng of recombinant plasmid (positive control) and 100 ng of the appropriate plasmid without insert (negative control).</P><P>2. Layer reactions with 50 µl of mineral oil. If PCR machine is equipped with a nonevaporation unit, mineral oil is not required.</P><P>3. Load thermocycler and run the following program:</P><TABLE border="1"><TBODY><TR><TD>Step</TD><TD> Temperature</TD><TD>Time </TD><TD>Cycle</TD></TR><TR><TD>Hot</TD><TD> 94℃</TD><TD> 2min</TD><TD> 1</TD></TR><TR><TD>Denaturation</TD><TD>94℃</TD><TD>1min </TD><TD rowspan="3"> 25 </TD></TR><TR><TD>Annealing</TD><TD> 55℃</TD><TD>1min</TD></TR><TR><TD>Extension</TD><TD> 72℃ </TD><TD> 1min</TD></TR><TR><TD>Final Extension</TD><TD>72℃</TD><TD>7min </TD><TD> 1</TD></TR></TBODY></TABLE><P>4. Analyze 10 µl on a 1x TAE, 1.0 % agarose gel.</P><P>5.If screening Mut+ integrants, two bands will be present. One will correspond to the size of your gene of interest, the other to the AOX1 gene (approximately 2.2 kb). If screening MutS integrants in GS115, only the band that corresponds to the gene of interest will be seen. In KM71, because of the ARG4 insert in AOX1, the PCR product is 3.6 kb. Parent plasmids will produce the following sized PCR products: </P><TABLE border="1"><TBODY><TR><TD>Vector</TD><TD> PCR Product</TD></TR><TR><TD>pHIL-D2</TD><TD> 188bp</TD></TR><TR><TD>pPIC3.5</TD><TD> 214bp</TD></TR><TR><TD>pHIL-S1</TD><TD> 262bp</TD></TR><TR><TD>pPIC9(using the 5’AOX1 primer)</TD><TD> 492bp</TD></TR><TR><TD>Ppic9(using the α-Factor primer)</TD><TD> 195bp</TD></TR></TBODY></TABLE><P>Remember to add these fragments to the size of insert to interpret PCR results.</P></LI><LI><H3>Expression of Recombinant Pichia Strains</H3><P><SPAN style="font-weight:bold">Mut+ Secreted:</SPAN></P><P>Test the effectiveness of your expression conditions by growing GS115 β-Gal which is Mut+ and expresses β-galactosidase intracellularly. Remember to include GS115 or KM71 transformed with the parent vector as a control for background intracellular expression.</P><P>1. Using a single colony, inoculate 25 ml of MGY, BMG, or BMGY in a 250 ml baffled flask. Grow at 28-30°C in a shaking incubator (250-300 rpm) until culture reaches an OD600 = 2-6 (~16-18 hours). The cells will be in log-phase growth.</P><P>2. Harvest the cells by centrifuging at 1500-3000 x g for 5 minutes at room temperature. Decant supernatant and resuspend cell pellet to an OD600 of 1.0 in MM, BMM, or BMMY medium to induce expression (approximately 100-200 ml).</P><P>3. Place culture in a 1 liter baffled flask. Cover the flask with 2 layers of sterile gauze or cheesecloth and return to incubator to continue growth.</P><P>4. Add 100% methanol to a final concentration of 0.5% methanol every 24 hours to maintain induction.</P><P>5. At each of the times indicated below, transfer 1 ml of the expression culture to a 1.5 ml microcentrifuge tube. These samples will be used to analyze expression levels and determine the optimal time post-induction to harvest. Centrifuge at maximum speed in a microcentrifuge for 2-3 minutes at room temperature.Time points (hours): 0, 6, 12, 24, 36, 48, 60, 72, 84, and 96(or more if you need).</P><P>6. For secreted expression, transfer the supernatant to a separate tube. Store the supernatant and the cell pellets at -80°C until ready to assay. Freeze quickly in liquid N2 or a dry ice/alcohol bath.</P><P>7. Analyze the supernatants and cell pellets for protein expression by Coomassiestained SDS-PAGE and Western blot or functional assay.</P></LI><LI><H3>Analysis by SDS-Polyacrylamide Gel Electrophoresis</H3><P><SPAN style="font-weight:bold">Solutions:</SPAN></P><P>purified water</P><P>0.5 M Tris-HCl, pH 6.8</P><P>0.5 M Tris-HCl, pH 8.8</P><P>10% (w/v) SDS</P><P>Acrylamide /Bis-acrylamide (30%/0.8% w/v)</P><P>10% (w/v) ammonium persulfate (AP)</P><P>TEMED</P><P>Coomassiae billiant blue R250</P><P>5x Sample Buffer</P><TABLE border="1"><TBODY><TR><TD>10% w/v</TD><TD> SDS</TD></TR><TR><TD>10 mM</TD><TD> Dithiothreitol,or bata-mercapto-ethanol</TD></TR><TR><TD>20% v/v</TD><TD> Glycerol</TD></TR><TR><TD>0.2M</TD><TD> Tris-HCl,pH 6.8</TD></TR><TR><TD>0.05% w/v</TD><TD> Bromophenolbule</TD></TR></TBODY></TABLE><P>1x Running Buffer</P><TABLE border="1"><TBODY><TR><TD>25mM</TD><TD> Tris-HCl</TD></TR><TR><TD>200mM</TD><TD>Glycine </TD></TR><TR><TD>0.1%(w/v)</TD><TD> SDS</TD></TR></TBODY></TABLE><P><SPAN style="font-weight:bold">Preparation of supernatant (Secreted Expression):</SPAN></P><P>1. Thaw supernatants and place on ice.</P><P>2. Mix 50 µl of the supernatant with 50 µl of SDS-PAGE Gel Loading buffer.</P><P>3. Boil 10 minutes, then load 10-30 µl onto the gel. Remaining sample may be stored at -20°C for Western blots, if necessary. Supernatants may be stored at -80°C for further analysis.</P><P><SPAN style="font-weight:bold">SDS-page:</SPAN></P><P>1. Make the separating gel:</P><P>Set the casting frames (clamp two glass plates in the casting frames) on the casting stands. Prepare the gel solution in a separate small beaker.Swirl the solution gently but thoroughly. Pipet appropriate amount of separating gel solution into the gap between the glass plates.To make the top of the separating gel be horizontal, fill in water  into the gap until a overflow.Wait for 20-30min to let it gelate.</P><P>2.Make the stacking gel:</P><P>Discard the water and you can see separating gel left. Pipet in stacking gel untill a overflow.
Insert the well-forming comb without trapping air under the teeth. Wait for 20-30min to let it gelate.
</P><P>3. Make sure a complete gelation of the stacking gel and take out the comb. Take the glass plates out of the casting frame and set them in the cell buffer dam. Pour the running buffer (electrophoresis buffer) into the inner chamber and keep pouring after overflow untill the buffer surface reaches the required level in the outer chamber.</P><P>4. Load prepared samples into wells and make sure not to overflow. Don't forget loading protein marker into the first lane. Then cover the top and connect the anodes.</P><P>5. Set an appropriate volt and run the electrophoresis when everything's done.</P><P>6. As for the total running time, stop SDS-PAGE running when the downmost sign of the protein marker (if no visible sign, inquire the manufacturer) almost reaches the foot line of the glass plate. Generally, about 2.5 hours for a 100V voltage and a 12% separating gel. </P><P>7. Stain the gel with Coomassiae billiant blue R250 for 4 hours and destain the gel with water at least 2 hours. </P><P>For a 5 ml stacking gel:</P><TABLE border="1"><TBODY><TR><TD>H2O</TD><TD> 2.975ml</TD></TR><TR><TD>0.5M Tris-HCl,pH 6.8</TD><TD>0.05ml</TD></TR><TR><TD>10%(w/v)SDS</TD><TD> SDS</TD></TR><TR><TD>Acrylamide/Bis-acrylamide(30%/0.8 w/v)</TD><TD> 0.67ml</TD></TR><TR><TD>10%(w/v)ammonium persulfate(AP)</TD><TD>0.05ml</TD></TR><TR><TD>TEMED</TD><TD> 0.005ml</TD></TR></TBODY></TABLE><P>For a 10ml separating gel:</P><TABLE border="1"><TBODY><TR><TD>Acrylamide percentage</TD><TD> 6%</TD><TD>8% </TD><TD>10%</TD><TD>12% </TD><TD>15%</TD></TR><TR><TD>H2O</TD><TD>5.2ml </TD><TD>4.6ml</TD><TD>3.8ml </TD><TD>3.2ml</TD><TD>2.2ml</TD></TR><TR><TD>Acrylamide/Bis-acrylamide(30%/0.8 w/v)</TD><TD> 2ml</TD><TD>2.6ml</TD><TD>3.4ml</TD><TD>4ml</TD><TD>5ml</TD></TR><TR><TD>1.5M Tris(pH=8.8)</TD><TD>2.6ml</TD><TD>2.6ml</TD><TD>2.6ml</TD><TD>2.6ml </TD><TD>2.6ml</TD></TR><TR><TD>10%(w/v)SDS</TD><TD>0.1ml</TD><TD>0.1ml</TD><TD>0.1ml</TD><TD>0.1ml</TD><TD>0.1ml</TD></TR><TR><TD>10%(w/v)ammonium persulfate(AP)</TD><TD>100uL</TD><TD>100uL</TD><TD>100uL</TD><TD>100uL</TD><TD>100uL</TD></TR><TR><TD>Acrylamide percentage</TD><TD>10uL</TD><TD>10uL </TD><TD>10uL</TD><TD>10uL </TD><TD>10uL</TD></TR></TBODY></TABLE></LI></UL><H2>Protein BCA quantification assay</H2><P><SPAN style="font-weight:bold">Equipment:</SPAN></P><P>Multiskan plate reader, TaKaRa BCA Protein Assay Kit (No. T9300A), transparent 96-well plate, Amicon Ultra -4mL Millipore</P><P><SPAN style="font-weight:bold">Sample preparation:</SPAN></P><P>1. Culture yeast strain(Y.Lipolytica) in 5ml YPD medium,30℃,200r/min,overnight.</P><P>2. Transfer 1% culture to 50ml YPD for amplification,30℃,200r/min,cultivate for 60~70h.</P><P>3. Centrifuge the sample at 3000rpm to collect supernatant.</P><P>4. Get 8 times concentrated sample by ultrafiltration (MWCO 19kDa,3000rpm,4℃).</P><P><SPAN style="font-weight:bold">Measuring:</SPAN></P><P>1. Make working reagent, BCA kit,A reagent and B reagent mix with ratio 100:1.</P><P>
                    2. Dilute protein standard reagent in 96-well plate to make standard curve.</P><P>
                    3. Add 50ul concentrated samples in 96-well plate for each 3 parallels.</P><P>
                    4. Add 200ul working reagent, vibrate to mix up,incubate in 37℃ for 30min.</P><P>
                    5. Take YPD medium as blank to measure 562nm absorption under Multiskan plate reader.
                </P><H2>β-galactosidase activity assay</H2><P><SPAN style="font-weight:bold">Materials:</SPAN></P><P>1)E.coli transformed with the whole application circuit plasmid 
(placm-pRE-RBS-Cro-RBS-CII-TT-patp2-RBS-CI-TT-pR-RBS-CIII-RBS-ompA-iLDH-TT-pRM-RBS-beta-galactosidase)</P><P>2)1M Na2CO3,ONPG,β-ME,LB medium</P><P>3)Z-buffer：0.04M NaH2PO4, 0.06M Na2HPO4, 0.001M MgSO4, 0.01M KCl</P><P>4)Z-buffer/β-ME: Per 10mL Z-buffer with 27uL β-ME</P><P>5)4mg/mL ONPG: 0.008gONPG dissolved in 2mL Z-buffer/β-ME</P><P><SPAN style="font-weight:bold">Procedures:</SPAN></P><P>1) Inoculate the bacetria containing the plasmid into LB liquid medium which has different levels of PH(PH=6.5, 7.5, 8.5, respectively).Then,culture them overnight in shake incubator at 37℃</P><P>2) Add 250uL above-mentioned cultures into 2mL LB liquid medium（in 20mL small glass bottle）. Each colony is inoculated to 3 bottles. Culture them at 28℃ for 2.5h. Then,measure the OD value.</P><P>3) Pellet 2mL culture by centrifugation at 12000rpm for 30s. Discard the supernatant.</P><P>Add 2mL Z-buffer and resuspend. Centrifuge the mixture at 12000rpm for 30s and discard the supernatant. </P><P>Add 300uL Z-buffer and resuspend. Add 6uL lysozyme and incubate it at room temperature for 2min.</P><P>Cool the mixture with liquid nitrogen for 30s and thaw it at room temperature. Repeat this procedure for 3 times. </P><P>Aspirate 100uL mixture to a new centrifuge tube. Add 700uL Z-buffer/β-ME and 200uL ONPG. Fully mix. Warm it at 30℃ in water bath and do the timekeeping until yellow substance appear. Add 400uL 1M Na2CO3 and fully mix to terminate the reaction.</P><P>Centrifuge the mixture at 12000rpm for 2min. Aspirate the supernatant and add them to 96-well plates (six wells for each tube and 200uL for each well ). Use the Multiskan Spectrum to determine the absorbance at wavelength of 420nm,550nm,and 600nm,respectively.</P><P>Enzyme-activity unit (U) = 1000*OD420/(OD600*V) *TIME *Dilution Ratio</P><P>1 U：The amount that one cell catalyzes the conversion of 1μmol substrate (ONPG) per minute. </P><P>Unit：Miller</P><P>NOTE:ONPG should be stored at -20℃.</P></ARTICLE></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>