<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Illinois_Research_articles"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:Illinois/Research_articles">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Illinois/Research_articles&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Illinois/Research_articles&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Illinois/Research_articles&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Illinois/Research_articles" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Illinois/Research articles</H1><DIV id="bodyContent"><H3 id="siteSub">From 2008.igem.org</H3><P><TABLE class="links"><TBODY><TR><TD align="center"><A class="mainLinks" href="https://2008.igem.org/Team:Illinois">Home</A></TD><TD align="center"><A class="mainLinks" href="https://2008.igem.org/Team:Illinois/Team">Team</A></TD><TD align="center"><A class="mainLinks" href="https://2008.igem.org/Team:Illinois/Project">Project</A></TD><TD align="center"><A class="mainLinks" href="https://2008.igem.org/Team:Illinois/Notebook">Notebook</A></TD><TD align="center" id="navactive"><A class="mainLinks" href="https://2008.igem.org/Team:Illinois/Research_articles">Research Articles</A></TD><TD align="center"><A class="mainLinks" href="https://2008.igem.org/Team:Illinois/Parts">Parts</A></TD><TD align="center"><A class="mainLinks" href="https://2008.igem.org/Team:Illinois/Current_Tasks">Protocols</A></TD><TD align="center"><A class="mainLinks" href="https://2008.igem.org/Team:Illinois/Pictures">Pictures</A></TD></TR></TBODY></TABLE></P><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Directed_Evolution"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Directed Evolution</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Method_for_Directed_Evolution_and_functional_Cloning_of_Enzymens"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Method for Directed Evolution and functional Cloning of Enzymens</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Creation_of_GPCR-based_chemical_sensors_by_directed_evolution_in_yeast"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Creation of GPCR-based chemical sensors by directed evolution in yeast</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-4"><A href="#Bio-sensors"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Bio-sensors</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Construction_of_a_Fluorescent_Biosensor_Family"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Construction of a Fluorescent Biosensor Family</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Protein_Engineering_and_the_Development_of_Generic_Biosensors"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">Protein Engineering and the Development of Generic Biosensors</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#Heterologous_Expression_of_the_human_D2s_dopamine_receptor_in_protease-deficient_Saccharomyces_cerevisiae_strains"><SPAN class="tocnumber">2.3</SPAN><SPAN class="toctext">Heterologous Expression of the human D2s dopamine receptor in protease-deficient Saccharomyces cerevisiae strains</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#Functional_expression_of_olfactory_receptors_in_yeast_and_development_of_a_bioassay_for_odorant_screening"><SPAN class="tocnumber">2.4</SPAN><SPAN class="toctext">Functional expression of olfactory receptors in yeast and development of a bioassay for odorant screening</SPAN></A></LI><LI class="toclevel-2 tocsection-9"><A href="#Foreign_Gene_Expression_in_Yeast:_A_Review"><SPAN class="tocnumber">2.5</SPAN><SPAN class="toctext">Foreign Gene Expression in Yeast: A Review</SPAN></A></LI><LI class="toclevel-2 tocsection-10"><A href="#Gentically_Engiueered_Whole-Cell_Sensing_Systems:_Coupling_Biological_Recognition_with_Reporter_Genes"><SPAN class="tocnumber">2.6</SPAN><SPAN class="toctext">Gentically Engiueered Whole-Cell Sensing Systems: Coupling Biological Recognition with Reporter Genes</SPAN></A></LI><LI class="toclevel-2 tocsection-11"><A href="#Immunosensor_for_the_detection_of_Vibrio_cholerae_O1_using_surface_plasmon_resonance"><SPAN class="tocnumber">2.7</SPAN><SPAN class="toctext">Immunosensor for the detection of Vibrio cholerae O1 using surface plasmon resonance</SPAN></A></LI><LI class="toclevel-2 tocsection-12"><A href="#Salmonella_and_salmonellosis"><SPAN class="tocnumber">2.8</SPAN><SPAN class="toctext">Salmonella and salmonellosis</SPAN></A></LI><LI class="toclevel-2 tocsection-13"><A href="#Yeast_System_as_a_Screening_Tool_for_Pharmacological_Assessment_of_G_Protein_Coupled_Receptors"><SPAN class="tocnumber">2.9</SPAN><SPAN class="toctext">Yeast System as a Screening Tool for Pharmacological Assessment of G Protein Coupled Receptors</SPAN></A></LI><LI class="toclevel-2 tocsection-14"><A href="#Detection_of_water-borne_E._coli_O157_using_the_integrating_waveguide_biosensor"><SPAN class="tocnumber">2.10</SPAN><SPAN class="toctext">Detection of water-borne E. coli O157 using the integrating waveguide biosensor</SPAN></A></LI><LI class="toclevel-1 tocsection-15"><A href="#E.coli"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">E.coli</SPAN></A></LI><LI class="toclevel-2 tocsection-16"><A href="#Host_strain_influences_on_supercoiled_plasmid_DNA_production_in_Escherichia_coli:_Implications_for_efficient_design_of_large-scale_processes."><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">Host strain influences on supercoiled plasmid DNA production in Escherichia coli: Implications for efficient design of large-scale processes.</SPAN></A></LI><LI class="toclevel-1 tocsection-17"><A href="#Yeast_signal_transduction"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Yeast signal transduction</SPAN></A></LI><LI class="toclevel-2 tocsection-18"><A href="#Yeast-Based_Fluorescence_Reporter_Assay_of_G_Protein-coupled_Receptor_Signalling_for_Flow_Cytometric_Screening:_FAR1-Disruption_Recovers_Loss_of_Episomal_Plasmid_Caused_by_Signalling_in_Yeast"><SPAN class="tocnumber">4.1</SPAN><SPAN class="toctext">Yeast-Based Fluorescence Reporter Assay of G Protein-coupled Receptor Signalling for Flow Cytometric Screening: FAR1-Disruption Recovers Loss of Episomal Plasmid Caused by Signalling in Yeast</SPAN></A></LI><LI class="toclevel-1 tocsection-19"><A href="#Yeast_Cell_Wall"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Yeast Cell Wall</SPAN></A></LI><LI class="toclevel-2 tocsection-20"><A href="#Functional_analysis_of_heterologous_GPCR_signalling_pathways_in_yeast"><SPAN class="tocnumber">5.1</SPAN><SPAN class="toctext">Functional analysis of heterologous GPCR signalling pathways in yeast</SPAN></A></LI><LI class="toclevel-2 tocsection-21"><A href="#C5a_Receptor_Activation"><SPAN class="tocnumber">5.2</SPAN><SPAN class="toctext">C5a Receptor Activation</SPAN></A></LI></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Directed_Evolution">Directed Evolution</SPAN></H2><H3><SPAN class="mw-headline" id="Method_for_Directed_Evolution_and_functional_Cloning_of_Enzymens">Method for Directed Evolution and functional Cloning of Enzymens</SPAN></H3><P>Henrik Pedersen, Swen Holder, Daniel P, Sutherlin, Urs Schwitter, David S. King. and Peter G. Schultz. Proc. Natl. Acad. Sci. USA Vol. 95, pp. 10523-10528, September 1998.<A href="http://igem-illini.googlegroups.com/web/Method%20for%20Directed%20Evolution%20and%20functional%20Cloning%20of%20Enzymens.pdf" class="external autonumber" rel="nofollow">[1]</A></P><UL><LI>Summary
</LI><LI>Bacically give the methodology for preforming directed evolution on a specific enzyme, but can probably be adapted for any other protein provided we can isolate the ligand we are trying to isolate
</LI><LI>The methods are very involved and complex, but are very specific for how to prepare substrates.
</LI><LI>Create substrate with proteins to be identified and then create conditions so that only very specific bonding can occur. Get rid of elements that do not have a high bond affinity, then un-bond the proteins. 
</LI></UL><LI>Note: Some one with more biology background might want to take a look at this, as it deals with really specific topics, that I do not have experience with. Also some relevant over-view things were highlighted.
</LI><LI>--Dan Knipmeyer
</LI><H3><SPAN class="mw-headline" id="Creation_of_GPCR-based_chemical_sensors_by_directed_evolution_in_yeast">Creation of GPCR-based chemical sensors by directed evolution in yeast</SPAN></H3><P>Addison D.Ault and James R.Broach. <I>Protein Engineering, Design &amp; Selection</I><A href="http://igem-illini.googlegroups.com/web/Creation%20of%20GPCR-based%20chemical%20sensors%20by%20directed%20evolution%20in%20yeast.pdf" class="external autonumber" rel="nofollow">[2]</A></P><UL><LI>Relevance to iGEM: High
</LI><LI>Subjects:
</LI><LI>G protein coupled receptors (GPCRs)specificity vs. sensitiyity
</LI><LI>Directed evolution in GPCRs
</LI><LI>Functional expression of heterologous GPCRs in yeast cells
</LI></UL><LI>Summary:
</LI><LI>The authors took a Human GPCR and used simmilarites between yeast mating response pathways and the human signal transduction pathways to couple a human GPCR to the yeast pheromone respone pathway. They then used  random mutagenesis to create hybrids. Ultimately the hybrids were more sensitive, but had the same specificity for the compounds they sere tested on.
</LI><LI>--Dan Knipmeyer
</LI><H2><SPAN class="mw-headline" id="Bio-sensors">Bio-sensors</SPAN></H2><H3><SPAN class="mw-headline" id="Construction_of_a_Fluorescent_Biosensor_Family">Construction of a Fluorescent Biosensor Family</SPAN></H3><P>Robert M. De Lorimier, J. Jeff Smith, Mary A. Dwyer, Loren L. Looger, Kevin M. Sali, Chad D. Paavola, Shahir S. Rizk, Shamil Sadigov, David W. Conrad, Leslie Loew and Homme W. Hellinga 
<I>Protein Sci. </I>2002 11: 2655-2675<A href="http://igem-illini.googlegroups.com/web/Construction%20of%20fluorescent%20biosensor.pdf" class="external autonumber" rel="nofollow">[3]</A></P><UL><LI>Relevance to iGEM: medium/low
</LI><LI>Subjects: 
</LI><LI>bacterial periplasmic (between the cell wall and cell membrane) binding proteins (bPBPs); 
</LI><LI>fluorescence; 
</LI><LI>fusion proteins
</LI></UL><LI>Summary:  
</LI><LI>The authors took bPBPs and did two things to them: they mutated them to change their ligand specificity on a limited scale, and they attached a fluorescent domain to the bPBP so that ligand binding would cause a change in fluorescence without the need for a full signal transduction pathway.  
</LI><LI>The authors' main focus was on the fluorescence.  We could conceivably use this type of protein to accomplish the biosensor; however, the authors state that not much is known about the natural bPBPs, so it might be difficult to find a starting point in terms of a specific bPBP.
</LI><LI>--Dave Luedtke
</LI><H3><SPAN class="mw-headline" id="Protein_Engineering_and_the_Development_of_Generic_Biosensors">Protein Engineering and the Development of Generic Biosensors</SPAN></H3><P>Homme W. Hellinga and Jonathan S. Marvin. <I>Trends in Biotechnology</I> Vol. 16, Issue 4 April 1998 pp. 183-189 <A href="http://igem-illini.googlegroups.com/web/Genetic%20Biosensors.pdf" class="external autonumber" rel="nofollow">[4]</A></P><UL><LI>Relevent quotes.
</LI><LI>Modular molecular engineering system in which the integrated signal transduction site are so constructed so that each can be changed separately with out destroying the communication between them.
</LI><LI>Maintain constant signal transduction function creating generic biosensor to share same detection instrumentation.
</LI><LI>Identify protein with particularly well behaved intrinsic signal transduction function and construct appropriate binding site.
</LI><LI>Use of alpha-haemolysin and GFP
</LI></UL><LI>Hinge action proteins for FRET analysis.
</LI><LI>--Dan Knipmeyer
</LI><H3><SPAN class="mw-headline" id="Heterologous_Expression_of_the_human_D2s_dopamine_receptor_in_protease-deficient_Saccharomyces_cerevisiae_strains">Heterologous Expression of the human D2s dopamine receptor in protease-deficient Saccharomyces cerevisiae strains</SPAN></H3><P>Peter Sander, Sylvia GrÃ¼newald, Marion Bach, Winfried Haase, Helmut ReilÃ¤nder, Hartmut Michel (1994) Heterologous Expression of the Human D2S Dopamine Receptor in Protease-Deficient Saccharomyces cerevisiae Strains
European Journal of Biochemistry 226 (2) , 697â705 <A href="http://igem-illini.googlegroups.com/web/heterologous%20expression%20of%20receptor%20in%20yeast%20strain.pdf" class="external autonumber" rel="nofollow">[5]</A></P><UL><LI>Functional expression of human receptor in yeast
</LI><LI>Overcome problem of protein degradation during growth or during protein preparation
</LI></UL><P>-Aleem Zafar
</P><H3><SPAN class="mw-headline" id="Functional_expression_of_olfactory_receptors_in_yeast_and_development_of_a_bioassay_for_odorant_screening">Functional expression of olfactory receptors in yeast and development of a bioassay for odorant screening</SPAN></H3><P>Jasmina Minic, Marie-Annick Persuy, Elodie Godel, Josiane Aioun, Ian Connerton, Roland Salesse, Edith Pajot-Augy (2005) Functional expression of olfactory receptors in yeast and development of a bioassay for odorant screening
FEBS Journal 272 (2) , 524â537 <A href="http://igem-illini.googlegroups.com/web/Functional%20expression%20of%20olfactory%20receptors%20in%20yeast.pdf" class="external autonumber" rel="nofollow">[6]</A></P><UL><LI>Relavent quotes
</LI><LI>functional similarities between signal transduction cascades of G protein-coupled receptor in mammalian cells and the pheromone response pathway in yeast to develop a novel biosensor for odorant screening.
</LI><LI>The aim of this study was to optimize the baker's yeast Saccharomyces cerevisiae as a host system for properly expressing an OR at the plasma membrane, and for its efficient coupling to a signaling pathway that produces a measurable response to odorant stimulation.
</LI><LI>The yeast system was chosen for several reasons. Firstly, S. cerevisiae has been successfully used for functional expression of many GPCRs [22â28]. Secondly, yeast constitutes an attractive system to study membrane receptors providing a null background for mammalian GPCRs and G proteins. Finally, yeast cells may provide a means for detailed investigation of receptor pharmacology in vivo through the use of sensitive reporter systems that take advantage of the functional homologies between yeast pheromone and mammalian GPCR signaling pathways.
</LI></UL><P>-Aleem Zafar
</P><H3><SPAN class="mw-headline" id="Foreign_Gene_Expression_in_Yeast:_A_Review">Foreign Gene Expression in Yeast: A Review</SPAN></H3><P>Michael A. Romanos, Carol A. Scorer, Jeffrey J. Clare. Foreign gene expression in yeast: a review. Yeast Volume 8 (6) pg 423-488. 1992 <A href="http://igem-illini.googlegroups.com/web/foreign%20gene%20expression%20in%20yeast.pdf" class="external autonumber" rel="nofollow">[7]</A></P><UL><LI>A great review
</LI><LI>Table of Contents
</LI><LI>Introduction, Transformation and selectable markers, Episomal vectors, Integrating vectors, Transcriptional promoters and terminators, Factors affecting intracellular expression, Secretion of foreign proteins, Expression in non-Saccharomyces yeasts, Physiology of foreign gene expression
</LI></UL><H3><SPAN class="mw-headline" id="Gentically_Engiueered_Whole-Cell_Sensing_Systems:_Coupling_Biological_Recognition_with_Reporter_Genes">Gentically Engiueered Whole-Cell Sensing Systems: Coupling Biological Recognition with Reporter Genes</SPAN></H3><P>Genetically Engineered Whole-Cell Sensing Systems: Coupling Biological Recognition with Reporter Genes
Daunert, S., Barrett, G., Feliciano, J.S., Shetty, R.S., Shrestha, S., and Smith-Spencer, W. <I>Chem. Rev.</I>, 100, 7, 2705 - 2738, 2000
<A href="http://pubs.acs.org/cgi-bin/abstract.cgi/chreay/2000/100/i07/abs/cr990115p.html" class="external text" rel="nofollow">Genetically Engineered Whole-Cell Sensing Systems: Coupling Biological Recognition with Reporter Genes</A></P><UL><LI>Discusses several reporters for biosensing
</LI><LI>Non-specific and specific biosensing systems
</LI><LI>Further sensing systems
</LI></UL><P><A href="/User:Alzar" title="User:Alzar">Alzar</A></P><H3><SPAN class="mw-headline" id="Immunosensor_for_the_detection_of_Vibrio_cholerae_O1_using_surface_plasmon_resonance">Immunosensor for the detection of Vibrio cholerae O1 using surface plasmon resonance</SPAN></H3><P>Jy-Young Jyounga, SaHyun Hongc, Woochang Leea, and Jeong-Woo Choi
</P><P>Biosensors and Bioelectronics
Volume 21, Issue 12, 15 June 2006, Pages 2315-2319
</P><P><A href="http://www.sciencedirect.com.proxy2.library.uiuc.edu/science?_ob=ArticleURL&amp;_udi=B6TFC-4HPD3MH-1&amp;_user=571676&amp;_coverDate=06%2F15%2F2006&amp;_rdoc=1&amp;_fmt=&amp;_orig=search&amp;_sort=d&amp;view=c&amp;_acct=C000029040&amp;_version=1&amp;_urlVersion=0&amp;_userid=571676&amp;md5=dd1529898e6d906dd6b727814c1f6d4e" class="external text" rel="nofollow">Immunosensor for the detection of Vibrio cholerae O1 using surface plasmon resonance</A></P><UL><LI>Quick explanation of the genus Vibrio
</LI><LI>Discusses an alternative to Finkelstein's plating technique by using immunofluorescense
</LI><LI>The method detects cholera in aquatic samples, like what we hope to do
</LI></UL><P>--Joleen Su
</P><H3><SPAN class="mw-headline" id="Salmonella_and_salmonellosis">Salmonella and salmonellosis</SPAN></H3><P>Kenneth Todar University of Wisconsin-Madison Department of Bacteriology
<A href="http://www.textbookofbacteriology.net/salmonella.html" class="external autonumber" rel="nofollow">[8]</A>
genus Salmonella has three kinds of major antigens with diagnostic or identifying applications:
</P><UL><LI> Somatic (O) or Cell Wall Antigens 
</LI><LI> 20 types of Antigen O
</LI><LI> O antigen results in partial loss of virulence, suggesting that this portion of LPS is important during a host-parasite interaction 
</LI></UL><UL><LI> Surface (Envelope) Antigens 
</LI><LI> Surface antigen that is well known: the Vi antigen. The Vi antigen occurs in only three Salmonella serovars (out of about 2,200): Typhi, Paratyphi C, and Dublin
</LI></UL><UL><LI> Flagellar (H) Antigens 
</LI><LI> Mixing salmonella cells with flagella-specific antisera  gives a characteristic pattern of agglutination (bacteria are loosely attached to each other by their flagella and can be dissociated by shaking)
</LI></UL><P><A href="http://images.google.com/imgres?imgurl=http://staff.vbi.vt.edu/pathport/pathinfo_images/Salmonella/Salmonella_typhi/Ty2_genome.JPG&amp;imgrefurl=http://pathport.vbi.vt.edu/pathinfo/pathogens/enterica.html&amp;h=662&amp;w=644&amp;sz=84&amp;hl=en&amp;start=8&amp;sig2=aGAtw4DyYOOkCxFGCEYKRQ&amp;um=1&amp;tbnid=vWIgKkmDzYUYTM:&amp;tbnh=138&amp;tbnw=134&amp;ei=m3dNSKXwD43y0QS6pNSrCQ&amp;prev=/images%3Fq%3DSurface%2Bof%2Bsalmonella%2Btyphi%26um%3D1%26hl%3Den%26rlz%3D1G1GGLQ_ENUS263%26sa%3DN" class="external autonumber" rel="nofollow">[9]</A> This website has a description of all the current tests used to detect Salmonella Typhi:
</P><UL><LI> Widal tube agglutination test: involves the use of bacterial suspensions of S typhi and S paratyphi `A' and `B', treated to retain only the 'O' and 'H' antigens. These antigens are employed to detect corresponding antibodies in the serum of a patient.
</LI><LI> Typhidot: allows separate evaluation of the presence of specific serum immunoglobulin (Ig) g and m antibodies to OMP (50 kDa Outer membrane protein). Identification of specific IgG but not IgM in the serum of a febrile child is taken as a positive result.
</LI><LI> Serotype Typhi IgM dipstick assay: based on the binding of S. Typhi specific IgM antibodies in samples to S. Typhi lipopolysaccharide (LPS) antigen and the staining of bound antibodies, by an anti-human IgM antibody, conjugated to colloidal dye particles.
</LI></UL><P>-- Kiruthika
</P><H3><SPAN class="mw-headline" id="Yeast_System_as_a_Screening_Tool_for_Pharmacological_Assessment_of_G_Protein_Coupled_Receptors">Yeast System as a Screening Tool for Pharmacological Assessment of G Protein Coupled Receptors</SPAN></H3><P><A href="http://web.ebscohost.com/ehost/detail?vid=1&amp;hid=14&amp;sid=5f2b7f53-d363-47bc-b714-de3ad9f12a87%40sessionmgr9" class="external text" rel="nofollow">Yeast System as a Screening Tool for Pharmacological Assessment of G Protein Coupled Receptors</A></P><H3><SPAN class="mw-headline" id="Detection_of_water-borne_E._coli_O157_using_the_integrating_waveguide_biosensor">Detection of water-borne E. coli O157 using the integrating waveguide biosensor</SPAN></H3><P><A href="http://www.creatvmicrotech.com/Biosensor-journal%20papers/1.paper_15-2005-Zhu-BB1.pdf" class="external text" rel="nofollow">Detection of water-borne E. coli O157 using the integrating waveguide biosensor</A></P><H2><SPAN class="mw-headline" id="E.coli">E.coli</SPAN></H2><H3><SPAN class="mw-headline" id="Host_strain_influences_on_supercoiled_plasmid_DNA_production_in_Escherichia_coli:_Implications_for_efficient_design_of_large-scale_processes.">Host strain influences on supercoiled plasmid DNA production in Escherichia coli: Implications for efficient design of large-scale processes.</SPAN></H3><P>Yau SY, Keshavarz-Moore E, Ward J.
</P><P>Advanced Centre of Biochemical Engineering, Department of Biochemical Engineering, University College London, Torrington Place, London WC1E 7JE, United Kingdom; telephone: +44â2076792961; fax: +44â2079163943.
</P><P>Summary:
</P><UL><LI> Set out to investigate if E. coli genotype plays a significant role in host strain selection for optimal processing of plasmid DNA based on both quality and quantity of supercoiling
</LI><LI> 17 E. coli commercial and non-commercial strains were selected and their available genetic backgrounds were researched in the open literature. Growth characteristics of all the strains were considered and made impartial by using a common medium and growth condition platform.
</LI><LI> Can screen the number of strains which are likely to give good productivity early on, before any further optimisation and verification is performed, both for small and large plasmids.
</LI><LI> Found that high specific growth rates of some strains did not affect the level of DNA supercoiling but did influence the total plasmid yield, potentially an important aspect in the design of fermentation strategy. Interestingly, a few host/plasmid combinations result in what appears to be runaway plasmid replication.
</LI></UL><UL><LI>Potentially helpful for deciding strains to use to amplify plasmids
</LI></UL><P>-Bobak</P><H2><SPAN class="mw-headline" id="Yeast_signal_transduction">Yeast signal transduction</SPAN></H2><H3><SPAN class="mw-headline" id="Yeast-Based_Fluorescence_Reporter_Assay_of_G_Protein-coupled_Receptor_Signalling_for_Flow_Cytometric_Screening:_FAR1-Disruption_Recovers_Loss_of_Episomal_Plasmid_Caused_by_Signalling_in_Yeast">Yeast-Based Fluorescence Reporter Assay of G Protein-coupled Receptor Signalling for Flow Cytometric Screening: FAR1-Disruption Recovers Loss of Episomal Plasmid Caused by Signalling in Yeast</SPAN></H3><P>Jun Ishii1, Tsutomu Tanaka2, Shizuka Matsumura3, Kenji Tatematsu4,
Shunâichi Kuroda4, Chiaki Ogino3, Hideki Fukuda2 and Akihiko Kondo3,*
1Department of Molecular Science and Material Engineering, Graduate School of Science and Technology;
2Organization of Advanced Science and Technology; 3Department of Chemical Science and Engineering,
Graduate School of Engineering, Kobe University; and 4Department of Structural Molecular Biology,
Institute of Scientific and Industrial Research, Osaka University, Japan

J. Biochem. 143, 667â674 (2008)
doi:10.1093/jb/mvn018
</P><UL><LI>covers signal transduction concepts within yeast
</LI><LI>discusses timetable of signal transduction activation
</LI></UL><H2><SPAN class="mw-headline" id="Yeast_Cell_Wall">Yeast Cell Wall</SPAN></H2><H3><SPAN class="mw-headline" id="Functional_analysis_of_heterologous_GPCR_signalling_pathways_in_yeast">Functional analysis of heterologous GPCR signalling pathways in yeast</SPAN></H3><P>Graham Ladds, Alan Goddard and John Davey
Department of Biological Sciences, University of Warwick, Coventry, CV4 7AL, UK
</P><UL><LI>This article is the one that brought the cell wall to light as a potential problem for our purposes
</LI><LI>It claims that &quot;Potencies for small ligands (Mw 1000) are similar in yeast and mammals, those for intermediate ligands (Mw 5,000) are about two orders of magnitude less in yeast, and large ligands such as chemokines (polypeptides with 70â80 residues) are often unable to activate receptors when applied exogenously to yeast.&quot;
</LI></UL><H3><SPAN class="mw-headline" id="C5a_Receptor_Activation">C5a Receptor Activation</SPAN></H3><P>Thomas J. Baranski, Paul Herzmark, Olivier Lichtarge, Basil O. Gerber, Joshua Trueheart, Elaine C. Meng, Taroh Iiri, SÃ¸ren P. Sheikh, and Henry R. Bourne
From the Department of Cellular and Molecular Pharmacology, University of California, San Francisco, California 94143 and Cadus Pharmaceutical Corporation, Tarrytown, New York 10591-6705
</P><UL><LI>This article is one of the articles cited by the previous article
</LI><LI>The GPCR used was activated by C5a, a chemokine.
</LI><LI>The only mention of the cell wall is:
</LI><LI>&quot;A separate plasmid ... allowed autocrine expression of the C5a ligand as an a-factor prepro/C5a ligand fusion protein; <B>autocrine expression was necessary because C5a cannot traverse the yeast cell wall.&quot; </B></LI></UL><P><TABLE class="links"><TBODY><TR><TD align="center"><A class="mainLinks" href="https://2008.igem.org/Team:Illinois">Home</A></TD><TD align="center"><A class="mainLinks" href="https://2008.igem.org/Team:Illinois/Team">Team</A></TD><TD align="center"><A class="mainLinks" href="https://2008.igem.org/Team:Illinois/Project">Project</A></TD><TD align="center"><A class="mainLinks" href="https://2008.igem.org/Team:Illinois/Notebook">Notebook</A></TD><TD align="center"><A class="mainLinks" href="https://2008.igem.org/Team:Illinois/Research_articles">Research Articles</A></TD><TD align="center"><A class="mainLinks" href="https://2008.igem.org/Team:Illinois/Parts">Parts</A></TD><TD align="center"><A class="mainLinks" href="https://2008.igem.org/Team:Illinois/Current_Tasks">Protocols</A></TD><TD align="center"><A class="mainLinks" href="https://2008.igem.org/Team:Illinois/Pictures">Pictures</A></TD></TR></TBODY></TABLE></P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2008.igem.org/Team:Illinois/Research_articles">http://2008.igem.org/Team:Illinois/Research_articles</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Illinois/Research_articles" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Illinois/Research_articles" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Illinois/Research_articles&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Illinois/Research_articles&amp;oldid=75240" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2008.igem.org:Privacy_policy" title="2008.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2008.igem.org:General_disclaimer" title="2008.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>