<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Illinois_Antibody_GPCR_Fusion"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:Illinois/Antibody_GPCR_Fusion">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Illinois/Antibody_GPCR_Fusion&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Illinois/Antibody_GPCR_Fusion&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Illinois/Antibody_GPCR_Fusion&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Illinois/Antibody_GPCR_Fusion" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Illinois/Antibody GPCR Fusion</H1><DIV id="bodyContent"><H3 id="siteSub">From 2008.igem.org</H3><P><TABLE class="links"><TBODY><TR><TD align="center"><A class="mainLinks" href="https://2008.igem.org/Team:Illinois">Home</A></TD><TD align="center"><A class="mainLinks" href="https://2008.igem.org/Team:Illinois/Team">Team</A></TD><TD align="center"><A class="mainLinks" href="https://2008.igem.org/Team:Illinois/Project">Project</A></TD><TD align="center"><A class="mainLinks" href="https://2008.igem.org/Team:Illinois/Notebook">Notebook</A></TD><TD align="center"><A class="mainLinks" href="https://2008.igem.org/Team:Illinois/Research_articles">Research Articles</A></TD><TD align="center"><A class="mainLinks" href="https://2008.igem.org/Team:Illinois/Parts">Parts</A></TD><TD align="center"><A class="mainLinks" href="https://2008.igem.org/Team:Illinois/Current_Tasks">Protocols</A></TD><TD align="center"><A class="mainLinks" href="https://2008.igem.org/Team:Illinois/Pictures">Pictures</A></TD></TR></TBODY></TABLE></P><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Core_Team_Members"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Core Team Members</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Project_Abstract"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Project Abstract</SPAN></A></LI><LI class="toclevel-1 tocsection-3"><A href="#Specific_Plans.2C_Supplies.2C_and_Protocols"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Specific Plans, Supplies, and Protocols</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Current_Plan"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">Current Plan</SPAN></A></LI><LI class="toclevel-3 tocsection-5"><A href="#Preventing_Cell_Cycle_arrest_by_mutating_the_FAR1_gene"><SPAN class="tocnumber">3.1.1</SPAN><SPAN class="toctext">Preventing Cell Cycle arrest by mutating the FAR1 gene</SPAN></A></LI><LI class="toclevel-3 tocsection-6"><A href="#Fusion_Protein_Plasmid"><SPAN class="tocnumber">3.1.2</SPAN><SPAN class="toctext">Fusion Protein Plasmid</SPAN></A></LI><LI class="toclevel-3 tocsection-7"><A href="#Integration_of_Fluorescence_Reporter_Controlled_by_FUS1_Promoter"><SPAN class="tocnumber">3.1.3</SPAN><SPAN class="toctext">Integration of Fluorescence Reporter Controlled by FUS1 Promoter</SPAN></A></LI><LI class="toclevel-3 tocsection-8"><A href="#Integrate_or_Plasmid"><SPAN class="tocnumber">3.1.4</SPAN><SPAN class="toctext">Integrate or Plasmid</SPAN></A></LI><LI class="toclevel-3 tocsection-9"><A href="#Directed_Evolution.2FMutagenesis"><SPAN class="tocnumber">3.1.5</SPAN><SPAN class="toctext">Directed Evolution/Mutagenesis</SPAN></A></LI><LI class="toclevel-3 tocsection-10"><A href="#Protocols"><SPAN class="tocnumber">3.1.6</SPAN><SPAN class="toctext">Protocols</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-11"><A href="#Possible_Experiments"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Possible Experiments</SPAN></A></LI><LI class="toclevel-1 tocsection-12"><A href="#Previous_Plan"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Previous Plan</SPAN></A></LI><LI class="toclevel-1 tocsection-13"><A href="#To_Research"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">To Research</SPAN></A></LI><LI class="toclevel-1 tocsection-14"><A href="#G_Proteins_and_their_Receptors"><SPAN class="tocnumber">7</SPAN><SPAN class="toctext">G Proteins and their Receptors</SPAN></A></LI><LI class="toclevel-1 tocsection-15"><A href="#Literature_Research_and_References"><SPAN class="tocnumber">8</SPAN><SPAN class="toctext">Literature Research and References</SPAN></A></LI></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Core_Team_Members">Core Team Members</SPAN></H2><P><A href="/User:Dluedtk2" title="User:Dluedtk2">Dave Luedtke</A>, <A href="/User:Bhadidi" title="User:Bhadidi">Bobak Hadidi</A>, <A href="/User:Kselvad2" title="User:Kselvad2">Kiruthika Selvadurai</A>, <A href="/User:Nbakshi2" title="User:Nbakshi2">Namita Bakshi</A></P><H2><SPAN class="mw-headline" id="Project_Abstract">Project Abstract</SPAN></H2><P>G protein-coupled receptors, or GPCRs, are transmembrane receptors that sense extracellular objects on the scale of small molecules to large proteins. &quot;The importance of GPCR systems is illustrated by the fact that 30% of all clinically prescribed drugs function as GPCR agonists or antagonists&quot; [2]. Activation of the GPCR by ligand binding begins a signal transduction pathway that ultimately results in the transcriptional activation or repression of one or more genes. The signal is transduced with G-proteins, which are signalling proteins that associate with GTP and GDP, as well as kinase cascades. Yeast cells are known to utilize two GPCRs signal transduction pathways, one to detect the presence of glucose and the other to initiate mating. We hope to engineer the well-characterized mating pathway to produce a colorimetric change in the cell upon detecting a novel molecule-- a surface protien of some water-borne pathogen, or possibly a toxin secreted by such a water-borne pathogen.
</P><H2><SPAN class="mw-headline" id="Specific_Plans.2C_Supplies.2C_and_Protocols">Specific Plans, Supplies, and Protocols</SPAN></H2><H3><SPAN class="mw-headline" id="Current_Plan">Current Plan</SPAN></H3><P>We plan to use a steric hindrance mechanism for indirect detection of the presence of the cholera toxin. Rather than modifying the GPCR's target ligand, the addition of the Antibody on the N-terminus of the protein would act as a gate to allow or block binding in the original receptor domain. In this method, the addition of the AB on the end of the GPCR could conceivably:
</P><UL><LI>Block the normal binding site for the alpha factor, unless the AB was bound. After adding a LacZ or GFP gene under the control of the FUS1 promoter, if the yeast was not fluorescing, then we would know toxin was present.
</LI><LI>Vice versa; that is, if the AB was bound by the toxin it would block the binding of the natural ligand to the GPCR so the signal would not be induced. Thus, once production of LacZ or HIS or GFP halts we could know toxin was present.
</LI></UL><P>In either case we would modify the original GPCR gene with spacer dna attached to the antibody gene. After transforming the cells with that plasmid as well as the Fus1-GFP plasmid, we would test the mechanism by adding purified alpha factor to the solution. 
</P><P>Alpha factor has the effect of causing cell cycle arrest on MATa yeast, through the very same pathway. This would obviously have adverse affects on our cell cultures; in order to prevent this, we will have to knockout the Far1 gene.
</P><P>Once we know the original working parts are in order, rounds of directed evolution, selecting for candidates showing the steric mechanism detailed, would be conducted. DE should be more focused on the AB chain, leaving the receptor mostly untouched.
</P><P>It seems the most important variable, outside of the directed evolution, is the length of the spacer dna. It must be long enough so that the two fused proteins can fold without interfering with each other, as well as the correct length so that the AB may cover the ligand binding domain. I think alpha factor typically binds to the 3rd extracellular loop of GPCR. 
</P><P>Steps:
</P><OL><LI>Obtain <A href="http://www.lgcpromochem-atcc.com/common/catalog/numSearch/numResults.cfm?atccNum=4005645" class="external text" rel="nofollow">Yeast</A> lacking Ste2, alpha factor, plasmids, and cholera toxin
</LI><LI>Knockout FAR1 gene
</LI><LI>Add GPCR-Antibody fusion protein plasmid
</LI><LI>Add FUS1-Reporter (GFP or LacZ) plasmid
</LI><LI>Use directed evolution iterations on ABSte2 plasmid
</LI><LI>New yeast strain that detects for cholera toxin through the pheromone pathway
</LI></OL><H4><SPAN class="mw-headline" id="Preventing_Cell_Cycle_arrest_by_mutating_the_FAR1_gene">Preventing Cell Cycle arrest by mutating the FAR1 gene</SPAN></H4><UL><LI>This gene is situated on chromosome X; it is a &quot;Factor ARrest&quot; gene
</LI><LI>FAR1 gene is induced and cell-cycle arrest begins with the secretion of peptide pheromones a and alpha by haploid yeast cells; arrest occurs in G1 phase
</LI><LI>Creating null mutants:
</LI><LI>Two substitutions in FAR1 lead to the far1-D1 mutation--glycine 646 and proline 671 are replaced by aspartic acid and leucine, respectively (Figure 3).  This null mutant can still induce the signal transduction pathway as in the wild-type, but is unable to halt the cell-cycle at the G1 phase when exposed to mating pheromone.
</LI><LI><A href="http://www.bio.davidson.edu/courses/genomics/2001/madden/assignment2.html" class="external autonumber" rel="nofollow">[1]</A>&lt;--- Scroll down to figure 3. The Glycine 646 and proline 671 that have to be mutated are labeled in blue. 
</LI></UL><P>Homologous Recombination Method <A href="http://www.bio.davidson.edu/Courses/genomics/method/homolrecomb.html" class="external autonumber" rel="nofollow">[2]</A>:
</P><UL><LI>Know the DNA sequence of the gene we want to replace
</LI><LI>Construct a DNA sequence with gene of interest (GOI) which has extra DNA on either side of GOI which is identical to  the DNA on the genome (The picture on the website makes it clearer) + attach antibiotic resistance gene
</LI></UL><P>Problem: The mutated FAR1 gene is different from the original FAR1 gene by 2 nucleotides, which could mean that recombination could occur anywhere. We can also replace it with the URA3 gene instead.
</P><P>Protocol for Homologous Recombination <A href="http://www.mrw.interscience.wiley.com/emrw/9780471142720/cp/cpmb/article/mb1310/current/html" class="external autonumber" rel="nofollow">[3]</A>
- We can use Basic Protocol 2 or 3
</P><P>Sequence of FAR1 <A href="http://www.ncbi.nlm.nih.gov/entrez/viewer.fcgi?val=1015578&amp;from=888&amp;to=3380&amp;view=gbwithparts" class="external autonumber" rel="nofollow">[4]</A></P><P>Sequence for URA3 gene <A href="http://www.ncbi.nlm.nih.gov/entrez/viewer.fcgi?val=7276232&amp;from=116167&amp;to=116970&amp;view=gbwithparts" class="external autonumber" rel="nofollow">[5]</A></P><H4><SPAN class="mw-headline" id="Fusion_Protein_Plasmid">Fusion Protein Plasmid</SPAN></H4><UL><LI>Linker DNA must be between AB and Ste2 proteins so they can fold properly
</LI><LI>Ensure fusion protein sent to the plasma membrane
</LI><LI> &quot;Prenylation&quot; adds hydrophobic molecules to proteins, usually with the goal of localizing them to the cell membrane. In Yeast this can be accomplished by addition of the CaaX amino acid sequence 4 aa's from the C terminus of the protein; where C stands for cysteine, a is any aliphatic amino acid, and the identity of X determines which enzyme acts on the protein; <A href="http://www.biology.arizona.edu/biochemistry/problem_sets/aa/Aliphatic.html" class="external text" rel="nofollow">List of Aliphatic AA's</A>.
</LI><LI> The yeast a factor is shown to contain the CaaX motif. <A href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=85805" class="external text" rel="nofollow">Amino Acid selection for Caax in yeast LINK</A></LI></UL><UL><LI> We plan to extract Ste2 gene (~1200 base pairs) from the yeast genome and put it on a plasmid <A href="http://db.yeastgenome.org/cgi-bin/getSeq?map=nmap&amp;seq=S000001868&amp;flankl=&amp;flankr=&amp;rev=" class="external text" rel="nofollow">Coding Sequence</A></LI></UL><P>Outline:
</P><P>RE1: Eco RI        5' G^AATTC
</P><P>RE2: Sac I         5' G^AGCTC
</P><P>RE3: Xba I         5' T^CTAGA
</P><P>RE5: Hind III      5' A^AGCTT
</P><P>
== plasmid ==== RE1 == PGK promoter == RE2 ==
== more restriction sites which include REX/Y (may be RE2/3) for antibody gene ==
== RE3 == Ste2 == RE4 == PGK terminator == RE5 ==== plasmid == 

</P><H4><SPAN class="mw-headline" id="Integration_of_Fluorescence_Reporter_Controlled_by_FUS1_Promoter">Integration of Fluorescence Reporter Controlled by FUS1 Promoter</SPAN></H4><P>Scroll down to the 'Materials and Methods' <A href="http://jb.oxfordjournals.org/cgi/content/full/143/5/667" class="external autonumber" rel="nofollow">[6]</A></P><P>Sequence of FUS1 <A href="http://www.ncbi.nlm.nih.gov/entrez/viewer.fcgi?db=Nucleotide&amp;dopt=GenBank&amp;val=171530" class="external autonumber" rel="nofollow">[7]</A></P><H4><SPAN class="mw-headline" id="Integrate_or_Plasmid">Integrate or Plasmid</SPAN></H4><P>We are currently considering direct integration of the FUS1-Fluo.Prot. fusion into the yeast genome, which seems to be the standard for this type of thing.
</P><P>Basically the target gene would be integrated by flanking it with complementary bases so the entire DNA fragment would undergo homologous recombination with the appropriate section of the yeast chromosome, replacing the original DNA.
</P><P>Our required materials include 4 Biobrick parts and 5 regions of DNA cloned out of the yeast genome (flanking where we would integrate).
</P><P>Biobricks:
</P><UL><LI>EYFP .................... BBa_J63001 ........ 744 bp
</LI><LI>ADH1 Terminator ......... BBa_J63002 ........ 225 bp
</LI><LI>HIS3 Marker ............. BBa_Y1006 ......... 660 bp
</LI><LI>includes its own promoter and terminator
</LI></UL><LI>Biofusion Plasmid ....... BBa_J63010 ........ &gt;4k bp
</LI><LI>includes ampicillin resistance gene
</LI><LI>EcoRI site found in the sequence at base pair 1991.
</LI><LI>XbaI site found in the sequence at base pair 2006.
</LI><LI>SpeI site found in the sequence at base pair 2012.
</LI><LI>PstI site found in the sequence at base pair 2026.
</LI><LI>NotI site found in the sequence at base pair 1997.
</LI><LI>NotI site found in the sequence at base pair 2019.
</LI><P>Yeast DNA:
</P><UL><LI>FUS1 promoter with ~250 amino acids of FUS1 gene
</LI><LI>1kb region downstream of FUS1 (~1000 bps)
</LI><LI>Ste2 gene (1296 bps)
</LI><LI>PGK Promoter (657 bps)
</LI><LI>PGK Terminator (288 bps)
</LI></UL><H4><SPAN class="mw-headline" id="Directed_Evolution.2FMutagenesis">Directed Evolution/Mutagenesis</SPAN></H4><UL><LI>Error Prone PCR in vivo
</LI><LI>Site Directed Mutagenesis
</LI></UL><P><A href="http://www.mrw.interscience.wiley.com/emrw/9780471142720/cp/cpmb/article/mb0805/current/html" class="external text" rel="nofollow">Protocol LINK</A></P><H4><SPAN class="mw-headline" id="Protocols">Protocols</SPAN></H4><UL><LI>Preparation of Yeast Media <A href="http://www.mrw.interscience.wiley.com/emrw/9780471142720/cp/cpmb/article/mb1301/current/pdf" class="external autonumber" rel="nofollow">[8]</A></LI><LI>Basic Protocol 1: Growth in Liquid Media<A href="http://www.mrw.interscience.wiley.com/emrw/9780471142720/cp/cpmb/article/mb1302/current/html" class="external autonumber" rel="nofollow">[9]</A></LI><LI>Rapid Isolation of Yeast Chromosomal DNA <A href="http://www.mrw.interscience.wiley.com/emrw/9780471142720/cp/cpmb/article/mb1311/current/html" class="external autonumber" rel="nofollow">[10]</A></LI></UL><H2><SPAN class="mw-headline" id="Possible_Experiments">Possible Experiments</SPAN></H2><UL><LI>Test the effect of alpha factor on GPCR pheromone-deficient (Ste2) yeast strain
</LI><LI>Compare growth of control plate versus growth on plate+alpha factor
</LI></UL><UL><LI>Create Ste2-plasmid using restriction enzymes
</LI><LI>Add plasma membrane shipping tag sequence (we need to find it for yeast)
</LI></UL><UL><LI>Create plasmid with GFP or LacZ under control of Fus1
</LI><LI>Add to regular yeast strain with MATa and MATalpha type, should see regular fluorescence.
</LI><LI>Add to Ste2 deficient strain, shouldn't see any fluorescence.
</LI><LI>Add to chromosomal Ste2 deficient strain with unmodified Ste2 plasmid, should see regular fluorescence.
</LI></UL><UL><LI>Create Toxin Antibody Plasmid by combining heavy and light chain sequences
</LI><LI>Need some way to test if it folds properly/works, maybe use protein scaffold idea.
</LI></UL><UL><LI>Test transformation of Ste2 deficient strain with Ste2-plasmid 
</LI><LI>Growth should be comparable to strains with Ste2 in genome
</LI></UL><UL><LI>Modify Ste2-plasmid by adding linker dna and complete antibody dna (ABSte2)
</LI><LI>Have to find appropriate length/sequence for linker dna so that receptor folding and antibody folding once translated don't interfere with each other.
</LI></UL><UL><LI>Transform deficient and wild type strains with ABSte2
</LI><LI>Test effects of alpha factor on strains. See if AB interferes with signal transduction.
</LI></UL><H2><SPAN class="mw-headline" id="Previous_Plan">Previous Plan</SPAN></H2><P>In theory, this is how we will progress:
</P><UL><LI>Create a fusion protein that links an antibody against cholera toxin to the Ste2 GPCR of S. cereviviae, the pheremone response GPCR.
</LI><LI>1)Find the sequences of the GPCR <A href="http://www.bio.davidson.edu/Courses/genomics/2002/Statler/STE2.htm" class="external autonumber" rel="nofollow">[11]</A> and the antibody. Did someone already find the sequence of the antibody? <A href="http://www.hytest.fi/product/cholera-toxin-antibody.html" class="external autonumber" rel="nofollow">[12]</A>&lt;--- We can buy the antibody to the beta subunit here. 
</LI><LI>2)Select site of fusion
</LI><LI> Cys187 in EL2 and Cys110 in TM3: highly conserved cysteines found in most GPCRs
</LI><LI> &quot;extracellular loop 2 has extensive contacts with the other extracellular domains and also plunges down into the transmembrane bundle, contacting the retinal ligand.&quot; If we add the antibody here, will it interfere with the proper folding of the surface protein? <A href="https://netfiles.uiuc.edu/dluedtk2/shared/iGEM/day2/structure%20of%20natural%20yeast%20GPCR.pdf" class="external autonumber" rel="nofollow">[13]</A></LI><LI> &quot;the first extracellular loop is important in ligand-mediated activation of the receptor, whereas the second loop functions as a negative regulator of receptor activation and serves to stabilize the inactive state of the receptor&quot; <A href="https://netfiles.uiuc.edu/dluedtk2/shared/iGEM/day2/structure%20of%20natural%20yeast%20GPCR.pdf" class="external autonumber" rel="nofollow">[14]</A></LI><LI> Possible sites for fusion of antibody: Tyr101 through Gln135 in EL1 domain (because they did not have major effects on binding and signal transduction when they were mutated to Cysteine) 
</LI><LI>Also, we want to add the antibody at a point where the unbound state (of the GPCR) is in contact with the solvent: residues 101 and 106
</LI><LI> &quot;mutation of residues Leu102, Asn105, Ser108, Tyr111, and Thr114 to cysteine resulted in receptors that could effectively bind pheromone, but they were partially (Leu102 and Gln135) or severely (Asn105, Ser108, and Tyr111) compromised with respect to signal transduction.&quot; <A href="https://netfiles.uiuc.edu/dluedtk2/shared/iGEM/day2/structure%20of%20natural%20yeast%20GPCR.pdf" class="external autonumber" rel="nofollow">[15]</A></LI></UL><LI>3)Have the gene sequenced--This will be a lengthy gene.  We might be better off somehow obtaining a gene containing the natural GPCR (perhaps from a research group) and inserting the gene of the antibody.
</LI><LI> Professor Chris Rao explained to us an online tool (it was some company) for determining what restriction enzymes and where in the gene they could act to aid us in this.
</LI><LI> One yeast Ste2 gene was sequenced to be ~1.5kb, including restriction sites for EcoRI and BamHI [9].
</LI><LI>Express the antibody/GPCR fusion protein in yeast that lack the wild type receptor.
</LI><LI> Found <A href="http://www.lgcpromochem-atcc.com/common/catalog/yeastGeneticStock/yeastGeneticStockIndex.cfm" class="external text" rel="nofollow">6 strains</A> (type &quot;ygk ste3|ste2&quot; without quotes into the text search) of yeast of interest from the Yeast Genetic Stock Center, 2 of mating type a, 2 of mating type alpha, and 2 that are diploid (both). Three of each type have their Ste2 gene knocked out; the other 3 likewise have their Ste3 gene knocked out. Each costs about $150, in addition the media for their growth conditions is listed, WOW!
</LI><LI>MATa expresses Ste2 (the alpha-factor receptor) while MATalpha expresses Ste3 (the a-factor receptor) [1]. We should thus either order MATa without Ste2 or MATalpha without Ste3.
</LI><LI>Which mating type would better suite our purposes?
</LI><LI>The gene will most likely be on a plasmid. We may have to integrate the gene into the yeast's chromosome, however.
</LI><LI> The size of the gene may be substantial, if it turns out to be too large for a plasmid, integration into the yeast chromosome may be the way to go (hopefully not).
</LI><LI> [9] cites that a form of the Ste2 gene has been carried by a vector into yeast, size should not be a problem unless the antibody module is large.
</LI><LI>Measure the activation of the GPCR by the toxin (and by the natural pheremone) using a reporter gene.
</LI><LI>Several studies have used the FUS1 promoter in conjunction with HIS3 selection.  The FUS1 promoter seems to be a good choice for our purposes.
</LI><LI>Here's a recombinant plasmid we will need to contruct:
</LI><P><A href="http://www.freepatentsonline.com/5063154.html" class="external autonumber" rel="nofollow">[16]</A>
a. the FUSI promoter isolated from Saccharomyces cerevisiae;
</P><P>b. DNA encoding approximately the first 254 amino acids of said FUSI gene;
FUS1 gene in Yeast (FASTA format):
&gt;FUS1_YEAST
MVATIMQTTTTVLTTVAAMSTTLASNYISSQASSSTSVTTVTTIATSIRSTPSNLLFSNVAAQPKSSSAS
TIGLSIGLPIGIFCFGLLILLCYFYLKRNSVSISNPPMSATIPREEEYCRRTNWFSRLFWQSKCEDQNSY
SNRDIEKYNDTQWTSGDNMSSKIQYKISKPIIPQHILTPKKTVKNPYAWSGKNISLDPKVNEMEEEKVVD
AFLYTKPPNIVHIESSMPSYNDLPSQKTVSSKKTALKTSEKWSYESPLSRWFLRGSTYFKDYGLSKTSLK
TPTGAPQLKQMKMLSRISKGYFNESDIMPDERSPILEYNNTPLDANDSVNNLGNTTPDSQITSYRNNNID
LITARPHSVIYGTTAQQTLETNFNDHHDCNKSTEKHELIIPTPSKPLKKRKKRRQSKMYQHLQHLSRSKP
LPLTPNSKYNGEASVQLGKTYTVIQDYEPRLTDEIRISLGEKVKILATHTDGWCLVEKCNTQKGSIHVSV
DDKRYLNEDRGIVPGDCLQEYD
c. DNA encoding a protein or a polypeptide of interest (Could be His3), the DNA positioned so as to be under the control of said FUSI promoter;
</P><P>d. a two micron autonomously replicating sequence;
</P><UL><LI>The usual response of haploid yeast cells to the mating pheromone leads to the activation of Pheromone Responsive genes and also cell cycle arrest. For the phenotype to be removed and the cells to continue growing when the pheromone response pathway is activated, the FAR1 gene should  be deleted. 
</LI></UL><LI>Use site specific directed evolution to increase the effectiveness of the new GPCR.
</LI><LI><A href="http://www.escience.ws/b572/L4/L4.htm" class="external autonumber" rel="nofollow">[17]</A>Methods of site-directed mutagenesis, including non-PCR techniques
</LI><LI><A href="http://web.archive.org/web/20010222093603/http://www.labmarket.co.kr/methodpcr5.htm" class="external autonumber" rel="nofollow">[18]</A>Protocol for site-directed mutagenesis using PCR
</LI><LI>Altered-States Method<A href="http://www.escience.ws/b572/L4/L4.htm" class="external autonumber" rel="nofollow">[19]</A>:
</LI><LI>   1.  Start with a plasmid carrying a defective selectable marker (e.g. Amp)
</LI><LI>2. Link the mutation you are making elsewhere in the plasmid to a correction of the defective selectable marker.
</LI><LI>3. Select for correction of the selectable marker, and you are likely to also find plasmids with your specific mutation introduced as well 
</LI><LI>QuikChange Method<A href="http://www.bio.davidson.edu/courses/Molbio/MolStudents/spring99/sarah/method.html" class="external autonumber" rel="nofollow">[20]</A></LI><LI>Gene of interest in plasmid, but it is modified (bp's added, removed, substituted), DNA is denatures and primers anneal to both strands, there are &quot;staggered nicks&quot; where mutation is, Dpn I added (digests methylated DNA), parent DNA is methylated but new mutated DNA is not, result is that only new mutant DNA (see below, brown region w/ green x's) is transformed in bacteria cell 
</LI><P>
Alternatively, we might begin looking into completely replacing the yeast receptor with a mammalian or novel receptor with an affinity for our target protein. If we go down this route, we will also have to replace the G-alpha subunit with a chimeric one that has affinity for both the new receptor and the yeast G-betagamma complex. These have already been synthesized/cloned/will be fairly easy to procure, and may or may nor be receptor specific. This method is the major procedure for researching GPCRs today, typically used in pharmaceutical research -- knock out the pheromone GPCR while leaving the endogenous signal pathway/reporter intact, then introduce the uncharacterized GPCR of interest (with its G-aplha). In our case, we would not be studying an unknown GPCR to find what its target ligand or purpose is; instead we would simply already have one that we had/would evolve using mutagenesis to have an affinity for the toxin.
</P><H2><SPAN class="mw-headline" id="To_Research">To Research</SPAN></H2><UL><LI>Effect of constitutive expression of alpha factor on MATa yeast cells. While they could still divide by budding, the signal pathway causes cell growth arrest in G1; is this a problem??
</LI><LI> How to knockout the gene responsible for growth arrest when alpha factor is present.
</LI></UL><UL><LI>Cell Wall Issue
</LI><LI>&quot;The potency of some ligands might be reduced in yeast compared with mammals [for receptors taken from mammalian cells]. Using Gpa1âGa chimeras probably explains part of this discrepancy [original yeast G-alpha subunits have low affinity for the new receptors and the mammalian G-alpha has low affinity for yeast G-betagamma, we will bypass this problem altogether by fusing the antibody to the original yeast receptor], but another factor might be the ability of ligands to penetrate the yeast cell wall. Potencies for small ligands are similar in yeast and mammals, those for intermediate ligands are about two orders of magnitude less in yeast, and large ligands such as chemokines (polypeptides with 70â80 residues) are often unable to activate receptors when applied exogenously to yeast. The fact that smaller versions of these ligands can be more efficient agonists suggests that accessibility is an issue.&quot;[1]
</LI></UL><UL><LI>Antibody Sequence (we have)
</LI><LI>Receptor Sequence (we have)
</LI><LI>Alpha Factor Sequence - 13 AAs (WHWLQLKPGQPMY)
</LI><LI>Sources of yeast strains deficient in specific genes (we have)
</LI><LI>Cholera Toxin or possible target protein
</LI></UL><H2><SPAN class="mw-headline" id="G_Proteins_and_their_Receptors">G Proteins and their Receptors</SPAN></H2><DIV class="thumb tleft"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">G protein Heterotrimer: Alpha chain is yellow, Beta-Gamma complex is Blue, GDP is black, membrane is shown in grey.</DIV></DIV></DIV><P>G proteins are comprised of 3 subunits, collectively known as a heterotrimeric or large G protein. The individual pieces are GÎ±, GÎ² and GÎ³. When the appropriate molecule is bound to the receptor, the GDP bound to the GÎ± unit  becomes GTP. More conformational changes are induced causing GÎ²Î³ and GÎ± to dissociate. Depending on the signal cascade, either complex, GÎ±-GTP or GÎ²Î³ may now induce the appropriate linked signal pathway.
</P><P>As opposed to other signal receptors, the ligands that bind GPCRs typically bind within the transmembrane domain.
When the receptor is bound by its ligand, it shifts shape, activating the G protein (by allowing the exchang of GDP for GTP). Also, G protein complexes may or may not be coupled with the receptors, instead, if they happen across a receptor with its activating ligand bound, the G protein will be activated (GDP -&gt; GTP) and induce its signal cascade. (wikipedia).
</P><H2><SPAN class="mw-headline" id="Literature_Research_and_References">Literature Research and References</SPAN></H2><P>[1]<A href="https://netfiles.uiuc.edu/dluedtk2/shared/iGEM/day2/Functional%20analysis%20of%20heterologous%20GPCR%20signaling%20pathways%20in%20yeast.pdf" class="external text" rel="nofollow">Functional analysis of heterologous GPCR signaling pathways in yeast</A></P><P>[2]<A href="http://www.nature.com/embor/journal/v2/n7/full/embor385.html" class="external free" rel="nofollow">http://www.nature.com/embor/journal/v2/n7/full/embor385.html</A></P><P>Ste2 information, includes sequence:<A href="http://www.bio.davidson.edu/Courses/genomics/2002/Statler/STE2.htm" class="external free" rel="nofollow">http://www.bio.davidson.edu/Courses/genomics/2002/Statler/STE2.htm</A></P><P>[9] <A href="http://www.pubmedcentral.nih.gov/picrender.fcgi?artid=280318&amp;blobtype=pdf" class="external text" rel="nofollow">Ste2 Protein</A></P><P>
2 GPCR pathways in yeast: </P><P><TABLE class="links"><TBODY><TR><TD align="center"><A class="mainLinks" href="https://2008.igem.org/Team:Illinois">Home</A></TD><TD align="center"><A class="mainLinks" href="https://2008.igem.org/Team:Illinois/Team">Team</A></TD><TD align="center"><A class="mainLinks" href="https://2008.igem.org/Team:Illinois/Project">Project</A></TD><TD align="center"><A class="mainLinks" href="https://2008.igem.org/Team:Illinois/Notebook">Notebook</A></TD><TD align="center"><A class="mainLinks" href="https://2008.igem.org/Team:Illinois/Research_articles">Research Articles</A></TD><TD align="center"><A class="mainLinks" href="https://2008.igem.org/Team:Illinois/Parts">Parts</A></TD><TD align="center"><A class="mainLinks" href="https://2008.igem.org/Team:Illinois/Current_Tasks">Protocols</A></TD><TD align="center"><A class="mainLinks" href="https://2008.igem.org/Team:Illinois/Pictures">Pictures</A></TD></TR></TBODY></TABLE></P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2008.igem.org/Team:Illinois/Antibody_GPCR_Fusion">http://2008.igem.org/Team:Illinois/Antibody_GPCR_Fusion</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Illinois/Antibody_GPCR_Fusion" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Illinois/Antibody_GPCR_Fusion" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Illinois/Antibody_GPCR_Fusion&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Illinois/Antibody_GPCR_Fusion&amp;oldid=61717" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2008.igem.org:Privacy_policy" title="2008.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2008.igem.org:General_disclaimer" title="2008.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>