<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_CU-Boulder_AzoPhe skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:CU-Boulder/AzoPhe</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><UL class="ulStuff"><LI class="liStuff"><A class="aStuffHead" href="https://2017.igem.org/Team:CU-Boulder"><SPAN style="color: red"><B>μ</B></SPAN> s t o r a g e</A></LI><LI class="dropdown"><A class="aStuff" href="javascript:void(0)" class="dropbtn">Project</A><DIV class="dropdown-content"><A class="dropstuff" href="https://2017.igem.org/Team:CU-Boulder/Description">Description</A><A class="dropstuff" href="https://2017.igem.org/Team:CU-Boulder/Demonstrate">Results</A><A class="dropstuff" href="https://2017.igem.org/Team:CU-Boulder/Model">Modelling</A><A class="dropstuff" href="https://2017.igem.org/Team:CU-Boulder/Notebook">Notebook</A></DIV></LI><LI class="dropdown"><A class="aStuff" href="javascript:void(0)" class="dropbtn">Team</A><DIV class="dropdown-content"><A class="dropstuff" href="https://2017.igem.org/Team:CU-Boulder/Team">Team</A><A class="dropstuff" href="https://2017.igem.org/Team:CU-Boulder/Attributions">Attributions</A></DIV></LI><LI class="dropdown"><A class="aStuff" href="javascript:void(0)" class="dropbtn">Parts</A><DIV class="dropdown-content"><A class="dropstuff" href="https://2017.igem.org/Team:CU-Boulder/Basic_Part">Basic Parts</A></DIV></LI><LI class="liStuff"><A class="aStuff" href="https://2017.igem.org/Team:CU-Boulder/Safety">Safety</A></LI><LI class="dropdown"><A class="aStuff" href="javascript:void(0)" class="dropbtn">Community</A><DIV class="dropdown-content"><A class="dropstuff" href="https://2017.igem.org/Team:CU-Boulder/HP/Silver">Human Practices</A><A class="dropstuff" href="https://2017.igem.org/Team:CU-Boulder/Collaborations">Collaborations</A></DIV></LI><LI class="liStuff"><A class="aStuff" href="https://2017.igem.org/Team:CU-Boulder/InterLab">Interlab</A></LI></UL></P><DIV class="mainhead"><H><I>• Mutation •</I></H></DIV><SECTIONTWO><DIV class="pageBuffer"><DIV class="textboxTop"><P> After proving our systems for visualization and isolation, we now had to determine which (if any) mutants could be used to form a successful compartment incorporating AzoPhe residues, validate it via visualization, and then move on to isolate it for use in proof-of-concept assays. </P></DIV></DIV><DIV class="pageBuffer"><H>Compartment Mutation</H><DIV class="textboxTop"><P> Last year’s CU Boulder iGEM team was able to demonstrate some hopeful preliminary data for several attempted mutants. This year however, all of our successful transfections did not show fluorescence. 

This is almost certainly the fault of the LVA tag on Fusion Red. If we trust last year’s team’s data and still believe that some mutants can still form compartments using AzoPhe, there are several reasons that this could be the case. For one, non-canonical amino acid incorporation systems are known not to be particularly robust: for every EutS subunit formed, there are likely several that truncate at the Amber codon where we hope to incorporate the ncAA. Because the Fusion Red tagged with LVA depends on the compartment to protect it from protease degradation, any irregularities in the compartment or a slow rate of compartment expression could allow the protease to find and degrade all our fluorescent markers. 

An additional problem we discovered was the difficulty of transfection with a three plasmid system. Even with very highly competent cells, most transfections with three plasmids fail or fail to grow in liquid media once they are cultured from the plate. Last year’s team seemed to have had much better luck with this system than we had, and a likely 90% of our attempted transfections failed, if not more.

Still yet another problem we encountered was the stunted growth of the transfected E.Coli in media. The strain we used still has the Amber stop codon present in its genome, and if too much AzoPhe makes its way into the normal proteins of the cell, it could easily present a large problem for the growth and division of those cells. Worse still, we want this AzoPhe to be available for the compartments!

We intend to resolve these problems by first moving the entire AzoPhe incorporation system onto the same plasmid as our Fusion Red marker. This takes time, as the plasmid that expresses that system is not BioBrick and has many separate elements that allow it to function, but it would be worthwhile to make transformation more reliable. Once done, we require an engineered E.Coli strain without Amber stop codons in its genome, which would allow the cells to grow and reserve the AzoPhe for compartment formation. With these changes, we believe that our system may well still prove viable.
 </P></DIV></DIV></SECTIONTWO></DIV></DIV></DIV></DIV></DIV></BODY></HTML>