Team:NEU China B/Demonstrate
Ruby - Responsive Corporate Tempalte
go to top
Project
Background
Description
Applied Design
Demonstration & Result
Models
Lab
InterLab
Protocols
Notebook
Safety
Parts
Basic Parts
Composite Parts
Improve
Human practice
Human Practices
Collaborations
Team
Team Members
Attributions
Demonstration
Overview
In this project, in order to detect and monitor the lactic acid concentration during milk fermentation, we constructed two plasmids with GFP reporter gene in engineered bacteria to respond exogenous lactic acid.
To date, the lactic acid detected plasmids have been demonstrated with two critical achievements:
1. Under the induction of lactic acid, the reporter gene, GFP, with significantly upregulated expression level than the negative control group;
2. We successfully designed an optical fiber based model to detect and monitor the lactic acid induced GFP signals during the fermentation.
Principle
Nature offers a potential solution in the form of bacterial genetic operons, which are designed to sense the concentration of important metabolites in the environment and activate gene expression in response. The sensitivity of such systems is very high—often compounds are detected at micromolar or even nanomolar concentrations and a wealth of such systems that can detect important metabolites for mammalian cell culture such as sugars, amino acids, and metabolic waste products have been identified [1].
Schematic of the LldPRD operon and biochemical mechanism (Figure 1): In the absence of lactate, dimers of LldR bind to the operator sites in the lldPRD promoter and form a tetramer, sequestering the DNA and preventing transcription of the operon. Bottom: Lactate enters the cell via the glycolate permease (GlcA) or LldP and interacts with the LldR regulator protein. The LldR dimer bound to O2 dissociates when bound to lactate, but the dimer bound to O1 becomes a transcriptional activator that promotes transcription of the operon when lactate binds[1].
In quorum sensing (QS) process, bacteria regulate gene expression by utilizing small signaling molecules called autoinducers in response to a variety of environmental cues. Autoinducer 2 (AI-2), a QS signaling molecule proposed to be involved in interspecies communication, is produced by many species of gram-negative and gram-positive bacteria. In Escherichia coli and Salmonella typhimurium, the extracellular AI-2 is imported into the cell by a transporter encoded by the Lsr operon. In every case, AI-2 is synthesized by LuxS, which functions in the pathway for metabolism of S-adenosylmethionine (SAM), a major cellular methyl donor. In a metabolic pathway known as the activated methyl cycle, SAM is metabolized to Sadenosylhomocysteine, which is subsequently converted to adenine, homocysteine, and 4,5-dihydroxy-2,3-pentanedione (DPD, the precursor of AI-2) by the sequential action of the enzymes Pfs and LuxS. DPD is a highly reactive product that can rearrange and undergo additional reactions, suggesting that distinct but related molecules derived from DPD may be the signals that different bacterial species recognize as AI-2. The regulatory network for AI-2 uptake is comprised of two other important components, lsrR and lsrK, both of which are located adjacent, but divergently transcribed from the lsr operon (Figure 2). LsrR is the repressor of the lsr operon and itself. LsrK is a kinase responsible for converting AI-2 to phospho-AI-2, which is required for relieving LsrR repression [2].
Figure 1. (a) Organization of the lldPRD operon. O1 and O2 represent the operator sites in the lldPRDp promoter. The three genes in the operon are (from left to right) LldP: lactate permease to allow lactate transport, LldR: regulatory protein, LldD: Lactate dehydrogenase for lactate utilization. (b) Diagram of the mechanism of lactate-dependent induction of lldPRD operon in E.coli cells.
Figure 2. (Left) Model for regulation, transportation, and modification of AI-2 by the Lsr proteins in E. coli. AI-2 is synthesized by LuxS and accumulates extracellularly. The AI-2 uptake repressor LsrR represses the lsr operon (comprised of lsrACDBFG) and the lsrRK. Basal expression of the LsrACDB transporter allows some AI-2 to enter the cytoplasm, where it is phosphorylated by LsrK. Phospho-AI-2 has been reported to bind to LsrR and relieve its repression effect on the lsr transporter genes, thus stimulating additional AI-2 uptake.      Figure 3. (Right) Illustration of project principle.
Therefore, we hope to combine QS system and lldPRD operon for constructing two-expression plasmids (Figure 3).
Engineered Bacteria Composition
According to the modeling results, we finally decided to choose following engineered E.coli as our biosense detector.
lldPRD operon promoter-Luxs-Lldr × LsrA promoter-GFP
Outcomes
1. After constructing two plasmids pCDFDuet-1 and pET-28a(+) contained lldPRD operon promoter-Luxs-Lldr and LsrA promoter-GFP, respectively, we transformed them into one K12 competent cell (Figure 4). For transformants selection, pCDFDuet-1 and pET-28a(+) contained streptomycin and kanamycin, respectively. As described previously, the lldPRD operon should be activated under the lactate induction. Thus, we used Western Blot for detecting the expression of LuxS and lldR proteins from lldPRD operon (Figure 5).
Figure 4. K12 E.coli can grow on two-resistant-media containing both Kanamycin and streptomycin.
Figure 5. Western Blot result. (a) Characterization of LuxS under different concentrations of lactic acid and IPTG. The LuxS protein had been an obvious protein band between the last two marker bands of 15 kDa and 25 kDa. The molecular weight of the LuxS protein is about 17 kDa. Lane 1: IPTG 0 mM, lactic acid 0 mM; Lane 2: IPTG 0.5 mM, lactic acid 0 mM; Lane 3: IPTG 1 mM, lactic acid 0 mM; Lane 4: IPTG 0 mM, lactic acid 2 mM; Lane 5: IPTG 0.5 mM, lactic acid 2 mM; Lane 6: IPTG 1 mM, lactic acid 2 mM; Lane 7: control( IPTG 0 mM, lactic acid 0 mM); Lane 8: IPTG 0 mM, lactic acid 0 mM; Lane 9: IPTG 0.5 mM, lactic acid 0 mM; Lane 10: IPTG 1 mM, lactic acid 0 mM; Lane 11: IPTG 0 mM, lactic acid 2mM; Lane 12: IPTG 0.5 mM, lactic acid 2 mM; Lane 13: IPTG 1 mM, lactic acid 2 mM. (b) Characterization of Lldr under different concentrations of lactic acid and IPTG. The lldR protein had been an obvious protein band between two marker bands of 25 kDa and 35 kDa. The molecular weight of the lldR protein is about 29 kDa. Lane 1: IPTG 0mM, lactic acid 0mM; Lane 2: IPTG 0.5mM, lactic acid 0mM; Lane 3, IPTG 1mM, lactic acid 0mM; Lane 4: IPTG 0mM, lactic acid 2mM; Lane 5: IPTG 0.5mM, lactic acid 2mM; Lane 6, IPTG 1mM, lactic acid 2mM.     Note: we have used two kind of plasmids: (1) pCDFDuet-1, its resistance is streptomycin; (2) pET-28b(+), its resistance is Kanamycin.
2. Then, we used optical fibers to detect the green fluorescent signal from GFP expression and the fluorescence intensity was quantified spectrophotometer (Figure 7.).
Figure 6. Equipment of optical fibers.
Figure 7. Fluorescence Intensity results from spectrophotometer.      Note: EG refers to Experimental Group; CG refers to Control Group; [Lactate] refers to applied lactate concentration, mM.
3. From above data, we calculated their net value as well as got fitting figures and functions (Figure 8). Along with series lactate concentration variety, all net value is above 0. This result indicated  that this engineered K12 E.coli can emit distinct GFP fluorescence intensities under different lactate concentrations. Except for the initiation of the reaction, the rest of the functional curves showed similar trends, especially when the reaction time was 5 min and 10 min, and the peak value was reached before the concentration of lactate concentration is 1 mM. From Figure 8, we could conclude  the engineered bacteria was best used to detect the lactic acid concentration in yogurt at the reaction time of 5 min and 10 min due to the relative high sensitivity. In order to reduce the reaction time of engineered bacteria, the reaction time of 5 min was selected.
Figure 8. Fitting Results.
4. The diagram indicated that the system for monitoring and quantifying lactic acid in fermentation via using our engineered GFP bacteria (Figure 9).
Figure 9. Design of lactate biosensor device. The container is made by 3D printing (details).
Reference
1. Goers, L., et al., Whole-cell Escherichia coli lactate biosensor for monitoring mammalian cell cultures during biopharmaceutical production. Biotechnol Bioeng, 2017. 114(6): p. 1290-1300.
2. Xue, T., et al., LsrR-binding site recognition and regulatory characteristics in Escherichia coli AI-2 quorum sensing. Cell Res, 2009. 19(11): p. 1258-68.
