<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_NEU_China_B_demostration skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:NEU China B/demostration</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Ruby - Responsive Corporate Tempalte</TITLE></P><DIV style="position:fixed;bottom:50px;right:50px;"><A class="btn btn-default" href="#header" role="button">go to top</A></DIV><HEADER id="header" class="header"><DIV class="header-inner"><DIV class="container"><DIV class="row"><DIV class="col-md-9 col-sm-10"><NAV class="navbar navbar-default"><DIV class="collapse navbar-collapse"><UL id="nav" class="nav navbar-nav"><LI><A href="#">Project</A></LI><LI><A href="https://2018.igem.org/Team:NEU_China_B/background">Background</A></LI><LI><A href="https://2018.igem.org/Team:NEU_China_B/Description">
													Description
												</A></LI><LI><A href="https://2018.igem.org/Team:NEU_China_B/Design">
													Applied Design
												</A></LI><LI><A href="https://2018.igem.org/Team:NEU_China_B/Demonstrate">
													Demonstration &amp; Result
												</A></LI><LI><A href="https://2018.igem.org/Team:NEU_China_B/Model">
													Models
												</A></LI></UL><LI><A href="#">Lab</A></LI><LI><A href="https://2018.igem.org/Team:NEU_China_B/InterLab">
														InterLab
													</A></LI><LI><A href="https://2018.igem.org/Team:NEU_China_B/protocols">Protocols</A></LI><LI><A href="https://2018.igem.org/Team:NEU_China_B/Notebook">
														Notebook
													</A></LI><LI><A href="https://2018.igem.org/Team:NEU_China_B/Safety">Safety</A></LI><LI><A href="#">Parts</A></LI><LI><A href="https://2018.igem.org/Team:NEU_China_B/Basic_Part">
														Basic Parts
													</A></LI><LI><A href="https://2018.igem.org/Team:NEU_China_B/Composite_Part">
														Composite Parts
													</A></LI><LI><A href="https://2018.igem.org/Team:NEU_China_B/Improve">
														Improve
													</A></LI><LI><A href="#">Human practice</A></LI><LI><A href="https://2018.igem.org/Team:NEU_China_B/Human_Practices">
														Human Practices
													</A></LI><LI><A href="https://2018.igem.org/Team:NEU_China_B/Collaborations">
														Collaborations
													</A></LI><LI><A href="#">Team</A></LI><LI><A href="https://2018.igem.org/Team:NEU_China_B/Team">
														Team Members
													</A></LI><LI><A href="https://2018.igem.org/Team:NEU_China_B/Attributions">
														Attributions
													</A></LI></DIV></NAV></DIV></DIV></DIV></DIV></HEADER><H1>
				Demonstration
			</H1><H2>
				Overview
			</H2><P>
				In the initial phase of the project, we planned to complete the whole project with the coliform effect and the lactic acid operator. We plan to construct two plasmids to complete the whole project and realize the whole process from detect lactic acid to express Green Fluorescence Protein. After a long effort, we have completed the entire process under formal conditions. The main achievements are stated as follows:
			</P><P>1.	With the induction of lactic acid, our GFP could emit green fluorescence, and the fluorescence value was significantly higher than that of the control group.
					2.	Isolated proteins also have good western bloting results.
					3.	We have constructed a device to detect lactic acid by using optical fiber, which is convenient and simple and fast fluorescence value of GFP, so as to determine the concentration of lactic acid.
			</P><H2>
				Principle
			</H2><P>
				Nature offers a potential solution in the form of bacterial genetic operons, which are designed to sense the concentration of important metabolites in the environment and activate gene expression in response. The sensitivity of such systems is very high—often compounds are detected at micromolar or even nanomolar concentrations and a wealth of such systems that can detect important metabolites for mammalian cell culture such as sugars, amino acids, and metabolic waste products have been identified [1]. 
			</P><P>
				Schematic of the LldPRD operon and biochemical mechanism (Figure 1): In the absence of lactate, dimers of LldR bind to the operator sites in the lldPRD promoter and form a tetramer, sequestering the DNA and preventing transcription of the operon. Bottom: Lactate enters the cell via the glycolate permease (GlcA) or LldP and interacts with the LldR regulator protein. The LldR dimer bound to O2 dissociates when bound to lactate, but the dimer bound to O1 becomes a transcriptional activator that promotes transcription of the operon when lactate binds [1].
			</P><P>
				In quorum sensing (QS) process, bacteria regulate gene expression by utilizing small signaling molecules called autoinducers in response to a variety of environmental cues. Autoinducer 2 (AI-2), a QS signaling molecule proposed to be involved in interspecies communication, is produced by many species of gram-negative and gram-positive bacteria. In Escherichia coli and Salmonella typhimurium, the extracellular AI-2 is imported into the cell by a transporter encoded by the Lsr operon. In every case, AI-2 is synthesized by LuxS, which functions in the pathway for metabolism of S-adenosylmethionine (SAM), a major cellular methyl donor. In a metabolic pathway known as the activated methyl cycle, SAM is metabolized to Sadenosylhomocysteine, which is subsequently converted to adenine, homocysteine, and 4,5-dihydroxy-2,3-pentanedione (DPD, the precursor of AI-2) by the sequential action of the enzymes Pfs and LuxS. DPD is a highly reactive product that can rearrange and undergo additional reactions, suggesting that distinct but related molecules derived from DPD may be the signals that different bacterial species recognize as AI-2. The regulatory network for AI-2 uptake is comprised of two other important components, lsrR and lsrK, both of which are located adjacent, but divergently transcribed from the lsr operon (Figure 2). LsrR is the repressor of the lsr operon and itself. LsrK is a kinase responsible for converting AI-2 to phospho-AI-2, which is required for relieving LsrR repression [2].
			</P><P>
				Figure 1. (a) Organization of the lldPRD operon. O1 and O2 represent the operator sites in the lldPRDp promoter. The three genes in the operon are (from left to right) LldP: lactate permease to allow lactate transport, LldR: regulatory protein, LldD: Lactate dehydrogenase for lactate utilization. (b) Diagram of the mechanism of lactate-dependent induction of lldPRD operon in E.coli cells. 
			</P><P>
				Figure 2. (Left) Model for regulation, transportation, and modification of AI-2 by the Lsr proteins in E. coli. AI-2 is synthesized by LuxS and accumulates extracellularly. The AI-2 uptake repressor LsrR represses the lsr operon (comprised of lsrACDBFG) and the lsrRK. Basal expression of the LsrACDB transporter allows some AI-2 to enter the cytoplasm, where it is phosphorylated by LsrK. Phospho-AI-2 has been reported to bind to LsrR and relieve its repression effect on the lsr transporter genes, thus stimulating additional AI-2 uptake.
				Figure 3. (Right) Illustration of project principle.
			</P><P>
				Therefore, we hope to combine QS system and lldPRD operon for constructing two-expression plasmids (Figure 3).
			</P><P>
				Engineered Bacteria Composition 
			</P><P>
				According to the modeling results, we finally decided to choose following engineered E.coli as out biosensor detector. 
			</P><P>
				lldPRD operon promoter-Luxs-Lldr × LsrA promoter-GFP
			</P>
			x
			<H2>
				Outcomes 
			</H2><P>
				1.	After constructing two plasmids containing lldPRD operon promoter-Luxs-Lldr and LsrA promoter-GFP, respectively, we transformed them into one K12 (Figure 4). Then we using Western Blotting method for expressing LuxS and lldR separately (Figure 5). 
			
			</P><P>
				Figure 4. K12 E.coli can grow on two-resistant-media containing both Kanamycin and streptomycin. 
			</P>
			
			(a)	                                      (b)
			Luxs                                            lldr
			
			
			
			
			
			
			<P>
				Figure 5. Western Blotting results
			</P><P>
				Note: we have used two kind of plasmids: (1) pCDFDuet-1, its resistance is streptomycin; (2) pET-28b(+), its resistance is Kanamycin.
			</P><P>
				2.	We took advantage of optical fibers (Figure 6) for detecting fluorescence intensity along with a series lactate concentration variety as well as time, resulting in the following data (Figure 7).  
			</P><P>
				Figure 6. Equipment of optical fibers.
			</P><P>
				Figure 7. Fluorescence Intensity results.
			</P><P>
				Note: EG refers to Experimental Group; CG refers to Control Group; [Lactate] refers to applied lactate concentration, mM.
			</P><P>
				3.	By using above data, we calculated their net value as well as got fitting figures and functions (Figure 8). Along with series lactate concentration variety, all net value is over 0, meaning that this engineered E.coli can detect the lactate concentration as well as change fluorescence intensity. Except for the beginning of the reaction, the rest of the functional curves showed similar trends, especially when the reaction time was 5 min and 10 min, and the peak value was reached before the concentration of lactate concentration is 1 mM. It proved that the engineering bacteria was best used to detect the lactic acid concentration in yogurt at the reaction time of 5 min and 10 min with a high sensitivity. In order to reduce the reaction time of engineered bacteria, the reaction time of 5 min was selected. Based that total time of reaction time of optical fibers is 200 ms, therefore our reaction time can be defined as 5 min. 
				</P><P>
				Figure 8. Fitting results and functions. 
			</P><P>
				4.	Since we have little time for building complete device, we only drawled the design of following (Figure 9.).  
				Figure 9. Design of lactate biosensor device. The container is made by 3D printing (details). 
			</P><I>
					Reference
					
					[1]: Lisa, Goers, Catherine, Ainsworth, Cher, Hui, Goey, Cleo, Kontoravdi, Paul, S, Freemont, Karen, M, Polizzi. Whole-Cell Escherichia coli Lactate Biosensor for Monitoring Mammalian Cell Cultures During Biopharmaceutical Production[J]. Biotechnology and Bioengineering, 2017, 114(6): 1290-1300.
					
					[2]: Ting Xue, Liping Zhao, Haipeng Sun, Xianxuan Zhou, Baolin Sun. LsrR-binding site recognition and regulatory characteristics in Escherichia coli AI-2 quorum sensing [J]. Cell Research, 2009: 1258-1268. 
			</I></DIV></DIV></DIV></DIV></DIV></BODY></HTML>