<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_CPU_CHINA_Demonstrate skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:CPU CHINA/Demonstrate</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV><DIV class="btn-group" id="pro" style=" width: 100px; left: 23%;"><UL id="proul" class="dropdown-menu" style="background:url(https://static.igem.org/mediawiki/2018/d/d3/T--CPU_CHINA--dropdown1.png);background-size:150%;background-repeat:repeat;"><LI><A href="https://2018.igem.org/Team:CPU_CHINA/Background">Background</A></LI><LI><A href="https://2018.igem.org/Team:CPU_CHINA/Demonstrate">Demonstrate</A></LI><LI><A href="https://2018.igem.org/Team:CPU_CHINA/Experiments">Experiments</A></LI><LI><A href="https://2018.igem.org/Team:CPU_CHINA/Application">Application</A></LI><LI><A href="https://2018.igem.org/Team:CPU_CHINA/Parts">Parts</A></LI><LI><A href="https://2018.igem.org/Team:CPU_CHINA/InterLab">InterLab</A></LI><LI><A href="https://2018.igem.org/Team:CPU_CHINA/Improve">Improve</A></LI></UL></DIV><DIV class="btn-group" id="hp" style=" width: 190px;left: 32%;"><UL id="hpul" class="dropdown-menu" style="background:url(https://static.igem.org/mediawiki/2018/4/4a/T--CPU_CHINA--dropdown2.png)"><LI><A href="https://2018.igem.org/Team:CPU_CHINA/Human_Practices">Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:CPU_CHINA/HP/Silver">Silver HP</A></LI><LI><A href="https://2018.igem.org/Team:CPU_CHINA/HP/Gold_Integrated">Intergrated and Gold HP</A></LI><LI><A href="https://2018.igem.org/Team:CPU_CHINA/Engagement">Education and Public Engagement</A></LI></UL></DIV><DIV class="btn-group" id="nb" style=" width: 105px; left: 65%;"><UL id="nbul" class="dropdown-menu" style="background:url(https://static.igem.org/mediawiki/2018/a/a5/T--CPU_CHINA--dropdown3.png)"><LI><A href="https://2018.igem.org/Team:CPU_CHINA/Notebook/Log">Log</A></LI><LI><A href="https://2018.igem.org/Team:CPU_CHINA/Safety">Safety</A></LI></UL></DIV><DIV class="btn-group" id="team" style=" width: 95px; left: 75%;"><UL id="teamul" class="dropdown-menu" style="background:url(https://static.igem.org/mediawiki/2018/c/c5/T--CPU_CHINA--dropdown4.png)"><LI><A href="https://2018.igem.org/Team:CPU_CHINA/Team">Team</A></LI><LI><A href="https://2018.igem.org/Team:CPU_CHINA/Attributions">Attributions</A></LI><LI><A href="https://2018.igem.org/Team:CPU_CHINA/Collborations">Collaborations</A></LI></UL></DIV></DIV><DIV class="container" id="container1" style="position:relative;top:180px;display:block;"><BUTTON id="clickMe" style="width: 20%;position: absolute;height:100px;left: 40%;top: 820px;background-color: #feeca9;border: none;color: #c9b083;font-size: 23px; display:none;z-index:1;" onclick="showContent()">RNAi</BUTTON><DIV class="door"><H1 style="font-size:2.3rem;text-align:center">&quot;The AND gate is a basic digital logic gate that implements logical conjunction. A HIGH output (1) results only if all the inputs to the AND gate are HIGH (1).&quot;</H1><H3 style="text-align:right;font-size:2rem;">——Wikipedia</H3><DIV class="pumpkins"><DIV class="pumpkin-left"><DIV id="htert" style="width: 250%;/* height: 50px; */font-size: 1em;position: relative;top: -20px;left: -10%;color: #d01f1f !important;font-size: 20px;/* background-color:white; *//* border-radius:2rem; */display:block;">
    	I am <I>Promoter hTERT!</I> Click me!
   </DIV></DIV><DIV class="pumpkin-right"><DIV id="hulc" style="width: 250%;/* height: 50px; */font-size: 1em;position: relative;top: -20px;left: -10%;color: #d01f1f !important;font-size: 20px;/* background-color:white; *//* border-radius:2rem; */ display:block;">
    	I am <I>Promoter Hulc!</I> Click me!
   </DIV></DIV></DIV></DIV></DIV><DIV class="container" id="container2" style="position:relative;top:180px;display:none;"><H1>A gene therapy strategy to target hepatocellular carcinoma based on conditional RNA interference
	</H1><H2>1.Description
	</H2><H4>As described in <A href="https://2018.igem.org/Team:CPU_CHINA/Background"><U>Background</U></A> , spatial and/or temporal regulation of RNAi is of significant importance for basic research as well as practical applications. Since disease-specific promoters only have high activity in pathogenic cells, our RNAi becomes conditional and specific for pathogenic cells as we put genes of the RdRp and non-coding RNAs behind them (Figure 1). When the two devices become transcriptionally activated together, RNA interference occurs. This actually forms a logical “AND” gate - it behaves according to the truth table on the right.
	</H4><H5>Figure 1. The “AND” gate based on disease-specific promoters</H5><H3>1.1 Key procedures
	</H3><H4>DROSHA can’t cleave pri-miRNA (green) when the latter binds with the inhibitory strand (yellow), which means no RNAi (Figure 2A). 
	</H4><H4>SLD3 is a short template competent for <I>de novo</I> RNA synthesis. It is located at the 3’ end of the inhibitory strand. NS5B, the RNA dependent RNA polymerase, interacts with the 2’-OH and 3’-OH of the two cytosines at the 3’ end of the SLD3 sequence, then starts RNA polymerization using the RNA promoter and the inhibitory strand as the template. It separates the inhibitory strand from the pri-miRNA, exposing the single-stranded area. The Microprocessor then recognizes this substrate and cleaves it into precursor miRNAs, which eventually would become mature miRNAs (Figure 2B).
	</H4><H5>Figure 2. The key procedures of the design</H5><H4>In the Tet-off system (Figure 2C), when Tetracycline (Tc) or Doxycycline (Dox) is absent, tetracycline transactivator (tTA) binds with the tetracycline response element (TRE), which then activates the downstream gene expression of a miRNA sponge. MiRNA sponge absorbs the miRNA targeting MAP4K4 to again block the RNAi pathway. When Tc or Dox is present, tTA changes its conformation to bind with them, thus preventing the downstream genes from expression.
	</H4><H3>1.2 Features
	</H3><H4>1.2.1 Cancer-specific promoters</H4><H4>Our gene therapy strategy utilizes two cancer-specific promoters (one HCC specific) to open an AND-gated system to target HCC.
	</H4><H4><I>hTERT</I> promoter is the core promoter of human telomerase reverse transcriptase gene. It is one of the most well-known cancer-specific promoters. A number of factors like cellular transcriptional activator c-Myc, HIF-1 as well as the repressors p53, WT1, and Menin, most of which comprise tumor suppressor gene products, have been identified to directly or indirectly regulate the hTERT promoter, contributing its exclusive up-regulation in cancer.
	</H4><H4><I>HULC</I> encodes a long non-coding RNA (lncRNA) and plays an important role in tumorigenesis. It is one of the most up-regulated genes in hepatocellular carcinoma and such activation, like the one for <I>hTERT</I>, is at transcriptional level. Here we only use a 132bp <I>HULC</I> core promoter because literature suggests that the most proximal 84nt of the <I>Hulc</I> gene contains several transcription factor binding sites and is capable of initiating transcription just fine.
	</H4><H4>When these two promoters work together, which only happens in tumor cells (especially malignant ones), the AND gate is “opened”, releasing miRNA and damaging themselves. 
	</H4><H4>1.2.2 Controlled by small molecules</H4><H4>Our gene therapy strategy can be controlled by a small-molecule drug.
	</H4><H4>As we mentioned above, the Tet-off system can also adjust the amount of functional miRNA, which should be instructive for individual drug administration <I>in vivo</I>.
	</H4><H4>1.2.3 Wide application on other diseases</H4><H4>The gene therapy strategy shown here is actually a proof of concept. As a RNA interference strategy, it has its inherent flexibility to be adapted to target any messenger RNAs and to other disease settings as long as there are corresponding disease-specific promoters. This is discussed in detail in <I>Application</I>(link) page. 
	</H4><H3>1.3 Plasmid construction
	</H3><H4>PlasmidⅠcontains <I>hTERT</I> promoter, transactivator <I>tTA</I> and RNA dependent RNA polymerase <I>NS5B</I>. We loaded a nuclear location sequence (NLS) to NS5B (NS5B<SUP>NLS</SUP>) for this protein to be transported into the nucleus. PlasmidⅡcontains the HULC promoter, genes encoding the pri-miRNA and the inhibitory strand. Notably, <I>tTA</I> and <I>TRE</I> are located separately in two plasmids (Figure 3). More information can be found in <A href="https://2018.igem.org/Team:CPU_CHINA/Parts"><U><I>Part</I></U></A>.
	</H4><H5>Figure 3. Plasmid construction of our system</H5><H2>2.Results
	</H2><H4>To apply our parts, we chose pIRES and pcDNA3.1 as backbones for the plasmidⅠandⅡ vectors, respectively. But before connecting all the parts, we constructed plasmids containing only some of the parts to verify the proper function of each part. Here the plasmids we used involve p1 (pIRES-hTERT-tTA-NS5B<SUP>NLS</SUP>), p-H(pcDNA3.1-HULC-pri-miRNA), p-H-U6(pcDNA3.1-HULC-pri-miRNA-U6-inhSi-pri-miRNA), and p2 (pcDNA3.1-HULC-pri-miRNA(MAP4K4)-U6-inhSi-pri-miRNA-tre-sponge).
	</H4><H3>2.1 Determine the promoters’ activity and specificity
	</H3><H4>Our system includes two specific promoters: <I>hTERT</I> and <I>HULC</I>. We used pGL3-Basic vector, a promoter-less vector for the luciferase assay to determine the transcriptional activity of these <A href="https://2018.igem.org/Team:CPU_CHINA/Experiments?promoters=1"><U>promoters</U></A>. We added the promoters on pGL3-Basic vector and measure the OD value of the luciferase activity. We chose SV40, a highly activated promoter in both cancer cells (Figure 4A) and normal cells (Figure 4B) as positive control (PGL3-CON).
	</H4><H5>Figure 4. The efficacy and specificity of the cancer-specific promoters</H5><H3>2.2 Function verification of NS5B
	</H3><H4>Successful expression of NS5B<SUP>NLS</SUP> of p1 is first verified by western blot (data now shown). In Figure 5, the results of <A href="https://2018.igem.org/Team:CPU_CHINA/Experiments?IF=1"><U>immuno-fluorescence</U></A> shows the ability of nuclear translocation of NS5B<SUP>NLS</SUP> has improved. (See detailed information in <A href="https://2018.igem.org/Team:CPU_CHINA/Improve"><U><I>Improve</I></U></A>)
	</H4><H5>Figure 5. Location of NS5B</H5><H3>2.3 Conditional RNA interference
	</H3><H4>To test the efficiency of our system, we performed quantitative <A href="https://2018.igem.org/Team:CPU_CHINA/Experiments?qPCR=1">PCR (qPCR)</A> on the effector miRNA and the targeted MAP4K4 <U>mRNA</U>(Figure 6A). Pri-miRNA analogue was successfully encoded and processed into miRNA, however, with the presence of the inhibitory strand, the amount of miRNA sharply decreased since DROSHA cannot cleave the pri-miRNA. This can be further confirmed in Figure 6B where a significant increase of mRNA was observed after expression of the inhibitory strand, which also indicates that our miRNA can successfully target MAP4K4.
	</H4><H4>However, from Figure 6A and 7B we found that NS5B did not function as expected, the inhibitory strand hardly removed. Taken from the results discussed above this might be due to insufficient presence of NS5B in the nucleus. However, the efficiency of nucleus translocation can be improved with e.g. adding two NLSs, thus our system might still work. Efforts will be paid regarding this issue in the future.
	</H4><H5>Figure 6. The expression of miRNA(MAP4K4) (A) and MAP4K4 mRNA (B)</H5><H4>For the Tet-off system, when the two plasmids worked together, we saw an increase of MAP4K4 mRNA (Figure 6B), which indicates successful down-regulation of the miRNA by the miRNA sponge.
	</H4><H3>References</H3><H5>[1]	Cai, X., Hagedorn, C. H., &amp; Cullen, B. R. (2004). Human micrornas are processed from capped, polyadenylated transcripts that can also function as mrnas. <I>RNA, 10</I>(12), 1957.
	</H5><H5>[2]	Amuthan, G. (2004). The microprocessor complex mediates the genesis of micrornas. <I>Nature, 432</I>(7014), 235-40.
	</H5><H5>[3] Han, J., Lee, Y., Yeom, K. H., Nam, J. W., Heo, I., &amp; Rhee, J. K., et al. (2006). Molecular basis for the recognition of primary micrornas by the drosha-dgcr8 complex. <I>Cell, 125</I>(5), 887-901.
	</H5><H5>[4] Castanotto, D., &amp; Rossi, J. J. (2009). The promises and pitfalls of rna-interference-based therapeutics. <I>Nature, 457</I>(7228), 426-433.
	</H5><H5>[5]	Zeng, Y., &amp; Cullen, B. R. (2005). Efficient processing of primary microrna hairpins by drosha requires flanking nonstructured rna sequences. <I>Journal of Biological Chemistry, 280</I>(30), 27595-603.
	</H5><H5>[6]	Beisel, C. L., Chen, Y. Y., Culler, S. J., Hoff, K. G., &amp; Smolke, C. D. (2011). Design of small molecule-responsive micrornas based on structural requirements for drosha processing. <I>Nucleic Acids Research,39</I>(7), 2981-2994.
	</H5><H5>[7]	Kumar, D., An, C. I., &amp; Yokobayashi, Y. (2009). Conditional rna interference mediated by allosteric ribozyme. <I>Journal of the American Chemical Society, 131</I>(39), 13906-13907.
	</H5><H5>[8]	Cheng, H., Zhang, Y., Wang, H., Sun, N., Liu, M., &amp; Chen, H., et al. (2016). Regulation of map4k4 gene expression by rna interference through an engineered theophylline-dependent hepatitis delta virus ribozyme switch. <I>Molecular Biosystems, 12</I>(11), 3370-3376.
	</H5><H5>[9]	Zhang, Y., Wang, J., Cheng, H., Sun, N., Liu, M., &amp; Wu, Z., et al. (2017). Inducible bcl-2 gene rna interference mediated by aptamer-integrated hdv ribozyme switch. <I>Integrative Biology Quantitative Biosciences from Nano to Macro, 9</I>(7), 619.
	</H5><H5>[10] Moradpour, Darius, Volker, Gosert, Rainer, &amp; Wölk, et al. (2002). Hepatitis c: molecular virology and antiviral targets. <I>Trends in Molecular Medicine, 8</I>(10), 476-482.
	</H5><H5>[11] Brass, V., Gouttenoire, J., Wahl, A., Pal, Z., Blum, H. E., &amp; Penin, F., et al. (2010). Hepatitis c virus rna replication requires a conserved structural motif within the transmembrane domain of the ns5b rna-dependent rna polymerase. <I>Journal of Virology, 84</I>(21), 11580.
	</H5><H5>[12] Vo, N. V., Tuler, J. R., &amp; Lai, M. M. (2004). Enzymatic characterization of the full-length and c-terminally truncated hepatitis c virus rna polymerases: function of the last 21 amino acids of the c terminus in template binding and rna synthesis. <I>Biochemistry, 43</I>(32), 10579.
	</H5><H5>[13] Lee, K. J., Choi, J., Ou, J. H., &amp; Lai, M. M. (2004). The c-terminal transmembrane domain of hepatitis c virus (hcv) rna polymerase is essential for hcv replication in vivo. <I>Journal of Virology, 78</I>(7), 3797.
	</H5><H5>[14] Lohmann, V., Körner, F., Herian, U., &amp; Bartenschlager, R. (1997). Biochemical properties of hepatitis c virus ns5b rna-dependent rna polymerase and identification of amino acid sequence motifs essential for enzymatic activity. <I>Journal of Virology, 71</I>(11), 8416-8428.
	</H5><H5>[15] <A href="http://parts.igem.org/Part:BBa_K1442100"><U>http://parts.igem.org/Part:BBa_K1442100</U></A></H5><H5>[16] Kao, C. C., Yang, X., Kline, A., Wang, Q. M., Barket, D., &amp; Heinz, B. A. (2000). Template requirements for rna synthesis by a recombinant hepatitis c virus rna-dependent rna polymerase.<I> Journal of Virology,74</I>(23), 11121.
	</H5><H5>[17] <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1442304"><U>http://parts.igem.org/wiki/index.php?title=Part:BBa_K1442304</U></A></H5><H5>[18] O'Farrell, D., Trowbridge, R., Rowlands, D., &amp; Jager, J. (2003). Substrate complexes of hepatitis c virus rna polymerase (hc-j4): structural evidence for nucleotide import and de-novo initiation. <I>Journal of Molecular Biology,326</I>(4), 1025-1035.
	</H5><H5>[19] Rhyu, M. S. (1995). Telomeres, telomerase, and immortality. <I>J Natl Cancer Inst, 87</I>(12), 884-894.
	</H5><H5>[20] Buseman, C. M., Wright, W. E., &amp; Shay, J. W. (2012). Is telomerase a viable target in cancer?. <I>Mutation Research - Fundamental and Molecular Mechanisms of Mutagenesis, 730</I>(1-2), 90-97.
	</H5><H5>[21] Nakayama, J., Tahara, H., Tahara, E., Saito, M., Ito, K., &amp; Nakamura, H., et al. (1998). Telomerase activation by htrt in human normal fibroblasts and hepatocellular carcinomas.<I> Nature Genetics, 18</I>(1), 65-68.
	</H5><H5>[22] Poole, J. C., Andrews, L. G., &amp; Tollefsbol, T. O. (2001). Activity, function, and gene regulation of the catalytic subunit of telomerase (hTERT). <I>Gene</I>, 1-12.
	</H5><H5>[23] Kyo, S., Kanaya, T., Takakura, M., Tanaka, M., &amp; Inoue, M. (1999). Human telomerase reverse transcriptase as a critical determinant of telomerase activity in normal and malignant endometrial tissues. <I>International Journal of Cancer</I>, 80(1), 60-63.
	</H5><H5>[24] Aisner, D. L., Wright, W. E., &amp; Shay, J. W. (2002). Telomerase regulation: not just flipping the switch. <I>Current Opinion in Genetics &amp; Development,12</I>(1), 80-85.
	</H5><H5>[25] Nakamura, T. M., &amp; Cech, T. R. (1997). Telomerase catalytic subunit homologs from fission yeast and human. <I>Science, 277</I>(5328), 955.
	</H5><H5>[26] Meyerson, M., Counter, C. M., Eaton, E. N., Ellisen, L. W., Steiner, P., &amp; Caddle, S. D., et al. (1997). Hest2, the putative human telomerase catalytic subunit gene, is up-regulated in tumor cells and during immortalization. <I>Cell, 90</I>(4), 785-795.
	</H5><H5>[27] Zhou, X. U., Lu, J., &amp; Zhu, H. (2016). Correlation between the expression of htert gene and the clinicopathological characteristics of hepatocellular carcinoma. <I>Oncology Letters, 11</I>(1), 111.
	</H5><H5>[28] Kyo, S., Takakura, M., Fujiwara, T., &amp; Inoue, M. (2010). Understanding and exploiting htert promoter regulation for diagnosis and treatment of human cancers. <I>Cancer Science, 99</I>(8), 1528-1538.
	</H5><H5>[29] Takakura, M., Kyo, S., Kanaya, T., Hirano, H., Takeda, J., &amp; Yutsudo, M., et al. (1999). Cloning of human telomerase catalytic subunit (htert) gene promoter and identification of proximal core promoter sequences essential for transcriptional activation in immortalized and cancer cells. <I>Cancer Research, 59</I>(3), 551-557.
	</H5><H5>[30] <A href="http://parts.igem.org/Part:BBa_K1722002"><U>http://parts.igem.org/Part:BBa_K1722002</U></A></H5><H5>[31] <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1922001"><U>http://parts.igem.org/wiki/index.php?title=Part:BBa_K1922001</U></A></H5><H5>[32] <A href="http://parts.igem.org/Part:BBa_K1699001"><U>http://parts.igem.org/Part:BBa_K1699001&quot;&gt;</U></A></H5><H5>[33] <A href="http://parts.igem.org/Part:BBa_K1722001"><U>http://parts.igem.org/Part:BBa_K1722001</U></A></H5><H5>[34] Siegel, R. L., Miller, K. D., &amp; Jemal, A. (2015). Cancer statistics, 2015. <I>CA: A Cancer Journal for Clinicians, 65</I>(1), 5-29.
	</H5><H5>[35] Riordan, S. M., &amp; Williams, R. (2017). Medical management of hepatocellular carcinoma. <I>Journal of Oncology Practice, 13</I>(6), 356.
	</H5><H5>[36] Panzitt, K., Tschernatsch, M. M., Guelly, C., Moustafa, T., Stradner, M., &amp; Strohmaier, H. M., et al. (2007). Characterization of hulc, a novel gene with striking up-regulation in hepatocellular carcinoma, as noncoding rna. <I>Gastroenterology, 132</I>(1), 330-342.
	</H5><H5>[37] Wang, J., Liu, X., Wu, H., Ni, P., Gu, Z., &amp; Qiao, Y., et al. (2010). Creb up-regulates long non-coding rna, hulc expression through interaction with microrna-372 in liver cancer. <I>Nucleic Acids Research, 38</I>(16), 5366-5383.
	</H5><H5>[38] Collins, C. S., Hong, J., Sapinoso, L., Zhou, Y., Liu, Z., &amp; Micklash, K., et al. (2006). A small interfering rna screen for modulators of tumor cell motility identifies map4k4 as a promigratory kinase. <I>Proc Natl Acad Sci U S A, 103</I>(10), 3775-3780.
	</H5><H5>[39] Han, S. X., Zhu, Q., Ma, J. L., Zhao, J., Huang, C., &amp; Jia, X., et al. (2010). Lowered hgk expression inhibits cell invasion and adhesion in hepatocellular carcinoma cell line hepg2. <I>World Journal of Gastroenterology, 16</I>(36), 4541-4548.
	</H5><H5>[40] Gao, X., Gao, C., Liu, G., &amp; Hu, J. (2016). Map4k4: an emerging therapeutic target in cancer. <I>Cell &amp; Bioscience, 6</I>(1), 56.
	</H5><H5>[41] Das, A. T., Tenenbaum, L., &amp; Berkhout, B. (2016). Tet-on systems for doxycycline-inducible gene expression. <I>Current Gene Therapy, 16</I>(3), 
	</H5><H5>[42] Stieger, K., Belbellaa, B., Guiner, C. L., Moullier, P., &amp; Rolling, F. (2009). In vivo, gene regulation using tetracycline-regulatable systems ☆. <I>Advanced Drug Delivery Reviews, 61</I>(7), 527-541.
	</H5><H5>[43] Gu, J., Zhang, L., Huang, X., Lin, T., Yin, M., &amp; Xu, K., et al. (2002). A novel single tetracycline-regulative adenoviral vector for tumor-specific bax gene expression and cell killing in vitro and in vivo. <I>Oncogene,21</I>(31), 4757-4764.
	</H5><H5>[44] <A href="https://2013.igem.org/Team:SYSU-China/Project/Design"><U>https://2013.igem.org/Team:SYSU-China/Project/Design</U></A></H5></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>