<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_CPU_CHINA_Background skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:CPU CHINA/Background</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV><DIV class="btn-group" id="pro" style=" width: 100px; left: 23%;"><UL id="proul" class="dropdown-menu" style="background:url(https://static.igem.org/mediawiki/2018/d/d3/T--CPU_CHINA--dropdown1.png);background-size:150%;background-repeat:repeat;"><LI><A href="https://2018.igem.org/Team:CPU_CHINA/Background">Background</A></LI><LI><A href="https://2018.igem.org/Team:CPU_CHINA/Demonstrate">Demonstrate</A></LI><LI><A href="https://2018.igem.org/Team:CPU_CHINA/Experiments">Experiments</A></LI><LI><A href="https://2018.igem.org/Team:CPU_CHINA/Application">Application</A></LI><LI><A href="https://2018.igem.org/Team:CPU_CHINA/Parts">Parts</A></LI><LI><A href="https://2018.igem.org/Team:CPU_CHINA/InterLab">InterLab</A></LI><LI><A href="https://2018.igem.org/Team:CPU_CHINA/Improve">Improve</A></LI></UL></DIV><DIV class="btn-group" id="hp" style=" width: 190px;left: 32%;"><UL id="hpul" class="dropdown-menu" style="background:url(https://static.igem.org/mediawiki/2018/4/4a/T--CPU_CHINA--dropdown2.png)"><LI><A href="https://2018.igem.org/Team:CPU_CHINA/Human_Practices">Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:CPU_CHINA/HP/Silver">Silver HP</A></LI><LI><A href="https://2018.igem.org/Team:CPU_CHINA/HP/Gold_Integrated">Intergrated and Gold HP</A></LI><LI><A href="https://2018.igem.org/Team:CPU_CHINA/Engagement">Education and Public Engagement</A></LI></UL></DIV><DIV class="btn-group" id="nb" style=" width: 105px; left: 65%;"><UL id="nbul" class="dropdown-menu" style="background:url(https://static.igem.org/mediawiki/2018/a/a5/T--CPU_CHINA--dropdown3.png)"><LI><A href="https://2018.igem.org/Team:CPU_CHINA/Notebook/Log">Log</A></LI><LI><A href="https://2018.igem.org/Team:CPU_CHINA/Safety">Safety</A></LI></UL></DIV><DIV class="btn-group" id="team" style=" width: 95px; left: 75%;"><UL id="teamul" class="dropdown-menu" style="background:url(https://static.igem.org/mediawiki/2018/c/c5/T--CPU_CHINA--dropdown4.png)"><LI><A href="https://2018.igem.org/Team:CPU_CHINA/Team">Team</A></LI><LI><A href="https://2018.igem.org/Team:CPU_CHINA/Attributions">Attributions</A></LI><LI><A href="https://2018.igem.org/Team:CPU_CHINA/Collborations">Collaborations</A></LI></UL></DIV></DIV><DIV id="fixedAnchor" style="position:fixed;left:0;top:250px;"><UL class="dropdown-menu1" style="background:url(https://static.igem.org/mediawiki/2018/d/d3/T--CPU_CHINA--dropdown1.png);z-index:10000000000;"><LI><A href="#Pri-miRNA">Pri-miRNA</A></LI><LI><A href="#Conditional RNAi">Conditional RNAi</A></LI><LI><A href="#RdRP">RdRP</A></LI><LI><A href="#Cancer Gene Therapy">Cancer Gene Therapy</A></LI></UL></DIV><DIV class="container" style="position:relative;top:300px;"><H1 id="Pri-miRNA">Pri-miRNA</H1><H4>MiRNAs are a group of short non-coding RNAs that mediate post-transcriptional gene silencing (PTGS). Binding of a miRNA to its target mRNA mediates mRNA degradation and blocks translation. MiRNA biogenesis initiates with the transcription of the pri‑miRNA. General domains of a pri-miRNA is shown on the right. These long pri‑miRNAs are then cleaved by Microprocessor, which comprises the double‑stranded RNase III enzyme DROSHA and its essential cofactor, the double‑stranded RNA (dsRNA)‑binding protein DiGeorge syndrome critical region 8 (DGCR8). DROSHA requires long single-stranded regions flanking the short hairpin structure in its substrates. If the basal ssRNA is somehow blocked, the cleavage reaction would also be blocked.</H4><H5>For more information, please see <A class="ref" id="r1">Reference(1)</A>. The figure General domains of a pri-miRNA and Schematic diagram of the base-anchor model come from <A class="ref" id="r2">Reference (2)</A> and<A class="ref" id="r3">(3)</A>.</H5><H1 id="Conditional RNAi">Conditional RNAi</H1><H4>Historically, PTGS can now be called RNA interference (RNAi).Because synthetic dsRNA introduced into cells can selectively and robustly induce suppression of specific genes of interest, RNAi is now a valuable research tool.More recently, oligonucleotides and various RNA molecules based on RNA interference (RNAi) have proven their potential as reagents for gene therapy applications. However, RNAi itself has no selectivity over the sites of interest. For cancer therapy to be discussed below,RNAi impairs not only the tumor cells but also the normal cells. Besides, constitutive knock-down of genes may result in undesirable outcomes in such biomedical applications.Therefore, spatial and/or temporal regulation of RNAi is of significant importance for basic research as well as practical applications of RNAi. This is the basic idea of conditional RNAi and what we CPU_CHINA this year are trying to do.</H4><H5>For examples of condition RNAi, please see <A class="ref" id="r4">Reference (4)</A> and <A class="ref" id="r5">(5)</A>.</H5><H1 id="RdRP">RdRP</H1><H4>RNA-dependent RNA polymerase (RdRP) is an enzyme that catalyzes the replication of RNA from an RNA template. It is quite an essential protein for all RNA-containing viruses with no DNA stage i.e. only RNA viruses.With a positive sense single-stranded RNA as its genome, Hepatitis C virus is such a virus. The corresponding RdRP is the nonstructural protein 5B (NS5B) of the HCV polyprotein. NS5B could not direct RNA synthesis without an RdRP-specific/RNApromoter, which contains a stable secondary structure and a single-stranded sequence that contained at least one 3’cytidylate.This polymerase and its corresponding RNA promoters have actually been used by Warwick in the 2014 iGEM competition.</H4><H5>For more information, please see <A class="ref" id="r6">Reference (6)</A>. The figure Ribbon diagram showing the overall structures of HC-J4comes from <A class="ref" id="r7">Reference(7)</A>.</H5><H1 id="Cancer Gene Therapy">Cancer Gene Therapy</H1><H4>Gene therapy is the transfer of defined genetic material to specific target cells of a patient for the ultimate purpose of preventing or altering a particular disease state. Since cancer is mediated by somatic aberration in the host genome, it is not difficult to explain the enthusiasm about gene therapy among cancer researchers. It is anticipated that gene therapy will play an important role in future cancer therapy as part of a multimodality treatment, in combination with, or following other forms of cancer therapy. In this year’s iGEM competition, we focus on hepatocellular carcinoma (HCC), which is the sixth most common cancer and ranks third in cancer-related deaths. Although we have treatments such as transplantation, surgical resection and local ablation for early HCC, problems lie in the lack of donor, postoperative recurrence, etc.</H4><H4>1.	Han, J., Lee, Y., Yeom, K.-H., Nam, J.-W., Heo, I., Rhee, J.-K., Sohn, S.Y., Cho, Y., Zhang, B.-T. and Kim, V.N. (2006) Molecular Basis for the Recognition of Primary microRNAs by the Drosha-DGCR8 Complex. Cell, 125, 887-901.
</H4><H4>2.	Beisel, C.L., Chen, Y.Y., Culler, S.J., Hoff, K.G. and Smolke, C.D. (2011) Design of small molecule-responsive microRNAs based on structural requirements for Drosha processing. Nucleic Acids Research, 39, 2981-2994.
</H4><H4>3.	Burke, J.M., Kelenis, D.P., Kincaid, R.P. and Sullivan, C.S. (2014) A central role for the primary microRNA stem in guiding the position and efficiency of Drosha processing of a viral pri-miRNA. RNA, 20, 1068-1077.
</H4><H4>4.	Kumar, D., An, C.-I. and Yokobayashi, Y. (2009) Conditional RNA Interference Mediated by Allosteric Ribozyme. Journal of the American Chemical Society, 131, 13906-13907.
</H4><H4>5.	Cheng, H., Zhang, Y., Wang, H., Sun, N., Liu, M., Chen, H. and Pei, R. (2016) Regulation of MAP4K4 gene expression by RNA interference through an engineered theophylline-dependent hepatitis delta virus ribozyme switch. Molecular bioSystems, 12, 3370-3376.
</H4><H4>6.	Kao, C.C., Yang, X., Kline, A., Wang, Q.M., Barket, D. and Heinz, B.A. (2000) Template Requirements for RNA Synthesis by a Recombinant Hepatitis C Virus RNA-Dependent RNA Polymerase. Journal of Virology, 74, 11121-11128.
</H4><H4>7.	O'Farrell, D., Trowbridge, R., Rowlands, D. and Jager, J. (2003) Substrate Complexes of Hepatitis C Virus RNA Polymerase (HC-J4): Structural Evidence for Nucleotide Import and De-novo Initiation. Journal of Molecular Biology, 326, 1025-1035.


</H4></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>