Team:UNIK Copenhagen/Construct
Home
Project
Green Lab
Green Lab Overview
Experimental Design
Results
Red Lab
Red Lab Overview
Mars
Soil Experiments
Thermo Gradient Prototype
Meet Us
Team
Attributions
Collaborations
Outreach
Presentations
In the Media
Comic Strip
Social Media
Ethics
Achievements
Safety
FAQ
Follow @SpaceMossUCPH
Experimental Design
Antifreeze Protein
The antifreeze protein used in this project is normally produced by an insect, the Spruce Budworm, Choristoneura fumiferana [3]. The structure of the antifreeze protein is composed of beta-sheets stacked parallely. From a cross section the protein appears triangular with rectangular sides [4] (Fig. 1).
Figure 1: Protein structure models of the antifreeze protein from the spruce budworm. Structures were determined with X-ray diffraction, 2.30 Å [9]
Through Uniprot we found the amino acid sequence of the protein, where it had the entry name Q9GTP0 [3][5]⁠. We then converted the sequence of amino acids into DNA codons. This DNA sequence were then codon optimized for expression in Physcomitrella Patens. Lastly, we sent the DNA sequence to IDT where the gene was synthesized. The aim is to confirm expression of this novel gene in P. Patens and add it as a new Biobrick to the registry.
Resveratrol
Resveratrol is a phenolic compound found in grapes (Vitis vinifera) that is believed to have wide range of health benefits in mammals. Resveratrol exists as both cis- and trans-isomers, but only the trans-isomer has been found in grapes [2]. Resveratrol is produced with intermediates from the phenylpropanoid pathway and is derived from p-coumaric acid which is an intermediate in lignin production. The two key enzymes are Coenzyme A (CoA) Ligase (4CL) and Stilbene Synthase (STS) [6] (fig. 2).
Figure 2: The biosynthetic pathway of Resveratrol. The key enzymes are Coenzyme A (CoA) Ligase (4CL) and Stilbene Synthase (STS). 4CL couples CoA to p-coumaric acid forming coumaroyl-CoA and subsequently STS forms Resveratrol by adding 3 malonyl-CoA groups and releasing CO2 [6].
P. patens has been shown to produce enzymes similar to 4CL. These enzymes from the Pp4CL family (P. Patens) have been shown to have similar function as enzymes from the 4CL family of higher plants [7]. This means that P. Patens only lack the production of STS to produce Resveratrol.
The STS is already in the registry (Part:BBa_K1033002) but was not available when we started our work. Instead, the STS-gene were kindly provided by Bachberry (also  thanks to Brian King for helping us getting the gene). The aim is to improve the Biobrick (Bba_K1033002) by expressing STS in P. Patens and detect Resveratrol using Liquid chromatography–mass spectrometry (LC-MS).
Genetic Constructs and Transformation
P. Patens is able to do homologous recombination [1] and we would use this to our advantage when transforming our moss. We generated large genetic constructs with PCR, that consisted of separate, linear DNA pieces (fig. 3, top). When we made the primers, we ensured that our linear DNA pieces had matching overhangs so that they after transformation would be assembled in vivo due to homologous recombination.
Piece A (fig. 3, top) was amplified from the (004 vector) with PCR and included a region homologous to the 108 locus on the moss genome (accession number: GQ250943). Piece A then contained the neomycin phosphotransferase II gene (nptII) driven by the 35S Cauliflower Mosaic Virus promoter (35S CaMV). The nptII-resistance cassette produces an aminoglycoside 3'-phosphotransferase that confers resistance to kanamycin [8]⁠. Lastly, it had the Zea Maize Ubiquitin Promoter (ZmUbi), which is a strong constitutive promoter from corn that would drive our gene of interest. The second DNA piece was our gene of interest, which was either the antifreeze gene (Piece B) or STS (Piece C). The last DNA piece (Piece D) was amplified from the (007-venus vector) with PCR and contained Yellow Fluorescent Protein (YFP) to confirm a successful transformation. After YFP there was a terminator and then a similar 108 region. Between the gene of interest and YFP, we had the LP4 sequence, which was made entirely with primer overhangs. This sequence translates into a linker peptide, which has a recognition site that is cleaved by a protease11⁠. The LP4 sequence ensures that the antifreeze protein or STS was separated from YFP and that their function remained unchanged.
Using PEG-mediated transformation we transformed Piece A, B and D into moss protoplasts and Piece A, C and D into moss protoplasts [10]. The pieces would be assembled in vivo in the moss cells (fig. 3, bottom) and integrated into the genome (fig. 3, bottom) due to homologous recombination. Those two constructs will later be referred to as the antifreeze construct or the STS construct, depending on the gene of interest. The moss protoplasts were left to grow a few days on plates containing media with nutrients for P. patens (PhyB-media).
Click on the different regions in the construct below to learn about them.
Figure 3: Genetic construct used for transforming P.Patens. Top: Four different DNA  pieces were generated with PCR. Piece A with the 108 region,  nptII-resistance marker driven by the 35S CaMV promoter (omitted  from figure) and the Zea Mays ubiquitin promoter (ZmUbi). Piece B  consisted of our antifreeze gene. Piece C consisted of the STS-gene.  Piece B/C had the LP4 sequence at 3' end. Piece D had the LP4  sequence at the 5' end and consisted of Yellow Fluorescent Protein  (YFP), terminator and a similar 108 region. Bottom: in vitro assembly of the DNA pieces after transformation into P.Patens by homologous  recombination and stable integration of the construct into the 108 locus on the moss genome.
References:
[1] Kamisugi, Y. et al. The mechanism of gene targeting in Physcomitrella patens: Homologous recombination, concatenation and multiple integration. Nucleic Acids Res. 34, 6205–6214 (2006).
[2] Bhat KPL, Kosmeder, J. W. & Pezzuto, J. M. Biological effects of resveratrol. Antioxid. Redox Signal. 3, 1041–1064 (2001).
[3] Tyshenko, M. G., Doucet, D., Davies, P. L. & Walker, V. K. The antifreeze potential of the spruce budworm thermal hysteresis protein. Nat. Biotechnol. 15, 887–890 (1997).
[4]Graether, S. P. et al. Beta-helix structure and ice-binding properties of a hyperactive antifreeze protein from an insect. Nature 406, 325–328 (2000).
[5] Tyshenko, M. G., Doucet, D. & Walker, V. K. Analysis of antifreeze proteins within spruce budworm. Insect Mol. Biol. 14, 319–326 (2005).
[6] Beekwilder, J. et al. Production of resveratrol in recombinant microorganisms. Appl. Environ. Microbiol. 72, 5670–5672 (2006).
[7] Silber, M. V., Meimberg, H. & Ebel, J. Identification of a 4-coumarate:CoA ligase gene family in the moss, Physcomitrella patens. Phytochemistry 69, 2449–2456 (2008).
[8] Beck, E., Ludwig, G., Auerswald, E. a, Reiss, B. & Schaller, H. Nucleotide sequence and exact localization of the neomycin phosphotransferase gene from transposon Tn5. Gene 19, 327–336 (1982).
[9] Leinala, E. K., Davies, P. L. & Jia, Z. Crystal structure of ??-Helical antifreeze protein points to a general ice binding model. Structure 10, 619–627 (2002).
[10] Cove, D. J. et al. The Moss Physcomitrella patens: A Novel Model System for Plant Development and Genomic Studies. Cold Spring Harbor Protocols (2009).
