<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Harvard_BioDesign_Platform skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Harvard BioDesign/Platform</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Prologue by HTML5 UP</TITLE></P><DIV id="header"><DIV class="inner"><DIV class="top"><NAV id="nav"><H4> Content </H4><UL><LI><A href="https://2015.igem.org/Team:Harvard_BioDesign/Platform#introduction" id="top-link" class="skel-layers-ignoreHref"><SPAN class="icon fa-home">Introduction</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:Harvard_BioDesign/Platform#strains" id="overview-link" class="skel-layers-ignoreHref"><SPAN class="icon fa-th">Strains</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:Harvard_BioDesign/Platform#constructs" id="sitemap-link" class="skel-layers-ignoreHref"><SPAN class="icon fa-user">Constructs</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:Harvard_BioDesign/Platform#validation" id="sitemap-link" class="skel-layers-ignoreHref"><SPAN class="icon fa-user">Validation</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:Harvard_BioDesign/Platform#standardization" id="sitemap-link" class="skel-layers-ignoreHref"><SPAN class="icon fa-user">Standardization</SPAN></A></LI></UL></NAV></DIV><DIV class="bottom"><NAV id="nav"><H4> Main Pages</H4><UL><LI><A href="https://2015.igem.org/Team:Harvard_BioDesign">Home</A></LI><LI><A href="https://2015.igem.org/Team:Harvard_BioDesign/Team">Team</A></LI><LI><A class="curPage" href="https://2015.igem.org/Team:Harvard_BioDesign/Project">Project</A></LI><LI><A href="https://2015.igem.org/Team:Harvard_BioDesign/Results">Results</A></LI><LI><A href="https://2015.igem.org/Team:Harvard_BioDesign/Notebook">Notebook</A></LI><LI><A href="https://2015.igem.org/Team:Harvard_BioDesign/Parts">Parts</A></LI><LI><A href="https://2015.igem.org/Team:Harvard_BioDesign/Collaborations">Collaborations</A></LI><LI><A href="https://2015.igem.org/Team:Harvard_BioDesign/Practices">Practices</A></LI><LI><A href="https://2015.igem.org/Team:Harvard_BioDesign/Safety">Safety</A></LI><LI><A href="https://2015.igem.org/Team:Harvard_BioDesign/Attributions">Attributions</A></LI></UL></NAV></DIV></DIV></DIV><DIV id="main"><SECTION id="introduction" class="one cover"><DIV class="container"><HEADER><H2>BactoGrip: Our Platform</H2></HEADER><P>
                This summer we dedicated ourselves to building a modular, durable, biomimetic platform for
                synthetic biologists who need to control bacterial adhesion. We developed a set of assays to
                reliably determine how our system is functioning and selected relevant positive and negative
                control strains to differentiate real from imaginary results. Using the tools of synthetic biology,
                we created <A href="https://2015.igem.org/Team:Harvard_BioDesign/Parts">BioBricks</A> which can be
                integrated into complex biological circuits where specific interaction with the physical environment is a must. Furthermore, we created a workflow
                that can be used to integrate any desired binding specificity into our system, and modified the native
                protein so that it is easy to work with and measure under diverse experimental conditions. We call our
                system BactoGrip, and we hope you will find it as gripping as we do.
              </P><FOOTER>How do you use BactoGrip, you ask?<DIV id="buttonbox"><A href="https://2015.igem.org/Team:Harvard_BioDesign/Platform#strains" style="font-family:'Lato';color:#9DE8BD;font-size:23px;"><H4 class="mid" style="color:#000;font-size:100%;">Read On</H4></A></DIV></FOOTER></DIV></SECTION><SECTION class="two"><DIV id="strains" class="container"><HEADER><H2>Strains and Assays</H2></HEADER><P>
                Before we could begin engineering the Fim system we needed to establish a positive and negative baseline for the experiments we'd use to categorize our constructs. We selected two strains from the Yale's Keio collection of nonessential gene knockouts which are each missing one of the recombinases that comprises the fim “switch”. We hypothesized based on the literature that a knockout for the fimB recombinase (Keio strain name JW4276) will not produce pili because fimE is biased to switch in the direction of “on” to “off”. When fimB is absent, fimE would switch the operon “off”. Contrarily, a knockout for fim E (Keio name- JW4276) will overproduce pili because fimB switches both “on” to “off” and “off” to “on”, so the operon will be “on” more than it would if both were present.
              </P><P>
                To confirm this hypothesis we used an agglutination assay from the literature which is standard for measuring the expression of Type 1 Pili.
              </P><P>
                This protocol tests whether a culture of E. coli cells can agglutinate--clump together--a substrate which contains the binding partner of fimH, mannose sugar. We chose to try to clump together S. cervisiae (baker's yeast) because it expresses mannose on its cell surface, is easy to grow in lab, and has been used in this assay before.

We found that the fimE KO we hypothesized would overproduce type 1 pili indeed agglutinated the yeast, as indicated by a white clump at the bottom of the tube. The fim B KO samples showed no sign of clumping and were completely opaque:
              </P><P>
                To confirm these results, we examined equal volumes of OD standardized agglutinated samples from either strain under light microscopy. Slides were prepared according to the slide preparation protocol and to visualize the bacteria and yeast, we stained the slides according to the Gram Stain protocol.
                
                These photos are representative of the morphology of each sample. The large purple circular cells are S. cerevisiae and the small pink cocci are E. coli:
              </P><P>
                At this point we were confident in the agglutination assay’s ability to detect the presence of pili via mannose-mediated adhesion. However, we hypothesized that our future modifications of the fimH adhesin might lead to weakened mannose binding strength even while the pili were still properly assembling.  We required a diagnostic assay that would detect whether type 1 Pili were present on the cell surface even when fimH no longer bound to mannose. So we conducted extensive literature review and developed the with the aim of a rapid test to provide a “yes” or “no” answer to this question.
              </P><P>
                We first tested our assay on our control strains, growing both the overproducer and null strain overnight in LB liquid culture, then OD standardizing to ensure our results were not dependent on variable cell growth. Then a volume of each sample was spun down at low speed (so as not to lyse the cells), washed once in DPBS to remove secreted proteins, and resuspended in a smaller volume of DPBS to concentrate the signal in later steps. These samples were then heat treated for 20 minutes at 60°C to shear-off the pili from the cell surface and spun down again at low-speed so the cells would pellet but the sheared pili would remain in solution. This pili-containing supernatant was then denatured using formic acid, boiled in 1x Lamelli’s buffer and run according to the SDS-PAGE protocol. The resulting gels were stained according to the R-250 Coomassie protocol  and compared at the molecular weight of Type 1 Pili’s structural subunit protein, fimA, which is most abundant. We hypothesized that the pili overproducing fimE would show a strong band at this weight compared to the null fimB strain. The resulting gel is below:
              </P><P>
                A band is present at the expected weight in the purified pili of the fimE overproducer. We tried testing the purification with and without formic acid acid denaturation. The fimA band is stronger in the formic acid treated sample which indicates that the pili are insoluble and need to be broken up for detection
              </P><P>
                Indeed, we see a dark band at 16.5 kD, the molecular weight of fimA, but only for the fimE overproducing strain, not for our negative control. This shows our pili purification method gives a clear signal to detect pili production, and can be used to characterize our modified Type 1 Pili and by future teams using our system. With functional assays and positive and negative controls, we felt confident that we had the tools necessary to begin engineering control over pili expression.
              </P></DIV></SECTION><SECTION class="three"><DIV id="constructs" class="container"><HEADER><H2>Construct Design</H2></HEADER><P>
                Our type I pili-expression system is distributed across two plasmids
                wherein the pili genes on each are expressed under different, inducible promoters.
                The fimH gene was amplified from the E. coli K12 genome and placed under the control
                 of a rhamnose-inducible (<A href="http://parts.igem.org/Part:BBa_K902065">Bba_K902065</A>) promoter with a strong ribosome binding site (<A href="http://parts.igem.org/Part:BBa_B0034">B0034</A>).
                 The remaining  type I pili structural and transport genes from the fim operon were amplified from
                 E. coli K12 and placed under an arabinose inducible (<A href="http://parts.igem.org/Part:BBa_I13453">BBa_I13453</A>) promoter with the native ribosomal
                 binding sites. Gibson assembly was used to place the parts into pSB1C3, illegal cut sites were
                 eliminated to make the parts RFC10 compatible, and each circuit was transferred into low copy number
                 expression plasmid (<A href="https://benchling.com/s/A9YgHe9r/edit"><I>fim operon</I> expression backbone, </A><A href="https://benchling.com/s/RdjOAYgP/edit">BBa_J61002 <I>fimH</I> expression backbone,</A>).
              </P><P>
                The system was arranged on two plasmids so that we could modulate the amount of adhesive <I>fimH</I> component with respect to the other pili structural genes. Additionally, we found that it was easier to customize <I>fimH</I> on a smaller plasmid backbone since it eliminated cloning issues encountered when the entire fim operon was located on a single plasmid. Finally, we selected pAra (<A href="http://parts.igem.org/Part:BBa_I13453">BBa_I13453</A>) because it is a titratable promoter that allows for controlled expression of pili structural genes over a broad range of inducer concentrations. Rhamnose and arabinose were added into broth cultures of <I>E.coli</I> containing both of the plasmids to create synthetic type I pili.
              </P><P>
                We introduced protein fusions to three discrete sites of within the FimH adhesin protein. These included the N-terminus immediately after the secretory peptide as well as after amino acids 225 and 258 from this site.
              </P><P>
                These selections were based on previous literature wherein small peptide fusions were introduced successfully at these sites. Sites 225 and 258 showed strong evidence in the literature that small fusions such as his-tags were expressed and functional on assembled pili.Conversely, prior work at the N-terminus has not shown that peptide fusions could be expressed and properly assembled at this site.,  Thus, we added novel peptide fusions to these sites in fimH using site-directed mutagenesis to customize the adhesive properties of type I pili. Additionally, we sought to determine whether these sites could be modified to make fimH generalizable to myriad localization and binding properties in the iGEM community.
              </P></DIV></SECTION><SECTION class="four"><DIV id="validation" class="container"><HEADER><H2>Validating Expression</H2></HEADER><P>We established that our engineered system could control pili expression by <A href="https://2015.igem.org/Team:Harvard_BioDesign/Platform#strains">performing an agglutination</A>. Following the protocol, we mixed cultures of our induced and uninduced plasmid-containing fimB knockouts with <I>S. cervisiae</I> yeast along with controls. We found that our plasmids recovered agglutination in the negative control strain and that this agglutination was dependent on the addition of inducer molecules. This shows both that we have control over expression of the biosynthetic machinery and the fimH adhesin, and that the resulting pili were functional as determined by a standard assay for mannose binding. This assay was performed in biological triplicates with the same result. Image below:</P></DIV></SECTION><SECTION class="five"><DIV id="standardization" class="container"><HEADER><H2>Making it Measureable</H2></HEADER><P>
                For our controlled adhesion system to be a useful platform for other iGEM teams and synthetic biologists, we needed to develop a method to reliably measure expression of our recombinant adhesin, fimH. To that end we inserted a His Tag into the 225 fusion site via site-directed mutagenesis and tested whether we could detect induction via an α-His Western Blot. Cultures of our His Tagged-fimH plasmid containing pili knockout bacteria were OD standardized and separated into two subcultures, one of which was induced with Rhamnose, the other uninduced. Samples taken at three timepoints show increasing expression of fimH in the induced culture but not the uninduced. This demonstrates that we can express recombinant fimH and that the His Tag gives us traction to measure expression.
              </P><P>
                In principle, the His Tag on fimH could also be used to quantify recombinant expression in assays like whole-cell ELISA, both integrated with the pilus shaft and separately. In the future we hope to test the efficacy of these assays for determining quantitative expression levels. Additionally, the His Tag will allow future teams to purify recombinant pili using Nickel NTA purification for use in vitro.
              </P></DIV></SECTION><DIV id="footer"><UL class="copyright"><LI>Site template designed by: <A href="http://html5up.net">HTML5 UP</A></LI></UL></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>