<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UC_Davis_Project skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UC Davis/Project</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TABLE width="100%" height="100%" cellspacing="0"><TBODY><TR height="50px"><TD width="162px" align="center" onmouseover="this.bgColor='#BDBDBD'" onmouseout="this.bgColor='#6A181B'" bgcolor="#6A181B"><A href="https://2015.igem.org/Team:UC_Davis/Practices" style="text-decoration:none;color:#FFFFFF">POLICY &amp; PRACTICES </A></TD><TD width="162px" align="center" onmouseover="this.bgColor='#BDBDBD'" onmouseout="this.bgColor='#6A181B'" bgcolor="#6A181B"><A href="https://2015.igem.org/Team:UC_Davis/Project" style="text-decoration:none;color:#FFFFFF">PROJECT</A></TD><TD width="162px" align="center" onmouseover="this.bgColor='#BDBDBD'" onmouseout="this.bgColor='#6A181B'" bgcolor="#6A181B"><A href="https://2015.igem.org/Team:UC_Davis/Parts" style="text-decoration:none;color:#FFFFFF">ACHIEVEMENTS </A></TD><TD width="162px" align="center" onmouseover="this.bgColor='#BDBDBD'" onmouseout="this.bgColor='#6A181B'" bgcolor="#6A181B"><A href="https://2015.igem.org/Team:UC_Davis/Attributions" style="text-decoration:none;color:#FFFFFF">ATTRIBUTIONS </A></TD><TD width="162px" align="center" onmouseover="this.bgColor='#BDBDBD'" onmouseout="this.bgColor='#6A181B'" bgcolor="#6A181B"><A href="https://2015.igem.org/Team:UC_Davis/Safety" style="text-decoration:none;color:#FFFFFF">SAFETY &amp; PROTOCOLS </A></TD></TR></TBODY></TABLE><TABLE id="CE1" width="100%" cellspacing="0" height="200px"><TBODY><TR><TD width="975px" align="top" bgcolor="#FFFFFF">


After identifying public awareness as an effective catalyst for change, we thought about how we might <I>raise</I> public awareness. Who would our target audience be? How would we communicate our message? We were particularly inspired by Greg Neimeyer’s Black Cloud initiative and identified high school students as viable candidates and a lesson plan as a feasible medium.

Our motivation for developing a lesson plan to be used in conjunction with our biosensor was two-fold:<UL><LI><B>to engage students in STEM</B> by demonstrating that biology and chemistry have real world applications</LI><LI>and to <B>raise awareness and accountability around chemical use</B> by enlisting the help of students to use our device to monitor environmental levels of triclosan.</LI></UL>
 
To aid us in our development of a lesson plan, we reached out to Community Resources for Science (CRS), an organization that works with scientists to <A href="http://www.crscience.org/volunteers/aboutbasis"> bring their work into the classroom</A> and engage students in hands on, inquiry based learning experiences.<B>Sasha Stackhouse</B> and <B>Morgan Seag</B> at CRS provided valuable feedback on how to present our project more clearly and how to fit our lesson plan to next generation science standards. Once we got the approval that our lesson plan met their standards, CRS connected us with their network of teachers in Davis.

While we are excited to share our project/message with schools in the area, delivering the instructions ourselves somewhat limits our scope. Ultimately we want to reach students around the nation. To better understand how we could refine our lesson plan to allow for wider distribution, we spoke with <B>Ann Moriarty</B>, an AP Biology/biotechnology teacher at Davis Senior High School.
 
Ms. Moriarty has a background in research herself and propounds the value of hands on activities in engaging students. In her classroom, for example, students develop an appreciation for biotechnology through running protein assays and performing gel electrophoresis to verify that their restriction enzymes cut the plasmid in the right location.
 
Ms. Morariaty provided valuable insight on the considerations that go into developing a lesson plan- the most salient being the constraints imposed by lack of funding and teaching to a curriculum.

The first obstacle is funding. The school budget is variable, teachers fill out a petition, and the process can take as long as a year. 

This piece of insight further emphasized our need to drive the cost of our assay down.
 
For example, in the first iteration of the lesson plan, the activity involved taking a field trip out to a local body of water where students would collect a sample for measurement. Ms. Moriarty raised some obstacles. For example, what happens to students who don't want to go on field trip? Who is going to cover the cost of a substitute teacher, transportation costs, insurance?  In sum, field trips are a logistical nightmare.
 
So we brainstormed a work-around. Why not have students go out and collect water samples from their local environment as a pre-activity assignment? Not only did this circumvent the difficulty of organizing a field trip, this also gave students a sense of ownership - a sense of individual responsibility that we had previously identified as an important factor in a successful civic engagement initiative.

The second obstacle is the constraints imposed by teaching to a curriculum. Ms. Moriarty mentioned the tight schedules associated with teaching to a curriculum.

However, the AP Biology curriculum is __ into four big ideas and that The College Board recommends two lab activities per big idea. We identified AP Biology standards that the lesson plan could fulfill, allowing our lesson plan to be more easily integrated into an AP Biology curriculum. </TD></TR><TR><TD width="100%" align="left" bgcolor="#FFFFFF">

Our goal for the wet lab portion of the project was to develop an inexpensive biosensor for triclosan to implement in a high school laboratory setting. The problem we wanted to approach was to find a molecule that could interact specifically with triclosan and whose interaction with triclosan could be coupled to a measurable readout. In trying to understand the biology of triclosan, we discovered that its natural target might fit these requirements:<UL><LI>It is an enzyme (FabI) that is directly inhibited by triclosan and</LI><LI>Whose activity also depends on a commonly measured co-factor, NADH</LI></UL>

In addition, our device requirements needed an enzyme that could also:<UL><LI>over-express well in E.coli,</LI><LI>show activity on its respective substrate at &lt;2nM level of enzyme,</LI><LI>was inhibited by triclosan at the &gt;2nM level of inhibitor, the higher level of TCS found in treated wastewater and toxic to algae [18]</LI></UL>

While the FabI class of enzymes are promising, it was unclear which of the thousands of enzymes we should use since all Bacteria and Eukaryotes have a <I>fabI</I> gene. In addition, the commercially available native substrate for FabI (crotonyl-CoA) is unstable and costly.  Therefore alternative substrates would be needed to be used in a practical setting. The wet lab portion of our project focused on:<UL><LI>Discovery of an enzyme that met all of our criteria,</LI><LI>Identification of an alternative substrate that would enable practical use, </LI></UL></TD></TR></TBODY></TABLE></P><P class="big"><FONT size="5" face="Avenir"> 
Our wet lab effort can be broken up into five components:<A href="#WL1"> 1. Enzyme Selection</A><A href="#WL2"> 2. Chemical Biology Substrate Screening </A><A href="#WL3"> 3. Enzyme Engineering </A><A href="#WL4"> 4. Prototyping in Real World Waste Water </A><A href="#WL5"> 5. Future Directions</A></FONT></P><A href="#menu" style="text-decoration:none" color="#FFFFFF" fillerwordsyeeeeeah=""><FONT color="#6A181B" font="" size="11">    ↥</FONT></A><TABLE id="WL1" width="100%" cellspacing="0" height="200px"><TBODY><TR><TD width="975px" align="top" bgcolor="#FFFFFF"><B>The need: A FabI enzyme that can show nanomolar inhibition using triclosan</B>
(Why nanomolar inhibition? Work done by Chalew and Halden showed that levels of triclosan leaving waste-water treatment plants was up to 9 nM, which happens to also be the toxicity threshold level for algae. [18])<B>Strategy 1a: Scan the registry to find characterized FabIs with inhibition data</B>

We first scanned through the iGEM parts registry for existing BioBrick Parts that code for FabI. We found <A ref="http://parts.igem.org/Part:BBa_K771303">Bba_K771303</A> from the 2012 Shanghai Jiao Tong University iGEM team, however we were unable to find enzymatic characterization data on the part. We therefore proceeded with our literature search for FabI enzymes with inhibition data.

<B>Strategy 1b: Alternate candidates were found by mining the literature to find characterized FabIs with inhibition data</B>

Triclosan inhibits type 2 fatty acid synthesis (FASII), an essential pathway in the Bacterial and Eukaryotic domains by interacting directly with the enoyl acyl carrier protein reductase (FabI) [3]. Evidence that we could use the enzyme to detect triclosan came from binding studies and crystallographic data initially from Heath et al. They showed that triclosan binding increases the enzyme’s affinity for NAD+ and triclosan’s role as an effective inhibitor is due to the formation of a stable ternary complex between FabI, triclosan, and NAD+ [4].

The basis of our biosensor, therefore, is to use triclosan’s mechanism of action as an inhibitor of enoyl acyl carrier protein reductase (FabI) in order to detect it. To screen a representative subset of FabI’s from all available domains of life, we mined the literature and found every characterized FabI with published inhibition data. We found all living organisms except for the Archaea, who synthesize lipids based on isoprenoids, have a fabI gene [5].

Unlike the other enzymes of the FASII pathway, it’s very important to note there is considerable diversity in the structure of FabI’s from different organisms[5]. AND, not all of them have the same level of sensitivity towards triclosan[23]. This is why we needed to screen a panel of enzymes in order to find the enzyme most sensitive towards triclosan with a nanomolar inhibition constant. (See above for why we wanted to see nanomolar inhibition)

Reported triclosan inhibition constants:<U>Terms:</U> Ki is the dissociation constant of the enzyme- inhibitor complex. IC50 is the amount of an inhibitor needed to inhibit a biological process by 50%

From our literature search, we were able to find enzymatic activity and triclosan inhibition data on the <I>E. coli</I> FabI. We also found 2 other FabI proteins that had at least nanomolar triclosan inhibition kinetics to screen: <I>S. aureus</I> and <I>P. falciparum</I>. Even though, <I>H. influenzae</I> FabI does not have reported nanomolar inhibition kinetics, we still wanted to verify the literature value so we included it in our set. We also acknowledge that <I>A. thaliana</I> has nanomolar triclosan inhibition, but we didn’t learn about its inhibition kinetics until we were near the end of our project		
			
Of the organisms studied, algae is most sensitive to triclosan, where toxicity levels are roughly 9 nanomolar which are the same concentrations reported by Chalew and Halden of triclosan leaving Wastewater Treatment Plants (WWTPs)[18]. From our literature search, we noticed that there were no characterized FabI’s from algae, nor is it even known whether triclosan exerts its effects on algae by inhibiting its FabI enzyme [19]. Due to their extreme sensitivity, we hypothesized that triclosan exerts its effects on algae through inhibition of the FabI enzyme. Therefore we decided to test 3 FabIs from algae: <I>P. tricornutum, T. pseudonana, and A. protothecoides</I>. 

This left us with a candidate list of 7 FabI proteins. 1 previously submitted FabI gene in the registry (<A ref="http://parts.igem.org/Part:BBa_K771303">Bba_K771303</A>), 3 FabI proteins taken from the literature, and 3 FabI proteins from algae. We decided to codon optimize and synthesize these genes to clone into a common <I>E. coli</I> expression vector pET29b+. A nucleotide alignment showing the codon optimizations to <A ref="http://parts.igem.org/Part:BBa_K771303">Bba_K771303</A> is shown below. <B>We screened each enzyme based on three criteria:</B><UL><LI>The enzymes had to over-express well in E. coli,</LI><LI>They had to have activity at the nanomolar level,</LI><LI>And they had to be inhibited by nanomolar levels of triclosan, the concentration leaving wastewater plants [18]</LI></UL><B><FONT size="“3”">1. Overexpression:</FONT></B>
Protein purity assessed through SDS-PAGE gels shown below:

Protein concentrations were measured spectrophotometrically from absorbance at 280 nm (A280). A280 readings converted enzyme concentration to mg/ml. The molarity of each enzyme was determined by dividing mg/ml by the enzyme’s extinction coefficient, theoretically derived from each enzyme’s amino acid sequence from http://web.expasy.org/protparam/

Extinction Coefficients used:<UL><LI>influenzae 17,420</LI><LI>falciparum 46,885</LI><LI>protothecoides 34,380</LI><LI>tricornutum 37,360</LI><LI>coli 15,930</LI></UL><I>B. pseudomallei and T. pseudonana</I> did not express under our conditions, so they were eliminated from our FabI team.<B><FONT size="“3”">2. Enzyme Activity Screening:</FONT></B>

Enzyme activity was measured spectrophotometrically through the decrease in NADH absorbance over time on the native substrate analog crotonyl CoA. Activity is defined as the change in optical density (absorbance) per minute. Activity is normalized by dividing activity by the microgram of enzyme used for the assay. Each enzyme was assayed with 100 uM NADH and 100 uM crotonyl-CoA. Negative control was 100 uM NADH, 100 uM crotonyl-CoA, no enzyme. Observed enzyme activities were subtracted from negative control and plotted on a log scale. Two biological replicates of each enzyme was used.<I>S. aureus</I> FabI showed no activity even though the enzyme has been previously characterized [11][12]. We later found out we didn’t see activity because <I>S. aureus</I> FabI is NADPH dependent rather than NADH dependent! Notwithstanding, NADPH is significantly more expensive than NADH[24], so instead of accommodating for <I>S. aureus</I> FabI, we decided to remove it from the FabI team.<B><FONT size="“3”">3. Triclosan Inhibition Screening:</FONT></B>
We were down to the &quot;Fab 5&quot;. Since Chalew et al showed the levels of triclosan leaving Waste Water Treatment Plants (WWTPs) was up to 9 nanomolar [18], so we wanted to measure enzyme inhibition using a nanomolar level of triclosan. Under our conditions, however, not all of the fab 5 had measurable activity with a nanomolar amount of enzyme, and in order to see inhibition using a nanomolar amount of triclosan we needed to use a nanomolar amount of enzyme.

Triclosan inhibition was measured by running our standard enzyme activity assay with no triclosan and 1 nM triclosan. Negative control was 100 uM NADH, 100 uM crotonyl CoA, no enzyme, no triclosan. Observed enzyme activities were subtracted from negative control activities. Percent inhibition was calculated by:

( (uninhibited activity - inhibited activity) / uninhibited activity ) * 100

The enzyme concentrations ranged from 1.9 - 3.3 nM using two biological replicates of each enzyme. Nanomolar inhibition from <I>P. falciparum</I> has been previously reported [8], but we are the first to show triclosan inhibition using <I>A. protothecoides</I> Fabi! Interestingly, with <I>P. tricornutum</I>, a marine diatom, which have been shown to be sensitive towards triclosan [29] we were unable to see triclosan inhibition, which suggests there is a different biological mechanism of action for triclosan inhibition. However, nanomolar inhibition was also not observed with <I>H. influenza</I>, and <I>E.coli</I>, even though previously reported [6][7][10]. Therefore more detailed studies on conditional dependencies of inhibition are needed in order to elucidate the mechanism of action. However, the <I>A. protothecoides</I> inhibition data clearly indicates that triclosan effects FabI from algae and that this could be the biological mechanism of toxicity.

For our biosensor, <I>P. falciparum</I> FabI, being the enzyme most inhibited by triclosan, appears to be the best enzyme to use for our biosensor!

Additionally, we further characterized the previously submitted <I>E. coli</I> FabI biobrick <A ref="http://parts.igem.org/Part:BBa_K771303">Bba_K771303</A>. We have added our characterization data to the experience section on the parts registry! 

<B>Sources:</B></TD></TR></TBODY></TABLE><A href="#menu" style="text-decoration:none" color="#FFFFFF" fillerwordsyeeeeeah=""><FONT color="#6A181B" font="" size="11">    ↥</FONT></A><TABLE id="WL2" width="100%" cellspacing="0" height="200px"><TBODY><TR><TD width="975px" align="top" bgcolor="#FFFFFF">

We calculated the cost to run the enzyme assay using the native substrate analog crotonyl-CoA, and calculated it costs 67 cents. A report published in 2002 by the American Association of Physics Teachers recommended that the budget for high school laboratories to be $1 per student per week[20]. Assuming this is the recommended budget for other laboratory courses, students would be unable to run our assay more than once in a given week. We calculated the cost of our enzyme assay: and found that 89% of the cost actually came from crotonyl CoA.  In addition to being expensive, Coenzyme A (CoA) is not very stable in solution. Sigma has reported that solutions stored at -20C are only stable for 2 weeks! [21]

In order to implement our device in a high school laboratory setting, we wanted our assay to be under 10 cents to run. This would allow a student to run our assay ~ 10 times. We couldn’t change the cost to produce the enzyme, nor its cofactor NADH, but it seemed feasible to try to find a cheaper substrate to use that did not involve CoA. Just like our enzyme screening process above, we needed to understand the chemistry behind how crotonyl CoA reacted in order to find cheaper substrates to use. We knew the reaction involved the reduction of the C2-C3 double bond. Rafferty et al first proposed the mechanism in which a hydride from NADH transferred to C3, which formed an enolate anion on the carbonyl oxygen. A proton transfer from tyrosine then leads to a keto-enol tautomerization [25][26]. In vivo, crotonyl is covalently bonded to acyl carrier protein, but coenzyme A is used as an analog. The purpose of these two molecules is to carry acyl chains through the cytoplasm (Acyl refers to CH3-C=O groups) [27].

We therefore designed a chemical biology screening based on two parameters: functional group similarity to the crotonyl moiety and to mimic CoA’s role as an acyl carrier. We wanted to explore a large chemical space to increase our chances of finding a hit. We weren’t completely sure if FabI only reduced carbon-carbon double bonds, so we tested valeraldehyde to see if FabI could reduce the aldehyde to an alcohol. To see if FabI could reduce the C-C double bond of an unsaturated carboxylic acid, we tested crotonic acid. We then tested three unsaturated aldehydes, just like crotonyl-CoA, but without the CoA moiety. And finally, to try and find potential acyl carriers, we tested bulky substrates with rings (phenyl acetaldehyde, p-anisaldehyde, and 3-(5-methyl-2-furyl)butanal).




We discovered enzyme activity on the three unsaturated aldehydes (trans-2-pentenal, 2-ethyl-2-butenal, and trans,trans-2,4-heptadienal), but had no activity on any of the other substrates! 

We found the enzymes were most active on trans-2-pentenal. There was no measurable enzyme activity using the other 5 substrates. This is highly consistent with the enzyme mechanism in which an allylic double bond is reduced when adjacent to an activating group, such as an electrophilic carbonyl (e.g. an aldehyde or thioester).  Furthermore, crotonic acid has a less electrophilic group adjacent to the allylic double bond, highlighting the high selectivity of FabI for the electronic structure of its substrates. 

While significant activity is observed on these alternative substrates, there was a decrease in overall enzyme activity relative to the near-native Crotonyl CoA substrate of ~100-fold. This made it so &gt;500nM enzyme is needed to see enzyme activity in assays.  In order to be used to detect relevant levels of TCS in wastewater the enzyme must have measurable activity at concentrations lower than the concentrations of triclosan in wastewater ( up to 9nM).  Therefore, we have begun to explore the use of enzyme engineering to enhance activity on trans-2-pentenal!


Continue scrolling to read more or <A href="#2.1">click here</A> to advance to the next section!</TD></TR></TBODY></TABLE><A href="#menu" style="text-decoration:none" color="#FFFFFF" fillerwordsyeeeeeah=""><FONT color="#6A181B" font="" size="11">    ↥</FONT></A><TABLE id="WL3" width="100%" cellspacing="0" height="200px"><TBODY><TR><TD width="975px" align="top" bgcolor="#FFFFFF">    

Fortunately a crystal structure for the P. falciparum FabI enzyme had already been determined[28]. We used the computational tool Foldit (See <A href="“https://2011.igem.org/Team:Washington”">1</A>, <AHREF>2, <AHREF>3 )
to design 28 mutants. We hypothesized the trans-2-pentenal would occupy a highly similar structural space as triclosan. Analysis of triclosan bound to NADH in the active site revealed that the phenyl ring of triclosan lined up face to face with the ring of NADH forming a pi-stacking interaction. We hypothesized that if we could increase the pi-stacking interaction by mutating residues around the triclosan-NADH site to be aromatic residues, we might be able to increase the enzyme’s activity on trans-2-pentenal. 



Each mutant was generated using kunkel mutagenesis through the <A href="https://www.transcriptic.com/">transcriptic cloud laboratory</A>. The sequence verified mutant genes were cloned into our expression strain of E. coli and protein produced and purified as described in our Notebook [LINK]. From this initial round of 28 mutants, 23 expressed as soluble protein. Of the solubly expressed designs 4 of the designs had no effect on function, and 19 decreased activity.   However, one of the enzymes resulted in 1.5x increase in activity against the non-natural trans-2-pentenal substrate.  



We are currently exploring new mutants based on the data generated from this first round of screens. While the enzyme activity needs to be improved ~100-fold in order to achieve levels of activity observed on the native substrate, this is well within reach of enzyme engineering efforts based on previous successes [30][31][32]

<B>Sources:</B></AHREF></AHREF></TD></TR></TBODY></TABLE><A href="#menu" style="text-decoration:none" color="#FFFFFF" fillerwordsyeeeeeah=""><FONT color="#6A181B" font="" size="11">    ↥</FONT></A><TABLE id="H" width="100%" cellspacing="0" height="200px"><TBODY><TR><TD width="975px" align="top" bgcolor="#FFFFFF">
In the lab we used an EPOCH spectrophotometer to run our assays. At a price tag of ~$10,000 this device is definitely out of the budget range of a high school teacher, so we looked for a more appropriately priced alternative. 

We found a <A href="http://www.iorodeo.com/content/educational-colorimeter-kit">colorimeter device</A> from IO Rodeo, a company that develops open source hardware and software for educational purposes. They sold a spectrophotometer which we hypothesized would work for our assay for $80. One reason for the price difference is the EPOCH is a monochronometer covers a spectrum from 200 nm - 999 nm, selectable in 1 nm increments.  The IO Rodeo is based on an 365nm LED and requires hardware changes to adjust the wavelength of emission and detection.  Our enzyme assay is based on oxidation of the enzyme cofactor NADH into NAD.  The standard wavelength for detection of this reaction is 340nm, however the NADH has a broad spectrum (see figure below).  Based on the differences in NAD and NADH spectral signals we hypothesized that the 365nm LED should provide a sufficient signal to detect the NADH to NAD conversion.

We purchased the IO Rodeo spectrophotometer and compared NADH sensitivity on the IO Rodeo spectrophotometer and EPOCH spectrophotometer.<B><U>Test #1: NADH Sensitivities</U></B>
We first compared the linear ranges of the devices and found that there was a linear relationship between NADH concentration and absorbance between 6 micro molar and 400 micro molar for both devices. This means that both instruments had the required sensitivity for our assay under ideal conditions:<B>Protocol for colorimeter test #1:</B>
All test solutions were prepared from a single freshly made 0.5 mM NADH stock solution. Each test solution was measured in triplicate on both the IO Rodeo colorimeter and the EPOCH spectrophotometer. <B><U>Test #2:  FabI Inhibition Assays (i.e. functional prototype)</U></B>
In order to illustrate the utility of a device that meets our design requirements. We then tested to see if the IO Rodeo device could be used for an inhibition assay.  As illustrated in Figure below the NADH oxidation rate is significantly lower in the presence of nM triclosan levels than in the absence of triclosan.

The IO rodeo portable spectrophotometer is also able to detect various levels of triclosan inhibition...right out of the box!

This assay used triplicates of 2 nM <I>P. falciparum</I> FabI. 100 uM crotonyl-CoA, 100 uM NADH

We are now working in refining the assay to measure and improve robustness (accuracy when tested by multiple users), specificity (substrate and alternative inhibitors), and sensitivity (more detailed inhibition curves and optimization of conditions).  We will also be working with high school students to see if the assay can be used successfully in high schools.






</TD></TR></TBODY></TABLE><A href="#menu" style="text-decoration:none" color="#FFFFFF" fillerwordsyeeeeeah=""><FONT color="#6A181B" font="" size="11">    ↥</FONT></A><TABLE id="WL4" width="100%" cellspacing="0" height="200px"><TBODY><TR><TD width="975px" align="top" bgcolor="#FFFFFF">


Can we detect enzyme inhibition in waste water?

In order for our show we had a functional prototype, we needed to show enzyme inhibition in waste water. We performed this experiment using triplicates of 15 nM P. falciparum FabI. It appears as if life is a bit slower in waste water…

This shows that our biosensor works in waste water!</TD></TR></TBODY></TABLE><A href="#menu" style="text-decoration:none" color="#FFFFFF" fillerwordsyeeeeeah=""><FONT color="#6A181B" font="" size="11">    ↥</FONT></A><TABLE id="WL5" width="100%" cellspacing="0" height="200px"><TBODY><TR><TD width="975px" align="top" bgcolor="#FFFFFF"><UL><LI>Confirm inhibition data in waste water correlates to known levels of triclosan in a wide variety of waste water samples</LI><LI>Continue rounds of enzyme engineering to enhance another 60-fold (~10 more 1.5 folds… or 1 60-fold)</LI><LI>Forward predictions based on unknown samples where the biosensor, ELISA, and MS are used in parallel</LI></UL><B>Sources:</B>[1] J, Regös, Zak O, Solf R, Vischer WA, and Weirich EG. &quot;Antimicrobial Spectrum of Triclosan, a Broad-spectrum Antimicrobial Agent for Topical Application. II. Comparison with Some Other Antimicrobial Agents.&quot; National Center for Biotechnology Information. U.S. National Library of Medicine, 1979.       
           
[2] Kini, Suvarna, Anilchandra R. Bhat, Byron Bryant, John S. Williamson, and Franck E. Dayan. &quot;Synthesis, Antitubercular Activity and Docking Study of Novel Cyclic Azole Substituted Diphenyl Ether Derivatives.&quot; EUROPEAN JOURNAL OF MEDICINAL CHEMISTRY. N.p., May 2008. 
  
[3] McMurry, Laura M., Margret Oethinger, and Stuart B. Levy. &quot;Triclosan Targets Lipid Synthesis.&quot; Nature 394 (1998): 531-32. 
  
[4] Heath, R. J. , Yu, Y.-T. , Shapiro, M. A. , Olson, E. &amp; Rock, C. O. J. Biol. Chem. 273, 30316–30320 (1998)  
[5] RP, Massengo-Tiassé, and Cronan JE. &quot;Diversity in Enoyl-acyl Carrier Protein Reductases.&quot; Cell Mol Life Sci. (May 2009)
[6] RJ, Heath, Rubin JR, Holland DR, Zhang E, Snow ME, and Rock CO. &quot;Mechanism of Triclosan Inhibition of Bacterial Fatty Acid Synthesis.&quot; J Biol Chem (April 1999)
[7] Ward, Walter. &quot;Kinetic and Structural Characteristics of the Inhibition of Enoyl (acyl Carrier Protein) Reductase by Triclosan.&quot; Biochemistry (1999 Sep 21)
[8] Kapoor, Mili. &quot;Slow-tight-binding Inhibition of Enoyl-acyl Carrier Protein Reductase from Plasmodium Falciparum by Triclosan.&quot; Biochem (2004 August 1)
[9] Surolia, Namita, and Avadhesha Surolia. &quot;Triclosan Offers Protection against Blood Stages of Malaria by Inhibiting Enoyl-ACP Reductase of Plasmodium Falciparum.&quot; Nature Medicine (2001)
[10] Marcinkeviciene, J.et al, (2001). &quot;Enoyl-ACP Reductase (FabI) of Haemophilus influenzae: Steady-State Kinetic Mechanism and Inhibition by Triclosan and Hexachlorophene.&quot; Archives of Biochemistry and Biophysics 390(1): 101-108.
[11]  Courtney Slater-Radosti, Glenn Van Aller, Rebecca Greenwood, Richard Nicholas, Paul M. Keller, Walter E. DeWolf, Jr, Frank Fan, David J. Payne, and Deborah D. Jaworski Biochemical and genetic characterization of the action of triclosan on Staphylococcus aureus J. Antimicrob. Chemother. (2001) 48 (1): 1-6. doi: 10.1093/jac/48.1.1
[12] Mechanism and Inhibition of saFabI, the Enoyl Reductase from Staphylococcus aureus Hua Xu, Todd J. Sullivan, Jun-ichiro Sekiguchi, Teruo Kirikae, Iwao Ojima, Christopher F. Stratton, Weimin Mao, Fernando L. Rock, M. R. K. Alley, Francis Johnson, Stephen G. Walker and Peter J. Tonge Institute for Chemical Biology &amp; Drug Discovery, Department of Chemistry, Stony Brook University, Stony Brook, New York 11794-3400, School of Dental Medicine, Stony Brook University, Stony Brook, New York 11794, Department of Infectious Diseases, International Medical Center of Japan, Tokyo 162-8655, Japan, and Discovery Biology, Anacor Pharmaceuticals Inc., Palo Alto, California 94303
[13] Hoang TT, Schweizer HP. 1999. Characterization of Pseudomonas aeruginosaenoyl-acyl carrier protein reductase (FabI): a target for the antimicrobial triclosan and its role in acylated homoserine lactone synthesis. J. Bacteriol.181:5489–5497.  
[14] Parikh, S. L., Xiao, G. and Tonge, P. J. (2000) ‘Inhibition of InhA, the enoyl reductase from Mycobacterium tuberculosis, by triclosan and isoniazid’,Biochemistry, Vol. 39, No. 26, pp.7645-7650.
[15] Massengo-Tiassé, R. P., and J. E. Cronan. 2009.  Diversity  in  enoyl-acyl carrier protein reductases. Cell. Mol. Life Sci.66:1507–1517.
[16] Dayan FE, Ferreira D, Wang YH, Khan IA, McInroy JA, Pan Z
(2008) A pathogenic fungi diphenyl ether phytotoxin targets plant enoyl (acyl carrier protein) reductase. Plant Physiol 147: 1062–1071
[17] Liu N, Cummings JE, England K, Slayden RA, Tonge PJ. 2011. Mechanism and inhibition of the FabI enoyl-ACP reductase from Burkholderia pseudomallei. J. Antimicrob. Chemother. 66:564–573. 10.1093/jac/dkq509
[18] Chalew T. E., Halden R. U. (2009). Environmental exposure of aquatic and terrestrial biota to triclosan and triclocarban. J. Am. Water Works Assoc. 45, 4–13. 10.1111/j.1752-1688.2008.00284.x                
[19] Eriksson M, Johansson H, Fihlman V, Grehn A, Sanli K, Andersson MX, Blanck H, Arrhenius Å, Sircar T, Backhaus T. (2014) Long-term effects of the antibacterial agent triclosan on marine periphyton communities. PeerJ PrePrints 2:e489v1
[20] &quot;Guidelines for High School Physics Programs.&quot; HS Guidelines. 
[21] Corp., Sigma-Aldrich. Acetyl Coenzyme A Trilithium Salt (A2181) - Product Information Sheet (n.d.): n. pag. Sigma. 
[22] Vick JE, Clomburg JM, Blankschien MD, Chou A, Kim S, Gonzalez R.Escherichia coli enoyl-acyl carrier protein reductase (FabI) supports efficient operation of a functional reversal of β-oxidation cycle. Vol. 269, No. 8,Issue of February 25, pp. 5493-5496, 1994 The Journal of Biological Chemistry, 269, 5493-5496.
[23]. Pidugu, L. S., M. Kapoor, N. Surolia, A. Surolia and K. Suguna (2004). &quot;Structural basis for the variation in triclosan affinity to enoyl reductases.&quot; J Mol Biol 343(1): 147-155.
[24] Links to purchase NADPH and NADH: https://www.fishersci.com/shop/products/nadph-tetrasodium-salt-hydrate-96-extra-pure-acros-organics-2/p-171261, https://www.fishersci.com/shop/products/beta-nicotinamide-adenine-dinucleotide-disod-salt-hydrate-95-reduced-form-acros-organics-4/p-3737061
[25] White, S. W., J. Zheng, Y. M. Zhang and Rock (2005). &quot;The structural biology of type II fatty acid biosynthesis.&quot; Annu Rev Biochem 74: 791-831.
[26] Rafferty, J. B., J. W. Simon, C. Baldock, P. J. Artymiuk, P. J. Baker, A. R. Stuitje, A. R. Slabas and D. W. Rice (1995). &quot;Common themes in redox chemistry emerge from the X-ray structure of oilseed rape (Brassica napus) enoyl acyl carrier protein reductase.&quot; Structure 3(9): 927-938.
[27] Elovson, J. and P. R. Vagelos (1968). &quot;Acyl Carrier Protein: X. ACYL CARRIER PROTEIN SYNTHETASE.&quot; Journal of Biological Chemistry 243(13): 3603-3611.
[28] Perozzo, R., M. Kuo, A. Sidhu, J. T. Valiyaveettil, R. Bittman, W. R. Jacobs, Jr., D. A. Fidock and J. C. Sacchettini (2002). &quot;Structural elucidation of the specificity of the antibacterial agent triclosan for malarial enoyl acyl carrier protein reductase.&quot; J Biol Chem 277(15): 13106-13114.               
[29] Johansson, C. H., L. Janmar and T. Backhaus (2014). &quot;Triclosan causes toxic effects to algae in marine biofilms, but does not inhibit the metabolic activity of marine biofilm bacteria.&quot; Mar Pollut Bull 84(1-2): 208-212.
[30] Savile, C. K., J. M. Janey, E. C. Mundorff, J. C. Moore, S. Tam, W. R. Jarvis, J. C. Colbeck, A. Krebber, F. J. Fleitz, J. Brands, P. N. Devine, G. W. Huisman, and G. J. Hughes. &quot;Biocatalytic Asymmetric Synthesis of Chiral Amines from Ketones Applied to Sitagliptin Manufacture.&quot; Science 329.5989 (2010): 305-09. 
[31] J. B. Siegel et al., Science 329, 309 (2010)
[32] Bornscheuer, U. T., G. W. Huisman, R. J. Kazlauskas, S. Lutz, J. C. Moore and K. Robins (2012). &quot;Engineering the third wave of biocatalysis.&quot; Nature 485(7397): 185-194.</TD></TR></TBODY></TABLE><A href="#menu" style="text-decoration:none" color="#FFFFFF" fillerwordsyeeeeeah=""><FONT color="#6A181B" font="" size="11">    ↥</FONT></A><TABLE id="CE2" width="100%" cellspacing="0" height="200px"><TBODY><TR><TD width="975px" align="top" bgcolor="#FFFFFF"><FONT size="5px"><B>Design:</B></FONT>
When we first started our project, we took a trip to our local Safeway to catalog products containing triclosan. We discovered that many products had already phased out triclosan; some labels even read “Triclosan Free.”  Although triclosan had been removed from products, many of them had simply replaced it with a different antimicrobial. 

This trend reminded us of what Arlene Blum told us about how when chemicals are removed from use manufacturers look for a replacement; but because these chemicals need to serve similar functions they often have similar structures, and thus similar consequences. What results is a cycle whereby one toxic chemical is replaced by another toxic chemical.
  
We didn’t want to raise fear over triclosan use and contribute to this cycle. Instead we wanted to raise awareness around appropriate chemical use and reduce the use of chemicals in cases where there is no proven benefit.
 
This lead us to supplementing our triclosan biosensor with an, “antimicrobial footprint app,” to get consumers thinking about whether antimicrobial agents are even warranted in consumer products. <FONT size="5px"><B>Deliverable:</B></FONT>

We designed our app as a heuristic to raise awareness about the unnecessary ubiquity of antimicrobials in consumer products.  In the app, the user can click on an “About” tab to learn more about antimicrobials and how to be a responsible consumer.  They can then go on to calculate their “Antimicrobial Footprint.” The user is able to click on antimicrobial containing products that they use, and see how it affects their total footprint. After using the app’s antimicrobial calculator to calculate their footprint, the user can submit their footprint along with their location. On the final page of the app the user is able to see how their footprint compares to the average footprint of other users. The submitted data is used to calculate this average, as well as to create a heat map of antimicrobial usage in the United States. This is another deliverable that users can look at to become more educated consumers.  <FONT color="white"> space space space space</FONT><FONT size="5px"><B>How it Works:</B></FONT>

To create the antimicrobial calculator we found data on the levels of triclosan in selected consumer products, given in g triclosan/g products.  
We also found data on the daily use rates of consumer products, given in g triclosan/day. By combining this information we were able to calculate the users’ “antimicrobial footprint,” in grams triclosan/day. The app will also give you this metric in grams triclosan/year. <FONT color="white"> space space space </FONT><B>Sources:</B>
Rodricks, Joseph V. &quot;Triclosan: A Critical Review of the Experimental Data and Development of Margins of Safety for Consumer Products.&quot; Critical Reviews in Toxicology, 2010. Web.
</TD></TR></TBODY></TABLE><A href="#menu" style="text-decoration:none" color="#FFFFFF" fillerwordsyeeeeeah=""><FONT color="#6A181B" font="" size="11">    ↥</FONT></A></DIV></DIV></DIV></DIV></BODY></HTML>