"
Page
Discussion
View source
History
teams
Log in
 
Team:Hong Kong CUHK/pah
From 2013.igem.org
iGEM CUHK
Team Members | Instructors and Advisors
Overview | PAH Degradtion | Voltage Switch
Parts | Construction Notes | Characterizations
Protocols | Notebooks
PAH Degradation
T7-RBS-QsrR (Device BBa_K1092001 )
Quinone  sensing and response repressor (QsrR) is a novel kind of thiol-stress-sensing  regulator YodB family transcriptional regulator found in the notorious pathogen Staphylococcus aureus. [2] It specifically  binds to a palindromic DNA sequence, blocking RNA polymerase and thus  repressing transcription. [2] However, with the presence of quinone, it  reshapes to bind with quinone molecules and leaves the target DNA, which allows  the transcription process to start. [2] Since laccase is used at first to  degrade PAHs into quinones for the proposed pathway, the quinone-like compounds  produced as intermediates can function as signaling molecules that induce the  expression of other enzymes responsible for subsequent degradations. This  method can save energy for the cell that expresses those PAH-degrading enzymes.  The protein sequence of QsrR is readily available at GenBank, with the  reference code 4HQE_A for side chain A, and 4HQE_B for side chain B.
QsrR  Binding region:
5’ – GTATAN{5}TATAC – 3’ [2]
The  redox-sensing regulator YodB family utilizes disulfide bond formation between  cystine residuals for the transformation of repressor protein. [1][2] The  presence of these repressors in some pathogen bacteria may relate to bacterial  defense of quinone-like immune molecules. [2]
References:
[1] CHI, B.K.,  ALBRECHT, D., GRONAU, K., BECHER, D., HECKER, M. and ANTELMANN, H., 2010. The  redox-sensing regulator YodB senses quinones and diamide via a thiol-disulfide  switch in Bacillus subtilis. Proteomics, 10(17), pp. 3155-3164.
[2] JI, Q.,  ZHANG, L., JONES, M.B., SUN, F., DENG, X., LIANG, H., CHO, H., BRUGAROLAS, P.,  GAO, Y.N., PETERSON, S.N., LAN, L., BAE, T. and HE, C., 2013. Molecular  mechanism of quinone signaling mediated through S-quinonization of a YodB  family repressor QsrR. Proceedings of the National Academy of Sciences,  110(13), pp. 5010-5015.
 
T7 - RBS – Dioxygenase (Device BBa_K1092003 )
Catechol 1,2- dioxygenase (EC 1.13.11.1) from Pseudomonas putida KT2400 is an oxidative enzyme which induces  ortho-ring cleavage of catechol, a phenolic compound [1, 5]. The gene sequence  is available in GenBank (1045930).
 
T7-RBS-Laccase (Device BBa_K1092005 )
The laccase (cotA) found in Bacillus sp. HR03 is a bacterial laccase  that have the potential ability to degrade a variety of aromatic compounds. [3]  Previous study showed that it can be successfully expressed in Escherichia coli strain BL-21(DE3), and  was correctly folded. [4] Moreover, the complete gene sequence of this laccase  was readily recorded in GenBank, with the reference code of FJ663050.1. It is  thus used as one of the two major enzymes for PAH degradation.
Properties of the laccase:
Optimal temperature: ~70°C;
Optimal pH: ~7. [3]
Laccases (benzenediol oxygen  oxidoreductase; EC 1.10.3.2) are a type of oxidative enzymes that show the  ability of oxidizing a variety of compounds ranging from phenolic compounds to  polycyclic aromatic hydrocarbons (PAHs). [1][2][3] They can be found both in  bacteria and fungi, especially white-rot fungi that have the ability to degrade  lignin, an important structural component of plants. [1][5] The experiment of  Hadibarata et al. suggested that laccase could play a very significant role in  benzo[a]pyrene degradation, a reaction that hardly occurs. [1] And the previous  research done by Hadibarata et al. and Zeng et al. suggested that laccase  degraded PAHs by adding oxygen atoms into the ring, forming quinone-like  intermediates. [1][6] After such transformation, hydrophilic quinones can be  further converted into other metabolites by other enzymes.
References:
[1]  HADIBARATA, T. and KRISTANTI, R.A., 2012. Identification of metabolites from  benzo[a]pyrene oxidation by ligninolytic enzymes of Polyporus sp. S133. Journal of environmental management, 111(0),  pp. 115-119.
[2]  HADIBARATA, T., TEH, Z., RUBIYATNO, ZUBIR, M., KHUDHAIR, A., YUSOFF, A., SALIM,  M. and HIDAYAT, T., 2013. Identification of naphthalene metabolism by white rot  fungus Pleurotus eryngii. Bioprocess  and Biosystems Engineering, 36(10), pp. 1455-1461.
[3]  MOHAMMADIAN, M., FATHI-ROUDSARI, M., MOLLANIA, N., BADOEI-DALFARD, A. and  KHAJEH, K., 2010. Enhanced expression of a recombinant bacterial laccase at low  temperature and microaerobic conditions: purification and biochemical  characterization. Journal of industrial microbiology & biotechnology,  37(8), pp. 863-869.
[4]  MOLLANIA, N., KHAJEH, K., RANJBAR, B., RASHNO, F., AKBARI, N. and  FATHI-ROUDSARI, M., 2013. An efficient in vitro refolding of recombinant  bacterial laccase in Escherichia coli.  Enzyme and microbial technology, 52(6–7), pp. 325-330.
[5]  SHARMA, P., GOEL, R. and CAPALASH, N., 2007. Bacterial laccases. World Journal  of Microbiology and Biotechnology, 23(6), pp. 823-832.
[6]  ZENG, J., LIN, X., ZHANG, J., ZHU, H., CHEN, H. and WONG, M., 2013. Successive  transformation of benzo[a]pyrene by laccase of Trametes versicolor and pyrene-degrading Mycobacterium strains. Applied Microbiology and Biotechnology,  97(7), pp. 3183-3194.
 
T7 - QsrR Binding Site - RBS – Dioxygenase (Device BBa_K1092014 )
This biobrick is similar to BBa_K1092003: T7 - RBS - Dioxygenase, with an additional QsrR binding site that allows QsrR regulation.
Address: Rm. 184, Science Centre, CUHK
Email: kingchan@cuhk.edu.hk  Tel: (852)-39434420  Fax: (852)-26037246
TOP
Retrieved from " http://2013.igem.org/Team:Hong_Kong_CUHK/pah "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
