<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Biwako_Nagahama_Project"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Biwako_Nagahama/Project">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Biwako_Nagahama/Project&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Biwako_Nagahama/Project&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Biwako_Nagahama/Project&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Biwako_Nagahama/Project" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Biwako Nagahama/Project</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><DIV id="biwako-menubar"><UL><LI><A href="https://2013.igem.org/Team:Biwako_Nagahama/Contact">Contact</A></LI><LI><A href="https://2013.igem.org/Team:Biwako_Nagahama/Human Practice">Human Practice</A></LI><LI><A href="https://2013.igem.org/Team:Biwako_Nagahama/Safety">Safety</A></LI><LI><A href="https://2013.igem.org/Team:Biwako_Nagahama/Original 3A Assembly">Original 3A Assembly</A></LI><LI><A href="https://2013.igem.org/Team:Biwako_Nagahama/Parts">Parts</A></LI><LI><A href="https://2013.igem.org/Team:Biwako_Nagahama/Material &amp; Method">Material &amp; Method</A></LI><LI><A href="https://2013.igem.org/Team:Biwako_Nagahama/Project">Project</A></LI><LI><A href="https://2013.igem.org/Team:Biwako_Nagahama/Team">Team</A></LI><LI><A href="https://2013.igem.org/Team:Biwako_Nagahama">Start</A></LI><UL><DIV class="main-contents"><DIV class="top-wrapper"><DIV class="top-sentence"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#AgRePaper.26E.coli-ink"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">AgRePaper&amp;E.coli-ink</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#AgRePaper"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">AgRePaper</SPAN></A></LI><LI class="toclevel-3 tocsection-3"><A href="#Background"><SPAN class="tocnumber">1.1.1</SPAN><SPAN class="toctext">Background</SPAN></A></LI></UL><LI class="toclevel-2"><A href="#.E3.83.BBCellulose_synthesis_operon"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">・Cellulose synthesis operon</SPAN></A></LI><LI class="toclevel-2"><A href="#.E3.83.BBCurdlan_synthesis_enzyme"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">・Curdlan synthesis enzyme</SPAN></A></LI><LI class="toclevel-2"><A href="#.E3.83.BBCelC"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">・CelC</SPAN></A></LI><LI class="toclevel-3 tocsection-4"><A href="#Secretion_of_fibrils_from_Agro"><SPAN class="tocnumber">1.4.1</SPAN><SPAN class="toctext">Secretion of fibrils from Agro</SPAN></A></LI><LI class="toclevel-3 tocsection-5"><A href="#Curdlan.27s_introduction"><SPAN class="tocnumber">1.4.2</SPAN><SPAN class="toctext">Curdlan's introduction</SPAN></A></LI><LI class="toclevel-3 tocsection-6"><A href="#Binary_Vector"><SPAN class="tocnumber">1.4.3</SPAN><SPAN class="toctext">Binary Vector</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#E.coli-ink"><SPAN class="tocnumber">1.5</SPAN><SPAN class="toctext">E.coli-ink</SPAN></A></LI></TD></TR></TBODY></TABLE><H2>AgRePaper&amp;E.coli-ink<P>Cellulose is used as raw material for paper, so our team experimented various ways to increase the amount of cellulose produced by agrobacterium and using it to make papers. For this we developed the different parts to insert into the system of agrobacterium. Among them were the genes used for expression of the curdlan. Similarly, genetic parts in order to increase the expression of the cellulose, along with the agrobacterium type binary vector were also developed . We also worked on recycling of the produced paper by degrading the cellulose to D-Glucose using various enzymes. We worked for the preparation of the biological ink using the sperm whale's cells by genetically modification to increase amount of myoglobin. Then, we observed the change on the color of the product by altering the formation of myoglobin and the production amount of myoglobin with the insertion of T7 promoter to the cell system.
</P></H2><H2>AgRePaper</H2><H5>Background<P>Today, Our Earth is confronted with global warming and depletion of ozone layer, extinction of organism along with energy crisis as few of many environmental issues.</P><P>We think deforestation as its major cause. Many great forests have fallen for production of paper, which we usually use in lots of day to day life for writing and Printing papers,newspaper, origami, wrappers etc…  .</P><P>To solve this problem,we tried to make paper from resources except wood and raw materials that do not affect environment. So, we can save forests for future.</P><P>How to solve it?</P><P>First, I need to study Paper.</P><P>Cellulose is the main ingredients of plant fiber whereas Paper is made by Cellulose and Hemicellulose, Lignin. Cellulose functions as scaffold, Hemicellulose functions as adhesion, Lignin functions as Space-filling.</P><P>Cellulose and Hemicellulose can be produced from synthetic biology.  So, the issue regarding paper can surely be solved.</P><P>Team Biwako Nagahama try to solve this issue for Agrobacterium tumefaciens C58.</P><P>Non-recombinant Agrobacterium tumefaciens synthesize cellulose and other glycan. Publication of Synthesized biofilm with Agrobacterium submitted.  In this view, Agrobacterium synthesize cellulose. I suspect it's so.</P><P>But, Agrobacterium doesn’t make hemicellulose. So, Hemicellulose like Curdlan(beta-1,3-glycan) pick one out of synthesized glycan from Agrobacterium.</P><P>And, Agrobacterium have Cellulose synthase enzyme, including catabolic enzyme. 
We try to reuse glycan by this enzyme.</P><P>In this project, we try to make Cellulose and Curdlan with Agrobacterium. This Cellulose and Curdlan use to make Paper. And, this Paper reuse by catabolic enzyme. So, we approach to an environmental issue!!</P></H5><H2><SPAN class="mw-headline" id=".E3.83.BBCellulose_synthesis_operon">・Cellulose synthesis operon</SPAN></H2><P>Cellulose production requires genes encoded within the CelABCG and CelDE operons on the C58 linear chromosome.

CelA is a membrane-associated cellulose synthase (CS) enzyme, utilizing the precursor UDP-glucose.Homologues of CelB bind cyclic diguanosine monophosphate(c-di-GMP),an allosteric regulator of CS activity,and physically interact with CS in the membrane.CelC is a secreted protein similar to endoglucanases.Regulation of cel gene expression is not well understood.

The CS activity of A.tumefaciens is allosterically regulated by the intracellular signal molecule c-di-GMP, which strongly stimulates cellulose synthesis in cell extracts. 
</P><H2><SPAN class="mw-headline" id=".E3.83.BBCurdlan_synthesis_enzyme">・Curdlan synthesis enzyme</SPAN></H2><P>Curdlan synthesis enzyme requires genes encoded within the CrdS on the C58 linear chromosome.</P><P>Providing genetic confirmation that crdS is essential for curdlan production.</P><P>CrdS was shown to be an integral membrane protein with seven transmembrane helices.</P><H2><SPAN class="mw-headline" id=".E3.83.BBCelC">・CelC</SPAN></H2><P>Cellulase requires genes encoded within the CelC on the C58 linear chromosome.</P><P>CelC belong Glycoside Hydrolase 8(GH8). CelC work as Endo-glucanase gene.CelC is enzyme that catabolized Cellulose to Cellobiose.</P><H2>Secretion of fibrils from Agro<P>For this year’s project our team aimed at making the paper by using Agro bacterium.
So, we confirmed whether carbohydrate developed from Agrobacterium could be used as the raw material for making paper.</P><P><B>Materials</B></P><P>Agro bacterium tumefaciens C58　</P><P>Agro bacterium tumefaciens C58　bacterial liquid</P><P>   　　　　＋LB medium　　　　   　  2 ml</P><P>   　　　　＋ampicillin　　　　　　         　2 µl</P><P>   　　　　＋Agro bacterium tumefaciens C58</P><P>Congo red</P><P>LB medium</P><P>　　　　＋1.0％(w/v)　 Triptone　　　　 2 g</P><P>　　　　＋0.5％(w/v)　 yeast extract　   1 g</P><P>　　　　＋1.0％(w/v)　　NaCl　　　　　 2 g</P><P><B>Experiment1</B></P><P>Sample 1 curdlan 15 µl ＋Congo red 5 µl ＋Agro bacterium precipitation</P><P>Sample 2 de-ionized water 15 µl　＋Congo red 5 µl　＋Agro bacterium precipitation</P><P>We centrifuged Sample 1 and 2 in 5 minutes with cooled centrifuges(4 ℃,13000 rpm).</P><P>As a result, Sample 1 had red precipitates. Sample 2 rationally melted.</P><P>So, we confirmed the presence of carbohydrate in the precipitates obtained from Agro bacterium.</P><P><B>Experiment2</B></P><P>Next, we found that carbohydrate could be produced when soy milk is added to the fluid bacteria</P><P>So, we experimented for the confirmation of the above findings.</P><P>Sample 3　LB medium 7 ml ＋Agro bacterium＋soy 40 µl</P><P>Sample 4　LB medium 7 ml ＋Agro bacterium</P><P>Sample 5　LB medium 7 ml　　          　 ＋soy 40 µl</P><P>Sample 6　LB medium 7 ml</P><P>We cultivated Sample 3 ~ 6 at 28 ℃.for 12 hours</P><P>Then, Sample1 and 2 produced a white film like substance but Sample 3 and 4 did not.</P><P>We put the white films from Sample 1 and 2 into micro-tubes.</P><P>‎ ‎</P><P>　　　　　　　　　　　　　　　　Sample3　　　　　　　　　　　　　　　　　　　　　　Sample4</P><P>Sample 7　 supernatant(Sample 1) 15 µl　＋Congo red 5 µl</P><P>Sample 8　 supernatant(Sample 1) 15 µl　＋Congo red 5 µl　＋de-ionized water 1 ml</P><P>Sample 9　 supernatant(Sample 2) 15 µl　＋Congo red 5 µl　</P><P>Sample 10　supernatant(Sample 2) 15 µl　＋Congo red 5 µl　＋de-ionized water 1 ml</P><P>Sample 11　Congo red 5 µl　＋de-ionized water 1 ml</P><P>We centrifuged all the samples for 5 minutes at 4 ℃,13000 rpm.</P><P>As a result, Sample 7 ~ 10 gave red precipitates but Sample 11 did not.</P><P>So, we confirmed Sample 7 ~ 10 had carbohydrate.</P><P>Through these experiments, we confirmed that Agro bacterium can discharge carbohydrate without soy milk.</P><P>Reference</P><P>1, Laboratory maintenance of Agrobacterium.</P><P>(<A href="http://europepmc.org/articles/PMC3350319" class="external free" rel="nofollow">http://europepmc.org/articles/PMC3350319</A>)</P><P>2, Coordination of Division and Development Influences Complex Multicellular Behavior in Agrobacterium tumefaciens (Jinwoo Kim.¤, Jason E. Heindl., Clay Fuqua)</P><P>3, Kirin Kyowa Foods Company (<A href="http://www.kirinkyowa-foods.co.jp/products/curdlan/" class="external free" rel="nofollow">http://www.kirinkyowa-foods.co.jp/products/curdlan/</A>)</P><P>4, Howard Hughes Molecular Biology Summer Research Program Poster, Austin, TX , August, 1995 Microcopy of Curdlan  Structure</P><P>5, Iain M. Cheeseman and R.Malcom Brown,Jr, Department of Botanty , The University of Texas at Austin,TX,78713</P><P>(<A href="http://www.botanty.utexas.edu/facstaff/facpages/mbroun/ongres/icheeze.htm" class="external free" rel="nofollow">http://www.botanty.utexas.edu/facstaff/facpages/mbroun/ongres/icheeze.htm</A>)</P></H2><H2>Curdlan's introduction<P>Agrobacterim makes  carbohydrate chain. For exampe it makes curdlan on culture midium.We'll explain curdlan  on this page.</P><P>About</P><P>It is fermentative polysaccharide that microorganizm makes. It polymerizes two position;C₁and C₃on D-glucose.  Its binding  position is not straight. So it is cycloid structure.</P><P><A href="/File:Biwako-nagahama_curdlan%27s-picture_yoshiharu-otaki1.png" title="File:Biwako-nagahama curdlan's-picture yoshiharu-otaki1.png">File:Biwako-nagahama curdlan's-picture yoshiharu-otaki1.png</A></P><P>Property</P><P>It is insolubility in water.  But it can become dispersing solution in cold water. And after high speed shuffle,the solution go to uniform dispersing solution. Moreover it become complete dispersing solution if it is mixxed by alkaline solution(avobe pH12). And last, dispersing solution become gel unnder the situation that it's heated by high temperature.</P><P>Gel</P><P>Dispersing solution of curdlan rizes viscosity if its temperature incleses to 50℃,and become gel if it is heated moreover.  The strenth of gel is ploportional by concentration of curdlan and/or temperature of heating.</P><P>The lelation curdlan and our life</P><P>It is used as food additive. The object is improving quolity ,mainting water-holding capasity and improving stability of hot. It is used in nudle,processed food of meet and garnish food. So curdlan isn't distant in our environment. It is relating in our life very much.</P><P>Gelatification of the Curdllan</P><P>Objective:Gelatification of the curdlan by heating the crdlan ＆producting3％(w/v) curdlan gel</P><P>Procedure</P><P>↓10ml of de-ionized water in the beaker and cool the beaker using ice pack.(Curdlan is soluble to cool water)</P><P>↓Take 0.30g of  curdlan and mux with the cooled de-ionized water using starrer for 20mins.(Occasional cooling of the beaker)</P><P>↓Heat the solution until vapors are seen ＆cool at room temp</P><P>↓Throw out the excess of water</P><P>↓@30℃ incubator for overnight for dehydrating</P><P>Result</P><P>The Gel shown in figure below was made</P><P>Left 1st day     Middle＆Right  2nd day</P><P>1st days gel has elasticity and softly but 2nd days gel was almost dry. </P></H2><H2>Binary Vector<P>We named BBa_K1044006 &quot;pBI107&quot;.This plasmid length is about 9,000 bp.</P><P>
●Function</P><P>pBI107 is improved form of empty backbone vector obtained from pre-existing binary vector of E.coli &amp; Agrobacterium. Selective with Kanamycin, there is possibility of reproduction by both E.coli and Agrobacterium.
pBI107 having both prefix and suffix, we can freely insert the cassette into the cloning site of the BioBrick which gives possibility of different expressions of BioBricks with Agrobacterium. </P><P>Also the possibility of use of OriV can just not be limited with Agrobacterium but also may broaden to be used with other organisms.</P><P>
●Proof for the reproduction of E.coli’s pBI107 and selection (NPT III)
After cloning the E.coli JM109 with pBI107, we plated the contents to a LB-Agar plate with 50 µg/ml kanamycin and cultured it at 37℃ for overnight. Similar experiment was performed for the E.coli JM109 without cloning to pBI107. From the experiment we found that under 50 µg/ml kanamycin, E.coliJM109 with pBI107 cultured and developed colony but the E.coli without pBI107 did not develop any colony.
From above result it was confirmed that E.coli JM109 cloned with pBI107 can reproduce under the influence of kanamycin. Hence, the functionality of ColE &amp; NPTⅢwith regard to E.coli JM109 was proved.</P><P>●Proof for the reproduction of Agrobacterium’s pBI107 and selection (NPT III)
After cloning Agrobacterium C58 with pBI107, it was plated to LB-Agar plate with 50 µg/ml kanamycin and cultured for 2 days at 30℃. Similar, experiment was carried out for the Agrobacterium C58 without cloning it. Colonies were confirmed for the cloned Agrobacetrium C58 under 50 µg/ml kanamycin but no colonies were found for the normal Agrobacterium C58.
From above result it was confirmed that Agrobacterium C58 cloned with pBI107 can reproduce under the influence of kanamycin. Hence, the functionality of OriV &amp; NPTⅢwith regard to Agrobacterium C58 was proved.</P><P>
●Proof for the prefix and suffix of pBI107
We determined the prefix and suffix of the BioBrick cloning site and inserted RFP device(BBa_J04450) into pBI107and cloned E.coli JM109. 
Then it was cultured in LB-Agar plate with 50 µg/ml kanamycin and IPTG at 37℃. After 6 hours of culturing, colonies were detected and after 24 hours of culturing the colonies with inserted RFP device were confirmed by the colonies which were shining red.</P><P>The shining red colonies were extracted and treated with restriction enzyme used for extraction of prefix and suffix. Then the length of vector(pBI107) &amp; insert(RFP device) were confirmed through electrophoresis. 
As expected the actual length of pBI107 was 9000bp whereas the extracted RFP device’s length was around 1100bp.
</P><P>From the above result, we found out the existence of prefix and suffix as the iGEM standard in pBI107 and hence BioBrick was expressed with E.coli JM109.</P></H2><H2>E.coli-ink</H2><P>Another one of our project is E.coli ink.</P><P>We are planning to use Escherichia coli expressing the sperm whale myoglobin as ink. We are also trying to change the color of transgenic Escherichia coli by changing the growth and medium conditions.</P><P>Why do we use myoglobin as ink ? </P><P>Recently, fluorescent pigment is very famous and popular. It is nice to look at.</P><P>But, Many fluorescent pigment is only colouring when it’s exposed to ultraviolet rays.</P><P>We want to make pigment colouring under natural light. Thus,We decided to use myoglobin as ink.</P><P>Finally, We will draw my advisor’ face on Agrepaper with this E.coli ink !</P><P>The nucleotide sequence of myoglobin of sperm whale was already published, and We have synthesis company to synthesis the DNA fragment encoding the myoglobin. </P><P>Fig1 shows　this DNA fragment sequence. Fig1 is published on paper.</P><P>Fig1.DNA fragment encoding myoglobin.</P><P>DNA synthesis company send us the DNA fragment with pTAKN-2. pTAKN-2 is plasmid that .Thus,We get DNA fragment with pTAKN-2.</P><P>Next,We insert to pUC18 to confirm the function of this DNA fragment.</P><P>We did double digestion pUC18 and pTAKN-2 with BamHⅠand SmaⅠ.This experiment protocol shows below.</P><P>two samples(include smaⅠ,plasmid,10×buf) Incubated at 37℃ for 2h.add phenol chloroform mixture 50μℓ and Vortex. centrifugal at 4℃ with 13000rpm for 5min. pour supernatant into another tube. add 3M CH3COONa 5μℓ and 99.5% EtOH 125μℓ to supernatant and Vortex. centrifugal at 4℃ with 13000rpm for 10min. add mixture solution to pellet . mixture solution composition is  BamHⅠ,plasmid,10×buf.</P><P>This sample again incubated at 37℃ for 1h.add phenol chloroform mixture 50μℓ and Vortex.
centrifugal at 4℃ with 13000rpm for 5min. pour supernatant into another tube. add 3M CH3COONa 5μℓ and 99.5% EtOH 125μℓ to supernatant and Vortex. centrifugal at 4℃ with 13000rpm for 10min. add 70% EtOH 500μℓ to pellet and up side down. Remove supernatant . dyr up at 65℃.add TEbuffer 10μℓ to pellet. Mix two samples and Ligation kit Ver2. 
We use Ligation kit Ver2 on Ligate reaction.</P><P>Transformation (show protocol page)
Plating to LB-Amp plate.
</P><P>Next day.</P><P>The Colony is on plate. We inoculate this colony into 2μℓ Amp in 2ml 2YT medium in culture tube</P><P>Next day</P><P>We did Miniprep of this sample.</P><P>Theoretical value is  but this plasmid is supercoil structure. Therefore, Theoretical value is  .</P><P>We confirm bund size is theoretical value with agarose electrophoresis.</P><P>We again inoculate this colony into 3μℓ 50ng/μℓ Amp in 30μℓ 100mM IPTG in 3ml 2YT medium in culture tube .</P><P>Next day, centrifugal this bacterial cell medium at 4℃ with 13000rpm for 1min.</P><P>Fig2 shows part’s function!</P><P>Fig2.myoglobin is changed it’s color in Ecoli</P><P>left:pellet of JM109 with pUC18.  right:pellet of JM109 with pUC18/Mb</P><P>Next,We use SDS/PAGE to confirm E.coli cloned pUC18/Mb produce myoglobin.</P><P>SDS/PAGE sample preparation protocol below.</P><P>Centrifugal Bacterial cells 200μℓ at 4℃ with 13000rpm for 1min. remove supernatant. Add 1×sample buffer to pellet and vortex. dip this sample into water bath at 98℃(not heat block). Let it sit on ice 30sec. Centrifugal the sample at 4℃ with 13000rpm for 10min.We use this sample 10μℓ for SDS/PAGE.</P><P>Fig3 shows Contative date of part’s function !</P><P>Fig3. SDS/PAGE analysis(CBB stain one super)with 15%gel</P><P>Lane1:JM109 with pUC18   Lane2:JM109 with pUC18/Mb</P><P>Next, We have to mutate this DNA fragment encoding myoglonbin because that includes EcoRⅠand XbaⅠsite. GAATTC that is EcoRⅠsite is changed to GAATTT. As same TCTAGA that is XbaⅠsite is changed to TCCAGA.</P><P>EcoRⅠand XbaⅠsite pointmutation protocol below</P><P>First,We design the primer occurring pointmutation.Primer sequence below.</P><P>F1:T T C T A G A     G      A G G A G A A C A A C A A C A</P><P>R1:ACTAGT A TTATTA TCA  T T A A C C C T G G T</P><P>F2:C T G G A A T T T A T C T C T G A A G C G</P><P>R2:C G C T T C A G A G A T A A A T T C C A G</P><P>F3:C T G C A T TC C A G A C A T C C A G G T A A C</P><P>R3:G T T A C C T G G A T G T C T G G A A T G C A G</P><P>Fw:GTTTCTTCGAATTCGCGGCCGCTTCTAGATG</P><P>Rv:GTTTCTTCCTGCAGCGGCCGCTACTAGTATTATTATC</P><P>We use primer F1 and R2 to make the left end fragment on PCR. PCR productis creaned up with exosap</P><P>We use primer F2 and R3 to make middle fragment on PCR.</P><P>We use thermal cycler on PCR. but,middle fragment theoretical value is about 60 base pair, it is very short. So this PCR don’t amplify DNA fragment we targeting but plasmid itself. PCR enzyme using by us is Prime star Max. Prime star Max is highly efficient enzyme. We replace Prime star Max with prime star hot start , but result is same. </P><P>Hereupon we hit upon an idea. We worked PCR by hand with water bath at 55℃, 70℃ and 98℃ because We think thermal cycler appear gradient temperature and this is a cause of failure. We dip sealed sample in water bath one by one. 98℃,55℃,70℃,98℃,55℃・・・・.</P><P>Then,We success PCR that amplify 60 base pair DNA fragment.the PCR productis creaned up with exosap</P><P>We use primer F3 and R1 to make the right end fragment . PCR products creaned up with exosap</P><P>mix the left end fragment and middle fragment and do overlap PCR. PCR products creaned up with exosap</P><P>mix the constructed fragment and the right end fragment and do overlap PCR. PCR products creaned up with exosap</P><P>Do PCR of constructed fragment with Fw and Rv primer.   </P><P>Thus We construct DNA fragment encoding myoglobin not including EcoRⅠ and XbaⅠ site .</P><P>We read sequence of this DNA fragment to confirm pointmutation </P><P>result of ncbi below</P><P>referance:High-level expression of sperm whale myoglobin in Escherichia coli
(gene synthesis/codon bias/protein engineering)
BARRY A. SPRINGER AND STEPHEN G. SLIGAR
Department of Biochemistry, University of Illinois, Urbana, IL 61801
Communicated by Gregorio Weber, September 4, 1987</P></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:Biwako_Nagahama/Project">http://2013.igem.org/Team:Biwako_Nagahama/Project</A>&quot;</DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Biwako_Nagahama/Project" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Biwako_Nagahama/Project" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Biwako_Nagahama/Project&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Biwako_Nagahama/Project&amp;oldid=240912" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></UL></UL></DIV></DIV></DIV></DIV></BODY></HTML>