Team:SCU-China/Drake
Toggle navigation
Home Page
Project
Rihanna
Overview
CecropinXJ
Lysis
Drake
Overview
Acetic Acid
Leucine
Enhancement
Overview
Knockout
VHb
Hardware
Modeling
Parts
Collaboration
Human Practice
Education
ACMA
Community Class
Teach-in for Freshmen
Survey
In Campus
In Longquan
NDS
Meet Up
IngenuityLab_Canada
iSWU
CCiC
Integrated Human Practice
Notebook
Team
Protocol
Drake
Work, work, work, work, you see me I be work with “Rihanna”. Yes, the insole is “Rihanna” featuring “Drake”. Strain “Drake” includes gene aarC to covert acetic acid into TCA cycle. It also competitively utilizes leucine to prevent the production of isovaleric acid in shoes.
Overview
Besides releasing AMP, we want to absorb the foot odor as well. Researches show that the isovaleric acid is key component to cause uncomfortable odor for human beings, though it only occupies 2.3% in the fatty acid components of foot odor. And another major component in the odor, acetic acid, which occupies 85.1% in the fatty acid components, contributes to the odor of sweat. (Figure 1)[1]. As a result, we decided to remove those two fatty acid components.
Figure 1
In order to reduce the acetic acid production, we utilize Succinyl-Coenzyme A (CoA): Acetate CoA-Transferase (aarC), which is derived from Acidophile Acetobacter aceti. aarC can transform succinyl coenzyme A and acetic acid into acetyl coenzyme A and succinic acid.
But for isovaleric acid, it’s hard to deal because we didn’t find specific enzyme to degrade it because it enters branched chain fatty acid metabolism pathway. But isovaleric acid is the degraded product of leucine which comes from the skin. As a result, we decide to competitively utilize leucine to prevent the production of isovaleric acid. (Figure 2)[2]
Figure 2
We want to overexpress the ABC transport of leucine, tRNA and aminoacyl-tRNA synthetase, a special synthesized gene called polyleu, whose coding protein is full of leucine to enhance the whole protein synthesis process. But at last, we only overexpress the transport.
The Drake system works as follows.
Figure 3
[1] Ara, K, et al. "Foot odor due to microbial metabolism and its control. " Canadian Journal of Microbiology 52.4(2006):357-364(8).
[2] James, A. Gordon, D. Cox, and Kathryn Worrall ‡. "Microbiological and biochemical origins of human foot malodour †." Flavour & Fragrance Journal 28.4(2013):231–237.
Acetic Acid
Goal: To reduce the acetic acid metabolized by microorganisms.
Achievement: The enzyme have not been successfully expressed in E. coli BL21
Part: BBa_K1919233
Introduction
Acetic acid is the metabolic product of microorganisms on foot that contributes sweat odor in foot odor. In order to reduce the acetic acid production, we utilize Succinyl-Coenzyme A (CoA): Acetate CoA-Transferase (aarC), which is derived from Acidophile Acetobacter aceti. aarC can transform succinyl coenzyme A and acetate to acetyl coenzyme A and succinate
There are two benefits owing to the metabolic pathway: First, acetic acid can be involved in tricarboxylic acid cycle (TCA) rather than be discarded as waste. Second, E. coli can also use the acetic acid or acetate from outside for its growth.
Design & Result
Figure 1 shows how aarC works (Figure 1).
Figure 1
We got the coding sequence of the enzyme in the NCBI database. After codon optimization, the gene is synthesized by Synbio Technologies Co. Ltd. In order to preliminary detection and optimize subsequent experiments, we use PCR method to clone gene then use enzyme cutting and connection to load it into pET 32 vector for expression. Product detecting by SDS-PAGE method
Figure 2. Line 1 is supernate after cell disruption. Line 2 is precipitate. Line 3 is whole cell. Line 4, 5, 6 are half loading.
pET32a+ vector involves His-tag and T7 terminator. His-tag can be used for Western-blot to detect. We cloned aarC coding sequence, His-tag and T7 terminator from the pET32+ vector together by PCR.
Considering the expression may bring effects on the growth of E.coli, we use four constitutive promoters at different strengths. We use long primers in to ligate standard RBS and five kinds of promoters J23100, J23104, J23105, J23106 and J23114 to the front of aarC coding sequence with His-tag and T7 terminator cloned from pET32a+ vector. (Figure 3).
Figure 3. AarC added promoter, RBS, His-tag and terminator.
After promoter, RBS and aarC coding sequence have been assembled as a completed expression system, we ligate them to the pSB1C3 vector to get aarC expression vector of five kinds of expression intensity (Figure. 4).
Figure 4. Use Gibson Assembly to assemble plasmids.
After the expression vectors were introduced into BL21 (DE3) strains.
Functional test of AarC
We test two kinds AarC expression intensity (J23100 & J23114. J23100 is ten times stronger than J23114) BL21 (DE3) and control group (BL21 without any plasmids) in the same condition. After adding additional acetic acid and acetate (2mg/mL) in the culture medium 8 hour, we detect the acetic acid and acetate concentration in the medium with HPLC. Because LB medium composition is very complex, we can’t do quantitative analysis, so we give the raw data (Figure 5-10) here and get some qualitative conclusion. The HPLC method is in protocol.
Figure 5. HPLC result of added sodium acetate in control group. Arrow points the acetate peak. Peak area is 3419002.
Figure 6. HPLC result of added sodium acetate in J23114. Arrow points the acetate peak. Peak area is 2142511.
Figure 7. HPLC result of added sodium acetate in J23100. Arrow points the acetate peak. Peak area is 1443872.
Figure 8. HPLC result of added acetic acid in control group. Arrow points the acetic acid peak. Peak area is 2816616.
Figure 9. HPLC result of added acetic acid in J23114. Arrow points the acetic acid peak. Peak area is 5382066.
Figure 10. HPLC result of added acetic acid in J23100. Arrow points the acetic acid peak. Peak area is 5664716.
In summary, we know that under the control of the both promoter, AarC can express successfully. The results showed that AarC can let E.coli use acetate well but may can’t consume acetic acid as our tentative idea.
We have a dilemma that if we didn’t add so much acetic acid, we can’t detect it by HPLC because LB medium also have a same peak there. But we found that E.coli growth badly in 2mg/mL acetic acid. We guess because our E.coli have AarC enzyme, so when they dead in LB medium, AarC released and transformed succinate in medium and themselves to acetate. That is why in figure 9 & 10 the acetic acid is higher than control group. There are many problems and we need more studying in future.
Leucine Utilization
Overview
Goal: With the aim to enhance E.coli utilize leucine dramatically, we express liv operon heterogeneously in BL21(DE3)
Achievements: Successfully construct 6 plasmids in which the six gene from liv operon reside flanking with the same promoter, RBS and terminator. In addition, a similar plasmid containing whole liv operon was constructed as well. The expression level of the six genes has been scrutinized via quantitative PCR. The functions of all seven devices have been tested via spectrophotometer, the results of which suggest that all of the gene can reduce the amount of leucine in LB liquid medium.
Parts: BBa_K1919300, BBa_K1919301, BBa_K1919302, BBa_K1919303, BBa_K1919304, BBa_K1919305, BBa_K1919306.
Introduction
Leucine is one of the element that can be metabolized to isovaleric acid by microorganisms on foot and then delivers awful odor. Thus we design a kind of special strain can utilize leucine dramatically so that other microbe can’t have this kind of raw materials to produce chemicals that have unpleasant smell.
Design
Liv operon contain six genes which is regarded as a operon coding a leucine transporter complex. All six genes used in this project derived from E.coli DH5a and in normal condition, this operon can not start transcript because of inhibition. Therefore, we clone all six genes into plasmids under control of constitutive promoter J23100. In order to make sure that all the genes in the plasmids are expressed, we utilize quantitative PCR to scrutinize the expression level of all six gene in the operon respectively in both BL21(DE3) and DH5a. Although it’s well known that BL21(DE3) is quite suitable for gene expression, we considered that the genes are derived from DH5a and thus it’s necessary to observe the expression level of genes in BL21(DE3) compared with those in DH5a. What’s more, with the aim to promote microbe to absorb leu in medium further, we attempt to clone tRNA-leu, aminoacyl tRNA synthetase and artificially synthesized polyleucine fragment which contain about 30 tandem leucines into overexpression vectors accordingly. Unfortunately, all those vectors are failed to be constructed, except for polyleucine fragment. Nevertheless, as figure A shows, the secondary structure of this gene’s transcript is too complex and are believed can not transcript properly in vivo. Thus, the polyleucine was discarded later in the project. The results of qPCR show that the expression levels of genes are accelerated when the E.coli was introduced into the plasmids, which means all six genes have been overexpressed in the microbe.
figureA1  polyleucine RNA secondary structure prediction based on MFE(minimum free energy). The color indicates base-pair probabilities.
FigureA2  Sequence of polyleucine fragment
Figure A3 entropy of polyleucine fragment
Figure A4  the dot plot containing the base pair probabilities of polyleucine fragment.
FigureA5
Since the polyleucine fragment is artificially synthesized by company and they do not make it completely, we repaired the fragment by adding promoter and RBS via Gibson assembly. The figure shows different condition to amplify the fragment of polyleucine. As mentioned before, there are numerous short repeats in the fragment which can account for why there are ladder like bands in the figure.
In addition, we noted that the expression level of livJ is increased drastically compared with other gene in the operon, which may indicate that the livJ plays a crucial role in metabolism of E.coli, which is consistent with the references that livJ is the most sensitive gene to lrp regulation in the operon. What’s more, we interestingly find that all expression levels of six genes are reduced generally in the same sequence of their position in operon, which might shed light on the fact that the stability of RNA of those genes in the operon is determined by coding sequence. Definitely, we can’t neglect the role of UTR (untranslated region) of liv operon played in regulation of stability of RNA in wild type of E.coli. However, as we have mentioned before, all these genes’ coding sequence starting from ATG to TAA are cloned to the overexpression vectors, in another word, the only discrepancy of overexpression vectors is the coding sequence and 5’ UTR & 3’UTR are all the same because of identical promoter, RBS and terminator. What’s more, not only the BL21(DE3) shows this kind of tendency that the expression level of six gene decrease in order, but the DH5a shows the tendency as well. Thus, we have sufficient evidence to support our belief that the determinant of stability of gene’s RNA includes in ORF (open reading frame).
After ensuring that the gene can be overexpressed in E.coli, we go on to examine the function of these genes. Although there are service in company to test the amount of amino acid in liquid sample, we incline to finish our project by ourselves as much as possible. We choose spectrophotometer to observe function of genes. In order to ensure the amount of leucine can be detected by spectrophotometer, we artificially increase the concentration of leucine in liquid LB to about 1 g/L. Thus, the spectrophotometer can detect the alternation of concentration of leucine after incubation for more than 24 hours with engineered E. coli.
First of all, we established a standard method to dispose the LB medium in this project so that all the data can be comparable. The results of spectrophotometer indicate that all six genes can deduce the concentration of leucine in liquid medium, albeit in different level. However, the results are not very convincing since the background of amino acid in LB medium can disturb the measurement profoundly. Then, we try to utilize the inorganic medium, M9 medium, to inoculate the engineered E.coli and the medium is artificially laced with additionally leucine. The results suggest that our engineered microbe do own desired function, that is, absorb the leucine from ambiance.
Result
1. Gibson Assembly
Figure1A
Figure 1B
Figure 1C
As the figure shows, we construct our overexpression vectors via Gibson assembly. However, because some of unknown problem occur in the reaction buffer, we did not get clear bands. It may result from long time storage of the assembly mix and the activity of three different enzyme in the mix altered, the activity of exonuclease may preponderate over other enzyme and digest the fragment into piece. Fortunately, we have screen out the positive colony after transformation.
Figure 2A
Figure 2B The six gene of the liv operon and the position of the six genes
Figure 3A
The result of relatively quantitative PCR. We utilize two gene as reference in the PCR, one is rrsA which is a gene of 16s RNA and the other is dnaA which is a gene of DNA polymerase element. Generally speaking, the expression level of rrsA is very high, whereas the expression level of dnaA is relatively low. In this table, we presume that the discrepancy of dnaA’s and rrsA’s expression level are the same in BL21 and DH5a, which is 10^10 folds.
Figure3B
Figure3C
However, we found the discrepency of expression level of dnaA and rrsA are quite distinct between BL21(DE3) and DH5a. T-test indicates p<0.01 (table 1B). Thus, we take this factor that the expression level of rrsA is 100000 folds over expression level of dnaA in DH5a and 160000 folds in BL21(DE3) into account.
figure 3D
figure 3E
After take the factor mentioned before into account, we get the figure 3D and 3E which reveal more reality than figure 3A does.
To exam which overexpression vector can promote leu uptake most drastically, we transferred all six genes into BL21 and DH5a. We designed a novel protocol to exam fluctuation of leu concentration in liquid LB medium. The protocol goes that:
Materials that should be prepared in advance
1 sulfosalicylic acid solution   20 g solid sulfosalicylic acid in dissolved in 100ml ddwater
2 ninhydrin solution         1 g ninhydrin is dissolved in 35ml ddwater
3 pH8.03 PBS
A solution  4.5350g KH2PO4 is dissolved in 500ml ddwater
B solution  11.938g Na2HPO4 is dissolved in 500ml ddwater
pH8.03 PBS = 10ml A solution +190ml B solution
1 1ml sterilized LB medium is inoculated with microbe in 1.5 ml EP tube and shaking in 37 ℃ overnight
2 10000g*1min for centrifugation and then pipette 800ul supernatant to another new 1.5 ml EP tube
3 pipette 300 ml LB to other two new 1.5 ml tubes and add 100ul sulfosalicylic acid solution
4 shaking violently for 1 min
5 20000g*10min and then pipette 300ul supernatant to a new 1.5 EP tube
6 add 1ml pH8.4 PBS buffer and mix them by reversing the tube several times
7 all 0.15ml ninhydrin solution and mix them by reversing the tube several times
8 incubate the tube in room temperature for 1hr
9 detect the optical density via spectrophotometer
Figure 6 full wavelength scanning of liquid LB implemented with leucine (A) or without leucine (B). We detected that there is a high peak of absorption in 570 nm.
Before further test, we exam two kinds of liquid LB medium that are with or without additional leucine implement. The liquid medium was processed follow the protocol above and tested by spectrophotometer for full wavelength scanning from 300nm to 1000nm. Since liquid LB medium teems with amino acid, short peptide and other nutrition, we can see a peak in B (without leucine implementation) 570nm as well.
Figure 7 liquid mediums shaking with DH5a harboring different overexpress vectors are tested by spectrophotometer at 570nm.
The optical density as 570nm of liquid LB implemented with leucine. Nctr is a negative control set that do not contain microbe. 2101 is a negative control as well but inoculates with an E.coli named 2101 that harbors chloromycetin resistance but only expresses a short non-coding RNA. LivJKHMGF indicates medium after shaking overnight with E.coli DH5a contain corresponding overexpression vectors. The data of BL21 is not shown here since the data of different sets only owns tiny discrepency.
We try to explain these results, but we are not sure the following explanation reflects reality to what extent. To ascertain this question, there need more sophisticated investigation.
As we have mentioned before, liv JKHMGF are not resides on genome of Bl21. In addition, the leucine ABC transporter will not work if there is not other component expressed in cell. Thus, it can be understood that there are tiny discrepency among different sets of experiments in BL21.
For DH5a, Albeit the DH5a owns livJKHMGF in genome, the expressions of them are repressed by a regulator, Lrp (leucine-responsive regulator protein). According to the result of BLAST, Lrp not only resides on genome of DH5a but that of BL21. Lrp is a dual transcriptional regulator for at least 10% of the genes in Escherichia coli. These genes are involved in amino acid biosynthesis and catabolism, nutrient transport, pili synthesis, and other cellular functions. It is believed that Lrp senses the presence of rich nutrition based on the concentration of leucine and positively regulates genes that function during famine and negatively regulates genes that function during a feast. Lrp-leucine DNA bingding transcriptional dual regulator inhibits livJ transcription initiation, whereas L-leucine binds to Lrp transcriptional dual regulator to block inhibition of livKHGMF.
As shows in figure 7, livJK seem do not increase uptake of leucine compared with 2101, livHM inhibit uptake of leucine compared with 2101 and livGF promote uptake of leucine dramatically, especially for livF. We can brace these six genes into 3 group, livJK, livHM and livGF, according to the result of spectrophotometer. Then we attempt to ascertain the inner relationship that brace these genes into the 3 group.
Figure 8 number of leucine in these six genes
Figure 9 percentage of leucine in these genes =number of leucine/number of amino acid
Firstly, we list leucine number and percentage in these genes, and we considered that if there are more leucine in the protein, it will uptake more leucine from medium. But we did not find any relevance between these data and results in figure 7.
Then we scrutinize the construction of the leucine ABC transporter. Although there is a little information for the transporter, we find that the hydrophilicity plots can brace the six genes into three categories.
figure 10 hydrophilicity plot of livJK
figure 11 hydrophilicity plot of livHM
figure 12 hydrophilicity plot of livGK
As figure 10,11,12 shows, livJK owns a hydrophobic N terminal. It is the signal peptide that guides livJ and livK to transport inner membrane and to be trapped in periplasmic space. livHM should be transmembrane protein since they have very hydrophobic peptide span. LivGF do not contain obvious very hydrophobic peptide span but contains Walker A and Walker B motifs involved in the binding and hydrolysis of ATP (according to results of BLAST).  Thus we can imagine the structure of leucine ABC transporter which is shown in figure 13.
Figure 13 predicted structure of leucine ABC transporter.
Oversxpress livJK in DH5a will not increase uptake of leucine in high leucine concentration medium (around 1ug/ml). Overexpress livMH in DH5a inhibits leucine uptake of leucine in high leucine concentration medium (around 1ug/ml). Overexpress livFG in DH5a promotes leucine uptake of leucine in high leucine concentration medium (around 1ug/ml). It may be explained that livFG works like a pump to hydrolysis ATP and obtain energy to transport leucine from periplasmic space into cytosol. More livFG means more energy is used to pump leucine into cytosol.
Overexpress livHM may result in more transmembrane pathway which may be a handicap to retain leucine in cell cytosol without futher energy to determine direction of transportation, that is, there will be some leak of leucine when overexpress livHM.
Overexpress livJK can not increase uptake of leucine may be explained that livJ and livK are saturated for transporter livHMGF, that is, more livJK to pass leucine to transporter livHMGF than the transporter needs. In addition, overexpressed livJK do not have enough time to be transported into periplasmic space and congregated into together in cell because of hydrophobic N terminal and livJK do not have enough enzyme to cleave N-terminal of livJK, which may be another explanation for performance of overexpressed livJK. The two explanation for livJK are not mutual excluded.
Please note that all of those explanation of leucine uptake derived from conjecture. To ascertain the puzzle results, further and sophisticated experiment should be carried out.
