<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_NCTU_Formosa_Model skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:NCTU Formosa/Model</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="main-container"><UL class="main-menu" id="menu" style="list-style-image: none;"><LI class="dropdown"><A href="https://2016.igem.org/Team:NCTU_Formosa" class="dropbtn">Home</A></LI><LI class="dropdown"><A href="https://2016.igem.org/Team:NCTU_Formosa/Project" class="dropbtn"><SPAN class="menu_sign">Project</SPAN></A><DIV class="dropdown-content"><DIV><A href="https://2016.igem.org/Team:NCTU_Formosa/Description">Description</A></DIV><DIV><A href="https://2016.igem.org/Team:NCTU_Formosa/Design">Design</A></DIV><DIV><A href="https://2016.igem.org/Team:NCTU_Formosa/Demonstrate">Device</A></DIV><DIV><A href="https://2016.igem.org/Team:NCTU_Formosa/Proof">Results</A></DIV><DIV><A href="https://2016.igem.org/Team:NCTU_Formosa/Model">Modeling</A></DIV><DIV><A href="https://2016.igem.org/Team:NCTU_Formosa/Safety">Safety</A></DIV></DIV></LI><LI class="dropdown"><A href="https://2016.igem.org/Team:NCTU_Formosa/Human_Practices" class="dropbtn"><SPAN class="menu_sign">Human Practice</SPAN></A><DIV class="dropdown-content"><DIV><A href="https://2016.igem.org/Team:NCTU_Formosa/Integrated_Practices">Study</A></DIV><DIV><A href="https://2016.igem.org/Team:NCTU_Formosa/Collaborations">Collaborations</A></DIV><DIV><A href="https://2016.igem.org/Team:NCTU_Formosa/Engagement">Education</A></DIV><DIV><A href="https://2016.igem.org/Team:NCTU_Formosa/Meetups">Meetups</A></DIV><DIV><A href="https://2016.igem.org/Team:NCTU_Formosa/Further_Considerations">Further Considerations</A></DIV></DIV></LI><LI class="dropdown"><A href="https://2016.igem.org/Team:NCTU_Formosa/Entrepreneurship" class="dropbtn"><SPAN class="menu_sign">Entrepreneurship</SPAN></A><DIV class="dropdown-content"><DIV><A href="https://2016.igem.org/Team:NCTU_Formosa/Entrepreneurship#Businessmodel">Business Model</A></DIV><DIV><A href="https://2016.igem.org/Team:NCTU_Formosa/Entrepreneurship#MarketingFuture">Marketing &amp; Future</A></DIV><DIV><A href="https://2016.igem.org/Team:NCTU_Formosa/Entrepreneurship#RegulationLaw">Law &amp; Regulations</A></DIV><DIV><A href="https://2016.igem.org/Team:NCTU_Formosa/Entrepreneurship#Business_competition">Business Competition</A></DIV><DIV><A href="https://2016.igem.org/Team:NCTU_Formosa/Entrepreneurship#ourSponsor">Sponsor</A></DIV></DIV></LI><LI class="dropdown"><A href="https://2016.igem.org/Team:NCTU_Formosa/Parts" class="dropbtn"><SPAN class="menu_sign">Parts</SPAN></A><DIV class="dropdown-content"><DIV><A href="https://2016.igem.org/Team:NCTU_Formosa/Basic_Part">Basic Parts</A></DIV><DIV><A href="https://2016.igem.org/Team:NCTU_Formosa/Composite_Part">Composite Parts</A></DIV></DIV></LI><LI class="dropdown"><A href="#" class="dropbtn"><SPAN class="menu_sign">Notebook</SPAN></A><DIV class="dropdown-content"><DIV><A href="https://2016.igem.org/Team:NCTU_Formosa/Notebook">Lab Note</A></DIV><DIV><A href="https://2016.igem.org/Team:NCTU_Formosa/Protocol">Protocol</A></DIV></DIV></LI><LI class="dropdown"><A href="https://2016.igem.org/Team:NCTU_Formosa/Attributions" class="dropbtn">Team</A></LI><LI class="dropdown"><A href="https://2016.igem.org/Team:NCTU_Formosa/Achievement" class="dropbtn">Achievement</A><DIV class="dropdown-content"><DIV><A href="https://2016.igem.org/Team:NCTU_Formosa/HP/Gold">HP Gold</A></DIV><DIV><A href="https://2016.igem.org/Team:NCTU_Formosa/HP/Silver">HP Silver</A></DIV></DIV></LI></UL></DIV></DIV><NAV id="scroll"><DIV class="wrapper"><DIV class="nodes"><DIV class="node" id="node1"><DIV class="topic"><P class="text_color"><A href="#title1" style="text-decoration:none;color:#F3F7F7;">Software</A></P></DIV></DIV><DIV class="node" id="node2"><DIV class="topic"><P class="text_color"><A href="#title2" style="text-decoration:none;color:#F3F7F7;">Pest Prediction System</A></P></DIV></DIV><DIV class="node" id="node3"><DIV class="topic"><P class="text_color"><A href="#title3" style="text-decoration:none;color:#F3F7F7;">Degradation Rate</A></P></DIV></DIV></DIV></DIV></NAV><SECTION class="content_container"><DIV><DIV id="modelingContainer"><P class="title" id="title1">Software—Toxin selection</P><DIV id="modelingContent"><DIV class="modelingPart"><H2 class="content-1" id="titleA" style="color:#33FFCC">I. Purpose</H2><DIV class="modelingPartContent" id="partA"><P class="content">To prove the concept of Pantide, we wanted to select three existing distinct spider toxin peptides with probable oral toxicity against the testee－Spodoptera litura(Tobacco cutworms). For the actual application of Pantide, we needed some more knowledge base of peptides which have different molecular targets to promote Pantide applying to other orders of insects, and a different toxic mechanism to regularly alternate so as to avoid drug resistance.</P><P class="content">To date, about 1500 toxin peptides from 97 spider species have been studied, though the number of spider toxin peptides is conservatively estimated up to 10 million. <SUP>[1]</SUP> So, our purpose is to establish a database collecting the information of those peptides, such as molecular target, taxon, toxicity, sequence. According to the database, if we first choose a target insect, then we can easily find out groups of suitable peptides used as Pantide. Therefore, we also need to create a method to select peptides from the database.</P></DIV></DIV><DIV class="modelingPart"><H2 class="content-1" id="titleB" style="color:#44FCCE">II. Method</H2><DIV class="modelingPartContent" id="partB"><P class="content">The method of toxin selection can be separated into three part: crawler, filter, and selection.</P><UL style="list-style-image:none;list-style-type:decimal;"><LI class="list">Toxin Collection—we planned to collect information of toxin peptides to establish our own database for Pantide from protein databases and some research results like taxon and toxicity from published papers. </LI><LI class="list">Toxin Filtering—based on background knowledge of toxin peptides, we set up some conditions to filter out those unsuitable to use as Pantide.</LI><LI class="list">Toxin Processing—we used online protein analytic tools to classify the remained peptides into groups by their similarity. Finally, we select out three distinct peptides from different groups to proof concept of Pantide.</LI></UL></DIV></DIV><DIV class="modelingPart"><H2 class="content-1" id="titleC" style="color:#5BFCD4">III. Step 1: Crawler</H2><DIV class="modelingPartContent" id="partC"><P class="content">In the beginning, we searched on UniProtKB/Swiss-Prot. It is a freely accessible database of protein sequence and functional information that is the manually annotated and reviewed section. (<A href="http://www.uniprot.org" style="color:#44E287;">http://www.uniprot.org/</A>) By searching the keyword “insecticidal NOT crystal” we wanted to find all the proteins that have insecticidal activity excluding those crystal proteins of Bacillus thuringiensis, and we got 216 proteins as results.</P><P class="content">Using the result, we established our Pantide database by crawling 11 entries of the protein information from UniProt. The entries are as follows.</P><UL style="list-style-image:none;list-style-type:disc;"><LI class="list">The name of the protein</LI><LI class="list">The description of protein function</LI><LI class="list">The organisms/source of the protein sequence</LI><LI class="list">The length of amino acids</LI><LI class="list">The number of disulfides bonds</LI><LI class="list">Propeptide &amp; signal peptide—If the proteins have an N-terminal signal peptide and propeptide, a part of protein will be cleaved during maturation or activation.</LI><LI class="list">Uniprot entry &amp; Arachnoserver id—the accession number of protein in UniProtKB and ArachnoServer*.</LI></UL><P class="content-2">*ArachnoServer is a manually curated database for protein toxins derived from spider venom.(<A href="http://www.arachnoserver.org/" style="color:#44E287;">http://www.arachnoserver.org/</A>).</P><P class="content">We also crawled other seven entries of protein toxicity recorded by Arachnoserver—molecular target, taxon, ED50, LD50, PD50, qualitative information, protein sequence from Arachnoserver. The term, Molecular target, is the effect site of toxin peptides, such as voltage-gated ion channels, GABA receptors and so on. Taxon, ED50, LD50, PD50, and the qualitative information are the toxicity against taxon that had been tested by experiments. The protein sequence from two databases is entirely the same.</P><P class="content">We utilized BeautifulSoup 4.4.0, sqlite3 and gevent modules in Python 3.5 to develop our crawler. Moreover, we have submitted the code to GitHub.(Link:<A href="https://github.com/chengchingwen/iGEM/blob/master/crawler.py" style="color:#44E287;">https://github.com/chengchingwen/iGEM/blob/master/crawler.py</A>)</P></DIV></DIV><DIV class="modelingPart"><H2 class="content-1" id="titleD" style="color:#6EFFDB">IV. Step 2: Filter</H2><DIV class="modelingPartContent" id="partD"><P class="content">After crawling the data, we used DB Browser for SQLite software to browse and used SQL to process our Pantide database. We tried to build a filter to find out peptides suitable to use as Pantide.</P><P class="content">According to the previous articles, we knew that around 90% of spider venom toxin peptides contain ICK structure which is the most important domain that reacts with the voltage-gated ion channels of insects and some other receptors specifically. <SUP>[2]</SUP></P><P class="content">Therefore, to find these spider venom toxin peptides from Pantide database, we could start from searching for ICK structure, whose mass is among 1-10 kDa containing at least three disulfide bonds. <SUP>[2]</SUP> So we set a filter with three conditions.</P><UL style="list-style-image:none;list-style-type:decimal;"><LI class="list">The organism we choose must be spiders or tarantulas.</LI><LI class="list">The length of the a.a. sequences are between 27 and 271 base pairs (1 kDa of protein has averagely nine amino acids, encoded by 27 base pairs)</LI><LI class="list">The number of disulfide bonds is greater or equal to 3.
After filtering with the three conditions, 113 peptides remained. Next, we set another filter to find out insecticidal peptides.</LI><LI class="list">Molecular target contains “invertebrate,” but we also remain peptides without data.
                 The reason why we keep the peptides without data was that they have the probability to be effective. In this stage, we got 63 candidates.</LI><P class="content-2">For efficacy experiment of Pantide, we choose our testee－Spodoptera litura as target insect. While there are 14 kinds of distinct Taxon in our database, including 4 Lepidoptera genus. Thus, we also set the other filter to find out peptides against Lepidoptera:</P><LI class="list">Taxon contains at least one of <I>Spodoptera litura</I>, <I>Heliothis virescens</I>, <I>Manduca sexta</I> and <I>Spodoptera exigua</I>, but we also remain peptides without data
                 On the other hand, because we designed to produce Pantide by <I>E.coli</I>, that is difficult to express proteins containing disulfide bonds. We had chosen <I>E.coli</I> Rosetta-gami strain for enhanced disulfide bond formation, but to express a protein with more than four disulfide bonds is still a heavy load. So we finally filtered out those peptides containing too much disulfide bonds.</LI><LI class="list">The number of disulfide bonds is less than or equal to four.
                 The result was that we got 46 peptides which have the possibility to use as Pantide in proof concept experiment, and all of them is targeted to insects’ voltage-gated ion channels (excluding NULL).</LI></UL></DIV></DIV><DIV class="modelingPart"><H2 class="content-1" id="titleE" style="color:#80FFDF">V. Step 3: Selection</H2><DIV class="modelingPartContent" id="partE"><P class="content">In this step, we tried to find three peptides that have different molecular target or mechanism from filtering result to do the test experiment. The method we used was to classify the remained peptides into groups by their structure similarity.</P><P class="content">We used online analytic tools on NCBI to process those peptides.</P><P class="content">We started with using Protein BLAST (Basic Local Alignment Search Tool) to search from the whole protein database for the similar query protein sequences related to all the 46 peptides and put those related peptides into groups.</P><P class="content">The next was using COBALT (Constraint-based Multiple Alignment Tool) to align the sequence between groups to find out whether or not the two groups have the similar structure while they were not got together on the last step because of side chains and other factors. At last, we separated 46 peptides into four groups, containing 27, 12, 3, 2 peptides, and two alone.</P><P class="content">Then we chose the three larger groups and used Conserved Domains Search, and found out that they belonged to the three conserved protein domain family. There are Omega-toxin Superfamily (cl05707), Toxin_28 Superfamily (cl06928) and Toxin_20 Superfamily (cl06915). The strings in brackets are unique ID of superfamilies in the conserved protein domain family database. Finally, we selected the representative peptides from each superfamily and got these three peptides, ω-hexatoxin-Hv1a, μ-segestritoxin-Sf1a and Orally active insecticidal peptide (OAIP).</P></DIV></DIV><DIV class="modelingPart"><H2 class="content-1" id="titleF" style="color:#88FCDF">VI. Future</H2><DIV class="modelingPartContent" id="partF"><P class="content">To promote the applicability of Pantide, we still need to extend our database. The next step is to integrate with other toxin peptide databases, such as scorpions or cone snails, collect more peptides’ information from research results, and even combine with bioinformatics to build a new scoring system, and search for new potential peptides.</P></DIV></DIV><DIV class="modelingPart"><H2 class="content-1" id="titleG" style="color:#9CFFE6">Reference</H2><DIV class="modelingPartContent" id="partG"><P class="reference-content">[1] King, G.F.; Gentz, M.C.; Escoubas, P.; Nicholson, G.M. A rational nomenclature for naming peptide toxins from spiders and other venomous animals. Toxicon 2008, 52, 264–276.</P><P class="reference-content">[2] Monique J. Windley, Volker Herzig, Sławomir A. Dziemborowicz, Margaret C. Hardy, Glenn F. King and Graham M. Nicholson (2012). Spider-Venom Peptides as Bioinsecticides. Toxins, 4, 191-227.</P></DIV></DIV></DIV><DIV><DIV id="modelingContainer"><P class="title" id="title2">The Pest Prediction System</P><DIV class="modelingPart"><H2 class="content-1" id="titleH" style="color:#FFFF82">I. The Factors of Prediction</H2><DIV class="modelingPartContent" id="partH"><P class="content">We built a prediction model for our device to support our spraying system. The model will predict the number of the pest, so we can know when our farm will be in the pest threat in the future. Thus we can open our spraying system spraying PANTIDE to protect our farmland. To know the number of pests in the future, we used seven weather data in the past 20 days including air pressure, the highest temperature, the lowest temperature, average temperature, humidity, precipitation, wind velocity, and also the accumulated number of bugs from the two periods including twenty to ten days ago and ten to one days ago. Because Pantide only influences on the larvae, we cannot easily predict the number of moths tomorrow or two or three days later. We needed to know the number of larvae in the future.</P></DIV></DIV><DIV class="modelingPart"><H2 class="content-1" id="titleI" style="color:#FFFF82">II. The Life History of Moth</H2><DIV class="modelingPartContent" id="partI"><P class="content">After understanding the life history of the moths from Dr. Huang in Taiwan Agriculture Research Institute, we can know that the time of pupa becoming moth ranges from six days to 14 days. So the moths we caught in 20 days must be the larvae in our farm right now. Therefore, we utilized the data of the accumulated number of moth in the next 20 days as the target of our prediction system, and we can use the output to know the time that requires spraying Pantide for prevention.</P></DIV></DIV><DIV class="modelingPart"><H2 class="content-1" id="titleJ" style="color:#FFFF82">III.The Software Design of Prediction Model</H2><DIV class="modelingPartContent" id="partJ"><P class="content">The method we used was called neural network, one of the popular machine learning solutions, or another well-known name called deep learning. The model we used was a combination of two kinds of neural network, Recurrent Neural Network (RNN) and Artificial Neural Network (ANN). Becuase we got seven features of 20 days, equals 140 features in total, we cannot simply put it into ANN to train the model. Therefore, we used RNN which was good for this kind of time series data to compress the 140 features into seven compressed data and got the seven feature and the other two feature which we have previously mentioned. Therefore, we only had nine features in total and then put them into the ANN as the input to get the final prediction. Then, it will compute the errors between the actual answer and the output and use gradient descent and backpropagation to modify the weight in each network each neuron. After a bunch of train steps, we got our model with about 80% of accuracy.</P></DIV></DIV><DIV class="modelingPart"><H2 class="content-1" id="titleK" style="color:#FFFF82">IV. The Quality of the Model</H2><DIV class="modelingPartContent" id="partK"><P class="content">The prediction model is also a part of our device system. After using the device to collect data in a farm, we can not only predict the number of moth with the model we already trained but also retrain our model with the data we collect in that farm. So, now we can modify the model according to each farm to get a specific model for it.</P><P class="content">The programming language we use is python3.5, and the deep learning framework is the one developed by Google called tendorflow.(See on<A href="https://github.com/chengchingwen/moth_prediction" style="color:#44E287;"> GitHub</A>)</P></DIV></DIV></DIV><DIV><DIV id="modelingContainer"><P class="title" id="title3">Degradation Rate</P><DIV class="modelingPart"><H2 class="content-1" id="titleL" style="color:#FFBB66">I. Summary</H2><DIV class="modelingPartContent" id="partL"><P class="content">The aim of this modeling was to predict and simulate the degradation rate of Pantide. Practically, the results would be integrated into our device to promote automatic control system. Once Pantide degraded below the effective level, it will spray the solution to replenish.</P><P class="content">After discussion and experimental verification, we show that UV radiolytic oxidation contributes to the degradation of Pantide. We tried to build a model for simulating the relationship between applied UV light and degradation rate and then combined with UV intensity sensor to calculate the period of spraying Pantide.</P></DIV></DIV><DIV class="modelingPart"><H2 class="content-1" id="titleM" style="color:#FFBB66">II. Pantide degradation process</H2><DIV class="modelingPartContent" id="partM"><P class="content">For the proteins used as Pantide, the inhibitor cystine knot (ICK) is significant to their function. Pantide with several disulfide bonds is often more stable in solution. If “native protein” is denatured, it loses disulfide bonds and becomes a less stable form, normally called “linear protein,” which is easily degradable.</P><P class="content">Recent research shows that the spider toxin proteins containing ICK structure, for example, ω-hexatoxin-Hv1a (Hv1a), have high stability against temperature, pH, solvents and protease. In contrast, when Hv1a is denatured to linear form, it loses its stability and then degrades rapidly. <SUP>[1]</SUP></P><P class="content">There are many possible processes of Pantide degradation we discussed below. (Figure 1) Pantide may have a chance to be reduced to a linear form by reductants or reductases. For both native form and linear form proteins, it may suffer hydrolysis and proteolysis, resulting in denaturing or amino acid cleavage. Also, UV light of sun also leads to Pantide degradation. Though the energy of UV light may not be not enough to break the covalent bonds efficiently, proteins still could undergo radiolytic oxidation.</P><DIV><P class="content-image" style="text-align:center !important;">Figure 1. Pantide degradation process</P></DIV><P class="content">According to this degradation process, we attempted to build up our model. The degradation rate of Pantide is contributed by hydrolysis, proteolysis, UV radiolytic oxidation and the reduction to linear form. We considered that the rate Pantide transformed to linear form is a part of degradation rate because linear form protein loses its function and it is also likely to be degraded.</P><P class="content">However, since the whole process is too complex to verify, we divided the experiment into three parts, hydrolysis test, proteolysis test and UV radiolytic oxidation test, and further summarized the results to conclude. The purpose is to find out the degradation rate of Pantide and to verify the less stable linear form protein has.</P><P class="content">On the other hand, because ICK structure domain mainly contributes the stability of these proteins; we could assume that the degradation processes of three target proteins are roughly the same, for the reason that, we chose Hv1a and Hv1a with lectin (Hv1a-lectin) to demonstrate.</P></DIV></DIV><DIV class="modelingPart"><H2 class="content-1" id="titleN" style="color:#FFBB66">III. Hydrolysis test</H2><DIV class="modelingPartContent" id="partN"><P class="content-1" style="color:#00E600"> i. Theory</P><P class="content">Like the reverse direction of polypeptide formation, proteins can be hydrolyzed into their constituent amino acids. The mechanism undergoes an E2 elimination since a nucleotide attach to sp2 hybridized acyl carbon (amide). [2] Therefore, the reaction rate depends on the concentration of both nucleotides and the protein: </P><P class="content">Where, [<I>P </I>] is the concentration of Pantide, <I>K<SUB>h</SUB></I> is the reaction constant of hydrolysis, and <I>k<SUB>A</SUB></I> , <I>k<SUB>N</SUB></I> , <I>k<SUB>B</SUB></I> are the three contributing components of overall hydorlysis which are dependent to condition.</P><P class="content"> For Pantide, we would apply the protein solution in the farm at a constant pH. Thus, the rate of hydrolysis is proportional to the concentration of Pantide by a constant <I>K<SUB>h</SUB></I> , and the concentration of Pantide undergoes an exponential decay as time goes on.</P><P class="content-1" style="color:#00E600"> ii. Experimental proof</P><P class="content">We tested the chemical stability of both native and linear types of Hv1a and Hv1a-lectin in neutral PBS solvent (phosphate buffered saline, pH=7.4) in 4 ℃; for one day and seven days. SDS-PAGE showed the remained protein concentration and calculated by software ImageJ. (Figure 2, Figure 3)</P><DIV><P class="content-image" style="text-align:center !important;">Figure 2. SDS-PAGE gel and the concentrations of hydrolysis test to Hv1a (5.3 kDa). The samples were marked on the top of the gel.</P></DIV><DIV><P class="content-image" style="text-align:center !important;">Figure 3. SDS-PAGE gel and the concentrations of hydrolysis test to Hv1a-lectin (HL, 17.1 kDa). The samples were marked on the top of the gel.</P></DIV><P class="content">The difference of the concentration of native Hv1a was not significant during seven days (1 day: 98%, seven days: 87%), while linear Hv1a only remained 9% after one day, and almost all degraded after seven days.</P><P class="content">The test to Hv1a-lectin showed the similar result. Native Hv1a-lectin nearly did not degrade in 7 days (1 day: 110% , 7 days: 105%), but linear Hv1a-lectin remained 16% after 1 day, 3% after 7 days.</P><P class="content">So, we could conclude that native proteins dissolved in neutral PBS solvent did not undergo hydrolysis (or at a very slow rate) in 4℃ for seven days, on the other side, linear form proteins degrade as time went on, and remained merely little after seven days.</P></DIV></DIV><DIV class="modelingPart"><H2 class="content-1" id="titleO" style="color:#FFBB66">IV. Proteolysis test</H2><DIV class="modelingPartContent" id="partO"><P class="content-1" style="color:#00E600"> i. Theory</P><P class="content">To simulate the degradation by protease, we used Michaelis-Menten kinetics model to express the rate equation. It is useful to describe enzymatic reactions, especially the one without ligand participation, such as proteolysis with proteases whose mechanism is first to find out the specific active site and then react to break it down. [3]</P><P class="content">The proteolysis process is irreversible in vivo because the product, linear form protein, is degraded quickly. We used differential equation to describe the major proteolysis process:</P><P class="content">Where [<I>P </I>]: the concentation of Pantide, <I>V<SUB>m,p </SUB></I> : the maximum reaction rate of proteolysis, which is equal to the product of the concentration of total enzyme and  turnover number <I>k<SUB>cat </SUB></I> of specific protease and substrate, <I>K<SUB>M,p </SUB></I>: Michaelis constant, which is the substrate concentration at which the reaction rate is half of <I>V<SUB>m,p </SUB></I></P><P class="content">We had assumed that the <I>V<SUB>m,p </SUB></I> is a constant because the proteases in vivo are in active and in equilibrium, and <I>V<SUB>m,p </SUB></I> represents the equivalent value for all kinds of protease in environment.</P><P class="content-1" style="color:#00E600"> ii. Experimental proof</P><P class="content">We designed two experiments to test the enzymatic stability towards protease of Hv1a and Hv1a-lectin. One was to observe the degrade level of both native and linear types of proteins applied by protease for one day, and the other one was to obtain the curve of the degradation rate of only linear form proteins in the period of four hours, because of the resistance against the protease.</P><P class="content">For the first experiment, we dissolved the protein solutions in neutral PBS (pH=7.5) solvent, applied with 0.25% trypsin-EDTA(1:250), a serine protease, and then incubated the samples in work temperature 37℃ for one day. (Figure 4, Figure 5)</P><DIV><P class="content-image" style="text-align:center !important;">Figure 4. SDS-PAGE gel and the concentrations of trypsin resistance test to Hv1a (5.3 kDa). The samples were marked on the top of the gel.</P></DIV><DIV><P class="content-image" style="text-align:center !important;">Figure 5. SDS-PAGE gel and the concentrations of trypsin resistance test to Hv1a-lectin (HL, 17.1 kDa). The samples were marked on the top of the gel.</P></DIV><P class="content">Compared with the sample incubated for one day without trypsin treatment, native Hv1a and Hv1a-lectin showed the resistance to trypsin protease (111% and 100%), but linear proteins were degraded by proteolysis (67% and 18%).</P><P class="content">We next tested the proteolysis rate of only linear proteins in the period of four hours, and we drew the Lineweaver–Burk plot (also called “double reciprocal plot”) (Figure 6) to obtain <I>V<SUB>m,p </SUB></I> and <I>K<SUB>M,p </SUB></I> for linear form Hv1a and Hv1a-lectin. (Table 1)</P><DIV><P class="content-image" style="text-align:center !important;">Figure 6. Lineweaver–Burk plot of proteolysis test of linear Hv1a (blue line) and Hv1a-lectin (orange line). The horizontal axis represents the reciprocal of substrate concentration, and the longitudinal axis represents the reciprocal of rate. The straight line’s x-intercept means the reciprocal of <I>-K<SUB>M,p </SUB></I>, and y-intercept means the reciprocal of <I>V<SUB>m,p</SUB></I>.</P></DIV><DIV><P class="content-image" style="text-align:center !important;">Table 1. The<I>V<SUB>m,p </SUB></I> and <I>K<SUB>M,p </SUB></I> of linear form Hv1a and Hv1a-lectin.</P></DIV></DIV></DIV><DIV class="modelingPart"><H2 class="content-1" id="titleP" style="color:#FFBB66">V. UV radiolytic oxidation test</H2><DIV class="modelingPartContent" id="partP"><P class="content-1" style="color:#00E600"> i. Theory</P><P class="content">In nature, proteins are also probable to degrade under the sunlight, and the reason is that the energy of light excites the solvent (almost water) and initially generate radicals, which afterward propagate and attack protein to break it down until termination.</P><P class="content">Ionizing radiation causes the radiolysis of water is, the major process is shown below. (Figure 7)</P><DIV><P class="content-image" style="text-align:center !important;">Figure 7. The radiolysis process of water.</P></DIV><P class="content">Though the real mechanism that radicals attack to protein is quite complex, we can simply indicate the rate of protein been attacked by an unknown power, n, of the total concentration of radicals. Then we derived the rate formula as differential equations.</P><P class="content"> Where [<I>radical </I>]: the total concentation of radicals, [<I>P </I>]: the concentation of Pantide, <I>G<SUB>γ </SUB></I>: the dose rate of absorbing γ-ray (radiation energy absorption rate per mass, for water, 1.42 Gy/s) [5], <I>A<SUB>γ </SUB></I>: the number of radicals created per energy (for water, 0.045 μmol/J) [5], <I>I </I>: the intensity of UVB from sunlight measured by UV Sensor (UVM30A), <I>R<SUB>T </SUB></I>: the rate constant of radicals termination, which is equal to 2.365×10<SUP>-7 </SUP>mol<SUP>-1 </SUP>s<SUP>-1 </SUP> [6], <I>K<SUB>UV </SUB></I>: the rate const ant of UV radiolytic oxidation to protiens, which is set to 44 mol<SUP>-1 </SUP>s<SUP>-1 </SUP>at the beginning [6]</P><P class="content">We then used software MATLAB to simulate the degradation rate by UV radiolytic oxidation on the intensity of UVB from sunlight. The results showed that the degradation rate increases as rising intensity whatever n is, and eventually it tends to be fully degraded. (Figure 8)</P><DIV><P class="content-image" style="text-align:center !important;">Figure 8. The simulation of degradation rate by UV radiolytic oxidation on the intensity of UVB from sunlight.</P></DIV><P class="content-1" style="color:#00E600"> ii. Experimental proof</P><P class="content">We applied the four kinds of native protein solutions to the UVB light from UV transilluminator (302 nm, 50 mW/m<SUP>2 </SUP>) in the period of 2 hours, where the environment temperature was 36.8℃ in average. The results showed the six proteins degraded under UVB light treatment and the tendency of the reduction of proteins corresponded to our model. (Figure 9)</P><DIV><P class="content-image" style="text-align:center !important;">Figure 9. The degradation rate of four proteins by UV radiolytic oxidation as time goes on.</P></DIV><P class="content">The degradation rates are obviously different between the protein with and without fusing to lectin. The possible reason we concerned was the difference in amino length; the longer protein has a relatively high probability to be attacked.</P><P class="content">To determine the relation between the UVB light intensity and the degradation, we fit the result from experiments to our model. We calculated the parameter n in formula (4) must be equal to 0.78. The rate constant <I>K<SUB>UV </SUB></I> for Hv1a, Sf1a, OAIP 7.8 15.7and Hv1a-lectin are 2.4, 7.8, 15.7 and 90.3.</P><P class="content">We also applied native Hv1a-lectin with another UV transilluminator (286 nm, 36.4 mW/m<SUP>2 </SUP>), and compared with the prediction from our model. (Figure 10)</P><DIV><P class="content-image" style="text-align:center !important;">Figure 10. The model prediction compared with the experiment data.</P></DIV><P class="content">The figure showed our model is on the right way, but to accomplish our purpose; we still need more experiment data to correct the parameters.</P></DIV></DIV><DIV class="modelingPart"><H2 class="content-1" id="titleQ" style="color:#FFBB66">VI. Summarization</H2><DIV class="modelingPartContent" id="partQ"><P class="content">The whole equation of degradation rate could be expressed by the summation of the rates of three possible degrade processes, that is hydrolysis, proteolysis, and UV radiolytic oxidation, and the rate that proteins transfer from native form to linear form indicated as <I>R<SUB>SS </SUB></I></P><P class="content">However, according to the previous experiments, if we only considered about the protein in native form, because of the high chemical stability and protease resistance, <I>K<SUB>h </SUB></I> and <I>V<SUB>m,p </SUB></I> is much smaller than <I>K<SUB>UV </SUB></I>, and the first two terms on the right side of the equal sign is relative insignificant. As for the reduction of disulfide bonds, since proteins are most stable in their favorable dimensional structure, it does not tend to break this strong bond down, so we assumed that <I>R<SUB>SS </SUB></I> is not contributed a lot for degradation in the nature. Then the equation was simplified to only one term.</P><P class="content">and</P><P class="content">As results, the degradation rate of Pantide mainly related to UV intensity expressed by (6) and (7). To verify our degradation rate model, and the practical use of our device, we had done the UV radiolytic oxidation test outdoor.</P><P class="content">We put the samples on a wide square in a transparent and closed acrylic box outdoor for 4 hours at a different time in a day. Combined with UV intensity sensor, we got the remained protein concentration with the average UVB light intensity in each period. (Figure 11)</P><DIV><P class="content-image" style="text-align:center !important;">Figure 11. The remained protein concentration and the average UVB light intensity in each 4 hours at a different time in a day.</P></DIV></DIV></DIV><DIV class="modelingPart"><H2 class="content-1" id="titleR" style="color:#FFBB66">Reference</H2><DIV class="modelingPartContent" id="partR"><P class="reference-content">[1] Volker Herzig and Glenn F. King (2015). The Cystine Knot Is Responsible for the Exceptional Stability of the Insecticidal Spider Toxin ω-Hexatoxin-Hv1a. Toxins, 7, 4366-4380.</P><P class="reference-content">[2] Anonymous. (n.d.). HYDROLYSIS. Retrieved October 16, 2016 , from https://zh.scribd.com/document/79207692/Hydrolysis-2006</P><P class="reference-content">[3] Hedstrom, L. (2002, May 14). Serine Protease Mechanism and Specificity. Chem. Rev. 2002, 102, 4501-4523.</P><P class="reference-content">[4] Guozhong Xu &amp; Mark R. Chance (2005). Radiolytic Modification of Sulfur-Containing Amino Acid Residues in Model Peptides: Fundamental Studies for Protein Footprinting. Anal. Chem, 77, 2437-2449.</P><P class="reference-content">[5] Mary E. Dzaugis, Arthur J. Spivack, Steven D'Hondt (2015, April 10). A quantitative model of water radiolysis and chemical production rates near radionuclide-containing solids. Radiation Physics and Chemistry, 115, 127-134.</P><P class="reference-content">[6] Bachari, T. S. (n.d.). Theoretical Investigation on The Kinetics of Free Radical Reactions of Styrene Emulsion Polymerization . Retrieved from http://www.iasj.net/iasj?func=fulltext&amp;aId=13996</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></SECTION><DIV id="index_footer"><DIV class="conent"><UL id="index_foot_list" style="list-style-image: none;list-style-type:none;"><LI><A href="https://2016.igem.org/Team:NCTU_Formosa" style="font-size:16px;color:#FFF;"><B>Home</B></A></LI></UL><LI><A href="https://2016.igem.org/Team:NCTU_Formosa/Project" style="font-size:16px;color:#FFF;"><B>Project</B></A></LI><LI><A href="https://2016.igem.org/Team:NCTU_Formosa/Description">Description</A></LI><LI><A href="https://2016.igem.org/Team:NCTU_Formosa/Design">Design</A></LI><LI><A href="https://2016.igem.org/Team:NCTU_Formosa/Demonstrate">Device</A></LI><LI><A href="https://2016.igem.org/Team:NCTU_Formosa/Proof">Results</A></LI><LI><A href="https://2016.igem.org/Team:NCTU_Formosa/Modeling">Modeling</A></LI><LI><A href="https://2016.igem.org/Team:NCTU_Formosa/Safety">Safety</A></LI><LI><A href="https://2016.igem.org/Team:NCTU_Formosa/Human_Practices" style="font-size:16px;color:#FFF;"><B>Human Practices</B></A></LI><LI><A href="https://2016.igem.org/Team:NCTU_Formosa/Integrated_Practices">Study</A></LI><LI><A href="https://2016.igem.org/Team:NCTU_Formosa/Collaborations">Collaborations</A></LI><LI><A href="https://2016.igem.org/Team:NCTU_Formosa/Engagement">Education</A></LI><LI><A href="https://2016.igem.org/Team:NCTU_Formosa/Meetups">Meetups</A></LI><LI><A href="https://2016.igem.org/Team:NCTU_Formosa/Further_Considerations">Further Considerations</A></LI><LI><A href="https://2016.igem.org/Team:NCTU_Formosa/Entrepreneurship" style="font-size:16px;color:#FFF;"><B>Entrepreneurship</B></A></LI><LI><A href="https://2016.igem.org/Team:NCTU_Formosa/Entrepreneurship#Businessmodel">Business Model</A></LI><LI><A href="https://2016.igem.org/Team:NCTU_Formosa/Entrepreneurship#MarketingFuture">Marketing &amp; Future</A></LI><LI><A href="https://2016.igem.org/Team:NCTU_Formosa/Entrepreneurship#RegulationLaw">Law &amp; Regulations</A></LI><LI><A href="https://2016.igem.org/Team:NCTU_Formosa/Entrepreneurship#Business_competition">Business Competition</A></LI><LI><A href="https://2016.igem.org/Team:NCTU_Formosa/Entrepreneurship#ourSponsor">Sponsor</A></LI><LI><B>Parts</B></LI><LI><A href="https://2016.igem.org/Team:NCTU_Formosa/Basic_Part">Basic Parts</A></LI><LI><A href="https://2016.igem.org/Team:NCTU_Formosa/Composite_Part">Composite Parts</A></LI><LI><B>Notebook</B></LI><LI><A href="https://2016.igem.org/Team:NCTU_Formosa/Notebook">Lab Note</A></LI><LI><A href="https://2016.igem.org/Team:NCTU_Formosa/Protocol">Protocol</A></LI><LI><A href="https://2016.igem.org/Team:NCTU_Formosa/Attributions" style="font-size:16px;color:#FFF;"><B>Team</B></A></LI><LI><A href="https://2016.igem.org/Team:NCTU_Formosa/Achievement" style="font-size:16px;color:#FFF;"><B>Achievement</B></A></LI><LI><A href="https://2016.igem.org/Team:NCTU_Formosa/HP/Gold">HP Gold</A></LI><LI><A href="https://2016.igem.org/Team:NCTU_Formosa/HO/Silver">HP Silver</A></LI></DIV><DIV class="footer_logo">NCTU_Formosa
    	<A href="http://www.nctu.edu.tw/" style="text-decoration:none;color:#999;" target="_blank">National Chiao Tung University</A>
      Engineering Building 6 EF455, 1001 University Road, Hsinchu 300, Taiwan, ROC. 
      </DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>