"
Page
Discussion
View source
History
teams
Log in
 
Team:Macquarie Australia/Notebook
From 2010.igem.org
Home
Team
Project
Parts Submitted to the Registry
Glossary
Human practice
Notebook 1: Agrobacterium Tumefaciens
Notebook 2: Deinococcus Radiodurans
Notebook 3: Cloning
Protocols and Other Methods
Safety
About Us
References
Just for fun!
Acknowledgements
Official Team Profile
PROJECT LAB BOOK
Welcome to the Macquarie University project lab book page!
Here you will find a day-by-day account of our triumphs and failures.
A day-by-day progress for Agrobacterium Tumefaciens Bacteriophytochrome
20th August 2010
Genomic DNA extraction
A. tumefaciens genomic DNA extracted using the Invitrogen PureLinkTM Genomic DNA Purification kit as per the manufacturer’s protocols.
The extractions were run on a GelRed stained 1% agarose gel and photo taken for visualization (see below).
A NanoDrop spectrophotometer reading was also recorded to check the quality of the extracted genomic DNA.
The extraction was successful for all A. tumefaciens cell lysate samples (labeled DNA1.1, DNA1.2, DNA2.1, DNA2.2 (See figure below)
A. tumefaciens genomic DNA extraction agarose results:
All four DNA samples show a smear of gDNA. Because the samples were treated with RNase there is no band indicative of RNA visible = SUCCESS!
Figure 1.  Results of A. tumefaciens genomic DNA extraction
Figure 1. GelRed stained 1% agarose gel of genomic DNA extraction from A. tumefaciens.
In lane 1 there is a 1kb ladder. In lane 2 is the DNA1.1 flow through, lane 3 is the DNA1.2 flow through, lane 4 is the DNA2.1 flow through and lane 5 is the DNA2.2 flowthrough.  All four samples show a smear that is indicative of genomic DNA. The extraction has been successful.
Nanodrop absorbance readings:
Genomic DNA sample
260/280 OD ratio
Concentration (ng/mL)
DNA1.1
1.88
351.5
DNA1.2
1.98
203.1
DNA2.1
2.14
738.9
DNA2.2
2.16
709.7
Overall, the Nanodrop readings show that we have obtained good DNA concentration with minimal protein contamination
= SUCCESS!
27th August 2010
Primer design
Various primers were designed manually and using Primer 3 Software package for PCR amplification.
The primers were ordered and supplied through Integrated DNA Technologies.
There was an array of various primers ordered for amplification of different products. The details of the primers are described below.
Fwd and Rvs primers for amplification of the full length A. tumefaciens BphP gene:
Primer name
Primer Sequence
(AT-FWD-1)
5’-ATG AGT TCA CAT ACG CCG-3’
(AT-RVS-1)
5’-TCA GGC AAT TTT TTC CTC-3’
Fwd and Rvs primers for amplification of the full length A. tumefaciens BphP gene for insertion in the operon BEFORE the HO gene:
Primer name
Primer Sequence
(AT-BHO-F)
5’- AAG GAG ATA TAC ATA TGA TGA GTT CAC ATA CGC CG – 3’
(AT-BHO-R)
5’- AAG TTG ACA CTC ATA TGA GCC CTC CTT TCA GGC – 3’
Fwd and Rvs primers for amplification of the full length A. tumefaciens BphP gene for insertion in the operon AFTER the HO gene in the operon:
Primer name
Primer Sequence
(AT-AHO-F)
5’- CCG AAG GCT AGG ATC CAG GAG GGC TGC TAT GAG – 3’
(AT-AHO-R)
5’- GTT AGC CGG ATC CTC AGG CAA TTT TTT CCT – 3’
Fwd and Rvs primers for amplification of the full length A. tumefaciens BphP gene for insertion in the operon AFTER the HO gene as well as the addition of a ribosome binding site or Shine Delgano sequence:
Primer name
Primer Sequence
(AT-FWD-RBS)
5’- AGG AGG GCT ATG AGT TCA CAT ACG CCG -3’
Initial PCR
The reaction mastermix for the initial PCR was set up as per the following recipe (per sample):
Mastermix:
Amount per sample (ul)
Gibco H2O
13.75
10x Buffer
2.00
Polymerase enzyme
0.25
dNTP
1.00
Fwd primer
1.00
Rvs primer
1.00
Genomic DNA
1.00
Total
20.00
The PCR program was set up as per the following:
94˚C for 2 minutes
94˚C for 30 seconds
60˚C for 30 seconds
72˚C for 2 minutes & 30 seconds
(This was repeated for another 25 cycles)
72˚C for 10 minutes
4˚C to end.
Different combinations of the primers were used for the PCR reaction (see below ‘Experimental Design’ section following)
Not all possible primer combinations were used due to limitations on the amount of polymerase enzyme available to us
The PCR products were run on a 1.2% GelRed post-stained agarose gel for visualisation (see picture of gel below).
Experimental Design – Primer combinations:
Template DNA
Fwd Primer
Rvs primer
DNA1.2
(AT-FWD-1)
(AT-RVS-1)
DNA2.2
(AT-FWD-1)
(AT-RVS-1)
DNA1.2
(AT-BHO-F)
(AT-RVS-1)
DNA2.2
(AT-BHO-F)
(AT-RVS-1)
DNA1.2
(AT-FWD-RBS)
(AT-RVS-1)
DNA2.2
(AT-FWD-RBS)
(AT-RVS-1)
SSH20 (negative control)
(AT-FWD-1)
(AT-RVS-1)
A band was seen in lane 9. This lane was using the primer and DNA template combinations of DNA2.2, (AT-FWD-RBS) and (AT-RVS-1) primers
= SUCCESS!!!
Figure 2. Results of the initial PCR
Figure 2. GelRed post-stained 1.2% agarose gel of initial PCR of A. tumefaciens using various primers.
In lanes 1 and 11 there is a 1kb ladder. In lane 2 is the DNA1.2 template with (AT-FWD-1) and (AT-RVS-1)primer pair.
In lane 3 is the DNA2.2 template with (AT-FWD-1) and (AT-RVS-1)primer pair. In lane 4 is the DNA1.2 template with (AT-BHO-F) and (AT-RVS-1)primer pair. In lane 5 is the DNA2.2 template with the (AT-BHO-F) and (AT-RVS-1)primer pair. In lanes 6 and 7 there is nothing loaded as the wells were damaged. In lane 8 there is the DNA1.2 template with (AT-FWD-RBS) and (AT-RVS-1)primer pair. In lane 9 there is the DNA2.2 template with the (AT-FWD-RBS)and (AT-RVS-1)primer pair and in lane 10 there is the ssH2O negative control. A product is seen in lane 9 – this is the DNA2.2 template with the (AT-FWD-RBS)and (AT-RVS-1)primers! This means that we have a A. tumefaciens bacteriophytochrome product with a ribosome binding sight inserted.
30th August 2010
PCR Optimization
Now that we have a product obtained by the initial PCR it is time to optimize the PCR conditions using the successful DNA2.2 PCR product obtained from last week and the original DNA2.2 template
The reaction mastermix for the PCR was set up as per the following recipe (per sample):
Mastermix:
Amount per sample (ul)
Gibco H2O
13.75
10x Buffer
2.00
Polymerase enzyme
0.25
dNTP
1.00
Fwd primer
1.00
Rvs primer
1.00
Genomic DNA
1.00
Total
20.00
The PCR program was set up as per the following:
94˚C for 2 minutes
94˚C for 30 seconds
60˚C for 30 seconds
72˚C for 2 minutes & 30 seconds
(This was repeated for another 25 cycles)
72˚C for 10 minutes
4˚C to end.
Again, different combinations of the primers were used for the PCR reaction (see below ‘Experimental Design’ section following)
Now that we don’t have a limited enzyme supply, even more primer combinations can be used!!
The PCR products were run on a GelRed post-stained 2% agarose gel using a 100bp ladder for visualization
All primer combinations tested worked with bands visible in each lane! The strange band pattern seen in lane 6 is most probably due to non-specific binding.
Experimental Design – Primer combinations:
Template DNA
Fwd Primer
Rvs primer
DNA2.2
(AT-FWD-1)
(AT-RVS-1)
PCR product (from DNA2.2)
(AT-FWD-1)
(AT-RVS-1)
DNA2.2
(AT-BHO-F)
(AT-RVS-1)
PCR product (from DNA2.2)
(AT-BHO-F)
(AT-RVS-1)
DNA2.2
(AT-FWD-RBS)
(AT-RVS-1)
PCR product (from DNA2.2)
(AT-FWD-RBS)
(AT-RVS-1)
DNA2.2
(AT-FWD-1)
(AT-AHO-R)
PCR product (from DNA2.2)
(AT-FWD-1)
(AT-AHO-R)
DNA2.2
(AT-BHO-F)
(AT-AHO-R)
PCR product (from DNA2.2)
(AT-BHO-F)
(AT-AHO-R)
DNA2.2
(AT-FWD-RBS)
(AT-AHO-R)
PCR product (from DNA2.2)
(AT-FWD-RBS)
(AT-AHO-R)
ssH2O (-) control
(AT-FWD-1)
(AT-RVS-1)
Figure 3. PCR optimization results
Figure 3. GelRed post-stained 2% agarose gel of optimized PCR of A. tumefaciens bacteriophytochrome using various primers. In lanes 1 and 15 there is a 100bp ladder. In lane 2 is the DNA2.2 sample with (AT-FWD-1) and (AT-RVS-1)primer pair. In lane 3 there is PCR product template (from DNA2.2 sample) with (AT-FWD-1)and (AT-RVS-1)primer pair. In lane 4 there is DNA sample with (AT-BHO-F) and (AT-RVS-1)primer pair. In lane 5 is the PCR product (from DNA2.2 sample) with (AT-BHO-F)and (AT-RVS-1)primer pair. In lane 6 there is DNA2.2 sample with (AT-FWD-RBS)and (AT-RVS-1)primer pair. In lane 7 there is PCR product template (from DNA2.2 sample) with (AT-FWD-RBS)and (AT-RVS-1)primer pair. In lane 8 there is DNA2.2 sample with (AT-FWD-1) and (AT-AHO-R)primer pairs. In lane 9 there is PCR product template (from DNA2.2 sample) with (AT-FWD-1) and (AT-AHO-R)primer pairs. In lane 10 there is DNA2.2 sample with (AT-BHO-F) and (AT-AHO-R)primer pairs. In lane 11 there is PCR product (from DNA2.2 sample) with (AT-BHO-F)and (AT-AHO-R)primer pair. In lane 12 there is DNA2.2 sample with (AT-BHO-F)and (AT-AHO-R)primer pair. In lane 13 there is PCR product (from DNA2.2 sample) with (AT-FWD-RBS)and (AT-AHO-R)primer pair. In lane 14 there is ssH2O negative control with (AT-FWD-1)and (AT-RVS-1)primer pair. All the lanes worked. There is a weird result in lane 7 which is probably due to non-specific binding.  = SUCCESS!
3rd September 2010
PCR Optimization (using Gradient PCR)
Now that we have the A. tumefaciens bacteriophytochrome product as well as the RBS inserted it is now time for us to amplify the bacteriophytochrome product with the heme oxygenase gene
We will be inserting the two genes in two different orientations in the operon
We will achieve this using two different primer pairs
The first primer pair is (AT-BHO-F) with (AT-BHO-R) [this will insert the bacteriophytchrome gene BEFORE the heme oxygenase gene in the operon]
The second primer pair is (AT-AHO-F) with (AT-AHO-R) [this will insert the bacteriophytochrome gene AFTER the heme oxygenase gene in the operon]
A technique called gradient PCR will be used here. This PCR includes different annealing temperatures so that the optimum annealing temperature for the primers can be determined.
This should also result in reduced non-specific binding that was observed in the previous PCR result
The reaction mastermix for the PCR was set up as per the following recipe (per sample):
Mastermix:
Amount per sample (ul)
Gibco H2O
13.75
10x Buffer
2.00
Polymerase enzyme
0.25
dNTP
1.00
Fwd primer
1.00
Rvs primer
1.00
Genomic DNA
1.00
Total
20.00
The PCR program was set up as per the following:
94˚C for 2 minutes
94˚C for 30 seconds
60˚C for 30 seconds
72˚C for 2 minutes & 30 seconds
(This was repeated for another 25 cycles)
72˚C for 10 minutes
4˚C to end.
The PCR products were run on a GelRed post-stained 2% agarose gel using a 1kb ladder for visualization
The products were run on a GelRed post-stained 2% agarose gel for visualization
No products were observed
Experimental Design – Primer combinations and annealing temperatures:
Fwd Primer
Rvs primer
Temp 1 (Degrees Celsius)
Temp 2 (Degrees Celsius)
Temp 3 (Degrees Celsius)
Temp 4 (Degrees Celsius)
Temp 5 (Degrees Celsius)
(AT-BHO-F)
(AT-BHO-R)
57.1
58.7
60.6
63.4
64.8
(AT-AHO-F)
(AT-AHO-R)
57.1
58.7
60.6
63.4
64.8
Figure 4. PCR Optimisation (gradient PCR) results
Figure 4. No product amplification is seen in any lanes. The anticipated product is approximately 1.6 to 3.0kb. The gel had been over run and at the very bottom some bands can be seen but as these are so small they are possibly dimers that are less than 300bp.
No products were observed. The anticipated product size was between 1.6kb and 3kb. The gel was over run but the only products that were over run were probably primer dimers, which are less than 300bp in size.  = FAIL!
7th September 2010
PCR Optimization (using Gradient PCR)
The PCR run on 3rd September wasn’t successful so this was repeated
This time however, only the PCR product was used as a template in two different dilutions (1:100 and 1:200)
Additionally, two different annealing temperatures were used: 60 and 65C
The primer pairs used in the previous PCR were also used again for the insertion of the bacteriophytochrome gene and heme oxygenase in different orientations in the operon
The reaction mastermix for the PCR was set up as per the following recipe (per sample):
Mastermix:
Amount per sample (ul)
Gibco H2O
13.75
10x Buffer
2.00
Polymerase enzyme
0.25
dNTP
1.00
Fwd primer
1.00
Rvs primer
1.00
Genomic DNA
1.00
Total
20.00
The PCR program was set up as per the following:
94˚C for 2 minutes
94˚C for 30 seconds
60˚C for 30 seconds
72˚C for 2 minutes & 30 seconds
(This was repeated for another 25 cycles)
72˚C for 10 minutes
4˚C to end.
The PCR products were run on a GelRed post-stained 2% agarose gel using a 1kb ladder for visualization
There was no amplification observed in any of the lanes so the picture of this gel is not included = FAIL!
Experimental Design – Primer combinations and annealing temperatures:
DNA template
Dilution
Fwd primer
Rvs primer
Annealing temp (Degrees Celsius)
PCR product (from DNA2.2)
1:100
(AT-BHO-F)
(AT-BHO-R)
60
PCR product (from DNA2.2)
1:200
(AT-BHO-F)
(AT-BHO-R)
60
PCR product (from DNA2.2)
1:100
(AT-AHO-F)
(AT-AHO-R)
60
PCR product (from DNA2.2)
1:200
(AT-AHO-F)
(AT-AHO-R)
60
PCR product (from DNA2.2)
1:100
(AT-BHO-F)
(AT-BHO-R)
65
PCR product (from DNA2.2)
1:200
(AT-BHO-F)
(AT-BHO-R)
65
PCR product (from DNA2.2)
1:100
(AT-AHO-F)
(AT-AHO-R)
65
PCR product (from DNA2.2)
1:200
(AT-AHO-F)
(AT-AHO-R)
65
10th September 2010
PCR Optimization (repeated)
As we were having trouble with one of the primers annealing we attempted a PCR technique that required a different PCR program to be set
The (AT-AHO-F)– (AT-AHO-R)primer pair were run on the normal PCR program being used for all other PCR’s as these primers were annealing properly
The primer pair that we were having difficulty with ((AT-BHO-F)-(AT-BHO-R)) required a special PCR program. This would allow for the (AT-BHO-R)primer to anneal to the template first for single stranded amplification and then the (AT-BHO-F)primer to bind later.
The reaction mastermix for the PCR was set up as per the following recipe (per sample):
Mastermix:
Amount per sample (ul)
Gibco H2O
13.75
10x Buffer
2.00
Polymerase enzyme
0.25
dNTP
1.00
Fwd primer
1.00
Rvs primer
1.00
Genomic DNA
1.00
Total
20.00
The first PCR program was set up as per the following:
94˚C for 2 minutes
94˚C for 30 seconds
60˚C for 30 seconds
72˚C for 2 minutes & 30 seconds
(This was repeated for another 35 cycles)
72˚C for 10 minutes
4˚C to end.
The second PCR program was set up as per the following (this was for the amplification of product using the (AT-FWD-RBS) and (AT-AHO-R) primer pair):
94˚C for 2 minutes
94˚C for 30 seconds
40˚C for 30 seconds
72˚C for 2 minutes & 30 seconds
(This was repeated for another 35 cycles)
72˚C for 10 minutes
4˚C to end.
The second PCR program was set up as per the following. There were two parts to this PCR to allow for the (AT-BHO-F) primer to anneal properly after initial annealing of the (AT-BHO-R) primer.
94˚C for 2 minutes
94˚C for 30 seconds
40˚C for 30 seconds
72˚C for 2 minutes and 30 seconds
(This was repeated for another 4 cycles)
Now we can add the (AT-BHO-F) primer to the reaction.
4˚C for 5 minutes
94˚C for 30 seconds
60˚C for 30 seconds
72˚C for 2 minutes and 30 seconds
(This was repeated for another 31 cycles)
72˚C for 10 minutes
4˚C to end
The PCR products were run on a GelRed post-stained 2% agarose gel using a 1kb ladder for visualization
Experimental Design – Primer combinations and annealing temperatures:
Template
Primer pair
Annealing temp (degrees Celsius)
Number of cycles
(AT-FWD-RBS)-(AT-RVS-1)PCR product
(AT-BHO-F)-(AT-BHO-R)
40, then 60
4, then 35
(AT-FWD-RBS)-(AT-AHO-R)PCR product
(AT-AHO-F)-(AT-AHO-R)
60
35
(AT-FWD-RBS)-(AT-RVS-1)PCR product (1:10 dilution)
(AT-BHO-F)-(AT-BHO-R)
40, then 60
4 then 35
(AT-FWD-RBS)-(AT-AHO-R)PCR product (1:10 dilution)
(AT-AHO-F)-(AT-AHO-R)
60
35
Figure 5. PCR optimization (with ssPCR amplification for (AT-BHO-F)primer)
Figure 5. GelRed post-stained 2% agarose gel of optimized PCR using primer pairs for insertion of the heme oxygenase gene and bacteriophytochrome gene. In lanes 1 and 6 there is a 1kb ladder. In lane 2 there is the (AT-FWD-RBS)-(AT-RVS-1)PCR product amplified with (AT-BHO-F)-(AT-BHO-R)primer pair on the special PCR program allowing for the (AT-BHO-R)primer to bind first. In lane 3 there is the (AT-FWD-RBS)-(AT-AHO-R)PCR product amplified with (AT-AHO-F)-(AT-AHO-R)primer pair on the normal PCR program. In lane 4 there is the (AT-FWD-RBS)-(AT-RVS-1)PCR product diluted 1:10 amplified with (AT-BHO-F)-(AT-BHO-R)primer pair on the special PCR program allowing for the (AT-BHO-R)primer to bind first. In lane 5 there is the (AT-FWD-RBS)-(AT-AHO-R) PCR product diluted 1:10 amplified with (AT-AHO-F)-(AT-AHO-R)primer pair on the normal PCR program. There is a product band seen in lane 3 – SUCCESS! This is the product with the bacteriophytochrome gene inserted AFTER the heme oxygenase gene.
Discussion of gradient PCR:
The gradient PCR was used because the (AT-BHO-R) primer that was ordered was 12 base pairs shorter than what was supposed to be ordered. The (AT-BHO-R)primer was supposed to include the whole reverse primer sequence (18 base pairs) as well as an additional sequence for the HO site. This caused an issue with this primer annealing in previous PCR reactions. The gradient PCR will allow for this primer to anneal prior to any other primer annealing increasing the chance of annealing.
Retrieved from " http://2010.igem.org/Team:Macquarie_Australia/Notebook "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
