"
Page
Discussion
View source
History
teams
Log in
 
Team:Macquarie Australia/Notebook2
From 2010.igem.org
Home
Team
Project
Parts Submitted to the Registry
Glossary
Human practice
Notebook 1: Agrobacterium Tumefaciens
Notebook 2: Deinococcus Radiodurans
Notebook 3: Cloning
Protocols and Other Methods
Safety
About Us
References
Just for fun!
Acknowledgements
Official Team Profile
PROJECT LAB BOOK
Welcome to the Macquarie University project lab book page!
A day-by-day progress for Deinococcus Radiodurans Bacteriophytochrome Operon Construct
16th September 2010
Genomic DNA extraction
D. radiodurans genomic DNA extracted using the BioLine Genomic DNA Extraction kit as per the manufacturer’s protocols.
The extractions were run on a GelRed stained 1% agarose gel and photo taken for visualization (see figure 6).
A NanoDrop spectrophotometer reading was also recorded to check the quality of the extracted genomic DNA.
The extraction was successful for all D. radiodurans cell lysate samples (labeled DEINO1, DEINO2, DEINO3 (See figure below)
D. radiodurans genomic DNA extraction agarose results:
All three DNA samples show a smear of gDNA. Because the samples were treated with RNase there is no band indicative of RNA visible = SUCCESS!
Figure 6.  Results of D. radiodurans genomic DNA extraction
Figure 6. GelRed post-stained 1% agarose gel of genomic DNA extraction from D. radiodurans . In lanes 1 and 5 there is a 1kb ladder. In lane 2 is the DEINO1 sample, lane 3 is the DEINO2 sample, lane 4 is the DEINO3 sample.  All three samples show a smear that is indicative of genomic DNA.  The extraction has been successful!!!
Nanodrop absorbance readings:
Genomic DNA sample
260/280 OD ratio
Concentration (ng/mL)
DEINO1
2.43
1.443
DEINO2
2.55
51.8
DEINO3
2.99
88.7
Overall, the Nanodrop readings show that we have obtained good DNA concentration with minimal protein contamination
= SUCCESS!
27th August 2010
Primer design
Various primers were designed manually and using Primer 3 Software package for PCR amplification.
The primers were ordered and supplied through Integrated DNA Technologies.
There was an array of various primers ordered for amplification of different products. The details of the primers are described below.
Fwd and Rvs primers for amplification of the full length D. radiodurans BphP gene:
Primer name
Primer Sequence
(DR-FWD-1)
5’-ATG AGC CGG GAC CCG TTG -3’
(DR-RVS-1)
5’-TCA GGC ATC GGC GGC TCC -3’
Fwd and Rvs primers for amplification of the full length D. radiodurans BphP gene for insertion in the operon BEFORE the HO gene:
Primer name
Primer Sequence
(DR-BHO-F)
5’- AAG GAG ATA TAC ATA TGA TGA GCC GGG ACC CGT TG – 3’
(DR-BHO-R)
5’- AAG TTG ACA CTC ATA TGA GCA GCC CTC CTT CAG GC – 3’
Fwd and Rvs primers for amplification of the full length D. radiodurans BphP gene for insertion in the operon AFTER the HO gene in the operon:
Primer name
Primer Sequence
(DR-AHO-F)
5’- CCG AAG GCT AGG ATC CAG GAG GGC TGC TAT GAG C – 3’
(DR-AHO-R)
5’- GTT AGC AGC CGG ATC CTC AGG CAT GGG CGG CTC C – 3’
Fwd and Rvs primers for amplification of the full length D. radiodurans BphP gene for insertion in the operon AFTER the HO gene as well as the addition of a ribosome binding site or Shine Delgano sequence:
Primer name
Primer Sequence
(DR-FWD-RBS)
5’- AGG AGG GCT GCT ATG AGC CGG GAC CCG TTG -3’
22nd September 2010
Initial PCR (gradient PCR)
The reaction mastermix for the PCR was set up as per the following recipe (per sample):
Mastermix:
Amount per sample (ul)
Gibco H2O
13.75
10x Buffer
2.00
Polymerase enzyme
0.25
dNTP
1.00
Fwd primer
1.00
Rvs primer
1.00
Genomic DNA
1.00
Total
20.00
The PCR program was set up as per the following:
94˚C for 2 minutes
94˚C for 30 seconds
55-65˚C for 30 seconds
72˚C for 2 minutes & 30 seconds
(This was repeated for another 25 cycles)
72˚C for 5 minutes
4˚C to end.
Different combinations of the primers were used for the initial PCR reaction (see below ‘Experimental Design’ section following)
Not all possible primer combinations were used
The PCR products were run on a 2% GelRed post-stained agarose gel for visualisation (see picture of gel below).
Experimental Design – Primer combinations and annealing temperatures:
DNA template
Dilution
Fwd primer
Rvs primer
Annealing temp (Degrees Celsius)
DEINO1
1:20
(DR-FWD)
(DR-RVS)
55, 57, 60, 62, 65
DEINO2
1:20
(DR-FWD)
(DR-RVS)
55, 57, 60, 62, 65
DEINO1
1:20
(DR-FWD-RBS)
(DR-RVS)
55, 57, 60, 62, 65
DEINO2
1:20
(DR-FWD-RBS)
(DR-RVS)
55, 57, 60, 62, 65
DEINO1
1:20
(DR-FWD)
(DR-AHO-R)
55, 57, 60, 62, 65
DEINO2
1:20
(DR-FWD)
(DR-AHO-R)
55, 57, 60, 62, 65
DEINO1
1:20
(DR-FWD-RBS)
(DR-AHO-R)
55, 57, 60, 62, 65
DEINO2
1:20
(DR-FWD-RBS)
(DR-AHO-R)
55, 57, 60, 62, 65
DEINO1
1:50
(DR-FWD)
(DR-RVS)
55, 57, 60, 62, 65
DEINO2
1:50
(DR-FWD)
(DR-RVS)
55, 57, 60, 62, 65
DEINO1
1:50
(DR-FWD-RBS)
(DR-RVS)
55, 57, 60, 62, 65
DEINO2
1:50
(DR-FWD-RBS)
(DR-RVS)
55, 57, 60, 62, 65
DEINO1
1:50
(DR-FWD)
(DR-AHO-R)
55, 57, 60, 62, 65
DEINO2
1:50
(DR-FWD)
(DR-AHO-R)
55, 57, 60, 62, 65
DEINO1
1:50
(DR-FWD-RBS)
(DR-AHO-R)
55, 57, 60, 62, 65
DEINO2
1:50
(DR-FWD-RBS)
(DR-AHO-R)
55, 57, 60, 62, 65
(-) control
-
(DR-FWD)
(DR-RVS)
55, 57, 60, 62, 65
All 85 reactions were unsuccessful and no bands were visible on the gel = BACK TO THE DRAWING BOARD!
27th September 2010
PCR with FailSafeTM PCR system
As the 85 reactions that were run were unsuccessful we decided to purchase a commercial PCR system called the FailSafeTM System from EpiCentre Biotechnologies
It was speculated that the amplification didn’t work due to either the enzyme / buffer or the combination. It didn’t seem to be the annealing temperatures because a range of temperatures were tested in the previous PCR
The FailSafeTM PCR System includes 12 different enzyme / buffer combinations so we decided to try this with two different annealing temperatures (56˚C and 60˚C)
The PCR program was set up as per the following:
94˚C for 2 minutes
94˚C for 30 seconds
56˚C or 60˚C for 30 seconds
72˚C for 3 minutes
(This was repeated for another 25 cycles)
72˚C for 10 minutes
4˚C to end.
The PCR products were run on a GelRed stained 2% agarose gel using a 1kb ladder for visualization (see figure 2 below)
We have successfully amplified the D. radiodurans bacteriophytochrome gene! SUCCESS!
Experimental Design – Primer combinations:
Template DNA
Dilution
Fwd Primer
Rvs primer
FailSafe system
DEINO1
1:100
DR-FWD-1
DR-RVS-1
(All 12 FailSafe premixed combinations)
DEINO2
1:100
DR-FWD-1
DR-RVS-1
(All 12 FailSafe premixed combinations)
Figure 7. FailSafe PCRTM System results:
Figure 7. GelRed post-stained 2% agarose gel of FailSafeTM PCR System. In lanes 1, 15, 16 and 30 there is a 1kb ladder. In lanes 2 to 29 is the D. radiodurans DNA template amplified with the (DR-FWD-1) and (DR-RVS-1) primer pair using the 12 different premixed enzyme / buffer combinations from the FailSafeTM PCR System. The top lanes (2-15) have a 56˚C annealing temperature. The bottom rows have a 60˚C annealing temperature. It is obvious by looking at the gel that there are differences in the non-specific binding patterns using the different temperature, enzyme and buffer combinations.
8th October 2010
PCR (with FailSafeTM PCR System and DR-FWD-RBS primer)
We used the FailSafeTM PCR System to amplify the D. radiodurans bacteriophytochrome with the additional RBS inserted (using DR-FWD-RBS primer)
The most successful enzyme/buffer combinations (J & K) from the previous PCR were used with an annealing temperature of 60C for this amplification
The PCR program was set up as per the following:
94˚C for 2 minutes
94˚C for 30 seconds
60˚C for 30 seconds
72˚C for 3 minutes
(This was repeated for another 25 cycles)
72˚C for 10 minutes
4˚C to end.
The PCR products were run on a GelRed stained 2% agarose gel using a 1kb ladder for visualization (see figure 3 below)
We have successfully amplified the D. radiodurans bacteriophytochrome gene with the additional RBS site! SUCCESS!
Figure 8: PCR results using DR-FWD-RBS primer with FailSafe PCR system:
Figure 8. GelRed post-stained 2% agarose gel of FailSafeTM PCR System. In lanes 1 and 5 there is a 1kb ladder. In lane 2 there is the XXX PCR product amplified with the (DR-FWD-RBS) and (DR-RVS-1) primer pair using buffer J from the FailSafeTM PCR System. In lane 3 there is the XXX PCR product amplified with the (DR-FWD-RBS) and (DR-RVS-1) primer pair using buffer K from the FailSafeTM PCR System.
9th October 2010
PCR with FailSafeTM PCR System and (DR-AHO-F) and (DR-AHO-R) primer pair
We used the FailSafeTM PCR System to amplify the D. radiodurans bacteriophytochrome with the additional RBS product obtained from the previous PCR using the (DR-AHO-F) and (DR-AHO-R) primer pair to insert the HO site
Again, the most successful enzyme/buffer combinations (J & K) from the previous PCR were used with an annealing temperature of 60˚C for this amplification
The PCR program was set up as per the following:
94˚C for 2 minutes
94˚C for 30 seconds
60˚C for 30 seconds
72˚C for 3 minutes
(This was repeated for another 35 cycles)
72˚C for 10 minutes
4˚C to end.
The PCR products were run on a GelRed stained 2% agarose gel using a 1kb ladder for visualization (see figure 3 below)
We have successfully amplified the D. radiodurans bacteriophytochrome gene with the additional RBS site as well as the HO site! SUCCESS!
We are ready for cloning!
Figure 9: PCR results of bacteriophytochrome and RBS template using (DR-AHO-F) and (DR-AHO-R) primer pair with FailSafeTM PCR System:
Figure 9. GelRed post-stained 2% agarose gel of FailSafeTM PCR System. In lanes 1 and 5 there is a 1kb ladder. In lane 2 there is the 1st PCR product (bacteriophytochrome and RBS) amplified with the (DR-AHO-F) and (DR-AHO-R) primer pair using buffer J from the FailSafeTM PCR System. In lane 3 there is the 2nd PCR product (bacteriophytochrome and RBS) amplified with the (DR-AHO-F) and (DR-AHO-R) primer pair using buffer K from the FailSafeTM PCR System.  We have successfully amplified the D. radiodurans bacteriophytochrome gene with both the RBS and HO site inserted. This is now ready for cloning!!
Retrieved from " http://2010.igem.org/Team:Macquarie_Australia/Notebook2 "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
