<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Macquarie_Australia_Notebook2"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Macquarie_Australia/Notebook2">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Macquarie_Australia/Notebook2&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Macquarie_Australia/Notebook2&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Macquarie_Australia/Notebook2&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Macquarie_Australia/Notebook2" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Macquarie Australia/Notebook2</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><DIV id="sidebar"><UL id="navigation"><LI><A href="https://2010.igem.org/Team:Macquarie_Australia">Home</A></LI><LI><A href="https://2010.igem.org/Team:Macquarie_Australia/Team">Team</A></LI><LI><A href="https://2010.igem.org/Team:Macquarie_Australia/Project">Project</A></LI><LI><A href="https://2010.igem.org/Team:Macquarie_Australia/Parts">Parts Submitted to the Registry</A></LI><LI><A href="https://2010.igem.org/Team:Macquarie_Australia/Glossary">Glossary</A></LI><LI><A href="https://2010.igem.org/Team:Macquarie_Australia/humanpractice">Human practice</A></LI><LI><A href="https://2010.igem.org/Team:Macquarie_Australia/Notebook">Notebook 1: <I>Agrobacterium Tumefaciens</I></A></LI><LI><A href="https://2010.igem.org/Team:Macquarie_Australia/Notebook2">Notebook 2: <I>Deinococcus Radiodurans</I></A></LI><LI><A href="https://2010.igem.org/Team:Macquarie_Australia/Notebook3">Notebook 3: Cloning 
</A></LI><LI><A href="https://2010.igem.org/Team:Macquarie_Australia/Protocols and Other Methods">Protocols and Other Methods</A></LI><LI><A href="https://2010.igem.org/Team:Macquarie_Australia/Safety">Safety</A></LI><LI><A href="https://2010.igem.org/Team:Macquarie_Australia/aboutus">About Us</A></LI><LI><A href="https://2010.igem.org/Team:Macquarie_Australia/References">References</A></LI><LI><A href="https://2010.igem.org/Team:Macquarie_Australia/Just for fun!">Just for fun!</A></LI><LI><A href="https://2010.igem.org/Team:Macquarie_Australia/Acknowledgements">Acknowledgements</A></LI><LI><A href="https://igem.org/Team.cgi?year=2010">Official Team Profile</A></LI></UL></DIV><DIV id="body"><CENTER><H1><FONT color="#47484c">PROJECT LAB BOOK</FONT></H1><H1><FONT color="#47484c">Welcome to the Macquarie University project lab book page!</FONT></H1><H2><FONT color="#47484c"><CENTER>A day-by-day progress for  <I>Deinococcus Radiodurans </I> Bacteriophytochrome Operon Construct </CENTER></FONT></H2></CENTER><P><BIG><B>

16th September 2010 </B></BIG></P><P>

Genomic DNA extraction </P><P><LI type="disc"><I> D. radiodurans </I>genomic DNA extracted using the BioLine Genomic DNA Extraction kit as per the manufacturer’s protocols.</LI><LI type="disc"> 
The extractions were run on a GelRed stained 1% agarose gel and photo taken for visualization (see figure 6).</LI><LI type="disc"> 
A NanoDrop spectrophotometer reading was also recorded to check the quality of the extracted genomic DNA.</LI><LI type="disc"> 
The extraction was successful for all <I>D. radiodurans </I>cell lysate samples (labeled DEINO1, DEINO2, DEINO3 (See figure below) </LI></P><H3><I><B>D. radiodurans  </B></I><B> genomic DNA extraction agarose results: </B></H3>

All three DNA samples show a smear of gDNA. Because the samples were treated with RNase there is no band indicative of RNA visible = SUCCESS!  <H3> Figure 6.  Results of <I> D. radiodurans </I> genomic DNA extraction  </H3><P><B>Figure 6. </B> GelRed post-stained 1% agarose gel of genomic DNA extraction from <I> D. radiodurans </I>. In lanes 1 and 5 there is a 1kb ladder. In lane 2 is the DEINO1 sample, lane 3 is the DEINO2 sample, lane 4 is the DEINO3 sample.  All three samples show a smear that is indicative of genomic DNA.  The extraction has been successful!!!</P><H4>Nanodrop absorbance readings: </H4><CENTER><TABLE><TBODY><TR><TH>Genomic DNA sample</TH><TH>260/280 OD ratio</TH><TH>Concentration (ng/mL)</TH></TR><TR><TD>DEINO1</TD><TD>2.43</TD><TD>1.443</TD></TR><TR><TD>DEINO2</TD><TD>2.55</TD><TD>51.8</TD></TR><TR><TD>DEINO3</TD><TD>2.99</TD><TD>88.7</TD></TR></TBODY></TABLE></CENTER><P> 
Overall, the Nanodrop readings show that we have obtained good DNA concentration with minimal protein contamination </P><P>= <U>SUCCESS! </U></P><B>
27th August 2010 <P>

Primer design </P></B><LI type="disc"> 
Various primers were designed manually and using Primer 3 Software package for PCR amplification. </LI><P><LI type="disc"> The primers were ordered and supplied through Integrated DNA Technologies.</LI></P><P><LI type="disc"> There was an array of various primers ordered for amplification of different products. The details of the primers are described below.
</LI></P><H4>Fwd and Rvs primers for amplification of the full length <I> D. radiodurans </I> BphP gene: </H4><CENTER><TABLE><TBODY><TR><TH>Primer name</TH><TH>Primer Sequence</TH></TR><TR><TD> (DR-FWD-1)</TD><TD>5’-ATG AGC CGG GAC CCG TTG -3’</TD></TR><TR><TD>(DR-RVS-1)</TD><TD>5’-TCA GGC ATC GGC GGC TCC -3’</TD></TR></TBODY></TABLE></CENTER><H4>Fwd and Rvs primers for amplification of the full length <I> D. radiodurans  </I>BphP gene for insertion in the operon <U>BEFORE</U> the HO gene:
</H4><CENTER><TABLE><TBODY><TR><TH>Primer name</TH><TH>Primer Sequence</TH></TR><TR><TD>(DR-BHO-F)</TD><TD>5’- AAG GAG ATA TAC ATA TGA TGA GCC GGG ACC CGT TG – 3’</TD></TR><TR><TD>(DR-BHO-R)</TD><TD> 5’- AAG TTG ACA CTC ATA TGA GCA GCC CTC CTT CAG GC – 3’</TD></TR></TBODY></TABLE></CENTER><H4> 
Fwd and Rvs primers for amplification of the full length <I>D. radiodurans  </I> BphP gene for insertion in the operon <U>AFTER </U>the HO gene in the operon:

</H4><CENTER><TABLE><TBODY><TR><TH>Primer name</TH><TH>Primer Sequence</TH></TR><TR><TD>(DR-AHO-F)</TD><TD>5’- CCG AAG GCT AGG ATC CAG GAG GGC TGC TAT GAG C – 3’</TD></TR><TR><TD> (DR-AHO-R)</TD><TD> 5’- GTT AGC AGC CGG ATC CTC AGG CAT GGG CGG CTC C – 3’ </TD></TR></TBODY></TABLE></CENTER><H4> 
Fwd and Rvs primers for amplification of the full length <I>D. radiodurans   </I> BphP gene for insertion in the operon <U>AFTER </U>the HO gene as well as the addition of a ribosome binding site or Shine Delgano sequence:

</H4><CENTER><TABLE><TBODY><TR><TH>Primer name</TH><TH>Primer Sequence</TH></TR><TR><TD>(DR-FWD-RBS)</TD><TD>  5’- AGG AGG GCT GCT ATG AGC CGG GAC CCG TTG -3’</TD></TR></TBODY></TABLE></CENTER><BIG><B>


22nd September 2010  <P>

Initial PCR (gradient PCR)</P></B></BIG><LI type="disc"> The reaction mastermix for the PCR was set up as per the following recipe (per sample): </LI><CENTER><TABLE><TBODY><TR><TH>Mastermix:</TH><TH>Amount per sample (ul)</TH></TR><TR><TD>Gibco H2O</TD><TD> 13.75</TD></TR><TR><TD>10x Buffer</TD><TD> 2.00</TD></TR><TR><TD>Polymerase enzyme</TD><TD> 0.25</TD></TR><TR><TD>dNTP</TD><TD> 1.00</TD></TR><TR><TD>Fwd primer </TD><TD> 1.00</TD></TR><TR><TD>Rvs primer </TD><TD> 1.00</TD></TR><TR><TD>Genomic DNA</TD><TD> 1.00</TD></TR><TR><TD><B>Total	</B></TD><TD><B>20.00</B></TD></TR></TBODY></TABLE></CENTER><H4> The PCR program was set up as per the following: </H4><OL><LI> 94˚C for 2 minutes </LI><LI>94˚C for 30 seconds </LI><LI> 55-65˚C for 30 seconds </LI><LI>72˚C for 2 minutes &amp; 30 seconds</LI><P>

(This was repeated for another 25 cycles)

<LI> 72˚C for 5 minutes </LI><LI> 4˚C to end. </LI></P></OL><P><LI type="disc">

Different combinations of the primers were used for the initial PCR reaction (see below ‘Experimental Design’ section following) </LI><LI type="disc">
Not all possible primer combinations were used </LI><LI type="disc">
The PCR products were run on a 2% GelRed post-stained agarose gel for visualisation (see picture of gel below). </LI></P><H4>Experimental Design – Primer combinations and annealing temperatures:  </H4><CENTER><TABLE><TBODY><TR><TH> DNA template </TH><TH> Dilution </TH><TH> Fwd primer </TH><TH> Rvs primer </TH><TH> Annealing temp (Degrees Celsius) </TH></TR><TR><TD> DEINO1 </TD><TD> 1:20</TD><TD> (DR-FWD)</TD><TD> (DR-RVS)</TD><TD> 55, 57, 60, 62, 65 </TD></TR><TR><TD> DEINO2 </TD><TD> 1:20</TD><TD> (DR-FWD)</TD><TD> (DR-RVS)</TD><TD> 55, 57, 60, 62, 65</TD></TR><TR><TD> DEINO1 </TD><TD> 1:20</TD><TD> (DR-FWD-RBS) </TD><TD> (DR-RVS) </TD><TD> 55, 57, 60, 62, 65 </TD></TR><TR><TD> DEINO2 </TD><TD> 1:20</TD><TD> (DR-FWD-RBS) </TD><TD> (DR-RVS) </TD><TD> 55, 57, 60, 62, 65 </TD></TR><TR><TD> DEINO1 </TD><TD> 1:20</TD><TD> (DR-FWD)		</TD><TD> (DR-AHO-R) </TD><TD> 55, 57, 60, 62, 65 </TD></TR><TR><TD> DEINO2 </TD><TD> 1:20</TD><TD> (DR-FWD)		</TD><TD> (DR-AHO-R) </TD><TD> 55, 57, 60, 62, 65 </TD></TR><TR><TD> DEINO1 </TD><TD> 1:20</TD><TD> (DR-FWD-RBS)		</TD><TD> (DR-AHO-R) </TD><TD> 55, 57, 60, 62, 65  </TD></TR><TR><TD> DEINO2 </TD><TD>1:20</TD><TD> (DR-FWD-RBS)		</TD><TD> (DR-AHO-R)</TD><TD> 55, 57, 60, 62, 65  </TD></TR><TR><TD> DEINO1 </TD><TD>1:50</TD><TD> (DR-FWD)		</TD><TD> (DR-RVS)</TD><TD> 55, 57, 60, 62, 65  </TD></TR><TR><TD> DEINO2 </TD><TD>1:50</TD><TD> (DR-FWD)		 </TD><TD> (DR-RVS)</TD><TD> 55, 57, 60, 62, 65</TD></TR><TR><TD> DEINO1 </TD><TD>1:50</TD><TD> (DR-FWD-RBS)		</TD><TD> (DR-RVS)</TD><TD> 55, 57, 60, 62, 65  </TD></TR><TR><TD> DEINO2 </TD><TD>1:50</TD><TD> (DR-FWD-RBS)		</TD><TD> (DR-RVS)</TD><TD> 55, 57, 60, 62, 65  </TD></TR><TR><TD> DEINO1 </TD><TD>1:50</TD><TD> (DR-FWD)		</TD><TD> (DR-AHO-R)</TD><TD> 55, 57, 60, 62, 65  </TD></TR><TR><TD> DEINO2 </TD><TD>1:50</TD><TD> (DR-FWD)		</TD><TD> (DR-AHO-R)</TD><TD> 55, 57, 60, 62, 65  </TD></TR><TR><TD> DEINO1 </TD><TD>1:50</TD><TD> (DR-FWD-RBS)		</TD><TD> (DR-AHO-R)</TD><TD> 55, 57, 60, 62, 65  </TD></TR><TR><TD> DEINO2 </TD><TD>1:50</TD><TD> (DR-FWD-RBS)		</TD><TD> (DR-AHO-R)</TD><TD> 55, 57, 60, 62, 65  </TD></TR><TR><TD> (-) control </TD><TD>-</TD><TD> (DR-FWD)		</TD><TD> (DR-RVS)</TD><TD> 55, 57, 60, 62, 65  </TD></TR></TBODY></TABLE></CENTER><P><LI type="disc">All 85 reactions were unsuccessful and no bands were visible on the gel = BACK TO THE DRAWING BOARD! </LI></P><BIG><B>
27th September 2010 <P>

PCR with FailSafeTM PCR system</P></B></BIG><LI type="disc">As the 85 reactions that were run were unsuccessful we decided to purchase a commercial PCR system called the FailSafeTM System from EpiCentre Biotechnologies </LI><LI type="disc">It was speculated that the amplification didn’t work due to either the enzyme / buffer or the combination. It didn’t seem to be the annealing temperatures because a range of temperatures were tested in the previous PCR </LI><LI type="disc">The FailSafeTM PCR System includes 12 different enzyme / buffer combinations so we decided to try this with two different annealing temperatures (56˚C and 60˚C) </LI><H4> The PCR program was set up as per the following: </H4><OL><LI> 94˚C for 2 minutes </LI><LI>94˚C for 30 seconds </LI><LI> 56˚C or 60˚C for 30 seconds </LI><LI>72˚C for 3 minutes </LI><P>

(This was repeated for another 25 cycles)

<LI> 72˚C for 10 minutes </LI><LI> 4˚C to end. </LI></P></OL><P><LI type="disc"> 
 
The PCR products were run on a GelRed stained 2% agarose gel using a 1kb ladder for visualization (see figure 2 below)</LI><LI type="disc"> 
We have successfully amplified the D. radiodurans bacteriophytochrome gene! SUCCESS! </LI></P><H4>Experimental Design – Primer combinations: </H4><CENTER><TABLE><TBODY><TR><TH>Template DNA</TH><TH>Dilution</TH><TH>Fwd Primer</TH><TH>Rvs primer</TH><TH>FailSafe system</TH></TR><TR><TD>DEINO1</TD><TD> 1:100 </TD><TD> DR-FWD-1</TD><TD> DR-RVS-1</TD><TD> (All 12 FailSafe premixed combinations)</TD></TR><TR><TD>DEINO2</TD><TD> 1:100</TD><TD> DR-FWD-1</TD><TD> DR-RVS-1</TD><TD> (All 12 FailSafe premixed combinations)</TD></TR></TBODY></TABLE></CENTER><H3><B> Figure 7. FailSafe PCRTM System results: </B></H3><P><B>Figure 7. </B> GelRed post-stained 2% agarose gel of FailSafeTM PCR System. In lanes 1, 15, 16 and 30 there is a 1kb ladder. In lanes 2 to 29 is the <I> D. radiodurans </I> DNA template amplified with the (DR-FWD-1) and (DR-RVS-1) primer pair using the 12 different premixed enzyme / buffer combinations from the FailSafeTM PCR System. The top lanes (2-15) have a 56˚C annealing temperature. The bottom rows have a 60˚C annealing temperature. It is obvious by looking at the gel that there are differences in the non-specific binding patterns using the different temperature, enzyme and buffer combinations.  </P><BIG><B>
8th October 2010  <P>

PCR (with FailSafeTM PCR System and DR-FWD-RBS primer)</P></B></BIG><LI type="disc">We used the FailSafeTM PCR System to amplify the D. radiodurans bacteriophytochrome with the additional RBS inserted (using DR-FWD-RBS primer)</LI><LI type="disc">The most successful enzyme/buffer combinations (J &amp; K) from the previous PCR were used with an annealing temperature of 60C for this amplification </LI><H4> The PCR program was set up as per the following: </H4><OL><LI> 94˚C for 2 minutes </LI><LI>94˚C for 30 seconds </LI><LI> 60˚C for 30 seconds </LI><LI>72˚C for 3 minutes </LI><P>

(This was repeated for another 25 cycles)

<LI> 72˚C for 10 minutes </LI><LI> 4˚C to end. </LI></P></OL><P><LI type="disc">

The PCR products were run on a GelRed stained 2% agarose gel using a 1kb ladder for visualization (see figure 3 below) </LI><LI type="disc">We have successfully amplified the D. radiodurans bacteriophytochrome gene with the additional RBS site! SUCCESS! </LI></P><H3><B> Figure 8: PCR results using DR-FWD-RBS primer with FailSafe PCR system: </B></H3><P><B>Figure 8. </B> GelRed post-stained 2% agarose gel of FailSafeTM PCR System. In lanes 1 and 5 there is a 1kb ladder. In lane 2 there is the XXX PCR product amplified with the (DR-FWD-RBS) and (DR-RVS-1) primer pair using buffer J from the FailSafeTM PCR System. In lane 3 there is the XXX PCR product amplified with the (DR-FWD-RBS) and (DR-RVS-1) primer pair using buffer K from the FailSafeTM PCR System.   </P><BIG><B>
9th October 2010  <P>

PCR with FailSafeTM PCR System and (DR-AHO-F) and (DR-AHO-R) primer pair</P></B></BIG><LI type="disc">We used the FailSafeTM PCR System to amplify the D. radiodurans bacteriophytochrome with the additional RBS product obtained from the previous PCR using the (DR-AHO-F) and (DR-AHO-R) primer pair to insert the HO site </LI><LI type="disc">Again, the most successful enzyme/buffer combinations (J &amp; K) from the previous PCR were used with an annealing temperature of 60˚C for this amplification </LI><H4> The PCR program was set up as per the following: </H4><OL><LI> 94˚C for 2 minutes </LI><LI>94˚C for 30 seconds </LI><LI> 60˚C for 30 seconds </LI><LI>72˚C for 3 minutes</LI><P>

(This was repeated for another 35 cycles)

<LI> 72˚C for 10 minutes </LI><LI> 4˚C to end. </LI></P></OL><LI type="disc"> 
The PCR products were run on a GelRed stained 2% agarose gel using a 1kb ladder for visualization (see figure 3 below) </LI><LI type="disc"> 
We have successfully amplified the D. radiodurans bacteriophytochrome gene with the additional RBS site as well as the HO site! SUCCESS! </LI><LI type="disc"> 
We are ready for cloning! </LI><H3><B> Figure 9: PCR results of bacteriophytochrome and RBS template using (DR-AHO-F) and (DR-AHO-R) primer pair with FailSafeTM PCR System: </B></H3><P><B>Figure 9. </B> GelRed post-stained 2% agarose gel of FailSafeTM PCR System. In lanes 1 and 5 there is a 1kb ladder. In lane 2 there is the 1st PCR product (bacteriophytochrome and RBS) amplified with the (DR-AHO-F) and (DR-AHO-R) primer pair using buffer J from the FailSafeTM PCR System. In lane 3 there is the 2nd PCR product (bacteriophytochrome and RBS) amplified with the (DR-AHO-F) and (DR-AHO-R) primer pair using buffer K from the FailSafeTM PCR System.  We have successfully amplified the D. radiodurans bacteriophytochrome gene with both the RBS and HO site inserted. This is now ready for cloning!!   </P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:Macquarie_Australia/Notebook2">http://2010.igem.org/Team:Macquarie_Australia/Notebook2</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Macquarie_Australia/Notebook2" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Macquarie_Australia/Notebook2" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Macquarie_Australia/Notebook2&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Macquarie_Australia/Notebook2&amp;oldid=175373" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></BODY></HTML>