<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Warsaw_Modelling_Structural"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:Warsaw/Modelling/Structural">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Warsaw/Modelling/Structural&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Warsaw/Modelling/Structural&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Warsaw/Modelling/Structural&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Warsaw/Modelling/Structural" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Warsaw/Modelling/Structural</H1><DIV id="bodyContent"><H3 id="siteSub">From 2009.igem.org</H3><DIV id="war_menu"><DIV class="war_menu_frame"><OL><LI class="main"><A href="/Team:Warsaw" title="Team:Warsaw">Home page</A></LI><P><LI class="main" onclick="switchMenu('team_submenu','menu_item')"><A name="team" style="cursor: hand; cursor: pointer;">The Team</A></LI></P><DIV id="team_submenu" style="display: none;"><LI class="sub"><A href="/Team:Warsaw/Team" title="Team:Warsaw/Team">About us</A></LI><LI class="sub"><A href="/Team:Warsaw/Team/Supervisors" title="Team:Warsaw/Team/Supervisors">Supervisors</A></LI><LI class="sub"><A href="/Team:Warsaw/Team/Miecznikowa" title="Team:Warsaw/Team/Miecznikowa">The M. Group</A></LI><LI class="sub"><A href="/Team:Warsaw/Team/Pawinskiego" title="Team:Warsaw/Team/Pawinskiego">The P. Group</A></LI></DIV><P><LI class="main" onclick="switchMenu('project_submenu')"><A name="project" style="cursor: hand; cursor: pointer;">The Project</A></LI></P><DIV id="project_submenu" style="display: none;"><LI class="sub"><A href="/Team:Warsaw/Project/introduction" title="Team:Warsaw/Project/introduction">Introduction</A></LI><LI class="sub"><A href="/Team:Warsaw/Project/invasion" title="Team:Warsaw/Project/invasion">Invasiveness</A></LI><LI class="sub"><A href="/Team:Warsaw/Project/endosome" title="Team:Warsaw/Project/endosome">Endosome Module</A></LI><LI class="sub"><A href="/Team:Warsaw/Project/cytoplasm" title="Team:Warsaw/Project/cytoplasm">Cytoplasmatic Module</A></LI><LI class="sub"><A href="/Team:Warsaw/Project/future" title="Team:Warsaw/Project/future">Future plans</A></LI><LI class="sub"><A href="/Team:Warsaw/Bibliography" title="Team:Warsaw/Bibliography">Bibliography</A></LI></DIV><LI class="main"><A href="/Team:Warsaw/results" title="Team:Warsaw/results">Results</A></LI><LI class="main" onclick="switchMenu('modelling_submenu')"><A name="modelling" style="cursor: hand; cursor: pointer;">Modelling</A></LI><DIV id="modelling_submenu" style="display: block"><LI class="sub"><A href="/Team:Warsaw/Modelling" title="Team:Warsaw/Modelling">System Overview</A></LI><LI class="sub"><A href="/Team:Warsaw/Modelling/Apoptosis" title="Team:Warsaw/Modelling/Apoptosis">Apoptosis</A></LI><LI class="sub"><STRONG class="selflink">Structural</STRONG></LI><LI class="sub"><A href="/Team:Warsaw/Modelling/Gene_Networks" title="Team:Warsaw/Modelling/Gene Networks">Gene Networks</A></LI><LI class="sub"><A href="/Team:Warsaw/Modelling/Invasiveness" title="Team:Warsaw/Modelling/Invasiveness">Invasiveness</A></LI></DIV><P><LI class="main" onclick="switchMenu('lab_submenu')"><A name="lab" style="cursor: hand; cursor: pointer;">Lab Work</A></LI></P><DIV id="lab_submenu" style="display: none"><LI class="sub"><A href="/Team:Warsaw/Notebook" title="Team:Warsaw/Notebook">Notebook</A></LI><LI class="sub"><A href="/Team:Warsaw/protocols" title="Team:Warsaw/protocols">Protocols</A></LI><LI class="sub"><A href="/Team:Warsaw/Resources" title="Team:Warsaw/Resources">Resources</A></LI></DIV><LI class="main"><A href="/Team:Warsaw/Parts" title="Team:Warsaw/Parts">Submitted Parts</A></LI><LI class="main"><A href="/Team:Warsaw/Glossary" title="Team:Warsaw/Glossary">Glossary</A></LI><LI class="main"><A href="/Team:Warsaw/Biosafety" title="Team:Warsaw/Biosafety">Biosafety</A></LI><LI class="main"><A href="/Team:Warsaw/Human" title="Team:Warsaw/Human">Human Practices</A></LI><LI class="main"><A href="/Team:Warsaw/Acknowledgements" title="Team:Warsaw/Acknowledgements">Acknowledgements</A></LI></OL></DIV><DIV class="war_menu_frame_bottom"> </DIV></DIV><DIV id="war_content"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1"><A href="#Introduction"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Introduction</SPAN></A></LI><LI class="toclevel-1"><A href="#Fundamental_basis"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Fundamental basis</SPAN></A></LI><LI class="toclevel-2"><A href="#Protein_folding"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Protein folding</SPAN></A></LI><LI class="toclevel-2"><A href="#Fusion_proteins"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">Fusion proteins</SPAN></A></LI></UL><LI class="toclevel-1"><A href="#Methods"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Methods</SPAN></A></LI><LI class="toclevel-2"><A href="#Computation"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">Computation</SPAN></A></LI><LI class="toclevel-3"><A href="#BioInfoBank_Meta_Server"><SPAN class="tocnumber">3.1.1</SPAN><SPAN class="toctext">BioInfoBank Meta Server</SPAN></A></LI><LI class="toclevel-3"><A href="#TASSER"><SPAN class="tocnumber">3.1.2</SPAN><SPAN class="toctext">TASSER</SPAN></A></LI><LI class="toclevel-3"><A href="#Robetta"><SPAN class="tocnumber">3.1.3</SPAN><SPAN class="toctext">Robetta</SPAN></A></LI><LI class="toclevel-3"><A href="#The_ModFOLD_Model_Quality_Assessment_Server"><SPAN class="tocnumber">3.1.4</SPAN><SPAN class="toctext">The ModFOLD Model Quality Assessment Server</SPAN></A></LI><LI class="toclevel-2"><A href="#Evaluation"><SPAN class="tocnumber">3.2</SPAN><SPAN class="toctext">Evaluation</SPAN></A></LI><LI class="toclevel-1"><A href="#Results"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Results</SPAN></A></LI><LI class="toclevel-2"><A href="#2D-predictions"><SPAN class="tocnumber">4.1</SPAN><SPAN class="toctext">2D-predictions</SPAN></A></LI><LI class="toclevel-2"><A href="#Full_models"><SPAN class="tocnumber">4.2</SPAN><SPAN class="toctext">Full models</SPAN></A></LI><LI class="toclevel-3"><A href="#Secretion_peptid"><SPAN class="tocnumber">4.2.1</SPAN><SPAN class="toctext">Secretion peptid</SPAN></A></LI><LI class="toclevel-3"><A href="#Bax_with_secretion_domain_secreted"><SPAN class="tocnumber">4.2.2</SPAN><SPAN class="toctext">Bax with secretion domain secreted</SPAN></A></LI><LI class="toclevel-3"><A href="#p53_wih_both_signal_domains"><SPAN class="tocnumber">4.2.3</SPAN><SPAN class="toctext">p53 wih both signal domains</SPAN></A></LI><LI class="toclevel-3"><A href="#Listeriolysin_with_secretion_domain"><SPAN class="tocnumber">4.2.4</SPAN><SPAN class="toctext">Listeriolysin with secretion domain</SPAN></A></LI><LI class="toclevel-3"><A href="#Invasin"><SPAN class="tocnumber">4.2.5</SPAN><SPAN class="toctext">Invasin</SPAN></A></LI><LI class="toclevel-3"><A href="#PhoP_and_PhoQ"><SPAN class="tocnumber">4.2.6</SPAN><SPAN class="toctext">PhoP and PhoQ</SPAN></A></LI></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Introduction">Introduction</SPAN></H2><DIV class="important">Because the native conformation of secretion peptide from hemolysin A is not determined we decided to use several computational structure prediction methods to find the three-dimensional structure of this domain. Additionally we attempt to obtain the theoretical models of proapoptotic fusion proteins and naturally occuring proteins which are used in our project.</DIV><H2><SPAN class="mw-headline" id="Fundamental_basis">Fundamental basis</SPAN></H2><H3><SPAN class="mw-headline" id="Protein_folding">Protein folding</SPAN></H3><P>Protein folding is the physical phenomena by which a polypeptide chain folds into highly specific and functional three-dimensional structure from a random coil. Shortly after translation from mRNA each protein molecule exists as an unfolded chain with no characteristic conformation. However aminoacids interact with each other to create a well-defined three dimensional structure known as the native state. This resulting conformation is determined by the amino acid sequence.
</P><H3><SPAN class="mw-headline" id="Fusion_proteins">Fusion proteins</SPAN></H3><P>Fusion proteins are proteins which are created by joining two or more genes which originally encoded separate polypeptide chain. Expression of that fusion gene results in a single polypeptide with functional properties derived from each of the proteins encoded by used genes. Recombinant fusion proteins are created artificially via DNA recombination  for use in biological research or to produce altered proteins with new features.
</P><P>In most cases the functionality of fusion proteins is not interrupted. It is possible due to intrinsic protein domains modularity. The fragment of polypeptide which corresponds to a given domain may be removed or added to the rest of the molecule without destroying its native capabilities.
</P><P>However it is highly recommended to predict the three-dimensional structure of fusion protein or the artificially attached domains.  The knowledge of the spatial organization of any given protein is an extremely useful prerequisite for the understanding of the function and for the rational modifications of the proteins.
</P><H2><SPAN class="mw-headline" id="Methods">Methods</SPAN></H2><H3><SPAN class="mw-headline" id="Computation">Computation</SPAN></H3><P>We choose following servers to compute the secondary structures and full models for proteins of interest. 
</P><H4><SPAN class="mw-headline" id="BioInfoBank_Meta_Server"><A href="http://www.bioinfo.pl/" class="external text" rel="nofollow">BioInfoBank Meta Server</A></SPAN></H4><P>This server offers a set of structural models collected from the prediction servers are assessed using the powerful 3D-jury consensus approach.
</P><H4><SPAN class="mw-headline" id="TASSER"><A href="http://zhang.bioinformatics.ku.edu/I-TASSER/" class="external text" rel="nofollow">TASSER</A></SPAN></H4><P>I-TASSER server is an Internet service for protein structure and function predictions. Models are built based on multiple-threading alignments by LOMETS and iterative TASSER simulations. 
</P><H4><SPAN class="mw-headline" id="Robetta"><A href="http://robetta.bakerlab.org/" class="external text" rel="nofollow">Robetta</A></SPAN></H4><P>Robetta is a full-chain protein structure prediction server. It parses protein chains into putative domains and models those domains either by <A href="https://2009.igem.org/Team:Warsaw/Glossary#comparative_modeling" class="external text" rel="nofollow">homology modeling</A> or by <A href="https://2009.igem.org/Team:Warsaw/Glossary#de_novo_modeling" class="external text" rel="nofollow"><I>de novo</I> modeling</A></P><H4><SPAN class="mw-headline" id="The_ModFOLD_Model_Quality_Assessment_Server"><A href="http://www.reading.ac.uk/bioinf/ModFOLD/" class="external text" rel="nofollow">The ModFOLD Model Quality Assessment Server</A></SPAN></H4><P>ModFOLD is a server which can provide a single score and a p-value relating to the predicted quality of a single 3D model of a protein structure and rankings for multiple 3D models for the same protein target according to predicted model quality. It also may do some predictions of the local quality within multiple models.
</P><P><STRONG>More detailed description of used methods is available <A href="https://2009.igem.org/Team:Warsaw/Modeling_methods" class="external text" rel="nofollow">here</A></STRONG></P><H3><SPAN class="mw-headline" id="Evaluation">Evaluation</SPAN></H3><P>We used the following measures of the models validity
</P><UL><LI> Ramachandran plot
</LI><LI> RMSD
</LI><LI> TM-score
</LI><LI> C-score
</LI></UL><P><STRONG>More detailed description of used methods is available <A href="https://2009.igem.org/Team:Warsaw/Models_evaluation" class="external text" rel="nofollow">here</A></STRONG></P><H2><SPAN class="mw-headline" id="Results">Results</SPAN></H2><H3><SPAN class="mw-headline" id="2D-predictions">2D-predictions</SPAN></H3><P>By means of some programs available on bioinformatics metaservers the secondary structures for our proteins of interest have been found. All structures (except one which was found by i-TASSER) were predicted using the meta.bioinfo.pl server. If you want to know the detailed information about secondary structures click <A href="https://2009.igem.org/wiki/index.php?title=Team:Warsaw/Modelling/Structural/secondary_structures" class="external text" rel="nofollow">here</A></P><H3><SPAN class="mw-headline" id="Full_models">Full models</SPAN></H3><H4><SPAN class="mw-headline" id="Secretion_peptid">Secretion peptid</SPAN></H4><P>It was recommended to elucidate the three dimensional structure of the hemolysin A domain responsible for its secretion. In the case of commonly used large proteins tag such as GST it is known that the added domain usually folded autonomically and do not interrupt the native structure of the rest of molecule. Although there is no available data concerning the influence of aforementioned secretion domain on the correct folding. 
</P><PRE>Models score quality (TM  score calculated by the ModFOLD)  0.3015 – 0.1873
</PRE><PRE>0.3015 – model1 (Muster)
0.2813 – model2 (Modeller) 
0.2754 – model3 (Modeller)
0.2585 – model4 (Tasser)
</PRE><PRE>Some calculated RMSD values (expressed in Ångström units)
Model1(Tasser) vs Model3(Modeller) - 4.315
Model1(Tasser) vs Model1(Muster) - 3.116
Model2(Tasser) vs Model3(Modeller) - 3.468
Model2(Tasser) vs Model1(Muster) - 3.874
</PRE><DIV class="thumb tleft"><DIV class="thumbinner" style="width:332px;"><DIV class="thumbcaption">Visualisation of the modelled domain (predicted by Modeller)</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:292px;"><DIV class="thumbcaption">Structural alignment of the three top  models</DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:682px;"><DIV class="thumbcaption">Predicted residue errors for model from Modeller (left) and i-TASSER (right)</DIV></DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:482px;"><DIV class="thumbcaption">Ramachandran plots for model from Modeller (left) and i-TASSER (right)</DIV></DIV></DIV><P>The accuracy of predicted structures is moderate. All generated models resemble each other however the RMSD values among them show that the similarity of these structures is not very significant. The global geometry of the modelled domains is not altered however in each case the spatial localisation of amino acid residues is different. Mediocre TM score indicates that the global topology of the models may not correspond to the real structure of the secretion peptid.
</P><H4><SPAN class="mw-headline" id="Bax_with_secretion_domain_secreted">Bax with secretion domain secreted</SPAN></H4><DIV class="thumb tleft"><DIV class="thumbinner" style="width:312px;"><DIV class="thumbcaption">Structural alignment of one predicted model (green) with secretion peptide model (purple) and crystal structure of bax core domain (yellow)</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:312px;"><DIV class="thumbcaption">Structural alignment of one predicted model (green) with secretion peptide model (purple) and NMR structure of bax core domain (yellow)</DIV></DIV></DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:252px;"><DIV class="thumbcaption">3D histogram representing Ramachandran plot for the best obtained model</DIV></DIV></DIV><P>Most of obtained models are at first sight incorrect and they do not form valid proteins. Only structured predicted by TASSER server seems to be physically acceptable. It should be remarked that the resolution of these models is low especially for the secretion protein. Many residues has improper values of dihedral angles. In spite of these results it appear that the conformation of the protein part which is corresponded to the bax appears to have not been altered by the presence of the additional domain. Unfortunately calculated TM score indicates that the global topology of the models may not correspond to the real structure of the fusion protein.
</P><PRE>Top models score quality 
(TM score calculated by the ModFOLD)  
</PRE><PRE>0.2470 - model1
0.2463 – model2
</PRE><PRE>Some calculated RMSD values (expressed in Ångström units)
1.083 (crystal structure of bax vs model)
13.411 (compared models)
</PRE><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:682px;"><DIV class="thumbcaption">Predicted residue errors for the best models</DIV></DIV></DIV></DIV><H4><SPAN class="mw-headline" id="p53_wih_both_signal_domains">p53 wih both signal domains</SPAN></H4><DIV class="thumb tleft"><DIV class="thumbinner" style="width:312px;"><DIV class="thumbcaption">Structural alignment of model generated by LOMETS (green), modelled secretion domain (cyan) and experimentally resolved p53 core domain (purple)</DIV></DIV></DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:312px;"><DIV class="thumbcaption">Structural alignment of model generated by i-TASSER (red), modelled secretion domain (creamy) and experimentally resolved p53 core domain (purple)</DIV></DIV></DIV><P>The best models were created by simple threading programs Lomets but the structures found by TASSER server were almost the same. The major distinction between models created by these programs was the topology of the secretion domain. Lomets recognized correctly the structure of secretion peptide but it was unable to reconstruct the geometry of p53. The physical quality of all obtained models was evaluated by the Modfold server. The assessment reveals that the resolution of all predicted structures is low. 
</P><PRE>Top models score quality 
(TM score calculated by the ModFOLD) 
</PRE><PRE>0.1995 (LOMETS)
0.1790 (i-TASSER)
</PRE><PRE>Some calculated RMSD values (expressed in Ångström units)
</PRE><PRE>mod5 (i-TASSER) vs secretion domain (Modeller)
RMSD = 11.635 (175 to 175 atoms)
</PRE><PRE>mod5 (i-TASSER) vs crystal structure
192 atoms aligned.
RMSD = 1.135 (171 to 171 atoms
</PRE><PRE>mod1 (LOMETS) vs secretion domain (Modeller)
190 atoms aligned.
RMSD = 3.129 (170 to 170 atoms)
</PRE><P>However one can find the structural similarity between the secretion domain from hemolysin A and obtained model of the domain. In the case of p53 core domain situation is better. RMSD between modelled protein core and experimentally resolved structures collected from PDB is surprisingly high. Unfortunately the validity of the other parts of the molecule is below the level of confidence and it appears to be without significant statistical meaning. As it was mentioned before in the case of models created by LOMETS the resolution of secretion domain is acceptable.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:682px;"><DIV class="thumbcaption">Ramachandran plots for model from LOMETS (left) and i-TASSER (center) in contrast to plot calculated for native structure of p53 (rigth)</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:682px;"><DIV class="thumbcaption">Predicted residue errors for the best models created by LOMETS (left) and i-TASSER</DIV></DIV></DIV></DIV><H4><SPAN class="mw-headline" id="Listeriolysin_with_secretion_domain">Listeriolysin with secretion domain</SPAN></H4><DIV class="thumb tleft"><DIV class="thumbinner" style="width:312px;"><DIV class="thumbcaption">Structural alignment of listeriolysin model (green) with closely related perfringolysin O crystal structure (orange) and the theoretical model of secretion domain (grey)</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:312px;"><DIV class="thumbcaption">Predicted residue errors for the top model created by LOMETS</DIV></DIV></DIV><P>Most of obtained models are physically incorrect and it is unlikely they represented valid proteins. Only one structured predicted by Tasser seems to be acceptable. It should be remarked that the resolution of this models is low especially for the secretion protein. Many residues have improper values of dihedral angles. Despite of these findings it appears that the conformation of the protein part which is corresponded to the listeriolysin appears to have not been altered by the presence of the additional domain. Unfortunately calculated TM score suggests that the global topology of the models may not correspond to the real structure of the fusion protein.
</P><PRE>Calculated RMSD value (expressed in Ångström units):
mod1 (i-TASSER) vs crystal structure of perfringolysin
RMSD = 11.635 (175 to 175 atoms)
</PRE><PRE>Top model score quality (TM score calculated by the ModFOLD)
0.2296
</PRE><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption">Ramachandran plots for model from i-TASSER (left) and  for native structure of perfringolysin (rigth)</DIV></DIV></DIV></DIV><H4><SPAN class="mw-headline" id="Invasin">Invasin</SPAN></H4><DIV class="thumb tleft"><DIV class="thumbinner" style="width:312px;"><DIV class="thumbcaption">Alignment of two theoretical models: first is generated by Modeller (orange) and the latter by i-TASSER (blue)</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:312px;"><DIV class="thumbcaption">Alignment of model calculated by i-TASSER and the crystal structure of invasin from <I>Yersinia pseudotuberculosis</I></DIV></DIV></DIV><P>Due to lack of proper structural template the best solution was to create a model of the domain responsible for invasiveness. Employing the crystal structure of related invasin from <I>Yersinia pseudotuberculosis</I> the full structure model of invasive domain was created. Alignment to the known structure of similar invasin indicate the global geometry of both protein is closely related. It should be underlined that the spatial organisation of essential amino acid residues in the part of molecule which interact with the integrin receptor is almost the same.
</P><PRE>Evaluation of the models
</PRE><PRE>RMSD =  3.274 (between both theoretical models)
Top model score quality 
0.2236 (i-TASSER)
0.2205 (Modeller)
</PRE><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:552px;"><DIV class="thumbcaption">Ramachandran plots for model from i-TASSER (left) and  for native structure of related invasin (rigth)</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:682px;"><DIV class="thumbcaption">Predicted residue errors for the models created by i-TASSER (left) and Modeller (rigth)</DIV></DIV></DIV></DIV><H4><SPAN class="mw-headline" id="PhoP_and_PhoQ">PhoP and PhoQ</SPAN></H4><P>The two proteins are responsible for induction of pH-dependent promoter which is pivotal element of our system. Because the full structures of both PhoP and PhoQ were unknown we decided to find them by means of structural modeling. 
</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:317px;"><DIV class="thumbcaption">Accurate model of PhoP created by Modeller</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Structural alignment of four models of PhoP protein</DIV></DIV></DIV><P>PhoP was an easy target due to presence  of a very similar proteins in PDB database.  Models generated by four programs have the same geometry and the RMSD between the structures is minimal (&lt;0.25). All structures were validated using modFOLD server. The analysis reveals that all of them appear to be correct and the distinctions among the best structures are not significant. These findings indicate the obtained models are notable congruent to the native structure.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:682px;"><DIV class="thumbcaption">Predicted residue errors for the models created by Modeller (left) i-TASSER (center) and MUSTER (right)</DIV></DIV></DIV></DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">Ramachandran plot for top model created by Modeller</DIV></DIV></DIV><PRE>Calculated RMSD value (expressed in Ångström units):

RMS = 0.340 (Modeller vs i-TASSER)
RMS = 0.561 (Modeller vs MUSTER)
RMS = 0.151 (Modeller vs LOMETS)
</PRE><PRE>For the 15 top models TM score 
calculated by the ModFOLD were above 0.6
</PRE><P>PhoQ was more difficult target for all used programs. Most of found models were at first sight physically incorrect. Only i-TASSER was able to created structures which resemble protein molecules. However, the structural alignments between the models reveal low level of similarity for these structures. Some parts were close related but their spatial organisation within the molecule was different in each case. Low TM score suggesting the probability that the structures are valid may not be significant.
</P><PRE>Top models score quality (TM score calculated by the ModFOLD)
0.2908
0.2456
0.2393
</PRE><DIV class="thumb tleft"><DIV class="thumbinner" style="width:347px;"><DIV class="thumbcaption">Structural alignment of three models of PhoQ generated by i-TASSER</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:237px;"><DIV class="thumbcaption">Ramachandran plot for one model created by i-TASSER</DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:682px;"><DIV class="thumbcaption">Predicted residue errors for the models created by i-TASSER</DIV></DIV></DIV></DIV><DIV id="footer-div" style="display: none;">

(C) iGEM 2009 Warsaw Team
</DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2009.igem.org/Team:Warsaw/Modelling/Structural">http://2009.igem.org/Team:Warsaw/Modelling/Structural</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Warsaw/Modelling/Structural" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Warsaw/Modelling/Structural" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Warsaw/Modelling/Structural&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Warsaw/Modelling/Structural&amp;oldid=102147" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2009.igem.org:Privacy_policy" title="2009.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2009.igem.org:General_disclaimer" title="2009.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>