<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Warsaw_Project_detailed"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:Warsaw/Project/detailed">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Warsaw/Project/detailed&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Warsaw/Project/detailed&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Warsaw/Project/detailed&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Warsaw/Project/detailed" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Warsaw/Project/detailed</H1><DIV id="bodyContent"><H3 id="siteSub">From 2009.igem.org</H3><DIV id="war_menu"><DIV class="war_menu_frame"><OL><LI class="main"><A href="/Team:Warsaw" title="Team:Warsaw">Home page</A></LI><P><LI class="main" onclick="switchMenu('team_submenu','menu_item')"><A name="team" style="cursor: hand; cursor: pointer;">The Team</A></LI></P><DIV id="team_submenu" style="display: none;"><LI class="sub"><A href="/Team:Warsaw/Team" title="Team:Warsaw/Team">About us</A></LI><LI class="sub"><A href="/Team:Warsaw/Team/Supervisors" title="Team:Warsaw/Team/Supervisors">Supervisors</A></LI><LI class="sub"><A href="/Team:Warsaw/Team/Miecznikowa" title="Team:Warsaw/Team/Miecznikowa">The M. Group</A></LI><LI class="sub"><A href="/Team:Warsaw/Team/Pawinskiego" title="Team:Warsaw/Team/Pawinskiego">The P. Group</A></LI></DIV><P><LI class="main" onclick="switchMenu('project_submenu')"><A name="project" style="cursor: hand; cursor: pointer;">The Project</A></LI></P><DIV id="project_submenu" style="display: block;"><LI class="sub"><A href="/Team:Warsaw/Project/introduction" title="Team:Warsaw/Project/introduction">Introduction</A></LI><LI class="sub"><A href="/Team:Warsaw/Project/invasion" title="Team:Warsaw/Project/invasion">Invasiveness</A></LI><LI class="sub"><A href="/Team:Warsaw/Project/endosome" title="Team:Warsaw/Project/endosome">Endosome Module</A></LI><LI class="sub"><A href="/Team:Warsaw/Project/cytoplasm" title="Team:Warsaw/Project/cytoplasm">Cytoplasmatic Module</A></LI><LI class="sub"><A href="/Team:Warsaw/Project/future" title="Team:Warsaw/Project/future">Future plans</A></LI><LI class="sub"><A href="/Team:Warsaw/Bibliography" title="Team:Warsaw/Bibliography">Bibliography</A></LI></DIV><LI class="main"><A href="/Team:Warsaw/results" title="Team:Warsaw/results">Results</A></LI><LI class="main" onclick="switchMenu('modelling_submenu')"><A name="modelling" style="cursor: hand; cursor: pointer;">Modelling</A></LI><DIV id="modelling_submenu" style="display: none"><LI class="sub"><A href="/Team:Warsaw/Modelling" title="Team:Warsaw/Modelling">System Overview</A></LI><LI class="sub"><A href="/Team:Warsaw/Modelling/Apoptosis" title="Team:Warsaw/Modelling/Apoptosis">Apoptosis</A></LI><LI class="sub"><A href="/Team:Warsaw/Modelling/Structural" title="Team:Warsaw/Modelling/Structural">Structural</A></LI><LI class="sub"><A href="/Team:Warsaw/Modelling/Gene_Networks" title="Team:Warsaw/Modelling/Gene Networks">Gene Networks</A></LI><LI class="sub"><A href="/Team:Warsaw/Modelling/Invasiveness" title="Team:Warsaw/Modelling/Invasiveness">Invasiveness</A></LI></DIV><P><LI class="main" onclick="switchMenu('lab_submenu')"><A name="lab" style="cursor: hand; cursor: pointer;">Lab Work</A></LI></P><DIV id="lab_submenu" style="display: none"><LI class="sub"><A href="/Team:Warsaw/Notebook" title="Team:Warsaw/Notebook">Notebook</A></LI><LI class="sub"><A href="/Team:Warsaw/protocols" title="Team:Warsaw/protocols">Protocols</A></LI><LI class="sub"><A href="/Team:Warsaw/Resources" title="Team:Warsaw/Resources">Resources</A></LI></DIV><LI class="main"><A href="/Team:Warsaw/Parts" title="Team:Warsaw/Parts">Submitted Parts</A></LI><LI class="main"><A href="/Team:Warsaw/Glossary" title="Team:Warsaw/Glossary">Glossary</A></LI><LI class="main"><A href="/Team:Warsaw/Biosafety" title="Team:Warsaw/Biosafety">Biosafety</A></LI><LI class="main"><A href="/Team:Warsaw/Human" title="Team:Warsaw/Human">Human Practices</A></LI><LI class="main"><A href="/Team:Warsaw/Acknowledgements" title="Team:Warsaw/Acknowledgements">Acknowledgements</A></LI></OL></DIV><DIV class="war_menu_frame_bottom"> </DIV></DIV><DIV id="war_content"><H2><SPAN class="mw-headline" id="Detailed_research_project">Detailed research project</SPAN></H2><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Detailed_research_project"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Detailed research project</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Introduction"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Introduction</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Invasion_operon"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Invasion operon</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Endosome_detection_operon"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Endosome detection operon</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#The_cytoplasmic_operon."><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">The cytoplasmic operon.</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#The_mechanism_of_proposed_regulatory_system."><SPAN class="tocnumber">1.5</SPAN><SPAN class="toctext">The mechanism of proposed regulatory system.</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#Conjugation_with_mitochondria."><SPAN class="tocnumber">1.6</SPAN><SPAN class="toctext">Conjugation with mitochondria.</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#Secretion_of_p53_into_the_cytoplasm"><SPAN class="tocnumber">1.7</SPAN><SPAN class="toctext">Secretion of p53 into the cytoplasm</SPAN></A></LI><LI class="toclevel-2 tocsection-9"><A href="#Directing_bacterial_proteins_to_mammalian_mitochondria."><SPAN class="tocnumber">1.8</SPAN><SPAN class="toctext">Directing bacterial proteins to mammalian mitochondria.</SPAN></A></LI><LI class="toclevel-2 tocsection-10"><A href="#Targeting_the_tumor_cells_via_recognition_the_specific_receptors_on_the_cell_surface"><SPAN class="tocnumber">1.9</SPAN><SPAN class="toctext">Targeting the tumor cells via recognition the specific receptors on the cell surface</SPAN></A></LI><LI class="toclevel-2 tocsection-11"><A href="#Error_proofing_and_disposing_of_the_run_down_bacteria"><SPAN class="tocnumber">1.10</SPAN><SPAN class="toctext">Error proofing and disposing of the run down bacteria</SPAN></A></LI></UL></TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Introduction">Introduction</SPAN></H3><P>
Fig 1. The overview of the system. Gene regulatory network is composed of three modules: invasion operon, endosome detection operon and cytoplasmic operon. Invasion and cytoplasmic operon are bistable switches regulated by endosome detection operon. 
</P><P>This project attempts to create <I>E. coli</I> strain capable of entering the eukaryotic cells by means of controlled expression of <A href="https://2009.igem.org/Team:Warsaw/Glossary#invasin" class="external text" rel="nofollow">invasin</A> gene from <I>Yersinia</I> sp. together with <A href="https://2009.igem.org/Team:Warsaw/Glossary#internalin" class="external text" rel="nofollow">internalinA</A> gene from <I>Listeria monocytogenes</I> st. EDG-e. This leads to internalization of the bacteria into the <A href="https://2009.igem.org/Team:Warsaw/Glossary#endosome" class="external text" rel="nofollow">phagosome</A> of the host cell. To escape the <A href="https://2009.igem.org/Team:Warsaw/Glossary#endosome" class="external text" rel="nofollow">endosome</A> we will apply bacterial hemolysin (<A href="https://2009.igem.org/Team:Warsaw/Glossary#listeriolysin" class="external text" rel="nofollow">listeriolysin</A>) from <I>Listeria monocytogenes</I>. The coordination of protein expression is based on switching the <I>E. coli</I> cells between four physiological states: growth in the medium, ability to enter mammalian cells, presence in the <A href="https://2009.igem.org/Team:Warsaw/Glossary#endosome" class="external text" rel="nofollow">phagosome</A>, and escape from the <A href="https://2009.igem.org/Team:Warsaw/Glossary#endosome" class="external text" rel="nofollow">phagosome</A> into the cytoplasm. (Fig1.). Two component system <A href="https://2009.igem.org/Team:Warsaw/Glossary#phoP.2FPhoQ" class="external text" rel="nofollow">PhoP/PhoQ</A> from <I>Salmonella enterica</I> ser. <I>typhimurium</I> LT2 will be used to detect the states. It activates PhoP dependent promoters in the endosomal environment (low pH, low metal ions concentration). Each of the states will be stabilised by two feedback loops based on natural regulatory systems. System is composed of three synthetic operons [18]. 
</P><H3><SPAN class="mw-headline" id="Invasion_operon">Invasion operon</SPAN></H3><P>
Fig 2. The overview of the invasion operon. It is based on a lacI/cI bistable switch. After thermal activation (42°C) the expression of genes that enable invasion of mammalian cells is activated. The invasion operon is composed of lacI – lactose operon repressor, llo – hemolysin from <I>Listeria monocytogenes</I>, inv – invasin from <I>Yersinia</I> sp., phoQ/phoP two component system detecting endosomal conditions from <I>Salmonella typhimurium</I>, GFP – Green Fluorescent Protein from <I>Aqueora victoria</I>. Proteins are fused with LVA sequence which provides quick proteolytic degradation. 
</P><P>The invasion operon includes <A href="https://2009.igem.org/Team:Warsaw/Glossary#lacI" class="external text" rel="nofollow">lacI</A>– lactose operon repressor, llo – hemolysin from <I>Listeria monocytogenes</I> (<A href="https://2009.igem.org/Team:Warsaw/Glossary#listeriolysin" class="external text" rel="nofollow">listeriolysin</A>), inv – <A href="https://2009.igem.org/Team:Warsaw/Glossary#invasin" class="external text" rel="nofollow">invasin</A> from <I>Yersinia</I> sp., <A href="https://2009.igem.org/Team:Warsaw/Glossary#phoP.2FPhoQ" class="external text" rel="nofollow">PhoP/PhoQ</A> two component system that detects endosomal conditions from <I>Salmonella typhimurium</I>, <A href="https://2009.igem.org/Team:Warsaw/Glossary#Green_Fluorescent_Protein" class="external text" rel="nofollow">GFP</A> – green fluorescent protein. (Fig.2.). These genes are under the control of bacteriophage promoter (PR) that is repressed by <A href="https://2009.igem.org/Team:Warsaw/Glossary#cI" class="external text" rel="nofollow">cI</A> protein. The second part of the operon is gene cI<SUB>ts</SUB> that encodes for thermosensitive version of the <A href="https://2009.igem.org/Team:Warsaw/Glossary#cI" class="external text" rel="nofollow">cI repressor</A> protein from phage λ. It is under the control of lactose promoter (Plac). This λ system of promoters and regulatory genes creates two mutually exclusive negative feedback loops and is dubbed the bistable switch [19]. It has two states depending on conditions within the cell. <A href="https://2009.igem.org/Team:Warsaw/Glossary#cI" class="external text" rel="nofollow">cI</A> protein binds the PR promoter inhibiting its activity and expression of the invasion operon genes. cI<SUB>ts</SUB> protein is inactivated by high temperature 42°C it allows the expression of invasion operon that codes for <A href="https://2009.igem.org/Team:Warsaw/Glossary#lacI" class="external text" rel="nofollow">lacI</A>. <A href="https://2009.igem.org/Team:Warsaw/Glossary#lacI" class="external text" rel="nofollow">lacI</A> stops gene expression from the Plac promoter ( the expression of <A href="https://2009.igem.org/Team:Warsaw/Glossary#cI" class="external text" rel="nofollow">cI</A> protein). IPTG that binds and inactivates <A href="https://2009.igem.org/Team:Warsaw/Glossary#lacI" class="external text" rel="nofollow">lacI</A> can be used to switch on the expression from Plac promoter. Another way to control the operon is <I>in trans</I> expression of <A href="https://2009.igem.org/Team:Warsaw/Glossary#cI" class="external text" rel="nofollow">cI</A> or <A href="https://2009.igem.org/Team:Warsaw/Glossary#lacI" class="external text" rel="nofollow">lacI</A> genes. Activated invasion operon causes internalisation of bacteria by mammalian cells. It also detects endosomal localisation using <A href="https://2009.igem.org/Team:Warsaw/Glossary#phoP.2FPhoQ" class="external text" rel="nofollow">PhoP/PhoQ system</A>. Expression of green fluorescence protein allows observation of the process using confocal microscopy. 
</P><H3><SPAN class="mw-headline" id="Endosome_detection_operon">Endosome detection operon</SPAN></H3><P>
Fig 3. Overview of endosomal detection operon. It is composed of cro – antirepressor from phage λ, tetR – repressor of tetracycline resistance operon, cI – repressor from phage λ and cfp – Cyan Fluorescent Protein. 
</P><P>Low endosomal pH and low concentration of metal ions lead to decrease the likelihood of transmembrane receptor kinase phoQ binding its ligands – divalent cations. This enables the phosphorylation of phoP by phoQ. Phosphorylated phoP activates the gene expression from <A href="https://2009.igem.org/Team:Warsaw/Glossary#mgtc_gene_promoter" class="external text" rel="nofollow">MgtC promoter</A> leading to production of proteins encoded by endosome detection operon. <A href="https://2009.igem.org/Team:Warsaw/Glossary#cI" class="external text" rel="nofollow">cI</A> protein inactivates the invasion operon. TetR protein activates cytoplasmic operon. Not all genes encoded by cytoplasmic operon are expressed in endosome, due to the presence of cro box sequence, which binds cro protein thus stopping the trancription Cro protein is encoded by endosome detection operon. After bacterium escapes into cytoplasm the endosome detection operon genes are no longer expressed, since phoP is not phosphorylated in cytoplasmic conditions. The process can be monitored using CFP fluorescence protein. 
</P><H3><SPAN class="mw-headline" id="The_cytoplasmic_operon.">The cytoplasmic operon.</SPAN></H3><P>
Fig 4. Overview of endosomal detection operon. It is composed of YFP – Yellow Fluorescent Protein, tetR – repressor of a tetracycline resistance operon, cro box regulatory sequence from phage and λ desired genes and genes specific for desired function of bacterial strain. In this case p53 – apoptosis inducer. 
</P><P>Cytoplasmic operon, like the invasion operon, is a bistable switch. The regulatory elements are: synthetic promoter inhibited by TetR, promoter inhibited by araC, araC protein (inactivated by L-arabinose) and TetR protein (repressor of tetracycline resistance gene inactivated by tetracycline). [18] This switch controls genes expressed in the cytoplasm. Cro box confines the gene expression to the cytoplasm only. Cro protein bound to it stops the transcription while the endosome detection operon is expressed. The yellow florescence protein is expressed to allow detection of the cytoplasmic operon activity. The set of genes in the cytoplasmic operon depends on the desired functionality of bacterial strain. In case of antitumor therapy could be the p53 protein with mitochondrial leader sequence. In the strain capable of conjugation with mitochondria cytoplasmic operon will control genes responsible for the conjugation process. 
</P><H3><SPAN class="mw-headline" id="The_mechanism_of_proposed_regulatory_system.">The mechanism of proposed regulatory system.</SPAN></H3><P>In the growth medium (at 37 °C) the expression of all genetic circuits is inactive. After temperature increases to 42°C the system is activated by thermal degradation of cI<SUB>ts</SUB>. Genes from invasion operon are transcribed. They maintain their own expression, initiate internalisation of <I>E. coli</I> by mammalian cells and enable escape from endosome. When bacterial cell is inside the endosome the endosome detection module is activated by <A href="https://2009.igem.org/Team:Warsaw/Glossary#phoP.2FPhoQ" class="external text" rel="nofollow">PhoP/PhoQ system</A>. In trans expression of regulatory genes <A href="https://2009.igem.org/Team:Warsaw/Glossary#cI" class="external text" rel="nofollow">cI</A> and tetR deactivates invasion operon and activates cytoplasmic operon. However its genes are not expressed as long as bacterial cell is inside endosome due to cro protein activity. When bacterium escapes from the endosome, the transcription from endosome detection operon is repressed and cytoplasmic operon genes are expressed. 
</P><H3><SPAN class="mw-headline" id="Conjugation_with_mitochondria.">Conjugation with mitochondria.</SPAN></H3><P>To locate the copy of the mitochondrial genome into bacterial cell the pBACrNESd plasmid will be used. It is built basing the Bacterial Artificial Chromosome. Replication origin (ori R6K), kanamycin resistance gen (kanR) and replication complex genes (repE, parA, parC) from pBACrNESd plasmid will be fused with origin of transfer (oriTF) from natural conjugative F plasmid. Unique restriction site will be introduced into the mitochondrial genome by PCR reaction and the described construct will be inserted into mitochondrion. To enable conjugation proteins encoded on RP4 plasmid (Tc, MuKm, Tn7) will be placed in the cytoplasmic operon. These genes are responsible for pillus formation and transport of single stranded DNA. The cell with this system will conjugate with any structure surround by lipid bilayer. We will prepare mitochondrial DNA deficient cells using a previously described procedure [2]. The occurrence of conjugation will be confirmed by PCR reaction specific for sequences added to mitochondrial DNA. The origin of the DNA will be determined by analysis of restriction patterns. The expression of mitochondrial genes will be confirmed by semi-quantitative PCR [2]. 
</P><H3><SPAN class="mw-headline" id="Secretion_of_p53_into_the_cytoplasm">Secretion of p53 into the cytoplasm</SPAN></H3><P>To efficiently transport p53 to the eukaryotic cells the endogenic <I>E. coli</I> secretion system responsible for secretion of hemolysin by uropathogenic strains will be used. It is composed of three transport proteins HlyB, HlyD and TolC [10]. TolC is present in the genome of the lab strain derived from K12 strain. To ensure efficient secretion hlyB and hlyD have to be introduced into the genome and proapoptotic protein has to be fused with 60 amino acid long C-terminal sequence of hemolysin. [5].
</P><H3><SPAN class="mw-headline" id="Directing_bacterial_proteins_to_mammalian_mitochondria.">Directing bacterial proteins to mammalian mitochondria.</SPAN></H3><P>One of the nescessary objectives is to direct a fraction of secreted proteins into the mitochondrion. This will be achieved by fusion of mitochondrial leader sequence from human protein Cox1 to the N-terminus of proteins. It has been shown that  40 amino acids long N-terminal sequence is enough for effective transport of Cox1 across the mitochondrial membrane. First the efficiency of the process will be assessed by detection of Red Fluorescent Protein with confocal microscopy. The red fluorescence should be visible in the mitochondria of the cells invaded by bacteria expressing RFP fused with the mitochondrial leader sequence. Then the ability to induce apoptosis by mitochondrially directed p53 will be investigated. Apoptosis induction will be confirmed by flow cytometry [8]. In addition we will use confocal microscopy to monitor the process. In both cases cells will be stained with propidium iodide and annexin V. 
</P><H3><SPAN class="mw-headline" id="Targeting_the_tumor_cells_via_recognition_the_specific_receptors_on_the_cell_surface">Targeting the tumor cells via recognition the specific receptors on the cell surface</SPAN></H3><P>The specific delivery of drug molecule or larger particles like artificial liposomes, dendrimers or even whole bacterial cells  to tumor cells mainly relies on the potential of the associated vector to specifically head for the malignant center. This implies the identification of tumor markers which are appropriate target. One of the major strategy consists in targeting proteins that are highly specific or directly over-expressed on tumor cells. Nowadays,  the set of potentially interested ligands for the specifically targeted anticancer agents is remarkable large but only a few class of these markers appears to be promising as a targets for the new generation anticancer drugs. Nowadays plethora of proteins that are responsible for cellular signaling pathways are known, involved in the proliferation, angiogenesis, and differentiation in neoplasms with the targets amenable to therapeutic interventions in cancer therapy. Membrane-bound human epidermal growth factor receptors (HER), c-MET, and insulin-like growth factor 1 receptor (IGF-1R) mediate mitogenic signals from extracellular ligands, such as epidermal growth factor (EGF), hepatocyte growth factor (HGF), and insulin growth factors (IGF) are the most popular ligands.
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Abnormal expression of integrins in tumor</DIV></DIV></DIV><P>It is strongly believed that the most promising type of tumor-specific markers are vascular receptors mainly involved in the neoangiogenesis of the solid tumors and the promotion of cancer cells migration and their invasiveness. Angiogenesis, the process by which new blood capillaries are formed from preexisting vessels, is a hallmark of cancer. The process is a common feature to many solid tumors and angiogenic targets are easily accessible for systemic treatments and are genetically stable. These targets are also could be exploited for imaging and destructing tumor blood vessels. 
</P><P>Integrins are the crucial family of receptors for for cell-matrix adhesion and are also involved in various cell-cell adhesion processes. Furthermore, integrins create transmembrane connections to the cytoskeleton and are pivotal for working of many intracellular signaling pathways. Therefore, integrins and their ligands play key roles in development, immune response, cellular hemostasis and cancer. Equipped with these various functions, integrins are desired targets of effective therapeutic drugs and are objects of intensive continuous research in the area for development and targeting of drugs.
</P><P>Until now, 18 α and 8 β integrins subunits are known in mammalian cells to associate non-covalently forming 24 integrin heterodimers. Each type of subunit is a peculiar glycoprotein which consists of a large extracellular domain, a transmembrane helix and a short cytoplasmic tail. It should be remark that each of the 24 integrins appears to have a specific, nonredundant function (this can be concluded both from their interactors specificities and from phenotypes of knockout mice).
</P><P>The relevance of α5β1 integrin as a proangiogenic factor arises from some knock-out experiments. It is assumed that both fibronectin and aforementioned integrin directly regulate angiogenesis. Hence, it is reasonable to expect the α5β1 integrin to play an important role as a potent anticancer drug target. Additionally, recent studies suggested a key role for the α5β1  integrin in certain bacterial invasion of human host cells leading to antibiotic resistance. The bacteria exhibit on their surface proteins belong to the invasin family which are capable of selectively recognition of this integrins. These findings suggest that usage of bacteria containing the invasin outside of the cell membrane to targeting cancer cell is resonable idea.
</P><H3><SPAN class="mw-headline" id="Error_proofing_and_disposing_of_the_run_down_bacteria">Error proofing and disposing of the run down bacteria</SPAN></H3><P>During the course of planing we have given much thought to the issue of error proofing. When the project will be finalised we'll be dealing with a potentially dangerous microbe, capable of entering eukaryotic cells. This forced us develop a method of emergency killing the bacteria at any given time, a so called &quot;kill switch&quot;. We chose to use a toxin/antitoxin system (MazF/MazE for example) in which both proteins are produced at equal levels. The antitoxin will be expressed from a promoter that is easily inhibited by small size particles capable of penetrating inside the cell. If at any moment we decide to terminate the treatment the introduction of such a particle to the blood stream will cause the bacterium to halt the production of the antitoxin thus effectively killing itself.
</P><P>The other problem we have faced was what to do with the bacteria that have already completed their task. If such a bacterium finds itself absorbed by a white blood cell along with the apoptosomes it might start inducing apoptosis to its new host. Our solution to that problem was inspired by the Team Waterloo's 2008 project. The cytoplasmic operon could be supplemented with an additional endonucleolitic enzyme, one that would dispose of every unnecessary peace of DNA including bacteria's genome and previous operons. Such a vessel could only carry out the one final step (be it protein secretion or conjugation) over and over again until it depletes all its resources becoming eventually digested by either the host cell or along with the apoptotic remnants.
</P><DIV id="footer-div" style="display: none;">

(C) iGEM 2009 Warsaw Team
</DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2009.igem.org/Team:Warsaw/Project/detailed">http://2009.igem.org/Team:Warsaw/Project/detailed</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Warsaw/Project/detailed" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Warsaw/Project/detailed" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Warsaw/Project/detailed&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Warsaw/Project/detailed&amp;oldid=155046" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2009.igem.org:Privacy_policy" title="2009.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2009.igem.org:General_disclaimer" title="2009.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>