<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Warsaw_Glossary"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:Warsaw/Glossary">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Warsaw/Glossary&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Warsaw/Glossary&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Warsaw/Glossary&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Warsaw/Glossary" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Warsaw/Glossary</H1><DIV id="bodyContent"><H3 id="siteSub">From 2009.igem.org</H3><DIV id="war_menu"><DIV class="war_menu_frame"><OL><LI class="main"><A href="/Team:Warsaw" title="Team:Warsaw">Home page</A></LI><P><LI class="main" onclick="switchMenu('team_submenu','menu_item')"><A name="team" style="cursor: hand; cursor: pointer;">The Team</A></LI></P><DIV id="team_submenu" style="display: none;"><LI class="sub"><A href="/Team:Warsaw/Team" title="Team:Warsaw/Team">About us</A></LI><LI class="sub"><A href="/Team:Warsaw/Team/Supervisors" title="Team:Warsaw/Team/Supervisors">Supervisors</A></LI><LI class="sub"><A href="/Team:Warsaw/Team/Miecznikowa" title="Team:Warsaw/Team/Miecznikowa">The M. Group</A></LI><LI class="sub"><A href="/Team:Warsaw/Team/Pawinskiego" title="Team:Warsaw/Team/Pawinskiego">The P. Group</A></LI></DIV><P><LI class="main" onclick="switchMenu('project_submenu')"><A name="project" style="cursor: hand; cursor: pointer;">The Project</A></LI></P><DIV id="project_submenu" style="display: block;"><LI class="sub"><A href="/Team:Warsaw/Project/introduction" title="Team:Warsaw/Project/introduction">Introduction</A></LI><LI class="sub"><A href="/Team:Warsaw/Project/invasion" title="Team:Warsaw/Project/invasion">Invasiveness</A></LI><LI class="sub"><A href="/Team:Warsaw/Project/endosome" title="Team:Warsaw/Project/endosome">Endosome Module</A></LI><LI class="sub"><A href="/Team:Warsaw/Project/cytoplasm" title="Team:Warsaw/Project/cytoplasm">Cytoplasmatic Module</A></LI><LI class="sub"><A href="/Team:Warsaw/Project/future" title="Team:Warsaw/Project/future">Future plans</A></LI><LI class="sub"><A href="/Team:Warsaw/Bibliography" title="Team:Warsaw/Bibliography">Bibliography</A></LI></DIV><LI class="main"><A href="/Team:Warsaw/results" title="Team:Warsaw/results">Results</A></LI><LI class="main" onclick="switchMenu('modelling_submenu')"><A name="modelling" style="cursor: hand; cursor: pointer;">Modelling</A></LI><DIV id="modelling_submenu" style="display: none"><LI class="sub"><A href="/Team:Warsaw/Modelling" title="Team:Warsaw/Modelling">System Overview</A></LI><LI class="sub"><A href="/Team:Warsaw/Modelling/Apoptosis" title="Team:Warsaw/Modelling/Apoptosis">Apoptosis</A></LI><LI class="sub"><A href="/Team:Warsaw/Modelling/Structural" title="Team:Warsaw/Modelling/Structural">Structural</A></LI><LI class="sub"><A href="/Team:Warsaw/Modelling/Gene_Networks" title="Team:Warsaw/Modelling/Gene Networks">Gene Networks</A></LI><LI class="sub"><A href="/Team:Warsaw/Modelling/Invasiveness" title="Team:Warsaw/Modelling/Invasiveness">Invasiveness</A></LI></DIV><P><LI class="main" onclick="switchMenu('lab_submenu')"><A name="lab" style="cursor: hand; cursor: pointer;">Lab Work</A></LI></P><DIV id="lab_submenu" style="display: none"><LI class="sub"><A href="/Team:Warsaw/Notebook" title="Team:Warsaw/Notebook">Notebook</A></LI><LI class="sub"><A href="/Team:Warsaw/protocols" title="Team:Warsaw/protocols">Protocols</A></LI><LI class="sub"><A href="/Team:Warsaw/Resources" title="Team:Warsaw/Resources">Resources</A></LI></DIV><LI class="main"><A href="/Team:Warsaw/Parts" title="Team:Warsaw/Parts">Submitted Parts</A></LI><LI class="main"><STRONG class="selflink">Glossary</STRONG></LI><LI class="main"><A href="/Team:Warsaw/Biosafety" title="Team:Warsaw/Biosafety">Biosafety</A></LI><LI class="main"><A href="/Team:Warsaw/Human" title="Team:Warsaw/Human">Human Practices</A></LI><LI class="main"><A href="/Team:Warsaw/Acknowledgements" title="Team:Warsaw/Acknowledgements">Acknowledgements</A></LI></OL></DIV><DIV class="war_menu_frame_bottom"> </DIV></DIV><DIV id="war_content"><H2><SPAN class="mw-headline" id="Glossary">Glossary</SPAN></H2><H2><SPAN class="mw-headline" id="A">A</SPAN></H2><H4><SPAN class="mw-headline" id="apoptosis">apoptosis</SPAN></H4><DIV class="glossary_text">Apoptosis is a natural process of programmed cell death. Apoptosis can be induced by many factors and its function is to remove damaged and unnecessary cells from the organism. In our project we want to induce apoptosis of tumor cells, using <A href="https://2009.igem.org/Team:Warsaw/Glossary#p53" class="external text" rel="nofollow">p53</A> or <A href="https://2009.igem.org/Team:Warsaw/Glossary#bax" class="external text" rel="nofollow">bax</A> proteins, both involved in control of this process</DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:362px;"><DIV class="thumbcaption">Overview of basic signal transduction pathways.</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:272px;"><DIV class="thumbcaption">A section of mouse liver stained to show cells undergoing apoptosis (orange)</DIV></DIV></DIV><H2><SPAN class="mw-headline" id="B">B</SPAN></H2><H4><SPAN class="mw-headline" id="bax">bax</SPAN></H4><DIV class="thumb tright"><DIV class="thumbinner" style="width:397px;"><DIV class="thumbcaption">Soluble form of bax protein</DIV></DIV></DIV><DIV class="glossary_text">Bcl-2–associated X protein (bax) is a pro-apoptotic protein. Although it's found mainly in the cytosol, upon initiation of <A href="https://2009.igem.org/Team:Warsaw/Glossary#apoptosis" class="external text" rel="nofollow">apoptosis</A> it's shifted to organellar membranes. It's believed that bax is responsible for opening special channels in the mitochondrial outer membrane, causing release of pro-apoptotic factors like cytochrome c. These proteins subsequently assembly a multiprotein complex named apoptosome which activated special proteases called caspases which destine the cell to apoptosis.</DIV><H2><SPAN class="mw-headline" id="C">C</SPAN></H2><H4><SPAN class="mw-headline" id="cI">cI</SPAN></H4><DIV class="glossary text">This protein is also known as the lambda repressor. cI is sole protein expressed in the lysogenic state of Lambda phage. It act as transcription inhibitor which binds to specific sites in phage genome. At low concentrations blocks the pR promoter (preventing cro production). At high concentrations downregulates its own production through OR3 binding</DIV>. 
<H4><SPAN class="mw-headline" id="comparative_modeling">comparative modeling</SPAN></H4><DIV class="glossary text">These methods use previously solved structures as templates. This methodology is effective because of appearance only a limited set of tertiary structural motifs to which most proteins belong. These methods may be split into two groups:</DIV><DIV class="glossary text"><B>Homology modeling</B> is based on the assumption that two homologous proteins will share very similar structures. Because a protein's fold is more evolutionarily conserved than its amino acid sequence, a target sequence can be modeled with reasonable accuracy on a very distantly related template, provided that the relationship between target and template can be discerned through sequence alignment.</DIV><DIV class="glossary text"><B>Protein threading</B> analyse the amino acid sequence of an unknown structure against a database of solved structures. In each case, a scoring function is used to assess the compatibility of the sequence to the structure, thus yielding possible three-dimensional models.</DIV><H4><SPAN class="mw-headline" id="cro">cro</SPAN></H4><DIV class="glossary text">Cro repressor is a small dimeric protein which binds to specific sites on the DNA of bacteriophage lambda. Fragment of the phage DNA that is recognized by cro is known as cro-box. Recognition of specific DNA binding sites appears to occur via multiple hydrogen bonds between amino acid side chains of the protein and base pair atoms exposed within the major groove of DNA. Cro act as a transcription inhibitor which binds OR3, OR2 and OR1 sites. At low concentrations blocks the pRM promoter preventing cI production. At high concentrations downregulates its own production through OR2 and OR1 binding.</DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:317px;"><DIV class="thumbcaption">Cro is a very small protein composed of 61 aminoacids</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:317px;"><DIV class="thumbcaption">Interaction between cro and DNA</DIV></DIV></DIV><H4><SPAN class="mw-headline" id="cro-box">cro-box</SPAN></H4><DIV class="glossary text">Cro-box is the particular fragment of phage lambda DNA which is recognized by aforementioned cro repressor. In our project we used artificial designed cro-box sequence:</DIV><PRE> 5' ATCTAGATACCTCTGGCGGTGATACTAGTGT 3'</PRE><H4><SPAN class="mw-headline" id="cytochrome_C_oxidase">cytochrome C oxidase</SPAN></H4><DIV class="glossary text">The enzyme cytochrome c oxidase or Complex IV is a large transmembrane protein complex found in bacteria and the mitochondrion. It is the last enzyme in the respiratory electron transport chain located in the mitochondrial (or bacterial) membrane. It receives an electron from each of four cytochrome c molecules, and transfers them to one oxygen molecule, converting molecular oxygen to two molecules of water. In the process, it binds and translocates four protons across the membrane, helping to establish a transmembrane proton concentration gradient which is crucial for the ATP synthase to synthesize ATP.</DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:317px;"><DIV class="thumbcaption">Cytochrome c oxidase complex. Each subunit is depicted using different color.</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:317px;"><DIV class="thumbcaption">Subunit VII of the complex has signal pepting which has been used in our project</DIV></DIV></DIV><H2><SPAN class="mw-headline" id="D">D</SPAN></H2><H4><SPAN class="mw-headline" id="de_novo_modeling"><I>de novo</I> modeling</SPAN></H4><DIV class="glossary text"><I>De novo</I> protein modelling methods attempt to build three-dimensional protein models  based on physical principles. There are few possible procedures that are able to reconstruct folding pathway or apply some stochastic method to search native conformations. These procedures tend to require vast computational resources, and have thus only been carried out for small proteins.</DIV><H2><SPAN class="mw-headline" id="E">E</SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:102px;"><DIV class="thumbcaption">Endosomes in mammalian cell</DIV></DIV></DIV><H4><SPAN class="mw-headline" id="endosome">endosome</SPAN></H4><DIV class="glossary text">endosome is a membrane-bound compartment inside a cell which is responsible for the sorting of material previously endocytosed by the cell before it being transported to lysosomes. This allows some material to be returned to the plasma membrane.</DIV><H2><SPAN class="mw-headline" id="G">G</SPAN></H2><H4><SPAN class="mw-headline" id="Green_Fluorescent_Protein">Green Fluorescent Protein</SPAN></H4><DIV class="glossary text">The green fluorescent protein (GFP) is protein which exhibits intensive green fluorescence when exposed to blue light. Although many other marine organisms have similar green fluorescent proteins, GFP traditionally refers to the protein first isolated from the jellyfish <I>Aequorea victoria</I>. The GFP from <I>A. victoria</I> has a major excitation peak at a wavelength of 395 nm and a minor one at 475 nm. Its emission peak is at 509 nm which is in the lower green portion of the visible spectrum. In the field of cell and molecular biology, the GFP gene is frequently used as a reporter of expression. Many different mutants of GFP have been engineered since their potential for widespread usage. The collection of the mutagenized forms include many color mutants.</DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:319px;"><DIV class="thumbcaption">Yellow fluorescent protein</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:319px;"><DIV class="thumbcaption">Focus on the fluorescent center</DIV></DIV></DIV><H2><SPAN class="mw-headline" id="I">I</SPAN></H2><H4><SPAN class="mw-headline" id="integrins">integrins</SPAN></H4><DIV class="glossary text">Integrins are cellular receptors that mediate attachment between a cell and the tissues surrounding it, which may be other cells or the extracellular matrix (ECM). They are also involved in cell signaling and thereby define cellular shape, mobility, and regulate the cell cycle. Many types of integrin exist, and some cells have multiple types of integrins on their surface. Integrins are of vital importance to all animals and have been found in each investigated animal plylum. Several integrin types are recognized by invasins, bacterial surface proteins which are responsible for entrance of bacterium into the cell.</DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:277px;"><DIV class="thumbcaption">Integrin β 1 which is recognized by invasin.</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption">Fragment of integrin β V structure.</DIV></DIV></DIV><H4><SPAN class="mw-headline" id="internalin">internalin</SPAN></H4><DIV class="glossary text">Internalins are surface proteins found on <I>Listeria monocytogenes</I>. There are two diiferent forms of these proteins, InlA and InlB. Both of them are used by the bacteria to invade mammalian cells via cadherins transmembrane proteins which exist on the cellular membrane. The exact role of these proteins and their invasiveness in vivo is in not yet completely understood. In cultured cells, InlA is necessary to facilitate <I>Listeria</I> entry into human epithelial cells. However InlB is necessary for bacterial cell internalisation in several other cell types, including hepatocytes and fibroblasts.</DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:317px;"><DIV class="thumbcaption">Structure of internalin B</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:317px;"><DIV class="thumbcaption">Internalin B (orange) interacting with cadherin E (violet)</DIV></DIV></DIV><H4><SPAN class="mw-headline" id="invasin">invasin</SPAN></H4><DIV class="glossary_text"><DIV class="thumb tright"><DIV class="thumbinner" style="width:317px;"><DIV class="thumbcaption">&quot;Head domain&quot; responsible for interaction with integrins</DIV></DIV></DIV>Invasin belongs to widespread protein family whose members are crucial for pathogenic features of enteropathogenic bacteria such as <I>Yersinia</I> sp. or <I>Salmonella</I> sp. Invasin (and related proteins) has a strong affinity to and therefore interacts with integrins, a family of receptors occurring on the surface of eukaryotic cell membranes. This interaction triggers a signalling cascade leading to endocytosis of the whole pathogenic bacteria. This step is obligatory for the invasion of the bacteria into the cells. The process is highly selective and bacteria cannot invade cells which are depleted of integrins.
<DIV><H2><SPAN class="mw-headline" id="L">L</SPAN></H2><H4><SPAN class="mw-headline" id="lacI">lacI</SPAN></H4><DIV class="glossary text"><DIV class="thumb tleft"><DIV class="thumbinner" style="width:292px;"><DIV class="thumbcaption">Dimer of LacI with IPTG molecules (white)</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:332px;"><DIV class="thumbcaption">LacI protein bound to DNA</DIV></DIV></DIV>The lac repressor is a bacterial DNA-binding protein which inhibits the expression of genes which encode proteins involved in the metabolism of lactose. It is active in the absence of lactose, ensuring that the bacterium only invests its resources for the synthesis of proteins required for the uptake and metabolism of lactose when this saccharide is found in the environment. When lactose becomes available, it is converted into allolactose, which inhibits the lac repressor's DNA binding ability.</DIV><H4><SPAN class="mw-headline" id="listeriolysin">listeriolysin</SPAN></H4><DIV class="glossary text"><DIV class="thumb tleft"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Pore formed by 12 molecules of hemolysin</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:312px;"><DIV class="thumbcaption">Structure of hemolysin which is close related to listeriolysin</DIV></DIV></DIV>Listeriolysin O (LLO) is a pore-forming protein from <I>Listeria monocytogenes</I> which belong to hemolysin family.The protein is selectively activated within the acidic phagosomes (average pH ~ 5.9) of cells that have phagocytosed <I>L. monocytogenes</I>. After LLO lyses the phagosome, the bacterium escapes into the cytosol, where it can grow intracellularly. Upon release from the phagosome, the toxin has reduced activity in the more basic cytosol.</DIV><H2><SPAN class="mw-headline" id="M">M</SPAN></H2><H4><SPAN class="mw-headline" id="mgtc_gene_promoter">mgtc gene promoter</SPAN></H4><DIV class="glossary_text">This is one of the <I>Salmonella typhimurium</I> PhoP-dependent promoters. MgtC gene is a virulence factor in <I>Salmonella typhimurium</I> that is required for growth at low-Mg2+ concentrations and intramacrophage survival</DIV><DIV class="glossary text"><DIV class="thumb tright"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">Regions of the mtDNA connected to particular disorders</DIV></DIV></DIV><H4><SPAN class="mw-headline" id="mitochondrial_diseases">mitochondrial diseases</SPAN></H4>
 Mitochondrial diseases are a group of disorders relating to the interrupted function of the mitochondrion. They comprise the disorders that affect the function of the proteins in the mitochondria or are due to mitochondrial DNA damage. Mitochondrial diseases take on unique characteristics both because of the way the diseases are often inherited and because mitochondria are pivotal to cell survivability. The subclass of these diseases that have neuromuscular disease symptoms are often referred to as a mitochondrial myopathies.</DIV><H2><SPAN class="mw-headline" id="P">P</SPAN></H2><H4><SPAN class="mw-headline" id="p53">p53</SPAN></H4><DIV class="glossary_text"><DIV class="thumb tright"><DIV class="thumbinner" style="width:242px;"><DIV class="thumbcaption">structure of monomeric p53 protein</DIV></DIV></DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:387px;"><DIV class="thumbcaption">Dimeric form of p53 interacting with DNA</DIV></DIV></DIV>p53 is a protein well-known amongst people interested in cancer. This protein is a tumor suppressor and one of the main factors involved in cell-cycle control. Its mutations are observed in more than half of tumors. Some data shows that the presence of p53 in mitochondria is able to induce <A href="https://2009.igem.org/Team:Warsaw/Glossary#apoptosis" class="external text" rel="nofollow">apoptosis</A>.
</DIV><H4><SPAN class="mw-headline" id="phoP.2FPhoQ">phoP/PhoQ</SPAN></H4><DIV class="glossary_text">PhoP/PhoQ is a two-component regulatory system which controls the virulence of <I>Salmonella typhimurium</I>. Under conditions of low pH and/or low metal ions concentration PhoP activates promoters responsible for virulence and survival of Salmonella within macrophages, like the <A href="https://2009.igem.org/Team:Warsaw/Glossary#mgtc_gene_promoter" class="external text" rel="nofollow">promoter of mgtc gene</A>. Whole system will be used to control bacteria escape from <A href="https://2009.igem.org/Team:Warsaw/Glossary#endosome" class="external text" rel="nofollow">endosome</A></DIV><H2><SPAN class="mw-headline" id="T">T</SPAN></H2><H4><SPAN class="mw-headline" id="Type_I_secretion_system">Type I secretion system</SPAN></H4><DIV class="glossary text"> Type I secretion system (TOSS) is a simple system, which consists of only three protein subunits: the ABC protein, membrane fusion protein (MFP), and outer membrane protein (OMP). Type I secretion system transports molecules of various size, from ions, drugs to even large proteins. The best characterized proteins secret via TOSS are the RTX toxins and the lipases. Type I secretion is also involved in export of non-proteinaceous substrates like cyclic β-glucans and polysaccharides. Many secreted proteins are particularly important in bacterial pathogenesis.</DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:317px;"><DIV class="thumbcaption">Tetrameric ABC protein</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:317px;"><DIV class="thumbcaption">ABC protein binding site and ATP molecule (white)</DIV></DIV></DIV><H4><SPAN class="mw-headline" id="TetR">TetR</SPAN></H4><DIV class="glossary text">TetR is an abbreviation of a family of bacterial transcriptional regulators which control the expression of genes responsible for resistance against tetracycline. Tetracyclines are amid of the most commonly used antibiotics and many gram-negative bacteria have developed mechanism of resistance against these antibiotics. The most abundant mechanism involved a membrane-associated proteins which exports tetracycline out of the bacterial membrane before it may act within the cell.</DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:317px;"><DIV class="thumbcaption">TetR protein with tetracycline molecule (silver)</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:317px;"><DIV class="thumbcaption">TetR protein bound to single-stranded DNA</DIV></DIV></DIV><DIV id="footer-div" style="display: none;">

(C) iGEM 2009 Warsaw Team
</DIV></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2009.igem.org/Team:Warsaw/Glossary">http://2009.igem.org/Team:Warsaw/Glossary</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Warsaw/Glossary" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Warsaw/Glossary" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Warsaw/Glossary&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Warsaw/Glossary&amp;oldid=119235" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2009.igem.org:Privacy_policy" title="2009.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2009.igem.org:General_disclaimer" title="2009.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>