<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Warsaw_Project_future"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:Warsaw/Project/future">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Warsaw/Project/future&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Warsaw/Project/future&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Warsaw/Project/future&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Warsaw/Project/future" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Warsaw/Project/future</H1><DIV id="bodyContent"><H3 id="siteSub">From 2009.igem.org</H3><DIV id="war_menu"><DIV class="war_menu_frame"><OL><LI class="main"><A href="/Team:Warsaw" title="Team:Warsaw">Home page</A></LI><P><LI class="main" onclick="switchMenu('team_submenu','menu_item')"><A name="team" style="cursor: hand; cursor: pointer;">The Team</A></LI></P><DIV id="team_submenu" style="display: none;"><LI class="sub"><A href="/Team:Warsaw/Team" title="Team:Warsaw/Team">About us</A></LI><LI class="sub"><A href="/Team:Warsaw/Team/Supervisors" title="Team:Warsaw/Team/Supervisors">Supervisors</A></LI><LI class="sub"><A href="/Team:Warsaw/Team/Miecznikowa" title="Team:Warsaw/Team/Miecznikowa">The M. Group</A></LI><LI class="sub"><A href="/Team:Warsaw/Team/Pawinskiego" title="Team:Warsaw/Team/Pawinskiego">The P. Group</A></LI></DIV><P><LI class="main" onclick="switchMenu('project_submenu')"><A name="project" style="cursor: hand; cursor: pointer;">The Project</A></LI></P><DIV id="project_submenu" style="display: block;"><LI class="sub"><A href="/Team:Warsaw/Project/introduction" title="Team:Warsaw/Project/introduction">Introduction</A></LI><LI class="sub"><A href="/Team:Warsaw/Project/invasion" title="Team:Warsaw/Project/invasion">Invasiveness</A></LI><LI class="sub"><A href="/Team:Warsaw/Project/endosome" title="Team:Warsaw/Project/endosome">Endosome Module</A></LI><LI class="sub"><A href="/Team:Warsaw/Project/cytoplasm" title="Team:Warsaw/Project/cytoplasm">Cytoplasmatic Module</A></LI><LI class="sub"><STRONG class="selflink">Future plans</STRONG></LI><LI class="sub"><A href="/Team:Warsaw/Bibliography" title="Team:Warsaw/Bibliography">Bibliography</A></LI></DIV><LI class="main"><A href="/Team:Warsaw/results" title="Team:Warsaw/results">Results</A></LI><LI class="main" onclick="switchMenu('modelling_submenu')"><A name="modelling" style="cursor: hand; cursor: pointer;">Modelling</A></LI><DIV id="modelling_submenu" style="display: none"><LI class="sub"><A href="/Team:Warsaw/Modelling" title="Team:Warsaw/Modelling">System Overview</A></LI><LI class="sub"><A href="/Team:Warsaw/Modelling/Apoptosis" title="Team:Warsaw/Modelling/Apoptosis">Apoptosis</A></LI><LI class="sub"><A href="/Team:Warsaw/Modelling/Structural" title="Team:Warsaw/Modelling/Structural">Structural</A></LI><LI class="sub"><A href="/Team:Warsaw/Modelling/Gene_Networks" title="Team:Warsaw/Modelling/Gene Networks">Gene Networks</A></LI><LI class="sub"><A href="/Team:Warsaw/Modelling/Invasiveness" title="Team:Warsaw/Modelling/Invasiveness">Invasiveness</A></LI></DIV><P><LI class="main" onclick="switchMenu('lab_submenu')"><A name="lab" style="cursor: hand; cursor: pointer;">Lab Work</A></LI></P><DIV id="lab_submenu" style="display: none"><LI class="sub"><A href="/Team:Warsaw/Notebook" title="Team:Warsaw/Notebook">Notebook</A></LI><LI class="sub"><A href="/Team:Warsaw/protocols" title="Team:Warsaw/protocols">Protocols</A></LI><LI class="sub"><A href="/Team:Warsaw/Resources" title="Team:Warsaw/Resources">Resources</A></LI></DIV><LI class="main"><A href="/Team:Warsaw/Parts" title="Team:Warsaw/Parts">Submitted Parts</A></LI><LI class="main"><A href="/Team:Warsaw/Glossary" title="Team:Warsaw/Glossary">Glossary</A></LI><LI class="main"><A href="/Team:Warsaw/Biosafety" title="Team:Warsaw/Biosafety">Biosafety</A></LI><LI class="main"><A href="/Team:Warsaw/Human" title="Team:Warsaw/Human">Human Practices</A></LI><LI class="main"><A href="/Team:Warsaw/Acknowledgements" title="Team:Warsaw/Acknowledgements">Acknowledgements</A></LI></OL></DIV><DIV class="war_menu_frame_bottom"> </DIV></DIV><DIV id="war_content"><H1><SPAN class="mw-headline" id="Future_plans">Future plans</SPAN></H1><P>In this place we want to present you how we want to improve our BacInVader system in the future. We've spent hours working on the idea of the project and we've realized that it's impossible to make everything we wanted to include in the project. During the wet lab work we have found out that it is even really hard to finish all steps we wanted to include in the the core of the project (described in the <A href="/Team:Warsaw/Project/introduction" title="Team:Warsaw/Project/introduction">Project</A> part of this website). We know that to make our system fully functional we should ensure that it will target only defective (for example tumor) cells. For safety purposes we should also take full control of bacteria action inside the cell, in particular we should include some &quot;kill-switch&quot; device for error-proofing. 
We also remember about our second idea of BacInvader functionality: conjugation with mitochondria. Further plan is also to move whole system to chassis better than popular <I>E. coli</I> - we think that<I>Bacillus subtilis</I> will be better for operating of BacInVader system.
</P><P>Below we discuss some of our future plans in more details
</P><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Future_plans"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Future plans</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Targeting_the_tumor_cells_via_recognition_the_specific_receptors_on_the_cell_surface"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Targeting the tumor cells via recognition the specific receptors on the cell surface</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Error_proofing_and_disposing_of_the_run_down_bacteria"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Error proofing and disposing of the run down bacteria</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Mitochondrial_transformation"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Mitochondrial transformation</SPAN></A></LI><LI class="toclevel-3 tocsection-5"><A href="#Theoretical_basis"><SPAN class="tocnumber">1.3.1</SPAN><SPAN class="toctext">Theoretical basis</SPAN></A></LI><LI class="toclevel-3 tocsection-6"><A href="#Details"><SPAN class="tocnumber">1.3.2</SPAN><SPAN class="toctext">Details</SPAN></A></LI></UL></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Targeting_the_tumor_cells_via_recognition_the_specific_receptors_on_the_cell_surface">Targeting the tumor cells via recognition the specific receptors on the cell surface</SPAN></H2><P>The specific delivery of drug molecule or larger particles like artificial liposomes, dendrimers or even whole bacterial cells  to tumor cells mainly relies on the potential of the associated vector to specifically head for the malignant center. This implies the identification of tumor markers which are appropriate target. One of the major strategy consists in targeting proteins that are highly specific or directly over-expressed on tumor cells. Nowadays,  the set of potentially interested ligands for the specifically targeted anticancer agents is remarkable large but only a few class of these markers appears to be promising as a targets for the new generation anticancer drugs. Nowadays plethora of proteins that are responsible for cellular signaling pathways are known, involved in the proliferation, angiogenesis, and differentiation in neoplasms with the targets amenable to therapeutic interventions in cancer therapy. Membrane-bound human epidermal growth factor receptors (HER), c-MET, and insulin-like growth factor 1 receptor (IGF-1R) mediate mitogenic signals from extracellular ligands, such as epidermal growth factor (EGF), hepatocyte growth factor (HGF), and insulin growth factors (IGF) are the most popular ligands.
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Abnormal expression of integrins in tumor</DIV></DIV></DIV><P>It is strongly believed that the most promising type of tumor-specific markers are vascular receptors mainly involved in the neoangiogenesis of the solid tumors and the promotion of cancer cells migration and their invasiveness. Angiogenesis, the process by which new blood capillaries are formed from preexisting vessels, is a hallmark of cancer. The process is a common feature to many solid tumors and angiogenic targets are easily accessible for systemic treatments and are genetically stable. These targets are also could be exploited for imaging and destructing tumor blood vessels. 
</P><P>Integrins are the crucial family of receptors for for cell-matrix adhesion and are also involved in various cell-cell adhesion processes. Furthermore, integrins create transmembrane connections to the cytoskeleton and are pivotal for working of many intracellular signaling pathways. Therefore, integrins and their ligands play key roles in development, immune response, cellular hemostasis and cancer. Equipped with these various functions, integrins are desired targets of effective therapeutic drugs and are objects of intensive continuous research in the area for development and targeting of drugs.
</P><P>Until now, 18 α and 8 β integrins subunits are known in mammalian cells to associate non-covalently forming 24 integrin heterodimers. Each type of subunit is a peculiar glycoprotein which consists of a large extracellular domain, a transmembrane helix and a short cytoplasmic tail. It should be remark that each of the 24 integrins appears to have a specific, nonredundant function (this can be concluded both from their interactors specificities and from phenotypes of knockout mice).
</P><P>The relevance of α5β1 integrin as a proangiogenic factor arises from some knock-out experiments. It is assumed that both fibronectin and aforementioned integrin directly regulate angiogenesis. Hence, it is reasonable to expect the α5β1 integrin to play an important role as a potent anticancer drug target. Additionally, recent studies suggested a key role for the α5β1  integrin in certain bacterial invasion of human host cells leading to antibiotic resistance. The bacteria exhibit on their surface proteins belong to the invasin family which are capable of selectively recognition of this integrins. These findings suggest that usage of bacteria containing the invasin outside of the cell membrane to targeting cancer cell is resonable idea.
</P><H2><SPAN class="mw-headline" id="Error_proofing_and_disposing_of_the_run_down_bacteria">Error proofing and disposing of the run down bacteria</SPAN></H2><P>During the course of planing we have given much thought to the issue of error proofing. When the project will be finalised we'll be dealing with a potentially dangerous microbe, capable of entering eukaryotic cells. This forced us develop a method of emergency killing the bacteria at any given time, a so called &quot;kill switch&quot;. We chose to use a toxin/antitoxin system (MazF/MazE for example) in which both proteins are produced at equal levels. The antitoxin will be expressed from a promoter that is easily inhibited by small size particles capable of penetrating inside the cell. If at any moment we decide to terminate the treatment the introduction of such a particle to the blood stream will cause the bacterium to halt the production of the antitoxin thus effectively killing itself.
</P><P>The other problem we have faced was what to do with the bacteria that have already completed their task. If such a bacterium finds itself absorbed by a white blood cell along with the apoptosomes it might start inducing apoptosis to its new host. Our solution to that problem was inspired by the Team Waterloo's 2008 project. The cytoplasmic operon could be supplemented with an additional endonucleolitic enzyme, one that would dispose of every unnecessary peace of DNA including bacteria's genome and previous operons. Such a vessel could only carry out the one final step (be it protein secretion or conjugation) over and over again until it depletes all its resources becoming eventually digested by either the host cell or along with the apoptotic remnants.
</P><H2><SPAN class="mw-headline" id="Mitochondrial_transformation">Mitochondrial transformation</SPAN></H2><H3><SPAN class="mw-headline" id="Theoretical_basis">Theoretical basis</SPAN></H3><P>Electron transport chain functionality can be impaired by a variety of defects on the DNA level: not only point mutation and local deletions in genes encoding proteins of the chain, but also deletions of larger fragments or even the whole mitochondrial genome (mtDNA). Normal mitochondrion contains from 2 to 10 copies of mtDNA. Defects in mitochondrial DNA result in severe medical disorders which are especially evident in tissues which require large amounts of energy (such as nervous and muscle tissue). Effects of these impairments are neuro and myopathies [20]. The standard therapy based on bypassing interrupted pathways does not apply under these circumstances. The administration of enzyme cofactors and other metabolically active substances is not largely effective and may be applied only as supplementary therapy. We are forced to seek novel treatment strategies to cure patients suffering from mitochondrial disorders [7]. Currently the most commonly used method of mitochondrial transformation is the &quot;gene gun&quot; approach which is based on the delivery of metal beads coated with DNA into the cell. This method is mostly limited to plants and fungi and has low therapeutic potential [7]. Current mammalian mitochondrial transformation techniques like electroporation also have serious limitations. Whole process takes place ex vivo and causes irreversible damage to the mitochondrion which makes its efficient reintroduction into the cell almost impossible. The conjugation between bacteria and mitochondria is free from these side effects. Moreover, this phenomenon has been preformed in vitro.[22]. More recently a conjugation between two strains of bacteria within the cytoplasm of a mammalian cell has been observed. These findings suggest that conjugation between a bacterium and a mitochondrion is feasible [10].
</P><H3><SPAN class="mw-headline" id="Details">Details</SPAN></H3><P>To locate the copy of the mitochondrial genome into bacterial cell the pBACrNESd plasmid will be used. It is based upon the Bacterial Artificial Chromosome. Replication origin (ori R6K), kanamycin resistance gen (kanR) and replication complex genes (repE, parA, parC) from pBACrNESd plasmid will be fused with origin of transfer (oriTF) from a natural conjugative F plasmid. Unique restriction site will be introduced into the mitochondrial genome by PCR reaction and the described construct will be inserted into mitochondrion. To enable conjugation proteins encoded on RP4 plasmid (Tc, MuKm, Tn7) will be placed in the cytoplasmic operon. These genes are responsible for pillus formation and transport of a single stranded DNA. The cell with this system will conjugate with any structure surrounded by a lipid bilayer. We will prepare mitochondrial DNA deficient cells using a previously described procedure [2]. The occurrence of conjugation will be confirmed by PCR reaction specific for sequences added to mitochondrial DNA. The origin of the DNA will be determined by analysis of restriction patterns. The expression of mitochondrial genes will be confirmed by semi-quantitative PCR [2].
</P><DIV id="footer-div" style="display: none;">

(C) iGEM 2009 Warsaw Team
</DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2009.igem.org/Team:Warsaw/Project/future">http://2009.igem.org/Team:Warsaw/Project/future</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Warsaw/Project/future" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Warsaw/Project/future" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Warsaw/Project/future&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Warsaw/Project/future&amp;oldid=161161" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2009.igem.org:Privacy_policy" title="2009.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2009.igem.org:General_disclaimer" title="2009.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>