<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Warsaw_Modeling_methods"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:Warsaw/Modeling_methods">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Warsaw/Modeling_methods&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Warsaw/Modeling_methods&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Warsaw/Modeling_methods&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Warsaw/Modeling_methods" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Warsaw/Modeling methods</H1><DIV id="bodyContent"><H3 id="siteSub">From 2009.igem.org</H3><DIV id="war_menu"><DIV class="war_menu_frame"><OL><LI class="main"><A href="/Team:Warsaw" title="Team:Warsaw">Home page</A></LI><P><LI class="main" onclick="switchMenu('team_submenu','menu_item')"><A name="team" style="cursor: hand; cursor: pointer;">The Team</A></LI></P><DIV id="team_submenu" style="display: {{{team}}};"><LI class="sub"><A href="/Team:Warsaw/Team" title="Team:Warsaw/Team">About us</A></LI><LI class="sub"><A href="/Team:Warsaw/Team/Supervisors" title="Team:Warsaw/Team/Supervisors">Supervisors</A></LI><LI class="sub"><A href="/Team:Warsaw/Team/Miecznikowa" title="Team:Warsaw/Team/Miecznikowa">The M. Group</A></LI><LI class="sub"><A href="/Team:Warsaw/Team/Pawinskiego" title="Team:Warsaw/Team/Pawinskiego">The P. Group</A></LI></DIV><P><LI class="main" onclick="switchMenu('project_submenu')"><A name="project" style="cursor: hand; cursor: pointer;">The Project</A></LI></P><DIV id="project_submenu" style="display: {{{project}}};"><LI class="sub"><A href="/Team:Warsaw/Project/introduction" title="Team:Warsaw/Project/introduction">Introduction</A></LI><LI class="sub"><A href="/Team:Warsaw/Project/invasion" title="Team:Warsaw/Project/invasion">Invasiveness</A></LI><LI class="sub"><A href="/Team:Warsaw/Project/endosome" title="Team:Warsaw/Project/endosome">Endosome Module</A></LI><LI class="sub"><A href="/Team:Warsaw/Project/cytoplasm" title="Team:Warsaw/Project/cytoplasm">Cytoplasmatic Module</A></LI><LI class="sub"><A href="/Team:Warsaw/Project/future" title="Team:Warsaw/Project/future">Future plans</A></LI><LI class="sub"><A href="/Team:Warsaw/Bibliography" title="Team:Warsaw/Bibliography">Bibliography</A></LI></DIV><LI class="main"><A href="/Team:Warsaw/results" title="Team:Warsaw/results">Results</A></LI><LI class="main" onclick="switchMenu('modelling_submenu')"><A name="modelling" style="cursor: hand; cursor: pointer;">Modelling</A></LI><DIV id="modelling_submenu" style="display: {{{modelling}}}"><LI class="sub"><A href="/Team:Warsaw/Modelling" title="Team:Warsaw/Modelling">System Overview</A></LI><LI class="sub"><A href="/Team:Warsaw/Modelling/Apoptosis" title="Team:Warsaw/Modelling/Apoptosis">Apoptosis</A></LI><LI class="sub"><A href="/Team:Warsaw/Modelling/Structural" title="Team:Warsaw/Modelling/Structural">Structural</A></LI><LI class="sub"><A href="/Team:Warsaw/Modelling/Gene_Networks" title="Team:Warsaw/Modelling/Gene Networks">Gene Networks</A></LI><LI class="sub"><A href="/Team:Warsaw/Modelling/Invasiveness" title="Team:Warsaw/Modelling/Invasiveness">Invasiveness</A></LI></DIV><P><LI class="main" onclick="switchMenu('lab_submenu')"><A name="lab" style="cursor: hand; cursor: pointer;">Lab Work</A></LI></P><DIV id="lab_submenu" style="display: {{{lab}}}"><LI class="sub"><A href="/Team:Warsaw/Notebook" title="Team:Warsaw/Notebook">Notebook</A></LI><LI class="sub"><A href="/Team:Warsaw/protocols" title="Team:Warsaw/protocols">Protocols</A></LI><LI class="sub"><A href="/Team:Warsaw/Resources" title="Team:Warsaw/Resources">Resources</A></LI></DIV><LI class="main"><A href="/Team:Warsaw/Parts" title="Team:Warsaw/Parts">Submitted Parts</A></LI><LI class="main"><A href="/Team:Warsaw/Glossary" title="Team:Warsaw/Glossary">Glossary</A></LI><LI class="main"><A href="/Team:Warsaw/Biosafety" title="Team:Warsaw/Biosafety">Biosafety</A></LI><LI class="main"><A href="/Team:Warsaw/Human" title="Team:Warsaw/Human">Human Practices</A></LI><LI class="main"><A href="/Team:Warsaw/Acknowledgements" title="Team:Warsaw/Acknowledgements">Acknowledgements</A></LI></OL></DIV><DIV class="war_menu_frame_bottom"> </DIV></DIV><DIV id="war_content"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1"><A href="#Short_description_of_used_programs_and_servers"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Short description of used programs and servers</SPAN></A></LI><LI class="toclevel-2"><A href="#Robetta_and_Rosetta"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Robetta and Rosetta</SPAN></A></LI><LI class="toclevel-2"><A href="#i-TASSER"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">i-TASSER</SPAN></A></LI><LI class="toclevel-2"><A href="#MUSTER"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">MUSTER</SPAN></A></LI><LI class="toclevel-2"><A href="#MODELLER"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">MODELLER</SPAN></A></LI><LI class="toclevel-2"><A href="#3D-Jury"><SPAN class="tocnumber">1.5</SPAN><SPAN class="toctext">3D-Jury</SPAN></A></LI><LI class="toclevel-2"><A href="#ModFOLD"><SPAN class="tocnumber">1.6</SPAN><SPAN class="toctext">ModFOLD</SPAN></A></LI></UL><LI class="toclevel-1"><A href="#References"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">References</SPAN></A></LI></TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Short_description_of_used_programs_and_servers">Short description of used programs and servers</SPAN></H3><H4><SPAN class="mw-headline" id="Robetta_and_Rosetta"><A href="http://robetta.bakerlab.org/" class="external text" rel="nofollow">Robetta and Rosetta</A></SPAN></H4><P>Robetta is a full-chain protein structure prediction server. It parses protein chains into putative domains with the Ginzu protocol, and models those domains either by <A href="https://2009.igem.org/Team:Warsaw/Glossary#comparative_modeling" class="external text" rel="nofollow">homology modeling</A> or by <A href="https://2009.igem.org/Team:Warsaw/Glossary#de_novo_modeling" class="external text" rel="nofollow"><I>de novo</I> modeling</A></P><P>The first part of the modeling process consists of determination of the locations of putative domains in the query
sequence, assignment of domains to the appropriate protocol, and identification of any likely homologs with experimentally characterized structures. The approach consists of scanning the target sequence with successively less confident methods to assign predicted domains. Once those regions are identified, cut points in the putative linkers are determined, if possible a single parent PDB chain is associated with each putative domain, and for each putative domain the  <A href="https://2009.igem.org/Team:Warsaw/Glossary#comparative_modeling" class="external text" rel="nofollow">homology modeling</A> or de novo protocol is subsequently initiated.
</P><P>Robetta uses specific alignment method to align the query sequence onto the parent structure. It then models the variable regions by allowing them to explore conformational space with fragments in fashion similar to the de novo protocol, but in the context of the template. When no structural homolog is available, domains are modeled with the Rosetta <I>de novo</I> protocol, which allows the full length of the domain to explore conformational space via fragment-insertion, producing a large decoy ensemble from which the final models are selected. To realize this attempt Robetta generates three- and nine-residue fragment libraries that represent local conformations seen in the PDB, and then assembles models by fragment insertion using a scoring function that favors protein-like features [1, 2].
</P><H4><SPAN class="mw-headline" id="i-TASSER"><A href="http://zhang.bioinformatics.ku.edu/I-TASSER/" class="external text" rel="nofollow">i-TASSER</A></SPAN></H4><P>Tasser is a full-chain proteins structure server which is base on iterative algoritm for <A href="https://2009.igem.org/Team:Warsaw/Glossary#de_novo_modeling" class="external text" rel="nofollow"><I>de novo</I> modeling</A>. After the protein submision , the server tries to retrieve template proteins of similar folds (or super-secondary structures) from the PDB library. 
</P><P>In the next step fragments excited from the PDB templates are reasembled into full-length models by replica-exchange Monte Carlo simulations with the threading unaligned regions built by <A href="https://2009.igem.org/Team:Warsaw/Glossary#de_novo_modeling" class="external text" rel="nofollow"><I>de novo</I> modeling</A>. In case where no appropriate template is identified I-TASSER will build whole structures without structural template.
</P><P>Finally, the fragments assembly simulation is performed again starting from cluster centroids where the spatial restrains collected by additional software are used to guide the simulations. The purpose of the second iteration is to remove the steric clash as well to refine the global topology of the cluster centroids. The decoys generated in this simulation are subsequently clustered and the structures with lowest energy are selected. The final atomic details are built from the selected decoys through the optimalization of the hydrogen-bonding network. [3, 4]
</P><H4><SPAN class="mw-headline" id="MUSTER"><A href="http://zhang.bioinformatics.ku.edu/MUSTER/" class="external text" rel="nofollow">MUSTER</A></SPAN></H4><P>MUSTER is a <A href="https://2009.igem.org/Team:Warsaw/Glossary#comparative_modeling" class="external text" rel="nofollow">threading</A> program which extend the secondary structure enhanced sequence profile-profile algorithm (PPA). It improve the PPA results due to addition some structural-derived features such as depth-dependent structure profiles ans hydrophobic scoring matrix. These features are implemented in the dynamic programming procedure [5].
</P><H4><SPAN class="mw-headline" id="MODELLER"><A href="http://salilab.org/modeller/" class="external text" rel="nofollow">MODELLER</A></SPAN></H4><P>MODELLER is a computer program which is used for <A href="https://2009.igem.org/Team:Warsaw/Glossary#comparative_modeling" class="external text" rel="nofollow">comparative modeling</A> of protein three-dimensional structures. The user provides an alignment of a to create a spatial constrains that are necessary to limit the exploration of conformational space. The restraints can operate on distances, angles, dihedral angles, pairs of dihedral angles and some other spatial features defined by atoms or pseudo atoms. Presently, MODELLER automatically derives the restraints only from the known related structures and their alignment with the target sequence.
</P><P>A 3D model is obtained by optimization of a molecular probability density function (PDF). The density function for comparative modeling is optimized with the variable target function procedure in Cartesian space that employs methods of conjugate gradients and molecular dynamics with simulated annealing. The PDFs restrain C alpha-C alpha distances, main-chain N-O distances, main-chain and side-chain dihedral angles. A smoothing procedure is used in the derivation of these relationships to minimize the problem of a sparse database. The 3D model of a protein is obtained by optimization of the molecular density function such that the model violates the input restraints only a little [6].
</P><P>This program can perform many additional tasks, including <A href="https://2009.igem.org/Team:Warsaw/Glossary#de_novo_modeling" class="external text" rel="nofollow"><I>de novo</I> modeling</A> of loops in protein structures, optimization of various models of protein structure with respect to a flexibly defined objective function, multiple alignment of protein sequences and/or structures, clustering, searching of sequence databases, comparison of protein structures [7].
</P><H4><SPAN class="mw-headline" id="3D-Jury"><A href="http://meta.bioinfo.pl/submit_wizard.pl" class="external text" rel="nofollow">3D-Jury</A></SPAN></H4><P>The 3D-Jury is a versatile system that generates meta-predictions from sets of models created using variable set of methods. It is not necessary to know prior characteristics of these methods.  It incorporates the comparison of models as the main processing step. It follows an approach similar to that employed in the field of ab initio fold recognition. 3D-Jury takes as input groups of models which are generated by a set of servers, however, neglecting the assigned confidence scores. All models are compared with each other and a similarity score is assigned to each pair which is enough congruent after optimal superposition [8]. 
</P><H4><SPAN class="mw-headline" id="ModFOLD"><A href="http://www.reading.ac.uk/bioinf/ModFOLD/" class="external text" rel="nofollow">ModFOLD</A></SPAN></H4><P>modFOLD is a tool for evaluation of models accuracy which rely on neural networks which is trained to discriminate between models based on the TM-score. The models for the training set were built from large alignments set using an in-house program, which simply mapped aligned residues in the target to the full backbone coordinates of the template and carried out renumbering. The target-template pairs were then generated from an all against all comparison of the sequences from non-redundant fold library [9]. 
</P><H3><SPAN class="mw-headline" id="References">References</SPAN></H3><OL><LI>Bonneau R, Strauss CE, Rohl CA, Chivian D, Bradley P, Malmström L, Robertson T, Baker D. De novo prediction of three-dimensional structures for major protein families. J Mol Biol. 2002 Sep 6;322(1):65-78. <A href="http://www.ncbi.nlm.nih.gov/pubmed/12215415" class="external text" rel="nofollow">PMID:12215415</A></LI><LI>Chivian D, Baker D. Homology modeling using parametric alignment ensemble generation with consensus and energy-based model selection. Nucleic Acids Res. 2006;34(17):e112. Epub 2006 Sep 13.Click here to read <A href="http://www.ncbi.nlm.nih.gov/pubmed/16971460" class="external text" rel="nofollow">PMID: 16971460</A></LI><LI>Zhang Y. I-TASSER server for protein 3D structure prediction. BMC Bioinformatics. 2008 Jan 23;9:40. <A href="http://www.ncbi.nlm.nih.gov/pubmed/18215316" class="external text" rel="nofollow">PMID: 18215316</A></LI><LI>Wu S, Skolnick J, Zhang Y. Ab initio modeling of small proteins by iterative TASSER simulations. BMC Biol. 2007 May 8;5:17. <A href="http://www.ncbi.nlm.nih.gov/pubmed/17488521" class="external text" rel="nofollow">PMID: 17488521</A></LI><LI>Wu S, Zhang Y. MUSTER: Improving protein sequence profile-profile alignments by using multiple sources of structure information.  Proteins. 2008 Aug;72(2):547-56. <A href="http://www.ncbi.nlm.nih.gov/pubmed/18247410" class="external text" rel="nofollow">PMID: 18247410</A></LI><LI>Sali A, Blundell TL. Comparative protein modelling by satisfaction of spatial restraints. J Mol Biol. 1993 Dec 5;234(3):779-815. <A href="http://www.ncbi.nlm.nih.gov/pubmed/8254673" class="external text" rel="nofollow">PMID: 8254673</A></LI><LI>Fiser A, Do RK, Sali A. Modeling of loops in protein structures.  Protein Sci. 2000 Sep;9(9):1753-73. <A href="http://www.ncbi.nlm.nih.gov/pubmed/11045621" class="external text" rel="nofollow">PMID: 11045621</A></LI><LI>Ginalski K, Elofsson A, Fischer D, Rychlewski L. 3D-Jury: a simple approach to improve protein structure predictions.  Bioinformatics. 2003 May 22;19(8):1015-8. <A href="http://www.ncbi.nlm.nih.gov/pubmed/12761065" class="external text" rel="nofollow">PMID: 12761065</A></LI><LI>McGuffin LJ. Benchmarking consensus model quality assessment for protein fold recognition. BMC Bioinformatics. 2007 Sep 18;8:345. <A href="http://www.ncbi.nlm.nih.gov/pubmed/17877795" class="external text" rel="nofollow">PMID: 17877795</A></LI></OL><DIV id="footer-div" style="display: none;">

(C) iGEM 2009 Warsaw Team
</DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2009.igem.org/Team:Warsaw/Modeling_methods">http://2009.igem.org/Team:Warsaw/Modeling_methods</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Warsaw/Modeling_methods" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Warsaw/Modeling_methods" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Warsaw/Modeling_methods&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Warsaw/Modeling_methods&amp;oldid=80913" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2009.igem.org:Privacy_policy" title="2009.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2009.igem.org:General_disclaimer" title="2009.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>