"
Page
Discussion
View source
History
teams
Log in
 
Team:Aberdeen Scotland/Parts/ 9001
From 2014.igem.org
Team:Aberdeen Scotland/Parts - 2014.ogem.org
Home
Team
Project
Parts
Modelling
Notebook
Safety
Attributions
Ethics & Outreach
Background
Created
Bba_K1352000
Bba_K1352001
Bba_K1352002
Bba_K1352003
Bba_K1352004
Bba_K1352006
Device Data
Improved
Bba_K759001
Bba_K542009
Bba_K346007
Bba_K1090000
Bba_T9002
Our characterisation of existing BioBrick BBa_K759001
Aggregation Module inducible by arabinose in E.coli
Aim
To verify Assembly Standard 10 compliance
Methods
1. Standard restriction digest and gel electrophoresis
Escherichia coli bacterial culture containing plasmids BBa_K759001 (coding region of Ag43 only), was inoculated overnight in LB-chloramphenicol medium in a shaking water bath at 370C. Plasmid mini preps were conducted with a Qiagen plasmid mini prep kit. Single and double restriction digests were carried out accordingly:
Restriction digests with EcoRI, XbaI, SpeI and PstI was performed to linearize the vector and to confirm the existence of unique restriction sites, as required by the iGEM biobrick Assembly standard 10 rules. Double restriction digests with EcoRI plus PstI and XbaI plus SpeI were performed to separate the BBa_K759001 insert from the pSB1C3 plasmid backbone. Digests were incubated at 37370C for 2 hours and analysed by gel electrophoresis. Theoretical maps were used to calculate the expected sizes of the fragments and to compare them against the gel electrophoresis results.
2. Sequencing data
The plasmid was sequenced by the DNA Sequencing Services at University of Dundee .Sequencing analysis was performed on BBa_K759001 to confirm the suspected additional PstI sites. 6 sequencing primers (see Fig.1) were designed that were located evenly across BBa_K759001 (sequence data was taken from the Registry of Standard Biological Parts) plus both BBa_G00100 (forward primer that amplifies from the prefix) and BBa_G00101 (reverse primer that amplifies from the suffix), as defined by iGEM, were used to sequence Ag43 flanked by the prefix and the suffix.
Results
Fig.1    Gel electrophoresis on standard restriction digests of BBa_K759001
Restriction digests were separated on a 0.8% agarose gel. Lane 1; NEB 1 kb size standards ladder, Lane 2; uncut plasmid, Lanes 3-6; plasmid cut with either EcoRI, XbaI, SpeI or PstI respectively, Lane 7; plasmid cut with EcoRI+PstI, Lane 8; plasmid cut with XbaI+SpeI.
Fig.2     Fragment sizes resulted from standard restriction digests on BBa_K759001
Assembly of the sequencing data was performed manually using the CLUSTAL OMEGA multiple alignment program. The resulting sequence (query) was then compared against the original sequence for BBa_K759001 (subject) (sequence of 4493 bp, as taken from the Registry of Standard Biological Parts), using BLAST Pairwise sequence alignment viewer:
Based on the sequencining data, a PstI restriction map of the pSB1C3-BBa_K759001 construct was drawn using SnapGene Viewer:
Conclusions
Both restriction digest and sequencing data show that biobrick BBa_K759001 contains 6 additional PstI sites (CTGCA^G) within the coding region of Antigen 43. Data suggests that the theoretical removal of the 6 additional sites from the coding region of Ag43 within BBa_K759001 is not confirmed by sequencing; therefore the construct is not standardized.
Based on the availability of the sequencing data and the results mentioned above, pSB1C3- BBa_K759001 has been chosen as the subject for removal of the PstI sites to ensure that an Ag43 BioBrick is created that meets Assembly Standard 10 requirements (see the RFC10 compatible version of this Biobrick created by 2014 iGEM Aberdeen team: Bba_K1352000)
Retrieved from " http://2014.igem.org/Team:Aberdeen_Scotland/Parts/_9001 "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
