"
Page
Discussion
View source
History
teams
Log in
 
Team:Aberdeen Scotland/Parts/ 2006
From 2014.igem.org
Team:Aberdeen Scotland/Parts - 2014.ogem.org
Home
Team
Project
Parts
Modelling
Notebook
Safety
Attributions
Ethics & Outreach
Background
Created
Bba_K1352000
Bba_K1352001
Bba_K1352002
Bba_K1352003
Bba_K1352004
Bba_K1352006
Device Data
Improved
Bba_K759001
Bba_K542009
Bba_K346007
Bba_K1090000
Bba_T9002
INP+FLAG
K1352006 is a part which has a FLAG-tag octapeptide flanked by a multiple cloning site (MCS) inserted into Bba_K523013 (K523013 expresses ice nucleation protein (INP)); specifically, on the end of a linker, which is in turn on the C-terminus of INP and is in the same reading frame. The FLAG-tag is a BglII restriction site, followed by a FLAG octapeptide, followed by a HindIII restriction site; the sequence of which is as follows; the octapeptide is uppercase, the restriction sites are lowercase:
(agatctGATTATAAAGATGATGATGATAAAaagctt)
Attaching a FLAG-tag to the end of INP means that it will be expressed on the surface of the cell, the purpose of this is to allow the rapid insertion (via the MCS) of polypeptides (an antigen for example) to the C-terminus end of INP; this allows the surface expression of said polypeptide. For the 2014 Aberdeen iGEM project, this part was intended to be used as a trypanosome antigen displayer.
Creation of INP-YFP-FLAG fragments followed by InFusion cloning
Four infusion primers (primers with one homologous half (lower case) to the template, and one “overhang” half (upper case), the last 15 nucleotides of which is homologous to another DNA fragment) were designed.
“INP-VEC-F” (CGCGGCCGCTTCTAGAttaatacgactcactataggg)
“INP-Rem-R” (aagctttttatcatcatcatctttataatcagatctTCCCGCCACGCTGC)
“INP-FLAG-F” (AGATCTGATTATAAAGATGATGATGATAAAAAGCTTtaataatactagcaacatatcataacggagtg)
“INP-VEC-R” (AGCGGCCGCTACTAGTtataaacgcagaaaggccc)
Two INP-YFP-FLAG fragments were created by PCR amplification, both use K523013 as the template, the first uses “INP-VEC-F” and “INP-FLAG-R” primers, the second uses “INP-VEC-R” and “INP-FLAG-F” primers. The Clontech InFusion kit was used to recombine the two INP-YFP-FLAG fragments with pSB1C3 backbone (cut with Xba1 and Spe1); this kit was followed according to the manufacturer’s instructions.
Restriction digest + Gel electrophoresis
Figure 1;  Xba1 + HindIII restriction digest screen of recombinants. The recombinant which went on to become K1352006 is in lane 5. The “L” lane is DNA marker (ladder). The arrows are to highlight the distance travelled by the HindIII-negative recombinants.
Figure 2; restriction digest verification of plasmid K1352006
LEGEND:
L 10,000bp – 500bp DNA marker “ladder”
N K1352006 plasmid digested with no enzymes
E K1352006 plasmid digested with EcoRI
X K1352006 plasmid digested with XbaI
S K1352006 plasmid digested with SpeI
P K1352006 plasmid digested with PstI
EP K1352006 plasmid digested with EcoRI and PstI
XS K1352006 plasmid digested with XbaI and Spe1
DNA Sequencing
The recombinant plasmid was Sanger-sequenced with the following sequencing primers. “G101” is a reverse primer, the rest are forward primers.
“G101” (attaccgcctttgagtgagc)
“G100” (tgccacctgacgtctaagaa)
“35 INP-SEQ 1” (ccgattcattaatgcagctgg)
“36 INP-SEQ 2” (gaggttgctgttgccgac)
“37 INP-SEQ 3” (ggtgtggaagccgacattc)
The plasmid insert was found to be exactly as desired. Highlighted below is the MCS excerpt from the “G101” primer results (“G101” is a reverse primer which is why the sequence is in reverse complement).
Conclusions
The part is RFC 10 compatible. The construct sequence was produced exactly as designed; the full FLAG-tag – with BglII and HindIII sites – is present at the N-terminus of the linker sequence after INP.
Retrieved from " http://2014.igem.org/Team:Aberdeen_Scotland/Parts/_2006 "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
