"
Page
Discussion
View source
History
teams
Log in
 
Team:Aberdeen Scotland/Parts/ 2000
From 2014.igem.org
Team:Aberdeen Scotland/Parts - 2014.ogem.org
Home
Team
Project
Parts
Modelling
Notebook
Safety
Attributions
Ethics & Outreach
Background
Created
Bba_K1352000
Bba_K1352001
Bba_K1352002
Bba_K1352003
Bba_K1352004
Bba_K1352006
Device Data
Improved
Bba_K759001
Bba_K542009
Bba_K346007
Bba_K1090000
Bba_T9002
Ag43 with PstI sites removed under the control
of pBad/araC promoter
Abstract
We describe the creation of BioBrick BBa_K1352000, the modularized version of BBa_K759001, which is compatible with iGEM Assemly Standard RCF10. Previous work by the Aberdeen 2014 iGEM team had clearly shown that both Ag43 containing BioBricks BBa_K346007 and BBa_K759001 contained the 6 native PstI sites that characterise the native Ag43 sequence. Also, BBa_K542009 was shown to contain an additional XbaI site and to have a smaller size than predicted. They were thus found to be non-compliant with assembly Standard 10. We describe the design and construction of the new BioBrick BBa_K1352000, and its verification using restriction digestion and DNA sequencing. Finally, we show that antigen 43 protein expression of biobrick BBa_K1352010 was not affected by the PCR-based Site-Directed Mutagenesis process. The biobrick works biologically as intended after modification and has the same response to arabinose induction: formation of culture aggregation.
1. Structure and function
Antigen 43 (Ag43), the product of the flu gene, is a cell-surface autotransporter protein found in Escherichia coli. It is expressed at about 50, 000 copies/cell and is initially synthesised as a precursor of 1039 amino acids. Upon removal of the signal peptide, the protein is transported to the cell surface and is composed of an α subunit (499 amino acids) at the N-terminus and a β subunit (488 amino acids) at the C-terminus (Kj\aergaard et al., 2002). Ag43 is mainly known to induce cell-to-cell aggregation and be involved in biofilm formation. However, as the necessary information required for auto transportation resides in the protein itself (Kj\aergaard et al., 2002), the main of our project was to use it as a platform for displaying specific peptides on the surface of E. coli. To accomplish this aim, we have modularized and characterized Biobrick BBa_K759001.
Fig.1    Representation of the autotransportation process through the Ag43 autotransporter.
Source: Westfälische Wilhelms-Universität Münster, I. (2014).
2. Aim
To eliminate unwanted PstI sites from a pre-existing, non-compliant Ag43 BioBrick (BBa_K759001)
3. Methods and Results
a. Site-Directed Mutagenesis
Synonymous mutations at the 6 additional PstI sites were thus introduced through PCR-based Site-Directed Mutagenesis (Agilent Lightning Quik-Change mutagenesis kit) in the BBa_K759001 sequence, by using the following set of primers:
Highlighted nucleotides represent the synonymous mutations introduced within the coding region of Ag43, to remove the PstI sites. For primer pairs 1,2,3,4 and 6, the restriction site for the PstI site (CTGCA^G) has been converted to CTGCAA. The amino acid glutamine was therefore preserved in the open reading frame of the sequence and has been selected according to the higher frequency of codon usage in E. coli. For primer pair 5, CTGCA^G has been changed to CTGCGG, to maintain the open reading frame of Ag43. By changing GCA TO GCC, alanine was conserved within the sequence.
b.  Restriction digest and gel electrophoresis
Escherichia coli bacterial cultures containing both plasmids BBa_K759001 and BBa_K1352000 were inoculated overnight in LB-chloramphenicol medium in a shaking water bath at 37oC. Plasmid mini preps were conducted with a Qiagen plasmid mini prep kit. Single and double restriction digests were carried out accordingly:
Restriction digests with EcoRI, XbaI, SpeI and PstI was performed to linearize the vector and to confirm the existence of unique restriction sites, as required by the iGEM biobrick Assembly standard 10 rules. Double restriction digests with EcoRI plus PstI and XbaI plus SpeI were performed to separate the inserts from the pSB1C3 plasmid backbone. Digests were incubated at 37oC for 2 hours and analysed by gel electrophoresis. Theoretical maps were used to calculate the expected sizes of the fragments and to compare them against the gel electrophoresis results.
Fig.2    Gel electrophoresis on standard restriction digests of BBa_K759001 (top tier) versus BBa_K1352000 (bottom).
Restriction digests were separated on a 1% agarose gel. Lane 1; NEB 1 kb size standards ladder, Lane 2; uncut plasmid, Lanes 3-6; plasmid cut with either EcoRI, XbaI, SpeI or PstI respectively, Lane 7; plasmid cut with EcoRI+PstI, Lane 8; plasmid cut with XbaI+SpeI, Lane 9; NEB 1 kb size standards ladder.
c.  Sequencing data and plasmid map
The plasmid was sequenced by the DNA Sequencing Services at University of Dundee. Sequencing analysis was further performed on BBa_ K1352000 with the following sequencing primers:
Assembly of the sequencing data was performed manually using the CLUSTAL OMEGA multiple alignment program. The resulting sequence (query) was then compared against the sequenced BBa_K759001 (subject) using BLAST Pairwise sequence alignment viewer:
Data suggests that the removal of the 6 additional sites through Site-Directed Mutagenisis from the coding region of Ag43 within BBa_K759001 is confirmed by sequencing.
The ExPASy Translate Tool has been used to translate the nucleotide sequences of both the BBa_K759001 and BBa_K1352000 constructs. The protein BLAST alignment tool has consequently used to compare the open reading frames of the two constructs. No indels were observed within the coding sequence of the BBa_K1352000 construct.
d. Physiology of BBa_K1352000
Both BBa_K1352000 and a negative control were grown overnight in 5 mls LB medium+ chloramphenicol. In the morning, both liquid cultures were induced with 0.2% arabinose for over 2 hours to detect aggregation. The E.coli culture aggregates (as shown below, Figure 1), demonstrating the engineered gene now lacking PstI sites is functional.
Fig.3    Physiology of BBa_K1352000 in response to 0.2% arabinose induction.
Left: BioBrick BBa_K1352000; right: negative control (untransformed E.coli).
4. Conclusion
BBa_K1352010 is the correct, iGEM RFC10-compatible version of BBa_K759001. In our experience, BBa_K759001 is not RFC10 compatible, having 6 Pst1 sites (see 'Design' for BBa_K1352010). During the construction of BBa_K1352010, the removal of the 6 additional PstI sites through Site-Directed Mutagenesis was confirmed by gel electrophoresis and sequence analysis, with no indels being introduced within the coding region of the construct.
Antigen 43 protein expression of biobrick BBa_K1352010 was shown not to be affected by the PCR-based Site-Directed Mutagenesis process. The biobrick works biologically as intended after modification and has the same response to arabinose induction: formation of culture aggregation.
References
1.      Kjaergaard, K., Hasman, H., Schembri, M. and Klemm, P. (2002). Antigen 43-mediated autotransporter display, a versatile bacterial cell surface presentation system. Journal of bacteriology, 184(15), pp.4197--4204. 2.Westfälische Wilhelms-Universität Münster, I. (2014). Topics of Research. [online] Uni-muenster.de. Available at: http://www.unimuenster.de/Chemie.pz/forschen/ag/jose/topicsofresearch.html, [Accessed 29 Sep. 2014].
Retrieved from " http://2014.igem.org/Team:Aberdeen_Scotland/Parts/_2000 "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
