Team:INSA-UPS France/Model/Simulation
Croc'n Cholera
iGEM UPS-INSA Toulouse 2017
Team
Members
Supports
Attributions
Collaborations
Strategy
Description
Design
Parts
Model
Overview
Simulation
Analysis
Interface
Realisations
Overview
Notebook
Clonings
Results
Contribution
Protocols
Safety
Human Practices
Overview
Silver
Gold
Public Engagement
Microbioworld
Application
Applied Design
Testimonies
Entrepreneurship
Device Conception
Simulation
Kinetic model
Data and parameters
Solver
Simulation results
References
Kinetic model
Every biological and physical processes including in the SBGN representation of the model was described mathematically using appropriate kinetic rate laws (passive diffusion, Michaelis-Menten enzyme kinetics, first order degradation of mRNAs, exponential growth law, etc). The final model contains 12 entities (genes, RNAs, proteins, metabolites, and cells) and 24 reactions from a broad range of processes (transcription, translation, RNAs and proteins degradation, growth and death of each micro-organism, metabolite poduction, transport between compartments, diffusion, etc). The dynamics of our microbial consortium was summed up in twelve differential equations as a system of ODEs.
For fearless people, our complete mathematical model and demonstration can be found there ! The final set of ODEs is briefly described below.
SBGN representation of the model developed in the project
We need to consider the growth and death of the three microorganisms: Vibrio cholerae (Vc) in water (W), and Vibrio harveyi (Vh) and Pichia pastoris (Pp) in the device (D). \begin{equation}     \frac{d[\textit{Vc}]_W}{dt} = V_{growth,Vc} - V_{death,Vc} \end{equation} \begin{equation}     \frac{d[\textit{Vh}]_D}{dt} = V_{growth,Vh} - V_{death,Vh} \end{equation} \begin{equation}     \frac{d[\textit{Pp}]_D}{dt} = V_{growth,Pp} - V_{death,Pp} \end{equation}
Microorganisms death is impacted by antimicrobial peptides production (AMPpeptide,Pp), produced by translation of antimicrobial peptides mRNA (AMPRNA). Peptides and mRNA are also degraded. \begin{equation}     \frac{d[AMP_{RNA}]_{Pp}}{dt}=V_{transcription,AMP} - V_{degradation,AMP RNA} \end{equation}  \begin{equation}     \frac{d[AMP_{peptide}]_{D}}{dt}=V_{translation,AMP} - V_{degradation,AMP} + \frac{V_{diff,AMP,W \to D}}{\mathcal{V}_D} \end{equation}
These peptides are transfered between compartments: from the device (D) to water (W).  \begin{equation}     \frac{d[AMP]_W}{dt} = -V_{diff,AMP,W\to D} \end{equation}
To produce antimicrobial peptides, an activation by diacetyl (dac) is needed. Diacetyl can freely diffuse from the device (D) to water (W). \begin{equation}     \frac{d[dac]_D}{dt}=V_{prod,dac}+\frac{V_{diff,dac,W \to D}}{\mathcal{V}_D} \end{equation}  \begin{equation}     \frac{d[dac]_W}{dt}=- V_{diff,dac,W \to D} \end{equation}
Diacetyl is produced by the enzyme acetolactate synthase (ALSenzyme). The als gene is first transcribed into ALSRNA, which is then translated into the protein. Both the enzyme and its mRNA can be degraded. \begin{equation}     \frac{d[ALS_{RNA}]_{Vh}}{dt} = V_{transcription,ALS} - V_{degradation,ALS RNA} \end{equation}  \begin{equation}     \frac{d[ALS_{enzyme}]_{Vh}}{dt} = V_{translation,ALS} - V_{degradation,ALSenzyme} \end{equation}
ALS production has to be activated by the quorum sensing molecule CAI-1, initially in water (W), after diffusing into the device (D).   \begin{equation}     \frac{d[CAI\text{-}1]_D}{dt} = \frac{V_{diff,CAI\text{-}1,W\to D}}{\mathcal{V}_D} \end{equation}  \begin{equation}     \frac{d[CAI\text{-}1]_W}{dt} = -V_{diff,CAI\text{-}1,W\to D} \end{equation}
Data and Parameters
Model parameters were mostly collected from publications, because a large part of the required data necessitates a complex set of hundreds of experiments that could not have been performed in the course of the project.
Name
Notation
Unit
Value
Reference
Vibrio cholerae maximum growth rate
μMAX,Vc
s-1
3.10-4
BioNumbers ID 112369 1
Leucrocine I MIC for V. cholerae
MICLeucro,Vc
mol/L
6.4.10-5
Pata et al., 2011 2
Leucrocine I MIC for V. harveyi
MICLeucro,Vh
mol/L
6.4.10-5
Pata et al., 2011 2 - Extrapolation from V. cholerae result
Leucrocine I MIC for P. pastoris
MICLeucro,Pp
mol/L
1015
Assuming no effects on Pichia pastoris
Leucrocine I IC50 for V. cholerae
MICLeucro,Vc
mol/L
1.92.10-4
Considering IC50 = 3.MIC - Extrapolation of data from standard antibiotics) 3
Leucrocine I IC50 for V. harveyi
IC50Leucro,Vh
mol/L
1.92.10-4
Considering IC50 = 3.MIC - Extrapolation of data from standard antibiotics 3
Leucrocine I IC50 for P. pastoris
IC50Leucro,Pp
mol/L
1015
Assuming no effects on Pichia pastoris
cOT2 MIC for V. cholerae
MICcOT2,Vc
mol/L
8.1.10-6
Prajanban et al., 2017 4
cOT2 MIC for V. harveyi
MICcOT2,Vh
mol/L
8.1.10-6
Prajanban et al., 2017 4 - Extrapolation from V. cholerae result
cOT2 MIC for P. pastoris
MICcOT2,Pp
mol/L
1015
Assuming no effects on Pichia pastoris
cOT2 IC50 for V. cholerae
IC50cOT2,Vc
mol/L
2.43.10-5
Considering IC50 = 3.MIC - Extrapolation of data from standard antibiotics 3
cOT2 IC50 for V. harveyi
IC50cOT2,Vh
mol/L
2.43.10-5
Considering IC50 = 3.MIC - Extrapolation of data from standard antibiotics 3
cOT2 IC50 for P. pastoris
IC50cOT2,Pp
mol/L
1015
Assuming no effects on Pichia pastoris
D-NY15 MIC for V. cholerae
MICD-NY15,Vc
mol/L
1.54.10-5
Yaraksa et al., 2014 5
D-NY15 MIC for V. harveyi
MICD-NY15,Vh
mol/L
1.54.10-5
Yaraksa et al., 2014 5 - Extrapolation from V. cholerae result
D-NY15 MIC for P. pastoris
MICD-NY15,Pp
mol/L
1015
Assuming no effects on Pichia pastoris
D-NY15 IC50 for V. cholerae
IC50D-NY15,Vc
mol/L
4.62.10-5
Considering IC50 = 3.MIC - Extrapolation of data from standard antibiotics 3
D-NY15 IC50 for V. harveyi
IC50D-NY15,Vh
mol/L
4.62.10-5
Considering IC50 = 3.MIC - Extrapolation of data from standard antibiotics 3
D-NY15 IC50 for P. pastoris
IC50D-NY15,Pp
mol/L
1015
Assuming no effects on Pichia pastoris
V. cholerae death rate with AMP
kkill,Vc
s-1
3.10-3
Yaraksa et al., 2014 5
V. harveyi death rate with AMP
kkill,Vh
s-1
3.10-3
Extrapolation from Yaraksa et al., 2014 5
P. pastoris death rate with AMP
kkill,Pp
s-1
0
Assuming no effects
Transfer coefficient through the membrane
K
s-1
1
Arbitrary value
Number of als gene per cell
alsDNA,0
Nb/cell
15
Considering a low copy plasmid 6
Number of AMP gene per cell
AMPDNA,0
Nb/cell
1
Protocol: genomic integration
V. harveyi transcription rate
ktranscript,Vh
nt/s
30
Molecular Biology course, Transcription - Faculté des Sciences - Rabat 7
Vibrio harveyi transcription rate
ktranslation,Vh
nt/s
15
Molecular Biology course, Translation - Faculté des Sciences - Rabat 8
als gene promoter influence
kP,als
/
1
Inductible promoter
AMP gene promoter influence
kP,AMP
/
1
Inductible promoter
mRNA degradation constant
Kdeg,mRNA
s-1
5.10-3
Esquerré et al., 2015 9
als gene length
DNA length
nucleotides
1662
UniProtKB - Q7DAV2 10
als mRNA length
RNA length
nucleotides
1730
Parts design
Number of CqsS* receptor per cell
CqsS*/cell
Nb/cell
1015
Arbitrary value
Number of Odr10 receptor per cell
Odr10/cell
Nb/cell
1015
Arbitrary value
Pichia pastoris transcription rate
ktranscript,Pp
nt/s
50
Molecular Biology course - INSA Toulouse(11)
Pichia pastoris translation rate
ktranslation,Pp
nt/s
48
Molecular Biology course, Translation - Faculté des Sciences - Rabat 8
AMP gene length
DNA length
nucleotides
360
Parts design
AMP mRNA length
RNA length
nucleotides
651
Parts design
Vibrio cholerae minimal pathogenic concentration
[Vc]pathogenic
cell/L
4.104
Medical Microbiology 4th edition (12)
CAI-1 initial concentration
[CAI-1]0
mol/L
1.10-5
Ng et al., 2011 13
Michaelis-Menten constant of acetolactate synthase
KM,ALS
mol/L
1.36.10-2
Atsumi et al., 2009 14
Catalytic rate constant of acetolactate synthase
kcat,ALS
s-1
1.21.102
Atsumi et al., 2009 14
Degradation constant of acetolactate synthase
Kdeg,ALS
s-1
0
Assuming a negligeable value in our conditions
Degradation constant of antimicrobial peptides
Kdeg,AMP
s-1
0
Aleinein et al., 2013 15
Activation constant of CqsS*-CAI-1 complex
Ka,CqsS*-CAI-1
mol/L
3.6.10-8
Ng et al., 2011 13
High Vibrio cholerae concentration in water
[Vc]0
cell/L
107
Huq et al., 1984 16
Dry weight of a bacterial cell
DWVh
pg/cell
0.28
BioNumbers ID 100008
Volume of a bacterial cell
Vintra,Vh
μm3
1
BioNumbers ID 100004
Volume of a yeast cell
Vintra,Pp
μm3
66
BioNumbers ID 100452
Dry weight of a yeast cell
DWPp
pg/cell
18.48
Estimation from yeast cell volume and ratio dry weight/volume for a bacterial cell
Ribosome density on a yeast cell
Nb/kb
6.5
BioNumbers ID 103026
Number of ribosome on AMP mRNA
Ribosome/RNA
Nb/RNA
2.34
Deduced from ribosome density and mRNA length
RNA polymerase density on a yeast gene
Nb/kb
6.5
BioNumbers ID 103026
Number of RNA polymerase on AMP DNA
RNA polymerase/DNA
Nb/DNA
4.95
BioNumbers ID 108308
Ribosome density on a bacterial mRNA
Nb/kb
6.6
BioNumbers ID 107727
Number of ribosome on als mRNA
Ribosomes/RNA
Nb/RNA
11
Deduced from ribosome density and mRNA length
RNA polymerase density on a bacterial gene
Nb/kb
7.6
Assuming the same density than yeast (BioNumbers ID 108308)
Number of RNA polymerase on als DNA
RNA Polymerase/DNA
Nb/DNA
13
Deduced from RNA polymerase density and DNA length
Pyruvate concentration in a bacterial cell
[S]
mol/L
3.9.10-4
BioNumbers ID 101192
Some preliminary experimental results were also expoited to refine important parameters of the model and verify some of our hypothesis. Indeed, we needed an experimental estimation of the growth rate and lag time of the two chassis microorganisms grown on a common medium, and these parameters were measured experimentally. The membrane diffusion tests we have performed also revealed a rapid molecular diffusion (equilibrium reached in less than one minute). This result confirmed the initial, arbitrary hypothesis of a high transfer coefficient (K) which would not slow down the system's dynamics.
Name
Notation
Unit
Value
Reference
Vibrio harveyi JMH626 maximum growth rate
μMAX,Vh
s-1
2.10-4
Experiment - 21/06/17
Pichia pastoris SMD1168 maximum growth rate
μMAX,Pp
s-1
4.10-5
Experiment - 21/06/17
Vibrio cholerae lag time
tl,Vc
s
1800
Experiment - 21/06/17, extrapolation from V. harveyi growth
Vibrio harveyi lag time
tl,Pp
s
1800
Experiment - 21/06/17
Pichia pastoris lag time
tl,Pp
s
1400
Experiment - 21/06/17
Solver
The system of ODEs was solved using Matlab R2017a, thanks to the free offer from iGEM. We used the ode15s solver, which was in this situation more efficient than the ode45 solver, very likely because the different time scales between some processes (e.g. slow growth - in the range of hours - vs fast signalling - in the range of milliseconds -) renders the problem partially stiff. 17 The proposed implementation proved to be very efficient, with simulations performed in less than 1 second.
You can freely re-use our code: General_resolution + System_of_ODEs .
Simulation results
At the beginning of the project, we needed to know if our microbial synthetic consortium would work in practice and if the information transmission between the different modules was possible and sufficiently fast. We thus carried out simulations by solving the ODEs system to have a first estimation of the dynamics of our synthetic system.
The initial conditions, such as the concentrations of Vibrio harveyi and Pichia pastoris in the device and the device volume, were set to biologically plausible values. Vibrio cholerae initial concentration was set to 4.107 cell/L,  which is in the range of concentrations observed in contaminated water. 16
[Vh]0,D = 1012 cell/L (OD600nm ≈ 1.5 18 )
[Pp]0,D = 1012 cell/L (OD600nm ≈ 1.5 18 )
[Vc]0,W = 4.107 cell/L
VD = 20.10-3 L
Temporal evolution of Vibrio cholerae concentration
under realistic conditions (device size, initial concentrations of other microorganisms in the device)
The response time to reach a non-pathogenic concentration is estimated to 53.6 min. This result confirmed the feasibility of our project: our microbial consortium is predicted to efficiently and rapidly sense the presence of V. cholerae, transmit this information to P. Pastoris which, in response, may produce enough antimicrobial peptides to kill V. cholerae in less than one hour.
Evolution of the concentration of each entity considered in the model
This visual representation of the system's dynamics also allowed us to check that each variable evolves in a realistic range of concentrations, hence indicating the model predicts a consistent behavior. Further analyses were then perfomed to better understand the system functioning, and in particular to evaluate its sensibility and robustness. We first tested whether the synthetic system would be robust to the variation of some parameters that could arise during the manufacturing or transport processes. Then, we identified parameters that control the response time (time to reach a non-pathogenic concentration) to guide the rational design of the system and of the device.
→ Sensitivity analysis methods and their results are described in our model analysis page
References
(Milo R, Jorgensen P, Moran U, Weber G, Springer M. BioNumbers—the database of key numbers in molecular and cell biology. Nucleic Acids Res. 2010;38(suppl 1):D750–D753
https://www.ncbi.nlm.nih.gov/pubmed/19854939
Pata S, Yaraksa N, Daduang S, Temsiripong Y, Svasti J, Araki T, Thammasirirak S, Characterization of the novel antibacterial peptide Leucrocin from crocodile (Crocodylus siamensis) white blood cell extracts. Dev. Comp. Immunol. 2011, 35, 545–553
https://www.ncbi.nlm.nih.gov/pubmed/21184776
Farrag A, Elsehrawi H, Younes A, Synthesis, Antimicrobial Evaluation and Docking Study of Novel Heterocyclic Compounds Bearing a Biologically Active Sulfonamide Moiety. J. Pharm. Appl. Chem. 2015, 1, No. 1, 27-35
http://www.naturalspublishing.com/files/published/23lc4s4b389433.pdf
Prajanban B, Jangpromma N, Araki T, Klaynongsruang S, Antimicrobial effects of novel peptides cOT2 and sOT2 derived from Crocodylus siamensis and Pelodiscus sinensis ovotransferrins. Biochimica et Biophysica Acta 1859. 2017, 860–869
https://www.ncbi.nlm.nih.gov/pubmed/28159460
Yaraksa N, Anunthawan T, Theansungnoen T, Daduang S, Araki T, Apisak D, Sompong T, Design and synthesis of cationic antibacterial peptide based on Leucrocin I sequence, antibacterial peptide from crocodile (Crocodylus siamensis) white blood cell extracts. Journal of Antibiotics. Mar 2014, Tokyo 67.: 205-212.
https://www.ncbi.nlm.nih.gov/pubmed/24192554
QIAGEN, Origins of replication and copy numbers of various plasmids and cosmids In: Growth Of Bacterial Cultures, 2013 - 2017. https://www.qiagen.com/dk/resources/technologies/plasmid-resource-center/growth%20of%20bacterial%20cultures/
Belkadi B, Expression génétique - Transcription In: Biologie Moléculaire, Faculté des Sciences - Rabat, 2010-2011. http://www.fsr.ac.ma/cours/biologie/BELKADI/transcription.pdf
Belkadi B, Expression génétique - Traduction In: Biologie Moléculaire, Faculté des Sciences - Rabat, 2009-2010. http://www.fsr.ac.ma/cours/biologie/BELKADI/traduction.pdf
Esquerre T, Moisan A, Chiapello H, Arike L, Vilu R, Gaspin C, Cocaign-Bousquet M, Girbal L, Genome-wide investigation of mRNA lifetime determinants in Escherichia coli cells cultured at different growth rates. BMC Genomics. 2015, 16, 275,
https://www.ncbi.nlm.nih.gov/pubmed/25887031
UNIPROT, Q7DAV2, Alpha-acetolactate synthase - Lactococcus lactis subsp. lactis (strain IL1403) : http://www.uniprot.org/uniprot/Q7DAV2
Capp JP, Chap.3 : Transcription In: Cours de Biologie Moléculaire, INSA Toulouse, 2016
Baron S, editor. Medical Microbiology. 4th edition. Galveston (TX): University of Texas Medical Branch at Galveston; 1996.
Ng WL, Perez LJ, Wei Y, Kraml C, Semmelhack MF, Bassler BL, Signal production and detection specificity in Vibrio CqsA/CqsS quorum-sensing systems. Mol. Microbiol. 2011, 79: 1407–1417.
https://www.ncbi.nlm.nih.gov/pubmed/21219472
Atsumi S, Li Z, Liao JC, Acetolactate synthase from Bacillus subtilis serves as a 2-ketoisovalerate decarboxylase for isobutanol biosynthesis in Escherichia coli. Appl. Environ. Microbiol. 2009a, 75:6306–6311
https://www.ncbi.nlm.nih.gov/pubmed/21219472
Aleinein R.A., Schäfer H., Wink M., Secretory ranalexin produced in recombinant Pichia pastoris exhibits additive bactericidal activity when used in combination with polymyxin B or linezolid against multi-drug resistant bacteria. Biotechnol. J. 2014, 9, 110–119.
https://www.ncbi.nlm.nih.gov/pubmed/24166764
Huq A., West, P. A., Small, E. B., Huq, M. I., and Colwell, R. R., Influence of water temperature, salinity and pH on survival and growth of toxigenic Vibrio cholerae serovar O1 associated with live copepods in laboratory microcosms, Appl. Environ. Microbiol. 1984, 48: 420–424.
https://www.ncbi.nlm.nih.gov/pubmed/6486784
MATHWORKS, Basic Solver Selection In: Choose an ODE Solver: https://fr.mathworks.com/help/matlab/math/choose-an-ode-solver.html
AGILENT. E. coli Cell Culture Concentration from OD600 Calculator http://www.genomics.agilent.com/biocalculators/calcODBacterial.jsp
Model pages
Overview
Simulation
Analysis
Interface
