<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_INSA-UPS_France_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:INSA-UPS France/Results</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container" style="position:relative; "><DIV class="site-header__title">
    Croc'n Cholera <SPAN style="font-size:12pt;">iGEM UPS-INSA Toulouse 2017</SPAN></DIV><NAV class="site-menu" style="text-align: center;"><DIV class="site-menu__item"><DIV class="cat_name">Team </DIV><UL class="subnav" style="width:150%;"><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Team">Members</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Supports">Supports</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Attributions">Attributions</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Collaborations">Collaborations</A></LI></UL></DIV><DIV class="site-menu__item"><DIV class="cat_name">Strategy </DIV><UL class="subnav"><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Description">Description</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Design">Design</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Parts">Parts</A></LI></UL></DIV><DIV class="site-menu__item"><DIV class="cat_name">Model </DIV><UL class="subnav"><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Model">Overview</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Model/Simulation">Simulation</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Model/Analysis">Analysis</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Model/Interface">Interface</A></LI></UL></DIV><DIV class="site-menu__item"><DIV class="cat_name">Realisations </DIV><UL class="subnav"><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Experiments">Overview</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Notebook">Notebook</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Experiments/Clonings">Clonings</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Results">Results</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Contribution">Contribution</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Protocols">Protocols</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Safety">Safety</A></LI></UL></DIV><DIV class="site-menu__item"><DIV class="cat_name">Human Practices</DIV><UL class="subnav"><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Human_Practices">Overview</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/HP/Silver">Silver</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/HP/Gold_Integrated">Gold</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Engagement">Public Engagement</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Microbioworld">Microbioworld</A></LI></UL></DIV><DIV class="site-menu__item"><DIV class="cat_name">Application </DIV><UL class="subnav"><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Applied_Design">Applied Design</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Entrepreneurship/Testimonies">Testimonies</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Entrepreneurship">Entrepreneurship</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Entrepreneurship/Device">Device Conception</A></LI></UL></DIV></NAV></DIV><DIV class="main_content"><DIV class="middle_container"><DIV class="section_container"><SECTION style="display:table;background:none;padding:0px !important;z-index:100; "><H1 style="vertical-align:bottom;display:table-cell; width:60%;font-size:40pt;letter-spacing: 0.1em;z-index:120;text-align: center;">Results</H1></SECTION><DIV class="left_container"><DIV class="left_container__inside"><DIV class="aside-nav__item"><A href="#result1" data-number="1">
1. Mimicking Vibrio sp. presence</A></DIV><DIV class="aside-nav__item"><A href="#result2" data-number="2">
2. <I>E. coli</I> producing C8-CAI molecules sensed by <I>V. harveyi</I></A></DIV><DIV class="aside-nav__item"><A href="#result3" data-number="3">
3. Modification of <I>V. harveyi</I></A></DIV><DIV class="aside-nav__item"><A href="#result4" data-number="4">
4. Production of diacetyl
      </A></DIV><DIV class="aside-nav__item"><A href="#result5" data-number="5">
5. <I>P. pastoris</I> is able to detect diacetyl
      </A></DIV><DIV class="aside-nav__item"><A href="#result6" data-number="6">
6. <I>P. pastoris</I> is able to produce functional AMP
      </A></DIV><DIV class="aside-nav__item"><A href="#result7" data-number="7">
7. Co-cultivation of <I>P. pastoris</I> and <I>V. harveyi</I></A></DIV><DIV class="aside-nav__item"><A href="#result8" data-number="8">
8. Permeability assay of the membranes
      </A></DIV></DIV></DIV><DIV class="right_container"><SECTION><P>The experimental results described here provide insights of the functionality of our project and clues about how our synthetic consortium will work.
</P></SECTION><SECTION><H1>1. Mimicking <I>Vibrio sp</I>. presence with an engineered <I>E. coli</I></H1><H2>Background</H2><P>
			For safety reasons, we cannot manipulate <I>V. cholerae</I>. Therefore to mimic its presence, we engineered <I>E. coli</I> to produce a quorum sensing molecule from a non-pathogenic <I>Vibrio</I> species i.e. C8-CAI-1 from <I>V. harveyi</I> (<A href="https://2017.igem.org/Team:INSA-UPS_France/Experiments/Clonings"> cloning </A>). C8-CAI-1 is an analogue of <I>V. cholerae</I> CAI-1 quorum sensing molecule and is produced by the enzyme CqsA. Besides, it allows to investigate our capacity to create synthetic communication between both engineered <I>E. coli</I> and <I>V. harveyi</I>.
		</P><H2>Materials and methods</H2><P>
Strain construction can be found here (<A href="https://2017.igem.org/Team:INSA-UPS_France/Experiments/Clonings"> clonings </A>) and protocols used can be found here <A href=" https://2017.igem.org/Team:INSA-UPS_France/Protocols#prot7"> Protein production and sampling</A>, <A href=" https://2017.igem.org/Team:INSA-UPS_France/Protocols#prot3"> NMR analysis </A>, <A href=" https://2017.igem.org/Team:INSA-UPS_France/Protocols#prot1"> Cultivation condition </A> and <A href=" https://2017.igem.org/Team:INSA-UPS_France/Protocols#prot0"> Medium composition </A> ).
</P><H2>Results and discussion</H2><P>
The figure 1 presents the results obtained with the supernatants of wild type <I>E. coli</I> (as a negative control with no C8-CAI-1 produced), wild type <I>V. harveyi</I>  (as a positive control producing C8-CAI-1), and our engineered <I>E. coli</I> strain expressing <I>cqsA</I> from <I>V. harveyi</I> (i.e. <I>VhCqsA</I>, <A href=" http://parts.igem.org/wiki/index.php/Part:BBa_K2278001 "> Part:BBa_K2278001 </A>). These supernatants were analysed by <SUP>1</SUP>H Nuclear Magnetic Resonance (NMR). Characteristic and observable proton signals of C8-CAI-1 in <SUP>1</SUP>H NMR should appears at approximatively (δ, in ppm) 4.14 as a triplet dedoubled (CH in α of both carbonyl and hydroxyl groups), 2.50-2.36 (CH2 in α of carbonyl group) and 1.92-1.84 (1H of CH2 in β of hydroxyl group) as multiplets.
Unfortunately, the NMR profiles of the three supernatants were similar and no characteristic <SUP>1</SUP>H NMR signal of C8-CAI-1 had been detected neither in wild type <I>V. harveyi</I> nor in <I>E. coli-VhCqsA</I> strains. 
So, NMR approaches failed to confirm the production of C8-CAI-1 molecule in both <I>E. coli-VhCqsA</I> and in <I>V. harveyi</I> WT strains. This does not mean the tested construction is not functional but more probably that C8-CAI-1 production was below the sensitivity limit of NMR. As a more sensitive alternative, mass spectrometry analyses were also applied on the same samples. However, no conclusive results were obtained (data not shown) and optimization of the method required extra time. We thus cannot exclude that the expression of CqsA is not functional but a more sensitive approach such as <I>in vivo</I> strategy should be tested to check the functionality of this part.
</P><FIGURE><FIGCAPTION>
        Fig. 1: <B>Detection of C8-CAI-1 by NMR analysis (500MHz, CDCl3, 298K) in culture supernatant.</B> Overlaid <SUP>1</SUP>H NMR spectra of freeze-dried supernatants from <I>E. coli</I>-pSB1C3 (negative control) in green, <I>E. coli</I>-VhCqsA (assay) in red, wild type <I>V. harveyi</I> (positive control) in blue. Supernatant have been freeze-dried and re-suspended into deuterated chloroform (CDCl3) before <SUP>1</SUP>H NMR analysis.
        </FIGCAPTION></FIGURE><H2>Perspectives</H2><P>
Culture parameters, induction by IPTG or extraction protocol could be optimized to increase C8-CAI-1 production. Alternatively, MS method could be improved to detect the C8-CAI-1.
Alternatively, we can focus on the expression of the <I>cqsA</I> gene from <I>V. cholerae</I> to create a safe <I>V. cholerae</I> effector.
</P></SECTION><SECTION><H1>2. <I>E. coli</I> producing C8-CAI-1 molecules can be sensed by <I>V. harveyi</I></H1><H2>Background</H2><P>In response to its quorum sensing molecule, i.e. C8-CAI-1, <I>V. harveyi</I> becomes bioluminescent. Here we used it as a sensor to detect the production of C8-CAI-1 by our engineered <I>E. coli-VhCqsA</I> strain (i.e., <A href=" http://parts.igem.org/wiki/index.php/Part:BBa_K2278001"> Part:BBa_K2278001 </A>) 
.</P><H2>Materials and methods</H2><P>
Protocols can be found here: <A href=" https://2017.igem.org/Team:INSA-UPS_France/Protocols#prot0"> Medium composition </A>, <A href=" https://2017.igem.org/Team:INSA-UPS_France/Protocols#prot4"> Solid Bioluminescence assay</A>.
</P><H2> Results and discussion</H2><P>
We used the JMH626 mutant strain of <I>V. harveyi</I>, a strain with deletion of all its quorum sensing production genes, including <I>cqsA</I><SUP><A href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3285556/" target="_blank">1</A></SUP>. Thus, it is unable to produce its own C8-CAI-1. However, the whole apparatus for response to quorum sensing is still present in this strain. So, the supernatant of the <I>E. coli-VhCqsA</I> strain was applied on this strain to check if it was able to trigger luminescence, as a natural property of <I>V. harveyi</I> exposed to C8-CAI-1. 
Results are presented in Figure 2. In the negative control with <I>V. harveyi ∆cqsA</I> supernatant applied to the same strain, a very low basal bioluminescent is observed (likely from promoter leaking). The same very low basal luminescence level was observed with supernatant of wild type <I>E. coli</I> strain. As expected, when the supernatant of <I>V. harveyi</I> wild type strain was applied to <I>V. harveyi ∆cqsA</I>, bioluminescence could be restored. When supernatant of <I>E. coli-VhCqsA</I> strain was applied to the <I>V. harveyi ∆cqsA</I> strain, bioluminescence was observed with a comparable level to the one observed with wild type <I>V. harveyi</I>. 
Together, these data demonstrate that C8-CAI-1 is efficiently produced by <I>E. coli-VhCqsA</I> and thus validate our <A href=" http://parts.igem.org/wiki/index.php/Part:BBa_K2278001 "> BBa_K2278001 </A>part. More importantly, this result also demonstrates that we successfully created synthetic communication between engineered <I>E. coli</I> and <I>V. harveyi</I> strains.
</P><FIGURE><FIGCAPTION>
        Fig. 2: <B>Solid bioluminescence assay</B>. Two strain of <I>V. harveyi</I> were plated (WT and Δ<I>cqsA</I>) with the addition of supernatant from <I>V. harveyi</I> WT, <I>V. harveyi </I>Δ<I>cqsA</I> and supernatant from <I>E. coli</I> with an empty pSB1C3 vector (<I>E. coli</I>) or <I>E. coli</I> with pSB1C3-<I>cqsA</I> (<I>E. coli-cqsA</I>). This picture is a representative picture of the 4 times repeated experiment.
        </FIGCAPTION></FIGURE><H2>Perspectives</H2><P>
Next step will be to prove the communication between <I>E coli</I> producing <I>V. cholerae</I> CAI-1 (<A href=" http://parts.igem.org/wiki/index.php/Part:BBa_K2278002"> Part:BBa_K2278002 </A>) and <I>V. harveyi</I> expressing the gene encoding the modified sensor CqsS* that could sense both C8-CAI-1 and CAI-1 (see next Results module). 
</P></SECTION><SECTION><H1>3. Modification of <I>V. harveyi</I> to detect both C8-CAI-1 and CAI-1</H1><H2>Background</H2><P><I>V. harveyi</I> has many assets to be a good sensor for <I>V. cholerae</I>: it possesses its own pathway of detection for C8-CAI-1, very similar to the <I>V. cholerae</I> pathway to detect CAI-1. It has been shown that only a single point mutation on its receptor CqsS allows <I>V. harveyi</I> detecting <I>V. cholerae</I> CAI-1 molecule<SUP><A href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3285556/" target="_blank">1</A></SUP>. Because genetic in <I>V. harveyi</I> is limited (i.e. classical transformation does not work), we first had to set up a triparental conjugation protocol.

		</P><H2>Materials and methods</H2><P>

Strain construction can be found <A href="https://2017.igem.org/Team:INSA-UPS_France/Experiments/Clonings">here </A>and protocols used can be found there: <A href=" https://2017.igem.org/Team:INSA-UPS_France/Protocols#prot5"> Triparental conjugation </A>, <A href=" https://2017.igem.org/Team:INSA-UPS_France/Protocols#prot6"> Fluorescence microscopy </A>, <A href=" https://2017.igem.org/Team:INSA-UPS_France/Protocols#prot1"> Cultivation condition </A>, <A href=" https://2017.igem.org/Team:INSA-UPS_France/Protocols#prot0"> Medium composition</A>. 
</P><H2>Results and discussion</H2><P>
A protocol of triparental conjugation to transfer plasmids to <I>V. harveyi</I> was set up from an original conjugation method provided by M. Arlat (LIPM, Toulouse) usually used with <I>Xanthomonas campestris</I>. It was adapted to our <I>V harveyi</I> strain. It requires an <I>E. coli</I> donor with the conjugative plasmid pBBR1MCS-4 + insert (BBa_J04450), an <I>E. coli</I> helper with the helper plasmid pRK2073, and a receiver <I>V. harveyi</I>.
Transformants were analyzed by fluorescence microscopy and results are presented in Figure 3. The pictures show that the strain of <I>V. harveyi</I> conjugated with the plasmid pBBR1MCS-4 containing the BBa_J04450 construction is able to express RFP using a promoter and terminator from <I>E. coli</I>. 
This raised several important conclusions. (i) It clearly demonstrated that <I>V. harveyi</I> can be engineered. This is the first use of <I>V. harveyi</I> as a genetic chassis in the IGEM competition. (ii) It proved that iGEM registry elements could be used in <I>V. harveyi</I> (promoter and terminator for instance). (iii) To the best of our knowledge, this is the first reported use of RFP in <I>V. harveyi</I>.
</P><FIGURE><FIGCAPTION>
        Fig. 3: <B>V. harveyi can be modified to produce RFP</B>. <I>V. harveyi</I> JMH626 (panels C &amp; D) and conjugated with pBBR1MCS-4-RFP (panels A &amp; B) in optic microscopy (panel A &amp; C) or fluorescence microscopy (panels B &amp; D). Emission: 530-560nm/Excitation 572-647nm.
On the panel (A), which shows control <I>V. harveyi</I> JMH626, no red bioluminescence is observed and the panel (C) prove the presence of cells by optic microscopy. On (B), red bioluminescence is clearly visible. The position of the red stains are correlated to with the location of the <I>V. harveyi</I> pBBR1MCS-5-RFP conjugated cells observed by optic microscopy (D). 
        </FIGCAPTION></FIGURE><H2>Perspectives</H2><P>
Next step will be to integrate the plasmid containing the engineered CqsS* receptor into <I>V. harveyi</I> to create a sensor able to detect both <I>V. cholerae</I> CAI-1 and <I>V. harveyi</I> C8-CAI-1. Likewise, the Als construction tested in <I>E. coli</I> could be now tested in <I>V. harveyi</I> (see next module).
 </P></SECTION><SECTION><H1>4. Production of diacetyl to establish communication between prokaryotic and eukaryotic cells</H1><H2>Background</H2><P>
			The quorum sensing signal detected by <I>V. harveyi</I> has to be transmitted to the yeast <I>P. pastoris</I>. For that, we chose to engineer <I>V. harveyi</I> so that it will conditionally produce <A href="https://2017.igem.org/Team:INSA-UPS_France/Design"> diacetyl </A> as the communication molecule between the bacterium and the yeast. Here, we tested the functionality of our construction containing the <I>als</I> gene, encoding the acetolactate synthase that enable diacetyl production ( <A href="http://parts.igem.org/wiki/index.php/Part:BBa_K2278011"> Part:BBa_K2278011 </A>).
		</P><H2>Materials and methods</H2><P>
Strain construction can be found<A href="https://2017.igem.org/Team:INSA-UPS_France/Experiments/Clonings"> here</A> and protocols used can be found there: <A href=" https://2017.igem.org/Team:INSA-UPS_France/Protocols#prot1"> Cultivation condition</A>, <A href=" https://2017.igem.org/Team:INSA-UPS_France/Protocols#prot3"> NMR analysis </A> and <A href=" https://2017.igem.org/Team:INSA-UPS_France/Protocols#prot7"> Protein production and sampling</A>.
</P><H2>Results and discussion</H2><P>
To check the functionality of the plasmid expressing <I>als</I>, we first used <I>E. coli</I>. Figure 4 presents the results obtained for the <SUP>1</SUP>H NMR analysis of the supernatant of <I>E. coli</I> strains with empty pSB1C3 or pSB1C3-<I>als</I>. Diacetyl specific shift was confirmed by spiking experiment. A small peak of diacetyl could be detected in the strain expressing <I>als</I> but not in the control. However, the amount seems very low. This could be related to two punctual mutations observed in the <I>als</I> sequence of this clone. However, only a single sequencing run was performed for this part, which is insufficient to be affirmative about the reality of these mutations.
</P><FIGURE><FIGCAPTION>
        Figure 4: <B>Validation of diacetyl production by NMR analysis</B> (800MHz, 10% D2O, 280K). Overlaid <SUP>1</SUP>H NMR spectra of <I>E. coli</I> supernatants from <I>E. coli</I> pSB1C3 (negative control) in green; <I>E. coli</I> pSB1C3 + 0.50 mM diacetyl standard solution (positive control underlying the signature of diacetyl as a chemical shift of δ = 2.34 ppm) in red; two biological replicates of E. coli expressing <I>als</I> in blue colors.
        </FIGCAPTION></FIGURE><H2>Perspectives</H2><P>
Next step will be to verify the sequence of als and to express the plasmid encoding for the right sequence of Als in <I>V. harveyi</I>. Diacetyl production will be then checked again by NMR.
 </P></SECTION><SECTION><H1>5. Diacetyl detection by <I>Pichia pastoris</I>: </H1><H2>Background</H2><P>
			To allow a communication between <I>V. harveyi</I> and <I>P. pastoris</I> mediated by diacetyl, the Odr-10 receptor <A href="https://2017.igem.org/Team:INSA-UPS_France/Design"> (design) </A> had to be expressed in <I>P. pastoris</I>. To check if Odr-10 was functional <I>in vivo</I>, we used a pFUS1-RFP reporter system. Indeed when diacetyl binds to Odr-10 a cascade of activation of Ste proteins (endogenous to <I>P. pastoris</I>) will lead to the binding of Ste12 on pFUS1 promoter, and the expression of RFP should be activated.
		</P><H2>Materials and methods</H2><P>
Strain construction can be found <A href="https://2017.igem.org/Team:INSA-UPS_France/Experiments/Clonings">here</A> and protocols used can be found there: <A href=" https://2017.igem.org/Team:INSA-UPS_France/Protocols#prot1"> Cultivation condition </A>, <A href=" https://2017.igem.org/Team:INSA-UPS_France/Protocols#prot0"> Medium composition </A> and <A href=" https://2017.igem.org/Team:INSA-UPS_France/Protocols#prot8"> Plate reader</A>.
 </P><H2>Results and discussion</H2><P>
We tested the functionality of the complete system Odr-10/pFUS1-RFP by growing the cells on a medium specifically designed to induce the activation of Ste proteins (Johnston et al., 1977). Diacetyl was added at various concentration. Absorbance and fluorescence production by <I>P. pastoris</I> strain having integrated the empty plasmid or the plasmid containing Odr-10/pFUS1-RFP system was followed over the time on a microplate reader. A positive control was obtained by growing a <I>Pichia</I> strain with a constitutive high expression of RFP (pGAP promoter fused to RFP gene). Results are presented in Figure 5. 

In the conditions without diacetyl or with 500 µM of diacetyl, no difference was observed between the control and the strain expressing Odr-10/pFUS1-RFP. However, when diacetyl is present at higher concentration (i.e. 1000 µM), higher expression is 
 observed than in the negative control. 
These data demonstrate the functionality of the complete detection pathway. Incidently, they also show that pGAP (our positive control promoter) is functional in <I>P. pastoris</I>, which increase the range of application of this promoter biobrick (<A href="http://parts.igem.org/wiki/index.php/Part:BBa_K431009"> Part:BBa_K431009 </A>).  This was demonstrated again by RT-qPCR in the next Results module. Finally, this result suggests that our engineered <I>P. pastoris</I> is ready to communicate with <I>Vibrio</I> species if this former one produce diacetyl.
 </P><FIGURE><FIGCAPTION>
        Figure 5: <B>Measurement of pFUS1 activity</B>. <I>P. pastoris</I> was grown in CMM media supplemented with glutamine. Negative control (T-) is performed using<I>P. pastoris</I> with a genomic integration of pPICZα, <I>P. pastoris</I> with a pPICZα-ODR10/pFUS1-RFP genomic integration were grown with 500 µM or 1000 µM of diacetyl. <I>P. pastoris</I> with genomic integration of pPICZα-pGAP-RFP (T+) is the positive control. Results are presented as the ratio of RFP fluorescence at 600(+/- 10) nm normalized by absorbance at 595 nm (measure of cell density). Results are from duplicated experiments. 
        </FIGCAPTION></FIGURE><H2>Perspectives</H2><P>
Next step will be to grow together our <I>V. harveyi</I> expressing Als and our <I>P. pastoris</I> expressing Odr-10/pFUS-RFP to demonstrate synthetic communication between them. </P></SECTION><SECTION><H1>6. <I>P. pastoris</I> is able to produce functional antimicrobial peptides</H1><H2>Background</H2><P>
			The final goal of our synthetic consortia was to kill <I>V. cholerae</I> with newly described antimicrobial peptides from crocodile. Here, we tested the production by <I>P. pastoris</I> of three inovative antimicrobial peptides (AMP), i.e. D-NY15, cOT2 and Leucrocin I (respectively <A href="http://parts.igem.org/wiki/index.php/Part:BBa_K2278021"> Part:BBa_K2278021</A>,<A href="http://parts.igem.org/wiki/index.php/Part:BBa_K2278023"> Part:BBa_K2278023 </A> and <A href="http://parts.igem.org/wiki/index.php/Part:BBa_K2278022"> Part:BBa_K2278022 </A> )
		</P><H2>Materials and methods</H2><P>
Strain construction can be found <A href="https://2017.igem.org/Team:INSA-UPS_France/Experiments/Clonings">here</A> and protocols used can be found there:  <A href=" https://2017.igem.org/Team:INSA-UPS_France/Protocols#prot1"> Cultivation condition </A>, <A href=" https://2017.igem.org/Team:INSA-UPS_France/Protocols#prot0"> Medium composition </A>, <A href=" https://2017.igem.org/Team:INSA-UPS_France/Protocols#prot8"> Plate reader</A>, <A href=" https://2017.igem.org/Team:INSA-UPS_France/Protocols#prot9"> Semi quantitative RT-PCR</A> and <A href=" https://2017.igem.org/Team:INSA-UPS_France/Protocols#prot10"> On plate toxicity assay</A>.
 </P><H2>Results and discussion</H2><P><UL><LI><B>Expression assay</B></LI></UL></P><P>
Because our AMPs have never been produced by <I>P. pastoris</I> and their toxicity on this yeast was not assessed, we first validated the expression of the AMPs by RT-qPCR. Figure 6 showed the results obtained for D-NY15 expression in <I>P. pastoris</I>. The amount of fluorescence provided by the RT-qPCR with the D-NY15 amorces (curves A1, A2 and A3) is raising after few cycles (8.32 +/- 0.03) whereas the negative control (pPICZα only, curves C1, C2 and C3) starts to be amplified at over 29 cycles (i.e. non-specific amplification). This means that the D-NY15 encoding gene is expressed in <I>P. pastoris</I> and non-lethal.
</P><FIGURE><FIGCAPTION>
       Fig 6. <B>RT-qPCR of D-NY15</B>. RNA were from <I>P. pastoris</I> strains having integrated pPICZ or pPICZα-D-NY15. Total RNAs were extracted from transformants and reverse transcriptions were performed using Superscript II reverse transcriptase (Invitrogen). Resulting D-NY15 cDNA were then amplified by quantitative PCR. The curves correspond to <I>P. pastoris</I> with integrated pPICZα-D-NY15 (A1, A2, A3), water control (B1 &amp; B2), and <I>P. pastoris</I> with integrated pPICZα (C1, C2, C3).
        </FIGCAPTION></FIGURE><P><UL><LI><B>Toxicity assay</B></LI></UL></P><P>
To test the toxicity of the three AMPs, disks soaked with the supernatant of <I>P. Pastoris</I> with genomic integration of pPICZα, pPICZα-D-NY15, pPICZα-cOT2 or pPICZα-Leucrocin I were put on freshly inoculated solid culture of <I>V. harveyi</I>. Results are presented on Figure 7. Inhibition haloes were observed around both the positive control and the patch containing the supernatant from a D-NY15 expressing strain. No growth inhibition were observed around the disks containing the supernatant from COT2 and Leucrocin I expressing strains. These data nicely demonstrated the capacity of the engineered crocodile peptide D-NY15 to inhibit <I>V. harveyi</I> growth but also the capacity of <I>P. pastoris</I> to produce antimicrobial peptides. This is the first demonstration of AMP production by <I>P. pastoris</I> by an IGEM Team and the first use of sequence coming from crocodile in iGEM!
</P><FIGURE><FIGCAPTION>
       Figure 7 : <B>AMP halo assays</B>. Positive control was performed with chloramphenicol (25 ng/µL), the negative control was performed with the empty plasmid integrated in <I>P. pastoris</I>, the assays were performed with the supernatants from <I>P. pastoris</I> strains expressing either pPICZα-D-NY15 (D-NY15), pPICZα-Leucrocin I (Leucrocin I) or pPICZα-cOT2 (cOT2).
        </FIGCAPTION></FIGURE><H2>Perspectives</H2><P>
Next step will be to prove the inhibitory activity of AMPs in liquid medium containing both modified <I>P. pastoris</I> with <I>V. harveyi</I>. For cOT2 or Leucrocin I, which cytoxicity was not demonstrated yet, a higher concentration of supernatant could be tested.
 </P></SECTION><SECTION><H1>7. Co-cultivation of <I>P. pastoris</I> and <I>V. harveyi</I> is possible</H1><H2>Background</H2><P>
			Ultimately our synthetic consortia will be freeze-dried and placed into a plastic bag containing dried cultivation medium. As each microorganism of the consortia has its own cultivation requirements, a difficulty was to find a common media on which <I>V. harveyi</I> and <I>P. pastoris</I> can grow together.
		</P><H2>Materials and methods</H2><P>
Protocols used can be found here: <A href=" https://2017.igem.org/Team:INSA-UPS_France/Protocols#prot1">Cultivation condition </A>, <A href=" https://2017.igem.org/Team:INSA-UPS_France/Protocols#prot0"> Medium composition</A>.
 </P><H2>Results and discussion</H2><P><UL><LI><B>Selection of the cultivation medium</B></LI></UL></P><P><I>P. pastoris</I> and <I>V. harveyi</I> were grown on six media corresponding to variants of their standard media (LB, LB acetate 2%, LM, LM acetate 2%, YPD, and YPA 1%, see <A href=" https://2017.igem.org/Team:INSA-UPS_France/Protocols#prot0"> Medium composition</A> for details). 
</P><FIGURE><FIGCAPTION>
        Figure 8: <B>Table of growth rates (h<INF>-1</INF>) of V. harveyi and P. pastoris</B> on six media. Media were inoculated at an initial OD600nm = 0.05. Optical density was measured very hours. Results are issued from a single biological replicate.
        </FIGCAPTION></FIGURE><P>
As expected, <I>V. harveyi</I> and <I>P. pastoris</I> were growing well on their respective usual media (LM and YPD ; figure 8), but not as performant on each other medium. Acetate inhibited growth of both microorganism in YPA &amp; LMA but it was better tolerated in LBA. Eventually, LM medium appears as good compromise to ensure a satisfying growth of both strains. This is an interesting result for this project since LM medium is cheap and easy to produce. 

All these data have been used to fit our <A href=" https://2017.igem.org/Team:INSA-UPS_France/Model/Simulation"> model page</A> in order to help in the design of the <A href=" https://2017.igem.org/Team:INSA-UPS_France/Entrepreneurship/Device">device</A>.
</P><H2>Perspectives</H2><P>
Next step will be to challenge the capacity of our strains to recover from freeze-dry since they will be lyophylized in our <A href=" https://2017.igem.org/Team:INSA-UPS_France/Entrepreneurship/Device">device</A>. Preliminary tries suggested they survived but more accurate measurements will have to be performed.
 </P></SECTION><SECTION><H1>8. Permeability assay of the membranes used for the devices:</H1><H2>Background</H2><P>
			We selected two types of membrane materials, i. e. TPX® X44B 50 PVDF 0.1 µm for making the sachet containing our synthetic biology system composed of <I>V. harveyi</I> and <I>P. pastoris</I> with nutrient medium. Here we tested the permeability of both to check whether this membrane could contain our synthetic organisms while allowing the antimicrobial peptides and quorum sensing molecules go through from the device compartment to the treated water. 
		</P><H2>Results and discussion</H2><P>
We first used the TPX® X44B 50µm material used by  <A href=" https://2012.igem.org/Team:Groningen"> iGEM Groeningen 2012</A> and <A href=" https://2015.igem.org/Team:Toulouse"> iGEM Toulouse 2015</A>. To assess the porosity of the material, we performed a liquid/liquid transfer assay with bengale pink dye. Bengale pink dye was chosen because of its high molecular mass for a dye (977 g/mol) to get closer to the size of our AMPs. The test was done with a Hele-Shaw measurement cell (two parallel planes spaced by 2 mm) with a camera (EDMUND OPTIC) to monitor the diffusion through the membrane (figure 9). The cell was filled with water and bengale pink was introduced from the top in a cone of membrane (or absorbent paper as a control), in contact with the cell water.
</P><FIGURE><FIGCAPTION>
       Figure 9: <B>Hele-Shaw measurement cell during the experiment</B>.
 </FIGCAPTION></FIGURE><P>
We did not observe diffusion of the dye through the TPX membrane in this experiment (Data not shown). Since we were concerned with the hydrophobicity of the material, we also tried to check if water can go through by putting the material with a drop of water on top of absorbent paper. No moisture on the absorbent paper was observed during the course of the experiment. We conclude that the chemical characteristics of the TPX material does not suit to our project.

Next we tested the material proposed by our partner Sunwaterlife, i.e. a membrane used in their water cleaning system (PVDF 0.1 µm from Orelis Environnement SAS, France). We tested this material with the same Hele-Shaw measurement cell. As shown on figure 10, the dye is able to go through the material. Diffusion was observed in less than 1 minute. This means this material is entirely suitable for our project since the 0.1 µm porosity prevents bacterial diffusion but allows peptides diffusion.
</P><FIGURE style="max-width:500px;"><FIGCAPTION>
       Figure 10: <B>Bengale pink dye diffusion</B> through the PVDF membrane 0.1 µm five minutes after the experiment start.
        </FIGCAPTION></FIGURE><H2>Perspectives:</H2><P>
Next step will be to test this membrane with our synthetic consortia to check whether bacterial diffusion is prevented while peptides diffusion allowed.
</P></SECTION><SECTION><H1>References:</H1><OL><LI>Ng W-L, Perez LJ, Wei Y, Kraml C, Semmelhack MF &amp; Bassler BL (2011) Signal production and detection specificity in Vibrio CqsA/CqsS quorum-sensing systems: Vibrio quorum-sensing systems. Molecular Microbiology 79 1407–1417
<A href="https://www.ncbi.nlm.nih.gov/pubmed/21219472">https://www.ncbi.nlm.nih.gov/pubmed/21219472</A></LI><LI>Johnston G. C., Singer R. A. &amp; McFarlane S. Growth and cell division during nitrogen starvation of the yeast Saccharomyces cerevisiae. J. Bacteriol. 132, 723–730 (1977). 
<A href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC221916/">https://www.ncbi.nlm.nih.gov/pmc/articles/PMC221916/</A></LI></OL></SECTION><SECTION class="links_end"><TABLE><TBODY><TR><TH colspan="6">
            Realisations pages
          </TH></TR><TR><TD><A href="https://2017.igem.org/Team:INSA-UPS_France/Experiments">Overview</A></TD><TD><A href="https://2017.igem.org/Team:INSA-UPS_France/Notebook">Notebook</A></TD><TD><A href="https://2017.igem.org/Team:INSA-UPS_France/Experiments/Clonings">Clonings</A></TD><TD><I>Results</I></TD><TD><A href="https://2017.igem.org/Team:INSA-UPS_France/Contribution">Contribution</A></TD><TD><A href="https://2017.igem.org/Team:INSA-UPS_France/Protocols">Protocols</A></TD><TD><A href="https://2017.igem.org/Team:INSA-UPS_France/Safety">Safety</A></TD></TR></TBODY></TABLE></SECTION></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>