<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_INSA-UPS_France_Model_ODE skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:INSA-UPS France/Model/ODE</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container" style="position:relative; "><DIV class="site-header__title">
    Croc'n Cholera <SPAN style="font-size:12pt;">iGEM UPS-INSA Toulouse 2017</SPAN></DIV><NAV class="site-menu" style="text-align: center;"><DIV class="site-menu__item"><DIV class="cat_name">Team </DIV><UL class="subnav" style="width:150%;"><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Team">Members</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Supports">Supports</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Attributions">Attributions</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Collaborations">Collaborations</A></LI></UL></DIV><DIV class="site-menu__item"><DIV class="cat_name">Strategy </DIV><UL class="subnav"><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Description">Description</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Design">Design</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Parts">Parts</A></LI></UL></DIV><DIV class="site-menu__item"><DIV class="cat_name">Model </DIV><UL class="subnav"><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Model">Overview</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Model/Simulation">Simulation</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Model/Analysis">Analysis</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Model/Interface">Interface</A></LI></UL></DIV><DIV class="site-menu__item"><DIV class="cat_name">Realisations </DIV><UL class="subnav"><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Experiments">Overview</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Notebook">Notebook</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Experiments/Clonings">Clonings</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Results">Results</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Contribution">Contribution</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Protocols">Protocols</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Safety">Safety</A></LI></UL></DIV><DIV class="site-menu__item"><DIV class="cat_name">Human Practices</DIV><UL class="subnav"><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Human_Practices">Overview</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/HP/Silver">Silver</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/HP/Gold_Integrated">Gold</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Engagement">Public Engagement</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Microbioworld">Microbioworld</A></LI></UL></DIV><DIV class="site-menu__item"><DIV class="cat_name">Application </DIV><UL class="subnav"><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Applied_Design">Applied Design</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Entrepreneurship/Testimonies">Testimonies</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Entrepreneurship">Entrepreneurship</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Entrepreneurship/Device">Device Conception</A></LI></UL></DIV></NAV></DIV><DIV class="main_content"><DIV class="middle_container"><DIV class="section_container"><DIV class="main_title"><DIV><P>System of ODEs</P></DIV></DIV><SECTION><H1>Equations</H1><P>The dynamics of our microbial consortium was summed up in thirteen differential equations. Every biological or physical process was described mathematically, and was gathered to constitute our system of ODEs. For fearless people, our complete mathematical model and demonstration can be found <A href="https://2017.igem.org/Team:INSA-UPS_France/Model/Complete_Model">there</A>!</P><P>We need to characterize the growth and death of the three microorganisms: <I><B>Vibrio cholerae</B></I> (<I>Vc</I>) in water (W), <I><B>Vibrio harveyi</B></I> (<I>Vh</I>) and <I><B>Pichia pastoris</B></I> (<I>Pp</I>) in the device (D).
\begin{equation}
    \frac{d[\textit{Vc}]_W}{dt} = V_{growth,Vc} - V_{death,Vc}
\end{equation}
\begin{equation}
    \frac{d[\textit{Vh}]_D}{dt} = V_{growth,Vh} - V_{death,Vh}
\end{equation}
\begin{equation}
    \frac{d[\textit{Pp}]_D}{dt} = V_{growth,Pp} - V_{death,Pp}
\end{equation}
</P><P>
Microorganisms death is impacted by <B>antimicrobial peptides</B> production(AMP<SUB>peptide,Pp</SUB>), produced by translation of antimicrobial peptides mRNA (AMP<SUB>RNA</SUB>). Peptides and mRNA are also degraded.
\begin{equation}
    \frac{d[AMP_{RNA}]_{Pp}}{dt}=V_{transcription,AMP} - V_{degradation,AMP RNA}
\end{equation}

\begin{equation}
    \frac{d[AMP_{peptide}]_{D}}{dt}=V_{translation,AMP} - V_{degradation,AMP} + \frac{V_{diff,AMP,W \to D}}{\mathcal{V}_D}
\end{equation}
</P><P>These peptides are transfered from the <B>device</B> (D) to <B>water</B> (W).

\begin{equation}
    \frac{d[AMP]_W}{dt} = -V_{diff,AMP,W\to D}
\end{equation}
</P><P>To produce antimicrobial peptides, an activation by <B>diacetyl</B> (dac) is needed. Diacetyl can freely diffuse from the device (D) to water (W).
\begin{equation}
    \frac{d[dac]_D}{dt}=V_{prod,dac}+\frac{V_{diff,dac,W \to D}}{\mathcal{V}_D}
\end{equation}

\begin{equation}
    \frac{d[dac]_W}{dt}=- V_{diff,dac,W \to D}
\end{equation}
</P><P>Diacetyl is produced by the enzyme <B>acetolactate synthase</B> (ALS<SUB>enzyme</SUB>). Als gene is first transcribed into ALS<SUB>RNA</SUB>, which is then translated into the protein. Both the enzyme and its mRNA can be degraded.
\begin{equation}
    \frac{d[ALS_{RNA}]_{Vh}}{dt} = V_{transcription,ALS} - V_{degradation,ALS RNA}
\end{equation}

\begin{equation}
    \frac{d[ALS_{enzyme}]_{Vh}}{dt} = V_{translation,ALS} - V_{degradation,ALSenzyme}
\end{equation}
</P><P>ALS production has to be activated by the quorum sensing molecule <B>CAI-1</B>, initially in water (W), after diffusing into the device (D). 

\begin{equation}
    \frac{d[CAI\text{-}1]_D}{dt} = \frac{V_{diff,CAI\text{-}1,W\to D}}{\mathcal{V}_D}
\end{equation}

\begin{equation}
    \frac{d[CAI\text{-}1]_W}{dt} = -V_{diff,CAI\text{-}1,W\to D}
\end{equation}

</P></SECTION><SECTION><H1>Data</H1><P>Our model is mostly set by data from publications, because the majority of the required data necessitates complex experiments we could not perform during our project.</P><TABLE style="font-size:8pt;"><TBODY><TR><TH>Name</TH><TH>Notation</TH><TH>Unit</TH><TH>Value</TH><TH>Reference</TH></TR><TR><TD><I>Vibrio cholerae</I> maximum growth rate</TD><TD>μ<SUB>MAX,<I>Vc</I></SUB></TD><TD>s<SUP>-1</SUP></TD><TD>3.10<SUP>-4</SUP></TD><TD>BioNumbers 112369 (1)</TD></TR><TR><TD>Leucrocine I MIC for <I>V. cholerae</I></TD><TD>MIC<SUB>Leucro,<I>Vc</I></SUB></TD><TD>mol/L</TD><TD>6.4.10<SUP>-5</SUP></TD><TD>Pata <I>et al.</I>, 2011 (2)</TD></TR><TR><TD>Leucrocine I MIC for <I>V. harveyi</I></TD><TD>MIC<SUB>Leucro,<I>Vh</I></SUB></TD><TD>mol/L</TD><TD>6.4.10<SUP>-5</SUP></TD><TD>Pata <I>et al.</I>, 2011 (2) - Extrapolation from <I>V. cholerae</I> result</TD></TR><TR><TD>Leucrocine I MIC for <I>P. pastoris</I></TD><TD>MIC<SUB>Leucro,<I>Pp</I></SUB></TD><TD>mol/L</TD><TD>∞</TD><TD>Assuming no effects on <I>Pichia pastoris</I></TD></TR><TR><TD>Leucrocine I IC50 for <I>V. cholerae</I></TD><TD>MIC<SUB>Leucro,<I>Vc</I></SUB></TD><TD>mol/L</TD><TD>1.92.10<SUP>-4</SUP></TD><TD>Considering IC50 = 3.MIC - Extrapolation of data from standard antibiotics (Farrag <I>et al.</I>, 2015)</TD></TR><TR><TD>Leucrocine I IC50 for <I>V. harveyi</I></TD><TD>IC50<SUB>Leucro,<I>Vh</I></SUB></TD><TD>mol/L</TD><TD>1.92.10<SUP>-4</SUP></TD><TD>Considering IC50 = 3.MIC - Extrapolation of data from standard antibiotics (Farrag <I>et al.</I>, 2015 (3))</TD></TR><TR><TD>Leucrocine I IC50 for <I>P. pastoris</I></TD><TD>IC50<SUB>Leucro,<I>Pp</I></SUB></TD><TD>mol/L</TD><TD>∞</TD><TD>Assuming no effects on <I>Pichia pastoris</I></TD></TR><TR><TD>cOT2 MIC for <I>V. cholerae</I></TD><TD>MIC<SUB>cOT2,<I>Vc</I></SUB></TD><TD>mol/L</TD><TD>8.1.10<SUP>-6</SUP></TD><TD>Prajanban <I>et al.</I>, 2017 (4)</TD></TR><TR><TD>cOT2 MIC for <I>V. harveyi</I></TD><TD>MIC<SUB>cOT2,<I>Vh</I></SUB></TD><TD>mol/L</TD><TD>8.1.10<SUP>-6</SUP></TD><TD>Prajanban <I>et al.</I>, 2017 (4) - Extrapolation from <I>V. cholerae</I> result</TD></TR><TR><TD>cOT2 MIC for <I>P. pastoris</I></TD><TD>MIC<SUB>cOT2,<I>Pp</I></SUB></TD><TD>mol/L</TD><TD>∞</TD><TD>Assuming no effects on <I>Pichia pastoris</I></TD></TR><TR><TD>cOT2 IC50 for <I>V. cholerae</I></TD><TD>IC50<SUB>cOT2,<I>Vc</I></SUB></TD><TD>mol/L</TD><TD>2.43.10<SUP>-5</SUP></TD><TD>Considering IC50 = 3.MIC - Extrapolation of data from standard antibiotics (Farrag <I>et al.</I>, 2015)</TD></TR><TR><TD>cOT2 IC50 for <I>V. harveyi</I></TD><TD>IC50<SUB>cOT2,<I>Vh</I></SUB></TD><TD>mol/L</TD><TD>2.43.10<SUP>-5</SUP></TD><TD>Considering IC50 = 3.MIC - Extrapolation of data from standard antibiotics (Farrag <I>et al.</I>, 2015)</TD></TR><TR><TD>cOT2 IC50 for <I>P. pastoris</I></TD><TD>IC50<SUB>cOT2,<I>Pp</I></SUB></TD><TD>mol/L</TD><TD>∞</TD><TD>Assuming no effects on <I>Pichia pastoris</I></TD></TR><TR><TD>D-NY15 MIC for <I>V. cholerae</I></TD><TD>MIC<SUB>D-NY15,<I>Vc</I></SUB></TD><TD>mol/L</TD><TD>1.54.10<SUP>-5</SUP></TD><TD>Yaraksa <I>et al.,</I> 2014 (5)</TD></TR><TR><TD>D-NY15 MIC for <I>V. harveyi</I></TD><TD>MIC<SUB>D-NY15,<I>Vh</I></SUB></TD><TD>mol/L</TD><TD>1.54.10<SUP>-5</SUP></TD><TD>Yaraksa <I>et al.,</I> 2014 (5) - Extrapolation from <I>V. cholerae</I> result</TD></TR><TR><TD>D-NY15 MIC for <I>P. pastoris</I></TD><TD>MIC<SUB>D-NY15,<I>Pp</I></SUB></TD><TD>mol/L</TD><TD>∞</TD><TD>Assuming no effects on <I>Pichia pastoris</I></TD></TR><TR><TD>D-NY15 IC50 for <I>V. cholerae</I></TD><TD>IC50<SUB>D-NY15,<I>Vc</I></SUB></TD><TD>mol/L</TD><TD>4.62.10<SUP>-5</SUP></TD><TD>Considering IC50 = 3.MIC - Extrapolation of data from standard antibiotics (Farrag <I>et al.</I>, 2015)</TD></TR><TR><TD>D-NY15 IC50 for <I>V. harveyi</I></TD><TD>IC50<SUB>D-NY15,<I>Vh</I></SUB></TD><TD>mol/L</TD><TD>4.62.10<SUP>-5</SUP></TD><TD>Considering IC50 = 3.MIC - Extrapolation of data from standard antibiotics (Farrag <I>et al.</I>, 2015)</TD></TR><TR><TD>D-NY15 IC50 for <I>P. pastoris</I></TD><TD>IC50<SUB>D-NY15,<I>Pp</I></SUB></TD><TD>mol/L</TD><TD>∞</TD><TD>Assuming no effects on <I>Pichia pastoris</I></TD></TR><TR><TD><I>V. cholerae</I> death rate with AMP</TD><TD>k<SUB>kill,<I>Vc</I></SUB></TD><TD>s<SUP>-1</SUP></TD><TD>3.10<SUP>-3</SUP></TD><TD>Yaraksa <I>et al.</I>, 2014 (5)</TD></TR><TR><TD><I>V. harveyi</I> death rate with AMP</TD><TD>k<SUB>kill,<I>Vh</I></SUB></TD><TD>s<SUP>-1</SUP></TD><TD>3.10<SUP>-3</SUP></TD><TD>Extrapolation from Yaraksa <I>et al.</I>, 2014 (5)</TD></TR><TR><TD><I>P. pastoris</I> death rate with AMP</TD><TD>k<SUB>kill,<I>Pp</I></SUB></TD><TD>s<SUP>-1</SUP></TD><TD>0</TD><TD>Assuming no effects</TD></TR><TR><TD>Transfer coefficient through the membrane</TD><TD>K</TD><TD>s<SUP>-1</SUP></TD><TD>1</TD><TD>Arbitrary value</TD></TR><TR><TD>Number of <I>als</I> gene per cell</TD><TD>als<SUB>DNA,0</SUB></TD><TD>Nb/cell</TD><TD>15</TD><TD>Considering a low copy plasmid (6)</TD></TR><TR><TD>Number of <I>AMP</I> gene per cell</TD><TD>AMP<SUB>DNA,0</SUB></TD><TD>Nb/cell</TD><TD>1</TD><TD>Protocol: genomic integration</TD></TR><TR><TD><I>V. harveyi</I> transcription rate</TD><TD>k<SUB>transcript,<I>Vh</I></SUB></TD><TD>nt/s</TD><TD>30</TD><TD>Molecular Biology course, Transcription - Faculté des Sciences - Rabat (7)</TD></TR><TR><TD><I>Vibrio harveyi</I> transcription rate</TD><TD>k<SUB>translation,<I>Vh</I></SUB></TD><TD>nt/s</TD><TD>15</TD><TD>Molecular Biology course, Translation - Faculté des Sciences - Rabat (8)</TD></TR><TR><TD><I>als</I> gene promoter influence</TD><TD>k<SUB>P,<I>als</I></SUB></TD><TD>/</TD><TD>1</TD><TD>Inductible promoter</TD></TR><TR><TD><I>AMP</I> gene promoter influence</TD><TD>k<SUB>P,<I>AMP</I></SUB></TD><TD>/</TD><TD>1</TD><TD>Inductible promoter</TD></TR><TR><TD>mRNA degradation constant</TD><TD>K<SUB>deg,mRNA</SUB></TD><TD>s<SUP>-1</SUP></TD><TD>5.10<SUP>-3</SUP></TD><TD>Esquerré 2015 (9)</TD></TR><TR><TD><I>als</I> gene length</TD><TD>DNA length</TD><TD>nucleotides</TD><TD>1662</TD><TD>UniProtKB - Q7DAV2 (10)</TD></TR><TR><TD><I>als</I> mRNA length</TD><TD>RNA length</TD><TD>nucleotides</TD><TD>1730</TD><TD>Parts design</TD></TR><TR><TD>Number of CqsS* receptor per cell</TD><TD>CqsS*/cell</TD><TD>Nb/cell</TD><TD>10<SUP>15</SUP></TD><TD>Arbitrary value</TD></TR><TR><TD>Number of Odr10 receptor per cell</TD><TD>Odr10/cell</TD><TD>Nb/cell</TD><TD>10<SUP>15</SUP></TD><TD>Arbitrary value</TD></TR><TR><TD><I>Pichia pastoris</I> transcription rate</TD><TD>k<SUB>transcript,<I>Pp</I></SUB></TD><TD>nt/s</TD><TD>50</TD><TD>Molecular Biology course - INSA Toulouse</TD></TR><TR><TD><I>Pichia pastoris</I> translation rate</TD><TD>k<SUB>translation,<I>Pp</I></SUB></TD><TD>nt/s</TD><TD>48</TD><TD>Molecular Biology course, Translation - Faculté des Sciences - Rabat (8)</TD></TR><TR><TD><I>AMP</I> gene length</TD><TD>DNA length</TD><TD>nucleotides</TD><TD>360</TD><TD>Parts design</TD></TR><TR><TD><I>AMP</I> mRNA length</TD><TD>RNA length</TD><TD>nucleotides</TD><TD>651</TD><TD>Parts design</TD></TR><TR><TD><I>Vibrio cholerae</I> minimal pathogenic concentration</TD><TD>[<I>Vc</I>]<SUB>pathogenic</SUB></TD><TD>cell/L</TD><TD>4.10<SUP>4</SUP></TD><TD>Medical Microbiology 4th edition (11)</TD></TR><TR><TD>CAI-1 initial concentration</TD><TD>[CAI-1]<SUB>0</SUB></TD><TD>mol/L</TD><TD>1.10<SUP>-5</SUP></TD><TD>Ng <I>et al.</I>, 2011 (12)</TD></TR><TR><TD>Michaelis-Menten constant of acetolactate synthase</TD><TD>K<SUB>M,ALS</SUB></TD><TD>mol/L</TD><TD>1.36.10<SUP>-2</SUP></TD><TD>Atsumi <I>et al.</I>, 2009 (13)</TD></TR><TR><TD>Catalytic rate constant of acetolactate synthase</TD><TD>k<SUB>cat,ALS</SUB></TD><TD>s<SUP>-1</SUP></TD><TD>1.21.10<SUP>2</SUP></TD><TD>Atsumi <I>et al.</I>, 2009 (13)</TD></TR><TR><TD>Degradation constant of acetolactate synthase</TD><TD>K<SUB>deg,ALS</SUB></TD><TD>s<SUP>-1</SUP></TD><TD>0</TD><TD>Assuming a negligeable value in our conditions</TD></TR><TR><TD>Degradation constant of antimicrobial peptides</TD><TD>K<SUB>deg,AMP</SUB></TD><TD>s<SUP>-1</SUP></TD><TD>0</TD><TD>Aleinein <I>et al.</I>, 2013 (14)</TD></TR><TR><TD>Activation constant of CqsS*-CAI-1 complex</TD><TD>K<SUB>a,CqsS*-CAI-1</SUB></TD><TD>mol/L</TD><TD>3.6.10<SUB>-8</SUB></TD><TD>Ng <I>et al.</I>, 2011 (12)</TD></TR><TR><TD>High <I>Vibrio cholerae</I> concentration in water</TD><TD>[<I>Vc</I>]<SUB>0</SUB></TD><TD>cell/L</TD><TD>1.10<SUB>7</SUB></TD><TD>Huq <I>et al.</I>, 1984 (15)</TD></TR><TR><TD>Dry weight of a bacterial cell</TD><TD>DW<SUB>Vh</SUB></TD><TD>pg/cell</TD><TD>0.28</TD><TD>BioNumbers 100008</TD></TR><TR><TD>Volume of a bacterial cell</TD><TD>V<SUB>intra,<I>Vh</I></SUB></TD><TD>μm<SUP>3</SUP></TD><TD>1</TD><TD>BioNumbers 100004</TD></TR><TR><TD>Volume of a yeast cell</TD><TD>V<SUB>intra,<I>Pp</I></SUB></TD><TD>μm<SUP>3</SUP></TD><TD>66</TD><TD>BioNumbers 100452</TD></TR><TR><TD>Dry weight of a yeast cell</TD><TD>DW<SUB>Pp</SUB></TD><TD>pg/cell</TD><TD>18.48</TD><TD>Estimation from yeast cell volume and ratio dry weight/volume for a bacterial cell</TD></TR><TR><TD>Ribosome density on a yeast cell</TD><TD>Nb/kb</TD><TD>6.5</TD><TD>BioNumbers 103026</TD></TR><TR><TD>Number of ribosome on <I>AMP</I> mRNA</TD><TD>Ribosome/RNA</TD><TD>Nb/RNA</TD><TD>2.34</TD><TD>Deduced from ribosome density and mRNA length</TD></TR><TR><TD>RNA polymerase density on a yeast gene</TD><TD>Nb/kb</TD><TD>6.5</TD><TD>BioNumbers 103026</TD></TR><TR><TD>Number of RNA polymerase on <I>AMP</I> DNA</TD><TD>RNA polymerase/DNA</TD><TD>Nb/DNA</TD><TD>4.95</TD><TD>BioNumbers 108308</TD></TR><TR><TD>Ribosome density on a bacterial mRNA</TD><TD>Nb/kb</TD><TD>6.6</TD><TD>BioNumbers 107727</TD></TR><TR><TD>Number of ribosome on <I>als</I> mRNA</TD><TD>Ribosomes/RNA</TD><TD>Nb/RNA</TD><TD>11</TD><TD>Deduced from ribosome density and mRNA length</TD></TR><TR><TD>RNA polymerase density on a bacterial gene</TD><TD>Nb/kb</TD><TD>7.6</TD><TD>Assuming the same density than yeast (BioNumbers 108308)</TD></TR><TR><TD>Number of RNA polymerase on <I>als</I> DNA</TD><TD>RNA Polymerase/DNA</TD><TD>Nb/DNA</TD><TD>13</TD><TD>Deduced from RNA polymerase density and DNA length</TD></TR><TR><TD>Pyruvate concentration in a bacterial cell</TD><TD>[S]</TD><TD>mol/L</TD><TD>3.9.10<SUP>-4</SUP></TD><TD>BioNumbers 101192</TD></TR></TBODY></TABLE><P>Even if we mostly describe our model as a predictive one, some preliminary experimental results could have been implemented into our model. Indeed, an experimental estimation of our two chassis behaviour on a common medium was needed.</P><TABLE style="font-size:8pt;"><TBODY><TR><TH>Name</TH><TH>Notation</TH><TH>Unit</TH><TH>Value</TH><TH>Reference</TH></TR><TR><TD><I>Vibrio harveyi</I> JMH626 maximum growth rate</TD><TD>μ<SUB>MAX,<I>Vh</I></SUB></TD><TD>s<SUP>-1</SUP></TD><TD>2.10<SUP>-4</SUP></TD><TD>Experiment - 21/06/17</TD></TR><TR><TD><I>Pichia pastoris</I> SMD1168 maximum growth rate</TD><TD>μ<SUB>MAX,<I>Pp</I></SUB></TD><TD>s<SUP>-1</SUP></TD><TD>4.10<SUP>-5</SUP></TD><TD>Experiment - 21/06/17</TD></TR><TR><TD><I>Vibrio cholerae</I> lag time</TD><TD>t<SUB>l,<I>Vc</I></SUB></TD><TD>s</TD><TD>1800</TD><TD>Experiment - 21/06/17, extrapolation from <I>V. harveyi</I> growth</TD></TR><TR><TD><I>Vibrio harveyi</I> lag time</TD><TD>t<SUB>l,<I>Pp</I></SUB></TD><TD>s</TD><TD>1800</TD><TD>Experiment - 21/06/17</TD></TR><TR><TD><I>Pichia pastoris</I> lag time</TD><TD>t<SUB>l,<I>Pp</I></SUB></TD><TD>s</TD><TD>14 400</TD><TD>Experiment - 21/06/17</TD></TR></TBODY></TABLE></SECTION><SECTION><H1>Solver</H1><P>The system of ODEs was solved using <B>Matlab R2017a</B>, thanks to the free offer from iGEM. Initially set with <I>ode45</I> solver, the recommanded Matlab ODE solver, the final script uses <I>ode15s</I> because <I>ode45</I> was not enough efficient. <I>ode15s</I> is the solver recommanded when having problems or inefficiency with <I>ode45</I>, and is adapted to stiff problems.(16)</P><P>You can freely re-use our code: <A href="" alt="">General_resolution + System_of_ODEs</A>.</P></SECTION><SECTION><H1>References</H1><UL><LI>(6) : http://bitesizebio.com/22824/how-to-manipulate-plasmid-copy-number/
</LI><LI>(16) : https://fr.mathworks.com/help/matlab/math/choose-an-ode-solver.html
</LI></UL></SECTION></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>