<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_INSA-UPS_France_Collaborations skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:INSA-UPS France/Collaborations</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container" style="position:relative; "><DIV class="site-header__title">
    Croc'n Cholera <SPAN style="font-size:12pt;">iGEM UPS-INSA Toulouse 2017</SPAN></DIV><NAV class="site-menu" style="text-align: center;"><DIV class="site-menu__item"><DIV class="cat_name">Team </DIV><UL class="subnav" style="width:150%;"><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Team">Members</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Supports">Supports</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Attributions">Attributions</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Collaborations">Collaborations</A></LI></UL></DIV><DIV class="site-menu__item"><DIV class="cat_name">Strategy </DIV><UL class="subnav"><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Description">Description</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Design">Design</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Parts">Parts</A></LI></UL></DIV><DIV class="site-menu__item"><DIV class="cat_name">Model </DIV><UL class="subnav"><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Model">Overview</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Model/Simulation">Simulation</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Model/Analysis">Analysis</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Model/Interface">Interface</A></LI></UL></DIV><DIV class="site-menu__item"><DIV class="cat_name">Realisations </DIV><UL class="subnav"><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Experiments">Overview</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Notebook">Notebook</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Experiments/Clonings">Clonings</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Results">Results</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Contribution">Contribution</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Protocols">Protocols</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Safety">Safety</A></LI></UL></DIV><DIV class="site-menu__item"><DIV class="cat_name">Human Practices</DIV><UL class="subnav"><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Human_Practices">Overview</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/HP/Silver">Silver</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/HP/Gold_Integrated">Gold</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Engagement">Public Engagement</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Microbioworld">Microbioworld</A></LI></UL></DIV><DIV class="site-menu__item"><DIV class="cat_name">Application </DIV><UL class="subnav"><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Applied_Design">Applied Design</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Entrepreneurship/Testimonies">Testimonies</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Entrepreneurship">Entrepreneurship</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Entrepreneurship/Device">Device Conception</A></LI></UL></DIV></NAV></DIV><DIV class="main_content"><DIV class="middle_container"><DIV class="section_container"><SECTION style="display:table;background:none;padding:0px !important;z-index:100; "><H1 style="vertical-align:bottom;display:table-cell; width:50%;font-size:40pt;letter-spacing: 0.1em;z-index:120;text-align: center;">Collaborations</H1></SECTION><SECTION><H1>Freeze-dried Lactobacillus Experiment</H1><P><I>Done for: Purdue team</I></P><H2>Their project:</H2><P><A href="https://2017.igem.org/Team:Purdue">Purdue team</A> is working on a spray device to purify lungs from benzene and toluene. Their idea is to spray engineered bacteria in the lungs that could purify lungs, thanks to their device. So they imagined a spray device supposed to be kept at -20°C and they were looking for a way to efficiently conserve bacteria at such temperature in a medium that human could ingest.
      </P><H2>Their experiment:</H2><P>
        It’s known that <I>E. coli</I> can be conserved for a long time in a milk/sucrose mix at -80°C. They enhanced the conservation protocol to maintain <I>E. coli</I> as long as possible in the milk/sucrose mix at -20°C.
      </P><H2>Our experiment:</H2><P>
        We obtained a lung microbiota strain, <I>Lactobacillus spp</I> ( coming from the Micalis lnstitute), to know whether the protocol they established could be used to maintain a real microbiota strain.
      </P><H2>Procotol used:</H2><P>
        In order to know if the lung strain could be maintained in milk for 2 weeks we made aliquots of bacteria in the milk mixture and everyday tubes were defrosted, dilutions were made and plated on petri dish. Two days after CFU were counted. Everyday 3 tubes of milk aliquots to check repeatability of countings were defrosted and 1 containing bacteria in water as a control.
      </P><P>
        The milk mixture was made with 18% of milk powder + 4% sucrose + 78% of sterilized water. The mixture was then pasteurized.
      </P><FIGURE><FIGCAPTION>Petri dishes (MRS) with 10<SUP>-4</SUP>, 10<SUP>-5</SUP>, 10<SUP>-6</SUP> and 10<SUP>-7</SUP> dilutions of Lactobacillus spp</FIGCAPTION></FIGURE><H2>Results</H2><P>
        CFU counts show that <B>the freezing  induced an expected decrease of CFU</B> in both milk and water (approximatively 35% of decrease after one day of freezing). In the following days, <B>CFU counts from milk have stabilized</B> to approximatively 4.10<SUP>9</SUP> CFU/ml while CFU counts from water have stabilized to 3.10<SUP>9</SUP> CFU/ml. Despite there were more CFU counted from the milk Eppendorfs <B>we can’t conclude milk is a better way to maintain <I>Lactobacillus spp</I> than water</B> because bacteria quantities were not the same at J0.
      </P><P>
        But we made an <B>interesting observation</B> about <I>Lactobacillus</I> colonies. Indeed we noticed that after a while (approximatively 10 days), half of the colonies coming from the sterilized water conditions were significantly smaller than the ones which were maintained in milk.
      </P><P>
        Our conclusion is that the freezing induces a loss of approximatively 35% of bacteria and that both water and milk are good choices to maintain <I>Lactobacillus</I> spp at -20°C. It’s interesting to add that it looks like this bacterial strain could have been keep longer in the fridge and that <B>the milk solution seems to offer better maintaining conditions because colonies were bigger on petri dishes</B>.
      </P></SECTION><SECTION><H1>Cloning</H1><P><I>From : BOKU-Vienna</I></P><P>
        During our 3 months of experiments we did several genomic integrations in <I>Pichia pastoris</I>. But a part remained non-integrated despite all our efforts.
We sent to the <A href="https://2017.igem.org/Team:BOKU-Vienna">BOKU-Vienna Team</A> our yeast strain and the construction below.
They succeeded the genomic integration and thanks to them we did a fluorescence experiment to characterize the GAP promoter activity. 
      </P><P>
Thanks to their work we obtained results showing that the GAP promoter we were working with is functional. As you can see on the figure, the <I>Pichia pastoris</I> strain they engineered shows a higher fluorescence than the wild type, proving the functionality of pGAP.
The functionality of pGAP has been further demonstrated later with an halo assay. 
      </P></SECTION><SECTION><H1>Creation of mazes</H1><P><I>Done for : NAWI-Graz Team</I></P><H2>Their project:</H2><P>
        The aim is to create a robot-bacteria interface in which information processing is done by a bacterial culture that exhibits a feedback loop with a mobile robot. In this concrete case changes in pH and a corresponding change in fluorescence of the bacterial culture are the means by which communication is achieved. Additionally, one of the fluorescence proteins used is under control of an acid inducible promoter, which they aim to further develop and characterize.
      </P><H2>Their tool:</H2><P>
        To test their algorithm they created a tool that iGEMers can use. The idea is to let people draw different mazes so they can test them.
      </P><H2>Our mazes:</H2><P>
        The robot is mainly going in circle and can’t find the exit when the way is too small. It’s easily trapped. It would have been interesting to have a bigger space designed for drawing.
      </P></SECTION><SECTION><H1>Combination of two projects about cholera</H1><P><I>Done with: Grenoble Team</I></P><P>
        We met the <A href="https://2017.igem.org/Team:Grenoble-Alpes">Grenoble Team</A> to talk about our respective projects as they are both dealing with the cholera disease. 
      </P><P>
        We designed a device using synthetic communication between 2 micro-organisms to detect and kill <I>Vibrio cholerae</I> in water. This device is user-friendly and can treat water before consumption thus avoiding cholera epidemic. 
      </P><P>
        iGEM Grenoble designed a pathogen sensitive detector: SnapLab, a portable kit that will detect cholera. The greatest advantage of this device will be its capability to communicate the results to its user and to map the cases of cholera. One of the biggest advantages of the project is the fact that it can provide an energy independent solution available for physicians and on-site helpers to provide a relevant diagnosis. On the other hand, this solution is ready to use, avoiding all the preparation steps required in a laboratory.
      </P><P>
        Both teams were really aware that the crisis management must be enhanced to reduce the lethal cases of a disease that shouldn’t kill anymore! Our solutions are complementary and can really help locals NGO. Indeed, we provided a really helpful solution to decontaminate infected waters (one of the biggest vehicles of transmission) beforehand, while iGEM Grenoble can detect and monitors the evolution of an outbreak by providing an easy way to diagnose that disease. Both projects are complementary as one act to avoid epidemic to start and the other one enables a precise detection of the bacteria.
      </P><P>
        We could imagine a whole device integrating both project: the Croc’n cholera project of the INSA-UPS_France team does not integrate any tangible detection that could be compensated by the SnapLab. On the other hand Croc’n cholera leads to <I>Vibrio cholerae</I> destruction while SnapLab remains a detection tool.
      </P><P>
        In the end, it appears that a combination of our projects can reduce the length of an outbreak because we propose to find the sources of spreading and on the other hands to facilitates the access of healthcare by providing a good diagnosis solution.
      </P></SECTION><SECTION><H1>Survey</H1><P><I>From: Greece team</I></P><P>
        We made a survey about the cholera disease to know if people are aware of this disease as it does not occur in Western Countries. We already had many answers in France and the Greece Team proposed to share it in their country. Here we provide results of this survey in both countries.
      </P><P>
        96% of people who answered to the survey in Greece are from scientific studies while only 70% for France. Despite that, it looks like french people answered with more accuracy than greek people. But it may be because of statistics, indeed, we received approximately 550 french answers and only 100 greek answers.
      </P></SECTION><SECTION><H1>Postcard project</H1><P><I>Done for : Gröningen Team</I></P><H2>Description</H2><P>
        The Gröningen Team set up a project to exchange postcards about synthetic biology. Each participating team creates its own postcard.
      </P><H2>Our postcard</H2><P>
        We chose to draw our logo on a petri dish with orange and yellow <I>E. coli</I>. Indeed, after a brainstorming was done in May, we still had two iGEM projects. One about cholera/crocodile and the other about modifying bacteria depending on light so it could form a drawing. We ended choosing the crocodile project so this postcard was the opportunity to mix our 2 main projects: crocodile and art.
      </P></SECTION><SECTION><H1>Database of host organisms called ChassiDex</H1><P>
           After filling out the survey of <A href="https://2017.igem.org/Team:IIT-Madras">IIT Madras Team</A> they sent us their database and asked for help to fill it. Indeed, ChassiDex is a database to gather the main information on organisms in order to modify them easily. Information such as biosafety level, genotype of strains, protocols of transformations and also iGEM BioBricks already available are collected to help future iGEM teams to quickly choose their organisms and to find relevant literature about it. 
       </P><P>

In the context of our project Croc’n cholera we entered data for <I>Pichia pastoris</I> SMD1168H and <I>Vibrio harveyi</I> JMH626

       </P></SECTION><SECTION><H1>Meetups</H1><P>
        We participated in the European iGEM Meetup in Delft on the 7th of July and also in the French Meetup in Paris on the 22nd of July.
      </P><P>
        It was obviously a nice way to practice our presentation and our poster before going to the Giant Jamboree, but it also enabled us to discuss our project with other iGEMers and scientists. As we presented the context of cholera and our ideas of a solution with synthetic biology, we got to confront with a lot of questions about ethics, safety and usefulness of our project. People gave us interesting feedbacks and it made us question our solution from other perspectives.

      </P><H2>European Meetup at TU Delft</H2><P>
On the 7th and 8th of July, our team participated to the European iGEM Meetup in Delft. 
The early presentations opened our mind to a lot of issues mainly about integrated human practices and the possibilities offered by bioinformatics to support the synthetic biology work. 
We also had the opportunity to practice the poster session by introducing our project to the other teams. We shared a lot of feedbacks about projects, poster tips, suggestions about experiments, encouragements for the wet lab work and best luck for the clonings. 
We truly had the feeling to establish beneficial dialogues with the other teams in order to do our best for the giant competition. 
</P><P>
It was also a way to meet our interlocutors for collaborations and to build ephemera collaborations or longer collaborations to challenge other iGEMers for a Volley Ball play. 
Indeed, this meetup was also a multicultural and socializing event. Teams coming from all corners of Europe were there. We shared our experience of the 2017 competition: the &quot;goods&quot; and the &quot;bads&quot; and learned to know each other through diversity. 
The meetup ended with a tour in Delft. We discovered a city, between lands and canals and its historical attractions. This moment shared with our iGEM fellows had its own importance making us practice observation, curiosity and sensitivity which are the best allies for young scientists.
</P><P>
As final teaching from our expedition to Delft: <I>gnothi seauton</I> or know thyself. As the team just started lab adventure, this event allows to improve the cohesion and the complicity between three members of our team during a 22 hours bus trip to Netherlands and a 22 hours trip back to Toulouse. What an experience!  
</P><H2>3rd Parisian Meetup</H2><P>
        On the 21th of July, the Pasteur Team organized an event to meet other French teams. We had the opportunity for the first time to introduce our project in front of a judging panel and to discover other teams’ projects. It was a rewarding experience to have the point of view of other iGEMers and interesting reviews from scientists. At the Parisian Meetup, judges were here to point out problems in our mock-up Jamboree presentations, but it revealed itself useful to improve our general communication. Here are some examples of the feedback we received and the conclusions we made:
      </P><UL><LI>No explanation of what to do with waste (tea bag)</LI><LI>Figure out how to deal with resistance problem and reaction (AMPs) </LI><LI>Drinking water with GMOs implies other risks than drinking water with cholera</LI></UL><P>
        Our team was also pleased to see how others managed to progress with their own projects. It was interesting to confront our ideas of human practices with theirs and particularly when parts of our projects were similar. (e.g. peptide production, treating water, dealing with worldwide health issues)
      </P><P>
        At this point of the competition, all teams were unsure about some aspects of their projects, and these meetups contributed to make progress on how to answer interrogations / solve ethical issues.
      </P></SECTION><SECTION><H1>Skype meeting</H1><H2><A href="2017.igem.org/Team:Greece">Greece</A></H2><P> 
        It was our first skype meeting. We had the opportunity to talk about modeling because the Greece team already had modeled the comportment of a quorum sensing molecule in a cell population. This modeling could have been related to our project with the diacetyl acting like a quorum sensing molecule between <I>V. harveyi</I> and <I>P. pastoris</I>. Moreover they were interested in NMR facilities we use to detect quorum sensing molecule.
        </P><H2><A href="2017.igem.org/Team:NUS_Singapore">Singapour</A></H2><P>
        We did a Skype to meet each other and to introduce our project. We talked about collaboration but didn’t find how to help each other.
      </P><H2><A href="2017.igem.org/Team:Purdue">Purdue</A></H2><P>
        We had an interesting exchange about collaboration. Indeed, their project is about using lung microbiota to detoxify lungs and because one of our members already dealt with strains coming from the lung microbiota, it was the opportunity to do an experiment for them with these lung strains (experiments and obtained results are described above). More than that, in their lab they had the opportunity to use rats organs which could have been useful to test the cytotoxicity of our AMP once we’d produced them. But we couldn’t produce them soon enough.

      </P><H2><A href="2017.igem.org/Team:NortheasternU-Boston">Boston</A></H2><P>
        Their project is about producing AMPs against a wide-range of bacteria causing world-wide disease like malaria or cholera. with an in vitro system. Because their system is based upon AMP without a methionine in N-ter they were interested in our project. As their production is based on an in vitro production and ours is based on the communication of several microorganisms, we had a lot of points to compare. We talked about our protocol, pros and cons of our projects and so on. We would have liked to collaborate with this team but while they were starting their experiments we were ending ours on the shareable part.
      </P><H2><A href="2017.igem.org/Team:Groningen">Groningen</A></H2><P>
        Both of our projects deal with the detection of  a microorganism in a food product. They aim to detect bacteriophages in milk while our goal is to detect <I>V. cholerae</I> in water. We had a talk about GMOs legislation in food product and exchanged our ideas about the way we would keep our engineered microorganisms away from human consumption (filter membrane for instance).
      </P></SECTION><SECTION><H1>Surveys</H1><P>
        During this year, our team created a survey in order to better know how far is the knowledge of common people about cholera and to start doing a market research/study. Beside that we filled surveys for other teams:
      </P><UL><LI><A href="2017.igem.org/Team:Cardiff_Wales">Cardiff</A>: Views on Genetic Modification to Produce Medicine</LI><LI><A href="2017.igem.org/Team:Amazonas_Brazil">Amazonas Brazil</A>: Biological Material Transport Survey</LI><LI><A href="2016.igem.org/Team:BostonU_HW">BU-Hardware</A>: Synbio/Microfluidics Survey</LI><LI><A href="2017.igem.org/Team:IIT-Madras">Madras</A>: What's your chassis?</LI><LI><A href="2017.igem.org/Team:Sydney_Australia">Sydney</A>:  Insuline Accessibility survey</LI><LI><A href="2017.igem.org/Team:UNBC-Canada">UNBC</A>: Resistant bacteria and synthetic biology</LI><LI><A href="2017.igem.org/Team:IONIS-PARIS">IONIS</A>: Sondage sur la perception du public envers les Organismes Génétiquement Modifiés (OGM)</LI><LI><A href="2017.igem.org/Team:Pasteur_Paris">Pasteur</A>: Quality of air</LI><LI><A href="2017.igem.org/Team:Harvard">Harvard</A>: iGEM Global Perspectives Outreach Survey </LI><LI><A href="2017.igem.org/Team:IISER-Mohali-INDIA">Ilser Mohali</A>: Green Survey</LI><LI><A href="2017.igem.org/Team:Evry_Paris-Saclay">Evry Paris Saclay</A>: The Pleasure of Sugar without the disadvantages: Psicose</LI><LI><A href="2017.igem.org/Team:Munich">Team Munich</A>: Diagnosis device survey</LI><LI><A href="2017.igem.org/Team:ColumbiaNYC">ColumbiaNYC</A>: Perspectives on Treatments for Illnesses</LI><LI><A href="2017.igem.org/Team:WPI_Worcester">WPI_Worcester Team</A>:  Lead Contamination in Your Drinking Water</LI><LI><A href="2017.igem.org/Team:Worldshaper-XSHS">Worldshaper-Wuhan</A>: A survey on colorectal cancer</LI><LI><A href="2017.igem.org/Team:Heidelberg">Heidelberg</A>: Directed Evolution and Artificial Intelligence Survey</LI><LI><A href="2017.igem.org/Team:UNebraska-Lincoln">
        UNebraska-Lincoln</A>: General Public Survey</LI></UL></SECTION><SECTION class="links_end"><TABLE><TBODY><TR><TH colspan="4">
            Team pages
        </TH></TR><TR><TD><A href="https://2017.igem.org/Team:INSA-UPS_France/Team">Members</A></TD><TD><A href="https://2017.igem.org/Team:INSA-UPS_France/Supports">Supports</A></TD><TD><A href="https://2017.igem.org/Team:INSA-UPS_France/Attributions">Attributions</A></TD><TD><I>Collaborations</I></TD></TR></TBODY></TABLE></SECTION></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>