<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_INSA-UPS_France_Protocols skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:INSA-UPS France/Protocols</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container" style="position:relative; "><DIV class="site-header__title">
    Croc'n Cholera <SPAN style="font-size:12pt;">iGEM UPS-INSA Toulouse 2017</SPAN></DIV><NAV class="site-menu" style="text-align: center;"><DIV class="site-menu__item"><DIV class="cat_name">Team </DIV><UL class="subnav" style="width:150%;"><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Team">Members</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Supports">Supports</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Attributions">Attributions</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Collaborations">Collaborations</A></LI></UL></DIV><DIV class="site-menu__item"><DIV class="cat_name">Strategy </DIV><UL class="subnav"><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Description">Description</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Design">Design</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Parts">Parts</A></LI></UL></DIV><DIV class="site-menu__item"><DIV class="cat_name">Model </DIV><UL class="subnav"><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Model">Overview</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Model/Simulation">Simulation</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Model/Analysis">Analysis</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Model/Interface">Interface</A></LI></UL></DIV><DIV class="site-menu__item"><DIV class="cat_name">Realisations </DIV><UL class="subnav"><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Experiments">Overview</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Notebook">Notebook</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Experiments/Clonings">Clonings</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Results">Results</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Contribution">Contribution</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Protocols">Protocols</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Safety">Safety</A></LI></UL></DIV><DIV class="site-menu__item"><DIV class="cat_name">Human Practices</DIV><UL class="subnav"><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Human_Practices">Overview</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/HP/Silver">Silver</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/HP/Gold_Integrated">Gold</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Engagement">Public Engagement</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Microbioworld">Microbioworld</A></LI></UL></DIV><DIV class="site-menu__item"><DIV class="cat_name">Application </DIV><UL class="subnav"><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Applied_Design">Applied Design</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Entrepreneurship/Testimonies">Testimonies</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Entrepreneurship">Entrepreneurship</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Entrepreneurship/Device">Device Conception</A></LI></UL></DIV></NAV></DIV><MAIN class="site-main"><DIV class="main_content"><DIV class="middle_container"><DIV class="section_container"><DIV class="main_title"><DIV><P>Protocols</P></DIV></DIV><SECTION class="protocols"><H1>Medium and antibiotics</H1><DIV class="prot-inside"><H2>Introduction</H2><P>
            Here are listed antibiotics concentration and media recipe used during the experiments.
          </P><H2>LB medium</H2><P><TABLE><TBODY><TR><TH>Tryptone</TH><TD>10 g/L</TD></TR><TR><TH>Yeast extract</TH><TD>5 g/L</TD></TR><TR><TH>NaCl</TH><TD>10 g/L</TD></TR><TR><TH>Water</TH><TD>Up to 1 L</TD></TR></TBODY></TABLE></P><P>
For solid medium, add 15 g/L of agar. 
Medium need to be autoclaved before use.
          </P><H2>LM medium</H2><P><TABLE><TBODY><TR><TH>Tryptone</TH><TD>10 g/L</TD></TR><TR><TH>Yeast extract</TH><TD>5 g/L</TD></TR><TR><TH>NaCl</TH><TD>20 g/L</TD></TR><TR><TH>Water</TH><TD>Up to 1 L</TD></TR></TBODY></TABLE></P><P>
For solid medium, add 15 g/L of agar. 
Medium need to be autoclaved before use.
          </P><H2>M9 medium</H2><P><TABLE><TBODY><TR><TH><U>5X Salts</U></TH><TD><B>For 1 L of final solution</B></TD><TD><B>[Final] in M9</B></TD></TR><TR><TH>Na<SUB>2</SUB>, H<SUB>2</SUB>PO<SUB>4</SUB>, 12 H<SUB>2</SUB>O</TH><TD>90 g</TD><TD>18 g/L</TD></TR><TR><TH>KH<SUB>2</SUB>PO<SUB>4</SUB></TH><TD>15.65 g</TD><TD>3.03 g/L</TD></TR><TR><TH>NaCl</TH><TD>2.5 g</TD><TD>0.5 g/L</TD></TR><TR><TH>NH<SUB>4</SUB>Cl</TH><TD>10.55 g</TD><TD>2.11 g/L</TD></TR></TBODY></TABLE></P><P><TABLE><TBODY><TR><TH><U>MgSO<SUB>4</SUB> 1M</U></TH><TD><B>For 50 mL of final solution</B></TD><TD><B>[Final] in M9</B></TD></TR><TR><TH> MgSO<SUB>4</SUB></TH><TD>12.3 g</TD><TD>0.49 g/L</TD></TR></TBODY></TABLE></P><P><TABLE><TBODY><TR><TH><U>CaCl<SUB>2</SUB> 0.01M</U></TH><TD><B>For 50 ùL of final solution</B></TD><TD><B>[Final] in M9</B></TD></TR><TR><TH> CaCl<SUB>2</SUB></TH><TD>0.073 g</TD><TD>4.38 mg/L</TD></TR></TBODY></TABLE></P><P><TABLE><TBODY><TR><TH><U>1000X Salts</U></TH><TD><B>For 100 mL of final solution</B></TD><TD><B>[Final] in M9</B></TD></TR><TR><TH>Na<SUB>2</SUB>EDTA, 2 H<SUB>2</SUB>O</TH><TD>1.5 g</TD><TD>15 mg/L</TD></TR><TR><TH>ZnSO<SUB>4</SUB>, 7 H<SUB>2</SUB>O </TH><TD>0.45 g</TD><TD>4.5 mg/L</TD></TR><TR><TH>CoCl<SUB>2</SUB>, 6 H<SUB>2</SUB>O </TH><TD>0.03 g</TD><TD>0.3 mg/L</TD></TR><TR><TH>MnCl<SUB>2</SUB>, 4 H<SUB>2</SUB>O</TH><TD>1 g</TD><TD>10 mg/L</TD></TR><TR><TH>H<SUB>3</SUB>BO H<SUB>3</SUB></TH><TD>0.1 g</TD><TD>1 mg/L</TD></TR><TR><TH>Na<SUB>2</SUB>MoO H<SUB>4</SUB>, 2 H<SUB>2</SUB>O</TH><TD>0.04 g</TD><TD>0.4 mg/L</TD></TR><TR><TH>FeSO<SUB>4</SUB>, 7 H<SUB>2</SUB>O</TH><TD>0.3 g</TD><TD>3 mg/L</TD></TR><TR><TH> CuSO<SUB>4</SUB>, 5 H<SUB>2</SUB>O </TH><TD>0.03 g</TD><TD>0.3 mg/L</TD></TR></TBODY></TABLE></P><P>
EDTA and ZnSO<SUB>4</SUB> are dissolved in 80 mL of mQ water and pH is adjusted to 6. Other compound are added and pH is maintained to 6. Once all compounds are dissolved, water is adjusted to 100 mL and pH to 4. Solution is filtered on 0.2 µm and stored at -4 ° C
          </P><P><TABLE><TBODY><TR><TH><U>100X thiamine</U></TH><TD><B>For 10 mL of final solution</B></TD><TD><B>[Final] in M9</B></TD></TR><TR><TH> Hypochloride thiamine&gt;</TH><TD>0.1 g</TD><TD>0.1 g/L</TD></TR></TBODY></TABLE></P><P>
pH is adjusted to 2 with HCl, solution is filtered (0.2 µm) and stored at -4 ° C. this product is light sensitive.
          </P><P>
For 1 L of M9 media, all the following recipe are mixed together under sterile condition.
          </P><TABLE><TBODY><TR><TH>Solution</TH><TD><B>Sterilisation</B></TD><TD><B>Volume</B></TD></TR><TR><TH><B>5X salts</B></TH><TD>autoclave</TD><TD>200 mL</TD></TR><TR><TH><B> MgSO<SUB>4</SUB> 1M </B></TH><TD>autoclave</TD><TD>2 mL</TD></TR><TR><TH><B> CaCl<SUB>2</SUB> 0.01M </B></TH><TD>autoclave</TD><TD>3 mL</TD></TR><TR><TH><B>1000X Salts </B></TH><TD>filtration (0.2 µm)</TD><TD>1 mL</TD></TR><TR><TH><B>100X thiamine</B></TH><TD>filtration (0.2 µm)</TD><TD>10 mL</TD></TR><TR><TH><B>Carbon source 40X</B></TH><TD>filtration (0.2 µm)</TD><TD>25 mL</TD></TR><TR><TH><B>Water</B></TH><TD>autoclave</TD><TD>759 mL</TD></TR></TBODY></TABLE><H2>YPB medium</H2><P><TABLE><TBODY><TR><TH>Baceriological peptone</TH><TD>20 g/L</TD></TR><TR><TH>Yeast extract</TH><TD>10 g/L</TD></TR><TR><TH>Glucose</TH><TD>20 g/L</TD></TR><TR><TH>Water</TH><TD>Up to 1 L</TD></TR></TBODY></TABLE></P><P>
For solid medium, add 15 g/L of agar. 
Medium need to be autoclaved before use. Glucose is added after autoclave.
          </P><H2>Complete Minimal Medium + glutamine</H2><P>
For CMM 2X
                      <TABLE><TBODY><TR><TH>YNB without amino acid</TH><TD>50 mL</TD></TR><TR><TH>Glucose 10%</TH><TD>100 mL</TD></TR><TR><TH>Adenine 1 mg/mL</TH><TD>10 mL</TD></TR><TR><TH>Histidine 10 mg/mL</TH><TD>1 mL</TD></TR><TR><TH>Leucine 10 mg/mL</TH><TD>1 lL</TD></TR><TR><TH>Isoleucine 10 mg/mL</TH><TD>1 mL</TD></TR><TR><TH>Lysine 10 mg/mL</TH><TD>1.5 mL</TD></TR><TR><TH>Methionine 10 mg/mL</TH><TD>1 mL</TD></TR><TR><TH>Phenylalanine 10 mg/mL</TH><TD>2.5 mL</TD></TR><TR><TH>Tryptophane 5 mg/mL</TH><TD>2 mL</TD></TR><TR><TH>Tyrosine 0.5 mg/mL</TH><TD>30 mL</TD></TR><TR><TH> Uracile 2 mg/mL </TH><TD>5 mL</TD></TR><TR><TH>Water</TH><TD>45 mL</TD></TR></TBODY></TABLE></P><P>
For CMM glutamine:
                      <TABLE><TBODY><TR><TH>CMM 2X</TH><TD>75 mL</TD></TR><TR><TH>Glutamine 2%</TH><TD>15 mL</TD></TR><TR><TH>Water</TH><TD>60 mL</TD></TR></TBODY></TABLE></P><H2>Antibiotics</H2><P>
            Antibiotics are prepared as stock solution of 1000X to facilitate further utilization.
          </P><P><B>Materials</B></P><P><UL><LI> Antibiotic powders (usually stocked as CMR products or at 4°C)</LI><LI> Solvents (pipettes + falcon tube)</LI><LI> Filtration kits </LI><LI> Weighing instrument </LI><LI> Steril Eppendorf tubes</LI></UL></P><P>
Procedure :
          </P><P><OL><LI> Under a safety cabinet, weight 0,5g of antibiotic powder and transfer it in a 50 mL falcon tube. </LI><LI> Add 10 mL of the appropriate solvent, mix briefly. </LI><LI> Under PSM, sterilize the antibiotics by filtration and distribute in sterile eppendorf tubes. Fill the seringue before using the filtration membrane.</LI><LI> Annotate the tubes and store at -20°C in the appropriate box. </LI><LI> Dilute 1000x the stock solution in your media to get the fine working concentration. (5μL in 5mL of LB media). </LI><LI> Trash treatment : containers or vessel are not biological waste : use common trash and wash the reusable materials. Diluted antibiotics can be disposed of down the drain.</LI></OL></P><P><TABLE><TBODY><TR><TH><U>Antibiotic</U></TH><TD><B>Abbreviation</B></TD><TD><B>Solvent</B></TD><TD><B>[1000X stock]</B></TD><TD><B>[Culture] </B></TD></TR><TR><TH>Ampicillin</TH><TD>Amp</TD><TD>water</TD><TD>50 mg/mL</TD><TD><B>50 µg/mL</B></TD></TR><TR><TH>Chloramphenicol</TH><TD>Cm</TD><TD>ethanol</TD><TD>25 mg/mL</TD><TD><B>25 µg/mL</B></TD></TR><TR><TH>Kanamycin</TH><TD>Kan</TD><TD>water</TD><TD>50 mg/mL</TD><TD><B>50 µg/mL</B></TD></TR><TR><TH>Streptomycin</TH><TD>Sm</TD><TD>water</TD><TD>50 mg/mL</TD><TD><B>50 µg/mL</B></TD></TR><TR><TH>Tetracycline</TH><TD>Tet</TD><TD>ethanol</TD><TD>50 mg/mL</TD><TD><B>50 µg/mL</B></TD></TR><TR><TH>Zeocin</TH><TD>Zeo</TD><TD>water</TD><TD>25 mg/mL</TD><TD><B>50 µg/mL</B></TD></TR></TBODY></TABLE></P></DIV></SECTION><SECTION class="protocols"><H1>Cultivation conditions</H1><DIV class="prot-inside"><H2><I>E.coli</I></H2><P>
            Unless specified, <I>E.coli</I> K12 MG 1655 was grown at 37°C at 160 rpm and 37 °C for solid media
          <UL><LI> LB medium</LI><LI> M9 medium</LI></UL></P><H2><I>V. harveyi</I></H2><P> Unless specified, <I>V. harveyi</I> BB120 and JMH626 were grown at 30°C and 160 RPM for liquid media and 30 °C for solid media.
          <UL><LI> LB medium</LI><LI> LM medium</LI></UL></P><H2><I>P. pastoris</I></H2><P>
            Unless specified, <I>P. pastoris</I> was grown at 30°C and 160 RPM for liquid media and 30 °C for solid media
          <UL><LI> YPD medium</LI><LI> CMM glutamine medium</LI></UL></P></DIV></SECTION><SECTION class="protocols"><H1>DNA manipulation </H1><DIV class="prot-inside"><H2>PCR</H2><P>
        We used the <A href="https://tools.thermofisher.com/content/sfs/manuals/MAN0012393_Phusion_HighFidelity_DNAPolymerase_UG.pdf">Thermo Scientific Phusion High-Fidelity DNA Polymerase</A>. Amplification of templates with high GC content, high secondary structure, low template concentrations or long amplicons may require further optimization.
      </P><H3>Materials</H3><UL><LI>PCR thermocycler</LI><LI>PCR tubes</LI><LI>nuclease-free water</LI><LI>dNTP</LI><LI>Phusion HF Buffer (X5) or GC Buffer</LI><LI>Primers (both forward and reverse)</LI><LI>Template DNA</LI><LI>Phusion polymerase</LI></UL><H3>Procedure</H3><P>
        All components <B>should be mixed and centrifuged prior to use</B>. It is important to add Phusion DNA Polymerase last in order to prevent any primer degradation caused by the 3´→ 5´ exonuclease activity. 
      </P><P>
        Phusion DNA Polymerase may be diluted in 1X HF or GC Buffer just prior to use in order to reduce pipetting errors.
      </P><P>
        Use of high quality, purified DNA templates greatly enhances the success of PCR. 
      </P><OL><LI>
          We recommend assembling all reaction components on ice and quickly transferring the reactions to a thermocycler preheated to the denaturation temperature (98°C). 
          <TABLE><TBODY><TR><TH>Component</TH><TH>50 μL</TH><TH>final concentration</TH></TR><TR><TD>Nuclease-free water</TD><TD>qs 50 μL</TD></TR><TR><TD>Buffer Phusion HF (5X)</TD><TD>10μL</TD><TD>1X</TD></TR><TR><TD>10 mM dNTPs</TD><TD>1 μL</TD><TD>200 μM</TD></TR><TR><TD>10 μM Forward primer</TD><TD>2.5 μL</TD><TD>0.5 μM</TD></TR><TR><TD>10 μM Reverse Primer</TD><TD>2.5 μL</TD><TD>0.5 μM</TD></TR><TR><TD> DNA template (10 ng/μL)</TD><TD>1 μL</TD><TD>10ng</TD></TR><TR><TD>Phusion DNA Polymerase</TD><TD>0.5 μL</TD><TD>1.0 U/0.5 μL of reaction</TD></TR></TBODY></TABLE>
          Notes: Gently mix the reaction. Collect all liquid to the bottom of the tube by a quick spin if necessary
        </LI><LI>
          Transfer PCR tubes from ice to a PCR machine with the block preheated to 98°C and begin thermocycling:
          <TABLE><TBODY><TR><TH>Step</TH><TH>Temperature</TH><TH>Time</TH></TR><TR><TD>Initial denaturation</TD><TD>98°C</TD><TD>45 sec</TD></TR><TR><TD rowspan="3">30 cycles</TD><TD>98°C</TD><TD>15 sec</TD></TR><TR><TD>55°C</TD><TD>30 sec</TD></TR><TR><TD>72°C</TD><TD>30 sec/kb</TD></TR><TR><TD>Final extension</TD><TD>72°C</TD><TD>5 min</TD></TR><TR><TD>Hold</TD><TD>4°C</TD><TD>hold</TD></TR></TBODY></TABLE><TABLE><TBODY><TR><TH>Parts</TH><TH>Length</TH><TH>Time of extension</TH></TR><TR><TD>pGAP-cOT2 / pGAP-DNY15 / pGAP-Leucro / YFP / DsRed</TD><TD>1 kb</TD><TD>30 sec</TD></TR><TR><TD>harveyi 1 / 2 / 3 / Vc and Vh</TD><TD>2 kb</TD><TD>60 sec</TD></TR><TR><TD>Odr10-cOT2</TD><TD>3 kb</TD><TD>90 sec</TD></TR></TBODY></TABLE></LI><LI>
          Then purify the products thanks to PCR purification kit
        </LI></OL><H2>PCR purification</H2><H3>Introduction</H3><P>
        This protocol was extracted from <A href="https://tools.thermofisher.com/content/sfs/manuals/purelink_pcr_man.pdf">Invitrogen PureLink® PCR Purification Kit</A>. Refer to this protocol for troubleshooting. Use the PureLink® PCR Purification Kit to efficiently remove primers, dNTPs, enzymes, and salts from PCR products in less than 15 minutes. Use the kit with Binding Buffer High-Cutoff (B3) to remove primer dimers or short spurious PCR products. The purified PCR product is suitable for automated fluorescent DNA sequencing, restriction enzyme digestion, and cloning.
      </P><H3>Materials</H3><UL><LI>Binding Buffer (B2)</LI><LI>Binding Buffer High-Cutoff (B3)</LI><LI>Wash Buffer (W1)</LI><LI>Elution Buffer; 10 mM Tris-HCl, pH 8.5 (E1)</LI><LI>PureLink® PCR Spin Columns with Collection Tubes</LI><LI>PureLink® Elution Tubes (1.7 mL)</LI><LI>50–100 μL PCR product</LI><LI>100% isopropanol</LI><LI>96–100% ethanol</LI><LI>Sterile, distilled water (pH&gt;7.0)</LI><LI>Microcentrifuge capable of achieving &gt;10,000 × g</LI></UL><H3>Procedure</H3><P><I>/!\ The PureLink® PCR Purification Kit buffers contain guanidine hydrochloride and isopropanol. Always wear a laboratory coat, disposable gloves, and eye protection when handling buffers.</I></P><P><I>/!\ Do not add bleach or acidic solutions directly to solutions containing guanidine hydrochloride or sample preparation waste because it forms reactive compounds and toxic gases when mixed with bleach or acids.</I></P><P>
        Follow the recommendations below to obtain the best results:
      </P><UL><LI>Maintain a PCR volume of 50–100 μL</LI><LI>Save an aliquot of PCR products before purification to verify and check the amplicon on the gel</LI><LI>Use a centrifuge at room temperature for all steps</LI><LI>Pipet the Elution Buffer (E1) in the center of the column and perform a 1 minute incubation</LI><LI>Always use sterile water with pH 7–8.5, if you are using water for elution</LI></UL><OL><LI><B>Before starting</B>. Add isopropanol to the Binding Buffers and ethanol to the Wash Buffer according to the following table. After adding isopropanol or ethanol, store all buffers at room temperature.
          <TABLE><TBODY><TR><TH>Buffer</TH><TH>Cat. no. K3100-01</TH></TR><TR><TD>Binding Buffer (B2)</TD><TD>10mL 100% isopropranol</TD></TR><TR><TD>Binding Buffer HC (B3)</TD><TD>2.3mL 100% isopropranol</TD></TR><TR><TD>Wash Bufer (W1)</TD><TD>64mL 96-100% isopropranol</TD></TR></TBODY></TABLE></LI><LI><B>Binding DNA. </B></LI><LI>Add 4 volumes of PureLink® Binding Buffer (B2) with isopropanol (see before starting) or Binding Buffer HC (B3) with isopropanol (see before starting) to 1 volume of the PCR product (50–100 μL). Mix well.</LI><LI>Remove a PureLink® Spin Column in a Collection Tube from the package.</LI><LI>Add the sample with the appropriate Binding Buffer (from step 1 of this procedure) to the PureLink® Spin Column.</LI><LI>Centrifuge the column at room temperature at 10,000 × g for 1 minute.</LI><LI>Discard the flow through and place the spin column into the collection tube.</LI><LI><B>Washing DNA</B></LI><LI>Add 650 μL of Wash Buffer with ethanol (see before starting) to the column.</LI><LI>Centrifuge the column at room temperature at 10,000 × g for 1 minute. Discard the flow through from the collection tube and place the column into the tube.</LI><LI>Centrifuge the column at maximum speed at room temperature for 2–3 minutes to remove any residual Wash Buffer. Discard the collection tube. Then let the residual ethanol evaporate by placing the open column on the collection tube and let it sit for 5 mins.</LI><LI><B>Eluting DNA.</B></LI><LI>Place the spin column in a clean 1.7-mL PureLink® Elution Tube supplied with the kit.</LI><LI>Add 30 μL of Elution Buffer (10 mM Tris-HCl, pH 8.5) or sterile, distilled water (pH &gt;7.0) to the center of the column.</LI><LI>Incubate the column at room temperature for 1 minute.</LI><LI>Centrifuge the column at maximum speed for 2 minutes.</LI><LI>The elution tube contains the purified PCR product. Remove and discard the column. The recovered elution volume is ~48 μL. Store the purified PCR product at –20°C or use the PCR product for the desired downstream application.</LI></OL><H2>Colony PCR</H2><H3>Introduction</H3><P>This protocol was elaborated thanks to the help of Anthony Henras.</P><H3>Materials</H3><P>
        10 μL of 0.02N NaOH / 1 PCR
      </P><H3>Procedure</H3><OL><LI>Resuspend the equivalent of the tip of a P1000 pipette of the colony in 10 μL of 0.02N NaOH</LI><LI>Mix well (vortex)</LI><LI>Incubate 5 min at 95°C and then chill on ice for 10 min at 4°C (program the thermocycler to do so (Program YeastLysis))</LI><LI>
          For each PCR mix: <I>NOTE: mix on ice and put on the thermocycler directly after mixing</I><TABLE><TBODY><TR><TH>Component</TH><TH>Volume (μL)</TH></TR><TR><TD>Previous cell extract</TD><TD>2</TD></TR><TR><TD>Taq Pol Buffer</TD><TD>10</TD></TR><TR><TD>Forward oligo 100 10 μM</TD><TD>0.5</TD></TR><TR><TD>Reverse oligo 100 10 μM</TD><TD>0.5</TD></TR><TR><TD>dNTP</TD><TD>1</TD></TR><TR><TD>H2O</TD><TD>35.6</TD></TR><TR><TD>Taq DNA polymerase</TD><TD>0.4</TD></TR></TBODY></TABLE></LI><LI>
          Put on a thermocycler and start this cycle:
          <TABLE><TBODY><TR><TD>95°C</TD><TD>5 min</TD></TR><TR><TD rowspan="3">35 cycles</TD><TD>95°C</TD><TD>30 sec</TD></TR><TR><TD>55°C</TD><TD>1 min</TD></TR><TR><TD>72°C</TD><TD>3 min</TD></TR><TR><TD>72°C</TD><TD>10 min</TD></TR><TR><TD>22°C</TD><TD>∞</TD></TR></TBODY></TABLE></LI><LI>Migration on gel to check the results</LI></OL><H2> Gel extraction of DNA </H2><H3> Procedure </H3><P><I>Please, before doing your preparative gel, use one sample to make an analityc one !</I></P><OL><LI>Equilibrate a water bath or heat block to 50°C.</LI><LI>
        Excise a minimal area of gel containing the DNA fragment of interest.</LI><LI><B>Crucial:</B> To protect the UV box, it is a good idea to place the gel on a glass plate if available. </LI><LI>Try to get as little excess gel around the band as possible.</LI><LI>Weigh the gel slice containing the DNA fragment using a scale sensitive to 0.001 g.</LI><LI> 
          Add Gel Solubilization Buffer (L3) to the excised gel in the tube size indicated in the following table:
          <TABLE><TBODY><TR><TH>Gel</TH><TH>Tube</TH><TH>Buffer L3 Volume</TH></TR><TR><TD>≤2% agarose</TD><TD>1.7 mL polypropylene</TD><TD>3:1 (i.e., 1.2 mL Buffer L3: 400 mg gel piece)</TD></TR><TR><TD>&gt;2% agarose</TD><TD>5 mL polypropylene</TD><TD>6:1 (i.e., 2.4 mL Buffer L3: 400 mg gel piece)</TD></TR></TBODY></TABLE></LI><LI>
          Place the tube with the gel slice and Buffer L3 into a 50°C water bath or heat block. Incubate the tube at 50°C for 10 minutes. Invert the tube every 3 minutes to mix and ensure gel dissolution.
          </LI><LI>Note: High concentration gels (&gt;2% agarose) or large gel slices may take longer than 10 minutes to dissolve.</LI><LI>
          After the gel slice appears dissolved, incubate the tube for an additional 5 minutes.
          </LI><LI>
              Optional: For optimal DNA yields, add 1 gel volume of isopropanol to the dissolved gel slice. Mix well.
            </LI><LI><B>Before Starting:</B> Add ethanol to the Wash Buffer (W1) according to the label on the bottle.
        </LI><LI><B>Purifying DNA Using a Centrifuge</B></LI><LI><B>Load.</B> Pipet the dissolved gel piece onto a Quick Gel Extraction Column inside a Wash Tube. Use 1 column per 400 mg of agarose gel.
          </LI><LI>
              Note: The column reservoir capacity is 850 μL.
            </LI><LI><B>Bind.</B> Centrifuge the column at &gt;12,000 × g for 1 minute. Discard the flow-through and place the column into the Wash Tube.
        </LI><LI><B>Wash.</B> Add 500 μL Wash Buffer (W1) containing ethanol to the column.
        </LI><LI><B>Remove Buffer.</B> Centrifuge the column at &gt;12,000 × g for 1 minute. Discard the flow-through and place the column into the Wash Tube.
        </LI><LI><B>Remove Ethanol.</B> Centrifuge the column at maximum speed for 1–2 minutes. Discard the flow-through.
        </LI><LI><B>Elute.</B> Place the column into a Recovery Tube. Add 30 μL Elution Buffer (E5) to the center of the column. Incubate the tube for 1 minute at room temperature.
        </LI><LI><B>Collect.</B> Centrifuge the tube at &gt;12,000 × g for 1 minute.
        </LI><LI><B>Store.</B> The elution tube contains the purified DNA. Store the purified DNA at 4°C for immediate use or at −20°C for long-term storage.
        </LI></OL><H2> Migration on agarose gel </H2><H3>Introduction</H3><P>
        This protocol is the classical one used for electrophoresis. - You can adapt the concentration of agar according to the length of your fragment 1% agar if the DNA fragments are big 2% agar if the DNA fragments are small (the bigger fragment are sticked together) - Adapt the volume of the gel 15 to 30 mL for small gels and 150 to 200 mL for big gels 
      </P><H3>Procedure</H3><OL><LI>
          Thoroughly rinse gel housing and well-comb with dH2O.
        </LI><LI>
          Place gel mold perpendicular to flow direction, ensuring proper sealing of rubber gaskets.
        </LI><LI>
          Add the calculated amounts of 0.5xTBE and agarose to a fresh Erlenmeyer flask.
        </LI><LI>
          Heat in microwave until mixture can be dissolved.
          </LI><LI><B>CRITICAL:</B> Do not let the mixture boil over and out of the flask. Typical heating time for 50mL in a 2.45GHz microwave oven at full power is 30s. USE HEAT GLOVES
            </LI><LI>
          Gently swirl until well mixed and gently swirling periodically until ~55°C.
        </LI><LI>
          Gently pour molten agarose gel into housing, avoiding air bubbles.
        </LI><LI>
          Place desired well comb in desired position.
        </LI><LI>
          Once gelled, carefully remove well comb in a uniform fashion.
        </LI><LI>
          Remove gel mold and place in parallel direction to flow 
          </LI><LI><B>CRITICAL:</B> the deposit line has to be at the anode (negative pole)
            </LI><LI>
          Fill gel box with 0.5 xTAE until the gel is well covered.
        </LI><LI>
          Place the ladder on the gel, the native and digested plasmid (write down the gel map)
          </LI><LI><B>TIP:</B> When loading the sample in the well, maintain positive pressure on the sample to prevent bubbles or buffer from entering the tip. </LI><LI>
          Run the electrophoresis for 20-30min at 100V until the dye line is approximately 80% of the way down the gel
        </LI><LI>
          Turn OFF power, disconnect the electrodes from the power source, and then carefully remove the gel from the gel box. 
        </LI><LI>
          Place the gel into a container filled with 100 mL of TAE running buffer and 5 μL of EtBr, place on a rocker for 20-30 mins, r
        </LI><LI>
          Place the gel into a container filled with water and destain for 5 mins.
        </LI><LI>
          Reaveal under UV lamp, visualize your DNA fragments
        </LI></OL><H2> Miniprep. </H2><H3>Introduction</H3><P>
        This protocol was taken from the <A href="https://tools.thermofisher.com/content/sfs/manuals/MAN0013117_GeneJET_Plasmid_Miniprep_UG.pdf">ThermoScientific GeneJET Plasmid Miniprep Kit</A>. Safety: Both the Lysis Solution and the Neutralization Solution contain irritants. <B>Wear gloves when handling these solutions.</B></P><H3>Procedure</H3><UL><LI>
          Note: All steps should be carried out at room temperature. All centrifugations should be carried out in a microcentrifuge at ≥ 12 000 x g (10 000-14 000 rpm, depending on the rotor type).
        </LI><LI>
          Be sure that the concentrated solutions have been diluted with the appropriated buffer
     </LI></UL><OL><LI>
          Pick a single colony from a freshly streaked selective plate to inoculate 5mL of LB medium supplemented with the appropriate selection antibiotic.
        </LI><LI>
          Incubate for 12-16 hours at 37°C while shaking at 200-250 rpm
        </LI><LI>
          Centrifugate the bacterial culture, &gt;12 000 g in a microcentrifuge for 2 minutes at room temperature. Repeat until there is no more media.
        </LI><LI>
          Add to the pelleted cells:
          </LI><LI>250 μL of Resuspension Solution and vortex</LI><LI>250 μL of Lysis Solution and invert the tube 4-6 times. WAIT 2 min</LI><LI>350 μL of Neutralization Solution and invert the tube 4-6 times.</LI><LI>Lysis buffer must be neutralized before 5 minutes</LI><LI>Centrifuge 5 minutes.</LI><LI>Transfer the supernatant to the Thermo Scientific GeneJET Spin Column. Centrifuge 1 minute</LI><LI>Add 500 μL of Wash Solution and centrifuge for 60 s and discard the flow-through</LI><LI>Repeat step 5.</LI><LI>Centrifuge empty column for 1 minute.</LI><LI>Dry for 5 minutes </LI><LI>Transfer the column into a new tube.</LI><LI>Add 30 μL of Elution Buffer to the column and incubate 2 minutes. </LI><LI>Centrifuge 2 minutes.</LI><LI>Collect the flow-through.</LI></OL><H2>Ligation</H2><H3>Introduction</H3><P>
        Please see the <A href="https://www.neb.com/protocols/1/01/01/dna-ligation-with-t4-dna-ligase-m0202"> NEB website </A>for supporting information on this protocol.
      </P><H3>Materials</H3><UL><LI><A href="https://www.neb.com/products/b0202-t4-dna-ligase-reaction-buffer">10X T4 DNA Ligase Reaction Buffer</A></LI><LI><A href="https://www.neb.com/products/m0202-t4-dna-ligase">T4 DNA Ligase</A></LI><LI>Vector DNA (4kb) </LI><LI>Insert DNA (1kb) </LI><LI>nuclease-free water</LI></UL><H3>Procedure</H3><P><I>Note: T4 DNA Ligase should be added last. The table shows a ligation using a molar ratio of 1:3 vector to insert for the indicated DNA sizes.  Use <A href="http://nebiocalculator.neb.com/#!/"> NEB calculator</A> to calculate molar ratios.</I></P><OL><LI>Thaw the T4 DNA Ligase Buffer and resuspend at room temperature. <I>Tip: Alicuote the 10x buffer less concentrated so when thawing, the DTT gets soluble more easily.</I></LI><LI>
          Set up the following reaction in a microcentrifuge tube on ice:
          <TABLE><TBODY><TR><TH>Component</TH><TH>Volume (µL)</TH></TR><TR><TD>10X T4 DNA Ligase Buffer</TD><TD>2</TD></TR><TR><TD>Vector DNA: 50 ng (0.020 pmol)</TD></TR><TR><TD>Insert DNA: 37.5 ng (0.060 pmol)</TD></TR><TR><TD>Nuclease-free water</TD><TD>17</TD></TR><TR><TD>T4 DNA Ligase</TD><TD>1</TD></TR><TR><TD>Total</TD><TD>20</TD></TR></TBODY></TABLE></LI><LI>Gently mix the reaction by pipetting up and down and microfuge briefly.</LI><LI>For cohesive (sticky) ends, incubate at 16°C overnight or room temperature for 10 minutes. For blunt ends or single base overhangs, incubate at 16°C overnight or room temperature for 2 hours.</LI><LI>Heat inactivate at 65 degrees C for 10 minutes.</LI><LI>Chill on ice and transform 1-5 μl of the reaction into 50 μl competent cells. <I>Use 25 uL DH5α cells, and add 2 uL of reaction mixture.</I></LI></OL><H2>Chemical transformation (RbCl method)</H2><H3>Introduction</H3><P>
        This protocol was given by Stéphanie. The aim is to make yourself Top10 competent cells.
      </P><H3>Materials</H3><UL><LI>2 * Steri cup 250mL</LI><LI>TrisEDTA</LI></UL><H2>Procedure</H2><H3>Media and Solutions</H3><OL><LI>500 mL LB</LI><LI>
          200 mL TFB1:
          </LI><LI>0.59 g KOAc (Cf=30 mM)</LI><LI>2.42 g RbCl (100 mM)</LI><LI>0.29 g CaCl2 2H2O (10 mM)</LI><LI>1.98 g MnCl2 4H2O (50 mM)</LI><LI>30 g Glycerol (15% wt/vol)</LI><LI>Adjust to pH 5.8 with 0.2 M acetic acid (do not adjust pH with KOH). Add dH2O to 200 mL. Filter sterilize. Store refrigerated at 4°C.</LI><LI>
          200 mL TFB2:
          </LI><LI>0.42 g MOPS (10 mM)</LI><LI>2.21 g CaCl2 2H2O</LI><LI>0.24 g RbCl (10 mM)</LI><LI>30 g Glycerol (15% wt/vol)</LI><LI>Adjust to pH 6.5 with KOH. Add dH2O to 200 mL. Filter sterilize. Store refrigerated at 4°C.</LI></OL><H3>Preparation of Competent Cells</H3><OL start="4"><LI>Streak cells from frozen stock onto LB plate. Incubate O/N at 37°C.</LI><LI>Pick a single fresh colony to inoculate 5 mL of LB medium. Grow O/N at 37°C. <I>Do not vortex cells at any time after this point in the procedure.</I></LI><LI>Dilute 1 mL of culture into 50 mL LB medium prewarmed to 37°C. Grow at 37°C for 2 hours with agitation. Volumes can be scaled up 5X and all of the 5 mL overnight culture can be used.</LI><LI>Transfer culture to sterile 50 mL tube. Chill on ice 10-15 minutes.</LI><LI>Centrifuge for 10 mintutes at 2000 rpm at 4°C. Immediately aspirate off all the supernatant. <I>Do not allow cells to warm above 4°C at any time in this procedure.</I></LI><LI>Resuspend cells in 10 mL of ice-cold TFB1 with gentle re-pipetting. Use chilled glass of plastic pipette.</LI><LI>Incubate cells on ice for 5 minutes.</LI><LI>Repeat step 8</LI><LI>Resuspend cells in 2 mL of ice-cold TFB2 with gentle re-pipetting. Use micropipet tip (plastic).</LI><LI>Incubate cells on ice for 15 minutes. Cells may be used for transformation or frozen. <I>To freeze: aliquot cells 100 µL volumes into prechilled 0.5 mL microcentrifuge tubes (on ice). Freeze immediately on dry ice. Stire cells frozen at -80°C.</I></LI></OL><H3>Transformation of competent cells</H3><OL start="14"><LI>If starting with frozen competent cells, warm tube/cells by gently twirling between your fingers until just thawed (i.e., at ~0°C). Then, immediately place on ice for about 5 minutes.</LI><LI>
          Set up transformation as follows: 
          Add to 15 mL plastic round bottom tube on ice: 
          </LI><LI>0-9 µL TE (Tris 10mM + EDTA 1mM)</LI><LI>1-10 µL DNA (10-100 ng)</LI><LI>10 µL final volume → /!\ 10% max of the cell competent volume</LI><LI>Add 100 µL of competent cells and mix by gentle repipetting. <I>This method can be scaled down 2- to 4-fold. The maximum volume of DNA should be ~1/10 volume of cells and the maximum mass should be &lt;= 100 ng of DNA for 100 µL of cells.</I></LI><LI>Incubate cells on ice for 20-30 minutes.</LI><LI>Heat shock the cells exactly 90 seconds at 42°C.</LI><LI>Return cells on ice 2 minutes.</LI><LI>Add 1 mL of LB medium. Incubate at 37°C for 45-60 minutes with slow gentle shaking. <I>For blue/white color selection, spread IPTG and X-gal on plates now and hold at 37°C until use</I></LI><LI>Plate 0.1 - 0.2 mL of transformed cells on LB-plate containing the appropriate antibiotic (adn IPTG and X-gal if needed). Incubate overnight at 37°C. Place at 4°C to store and/or enhance blue color. <I>Note: The next day, liquid cultures of the transformants can be left 8 hours before the miniprep. In the best-case scenario, do the liquid culture at 8am and do the miniprep at 4pm.</I></LI></OL><H3>Testing competent cells</H3><OL start="22"><LI>Transform 100 µL of cells with 1 µL (10 pg) of pUC19 monomer (0.01 µg/µL).</LI><LI>Plate 0.25 mL of transformation mixture. Incubate overnight at 37°C.</LI><LI>Count CFU and calculate efficiency. Efficiency =# of colonies per µg =# of colonies X4 X 105. You should obtain 1-5 X 10<SUP>7</SUP>/µg from competent cells after one freeze-thaw cycle.</LI></OL><H2> Enzymatic digestion of DNA. </H2><H3>Introduction</H3><P>This protocol was extracted from the <A href="https://www.protocols.io/view/Single-temperature-Double-Digest-imsuj5">protocol from NEB website.</A></P><H3>Materials</H3><UL><LI>
          For analysis digestion:
          </LI><LI>Eppendorfs</LI><LI>1 μg of DNA </LI><LI>1 μL 10X buffer (most enzymes can be used in Cutsmart buffer, check on NEB website)</LI><LI>1U enzyme pour 1 μg ADN (0,5 μL for 1 μg DNA)</LI><LI>H<SUB>2</SUB>O qsp 10 μL</LI><LI>heating plate</LI></UL><LI>
          For preparative digestion:
          </LI><LI>Keep the same proportions and scale up for 30µL of DNA on 100µL final</LI><LI>If cut by the same Enzyme, please prepare a MIX with n+1 (n = number of sample)</LI><LI>For gel migration, add 2 μL of loading dye for each 10 μL mix</LI><H3>Procedure</H3><OL><LI>Mix all the elements</LI><LI>Incubate 1h at enzyme specific temperature (usually 37°C)</LI><LI>Check if heat inactivation is required and do it accordingly <I>/!\ if inactivation is done at high temperature put on ice after inactivation and then centrifuge to keep the  evaporated water.</I></LI></OL><H2> Electroporation of <I>P. pastoris.</I></H2><H3>Introduction</H3><P>
        Protocol from Lin-Cereghino, J., Wong, W., Xiong, S., Giang, W., Luong, L., Vu, J., Johnson, S. and Lin-Cereghino, G. (2005). Condensed protocol for competent cell preparation and transformation of the methylotrophic yeast Pichia pastoris. <I>BioTechniques</I>, 38(1), pp.44-48.
      </P><H3>Materials</H3><UL><LI>ice</LI><LI>linearized plasmid (with AvrII)</LI><LI>competent cells from the protocol cell preparation</LI><LI>electroporation apparatus</LI><LI>1.0M sorbitol</LI><LI>YPD</LI><LI>plates with gradient of zeocin</LI></UL><H3>Procedure</H3><OL><LI>Mix approximately 4-8μL (50–100 ng) of dialysed linearized plasmid DNA with 40 μL of competent cells in an electroporation cuvette</LI><LI>Incubate for 2 min on ice</LI><LI>Pulse 1500V, 25μF, 200Ω (You should have a Ꞇ between 4 and 5 ms. If it is &gt;5ms, there were too many ions in the mix. It can kill cells.) (was done previously with 1500V, 10μF, 600Ω -&gt; worked)</LI><LI>Resuspend immediately samples in 0.5 mL 1.0 M sorbitol and 0.5 mL YPD, incubate in a 30°C shaker for 1h30, and then plate on media containing increasing concentrations of zeocin (100, 250, 500, or 1000 μg/mL) for the selection of multicop</LI></OL></DIV></SECTION><SECTION class="protocols"><H1>NMR analysis</H1><DIV class="prot-inside"><H2>C8-CAI-1 analysis</H2><P>
            This protocol is used to quantify C8-CAI-1 production by NMR spectroscopy.
          </P><P>
Supernatant obtained by centrifugation of 50mL of total broth are freeze-dried, resuspended in 500µL of CDCl3, and spiked with 100 µL of TSP-d4 (1mM, in D2O) used as internal standard for quantification and as reference for chemical shifts. The resulting samples are analyzed at 280K by 1D 1H NMR on an Avance 800 MHz spectrometer (Bruker, Rheinstetten, Germany) equipped with a 5-mm z-gradient TPI probe, using a zgpr sequence with a 90° pulse of 7µs and a relaxation delay between scans of 5 s. A total of 64 scans were accumulated (128k data points with a spectral width of 10 ppm) after 4 dummy scans. All the spectra were acquired and processed on TopSpin 3.2 (Bruker).
          </P><H2>Diacetyl analysis</H2><P>
            This protocol is used to quantify diacetyl production by NMR spectroscopy.
          </P><P>
Supernatant (500µL) obtained by filtration of total broth (Sartolon polyamide 0.2µm, Sartorius) are spiked with 100 µL of TSP-d4 (1mM, in D2O) used as internal standard for quantification and as reference for chemical shifts. The resulting samples are analyzed at 280K by 1D 1H NMR on an Avance 500 MHz spectrometer (Bruker, Rheinstetten, Germany) equipped with a 5-mm z-gradient BBI probe, using a zgpr sequence for water suppression with a 90° pulse of 7µs and a relaxation delay between scans of 5 s. A total of 64 scans were accumulated (128k data points with a spectral width of 10 ppm) after 4 dummy scans. All the spectra were acquired and processed on TopSpin 3.2 (Bruker).          </P></DIV></SECTION><SECTION class="protocols"><H1>Solid Bioluminescence assay</H1><DIV class="prot-inside"><H2>Introduction</H2><P>    This protocol is based on the Experimental Procedure provided in the following publication: 
          </P><P>
Ng W-L, Perez LJ, Wei Y, Kraml C, Semmelhack MF &amp; Bassler BL (2011). “Signal production and detection specificity in Vibrio CqsA/CqsS quorum-sensing systems: Vibrio quorum-sensing systems.” Molecular Microbiology 79 1407–1417. <A href="https://www.ncbi.nlm.nih.gov/pubmed/21219472">https://www.ncbi.nlm.nih.gov/pubmed/21219472</A></P>
The aim is to detect the Vibrio harveyi quorum sensing molecule C8-CAI-1 in vivo from a spent culture fluid. Here, a clone of an Escherichia coli MG1655 strain able to synthesis the molecule is taken as an example. Please note that the whole procedure lasts 3 days.
          <H2>Materials</H2><P><UL><LI>Liquid LB medium</LI><LI> Solid LB medium</LI><LI> Sterile glucose solution </LI><LI> Sterile IPTG solution, final concentration in culture: 0.5 mM</LI></UL></P><H2>Procedure</H2><P><B>1<SUP>st</SUP> day.</B></P><P>
Liquid precultures of <I>E. coli</I><OL><LI> Prepare 2 test tubes with 5 mL of liquid LB medium complemented with chloramphenicol (final concentration in the medium: 25 mg/L) and glucose (final concentration in the medium: 10 g/L). Vortex to mix all the components.</LI><LI> From agar plates, inoculate one of the previous tubes with the <I>E. coli</I> MG1655 clone. Inoculate the other with the <I>E. coli</I> MG1655 negative control.</LI><LI> Incubate the tubes over-night at 37°C with shaking (160 rpm). </LI></OL></P><P>
Liquid precultures of <I>V. harveyi</I><OL><LI> Prepare 2 flasks with 10 mL of liquid LB medium.</LI><LI> From agar plates, inoculate one of the previous flasks with <I>V. harveyi</I> JMH626. Inoculate the other with <I>V. harveyi</I> BB120.</LI><LI> Incubate the flasks over-night at 30°C with shaking (160 rpm).</LI></OL></P><P><B>2<SUP>nd</SUP> day.</B></P><P>
Expression cultures of <I>E. coli</I><OL><LI> Measure OD of the 2 <I>E. coli</I> precultures at 600 nm.</LI><LI> Prepare 2 flasks with 20 mL of liquid LB medium complemented with chloramphenicol (final concentration in the medium: 25 mg/L) and glucose (final concentration in the medium: 10 g/L).</LI><LI> Vortex to mix all the components.</LI><LI> From the precultures, inoculate one of flasks with the <I>E. coli</I> MG1655 clone at OD = 0.1 Inoculate the other with the <I>E. coli</I> MG1655 negative control at OD = 0.1.</LI><LI> Incubate the flasks at 37°C with shaking (160 rpm).</LI><LI> When OD = 0.3, put the flasks at 30°C with shaking (160 rpm) for 15 minutes.
Then, add IPTG (final concentration in the medium: 0.5 mM) and incubate at 30°C with shaking (160 rpm).</LI><LI> When OD = 0.9, retrieve the supernatants as follows:
centrifugate all the cultures at maximum speed for 10 min.
filter the resulting supernatants through a 0.2 µm filter.</LI><LI> Store both supernatants at -20°C for subsequent use.</LI></OL></P><P>
Expression culture of <I>V. harveyi</I> BB120
          <OL><LI> Measure OD of the <I>V. harveyi</I> BB120 preculture at 600 nm.</LI><LI> Prepare 2 flasks with 20 mL of liquid LB medium.</LI><LI> From the preculture, inoculate the previous flasks at OD = 0.1</LI><LI> Incubate at 30°C with shaking (160 rpm).</LI><LI> Check bioluminescence regularly. When <I>V. harveyi</I> BB120 shows bright bioluminescence, retrieve the supernatant of one the cultures as follows:
centrifugate the whole culture at maximum speed for 10 min
filter the resulting supernatants through a 0.2 µm filter.</LI><LI> Store the supernatant at -20°C for subsequent use.
Keep the other <I>V. harveyi</I> BB120 culture at 30°C with shaking (160 rpm).</LI><LI>Puce 1</LI><LI>Puce 1</LI><LI>Puce 1</LI><LI>Puce 1</LI></OL></P><P>
Expression culture of <I>V. harveyi</I> JMH626
          <OL><LI> Measure OD of the <I>V. harveyi</I> JMH626 preculture at 600 nm.</LI><LI> Prepare 4 flasks with 10 mL of liquid LB medium.</LI><LI> From the preculture, inoculate the previous flasks at OD = 0.1</LI><LI> Incubate at 30°C with shaking (160 rpm) until OD = 0.6. </LI><LI> Centrifugate the 4 cultures at 4500 rpm for 6 min.</LI><LI> Discard the resulting supernatant and resuspend each of the pellets with 5 mL of liquid LB medium.</LI><LI> Add 5 mL of the supernatants obtained previously.
one flask must be complemented with the <I>E. coli</I> MG1655 clone supernatant.
one flask must be complemented with the <I>E. coli</I> MG1655 negative control supernatant.
one flask must be complemented with the <I>V. harveyi</I> BB120 supernatant.
one flask must be complemented with additional liquid LB medium.
</LI><LI> Prepare 3 LB agar plates and divide each of them into 5 identical zones. </LI><LI> On the plates, drop-off 70 µL of each of the <I>V. harveyi</I> JMH626 resuspended cultures.
On the fifth zone, drop-off 70 µL of the last <I>V. harveyi</I> BB120 liquid culture .</LI><LI> Incubate the 3 plates at 30°C over-night.</LI></OL></P><P><B>3<SUP>rd</SUP> day</B><OL><LI> Observe each of the plates in total darkness.</LI></OL></P></DIV></SECTION><SECTION class="protocols"><H1>Triparental conjugation</H1><DIV class="prot-inside"><H2>Materials</H2><P><B>Plates:</B></P><P>For conjugation:</P><UL><LI>1 plate of LB</LI><LI>1 plate of LB - Cmp - Gen or LB - Cmp - Amp (depends on the resistance cassette on the conjugative plasmid)</LI></UL><P><B>For control:</B></P><UL><LI>1 plate of LB</LI><LI>1 plate of LB - Cmp - Gen or LB - Cmp - Amp (depends on the resistance cassette on the conjugative plasmid)</LI></UL><P><B>Strains:</B></P><UL><LI>Recipient: <I>Vibrio harveyi</I> JMH626 Cmp<SUP>R</SUP> Kan<SUP>R</SUP></LI><LI>Helper: <I>Escherichia coli</I> - pRK2073 (helper plasmid)</LI><LI>Donor: <I>Escherichia coli</I> transformed with the gene of interest in a conjugative plasmid (pBBR1MCS-4 Amp<SUP>R</SUP> or pBBR1MCS-5 Gen<SUP>R</SUP>)</LI></UL><P><B>Other:</B></P><UL><LI>LB liquid media</LI><LI>Antibiotics: Spec, Cmp, Gen or Amp (depends on the resistance cassette on the conjugative plasmid)</LI><LI>Membranes for the conjugative culture</LI></UL><H2>Plan of experiment</H2><P><B>1. Liquid cultures overnight from glycerol stock or plates:</B></P><UL><LI><I>V. harveyi</I> JMH626: 5 mL of LB, 30°C</LI><LI><I>E. coli</I> helper - pRK2073: 5 mL of LB + Spec (at the recommended concentration), 37°C</LI><LI><I>E. coli</I> donor - conjugative plasmid: 5 mL of LB + Amp or Gen (at the recommended concentration), 37°C</LI></UL><P><B>2. Centrifugation steps:</B></P><UL><LI>Centrifugate 1 mL of each culture 4 min at 10,000 rpm</LI><LI>Throw the supernatant</LI><LI>Resuspend the pellet in 1 mL LB</LI><LI>Repeat the three steps with the resuspended solution</LI></UL><P><B>3. Conjugation</B></P><P>Conjugation mix:</P><P>In an eppendorf, add:</P><UL><LI>40 µL of the donor strain (<I>E. coli</I> with the conjugative plasmid)</LI><LI>40 µL of the helper strain (<I>E. coli</I> - pRK2073)</LI><LI>40 µL of the recipient strain (<I>V. harveyi</I> JMH626)</LI></UL><P>Depose a membrane on an LB plate and add 100 µL of the mix at the center of the membrane.</P><P>Control mix:</P><P>In an eppendorf tube, add:</P><UL><LI>50 µL of the helper strain (<I>E. coli</I> - pRK2073)</LI><LI>50 µL of the recipient strain (<I>V. harveyi</I> JMH626)</LI></UL><P>Depose a membrane on an LB plate and add 80 µL of the mix at the center of the membrane.</P><P>Incubate the two plates overnight at 30°C.</P><P><B>4. Resuspension and incubation</B></P><P>For both control and conjugation:</P><P>Put the membrane in 5 mL H<SUB>2</SUB>O (use falcone tube) and vortex it until the solution becomes unclear. Place 1.5 mL of the solution into an eppendorf tube and centrifugate 4 min at 10,000 rpm. Throw approximately 1200 µL of supernatant and resuspend the pellet in the remaining supernatant. Spread the solution on a LB - Cmp - Amp or LB - Cmp - Gen (depending on the resistance cassette on the donor plasmid). Incubate 48h at 30°C. </P><P><B>To sum up:</B></P></DIV></SECTION><SECTION class="protocols"><H1>Fluorescence microscopy</H1><DIV class="prot-inside"><H2>Sample preparation</H2><P><I>V. harveyi</I> were grown at 30°C and 160 RPM in liquid media LM with chloramphenicol over-night.
          </P><P>
            Microscope slide has been prepared with 2x diluted culture of <I>V. harveyi</I>.
          </P><H2>Observations under fluorescent microscope</H2><P>
            Lorem ipsum dolor sit amet, consectetur adipisicing elit. Harum, perspiciatis unde quasi enim ab repellendus deleniti vel earum, facere aspernatur magnam! Maiores in, eos at id! Necessitatibus eligendi modi, unde.
          </P></DIV></SECTION><SECTION class="protocols"><H1>Protein production and sampling</H1><DIV class="prot-inside"><H2>C8-CAI-1 production</H2><P><OL><LI> Precultures LB: Inoculation on the morning of <I>E. coli</I> K-12 MG1655 - VhCqsA and <I>E. coli</I> K-12 MG1655 - pSB1C3 (no insert, used as negative control) on liquid culture LB (10 mL) with Cmp, at 37°C, 160 RPM.</LI><LI> Culture on M9: On the morning, inoculation at OD(600nm) = 0.1 into 50 mL of M9  after 2 washing step (centrifuged 4000 g twice, resuspension of pellet in M9) with the glucose as carbon source (20 g/L) at 37°C, 160 RPM </LI><LI>At OD= 0.9, IPTG induction (0.5 mM). Growth overnight at 30 °C, 160 RPM</LI><LI> Samples are collected after a night of production. </LI></OL></P><H2> Diacetyl production </H2><P><OL><LI> Precultures LB: Inoculation on the morning of <I>E. coli</I> K-12 MG1655 - als and <I>E. coli</I> K-12 MG1655 - pSB1C3 (no insert, used as negative control) on liquid culture LB (10 mL) with Cmp, at 37°C, 160 RPM.</LI><LI> Precultures M9: On the evening, inoculation at OD(600nm) = 0.1 into 50 mL of M9 with the xylose as carbon source (30 mM) and previously prewarmed at 37°C. Growth overnight at 37°C, 160 RPM.</LI><LI> Culture M9: Pyruvate is added to the medium (30 nM), and 50 mL flasks with or without citrate (2.2 g/L) were prepared. On the morning, precultures M9 were inoculated into 50 mL flasks with or without citrate at OD(600nm) = 0.1. Growth at 30°C, 160 RPM.</LI><LI> Culture M9: The culture is followed during two days. Samples are collected at the end of exponential growth phase (OD(600nm) between 1.4 and 1.9 depending on the culture)</LI></OL></P><H2> AMP production </H2><H3>DNY-15</H3><OL><LI> One colony of <I>P. pastoris</I> positive transformant for pPICZα-D-NY15 was grown in 50 mL YPD 4% glucose medium in a 250mL culture flask. Same in YPD 5% glucose medium. The negative control was <I>P. pastoris</I> that as integrated the empty pPICZα plasmid.</LI><LI> The 4 cultures were shaken for 96 hours, 30°C, 160 RPM.</LI><LI> Cultures were centrifuged 10 min at 3000 g, then supernatant was collected for sampling.</LI></OL><H3>Leucrocine I and cOT2</H3><P>
See protocol  <A href="https://www.ncbi.nlm.nih.gov/pubmed/26854372">here</A>here: in M.R. Kuddus, F. Rumi, M. Tsutsumi, R. Takahashi, M. Yamano, M. Kamiya, T. Kikukawa, M. Demura, T. Aizawa, Expression, purification and characterization of the recombinant cysteine-rich antimicrobial peptide snakin-1 in Pichia pastoris, Protein Expression and Purification (2016).
</P><H2>Sampling</H2>
After recuperation of supernantant from culture,
<UL><LI> Supernatant was filtered (0.2 µm) for further analysis, Diacetyl</LI><LI> Supernantant was filtered (0.2 µm) then freeze dried then stored a -80°C for C8-CAI-1 and AMP</LI></UL></DIV></SECTION><SECTION class="protocols"><H1>Plate reader</H1><DIV class="prot-inside"><H2>Materials</H2><P><UL><LI>CMM medium</LI><LI>YNB solution</LI><LI>YPD solution</LI><LI>Plate reader</LI><LI>Centrifuge</LI><LI>Falcon 50</LI><LI>24-well plate</LI></UL></P><H2>Procedure</H2><P><OL><LI>One colony of <I>P. pastoris</I> Odr10-RFP was grown in 50mL YPD medium for 48h at 30°C with shaking. Same for the negative (<I>P. pastoris</I> with empty pPICZα) and positive (<I>P. pastoris</I> with pGAP-RFP) controls.</LI><LI>Inoculation of 50mL CMM Ammonium sulfate and 50mL CMM Glutamine at OD 0.2 for each clone.</LI><LI>The next day, inoculation of 50mL CMM Ammonium sulfate and 50mL CMM Glutamine at OD 0.2 for each clone and culture over day from OD600 = 0.2 to 0.6</LI><LI>Split each culture in 2 in falcons 50 and centrifuge 3 min at 3000 rpm at RT.</LI><LI>Wash cells with 10 mL of CMM Glutamine or YNB (to start Glutamine depletion) for the cultures coming from CMM Glutamine medium and 10mL of CMM Ammonium sulfate or YNB (to start Ammonium sulfate depletion) for the cultures coming from CMM Ammonium Sulfate.</LI><LI>Centrifuge 3 min at 3000 rpm at RT.</LI><LI>Resuspend each pellet in 1mL of CMM Glutamine or YNB for the cultures coming from CMM Glutamine medium and 1mL of CMM Ammonium sulfate or YNB for the cultures coming from CMM Ammonium Sulfate.</LI><LI>Inoculation plate at OD 0.2 in 1mL of the corresponding medium.</LI><LI>Add diacetyl to 500µM and 1000µM for each condition.</LI></OL></P><H2> Plate reader set-ups</H2><P>
Cells were cultivated in a FLUOstar Optima plate reader (BMG Labtech, Offenburg, Germany) at 30°C and 600 rpm (orbital) with a shaking diameter of 1 mm. The shaking and measurement procedures were as follows: 
<UL><LI>shake duration, 300 s</LI><LI>ﬂuorescent measurement duration, 123 s per 48 wells, 20 ﬂashes; shaking duration, 60 s</LI><LI>absorbance measurement duration, 50 s per 48 wells, 20 ﬂashes</LI><LI>180 cycles, 24 h, ﬂash</LI>.
</UL></P><P> Biomass was determined by measurement of optical density (OD) at 600 nm. The experiments were carried out with standard round 48 or 24--well plates with ﬂat bottoms from Starlab, France (catalog number 1830048), covered with lids. If not otherwise speciﬁed, the experiments were conducted with a 500-�l working volume of medium.  RFR was excited with a wavelength of 544 nm and the emission was detected at 600nm. 
</P></DIV></SECTION><SECTION class="protocols"><H1>RT-qPCR</H1><DIV class="prot-inside"><H2> RNA extraction </H2><H3>Culture of P.pastoris SMD1168H</H3><UL><LI> Cells are centrifuged max speed and washed with sterilized water </LI><LI> Another centrifugation max speed and the supernatant is removed </LI><LI> Pellets are freeze at -80°C overnight </LI></UL><P><OL><LI>500 µl of GTC mix + 500 µl of Water/Phenol + 1 ml of ice-cold glass beads are added to each falcon </LI><LI>2 min of vortex full power and then chill on ice for 1 min, this step is repeated 2 more times </LI><LI>Add 7.5 ml of GTC mix and 7,5 ml of Water/phenol </LI><LI>1 min of vortex full speed and then 5 min of incubation at 65°C </LI><LI>Add 7.5 ml of CH3Cl + 4 ml of NaAc 100mM in TE </LI><LI>1 min of vortex full power </LI><LI>centrifugation 4000 rpm, 5 min at 4°C </LI><LI>12 ml of the aqueous phase is extracted and transferred into a new falcon containing 6 ml of Water/phenol + 6 ml of CH3Cl </LI><LI> centrifugation 4000 rpm, 5 min at 4°C </LI><LI>10 ml of the aqueous phase are extracted and transferred into a new falcon containing 5 ml of Water/phenol + 5 ml of CH3Cl </LI><LI>1 min of vortex full speed </LI><LI>The aqueous phase is collected (5 ml) and 3 volume of Ethanol are added (25ml)</LI><LI> centrifugation 4000rpm, 20 min at 4°C and the supernatant is thrown away</LI><LI>Add 5 ml of Ethanol 70%</LI><LI>centrifugation 4000rpm, 1 min at 4°C and the supernatant is thrown away (this step is repeated 2 more times)</LI><LI>ARNs are dried and diluted in 200 µl of water </LI><LI>Nanodrop quantification </LI></OL></P><H2> Reverse transcriptions</H2><P>
RT reactions using total RNAs extracted from Pichia pastoris cells transformed with plasmids pPIC-DNY15 or pPIC (empty vector).
          </P><P><UL><LI> Reactions using 1 µg of total RNAs (1.5 ml microtube):</LI><OL><LI>2 µl Primer DNY15-RT-r (1 µM = 1 pmol/µl)</LI><LI>1 µl Total RNAs (1 µg/µl)</LI><LI>1 µl dNTP mix (10 mM each)</LI><LI>8 µl MQ H2O </LI></OL><LI> Reactions using 5 µg of total RNAs:</LI><OL><LI>2 µl Primer DNY15-RT-r, 1 µM (= 1 pmol/µl)</LI><LI>5 µl Total RNAs (1 µg/µl)</LI><LI>1 µl dNTP mix (10 mM each)</LI><LI>4 µl MQ H2O </LI></OL><LI> Mix gently with vortex </LI><LI> Incubate 5 min at 65°C (water bath)</LI><LI> Chill immediately on ice and incubate for 5 min </LI><LI> Pulse spin at 4°C and add the following:</LI><UL><LI>4 µl 5X First-Strand Buffer </LI><LI>2 µl 100 mM DTT </LI><LI>1 µl RNasin (40 U./µl, Promega)</LI></UL><LI> Mix with vortex gently </LI><LI> Incubate 2 min at 42°C (Thermomixer Eppendorf)</LI><LI> Add 1 µl SuperScript II RT (Invitrogen, reference 18064-022)</LI><LI> Mix by pipeting gently up &amp; down </LI><LI> Incubate 1 hour at 42°C </LI><LI> Inactivate the reaction by heating at 70°C for 15 min (water bath)</LI></UL></P><H2> qPCR reactions </H2><UL><LI>2 µl RT mix</LI><LI>0.5 µl primer DNY15-RT-f (12 µM = 100 ng/µl)</LI><LI>0.5 µl primer DNY15-RT-r (12 µM = 100 ng/µl)</LI><LI>4.5 µl MQ H2O </LI><LI>7.5 µl IQ SYBR Green Supermix (Biorad, reference 1725006CUST)</LI><LI> PCR program (CFX96 Touch Real-Time PCR detection system)</LI><UL><LI>95°C 3 min </LI><LI>95°C 15 sec </LI><LI>40X 60°C 30 sec </LI><LI>72°C 30 sec </LI></UL><LI> Melting curve from 60°C to 95°C, 0.5°C/cycle, 5 sec 	</LI></UL></DIV></SECTION><SECTION class="protocols"><H1>On plate toxicity assay</H1><DIV class="prot-inside"><H2>Introduction</H2><P>
           To test the cytotoxicity of the AMPs, the supernatant of recombinant <I>P. pastoris</I> were tested by a Growth Inhibition Assay on freshly plated <I>V. harveyi</I> cultures. 
          </P><H2>Materials</H2><UL><LI>LB agar plate</LI><LI>50 mL culture flasks</LI><LI>YPD liquid medium</LI><LI>Paper discs (6 mm diameter)</LI><LI>Recombinant <I>P. pastoris</I> (producing AMPs and empty vector)</LI><LI>Chloramphenicol solution (25 μg/mL)</LI></UL><H2>Procedure</H2><P><OL><LI>Produce the AMPs with the AMP production protocol, a negative control is also needed containing only the empty cloning vector.</LI><LI>Grow wild type <I>V. harveyi</I> to OD 0.3, and then plate 200 μL on an LB plate.</LI><LI>Let it dry a few minutes. </LI><LI>Dip the paper discs in: the AMP containing supernatant, the negative supernatant and the chloramphenicol solution. And place them on the plate.</LI><LI>Grow overnight at 30°C.</LI></OL></P></DIV></SECTION><SECTION class="links_end"><TABLE><TBODY><TR><TH colspan="6">
            Realisations pages
          </TH></TR><TR><TD><A href="https://2017.igem.org/Team:INSA-UPS_France/Experiments">Overview</A></TD><TD><A href="https://2017.igem.org/Team:INSA-UPS_France/Notebook">Notebook</A></TD><TD><A href="https://2017.igem.org/Team:INSA-UPS_France/Experiments/Clonings">Clonings</A></TD><TD><A href="https://2017.igem.org/Team:INSA-UPS_France/Results">Results</A></TD><TD><A href="https://2017.igem.org/Team:INSA-UPS_France/Contribution">Contribution</A></TD><TD><I>Protocols</I></TD><TD><A href="https://2017.igem.org/Team:INSA-UPS_France/Safety">Safety</A></TD></TR></TBODY></TABLE></SECTION></DIV></DIV></DIV></MAIN></DIV></DIV></DIV></DIV></DIV></BODY></HTML>