<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_INSA-UPS_France_Notebook skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:INSA-UPS France/Notebook</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container" style="position:relative; "><DIV class="site-header__title">
    Croc'n Cholera <SPAN style="font-size:12pt;">iGEM UPS-INSA Toulouse 2017</SPAN></DIV><NAV class="site-menu" style="text-align: center;"><DIV class="site-menu__item"><DIV class="cat_name">Team </DIV><UL class="subnav" style="width:150%;"><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Team">Members</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Supports">Supports</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Attributions">Attributions</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Collaborations">Collaborations</A></LI></UL></DIV><DIV class="site-menu__item"><DIV class="cat_name">Strategy </DIV><UL class="subnav"><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Description">Description</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Design">Design</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Parts">Parts</A></LI></UL></DIV><DIV class="site-menu__item"><DIV class="cat_name">Model </DIV><UL class="subnav"><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Model">Overview</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Model/Simulation">Simulation</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Model/Analysis">Analysis</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Model/Interface">Interface</A></LI></UL></DIV><DIV class="site-menu__item"><DIV class="cat_name">Realisations </DIV><UL class="subnav"><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Experiments">Overview</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Notebook">Notebook</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Experiments/Clonings">Clonings</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Results">Results</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Contribution">Contribution</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Protocols">Protocols</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Safety">Safety</A></LI></UL></DIV><DIV class="site-menu__item"><DIV class="cat_name">Human Practices</DIV><UL class="subnav"><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Human_Practices">Overview</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/HP/Silver">Silver</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/HP/Gold_Integrated">Gold</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Engagement">Public Engagement</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Microbioworld">Microbioworld</A></LI></UL></DIV><DIV class="site-menu__item"><DIV class="cat_name">Application </DIV><UL class="subnav"><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Applied_Design">Applied Design</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Entrepreneurship/Testimonies">Testimonies</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Entrepreneurship">Entrepreneurship</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Entrepreneurship/Device">Device Conception</A></LI></UL></DIV></NAV></DIV><DIV class="main_content"><DIV class="middle_container"><DIV class="section_container"><SECTION style="display:table;background:none;padding:0px !important;z-index:100; "><H1 style="vertical-align:bottom;display:table-cell; width:50%;font-size:60pt;letter-spacing: 0.2em;z-index:120;text-align: center;">Notebook</H1></SECTION><SECTION style="margin-top:50px;"><P>
        Here is a summary of what we did for our project, week by week. 
        You can go on our <A href="https://2017.igem.org/Team:INSA-UPS_France/HP/Silver">Human Practices labbook</A> page to know more about how each event affected us to take a decision about the future of our project. 
      </P></SECTION><SECTION style="background:none;margin:0;"><DIV><DIV style="display:inline-block;">Brainstorming meeting</DIV></DIV><DIV><DIV style="display:inline-block;">Team day</DIV></DIV><DIV><DIV style="display:inline-block;">Meetup with other iGEM teams</DIV></DIV><DIV><DIV style="display:inline-block;">Public Engagement event</DIV></DIV><DIV><DIV style="display:inline-block;">Integrated Human Practices / Entrepreneurship major step</DIV></DIV><DIV><DIV style="display:inline-block;">Experiments</DIV></DIV></SECTION><DIV class="calendar"><DIV class="first-row"><DIV class="month_container"><H3>January</H3><TABLE><TBODY><TR data-target="1"><TD>1</TD><TD>2</TD><TD>3</TD><TD>4</TD><TD>5</TD><TD>6</TD><TD>7</TD></TR><TR data-target="2"><TD>8</TD><TD>9</TD><TD>10</TD><TD>11</TD><TD>12</TD><TD>13</TD><TD>14</TD></TR><TR data-target="3"><TD>15</TD><TD>16</TD><TD>17</TD><TD>18</TD><TD>19</TD><TD>20</TD><TD>21</TD></TR><TR data-target="4" class="current-week"><TD>22</TD><TD>23</TD><TD>24</TD><TD class="team_day">25</TD><TD>26</TD><TD>27</TD><TD>28</TD></TR><TR data-target="5"><TD>29</TD><TD class="meeting_day">30</TD><TD>31</TD><TD class="lastmonth">1</TD><TD class="lastmonth">2</TD><TD class="lastmonth">3</TD><TD class="lastmonth">4</TD></TR></TBODY></TABLE></DIV><DIV class="month_container"><H3>February</H3><TABLE><TBODY><TR data-target="5"><TD class="lastmonth">29</TD><TD class="lastmonth">30</TD><TD class="lastmonth">31</TD><TD>1</TD><TD>2</TD><TD>3</TD><TD>4</TD></TR><TR data-target="6"><TD>5</TD><TD class="meeting_day">6</TD><TD>7</TD><TD>8</TD><TD>9</TD><TD>10</TD><TD>11</TD></TR><TR data-target="7"><TD class="experiment_day">12</TD><TD class="experiment_day">13</TD><TD class="experiment_day">14</TD><TD class="experiment_day">15</TD><TD class="meeting_day experiment_day">16</TD><TD class="experiment_day">17</TD><TD class="experiment_day">18</TD></TR><TR data-target="8"><TD>19</TD><TD>20</TD><TD>21</TD><TD>22</TD><TD class="meeting_day">23</TD><TD>24</TD><TD>25</TD></TR><TR data-target="9"><TD>26</TD><TD>27</TD><TD>28</TD><TD class="lastmonth">1</TD><TD class="lastmonth">2</TD><TD class="lastmonth">3</TD><TD class="lastmonth">4</TD></TR></TBODY></TABLE></DIV><DIV class="month_container"><H3>March</H3><TABLE><TBODY><TR data-target="9"><TD class="lastmonth">26</TD><TD class="lastmonth">27</TD><TD class="lastmonth">28</TD><TD>1</TD><TD class="meeting_day">2</TD><TD>3</TD><TD>4</TD></TR><TR data-target="10"><TD class="team_day">5</TD><TD>6</TD><TD>7</TD><TD>8</TD><TD class="meeting_day">9</TD><TD>10</TD><TD>11</TD></TR><TR data-target="11"><TD class="team_day">12</TD><TD>13</TD><TD class="meeting_day">14</TD><TD>15</TD><TD>16</TD><TD>17</TD><TD>18</TD></TR><TR data-target="12"><TD>19</TD><TD>20</TD><TD class="meeting_day">21</TD><TD>22</TD><TD>23</TD><TD>24</TD><TD>25</TD></TR><TR data-target="13"><TD class="team_day">26</TD><TD>27</TD><TD>28</TD><TD>29</TD><TD class="meeting_day">30</TD><TD>31</TD><TD class="lastmonth">1</TD></TR></TBODY></TABLE></DIV><DIV class="month_container"><H3>April</H3><TABLE><TBODY><TR data-target="13"><TD class="lastmonth">26</TD><TD class="lastmonth">27</TD><TD class="lastmonth">28</TD><TD class="lastmonth">29</TD><TD class="lastmonth">30</TD><TD class="lastmonth">31</TD><TD>1</TD></TR><TR data-target="14"><TD>2</TD><TD>3</TD><TD>4</TD><TD>5</TD><TD class="meeting_day">6</TD><TD class="team_day">7</TD><TD>8</TD></TR><TR data-target="15"><TD class="experiment_day">9</TD><TD class="experiment_day">10</TD><TD class="experiment_day">11</TD><TD class="experiment_day">12</TD><TD class="meeting_day experiment_day">13</TD><TD class="experiment_day">14</TD><TD class="experiment_day">15</TD></TR><TR data-target="16"><TD>16</TD><TD>17</TD><TD>18</TD><TD class="ihp_day">19</TD><TD class="meeting_day">20</TD><TD>21</TD><TD>22</TD></TR><TR data-target="17"><TD class="team_day" s="">23</TD><TD>24</TD><TD class="ihp_day">25</TD><TD class="meeting_day">26</TD><TD class="team_day">27</TD><TD>28</TD><TD>29</TD></TR><TR data-target="18"><TD>30</TD><TD class="lastmonth">1</TD><TD class="lastmonth">2</TD><TD class="lastmonth">3</TD><TD class="lastmonth">4</TD><TD class="lastmonth">5</TD><TD class="lastmonth">6</TD></TR></TBODY></TABLE></DIV><DIV class="month_container"><H3>May</H3><TABLE><TBODY><TR data-target="18"><TD class="lastmonth">30</TD><TD>1</TD><TD class="meeting_day">2</TD><TD>3</TD><TD>4</TD><TD class="pe_day">5</TD><TD>6</TD></TR><TR data-target="19"><TD>7</TD><TD>8</TD><TD>9</TD><TD>10</TD><TD class="meeting_day">11</TD><TD>12</TD><TD class="team_day">13</TD></TR><TR data-target="20"><TD>14</TD><TD>15</TD><TD>16</TD><TD class="meeting_day">17</TD><TD class="experiment_day">18</TD><TD>19</TD><TD>20</TD></TR><TR data-target="21"><TD class="team_day">21</TD><TD>22</TD><TD>23</TD><TD>24</TD><TD class="team_day">25</TD><TD>26</TD><TD>27</TD></TR><TR data-target="22"><TD class="team_day experiment_day">28</TD><TD class="experiment_day">29</TD><TD class="pe-event_day experiment_day">30</TD><TD class="pe-event_day experiment_day">31</TD><TD class="lastmonth">1</TD><TD class="lastmonth">2</TD><TD class="lastmonth">3</TD></TR></TBODY></TABLE></DIV></DIV><DIV class="second-row"><DIV class="month_container"><H3>June</H3><TABLE><TBODY><TR data-target="22"><TD class="lastmonth">28</TD><TD class="lastmonth">29</TD><TD class="lastmonth">30</TD><TD class="lastmonth">31</TD><TD class="pe-event_day experiment_day">1</TD><TD class="experiment_day">2</TD><TD class="experiment_day">3</TD></TR><TR data-target="23"><TD class="experiment_day">4</TD><TD class="experiment_day">5</TD><TD class="ihp_day experiment_day">6</TD><TD class="experiment_day">7</TD><TD class="meeting_day experiment_day">8</TD><TD class="pe-event_day experiment_day">9</TD><TD class="experiment_day">10</TD></TR><TR data-target="24"><TD class="experiment_day">11</TD><TD class="experiment_day">12</TD><TD class="experiment_day">13</TD><TD class="ihp_day experiment_day">14</TD><TD class="meeting_day experiment_day">15</TD><TD class="igemers_day experiment_day">16</TD><TD class="experiment_day">17</TD></TR><TR data-target="25"><TD class="experiment_day">18</TD><TD class="ihp_day experiment_day">19</TD><TD class="ihp_day experiment_day">20</TD><TD class="experiment_day">21</TD><TD class="igemers_day experiment_day">22</TD><TD class="meeting_day experiment_day">23</TD><TD class="experiment_day">24</TD></TR><TR data-target="26"><TD class="experiment_day">25</TD><TD class="igemers_day experiment_day">26</TD><TD class="experiment_day">27</TD><TD class="ihp_day experiment_day">28</TD><TD class="meeting_day experiment_day">29</TD><TD class="ihp_day experiment_day">30</TD><TD class="lastmonth">1</TD></TR></TBODY></TABLE></DIV><DIV class="month_container"><H3>July</H3><TABLE><TBODY><TR data-target="26"><TD class="lastmonth">25</TD><TD class="lastmonth">26</TD><TD class="lastmonth">27</TD><TD class="lastmonth">28</TD><TD class="lastmonth">29</TD><TD class="lastmonth">30</TD><TD class="experiment_day">1</TD></TR><TR data-target="27"><TD class="experiment_day">2</TD><TD class="experiment_day">3</TD><TD class="experiment_day">4</TD><TD class="experiment_day">5</TD><TD class="meeting_day experiment_day">6</TD><TD class="igemers_day experiment_day">7</TD><TD class="experiment_day">8</TD></TR><TR data-target="28"><TD class="experiment_day">9</TD><TD class="experiment_day">10</TD><TD class="experiment_day">11</TD><TD class="experiment_day">12</TD><TD class="meeting_day">13</TD><TD class="ihp_day experiment_day">14</TD><TD class="experiment_day">15</TD></TR><TR data-target="29"><TD class="experiment_day">16</TD><TD class="experiment_day">17</TD><TD class="experiment_day">18</TD><TD class="experiment_day">19</TD><TD class="meeting_day experiment_day">20</TD><TD class="igemers_day experiment_day">21</TD><TD class="experiment_day">22</TD></TR><TR data-target="30"><TD class="igemers_day experiment_day">23</TD><TD class="experiment_day">24</TD><TD class="experiment_day">25</TD><TD class="experiment_day">26</TD><TD class="ihp_day experiment_day">27</TD><TD class="experiment_day">28</TD><TD class="team_day experiment_day">29</TD></TR><TR data-target="31"><TD class="team_day experiment_day">30</TD><TD class="experiment_day">31</TD><TD class="lastmonth">1</TD><TD class="lastmonth">2</TD><TD class="lastmonth">3</TD><TD class="lastmonth">4</TD><TD class="lastmonth">5</TD></TR></TBODY></TABLE></DIV><DIV class="month_container"><H3>August</H3><TABLE><TBODY><TR data-target="31"><TD class="lastmonth">30</TD><TD class="lastmonth">31</TD><TD class="meeting_day experiment_day">1</TD><TD class="experiment_day">2</TD><TD class="experiment_day">3</TD><TD class="experiment_day">4</TD><TD class="experiment_day">5</TD></TR><TR data-target="32"><TD class="experiment_day">6</TD><TD class="igemers_day experiment_day">7</TD><TD class="experiment_day">8</TD><TD class="experiment_day">9</TD><TD class="experiment_day">10</TD><TD class="experiment_day">11</TD><TD class="experiment_day">12</TD></TR><TR data-target="33"><TD class="experiment_day">13</TD><TD class="experiment_day">14</TD><TD class="experiment_day">15</TD><TD class="igemers_day experiment_day">16</TD><TD class="team_day experiment_day">17</TD><TD class="experiment_day">18</TD><TD class="experiment_day">19</TD></TR><TR data-target="34"><TD class="experiment_day">20</TD><TD class="experiment_day">21</TD><TD class="experiment_day">22</TD><TD class="experiment_day">23</TD><TD class="meeting_day experiment_day">24</TD><TD class="experiment_day">25</TD><TD class="experiment_day">26</TD></TR><TR data-target="35"><TD class="experiment_day">27</TD><TD class="experiment_day">28</TD><TD class="experiment_day">29</TD><TD class="experiment_day">30</TD><TD class="meeting_day">31</TD><TD class="lastmonth">1</TD><TD class="lastmonth">2</TD></TR></TBODY></TABLE></DIV><DIV class="month_container"><H3>September</H3><TABLE><TBODY><TR data-target="35"><TD class="lastmonth">27</TD><TD class="lastmonth">28</TD><TD class="lastmonth">29</TD><TD class="lastmonth">30</TD><TD class="lastmonth">31</TD><TD class="experiment_day">1</TD><TD class="experiment_day">2</TD></TR><TR data-target="36"><TD class="experiment_day">3</TD><TD class="pe-event_day">4</TD><TD class="experiment_day">5</TD><TD class="experiment_day">6</TD><TD class="meeting_day experiment_day">7</TD><TD class="team_day experiment_day">8</TD><TD class="ihp_day experiment_day">9</TD></TR><TR data-target="37"><TD class="experiment_day">10</TD><TD class="experiment_day">11</TD><TD class="experiment_day">12</TD><TD class="experiment_day">13</TD><TD class="meeting_day experiment_day">14</TD><TD class="experiment_day">15</TD><TD class="experiment_day">16</TD></TR><TR data-target="38"><TD class="experiment_day">17</TD><TD class="experiment_day">18</TD><TD class="experiment_day">19</TD><TD class="experiment_day">20</TD><TD class="meeting_day experiment_day">21</TD><TD class="ihp_day experiment_day">22</TD><TD class="experiment_day">23</TD></TR><TR data-target="39"><TD class="experiment_day">24</TD><TD class="experiment_day">25</TD><TD class="experiment_day">26</TD><TD class="meeting_day experiment_day">27</TD><TD class="experiment_day">28</TD><TD class="pe-event_day experiment_day">29</TD><TD class="experiment_day">30</TD></TR></TBODY></TABLE></DIV><DIV class="month_container"><H3>October</H3><TABLE><TBODY><TR data-target="40"><TD>1</TD><TD>2</TD><TD>3</TD><TD class="pe-event_day">4</TD><TD class="meeting_day">5</TD><TD>6</TD><TD>7</TD></TR><TR data-target="41"><TD>8</TD><TD>9</TD><TD>10</TD><TD>11</TD><TD>12</TD><TD>13</TD><TD>14</TD></TR><TR data-target="42"><TD>15</TD><TD>16</TD><TD>17</TD><TD>18</TD><TD>19</TD><TD>20</TD><TD>21</TD></TR><TR data-target="43"><TD>22</TD><TD>23</TD><TD>24</TD><TD>25</TD><TD>26</TD><TD>27</TD><TD>28</TD></TR><TR data-target="44"><TD>29</TD><TD>30</TD><TD>31</TD><TD class="lastmonth">1</TD><TD class="lastmonth">2</TD><TD class="lastmonth">3</TD><TD class="lastmonth">4</TD></TR></TBODY></TABLE></DIV></DIV></DIV><DIV class="week" id="week-1"><SECTION><H1>Week 1 </H1></SECTION><SECTION><P>
            Our team was being recruited. Nothing happened this week!
          </P></SECTION></DIV><DIV class="week" id="week-2"><SECTION><H1> Week 2 </H1></SECTION><SECTION><P>
          Our team was being recruited. Nothing happened this week!
        </P></SECTION></DIV><DIV class="week" id="week-3"><SECTION><H1> Week 3 </H1></SECTION><SECTION><P>
            Our team was being recruited. Nothing happened this week!
          </P></SECTION></DIV><DIV class="week visible-week" id="week-4"><SECTION><H1> Week 4 </H1></SECTION><SECTION class="week-team"><H2>01-25: Kick Off Meeting</H2><P>
          Gathering the team members and distribute the tasks... The adventure begins!
        </P></SECTION></DIV><DIV class="week" id="week-5"><SECTION><H1> Week 5 </H1></SECTION><SECTION class="week-meeting"><H2>
          01-30: 2th brainstorming: on the track of crocodile antimicrobial peptides... 
        </H2><P>
          We’ve took charge of differents accounts of the association (facebook, twitter, email, bank account, google drive…), and made a listing of 45 potential subjects for the competition.
        </P></SECTION></DIV><DIV class="week" id="week-6"><SECTION><H1> Week 6 </H1></SECTION><SECTION class="week-meeting"><H2>02-06: 3th brainstorming meeting</H2><P>
          We’ve selected 17 subjects that we considered the most originals, the most feasible, and the most interesting. We’ve imagined to make a dressing against nosocomial infections containing bacteria that produces antimicrobial peptides from crocodile. We’ve read that crocodile peptides are not toxic for humans at microbicide quantity.
        </P></SECTION></DIV><DIV class="week" id="week-7"><SECTION><H1> Week 7 </H1></SECTION><SECTION class="week-meeting"><H2>02-16: 4th brainstorming</H2><P>
          Only 7 subjects left! To make a safe device, we’ve thought of switch on the dressing by using spores that can be activated with the temperature. 
        </P></SECTION><SECTION class="week-experiment"><H2>02-13 to 02-17: Training course for INSA students at LISBP with our instructor Brice Enjalbert</H2></SECTION></DIV><DIV class="week" id="week-8"><SECTION><H1> Week 8 </H1></SECTION><SECTION class="week-meeting"><H2>02-23: 5th brainstorming</H2><P>
          We made a list of our public engagement events. For our subject, we’ve focused our research on the originality, and we’ve noted that a similar dressing has already been made by an iGEM team before. So we imagined the use of anticoagulant molecules in the dressing. We’ve thought of a bandage for pets.
        </P></SECTION></DIV><DIV class="week" id="week-9"><SECTION><H1> Week 9 </H1></SECTION><SECTION class="week-meeting"><H2>03-02: 6th brainstorming</H2><P>
          We’ve analysed the 7 subjects, and we’ve noted some technical issues. For the “crocodile peptides project”, we’ve raised several questions : what organisme should we use to produce antimicrobial peptides? We need to find an organism which is not susceptible to their antimicrobial activity. We need to use a special membrane to contain GMOs inside the dressing and not directly in contact to the skin.
        </P></SECTION></DIV><DIV class="week" id="week-10"><SECTION><H1> Week 10 </H1></SECTION><SECTION class="week-team"><H2>03-05: Team building at the restaurant</H2><P>
          Our team met for the first time outside the INSA buildings and shared a delicious korean meal after brainstorming the whole afternoon!
        </P></SECTION><SECTION class="week-meeting"><H2>03-09: 7th brainstorming meeting</H2><P>
           We’ve voted for our 4 favourite subjects : “crocodile antimicrobial peptides”, “bioluminescent living board”, “expiration pastille”, and “colourful cellular cycle”.
        </P></SECTION></DIV><DIV class="week" id="week-11"><SECTION><H1> Week 11 </H1></SECTION><SECTION class="week-team"><H2>03-12: Brainstorming in a café</H2></SECTION><SECTION class="week-meeting"><H2>03-14: 8th brainstorming</H2><P>
          The problem of originality persists for the application: what about a suture thread? For the peptide-producer microorganism, a yeast could be a good solution because the antimicrobial activity of our peptide is bacterial-specific. For the activation of the peptide production, we’ve thought about lyophilisating the engineered yeast. 
        </P></SECTION></DIV><DIV class="week" id="week-12"><SECTION><H1> Week 12 </H1></SECTION><SECTION class="week-meeting"><H2>03-21: 9th brainstorming</H2><P>
          In view of what has already been done in iGEM for  detection of quorum sensing molecules, it seems that we’ll need a bacteria to detect other quorum sensing molecules because it’s difficult to express a prokaryotic membrane-receptor into a yeast. So we’ve imagined a system with a prokaryote that detect a quorum sensing signal from a pathogenic bacteria, and that triggers peptide production by a yeast.
        </P></SECTION></DIV><DIV class="week" id="week-13"><SECTION><H1> Week 13 </H1></SECTION><SECTION class="week-team"><H2>03-26: Brainstorming in a café</H2></SECTION><SECTION class="week-meeting"><H2>03-30: 10th brainstorming</H2><P>
          A second device is born: a shaker containing our too organisms (prokaryote-detector and yeast-peptide-producer) to detect and kill <I>Vibrio cholerae</I> in the water. We had a idea for our public engagement project: creating a pedagogic card game about microbiology. <I>MicrobioWorld</I> conception is in process... 
        </P></SECTION></DIV><DIV class="week" id="week-14"><SECTION><H1> Week 14 </H1></SECTION><SECTION class="week-meeting"><H2>04-06: 11th brainstorming</H2><P>
          Dropping of the first device because ALS detection of one single Staphylococcus aureus is going to be difficult. In contrary, <I>Vibrio cholerae</I> produces a high quantity of CAI-1, a specific quorum sensing molecule, at pathogenic quantity. 
        </P></SECTION><SECTION class="week-team"><H2>04-07: choice of the subject</H2><P>
          We’ve finally choose to work on cholera water treatment using a prokaryotic-eukaryotic communication.
        </P></SECTION></DIV><DIV class="week" id="week-15"><SECTION><H1> Week 15 </H1></SECTION><SECTION class="week-meeting"><H2>04-13: 12th brainstorming</H2><P>
          Actual solution to fight against cholera are not sufficient to prevent great epidemics, we need to contact NGOs to benefit from their experience of the field of a cholera epidemy. For the proof of concept, we need to find an effective cholera detector, and a good peptide producer. The detector bacteria could be <I>E. coli</I>, or maybe <I>V. harveyi</I>, that already possess the quorum sensing receptor of CAI-1, and all the proteins to do the phosphorylation cascade in response of CAI-1.
        </P></SECTION><SECTION class="experiment-meeting"><H2>04-10 to 04-14 Training course for the UPS students at LBME with our instructors Anthony Henras and Yves Romeo</H2></SECTION></DIV><DIV class="week" id="week-16"><SECTION><H1> Week 16 </H1></SECTION><SECTION class="week-ihp"><H2>04-19: Meeting with Marc Lemonnier, CEO of Antabio</H2><P>
          We met Marc Lemonnier, founding CEO of ANTABIO: a start-up that develops novel therapies to treat drug-resistant infections by the most critical Gram-negative pathogens. This was our first meeting within the Entrepreneurship part of the project. The aim was to learn about the approach to follow to start a business and to have a first external opinion regarding the integration of our biological system into the market. You can read the result of <A href="https://2017.igem.org/Team:INSA-UPS_France/Entrepreneurship/Testimonies#a1">his testimony here.</A></P></SECTION><SECTION class="week-meeting"><H2>04-20: 13th brainstorming</H2><P>
          We’ve found a company called “Sunwaterlife” that work on water treatment by filtrating water in Toulouse. We’ve planned to meet them to take advices from their experience, and ask if they could help us to build our device to contain GMMs into a membrane to avoid their dispersion into the environment. For the peptide-producer yeast, we’ve selected <I>Pichia pastoris</I>, famous for it’s capacity of fast production of proteins. To make a communication between our two organisms, we’ve thought of using diacetyl: our detector organism produces diacetyl in response of CAI-1 detection, that triggers AMP by <I>P. pastoris</I>.
        </P></SECTION></DIV><DIV class="week" id="week-17"><SECTION><H1> Week 17 </H1></SECTION><SECTION class="week-team"><H2>04-23: First drafts of Sobki, our mascot!</H2></SECTION><SECTION class="week-ihp"><H1>04-25:</H1><P>
          Following our appointment with Marc Lemonnier, we began to think about a device so that our system could treat the water in a tank for villagers or the water of a flask for the tourists. The challenge is to confine the microorganisms so that they do not go into the water.
        </P></SECTION><SECTION class="week-meeting"><H2>04-26: 14th brainstorming</H2><P>
          For safety reasons we can’t use <I>V. cholerae</I> for our proof of concept, and we can’t use any gene from that bacteria. As <I>V. harveyi</I> is genetically close to <I>V. cholerae</I> but not pathogenic for human, we can use it’s quorum sensing molecule, the C8-CAI-1 that looks very much as the CAI-1, to mimic <I>V. cholerae</I>. Here, the idea is to engineer a bacteria to be a “cholerae-like bacteria” that produces a lot quorum sensing molecules to be detected by our detector <I>V. harveyi</I>.
        </P></SECTION><SECTION class="week-team"><H2>04-27:</H2><P>
          Téo and Margaux went to a conference about <I>V. cholerae</I> to learn more about genetics of that bacteria, and the main differences between <I>V. cholerae</I> and <I>V. harveyi</I>.
        </P></SECTION></DIV><DIV class="week" id="week-18"><SECTION><H1> Week 18 </H1></SECTION><SECTION class="week-meeting"><H2>05-02: 15th brainstorming</H2><P>
          We need to evaluate the volume of water to treat. In the case of our device contain 1L water, it’s for only 1 person but we can imagine to treat a high volume for an entire village. We’ve prepared a list of questions to ask to NGOs to make a user friendly device.
        </P></SECTION><SECTION class="week-pe"><H2>05-07:</H2><P>
          We’ve met 2 game designers and graphists students, Vincent and Julien, to help us in the conception of our card game about microbiology! They will make the graphism designs of our card game.
        </P></SECTION></DIV><DIV class="week" id="week-19"><SECTION><H1> Week 19 </H1></SECTION><SECTION class="week-meeting"><H2>05-11: 16th brainstorming</H2><P>
          For the detection of diacetyl, we found the receptor Odr10, that activate the promoter pFUS for peptide production. For diacetyl production, the gene ALS is known to enhance diacetyl production, so we’ll need to express ALS gene in <I>V. harveyi</I> only when C8-CAI-1 is detected. This is complicated because the receptor of C8-CAI-1 induces a repression of the promoter pqrr4. We need to find a inhibitor regulation in our system.
        </P></SECTION><SECTION class="week-team"><H2>05-13: Paul &amp; Margaux officialy joined our team!</H2></SECTION></DIV><DIV class="week" id="week-20"><SECTION><H1> Week 20 </H1></SECTION><SECTION class="week-meeting"><H2>05-17: 17th brainstorming</H2><P>
           We planed to do NMR analysis to detect C8-CAI-1 and diacetyl for the proof of concept. We’ve asked to Bassler’s lab to give us 2 <I>V. harveyi</I> strains: a WT one, and the JMH626 strain deleted from the gene CqsA, which can’t produce C8-CAI-1 by itself. We’ll need that strain to detect C8-CAI-1 from our engineered “<I>E. coli</I> cholera-like”. Purified diacetyl has been commanded to do induction experiments on <I>P. pastoris</I>. We find the Tet on/ Tet off repression system for diacetyl production in response to C8-CAI-1.
        </P></SECTION><SECTION class="week-experiment"><H2>05-18: iGEM distribution kit received</H2></SECTION><SECTION class="week-team"><H2>05-21 Sobki is born!</H2></SECTION></DIV><DIV class="week" id="week-21"><SECTION><H1> Week 21 </H1></SECTION><SECTION class="week-ihp"><P>
          We made a survey about cholera disease and about our project to know if people are aware of cholera epidemics in Africa and the existing solutions, and to know if people would be ready to use a device containing GMMs to treat the water they drink.
        </P></SECTION><SECTION class="week-team"><H2>05-25: Parts design day at Brice's</H2></SECTION></DIV><DIV class="week" id="week-22"><SECTION><H1> Week 22 </H1></SECTION><SECTION class="week-team"><H2>05-28: Official announcement of our subject</H2><P>
          We officialy the subject of our project on social media!
        </P></SECTION><SECTION class="week-pe"><H2>05-30 and 31: Exposciences</H2><P>
          We took part to “Exposciences” which is a scientific festival. During this event we made children extract banana DNA thanks to simple ingredients that they can find in their kitchen. We also discussed with them about microorganisms thanks to paper fortune teller and this support led them to be curious on the microscopic world around them.
        </P><H2>06-01: Interventions in schools</H2><P>
          The main goals of these interventions were the discovery of biology and research at school with two workshops: Microorganisms and their environment and growth of microorganisms on a Petri dish.
        </P></SECTION><SECTION class="week-experiment"><H2>Experiments</H2><DIV class="wet-lab-notebook"><DIV><DIV><H2>Design and orders</H2><UL><LI>Design of the gBlocks and launching of the synthesis order for IDT</LI><LI>Design of the PCR oligos</LI></UL></DIV><DIV><H2>Design and orders</H2><UL><LI>Design of the gBlocks and launching of the synthesis order for IDT</LI><LI>Design of the PCR oligos</LI></UL></DIV></DIV></DIV></SECTION></DIV><DIV class="week" id="week-23"><SECTION><H1> Week 23 </H1></SECTION><SECTION class="week-ihp"><H2>06-06: Meeting with the CEO of Sunwaterlife</H2><P>
          We presented our project to Christophe Campéri-Ginestet who seems to be interested, especially by the detection function of <I>V. cholerae</I>. He would like to adapt it to the Sunwaterlife filters in order to have a direct colored response according to the water contamination. In addition, he would be willing to invest in our project in order to apply for JEI status (or “Young Innovative Enterprises” in english), a very interesting status for start-ups allowing them to have financial advantages. Besides, a collaboration is discussed so that Sunwaterlife provide us with their membranes, allowing us to confine our microorganisms within a device. He told us that he might be able to put us in touch with a UNICEF person in charge of cholera epidemics.
        </P><P>
          You can read the result of <A href="https://2017.igem.org/Team:INSA-UPS_France/Entrepreneurship/Testimonies#a4">his testimony here.</A></P></SECTION><SECTION class="week-meeting"><H2>06-08: 18th brainstorming</H2><P>
          We’ve divided the subject in 3 modules. The “Quorum” module, with <I>E. coli</I> engineered to produce C8-CAI-1; the “Detection” module, with V. harveyi engineered to detect C8-CAI-1 and produce diacetyl in response; and the “Killing” module with <I>P. pastoris</I> engineered to detect diacetyl and produce crocodile AMP in response. 
        </P></SECTION><SECTION class="week-pe"><H2>06-09: Intervention in school</H2><P>
          The main goals of these interventions were the discovery of biology and research at school with two workshops: Microorganisms and their environment and growth of microorganisms on a Petri dish.
        </P></SECTION><SECTION class="week-experiment"><H2>Experiments</H2><DIV class="wet-lab-notebook"><DIV><DIV><H2>Taking care of <I>V. harveyi</I></H2><UL><LI>Reception and storage of <I>V. harveyi</I> BB120 (WT) and <I>V. harveyi</I> JMH626 (ΔcqsA ΔluxQ ΔluxN) CmR at -80 °C.</LI><LI>Investigation of <I>V. harveyi</I> antibiotics resistances.</LI></UL></DIV></DIV></DIV></SECTION></DIV><DIV class="week" id="week-24"><SECTION><H1> Week 24 </H1></SECTION><SECTION class="week-experiment"><H2>06-13: Model</H2><P>
          A first version of the SBGN (Systems Biology Graphical Notation) summarizing our synthetic system have been drawn.
        </P></SECTION><SECTION class="week-ihp"><H2>06-14: Scope statement start</H2><P>
          A scope statement was created for a device that treats the water of a tank and another for a device that treats the water in a flask.
        </P></SECTION><SECTION class="week-meeting"><H2>06-15: 19th brainstorming</H2><P>
          We’ve received our IDT parts! We finally choose the name of our project: Croc’n cholera.
        </P></SECTION><SECTION class="week-igemers"><H2>06-16: Skype with Greece iGEM team</H2></SECTION><SECTION class="week-experiment"><H2>Experiments</H2><DIV class="wet-lab-notebook"><DIV><DIV><H2>Amplifying pSB1C3</H2><UL><LI>Reception of IDT gBlocks and storage</LI><LI>Preparative work with the cloning vectors : Amplification and stock of pSB1C3</LI></UL></DIV><DIV><H2>Taking charge of the lab and of <I>V. harveyi</I></H2><UL><LI>Growth kinetic assay in order to determine the adequate NaCl concentration for <I>V. harveyi</I> growth. </LI><LI>Reception of IDT gBlocks and storage</LI><LI>Transformation of DH5α competent cells with pYFP, pDsRed, pPIZα, pBR322 and amplification of the plasmids. The plasmids are double digested with the appropriate restriction enzymes.</LI></UL></DIV><DIV><H2>Cloning</H2><UL><LI>pPICzα, jmp62-DsRed (pDsRed) and jmp62-YFP (pYFP) were transformed into E.coli DHAα in order to amplify them.  </LI><LI>Bacteria grown were then put into liquid culture to do miniprep.</LI><LI>Digestion BamHI/EcoRI to check pPICzα gel migration profile
Digestion XbaI/NheI to check jmp62-DsRed gel migration profile
Digestion BamHI/NheI to check jmp62-YFP gel migration profile</LI></UL></DIV></DIV></DIV></SECTION></DIV><DIV class="week" id="week-25"><SECTION><H1> Week 25 </H1></SECTION><SECTION class="week-ihp"><H2>06-19: Meeting with Doctors Without Borders, regional manager</H2><P>
          She explained that the detection of cholera in the patient is simple because the symptoms are known. She told us that our system would be more useful treating water tanks in villages far from cholera camps rather than tourists. Indeed, governments often hide epidemics from tourists and tourist places take enough precautions to avoid tourists’ contamination. 
        </P><P>
          You can read the result of <A href="https://2017.igem.org/Team:INSA-UPS_France/Entrepreneurship/Testimonies#a5">her testimony here.</A></P><H2>06-23: Beginning of the marketing approach</H2><P>
          A marketing approach has been initiated to study the strengths and weaknesses of our two potential products and to determine the approach to sell them. Thanks to a SWOT analysis on both products, we realized that the product to treat water in a flask is not adapted to the market. This product is therefore abandoned and we focus on a product that processes a larger volume of water, suitable for villagers in the affected countries.
        </P></SECTION><SECTION class="week-igemers"><H2>06-22 Skype with NUS iGEM team</H2></SECTION><SECTION class="week-meeting"><H2>06-23: 20th brainstorming</H2><P>
          Meeting with all the team members and supervisors and final choice of the global device purpose: made for people living in remote areas in Africa, which are far from NGOs camps and more vulnerable to a cholera epidemy.
        </P></SECTION><SECTION class="week-experiment"><H2>06-23: Model</H2><P>
          Our mathematical model based on twelve ODEs was built!
        </P></SECTION><SECTION class="week-team"><H2>06-23: First appearance of Sobki, our mascott!</H2></SECTION><SECTION class="week-experiment"><H2>Experiments</H2><DIV class="wet-lab-notebook"><DIV><DIV><H2>Plasmids preparation</H2><UL><LI>Transformation of DH5α competent cells with pSB1C3 and amplification of the plasmids. The plasmids are double digested with the appropriate restriction enzymes.</LI><LI>Scale up of the preparative digestion, in order to have more DNA matrix.</LI><LI>Reception of the PCR primer.</LI></UL></DIV><DIV><H2>Plasmids preparation</H2><UL><LI>Transformation of DH5α competent cells with pGAP, pSB1C3 and amplification of the plasmids. The plasmids are double digested with the appropriate restriction enzymes.</LI><LI>Scale up of the preparative digestion, in order to have more DNA matrix.</LI><LI>Reception of the PCR primer.</LI></UL></DIV><DIV><H2>Cloning</H2><UL><LI>Preparative digestion of pPICZα with BamHI/BglII to clone our gene followed by a gel extraction to purify the open plasmid.</LI><LI>Glycerol stocks of <I>E.coli</I> DHAα containing pPICZα, pDSRed and pYFP were made after liquid culture. </LI></UL></DIV></DIV></DIV></SECTION></DIV><DIV class="week" id="week-26"><SECTION><H1> Week 26 </H1></SECTION><SECTION class="week-igemers"><H2>06-26: Skype with the iGEM team of Purdue</H2></SECTION><SECTION class="week-ihp"><H2>06-28: Meeting with the CEO of Sunwaterlife, building a collaboration</H2><P>
          Really interested in the ability to detect <I>V. cholerae</I> in water, Christophe Campéri-Ginestet asked us to provide our bibliographic research and results at the end of the iGEM project. In addition, in exchange for our financial investment in our project, we have to fill out the JEI document which attests to our R&amp;D activity for Sunwaterlife. He also put us in touch with Alama Keita, cholera leader of UNICEF in Niger. Finally, we talked to him about our product to contain our microorganisms. He helped us to design a cylindrical device whose sections would be these membranes to ensure GMMs containment.
        </P><P>
          You can read the result of <A href="https://2017.igem.org/Team:INSA-UPS_France/Entrepreneurship/Testimonies#a4">his testimony here.</A></P><H2>06-30: First steps of our ethical matrix</H2></SECTION><SECTION class="week-meeting"><H2>06-29: 21th brainstorming</H2><P>
          We choose to made a Prezi for the presentation in Boston, the structure has been validated by instructors and students, and we made a first version of our poster for the meetup in Delft. Our wiki is online next day! Our survey has around 400 answers, so we can start analyses of results.
        </P></SECTION><SECTION class="week-experiment"><H2>Experiments</H2><DIV class="wet-lab-notebook"><DIV><DIV><H2>Cloning VhCqsA</H2><UL><LI>Amplification of the IDT gblocks and optimisation of the PCR cycle condition for VhCqsA part</LI><LI>Digestion of the PCR amplicons and ligation with the appropriate cloning vector. </LI></UL></DIV><DIV><H2>Assembly of <I>V. harveyi</I> gene circuit : starting the sub-cloning work</H2><UL><LI>Amplification of the IDT gblocks and optimisation of the PCR cycle condition for VhCqsA part</LI><LI>Digestion of the PCR amplicons and ligation with the appropriate cloning vector. </LI><LI>Cloning attempt #1 and #2 to sub-clone  Vh1, Vh2 and  Vh3 in their respective vector.  </LI></UL></DIV><DIV><H2>Cloning</H2><UL><LI>PCR of DsRed and YFP with primers containing restriction sites. PCR products for these genes have right size. PCR products are purified.</LI><LI>PCR of our parts (cOT2 ; leucrocine ; DNY15; Odr10) from IDT GBlocks. All PCRs succeeded.</LI><LI>cOT2 ; leucrocine ; DNY15 ; Odr10 were digested BamHI/BglII to be inserted in pPICZα.</LI><LI>Transformation of pPICZα-cOT2 ; pPICZα-leucrocine ; pPICZα-Odr10 and pPICZα-DNY15 in <I>E. coli</I> DH5α</LI><LI>Miniprep of liquid culture transformants</LI><LI>Digestion BmaHI/BglII to check the gel migration profile. → It appears that the plasmid was well digested but the insert does not appear. There is an intense band around 1300bp that might correspond to supercoiled plasmid. To conclude it seems like pPICZα closed itself without the insert: BamHI and BglII are compatibles.
                </LI><LI>New primers were ordered replace by PCR the BglII site by KpnI site to remove the possibility for the plasmid to close itself.</LI></UL></DIV></DIV></DIV></SECTION></DIV><DIV class="week" id="week-27"><SECTION><H1> Week 27 </H1></SECTION><SECTION class="week-meeting"><H2>07-06: 22th brainstorming</H2><P>
          We’ve prepared some public engagement events for september : an exhibition at the campus science in Toulouse to showcase ancient iGEM Toulouse projects, and synthetic biology, a practical work in an high school, and we’ve planned to participate to the “European Researcher’s Night” with the LISBP by creating a game called “Possible or Impossible”, to open a dialogue about GMO legislation with people.
        </P></SECTION><SECTION class="week-igemers"><H2>07-07: Delft European iGEM Meetup</H2></SECTION><SECTION class="week-experiment"><H2>Experiments</H2><DIV class="wet-lab-notebook"><DIV><DIV><H2>Cloning VhCqsA</H2><UL><LI>Optimization of PCR cycle </LI><LI>Cloning of VhCqsA into pSB1C3 : positives clones grew </LI></UL></DIV><DIV><H2>Assembly of <I>V. harveyi</I> gene circuit : continuing the sub-cloning work</H2><UL><LI>Assembly of V.harveyi gene circuit: Analysis of the transformants of V.harveyi module (attempt #1 and #2) by checking of the restriction map. </LI><LI>New attempt (attempt #3) to sub-clone Vh1, Vh2 and  Vh3 in their respective vector.  </LI></UL></DIV><DIV><H2>Cloning</H2><UL><LI>While waiting for the new primers, another try was made using for the ligation a 1:10 ratio (pPICZα:insert).</LI><LI>Transformants appeared in all plates except for Odr10.</LI><LI>Miniprep and digestions of transformants to check the gel migration profile.</LI><LI>We obtained 2 clones pPICZα-DNY15. The gel confirmed that D1 and D2 are clones with the entire plasmid pPICZa A open only in BamHI site with one insert in this site. D1 has the insert in the good direction and D2 has the insert in the opposite direction.</LI><LI>Clones D1 and D2 were stored in glycerol.</LI><LI>At the end of the week, PCR with new primers containing KpnI instead of BglII were made for all the parts (cOT2 ; leucrocine ; DNY15 ; Odr10). All the PCR worked perfectly, PCR tubes were pooled. Odr10 has secondary products which need gel extraction. </LI><LI>Digestion of plasmid and parts with KpnI and BamHI</LI></UL></DIV></DIV></DIV></SECTION></DIV><DIV class="week" id="week-28"><SECTION><H1> Week 28 </H1></SECTION><SECTION class="week-meeting"><H2>07-13: 23th brainstorming</H2><P>
           Citation from Cédric : “we must be humble before molecular biology”. Our card game conception progressing well! We find a name : MicrobioWorld, and the main rules has been determined after several crazy gameplay! 
        </P></SECTION><SECTION class="week-ihp"><H2>07-14: Meeting with Doctors Without Borders, cholera specialist</H2><P>
          Fransisco Luquero assured us that the current solutions (bleach, boil water) were not the most suitable to treat water from cholera. Our device could be interesting if it is suitable to treat several liters of water and even if it contains GMM, it could be easily accepted by the population as long as it does not change the color and taste of water.
        </P></SECTION><SECTION class="week-experiment"><H2>Experiments</H2><DIV class="wet-lab-notebook"><DIV><DIV><H2>Assembly of <I>V. harveyi</I> gene circuit : continuing the sub-cloning work</H2><UL><LI>Assembly of V.harveyi gene circuit: Analysis of the transformants of V.harveyi module (attempt #1 and #2) by checking of the restriction map. </LI><LI>New attempt (attempt #3) to sub-clone Vh1, Vh2 and  Vh3 in their respective vector.  </LI></UL></DIV><DIV><H2>Cloning</H2><UL><LI>Ligation and transformation of pPICZα-cOT2 ; pPICZα-leucrocine ; pPICZα-Odr10 and pPICZα-DNY15.</LI><LI>Miniprep of transformants and gel migration were made. The ladder was clear but there was no DNA on tranformants colums.</LI><LI>Nanodrop analysis showed that there was not enough DNA.→ The hypothesis is that the liquid culture was too long so <I>E. coli</I> lost the plasmid.</LI></UL><H2>Experiments with <I>P. pastoris</I></H2><UL><LI>We did plates of YPD with increasing concentration of zeocin to select <I>P. pastoris</I> clones which integrated the most copies of the construction.</LI><LI>Preparation of competent yeast (<I>Pichia pastoris</I>)</LI><LI>Digestion of D1 clone (pPICZα-DNY15) with HindIII to integrate the plasmid construction at the AOX1 promotor in <I>P. pastoris</I> genome.</LI><LI>Electroporation of competent yeast with the digested DNA.</LI></UL></DIV></DIV></DIV></SECTION></DIV><DIV class="week" id="week-29"><SECTION><H1> Week 29 </H1></SECTION><SECTION class="week-experiment"><H2>07-17: Model</H2><P>
          Our system of ODEs script and function have been developed in Matlab.
        </P></SECTION><SECTION class="week-meeting"><H2>07-20: 24th brainstorming</H2><P>
          We’ve finished a first version of our presentation, which will be presented during the Parisian Meetup. The Matlab programming is done, the next step is to add values from bibliography.
        </P></SECTION><SECTION class="week-ihp"><H2>07-20: First meeting with Pierre-Alain Hoffmann</H2><P>
          We met Pierre-Alain Hoffmann, Deputy Director of CRITT Bio-Industries for the business plan elaboration. The knowledge in creation and development of companies of Pierre-Alain Hoffmann allowed us to understand the goal of a business plan. He accompanied us in this approach, explaining the points to be detailed in order to better integrate our system in the market.
        </P></SECTION><SECTION class="week-igemers"><H2>07-21: Parisian meetup</H2></SECTION><SECTION class="week-experiment"><H2>Experiments</H2><DIV class="wet-lab-notebook"><DIV><DIV><H2>Expressing VhCqsA (2nd)</H2><UL><LI>Production of C8-CAI-1 in minimal media, extration on LLE dichloromethane and analysis on NMR (fail)</LI><LI>SDS page fo the protein overexpression of our clones (fail)</LI></UL></DIV><DIV><H2>Assembly of <I>V. harveyi</I> gene circuit : A new strategy</H2><UL><LI>Amplification of the IDT gblocks, digestion of the PCR amplicons. </LI><LI>Amplification and digestion of the sub-cloning vectors. </LI><LI>Ligation and transformation in <I>E. coli</I> STELLAR competent cells (attempt #8). </LI></UL></DIV><DIV><H2>Cloning</H2><UL><LI>This time transformation were made with <I>E. coli</I> XL1 Blue which are supposed to be more efficient.</LI><LI>No DNA after gel migration like the previous week.</LI></UL><H2>Experiments with <I>P. pastoris</I></H2><UL><LI>Colony PCR of <I>P. pastoris</I> clone which grew on YPD zeocin. → All clones have the gene construction in their genome!￼</LI><LI>The 11 clones of Pichia were grown on plates with 250 µg/mL, 500 µg/mL and 1000 µg/mL to see which one has the best integration of the construction. All clones grew onto 250, 500 and 1000 µg/mL zeocin.</LI></UL></DIV></DIV></DIV></SECTION></DIV><DIV class="week" id="week-30"><SECTION><H1> Week 30 </H1></SECTION><SECTION class="week-igemers"><H2>07-23: Skype with the iGEM team of Greece</H2></SECTION><SECTION class="week-ihp"><H2>07-27: Meeting with Pierre Monsan at TWB</H2><P>
          As a preindustrial demonstrator recognized in France, TWB helps many projects to become companies. We presented them our project and had a positive feedback. It is even one of our biggest sponsors.
        </P><P>
          You can read the result of <A href="https://2017.igem.org/Team:INSA-UPS_France/Entrepreneurship/Testimonies#a2">their testimony here.</A></P></SECTION><SECTION class="week-team"><H2>07-29: Team building in Pyrenees</H2></SECTION><SECTION class="week-experiment"><H2>Experiments</H2><DIV class="wet-lab-notebook"><DIV><DIV><H2>Expressing VhCqsA (3rd)</H2><UL><LI>Production of C8-CAI-1 in minimal media, extration on LLE dichloromethane and analysis on NMR (fail)</LI></UL></DIV><DIV><H2>Assembly of <I>V. harveyi</I> gene circuit : The first transformant</H2><UL><LI>TA cloning (attempt #9): amplification of the IDT gblocks with KAPA2G polymerase , digestion of the PCR amplicons. Ligation with pGEM vector TA cloning vector. </LI><LI>Analysis of the transformants (attempt #8 and #9)  by checking the restriction map. </LI><LI>The part 1 and 2 were successfully cloned in pBR322 and pGEM.  </LI></UL></DIV><DIV><H2>Cloning</H2><UL><LI>PCR were made again of all parts in order to have enough materials to do ligation.</LI><LI>This time lot of DNA was digested.</LI><LI>Several strategies were tried to obtain pPICZα-cOT2 ; pPICZα-leucrocine  and pPICZα-Odr10 with DHAα and XL1 Blue strains.</LI><LI>At the end of the week we got 2 clones of pPICZα-Odr10 in E. coli DHAα</LI></UL><H2>Experiments with <I>P. pastoris</I></H2><UL><LI>The supernatant of <I>P. pastoris</I> containing the DNY15 gene was extracted and then concentrated via rotavapor and freeze-drying.</LI><LI>In a 24 wells plate, kinetic growths of <I>V. harveyi</I> were followed with or without <I>P. pastoris</I> supernatant to check the bactericide effect of DNY15. → this first try was not perfectly made</LI><LI>Inhibition test with the same supernatant was made on petri dish. → this first try was not perfectly made</LI></UL></DIV></DIV></DIV></SECTION></DIV><DIV class="week" id="week-31"><SECTION><H1> Week 31 </H1></SECTION><SECTION class="week-meeting"><H2>08-01: 25th brainstorming</H2><P>
          Our collaborations with other iGEM teams are in process! We’ve started an experiment for iGEM Purdue.
        </P></SECTION><SECTION class="week-experiment"><H2>Experiments</H2><DIV class="wet-lab-notebook"><DIV><DIV><H2>Expressing VhCqsA (aborted) + Cloning of VcCqsA </H2><UL><LI>Production of C8-CAI-1 in minimal media, extration on LLE dichloromethane and analysis on NMR aborted : no growth of bacteria</LI></UL></DIV><DIV><H2>Step 2 : reconstruction of CqsS Rc by ligating Vh2 to Vh1 in pBR322. </H2><UL><LI>Storage of the transformants with Vh1 and Vh2 in glycerol at -80°C.</LI><LI>Amplification, digestion of pBR322-Vh1,  pGEM-Vh2 and ligation in <I>E. coli</I> TOP10 competent cells (attempt A)</LI><LI>Transformation of Vh3 (attempt #10) in <I>E. coli</I> DH5α competent cells </LI><LI>Analysis of the transformants (attempt #A and #10)  by checking the restriction map.</LI></UL></DIV><DIV><H2>Cloning</H2><UL><LI>Different strategies were made again to obtain pPICZα-cOT2 and pPICZα-leucrocine in E.coli. → Several clones of pPICZα-cOT2 and pPICZα-leucrocine were obtained!!!</LI><LI>Liquid culture of DHAα containing pSB1C3 was made to get enough material to do cloning of our parts in it.</LI><LI>Ligation of YFP in pPICZα-Odr10 and in pPICZα-DNY15 instead of the AMP gene.</LI><LI>Integration of AMP genes in pSBIC3 (=iGEM parts).</LI></UL><H2>Experiments with <I>P. pastoris</I>￼</H2><UL><LI>Cytotoxicity test of DNY15 was tried again with kinetic growths of <I>V. harveyi</I>.</LI><LI>Cytotoxicity test of DNY15 was tried again on petri dish: No effect of DNY15 can be seen. </LI><LI>Genomic integration of pPICZα-cOT2 , pPICZα-leucrocine and pPICZα-Odr10.</LI></UL></DIV></DIV></DIV></SECTION></DIV><DIV class="week" id="week-32"><SECTION><H1> Week 32 </H1></SECTION><SECTION class="week-igemers"><H2>08-07: Skype with the iGEM team of Boston</H2></SECTION><SECTION class="week-experiment"><H2>08-07: Model</H2><P>
          All the experimental and literature values have been implemented in our Matlab script, which provides promising results. These first results will be used for preliminary work in design and entrepreneurship.
        </P></SECTION><SECTION class="week-experiment"><H2>Experiments</H2><DIV class="wet-lab-notebook"><DIV><DIV><H2>Expressing VhCqsA (4th time) + VcCqsA cloning</H2><UL><LI>Production of C8-CAI-1 in minimal media, extration on LLE dichloromethane and analysis on NMR (fail)</LI><LI>PCR on VcCqsA</LI></UL></DIV><DIV><H2>Step 2 CqsS Rc assembly in progress…still no success for Vh3</H2><UL><LI>Amplification, digestion of pBR322-Vh1  pGEM-Vh2 and ligation in <I>E. coli</I> TOP10 competent cells (attempt B and C)</LI><LI>Transformation of Vh3 (attempt #11 to #13) in <I>E. coli</I> TOP10 competent cells using an alternative restriction enzyme :  PstI-HF restriction enzyme is remplaced  by SpeI.</LI></UL></DIV><DIV><H2>Cloning</H2><P>
                We obtained the YFP gene after the GAP promotor and the FUS1 promotor in <I>E. coli</I>.
              </P><H2>Experiments with <I>P. pastoris</I></H2><UL><LI>pGAP-YFP and pFUS-YFP were transformed in <I>P. pastoris</I>.</LI><LI><I>P. pastoris</I> clones were obtained with pPICZα-Odr10 ; pPICZα-leucrocine and pPICZα-cOT2 but the PCR colony didn’t work. After several trials and controls we concluded that the DNA extraction the way we did it was not enough to do a PCR of the GAP promotor in the pichia genome.</LI><LI>A kinetic study of <I>V. harveyi</I> growth was made using pichia supernatant. → A difference was observed but the experiment needs to be done again.</LI></UL></DIV></DIV></DIV></SECTION></DIV><DIV class="week" id="week-33"><SECTION><H1> Week 33 </H1></SECTION><SECTION class="week-igemers"><H2>08-16: Skype with the iGEM team of Groningen</H2></SECTION><SECTION class="week-team"><H2>08-17: “iGEM on ice”</H2></SECTION><SECTION class="week-experiment"><H2>Experiments</H2><DIV class="wet-lab-notebook"><DIV><DIV><H2>Cloning of VcCqsA</H2><UL><LI>From ligation to transformation : transformants grew.</LI></UL></DIV><DIV><H2>Step 2 CqsS Rc assembly in progress, preparation of the conjugation</H2><UL><LI>Analysis of the transformants (attempt #13)  by checking the restriction map : the part 3 is successfully cloned in pSB1C3. The transformant is stored  in glycerol at -80°C.</LI><LI>Amplification of pBBR1MCS-4 and pBBR1MCS-5 for cloning of Vh3 and for a conjugation assay. </LI><LI>Ligation of Vh3 in pBBR1MCS-4 and transformation in <I>E. coli</I> TOP10 (attempt α). </LI><LI>Amplification, digestion of pBR322-Vh1  pGEM-Vh2 and ligation in <I>E. coli</I> TOP10 competent cells (attempt D to F) </LI></UL></DIV><DIV><UL><LI><I>P. pastoris</I> clones were obtained with the YFP reporter behind both promotor: GAP and FUS1.</LI><LI>Fluorescence test was made but it didn’t work.</LI><LI>A cytotoxicity test was made on V.harveyi with pichia-AMP supernantant but we had trouble because of volumes used.</LI><LI>We tried a new DNA extraction protocol to do the PCR colony on <I>P. pastoris</I> and it did work.</LI></UL></DIV></DIV></DIV></SECTION></DIV><DIV class="week" id="week-34"><SECTION><H1> Week 34 </H1></SECTION><SECTION class="week-meeting"><H2>08-24: 26th brainstorming</H2><P>
          We finally choose our track! We will be candidate in “information processing”, to highlight the prokaryote-eukaryote communications in our multimicrobial system!
        </P></SECTION><SECTION class="week-experiment"><H2>Experiments</H2><DIV class="wet-lab-notebook"><DIV><DIV><H2>Confirming VcCqsa clone + Expressing VhCqsA (5th time)</H2><UL><LI>Transformant were good</LI><LI>Production of C8-CAI-1 in minimal media, extration on LLE dichloromethane and analysis on NMR (fail)</LI></UL></DIV><DIV><UL><LI>Analysis of the transformants (attempt #F)  by checking the restriction map : the part Vh1+Vh2 is successfully cloned in pBR322. The transformant is stored  in glycerol at -80°C.</LI><LI>Amplification, digestion of pBR322-Vh1+Vh2, pSB1C3-Vh3. Ligation and transformation (attempt #1 and #2) in <I>E. coli</I> TOP10 competent cells. </LI><LI>Ligation of Vh3 in pBBR1MCS-4 and transformation in <I>E. coli</I> TOP10 (attempt β and γ)</LI><LI>Vh3 part validation : diacetyl production assay by NMR detection with <I>E. coli</I> BL21. </LI></UL></DIV><DIV><UL><LI>Cytotoxicity test was made again but this time on a petri dish. We had a halo for the antibiotic but <I>P. pastoris’</I> supernatant didn’t stop <I>V. harveyi’</I>s growth.</LI><LI>We’ve tried to characterize pFUS1 promoter in different media but we got no positive result.</LI></UL></DIV></DIV></DIV></SECTION></DIV><DIV class="week" id="week-35"><SECTION><H1> Week 35 </H1></SECTION><SECTION class="week-ihp"><H2>08-28: First try for the device modeling</H2><P>
           A 3D modeling of the device was initiated on the Solidworks software with the help of a professional in the design of industrial parts. The objective is to model the device in 3D and then to print the prototype.
        </P></SECTION><SECTION class="week-meeting"><H2>08-31: 27th brainstorming</H2><P>
          After much thinking, we are agreed that we want Margaux, Maxant and Margaux to be our speakers during the competition in Boston!
        </P></SECTION><SECTION class="week-experiment"><H2> 08-31: Model</H2><P>
          First MCA results! The control exerted by each parameter on the response time can now be analysed.
        </P></SECTION><SECTION class="week-experiment"><H2>Experiments</H2><DIV class="wet-lab-notebook"><DIV><DIV><H2>Expressing VhCqsA (6th time) + Bioluminescence</H2><UL><LI>Production of C8-CAI-1 in minimal media, extration on LLE dichloromethane and analysis on NMR (fail)</LI><LI>Optimisation of <I>Vibrio harveyi</I> JMH626 bioluminescent assay</LI></UL></DIV><DIV><UL><LI>2nd  diacetyl production assay by NMR detection with <I>E. coli</I> BL21.</LI><LI>Success of the conjugation assay of pBBR1MCS-4. </LI></UL></DIV><DIV><UL><LI>This time, we tested the production of DNY15 in YPD with 40g/L of glucose instead of 30g/L because publications which use pGAP as a promotor to produce AMP sometimes have a higher production with more glucose.</LI><LI>Cytotoxicity tests using the supernatant from the 40g/L glucose culture media was made. Supernatants were filtered beforehand and half was freeze-dried.  We got a weak halo for the DNY15 production in glucose 40 and supernatant freeze-dried!</LI></UL></DIV></DIV></DIV></SECTION></DIV><DIV class="week" id="week-36"><SECTION><H1> Week 36 </H1></SECTION><SECTION class="week-pe"><H2>09-04: Start of our Ulule campaign</H2></SECTION><SECTION class="week-meeting"><H2>09-07: 28th brainstorming</H2></SECTION><SECTION class="week-team"><H2>09-08: Farewell party</H2></SECTION><SECTION class="week-ihp"><H2>09-09: Meeting with Marie-Pierre Escudié</H2></SECTION><SECTION class="week-experiment"><H2>Experiments</H2><DIV class="wet-lab-notebook"><DIV><DIV><H2>Expressing VhCqsA (7th time)</H2><UL><LI>Production of C8-CAI-1 in minimal media, and analysis on MS (fail) extration on LLE dichloromethane and analysis on NMR (fail)</LI></UL></DIV><DIV><UL><LI>3rd  Diacetyl production assay by NMR detection with <I>E. coli</I> MG1655. </LI><LI>Amplification of pBBR1MCS-4 and pBBR1MCS-5. </LI></UL></DIV><DIV><UL><LI>Because we still had trouble integrating some parts in <I>P. pastoris’</I> genome we tried the integration via electroporation and via chemical way at the same time.</LI><LI>We launched one again <I>P. pastoris</I> culture in YPD 40g/L and 50g/L glucose to try producing more DNY15.</LI></UL></DIV></DIV></DIV></SECTION></DIV><DIV class="week" id="week-37"><SECTION><H1> Week 37 </H1></SECTION><SECTION class="week-meeting"><H2>09-14: 29th brainstorming</H2><P>
          We’ve prepared the abstract, and choose a subtitle : “a synthetic microbial consortium”</P></SECTION><SECTION class="week-experiment"><H2>Experiments</H2><DIV class="wet-lab-notebook"><DIV><DIV><H2>Bioluminescence assay</H2><UL><LI>Trial on the expression of C8-CAI-1 with induction on bioluminescence on liquid media : fail</LI></UL></DIV><DIV><UL><LI>4th diacetyl production assay by NMR detection with <I>E. coli</I> MG1655 in M9 media supplemented with xylose and pyruvate. </LI><LI>Transformation of <I>E. coli</I> TOP10 competent cells with GFP from iGEM kit</LI><LI>Ligation and transformation of pBBR1MCS-4 to RFP from iGEM kit in <I>E. coli</I> TOP10 competent cells for conjugation to <I>V. harveyi</I>. </LI><LI>Preparation of the clones for sequencing. </LI></UL></DIV><DIV><UL><LI>We prepared sample to sequence the AMP genes we wanted to give to the iGEM registry.</LI><LI>Culture media from <I>P. pastoris</I> culture of last week were freeze-dried. And the cytotoxicity test on petri dish was made again but this time, even the positive control with antibiotic didn’t work.</LI><LI><I>P. pastoris</I> clones were obtained with pAOX1-leucrocine, pAOX1-cOT2, Odr10-pFUS-RFP.</LI></UL></DIV></DIV></DIV></SECTION></DIV><DIV class="week" id="week-38"><SECTION><H1> Week 38 </H1></SECTION><SECTION class="week-meeting"><H2>09-21: 30th brainstorming</H2><P>
          We’ve finished the experiment time, it’s time to write results for the wiki!
        </P></SECTION><SECTION class="week-ihp"><H2>09-22: Meeting with UNICEF</H2><P>
          He warned us that NGOs would agree to use a GMM system only if efficacy is proven and there is no danger of GMM proliferation. He also gave us an idea of the volume of water a village needs. Moreover, it is also important to make a system with as little waste as possible.
        </P><P>
          You can read the result of <A href="https://2017.igem.org/Team:INSA-UPS_France/Entrepreneurship/Testimonies#a6">Alama Keita's testimony here.</A></P></SECTION><SECTION class="week-experiment"><H2>Experiments</H2><DIV class="wet-lab-notebook"><DIV><DIV><H2>Bioluminescence assay</H2><UL><LI>Trial on the expression of C8-CAI-1 with induction on bioluminescence on plate : success</LI></UL></DIV><DIV><UL><LI>Results of the 4th Diacetyl production assay by NMR detection: a small peak of diacetyl is visible. </LI><LI>5th diacetyl production assay by NMR detection with <I>E. coli</I> MG1655 in M9 media supplemented with xylose and pyruvate. </LI><LI>Results of ligation of RFP in conjugative plasmid: all transformants are red. Two transformants are stored at -80°C in order to be used for conjugation assay. </LI><LI>Preparative work for the ligation of Vh3 in pBBR1MCS-4 with fresh material</LI></UL></DIV><DIV><UL><LI>AMP genes were sent to be sequenced.</LI><LI><I>P. pastoris</I> producing DNY15 was cultured to do a RT-qPCR experiment to compare it to the WT <I>P. pastoris</I>.</LI><LI>New cytotoxicity tests on petri dish were made but this time it didn’t work because of the paper disk we used. Once again the positive control didn’t work.</LI><LI>Cultures in BMMY adding methanol every 24h for 120h were made for both pAOX1-leucrocine and pAOX1-cOT2 P.pastoris clones. Half of each supernatant was stored at 4°C and the other half was evaporated to concentrate 10 times.</LI></UL></DIV></DIV></DIV></SECTION></DIV><DIV class="week" id="week-39"><SECTION><H1> Week 39 </H1></SECTION><SECTION class="week-meeting"><H2>09-27: 31th brainstorming</H2><P>
          We’ve started to make rehearsals for the presentation in Boston.
        </P></SECTION><SECTION class="week-pe"><H2>09-29: European Researcher's Night</H2><P>
          We’ve been to the European Researcher’s Night, a major scientific event that gather researchers and general public in a convivial atmosphere. In 2017, the topic of the event was “Impossible”. In order to follow this thematic, we chose to focus our workshop on the incredible features of biodiversity, how to use it in synthetic biology, and what are legal limits on GMO use in France.
        </P></SECTION><SECTION class="week-experiment"><H2>Experiments</H2><DIV class="wet-lab-notebook"><DIV><DIV><UL><LI>Observation by microscopy of <I>V. harveyi</I> (rfp) transformant</LI><LI>Ligation of pBBR1MCS-4 and Vh3 at 16°C overnight, and transformation in competent <I>E. coli</I> (Stellars)</LI></UL></DIV><DIV><UL><LI>New cytotoxicity tests on petri dishes were made with the leucrocine and cOT2 supernatants. This time with appropriate discs. Positive control did not work again.</LI><LI> New cytotoxicity tests on petri dishes were made but using pure chloramphenicol. A weak halo was seen. We found out that the <I>V. harveyi</I> strain we used to do cytotoxicity test was maybe! the JMH626 one which was chloramphenicol-resistant.</LI><LI>RT-PCR experiment was made.</LI><LI>To test the Odr10-RFP system and activate pFUS, nitrogen depletion cultures were done and two sources of nitrogen were tested: sulfate ammonium and glutamine. The fluorescence following was made in a plate reader.</LI><LI>New cytotoxicity tests on petri dishes were made starting from a fresh <I>V. harveyi</I> cuture. Leucrocine, cOT2 and D-NY15 (40g/L and 50g/L glucose) productions were tested. Halos are clearly visible for D-NY15 40g / L not concentrated and concentrated 10 times.</LI><LI>A 24 well plate was launched by adding leucrocine, cOT2 and D-NY15 supernatant to <I>V. harveyi</I> cultures in exponential phase.</LI></UL></DIV></DIV></DIV></SECTION></DIV><DIV class="week" id="week-40"><SECTION><H1> Week 40 </H1></SECTION><SECTION class="week-ihp"><H2>10-02: Meeting with Pierre-Alain Hoffmann, director of CRITT Bio-industries.</H2><P>
          Between these two appointments, we have elaborated the business plan ourselves. Pierre-Alain Hoffmann read it and made a feed-back. He was pleased with our work, ensuring that some start-ups do not start their business with such a business plan.
        </P></SECTION><SECTION class="week-meeting"><H2>10-05: 32th brainstorming</H2><P>
           Rehearsals, and writing of different pages of the wiki.
        </P></SECTION><SECTION class="week-pe"><H2>10-04: High school practical work</H2><P>
          We prepared an intervention for High School senior students in scientific classes. Our involvement was focused on 3 main goals: implement a learner-centred pedagogical approach in a high school class, make discover biotechnologies through a practical approach, and discuss on “how to be part of science, and which paths to choose after high school?”.
        </P></SECTION></DIV><DIV class="week" id="week-41"><SECTION><H1> Week 41 </H1></SECTION><SECTION class="week-meeting"><H2>33th brainstorming</H2><P>
          Rehearsals, and writing of different pages of the wiki.
        </P></SECTION><SECTION class="week-pe"><H2>All week: exhibition at Bib'INSA</H2><P>
          In order to make former iGEM Toulouse projects understandable for all scientist student (and not only for biologists), we redesigned posters of previous project E. calculus, SubtiTree, ApiColi, Paleotilis, and of our project Croc’n Cholera.
        </P></SECTION></DIV><DIV class="week" id="week-42"><SECTION><H1> Week 42 </H1></SECTION><SECTION class="week-ihp"><H2>10-10: End of the 3D modeling of the device and 3D printing</H2><P>
          The device was printed in 3D by the High school Georges Cabanis at Brive-La-Gaillarde.￼
        </P></SECTION><SECTION class="week-experiment"><H2>10-11: Model</H2><P>
          Some inaccuracies in our mathematical model have been corrected, we have now the final version of all our Matlab files! Results will be used for our entrepreneurship and design work.
        </P><H2>10-13</H2><P>
          We have now our final MCA, and performed a global analysis of our system. We can now definitely conclude about sensitivity and robustness of our model.
        </P></SECTION><SECTION class="week-pe"><H2>13-10: Seminar at the campus science Toulouse</H2><P>
          After working during several months on synthetic biology for our iGEM project, we needed to organize an event to present our work and the one achieved by the previous iGEM teams. We also wanted to have an exchange of opinions with people who possibly have different values and considerations than us. The intended public was mainly composed of the Paul Sabatier university and INSA Toulouse students, two of the schools we are intending.
        </P></SECTION></DIV><DIV class="week" id="week-43"><SECTION><H1> Week 43 </H1></SECTION><SECTION class="week-experiment"><H2>10-21: Model</H2><P>
          An intuitive modeling interface have been built for displaying our model results to non-specialists.
        </P></SECTION></DIV><DIV class="week" id="week-44"><SECTION><H1> Week 44 </H1></SECTION></DIV><SECTION class="links_end"><TABLE><TBODY><TR><TH colspan="6">
            Realisations pages
          </TH></TR><TR><TD><A href="https://2017.igem.org/Team:INSA-UPS_France/Experiments">Overview</A></TD><TD><I>Notebook</I></TD><TD><A href="https://2017.igem.org/Team:INSA-UPS_France/Experiments/Clonings">Clonings</A></TD><TD><A href="https://2017.igem.org/Team:INSA-UPS_France/Results">Results</A></TD><TD><A href="https://2017.igem.org/Team:INSA-UPS_France/Contribution">Contribution</A></TD><TD><A href="https://2017.igem.org/Team:INSA-UPS_France/Protocols">Protocols</A></TD><TD><A href="https://2017.igem.org/Team:INSA-UPS_France/Safety">Safety</A></TD></TR></TBODY></TABLE></SECTION></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>