<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_INSA-UPS_France_Model_Simulation skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:INSA-UPS France/Model/Simulation</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container" style="position:relative; "><DIV class="site-header__title">
    Croc'n Cholera <SPAN style="font-size:12pt;">iGEM UPS-INSA Toulouse 2017</SPAN></DIV><NAV class="site-menu" style="text-align: center;"><DIV class="site-menu__item"><DIV class="cat_name">Team </DIV><UL class="subnav" style="width:150%;"><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Team">Members</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Supports">Supports</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Attributions">Attributions</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Collaborations">Collaborations</A></LI></UL></DIV><DIV class="site-menu__item"><DIV class="cat_name">Strategy </DIV><UL class="subnav"><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Description">Description</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Design">Design</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Parts">Parts</A></LI></UL></DIV><DIV class="site-menu__item"><DIV class="cat_name">Model </DIV><UL class="subnav"><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Model">Overview</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Model/Simulation">Simulation</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Model/Analysis">Analysis</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Model/Interface">Interface</A></LI></UL></DIV><DIV class="site-menu__item"><DIV class="cat_name">Realisations </DIV><UL class="subnav"><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Experiments">Overview</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Notebook">Notebook</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Experiments/Clonings">Clonings</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Results">Results</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Contribution">Contribution</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Protocols">Protocols</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Safety">Safety</A></LI></UL></DIV><DIV class="site-menu__item"><DIV class="cat_name">Human Practices</DIV><UL class="subnav"><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Human_Practices">Overview</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/HP/Silver">Silver</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/HP/Gold_Integrated">Gold</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Engagement">Public Engagement</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Microbioworld">Microbioworld</A></LI></UL></DIV><DIV class="site-menu__item"><DIV class="cat_name">Application </DIV><UL class="subnav"><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Applied_Design">Applied Design</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Entrepreneurship/Testimonies">Testimonies</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Entrepreneurship">Entrepreneurship</A></LI><LI><A href="https://2017.igem.org/Team:INSA-UPS_France/Entrepreneurship/Device">Device Conception</A></LI></UL></DIV></NAV></DIV><DIV class="main_content"><DIV class="middle_container"><DIV class="section_container"><SECTION style="display:table;background:none;padding:0px !important;z-index:100; "><H1 style="vertical-align:bottom;display:table-cell; width:50%;font-size:60pt;letter-spacing: 0.2em;z-index:120;text-align: center;">Simulation</H1></SECTION><DIV class="left_container"><DIV class="left_container__inside"><DIV class="aside-nav__item"><A href="#a1" data-number="1">
        Kinetic model
      </A></DIV><DIV class="aside-nav__item"><A href="#a2" data-number="2">
        Data and parameters
      </A></DIV><DIV class="aside-nav__item"><A href="#a3" data-number="3">
        Solver
      </A></DIV><DIV class="aside-nav__item"><A href="#a4" data-number="4">
        Simulation results
      </A></DIV><DIV class="aside-nav__item"><A href="#a5" data-number="5">
        References
      </A></DIV></DIV></DIV><DIV class="right_container"><SECTION><H1>Kinetic model</H1><P>Every biological and physical processes including in the SBGN representation of the model was described mathematically using appropriate kinetic rate laws (passive diffusion, Michaelis-Menten enzyme kinetics, first order degradation of mRNAs, exponential growth law, etc). The final model contains <B>12 entities</B> (genes, RNAs, proteins, metabolites, and cells) and <B>24 reactions</B> from a broad range of processes (transcription, translation, RNAs and proteins degradation, growth and death of each micro-organism, metabolite poduction, transport between compartments, diffusion, etc). The dynamics of our microbial consortium was summed up in twelve differential equations as a system of ODEs. 
</P><P>For fearless people, our complete mathematical model and demonstration can be found <A href="https://static.igem.org/mediawiki/2017/2/2c/T--INSA-UPS_France--Complete_model.pdf">there</A>! The final set of ODEs is briefly described below.</P><FIGURE><FIGCAPTION>SBGN representation of the model developed in the project
</FIGCAPTION></FIGURE><P>We need to consider the growth and death of the three microorganisms: <I><B>Vibrio cholerae</B></I> (<I>Vc</I>) in water (W), and <I><B>Vibrio harveyi</B></I> (<I>Vh</I>) and <I><B>Pichia pastoris</B></I> (<I>Pp</I>) in the device (D).
\begin{equation}
    \frac{d[\textit{Vc}]_W}{dt} = V_{growth,Vc} - V_{death,Vc}
\end{equation}
\begin{equation}
    \frac{d[\textit{Vh}]_D}{dt} = V_{growth,Vh} - V_{death,Vh}
\end{equation}
\begin{equation}
    \frac{d[\textit{Pp}]_D}{dt} = V_{growth,Pp} - V_{death,Pp}
\end{equation}
</P><P>
Microorganisms death is impacted by <B>antimicrobial peptides</B> production (AMP<SUB>peptide,Pp</SUB>), produced by translation of antimicrobial peptides mRNA (AMP<SUB>RNA</SUB>). Peptides and mRNA are also degraded.
\begin{equation}
    \frac{d[AMP_{RNA}]_{Pp}}{dt}=V_{transcription,AMP} - V_{degradation,AMP RNA}
\end{equation}

\begin{equation}
    \frac{d[AMP_{peptide}]_{D}}{dt}=V_{translation,AMP} - V_{degradation,AMP} + \frac{V_{diff,AMP,W \to D}}{\mathcal{V}_D}
\end{equation}
</P><P>These peptides are transfered between compartments: from the <B>device</B> (D) to <B>water</B> (W).

\begin{equation}
    \frac{d[AMP]_W}{dt} = -V_{diff,AMP,W\to D}
\end{equation}
</P><P>To produce antimicrobial peptides, an activation by <B>diacetyl</B> (dac) is needed. Diacetyl can freely diffuse from the device (D) to water (W).
\begin{equation}
    \frac{d[dac]_D}{dt}=V_{prod,dac}+\frac{V_{diff,dac,W \to D}}{\mathcal{V}_D}
\end{equation}

\begin{equation}
    \frac{d[dac]_W}{dt}=- V_{diff,dac,W \to D}
\end{equation}
</P><P>Diacetyl is produced by the enzyme <B>acetolactate synthase</B> (ALS<SUB>enzyme</SUB>). The <I>als</I> gene is first transcribed into ALS<SUB>RNA</SUB>, which is then translated into the protein. Both the enzyme and its mRNA can be degraded.
\begin{equation}
    \frac{d[ALS_{RNA}]_{Vh}}{dt} = V_{transcription,ALS} - V_{degradation,ALS RNA}
\end{equation}

\begin{equation}
    \frac{d[ALS_{enzyme}]_{Vh}}{dt} = V_{translation,ALS} - V_{degradation,ALSenzyme}
\end{equation}
</P><P>ALS production has to be activated by the quorum sensing molecule <B>CAI-1</B>, initially in water (W), after diffusing into the device (D). 

\begin{equation}
    \frac{d[CAI\text{-}1]_D}{dt} = \frac{V_{diff,CAI\text{-}1,W\to D}}{\mathcal{V}_D}
\end{equation}

\begin{equation}
    \frac{d[CAI\text{-}1]_W}{dt} = -V_{diff,CAI\text{-}1,W\to D}
\end{equation}

</P></SECTION><SECTION><H1>Data and Parameters</H1><P>Model parameters were mostly collected from publications, because a large part of the required data necessitates a complex set of hundreds of experiments that could not have been performed in the course of the project.</P><TABLE><TBODY><TR><TH>Name</TH><TH>Notation</TH><TH>Unit</TH><TH>Value</TH><TH>Reference</TH></TR><TR><TD><I>Vibrio cholerae</I> maximum growth rate</TD><TD>μ<SUB>MAX,<I>Vc</I></SUB></TD><TD>s<SUP>-1</SUP></TD><TD>3.10<SUP>-4</SUP></TD><TD>BioNumbers ID 112369 <SUP><A href="https://www.ncbi.nlm.nih.gov/pubmed/19854939" alt="">1</A></SUP></TD></TR><TR><TD>Leucrocine I MIC for <I>V. cholerae</I></TD><TD>MIC<SUB>Leucro,<I>Vc</I></SUB></TD><TD>mol/L</TD><TD>6.4.10<SUP>-5</SUP></TD><TD>Pata <I>et al.</I>, 2011 <SUP><A href="https://www.ncbi.nlm.nih.gov/pubmed/19854939" alt="">2</A></SUP></TD></TR><TR><TD>Leucrocine I MIC for <I>V. harveyi</I></TD><TD>MIC<SUB>Leucro,<I>Vh</I></SUB></TD><TD>mol/L</TD><TD>6.4.10<SUP>-5</SUP></TD><TD>Pata <I>et al.</I>, 2011 <SUP><A href="https://www.ncbi.nlm.nih.gov/pubmed/21184776" alt="">2</A></SUP> - Extrapolation from <I>V. cholerae</I> result</TD></TR><TR><TD>Leucrocine I MIC for <I>P. pastoris</I></TD><TD>MIC<SUB>Leucro,<I>Pp</I></SUB></TD><TD>mol/L</TD><TD>10<SUP>15</SUP></TD><TD>Assuming no effects on <I>Pichia pastoris</I></TD></TR><TR><TD>Leucrocine I IC50 for <I>V. cholerae</I></TD><TD>MIC<SUB>Leucro,<I>Vc</I></SUB></TD><TD>mol/L</TD><TD>1.92.10<SUP>-4</SUP></TD><TD>Considering IC50 = 3.MIC - Extrapolation of data from standard antibiotics) <SUP><A href="https://www.ncbi.nlm.nih.gov/pubmed/21184776" alt="">3</A></SUP></TD></TR><TR><TD>Leucrocine I IC50 for <I>V. harveyi</I></TD><TD>IC50<SUB>Leucro,<I>Vh</I></SUB></TD><TD>mol/L</TD><TD>1.92.10<SUP>-4</SUP></TD><TD>Considering IC50 = 3.MIC - Extrapolation of data from standard antibiotics <SUP><A href="https://www.ncbi.nlm.nih.gov/pubmed/21184776" alt="">3</A></SUP></TD></TR><TR><TD>Leucrocine I IC50 for <I>P. pastoris</I></TD><TD>IC50<SUB>Leucro,<I>Pp</I></SUB></TD><TD>mol/L</TD><TD>10<SUP>15</SUP></TD><TD>Assuming no effects on <I>Pichia pastoris</I></TD></TR><TR><TD>cOT2 MIC for <I>V. cholerae</I></TD><TD>MIC<SUB>cOT2,<I>Vc</I></SUB></TD><TD>mol/L</TD><TD>8.1.10<SUP>-6</SUP></TD><TD>Prajanban <I>et al.</I>, 2017 <SUP><A href="https://www.ncbi.nlm.nih.gov/pubmed/28159460" alt="">4</A></SUP></TD></TR><TR><TD>cOT2 MIC for <I>V. harveyi</I></TD><TD>MIC<SUB>cOT2,<I>Vh</I></SUB></TD><TD>mol/L</TD><TD>8.1.10<SUP>-6</SUP></TD><TD>Prajanban <I>et al.</I>, 2017 <SUP><A href="https://www.ncbi.nlm.nih.gov/pubmed/28159460" alt="">4</A></SUP> - Extrapolation from <I>V. cholerae</I> result</TD></TR><TR><TD>cOT2 MIC for <I>P. pastoris</I></TD><TD>MIC<SUB>cOT2,<I>Pp</I></SUB></TD><TD>mol/L</TD><TD>10<SUP>15</SUP></TD><TD>Assuming no effects on <I>Pichia pastoris</I></TD></TR><TR><TD>cOT2 IC50 for <I>V. cholerae</I></TD><TD>IC50<SUB>cOT2,<I>Vc</I></SUB></TD><TD>mol/L</TD><TD>2.43.10<SUP>-5</SUP></TD><TD>Considering IC50 = 3.MIC - Extrapolation of data from standard antibiotics <SUP><A href="https://www.ncbi.nlm.nih.gov/pubmed/21184776" alt="">3</A></SUP></TD></TR><TR><TD>cOT2 IC50 for <I>V. harveyi</I></TD><TD>IC50<SUB>cOT2,<I>Vh</I></SUB></TD><TD>mol/L</TD><TD>2.43.10<SUP>-5</SUP></TD><TD>Considering IC50 = 3.MIC - Extrapolation of data from standard antibiotics <SUP><A href="https://www.ncbi.nlm.nih.gov/pubmed/21184776" alt="">3</A></SUP></TD></TR><TR><TD>cOT2 IC50 for <I>P. pastoris</I></TD><TD>IC50<SUB>cOT2,<I>Pp</I></SUB></TD><TD>mol/L</TD><TD>10<SUP>15</SUP></TD><TD>Assuming no effects on <I>Pichia pastoris</I></TD></TR><TR><TD>D-NY15 MIC for <I>V. cholerae</I></TD><TD>MIC<SUB>D-NY15,<I>Vc</I></SUB></TD><TD>mol/L</TD><TD>1.54.10<SUP>-5</SUP></TD><TD>Yaraksa <I>et al.,</I> 2014 <SUP><A href="https://www.ncbi.nlm.nih.gov/pubmed/24192554" alt="">5</A></SUP></TD></TR><TR><TD>D-NY15 MIC for <I>V. harveyi</I></TD><TD>MIC<SUB>D-NY15,<I>Vh</I></SUB></TD><TD>mol/L</TD><TD>1.54.10<SUP>-5</SUP></TD><TD>Yaraksa <I>et al.,</I> 2014 <SUP><A href="https://www.ncbi.nlm.nih.gov/pubmed/24192554" alt="">5</A></SUP> - Extrapolation from <I>V. cholerae</I> result</TD></TR><TR><TD>D-NY15 MIC for <I>P. pastoris</I></TD><TD>MIC<SUB>D-NY15,<I>Pp</I></SUB></TD><TD>mol/L</TD><TD>10<SUP>15</SUP></TD><TD>Assuming no effects on <I>Pichia pastoris</I></TD></TR><TR><TD>D-NY15 IC50 for <I>V. cholerae</I></TD><TD>IC50<SUB>D-NY15,<I>Vc</I></SUB></TD><TD>mol/L</TD><TD>4.62.10<SUP>-5</SUP></TD><TD>Considering IC50 = 3.MIC - Extrapolation of data from standard antibiotics <SUP><A href="https://www.ncbi.nlm.nih.gov/pubmed/21184776" alt="">3</A></SUP></TD></TR><TR><TD>D-NY15 IC50 for <I>V. harveyi</I></TD><TD>IC50<SUB>D-NY15,<I>Vh</I></SUB></TD><TD>mol/L</TD><TD>4.62.10<SUP>-5</SUP></TD><TD>Considering IC50 = 3.MIC - Extrapolation of data from standard antibiotics <SUP><A href="https://www.ncbi.nlm.nih.gov/pubmed/21184776" alt="">3</A></SUP></TD></TR><TR><TD>D-NY15 IC50 for <I>P. pastoris</I></TD><TD>IC50<SUB>D-NY15,<I>Pp</I></SUB></TD><TD>mol/L</TD><TD>10<SUP>15</SUP></TD><TD>Assuming no effects on <I>Pichia pastoris</I></TD></TR><TR><TD><I>V. cholerae</I> death rate with AMP</TD><TD>k<SUB>kill,<I>Vc</I></SUB></TD><TD>s<SUP>-1</SUP></TD><TD>3.10<SUP>-3</SUP></TD><TD>Yaraksa <I>et al.</I>, 2014 <SUP><A href="https://www.ncbi.nlm.nih.gov/pubmed/24192554" alt="">5</A></SUP></TD></TR><TR><TD><I>V. harveyi</I> death rate with AMP</TD><TD>k<SUB>kill,<I>Vh</I></SUB></TD><TD>s<SUP>-1</SUP></TD><TD>3.10<SUP>-3</SUP></TD><TD>Extrapolation from Yaraksa <I>et al.</I>, 2014 <SUP><A href="https://www.ncbi.nlm.nih.gov/pubmed/24192554" alt="">5</A></SUP></TD></TR><TR><TD><I>P. pastoris</I> death rate with AMP</TD><TD>k<SUB>kill,<I>Pp</I></SUB></TD><TD>s<SUP>-1</SUP></TD><TD>0</TD><TD>Assuming no effects</TD></TR><TR><TD>Transfer coefficient through the membrane</TD><TD>K</TD><TD>s<SUP>-1</SUP></TD><TD>1</TD><TD>Arbitrary value</TD></TR><TR><TD>Number of <I>als</I> gene per cell</TD><TD>als<SUB>DNA,0</SUB></TD><TD>Nb/cell</TD><TD>15</TD><TD>Considering a low copy plasmid <SUP><A href="https://www.qiagen.com/dk/resources/technologies/plasmid-resource-center/growth%20of%20bacterial%20cultures/" alt="">6</A></SUP></TD></TR><TR><TD>Number of <I>AMP</I> gene per cell</TD><TD>AMP<SUB>DNA,0</SUB></TD><TD>Nb/cell</TD><TD>1</TD><TD>Protocol: genomic integration</TD></TR><TR><TD><I>V. harveyi</I> transcription rate</TD><TD>k<SUB>transcript,<I>Vh</I></SUB></TD><TD>nt/s</TD><TD>30</TD><TD>Molecular Biology course, Transcription - Faculté des Sciences - Rabat <SUP><A href="http://www.fsr.ac.ma/cours/biologie/BELKADI/transcription.pdf" alt="">7</A></SUP></TD></TR><TR><TD><I>Vibrio harveyi</I> transcription rate</TD><TD>k<SUB>translation,<I>Vh</I></SUB></TD><TD>nt/s</TD><TD>15</TD><TD>Molecular Biology course, Translation - Faculté des Sciences - Rabat <SUP><A href="http://www.fsr.ac.ma/cours/biologie/BELKADI/traduction.pdf" alt="">8</A></SUP></TD></TR><TR><TD><I>als</I> gene promoter influence</TD><TD>k<SUB>P,<I>als</I></SUB></TD><TD>/</TD><TD>1</TD><TD>Inductible promoter</TD></TR><TR><TD><I>AMP</I> gene promoter influence</TD><TD>k<SUB>P,<I>AMP</I></SUB></TD><TD>/</TD><TD>1</TD><TD>Inductible promoter</TD></TR><TR><TD>mRNA degradation constant</TD><TD>K<SUB>deg,mRNA</SUB></TD><TD>s<SUP>-1</SUP></TD><TD>5.10<SUP>-3</SUP></TD><TD>Esquerré <I>et al.</I>, 2015 <A href="https://www.ncbi.nlm.nih.gov/pubmed/25887031" alt="">9</A></TD></TR><TR><TD><I>als</I> gene length</TD><TD>DNA length</TD><TD>nucleotides</TD><TD>1662</TD><TD>UniProtKB - Q7DAV2 <SUP><A href="http://www.uniprot.org/uniprot/Q7DAV2">10</A></SUP></TD></TR><TR><TD><I>als</I> mRNA length</TD><TD>RNA length</TD><TD>nucleotides</TD><TD>1730</TD><TD>Parts design</TD></TR><TR><TD>Number of CqsS* receptor per cell</TD><TD>CqsS*/cell</TD><TD>Nb/cell</TD><TD>10<SUP>15</SUP></TD><TD>Arbitrary value</TD></TR><TR><TD>Number of Odr10 receptor per cell</TD><TD>Odr10/cell</TD><TD>Nb/cell</TD><TD>10<SUP>15</SUP></TD><TD>Arbitrary value</TD></TR><TR><TD><I>Pichia pastoris</I> transcription rate</TD><TD>k<SUB>transcript,<I>Pp</I></SUB></TD><TD>nt/s</TD><TD>50</TD><TD>Molecular Biology course - INSA Toulouse(11)</TD></TR><TR><TD><I>Pichia pastoris</I> translation rate</TD><TD>k<SUB>translation,<I>Pp</I></SUB></TD><TD>nt/s</TD><TD>48</TD><TD>Molecular Biology course, Translation - Faculté des Sciences - Rabat <SUP><A href="http://www.fsr.ac.ma/cours/biologie/BELKADI/traduction.pdf" alt="">8</A></SUP></TD></TR><TR><TD><I>AMP</I> gene length</TD><TD>DNA length</TD><TD>nucleotides</TD><TD>360</TD><TD>Parts design</TD></TR><TR><TD><I>AMP</I> mRNA length</TD><TD>RNA length</TD><TD>nucleotides</TD><TD>651</TD><TD>Parts design</TD></TR><TR><TD><I>Vibrio cholerae</I> minimal pathogenic concentration</TD><TD>[<I>Vc</I>]<SUB>pathogenic</SUB></TD><TD>cell/L</TD><TD>4.10<SUP>4</SUP></TD><TD>Medical Microbiology 4th edition (12)</TD></TR><TR><TD>CAI-1 initial concentration</TD><TD>[CAI-1]<SUB>0</SUB></TD><TD>mol/L</TD><TD>1.10<SUP>-5</SUP></TD><TD>Ng <I>et al.</I>, 2011 <SUP><A href="https://www.ncbi.nlm.nih.gov/pubmed/21219472" alt="">13</A></SUP></TD></TR><TR><TD>Michaelis-Menten constant of acetolactate synthase</TD><TD>K<SUB>M,ALS</SUB></TD><TD>mol/L</TD><TD>1.36.10<SUP>-2</SUP></TD><TD>Atsumi <I>et al.</I>, 2009 <SUP><A href="https://www.ncbi.nlm.nih.gov/pubmed/212194722" alt="">14</A></SUP></TD></TR><TR><TD>Catalytic rate constant of acetolactate synthase</TD><TD>k<SUB>cat,ALS</SUB></TD><TD>s<SUP>-1</SUP></TD><TD>1.21.10<SUP>2</SUP></TD><TD>Atsumi <I>et al.</I>, 2009 <SUP><A href="https://www.ncbi.nlm.nih.gov/pubmed/21219472" alt="">14</A></SUP></TD></TR><TR><TD>Degradation constant of acetolactate synthase</TD><TD>K<SUB>deg,ALS</SUB></TD><TD>s<SUP>-1</SUP></TD><TD>0</TD><TD>Assuming a negligeable value in our conditions</TD></TR><TR><TD>Degradation constant of antimicrobial peptides</TD><TD>K<SUB>deg,AMP</SUB></TD><TD>s<SUP>-1</SUP></TD><TD>0</TD><TD>Aleinein <I>et al.</I>, 2013 <SUP><A href="https://www.ncbi.nlm.nih.gov/pubmed/24166764" alt="">15</A></SUP></TD></TR><TR><TD>Activation constant of CqsS*-CAI-1 complex</TD><TD>K<SUB>a,CqsS*-CAI-1</SUB></TD><TD>mol/L</TD><TD>3.6.10<SUP>-8</SUP></TD><TD>Ng <I>et al.</I>, 2011 <SUP><A href="https://www.ncbi.nlm.nih.gov/pubmed/21219472" alt="">13</A></SUP></TD></TR><TR><TD>High <I>Vibrio cholerae</I> concentration in water</TD><TD>[<I>Vc</I>]<SUB>0</SUB></TD><TD>cell/L</TD><TD>10<SUP>7</SUP></TD><TD>Huq <I>et al.</I>, 1984 <SUP><A href="https://www.ncbi.nlm.nih.gov/pubmed/6486784" alt="">16</A></SUP></TD></TR><TR><TD>Dry weight of a bacterial cell</TD><TD>DW<SUB>Vh</SUB></TD><TD>pg/cell</TD><TD>0.28</TD><TD>BioNumbers ID 100008</TD></TR><TR><TD>Volume of a bacterial cell</TD><TD>V<SUB>intra,<I>Vh</I></SUB></TD><TD>μm<SUP>3</SUP></TD><TD>1</TD><TD>BioNumbers ID 100004</TD></TR><TR><TD>Volume of a yeast cell</TD><TD>V<SUB>intra,<I>Pp</I></SUB></TD><TD>μm<SUP>3</SUP></TD><TD>66</TD><TD>BioNumbers ID 100452</TD></TR><TR><TD>Dry weight of a yeast cell</TD><TD>DW<SUB>Pp</SUB></TD><TD>pg/cell</TD><TD>18.48</TD><TD>Estimation from yeast cell volume and ratio dry weight/volume for a bacterial cell</TD></TR><TR><TD>Ribosome density on a yeast cell</TD><TD>Nb/kb</TD><TD>6.5</TD><TD>BioNumbers ID 103026</TD></TR><TR><TD>Number of ribosome on <I>AMP</I> mRNA</TD><TD>Ribosome/RNA</TD><TD>Nb/RNA</TD><TD>2.34</TD><TD>Deduced from ribosome density and mRNA length</TD></TR><TR><TD>RNA polymerase density on a yeast gene</TD><TD>Nb/kb</TD><TD>6.5</TD><TD>BioNumbers ID 103026</TD></TR><TR><TD>Number of RNA polymerase on <I>AMP</I> DNA</TD><TD>RNA polymerase/DNA</TD><TD>Nb/DNA</TD><TD>4.95</TD><TD>BioNumbers ID 108308</TD></TR><TR><TD>Ribosome density on a bacterial mRNA</TD><TD>Nb/kb</TD><TD>6.6</TD><TD>BioNumbers ID 107727</TD></TR><TR><TD>Number of ribosome on <I>als</I> mRNA</TD><TD>Ribosomes/RNA</TD><TD>Nb/RNA</TD><TD>11</TD><TD>Deduced from ribosome density and mRNA length</TD></TR><TR><TD>RNA polymerase density on a bacterial gene</TD><TD>Nb/kb</TD><TD>7.6</TD><TD>Assuming the same density than yeast (BioNumbers ID 108308)</TD></TR><TR><TD>Number of RNA polymerase on <I>als</I> DNA</TD><TD>RNA Polymerase/DNA</TD><TD>Nb/DNA</TD><TD>13</TD><TD>Deduced from RNA polymerase density and DNA length</TD></TR><TR><TD>Pyruvate concentration in a bacterial cell</TD><TD>[S]</TD><TD>mol/L</TD><TD>3.9.10<SUP>-4</SUP></TD><TD>BioNumbers ID 101192</TD></TR></TBODY></TABLE><P>Some preliminary experimental results were also expoited to refine important parameters of the model and verify some of our hypothesis. Indeed, we needed an experimental estimation of the growth rate and lag time of the two chassis microorganisms grown on a common medium, and these parameters were measured experimentally. The membrane diffusion tests we have performed also revealed a rapid molecular diffusion (equilibrium reached in less than one minute). This result confirmed the initial, arbitrary hypothesis of a high transfer coefficient (K) which would not slow down the system's dynamics. </P><TABLE style="font-size:8pt;"><TBODY><TR><TH>Name</TH><TH>Notation</TH><TH>Unit</TH><TH>Value</TH><TH>Reference</TH></TR><TR><TD><I>Vibrio harveyi</I> JMH626 maximum growth rate</TD><TD>μ<SUB>MAX,<I>Vh</I></SUB></TD><TD>s<SUP>-1</SUP></TD><TD>2.10<SUP>-4</SUP></TD><TD>Experiment - 21/06/17</TD></TR><TR><TD><I>Pichia pastoris</I> SMD1168 maximum growth rate</TD><TD>μ<SUB>MAX,<I>Pp</I></SUB></TD><TD>s<SUP>-1</SUP></TD><TD>4.10<SUP>-5</SUP></TD><TD>Experiment - 21/06/17</TD></TR><TR><TD><I>Vibrio cholerae</I> lag time</TD><TD>t<SUB>l,<I>Vc</I></SUB></TD><TD>s</TD><TD>1800</TD><TD>Experiment - 21/06/17, extrapolation from <I>V. harveyi</I> growth</TD></TR><TR><TD><I>Vibrio harveyi</I> lag time</TD><TD>t<SUB>l,<I>Pp</I></SUB></TD><TD>s</TD><TD>1800</TD><TD>Experiment - 21/06/17</TD></TR><TR><TD><I>Pichia pastoris</I> lag time</TD><TD>t<SUB>l,<I>Pp</I></SUB></TD><TD>s</TD><TD>1400</TD><TD>Experiment - 21/06/17</TD></TR></TBODY></TABLE></SECTION><SECTION><H1>Solver</H1><P>The system of ODEs was solved using <B>Matlab R2017a</B>, thanks to the free offer from iGEM. We used the <I>ode15s</I> solver, which was in this situation more efficient than the <I>ode45</I> solver, very likely because the different time scales between some processes (e.g. slow growth - in the range of hours - vs fast signalling - in the range of milliseconds -) renders the problem partially stiff.<SUP><A href="https://fr.mathworks.com/help/matlab/math/choose-an-ode-solver.html">17</A></SUP> The proposed implementation proved to be very efficient, with simulations performed in less than 1 second.</P><P>You can freely re-use our code: <A href="https://static.igem.org/mediawiki/2017/e/ec/T--INSA-UPS_France--iGEM_INSA-UPS_France_2017_Model.zip" alt="">General_resolution + System_of_ODEs</A>.</P></SECTION><SECTION><H1>Simulation results</H1><P>At the beginning of the project, we needed to know if our microbial synthetic consortium would work in practice and if the information transmission between the different modules was possible and sufficiently fast. We thus carried out simulations by solving the ODEs system to have a first estimation of the dynamics of our synthetic system.</P><P>The initial conditions, such as the concentrations of <I>Vibrio harveyi</I> and <I>Pichia pastoris</I> in the device and the device volume, were set to biologically plausible values. <I>Vibrio cholerae</I> initial concentration was set to 4.10<SUP>7</SUP> cell/L,
 which is in the range of concentrations observed in contaminated water. <A href="https://www.ncbi.nlm.nih.gov/pubmed/6486784" alt="">16</A></P><UL><LI>[Vh]<SUB>0,D</SUB> = 10<SUP>12</SUP> cell/L (OD<SUB>600nm</SUB>≈ 1.5 <A href="http://www.genomics.agilent.com/biocalculators/calcODBacterial.jsp">18</A>)
</LI><LI>[Pp]<SUB>0,D</SUB> = 10<SUP>12</SUP> cell/L (OD<SUB>600nm</SUB>≈ 1.5 <A href="http://www.genomics.agilent.com/biocalculators/calcODBacterial.jsp">18</A>)
</LI><LI>[Vc]<SUB>0,W</SUB> = 4.10<SUP>7</SUP> cell/L
</LI><LI>V<SUB>D</SUB> = 20.10<SUP>-3</SUP> L
</LI></UL><DIV class="section-inside"><FIGURE><FIGCAPTION>Temporal evolution of <I>Vibrio cholerae</I> concentration under realistic conditions (device size, initial concentrations of other microorganisms in the device)</FIGCAPTION></FIGURE><P><B>The response time to reach a non-pathogenic concentration is estimated to 53.6 min</B>. This result confirmed the <B>feasibility</B> of our project: our microbial consortium is predicted to efficiently and rapidly sense the presence of <I>V. cholerae</I>, transmit this information to <I>P. Pastoris</I> which, in response, may produce enough antimicrobial peptides to kill <I>V. cholerae</I> in less than one hour. </P></DIV><DIV class="section-inside"><FIGURE class="float-figure"><FIGCAPTION>Evolution of the concentration of each entity considered in the model</FIGCAPTION></FIGURE><P>This visual representation of the system's dynamics also allowed us to check that each variable evolves in a realistic range of concentrations, hence indicating the model predicts a consistent behavior. Further analyses were then perfomed to better understand the system functioning, and in particular to evaluate its sensibility and robustness. We first tested whether the synthetic system would be robust to the variation of some parameters that could arise during the manufacturing or transport processes. Then, we identified parameters that control the response time (time to reach a non-pathogenic concentration) to guide the rational design of the system and of the device.
</P><H2><B>→</B> Sensitivity analysis methods and their results are described in <A href="https://2017.igem.org/Team:INSA-UPS_France/Model/Analysis">our model analysis page</A></H2></DIV></SECTION><SECTION><H1>References</H1><OL><LI>(Milo R, Jorgensen P, Moran U, Weber G, Springer M. BioNumbers—the database of key numbers in molecular and cell biology. <I>Nucleic Acids Res.</I> 2010;38(suppl 1):D750–D753 <A href="https://www.ncbi.nlm.nih.gov/pubmed/19854939" alt="">https://www.ncbi.nlm.nih.gov/pubmed/19854939</A></LI><LI>Pata S, Yaraksa N, Daduang S, Temsiripong Y, Svasti J, Araki T, Thammasirirak S, Characterization of the novel antibacterial peptide Leucrocin from crocodile (<I>Crocodylus siamensis</I>) white blood cell extracts. <I>Dev. Comp. Immunol.</I> 2011, 35, 545–553 <A href="https://www.ncbi.nlm.nih.gov/pubmed/21184776" alt="">https://www.ncbi.nlm.nih.gov/pubmed/21184776</A></LI><LI>Farrag A, Elsehrawi H, Younes A, Synthesis, Antimicrobial Evaluation and Docking Study of Novel Heterocyclic
Compounds Bearing a Biologically Active Sulfonamide Moiety. <I>J. Pharm. Appl. Chem.</I> 2015, 1, No. 1, 27-35 <A href="http://www.naturalspublishing.com/files/published/23lc4s4b389433.pdf" alt="">http://www.naturalspublishing.com/files/published/23lc4s4b389433.pdf</A></LI><LI>Prajanban B, Jangpromma N, Araki T, Klaynongsruang S, Antimicrobial effects of novel peptides cOT2 and sOT2 derived from <I>Crocodylus siamensis</I> and <I>Pelodiscus sinensis</I> ovotransferrins. <I>Biochimica et Biophysica Acta 1859</I>. 2017, 860–869 <A href="https://www.ncbi.nlm.nih.gov/pubmed/28159460" alt="">https://www.ncbi.nlm.nih.gov/pubmed/28159460</A></LI><LI>Yaraksa N, Anunthawan T, Theansungnoen T, Daduang S, Araki T, Apisak D, Sompong T, Design and synthesis of cationic antibacterial peptide based on Leucrocin I sequence, antibacterial peptide from crocodile (<I>Crocodylus siamensis</I>) white blood cell extracts. <I>Journal of Antibiotics</I>. Mar 2014, Tokyo 67.: 205-212.<A href="https://www.ncbi.nlm.nih.gov/pubmed/24192554" alt="">https://www.ncbi.nlm.nih.gov/pubmed/24192554</A></LI><LI>QIAGEN, Origins of replication and copy numbers of various plasmids and cosmids <B>In:</B><I>Growth Of Bacterial Cultures</I>, 2013 - 2017. <A href="https://www.qiagen.com/dk/resources/technologies/plasmid-resource-center/growth%20of%20bacterial%20cultures/">https://www.qiagen.com/dk/resources/technologies/plasmid-resource-center/growth%20of%20bacterial%20cultures/</A></LI><LI>Belkadi B, Expression génétique - Transcription <B>In:</B><I>Biologie Moléculaire</I>, Faculté des Sciences - Rabat, 2010-2011. <A href="http://www.fsr.ac.ma/cours/biologie/BELKADI/transcription.pdf" alt="">http://www.fsr.ac.ma/cours/biologie/BELKADI/transcription.pdf</A></LI><LI>Belkadi B, Expression génétique - Traduction <B>In:</B><I>Biologie Moléculaire</I>, Faculté des Sciences - Rabat, 2009-2010. <A href="http://www.fsr.ac.ma/cours/biologie/BELKADI/traduction.pdf" alt="">http://www.fsr.ac.ma/cours/biologie/BELKADI/traduction.pdf</A></LI><LI>Esquerre T, Moisan A, Chiapello H, Arike L, Vilu R, Gaspin C, Cocaign-Bousquet M, Girbal L, Genome-wide investigation of mRNA lifetime determinants in <I>Escherichia coli</I> cells cultured at different growth rates. <I>BMC Genomics</I>. 2015, 16, 275,<A href="https://www.ncbi.nlm.nih.gov/pubmed/25887031" alt="">https://www.ncbi.nlm.nih.gov/pubmed/25887031</A></LI><LI>UNIPROT, Q7DAV2, Alpha-acetolactate synthase - <I>Lactococcus lactis</I> subsp. lactis (strain IL1403) : <A href="http://www.uniprot.org/uniprot/Q7DAV2">http://www.uniprot.org/uniprot/Q7DAV2</A></LI><LI>Capp JP, Chap.3 : Transcription <B>In:</B><I>Cours de Biologie Moléculaire</I>, INSA Toulouse, 2016</LI><LI>Baron S, editor. <I>Medical Microbiology</I>. 4th edition. Galveston (TX): University of Texas Medical Branch at Galveston; 1996.
</LI><LI>Ng WL, Perez LJ, Wei Y, Kraml C, Semmelhack MF, Bassler BL, Signal production and detection specificity in <I>Vibrio</I> CqsA/CqsS quorum-sensing systems. <I>Mol. Microbiol.</I> 2011, 79: 1407–1417. <A href="https://www.ncbi.nlm.nih.gov/pubmed/21219472" alt="">https://www.ncbi.nlm.nih.gov/pubmed/21219472</A></LI><LI>Atsumi S, Li Z, Liao JC, Acetolactate synthase from <I>Bacillus subtilis</I> serves as a 2-ketoisovalerate decarboxylase for isobutanol biosynthesis in Escherichia coli. <I>Appl. Environ. Microbiol.</I> 2009a, 75:6306–6311 <A href="https://www.ncbi.nlm.nih.gov/pubmed/21219472" alt="">https://www.ncbi.nlm.nih.gov/pubmed/21219472</A></LI><LI>Aleinein R.A., Schäfer H., Wink M., Secretory ranalexin produced in recombinant <I>Pichia pastoris</I> exhibits additive bactericidal activity when used in combination with polymyxin B or linezolid against multi-drug resistant bacteria. <I>Biotechnol. J.</I> 2014, 9, 110–119. <A href="https://www.ncbi.nlm.nih.gov/pubmed/24166764" alt="">https://www.ncbi.nlm.nih.gov/pubmed/24166764</A></LI><LI>Huq A., West, P. A., Small, E. B., Huq, M. I., and Colwell, R. R., Influence of water temperature, salinity and pH on survival and growth of toxigenic <I>Vibrio cholerae</I> serovar O1 associated with live copepods in laboratory microcosms, <I>Appl. Environ. Microbiol.</I> 1984, 48: 420–424. <A href="https://www.ncbi.nlm.nih.gov/pubmed/6486784" alt="">https://www.ncbi.nlm.nih.gov/pubmed/6486784</A></LI><LI>MATHWORKS, Basic Solver Selection <B>In:</B><I>Choose an ODE Solver</I>: <A href="https://fr.mathworks.com/help/matlab/math/choose-an-ode-solver.html">https://fr.mathworks.com/help/matlab/math/choose-an-ode-solver.html</A></LI><LI>AGILENT. <I>E. coli</I> Cell Culture Concentration from OD600 Calculator <A href="http://www.genomics.agilent.com/biocalculators/calcODBacterial.jsp">http://www.genomics.agilent.com/biocalculators/calcODBacterial.jsp</A></LI></OL></SECTION></DIV><SECTION class="links_end"><TABLE><TBODY><TR><TH colspan="4">
            Model pages
          </TH></TR><TR><TD><A href="https://2017.igem.org/Team:INSA-UPS_France/Model">Overview</A></TD><TD><I>Simulation</I></TD><TD><A href="https://2017.igem.org/Team:INSA-UPS_France/Model/Analysis">Analysis</A></TD><TD><A href="https://2017.igem.org/Team:INSA-UPS_France/Model/Interface">Interface</A></TD></TR></TBODY></TABLE></SECTION></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>