Team:INSA-UPS France/Model/ODE
Croc'n Cholera
iGEM UPS-INSA Toulouse 2017
Team
Members
Supports
Attributions
Collaborations
Strategy
Description
Design
Parts
Model
Overview
Simulation
Analysis
Interface
Realisations
Overview
Notebook
Clonings
Results
Contribution
Protocols
Safety
Human Practices
Overview
Silver
Gold
Public Engagement
Microbioworld
Application
Applied Design
Testimonies
Entrepreneurship
Device Conception
System of ODEs
Equations
The dynamics of our microbial consortium was summed up in thirteen differential equations. Every biological or physical process was described mathematically, and was gathered to constitute our system of ODEs. For fearless people, our complete mathematical model and demonstration can be found there !
We need to characterize the growth and death of the three microorganisms: Vibrio cholerae (Vc) in water (W), Vibrio harveyi (Vh) and Pichia pastoris (Pp) in the device (D). \begin{equation}     \frac{d[\textit{Vc}]_W}{dt} = V_{growth,Vc} - V_{death,Vc} \end{equation} \begin{equation}     \frac{d[\textit{Vh}]_D}{dt} = V_{growth,Vh} - V_{death,Vh} \end{equation} \begin{equation}     \frac{d[\textit{Pp}]_D}{dt} = V_{growth,Pp} - V_{death,Pp} \end{equation}
Microorganisms death is impacted by antimicrobial peptides production(AMPpeptide,Pp), produced by translation of antimicrobial peptides mRNA (AMPRNA). Peptides and mRNA are also degraded. \begin{equation}     \frac{d[AMP_{RNA}]_{Pp}}{dt}=V_{transcription,AMP} - V_{degradation,AMP RNA} \end{equation}  \begin{equation}     \frac{d[AMP_{peptide}]_{D}}{dt}=V_{translation,AMP} - V_{degradation,AMP} + \frac{V_{diff,AMP,W \to D}}{\mathcal{V}_D} \end{equation}
These peptides are transfered from the device (D) to water (W).  \begin{equation}     \frac{d[AMP]_W}{dt} = -V_{diff,AMP,W\to D} \end{equation}
To produce antimicrobial peptides, an activation by diacetyl (dac) is needed. Diacetyl can freely diffuse from the device (D) to water (W). \begin{equation}     \frac{d[dac]_D}{dt}=V_{prod,dac}+\frac{V_{diff,dac,W \to D}}{\mathcal{V}_D} \end{equation}  \begin{equation}     \frac{d[dac]_W}{dt}=- V_{diff,dac,W \to D} \end{equation}
Diacetyl is produced by the enzyme acetolactate synthase (ALSenzyme). Als gene is first transcribed into ALSRNA, which is then translated into the protein. Both the enzyme and its mRNA can be degraded. \begin{equation}     \frac{d[ALS_{RNA}]_{Vh}}{dt} = V_{transcription,ALS} - V_{degradation,ALS RNA} \end{equation}  \begin{equation}     \frac{d[ALS_{enzyme}]_{Vh}}{dt} = V_{translation,ALS} - V_{degradation,ALSenzyme} \end{equation}
ALS production has to be activated by the quorum sensing molecule CAI-1, initially in water (W), after diffusing into the device (D).   \begin{equation}     \frac{d[CAI\text{-}1]_D}{dt} = \frac{V_{diff,CAI\text{-}1,W\to D}}{\mathcal{V}_D} \end{equation}  \begin{equation}     \frac{d[CAI\text{-}1]_W}{dt} = -V_{diff,CAI\text{-}1,W\to D} \end{equation}
Data
Our model is mostly set by data from publications, because the majority of the required data necessitates complex experiments we could not perform during our project.
Name
Notation
Unit
Value
Reference
Vibrio cholerae maximum growth rate
μMAX,Vc
s-1
3.10-4
BioNumbers 112369 (1)
Leucrocine I MIC for V. cholerae
MICLeucro,Vc
mol/L
6.4.10-5
Pata et al., 2011 (2)
Leucrocine I MIC for V. harveyi
MICLeucro,Vh
mol/L
6.4.10-5
Pata et al., 2011 (2) - Extrapolation from V. cholerae result
Leucrocine I MIC for P. pastoris
MICLeucro,Pp
mol/L
∞
Assuming no effects on Pichia pastoris
Leucrocine I IC50 for V. cholerae
MICLeucro,Vc
mol/L
1.92.10-4
Considering IC50 = 3.MIC - Extrapolation of data from standard antibiotics (Farrag et al., 2015)
Leucrocine I IC50 for V. harveyi
IC50Leucro,Vh
mol/L
1.92.10-4
Considering IC50 = 3.MIC - Extrapolation of data from standard antibiotics (Farrag et al., 2015 (3))
Leucrocine I IC50 for P. pastoris
IC50Leucro,Pp
mol/L
∞
Assuming no effects on Pichia pastoris
cOT2 MIC for V. cholerae
MICcOT2,Vc
mol/L
8.1.10-6
Prajanban et al., 2017 (4)
cOT2 MIC for V. harveyi
MICcOT2,Vh
mol/L
8.1.10-6
Prajanban et al., 2017 (4) - Extrapolation from V. cholerae result
cOT2 MIC for P. pastoris
MICcOT2,Pp
mol/L
∞
Assuming no effects on Pichia pastoris
cOT2 IC50 for V. cholerae
IC50cOT2,Vc
mol/L
2.43.10-5
Considering IC50 = 3.MIC - Extrapolation of data from standard antibiotics (Farrag et al., 2015)
cOT2 IC50 for V. harveyi
IC50cOT2,Vh
mol/L
2.43.10-5
Considering IC50 = 3.MIC - Extrapolation of data from standard antibiotics (Farrag et al., 2015)
cOT2 IC50 for P. pastoris
IC50cOT2,Pp
mol/L
∞
Assuming no effects on Pichia pastoris
D-NY15 MIC for V. cholerae
MICD-NY15,Vc
mol/L
1.54.10-5
Yaraksa et al., 2014 (5)
D-NY15 MIC for V. harveyi
MICD-NY15,Vh
mol/L
1.54.10-5
Yaraksa et al., 2014 (5) - Extrapolation from V. cholerae result
D-NY15 MIC for P. pastoris
MICD-NY15,Pp
mol/L
∞
Assuming no effects on Pichia pastoris
D-NY15 IC50 for V. cholerae
IC50D-NY15,Vc
mol/L
4.62.10-5
Considering IC50 = 3.MIC - Extrapolation of data from standard antibiotics (Farrag et al., 2015)
D-NY15 IC50 for V. harveyi
IC50D-NY15,Vh
mol/L
4.62.10-5
Considering IC50 = 3.MIC - Extrapolation of data from standard antibiotics (Farrag et al., 2015)
D-NY15 IC50 for P. pastoris
IC50D-NY15,Pp
mol/L
∞
Assuming no effects on Pichia pastoris
V. cholerae death rate with AMP
kkill,Vc
s-1
3.10-3
Yaraksa et al., 2014 (5)
V. harveyi death rate with AMP
kkill,Vh
s-1
3.10-3
Extrapolation from Yaraksa et al., 2014 (5)
P. pastoris death rate with AMP
kkill,Pp
s-1
0
Assuming no effects
Transfer coefficient through the membrane
K
s-1
1
Arbitrary value
Number of als gene per cell
alsDNA,0
Nb/cell
15
Considering a low copy plasmid (6)
Number of AMP gene per cell
AMPDNA,0
Nb/cell
1
Protocol: genomic integration
V. harveyi transcription rate
ktranscript,Vh
nt/s
30
Molecular Biology course, Transcription - Faculté des Sciences - Rabat (7)
Vibrio harveyi transcription rate
ktranslation,Vh
nt/s
15
Molecular Biology course, Translation - Faculté des Sciences - Rabat (8)
als gene promoter influence
kP,als
/
1
Inductible promoter
AMP gene promoter influence
kP,AMP
/
1
Inductible promoter
mRNA degradation constant
Kdeg,mRNA
s-1
5.10-3
Esquerré 2015 (9)
als gene length
DNA length
nucleotides
1662
UniProtKB - Q7DAV2 (10)
als mRNA length
RNA length
nucleotides
1730
Parts design
Number of CqsS* receptor per cell
CqsS*/cell
Nb/cell
1015
Arbitrary value
Number of Odr10 receptor per cell
Odr10/cell
Nb/cell
1015
Arbitrary value
Pichia pastoris transcription rate
ktranscript,Pp
nt/s
50
Molecular Biology course - INSA Toulouse
Pichia pastoris translation rate
ktranslation,Pp
nt/s
48
Molecular Biology course, Translation - Faculté des Sciences - Rabat (8)
AMP gene length
DNA length
nucleotides
360
Parts design
AMP mRNA length
RNA length
nucleotides
651
Parts design
Vibrio cholerae minimal pathogenic concentration
[Vc]pathogenic
cell/L
4.104
Medical Microbiology 4th edition (11)
CAI-1 initial concentration
[CAI-1]0
mol/L
1.10-5
Ng et al., 2011 (12)
Michaelis-Menten constant of acetolactate synthase
KM,ALS
mol/L
1.36.10-2
Atsumi et al., 2009 (13)
Catalytic rate constant of acetolactate synthase
kcat,ALS
s-1
1.21.102
Atsumi et al., 2009 (13)
Degradation constant of acetolactate synthase
Kdeg,ALS
s-1
0
Assuming a negligeable value in our conditions
Degradation constant of antimicrobial peptides
Kdeg,AMP
s-1
0
Aleinein et al., 2013 (14)
Activation constant of CqsS*-CAI-1 complex
Ka,CqsS*-CAI-1
mol/L
3.6.10-8
Ng et al., 2011 (12)
High Vibrio cholerae concentration in water
[Vc]0
cell/L
1.107
Huq et al., 1984 (15)
Dry weight of a bacterial cell
DWVh
pg/cell
0.28
BioNumbers 100008
Volume of a bacterial cell
Vintra,Vh
μm3
1
BioNumbers 100004
Volume of a yeast cell
Vintra,Pp
μm3
66
BioNumbers 100452
Dry weight of a yeast cell
DWPp
pg/cell
18.48
Estimation from yeast cell volume and ratio dry weight/volume for a bacterial cell
Ribosome density on a yeast cell
Nb/kb
6.5
BioNumbers 103026
Number of ribosome on AMP mRNA
Ribosome/RNA
Nb/RNA
2.34
Deduced from ribosome density and mRNA length
RNA polymerase density on a yeast gene
Nb/kb
6.5
BioNumbers 103026
Number of RNA polymerase on AMP DNA
RNA polymerase/DNA
Nb/DNA
4.95
BioNumbers 108308
Ribosome density on a bacterial mRNA
Nb/kb
6.6
BioNumbers 107727
Number of ribosome on als mRNA
Ribosomes/RNA
Nb/RNA
11
Deduced from ribosome density and mRNA length
RNA polymerase density on a bacterial gene
Nb/kb
7.6
Assuming the same density than yeast (BioNumbers 108308)
Number of RNA polymerase on als DNA
RNA Polymerase/DNA
Nb/DNA
13
Deduced from RNA polymerase density and DNA length
Pyruvate concentration in a bacterial cell
[S]
mol/L
3.9.10-4
BioNumbers 101192
Even if we mostly describe our model as a predictive one, some preliminary experimental results could have been implemented into our model. Indeed, an experimental estimation of our two chassis behaviour on a common medium was needed.
Name
Notation
Unit
Value
Reference
Vibrio harveyi JMH626 maximum growth rate
μMAX,Vh
s-1
2.10-4
Experiment - 21/06/17
Pichia pastoris SMD1168 maximum growth rate
μMAX,Pp
s-1
4.10-5
Experiment - 21/06/17
Vibrio cholerae lag time
tl,Vc
s
1800
Experiment - 21/06/17, extrapolation from V. harveyi growth
Vibrio harveyi lag time
tl,Pp
s
1800
Experiment - 21/06/17
Pichia pastoris lag time
tl,Pp
s
14 400
Experiment - 21/06/17
Solver
The system of ODEs was solved using Matlab R2017a, thanks to the free offer from iGEM. Initially set with ode45 solver, the recommanded Matlab ODE solver, the final script uses ode15s because ode45 was not enough efficient. ode15s is the solver recommanded when having problems or inefficiency with ode45, and is adapted to stiff problems.(16)
You can freely re-use our code: General_resolution + System_of_ODEs .
References
(6) : http://bitesizebio.com/22824/how-to-manipulate-plasmid-copy-number/
(16) : https://fr.mathworks.com/help/matlab/math/choose-an-ode-solver.html
