Team:INSA-UPS France/Protocols
Croc'n Cholera
iGEM UPS-INSA Toulouse 2017
Team
Members
Supports
Attributions
Collaborations
Strategy
Description
Design
Parts
Model
Overview
Simulation
Analysis
Interface
Realisations
Overview
Notebook
Clonings
Results
Contribution
Protocols
Safety
Human Practices
Overview
Silver
Gold
Public Engagement
Microbioworld
Application
Applied Design
Testimonies
Entrepreneurship
Device Conception
Protocols
Medium and antibiotics
Introduction
Here are listed antibiotics concentration and media recipe used during the experiments.
LB medium
Tryptone
10 g/L
Yeast extract
5 g/L
NaCl
10 g/L
Water
Up to 1 L
For solid medium, add 15 g/L of agar.  Medium need to be autoclaved before use.
LM medium
Tryptone
10 g/L
Yeast extract
5 g/L
NaCl
20 g/L
Water
Up to 1 L
For solid medium, add 15 g/L of agar.  Medium need to be autoclaved before use.
M9 medium
5X Salts
For 1 L of final solution
[Final] in M9
Na2, H2PO4, 12 H2O
90 g
18 g/L
KH2PO4
15.65 g
3.03 g/L
NaCl
2.5 g
0.5 g/L
NH4Cl
10.55 g
2.11 g/L
MgSO4 1M
For 50 mL of final solution
[Final] in M9
MgSO4
12.3 g
0.49 g/L
CaCl2 0.01M
For 50 ùL of final solution
[Final] in M9
CaCl2
0.073 g
4.38 mg/L
1000X Salts
For 100 mL of final solution
[Final] in M9
Na2EDTA, 2 H2O
1.5 g
15 mg/L
ZnSO4, 7 H2O
0.45 g
4.5 mg/L
CoCl2, 6 H2O
0.03 g
0.3 mg/L
MnCl2, 4 H2O
1 g
10 mg/L
H3BO H3
0.1 g
1 mg/L
Na2MoO H4, 2 H2O
0.04 g
0.4 mg/L
FeSO4, 7 H2O
0.3 g
3 mg/L
CuSO4, 5 H2O
0.03 g
0.3 mg/L
EDTA and ZnSO4 are dissolved in 80 mL of mQ water and pH is adjusted to 6. Other compound are added and pH is maintained to 6. Once all compounds are dissolved, water is adjusted to 100 mL and pH to 4. Solution is filtered on 0.2 µm and stored at -4 ° C
100X thiamine
For 10 mL of final solution
[Final] in M9
Hypochloride thiamine>
0.1 g
0.1 g/L
pH is adjusted to 2 with HCl, solution is filtered (0.2 µm) and stored at -4 ° C. this product is light sensitive.
For 1 L of M9 media, all the following recipe are mixed together under sterile condition.
Solution
Sterilisation
Volume
5X salts
autoclave
200 mL
MgSO4 1M
autoclave
2 mL
CaCl2 0.01M
autoclave
3 mL
1000X Salts
filtration (0.2 µm)
1 mL
100X thiamine
filtration (0.2 µm)
10 mL
Carbon source 40X
filtration (0.2 µm)
25 mL
Water
autoclave
759 mL
YPB medium
Baceriological peptone
20 g/L
Yeast extract
10 g/L
Glucose
20 g/L
Water
Up to 1 L
For solid medium, add 15 g/L of agar.  Medium need to be autoclaved before use. Glucose is added after autoclave.
Complete Minimal Medium + glutamine
For CMM 2X
YNB without amino acid
50 mL
Glucose 10%
100 mL
Adenine 1 mg/mL
10 mL
Histidine 10 mg/mL
1 mL
Leucine 10 mg/mL
1 lL
Isoleucine 10 mg/mL
1 mL
Lysine 10 mg/mL
1.5 mL
Methionine 10 mg/mL
1 mL
Phenylalanine 10 mg/mL
2.5 mL
Tryptophane 5 mg/mL
2 mL
Tyrosine 0.5 mg/mL
30 mL
Uracile 2 mg/mL
5 mL
Water
45 mL
For CMM glutamine:
CMM 2X
75 mL
Glutamine 2%
15 mL
Water
60 mL
Antibiotics
Antibiotics are prepared as stock solution of 1000X to facilitate further utilization.
Materials
Antibiotic powders (usually stocked as CMR products or at 4°C)
Solvents (pipettes + falcon tube)
Filtration kits
Weighing instrument
Steril Eppendorf tubes
Procedure :
Under a safety cabinet, weight 0,5g of antibiotic powder and transfer it in a 50 mL falcon tube.
Add 10 mL of the appropriate solvent, mix briefly.
Under PSM, sterilize the antibiotics by filtration and distribute in sterile eppendorf tubes. Fill the seringue before using the filtration membrane.
Annotate the tubes and store at -20°C in the appropriate box.
Dilute 1000x the stock solution in your media to get the fine working concentration. (5μL in 5mL of LB media).
Trash treatment : containers or vessel are not biological waste : use common trash and wash the reusable materials. Diluted antibiotics can be disposed of down the drain.
Antibiotic
Abbreviation
Solvent
[1000X stock]
[Culture]
Ampicillin
Amp
water
50 mg/mL
50 µg/mL
Chloramphenicol
Cm
ethanol
25 mg/mL
25 µg/mL
Kanamycin
Kan
water
50 mg/mL
50 µg/mL
Streptomycin
Sm
water
50 mg/mL
50 µg/mL
Tetracycline
Tet
ethanol
50 mg/mL
50 µg/mL
Zeocin
Zeo
water
25 mg/mL
50 µg/mL
Cultivation conditions
E.coli
Unless specified, E.coli K12 MG 1655 was grown at 37°C at 160 rpm and 37 °C for solid media
LB medium
M9 medium
V. harveyi
Unless specified, V. harveyi BB120 and JMH626 were grown at 30°C and 160 RPM for liquid media and 30 °C for solid media.
LB medium
LM medium
P. pastoris
Unless specified, P. pastoris was grown at 30°C and 160 RPM for liquid media and 30 °C for solid media
YPD medium
CMM glutamine medium
DNA manipulation
PCR
We used the Thermo Scientific Phusion High-Fidelity DNA Polymerase . Amplification of templates with high GC content, high secondary structure, low template concentrations or long amplicons may require further optimization.
Materials
PCR thermocycler
PCR tubes
nuclease-free water
dNTP
Phusion HF Buffer (X5) or GC Buffer
Primers (both forward and reverse)
Template DNA
Phusion polymerase
Procedure
All components should be mixed and centrifuged prior to use. It is important to add Phusion DNA Polymerase last in order to prevent any primer degradation caused by the 3´→ 5´ exonuclease activity.
Phusion DNA Polymerase may be diluted in 1X HF or GC Buffer just prior to use in order to reduce pipetting errors.
Use of high quality, purified DNA templates greatly enhances the success of PCR.
We recommend assembling all reaction components on ice and quickly transferring the reactions to a thermocycler preheated to the denaturation temperature (98°C).
Component
50 μL
final concentration
Nuclease-free water
qs 50 μL
Buffer Phusion HF (5X)
10μL
1X
10 mM dNTPs
1 μL
200 μM
10 μM Forward primer
2.5 μL
0.5 μM
10 μM Reverse Primer
2.5 μL
0.5 μM
DNA template (10 ng/μL)
1 μL
10ng
Phusion DNA Polymerase
0.5 μL
1.0 U/0.5 μL of reaction
Notes: Gently mix the reaction. Collect all liquid to the bottom of the tube by a quick spin if necessary
Transfer PCR tubes from ice to a PCR machine with the block preheated to 98°C and begin thermocycling:
Step
Temperature
Time
Initial denaturation
98°C
45 sec
30 cycles
98°C
15 sec
55°C
30 sec
72°C
30 sec/kb
Final extension
72°C
5 min
Hold
4°C
hold
Parts
Length
Time of extension
pGAP-cOT2 / pGAP-DNY15 / pGAP-Leucro / YFP / DsRed
1 kb
30 sec
harveyi 1 / 2 / 3 / Vc and Vh
2 kb
60 sec
Odr10-cOT2
3 kb
90 sec
Then purify the products thanks to PCR purification kit
PCR purification
Introduction
This protocol was extracted from Invitrogen PureLink® PCR Purification Kit . Refer to this protocol for troubleshooting. Use the PureLink® PCR Purification Kit to efficiently remove primers, dNTPs, enzymes, and salts from PCR products in less than 15 minutes. Use the kit with Binding Buffer High-Cutoff (B3) to remove primer dimers or short spurious PCR products. The purified PCR product is suitable for automated fluorescent DNA sequencing, restriction enzyme digestion, and cloning.
Materials
Binding Buffer (B2)
Binding Buffer High-Cutoff (B3)
Wash Buffer (W1)
Elution Buffer; 10 mM Tris-HCl, pH 8.5 (E1)
PureLink® PCR Spin Columns with Collection Tubes
PureLink® Elution Tubes (1.7 mL)
50–100 μL PCR product
100% isopropanol
96–100% ethanol
Sterile, distilled water (pH>7.0)
Microcentrifuge capable of achieving >10,000 × g
Procedure
/!\ The PureLink® PCR Purification Kit buffers contain guanidine hydrochloride and isopropanol. Always wear a laboratory coat, disposable gloves, and eye protection when handling buffers.
/!\ Do not add bleach or acidic solutions directly to solutions containing guanidine hydrochloride or sample preparation waste because it forms reactive compounds and toxic gases when mixed with bleach or acids.
Follow the recommendations below to obtain the best results:
Maintain a PCR volume of 50–100 μL
Save an aliquot of PCR products before purification to verify and check the amplicon on the gel
Use a centrifuge at room temperature for all steps
Pipet the Elution Buffer (E1) in the center of the column and perform a 1 minute incubation
Always use sterile water with pH 7–8.5, if you are using water for elution
Before starting. Add isopropanol to the Binding Buffers and ethanol to the Wash Buffer according to the following table. After adding isopropanol or ethanol, store all buffers at room temperature.
Buffer
Cat. no. K3100-01
Binding Buffer (B2)
10mL 100% isopropranol
Binding Buffer HC (B3)
2.3mL 100% isopropranol
Wash Bufer (W1)
64mL 96-100% isopropranol
Binding DNA.
Add 4 volumes of PureLink® Binding Buffer (B2) with isopropanol (see before starting) or Binding Buffer HC (B3) with isopropanol (see before starting) to 1 volume of the PCR product (50–100 μL). Mix well.
Remove a PureLink® Spin Column in a Collection Tube from the package.
Add the sample with the appropriate Binding Buffer (from step 1 of this procedure) to the PureLink® Spin Column.
Centrifuge the column at room temperature at 10,000 × g for 1 minute.
Discard the flow through and place the spin column into the collection tube.
Washing DNA
Add 650 μL of Wash Buffer with ethanol (see before starting) to the column.
Centrifuge the column at room temperature at 10,000 × g for 1 minute. Discard the flow through from the collection tube and place the column into the tube.
Centrifuge the column at maximum speed at room temperature for 2–3 minutes to remove any residual Wash Buffer. Discard the collection tube. Then let the residual ethanol evaporate by placing the open column on the collection tube and let it sit for 5 mins.
Eluting DNA.
Place the spin column in a clean 1.7-mL PureLink® Elution Tube supplied with the kit.
Add 30 μL of Elution Buffer (10 mM Tris-HCl, pH 8.5) or sterile, distilled water (pH >7.0) to the center of the column.
Incubate the column at room temperature for 1 minute.
Centrifuge the column at maximum speed for 2 minutes.
The elution tube contains the purified PCR product. Remove and discard the column. The recovered elution volume is ~48 μL. Store the purified PCR product at –20°C or use the PCR product for the desired downstream application.
Colony PCR
Introduction
This protocol was elaborated thanks to the help of Anthony Henras.
Materials
10 μL of 0.02N NaOH / 1 PCR
Procedure
Resuspend the equivalent of the tip of a P1000 pipette of the colony in 10 μL of 0.02N NaOH
Mix well (vortex)
Incubate 5 min at 95°C and then chill on ice for 10 min at 4°C (program the thermocycler to do so (Program YeastLysis))
For each PCR mix:
NOTE: mix on ice and put on the thermocycler directly after mixing
Component
Volume (μL)
Previous cell extract
2
Taq Pol Buffer
10
Forward oligo 100 10 μM
0.5
Reverse oligo 100 10 μM
0.5
dNTP
1
H2O
35.6
Taq DNA polymerase
0.4
Put on a thermocycler and start this cycle:
95°C
5 min
35 cycles
95°C
30 sec
55°C
1 min
72°C
3 min
72°C
10 min
22°C
∞
Migration on gel to check the results
Gel extraction of DNA
Procedure
Please, before doing your preparative gel, use one sample to make an analityc one !
Equilibrate a water bath or heat block to 50°C.
Excise a minimal area of gel containing the DNA fragment of interest.
Crucial: To protect the UV box, it is a good idea to place the gel on a glass plate if available.
Try to get as little excess gel around the band as possible.
Weigh the gel slice containing the DNA fragment using a scale sensitive to 0.001 g.
Add Gel Solubilization Buffer (L3) to the excised gel in the tube size indicated in the following table:
Gel
Tube
Buffer L3 Volume
≤2% agarose
1.7 mL polypropylene
3:1 (i.e., 1.2 mL Buffer L3: 400 mg gel piece)
>2% agarose
5 mL polypropylene
6:1 (i.e., 2.4 mL Buffer L3: 400 mg gel piece)
Place the tube with the gel slice and Buffer L3 into a 50°C water bath or heat block. Incubate the tube at 50°C for 10 minutes. Invert the tube every 3 minutes to mix and ensure gel dissolution.
Note: High concentration gels (>2% agarose) or large gel slices may take longer than 10 minutes to dissolve.
After the gel slice appears dissolved, incubate the tube for an additional 5 minutes.
Optional: For optimal DNA yields, add 1 gel volume of isopropanol to the dissolved gel slice. Mix well.
Before Starting: Add ethanol to the Wash Buffer (W1) according to the label on the bottle.
Purifying DNA Using a Centrifuge
Load. Pipet the dissolved gel piece onto a Quick Gel Extraction Column inside a Wash Tube. Use 1 column per 400 mg of agarose gel.
Note: The column reservoir capacity is 850 μL.
Bind. Centrifuge the column at >12,000 × g for 1 minute. Discard the flow-through and place the column into the Wash Tube.
Wash. Add 500 μL Wash Buffer (W1) containing ethanol to the column.
Remove Buffer. Centrifuge the column at >12,000 × g for 1 minute. Discard the flow-through and place the column into the Wash Tube.
Remove Ethanol. Centrifuge the column at maximum speed for 1–2 minutes. Discard the flow-through.
Elute. Place the column into a Recovery Tube. Add 30 μL Elution Buffer (E5) to the center of the column. Incubate the tube for 1 minute at room temperature.
Collect. Centrifuge the tube at >12,000 × g for 1 minute.
Store. The elution tube contains the purified DNA. Store the purified DNA at 4°C for immediate use or at −20°C for long-term storage.
Migration on agarose gel
Introduction
This protocol is the classical one used for electrophoresis. - You can adapt the concentration of agar according to the length of your fragment 1% agar if the DNA fragments are big 2% agar if the DNA fragments are small (the bigger fragment are sticked together) - Adapt the volume of the gel 15 to 30 mL for small gels and 150 to 200 mL for big gels
Procedure
Thoroughly rinse gel housing and well-comb with dH2O.
Place gel mold perpendicular to flow direction, ensuring proper sealing of rubber gaskets.
Add the calculated amounts of 0.5xTBE and agarose to a fresh Erlenmeyer flask.
Heat in microwave until mixture can be dissolved.
CRITICAL: Do not let the mixture boil over and out of the flask. Typical heating time for 50mL in a 2.45GHz microwave oven at full power is 30s. USE HEAT GLOVES
Gently swirl until well mixed and gently swirling periodically until ~55°C.
Gently pour molten agarose gel into housing, avoiding air bubbles.
Place desired well comb in desired position.
Once gelled, carefully remove well comb in a uniform fashion.
Remove gel mold and place in parallel direction to flow
CRITICAL: the deposit line has to be at the anode (negative pole)
Fill gel box with 0.5 xTAE until the gel is well covered.
Place the ladder on the gel, the native and digested plasmid (write down the gel map)
TIP: When loading the sample in the well, maintain positive pressure on the sample to prevent bubbles or buffer from entering the tip.
Run the electrophoresis for 20-30min at 100V until the dye line is approximately 80% of the way down the gel
Turn OFF power, disconnect the electrodes from the power source, and then carefully remove the gel from the gel box.
Place the gel into a container filled with 100 mL of TAE running buffer and 5 μL of EtBr, place on a rocker for 20-30 mins, r
Place the gel into a container filled with water and destain for 5 mins.
Reaveal under UV lamp, visualize your DNA fragments
Miniprep.
Introduction
This protocol was taken from the ThermoScientific GeneJET Plasmid Miniprep Kit . Safety: Both the Lysis Solution and the Neutralization Solution contain irritants. Wear gloves when handling these solutions.
Procedure
Note: All steps should be carried out at room temperature. All centrifugations should be carried out in a microcentrifuge at ≥ 12 000 x g (10 000-14 000 rpm, depending on the rotor type).
Be sure that the concentrated solutions have been diluted with the appropriated buffer
Pick a single colony from a freshly streaked selective plate to inoculate 5mL of LB medium supplemented with the appropriate selection antibiotic.
Incubate for 12-16 hours at 37°C while shaking at 200-250 rpm
Centrifugate the bacterial culture, >12 000 g in a microcentrifuge for 2 minutes at room temperature. Repeat until there is no more media.
Add to the pelleted cells:
250 μL of Resuspension Solution and vortex
250 μL of Lysis Solution and invert the tube 4-6 times. WAIT 2 min
350 μL of Neutralization Solution and invert the tube 4-6 times.
Lysis buffer must be neutralized before 5 minutes
Centrifuge 5 minutes.
Transfer the supernatant to the Thermo Scientific GeneJET Spin Column. Centrifuge 1 minute
Add 500 μL of Wash Solution and centrifuge for 60 s and discard the flow-through
Repeat step 5.
Centrifuge empty column for 1 minute.
Dry for 5 minutes
Transfer the column into a new tube.
Add 30 μL of Elution Buffer to the column and incubate 2 minutes.
Centrifuge 2 minutes.
Collect the flow-through.
Ligation
Introduction
Please see the NEB website for supporting information on this protocol.
Materials
10X T4 DNA Ligase Reaction Buffer
T4 DNA Ligase
Vector DNA (4kb)
Insert DNA (1kb)
nuclease-free water
Procedure
Note: T4 DNA Ligase should be added last. The table shows a ligation using a molar ratio of 1:3 vector to insert for the indicated DNA sizes.  Use NEB calculator to calculate molar ratios.
Thaw the T4 DNA Ligase Buffer and resuspend at room temperature. Tip: Alicuote the 10x buffer less concentrated so when thawing, the DTT gets soluble more easily.
Set up the following reaction in a microcentrifuge tube on ice:
Component
Volume (µL)
10X T4 DNA Ligase Buffer
2
Vector DNA: 50 ng (0.020 pmol)
Insert DNA: 37.5 ng (0.060 pmol)
Nuclease-free water
17
T4 DNA Ligase
1
Total
20
Gently mix the reaction by pipetting up and down and microfuge briefly.
For cohesive (sticky) ends, incubate at 16°C overnight or room temperature for 10 minutes. For blunt ends or single base overhangs, incubate at 16°C overnight or room temperature for 2 hours.
Heat inactivate at 65 degrees C for 10 minutes.
Chill on ice and transform 1-5 μl of the reaction into 50 μl competent cells. Use 25 uL DH5α cells, and add 2 uL of reaction mixture.
Chemical transformation (RbCl method)
Introduction
This protocol was given by Stéphanie. The aim is to make yourself Top10 competent cells.
Materials
2 * Steri cup 250mL
TrisEDTA
Procedure
Media and Solutions
500 mL LB
200 mL TFB1:
0.59 g KOAc (Cf=30 mM)
2.42 g RbCl (100 mM)
0.29 g CaCl2 2H2O (10 mM)
1.98 g MnCl2 4H2O (50 mM)
30 g Glycerol (15% wt/vol)
Adjust to pH 5.8 with 0.2 M acetic acid (do not adjust pH with KOH). Add dH2O to 200 mL. Filter sterilize. Store refrigerated at 4°C.
200 mL TFB2:
0.42 g MOPS (10 mM)
2.21 g CaCl2 2H2O
0.24 g RbCl (10 mM)
30 g Glycerol (15% wt/vol)
Adjust to pH 6.5 with KOH. Add dH2O to 200 mL. Filter sterilize. Store refrigerated at 4°C.
Preparation of Competent Cells
Streak cells from frozen stock onto LB plate. Incubate O/N at 37°C.
Pick a single fresh colony to inoculate 5 mL of LB medium. Grow O/N at 37°C. Do not vortex cells at any time after this point in the procedure.
Dilute 1 mL of culture into 50 mL LB medium prewarmed to 37°C. Grow at 37°C for 2 hours with agitation. Volumes can be scaled up 5X and all of the 5 mL overnight culture can be used.
Transfer culture to sterile 50 mL tube. Chill on ice 10-15 minutes.
Centrifuge for 10 mintutes at 2000 rpm at 4°C. Immediately aspirate off all the supernatant. Do not allow cells to warm above 4°C at any time in this procedure.
Resuspend cells in 10 mL of ice-cold TFB1 with gentle re-pipetting. Use chilled glass of plastic pipette.
Incubate cells on ice for 5 minutes.
Repeat step 8
Resuspend cells in 2 mL of ice-cold TFB2 with gentle re-pipetting. Use micropipet tip (plastic).
Incubate cells on ice for 15 minutes. Cells may be used for transformation or frozen. To freeze: aliquot cells 100 µL volumes into prechilled 0.5 mL microcentrifuge tubes (on ice). Freeze immediately on dry ice. Stire cells frozen at -80°C.
Transformation of competent cells
If starting with frozen competent cells, warm tube/cells by gently twirling between your fingers until just thawed (i.e., at ~0°C). Then, immediately place on ice for about 5 minutes.
Set up transformation as follows:
Add to 15 mL plastic round bottom tube on ice:
0-9 µL TE (Tris 10mM + EDTA 1mM)
1-10 µL DNA (10-100 ng)
10 µL final volume → /!\ 10% max of the cell competent volume
Add 100 µL of competent cells and mix by gentle repipetting. This method can be scaled down 2- to 4-fold. The maximum volume of DNA should be ~1/10 volume of cells and the maximum mass should be <= 100 ng of DNA for 100 µL of cells.
Incubate cells on ice for 20-30 minutes.
Heat shock the cells exactly 90 seconds at 42°C.
Return cells on ice 2 minutes.
Add 1 mL of LB medium. Incubate at 37°C for 45-60 minutes with slow gentle shaking. For blue/white color selection, spread IPTG and X-gal on plates now and hold at 37°C until use
Plate 0.1 - 0.2 mL of transformed cells on LB-plate containing the appropriate antibiotic (adn IPTG and X-gal if needed). Incubate overnight at 37°C. Place at 4°C to store and/or enhance blue color. Note: The next day, liquid cultures of the transformants can be left 8 hours before the miniprep. In the best-case scenario, do the liquid culture at 8am and do the miniprep at 4pm.
Testing competent cells
Transform 100 µL of cells with 1 µL (10 pg) of pUC19 monomer (0.01 µg/µL).
Plate 0.25 mL of transformation mixture. Incubate overnight at 37°C.
Count CFU and calculate efficiency. Efficiency =# of colonies per µg =# of colonies X4 X 105. You should obtain 1-5 X 107/µg from competent cells after one freeze-thaw cycle.
Enzymatic digestion of DNA.
Introduction
This protocol was extracted from the protocol from NEB website.
Materials
For analysis digestion:
Eppendorfs
1 μg of DNA
1 μL 10X buffer (most enzymes can be used in Cutsmart buffer, check on NEB website)
1U enzyme pour 1 μg ADN (0,5 μL for 1 μg DNA)
H2O qsp 10 μL
heating plate
For preparative digestion:
Keep the same proportions and scale up for 30µL of DNA on 100µL final
If cut by the same Enzyme, please prepare a MIX with n+1 (n = number of sample)
For gel migration, add 2 μL of loading dye for each 10 μL mix
Procedure
Mix all the elements
Incubate 1h at enzyme specific temperature (usually 37°C)
Check if heat inactivation is required and do it accordingly /!\ if inactivation is done at high temperature put on ice after inactivation and then centrifuge to keep the  evaporated water.
Electroporation of P. pastoris.
Introduction
Protocol from Lin-Cereghino, J., Wong, W., Xiong, S., Giang, W., Luong, L., Vu, J., Johnson, S. and Lin-Cereghino, G. (2005). Condensed protocol for competent cell preparation and transformation of the methylotrophic yeast Pichia pastoris. BioTechniques, 38(1), pp.44-48.
Materials
ice
linearized plasmid (with AvrII)
competent cells from the protocol cell preparation
electroporation apparatus
1.0M sorbitol
YPD
plates with gradient of zeocin
Procedure
Mix approximately 4-8μL (50–100 ng) of dialysed linearized plasmid DNA with 40 μL of competent cells in an electroporation cuvette
Incubate for 2 min on ice
Pulse 1500V, 25μF, 200Ω (You should have a Ꞇ between 4 and 5 ms. If it is >5ms, there were too many ions in the mix. It can kill cells.) (was done previously with 1500V, 10μF, 600Ω -> worked)
Resuspend immediately samples in 0.5 mL 1.0 M sorbitol and 0.5 mL YPD, incubate in a 30°C shaker for 1h30, and then plate on media containing increasing concentrations of zeocin (100, 250, 500, or 1000 μg/mL) for the selection of multicop
NMR analysis
C8-CAI-1 analysis
This protocol is used to quantify C8-CAI-1 production by NMR spectroscopy.
Supernatant obtained by centrifugation of 50mL of total broth are freeze-dried, resuspended in 500µL of CDCl3, and spiked with 100 µL of TSP-d4 (1mM, in D2O) used as internal standard for quantification and as reference for chemical shifts. The resulting samples are analyzed at 280K by 1D 1H NMR on an Avance 800 MHz spectrometer (Bruker, Rheinstetten, Germany) equipped with a 5-mm z-gradient TPI probe, using a zgpr sequence with a 90° pulse of 7µs and a relaxation delay between scans of 5 s. A total of 64 scans were accumulated (128k data points with a spectral width of 10 ppm) after 4 dummy scans. All the spectra were acquired and processed on TopSpin 3.2 (Bruker).
Diacetyl analysis
This protocol is used to quantify diacetyl production by NMR spectroscopy.
Supernatant (500µL) obtained by filtration of total broth (Sartolon polyamide 0.2µm, Sartorius) are spiked with 100 µL of TSP-d4 (1mM, in D2O) used as internal standard for quantification and as reference for chemical shifts. The resulting samples are analyzed at 280K by 1D 1H NMR on an Avance 500 MHz spectrometer (Bruker, Rheinstetten, Germany) equipped with a 5-mm z-gradient BBI probe, using a zgpr sequence for water suppression with a 90° pulse of 7µs and a relaxation delay between scans of 5 s. A total of 64 scans were accumulated (128k data points with a spectral width of 10 ppm) after 4 dummy scans. All the spectra were acquired and processed on TopSpin 3.2 (Bruker).
Solid Bioluminescence assay
Introduction
This protocol is based on the Experimental Procedure provided in the following publication:
Ng W-L, Perez LJ, Wei Y, Kraml C, Semmelhack MF & Bassler BL (2011). “Signal production and detection specificity in Vibrio CqsA/CqsS quorum-sensing systems: Vibrio quorum-sensing systems.” Molecular Microbiology 79 1407–1417. https://www.ncbi.nlm.nih.gov/pubmed/21219472
The aim is to detect the Vibrio harveyi quorum sensing molecule C8-CAI-1 in vivo from a spent culture fluid. Here, a clone of an Escherichia coli MG1655 strain able to synthesis the molecule is taken as an example. Please note that the whole procedure lasts 3 days.
Materials
Liquid LB medium
Solid LB medium
Sterile glucose solution
Sterile IPTG solution, final concentration in culture: 0.5 mM
Procedure
1st day.
Liquid precultures of E. coli
Prepare 2 test tubes with 5 mL of liquid LB medium complemented with chloramphenicol (final concentration in the medium: 25 mg/L) and glucose (final concentration in the medium: 10 g/L). Vortex to mix all the components.
From agar plates, inoculate one of the previous tubes with the E. coli MG1655 clone. Inoculate the other with the E. coli MG1655 negative control.
Incubate the tubes over-night at 37°C with shaking (160 rpm).
Liquid precultures of V. harveyi
Prepare 2 flasks with 10 mL of liquid LB medium.
From agar plates, inoculate one of the previous flasks with V. harveyi JMH626. Inoculate the other with V. harveyi BB120.
Incubate the flasks over-night at 30°C with shaking (160 rpm).
2nd day.
Expression cultures of E. coli
Measure OD of the 2 E. coli precultures at 600 nm.
Prepare 2 flasks with 20 mL of liquid LB medium complemented with chloramphenicol (final concentration in the medium: 25 mg/L) and glucose (final concentration in the medium: 10 g/L).
Vortex to mix all the components.
From the precultures, inoculate one of flasks with the E. coli MG1655 clone at OD = 0.1 Inoculate the other with the E. coli MG1655 negative control at OD = 0.1.
Incubate the flasks at 37°C with shaking (160 rpm).
When OD = 0.3, put the flasks at 30°C with shaking (160 rpm) for 15 minutes. Then, add IPTG (final concentration in the medium: 0.5 mM) and incubate at 30°C with shaking (160 rpm).
When OD = 0.9, retrieve the supernatants as follows: centrifugate all the cultures at maximum speed for 10 min. filter the resulting supernatants through a 0.2 µm filter.
Store both supernatants at -20°C for subsequent use.
Expression culture of V. harveyi BB120
Measure OD of the V. harveyi BB120 preculture at 600 nm.
Prepare 2 flasks with 20 mL of liquid LB medium.
From the preculture, inoculate the previous flasks at OD = 0.1
Incubate at 30°C with shaking (160 rpm).
Check bioluminescence regularly. When V. harveyi BB120 shows bright bioluminescence, retrieve the supernatant of one the cultures as follows: centrifugate the whole culture at maximum speed for 10 min filter the resulting supernatants through a 0.2 µm filter.
Store the supernatant at -20°C for subsequent use. Keep the other V. harveyi BB120 culture at 30°C with shaking (160 rpm).
Puce 1
Puce 1
Puce 1
Puce 1
Expression culture of V. harveyi JMH626
Measure OD of the V. harveyi JMH626 preculture at 600 nm.
Prepare 4 flasks with 10 mL of liquid LB medium.
From the preculture, inoculate the previous flasks at OD = 0.1
Incubate at 30°C with shaking (160 rpm) until OD = 0.6.
Centrifugate the 4 cultures at 4500 rpm for 6 min.
Discard the resulting supernatant and resuspend each of the pellets with 5 mL of liquid LB medium.
Add 5 mL of the supernatants obtained previously. one flask must be complemented with the E. coli MG1655 clone supernatant. one flask must be complemented with the E. coli MG1655 negative control supernatant. one flask must be complemented with the V. harveyi BB120 supernatant. one flask must be complemented with additional liquid LB medium.
Prepare 3 LB agar plates and divide each of them into 5 identical zones.
On the plates, drop-off 70 µL of each of the V. harveyi JMH626 resuspended cultures. On the fifth zone, drop-off 70 µL of the last V. harveyi BB120 liquid culture .
Incubate the 3 plates at 30°C over-night.
3rd day
Observe each of the plates in total darkness.
Triparental conjugation
Materials
Plates:
For conjugation:
1 plate of LB
1 plate of LB - Cmp - Gen or LB - Cmp - Amp (depends on the resistance cassette on the conjugative plasmid)
For control:
1 plate of LB
1 plate of LB - Cmp - Gen or LB - Cmp - Amp (depends on the resistance cassette on the conjugative plasmid)
Strains:
Recipient: Vibrio harveyi JMH626 CmpR KanR
Helper: Escherichia coli - pRK2073 (helper plasmid)
Donor: Escherichia coli transformed with the gene of interest in a conjugative plasmid (pBBR1MCS-4 AmpR or pBBR1MCS-5 GenR)
Other:
LB liquid media
Antibiotics: Spec, Cmp, Gen or Amp (depends on the resistance cassette on the conjugative plasmid)
Membranes for the conjugative culture
Plan of experiment
1. Liquid cultures overnight from glycerol stock or plates:
V. harveyi JMH626: 5 mL of LB, 30°C
E. coli helper - pRK2073: 5 mL of LB + Spec (at the recommended concentration), 37°C
E. coli donor - conjugative plasmid: 5 mL of LB + Amp or Gen (at the recommended concentration), 37°C
2. Centrifugation steps:
Centrifugate 1 mL of each culture 4 min at 10,000 rpm
Throw the supernatant
Resuspend the pellet in 1 mL LB
Repeat the three steps with the resuspended solution
3. Conjugation
Conjugation mix:
In an eppendorf, add:
40 µL of the donor strain (E. coli with the conjugative plasmid)
40 µL of the helper strain (E. coli - pRK2073)
40 µL of the recipient strain (V. harveyi JMH626)
Depose a membrane on an LB plate and add 100 µL of the mix at the center of the membrane.
Control mix:
In an eppendorf tube, add:
50 µL of the helper strain (E. coli - pRK2073)
50 µL of the recipient strain (V. harveyi JMH626)
Depose a membrane on an LB plate and add 80 µL of the mix at the center of the membrane.
Incubate the two plates overnight at 30°C.
4. Resuspension and incubation
For both control and conjugation:
Put the membrane in 5 mL H2O (use falcone tube) and vortex it until the solution becomes unclear. Place 1.5 mL of the solution into an eppendorf tube and centrifugate 4 min at 10,000 rpm. Throw approximately 1200 µL of supernatant and resuspend the pellet in the remaining supernatant. Spread the solution on a LB - Cmp - Amp or LB - Cmp - Gen (depending on the resistance cassette on the donor plasmid). Incubate 48h at 30°C.
To sum up:
Fluorescence microscopy
Sample preparation
V. harveyi were grown at 30°C and 160 RPM in liquid media LM with chloramphenicol over-night.
Microscope slide has been prepared with 2x diluted culture of V. harveyi.
Observations under fluorescent microscope
Lorem ipsum dolor sit amet, consectetur adipisicing elit. Harum, perspiciatis unde quasi enim ab repellendus deleniti vel earum, facere aspernatur magnam! Maiores in, eos at id! Necessitatibus eligendi modi, unde.
Protein production and sampling
C8-CAI-1 production
Precultures LB: Inoculation on the morning of E. coli K-12 MG1655 - VhCqsA and E. coli K-12 MG1655 - pSB1C3 (no insert, used as negative control) on liquid culture LB (10 mL) with Cmp, at 37°C, 160 RPM.
Culture on M9: On the morning, inoculation at OD(600nm) = 0.1 into 50 mL of M9  after 2 washing step (centrifuged 4000 g twice, resuspension of pellet in M9) with the glucose as carbon source (20 g/L) at 37°C, 160 RPM
At OD= 0.9, IPTG induction (0.5 mM). Growth overnight at 30 °C, 160 RPM
Samples are collected after a night of production.
Diacetyl production
Precultures LB: Inoculation on the morning of E. coli K-12 MG1655 - als and E. coli K-12 MG1655 - pSB1C3 (no insert, used as negative control) on liquid culture LB (10 mL) with Cmp, at 37°C, 160 RPM.
Precultures M9: On the evening, inoculation at OD(600nm) = 0.1 into 50 mL of M9 with the xylose as carbon source (30 mM) and previously prewarmed at 37°C. Growth overnight at 37°C, 160 RPM.
Culture M9: Pyruvate is added to the medium (30 nM), and 50 mL flasks with or without citrate (2.2 g/L) were prepared. On the morning, precultures M9 were inoculated into 50 mL flasks with or without citrate at OD(600nm) = 0.1. Growth at 30°C, 160 RPM.
Culture M9: The culture is followed during two days. Samples are collected at the end of exponential growth phase (OD(600nm) between 1.4 and 1.9 depending on the culture)
AMP production
DNY-15
One colony of P. pastoris positive transformant for pPICZα-D-NY15 was grown in 50 mL YPD 4% glucose medium in a 250mL culture flask. Same in YPD 5% glucose medium. The negative control was P. pastoris that as integrated the empty pPICZα plasmid.
The 4 cultures were shaken for 96 hours, 30°C, 160 RPM.
Cultures were centrifuged 10 min at 3000 g, then supernatant was collected for sampling.
Leucrocine I and cOT2
See protocol here here: in M.R. Kuddus, F. Rumi, M. Tsutsumi, R. Takahashi, M. Yamano, M. Kamiya, T. Kikukawa, M. Demura, T. Aizawa, Expression, purification and characterization of the recombinant cysteine-rich antimicrobial peptide snakin-1 in Pichia pastoris, Protein Expression and Purification (2016).
Sampling
After recuperation of supernantant from culture,
Supernatant was filtered (0.2 µm) for further analysis, Diacetyl
Supernantant was filtered (0.2 µm) then freeze dried then stored a -80°C for C8-CAI-1 and AMP
Plate reader
Materials
CMM medium
YNB solution
YPD solution
Plate reader
Centrifuge
Falcon 50
24-well plate
Procedure
One colony of P. pastoris Odr10-RFP was grown in 50mL YPD medium for 48h at 30°C with shaking. Same for the negative (P. pastoris with empty pPICZα) and positive (P. pastoris with pGAP-RFP) controls.
Inoculation of 50mL CMM Ammonium sulfate and 50mL CMM Glutamine at OD 0.2 for each clone.
The next day, inoculation of 50mL CMM Ammonium sulfate and 50mL CMM Glutamine at OD 0.2 for each clone and culture over day from OD600 = 0.2 to 0.6
Split each culture in 2 in falcons 50 and centrifuge 3 min at 3000 rpm at RT.
Wash cells with 10 mL of CMM Glutamine or YNB (to start Glutamine depletion) for the cultures coming from CMM Glutamine medium and 10mL of CMM Ammonium sulfate or YNB (to start Ammonium sulfate depletion) for the cultures coming from CMM Ammonium Sulfate.
Centrifuge 3 min at 3000 rpm at RT.
Resuspend each pellet in 1mL of CMM Glutamine or YNB for the cultures coming from CMM Glutamine medium and 1mL of CMM Ammonium sulfate or YNB for the cultures coming from CMM Ammonium Sulfate.
Inoculation plate at OD 0.2 in 1mL of the corresponding medium.
Add diacetyl to 500µM and 1000µM for each condition.
Plate reader set-ups
Cells were cultivated in a FLUOstar Optima plate reader (BMG Labtech, Offenburg, Germany) at 30°C and 600 rpm (orbital) with a shaking diameter of 1 mm.
The shaking and measurement procedures were as follows:
shake duration, 300 s
ﬂuorescent measurement duration, 123 s per 48 wells, 20 ﬂashes; shaking duration, 60 s
absorbance measurement duration, 50 s per 48 wells, 20 ﬂashes
180 cycles, 24 h, ﬂash
.
Biomass was determined by measurement of optical density (OD) at 600 nm. The experiments were carried out with standard round 48 or 24--well plates with ﬂat bottoms from Starlab, France (catalog number 1830048), covered with lids.
If not otherwise speciﬁed, the experiments were conducted with a 500-�l working volume of medium.  RFR was excited with a wavelength of 544 nm and the emission was detected at 600nm.
RT-qPCR
RNA extraction
Culture of P.pastoris SMD1168H
Cells are centrifuged max speed and washed with sterilized water
Another centrifugation max speed and the supernatant is removed
Pellets are freeze at -80°C overnight
500 µl of GTC mix + 500 µl of Water/Phenol + 1 ml of ice-cold glass beads are added to each falcon
2 min of vortex full power and then chill on ice for 1 min, this step is repeated 2 more times
Add 7.5 ml of GTC mix and 7,5 ml of Water/phenol
1 min of vortex full speed and then 5 min of incubation at 65°C
Add 7.5 ml of CH3Cl + 4 ml of NaAc 100mM in TE
1 min of vortex full power
centrifugation 4000 rpm, 5 min at 4°C
12 ml of the aqueous phase is extracted and transferred into a new falcon containing 6 ml of Water/phenol + 6 ml of CH3Cl
centrifugation 4000 rpm, 5 min at 4°C
10 ml of the aqueous phase are extracted and transferred into a new falcon containing 5 ml of Water/phenol + 5 ml of CH3Cl
1 min of vortex full speed
The aqueous phase is collected (5 ml) and 3 volume of Ethanol are added (25ml)
centrifugation 4000rpm, 20 min at 4°C and the supernatant is thrown away
Add 5 ml of Ethanol 70%
centrifugation 4000rpm, 1 min at 4°C and the supernatant is thrown away (this step is repeated 2 more times)
ARNs are dried and diluted in 200 µl of water
Nanodrop quantification
Reverse transcriptions
RT reactions using total RNAs extracted from Pichia pastoris cells transformed with plasmids pPIC-DNY15 or pPIC (empty vector).
Reactions using 1 µg of total RNAs (1.5 ml microtube):
2 µl Primer DNY15-RT-r (1 µM = 1 pmol/µl)
1 µl Total RNAs (1 µg/µl)
1 µl dNTP mix (10 mM each)
8 µl MQ H2O
Reactions using 5 µg of total RNAs:
2 µl Primer DNY15-RT-r, 1 µM (= 1 pmol/µl)
5 µl Total RNAs (1 µg/µl)
1 µl dNTP mix (10 mM each)
4 µl MQ H2O
Mix gently with vortex
Incubate 5 min at 65°C (water bath)
Chill immediately on ice and incubate for 5 min
Pulse spin at 4°C and add the following:
4 µl 5X First-Strand Buffer
2 µl 100 mM DTT
1 µl RNasin (40 U./µl, Promega)
Mix with vortex gently
Incubate 2 min at 42°C (Thermomixer Eppendorf)
Add 1 µl SuperScript II RT (Invitrogen, reference 18064-022)
Mix by pipeting gently up & down
Incubate 1 hour at 42°C
Inactivate the reaction by heating at 70°C for 15 min (water bath)
qPCR reactions
2 µl RT mix
0.5 µl primer DNY15-RT-f (12 µM = 100 ng/µl)
0.5 µl primer DNY15-RT-r (12 µM = 100 ng/µl)
4.5 µl MQ H2O
7.5 µl IQ SYBR Green Supermix (Biorad, reference 1725006CUST)
PCR program (CFX96 Touch Real-Time PCR detection system)
95°C 3 min
95°C 15 sec
40X 60°C 30 sec
72°C 30 sec
Melting curve from 60°C to 95°C, 0.5°C/cycle, 5 sec
On plate toxicity assay
Introduction
To test the cytotoxicity of the AMPs, the supernatant of recombinant P. pastoris were tested by a Growth Inhibition Assay on freshly plated V. harveyi cultures.
Materials
LB agar plate
50 mL culture flasks
YPD liquid medium
Paper discs (6 mm diameter)
Recombinant P. pastoris (producing AMPs and empty vector)
Chloramphenicol solution (25 μg/mL)
Procedure
Produce the AMPs with the AMP production protocol, a negative control is also needed containing only the empty cloning vector.
Grow wild type V. harveyi to OD 0.3, and then plate 200 μL on an LB plate.
Let it dry a few minutes.
Dip the paper discs in: the AMP containing supernatant, the negative supernatant and the chloramphenicol solution. And place them on the plate.
Grow overnight at 30°C.
Realisations pages
Overview
Notebook
Clonings
Results
Contribution
Protocols
Safety
