Team:INSA-UPS France/Experiments/Clonings
Croc'n Cholera
iGEM UPS-INSA Toulouse 2017
Team
Members
Supports
Attributions
Collaborations
Strategy
Description
Design
Parts
Model
Overview
Simulation
Analysis
Interface
Realisations
Overview
Notebook
Clonings
Results
Contribution
Protocols
Safety
Human Practices
Overview
Silver
Gold
Public Engagement
Microbioworld
Application
Applied Design
Testimonies
Entrepreneurship
Device Conception
Clonings
1. Mimicking Vibrio sp. presence
2. E. coli producing C8-CAI molecules sensed by V. harveyi
3. Modification of V. harveyi
4. Production of diacetyl
5. P. pastoris is able to detect diacetyl
6. P. pastoris is able to produce functional AMP
7. Co-cultivation of P. pastoris and V. harveyi
8. Permeability assay of the membranes
Manipulating DNA and inserting it into different chassis organism is a crucial step to begin synthetic biology, here you can find the different part that we cloned for each step of our experiments.
1. Mimicking Vibrio sp. presence with an engineered E .coli
Part Part:BBa_K2278001 and Part: Part:BBa_K2278002 were constructed to produce respectively C8-CAI-1 and CAI-1 in E .coli. The cqsA from V. harveyi (i.e. Vh-CqsA) or V. cholerae (i.e. Vc-CqsA) coding gene was placed under the control of the pLac promoter (part: Part: BBa_R0040 ), a strong RBS ( Part: BBa_B0034 ), and a terminator ( Part: BBa_B1006 ) (Figure 1). IDT performed the DNA synthesis and delivered the part as gBlock. The constructs were cloned by conventional ligation into the pSB1C3 plasmid and then transformed into E .coli DH5α or TopTen strain. Three transformants of each were tested (Figure 2).  Sequencing (figure 3) revealed that the Vh-CqsA construction slightly differs from the initial design, with a loss of the 9 last amino acids of the protein (position 382 to 391; confirmed on two different runs).
Figure 1: Schematic view of the constructs used to express cqsA in E .coli.  A : Part:BBa_K2278001 and B: Part:BBa_K2278002 .
Figure 2: Analytical gel of the restriction map of  pSB1C3_Vh cqsA  and pSB1C3_Vc cqsA   A: pSB1C3_Vh cqsA digested by SacI are electrophoresed through a 1% agarose gel. Lane 1 is the DNA ladder (New England biolab), the 0.5kb, 1 kb and 3kb DNA fragments are annotated. Lane 2 is the linearized pSB1C3 vector containing a 700bp insert (2700bp). Lanes 3-5 are the digested plasmids resulting from DNA extraction of the 3 obtained clones. We expected two bands at 1901 and 1458 bp. B: pSB1C3-VcCqsA digested by (EcoRI and Pst1) are electrophoresed through a 1% agarose gel. After digestion, the 2029 bp fragment corresponds to the pSB1C3 vector while the 1318 bp fragment corresponds to the cqsA gene.
Figure 3: Sequencing of pSB1C3-VhCqsA 1500 ng of plasmid were sequenced. 3 oligos were used to perform the sequencing. The obtained sequence were blast on the Part:BBa_K2278001 sequence with the iGEM sequencing online tools.
2. E .coli producing C8-CAI molecules can be sensed by V. harveyi
No cloning was made there.
3. Modification of V. harveyi to detect both C8-CAI-1 and CAI-1
This part (Figure 4) was not submitted in the registry and was created to implement the protocol of triparental conjugation in V. harveyi. The part: Part:BBa_J04450 containing the LacI promoter + the RFP encoding gene + a terminator was cloned by conventional ligation into pBBR1MCS-4 and pBBR1MCS-5 (two conjugative plasmids) and then transformed into E .coli Top10 (Figure 5).  These plasmids were then used conjugated into V. harveyi by triparental conjugation to validate both the biobrick in V. harveyi and our conjugation protocol.
Figure 4 : Schematic view of the contruct used to express RFP in V. harveyi
Figure 5: Plates with red transformants from pBBR1MCS-4 – RFP (left) and pBBR1 – MCS-5 – RFP  (right) clonings Transformants show RFP activity.
This part (Figure 6) was not submitted on the registry and was created to allow V. harveyi to recognize both the C8-CAI-1 and CAI-1 from V. cholerae . This part includes the complete cqsS* gene driven by a constitutive promoter ( Part:Bba_J23106 ) and the tetracyclin repressor under the control of pQRR4 promoter ( Part:Bba_K1311017 ) which activation depends on CqsS* detection of C8-CAI-1 and CAI-1. Strong RBS ( Part:Bba_BBa_B0034 ) and terminator ( Part:Bba_BBa_B1006 ) were surrounded the ORF. IDT performed the DNA synthesis. Because of its length, it delivered the part as two gBlocks subparts to assembly. Both subparts were cloned separately by conventional ligation into pBR322, then transformed into E .coli Stellar strain. The final part was cloned by another conventional ligation of the first subpart into the second one, and transformed into E .coli Top10. Six transformants were tested (Figure 7).
Figure 6 : Schematic view of the contruct used to express modified receptor of CqsS* Transformants show RFP activity.
Figure 7: Analytical gel of the restriction map of pBR322-VhCqsS* 1, 2, 3, 4, 5, 6 lanes are transformants digested with BamHI/XhoI. We expected 2 fragments of 4691 and 2026 bp for pBR322-VhCqsS*(total length: 6717 bp)
4. Production of diacetyl to establish communication between prokaryotic and eukaryotic cells
( Part:BBa_K2278011 ) (Figure 8) was constructed to test the diacetyl production in E .coli before implementing the pathway in V. harveyi. The gene als encoding for the acetolactate synthase responsible for the diacetyl production, was placed  under the control of the pTet promoter ( Part: BBa_R0040 ), a strong RBS ( Part: BBa_B0034 ), and a terminator ( Part: BBa_B1006 ). IDT performed the DNA synthesis and delivered the part as gBlock. The construct was cloned by conventional ligation into the pSB1C3 plasmid and transformed into E .coli Top10 strain. 5 transformants were tested (Figure 9).
Figure 8: Schematic view of the contruct used to express als in E .coli .
Figure 9: Analytical gel of the restriction map of pSB1C3-als A to G lanes are transformants digested with NcoI and ApaI. C and E lane have the expected length: 876 bp and 3083 bp (total length: 3959 bp) containing pSB1C3 and als.
5. P. pastoris is able to detect diacetyl from the environment
This part (Figure 10), not submitted to the registry, was created to allow P. pastoris to sense diacetyl and produce AMPs in response. This part includes Odr-10 receptor driven by a constitutive yeast promoter pGAP ( Part:BBA_K431009 ) and flanked by a kozac sequence ( Part:BBA-J63003 ) and a stop sequence ( Part:BBA-J63002 ). It also includes cOT2-coding-gene (AMP) under the control of pFUS1 promotor ( Part:BBA_K1072023 ) inductible by the Ste12 protein activated by Odr-10 pathway when diacetyl is detected. The gene of cOT2 is flanked by kozac sequence ( Part:BBA-J63003 ) and a stop sequence ( Part:BBA-J63002 ). This part has been successfully integrated in P. pastoris genome (Figure 11).
Figure 10: Schematic view of the contruct used to produce AMPs thanks to the ODR10 receptor
Figure 11: Analytical gel of the colony PCR of the part Odr10-pFUS1-cOT2 The positive control of the colony PCR gives a clear and expected band at 800 bp. All P. pastoris clones have the expected band of 3000 bp.
A second part was made by replacing the cOT2 gene by the RFP to check if Odr-10 was functional in vivo (Figure 12) and this part has also been successfully integrated (Figure 13).
Figure 12: Shematic view of the contruction used to test Odr-10 in vivo functionality with RFP
Figure 13: Analytical gel of the colony PCR of Odr10-pFUS1-RFP   The positive control of the colony PCR gives a clear and expected band at 800 bp. The 5th clone of P. pastoris has the expected band of 2700 bp, despite being weak the integration of this part has been confirmed with fluorescence assay.
6. P. pastoris is able to produce functional antimicrobial peptides
Parts ( Part:Bba_K2278021 (D-NY15 gene), ( Part:Bba_K2278022 (Leucrocine I gene) and ( Part:Bba_K2278023 (cOT2 gene) were built to test the production of AMPs in P. pastoris (Figure 14). Genes encoding for Leucrocine I, D-YN15, and cOT2 were placed under the control of an alpha factor signal ( Part:BBA_K1800001 ). IDT performed the DNA synthesis and delivered the part as gBlock. The constructions were cloned by conventional ligation into the pPICZα yeast vector containing pAOX1 or pGAP (Figure 15) and integrated into the yeast genome (Figure 16). Sequencing (Figure 17) revealed that the AMP constructions do not contain any mutation.
Figure 14: Schematic view of the contructions used to secrete AMPs thanks to the α-factor sequence
Figure 15: Schematic view of the contructions used to secrete AMPs thanks to two promotors pGAP (constitutive) and pAOX1 (methanol-inducible)
Figure 16: Validation of genomic integration of the antimicrobial peptide sequences by PCR on colony a) Integration of pAOX1-cOT2 and pAOX1-leucrocine I at the expected length of respectively 307 bp and 241 bp. b) Integration of pGAP-D-NY15 with an expected length of 850 bp.
Figure 17: Sequencing of AMP 1500 ng of plasmid were sequenced. a) Sequencing of D-NY15 AMP performed with 1 oligo. The obtained sequence was blast on the Part:Bba_K2278021 sequence with the iGEM sequencing online tools. b) Sequencing of Leucrocine I AMP performed with 2 oligos. The obtained sequence was blast on the Part:Bba_K2278022 sequence with the iGEM sequencing online tools. c) Sequencing of cOT2 AMP performed with 2 oligos. The obtained sequence was blast on the Part:Bba_K2278023 sequence with the iGEM sequencing online tools.
To prove the functionality of the pGAP promotor, the RFP gene has been cloned in the pPICZα-D-NY15 instead of the D-NY15 gene (Figure 18) and this part has been integrated into the genome. This work has been performed by the IGEM team of Vienna as collaboration and they send us the engineered strain as well as the proof of integration by PCR colony gel (Figure 19).
Figure 18: Schematic view of the contructions used to test pGAP activity thanks to RFP fluorescence
Figure 19: Validation of genomic integration of pGAP-RFP by PCR on colony  The size of the integrated construct should be 2119 bp. Note that the DNA Ladder they used (MassRuler DNA Ladder Mix) is different from the one we used in the lab.
7. Co-cultivation of P. pastoris and V. harveyi is possible
No cloning was made there.
8. Permeability assay of the membranes used for the devices
No cloning was made there.
Realisations pages
Overview
Notebook
Clonings
Results
Contribution
Protocols
Safety
