"
Page
Discussion
View source
History
teams
Log in
 
Team:ULB-Brussels/Conclusion
From 2012.igem.org
Home
Team
Project
Parts
Modeling
Conclusion & Perspectives
Safety
Older wiki's
 
Team ULB-Brussels, conclusion &
perspectives!
Construction of the integron
     As said in the introduction, an integron is composed of different gene cassettes located downstream of a recombinase-encoding conserved sequence. To allow genes rearrangement, they must be flanked by attc sites. Indeed, these sites are recognized by the recombinase that allows the excision of the different gene cassettes and their integration in another attc site. In the results, we showed that we had to add an RFC10 prefix and a RBS sequence upstream of each gene and an attc site followed by a RFC10 suffix downstream of the gene. In the future, we should build the remaining microcin biobricks. The next step will be to assemble all the biobrick parts of the Microcin B17 and C7. Those genes could be assembled in the natural or in a random order. Thereafter, the integron biobrick has to be put in an iGEM plasmid provided by Mazel D. – pSWlib. This plasmid contains – between the promoter and the RFC10 region – an attI site which is required for the recombination by the integrase to occur.
Competitions
     In order to select the most productive genes rearrangement, we have to exert a strong selection pressure on the bacteria containing the integron and integrase biobricks. To this end, we could use the properties of microcin and put into competition two E. coli strains. The first strain should carry the microcin B17 integron plasmid, the integrase plasmid and a plasmid that contains the immune genes of microcin C7. The other strain should carry the microcin C7 integron plasmid, the integrase plasmid and a plasmid that contains the immune gene of microcin B17. The upstream promoter of the immune genes has to be carefully chosen to confer the bacteria a low resistance against the microcin they don’t express. Therefore, the more bacteria produce one type of microcin, the more they will kill the bacteria producing the other type of microcin. This way, we could select a population of bacteria that produces the more efficiently its microcin. We could simplify our experiment by putting into competition a strain of E.coli that contains the microcin B17 with the integrase against a strain of E. coli that contains the immune gene that protects it against the microcin B17. This immune gene must be under the control of an inducible promoter to be able to modulate its rate. Some times after the beginning of the experiment, we could increase the expression of the immune gene of microcin B17 to exert a stronger selective pressure in order to favor the best gene rearrangement of the microcin operon. The same experiment should also be done with microcin C7.
Team Slovenia 2010
     By reading previous iGEM projects, we saw that the Slovenia team from 2010 wanted to improve protein production like us, but in another way. In their project, they use DNA like a scaffold to bring together all the enzymes involved in a biosynthetic pathway. In fact, they fused each of these enzymes with a zinc finger motif, whose are able to bind DNA. The zinc finger motifs used were specific to some nucleotide sequences that were close to each other. The results of their experiments shown an increase of the rate of the protein tested.
     Our approach could complement them by making sure each enzyme in the pathway was produced at an optimal rate.
Could our system do better than « nature »?
     Our entire goal was to prove, using microcins, that the integron could optimize any biological pathway. We did not manage to build all our biobricks and assemble them in the integron. However, we can still try to answer the following question: Could it be possible to use the integron to increase the microcin production to a higher degree than what is found in the environment? There are evidences to back up and debunk our hypothesis.
     Firstly, there is a pattern in the way the original microcin B17 and C7 operons are built. The gene coding for the pro-microcin is always first, followed by genes crucial for microcin maturation and finally immunity genes. This gene arrangement has been selected over a very large period of time in the environment and therefore we could consider that the microcin operon is already a highly optimized pathway. Using the integron in this case could have been useless.  However, the way our competition experiments have been thought does not strictly represent what is found in nature. Putting 2 microcins producing bacteria in direct contact would apply a strong selective pressure on both of them. That selective pressure could lead to gene rearrangements in the integron that could never have been seen in the environment.
     Secondly, it has already been proven that the integron is a powerful tool when it comes to optimize pathways. Bikard et al. (2010) managed to use the integron to enhance tryptophan production in bacteria. The experimental set up was similar to ours albeit with a few differences. The 5 genes from the tryptophan operon were packaged into gene cassettes: the genes of interests were preceded by an RBS and followed by an attC recombination site. The different cassettes were then assembled in an integron on a plasmid and in an arbitrary order. To make sure that the primary gene order could not allow tryptophan production, 2 transcription terminator cassettes were assembled along with the gene cassettes. Tryptophan auxotroph Escherichia coli strains were transformed with the plasmid and with another plasmid containing the integrase controlled by a PBAD promoter. After an overnight culture with arabinose, tryptophan prototrophs were recovered by the researchers. The tryptophan productions were measured and some prototrophs had productions that ranged from 4-fold less to a 2.8-fold more than the original strain carrying the natural tryptophan operon. Bikard et al. used the characteristics of an auxotroph strain to isolate tryptophan producing cells. Similarly, we proposed to put bacteria in competition to isolate microcin producing cells.
     To conclude, according to our model (see modeling section) it would not be possible to select any population producing a higher amount of microcin in our competition assays. Actually, it seems that the subpopulations of bacteria producing lower amounts of one microcin type would be selected as well as those producing larger amount of microcin. The bacteria producing low amounts would benefit from the protection granted by the high producers. Moreover, the selection pressure of this model should be the resistance of the bacteria and not the production. Putting a sensitive or lightly immune bacterial strain in competition with a microcin producing strain, will have the same results except that the selection pressure should be on the production of microcin. On the other hand, we could select the subpopulation that produces the largest amount of microcin according to our model.
     Reference: Bikard D., Julié-Galau S., Cambray G., Mazel D., 2010, The synthetic integrin: an in vivo genetic shuffling device, Nucleic Acids Res., 38 (15), e153
    
Retrieved from " http://2012.igem.org/Team:ULB-Brussels/Conclusion "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
