<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_NCHU_Taichung_Design skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:NCHU Taichung/Design</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>NCHU_Taichung</TITLE></P><DIV class="ui borderless main inverted menu fixed grid"><DIV class="ui text container tablet computer only row"><A class="item" href="https://2018.igem.org/Team:NCHU_Taichung">Home</A><DIV class="ui dropdown item">Project
      <DIV class="menu"><A class="item" href="https://2018.igem.org/Team:NCHU_Taichung/Description">Description</A><A class="item" href="https://2018.igem.org/Team:NCHU_Taichung/Background">Background</A><A class="item" href="https://2018.igem.org/Team:NCHU_Taichung/Design">Design</A><A class="item" href="https://2018.igem.org/Team:NCHU_Taichung/Model">Model</A><A class="item" href="https://2018.igem.org/Team:NCHU_Taichung/Result">Result</A><A class="item" href="https://2018.igem.org/Team:NCHU_Taichung/Demonstrate">Demonstrate</A></DIV></DIV><DIV class="ui dropdown item">Parts
      <DIV class="menu"><A class="item" href="https://2018.igem.org/Team:NCHU_Taichung/Parts">Overview</A><A class="item" href="https://2018.igem.org/Team:NCHU_Taichung/Parts_Improved">Improve</A></DIV></DIV><DIV class="ui dropdown item">Human Practice
      <DIV class="menu"><A class="item" href="https://2018.igem.org/Team:NCHU_Taichung/Human_Practices">Integrated</A><A class="item" href="https://2018.igem.org/Team:NCHU_Taichung/Human_Practices/Introduction">Introduction</A><A class="item" href="https://2018.igem.org/Team:NCHU_Taichung/Human_Practices/Conference">Conference</A><A class="item" href="https://2018.igem.org/Team:NCHU_Taichung/Human_Practices/Engagement">Engagement</A><A class="item" href="https://2018.igem.org/Team:NCHU_Taichung/Entrepreneurship">Entrepreneurship </A></DIV></DIV><DIV class="ui dropdown item">Lab
      <DIV class="menu"><A class="item" href="https://2018.igem.org/Team:NCHU_Taichung/InterLab">Interlab</A><A class="item" href="https://2018.igem.org/Team:NCHU_Taichung/Notebook">Notebook</A><A class="item" href="https://2018.igem.org/Team:NCHU_Taichung/Protocol">Protocol</A><A class="item" href="https://2018.igem.org/Team:NCHU_Taichung/Safety">Safety</A></DIV></DIV><DIV class="ui dropdown item">Team
      <DIV class="menu"><A class="item" href="https://2018.igem.org/Team:NCHU_Taichung/Team">Members</A><A class="item" href="https://2018.igem.org/Team:NCHU_Taichung/Attributions">Attributions</A><A class="item" href="https://2018.igem.org/Team:NCHU_Taichung/Collaborations">Collaborations</A></DIV></DIV><DIV class="ui dropdown item"><DIV class="menu"><A class="item" href="https://2018.igem.org/Team:NCHU_Taichung/Judging#Medals">Medals</A><A class="item" href="https://2018.igem.org/Team:NCHU_Taichung/Judging#Prizes">Prizes</A></DIV></DIV></DIV></DIV><DIV class="ui inverted menu" id="sidemenu"><A class="item" href="https://2018.igem.org/Team:NCHU_Taichung">Home</A><DIV class="ui dropdown item">Project
    <DIV class="menu"><A class="item" href="https://2018.igem.org/Team:NCHU_Taichung/Description">Description</A><A class="item" href="https://2018.igem.org/Team:NCHU_Taichung/Background">Background</A><A class="item" href="https://2018.igem.org/Team:NCHU_Taichung/Design">Design</A><A class="item" href="https://2018.igem.org/Team:NCHU_Taichung/Model">Model</A><A class="item" href="https://2018.igem.org/Team:NCHU_Taichung/Result">Result</A><A class="item" href="https://2018.igem.org/Team:NCHU_Taichung/Demonstrate">Demonstrate</A></DIV></DIV><DIV class="ui dropdown item">Parts
    <DIV class="menu"><A class="item" href="https://2018.igem.org/Team:NCHU_Taichung/Parts">Overview</A><A class="item" href="https://2018.igem.org/Team:NCHU_Taichung/Parts_Improved">Improve</A></DIV></DIV><DIV class="ui dropdown item">Human Practice
    <DIV class="menu"><A class="item" href="https://2018.igem.org/Team:NCHU_Taichung/Human_Practices">Integrated</A><A class="item" href="https://2018.igem.org/Team:NCHU_Taichung/Human_Practices/Introduction">Introduction</A><A class="item" href="https://2018.igem.org/Team:NCHU_Taichung/Human_Practices/Conference">Conference</A><A class="item" href="https://2018.igem.org/Team:NCHU_Taichung/Human_Practices/Engagement">Engagement</A><A class="item" href="https://2018.igem.org/Team:NCHU_Taichung/Entrepreneurship">Entrepreneurship </A></DIV></DIV><DIV class="ui dropdown item">Lab
    <DIV class="menu"><A class="item" href="https://2018.igem.org/Team:NCHU_Taichung/InterLab">Interlab</A><A class="item" href="https://2018.igem.org/Team:NCHU_Taichung/Notebook">Notebook</A><A class="item" href="https://2018.igem.org/Team:NCHU_Taichung/Protocol">Protocol</A><A class="item" href="https://2018.igem.org/Team:NCHU_Taichung/Safety">Safety</A></DIV></DIV><DIV class="ui dropdown item">Team
    <DIV class="menu"><A class="item" href="https://2018.igem.org/Team:NCHU_Taichung/Team">Members</A><A class="item" href="https://2018.igem.org/Team:NCHU_Taichung/Attributions">Attributions</A><A class="item" href="https://2018.igem.org/Team:NCHU_Taichung/Collaborations">Collaborations</A></DIV></DIV><DIV class="ui dropdown item"><DIV class="menu"><A class="item" href="https://2018.igem.org/Team:NCHU_Taichung/Judging#Medals">Medals</A><A class="item" href="https://2018.igem.org/Team:NCHU_Taichung/Judging#Prizes">Prizes</A></DIV></DIV></DIV><DIV class="ui main text container"><DIV class="ui segments" id="mainbox"><DIV class="ui left very close rail" id="railMenu"><DIV class="ui sticky"><DIV class="ui secondary vertical pointing menu"><A class="active item" href="#Genes">Design</A><A class="item" href="#Shuttlevectors">Shuttle vectors</A><A class="item" href="#ExperimentalPlan">Experimental Plan</A><A class="item" href="#Reference">Reference</A></DIV></DIV></DIV><DIV class="ui segment" id="mobileRailMenu"><DIV class="ui sticky"><DIV class="ui secondary vertical pointing menu"><A class="active item" href="#Genes">Design</A><A class="item" href="#Shuttlevectors">Shuttle vectors</A><A class="item" href="#ExperimentalPlan">Experimental Plan</A><A class="item" href="#Reference">Reference</A></DIV></DIV></DIV><DIV class="ui segment" id="topSeg"><H1 class="ui huge header">Design</H1><P class="longP">Our motivation is to solve large area soil contamination using plant-endophyte synergism. We combine phytoremediation
          and our engineered endophyte together to clean dioxin in soil. Two of our genes are derived from the transcriptomic
          data of
          <I>Burkholderia cenocepacia</I> 869T2 under dioxin-containing condition.</P><H2 class="ui dividing header" id="Genes">Genes</H2><H3 class="ui header">TonB</H3><P class="longP">TonB-dependent receptors are membrane bound proteins of Gram-negative bacteria that can actively transport the ferric ion carrier, siderophore, which is similar in structure with dioxin<SUP>[3]</SUP>. </P><P class="longP">Researches have shown that, some macromolecule such as carbonhydrate, heme or vitamin B12 can permeate TonB-dependent receptor<SUP>[1,6]</SUP>.
</P><P class="longP">Our TonB protein is derived from transcriptomic data under dioxin-stressed condition of Burkholderia cenocepacia 869T2. It is up-regulated during early growth period (pre-log phase)<SUP>[2,13]</SUP>. It can be also found in famous dioxin degradable bacteria, <I>Sphingomonas wittichii</I> RW1<SUP>[9]</SUP>.
</P><P class="longP">The docking results have shown that the protein structure can allow the passage of TCDD and some other dioxin compound.</P><DIV class="ui fluid image" id="labelImg"><DIV class="ui pointing label">Figure 1. Dioxin Structure (a) and Prototypical catechol-type siderophore (b). Modified from<SUP>[10]</SUP></DIV></DIV><H3 class="ui header">HAD</H3><P class="longP">Haloacid dehalogenases (HAD) are a family of bacterial enzymes. Generally, most of HAD superfamily members catalyze the dephosphorylation of various compounds, but the catalyzation of carbon group transfer is unique<SUP>[15]</SUP>. 2-haloacid dehalogenase, which we used in our project, is a representative of haloacid dehalogenase (HAD) superfamily, that can catalyze the cleavage of carbon-halogen bonds of halogenated compounds, which are produced worldwide in a large amount of served human usages as solvents, synthetic precursors, agrochemicals, etc<SUP>[16]</SUP>. The catalyzation by 2-HAD needs without a metal ion cofactor while other HAD enzymes need, and that a water nucleophile attacks the Asp C=O in the hydrolysis partial reaction<SUP>[15]</SUP>.</P><P class="longP">2-HADs are classified into three kinds based on substrate specificities, including L-2-haloacid dehalogenase, D-2-haloacid dehalogenase and DL-2-haloacid dehalogenase<SUP>[17]</SUP>. There are two reaction mechanisms proposed for L-2-haloacid dehalogenase. For the first reaction mechanism in Fig. N-A, a carboxylate group of the enzyme attacks the α-carbon atom of the substrate to release the halogen atom to form an ester intermediate. After that, the ester intermediate is hydrolyzed to produce D-2-hydroxyalkanoic acid and regenerate the enzyme. As for the second mechanism, a water molecule directly attacks the substrate to expel the halogen atom in the single step in Figure 2 below<SUP>[17]</SUP>.</P><DIV class="ui fluid image" id="labelImg"><DIV class="ui pointing label">Figure 2. HAD Mechanism. Modified from<SUP>[17]</SUP></DIV></DIV><P class="longP">Since numerous haloacid dehalogenases isolated from various bacteria species, the potential of bacteria using HAD
          to do bioremediation and utilize these compounds as its carbon source increases.</P><H3 class="ui header">Laccase</H3><P class="longP">The laccase we adopt in this system is from a symbiotic fungus of termite (
          <I>Odontotermes formosanus</I>),
          <I>Termitomyces</I> T984, which was isolated from fecal material and responsible for lignin degradation where laccase
          plays a crucial role.</P><P class="longP">The laccase are a kind of peroxidase that has great oxidation ability and has potential to degrade organic compound
          such as diesel, pesticide, antibiotics, PCBs, dioxins etc. In former research, the laccase we use has shown</P><P class="longP">which can attack the ring structure on the dioxin and turn it into some products that can enter lower metabolic pathway
          of bacteria. (Mathew2013, Jiang2016, Zhang2012)</P><H3 class="ui header">Circuit</H3><P class="longP">We use constitutive Anderson promoter J23100 and strong RBS B0034 to express our genes as a polygene (polycistronic)
          circuit. Since the total length of our genes is short enough (less than 5000) we can express it right away.</P><DIV class="ui fluid image" id="labelImg"><DIV class="ui pointing label">Figure 3.</DIV></DIV><H2 class="ui dividing header" id="Shuttlevectors">Shuttle vectors</H2><P class="longP">To express our genes in endophyte, we found some broad host range shuttle vector and get it from addgene. </P><P class="longP">We decided to replace the ori from pSB1C3 with the replication-related proteins and ori on the shuttle vectors. After
          we searched for the restriction sites, we decide to add BamHI and HindIII by PCR on the shuttle vectors and pSB1C3,
          then, digest it and ligate it (Figure 4). </P><DIV class="ui fluid image" id="labelImg"><DIV class="ui pointing label">Figure 4. Shuttle vectors construction process.</DIV></DIV><P class="longP"><B>pME6010</B> is a small, stable, high copy number (n = 5.9 ± 1.8) shuttle vector based on the minimal pVS1 replicon for use in gram-negative especially plant-associated bacteria with tetracycline resistance. The vector we used was originally from Dr. Herman Spaink’s Lab, Leiden University. We bought it from addgene as the name, pMP4657, which contains pLacZ-EGFP<SUP>[4,12]</SUP>.</P><H3 class="ui header">pBBR1</H3><P class="longP">pBBR1 is relatively smaller in size (&lt;5.3kb), and they possess an extended multiple cloning site (MCS). They have the LacZα peptide that allow direct selection of recombinant plasmid in <I>Escherichia coli</I>. Moreover, pBBR1 provide four different antibiotics including kanamycin, ampicillin, tetracycline, gentamycin. This let pBBR1 broaden it usage. The plasmids have been tested and found to replicate in <I> Pseudomonas fluorescens</I>. According <I>Burkholderia </I> genus and <I> Pseudomonas fluorescens</I> are close, pBBR1 can express in <I>Burkholderia </I> genus<SUP>[7]</SUP>.</P><TABLE class="ui celled table"><THEAD><TR><TH>Template</TH><TH>Primers</TH></TR></THEAD><TBODY><TR><TD>pSB1C3 or pSB series plasmid backbone</TD><TD>BBaddBam F
                 CGCGGATCCGCGcgaaaactcacgttaagg
                 BBaddHin R
                 CCCAAGCTTGGGgtattaccgcctttgag
                </TD></TR><TR><TD>pMP4657 (pME6010 with GFP) </TD><TD>V4657addBamF
                 CGCGGATCCgcggctgcatgaaatcct
                 V4657addHinR
                 CCCAAGCTTgggtgtcagtgaagtgct
                </TD></TR><TR><TD>pBBR1MCS </TD><TD>BBRaddBamF
                 CGCGGATCCgcgccatcagatccttg
                 BBRaddBamF2
                 CGCGGATCCgcgggccactcaatgctt
                 BBRaddHinR
                 CCCAAGCTTgggcagggtcgttaaatag
                </TD></TR></TBODY></TABLE><H2 class="ui dividing header" id="ExperimentalPlan">Experimental Plan</H2><P class="longP">
We use 3A method to construct the gene circuit. For the functional test, we use pET24a and pET21b expression vector to test growth curve with and without dioxin and purify proteins. Not only doing HAD activity test, the purified protein was tested by MTT assay for testing the toxicology-cell viability. We use <I>Vetiver</I> and <I>Arabidopsis</I> to analyze TCDD concentration in plants. 

</P><H2 class="ui dividing header" id="Reference">Reference</H2><OL><LI>
        Blanvillain, Servane, Meyer, Damien, Boulanger, Alice, Lautier,
        Martine, Guynet, Catherine, Denancé, Nicolas, . . . Arlat, Matthieu.
        (2007). Plant carbohydrate scavenging through TonB-dependent receptors:
        a feature shared by phytopathogenic and aquatic bacteria. PLoS one,
        2(2), e224.
    </LI><LI>
Chang, Kung-Hao. (2016). To study the change in transcriptome of        <EM>Burkholderia cenocepacia</EM> 869T2, when exposed to dioxin, and to
        predict the relationship between TonB-dependent siderophore receptor
        and dioxin. Master Thesis from the National Chung Hsing University.
    </LI><LI>
        Chimento, David P, Kadner, Robert J, &amp; Wiener, Michael C. (2003).
        The Escherichia coli outer membrane cobalamin transporter BtuB:
        structural analysis of calcium and substrate binding, and
        identification of orthologous transporters by sequence/structure
        conservation. <EM>Journal of molecular biology, 332</EM>(5), 999-1014.
    </LI><LI>
        Heeb, S., Itoh, Y., Nishijyo, T., Schnider, U., Keel, C., Wade, J., ...
        &amp; Haas, D. (2000). Small, stable shuttle vectors based on the
        minimal pVS1 replicon for use in gram-negative, plant-associated
        bacteria. <EM>Molecular Plant-Microbe Interactions</EM>, <EM>13</EM>
        (2), 232-237.
    </LI><LI>
        Jiang, Yu-Han. (2016).The possibility of the degradation of dioxin with
        recombinant Laccase protein from <EM>Odontotermes formosanus</EM>
        symbiosis strain<EM> Termitomyces</EM> T984. Master Thesis from the
        National Chung Hsing University.
    </LI><LI>
        Kai Tang, Nianzhi Jiao, Keshao Liu, Yao Zhang and Shuhui Li. (2012).
        Distribution and Functions of TonB-Dependent Transporters in Marine
        Bacteria and Environments: Implications for Dissolved Organic Matter
        Utilization. <EM>PLoS ONE, 7</EM>(7).
    </LI><LI>
        Kovach, M. E., Elzer, P. H., Hill, D. S., Robertson, G. T., Farris, M.
        A., Roop, R. M., &amp; Peterson, K. M. (1995). Four new derivatives of
        the broad-host-range cloning vector pBBR1MCS, carrying different
        antibiotic-resistance cassettes. <EM>Gene</EM>, <EM>166</EM>(1),
        175-176.
    </LI><LI>
        Mathew, G. M., Mathew, D. C., Lo, S. C., Alexios, G. M., Yang, J. C.,
        Sashikumar, J. M., ... &amp; Huang, C. C. (2013). Synergistic
        collaboration of gut symbionts in Odontotermes formosanus for
lignocellulosic degradation and bio-hydrogen production.        <EM>Bioresource technology</EM>, <EM>145</EM>, 337-344.
    </LI><LI>
        Miller, T. R., Delcher, A. L., Salzberg, S. L., Saunders, E., Detter,
        J. C., &amp; Halden, R. U. (2010). Genome sequence of the
dioxin-mineralizing bacterium Sphingomonas wittichii RW1.        <EM>Journal of bacteriology</EM>, <EM>192</EM>(22), 6101-6102.
    </LI><LI>
        Neilands, J. B. (1995). Siderophores: structure and function of
microbial iron transport compounds.        <EM>Journal of Biological Chemistry</EM>, <EM>270</EM>(45),
        26723-26726.
    </LI><LI>
        Nguyen, Thi Bao Anh. (2017). Biodegradation of dioxins by the
        endophytic bacterium <EM>Burkholderia cenocepacia</EM> 869T2: Role of
        2-haloacid dehalogenase in dehalogenation. Master Thesis from the
        National Chung Hsing University.
    </LI><LI>
        Stuurman, N., Bras, C. P., Schlaman, H. R., Wijfjes, A. H., Bloemberg,
        G., &amp; Spaink, H. P. (2000). Use of green fluorescent protein color
        variants expressed on stable broad-host-range vectors to visualize
rhizobia interacting with plants.        <EM>Molecular Plant-Microbe Interactions</EM>, <EM>13</EM>(11),
        1163-1169.
    </LI><LI>
        Tsai, Chia-Jung. (2018). Transcriptomic analysis of endophytic
        bacterium Burkholderia sp. 869T2 during
        2,3,7,8-Tetrachlorodibenzodioxin decomposing. Master Thesis from the
        National Chung Hsing University.
    </LI><LI>
        Zhang, X., Zhang, S., Pan, B., Hua, M., &amp; Zhao, X. (2012). Simple
        fabrication of polymer-based Trametes versicolor laccase for
decolorization of malachite green. <EM>Bioresource technology</EM>,        <EM>115</EM>, 16-20.
    </LI><LI>
        Burroughs, A. M., Allen, K. N., Dunaway-Mariano, D., &amp; Aravind, L.
        (2006). Evolutionary genomics of the HAD superfamily: understanding the
        structural adaptations and catalytic diversity in a superfamily of
phosphoesterases and allied enzymes.        <EM>Journal of molecular biology</EM>, <EM>361</EM>(5), 1003-1034.
    </LI><LI>
        Kurihara, T. (2011). A mechanistic analysis of enzymatic degradation of
organohalogen compounds.        <EM>Bioscience, biotechnology, and biochemistry</EM>,<EM>75</EM>(2),
        189-198.
    </LI><LI>
        Kurihara, T., Esaki, N., &amp; Soda, K. (2000). Bacterial 2-haloacid
dehalogenases: structures and reaction mechanisms.        <EM>Journal of Molecular Catalysis B: Enzymatic</EM>, <EM>10</EM>(1-3),
        57-65.
    </LI></OL></DIV></DIV></DIV><DIV class="ui inverted vertical footer segment"><DIV class="ui center aligned container"><H4 class="ui inverted header" id="foot">Follow us on</H4></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>