<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_ATOMS-Turkiye_Project_Module1_Experiment"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:ATOMS-Turkiye/Project/Module1:Experiment">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:ATOMS-Turkiye/Project/Module1:Experiment&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:ATOMS-Turkiye/Project/Module1:Experiment&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:ATOMS-Turkiye/Project/Module1:Experiment&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:ATOMS-Turkiye/Project/Module1:Experiment" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:ATOMS-Turkiye/Project/Module1:Experiment</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Experiment_Results_.2F_Discussion"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Experiment Results / Discussion</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#-The_Experiments_about_EpCAM_.E2.80.93_Anti_EpCAM_Binding"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">-The Experiments about EpCAM – Anti EpCAM Binding</SPAN></A></LI><LI class="toclevel-3 tocsection-3"><A href="#Immunofluorescence_c215"><SPAN class="tocnumber">1.1.1</SPAN><SPAN class="toctext">Immunofluorescence c215</SPAN></A></LI><LI class="toclevel-3 tocsection-4"><A href="#Immunofluorescence_OmpA"><SPAN class="tocnumber">1.1.2</SPAN><SPAN class="toctext">Immunofluorescence OmpA</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-5"><A href="#Ellmans_Assay"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Ellmans Assay</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Lsrk.2F_Lslr_Inducible_promoter_experiment"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Lsrk/ Lslr Inducible promoter experiment</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#Discussion_of_ellman.E2.80.99s_assay"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">Discussion of ellman’s assay</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#Protein_G_binding_experiment"><SPAN class="tocnumber">2.3</SPAN><SPAN class="toctext">Protein G binding experiment</SPAN></A></LI><LI class="toclevel-2 tocsection-9"><A href="#DISCUSSION"><SPAN class="tocnumber">2.4</SPAN><SPAN class="toctext">DISCUSSION</SPAN></A></LI><LI class="toclevel-3 tocsection-10"><A href="#Optimization_of_constitutive_promoter_for_protein_over_expression"><SPAN class="tocnumber">2.4.1</SPAN><SPAN class="toctext">Optimization of constitutive promoter for protein over expression</SPAN></A></LI><LI class="toclevel-2 tocsection-11"><A href="#Signal_Peptide"><SPAN class="tocnumber">2.5</SPAN><SPAN class="toctext">Signal Peptide</SPAN></A></LI><LI class="toclevel-2 tocsection-12"><A href="#DISCUSSION_2"><SPAN class="tocnumber">2.6</SPAN><SPAN class="toctext">DISCUSSION</SPAN></A></LI></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Experiment_Results_.2F_Discussion">Experiment Results / Discussion </SPAN></H2><H3><SPAN class="mw-headline" id="-The_Experiments_about_EpCAM_.E2.80.93_Anti_EpCAM_Binding">-The Experiments about EpCAM – Anti EpCAM Binding</SPAN></H3><P>First of all, we wanted to test if our bacteria were able to produce C215 (Anti-EpCAM). To determine this, we performed Immunofluorescence experiments by placing OmpA (a signal peptide which sends the proteins into the membrane) signal peptide and his-tag before the C215 protein sequence in one of our parts. We then accomplished immunofluorescence colouring and were able to see red circles around our bacteria. <I>Including his-tag in the sequence was to our bacteria.</I> During the experiments we used anti-his-tag which was situated after the colouring substance in the sequence. This allowed us to comprehend if our bacteria were able to produce C215; in which case they were.
</P><P>Our second set of immunofluorescence experiments was to confirm the binding of EpCAM to C215. The first set of experiments had proved that our bacteria were successfuly producing C215, however we were unable to comprehend if C215 was able to bind to EpCAM or not. Therefore we carried out another experiment, where purified C215 was presented to cancer cells in well-plates. Our expectations were fulfilled and we were able to see red coloured cancer cells and blue nucleus'.
</P><P>
---
</P><H4><SPAN class="mw-headline" id="Immunofluorescence_c215">Immunofluorescence c215</SPAN></H4><P>For our project to work, the active domain of anti EpCAM was required to bind to the membranes of cancer cells. Pic. 1 show vague red shining cells which prove that our C215 is working. With these results we were able to say that our nanofactories would be able to bind to our cancer cells successfully. 
</P><H4><SPAN class="mw-headline" id="Immunofluorescence_OmpA">Immunofluorescence OmpA</SPAN></H4><P>Before beginning with experiments we decided to use an alternative method to detect the cancer cells. We decided to use our bacteria as nanofactory itself (quorum sensing). Therefore OmpA was placed before C215 to determine so that our bacteria would bind to the cancer cells. We tested the production of OmpA by our bacteria with immunofluorescence experiments and were able to see shinning red lights in the Pic 2 which proved that our OmpA was presented by our cell’s outer membrane successfully; meaning that our bacteria could be used as a nanofactory.
</P><H1><SPAN class="mw-headline" id="Ellmans_Assay">Ellmans Assay</SPAN></H1><P>Ellman’s assay is a test which enables us to measure the concentration of AI-2. In order to perform Ellman’s assay liquid culture of bacteria which produce the enzymes Luxs-Pfs were required. The liquid cultures resistant to CHL were incubated for 16 hours which then followed on by adding 0,1 mM substrate (SAH in 10X PBS containing %1 BSA) and SAH containing liquid cultures were continued to be incubated but this time at room temprature for a period of two hours. During the incubations, our enzymes were producing autoinducer 2. To determine the quantity of autoinducer-2 produced, we centrifuged and measured light density of our supernatants with the light spectrophotometer. Our negative control  were transformed bacteria. The results of the Ellman’s assay our below:
</P><P>This results show that enzymes are expressed by e.coli are succesfully producing autoinducer2.
</P><H3><SPAN class="mw-headline" id="Lsrk.2F_Lslr_Inducible_promoter_experiment">Lsrk/ Lslr Inducible promoter experiment</SPAN></H3><P>To observe inducible promoters response, we co incubated two different bacteria. The first bacteria culture was expressing the enzyme system luxs-pfs and the second culture included our inducible promoter which produce RFP in the presence of AI-2. During the experiment, we incubated them seperately for 16 hours then mixed them into one flask and waited for 4 hours, we then add 0,1 mM SAH (with 10x PBS containing %1 BSA). After incubating them with SAH for one day, we took 2 ml of liquid culture into centrifuge tubes for centrifugation. Below is a photo which shows the resulting red pellet of our experiment meaning that our inducible promoter was succesfully working to produce AI-2.
</P><H3><SPAN class="mw-headline" id="Discussion_of_ellman.E2.80.99s_assay">Discussion of ellman’s assay</SPAN></H3><P>Our results proved that our system was working as expected and the challenges were overcome. According to our Ellman’s assay experience, our enzyme system is succesfully producing auto inducer 2. As described in the ‘design’ section, AI-2 will induce bacteria to move towards the cancer cells.
We searched for an inducible system which would become activated at the presence of antigen on bacteria. As an activating molecule, autoinducer2 activates LsrR-LsrK promoter system for express the killer protein by bacteria.
The centrifuged tubes of liquid cultures were addedwith an inducible promoter containing the RFP gene which would be a marker of its activation. Therefore the results of co-incubation significantly show us that the promoter system is activated in the presence of autoinducer-2.
</P><H2><SPAN class="mw-headline" id="Protein_G_binding_experiment">Protein G binding experiment</SPAN></H2><P>Our nanofactory is formed with an antibody, Protein G and an enzyme complex. A connection between the enzyme system and antibody was required, therefore we used a linker. We used a molecule called protein G which was derived from group C and G Streptococcus strains that binds to the Fc region of numerous immunogloblulins. Protein G facilitates binding of enzyme construct to the Fc region of the targeting antibody. We also cloned the nanofactories into pSB1C3 and we expressed this protein in the BL21 strain of E. coli. Futhermore we tagged his6x at N terminus of protein G for purification.
</P><P>The aim of protein G and Ig G binding experiment was to construct a nanofactory complex composed of two parts that is the antibody and enzyme system. 
Protein G has a natural affinity  to bind to the antibody in phosphate buffer. Therefore we incubated protein G with the antibodies in 10mM phosphate buffer for 2,5 hours and measured the quantity of fused proteins via Western Blotting experiment.
Result;
</P><P>
Our protein G weight was 25 kDas and our combined Protein G + antibody protein’s weight was about 50 kDas.  In our view, the experimental results were acceptable as the weight of protein G is known to be approximately 25 kDas line as well as the complex being close 50 kDas. 
</P><H2><SPAN class="mw-headline" id="DISCUSSION">DISCUSSION</SPAN></H2><P>First of all, liquid culture was prepared for our TorA-GFP and GFP-HlyA experiments. We added TritonX into half of our TorA-GFP samples and didn’t add any chemicals into the other half of TorA-GFP samples . After the Overnight process of incubation we centrifuged to receive our supernatants from our samples because our aim was to extract the proteins.
The results of the TorA-GFP supernatant samples showed that the samples with TritonX &amp; Glycine had no GFP. We assumed this was due to the cellular necroses due to TritonX &amp; glycine being high in concentration  caused secretion of low amount of GFP. Because of the secretion of the GFP in periplasmic gap we used %2 TritonX*+%1 Glycine*  that wrote in article and we destroy outer memrane. We got this concentration from article.
Our TorA-GFP experiment results show that the samples with only TritonX added contains more GFP than the Negative(-) Control. The results show us that our signal peptides worked properly and our chemicals TritonX and Glycine showed the right effects of our desires. 
GFP-HlyA also resulted in the production of high amount of GFP in our Negative(-) Controls.
Another set of results showed that liquid culture of only LB produced maximum amount of GFP. 
</P><UL><LI>Tang et al 2008 ., Jin-bao Tanga, Hong-ming Yanga, Shu-liang Songb, Peng Zhub, Ai-guo Jib
</LI></UL><P>(Effect of Glycine and Triton X-100 on secretion and expression of ZZ–EGFP fusion protein)
1)Blobel G, Dobberstein B. (Dec. 1975). &quot;Transfer of proteins across membranes. I. Presence of proteolytically processed and unprocessed nascent immunoglobulin light chains on membrane-bound ribosomes of murine myeloma.&quot;
</P><H3><SPAN class="mw-headline" id="Optimization_of_constitutive_promoter_for_protein_over_expression">Optimization of constitutive promoter for protein over expression</SPAN></H3><P>The over expression experiments aim was to increase the synthesis of our protein in a new medium we prepared. We predicted that the over expression of our desired protein by our bacteria would be a good idea to observe the net results. So we decided to prepare the “best medium” for the bacteria. Then we generated different media combinations. We generated the combinations by altering conditions of bacteria, liquid culture incubation time, bacteria strain and temperature of incubator. So at the end of the experiment there were 60 different conditions and media we designed.
Different media and conditions in our experiment: 
</P><P>After preparing the media we started the experiment stage. Our experiment includes protein isolation and SDS page parts.  We got the total amount of protein of the bacteria after the protein isolation by using sonicator. Then we measure the band of our proteins in the SDS Page experiment.  Finally we chose the winner a medium called TNY, BL21, 24H, 37C
SDS Page results; 
</P><PRE>           TNY, BL21, 37C, 24Hr                        TNY, NEB10, 37C, 24Hr
</PRE><P>
We were excepting LB as the expression maximizer but the results were surpising as the TNY medium resulted in high level of protein. İngredients of LB and TNY were the same. The only difference was the concentration of each component. Interestingly, the medium with low density was better than our routine medium, LB. There was small percentage dissimilarity between the medias like 7/200 and 4/200. Our TNY was prepared by diluting 4gr TNY extract with distilled water.
To conclude, protein expression level of TNY is 10  times higher than LB(see above).
</P><H2><SPAN class="mw-headline" id="Signal_Peptide">Signal Peptide </SPAN></H2><P>A signal peptide (sometimes referred to as signal sequence, leader sequence or leader peptide) is a short (5-30 amino acids long) peptidepresent at the N-terminus of the majority of newly synthesized proteins that are destined towards the secretory pathway.[1] 
In prokaryotes, signal peptides direct the newly synthesized protein to the SecYEG protein-conducting channel, which is present in the plasma membrane.
In this part of our project we tested are our signal peptides is working or not
To secrete GFP protein via signal peptide to extracellular membrane. 
In overall project, our aim related with signal peptide is to secrete apoptotic proteins outer membrane. If this construct secrete GFP successfully, that means signal peptide works and TorA can be used in Apoptin or E4orf4 export.  
And also we use constitutive promoter in our designs because we need high amount of protein and we want to our bacteria always produce our proteins. 
We made santrifuge our signal peptides’ (TorA-GFP ve GFP-HlyA) liquid culture falcons and then get supernatant from falcons. Then we measured values of absorption of GFP in our supernatant in suitable wavelength. Results show that we found high amount of GFP in our supernatants.                         
Result; 
</P><P>Column scale(y)  shows us quantity amount of GFP in the solution.  
</P><H2><SPAN class="mw-headline" id="DISCUSSION_2">DISCUSSION</SPAN></H2><P>Firstly we made liquid culture in our TorA-GFP and GFP-HlyA experiments. We added TritonX half of our TorA-GFP samples and didn’t add any chemicals our other TorA-GFP samples . After the Overnight process we made santrifuge and we get supernatants from our samples because we wait our proteins in supernatant.
The results of the supernatant of the TorA-GFP samples that we didn’t add TritonX &amp; Glycine were over 0. So we thought that we got this results becuse of the cellular necroses that caused secretion of low amount of GFP. Because of the secretion of the GFP in periplasmic gap we used %2 TritonX*+%1 Glycine*  that wrote in article and we destroy outer memrane. We got this concentration from article.
Our results in our TorA-GFP experiments that our samples that added TritonX contains more GFP amount than Negative(-) Control. So this results showed us that our signal peptides worked properly and our chemicals TritonX and Glycine showed their effects. 
Also we get another fine results from our GFP-HlyA that we found high amount of GFP in our samples more than our Negative(-) Control.
Another good results that our Positive Control which is GFP that added in LB showed maximum amount of GFP in graphics. So we can said that our controls were working too.
</P><UL><LI>Tang et al 2008 ., Jin-bao Tanga, Hong-ming Yanga, Shu-liang Songb, Peng Zhub, Ai-guo Jib
</LI></UL><P>(Effect of Glycine and Triton X-100 on secretion and expression of ZZ–EGFP fusion protein)
1)Blobel G, Dobberstein B. (Dec. 1975). &quot;Transfer of proteins across membranes. I. Presence of proteolytically processed and unprocessed nascent immunoglobulin light chains on membrane-bound ribosomes of murine myeloma.&quot;
</P><P>We were excepting the LB for the winner of the experiment but we surprised by seeing the result.  Because the medium that prepared by us and called TNY was the best. The content of TNY and LB is the similar but the component percentage of media is different. It is very interesting that low rated medium is better than the LB. There is small percentage dissimilarity between the media like 7/200 and 4/200. Our TNY is preparing by diluting 4 gr TNY extract with distilled water. 
We observed the protein expression level of TNY 10 times more than LB as you see above.
</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:ATOMS-Turkiye/Project/Module1:Experiment">http://2013.igem.org/Team:ATOMS-Turkiye/Project/Module1:Experiment</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:ATOMS-Turkiye/Project/Module1:Experiment" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:ATOMS-Turkiye/Project/Module1:Experiment" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:ATOMS-Turkiye/Project/Module1:Experiment&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:ATOMS-Turkiye/Project/Module1:Experiment&amp;oldid=363499" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>