"
Page
Discussion
View source
History
teams
Log in
 
Team:CBNU-Korea/Project
From 2010.igem.org
Home
Team
Minimal Chromosome
DB&Program
Parts
Notebook
Safety
Contact
The CBNU 2009 iGEM team, our team's goal was to manage and retrieve required data and explore informations through mapping essential genes into orthologs of interesting genomes and through statistical analyses of gene order and direction, so on.
Previous iGEM teams have planned about making synthetic gemone for synthetic cell and programe to design the minimal genome.
We managed and retrieved required data and explored informations through mapping essential gens into orthologs of interesting genomes and through statistical analyses of gene order and direction, so on.
Now we are trying putting the plan in practice.  Our final main purpose is synthesis a synthetic cell.  If you want synthesis a synthetic cell, you need a minimal genome.
The minimal genome is the smallest possible group of genes that would be sufficient to sustain a cellular life form under the most favorable conditions imaginable.  There are two ways to synthesize a minimal genome, which are top-down and bottom-up approaches.  We are interested in the bottom-up approach, in which the major hurdle is how to degine the minimal genome.  We don't know the principle of the organization of the genome.  Moreover to verify the design experimentally, we should to deal with some 200 or more essential genes.  So, we use the divide-and-conquer strategy to solve the problems in designing the minimal genome.
We will make E.coli having two chromosomes.  One is a normal chromosome and the other is a normal chromosome and the other is a model minimal chromosomes, in which the replication and segregation processes and their regulation are occurred independently.  Studies on the minimal chromosome will provide us useful information about designing the minimal genome.
This project's purpose is design and making of Minimal Synthetic Chromosome(MSC).  Then we need this step for successful process.
(1) Methodical classification of bacterial chromosome copy and division
(2) Development of oriC-finder program for finding Replication Origin of Chromosome(oriC)
(3) Making Bacterial chromosome copy and Division database
(4) Stepwise production and investigation of Minimal Synthetic Chromosome(MSC)
ABSTRACT
Most of all bacteria have single circular  chromosome. But some bacteria have two or more circular chromosomes. In Vibrio cholerae O1 biovar eltor str.  N16961, there are two circular chromosomes, chromosome I and chromosome II, and  each perfectly works as a chromosome.
We’ve been motivated by V.cholerae two chromosome system. So we  employed some essential genes, parA,  parB, parS, dif, rctA, rctB and  origin of chromosome II and constructed a tiny miniature of V.cholerae two chromosome system in E.coli, using BioBrick assembly method.
Also, we built software and database of  essential genes for designing of minimal synthetic chromosome and genome.  Essential gene information was gathered from some databases, DEG, NCBI and java  language was used.
PROJECT
Introduction
Synthetic  minimal genome is the smallest possible group of genes that would be sufficient  to sustain a cellular life form under the most favorable conditions imaginable  that working like natural chromosome, bacteria have. For synthesizing minimal  genome, there are many things that we have to consider such as kind of genes,  gene arrangement, direction of genes, codon usage, genome structure, stability  and so on. Therefore it is very difficult to synthesize minimal genome  considering ALL of these. So we synthesize simple minimal chromosome employing V.cholerae O1 biovar eltor str. N16961chromosome II replication and partition  system as the first step of synthetic minimal genome. Also, we re-build essential  gene database from DEG and NCBI. And using this database, we build genome  designer easy to design, too.
Approach  1: Replication and Partition system of V.  cholerae
In Vibrio cholerae, there are two circular  chromosomes, chromosome I(2.96Mb) and chromosome II(1.07Mb), and each perfectly  works as a chromosome. Because chromosome II has different replication protein  from main chromosome (chromosome I), RctA and RctB. Also chromosome replication  origin structure is different each other. Replication origin of chrI(  chromosome I ), oriC1vc is similar in  sequence to the origin of replication of the Escherichia coli chromosome, oriC.oriC2vc has very little similarity to oriC1vc. oriC2vc has plasmid-like feature(sequences resembling iterons)(Egan  and Waldor 2003 and Vendoka-Canova et al 2006).
Fig. 1. Vibrio cholerae O1 biovar eltor str. N16961 recombinant structure of oriC2vc
 
The  two genes rctA and rctB flanking oriC2vc are so far known unique to replication in the Vibrio family(  Egan et al, 2006). The rctA gene is  not actually protein coding gene. It codes of an untranslated RNA and acts as a  DNA to inhibit oriC2vc function by  binding RctB. RctA is not conserved in other vibrio(Venkova-Canova et al 2006).  RctB is initiator protein of V.cholerae chrII replication. The RctB protein binds to disparate specific DNA sequences,  which gives RctB multiple functions. Also this initiator protein regulates the  transcription of rctA and rctB. RctB acts an autorepressor  repressing its own transcription as well as rctA.  RctB promoter activity is autoregulated by RctB. Also,  we searched some articles to study. According to article about replication of V.cholerae, RctB protein is needed to  start replication as initiator. So that research has two vectors. First one includes  oriC2vc and another one has pBAD+rctB for start rctB replication system. So, we  make synthetic promoter and insert it into oriC2vc, upstream of rctB gene. PrctB(syn)  have R13 that repeat sequence essential for replication. But PrctB(syn) have strong  promoter from E.coli sigma70 factor. And have strong SD sequence. PrctB (syn) for the minimal chromosome  replication initiation and chromosome RctB will allow themselves could create.Also, we searched some articles to study. According  to article about replication of V.cholerae, RctB protein is needed to start replication as initiator. So that research  has two vectors. First one includes oriC2vc and another one has pBAD+rctB for start  rctB replication system. So, we make synthetic promoter and insert it into oriC2vc, upstream of rctB gene. PrctB(syn) have R13 that repeat  sequence essential for replication. But PrctB(syn) have strong promoter from  E.coli sigma70 factor. And have strong SD sequence. PrctB (syn) for the minimal chromosome replication  initiation and chromosome RctB will allow themselves could create.Termination of replication is  accomplished at the dif DNA site.
Partition of V.cholerae chrII is accomplished by parA, parB and parB binding site parS. Partitioning  genes(par genes) have been known to play an essential role in the stable  inheritance of certain plasmids. ParB proteins bind to their cognate pars sites and form ParB-parS nucleoprotein complex. Then, ParA  protein interacts with the ParB-parS complex and their ATPase activity is essential for partitioning. Still, the  molecular mechanism by which the interaction of ParA with the ParB-parS mediates localization and  partitioning are not well understood.
Fig. 2. Recombinant structure of partitioning system of v.cholerae chromosome II.
Retrieved from " http://2010.igem.org/Team:CBNU-Korea/Project "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
