"
Page
Discussion
View source
History
teams
Log in
 
Team:CBNU-Korea/Parts
From 2010.igem.org
Home
Team
Minimal Chromosome
DB&Program
Parts
Notebook
Safety
Contact
To synthesize the minimal chromosome, we  selected some essential genes and got each gene sequence from NCBI Genome  database. Replication origin(including rctA, rctB) , parA, parB, parS, dif and related promoter. Designing primer including BioBrick prefix and suffix using  GinkoPrimer tool.
parS and dif is very short  sequence DNA. So we decided to make these genes using PCR primer including  BioBrick prefix and suffix.
   parS, ParB protein binding sites have 9  different sequences and positions in V.cholerae chrII each other.  So we take consensus  sequence of 9 parS sequences.
 To  checking the V.cholerae origin is working  well, we first used BioBrick base vector BBa_I51020. But there is ccdb positive selection marker that  killed the host. So we decide to re-design the pSB1C3 BioBrick vector using  PCR. And we got the re-designed vector, Cn. The Cn has 2 NheI restriction  enzyme sites instead rep protein coding gene. XbaI/SpeI can make mixed sequence  with NheI.
EcoRI and PstI sites are located in parB gene. Because of this, we should  cut parB to XbaI/SpeI. It could be  inserted into Cn, NheI site. But we have to consider gene arrangement.
DNA Amplification
Concentration of  template DNA and primer. Add distilled water to AccuPower™ HF PCR PreMix(Bioneer) tubes to a  total volume of 20㎕.
Step
Temperature
Time
Number of cycles
Initial    Denaturation
94℃
5min
1cycle
Denaturation
94℃
1min
The cycle number is    dependent on the amount of template DNA.
Annealing
68℃
1min
Extension
72℃
1min/kb
Final Extension
72℃
5min
1cycle
In case of long PCR  It is recommended to use two-step PCR methods.
Step
Temperature
Time
Number of cycles
Initial    Denaturation
94℃
5min
1cycle
Denaturation
94℃
20sec
The cycle number is    dependent on the amount of template DNA.
Annealing/Extension
68℃
1min
Final Extension
72℃
5min
1cycle
Plasmid Mini  Extraction
The overall principle  combines modified alkaline lysis method. Collected cells are re-suspended in Resuspension  Buffer. Following the addition of Lysis Buffer and Neutralization Buffer to the  lysate, the chromosomal DNA and cell debris will be forms an insoluble  aggregate. The insoluble protein aggregate is removed following centrifugation  and transfer the clear lysate to the DNA binding filter tube. The cleared  lysate contains a chaotropic salt originating from Neutralization Buffer which  helps the binding of the plasmid DNA on the membrane surface. The DNA binding  filter which is packed with silica based membrane has enough surface area to  bind up to 20㎕ of plasmid DNA. Any  salts and precipitates are eliminated by addition of the Washing Buffer.  Finally, highly purified plasmid DNA can be eluted with Elution Buffer or  Nuclease free autoclaved distilled water.
Gel Purification
Gel Purification is  designed for purification of up to 10㎕ of fragment DNA from low-melting, TAE and TBE agarose gel within  15minutes. The size range for effective purification is about 70bp~10kb. The  average recovery yield exceeds 70%~90%. Elution volume can be as little as 30㎕ when concentrated product is needed. The principle of this kit is  based on adsorption of DNA onto the silica based membrane by chaotropic salt.  And chaotropic salts enhance melting of agarose gel but binding of DNA onto the  silica based membrane that is packed in a binding column tube. Adsorption of  DNA is so selective that molten agarose and salts are not adsorbed and pass  through the binding column tube. Washing eliminates salts and residual agarose  gel. High-purity DNA fragments are eluted with provided Elution Buffer or  distilled water.
Electrophoresis
Dissolve the proper  amount of agarose in 1X TAE buffer and boil, then let cool to about 60℃. Add EtBr. After carefully mixing the  EtBr, slowly pour the agarose solution into the assembled gel caster. Pour  slowly to avoid the formation of bubbles. Let the gel solidify at room  temperature. After the gel has solidified, remove each comb carefully. If the  comb seems to be stuck, pour a little D.W then remove. Check that the gel in  the caster does not slide, and that the gel is fully submerged in the buffer  after being fitted in the Agaro Tank. Place the Agaro Tank Lid over the Agaro  Tank. Make sure that the (-) side faces up. Load the prepared samples using a  multi-channel pipette onto the loading adapter. The samples will be accurately  loaded onto each well. Also, check the number of the holes to reduce loading  errors. Select the voltage and proceed with electrophoresis.
Parts
<groupparts>iGEM010 CBNU-Korea</groupparts>
Retrieved from " http://2010.igem.org/Team:CBNU-Korea/Parts "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
