<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_BCCS-Bristol_Notebook"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:BCCS-Bristol/Notebook">Page               </A></LI><LI><A href="/Talk:Team:BCCS-Bristol/Notebook">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:BCCS-Bristol/Notebook&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:BCCS-Bristol/Notebook&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:BCCS-Bristol/Notebook" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:BCCS-Bristol/Notebook</H1><DIV id="bodyContent"><H3 id="siteSub">From 2008.igem.org</H3><DIV class="bccsNavBar"><TABLE align="center"><TBODY><TR><TH align="center"><A href="/Team:BCCS-Bristol" title="Team:BCCS-Bristol">Home</A></TH><TH align="center"><A href="/Team:BCCS-Bristol/Team" title="Team:BCCS-Bristol/Team">The Team</A></TH><TH align="center"><A href="/Team:BCCS-Bristol/Project" title="Team:BCCS-Bristol/Project">The Project</A></TH><TH align="center"><A href="/Team:BCCS-Bristol/Parts" title="Team:BCCS-Bristol/Parts">Submitted Parts</A></TH><TH align="center"><A href="/Team:BCCS-Bristol/Modeling" title="Team:BCCS-Bristol/Modeling">Modelling</A></TH><TH align="center"><STRONG class="selflink">Wet Lab</STRONG></TH><TH align="center"><A href="/Team:BCCS-Bristol/Calendar" title="Team:BCCS-Bristol/Calendar">Calendar</A></TH><TH align="center"><A href="/Team:BCCS-Bristol/Misc" title="Team:BCCS-Bristol/Misc">Miscellaneous</A></TH></TR></TBODY></TABLE></DIV><DIV class="bccsMain"><H2><SPAN class="mw-headline" id="Protocols">Protocols</SPAN></H2><UL><LI><A href="/Team:BCCS-Bristol/Protocols-Agarose_Gel_Electrophoresis" title="Team:BCCS-Bristol/Protocols-Agarose Gel Electrophoresis"><B>Agarose Gel Electrophoresis</B></A></LI><LI><A href="/Team:BCCS-Bristol/Protocols-Colony_PCR" title="Team:BCCS-Bristol/Protocols-Colony PCR"><B>Colony PCR</B></A></LI><LI><A href="/Team:BCCS-Bristol/Protocols-DMSO_stocks" title="Team:BCCS-Bristol/Protocols-DMSO stocks"><B>DMSO stocks</B></A></LI><LI><A href="/Team:BCCS-Bristol/Protocols-Preparation_of_electro_competent_cells" title="Team:BCCS-Bristol/Protocols-Preparation of electro competent cells"><B>Preparation of electro competent cells</B></A></LI><LI><A href="/Team:BCCS-Bristol/Protocols-Media" title="Team:BCCS-Bristol/Protocols-Media"><B>Media</B></A></LI><LI><A href="/Team:BCCS-Bristol/Protocols-Resuspending_Primers" title="Team:BCCS-Bristol/Protocols-Resuspending Primers"><B>Resuspending Primers</B></A></LI><LI><A href="/Team:BCCS-Bristol/Protocols-Transformation_with_heat_shock" title="Team:BCCS-Bristol/Protocols-Transformation with heat shock"><B>Transformation with heat shock</B></A></LI><LI><A href="/Team:BCCS-Bristol/Protocols-Transformation_using_electroporation" title="Team:BCCS-Bristol/Protocols-Transformation using electroporation"><B>Transformation using electroporation</B></A></LI><LI><A href="/Team:BCCS-Bristol/Protocols-Testing_the_activity_of_biobrick_T9002" title="Team:BCCS-Bristol/Protocols-Testing the activity of biobrick T9002"><B>Testing the activity of biobrick T9002</B></A></LI></UL><H2><SPAN class="mw-headline" id="Progress_Reports">Progress Reports</SPAN></H2><UL><LI><A href="/Team:BCCS-Bristol/Notebook#1st_July_-_8th_July" title="Team:BCCS-Bristol/Notebook"><B>1st July - 8th July </B></A></LI><LI> Swimming agar assay 
</LI></UL><UL><LI><A href="/Team:BCCS-Bristol/Notebook#9th_July_-_15th_July" title="Team:BCCS-Bristol/Notebook"><B>9th July - 15th July </B></A></LI><LI>Swimming agar assay with L-aspartic acid
</LI></UL><LI><A href="/Team:BCCS-Bristol/Notebook#16th_July_-_22nd_July" title="Team:BCCS-Bristol/Notebook"><B>16th July - 22nd July </B></A></LI><UL><LI><A href="/Team:BCCS-Bristol/Notebook#23rd_July_-_29th_July" title="Team:BCCS-Bristol/Notebook"><B>23rd July - 29th July </B></A></LI></UL><UL><LI><A href="/Team:BCCS-Bristol/Notebook#30th_July_-_6th_August" title="Team:BCCS-Bristol/Notebook"><B>31st July - 6th August</B></A></LI><LI> BioBrick Transformation
</LI><LI> Heat shock transformations with BioBrick DNA resulted in zero, one or two colonies per attempt
</LI><LI> All colonies have been verified to possess the plasmids
</LI><LI> Electroporation is much more successful
</LI></UL><LI> Agar plug assay
</LI><LI><I>E. coli</I> MG1655 moved apparently some beads!!!
</LI><LI>but unfortunenately, no chemotaxis was observed
</LI><UL><LI><A href="/Team:BCCS-Bristol/Notebook#7th_August_-_13th_August" title="Team:BCCS-Bristol/Notebook"><B>7th August - 13th August</B></A></LI><LI> BioBrick Transformation
</LI><LI>The self-made electro competent <I>E. coli</I> DH5α cells are working
</LI><LI>A longer incubation time of the punched paper disc in water seems to resolve more BioBrick DNA
</LI></UL><UL><LI><A href="/Team:BCCS-Bristol/Notebook#14th_August_-_20th_August" title="Team:BCCS-Bristol/Notebook"><B>14th August - 20th August</B></A></LI><LI> Agar plug assay
</LI><LI> Repulsion achieved to some degree in the agar-plug assay. 
</LI></UL><LI> BioBrick transformation
</LI><LI> DH5α transformed with pRZ1(pRS415 containing cpxR responsive promotor - sent to us by T.Silhavy, Princeton)
</LI><UL><LI><A href="/Team:BCCS-Bristol/Notebook#2nd_September_-_9th_September" title="Team:BCCS-Bristol/Notebook"><B>2nd September - 9th September</B></A></LI><LI> BioBrick transformation
</LI></UL><UL><LI><A href="/Team:BCCS-Bristol/Notebook#10th_September_-_16th_September" title="Team:BCCS-Bristol/Notebook"><B>10th September - 16th September</B></A></LI><LI> BioBrick transformation
</LI></UL><UL><LI><A href="/Team:BCCS-Bristol/Notebook#17th_September_-_25th_September" title="Team:BCCS-Bristol/Notebook"><B>17th September - 25th September</B></A></LI><LI> Building the new BioBricks
</LI></UL><UL><LI><A href="/Team:BCCS-Bristol/Notebook#26th_September_-_16th_October" title="Team:BCCS-Bristol/Notebook"><B>26th September - 16th October</B></A></LI><LI> BioBrick testing and sequencing
</LI></UL><H2><SPAN class="mw-headline" id="Progress_Report_Details">Progress Report Details</SPAN></H2><H3><SPAN class="mw-headline" id="1st_July_-_8th_July">1st July - 8th July</SPAN></H3><H4><SPAN class="mw-headline" id="Swimming_agar_assay">Swimming agar assay</SPAN></H4><P>We wanted to test different conditions for the swimming agar assay: 
</P><P>- different agar concentrations (0.2 %; 0.3 %; 0.5 %)
</P><P>- different incubation temperatures (25°C; 30°C; 37°C)
</P><P>- different inoculation depths (On the surface; Under the surface; Deep [but not on the bottom])
</P><P>Therefore, we used an overnight culture of <I>Escherichia coli</I> K-12 MG1655 grown in <A href="/Team:BCCS-Bristol/Protocols-Media" title="Team:BCCS-Bristol/Protocols-Media">LB broth</A>. 10 µl of the bacteria solution was inoculated with a pipette tip in swimming agar medium plates (25 ml of <A href="/Team:BCCS-Bristol/Protocols-Media" title="Team:BCCS-Bristol/Protocols-Media">Swimming agar medium</A> in petri dishes).
</P><P>
After an incubation overnight, we measured the diameter of the <I>E. coli</I> MG1655 swarms. Surprisingly, we observed two swarm layers: The lower one had a bigger diameter than the upper. The measured diameters are depicted in the table. 
</P><TABLE align="center"><TBODY><TR><TH align="center" width="10%"><I>E. coli</I> MG1655: Lower and upper layer in swimming agar plates
</TH></TR></TBODY></TABLE><TABLE align="center" border="1" cellpadding="4" cellspacing="0" style="border:#c9c9c9 1px solid; margin: 1em 1em 1em 0; border-collapse: collapse;"><TBODY><TR><TD align="center" style="background:#f0f0f0;"><B>Agar (%)</B></TD><TD align="center" style="background:#f0f0f0;"><B>Temperature (°C)</B></TD><TD align="center" style="background:#f0f0f0;"><B>Inoculation Depth</B></TD><TD align="center" style="background:#f0f0f0;"><B>Number</B></TD><TD align="center" style="background:#f0f0f0;"><B>JPG File</B></TD><TD align="center" style="background:#f0f0f0;"><B>Inner Ring (cm)</B></TD><TD align="center" style="background:#f0f0f0;"><B>Outer Ring (cm)</B></TD></TR><TR><TD> 0.3</TD><TD>30</TD><TD>Deep</TD><TD>1</TD><TD>4828</TD><TD>4.6</TD><TD>5.3
</TD></TR><TR><TD> 0.3</TD><TD>30</TD><TD>Deep</TD><TD>2</TD><TD>4827</TD><TD>4.6</TD><TD>5.3
</TD></TR><TR><TD> 0.3</TD><TD>30</TD><TD>Deep</TD><TD>3</TD><TD>4829</TD><TD>4.7</TD><TD>5.2
</TD></TR><TR><TD> 0.3</TD><TD>30</TD><TD>Under Surface</TD><TD>1</TD><TD>4830</TD><TD>4.6</TD><TD>5.2
</TD></TR><TR><TD> 0.3</TD><TD>30</TD><TD>Under Surface</TD><TD>2</TD><TD>4831</TD><TD>4.6</TD><TD>5.2
</TD></TR><TR><TD> 0.3</TD><TD>30</TD><TD>Under Surface</TD><TD>3</TD><TD>4832</TD><TD>4.6</TD><TD>5.1
</TD></TR><TR><TD> 0.3</TD><TD>30</TD><TD>On Surface</TD><TD>1</TD><TD>4833</TD><TD>2.9</TD><TD>3
</TD></TR><TR><TD> 0.3</TD><TD>30</TD><TD>On Surface</TD><TD>2</TD><TD>4834</TD><TD>3.8</TD><TD>4.4
</TD></TR><TR><TD> 0.3</TD><TD>30</TD><TD>On Surface</TD><TD>3</TD><TD>4835</TD><TD>4.4</TD><TD>4.9
</TD></TR><TR><TD> 0.3</TD><TD>37</TD><TD>Under Surface</TD><TD>1</TD><TD>4836</TD><TD>7.2</TD><TD>7.6
</TD></TR><TR><TD> 0.3</TD><TD>37</TD><TD>Under Surface</TD><TD>2</TD><TD>4837</TD><TD>7.4</TD><TD>7.7
</TD></TR><TR><TD> 0.3</TD><TD>37</TD><TD>Under Surface</TD><TD>3</TD><TD>4838</TD><TD>7.1</TD><TD>7.4
</TD></TR><TR><TD> 0.3</TD><TD>25</TD><TD>Under Surface</TD><TD>1</TD><TD>4840</TD><TD>2.8</TD><TD>3.5
</TD></TR><TR><TD> 0.3</TD><TD>25</TD><TD>Under Surface</TD><TD>2</TD><TD>4839</TD><TD>3</TD><TD>3.6
</TD></TR><TR><TD> 0.3</TD><TD>25</TD><TD>Under Surface</TD><TD>3</TD><TD>4841</TD><TD>3.1</TD><TD>3.7
</TD></TR><TR><TD> 0.2</TD><TD>30</TD><TD>Under Surface</TD><TD>1</TD><TD>4842</TD><TD>8.6</TD><TD>8.6
</TD></TR><TR><TD> 0.2</TD><TD>30</TD><TD>Under Surface</TD><TD>2</TD><TD>4843</TD><TD>8.6</TD><TD>8.6
</TD></TR><TR><TD> 0.2</TD><TD>30</TD><TD>Under Surface</TD><TD>3</TD><TD>4844</TD><TD>8.6</TD><TD>8.6
</TD></TR><TR><TD> 0.5</TD><TD>30</TD><TD>Under Surface</TD><TD>1</TD><TD>4847</TD><TD>2.3</TD><TD>2.3
</TD></TR><TR><TD> 0.5</TD><TD>30</TD><TD>Under Surface</TD><TD>2</TD><TD>4845</TD><TD>1.9</TD><TD>1.9
</TD></TR><TR><TD> 0.5</TD><TD>30</TD><TD>Under Surface</TD><TD>3</TD><TD>4846</TD><TD>1.5</TD><TD>1.5
</TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="9th_July_-_15th_July">9th July - 15th July</SPAN></H3><H4><SPAN class="mw-headline" id="Swimming_agar_assay_with_L-aspartic_acid">Swimming agar assay with L-aspartic acid</SPAN></H4><P>We conducted a second swimming agar assay with <I>E. coli</I> MG1655. This time, we added paper discs that were soaked with a L-aspartic acid solution. We wanted to observe whether the bacteria are attracted by the amino acid. We tested different parameters:
</P><P>- agar concentrations of 0.15 %, 0.3 % and 0.5 %
</P><P>- incubation temperatures of 25 °C, 30 °C and 37 °C
</P><P>- inoculations depths of &quot;on the surface&quot;, &quot;under the surface&quot; and &quot;deep in the medium&quot;
</P><P>- aspartic acid concentrations of 0.2 %, 0.1 % and 0.01 %
</P><P>Photos of swimming agar were taken and travelling distances measured. 0.3% agar was found to be the best for an overnight culture. A temperature of 37°C was too high, since the swarms spread over the whole plate. All plates showed swarm spread at different levels giving inner and outer rings. Unfortunately, we observed no different movement/growth towards the L-aspartic acid compared with the control (water). It was assumed that this is due to the low concentration, but it wasn't possible to increase the molar percentage of solution past ~0.4%.
</P><P>
The swimming agar experiment was repeated. Since the OD<SUB>600</SUB> of the bacterial culture for the inoculation was too low The plates were inoculated the next day. More concentrated aspartate solution was made by reacting the acid of aspartate with its salt. A concentration up to 7 % was reached. We ordered the salt to save having to do this in future.
</P><P>This time, only 0.3 % agar plates were used. Two wells were made using the thick end of a 1000 µl pipette tip. Aspartate solution was pippeted into the left hole and water into the right (as a control) . Half of the plates had thr solutions added in the morning, in order to set up a chemotactic gradient to set up. The other half of the plates had their solutions added immediately before the inoculation in the afternoon (arond 5 hours later). The <I>E. coli</I> MG1655 strain was inoculated three times on a line in the centre between the two holes and incubated overnight at the different temperatures: 25°C, 30°C and 37°C. 
</P><P>The plates were photographed and measured. There was some significant increase in movement towards the aspartate (chemoattractant), but the difference should be more visible. Thus, various conditions should be tested during the next experiments.
</P><H3><SPAN class="mw-headline" id="16th_July_-_22nd_July">16th July - 22nd July</SPAN></H3><H4><SPAN class="mw-headline" id="BioBrick_Transformation">BioBrick Transformation</SPAN></H4><P><A href="/Team:BCCS-Bristol/Protocols-Media" title="Team:BCCS-Bristol/Protocols-Media">SOC medium</A> was made in preparation for Biobrick transformation.
</P><H4><SPAN class="mw-headline" id="Chambers_for_bacteria-moving-bead_assay">Chambers for bacteria-moving-bead assay</SPAN></H4><P>The first trial in making wells as in the diagram (See below) was made, but it was found that a different glue is needed. Nevertheless, in the first chambers a blue dye in 0.3% agar (to represent aspartate) and an orange dye in 0.1% agar to represent the bacteria was put (beads were added in later experiments). Pictures were taken at 30 minutes intervals. This was to see how long a chemotatic gradient would take to set up. However, after three hours no gradient was obvious, so the chambers were left overnight.
</P><P>The dye plates were observed: They had dried out somewhat overnight, however the dyes had mixed well. The experiment was carried out again with whatman paper soaked in water in 15 cm petri dishes to make a humid environment. This time swimming medium (see table below) was also used instead of the 0.1% agar used before as this may be more like what we use in the final experiment. 
</P><P><B>Swimming Medium</B></P><TABLE align="center" border="1" cellpadding="4" cellspacing="0" style="border:#c9c9c9 1px solid; margin: 1em 1em 1em 0; border-collapse: collapse;"><TBODY><TR><TD align="center" style="background:#f0f0f0;"><B>In 100ml of water</B></TD><TD align="center" style="background:#f0f0f0;"><B>weight (g)</B></TD></TR><TR><TD> Potassium Phosphate</TD><TD>0.1742
</TD></TR><TR><TD> NaCl</TD><TD>0.392
</TD></TR><TR><TD> Glucose</TD><TD>0.18016
</TD></TR><TR><TD> EDTA</TD><TD>0.003722
</TD></TR><TR><TD> Tween -20 aka polysorbate 20</TD><TD>0.2 µl
</TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="Comparision_of_motility_between_E._coli_MG1655_and_MC1000">Comparision of motility between <I>E. coli</I> MG1655 and MC1000</SPAN></H4><P>The motility of both strains was tested under different conditions:
</P><P>- agar concentration: 0.3 % and 0.5 %
</P><P>- incubation temperatures: 25<SUP>o</SUP>C and 30<SUP>o</SUP>C
</P><P>The OD<SUB>600</SUB> was adjusted to 1.25 and the bacteria were inoculated in the middle of an LB-plate without yeast extract, but with streptomycin. After an incubation overnight there was no growth of MG1655 visible, because only MC1000 is resistant. Therefore, the experiment should be repeated.
</P><P>The motility comparison experiment was repeated, but this time only the plates for MC1000 contained streptomycin. None of the 0.5% agar showed growth. Overall MG1655 swarms were possibly slightly larger than MC1000 (in Cambridge 06' MC1000 had much greater motility than MG1655). The growth/motility of MC1000 might be disturbed because, in contrast to MG1655, the bacteria had to deal with an antibiotic. Thus, the experiment will be repeated without any antibiotics in the agar.
</P><H3><SPAN class="mw-headline" id="23rd_July_-_29th_July">23rd July - 29th July</SPAN></H3><H4><SPAN class="mw-headline" id="BioBrick_Transformation_2">BioBrick Transformation</SPAN></H4><P>The cells transformed with BioBrick were grown on ampicillin plates as if transformed are able to do so, however there was no growth of bacteria except when pUC19 was used. This highlights some problems with the DNA and not the actual transformation. The experiment was repeated with larger amounts of TE buffer around spot and with a different BioBrick. Results see weekly summary.
</P><H4><SPAN class="mw-headline" id="Bead_experiment_1">Bead experiment 1</SPAN></H4><P>The bead we had focused on overnight (see <A href="/Team:BCCS-Bristol/Calendar-Notebook/22_July_2008" title="Team:BCCS-Bristol/Calendar-Notebook/22 July 2008"> 22 July 2008</A>) had gone!! There were observed lots of beads clumping together, areas of dead/non-motile bacteria and also filamentous bacteria (possible conaminant). There were also areas of still motile bacteria. 
The bacteria had travelled 40 % of way across bridge, thus a smaller bridge was suggested.
</P><H4><SPAN class="mw-headline" id="Swimming_agar_assay_2">Swimming agar assay</SPAN></H4><P>The experiment from <A href="/Team:BCCS-Bristol/Calendar-Notebook/21_July_2008" title="Team:BCCS-Bristol/Calendar-Notebook/21 July 2008"> 21 July 2008</A> was repeated. The aspartate solution was added eigth hours before the inoculation. The Results for showed that both strains were actually repelled by aspartate, possibly because the concentration was too high. Aspartate is the well on the right, and the control (water) is on the left.
</P><TABLE align="center" width="80%"><TBODY><TR><TH align="center" width="10%">MC1000 25<SUP>o</SUP>C
</TH><TH align="center" width="10%">MC1000 30<SUP>o</SUP>C
</TH><TH align="center" width="10%">MG1655 25<SUP>o</SUP>C
</TH><TH align="center" width="10%">MG1655 30<SUP>o</SUP>C
</TH></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="Results_from_different_aspartate_concentrations">Results from different aspartate concentrations</SPAN></H4><P>As you can see there wasn't much difference between control and aspartate, possibly due to the chemtactic gradient not being set up. motility comparison gave much better results
</P><H3><SPAN class="mw-headline" id="30th_July_-_6th_August">30th July - 6th August</SPAN></H3><H4><SPAN class="mw-headline" id="BioBrick_Transformation_3">BioBrick Transformation</SPAN></H4><P>Last Thursday, the primers <A href="http://partsregistry.org/wiki/index.php/Part:BBa_G00100" class="external text" rel="nofollow">VF2</A> and <A href="http://partsregistry.org/Part:BBa_G00101" class="external text" rel="nofollow">VR</A> arrived. These primers allow a confirmation of a succesful transformation of most BioBrick parts by means of PCR. The resulting fragment length depends on the specific BioBrick DNA. 
</P><P>Using VF2 and VR, a colony PCR are was conducted with all colonies that were obtained with chemical competent cells so far. Two colonies with the <A href="http://partsregistry.org/Part:BBa_J63005" class="external text" rel="nofollow">yeast ADH1 promoter</A> (No. 1 and 2, see left photo) and three colonies with a <A href="http://partsregistry.org/Part:BBa_E0240" class="external text" rel="nofollow">GFP generator</A> (No. 3-5, see left photo) were identified as positive. Thus, all colonies are positive, but the transformation efficiency is too low, since the result was only one or zero colonies per transformation attempt. 
</P><P>Electroporation should give higher efficiencies. An attempt with a new BioBrick (<A href="http://partsregistry.org/Part:BBa_J63002" class="external text" rel="nofollow">ADH1 terminator</A>) resulted in 69 colonies! Ten of them were analysed using PCR. The resulting fragment should have a length of 461 bp, but from the eigth positive colonies fragments between 650-750 bp were obtained (see right photo). This is probably due to a mutation which might result in a different binding location for one of the primers. Another explanation might be that the denoted length is wrong. Since this BioBrick is not important for our project, it will be disregarded. 
</P><TABLE align="center" border="1"><TBODY><TR><TD><P>L= HyperLadder I (BIOLINE)
</P><P>N= Negative control
</P><P>Numbers= See text 
</P></TD></TR></TBODY></TABLE><P>The transformation efficiency of the electroportation was significant higher than of the chemical competent cells. Therefore, an own stock of electric competent cells was made.
</P><H4><SPAN class="mw-headline" id="Agar_plug_assay">Agar plug assay</SPAN></H4><P>TR235 (E. coli with construct to detect adhesion to hydrophobic surfaces) arrived from Princeton (Thomas J. Silhavy). Cells are viable, growth curve done. 11µm PS beads shown to be moved by bacteria (MG1655); we observed ‘jiggling’ of beads when bacteria present (not seen in no-bacteria control). Also, we observed significant movement in different directions (distances of approx 1x to 5x length of the beads by eye) of beads that were close together, suggesting that movement was not due to currents in the medium. We saw no distance covered in almost all the control beads, a few moved small distances, so some currents are present. Bacteria and beads were left together for ~30mins to allow time for adhesion. No movement has yet been seen when the bacteria and beads are not left to adhere. We will look again at this, as we may have not previously looked closely enough (photographing over a time course of a few minutes). 
</P><TABLE align="center"><TBODY><TR><TH align="center" width="10%">0 mins
</TH><TH align="center" width="10%">1 min
</TH><TH align="center" width="10%">2 mins
</TH></TR></TBODY></TABLE><TABLE align="center"><TBODY><TR><TH align="center" width="80%"><B>Circles represent position at 0 mins, one colour for each bead. For example, the bead that starts in red circle    moves down and right, while that in the yellow circle moves left.</B></TH></TR></TBODY></TABLE><P>Chemotaxis assay (Yu, H.S. et al. (1997) FEMS Microbiology Letters, 156, 265-269.) has not worked so far. Currents can be seen flowing from the aspartate plug, pushing bacteria away. The focus of this week will be on getting this fixed, with a view to combining chemotaxis and the ability to move beads, allowing us to move beads in a particular direction. 
</P><P>
The GRN for switching on the chemotactic response by binding to a bead, and signaling to nearby bacteria that a particle is in the vicinity, is almost finalised.
</P><H3><SPAN class="mw-headline" id="7th_August_-_13th_August">7th August - 13th August</SPAN></H3><H4><SPAN class="mw-headline" id="BioBrick_Transformation_4">BioBrick Transformation</SPAN></H4><P>The self-made electro competent <I>E. coli</I> DH5α cells were tested twice. The transformation efficiency with pUC19 was 4.45 x 10<SUP>8</SUP> cfu/µg and 5.67 x 10<SUP>8</SUP> cfu/µg. The first attempt with BioBrick DNA failed, but this might be due to a too low cell amount compared to all the other decanted tubes. In the following transformations, the incubation of the punched paper disc with the DNA was varied in time (3 h and 5.5 h). It seems that a longer incubation increases the transformation efficiency. This has to be confirmed, since the transformations were conducted by different persons. Thereby, next to the <A href="http://partsregistry.org/Part:BBa_J63005" class="external text" rel="nofollow">yeast ADH1 promoter</A> another BioBrick (<A href="http://partsregistry.org/Part:BBa_J63001" class="external text" rel="nofollow">enhanced version of EYFP, yeast-optimized YFP</A>) was tested because the VF2-VR value of <A href="http://partsregistry.org/Part:BBa_J63005" class="external text" rel="nofollow">yeast ADH1 promoter</A> we got didn't coincide with the value given on the iGEM page.
</P><H4><SPAN class="mw-headline" id="Agar_Plug_Assay_2">Agar Plug Assay</SPAN></H4><P>Evaporation and currents that were occurring in previous attempts at the agarose-in-plug assay were eliminated by ensuring water potentials of plug and chemotaxis buffer were as similar as possible.
Both the beads and the bacteria were found to sink to the bottom of the chamber shortly after inoculation.  This makes the model much simpler, as everything can be modelled in 2D. 
We showed that beads need to be left in bacteria for around 30mins in order for significant bead movement to occur.  We currently have not determined if this is because the bacteria form a layer under the beads which stops them sticking to the slide, or if adherence itself is needed for bead movement.
We determined the diffusion speed of bromophenol blue dye (which is three times larger than aspartate), and concluded that by 30mins aspartate would have diffused throughout the slide.
</P><H3><SPAN class="mw-headline" id="14th_August_-_20th_August">14th August - 20th August</SPAN></H3><H4><SPAN class="mw-headline" id="Agar_Plug_Assay_3">Agar Plug Assay</SPAN></H4><P>We repeated the chemotaxis assays and found that the bacterial density surrounding the plug varied and therefore we obtained more valid results by using lower magnification and scanning around the whole plug (as opposed to focussing on one area for the whole time course).  This gave a much better overview of changes in bacterial density.
</P><P>
We tried testing bacterial chemotaxis to 1% serine and 1% aspartate in the plug.  Bacterial density around the plug was monitored for 1hr.  Bacteria appeared to be repelled by both the serine (see below)and aspartate at this concentration compared with the control where no chemoattractant was present in the plug.  Research group page (Professor Jong Yoon, Micro/nanofluidic Bio MEMS group) indicates that serine becomes chemorepellent 1M concentration (the same may be true of aspartate).
</P><TABLE align="center"><TBODY><TR><TH align="center" width="10%">Control
</TH><TH align="center" width="12%">1%Serine in plug
</TH></TR></TBODY></TABLE><P>We have been attempting a 'dead bacteria' control, but have been unable to successfully kill our bacteria without using a method that physically alters them.  We have tried sodium azide (bacteriostatic)- but resuspension allows them to recover.  Kanamycin and chloramphenicol were also used, but only for a few hours.  We might try leaving a culture overnightin these two antibiotics - this should be sufficient to kill them!
</P><H4><SPAN class="mw-headline" id="BioBrick_Transformation_5">BioBrick Transformation</SPAN></H4><P>The transformation efficiency should be increased. It was assumed that the quality of the BioBrick DNA is the most important factor to improve. Thus, two punched paper discs of the same BioBrick were incubated overnight at two different temperatures: 37<SUP>o</SUP>C and room temperature. The room temparature DNA resulted in 63 colonies (compared to 14 colonies with the 37<SUP>o</SUP>C-DNA), but it was not comparable since the 63 colonies were a result from a double electro shock. This may enhance the transformation efficiency. Thus, further tests should be conducted considering different incubation times and temperatures of the paper disc in water and as well single and double electroporation. 
</P><P>Electro competent cell of <I>E. coli</I> MG1655 were made and tested with pUC19. This resulted in zero colonies and may due to the high OD<SUB>600</SUB> of the cells when they were prepared. New cells should be made. 
</P><P>DH5α cells were transformed with pRZ1 (pRS415 containing cpxR responsive promotor). The efficiency was very high and 15 colonies were isolated. 
</P><H3><SPAN class="mw-headline" id="21st_August_-_1st_September">21st August - 1st September</SPAN></H3><H4><SPAN class="mw-headline" id="Change_to_a_proof-of-concept">Change to a proof-of-concept</SPAN></H4><P>Unfortunately it was discovered that the Cpx response (activated by adhesion to hydrophobic surfaces) actually downregulates, via CpxR, chemotaxis through a number of genes. This means that our initial plan of using the Cpx system to detect adhesion to the beads will not work in its current form as the bacteria adhered to the bead will stop chemotaxing. Indeed, bacteria that adhere to the bead because of short or long-range co-ordination will also stop chemotaxing soon after they reach and adhere to the bead. There would be two ways around this problem: either we stop CpxR switching off chemotaxis by knocking-out the two binding regions that allow CpxR to downregulate the two operons downregulating the chemotactic genes, or we place these genes (eight of them) under the control of an activating CpxR region. In the time remaining, neither of these are achievable. Furthermore, it is likely that there are additional unknown genes required for chemotaxis, that are also downreguated by CpxR upon adhesion, therefore there is a good chance that neither strategy would work initially. As our experimental work has not yet confirmed whether bacteria adhered to beads, and able to chemotax, can actually direct bead movement towards a goal chemoattractant, at the least we would require the modelling work to suggest that this was indeed possible before any effort was made to maintain chemotaxis upon adhesion by either of the above strategies. 
</P><P>
In light of this, the focus of the lab work will move towards producing proof-of-concept BioBricks that will allow us to demonstrate the communication of an adhesion event from an adhered to an unadhered bacteria. This can be achieved  making extensive use of the available BioBricks. Specifically, sender cells (cells that have adhered to beads and are trying to communicate with unadhered cells), will contain the LuxI-GFP portion of BBa_J37015, fused downstream of a CpxR responsive promoter (modified from current version fused to LacZ provided courtesy of T.Silhavy, Princeton). Upon adhesion to beads, LuxI will produce AHL, which will diffuse into the medium, while GFP will give confirmation that the transcript is working. For receiver cells we will test both BBa_T9002 and BBa_I731014, which produce GFP and mCherry respectively upon activation by AHL. We will also BioBrick the CpxR responsive promoter itself, and aim to fuse BBa_E0840 (GFP) to the CxpR responsive promoter, creating a nice visual assay for adhesion (the CpxR-LuxI-GFP BioBrick will also do this, but producing AHL in addition may not be ideal for certain future applications). Therefore in total we aim to produce 3 BioBricks in total.
</P><H3><SPAN class="mw-headline" id="2nd_September_-_9th_September">2nd September - 9th September</SPAN></H3><H4><SPAN class="mw-headline" id="Biobrick_transformations">Biobrick transformations</SPAN></H4><P>A series of biobrick transformations (J37015, T9002, E0840 and I731014) were performed using electro-competent DH5α cells.  After several attempts, only J37015 and T9002 appeared to work, but after carrying out colony PCR on the transformed bacteria we found that the J37015 part doesn’t appear to be in its plasmid (pSB1A3).  The T9002 transformation, however, was successful, and this was verified by plasmid mini-prep and subsequent test digestion.  Its activity as a luxR GFP receiver construct was also confirmed, by testing the fluorescence of the transformed bacteria when incubated with two different concentrations of AHL.  (No GFP was produced in the control, un-transformed, bacteria).   Since we have tried several times, unsuccessfully, to transform the other biobricks, strains already transformed with these biobricks were requested from iGEM HQ.
</P><TABLE align="center"><TBODY><TR><TH align="center" width="11%">DH5α T9002, 10E-6M AHL, Bright Field
</TH><TH align="center" width="10%">DH5α T9002, 10E-6M AHL, GFP
</TH></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="10th_September_-_16th_September">10th September - 16th September</SPAN></H3><P>We received strains from iGEM HQ transformed with the following biobricks: J37015, pSB1A3, I731014, T9002, E0840.  
</P><P>The AHL test used to verify the activity of our T9002-transformed DH5α cells was repeated (see protocols page) to test the I731014 and T9002 strains provided by iGEM.  We used pSB1A3-transformed strains as our control.  Both test strains displayed fluorescence at both 10<SUP>-6</SUP>M and 10<SUP>-8</SUP>M AHL, whilst no fluorescence was detected in the negative control.  The proportion of total I731014-transformed bacteria that actually fluoresced was quite low however.  (T9002 fluorescence, in contrast, was near to 100% of total bacteria in the sample that was viewed).
</P><P>Plasmid mini-preps and restriction digests were performed to test the identity of the plasmids.  Single and double digests were carried out for each biobrick.  The results for T9002 and E0840 were as expected and corresponded to the part sequence data.  
Both the single and double digests for J37015, however, produced only one band – all running just above the 2000bp marker.  SpeI was used in both the single and double digest and according to the sequence data cuts once in the plasmid containing the J37015 part (pSB1A2).  HindIII was the second restriction enzyme used in the double digests and should cut once in the J37015 part.  We concluded from this information that the J37015 part is not in the plasmid – and the resulting single band corresponds to the linearised pSB1A2 plasmid!
The digests for pSB1A3 and I731014 were rather confusing.  We don’t know what went wrong here, especially as we had shown previously in the AHL test that the I731014 part appears to be functioning.
</P><P>This week we also started on the road to making our first biobricks.  The CpxR promoter was cloned out of the TR235 and MG1655 genomes, and was verified on a gel after restriction digest.  The results of the digest of pSB1A3, however, did not make sense again, and doesn’t appear to correspond to what it is meant to be.  We have now decided to obtain this plasmid by using the T9002 biobrick, cutting out the T9002 part, and inserting our CpxR promoter.  
</P><P>The second biobrick we want to make is luxI and GFP under the control of our CpxR promoter.  Originally the LuxI and GFP portion were to be obtained from the J37015 biobrick, but since this doesn’t appear to be in the plasmid, we have started on building it ourselves using the separate LuxI (I15030) and GFP (E0840) parts.  Transformations of electro-competent DH5α with the I15030 part were carried out.
</P><H3><SPAN class="mw-headline" id="17th_September_-_25th_September">17th September - 25th September</SPAN></H3><P>We had problems using the pSB1A3 BioBrick so instead digested T9002 to remove the insert, leaving the pSB1A3 vector for Biobrick Building.
</P><H4><SPAN class="mw-headline" id="Building_pCpxR_pSB1A3">Building pCpxR_pSB1A3</SPAN></H4><P>We digested pCpxR (from both MG1655 and TR235 - see previous entry) to give it sticky ends and ligated it into pSB1A3. There were transformed into DN5 alpha. 4 out of 7 colonies were in the correct orientation. 3 out of the 4 colonies were of the right length when these were analysed by restriction digest. Out of the 3 colonies, 2 were MG1655 pCpxR and 1 was TR235 pCpxR. 
</P><H4><SPAN class="mw-headline" id="Building_pCpxR_E0840_pSB1A3">Building pCpxR_E0840_pSB1A3</SPAN></H4><P>BioBrick BBa_E0840 was cloned and digested to make it sticky. This was ligated into the pCPXR_pSB1A3 (both MG1655 and TR235 pCpxR) vector and transformed into DH5 alpha.
</P><P>Transformation of DH5 alpha with I15030 (in an attempt to aquire LuxI-GFP this way) showed that the part doesn't appear to be in the BBa_I731014 Biobrick. There are issues with the BBa_I731014 part documented the parts registry. We have contacted the Imperial IGEM group who are sending us the part.
</P><H3><SPAN class="mw-headline" id="26th_September_-_16th_October">26th September - 16th October</SPAN></H3><P>The part from Imperial arrived too late, so could not be used. We therefore concentrated on the pCpxR_pSB1A3 and the pCpxR_E0840_pSB1A3 parts. 
There were three different pCpxR_E0840_pSB1A3 constructs to test, two using the pCpxR from MG1655 and one using pCpxR from TR235. Of these three, DH5alpha transformed with the TR235 pCpxR_E0840_pSB1A3 construct and one of the MG1655 pCpxR_E0840_pSB1A3 constructs fluoresced after being placed on glass microscope slides (a longer form of pCpxR had previously been shown to be activated by adhesion to glass microspheres). Fluorescence in the TR235 pCpxR_E0840_pSB1A3 transformation was mainly restricted to long filamentous bacteria, while smaller bacteria were seen to fluoresce in the MG1655_pCpxR_E0840_pSB1A3 transformation. Restriction digests confirmed that TR235 pCpxR_E0840_pSB1A3 had been formed correctly with E0840 inserting in the correct orientation. There was some confusion however as the MG1655 pCpxR_E0840_pSB1A3 construct that fluoresced was suggested by the restriction digest to have an E0840 insertion in the incorrect orientation, while the MG1655 pCpxR_E0840_pSB1A3 that did not fluoresce was suggested to be in the correct orientation. All three samples, plus the two pCpxR_pSB1A3 constructs (one from TR235 and one from MG1655) were sent off for sequencing to confirm their identity.
</P><P>Sequencing confirmed the identity of the TR235 pCpxR_E0840_pSB1A3 confirmed its correct orientation, and showed an exact match to the target sequence. The restriction digest results were also confirmed, with the MG1655 pCpxR_E0840_pSB1A3 that fluoresced having the E0840 in the incorrect orientation, while the MG1655 pCpxR_E0840_pSB1A3 that didn't fluoresce having E0840 in the correct orientation, again with an exact match to the target sequence. It appears that E0840 in the incorrect orientation had been activated by another route, although this has not been looked into. It is also unclear as to why the MG1655 pCpxR_E0840_pSB1A3 with E0840 in the correct orientation did not fluoresce (as it was also an exact match for the target sequence), although more repetitions of the experiments would be required to ascertain the efficiency of the different constructs, as the results here could be an anomaly (there was not enough time to perform these experiments here). Of the two pCpxR_pSB1A3 constructs, the TR235 pCpxR_pSB1A3 was an exact match for the target, while the MG1655 pCpxR_pSB1A3 had one mismatch within the promoter region, although not in the CpxR consensus binding sequence. Note: as the MG1655 pCpxR_E0840_pSB1A3 construct was shown to be an exact match for the target sequence, the mutation in MG1655 pCpxR_pSB1A3 must have occured after the construction of MG1655 pCpxR_E0840_pSB1A3.
</P><P>Two fully functioning BioBricks can therefore be submitted; the TR235 pCpxR_pSB1A3 construct (<A href="http://partsregistry.org/Part:BBa_K135000" class="external text" rel="nofollow">BBa_K13500</A>), and the TR235 pCpxR_E0840_pSB1A3 construct (<A href="http://partsregistry.org/Part:BBa_K135001" class="external text" rel="nofollow">BBa_K135001</A>). 
</P><P>Project work has now ended. See you in Boston! :)
</P></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2008.igem.org/Team:BCCS-Bristol/Notebook">http://2008.igem.org/Team:BCCS-Bristol/Notebook</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:BCCS-Bristol/Notebook" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:BCCS-Bristol/Notebook" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:BCCS-Bristol/Notebook&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:BCCS-Bristol/Notebook&amp;oldid=82857" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2008.igem.org:Privacy_policy" title="2008.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2008.igem.org:General_disclaimer" title="2008.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>