<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Penn_MethylaseCharacterization"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Penn/MethylaseCharacterization">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Penn/MethylaseCharacterization&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Penn/MethylaseCharacterization&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Penn/MethylaseCharacterization&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Penn/MethylaseCharacterization" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Penn/MethylaseCharacterization</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><P><TITLE>Penn iGEM</TITLE></P></DIV><DIV id="text"><DIV class="textwrap"><HEADER><H1><CENTER><B>Methylase Characterization</B></CENTER></H1></HEADER>
For a detailed, graphical explanation of the MaGellin work flow, please download the <A href="https://static.igem.org/mediawiki/2013/e/e5/Spec_Sheet.pdf">MaGellin Workflow Specifications Sheet</A>, which includes all of the steps in the MaGellin workflow.

We developed the MaGellin assay to optimize the development process for site-specific methylases. Having validated the assay, we determined to design and test three site-specific methylases, two of which had never been constructed before.<H7>The process further validated our MaGellin assay:1. We recapitulated published results with a zinc finger-methylase and shed light on the significant magnitude of its off target effects. MaGellin is an excellent assay for this purpose, because of the noiseless chassis and because it's simpler to detect off target effects on a plasmid than a genome.2. We further characterized our promising novel TALE-methylase and were able to de-noise this noisy, complex system. MaGellin was in agreement with COBRA that the TALE exhibited on and off target methylation. This could have implications for the multitude of TALE-effector systems that have recently been developed. The MaGellin workflow is well suited to solve this optimization problem.3. We demonstrated the ease-of-use of the MaGellin workflow by assaying 20 conditions in less than one week.</H7><H1>Zinc Finger-M.SssI Fusion</H1>
The zinc finger is a small DNA binding domain, with limited sequence specificity. Previous studies showed it was prone to off-target methylation, which we verified (Xu et al., 1997).<H7> MaGellin agreed with previously published work</H7>, from various groups with no common, standardized assay. It's possible MaGellin is more sensitive to off target methylation than their assays because <H7>MaGellin only reports site-specific methylation if 100% of the methylations is site-specific for that plasmid.</H7><H4><CENTER>ZF-M.SssI fully methylates MaGellin plasmid</CENTER></H4><DIV style="margin-left:auto;margin-right:auto;text-align:center"><FIGURE><FIGCAPTION><I>Figure 1: A ZF-M.SssI was cloned into and expressed from MaGellin, then induced with IPTG in T7 Express cells. The NEB10 (N) control has no T7 polymerase and no possibility of leaky expression. The linearized control (L)  is the same band length as blanket methylation.</I></FIGCAPTION></FIGURE></DIV>

To be sure of the targeting specificity, we cloned the MaGellin plasmid with and without the zinc finger’s binding site present at the target cut site. MaGellin reported off target activity, irrespective of zinc finger binding (Figure 1). <H7>MaGellin is sensitive to off target effects</H7> because there is no background CpG methylation in E.coli and the plasmid is short compared to a mammalian genome. We are interested in examining this initial finding further to determine its reproducibility and relevance to previously published zinc finger-methylases.
<H1>TALE-M.SssI Fusion</H1>TALEs have a greater sequence specificity than zinc fingers, and are easier to customize and less expensive to construct (Cong et al., 2012). They have already been validated for use in genome engineering and are replacing zinc fingers for some applications. We followed the MaGellin protocol to clone a TALE-M.SssI fusion and induced its expression. We repeated this experiment numerous times with varying induction conditions and found the TALE-M.SssI was methylating at both sites, as reported by the MaGellin software.
<H4><CENTER>TALE-M.SssI partially methylates on and off target sites on MaGellin plasmid</CENTER></H4><DIV style="margin-left:auto;margin-right:auto;text-align:center"><FIGURE><FIGCAPTION><I>Figure 2: A TALE-M.SssI was cloned into and expressed from MaGellin, then induced with IPTG in T7 Express cells. The NEB10 control has no T7 polymerase and no possibility of leaky expression. The linearized control is the same band length as blanket methylation.</I></FIGCAPTION></FIGURE></DIV>

MaGellin reported our TALE-M.SssI was detectably methylating DNA at both the target site and off-target site. The off target methylation was significantly reduced compared to what MaGellin reported for the zinc finger fusion. <H7>This showed MaGellin can detect changes in the site-specificity of methylation due to swapping DNA binding domains</H7>. We still expected a certain degree of off target methylation simply because the TALEs could occupy all the binding sites on our low copy plasmid; the molar ratio is one of the problems in developing site-specific methylases that the inducible MaGellin system is designed to address. MaGellin is designed to screen multiple fusion protein constructs in a high-throughput manner, and a user would normally select only constructs that methylate in a highly site-specific manner. However, we were interested in using MaGellin to study the TALE-M.SssI further before going back to the drawing board to redesign the linker length, binding site, and other variables.
<CENTER><H4>Validated COBRA is in agreement with our new MaGellin Assay</H4>
Given the novel nature of our MaGellin assay, we wanted to see if a traditional, published methylation assay would be in agreement about the TALE-M.SssI result. <DIV style="margin-left:auto;margin-right:auto;text-align:center"><FIGURE><FIGCAPTION><I>Figure 3: COBRA on induced TALE-M.SssI. The plasmid was bisulfite treated and the  target and off target sites were amplified with our standard bisulfite sequencing primers. The amplicons were digested with TaqαI, which only cuts methylated sites, and BamHI, which only cuts untreated DNA. In COBRA, as opposed to MaGellin, digestion means methylation, and no digestion means no methylation.</I></FIGCAPTION></FIGURE></DIV>

We bisulfite converted the plasmid and used our <A href="https://2013.igem.org/Team:Penn/AssayValidation">validated bisulfite sequencing primers</A> on both the target and off target site, then used the <A href="https://2013.igem.org/Team:Penn/AssayOverview">COBRA assay</A>. The controls recapitulated that our primers are biased for only bisulfite converted DNA, as desired. Unconverted DNA would have been digested by the control enzyme, otherwise. Methylation-sensitive TaqαI digested both the on and off target sites, confirming that the TALE was partially methylating both sites, as MaGellin reported (Figure 3). <H7>This validated our assay further, as MaGellin reports the same biological outcome as the published COBRA method, but at a fraction of the cost, time, and technical difficulty (Xiong et al., 1997).</H7><CENTER><H4>Varied Induction Conditions Suggests TALE-M.SssI is Prone to Off Target Activity</H4>
Given the quick turnaround and cost-effectiveness of the MaGellin assay, <H7>it was feasible to test our TALE-M.SssI construct at 20 different conditions to get a better idea of its fuction <I>in vivo</I></H7>. We hoped to find the optimal induction point for reducing off target methylation. This study would have cost us approximately $7,000 to do by bisulfite sequencing, based on the prices at our university core facility. MaGellin only required restriction enzymes and gel electrophoresis (less than 5% the cost of bisulfite sequencing).
<DIV style="margin-left:auto;margin-right:auto;text-align:center"><FIGURE><FIGCAPTION><I>Figure 4: The TALE-M.SssI with and without the TALE binding site present was induced with 0, .1, 1, and 2 mM of IPTG for 0, 2, 6, and 24 hours to find optimal expression conditions. Representative bands’ intensities were quantified and normalized by background intensity. The plot is missing data for samples marked *. The dotted white circle marks the conditions of our initial experiments.</I></FIGCAPTION></FIGURE></DIV>

We varied induction conditions, expecting one might be more optimal than our initial inductions. As expected, the &quot;without binding site&quot; negative control showed significant off target effects, presumably because the TALE did not bind and the methylase was still active. However, as we induced for longer times and with more IPTG than we had originally used, MaGellin reported off target methylation increasing even for the TALE with the binding site. 
This makes sense, as the TALEs could saturate the binding sites on a low copy plasmid or genome. We expect this has implications for recently published TALE-effector systems, which may be noisier than expected as it is much more difficult to assay off-target effects in mammalian systems. We propose the use of split-reconstitution systems that are activated by co-localization of effector subunits at a target site. Our future direction is to design this sort of chimeric TALE, cloning and screening the construct will be quick and easy with MaGellin. <H7>Notably, we would not have noticed the off target methylation so quickly on a mammalian genome, but it was simple to de-noise this complex system with MaGellin.</H7><B>Up Next:</B> We have designed and cloned a dCas9-M.SssI fusion, which we will characterize with MaGellin.
<H1>Summary</H1><H7>MaGellin was designed to optimize the development of robust tools for site-specific methylation.</H7> To those ends, we successfully cloned and expressed three fusion methylases, two of which are novel constructs with advantages over the previously published zinc finger. Our constructs have shown methylase activity and DNA binding activity, which we could measure with our new assay. They are ready to be further optimized, using our MaGellin workflow.
<P>MaGellin agreed with previously published work that zinc finger methylases are prone to off target methylation. <H7>We were able to construct and test this construct in the three weeks after regionals because of MaGellin's simple workflow.</H7></P><P><H7>We picked up on the noisiness of our TALE-M.SssI using MaGellin,</H7> which could have implications for the noisiness of other TAL-Effector systems being used in mammalian systems. To do so, we used MaGellin to its full extent: swapping out DNA binding domains and binding sites, varying induction conditions, applying COBRA, and depending on our original algorithm to properly predict methylation-sensitive digestion patterns. <H7>Importantly, we could not have reached this result without MaGellin,</H7> because the one-plasmid system in a noiseless chassis makes it simple, even unavoidable, to detect off target methylation. Conversely, for the previously published work in mammalian systems, it was not as feasible to detect off target effects across a long genome with background signal.<H7> Based on our data, future improvements on genome engineering tools should include the construction of two targeted fusions with subunits of effectors that only dimerize and show activity at the binding sites,</H7> along the lines of how TALE-Nucleases cleave DNA (Li et al, 2007). 
<CENTER><A href="https://2013.igem.org/Team:Penn/MethylaseOverview">←Previous</A><A href="https://2013.igem.org/Team:Penn/MaGellinFutureDirections">Next→</A></CENTER></P></CENTER></CENTER></DIV></DIV></DIV></DIV><DIV id="pagefooter"><CENTER><A href="https://2013.igem.org/Team:Penn"> Home  </A><A href="https://static.igem.org/mediawiki/2013/e/e5/Spec_Sheet.pdf">Spec Sheet</A><A href="https://2013.igem.org/Team:Penn/sitemap">Sitemap</A></CENTER>
Penn iGem © 2013
</DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:Penn/MethylaseCharacterization">http://2013.igem.org/Team:Penn/MethylaseCharacterization</A>&quot;</DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Penn/MethylaseCharacterization" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Penn/MethylaseCharacterization" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Penn/MethylaseCharacterization&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Penn/MethylaseCharacterization&amp;oldid=360418" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>