<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Penn_MaGellinFutureDirections"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Penn/MaGellinFutureDirections">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Penn/MaGellinFutureDirections&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Penn/MaGellinFutureDirections&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Penn/MaGellinFutureDirections&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Penn/MaGellinFutureDirections" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Penn/MaGellinFutureDirections</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><P><TITLE>Penn iGEM</TITLE></P></DIV><DIV id="text"><DIV class="textwrap"><CENTER><B><H1>
Future Directions
</H1>
For a detailed, graphical explanation of the MaGellin work flow, please download the <A href="https://static.igem.org/mediawiki/2013/e/e5/Spec_Sheet.pdf">MaGellin Workflow Specifications Sheet</A>, which includes all of the steps in the MaGellin workflow.
<H4><B>Advantages of a TALE-methylase</B></H4><P>Our new TALE fusion is much more modular and easy to customize than the old zinc finger (Table 1). TALE construction is not heavily patented like zinc-finger design, and TALE’s are considerably cheaper to produce (Sanjana 2012 and Zhang 2011). They have made the zinc-fingers nearly obsolete as a tool for genetic engineering, including the creation of transgenic animal models (Tesson 2011, Sander 2011, Huang 2011, and Zhang 2011). We expect our new fusion protein could be translated to enable the creation of differentially methylated animal models, which would be revolutionary for epigenetic disease research (Klose 2006). In effect, this could transform epigenetics from a largely observational discipline to one of active intervention and manipulation, similar to the transition from early classical genetics to genetic engineering and synthetic biology.
</P><H4><B>CRISPR System</B></H4><P>We also fused a methyltransferase to a dCas9 DNA binding domain from the CRISPR-Cas system. We modified our MaGellin vector to include a cassette that expresses the necessary guiding sgRNA1. The sgRNA can easily be swapped out to quickly re-target the Cas system, much more quickly and with less synthesis than it would take to re-target the TALE. We are intrigued by the prospect of optimizing the Cas system, which is considered to possibly have many advantages over the TALE (Gaj 2013). It has already been fused to transcription activators that similarly target promoters, so the strategy is theoretically sound (Bikard 2013). Particularly, it’s ability to multiplex and target multiple sites for methylation would be very useful (Wang 2013). However, many recent papers have described problems with Cas specificity that seriously limit its usefulness, so we expect the excitement around Cas may dim as scientists begin a rigorous re-evaluation of the comparison between Cas and TALE systems. We propose MaGellin as a simple, fast, and inexpensive assay to aid the development of superior Cas systems, as they are optimized by site-directed mutagenesis, directed evolution, or other methods. As outlined above and demonstrated with our TALE, fusing a methylase to the Cas serves as a detectable marker for its DNA binding.</P><H4><B>Split-Reconstitution Site-Specific Methylases</B></H4>Most previously published zinc finger-methylase studies have used the same two components as our TALE, the DNA binding domain linked to the methylase. In the noiseless MaGellin system, we were able to detect significant off target methylation at long induction times and high IPTG concentrations. This suggests these fusions still methylate in a non-specific manner once the binding sites are saturated. This issue must be harder to detect in mammalian systems. We are proposing the best strategy would be to split the methylase into subunits that only dimerize and activate at the target site when co-localized by their linked TALEs. This has been effective for TALE nucleases.
We are going to try this strategy by using the other multiple cloning site on the MaGellin plasmid, which we have so far used to express an sgRNA for the dCas9-M.SssI.

<H4><B>Translational Potential</B></H4><P>It is worth noting that the TALE-methylase can be delivered to mammalian cells by adeno-associated vectors (AAV). This viral vector is the best currently available in terms of safety and efficiency (Daya 2008). Different serotypes have different cell tropisms, which can provide efficient cell-type targeting when used in conjunction with a cell-specific promoter (Ellis 2013). As we consider different model organisms, we may also want to try different methyltransferases. Importantly, the TALE, at 2.5kb long, can easily be packaged with a methyltransferase in an AAV vector, whereas a Cas would not fit (Konermann 2013).

</P><H4><B>Epigenome Engineering is a Reality Today</B></H4><P>We were not the only synthetic biologists developing epigenetic engineering tools this summer. In August, we were excited to see George Church’s lab at Harvard had fused TALEs to histone modifiers, enabling another distinct form of epigenetic engineering (Konermann 2013). They delivered these TALE fusions to mammalian cells with AAV, which gives us confidence our system can be translated into more complex organisms. With their targeted histone modification and our targeted DNA methylation, it appears the era of serious epigenetic engineering efforts is now upon us. 

</P><H4><B>Transcriptional Silencing in Bacteria with Methylation</B></H4><P>At the regional jamboree, we were asked how precisely we could use CpG methylation to silence transcription in E.coli, which have no such endogenous mechanism. In response, we have designed an experiment using MeCP2, a DNA binding protein that only binds when its recognition site is methylated. We hope to use this protein to hinder the polymerase. So far, we have cloned and purified the protein, including a variant which was truncated to remove the DNA binding domain, to serve as our negative control.
</P><DIV style="margin-left:auto;margin-right:auto;text-align:center"><FIGURE><FIGCAPTION><I>A SDS-Page gel shows expression of the MECP2 gene.</I></FIGCAPTION></FIGURE></DIV><DIV align="left"><H4><B>Alternative to <I>ChIP.</I></B></H4><OL><LI>MaGellin can also be used to screen the binding specificity of transcription factors (TF) and other DNA binding domains, with methylation serving as an effective reporter for targeted binding. </LI><LI>By fusing a TF to the methyltransferase, its binding with the plasmid becomes easily detected by the associated methylation</LI><LI>In this situation, one would vary the “target site” to see which DNA sequences have the strongest binding to the TF</LI><LI>This technique would be significantly faster, simpler, and cheaper than a ChIP-based method</LI></OL><CENTER><A href="https://2013.igem.org/Team:Penn/MethylaseCharacterization">←Previous</A></CENTER></DIV></B></CENTER></DIV></DIV><DIV id="pagefooter"><CENTER><A href="https://2013.igem.org/Team:Penn"> Home  </A><A href="https://static.igem.org/mediawiki/2013/e/e5/Spec_Sheet.pdf">Spec Sheet</A><A href="https://2013.igem.org/Team:Penn/sitemap">Sitemap</A></CENTER>
Penn iGem © 2013
</DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:Penn/MaGellinFutureDirections">http://2013.igem.org/Team:Penn/MaGellinFutureDirections</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Penn/MaGellinFutureDirections" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Penn/MaGellinFutureDirections" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Penn/MaGellinFutureDirections&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Penn/MaGellinFutureDirections&amp;oldid=359806" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>