"
Page
Discussion
View source
History
teams
Log in
 
Team:Penn State/Notebook
From 2013.igem.org
Cas9 Project
Promoter Project
CesA Project
Butanol Project
Vanillin Project
Cas9 Project
June 23-29
PCR amplification of NOS 1 and NOS 2 terminators from pmdc107. PCR amplification of GFP, NOS 3, CMV35s, CMV35s(2), and mCherry. All PCR products were gel extracted and then PCR amplified again.
June 30-July 6
PCR amplification of mCherry again from pNigel7 for higher yields, then it was gel extracted. Picked colonies and grew cultures of bacteria containing pMDC107.
July 7-13
Miniprep of bacteria containing pmdc107 and pNigel7, made save cultures of both. Ran gradient PCR of mCherry due to it’s failure the first tries.
July 14-20
Gradient PCR for NOS 2 terminator then a “band stab” PCR from the band in the gel. PCR amplification of Actin 8. PCR amplification of GFP, Nos1, and Nos2 from newer pmdc107.
July 21-27
Having trouble isolating mCherry still, PCR with taq polymerase and phusion polymerase to eliminate factors. Also, PCR of Nos2 with taq polymerase for same reasons as mCherry. PCR amplification of dCas9 with phusion polymerase.
July 28- Aug 3
PCR amplification of dCas9 from the band last week, in addition to starting from plasmid DNA. Tried again with lower annealing temperatures. They were successful and the bands were then gel extracted. At this point all of the components have been isolated and PCR amplified to our specifications.
Aug 4-10
Calculations were started for a Gibson Assembly for all of the components. In addition, dCas9 was PCR amplified again using different annealing temperatures and again using different DNA. Two CBAR’s were ran to assemble CMV35s, GFP, Nos1, Nos2, CMV35s(2), mCherry, and Nos3. It was unsuccessful and more calculations were done to troubleshoot the problem.
Aug 11-17
Tried smaller Gibson Assemblies putting together CMV35s with GFP, Nos1 with Nos2, and CMV35s(2) with mcherry and Nos3. All three CBAR’s were successful and were rescue PCR amplified. Then, CBAR of g-blocks for viral promoters, all were successful and also rescue PCR amplified. PCR amplification of Actin, Myosin, Profilin from the Arabidopsis thaliana genome.
Aug 18-24
PCR products were all ran on a gel and only 2 viral promoters and the Actin, Myosin, and Profilin were successful.
**From here the Project switched focus from Cas9 to the promoter project for consolidation purposes and time constraints**
Promoter Project
**The components of this project are the same as those in Cas9 and because of this, the notebook starts in August**
Aug 11-17
Re-isolated cytoskeletal promoter, Actin, from Arabidopsis thaliana genome. Actin was unsuccessful so it was PCR amplified again. A genomic isolation was performed from Arabidopsis leaves. Gel extractions of Myosin, Actin, MMV, and ClCuV.
Aug 18-24
PCR of myosin and profilin were both successful. CBAR 1 and CBAR 2 were performed again as described in cas9 notebook. In addition, FiMV promoter was put together again with CBAR. All were successful except the CBAR’s that were rescue PCR amplified. It was determined that the annealing temperatures were not compatible for CBAR 1 and 2 amplification so new primers were ordered. FiMV was successfully PCR amplified and was gel extracted. An intermediate CBAR of CMV35s(2) with mCherry (CBAR 3.5) was conducted and successfully amplified.  Each component of the construct was CBARed again successfully and a CBAR of those parts was conducted to assemble the entire construct. It was unsuccessful.
Aug 25-31
More intermediate CBAR’s were conducted to piece together CBAR 1 with CBAR 2 from above, named CA. Also, CBAR 3.5 and Nos3 were joined together and named CB. Both attempts were unsuccessful and were tried again.
Sept 1-7
Began biobrick assembly by PCR amplifying Actin, Profilin, ClCuV, MMV, CesA 7, CesA 8, and Myosin with new primers that had the correct restriction enzymes on them. Continuation of trying for successful CBAR reactions with a series of new calculations, new primers, and new annealing temperatures during rescue PCR.
Sept 8-14
Restriction enzyme digest of all of the biobrick components and pSB1C3. The was some success with CBAR 1 and CBAR 3.5, but nothing could be amplified to standards.
Sept 15-21
Biobrick parts were individually ligated into vectors and transformed. Colonies were picked, cultures were grown and mini prepped to be sent to sequencing. Continued to troubleshoot CBAR reactions and look into a different method for assembly.
CesA Project
June 2-8
Growing cultures of pEarlygate101 and pmdc107
June 9-15
PCR of CesA1 promoter from Arabidopsis thaliana genome and it was unsuccessful. Went into the troubleshooting phase
June 16-29
After consulting with graduate students and advisors we tried to PCR amplify the CesA ! promoter using different annealing temperatures, concentrations, polymerases, and elongation times. After 12 unsuccessful attempts and two weeks of troubleshooting, we had our first successful band.
June 30-July 13
Gel extraction of successful band and PCR amplified again for better quality and higher concentrations. Moved into PCR amplification of CesA 4, CesA 7, and CesA 8 from the genomic DNA. Just as with the promoter, we encountered many problems and did not have a successful band with genomic DNA. cDNA had been ordered and arrived at the end of the week.
July 14-20
After cDNA arrived, we attempted to PCR amplify CesA 7 and CesA 8 as CesA 4 was backordered. It was successful after many failed attempts.
July 21-27
The successful bands were PCR amplified and we achieved high enough yields to proceed. Although, the construct could not be complete so cDNA was then ordered from Japan.
July 28- Aug 3
Since the cDNA from Japan did not arrive, we redesigned the construct without it and had to wait for primers to come in. CesA 7 and CesA 8 were then sent for sequencing.
Aug 4-10
Tried multiple times to isolate CesA 4 from a cDNA library, but we were unsuccessful.
**After meeting with our advisors, we decided to put this project on hold and focus our efforts on the promoter project**
Butanol Project
April 21-28
A rough draft of Castor and Pollux were designed using a 2A sequence to separate the different genes in the pathway. The amino acid sequences for the genes were tracked down and converted back into optimized nucleic acid sequences for Nicotiana and Physcomitrella. Sequences for the CMV promoter and NOS terminator were also found and added to the sequences.
May 12-18
More research was done into the 2A sequence, and we are more confident that we can chain together the different proteins for Castor and Pollux. This also raised concerns that the 2A sequence would leave “scars” on the the upstream and downstream coding sequences. This was addressed by searching for the structures of the proteins in Castor and Pollux and placing the 2A sequence at the least obstructed end of the protein.
May 19-25
Other pathways that utilized NADH were found that could synthesize butanol, but the team decided to stay with a NADPH dependent pathway. It was also suggested by our advisors that we could divert one of the intermediates in the butanol synthesis pathway for making a plastic as a secondary product. The proteins in the butanol synthesis pathway were checked again to make sure that the 2A sequence would interfere as little as possible with protein folding. The drafts of the constructs were also checked for stop codons and edited to remove restriction sites that could be problematic in the future.
May 26-June 1
An enzyme that could catalyze the synthesis of the plastic, polyhydroxybutyrate, was tracked down. This enzyme will be in its own construct to allow for co-transformation to make either plastic or butanol. The drafts of the constructs were also checked for errors and corrected. They were also optimized for assembly.
June 2-8
The three constructs – PhaC, Castor, and Pollux –were run through final checks.  Vector plasmids were also cultured.
June 9-15
DNA was ordered for the synthesis of PhaC and Castor. Various issues were addressed concerning problems with synthesis. As a result the NOS terminator would have to be amplified from a source, and could not be synthesized.
June 16-22
The NOS terminators were amplified in preparation for assembly of PhaC and Castor.
June 23-29
The DNA arrived, was hydrated, and a preliminary CBAR reaction was completed to assemble the two constructs. The desired product was then PCR amplified from the CBAR reaction.
June 30-July 6
A gel of the PCR amplified CBAR did not yield either of the constructs.
July 7-13
The PCR amplification of both CBAR were again run on a gel, but again yielded nothing. The CBAR were run again, but were heat killed this time. This did not solve the problem and the CBAR were still unsuccessful. Two new vector plasmids were also cultured. A gradient PCR was also tried for PhaC.
July 14-20
Pollux was optimized for synthesis. This required breaking it into two major segments that would later need to be assembled. It was also suggested that PhaC and Castor be assembled with intermediates instead of all at once. New primers were designed to implement this idea.
July 21-27
Intermediate components of PhaC and Castor were prepared via PCR. These were then run through a CBAR to assemble them into their final constructs.
July 28-August 3
PhaC was digested and ligated into its final construct, minus the NOS terminator.
August 4-10
The PhaC construct was transformed and colonies were picked for a screen.
August 11-17
The screen of PhaC looked promising. Castor was digested and ligated into its final vector plasmid. Colonies were picked and cultured for a screen.
August 18-24
PhaC was sequenced and found to be correct.
August 25-31
The PhaC construct was digested and the NOS terminator prepared previously was ligated. The screen colonies for Castor were digested and run on a gel. The results looked promising for Castor being in the desired construct.
September 1-7
Castor was submitted to sequencing and found to be correct. However one of the sequencing reactions went poorly and will need to be re-run. Primers were also designed for amplifying parts to be submitted to the registry.
September 8-14
The failed sequencing reaction for Castor was repeated and confirmed the sequence. Parts were also amplified and prepared to be submitted to the registry. The PhaC and Castor constructs were transformed into Agrobacterium.
September 15-21
The parts to be submitted were prepared and sent to the registry. PhaC and Castor were also transiently transformed into Nicotiana tabacum.
September 22-28
The sections of the transformed Nicotiana leaves were cut and stained. These were viewed with fluorescence microscopy for the presence of the plastic polyhydroxybutyrate. Due to issues with staining, the results were inconclusive.
Vanillin Project
May 27-31
Received Pcambia 1281z and 1201 vectors from Dr. Gu’s lab. Plasmids we transformed and amplified. Received Pseudomonas putida KT2440 from Dr. Salis’ lab. Genomic DNA extraction was performed on the Pseudomonas putida.  Genomic DNA was confirmed by Nano-drop. Primers for PCR of the enzyme 4-hydroxycinnamoyl-CoA hydratase/lyase (HCHL), were ordered through IDT.
June 3 – 7
Genomic DNA was amplified by PCR using phusion polymerase and following manufacturer instructions, in order to amplify the gene of 4-hydroxycinnamoyl-CoA hydratase/lyase (HCHL). The PCR amplified product was run on an electrophoresis gel and a  ~ 800 base pairs was observed.  DNA was purified through gel extraction.
June 10-14
HCHL gene was ligated to pCambia vectors, and then the ligated vector were transformed. No colonies were observed.  A new approach was developed and new primers were ordered.
June 17 -21
A new approach and corresponding primers for amplifying the 35s promoter, HCHL gene and NOS terminator with sticky ends interconnectable between them and the Pcambia vectors were developed. These primers were ordered through IDT.
Junes 24-28
The 35s promoter and NOS terminator were successfully amplified. The PCR amplified product was run on an electrophoresis gel and the corresponding bands of ~450 and  ~150 base pairs were observed. The DNA was later purified by gel extraction. HCHL gene was amplified and purified with new primers.
July 1-5
A double digestion of PCambia 1201 and 1281z vectors was run and then purified via gel electrophoresis and gel extraction. Biobrick alcohol acetyltransferase, which catalyzes the conversion of butanol or 2-methylbutanol to butyl acetate, was successfully transformed. The corresponding plasmids were purified and the gene was amplified via PCR.
July 8-12
The 35s promoter, NOS terminator and HCHL gene were double digested. The HCHL gene was successfully ligated to the NOS terminator. When run on an electrophoresis gel, a ~ 950 base pair was observed. A pure HCHL gene was also run as a control and the ligated band was clearly bigger than the control.
July 15-19
The construct of containing the HCHL gene and NOS terminator was amplified via PCR.  This construct was successfully ligated with the 35s promoter. When run on an electrophoresis gel, a ~ 1500 band was observed.
July 22-26
The construct of the 35s promoter, NOS terminator and HCHL gene was amplified via PCR using forward primer of 35s promoter and reverse primer of NOS terminator. An electrophoresis gel confirmed the successful amplification of the construct, proving that our ~1500 band is our desired construct.
July 29 – August 2
Attempted to ligated construct containing the 35s promoter, HCHL gene and NOS terminator in pCambia p1281z and p1201 vectors. Later transformed into competent cells.  No colonies were observed.
Home
Team
Notebook
Promoter
Project
Cas9
Project
CesA
Project
Butanol
Project
Vanillin
Project
Parts
Human Practices
Attributions
Retrieved from " http://2013.igem.org/Team:Penn_State/Notebook "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
