<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Penn_MaGellinResults"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Penn/MaGellinResults">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Penn/MaGellinResults&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Penn/MaGellinResults&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Penn/MaGellinResults&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Penn/MaGellinResults" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Penn/MaGellinResults</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><P><TITLE>Penn iGEM</TITLE></P></DIV><DIV id="text"><DIV class="textwrap"><HEADER><H1><CENTER><B>Assay Validation</B></CENTER></H1></HEADER>
For a detailed, graphical explanation of the MaGellin work flow, please download the <A href="https://dl.dropboxusercontent.com/u/11828463/MaGellin%20Spec%20Sheet.pdf">MaGellin Workflow Specifications Sheet</A>, which includes all of the steps in the MaGellin workflow.
<B>Our Team’s Solution.</B> In order to address the challenges associated with developing new site-specific methylase proteins, our team proposed several different strategies. First, we proposed a migration away from mammalian systems and into E. coli. E. coli does not have a native cytosine methyltransferase, and therefore offers a noise-free environment for methylation studies. Any methylation of CpG sites in E Coli would be a product of a candidate engineered protein rather than the native organism. Second, we envisioned a modular one-plasmid system that can be employed for quickly and cheaply screening the activity and specificity of any DNA binding domain – methyltransferase fusion protein. This plasmid-based methylation assay is called MaGellin.
<DIV style="margin-left:auto;margin-right:auto;text-align:center"><FIGURE><FIGCAPTION><I>The MaGellin plasmid includes all the features needed to clone, express, and assay site-specific methylases.</I></FIGCAPTION></FIGURE></DIV><B>Plasmid Features.</B> To accommodate the MaGellin assay, our team designed a plasmid with several key features:
<OL><LI>CpG Methyltransferase (M.SssI) with a generic linker sequence in the cloning site. For a working fusion protein and assay, only a DNA binding domain must be cloned into the plasmid. This inherently standardizes MaGellin and lessens the time a user of the assay must spend cloning.</LI><LI>Multiple cloning site downstream of T7 promoter for orthogonal expression of fusion protein in T7 Express competent E. coli.</LI><LI>Cloning site for a smaller DNA sequence, specific to the fusion protein being screened – named the “target site”, where the protein will bind. This can be the binding site for a CRISPR-Cas, TALE, Zinc Finger, or transcription factor</LI><LI>AvaI restriction site 4 bases downstream of the target site – the AvaI restriction enzyme is blocked by methylated CpG sites, thus screening for site specific methylation becomes equivalent to screening for AvaI digestion</LI><LI>AvaI restriction site sufficiently further downstream of the target site – named the off-target site. This site screens for non-specific DNA methylation as it is spatially removed from where the fusion protein binds to the plasmid.</LI><LI>XbaI site for linearization of the plasmid. Linearizing the plasmid simplifies analysis of the AvaI digestion by gel electrophoresis.</LI><LI>Validated bisulfite conversion primer binding sites, so users do not need to go through the time-consuming primer design process if they choose to fortify MaGellin’s results with bisulfite sequencing results</LI><LI>sgRNA cloning site for users who want to target a CRISPR-Cas binding domain. The sgRNA is constitutively expressed and can be swapped by restriction digest.</LI><LI>Validated bisulfite conversion primers for users who choose to advance to bisulfite sequencing, for even higher resolution in detecting methylation, after proving their enzyme’s efficacy with our MaGellin assay.</LI><LI>Kanamycin resistance as a selection marker</LI></OL><B>Noiseless Chassis.</B> After cloning this plasmid, we faced the challenge of choosing the correct cell line for the assay. We chose to transform into T7 Express cells for several reasons:
<OL><LI>T7 RNA Polymerase in the lac operon allows us to turn on expression of fusion protein after induction with IPTG</LI><LI>In the T7 Express cell line, genes for several restriction enzymes known to target methylated DNA are knocked out (McrA-, McrBC-, EcoBr-m-, Mrr-). This ensures that our assay plasmid is not cleaved in vivo. Results are difficult, if not impossible, to interpret in the commonly used BL21 cell line.</LI></OL><B>MaGellin Workflow.</B> The workflow for screening new fusion proteins with the one plasmid MaGellin bacterial system is as follows:<B>Assemble</B> the MaGellin backbone together with a DNA-binding protein and target sequence of your choosing.
<LI>Digest BBa_K1128001 (the MaGellin backbone) and BBa_K1128002 (the linker-M.ssI construct) with EcoRI and PstI.
</LI><LI>Ligate K1128002  into the K1128001 backbone. </LI><LI>PCR amplify your DNA-binding protein of choice.  In order to keep everything in frame, use the following 5’ extensions on the PCR primers:</LI><LI> Forward: CAGGAGGAATTC[ATG] (add start codon only if not included in gene).</LI><LI> Reverse: CTCTAGAAGCGGC (make sure to remove the stop codon). </LI><LI> Use EcoRI and XbaI to ligate the DNA-binding protein into the MaGellin backbone, fusing it in frame to the linker-M.sssI construct.</LI><LI> Clone in your target sequence using BamHI and XhoI.</LI><B>Methylate</B> the MaGellin plasmid <I>in vivo</I>.<OL><LI>Transform the completed MaGellin plasmid into T7 Express. </LI><LI>Induce culture with 1 mM IPTG.</LI><LI>Incubate in a shaker at 37C for 5 hours.</LI><LI>Miniprep to isolate the plasmid.</LI></OL><B>Digest</B> the methylated plasmid.<OL><LI> Digest 600 ng of miniprep DNA in a 15 uL reaction with 10 U of both XbaI and AvaI.</LI><LI> Incubate reaction for 1 hour at 37C.</LI></OL><B>Analyze</B> the data using the MaGellin Software Package.
<LI>Run the entire digestion reaction on a 1% agarose gel.</LI><LI>Take a photo of the gel.</LI><LI>Upload and analyze the gel photo using the MaGellin Software Package. </LI><OL><LI>Look for 3 distinct band patterns that correspond to specific and interpretable methylation outcomes.</LI><OL><LI>The presence of large one band corresponds to non-site-specific DNA methylation (AvaI was blocked at both the target and off target sites, and thus only XbaI cut the plasmid)</LI><LI>The presence of two bands corresponds to site-specific DNA methylation (AvaI was only blocked at the target site, thus AvaI cut in the off target site and XbaI cut the plasmid)</LI><LI>The presence of three bands corresponds to no DNA methylation – or an inactive fusion protein (AvaI was not blocked at either the target or off target sites and XbaI cut the plasmid) </LI></OL></OL></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:Penn/MaGellinResults">http://2013.igem.org/Team:Penn/MaGellinResults</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Penn/MaGellinResults" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Penn/MaGellinResults" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Penn/MaGellinResults&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Penn/MaGellinResults&amp;oldid=335337" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>