"
Page
Discussion
View source
History
teams
Log in
 
Team:Penn/AssayOverview
From 2013.igem.org
Penn iGEM
Assay Overview
For a detailed, graphical explanation of the MaGellin work flow, please download the MaGellin Workflow Specifications Sheet , which includes all of the steps in the MaGellin workflow. Click on the image below to play a step-by-step animation that details the process.
Early on, it became clear that there was no standardized assay for testing the activity and specificity of site-specific methylases. Importantly, it seemed many groups were not focusing enough on their methylase's tendency to also methylate off-target sequences; and were only measuring site-specific methylation.
Measuring Methylation
Two techniques have traditionally been employed to measure DNA methylation: Restriction Based. The first, called Combined Bisulfite Restriction Analysis (COBRA), involves chemically converting unmethylated cytosines into uracils (a process called bisulfite conversion), while leaving methylated cytosines intact. Performing PCR that amplifies the region of interest leaves methylated cytosines intact and converts unmethylated cytosines to thymines (Figure 1). Samples are then digested using an enzyme that will only cut the unconverted (originally methylated) cytosines. The enzyme can no longer recognize unmethylated sites, as they are “TG” instead of “CG”. Even with the help of advanced algorithms, designing primers for this process is not always feasible, and the process needs to be optimized each time a new site is to be analyzed. Furthermore, the workflow takes several days, is expensive, and is not high throughput enough to accommodate screening libraries of candidate DNA-binding-domain-methylase fusion proteins. It has recently fallen out of favor because it is difficult to interpret and does not consider all CpG sites, but only ones which fall within a restriction enzyme’s recognition sequence. (Xiong 1997 and Li 2002). Our methylation assay, MaGellin, is also restriction-based but is much simpler than COBRA because it does not require bisulfite conversion of the DNA. This eliminates most of the problems that made COBRA unwieldy.
Figure 1: COBRA. This restriction based assay detects methylation at CpG sites that fall within restriction enzyme recognition sequences.
Sequencing Based. The second, and more commonly employed technique, is bisulfite sequencing, which employs the same bisulfite conversion step previously described, and is immediately followed by sequencing. Unmethylated cytosines are read as thymines and methylated cytosines are read as cytosines. Comparing converted and unconverted sequences reveals the methylation pattern with high resolution. Despite its advantages, this method is time consuming and can become very expensive as more and more constructs are screened for activity and specificity (Darst 2010). We determined to include standardized bisulfite sequencing primers on the MaGellin plasmid, so users would have the option of bisulfite sequencing after screening functional site-specific methylases with our quicker, less expensive, and intuitive digestion based assay.
The MaGellin Methylation Assay
Our Team’s Solution.
In order to address the challenges associated with developing new site-specific methylase proteins, our team proposed several different strategies. First, we proposed a migration away from mammalian systems and into E. coli. E. coli does not have a native cytosine methylase, and therefore offers a noise-free environment for methylation studies. Any methylation of CpG sites in E Coli would be a product of a candidate engineered protein rather than the native organism. Second, we envisioned a modular one-plasmid system that can be employed for quickly and cheaply screening the activity and specificity of any DNA binding domain – methylase fusion protein. This plasmid-based methylation assay is called MaGellin.
Figure 2: The MaGellin plasmid includes all the features needed to clone, express, and assay site-specific methylases.
Plasmid Features.
To accommodate the MaGellin assay, our team designed a plasmid with several key features:
CpG methylase(M.SssI) with a generic linker sequence in the cloning site. Only a DNA binding domain needs to be cloned into the plasmid for a working fusion protein and assay, . This inherently standardizes MaGellin and minimizes the time a user of the assay needs to spend cloning.
Multiple cloning site downstream of T7 promoter for orthogonal expression of fusion protein in T7 Express competent E. coli.
Cloning site for a smaller DNA sequence, specific to the fusion protein being screened – named the “target site”, where the protein will bind. This can be the binding site for a Zinc Finger, TALE, CRISPR-Cas, or transcription factor.
AvaI restriction site 4 bases downstream of the target site – the AvaI restriction enzyme is blocked by methylated CpG sites, thus screening for site specific methylation becomes equivalent to screening for AvaI digestion
AvaI restriction site sufficiently further downstream of the target site – named the "off-target site". This site screens for non-specific DNA methylation as it is spatially removed from where the fusion protein binds to the plasmid.
XbaI site for linearization of the plasmid. Linearizing the plasmid simplifies analysis of the AvaI digestion by gel electrophoresis.
Validated bisulfite conversion primer binding sites, so users do not need to go through the time-consuming primer design process if they choose to complement MaGellin’s results with bisulfite sequencing for even higher resolution in detecting methylation, after proving their enzyme’s efficacy with our MaGellin assay.
sgRNA cloning site for users who want to target a CRISPR-Cas binding domain. The sgRNA is constitutively expressed and can be swapped by restriction digest.
Kanamycin resistance as a selection marker.
Noiseless Chassis.
After cloning this plasmid, we faced the challenge of choosing the correct cell line for the assay. We chose to transform into T7 Express cells for several reasons:
T7 RNA Polymerase in the lac operon allows us to turn on expression of fusion protein after induction with IPTG
In the T7 Express cell line, genes for several restriction enzymes known to target methylated DNA are knocked out (McrA-, McrBC-, EcoBr-m-, Mrr-). This ensures that our assay plasmid is not cleaved in vivo. Results are difficult, if not impossible, to interpret in the commonly used BL21 cell line.
MaGellin Workflow.
For a detailed, graphical explanation of the MaGellin work flow, please download the MaGellin Workflow Specifications Sheet , which includes all of the steps in the MaGellin workflow. The workflow for screening new fusion proteins with the one plasmid MaGellin bacterial system is as follows:
Assemble the MaGellin backbone together with a DNA-binding protein and target sequence of your choosing.
Digest BBa_K1128001 (the MaGellin backbone) and BBa_K1128002 (the linker-M.ssI construct) with EcoRI and PstI.
Ligate K1128002  into the K1128001 backbone.
PCR amplify your DNA-binding protein of choice.  In order to keep everything in frame, use the following 5’ extensions on the PCR primers:
Forward: CAGGAGGAATTC[ATG] (add start codon only if not included in gene).
Reverse: CTCTAGAAGCGGC (make sure to remove the stop codon).
Use EcoRI and XbaI to ligate the DNA-binding protein into the MaGellin backbone, fusing it in frame to the linker-M.sssI construct.
Clone in your target sequence using BamHI and XhoI.
Methylate the MaGellin plasmid in vivo.
Transform the completed MaGellin plasmid into T7 Express.
Induce culture with 1 mM IPTG.
Incubate in a shaker at 37C for 5 hours.
Miniprep to isolate the plasmid.
Digest the methylated plasmid.
Digest 600 ng of miniprep DNA in a 15 uL reaction with 10 U of both XbaI and AvaI.
Incubate reaction for 1 hour at 37C.
Analyze the data using the MaGellin Software Package.
Run the entire digestion reaction on a 1% agarose gel.
Take a photo of the gel.
Upload and analyze the gel photo using the MaGellin Software Package.
Look for 3 distinct band patterns that correspond to specific and interpretable methylation outcomes.
The presence of large one band corresponds to non-site-specific DNA methylation (AvaI was blocked at both the target and off target sites, and thus only XbaI cut the plasmid)
The presence of two bands corresponds to site-specific DNA methylation (AvaI was only blocked at the target site, thus AvaI cut in the off target site and XbaI cut the plasmid)
The presence of three bands corresponds to no DNA methylation – or an inactive fusion protein (AvaI was not blocked at either the target or off target sites and XbaI cut the plasmid)
Figure 3: The full workflow to use MaGellin, available from the BioBrick registry.
←Previous Next→
Home Spec Sheet Sitemap
Penn iGem © 2013
Retrieved from " http://2013.igem.org/Team:Penn/AssayOverview "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
