"
Page
Discussion
View source
History
teams
Log in
 
Team:Penn/MaGellinMotivation
From 2013.igem.org
Penn iGEM
Background Information
For a detailed, graphical explanation of the MaGellin work flow, please download the MaGellin Workflow Specifications Sheet , which includes all of the steps in the MaGellin workflow.
Epigenetics.
The code of life is more than a sequence of A’s, C’s, T’s, and G’s. Heart cells in the human body contain the same DNA as skin cells in the foot, yet these two cell types behave in radically different ways. Both contain the DNA for every one of over 20,000 human genes but express only the ones needed for their own form and function. These differences are due to epigenetic controls. Epigenetics refers to any regulation of gene expression and phenotype that is not based on the sequence of bases in DNA. In addition to governing cellular differentiation, epigenetic mechanisms facilitate the proper functioning of a cell. When these mechanisms go awry, neurodevelopmental disorders, immunodeficiency, and cancer can result. Epigenetic phenomena are amongst the primary ways gene expression is regulated; yet, our current understanding of them is limited, especially due to the challenge of studying them in noisy mammalian systems. By virtue of its emphasis on the isolation and testing of biological networks in engineered systems with reduced complexity, synthetic biology offers the promise of fostering understanding of phenomena difficult to study in their native environments. It is thus an ideal forum for epigenetic studies. At its core, synthetic biology also involves the engineering of non-native networks for useful purposes. Well-tuned gene expression is essential to the proper functioning of synthetic biological circuitry, yet epigenetics has not yet been fully explored as a tool for this application.
DNA Methylation.
DNA methylation is one of the most prominent and powerful mechanisms of epigenetic control. In mammals, enzymes called methyltransferases catalyze the addition of methyl groups to cytosines in CpG sites (a cytosine next to a guanine); these sites are abundant throughout the human genome (Venter 2001). Methylated cytosines in CpG sites can block transcription factors from binding to DNA and repress gene expression (Jones 1998 and Nan 1998). Although epigenetic factors do not affect genotype, they do affect phenotype. Specific methylation patterns are necessary for healthy development and disruption of methylation patterns has been shown to lead to many diseases (Cooper 1988, Rideout III 1990, Baylin 1998, Jones 1999, and Amir 1999).
An Unmet Need
Epigenetic Engineering.
Synthetic biology largely involved engineering genetic networks in bacterial chasses at its inception, but there have been increasing efforts to engineer more complex mammalian systems. Yet, despite the dramatic effects that subtle epigenetic modifications can have on phenotype, there are no robust and well-characterized tools for engineering the epigenome. If synthetic biologists intend to successfully transition from bacterial to mammalian chasses, they must appreciate the epigenetic control of gene expression. An epigenetic toolbox for synthetic biology would enable the creation of engineered organisms that more closely rival their natural counterparts with regard to the subtlety and robustness of cellular control. We decided to make one of these tools, an enzyme that can direct DNA methylation to specific sequences.
Methylation-mediated transcriptional silencing could grant synthetic biologists access to an additional layer of genetic control.
Epigenetic Disease.
Methylation abnormalities are linked to a wide range of diseases. Many types of cancer can be characterized by their DNA methylation profiles. In fact, DNA methylation has been called “the hallmark of cancer” (Syzf, 2004).  Specifically, hypomethylation of oncogenes has been linked to tumorigenesis and loss of CpG methylation at specific sites has been implicated as a main cause of cancer. No drugs, approved or in testing, can restore methylation. We determined to engineer enzymes that are up to the task.
Site-specific methylation could potentially restore normal methylation levels to hypomethylated cells.
Existing Technologies
Zinc-Finger Systems.
Some progress has been made towards developing a tool that can make methylate DNA in a controlled manner (Xu 1997, Carvin 2003, and van Steensel 2000). Methylases are the enzymes which catalyze DNA methylation, but they are not inherently targeted to any specific DNA sequence. Since the 1990s, zinc finger proteins that bind a given DNA target sequence have been fused to methylases in an attempt to create an enzyme capable of methylating predetermined DNA sequences (Xu 1997). Although these fusion proteins have been somewhat successful in directing and controlling DNA methylation, they are known to methylate “off-target” DNA sequences distinct from the region intended to be methylated, it is difficult to modify the zinc finger domain to target unique DNA sequences, and the protein engineering process is expensive (Li 2006, Papwort 2005, and Desjarlais 1992). For these reasons, zinc finger methyltransferase fusion proteins have not gained wide spread use in epigenetic studies, and have not been considered for therapeutic purposes. We recapitulated the results with published zinc finger fusions, but were eager to improve on these existing technologies.
Molecular model of M.SssI (Generated using Phyre2)
TAL Effector Systems.
The TALE is a more site specific and easily customizable DNA-binding domain than the zinc finger, which has increasingly been used in synthetic biology DNA targeting experiments. As we worked on our project this summer and fall, we were thrilled to see other labs thinking about epigenetic engineering. In the past few months, there has been published work on TALEs  fused with histone methylases, histone demethylases, and DNA demethylases (Konermann 2013, Mendenhall 2013, Maedner 2013). The contribution of our TALE DNA methylase completes this initial suite of tools for epigenetic engineering.
Slowed Progress.
DNA binding proteins that are more modular and specific to their target sequence than zinc-fingers, such as the TALE and CRISPR-Cas systems, have gained widespread attention for their utility in genetic studies, but they have not been leveraged for improved targeted methylation. Screening new methyltransferase fusions for activity and specificity is difficult and expensive, which could hamper protein-engineering efforts (Gaj 2013). So, we developed a new methylation assay, MaGellin, to accelerate development and cut costs. Then, we used MaGellin to prove the efficacy of the first TALE-methylase fusion.
Noise Problem.
An additional challenge that makes the design and characterization of proteins which methylate specific DNA sequences is the signal:noise ratio in mammalian cells. With a large amount of CpG sites naturally methylated in mammalian cells, it is difficult to differentiate protein activity from background noise. However, CpG methylation is completely orthogonal to E.coli so we developed our methylation assay, MaGellin, in this bacterial chassis.
Lack of a Standardized Assay.
The most pressing challenge, however, was the lack of a standardized assay for the activity of site-specific methylases that was inexpensive, fast, robust, and easy to use. So, the first element of our project and our most important contribution to the community was the development of the MaGellin assay.
Next→
Home Spec Sheet Sitemap
Penn iGem © 2013
Retrieved from " http://2013.igem.org/Team:Penn/MaGellinMotivation "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
