"
Page
Discussion
View source
History
teams
Log in
 
Team:Penn/FusionMotivation
From 2013.igem.org
Penn iGEM
An Unmet Need
Epigenetic Disease. Methylation abnormalities are linked to a wide range of diseases. Many types of cancer can be characterized by their DNA methylation profiles. Specifically, hypomethylation of oncogenes has been linked to tumorigenesis and loss of CpG methylation at specific sites has been implicated as a main cause of cancer. No drugs, approved or in testing, can restore methylation. We determined to engineer enzymes that are up to the task.
Epigenetic Engineering. Synthetic biology largely involved engineering genetic networks in bacterial chasses at its inception, but there have been increasing efforts to engineer more complex mammalian systems. Yet, despite the dramatic effects that subtle epigenetic modifications can have on phenotype, there are no robust and well-characterized tools for engineering the epigenome. If synthetic biologists intend to successfully transition from bacterial to mammalian chasses, they must appreciate the epigenetic control of gene expression. An epigenetic toolbox for synthetic biology would enable the creation of engineered organisms that more closely rival their natural counterparts with regard to the subtlety and robustness of cellular control. We decided to make one of these tools, an enzyme that can direct DNA methylation to specific sequences.
Existing Technologies Zinc-Finger Systems. Some rudimentary progress has been made towards developing a tool that can make epigenetic modifications in a controlled manner (Xu 1997, Carvin 2003, and van Steensel 2000). Since the 1990s, zinc finger proteins that bind a given DNA target sequence have been fused to methyltransferases in an attempt to create an enzyme capable of methylating predetermined DNA sequences (Xu 1997). Although these chimeric proteins have been somewhat successful in directing and controlling DNA methylation, they are known to methylate “off-target” DNA sequences distinct from the region intended to be methylated, it is difficult to modify the zinc finger domain to target unique DNA sequences, and the protein engineering process is expensive (Li 2006, Papwort 2005, and Desjarlais 1992). For these reasons, zinc finger methyltransferase fusion proteins have not gained wide spread use in epigenetic studies, and have not been considered for therapeutic purposes. We recapitulated the results with published zinc finger fusions, but were eager to improve on these existing technologies.
Slowed Progress. DNA binding proteins that are more modular and specific to their target sequence than zinc-fingers, such as the TALE and CRISPR-Cas systems, have gained widespread attention for their utility in genetic studies, but they have not been leveraged for improved targeted methylation. Screening new methyltransferase fusions for activity and specificity is difficult and expensive, which could hamper protein-engineering efforts (Gaj 2013). So, we developed a new methylation assay, MaGellin, to accelerate development and cut costs. Then, we used MaGellin to prove the efficacy of the first TALE-methyltransferase fusion.
Noise Problem An additional challenge that makes the design and characterization of proteins which methylate specific DNA sequences is the signal:noise ratio in mammalian cells. With a large amount of CpG sites naturally methylated in mammalian cells, it is difficult to differentiate protein activity from background noise. However, CpG methylation is completely orthogonal to E.coli so we developed our methylation assay, MaGellin, in this bacterial chassis.
Next→
Retrieved from " http://2013.igem.org/Team:Penn/FusionMotivation "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
