"
Page
Discussion
View source
History
teams
Log in
 
Team:Penn/AssayValidation
From 2013.igem.org
Penn iGEM
Assay Validation
For a detailed, graphical explanation of the MaGellin work flow, please download the MaGellin Workflow Specifications Sheet , which includes all of the steps in the MaGellin workflow.
We have designed standardized bisulfite sequencing primers.
Bisulfite sequencing is a good next step after restriction digest to further characterize functional site-specific methylases, but it is inherently very difficult to design good primers. People use advanced algorithms for primer design that are still not guaranteed to successfully sequence some sequences. We went through 8 sets of primers, most of which did not show the proper bias to amplify only bisulfite converted DNA. Primer Set 2 was successful and is included with our MaGellin plasmid, much like VF and VR are included as standardized biobrick sequencing primers (Figure 1).
Figure 1: Validating standardized bisulfite sequencing primers. Primer Set 2 successfully amplifies bisulfite converted DNA but not unconverted DNA, as desired.
MaGellin effectively detects methylation in vitro.
First, we tested MaGellin with a purified methylase in vitro. The results made it clear that MaGellin can detect methylation at both the “target” and “off-target” site (Figure 2). MaGellin is also sensitive to various degrees of methylation (Figure 3). These experiments helped us optimize the ideal amount of plasmid and restriction enzyme to use in any study moving forward.
Figure 2: Plasmid DNA treated in vitro with purified M.SssI. The first three lanes were not treated and show zero methylation detection by our assay. The last three lanes were methylated and show 100% methylation. This figure validates that MaGellin is capable of clear input/output.
Figure 3: Plasmid DNA treated in vitro with purified M.SssI. Each lane was treated for a different amount of time, this figure shows that MaGellin, alongside our software package , can report relative differences in the level of methylation.
MaGellin detects methylation in vivo.
We expressed M.SssI in vivo and compared it with purified M.SssI used on the plasmid in vitro. In both cases, we saw similar full methylation of the plasmid, confirming that MaGellin can express methylases and report their activity in vivo (Figure 4).
Figure 4: M.SssI expressed in vivo compared with in vitro methylation.
Summary
We have created MaGellin, a new technology that facilitates screening novel DNA binding domain – methylase fusion proteins
Our assay is less expensive and faster than existing methods
We have eliminated noise associated with previous studies
We have a system with clear input/output
Our assay lends itself to high throughput screening of many different proteins
We are releasing it alongside an open source data analysis software package which streamlines the entire screening process
←Previous Next→
Home Spec Sheet Sitemap
Penn iGem © 2013
Retrieved from " http://2013.igem.org/Team:Penn/AssayValidation "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
