"
Page
Discussion
View source
History
teams
Log in
 
Team:Penn/MaGellinToolbox
From 2013.igem.org
Penn iGEM
Toolbox
For a detailed, graphical explanation of the MaGellin work flow, please download the MaGellin Workflow Specifications Sheet , which includes all of the steps in the MaGellin workflow.
Constructing a Toolbox
Our goal was to create a toolbox for developing customized site-specific methylases. Like any good toolbox, it includes three components: the tool, the assay, and the automation.
The Tool: a fusion between a DNA binding domain and a methylase enzyme
The Assay: a digestion based assay, called MaGellin, to measure both methylase activity and sequence specificity
The Automation: a unique software package that predicts experimental outcomes and analyzes gel electrophoresis images to measure methylase functionality
Toolbox with the three essential components.
Our toolbox satisfies 6 requirements:
Robust – includes functional fusion proteins for comparison
Standardized – all inclusive in one-plasmid with simple multiple cloning sites, standard bisulfite sequencing primers, and a digestion based methylation assay built in
Easy to Use – we programmed a software package to predict experimental outcomes and automate gel electrophoresis analysis
Inexpensive – methylase activity and specificity can be screened by simply digesting and running a gel
Noiseless – the bacterial chassis has no background CpG methylation
Open Source – we BioBrick’ed the full plasmid and the methylases
←Previous Next→
Home Spec Sheet Sitemap
Penn iGem © 2013
Retrieved from " http://2013.igem.org/Team:Penn/MaGellinToolbox "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
