<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Penn_Biobricks"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Penn/Biobricks">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Penn/Biobricks&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Penn/Biobricks&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Penn/Biobricks&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Penn/Biobricks" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Penn/Biobricks</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><P><TITLE>Penn iGEM</TITLE></P></DIV><DIV id="text"><DIV class="textwrap"><HEADER><H1><CENTER><B>Parts Submitted</B></CENTER></H1></HEADER>
For a detailed, graphical explanation of the MaGellin work flow, please download the <A href="https://static.igem.org/mediawiki/2013/e/e5/Spec_Sheet.pdf">MaGellin Workflow Specifications Sheet</A>, which includes all of the steps in the MaGellin workflow.
<H4><B>MaGellin Plasmid Backbone <A href="http://parts.igem.org/Part:BBa_K1128001">(BBa_K1128001)</A></B></H4>
MaGellin is a platform that streamlines the process of testing the specificity of DNA binding domains and the activity of methyltransferase-DNA binding domain fusion proteins. This technology combines the use of methylation-sensitive restriction enzymes with a powerful software package to identify and quantify the location and amount of CpG DNA methylation. The one-plasmid design ensures simple customization of the DNA-binding domain and targeting sequence, easy readout by agarose gel electrophoresis and analysis by included built-in software, and pre-designed bisulfite sequencing primers. 

The plasmid itself is a modified version of Novagen's pET-26(b)+ vector with additional sites cloned in, as well as the standard BioBrick prefix and suffix for cloning in standard parts from the registry. The plasmid backbone contains two AvaI restriction sites, which are composed of CYCGRG and are cut by the AvaI restriction enzyme. These sites can be CpG methylated, which blocks the activity of the AvaI enzyme. The backbone also contains a 9 base pair region that can be removed and replaced. This site is upstream of one of the AvaI sites and can act as a transcription factor &quot;target site&quot;, used to determine where certain DNA transcription factors bind. This biobrick has the BBa_J04450 insert, which is an RFP expression cassette with a pLac promoter. This backbone has a lac repressor gene, and will NOT express the RFP in vivo. The backbone has a gene for Kanamycin resistance.
<DIV style="margin-left:auto;margin-right:auto;text-align:center"><FIGURE><FIGCAPTION><I>The MaGellin plasmid includes all the features needed to clone, express, and assay site-specific methylases.</I></FIGCAPTION></FIGURE></DIV><I>Click on the image below to start the animation.</I><CENTER><FIGCAPTION><I>Figure 3: Interactive animation of MaGellin workflow</I></FIGCAPTION></CENTER><H4><B>CpG Methylase M.SssI <A href="http://parts.igem.org/Part:BBa_K1128000">(BBa_K1128000)</A></B></H4>
Epigenetics, the study of modifications made to DNA in order to regulate and control the functions of a cell, has several components, one of which is known as DNA methylation. This involves adding a methyl group (-CH<SUB>3</SUB>) to a base pair residue, which helps the cell determine if the methylated gene should be off, or if it should be on, and how much protein it should be expressing. The enzymes that catalyze the addition of methyl groups to the DNA are called methyltransferases, and each type of methyltransferase has its own unique activity. The most common type of methylation in a eukaryotic cell is called CpG methylation, which stands for <I>Cytosine-Phosphate-Guanine</I>, and it is the addition of a methyl group to the Cytosine at a CpG site. 
M.SssI is a methyltransferase derived from <I>Spiroplasma</I>, and is a very common protein used in epigenetics studies. It has been shown to work effectively both <I>in vivo</I> and <I>in vitro</I>, and the CpG methylation that M.SssI exhibits is orthogonal to the bacterial form of methylation, which methylates GATC sites in the genome using an enzyme called the DNA Adenine Methyltransferase (dam). In most bacteria, there are certain restriction enzymes native to the genome that digest CpG methylated sites as protection for the cell against foreign DNA. However, if the M.SssI is expressed in the cell <I>in vivo</I>, it will methylate the genome of the cell and cause the genome to be digested, leading to death of the cell. Therefore, when using this part, it should be noted that experiments and cloning involving this part should be done in cells that don’t have this restriction mechanism, called Mcr and Mrr restriction, which can be purchased from vendors such as New England Biolabs. For the Penn iGEM project,  the experiments involving this protein were done in T7 Express cells, purchased from New England Biolabs, which are classified as an Mcr- and Mrr- strain.

Part of Penn iGEM's project in 2013 was creating novel targeted methyltransferases, which are fusion proteins consisting of a DNA binding domain, such as a TAL effector, linked to a methyltransferse, M.SssI in our case, and testing those proteins for site specific methylation in vivo using the MaGellin assay workflow. We used this part as a positive control for those studies, in order to compare genome wide methylation and site specific methylation using our novel fusion proteins.
<DIV style="margin-left:auto;margin-right:auto;text-align:center"><FIGURE><FIGCAPTION><I>Molecular model of M.SssI (Generated using Phyre2)</I></FIGCAPTION></FIGURE></DIV><H4><B>CpG Methylase M.SssI with Linker <A href="http://parts.igem.org/Part:BBa_K1128002">(BBa_K1128002)</A></B></H4>
For full characterization see BBa_K1128000. This is an M.SssI methyltransferase protein with a linker on the N terminus. The part is ready for fusion to transcription factors. The linker is composed of a string of glycine residues.
<HEADER><H1><CENTER><B>Planned Future Submissions</B></CENTER></H1></HEADER><H4><B>Methyl CpG Binding Protein 2 (MECP2)<A href="http://parts.igem.org/Part:BBa_K1128007">(BBa_K1128007)</A></B></H4>
Methyl CpG binding protein 2 (MECP2) is a mammalian protein that binds to methylated CpG sites by recruiting histone deacetylases, and is essential for embryonic development. The Penn iGEM team is using this protein to test the feasibility of transcriptional silencing using CpG methylation in bacteria using a cell-free assay and a methylated red flourescent protein gene. The possibility of transcriptional silencing in E. Coli using methylation that is completely orthogonal to native E. Coli methylation can open up whole new for synthetic biologists, giving them a layer of control on genetic systems that has yet to be touched by traditional synthetic biology.
<DIV style="margin-left:auto;margin-right:auto;text-align:center"><FIGURE><FIGCAPTION><I>Molecular model of MECP2 </I></FIGCAPTION></FIGURE></DIV><DIV style="margin-left:auto;margin-right:auto;text-align:center"><FIGURE><FIGCAPTION><I>A SDS-Page gel shows expression of the MECP2 gene.</I></FIGCAPTION></FIGURE></DIV><H4><B>ΔC MECP2 <A href="http://parts.igem.org/Part:BBa_K1128008">(BBa_K1128008)</A></B></H4>
This protein is a version of MECP2 that has a truncated C-terminus. 
<H4><B>ΔN MECP2 <A href="http://parts.igem.org/Part:BBa_K1128009">(BBa_K1128009)</A></B></H4>
This protein is a version of MECP2 that has a truncated N-terminus.
</DIV></DIV><DIV id="pagefooter"><CENTER><A href="https://2013.igem.org/Team:Penn"> Home  </A><A href="https://static.igem.org/mediawiki/2013/e/e5/Spec_Sheet.pdf">Spec Sheet</A><A href="https://2013.igem.org/Team:Penn/sitemap">Sitemap</A></CENTER>
Penn iGem © 2013
</DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:Penn/Biobricks">http://2013.igem.org/Team:Penn/Biobricks</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Penn/Biobricks" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Penn/Biobricks" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Penn/Biobricks&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Penn/Biobricks&amp;oldid=359641" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>