"
Page
Discussion
View source
History
teams
Log in
 
Team:Penn/Abstract
From 2013.igem.org
Penn iGEM
Project Overview
The code of life is more than a sequence of A’s, C’s, T’s, and G’s. Heart cells in the human body contain the same DNA as skin cells in the foot, yet these two cell types behave in radically different ways. Both contain the DNA for every one of over 20,000 human genes but express only the ones needed for their own form and function. These differences are due to epigenetic processes. Epigenetics refers to any regulation of gene expression that is not based on the sequence of bases in DNA. DNA methylation (the heritable modification of cytosine groups by covalent addition of methyl groups) is one of the most important examples of epigenetic modification. DNA methylation governs critical cellular processes, and hypomethylation in certain regions of the genome can cause myriad diseases including neurodevelopmental disorders, immunodeficiency disorders, and cancer. Currently, there are no effective methods to target methylation to precise regions of the genome, and no simple or inexpensive assays to determine the regions which have been methylated. These problems are compounded by the fact that DNA methylation is almost always studied in mammalian systems with high background methylation activity, resulting in noisy or uninterpretable data. By virtue of its emphasis on the isolation and testing of biological networks in engineered systems with reduced complexity, synthetic biology offers the promise of developing an understanding of phenomena difficult to study in their native environments. It is thus an ideal forum for epigenetic studies, additionally, E. coli does not naturally methylate its DNA in the same way that mammalian systems do, making it a noiseless system to study DNA methylation. We thus sought to adopt a synthetic biology approach to accelerate the development of targeted methylases. We created a toolbox to enable researchers to engineer the epigenome, it consists of three elements: an economic, quick, and noiseless assay for targeted methylation, novel methylases currently being tested for targeting specificity, and software to enable quick screening of these selective enzymes. We have discovered subtle characteristics of targeted methylations which were not possible to study using conventional methods, and have made our assay and software open source through the BioBrick registry. We hope that our assay will allow for the accelerated screening and development of targeted methylases, which could be used to finely tune biological circuitry in synthetic systems, or eventually used to develop targeted methylation therapies. We also hope that our project will draw attention to epigenetic modifications as another potential layer of control for synthetic biologists.
There are three major components to our toolbox: the assay , the software , and the fusion protein
Home Spec Sheet Sitemap
Penn iGem © 2013
Retrieved from " http://2013.igem.org/Team:Penn/Abstract "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
