"
Page
Discussion
View source
History
teams
Log in
 
Team:Edinburgh/Bacterial/Red light sensor
From 2010.igem.org
home
team - illuminati
official
supervisors
advisors
students
environment
gallery
genomic BRIDGEs
the protocol
submitted parts
results
the future
references
bacterial BRIDGEs
the project
red light
red sensor
blue light
blue sensor
green light
green sensor
submitted parts
results
the future
references
modelling BRIDGEs
kappa
the genomic model
the bacterial model
the signalling model
tools
results
the future
references
human BRIDGEs
communication of science
iGEM survey
conversations
the epic
future applications
further thoughts
references
lab notes   
collaboration
attribution
BRIDGE
red light
red sensor
blue light
blue sensor
green light
green sensor
modelling
safety
protocols
acknowledgements
Overview: The red light sensor
The red light sensor was first engineered by UT Austin in iGEM 2004 . In this project, we will try to use it as one of the photo-activated sensors for our FORTH framework . The maximum response of the sensor is at a wavelength of 660nm.
The red-light sensor (Cph8) contains three parts:
PCB - Phycocyanobilin, coded for by HO and pcyA, required for light sensing
Cph8, a combination of Cph1 and EnvZ which, together with PCB, form a transmembrane, histidine kinase-based light sensing system
OmpR, a natural E. coli protein which couples with EnvZ and, when phosphorylated, activates genes downstream of the ompC promoter
In the absence of red light, phosphorylated OmpR activates EnvZ, which in turn promotes transcription from the ompC promoter and represses transcription from the ompF promoter, thus leading to the expression of LacZ (in this case under the regulation of the aforementioned promoter). This catalyses the formation of a black precipitate from S-gal(3,4-cyclohexenoesculetin-β-D-galactopyranoside).
When exposed to red light, an isomerization in the Cph1 and a structure change in the phycocyanobilin (PCB) part of the photoreceptor inactivates the histidine kinase acitity of EnvZ ( Figure 1 ). This then cascades down the OmpR pathway as shown in Figure 2 .
Figure 1: The activity of the chimaeric light receptor Cph8, described in further detail below..
Image: Levskaya et al. (2005)
The chimaeric light receptor Cph8 contains the photoreceptor from Cph1 (green) and the histidine kinase and response regulator from EnvZ–OmpR (orange); inset, conversion of haem to phycocyanobilin (PCB), which forms part of the photoreceptor. Red light drives the sensor to a state in which autophosphorylation is inhibited (right), turning off gene expression.
Miller assay showing that Cph8 is active in the dark (black bars) in the presence of PCB and inactive in the light(white bars). There is no light dependent activity in the absence of Cph8 (-) and there is constitutive activity when only the histidine kinase domain of EnvZ is expressed (+), or when the PCB metabolic pathway is not included (-PCB).
When an image is projected on to a bacterial lawn, the LacZ reporter is expressed only in the dark regions.
Transfer function of the circuit. As the intensity of the light is increased by using a light gradient projected from a 35mm slide, the circuit output gives a graded response.
Strategy
Our original plan was to revive UT Austin 2004 's BioBricks BBa_I15008 , BBa_I15009 , and BBa_I15010 from the provided Registry plates. Once revived, we aimed to combine them into a single red light sensing system, and transform cells with for characterisation of the system and for analysis of their compatibility with the mutated red luciferases .
Figure 2: Another view of the red light sensor.
Problems
The composite construct of all three BioBricks would not transform, and transformations of the individual BioBricks also failed. We attempted to amplify products out of the BioBrick to see if they were actually there, but the only component that we were able to recover was the sensing component, Cph8.
Thus, we retrieved transformants from the kanamycin-resistant version of BBa_K098010 , the HO-pcyA fusion deposited by Harvard 2008 , to supplement this and to complete our red light sensor.
BioBricks
The red light sensor has seen frequent use throughout the history of iGEM, beginning with the original coliroid parts by UT Austin 2004 to their adaptation by Harvard 2008 . We have updated and adapted their parts to the pSB1C3 chassis along with a number of different reporter systems for characterisation.
BBa_K322122 : phycocyanobilin synthesis operon (Harvard 2008's BBa_K098010 in pSB1C3).
BBa_K322123 : phycocyanobilin synthesis operon without terminator.
BBa_K322124 : Cph8 light sensing protein (UT Austin 2004's BBa_I15010 in pSB1C3).
BBa_K322125 : Cph8 with lacZ reporter system.
BBa_K322126 : Cph8 with EYFP reporter system.
BBa_K322127 : phycocyanobilin synthesis genes with cph8.
BBa_K322128 : phycocyanobilin synthesis genes with cph8 and EYFP reporter system.
Characterisation
Strains made by transformation with required DNA. Overnight cultures with cells of the required strains in 2.5ml LB + 40mg/ml Cml were grown in the dark at 37C with shaking. ONs were as follows:
JM109 – RLS.lacZ.YFP
envZ – RLS.lacZ.YFP
JM109 – YFP Control
envZ – YFP Control
100μl of ONs were used to inoculate two sterile LB + 40mg/ml Cml making up a total volume of 4ml in 5ml flasks which were then incubated in both light and dark, at 37C with shaking, as follows:
JM109 – RLS.lacZ.YFP (Light / Dark)
envZ – RLS.lacZ.YFP (Light / Dark)
JM109 – YFP Control (Light / Dark)
envZ – YFP Control (Light / Dark)
At 50 minute time intervals 200μl of each sample was taken and mixed with 800μl of sterile water in plastic cuvettes. The optical density of each sample was taken after the spectrophotometer was set with a sample of 200μl of sterile LB and 800μl water as a control, as was the luminescence, with readings for the background luminescence being taken for the sample of LB and water.
Two readings for each sample at each time interval were taken and then an average was calculated for each time interval for both optical density and luminescence.
For each sample at each time interval the average luminescence (normalised by the background luminescence) was divided by the optical density and plotted on a graph, shown below as Figure 3 .
Figure 3: Characterisation data for the red light sensor.
Further characterisation of the RLS was attempted using the Beta-Galactosidase Assay (A Better Miller) from OpenWetWare and preliminary data was recorded and calculated as shown in Figure 4 . The cultures grown were as follows:
JM109 – RLS.lacZ.YFP (Light / Dark)
envZ – RLS.lacZ.YFP (Light / Dark)
JM109 – lacZ Control (Light / Dark)
JM109 – RLS.lacZ (Light / Dark)
The conditions were Cml 40 and IPTG 90 in similar ONs to those used in the YFP characterisation.
YFP RLS Characterisation
Red light = less yellow colour fluorescence.
No Red light = more yellow colour fluorescence.
This assay shows the fluorescence per unit of estimated biomass for a number of different times of incubation. We can see that the controls for YFP in both the JM109 and envZ mutant strains show reasonable paired results in both the light and the dark. This means that the controls work as expected, however the large difference between the JM109 and the envZ control pairs implies that most of the result seen in JM109 is due to background envZ activity.
The rest of the results however are of limited usefulness beyond the testing of our assay procedure and the benchmark they represent of the RLS system which we can use to further improve our characterisation assays. The reasons for the affect seen are similar to those listed above in the lacZ RLS characterisation, listed below.
Figure 4: Further characterisation data for the red light sensor.
lacZ RLS Characterisation
Red light = less yellow colour ONPG
No Red light = more yellow colour ONPG
This characterisation data was acquired using a β-Galactosidase substrate o-nitrophenyl-β-D-galactoside (ONPG) which is cleaved to form a yellow dye that can be easily visualised and absorbance measurements can be taken. This can be used as a direct measure of lacZ activity as lacZ encodes for β-Galactosidase and when ONPG is in excess the production of the visible yellow dye can be measured over time to shown lacZ activation and control via promoter systems.
The absorbance at 420nm was divided by the time taken for the colour of o-nitrophenyl to become visible and was used to calculate a rate of expression.
This preliminary Miller-like assay shows limited useful characterisation date of the red light sensor (RLS) system, and is a good start towards accurate characterisation of the system, which we hope to continue now that a preliminary methodology has been found and tested.
The envZ – RLS.lacZ.YFP (light / dark) cultures did not produce Miller-like results, as was expected and have such not been included in the graphs, they were however run as a negative control to prove that any effects seen were not due to an artefact from the envZ mutant.
Though the data is not entirely useful we believe it may be due to a number of potential causes – such as the cofactor not being expressed properly, weak light intensity unable to activate the RLS system, or insufficient expression of the system as a whole - all of which we could test for given sufficient time, allowing us to improve our experimental technique to account for accordingly. We will endeavour to do so and bring further results to the Jamboree.
References
Levskaya, A., Chevalier, A. A., Tabor, J. J. & other authors (2005). Synthetic biology: Engineering Escherichia coli to see light. Nature 438, 441-442.
Gambetta, G. A. & Lagarias, J. C. (2001). Genetic engineering of phytochrome biosynthesis in bacteria. Proceedings of the National Academy of Sciences of the United States of America 98, 10566-10571.
Levskaya, A., O. D. Weiner, et al. (2009). Spatiotemporal control of cell signalling using a light-switchable protein interaction. Nature Vol 461
UT Austin 2004 team wiki (Registry coliroid page), http://partsregistry.org/Coliroid.
Harvard 2008 team wiki, https://2008.igem.org/Team:Harvard.
Throughout this wiki there are words in bold that indicate a relevance to human aspects. It will become obvious that human aspects are a part of almost everything in iGEM.
Retrieved from " http://2010.igem.org/Team:Edinburgh/Bacterial/Red_light_sensor "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
