<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Edinburgh_Notebook_Blue_light_sensor"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Edinburgh/Notebook/Blue_light_sensor">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Edinburgh/Notebook/Blue_light_sensor&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Edinburgh/Notebook/Blue_light_sensor&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Edinburgh/Notebook/Blue_light_sensor&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Edinburgh/Notebook/Blue_light_sensor" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Edinburgh/Notebook/Blue light sensor</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><DIV id="nav"><UL class="dropdown dropdown-horizontal" style="background: none;"><LI><A href="https://2010.igem.org/Team:Edinburgh" class="dir">home</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Team" class="dir">team - illuminati</A></LI><LI><A href="https://igem.org/Team.cgi?id=322">official</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Team/Supervisors">supervisors</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Team/Advisors">advisors</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Team/Students">students</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Team/Environment">environment</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Team/Gallery" class="dir">gallery</A></LI></UL><LI><A href="https://2010.igem.org/Team:Edinburgh/Project" class="dir">genomic BRIDGEs</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Project/Protocol">the protocol</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/BioBricks#Genomic">submitted parts</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Results#Genomic">results</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Project/Future">the future</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Project/References">references</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Bacterial" class="dir">bacterial BRIDGEs</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Bacterial/Core_repressilator">the project</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Bacterial/Red_light_producer">red light</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Bacterial/Red_light_sensor">red sensor</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Bacterial/Blue_light_producer">blue light</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Bacterial/Blue_light_sensor">blue sensor</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Bacterial/Green_light_producer">green light</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Bacterial/Green_light_sensor">green sensor</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/BioBricks#Bacterial">submitted parts</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Results#Bacterial">results</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Bacterial/Future">the future</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Bacterial/References">references</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Modelling" class="dir">modelling BRIDGEs</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Modelling/Kappa">kappa</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Modelling/Genomic">the genomic model</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Modelling/Bacterial">the bacterial model</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Modelling/Signalling">the signalling model</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Modelling/Tools">tools</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Results#Modelling">results</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Modelling/Future">the future</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Modelling/References">references</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Human" class="dir">human BRIDGEs</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Human/Communication">communication of science</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Human/Branding">iGEM survey</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Human/Conversations">conversations</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Human/Epic">the epic</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Human/FutureApps">future applications</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Results#Human">further thoughts</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Human/References">references</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Notebook" class="dir">lab notes   </A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Notebook/Collaboration">collaboration</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Notebook/Attribution">attribution</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Notebook/BRIDGE">BRIDGE</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Notebook/Red_light_producer">red light</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Notebook/Red_light_sensor">red sensor</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Notebook/Blue_light_producer">blue light</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Notebook/Blue_light_sensor">blue sensor</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Notebook/Green_light_producer">green light</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Notebook/Green_light_sensor">green sensor</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Notebook/Modelling">modelling</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Notebook/Safety">safety</A></LI><LI><A href="http://www.openwetware.org/wiki/French_Lab">protocols</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Acknowledgements" class="dir">acknowledgements</A></LI></DIV><DIV id="windowbox" style="border: .2em solid #660000; padding: 5px; position:fixed; top:50%; right:30px; width:8%;"><SPAN style="color:ivory;">These <B>notes</B> correspond to the part of the <B>project</B> detailed <A href="https://2010.igem.org/Team:Edinburgh/Bacterial/Blue_light_sensor">here</A>.</SPAN></DIV><H2><SPAN class="mw-headline" id="Blue_Light_Sensor">Blue Light Sensor</SPAN></H2><P><B>14/07/10</B></P><P>Received parts from Lausanne:
</P><UL><LI>BBa_191006 (113 ng/µL) – LovTAP alone
</LI><LI>BBa_191004 (166 ng/µL) – Readout 
</LI><LI>BBa_191003 (230 ng/µL) – whole system
</LI><LI>BBa_191009 (85 ng/µL) – mutant 
</LI><LI>Sosnick vector containing TrpR (393 ng/µL) – vector 
</LI></UL><P><B>16/07/10</B></P><P>Transformations using 100 µL of cells, 900 µL broth
</P><UL><LI>K191004 - amp
</LI><LI>K191003 - kanamycin
</LI><LI>K191009 – kanamycin
</LI><LI>Sosnick vector with TrpR – amp
</LI></UL><P>Note: Labeling on tubes is unclear – K191009 and K191004 got mixed up, so plating both tubes on both Kan and amp
</P><P><B>22/07/10</B></P><UL><LI>Minipreps: K0998010 – 5x (1 didn’t grow, but did anyway)
</LI><LI>Blue sensors (3 biobricks and Sosnick vector) – cultures set to grow overnight
</LI></UL><P><B>25/07/10</B>
Digest of the blue light sensor minipreps was performed with EcoRI and BamHI; samples:
</P><UL><LI>3- K19100<B>4</B>; 900ul, (3)
</LI><LI>4- K19100<B>4</B>; 100ul, (2)
</LI><LI>5-K19100<B>9</B>; 100ul, (1)
</LI><LI>6- K19100<B>9</B>; 100ul, (6)
</LI><LI>7- K19100<B>9</B>; 900ul, (4)
</LI><LI>8- K19100<B>9</B>; 100ul, (2)
</LI><LI>9- K19100<B>3</B>; 900ul, (2)
</LI><LI>10- K19100<B>3</B>; 900ul, (5)
</LI><LI>11- K19100<B>3</B>; 100ul, (4)
</LI><LI>12- K19100<B>3</B>; 100ul, (1)
</LI><LI>13- K19100<B>4</B>; 100ul, (3)
</LI><LI>14- K19100<B>4</B>; 900ul, (5)
</LI></UL><P>Lanes- 1- marker, 2-9 sampels as in order, 10-empty, 11, 12- marker instead of the samples- need to run last two again
</P><P>
LAST TWO LANES- NSAMPLES NO 13 AND 14 FROM THE ABOVE LIST
</P><P><B>28/07/10</B></P><P>Double transformants made using a novel cowboy procedure- double transformation with 4ul of each piece od DNA. Chances of succes- very very low... But at least it will be proved that it won't work!
Trans. 1: BBa_K191004+ K191003- read out + LovTap
Trans. 2: BBa_K191004+ K191009- read out + mutated LovTap
</P><P><B>30/7/10</B>
set up experiments to see if the blue light sensor will activate the response system under the daylight. Both liquid cultures and plates were set up, with red and white colonies- first supposed to turn white in the daylight and stay red in the dark (under the foil), second ones- no change of the colour. The same applies to the liquid cultures. However, the experiment failed- no proper growth was observed, and if there was a growth - it was very weak. No pigment observed, apart from the small points in the centre. 
Hypothesis- not stable plasmids-- cells look like they are not really resistant. Recorded growth:
</P><TABLE border="1"><TBODY><TR><TD>K191004+9 control
</TD><TD>K191004+9
</TD><TD>K191004+3 control
</TD><TD>K191004+3
</TD></TR><TR><TD>DARK
</TD><TD> ++ , white
</TD><TD> +++ , white
</TD><TD> ++  , white
</TD><TD> +++ , white with red points
</TD></TR><TR><TD> LIGHT
</TD><TD> + , white
</TD><TD> +++ , white with red point
</TD><TD> + -almost no growth
</TD><TD> ++ white
</TD></TR></TBODY></TABLE><P><B>2/8/10</B></P><P>Run the digest of the minipreps  from the white and red cultures of double transformants. No proper DNA band in the solution- repeat the procedure or not? Don't know what to do.... again. 
</P><P>Collect the thoughts and ideas and start again tomorrow!
</P><P><B>6/8/10</B></P><P>double transformants were prepared again, using non-cowboy procedure. 04 cells were first MADE COMPETENT, then transformed with 03 and 09, also 1 neg control was plated. Grow white on the intitial plate, then turned red if subcultured. 
</P><P><B>7-8/08</B></P><P>'Subbed these to duplicate fresh plates and incubated overnight at room temperature 
either under a fluorescent lamp or wrapped in foil - all white in both cases (and growth 
not great, probably because of low temperature)'. 
The plates were moved to 37C with the lamp /one under the light, one in the dark. so far -no results, checking in an hour/. 
</P><P>Another idea- check for stability of the insert- we don't know if the plasmids are compatibile. Streaking from original plates to check if single colonies will be a mixture of white and red or just red- origin? no plasmid maps from Lausanne. 
by Chairman--&gt;' If we can't find this information, I think our best course is to 
PCR out both inserts and combine them in a single vector, preferably pSB1C3.'
</P><P><B>9/8/2010</B></P><P>to do:
</P><UL><LI>primers for the parts to put them in a vector
</LI><LI>check the lamp every hour!!
</LI></UL><P>go thruough Lausanne website to find the plasmid maps
get back to Mexico about the parts and etc.
find out the conditions for the blue light sensor assay!
</P><P><B>10/08/10</B></P><UL><LI> assay with dark/light liquid cultures - fluorescence measurment and OD, after 5 h in the shaker under lamp- see video and pictures
</LI><LI>email to Mexico
</LI></UL><P><B>11/08/10</B></P><UL><LI>primer designed for combining LOVTap and Readout1 into 1 cell
</LI><LI>repeated assay with the smae samples, after overngiht in RT and %5h under lamp (for the light samples) and RT without shaking
</LI><LI>primary cultures streaked to amp/kan/IPTG plates- part 4+9 and 4+3 -three clones each
</LI></UL><P><B>12/08/10</B></P><UL><LI>results analysis after LOVTap experiment:check file, add graphs later.
</LI><LI>email from Mexico about Blue promoter
</LI></UL><P><B>16/08/10</B></P><UL><LI> streaked single clone for 9+4 and 3+4 (clones number 5 and 2, respectively)- for tomorrow's experiment
</LI><LI> minipreped mysterious double transformants and Sacb smples
</LI><LI> tried out the box on the shaker- unscrewed the things. 
</LI><LI> made the graphs from the second reading
</LI><LI> 03 and 09 LovTap taken for sequencing. 
</LI></UL><P>to do: decide on the illumination length and think about controls and how many samples do you want to put. check Lausanne website
t-test to check to importancy of the differences between the means.
</P><P><B>17/07/10</B></P><UL><LI> desinged experimental set up for the LovTap blue light response experiment /see next day/
</LI><LI> BAD NEWS: there is a suspicion of 1bp deletion in LovTap!!! Chairman noticed it when he was checking the parts base by base. Should have done it much much earlier probably to save the time... 
</LI><LI> BAD NEWS: lack of the proper thinking put in doubt all the results from previous two experiments (that is why you can't find any files of it attached)- there was no  IPTG added to the samples, therefore LOVTap expression was not induced- no point to do T-test right now... :( Well, at least I will learn from it and add it next time!
</LI></UL><P><B>18/08/10</B></P><UL><LI> tested LovTap under the blue light (blue LEDs, 470 nm wavelength, 2200 mcd/B, V=4.5 V; RS 466-3548).
</LI><LI> conditions:
</LI></UL><P>Liquid cultures (same as for minipreps) were grown overnight in 37C with  antibiotics. 
The next day, bottles with 3.5 ml of broth were inoculated with the 0.5 ml of appropriate cultures (0.5. ml of broth added as a control).
</P><UL><LI>sample designation: (# refers to the clone number form the original plate). Original plates are the ones made on 25/7 and have 2 clones on each plates (clones1-3 are K191003&amp;4 double transformants, clones 4-6 are K191009+4 double transformants). 
</LI></UL><OL><LI>	K191003+4 (#1) + Amp + Kan + IPTG + LIGHT		
</LI><LI>	K191003+4 (#1) + Amp + Kan IPTG + DARK		
</LI><LI>	K191003+4 (#2) + Amp + Kan IPTG + LIGHT		
</LI><LI>	K191003+4 (#2) + Amp + Kan IPTG + DARK		
</LI><LI>	K191009+4 (#4) + Amp + Kan IPTG +LIGHT		
</LI><LI>	K191009+4 (#4) + Amp + Kan IPTG +DARK		
</LI><LI>	K191009+4 (#5) + Amp + Kan IPTG +LIGHT		
</LI><LI>	K191009+4 (#5) + Amp + Kan IPTG +DARK		
</LI><LI>	K191004 ONLY #2+ Amp + LIGHT		
</LI><LI>	K191004 ONLY #2+ Amp + DARK		
</LI><LI>	neg. control- broth, Amp + Kan + IPTG + LIGHT
</LI></OL><UL><LI> samples designated DARK were wrapped in the aluminium foil straight after the inoculation; the cap was not wrapped (not see through anyway)so that samples could be easily taken without unwrapping the bottles from the foil. 
</LI><LI>All the samples were then taken to the shaker in the hot room and placed in prepared spots: for the dark- anywhere on an easily accessible holder, for light- in 2 rows of holders. 
</LI><LI>Light samples were covered with the cardboard box with blue LEDs attached on the sides, in a way to optimize light accession to the samples. Box was securely taped to the shaker. 
</LI><LI> Measurments were taken every half an hour: 200ul of the samples were placed in the numbered cuvette and then 800ul of water was added. Readings of fluroscence (green mode) were taken in standard fluorescence units (SFU, two readings each). Then optical density was measured (upstairs in the incubator room) at 360 nm wavelength. Control was used to calibrate the machine (set ref button). Also 2 readings were taken. Samples were shaken by inversion prior to taking measurment. 
</LI><LI> Experiment was stopped after 3 hours (6 sets of reading taken). 
</LI></UL><UL><LI><B>Result analysis</B></LI></UL><P>To generate the data for the graph:
</P><OL><LI> means were calculated from 2 readings of optical density and fluorescence
</LI><LI> (mean SFU minus background SFU) was divided by (mean OD minus background OD, if different from 0).
</LI><LI> table was made for the samples at different points of time- e.g. sample 1 after 30 min, 60 min, 90 min, 120 min etc. 
</LI><LI> graph was made with series named the same as the samples
</LI></OL><UL><LI>BAD BAD VERY BAD NEWS: after 2 hours of measurments the results of the sequencing were know and revealed that THERE IS A 1BP DELETION IN LOVTAP 03 AND 09 WHICH RESULTS IN FRAMESHIFT AND LACK OF THE FUNCTIONAL PRODUCT!!!! :(:(:( despite this information, it was decided that the experiment will be finished and analysed as initially planned.
</LI></UL><P><B>19/08/2010</B></P><UL><LI> Pcr- mutagenensis of LovTap- Mabel, adding missing bp. 
</LI><LI> Puryfing PCR product from the solution. 
</LI></UL><P><B>20/8/2010</B></P><UL><LI> PCR run on the gel -correct band of ~5.5. kb size, suggest that the PCR worked well. couple of other fainter bands 
</LI><LI> Self -ligation overnight (MABEL self ligation- 12ul of H2O, 4ul of DNA, 2 ul of Ligase buffer, 1 ul of T4 DNA ligase, 1ul of T4 polynucleotide kinase. 
</LI><LI> transformation and setting up the liquid cultures for the minipreps over the weekend. 
</LI></UL><P><B>23/08/2010</B></P><UL><LI> mini-prep DNA from the colonies after PCR- marker worn off in the shaker,so 3.2. and 9.1. got mixed up- probably sequencing necessary to distinguish the mutated version from non-mutated 
</LI><LI> digest minipreps with EcoRI, run the gel[photo]
</LI><LI> run 5ul of the ligated 03 and 09 on the gel /also Richards samples/.
</LI></UL><P><B>24/08/2010</B></P><UL><LI> repeated digest of mutated LovTap samples: 3.1, 3.?, 3.3., 3.4 9.?, 9.2 with EcoRi
</LI><LI> minipreped 6 more clones of mutated LovTap 03 and 09 from the same plate, labelled- 3.6, 3.7, 3.8, 9.4, 9.5, 9.6. Run the digest with EcoRI. Note: 3.7 sample is higly diluted as 80ul of EB was used instead of 40ul. oops. 
</LI><LI> delivered mutated 3.? and 9.2 minipreped clones for sequencing /only 2 reactions: each sample- forward/.
</LI></UL><P><B>25/9/2010</B></P><UL><LI> trpR mutant received form KEIO collection- generously privided by Lucas Black (The University of Edinburgh). 
</LI></UL><P><B>1/10/2010</B></P><UL><LI> LOVTap-RFP in JM109
</LI></UL><P><B>04/10/10</B>
We may have a working LovTap construct + red/white (RFP) reporter system thanks to Mexico.
</P><UL><LI> Cutting Mexico's LOVTap with EcoRi/PstI &amp; transfer into pSB1C3 plasmid (Part 1)
</LI><LI> Adding reporter- cut out with XbaI/PstI from K191004 part provided by Lausanne (PArt 2)
</LI><LI> Ligation of Part 1 and Part 2., selection for red colonies. 
</LI></UL><P>Minor characterisation experiment:
-LovTap constructed with red-white RFP reporter
-If activated, the blue light sensor transformants should produce red colonies
-Blue light sensor containing-colonies from transformant plate streaked onto two plates
-One plate left in the light, the other left in the dark
-If the one in the light grows red and the one in the dark doesn't, it is a good indication that the construct is working</P><P><B>5/10/2010</B></P><UL><LI> transforming trpR mutant with LOVTap- RFP plasmid. 
</LI></UL><P><B>8/10/2010</B></P><UL><LI> repetaed LOVTap charactierzation- protocol followed as on 18th of August 2010- with TrpR mutant and WT under light/dark conidtion. 
</LI><LI> no clear response was seen. 
</LI></UL><P><B>11/10/2010</B></P><UL><LI> Transforming plasmid Ptrp with RFP reporter to JM109 strian- control for further experiments.
</LI></UL><P><B>14/10/2010</B></P><UL><LI> PRi in trpR host- control plasmid
</LI></UL><P><B>22/10/2010</B></P><UL><LI> checking temperature response of LOVTap
</LI><LI> setting up samples with CMM minum medium, enriched in thymine, trace elements and glucose--&gt; for TrpR mutant.
</LI></UL><P>CELLS FAIL TO GROW!:(
</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:Edinburgh/Notebook/Blue_light_sensor">http://2010.igem.org/Team:Edinburgh/Notebook/Blue_light_sensor</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Edinburgh/Notebook/Blue_light_sensor" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Edinburgh/Notebook/Blue_light_sensor" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Edinburgh/Notebook/Blue_light_sensor&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Edinburgh/Notebook/Blue_light_sensor&amp;oldid=205026" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>