<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Edinburgh_Bacterial_Core_repressilator"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Edinburgh/Bacterial/Core_repressilator">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Edinburgh/Bacterial/Core_repressilator&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Edinburgh/Bacterial/Core_repressilator&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Edinburgh/Bacterial/Core_repressilator&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Edinburgh/Bacterial/Core_repressilator" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Edinburgh/Bacterial/Core repressilator</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><DIV id="nav"><UL class="dropdown dropdown-horizontal"><LI><A href="https://2010.igem.org/Team:Edinburgh" class="dir">home</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Team" class="dir">team - illuminati</A></LI><LI><A href="https://igem.org/Team.cgi?id=322">official</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Team/Supervisors">supervisors</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Team/Advisors">advisors</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Team/Students">students</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Team/Environment">environment</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Team/Gallery" class="dir">gallery</A></LI></UL><LI><A href="https://2010.igem.org/Team:Edinburgh/Project" class="dir">genomic BRIDGEs</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Project/Protocol">the protocol</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/BioBricks#Genomic">submitted parts</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Results#Genomic">results</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Project/Future">the future</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Project/References">references</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Bacterial" class="dir">bacterial BRIDGEs</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Bacterial/Core_repressilator">the project</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Bacterial/Red_light_producer">red light</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Bacterial/Red_light_sensor">red sensor</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Bacterial/Blue_light_producer">blue light</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Bacterial/Blue_light_sensor">blue sensor</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Bacterial/Green_light_producer">green light</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Bacterial/Green_light_sensor">green sensor</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/BioBricks#Bacterial">submitted parts</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Results#Bacterial">results</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Bacterial/Future">the future</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Bacterial/References">references</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Modelling" class="dir">modelling BRIDGEs</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Modelling/Kappa">kappa</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Modelling/Genomic">the genomic model</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Modelling/Bacterial">the bacterial model</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Modelling/Signalling">the signalling model</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Modelling/Tools">tools</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Results#Modelling">results</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Modelling/Future">the future</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Modelling/References">references</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Human" class="dir">human BRIDGEs</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Human/Communication">communication of science</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Human/Branding">iGEM survey</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Human/Conversations">conversations</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Human/Epic">the epic</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Human/FutureApps">future applications</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Results#Human">further thoughts</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Human/References">references</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Notebook" class="dir">lab notes   </A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Notebook/Collaboration">collaboration</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Notebook/Attribution">attribution</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Notebook/BRIDGE">BRIDGE</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Notebook/Red_light_producer">red light</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Notebook/Red_light_sensor">red sensor</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Notebook/Blue_light_producer">blue light</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Notebook/Blue_light_sensor">blue sensor</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Notebook/Green_light_producer">green light</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Notebook/Green_light_sensor">green sensor</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Notebook/Modelling">modelling</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Notebook/Safety">safety</A></LI><LI><A href="http://www.openwetware.org/wiki/French_Lab">protocols</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Acknowledgements" class="dir">acknowledgements</A></LI></DIV><DIV id="body" style="padding: 0px 60px 10px 60px; height: 1356px"><H2>Overview: The core repressilator</H2><P>In 2000, <A href="#References">Elowitz and Leibler</A><B>presented</B> the first synthetic oscillatory system in <I>E. coli</I>. Termed the repressilator, this combined three transcriptional repressor systems that were not naturally part of any oscillating system. This system oscillated but was not very precise (it had an oscillation period of 120 +/- 40min, and although mother cells transmitted their state to their daughters when dividing, the bacteria tended to get out of synchrony after time).</P><P>Over the next decade, a number of <B>efforts</B> were made to improve the precision of the system. <A href="#References">Garcia-Ojalvo et al. (2004)</A><B>presented</B> a model of an improved repressilator which used quorum sensing to make the bacteria function as a single unit; <A href="https://static.igem.org/mediawiki/2010/c/c4/Ed10-QuorumRepressilator.png">Figure 1</A> shows a diagram of this system. <A href="#References">Danino et al. (2009)</A><B>produced</B> an oscillating genetic circuit synchronised by quorum sensing, confirming that this method could be used to improve synthetic biological oscillators.</P><CENTER><P><B>Figure 1:</B> A quorum sensor synchronised repressilator.</P><P>Image: <A href="#References">Garcia-Ojalvo et al. (2004)</A></P></CENTER><P>The repressilator <B>designed</B> by <A href="#References">Danino et al.</A> in 2009 was not only advantageous in that it was more stable than the original Elowitz repressilator, but also in that the oscillation period could be <B>controlled</B>. The bacteria were grown in a trapping chamber, and when the flow rate through the main tube was increased or decreased, the quorum sensing molecules diffused away faster or slower, altering the oscillation period (see <A href="https://static.igem.org/mediawiki/2010/c/ca/Ed10-Flowrate.png">Figure 2</A>).</P><CENTER><P><B>Figure 2:</B> An oscillator in a flow chamber. By increasing or lowering the flow rate of water through the tube, the autoinducer (AI, which promotes transcription when bound to LuxR) diffuses more or less quickly out of the chamber, allowing control of the oscillation rate.</P><P>Image: <A href="#References">Danino et al. (2009)</A></P></CENTER><H2>Strategy: The core repressilator</H2><P>Our original <B>intentions</B> were to build upon this pre-existing work on the familiar and famous repressilator system, replacing the <B>unwieldy</B> quorum-sensing systems with a more <B>elegant</B> light-based solution. <B>Communication</B> via light would not only eliminate the need for chemical signalling, but also produce a highly <B>visible</B> and easily <B>recognisable</B> means of <B>viewing</B> and <B>verifying</B> the system in action. These dual <B>advantages</B> would hopefully serve as a proof of concept for a methodology that would have wide-ranging <B>benefits</B> throughout synthetic biology.</P><P>The overall oscillating <B>design</B> of our system would be composed of two parallel networks working in the same direction as an associated repressilator, with the addition of red, blue, and green light as a signalling mechanism. The inner network would function in the same manner as the repressilator designed by <A href="#References">Garcia-Ojalvo and Elowitz</A> in 2004, containing three genes in an ordered circular fashion (<I>lambda-cI</I>, <I>lacI</I>, <I>tetR</I>). The outer network, on the other hand, would consist of the three light sensors and their associated light producers.</P><P>The integrated network (<A href="https://static.igem.org/mediawiki/2010/5/58/1repressilatordiagram03.jpg">Figure 3</A>) could thus be <B>considered</B> in terms of its three separate components:</P><UL><LI> Component 1: red luciferase, LacI, red sensor. </LI><LI> Component 2: blue luciferase, TetR, blue sensor. </LI><LI> Component 3: green luciferase, lambda-cI, green sensor. </LI></UL></DIV><DIV id="body" style="padding: 0px 60px 10px 60px; height: 1356px"><CENTER><P><B>Figure 3:</B> A diagrammatic representation of the repressilator coupled to the light sensing and production pathways.</P></CENTER><P>Within each component, the light (the luciferase of the associated wavelength) and the repressor would be both either active or inactive at the same time. For example, in Component 1, high levels of lambda-cI would repress the activity of both the red luciferase and LacI; as soon as the levels of lambda-cI begin to fall, the concentration of red luciferase and LacI in the system would increase in tandem.</P><P>The expression of the light luciferase, which to the <B>observer</B> would produce light of a visible and distinguishable colour, would inhibit the following promoter through the action of the associated light sensor. The promoter thus inhibited would repress both the following light luciferase and the following promoter, similar to its functionality in the standard repressilator.</P><P>The light sensors would act in different ways according to the peculiarities of their associated pathways. The red light sensor would absorb red photons and repress the phosphorylation of OmpR, thus inhibiting the fusion between the OmpR-dependent <I>ompC</I> promoter and the <I>tetR</I> coding sequence. The green light sensor would work in a similar manner upon the PhoR pathway and the <I>lacI</I> coding sequence, whilst the blue light sensor would modify the α-helical domain linker of LovTAP as a conduit for an allosteric signal upon the <I>lambda-CI</I> coding sequence.</P><P>The <B>significance</B> of our design would be in the addition of the outer network (the light repressilator), thus theoretically introducing the light signal into the cell community and realising multicellular <B>communication</B>. Eventually, we hoped to be able to demonstrate <B>synchronous behaviour</B> across populations of bacteria that communicate via light to maintain the same state in <B>unison</B>.</P><H2>Strategy: Communication via light</H2><P>The response of light receptors Cph8, LovTAP and our green light receptor to different wavelengths of light have not been fully <B>characterised</B>. The absorbance spectra of the light sensitive proteins (<A href="https://static.igem.org/mediawiki/2010/a/a2/Ed10-Lightsensorspectra1.jpg">Figure 4</A>) might not exactly mirror their response to light, but should give us a good <B>idea</B> of their properties until this is possible.</P><P>In LovTAP, the light sensitive domain of the protein is the Lov2 domain from the <I>Avena sativa</I> blue light receptor phototropin. This binds flavin mononucleotide (FMN) which is its co-factor. <A href="#References">Schüttrigkeit et al. (2003)</A> measured the absorbance of the wildtype Lov2 bound to FMN. Since this is the active part of LovTAP, we <B>expect</B> our blue light receptor to have a similar response <I>in vivo</I>. The red light absorbing form of Cph1 is what responds to red light in Cph8. The absorbance of Cph1 was measured by <A href="#References">Gambetta and Lagarias (2001)</A>. Similarly, the absorbance of the green light absorbing form of CcaS, which we are planning on using in our green light receptor, was measured by <A href="#References">Hirose et al. (2008)</A>.</P><P>In accordance with the above, the <B>challenge</B> was then to design and characterise luciferases compatible with the sensors. In this, we decided to <B>build on</B> previous work by <A href="https://2007.igem.org/Ljubljana">Ljubljana 2007</A> and <A href="https://2009.igem.org/Team:Edinburgh">Edinburgh 2009</A>. The firefly luciferase from <I>Photinus pyralis</I> deposited as a BioBrick by the former formed the base of our red and green light producing proteins, while the bacterial luciferase LuxAB from <I>Xenorhabdus luminescens</I> BioBricked by last year's Edinburgh team was our blue light protein.</P><P>The emission peak of the wildtype firefly luciferase is roughly 557nm at pH 7.8 according to <A href="#References">Branchini et al. (2005)</A>. We needed to mutate this towards the red spectrum in order to activate the Cph8-based red light sensor, and also <B>proposed</B> to introduce a mutation towards the green spectrum so as to better match it with our hypothetical green light sensor. The LuxAB-LumP fusion already available in the Registry was theoretically already of the correct wavelength to activate the blue light sensor, although further characterisation was necessary. The theoretical spectra of the luciferases we sought to create are displayed in <A href="https://static.igem.org/mediawiki/2010/4/47/Ed10-emissiongraph.jpg">Figure 5</A>.</P></DIV><DIV id="body" style="padding: 0px 60px 10px 60px; height: 1356px"><CENTER><P><B>Figure 4:</B> Normalised absorbance spectra of:<B>a.</B> the Lov2 domain of <I>Avena sativa</I> with bound FMN. Adapted from <A href="#References">Schüttrigkeit et al. (2003)</A>.<B>b.</B> Green light absorbing form of cyanobacteriochrome CcaS from <I>Synechocystis</I> sp. PCC 6803. Adapted from <A href="#References">Hirose et al. (2008)</A>.<B>c.</B> Red light absorbing form of phytochrome Cph1-PCB adduct, from <I>Synechocystis</I> sp. PCC 6803. Adapted from <A href="#References">Gambetta and Lagarias (2001)</A>.
Note that the relative absorbance of each spectrum is normalised to one.</P></CENTER><CENTER><P><B>Figure 5:</B> Normalised emission spectra of:<B>a.</B> bacterial luciferase LuxAB from <I>V. campbellii</I>. Adapted from <A href="#References">Suadee et al. (2003)</A>.<B>b.</B> firefly luciferase from <I>P. pyralis</I>, wildtype. Adapted from <A href="#References">Shapiro et al. (2009)</A>.<B>c.</B> firefly luciferase from <I>P. pyralis</I>, substitution mutant S284T. Adapted from <A href="#References">Branchini et al. (2007)</A>.
Note that the relative emission of each spectrum is normalised to one.</P></CENTER><P>Difficulties were <B>foreseen</B> in making the various light producing proteins bright enough to activate their corresponding sensors, as well as in providing the substrates necessary for the bacteria to constantly emit light. However, once we received <B>word</B> that <A href="https://2010.igem.org/Team:Cambridge">this year's Cambridge team</A> was working to alleviate these problems, we decided to <B>focus</B> our efforts on the process of creating and characterising the various light producing and light sensing proteins.</P><P>During the course of the project, we <B>collaborated</B> closely with the <A href="https://2010.igem.org/Team:UNAM-Genomics_Mexico">UNAM-Genomics team</A> from Mexico, since it was apparent from an early stage that we were pursuing very closely related <B>ideas</B>.</P><H2>Problems</H2><P>The overall design was very <B>ambitious</B>, and we were thus unable to assemble the light controlled repressilator within the allocated time period, especially given the problems we <B>experienced</B> with the individual light based components. However we believe that our work makes <B>progress</B> towards it as well as other <A href="https://2010.igem.org/Team:Edinburgh/Bacterial/Future#Applications">applications</A> in which light signalling can be <B>useful</B>, such as <B>communication</B> between bacteria and computers.</P></DIV><DIV id="body2" style="padding: 0px 60px 10px 60px; height: 998px"><H2>BioBricks (Proposed)</H2><P>The BioBricks that would have been <B>used</B> in the proposed repressilator system would have been as follows:</P><P><A href="http://partsregistry.org/Part:BBa_B0034">BBa_B0034</A>: ribosome binding site based on the Elowitz repressilator.</P><P><A href="http://partsregistry.org/Part:BBa_R0010">BBa_R0010</A>: the LacI-sensitive inhibitory promoter region.</P><P><A href="http://partsregistry.org/Part:BBa_R0040">BBa_R0040</A>: the TetR-sensitive inhibitory promoter region.</P><P><A href="http://partsregistry.org/Part:BBa_R0051">BBa_R0051</A>: the lambda-CI-sensitive inhibitory promoter region.</P><P><A href="http://partsregistry.org/Part:BBa_C0012">BBa_C0012</A>: the coding region for the LacI protein.</P><P><A href="http://partsregistry.org/Part:BBa_C0040">BBa_C0040</A>: the coding region for the TetR protein.</P><P><A href="http://partsregistry.org/Part:BBa_C0051">BBa_C0051</A>: the coding region for the lambda-CI protein.</P><P><A href="http://partsregistry.org/Part:BBa_B0011">BBa_B0011</A>: transcriptional terminator.</P><P>In addition to the above, we would also have <B>integrated</B> the light sensing and light producing BioBrick parts <B>described</B> elsewhere in this section.</P><H2>References</H2><P><B>Elowitz, M. B. and S. Leibler (2000).</B> A synthetic oscillatory network of transcriptional regulators. <I>Nature</I><B>403</B>(6767): 335-338.</P><P><B>Danino, T., O. Mondragon-Palomino, et al (2009).</B> A synchronized quorum of genetic clocks. <I>Nature</I><B>463</B>(7279): 326-330.</P><P><B>Garcia-Ojalvo, J., M. B. Elowitz, et al. (2004).</B> Modeling a synthetic multicellular clock: Repressilators coupled by quorum sensing. <I>Proceedings of the National Academy of Sciences of the United States of America</I><B>101</B>(30): 10955-10960.</P><P><B>Branchini, B. R., Ablamsky, D. M., Murtiashaw, M. H., Uzasci, L., Fraga, H. &amp; Southworth, T. L. (2007).</B> Thermostable red and green light-producing firefly luciferase mutants for bioluminescent reporter applications. <I>Analytical Biochemistry</I><B>361</B>, 253-262.</P><P><B>Suadee, C., Nijvipakul, S., et al. (2008).</B> LuxG Is a Functioning Flavin Reductase for Bacterial Luminescence. <I>J. Bacteriol.</I><B>190</B>(5): 1531-1538</P><P><B>Shapirol, E., Lu, C., Baneyx, F. (2009).</B> Design and characterization of novel trypsin-resistant firefly luciferases by site-directed mutagenesis. <I>Protein Eng Des Sel</I><B>22</B>(11): 655-663.</P><P><B>Gambetta, G. A. &amp; Lagarias, J. C. (2001).</B> Genetic engineering of phytochrome biosynthesis in bacteria. <I>Proceedings of the National Academy of Sciences of the United States of America</I><B>98</B>, 10566-10571.</P><P><B>Schüttrigkeit, T. A., Kompa, C. K., Salomon, M., Rüdiger, W. &amp; Michel-Beyerle, M. E. (2003).</B> Primary photophysics of the FMN binding LOV2 domain of the plant blue light receptor phototropin of Avena sativa. <I>Chemical Physics</I><B>294</B>, 501-508.</P><P><B>Hirose, Y., Shimada, T., Narikawa, R., Katayama, M. &amp; Ikeuchi, M. (2008).</B> Cyanobacteriochrome CcaS is the green light receptor that induces the expression of phycobilisome linker protein. <I>Proceedings of the National Academy of Sciences</I><B>105</B>, 9528-9533.</P><P>Cambridge 2010 team wiki, <I>https://2010.igem.org/Team:Cambridge</I>.</P><P>UNAM-Genomics Mexico 2010 team wiki, <I>https://2010.igem.org/Team:UNAM-Genomics_Mexico</I>.</P></DIV><DIV id="windowbox" style="border: .2em solid #660000; padding: 5px; position:fixed; top:50%; right:30px; width:8%;"><SPAN style="color:ivory;">Throughout this wiki there are words in <B>bold</B> that indicate a relevance to <B>human aspects</B>. It will become obvious that <B>human aspects</B> are a part of almost everything in <B>iGEM</B>.</SPAN></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:Edinburgh/Bacterial/Core_repressilator">http://2010.igem.org/Team:Edinburgh/Bacterial/Core_repressilator</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Edinburgh/Bacterial/Core_repressilator" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Edinburgh/Bacterial/Core_repressilator" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Edinburgh/Bacterial/Core_repressilator&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Edinburgh/Bacterial/Core_repressilator&amp;oldid=205446" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>