"
Page
Discussion
View source
History
teams
Log in
 
Team:Edinburgh/Bacterial/Green light producer
From 2010.igem.org
home
team - illuminati
official
supervisors
advisors
students
environment
gallery
genomic BRIDGEs
the protocol
submitted parts
results
the future
references
bacterial BRIDGEs
the project
red light
red sensor
blue light
blue sensor
green light
green sensor
submitted parts
results
the future
references
modelling BRIDGEs
kappa
the genomic model
the bacterial model
the signalling model
tools
results
the future
references
human BRIDGEs
communication of science
iGEM survey
conversations
the epic
future applications
further thoughts
references
lab notes   
collaboration
attribution
BRIDGE
red light
red sensor
blue light
blue sensor
green light
green sensor
modelling
safety
protocols
acknowledgements
Overview: The green light producer
For the production of green light, we relied on tried and trusted firefly luciferase from P. pyralis. This has a recorded luminescence emission peak of roughly 557nm, as recorded in Shapiro et al. (2009) and other references ( Figure 1 ). This part was originally submitted by Ljubljana 2007. The sequence analysis revealed that there were 6 bases inserted upstream of the coding sequence. The distance between the ribosome binding site and the start codon was thus not optimal, and this resulted in a decreased luminescence intensity of the cells. We removed these 6 bases by PCR and resubmitted the part in pSB1C3.
Figure 1: Emission spectra of the P. pyralis luciferase (in solid black line).
Image: Shapiro et al. (2009)
An alternative approach was planned during the early stages of the project: the combination of LuxAB with another protein: YFP from Vibrio fischeri. This would shift the wavelength produced by LuxAB towards the green regions of the spectrum, to activate our green light sensor. During our discussions with UNAM-Genomics Mexico we realised that they planned on making the same part, so we stopped working on it, planning on testing its combination with our green light sensor later on. We unfortunately have not yet reached that stage, but the part is available in the registry as BBa_K360010 .
In addition to the above, we created a codon-optimised mutant of the firefly luciferase, reported by Fujii et al. (2007) to be 12.5 times brighter than the wildtype. This may help to alleviate the problems foreseen with a lack of luminescence intensity failing to activate the light sensors of the corresponding wavelength.
LUCIE (LUCiferase with Intense Emission) has three mutations compared to the wild type: Ile423Leu, Asp436Gly, and Leu530Arg. This causes more efficient binding of ATP and luciferin, which are used up faster and result in increased brightness of the cells.
Strategy
The wildtype version of the luciferase was cloned into pSB1C3. The 6 extra bases between the start codon and the start codon were removed by PCR, using a forward primer starting at the start codon, and an RBS based reverse primer. The part and the vector were thus amplified, but without the 6 extra bases.
The codon optimised LUCIE luciferase was synthesised by GENEART , then transferred to pSB1C3.
Problems
Unfortunately, GENEART is often overloaded with synthesis requests at this time of year. The bright luciferase arrived at the last moment, and the photos were taken the same day as the wiki freeze. We did not have time to characterise it further.
BioBricks
Firefly luciferase was originally deposited in the Registry by Ljubljana 2007 . We modified this slightly and re-submitted in pSB1C3, along with a codon-optimised mutant and a simple reporter system.
BBa_K322237 : firefly luciferase from Photinus pyralis, modified BBa_I712019 .
BBa_K322238 : firefly luciferase from Photinus pyralis under lac promoter.
BBa_K322451 : firefly luciferase from Photinus pyralis, codon optimised and mutated for brighter bioluminescence.
Characterisation
Figure 2: Results of spectrum analysis of our wildtype firefly luciferase.
Figure 2 shows the results of the spectral analysis of our wildtype firefly luciferase BBa_K322237 . Unlike the emission spectrum for our red-light producing S248T variant , the spectrum differs greatly from the expected; there is a small peak at roughly the correct location (557nm), but also a shoulder at approximately 580nm and a further peak at 600nm. This is due to the fact that the cells were suspended in citrate buffer, pH4.8, to help the luciferin enter the cells. At pH 7, the luciferase peaks in the yellow green region and does not make red light.
Figure 3 and Figure 4 on the next page demonstrate the difference in spectral output due to the pH of the buffer solution.
Figure 3: Emission output (green light) of the wildtype firefly luciferase in neutral buffer.
Figure 4: Emission output (yellow light) of the wildtype firefly luciferase in citrate buffer (pH 4.8).
References
Fujii, H., Noda, K., Asami, Y., Kuroda, A., Sakata, M. & Tokida, A. (2007). Increase in bioluminescence intensity of firefly luciferase using genetic modification. Analytical Biochemistry 366, 131-136.
Shapirol, E., Lu, C., Baneyx, F. (2009). Design and characterization of novel trypsin-resistant firefly luciferases by site-directed mutagenesis. Protein Eng Des Sel 22(11): 655-663.
Ljubljana 2007 team wiki, https://2007.igem.org/Ljubljana.
Throughout this wiki there are words in bold that indicate a relevance to human aspects. It will become obvious that human aspects are a part of almost everything in iGEM.
Retrieved from " http://2010.igem.org/Team:Edinburgh/Bacterial/Green_light_producer "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
