<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Edinburgh_Notebook_Red_light_producer"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Edinburgh/Notebook/Red_light_producer">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Edinburgh/Notebook/Red_light_producer&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Edinburgh/Notebook/Red_light_producer&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Edinburgh/Notebook/Red_light_producer&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Edinburgh/Notebook/Red_light_producer" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Edinburgh/Notebook/Red light producer</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><DIV id="nav"><UL class="dropdown dropdown-horizontal" style="background: none;"><LI><A href="https://2010.igem.org/Team:Edinburgh" class="dir">home</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Team" class="dir">team - illuminati</A></LI><LI><A href="https://igem.org/Team.cgi?id=322">official</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Team/Supervisors">supervisors</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Team/Advisors">advisors</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Team/Students">students</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Team/Environment">environment</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Team/Gallery" class="dir">gallery</A></LI></UL><LI><A href="https://2010.igem.org/Team:Edinburgh/Project" class="dir">genomic BRIDGEs</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Project/Protocol">the protocol</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/BioBricks#Genomic">submitted parts</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Results#Genomic">results</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Project/Future">the future</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Project/References">references</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Bacterial" class="dir">bacterial BRIDGEs</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Bacterial/Core_repressilator">the project</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Bacterial/Red_light_producer">red light</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Bacterial/Red_light_sensor">red sensor</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Bacterial/Blue_light_producer">blue light</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Bacterial/Blue_light_sensor">blue sensor</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Bacterial/Green_light_producer">green light</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Bacterial/Green_light_sensor">green sensor</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/BioBricks#Bacterial">submitted parts</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Results#Bacterial">results</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Bacterial/Future">the future</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Bacterial/References">references</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Modelling" class="dir">modelling BRIDGEs</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Modelling/Kappa">kappa</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Modelling/Genomic">the genomic model</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Modelling/Bacterial">the bacterial model</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Modelling/Signalling">the signalling model</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Modelling/Tools">tools</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Results#Modelling">results</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Modelling/Future">the future</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Modelling/References">references</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Human" class="dir">human BRIDGEs</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Human/Communication">communication of science</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Human/Branding">iGEM survey</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Human/Conversations">conversations</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Human/Epic">the epic</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Human/FutureApps">future applications</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Results#Human">further thoughts</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Human/References">references</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Notebook" class="dir">lab notes   </A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Notebook/Collaboration">collaboration</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Notebook/Attribution">attribution</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Notebook/BRIDGE">BRIDGE</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Notebook/Red_light_producer">red light</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Notebook/Red_light_sensor">red sensor</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Notebook/Blue_light_producer">blue light</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Notebook/Blue_light_sensor">blue sensor</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Notebook/Green_light_producer">green light</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Notebook/Green_light_sensor">green sensor</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Notebook/Modelling">modelling</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Notebook/Safety">safety</A></LI><LI><A href="http://www.openwetware.org/wiki/French_Lab">protocols</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Acknowledgements" class="dir">acknowledgements</A></LI></DIV><DIV id="windowbox" style="border: .2em solid #660000; padding: 5px; position:fixed; top:50%; right:30px; width:8%;"><SPAN style="color:ivory;">These <B>notes</B> correspond to the part of the <B>project</B> detailed <A href="https://2010.igem.org/Team:Edinburgh/Bacterial/Red_light_producer">here</A>.</SPAN></DIV><H2><SPAN class="mw-headline" id="Red_Light_Producer"> Red Light Producer </SPAN></H2><P><B>29/6/10</B></P><P>Now preparing primers to make a set of different spectra bioluminescent proteins. Need to order primers for Site directed mutagenesis of the Firely luciferase to make it red (MABEL protocol), and to make click beetle luciferase Rluc8 (which emits light at about 420nm in presence of bisdeoxycoelenterazine - about the same price a coelenterazine though)
</P><P><B>8/7/2010</B></P><UL><LI> Red luciferase mutagenesis primers arrived- 3 sets:
</LI><LI> S248T
</LI><LI> 356 K
</LI><LI> 356 R
</LI></UL><UL><LI> Followed the protocol with 34ul water + 0.5 template DNA
</LI><LI> Put the primers and template in green box <B>(???)</B></LI><LI>Ran Gel: 5 µL PCR product, 5 µL water, 2.5 µL buffer
</LI></UL><TABLE border="1"><TBODY><TR><TD> Marker
</TD><TD>S284T
</TD><TD> 356K
</TD><TD> 356R
</TD></TR></TBODY></TABLE><P><A href="/File:07-08.JPG" title="File:07-08.JPG">File:07-08.JPG</A></P><UL><LI>356K and 356R look fine
</LI><LI>going to run S284T at a higher ** temp tomorrow to see if it works better
</LI><LI>all PCR tubes stored in freezer
</LI></UL><P><B>09/7/10</B></P><P>Redo PCR of Red Luciferase
</P><UL><LI>Everything except template DNA + Kod polymerase kept on ice
</LI><LI>Ran PCR
</LI><LI>Template DNA used PyeaR  and P. pyralis luciferase BBa_K216015
</LI><LI>Purification done for 356K and 356R; stored in iGEM box (-20C).
</LI></UL><P><A href="/File:07-09straight.JPG" title="File:07-09straight.JPG">File:07-09straight.JPG</A></P><P><B>9/7/2010</B></P><UL><LI>Ligations set up overnight- might have put too much ligase in 356K... Incubated at 15C.
</LI><LI>Oure 356R and 356K are in the freezer- row 10 (G &amp;H)?
</LI></UL><P><B>13/07/10</B>
Goal of the day: 
</P><P>Transformations:
</P><UL><LI>356R (2): one from 16 °C and one from room temp(RT)
</LI><LI>356K (2): one from 16 °C and one from room temp(RT)
</LI><LI>S284T (2): one with 25/5 ligase, one with 1/7 ligase
</LI></UL><UL><LI> I had a burger for the lunch. It was very nice.
</LI><LI> Now we are transforming cells:
</LI></UL><PRE>** 356R (2)&amp; 356K (2): one form 16C, one from RT
** S248T (2): 1 with 25/5 ligase, other with 1.7.10 ligase/
</PRE><P>Empty edinbrick vector for control. 
</P><UL><LI> LuxAB + edinbrick ligation.
</LI></UL><UL><LI> Using 100 ul transformed cells + 800 ul LB for all. Incubation started half 5. 
</LI></UL><P><B>22/07/10</B></P><P>Liquid Cultures (amp 80):
</P><UL><LI>356R (RT) – 1 with NaNO3 and 1 without
</LI><LI>(16°C) – 2 with NaNO3 and 2 without
</LI></UL><LI>356K (RT) – 1 with NaNO3 and 1 without
</LI><LI>(16°C) – 2 with NaNO3 and 2 without 
</LI><LI>S284T (100 µL and 25/5 ligase) – 2 with NaNO3 and 2 without
</LI><LI>(900 µL and 25/5 ligase) – 3 with NaNO3 and 3 without
</LI><LI>(900 µL and 1/7 ligase) – 5 with NaNO3 and 5 without
</LI><P>NaNO3 = 30mM Sodium Nitrate
</P><P><B>23/07/10</B></P><P>Double digest of K098010
</P><P><B>28/07/10</B></P><P>Primer design to produce a stable red light luciferase. The mutations for increasing bioluminescence come from <A href="http://www.sciencedirect.com/science?_ob=ArticleURL&amp;_udi=B6W9V-4NHM57C-1&amp;_user=809099&amp;_coverDate=07%2F15%2F2007&amp;_rdoc=1&amp;_fmt=high&amp;_orig=search&amp;_sort=d&amp;_docanchor=&amp;view=c&amp;_searchStrId=1414502247&amp;_rerunOrigin=google&amp;_acct=C000043939&amp;_version=1&amp;_urlVersion=0&amp;_userid=809099&amp;md5=00d9c92f6452bd276c6ea606ed7d3bbf" class="external text" rel="nofollow">Fujii et al, 2007</A> and the paper from <A href="http://www.sciencedirect.com/science?_ob=ArticleURL&amp;_udi=B6W9V-4X6MSYB-7&amp;_user=809099&amp;_coverDate=01%2F15%2F2010&amp;_rdoc=1&amp;_fmt=high&amp;_orig=search&amp;_sort=d&amp;_docanchor=&amp;view=c&amp;_searchStrId=1414503636&amp;_rerunOrigin=google&amp;_acct=C000043939&amp;_version=1&amp;_urlVersion=0&amp;_userid=809099&amp;md5=297be52832193b4d026953632a1edf3c" class="external text" rel="nofollow">Branchini et al, 2010</A></P><P><B>29/7/2010</B></P><UL><LI>Double digest with EcoRI HF&amp;PstI in buffer 2 (neb).
</LI><LI> 2 bands: with and without vector?
</LI><LI> did not leave the digest for too long... 
</LI></UL><TABLE border="1"><TBODY><TR><TD> Lane 1
</TD><TD> Lane 2
</TD><TD> Lane 3
</TD><TD> Lane 4
</TD><TD> Lane 5
</TD><TD> Lane 6
</TD></TR><TR><TD> Ladder
</TD><TD> S284T (4)
</TD><TD> S284T (6)
</TD><TD> 356R (6, 900)
</TD><TD> 356R (6)
</TD><TD> 356K (4)
</TD></TR></TBODY></TABLE><P><B>2/8/2010</B></P><UL><LI> Red luciferases EcoRI HF&amp;PstI double digests- lanes 2-7 in the second miniprep set. 
</LI><LI>Ladder didn't work too well ,though...
</LI></UL><UL><LI> Transform last years luciferase
</LI><LI> Look at papers for expression in E. coli
</LI><LI> Replace B0034 by stronger RBS?
</LI><LI> Confirm that we order bright luc.
</LI><LI> Single digest of constructs
</LI><LI> Test glowing with positive control
</LI></UL><P><B>9/8/2010</B></P><P>Digest: 
</P><UL><LI>WT1
</LI><LI>356 R: (2)
</LI><LI> 356 K: (4)
</LI><LI> S284T: (6)
</LI><LI> 3 eith EcoRI; 3 with EcoRi + PstI
</LI></UL><P><B>10/8/2010</B></P><UL><LI> Sequencing of red mutant luciferases. 
</LI><LI> Forward primers for S284T and reverse for 356R and K should do. 
</LI><LI> Concentration of primers for sequencing: 5uM.
</LI><LI> sequencing tubE: 6 ul liquid: 3ul DNA + 2ul water, 1 ul primer mix.
</LI><LI> when using miniprep DNA- remember to spin down the dirt- 1min, max speed, then take the DNA from the top to avoid disturbing any dirt that is settled. 
</LI></UL><P><B>12/8/2010</B></P><UL><LI> Got the sequences for red luciferases
</LI><LI>see: if we can detect Red Light with the luminometer?
</LI><LI> Red Luciferases 356R and S284T were nNOT MUTANTS- just WT
</LI><LI> set up ;iquid cultures with 30 ul nitrate for: 
</LI><LI> 356R - no 3,1 from RT plate, no 5 from 16C plate; no 1 from Master plate
</LI><LI> S284T - 5 from masterplate; 2, 4, 6 from masterplate
</LI><LI> 356K: 4 from masterplate
</LI></UL><P><B>13/8/2010</B>
Big day today... TESTING YESTERDAYS PLAN- LUMINESCENCE
</P><P><B>16/8/2010</B></P><UL><LI> Sequecning of: 
</LI><LI> WR1- 356R -use S284T forward primer
</LI><LI> WR2 -S284T -use 356 R reverse primer
</LI><LI> WR3- LovTap 03
</LI><LI>WR4 -LovTap 09
</LI></UL><UL><LI> RESULTS: ONLY 356K WAS A REAL MUTANT; S284T AND 356R TURNED OUT TO BE STILL A WT
</LI></UL><P><B>19/8/2010</B></P><UL><LI> star6: 56,869
</LI><LI> star4: 13,471
</LI><LI> 356K: 263,778
</LI><LI> star 2: 3,717
</LI><LI> WT: 1,617832
</LI></UL><UL><LI>Gel of the ligations: marker (lane 1), 356K (lane 2,3), S284T (lane 3,4), 358Rs (lane 5,6).
</LI></UL><P><B>25/8/2010</B></P><UL><LI> Next step:- redo ligations for 356R; redo PCR for S284T
</LI><LI> Ligations
</LI><LI>Did MABEL PCR PPyInc + S284T primers
</LI><LI>used 0.5 ul template DNA
</LI><LI>Gel: marker (Lane1), ligation of PCR product for 356R (Lane2), Ligation of 07 PCR for S284T (lane3), Pure PCR from S284T. 
</LI></UL><P><B>26/8/2010</B></P><UL><LI> check the Fuji et. al paper, see what sequence they used to start off with. 
</LI><LI> check IAV in other luciferase
</LI><LI> Fix scar before RBS and in teh second stop codon. 
</LI><LI> compared to WT, luciferase sequence has: 
</LI><LI> N 50 P
</LI><LI> N 99 G
</LI><LI> SKL- IAV
</LI></UL><P><B>27/8/2010</B></P><UL><LI> Ligations from yesterday out of freezer.
</LI><LI> Running on gel
</LI><LI> Column 3 9with 356R) has less DNA in it- not enough from ligation and messed up loading of gel). 
</LI></UL><TABLE border="1"><TBODY><TR><TD> Lane 1
</TD><TD> Lane 2
</TD><TD> Lane 3
</TD><TD> Lane 4
</TD></TR><TR><TD> Ladder
</TD><TD> New S284T PCR - new ligase and the buffer
</TD><TD> 356 R ligation
</TD><TD> S284T PCR + old ligase
</TD></TR></TBODY></TABLE><P><B>30/8/2010</B></P><UL><LI> Primers have arrived for fixing the luciferase- removing 6 bases
</LI><LI> Primers have arrived for up and downstream sequences of TnaA region.
</LI><LI> Send Chairman sequence for T7 promoter
</LI><LI> transformed 356R and S284T ligations from Friday
</LI><LI> Used PCR product as a control
</LI><LI> Plated out the transformed cells. Used 900 rather than 100 if possible
</LI></UL><P><B>31/8/2010</B></P><UL><LI> We have S284T transformants but 
</LI><LI> 14 colonies on ligation plate
</LI><LI> 4 colonies on control where we used PCR product instead of ligation
</LI></UL><UL><LI> For sequencing- 
</LI><LI> if sequencing a S284T mutant, use 356R/K reverse primer
</LI><LI> if sequencing 356R mutant, use S284T forward primer.
</LI></UL><P><B>2/9/2010</B></P><UL><LI> minipreped 14 colonies (1-14)
</LI><LI> run the gel with samples digested with EcoRI
</LI><LI> here: samples 1-7
</LI></UL><P><B>3/9/2010</B></P><UL><LI> S284T 8-14 (from yesterday)
</LI></UL><P><B>8/9/2010</B></P><UL><LI> wWT of luciferase (left)
</LI><LI> mutation of S284T (right)
</LI></UL><P><B>9/9/2010</B></P><TABLE border="1"><TBODY><TR><TD> Lane 1
</TD><TD> Lane 2
</TD><TD> Lane 3
</TD><TD> Lane 4
</TD><TD> Lane 5
</TD></TR><TR><TD> Ladder
</TD><TD> WT
</TD><TD> S284T 1
</TD><TD> luxCDE 9:4
</TD><TD> LuxCDE 9:1
</TD></TR></TBODY></TABLE><P><B>20/9/2010</B></P><UL><LI>send for sequencing: 356K, S284T, WT (samples from 16/9: WT- 900, 2; also 16/9 S284T- 900, 4.)
</LI><LI> all in psB1C3 vector. 
</LI><LI> Transform 356K and S284T with RBS and promoter (lacZ, LacI, RBS). 
</LI><LI> Run gel of 365K in psb1C3 of the digest.  --&gt; control: RFP + psb1C3
</LI><LI> then add promoter
</LI></UL><P><B>23/9/2010</B></P><UL><LI> Added in incubator room liquid culture  <B>?)</B> LacP + RBS + Luciferase for S284T, 356K and WT. 
</LI><LI> miniprep- 6 bottles
</LI><LI> GLOWING
</LI></UL><UL><LI> S248T- 
</LI><LI>9.2; 5 
</LI><LI> 9.4; 1 
</LI><LI>5; 4
</LI></UL><LI> 356
</LI><LI> 3.3
</LI><LI> WT
</LI><LI> 3.2; 4 
</LI><LI> 1.4, 5
</LI><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:Edinburgh/Notebook/Red_light_producer">http://2010.igem.org/Team:Edinburgh/Notebook/Red_light_producer</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Edinburgh/Notebook/Red_light_producer" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Edinburgh/Notebook/Red_light_producer" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Edinburgh/Notebook/Red_light_producer&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Edinburgh/Notebook/Red_light_producer&amp;oldid=205002" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>