<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Edinburgh_Notebook_Red_light_sensor"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Edinburgh/Notebook/Red_light_sensor">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Edinburgh/Notebook/Red_light_sensor&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Edinburgh/Notebook/Red_light_sensor&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Edinburgh/Notebook/Red_light_sensor&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Edinburgh/Notebook/Red_light_sensor" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Edinburgh/Notebook/Red light sensor</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><DIV id="nav"><UL class="dropdown dropdown-horizontal" style="background: none;"><LI><A href="https://2010.igem.org/Team:Edinburgh" class="dir">home</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Team" class="dir">team - illuminati</A></LI><LI><A href="https://igem.org/Team.cgi?id=322">official</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Team/Supervisors">supervisors</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Team/Advisors">advisors</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Team/Students">students</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Team/Environment">environment</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Team/Gallery" class="dir">gallery</A></LI></UL><LI><A href="https://2010.igem.org/Team:Edinburgh/Project" class="dir">genomic BRIDGEs</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Project/Protocol">the protocol</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/BioBricks#Genomic">submitted parts</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Results#Genomic">results</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Project/Future">the future</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Project/References">references</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Bacterial" class="dir">bacterial BRIDGEs</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Bacterial/Core_repressilator">the project</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Bacterial/Red_light_producer">red light</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Bacterial/Red_light_sensor">red sensor</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Bacterial/Blue_light_producer">blue light</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Bacterial/Blue_light_sensor">blue sensor</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Bacterial/Green_light_producer">green light</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Bacterial/Green_light_sensor">green sensor</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/BioBricks#Bacterial">submitted parts</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Results#Bacterial">results</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Bacterial/Future">the future</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Bacterial/References">references</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Modelling" class="dir">modelling BRIDGEs</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Modelling/Kappa">kappa</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Modelling/Genomic">the genomic model</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Modelling/Bacterial">the bacterial model</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Modelling/Signalling">the signalling model</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Modelling/Tools">tools</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Results#Modelling">results</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Modelling/Future">the future</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Modelling/References">references</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Human" class="dir">human BRIDGEs</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Human/Communication">communication of science</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Human/Branding">iGEM survey</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Human/Conversations">conversations</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Human/Epic">the epic</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Human/FutureApps">future applications</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Results#Human">further thoughts</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Human/References">references</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Notebook" class="dir">lab notes   </A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Notebook/Collaboration">collaboration</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Notebook/Attribution">attribution</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Notebook/BRIDGE">BRIDGE</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Notebook/Red_light_producer">red light</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Notebook/Red_light_sensor">red sensor</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Notebook/Blue_light_producer">blue light</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Notebook/Blue_light_sensor">blue sensor</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Notebook/Green_light_producer">green light</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Notebook/Green_light_sensor">green sensor</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Notebook/Modelling">modelling</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Notebook/Safety">safety</A></LI><LI><A href="http://www.openwetware.org/wiki/French_Lab">protocols</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Acknowledgements" class="dir">acknowledgements</A></LI></DIV><DIV id="windowbox" style="border: .2em solid #660000; padding: 5px; position:fixed; top:50%; right:30px; width:8%;"><SPAN style="color:ivory;">These <B>notes</B> correspond to the part of the <B>project</B> detailed <A href="https://2010.igem.org/Team:Edinburgh/Bacterial/Red_light_sensor">here</A>.</SPAN></DIV><H2><SPAN class="mw-headline" id="Red_Light_Sensor"> Red Light Sensor </SPAN></H2><P><B>22/6/10</B></P><P>Transformation of BBa_M30109; well 14 H, distribution plate 3 (2010)--&gt; red light sensor 
</P><P><B>24/6/10</B></P><P>Subcultured transformants from ampicillin plates to new plates
</P><P><B>25/6/10</B></P><P>1) DNA mini-preps were prepared from the transformants
2) DNA mini-preps were cut with EcoRI enzyme to check for the presence of the plasmid (and DNA); agarose gel was prepared for DNA electrophoresis after cutting.
3) Results from cutting showed no presence of the sensor.
Troubleshooting: was the plate number correct? Was procedure performed correct?
by:Maria, Will, Marta 
</P><P><B>28/6/10</B></P><P>Tried to determine what was wrong with the red sensor that failed to appear in the gel. We definitely had the correct plate and well. We checked the seqeunce of the part and found that the sequence given does not match any given in BLAST. We suspect either that part of the plasmid was added to the distribution/registry or that the plasmid contains an illegal EcoR1 site, but are re-transforming cells just in case there was an experimental error.
Due to failure of the previous transformation competent cells were retransformed with the two plasmids, as before. Two Chloramphenicol spread plates for each plasmid were made one  of each pair containing 100micro-l of cells the other containing the 900micro-l remainder spun down and resuspended in 100micro-l of sterile LB.
by: Richard, Lu
</P><P><B>29/6/10</B></P><P>Streaking the colonies from the chloramphenicol plates on the new agar + chloramphenicol plates.
</P><P><B>30/6/10</B></P><P>No growth on plates.
</P><P><B>5/7/10</B></P><P>Made kanomycin 32mg/ml, IPTG 90mg/ml plates. Transformed cells with individual red light sensor parts (BBa_I15008 - 2010 distribution Plate 2 Well 13J; BBa-I15009 - distribution plate 1 Well 20F; BBa_I15010 - distribution Plate 3 Well 20D; &amp; K098010 - distribution Plate 3 11N). Plated cells onto kanomycin plates and left to incubate overnight. 
by: Hannah and Richard
</P><P><B>6/7/10</B></P><P>Richard/Hannah - made subcultures of red light sensor part transformants.
</P><UL><LI>Agarose Gel Eleckrophoresis of composite red sensor M30109
</LI><LI>The bands are not the right size
</LI><LI>Sequencing done by the registry starts in the middle of the sequence (for forward sequence)
</LI><LI>So it seems the plasmid doesn’t contain the right sequence
</LI><LI>We will try to assemble it from individual parts
</LI></UL><P><B>8/7/10</B></P><P>Plasmid minipreps of I15009 (the only red sensor part transformant that grew in liquid culture)
by: Richard and Hannah
</P><P><B>9/7/10</B></P><P>Gel run with I15009. 
</P><TABLE border="1"><TBODY><TR><TD> Lane 1
</TD><TD> Lane 2
</TD><TD> Lane 3
</TD><TD> Lane 4
</TD><TD> Lane 5
</TD><TD> Lane 6
</TD><TD> Lane 7
</TD><TD> Lane 8
</TD></TR><TR><TD> Ladder
</TD><TD> Pure SacB
</TD><TD> Pure SacB &amp; EcoRI
</TD><TD> sacB &amp; catR PCR
</TD><TD> I15009 digest
</TD><TD> S248T PCR Product
</TD><TD> Purified 356K
</TD><TD> Purified 356R
</TD></TR></TBODY></TABLE><UL><LI>Insert Picture of Gel*
</LI></UL><P><B>13/7/2010</B>
NOTHING WORKS. No idea why...
</P><P><B>16/7/10</B></P><P>Because the red light sensing parts were transformed with 50 µL or water instead of 15 µL during the first transformation, another attempted was tried.
Transforming:
</P><UL><LI>I15008 (all red light sensor parts)
</LI><LI>I15009
</LI><LI>I15010
</LI></UL><P>used 200 µL of cells, 800 µL of broth. Kan resistance.
</P><P><B>23/07/10</B></P><P>Double digest of K098010. Ran gel. Lane was empty:
</P><UL><LI>Not going to bother saying what is in each lane. 
</LI></UL><P><B>30/7/10</B></P><P>Primers were ordered in order to PCR out the Red Light Sensor Parts from the  <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_M30109" class="external text" rel="nofollow">Bio Brick BBa_M30109</A></P><P>Forward Primer for PBad promoter <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_I13453" class="external text" rel="nofollow">Bio Brick BBa_I13453</A>
Reverse Primer for Heme Oxygenase (ho1) <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_I15008" class="external text" rel="nofollow">Bio Brick BBa_I15008</A>
Reverse Primer for Phycocyanobilin:Ferredoxin Oxidoreductase (PcyA) <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_I15009" class="external text" rel="nofollow">Bio Brick BBa_I15009</A>
Forward Primer for TetR repressible promoter <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_R0040" class="external text" rel="nofollow">Bio Brick BBa_R0040</A>
Reverse Primer for cph8 (Cph1/EnvZ fusion)Chimeric Cph1 light receptor/EnvZ protein <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_I15010" class="external text" rel="nofollow">Bio Brick BBa_I15010</A></P><P><B>3/8/10</B></P><P>A PCR was run with the above primers in 3 distinct mixtures in an attempt to amplify out the Red Light Sensor parts using Bio Brick BBa_M30109 as a template.
PCR 1 Contained The PBad promoter Forward Primer &amp; the PcyA Reverse Primer
PCR 2 Contained The TetR repressible promoter Forward Primer &amp; the cph8 Reverse Primer 
PCR 3 Contained The PBad promoter Forward Primer &amp; the cph8 Reverse Primer
</P><P>And a gel was run of the PCR products: Marker(Lane 1&amp;6), only visible product- suggested PCR I15010 with RBS &amp; promoter (Lane 3). 
</P><P>Due to the successful amplification of the I15010 cph8 Red light sensor part, the DNA was ligated into plasmid psb1C3.
</P><P><B>4/8/10</B></P><P>Competent cells were transformed with the ligated psb1C3 plasmid containing I15010 and plated out on Cml 40 + IPTG 90 plates and incubated O/N at 37 degrees Celsius.
</P><P>A PCR was run using the following primers:
PCR 1 Contained The BBa_B0034 RBS Forward Primer &amp; the ho1 Reverse Primer
PCR 2 Contained The BBa_B0034 RBS Forward Primer &amp; the PcyA Reverse Primer
PCR 3 Contained The PBad promoter Forward Primer &amp; the ho1 Reverse Primer
PCR 4 Contained The PBad promoter Forward Primer &amp; the PcyA Reverse Primer</P><P>And a gel was run of the PCR products.
</P><UL><LI>Insert picture of Gel*
</LI></UL><P>No successful amplification was seen.
</P><P><B>11/8/10</B></P><P>Update from Maria. 
The transformations and PCR amplification of the HO1-pcyA fusion construct have nearly all failed. 
This weekend the kanamycin resistant version of the construct (K089010 in pSB3K3) was transformed and cultures grew.
We already have a PCR product of the fused cph8-envZ sensing part. We now need to put the newly transformed K089010 in front of the sensor in order to rebuild the original sensor, which is well and truly fucked.
</P><P><B>16/8/10</B></P><P>Successful PCR of K098010 using primers for EcoRI &amp; I15009 reverse primer.
</P><P><B>17/8/10</B></P><UL><LI> The PCR product of the Harvard K098010 part was purified and then digested with both XbaI &amp; SpeI as was the PCR product of cph8.
</LI><LI> These digestions were then ligated together ON using in two orientations.
</LI></UL><P><B>18 &amp;19/8/2010</B></P><UL><LI> Fusion PCR C + H with R004081 &amp; I15009-1 
</LI><LI> also running H + C same primers as negative control
</LI><LI> run gel of PCR prod &amp; 5 ul of ligation
</LI><LI> luxAB -SpeI; lumP -XbaI, ;uxCDE -XbaI
</LI><LI> lumP = P1024
</LI><LI> luxAB + lumP and luxAB + lyxCDE ligation
</LI></UL><P><B>20/8/2010</B></P><UL><LI> Fusion PCR with ?????
</LI><LI> get: 
</LI></UL><P><B>24/8/2010</B></P><UL><LI> Digest cph8 (XbaI- yello) &amp; Harvard  (SpeI -brown)
</LI><LI> Ligate Harvard and cph8 = harvard + cph*8
</LI><LI> mutagenic PCR of lux CDE to mutate out site in LuxD
</LI><LI> subculture luxAB lumP transformants
</LI><LI> miniprep ON'S <B>(?)</B> set up
</LI></UL><P><B>25/8/2010</B></P><UL><LI> Minipreps of subcultures luxAB lumP,
</LI><LI> digest &amp; run gel
</LI><LI> Purify luxCDE PCR product- from gel and froem the product; run the gel of purifications --&gt; do MABEL self ligation- set 16C ON
</LI><LI> Fusion PCR harv + cph8- run gel of product
</LI></UL><P><B>26/8/2010</B></P><UL><LI> Gel1 - minipreps 1-5, gluxCDE, PluxCDE
</LI><LI> Gel2 - Minipreps 6-10, H + C ligation
</LI></UL><UL><LI>Lane 7: Harvard + Cph8 ligation
</LI></UL><P><B>27/8/2010</B></P><P>Transform cells with self ligated MABEL mutated luxCDE from gel and PCR product
</P><P><B>30/8/2010</B></P><UL><LI> Lu wants a broad sward (?).
</LI><LI> Re-transform MABEL mutated
</LI><LI> luxCDE from gel- PCR products
</LI><LI> luxAB and luxABlumP double digest with EcoRi/PstI
</LI><LI> Fusion PCR if H + cph8 using H promoter forward and cph8 reverse primer
</LI></UL><P><B>31/8/2010</B></P><UL><LI> Lane 1- marker, Lane 2- H+ Cph8 fPCR
</LI></UL><P><B>6/9/2010</B></P><UL><LI> ligated H+ C ON in psB1C3
</LI><LI> Double digest luxCDE with EcoRI and PstI - run gel
</LI></UL><P><B>7/9/2010</B></P><UL><LI> Run digest luxCDE with XbaI  --&gt; gel
</LI><LI> Gel of H + C ligation
</LI><LI> transform cells with
</LI></UL><P><B>8/9/2010</B></P><UL><LI> PCR with biobrick pre/suff - primers luxCDE 1 and 4
</LI></UL><P><B>9/9/2010</B></P><UL><LI> Purify luxCDE PCR
</LI><LI> Subculture transformants H + C
</LI></UL><P><B>10/9/2010</B></P><UL><LI> Add promoter to luxAB and luxABlumP
</LI><LI> Liquid ON'S of H + C subculture transformants 
</LI></UL><UL><LI> Lab meeting: small--&gt;</LI><LI>sacB - some preeliminary sucrose sensitivity characterisation data.
</LI><LI> promoter: luxAB, luxAb-lumP- eission spectra, testing thing
</LI><LI> red light luciferase: S184T, 356 K
</LI></UL><TABLE border="1"><TBODY><TR><TD> Lane 1
</TD><TD> Lane 2
</TD><TD> Lane 3
</TD><TD> Lane 4
</TD></TR><TR><TD> Ladder
</TD><TD> Tna up
</TD><TD> Cph8- fixing prefix
</TD><TD> Cph8 - fixing PstI site
</TD></TR></TBODY></TABLE><P><B>13/9/2010</B></P><UL><LI> H + C + pSB1C3 minipreps
</LI></UL><P><B>14/9/2010</B></P><UL><LI> Digest H+ C minipreps with EcoRi. 100: 1,4,5. 900: 3,4,6
</LI></UL><P><B>???</B><B>20/9/2010</B></P><UL><LI> Test luxCDE
</LI><LI> luxCDE digest with XBaI and PstI (buffer H)--&gt; purify and ligate to luxAB digested with SPEI PstI in buffer B after purification.
</LI><LI> conclusion: purify digestions, ligate luxCDE into luxAB + promoter
</LI></UL><P><B>21/9/2010</B></P><UL><LI> Transform cells with luxAB + luxCDE ligation, plated on CML 40, IPTG 90
</LI><LI> Made up 50 ml liquid cultures of luxAB lumP and luxAB with Cml40, IPTG 90
</LI></UL><P><B>04/10/10</B>
Chris has managed to construct what we hope is a working red light sensor. It is being sent in for sequencing today. 
Minor characterisation experiment:
-Red light sensor constructed with blue-white lacZ reporter
-If activated, the red light sensor transformants should produce blue colonies
-Red light sensor containing-colonies from transformant plate streaked onto two plates
-One plate left in the light, the other left in the dark
-If the one in the light grows blue and the one in the dark doesn't, it is a good indication that the construct is working</P><P><B>12/10/10</B></P><UL><LI>Transform <I>envZ</I> mutant strain with YFP
</LI><LI>Plate out onto 40mg/ml Cml
</LI></UL><P><B>13/10/10</B></P><UL><LI>Subcultured <I>envZ</I> transformants onto Cml 40 plates
</LI></UL><P><B>15/10/10</B></P><UL><LI>Liquid Cultures made of transformants with Cml 40 &amp; grown ON
</LI></UL><P><B>16/10/10</B></P><UL><LI>Liquid ONs viewed under blue light, yellow fluorescence seen showing successful transformation.
</LI></UL><P><B>20/10/10</B></P><UL><LI>RLS characterisation begins.
</LI><LI>Liquid ON's of
</LI></UL><UL><LI>JM109 – RLS.lacZ.YFP</LI><LI>envZ – RLS.lacZ.YFP</LI><LI>JM109 – YFP Control</LI><LI>envZ – YFP Control</LI></UL><P>were made in 5ml LB + Cml 40 and grown at 37 degrees with shaking in the dark.
</P><P>100μl of ONs were used to inoculate two sterile LB + 40mg/ml Cml making up a total volume of 4ml in 5ml flasks which were then incubated in both light and dark, at 37C with shaking, as follows:</P><UL><LI>JM109 – RLS.lacZ.YFP (Light / Dark)</LI><LI>envZ – RLS.lacZ.YFP (Light / Dark)</LI><LI>JM109 – YFP Control (Light / Dark)</LI><LI>envZ – YFP Control (Light / Dark)</LI></UL><P>At 50 minute time intervals 200μl of each sample was taken and mixed with 800μl of sterile water in plastic cuvettes. The optical density of each sample was taken after the spectrophotometer was set with a sample of 200μl of sterile LB and 800μl water as a control, as was the luminescence, with readings for the background luminescence being taken for the sample of LB and water.</P><P>Two readings for each sample at each time interval were taken and then an average was calculated for each time interval for both optical density and luminescence.</P><P>For each sample at each time interval the average luminescence (normalised by the background luminescence) was divided by the optical density and plotted on a graph, shown below as Figure 2.</P><UL><LI> PCR of PhoA promoter (from genomic DNA).
</LI></UL><P><B>25/10/10</B></P><UL><LI>Liquid ON's made up of 
</LI></UL><UL><LI>JM109 – RLS.lacZ.YFP</LI><LI>envZ – RLS.lacZ.YFP</LI><LI>JM109 – RLS.lacZ</LI><LI>JM109 - lacZ Control</LI></UL><P>in 2.5ml LB + 40 Cml
</P><P><B>26/10/10</B></P><P>A modified version of the Beta-Galactosidase (Better Miller) Assay from OpenWetWare page <A href="http://openwetware.org/wiki/Beta-Galactosidase_Assay_(A_better_Miller)" class="external free" rel="nofollow">http://openwetware.org/wiki/Beta-Galactosidase_Assay_(A_better_Miller)</A> was used to attempt to characterise the RLS with lacZ. Aliquots were taken every hour of the cultures which were set up as before,100μl of ONs were used to inoculate two sterile LB + 40mg/ml Cml making up a total volume of 4ml in 5ml flasks which were then incubated in both light and dark, at 37C with shaking, as follows:&lt;/p&gt;</P><UL><LI>JM109 – RLS.lacZ.YFP (Light/Dark)</LI><LI>envZ – RLS.lacZ.YFP (Light/Dark)</LI><LI>JM109 – RLS.lacZ (Light/Dark)</LI><LI>JM109 - lacZ Control (Light/Dark)</LI></UL><P> For each sample at each of the time points, Abs420 / reaction time was calculated and plotted onto a graph, shown below.


</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:Edinburgh/Notebook/Red_light_sensor">http://2010.igem.org/Team:Edinburgh/Notebook/Red_light_sensor</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Edinburgh/Notebook/Red_light_sensor" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Edinburgh/Notebook/Red_light_sensor" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Edinburgh/Notebook/Red_light_sensor&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Edinburgh/Notebook/Red_light_sensor&amp;oldid=205009" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>