<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Edinburgh_Project_Protocol"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Edinburgh/Project/Protocol">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Edinburgh/Project/Protocol&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Edinburgh/Project/Protocol&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Edinburgh/Project/Protocol&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Edinburgh/Project/Protocol" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Edinburgh/Project/Protocol</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><DIV id="nav"><UL class="dropdown dropdown-horizontal"><LI><A href="https://2010.igem.org/Team:Edinburgh" class="dir">home</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Team" class="dir">team - illuminati</A></LI><LI><A href="https://igem.org/Team.cgi?id=322">official</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Team/Supervisors">supervisors</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Team/Advisors">advisors</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Team/Students">students</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Team/Environment">environment</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Team/Gallery" class="dir">gallery</A></LI></UL><LI><A href="https://2010.igem.org/Team:Edinburgh/Project" class="dir">genomic BRIDGEs</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Project/Protocol">the protocol</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/BioBricks#Genomic">submitted parts</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Results#Genomic">results</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Project/Future">the future</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Project/References">references</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Bacterial" class="dir">bacterial BRIDGEs</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Bacterial/Core_repressilator">the project</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Bacterial/Red_light_producer">red light</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Bacterial/Red_light_sensor">red sensor</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Bacterial/Blue_light_producer">blue light</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Bacterial/Blue_light_sensor">blue sensor</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Bacterial/Green_light_producer">green light</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Bacterial/Green_light_sensor">green sensor</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/BioBricks#Bacterial">submitted parts</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Results#Bacterial">results</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Bacterial/Future">the future</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Bacterial/References">references</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Modelling" class="dir">modelling BRIDGEs</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Modelling/Kappa">kappa</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Modelling/Genomic">the genomic model</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Modelling/Bacterial">the bacterial model</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Modelling/Signalling">the signalling model</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Modelling/Tools">tools</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Results#Modelling">results</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Modelling/Future">the future</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Modelling/References">references</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Human" class="dir">human BRIDGEs</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Human/Communication">communication of science</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Human/Branding">iGEM survey</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Human/Conversations">conversations</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Human/Epic">the epic</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Human/FutureApps">future applications</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Results#Human">further thoughts</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Human/References">references</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Notebook" class="dir">lab notes   </A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Notebook/Collaboration">collaboration</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Notebook/Attribution">attribution</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Notebook/BRIDGE">BRIDGE</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Notebook/Red_light_producer">red light</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Notebook/Red_light_sensor">red sensor</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Notebook/Blue_light_producer">blue light</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Notebook/Blue_light_sensor">blue sensor</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Notebook/Green_light_producer">green light</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Notebook/Green_light_sensor">green sensor</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Notebook/Modelling">modelling</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Notebook/Safety">safety</A></LI><LI><A href="http://www.openwetware.org/wiki/French_Lab">protocols</A></LI><LI><A href="https://2010.igem.org/Team:Edinburgh/Acknowledgements" class="dir">acknowledgements</A></LI></DIV><DIV id="body" style="padding: 0px 60px 10px 60px; height: 1356px"><H2>BRIDGE: The concept</H2><P><B>BRIDGE</B> stands for BioBrick Recombination In Direct Genomic Editing. It is an <B>alternative</B> method for inserting BioBricks into the genome by using homologous recombination instead of restriction digestion, with the added <B>bonus</B> of not leaving a marker behind in the product (<A href="https://static.igem.org/mediawiki/2010/5/5c/Ed10-OriginalBridge.JPG">Figure 1</A>).</P><CENTER><P><B>Figure 1:</B> The strategy for markerless deletion of a chromosomal gene by two-step recombination. (A) A DNA fragment carrying the <I>cat/sacB</I> genes, flanked by two regions homologous to the DNA sequences bordering the target site, is integrated into the chromosome. (B) A DNA fragment carrying the desired deletion or insertion, again flanked by two long regions homologous to the DNA sequences bordering the target sites, replaces the <I>cat/sacB</I> genes through homologous recombination.</P><P>Image: <A href="#References">Sun et al. (2008)</A></P></CENTER><P>The first step of BRIDGE requires the deletion of existing DNA (probably a non-coding piece or a non-essential gene) to introduce a construct of two genes; a positive selection marker and a negative selection marker. We have <B>chosen</B> an antibiotic resistance gene (<I>cat</I> - chloramphenicol resistance) for the former, and <I>sacB</I>, which prevents the host from growing on sucrose, for the latter. After the first step we can thus <B>select</B> for cells which have taken up the construct by growing them on chloramphenicol.</P><P>The second step involves swapping the construct for another piece of DNA (e.g. a <A href="http://partsregistry.org/Main_Page">BioBrick</A> construct) through homologous recombination. After this we can <B>select</B> for those with the new gene by growing the cells on sucrose. Neither selection marker is left in the genome, but the original DNA is replaced with the desired insert (or simply deleted).</P><P>The gene we have <B>chosen</B> for preliminary targetted deletion is <I>tnaA</I>, the indole producing gene. To this effect, we have <B>submitted</B> its <A href="http://partsregistry.org/Part:BBa_K322705">up-</A> and <A href="http://partsregistry.org/Part:BBa_K322706">downstream</A> sequences as BioBricks.</P><P>Eventually our <B>goal</B> would be to <B>apply</B> this protocol to our other project, <A href="https://2010.igem.org/Team:Edinburgh/Bacterial">FORTH</A>. For example, we could directly <B>replace</B><I>trpR</I> with the <A href="https://2010.igem.org/Team:Edinburgh/Bacterial/Blue_light_sensor">LovTAP</A> construct and readout system, thus in one step introducing a key light sensor and removing background noise.</P><H2>BRIDGE: The advantages</H2><P>BRIDGE has two significant <B>advantages</B> over the current method of BioBrick insertion. The first is that it is vector independent - whole PCR constructs can be inserted directly into the genome in two steps in under a week, compared to the <B>lengthy</B> process of vector digestion and ligation required with normal BioBricks.</P><P>The other major <B>advantage</B> is that it will not leave a lasting marker in the genome. With most BioBricks we have to leave a marker (antibiotic resistance, GFP, etc) in our constructs so that we can <B>guarantee</B> their presence. This becomes an issue, a) when you want to <B>use</B> the organism in an industrial or environmental <B>capacity</B>, and b) when you want to <B>insert</B> multiple constructs into the same host organism (there are only a limited number of markers out there). With this system, the markers are removed every time a new gene is inserted, so they can be used again and again indefinitely. You could essentially <B>replace</B> the entire genome with new genes, should you be so <B>inclined</B>!</P></DIV><DIV id="body" style="padding: 0px 60px 10px 60px; height: 1356px"><H2>BRIDGE: The protocol</H2><P>The <B>idea</B> here is that I will write up the method that gave us the best results in the shortest time. The protocol that is up here by the wiki freeze may not be the optimum, but will be based on our <B>success</B> so far. To view the progression of this protocol over time, please visit the <A href="https://2010.igem.org/Team:Edinburgh/Notebook/BRIDGE">labnotes</A> for this component of the project.</P><H3>Introduction</H3><P><B>IMPORTANT</B>: In the BRIDGE protocol the <B>sections in bold indicate points of particular importance</B>, not links to human aspects, unlike elsewhere on the wiki.

</P><P>The bulk of this protocol has been developed from the <A href="http://genebridges.com/">Gene Bridges &quot;Quick and Easy BAC Modification Kit by Red/ET Recombination&quot; protocol, version 2.4 from 2005</A>. We have had to edit in places as we are not using vectors and have two strains of <I>E. coli</I>. For clarification on why certain steps are taken you should refer to the full protocol. If you are using an alternative plasmid then you should refer to a protocol relevant to this plasmid for sections 1 and 2 (i.e. growth conditions). I will give you a brief overview of how to make your constructs by traditional BioBricking methods and will then go into detail about how to transform and select for you recombinants.</P>The two <I>E. coli</I> strains we used were JM109 and DH5alpha. When transformed with the Red/ET these are tetracycline resistant, but you be aware that <B>the two strains require different concentrations of tetracycline</B> in liquid cultures as JM109 has greater antibiotic resistance than DH5alpha. These concentrations are given in the protocol but remember that if you alter the volume of you liquid cultures you should also alter the volume of tet15 added.<P>NB: <B>JM109 and DH5alpha do not make good hosts for recombination</B>. We did not discover this until too late and recommend using another strain such as K12.</P><P>You should also bear in mind that <B>the Red/ET plasmid will not replicate at 37C</B> so will be lost from cells grown at 37C for more than an hour. For this reason we grew our transformants at both 30C and 37C to retain the plasmid and to gain decent growth for characterisation and determination of results.</P><P>For this experiment <B>you will need negative controls</B> for your transformations to confirm that growth is definitely due to the presence of the chloramphenicol resistance gene. We grew 1 liquid culture for each strain, prepared 2 transformations for each (one with and one without DNA), grew each transformant at both 30C and 37C and then plated each of these onto cml40 at both a high and a low concentration of cells. By the end of the first step you should have 16 plates: JM109/DH5alpha +/-DNA at 30/37C and concentrated/not concentrated.</P><P>Sections:</P><OL><LI>Creating homology between target and constructs</LI><LI>Preparing cells with Red/ET</LI><LI>Preparing cells for electroporation</LI><LI>Electroporation and recovery steps</LI><LI>Selecting for recombinants</LI><LI>Starting again for the second step</LI><LI>Control experiments</LI></OL><P>Materials: (these will be given again for each section)</P><UL><LI>Ingredients and equipment required for PCR and restriction enzyme disgestion</LI><LI>A good stock of sterile LB</LI><LI>A large supply of agar plates with cml40</LI><LI>Agar plates with tet15</LI><LI>Tetracycline (15mg/ml)</LI><LI>Chloramphenicol (40mg/ml)</LI><LI>L-arabinose</LI><LI>Access to electroporation technology</LI><LI>Sterile electroporation cuvettes</LI><LI>Sterile water</LI><LI>Eppendorfs</LI></UL><H3>Section 1 - Creating homology between target and constructs</H3><P>In order to perform BRIDGE, the <I>cat/sacB</I> construct (or whichever combination of markers you choose) and the gene you wish to insert must have homology with a region of the <I>E. coli</I> genome. This could be a non-essential gene, for example we used <I>tnaA</I> which codes for indole production, or it could be a non-coding, non-functional region of the genome.</P><P>Once you have chosen a section of DNA for deletion, you need to identify and obtain the flanking (upstream and downstream) sequences of that region. The <B>upstream sequence should contain the promoter region</B>, so that this is not deleted along with the rest of the gene, leaving your markers silent. This is best done by designing and synthesising primers with the correct BioBrick restriction sites for the two sequences and amplifying them directly out of <I>E. coli</I> by PCR. Alternatively you could have the full sequences synthesised, but this is less cost-effective. <B>The flanking sequences need to be BioBricks</B>, i.e. they should have EcoRI and XbaI sites at their upstream end and SpeI and PstI at their downstream end.</P><P>Once you have your flanking sequences you need to digest the marker construct (which should already be in BioBrick format) with XbaI and your upstream sequence with SpeI before ligating them together. This should then be amplified using the upstream forward primer and the marker construct reverse primer.</P><P>You now need to redigest the new construct with SpeI and digest your downstream sequence with XbaI and ligate them. Using the forward primer of the upstream sequence and the reverse primer of the downstream sequence you can now amplify the final contruct from this ligation.
Repeat this process with the gene you wish to insert at the final step and clean both PCR products. These can not be used in the protocol.</P><H3>Section 2 - Preparing cells with Red/ET</H3><P>The initial protocol suggests an electroporation procedure for inserting the plasmid. We prefer to use a simple cold shock-heat shock method, which seems to work just as efficiently. To see the exact protocol used, please see the &quot;Preparing and using competent <I>E. coli</I> cells&quot; protocol on our <A href="http://www.openwetware.org/wiki/French_Lab">protocols</A> page.</P><P>Spread the transformants to tetracycline (15mg/ml) (or relevant antibiotic if not using Red/ET) plates and then subculture to a second plate as a stock of cells.</P></DIV><DIV id="body" style="padding: 0px 60px 10px 60px; height: 1356px"><H3>Section 3 - Preparing the cells for electroporation</H3><P>NB: This section of the protocol assumes you're only using one strain. You need to repeat this for any others.</P><P>Wherever cells are exposed to the air, be sure to <B>use a clean hood</B> or lit bunsen burner to reduce risk of contamination.</P><P><B>1</B> - Prepare 5ml LB in a 10ml vial with tetracycline at 15mg/ml (tet15) (NB: if using a less resistant strain such as DH5alpha, use a lower concentration of tetracycline)<B>2</B> - Pick cells from your stock plate and innoculate the vial<B>3</B> - Incubate culture overnight at 30C with shaking (200rpm)</P><P><B>4</B> - Prepare another vial with 5ml LB and tet15<B>5</B> - Use 0.5ml of the overnight cultures to innoculate the new vial<B>6</B> - Grow the new liquid culture at 30C for 2 hours with shaking (200rpm)</P><P><B>7</B> - Add 100 microlitres of L-arabinose (the promoter on the plasmid is an L-arabinose-activated promoter) to the vial and grow the cells at 37C for one hour
Step 7 is what is recommended by the original protocol to induce the recombinase genes. This has not worked with Red/ET for us so far. We are looking into alternatives, we suggest you do the same or use an alternative plasmid.</P><P><B>8</B> - Prepare: 1 eppendorf (for collecting and washing cells); 2 electroporation cuvettes (one for transformation, one for a control); 4 extra eppendorfs, two with 1.5ml LB (for the recovery step); put the cuvettes and a bottle of sterile water on ice<B>9</B> - Transfer 1.5ml of culture from the vial to a prepared eppendorf and spin in a centrifuge at 8000rpm for 3 minutes. Remove the supernatant to a waste bottle/beaker. Repeat this until you have removed all the cell culture from the vial. <B>10</B> - Resuspend the cells in 1ml of cold sterile water, spin at 8000rpm for 3 minutes, remove supernatant. <B>11</B> - Repeat step 10 twice more<B>12</B> - Resuspend the cells in 80-100 microlitres of cold sterile water</P><P><B>13</B> - Add 40 microlitres of the washed cells to two prepared chilled electroporation cuvettes<B>14</B> - Add 1 microlitre of your cleaned up-marker/marker-down DNA construct to one of the cuvettes (<B>do not add DNA to your controls</B>)</P><H3>Section 4 - Electroporation and recovery steps</H3><P><B>15</B> - Electroporate all cells in cuvettes with normal <I>E. coli</I> settings<B>16</B> - Add 500 microlitres of LB from one of the eppendorfs to the cuvette to resuspend the cells, then immediately transfer the LB with cells back to the eppendorf<B>17</B> - Transfer 0.75ml of the innoculated lB to one of the empty eppendorfs<B>18</B> - Grow each culture at 37C for 1 hour</P><H3>Section 5 - Selecting for recombinants</H3><P><B>19</B> - Prepare 4 5ml vials of LB with cml40 (or lower if you have a less resistant strain)<B>20</B> - Transfer all 0.75ml of each culture to a vial and grow overnight at either 30C or 37C with shaking (200rpm)
By now you should have:</P><UL><LI>Strain+markers at 37C</LI><LI>Strain+markers at 30C</LI><LI>Control strain at 37C</LI><LI>Control strain at 30C</LI></UL><P><B>21</B> - If you do get growth in your positive transformant vials it is worth performing a PCR using the cell sulture as template and the UP forward and DOWN reverse primers from you deleted gene. Run the same PCR with a negative control and check for a difference in band size that resembles your construct (<I>cat/sacB</I> should be ~2.5kb whereas <I>tnaA</I> is ~2kb).<B>22</B> - Transfer some of your positive 30C cell culture (remember that the 37C ones will have lost their plasmid) to an eppendorf and dilute with sterile water (you can use your judgement here, but 100 microlitres in 0.9ml should give a good dilution for spreading)<B>23</B> - Spread 100 microlitres of the dilution onto a cml40 plate and incubate overnight at 30C<B>24</B> - If the transformants grow on the plates, subculture a few colonies to a second cml40+tet15 plate for future stock and incubate at 30C overnight
Although the protocol states that cultures should grow on plates overnight, sometimes this takes slightly longer. Allow an extra day for every set of transformants and subcultures to grow on cml40, especially if using <I>cat</I>.</P><H3>Section 6 - Starting again for the second step</H3><P>Repeat steps 1-24 from above with some adjustments:</P><UL><LI>Initial liquid cultures should be in tet15 and cml40 (or you antibiotic)</LI><LI>After electroporation cells should be grown in/on 10% sucrose (and tet15 if you wish to avoid contamination)</LI><LI>Use the final gene construct DNA instead of the marker construct DNA for electroporation</LI></UL></DIV><DIV id="body" style="padding: 0px 60px 10px 60px; height: 1356px"><H3>Section 7 - Control Experiments</H3><P>This is a long experiment with much room for error and little for quantification. These additional experiments were run when we needed to confirm or rule out problems with the protocol. We recommend you use them in the same way to solve similar problems.</P><H4>Contamination:</H4><P>To be used for narrowing down stage of entry of contaminants.</P><P>For the first two runs of BRIDGE we attempted, the cultures became contaminated. One type of the contaminants was RFP containing <I>E. coli</I>, another was micrococcus (which probably came from the experimenter) and the third was an unidentified organism which grew into white, runny colonies and is apparently chlromphenicol resistant and prefers 30C to 37C (see lab notes for details).</P><P>To determine where these were coming from we decided to take samples of the cells after every step of the protocol which could involve human error or required contact with another solution or object and plate them to chloramphenicol 40 plates. This allowed us to narrow down the source. If you suspect contamination on your final plates you might want to follow these steps in addition to the protocol above. </P><P>We took samples at 5 stages prior to the final spreading of the transformants:</P><OL><LI>After growing the strains overnight at 30C</LI><LI>After growing the cells for 2 hours at 30C in new LB</LI><LI>After growing the cells at 37C for one hour with arabinose</LI><LI>After cleaning the cells with sterile water</LI><LI>After growing for a couple of hours after electroporation</LI></OL><P>If the contamination appears after:</P><UL><LI>Sample 1- either the original strains are contaminated or they are getting by human error</LI><LI>Sample 2- they are probably getting in by human error</LI><LI>Sample 3- the arabinose is contaminated</LI><LI>Sample 4- either the sterile water or the eppendorfs are contaminated (or possibly too much exposure to non-sterile air)</LI><LI>Sample 5- contaminants are coming from the electroporation lab/area</LI><LI>The final spreading of the transformants - human error in spreading</LI></UL><H4>Negative Controls:</H4><P>To be used to pinpoint source of contamination.</P><P>The previous experiment narrowed down the contamination to the washing step. This meant it could either be the sterile water, the unsterile air or the eppendorfs.</P><P>The next time we performed the BRIDGE protocol we took 4 samples at the washing step:</P><OL><LI>DH5alpha after being washed under the clean hood</LI><LI>JM109 after being washed under the clean hood</LI><LI>Sterile water from previous run of experiment kept in an eppendorf for ten minutes</LI><LI>Fresh sterile LB kept in an eppendorf for ten minutes</LI></OL><P>Each were plated to cml40 plates. If contamination occurs on:</P><UL><LI>Bacterial samples only - samples were already contaminated</LI><LI>Sterile water but not LB - sterile water is contaminated</LI><LI>Both sterile water and LB - eppendorfs are contaminated</LI><LI>None - using the clean hood has prevented contamination</LI></UL><P>We got no growth from these controls and samples. We concluded that the exposure to non-sterile air had lead to the contamination in the previous experiments.</P><H4>Lethality of Electroporation:</H4><P>To be used if cells are not growing in order to rule out death by electroporation.</P><P>At one point we became concerned that the electroporation was damaging the cells and preventing them from growing after transformation. To determine if this was the case we set up extra liquid cultures after the recovery step.</P><P>As well as the normal 8 cultures in chloramphenicol (see above) we set up each of the positive transformants (those electroporated with the <I>cat/sacB</I> construct) in tetracycline at the concentrations used in the initial overnight cultures (tet15/5microlitres for JM109 and tet3.75/1.75microlitres for DH5alpha).</P><P>Whereas none of the chloramphenicol-containing cultures grew, 3 of the tetracycline-containing cultures did (see lab notes for details). This confirmed that, in the majority of cases, elecrtroporation was not killing the cells. We then had to conclude that either the Red/ET plasmid is incorrect (i.e. we were using the wrong one) or that the recombinase genes were not being properly induced.</P></DIV><DIV id="body" style="padding: 0px 60px 10px 60px; height: 1356px"><H4>Titre of Resistance:</H4><P>To be used for weak antibiotic resistance genes to confirm if a protocol works or to characterise said gene.</P><P>The <I>cat</I> gene we used for chloramphenicol resistance in this procedure is not a very strongly induced gene. When the cells still wouldn't grow in lowered concentrations of chloramphenicol (see lab notes) we decided to confirm if they could grow in the presence of any chloramphenicol. For this we designed and carried out a simple experiment which can be used to titre the antibiotic resistance of a gene.</P><P>Each transformant (JM109+<I>cat/sacB</I> and DH5alpha+<CAT>) was grown in 5ml liquid LB cultures with increasing concentrations of chloramphenicol -  0(0microlitres) to 24(3microlitres) - along with non-transformant controls of both strains.</CAT></P><P>This experiment can be interperated both qualitatively and quantitatively:</P><UL><LI>Qualitatively - determine the presence of growth by eye (if characterisation is unimportant)</LI><LI>Quantitatively - measure the optical density of each culture relative to that of LB</LI></UL><CENTER><TABLE border="1" style="text-align: center; background: transparent;"><TBODY><TR><TH colspan="2"> Ju109 (tet15 / 5ul)</TH><TH colspan="2"> DH5a (tet3.75 / 1.75ul)</TH></TR><TR><TH> Cul </TH><TH> + cat/sacB </TH><TH> control </TH><TH> + cat/sacB </TH><TH> control </TH></TR><TR><TD> 0 / 0 ul </TD><TD> + </TD><TD> + </TD><TD> + </TD><TD> + </TD></TR><TR><TD> 4 / 0.5 ul </TD><TD> - </TD><TD> - </TD><TD> - </TD><TD> - </TD></TR><TR><TD> 8 / 1 ul </TD><TD> - </TD><TD> - </TD><TD> - </TD><TD> - </TD></TR><TR><TD> 12 / 1.5 ul </TD><TD> - </TD><TD> - </TD><TD> - </TD><TD> - </TD></TR><TR><TD> 16 / 2 ul </TD><TD> - </TD><TD> - </TD><TD> - </TD><TD> - </TD></TR><TR><TD> 20 / 2.5 ul </TD><TD> - </TD><TD> - </TD><TD> - </TD><TD> - </TD></TR><TR><TD> 24 / 3 ul </TD><TD> - </TD><TD> - </TD><TD> - </TD><TD> - </TD></TR></TBODY></TABLE><P><B>Table 1:</B> The results of our titre: &quot;+&quot; indicates growth, &quot;-&quot; indicates no visible growth; there was no visible growth in any of the cultures containing chloramphenicol so we did not continue with the optical density readings.</P></CENTER><P>From this we concluded that <I>cat</I> is definitely not present in the transformants.</P><H2>BioBricks</H2><P>The BioBricks <B>created</B> for this component of the project include the <I>cat</I> and <I>sacB</I> genes, the composite construct containing them both, and sample upstream and downstream sequences for targetting the <I>E. coli</I> tryptophanase locus using BRIDGE.</P><P><A href="http://partsregistry.org/Part:BBa_K322210">BBa_K322210</A>: chloramphenicol resistance gene (chloramphenicol acetyltransferase)</P><P><A href="http://partsregistry.org/Part:BBa_K322921">BBa_K322921</A>: <I>Bacillus subtilis</I> levansucrase, lethal to <I>E. coli</I> in the presence of sucrose.</P><P><A href="http://partsregistry.org/Part:BBa_K322922">BBa_K322922</A>: composite construct of both <I>cat</I> and <I>sacB</I></P><P><A href="http://partsregistry.org/Part:BBa_K322705">BBa_K322705</A>: Upstream region of <I>E. coli</I> tryptophanase locus, used for targetting genes to this locus using BRIDGE.</P><P><A href="http://partsregistry.org/Part:BBa_K322706">BBa_K322706</A>: Downstream region of <I>E. coli</I> tryptophanase locus, used for targetting genes to this locus using BRIDGE.</P></DIV><DIV id="body" style="padding: 0px 60px 10px 60px; height: 1356px"><H2>Characterisation</H2><P><B>Characterisation</B> of the BRIDGE protocol focused on characterisation of the individual BioBricks that make up the BRIDGE construct: <A href="http://partsregistry.org/Part:BBa_K322210"><I>cat</I></A> and <A href="http://partsregistry.org/Part:BBa_K322921"><I>sacB</I></A>. By <B>ensuring</B> that these two BioBricks worked correctly, we would confirm the activity of the basic positive and negative selection markers so crucial to the protocol.</P><CENTER><P><B>Figure 2:</B> Characterisation plates for <I>cat</I>: cell lines containing <I>cat</I> in pSB1A2 (red boxes) grew on both ampicillin (100mg/ml) chloramphenicol (40mg/ml) whereas cell lines containing a pSB1A2 control (Edinbrick, blue boxes) only grew on ampicillin (100mg/ml).</P></CENTER><CENTER><P><B>Figure 3:</B> Characterisation plates for <I>sacB</I>: the control strain which does not contain <I>sacB</I> has grown well on both plates whereas the <I>sacB</I>-containing strain X2 grows poorly on 10% sucrose and E6 does not grow at all.</P></CENTER><P><A href="">Figure 2</A> shows characterisation plates for <I>cat</I>, while <A href="https://static.igem.org/mediawiki/2010/8/8f/Ed10-sacBChar.jpg">Figure 3</A> shows characterisation plates for <I>sacB</I>. The results of both sets of experiments confirm that the selection markers (both positive and negative) work as <B>expected</B> for our protocol. </P></DIV><DIV id="body2" style="padding: 0px 60px 10px 60px; height: 998px"><H2>References</H2><P><B>Sun, W., Wang, S. &amp; Curtiss, R. (2008).</B> Highly Efficient Method for Introducing Successive Multiple Scarless Gene Deletions and Markerless Gene Insertions into the <I>Yersinia pestis</I> Chromosome <I>Appl Environ Microbiol. 2008 July</I>; <B>74</B>(13): 4241–4245.</P><P><B>Gene Bridges</B>: <I>Quick and Easy BAC Modification Kit</I> By Red/ET Recombination. Version 2.4 (February 2005)</P></DIV><DIV id="windowbox" style="border: .2em solid #660000; padding: 5px; position:fixed; top:50%; right:30px; width:8%;"><SPAN style="color:ivory;">Throughout this wiki there are words in <B>bold</B> that indicate a relevance to <B>human aspects</B>. It will become obvious that <B>human aspects</B> are a part of almost everything in <B>iGEM</B>.</SPAN></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:Edinburgh/Project/Protocol">http://2010.igem.org/Team:Edinburgh/Project/Protocol</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Edinburgh/Project/Protocol" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Edinburgh/Project/Protocol" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Edinburgh/Project/Protocol&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Edinburgh/Project/Protocol&amp;oldid=205142" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>