<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Penn_Notebook"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Penn/Notebook">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Penn/Notebook&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Penn/Notebook&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Penn/Notebook&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Penn/Notebook" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Penn/Notebook</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><P><TITLE>Penn 2012 iGEM Wiki</TITLE></P></DIV><UL class="nav"><LI class="dropdown"><A href="/Team:Penn/ProjectOverview" style="font-size:13px;">Our Project</A></LI><LI><A href="/Team:Penn/ProjectOverview">Motivation</A></LI><LI><A href="/Team:Penn/ProjectResults">Our Therapeutic</A></LI><LI><A href="/Team:Penn/Achievements">Achievements</A></LI></UL><LI class="dropdown"><A href="/Team:Penn/LightActivatedOverview" style="font-size:13px;">Light Activation</A></LI><LI><A href="/Team:Penn/LightActivatedOverview">Overview</A></LI><LI><A href="/Team:Penn/BLSensor">YF1/FixJ BL Sensor</A></LI><LI><A href="/Team:Penn/LightActivatedLysis">Light Activated Cell Lysis</A></LI><LI class="dropdown"><A href="/Team:Penn/SurfaceDisplayOverview" style="font-size:13px;">Surface Display</A></LI><LI><A href="/Team:Penn/SurfaceDisplayOverview">Overview</A></LI><LI><A href="/Team:Penn/SurfaceDisplay">INPNC Surface Display</A></LI><LI><A href="/Team:Penn/Targeting">HER2 Targeting </A></LI><LI><A href="/Team:Penn/SurfaceDisplayBBa">BBa Surface Display Platform</A></LI><LI class="dropdown"><A href="#" style="font-size:13px;">Wet Lab</A></LI><LI><A href="/Team:Penn/Notebook">Notebook</A></LI><LI><A href="/Team:Penn/Safety">Safety</A></LI><LI><A href="/Team:Penn/Protocols">Protocols</A></LI><LI><A href="/Team:Penn/Parts">Parts</A></LI><LI class="dropdown"><A href="/Team:Penn/HumanPracticesOverview" style="font-size:13px;">Human Practices</A></LI><LI><A href="/Team:Penn/HumanPracticesOverview">Overview</A></LI><LI><A href="/Team:Penn/Barriers">Bacterial Therapeutics</A></LI><LI><A href="/Team:Penn/Outreach">Outreach</A></LI><LI><A href="/Team:Penn/Perceptions">Public Perception</A></LI><LI><A href="/Team:Penn/Nissle">E. Coli Nissle 1917</A></LI><LI><A href="/Team:Penn/VerifiGEM">VerifiGEM</A></LI><LI class="dropdown"><A href="/Team:Penn/Team" style="font-size:13px;">About Us</A></LI><LI><A href="/Team:Penn/Team">Team</A></LI><LI><A href="/Team:Penn/Sponsors">Sponsors</A></LI><P style="text-align:center;color:white;">June 2012 Notebook</P><DIV id="accordion-container"><H2 class="accordion-header">Week 1</H2><DIV class="accordion-content"><P><B>June 6th</B></P><UL><LI>Set up some lab equipment</LI><LI>Autoclaved for a while</LI><LI>Organized biobrick stuff</LI><LI>Called Vinoo about DNA planning</LI></UL><P><B>June 7th</B></P><UL><LI>Transformed Cph8, pLsr, and LuxS</LI><LI>Placed order with Vinoo</LI><LI>Developed idea using PGY/PCN system to activate a gene</LI></UL></DIV><H2 class="accordion-header">Week 2</H2><DIV class="accordion-content"><P><B>June 11th</B></P> <P>Wet Lab</P><UL><LI>PCR'd mCherry from NAS157</LI><LI>Ran 1% Gel and purified product</LI></UL>  <P>Dry Lab</P><UL><LI>Designed primers for LsR promoter</LI><LI>Meeting with Dr. Sarkar</LI></UL><P><B>June 12th</B></P> <P>Wet Lab</P><UL><LI>Digested mCherry PCR product with BamHI and NotI</LI><LI>Column purified mCherry and ligated into NAS152 backbone</LI><LI>Transformed NAS152-mCherry into DH5alpha</LI><LI>Poured 25 LB-Kan plates</LI></UL>  <P>Dry Lab</P><UL><LI>Research more information about bacterial drug delivery system</LI><LI>More research into biofilm project</LI></UL><P><B>June 14th</B></P>  <P>Dry Lab</P><UL><LI>Met with Dr. Goulian, obtained pDawn and pDusk</LI><LI>Identified inaK as a surface display gene we can use</LI></UL></DIV><H2 class="accordion-header">Week 3</H2><DIV class="accordion-content"><P>June 18th</P>  <P>Wet Lab</P><UL><LI>Miniprep pDawn and pDusk</LI><LI>Test cut pDawn and pDusk with XmaI, analytical gel was correct</LI><LI>Prep cut pDawn and pDusk with BamHI and NotI, gel purified</LI></UL>  <P>Dry Lab</P><UL><LI>Ordered and picked up PCR purification kit from cell center</LI><LI>Additional orders through cell center</LI><LI>Designed primers for one of Peter's components (forgot which)</LI></UL><P>June 20</P>  <P>Wet Lab</P><UL><LI>Picked 2 colonies of pDawn-mCherry, innoculated in 5 mL of LB and 50 ug/mL of Kan</LI><LI>PCR purified fragments (Peter), then ran gel?</LI></UL>  <P>Dry Lab</P><UL><LI>Researched DARPin binding domains and linkers</LI><LI>Finalized some biobrick orders</LI><LI>Finalized synthesis order (minus linker)</LI></UL></DIV><H2 class="accordion-header">Week 4</H2><DIV class="accordion-content"><P><B>June 22</B></P>  <P>Wet Lab</P><UL><LI>Ashwin - Repeated miniprep on pDawn and did test cut</LI><LI>Peter  - Miniprepped pet26b and digested with BglII and EcorRI</LI><LI>Avin - Miniprepped pet26b, made glycerol stock and digested with BamH1 and Not1</LI></UL>  <P>Dry Lab</P><UL><LI>Avin - Finalized and sent in synthesis order (still awaiting order confirmation)</LI></UL><P><B>June 25</B></P>  <P>Wet Lab</P><UL><LI>Ashwin/Avin - Column purification, ligation, and transformation of Pet26b-mCherry</LI><LI>Peter -run gel of eGFP/plsr ligation</LI></UL>  <P>Dry Lab</P><UL><LI>Avin - Sent in final gene synthesis order</LI><LI>Mike - reviewed pDawn protocol, reviewed TetR sequences</LI><LI>Peter- Order restriction enzymes from cell center</LI></UL><P><B>June 26</B></P>  <P>Wet Lab</P><UL><LI>Avin - Picked colonies for Pet26b-mCherry and pJT106b</LI><LI>Mike/Ashwin - Plated pJT122</LI></UL>  <P>Dry Lab</P><UL><LI>Everyone - Brainstormed human practices, Wiki design</LI></UL><P><B>June 27</B></P>  <P>Wet Lab:</P><UL><LI>Miniprepped pet26b-mCherry and pDawn-mCherry</LI><LI>Test cut pet26b-mCherry with HindIII and ClaI (picture temporarily saved in MEAM folder of pqiao account), bands looked okay, chose C2 for remainder of experiments and made a glycerol stock, transformed both pet26b-mCherry and pDawn-mCherry into BL21(DE3)- Gold (10 ul cells, ~1ng DNA)
Plated 200 ul and 20 ul</LI><LI>Inoculated colonies for pJT122, pJT106b, PhyB (BBa), PIF3 (BBa), and Cph8 (BBa)</LI></UL>  <P>Dry Lab:</P><UL><LI>Started exploring possible wiki designs and coding</LI></UL><P><B>June 28</B></P>  <P>Wet Lab:</P><UL><LI>Miniprepped pJT122, pJT106b, PhyB, PIF3, Cph8, made glycerol stocks</LI><LI>Picked colonies for pET26b-mCherry and pDawn-mCherry, innoculated, grew up, then diluted
Measured OD along the way</LI></UL>  <P>Dry Lab:</P><UL><LI>Set up light and dark incubators</LI></UL><P><B>June 29</B></P>  <P>Wet Lab:</P><UL><LI>Tested pDawn-mCherry and pET-mCherry, pDawn-mCherry expressed mCherry only after light induction</LI><LI>Nanodropped pJT122, pJT106b, PhyB, PIF3, Cph8</LI><LI>Performed digest of pET-26b, eGFP, and plsr</LI><LI>Spread Biobrick shipment on Amp plates (lsrR, lsrK)</LI></UL>  <P>Dry Lab:</P><UL><LI>Opened Bio-Rad shipments</LI></UL></DIV></DIV><P style="text-align:center; color:white;">July 2012 Notebook</P><DIV id="accordion-container"><H2 class="accordion-header">Week 5</H2><DIV class="accordion-content"><P><B>July 2</B></P>  <P>Wet Lab:</P><UL><LI>Transformed ho1 and pcyA BioBricks</LI><LI>Peter: ligations</LI></UL>  <P>Dry Lab:</P><UL><LI>Contacted labs for JT2 and pPL-PCB</LI><LI>Worked on Human Practices</LI></UL><P><B>July 3</B></P>  <P>Wet Lab</P><UL><LI>Nothing to be done for drug delivery</LI></UL>  <P>Dry Lab</P><UL><LI>Avin - digital schematic, helped with primers, read human practices bacterial therapy stuff</LI><LI>Mike - talked to Dan, developed cloning strategy for getting Cph8 into the pDawn backbone, started primer design</LI></UL><P><B>July 5</B></P>  <P>Wet Lab</P><UL><LI>Picked colonies of ho1 and pcyA</LI><LI>Inoculated pDawn-mCherry cells for timecourse</LI></UL>  <P>Dry Lab</P><UL><LI>worked on primer design/cloning strategy and talked to Dan</LI><LI>worked on schematic and wiki</LI><LI>worked on human practices</LI></UL><P><B>July 6-7</B></P>  <P>Wet Lab:</P><UL><LI>Took pDawn-mCherry time course</LI><LI>Sterilized incubator and TC hood</LI><LI>Moved lab stuff to small room</LI><LI>Redesigned primers for plsr &amp; egfp</LI></UL>  <P>Dry Lab:</P><UL><LI>
Contacted FDA contacts for human practices
</LI></UL></DIV><H2 class="accordion-header">Week 6</H2><DIV class="accordion-content"><P><B>July 9</B></P>  <P>Wet Lab:</P><UL><LI>Inoculated 50mL cultures of pDawn-mCherry</LI><LI>Set up pH meter, picked up JT2</LI><LI>Organized lab area </LI><LI>Made bacterial streaks for biobricks</LI></UL>  <P>Dry Lab:</P><UL><LI>Ordered mammalian cell culture media</LI><LI>Finalized wiki template</LI><LI>Worked on cloning steps for cph8</LI></UL><P><B>July 10</B></P>  <P>Wet Lab:</P><UL><LI>Did time course for pDawn-mCherry, but since it was from a glycerol stock, saw no growth → will let grow overnight, then dilute in the morning and start a new time course</LI><LI>Picked colonies from transformed Biobricks and innoculated</LI><LI>Resuspended ClyA and LuxS IDT DNA and transformed into DH5alpha</LI><LI>TC/Sterile practice training</LI></UL>  <P>Dry Lab:</P><UL><LI>Ordered primers for ClyA-RFP, Peters primers</LI><LI>Worked on wiki</LI><LI>Emailed Tabor for pJT106b plasmid map</LI></UL><P><B>July 11</B></P>  <P>Wet Lab:</P><UL><LI>
Recorded BL21 pDawn-mCherry growth curve and calculated doubling time</LI><LI><UL> Miniprepped biobricks
</UL></LI><LI>
BBa_K265008 - Ice Nucleation Protein NC
</LI><LI>
BBa_K523013 - Plac + INP-EYFP 
</LI><LI>
BBa_K299810 - B0032 + Invasin
</LI><LI>
BBa_K257010 - ClyA + RFP
</LI></UL>  <P>Dry Lab:</P><UL><LI>Worked on biofilm project schematic</LI><LI>More work on wiki</LI></UL><P><B>July 13</B></P>  <P>Wet Lab:</P><UL><LI>Diluted to OD600=0.01 and take 0-8hr time points of pDawn-mCherry fluorescence time course</LI><LI>Thawed out HEK293T cells</LI></UL>  <P>Dry Lab:</P><UL><LI>Worked on wiki/schematic</LI></UL><P><B>July 17</B></P><P>  </P><P>Wet Lab:</P><UL><LI>Re-ligation of pDawn with IDTClyA and ClyA-RFP, Transformation into DH5alpha/XO1blue</LI><LI>Performed PCR of plsr &amp; gfp w/new primers.</LI><LI>Picked colonies of T7 and INPNC-DARPin</LI></UL>  <P>Dry Lab:</P><UL><LI>Designed experiments for ClyA and INPNC-DARPin assays</LI></UL></DIV><H2 class="accordion-header">Week 7</H2><DIV class="accordion-content"><P><B>July 16</B></P>  <P>Wet Lab:</P><UL><LI>
Transformed INPNC-DARPin and T7 BioBrick
</LI></UL>  <P>Dry Lab:</P><UL><LI>SAAST presentation</LI><LI>Met with Lazzara lab to discuss DARPin experiments</LI></UL><P><B>July 18</B></P>  <P>Wet Lab:</P><UL><LI>
Re-inoculated T7 and INPNC-DARPin</LI><LI>PCR ClyA-RFP, Gel purify PCR ClyA-RFP product</LI></UL>  <P>Dry Lab:</P><UL><LI>Worked on DARPin assay experiment design</LI></UL><P><B>July 19</B></P>  <P>Wet Lab:</P><UL><LI>Trypan Blue Assay</LI><LI>Digested pDawn-mCherry with BamHI, NotI, pDawn-mCherry with BamHI,Digest pDawn-mCherry with NotI, Digested IDTsmart-ClyA with BamHI, NotI (Gel for Digests showed possible degeneration of NotI enzyme...) </LI><LI>Ligated pDawn backbone (old) with 1) ClyA and 2) ClyA-RFP and Ligated pDawn backbone (new) with ClyA and ClyA-RFP, as well as pDawn old backbone with H20 and pDawn new backbone with H20 (ligation controls)</LI><LI>Transformed all of above ligations into DH5alpha</LI></UL><P><B>July 23</B></P>  <P>Wet Lab:</P><UL><LI>Ashwin-Design test cuts for pDawn-Clya, pDawn-ClyA+RFP </LI><LI>Ashwin-Digest pDawn-ClyA+pDawn-ClyA+RFP, Run on gel </LI><LI>Nikita - Miniprep pDawn culture in Sarkar cold room</LI><LI>Digest of pET-26b w/ BglII, EcoRI-Peter</LI><LI>Digest of plsr product w/ BglII, PseI-Peter</LI><LI>Digest egfp w/ SpeI, EcoRI-Peter</LI><LI>Ligate &amp; transform pET-26-plsr-GFP-Peter</LI><LI>Digest INPNC-DARPin</LI><LI>Avin - Try to rescue 293T cells after CO2 arrives if can’t be rescued, thaw out more 293T</LI><LI>Transform pET-26b-luxS - Peter</LI><LI>Avin - pick GAVPO </LI></UL>  <P>Dry Lab:</P><UL><LI>Ordered AI-2 ASAP-peter</LI><LI>Nikita - Researched whether ClyA is toxic to E. coli BL21, JT2, DH5 alpha, and whether it is secreted in these strains</LI><LI>Read Daphne+Dr. Sarkar’s ligation paper </LI><LI><UL>Avin - Buy: 
</UL></LI><LI>DH5 alpha</LI><LI>20 kan plates, 10 amp plates</LI><LI>Cell Counter</LI></UL><LI>
Write out competent cell production protocol-Peter</LI><LI>Obtained BL21 transformation protocol from daphne/find it-Peter</LI><LI>Avin - Logo design, human practices</LI><LI>Ashwin - wiki</LI><P><B>July 24</B></P>  <P>Wet Lab:</P><UL><LI>Pick colonies of pDawn-ClyA and pDawn-ClyA+RFP (DH5alpha)</LI><LI>Avin - Picked 2 colonies (light+dark) of pDawn-ClyA+RFP (BL21), set up pDawn culture experiment, check on HEK293T and passage into 6 well plates if confluent</LI><LI>Avin - Ran Gel, no DNA</LI></UL>  <P>Dry Lab:</P><UL><LI>Peter: order the promoter (or figure out an alternative method..)</LI><LI>Ashwin: wiki</LI><LI>Avin - Edit/Order primers for HA tag and His tag, design pDawn-ClyA experiments, human practices</LI><LI>Nikita - Write human practices</LI></UL></DIV><H2 class="accordion-header">Week 8</H2><DIV class="accordion-content"><P><B>July 26</B></P>  <P>Wet Lab:</P><UL><LI>Avin - Finish ClyA-RFP time course, collect ClyA supernatant, measure fluorescence of ClyA-RFP triplicates, spin down + pictures if red, Miniprep of DARPin, possible ClyA assay</LI><LI>Peter- ligation, transformation, plating of ligations, help Avin with ClyA cell lysis experiments </LI><LI>Transform luxS6-8 into BL21 </LI><LI> Nikita - Miniprep GAVPO and DARPin </LI></UL>  <P>Dry Lab:</P><UL><LI>Avin - Call IDT at 9am, order blood agar plates, nissl stuff after we do background research</LI><LI>Ashwin - wiki, research getting recombinant ClyA</LI><LI>Nikita - Human practices, Nissle research, plan experiments</LI><LI><UL>Peter
</UL></LI><LI>Complete FDA writeup</LI><LI>Order ATCC Strains</LI><LI>Order lss primers</LI><LI>Order AI-2</LI><LI>Order Crystal Violet Stain</LI><LI>Plan ClyA experiment in JMOutline</LI></UL><P><B>July 30</B></P>  <P>Wet Lab:</P><UL><LI>Avin - Transform pBAD33 and pSB4A5, Inoculate BL21 Intein colonies</LI><LI>Redo the test cuts for pDawn and pDawn-ClyA-RFP with XmaI and ClaI on a 0.7% gel</LI></UL>  <P>Dry Lab:</P><UL><LI>Avin - Human Practices</LI><LI><UL>Mike - Primers
</UL></LI><LI>pET26b-ClyA (IDT) keeping pelB with 6xHis C terminus - use NcoI on forward primer, figure out reverse</LI><LI>pET26b-ClyA (IDT) removing pelB with 6xHis C terminus - use NdeI on forward primer, figure out reverse</LI><LI>pet26b-ClyA+RFP keeping pelB with 6xHis C terminus - use NcoI on forward primer, figure out reverse</LI><LI>pet26B-CLlyA+RFP removing pelB with 6x His C terminus - use NdeI on forward primer, figure out reverse</LI><LI>fix frame shift in pDawn and include 6xHis N terminus - use NdeI on forward primer, figure out reverse</LI><LI>ClyA </LI><LI>mCherry</LI><LI>ClyA-RFP</LI><LI>Cph8 primers - figure out pJT106b primers</LI></UL><P><B>July 31</B></P>  <P>Wet Lab:</P><UL><LI>
Test Cut pDawn-ClyA-RFP colony 4 w/ XmaI and Cla</LI><LI>
Transform the pDawn-ClyA-RFP colony 4 plasmid (assuming test cuts look good) into BL21 and Dh5alpha</LI><LI>
Check for growth of Intein-mCherry plasmid</LI><LI><UL>
Assuming growth: 
</UL></LI><LI>miniprep DH5alpha culture</LI><LI>induce BL21 culture with IPTG - see Jordan for protocol</LI><DILUTE in="" morning=""><LI>If primers arrive → PCR (x2) for DARPin (check order status for IDT in the morning, call cell center if it says its been shipped)</LI><LI>Pick colonies and innoculate pBAD33 (chloramphenicol) and pSB4A5 (ampicillin)</LI></DILUTE></UL>  <P>Dry Lab:</P><UL><LI>Mike - Order Primers</LI><LI>
pET26b-ClyA (IDT) keeping pelB with 6xHis C terminus - use NcoI on forward primer, XhoI on reverse (no stop codon)</LI><LI>
pET26b-ClyA (IDT) removing pelB with 6xHis C terminus - use NdeI on forward primer, XhoI on reverse (no stop codon)
</LI><LI>
fix frame shift in pET26b for ClyA-RFP (NdeI on forward, NotI on reverse, include stop) - can’t use NdeI because there is an NdeI cut site inside ClyA-RFP
</LI><LI>
fix frame shift in pDawn and include 6xHis N terminus 
</LI><LI>
ClyA - NdeI on forward, BamHI on reverse
</LI><LI>
mCherry - NdeI on forward, NotI on reverse
</LI><LI>
ClyA-RFP - NdeI on forward, NotI on reverse - can’t use NdeI because there’s an NdeI cut site inside ClyA-RFP
</LI><LI>
Cph8 primers - figure out pJT106b primers
</LI></UL><LI>
Mike - Take out Biohazard for Sevile
</LI><LI>
Ashwin - wiki
</LI></DIV></DIV><P style="text-align:center;color:white;">August 2012 Notebook</P><DIV id="accordion-container"><H2 class="accordion-header">Week 9</H2><DIV class="accordion-content"><P><B>August 1st</B></P>  <P>Wet Lab:</P><LI>pET-26-plsr-gfp triple ligation ( (+) indicates positive control, (-) indicates negative control)</LI><LI><UL>
Gel Purification 8:30-12:00
</UL></LI><LI>
1% gel 50mL +5uL SyberSafe</LI><LI>
pET-26b cut w/ EcoRI-HF (+)</LI><LI>
pET-26b cut w/ EcoRI-HF &amp; BglII</LI><LI>
run against pET-26b uncut for reference
</LI><LI><UL>
Ligation (Start w/ 25ng vector) 12:00-3:00
</UL></LI><LI>
Vector:GFP:plsr</LI><LI>1:6:6</LI><LI>1:4:4</LI><LI>1:1:1</LI><LI>pET-26b cut w/ EcoRI-HF (+)</LI><LI>pET-26b cut w/ EcoRI-HF &amp; BglII (-)</LI><LI>INCUBATE @ RT 1hr</LI><LI><UL>
Transform Into DH5a Max Efficiency 3:00-6:00
</UL></LI><LI>Transform pET-26b (+)</LI><LI>Transform H2O (-)</LI><LI>Transform 1:6:6</LI><LI>Transform 1:4:4</LI><LI>Transform 1:1:1</LI><LI>
Total Plates Needed=8</LI><LI>
Check IDT primer order in the morning</LI><LI>
Avin - Pick colonies of pDawn-ClyA-RFP (DH5a and BL21) and inoculate into 5 mL LB culture
</LI><LI>
Avin - Miniprep pSB4A5, pBAD33, Intein-mCherry (DH5a)
Grow up BL21 Intein-mCherry and induce for fun?
</LI><P><B>August 2</B></P>  <P>Wet Lab:</P><UL><LI>
pDawn-ClyA-RFP Time course - Avin </LI><LI>
growth to 0.8 and 1:1000 IPTG induction of 50mL BL21 Intein-mCherry - Avin </LI><LI>
Miniprep of DH5 alpha pDawn-ClyA-RFP - Ashwin </LI><LI>
PCR purify INPNC-DARPin-HA Tag - Mike </LI><LI>
Digest INPNC-DARPin-HA Tag and pET26b with EcoRI and NdeI
</LI><LI>
Column purify INPNC-DARPin-HA Tag</LI><LI>
Gel purify pET26b </LI><LI>
Digest INPNC, INPNC-HA Tag, 6xHis-DARPin-Ha Tag with NdeI and BamHI </LI><LI>Column purify INPNC, INPNC-HA Tag, 6xHis-DARPin-HA Tag</LI><LI>
Gel purify PET26b</LI><LI>
Gel purification of pET-26b digested with BglII and EcoRI-HF </LI><LI>
Plated S. Epidermis and E. Coli containing Lysostaphin</LI></UL>  <P>Dry Lab:</P><UL><LI>Logo - Avin</LI></UL><P><B>August 3</B></P>  <P>Wet Lab:</P><UL><LI>BL21 pDawn-ClyA-RFP time course, store blood agar plates in fridge - Avin</LI><LI>BL21 Intein-mCherry Protein purification and Coomassie gel - Avin/Peter</LI><LI>Ligate and transform pET26b-INPNC, pET26b-INPNC-HATag, pET26b-INPNC-DARPin-HATag, pET26b-6xHis-DARPin-HATag</LI><LI>Saturday - pick colonies</LI><LI>Sunday/Monday - Miniprep, test cuts, transformation</LI><LI>Check if ClyA primers are shipped, if so pick up at cell center</LI><LI>PCR ClyA with various primers</LI><LI>Run on Gel, Gel Purify, Digest, Column Purify, Ligate, Transform</LI></UL>  <P>Dry Lab:</P><UL><LI>Pick up primers</LI><LI>Meeting with Dr. Sarkar</LI></UL></DIV><H2 class="accordion-header">Week 10</H2><DIV class="accordion-content"><P><B>August 13</B></P><UL><LI>
Started pDawn ClyA &amp; pDawn ClyA-RFP cultures for blood agar experiment</LI><LI>
Meeting with Dr. Sarkar to discuss progress on both projects</LI></UL><P><B>August 14 </B></P><UL><LI><UL>
Reviewed sequencing results and transformed the following into BL21 and DH5 alpha:
</UL></LI><LI>
pET26b-INPNC
</LI><LI>
pET26b-INPNC-HA
</LI><LI>
pET26b-6xHis-DARPin-HA
</LI><LI>
pET26b-INPNC-DARPin-HA
</LI><LI>
pET26b-ClyA-6xHis
</LI><LI>
pET26b-PelB-ClyA-6xHis
</LI><LI>
pDawn-ClyA-6xHis
</LI></UL><LI>
Serial dilution and plating of pDawn ClyA &amp; pDawn ClyA-RFP cultures for blood agar experiment</LI></DIV><H2 class="accordion-header">Week 11</H2><DIV class="accordion-content"><P><B>August 15</B></P><UL><LI>Re-plated all constructs</LI><LI>pDawn-ClyA and pDawn-ClyA-RFP demonstrated light-dependent hemolysis!</LI><LI>BL21 washing titration experiment for flow cytometry experiment - obtained optimal starting OD600 of 0.05 to obtain ~1E6 cells after all washes and incubations</LI><LI>Made Incubation Buffer for flow cytometry experiment</LI><LI>Designed/ordered pBAD33-eGFP Primers</LI></UL><P><B>August 16</B></P><UL><LI>
Picked colonies on all BL21 and DH5 constructs, including inducing and non-inducing conditions, set up flow cytometry stuff, booked flow core
</LI><LI>
Started cultures of pDawn-ClyA, pDawn-ClyA-RFP, and pDawn-ClyA-mCherry for repeat blood agar experiment</LI><LI>Set up light incubator properly</LI><LI>Started protein purification cultures of pDawn-ClyA-6xHis, pET26b-ClyA-6xHis, and pET26b-PelB-ClyA-6xHis, grew to OD600=0.8, induced with IPTG or light</LI></UL><P><B>August 22 </B></P><UL><LI>
Image INPNC-DARPin-HA, INPNC-HA, DARPin-HA (both induced and not induced) on the confocal - Avin
</LI><LI>
Figure out ClyA imaging system and maybe design a stencil (not a high priority)
</LI><LI>
Pick up media and everything else from cell center - Mike
</LI><LI>
Ligate ClyA into lactococcus? -- Talk to Daphne about vectors
</LI><LI>
Order the cytotoxicity assay from Promega - Avin
</LI><LI>
Ask Dr. Sarkar for cell center account - Mike
</LI><LI>
Order pDawn sequencing primer - Avin
</LI><LI>
Run GFP PCR product on Gel, 
Gel didn’t work → rerun PCR using gradient annealing temp
Run PCR products on gel, check if works
</LI><LI>
Look into biobrick format - Mike
</LI><LI>
Make an in-depth plan for the incoming weeks, prioritizing what needs to be done
Cloning! what is the final construct? Design and execute
add RBS to pBAD33 primer
</LI></UL><P><B>Plan for August 23</B></P><UL><LI>
Add RBS/SD to pBAD33 forward primer and re-order it
</LI><LI>
Send out sequencing for INPNC-DARPin-HA (and what else?)
</LI><LI>
Look at old sequencing data just to make sure everything worked
</LI><LI>
Pick up IPTG from the cell center and make stock solution/aliquots
</LI><LI>
Using new IPTG, spread on blood agar plates (final dilution of 1 to 1000)
</LI><LI>
Spread Kan on Blood Agar Plates
</LI><LI>
Plate pET26b-ClyA-His (+/-), pET26b-pelB-ClyA-His (+/-), and pDawn-ClyA(FS) (+/-) on Blood Agar + Kan plates 
</LI></UL><P><B>August 24</B></P><UL><LI>
If colonies were picked, miniprep, then test cut, then run on diagnostic gel → transform into BL21</LI><LI>
If colonies were not picked, then pick colonies</LI><LI>
Analyze sequencing results</LI><LI>
Plan out INPNC-DARPin-HA experiments
</LI><LI>
Figure out cloning for 1 plasmid system
</LI></UL></DIV><H2 class="accordion-header">Week 12</H2><DIV class="accordion-content"><P><B>Goals in order of priority:</B></P><UL><LI>
Show that INPNC-DARPin-HA binds to HER2 in vitro
</LI><LI>
Show that purified 6xHis-DARPin-HA binds to Her2 - Monday?
</LI><LI>
Show that INPNC-DARPin-HA is displayed on the surface
</LI><LI>
Show INPNC-mCherry is displayed on surface (confocal?)
</LI><LI>
Construct ClyA and ClyA-His BioBricks 
</LI><LI>
Construct INPNC-mCherry Biobrick
</LI><LI>
Construct Surface Display BioBrick (need INPNC-mCherry to work)
</LI><LI>
Transform ClyA into Lactococcus and plate onto blood agar for Human Practices-
</LI><LI>
Stencil experiment showing spatial control of pDawn-ClyA (could be done next week, only takes a little time and 
artistic ability)
</LI><LI>
Show that cph8 works with a reporter
</LI><LI>
Show that cph8-ClyA works
</LI></UL><P><B>Experiments in order of priority:</B></P><UL><LI>
Construction of pBAD33-eGFP (Mike)
</LI><LI>
Order primers for ClyA-His and ClyA to clone them into BioBrick backbone
</LI><LI>
Order primers for INPNC-mCherry
</LI><LI>
Order primers for cph8-reporter, cph8-clyA-his
</LI><LI>
SKBR3 imaging with pBAD33-eGFP/pET26b-INPNC-DARPin-HA bacteria on confocal
</LI><LI>
INPNC-DARPin immunos, 0.2mM IPTG, overnight induction
</LI><LI>
INPNC-DARPin bacterial flow cytometry on FACS machine (can use same bacteria as immuno)
</LI><LI>
Create stable cell line of SKBR3-mCherry (Jordan)
</LI><LI>
Obtain Lactococcus expression vector, order primers to clone in ClyA-His
</LI><LI>
Design surface display biobrick
</LI><LI>
Cytotoxicity experiment on SKBR3</LI><LI>
Protein purification on His-DARPin-HA (Peter)
</LI><LI>
Flow cytometry of SKBR3 cells incubated with his-DARPin-HA (someone/Najaf)
</LI><LI>
Construct cph8-ClyA plasmid
</LI><LI>
cph8-reporter timecourse
</LI><LI>
cph8-ClyA blood agar
</LI></UL><P><B>August 27</B></P><UL><LI>
PCR eGFP out of PHAT (annealing = 65), run on 1% gel, gel purify
</LI><LI>
Digest eGFP and pBAD33 with PstI and XmaI, column purify
</LI><LI>
Ligate at RT for 1 hour, transform into DH5a 
depending on time, otherwise ligate at 16C/4C overnight and transform the next day
</LI><LI>
Order Primers for (in order of priority)
</LI><LI>
ClyA and ClyA-His Biobricks
</LI><LI>
INPNC-mCherry construct
</LI><LI>
ClyA into lactococcus plasmid
</LI><LI>
Cph8-reporter plasmid
</LI><LI>
Cph8-ClyA-His plasmid
</LI><LI>
INPNC surface display vector (provided it works)
</LI><LI>
Start cell culture of SK-BR-3 cells?
</LI><LI>
Get protocols
</LI></UL><P><B>August 28</B></P><UL><LI>
start cultures for purification of His-DARPin-HA
</LI><LI>
HA affinity purification of INPNC-DARPin-HA?
</LI><LI>
Print and hand in safety form to EHRS
</LI><LI>
Pick up SK-BR-3 plate from Cal, learn protocols
</LI><LI>
Add FBS to our McCoy’s media
</LI><LI>
Design and order primers to put ClyA into lactococcus expression vector
</LI><LI>
Set up preliminary stencil experiment, send avin plate base diameter (approximate 86mm)
</LI><LI>
Design INPNC-mCherry-DARPin primers
</LI><LI>
pBAD33-eGFP
</LI><LI>
Run Gradient PCR  products (which ran overnight) on gel, gel purify the best band
</LI><LI>
Digest eGFP and pBAD33 with PstI and XmaI, column purify/gel purify
</LI><LI>
Ligate at RT for 1 hour, transform into DH5a 
depending on time, otherwise ligate at 16C/4C overnight and transform the next day
</LI><LI>
Start DH5 alpha cultures of INPNC, INPNC-HA, DARPin-HA, INPNC-DARPin-HA</LI><LI> Also start cultures of pET-ClyA-His, pET-PelB-ClyA-His and pDawn-ClyA-His for minipreps
</LI></UL><P><B>August 29</B></P><UL><LI>
Induce His-DARPin-HA culture (100 microliters of 1 M IPTG)</LI><LI>
Monitor SK-BR-3 </LI><LI>
Primers for ClyA into lactococcus</LI><LI>
Primers for INPNC-mCherry-DARPin </LI><LI>
Pick colonies for pBAD33-eGFP if the RT ligation and transformation worked - Peter/Ashwin</LI><LI>
If RT ligation and transformation looks bad, transform the 4C and 16C ligations - Mike</LI><LI>
When primers arrive, PCR out mCherry from JMIL intein plasmid, PCR out INPNC, then assembly PCR for INPNC-mCherry</LI></UL><P><B>August 30</B></P><UL><LI>
Protein purification of His-DARPin-HA
</LI><LI>
PCR purify INPNC-mCherry, run a few ul on a diagnostic gel → results call for gel purifiation → Gel purify INPNC-mCherry → yield was bad, redo PCR  → gel purify 
</LI><LI>
Digest INPNC-mCherry and pET26b with NdeI and HindIII, ligate at RT for 1 hr (and maybe some at 4C or 16C overnight), and transform into pET26b
</LI><LI>
Start DH5a cultures for INPNC, INPNC-HA, His-DARPin-HA, pET-ClyA-His, pET-PelB-ClyA-His and pDawn-ClyA-His for minipreps
</LI><LI>
Repick colonies of pBAD33-eGFP from 16C or 4C plate
</LI></UL></DIV></DIV><P style="text-align:center;color:white;">September 2012 Notebook</P><DIV id="accordion-container"><H2 class="accordion-header">Week 13</H2><H2 class="accordion-header">Week 14</H2><DIV class="accordion-content"><P><B>September 10</B></P><UL><LI>
Mike - pick INPNC-mCherry colony (transformed Sample #2) and split into + and - cultures
</LI><LI>Start BL21 cultures of INPNC, INPNC-HA, DARPin-HA, INPNC-DARPin-HA, His-Intein-mCherry
</LI><LI>
Peter - Protein purification of His-DARPin-HA, ClyA-His
</LI><LI>
Avin - Dilute (9am) &amp; Induce (?) His-DARPin-HA, INPNC-HA, INPNC-DARPin-HA at 0.5
</LI><LI>
Ashwin - Wiki work
</LI></UL><P><B>September 11</B></P><UL><LI>
Mike - Make glycerol stock of INPNC-mCherry, Dilute and induce INPNC-mCherry, INPNC, INPNC-HA, DARPin-HA, INPNC-DARPin HA, His-Intein-mCherry
</LI><LI>
Peter - Make electro/chemically competent cells, transform
</LI><LI>
Ashwin - Pick up S. typhimurium from Mark Goulian
</LI></UL><P><B>September 12</B></P><UL><LI>
Peter - Lysis and spin down (for INPNC-mCherry, aliquot  1mL of both (+) and (-) for Avin
</LI><LI>Peter- spin down and check if red, then lyse) , put in 4C, check Nissle eGFP colonies
</LI><LI>
Avin - Immuno experiment on His-DARPin-HA, INPNC-HA, INPNC-DARPin-HA, Take INPNC-mCherry, His-Intein-mCherry, fix and mount onto scope slides
</LI></UL><P><B>September 13</B></P><UL><LI>
Mike - Bradford assay, Run protein gel</LI><LI>Peter - </LI><LI>
Avin - Observe BL21 transformatons under confocal
</LI><LI>
Ashwin - colony PCR on S. typhimurium for MisL
</LI></UL><P><B>September 14</B></P><UL><LI>
Mike - 
</LI><LI>
Peter -
</LI><LI>
Avin - core confocal?
</LI><LI>
Ashwin - 
</LI></UL>  <P>Other</P><P><UL><LI>Order scfv plasmid, scfv primers</LI><LI>
Order MisL Primers, pick up misL, PCR on S. typhimurium
</LI><LI>
Chemical/Electrocompetent Nissle
</LI><LI>
Make LB, Glycerol
</LI><LI>
Wiki
</LI></UL></P></DIV><H2 class="accordion-header">Week 15</H2><H2 class="accordion-header">Week 16</H2></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:Penn/Notebook">http://2012.igem.org/Team:Penn/Notebook</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Penn/Notebook" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Penn/Notebook" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Penn/Notebook&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Penn/Notebook&amp;oldid=272282" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>