"
Page
Discussion
View source
History
teams
Log in
 
Team:ULB-Brussels/Homologous
From 2010.igem.org
  Home  
  Team  
  Project  
  Parts  
  Modeling  
  Notebook  
  Safety  
  Sponsors  
Hydrogen Production
   
Bacteria planned death system
Homologous recombination
The aim of this module is to provide the iGEM  community with the tools we used to delete genes in Escherichia coli. The deletions were performed using the phage λ Red system, as described by Datsenko and Wanner [1] .  The toolbox is composed of two plasmid helpers and two antibiotic resistance  cassettes BioBricks, adapted to iGEM standards. Basically, one could delete a  gene with only this toolbox and a couple of primers designed specifically for  the target gene.
We  think that a simple way to delete genes could be a great asset in the context  of designing new biological systems, and thus could be useful to future iGEM  participants. Provided that the gene is not essential to the microorganism, the  deletion could be constructed for different purposes, and open up new  approaches in iGEM's synthetic biology projects, together with the BioBricks  system. For example, removing a specific gene could enhance a particular  metabolic pathway, like we did for the hydrogen production. Moreover, it could  be easier to delete a gene hindering the production of a desired product than  to over-express enhancer genes. It could also be useful if a gene is  interfering with the function of a BioBrick.
 
How it works
 
In  order to delete a gene, the first step is to perform a PCR using specific  primers and a resistance cassette as template. The fragment is used by the Red  system to replace the target gene with the resistance cassette, and the  recombinant clones are then selected on an appropriate selection medium. The  antibiotic resistance gene is then removed, so as to allow several deletions in  the same strain while avoiding multiple resistances. Before explaining in  details those steps, we will first introduce the different elements.
The  first plasmid helper (PH λ) contains the three genes of  the λ Red system: γ, β, and exo, necessary for  homologous recombination. The γ gene encodes Gam, which prevents an  exonuclease of E. coli from inhibiting the homologous recombination,  while Exo is a 3’-5’ exonuclease preparing the PCR product for the homologous  recombination reaction. The product of β, Bet, is the recombination enzyme.
The  second plasmid helper (PHFlp) contains the flp gene encoding the  Flippase, a site-specific recombination enzyme (Flp). Flp recognizes small  sequences called Flippase recognition target (frt), and, in a way similar to  the Cre-Lox system, catalyzes a site-specific recombination reaction between  the two frt sites. In this case, it is used to remove the frt-flanked  antibiotic resistance gene that replaces the deletion target gene.
Both  plasmid helpers are low copy thermosensitive replicons, which means the  plasmids do not replicate at 42°C. Thus, the plasmids are easily cured after  the different recombination steps and the strain deleted of the target gene do  not contain neither plasmids nor antibiotic resistance gene. Moreover,  expression of the different genes is conditionally induced at 42°C which  provides a strict control of activity.
 
Figure 1: plasmids helper lambda and Flp
The  antibiotic resistance cassettes plasmids (RCP) are plasmids with a resistance  gene flanked by two frt sequences (fig.2). There are two resistance cassette  plasmids, one with a kanamycin resistance marker (RCPKan) and the other with a  chloramphenicol resistance marker (RCPCm)
 
Figure 2 : Resistance cassette kanamycin
The  primers used for the homologous recombination step contain a constant 3’region  and a variable 5’ region (fig.3).  The  sequence of the 5’ variable region depends on the target gene and is homologous  to the sequence flanking the target gene.    The 3’ constant region is the priming sequence for the resistance gene  PCR amplification.
 
Figure 3: Homologous recombination deletion primers
With these elements, a gene deletion can be constructed in three main steps (fig.4).
 
Figure 4 : Deletion procedure
Step 1: PCR
The first step consists of a PCR to produce the  sequence which will replace the target gene. Using the frt-flanked antibiotic  resistance gene carried by the RCP as template together with the primers  designed specifically for the target gene, the PCR product is a sequence which  can be inserted into the E. coli chromosome by homologous recombination with the Red system (fig.5).
 
Figure 5 : Deletion procedure step 1
Step 2: Homologous recombination
An E.  coli strain containing the PH λ is then electroporated with the  purified PCR product obtained in Step 1. After heat induction, Gam, Bet and Exo  catalyze the homologous recombination reaction.
 
Figure 6 : Deletion procedure step 2
Step  3: Antibiotic resistance elimination
After selection on an appropriate selection medium,  the mutant strain is transformed with the PHFlp using the TSS [2] method.  The recombinant candidates are then grown at  42°C.  This allows the elimaination of  the resistance marker by Flp-dependent site-specific recombination as well as  the curing of PHFlp (fig.7).
 
Figure 7 : Deletion procedure step 3
 
Parts Design
 
PH  λ and PHFlp are derived from the Red recombinase plasmid pKD46 [3] .  Our goal was to adapt them to at least one iGEM standard, and to change the  original arabinose inducible promoter controlling the expression of the Red  system to a thermo-inducible promoter.
 
Figure 8 : plasmid pKD46 and priming sites
For  PH λ, the RCF21 standard was selected. The idea was to preserve the  thermosensitive replication origin (repA101 and oriR101) and the ampicillin  resistance of pKD46 while removing the red operon. The pKD46 backbone was amplified using primers flanking the red operon (γ, β, and exo genes) (fig.8),  extended with RCF21 prefix and suffix.   In a second step, we amplified by PCR gam and bet, with primers  containing RCF21 prefixes and suffixes.   The exo gene was synthesised [4] with a silent mutation at the EcoR1 restriction site, to make it  compatible with RCF 21 and 10. The red operon is reconstructed and placed under the control of a thermoinducible  promoter (BBa_K098995),  RBS (B0034)  and followed by a terminator (B0024) using the BioBrick system.
The primer sequences are given below, with RCF21  prefix and suffix in green.
pKD46 :
pKD46_RCF21_for : 5’-cccGGATCCTAACTCGAGCGCATCCTCACGATAATATC-3’
pKD46_RCF21_rev : 5’-cccAGATCTCATGAATTCTATGGCATAGCAAAGTGTGA -3’
γ :
gam_RCF21_for : 5’-cccGAATTCATGAGATCTATGGATATTAATACTGAAAC-3’
gam_RCF21_rev : 5’-cccCTCGAGTTAGGATCCTTATACCTCTGAATCAATAT-3’
β :
bet_RCF21_for : 5’-cccGAATTCATGAGATCTATGAGTACTGCACTCGCAAC-3’
bet_RCF21_rev : 5’-cccCTCGAGTTAGGATCCTCATGCTGCCACCTTCTGCT-3’
Three  C's were added at the 5’-end of prefixes and suffixes to facilitate digestion  by restriction enzymes (endonucleic enzymes).
We  also started from pKD46 to construct PHFlp, except that the RCF10 standard was  selected. The sequence of primers used for the pKD46 PCR were the same except  for the RCF10 prefix and suffix extensions. We also preformed a PCR  site-directed mutagenesis on the pKD46 PCR fragment to eliminate a SpeI  restriction site in the oriR101 sequence. The flp gene was  synthesized to be compatible with RCF10 and 21 standards, then cloned together  with the pKD46 fragment, promoter, rbs and terminator.
The primer sequences are given below, with RCF10  prefix and suffix in green.
pKD46_RCF10_for :  5’-cccTACTAGTAGCGGCCGCTGCAGCGCATCCTCACGATAATATC-3’
pKD46_RCF10_rev :  5’-cccCTCTAGAAGCGGCCGCGAATTCTATGGCATAGCAAAGTGTGA-3’
For  the RCPs, we used primers with R1 and R2 priming sites, extended with RCF10  prefix and suffix, to amplify the kanamycin and chloramphenicol resistance  cassettes. The same primers are used for both resistance cassettes, as the R1  and R2 priming sequences are identical. The templates used are pKD4 [5] for the kanamycin resistance cassette and pKD3 [6] for the chloramphenicol resistance cassette.   The PCR fragments are then inserted in pSB1C3 plasmid backbones.
The primer sequences are given below, with RCF10  prefix and suffix in green.
RCF10prefix_R1 : 5’-cccGAATTCGCGGCCGCTTCTAGAGGTGTAGGCTGGAGCTGCTTC-3’
RCF10suffix_R2 : 5’-cccCTGCAGCGGCCGCTACTAGTACATATGAATATCCTCCTTA-3’
 
Materials and methods
 
The  concentrations of the different antibiotics mentioned are 100µg/l, 50µg/l and 20µg/l, respectively for  ampicillin, kanamycin and chloramphenicol.
 
Resistance gene PCR
 
The  primers are designed by adding a ~ 40 bp homologous sequence (H) to a resistance  gene priming site (R)(fig.9). The homologous sequences are complementary to  sequences flanking the sequence to be deleted.   The forward primer is H1R1 and the reverse primer H2R2.
 
Figure 9 : Target gene, primer H1R1 and resistance cassette
R1  and R2 sequences are given below:
R1 (forward): 5’-GTGTAGGCTGGAGCTGCTTC-3’
R2  (reverse): 5’-CATATGAATATCCTCCTTA-3’
For  instance, here is the sequence of the ldhA gene (encoding D-lactate dehydrogenase) (in blue):
If we  take 40 bp homologous sequence extension (in green), the primers associated to ldha are the following:
H1R1 (forward):
5’-ATCTGAATCAGCTCCCCTGGAATGCAGGGGAGCGGCAAGAGTGTAGGCTGGAGCTGCTTC-3’
H2R2 (reverse):
5’-AGTAGCTTAAATGTGATTCAACATCACTGGAGAAAGTCTTCATATGAATATCCTCCTTA-3’
PCR were performed with the Promega®  PCR kit (catalog # M8305) with 5% DMSO using the following thermocycler  program:
 
Figure 9 : Target gene, primer H1R1 and resistance cassette
PCR  products are then gel-purified. To eliminate the template DNA, digestion with  DpnI and a second gel-purification are performed on the PCR product.
 
Homologous recombination
 
MG1655 containing PHλ  are grown in LB supplemented with ampicillin at 30°C to an OD600 ~ 0.6. The culture is then incubated  with agitation at 42°C for 20-30 min (depending of the volume of culture). For  volume greater than 120 ml, the culture should be split into smaller volumes  (40 ml) to ensure a homogenous heating to 42°C.   Electro-competent bacteria are prepared by cooling the culture on ice  for 20-30 min, followed by a series of centrifugation (4) and wash with  ice-cold water to concentrate bacteria 1000-fold.
50 µl of this suspension are electroporated with  10-100 ng of PCR product.  One ml of LB  is added to the shocked cells. The cells are incubated 1 h at 37°C, then plated  on agar plates with kanamycin or chloramphenicol to select recombinants.
After the first selection, recombinant colonies are  isolated by streaking on agar medium with the appropriate antibiotic, and  incubated 24 h at 37°C. This is done three times, to avoid undisrupted  surnumerous chromosome copies.  After the  third isolation, recombinants are PCR tested for the loss of the targeted  gene.  
 
Resistance gene removal
 
Recombinants are grown in 5ml LB suppelemented  with kanamycin or chloramphenicol at 37°C to an OD600 ~ 0.3.  One ml of culture is centrifuged (3,250 x g,  7 min). The pellet is resuspended in 100 µl of 1 X TSS [7] . 
Mix 100 µl of resuspended pellet with ~100µg of PHFlp,  incubate 30 min on ice, then add 100 µl LB supplemented with kanamycin or chloramphenicol.  Incubate 1 h at 30°C, then plate 50 µl on agar plates with ampicillin and  incubate overnight at 30°C.
Inoculate in LB + ampicillin + kanamycin or  chloramphenicol and incubate overnight at 30°C. Colonies are streaked on agar  plate without antibiotic and incubate overnight at 42°C.
Streak successively on agar plates with ampicillin, on  agar plate with kanamycin or chloramphenicol, and on agar plate without  antibiotics. PCR-test colonies grown on agar plates without antibiotic to check  for resistance removal. If the resistance has been successfully removed and the  PHFlp cured, colonies should grow only on the antibiotic-free medium.
 
PCR verifications
 
The verifications for correct recombination and  resistance gene removal are preformed by PCR, using gene-specific primers. The  primers are designed with priming sites around180-200 bp away from the targeted  gene, so that in case of a successful resistance gene removal, the amplified  PCR fragments would be long enough to be easily detectable in agar gel  electrophoresis.
 
Results and discussion
 
While we  succeeded in deleting genes for the H2 production part, with a  method very similar to the one exposed above, this part of the project remained  mostly theoritical. On the workbench, we first focused on the other parts of  the project, our priority being gene deletions with the tools available.
In addition, we encountered many difficulties in  putting together the different elements in the plasmid backbones. The lack of  time prevented us from achieving any usable parts for the toolbox we proposed.  Even though, we thought it interesting enough to present what we initially  wanted to do.
[1]  One-step inactivation of chromosomal gene  in Escherichia coli K-12 using PCR products, K.A. Datsenko and B.L. Wanner  (2000)
[2] Transformation  and Storage Solution
[3] See http://www.ncbi.nlm.nih.gov/nuccore/15554345?report=genbank for features and complete sequence.
[4]   Synthetized by Mr Gene, http://mrgene.com/desktopdefault.aspx/tabid-2/
[5] http://www.ncbi.nlm.nih.gov/nucleotide/15554332?report=genbank&log$=nucltop&blast_rank=1&RID=CDKUK6A2011
[6] http://www.ncbi.nlm.nih.gov/nucleotide/15554329?report=genbank&log$=nucltop&blast_rank=2&RID=CDM9J2WY016
[7] 1 X TSS =  10% PEG (wt/vol) , 5% (DMSO)vol/vol, MgCl2 50 mM, LB
Hydrogen Production
   
Bacteria planned death system
 
Retrieved from " http://2010.igem.org/Team:ULB-Brussels/Homologous "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
