<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_ULB-Brussels_QAModule"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:ULB-Brussels/QAModule">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:ULB-Brussels/QAModule&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:ULB-Brussels/QAModule&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:ULB-Brussels/QAModule&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:ULB-Brussels/QAModule" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:ULB-Brussels/QAModule</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><UL id="css3menu"><LI class="topfirst"><A href="https://2010.igem.org/Team:ULB-Brussels" title="Home">  Home  </A></LI><LI class="topitem"><A href="https://2010.igem.org/Team:ULB-Brussels/Team" title="Team"><SPAN>  Team  </SPAN></A></LI><LI class="topitem"><A href="https://2010.igem.org/Team:ULB-Brussels/Project" title="Project"><SPAN>  Project  </SPAN></A></LI><LI class="topitem"><A href="https://2010.igem.org/Team:ULB-Brussels/Parts" title="Parts"><SPAN>  Parts  </SPAN></A></LI><LI class="topitem"><A href="https://2010.igem.org/Team:ULB-Brussels/Modeling" title="Modeling"><SPAN>  Modeling  </SPAN></A></LI><LI class="topitem"><A href="https://2010.igem.org/Team:ULB-Brussels/Notebook" title="Notebook"><SPAN>  Notebook  </SPAN></A></LI><LI class="toplast"><A href="https://2010.igem.org/Team:ULB-Brussels/Safety" title="Safety"><SPAN>  Safety  </SPAN></A></LI><LI class="topitem"><A href="https://2010.igem.org/Team:ULB-Brussels/Sponsors" title="Sponsors"><SPAN>  Sponsors  </SPAN></A></LI></UL><DIV align="center"><TABLE border="0"><TBODY><TR><TD><A href="https://2010.igem.org/Team:ULB-Brussels/Homologous">Homologous recombination</A></TD><TD>   </TD><TD><A href="https://2010.igem.org/Team:ULB-Brussels/Modeling">Modeling</A></TD></TR></TBODY></TABLE></DIV><DIV align="center"><GH3><P>Quorum Addiction</P></GH3></DIV><P><GH4>Table of content</GH4></P><UL><LI><A href="#UA">List of used abreviations</A></LI><LI><A href="#I">Introduction</A></LI><LI><A href="#MM">Materials and methods</A></LI><LI><A href="#AGS">Assembly of genetic sequences</A></LI></UL><LI><A href="#R">Results</A></LI><LI><A href="#A">Annex</A></LI><LI><A href="#PS">Primers sequences
        </A></LI><LI><A href="#AP">Antidote plasmid</A></LI><LI><A href="#PP">Poison plasmid </A></LI><GH4><P> </P><P>List of used abreviations</P></GH4><TABLE border="0" cellspacing="0" cellpadding="0"><TBODY><TR><TD width="218" valign="top"><P align="center"><STRONG>Abreviation</STRONG></P></TD><TD width="414" valign="top"><P align="center"><STRONG>Signification</STRONG></P></TD></TR><TR><TD width="218" valign="top"><P align="center">HSL</P></TD><TD width="414" valign="top"><P align="center">3-oxo-hexanoyl-homoserine    lactone</P></TD></TR><TR><TD width="218" valign="top"><P align="center">aiiA</P></TD><TD width="414" valign="top"><P align="center">HSL degrading protein (BBa_C0060)</P></TD></TR></TBODY></TABLE><GH4><P>Introduction</P></GH4><P>In our  project, it was important to make sure that Hydrocoli would not be released  into the environment of the sewage treatment plant. The QUORUM ADDICTION MODULE ensures that bacteria escaping the sewage treatment plant are  killed.  </P><P><EM>Concept  used </EM><EM>Quorum sensing : </EM>The quorum sensing is a mechanism  used by bacteria to perceive their own concentration,  or the concentration of other species in their environment. To do that, the  bacteria secretes homoserine lactones which vary  depending on the bacterial species. These molecules control the expression of  hundreds of genes such as genes involved in  biofilm formation or virulence factor.  In the Hydrocoli project, we used  two genes belonging to the <EM>lux</EM> operon  of <EM>Vibrio fischeri</EM> (<EM>luxR</EM> and <EM>luxI</EM>). <EM>The parDE poison-antidote system : </EM>Poison-antidote systems are composed  of 2 genes; one encoding a toxic protein and the other encoding its cognate  antidote.  These systems are abundant in  bacterial genomes, although their role in bacterial physiology is still  uncertain.  When they are located on  plasmids, these systems contribute to plasmid stability.  
  The  Quorum addiction module is based on two plasmids that ‘interact’ with each  other. These plasmids, once integrated into the bacterial strain, allow the  bacteria to survive in condition in which the bacterial density is high enough.  In condition in which the bacterial density decreases, a toxin gene is  expressed and the toxin will eventually lead to cell death.</P><P>For the  construction of this module, BioBricks of cell-cell signaling pathways were  used as well as the well-characterized <EM>parDE</EM> poison-antidote system.</P><P>The first  construction was named the <U>antitoxin plasmid</U>.  It contains the ‘HSL-LuxR activated’ promoter  (BBa_R0062), the <EM>luxI</EM> gene  (BBa_C0061), the <EM>luxR</EM> gene  (BBa_C0062), the <EM>parD</EM> (antitoxin)  gene and a double terminator sequence (BBa_B0015). </P><P>The second  construction was named the <U>toxin plasmid</U>.  It contains the ‘HSL-LuxR  repressed’ promoter (BBa_R0061), the <EM>aiiA</EM> gene (BBa_C0060), the <EM>parE</EM> (toxin)  gene and a double terminator sequence (BBa_B0015).</P><P><EM>In the sewage treatment plant:</EM>
  When  bacteria are in the sewage treatment plant, the genes of the antitoxin plasmid are expressed to a basal level and  the LuxI, LuxR and ParD proteins are produced.   Expression of LuxI leads to HSL production.  As HSL is able to diffuse ‘in and out’ of the  bacterial cell, both the ‘outside’ and intracellular HSL concentration will be  high.  This will have an opposite effect  on the antitoxin and toxin plasmids.  On  the one hand, the LuxR-HSL complex binds to the ‘HSL-LuxR activated’ promoter and activates the  transcription of <EM>luxI</EM>, <EM>luxR</EM> and <EM>parD</EM> genes located on the antitoxin  plasmid.  This is a positive  feedback loop. More and more HSL, LuxR and ParD are produced.  On the other hand,  the same LuxR-HSL complex has an opposite effect on expression of the genes  encoded by the toxin plasmid.  It binds  to the ‘HSL-LuxR repressed’ promoter and inhibits the <EM>aiiA</EM> and <EM>parE</EM> gene expression.   In  these conditions, bacteria are growing ‘happily’ and do their job in the sewage  treatment plan</P><P><EM>Out of  the sewage treatment plant:</EM>
  If bacteria escape  out of the sewage treatment plant and end up in the outside environment, the  HSL concentration will immediately drop.   This will lead to a drastic drop of the intracellular HSL concentration  and therefore, expression of the antitoxin plasmid genes will be inhibited  while that of the toxin plasmid genes will be activated.  The ParE toxin will be produced as well as  AiiA, an HSL degrading enzyme (<EM>aiiA</EM>, (BBa_C0060)).   The ParE toxin  will kill the bacteria and the AiiA enzyme will ensure a complete removal of  HSL.  </P><P><GH4>Materials and methods </GH4></P><P> </P><P><GH5>Assembly of genetic sequences</GH5></P><P>Given the many parts to be assembled, a different technique from the  usual technique was used. The assembly of the two constructions was performed  by PCR.  Primers have been designed to &quot;stick&quot; together two genes (see  primer sequences in annex).
The  first step was to amplify each part separately. </P><P> </P><P>Here (at the top  and bottom of the image) the oligos of the first step “stick” to the gene of  interest (in orange in the center). After this first PCR we obtain the gene of  interest able to “stick”together in the right order. Indeed,  each gene has a sticky end with one upstream and another with the downstream.
  For the second step  we put all genes involved in the construction together and adding just 2 oligos  (one with préfix+beginning of the promotor (5’-3’) second with suffix+end of  the terminator (3’-5’)) for replicate by PCR all the construction.</P><P> </P><GH4>Results </GH4><P> </P><P>As  explained above we started our PCR assembly. The first gel below shows the PCR  products (first step) obtained for <EM>luxI</EM>, <EM>luxR</EM>, <EM>parD</EM>, <EM>parE</EM> and the double  terminator. The second gel shows the PCR product for <EM>aiiA</EM>. </P><P>PCR  amplification of the individual  parts.  
  1.  SMART ladder ; 2. Lux pR Promoter ; 3. Lux pR Promotor negative control;  4. LuxI ; 5. LuxI negative control ; 6. LuxR ; 7. LuxR negative  control ; 8. ParD ; 9. ParD negative control ; 10. Double  terminator ; 11. Double terminator negative control  ; 12.  iiai ; 13. iiai negative control ; 14. ParE ; 15. ParE negative  control ; 16. SMART ; 17. SMART ; 18. Double terminator ;  19. Double terminator negative control.</P><P> </P><P>A gel purification was then performed in order to  extract the DNA fragment before the second step of PCR assembly.</P><P>We were able to obtain PCR amplification  
  Despite numerous attempts, it was impossible to  amplify the promoter by PCR (HSL-LuxR activated (BBa_R0062)). We don't know why it failed, although we tried several  annealing temperature and primer concentrations. Because of this problem  we couldn't put together the genes of the first construction. As such, it was  impossible to proceed with transformation and measurements.
  In contrast, for the second construction we  succeeded in the assembly of all the genes   (see gel below).</P><P>However, due to its inherent  toxicity, the second construction cannot be transformed in bacteria, the first  construction is a prerequisite of the second construction's implementation. </P><P> </P><GH4>Annex </GH4><P> </P><GH4>Primer  sequences</GH4><P> </P><GH5>Antidote plasmid </GH5><P> </P><P><STRONG>‐ Prefix + promoter :</STRONG></P><P>CCCCGAATTCGCGGCCGCTTCTAGAGACCTGTAGGATCGTACAGG</P><P><STRONG>‐ Promoter+RBS+LuxI :</STRONG></P><P>GTTTGTTATAGTCGAATAAAAAAGAGGAGAAAATGACTATAATGATAAAAAAATCGCGATTTTTTTATCATTATAGTCATTTTCTCCTCTTTTTTATTCGACTATAACAAAC</P><P><STRONG>‐ LuxI + RBS + LuxR :</STRONG></P><P>CGCTTTAGTAGCTTAATAAAAAGAGGAGAAAATGAAAAACATAAATGCCGCGGCATTTATGTTTTTCATTTTCTCCTCTTTTTATTAAGCTACTAAAGCG</P><P><STRONG>‐ LuxR + RBS + ParD :</STRONG></P><P>CATACTTTAAAAATTAATAAAAAGAGGAGAAAATGAGCCGCCTGACAATCGCGATTGTCAGGCGGCTCATTTTCTCCTCTTTTTATTAATTTTTAAAGTATG</P><P><STRONG>‐ ParD + double terminator :</STRONG></P><P>CAGCGGGGATCGCGCTTGACCAGGCATCAAATAAAACGCGTTTTATTTGATGCCTGGTCAAGCGCGATCCCCGCTG</P><P><STRONG>‐ Double terminator + suffix :</STRONG></P><P>CCCCCTGCAGCGGCCGCTACTAGTATATAAACGCAGAAAGGCCC</P><P> </P><GH5>Poison plasmid</GH5><P> </P><P><STRONG>‐ Prefix + promoter +RBS + aiiA :</STRONG></P><P>CCCCGAATTCGCGGCCGCTTCTAGAGTTGACACCTGTAGGATCGTACAGGTATAATAAAGAGGAGAAAATGACAGTAAAGAAGCTTTATTTCG</P><P><STRONG>- aiiA + RBS + ParE:</STRONG></P><P>CGCTTTAGTAGCTTAATAAAAAGAGGAGAAATTGACGGCCTACATCCTCACGTGAGGATGTAGGCCGTCAATTTCTCCTCTTTTTATTAAGCTACTAAAGCG</P><P><STRONG>‐ParE+doubleterminator :</STRONG></P><P>CCGACAGGCTCAAGGGCTGACCAGGCATCAAATAAAACGCGTTTTATTTGATGCCTGGTCAGCCCTTGAGCCTGTCGG</P><P><STRONG>- Doubleterminator+suffix : </STRONG></P><P>CCCCCTGCAGCGGCCGCTACTAGTATATAAACGCAGAAAGGCCC</P><DIV align="center"><TABLE border="0"><TBODY><TR><TD><A href="https://2010.igem.org/Team:ULB-Brussels/Homologous">Homologous recombination</A></TD><TD>   </TD><TD><A href="https://2010.igem.org/Team:ULB-Brussels/Modeling">Modeling</A></TD></TR></TBODY></TABLE></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:ULB-Brussels/QAModule">http://2010.igem.org/Team:ULB-Brussels/QAModule</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:ULB-Brussels/QAModule" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:ULB-Brussels/QAModule" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:ULB-Brussels/QAModule&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:ULB-Brussels/QAModule&amp;oldid=195750" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>