"
Page
Discussion
View source
History
teams
Log in
 
Team:ULB-Brussels/H2
From 2010.igem.org
  Home  
  Team  
  Project  
  Parts  
  Modeling  
  Notebook  
  Safety  
  Sponsors  
Introduction
   
Homologous recombination
Hydrogen production
E. coli naturally produces hydrogen through the mixed  acid fermentation. Our goal in this part of the project is to increase the H2  production by modifying the carbon flow through the pathway of the mixed acid  fermentation (MAF). The pathways of this fermentation are quite simple.  Phosphoenolpyruvate (PEP), which is produced by the glycolysis, is transformed  into pyruvate or into oxaloacetate. Those two molecules are then transformed  into several products after a chain of reactions. Hydrogen is one of those  products. It is produced from formate to regulate the intracellular pH of the  bacteria [1] .
Here is a simplified global overview  of the mixed acid fermentation (fig.1):
Fig. 1 : Mixed acid fermentation pathway [2] .
 Each reaction is controlled by  one or several enzymes. By overexpressing or deleting specific genes of the MAF  pathway, we expect to increase the hydrogen production. The part of MAF pathway in which we are  interested is indicated by red lines (fig. 2).
Fig. 2: The interesting pathway for hydrogen production via the mixed acid fermentation.
All the reactions crossing the red lines should be avoided or limited, and the three main reactions which lead to hydrogen production should be increased. The hydrogen is produced from formate by the formate hydrogenase lyase system (FHL), an intracellular membrane-bound complex composed of the formate dehydrogenase (FDHH) and the hydrogenase 3 [1] . This complex catalyzes the following reaction:
Formate + H+ <=> H2 + CO2
Hydrogen is not excreted outside of the cell but  it is immediately consumed by two uptake hydrogenases, the hydrogenase 1 and 2 [1] [2] :
A menaquinone  + H2 + 2H+ <=> A menaquinol + 2H+
The expression of the FHL system is activated at  the level of transcription by the FhlA activator (formate hydrogen-lyase  transcriptional activator). Expression of fhlA is positively regulated by FNR and repressed by HycA [1] . FNR is the main  transcriptional regulator that mediates the transition between aerobic and  anaerobic growth through the regulation of hundreds of genes [2] . All the  regulations linked to the transformation of formate into hydrogen are shown  here (fig.3):
Fig. 3 : Regulation of hydrogen production in E.coli [1] .
 
MAF pathway modification: genes to delete
Several steps of the MAF pathway are irrelevant for hydrogen production (fig. 2). To inactivate these reactions, the genes encoding enzymes that catalyze these reactions have to be deleted using the technique described in the homologous recombination module. After exhaustive analyzes of the literature, we decided which genes we would overexpress and which we would disrupt.  Here is the list of all the genes that we planned to delete. By using the KEIO Collection database, we checked that none of those genes were essential [4] .
ppc
The ppc gene encodes the phosphoenolpyruvate carboxylase enzyme (Ppc). This enzyme  catalyzes the following reaction [2] :
Phosphoenolpyruvate + CO2 + H2O <=> Oxaloacetate  + Phosphate + H+
Deletion of the ppc gene will therefore prevent the PEP to be transformed into  oxaloacetate which is not interesting for our application. As shown on figures  1 and 2, the left part of the MAF pathway does not lead to hydrogen production.  Thus, deletion of ppc will inactivate  this part of the pathway and all the PEP will be transformed into pyruvate.
ldhA
Pyruvate is transformed into  (R)-lactate or into formate + acetyl-CoA. The second reaction is interesting  for our application. The first reaction is mediated by the D-lactate  dehydrogenase (LdhA) which catalyzes the following reaction:
NADH + Pyruvate + H+ <=> NAD+ + (R)-lactate
Deletion of ldhA should ensure that all the pyruvate is transformed into formate + acetyl-CoA.
focA
The focA gene encodes for a formate transporter located in the plasmic membrane. FocA  transports formate outside the cell. It has been shown that disabling this transporter  increases the intracellular formate concentration [5] . A higher formate intracellular concentration is  obviously interesting for hydrogen production.
hyaB and hybC
To prevent hydrogen uptake by the two hydrogenases, HyaB and HybC, we have to inactivate them. Both hydrogenases-encoding genes are located in operons.  The hya operon is composed of 6 genes (from hyaA to hyaF). The hyaB gene encodes the largest subunit of the hydrogenase [2] . This subunit is  required for the hydrogenase to be functional [2] . Deleting this gene should inactivate the hydrogenase 1 [1] .
The second hydrogenase-encoding gene is located in an operon composed of 7 genes (from hybA to hybG). The hybC gene encodes the largest subunit of the second hydrogenase. As for hyaB, deleting the gene encoding the largest subunit should inactivate it [1] .
 
Materials and methods
 
Gene  deletion by the λ phage Red system
In order to delete  the five genes listed above, we used a method based on the λ phage Red system. As  this method is described in great details in the ‘homologous recombination  module’, we will briefly summarize it here. This method consists in the  production of a gene-specific PCR fragment which contains an antibiotic  resistance gene and site-specific recombination sites (FRT) flanking this gene.  This PCR fragment is then electroporated in a strain containing the plasmid  helper λ (PH λ) (see homologous recombination module). This plasmid encodes the  genes of the Red system. The target gene is replaced by the resistance gene  through homologous recombination using the Red system. We worked with two  different antibiotic resistances: chloramphenicol (cm) and kanamicine (kan).
The antibiotic resistance  cassette is eventually removed by site-specific recombination using the Flp  recombinase. This method leads to the ‘clean’ deletion of the gene of interest  without any traces of antibiotic resistance gene. It is also adapted to delete  genes in operons since the resistance-FRT cassettes used in this work also contain  translation signal (RBS). This allows a proper expression of the genes  downstream of the deletion (no polar effect). The  primers used deleting the 5 genes of interest are listed in Table 1. 
Phage P1  transduction
P1vir stock :
Overnight cultures of the donor strain are  diluted 100 fold in 20 ml of LB medium supplemented with MgSO4 10-2M  and of CaCl2 5.10-3M and grown at 37°C. At an optical density at  600nm of 0.2, 10µl of a P1 vir stock is added (approximately 109  pfu/ml). Once the bacteria are lysed, 1/20 (v/v) of chloroform is added and the  culture is vigorously shacked. The mix is centrifuged at 4°C at 3,700 g and the  supernatant is conserved at 4°C. 1/20 of the total volume of chloroform is  added and vigorously shacked, then centrifuged again in the same conditions.  The surnageant containing the P1 particles is stored at 4°C.
P1vir  transduction :
Overnight culture of the recipient strain is  pelleted and resuspended in the same volume of MgSO4 10-2M and of  CaCl2 5.10-3M. To 100µl of the resuspended pellet, 100µl and 10µl of  the P1 lysat is added. The mix is conserved 30 min at 37°C. 3 ml of agar 0.7%  is added as well as 100µl of Na citrate 1M. The entire mix is plated on  selective plates at 37°C for 16h. The candidates are streaked 3 times on  selective plates and a PCR is then performed to confirm the transduction of the  marker of interest. 
 
Results
 
Deletion  of the ldhA gene by homologous recombination
The amplification of the resistance cassettes with  the primers specific to our 5 genes of interest by PCR took us a little more  than a week. Because of the size of the primers used, we had to try several  times the PCR and vary the parameters.
Here is the picture of the electrophoresis of  those PCR for the 5 genes:
For each genes we tried to obtain both  chloramphenicol (about 1100bp) and kanamicine (about 1500bp) resistance  cassettes. From left to right we have the genes ldhA (1 and 2), focA (4  and 5), ppc (7 and 8), hybC (10 and 11) and hyaB (13 and 14). The sinks 3, 6, 9, 12  and 15 are the five negative controls.
        Once  we obtained these cassettes, it took us about 1 month to obtain our first  deletion candidate. The electroporation efficiency of the MG1655/PHl strain was not good. We obtained a lot of false-positive deletion  candidates (i.e. clones growing on the selective medium). As explained in the homologous  recombination module, we had to purify at least three times each candidate  (this process takes therefore at least 3 days) before checking the  deletion/insertion of the resistance cassette. Using primers complementary to  the flanking regions of the genes of interest, PCR were performed on candidate  colonies (data shown below). Unfortunately, the candidates were most of the time  wild-type for the gene of interest. Our hypothesis is that the resistance  cassette was inserted at another location in the genome. The reason for that is  unknown since the genes of interest are not essential for E. coli viability, at least in the conditions we used. 
After  several weeks of unsuccessful attempts, we finally obtained the MG1655 DldhA::cm strain.  Here is the picture of the  electrophoresis where we tested different deletion candidates:
The sinks we are interested in are the 4 first  ones. The wild type version of the gene ldhA is in the third sink. The two first sinks are the deletion candidates for ldhA with the cm resistance cassette. The fourth sink is the negative  control for this gene. For information, the other sinks are the tests for the 4  other genes: 5 and 6 are the deletion candidates for ppc; 9 and 10 are the  deletion candidates for focA; 17 and 18 are the deletion candidates for hybC:  21 and 22 are the deletion candidates for hyaB. Unfortunately, the cm  resistance cassette (about 1100 bp) and the gene ldhA (990 bp) have approximately the same length. In order to  testify if we really had the MG1655 DldhA::cm strain,  we did a test with a restriction enzyme which cut the gene ldhA.
The  result of this test is shown here:
After the purification process, we had 4 candidates  for the MG1655 DldhA::cm strain. For those 4 candidates, we did a PCR on  the gene ldhA (not knowing if it  really was ldhA or the cm resistance  cassette that we amplified) and we digest the product of the PCR with an enzyme  which cut ldhA. The digested gene ldhA is in the sink 7, the digested cm  resistance cassette is in the sink 6. The four digested candidates are in the 4  first sinks. The result is therefore a success, because the candidates are not  digested like ldhA. The wild-type  version of the gene has been successfully replaced with the cm resistance  cassette. We have our MG1655 DldhA::cm strain.
Construction  of the MG1655 Dppc::kan, MG1655 DfocA::kan, MG1655 DhyaB::kan, MG1655 DldhA::cm Dppc::kan, MG1655 DldhA::cm DfocA::kan andMG1655 DldhA::cm DhyaB::kan mutant strains by P1  transduction
By  that time, we obtained the KEIO strains deleted for our 5 genes of interest. We  were thus able to make a P1 stock on these strains (see materials and methods)  and to transduce the Dppc::kan, DfocA::kan, DhyaB::kan and DhybC::kan markers in the MG1655 wild-type strain and in the MG1655 DldhA::cm strain.  We obtained 6 mutant strains: the single MG1655 Dppc::kan, MG1655 DfocA::kan and MG1655 DhyaB::kan mutant strains as  well as the double MG1655 DldhA::cm Dppc::kan, MG1655 DldhA::cm DfocA::kan andMG1655 DldhA::cm DhyaB::kan mutant strains.  These strains were checked by PCR. 
The results are shown here:
 
For  each strain tested, we have: four candidates, one negative control and the wild  type version of the gene that is supposed to have been deleted. If the  amplified DNA segment of a candidate has the size of the reistance cassette instead  of the size of the wild type version of the gene, the candidate is a mutant.  Here are the results:
From 1 to  6 is the test for the MG1655 DhyaB::kan mutant:  the 4 candidates are mutants.
From 8 to 13 is the test for the MG1655 DldhA::kan mutant: the  candidates in the sinks 10 and 11 are mutants.
From 17 to 22 is the test for the gene ldhA of the double MG1655 DldhA::cm Dppc::kan mutant: the 4  candidates are mutants.
From 24 to 29 is the test for the gene ldhA of the double MG1655 DldhA::cm DfocA::kan mutant:  the 4 candidates are mutants.
From 33 to 38 is the test for the gene focA of the  double MG1655 DldhA::cm DfocA::kan mutant:  the candidates in the sinks 35 and 36 are mutants.
From 40 to 45 is the test for the gene ppc of the double MG1655 DldhA::cm Dppc::kan mutant: the  candidates in the sinks 40, 42 and 43 are mutants.
        We  managed to obtain different double mutant strains, ready for another  transduction. Unfortunately, due to the lack of time we were unable to continue  our progression and we were also unable to characterize the double mutant strains for  growth in different medium, at different temperature, …
 
MAF pathway modification: Genes to overexpress
 
In order to enhance  hydrogen production, we proposed to over-express some genes involved in the  hydrogen production. Three steps in the mixed acid fermentation pathway are of  interest:
The  transformation of phosphoenolpyruvate into pyruvate:
Phosphoenolpyruvate + ADP + 2H+ <=> Pyruvate + ATP
This reaction is present in  other pathway. We thus decided not to improve it assuming it will not be the  limiting step.
The  transformation of pyruvate into formate:
Pyruvate + Coenzyme A <=>  Formate + Acetyl-CoA
This reaction is catalyzed  by two pyruvate formate lyase: PflB and TdcE. The formate is the key molecule  for hydrogen production and intracellular elevated formate concentration is  needed for an efficient hydrogen production [6] .
The  transformation of formate into hydrogen:
Formate + H+ <=>  CO2 + H2
This reaction is under the  control of the FHL complex (formate hydrogenase lyase composed of the formate dehydrogenase  (FDHH) and the hydrogenase 3). As stated above, the FHL complex is  regulated by FNR, a transcriptional regulator that mediates the transition  between aerobic and anaerobic growth, and repressed by HyaC. It has been showed  that FHL activity is not the limiting step for hydrogen production. We decided  not to over-express the two enzymes of the FHL complex [3] .
On the basis of these three  reactions, we decided to overexpress the following genes:
fnr
        fnr is quite an obvious choice for the first gene to overexpress.  FNR is the main transcriptional regulator that mediates the transition from  aerobic to anaerobic growth [2] . The concentration of the FNR protein is similar  under aerobic and anaerobic conditions. The regulation occurs at the level of FNR  activity: in the presence of O2, FNR is inactivated. Under anaerobic  conditions, FNR is functional and activates expression of hundreds of genes  involved in anaerobic metabolism. In these conditions, FNR represses genes  involved in aerobic metabolism [2] . Over-expressing the fnr gene should increase FNR concentration and should thus enhance  the anaerobic metabolism. The three reactions of the mixed acid fermentation pathway  leading to hydrogen described above should therefore be increased.
pflB
        The pflB gene encodes one of the two pyruvate formate lyases which  catalyze the transformation of pyruvate into formate [2] . This reaction is the  most important one for hydrogen production. Over-expressing pflB should increase formate concentration  and therefore hydrogen production. Although pyruvate is the substrate for many  other reactions, high level of PflB should direct pyruvate to formate synthesis  and therefore hydrogen production.
tdcE
The tdcE gene encodes the second pyruvate formate lyase [2] . The reasons to overexpress  this gene are the same as for pflB.
Materials  and methods
To over-express these three  genes of interest, we decided to insert them into plasmid pSB1C3 with a strong  RBS, a constitutive promoter and a terminator. We decided to work in the  BioBrick standard assembly 10 because it is compatible with pSB1C3. 
First of all, we had to  obtain the three genes to ligate them with the RBS, the promoter and the  terminator. Unfortunately, the fnr gene contains an EcoR1 restriction site. Therefore, we had to introduce a  silent mutation in the sequence of this restriction site and ordered the  modified fnr on mrgene.com.
The tdcE gene was obtained by PCR using specific primers and the wild-type  genome of E. coli.
The pflB sequence contains a restriction enzyme present in the standard  assembly 10: Pst1. We could have also introduced a silent mutation in this sequence  and ordered it on mrgene.com. However, due to the size of the pflb gene (2283 bp) and our limited  budget, we were not able to afford it.
The RBS, the promoter and  the terminator that were selected are the same for the three genes and are  described as being the ‘strongest’ ones received in spring 2010 distribution.
The promoter is: BBa_J23100
The RBS is: BBa_B0034
The terminator is: BBa_B0024
After obtaining the gene  sequences either by PCR or by in vitro synthesis, the plan was to ligated them into the plasmid containing the RBS  opened with Spe1 and Pst1 in order to clone the genes downstream of the RBS.  The genes were also digested by Xba1 and Pst1. The second step was to ligate  the part RBS + gene with the promoter. The third step consisted in extracting  the construction promoter + RBS + gene and inserting upstream of the  terminator. The last step was to extract the promoter+RBS+gene+terminator and  insert it into the pSB1C3 plasmid.
Unfortunately, due to lack  of time, we were unable to do all these constructs.
We managed to obtain the  gene tdcE by PCR as we can see here:
The gene tdcE is in the sink 3  and 5. The two sinks correspond to two different PCR programs. The sinks 2 and  4 are the negative controls for those PCR.
           We digested tdcE and inserted it in the pSB1C3 to  send it as a BioBrick: BBa_K348000.
Genes of the MAF pathway that we did not consider
 
Several other genes of the MAF pathway could have been potential  candidates for modifications (deletion or overexpression). However, after  exhaustive literature readings, we did not consider them. 
hycA
        This gene encodes HycA, a  regulator of the hydrogenase 3 in the FHL system [2] . This regulator limits the  amount of hydrogenase 3 in the cell. Deleting hycA should therefore lead to an increase of the concentration of  hydrogenase 3. Note that as stated above, the hydrogenase 3 catalyzes the  transformation of formate into hydrogen. We did not consider this candidate  since transformation of formate into hydrogen do not appear to be the limiting  step for hydrogen production and normal concentration of hydrogenase 3 is fully  adequate [3] .
aceE
        This gene encodes a subunit  of the pyruvate dehydrogenase complex. This complex catalyzes the following  reaction [2] :
Pyruvate + NAD+ +  coenzyme A <=> Acetyl-CoA + CO2 + NADH
For hydrogen production,  transformation of pyruvate into formate is needed. The AceE enzyme catalyzes a  variety of reactions, and deleting the corresponding gene reduces E. coli viability [1] . We therefore  decided not to delete this gene. We focused on the transformation of pyruvate  into formate and to overexpress the tdcE gene (see above).
poxB
        The poxB gene encodes a pyruvate oxydase, which catalyzes the  transformation of pyruvate into acetate and CO2 [1] . It has been  described that deleting poxB does not  increase hydrogen production [1] .
focB
        FocB is a formate  transporter (the other formate transporter is FocA, see above). Unlike FocA, FocB  is less efficient and less specific to formate. We could have deleted focB as well but deleting the two formate  transporter would have reduced the growth rate [1] .
fdoG and fdnG
       
fdoG encodes the formate dehydrogenase-O  and fdnG the formate dehydrogenase-N.  Those two formate dehydrogenase catalyze the transformation of formate into CO2  + H+. The formate dehydrogenase-O is active under aerobic  conditions. We can therefore avoid deleting fdoG simply because our bacteria will be used under anaerobic conditions. The  formate dehydrogenase-N needs a source of nitrate to be active. In the  laboratory, it is easy to work on a medium which does not contain nitrogen and  we can avoid deleting this gene. In reality, in a sewage treatment facility,  this could be much more problematic. We can propose that our bacteria will be  used to produce hydrogen after the nitrogen removal step. Deleting fdnG would be necessary if we want to  produce hydrogen in a medium that contains nitrogen. Moreover, the formate  dehydrogenase-N is a much more effective enzyme than the FHL system. Most of  the formate will thus be consumed by the formate dehydrogenase-N instead of  being transformed into hydrogen.
Table  1
All the sequences are  listed in the directionality 5' -> 3'.
PPC
H1R1: ACCCTCGCGCAAAAGCACGAGGGTTTGCAGAAGAGGAAGAGTGTAGGCTGGAGCTGCTTC
H2R2: ACAGGGCTATCAAACGATAAGATGGGGTGTCTGGGGTAATCATATGAATATCCTCCTTA
LdhA
H1R1: ATCTGAATCAGCTCCCCTGGAATGCAGGGGAGCGGCAAGAGTGTAGGCTGGAGCTGCTTC
H2R2: AGTAGCTTAAATGTGATTCAACATCACTGGAGAAAGTCTTCATATGAATATCCTCCTTA
FocA
For FocA, we didn't  try to delete the entire gene because FocA is located next to pflB (one of the  genes useful for hydrogen production, see above). The RBS and the promoter of  pflB are included into the sequence of FocA. We decided to remove only the half  of FocA, leaving the RBS and promoter of pflB untouched.
H1R1: GCTGCGGCCAGAATAACTCATCCATACTGCCAGACATACCGTGTAGGCTGGAGCTGCTTC
H2R2: GTTAGTATCTCGTCGCCGACTTAATAAAGAGAGAGTTAGTCATATGAATATCCTCCTTA
HyaB
H1R1: CAGAAACCGAACATCAGCCAGGCAATGAGGATAAACAGGCGTGTAGGCTGGAGCTGCTTC
H2R2: CGTTGTCGCTTTTCTGTTGCATGATGATTCTCCTTCGCTGCATATGAATATCCTCCTTA
HybC
H1R1: GTCAGCAAAATATTGCCGACCCCTAAGACTAAAATACGCAGTGTAGGCTGGAGCTGCTTC
H2R2: TAAAACAAAACGATCATAATCGTCATGAGGCGAGCAAAGCCATATGAATATCCTCCTTA
 
References
[1] Maeda, T.; Sanchez-Torres, V.; K.  Wood, Thomas. - Enhanced hydrogen production from glucose by metabolically  engineered Escherichia coli - Appl Microbiol Biotechnol (2007) 77:879–890 –  September 2007.
[2] The  database Ecocyc for the bacterium Escherichia coli K-12 MG1655,  consulted online in July, August and September 2010: http://ecocyc.org/
[3] Node, K.; Watanabe, M.;  Makimoto, H.; Tomiyama, M. - Effect of hydrogenase 3 over-expression and  disruption of nitrate reductase on fermamentive hydrogen production in Escherichia  coli
[4] The KEIO Collection consulted  online in July, August and September 2010: http://ecoli.naist.jp/gb6/Resources/deletion/deletion.html
[5] Suppmann, B.; Sawers, G. - Isolation  and characterization of hypophosphite--resistant mutants of Escherichia coli:  identification of the FocA protein, encoded by the pfl operon, as a putative  formate transporter - Lehrstuhl für Mikrobiologie der Universität München,  Germany. - Mol Microbiol. 1994 Mar;11(5):965-982. – March 1994.
[6 ] Sawers, R.G. - Formate and its  role in hydrogen production in Escherichia coli - Biochemical Society  Transactions (2005) Volume 33, part 1 – Pages 42-46 – September 2004.
Introduction
   
Homologous recombination
 
Retrieved from " http://2010.igem.org/Team:ULB-Brussels/H2 "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
