<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Washington_Aptazyme skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Washington/Aptazyme</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="menuContainer"><UL><LI><A href="https://2015.igem.org/Team:Washington">Home</A></LI><LI><A href="https://2015.igem.org/Team:Washington/Auxin">Auxin  
                    </A></LI><LI><A href="https://2015.igem.org/Team:Washington/Auxin#Introduction">Introduction</A></LI><LI><A href="https://2015.igem.org/Team:Washington/Auxin#Methods">Methods</A></LI><LI><A href="https://2015.igem.org/Team:Washington/Auxin#Results">Results</A></LI><LI><A href="https://2015.igem.org/Team:Washington/Auxin#Conclusion">Conclusion</A></LI><LI><A href="https://2015.igem.org/Team:Washington/Auxin#Future_Direction">Future Direction</A></LI><LI><A href="https://2015.igem.org/Team:Washington/Parts#Auxin">Biobrick</A></LI></UL><LI><A href="https://2015.igem.org/Team:Washington/Aptazyme">Aptazyme
                  </A></LI><LI><A href="#" class="scroll-link" data-id="Introduction">Introduction</A></LI><LI><A href="#" class="scroll-link" data-id="Methods">Methods</A></LI><LI><A href="#" class="scroll-link" data-id="Results">Results</A></LI><LI><A href="#" class="scroll-link" data-id="Conclusion">Conclusion</A></LI><LI><A href="https://2015.igem.org/Team:Washington/Parts#Aptazyme">Biobrick</A></LI></DIV><LI><A href="https://2015.igem.org/Team:Washington/Paper_Device">Paper Device               
                    </A></LI><LI><A href="https://2015.igem.org/Team:Washington/Paper_Device#Introduction">Introduction</A></LI><LI><A href="https://2015.igem.org/Team:Washington/Paper_Device#Methods">Methods</A></LI><LI><A href="https://2015.igem.org/Team:Washington/Design">Design</A></LI><LI><A href="https://2015.igem.org/Team:Washington/Paper_Device#Results">Results</A></LI><LI><A href="https://2015.igem.org/Team:Washington/Paper_Device#Conclusion">Conclusion</A></LI><LI><A href="https://2015.igem.org/Team:Washington/Modeling">Modeling               
                    </A></LI><LI><A href="#" class="scroll-link" data-id="Paper_Device">Paper Device</A></LI><LI><A href="#" class="scroll-link" data-id="Aptazyme">Aptazyme</A></LI><LI><A href="https://2015.igem.org/Team:Washington/Practices">Human Practices               
                    </A></LI><LI><A href="https://2015.igem.org/Team:Washington/Practices#Outreach">Outreach</A></LI><LI><A href="https://2015.igem.org/Team:Washington/Practices#Integrated">Integrated</A></LI><LI><A href="https://2015.igem.org/Team:Washington/Protocols">Protocols
                    </A></LI><LI><A href="https://2015.igem.org/Team:Washington/Protocols#Experiments">Experiments</A></LI><LI><A href="https://2015.igem.org/Team:Washington/Protocols#Safety">Safety</A></LI><LI><A href="https://2015.igem.org/Team:Washington/Team">Team                         
                    </A></LI><LI><A href="https://2015.igem.org/Team:Washington/Team#Members">Members</A></LI><LI><A href="https://2015.igem.org/Team:Washington/Attributions">Attributions</A></LI><LI><A href="https://2015.igem.org/Team:Washington/Team#Sponsors">Sponsors</A></LI><LI><A href="https://2015.igem.org/Team:Washington/Team#Judging_Form">Judging Form</A></LI></DIV><DIV id="main"><DIV class="container clearfix"><DIV id="sidebar"><NAV><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav" style="list-style-type: none;"><LI><A href="#" class="scroll-link" data-id="Introduction">Introduction</A></LI><LI><A href="#" class="scroll-link" data-id="Methods">Methods</A></LI><LI><A href="#" class="scroll-link" data-id="Results">Results</A></LI><LI><A href="#" class="scroll-link" data-id="Conclusion">Conclusion</A></LI><LI><A href="#" class="scroll-link" data-id="Future_Direction">Future Direction</A></LI><LI><A href="https://2015.igem.org/Team:Washington/Parts#Aptazyme">Biobrick</A></LI><LI><A href="#" class="scroll-link" data-id="nav-anchor">Top</A></LI></UL></DIV></NAV></DIV><DIV id="contentContainer"><H2><DIV id="Introduction">Aptazyme Background</DIV></H2><P align="justify"> An aptamer is a single strand of RNA which folds into a structure that is able to bind to a variety of small molecules and proteins. Ribozymes are in some contexts self-cleaving pieces of RNA which can be utilized to destabilize RNA transcripts. Aptazymes are a combination of both aptamers and ribozymes. Putting these components together allows for the selectivity of aptamers which in turn results in the activity of the ribozyme section of the RNA. Overall this is a reactive strand of RNA with selectivity. Additionally, aptazymes can control protein expression at the level of translation. This allows for quicker response times compared to the traditional method of modifying rates of transcription through interactions with promoters. Theophylline is a commonly used target molecule for academic studies on aptamers due to its ability to permeate membranes.</P><H2><DIV id="Methods"> Design and Methods</DIV></H2><P>In pursuit of developing a new model for detecting small molecules (like shellfish toxins), we designed an RNA aptazyme that would stabilize itself in the presence of a toxin. Our system is based on one main preface- under non-toxic conditions, the aptamer part of our aptazyme takes on a special conformation which activates the ribozyme subunit of the aptazyme to cleave itself. The cleavage results in no translation. However, in the presence of our chosen toxin, theophylline, the aptamer assumes a new conformation which stabilizes the mRNA so translation can happen. The mRNA contains a YFP gene called Venus, which fluoresces once translated. In short, the presence of a toxin is noted by yellow fluorescence. We then implemented our aptazyme in yeast and transposed it on paper. Integrating this system in yeast on a paper device creates a safe and easy way to test for the presence of toxins.</P><H2><B>Plasmid Design and Construction</B></H2><P>Our plasmid contains an aptazyme sequence that was contributed by the Smolke lab (Theo(A)-AAAGA) that was found to be highly functional in yeast. This sequence was integrated with Venus, an optimized form of a yellow fluorescent protein (YFP); a yeast GPD Promoter; a cyc1 terminator; and flanked with a uracil auxotrophic marker, all contributed by the Klavins Lab. Because we received each gene separately, we performed PCR and Gibson Assembly to amplify and make our plasmid. We cloned the plasmid in E. coli and then sequenced to ensure our plasmid was linked correctly.</P><P>Using the same technique, we constructed a plasmid, containing all of the above parts, but with a different version of YFP, which was destabilized by fusing a PEST rich sequence to the C-terminal region of YFP. The PEST sequence came from the last 178 C-terminal residues of G1-cyclin Cln2.</P><P>We also constructed a plasmid with yeVenus/Theo(A)-AAAGA fusion and also the destabilized yeVenus/Theo(A)-AAAGA with a GAL1 galactose-inducible promoter by restriction cloning our construct with SpeI and XhoI into a modified version of pRS426 which already contained the GAL1 promoter and cyc1 terminator. This is a yeast multi-copy plasmid containing a uracil auxotrophic marker and E. coli ampicillin resistance.</P><H2><B>Yeast Transformations</B></H2><P>We digested the GPD promoter containing plasmids with PmeI restriction enzyme which linearizes the plasmid in order to insert the gene into the yeast genome via homologous integration by the uracil cassette. We transformed the linearized plasmid using a high-efficiency yeast transformation protocol.
We transformed the GAL1 promoter containing plasmids following the PLATE “Lazy Bones Protocol.” This protocol is a rapid yeast transformation technique.</P><H2><B>Testing</B></H2><P>We first tested our genetically engineered yeast by having a theophylline-induced and non-induced conditions after incubation of 4 hours. Fluorescence would be measured by the Flow Cytometer.
Secondly, we tested the yeast by inducing at different concentrations of theophylline, namely 0, 5, 10, &amp; 30 mM concentrations of theophylline. The Flow Cytometer measured fluorescence after 4 hours of incubation.
Thirdly, we attached our genetically engineered yeast transformed with pRS426 plasmid containing GALI promoter on filter paper. Next, we allowed the yeast to be exposed to different conditions, namely presence/absence of galactose and theophylline or caffeine. Fluorescence was observed on a dark reader.</P><H2><DIV id="Results"> Results </DIV></H2><P>Our initial results from the experiment with the theophylline induced and non-induced conditions showed a systematic difference with all induced trials having a higher mean fluorescence/cell . yeast transformant 3 showed a greater response than the other transformants.</P><P>Our subsequent concentration dependence experiment showed a increase in fluorescence with increasing concentrations of theophylline. Here again the transformant 3 showed the best concentration dependence. </P><P>The paper device/Gal promoter tests showed fluorescence in all galactose present conditions. The difference in fluorescence between the caffeine and theophylline conditions was very hard to spot, though we expected the caffeine condition to show lesser fluorescence. The device which was plated on sugar free media and had galactose injected prior to theophylline induction also showed fluorescence. </P><H2><DIV id="Conclusion"> Conclusion </DIV></H2><P> The initial flow cytometry results suggest that theophylline might be responsible for the changes in fluorescence. However, the difference in fluorescence is hard to spot visually. We hypothesized that this is caused by leaky expression of YFP, which accumulates in the cell. To address this we tried several approaches. First we replaced our promoter for an inducible one, which is only active in the presence of galactose. This proved to be an impractical solution as it takes too much time for the new promoter to respond to the change of sugar. Also, it didn’t seem to have helped to visually discern the difference in fluorescence. Next we tried a version of YFP, which is less stable. Unfortunately we haven’t, had time to do any tests with this construct. different ways of optimizing our construct are needed in the future. One of them is trying to create a construct with multiple aptazymes attached in series to increase the cleavage efficiency and to decrease the leakiness of expression of YFP.</P><H2><DIV id="Future_Direction"> Future Directions </DIV></H2><P align="justify"> Given that the aptazyme is a model system instead of actually selecting for specific toxins, the next step would be to create aptamers that select for the toxins of interest. Most likely, the selection process would follow evolution based methods (Goler et al., 2014) to optimize a sequence of RNA that would react with the selected molecules. However, this lengthy process does not guarantee results in a set timeframe which is why this portion of the project was not undertaken over the summer.
There are multiple methods to improve the user friendliness. One route to take would be to lower the basal expression levels or increase the expression levels of YFP when the theophylline/toxin is present. A method we are undertaking that will not produce results in time for the wiki is the inclusion of additional aptazyme copies on the RNA (Wei et al, 2013; Win and Smolke, 2008) which should decrease the basal level of expression resulting in a greater proportion between active and basal expression. 
Another method to improve user friendliness would be to create more instantaneous colorimetric switches. A classic example is the creation of RNA-based cocaine sensors (Stojanovic et al., 2001). These sensors should provide feedback to the presence of toxins at a much faster rate than the aptazyme system in yeast. Additionally, the development of better colorimetric and fluorescent probes such as Spinach allow for more robust responses in these systems.   </P></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>