<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Washington_Protocols skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Washington/Protocols</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="menuContainer"><UL><LI><A href="https://2015.igem.org/Team:Washington">Home
                  </A></LI><LI><A href="https://2014.igem.org/Team:Washington">UW 2014</A></LI><LI><A href="https://2013.igem.org/Team:Washington">UW 2013</A></LI><LI><A href="https://2012.igem.org/Team:Washington">UW 2012</A></LI><LI><A href="https://2011.igem.org/Team:Washington">UW 2011</A></LI><LI><A href="https://2010.igem.org/Team:Washington">UW 2010</A></LI><LI><A href="https://2009.igem.org/Team:Washington">UW 2009</A></LI><LI><A href="https://2008.igem.org/Team:University_of_Washington">UW 2008</A></LI><LI><A href="https://2013.igem.org/Main_Page">iGEM Homepage</A></LI></UL><LI><A href="https://2015.igem.org/Team:Washington/Auxin">Auxin
                        </A></LI><LI><A href="https://2015.igem.org/Team:Washington/Auxin#Introduction">Introduction</A></LI><LI><A href="https://2015.igem.org/Team:Washington/Auxin#Methods">Methods</A></LI><LI><A href="https://2015.igem.org/Team:Washington/Auxin#Results">Results</A></LI><LI><A href="https://2015.igem.org/Team:Washington/Auxin#Conclusion">Conclusion</A></LI><LI><A href="https://2015.igem.org/Team:Washington/Auxin#Future_Direction">Future Direction</A></LI><LI><A href="https://2015.igem.org/Team:Washington/Parts#Auxin">Biobrick</A></LI><LI><A href="https://2015.igem.org/Team:Washington/Aptazyme">Aptazyme                               
                    </A></LI><LI><A href="https://2015.igem.org/Team:Washington/Aptazyme#Introduction">Introduction</A></LI><LI><A href="https://2015.igem.org/Team:Washington/Aptazyme#Methods">Methods</A></LI><LI><A href="https://2015.igem.org/Team:Washington/Aptazyme#Results">Results</A></LI><LI><A href="https://2015.igem.org/Team:Washington/Aptazyme#Conclusion">Conclusion</A></LI><LI><A href="https://2015.igem.org/Team:Washington/Parts#Aptazyme&quot;">Biobrick</A></LI><LI><A href="https://2015.igem.org/Team:Washington/Paper_Device">Paper Device               
                    </A></LI><LI><A href="https://2015.igem.org/Team:Washington/Paper_Device#Introduction">Introduction</A></LI><LI><A href="https://2015.igem.org/Team:Washington/Paper_Device#Methods">Methods</A></LI><LI><A href="https://2015.igem.org/Team:Washington/Design">Design</A></LI><LI><A href="https://2015.igem.org/Team:Washington/Paper_Device#Results">Results</A></LI><LI><A href="https://2015.igem.org/Team:Washington/Paper_Device#Conclusion">Conclusion</A></LI><LI><A href="https://2015.igem.org/Team:Washington/Modeling">Modeling               
                    </A></LI><LI><A href="https://2015.igem.org/Team:Washington/Modeling#Paper_Device">Paper Device</A></LI><LI><A href="https://2015.igem.org/Team:Washington/Modeling#Aptazyme">Aptazyme</A></LI><LI><A href="https://2015.igem.org/Team:Washington/Practices">Human Practices               
                    </A></LI><LI><A href="https://2015.igem.org/Team:Washington/Practices#Outreach">Outreach</A></LI><LI><A href="https://2015.igem.org/Team:Washington/Practices#Integrated">Integrated</A></LI><LI><A href="https://2015.igem.org/Team:Washington/Protocols">Protocols
                  </A></LI><LI><A href="#" class="scroll-link" data-id="Experiments">Experiments</A></LI><LI><A href="#" class="scroll-link" data-id="Safety">Safety</A></LI></DIV><LI><A href="https://2015.igem.org/Team:Washington/Team">Team                         
                    </A></LI><LI><A href="https://2015.igem.org/Team:Washington/Team#Members">Members</A></LI><LI><A href="https://2015.igem.org/Team:Washington/Attributions">Attributions</A></LI><LI><A href="https://2015.igem.org/Team:Washington/Team#Sponsors">Sponsors</A></LI><LI><A href="https://2015.igem.org/Team:Washington/Team#Judging_Form">Judging Form</A></LI></DIV><DIV id="main"><DIV class="container clearfix"><DIV id="sidebar"><NAV><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav" style="list-style-type: none;"><LI><A href="#" class="scroll-link" data-id="Protocols">Protocols</A></LI><LI><A href="#" class="scroll-link" data-id="Safety">Safety</A></LI><LI><A href="#" class="scroll-link" data-id="nav-anchor">Top</A></LI></UL></DIV></NAV></DIV><DIV id="contentContainer"><H2><DIV id="Protocols">Protocols</DIV></H2><UL id="tabs"><LI><A href="#Auxin">Auxin Protocols</A></LI><LI><A href="#Aptazyme">Aptazyme Protocols</A></LI><LI><A href="#Paper">Paper Protocols</A></LI></UL><DIV class="tabContent" id="Aptazyme"><DIV><H2>Yeast Chromosome Integration</H2><P1>Digest E. Coli plasmid using PmeI restriction enzyme</P1><UL list-style-image:url=""><LI>1 ug of DNA</LI><LI>5 uL of 10x NEB CutSmart buffer</LI><LI>1 uL of restriction enzyme</LI><LI>Fill to 50 uL with water</LI><LI>Incubate at 37 C for 15 minutes (1 hour if not using TimeSaver buffer)</LI><LI>Heat inactivate at 65 C for 20 minutes</LI><LI>Agarose gel purify (optional)</LI></UL><P1>Salmon Sperm Transformation</P1><UL list-style-type:circle=""><LI>Grow a yeast overnight </LI><LI>Check OD of culture. 0.5-0.6 are the preferred readings, if the reading is lower, wait for longer growth, if the reading is higher, dilute the sample.</LI><LI>Spin down 10 ml of cells per transformation.</LI><LI>Decant supernatant and wash with 10 ml ddH2O. Vortex to resuspend and spin down.</LI><LI>Remove the supernatant.</LI><LI>Resuspend cells in 300 uL .1 M LiOAc. Transfer to a 1.5 mL tube.</LI><LI>Incubate at 30 C for 15 min</LI><LI>Put salmon sperm DNA in boiling water for 5 minutes. Cool immediately on ice.</LI><LI>Spin down cells and remove supernatant.</LI><LI>Add the following in order:</LI><OL type="a"><LI>240 uL 50% PEG</LI><LI>36 uL 1.0 M LioAc</LI><LI>10 uL salmon sperm DNA</LI><LI>34 uL DNA</LI><LI>40 uL ddH2O</LI><LI>Final volume: 360 uL</LI></OL></UL><H2>Theophylline Stock</H2><P>50 mM Theophylline dissolved by DMSO</P><P>Replace occasionally due to possible interactions between theophylline &amp; DMSO</P><H2>Fluorescence Reading</H2><UL list-style-type:none=""><LI>Put yeast plate to a blue light imager.</LI><LI>Note differences in brightness between yeast colonies</LI></UL><H2>Flow Cytometer</H2><UL list-style-type:none=""><LI> 1. From the dilution previously made, measure the optical density, roughly 1.2-1.6.</LI><LI> 2. Make an aliquot of 500 μL of the dilution culture in a 1.5 mL centrifuge tube. </LI><LI> 3. Spin down the aliquot at 3000 rpm for 3 minutes. </LI><LI> 4. Decant the supernatant.</LI><LI> 5. Resuspend the cell pellet in 500 μL of PBSF.</LI><LI> 6. Spin down the resuspension at 3000 rpm for 3 minutes. </LI><LI> 7. Decant the supernatant. </LI><LI> 8. Resuspend the cell pellet in another 500 μL of PBSF.* </LI><LI> 9. Prepare the C6 Accuri Flow Cytometer by running a backflush cycle and a diH2O cycle. </LI><LI> 10. Load the sample onto the sip. </LI><LI> 11. Run the sample with 100,000 cell count. </LI><LI> 12. Repeat for all samples and make sure to change data cells otherwise the old data will be erased. </LI><LI> 13. Once finished, run a cleaning cycle with Accuri approved cleaning solution, then run a diH2O cycle. </LI><LI><B>Parameters: </B></LI><LI>Excitation: 515 nm</LI><LI>Emission: 530 nm</LI></UL></DIV></DIV><DIV class="tabContent" id="Auxin"><DIV><H2>X-GAL ASSAY</H2><UL><OL type="1"><LI>Prepare a solution of 50 mg/mL X-Gal in DMSO.</LI><LI>Take one ml of cells culture and spin down.</LI><LI>Remove supernatant and resuspend 1ml of water.</LI><LI>Spin down, and remove supernatant again.</LI><LI>Resuspend in 5ul X-Gal solution, 25 ml 2% SDS and 70ml water.</LI><LI>Incubate away from light at 37C for 30+ min</LI></OL></UL><H2>Auxin Assay.</H2><UL><OL type="1"><LI>Prepare a solution of 100 mMol auxin in ethanol.</LI><LI>Add 3ul auxin per ml of cell culture you are using.</LI><LI>Wrap your culture container in foil and put in the shake incubator @ 30C for 4+ hours.</LI><LI>Finish by doing the X-Gal assay described above. Alternatively, just add the SDS and X-Gal directly.</LI></OL></UL><H2>Transformation of E.Coli Competent Cell :</H2><UL><OL type="1"><LI>Thaw competent E.coli cells on ice (XL1-Blue for cloning)</LI><LI>Add 50 uL of competent cells to sterile 14 mL Falcon culture tube.</LI><LI>Add 1 uL of the miniprep to each culture tube</LI><LI>Equilibrate the cells on ice for 10 min</LI><LI>Heat shock the cells at 42C for 30-45 second.</LI><LI>Immediately place the cells back on ice for 3 min</LI><LI>Add 250 uL LB media and shake at 250 rpm and 37C for 30 min</LI><LI>Plate 10 ul and 290 ul of the recovered cells onto LB-agar plates supplemented with appropriate antibiotics﻿ (spread with ~150uL DiH2O)</LI><LI>Invert and incubate at 37C overnight</LI></OL></UL><H2>DIGESTION:</H2><UL><LI>Buffers are in 10X. Upper limit 120ng/uL for Plasmids. 50uL Reaction Volume</LI><LI>DNA/Plasmid: minimum 1ug - 5ug max (more or less depending on the amount from the      miniprep)</LI><LI>DO NOT EXCEED 120ng/uL</LI><LI>10x Buffer: 5uL of NEBuffer 2.1</LI><LI>Enzyme: 1uL of EcoRI and 1uL of Nhel (or other restriction enzyme)</LI><LI>*Add enzyme last</LI><LI>ADD water to 50uL total volume</LI><LI>Incubate at 37 C for 1 Hour</LI></UL><H2>High Efficiency Yeast Transformation Protocol</H2>
(Modified from Agatep, R. et al. (1998) Technical Tips Online)
<UL><OL type="1"><LI>Inoculate a 50 ml ypd liquid culture (with a single yeast colony). This volume depends on how many transformations are needed. Allow about 5-10ml of culture per sample. However, never start a culture less than 50ml.
</LI><LI>Grow overnight with shaking at 30° C.
</LI><LI>In the morning, read the OD of the culture. A culture in mid-log with an OD between 0.5 and 0.6 is ideal (which represents approximately 2.0 x 107total yeast cells per ml), but cultures ranging from 0.4 to 0.8 are also acceptable. If the culture is overgrown, dilute to approximately 0.2 and grow for 2 generations (2-3 hours). If efficiency is not essential, it is acceptable to use 10 ml of a culture at 0.3 OD. 
	</LI><LI>How to use the spectrometer. Cuvettes are located in a Styrofoam box on the right of the computer.</LI><LI>Make sure the reading is at 600 nm</LI><LI>Blank with 900 uL of ypd</LI><LI>Add 100 uL of culture to make a 1:10 dilution</LI><LI>Measure the OD. Keep in mind that the actual OD is 10x larger</LI><LI>Use vacuum line to empty the cuvette, cuvettes can be reused</LI><LI>Dilute or concentrate and repeat measurements until OD is in the range</LI></OL><LI>Spin down 10 ml of cells per transformation (2000 rpm / 2 min) in a 15 ml Falcon tube (which equals 2 x 108 total cells). Multiple transformations with the same strain can be combined into larger volumes up until step 7. Please note that these volumes are for haploid strains. A transformation with a diploid strain would require 5 ml of culture per transformation (equaling 1 x 108 total cells).</LI><LI>Decant supernatant and wash with 10 ml ddH2O. Vortex briefly to resuspend cells. Spin down cells (2000 rpm / 2 min). Completely remove supernatant.</LI><LI>Resuspend cells (by vortexing) in 300 µl 0.1 M LiOAc. Transfer to a sterile 1.5 ml eppendorf tube. (This step and those that follow list volume amounts that are PER TRANSFORMATION).</LI><LI>Incubate at 30° C for 15 min.</LI><LI>Begin boiling water to denature Salmon Sperm DNA. Boil 5 min. Cool immediately on ice to prevent re-annealing of single stranded DNA.</LI><LI>Spin down cells (2500 rpm/ 1 min). Remove supernatant with a pipetman.</LI><LI>Add the following components of the transformation mix in order: </LI><OL type="a"><LI>240 µl 50% PEG</LI><LI>36 µl 1.0 M LiOAc</LI><LI>10 µl Salmon Sperm DNA (10 mg/ml)</LI><LI>34 µl DNA (plasmid, PCR or digest)</LI><LI>40 µl ddH2O</LI></OL><P>FINAL VOLUME: 360 µl</P><P>Please note: a mastermix at this point is acceptable if transforming multiple samples into the same yeast strain. Add 326 µl of mastermix to each pellet, then the DNA sample.</P><LI>Vortex thoroughly to resuspend cells (approximately 30 sec). Be sure all cells are in suspension.</LI><LI>Incubate at 30° C for 2 hours. Mix cells gently every hour. Incubation time can range anywhere from 1.5 hours to 3 hours, resulting in the same transformation efficiency.</LI><LI>Add 36 µl of DMSO and mix by inverting tubes. (DO NOT VORTEX)</LI><LI>Heat shock for 5 min at 42° C.</LI><LI>Spin down (2500 rpm / 1 min). Remove supernatant with a pipetman.</LI><LI>Wash cells by gently pipeting cells in 1 ml ypd liquid (or selective media). Spin down (2500 rpm / 1 min). Remove supernatant.</LI><LI>Resuspend cells by gently pipeting in 1 ml liquid media. Dilute if necessary.</LI><LI>Incubate culture with shaking at 30° C overnight if expression is necessary, otherwise skip this step. Often cells expressing certain selectable markers require an additional incubation at this step to allow phenotypic expression of the antibiotic resistance marker.</LI><LI>Titer on selective plates (10-1,10-2,10-3 are the standard dilutions). </LI><OL type="a"><LI>If you re suspend the cells in 1 ml of -ura then plate 100ul on a -Ura plate, that is a 10-1 dilution.
</LI><LI>If you plate 100 or 200ul on a plate, you'll get colonies for sure
</LI></OL><LI>Plate on selective media.</LI></UL></DIV></DIV><DIV class="tabContent" id="Paper"><DIV><H2>Making 1mm-Thick Sheets of PDMS (Polydimethyl Siloxane)</H2><UL><LI>Obtain Sylgard 184 Silicone Elastomer Kit from Dow Corning</LI><LI>Obtain cell culture plates to use as templates</LI><LI>Calculate surface area of plate in order to determine the mass needed of PDMS</LI><LI>Make a 1:10 mixture of activator to PDMS</LI><LI>Stir well using a wood or plastic stick</LI><LI>Pour polymer onto the top of the lid or bottom of the culture plate.  Pour into the middle and work out towards the edges.  Tilt plate to let PDMS</LI><LI>flow to the edges of the plate.</LI><LI>Remove bubbles, either by putting the sample in a vacuum or by popping the bubbles with the stirring stick</LI><LI>Let polymer cure for 2 days in a flat place to ensure even distribution.</LI></UL><H2>Making the Paper Device</H2><UL><LI>Cut strips of Whatman 114 filter paper to dimensions 3cm x 4cm</LI><LI>Cut 2cm x 2cm pieces of PDMS sheet</LI><LI>Use heat-resistant tape (such as electrical tape or 3M heat-resistant packing tape) to tape polymer squares to either side of the paper strip.  On one side, tape around all four sides of the PDMS.  On the other, leave the top open in order to insert media and cells.</LI><LI>Autoclave devices in a sterile container on a dry cycle</LI></UL><H2>Inserting cells and media</H2><UL><LI>Make either liquid or gel media with conditions specific for the intended yeast strain (note that liquid media may dry out fast)</LI><LI>For liquid media, pipette no more than 100uL through the open side of the device, between the paper and the PDMS window</LI><LI>For gel media, pipette no more than 300uL between the paper and PDMS</LI><LI>Using a pipette tip or toothpick, insert a clump of cells and deposit them in the middle of the PDMS window.  For gel media, it may be necessary to use a toothpick to poke a pathway through the gel in order to ensure cells can be deposited easily.</LI><LI>Tape over the remaining side of the device to seal the cells and media inside</LI></UL><H2>Running detection assays</H2><UL><LI>The molecule of interest can either be pipetted directly between the paper and PDMS along with the yeast media, or the device can be stood up in a solution of the molecule, which can travel to the yeast via wicking.</LI></UL><H2>Yeast lysis on paper to view beta-Gal production</H2><UL><LI>Place yeast on liquid media inside a device.</LI><LI>Pipette 30uL of 0.2% SDS and 10uL of 50mM X-gal directly to the cells, or place end of device in solution of 300uL of 0.2% SDS and 100uL X-gal</LI><LI>When SDS is in contact with cells, massage PDMS window briefly to mix</LI><LI>Allow devices to sit at 37C for 30 minutes to 1 hour</LI></UL><H2>Observe blue color</H2><UL><LI>Theophylline detection on paper (assuming inducible Gal promoter)</LI><LI>Load device with theophylline-detecting strain and c-ura media, with galactose as the sugar*</LI><LI>Seal device</LI><LI>Allow device to sit in a beaker of 3mL of 50mM theophylline solution for 6-10 hours.  </LI><LI>Observe fluorescence</LI></UL><P>*Alternatively, use media with no sugar and add galactose the same way as theophylline</P><P>Possible controls</P><UL><LI>Strain that doesn’t produce theo aptazyme with theophylline (negative control)</LI><LI>Theo detection strain without theophylline (negative control)</LI><LI>Strain that constitutively produces YFP with theophylline (positive control)</LI><LI>Theo detection strain on arafinose or glucose c-ura media</LI><LI>Theo detection strain on galactose c-ura with caffeine (no theophylline)</LI></UL><H2>Auxin detection on paper</H2><P>Possible controls</P><UL><LI>Strain without auxin detection pathway with auxin (negative control)</LI><LI>Strain with auxin detection pathway without auxin (negative control)</LI><LI>Strain without gRNA, which constitutively produces beta-Gal/blue chromoprotein with auxin (positive control)</LI></UL></DIV></DIV><H2><DIV id="Safety">Safety</DIV></H2><H1>What risks does your project pose at the laboratory stage? What actions are you taking to reduce those risks?</H1><P>Our lab is BSL 1.  We work with common lab strains of yeast and E. coli, which are kept in closed containers and always handled with gloves.  Closed-toe shoes are required in our lab.  We autoclave all biohazard waste.  When running gel electrophoresis, we use Sybr Safe, which is a mutagen.  To minimize exposure, we wear gloves and work in a contained space (fume hood) and use equipment that is not used for other experiments.  Chloroform is occasionally used in a hood to contain fumes.  Theophylline is used in some experiments.  Even though Theophylline is used in medicine, high exposures to it may be harmful, so we only handle small amounts of it.</P><H1>What risks might your project pose, if it were fully developed into a real product that real people could use? What future work might you do to reduce those risks?</H1><P>Our product is meant to be a disposable test strip for use in the environment.  Since releasing engineered organisms into the environment is a significant risk with unknown consequences, we would want to engineer a biological kill switch or mode of codon security into our yeast to prevent growth outside of the device.  Yeast are also physically contained behind an impermeable PDMS shield within our device.    
</P><H4>Do the biological materials used in your lab work pose any of the following risks?</H4><H2>A.  Risks to the safety and health of team members or others working in the lab?</H2><P>Using E.coli and Theophylline poses a potential risk to the health and safety of our team members working in the lab if it is handled improperly or consumed. It may cause irritation to skin, eyes, and respiratory tract and may adversely affect kidneys.</P><P>Using Sybr Safe and Ethidium Bromide to make gels poses health risks because both chemicals are mutagens.</P><H2>B. Risks to the safety and health of the general public, if released by design or by accident?</H2><P>The risks to the safety and health of the general public are the same as those for individuals directly working with these biological materials.</P><H2>C. Risks to the environment, if released by design or by accident?</H2><P>Sybr Safe and Ethidium Bromide both cause risks to the environment because they are both mutagens.</P><H2>D. Risks to security through malicious misuse by individuals, groups, or countries?</H2><P>There is slight concern for the misuse of our systems, since any gene could be substituted in place of the existing outputs.</P><H4>What safety training have you received (or plan to receive in the future)? Provide a brief description, and a link to your institution’s safety training requirements, if available.</H4><P>All student members of our team have been trained by graduate student advisers about proper lab techniques, lab etiquette, and biohazard waste disposal. All team members have completed the University of Washington's online training in biosafety, fume hood use, and managing lab chemicals. Advisers have been trained by their respective labs in accordance with the University of Washington Environmental Health and Safety Committee's regulations. EH&amp;S inspections to make sure the lab is up to university lab standards</P><H4>Under what biosafety provisions will / do you work?</H4><H2>A. Please provide a link to your institution biosafety guidelines.</H2><P><A href="www.ehs.washington.edu">www.ehs.washington.edu</A></P><H2>B. Does your institution have an Institutional Biosafety Committee, or an equivalent group? If yes, have you discussed your project with them?</H2><P>The University of Washington Environmental Health and Safety (UW EHS) committee determines biosafety regulations and guidelines for all labs associated with our campus. We have not discussed this specific iGEM project with members of the EHS committee; however, we are working closely with our sponsor labs and have been trained according to the guidelines which they follow.</P><H2>C. Does your country have national biosafety regulations or guidelines? If so, please provide a link to these regulations or guidelines if possible?</H2><P>The United States of America has national biosafety regulations and guidelines determined by the Centers for Disease Control and Prevention (CDC). Specifics about their guidelines can be found at <A href="www.cdc.gov/biosafety/">www.cdc.gov/biosafety</A></P><H2>D. Does your country have national biosafety regulations or guidelines? If so, please provide a link to these regulations or guidelines if possible?</H2><P>The United States of America has national biosafety regulations and guidelines determined by the Centers for Disease Control and Prevention (CDC). Specifics about their guidelines can be found at <A href="www.cdc.gov/biosafety/">www.cdc.gov/biosafety/</A></P><H2>E. According to the WHO Biosafety Manual, what is the BioSafety Level rating of your lab? (Check the summary table on page 3, and the fuller description that starts on page 9.) If your lab does not fit neatly into category 1, 2, 3, or 4, please describe its safety features [see 2013.igem.org/Safety for help].</H2><P>The BioSafety level of our lab is category 2. The lab room used is equipped to deal with category 2 hazards, for example, it contains a fume hood. However, for this project, only category 1 cells were used; namely, non-pathogenic E. coli and yeast.</P></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>