<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Washington_Aptamer skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Washington/Aptamer</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="menuContainer"><UL><LI><A href="https://2015.igem.org/Team:Washington">Home</A></LI><LI><A href="https://2015.igem.org/Team:Washington/Auxin">Auxin  
                    </A></LI><LI><A href="https://2015.igem.org/Team:Washington/Auxin#Introduction">Introduction</A></LI><LI><A href="https://2015.igem.org/Team:Washington/Auxin#Methods">Methods</A></LI><LI><A href="https://2015.igem.org/Team:Washington/Auxin#Results">Results</A></LI><LI><A href="https://2015.igem.org/Team:Washington/Auxin#Conclusion">Conclusion</A></LI><LI><A href="https://2015.igem.org/Team:Washington/Auxin#Future_Direction">Future Direction</A></LI><LI><A href="https://2015.igem.org/Team:Washington/Parts#Auxin">Biobrick</A></LI></UL><LI><A href="https://2015.igem.org/Team:Washington/Aptamer">Aptamer                               
                  </A></LI><LI><A href="#" class="scroll-link" data-id="Introduction">Introduction</A></LI><LI><A href="#" class="scroll-link" data-id="Methods">Methods</A></LI><LI><A href="#" class="scroll-link" data-id="Results">Results</A></LI><LI><A href="#" class="scroll-link" data-id="Conclusion">Conclusion</A></LI><LI><A href="#" class="scroll-link" data-id="Biobrick">Biobrick</A></LI></DIV><LI><A href="https://2015.igem.org/Team:Washington/Paper_Device">Paper Device               
                    </A></LI><LI><A href="https://2015.igem.org/Team:Washington/Paper_Device#Introduction">Introduction</A></LI><LI><A href="https://2015.igem.org/Team:Washington/Paper_Device#Methods">Methods</A></LI><LI><A href="https://2015.igem.org/Team:Washington/Design">Design</A></LI><LI><A href="https://2015.igem.org/Team:Washington/Paper_Device#Results">Results</A></LI><LI><A href="https://2015.igem.org/Team:Washington/Paper_Device#Conclusion">Conclusion</A></LI><LI><A href="https://2015.igem.org/Team:Washington/Modeling">Modeling               
                    </A></LI><LI><A href="#" class="scroll-link" data-id="Paper_Device">Paper Device</A></LI><LI><A href="#" class="scroll-link" data-id="Aptamer">Aptamer</A></LI><LI><A href="https://2015.igem.org/Team:Washington/Practices">Human Practices               
                    </A></LI><LI><A href="https://2015.igem.org/Team:Washington/Practices#Outreach">Outreach</A></LI><LI><A href="https://2015.igem.org/Team:Washington/Practices#Integrated">Integrated</A></LI><LI><A href="https://2015.igem.org/Team:Washington/Protocols">Protocols
                    </A></LI><LI><A href="https://2015.igem.org/Team:Washington/Protocols#Experiments">Experiments</A></LI><LI><A href="https://2015.igem.org/Team:Washington/Protocols#Safety">Safety</A></LI><LI><A href="https://2015.igem.org/Team:Washington/Team">Team                         
                    </A></LI><LI><A href="https://2015.igem.org/Team:Washington/Team#Members">Members</A></LI><LI><A href="https://2015.igem.org/Team:Washington/Attributions">Attributions</A></LI><LI><A href="https://2015.igem.org/Team:Washington/Team#Sponsors">Sponsors</A></LI><LI><A href="https://2015.igem.org/Team:Washington/Team#Judging_Form">Judging Form</A></LI></DIV><DIV id="main"><DIV class="container clearfix"><DIV id="sidebar"><NAV><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav" style="list-style-type: none;"><LI><A href="#" class="scroll-link" data-id="Introduction">Introduction</A></LI><LI><A href="#" class="scroll-link" data-id="Methods">Methods</A></LI><LI><A href="#" class="scroll-link" data-id="Results">Results</A></LI><LI><A href="#" class="scroll-link" data-id="Conclusion">Conclusion</A></LI><LI><A href="#" class="scroll-link" data-id="Future_Direction">Future Direction</A></LI><LI><A href="#" class="scroll-link" data-id="Biobrick">Biobrick</A></LI><LI><A href="#" class="scroll-link" data-id="nav-anchor">Top</A></LI></UL></DIV></NAV></DIV><DIV id="contentContainer"><H2><DIV id="Introduction">Aptazyme Background</DIV></H2><P align="justify"> An aptamer is a single strand of RNA which folds into a structure that is able to bind to a variety of small molecules and proteins. Ribozymes are in some contexts self-cleaving pieces of RNA, which can be utilized to destabilize RNA transcripts. A specific subgroup of Aptazymes, called Riboswitches, are a combination of both aptamers and ribozymes. Putting these components together allows for the aptamer section of the RNA to regulate the activity of the ribozyme section of the RNA. Overall, riboswitch is a reactive strand of RNA under allosteric control. Additionally, aptazymes can control protein expression at the level of translation. This allows for quicker response times compared to the traditional method of modifying rates of transcription through interactions with promoters. Theophylline is commonly used target molecule (ligand for the aptazyme) for academic studies on aptamers due to its ability to permeate membranes.</P><H2><DIV id="Methods"> Design and Methods</DIV></H2><P>Since we thought yeast would be a better implementation on paper, our original concept was to utilize an aptazyme sequence that was contributed by the Smolke lab that was found to be highly functional in yeast. This sequence was integrated with Venus, an optimized form of a yellow fluorescent protein (YFP); a yeast GPD Promoter; a cyc1 terminator; and a uracil auxotrophic marker. The plasmid was cloned in e. coli and digested with a PmeI restriction enzyme in order to insert the gene into the yeast genome via homologous integration by the uracil casette. Experiments were carried out using these new strains of yeast. </P><H2><DIV id="Results"> Results </DIV></H2><P>Our initial results from the experiment with the theophylline induced and non-induced conditions showed a systematic difference with all induced trials having a higher mean fluorescence/cell (graph). yeast transformant 3 showed a greater response than the other transformants.</P><P>Our subsequent concentration dependence experiment showed a increase in fluorescence with increasing concentrations of theophylline. Here again the transformant 3 showed the best concentration dependence. (graph)</P><P>The paper device/Gal promoter tests showed fluorescence in all galactose present conditions. The difference in fluorescence between the caffeine and theophylline conditions was very hard to spot, though we expected the caffeine condition to show lesser fluorescence. The device which was plated on sugar free media and had galactose injected prior to theophylline induction also showed fluorescence. (picture)</P><H2><DIV id="Conclusion"> Conclusion </DIV></H2><P align="justify"> Contents </P><H2><DIV id="Future_Direction"> Future Directions </DIV></H2><P align="justify"> Given that the aptazyme is a model system instead of actually selecting for specific toxins, the next step would be to create aptamers that select for the toxins of interest. Most likely, the selection process would follow evolution based methods (Goler et al., 2014) to optimize a sequence of RNA that would react with the selected molecules. However, this lengthy process does not guarantee results in a set timeframe which is why this portion of the project was not undertaken over the summer.
There are multiple methods to improve the user friendliness. One route to take would be to lower the basal expression levels or increase the expression levels of YFP when the theophylline/toxin is present. A method we are undertaking that will not produce results in time for the wiki is the inclusion of additional aptazyme copies on the RNA (Wei et al, 2013; Win and Smolke, 2008) which should decrease the basal level of expression resulting in a greater proportion between active and basal expression. 
Another method to improve user friendliness would be to create more instantaneous colorimetric switches. A classic example is the creation of RNA-based cocaine sensors (Stojanovic et al., 2001). These sensors should provide feedback to the presence of toxins at a much faster rate than the aptazyme system in yeast. Additionally, the development of better colorimetric and fluorescent probes such as Spinach allow for more robust responses in these systems.   </P><H2><DIV id="Biobrick"> Biobricks </DIV></H2><P align="justify"> Contents</P></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>