<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Washington_Auxin skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Washington/Auxin</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="menuContainer"><UL><LI><A href="https://2015.igem.org/Team:Washington">Home
                  </A></LI><LI><A href="https://2014.igem.org/Team:Washington">UW 2014</A></LI><LI><A href="https://2013.igem.org/Team:Washington">UW 2013</A></LI><LI><A href="https://2012.igem.org/Team:Washington">UW 2012</A></LI><LI><A href="https://2011.igem.org/Team:Washington">UW 2011</A></LI><LI><A href="https://2010.igem.org/Team:Washington">UW 2010</A></LI><LI><A href="https://2009.igem.org/Team:Washington">UW 2009</A></LI><LI><A href="https://2008.igem.org/Team:University_of_Washington">UW 2008</A></LI><LI><A href="https://2013.igem.org/Main_Page">iGEM Homepage</A></LI></UL><LI><A href="https://2015.igem.org/Team:Washington/Auxin">Auxin 
                   </A></LI><LI><A href="#" class="scroll-link" data-id="Introduction">Introduction</A></LI><LI><A href="#" class="scroll-link" data-id="Methods">Methods</A></LI><LI><A href="#" class="scroll-link" data-id="Results">Results</A></LI><LI><A href="#" class="scroll-link" data-id="Conclusion">Conclusion</A></LI><LI><A href="#" class="scroll-link" data-id="Future_Direction">Future Direction</A></LI><LI><A href="https://2015.igem.org/Team:Washington/Parts#Auxin&quot;">Biobrick</A></LI></DIV><LI><A href="https://2015.igem.org/Team:Washington/Aptazyme">Aptazyme                               
                    </A></LI><LI><A href="https://2015.igem.org/Team:Washington/Aptazyme#Introduction">Introduction</A></LI><LI><A href="https://2015.igem.org/Team:Washington/Aptazyme#Methods">Methods</A></LI><LI><A href="https://2015.igem.org/Team:Washington/Aptazyme#Results">Results</A></LI><LI><A href="https://2015.igem.org/Team:Washington/Aptazyme#Conclusion">Conclusion</A></LI><LI><A href="https://2015.igem.org/Team:Washington/Parts#Aptazyme&quot;">Biobrick</A></LI><LI><A href="https://2015.igem.org/Team:Washington/Paper_Device">Paper Device               
                    </A></LI><LI><A href="https://2015.igem.org/Team:Washington/Paper_Device#Introduction">Introduction</A></LI><LI><A href="https://2015.igem.org/Team:Washington/Paper_Device#Methods">Methods</A></LI><LI><A href="https://2015.igem.org/Team:Washington/Design">Design</A></LI><LI><A href="https://2015.igem.org/Team:Washington/Paper_Device#Results">Results</A></LI><LI><A href="https://2015.igem.org/Team:Washington/Paper_Device#Conclusion">Conclusion</A></LI><LI><A href="https://2015.igem.org/Team:Washington/Modeling">Modeling               
                    </A></LI><LI><A href="https://2015.igem.org/Team:Washington/Modeling#Paper_Device">Paper Device</A></LI><LI><A href="https://2015.igem.org/Team:Washington/Modeling#Aptazyme">Aptazyme</A></LI><LI><A href="https://2015.igem.org/Team:Washington/Practices">Human Practices               
                    </A></LI><LI><A href="https://2015.igem.org/Team:Washington/Practices#Outreach">Outreach</A></LI><LI><A href="https://2015.igem.org/Team:Washington/Practices#Integrated">Integrated</A></LI><LI><A href="https://2015.igem.org/Team:Washington/Protocols">Protocols
                    </A></LI><LI><A href="https://2015.igem.org/Team:Washington/Protocols#Experiments">Experiments</A></LI><LI><A href="https://2015.igem.org/Team:Washington/Protocols#Safety">Safety</A></LI><LI><A href="https://2015.igem.org/Team:Washington/Team">Team                         
                    </A></LI><LI><A href="https://2015.igem.org/Team:Washington/Team#Members">Members</A></LI><LI><A href="https://2015.igem.org/Team:Washington/Attributions">Attributions</A></LI><LI><A href="https://2015.igem.org/Team:Washington/Team#Sponsors">Sponsors</A></LI><LI><A href="https://2015.igem.org/Team:Washington/Team#Judging_Form">Judging Form</A></LI></DIV><DIV id="main"><DIV class="container clearfix"><DIV id="sidebar"><NAV><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav" style="list-style-type: none;"><LI><A href="#" class="scroll-link" data-id="Introduction">Introduction</A></LI><LI><A href="#" class="scroll-link" data-id="Methods">Methods</A></LI><LI><A href="#" class="scroll-link" data-id="Results">Results</A></LI><LI><A href="#" class="scroll-link" data-id="Conclusion">Conclusion</A></LI><LI><A href="#" class="scroll-link" data-id="Future_Direction">Future Direction</A></LI><LI><A href="https://2015.igem.org/Team:Washington/Parts#Aptazyme&quot;">Biobrick</A></LI><LI><A href="#" class="scroll-link" data-id="nav-anchor">Top</A></LI></UL></DIV></NAV></DIV><DIV id="contentContainer"><H1 align="center"><DIV id="Introduction">Introduction: Auxin Pathway</DIV></H1><H2>CRISPR Transcriptional Factors</H2><P align="justify"> CRISPR transcriptional factors enable scientists to make targeted changes in gene expression through the use of a guide RNA (gRNA). The gRNA attaches to the CRISPR protein, and guides it to a specific DNA locus based on the sequence of the gRNA.  The CRISPR proteins we used known as dCas9 were obtained from the Klavins lab, and are designed so that they do not cleave the DNA strands themselves, this dead CRISPR will instead bind tightly and prevent access to the gene by other proteins. This effectively disables transcription and translation of the targeted gene.  CRISPR transcriptional factors were first developed by Perez-Pinera et al. in 2013. </P><H2>Auxin Background</H2><P align="justify"> Auxin is a class of plant hormone that is involved in developmental and behavioral signalling.  This type of molecule serves as a good proof-of-concept for the detection of small-molecules by our model system. This is because Auxin can pass through the cell membrane and bind to an F-box protein (AFB2) and a degron (deg1), which will then target our dCas9 protein for degradation. Additionally, auxin is orthogonal to the biochemical machinery of the yeast, allowing for uninterrupted signal processing. Indole-3-acetic acid (IAA) is the specific Auxin molecule that we are using in our model system.</P><H2> Model System Design </H2><P align="justify">  The pathway relies on several components. A dead-CRISPR transcription factor (dCas9) is  fused to a degron domain (Deg1) and a strong repressor domain (MXI1). Along with a guide RNA designed to target an a promoter region for a signal response. Combined, the dCas9 complex along with the guide RNA will suppress the expression of lacZ. However, in the presence of IAA and an F-Box protein (AFB2), the dCas9 complex will be degraded and lacZ will be expressed leading to the production of beta-galactosidase. Thus, causing another small molecule X-gal to be converted into indigo when cleaved by beta-galactosidase and appear blue. The system is designed to produce a response (a color response in our case) to the presence of the small molecule indole-3-acetic acid. </P><P style="font-size:12px" align="justify">(1) dCas9-deg1-MXI1 binds to a guide RNA which targets a sequence of a lacZ promoter causing the expression of beta-galactosidase to be supprefont size = 2ssed. (2) Auxin binds to both AFB2 (which also helps recruit the ubiquitin ligase)  and deg1 simultaneously bringing the two into close proximity allowing the ubiquitin ligase to ubiquitinate the dCas9 construct. (3) lacZ expression is no longer suppressed allowing beta galactosidase to be produced and in the presence of X-gal, indigo is then formed. </P><H2><DIV id="Methods"> Methods </DIV></H2><P align="justify"> Since the system requires 4 components (AFB2, dCas9-deg1-MXI1, lacZ, gRNA) to be transformed into a single cell, our team decided to integrate each component into the yeast genome sequentially using a &quot;cassette&quot;. A genomic integration has the advantage that cells are less likely to loose the genetic information once acquired into the cells genome. This saves time and effort and by-passes and potential problem with multiple dropout media for marker selections during each sequential transformation. The Klavins Lab at the University of Washington, allowed us access to their genetically modified Saccharomyces Cerevisiae strain that has a partially deleted uracil, histidine, leucine and tryptophan autotrophic gene, making the strain incapable of surviving without the aforementioned amino acids. With our strain selected we proceeded to sequentially transform each component of auxin detection pathway. </P><P align="justify"> Each of the 4 genetic components of our system were contained within plasmids containing an ampicillin resistance marker and were cloned and amplified in E.Coli. Furthermore, each the component's plasmid also contained a different yeast autotrophic selection marker as followed: AFB2 Leucine, dCas9 Tryptophan, lacZ Uracil, gRNA Histidine. The plasmid is digested with PME1 creating the gene cassette which, is then transformed into our yeast strain and each autotrophic marker-gene construct would then integrate into the yeast genome. A sequential transformation was carried out in which a component was transformed and the new strain grown on selective (amino acid drop-out) media and then transformed with another component and grown on a different selective media sequentially until all 4 components were successfully integrated. The final strain could then be grown on a selective media with all 4 amino acid removed which, acts as a final check to ensure all components have been integrated into the genome. </P><P align="justify"> X-gal assays were run as followed. An aliquot of an overnight culture was diluted and incubated at 30C for 5 hours, for cultures containing the gRNA, auxin was added to the aliquot. After 5 hours, the cultures were pelleted and lysed and X-gal was added. The subsequent lysate was incubated at 37C. Blue color begins to show faintly after 30 minutes to 1 hour. A much deeper blue that is much easier to visualize was seen after 8-24 hours either at room temperature or in an incubator. </P><P style="font-size:12px" align="center"> Two cell lysates in vials in which X-gal has been converted into indigo after a 24 hour incubation. Top vial shows that the system responds to the presence of auxin and produces beta-galactosidase. Bottom vial lacks cells that lack the gRNA and therefore does not require auxin for beta-galactosidase to be produced.</P><P style="font-size:12px" align="center"> Two cell lysates that contain X-gal and were incubated overnight at 37C. In the top sample auxin was not added to the pre-lysed culture and in bottom sample auxin was added. The top sample with auxin does not produces beta-galactosidase and X-gal does not turn into indigo while the bottom sample with auxin turns indigo. </P><H2><DIV id="Results"> Results </DIV></H2><P align="justify"> The transformation of the lacZ gene containing a uracil marker proved successful as yeast cells grew on plates or in liquid media lacking uracil. Furthermore, X-gal assays further showed that a constitutively expressing lacZ gene was successfully transformed and integrated into our yeast strain's genome. The transformation and genomic integration of the gRNA, the dCas9 construct and the AFB2 sequences was partially successful. Cell growth on plates lacking uracil and histidine showed that the cells did contain the both a lacZ gene containing a uracil marker as well as a gRNA gene containing a histidine marker. However cell (containing both the lacZ and gRNA genes) cultures were assayed in parallel in which one culture had auxin added while another lacked auxin yet both assays turned blue on occasion while other times only culture that had auxin turned blue and those lacking it remaining colorless (white). It is suspected that this was caused by an integration of the correct autotrophic marker but not the correct construct. </P><H2><DIV id="Conclusion">Conclusion</DIV></H2><P align="justify"> The system that our team developed does indeed work as expected albeit a few oddities. There is a visible signal, in the form of an indigo dye that is produced in response to the presences of a small molecule auxin. With this success in a liquid culture lysate we can now apply our system to a portable paper device. </P><H2><DIV id="Future_Direction">Future Direction</DIV></H2><H2>Introduce high-resolution, easily-quantifiable response gradient using ONPG</H2><P align="justify">ONPG is a molecule used in a liquid assay and can be measured quantitatively with very high-resolution. This works by using a response factor that is able to dissolve in liquid solution and is thus measurable via photospectroscopy. Using this method, our team can precisely measure the impact of varying concentrations of small molecules on our system. We can then use the measured, overall response of our system to predict the amount of analyte present.</P><H2>Alternatively, use a quicker and easily  visible response factor</H2><P align="justify">Currently, the response time of our system utilizing beta-galactosidase to cleave X-gal is somewhat lengthy taking at least 30 minutes to produce a response. By switching over to a rapidly produced, colored signal  response that is visible to the naked eye we hope to make the system easier to use. Colored response signals such as RFP are great because they can be visualized without the need for lab instruments.  </P><H2>Find or design a protein similar to AFB2 that can target other toxins/small molecules </H2><P align="justify">The limitations of our model system is that only Auxin like molecules can be detected. However, with future advancements in the field of protein engineering perhaps more complex molecules can be detected using our system. </P></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>