Team:Northeastern/Notebook
Northeastern IGEM Team: Notebook
6/18/2016
Made with Benchling
Project: Northeastern iGEM 2016
Authors: Asa Budnick
Date: 2016-06-18
Saturday, 6/18
Used competent W6 cells and Plasmids from minipreps and from the kits to do heat shock transformation creating stocks of W6 cells with pAWP78 and pCM66T on kanamycin plates and Bba_E0240 and Bba_K1218011 on chloramphenicol plates. I used the heat shock transduction protocol found here: https://www.addgene.org/plasmid-protocols/bacterial-transformation/
Note: The pAMP78 plasmid tube fell out of the shaker at some point during the 45min incubation so it may not function correctly or it may.
Heatshock transformed DH5alpha with K77302 and pSB1K3, plated on respective antibiotics
Amplified second RBS aliquot
6/19/2016
Made with Benchling
Project: Northeastern iGEM 2016
Authors: Joshua Timmons
Date: 2016-06-19
Sunday, 6/19
Subcultured the K77302 and pSB1K3 to Kan and Chlor
6/20/2016
Made with Benchling
Project: Northeastern iGEM 2016
Authors: David Rosenberg
Date: 2016-06-20
Monday, 6/20
Mini-prepped pSB1K3 and K77302. Did not nano-drop
Reconstituted B0015, the double terminator, and J23100, the Anderson promoter. Did not heat shock
Reconstituted the 30+ primer pairs that showed up
Arranged the 30+ primer pairs that showed up. Threw away the Proteorhodopsin reverse primers
6/21/2016
Made with Benchling
Project: Northeastern iGEM 2016
Authors: Ariela Esmurria
Date: 2016-06-21
Tuesday, 6/21
1hr 5 minute presentation to Dr. Godoy-Carter and Dr Lee-Parsons, albeit with intermittent questions
Olivia and Ariela nano-dropped all the plasmids and recorded their concentrations
6/22/2016
Made with Benchling
Project: Northeastern iGEM 2016
Authors: Ariela Esmurria
Date: 2016-06-22
Wednesday, 6/22
Heat shock transformed B0015 and J23100 in DH5Alpha then (straight from heat transformation protocol) subcultured both into LB with Chloro. There were no Chloro Agar plates left
Autoclaved 500mL LB Broth and 1L of LB Agar
Made 800x stock concentrations of Amp, Kan, and Chloro (water, water, and 95% EtOH). Meant to make 1000x but did it in wrong volume
Poured 200mL of Amp plates, 300mL of Kan, and 500mL of Chloro
Made GelRed staining dye with 5g NaCl, 1L water and 300uL Gel Red
Amplified pSB1C3 backbone using ProteoRho plasmid (30 cycles)
Extension 70 C 20 s
Annealing 72 C 70 s
Ran electrophoresis
IMG_2993.JPG
Image to the left is of pSB1C3 backbone cloned, by Aaron, directly from a plasmid with Phusion. Huge signal, considering it's 20uL of mixture spread across 5 wells. He cut the DNA from the gel and it is now in an eppindorf tube in the 4 deg C. Stained in Gel Red for 30 minutes with gel covered by ice box
Ariela
PCR using Thermo Scientific Phusion High-Fidelity PCR Kit
Need to do PCR for E0240R, E0240TC, J22106, K1218011, pAWP78, pCM66T custom gRNA(maybe??)
Specific Lengths:
E0240- 876bp
J22106- 192bp
K1218011- 4260bp
Custom gRNA- 142bp
E0240R PCR Rxn Table EFFECTIVE
A
B
C
D
E
F
G
H
1
Component
Amount
Total Final Concentration
Forward Primer Temp: 63.1C
Rev. Primer Temp: 66.6C
2
dH20
12.99ul
3
5x Phusion HF Buffer
4ul
1x
Initial Denaturing
98C
30s
4
10mM dNTPs
0.4ul
200uM
Denaturing
98C
7s
5
Forward Primer
0.96ul
0.5uM
Annealing
66.1C
30s
6
Reverse Primer
0.85ul
0.5uM
Extension
72
15s
30cycles
7
Template DNA
1ul of 1-15 dilution
5ng total
Final Extension
72
5min
8
Phusion DNA Poly.
0.2ul
0.02 U/ul
Hold
4
9
TOTAL Rxn Vol
20
Table1
K1218011 PCR Rxn Table INEFFECTIVE
A
B
C
D
E
F
G
H
1
Component
Amount
Total Final Concentration
Forward Primer Temp: 59.6C
Rev. Primer Temp: 61.0C
2
dH20
12.44ul
Saved on 705 pcr block as k1208T
3
5x Phusion HF Buffer
4ul
1x
Initial Denaturing
98C
30s
4
10mM dNTPs
0.4ul
200uM
Denaturing
98C
7s
5
Forward Primer
0.96ul
0.5uM
Annealing
62.6C
30s
6
Reverse Primer
1.0ul
0.5uM
Extension
72
63s
30cycles
7
Template DNA
1ul of a 1-12 dilution
5ng total
Final Extension
72
5min
8
Phusion DNA Poly.
0.2ul
0.02 U/ul
Hold
4
9
TOTAL Rxn Vol
20
Table3
-Asa took out PCR tubes
6/23/2016
Made with Benchling
Project: Northeastern iGEM 2016
Authors: Ariela Esmurria
Date: 2016-06-23
Thursday, 6/23
Heat shock to liquid culture + antibiotic did not work for B0015 or J23100. Heat shock transformed both of these plasmids
Olivia did the Interlab, transforming 5 trials of DH5alpha with positive control, negative control, test device 1, test device 2, and test device 3. They were all plated with glass beads and left in the incubator
I13452 (being amplified for its pSB1A3 backbone) was succesfully transformed yesterday. A colony was selected and subcultured
Five other plasmids from iGEM were plated onto plates with their respective antibiotic. These plasmids were: K1155000. K572005, K1604021, K274200, J22106
Purified pSB1A3 backbone from gel
Ran nanodrop on purified backbone
Concentration 3 ng/uL
260/280 2.43
260/230 0.01
No visible peak
Ariela
Gel Runs of PCR'd stuff
-Used a 1% agarose gel using 2mL of 50x TAE and 98mL of dH20
-Used 4ul of 6x purple loading dye(NEB) for what was initially 20ul, however I think some of it evaporated b/c there wasn't that much in the tubes.
-Ran gel w/ both the K1218011 and E0240 + 2-log DNA ladder(NEB) for 30min @ 130V
-Stained w/ gel red for 20 min
Gel wells are as followed: DNA Ladder, K1218011, E0240
K1218011: Supposed to be ~4200bp
E0240: Supposed to be ~900bp
DNA Ladder 2-log.jpg
6/24/2016
Made with Benchling
Project: Northeastern iGEM 2016
Authors: Joshua Timmons
Date: 2016-06-25
Saturday, 6/25
Heat shocked B0015 or J23100, both to plates this time, failed again
All of the Interlab transformations appeared to have worked (Test 2 has one very small colony along the edge)
K1155000. K572005, K1604021, K274200 were all subcultured to liquid with their respective antibiotic twice (two colonies were selected per plate)
DH5alpha was seeded for another round of comp cells
6/26/2016
Made with Benchling
Project: Northeastern iGEM 2016
Authors: David Rosenberg
Date: 2016-06-26
Sunday, 6/26
Miniprepped K1604021 (Blh composite), K572005 (PR), J23100, K1155000 (Pndh), and J22106 (recA promoter)
Streaked the plates from the Interlab: Test1, Test2, Test3, Negative Control and Positive Control. Will get rid of original plates (making space in fridge) if each restreaking is successful
Made ~25 100uL alliquots of CaCl competent cells using a different/shorter protocol
Diluted the remaining plasmid DNA for B0015. Did PCR on the first and second double terminator (Nox and pR) with both reactions at 65 degrees Celciusannealing with a 10 second elongation time. Ran the post-reaction mixture thru 1 % agarose, and stained in Gel Red for 20 minutes with the following result:
IMG_2998.JPG
I was suspect that thte bands above are just pools of primers at the bottom of the gel, but I had hopes since both bands were slightly above the 0.1 kb band at the bottom of the ladder (and our expected terminators are both around 120-140bps). Purified these both lanes of terminator, as well as E0240R, using the gel purification kit. Added Isopronal (from chem cabinet) since the pieces are < 500 bps
Nanodropped the resulting elutions. The determined concentrations were: 21.5 ng/uL for E0240R, a 260/280 of 1.57, a 260/230 of 0.7; 5.7 ng/uL for Term1 260/280 of 1.06 and 260/230 of 0.22; and Term2 260/280 4.0 and 260/230 0.27
Asa_
I designed PCR tables for most of the crispr/Cas project and ran PCR on the two backbones (pAWP78, pCM66T) and on the recA promoter part (J22106). pAWP78 and J22106 were run upstairs and both tubes were melted, J22106 to an unusable state but pAWP78 was only slightly damaged, pCM66T ran with minimal noticed disturbance but had some condensation.
6/27/2016
Made with Benchling
Project: Northeastern iGEM 2016
Authors: David Rosenberg
Date: 2016-06-28
Tuesday, 6/28
IMG_3004.JPG
Did PCR for the two double terminator regions from the B0015 template (there's ~6 uL of diluted plasmid left, there's probably around 150pg/uL). Annealing at 65 degrees and elongation for 10 seconds, 30 cycles.
IMG_3007.JPG
Did PCR for the anderson promoter from the diluted J23100 plasmid template. Annealing at 64 degrees and elongation for 8 seconds, 30 cycles. The only visible bands in the gel are the primer dimers
Did PCR for the Chlor and Kan backbone using the the same primers but different templates. Tried 10 cycles at 62 degrees celcius then increased to 68 degrees celcius for 25 cycles. Elongation at 1 minute 5 seconds. Left in the cycler overnight (4 degrees)
Did another round of PCR for the double terminators. Left in PCR machine upstairs with a greater concentration of primers (1/10, rather than 1/20). still 65 degrees and 10 seconds elongation.
From now out, based on this Gibson Assembly guide , we'll be doing agarose-gel/staining/UV light on 4-5 uL of the 20uL PCR product. If the results are positive for the desired DNA, we can purify that PCR result using the PCR clean-up kit. This has a higher yield than the gel purificaiton (and is easier). We also won't have to worry about the UV light causing mutations in our fragments
Gibson_test-scale_PCRs.jpg
If the PCRs continue to fail, we can start trying the multi-PCR approach of the guide (linked above). It's approach is outlined in the table above. Attempts PCR at two temperatures 3 degrees apart and with different concentratiosn of DMSO. DMSO promoters denaturing. Also improves amplification of GC rich reasons by structual changes that are caused by DMSO binding to cytosine residues
Aaron_
Nanodropped BBa_K1155000, K1604021, K572005, K274200, and P77302 from yesterday's miniprep.
A
B
C
D
1
Sample
Conc. (ng/uL)
260/280
260/230
2
K1155000
111.2
1.89
2.13
3
K1604021
50
1.89
1.36
4
K572005
32.1
2.08
1.5
5
K274200
178.7
1.91
2.05
6
P77302
90.1
1.88
1.76
Table1
Ran PCR on K1155000 (pndh), K572005 (proteorho), and K1604021 (blh)
10 cycles with 64.8 C annealing (check), 25 with 70 C annealing for all
17 second extension time
Ran electrophoresis on above
(picture == crap)
Double bands for blh, band for PR closer to well than either and very faint; probably origional amplicon for both. Pndh clear band where expected.
Cut out all bands and placed in centrifuge tubes. Stored in freezer.
6/28/2016
Made with Benchling
Project: Northeastern iGEM 2016
Authors: Joshua Timmons
Date: 2016-06-28
Tuesday, 6/28
The csiDp_iLOV gBlock from IDT arrived. I reconstituted this in 50 uL of TE. Given that we recieved 500ng, this comes out to a stock conc of 10 ng/uL
I made up an experimental and control Gibson Assembly, outlined in the table below. This was mixed and put on 50 degrees Celcius on the thermocycler for 20 minutes. Then, it was incubated with some thawed DH5alpha (from the batch just made competent a couple days ago) for 30 minutes. Then heat shocked at 42 degrees for 30 seconds (as opposed to teh 55 seconds that we've been doing). 950 uL of SOC added, then put in incubator w/ vigorous shaking for an hour. 100uL of the 1mL put onto two Chlor plates, and 100uL of the control 1mL onto an Amp plate w/ glass beads
A
B
1
Exp
Control
2
7.8 uL gBlock (78 ng, 0.22 pmoles)
10uL positive control mixture
3
2.5 uL pSB1C3 (63 ng)
4
10uL Gibson Mix
10uL Gibson Mix
Table1
The 6 PCR conditions, which David made, were put at two different reaction temperatures (in 2 PCR machines because I couldn't pull of the gradient option/step). The first was at 67.5 degrees Celsius, 0.5 degrees below the primers annealing temp, and the other was put at 64.5 degrees Celsius, 3.5 degrees below. As David will describe, they had differing concentrations of DMSO, as mentioned in yesterday's entry
I re-pcr'd J22106
Purified blh and proteorhodopsin for PR + blh plasmid from gel.
Ran gel on Chlor backbone, Kan backbone, and BBa_B0015 1 (double term for Nox with His). Got only primers for both backbones (possibly needed to run longer).
20160628_183605.jpg
Purified B0015 1 (accidentally used 50 uL elution buffer instead of 15, neglected to add isopropanol) and nanodropped.
Trial Concentration 260/280 260/230
1 7.1 2.18 0.41
2 10.2 1.75 0.48
6/29/2016
Made with Benchling
Project: Northeastern iGEM 2016
Authors: Joshua Timmons
Date: 2016-06-29
Wednesday, 6/29
The Gibson from yesterday did not work. I am trying the transformation again using two AMP plates (Streaking the newly transformed DH5alpha twice) and one Chlor plate, again trying the experimental plasmid. While it's good news that the positive control didn't work, in addition to the experimental, several things may have failed. Worse case, the freezer is at a higher temp than -20 and this had degraded the exonulceases or T4 ligase in the Gibson Assembly mixture. Best case, the concentration of plasmid was too high. I did not correct for the this best case scenario directly, but am trying the heat shock with a different set of competent cells
IMG_3008.JPG
The DNA to the left, from top to bottom are: Anderson promoter at 64.5 degrees, 5% DMSO; Terminator 2; PCMGGT?; PAW978; J22106. This gel was stained for around 15 minutes, and maybe could have gone longer in the Gel-Red, but I think the top four are probably all negative (with possible exception of the Terminator 2). I'll put that Terminator two should be PCR purified, but, if yield is < 10 ng/uL, it's probably worth tossing/trying again
IMG_3010.JPG
The DNA bands to left, from top to bottom are (all Anderson promoters): 67.5 degrees 0% DMSO, 67.5 degrees 2.5% DMSO, 67.5 degrees 5% DMSO, 64.5 degrees 0% DMSO, 64.5 degrees 2.5% DMSO. Isn't as clear in the picture, but the top band (67.5 degrees 0% DMSO) was very crisp and aparent. The others looked cloudy like possible primer-dimers. This was stained for >30 minutes in Gel-Red, which is apprent by how much brighter the ladder is opposed to the gel above. I'll mark that the 67.5 degrees 0% DMSO should be PCR purified in the table
6/30/16
Made with Benchling
Project: Northeastern iGEM 2016
Authors: Ariela Esmurria
Date: 2016-06-30
Thursday, 6/30
PCR of K1218011 Redone Not Very Effective
A
B
C
D
E
F
G
H
1
Component
Amount
Total Final Concentration
Forward Primer Temp: 59.6C
Rev. Primer Temp: 61.0C
Lid Temp: 70
2
dH20
12.4ul
Saved in 605 as K121ACE
3
5x Phusion HF Buffer
4ul
1x
Initial Denaturing
98C
30s
4
10mM dNTPs
0.4ul
200uM
Denaturing
98C
10s
5
Forward Primer
1 of a 1-10 dilution
0.5uM
Annealing
60C
30s
6
Reverse Primer
1 of a 1-10 dilution
0.5uM
Extension
72
84s (20s x 4.2kB)
30cycles
7
Template DNA
1ul of a 1-12 dilution
5ng total
Final Extension
72
5min
8
Phusion DNA Poly.
0.2ul
0.02 U/ul
Hold
30
9
TOTAL Rxn Vol
20
Table3
Hannah
PCR using Thermo Scientific Phusion High-Fidelity PCR Kit
Chloremphenecol backbone- 6 total reactions with varying temperatures and DMSO concentrations: three with DMSO at 66°C, 69°C, and 72°C, and three w/o DMSO at 66°C, 69°C, and 72°C.
A
B
C
D
E
F
G
1
Component
Amount
Forward Primer Temps: 66°C, 69°C, 72°C
Rev. Primer Temps: 66°C, 69°C, 72°C
Lid Temp: 95
2
RODI H2O
13ul
3
5x Phusion HF Buffer
4ul
Initial Denaturing
98°C
30s
4
10mM dNTPs
0.4ul
Denaturing
98°C
10s
5
Forward Primer (pSBIC3 FWD1)
1 of a 1-10 dilution
Annealing
66°C, 69°C, 72°C
30s
6
Reverse Primer (sBIC3 REV1)
1 of a 1-10 dilution
Extension
72°C
30cycles
7
Template DNA (K77302)
0.5ul of a 1-5 dilution
Final Extension
72°C
5min
8
DMSO*  (only in 3 reactions)
0.6ul
Hold
4°C
9
Phusion DNA Polymerase
0.2ul
10
TOTAL Rxn Vol
20ul
Table1
Kanamycin backbone- 6 total reactions with varying temperatures and DMSO concentrations: three with DMSO at 66°C, 69°C, and 72°C, and three w/o DMSO at 66°C, 69°C, and 72°C.
A
B
C
D
E
F
G
1
Component
Amount
Forward Primer Temps: 66°C, 69°C, 72°C
Rev. Primer Temps: 66°C, 69°C, 72°C
Lid Temp: 95
2
RODI H2O
13ul
3
5x Phusion HF Buffer
4ul
Initial Denaturing
98°C
30s
4
10mM dNTPs
0.4ul
Denaturing
98°C
10s
5
Forward Primer (PR&BIh PR FWD)
1 of a 1-10 dilution
Annealing
66°C, 69°C, 72°C
30s
6
Reverse Primer (PR&BIh PR REV)
1 of a 1-10 dilution
Extension
72°C
30cycles
7
Template DNA (pSB1K3)
0.5ul of a 1-40 dilution
Final Extension
72°C
5min
8
DMSO*  (only in 3 reactions)
0.6ul
Hold
4°C
9
Phusion DNA Polymerase
0.2ul
10
TOTAL Rxn Vol
20ul
Table2
Aaron
Purified B0015 2 , J23100? - neglected to add isopropanol
Nanodropped - all had peaks at 230 and nothing at 260 - discarded
Diluted template and primers for PCR for B0015 2 and J23100
All templates 1:5 (0.5 uL template in 2.5 uL water)
All primers 1:9 (1 uL primer in 9 uL water)
Moved Olivia's Gibson transformation to plates with glass beads and placed in the incubator
7/1/16
Made with Benchling
Project: Northeastern iGEM 2016
Authors: Ariela Esmurria
Date: 2016-07-01
Friday, 7/1
The positive control from yesterday, with dilutions, still failed. Trying the positive control Gibson Assembly again today. Doing 3uL of positive control, 3uL of Gibson Mixture, and putting the PCR tube containing the mixture into a centrifuge tube, full of water, which has been pre-heated on a 50 degree hot plate. Bascially: a mini-water bath. Water bath is to ensure even heating on all sides of the PCR tube. Going to transform with 1:1 and 1:4 dilutions of the mixture post-Gibson, 2uL total volume of mixture. Going to heat shock with the same protocol that we've been trying so far. 42 degrees for 55 seconds (55 because it's a dry plate and it may not be heating fast enough), 950 uL of SOC, 1 hr incubator at high-speed, 100uL onto selection plate with glass beads
Subcultured each of the Interlab strains into 7mL of liquid LB with chloramphenicol. Put in incubator
IMG_3013.JPG
Ran this gel at 110V for 30 minutes (lower voltage for the sake of doing two on one). Contents are below. Cell locations in the table correspond to the part locations in the gel. There are faint bands for both K1218011 and JJJ, however, the primer dimers are the predominant factors
A
B
1
Ladder
Ladder
2
K1218011
Chlor BB 1
3
JJJ
Chlor BB 2
4
Chlor BB 3
5
Chlor BB 6
Chlor BB 4
6
Ladder
Chlor BB 5
Table1
IMG_3015.JPG
Ran this gel at 120V for 30 minutes (this was, obviously, too high a voltage). Contents are in the table below
A
B
1
PR5
2
PR4
3
PR3
4
PR2
5
PR1
PR6
6
Ladder
Ladder
Table2
Aaron_
PCR on his Nox double terminator (B0015 2) and promoter (J23100) with conditions based on last successful attempt
B0015 2
A
B
C
D
E
F
G
1
Component
Amount
Forward Primer Temps: 66°C, 69°C, 72°C
Rev. Primer Temps: 66°C, 69°C, 72°C
Lid Temp: 95
2
RODI H2O
13ul
3
5x Phusion HF Buffer
4ul
Initial Denaturing
98°C
30s
4
10mM dNTPs
0.4ul
Denaturing
98°C
5s
5
Forward Primer (BBa_B0015 FWD 2)
0.5ul of a 1-10 dilution
Annealing
72°C
10s
6
Reverse Primer (BBa_B0015 REV 2)
0.5ul of a 1-10 dilution
Extension
65°C
12s
30cycles
7
Template DNA (B0015)
0.5ul of a 1-2 dilution
Final Extension
72°C
5min
8
DMSO*
0ul
Hold
4°C
9
Phusion DNA Polymerase
0.2ul
10
TOTAL Rxn Vol
19.2ul
Table3
J23100 (check machine)
A
B
C
D
E
F
G
1
Component
Amount
Forward Primer Temps
Rev. Primer Temps: 66°C, 69°C, 72°C
Lid Temp: 95
2
RODI H2O
13ul
3
5x Phusion HF Buffer
4ul
Initial Denaturing
98°C
30s
4
10mM dNTPs
0.4ul
Denaturing
98°C
10s
5
Forward Primer (pSBIC3 FWD1)
0.5 of a 1-10 dilution
Annealing
67.5°C
12s
6
Reverse Primer (sBIC3 REV1)
0.5 of a 1-10 dilution
Extension
72°C
10s
30cycles
7
Template DNA (K77302)
0.5ul of a 1-2 dilution
Final Extension
72°C
5min
8
DMSO*
0ul
Hold
4°C
9
Phusion DNA Polymerase
0.2ul
10
TOTAL Rxn Vol
19.2ul
Table4
7/2/16
Made with Benchling
Project: Northeastern iGEM 2016
Authors: Ariela Esmurria
Date: 2016-07-02
Saturday, 7/2
The Gibson Assembly positive control test failed again, with the reaction taking place in a make-shift PCR-tube waterbath at 50 degrees. Going to make new comp-cells tomorrow with newly made 0.1M CaCl2 and 0.1M CaCl2 + 15% Glycerol. Today I have autoclaved 300mL of 0.1M CaCl2. I will divide this into 200mL and 100mL after it's done autoclaving. I will add 18mL of Glycerol to the 100mL beaker, making it 15% Glycerol. Streaked DH5alpha onto the last remaining LB Agar plate to be subcultured tomorrow for comp-cell generation
Re-subcultured the 5 Interlab strains for growth and measurement tomorrow
Made 500mL of LB Agar. 300mL for Amp plates (for more testing of the Gibson Positive control, and to have fresh plates for that) and 200mL for LB Agar plates without antibiotics.
Aaron
PCR on J23100 promoter again. 4 trials at 2 diff temps and with or without .5 uL (2.5%) DMSO
A
B
C
D
E
F
G
H
1
H2O
13.9 uL
13.4 uL
Initial denaturation
98
30s
2
5x HF buffer
4 uL
4 uL
Denaturation
98
6s
30 cycles
3
dNTPs
0.4 uL
0.4 uL
Annealing
67.5, 70.5
12s
4
BBa_J23100 FWD (1:10 dilution)
0.5 uL
0.5 uL
Extension
72
10s
5
Nox Anderson REV (1:10 dilution)
0.5 uL
0.5 uL
FInal Extension
72
5min
6
J23100 (1:5 dilution)
0.5 uL
0.5 uL
Hold
4
-
7
DMSO
0 uL
0.5 uL
8
Phusion
0.2 uL
0.2 uL
Table2
Electrophoresis on same.
20160702_163426.jpg
Top to bottom: 70.5 C, 2.5% DMSO; 70.5 C, 0% DMSO; 60.5 C 2.5% DMSO; 60.5 C 0% DMSO
PCR on B0015 terminator. 2 trials at 65 and 68 degrees. No DMSO
A
B
C
D
E
F
1
H2O
13.9 uL
Initial denaturation
98
30s
2
5x HF buffer
4 uL
Denaturation
98
6s
3
dNTPs
0.4 uL
Annealing
65,68
13s
4
BBa_B0015 FWD 2 (1:10 dilution)
0.5 uL
Extension
72
12s
5
BBa_B0015 REV 2 (1:10 dilution)
0.5 uL
Final extension
72
5 min
6
B0015 template (1:5) dilution
0.5 uL
Hold
4
7
Phusion
0.2 uL
Table5
Olivia
Worked on calibration steps of Interlab
Used square cuvettes instead of round test tubes so that smaller volume ( 1mL) could be accurately measured by spectrophotometer.
Measured OD600 reference point and FITC standard curve, will be doing cell measurements tomorrow.
OD600 reference point:
A
B
C
1
100
0
2
replicate 1
0.07
0.045
3
replicate 2
0.06
0.041
4
replicate 3
0.061
0.041
5
replicate 4
0.066
0.045
6
average
0.06425
0.043
7
corrected Abs600
0.02125
8
reference OD600
0.01475
9
correction factor
0.694118
Table4
clipboard_2016-07-02_16:06:58.png
FITC standard curve
wavelength range: 395nm-509nm
A
B
C
D
E
F
G
H
I
J
K
L
M
1
2.5
1.25
0.625
0.3125
0.15625
0.078125
0.0390625
0.01953125
0.009765625
0.004882813
0.002441406
0
2
replicate 1
0.325
0.155
0.074
0.036
0.017
0.008
0.003
0.001
0.001
0.001
0
0
3
replicate 2
0.325
0.155
0.075
0.036
0.017
0.009
0.004
0.002
0.001
0.001
0
0
4
replicate 3
0.324
0.155
0.074
0.036
0.017
0.008
0.003
0.001
0
0
0
0
5
replicate 4
0.324
0.156
0.075
0.036
0.017
0.008
0.004
0.001
0
0
0
0
Table3
clipboard_2016-07-02_16:07:35.png
7/3/2016
Made with Benchling
Project: Northeastern iGEM 2016
Authors: David Rosenberg
Date: 2016-07-03
Sunday, 7/3
The plated DH5 alpha from yesterday did not grow. This may be because the glycerol stock at -20 is no longer alive, becuase I didn't pre-heat the LB plate to 37 degrees, or because I didn't (with 100uL) get enough of the DH5 alpha onto the LB plate. I've restreaked two LB agar plates with DH5 alpha today. One from the -80 and another from the -20. This time I incubated the DH5 alpha from the -20 with 1 mL of LB while shaking in the incubator for an hour before streaking onto its plate.
I' ve PCR'ed the SB1C3 backbone from a newly reconstituted part from iGEM plate 1, well 10D. This well corresponds to BBa_K654058. This should work, because every part in iGEM plates 1 thru 3 are on the SB1C3 backbone. I PCR'ed this with six reaction conditions. 3 with DMSO and 3 without. I messed up the PCR gradient. I meant to have the gradient be 1 degree celsius per gradation (meaning the PCR tubes should have been 3 wells apart). Instead, I set the gradation at 3 degrees, and still put the PCR tubes 3 wells apart. This means that my temperatures were 56, 65, and 74 degrees, which is obviously not what I meant to do.
IMG_3022.JPG
Apart from the fact that this gel is completely and absolutely useless, it is a good looking gel. Total separation of ladder bands, relatively linear across the gel, and full use of the gel space. But it is a bunch of crap. The primers could be reduced, it appears, and perhaps more of the template could be added (since none was apparent, even as a small band). 6D is the only PCR tube not on this gel.
For the next attempt at pSB1C3, I will start the PCR annealing temperature at the the lower annealing temp (direct to template). I.e. I will set the thermocycler at 58 degrees for all lanes for 10 cycles, then run it at 62, 65, and 68 (gradations around the final annealing temp) for 25 cycles.
I' ve PCR'ed PR from BBa_K572005 at three reaction temperatures with and without DMSO. I also messed this one up. The temperatures were 60 degrees celcius, 69 degrees celcius and 78 degrees celsius, which, again, is not at all what I had intended to do.
IMG_3024 (1).JPG
Another wonderfully worthless gel. On the plus side, it appears that I have enough primers in each lane. Going to also, as with pSB1C3, try again with a multi-step PCR that begins at a lower temperature then increases, because the two bands with promise, 4 and 5, in the gel to the left, are completely empty.
Aaron
Ran electrophoresis on the PCR products from yesterday for the terminator in the PR+Blh circuit (BBa_B0015 2) and Asa's part labelled AAA
20160703_125815.jpg
Top to bottom: Terminator at 65 C, terminator at 68 C, AAA. Excellent seperation of the ladder bands clearly shows that the bottom two have only primers (AAA is several kb long). The lower temp terminator is higher, if a little messy so I'm assuming we had some amplification.
Cut out the bands and purified the lower temp terminator, then nanodropped. Repeating analysis or reblanking without removing the PCR product produced gradually better results (higher peak, higher conc., lower 260/280, higher 260/230) (data not shown). Cleaned with dH2O and ran a new blank between trials 1 and 2.
A
B
C
D
1
Trial
Concentration
260/280
260/230
2
1
14.2
3.27
0.03
3
2
13.2
3.21
0.03
Table1
Ran PCR on purified electrophoresis product. Made 5 replicates with the same conditions.
A
B
C
D
E
F
1
H2O
12.9 uL
Initial denaturation
98
30s
2
5x HF buffer
4 uL
Denaturation
98
6s
3
dNTPs
0.4 uL
Annealing
65
13s
4
DMSO
0.5 uL
5
BBa_B0015 FWD 2 (1:10 dilution)
0.5 uL
Extension
72
12s
6
BBa_B0015 REV 2 (1:10 dilution)
0.5 uL
Final extension
72
5 min
7
B0015 template (1:10) dilution
1 uL
Hold
4
8
Phusion
0.2 uL
Table5
Nandropped PCR product from successful gel on the Anderson promoter (J23100 at 70.2 C and 2.5% DMSO)
Conc 12.6 ng/uL 260/280 1.40 260/230 0.05
Curve had no peak.
Olivia
Ran cell measurements for interlab
Absorbance settings:
●
min wavelength: 500nm
●
measured wavelength: 600nm
Fluorescence settings
●
min wavelength: 395nm
●
measured wavelength: 509nm
clipboard_2016-07-03_18:37:14.png
clipboard_2016-07-03_18:37:20.png
clipboard_2016-07-03_18:37:27.png
7/4/2016
Made with Benchling
Project: Northeastern iGEM 2016
Authors: Joshua Timmons
Date: 2016-07-04
Monday, 7/4
IMG_3033.JPG
This was the pSB1C3 run at 59 degrees for 10 cycles, followed by 25 cycles at 69 degrees with a gradient of two degrees celcius. Effect temps were 65, 67, 69, and 71 degrees celcius. Picture of the gel not posted, but PR also showed nothing. Found out later that the PR FWD primer was incorrect. Fixed for later run.
IMG_3099.JPG
Tried amplifying pSB1C3 at 63 degrees, 65 degrees and 67 degrees using the NEB PCR kit. Elongation of 1 hour 30 seconds. Using just the lower of the two temperatures.
IMG_3102.JPG
Tried amplifying PR at 58, 60 and 62 degrees with an elongation time of 25 seconds using the NEB PCR kit. Using just the lower of the two temperatures. Checked the part sequence in the registry versus the primers being used (ProteoRho FWD 2 & PR REV 2), and it appears that BBa_K572005 is the incorrect template. The primers are for K773002 instead. Will try again, same temps, but for K773002
Aaron
Ran gel on unknown set of centrifuge tubes labeled 1-5 from the white tip container in the freezer. 1-5 from top to bottom. Many faint bands visible on last 3 lanes between 1100 and 1400 bp ladder rungs. 1,3, and 5 had clearer bands around 100-300 rungs.
20160704_170533.jpg
Mystery DNA. 1-5 from top to botom. Many faint bands visible on last 2 lanes between 400 and 1400 bp. 1,4, and 5 appear to have a < 100 bp construct with primers.
Then I found the terminator 2 PCR tubes that used the purified successful previous gel band as a template, which I originally intended to use, and ran a gel on that. Worked exactly as intended.
20160704_174615.jpg
From top to bottom tubes 1-5. All terminator at 65 C and 2.5% DMSO. Broad, bright bands between 100-200 bp indicating that the terminator contstruct with gibson arms was properly amplified.
7/5/2016
Made with Benchling
Project: Northeastern iGEM 2016
Authors: David Rosenberg
Date: 2016-07-05
Tuesday, 7/5
JOSH
Ran the remaining 15uL of each pSB1C3 amplification through an agarose gel and cut out each band. Did not take a picture because of possible (albeit unlikely) UV damage to the DNA. Each band was in the correct location, just below the 3kbps band, where it would be expected. Will gel purify and use as a template for PCR tomorrow
Made competent cells. Grew up the DH5A in two 50mL tubes. Stopped and put the tubes on ice at different cell densities. The first of the two was stopped (put on ice) at around 0.22 OD600. The other was around 0.32 OD600. For both I centrifuged at 4 degrees and generally followed the protocol outlined here. One change (made after first centrifuge of the 0.22 OD600 cells (marked in blue with "5")) was to centrifuge at 4500rpm, rather than 6000 rpm. This was because the pellet was too firmly attached after the first centrifugation run. Additionally, I no longer used a pipette to resuspend. Instead (for the last 3 resuspensions)) I just tapped the side of the 50mL tube repeatedly until it resuspended. This was after reading several forum posts outline the importance of being gentle. Therefore, for both reasons of density and "gentleness," I believe that the tubes marked with an orange "5" will be the preferrable ones. TBD
Reran PCR on PR, using the correct template: K773002. Used the same temp and gradient as the prior run.
Aaron
Purified the terminator 2 construct analyzed yesterday, combining all of the liquid in tubes 1-4 in tube 4 before conducting the PCR purification protocol. Clear peak at 260.
Concentration 260/280 260/230
140.5 ng/uL 1.90 0.81
7/6/2016
Made with Benchling
Project: Northeastern iGEM 2016
Authors: David Rosenberg
Date: 2016-07-06
Wednesday, 7/6
IMG_3103.JPG
PR when amplified from K773002 at a range of temperatures. All seem to work... The range of different sizes are likely the result of too much template plasmid. The whole range of DNA pieces, from the full length of the plasmid down to the expected size were likely generated after each cycle, resulting in that perfect smear. For that reason, it's probably smart to do another round of PCR with the excized DNA as a template, to ensure that the resulting PR doesn't have any of the incorrectly amplified DNA. The line between the brighest band and the rest of the smear above is from the cut I put into the gel with a razor, i.e. it's not real and not useful.
The pSB1C3 was gel purified and nanodropped. It had a concentration of 15 ng/uL. I diluted the DNA and used it as a template for another round of pSB1C3 PCR. Given the high annealing temp of the lower temp primer (69 degrees) I did a two-step PCR, where it hangs at 72 degrees for 1:30 for both annealing and elongation, then goes back to 98 for 10 seconds for denaturation. Will do a diagnostic gel tomorrow.
Aaron
Ran PCR on the blh coding region with the RBS 3 reverse primer. 2 each 2.5% DMSO and no DMSO. 10 cycles at 55.7 C, 10 cycles at 58.7 C, and 10 cycles at 63.7 C annealing temps.
7/7/2016
Made with Benchling
Project: Northeastern iGEM 2016
Authors: David Rosenberg
Date: 2016-07-07
Thursday, 7/7
Ran Gibson assembly on positive control mix according to Josh's protocol from the 28th:
"I made up an experimental and control Gibson Assembly, outlined in the table below. This was mixed and put on 50 degrees Celcius on the thermocycler for 20 minutes. Then, it was incubated with some thawed DH5alpha (from the batch just made competent a couple days ago) for 30 minutes. Then heat shocked at 42 degrees for 30 seconds (as opposed to teh 55 seconds that we've been doing). 950 uL of SOC added, then put in incubator w/ vigorous shaking for an hour. 100uL of the 1mL put onto two Chlor plates, and 100uL of the control 1mL onto an Amp plate w/ glass beads
A
B
1
Exp
Control
2
7.8 uL gBlock (78 ng, 0.22 pmoles)
10uL positive control mixture
3
2.5 uL pSB1C3 (63 ng)
4
10uL Gibson Mix
10uL Gibson Mix
Table1
"
1.
Modified it so that the total volume was only 4, by taking 2uL of each component instead of 10.
2.
Heated at 50C for 20 mins.
3.
Diluted assembled products 1/4 by adding 12uL of H2O. Added 2uL to each line of competent cells, placed on ice for 30 mins .
4.
Put on ampicillin plates (matches resistance of pos control mix).
Rest of protocol unchanged
Aaron
Ran gel on blh constructs from yesterday. Gel was poured too thin and did not fill entire chamber, DNA ended up at the edge.
7/8/2016
Made with Benchling
Project: Northeastern iGEM 2016
Authors: David Rosenberg
Date: 2016-07-08
Friday, 7/8
Ariela
Comp Cell test w/ new comp cells Josh made.
Spin down the DNA tubes from the Competent Cell Test Kit/Transformation Efficiency Kit to collect all of the DNA into the bottom of each tube prior to use. A quick spin of 20-30 seconds at 8,000-10,000 rpm will be sufficient. Note: There should be 50 µL of DNA in each tube sent in the Kit.
1.
Thawed 2 tubes of 100ul competent cells on ice.
2.
Pipet 2µL of DNA into each microcentrifuge tube. For each concentration, use a separate tube.
3.
Flick the tube gently with your finger to mix. Incubate on ice for 30 minutes.
4.
Heat-shock the cells by placing into the waterbath @ 42C for 1 minute.
5.
Immediately transfer the tubes back to ice, and incubate on ice for 5 minutes.
6.
Added 400 µL of SOC media per tube, and incubate at 37°C for 2 hours. Prepare the agar plates during this time: label them, and add sterile glass beads if using beads to spread the mixture.
7.
Pipet 100 µL from each tube onto the appropriate plate, and spread the mixture evenly across the plate. Incubate at 37°C overnight or approximately 16 hours. Position the plates so the agar side is facing up, and the lid is facing down.
8.
Count the number of colonies on a light field or a dark background, such as a lab bench. Use the following equation to calculate your competent cell efficiency. If you've done triplicates of each sample, use the average cell colony count in the calculation.
○
(colonies on plate) / ng of DNA plated x 1000ng/µg
○
Note: The measurement "ng of DNA plated" refers to how much DNA was plated onto each agar plate, not the total amount of DNA used per transformation. You can calculate this number using the following equation:
○
1 µL x concentration of DNA (refer to vial) x (volume plated / total reaction volume)
-Only plated 1 plate of each. Went in incubator at 7:30 PM.
Observations: When i took tubes out of incubator there was stuff that was pelleted on the bottom of the tubes. I have no idea what it was (maybe cells??). Resuspended whatever it was then plated 100ul onto each plate.
Aaron
Ran a second gel on the blh+RBS constructs. The entire top half seemed to glow slightly red, but the only bands apparent in the samples were primers
20160708_172837.jpg
Top 2 with 2.5% DMSO, bottom 2 without DMSO. All samples were run 10 cycles each at 55.7, 58.7, and 63.7 C.
Ran PCR on both blh and proteorhodopsin with added RBS primers using NEB PCR mix.
A
B
C
D
E
F
G
1
Component
Proeorhodopsin
Blh
Step
Proteorhodopsin
Blh
2
H2O
9.5 uL
9.5 uL
Init denaturation
98 C, 30s
98 C, 30s
3
Primers (1:10 dilution)
RBS 2 FWD, 1uL   PR+Blh PR REV, 1uL
Promoter+RBS+blh FWD, 1 uL RBS 2 REV, 1 uL
denaturation
53.8 C                     55.8 C     20s           57.8 C
51.8 C                      53.8 C      20s                      55.8 C
4
Template (1:5 dilution)
1 uL
1 uL
extension
72 C, 20s
72 C, 20s
5
HF master mix
12.5 uL
12.5 uL
final extension
72C, 2 min
72 C, 2 min
Table1
7/9/2016
Made with Benchling
Project: Northeastern iGEM 2016
Authors: Ariela Esmurria
Date: 2016-07-09
Saturday, 7/9
Ariela
K1218011 PCR: Kinda Effective, but something messed up with the gel
-Left PCR rxn in -20 after PCR was complete, no time to run gel today.
A
B
C
D
E
F
G
H
I
J
1
Component
Amount
Total Final Concentration
Forward Primer Temp: 59.6C
Rev. Primer Temp: 61.0C
Lid Temp: 98
2
dH20
11.8ul
Saved in 605 as K121ACE
3
5x Phusion HF Buffer
4ul
1x
Initial Denaturing
98C
30s
4
10mM dNTPs
0.4ul
200uM
Denaturing
98C
10s
5
Forward Primer
1 of a 1-10 dilution
0.5uM
Annealing
60C
30s
6
Reverse Primer
1 of a 1-10 dilution
0.5uM
Extension
72
84s (20s x 4.2kB)
30cycles
7
Template DNA
1ul of a 1-12 dilution
5ng total
Final Extension
72
5min
8
DMSO
0.6ul
3%
Hold
30
9
Phusion DNA Poly.
0.2ul
0.02 U/ul
10
TOTAL Rxn Vol
20
Table1
Benchling: For some reason benchling doesn't seem to be working on the computers in the lab. I wonder if it's just my login.
Comp Cell Test Results:
50pg/ul: 28 colonies
IMG_3245.JPG
10pg/ul: 5 colonies
IMG_3246.JPG
(colonies on plate) / ng of DNA plated x 1000ng/µg
○
Note: The measurement "ng of DNA plated" refers to how much DNA was plated onto each agar plate, not the total amount of DNA used per transformation. You can calculate this number using the following equation:
○
1 µL x concentration of DNA (refer to vial) x (volume plated / total reaction volume)
Gibson Results:
positive chlor: 9 colonies (do not have red, normal color)
IMG_3242.JPG
Positive+ H.F(?) Amp: no colonies
IMG_3244.JPG
Positive + Amp: 1 colony (looks red in the middle)
IMG_3243.JPG
Aaron
Ran gels on PR and blh with added RBS arms from yesterday.
RBS + Proteorhodopsin:
20160709_132340.jpg
All three PCR lanes had clear bands between rungs 700 and 800, indicating successful amplificaiton.
Blh + RBS :
7/11/2016
Made with Benchling
Project: Northeastern iGEM 2016
Authors: David Rosenberg
Date: 2016-07-11
Monday, 7/11
Aaron
Ran gel on blh template DNA (one labelled K1604021 blh 50 n/u; one labelled blh purified 13.9 ng/uL)
20160711_175105.jpg
Not sure if that's at all visible. Top to bottom 13.9 ng/uL, 50 ng/uL. One band showed up for the higher concentration at around 1200 bp on the ladder. With Gibson arms the blh gene should be ~900 bp. Our initial primers were bad and the new ones were mislabelled, so quite possibly PCR was done with the wrong ones.
Purified proteorhodopsin w/ RBS from yesterday, combining all 3 PCR products in one column. High peak at around 235 nm dropping to and inflection at 260.
Concentration 260/280 260/230
207 ng/uL 1.91 0.63
Replated blh construct from registry on 2 chlor plates
7/11/2016
Made with Benchling
Project: Northeastern iGEM 2016
Authors: Joshua Timmons
Date: 2016-07-11
Monday, 7/11
Given that the Chloro iGEM plasmid was successfully transformed into the newly competent E coli, as was the competency test RFP plasmid, I'm going to again try a Gibson Assembly of the iLOV plasmid and then transform 3 tubes worth of DH5A to see whether that yields some of the expected colonies Using the HiFi Assembly mix
A
B
1
Exp
Control
2
7.8 uL gBlock (78 ng, 0.22 pmoles)
10uL positive control mixture
3
2.5 uL pSB1C3 (63 ng)
4
10uL Gibson Mix
10uL Gibson Mix
Table1
The original table is above. This will adapted slightly, to the version below, to give a higher ratio of insert to backbone. Additionally, the entire volume will be scaled down
A
B
1
Exp
2
1.95 uL gBlock (19.5 ng, 0.22 pmoles)
3
0.20 uL pSB1C3 (5 ng, )
4
2.85uL H2O
5
5uL Gibson Mix
Table2
7/12/2016
Made with Benchling
Project: Northeastern iGEM 2016
Authors: Ariela Esmurria
Date: 2016-07-12
Tuesday, 7/12
Ariela
What to Do: Need to run gel today on K1218011 PCR product; if this doesn't work then I may try using GC-Rich phusion buffer.
-If it doesn't work I may need to try and do a gradient with varying annealing temperatures
Used leftover 1% agarose, did not make new one.
1% Agarose Gel Recipe
A
B
1
TAE 50x
1ml
2
dH20
49ml
3
Agarose
0.5g
4
Total Amount
50ml of 1% gel
Table1
-Use 4ul of 6x loading dye(ThermoFisher) for 20ul
-Run gel w/ K1218011 and 4ul 2-log DNA ladder(NEB) for 30min @ 140V
-Stain w/ gel red for 25 min under bucket.
Results: Gel WORKED!!!! Excised band and stuck it in the fridge for later.
Did a PCR of PR with the RBS forward primer. Ran at 51, 53, 55, 57 degrees Celcius, with 20 second annealing.
Realized that the pSB1C3 backbone, that comes in the linearized kit, is the not the one that we expect. Ordered primers from IDT that will make a region of the backbone for Gibson Assembly. This construction is highlighted in the starvation sensor plasmid design.
7/13/2016
Made with Benchling
Project: Northeastern iGEM 2016
Authors: Joshua Timmons
Date: 2016-07-13
Wednesday, 7/13
IMG_3115.JPG
PCR results from yesterday of PR + RBS turned out well. Still has the weird trailing streaks that plagued the earlier ones, but I think that's an artifact of some gel impurify/size/etc. Did a PCR cleanup on this sample. Had around 200 ng/uL and a 260/230 that was acceptable (although not completely remarkable). Think it may be time for the PCR of this part.
7/14/2016
Made with Benchling
Project: Northeastern iGEM 2016
Authors: Ariela Esmurria
Date: 2016-07-14
Thursday, 7/14
Ariela
Another PCR of K1218011
A
B
C
D
E
F
G
H
1
Components
Forward Primer Temp: 59.6C
Rev. Primer Temp: 61.0C
Lid Temp: 98
2
PCR Master Mix (Neb)
12.5
Saved in 605 as K121ACE
3
H20
8.9
Initial Denaturing
98C
30s
4
Fwd primer
1ul of 1-10 dilution
Denaturing
98C
10s
5
Rev primer
1ul of 1-10 dilution
Annealing
60C
30s
6
Template
1ul of 1-12 dilution
Extension
72
84s (20s x 4.2kB)
30cycles
7
DMSO
0.6 ul
Final Extension
72
5min
8
Total
20 ul
x2 for 40ul rxn
Hold
30
9
Table1
IMG_3117.JPG
Aaaannnnddddd it's upside down. Don't know how that happened. Left three lanes (from this vantage) are Blh. I have PCR clean-up'ed those three tubes and will nano-drop tomorrow. Amplification of the pSB1K3 backbone for Gibson Assembly into the csiDp-iLOV plasmid failed. Will try again with a temp gradient and Benchling annealing temperatures. Will only have time for one Gibson tomorrow, but will maybe be able to have the backbone by Saturday. pSB1C3 (not pSB1K3, because that's actually not available, pSB3K3 (for which we don't have primers) or pSB1K3.m1 (which we have in the backbone kit but for which I can't find sequence data) are available) was PCR'ed at 66, 68, 70, and 72 degrees with iLOV FWD and REV primers for 50 seconds of elongation.
Screen Shot 2016-07-14 at 8.14.28 PM.png
Was going to PCR Blh + RBS but am not going to for two reasons. 1) The current RBS 2 REV primer won't work since there's only 7 bp's of overlap between RBS 2 REV and the Blh I just amplified. That's a denaturation temp of 3 degrees Celcius. And 2) There's already > 20 bps of overlap between the Blh region and the PR + RBS region. A picture of this area, with the Blh REV (lime green), PR FWD (teal), and PR+RBS2 FWD (gray) primers marked is to the left. The lime green and gray region of overlap is the current homology, which isn't too bad.
7/15/2016
Made with Benchling
Project: Northeastern iGEM 2016
Authors: Joshua Timmons
Date: 2016-07-15
Friday, 7/15
Doing the Gibson Assembly today for the PR + Blh plasmid. Nanodropped the 4 parts and got the following concentrations. Used pmols = (weight in ng) x 1,000 / (base pairs x 650 daltons) to determine pmols / uL. This is concentration of DNA in pmols/uL and will be useful for ensuring an equal ratio of parts during the Assembly.
A
B
C
D
1
pSB1C3
63 ng/uL
2070 bps
0.05 pmols/uL
2
Blh
132 ng/uL
890 bps
0.23 pmols/uL
3
PR + RBS
243 ng/uL
780 bps
0.48 pmols/uL
4
Double Term
153 ng/uL
130 bps
1.81 pmols/uL
Table1
Paraphrased notes from NEB's site: "Optimized cloning efficiency is 50-100 ng of vector with 2-3 fold excess of inserts. 5 times more inserts if size is less than 200 bps. Total volume should not exceed 20%."
A
B
C
D
1
Volume (uL)
DNA mass (ng)
DNA mols (pmols)
2
pSB1C3
1.2
76
0.06
3
Blh
0.52
69
0.12
4
PR + RBS
0.25
61
0.12
5
Double Term
0.17
26
0.3
6
Master Mix
5
0
0
7
H20
2.86
0
0
8
2.14
232
0.6
Table2
7/16/2016
Made with Benchling
Project: Northeastern iGEM 2016
Authors: Ariela Esmurria
Date: 2016-07-17
Sunday, 7/17
I killed off all of the comp cells yesterday by closing the freezer before I was done with it. Went downstairs thinking I would remember to put the cells back, and did not. I'll make more tomorrow
The PCR on the pSB1C3 backbone, with the iLOV primers, has failed again. I'll double check the backbone sequence and primers before trying again. May also need some DMSO
7/17/2016
Made with Benchling
Project: Northeastern iGEM 2016
Authors: Joshua Timmons
Date: 2016-07-17
Sunday, 7/17
Subcultured the DH5A and the W6 1/200 into 100mL of LB. Going to shake to OD600's > 0.3
Going to again try the amplification of the pSB1C3 backbone using iLOV BB FWD and iLOV BB REV. Noticed that iLOV BB REV has a - delta G of -14.5, which is very high. Though the NEB PCR master mix is optimized for GC-rich regions, this does not translate to primer binding efficiency. Several sources suggest that the DMSO may help with primer secondary structure. Just in case, and for the sake of time as a backup, I have ordered a new iLOV BB REV with a lower delta g. Additionally, I will run a PCR today with 0.8 uL of DMSO, 3% of volume, hoping that takes care of the issue with the primer secondary structure.
7/18/2016
Made with Benchling
Project: Northeastern iGEM 2016
Authors: Joshua Timmons
Date: 2016-07-19
Tuesday, 7/19
The pSB1C3 for csiDp failed again. I will now wait until the new reverse primer shows up, because the DMSO at 3% did not help.
Heat shock transformed two tubes of DH5A with the Gibson Assembly for PR+Blh. Split each tube onto two Chlor plates, resulting in four plates total in the incubator
7/19/2016
Made with Benchling
Project: Northeastern iGEM 2016
Authors: Asa M. Budnick
Date: 2016-07-19
Tuesday, 7/19
The PR+Blh heat shock transformants from yesterday worked... I have subcultured 4 colonies (one from 2 of the plates and two from a third), into LB with Chlor. I should technically do a colony PCR to check which of the colonies warrants subculturing, but I do not have time and no one else (except for Asa, who's working on his own things), is here, so I'm let someone else do that after mini-prepping the resulting plasmids.
The full Nox construct (with the promoter, GOI, and double terminator all fused), has arrived from IDT. I've reconstituted into 100 uL of TE for a final concentration of 10ng/uL. Going to perform a Gibson Assembly on these pieces today, using the volumes and concentrations beneath. Used pmols = (weight in ng) x 1,000 / (base pairs x 650 daltons) to determine pmols / uL. This is concentration of DNA in pmols/uL and will be useful for ensuring an equal ratio of parts during the Assembly. The Nox Construct is current at 0.01 pmols/uL
A
B
C
D
E
1
Concentration (ng/uL)
Conc (pmol/uL)
Total mols (pmols)
Volume (uL)
2
pSB1C3
76
0.05
0.025
0.5
3
Nox-Construct
10
0.01
0.04
4
4
Water
2.5
5
Gibson Hi-Fi Mix
7
6
Totals
0.065
14
Table1
Heat shock transformed the DH5A with 2uL of the completed Gibson Assembly and plated onto Chloramphenicol plates. Will check for colonies tomorrow.
I have retried the PCR of the pSB1C3 for the Starvation Sensor plasmid. This was with the new pSB1C3 reverse primer that has a substatially more positive delta g (4 vs 14). Ran the PCR at (62, 64, 66, 68, 70). If this works, I will attempt that Gibson again tomorrow.
7/21/2016
Made with Benchling
Project: Northeastern iGEM 2016
Authors: Joshua Timmons
Date: 2016-07-21
Thursday, 7/21
It appears that the Gibson for Nox has also worked. I restreaked two colonies. Additionally, I've set up and run a colony PCR on the two streaked colonies from PR+Blh and the 2 colonies from Nox. Both sets are being run with the verification primers at 62 degrees Celcius with an elongation time of a minute for 30 cycles. This will determine whether the two Gibson have truly worked.
I have gel purified the iLOV Backbone from the gel. I have attached a figure of it post-purification with the gel-purification kit, followed by an additional purification with the PCR-purification kit. These is an abnorammyl large protein spike (alhtough this seems ot have become the norm, for whatever reason. I don't think it will have a major negative effect on the Gibson, since the Gibson guide says to not run a Gibson with > 20% of unpurified product, which would seem to suggest that some spurious proteins/enzyme are acceptable (and this is purified, in theory).
IMG_3128.JPG
Crazy high protein concentration (relative). Regardless, going to try a Gibson this afternoon using this backbone.
IMG_3126.JPG
Band at the expected location. I cut it out of the gel, rather than directly purifying from the PCR mixture, because there were some faint bands beneath the pSB1C3 that I was suspicious of. Also lots of primers (not suprisingly).
The Gibson table for the starvation sensor (csiDp + iLOV), with the newly amplified backbone, is below. Used pmols = (weight in ng) x 1,000 / (base pairs x 650 daltons) to determine pmols / uL.
A
B
C
D
E
1
Concentration (ng/uL)
Conc (pmol/uL)
Total mols (pmols)
Volume (uL)
2
pSB1C3 (modified)
19
0.01
0.03
3.00
3
csiDp-iLOV
10
0.03
0.12
4.00
4
Water
3.00
5
Gibson Hi-Fi Mix
10.00
6
Totals
0.15
20.00
Table1
7/22/2016
Made with Benchling
Project: Northeastern iGEM 2016
Authors: Joshua Timmons
Date: 2016-07-22
Friday, 7/22
Aaron and Ezra have run two colony PCRs. Ezra colony PCR'ed the iLOV construct that was Gibson'ed yesterday. There were two colonies on the plate. Both were restreaked after being PCR'ed. Aaron colony PCR'ed the Nox constructs. There were five total, of which two of the PCR'ed products are below (both failed). Therefore, he PCR'ed the remaining three, re-streaking each onto a new plate.
Screen Shot 2016-07-22 at 5.47.07 PM.png
Colony PCR for 1) PR+Blh "1" 2) PR+Blh "2" 3) Nox "1" 4) Nox "4". PR+Blh "1" looks correct. The others do not. I have restreaked PR+Blh "1" and have dropped/thrown-out the rest. Expected a band around 2,200 bps, which is right where it is at. Will probably re-PCR the streaked colony tomorrow
There were no band on Ezra's colony PCR for iLOV + csiDp. Will wait an see whether the streaked colonies grow by tomorrow, then run again.
7/24/2016
Made with Benchling
Project: Northeastern iGEM 2016
Authors: Ariela Esmurria
Date: 2016-07-24
Sunday, 7/24
Ran three PCRs, two colony PCRs and one attempt at a backbone amplification. One interesting note is that the 4 lanes on the left, the colony PCR for Nox, failed even with their control (lane 5). This was the last of the NEB master mix from that tube, which suggests that it was lacking something. The rest were run with the more full tube of NEB PCR master mix. Despite the negative results for Nox (again), it's worthwhile to run these again with the NEB master mix from the new tube.
IMG_3134.JPG
Lanes, left to right, are: 1) LD, Nox 1, Nox 2, Nox 3, Nox Control, iLOV+csiDp Control. 2) LD, iLOV+csiDp 1, iLOV+csiDp 2, iLOV BB 1, iLOV BB 2, iLOV BB 3
Unlike the Nox, the csiDp + iLOV construct failed outright. The control was successful AND the bands that showed up are at improper heights. It appears that the linearized backbone has somehow self-annealed. Unforunately, the iLOV BB was also unsuccessfully amplified. There is the faint appearance of a backbone band at the proper height, but it's overwhelmed by the large streaks of DNA running through the gel.
Going to re-PCR the iLOV+csiDp backbone with a fresh template. Also going to re-colony PCR the 3 Nox constructs, again with a positive control.
Ariela
Started working on SDS-Page of protein from PR andNOX. Made plate stock of bacteria w/ PR (on chlor plate) and control (lb plate).
7/25/2016
Made with Benchling
Project: Northeastern iGEM 2016
Authors: Joshua Timmons
Date: 2016-07-25
Monday, 7/25
trying the Nox colony PCR's and the iLOV+csiDp Backbone again. Both failed the first time, however, I am now trying to re-do the Nox (same everything, except with the second NEB master mix [since the first one failed]). Also going to try the iLOV+csiDp with a regular template (straight out of the iGEM Plate 1). Will then attempt to run both through a gel.
7/26/2016
Made with Benchling
Project: Northeastern iGEM 2016
Authors: Joshua Timmons
Date: 2016-07-26
Tuesday, 7/26
IMG_3136.JPG
The PCR results, colony of NOX and another attempt at the pSB1C3 backbone. The pSB1C3 failed outright. NOX_3 appears to have worked. I will restreak this bacteria today. Also, I will retry the pSB1C3 backbone for the csiDp + iLOV backbone.
7/28/2016
Made with Benchling
Project: Northeastern iGEM 2016
Authors: Joshua Timmons
Date: 2016-07-28
Thursday, 7/28
Ran a new PCR on the backbone for iLOV using the template from before with the origional primers. Did not dilute the template again. Diluted primers 1:10. 4 identical reactions at 68 C
A
B
C
D
E
F
G
1
Component
Volume (uL)
Step
Temperature (C)
Time
2
H2O
8.5
Init Denaturation
98
30s
3
Master Mix
12.5
Denaturation
98
8s
4
DMSO
1
Annealing
68
20s
5
Fwd Primer
1
Extension
72
50s
6
Rev Primer
1
35 cycles
7
Template
1
Final Extension
72
2min
8
Total
25
Hold
4
Table1
7/30/2016
Made with Benchling
Project: Northeastern iGEM 2016
Authors: David Rosenberg
Date: 2016-07-30
Saturday, 7/30
Subcultured DH5 controls, Blh and PR plasmid containing cells.
Transformed cells with iLOV assembly using the Gibson mix, plated on chlor and no antibiotic plates.
Accidentally plated on Kan instead of chlor
Made 41 new chlor plates.
7/31/2016
Made with Benchling
Project: Northeastern iGEM 2016
Authors: Ariela Esmurria
Date: 2016-07-31
Sunday, 7/31
Examined growth curves for PR and Blh Gibson constructs with chloramphenicol under standard conditions (37 C, setting 4 on shaker). The PR and Blh cells were too turbid so 2 mL ali quots were added to 5 mL fresh LB and let grow for 30 minutes.
A
B
C
D
E
F
G
H
I
J
1
Time
Control 1
Control 2
Control 3
PR 1
PR 2
PR 3
Nox 1
Nox 2
Nox 3
2
0
0.089
0.076
0.102
0.096
0.105
0.109
0.103
0.105
0.1
3
20
0.107
0.094
0.136
0.122
0.13
0.135
0.121
0.132
0.135
4
40
0.135
0.154
0.191
0.198
0.178
0.211
0.196
0.205
0.205
5
60
0.166
0.189
0.237
0.244
0.239
0.28
0.257
0.28
0.287
6
80
0.225
0.265
0.282
0.299
0.305
0.326
0.322
0.321
0.331
7
100
0.252
0.298
0.312
0.352
0.331
0.355
0.371
0.377
0.359
8
120
0.312
0.324
0.365
0.379
0.365
0.397
0.392
0.412
0.406
9
140
0.341
0.379
0.415
0.41
0.421
0.447
0.437
0.455
0.467
10
160
0.413
0.416
0.463
0.452
0.461
0.491
0.47
0.495
0.493
11
180
0.428
0.469
0.483
0.464
0.489
0.499
0.497
0.508
0.525
12
200
0.437
0.457
0.489
0.474
0.484
0.508
0.486
0.505
0.512
13
220
0.479
0.489
0.532
0.513
0.527
0.538
0.526
0.547
0.541
14
250
0.548
0.587
0.602
0.536
0.566
0.581
0.563
0.575
0.593
15
270
0.545
0.553
0.618
0.563
0.58
0.601
0.581
0.614
0.589
16
290
0.558
0.572
0.607
0.609
0.6
0.608
0.605
0.623
0.615
17
310
0.61
0.614
0.636
0.61
0.625
0.638
0.635
0.645
0.64
18
330
0.582
0.598
0.64
0.608
0.633
0.647
0.634
0.654
0.66
19
600
0.621
0.608
0.639
0.615
0.634
0.658
0.658
0.685
0.665
Table1
clipboard_2016-08-06_16:18:02.png
Plot of mean values. Not sure what to make of this. All strains clearly had similar growth. However no log phase is apparent. May have started with too high an OD or needed more time. Also, values above the linear range (.4) were taken as measured instead of diluting - thus the linear correlation of OD600 to microbe concentration breaks down at around 160 minutes. Also notable and unusual that the controls grew more slowly than the cells with added plasmids - this is true until 200 minutes even for control 3 which had a higher starting OD.
Ariela
Did SDS-Page on NOX, PR, and DH5A control. For all there were three dilutions with SDS (1X [no SDS], 1/2X [1/2 SDS], 1/4X [3/4 SDS]). Ran on a gel at 200V for 40 minutes and ran with a ladder.
Protocol from lysis: "If you want to make quick extracts from whole cells, I would suggest to take 1ml of culture at OD(600)=2.0 (or more or less depending on the density of your culture), spin down the cells and resuspend the pellet in 50 µl of 1xSDS loading buffer. Boil for 10 min, then spin at max. speed for 10 min. The sample will be viscous, but if you take 8 µl from the very top of the liquid, you should easily be able to pipet it." came from https://www.researchgate.net/post/How_can_I_carry_out_SDS-PAGE_with_whole_cells_Ecoli
-When lysing the cells there were a couple of problems. First i resuspended in 1mL SDS running buffer when it was supposed to be 50ul (took 950 ul out of tubes), 2nd the OD wasn't exactlly known and OD for NOX and PR was 2x as high as DH5A cells. Centrifugation was run at 4000xg for 3 minutes.
-Left stain overnight, will destain tomorrow night
Gel wells: Ladder, DH5A dilutions(1, 1/2, 1/4), NOX dilutions, PR dilutions).
clipboard_2016-10-10_22:33:06.png
8/1/2016
Made with Benchling
Project: Northeastern iGEM 2016
Authors: Ariela Esmurria
Date: 2016-08-01
Monday, 8/1
Set up another PCR of the iLOV-csiDp backbone. The template is still the linearized backbone from the distribution kit. Ran 8 conditions from around 52 degrees up to 70.
Ariela
Destained. Gel looked good, however I think 30 minutes at 200V was far too long and too high. Suggestion: 170 for 40 minutes.
Pictures after destaining for 1 1/2 hr:
IMG_3255.JPG
IMG_3256.JPG
After 2hr:
clipboard_2016-10-10_22:33:49.png
IMG_3260.JPG
8/2/2016
Made with Benchling
Project: Northeastern iGEM 2016
Authors: Ariela Esmurria
Date: 2016-08-02
Tuesday, 8/2
Ariela
Conjugation Assay:
Using PSBC3 w/ chloramphenicol resistance as receiving bacteria. Host bacteria are PAGGR and PCGGR with kan resistance.
1.
made sure all the OD600 concentrations were roughly the same.
2.
alloquated 500ul of bacteria in 3 ratios of PSBC3:Host [1:1(500ul:500uL), 1:3(500ul:1500ul), 1:9].
3.
Brought volumn of conical tube up to 10mL
4.
Shook at 37 for 18 hr, plated alloquats onto chlor + kan plates
5.
Continued shaking until time reached 24 hr, plated alloquats onto chlor + kan plates
8/3/2016
Made with Benchling
Project: Northeastern iGEM 2016
Authors: Joshua Timmons
Date: 2016-08-03
Wednesday, 8/3
IMG_3146.JPG
The results from yesterday's PCR. Amplifying the pSB1C3 backbone for the iLOV-csiDp construct using the linearized pSB1C3, from iGEM, as a tempalte. Ran at a range of temperatures around 66 degrees. While there are a couple unfortunate bands around 150 and 75 bps, cutting from the gel consistently leads to huge protein spikes (and may have caused our prior issues). Therefore, I just PCR purified these 4 PCR tubes, hoping that the primers, non-specific bands would account for a lower final proportion
IMG_3147.JPG
Results of the DNA purification. High concentration from the 4 wells and a not perfect, but better gel-purification, curve. Will now set up a csiDp-iLOV Gibson assembly.
A
B
C
D
E
1
Concentration (ng/uL)
Conc (pmol/uL)
Total mols (pmols)
Volume (uL)
2
pSB1C3 (modified)
104
0.07
0.04
0.50
3
csiDp-iLOV
10
0.03
0.12
4.00
4
Water
5.50
5
Gibson Hi-Fi Mix
10.00
6
Totals
0.15
20.00
Table1
I Gibson Assembled the above in a heat-plate, centrifuge tube bath at 51 degrees Celcius for 40 minutes before heat shocking into DH5A with selection onto Cloramphenicol plates.
We recieved 4 PETase containing plasmids from Harvard's iGEM team today, however, we don't have the T7 polymerase expressing bacteria necessary to test their constructs. I've emailed Violetta. Depending on her response, I will email Dr. Godoy-Carter, after which I'll email the Harvard iGEM team back about getting the T7 polymerase containing bacteria from them.
8/5/2016
Made with Benchling
Project: Northeastern iGEM 2016
Authors: Joshua Timmons
Date: 2016-08-05
Friday, 8/5
IMG_3150.JPG
Tried to confirm that the two GOI's were at the expected range. While PR+Blh looked good, NOX showed two different sized pieces. It looked like there might be contamination. Therefore, I ran a colony PCR on a bunch of single colonies in the plate
IMG_3153.JPG
Indeed, some of the plasmids had both while some had just the single--correctly sized--part. I re-streaked #4 and will re-PCR the NOX construct and PR+Blh for overnight sequencing overnight at the needed 8 uL of 40-60 ng/uL concentration necessary.
A
B
C
D
E
1
Concentration (ng/uL)
Conc (pmol/uL)
Total mols (pmols)
Volume (uL)
2
pSB1C3 (modified)
104
0.07
0.04
0.50
3
csiDp-iLOV
10
0.03
0.18
6.00
4
Water
1.50
5
Gibson Hi-Fi Mix
8.00
6
Totals
0.21
16.00
Table1
8/6/2016
Made with Benchling
Project: Northeastern iGEM 2016
Authors: Joshua Timmons
Date: 2016-08-06
Saturday, 8/6
Miniprepped E0240 and put into a green marked tube in the plasmids box
Re-PCR'ed the iLOV BB and purified. Resulted in 76 ng/uL with a good looking curve
PCR purified printed csiDp-iLOV. Resulted in 13.3 ng.uL with a shit curve.
A
B
C
D
E
1
Concentration (ng/uL)
Conc (pmol/uL)
Total mols (pmols)
Volume (uL)
2
pSB1C3 (modified)
76
0.05
0.053
1.00
3
csiDp-iLOV
13.3
0.04
0.141
4.00
4
Water
2.00
5
Gibson Hi-Fi Mix
7.00
6
Totals
0.194
14.00
Table1
Performed the Gibson Assembly above and heat shocked DH5A
Harvard Assay
Made with Benchling
Project: Northeastern iGEM 2016
Authors: Ariela Esmurria
Dates: 2016-08-08 to 2016-08-10
Monday, 8/8
Ariela
pos cont: 13H well on plate 4 (BBa_E0044) has pSB1A3 backbone containing amp resistance
neg cont: no plasmid
Transformations of Harvard Plasmids into T7 promoter-compatible expression cells (B-something or other), ask Josh:
1) sfGFP
Construct of pSB1A3 + T7 promoter + RBS 34 + sfGFP + PETase + His-tag
Concentration: 30 ng/ul
2) sfGFP + ompT
Construct of pSB1A3 + T7 promoter + RBS 34 + ompT + sfGFP + PETase + His-tag
Concentration: 37 ng/ul
3) sfGFP + pelB
Construct of pSB1A3 + T7 promoter + RBS 34 + pelB + sfGFP + PETase + His-tag
Concentration: 30 ng/ul
4) sfGFP + YebF
Construct of pSB1A3 + T7 promoter + RBS 34 + YebF + sfGFP + PETase + His-tag
Concentration: 23 ng/ul
Protocol:
-Place 50ul of cells into each tube (6 in total w/ controls)
-Add 1ul of DNA to each respective tube
-Let sit on ice for 15 minutes
-Heat shock at 42C for 1 minute
-Immediately place back on ice for 5 minutes
-Add 200ul of LB to each tube
-Let shake at 37C for 45 min
-Plate 100ul on Amp Plates and let grow overnight
Harvard Assay
Made with Benchling
Project: Northeastern iGEM 2016
Authors: Ariela Esmurria
Dates: 2016-08-08 to 2016-08-10
Tuesday, 8/9
IMG_3270.JPG
IMG_3269.JPG
IMG_3268.JPG
IMG_3267.JPG
IMG_3266.JPG
A
B
C
1
Plate
Number of Colonies
2
- Control
0
3
+ Control
0
????
4
1
45
5
2
37
6
3
38
7
4
83
Table1
+ Control having no colonies is weird because they were supposed to have been transformed with a backbone containing amp resistance. Perhaps not enough DNA used in transformation.
Today I am making 3ml liquid cultures (3x) of each, including that of the bacteria w/ no plasmid.
-Plan to induce w/ IPTG tomorrow morning and then read fluorescence w/ plate reader
IPTG Induction Protocol
1.
Pick fresh colonies and incubate at 37°C until OD600 reaches 0.4–0.8.
2.
Induce with 10ul of a 100 mM stock of IPTG, and induce for 3 to 5 hours at 37°C.
8/10/2016
Made with Benchling
Project: Northeastern iGEM 2016
Authors: Asa M. Budnick
Date: 2016-08-10
Wednesday, 8/10
Trying another Gibson for the starvation sensor, this time with a PCR'ed backbone
A
B
C
D
E
1
Concentration (ng/uL)
Conc (pmol/uL)
Total mols (pmols)
Volume (uL)
2
pSB1C3 (modified)
76
0.05
0.066
1.25
3
csiDp-iLOV
164
0.44
0.218
0.50
4
Water
4.00
5
Gibson Hi-Fi Mix
5.75
6
Totals
0.284
11.50
Table1
Did the above Gibson Assembly and then heat shocked and plated on Chloro.
Subcultured some DH5A and NOX for His-tag purification tomorrow + coomasii
Harvard Assay
Made with Benchling
Project: Northeastern iGEM 2016
Authors: Ariela Esmurria
Dates: 2016-08-08 to 2016-08-10
Wednesday, 8/10
Excitation: 485/20, Emission: 528/20
clipboard_2016-08-15_08:56:37.png
Fluorescence
clipboard_2016-08-15_15:10:56.png
OD @ 630
Control (BL21 Cells):
A) A1, A2, A3
B) B1, B2, B3
C) C1, C2, C3
sfGFP (1)
A) F1, F2, F3
B) G1, G2, G3
C) H1, H2, H3
sfGFP + ompT (2)
A) A5, A6, A7
B) B5, B6, B7
C) C5, C6, C7
sfGFP + pelB (3)
A) F5, F6, F7
B) G5, G6, G7
C) H5, H6, H7
sfGFP + YebF (4)
A) A9, A10, A11
B) B9, B10, B11
C) C9, C10, C10
Data Analysis
One Way Anova done with: http://vassarstats.net/anova1u.html
8/12/2016
Made with Benchling
Project: Northeastern iGEM 2016
Authors: David Rosenberg
Date: 2016-08-12
Friday, 8/12
Plan for today is to try purify some HIS proteins through the columns.
Will run 6 lanes in SDS gel: 1) DH5A 2) DH5A His-purified First 3) DH5A His-purified Second 4) Nox-plasmid in SDS 5) Nox His-purified First 6) Nox His-purified Second
Heat shocked comp cells with the new starvation reporter plasmid. There are now 5 boxes of comp cells in the lower compartment of the -80 because of biochemstry. Hopefully what I used was DH5alpha, as it was not ours. Cells were in 42 C for 90 instead of 45 seconds, as for some reason our printed protocol calls for 2 minute incubation at 42.
8/17/2016
Made with Benchling
Project: Northeastern iGEM 2016
Authors: Joshua Timmons
Date: 2016-08-17
Wednesday, 8/17
Many "successful" colonies for the starvation sensor after heat shocking into BL21 last night, rather than DH5alpha. Weird Colony-PCR results though, suggesting the creation of something other that what was intended. The first gel is below. This PCR will be performed again.
IMG_3180.JPG
the expected band size is around 800bps. Here, however, the bands are hovering around 300-400 bps.
8/18/2016
Made with Benchling
Project: Northeastern iGEM 2016
Authors: Ariela Esmurria
Date: 2016-08-18
Thursday, 8/18
SDS-Page of PR and Nox Take 2
Cell Lysis Protocol
-Grew overnight cultures of bacteria w/ parts and control
-Spun down cells from 2 ml of culture
-Resuspended in 50ul of SDS buffer
-Boiled for 10 minutes at 100C using a heat block
-Spun down cell lysate for 11 minutes at 13200rmp
-Look 5ul off top of supernatant and placed into pcr tube
Gel Run and Protocol
-Added 5 ul of 2x Laemmli sample buffer to each 5ul sample
-Heated at 95C for 5 minutes in water bath
-Placed into gel and ran at 170V for 40 minutes
Nox ~ 55 KDA
PR ~ 37 KDA
Destained on 8/19 for ~1 1/2 hr
Gel: Ladder, Nox, Control, PR
-Gel was fairly faint, might have destained too long
-Lysis didn't seem to work very well
-Control, Nox, and PR all look the same
IMG_3285.JPG
8/23/2016
Made with Benchling
Project: Northeastern iGEM 2016
Authors: Ariela Esmurria
Date: 2016-08-23
Tuesday, 8/23
Ariela
K1218011 Effective
A
B
C
D
E
F
G
H
1
Component
Amount (uL)
Total Final Concentration
2
Phusion Flash
20
3
Initial Denaturing
98C
30s
4
Denaturing
98C
7s
5
Forward Primer
2ul of 1:10 dilution
Annealing
58
30s
6
Reverse Primer
2ul of 1:10 dilution
Extension
72
20s
30cycles
7
Template DNA
2ul of 1:10 dilution
10ng total
Final Extension
72
5min
8
DMSO
1.2
3%
Hold
4
9
TOTAL Rxn Vol
40
Table5
A
B
C
D
E
F
G
H
I
J
1
NOX (Days 1 and 2)
2
NAD+
31000
28000
27000
21000
22000
22000
3
NADH
21000
23000
20000
21000
20000
18000
4
1.476190476
1.217391304
1.35
1
1.1
1.222222222
1.227634
5
6
Control (Days 1 and 2)
7
NAD+
29000
28000
26000
8
NADH
22000
21000
21000
9
1.318181818
1.333333333
1.238095238
1.296536797
Table1
8/27/16
Made with Benchling
Project: Northeastern iGEM 2016
Authors: David Rosenberg
Date: 2016-08-27
Saturday, 8/27
Finished mixing acetate solution based on Logan's simple formula.
No cells grew on any of the transformations from yesterday.
Acetate media for MEC
Introduction
Protocol based on Logan's simple formula, 1 L batch
Materials
Sodium acetate
BOD nutrient buffer solution
Dibasic sodium phosphate
Monobasic potassium phosphate
Procedure
Stir 1 L tap water overnight to remove chloride
Add nutrients
A
B
1
NaAc x 3H2O
1.66 g
2
Nutrient buffer
1 pillow
3
Na2HPO4
1.25 g
4
KH2PO4
6.7 g
Table1
Stir
8/28/16
Made with Benchling
Project: Northeastern iGEM 2016
Authors: David Rosenberg
Date: 2016-08-28
Sunday, 8/28
Large amount of precipitate in media; may be due to substituting less soluble KH2PO4 for NaH2PO4, retrying with 0.2 M NaH2PO4 solution
1:18 put 750 mL tap water on stir
Acetate media for MEC
Introduction
Protocol based on Logan's simple formula, 1 L batch
Materials
Sodium acetate
BOD nutrient buffer solution
Dibasic sodium phosphate
Monobasic sodium phosphate
Procedure
Stir 1 L tap water overnight to remove chloride
Add nutrients
A
B
1
NaAc x 3H2O
1.66 g
2
Nutrient buffer
1 pillow
3
Na2HPO4
1.25 g
4
NaH2PO4
.2 M * 250 mL
Table1
Stir
8/24/2016
Made with Benchling
Project: Northeastern iGEM 2016
Authors: Ariela Esmurria
Date: 2016-08-29
Monday, 8/29
Golden Gate Assembly (Used Golden protocol in thermocycler)
A
B
C
D
1
Successful Golden Gate Mixture
8.24 (Active)
8.24 (inactive)
2
T4 Buffer - Mix well
2
2
3
T4 Ligase
1.7
1.7
4
Eco 31I
1.5
1.5
5
BSA (Bovine Serum Albumin) 20mg/l
1.5
1.5
6
1:3 of E0420REP/TC
2
2
7
1:8 J22106
2
2
8
pCM66T or pAWP78
2.7
2.7
9
4.3 H20
0.6
0.6
10
gRNA
2
2
11
k1218011
4
4
Table3
clipboard_2016-07-16_12:54:44.png
1.675 uL T4
8/25/2016
Made with Benchling
Project: Northeastern iGEM 2016
Authors: Ariela Esmurria
Dates: 2016-08-29 to 2016-08-30
Monday, 8/29
Transformtion of W6 E. coli w/ Golden Gate assembly plasmid
A
B
1
Active Plasmid
Inactive Plasmid
2
1ul in rxn
1ul in rxn
3
3ul in rxn
3ul in rxn
4
5ul in rxn
5ul in rxn
Table1
Protocol
-Thaw W6 cells on ice
-Place specified amounts of Golden Gate Assembly plasmids into clean microcent. tubes
-Add 50ul of cells to each tube and mix thoroughly, let sit on ice for 15 minutes
-Shock in 42C heat block/bath for 1 minute, immediately put back on ice for 5 minutes
-Add 200ul of LB or superbroth (LB was used on 8/25)
-Shake at 37C for 45 min -1 hr
-Plate on Kanamycin plates
-Grow at 37C overnight
8/25/2016
Made with Benchling
Project: Northeastern iGEM 2016
Authors: Ariela Esmurria
Dates: 2016-08-29 to 2016-08-30
Tuesday, 8/30
Results!
22735.jpg
9/3/2016
Made with Benchling
Project: Northeastern iGEM 2016
Authors: Ariela Esmurria
Date: 2016-09-05
Monday, 9/5
Protein Isolation of PR +BLH in BL21
-Grew liquid culture overnight
-Spun down cells until the mass reached 0.02g, took off supernatant (12,000rpm for 10 minutes @ time until mass reached)
-Added 5ul of 1X BugBuster Lysis Buffer per 1mg of cells (100 ul total added), then pipetted up and down to resuspend
-Shook on shaking plate for 20 minutes @ room temp
-Spun down at 13,000rpm for 25 minutes to spin down cell debris
*For some reason it still looked like a pellet of cells and not cell debris (I dont really know ???)
-Collected supernatant into clean microcentrifuge tube, stored at -20
I also set up overnight cultures for BL21 bacteria containing Nox and then a Bl21 negative control.
9/5/2016
Made with Benchling
Project: Northeastern iGEM 2016
Authors: Ariela Esmurria
Date: 2016-09-05
Monday, 9/5
Took out overnight cultures (grew for ~48 hrs instead of 24)
-Spun down cells @ 5000g for 15 minutes, took off supernatant
-Weighed cell pellet mass in tubes (0.048g for Nox and 0.078g for Bl21)
-Added 5ul of 1X BugBuster Lysis Buffer per 1mg of cells (240ul for Nox and 390ul for BL21), then pipetted up and down to resuspend
-Shook on shaking plate for 20 minutes @ room temp
-Spun down at 13,000rpm for 25 minutes to spin down cell debris
-Collected supernatant into clean microcentrifuge tube, store at -20
SDS-Page
-Ran SDS/PAGE gel at 120V for 60 minutes
-Stained with Coomassie brilliant Blue for 1 1/2 hr
-Destained until gel bands were clear
-Gel Order: Ladder, PR + BLH, Bl21, Nox
9/11/2016
Made with Benchling
Project: Northeastern iGEM 2016
Authors: Ariela Esmurria
Date: 2016-09-16
Friday, 9/16
Ran Coomassie Stain for His tag purification of PR + BLH, Nox, and Bl21
Order: PR + BLH, NOX, Bl21
To Bind loading Dye to Protein:
1.
Heat water bath or heat block to 95-100 C, (I like to put microcentrifuge tubes filled with water into the heat block spaces while heating up)
2.
Put 10ul of sample into pcr tube
3.
Add 10 ul of 2x laemmli sample buffer to tube and mix well
4.
Heat sample for 5 minutes at whatever temperature
5.
Set up gel box
6.
Add clean sds page running buffer into middle of gel box until it overflows, clean out gel wells using a disposable bulb pipette
7.
You can use used sds page running buffer on the outside of the gel box, fill until 1-2 cm below lowest point ontop of the box
8.
Load samples into each well, add ladder, record order
9.
Run at 160 V for ~45 minutes, make sure blue lines from loading dye are ~ 1cm from bottom
10.
Crack cast of the gel, gently place into container
11.
Fill container with Coomassie blue stain and cover
12.
Put shaker table on lowest speed and stain for 1.5 hr
13.
Carefully pour liquid out into waste bottle
14.
Wash gel with destain for 5 minutes, pour out liquid into waste bottle
15.
Add more destain liquid and set on shaker for 30 minutes, pour out waste and refil container with destain
16.
Respeat every 30 minutes until gel is no longer blue, but bands are visible
17.
Take picture of gel, dispose of gel into biohazard bin
clipboard_2016-10-10_21:53:05.png
10/1/2016
Made with Benchling
Project: Northeastern iGEM 2016
Authors: Ariela Esmurria
Date: 2016-10-01
Saturday, 10/1
Ran colony PCR on Nox plasmid from solid cultures. 2 reactions each with and without DMSO. Denature-anneal-extend cycle repeated 35 times.
A
B
C
D
E
F
1
Component
Volume (uL)
Step
Temp (C)
Time
2
Rodi H2O
9.5-10.5
Init Denaturation
98
30s
3
Master Mix
12.5
Denaturation
98
10s
4
pSB1C3 FWD 1 primer (diluted 1:10)
1
Annealing
67
23s
5
B0015 REV 1 primer (diluted 1:10)
1
Extension
72
65s
6
DMSO
0-1
Final extension
72
10min
7
Total
25
hold
4
Table1
Resuspended the Nox BB primers from 8/28 for biobrick submission in 248 uL or RODI each (100 uM concentration).
Ran PCR on Nox BB. Two identical reactions with DMSO. Denature-anneal-extension cycle repeated 35 times.
A
B
C
D
E
F
1
Component
Volume (uL)
Step
Temp (C)
Time
2
Rodi H2O
9.5
Init Denaturation
98
30s
3
Master Mix
12.5
Denaturation
98
10s
4
Nox BB FWD primer (diluted 1:100)
1
Annealing
55.8 (check)
20s (check)
5
Nox BB REV primer (diluted 1:100)
1
Extension
72
30s
6
Biobrick Nox submission Gblock diluted (1:5)
1
Final extension
72
5min
7
DMSO
1
hold
4
8
Total
26
Table2
10/2/2016
Made with Benchling
Project: Northeastern iGEM 2016
Authors: David Rosenberg
Date: 2016-10-02
Sunday, 10/2
Ran gel on Nox biobrick from yesterday.
20161002_132757.jpg
This isn't a great picture, but both Nox samples ended up exactly where expected.
Did colony PCR on PR + blh plasmid. 1 with and 1 w/o DMSO.
A
B
C
D
E
F
1
Component
Volume (uL)
Step
Temp (C)
Time
2
RODI H2O
9.5-10.5
Init Denaturation
98
30s
3
Master Mix
12.5
Denaturation
98
7s
4
pSB1C3 FWD 1 primer (1:10)
1
Annealing
67 (check)
20s (check)
5
B0015 1 REV primer (1:10)
1
Extension
72
80s
6
DMSO
0-1
Final Extension
72
5min (check)
7
Total
25
Hold
4
Table1
Ran gel on result:
10/11/16
Project: Northeastern iGEM 2016
Authors: David Rosenberg
Date: 2016-10-11
Tuesday, 10/11
Ran gel on CsidP biobrick. Box was set too high and DNA fell off.
Ran EcoRI/PstI digests on the purified Nox biobrick and a plasmid with the pSB1C3 backbone from miniprep. Initial concentrations were 52.5 ng/uL for Nox and 122 ng/uL for the plasmid.
A
B
C
1
Component
Volume (Nox)
Volume (plasmid)
2
RODI  H2O
44 uL
50 uL
3
FD buffer
4 uL
4 uL
4
DNA
10 uL
4 uL
5
EcoRI
1 uL
1 uL
6
PstI
1 uL
1 uL
7
Total
60 uL
60 uL
Table1
Gel on both digests and CsidP
Digest gel 1.jpg
Bands where they should be for Nox and backbone, though a little faint. Nothing for CsiDp.
10/12/2016
Project: Northeastern iGEM 2016
Authors: David Rosenberg
Date: 2016-10-12
Wednesday, 10/12
Analyzed concentration from Nox and pSB1C3 digests. 9.5 and 3.1 ng/uL respectively. Ran another gel with entire digest volume of each to purify more DNA. Bands cut out and pooled.
20161013_000120.jpg
Nox has a band around 1300 bp, in agreement with the length of the expected construct. No clear bands for pSB1C3 backbone, even though it showed up on the previous gel.
10/13/2016
Project: Northeastern iGEM 2016
Authors: David Rosenberg
Date: 2016-10-13
Thursday, 10/13
Ran another digest on the plasmid backbone containing pSB1C3. Ran entirety through gel as before.
20161013_215344.jpg
The entirety is pretty faint, but there are bands where expected at around 2kb.
Purified Nox and pSB1C3, using 2 elutions of 10 uL for both.
10/14/2016
Project: Northeastern iGEM 2016
Authors: David Rosenberg
Date: 2016-10-14
Friday, 10/14
Analyzed concentrations of Nox and pSB1C3 digests. 5.9 and 10.1 ng/uL, respectively
Performed ligation in 2 x 20 uL reactions. 10 uL of Nox and 5 uL of pSB1C3 were added for each. Incubated at room temp for 1 hour.
Ligation Protocol WITH T4 DNA Ligase (M0202)
Introduction
Please see the NEB website for supporting information on this protocol.
Materials
10X T4 DNA Ligase Reaction Buffer
T4 DNA Ligase
Vector DNA (4kb)
Insert DNA (1kb)
Nuclease-free water
Procedure
Set up the T4 DNA Ligase Reaction
Note: T4 DNA Ligase should be added last. The table shows a ligation using a molar ratio of 1:3 vector to insert for the indicated DNA sizes. Use NEB calculator to calculate molar ratios.
Thaw the T4 DNA Ligase Buffer and resuspend at room temperature.
Tip: Alicuote the 10x buffer less concentrated so when thawing, the DTT gets soluble more easily.
Set up the following reaction in a microcentrifuge tube on ice:
A
B
1
Component
Volume (μl)
2
10X T4 DNA Ligase Buffer
2
3
Vector DNA: 50 ng (0.020 pmol)
4
Insert DNA: 37.5 ng (0.060 pmol)
5
Nuclease-free water
17
6
T4 DNA Ligase
1
7
Total
20
Table1
Gently mix the reaction by pipetting up and down and microfuge briefly.
For cohesive (sticky) ends, incubate at 16°C overnight or room temperature for 10 minutes. For blunt ends or single base overhangs, incubate at 16°C overnight or room temperature for 2 hours.
Heat inactivate at 65 degrees C for 10 minutes.
00:10:00
Chill on ice and transform 1-5 μl of the reaction into 50 μl competent cells.
Use 25 uL DH5α cells, and add 2 uL of reaction mixture.
Ligation Protocol WITH T4 DNA Ligase (M0202)
Introduction
Please see the NEB website for supporting information on this protocol.
Materials
10X T4 DNA Ligase Reaction Buffer
T4 DNA Ligase
Vector DNA (4kb)
Insert DNA (1kb)
Nuclease-free water
Procedure
Set up the T4 DNA Ligase Reaction
Note: T4 DNA Ligase should be added last. The table shows a ligation using a molar ratio of 1:3 vector to insert for the indicated DNA sizes. Use NEB calculator to calculate molar ratios.
Thaw the T4 DNA Ligase Buffer and resuspend at room temperature.
Tip: Alicuote the 10x buffer less concentrated so when thawing, the DTT gets soluble more easily.
Set up the following reaction in a microcentrifuge tube on ice:
A
B
1
Component
Volume (μl)
2
10X T4 DNA Ligase Buffer
2
3
Vector DNA: 50 ng (0.020 pmol)
4
Insert DNA: 37.5 ng (0.060 pmol)
5
Nuclease-free water
17
6
T4 DNA Ligase
1
7
Total
20
Table1
Gently mix the reaction by pipetting up and down and microfuge briefly.
For cohesive (sticky) ends, incubate at 16°C overnight or room temperature for 10 minutes. For blunt ends or single base overhangs, incubate at 16°C overnight or room temperature for 2 hours.
Heat inactivate at 65 degrees C for 10 minutes.
00:10:00
Chill on ice and transform 1-5 μl of the reaction into 50 μl competent cells.
Use 25 uL DH5α cells, and add 2 uL of reaction mixture.
10/15/2016
Project: Northeastern iGEM 2016
Authors: David Rosenberg
Date: 2016-10-15
Saturday, 10/15
Performed heat shock on BL21 comp cells with nox biobrick and with PR+blh plasmid.
Asa ran DO assay between control and Nox transformed BL21
200ml of PBS solution was agitated on a magnetic stirring tray for 6 hours in order to aerate the solution.  Nox plasmid transformed Bl21 E. coli were inoculated into 30ml of LB +chloramphenicol at the same time that untransformed BL21 E. coli were inoculated into 30ml of LB. These cultures were grown at 37 degrees celsius for four hours. 200ml of PBS solution was agitated on a magnetic stirring tray for 6 hours in order to aerate the solution.  Optical density of the cultures was measured and the cultures were normalized to an OD of .018.
A
B
C
1
NOX   Dissolved Oxygen Consumption
2
Time(min)
Dissolved Oxygen (ppm)
3
NOX
BL21
4
0
9.062
8.872
5
0.5
9.041
8.851
6
1
9.151
8.881
7
1.5
8.704
8.804
8
2
8.008
8.488
9
2.5
7.479
8.579
10
3
7.06
8.45
11
3.5
6.763
8.413
12
4
6.529
8.439
13
4.5
6.383
8.403
14
5
6.2
8.37
15
5.5
6.013
8.383
16
6
5.988
8.378
17
6.5
5.884
8.344
18
7
5.781
8.311
19
7.5
5.705
8.325
20
8
5.689
8.319
21
8.5
5.6
8.36
22
9
5.529
8.279
23
9.5
5.405
8.335
24
10
5.357
8.397
25
10.5
5.311
8.291
26
11
5.118
8.348
27
11.5
5.179
8.359
28
12
5.087
8.347
29
12.5
5.067
8.257
30
13
5.056
8.406
31
13.5
5.061
8.361
32
14
4.998
8.388
33
14.5
4.937
8.367
34
15
4.938
8.368
35
15.5
4.845
8.335
36
16
4.879
8.409
37
16.5
4.886
8.406
38
17
4.855
8.385
39
17.5
4.835
8.385
40
18
4.888
8.338
41
18.5
4.739
8.409
42
19
4.787
8.417
43
19.5
4.744
8.364
44
20
4.757
8.457
45
20.5
4.778
8.438
46
21
4.667
8.507
47
21.5
4.653
8.463
48
22
4.606
8.376
49
22.5
4.627
8.437
50
23
4.624
8.454
51
23.5
4.595
8.455
52
24
4.55
8.46
53
24.5
4.597
8.477
54
25
4.551
8.451
55
25.5
4.566
8.476
56
26
4.519
8.429
57
26.5
4.469
8.429
58
27
4.523
8.433
59
27.5
4.5
8.48
60
28
4.528
8.508
61
28.5
4.491
8.481
62
29
4.467
8.487
Table8
Heat shock transformation
Introduction
This protocol is from Addgene (https://www.addgene.org/plasmid-protocols/bacterial-transformation/)
Materials
Procedure
Take comp cells out of -80 C freezer and thaw on ice 20-30 minutes
Mix 1-5 uL DNA into 20-50 uL comp cells in a microcentrifuge tube. Mix gently.
Incubate in ice 20-30 minutes
Heat shock at 42 C for 5 seconds
Quickly return cells to ice. incubate 2 minutes
Add 250-1,000 uL LB broth
Incubate 45 minutes at 37 C
Plate 50 uL on agar and/or innoculate 1-100 uL in liquid culture
NOX DO Colony Forming Units Comparison
Project: Northeastern iGEM 2016
Authors: Asa M. Budnick
Date: 2016-10-16
Sunday, 10/16
After Optical density normalization some liquid culture of both NOX and BL21 bacteria were reserved, they were diluted in a five tier 1:10 serial dilution and 50ul of dilution was plated in 4 replicates for each of the 1:1000 and 1:100000 dilutions. The 1:100000 dilutions were too low and variable for the NOX group so comparisons were drawn using the 1:1000 Dilution
A
B
C
D
E
F
G
H
I
1
Colony   Counts Per 50uL of Liquid Culture
2
Dilution:
1e-3
1e-5
3
Replicate:
1
2
3
4
1
2
3
4
4
NOX
151
141
159
157
5
5
2
0
5
Bl21
220
223
193
197
18
17
13
23
Table2
A
B
C
D
E
1
Average   CFU Comparison for 1:1000 Dilution
2
Average (CFU/ml)
Standard Error (CFU/ml)
Lower End of 95% Confidence Interval (CFU/ML
Upper End of 95% Confidence interval (CFU/ml)
3
NOX
3040000
70000
2902800
3177200
4
Bl21
4165000
133674
3902999
4427001
Table1
