<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Rock_Ridge_Virginia_Experiments skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Rock Ridge Virginia/Experiments</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="mainContainer"><DIV id="menuContainer"><UL><LI><A href="https://2015.igem.org/Team:Rock_Ridge_Virginia">HOME</A></LI><LI><A href="https://2015.igem.org/Team:Rock_Ridge_Virginia/Team">TEAM</A></LI><LI><A href="#">PROJECT
            					</A></LI><LI><A href="https://2015.igem.org/Team:Rock_Ridge_Virginia/Description">Description</A></LI><LI><A href="https://2015.igem.org/Team:Rock_Ridge_Virginia/Experiments">Experiments &amp; Protocols</A></LI><LI><A href="https://2015.igem.org/Team:Rock_Ridge_Virginia/Results">Results</A></LI><LI><A href="https://2015.igem.org/Team:Rock_Ridge_Virginia/Design">Design</A></LI></UL><LI><A href="#">PARTS
            					</A></LI><LI><A href="https://2015.igem.org/Team:Rock_Ridge_Virginia/Parts">Team Parts</A></LI><LI><A href="https://2015.igem.org/Team:Rock_Ridge_Virginia/Basic_Part">Basic Parts</A></LI><LI><A href="https://2015.igem.org/Team:Rock_Ridge_Virginia/Composite_Part">Composite Parts</A></LI><LI><A href="https://2015.igem.org/Team:Rock_Ridge_Virginia/Part_Collection">Part Collection</A></LI><LI><A href="https://2015.igem.org/Team:Rock_Ridge_Virginia/Notebook">NOTEBOOK</A></LI><LI><A href="https://2015.igem.org/Team:Rock_Ridge_Virginia/Attributions">ATTRIBUTIONS</A></LI><LI><A href="https://2015.igem.org/Team:Rock_Ridge_Virginia/Collaborations">COLLABORATIONS</A></LI><LI><A href="https://2015.igem.org/Team:Rock_Ridge_Virginia/Practices">HUMAN PRACTICES</A></LI><LI><A href="https://2015.igem.org/Team:Rock_Ridge_Virginia/Safety">SAFETY</A></LI><LI><A href="https://2015.igem.org/Team:Rock_Ridge_Virginia/Modeling">MODELING</A></LI><LI><A href="https://2015.igem.org/Team:Rock_Ridge_Virginia/Measurement">MEASUREMENT</A></LI><LI><A href="https://2015.igem.org/Team:Rock_Ridge_Virginia/Software">SOFTWARE</A></LI><LI><A href="https://2015.igem.org/Team:Rock_Ridge_Virginia/Entrepreneurship">ENTREPRENEURSHIP</A></LI></DIV><DIV id="contentContainer"><H2>Protocols</H2><DIV class="pure-u-1-1"><P>Here you can find the protocols we used throughout the summer</P><SECTION id="gluconacetobacter"><H2><EM>Gene</EM> Protocols</H2><DIV class="accordion"><H3>Vaccine Platform</H3><DIV><DIV class="pure-g"><DIV class="pure-u-1-2"><OL><LI>This project was based on a past UVA iGEM project on minicells. We tried to use this minicell as a vaccine platform using Wolbachia Surface protein as an immunogenicity factor/adjuvant for other protein that are not very immunogenic. Our goal was to immunize animal vectors and make them immune to the Lyme bacteria. We were going to use 4 common surface proteins from Lyme: OspA, OspC, DbpA, and DbPB. We selected these proteins because based on research they are the most common protein that are expressed in the different animal vectors. Unfortunately due to the lack of resources and the need to focus, we are putting this project in our drawer for future use.</LI></OL></DIV></DIV></DIV><H3><EM>Bacteria mimic/Living vaccine</EM> project</H3><DIV><DIV class="pure-g"><DIV class="pure-u-1-2"><UL><LI>Idea
                                                </LI><LI>After researching for the vaccine platform, we realized that the only component needed for the Lyme bacteria to colonize the Tick's gut was OspA. We started brain storming as how we could use it to combat Lyme disease. We first thought of making a bacteria that can secrete the Tick Receptor of OspA protein (TROSPA), which was shown to be active and able to bind in its soluble and truncated form to the Lyme bacteria's OspA. We quickly realized that our secreted protein may not co-localized with Borrelia burgdorferi (Lyme bacteria) and therefore be ineffective. This is how the idea of making a mimic came about, if we wanted our bacteria to co-localize with the Lyme bacteria then our bacteria would have to resemble Lyme. And the only protein needed to accomplish this is OspA. Through our research we also realized that this technique can potentially work for other insect born diseases. The only problem that we had was how to vertically transfer (parent to offspring) our mimic to the next generation of ticks, so they are also vaccinated against Lyme disease. We knew the key was the sex cells and we found our answer on another bacteria, a symbiot called Wolbachia. Wolbachia Surface Protein (WSP) is responsible for bacteria host interaction, it is the major surface protein in Wolbachia and if co-localizes well with the sex cells of female insects.  


<H2> Igem parts used</H2><P>BACTERIA MIMIC PROJECT ASSEMBLY OF PARTS</P><P>Strong Promoter</P></LI><LI>Part:BBa_J23100
	<P>ttgacggctagctcagtcctaggtacagtgctagc</P><P> RBS that goes with above promoter</P></LI><LI>BBa_B0034 Part-only sequence (12 bp)
     <P>aaagaggagaaa


</P><P>Terminator

</P></LI><LI>BBa_B0014






                                                      
<H2> Genes</H2></LI><LI>We used the Nucleotide Data Base from NCBI to find the OspA and WSP gene sequences.
                                            </LI><LI>Translate from Expasy was used to double, triple check that we have the right protein sequence. WSP is 2151 bp, and OspA is 2475 bp.</LI><LI> the sequences were then sent to Integrated DNA Technologies (IDT) and Genscrip for production.</LI><H3>Plasmid Mini Kit Protocol (modified protocol when hi speed centrifuge is not available</H3><LI> Harvest overnight bacterial culture by centrifuging at 6000 x g for 15 mins at 4 degrees 

celsius 
</LI><LI>Resuspend the bacterial pellet in 0.3 ml of buffer P1

</LI><LI>Add 0.3 ml of buffer P2, mix thoroughly  by vigorously inverting 4-6 times, and incubate 

at room temperature for 5 mins 

</LI><LI>Add 0.3 ml pre chilled buffer P3, mix thoroughly by vigorously inverting 4-6 times, and 

incubate on ice for 5 mins 

</LI><LI>centrifuge at 10,000 rpm for 10 mins. re centrifuge if the supernatant is not clear 

</LI><LI>Equilibrate a QIAGEN-tip 20 by applying 1 ml of Buffer QBT, and allow column to empty 

by gravity flow 

</LI><LI>Apply the supernatant from step 5 to the quiagen-tip and allow it to enter the resin by 

gravity flow 

</LI><LI>wash the QUIAGEN-tip with 2x2 ml buffer QC. allow BUffer QC to move through the 

QUIAGEN-tip by gravity flow 

</LI><LI>Elute DNA with 0.8 ml Buffer QF into a clean 2 ml vessel.

</LI><LI>precipitate DNA by adding 0.56 ml (0.7 volumes (70%)) room-temperature isopropanol 

to the eluted DNA and mix. Centrifuge at 15,000 xg for 30 mins. Carefully decant the 

supernatant </LI><LI> Add 0.7 volumes of isopropanol and keep on ice for 10 mins 

</LI><LI> Transfer solution into a mini-spin column and centrifuge for 1 min. at 10,000 rpm .
</LI><LI> Wash with buffer 500ul AW1 

</LI><LI> Then wash with buffer 500ul AW2
</LI><LI> Spin dry for 1 min at 10,000rpm

</LI><LI> Elute DNA with 100ul ultrapure water

                                
<H3>Digest</H3></LI><LI> Enzyme Master Mix for Plasmid Backbone (25ul total, for 5 rxns)

</LI><LI> Digest Plasmid Backbone

</LI><LI> 5 ul NEB Buffer
</LI><LI> 0.5 ul BSA

</LI><LI>0.5 ul EcoRI-HF

</LI><LI> 0.5 ul PstI

</LI><LI> 18 ul dH20

</LI><LI> Add 19 ul linearized plasmid backbone (25ng/ul for 100ng total)

</LI><LI> Add 4 ul of Enzyme Master Mix

</LI><LI> Digest 37C/30 min, heat kill 80C/20 min



<H3> PCR protocols </H3><H2>GFP PCR</H2></LI><LI>Volume of 25 ul:

</LI><LI>WA2F = 3 ul

</LI><LI>GSV-1R = 3 ul

</LI><LI>GFP DNA = 2 ul

</LI><LI>Distilled, autoclaved water = 17 ul

</LI><LI>Volume of 50ul:

</LI><LI>WA2F = 6 ul

</LI><LI>GSV-1R = 6 ul

</LI><LI>GFP DNA = 4ul 

</LI><LI>Distilled, autoclaved water = 34 ul

<H2> OspA PCR </H2></LI><LI>Volume of 25 ul:

</LI><LI>WA2F = 3 ul

</LI><LI>WA3R = 3 ul

</LI><LI>OspA DNA = 2 ul

</LI><LI>Distilled, autoclaved water = 17 ul

</LI><LI>Volume of 50 ul:

</LI><LI>WA2F = 6 ul

</LI><LI>WA3R = 6 ul

</LI><LI>OspA DNA = 4 ul

</LI><LI>Distilled, autoclaved water = 34 ul

<H2>WSP PCR </H2></LI><LI>Volume of 25 ul:

</LI><LI>VPW1F = 3 ul

</LI><LI>WA3R = 3 ul

</LI><LI>WSP DNA = 2 ul

</LI><LI>Distilled, autoclaved water = 17 ul

</LI><LI>Volume of 50 ul:

</LI><LI>VPW1F = 6ul

</LI><LI>WA3R = 6 ul

</LI><LI>WSP DNA = 4 ul

</LI><LI>Distilled, autoclaved water = 34 ul

                            <H3>Gel Extraction</H3></LI><LI>Cut out gel piece </LI><LI> Weigh the gel slice in a colorless tube. Add three volumes of buffer QG to 1 volume gel (100 mg 

gel – 100 microliters)</LI><LI> NOTE: the maximum amount of gel per spin column is 400 mg</LI><LI>  NOTE: for &gt;2% gels, add 6 volumes of buffer QG
</LI><LI>Incubate at 50 degrees Celsius for 10 mins  </LI><LI> Vortex the tube every 2-3 mins to help dissolve the gel. After the gel slice has dissolved 

completely, check that the color of the mixture is yellow. If the color of the mixture is orange or 

violet add 10 microliters of 3M sodium acetate, pH 5.0, and mix. The mixture should then turn 

yellow </LI><LI> Add 1 volume isopropanol to the sample and mix.</LI><LI>Place QUIAquick spin column in centrifuge in a provided 2 ml collection tub. To bind DNA, apply 

the sample to the QIAquick column and centrifuge for 1 min until all the samples have passed 

the column.  </LI><LI>Discard flow-through and place the QIAquick column back into the same tube. For sample 

volumes &gt; 800 microliters, load and spin/apply vacuum again. </LI><LI>If the DNA will subsequently be used for sequencing, in vitro transcription, or microinjection, 

add 500 microliters Buffer QG to the QIAquick column and centrifuge for 1 min. </LI><LI> Discard flow-through and place the QIAquick column vack into the same tube. (If the DNA will be 

used for salt-sensitive applications, let the column stand 2-5 min after addition of Buffer PE)</LI><LI> For a concentrated solution, add only 30 microliters of Buffer EB (rather than 50 microliters).
</LI><LI> For a concentrated solution, add only 30 microliters of Buffer EB (rather than 50 microliters).</LI><LI> Let the column stand for 2-5 minutes (rather than 1 minute… BECAUSE increasing the time 

increases the yield of purified DNA and is preferred when using it for salt-sensitive applications) </LI><LI> Centrifuge for 1 minute. </LI><H3>Gibson Assembly</H3><LI>  1ul of vector pSB1A3


	</LI><LI>10 ul of gibson assembly master mix 
	</LI><LI>2 ul of ampicillin resistant vector 
	</LI><LI>1.5 ul of GFP (2x the amount of vector)
	</LI><LI>1.5 ul of WSP (2x the amount of vector)
        </LI><LI>5 ul of autoclaved water 
        </LI><LI> 2ul of reaction mixture was used for transformation

<H3>E. coli Calcium Chloride competent cell protocol </H3></LI><LI>Inoculate a single colony into 3mL of LB
</LI><LI>Use 1mL to inoculate 40mL Falcon tube and Shake @ 37°C until the OD 600 is 0.4 
 
</LI><LI>Put the cells on ice for 10 mins (keep cold from now on). 
</LI><LI>Collect the cells by centrifugation in the big centrifugue for 3 mins @6krpm 
</LI><LI>Wash cells with 4mL of ice cold 0.1M CaCl and spin down for 5min at 10,000rpm
</LI><LI>Decant supernatant and gently resuspend on 4 mL of culture cold 0.1M CaCl (cells are susceptible to mechanical disruption, so treat them nicely). 
</LI><LI>Incubate on ice x 20 mins 
</LI><LI>Centrifuge as in 2 
</LI><LI>Discard supernatant and gently resuspend on 2mL cold 
0.1MCaCl/30%Glycerol 
</LI><LI>Dispense in microtubes (150μL/tube). Freeze in -80°C. 
 
<H3>Transformation of Ca++ competent cells</H3></LI><LI>Put 1μL of circular plasmid or all of a ligation reaction of plasmid DNA in a microtube. Gently add ~100μL of competent cells. Do NO DNA control tube with cells and no DNA. 
</LI><LI>Incubate for 30 mins on ice. 
</LI><LI>Heat shock for 2 mins @ 42°C. Put back on ice. 
</LI><LI>Add 900 μL of LB to tubes. Incubate @ 37°C for 30 mins. 
</LI><LI>Plate 100-1000 μL of the cells in LBAmp  (50µg/ml) plates. Plate 100 μL of the NO DNA control in a blood plate (to check for </LI><LI>quality of cells).
</LI><LI>If you need a lot of colonies or the ligation is of low efficiency, centrifuge the transformation for 1 min @ 8krpm, discard 900 μL of supernatant, resuspend o n the 100 μL left and plate the whole lot. 
</LI></UL></DIV></DIV></DIV></DIV></SECTION></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>