"
Page
Discussion
View source
History
teams
Log in
 
Team:WHU-China/templates/standardpage noteProtocal
From 2013.igem.org
Gibson assembly
In order to accomplish an efficient and economical DNA fragments assembly, we adopted an enzymatic assembly method named “Gibson assembly”[1]. Gibson assembly is wildly used because of its’ high efficiency in yielding deserted DNA fragments that may from several to hundreds kilobases. In our project, we adopt the original Gibson assembly protocol. This figure indicates the basic principle of Gibson assembly. In this figure, two DNA fragments which share the same overlapping pieces, are joined together in a one-step isothermal reaction (or precisely called One-step isothermal in vitro recombination). Because the two adjacent DNA fragments share terminal sequence overlap, when T5 exonuclease remove nucleotides from 5’ ends of double-stranded DNA, the single-strand overlap region can pair with each other, just like two sticky ends. After two overlapping single-strand DNA annealed, Phusion DNA polymerase fill the gaps and Taq ligase, seals the nicks. The Protocol of Gibson Assembly: The required key reagents in Gibson assembly: T5 exonuclease (NEB, 10U/ul, 1000U ) Phusion DNA polymerase (Thermo, 100 x 50 μl) Taq DNA ligase (NEB, 10 000U） First, prepare DNA assembly reagent-enzyme mix (which can be stored at -20℃ until needed). The reagent-enzyme is composed of reaction buffer and enzyme which are commercial available. Reaction buffers: 5× Isothermal reaction buffer preparing protocol:
1.5 ml
1 M Tris-HCl pH 7.5 (Prepared by adding HCl into 6.05g Tris-base to adjust pH to 7.5. The total volume is 50ml)
75 ul
2 M MgCl2,        (1.9g/10ml)
30 ul
100 mM dGTP,
30 ul
100 mM dATP,
30 ul
100 mM dTTP,
30 ul
100 mM dCTP,
150 ul
1M DTT,
0.75 g
PEG-8000，
150 ul
100 mM NAD        (0.33g/5ml)
ddH2O
3 ml.
Assembly master mixture:
320 ul 5* isothermal reaction buffer, 0.64 ul of 10 U /ul T5 exonuclease, 20 ul of 2 U/ul Phusion DNA polymerase,
160 ul of 40 U /ul Taq DNA ligase, Add water up to a final volume of 1.2 ml. The volume of mix and DNA is 3:1. For example, 15ul mix + 5ul DNA solution. To start a Gibson assembly, you can just simply mix the reagent-enzyme mixture with DNA fragments solution, and put the PCR tube that with the mix-DNA mixture into PCR instrument, incubate it at 50℃ from 15 to 60 minutes, and then the DNA fragments is recombined together. Using ordinary PCR procedure, we can subsequently amplify the product for further experiment.
Measurement
To fully test our Cas-9 mediated multi-stage modulation system, we adapt a method[2] that can measure a gene’s expression level comparatively preciously. We choose mRFP as our reporter. Check the absolute fluorescent intensity of mRFP by Multi-Functional microplate reader accurately. We define I as the relative fluorescence intensity and F as absolute fluorescence intensity. Because the absolute fluorescent intensity cannot reflect the RFP expressive level,we measured OD600 to define the cell density. Thus, the relative fluorescence intensity(I) can be formulated like this: I =F/OD600 Now, the relative fluorescence intensity can be measured, and it obviously can reflect the relatively strengths of different tandem double promoters.
OverlapPCR
Overlap extension polymerase chain reaction in construction of guide-RNA coding sequence. The overlap extension polymerase chain reaction (or OE-PCR, Overlap PCR), was developed in late 1980s. And until up to now, it has become a classical method. In this project, OE-PCR is used in bio-brick construction: mainly in guide-RNA coding sequence (shortly referred as gRNA) synthesis. Synthesizing a complete, long gRNA is very expensive. So, in order to save budget, we ordered several short fragments can anneal head-to-tail by short overlapping regions at two ends, and thus these fragments can cover the complete gRNA. By deliberately design primers, these DNA primers are fused into a complete, covalently sealed gRNA by OE-PCR. This figure indicates the basic principle of OE-PCR, and is quite simple and plain.  OE-PCR can be performed by ordinary PCR instrument. First, we separate the complete gRNA into 5 pieces, and asked bio-technological company to synthesize these designated primers. Each of them shares an overlapping region with its adjacent partner.
After annealing when replication occurs, the DNA piece is extended by a new sequence that is complementary to the molecule it is to be joined to. Once both DNA molecules are extended in such a manner, they are mixed and a PCR is carried out with only the primers for the far ends. The overlapping complementary sequences introduced will serve as primers and the two sequences will be fused. This method has an advantage over other gene splicing techniques in not requiring restriction sites. Finally, the gRNA construction is finished by 3 cycles of PCR.
Digestion & Ligation
Tandem Promoter
We digest plasmid with Spel&PstI to get a scaffold with promoter I and deal another plasmid with  Xbal&PstI  to get DNA fragment containing promoter II and reporter gene RFP.
And then ligate them to construct tandem double promoters.
Finally we tansfer the disegned gene to suitable vector. Double Fluorescence
Based on double promotor, we exchange reporter gene RFP with another fluorescence gene, mCheer. Because mCheer is a more sensitive for detection.
According to the smilar method, we get the second fluorescence gene with promoter, EYFP, as a control for double fluolrescence detection to elimiante the disturbance.
Then we add EYFP to the down stream of mCheer.
And transfer them to a suitable vector. Guide RNA
We use three cycle of overlap PCR to construct the whole length guide RNA, including prefix, promoter, N20 targeting region, dCas9 binding scaffold, terminator and suffix. dCas9 Device
We amplify dCas9 gene from plasmid through PCR.(We design the homologe terminals of primes preparing for Gibson Assambly).
Restriction enzyme digestion of Bba_J13002 with SpeI to get two sticky ends.
Ligate dCas9 gene with the back backbone by Gibson Assembly.
Restriction enzyme digestion with SpeI and then ligate w gene (got from E.Coli by PCR)through Gibson Assambly.
Ligase gRNA to the downstream of Ѡ to get the functional device. Or we can transfer dCas9 and w to BBa_pSB1C3 first and the add gRNA on to it through the enzyme degistion and ligation.
Retrieved from " http://2013.igem.org/Team:WHU-China/templates/standardpage_noteProtocal "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
