<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_WHU-China_Experiments skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:WHU-China/Experiments</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="header"><DIV id="nav-wrapper"><DIV class="nav"><UL class="nv"><LI id="nv1"><A>Project</A></LI><LI id="nv11"><A href="https://2017.igem.org/Team:WHU-China/Overview">Overview</A></LI><LI id="nv12"><A href="https://2017.igem.org/Team:WHU-China/Background">Background</A></LI><LI id="nv13"><A href="https://2017.igem.org/Team:WHU-China/Introduction">Introduction</A></LI><LI id="nv14"><A href="https://2017.igem.org/Team:WHU-China/Designs">Designs</A></LI><LI id="nv15"><A href="https://2017.igem.org/Team:WHU-China/Experiments">Experiments</A></LI><LI id="nv16"><A href="https://2017.igem.org/Team:WHU-China/Results">Results</A></LI></UL><LI id="nv2"><A>Parts</A></LI><LI id="nv21"><A href="https://2017.igem.org/Team:WHU-China/Description_Part">Description</A></LI><LI id="nv22"><A href="https://2017.igem.org/Team:WHU-China/Basic_Part">Basic</A></LI><LI id="nv23"><A href="https://2017.igem.org/Team:WHU-China/Composite_Part">Composite</A></LI><LI id="nv3"><A>Interlab</A></LI><LI id="nv31"><A href="https://2017.igem.org/Team:WHU-China/InterLab">Results</A></LI><LI id="nv32"><A href="https://2017.igem.org/Team:WHU-China/Calibration">Calibration</A></LI><LI id="nv33"><A href="https://2017.igem.org/Team:WHU-China/Cell_Measuremnet">CellMeasurement</A></LI><LI id="nv4"><A href="https://2017.igem.org/Team:WHU-China/Notebook">Notebook</A></LI><LI id="nv5"><A>Model</A></LI><LI id="nv51"><A href="https://2017.igem.org/Team:WHU-China/Model">Modeling</A></LI><LI id="nv52"><A href="https://2017.igem.org/Team:WHU-China/Hardware">Hardware</A></LI><LI id="nv6"><A>Human Practice</A></LI><LI id="nv61"><A href="https://2017.igem.org/Team:WHU-China/HP/Silver">Silver</A></LI><LI id="nv62"><A href="https://2017.igem.org/Team:WHU-China/HP/Gold_Integrated">Gold&amp;Integrated</A></LI><LI id="nv63"><A href="https://2017.igem.org/Team:WHU-China/Engagement">Engagement</A></LI><LI id="nv7"><A>Safety</A></LI><LI id="nv71"><A href="https://2017.igem.org/Team:WHU-China/Project_Design">ProjectDesign</A></LI><LI id="nv72"><A href="https://2017.igem.org/Team:WHU-China/Lab_Work">LabWork</A></LI><LI id="nv8"><A>Team</A></LI><LI id="nv81"><A href="https://2017.igem.org/Team:WHU-China/Member">Member</A></LI><LI id="nv82"><A href="https://2017.igem.org/Team:WHU-China/Attributions">Attributions</A></LI><LI id="nv83"><A href="https://2017.igem.org/Team:WHU-China/Collaborations">Collaborations</A></LI><LI id="nv9"><A href="https://2017.igem.org/Team:WHU-China/Achievement">Achievement</A></LI></DIV></DIV></DIV><DIV id="topbanner-wrapper" style="height: 299.462px;"><DIV id="topbanner"><UL id="cont"><H6>Experiments</H6><LI><A href="#Materials">1 Materials</A></LI><LI><A href="#Solutions">     1.1 Solutions</A><A href="#Plasmids">     1.2 Plasmids</A></LI><LI><A href="#Bacteria">     1.3	Bacteria</A><A href="#Primers">     1.4	Primers</A></LI><LI><A href="#Methods">2 Methods</A></LI><LI><A href="#Gener">     2.1 General construction of xylose-inducible expression vectors</A></LI><LI><A href="#Trans">     2.2 Transformation of <I>B. megaterium</I> with these plasmids</A></LI><LI><A href="#protein">     2.3 Induction of protein production</A></LI><LI><A href="#RBS">     2.4 Optimization of RBS</A></LI><LI><A href="#Verification">     2.5 Verification of RdhANP-overexpression <I>B. megaterium</I>’s Dehalogenation</A></LI><LI><A href="#hemAXCDBL">     2.6 Integration of hemAKK and xylA Promoter upstream of Operon hemAXCDBL</A></LI><LI><A href="#CBD_MAD">     2.7 Coexpression CBD and MAD</A></LI><LI><A href="#HMBR">     2.8 Introduction of engineered <I>B. megaterium</I> into HMBR</A></LI><LI><A href="#Model">     2.9 Detection and adjustment with Model</A></LI></UL></DIV></DIV><DIV id="contents-wrapper"><DIV id="contents"><DIV><H1 id="Materials">Materials</H1><H3 id="Solutions">1.Solutions</H3><TABLE border="1" align="center"><TBODY><TR bgcolor="#bed6ed"><TD width="20%" style="text-align:center;vertical-align:middle;">Solutions</TD><TD style="text-align:center;vertical-align:middle;">Ingredients</TD><TD width="30%" style="text-align:center;vertical-align:middle;">Sterilization</TD></TR><TR><TD style="text-align:center;vertical-align:middle;">2x AB3(Antibiotic Medium No.3)</TD><TD style="text-align:center;vertical-align:middle;">35g/L AB3(DIFCO) in dH2O</TD><TD style="text-align:center;vertical-align:middle;">Autoclave at 121℃ for 20 min</TD></TR><TR><TD style="text-align:center;vertical-align:middle;">2×SMM	</TD><TD style="text-align:center;vertical-align:middle;">40mM Maleic acid, 80mM NaOH, 40mM MgCl2•6H2O, 1M Sucrose in dH2O</TD><TD style="text-align:center;vertical-align:middle;"> 	Filtration</TD></TR><TR><TD style="text-align:center;vertical-align:middle;">PEG-P</TD><TD style="text-align:center;vertical-align:middle;">400g/L PEG-6000 in 1x SMM</TD><TD style="text-align:center;vertical-align:middle;">Autoclave at 121℃ for 11 min</TD></TR><TR><TD style="text-align:center;vertical-align:middle;">SMMP</TD><TD style="text-align:center;vertical-align:middle;">2x AB3 : 2×SMM= 1:1	</TD><TD style="text-align:center;vertical-align:middle;">Separately sterilized</TD></TR><TR><TD style="text-align:center;vertical-align:middle;">Solution A</TD><TD style="text-align:center;vertical-align:middle;">206g/L Sucrose, 13g/L MOPS, 1.2g/L NaOH in dH2O.Adjust to pH 7.3 with NaOH	</TD><TD style="text-align:center;vertical-align:middle;">Filtration</TD></TR><TR><TD style="text-align:center;vertical-align:middle;">Solution B</TD><TD style="text-align:center;vertical-align:middle;">14.04g/L Agar, 0.70g/L Casamino acids, 35.09g/L Yeast Extract in dH2O</TD><TD style="text-align:center;vertical-align:middle;">Autoclave at 121℃ for 15 min</TD></TR><TR><TD style="text-align:center;vertical-align:middle;">8x CR5 salts</TD><TD style="text-align:center;vertical-align:middle;">2g/L K2SO4, 80g/L MgCl2•6H2O, 0.4g/L KH2PO4, 17.6g/L CaCl2 in dH2O</TD><TD style="text-align:center;vertical-align:middle;">Autoclave at 121℃ for 15 min</TD></TR><TR><TD style="text-align:center;vertical-align:middle;">12% Proline	120g/L</TD><TD style="text-align:center;vertical-align:middle;">L-proline in dH2O</TD><TD style="text-align:center;vertical-align:middle;">Filtration</TD></TR><TR><TD style="text-align:center;vertical-align:middle;">20% Glucose</TD><TD style="text-align:center;vertical-align:middle;">200g/L Glucose in dH2O</TD><TD style="text-align:center;vertical-align:middle;">Filtration</TD></TR><TR><TD style="text-align:center;vertical-align:middle;">CR5-top agar(2.5ml)	</TD><TD style="text-align:center;vertical-align:middle;">1.25ml Solution A, 288μl CR5 salts, 125μl 12% Proline, 125μl 20% Glucose, 713μl Solution B(42℃)</TD><TD style="text-align:center;vertical-align:middle;">Separately sterilized and mix in order</TD></TR><TR><TD style="text-align:center;vertical-align:middle;">Lysis buffer</TD><TD style="text-align:center;vertical-align:middle;">100mM Na3PO4, 5mg/ml Lysozyme. Adjust to pH 6.3 with H3PO4. 1μl 1 M MgSO4 solution and 2 μl HS-Nuclease* (250 U/μl, cat.# GENUC10700-01, final concentration of 500 U/ml)</TD><TD style="text-align:center;vertical-align:middle;">Filtration</TD></TR></TBODY></TABLE><P>(Other basic molecular biological materials are not listed here)</P></DIV><DIV><H3 id="Plasmids">2.Plasmids</H3><TABLE border="1" align="center"><TBODY><TR bgcolor="#a9cf90"><TD width="15%" style="text-align:center;vertical-align:middle;">Plasmids</TD><TD width="35%" style="text-align:center;vertical-align:middle;">Purposes</TD><TD width="50%" style="text-align:center;vertical-align:middle;">Cite/Source</TD></TR><TR><TD style="text-align:center;vertical-align:middle;">pSPAsp-hp</TD><TD style="text-align:center;vertical-align:middle;">Shuttle vector <I>E. coli</I>/<I>B. mega</I>; Protein production</TD><TD style="text-align:center;vertical-align:middle;">pSPAsp-hp was a gift from Dieter Jahn (Addgene plasmid # 48122)</TD></TR><TR><TD style="text-align:center;vertical-align:middle;">pBE-2	</TD><TD style="text-align:center;vertical-align:middle;">Shuttle vector <I>E. coli</I>/<I>B. subtilis<I>; Protein production</I></I></TD><TD style="text-align:center;vertical-align:middle;">pBE-2 was obtained from China Center for Type Culture Collection(CCTCC)</TD></TR><TR><TD style="text-align:center;vertical-align:middle;">pBE-p43</TD><TD style="text-align:center;vertical-align:middle;">Shuttle vector <I>E. coli</I>/<I>B. subtilis</I>; Protein production</TD><TD style="text-align:center;vertical-align:middle;">pBE-p43 was obtained from CCTCC</TD></TR><TR><TD style="text-align:center;vertical-align:middle;">pET28a	</TD><TD style="text-align:center;vertical-align:middle;">Protein production in <I>E. coli</I></TD><TD style="text-align:center;vertical-align:middle;">	pET28a was obtained from Room 6133 of College of Life Sciences, Wuhan university(CLS-WHU)</TD></TR><TR><TD style="text-align:center;vertical-align:middle;">pET26b	</TD><TD style="text-align:center;vertical-align:middle;">Protein production in <I>E. coli</I></TD><TD style="text-align:center;vertical-align:middle;">pET28a was obtained from Room 6135 of CLS-WHU</TD></TR></TBODY></TABLE></DIV><DIV><H3 id="Bacteria">3.Bacteria</H3><TABLE border="1" align="center"><TBODY><TR bgcolor="#fee49d"><TD style="text-align:center;vertical-align:middle;">Strains</TD><TD style="text-align:center;vertical-align:middle;">Purposes</TD><TD style="text-align:center;vertical-align:middle;">Cite/Source</TD></TR><TR><TD style="text-align:center;vertical-align:middle;"><I>Escherichia coli</I> DH5α</TD><TD style="text-align:center;vertical-align:middle;">Plasmid replication, Protein production</TD><TD style="text-align:center;vertical-align:middle;"><I>E. coli</I> DH5α was obtained from Room 6133 of CLS-WHU</TD></TR><TR><TD style="text-align:center;vertical-align:middle;"><I>Bacillus subtilis</I> 168</TD><TD style="text-align:center;vertical-align:middle;">Protein production</TD><TD style="text-align:center;vertical-align:middle;"><I>B. subtilis</I> 168 was obtained from CCTCC</TD></TR><TR><TD style="text-align:center;vertical-align:middle;"><I>Bacillus megaterium</I> DSM B32</TD><TD style="text-align:center;vertical-align:middle;">	Protein production</TD><TD style="text-align:center;vertical-align:middle;"><I>B. megaterium</I> DSM B32 was obtained from CCTCC</TD></TR><TR><TD style="text-align:center;vertical-align:middle;"><I>Nitratireductor pacificus</I> pht-3B</TD><TD style="text-align:center;vertical-align:middle;">Genome PCR</TD><TD style="text-align:center;vertical-align:middle;"><I>Nitratireductor pacificus</I> pht-3B was obtained from CCTCC</TD></TR></TBODY></TABLE></DIV><DIV><H3 id="Primers">4.Primers</H3><TABLE border="1" align="center"><TBODY><TR bgcolor="#bdbbdb"><TD style="text-align:center;vertical-align:middle;">Primer names</TD><TD style="text-align:center;vertical-align:middle;">Sequence</TD></TR><TR><TD style="text-align:center;vertical-align:middle;">Prefix-PartA-F</TD><TD style="vertical-align:middle;">5’- TCTAGAGGCCGCTGCGATCCCGCGAAG-3’</TD></TR><TR><TD style="text-align:center;vertical-align:middle;">PartA-RBS-R</TD><TD style="vertical-align:middle;">5’- CTCCTCTTTAATCTCTAGACATCCTCTATGGTACTCGTGATGGCTTTATTG -3’</TD></TR><TR><TD style="text-align:center;vertical-align:middle;">RBS-Rdh-F</TD><TD style="vertical-align:middle;">5’- TCTAGAGATTAAAGAGGAGAAATACTAGATGAGACTTTACAGCAATAGAGATAGACCGAA-3’</TD></TR><TR><TD style="text-align:center;vertical-align:middle;">Rdh-Ter-R</TD><TD style="vertical-align:middle;">5’- GTTTTATTTGATGCCTGGTTATTATCCCGCTGATGATTTAAGTCTTG-3’</TD></TR><TR><TD style="text-align:center;vertical-align:middle;">Rdh-Ter-F</TD><TD style="vertical-align:middle;">5’- ATCATCAGCGGGATAATAACCAGGCATCAAATAAAACGAAAG-3’</TD></TR><TR><TD style="text-align:center;vertical-align:middle;">Ter-Postfix-R</TD><TD style="vertical-align:middle;">5’- CTGCAGCGGCCGCTACTAGTATATAAACGCAGAAAGGC -3’</TD></TR><TR><TD style="text-align:center;vertical-align:middle;">pSPAsp-hp-BsrG1-F</TD><TD style="vertical-align:middle;">5’- AAGGAGGTGAATGTACAATGAGACTTTACAGCAATAGAGATAGACCGAAT -3’</TD></TR><TR><TD style="text-align:center;vertical-align:middle;">pSPAsp-hp-BamH1-R</TD><TD style="vertical-align:middle;">5’- GCCGGTACCGGATCCTTATTATCCCGCTGATGATTTAA -3’</TD></TR><TR><TD style="text-align:center;vertical-align:middle;">pHIS1525-BsrG1-F</TD><TD style="vertical-align:middle;">5’- AAAGGGGGAAATGTACAATGAGACTTTACAGCAATAGAGATAGACCG-3’</TD></TR><TR><TD style="text-align:center;vertical-align:middle;">pHIS1525-BamH1-R</TD><TD style="vertical-align:middle;">5’- CCGGTACCGGATCCTTATTATCCCGCTGATGATTT-3’</TD></TR><TR><TD style="text-align:center;vertical-align:middle;">pHIS1525-2-F</TD><TD style="vertical-align:middle;">5’- AAGGGGGAAATGTACAATGAGACTTTACAGCAATAGAGATAGACCG-3’</TD></TR><TR><TD style="text-align:center;vertical-align:middle;">pHIS1525-2-R</TD><TD style="vertical-align:middle;">5’- CCGGTACCGGATCCGTCCCGCTGATGATTT-3’</TD></TR><TR><TD style="text-align:center;vertical-align:middle;">pSPAsp1-F</TD><TD style="vertical-align:middle;">5’- AAGGAGGTGAATGTACAATGAGACTTTACAGCAATAGAGATAGACCGAAT-3’</TD></TR><TR><TD style="text-align:center;vertical-align:middle;">pSPAsp1-R</TD><TD style="vertical-align:middle;">5’- GCCGGTACCGGATCCGTCCCGCTGATGATTTAA-3’</TD></TR></TBODY></TABLE><P>(Not all the primers are listed here but some important ones)</P></DIV><DIV><H1 id="Methods">Methods</H1><P>    The following methods cover 1) Construction and use of different xylose-inducible expression vectors, 2) Transformation of <I>B. megaterium</I> with these plasmids, 3) Induction of protein production, 4) Optimization of the Ribosome-Binding Site(RBS), 5) Verification of RdhANP-overexpression <I>B. megaterium</I>’s dehalogenation, 6) Integration of engineered version of hemA and xylA Promoter upstream of Operon hemAXCDBL, 7) Coexpression cellulose binding domain(CBD) and Membrane anchoring domain(MAD), 8) Introduction of engineered <I>B. megaterium</I> into Hybrid Membrane Bioreactor(HMBR), 9) Detection and adjustment with Model.</P><H3 id="Gener">1.General construction of xylose-inducible expression vectors</H3><P>    This year, we aim to construct several different xylose-inducible expression vectors compatible for <I>B. megaterium</I>, including RdhANP-pSPAsp-hp, RdhANP-pHIS1525, RdhANP-pHBintE, hemAKK-pHBintE and CBD-MAD-pSPAsp-hp. For further optimization of our project, more vectors are constructed but not listed here. Some things should be mentioned about plasmid pHIS1525, pSPAsp-hp and pHBintE are that, they are all shuttle vectors for <I>E. coli</I>&amp;<I>B. megaterium</I>.  And pHBintE is capable of chromosome integration due to its temperature sensitivity.  </P><H3 id="Trans">2.Transformation of <I>B. megaterium</I> with these plasmids</H3><P>    We’ve tried a plenty of transformation protocols, including 1 for electroporation, 3 for PEG-P mediated protoplast transformation, 1 for kit of <I>Bacilli</I> transformation and 1 for <I>B. subtilis</I>’s chemical transformation. Each protocol was tried several times with a few adjustments each time. And finally, we worked out with the help of FAFU-iGEM. We propose that our failure in transformation of <I>B. megaterium</I> may results from three reasons, our lack of experience, our lab environment and the protocols themselves. We have gathered and arranged the transformation protocols for expression in <I>B. megaterium</I> for convenience of other teams if they are in need. Here’s the final protocol.</P><P>    2.1 Protoplast preparation</P><P>    a.	A preculture of <I>B. megaterium</I> was grown overnight in 50 mL SMMP on a shaking platform (100 rpm) at 30 °C for a maximum of 16 h. </P><P>    b.	50 mL of SMMP was inoculated with 1 mL preculture, incubated at 37 °C, and shaken at 200 rpm until an OD578 of 1.0 was reached (~2 h). </P><P>    c.	Cells were harvested by centrifugation at 2,683 × g and 4 °C for 15 min. The supernatant was removed, and cells were resuspended in 5 mL SMMP. </P><P>    d.	Lysozyme (100 μL 5 mg/mL dissolved in SMMP and ﬁlter sterilized) was added and incubated at 37 °C with shaking at 100 rpm for 10–20 min until at least 50% of cells were protoplasts (as estimated by use of a light microscope). </P><P>    e.	Protoplasts were incubated on ice for 30 min and then harvested at 1,400 × g and 4 °C for 10 min. The supernatant was removed, and cells were resuspended in 5 mL SMMP. </P><P>    f.	Step 5 was repeated three times to remove excess lysozyme. </P><P>    g.	After the ﬁnal resuspension in 5 mL SMMP, 652.5μL 100% glycerol were added and mixed. Aliquots (500 μL) were prepared and used fresh or stored at −80 °C (viable for 2 month).</P><P>    2.2 Transformation protocol</P><P>    a.	Combine 500 μl of protoplast suspension and 3-5 μg of plasmid DNA in a 15 ml tube, one for each transformation. DNA should be purified using a commercial preparation kit. Elute the DNA from the column using water.</P><P>    b.	Add 1.5 ml of PEG-P, mix gently, and incubate for 2 minutes at RT.</P><P>    c.	Add 5 ml SMMP and mix carefully by rolling the tube. </P><P>    d.	Harvest cells by gentle centrifugation (1,300 x g for 10 minutes at RT), discard the supernatant immediately after centrifugation. Supernatant does not have to be removed completely. </P><P>    (Note: do not check for a cell pellet - most of the time it will be invisible) </P><P>    e.	Add 500 μl of SMMP to remaining supernatant (containing bacterial cells) and transfer to a 1.5 ml microcentrifuge tube. </P><P>    f.	Incubate at 30 °C for 90 minutes with gentle shaking or rolling of tubes (max. 100 rpm) or incubate for 45 min without shaking followed by another 45 min while shaking at 300 rpm. </P><P>    g.	Prepare 2.5 ml aliquots of CR5-top agar in sterile tubes. </P><P>    h.	After incubation at 30 °C add all cells to 2.5 ml top agar, mix gently by rolling the tube between both hands (do not vortex!), and pour onto a pre-warmed plate of LB containing desired antibiotic. </P><P>    i.	Incubate overnight at 30 °C - expect colonies of varying diameter because some will be covered with agar and others have easier access to air. 
(Note: bacterial colonies on the top of the agar surface will be shiny) </P><P>    j.	Streak several single colonies (at least 3) on fresh plates within two days.</P><P>    2.3 Controls</P><P>    a. Negative control: protoplasts without DNA</P><P>    This is a test reassuring that the protoplasts are not only fully viable but also free of contaminations before using them for transformation. Perform this test according to the transformation protocol as demonstrated below. After incubation at 30 °C, apply CR5-top agar to the protoplasts and split the sample into two portions. You may plate one sample on an LB plate with antibiotic such as tetracycline or chloramphenicol, and another one on a plate without any drug. In this case, bacterial colonies will grow only on a solid medium without antibiotics.</P><P>    b.Positive control: protoplasts transformed with an empty plasmid
</P><P>    This is a test control for a successful transformation and should yield lots of colonies on the plates supplemented with an antibiotic (here: tetracycline or chloramphenicol). If this transformation works well, but you have problems with the plasmid containing your target gene, the problem is most likely associated with your construct.</P><H3 id="protein">3.Induction of protein production</H3><P>    3.1	Test protein production</P><P>    a.	Prepare an overnight culture inoculated with <I>B. megaterium</I> single colonies from a plate (medium including antibiotic such as tetracycline or chloramphenicol) grown 14 h at 37 °C while agitation at 100 rpm. </P><P>    b.	Inoculate fresh medium with overnight culture in a dilution of 1:100. </P><P>    c.	Grow the recombinant <I>B. megaterium</I> cells in baffled flasks to an optical density (OD578nm) of 0.3 - 0.4 at 37 °C under vigorous shaking (250 rpm). </P><P>    d.	Take a sample as control before induction. </P><P>    e.	Induce the xylose inducible promoter by addition of 0.5% (w/v) of (D)-xylose. </P><P>    f.	Incubate at 37 °C with vigorous shaking at 250 rpm. </P><P>    g.	Withdraw samples every 30 to 60 minutes for OD578nm-measurement and protein analysis (up to 6 hours after induction). For extracellular protein analysis take 2 ml of cell culture. Intracellular protein analysis requires a higher volume than 2 ml of sample. </P><P>    h.	Centrifuge each sample to harvest cells and cell free supernatant. </P><P>    i.	For extracellular protein analysis store the cell free supernatant at 4 °C, and for intracellular protein analysis completely remove supernatant and freeze the cell pellet at -20 °C.</P><P>    3.2	Analysis of intracellular proteins</P><P>    a.	Resuspend cells in 30 μl of lysis buffer. </P><P>    b.	Incubate for 30 min at 37 °C while shaking at 1,000 rpm in a thermomixer. An effective cell lysis can be obtained by whirling the samples every 10 minutes. </P><P>    c.	To separate insoluble fraction (pellet) from the soluble fraction (supernatant), centrifuge cell lysate for 30 min at 4 °C and 13,000 rpm. </P><P>    d.	Mix 27 µl of supernatant containing soluble proteins with 13 µl of SDS sample buffer. </P><P>    e.	Completely remove the supernatant and resuspend the pellet in 30 µl of 8 M urea (w/v). Centrifuge for 30 min at 4 °C and 13,000 rpm. </P><P>    f.	Mix 27 µl of the supernatant with 13 µl of SDS sample buffer. </P><P>    g.	Heat each sample for 5 min at 95 °C. </P><P>    h.	Load 7.5 µl of each sample onto an SDS-PAGE gel.</P><H3 id="RBS">4.Optimization of RBS</H3><P>    As we’ve got to know in papers about <I>B. megaterium</I>, the RBS sequence plays a cardinally significant role in protein expression. It’s mostly preferred to have 7 to 10 bp of RBS between the mRNA and the 3’ end of the 16S rRNA. Cytosines or guanines shouldn’t be included in the spacer region between RBS and the start codon, which also has a preferred length of 7 to 9 bp. Since <I>N. pacificus</I> and <I>B. megaterium</I> belong to different phyla, we propose that they have low 16S rRNA gene sequence similarity and optimization of RBS is necessary for the heterogeneous expression of RdhANP in <I>B. megaterium</I>. But unfortunately we haven’t found the RBS for <I>rdhANP</I> in <I>N. pacificus</I>. In this. Therefore, we decided to recombine the promoters and RBSs in our expression vectors and adjust the length of spacer between the RBS and start codon of <I>rdhANP</I>. In this way, we have constructed 5 Biobricks listed below concerning to this subject for the sake of other potential users in subsequent iGEM competitions. </P><TABLE border="1"><TBODY><TR bgcolor="#c7d9f0"><TD style="text-align:center;vertical-align:middle;">Parts</TD><TD style="text-align:center;vertical-align:middle;">Descriptions</TD><TD style="text-align:center;vertical-align:middle;" width="47%">Links</TD></TR><TR><TD style="text-align:center;vertical-align:middle;">BBa_K2462003</TD><TD style="text-align:center;vertical-align:middle;">Promoters for <I>Bacillus megaterium</I></TD><TD style="vertical-align:middle;"><A href="http://parts.igem.org/Part:BBa_K2462003">http://parts.igem.org/Part:BBa_K2462003</A></TD></TR><TR><TD style="text-align:center;vertical-align:middle;">BBa_K2462004</TD><TD style="text-align:center;vertical-align:middle;">Promoters for <I>Bacillus megaterium</I></TD><TD style="vertical-align:middle;"><A href="http://parts.igem.org/Part:BBa_K2462004">http://parts.igem.org/Part:BBa_K2462004</A></TD></TR><TR><TD style="text-align:center;vertical-align:middle;">BBa_K2462001</TD><TD style="text-align:center;vertical-align:middle;">RBS for <I>Bacillus megaterium</I></TD><TD style="vertical-align:middle;"><A href="http://parts.igem.org/Part:BBa_K2462001">http://parts.igem.org/Part:BBa_K2462001</A></TD></TR><TR><TD style="text-align:center;vertical-align:middle;">BBa_K2462002</TD><TD style="text-align:center;vertical-align:middle;">RBS for <I>Bacillus megaterium</I></TD><TD style="vertical-align:middle;"><A href="http://parts.igem.org/Part:BBa_K2462002">http://parts.igem.org/Part:BBa_K2462002</A></TD></TR><TR><TD style="text-align:center;vertical-align:middle;">BBa_K2462006</TD><TD style="text-align:center;vertical-align:middle;">Promoter+RBS</TD><TD style="vertical-align:middle;"><A href="http://parts.igem.org/Part:BBa_K2462006">http://parts.igem.org/Part:BBa_K2462006</A></TD></TR></TBODY></TABLE><P>    If time and conditions permits, we plan to design synthetic RBS by 'Ribosome binding site calculator' and RBSDesigner and measure their effectiveness with eGFP and RdhANP. We also recommend other teams to do this job if they have similar intention with us.
	</P><H3 id="Verification">5.Verification of RdhANP-overexpression <I>B. megaterium</I>’s Dehalogenation</H3><P>    After succeeding in transformation and inducing the protein production, we are faced with one of the most important experiments in our project. Since our intention is to overexpress RdhANP, this novel reductive dehalogenase in <I>B. megaterium</I>, and although it has been reported in the paper <I>Reductive dehalogenase structure suggests a mechanism for B12-dependent dehalogenation</I> published in <I>Nature</I> in 2015, we still have to verify this enzyme’s dehalogenation in our constructed proof-to-concept experiment for the sake of assurance. 
</P><P>    In our proof-to-concept experiment, we designed three different groups as follows:</P><TABLE border="1"><TBODY><TR><TD style="text-align:center;vertical-align:middle;"><I>B. megaterium</I></TD><TD style="text-align:center;vertical-align:middle;" bgcolor="#fffd38">Transformed with <I>rdhANP</I></TD><TD style="text-align:center;vertical-align:middle;">Untransformed with <I>rdhANP</I></TD></TR><TR bgcolor="#bed6ed"><TD style="text-align:center;vertical-align:middle;">Induced by xylose</TD><TD style="text-align:center;vertical-align:middle;" bgcolor="#94ce58">Group 1</TD><TD style="text-align:center;vertical-align:middle;">Group 2</TD></TR><TR><TD style="text-align:center;vertical-align:middle;">Uninduced by xylose</TD><TD style="text-align:center;vertical-align:middle;" bgcolor="#fffd38">Group 3</TD><TD style="text-align:center;vertical-align:middle;">/</TD></TR></TBODY></TABLE><P>    By comparing the result of Group 1 and Group 2 both colored in light blue, we can see whether expression of RdhANP is the reason for dehalogenation. And by comparing the result of Group 1 and Group 3 both colored in yellow, we can measure the influence of xylose operon in the expression of RdhANP. Note that Group 2 is in both comparison, so it’s colored green.</P><P>    We mainly follow the formerly mentioned protocol of protein induction in Subtitle 2.3.1 and 2.3.2. And we used several different halogenated organics to test RdhANP’s ability of dehalogenation by HPLC done by professional technicians in Testing Center of Wuhan University. </P><H3 id="hemAXCDBL">6.Integration of hemAKK and xylA Promoter upstream of Operon hemAXCDBL</H3><P>    In Introduction-B12, we`ve introduced the synthetic pathway of B12 along with heme and siroheme from glutamyl-tRNA to uroporphyrinogenⅢ. And regular biosynthetic enzyme engineering involves in introducing <I>hemAKK</I> instead of <I>hemA</I> and the xylose-inducible promoter PxylA into <I>B. megaterium</I>’s chromosome. This work can be achieved by formerly mentioned plasmid pHBintE.
</P><H3 id="CBD_MAD">7.Coexpression CBD and MAD</H3><P>    In our hardware, we also designed a special membrane which is made of cellulose and has engineered <I>B. megaterium</I> attached to. Because when we took the cost of artificially synthesized membranes into account, we realized that there should be some substitution done here to make our hardware more available. In this way, we referred to one of the most well-known membrane in biology, cellulose membrane and used the surface display system constructed in <A href="https://2016.igem.org/Team:Kyoto/Description#surface">2016 Kyoto</A> for reference. And we shall repeat their experiment in the following process.</P><H3 id="HMBR">8.Introduction of engineered <I>B. megaterium</I> into HMBR</H3><P>    After finishing biosynthetic enzyme engineering in <I>B. megaterium</I>, we would come to the next stage of introduction of this engineered strain into HMBR. Several problems are encountered, for example, the livability of <I>B. megaterium</I> in organohalide-contained wastewater, cohabitation of <I>B. megaterium</I> and the other bacteria, biofilm formation on suspended carriers. We have consulted Prof. Mao Xuhui of SRES-WHU about these problems. He showed great interest in our project and promised that if we finished our engineering work, he would help us in applying our engineered strain into HMBR. </P><H3 id="Model">9.Detection and adjustment with <A href="https://2017.igem.org/Team:WHU-China/Model">Model</A></H3><P>    In this experiment, we used 3,5-dichloro-2-hydroxybenzoic acid (symbol as A) and 3,5-dibromo-2-hydroxybenzoicacid (symbol as B) to represent the organohalides in the wastewater for their relative higher toxicity and potential harm to <I>B. megaterium</I>. And we hope to test this strain’s growth conditions under the stress of this two pollutants and  to the optimal our project’s practicality.</P><P>     The experiments can be divided into four steps as listed below. Detailed results are written in Modeling.</P><P>    a.	Both A and B were dissolved in ethanol. And LB medium was then prepared with different concentrations of halides. The whole volume of each medium is constant. </P><P>    b.	On Oct.25, we did the experiment using the concentrations got from an article dealing with the effects these halides have on epithelial cells (We chose that because up to then, no more related articles had been found by us.), but the results weren’t very good. The concentrations we tested were too low to perform adverse effects on bacteria.</P><P>    c.	Preliminary tests were done and we tried to narrow down the range of concentrations by ourselves. Fortunately, this step went well. </P><P>    d.	We changed the gradients of concentration and got a better result.
</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>