<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Uppsala_University_Project"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Uppsala_University/Project">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Uppsala_University/Project&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Uppsala_University/Project&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Uppsala_University/Project&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Uppsala_University/Project" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Uppsala University/Project</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><P><TITLE>Team Uppsala University – iGEM 2012</TITLE></P><DIV id="links"><DIV id="logos"><A href="http://www.facebook.com/UppsalaIgem"><FONT>Facebook</FONT></A><A href="http://twitter.com/igemuppsala"><FONT>Twitter</FONT></A><A href="http://www.unt.se/blogg/blogg.aspx?blogg=1784672"><FONT>Blog (in Swedish)</FONT></A><A href="/Team:Uppsala_University/Team#contact"><FONT>Email</FONT></A></DIV><DIV id="menucontainer"><UL class="menu"><LI><A href="/Team:Uppsala_University">Home</A></LI><LI><A href="/Team:Uppsala_University/Project">Project</A></LI><LI><A href="/Team:Uppsala_University/Project#Description">Description</A></LI><LI><A href="/Team:Uppsala_University/Datapage">Data Page</A></LI><LI><A href="/Team:Uppsala_University/Notebook">Notebook</A></LI><LI><A href="/Team:Uppsala_University/Safety">Safety</A></LI><LI><A href="/Team:Uppsala_University/Humanpractice">Human Practice</A></LI><LI><A href="/Team:Uppsala_University/Collaborations">Collaborations</A></LI><LI><A href="/Team:Uppsala_University/Attributions">Attributions</A></LI></UL><LI><A href="/Team:Uppsala_University/Results">Results</A></LI><LI id="widerSubmenu"><A href="/Team:Uppsala_University/Translational">Silencing sRNA:s</A></LI><LI id="widerSubmenu"><A href="/Team:Uppsala_University/Backbones">New backbones</A></LI><LI id="widerSubmenu"><A href="/Team:Uppsala_University/Promoters">Promoter tests</A></LI><LI id="widerSubmenu"><A href="/Team:Uppsala_University/Chromoproteins">Chromoproteins</A></LI><LI id="widerSubmenu"><A href="/Team:Uppsala_University/Catsac">Gene deletion</A></LI><LI><A href="/Team:Uppsala_University/Parts">Parts</A></LI><LI><A href="/Team:Uppsala_University/Modelling">Modelling</A></LI><LI><A href="/Team:Uppsala_University/Modelling#sRNA">sRNA interactions</A></LI><LI><A href="/Team:Uppsala_University/Team">Team</A></LI><LI><A href="/Team:Uppsala_University/Team#aboutus">About Us</A></LI><LI><A href="/Team:Uppsala_University/Team#supervisors">Supervisors</A></LI><LI><A href="/Team:Uppsala_University/Team#instructors">Instructors</A></LI><LI><A href="/Team:Uppsala_University/Team#students">Students</A></LI><LI><A href="/Team:Uppsala_University/Team#advisors">Advisors</A></LI><LI><A href="/Team:Uppsala_University/Team#donors">Donors</A></LI><LI><A href="/Team:Uppsala_University/Team#contact">Contact us</A></LI></DIV><DIV style="position: absolute; top: 3px; right: 5px;"><A href="https://twitter.com/share" class="twitter-share-button" data-count="none">Tweet</A></DIV></DIV></DIV><DIV id="header"><DIV id="headertext">Project</DIV><DIV id="headlines"><A href="#overview">Overview</A> | <A href="#sRNA">Construction of artificial small RNA</A> | <A href="#indevelopement">In developement</A> | <A href="#future">Future developement</A> | <A href="#newsynbio">New SynBio tool</A></DIV></DIV><TABLE id="ttable" align="center"><TBODY><TR><TD class="subtext"><H2>Overview</H2></TD><TD valign="bottom"><A id="top" href="#top">Back to top</A></TD></TR></TBODY></TABLE><P>Antibiotic resistance is a major global health problem. During the last couple of decades, antibiotic resistance has become increasingly problematic, and there is no doubt that new approaches to solve this problem at a technical level are needed to alleviate the problem.

The data below is taken from the european center for disease prevention and control (ECDC) and shows the proportion of resistant isolates of <I>Klebsiella pneumoniae</I> in the member states of the European Union from 2000 to 2010.</P><TABLE id="ptable"><P><B>Spread of <I>Klebsiella</I> resistant to aminoglycosides</B></P></TABLE><P>Misuse and overuse of antibiotics are contributing factors to the severity of the problem, and it seems that we need new approaches to solve it. One approach to the problem is to keep developing new kinds of antibiotics, but unfortunately there is no certainty that we can keep up with the resistance development. We are fighting a never ending battle against bacterial evolution that we at the moment seem to be losing. An outside-the-box solution is to approach problem from the other way. Why don’t we try to make the bacteria sensitive to already developed antibiotics once again? This is our approach. We accomplish it by engineering silencing artificial small RNAs against the resistance genes.</P><P><B>Targeting resistance with sRNAs</B>
The graphs below shows that it is possible to engineer small RNAs to lower the resistance of <I>Escherichia coli</I> against the antibiotic kanamycin. The three different sRNAs UU17, UU37 and UU55 all had significant effects compared to expression of a wild-type regulating sRNA. The left graph shows the kanamycin resistance of <I>E coli</I> carrying the AAC(6') gene on an F plasmid, while the right graph shows the kanamycin resistance of <I>E coli</I> carrying the clinical resistance plasmid pUUH239.2.</P><P><B>Gene networks</B>
Targeting resistance with sRNAs is one approach to an alleviation of the problem with antibiotic resistance, but there are several more. It has been shown that site directed mutagenesis on certain proteins which natively repress antibiotic resistance genes can create superrepressor proteins. Another possible way to lower resistance to antibiotics is to overexpress proteins that repress DNA-repair, such as the LexA protein. It might be possible to use these proteins to repress antibiotic resistance, and this is something we are actively investigating.</P><P><B>Transcription-like effector nucleases</B>
There might be even more creative solutions to the problem of antibiotic resistance. Our team is investigating methods to radically increase plasmid loss rate by introducing plasmid-cutting enzymes in the bacteria. It may be possible to engineer modular proteins targeting very specific sequences of DNA using TAL Effector Nucleases, proteins originating from a plant pathogen DNA binder fused with a FokI DNA cleavage domain.</P><P><B>Improving the Registry of Standard Biological Parts</B>
We want to contribute by improving the common toolbox of the SynBio community by adding new characterized parts. We have constructed and characterized efficient low copy BioBrick standard vectors and added new chromoproteins. We have also developed a modular screening system and protocol for finding silencing sRNA:s against arbitrary genes. The intention is that these parts will be used in systems by other teams having the same vision as us; using synthetic biology as means to solve the daunting problems humanity are facing, and will be facing in the future.</P><P><B>Conclusions</B>
In conclusion, we have constructed several small synthetic antisense RNAs and shown that we can down-regulate resistance to kanamycin with more than 90 %. We are currently working with developing new small RNAs against other resistance genes and also combining small RNAs with targeting of gene networks described above to get a potential synergy effect when combining the different approaches. </P><TABLE><TBODY><TR><TD class="subtext"><H2>Construction of artificial small RNAs</H2></TD><TD valign="bottom"><A id="top" href="#top">Back to top</A></TD></TR></TBODY></TABLE><P><B>Challenge</B>
The primary goal was to construct synthetic small RNAs to silence antibiotic resistance genes. This could potentially be a way to make resistant bacteria sensitive against existing antibiotics, but the problem is to design or find the ideal sRNA sequences.
</P><P><B>Background</B>
Spot42 is one of many non-coding RNAs in <I>Escherichia coli</I>. It is a small regulatory RNA that acts as one of the factors involved in the central and secondary metabolism where it regulates the switch between fermentation and respiration [2]. Spot42 contains two distinctive regions, where one is the antisense region, the part of the sRNA that interacts with other RNAs, and one is the region that recruits an RNA binding protein called Hfq.
</P><P><B>Goal</B>

Our goal was to engineer the native sRNA spot42 to instead target the kanamycin resistance gene AAC(6’), isolated from an ESBL plasmid from an outbreak of multiresistent bacteria in a hospital in Sweden.
In theory, the sRNA with its modified antisense region would Watson-Crick base pair with the complementary mRNA sequence at the 5’UTR of the antibiotic resistance gene AAC(6´), blocking ribosomal binding. This would supposedly lead to an inhibition of the translation and henceforth a silencing of the antibiotic resistance gene AAC (6´). </P><P>
Earlier studies of how to design an artificial sRNA showed that there were unknown factors determining whether an sRNA would be effective in blocking translation or not. To find the optimal sRNA, a large randomised library of sRNAs was made to find sequences efficient enough to down regulate the antibiotic resistance with a combinatorial approach[1].
</P><P><B>Constructing a randomized library of small RNAs</B>
The native Spot42 gene spf from <I>E coli</I> was cloned in a BioBrick plasmid and placed in front of a synthetic constitutive promoter (J23101). Using this plasmid as template, primers binding to the Hfq binding region and the promoter with overhangs containing a randomized nucleotide sequence of 30 bp was designed (15 randomized nucleotides per primer). By running an inverse PCR on the plasmid with these primers and religating the mutagenized plasmids, a randomized library was created with a maximal theoretical size of 4^30 unique sRNA, only limited by the volume of the PCR reaction.
</P><P>
In order to screen for small RNAs with an antisense region hybridizing in an silencing manner to the 5’ UTR of the antibiotic resistance gene AAC (6´), the randomized sRNAs were transformed into an <I>E coli</I> MG1655 strain carrying a reporter system containing the native 5´UTR of AAC(6’) followed by an additional 15 codons of the coding sequence of AAC(6’) translationally fused via a linker (J18922) to the yellow fluorescent protein SYFP2 (K864100).
</P><P>
Since the reporter system and the sRNA library were on two different plasmids, two suitable plasmid backbones were chosen from different compatibility groups. To make the reporter system as similar to the expression levels of the natural resistance genes as possible, a low copy origin such as pSC101 (<A href="”http://partsregistry.org/Part:BBa_K864001”">BBa_K864001</A>) was required while the sRNA library used a medium copy backbone such as p15A. Unfortunately, the low copy backbones of the pSB4X5 serie in the registry did not display the predicted behavior of a low copy plasmid, henceforth the decision to construct new plasmid backbones that could meet our requirements was made <A href="/Team:Uppsala_University/Backbones">Read more</A></P><P>

A randomized antisense region of thirty bases resulted in an immense library of sRNAs, where the limiting factor was the transformation efficiency of or competent cells. To be able to find promising silencing sRNAs in this vast library, a Fluorescence Activated Cell Sorter (FACSAria II from BD) was used to sort out 20 000 cells that showed a downregulation of SYFP2 from a total of 10^7 cells. </P><P>
By using the cell sorter function on the FACS machine the amount of false postives were dramatically reduced. This is because the FACS it makes it possible to differentiate between cells with a very low fluorescence and cells that have no fluorescence at all, non-fluorescent cells were expected to might have picked up a loss of function mutations in the SYFP gene.
</P><P>
The cells sorted based on lowered fluorescence were plated on selective agar plates and studied under UV light in order to screen for colonies containing a small RNA that had downregulated the SYFP2 gene. This was problematic due to radiative DNA-damage, but this problem was solved by switching to a Visi-Blue transilluminator, avoiding damage to the cells and also simplified future screening.
</P><P>
Clones that showed lowered expression of SYFP2 were measured for fluorescence using flow cytometry. The fluorescence levels could be distinguished with a accuracy of just a few percent. 
</P><P>
The plasmids that contained sRNA down regulating SYFP2 were purified and transformed into DH5alpha, and then purified again to ensure pure plasmid clones free from reporter plasmids. 
</P><P>
Finally, the isolated sRNA plasmids were transformed into a new reporter strain to validate the downregulation with flow cytometry
</P><P>
The reporter system together with the native spot42 has been sent as parts to the registry. This is to give future iGEM teams the possibility to repress any gene by replacing the RFP region with the 5´UTR of the gene of interest.
</P><TABLE><TBODY><TR><TD class="subtext"><H2>In developement: gene networks</H2></TD><TD valign="bottom"><A id="top" href="#top">Back to top</A></TD></TR></TBODY></TABLE><P><B>LexA and MarR</B>
Multiple antibiotic resistance repressor (MarR) is a native protein in <I>E coli</I> that represses the multiple antibiotic resistance operon (MarO). Super-repressor mutants generated in order to study regions of MarR required for function showed altered inducer recognition properties and increased DNA binding to MarO in vitro [1]. By isolating MarR from MG1655 chromosome by PCR followed by site directed mutagenesis we acquired a marR(G95S) mutant which represses MarO 30 fold more efficiently compared to wildtype [1]. Overexpressing the MarR (G95S) mutant in a resistant strain might hence increase the sensitivity to antibiotics. 
</P><P>
In addition to MarR, we could also use LexA to target the SOS response system required for repairing DNA damage. By isolating LexA from the MG1655 chromosome using PCR and subsequent mutagenesis we acquired LexA3 [2]. RecA is also involved in this system by activation with single stranded DNA which interacts with the LexA repressor to facilitate autoprotolysis (self-cleavage) of the repressor from the operator which then leads to SOS response. The LexA3 [2] mutation protects from cleavage by RecA, which in turn increases the repression of the DNA repair system. This leads to weaker SOS response to antibiotics and increased DNA damage which will increase sensitivity to antibiotics and cell death.
</P><P>
These constructs are in separate vectors (<A href="”http://partsregistry.org/Part:pSB1C3”">pSB1C3</A>) and will be tested separately by transformation of LexA3 and MarR(G95S) into a ciprofloxacin resistant strain (DA25189). Potential reduction of resistance can be measured with Etest (BioMerieux).
</P><P><B>TetR repressor</B>
The TetA gene which confers resistance to tetracycline is regulated by the conformation changes of the repressor TetR in absence or presence of tetracylince. In presence of the antibiotic the TetR protein changes its conformation and will no longer repress the TetA gene. Mutations of the binding site on TetR and mutants that not bind to tetracycline have been characterized, which leads to mutants which has a constant repression of TetA [3].
</P><P>
Through three separate site directed mutagenesis PCR reactions of the Tet inverter (BBa_Q04400) we intend to create mutant versions of the TetR unable to interact with tetracycline, and as a result less likely to release from its binding site, lowering the expression of TetA. 
</P><P>
These different constructs will be in inserted into chlorophenicol backbone and later transformed into a strain carrying an F-plasmid containing the TetA gene to investigate potential reduction of resistance.
</P><P>
[1] Alekshun, M.N., Levy, S.B., 1999 Characterization of MarR Superrepressor Mutants. J Bacteriol 181, 3303–3306.
</P><P>
[2] Lin, L.L., Little, J.W., 1988 Isolation and characterization of noncleavable (Ind-) mutants of the LexA repressor of Escherichia coli K-12. J. Bacteriol. 170, 2163–2173.</P><P>
[3] Müller, G., Hecht, B., Helbl, V., Hinrichs, W., Saenger, W., Hillen, W., 1995. Characterization of non-inducible Tet repressor mutants suggests conformational changes necessary for induction. Nat. Struct. Biol. 2, 693–703.
</P><TABLE><TBODY><TR><TD class="subtext"><H2>Future developement</H2></TD><TD valign="bottom"><A id="top" href="#top">Back to top</A></TD></TR></TBODY></TABLE><P><B>Delivery of constructs</B>
 
Synthetic small RNAs and repressors of resistance genes need to be expressed inside the bacterial cell to be efficient, so to be used as a drug we would need a method to deliver them to as many bacterial cells as possible. Two different approaches were investigated: delivery thought M13 phage or through a conjugative plasmid.</P><P>
 
The M13 phage is an well characterized bacteriophage that specifically targets <I>E coli</I> and is often used in recombinant DNA technologies. It inserts a single stranded circular DNA and hijacka the bacterial systems to replicate their own DNA to make more phages. It has been shown that it is possible to deliver synthetic DNA into a resistant <I>E coli</I> strain using a phage, making the host cell less antibiotic resistant. [1]</P><P>
 
The phage approach was demonstrated by Timothy et al 2009, where they inserted  proteins LexA and OmpF in the phage M13 that we have also have worked with (see gene networks for explanation of the genes LexA and OmpF).The infected cells showed a decrease in antibiotic resistance. In the article they also tested to inject engineered phages in mice infected with antibiotic resistant bacteria and eight out of ten survived instead of one out of ten when not infected with phages. </P><P>

Conjugation is a process in bacteria that enables bacteria to share genetic information by spreading plasmids to each other. In <I>E coli</I>, conjugation is initiated when the donor bacteria develop pili that bind to the surface of the receiver bacteria. Through this pilus conjugative plasmids can be transferred to the receiver cell. It might be possible to use this functionality of conjugative plasmids to deliver our synthetic genes to resistant bacteria.</P><P>

We have not only manage to clone the ompF and LexA genes, but also constructed a super-repressing mutant of MarR (see Gene networks), all proteins targeting antibiotic resistance in different ways. It is very possible that we could see a synergistic effect on the decrease of resistance if you would combine these protein based approaches with our synthetic small RNAs in a phage or conjugative plasmid.
 </P><P>
[1]Timothy K. Lua,b and James J. Collinsb, 2009, Engineered bacteriophage targeting gene,PNAS N0.12 2009, p 4629 – 4634 

</P><TABLE><TBODY><TR><TD class="subtext"><H2>New SynBio tools</H2></TD><TD valign="bottom"><A id="top" href="#top">Back to top</A></TD></TR></TBODY></TABLE><P><B>Chromoproteins</B>

Chromoproteins are orginally derived from corals or sea anemones. The expressed pigment can be seen by the naked eye and without any external tool, making them very useful as reporter genes. During iGEM 2011, Team <A href="https://2011.igem.org/Team:Uppsala-Sweden/Parts">Uppsala-Sweden</A> contributed with chromoproteins to the registry. This year, we have contributed with an additional gene, aeBlue, and characterized two chromoproteins from last year.
<UL><LI>aeBlue, <A href="http://partsregistry.org/wiki/index.php/Part:BBa_K864401">BBa_K864401</A></LI><LI>eforRed, <A href="http://partsregistry.org/wiki/index.php/Part:BBa_K592012">BBa_K592012</A></LI><LI>cjBlue, <A href="http://partsregistry.org/wiki/index.php/Part:BBa_K592011">BBa_K592011</A></LI></UL></P><P><B>Characteriation of promoters:</B>
Promoters such as Plac, PlaciQ, J23-series, PLlacO, T5lac etc. have been characterized in a promoter test to see promoter strengths in different situations (different strains, lacI repressed, IPTG induced etc).</P><P><B>Scarless deletion - negative/positive selection:</B>
Using λ red recombineering, the cat-sacB genes can be placed on the chromosome deleting the sequence between the homologies added by PCR on the ends of the cat-sacB cassette. In a first step, successful clones are selected on chloramphenicol, then a second λ red is performed with a single stranded oligo joining the sequences of the homologies used in the first step. Successful clones are selected on sucrose (SacB converts sucrose into a toxic substance in <I>E coli</I>). <A href="https://2012.igem.org/Team:Uppsala_University/Catsac">Read more</A></P><P><B>New working low-copy backbones:</B>
Due to a need for a low copy BioBrick plasmid in our project, we have devoloped a new series of BioBrick standard vectors. The new pSB4x15 backbones have a low copy pSC101 replication origin (~5 copies per cell) and ampicillin, chloramphenicol, kanamycin or spectinomycin antibiotic resistance markers. They are especially designed for Lambda Red recombineering in <I>E coli</I>. The backbone sequence is based on pSB3T5, but the <I>E coli</I> His operon terminator BBa_B0053 has been replaced with the late terminator of the <I>Salmonella</I> phage P22, similar to BBa_K59200.


The pSB4x15 series in brief:


<UL><LI>Verified low copy number</LI><LI>More accurate annotation</LI><LI>No homologies to the <I>E coli</I> genome that would interfere with Lambda red recombineering</LI><LI>Easy introduction of new resistance casettes</LI><LI>Easy introduction of new replication origins</LI><LI>Smaller backbone size</LI><LI>Flp recombinase target sites around the resistance casette available</LI><LI>LacIq versions for tight repression available</LI><LI>Thermosensitive versions available</LI></UL></P></DIV><CENTER><DIV id="sponsors"><H2>Sponsors</H2></DIV></CENTER><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:Uppsala_University/Project">http://2012.igem.org/Team:Uppsala_University/Project</A>&quot;</DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Uppsala_University/Project" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Uppsala_University/Project" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Uppsala_University/Project&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Uppsala_University/Project&amp;oldid=296690" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>