"
Page
Discussion
View source
History
teams
Log in
 
Team:Uppsala University/Modelling
From 2012.igem.org
Team Uppsala University – iGEM 2012
Facebook
Twitter
Blog (in Swedish)
Email
Home
Project
Description
Data Page
Notebook
Safety
Human Practice
Collaborations
Attributions
Results
Silencing sRNA:s
New backbones
Promoter tests
Chromoproteins
Gene deletion
Parts
Modelling
sRNA interactions
Team
About Us
Supervisors
Instructors
Students
Advisors
Donors
Contact us
Tweet
Silencing
Modelling sRNA | sRNAs validated to downregulate AAC(6') | sRNAs matching SYFP2
Modelling of small RNA interacting with the AAC(6)’5-UTR
Back to top
The modelling of our synthetic sRNA is a vital part for evaluating hypotheses of how sRNA molecules interact with their target mRNA. One canonical approach for sRNA stability is to evaluate secondary structures by minimum free energy (MFE) approaches. This gives a pointer for approximating the probability of different kind of interactions.
Stabile structures prevents translation A thermodynamic scheme of RNA interactions at different stages were made to show how an interaction between mRNA and sRNA could possibly work. So called toeholds are belived to play an important role in the interaction between sRNA and mRNA. [1] Understanding the interaction between RNA with different structures is a key to learn how to design a sRNA. Often the hybridization reaction between the sRNA and the mRNA starts with the unpaired nucleotides, the so called toeholds. To start a basepairing reaction, the sRNA needs a few basepairs to start hybridize to. The sRNA and the mRNA can then create a more stable secondary complex, hindering translation.
Reaction diagram sRNA
The sRNA 17 and mRNA for the AAC(6)-UTR were first calculated in CLC main workbench separately, after that we calculated the interaction structure between them .
This is a model of how a secondary complex can be created between the sRNA and mRNA.
Using a native scaffold to stabilize the interactions
The wildtype small RNA of E.coli K-12 MG1655 that we used as a template for constructing our own smallRNA is named Spot42 and has been shown to be interacting with the Hfq protein.
The sRNA Spot42 has like many other sRNA two distinctive functional parts, a recognition and a scaffold sequence. The recognition sequence binds the mRNA, while the scaffold sequence interacts with Hfq, an RNA binding protein. The sRNA-Hfq complex play an important role in the function of sRNA translational regulation.  (Holmquist, 2012)
Figure above. Panel A shows the secondary structure of scaffold
Spot42 and panel B shows a MMB generated three dimensional of
the Spot42 scaffold. HP1 and HP2 shows the positions of the two hairpins.
The secondary structure of the scaffold of Spot42 (figA) is known. However, no 3D structure of spot42 has been solved to date.  We have generated a 3D structure from the information in the secondary structure with molecular dynamics software MacroMoleculeBuilder (MMB) [1] and the 3D structure of Spot42 scaffold is presented in figure B. MMB allows the user to have full control of the biomolecule (DNA, RNA and/or protein) by applying forces between atoms, constraints and flexibility to different parts of the biomolecule.  [3]
To gain better understanding of the mechanisms of small RNA translation inhibition we have made 2D models of all our synthetic small RNAs. The 2D models were calculated in CLC main workbench using adapted versions of maximum free energy algorithms developed by [2] and thermodynamical parameters of Mfold version 3. IntaRNA, an RNA-interaction prediction software adapted for sRNA and ncRNA interactions [5] was used  to predict the sRNA-mRNA interactions of the candidate sRNAs.
The sRNAs were isolated, sequenced and analyzed to find the hybridizing base pairs. The sequences of the sRNAs that showed downregulation of YFP mainly had matching sequences in the 5’UTR of our target mRNA, although some of the sRNAs was shown to hybridize at the SYFP2 coding mRNA region. Two of these were further studied and modelled. At last, a prediction of the structure between the sRNA UU17 and AAC(6’)UTR mRNA was modelled.
Candidate sRNAs
Back to top
As you can see below, our sRNAs that downregulate the SYFP2 and the antibiotic resistance gene also seem to hybridize at the region close to the RBS and the start codon. This supports the idea that many sRNAs prevent the ribosome from binding to the RBS, thereby preventing translation. (Erik Holmquist, 2012) Our structure prediction data from CLC also shows that the hybridizing region of our silencing synthetic sRNAs often have strong secondary structures, with small hairpin loops.
sRNA UU17
sRNA UU17 was found to be one of the two most effective sRNAs against the resistance gene AAC(6’).
∆G =  -31.2 kcal/mol
Length in base pairs = 82 bp
Number of hybridizing base pairs = 17
Maximum number of hybridizing bases in a row = 13
Result on Etest = 48 µg/ml
∆G = -14.3kcal/mol
SYFP2 downregulation = 78%
sRNA UU37
sRNA UU37 was the the other of the two most effective sRNAs against the resistance gene AAC(6’).
∆G = -31.9 kcal/mol
Length in base pairs = 83 bp
Number of hybridizing base pairs = 24
Maximum number of hybridizing bases in a row = 7
Result on Etest = 64 µg/ml
∆G = -15.7 kcal/mol
SYFP2 downregulation: 82%
sRNA UU46
Complimentary to AAC(6) UTR
∆G =  -28.7 kcal/mol
Length in base pairs = 65 bp
Number of hybridizing base pairs = 8 bp
Maximum number of hybridizing bases in a row = 8 bp
Result on Etest > 256µg/ml
SYFP2 downregulation: 83%    ∆G = -11.2kcal/mol
sRNA UU55
Complimentary to AAC(6) UTR
∆G =  -31.2 kcal/mol
Length in base pairs = 83
Number of hybridizing base pairs = 8
Maximum number of hybridizing bases in a row = 8
Result on Etest = 124 µg/ml
SYFP2 downregulation= 58%    ∆G = -10.2kcal/mol
sRNAs matching SYFP2
Back to top
Below are two examples of structures that were found to match regions in the coding sequence of the SYFP2. It is apparent that false positives will appear when using a reporter system that itself can be repressed, but by modelling interactions with the target mRNA, we save much time knowing which sRNAs are not of direct interest for downregulation of the actual target gene.
sRNA UU52
sRNA basepairing to reporter gene SYFP2
∆G =  -28.3kcal/mol
Length in base pairs =82
Number of hybridizing base pairs =10
Maximum number of hybridizing bases in a row = 6
∆G = -5.6 kcal/mol
sRNA UU53
sRNA basepairing to reporter gene SYFP2
∆G =  -34.5 kcal/mol
Length in base pairs =85
Number of hybridizing base pairs =21
Maximum number of hybridizing bases in a row =9
∆G = -10.9kcal/mol
Although a randomized library was used to find an optimized sRNA for the downregulation of sRNA, we asked ourselves how an sRNA with perfect complementarity to the RBS and UTR region would work. This complementary sRNA was synthesized and we tested its function.  The results were not simple to decipher, because it seems that this sRNA targets more than the AAC(6’) UTR. The construct confered a large fitness cost on the bacteria, with very slow growth and all sorts of different colony morphologies and measurements of SYFP2 activity on the reporter strain. Some results showed a significant decrease in SYFP2 activity, while measurements of a different clone showed only moderate to non existing downregulation of SYFP2.
In addition to this, the sRNA selected from the random library seemed to have secondary structure on the hybridizing part, while the optimal secondary structure of our perfectly matching sRNA is without any base pairing within the antisense region. This makes us speculate that this sRNA might have unspecific base pair matching with many different bacterial mRNAs.
The approach to construct a perfect complementary is maybe not the best way to make a working sRNA, because there are so many other mechanisms involved that are difficult to predict.
sRNA UU01
Perfect complimentary to AAC(6)
∆G =  -31.2 kcal/mol
Length in base pairs = 83
Number of hybridizing base pairs = 8
Maximum number of hybridizing bases in a row = 8
Result on Etest = 124 µg/ml
∆G = -10.2kcal/mol
AAC(6')-SYFP2 and sRNA UU17 interaction
The mRNA of AAC(6') 5-UTR and a part of the coding strand for SYFP2 was included.
∆G =  -23,4 kcal/mol
The mRNA and sRNA UU17 in a combined structure
∆G =  -60,8 kcal/mol kcal/mol
References
Back to top
[1] Rodrigo, G., Landrain, T.E., Jaramillo, A., 2012. De novo automated design of small RNA circuits for engineering synthetic riboregulation in living cells. Proc. Natl. Acad. Sci. U.S.A. 109, 15271–15276.
[2] Zuker, M., 1989b. The use of dynamic programming algorithms in  RNA secondary structure prediction, 159–184. In Waterman, M., ed. Mathematical Methods for DNA Sequences, CRC Press, Boca Raton, FL
[3]. Samuel C Flores and Russ B Altman. 2010 Turning limited experimental information intio 3D models of RNA. RNA 16(9):1769-78.
Notes about CLC workbench: Uses thermodynamical parameters of Mfold, version 3, found on http://www.bioinfo.rpi.edu/zukerm/rna/energy/
[4] Erik Holmquist Macromolecular Matchmaking Mechanisms and Biology of Bacterial Small RNAs, 2012
[5] Smith, C., Heyne, S., Richter, A.S., Will, S., Backofen, R., 2010. Freiburg RNA Tools: a web server integrating INTARNA, EXPARNA and LOCARNA. Nucleic Acids Research 38, W373–W377.
Sponsors
Retrieved from " http://2012.igem.org/Team:Uppsala_University/Modelling "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
