Team:LASATX/Protocols
Medals
Collaboration
Safety
Human Practices
Notebook
Protocols
Parts
Project
Description
Experiment
Results
Design
Team
Biographies
Attribution
Home
Protocols
Here is a list of the protocols we used in our project.
STEP 1: PCR Amplification
Traditional Protocol
Reactants
5x Buffer
20 uL
Phusion Enzyme
1 uL
dNTPs
5 µL
Template
1 µL
Primers
1 µL x 2
H2O
Volume Total
100 µL
Thermocycler (Ross Phusion)
Temp (C°)
Time
94°
2:00
80°
0:30
98°
0:20
Denature
65°
0:20
Anneal
73°
1:00 (30 Cycles)
Extend
94°
0:20
50°
0:20
68°
1:45 (10 Cycles)
4°
Forever
Non-Traditional Protocol
Hotstart AccuPrime Pfx MasterMix
45 µL
DNA Template
X µL (30 ng)
Primers (20 uM)
2 µL
MgCl2 (25 mM)
2 µL (1 mM)
Total Volume
49 + X µL
Pfx Enzyme
1 µL
10X Reaction Mix
5 µL
DNA Template
1 µL (30 ng)
Primers (20 uM each)
1 µL x 2
MgCl2 (25 mM)
1 µL (0.5 mM)
DMSO (Use Dilution)
10 µL
H2O
Total Volume
50 µL
Temp (C°)
Time
10 Cycles
95°
5:00
2:00 if NOT using Hotstart
95°
At 0.3°/s to 50°
98°
0:20
Denature
72°
X s
73°
1:00 (30 cycles)
Extend
15 Cycles
95°
0:20
50°
0:20
55°
0:20
72°
1:00
1 Cycle
72°
10:00
4°
Forever
STEP 2: Gel Electrophoresis
Making 1% TAE Gels
1. 100 mL TAE buffer + 1 g agarose (Nuseive 3:1)
2. Stir
3. Microwave
4. Add 1 drop of EtBr
5. Use large combs for 50 uL PCR reactions and small combs for 10 uL reactions
6. Let the gel cool
Gel Electrophoresis of PCR Product
1 kb ladder (2 µL), loading dye (2 µL), water (8 µL)
pcr product (45 µL), dye (7.5 µL) (6:1 ratio)
STEP 3: Gel Purification (Promega)
Load and run the dye
Weight a 1.5 mL microcentrifuge tube for each DNA fragment to be isolated and record weight.
Visualize and photograph the DNA using a long-wavelength UV lamp and an intercalating dye (EtBr). Excise DNA fragment of interest in a minimal volume of agarose using a clean scalpel or razor blade. Transfer the gel slice to the weighed microcentrifuge tube and record the weight. Subtract the weight of the empty tube from the total weight to obtain the weight of the gel slice. The gel slice may be stored at 4°C or at –20°C for up to one week in a tightly closed tube under nuclease-free conditions before purification.
Notes: The gel slice may be stored at 4°C or at –20°C for up to one week in a tightly closed tube under nuclease-free conditions before purification.
Add Membrane Binding Solution at a ratio of 1 mL of solution per 1 g of agarose gel slice.
Vortex the mixture and incubate at 50–65°C for 10 minutes or until the gel slice is completely dissolved. Vortex the tube every few minutes to increase the rate of agarose gel melting. Centrifuge the tube briefly at room temperature to ensure the contents are at the bottom of the tube. Once the agarose gel is melted, the gel will not resolidify at room temperature.
Place one SV Minicolumn in a Collection Tube for each dissolved gel slice or PCR amplification.
Transfer the dissolved gel mixture or prepared PCR product to the SV Minicolumn assembly and incubate for 1 minute at room temperature.
Centrifuge for 1 minute and discard liquid.
Wash with 700 µl of Membrane Wash Solution, centrifuge 1 minute, and discard.
Wash with 500 µl of Membrane Wash Solution, centrifuge 5 minutes, and discard.
Centrifuge 1 minute with the microcentrifuge lid open (or off ) to allow evaporation of any residual ethanol.
Transfer SV Minicolumn to a clean 1.5 ml microcentrifuge tube.
Apply 15 µl of Nuclease-Free Water directly to the center of the column. Make sure membrane is completely covered with Nuclease-free water. Incubate at room temperature for 1 minute. Centrifuge 1 minute at 16,000 × g. Repeat.
Discard SV Minicolumn, store tube containing the eluted DNA at 4°C or –20°C.
Notes for "DNA Purification by Centrifugation":
MWS = Membrane Wash Solution
RT = room temperature
Centrifuge at 16,000 x g
Nanodrop/Speed Vac: if concentration not high enough, speed vac and measure again.
STEP 4: PCR Clean-Up (Zymo) of Remaining PCR Product
Protocol
Protocol:All centrifugation steps should be performed between 10,000 - 16,000 x g
In a 1.5 ml microcentrifuge tube, add 2-7 volumes of DNA Binding Buffer to each volume of DNA sample (see table below). Mix briefly by vortexing.
Application
DNA Binding Buffer: Sample
Example
Genomic DNA (>2 kb)
2 : 1
200 uL : 100 uL
PCR Product         DNA Fragment
5 : 1
500 uL : 100 uL
ssDNA (eg. cDNA, M13 Phage)
7 : 1
700 uL : 100 uL
Transfer mixture to a provided Zymo-Spin™ Column in a Collection Tube.
Centrifuge for 30 seconds. Discard the flow-through.
Add 200 µl DNA Wash Buffer to the column. Centrifuge for 30 seconds. Repeat the wash step. 5. Add 7 uL water directly to the column matrix and incubate at room temperature for one minute. Transfer the column to a 1.5 ml microcentrifuge tube and centrifuge for 30 seconds to elute the DNA.
Ultra-pure DNA is now ready for use.
Note: All centrifugation steps should be performed between 10,000 - 16,000 x g
STEP 5: Gibson
Reactants
Note:
10 ng/kb
If <500 bp, add double or triple
Gibson Mix1, 1.3 X  15 uL
Insert2 (~20 ng/uL)  2.5 uL (50 ng)
Vector PCR 2.13 2.5 uL
Total Vol     20 uL
Protocol
Incubate 1 hour at 50 C in thermocycler.
15 uL aliquots in pcr tube box OR box next to antibiotics in B lab freezer
PCR product (gel purified or pcr clean-up’d)
Blue sticker
STEP 6A: Chemical Transformation: One Shot TOP10 Competent Cells
Chemical Transformation
Place Gibson reaction on ice.
Thaw on ice one 50 uL vial of One Shot TOP10 cells* for each ligation/transformation.
Pipette 5 uL of each Gibson reaction (OR 1 uL of miniprep) directly into the vial of competent cells and mix by tapping gently. Do not mix by pipetting up and down. The remaining Gibson mixture(s) can be stored at -20 C.
Incubate vials on ice for 30 minutes.
~take out vial of SOC~
Incubate for exactly 30 seconds in the 42C water bath. Do not mix or shake.
Remove vial(s) from 42C bath and place them on ice.
Add 250 uL of pre-warmed SOC medium to each vial. SOC is a rich medium; sterile technique must be practiced to avoid contamination.
Place vials in microcentrifuge rack on its side and secure with tape to avoid loss of the vial(s). Shake the vial(s) at 37 C for exactly 1 hour at 225 rpm in a shaking incubator.
~take out agar plates. place at 37 C~
Spin down and resuspend in 100 uL LB.
Spread 50 uL from each transformation vial on separate, labeled LB agar plates. Store the remaining transformation mix at 4C.
Invert the plates and incubate at 37 C overnight.
Select colonies and analyze by plasmid isolation, pcr or sequencing.
*B lab freezer, bottom row, right most, white pcr tube box.
STEP 6B: Electrotransformation of E. coli
Electroporate: Pulse at 2.5 kV
Inoculate 500 ml of L-broth with 1/100 volume of a fresh overnight E. coli culture.
Grow the cells at 37 °C shaking at 300 rpm to an OD600 of approximately 0.5–0.7 (the best results are obtained with cells that are harvested at early- to mid-log phase; the appropriate cell density therefore depends on the strain and growth conditions).
Chill cells on ice for ~20 min. For all subsequent steps, keep the cells as close to 0 °C as possible (in an ice/water bath) and chill all containers in ice before adding cells. To harvest, transfer the cells to a cold centrifuge bottle and spin at 4000 x g for 15 minutes at 4 °C.
Carefully pour off and discard the supernatant. It is better to sacrifice the yield by pouring off a few cells than to leave any supernatant behind.
Gently resuspend the pellet in 500 ml of ice-cold 10% glycerol. Centrifuge at 4000 x g for 15 minutes at 4 °C; carefully pour off and discard the supernatant
Resuspend the pellet in 250 ml of ice-cold 10% glycerol. Centrifuge at 4000 x g for 15 minutes at 4 °C; carefully pour off and discard the supernatant.
Resuspend the pellet in ~20 ml of ice-cold 10% glycerol. Transfer to a 30 ml sterile Oakridge tube. Centrifuge at 4000 x g for 15 minutes at 4 °C; carefully pour off and discard the supernatant.
Resuspend the cell pellet in a final volume of 1–2 ml of ice-cold 10% glycerol. The cell concentration should be about 1–3 x 1010 cells/ml. This suspension may be frozen in aliquots on dry ice and stored at -70 °C. The cells are stable for at least 6 months under these conditions.
STEP 7: Electroporation
Electroporate: Pulse at 2.5 kV
Thaw the cells on ice. For each sample to be electroporated, place a 1.5 ml microfuge tube and either a 0.1 or 0.2 cm electroporation cuvette on ice.
In a cold, 1.5 ml polypropylene microfuge tube, mix 40 µl of the cell suspension with 1 to 2 µl of DNA (DNA should be in a low ionic strength buffer such as TE). Mix well and incubate on ice for ~1 minute. (Note: it is best to mix the plasmids and cells in a microfuge tube since the narrow gap of the cuvettes prevents uniform mixing.)
Set the MicroPulser to “Ec1” when using the 0.1 cm cuvettes. Set it to "Ec2" or "Ec3" when using the 0.2 cm cuvettes. See Section 4 for operating instructions.
Transfer the mixture of cells and DNA to a cold electroporation cuvette and tap the suspension to the bottom. Place the cuvette in the chamber slide. Push the slide into the chamber until the cuvette is seated between the contacts in the base of the chamber. Pulse once.
Remove the cuvette from the chamber and immediately add 1 ml of SOC medium to the cuvette. Quickly but gently resuspend the cells with a Pasteur pipette. (The period between applying the pulse and transferring the cells to outgrowth medium is crucial for recovering E. coli transformants (Dower et al., 1988). Delaying this transfer by even 1 minute causes a 3-fold drop in transformation. This decline continues to a 20-fold drop by 10 minutes.
Transfer the cell suspension to a 17 x 100 mm polypropylene tube and incubate at 37 °C for 1 hour, shaking at 225 rpm.
Check and record the pulse parameters. The time constant should be close to 5 milliseconds. The field strength can be calculated as actual volts (kV) / cuvette gap (cm).
Plate on selective medium.
STEP 8: ON Cultures
Per Culture Tube:
Toothpick w/ Colony
5 mL LB
5 uL Antibiotic
STEP 9: Making Glycerol Stocks
In a microcentrifuge tube, add:
800 uL ON culture
200 uL 80% glycerol
Freeze at ~80 C
STEP 10: Miniprep (Qiagen)
Procedure
Resuspend pelleted bacterial cells in 250 µl Buffer P1 (w/ RNase A*) and transfer to a microcentrifuge tube. No cell clumps should be visible after resuspension of the pellet.
Add 250 µl Buffer P2 and gently invert the tube 4–6 times to mix. Mix gently by inverting the tube. Do not vortex, as this will result in shearing of genomic DNA. If necessary, continue inverting the tube until the solution becomes viscous and slightly clear. Do not allow the lysis reaction to proceed for more than 5 min.
Add 350 µl Buffer N3 and invert the tube immediately but gently 4–6 times. To avoid localized precipitation, mix the solution gently but thoroughly, immediately after addition of Buffer N3. The solution should become cloudy.
Centrifuge for 10 min at 13,000 rpm (~17,900 x g) in a table-top microcentrifuge. A compact white pellet will form.
Apply the supernatants from step 4 to the QIAprep Spin Column by decanting or pipetting.
Centrifuge for 30–60 s. Discard the flow-through.
(Optional): Wash the QIAprep Spin Column by adding 0.5 ml Buffer PB and centrifuging for 30–60 s. Discard the flow-through. This step is necessary to remove trace nuclease activity when using endA+ strains such as the JM series, HB101 and its derivatives, or any wild-type strain, which have high levels of nuclease activity or high carbohydrate content. Host strains such as XL-1 Blue and DH5α™ do not require this additional wash step.
Wash QIAprep Spin Column by adding 0.75 ml Buffer PE and centrifuging for 30–60 s.
Discard flow-through; centrifuge 1 min to remove residual wash buffer.
Place the QIAprep column in a clean 1.5 ml microcentrifuge tube.
To elute DNA, add 25 µl Buffer EB (10 mM Tris·Cl, pH 8.5) or water to the center of each QIAprep Spin Column, let stand for 1 min, and centrifuge for 1 min. Repeat for 2x DNA.
*want 50 ug/mL RNase A in P1
STEP 11: Sequencing Sample Submission
Add DNA to a total of 250 ng.
Use 10 pmol primer (1 uL of 10 uM) per sample.
Add water to 12 uL.
Send individual samples in 1.5 mL tubes.
Write order number on top of each tube and label.
Spin down.
Place in green racks in fridge next to  room 1.42G.
Note: Samples put in before 10 am will be sequenced.
STEP 12: Analyze Sequences, Sequence Verification
Geneious
Select for, rev and template sequences
Map to Reference
Consensus sequences: mismatches? Insertions/deletions?
If have plates, clone more (step 6)
No plates, quikchange (step 10)
STEP 13: Quikchange
Design Primers
5’- For: 10 bp  x 20 bp -3’
3’- Rev: 20 bp  x 10 bp -5’
Proceed to Step 11
STEP 14: PCR Transformation w/ Dpn1
Run PCR (touchdown protocol)
~make TAE gel w/ small wells~
PCR clean-up (changing buffers - eventually elute in water, which dpn1 can work in)
Run gel: b4 dpN1
product (4 uL), dye (2 uL), water (6 uL)
ladder (2 uL), dye (2uL), water (8 uL)
Add DpN1 to remaining pcr product
Add 10X dpn1 buffer (do first)
Add 3 uL dpn1
Incubate at 37 C 15min - 2h or overnight
Run gel: a/f dpN1 ← optional
product (4 uL), dye (2 uL)
ladder
Chemical transformation: 5 uL + 50 uL cells
Restriction Digest
Enzyme Master Mix for Plasmid Backbone (25 uL total, for 5 reactions)
5 uL NEB Buffer 2
0.5 uL BSA
0.5 uL EcoRI-HF
0.5 uL Pstl
0.5 uL Dpnl (Used to digest any template DNA from production)
18 uL dH2O
Digest Plasmid Backbone
Add 4 uL linearized plasmid backbone (25 ng/uL for 100 ng total)
Add 4 uL of Enzyme Master Mix
Enzyme Master Mix for Part A (25 uL total, for 5 reactions<
5 uL NEB Buffer 2
0.5 uL BSA
0.5 uL EcoRI-HF
0.5 uL Pstl
18.5 uL dH2O
Digest Part A
Add 4 uL Part A (25 ng/uL for 100 ng total)
Add 4 uL of Enzyme Master Mix
Digest all three reactions at 37 C/30 min, heat kill 80 C/20 min
Ligation
Add 2ul of digested Plasmid Backbone (25 ng)
Add equimolar amount of Part A (EcoRI-HF PstI digested) fragment (< 3 ul) (~75 ng insert)
Add 1 ul T4 DNA ligase buffer. Note: Do not use quick ligase
Add 0.5 ul T4 DNA ligase
Add water to 10 ul
Ligate 16C/30 min, heat kill 80C/20 min
Transform with 1-2 ul of product
Sequence Verification: TOPO-TA
STEP 1: PCR Amplification of Insert
use Hotstart taq PCR supermix (platinum) instead of Accuprime pfx
touchdown protocol for thermocycler
STEP 2B: Run Gel of 10 uL of Product
STEP 2C: Perform TOPO-TA Cloning (w/ Remaining Product)
Reagents
Fresh PCR Product
0.5 - 4 µL
Salt Solution
1 µL
Water
Add to Final Volume 5 µL
Template
1 µL
TOPO Vector
1 µL
Volume Total
6 µL
Mix reaction gently and incubate for 5 minutes at room temperature (22-23°C).
Note: For most applications, 5 minutes will yield plenty of colonies for analysis. Depending on your needs, the length of the TOPO® Cloning reaction can be varied from 30 seconds to 30 minutes. For routine subcloning of PCR products, 30 seconds may be sufficient. For large PCR products (> 1 kb) or if you are TOPO® Cloning a pool of PCR products, increasing the reaction time will yield more colonies.
Place reaction on ice.
Note: You may store the TOPO® Cloning reaction at -20°C overnight.
Continue to 5. Transforming TOP10 Competent cells
pchBA Genomic Extraction
Objective: extract gene pchBA out of bacterium Pseudomonas aeruginosa; use as possible replacement for pchBA gene in COWg construct.
Order primers to amplify pchBA gene and add RBS extensions on both ends of gene.
Michelle: ON culture, lyse cells
Miniprep: get genomic DNA out of cells
PCR: amplify pchBA gene out of genome (using genomic DNA (2), For/Rev primers (1), buffer --which polymerase? MM?)
Gel electrophoresis & DNA purification: purify DNA, nanodrop for concentration
Assembly of Entire COWg Construct
Starting With:
All four COWg fragments in cloning vectors
pET21
STEP 1: PCR Amplification (of fragment out of cloning vector 2.1)
COWg 1, F and R
COWg 2, F and R
COWg 3, F and R
COWg 4, F and R
pET21, F and R
STEP 2: Gel Electrophoresis of 10 µL Product
STEP 3: PCR Clean-Up of Remaining Product
STEP 4: Giant Gibson of COWg 1 + COWg 2 + COWg 3 + COWg 4 + pET21
STEP 5: Chemical Transformation into TOP10 cells
STEP 6: ON cultures
STEP 7: Miniprep
STEP 8: Sequencing Prep, Send in for Sequencing, Analyze Sequencing
Experimentation with Full Construct
STEP 1: Chemical Transformation of Sequence-Verified Minipreps into BL21(DE3) Cells (Expression Strain)
