<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_BostonU_Design skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:BostonU/Design</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="navBar"><DIV class="leoLogo"><H1>Project LEO</H1></DIV><DIV id="main1" class="dropdownbutton"><A id="navbutton" href="https://igem.org/2018_Judging_Form?team=BostonU">JUDGING</A><DIV class="drop-opt"><A href="https://2018.igem.org/Team:BostonU/Medals">Medals</A><A href="https://igem.org/2018_Judging_Form?team=BostonU">Judging Form</A></DIV></DIV><DIV id="main2" class="dropdownbutton"><A id="navbutton" href="https://2018.igem.org/Team:BostonU/Human_Practices">HUMAN PRACTICES</A><DIV class="drop-opt"><A href="https://2018.igem.org/Team:BostonU/Human_Practices">Integrated Human Practices</A><A href="https://2018.igem.org/Team:BostonU/Public_Engagement">Public Engagement</A><A href="https://2018.igem.org/Team:BostonU/GreenGEM">GreenGEM</A><A href="https://2018.igem.org/Team:BostonU/Challenge">2018 Sustainable Labs Challenge</A><A href="https://2018.igem.org/Team:BostonU/Safety">Safety</A></DIV></DIV><DIV id="main3" class="dropdownbutton"><A id="navbutton" href="https://2018.igem.org/Team:BostonU/Description">PROJECT</A><DIV class="drop-opt"><A href="https://2018.igem.org/Team:BostonU/Description">Motivation</A><A href="https://2018.igem.org/Team:BostonU/Design">Design</A><A href="https://2018.igem.org/Team:BostonU/Measurement">Measurement</A><A href="https://2018.igem.org/Team:BostonU/Experiments">Characterization</A><A href="https://2018.igem.org/Team:BostonU/Model">Modeling</A><A href="https://2018.igem.org/Team:BostonU/Parts">Submitted Parts</A><A href="https://benchling.com/s/etr-tAKlAtAknVaj3nnihozY" style="background-color:white;">Notebook</A><A href="https://2018.igem.org/Team:BostonU/InterLab">InterLab</A></DIV></DIV><DIV id="main4" class="dropdownbutton"><A id="navbutton" href="https://2018.igem.org/Team:BostonU/Team">PEOPLE</A><DIV class="drop-opt"><A href="https://2018.igem.org/Team:BostonU/Team">Team</A><A href="https://2018.igem.org/Team:BostonU/Attributions">Attributions</A><A href="https://2018.igem.org/Team:BostonU/Collaborations">Collaborations</A></DIV></DIV><DIV id="main5" class="dropdownbutton"><A id="navbutton" href="https://2018.igem.org/Team:BostonU">HOME</A></DIV></DIV><DIV class="banner"><DIV class="centered">Design</DIV></DIV><DIV class="description" style="font-size:20pt;">LOV2 and PhiReX</DIV><DIV class="center" style="font-size:14pt;text-align:center;">
  Figure 1: LOV2 dimerizes under blue light, then binds in the promoter region to activate transcription of mRuby.
</DIV><DIV class="description" style="float:left; margin: 10px; padding:0px 150px;">
  BostonU aimed to characterize two light-inducible systems: blue light inducible LOV2 <SUP>1</SUP>, and red light inducible PhiReX <SUP>2</SUP>. LOV2 is a robust transcriptional activation system with reduced toxicity to the host cell over other inducible methods<SUP>1</SUP>. When exposed to blue light, the LOV proteins dimerize, exposing DNA-binding domains that anchor the LOV2 protein in the promoter region. Once anchored, a pair of VP16 activating domains activate transcription of the gene of interest. After the blue light is turned off, LOV2 unbinds from the DNA and reverses dimerization, deactivating transcription.
<DIV class="center" style="margin-left:25%;font-size:14pt;text-align:center;">
Figure 2: PhiReX's chromophores absorb red light, causing a conformational change in the transcription factor PhyBNT, localized to  the promoter region, allowing PIF3 to bind and activate transcription of mRuby.
</DIV>
PhiReX is a multi-part system which uses plant-derived photoreceptors to achieve flexible regulation of a gene of interest. PhiReX requires exogenous chromophores to be expressed along with the binding and activating proteins at the integrated promoter<SUP>2</SUP>. When exposed to red light, these chromophores absorb light and cause a conformational change in the photoreceptor transcription factor, PhyBNT. PhyBNT is bound to the promoter region of the gene, and upon conformational change, opens to form a heterodimer with a PIF3 domain, which activates transcription of the gene of interest. When  the red light is removed, PIF3 uncouples from PhyBNT which returns to its original conformation, deactivating transcription.
<DIV class="center" style="font-size:14pt;text-align:center;">
  Figure 3: Absorbance spectra of LOV2 and PhiReX. LOV2 peaks at 450-490 nm and PhiReX peaks at 640-670 nm.
</DIV>
  Because the activation wavelengths of LOV2 and PhiReX lie on opposite ends of the visible light spectrum, the systems permit orthogonal control of gene expression (see <A href="Experiments">Characterization</A>) By characterizing the two orthogonal systems, BostonU provides a means for multiplexed control of gene expression in the same cell. 
Further, recent research using LOV2 has demonstrated that different light pulse dynamics can be used to attenuate gene expression in design of optimal activation programs<SUP>3</SUP>. The Khalil Labâ€™s culturing platform, the eVOLVER, is primedto characterize across multiple parameters, including OD and temperature, as described below. We leveraged the tunability of eVOLVER vials to probe the effects of light pulsing dynamics on reporter activation from our light inducible systems.

  To characterize across different light pulse inputs, we gathered data on a unit step input response to observe activation and deactivation curves for both systems. Using these curves, we generated a model that enabled us to predict the output of each system in response to different pulsatile inputs. By characterizing these outputs in response to different light pulsing programs, we obtained data which will help select an appropriate pulsatile input for a desired expression pattern.
  
  In addition, we characterized the effects of expressing each system component in varying yeast selection loci to discover the optimal integrations for  our systems. Using the Dueber Lab's modular cloning scheme and yeast cloning kit detailed below, we were able to rapidly swap  homology arms to build parts to integrate our systems across  the HO, LEU2, and URA3 loci in yeast. Thus, we obtained data on optimal part integration sites determined to achieve optimal  expression outputs.

</DIV><DIV class="description" style="font-size:20pt;">Assembling Plasmids</DIV><DIV class="description">
In order to demonstrate induction by our two systems, BostonU placed a fluorescent reporter, mRuby2, under the transcriptional control of LOV2 and PhiReX. To do so, we made use of the Dueber Lab Golden Gate Modular Cloning Kit<SUP>4</SUP>, a catalog of parts and connectors that uses Golden Gate assembly to build  plasmids for <I>S. cerevisiae</I> integration.

    
BostonU also followed the modular cloning scheme designed for  the Dueber Kit<SUP>4</SUP>, adopted in the Khalil Lab for assembly of Level 0, 1, and 2 parts. Level 0 parts (pKL0s) are building blocks assembled via a Golden Gate assembly using BsmBI and T7 DNA ligase, and contain individual pieces such as a promoter or a coding sequence. Level 1 parts (pKL1s) are assembled using NEB's Golden Gate Mix, which contains BsaI and T7 ligase, and contain executable genes, complete with a promoter, coding sequence, and terminator. Level 2 parts (pKL2s) are assembled with BsmBI and T7 ligase, and are cassettes containing suites of desired genes.
    <DIV class="center" style="font-size:14pt;text-align:center;">
  Figure 4: A figure from the Dueber paper showing a schematic view of different part levels.
</DIV>
All plasmids contain a bacterial origin of replication and antibiotic resistance marker, for easy replication in <I>E. coli</I>, as well as specific overhangs to ensure specific directional connections during assembly. Level 1 parts contain assembly connectors, promoters, coding sequences, terminators, and a yeast origin of replication and selection marker. Level 2 cassettes contain multiple Level 1 parts, and thus incorporate multiple promoters, coding sequences, and terminators.
    
Using the Dueber Kit and the modular cloning scheme allowed for easy exchange of different parts in our cloning designs. BostonU leveraged this easy exchange to rapidly construct variations of the LOV2 and PhiReX systems to determine the optimal configuration of parts as well as the site of integration into the yeast genome. During the Level 2 assembly, individual bacterial and yeast markers and origins of replication are excised, and replaced by the bacterial and yeast markers and origins contained in the backbone of the Level 2 cassette. Level 2 assemblies are the points at which we can choose homology arms to integrate multigene cassettes into different loci. Below is an example of a typical Level 2 part:
    <DIV class="center" style="font-size:14pt;text-align:center;">
      Figure 5: An example of a pKL2 (in this case, pKL2-226). This pKL2 contains LOV2 driving expression of mRuby, with homology for the HO locus. pKL2-226 confers hygromycin resistance as a selection marker.
    </DIV>
    Level 2 parts are linearized by digestion with NotI, which cuts out the bacterial marker and origin, exposing homology arms flanking the sequence for integration into the yeast genome.

We used different homology arms to characterize our systems in the HO, URA3, and LEU2 loci in yeast.
  
    We used different homology arms to characterize our systems in the HO, URA3, and LEU2 loci in yeast.
    </DIV><DIV class="description" style="font-size:20pt;">Strain Engineering and Testing</DIV><DIV class="description">
    Following assembly of Level 2 cassettes, plasmids were linearized and integrated into the yeast genome. Transformed yeast cells were then plated with either hygromycin antibiotic or with
    auxotrophy selection media.
  
  In order to test transformed colonies, we picked colonies into appropriate media in a culture block and grew overnight in a 30<SUP>o</SUP>C shaking incubator. Then, in the morning, we used a red or blue lamp to induce our systems. Notably, all of our strains contained a mutation that caused them to turn pink in the presence of adenine in growth media, a feature of the yPH500 parent strain. This pink color initially created noise in our measurements of mRuby expression. To minimize noise, we modulated this pink color by manually performing 60:40 cells:media dilutions every hour. Batch culture experiments in the shaking incubator provided first-pass data to find viable colonies for eVOLVER experiments.

</DIV><DIV class="description" style="font-size:20pt;">eVOLVER</DIV><DIV class="description">
  A robust, versatile culturing platform developed by Brandon Wong<SUP>5</SUP>, the eVOLVER empowers us to probe a vast parameter space to find the optimal conditions tailored for  PhiReX and LOV2 use. Equipped with red and blue LEDs, an LED/Diode pair to monitor OD, fluidics modules to control dilution rates, and temperature sensors, the eVOLVER provides fine control over light pulse dynamics, temperature, OD, and media flow rate, as well as a host of other parameters. The eVOLVER is capable of supporting 16 separate cultures at once, and separately controls each parameter of the 16 cultures, enabling us to rapidly obtain characterization data for our systems, here up to 16 hours. 
Marrying the best of both worlds, the eVOLVER augments the controllability  of a traditional bioreactor with the throughput of batch culture platforms. Using the eVOLVER, BostonU 2018 has gathered data on the optimal conditions under which PhiReX and LOV2 induce gene expression. See <A href="Experiments">Characterization</A> for details.
  
  eVOLVER experiments involved growing yeast cultures in customized glass vials overnight, then activating transcription in the morning using a red 660 nm LED or a blue 462 nm LED, chosen to coincide with the absorbance peaks of PhiReX and LOV2, respectively. The platform used fluidic lines to control dilution rates to keep cultures between 0.2 and 0.3 OD, in order to maximize light transmission through the culture in conjunction with maximizing expression of mRuby. Use of mRuby provided a convenient fluorescent reporter output, which was readily measured using flow cytometry of fixed culture samples. In addition, the eVOLVER maintained culture temperature at 30<SUP>o</SUP>C, the optimal temperature for yeast growth and allowed for live tracking of OD and temperature from remote locations via the CloudEvolution platform.

</DIV><DIV class="description" style="font-size:20pt;">References</DIV><DIV class="refs" style="padding:5px 150px; text-align:left;">

1. Zhao EM, Zhang Y, Mehl J, et al. Optogenetic regulation of engineered cellular metabolism for microbial chemical production. Nature. 2018;555:683. http://dx.doi.org/10.1038/nature26141.



2. Hochrein L, Machens F, Messerschmidt K, Mueller-Roeber B. PhiReX: A programmable and red light-regulated protein expression switch for yeast. Nucleic Acids Res. 2017;45(15):9193-9205. http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5587811/. doi: 10.1093/nar/gkx610.



3. Benzinger D, Khammash M. Pulsatile inputs achieve tunable attenuation of gene expression variability and graded multi-gene regulation. Nature Communications. 2018;9(1):3521. https://doi.org/10.1038/s41467-018-05882-2. doi: 10.1038/s41467-018-05882-2.



4. Wong BG, Mancuso CP, Kiriakov S, Bashor CJ, Khalil AS. A generalizable experimental framework for automated cell growth and laboratory evolution. bioRxiv. 2018. http://biorxiv.org/content/early/2018/03/12/280867.abstract.



5. Lee ME, DeLoache WC, Cervantes B, Dueber JE. A highly characterized yeast toolkit for modular, multipart assembly. ACS Synth Biol. 2015;4(9):975-986. https://doi.org/10.1021/sb500366v. doi: 10.1021/sb500366v.

</DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>