<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_NKU_CHINA_Improve skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:NKU CHINA/Improve</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container-fluid"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#myNavbar" aria-expanded="false"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse" id="myNavbar"><UL class="nav navbar-nav navbar-right"><LI><A href="https://2018.igem.org/Team:NKU_CHINA" style="margin-top:4px;">Home</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">Team
                  </A></LI><LI><A href="https://2018.igem.org/Team:NKU_CHINA/Team">Team</A></LI><LI><A href="https://2018.igem.org/Team:NKU_CHINA/Collaborations">Collaborations</A></LI></UL><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">Project
                  </A></LI><LI><A href="https://2018.igem.org/Team:NKU_CHINA/Description">Description</A></LI><LI><A href="https://2018.igem.org/Team:NKU_CHINA/Experiments">Experiments</A></LI><LI><A href="https://2018.igem.org/Team:NKU_CHINA/Model">Model</A></LI><LI><A href="https://2018.igem.org/Team:NKU_CHINA/Proof">Proof of Concept</A></LI><LI><A href="https://2018.igem.org/Team:NKU_CHINA/Demonstrate">Demonstrate</A></LI><LI><A href="https://2018.igem.org/Team:NKU_CHINA/Human_Practices" data-tooltip="true" title="Human Practices" style="margin-top:4px;">HP</A></LI><LI><A href="https://2018.igem.org/Team:NKU_CHINA/Safety" style="margin-top:4px;">Safety</A></LI><LI><A href="https://2018.igem.org/Team:NKU_CHINA/Attributions" style="margin-top:4px;">Attributions</A></LI><LI class="dropdown"><A class="data-toggle" data-toggle="dropdown" href="#">Parts
                   </A></LI><LI><A href="https://2018.igem.org/Team:NKU_CHINA/Basic_Part">Basic Parts</A></LI><LI><A href="https://2018.igem.org/Team:NKU_CHINA/Composite_Part">Composite Parts</A></LI><LI><A href="https://2018.igem.org/Team:NKU_CHINA/Improve">Improve</A></LI><LI><A href="https://2018.igem.org/Team:NKU_CHINA/InterLab" style="margin-top:4px;">InterLab</A></LI><LI class="dropdown"><A class="data-toggle" data-toggle="dropdown" href="#">Notebook
                  </A></LI><LI><A href="https://2018.igem.org/Team:NKU_CHINA/Notebook">Lab Notes</A></LI><LI><A href="https://2018.igem.org/Team:NKU_CHINA/Protocols">Protocols</A></LI></DIV></DIV><DIV id="loading_p"><P style="color: white;font-family:myTitle3;font-size:30px;position:relative;top:47%;left:46%">Loading...</P></DIV><MAIN><H1 class="text-center" style="font-size: 80px;font-weight: normal;color: white;padding-bottom: 0;margin-bottom: 20px; font-family: myTitle;margin-top: 30px;padding-top: 0;">Improved Parts</H1><DIV class="container"><DIV class="row"><DIV class="col-xs-12"><P class="improve_content">The 2018 NKU_CHINA team improved 2 existing parts, the promoter P<SUB><I>liaG</I></SUB> (<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K823000" style="color:yellow;">BBa_K823000</A>) of <I>Bacillus subtilis</I> (<I>B. subtilis</I>) and the tetracycline resistance protein TetA(C) (<A href="http://parts.igem.org/Part:BBa_J31006" style="color:yellow;">BBa_J31006</A>). Then we created TetA (optimized for LL3) (<A href="http://parts.igem.org/Part:BBa_K2705007" style="color:yellow;">BBa_K2705007</A>), P<SUB><I>liaG&amp;43</I></SUB> (<A href="http://parts.igem.org/Part:BBa_K2705008" style="color:yellow;">BBa_K2705008</A>), P<SUB><I>liaG&amp;43-1</I></SUB> (<A href="http://parts.igem.org/Part:BBa_K2705009" style="color:yellow;">BBa_K2705009</A>), P<SUB><I>liaG&amp;43-2</I></SUB> (<A href="http://parts.igem.org/Part:BBa_K2705010" style="color:yellow;">BBa_K2705010</A>), P<SUB><I>liaG&amp;43-3</I></SUB> (<A href="http://parts.igem.org/Part:BBa_K2705011" style="color:yellow;">BBa_K2705011</A>).</P></DIV></DIV></DIV><DIV class="container"><DIV class="row"><DIV class="col-xs-12"><H2 class="improve_header">1. P<SUB><I>liaG</I></SUB> Improvements</H2><H3 class="improve_header_two">Usage and Biology</H3><P class="improve_content">P<SUB><I>liaG</I></SUB> (BBa_K823000) is a weak, constitutive promoter from <I>B. subtilis</I>. It is responsible for the transcription of the last four genes of the <I>liaIHGFSR</I> cluster and therefore for the production of the components of the LiaRS system, which is important for the detection of cell wall antibiotics (Jordan et al., 2006). P<SUB><I>liaG</I></SUB> was evaluated with the <I>lux</I> operon as well as the LacZ as reporter. For more details, visit the Data page of the <A href="https://2012.igem.org/Team:LMU-Munich" style="color: yellow" target="_blank">LMU-Munich Team 2012</A> or get an overview of the whole project Beadzillus.</P></DIV></DIV></DIV><DIV class="container"><DIV class="row"><DIV class="col-xs-12"><H3 class="improve_header_two">Design</H3><P class="improve_content">We intended to improve the part P<SUB><I>liaG</I></SUB> (BBa_K823000). Inspired by the relatively strong constitutive promoter P<SUB>43</SUB>(BBa_K143013), we chose to mimic its structure, which can be recognized by both sigma factor 55 (the major sigma factor A) and sigma factor 37 (the lag phase sigma factor B). As the promoter P<SUB><I>liaG</I></SUB> is a constitutive <I>Bacillus subtilis</I>σ<SUP>A</SUP> promoter, we replace part of the promoter P<SUB><I>liaG</I></SUB> sequence with the σ<SUP>B</SUP> recognition site at the same relative position as the σ<SUP>B</SUP> recognition site to the σ<SUP>A</SUP> recognition site of the promoter P<SUB>43</SUB> to create the new promoter P<SUB><I>liaG&amp;43</I></SUB>. On the basis of this, we made some adjustments to the spacing between -35 region and -10 region, the sequence of -35 region and created P<SUB><I>liaG&amp;43-1</I></SUB>(BBa_K2705009), P<SUB><I>liaG&amp;43-2</I></SUB>(BBa_K2705010), P<SUB><I>liaG&amp;43-3</I></SUB> (BBa_K2705011).</P></DIV></DIV></DIV><DIV class="container"><DIV class="row"><DIV class="col-xs-12"><H3 class="improve_header_two">Luminescence Measurements</H3><P class="improve_content">The constitutive promoters P<SUB><I>liaG&amp;43</I></SUB> (BBa_K2705008), P<SUB><I>liaG&amp;43-1</I></SUB> (BBa_K2705009), P<SUB><I>liaG&amp;43-2</I></SUB> (BBa_K2705010), P<SUB><I>liaG&amp;43-3</I></SUB> (BBa_K2705011) were evaluated in the reporter vector pHT01-P<SUB><I>x</I></SUB>-GFP. The promoter activity results in gene expression and production of the green fluorescence protein (GFP). The fluorescence intensity (FI) was measured by the Multimode Plate Reader <I>Enspire</I> (PerkinElmer).</P><P class="improve_content">All clones showed a normal growth behavior. As the σ<SUP>B</SUP> recognition site of P<SUB>43</SUB> we added is a lag phase sigma factor binding site, we tested the FI at the very beginning (the lag phase, Fig.1a) of its life. All the modified promoters except for the promoter P<SUB><I>liaG&amp;43-2</I></SUB> (BBa_K2705010) outperformed the original promoter P<SUB><I>liaG</I></SUB> at the lag phase. The activity of the best promoter, P<SUB><I>liaG&amp;43</I></SUB> is about 1.5 times as high as the activity of P<SUB><I>liaG</I></SUB>. The promoter P<SUB><I>liaG&amp;43-2</I></SUB> (BBa_K2705010) still has a slightly higher activity than the promoter P<SUB><I>liaG</I></SUB> (Fig.1)</P><P class="tuzhu"><STRONG>Fig. 1 Luminescence measurement of the constitutive <SUB>Bacillus</SUB> sp. promoters P<SUB><I>liaG&amp;43</I></SUB>, P<SUB><I>liaG&amp;43-1</I></SUB>, P<SUB><I>liaG&amp;43-2</I></SUB>, P<SUB><I>liaG&amp;43-3</I></SUB> and P<SUB><I>liaG</I></SUB> in the reporter vector pHT01-P<SUB><I>x</I></SUB>-GFP.</STRONG> The strains were cultured in LB medium with 5 µg/mL chloramphenicol in dark for 2 hours. Fluorescence intensity of GFP was measured, which was normalized against OD<SUB>600</SUB>. Data indicate mean values ± standard deviations from three independent experiments performed in triplicates. ** Very significantly different (P &lt; 0.01) by Student's t-test.</P></DIV></DIV></DIV><DIV class="container"><DIV class="row"><DIV class="col-xs-12"><H3 class="improve_header_two">Result</H3><P class="improve_content">As these promoters are all constitutive promoters, the activity enhancement at the lag phase can surely enhance the promoters' effect, which has achieved the goal of promoter enhancement as well as the part P<SUB><I>liaG</I></SUB> (BBa_K823000)'s improvement.</P></DIV></DIV></DIV><DIV class="container"><DIV class="row"><DIV class="col-xs-12"><H2 class="improve_header">2. TetA Improvements</H2><H3 class="improve_header_two">Usage and Biology</H3><P class="improve_content">The tetracyclines (Tcs) is a group of antibiotics that have been widely used since the 1940s against both Gram-negative and Gram-positive bacteria. Resistance to Tcs has many mechanisms including the active efflux system of tetracycline, target gene mutations, enzymatic degradation of antibiotics, decreased drug permeability and so on. The tetracycline efflux pumps are encoded by several genes, including <I>tetA</I>, <I>tetC</I>, <I>tetE</I>, <I>tetG</I>, and <I>tetH</I>, which have been reported a major mechanism of Tcs resistance.</P></DIV></DIV></DIV><DIV class="container"><DIV class="row"><DIV class="col-xs-12"><H3 class="improve_header_two">Design</H3><P class="improve_content">Here we improved <I>tetA</I> (BBa_J31006) by codon optimizing for <I>Bacillus amyloliquefaciens</I>, which could result in Tcs resistance by encoding membrane-associated proteins that pump Tcs out of the cell, reducing intracellular drug concentrations and protecting intracellular ribosomes.</P><P class="improve_content">Here we entrust Genescript to help us do the optimization. In this case, the native gene employs tandem rare codons that can reduce the efficiency of translation or even disengage the translational machinery. We changed the codon usage bias in <I>Bacillus amyloliquefaciens</I> FZB42 by upgrading the CAI from 0.69 to 0.92. (Fig. 2) GC content and unfavorable peaks have been optimized to prolong the half-life of the mRNA. (Fig. 3) The Stem-Loop structures, which impact ribosomal binding and stability of mRNA, were broken. In addition, our optimization process has screened and successfully modified several negative cis-acting sites.</P></DIV></DIV></DIV></MAIN><FOOTER><DIV class="container-fluid myFooter"><DIV class="row"><DIV class="col-xs-4 footer-w3ls"><H3 style="color: white; font-size: 30px;">Contacts</H3><H3 style="color: white; font-size: 30px;">Address</H3><ADDRESS style="color: white; font-size: 17px;">
                    Nankai University
                    No.94 Weijin Road, Nankai District
                    Tianjin, P.R.China 300071</ADDRESS></DIV><DIV class="col-xs-12"><P class="Copyright" style="text-align: center; color: white;">Copyright © Team NKU_CHINA 2018</P></DIV></DIV></DIV></FOOTER><A href="#0" class="cd-top">Top</A></DIV></DIV></DIV></DIV></DIV></BODY></HTML>