<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_NKU_CHINA_Protocols skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:NKU CHINA/Protocols</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container-fluid"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#myNavbar" aria-expanded="false"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse" id="myNavbar"><UL class="nav navbar-nav navbar-right"><LI><A href="https://2018.igem.org/Team:NKU_CHINA" style="margin-top:4px;">Home</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">Team
                  </A></LI><LI><A href="https://2018.igem.org/Team:NKU_CHINA/Team">Team</A></LI><LI><A href="https://2018.igem.org/Team:NKU_CHINA/Collaborations">Collaborations</A></LI></UL><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">Project
                  </A></LI><LI><A href="https://2018.igem.org/Team:NKU_CHINA/Description">Description</A></LI><LI><A href="https://2018.igem.org/Team:NKU_CHINA/Experiments">Experiments</A></LI><LI><A href="https://2018.igem.org/Team:NKU_CHINA/Model">Model</A></LI><LI><A href="https://2018.igem.org/Team:NKU_CHINA/Proof">Proof of Concept</A></LI><LI><A href="https://2018.igem.org/Team:NKU_CHINA/Demonstrate">Demonstrate</A></LI><LI><A href="https://2018.igem.org/Team:NKU_CHINA/Human_Practices" data-tooltip="true" title="Human Practices" style="margin-top:4px;">HP</A></LI><LI><A href="https://2018.igem.org/Team:NKU_CHINA/Safety" style="margin-top:4px;">Safety</A></LI><LI><A href="https://2018.igem.org/Team:NKU_CHINA/Attributions" style="margin-top:4px;">Attributions</A></LI><LI class="dropdown"><A class="data-toggle" data-toggle="dropdown" href="#">Parts
                   </A></LI><LI><A href="https://2018.igem.org/Team:NKU_CHINA/Basic_Part">Basic Parts</A></LI><LI><A href="https://2018.igem.org/Team:NKU_CHINA/Composite_Part">Composite Parts</A></LI><LI><A href="https://2018.igem.org/Team:NKU_CHINA/Improve">Improve</A></LI><LI><A href="https://2018.igem.org/Team:NKU_CHINA/InterLab" style="margin-top:4px;">InterLab</A></LI><LI class="dropdown"><A class="data-toggle" data-toggle="dropdown" href="#">Notebook
                  </A></LI><LI><A href="https://2018.igem.org/Team:NKU_CHINA/Notebook">Lab Notes</A></LI><LI><A href="https://2018.igem.org/Team:NKU_CHINA/Protocols">Protocols</A></LI></DIV></DIV><DIV id="loading_p"><P style="color: white;font-family:myTitle3;font-size:30px;position:relative;top:47%;left:46%">Loading...</P></DIV><MAIN><H1 class="text-center" style="font-size: 80px;font-weight: normal;color: white; font-family: myTitle;margin-top: 20px;margin-bottom:0;">Protocols</H1><H2 class="Protocols_header text-center">1. The Markerless Gene Replacement Method for <I>Bacillus amyloliquefaciens</I> LL3.</H2><DIV class="container"><DIV class="row"><DIV class="col-xs-12"><H3 class="header_3">Abstract</H3></DIV><DIV class="col-xs-12"><P class="Protocols_content">This is a markerless gene replacement method that combines a temperature-sensitive plasmid pKSV7 with a counter-selectable marker, the <I>upp</I> gene encoding uracil phosphoribosyltransferase (UPRTase) for the <I>Bacillus amyloliquefaciens</I> LL3. This method allows us to adapt a two-step plasmid integration and excision strategy to perform markerless deletion of genes.</P></DIV></DIV></DIV><DIV class="container-fluid"><DIV class="row"><DIV class="col-xs-12"><DIV class="panel-group" id="first_one"><DIV class="panel panel-default" style="margin-right: 10%;margin-left: 10%;"><DIV class="panel-heading panelheadingcursor" data-toggle="collapse" data-parent="#first_one" data-target="#collapse1" style="background-color: #B5CACF;"><H4 class="panel-title" style="text-align:center;"><A href="javascript:void(0)" style="font-size:28px;">  Click on Me</A></H4></DIV><DIV id="collapse1" class="panel-collapse collapse"><DIV class="panel-body"><H3 class="header_3">Introduction</H3><DIV class="container"><DIV class="row"><DIV class="col-xs-7"><P class="protocol_panel_content">The temperature-sensitive plasmid pKSV7 of <I>Bacillus</I> can replicate at 30°C normally, but the plasmid will be lost at 42°C. Combined the plasmid with a counter-selectable marker, the markerless gene replacement method can be constructed.</P><P class="protocol_panel_content">The <I>upp</I> gene encodes uracil phosphoribosyl transferase (UPRTase), which makes U transfer to UMP. Thus, cell can use extrinsic uracil via the following pathway 5-FU ➔ 5F(fluorin)-UMP ➔ 5-F-dUMP. Since the products of this pathway are toxic, cellular growth can be restrained.
If <I>upp</I> gene is knocked out, bacteria can resist 5-FU toxicity.</P></DIV><DIV class="col-xs-5 col-xs-push-1"><DIV class="col-xs-5 col-xs-push-3"><P style="font-size: 20px;">Figure 1</P></DIV></DIV></DIV></DIV><H3 class="header_3">Procedures</H3><DIV class="container"><DIV class="row"><DIV class="col-xs-12"><OL class="protocol_panel_content"><LI>Δ<I>upp</I><I>Bacillus amyloliquefaciens</I> LL3 was constructed, which can resist 5-fluorouracil.</LI><LI>Construction of the <I>upp</I> cassette: An 850 bp DNA fragment carrying the <I>upp</I> gene with its 5' regulatory region and its 3'; transcription terminator was generated by PCR from B. subtilis 168 genomic DNA using primers <I>upp</I> -F and <I>upp</I> -R. After digesting with KpnI and BamHI, the fragment was cloned in the KpnI –BamHI site of pKSV7. The resulting counter-selective plasmid was designated pKSU.</LI><LI>The target gene was selected. The upstream sequence A and downstream sequence B of target gene were combined by over-lapping PCR and ligated into plasmid pKSU.</LI><LI>Plasmid pKSU is a temperature sensitive plasmid, which replicates at 30°C and gets expelled at 42°C, besides, contains an <I>upp</I> expression cassette.</LI><LI>The recombinant plasmid was transformed into the target strain and the resulting transformants were cultured at 42°C with chloramphenicol to select single-crossover colonies.</LI><LI>The single-crossover strains were then cultured in medium with 5-fluorouracil to select double-crossover colonies.</LI></OL><P class="protocol_panel_content">Reference: Zhang W et al. Applied Microbiology &amp; Biotechnology, 2014.</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><H2 class="Protocols_header text-center">2. ClonExpress II One Step Cloning Kit</H2><DIV class="container"><DIV class="row"><DIV class="col-xs-12"><H3 class="header_3">Abstract</H3></DIV><DIV class="col-xs-12"><P class="Protocols_content">ClonExpress II One Step Cloning is a simple, fast, and high effcient cloning technology and enables directional insertion of any amplied DNA product into any linearized vector at any site.</P></DIV></DIV></DIV><DIV class="container-fluid"><DIV class="row"><DIV class="col-xs-12"><DIV class="panel-group" id="second_two"><DIV class="panel panel-default" style="margin-right: 10%;margin-left: 10%;"><DIV class="panel-heading panelheadingcursor" data-toggle="collapse" data-parent="#second_two" data-target="#collapse2" style="background-color: #B5CACF;"><H4 class="panel-title" style="text-align:center;"><A href="javascript:void(0)" style="font-size:28px;">  Click on Me</A></H4></DIV><DIV id="collapse2" class="panel-collapse collapse"><DIV class="panel-body"><H3 class="header_3">Introduction</H3><DIV class="container"><DIV class="row"><DIV class="col-xs-12"><P class="protocol_panel_content">The ClonExpress II is a novel cloning Kit, independent of DNA ligase, significantly reducing the vself-ligated colonies and bringing a true positive rate &gt; 95%. The enhanced Exnase II and highly optimized buffer significantly improve the recombination efficiency and the tolerance to impurities. Both the linearized vector and the PCR products of insert can be directly used for recombination without purification, significantly simplifying the procedures.</P></DIV></DIV></DIV><H3 class="header_3">Steps</H3><DIV class="container"><DIV class="row"><DIV class="col-xs-12"><DL><DT class="dt_header">1. Preparation for linearized cloning vectors</DT><DD><P class="protocol_panel_content">Select an appropriate cloning site on the vector that will be linearized. It is recommended to select the cloning site from regions with no repetitive sequence and the GC content of the certain region (within 20 bp up and downstream of the site) stays between 40% and 60%.</P><P class="protocol_panel_content">The linearized vector can be obtained by digesting the circular vector with restriction enzymes or by reverse PCR. Double digestion is recommended because it brings complete linearization and low false positive rate. If single digestion is adopted, a longer digestion time is necessary to reduce intact plasmid residues and decrease the false positive rate. When using reverse PCR amplification to obtain linearized vector, it is highly recommended to use a high-fidelity DNA polymerase.</P></DD><DT class="dt_header">2. Acquisition of Inserts</DT><DD><P class="protocol_panel_content">The principle for the design of ClonExpress II primers is adding homologous sequences of linearized vector (15 bp∼ 20 bp) to the 5' -ends of both Forward (F) and Reverse (R) primers. This is to make the ends of amplified inserts and linearized vectors identical to each other (15 bp∼20 bp, excludes restriction enzyme cutting sites).</P><P class="protocol_panel_content">Firstly, design the forward and the reverse primer of insertion according to Fig. 2.</P><P class="protocol_panel_content">Forward primer of insert: 5' - homologous sequence of vector-upstream end + restriction enzyme cutting site (optional) + gene specific forward amplification sequence of insert -3'</P></DD><DD><P class="protocol_panel_content">Reverse primer of insert: 3' - gene specific reverse amplification sequence of insert + restriction enzyme cutting site (optional) + homologous sequence of vector-downstream end -5'</P><P class="protocol_panel_content">Notice: If the primer length exceeds 40 bp, PAGE purification of synthetized primers is recommended, which will benefit the recombination efficiency. When calculating the T<SUB>m</SUB> of primers, the homologous sequence of vector ends should be excluded and only gene specific amplification sequence should be counted.</P><P class="protocol_panel_content">Secondly, Inserts can be amplified by any PCR polymerase (i.e. conventional Taq DNA polymerase or high-fidelity DNA polymerase). It will not interfere with the recombination efficiency whether there is A-tail in the PCR products or not. To prevent possible mutations introduced during PCR, amplification with a high-fidelity polymerase are highly recommended.</P></DD><DT class="dt_header">3. Recombination</DT><DD><P class="protocol_panel_content">Firstly, calculation of the amount of vectors and insert. The optimal amount of vector for the recombination with ClonExpress II is 0.03 pmol, while the optimal amount of insert is 0.06 pmol (molar ratio of vector to insertion is 1:2), as roughly calculated as follows:</P><P class="protocol_panel_content">The optimal mass of vector = [0.02 × number of base pairs] ng (0.03 pmol)</P><P class="protocol_panel_content">The optimal mass of insert = [0.04 × number of base pairs] ng (0.06 pmol)</P><P class="protocol_panel_content">Secondly, set up the following reaction on ice. Spin briefly to bring the sample to the bottom before reacting. See Table 1.</P><DIV class="container"><DIV class="row"><DIV class="col-xs-12"><H5 class="text-center" style="font-size:20px;margin-top: 10px;">Table 1</H5></DIV></DIV></DIV></DD><DT class="dt_header">4. Transformation and plating</DT><DD><P class="protocol_panel_content">Add the entire recombination products to 200 µL of competent cells; flip the tube several times to mix it thoroughly and place the tube on ice for 30 min. Heat-shock the tube for 45∼90 sec at 42 °C and then place the tube on ice for 2 min. Add 900 µL of SOC or LB medium to competent cells and leave the tube in 37 °C water bath for 10 min to let the competent cells fully recovered. Then, shake the tube at 37 °C for 45 min to culture the bacteria. Take 100 of culture and plate evenly on agar plate which contains appropriate selection antibiotic. Place the plate at 37 °C overnight to culture.</P></DD><DT class="dt_header">5. Selection of positive colony</DT><DD><P class="protocol_panel_content">Colony PCR is the most convenient selection method. Pick a single colony with tips to 20∼50 µL of LB medium, mix thoroughly and take 1 µL as PCR template. To avoid false positive PCR, we recommend at least one sequencing primer of the cloning vector should be used. Inoculate the remaining medium of positive clones into fresh LB medium and culture overnight. Then, extract the plasmids for subsequent authentication.</P></DD></DL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><H2 class="Protocols_header text-center">3. ClonExpress MultiS One Step Cloning Kit</H2><DIV class="container"><DIV class="row"><DIV class="col-xs-12"><H3 class="header_3">Abstract</H3></DIV><DIV class="col-xs-12"><P class="Protocols_content">ClonExpress One Step technology is a simple, fast and highly efficient cloning kit which is based on homologous recombination technology. It allows to directly clone any amplified product(s) to any linearized vector, at any site.</P></DIV></DIV></DIV><DIV class="container-fluid"><DIV class="row"><DIV class="col-xs-12"><DIV class="panel-group" id="third_three"><DIV class="panel panel-default" style="margin-right: 10%;margin-left: 10%;"><DIV class="panel-heading panelheadingcursor" data-toggle="collapse" data-parent="#third_three" data-target="#collapse3" style="background-color: #B5CACF;"><H4 class="panel-title" style="text-align:center;"><A href="javascript:void(0)" style="font-size:28px;">  Click on Me</A></H4></DIV><DIV id="collapse3" class="panel-collapse collapse"><DIV class="panel-body"><H3 class="header_3">Introduction</H3><DIV class="container"><DIV class="row"><DIV class="col-xs-12"><P class="protocol_panel_content">ClonExpress MultiS One Step Cloning kit is a new version cloning kit based on ClonExpress One Step Cloning technology. Exnase MultiS and reaction buffer supplied in this kit are especially optimized for multi-insertion seamless cloning (MultiS for short). With the help of this kit, sequential assembly of up to five insertions can be realized in a single reaction. Additionally, Exnase MultiS is also compatible with the endonuclease digesting reaction and the PCR reaction. Thus, the digesting products or PCR products can be directly applied in recombination reaction without purification, which greatly simplifies the experimental procedures.</P><P class="protocol_panel_content">Firstly, the expression vector will be linearized at the cloning site of choice. A small sequence (15∼20 bp) overlapped with the end of the cloning site will be added onto the insert through a PCR step. After the inserts and the linearized vector are mixed in the presence of Exnase for only 30 min, the cloning DNA products can be directly subjected to <I>E.coli.</I> transformation with true positive rate over 95%.</P></DIV></DIV></DIV><H3 class="header_3">Steps</H3><DIV class="container"><DIV class="row"><DIV class="col-xs-12"><DL><DT class="dt_header">1. Preparation for linearized cloning vectors</DT><DD><P class="protocol_panel_content">Select appropriate cloning sites and linearize the cloning vector. GC content of 20 bp regions at both ends of linearized cloning vector has great impacts on the recombination efficiency. The maximum recombination efficiency can be realized when the GC content of these regions is within 40% ∼60%. Thus, it’s better to avoid regions with sequence repeats and select regions containing even GC content.</P><P class="protocol_panel_content">The cloning vectors can be linearized by restriction digesting with endonuclease or by reverse PCR amplification.</P></DD><DT class="dt_header">2. Design of PCR primers of the insertions</DT><DD><P class="protocol_panel_content">The principle for the design of ClonExpress MultiS primers is: introduce homologous sequences (15 bp∼20 bp) into 5' end of primers, aiming to making the ends of amplified insertions and linearized cloning vector identical to the ends of their neighbours which is required for recombination reaction. Taking sequential assembly of three insertions (assembly order from 5' to 3' is as follows: insertion 1, insertion 2, insertion 3) to pUC18 cloning vector as example, design the primers as below:</P><P class="protocol_panel_content">Firstly, design the forward primer of insertion 1 and the reverse primer of insertion 3 (two insertions next to cloning vector) according to Figure 1.</P><DIV class="container"><DIV class="row"><DIV class="col-xs-12"><DIV class="col-xs-12"><P class="text-center" style="font-size: 20px;">Fig. 3: Design of the forward primer of insertion 1 and the reverse primer of insertion 3</P></DIV></DIV></DIV></DIV><P class="protocol_panel_content">Notice: If the primer length exceeds 40 bp, PAGE purification of synthetized primers is recommended, which will benefit the recombination efficiency. When calculating the T<SUB>m</SUB> of primers, the homologous sequence of vector ends should be excluded and only gene specific amplification sequence should be counted.</P><P class="protocol_panel_content">Secondly, design the reverse primer of insertion 1 and the forward primer of insertion 2. Homologous sequence used for inter-recombination between insertions can be fully added to either the reverse primer of insertion 1or the forward primer of insertion 2, and also can be partially added to both of them. Taking the addition homologous sequence to the reverse primer of insertion 1 as an example, design the primer according to Figure 4.</P><DIV class="container"><DIV class="row"><DIV class="col-xs-12"><DIV class="col-xs-12"><P class="text-center" style="font-size: 20px;">Figure 4: Design of the reverse primer of insertion 1 and the forward primer of insertion 2</P></DIV></DIV></DIV></DIV><P class="protocol_panel_content">Notice: If the primer length exceeds 40 bp, PAGE purification of synthetized primers is recommended, which will benefit the recombination efficiency. When calculating the T<SUB>m</SUB> of primers, the homologous sequence of vector end should be excluded and only gene specific amplification sequence should be counted.</P><P class="protocol_panel_content">Lastly, design the reverse primer of insertion 2 and the forward primer of insertion 3. Design principles are similar to that of the reverse primer of insertion 1 and forward primer of insertion 2 respectively (see Fig. 4).</P></DD><DT class="dt_header">3. PCR amplification of insertions</DT><DD><P class="protocol_panel_content">Insertions can be amplified by any polymerase (Taq DNA polymerase or other high-fidelity polymerases). It will not interfere with the recombination efficiency whether there is A-tail in the PCR products or not, which will be removed during recombination and missing in the final construct.</P><P class="protocol_panel_content">Take a small amount of products and run agrose electrophoresis after PCR to confirm the yields and specificity of amplification. Exnase MultiS is compatible with most PCR reactions. As a result, PCR products can be directly applied to recombination reaction without further purification if the PCR templates are not circular plasmids which share the same antibiotic resistance with the cloning vector.</P></DD><DT class="dt_header">4. Recombination reaction</DT><DD><P class="protocol_panel_content">Set up the following reaction on ice. Spin briefly to bring the sample to the bottom before reacting.  See Table 2.</P><DIV class="container"><DIV class="row"><DIV class="col-xs-12"><H5 class="text-center" style="font-size:20px;margin-top: 10px;">Table 2</H5></DIV></DIV></DIV><P class="protocol_panel_content">The recommended amount of DNA for recombination reaction is 0.03 pmol per DNA fragment (including the cloning vector and insertions). Their corresponding mass can be roughly calculated according the following formula:</P><P class="protocol_panel_content">The mass of each fragment required = [0.02 × number of base pair] ng (0.03 pmol)</P></DD><DT class="dt_header">Notice:</DT><DD><OL style="font-size: 20px;"><LI>The mass of linearized cloning vector used should be between 50∼200 ng. Use 50 or 200 ng if the calculated mass is out of range.</LI><LI>The mass of insertions should be over 10 ng. Use 10 ng if the calculated mass is less.</LI><LI>When applied to recombination reaction without gel recovery, the total volume of unpurified DNA used should be less than 1/5 of that of recombination reaction, which is 4 µL.</LI></OL><P class="protocol_panel_content">After finishing setting up, gently pipette up and down several times with a pipettor to mix thoroughly and try to avoid the formation of bubbles. Incubate the reaction at 37°C for 30 min and immediately place it on ice for 5 min. Recombination product is now ready for transformation, or otherwise it can be stored at -20 °C before transformation.</P><P class="protocol_panel_content">Both “5. Transformation and plating.” and “6. Selection of positive colony.” are the same with that in ClonExpress II One Step Cloning Kit.</P></DD></DL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><H2 class="Protocols_header text-center">4. Glutamate Content Assay Kit</H2><DIV class="container"><DIV class="row"><DIV class="col-xs-12"><H3 class="header_3">Abstract</H3></DIV><DIV class="col-xs-12"><P class="Protocols_content">This is a simple, fast glutamate content assay kit. It allows to directly determination of intracellular glutamate concentration.</P></DIV></DIV></DIV><DIV class="container-fluid"><DIV class="row"><DIV class="col-xs-12"><DIV class="panel-group" id="fourth_four"><DIV class="panel panel-default" style="margin-right: 10%;margin-left: 10%;"><DIV class="panel-heading panelheadingcursor" data-toggle="collapse" data-parent="#fourth_four" data-target="#collapse4" style="background-color: #B5CACF;"><H4 class="panel-title" style="text-align:center;"><A href="javascript:void(0)" style="font-size:28px;">  Click on Me</A></H4></DIV><DIV id="collapse4" class="panel-collapse collapse"><DIV class="panel-body"><H3 class="header_3">Introduction</H3><DIV class="container"><DIV class="row"><DIV class="col-xs-12"><P class="protocol_panel_content">Glutamate content assay kit is based on the principle listed below.</P></DIV></DIV></DIV><H3 class="header_3">Steps</H3><DIV class="container"><DIV class="row"><DIV class="col-xs-12"><OL style="font-size: 20px;"><LI><STRONG>10 mmol/L standard stock solution preparation:</STRONG> Heat the dilution in a boiling water bath; dilute and dissolve a standard powder with a hot diluent to a volume of 5 mL.<STRONG>200 µmol/L standard application fluid preparation:</STRONG> Take 0.1 mL of stock solution and dilute to 5 mL with diluent (without heating).</LI><LI><STRONG>Sample Processing:</STRONG> Take 0.2 mL of bacteria solution and add 0.6 mL of reagent 1 (in a ratio of 1:3); mix thoroughly; 3500 rpm, centrifuge for 10 minutes; take the supernatant 0.5mL to be tested.</LI><LI><STRONG>Working fluid preparation:</STRONG> Formulate according to Reagent 2: Reagent 3: Reagent 4: ddH<SUB>2</SUB>O= 1.0:0.1:0.01:0.39, use it right after it was ready.</LI><LI><STRONG>Operation table:</STRONG>
        			    Mix, heated in a 37 °C water bath for 40 minutes; 340nm, 1 cm light path, zeroing with ddH<SUB>2</SUB>O, measure the absorbance of each tube as A<SUB>2</SUB>.
        			    </LI><LI><STRONG>Calculation:</STRONG></LI></OL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><H2 class="Protocols_header text-center">5. Fermentation and production treatment</H2><DIV class="container"><DIV class="row"><DIV class="col-xs-12"><H3 class="header_3">Abstract</H3></DIV><DIV class="col-xs-12"><P class="Protocols_content"><I>Bacillus amyloliquefaciens</I> LL3 is a glutamate independent γ-PGA production strain.</P></DIV></DIV></DIV><DIV class="container-fluid"><DIV class="row"><DIV class="col-xs-12"><DIV class="panel-group" id="fifth_five"><DIV class="panel panel-default" style="margin-right: 10%;margin-left: 10%;"><DIV class="panel-heading panelheadingcursor" data-toggle="collapse" data-parent="#fifth_five" data-target="#collapse5" style="background-color: #B5CACF;"><H4 class="panel-title" style="text-align:center;"><A href="javascript:void(0)" style="font-size:28px;">  Click on Me</A></H4></DIV><DIV id="collapse5" class="panel-collapse collapse"><DIV class="panel-body"><H3 class="header_3">Steps</H3><DIV class="container"><DIV class="row"><DIV class="col-xs-12"><P class="protocol_panel_content">For growth experiments, <I>Bacillus amyloliquefaciens</I> LL3 and its derivatives were first grown overnight in test tubes containing LB liquid medium and then inoculated into fermentation medium to an optical density at 600 nm (OD<SUB>600</SUB>) of approximately 0.05–0.1, and growth was monitored at OD<SUB>600</SUB>.</P><P class="protocol_panel_content">For fermentation experiments, <I>Bacillus amyloliquefaciens</I> strains were first grown for 16 h in LB at 37 °C in a shaking incubator and then a 1%–2% (v/v) inoculum was added to a 500 mL flask containing 100 mL of fermentation medium at 37 °C with shaking at 180 rpm for 48 h.</P><P class="protocol_panel_content">The fermentation broth was measured at OD<SUB>600</SUB> first, and centrifuged at 8,000 × g (4 °C) for 20 min. γ-PGA was extracted using an ethanol precipitation method: Fourfold volume of cold anhydrous ethanol was added to the supernatant containing PGA followed by incubation at 4 °C overnight. The precipitate was centrifuged at 4,000 × g (4 °C) and then dialyzed against distilled water and lyophilized to obtain γ-PGA.</P><P class="protocol_panel_content">Experiments were independently repeated at least three times.</P><P class="protocol_panel_content">Reference:</P><P class="protocol_panel_content">Zhang, W., <I>et al.</I> &quot;A markerless gene replacement method for <I>Bacillus amyloliquefaciens</I> LL3 and its use in genome reduction and improvement of poly-γ-glutamic acid production.&quot; Applied Microbiology &amp; Biotechnology 98.21(2014):8963-8973.</P><P class="protocol_panel_content">Zhang, W., <I>et al.</I> &quot;Chromosome integration of the Vitreoscilla hemoglobin gene (<I>vgb</I>) mediated by temperature-sensitive plasmid enhances γ-PGA production in <I>Bacillus amyloliquefaciens</I>.&quot; Fems Microbiology Letters 343.2(2013):127-134.</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></MAIN><FOOTER><DIV class="container-fluid myFooter"><DIV class="row"><DIV class="col-xs-4 footer-w3ls"><H3 style="color: white; font-size: 30px;">Contacts</H3><H3 style="color: white; font-size: 30px;">Address</H3><ADDRESS style="color: white; font-size: 17px;">
                    Nankai University
                    No.94 Weijin Road, Nankai District
                    Tianjin, P.R.China 300071</ADDRESS></DIV><DIV class="col-xs-12"><P class="Copyright" style="text-align: center; color: white;">Copyright © Team NKU_CHINA 2018</P></DIV></DIV></DIV></FOOTER><A href="#0" class="cd-top">Top</A></DIV></DIV></DIV></DIV></DIV></BODY></HTML>