<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_NKU_CHINA_Experiments skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:NKU CHINA/Experiments</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container-fluid"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#myNavbar" aria-expanded="false"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse" id="myNavbar"><UL class="nav navbar-nav navbar-right"><LI><A href="https://2018.igem.org/Team:NKU_CHINA" style="margin-top:4px;">Home</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">Team
                  </A></LI><LI><A href="https://2018.igem.org/Team:NKU_CHINA/Team">Team</A></LI><LI><A href="https://2018.igem.org/Team:NKU_CHINA/Collaborations">Collaborations</A></LI></UL><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">Project
                  </A></LI><LI><A href="https://2018.igem.org/Team:NKU_CHINA/Description">Description</A></LI><LI><A href="https://2018.igem.org/Team:NKU_CHINA/Experiments">Experiments</A></LI><LI><A href="https://2018.igem.org/Team:NKU_CHINA/Model">Model</A></LI><LI><A href="https://2018.igem.org/Team:NKU_CHINA/Proof">Proof of Concept</A></LI><LI><A href="https://2018.igem.org/Team:NKU_CHINA/Demonstrate">Demonstrate</A></LI><LI><A href="https://2018.igem.org/Team:NKU_CHINA/Human_Practices" data-tooltip="true" title="Human Practices" style="margin-top:4px;">HP</A></LI><LI><A href="https://2018.igem.org/Team:NKU_CHINA/Safety" style="margin-top:4px;">Safety</A></LI><LI><A href="https://2018.igem.org/Team:NKU_CHINA/Attributions" style="margin-top:4px;">Attributions</A></LI><LI class="dropdown"><A class="data-toggle" data-toggle="dropdown" href="#">Parts
                   </A></LI><LI><A href="https://2018.igem.org/Team:NKU_CHINA/Basic_Part">Basic Parts</A></LI><LI><A href="https://2018.igem.org/Team:NKU_CHINA/Composite_Part">Composite Parts</A></LI><LI><A href="https://2018.igem.org/Team:NKU_CHINA/Improve">Improve</A></LI><LI><A href="https://2018.igem.org/Team:NKU_CHINA/InterLab" style="margin-top:4px;">InterLab</A></LI><LI class="dropdown"><A class="data-toggle" data-toggle="dropdown" href="#">Notebook
                  </A></LI><LI><A href="https://2018.igem.org/Team:NKU_CHINA/Notebook">Lab Notes</A></LI><LI><A href="https://2018.igem.org/Team:NKU_CHINA/Protocols">Protocols</A></LI></DIV></DIV><DIV id="loading_p"><P style="color: white;font-family:myTitle3;font-size:30px;position:relative;top:47%;left:46%">Loading...</P></DIV><MAIN><H1 style="font-family: myTitle;color: white;font-size: 80px;text-align: center;margin-top:20px;margin-bottom:0;">Experiments</H1><DIV class="container-fluid"><DIV class="row"><DIV class="col-xs-12"><H2 class="ex_h2" style="color: #FFD600;font-size: 40px;text-align: center;">Part 0 Construction of High-glutamate-yield <I>B. amyloliquefaciens</I> LL3</H2></DIV></DIV></DIV><DIV class="container"><DIV class="row"><DIV class="col-xs-12"><P class="ex_content">In order to get a <I>Bacillus amyloliquefaciens</I> LL3 strain for a higher glutamate yield, we edited the genome by constructing promoters in tandem to overexpress glutamate related genes, <I>icd</I> and <I>gltAB</I>, which are responsible for 2-ketoglutarate and glutamate, separately. Due to the difficulties of genome manipulation, we only succeeded in inserting P<SUB><I>c2up</I></SUB> ahead of <I>icd</I> and get a high yield strain.</P></DIV></DIV></DIV><DIV class="container-fluid"><DIV class="row"><DIV class="col-xs-12"><H2 class="ex_h2" style="color: #FFD600;font-size: 40px;text-align: center;">Part 1 Verification of P<SUB><I>gltAB</I></SUB> Responding with Glutamate and GltC</H2></DIV></DIV></DIV><DIV class="container"><DIV class="row"><DIV class="col-xs-12"><P class="ex_content">In <I>Bacillus amyloliquefaciens</I> LL3, there is a <I>glt</I> operon which is responsible for the intracellular glutamate synthesis. Since we need to use the regulatory role of glutamate and GltC to P<SUB><I>gltAB</I></SUB>, we need to verify the effect of promoter P<SUB><I>gltAB</I></SUB> under different glutamate concentrations. So we constructed pHT01-P<SUB><I>gltAB</I></SUB>-GFP and pHT01-P<SUB>43</SUB>-GFP 2 plasmids and transformed them into <I>Bacillus amyloliquefaciens</I> LL3 <I>icd</I> strain. We fermented under different glutamate concentrations and tested the fluorescence intensity(FI) of GFP every 6 hours in M9 medium. For the strain transformed with pHT01-P<SUB><I>gltAB</I></SUB>-GFP, as the glutamate concentration goes up, the FI of GFP first went up and came down. On the other hand, for the strain transformed with pHT01-P<SUB>43</SUB>-GFP, FI is uncorrelated to the glutamate concentration.</P></DIV></DIV></DIV><DIV class="container-fluid"><DIV class="row"><DIV class="col-xs-12"><H2 class="ex_h2" style="color: #FFD600;font-size: 40px;text-align: center;">Part 2 Verification of the Repression Effect of LacI to P<SUB><I>grac</I></SUB></H2></DIV></DIV></DIV><DIV class="container"><DIV class="row"><DIV class="col-xs-12"><P class="ex_content">To make our circuit work, we need to test the inhibition effect to P<SUB><I>grac</I></SUB> which can respond to LacI protein at different <I>lacI</I> expression levels. So we expected to construct vectors pHT01-LacI-P<SUB>different</SUB>-P<SUB><I>grac</I></SUB>-TetA.</P></DIV></DIV></DIV><DIV class="container-fluid"><DIV class="row"><DIV class="col-xs-12"><H2 class="ex_h2" style="color: #FFD600;font-size: 40px;text-align: center;">Part 3 Constrction of the Whole <SPAN data-tooltip="true" title="Population Quality Control">PopQC</SPAN> System and Functional Test</H2></DIV></DIV></DIV><DIV class="container"><DIV class="row"><DIV class="col-xs-12"><P class="ex_content">Our project is aiming to construct a <SPAN data-tooltip="true" title="Population Quality Control">PopQC</SPAN> system in a single plasmid. So we construct the vector pHT01-LacI-P<SUB><I>gltAB</I></SUB>-P<SUB><I>grac</I></SUB>-TetA and ferment to test the function of our system. To furthermore simplify the procedures of functional test, we constructed another vector pHT01-mCherry-LacI-P<SUB><I>gltAB</I></SUB>-P<SUB><I>grac</I></SUB>-TetA-GFP(int) to test the fluorescence intensity instead of mRNA. The results are roughly in accordance with our expectation.</P></DIV></DIV></DIV><DIV class="container-fluid"><DIV class="row"><DIV class="col-xs-12"><H2 class="ex_h2" style="color: #FFD600;font-size: 40px;text-align: center;">Parts Creation</H2></DIV></DIV></DIV><DIV class="container"><DIV class="row"><DIV class="col-xs-12"><P class="ex_content">We constructed 12 standard parts, 1. P<SUB><I>gltAB</I></SUB>-LacI-P<SUB><I>grac</I></SUB>-TetA (BBa_K2705006); 2. LacI(BBa_K2705001); 3. P<SUB><I>gltAB</I></SUB>(BBa_K2705000); 4. P<SUB><I>grac</I></SUB>(BBa_K2705002); 5. TetA(BBa_K2705003); 6. GFP(BBa_K2705004) 7. P<SUB><I>gltAB</I></SUB>-GFP(BBa_K2705005); 8. PliaG&amp;43(BBa_K2705008); 9. PliaG&amp;43-1(BBa_K2705009); 10. PliaG&amp;43-2(BBa_K2705010); 11. PliaG&amp;43-3(BBa_K2705011); 12. TetA(optimized)(BBa_K2705003). The promoter P<SUB><I>gltAB</I></SUB>, P<SUB><I>grac</I></SUB>, <I>lacI</I>, <I>tetA</I>, and <I>gfp</I> genes are cloned by PCR. Then both these segments and pSB1C3 vector were treated with restriction enzymes. After that, these parts were separately ligated to linear pSB1C3 by ligase or recombinase, and the products were separately transformed into <I>E.coli</I> JM110. Verification PCR was performed to select the positive clones. The positive strains were chosen to be cultured overnight and plasmids were extracted. After restriction enzyme digestion verification, the positive clones were sequenced.</P></DIV></DIV></DIV><DIV class="container-fluid"><DIV class="row"><DIV class="col-xs-12"><H2 class="ex_h2" style="color: #FFD600;font-size: 40px;text-align: center;">Parts Improvements</H2></DIV></DIV></DIV><DIV class="container"><DIV class="row"><DIV class="col-xs-12"><P class="ex_content">We improved 2 parts, BBa_K823000 and BBa_J31006. For BBa_K823000 the weak, constitutive promoter from <I>Bacillus subtilis</I>, we chose to mimic the structure of P<SUB><I>43</I></SUB>, the relatively strong constitutive promoter which can be recognized by both sigma factor 55 (the major sigma factor A) and sigma factor 37 (the lag phase sigma factor B). After designing the new sequences of promoters, we send them to the company GenScript Biotech Corp. to have them synthesized. The P<SUB><I>liaG&amp;43</I></SUB>-0, P<SUB><I>liaG&amp;43</I></SUB>-1, P<SUB><I>liaG&amp;43</I></SUB>-2, P<SUB><I>liaG&amp;43</I></SUB>-3 are cloned by PCR. Then both these segments and pHT01-GFP vector were treated with restriction enzymes. After that, these parts were separately ligated to linear pHT01-GFP by recombinase, and the products were separately transformed into <I>E.coli</I> JM110. Verification PCR was performed to select the positive clones. The positive strains were chosen to be cultured overnight and plasmids were extracted. After restriction enzyme digestion verification, the positive clones were sequenced. Object plasmids are transformed into <I>Bacillus subtilis</I> 168 by electroporation transformation and verification PCR was performed. After that, we use the Microplate Reader to analyze the FI of the cultures after 2 hours. For BBa_J31006, we send the sequence to the company GenScript Biotech Corp. and get a codon optimized one for <I>Bacillus subtilis</I>. And then we constructed pHT01-P<SUB>43</SUB>-TetA plasmid and transformed into <I>Bacillus subtilis</I> LL3 to test its function by measuring the MIC.</P></DIV></DIV></DIV></MAIN><FOOTER><DIV class="container-fluid myFooter"><DIV class="row"><DIV class="col-xs-4 footer-w3ls"><H3 style="color: white; font-size: 30px;">Contacts</H3><H3 style="color: white; font-size: 30px;">Address</H3><ADDRESS style="color: white; font-size: 17px;">
                    Nankai University
                    No.94 Weijin Road, Nankai District
                    Tianjin, P.R.China 300071</ADDRESS></DIV><DIV class="col-xs-12"><P class="Copyright" style="text-align: center; color: white;">Copyright © Team NKU_CHINA 2018</P></DIV></DIV></DIV></FOOTER><A href="#0" class="cd-top">Top</A></DIV></DIV></DIV></DIV></DIV></BODY></HTML>