<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_ZJU-China_Design_Toxinmanufacture skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:ZJU-China/Design/Toxinmanufacture</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="gl-outer-wrapper" class="gl-zh cf" style=""><DIV id="gl-header" class="cf gl-header-fixed" style="max-width:1440px"><DIV id="gl-header-bottom" class="cf"><DIV id="gl-menu" class="cf"><DIV id="gl-menu-one" class="gl-has-submenu"><DIV id="gl-submenu-one" class="gl-submenu"><DIV class="gl-col-footer" style="border-top:0px solid #aaa;"><A href="https://2015.igem.org/Team:ZJU-China">Home</A></DIV></DIV></DIV><DIV id="gl-menu-two" class="gl-has-submenu"><DIV id="gl-submenu-three" class="gl-submenu"><DIV class="gl-col-wrapper"><DIV class="gl-col gl-col1"><A href="https://2015.igem.org/Team:ZJU-China/Parts" class="gl-hover"><SPAN>Team Part</SPAN><SPAN>Team Part</SPAN></A></DIV><DIV class="gl-col gl-col2"><A href="https://2015.igem.org/Team:ZJU-China/Basic_Part" class="gl-hover"><SPAN>Basic Part</SPAN><SPAN>Basic Part</SPAN></A></DIV><DIV class="gl-col gl-col3"><A href="https://2015.igem.org/Team:ZJU-China/Description" class="gl-hover"><SPAN>Improved  Part</SPAN><SPAN>Improved  Part</SPAN></A></DIV><DIV class="gl-col gl-col4"><A href="https://2015.igem.org/Team:ZJU-China/Composite_Part" class="gl-hover"><SPAN>Composite Part</SPAN><SPAN>Composite Part</SPAN></A></DIV></DIV><DIV class="gl-col-footer"><A href="https://2015.igem.org/Team:ZJU-China/Parts">Parts</A></DIV></DIV></DIV><DIV id="gl-menu-three" class="gl-has-submenu"><DIV id="gl-submenu-three" class="gl-submenu"><DIV class="gl-col-wrapper"><DIV class="gl-col gl-col1" style="width: 33.3%;"><A href="https://2015.igem.org/Team:ZJU-China/Project" class="gl-hover"><SPAN>Overview</SPAN><SPAN>Quick view for our project</SPAN></A><A href="https://2015.igem.org/Team:ZJU-China/Project/Termite_Issue" class="gl-hover"><SPAN>Termite Issue</SPAN><SPAN>The Termite Issue</SPAN></A></DIV><DIV class="gl-col gl-col2" style="width: 33.3%;"><DIV><SPAN><P>Design</P></SPAN><UL><LI><A href="https://2015.igem.org/Team:ZJU-China/Design/Toxinmanufacture">Toxins Manufacture</A></LI><LI><A href="https://2015.igem.org/Team:ZJU-China/Design/CNC" style="padding-left:32px">Bacteria Carrier</A></LI><LI><A href="https://2015.igem.org/Team:ZJU-China/Design/Termites" style="padding-left:32px">Termites</A></LI></UL></DIV></DIV><DIV class="gl-col gl-col3" style="width: 33.3%;"><A href="https://2015.igem.org/Team:ZJU-China/Project/Protocol" class="gl-hover"><SPAN>Protocol</SPAN><SPAN>Quick view for Protocol</SPAN></A></DIV></DIV><DIV class="gl-col-footer"><A href="https://2015.igem.org/Team:ZJU-China/Project">Project</A></DIV></DIV></DIV><DIV id="gl-menu-four" class="gl-has-submenu"><DIV id="gl-submenu-three" class="gl-submenu"><DIV class="gl-col-wrapper"><DIV class="gl-col gl-col1"><A href="https://2015.igem.org/Team:ZJU-China/Practices" class="gl-hover"><SPAN>Overview</SPAN><SPAN>Quick view for Outreach</SPAN></A><DIV><SPAN><P>Society</P></SPAN><UL><LI><A href="https://2015.igem.org/Team:ZJU-China/Practices/Society/Survey">Survey</A></LI><LI><A href="https://2015.igem.org/Team:ZJU-China/Practices/Society/Consultation" style="padding-left:32px">Consultation</A></LI><LI><A href="https://2015.igem.org/Team:ZJU-China/Practices/Society/Syn-bio_Debate" style="padding-left:32px">Syn-bio Debate</A></LI></UL></DIV></DIV><DIV class="gl-col gl-col2"><DIV><SPAN><P>Application</P></SPAN><UL><LI><A href="https://2015.igem.org/Team:ZJU-China/Design">Device</A></LI><LI><A href="https://2015.igem.org/Team:ZJU-China/Entrepreneurship" style="padding-left:32px">Promotion</A></LI><LI><A href="https://2015.igem.org/Team:ZJU-China/Practices/Application/Safety" style="padding-left:32px">Safety</A></LI><LI><A href="https://2015.igem.org/Team:ZJU-China/Practices/Application/Future" style="padding-left:32px">Future</A></LI></UL></DIV></DIV><DIV class="gl-col gl-col3"><DIV><SPAN><P>Public Promotion</P></SPAN><UL><LI><A href="https://2015.igem.org/Team:ZJU-China/Practices/Public_Promotion/Poly-cards">Poly-cards</A></LI><LI><A href="https://2015.igem.org/Team:ZJU-China/Practices/Public_Promotion/Syn-bio_Share" style="padding-left:32px">Syn-bio Share</A></LI><LI><A href="https://2015.igem.org/Team:ZJU-China/Practices/Public_Promotion/Community_Lab" style="padding-left:32px">Community Lab </A></LI><LI><A href="https://2015.igem.org/Team:ZJU-China/Practices/Public_Promotion/Seminar" style="padding-left:32px">Seminar</A></LI></UL></DIV></DIV><DIV class="gl-col gl-col4" style="padding-right:80px"><DIV><SPAN><P>Collaboration</P></SPAN><UL><LI><A href="https://2015.igem.org/Team:ZJU-China/Practices/Cooperate/Senergene/first">SynenergeneⅠ</A></LI><LI><A href="https://2015.igem.org/Team:ZJU-China/Practices/Cooperate/Senergene/second" style="padding-left:32px">SynenergeneⅡ</A></LI><LI><A href="https://2015.igem.org/Team:ZJU-China/Practices/Cooperate/Senergene/third" style="padding-left:32px">Synenergene Ⅲ</A></LI><LI><A href="https://2015.igem.org/Team:ZJU-China/Collaborations" style="padding-left:32px">Other Teams</A></LI></UL></DIV></DIV></DIV><DIV class="gl-col-footer"><A href="https://2015.igem.org/Team:ZJU-China/Practices">Outreach</A></DIV></DIV></DIV><DIV id="gl-menu-five" class="gl-has-submenu"><DIV id="gl-submenu-three" class="gl-submenu"><DIV class="gl-col-wrapper"><DIV class="gl-col gl-col1"><A href="https://2015.igem.org/Team:ZJU-China/Team" class="gl-hover"><SPAN>Members</SPAN><SPAN>Introduce our Members</SPAN></A></DIV><DIV class="gl-col gl-col2"><A href="https://2015.igem.org/Team:ZJU-China/Attributions" class="gl-hover"><SPAN>Attributions</SPAN><SPAN>Introduce the Attributions</SPAN></A></DIV><DIV class="gl-col gl-col3"><A href="https://2015.igem.org/Team:ZJU-China/Team/Acknowledgement" class="gl-hover"><SPAN>Acknowledgement</SPAN><SPAN>Introduce the Acknowledgement</SPAN></A></DIV><DIV class="gl-col gl-col4"><A href="https://2015.igem.org/Team:ZJU-China/Team/Sponsors" class="gl-hover"><SPAN>Sponsors</SPAN><SPAN>Introduce our Sponsors </SPAN></A></DIV></DIV><DIV class="gl-col-footer"><A href="https://2015.igem.org/Team:ZJU-China/Team">Team</A></DIV></DIV></DIV><DIV id="gl-menu-six" class="gl-has-submenu"><DIV id="gl-submenu-three" class="gl-submenu"><DIV class="gl-col-wrapper"><DIV class="gl-col gl-col1" style="border-right:2px solid #ddd"><A href="https://2015.igem.org/Team:ZJU-China/Modeling" class="gl-hover"><SPAN>Modeling Overview</SPAN><SPAN>Introduce our Members</SPAN></A></DIV><DIV class="gl-col gl-col2"><A href="https://2015.igem.org/Team:ZJU-China/Modeling/first" class="gl-hover"><SPAN>Modeling α</SPAN><SPAN>Termite Simulation</SPAN></A></DIV><DIV class="gl-col gl-col3"><A href="https://2015.igem.org/Team:ZJU-China/Modeling/second" class="gl-hover"><SPAN>Modeling β</SPAN><SPAN>Mass transfer model</SPAN></A></DIV><DIV class="gl-col gl-col4"><A href="https://2015.igem.org/Team:ZJU-China/Modeling/third" class="gl-hover"><SPAN>Modeling γ</SPAN><SPAN>Degradation model</SPAN></A></DIV></DIV><DIV class="gl-col-footer"><A href="https://2015.igem.org/Team:ZJU-China/Modeling">Modeling</A></DIV></DIV></DIV><DIV id="gl-menu-seven" class="gl-has-submenu"><DIV id="gl-submenu-seven" class="gl-submenu"><DIV class="gl-col-footer" style="border-top:0px solid #aaa;"><A href="https://2015.igem.org/Team:ZJU-China/Notebook">Notebook</A></DIV></DIV></DIV><DIV id="gl-menu-eight" class="gl-has-submenu"><DIV id="gl-submenu-three" class="gl-submenu"><DIV class="gl-col-wrapper"><DIV class="gl-col gl-col1"><A href="https://2015.igem.org/Team:ZJU-China/Results" class="gl-hover"><SPAN>Experiment Results</SPAN><SPAN>toxins manufacture</SPAN></A></DIV><DIV class="gl-col gl-col2"><A href="https://2015.igem.org/Team:ZJU-China/Results/second" class="gl-hover"><SPAN>Experiment Result</SPAN><SPAN>bacteria carrier</SPAN></A></DIV><DIV class="gl-col gl-col3"><A href="https://2015.igem.org/Team:ZJU-China/Results/third" class="gl-hover"><SPAN>Experiment Result</SPAN><SPAN>termites simulation</SPAN></A></DIV><DIV class="gl-col gl-col4"><A href="https://2015.igem.org/Team:ZJU-China/Results/social_result" class="gl-hover"><SPAN>Social Result</SPAN><SPAN>social results</SPAN></A></DIV></DIV><DIV class="gl-col-footer"><A href="https://2015.igem.org/Team:ZJU-China/Results">Results</A></DIV></DIV></DIV><DIV id="gl-menu-nine" class="gl-has-submenu"><DIV id="gl-submenu-nine" class="gl-submenu"><DIV class="gl-col-footer" style="border-top:0px solid #aaa;"><A href="https://2015.igem.org/Team:ZJU-China/Safety">Safety</A></DIV></DIV></DIV><DIV id="gl-menu-ten" class="gl-has-submenu" style="padding-right:20px"><DIV id="gl-submenu-three" class="gl-submenu"><DIV class="gl-col-wrapper"><DIV class="gl-col gl-col1"><A href="https://2015.igem.org/Team:ZJU-China/Design" class="gl-hover"><SPAN>Device</SPAN><SPAN>Introduce our Device and Instruction Manual</SPAN></A></DIV><DIV class="gl-col gl-col2"><A href="https://2015.igem.org/Team:ZJU-China/Entrepreneurship" class="gl-hover"><SPAN>Promotion</SPAN><SPAN>The Promotion</SPAN></A></DIV><DIV class="gl-col gl-col3"><A href="https://2015.igem.org/Team:ZJU-China/Practices/Application/Safety" class="gl-hover"><SPAN>Safety</SPAN><SPAN>The Safety</SPAN></A></DIV><DIV class="gl-col gl-col4"><A href="https://2015.igem.org/Team:ZJU-China/Practices/Application/Future" class="gl-hover"><SPAN>Future</SPAN><SPAN>Future</SPAN></A></DIV></DIV><DIV class="gl-col-footer"><A href="https://2015.igem.org/Team:ZJU-China/Design">Application</A></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV id="page-content-wrapper"><ARTICLE><H2 style="color:green;font-size:50px"> Toxin Manufacture </H2><H2 id="pos1"> Introduction </H2><P class="p1"> Biological pesticides can be divided into two types: small molecular compounds and biological macromolecules. On the one hand, small compounds are more prone to be absorbed by termites while more costly to produce. On the other hand, macromolecules are easier and cheaper to produce whereas sometimes not as effective as small molecules. Hence, to kill termites more efficiently and effectively, we choose both--We plan to overexpress avermectin in its host <I> Streptomyces avermitilis </I> and express four kinds of toxic protein in <I> Escherichia coli</I> BL21 (DE3)  . Then we embed the engineered <I> S. avermitilis </I> and <I> E.coli </I> with CNC carrier and feed termites with the CNC imbedded bacteria. For more information about CNC, please go to <A href="https://2015.igem.org/Team:ZJU-China/Design/CNC" title="about CNC"> the main page of CNCs </A> . </P><H2 id="pos2"> Avermectin manufacture </H2><P class="p1"> Judging that many toxic small compounds are harmful to human being, we choose avermectin, which is highly specific to insects and does little harm to human. For one thing, being a secondary metabolite produced by <I> Streptomyces avermitilis </I> , avermectin is encoded by an 80kb gene cluster, making it difficult to be engineered in other standardized strains, for instance, <I> Escherichia coli </I>. For another, the avermectin yield in wild type <I> S. avermitilis </I> strain is comparatively low. Nevertheless, we plan to engineer the wild <I> S. avermitilis </I> to improve the yield of avermectin, embed the engineered strain with CNCs and feed termites with CNC embedded <I> S. avermitilis </I> . </P><DIV id="show1"><H3> AVERMECTIN: EFFECTIVE AND BROAD-SPECTRUM PESTICIDE </H3><H3><P class="p1"> For years, people always adopt the organochlorine pesticides such as chlordane and mirex to achieve prevention and control of termites, but these organochlorine pesticides will produce pollution and potential harm to the environment. Avermectin is a new type of highly efficient biological pesticide, which has good control effect to termites and other pests, and no pollution to the environment. </P><DIV class="row"><DIV class="col-md-12" style="text-align:center"><DIV class="cpleft"><P class="kuvateksti"> Figure 1 Abstract process of self-assembly </P></DIV></DIV></DIV></H3><H3> HOST OF AVERMECTIN - <I> Streptomyces avermitilis </I></H3><P class="p1"><I> Streptomyces avermitilis </I> , a soil-dwelling gram-positive microorganism, is a rich source of numerous secondary metabolites. It's a kind of Actinomycetes with staghorn-like hypha (Figure 2). Now it has been industrialized to produce the commercially important antiparasitic agent avermectin(2). Early in 2003, the complete genome of <I> Streptomyces avermitilis </I> had been sequenced(3). </P><DIV class="row"><DIV class="col-md-12" style="text-align:center"><DIV class="cpleft"><P class="kuvateksti"> Figure 2 The picture of <I> Streptomyces avermitilis </I> under scanning electron microscope. </P></DIV></DIV></DIV><P class="p1"> In past years, scientists had been trying to transform gene into <I> S.avermitilis. </I> Until 1989, gene transformation into <I> S.avermitilis </I> was achieved through <A href="#CONJUGATION" title="more about conjugation"> conjugation </A> between <I> E.coli strains </I> (eg, <I> s17-1 </I> )and <I> S.avermitilis </I><I>(4)</I>. However, the efficiency was limited by the methyl-specific restriction system in <I> S.avermitilisi </I> , which shows strong restriction to gene methylated in normal <I> E.coli</I> strains  <I>(5)</I>. Eventually, high efficiency conjugation was achieved till the introduction of methylase-negative donor strain <I> E.coli </I><A href="#ET12567" title="about ET12567"> ET12567  </A> Now conjugation and strain ET12567  has been ubiquitously adopted in the gene transformation of <I> S.avermitilis. </I></P><H3> PROBLEMS AND SOLUTIONS </H3><P class="p1"> Environmentally friendly though avermectin is, the yield of avermectin in wild <I> S. avermitilis </I> doesn't fulfill our needs. Many efforts have been paid to increase its yield, including developing genome-minimized hosts, engineering the metabolic network<I>(2)</I>, etc. In our project, we plan to overexpress three genes, <I> frr, orfX, metK </I> in <I> S. avermitilis </I> to improve the yield of avermectin. </P><H3> CIRCUITS DESIGN </H3><P class="p1"> We have constructed three circuits to improve the yield of avermectin(Figure 3). PROMOTER: ermEp We chose ermEp, a strong constitutive promoter, to overexpress the three genes in <I> S.avermitilis </I> . It should be noticed that ermEp can only be expressed in <I> S.avermitilis </I> strains instead of <I> Escherichia coli </I> or any other chassis. </P><DIV class="row"><DIV class="col-md-12" style="text-align:center"><DIV class="cpleft"><P class="kuvateksti"> Figure 3 The circuits constructed for yield improvement of avermectin in <I> S.avermitilis </I> . </P></DIV></DIV></DIV><H3> BACKBONE: PL96 and PL97 </H3><P class="p1"> PL96 and PL97 are two high-copy vectors we used to overexpress our target genes. We get these vectors through commercial purchase. These vectors have pUC18 and pIJ101 replication origins for high-copy plasmid number in <I> Escherichia coli </I> and <I> S.avermitilis </I> , respectively, and the oriT (RK2) allows the efficient and convenient plasmid transfer from <I> E.coli </I> to <I> S.avermitilis </I> (6). </P><DIV class="row"><DIV class="col-md-12" style="text-align:center"><DIV class="cpleft"><P class="kuvateksti"> Figure 4 the map of plasmid backbone PL96. </P></DIV></DIV><DIV class="col-md-12" style="text-align:center"><DIV class="cpleft"><P class="kuvateksti"> Figure 5 The map of plasmid backbone PL97. </P></DIV></DIV></DIV><P class="p1"> To be noticed, we use special antibiotic aparamycin to choose final transformants. And there are aparamycin resistent gene <I> acc </I> in the backbone. </P><H3> EXPRESSION: </H3><P class="p1"> In order to  construct and express the three gene in <I> S.avermitilis </I> , we have adopted two hosts, <I> E.coli </I> DH5α and <I> E.coli </I> ET12567 . Then the target vectors are transferred from <I> E.coli </I> ET12567 to <I> S.avermitilis </I> by conjugation. </P><H3> PRIMARY HOST: <I> E.coli </I> DH5α </H3><P class="p1" id="ET12567"> As usual, we use <I> E.coli </I> DH5α to get plenty of recombinants in high quality and quantity. </P><H3> INTERMEDIA HOST: <I> E.coli </I> ET12567 </H3><P class="p1"> , <I> E.coli </I> ET12567 is a methylase-negative donor strain first used by MacNeil in 1988<I>(7)</I>. And we use <I> E.coli </I> ET12567 to demethylation the recombinants to better suit the methyl-specific restriction system in <A href="#avermitilisi" title="more about S.avermitilisi"><I> S.avermitilisi. </I></A></P><H3 id="CONJUGATION"> CONJUGATION: </H3><P class="p1"> Bacterial conjugation is the transfer of genetic material between bacterial cells by direct cell-to-cell contact or by a bridge-like connection between two cells. During conjugation the donor cell provides a conjugative or mobilizable genetic element that is most often a plasmid or transposon(8). In laboratories, successful transfers have been reported from bacteria to yeast(9), plants(10), mammalian cells(11), etc. In our project, we use the conjugation between <I> E.coli </I> ET12567 and <I> S.avermitilisi </I> to overexpress three target genes. </P><P class="p1"> To see the results of expression and toxic experiment on termites, please go to <A href="https://2015.igem.org/Team:ZJU-China/Results" title="Results"> results page </A> . </P><H3 id="avermitilisi"> CIRCUITS CONSTRUCTION </H3><P class="p1"> STEP ONE: PCR </P><P class="p1"> We amplify the target gene from the genome of <I> S.avermitilisi </I> by PCR. The primer and PCR program can be seen in our <A href="#biobrick" title="more about biobrick"> biobrick pages </A> . </P><P class="p1"> STEP TWO: TA CLONING </P><P class="p1"> We use TA cloning to efficiently clone the PCR products. In TA cloning, we use pMD19-T Vector, a vector transformed from pUC19 vector, to improve the efficiency of digestion and connection. As a result, we get three recombinant vectors of target genes and pMD19-T. </P><P class="p1"> STEP THREE: DIGESTION AND CONNECTION </P><P class="p1"> We digest the three recombinants and backbone PL96 with restriction enzymes NdeI, XbaI, then connect the fragments and backbone. Similarly, we use NdeI, Hind III to digest the three recombinants and backbone PL97 and connect the corresponding product. Then we get the target plasmids. </P><DIV class="row"><DIV class="col-md-6" style="text-align:center"><DIV class="cpleft"><P class="kuvateksti"> Figure 7 the sketch map of PL96 plasmid construction. </P></DIV></DIV><DIV class="col-md-6" style="text-align:center"><DIV class="cpleft"><P class="kuvateksti"> Figure 8 the sketch map of PL97 plasmid construction. </P></DIV></DIV></DIV><P class="p1"> For more detailed protocols, please go to <A href="#protocol" title="more about protocol"> protocol </A> . </P></DIV><H2 id="pos3"> Toxic protein manufacture </H2><P class="p1"> In order to kill the termites, we have chosen four types of insecticidal toxic proteins, respectively Tc protein tcdA1, tcdB1, bt-like Plu0840 and enterotoxin-like Plu1537, from <I> Photorhabdus luminescens</I> TT01,  a bacterium of native toxin storehouse. Then we clone these genes from the genome of TT01 , construct corresponding vectors, successfully express these proteins in <I> Escherichia coli</I> BL21 (DE3) and feed the termites with the raw engineered BL21 embedded with CNC. For more information about CNCs, please go to <A href="https://2015.igem.org/Team:ZJU-China/Design/CNC" title="about CNC">the main page of CNCs</A></P><DIV id="show2"><H3> HOST OF TOXIN -- <I> Photorhabdus luminescens </I></H3><P class="p1"><I> Photorhabdus luminescens </I> , one kind of gram-negative bacteria, is capable of producing and releasing a variety of insecticidal and bactericidal toxins. Living in symbiosis with nematodes, the bacteria are released and start to produce toxins that eventually kill the insect after insect larvae are invaded by nematodes, thereby generating a food resource for bacteria and nematodes (12). </P><DIV class="row"><DIV class="col-md-12" style="text-align:center"><DIV class="cpleft"><P class="kuvateksti"> Figure 9 Caterpillars infected with nematodes carrying symbiotic Photorhabdus luminescens2. Copyright 2003, Nature Publishing Group </P></DIV></DIV></DIV><P class="p1"> The whole genome of strain TT01, which has been sequenced in 2003, is predicted to encode 4839 kinds of protein<I>(12)</I>. And many of them are toxic proteins, most of which remain functionally unclear. Although they are toxic to insects and many other bacteria, <I> Photorhabdus luminescens </I> belongs to Risk Group 1 according to DSMZ (Deutsche Sammlung von Mikroorganismen und Zellkulturen) and has no toxic effect on human being at all. More than 50 years of field application of nematodes for controlling insect pests also showed that EN and their symbiotic bacteria ( <I> Photorhabdus luminescens </I> ) are safe to human and EN-based bio-pesticides were exempted from registration in many countries, including USA and all European countries <I>(13)</I>. </P><DIV class="row"><DIV class="col-md-12" style="text-align:center"><DIV class="cpleft"><P class="kuvateksti"> Figure 10 Circular representation of the <I> P. luminescens </I> genome. Copyright 2003, Nature Publishing Group </P></DIV></DIV></DIV><H3> TOXIN PROTEIN IN <I> P. luminescens </I> TT01 </H3><P class="p1"> Numerous toxins as there are in the genome of <I> P. luminescens </I> TT01, many of them have never been studied. Moreover, many small-molecule toxins are regulated by complex gene cluster, which makes it difficult to express in other standardized hosts, for instance <I> Escherichia coli. </I> Hence, on account of cost and safety, we chose four types of single-gene regulated toxic protein, TcdA1, TcdB1, Plu0840 and Plu1537, instead of small molecules because the former is easier to manipulate and less risky to the environment. </P><H3> tcdA1: PORE FORMING PROTEIN of Tc TOXIN FAMILY </H3><P class="p1"> The most remarkable toxin family till now is the Tc family, which are widely distributed among different gram-negative and gram-positive bacteria. </P><DIV class="row"><DIV class="col-md-12" style="text-align:center"><DIV class="cpleft"><P class="kuvateksti"> Figure 11 Structures of the TcA prepore and pore complex2. Copyright 2014, Nature Publishing Group </P></DIV></DIV></DIV><P class="p1"> Tcs are composed of TcA, TcB, and TcC. TcA is supposed to perforate the membrane by forming channel outside-in and translocating the toxic enzymes into the host. Meanwhile the TcB and TcC cooperate with a syringe-like mechanism during membrane insertion(14). </P><P class="p1"> In a 2008 study, researchers expressed tcdA1 and tcdB1 in  <I> Enterobacter cloacae </I> and fed the termites with <I> E. cloacae </I> to control termites<I>(15)</I>. Inspired by their experiment, we chose to express tcdA1 (Uniprot: Q7N7Y9_PHOLL) and tcdB1(Uniprot: Q7N7Z0_PHOLL) to kill termites. For more details, please go to <A href="https://2015.igem.org/Team:ZJU-China/Parts" title="part page"> parts </A></P><H3> Plu1537: Bt HOMOLOGOUS TOXIC PROTEIN </H3><P class="p1"> The exact function of Plu1537 is still unclear, but a research in 2009 indicated that Plu1537 had insecticidal activity against Galleria larvae <I>(16)</I>. </P><P class="p1"> Judging that the Plu1537 protein has 30% predicted amino acid sequence similarity to a 13.6 kDa insecticidal crystal protein cry34Ab1(figure 12) in <I> Bacillus thuringiensis </I> (Uniprot: Q939T0_BACTU), which belongs to Bt crystal protein family, it may have similar toxic effect with cry34Ab1 Bt protein. </P><P class="p1"> Bt protein may be the most well-known toxic protein till now. It is widely used in transgene plants to kill the larvae of worm. It also “interacts with membranes to form pores”(17). And there is abundant evidence to ensure the safety of Bt protein. </P><DIV class="row"><DIV class="col-md-12" style="text-align:center"><DIV class="cpleft"><P class="kuvateksti"> Figure 12 Structures of the Cry34Ab1 protein2. Copyright 2014, Worldwide Protein Data Bank </P></DIV></DIV></DIV><P class="p1"> We have successfully cloned the plu1537 gene and expressed the Plu1537 toxin protein in <I> E.coli </I>BL21 (DE3), for more details, please go to </P><H3> Plu0840: ENTEROTOXIN Ast HOMOLOGOUS PROTEIN </H3><P class="p1"> The exact function of Plu0840 is also unclear. A 2007 study confirmed that Plu0840 had weak oral toxicity against two kinds of moth ( <I> S. litura and S. exigua </I> )<I>(13)</I>. </P><P class="p1"> Sequence analysis showed that the plu0840 in the P. luminescens TT01 genome has 55% sequence identity with an enterotoxin Ast from Aeromonas hydrophila, therefore may play a similar role. (see figure 13) </P><DIV class="row"><DIV class="col-md-12" style="text-align:center"><DIV class="cpleft"><P class="kuvateksti"> Figure 13 Homologous alignment result of toxic protein Plu0840. Copyright 2014, Worldwide Protein Data Bank </P></DIV></DIV></DIV><P class="p1"> In 2001, researchers studied the function of enterotoxin Ast from Aeromonas hydrophila, concluded that it played an important role in A. hydrophila-induced gastroenteritis in a mouse model(18). </P><P class="p1"> We have successfully cloned the <I> plu0840 </I> gene and expressed he Plu0840 toxin protein in <I> E.coli </I> BL21 (DE3) , for more details, please go to the next page. </P><DIV class="row"><DIV class="col-md-12" style="text-align:center"><DIV class="cpleft"><P class="kuvateksti">Figure 14 The circuits constructed for toxic protein expression</P></DIV></DIV></DIV><H3> CIRCUITS DESIGN </H3><P class="p1"> As displayed in figure 14, we have constructed three devices to express corresponding toxic proteins, plu1537 (BBa_K1668010), plu0840 (BBa_K1668009) and tcdA1 (BBa_K1668008) </P><UL><LI><A href="http://parts.igem.org/Part:BBa_K1668010" title="go to part page"> BBa_K1668010 </A></LI><LI><A href="http://parts.igem.org/Part:BBa_K1668009" title="go to part page"> BBa_K1668009 </A></LI><LI><A href="http://parts.igem.org/Part:BBa_K1668008" title="go to part page"> BBa_K1668008 </A></LI><LI><A href="http://parts.igem.org/Part: BBa_I0500" title="go to part page"> BBa_I0500 </A></LI><LI><A href="http://parts.igem.org/Part: BBa_B0034" title="go to part page"> BBa_B0034 </A></LI><LI><A href="http://parts.igem.org/Part:BBa_K1668011)" title="go to part page"> BBa_K1668011 </A></LI></UL><H3> PROMOTER: pBad(BBa_I0500) </H3><P class="p1"> We chose arabinose inducible promoter pBad (BBa_I0500) because it's not only of medium strength with arabinose up to certain concentration, but also have little leakage. Moreover, the pBad promoter is repressed by glucose, giving the expression more controllability. In order to promote expression, we chose one of the strongest RBS in Parts Registry (BBa_B0034). </P><P class="p1"> BACOBONE: pSB1C3 </P><H3> EXPRESSION: </H3><P class="p1"> We adopted tandem expression of toxin and reporter mCherry (BBa_K1668011) to roughly judge whether toxin is expressed. </P><P class="p1"> We use <I> E.coli </I><I> DH5α </I> to get plenty recombinants in high quality and quantity. Then we transform the positive recombinants into <I> E.coli </I><I> BL21 (DE3) </I> for high-quality expression. </P><P class="p1"> To see the results of expression and toxic experiment on termites, please go to <A href="https://2015.igem.org/Team:ZJU-China/Results">results</A></P><H3> CIRCUITS CONSTRUCTION </H3><P class="p1"> STEP ONE: PCR </P><P class="p1"> We amplify the target gene from the genome of <I> S.avermitilisi </I> by PCR. We also clone the arabinose inducible promoter pBad from Part Registry. The primer and PCR program can be seen in our <A href="#biobrick" title="more about biobrick"> biobrick pages </A> . </P><P class="p1"> STEP TWO: BACKBONE DIGESTION </P><P class="p1"> We digest the part BBa_J06702, mCherry with RBS in front and double terminator behind, with restriction enzyme XbaI to make a linearized backbone. </P><P class="p1"> STEP THREE: SCARLESS ASSEMBLY </P><P class="p1"> We use the MultiS_one step cloning kit of Vazyme company to assemble the target gene and backbone. The mechanism is showed in figure 15. </P><P class="p1"> For more details about scarless assembly and any other protocols, please go to <A href="#protocol" title="more about protocol"> protocol </A></P><DIV class="row"><DIV class="col-md-12" style="text-align:center"><DIV class="cpleft"><P class="kuvateksti"> Figure 15 the mechanism of scarless cloning </P></DIV></DIV></DIV></DIV><H2> Reference </H2><P class="p1"> 1. X. Zhang et al., APPL MICROBIOL BIOT 72, 986 (2006-09-27, 2006). </P><P class="p1"> 2. H. Ikeda, K. Shin-ya, S. Omura, J IND MICROBIOL BIOT 41, 233 (2014). </P><P class="p1"> 3. H. Ikeda et al., NAT BIOTECHNOL 21, 526 (2003). </P><P class="p1"> 4. P. MAZODIER, R. PETTER, C. THOMPSON, J BACTERIOL 171, 3583 (1989). </P><P class="p1"> 5. F. Flett, V. Mersinias, C. P. Smith, FEMS MICROBIOL LETT 155, 223 (1997). </P><P class="p1"> 6. Ning Sun, BIOCHEM MOL BIOL EDU (2013). </P><P class="p1"> 7. D. J. MACNEIL, J BACTERIOL 170, 5607 (1988). </P><P class="p1"> 8. R. K. Holmes, M. G. Jobling, (1996-01-19, 1996). </P><P class="p1"> 9. J. A. HEINEMANN, G. F. SPRAGUE, NATURE 340, 205 (1989). </P><P class="p1"> 10. T. Kunik et al., P NATL ACAD SCI USA 98, 1871 (2001). </P><P class="p1"> 11. V. L. Waters, NAT GENET 29, 375 (2001). </P><P class="p1"> 12. E. Duchaud et al., NAT BIOTECHNOL 21, 1307 (2003). </P><P class="p1"> 13. M. Li, L. H. Qiu, Y. Pang, ANN MICROBIOL 57, 313 (2007). </P><P class="p1"> 14. C. Gatsogiannis et al., NATURE 495, 520 (2013-03-20, 2013). </P><P class="p1"> 15. R. Zhao et al., APPL ENVIRON MICROB 74, 7219 (2008-12-01, 2008). </P><P class="p1"> 16. M. Li et al., MOL BIOL REP 36, 785 (2009). </P><P class="p1"> 17. M. S. Kelker et al., PLOS ONE 9, (2014). </P><P class="p1"> 18. J. Sha, E. V. Kozlova, A. K. Chopra, INFECT IMMUN 70, 1924 (2002). </P></ARTICLE><ASIDER id="popView2"><HEADER>
      Toxin manufacture 
    </HEADER><UL id="nav2"><LI><A href="#pos1"> Introduction </A></LI><LI><A href="#pos2"> Avermectin manufacture </A></LI><LI><A href="#pos3"> Toxic protein manufacture </A></LI></UL></ASIDER></DIV><ASIDE id="popView1"><HEADER>
     Project 
   </HEADER><UL id="nav"><LI><A href="https://2015.igem.org/Team:ZJU-China/Project"> Over View </A></LI><LI><A href="https://2015.igem.org/Team:ZJU-China/Project/Termite_Issue"> Termite Issue </A></LI><LI><A href="https://2015.igem.org/Team:ZJU-China/Design/Toxinmanufacture"> Toxins </A></LI><LI><A href="https://2015.igem.org/Team:ZJU-China/Design/CNC"> CNC </A></LI><LI><A href="https://2015.igem.org/Team:ZJU-China/Design/Termites"> Termites </A></LI><LI><A href="https://2015.igem.org/Team:ZJU-China/Project/Protocol"> Protocol</A></LI></UL></ASIDE><DIV id="gl-footer" class="cf" style="max-width:1440px"><DIV id="gl-footer-copyright" class="cf"><DIV class="gl-col1 cf"><SPAN>© 2015 ZJU-IGEM TERMINATE.</SPAN></DIV><UL class="gl-col2 cf"><LI><A href="http://www.igem.org/Main_Page">IGEM</A></LI><LI><A href="http://www.zju.edu.cn">Zhejiang University</A></LI></UL></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>