<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_ZJU-China_Basic_Part skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:ZJU-China/Basic Part</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="gl-outer-wrapper" class="gl-zh cf" style=""><DIV id="gl-header" class="cf gl-header-fixed" style="max-width:1440px"><DIV id="gl-header-bottom" class="cf"><DIV id="gl-menu" class="cf"><DIV id="gl-menu-one" class="gl-has-submenu"><DIV id="gl-submenu-one" class="gl-submenu"><DIV class="gl-col-footer" style="border-top:0px solid #aaa;"><A href="https://2015.igem.org/Team:ZJU-China">Home</A></DIV></DIV></DIV><DIV id="gl-menu-two" class="gl-has-submenu"><DIV id="gl-submenu-three" class="gl-submenu"><DIV class="gl-col-wrapper"><DIV class="gl-col gl-col1"><A href="https://2015.igem.org/Team:ZJU-China/Parts" class="gl-hover"><SPAN>Team Part</SPAN><SPAN>Team Part</SPAN></A></DIV><DIV class="gl-col gl-col2"><A href="https://2015.igem.org/Team:ZJU-China/Basic_Part" class="gl-hover"><SPAN>Basic Part</SPAN><SPAN>Basic Part</SPAN></A></DIV><DIV class="gl-col gl-col3"><A href="https://2015.igem.org/Team:ZJU-China/Description" class="gl-hover"><SPAN>Improved  Part</SPAN><SPAN>Improved  Part</SPAN></A></DIV><DIV class="gl-col gl-col4"><A href="https://2015.igem.org/Team:ZJU-China/Composite_Part" class="gl-hover"><SPAN>Composite Part</SPAN><SPAN>Composite Part</SPAN></A></DIV></DIV><DIV class="gl-col-footer"><A href="https://2015.igem.org/Team:ZJU-China/Parts">Parts</A></DIV></DIV></DIV><DIV id="gl-menu-three" class="gl-has-submenu"><DIV id="gl-submenu-three" class="gl-submenu"><DIV class="gl-col-wrapper"><DIV class="gl-col gl-col1" style="width: 33.3%;"><A href="https://2015.igem.org/Team:ZJU-China/Project" class="gl-hover"><SPAN>Overview</SPAN><SPAN>Quick view for our project</SPAN></A><A href="https://2015.igem.org/Team:ZJU-China/Project/Termite_Issue" class="gl-hover"><SPAN>Termite Issue</SPAN><SPAN>The Termite Issue</SPAN></A></DIV><DIV class="gl-col gl-col2" style="width: 33.3%;"><DIV><SPAN><P>Design</P></SPAN><UL><LI><A href="https://2015.igem.org/Team:ZJU-China/Design/Toxinmanufacture">Toxins Manufacture</A></LI><LI><A href="https://2015.igem.org/Team:ZJU-China/Design/CNC" style="padding-left:32px">Bacteria Carrier</A></LI><LI><A href="https://2015.igem.org/Team:ZJU-China/Design/Termites" style="padding-left:32px">Termites</A></LI></UL></DIV></DIV><DIV class="gl-col gl-col3" style="width: 33.3%;"><A href="https://2015.igem.org/Team:ZJU-China/Project/Protocol" class="gl-hover"><SPAN>Protocol</SPAN><SPAN>Quick view for Protocol</SPAN></A></DIV></DIV><DIV class="gl-col-footer"><A href="https://2015.igem.org/Team:ZJU-China/Project">Project</A></DIV></DIV></DIV><DIV id="gl-menu-four" class="gl-has-submenu"><DIV id="gl-submenu-three" class="gl-submenu"><DIV class="gl-col-wrapper"><DIV class="gl-col gl-col1"><A href="https://2015.igem.org/Team:ZJU-China/Practices" class="gl-hover"><SPAN>Overview</SPAN><SPAN>Quick view for Outreach</SPAN></A><DIV><SPAN><P>Society</P></SPAN><UL><LI><A href="https://2015.igem.org/Team:ZJU-China/Practices/Society/Survey">Survey</A></LI><LI><A href="https://2015.igem.org/Team:ZJU-China/Practices/Society/Consultation" style="padding-left:32px">Consultation</A></LI><LI><A href="https://2015.igem.org/Team:ZJU-China/Practices/Society/Syn-bio_Debate" style="padding-left:32px">Syn-bio Debate</A></LI></UL></DIV></DIV><DIV class="gl-col gl-col2"><DIV><SPAN><P>Application</P></SPAN><UL><LI><A href="https://2015.igem.org/Team:ZJU-China/Design">Device</A></LI><LI><A href="https://2015.igem.org/Team:ZJU-China/Entrepreneurship" style="padding-left:32px">Promotion</A></LI><LI><A href="https://2015.igem.org/Team:ZJU-China/Practices/Application/Safety" style="padding-left:32px">Safety</A></LI><LI><A href="https://2015.igem.org/Team:ZJU-China/Practices/Application/Future" style="padding-left:32px">Future</A></LI></UL></DIV></DIV><DIV class="gl-col gl-col3"><DIV><SPAN><P>Public Promotion</P></SPAN><UL><LI><A href="https://2015.igem.org/Team:ZJU-China/Practices/Public_Promotion/Poly-cards">Poly-cards</A></LI><LI><A href="https://2015.igem.org/Team:ZJU-China/Practices/Public_Promotion/Syn-bio_Share" style="padding-left:32px">Syn-bio Share</A></LI><LI><A href="https://2015.igem.org/Team:ZJU-China/Practices/Public_Promotion/Community_Lab" style="padding-left:32px">Community Lab </A></LI><LI><A href="https://2015.igem.org/Team:ZJU-China/Practices/Public_Promotion/Seminar" style="padding-left:32px">Seminar</A></LI></UL></DIV></DIV><DIV class="gl-col gl-col4" style="padding-right:80px"><DIV><SPAN><P>Collaboration</P></SPAN><UL><LI><A href="https://2015.igem.org/Team:ZJU-China/Practices/Cooperate/Senergene/first">SynenergeneⅠ</A></LI><LI><A href="https://2015.igem.org/Team:ZJU-China/Practices/Cooperate/Senergene/second" style="padding-left:32px">SynenergeneⅡ</A></LI><LI><A href="https://2015.igem.org/Team:ZJU-China/Practices/Cooperate/Senergene/third" style="padding-left:32px">Synenergene Ⅲ</A></LI><LI><A href="https://2015.igem.org/Team:ZJU-China/Collaborations" style="padding-left:32px">Other Teams</A></LI></UL></DIV></DIV></DIV><DIV class="gl-col-footer"><A href="https://2015.igem.org/Team:ZJU-China/Practices">Outreach</A></DIV></DIV></DIV><DIV id="gl-menu-five" class="gl-has-submenu"><DIV id="gl-submenu-three" class="gl-submenu"><DIV class="gl-col-wrapper"><DIV class="gl-col gl-col1"><A href="https://2015.igem.org/Team:ZJU-China/Team" class="gl-hover"><SPAN>Members</SPAN><SPAN>Introduce our Members</SPAN></A></DIV><DIV class="gl-col gl-col2"><A href="https://2015.igem.org/Team:ZJU-China/Attributions" class="gl-hover"><SPAN>Attributions</SPAN><SPAN>Introduce the Attributions</SPAN></A></DIV><DIV class="gl-col gl-col3"><A href="https://2015.igem.org/Team:ZJU-China/Team/Acknowledgement" class="gl-hover"><SPAN>Acknowledgement</SPAN><SPAN>Introduce the Acknowledgement</SPAN></A></DIV><DIV class="gl-col gl-col4"><A href="https://2015.igem.org/Team:ZJU-China/Team/Sponsors" class="gl-hover"><SPAN>Sponsors</SPAN><SPAN>Introduce our Sponsors </SPAN></A></DIV></DIV><DIV class="gl-col-footer"><A href="https://2015.igem.org/Team:ZJU-China/Team">Team</A></DIV></DIV></DIV><DIV id="gl-menu-six" class="gl-has-submenu"><DIV id="gl-submenu-three" class="gl-submenu"><DIV class="gl-col-wrapper"><DIV class="gl-col gl-col1" style="border-right:2px solid #ddd"><A href="https://2015.igem.org/Team:ZJU-China/Modeling" class="gl-hover"><SPAN>Modeling Overview</SPAN><SPAN>Introduce our Members</SPAN></A></DIV><DIV class="gl-col gl-col2"><A href="https://2015.igem.org/Team:ZJU-China/Modeling/first" class="gl-hover"><SPAN>Modeling α</SPAN><SPAN>Termite Simulation</SPAN></A></DIV><DIV class="gl-col gl-col3"><A href="https://2015.igem.org/Team:ZJU-China/Modeling/second" class="gl-hover"><SPAN>Modeling β</SPAN><SPAN>Mass transfer model</SPAN></A></DIV><DIV class="gl-col gl-col4"><A href="https://2015.igem.org/Team:ZJU-China/Modeling/third" class="gl-hover"><SPAN>Modeling γ</SPAN><SPAN>Degradation model</SPAN></A></DIV></DIV><DIV class="gl-col-footer"><A href="https://2015.igem.org/Team:ZJU-China/Modeling">Modeling</A></DIV></DIV></DIV><DIV id="gl-menu-seven" class="gl-has-submenu"><DIV id="gl-submenu-seven" class="gl-submenu"><DIV class="gl-col-footer" style="border-top:0px solid #aaa;"><A href="https://2015.igem.org/Team:ZJU-China/Notebook">Notebook</A></DIV></DIV></DIV><DIV id="gl-menu-eight" class="gl-has-submenu"><DIV id="gl-submenu-three" class="gl-submenu"><DIV class="gl-col-wrapper"><DIV class="gl-col gl-col1"><A href="https://2015.igem.org/Team:ZJU-China/Results" class="gl-hover"><SPAN>Experiment Results</SPAN><SPAN>toxins manufacture</SPAN></A></DIV><DIV class="gl-col gl-col2"><A href="https://2015.igem.org/Team:ZJU-China/Results/second" class="gl-hover"><SPAN>Experiment Result</SPAN><SPAN>bacteria carrier</SPAN></A></DIV><DIV class="gl-col gl-col3"><A href="https://2015.igem.org/Team:ZJU-China/Results/third" class="gl-hover"><SPAN>Experiment Result</SPAN><SPAN>termites simulation</SPAN></A></DIV><DIV class="gl-col gl-col4"><A href="https://2015.igem.org/Team:ZJU-China/Results/social_result" class="gl-hover"><SPAN>Social Result</SPAN><SPAN>social results</SPAN></A></DIV></DIV><DIV class="gl-col-footer"><A href="https://2015.igem.org/Team:ZJU-China/Results">Results</A></DIV></DIV></DIV><DIV id="gl-menu-nine" class="gl-has-submenu"><DIV id="gl-submenu-nine" class="gl-submenu"><DIV class="gl-col-footer" style="border-top:0px solid #aaa;"><A href="https://2015.igem.org/Team:ZJU-China/Safety">Safety</A></DIV></DIV></DIV><DIV id="gl-menu-ten" class="gl-has-submenu" style="padding-right:20px"><DIV id="gl-submenu-three" class="gl-submenu"><DIV class="gl-col-wrapper"><DIV class="gl-col gl-col1"><A href="https://2015.igem.org/Team:ZJU-China/Design" class="gl-hover"><SPAN>Device</SPAN><SPAN>Introduce our Device and Instruction Manual</SPAN></A></DIV><DIV class="gl-col gl-col2"><A href="https://2015.igem.org/Team:ZJU-China/Entrepreneurship" class="gl-hover"><SPAN>Promotion</SPAN><SPAN>The Promotion</SPAN></A></DIV><DIV class="gl-col gl-col3"><A href="https://2015.igem.org/Team:ZJU-China/Practices/Application/Safety" class="gl-hover"><SPAN>Safety</SPAN><SPAN>The Safety</SPAN></A></DIV><DIV class="gl-col gl-col4"><A href="https://2015.igem.org/Team:ZJU-China/Practices/Application/Future" class="gl-hover"><SPAN>Future</SPAN><SPAN>Future</SPAN></A></DIV></DIV><DIV class="gl-col-footer"><A href="https://2015.igem.org/Team:ZJU-China/Design">Application</A></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV id="page-content-wrapper"><ARTICLE><H3 class="hh2" style="border:0;color:red" id="pos1">Part:BBa_K1668005</H3><H4 class="hh4-left" style="padding-top:30px" style="padding-top:30px">CDS <I>tcdA1</I></H4><P class="p1">
        The part CDS <I>tcdA1</I> is the coding sequence of insecticidal protein TcdA1, which is used for termite control in our project. 


    </P><P class="p1"><I>tcdA1</I> is one of the longest genes in bacteria. And the TcdA1 toxic protein is a 285kDa pore-forming protein, belonging to tc toxic family which is widely distributed among gram-positive and gram-positive bacteria. 
    </P><DIV style="width:100%l;text-align:center;font-size:30px;"><A href="http://parts.igem.org/Part:BBa_K1668005">Sequence and Features</A><H3 class="hh2">OVERVIEW<P class="p1">
      TcdA1, one of the biggest proteins in bacteria (285kDa), is first found in <I>Photorhabdus luminescens</I>. 
It forms channels and assists other toxins across the cell membrane. 
It belongs to Tc toxic protein family, which is widely distributed among different gram-negative and gram-positive bacteria.  
    </P><P class="p1">
      We clone and standardize the gene into standard plasmid pSB1C3 and confirmed it by digesting and sequencing. 

    </P></H3><H3 class="hh2">Background<DIV><DIV class="col-md-12 textcenter"><DIV class="cpleft"><P class="kuvateksti">
                Fig.1 The 3D structure of tcdA1. Copyright 2013, Nature Publishing Group.
              </P></DIV></DIV></DIV><P class="p1">
     TcdA1 is a pore-forming macro-protein, which can keep the ability to form a pore in a large pH range (from 4 to 11). To be noticed, at pH11, the pore-forming activity of TcdA1 is more than 100-fold greater than at pH6. As the midguts of most insects are alkaline, Tc toxic proteins are effective by feeding on insects, including termites. 
    </P><P class="p1">
      In 2013, the structure of TcdA1 was revealed by researchers and reported in nature(1). As displayed in figure1a&amp;b, the TcdA1 is composed of three parts: N-terminal a-helical domain(brown), the central b-sheet domain(green) and the C-terminal pore-forming domain(yellow). The protein has two states: pre-pore state and pore state. The pore-forming domain (figure 1c) sticks out to form pore, changing into pore state (figure 2). 
    </P><P class="p1">
     Moreover, the TcdA1 toxin helps other toxins to enter the cell membrane. Naturally in strain TT01, TcdA1 is expressed homologously with other toxins, for example, TcdB1 and Tcc toxins. TcdA1 helps to transfer the latter into the cell to maximum the toxic effect(figure 3). 
    </P></H3><H3 class="hh2">Construction</H3><H4 class="hh4-left">PCR<P class="p1">
        The CDS<I>tcdA1</I> gene was amplified by PCR by three pieces to remove three enzyme site EcoRI in its original squnce. The template is genomic DNA extracted from strain <I>Photorhabdus luminescens</I> TT01. 
    </P><P class="p1">
	We use primer tcdA1-left L and tcdA1-left R to amplify the left side of gene., tcdA1-middle L and tcdA1-middle R to amplify the middle part and tcdA1-right L and tcdA1-right R to amplify the right side. Primers are shown below.
	 </P></H4><H4 class="hh4-left">Seamless assembly<P class="p1">
     We used seamless assembly as our assembly method so restriction digestion and T4 ligation can be avoided. Detailed protocol and instruction for primer design can be seen in our Protocol. By this way prefix sequence, <I>tcdA1</I>-left, <I>tcdA1</I>-middle, <I>tcdA1</I>-right, and suffix sequence can be ligated seamlessly. 


    </P><P class="p1">
tcdA1-left F (F, 5’-3’): AATTCGCGGCCGCTTCTAGATGAACTCGCCTGTAAAAGAGA    </P><P class="p1">
tcdA1-left R (R, 5’-3’): AAATTCACAACAAGCGGATACATTACAC    </P><P class="p1">
tcdA1-middle F(F, 5’-3’): TATCCGCTTGTTGTGAATTTAATCCGG     </P><P class="p1">
tcdA1-middle R(R, 5’-3’): TAACATTAGCGGAAAATTCCATCACATAACCTGTTGC  </P><P class="p1">
tcdA1-right F(F, 5’-3’): GGAATTTTCCGCTAATGTTATGAATACCGAAGC   </P><P class="p1">
tcdA1-right R(R, 5’-3’): CTGCAGCGGCCGCTACTAGTATTATTATTTAATGGTGTA   </P></H4><H4 class="hh4-left">Transformation and confirmation<P class="p1">
      After seamless assembly, standard plasmid pSB1C3 containing <I>tcdA1</I> gene was transformed into <I> E.coli DH5α</I>. When single colony appeared on the LB plate, we picked out 10 colonies, respectively, as our template for bacteria solution PCR. In order to avoid the appearance of false positive clones, we used tcdA1_M1(a primer on the right part of tcdA1)/ VR as the universal primers for the CDS<I>tcdA1</I> is too long to amplify all the parts sequence with VF2/VR .Primers are shown below. The positive clone and its corresponding raw bacteria solution were stored and samples were sent to do DNA sequencing.
    </P><P class="p1">
tcdA1_M1(F, 5’-3’):  CTGCCAATCTATGCCACACC    </P><P class="p1">
VR(F, 5’-3’):  ATTACCGCCTTTGAGTGAGC    </P></H4><H4 class="hh4-left">Plasmid map<DIV class="col-md-12 textcenter"><DIV class="cpleft"><P class="kuvateksti">
                Fig.2 The plasmid map of BBa_K1668005 CDS<I>tcdA1</I></P></DIV></DIV></H4><H3 class="hh2">RESULTS</H3><H4 class="hh4-left">DNA CONSTRUCTION<DIV class="col-md-12 textcenter"><DIV class="cpleft"><P class="kuvateksti">
	Figure 4 PCR confirmation results of CDS <I>tcdA1</I>. We choose a 1k fragment in the middle of <I>tcdA1</I> gene as template because the whole gene is much too long.
            </P></DIV></DIV><P class="p1">
The template we use is recombinant plasmids. 
    </P><P class="p1">
It can be clearly seen the two recombinants shown in figure 4 is two positive cloning. 
    </P></H4><H4 class="hh4-left">DNA SEQUNCING<P class="p1">
We have sequenced the parts with standard primers VF2 and VR. The sequence of the 7.5k part shows 100% agreement with the desired sequence. 
    </P></H4><H3 class="hh2">Reference</H3><P class="p1">
        1. C. Gatsogiannis et al., NATURE 495, 520 (2013-03-20, 2013). 
		2. D. Meusch et al., NATURE 508, 61 (2014).
    </P></DIV></ARTICLE><ARTICLE><H3 class="hh2" style="border:0;color:red;padding-top:100px" id="pos2">BBa_K1668006</H3><H4 class="hh4-left" style="padding-top:30px" style="padding-top:30px">CDS plu0840</H4><P class="p1">
      The part CDS<I>plu0840</I> is coding sequence of toxin protein Plu0840, which is used for termite control in our project. 
    </P><P class="p1">
      Plu0840 is a 72kDa insecticidal toxic protein, which had weak oral toxicity against two kinds of moth according to a 2007 research and showed weak toxicity to termites by oral feeding.
    </P><H4 class="hh4-left" style="padding-top:30px" style="padding-top:30px">Characterization</H4><H3 class="hh2">OVERVIEW</H3><P class="p1">
        Plu0840 is a insecticidal toxin found in <I>Photorhabdus luminescens</I>, a native toxin storehouse. In our project, it is used for termite control. 
    </P><P class="p1">
        We clone and standardize the gene into standard plasmid pSB1C3, and confirmed the part by PCR and sequencing. Then we combine the CDS plu0840 with arabinose inducible promoter pBad in front reporter mCherry and double terminator behind into the device plu0840 to strongly express the toxin. 
    </P><H3 class="hh2">BACKGROUND</H3><P class="p1">
      In 2009 research Cloning and expression analysis of a predicted toxin gene from Photorhabdus sp. HB78, plu0840 fused with GST is expressed in <I>E.coli</I> BL21(DE3). Engineered strain BL21 was both orally fed and injected in hemocoel to two kind of moth (S. litura and S.exigua). 
    </P><P class="p1">
      According to the result, on the one hand, oral feeding effectively inhibits the growth of larva while has only weak oral toxic effect. On the other hand, hemocoel injection showed negative results. 
    </P><P class="p1">
      The research mentions that Plu0840 (figure 1) shares 55% sequence identity with an enterotoxin Ast from aeromonas hydrophila. Aaeromonas hydrophila, which is connected with gastroenteritis, may lead to altered fluid secretion in mouse. According to a 2002 research, enterotoxin Ast makes weak contributions to fluid secretion compared with two other genes (2) However, judging that TT01 is nontoxic to animals at all, we think Plu0840 may share little similarity with Ast.
    </P><DIV><DIV class="col-md-12 textcenter"><DIV class="cpleft"><P class="kuvateksti">
                Figure 1 The 3D structure of Plu0840. Copyright 2014, Worldwide Protein Data.
              </P></DIV></DIV></DIV><DIV><H3 class="hh2">Construction</H3><H4 class="hh4-left">PCR<P class="p1">
The CDS<I>plu0840</I> gene was amplified by PCR by two pieces to remove one enzyme site EcoRI in its original squnce. The template is genomic DNA extracted from strain <I>Photorhabdus luminescens TT01</I>. 	</P><P class="p1">
We use primer plu0840-left L and plu0840-left R to amplify the left side of gene. And we use plu0840-right L and plu0840-right R to amplify the right side. Primers are shown below.	 </P></H4><H4 class="hh4-left">Seamless assembly<P class="p1">
We used seamless assembly as our assembly method so restriction digestion and T4 ligation can be avoided. Detailed protocol and instruction for primer design can be seen in our Protocol. By this way prefix sequence, <I>plu0840</I>-left, <I>plu0840</I>-right, and suffix sequence can be ligated seamlessly. 



    </P><P class="p1">
plu0840-left F (F, 5’-3’): AATTCGCGGCCGCTTCTAGATGTTGTATAACACCCCAGT  </P><P class="p1">
plu0840-left R (R, 5’-3’): CAACGAATTAAGAGGGAACAGCGGCC   </P><P class="p1">
plu0840-right F(F, 5’-3’): TGTTCCCTCTTAATTCGTTGATGCTGCATGGC  </P><P class="p1">
plu0840-right R(R, 5’-3’): 
ACTGCAGCGGCCGCTACTAGTATTATTATTTCGATGGGGTCAAAG
 </P></H4><H4 class="hh4-left">Transformation and confirmation<P class="p1">
After seamless assembly, standard plasmid pSB1C3 containing <I>plu0840</I> gene was transformed into <I> E.coli</I> DH5α. When single colony appeared on the LB plate, we picked out 10 colonies, respectively, as our template for bacteria solution PCR. In order to avoid the appearance of false positive clones, we used VF2/VR as the universal primers. The positive clone and its corresponding raw bacteria solution were stored and samples were sent to do DNA sequencing.    </P></H4><H4 class="hh4-left">Plasmid map<DIV class="col-md-12 textcenter"><DIV class="cpleft"><P class="kuvateksti">
                Fig.2 The plasmid map of BBa_K1668006 CDS<I>plu0840</I></P></DIV></DIV></H4><H3 class="hh2">RESULTS</H3><H3>PLASMID CONSTRUCTION</H3><P class="p1">
        The template we use is recombinant plasmids. 
    </P><P class="p1">    
        It can be clearly seen the recombinants shown in figure 2 is positive cloning. 
    </P><DIV><DIV class="col-md-12 textcenter"><DIV class="cpleft"><P class="kuvateksti">
                Figure 2 PCR confirmation results of CDS plu0840 with standard primers VF2 and VR.
              </P></DIV></DIV></DIV><H4 class="hh4-left" style="padding-top:30px" style="padding-top:30px"> DNA Sequencing </H4><P class="p1">
      We have sequenced the parts with standard primers VF2 and VR. The sequence of the 1.8k part shows 100% agreement with the desired sequence. 
    </P><H3 class="hh2">REFERENCE</H3><P class="p1">
      1. M. Li et al., MOL BIOL REP 36, 785 (2009).
    </P><P class="p1">
      2. J. Sha, E. V. Kozlova, A. K. Chopra, INFECT IMMUN 70, 1924 (2002).
    </P><P class="p1">
      3. Plu0840 (uniprot):<A href="http://www.uniprot.org/uniprot/Q7N895">http://www.uniprot.org/uniprot/Q7N895</A> .
    </P><P class="p1">
      4. 3D structure of Plu0840:<A href="http://www.proteinmodelportal.org/?pid=modelDetail&amp;provider=MODBASE&amp;template=1uasA&amp;pmpuid=1000725544546&amp;range_from=1&amp;range_to=639&amp;ref_ac=Q7N895&amp;mapped_ac=Q7N895&amp;zid=async">Link Here</A> .
    </P></DIV></ARTICLE><ARTICLE><H3 class="hh2" style="border:0;color:red;padding-top:100px" id="pos3">Part:BBa_K1668007</H3><H4 class="hh4-left" style="padding-top:30px" style="padding-top:30px">CDS <I>plu1537</I></H4><P class="p1">
        The part <I>CDSplu1537</I>  is coding sequence of toxin protein Plu1537, which is used for termite control in our project. 

    </P><P class="p1">
        Plu1537 is a 14kDa insecticidal toxic protein, which has strong toxicity against termites by oral feeding according to our termites experiment. It has 30% homology with a kind of Bt toxic protein, therefore it may play a similar role.
    </P><DIV style="width:100%l;text-align:center;font-size:30px;"><A href="http://parts.igem.org/Part:BBa_K1668007">Sequence and Features</A><H4 class="hh4-left" style="padding-top:30px" style="padding-top:30px">Characterization</H4><H5 class="hh2"> OVERVIEW </H5><P class="p1">
        Plu1537 is a small insecticidal toxin found in <I>Photorhabdus luminescens</I>, a native toxin storehouse. In our project, it is used for termite control. 

    </P><H5 class="hh2"> BACKGROUND </H5><P class="p1">
        In 2009 research Expression and activity of a probable toxin from <I>Photorhabdus Luminescens</I>, toxin protein Pit, which is 94% homologous with Plu1537, is expressed in <I>E.coli</I> BL21(DE3). Engineered strain BL21 was both orally fed and injected in hemocoel to two kind of moths(Galleria mellonella &amp; Spodoptera litura)(1). As a result, hemocoel injection is more effective than oral feeding. However, our experiment showed that oral feeding is also effective.     </P><P class="p1">
        Plu1537 shares 30% amino acid sequence similarity with a 13.6 kDa insecticidal crystal protein Cry34Ab1 in <I>Bacillus thuringiensis</I> (figure 1), which belongs to Bt toxin family. 
    </P><P class="p1">
        Bt family may be the most famous insecticidal toxin up to now. It’s one of biological toxins first used for pest control. After years of study, Bt family is becoming bigger and bigger. Basically, most proteins in Bt family form pores in cell membrane to kill a cell, including cry34Ab1(2). 
    </P><P class="p1">
        In 2014, the structure of Cry34Ab1 was revealed and reported in PLOS ONE. As displayed in figure 1, the structure of Cry34Ab1 is simpler, compared with other two toxins we used. However, Cry34Ab1 (figure 2A) can only function with the assistant of Cry35Ab1 (figure 2B)(2), which differentiate CAb1 from Plu1537, which is toxic without any other assistant. 
    </P><DIV class="col-md-12 textcenter"><DIV class="cpleft"><P class="kuvateksti">
              Figure 1 Comparison between Cry34Ab1 and Cry35Ab1(2). Copyright 2014, Public Library of Science.
            </P></DIV></DIV><DIV><H3 class="hh2">Construction</H3><H4 class="hh4-left">PCR<P class="p1">
The CDS<I>plu1537</I> gene was amplified by PCR by two pieces to remove one enzyme site EcoRI in its original squnce. The template is genomic DNA extracted from strain <I>Photorhabdus luminescens</I> TT01.     </P><P class="p1">
We use primer plu1537-left F and plu1537-left R to amplify the left side of gene, which are shown below.	 </P></H4><H4 class="hh4-left">Seamless assembly<P class="p1">
We used seamless assembly as our assembly method so restriction digestion and T4 ligation can be avoided. Detailed protocol and instruction for primer design can be seen in our Protocol.By this way prefix sequence, <I>plu1537</I> and suffix sequence can be ligated seamlessly. 



    </P><P class="p1">
plu1537 F(F, 5’-3’): AATTCGCGGCCGCTTCTAGATGTCAGAGATCGAAGTAAT </P><P class="p1">
plu1537 R(R, 5’-3’): CGGACTGCAGCGGCCGCTACTAGTATTATTAGCTTACAATCATATATTC    </P></H4><H4 class="hh4-left">Transformation and confirmation<P class="p1">
After seamless assembly, standard plasmid pSB1C3 containing <I>plu1537</I> gene was transformed into <I> E.coli</I> DH5α. When single colony appeared on the LB plate, we picked out 10 colonies, respectively, as our template for bacteria solution PCR. In order to avoid the appearance of false positive clones, we used VF2/VR as the universal primers. The positive clone and its corresponding raw bacteria solution were stored and samples were sent to do DNA sequencing.   </P></H4><H4 class="hh4-left">Plasmid map<DIV class="col-md-12 textcenter"><DIV class="cpleft"><P class="kuvateksti">
                Fig.2 The plasmid map of BBa_K1668007 CDS<I>plu1537</I></P></DIV></DIV></H4><H4 class="hh4-left" style="padding-top:30px" style="padding-top:30px">Results</H4><DIV class="col-md-6 text-col-left"><H5 class="hh2"> PLASIMD CONSTRUCTION</H5><P class="p1">
                The template we use is recombinant plasmids. 
           </P><P class="p1">
                It can be clearly seen the recombinants shown in figure 3 is positive cloning. 

           </P></DIV><DIV class="col-md-6 img-100"><DIV class="cpleft"><P class="kuvateksti">
                Figure 2 PCR confirmation results of CDS plu1537 with standard primers VF2 and VR.
            </P></DIV></DIV></DIV><H4 class="hh4-left" style="padding-top:30px" style="padding-top:30px"> DNA Sequencing </H4><P class="p1">
      We have sequenced the parts with standard primers VF2 and VR. The sequence of the 1.8k part shows 100% agreement with the desired sequence. 

    </P><H3 class="hh2">Reference</H3><P class="p1">
1. M. Li et al., MOL BIOL REP 36, 785 (2009).
2. M. S. Kelker et al., PLOS ONE 9, (2014).
3. cry34Ab1(uniprot): http://www.uniprot.org/uniprot/Q939T0 
4. 3D structure of cry34Ab1: http://www.ebi.ac.uk/pdbe/entry/pdb/4JOX     </P></DIV></ARTICLE><ARTICLE><H3 class="hh2" style="border:0;color:red" id="pos4">BBa_K1668001: metK</H3><H3 class="hh2">Overview<P class="p1">
      The part metK is the coding sequence of S-adenosylmethionine synthetase in Streptomyces avermitilis. It was found to stimulate the production of avermectins, one kind of pesticide. 
    </P><P class="p1">
      This gene sequence could not function in E.coli. If you would like to express metK in Streptomyces avermitilis, remember to add ermEp <A href="http://parts.igem.org/Part:BBa_K1668004">(BBa_K1668004)</A> as its promoter.
    </P></H3><H3 class="hh2">Background<P class="p1">
        metK is the gene encoding S-adenosylmethionine synthetase, which has been found in almost every organism. Its output catalyzes the formation of S-adenosylmethionine from methionine and ATP. 
    </P></H3><H4 class="hh4-left">Function<P class="p1">
      In Streptomyces avermitilis, metK was found to stimulate the production of avermectins, a kind of pesticide. When wild-type S. avermitilis strain ATCC31267 was transformed with pYJ02 and pYJ03, two metK expression plasmids, avermectin production was increased about 2.0-fold and 5.5-fold compared with that in the control strains, respectively. (1)
    </P></H4><H4 class="hh4-left">Principle<P class="p1">
        As for the principle of improving the productivity, instead of changing cell growth or copy effect, metK stimulates the avermectin production by increasing the intracellular concentration of S-adenosylmethionine (SAM), an important intermediate product in avermectin production. However, there may be a maximum concentration of SAM for the production of avermectin in S. avermitilis, which means that SAM has no effect when its concentration achieve maximum. 
    </P></H4><H4 class="hh4-left">Limitation<P class="p1">
          The results of experiments in research paper showed that different metK expression levels have different influence on avermectin production in various S. avermitilis strains. The gene expression levels of metK in two engineered strain, GB-165 and 76-05, were much higher than those in wild-type strain, whereas the avermectin productivity in these two strains have not been significantly improved(1). It is probably because the high expression level of metK in engineered strains limited the improvement of avermectin productivity by overexpression of metK. 


    </P></H4><H4 class="hh4-left">Protein<P class="p1">
      The 3D structure of S-adenosylmethionine synthetase is as follows. This enzyme catalyzes the formation of S-adenosylmethionine from methionine and ATP and is involved in step1 of the subpathway that synthesizes S-adenosyl-L-methionine from L-methionine.
    </P><DIV class="row"><DIV class="col-md-12 textcenter"><DIV class="cpleft"><P class="kuvateksti">
                Fig.1 the 3D structure of S-adenosylmethionine synthetase (Uniprot #: Q3HW35)
              </P></DIV></DIV></DIV></H4><H3 class="hh2">Construction</H3><H4 class="hh4-left">PCR<P class="p1">
          The metK gene was amplified by PCR with genomic DNA extracted from S. avermitilis ATCC31267 strain as template. We commercially purchased this strain.
    </P><P class="p1">
          By PCR with primers metK1 and metK2 shown below, we added the standard prefix and suffix at both ends of the metK sequence. 
    </P></H4><H4 class="hh4-left">Seamless assembly<P class="p1">
      We used seamless assembly as our assembly method so restriction digestion and T4 ligation can be avoided. Detailed protocol and instruction for primer design can be seen in our <A href="https://2015.igem.org/Team:ZJU-China/Project/Protocol">Protocol</A>. By this way, prefix sequence, metK, and suffix sequence can be ligated seamlessly.
    </P><P class="p1">
      metK1 (F, 5’-3’): GAATTCGCGGCCGCTTCTAGATGTTCGGCTACGC
    </P><P class="p1">
      metK2 (R, 5’-3’): TGCAGCGGCCGCTACTAGTATTATTACAGCCCCACA
    </P></H4><H4 class="hh4-left">Transformation and confirmation<P class="p1">
      After seamless assembly, standard plasmid pSB1C3 containing metK gene was transformed into E.coli DH5α. When single colony appeared on the LB plate, we picked out 10 colonies, respectively, as our template for bacteria solution PCR. In order to avoid the appearance of false positive clones, we used VF2 and VR as the universal primers.
    </P><P class="p1">
      The positive clone and its corresponding raw bacteria solution were stored and samples were sent to do DNA sequencing.
    </P></H4><H4 class="hh4-left" style="font-size:30px">Plasmid map<DIV class="row"><DIV class="col-md-12 textcenter"><DIV class="cpleft"><P class="kuvateksti">
                Fig.2 The plasmid map of BBa_K1668001 metK
              </P></DIV></DIV></DIV></H4><H3 class="hh2">Results</H3><H4 class="hh4-left">Gel electrophoretic analysis<P class="p1">
      In Fig.3 (A), it is indicated that metK (816bp) has been successfully amplified by PCR. 
    </P><P class="p1">
      In Fig.3 (B), positive clones determined by bacteria solution PCR are indicated.
    </P><DIV class="row"><DIV class="col-md-12 textcenter"><DIV class="cpleft"><P class="kuvateksti">
                Fig.3 Gel electrophoretic analyses of PCR products (A) and selected examples of cloned products of seamless assembly reaction (B). (A) 5-μl samples of the PCR products for metK, (B) 5-μl samples of the bacteria solution PCR products were loaded onto a 1% BioRad Ready Agarose Mini Gel, then subjected to AGE. See (protocol) for AGE parameters. The DNA size standards was the DL1,000 DNA Marker (M1; Takara, Cat#3428A) and DL2,000 DNA Marker (M2; TaKaRa, Cat#3427A). Bands were visualized with a Shanghai Peiqing JS-380A Fluorescence Imager. PCR products, positive clones and negative clones are indicated. 
              </P></DIV></DIV></DIV></H4><H4 class="hh4-left">DNA sequencing<P class="p1">
      We have sequenced the parts with standard primers VF2 and VR. The sequence of the 816bp part shows 100% agreement with the desired sequence. 
    </P></H4><H3 class="hh2">Reference<P class="p1">
      X. Zhao, Q. Wang, W. Guo, Y. Cai, C. Wang, S. Wang, S. Xiang, Y. Song, Overexpression of metK shows different effects on avermectin production in various Streptomyces avermitilis strains. World journal of microbiology &amp; biotechnology 29, 1869-1875 (2013); published online EpubOct (10.1007/s11274-013-1350-0).
    </P></H3></ARTICLE><ARTICLE><H3 class="hh2" style="border:0;color:red" id="pos5">BBa_K1668002: frr</H3><H3 class="hh2">Overview<P class="p1">
      The part frr is the coding sequence of ribosome recycling factor (RRF) in Streptomyces avermitilis. It was found to promote cell growth and stimulate the production of avermectins, one kind of pesticide. 
    </P><P class="p1">
      This gene sequence could not function in E.coli. If you would like to express frr in Streptomyces avermitilis, remember to add ermEp <A href="http://parts.igem.org/Part:BBa_K1668004">(BBa_K1668004)</A> as its promoter.
    </P></H3><H3 class="hh2">Background</H3><H4 class="hh4-left">Function<P class="p1">
          frr gene encodes the ribosome recycling factor (RRF), which is involved in the release of ribosomes from the translational post-termination complex for a new round of initiation. RRF may increase the efficiency of translation by recycling ribosomes from one round of translation to another. Avermectin yield was increased significantly by 3- to 3.7-fold in transformants 31267(pFRR-1139) and 31267(pFRRerm-1139), compared with that in the wild-type strains and both of the transformants contained multiple frr copies.(1)  The avermectin productivity of each culture was quantitatively measured by HPLC analysis.
    </P></H4><H4 class="hh4-left">Principle<P class="p1">
      Research indicated that frr overexpression promoted cell growth as well as the expression of ave genes (including pathway-specific positive regulatory gene aveR for avermectin biosynthesis and ave structural genes), leading in turn to avermectin overproduction. Different from S-adenosylmethionine synthetase gene (metK), frr gene revealed a ‘‘copy number effect’’. That is to say, multiple copies of frr had a greater promoting effect on avermectin production than a single copy does. However, the detailed mechanism of frr enhancing antibiotic production remains to be clarified.
    </P></H4><H4 class="hh4-left">Limitation<P class="p1">
      Compared with the wild-type strain, the effect of frr on avermectin production in engineered strains 76-02-e and GB-165 was less obvious, probably because most of the negative stimulatory factors are downregulated and most of the positive factors are upregulated, resulting in relatively limited potential for further improvement of avermectin yield.(1)
    </P></H4><H4 class="hh4-left">Protein<P class="p1">
      The 3D structure of ribosome recycling factor is as above (Fig.1). This factor is responsible for the release of ribosomes from messenger RNA at the termination of protein biosynthesis. and may increase the efficiency of translation by recycling ribosomes from one round of translation to another. 
    </P><DIV class="col-md-12 textcenter"><DIV class="cpleft"><P class="kuvateksti">
                Fig.1 The 3D structure of ribosome recycling factor (Uniprot #: Q82JY0)
              </P></DIV></DIV></H4><H3 class="hh2">Construction</H3><H4 class="hh4-left">PCR<P class="p1">
        The frr gene was amplified by PCR with genomic DNA extracted from S. avermitilis ATCC31267 strain as template. We commercially purchased this strain. By PCR with primers frr1 and frr2 shown below, we added the standard prefix and suffix at both ends of the frr sequence.
    </P></H4><H4 class="hh4-left">Seamless assembly<P class="p1">
      We used seamless assembly as our assembly method so restriction digestion and T4 ligation can be avoided. Detailed protocol and instruction for primer design can be seen in our <A href="https://2015.igem.org/Team:ZJU-China/Project/Protocol">Protocol</A>. By this way, prefix sequence, metK, and suffix sequence can be ligated seamlessly. 
    </P><P class="p1">
      frr1 (F, 5’-3’): GAATTCGCGGCCGCTTCTAGATGCGCGGGTACGTC 
    </P><P class="p1">
      frr2 (R, 5’-3’): TGCAGCGGCCGCTACTAGTATTATTACATCAAGGTCGCC
    </P></H4><H4 class="hh4-left">Transformation and confirmation<P class="p1">
      After seamless assembly, standard plasmid pSB1C3 containing frr gene was transformed into E.coli DH5α. When single colony appeared on the LB plate, we picked out 10 colonies, respectively, as our template for bacteria solution PCR. In order to avoid the appearance of false positive clones, we used VF2/VR as the universal primers. The positive clone and its corresponding raw bacteria solution were stored and samples were sent to do DNA sequencing.
    </P></H4><H4 class="hh4-left">Plasmid map<DIV class="col-md-12 textcenter"><DIV class="cpleft"><P class="kuvateksti">
                Fig.2 The plasmid map of BBa_K1668002 frr
              </P></DIV></DIV></H4><H3 class="hh2">Results</H3><H4 class="hh4-left">Gel electrophoretic analysis<DIV class="col-md-12 textcenter"><DIV class="cpleft"><P class="kuvateksti">
                Fig.3 Gel electrophoretic analyses of PCR products (A) and selected examples of cloned products of seamless assembly reaction (B). (A) 5-μl samples of the PCR products for frr, (B) 5-μl samples of the bacteria solution PCR products were loaded onto a 1% BioRad Ready Agarose Mini Gel, then subjected to AGE. See (protocol) for AGE parameters. The DNA size standards was the DL1,000 DNA Marker (M; Takara, Cat#3428A). Bands were visualized with a Shanghai Peiqing JS-380A Fluorescence Imager. PCR products, positive clones and negative clones are indicated. 
              </P></DIV></DIV></H4><H4 class="hh4-left">DNA sequencing<P class="p1">
      We have sequenced the parts with standard primers VF2 and VR. The sequence of the 384bp part shows 100% agreement with the desired sequence. 
    </P></H4><H3 class="hh2">Reference</H3><P class="p1">
          L. Li, J. Guo, Y. Wen, Z. Chen, Y. Song, J. Li, Overexpression of ribosome recycling factor causes increased production of avermectin in Streptomyces avermitilis strains. Journal of industrial microbiology &amp; biotechnology 37, 673-679 (2010); published online EpubJul (10.1007/s10295-010-0710-0).
    </P></ARTICLE><ARTICLE><H3 class="hh2" style="border:0;color:red" id="pos6">BBa_K1668003: orfX</H3><H3 class="hh2">Overview<P class="p1">
      The part orfX is a putative membrane-bound putative regulatory gene and its product is a putative membrane protein. 
    </P><P class="p1">
      This gene sequence could not function in E.coli. If you would like to express frr in Streptomyces avermitilis, remember to add ermEp <A href="http://parts.igem.org/Part:BBa_K1668004">(BBa_K1668004)</A> as its promoter.
    </P></H3><H3 class="hh2">Background</H3><H4 class="hh4-left">Function<P class="p1">
      The results of PCR analysis and the gene disruption experiments strongly suggest that either a considerably conserved sequence of orfX or a combination of orfX and other assisting genes exists in the high producers. And the orfX product appears to play an essential role in the production and regulation of avermectin in both the normal strain and the high producers. When wild-type S. avermitilis was transformed with a 8.0-kb DNA fragment containing the orfX gene, avermectin production increased approximately 3.5-fold.(1) However, the nature of the stimulatory effect of orfX is still unclear.
    </P></H4><H4 class="hh4-left">Principle<P class="p1">
      The orfX gene reveals a “copy number effect”. That is to say, multiple fragment copies can substantially increase avermectin production in S. avermitilis. Different from metK and frr gene, the DNA fragment containing orfX gene also increased avermectin bio-synthesis in various S. avermitilis strains, including the high-producing mutant strain ATCC 31780 and a semi-industrial strain L-9. (1) 
    </P></H4><H3 class="hh2">Construction</H3><H4 class="hh4-left">PCR<P class="p1">
      The orfX gene was amplified by PCR with genomic DNA extracted from S. avermitilis ATCC31267 strain as template. We commercially purchased this strain. By PCR with primers orfX1 and orfX2 shown below, we added the standard prefix and suffix at both ends of the metK sequence.
    </P></H4><H4 class="hh4-left">Seamless assembly<P class="p1">
      We used seamless assembly as our assembly method so restriction digestion and T4 ligation can be avoided. Detailed protocol and instruction for primer design can be seen in our <A href="https://2015.igem.org/Team:ZJU-China/Project/Protocol">Protocol</A>. By this way, prefix sequence, metK, and suffix sequence can be ligated seamlessly. 
    </P><P class="p1">
      orfX1 (F, 5’-3’): GAATTCGCGGCCGCTTCTAGATGGTGAGCGCCT 
    </P><P class="p1">
      orfX2 (R, 5’-3’): TGCAGCGGCCGCTACTAGTATTATTATCTGCGGTCC
    </P></H4><H4 class="hh4-left">Transformation and confirmation<P class="p1">
      After seamless assembly, standard plasmid pSB1C3 containing orfX gene was transformed into E.coli DH5α. When single colony appeared on the LB plate, we picked out 10 colonies, respectively, as our template for bacteria solution PCR. In order to avoid the appearance of false positive clones, we used VF2/VR as the universal primers. The positive clone and its corresponding raw bacteria solution were stored and samples were sent to do DNA sequencing.
    </P></H4><H4 class="hh4-left">Plasmid map<DIV><DIV class="col-md-12 textcenter"><DIV class="cpleft"><P class="kuvateksti">
                Fig.1 The plasmid map of BBa_K1668003 orfX
              </P></DIV></DIV></DIV></H4><H3 class="hh2">Results</H3><H4 class="hh4-left">Gel electrophoretic analysis<P class="p1">
      In Fig.2 (A), it is indicated that orfX (942bp) has been successfully amplified by PCR. 
    </P><P class="p1">
      In Fig.2 (B), positive clones determined by bacteria solution PCR are indicated.
    </P><DIV><DIV class="col-md-12 textcenter"><DIV class="cpleft"><P class="kuvateksti">
                Fig.2 Gel electrophoretic analyses of PCR products (A) and selected examples of cloned products of seamless assembly reaction (B). (A) 5-μl samples of the PCR products for orfX, (B) 5-μl samples of the bacteria solution PCR products were loaded onto a 1% BioRad Ready Agarose Mini Gel, then subjected to AGE. See (protocol) for AGE parameters. The DNA size standards was the DL1,000 DNA Marker (M1; Takara, Cat#3428A) and DL2,000 DNA Marker (M2; TaKaRa, Cat#3427A). Bands were visualized with a Shanghai Peiqing JS-380A Fluorescence Imager. PCR products, positive clones and negative clones are indicated. 
              </P></DIV></DIV></DIV></H4><H4 class="hh4-left">DNA sequencing<P class="p1">
      We have sequenced the parts with standard primers VF2 and VR. The sequence of the 942bp part shows 100% agreement with the desired sequence. 
    </P></H4><H3 class="hh2">Reference</H3><P class="p1">
        Y. S. Hwang, E. S. Kim, S. Biro, C. Y. Choi, Cloning and Analysis of a DNA Fragment Stimulating Avermectin Production in Various Streptomyces avermitilis Strains. Applied and environmental microbiology 69, 1263-1269 (2003)10.1128/aem.69.2.1263-1269.2003).
    </P></ARTICLE><ARTICLE><H3 class="hh2" style="border:0;color:red" id="pos7">BBa_K1668004 ermEp</H3><H3 class="hh2">Overview<P class="p1">
      ermEp is a strong constitutive promoter in various S. avermitilis strains. It should be noticed that ermEp can only be expressed in S.avermitilis strains instead of Escherichia coli or any other chassis.
    </P></H3><H3 class="hh2">Background<P class="p1">
          The ermE promoter was originally characterised by cloning the entire putative promoter region upstream of the ermE gene of Saccharopolyspora erythraea in front of a kanamycin resistance gene (neo) in a replicative vector in Streptomyces lividans TK24.
    </P><P class="p1">
      The ermE promoter region contains two different promoters, ermEp1 and ermEp2. It was reported that a TGG deletion in the 35 region of the ermEp1 promoter resulted in a stronger variant called ermE* (ermEp2 and ermEp1 ΔTGG).(1)  The promoter strength was indirectly assessed according to the enzymatic activity of the reporter protein GUS. The ermE and ermE* promoters were approximately 1.8 times stronger than the ermEp1 and ermEp1* promoters. However, no significant difference was detected between the strengths of the native ermE promoter and its variant ermE* or between the ermEp1 and the ermEp1* promoter.(2)
    </P></H3><H3 class="hh2">Construction</H3><H4 class="hh4-left">PCR<P class="p1">
      The ermEp gene was amplified by PCR with genomic DNA extracted from S. avermitilis ATCC31267 strain as template. We commercially purchased this strain. By PCR with primers ermEp1 and ermEp2 shown below, we added the standard prefix and suffix at both ends of the metK sequence.
    </P></H4><H4 class="hh4-left">Seamless assembly<P class="p1">
      We used seamless assembly as our assembly method so restriction digestion and T4 ligation can be avoided. Detailed protocol and instruction for primer design can be seen in our <A href="https://2015.igem.org/Team:ZJU-China/Project/Protocol">Protocol</A>. By this way, prefix sequence, metK, and suffix sequence can be ligated seamlessly. 
    </P><P class="p1">
      ermEp1 (F, 5’-3’): ATTCGCGGCCGCTTCTAGAGGGCGGCTTGCGCC 
    </P><P class="p1">
      ermEp2 (R, 5’-3’): TGCAGCGGCCGCTACTAGTATACCAACCGGCACGAT
    </P></H4><H4 class="hh4-left">Transformation and confirmation<P class="p1">
      After seamless assembly, standard plasmid pSB1C3 containing ermEp gene was transformed into E.coli DH5α. When single colony appeared on the LB plate, we picked out 10 colonies, respectively, as our template for bacteria solution PCR. In order to avoid the appearance of false positive clones, we used VF2/VR as the universal primers. The positive clone and its corresponding raw bacteria solution were stored and samples were sent to do DNA sequencing.
    </P></H4><H4 class="hh4-left">Plasmid map<DIV><DIV class="row"><DIV class="col-md-12 textcenter"><DIV class="cpleft"><P class="kuvateksti">
                Fig.1 The plasmid map of BBa_K1668004 ermEp
              </P></DIV></DIV></DIV></DIV></H4><H3 class="hh2">Results</H3><H4 class="hh4-left">Gel electrophoretic analysis<P class="p1">
      In Fig.2 (A), it is indicated that ermEp (155bp) has been successfully amplified by PCR. In Fig.2 (B), positive clones determined by bacteria solution PCR are indicated.
    </P><DIV><DIV class="row"><DIV class="col-md-12 textcenter"><DIV class="cpleft"><P class="kuvateksti">
                Fig.2 Gel electrophoretic analyses of PCR products (A) and selected examples of cloned products of seamless assembly reaction (B). (A) 5-μl samples of the PCR products for ermEp, (B) 5-μl samples of the bacteria solution PCR products were loaded onto a 1% BioRad Ready Agarose Mini Gel, then subjected to AGE. See (protocol) for AGE parameters. The DNA size standards was the DL1,000 DNA Marker (M; Takara, Cat#3428A). Bands were visualized with a Shanghai Peiqing JS-380A Fluorescence Imager. PCR products, positive clones and negative clones are indicated. 
              </P></DIV></DIV></DIV></DIV></H4><H4 class="hh4-left">DNA sequencing<P class="p1">
      We have sequenced the parts with standard primers VF2 and VR. The sequence of the 155bp part shows 100% agreement with the desired sequence. 
    </P></H4><H3 class="hh2">Reference</H3><P class="p1">
      1.  T. Siegl, B. Tokovenko, M. Myronovskyi, A. Luzhetskyy, Design, construction and characterisation of a synthetic promoter library for fine-tuned gene expression in actinomycetes. Metabolic engineering 19, 98-106 (2013); published online EpubSep (10.1016/j.ymben.2013.07.006).
    </P><P class="p1">
      2.  M. J. Bibb, J. White, J. M. Ward, G. R. Janssen, The mRNA for the 23S rRNA methylase encoded by the ermE gene of Saccharopolyspora erythraea is translated in the absence of a conventional ribosome-binding site. Molecular microbiology 14, 533-545 (1994); published online EpubNov 
</P></ARTICLE><ASIDER id="popView2"><HEADER>Basic parts</HEADER><UL id="nav2"><LI><A href="#pos4">BBa_K1668001: metK	</A></LI><LI><A href="#pos5">BBa_K1668002: frr		</A></LI><LI><A href="#pos6">BBa_K1668003: orfX	</A></LI><LI><A href="#pos7">BBa_K1668004: ermEp</A></LI><LI><A href="#pos1">BBa_K1668005: tcdA1	</A></LI><LI><A href="#pos2">BBa_K1668006: plu0840	</A></LI><LI><A href="#pos3">BBa_K1668007: plu1537	</A></LI></UL></ASIDER></DIV><DIV id="gl-footer" class="cf" style="max-width:1440px"><DIV id="gl-footer-copyright" class="cf"><DIV class="gl-col1 cf"><SPAN>© 2015 ZJU-IGEM TERMINATE.</SPAN></DIV><UL class="gl-col2 cf"><LI><A href="http://www.igem.org/Main_Page">IGEM</A></LI><LI><A href="http://www.zju.edu.cn">Zhejiang University</A></LI></UL></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>