Team:ZJU-China/Description
Home
Team Part Team Part
Basic Part Basic Part
Improved  Part Improved  Part
Composite Part Composite Part
Parts
Overview Quick view for our project Termite Issue The Termite Issue
Design
Toxins Manufacture
Bacteria Carrier
Termites
Protocol Quick view for Protocol
Project
Overview Quick view for Outreach
Society
Survey
Consultation
Syn-bio Debate
Application
Device
Promotion
Safety
Future
Public Promotion
Poly-cards
Syn-bio Share
Community Lab
Seminar
Collaboration
SynenergeneⅠ
SynenergeneⅡ
Synenergene Ⅲ
Other Teams
Outreach
Members Introduce our Members
Attributions Introduce the Attributions
Acknowledgement Introduce the Acknowledgement
Sponsors Introduce our Sponsors
Team
Modeling Overview Introduce our Members
Modeling α Termite Simulation
Modeling β Mass transfer model
Modeling γ Degradation model
Modeling
Notebook
Experiment Results toxins manufacture
Experiment Result bacteria carrier
Experiment Result termites simulation
Social Result social results
Results
Safety
Device Introduce our Device and Instruction Manual
Promotion The Promotion
Safety The Safety
Future Future
Application
BBa_K1668001 metk
Preface
The part BBa_K417000 just briefly introduced metK and its output SAMsynth. The specific function and principle of SAMsynth (S-adenosylmethionine synthetase) isn't mentioned. Furthermore, the 3D structure and other detailed information isn't included, which may bring much inconvenience to potential users. Thus, connected with our project, we added plenty of related information of metK and sincerely wish to give a hand to future users.
Summary
1. Characterize the output of metK in a novel chassis: Streptomyces avermitilis
2. Create a visualization of S-adenosylmethionine synthetase
3. Add the 2D PAGE maps for the output of metK in E.coli
Part 1 Characterization
——the output of metK in Streptomyces avermitilis
Background
metK: S-adenosylmethionine synthetase encoding gene
metK is the gene encoding S-adenosylmethionine synthetase, which has been found in almost every organism. Its output catalyzes the formation of S-adenosylmethionine from methionine and ATP.
In Streptomyces avermitilis, it was found to stimulate the production of avermectins. When wild-type S. avermitilis strain ATCC31267 was transformed with pYJ02 and pYJ03, two metK expression plasmids, avermectin production was increased about 2.0-fold and 5.5-fold compared with that in the control strains, respectively.
As for the principle of improving the productivity, instead of changing cell growth or copy effect, metK stimulates the avermectin production by increasing the intracellular concentration of S-adenosylmethionine (SAM), an important intermediate product in avermectin production. However, there may be a maximum concentration of SAM for the production of avermectin in S. avermitilis, which means that SAM has no effect when its concentration achieve maximum.
The results of experiments in research paper showed that different metK expression levels have different influence on avermectin production in various S. avermitilis strains. The gene expression levels of metK in two engineered strain, GB-165 and 76-05, were much higher than those in wild-type strain, whereas the avermectin productivity in these two strains have not been significantly improved.  It is probably because the high expression level of metK in engineered strains limited the improvement of avermectin productivity by overexpression of metK.
Fig.1 The picture of Streptomyces avermitilis under scanning electron microscope (SEM). Copyright all reserved ZJU-China 2015
Host of avermectin——Streptomyces avermitilis
Streptomyces avermitilis, a soil-dwelling gram-positive microorganism, is a rich source of numerous secondary metabolites. Now it has been industrialized to produce the commercially important antiparasitic agent avermectin. Early in 2003, the complete genome of Streptomyces avermitilis had been sequenced.
In past years, scientists had been trying to transform gene into S.avermitilis. Until 1989, gene transformation in S.avermitilis was achieved through conjugation between E.coli strains(eg, s17-1) and S.avermitilis. However, the efficiency was limited by the methyl-specific restriction system in S.avermitilisi, which show strong restriction to gene methylated in normal E.coli strains. Eventually, high efficiency conjugation was achieved till the introduction of methylase-negative donor strain E.coli ET12567. Now conjugation and strain ET12567 has been ubiquitously adopted in the gene transformation of S.avermitilis.
avermectin: effective and broad-spectrum pesticide
Fig.2 The structure of avermectin and avermectin 1 Copyright 2006, Springer Verlag
For years, people always adopt the organochlorine pesticides such as chlordane and mirex to achieve prevention and control of termites, but these organochlorine pesticides will produce pollution and potential harm to the environment. Avermectin is a new type of high efficient biological pesticide, which has good control effect to the termites and other pests, and no pollution to the environment.
Experiment: Avermectin manufacture in S. avermitilis
Purpose
We attempt to test the function of metK gene by improving the yield of avermectin in S. avermitilis. Because as it has been described in research paper, metK was found to stimulate the production of avermectins. For one thing, being a secondary metabolite produced by Streptomyces avermitilis, avermectin is regulated by an 80kb gene cluster, making it difficult to express in other standardized strains, for instance, Escherichia coli. For another, the avermectin yield in wild type S. avermitilis strain is comparatively low. Therefore, we plan to engineer the wild S. avermitilis with metK to improve the yield of avermectin.
Fig.3 the circuits constructed for yield improvement of avermectin in S.avermitilis.
Circuit design
promoter: ermEp
We chose ermEp, a strong constitutive promoter, to overexpress the three genes in S.avermitilis. It should be noticed that ermEp can only be expressed in S.avermitilis strains instead of Escherichia coli or any other chassis.
backbone: PL96 and PL97
PL96 and PL97 are two high-copy vectors we used to overexpress our target genes. We get these vectors through commercial purchase. These vectors have pUC18 and pIJ101 replication origins for high-copy plasmid number in Escherichia coli and S.avermitilis, respectively, and the oriT (RK2) allows the efficient and convenient plasmid transfer from E.coli to S.avermitilis.
Fig.4 The plasmid map for PL96.
Fig.5 The plasmid map for PL97.
To be noticed, we use special antibiotic aparamycin to choose final transformants. And there are aparamycin resistent gene acc in the backbone.
expression
In order to construct and express the three gene in S.avermitilis, we have adopted two hosts, E.coli DH5αand E.coli ET12567. Then the target vectors are transferred from E.coli ET12567 to S.avermitilis by conjugation.
primary host: E.coli DH5α
As usual, we use E.coli DH5α to get plentiful recombinants in high quality and quantity.
intermedia host: E.coli ET12567
E.coli ET12567 is a methylase-negative donor strain first used by MacNeil in 1988. And we use E.coli ET12567 to demethylation the recombinants to better suit the methyl-specific restriction system in S.avermitilis.
conjugation
Bacterial conjugation is the transfer of genetic material between bacterial cells by direct cell-to-cell contact or by a bridge-like connection between two cells. During conjugation the donor cell provides a conjugative or mobilizable genetic element that is most often a plasmid or transposon. In laboratories, successful transfers have been reported from bacteria to yeast, plants, mammalian cells, etc.
Fig.6 The sketch map of conjugation between E.coli ET12567 and S.avermitilisi.
In our project, we use the conjugation between E.coli ET12567 and S.avermitilisi to overexpress three target genes.
Circuit Construction
STEP ONE: PCR
We amplify the target gene from the genome of S.avermitilisi ATCC13267 strain by PCR.
STEP TWO: TA CLONING
We use TA cloning to efficiently clone the PCR products. In TA cloning, we use pMD19-T Vector, a vector transformed from pUC19 vector, to improve the efficiency of digestion and connection. As a result, we get three recombinant vectors of target genes and pMD19-T.
STEP THREE: DIGESTION AND LIGATION
We digest the three recombinants and backbone PL96 with restriction enzymes NdeI, XbaI, then connect the fragments and backbone. Similarly, we use NdeI, HindⅢ to digest the three recombinants and backbone PL97 and connect the corresponding product. Then we get the target plasmids.
Fig.7 digestion and ligation in PL96.
Fig.8 digestion and ligation in PL97.
For more detailed protocols, please go to Protocol .
Result
Gel electrophoretic analysis
We successfully constructed the plasmid (PL96 and PL97) containing metK (sequence is shown in the part BBa_K1668001) gene in E.coli DH5α and E.coli ET12567. Then we transformed it into S.avermitilis by conjugation.
Fig.9 Gel electrophoretic analyses of PCR products (A) and double enzyme digestion products (B and C). (A) 5-μl samples of the PCR products for metK, (B and C) 5-μl samples of the double enzyme digestion products were loaded onto a 1% BioRad Ready Agarose Mini Gel, then subjected to AGE. See ( Protocol ) for AGE parameters. (B and C) Sizes of the NdeI and XbaI–cleaved assemblies were determined by AGE analysis. The DNA size standards was the DL2,000 DNA Marker (M1; TaKaRa, Cat#3427A) and 1kb DNA Ladder (Dye Plus) (M2; TaKaRa, Cat#3426A). Bands were visualized with a Shanghai Peiqing JS-380A Fluorescence Imager.
Part 2 Visualization
——S-adenosylmethionine synthetase in E.coli
We have found the 3D molecular graphic (Fig.2) and interaction figure (Fig.3) of S-adenosylmethionine synthetase in NCBI database.
Fig.10 3D molecular graphic of S-adenosylmethionine synthetase (MMDB ID: 52023)
Fig.11 the interaction figure of S-adenosylmethionine synthetase (MMDB ID: 52023 ). In the figure, ○ represents protein and ◇ represents chemical. The rhombus marked 1,2 and 3 represents phosphate ion, Co and K+, respectively.
Part 3 2DPAGE map
——the output of metK in E.coli
We have found the 2DPAGE maps for S-adenosylmethionine synthetase in E.coli. You can click on a highlighted spot in the figure from the website listed below to access all the associated protein entries of the spot.
Fig.12 2DPAGE maps for S-adenosylmethionine synthetase in E.coli (SWISS-2DPAGE: P0A817)
Reference
http://www.ncbi.nlm.nih.gov/Structure/mmdb/mmdbsrv.cgi?uid=52023
http://world-2dpage.expasy.org/swiss-2dpage/viewer&map=ECOLI4.5-5.5&ac=P0A817
X. Zhao, Q. Wang, W. Guo, Y. Cai, C. Wang, S. Wang, S. Xiang, Y. Song, Overexpression of metK shows different effects on avermectin production in various Streptomyces avermitilis strains. World journal of microbiology & biotechnology 29, 1869-1875 (2013); published online EpubOct (10.1007/s11274-013-1350-0).
BBa_K1668011 mCherry
:
Preface
:
There is little message about the characterrization and features of mCherry in old pages. In addition, the quality of parts varied from year to year. With an extra promoter in it, the sequnce of mCherry in 2015 (kit plate 3, well 15B) is incrrect according to the sequncing result. And mCherry in 2012 (kit plate 2, well 8E) is in a "pSB1A2" backbone, which is of 4k size and doesn't correspond with message of pSB1A2 in Part Registry. So we amplified the correct gene, assembled it with standardize pSB1C3 backbone, confirmed the sequnce by enzyme digestion and sequncing and submitted the part BBa_K1668011.
Summary
:
1. sequncing of parts in previous year.
2. correct the sequnce and resemble it in a standard backbone BBa_K16680111
3. add more features about the mCherry.
Background
We used mCherry (BBa_J06702) as a reporter, which is one registry star and therefore comparably more reliable. According to the message in the Registry, there is no promoter in the part therefore the reporter should not be expressed(Figure 1). But when we transformed the mCherry gene into E.coli DH5α, the E.coli turned out to become red. With the colorless control, we concluded that the mCherry was expressed and sequnced the parts. According to the results of the sequncing, the part has a promoter with it (Figure 2).
Figure 1 Information of mCherry (BBa_J06702) in Part Registry.
Figure 2 Sequencing results of 2015 mCherry in 2015 kit plate 3, well 15B.
hen we got the same part from kit plate in 2013 and 2012. At last, we used mCherry of 2012 in backbone pSB1A2, which is a 2k backbone. But after we have cut the backbone with XbaI and SpeI, we found that the backbone was as big as 4k (figure 3). So we had to sequnce the mCherry of 2012, which turned out to be correct (figure 4). And the backbone had an anti-ampicillin gene with it.
Figure 3 Double enzyme digestion of the 2012mCherry (BBa_J06702) in 2012 kit plate 2, well 8E.
Figure 4 Sequencing results of 2012 mCherry in 2012 kit plate 2, well 8E.
Source
The mCherry gene was amplified by PCR with the template genomic DNA extracted from strain Photorhabdus luminescens TT01. We got the strain from Shandong University.
Design Notes
PCR
The mCherry gene was amplified by PCR with the template genomic DNA extracted from strain Photorhabdus luminescens TT01. We use primer mCherry -left F and mCherry R to amplify the left side of gene, which are shown below.
Seamless assembly
We used seamless assembly as our assembly method so restriction digestion and T4 ligation can be avoided. By this way prefix sequence, mCherry and suffix sequence can be ligated seamlessly.  Detailed protocol and instruction for primer design can be seen in our Protocol .
mCherry F(F, 5’-3’): AGAAAGAGGAGAAATACTAGATGGTGAG
mCherry R(R, 5’-3’): CCGGACTGCAGCGGCCGCTACTAGTATAAACGCAGAAAGGCC
Transformation and confirmation
After seamless assembly, standard plasmid pSB1C3 containing mCherry gene was transformed into E.coli DH5α. When single colony appeared on the LB plate, we picked out 10 colonies, respectively, as our template for bacteria solution PCR. In order to avoid the appearance of false positive clones, we used VF2/VR as the universal primers. The positive clone and its corresponding raw bacteria solution were stored and samples were sent to do DNA sequencing.
Plasmid map
Fig.1 The plasmid map of BBa_K1668011 mCherry
Results
Gel electrophoretic analysis
5-μl samples of the double enzyme digestion products for improved mCherry BBa_K1668011 were loaded onto a 1% BioRad Ready Agarose Mini Gel, then subjected to AGE. See ( Protocol ) for AGE parameters. Sizes of the XbaI and PstI–cleaved assemblies were determined by AGE analysis. The DNA size standards were 1kb DNA Ladder (Dye Plus)(M2; TaKaRa, Cat#3426A). Bands were visualized with a Shanghai Peiqing JS-380A Fluorescence Imager. Digested plasmid backbone and mCherry fragment are indicated.
It can be clearly seen that the 900bp mCherry is in the right position in agarose gel.
Figure 5 double enzyme digestion of the improved mCherry.
DNA sequencing
We have sequenced the parts with standard primers VF2 and VR. The sequence of the 155bp part shows 100% agreement with the desired sequence.
EXPRESSION
We serial express our three toxins (TcdA1, Plu1537 and Plu0840) with MCherry, shown in figure 6. The redness of all three devices indicates that mCherry functions well.
Figure 6 Tandem expression of toxin protein and MCherry shown in pipet tube. 1ml bacterium solution was added in each pipet tube and centrifuged in a speed of 12000 rpm of for 2min.
Improved parts
BBa_K1668001
metk
Part 1
Part 2
Part 3
BBa_K1668011
mCherry
© 2015 ZJU-IGEM TERMINATE.
IGEM
Zhejiang University
