<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_ZJU-China_Description skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:ZJU-China/Description</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="gl-outer-wrapper" class="gl-zh cf" style=""><DIV id="gl-header" class="cf gl-header-fixed" style="max-width:1440px"><DIV id="gl-header-bottom" class="cf"><DIV id="gl-menu" class="cf"><DIV id="gl-menu-one" class="gl-has-submenu"><DIV id="gl-submenu-one" class="gl-submenu"><DIV class="gl-col-footer" style="border-top:0px solid #aaa;"><A href="https://2015.igem.org/Team:ZJU-China">Home</A></DIV></DIV></DIV><DIV id="gl-menu-two" class="gl-has-submenu"><DIV id="gl-submenu-three" class="gl-submenu"><DIV class="gl-col-wrapper"><DIV class="gl-col gl-col1"><A href="https://2015.igem.org/Team:ZJU-China/Parts" class="gl-hover"><SPAN>Team Part</SPAN><SPAN>Team Part</SPAN></A></DIV><DIV class="gl-col gl-col2"><A href="https://2015.igem.org/Team:ZJU-China/Basic_Part" class="gl-hover"><SPAN>Basic Part</SPAN><SPAN>Basic Part</SPAN></A></DIV><DIV class="gl-col gl-col3"><A href="https://2015.igem.org/Team:ZJU-China/Description" class="gl-hover"><SPAN>Improved  Part</SPAN><SPAN>Improved  Part</SPAN></A></DIV><DIV class="gl-col gl-col4"><A href="https://2015.igem.org/Team:ZJU-China/Composite_Part" class="gl-hover"><SPAN>Composite Part</SPAN><SPAN>Composite Part</SPAN></A></DIV></DIV><DIV class="gl-col-footer"><A href="https://2015.igem.org/Team:ZJU-China/Parts">Parts</A></DIV></DIV></DIV><DIV id="gl-menu-three" class="gl-has-submenu"><DIV id="gl-submenu-three" class="gl-submenu"><DIV class="gl-col-wrapper"><DIV class="gl-col gl-col1" style="width: 33.3%;"><A href="https://2015.igem.org/Team:ZJU-China/Project" class="gl-hover"><SPAN>Overview</SPAN><SPAN>Quick view for our project</SPAN></A><A href="https://2015.igem.org/Team:ZJU-China/Project/Termite_Issue" class="gl-hover"><SPAN>Termite Issue</SPAN><SPAN>The Termite Issue</SPAN></A></DIV><DIV class="gl-col gl-col2" style="width: 33.3%;"><DIV><SPAN><P>Design</P></SPAN><UL><LI><A href="https://2015.igem.org/Team:ZJU-China/Design/Toxinmanufacture">Toxins Manufacture</A></LI><LI><A href="https://2015.igem.org/Team:ZJU-China/Design/CNC" style="padding-left:32px">Bacteria Carrier</A></LI><LI><A href="https://2015.igem.org/Team:ZJU-China/Design/Termites" style="padding-left:32px">Termites</A></LI></UL></DIV></DIV><DIV class="gl-col gl-col3" style="width: 33.3%;"><A href="https://2015.igem.org/Team:ZJU-China/Project/Protocol" class="gl-hover"><SPAN>Protocol</SPAN><SPAN>Quick view for Protocol</SPAN></A></DIV></DIV><DIV class="gl-col-footer"><A href="https://2015.igem.org/Team:ZJU-China/Project">Project</A></DIV></DIV></DIV><DIV id="gl-menu-four" class="gl-has-submenu"><DIV id="gl-submenu-three" class="gl-submenu"><DIV class="gl-col-wrapper"><DIV class="gl-col gl-col1"><A href="https://2015.igem.org/Team:ZJU-China/Practices" class="gl-hover"><SPAN>Overview</SPAN><SPAN>Quick view for Outreach</SPAN></A><DIV><SPAN><P>Society</P></SPAN><UL><LI><A href="https://2015.igem.org/Team:ZJU-China/Practices/Society/Survey">Survey</A></LI><LI><A href="https://2015.igem.org/Team:ZJU-China/Practices/Society/Consultation" style="padding-left:32px">Consultation</A></LI><LI><A href="https://2015.igem.org/Team:ZJU-China/Practices/Society/Syn-bio_Debate" style="padding-left:32px">Syn-bio Debate</A></LI></UL></DIV></DIV><DIV class="gl-col gl-col2"><DIV><SPAN><P>Application</P></SPAN><UL><LI><A href="https://2015.igem.org/Team:ZJU-China/Design">Device</A></LI><LI><A href="https://2015.igem.org/Team:ZJU-China/Entrepreneurship" style="padding-left:32px">Promotion</A></LI><LI><A href="https://2015.igem.org/Team:ZJU-China/Practices/Application/Safety" style="padding-left:32px">Safety</A></LI><LI><A href="https://2015.igem.org/Team:ZJU-China/Practices/Application/Future" style="padding-left:32px">Future</A></LI></UL></DIV></DIV><DIV class="gl-col gl-col3"><DIV><SPAN><P>Public Promotion</P></SPAN><UL><LI><A href="https://2015.igem.org/Team:ZJU-China/Practices/Public_Promotion/Poly-cards">Poly-cards</A></LI><LI><A href="https://2015.igem.org/Team:ZJU-China/Practices/Public_Promotion/Syn-bio_Share" style="padding-left:32px">Syn-bio Share</A></LI><LI><A href="https://2015.igem.org/Team:ZJU-China/Practices/Public_Promotion/Community_Lab" style="padding-left:32px">Community Lab </A></LI><LI><A href="https://2015.igem.org/Team:ZJU-China/Practices/Public_Promotion/Seminar" style="padding-left:32px">Seminar</A></LI></UL></DIV></DIV><DIV class="gl-col gl-col4" style="padding-right:80px"><DIV><SPAN><P>Collaboration</P></SPAN><UL><LI><A href="https://2015.igem.org/Team:ZJU-China/Practices/Cooperate/Senergene/first">SynenergeneⅠ</A></LI><LI><A href="https://2015.igem.org/Team:ZJU-China/Practices/Cooperate/Senergene/second" style="padding-left:32px">SynenergeneⅡ</A></LI><LI><A href="https://2015.igem.org/Team:ZJU-China/Practices/Cooperate/Senergene/third" style="padding-left:32px">Synenergene Ⅲ</A></LI><LI><A href="https://2015.igem.org/Team:ZJU-China/Collaborations" style="padding-left:32px">Other Teams</A></LI></UL></DIV></DIV></DIV><DIV class="gl-col-footer"><A href="https://2015.igem.org/Team:ZJU-China/Practices">Outreach</A></DIV></DIV></DIV><DIV id="gl-menu-five" class="gl-has-submenu"><DIV id="gl-submenu-three" class="gl-submenu"><DIV class="gl-col-wrapper"><DIV class="gl-col gl-col1"><A href="https://2015.igem.org/Team:ZJU-China/Team" class="gl-hover"><SPAN>Members</SPAN><SPAN>Introduce our Members</SPAN></A></DIV><DIV class="gl-col gl-col2"><A href="https://2015.igem.org/Team:ZJU-China/Attributions" class="gl-hover"><SPAN>Attributions</SPAN><SPAN>Introduce the Attributions</SPAN></A></DIV><DIV class="gl-col gl-col3"><A href="https://2015.igem.org/Team:ZJU-China/Team/Acknowledgement" class="gl-hover"><SPAN>Acknowledgement</SPAN><SPAN>Introduce the Acknowledgement</SPAN></A></DIV><DIV class="gl-col gl-col4"><A href="https://2015.igem.org/Team:ZJU-China/Team/Sponsors" class="gl-hover"><SPAN>Sponsors</SPAN><SPAN>Introduce our Sponsors </SPAN></A></DIV></DIV><DIV class="gl-col-footer"><A href="https://2015.igem.org/Team:ZJU-China/Team">Team</A></DIV></DIV></DIV><DIV id="gl-menu-six" class="gl-has-submenu"><DIV id="gl-submenu-three" class="gl-submenu"><DIV class="gl-col-wrapper"><DIV class="gl-col gl-col1" style="border-right:2px solid #ddd"><A href="https://2015.igem.org/Team:ZJU-China/Modeling" class="gl-hover"><SPAN>Modeling Overview</SPAN><SPAN>Introduce our Members</SPAN></A></DIV><DIV class="gl-col gl-col2"><A href="https://2015.igem.org/Team:ZJU-China/Modeling/first" class="gl-hover"><SPAN>Modeling α</SPAN><SPAN>Termite Simulation</SPAN></A></DIV><DIV class="gl-col gl-col3"><A href="https://2015.igem.org/Team:ZJU-China/Modeling/second" class="gl-hover"><SPAN>Modeling β</SPAN><SPAN>Mass transfer model</SPAN></A></DIV><DIV class="gl-col gl-col4"><A href="https://2015.igem.org/Team:ZJU-China/Modeling/third" class="gl-hover"><SPAN>Modeling γ</SPAN><SPAN>Degradation model</SPAN></A></DIV></DIV><DIV class="gl-col-footer"><A href="https://2015.igem.org/Team:ZJU-China/Modeling">Modeling</A></DIV></DIV></DIV><DIV id="gl-menu-seven" class="gl-has-submenu"><DIV id="gl-submenu-seven" class="gl-submenu"><DIV class="gl-col-footer" style="border-top:0px solid #aaa;"><A href="https://2015.igem.org/Team:ZJU-China/Notebook">Notebook</A></DIV></DIV></DIV><DIV id="gl-menu-eight" class="gl-has-submenu"><DIV id="gl-submenu-three" class="gl-submenu"><DIV class="gl-col-wrapper"><DIV class="gl-col gl-col1"><A href="https://2015.igem.org/Team:ZJU-China/Results" class="gl-hover"><SPAN>Experiment Results</SPAN><SPAN>toxins manufacture</SPAN></A></DIV><DIV class="gl-col gl-col2"><A href="https://2015.igem.org/Team:ZJU-China/Results/second" class="gl-hover"><SPAN>Experiment Result</SPAN><SPAN>bacteria carrier</SPAN></A></DIV><DIV class="gl-col gl-col3"><A href="https://2015.igem.org/Team:ZJU-China/Results/third" class="gl-hover"><SPAN>Experiment Result</SPAN><SPAN>termites simulation</SPAN></A></DIV><DIV class="gl-col gl-col4"><A href="https://2015.igem.org/Team:ZJU-China/Results/social_result" class="gl-hover"><SPAN>Social Result</SPAN><SPAN>social results</SPAN></A></DIV></DIV><DIV class="gl-col-footer"><A href="https://2015.igem.org/Team:ZJU-China/Results">Results</A></DIV></DIV></DIV><DIV id="gl-menu-nine" class="gl-has-submenu"><DIV id="gl-submenu-nine" class="gl-submenu"><DIV class="gl-col-footer" style="border-top:0px solid #aaa;"><A href="https://2015.igem.org/Team:ZJU-China/Safety">Safety</A></DIV></DIV></DIV><DIV id="gl-menu-ten" class="gl-has-submenu" style="padding-right:20px"><DIV id="gl-submenu-three" class="gl-submenu"><DIV class="gl-col-wrapper"><DIV class="gl-col gl-col1"><A href="https://2015.igem.org/Team:ZJU-China/Design" class="gl-hover"><SPAN>Device</SPAN><SPAN>Introduce our Device and Instruction Manual</SPAN></A></DIV><DIV class="gl-col gl-col2"><A href="https://2015.igem.org/Team:ZJU-China/Entrepreneurship" class="gl-hover"><SPAN>Promotion</SPAN><SPAN>The Promotion</SPAN></A></DIV><DIV class="gl-col gl-col3"><A href="https://2015.igem.org/Team:ZJU-China/Practices/Application/Safety" class="gl-hover"><SPAN>Safety</SPAN><SPAN>The Safety</SPAN></A></DIV><DIV class="gl-col gl-col4"><A href="https://2015.igem.org/Team:ZJU-China/Practices/Application/Future" class="gl-hover"><SPAN>Future</SPAN><SPAN>Future</SPAN></A></DIV></DIV><DIV class="gl-col-footer"><A href="https://2015.igem.org/Team:ZJU-China/Design">Application</A></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV id="page-content-wrapper"><ARTICLE><H2 class="hh2" style="color:Green" id="pos1"><B>BBa_K1668001  <I>metk</I></B></H2><H2 class="hh2">Preface</H2><DIV class="row textcenter"><DIV style="    margin: auto; width: 80%;"><P class="p1">The part BBa_K417000 just briefly introduced metK and its output SAMsynth. The specific function and principle of SAMsynth (S-adenosylmethionine synthetase) isn't mentioned. Furthermore, the 3D structure and other detailed information isn't included, which may bring much inconvenience to potential users. Thus, connected with our project, we added plenty of related information of metK and sincerely wish to give a hand to future users. 
</P></DIV></DIV><H2 class="hh2">Summary</H2><DIV class="row textcenter"><DIV style="    margin: auto; width: 80%;"><P class="p1">1.	Characterize the output of metK in a novel chassis: <I>Streptomyces avermitilis</I></P><P class="p1">2.	Create a visualization of S-adenosylmethionine synthetase </P><P class="p1">3.	Add the 2D PAGE maps for the output of metK in E.coli</P></DIV></DIV><H2 class="hh2" id="pos2">Part 1 Characterization</H2><H3 style="padding:0;text-align:right">——the output of metK in <I>Streptomyces avermitilis</I></H3><H3 class="hh2">Background</H3><H4 class="hh4-left" style="padding-top:40px">metK: S-adenosylmethionine synthetase encoding gene<P class="p1">metK is the gene encoding S-adenosylmethionine synthetase, which has been found in almost every organism. Its output catalyzes the formation of S-adenosylmethionine from methionine and ATP. </P><P class="p1">
In <I>Streptomyces avermitilis</I>, it was found to stimulate the production of avermectins. When wild-type <I>S. avermitilis</I> strain ATCC31267 was transformed with pYJ02 and pYJ03, two metK expression plasmids, avermectin production was increased about 2.0-fold and 5.5-fold compared with that in the control strains, respectively. 
</P><P class="p1">
As for the principle of improving the productivity, instead of changing cell growth or copy effect, metK stimulates the avermectin production by increasing the intracellular concentration of S-adenosylmethionine (SAM), an important intermediate product in avermectin production. However, there may be a maximum concentration of SAM for the production of avermectin in <I>S. avermitilis</I>, which means that SAM has no effect when its concentration achieve maximum. 
</P><P class="p1">
The results of experiments in research paper showed that different metK expression levels have different influence on avermectin production in various <I>S. avermitilis</I> strains. The gene expression levels of metK in two engineered strain, GB-165 and 76-05, were much higher than those in wild-type strain, whereas the avermectin productivity in these two strains have not been significantly improved.  It is probably because the high expression level of metK in engineered strains limited the improvement of avermectin productivity by overexpression of metK. 
	</P><DIV class="row"><DIV class="col-md-12 textcenter"><DIV class="cpleft"><P class="kuvateksti">
                Fig.1 The picture of Streptomyces avermitilis under scanning electron microscope (SEM). Copyright all reserved ZJU-China 2015
            </P></DIV></DIV></DIV></H4><H4 class="hh4-left" style="padding-top:40px">Host of avermectin——<I>Streptomyces avermitilis </I><P class="p1"><I>Streptomyces avermitilis</I>, a soil-dwelling gram-positive microorganism, is a rich source of numerous secondary metabolites. Now it has been industrialized to produce the commercially important antiparasitic agent avermectin. Early in 2003, the complete genome of <I>Streptomyces avermitilis</I> had been sequenced.
</P><P class="p1">
In past years, scientists had been trying to transform gene into S.avermitilis. Until 1989, gene transformation in S.avermitilis was achieved through conjugation between E.coli strains(eg, s17-1) and S.avermitilis. However, the efficiency was limited by the methyl-specific restriction system in S.avermitilisi, which show strong restriction to gene methylated in normal E.coli strains. Eventually, high efficiency conjugation was achieved till the introduction of methylase-negative donor strain E.coli ET12567. Now conjugation and strain ET12567 has been ubiquitously adopted in the gene transformation of S.avermitilis.
</P></H4><H4 class="hh4-left" style="padding-top:40px">avermectin: effective and broad-spectrum pesticide<DIV class="row"><DIV class="col-md-12 textcenter"><DIV class="cpleft"><P class="kuvateksti">
                Fig.2 The structure of avermectin and avermectin 1 Copyright 2006, Springer Verlag
            </P></DIV></DIV></DIV><P class="p1">
For years, people always adopt the organochlorine pesticides such as chlordane and mirex to achieve prevention and control of termites, but these organochlorine pesticides will produce pollution and potential harm to the environment. Avermectin is a new type of high efficient biological pesticide, which has good control effect to the termites and other pests, and no pollution to the environment.
</P></H4><H3 class="hh2">Experiment: Avermectin manufacture in <I>S. avermitilis</I></H3><H4 class="hh4-left" style="padding-top:40px">Purpose</H4><P class="p1">
We attempt to test the function of metK gene by improving the yield of avermectin in <I>S. avermitilis</I>. Because as it has been described in research paper, metK was found to stimulate the production of avermectins. For one thing, being a secondary metabolite produced by <I>Streptomyces avermitilis</I>, avermectin is regulated by an 80kb gene cluster, making it difficult to express in other standardized strains, for instance, Escherichia coli. For another, the avermectin yield in wild type <I>S. avermitilis</I> strain is comparatively low. Therefore, we plan to engineer the wild <I>S. avermitilis</I> with metK to improve the yield of avermectin.
</P><DIV class="row"><DIV class="col-md-12 textcenter"><DIV class="cpleft"><P class="kuvateksti">
                Fig.3 the circuits constructed for yield improvement of avermectin in S.avermitilis.
            </P></DIV></DIV></DIV><H4 class="hh4-left" style="padding-top:40px">Circuit design</H4><H4 class="hh2-left">promoter: ermEp<P class="p1">
We chose ermEp, a strong constitutive promoter, to overexpress the three genes in S.avermitilis. It should be noticed that ermEp can only be expressed in S.avermitilis strains instead of Escherichia coli or any other chassis.
</P></H4><H4 class="hh2-left">backbone: PL96 and PL97<P class="p1">
PL96 and PL97 are two high-copy vectors we used to overexpress our target genes. We get these vectors through commercial purchase. These vectors have pUC18 and pIJ101 replication origins for high-copy plasmid number in Escherichia coli and S.avermitilis, respectively, and the oriT (RK2) allows the efficient and convenient plasmid transfer from E.coli to S.avermitilis.
</P><DIV class="row"><DIV class="col-md-12 textcenter"><DIV class="cpleft"><P class="kuvateksti">
               Fig.4 The plasmid map for PL96.
            </P></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-12 textcenter"><DIV class="cpleft"><P class="kuvateksti">
               Fig.5 The plasmid map for PL97.
            </P></DIV></DIV></DIV><P class="p1">
To be noticed, we use special antibiotic aparamycin to choose final transformants. And there are aparamycin resistent gene acc in the backbone. 
</P></H4><H4 class="hh2-left">expression<P class="p1">
In order to construct and express the three gene in S.avermitilis, we have adopted two hosts, E.coli DH5αand E.coli ET12567. Then the target vectors are transferred from E.coli ET12567 to S.avermitilis by conjugation.
</P></H4><H4 class="hh2-left">primary host: E.coli DH5α<P class="p1">
As usual, we use E.coli DH5α to get plentiful recombinants in high quality and quantity. 
</P></H4><H4 class="hh2-left">intermedia host: E.coli ET12567<P class="p1">
E.coli ET12567 is a methylase-negative donor strain first used by MacNeil in 1988. And we use E.coli ET12567 to demethylation the recombinants to better suit the methyl-specific restriction system in S.avermitilis.
</P></H4><H4 class="hh2-left">conjugation<P class="p1">
Bacterial conjugation is the transfer of genetic material between bacterial cells by direct cell-to-cell contact or by a bridge-like connection between two cells. During conjugation the donor cell provides a conjugative or mobilizable genetic element that is most often a plasmid or transposon. In laboratories, successful transfers have been reported from bacteria to yeast, plants, mammalian cells, etc.
</P><DIV class="row"><DIV class="col-md-12 textcenter"><DIV class="cpleft"><P class="kuvateksti">
               Fig.6 The sketch map of conjugation between E.coli ET12567 and S.avermitilisi.
            </P></DIV></DIV></DIV><P class="p1">
In our project, we use the conjugation between E.coli ET12567 and S.avermitilisi to overexpress three target genes. 
</P></H4><H4 class="hh4-left" style="padding-top:40px">Circuit Construction</H4><H2 class="hh2-left">STEP ONE: PCR</H2><P class="p1">	
We amplify the target gene from the genome of S.avermitilisi ATCC13267 strain by PCR. 
</P><H2 class="hh2-left">STEP TWO: TA CLONING</H2><P class="p1">
We use TA cloning to efficiently clone the PCR products. In TA cloning, we use pMD19-T Vector, a vector transformed from pUC19 vector, to improve the efficiency of digestion and connection. As a result, we get three recombinant vectors of target genes and pMD19-T.
</P><H2 class="hh2-left">STEP THREE: DIGESTION AND LIGATION</H2><P class="p1">
We digest the three recombinants and backbone PL96 with restriction enzymes NdeI, XbaI, then connect the fragments and backbone. Similarly, we use NdeI, HindⅢ to digest the three recombinants and backbone PL97 and connect the corresponding product. Then we get the target plasmids.
</P><DIV class="row"><DIV class="col-md-12 textcenter"><DIV class="cpleft"><P class="kuvateksti">
                Fig.7 digestion and ligation in PL96.
            </P></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-12 textcenter"><DIV class="cpleft"><P class="kuvateksti">
               Fig.8 digestion and ligation in PL97.
            </P></DIV></DIV></DIV><P class="p1">
For more detailed protocols, please go to <A href="https://2015.igem.org/Team:ZJU-China/Project/Protocol">Protocol</A>.
</P><H3 class="hh2">Result</H3><H4 class="hh2-left">Gel electrophoretic analysis</H4><P class="p1">
We successfully constructed the plasmid (PL96 and PL97) containing metK (sequence is shown in the part BBa_K1668001) gene in E.coli DH5α and E.coli ET12567. Then we transformed it into S.avermitilis by conjugation.
</P><DIV class="row"><DIV class="col-md-12 textcenter"><DIV class="cpleft"><P class="kuvateksti">
               Fig.9 Gel electrophoretic analyses of PCR products (A) and double enzyme digestion products (B and C). (A) 5-μl samples of the PCR products for metK, (B and C) 5-μl samples of the double enzyme digestion products were loaded onto a 1% BioRad Ready Agarose Mini Gel, then subjected to AGE. See (<A href="https://2015.igem.org/Team:ZJU-China/Project/Protocol">Protocol</A>) for AGE parameters. (B and C) Sizes of the NdeI and XbaI–cleaved assemblies were determined by AGE analysis. The DNA size standards was the DL2,000 DNA Marker (M1; TaKaRa, Cat#3427A) and 1kb DNA Ladder (Dye Plus) (M2; TaKaRa, Cat#3426A). Bands were visualized with a Shanghai Peiqing JS-380A Fluorescence Imager.
            </P></DIV></DIV></DIV><H2 class="hh2" id="pos3">Part 2 Visualization</H2><H3 style="padding:0;text-align:right">——S-adenosylmethionine synthetase in E.coli</H3><P class="p1">We have found the 3D molecular graphic (Fig.2) and interaction figure (Fig.3) of S-adenosylmethionine synthetase in NCBI database. </P><DIV class="row"><DIV class="col-md-12 textcenter"><DIV class="cpleft"><P class="kuvateksti">
                Fig.10 3D molecular graphic of S-adenosylmethionine synthetase (MMDB ID: 52023)
            </P></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-12 textcenter"><DIV class="cpleft"><P class="kuvateksti">
               Fig.11 the interaction figure of S-adenosylmethionine synthetase (MMDB ID: 52023 ). In the figure, ○ represents protein and ◇ represents chemical. The rhombus marked 1,2 and 3 represents phosphate ion, Co and K+, respectively.
            </P></DIV></DIV></DIV><H2 class="hh2" id="pos4">Part 3 2DPAGE map</H2><H3 style="padding:0;text-align:right">——the output of metK in E.coli</H3><P class="p1">
We have found the 2DPAGE maps for S-adenosylmethionine synthetase in E.coli. You can click on a highlighted spot in the figure from the website listed below to access all the associated protein entries of the spot.
</P><DIV class="row"><DIV class="col-md-12 textcenter"><DIV class="cpleft"><P class="kuvateksti">
            Fig.12 2DPAGE maps for S-adenosylmethionine synthetase in E.coli (SWISS-2DPAGE: P0A817)
            </P></DIV></DIV></DIV><H2 class="hh2" id="pos4">Reference</H2><P class="p1">
http://www.ncbi.nlm.nih.gov/Structure/mmdb/mmdbsrv.cgi?uid=52023
</P><P class="p1">
http://world-2dpage.expasy.org/swiss-2dpage/viewer&amp;map=ECOLI4.5-5.5&amp;ac=P0A817
</P><P class="p1"><A href="https://static.igem.org/mediawiki/parts/0/03/Overexpression_of_metK_shows_different_effects_on_avermectin_production_in_various_Streptomyces_avermitilis_strains.pdf">
X. Zhao, Q. Wang, W. Guo, Y. Cai, C. Wang, S. Wang, S. Xiang, Y. Song, Overexpression of metK shows different effects on avermectin production in various Streptomyces avermitilis strains. World journal of microbiology &amp; biotechnology 29, 1869-1875 (2013); published online EpubOct (10.1007/s11274-013-1350-0).
</A></P><H2 class="hh2" style="color:Green" id="pos6"><B>BBa_K1668011 <I>mCherry</I></B></H2>:
<DIV><H2 class="hh2">Preface</H2>:
<DIV class="row textcenter"><DIV style="    margin: auto; width: 80%;"><P class="p1">There is little message about the characterrization and features of mCherry in old pages. In addition, the quality of parts varied from year to year. With an extra promoter in it, the sequnce of mCherry in 2015 (kit plate 3, well 15B) is incrrect according to the sequncing result. And mCherry in 2012 (kit plate 2, well 8E) is in a &quot;pSB1A2&quot; backbone, which is of 4k size and doesn't correspond with message of pSB1A2 in Part Registry. So we amplified the correct gene, assembled it with standardize pSB1C3 backbone, confirmed the sequnce by enzyme digestion and sequncing and submitted the part BBa_K1668011.
</P></DIV></DIV><H2 class="hh2" id="">Summary</H2>:
<DIV class="row textcenter"><DIV style="    margin: auto; width: 80%;"><P class="p1">1. sequncing of parts in previous year.</P><P class="p1">2. correct the sequnce and resemble it in a standard backbone <A href="http://parts.igem.org/Part:BBa_K1668011">BBa_K16680111</A></P><P class="p1">3. add more features about the mCherry.

</P></DIV></DIV><H3 class="hh2">Background</H3><P class="p1">We used <I>mCherry</I> (BBa_J06702) as a reporter, which is one registry star and therefore comparably more reliable. According to the message in the Registry, there is no promoter in the part therefore the reporter should not be expressed(Figure 1). But when we transformed the <I>mCherry</I> gene into <I>E.coli DH5α</I>, the <I>E.coli</I> turned out to become red. With the colorless control, we concluded that the <I>mCherry</I> was expressed and sequnced the parts. According to the results of the sequncing, the part has a promoter with it (Figure 2). </P><DIV><DIV class="row"><DIV class="col-md-12 textcenter"><DIV class="cpleft"><P class="kuvateksti">
                Figure 1 Information of <I>mCherry</I> (BBa_J06702) in Part Registry. 
              </P></DIV></DIV></DIV></DIV><DIV><DIV class="row"><DIV class="col-md-12 textcenter"><DIV class="cpleft"><P class="kuvateksti">
                Figure 2 Sequencing results of 2015 <I>mCherry</I> in 2015 kit plate 3, well 15B.
              </P></DIV></DIV></DIV></DIV><P class="p1">
hen we got the same part from kit plate in 2013 and 2012. At last, we used <I>mCherry</I> of 2012 in backbone pSB1A2, which is a 2k backbone. But after we have cut the backbone with XbaI and SpeI, we found that the backbone was as big as 4k (figure 3). So we had to sequnce the <I>mCherry</I> of 2012, which turned out to be correct (figure 4). And the backbone had an anti-ampicillin gene with it.
	</P><DIV><DIV class="row"><DIV class="col-md-12 textcenter"><DIV class="cpleft"><P class="kuvateksti">
               Figure 3 Double enzyme digestion of the 2012<I>mCherry </I>(BBa_J06702) in 2012 kit plate 2, well 8E.
              </P></DIV></DIV></DIV></DIV><DIV><DIV class="row"><DIV class="col-md-12 textcenter"><DIV class="cpleft"><P class="kuvateksti">
                Figure 4 Sequencing results of 2012 <I>mCherry</I> in 2012 kit plate 2, well 8E.
              </P></DIV></DIV></DIV></DIV><H3 class="hh2">Source<P class="p1">
      The <I>mCherry</I> gene was amplified by PCR with the template genomic DNA extracted from strain <I>Photorhabdus luminescens TT01</I>. We got the strain from Shandong University.
    </P></H3><H3 class="hh2">Design Notes</H3><H4 class="hh4-left">PCR<P class="p1">
      The <I> mCherry </I> gene was amplified by PCR with the template genomic DNA extracted from strain <I>Photorhabdus luminescens TT01</I>. We use primer mCherry -left F and mCherry R to amplify the left side of gene, which are shown below.
    </P></H4><H4 class="hh4-left">Seamless assembly<P class="p1">
     We used seamless assembly as our assembly method so restriction digestion and T4 ligation can be avoided. By this way prefix sequence, <I> mCherry </I> and suffix sequence can be ligated seamlessly.  Detailed protocol and instruction for primer design can be seen in our <A href="https://2015.igem.org/Team:ZJU-China/Project/Protocol">Protocol</A>.
    </P><P class="p1">
     mCherry F(F, 5’-3’): AGAAAGAGGAGAAATACTAGATGGTGAG
    </P><P class="p1">
     mCherry R(R, 5’-3’): CCGGACTGCAGCGGCCGCTACTAGTATAAACGCAGAAAGGCC
    </P></H4><H4 class="hh4-left">Transformation and confirmation<P class="p1">
     After seamless assembly, standard plasmid pSB1C3 containing <I> mCherry </I> gene was transformed into <I> E.coli DH5α</I>. When single colony appeared on the LB plate, we picked out 10 colonies, respectively, as our template for bacteria solution PCR. In order to avoid the appearance of false positive clones, we used VF2/VR as the universal primers. The positive clone and its corresponding raw bacteria solution were stored and samples were sent to do DNA sequencing.
    </P></H4><H4 class="hh4-left">Plasmid map<DIV><DIV class="row"><DIV class="col-md-12 textcenter"><DIV class="cpleft"><P class="kuvateksti">
                Fig.1 The plasmid map of BBa_K1668011 mCherry
              </P></DIV></DIV></DIV></DIV></H4><H3 class="hh2">Results</H3><H4 class="hh4-left">Gel electrophoretic analysis<P class="p1">
     5-μl samples of the double enzyme digestion products for improved <I>mCherry</I> BBa_K1668011 were loaded onto a 1% BioRad Ready Agarose Mini Gel, then subjected to AGE. See (<A href="https://2015.igem.org/Team:ZJU-China/Project/Protocol">Protocol</A>) for AGE parameters. Sizes of the XbaI and PstI–cleaved assemblies were determined by AGE analysis. The DNA size standards were 1kb DNA Ladder (Dye Plus)(M2; TaKaRa, Cat#3426A). Bands were visualized with a Shanghai Peiqing JS-380A Fluorescence Imager. Digested plasmid backbone and <I>mCherry</I> fragment are indicated.
    </P><P class="p1">
    It can be clearly seen that the 900bp <I>mCherry</I> is in the right position in agarose gel.
    </P><DIV><DIV class="row"><DIV class="col-md-12 textcenter"><DIV class="cpleft"><P class="kuvateksti">
                Figure 5 double enzyme digestion of the improved <I>mCherry</I>. 
              </P></DIV></DIV></DIV></DIV></H4><H4 class="hh4-left">DNA sequencing<P class="p1">
      We have sequenced the parts with standard primers VF2 and VR. The sequence of the 155bp part shows 100% agreement with the desired sequence. 
    </P></H4><H4 class="hh4-left">EXPRESSION<P class="p1">
     We serial express our three toxins (TcdA1, Plu1537 and Plu0840) with MCherry, shown in figure 6. The redness of all three devices indicates that <I>mCherry</I> functions well. 
    </P><DIV><DIV class="row"><DIV class="col-md-12 textcenter"><DIV class="cpleft"><P class="kuvateksti">
                Figure 6 Tandem expression of toxin protein and MCherry shown in pipet tube. 1ml bacterium solution was added in each pipet tube and centrifuged in a speed of 12000 rpm of for 2min. 
              </P></DIV></DIV></DIV></DIV><ASIDER id="popView2"><HEADER>Improved parts</HEADER><UL id="nav2"><LI><A href="#pos1"><B>BBa_K1668001<I>metk</I></B></A></LI><LI><A href="#pos2">Part 1</A></LI><LI><A href="#pos3">Part 2</A></LI><LI><A href="#pos4">Part 3</A></LI><LI><A href="#pos6"><B>BBa_K1668011<I>mCherry</I></B></A></LI></UL></ASIDER></H4></DIV></ARTICLE></DIV><DIV id="gl-footer" class="cf" style="max-width:1440px"><DIV id="gl-footer-copyright" class="cf"><DIV class="gl-col1 cf"><SPAN>© 2015 ZJU-IGEM TERMINATE.</SPAN></DIV><UL class="gl-col2 cf"><LI><A href="http://www.igem.org/Main_Page">IGEM</A></LI><LI><A href="http://www.zju.edu.cn">Zhejiang University</A></LI></UL></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>