<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_ZJU-China_Composite_Part skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:ZJU-China/Composite Part</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="gl-outer-wrapper" class="gl-zh cf" style=""><DIV id="gl-header" class="cf gl-header-fixed" style="max-width:1440px"><DIV id="gl-header-bottom" class="cf"><DIV id="gl-menu" class="cf"><DIV id="gl-menu-one" class="gl-has-submenu"><DIV id="gl-submenu-one" class="gl-submenu"><DIV class="gl-col-footer" style="border-top:0px solid #aaa;"><A href="https://2015.igem.org/Team:ZJU-China">Home</A></DIV></DIV></DIV><DIV id="gl-menu-two" class="gl-has-submenu"><DIV id="gl-submenu-three" class="gl-submenu"><DIV class="gl-col-wrapper"><DIV class="gl-col gl-col1"><A href="https://2015.igem.org/Team:ZJU-China/Parts" class="gl-hover"><SPAN>Team Part</SPAN><SPAN>Team Part</SPAN></A></DIV><DIV class="gl-col gl-col2"><A href="https://2015.igem.org/Team:ZJU-China/Basic_Part" class="gl-hover"><SPAN>Basic Part</SPAN><SPAN>Basic Part</SPAN></A></DIV><DIV class="gl-col gl-col3"><A href="https://2015.igem.org/Team:ZJU-China/Description" class="gl-hover"><SPAN>Improved  Part</SPAN><SPAN>Improved  Part</SPAN></A></DIV><DIV class="gl-col gl-col4"><A href="https://2015.igem.org/Team:ZJU-China/Composite_Part" class="gl-hover"><SPAN>Composite Part</SPAN><SPAN>Composite Part</SPAN></A></DIV></DIV><DIV class="gl-col-footer"><A href="https://2015.igem.org/Team:ZJU-China/Parts">Parts</A></DIV></DIV></DIV><DIV id="gl-menu-three" class="gl-has-submenu"><DIV id="gl-submenu-three" class="gl-submenu"><DIV class="gl-col-wrapper"><DIV class="gl-col gl-col1" style="width: 33.3%;"><A href="https://2015.igem.org/Team:ZJU-China/Project" class="gl-hover"><SPAN>Overview</SPAN><SPAN>Quick view for our project</SPAN></A><A href="https://2015.igem.org/Team:ZJU-China/Project/Termite_Issue" class="gl-hover"><SPAN>Termite Issue</SPAN><SPAN>The Termite Issue</SPAN></A></DIV><DIV class="gl-col gl-col2" style="width: 33.3%;"><DIV><SPAN><P>Design</P></SPAN><UL><LI><A href="https://2015.igem.org/Team:ZJU-China/Design/Toxinmanufacture">Toxins Manufacture</A></LI><LI><A href="https://2015.igem.org/Team:ZJU-China/Design/CNC" style="padding-left:32px">Bacteria Carrier</A></LI><LI><A href="https://2015.igem.org/Team:ZJU-China/Design/Termites" style="padding-left:32px">Termites</A></LI></UL></DIV></DIV><DIV class="gl-col gl-col3" style="width: 33.3%;"><A href="https://2015.igem.org/Team:ZJU-China/Project/Protocol" class="gl-hover"><SPAN>Protocol</SPAN><SPAN>Quick view for Protocol</SPAN></A></DIV></DIV><DIV class="gl-col-footer"><A href="https://2015.igem.org/Team:ZJU-China/Project">Project</A></DIV></DIV></DIV><DIV id="gl-menu-four" class="gl-has-submenu"><DIV id="gl-submenu-three" class="gl-submenu"><DIV class="gl-col-wrapper"><DIV class="gl-col gl-col1"><A href="https://2015.igem.org/Team:ZJU-China/Practices" class="gl-hover"><SPAN>Overview</SPAN><SPAN>Quick view for Outreach</SPAN></A><DIV><SPAN><P>Society</P></SPAN><UL><LI><A href="https://2015.igem.org/Team:ZJU-China/Practices/Society/Survey">Survey</A></LI><LI><A href="https://2015.igem.org/Team:ZJU-China/Practices/Society/Consultation" style="padding-left:32px">Consultation</A></LI><LI><A href="https://2015.igem.org/Team:ZJU-China/Practices/Society/Syn-bio_Debate" style="padding-left:32px">Syn-bio Debate</A></LI></UL></DIV></DIV><DIV class="gl-col gl-col2"><DIV><SPAN><P>Application</P></SPAN><UL><LI><A href="https://2015.igem.org/Team:ZJU-China/Design">Device</A></LI><LI><A href="https://2015.igem.org/Team:ZJU-China/Entrepreneurship" style="padding-left:32px">Promotion</A></LI><LI><A href="https://2015.igem.org/Team:ZJU-China/Practices/Application/Safety" style="padding-left:32px">Safety</A></LI><LI><A href="https://2015.igem.org/Team:ZJU-China/Practices/Application/Future" style="padding-left:32px">Future</A></LI></UL></DIV></DIV><DIV class="gl-col gl-col3"><DIV><SPAN><P>Public Promotion</P></SPAN><UL><LI><A href="https://2015.igem.org/Team:ZJU-China/Practices/Public_Promotion/Poly-cards">Poly-cards</A></LI><LI><A href="https://2015.igem.org/Team:ZJU-China/Practices/Public_Promotion/Syn-bio_Share" style="padding-left:32px">Syn-bio Share</A></LI><LI><A href="https://2015.igem.org/Team:ZJU-China/Practices/Public_Promotion/Community_Lab" style="padding-left:32px">Community Lab </A></LI><LI><A href="https://2015.igem.org/Team:ZJU-China/Practices/Public_Promotion/Seminar" style="padding-left:32px">Seminar</A></LI></UL></DIV></DIV><DIV class="gl-col gl-col4" style="padding-right:80px"><DIV><SPAN><P>Collaboration</P></SPAN><UL><LI><A href="https://2015.igem.org/Team:ZJU-China/Practices/Cooperate/Senergene/first">SynenergeneⅠ</A></LI><LI><A href="https://2015.igem.org/Team:ZJU-China/Practices/Cooperate/Senergene/second" style="padding-left:32px">SynenergeneⅡ</A></LI><LI><A href="https://2015.igem.org/Team:ZJU-China/Practices/Cooperate/Senergene/third" style="padding-left:32px">Synenergene Ⅲ</A></LI><LI><A href="https://2015.igem.org/Team:ZJU-China/Collaborations" style="padding-left:32px">Other Teams</A></LI></UL></DIV></DIV></DIV><DIV class="gl-col-footer"><A href="https://2015.igem.org/Team:ZJU-China/Practices">Outreach</A></DIV></DIV></DIV><DIV id="gl-menu-five" class="gl-has-submenu"><DIV id="gl-submenu-three" class="gl-submenu"><DIV class="gl-col-wrapper"><DIV class="gl-col gl-col1"><A href="https://2015.igem.org/Team:ZJU-China/Team" class="gl-hover"><SPAN>Members</SPAN><SPAN>Introduce our Members</SPAN></A></DIV><DIV class="gl-col gl-col2"><A href="https://2015.igem.org/Team:ZJU-China/Attributions" class="gl-hover"><SPAN>Attributions</SPAN><SPAN>Introduce the Attributions</SPAN></A></DIV><DIV class="gl-col gl-col3"><A href="https://2015.igem.org/Team:ZJU-China/Team/Acknowledgement" class="gl-hover"><SPAN>Acknowledgement</SPAN><SPAN>Introduce the Acknowledgement</SPAN></A></DIV><DIV class="gl-col gl-col4"><A href="https://2015.igem.org/Team:ZJU-China/Team/Sponsors" class="gl-hover"><SPAN>Sponsors</SPAN><SPAN>Introduce our Sponsors </SPAN></A></DIV></DIV><DIV class="gl-col-footer"><A href="https://2015.igem.org/Team:ZJU-China/Team">Team</A></DIV></DIV></DIV><DIV id="gl-menu-six" class="gl-has-submenu"><DIV id="gl-submenu-three" class="gl-submenu"><DIV class="gl-col-wrapper"><DIV class="gl-col gl-col1" style="border-right:2px solid #ddd"><A href="https://2015.igem.org/Team:ZJU-China/Modeling" class="gl-hover"><SPAN>Modeling Overview</SPAN><SPAN>Introduce our Members</SPAN></A></DIV><DIV class="gl-col gl-col2"><A href="https://2015.igem.org/Team:ZJU-China/Modeling/first" class="gl-hover"><SPAN>Modeling α</SPAN><SPAN>Termite Simulation</SPAN></A></DIV><DIV class="gl-col gl-col3"><A href="https://2015.igem.org/Team:ZJU-China/Modeling/second" class="gl-hover"><SPAN>Modeling β</SPAN><SPAN>Mass transfer model</SPAN></A></DIV><DIV class="gl-col gl-col4"><A href="https://2015.igem.org/Team:ZJU-China/Modeling/third" class="gl-hover"><SPAN>Modeling γ</SPAN><SPAN>Degradation model</SPAN></A></DIV></DIV><DIV class="gl-col-footer"><A href="https://2015.igem.org/Team:ZJU-China/Modeling">Modeling</A></DIV></DIV></DIV><DIV id="gl-menu-seven" class="gl-has-submenu"><DIV id="gl-submenu-seven" class="gl-submenu"><DIV class="gl-col-footer" style="border-top:0px solid #aaa;"><A href="https://2015.igem.org/Team:ZJU-China/Notebook">Notebook</A></DIV></DIV></DIV><DIV id="gl-menu-eight" class="gl-has-submenu"><DIV id="gl-submenu-three" class="gl-submenu"><DIV class="gl-col-wrapper"><DIV class="gl-col gl-col1"><A href="https://2015.igem.org/Team:ZJU-China/Results" class="gl-hover"><SPAN>Experiment Results</SPAN><SPAN>toxins manufacture</SPAN></A></DIV><DIV class="gl-col gl-col2"><A href="https://2015.igem.org/Team:ZJU-China/Results/second" class="gl-hover"><SPAN>Experiment Result</SPAN><SPAN>bacteria carrier</SPAN></A></DIV><DIV class="gl-col gl-col3"><A href="https://2015.igem.org/Team:ZJU-China/Results/third" class="gl-hover"><SPAN>Experiment Result</SPAN><SPAN>termites simulation</SPAN></A></DIV><DIV class="gl-col gl-col4"><A href="https://2015.igem.org/Team:ZJU-China/Results/social_result" class="gl-hover"><SPAN>Social Result</SPAN><SPAN>social results</SPAN></A></DIV></DIV><DIV class="gl-col-footer"><A href="https://2015.igem.org/Team:ZJU-China/Results">Results</A></DIV></DIV></DIV><DIV id="gl-menu-nine" class="gl-has-submenu"><DIV id="gl-submenu-nine" class="gl-submenu"><DIV class="gl-col-footer" style="border-top:0px solid #aaa;"><A href="https://2015.igem.org/Team:ZJU-China/Safety">Safety</A></DIV></DIV></DIV><DIV id="gl-menu-ten" class="gl-has-submenu" style="padding-right:20px"><DIV id="gl-submenu-three" class="gl-submenu"><DIV class="gl-col-wrapper"><DIV class="gl-col gl-col1"><A href="https://2015.igem.org/Team:ZJU-China/Design" class="gl-hover"><SPAN>Device</SPAN><SPAN>Introduce our Device and Instruction Manual</SPAN></A></DIV><DIV class="gl-col gl-col2"><A href="https://2015.igem.org/Team:ZJU-China/Entrepreneurship" class="gl-hover"><SPAN>Promotion</SPAN><SPAN>The Promotion</SPAN></A></DIV><DIV class="gl-col gl-col3"><A href="https://2015.igem.org/Team:ZJU-China/Practices/Application/Safety" class="gl-hover"><SPAN>Safety</SPAN><SPAN>The Safety</SPAN></A></DIV><DIV class="gl-col gl-col4"><A href="https://2015.igem.org/Team:ZJU-China/Practices/Application/Future" class="gl-hover"><SPAN>Future</SPAN><SPAN>Future</SPAN></A></DIV></DIV><DIV class="gl-col-footer"><A href="https://2015.igem.org/Team:ZJU-China/Design">Application</A></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV id="page-content-wrapper"><ARTICLE><H3 class="hh2" style="border:0;color:red" id="pos1">Part:BBa_K1668010</H3><H4 class="hh4-left" style="padding-top:30px" style="padding-top:30px">plu1537-device</H4><P class="p1">
        The part <I>plu1537</I> device is composed of arabinose inducible promoter pBad <A href="http://parts.igem.org/Part:">BBa_I0500</A>, toxin protein <I>plu1537</I> coding sequence <A href="http://parts.igem.org/Part:">BBa_K1668007</A> and composite part mCherry <A href="http://parts.igem.org/Part:">BBa_K1668011</A>.
    </P><P class="p1">
        We use the device to tandem express toxic protein Plu1537 and mCherry in <I>E.coli</I> BL 21 (DE3). The gene plu1537 is for termite control and mCherry is a reporter.
    </P><DIV style="width:100%l;text-align:center;font-size:30px;"><A href="http://parts.igem.org/Part:BBa_K1668010">Sequence and Features</A><H4 class="hh4-left" style="padding-top:30px" style="padding-top:30px">Characterization</H4><H5 class="hh2"> OVERVIEW </H5><P class="p1">
        We builg the device <I>plu1537</I> to express insecticidal toxic protein Plu1537 together with the reporter mCherry. The 14kDa insecticidal protein Plu1537 is used to kill termites in our project. 
    </P><P class="p1">
        The exact mechanism of Plu1537 remains unclear. In a 2009 research, scientists first expressed the Plu1537 and purified it with GST tag. Consequently, Plu1537 showed its insecticidal activity against two moth species (Galleria&amp; Spodoptera) larvae via hemocoel injection instead of oral feeding.
    </P><P class="p1">
        We clone and standardize the gene into standard plasmid pSB1C3. After confirmation through enzyme digestion and whole sequencing, we transform the plasmid into <I>E.coli</I> BL21 (DE3) to achieve better expression level. 
    </P><P class="p1">
        We observe that transformants have obviously turned red and identify the expected protein band in SDS-PAGE. According to in vivo test on termites, the toxic effect of Plu1537 is far more ideal than our expectation by oral feeding.
    </P><P class="p1">
        In conclusion, we have successfully cloned and expressed the plu1537 in <I>E.coli</I> BL21(DE3). Toxic effects of oral feedings on termites are far better than that described in moth larvae research.  
    </P><H5 class="hh2"> BACKGROUND </H5><P class="p1">
        In 2009 research<I> Expression and activity of a probable toxin from Photorhabdus Luminescens</I>, toxin protein pit, which is 94% homologous with plu1537, is expressed in <I>E.colo</I> BL21(DE3). Engineered strain BL21 was both orally fed and injected in hemocoel to two kind of moth(Galleria mellonella &amp; Spodoptera litura). As a result, hemocoel injection is more effective than oral feeding. However, our experiment showed that oral feeding is also effectively. 
    </P><P class="p1">
        Plu1537 shares 30% amino acid sequence similarity with a 13.6 kDa insecticidal crystal protein Cry34Ab1 in <I>Bacillus thuringiensis</I> (Figure 1), which belongs to bt toxin family. 
    </P><P class="p1">
        Bt family is a type of biological insecticidal toxins and may be the most famous one up to now. After years of study, bt family is becoming bigger and bigger. Basically, most proteins in bt family such as Cry34Ab1 form pores in the membrane to kill the cell. 
    </P><P class="p1">
        In 2014, the structure of cry34Ab1 was revealed and reported in PLOS ONE. As displayed in figure 1, the structure of cry34Ab1 is simpler, compared with other two toxins we used. Notably, cry34Ab1 (figure 2A) functions only with the assistant of cry35Ab1 (figure 2B)(<I>2</I>), while plu1537 performs its toxicity directly.
    </P><DIV class="col-md-12 textcenter"><DIV class="col-md-6 img-100"><DIV class="cpleft"><P class="kuvateksti">
              Figure 1 The 3D structure of cry34Ab1. Copyright 2014, Worldwide Protein Data.
            </P></DIV></DIV><DIV class="col-md-6 img-100"><DIV class="cpright"><P class="kuvateksti">
              Figure 2 Comparison between cry34Ab1 and cry35Ab1(2). Copyright 2014, Public Library of Science.
            </P></DIV></DIV></DIV><H3 class="hh2">Construction</H3><H4 class="hh4-left">PCR<P class="p1">
        We use primer pBad F / pBad R to amplify <I>pBad</I> and standard F / plu1537 R for CDS <I>plu1537</I>.
    </P></H4><H4 class="hh4-left">Seamless assembly<P class="p1">
     We used seamless assembly as our assembly method so restriction digestion and T4 ligation can be avoided. Detailed protocol and instruction for primer design can be seen in our Protocol. By this way prefix sequence, <I>pBad</I>, CDS <I>plu1537</I> and front twenty bases of <I>mCherry</I> sequence can be ligated seamlessly.
    </P><P class="p1">
      pBad F (F, 5’-3’): ATTCGCGGCCGCTTCTAGAGTTATGACAACTTGACG
    </P><P class="p1">
      pBad R (R, 5’-3’): GCTAGCCCAAAAAAACG
    </P><P class="p1">
      standard F (F, 5’-3’): CCGTTTTTTTGGGCTAGCAGAAAGAGGAGAAATACTAG
    </P><P class="p1">
     plu1537 R (R, 5’-3’): CGGACTGCAGCGGCCGCTACTAGTATTATTAGCTTACAATCATATATTC
    </P></H4><H4 class="hh4-left">Transformation and confirmation<P class="p1">
      After seamless assembly, standard plasmid pSB1C3 containing <I>plu1537</I> gene was transformed into <I> E.coli DH5α</I>. When single colony appeared on the LB plate, we picked out 10 colonies, respectively, as our template for bacteria solution PCR. In order to avoid the appearance of false positive clones, we used VF/VR as the universal primers. The positive clone and its corresponding raw bacteria solution were stored and samples were sent to do DNA sequencing.
    </P></H4><H4 class="hh4-left">Plasmid map<DIV class="col-md-12 textcenter"><DIV class="cpleft"><P class="kuvateksti">
                Figure 3 The plasmid map of BBa_K1668010 <I>plu1537</I> device
              </P></DIV></DIV></H4><H4 class="hh4-left" style="padding-top:30px" style="padding-top:30px">Results</H4><DIV class="col-md-6 text-col-left"><H5 class="hh2"> PLASMID CONSTRUCTION</H5><P class="p1">
                5-μl samples of the single (L1) and double enzyme (L2) digestion products for plu1537-device were loaded onto a 1% BioRad Ready Agarose Mini Gel, then subjected to AGE. See  <A href=" https://2015.igem.org/Team:ZJU-China/Project/Protocol">Protocol</A> for AGE parameters. We use PstI for single digestion, XbaI and PstI for double digestion, then products were determined by AGE analysis. The DNA size standards were the DL5,000 DNA Marker (M2; TaKaRa, Cat#3428A). Bands were visualized with a Shanghai Peiqing JS-380A Fluorescence Imager.
                It can be clearly seen the plu1537 is constructed into the pSB1C3 backbone (Figure 4).
           </P></DIV><DIV class="col-md-6 img-100"><DIV class="cpleft"><P class="kuvateksti">
                Figure 4 digestion confirmation of tcdA1-device in pSB1A2 backbone.
            </P></DIV></DIV></DIV><H4 class="hh4-left" style="padding-top:30px" style="padding-top:30px"> Plasmid Sequencing </H4><P class="p1">
      We have sequenced the parts with standard primers VF2 and VR. The sequence of the 2.5k part shows 100% agreement with the desired sequence.
    </P><H4 class="hh4-left" style="padding-top:30px" style="padding-top:30px"> Toxin Expression</H4><H5 class="hh2"> BACTERIA CULTURE </H5><DIV class="row"><P class="p1">
                The solid or liquid culture medium is LB culture with 34ug/ml chloromycetin and 80mM arabinose.
            </P><P class="p1">
                Both the antibiotics and arabinose are added after the medium cools down to 60℃. 2% (w/v) of agar is added in solid medium. 
           </P><P class="p1">
                It can be clearly seen that the recombinant cology turns red, indicating the expression of reporter mCherry (figure 5, 6), showing that the target gene may be expressed to a great extent.
           </P><DIV class="col-md-6 img-100"><DIV class="cpleft"><P class="kuvateksti">
                Figure 5 Expression of reporter mCherry in LB solid medium with arabinose and chloromycetin.
            </P></DIV></DIV><DIV class="col-md-6 img-100"><DIV class="cpleft"><P class="kuvateksti">
                Figure 6 Expression of reporter mCherry in LB liqiud medium with arabinose and chloromycetin.
            </P></DIV></DIV></DIV><H5 class="hh2"> SDS-PAGE </H5><DIV class="row"><DIV class="col-md-6 text-col-left"><P class="p1">
                5ul sample is loaded in a 10% SDS-PAGE separation gel. We use 250kDa marker Precision Plus Protein? Dual Color Standards #161-0374. Parameters can be seen in protocols.
            </P><P class="p1">
                According to the result of SDS-PAGE (figure 7), target protein (14kDa) is strongly expressed (line 4) compared with the negative control--the wildtype BL21 (DE3) strain (line 1).  
           </P></DIV><DIV class="col-md-6 img-100"><DIV class="cpleft"><P class="kuvateksti">      Figure 7 SDS-PAGE results of four devices we constructed.
            </P></DIV></DIV></DIV><H5 class="hh2"> TERMITES <I>in vivo</I> EXPERIMENTS </H5><DIV class="row"><DIV class="col-md-12 textcenter"><DIV class="cpleft"><P class="kuvateksti">
                Figure 8  toxin effect of tcdA1 on termites.
            </P></DIV></DIV></DIV><P class="p1">We fed the same amount of termites separately with Plu1537-expressing BL21, Plu1537-expressing BL21 embedded with CNC and control group: mCherry-expressing BL21. For more details please go to our <A href=" https://2015.igem.org/Team:ZJU-China/Project/Protocol">Protocol</A></P><P class="p1">It can be clearly seen that Plu1537-expressing BL21 embedded with CNCs has strong toxicity on termites while raw Plu1537-expressing BL21 has little effect. According to the results, functions of <A href="https://2015.igem.org/Team:ZJU-China/Design/CNC">CNCs</A> as reducing the phobotaxis is confirmed. 
          </P><P class="p1">Judging from the SDS PAGE results and toxicity confirmation on termites, we conclude that the device <I>plu1537</I> functions well. We successfully kill termites with toxin protein and our killing strategy proves to be effective and powerful.
          </P><H5 class="hh2"> References</H5><P class="p1">1.	M. Li et al., MOL BIOL REP 36, 785 (2009).
</P><P class="p1">2.	M. S. Kelker et al., PLOS ONE 9, (2014).
</P><P class="p1"><A href="http://www.uniprot.org/uniprot/Q939T0">4.   Cry34Ab1(uniprot)</A></P><P class="p1"><A href="http://www.ebi.ac.uk/pdbe/entry/pdb/4JOX">5.   3D structure of Cry34Ab1</A></P></ARTICLE><ARTICLE><H3 class="hh2" style="border:0;color:red;padding-top:100px" id="pos2">Part: BBa_K1668009</H3><H4 class="hh4-left" style="padding-top:30px">plu0840-device</H4><P class="p1">
        the part <I>plu0840</I> device is composed of arabinose inducible promoter pBad <A href="http://parts.igem.org/Part:">BBa_I0500</A>, toxin protein <I>plu0840</I> coding sequence <A href="http://parts.igem.org/Part:">BBa_K1668006</A> and composite part mCherry <A href="http://parts.igem.org/Part:">BBa_K1668011</A>. Here we use the device to tandem express toxic protein Plu0840 and the reporter mCherry in<I> E.coli</I> BL 21(DE3).
    </P><DIV style="width:100%l;text-align:center;font-size:30px;"><A href="http://parts.igem.org/Part:BBa_K1668009">Sequence and Features</A><H4 class="hh4-left" style="padding-top:30px">Characterization</H4><H5 class="hh2"> OVERVIEW </H5><P class="p1">
        We construct the device <I>plu0840</I> to tandem express insecticidal toxic protein Plu0840 and reporter mCherry. The 72kDa insecticidal toxic protein Plu0840 is used to kill termites in our project.
    </P><P class="p1">
        The exact mechanism of insecticidal toxin protein Plu0840 remains to be revealed. In a 2007 research, scientists first expressed a GST-plu1537 fusion protein in <I>E.coli</I> BL21(DE3) with pGEX-4T-1 vector. The results showed that Plu0840 had weak oral toxicity against two kinds of moth(<I>S. litura and S. exigua</I>)
    </P><P class="p1">
        We clone and standardize the gene into standard plasmid pSB1C3. After confirmation of digestion and sequencing, we transform the plasmid into <I>E.coli</I> BL21(DE3) to achieve better expression level.
    </P><P class="p1">
        We observe that transformants have obviously turned red and figure out the expected protein band in SDS-PAGE. According to<I> in vivo </I>experiments on termites, the toxin effect of Plu0840 is comparatively weak than Plu1537, which is consistent with the research before.
    </P><P class="p1">
        In conclusion, we have successfully cloned and expressed the Plu0840 toxic protein in <I>E.coli</I> BL21(DE3). The Plu0840 is weak toxic to termites.
    </P><H5 class="hh2"> BACKGROUND </H5><P class="p1">
        In 2009 research<I> Cloning and expression analysis of a predicted toxin gene from Photorhabdus sp. HB78</I>, <I>plu0840</I> fused with GST is expressed in <I>.coli</I> BL21(DE3). Engineered strain BL21 was both orally fed and injected in hemocoel to two kind of moth (<I>S. litura </I>and <I>S.exigua</I>)(<I>1</I>).  
    </P><P class="p1">
        According to the result, on the one hand, oral feeding effectively inhibits the growth of larva while has only weak oral toxic effect. On the other hand, hemocoel injection showed negative results.
    </P><P class="p1">
        The research mentions that Plu0840 (Figure 1) shares 55% sequence identity with an enterotoxin Ast from <I>aeromonas hydrophila</I>. <I>Aaeromonas hydrophila</I>, which is connected with gastroenteritis, may lead to altered fluid secretion in mouse. According to a 2002 research, enterotoxin Ast makes weak contributions to fluid secretion compared with two other genes (<I>2</I>) However, judging that TT01 is nontoxic to animals at all, we think Plu0840 may share little similarity with Ast. 
    </P><DIV class="row textcenter"><DIV class="cpleft"><P class="kuvateksti">
              Figure 1 The 3D structure of Plu0840. Copyright 2014, Worldwide Protein Data.
            </P></DIV></DIV><DIV><H3 class="hh2">Construction</H3><H4 class="hh4-left">PCR<P class="p1">
        We use primer pBad F / pBad R to amplify <I>pBad</I> and standard F / plu0840-right R for CDS <I>plu0840</I>.
    </P></H4><H4 class="hh4-left">Seamless assembly<P class="p1">
     We used seamless assembly as our assembly method so restriction digestion and T4 ligation can be avoided. Detailed protocol and instruction for primer design can be seen in our Protocol. By this way prefix sequence, <I>pBad</I>, CDS <I>plu0840</I> and front twenty bases of <I>mCherry</I> sequence can be ligated seamlessly. 
    </P><P class="p1">
      pBad F (F, 5’-3’): ATTCGCGGCCGCTTCTAGAGTTATGACAACTTGACG
    </P><P class="p1">
      pBad R (R, 5’-3’): GCTAGCCCAAAAAAACG
    </P><P class="p1">
      standard F (F, 5’-3’): CCGTTTTTTTGGGCTAGCAGAAAGAGGAGAAATACTAG
    </P><P class="p1">
     plu0840 R (R, 5’-3’): CTAGTATTTCTCCTCTTTCTTTATTTCGATGGGGTCAAAG
    </P></H4><H4 class="hh4-left">Transformation and confirmation<P class="p1">
     After seamless assembly, standard plasmid pSB1C3 containing <I>plu0840</I> gene was transformed into <I> E.coli</I> DH5α. When single colony appeared on the LB plate, we picked out 10 colonies, respectively, as our template for bacteria solution PCR. In order to avoid the appearance of false positive clones, we used mCherry F(a primer on the beginning of <I>mCherry</I>)/ VR as the universal primers for the <I>plu0840</I> is too long to amplify all the parts sequence with VF2/VR .Primers are shown below. The positive clone and its corresponding raw bacteria solution were stored and samples were sent to do DNA sequencing.
    </P><P class="p1">
     mCherry F (F, 5’-3’):  AGAAAGAGGAGAAATACTAGATGGTGAG
    </P><P class="p1">
     VR(F, 5’-3’):  ATTACCGCCTTTGAGTGAGC
    </P></H4><H4 class="hh4-left">Plasmid map<DIV class="col-md-12 textcenter"><DIV class="cpleft"><P class="kuvateksti">
                Figure 2 The plasmid map of BBa_K1668009 <I>plu0840</I> device
              </P></DIV></DIV></H4><H4 class="hh4-left" style="padding-top:30px">RESULTS</H4><DIV class="col-md-6 text-col-left"><H5 class="hh2"> PLASMID CONSTRUCTION </H5><P class="p1">  5-μl samples of the single (L1) and double enzyme (L2) digestion products for plu0840-device were loaded onto a 1% BioRad Ready Agarose Mini Gel, then subjected to AGE. See<A href=" https://2015.igem.org/Team:ZJU-China/Project/Protocol">Protocol</A> for AGE parameters. We use PstI for single digestion, XbaI and PstI for double digestion, then products were determined by AGE analysis. The DNA size standards were the DL5,000 DNA Marker (M2; TaKaRa, Cat#3428A). Bands were visualized with a Shanghai Peiqing JS-380A Fluorescence Imager.
            </P><P class="p1">
                It can be clearly seen the Plu0840 is constructed into the pSB1C3 backbone (Figure 3).
           </P></DIV><DIV class="col-md-6 img-100"><DIV class="cpleft"><P class="kuvateksti">
                Figure 3 Digestion confirmation of device plu0840 in pSB1C3 backbone.
            </P></DIV></DIV></DIV><H4 class="hh4-left" style="padding-top:30px"> PLASMID SEQUENCING </H4><P class="p1">
      We have sequenced the parts with standard primers VF2 and VR. The sequence of the 3.9k part shows 100% agreement with the desired sequence.
    </P><H4 class="hh4-left" style="padding-top:30px"> TOXIN EXPRESSION </H4><H5 class="hh2"> BACTERIA CULTURE </H5><DIV class="row"><P class="p1">
                It can be clearly seen that the recombinant turned red, indicating the expression of  reporter mCherry(figure 4, 5). As mCherry is located behind target gene and shares a promoter with target gene, the target gene may be expressed to a great extent.
           </P><P class="p2">
                The solid or liquid culture medium is LB culture with 34ug/ml chloromycetin and 80mM arabinose. 
            </P><P class="p2">
                Both the antibiotics and arabinose are added after the culture cools down to 60℃. 2%(w/v) of agar is added in solid medium. 
           </P><DIV class="col-md-6 text-col-left"><DIV class="cpleft"><P class="kuvateksti">
                Figure 4 Expression of reporter mCherry in LB solid medium with arabinose and chloromycetin.
            </P></DIV></DIV><DIV class="col-md-6 img-100"><DIV class="cpleft"><P class="kuvateksti">
                Figure 5 Expression of reporter mCherry in LB liqiud medium with arabinose and chloromycetin.
            </P></DIV></DIV></DIV><H5 class="hh2"> SDS-PAGE </H5><DIV class="row"><DIV class="col-md-6 text-col-left"><P class="p1">
                5ul sample is loaded in a 10% SDS-PAGE separation gel. We use 250kDa marker Precision Plus Protein? Dual Color Standards #161-0374. Parameters can be seen in protocols.
            </P><P class="p1">
                According to the result of SDS-PAGE (figure 6), target protein Plu0840 (72kDa) is strongly expressed (line 3) compared with the negative control, native BL21 (DE3) strain without engineering (line 1). 
           </P></DIV><DIV class="col-md-6 img-100"><DIV class="cpleft"><P class="kuvateksti">
                Figure 6 SDS-PAGE results of four devices we constructed.
            </P></DIV></DIV></DIV><H5 class="hh2"> TERMITES <I>in vivo</I> EXPERIMENTS </H5><DIV class="row"><DIV class="col-md-12 textcenter"><DIV class="cpleft"><P class="kuvateksti">
                Figure 7  Toxin effect of tcdA1 on termites.
            </P></DIV></DIV></DIV><P class="p1">
      We fed the same amount of termites separately with Plu0840-expressing BL21, Plu0840-expressing BL21 embedded with CNCs and control group: mCherry-expressing BL21. For more details please go to our <A href=" https://2015.igem.org/Team:ZJU-China/Project/Protocol">Protocol</A></P><P class="p1">
      	In six days, three termites died in Plu0840-expressing group and two in CNCs group. We conclude that Plu0840 may have weak oral toxicity on termites, which also corresponds with results of related research. However, more experimental proof is needed to further confirm the toxicity of Plu0840.    
	</P><H5 class="hh2"> References </H5><P class="p1">1.	M. Li et al., MOL BIOL REP 36, 785 (2009).</P><P class="p1">2.	J. Sha, E. V. Kozlova, A. K. Chopra, INFECT IMMUN 70, 1924 (2002).</P><P class="p1">3.  Plu0840 (uniprot): http://www.uniprot.org/uniprot/Q7N895    </P><P class="p1"><A href="http://www.proteinmodelportal.org/?pid=modelDetail&amp;provider=MODBASE&amp;template=1uasA&amp;pmpuid=1000725544546&amp;range_from=1&amp;range_to=639&amp;ref_ac=Q7N895&amp;mapped_ac=Q7N895&amp;zid=async">4.  3D structure of Plu0840</A></P></DIV></ARTICLE><ARTICLE><H3 class="hh2" style="border:0;color:red;padding-top:100px" id="pos3">Part:BBa_K1668008</H3><H4 class="hh4-left" style="padding-top:30px">tcdA1 device</H4><P class="p1">
        the part TcdA1 device is composed of arabinose inducible promoter pBad <A href="http://parts.igem.org/Part:">BBa_I0500</A>, toxin protein tcdA1 coding sequnce<A href="http://parts.igem.org/Part:">BBa_K1668005</A> and composite part mCherry <A href="http://parts.igem.org/Part:">BBa_K1668011</A>.
    </P><P class="p1">
        We use the device to tandem express toxic protein TcdA1 and mCherry. Toxic protein TcdA1 is a macro channel forming toxin used for termite control in our project and mCherry is a reporter. 
    </P><DIV style="width:100%l;text-align:center;font-size:30px;"><A href="http://parts.igem.org/Part:BBa_K1668008">Sequence and Features</A><H4 class="hh4-left" style="padding-top:30px">Characterization</H4><H5 class="hh2"> OVERVIEW </H5><P class="p1">
        We construct the device TcdA1 to tandem express toxic protein TcdA1 and reporter mCherry. Toxic protein TcdA1 is used to kill termites in our project. 
    </P><P class="p1">
        TcdA1, one of the biggest proteins in bacteria (285kDa), is first found in <I>Photorhabdus luminescens</I>. It forms channels and assists other toxins across the cell membrane. It belongs to tc toxic protein family, which is widely distributed among different gram-negative and gram-positive bacteria.
    </P><P class="p1">
        We clone and standardize the gene into standard plasmid pSB1C3. After confirmation of digestion and sequencing, we transform the plasmid into  <I>E.coli</I> BL21(DE3) to achieve better expression level. Despite we observe that transformants have obviously turned red, we didn’t figure out the expected protein band in SDS-PAGE. Judging that the protein is considerably huge in bacteria, more improvements are needed.
    </P><DIV class="row"><H5 class="hh2"> BACKGROUND </H5><DIV class="col-md-6 text-col-left"><P class="p1">
            TcdA1 is a pore-forming macro-protein, which can keep the ability to form a pore in a large pH range (from 4 to 11). To be noticed, at pH11, the pore-forming activity of tcdA1 is more than 100-fold greater than at pH 6. As the midguts of most insects are alkaline, tc toxic proteins are effective by feeding on insects, including termites.
        </P><P class="p1">
            In 2013, the structure of TcdA1 was revealed by researchers and reported in nature(1). As displayed in figure1a&amp;b, the tcdA1 is composed of three parts: N-terminal a-helical domain(brown), the central b-sheet domain(green) and the C-terminal pore-forming domain(yellow). The protein has two states: pre-pore state and pore state. The pore-forming domain (Figure 1c) sticks out to form pore, changing into pore state (Figure 2). 
        </P><P class="p1">
            Moreover, the tcdA1 toxin helps other toxins to enter the cell membrane. Naturally in strain TT01, tcdA1 is expressed homologously with other toxins, for example, tcdB1 and tcc toxins. TcdA1 helps to transfer the latter into the cell to maximum the toxic effect(Figure 3).
        </P></DIV><DIV class="col-md-6 text-col-right"><DIV class="cpleft"><P class="kuvateksti">
                Figure 1 The 3D structure of tcdA1. Copyright 2013, Nature Publishing Group.
            </P></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-12 textcenter"><DIV class="cpleft"><P class="kuvateksti">
                Figure 2 Comparison between pre-pore state and pore state of tcdA1. Copyright 2014, Nature Publishing Group
            </P></DIV></DIV><DIV class="col-md-12 textcenter"><DIV class="cpleft"><P class="kuvateksti">
                Figure 3 The function of tcdA1 in toxin transportation(1). Copyright 2013, Nature Publishing Group.
            </P></DIV></DIV></DIV><DIV><H3 class="hh2">Construction</H3><H4 class="hh4-left">PCR<P class="p1">
        We use primer pBad F/pBad R to amplify <I>pBad</I> and standard F/ tcdA1-right R for CDS <I> tcdA1</I>.
    </P></H4><H4 class="hh4-left">Seamless assembly<P class="p1">
     We used seamless assembly as our assembly method so restriction digestion and T4 ligation can be avoided. Detailed protocol and instruction for primer design can be seen in our Protocol. By this way prefix sequence, <I>pBad</I>, CDS <I> tcdA1</I> and front twenty bases of <I>mCherry</I> sequence can be ligated seamlessly.
    </P><P class="p1">
      pBad F (F, 5’-3’): ATTCGCGGCCGCTTCTAGAGTTATGACAACTTGACG
    </P><P class="p1">
      pBad R (R, 5’-3’): GCTAGCCCAAAAAAACG
    </P><P class="p1">
      standard F (F, 5’-3’): CCGTTTTTTTGGGCTAGCAGAAAGAGGAGAAATACTAG
    </P><P class="p1">
     tcdA1 R (R, 5’-3’): CTAGTATTTCTCCTCTTTCTTTATTTAATGGTGTAGCGAA
    </P></H4><H4 class="hh4-left">Transformation and confirmation<P class="p1">
      After seamless assembly, standard plasmid pSB1C3 containing <I> tcdA1</I> gene was transformed into <I> E.coli</I> DH5α. When single colony appeared on the LB plate, we picked out 10 colonies, respectively, as our template for bacteria solution PCR. In order to avoid the appearance of false positive clones, we used tcdA1_M1(a primer on the right part of tcdA1)/VR as the universal primers for the CDS<I>tcdA1</I> is too long to amplify all the parts sequence with VF2/VR .Primers are shown below. The positive clone and its corresponding raw bacteria solution were stored and samples were sent to do DNA sequencing.
    </P><P class="p1">
     tcdA1_M1(F, 5’-3’):  CTGCCAATCTATGCCACACC
    </P><P class="p1">
     VR(F, 5’-3’):  ATTACCGCCTTTGAGTGAGC
    </P></H4><H4 class="hh4-left">Plasmid map<DIV class="col-md-12 textcenter"><DIV class="cpleft"><P class="kuvateksti">
                Figure 4 The plasmid map of BBa_K1668008 <I>tcdA1</I> device
              </P></DIV></DIV></H4><H4 class="hh4-left" style="padding-top:30px">RESULTS</H4><DIV class="col-md-6 text-col-left"><H5 class="hh2"> PLASMID CONSTRUCTION </H5><P class="p1">
                5μl samples of the double enzyme digestion products for tcdA1-device were loaded onto a 1% BioRad Ready Agarose Mini Gel, then subjected to AGE. See (protocol) for AGE parameters. Sizes of the XbaI and PstI–cleaved assemblies were determined by AGE analysis. The DNA size standards were the DL5,000 DNA Marker (M2; TaKaRa, Cat#3428A) and 1kb DNA Ladder (Dye Plus)(M2; TaKaRa, Cat#3426A). Bands were visualized with a Shanghai Peiqing JS-380A Fluorescence Imager.
            </P><P class="p1">
                First we construct the tcdA1 device in pSB1A2. Our target fragments can be clearly seen in the right position (figure 6). As the fragment is a little big(7.2k), the efficiency is low when we change the backbone to pSB1C3 and the unwanted fragment is hard to explain(Figure 7).
           </P></DIV><DIV class="col-md-6 img-100"><DIV class="cpleft"><P class="kuvateksti">
                Figure 6 Digestion confirmation of device tcdA1 in pSB1A2 backbone.
            </P></DIV><DIV class="cpleft"><P class="kuvateksti">
                Figure 7 Digestion confirmation of device tcdA1 in pSB1C3 backbone.
            </P></DIV></DIV></DIV><H4 class="hh4-left" style="padding-top:30px"> PLASMID SEQUENCING </H4><P class="p1">
      We have sequenced the parts with standard primers VF2 and VR. The sequence of the 9.7k part shows 100% agreement with the desired sequence.
    </P><H4 class="hh4-left" style="padding-top:30px"> TOXIN EXPRESSION </H4><H5 class="hh2"> BACTERIA CULTURE </H5><DIV class="row"><P class="p1">
                The solid or liquid culture medium is LB culture with 34ug/ml chloromycetin and 80mM arabinose. 
            </P><P class="p1">
                Both the antibiotics and arabinose are added after the culture cools down to 60℃. 2%(w/v) of agar is added in solid medium.  
           </P><P class="p1">
                It can be clearly seen that the recombinant turned red, indicating the expression of  reporter mCherry. As mCherry is located behind target gene and shares a promoter with target gene, the target gene may be expressed to a great extent.
           </P><DIV class="col-md-6 text-col-left"><DIV class="cpleft"><P class="kuvateksti">
                Figure 8 Expression of reporter mCherry in LB solid medium with arabinose and chloromycetin.
            </P></DIV></DIV><DIV class="col-md-6 img-100"><DIV class="cpleft"><P class="kuvateksti">
                Figure 9 Expression of reporter mCherry in LB liqiud medium with arabinose and chloromycetin.
            </P></DIV></DIV></DIV><H5 class="hh2"> SDS-PAGE </H5><DIV class="row"><DIV class="col-md-6 text-col-left"><P class="p1">
                5ul sample is loaded in a 10% SDS-PAGE separation gel. We use 250kDa marker Precision Plus Protein™ Dual Color Standards #161-0374. Parameters can be seen in protocols.

            </P><P class="p1">
                According to the result of SDS-PAGE, target protein(285kDa) is not eyeable(line 2) compared with the negative control, native BL21 (DE3) strain without engineering(line 1). However, the recombinant tcdA1 strain turns red, indicating that it expressed mCherry, which can be confirmed in SDS-PAGE.
           </P><P class="p1">
                There are two possible explanations to the results. One is that the expression level of macro protein like TcdA1 is extremely low, which is unrecognizable in SDS-PAGE. The other is that TcdA1 didn’t express out of unknown reason. 
           </P></DIV><DIV class="col-md-6 img-100"><DIV class="cpleft"><P class="kuvateksti">
                Figure 10 SDS-PAGE results of four devices we constructed.
            </P></DIV></DIV></DIV><H5 class="hh2"> TERMITES <I>in vivo</I> EXPERIMENTS </H5><DIV class="row"><DIV class="col-md-12 textcenter"><DIV class="cpleft"><P class="kuvateksti">
                Figure 11  Toxin effect of tcdA1 on termites.
            </P></DIV></DIV></DIV><P class="p1">
      We fed the same amount of termites separately with tcdA1-expressing BL21, tcdA1-expressing BL21 embedded with CNCs and control group mCherry-expressing BL21. For more details please go to our <A href=" https://2015.igem.org/Team:ZJU-China/Project/Protocol">Protocol</A></P><P class="p1">
      However, only one termite died in six days and further improvements are needed. In addition, further works are needed on the large-amount expression of TcdA1.
    </P><H5 class="hh2"> References </H5><P class="p1">
      1.	C. Gatsogiannis et al., NATURE 495, 520 (2013-03-20, 2013).
	</P><P class="p1">
      2.	D. Meusch et al., NATURE 508, 61 (2014).
	</P></DIV></ARTICLE><ASIDER id="popView2"><HEADER>Composite Part</HEADER><UL id="nav2"><LI><A href="#pos1">BBa_K1668010	device <I>plu1537</I></A></LI><LI><A href="#pos2">BBa_K1668009	device <I>plu0840</I></A></LI><LI><A href="#pos3">BBa_K1668008	device <I>tcdA1</I></A></LI></UL></ASIDER></DIV><DIV id="gl-footer" class="cf" style="max-width:1440px"><DIV id="gl-footer-copyright" class="cf"><DIV class="gl-col1 cf"><SPAN>© 2015 ZJU-IGEM TERMINATE.</SPAN></DIV><UL class="gl-col2 cf"><LI><A href="http://www.igem.org/Main_Page">IGEM</A></LI><LI><A href="http://www.zju.edu.cn">Zhejiang University</A></LI></UL></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>