Team:ZJU-China/Project/Protocol
Home
Team Part Team Part
Basic Part Basic Part
Improved  Part Improved  Part
Composite Part Composite Part
Parts
Overview Quick view for our project Termite Issue The Termite Issue
Design
Toxins Manufacture
Bacteria Carrier
Termites
Protocol Quick view for Protocol
Project
Overview Quick view for Outreach
Society
Survey
Consultation
Syn-bio Debate
Application
Device
Promotion
Safety
Future
Public Promotion
Poly-cards
Syn-bio Share
Community Lab
Seminar
Collaboration
SynenergeneⅠ
SynenergeneⅡ
Synenergene Ⅲ
Other Teams
Outreach
Members Introduce our Members
Attributions Introduce the Attributions
Acknowledgement Introduce the Acknowledgement
Sponsors Introduce our Sponsors
Team
Modeling Overview Introduce our Members
Modeling α Termite Simulation
Modeling β Mass transfer model
Modeling γ Degradation model
Modeling
Notebook
Experiment Results toxins manufacture
Experiment Result bacteria carrier
Experiment Result termites simulation
Social Result social results
Results
Safety
Device Introduce our Device and Instruction Manual
Promotion The Promotion
Safety The Safety
Future Future
Application
Protocol
Preparation
Formulations
1.TAE buffer (50×, 1 liter)
Tris
Acetic acid glacial
0.5mol/L EDTA (pH8.0)
242g
57.1mL
100mL
2.Gel
Dissolve agarose to a final concentration of 1%(by mass) in TAE buffer in a glass bottle.          Heat the solution in the microwave with frequent stirring to dissolve the agarose homogenously. ~1 minute/200ml solution          Place the solution in a warm water bath for 5 mins.          Add 2.5 µL ExRed (1:10000) per 25 mL of the solution and mix well.           Pour 25mL of solution per small gel tray. (the gel trays and combs should be pre-cleaned with water and wiped dry).          -Note for combs: 11-well combs hold about 15 µL liquid, 6-well combs hold about 50 µL          For the small set: small trays hold 25ml, large trays hold 50ml          Wait for the gels to solidify. ~15 mins
3.ddH2O
Sterilize the UP water. 1mL per EP tube. Store at -20℃.
4.LB liquid culture (1 liter, pH7.2~7.4)
Tryptone
Yeast Extract
NaCl
10g
5g
10g
Fill with sterile water up to 1000 mL mark.
5. LB agar plates
Pour 200mL of LB liquid culture into conical flask, add 20g (2%, w/v) agar powder, swirl the solution until the powder is dissolved.          Cap loosely and tape with autoclave tape.          Autoclave for 30 minutes under 121℃.          Wait till cool enough so that you can touch it for a minute (60℃)          Add appropriate volume of antibiotics (1000×). For 200 mL of LB Agar, add 200 uL of each antibiotic needed.          Swirl to mix, transfer 20 mL into each plate. Avoid bubbles.           Allow plates to cool and solidify. Mark antibiotic on side of plate.           (If LB Agar solidifies before being poured, microwave to liquefy again.)
6. PBS buffer (10×,1 liter, pH~6.8)
NaCl
KCl
Na2HPO4
KH2PO4
80g
2g
14.2g
2.7g
Fill with 800 mL sterile water. Stir to dissolve the powder.          Use HCl to adjust pH. Fill with sterile water until 1000 mL.          Autoclave for 30 minutes under 121℃.          Store under room temperature.
7.1M Tris (1 liter, pH6.8)
Dissolve 121.4g Tris into 1000mL ddH2O.           Use concentrated HCl (about 70mL) to adjust pH to 6.8.           Fill with sterile water up to 1000 mL mark.
Making competent cells
(From https://2014.igem.org/Team:ZJU-China/Protocol )
1. Preparation of heat shock competent cells.*The most important thing to remember is to keep the cell as cold as possible at all the time.
Preparation：
Autoclave sterilization：Erlenmeyer flask；1L reagent bottles (for TFB)；100ml centrifuge bottles (bottles and caps need to be sterilized separately)；vacuum filter flask; 0.22nm filter membrane；80%glycerol; LB broth.
Procedure:
1.  Streak out E. coli (DH5α, DH10β, BL21, BW25113 and so on) onto LB plate. Cultivate it inversely.
2.  Growing overnight @37℃
3.  Pick a single colony and inoculate overnight using 5mL LB Broth, shake it @37℃, 200r/min.
4.  Inoculate 2ml overnight culture to 100ml LB broth, shake it @37℃, 200r/min until OD600=0.4 or so. (About 3.5 hours)     * Remember to open spectrophotometer in advance and use LB as blank.
5.  Chill the cell on ice for 15min.
6.  Centrifuge the cells at 5000rpm for 10min @4℃.     *Remember to balance the centrifuge bottles/tubes before centrifugation.
7.  Discard the supernatant.
8.  Resuspend the cells with 2ml TFB. Then fill it with TFB.
9.  Centrifuge at 5000rpm for 10min @4℃.
10. Add some TFB buffer to resuspend the cell, and then fill the tube with TFB buffer.
11. Centrifuge at 5000rpm for 10min @4℃.
12. Discard the supernatant.
13. Resuspend the cell with 3ml TFB buffer, then add 700ul 80%glycerol to 15% of final concentration of glycerol.
14. Fill the 1.5ml EP tube with 50ul liquid. Quick-freeze in liquid nitrogen. Stock in -80℃.
Preparation of TFB:
Mother solution: CaCl2: 0.5M, MgCl2: 1M.
Reagent
CaCl2 (Final concentration 100mM)
MgCl2 (Final concentration 70mM)
NaAc(add powder) (Final concentration 40mM)
500ml
100ml
35ml
1.64g
Using acetic acid to adjust pH to 5.5, Filtrated, Stock @4℃.
2. Preparation of Electrocompetent Cells.
Preparation:
Autoclave sterilization：Erlenmeyer flask；1L ddH2O；1L 10% glycerol; 100ml centrifuge bottles (bottles and caps need to be sterilized separately)；LB broth.
Procedure:
1.  Inoculate 5ml L-broth with a single colony of E. coli. Incubate 5 hours to overnight at 37°C on a roller or with moderate shaking.
2.  Inoculate a volume of L-broth contained in an appropriately sized side-arm flask with one-tenth volume of the culture (i.e. 1 ml of culture to 100 ml L-broth). Grow cells at 37°C with shaking (200-300 rpm) to an OD600 of 0.55 to 0.6.
3.  Chill the cells in an ice-water bath for 10 to 15 minutes and transfer to a pre-chilled centrifuge bottle. (Divide the culture if required.)
4.  *Cells should be kept at 2°C for all subsequent steps.
5.  Pellet the cells by centrifugation at 5000 rpm, 4°C for 10 minutes.
6.  Pour off the supernatant and resuspend the cells in 1mL ice-cold ddH2O. Add 100mL ice-cold ddH2O. Centrifuge the cells as in step 4.
7.  Pour off the supernatant immediately and resuspend the cells in the small amount of fluid remaining in the bottle.
8.  *The pellet may be very loose. Exercise care and pour off the supernatant immediately.
9.  Add 50mL of ice-cold WB (10% glycerol). Centrifuge the cells, again as in step 4. Pour off the supernatant immediately and resuspend the cells in the remaining fluid.
10. Place the cell suspension in an appropriately-sized, narrow-bottom tube that has been pre-chilled.
11. Add to the cells an amount of ice-cold 10% glycerol equal to 0.01 of original culture volume (1 ml for a culture of originally 100 ml) and mix well.
12. Aliquot 40 ul of the cells to pre-chilled EP tubes. Freeze the cells by incubation in a liquid nitrogen bath. Store at -80°C.
Cloning
Plsmid miniprep
We use AxyPrep Plasmid Miniprep Kit.
Usually we use harvest E.coli liquid culture which is added antibiotic marker. Bacterium have grown for 12-16 hours.
https://static.igem.org/mediawiki/2014/6/6d/AxyPrep_Plasmid_Miniprep_Kit.pdf
Streptomyces genomic DNA extraction
1. Take the spores and inoculate them into YEME medium. Cultivate bacteria for 4-5 days. Draw 2 ml bacterium liquid into 2 ml centrifuge tube and centrifuge at 3000 rpm for 15 min to collect bacteria.
2. Add ddH2O and wash the bacteria twice.
3. Use 500ul lysozyme solution (RNaseA plused) to suspend mycelium and incubate at 37℃ for approximately 50 minutes, making the solution translucent.
4. Use 500ul 2% SDS solution to mix and oscillatefor about 1 minute until the viscosity of solution decreases significantly. Then open the lid and conserve at 55℃ for about 30 min.
5. Cool it to room temperature and then add 1/10 volume of 3 mol/L sodium acetate (246 g/L) and 200ul chloroform/isoamyl alcohol (24:1), vortex oscillation for about 1 min and centrifuge at 4℃, 12000rpmfor 5 min.
6. Extract the upper liquid,abandon medium white layer, and use chloroform/isoamyl alcohol solution to extract the layer until the middle tier disappears.
7. Take the supernatant, add 2 times volume of anhydrous ethanol, conserve at -20℃ for 30 min until the flocculent precipitate DNA appears.
8. Centrifuge and abandon supernatant, wash DNA precipitation with 70% ethanol and use spear to absorb liquid for a few minutes to dry it.
9. Dissolve it in moderate ddH2O, use spectrophotometer to measure DNA concentration and purity and save at -20℃.
PCR
We use TaKaRa PrimerSTAR HS DNA Polymeras to amplify target fragment.
PCR reaction system are usually 20μL.
Detailed PCR Reaction Mix and PCR conditions are in the manual.
Gel electrophoretic analysis
To detect the PCR product, we add 2μL or 5μL sample into 15μL-gel well. Set the timer and voltage to 80V (100V if using large tray) and 20 min.
To detect the double enzyme digetion products, we add 15μL sample into 15μL-gel well. Set the timer and voltage to 80V (100V if using large tray) and 20 min.
To do DNA gel extraction, we add 40μL  or 45μL  sample into 50μL-gel well. Set the timer and voltage to 80V (100V if using large tray) and 20 min.
Bands were visualized with a Shanghai Peiqing JS-380A Fluorescence Imager.
PCR Cleanup
We use AxyPrep PCR Cleanup to clean up the PCR product.
Usually put 4 or 8 tubes of PCR product (20μL per tube) into one EP tube and then clean up.
https://static.igem.org/mediawiki/2015/e/e6/Protocol_AxyPrep_PCR_Cleanup.pdf
DNA gel extraction
We use AxyPrep DNA Gel Extraction Kit.
https://static.igem.org/mediawiki/2014/a/a9/AxyPrep_DNA_Gel_Extraction_Kit_.pdf
Transformation
(From https://2014.igem.org/Team:ZJU-China/Protocol )
1. Heat shock transformation.
Preparation:
Water bath 42℃, rotary shaker @37℃, 150rpm, ice, E. coli @-80℃.
Procedure:
1.  Thaw the competent cells or super competent cells on ice. About 10 minutes.
2.  Add 1ul of plasmids (about 30ng) to cells and swirl it gently.
3.  Incubate the cells on ice for 30 minutes.
4.  Heat pulse the tube in a 42℃ water bath for 80~90 sec.     *The length of time of the heat pulse is critical for obtaining the highest efficiencies.
5.  Incubate the cells on ice for 2min.
6.  Add 1ml of LB medium to the tube. Then shake it @37℃ 150rpm for about 1h.
7.  Centrifuge the tube at less than 5000rpm for 2min, then discard the supernatant making the residue less than 50ul.
8.  Coat the plates containing specific antibiotics with the culture.
9.  Cultivate the plate inversely @37℃ for about 12h (12h~14h).
2. Electroporation of the Cells. (for plasmid)
Preparation:
0.1cm electroporation cuvettes; rotary shaker; ice; E. coli @-80℃.
Procedure:
1.  Set the MicroPulser to Eco 1 state (2.5 kV, 25 µF). Add 1 µl plasmid DNA to tubes containing 40 µl fresh or thawed cells on ice. Mix by swirling with pipette tip.   *The volume of DNA added to the cells should be kept small.     *Adding DNA up to one-tenth the volume of cells can reduce the efficiency of electroporation 2- to 3-fold.
2.  Transfer the DNA and cells to a pre-chilled electroporation cuvette (0.1 cm electrode gap) using a narrow pipette tip.
3.  Wipe any ice or water from sides of cuvette using a Kimwipe. Place the cuvette into the sample chamber.
4.  Energize MicroPulser and deliver the pulse by pushing in both charging buttons simultaneously and holding until a short beep is heard.       *Note the time constant of the pulse and the actual voltage delivered.
5.  Remove the cuvette from the sample chamber. Immediately add 1 ml SOC medium and transfer the cells to a sterile EP tube using a Pasture pipette.     *Failure to immediately add SOC to the electroporated cells can significantly reduce cell viability and decrease transformation efficiency.
6.  Incubate cultures for 1 hour at 37°C (if cells have pKD46, incubate them at 30°C) on a roller or with moderate shaking to allow for plasmid expression.
7.  Plate 100uL of the electroporation mixture on L-agar plates supplemented with the appropriate antibiotics. Incubate plates at 37°C (if cells have pKD46, incubate them at 30°C).
Preparation of SOC Broth (1 liter)
Bacto tryptone
Bacto yeast extract
NaCl
KCl
MgCl2
MgSO4
Glucose
20.0 g
5.0 g
0.6 g
0.5 g
10 mM
10 mM
20 mM
Note: SOC is identical to SOB, except that it contains 20 mM glucose.
Dissolve tryptone, yeast extract, sodium chloride, and potassium chloride in a final volume of 970 ml distilled H2O. Sterilize by autoclaving.
After autoclaving, allow the solution to cool to 60°C, and add 20 ml of a sterile 1 M glucose stock to make the media 20 mM with respect to glucose.
Just prior to using, add 10 ml of magnesium stock to the SOC broth to make the media 20 mM with respect to magnesium.
Ristriction digest
Usually use FastDigest Enzyme to digest plasmid and use Conventional Restriction Enzyme to digest PCR products.
Detailed digestion reaction system can be seen in the manual.
https://static.igem.org/mediawiki/2014/a/ae/PstI.pdf
https://static.igem.org/mediawiki/2014/5/54/XbaI.pdf
https://static.igem.org/mediawiki/2015/b/b1/FD-Bcul.pdf
https://static.igem.org/mediawiki/2015/4/40/FD-Ndel.pdf
https://static.igem.org/mediawiki/2015/5/5e/FD-Xbal.pdf
https://static.igem.org/mediawiki/2015/2/20/FD-Pstl.pdf
Ligation
We use Thermo T4 DNA Ligase to ligate the digested products. See the manual.
https://static.igem.org/mediawiki/2014/5/5e/T4_DNA_Ligase.pdf
Seamless assembly
We use Vazyme ClonExpress Ⅱ One Step Cloning Kit to ligate one target fragment with plasmid backbone and Vazyme ClonExpress MultiS One Step Cloning Kit to ligate several (up to 5) target fragments with plasmid backbone.
https://static.igem.org/mediawiki/2015/f/f9/One_step-%E2%85%A1.pdf
https://static.igem.org/mediawiki/2015/8/80/One_step-MultiS.pdf
Non-lab
Checking primers
Design primers manually. Use NCBI Primer-BLAST Tool
( http://www.ncbi.nlm.nih.gov/tools/primer-blast/index.cgi?LINK_LOC=BlastHome ) to check the paremeters of primers.
-no individual secondary structures i.e. hairpins
-no heterostructure with the forward and reverse primers together
-free energy of primers should be greater than -4 kCal
-GC content should be around 50% (40-60% is okay)
Also use DNAMAN7 to check the rRNA secondary structure.
Drawing plasmid maps
Draw plasmid maps by SnapGene 2.3.2 (http://www.snapgene.com/).
Confirmation
We usually adopt following four methods step by step to confirm our assembly products.
Bacterium solution PCR
We use CWBIO 2×Taq MasterMix to amplify the desired fragment in the plasmid of bacterium.
PCR reaction system (20μL)
RNase-Free Water
2×Taq MasterMix
Primer 1 (10μM)
Primer 2 (10μM)
Bacterium solution template
8μL
10μL
0.5μL
0.5μL
1μL
*Primer 1 and Primer 2 are universal primers (VF2 and VR) to prevent false positive clone.
**Bacterium solution template are added into the PCR tube in fume cupboard.
Programming The Thermocycler
Initial Denaturation: 94℃ for 10min
LOOP: 25-35 cycles
CYCLE:
Denaturation: 94℃ for 30s
Annealing: calculated temperature (typically 55-65℃) for 30s
Elongation: 72C for 30s per kb
Final Elongation: 72C for 10min
Store: 4℃
The positive clone will be tested by double enzyme digestion before sequenced.
Double enzyme digestion test
We use FastDigest enzymes to digest the plasmid containing our target fragment. Detailed digestion reaction system see above.
When doing gel electrophoretic analysis, keep in mind to add three kinds of control:
- circular plasmid
- linearized plasmid under single enzyme digestion
- correct linearized plasmid under double enzyme digestion
Usually 15μL digested products are added in the gel well.
SDS-PAGE
Reagents:
5x Sample Buffer
10% w/v
10 mM
20 % v/v
0.2 M
0.05% w/v
SDS
Dithiothreitol, or beta-mercapto-ethanol
Glycerol
Tris-HCl, pH 6.8
Bromophenolblue
Should add up to 8M urea for really hydrophobic proteins
1x Running Buffer
25 mM
200 mM
0.1% (w/v)
Tris-HCl
Glycine
SDS
1x Running Gel Solution
For different applications increase your desired percentage acrylamide, make up 30ml of running gel by selecting one of the following percentages and mixing the ingredients shown below. After adding TEMED and APS your gel will polymerize fairly quickly, so do not add these until you are sure you are ready to pour.
H2O
1.5 M Tris-HCl, pH 8.8
20% (w/v) SDS
Acrylamide/Bis-acrylamide(30%/0.8% w/v)
10%(w/v)ammonium persulfate(APS)
TEMED
7%:15.3 ml  /  10%:12.3 ml  /  12%:10.2 ml  /  15%:7.2 ml
7%:7.5 ml  /  10%:7.5 ml  /  12%:7.5 ml  /  15%:7.5 ml
7%:0.1 ml  /  10%:0.15 ml  /  12%:0.15 ml  /  15%:0.15 ml
7%:6.9 ml  /  10%:9.9 ml  /  12%:12.0 ml  /  15%:15.0 ml
7%:0.15 ml  /  10%:0.15 ml  /  12%:0.15 ml  /  15%:0.15 ml
7%:0.02 ml  /  10%:0.02 ml  /  12%:0.02 ml  /  15%:0.02 ml
Stacking Gel Solution (4% Acrylamide):
H2O
0.5 M Tris-HCl, pH 6.8
20% (w/v) SDS
Acrylamide/Bis-acrylamide  (30%/0.8% w/v)
10% (w/v) ammonium persulfate (APS)
TEMED
3.075 ml
1.25 ml
0.025 ml
0.67 ml
0.025 ml
0.005 ml
Pouring the Gels:
Choose a percentage acrylamide based on the molecular weight range of proteins you wish to separate:
% Gel
7
10
12
15
M.W. Range
50 kDa - 500 kDa
20 kDa - 300 kDa
10 kDa - 200 kDa
3 kDa - 100 kDa
Mix the ingredients needed for the chosen percentage and pour the solution quickly into gel casting form.
be sure to leave a some room for the stacking gel
usually leave about 2 centimeters below the bottom of the comb for the stacking gel.
Look for bubbles and remove them
Wait for about 30 minutes for the gel to polymerize completely. (If you always use fresh ammonium persulfate, you're gel may polymerize more quickly and reliably.)
While waiting mix the reagents for the stacking gel, but LEAVE OUT the APS and TEMED until you are ready to pour this gel
When the running gel is polymerized wash out the butanol completely or your stacker may separate from the gel and you will get ugly running artifacts.
Mix in the polymerizing reagents and pour the stacking gel on top of the running gel.
Insert your combs trying not to get bubbles stuck underneath and allow another 30 min - 1 hour for complete polymerization.
Preparing your Sample:
Mix your protein 4:1 with the sample buffer. Heat your sample by either:
a) Boiling for 5-10 minutes (Works for most proteins)
b) 65 degrees C for 10 minutes (If you have smearing using the above procedure)
c) 37 degrees for 30 minutes (Membrane proteins or others that do not enter the gel otherwise may benefit from this type of sample preparation)
Running your gel:
Clamp in your gel and fill both buffer chambers with gel running buffer according to the instructions for your specific apparatus.
Pipet your sample into the gel adjusting the volume according to the amount of protein in your sample. Be sure to include a lane with molecular weight standards.
Now attach your power leads and run the gel until the blue dye front reaches the bottom.
Run at 250 V constant which in a four to twenty percent mini gel needs about 30 minutes total run time, but adjust to the thickness of your gel, the power supply used and the resolution desired.
Remove the gel for the power supply and process further
Visualize proteins using Coomassie Brilliant Blue.
Reference:  Laemmli, U. K. (1970). Cleavage of structural proteins during the assembly of the head of bacteriophage T4. Nature 227, 680-685.
Sequencing
Last but not least, we send our tested samples to TSING KE Biological Technology (http://www.tsingke.net/) to do DNA sequencing.
Protocol
Preparation
Cloning
Non-lab
Confirmation
© 2015 ZJU-IGEM TERMINATE.
IGEM
Zhejiang University
