Team:UCC Ireland/Notebook
UCC iGEM
Toggle navigation
Home
Project
Limitless Lactis
iRFP In-Vitro Reporter
Leishmaniasis
Proof
Demonstrate
Outreach
HP Silver
HP Gold
Engagement
Integrated Practices
Team
Notes
Parts
Collaborations
iGEM Teams
Synenergene
Application Scenario
Techno-Moral Newsletter
Techno-Moral Blog
Interlab
Achievements
Attributions
Notebook
Week beginning 6th June
During this week we transformed parts from iGEM DNA distribution kit, these included:
BBa_J04450
BBa_J23119
BBa_J06702
BBa_K516032
We had a team meeting with our Secondary Instructor Dr. Mark Tangney to develop a workflow chart featuring the various stages of our project.
We also grew up the pNZ44 plasmid.
Week beginning 13th June
We performed a plasmid prep of pNZ44
We completed a restriction digest and pIRFP amplification
The following were transformed with the use of either Ampicillin or Chloramphenicol LB plates:
BBa_J04450- with Ampicillin
BBa_J04450 with Chloramphenicol
J23119 with Ampicillin
K584001 with Chloramphenicol
A plasmid prep of J23119 was performed, then a restriction digest was carried out.The J23119 backbone was linearised.
Subsequently J23119 and K516032 were ligated into psb1C3.This was then transformed into E-Coli DH5α.
K516032 and K608002 were also transformed into DH5α.
With the DNA distribution kit 7 transformations were completed
BFP (plated on plate 1)
YFP (plate 1 )
Terminator (plate 3)
Banana Fluorescent Protein (plate 4)
Week beginning 20th June
A restriction digest K516032 was performed with Xpa and Pst1, a gel was ran of the digest and then a gel extraction was carried out.
The following plasmid preps were completed
K864100
B0017
K608002
We resuspended the Mesothelin G-blocks which arrived from IDT.
pNZ44 was digested with XbaI
The ligation of K864100 (insert) and another parts into psb183 was performed.
Overnight cultures of K592100 was completed.
Week beginning 27th June
A plasmid prep of BBa_592100 was performed                   Several ligations were completed, these included:
PiRFP
PiRFP control
G block (F1)
G block (F3)
G block (LJM11)
Other ligations were carried out with:
K592160
K60800
PSB1A3
LJM11 was unsuccessfully transformed into DH5α, as was MSLN
These transformations but competent cells -C2987H completed.
A plasmid prep was performed with the generator (from DNA distribution kit).
To check the ligations carried out earlier in the week a colony PCR was completed for:
pIRFP
LJM11
Negative control
Positive pIRFP control
There was a gel electrophoresis with the PCR products.
Overnights were prepared with the positive colonies.
Uncut pNZ44 and Puc19 were transformed into competent cells C2987I in order to check the integrity of the transformation process.
Several plasmid preps were completed with a variety of digestion products
pIRFP1 (enzymes: EcoRI and NdeI)
pIRFP2 (enzymes: Pst)
LJM11 (Pst and HindIII)
Lane 1:
pIRFP1 (NDEI and ECORI)
Lane 2:
pIRFP8 (NDEI and ECORI)
Lane 3:
pIRFP1 (PSTI)
Lane 4:
pIRFP8 (PSTI)
Lane 5:
LJM11 (1) (HNDIII and PSTI)
Lane 6:
LJM11 (2) (HNDIII and PSTI)
DEDUCTION FROM GEL: Pirfp 1 cut twice but too high.
Pirfp 8 only cut once.
Single cuts didnt appear.
LJM11 didnt appear
With the aid of pIRFP Forward primer and pIRFP reverse primer a PCR was performed of pIRFP                   The PCR products were then digested with NdeI and EcoRI as well as with BamHI and HindIII                   pNZ44 was digested and a plasmid prep was performed.
Resulting gel can be seen below(two ladders used 1 kb and 100b), lanes are as follows;
Lane 1:
LJM11 1
Lane 2:
LJM11 2
Lane 3:
pIRFP1 (ECORI and NDEI)
Lane 4:
pIRFP1 (PSTI)
Lane 5:
pIRFP8 (ECORI and NDEI)
Lane 6:
pIRFP8 (PSTI)
DEDUCTION FROM GEL: LJM11 1 and 2 didnt work.
Pirfp 1 double cut cut twice but too high. Single cut didnt appear.
PIrfp 8 just didn’t appear.
Week beginning 4th July
A double digest was competed with pIRFP1 with the enzymes EcoRI and NdeI. The corresponding gel was run which included cut pNZ44 for comparison purposes.
Lanes are as follows:
Lane 1:
pnz44 cut with xba
Lane 2:
pirfp 1 cut with nde and ecor1
Lane 3:
pirfp 8 cut with nde1 and ecor1
DEDUCTION FROM GEL: bands all too short, also pirfp should be cut twice, not once.
The digestion of several other pIRFP colonies was attempted with EcoRI only and also with EcoRI and NdeI. pNZ44 was also digested with XbaI.
Lane 1:
pIRFP 7 (EcoRI)
Lane 2:
pIRFP 9 (EcoRI)
Lane 3:
pIRFP 10 (EcoRI)
Lane 4:
pIRFP 7 (EcoRI and NdeI)
Lane 5:
pIRFP 9 (EcoRI and NdeI)
Lane 6:
pIRFP 10 (EcoRI and NdeI)
Lane 7:
Lane 7: pnz44 with xba1
DEDUCTION FROM GEL: pIRFP single cuts: Appears to have cut twice at ~3,500 and ~5,500 instead of once at ~4,500. (5500 could be uncut)                   pIRFP double cuts: Cut twice at ~2,500 and ~3500. (minor cuts at ~1000) Should be ~1800 and ~2700.                   Pnz44 single: one strong band at ~3500/4000 as expected.
Week beginning 11th July
A gel was ran containing the following:
Uncut plasmid
Plasmid with mesothelin
Plasmid with LJM11
Plasmid with pIRFP
Split RAW 264.7 cells.
6 transformations were performed-for each one plate with chloramphenicol, one without 12 plates.
LJM11
MSLN
pIRFP
K584001
Control
Uncut pNZ44
Result from transformations: No plates had colonies with the exception of uncut pNZ44 which only had 3. All plates without chloramphenicol were contaminated.
PCR was carried out of the Gibson assembly products using USP45 forward primer and pNZ44 reverse primer. The expected amplicon was ~300bp. The gel, shown below, was run as follows: MSLN, LJM11, pIRFP, control, uncut.
DEDUCTION FROM GEL: It is clear that all three ligations worked to some extent. This proves that the problem is with transformation; DH5α is not taking up the plasmids.
It was decided that we would troubleshoot the procedure in order to increase transformation efficiency. The following were altered in order to do this:
(i) Heat shock time
(ii) Antibiotic concentration
There were six 1 microgram pNZ44 XbaI digests performed from a series of plasmid preps:
Prep 131 A
Prep 131 B
Prep 120 A
Prep 120 B
Prep 52.6 A
Prep 52.6 B
The digests were then ran on a gel and deduction is as follows:
    All appear to have cut cleanly as no traces of uncut visible on gel.
6 transformations were performed with pNZ44 to troubleshoot transformation procedure.                   (i) 4 with differing heat shock times
    (a) 30 seconds
    (b) 45 seconds
    (c) 60 seconds
    (d) 90 seconds
(ii) 2 with differing antibiotic concentrations
    (a) Half previous concentration used
    (b) Previous concentration used
Two chloramphenicol plates were streaked with with MSP712
Two ampicillin plates were streaked with pCMV-Luc
pNZ44 and Puc19 were transformed into C2987I with the aid of NEB protocol
    Uncut pNZ44 diluted and undiluted plates both yielded a significant number of colonies
    Significant number of colonies on two Puc19 plates
Week beginning 18th July
dCas9 was PCR amplified.
A gel was ran with the following:
DEDUCTION FROM GEL: 3 bands at ~4,500 bp showed pcr worked for each tube.
The purified PCR products were then pooled.
A restriction digest of MSP712 was performed with BamHI and XbaI
A restriction digest of pCMVLuc was performed with NdeI and KpnI
A plasmid prep of MSP712 was performed and result was a concentration of 64ng/ul
A plasmid prep of pCMVLuc was performed and result was a concentration of 276ng/ul
Repeated dcas9 PCR with the following adjustments (one variable altered in each of 3 dcas9 PCR’s)
Longer extension time
Longer initial denaturation time
Less DNA
A gel was ran with the resulting PCR products and a control.
DEDUCTION FROM GEL: 3 bands at ~4,500 bp showed pcr worked for each tube.
The purified PCR products were then pooled.
Split cells, P11, 2 from 7 into a T175 flask (20ml volume)
5 ligations were carried out with the aid of the NEB E5520S HiFi DNA Assembly Kit:
1: Dcas9
2: LJM11
3: Mesothelin
4: pIRFP
5: Control
The ligations were subsequently transformed into C2987I
DEDUCTION FROM TRANSFORMATION: ample colonies on 2 Puc19                 plates as well as the two Gibson control plates (A+B) There was one colony on the dcas9 plate and a few colonies                 on the LJM11 plate also. No colonies on pIRFP or Mesothelin plates
Colony PCR was carried out using USP45 fp and pNZ44 RP. Two gels were run. The first gel was run as follows:
Lanes 1-6: LJM11 colonies A1-A6, Lane 7: LJM11 colony B1
DEDUCTION FROM GEL: LJM11 colonies A1, A2, A3, A6 and B1 were positive with bands at ~300bp
The second gel was run as follows and is shown below:
MSLN A1
MSLN B1
MSLN B2
pIRFP B1
pIRFP B2
dCas9 B1
Week beginning 1st August
Following successful colony PCR and restriction mapping, LJM11 colony A3 was sent to MWG for sequencing. Further restriction mapping was carried out on dCas9, MSLN and pIRFP. dCas9-pNZ44 was digested with SmaI and KpnI. The expected fragment sizes were approximately 5kb and 3.5kb. The gel was run as follows:                   Colony B1, C1 ,C2, C3, C4, C5, pNZ44 digested with SmaI and KpnI, pNZ44 uncut.
The results were not as expected and these colonies were abandoned.
MSLN was digested with PstI, which was expected to cut once and yield a single fragment of approximately 5kb. On the same gel pIRFP digested with KpnI and HindIII was run, with fragments expected at 3.5kb and 1 kb.
The gel was run as follows:
MSLN colony A1, B1, B2, piRFP colony B1, B2, C1, C2, pNZ44 digested with KpnI and HindIII
None of the digests were of the expected size; they were all similar to the unligated vector. We decided to abandon these colonies.
A different approach to the assembly of the dCas9-IKKBsgRNA-pNZ44 construct was suggested and attempted. This consisted of splitting the Gibson Assembly into several steps. The first step assembled Gibson fragment 1 and the digested vector (pNZ44), yielding a product A. The second step assembled Gibson fragment 2 and Gibson fragment 3, yielding a product B. The third step assembled A and B to form the final product.
Two approaches to this final assembly were carried out. In the first approach, X, 5uL of both A and B was taken. 10uL of NEB 2x HiFi DNA Assembly Master Mix was added. It was incubated at 50 degrees for 1 hour.                   In the second approach, 10uL of both A and B were taken. NEB 2x HiFi DNA Assembly Master Mix was not addd. It was incubated at 50 degrees for 1 hour.
The product was transformed into E.coli C2987I cells. Two different volumes of the ligation products X and Y were added to 50uL cells. 100uL was then plated on two LB agar plates containing chloramphenicol, giving eight plates in total.
Week beginning 8th August
Sequencing results for LJM11 A3 arrived from MWG. Results were positive, LJM11 now ready to be transformed into L.lactis subsp. cremoris. 17 colonies in total were observed from the 8 plates onto which the dCas9-IKKBsgRNA-pNZ44 construct transformation was plated. Colony PCR was carried out to screen for successful colonies. pNZ44 forward primer and dCas9 reverse primer were used. A product of approximately 4500 bp was expected. Results were not as expected:
Ladder used was NEB 2-log purple ladder:
Gel 1: Lane 1-10, colonies 1-10 respectively.
Gel 2: Lane 1-6, colonies 11-17 respectively. Lane 7, positive control, MSP712 amplified using dCas9 forward primer and dCas9 reverse primer. Lane 8, negative control.
There were no bands of the correct size. Nonetheless, overnight liquid cultures were made of colonies 1, 3, 5, 9, 11, 16. It was reasoned that colony PCR was not a suitable technique for such a large amplicon.
Plasmid minipreps were made of the overnight liquid cultures. Single digests of these minipreps were carried out using XbaI. The expected size was approximately 8000 bp.
The gel was run as follows:  pNZ44-dCas9-IKK2 sgRNA colony 1, pNZ44-dCas9-IKK2 sgRNA colony 3, pNZ44-dCas9-IKK2 sgRNA colony 5, pNZ44-dCas9-IKK2 sgRNA colony 9, pNZ44-dCas9-IKK2 sgRNA colony 16, pNZ44 digested with XbaI, undigested pNZ44.
Colonies 1, 3, 5 and 9 gave the expected size, it was concluded that these colonies were likely correct and sequencing could be carried out.
Our initial strategy for the replacement of the IKK2 sgRNA with CMV sgRNA was discussed. A G-block containing the CMV sgRNA would be cloned in using standard cloning. Both the pNZ44-dCas9-IKK2 sgRNA and the CMV G-block would be digested with XbaI and SmaI. The pNZ44-dCas9-IKK2 sgRNA would be run on a gel and the top band removed using gel extraction and subsequently purified. It would then be dephosphorylated using Antarctic phosphatase. These would then be ligated using T4 DNA ligase.
However, we reasoned that this method would be difficult. It might not be possible to see if our double digest was successful - when run on a gel, as it would look the same as a single cut plasmid. This was because the dropout would be very small (approximately 200bp) compared to the original plasmid (approximately 8000 bp).
A meeting with our supervisors and our secondary investigator took place. A new method of assembly of the pNZ44-dCas9-IKK2 sgRNA construct was designed. A new Gibson fragment 3 was ordered, which contained sgRNA against the CMV promoter instead of against the IKK2 promoter. This would be cloned using the same Gibson assembly method which worked for the  pNZ44-dCas9-IKK2 sgRNA construct. This G-block was ordered from IDT.
A new method of assembling our pNZ44-piRFP construct was also designed. This method would require the assembly of one insert into the vector, instead of the assembly of three inserts into the vector, which was previously attempted. Planning for the next parts of the project was carried out, including protein work and cell culture work.
Glycerol stocks of the successful pNZ44-USP45-His-LJM11 construct were made.
https://www.addgene.org/plasmid-protocols/create-glycerol-stock/
Week beginning 15th August
GM17 broth and agar plates were made (find manufacturer of M17 for reference. Can also use this for reference for 11g/L agar).                           Transformation of pNZ44-USP45-His-LJM11 and pNZ44-dCas9-IKK2 sgRNA into L.lactis subsp. Cremoris LLO via electroporation was carried out. In preparation for the LLO validation experiment, pFX-Luc was transformed into L.lactis subsp. cremoris LLO and L.lactis subsp. cremoris wild-type  The protocol followed can be found
here???????
The plates were incubated aerobically for 48 hours. Colonies were seen on the plates and colony PCR was used in order to screen for positive colonies. Following the results of this, the transformation via electroporation was deemed successful.
Further restriction mapping of the pNZ44-dCas9-IKK2 sgRNA constructs was carried out. Single digests were carried out. One set of digests used an enzyme which cut once in the vector (XhoI). The other set of digests used an enzyme which cut in the insert (EcorV).
The gel was run as follows:
Lanes 1-4: pNZ44-dCas9-IKK2 sgRNA colonies 1, 3, 5, 9 digested with XhoI. Lane 5: pNZ44 digested with XhoI. Lanes 6-9: dCas9 colonies 1, 3, 5, 9 digested with EcoRV. Lane 10; MSP712 digested with EcoRV
This restriction mapping was deemed successful - the enzymes cut as expected and the correct size (approximately 8000 bp) was observed.
This week, preparation for an experiment for the validation of our genomically modified L.lactis
Week beginning 22nd August
Our initial strategy for the replacement of the IKK2 sgRNA with CMV sgRNA was attempted. A G-block containing the CMV sgRNA would be cloned in using standard cloning. Both the pNZ44-dCas9-IKK2 sgRNA and the CMV G-block would be digested with XbaI and SmaI. The pNZ44-dCas9-IKK2 sgRNA would be run on a gel and the top band removed using gel extraction and subsequently purified. It would then be dephosphorylated using Antarctic phosphatase. These would then be ligated using T4 DNA ligase.
However, this method was not successful. It was not possible to see if our double digest was successful; when run on a gel, it looked the same as a single cut plasmid. This was because the dropout was very small (approximately 200bp) compared to the original plasmid (approximately 8000 bp).
Thus, it was decided that we would use the second strategy, which was designed and for which the necessary G-block was ordered on the week beginning 8th August.
