<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UNebraska-Lincoln_Description skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UNebraska-Lincoln/Description</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><H1 style="font-size:15px;padding:2px;font-family:'Josefin Slab';">UNL 2017</H1><H2 style="font-size:12.5px;">Helping reduce methane emissions from livestock</H2><DIV class="menu"><DIV class="menuDrop"><DIV id="teamButton" class="menuButton"><A href="https://2017.igem.org/Team:UNebraska-Lincoln/TeamLanding">Team</A></DIV><DIV class="menuContent"><A href="https://2017.igem.org/Team:UNebraska-Lincoln/Team">Team</A><A href="https://2017.igem.org/Team:UNebraska-Lincoln/Collaborations">Collaborations</A></DIV></DIV><DIV class="menuDrop"><DIV id="projectButton" class="menuButton"><A href="https://2017.igem.org/Team:UNebraska-Lincoln/ProjectLanding">Project</A></DIV><DIV class="menuContent"><A href="https://2017.igem.org/Team:UNebraska-Lincoln/Description">Description</A><A href="https://2017.igem.org/Team:UNebraska-Lincoln/Design">Design</A><A href="https://2017.igem.org/Team:UNebraska-Lincoln/Experiments">Experiments</A><A href="https://2017.igem.org/Team:UNebraska-Lincoln/Notebook">Notebook</A><A href="https://2017.igem.org/Team:UNebraska-Lincoln/InterLab">InterLab</A><A href="https://2017.igem.org/Team:UNebraska-Lincoln/Contribution">Contribution</A><A href="https://2017.igem.org/Team:UNebraska-Lincoln/Model">Model</A><A href="https://2017.igem.org/Team:UNebraska-Lincoln/Results">Results</A><A href="https://2017.igem.org/Team:UNebraska-Lincoln/Attributions">Attributions</A></DIV></DIV><DIV class="menuDrop"><DIV id="partsButton" class="menuButton"><A href="https://2017.igem.org/Team:UNebraska-Lincoln/PartsLanding">Parts</A></DIV><DIV class="menuContent"><A href="https://2017.igem.org/Team:UNebraska-Lincoln/Parts">Parts</A><A href="https://2017.igem.org/Team:UNebraska-Lincoln/Basic_Part">Basic Parts</A><A href="https://2017.igem.org/Team:UNebraska-Lincoln/Composite_Part">Composite Parts</A></DIV></DIV><DIV class="menuButton"><A href="https://2017.igem.org/Team:UNebraska-Lincoln/Safety">Safety</A></DIV><DIV class="menuDrop"><DIV id="humanButton" class="menuButton"><A href="https://2017.igem.org/Team:UNebraska-Lincoln/HumanLanding">Human Practices</A></DIV><DIV class="menuContent"><A href="https://2017.igem.org/Team:UNebraska-Lincoln/HP/Silver">Silver HP</A><A href="https://2017.igem.org/Team:UNebraska-Lincoln/HP/Gold_Integrated">Integrated and Gold</A></DIV></DIV></DIV><SECTION class="full"><DIV class="reallyfull"><DIV class="column full_size"><H1 class="topOfPage" style="font-family:'Josefin Slab';padding-bottom:30px;">Project Description</H1><H3 class="leap">Overview</H3><P>The cattle industry provides us with countless goods we eat and utilize today. From hamburgers and steak to dairy products to even some materials used to insulate your home, all these items are the by-products of cows from the cattle industry. However, in exchange for these commodities, the price we pay is the emission of methane gas into our atmosphere. In order to restrict this methanogenesis process occurring in the rumen, we are simultaneously pursuing the biosynthesis of bromoform in the rumen as well as the facilitation of nitrite reduction to ammonia.
</P><P class="citeCaption">[1]</P><H3 class="leap">Motivation</H3><P>Methane production in the rumen of cattle is a serious issue, not only for our environment and our carbon footprint but also for the efficiency of the cattle industry at large. Our project is a double edged sword by attempting to solve both problems' common root cause.
</P><P>In the 21st century, one of the the most pressing issues we all face is climate change. Earth’s average temperature is increasing at a dangerously fast rate and is affecting our planet in detrimental ways; glaciers are shrinking, heat waves are becoming more intense, and ecosystems are being destroyed. As the Greenhouse Effect shows, certain greenhouse gases are trapping the sun’s radiation and heat, causing climate change. The most widely recognized greenhouse gas is carbon dioxide as it is the greatest contributor to global warming. Yet there is another gas which is 25 times more potent at trapping the sun’s radiation than carbon dioxide, methane. According to the Environmental Protection Agency, a quarter of methane emissions is created by livestock alone, mainly cattle (EPA, 2017).
</P><P class="citeCaption">[2]</P><P>In our local state of Nebraska, the beef industry dominates above all others. There are over 6 million cows in Nebraska that account for 6.89% of all cattle in the United States. Each cow releases 70 kg to 120 kg of methane annually, and coupled with the sheer magnitude of the cattle industry not only in Nebraska but across the globe, this has become a problem of increasing importance. If we are able to find a way to lessen the amount of methane produced by cattle, we as Nebraskans can help reduce the environmental footprint of our state.  
</P><P>Moving away from only the environmental aspect, another issue arises through the methanogenesis process as well. Methane production through enteric fermentation is a waste of feed energy for the animal. An average of 6.5% of the gross energy provided by a cow’s food source is lost here (Johnson <I>et al.,</I> 1996). By curbing methanogenesis, we may be able to help the cows utilize this energy themselves and in turn, benefit the cattle industry and the consumers as well. If we are able to increase the efficiency by even a mere 6%, about $20 per head of cattle could be saved. This could help drive down costs for local farmers to buy feed, and the price of beef could drop as a result.
</P><H3 class="leap">The Problem</H3><P>Methane is generated in the rumen of cattle during the digestive process. The rumen is the first of four compartments of the stomach and functions to enzymatically decompose plant carbohydrates during enteric fermentation. When the cow consumes food sources such as grass, its unique ruminant microbiome allows it to digest the high cellulose content unlike the monogastric digestive system of other animals and humans.</P><P class="citeCaption">[3]</P><P>Three volatile fatty acids (VFA), acetate, butyrate, and propionate, are produced along with hydrogen and carbon dioxide.  While the VFA’s follow one pathway and become energy for the cow and the microbiome living inside of it, the hydrogen and carbon dioxide are instead used as substrates for the methanogenesis process.  In this process, methanogens, microorganisms of the domain Archaea and phylum <I>Euryarchaeota</I>, convert the two gases, hydrogen and carbon dioxide, into methane. At the end, most of the methane is expelled from the rumen when the cow belches the gas into the air.</P><H3 class="leap">What can we do?</H3><P>There is research going on all over the world that is working towards lowering methane emissions from cattle. Common approaches involve diet changes, such as feed additives, that indirectly change the microbiome. The efficiency of these approaches can be vastly improved by directly changing the microbiome of the cow using genetically engineered <I>E. coli</I>.</P><P>One way to do this is by using the powers of seaweed. Surprisingly, humans have been feeding seaweed to cows in coastal regions since the time of the ancient greeks (Ogilvie, 2010). Seaweed is now being studied as a food additive because it has been shown to reduce the amount of methane produced by cattle.</P><P class="citeCaption">[4]</P><P>The main compound within seaweed that is responsible for reducing methane emissions in cattle is bromoform. Bromoform works by inhibiting the efficiency of the methyltransferase enzyme by reacting with the reduced vitamin B12 cofactor required for the second to last step of methanogenesis as shown below. 
</P><P>A study done by Australian scientists tested 20 different species of seaweed on methanogens found in the stomachs of cows. They discovered seaweed reduced methane production by up to 50 percent, depending on the amount administered. But methane reduction at notable levels required high doses of seaweed, almost 20 percent by weight of the sample. This worked fine in the lab, but outside of the lab this large percentage of seaweed would be difficult to implement and would likely have a negative effect on cow’s digestion (Kinley <I>et al.,</I> 2014). On top of this the bromoform produced by the seaweed farms is known to act as a catalyst for the recombination of ozone. In fact its ability to deplete ozone can be 10-20 times higher than the more well known molecule Freon-22. This is due to the high resistance of bromine to the termination reaction of ozone (Itoh <I>et al.,</I> 1994). </P><P>This bears the question; do we want to fix one environmental issue by creating another one? The UNebraska-Lincoln iGEM team found this to be a counterproductive option so we decided to utilize the powers of synthetic biology and seaweed to inhibit methanogenesis directly by using <I>E. coli</I> to change the microbiome of the cattle. To do this our team took the gene from the algae Corallina <I>pilulifera</I> that codes for the enzyme bromoperoxidase and cloned it into <I>E. coli</I> as shown below. 
</P><P>Our bacteria would be fed to the cow along with the necessary substrates as a food additive. This would eliminate the creation of bromoform caused by the massive seaweed farms needed to feed the world's cattle, while simultaneously eliminating digestive problems in cattle caused by too much seaweed. To obtain these benefits and protect the ozone we needed to make sure that bromoperoxidase will not be produced until the <I>E. coli</I> entered the cow’s rumen. This way there is no possibility for the creation of ozone depleting molecules outside of the cow. Yet if the <I>E. coli</I> were to exit the cow’s rumen, the bacteria would need to die.
</P><P>To solve this complicated issue we designed the kill switch shown below.</P><P>The kill switch was designed so that the bromoperoxidase gene would not be produced until it came into contact with hydrogen peroxide. We chose this condition because <I>E. coli’s</I> contact with hydrogen peroxide will be a factor that we can control. Also hydrogen peroxide is a good choice because it is needed to catalyze the reaction that makes bromoform. We used the oxyR protein (BBa_K1104200) along with the adjacent ahpC  promoter  (BBa_K362001) to complete this task. One of the Anderson Promoters (0.36 strength) will be constitutively producing the oxyR gene. OxyR is an <I>E. coli</I> transcription factor that senses hydrogen peroxide to activate transcription. OxyR then activates the ahpC promoter and starts producing bromoperoxidase. This allows us to control the production of bromoform by controlling when the <I>E. coli</I> makes contact with hydrogen peroxide. This will ensure that bromoform can only be produced inside the cow's rumen.</P><P>Now that our kill switch will only allow production of bromoperoxidase inside the rumen the next problem is making sure that the <I>E. coli</I> will not survive if it ever makes it outside of the rumen. Usually, the risk of bacteria passing the abomasum (final stomach compartment) of a cow is very low. Nevertheless, a kill switch will disable our <I>E. coli</I> so that it cannot live out of its provided rumen system, decreasing the risk of an uncontrolled distribution. To ensure that it will not survive outside of the rumen we have added a temperature controlled hairpin structure that can only be activated by the nrrA transcription factor-activated promoter after the <I>E. coli</I> has reached the rumen. The hairpin structure works as shown in the figure below.</P><P>At high temperatures RNase E has access to the sites in the stem and cuts up the transcript (Hoynes <I>et al.,</I> 2015). This means at temperatures of 37°C or higher the kill switch gene nucA (BBa_k1159105) will not be produced. The nucA gene is a deadly endo-exonuclease from <I>Staphylococcus aureus</I> that degrades dsDNA, ssDNA, dsRNA and ssRNA. This is perfect for use in cattle because the average temperature of the rumen is 37.8°C - 40°C. This ensures that as long as the <I>E. coli</I> is in the rumen and in contact with hydrogen peroxide it will be able to produce bromoform, but if our <I>E. coli</I> gets outside of the rumen the RNase E sites are not accessible and translation can occur. This will lead the nucA gene to be expressed. This will effectively kill the <I>E. coli</I> once it has reached a temperature of 27°C outside of the rumen.</P><H2 style="font-size:2em;">Nitrite Reductase</H2><P>The dietary addition of nitrate for ruminants is a well recognized method used to decrease methanogenesis (Asanuma et al., 2014). Methanogenesis, the biosynthesis of methane by archaea, utilizes electrons from H2 in order to maintain a healthy rate of ruminal fermentation (Janssen &amp; Kirs, 2008). The addition of nitrate promotes the growth of bacteria that have the capability to reduce nitrate and nitrite. This promotion induces a gradual change in the microbiome and ultimately alters the flow of electrons away from methanogenesis (Asanuma et al., 2014).</P><P>Unfortunately, nitrate is quickly reduced to nitrite while the reduction from nitrite to ammonia is relatively slow. This combination leads to nitrite accumulation in the rumen and can cause serious health issues for the cattle or even death (Rasby et al., 2014).</P><P>Our proposed solution is to stimulate ruminal nitrite reduction to ammonia. If successful, this will lead to cattle having increased tolerance to nitrate-rich diets. We attempted to achieve this goal by experimenting with two functional analogs of nitrite reductase: <A href="http://parts.igem.org/Part:BBa_K1866006">nrfA from <I>E. coli TOP10</I></A> and <A href="http://www.ncbi.nlm.nih.gov/nuccore/AP012292">nrfA with nrfH from a native rumen species,  <I>Selenomonas ruminantium</I> subsp. <I>lactilytica TAM6421</I></A>. We constructed <A href="https://2017.igem.org/Team:UNebraska-Lincoln/Composite_Part">composite parts</A> for each and cloned the genes into <I>E. coli Genehog</I> and <I>E. coli NEB DH5-α</I> respectively. Experimentation with nrfA from <I>E. coli TOP10</I> was eventually discontinued due to a deleterious mutation occurring in the coding sequence. Focus was then shifted to experimentation with our nrfHA composite part derived from <I>S. ruminantium</I>. A kill switch for this gene has not yet been proposed, as it was more necessary for the gene encoding bromoperoxidase to have a designed kill switch.</P><P>The image above displays our plasmid that encodes nitrite reductase, nrfHA. The reaction, given as a stoichiometric equation, which the gene catalyzes is as follows:
 3H<SUB>2</SUB> +  NO<SUB>2</SUB><SUP>-</SUP> + 2H<SUP>+</SUP> -&gt; NH<SUB>4</SUB><SUP>+</SUP> + 2[H<SUB>2</SUB>O] ΔG<SUP>o’</SUP> = -149 kJ/mol
 Redox pairs NO<SUB>2</SUB><SUP>-</SUP>/NH<SUB>4</SUB><SUP>+</SUP> ΔE<SUP>o’</SUP> = +0.34V, H<SUB>2</SUB>/H<SUP>+</SUP>  ΔE<SUP>o’</SUP> = -0.42V (Simon, 2002)</P><P>Continue the story <A href="https://2017.igem.org/Team:UNebraska-Lincoln/Design#experimental">HERE</A>!</P><P>Works Cited</P><UL><LI>(2017, April 14) Overview of Greenhouse Gases. <I>EPA.</I>  Environmental Protection Agency.</LI><LI>Asanuma, N., Yokoyama, S., and Hino, T. (2014) Effects of nitrate addition to a diet on fermentation and microbial populations in the rumen of goats, with special reference to <I>Selenomonas ruminantium</I> having the ability to reduce nitrate and nitrite. Animal Science Journal 86, 378–384.</LI><LI>Hoynes-Oconnor, A., Hinman, K., Kirchner, L., and Moon, T. S. (2015) De novo design of heat-repressible RNA thermosensors in <I>E. coli</I>. <I>Nucleic Acids Research 43, 6166–6179.</I></LI><LI>Itoh, N., and Shinya, M. (1994) Seasonal evolution of bromomethanes from coralline algae (Corallinaceae) and its effect on atmospheric ozone. <I>Marine Chemistry 45, 95–103.</I></LI><LI>Johnson, D. E., and Ward, G. M. (1996) Estimates of animal methane emissions. <I>Environmental Monitoring and Assessment 42.</I></LI><LI>Janssen, P. H., and Kirs, M. (2008) Structure of the Archaeal Community of the Rumen. Applied and Environmental Microbiology 74, 3619–3625.</LI><LI>Kinley, R. D., and Fredeen, A. H. (2014) In vitro evaluation of feeding North Atlantic storm toss seaweeds on ruminal digestion. <I>Journal of Applied Phycology 27,</I> 2387–2393.</LI><LI>Ogilvie, A. E. J. (2010) The iceberg in the mist: northern research in pursuit of a &quot;Little Ice Age&quot;. Kluwer Academic Publishers Dordrecht.</LI><LI>Simon, J. C. B. (2002) Enzymology and bioenergetics of respiratory nitrite ammonification. FEMS Microbiology Reviews 26, 285–309.</LI></UL><P>Image Citations:</P><OL><LI>https://ichef-1.bbci.co.uk/news/624/media/images/53263000/gif/_53263376_cow_624.gif</LI><LI>https://blog.csiro.au/wp-content/uploads/2016/10/cow-on-beach.jpg</LI><LI>https://www.epa.gov/ghgemissions/overview-greenhouse-gases#methane</LI><LI>https://i.pinimg.com/474x/51/35/86/5135861a25e8979594e2eecada02c9a9--ag-science-animal-science.jpg</LI></OL></DIV></DIV></SECTION><DIV class="sideMenu"><DIV class="sideLink"><A class="sideA" href="#overview">Overview</A></DIV><DIV class="sideLink"><A class="sideA" href="#motivation">Motivation</A></DIV><DIV class="sideLink"><A class="sideA" href="#problem">The Problem</A></DIV><DIV class="sideLink"><A class="sideA" href="#whatToDo">What Can We Do?</A></DIV></DIV><H1 class="topOfPage" style="font-family:'Josefin Slab';text-align:initial;margin-left:20%;">Thanks to Our Sponsors</H1></DIV></DIV></DIV></DIV></DIV></BODY></HTML>