<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Marburg_Description skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Marburg/Description</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="header_wrapper"><DIV class="menu_wrapper" role="navigation" aria-label="Main Menu"><UL class="menu"><LI><A href="#">Team ▾</A></LI><LI><A href="/Team:Marburg/Team">Team</A></LI><LI><A href="/Team:Marburg/Collaborations">Collaborations</A></LI><LI><A href="/Team:Marburg/Attributions">Attributions</A></LI></UL><LI><A href="#">Project ▾</A></LI><LI><A href="/Team:Marburg/Description">Description</A></LI><LI><A href="/Team:Marburg/Design">Design</A></LI><LI><A href="/Team:Marburg/Experiments">Experiments</A></LI><LI><A href="/Team:Marburg/Notebook">Notebook</A></LI><LI><A href="/Team:Marburg/InterLab">InterLab</A></LI><LI><A href="/Team:Marburg/Model">Model</A></LI><LI><A href="/Team:Marburg/Results">Results</A></LI><LI><A href="/Team:Marburg/Demonstrate">Demonstrate</A></LI><LI><A href="/Team:Marburg/Measurement">Measurement</A></LI><LI><A href="/Team:Marburg/Labautomation">Labautomation</A></LI><LI><A href="#">Parts ▾</A></LI><LI><A href="/Team:Marburg/Parts">Parts Overview</A></LI><LI><A href="/Team:Marburg/Basic_Part">Basic Parts</A></LI><LI><A href="/Team:Marburg/Composite_Part">Composite Part</A></LI><LI><A href="/Team:Marburg/Part_Collection">Part Collection</A></LI><LI><A href="/Team:Marburg/Improve">Improve</A></LI><LI><A href="/Team:Marburg/Safety">Safety</A></LI><LI><A href="#">Human Practices ▾</A></LI><LI><A href="/Team:Marburg/Human_Practices">Human Practices</A></LI><LI><A href="/Team:Marburg/Public_Engagement">Public Engagement</A></LI><LI><A href="#">Awards ▾</A></LI><LI><A href="/Team:Marburg/Human_Practices">Integrated Human Practices</A></LI><LI><A href="/Team:Marburg/Public_Engagement">Education and Public Engagement</A></LI><LI><A href="/Team:Marburg/Model">Model</A></LI><LI><A href="/Team:Marburg/Measurement">Measurement</A></LI><LI><A href="/Team:Marburg/Basic_Part">Best New Basic Part</A></LI><LI><A href="/Team:Marburg/Composite_Part">Best New Composite Part</A></LI><LI><A href="/Team:Marburg/Part_Collection">Best Part Collection</A></LI><LI><A href="/Team:Marburg/Software">Software</A></LI><LI><A href="https://igem.org/2018_Judging_Form?id=2560">Judging Form</A></LI></DIV></DIV><MAIN><DIV class="titleWrapper"><DIV class="title">Description</DIV></DIV><ARTICLE><P><I>you better start swimmin' 
Or you'll sink like a stone 
For the times they are a-changin' </I><B>--Bob Dylan</B></P><P>The last years have seen a renaissance of research into <I>Vibrio natriegens</I>, the world’s fastest growing bacterium.
<I>V. natriegens</I> has a lot of potential to become the next universally applicable chassis organism, challenging <I>E. coli</I>’s dominance. To fully realize this potential, several steps have to be taken. We intended to contribute to this coup d’état by providing the research community with open source strains, tailored specifically to cloning, protein expression and interaction. Additionally, we conducted basic microbial research, establishing procedures and protocols for cloning and molecular biology research.
</P><P>
Metabolic engineering relies heavily on iterative testing. To shorten the duration of these iterations means to enable broader and more in-depth acquisition of data. </P><P>
Traditional cloning methods are limited by their rigidity and the time it takes to assemble constructs from multiple fragments. Recently, cloning methods based on type IIs restriction enzymes have addressed these issues. The synergy generated by combining these with <I>V. natriegens</I> would result in the world's fastest cloning platform, the Marburg Collection.
 
</P><DIV class="collapsible"><DIV class="btn_expand"><DIV class="btn_e_title"><I>Vibrio natriegens</I>, a review </DIV><DIV class="btn_e_desc">
Science is a collaborative effort, we all build on the work of others. Sometimes, finding all relevant sources can be difficult, so we wrote a short review of the research into <I> V. natriegens</I> from the last 60 years.

 </DIV></DIV><DIV class="content">


  When we first started research about
  <I>
     V. natriegens,
  </I>
    we only came across the latest publications, as well as the original descriptions. Only very tentatively, we became aware of the full body of research that had accumulated since its discovery. In order to make it easier for the scientific community to become familiar with
  <I>
     V. natriegens,
  </I>
    we thoroughly collected all available publications about it. You can find the complete list of publications on our wiki. The following text is meant to give a first impression about what is known and what has been done so far regarding
  <I>
     V. natriegens.
  </I><DIV class="collapsible"><DIV class="btn_expand">Introduction</DIV><DIV class="content"><P>

  First isolated from salt marsh mud on Sapelo island off the coast of Georgia in 1958

   <A href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC290205/pdf/jbacter00506-0093.pdf"><ABBR title="STUDIES ON BACTERIAL UTILIZATION OF URONIC ACIDS III. INDUCTION OF OXIDATIVE ENZYMES IN A MARINE ISOLATE">(Payne <I> et al. </I> 1958),</ABBR></A>

     
  
    the gram-negative bacterium was first named
  <I>
     Pseudomonas natriegens</I><A href="https://link.springer.com/content/pdf/10.1007/BF02538432.pdf%20"><ABBR title="SOME OBSERVATIONS ON THE PHYSIOLOGY OF PSEUDOMONAS NATRIEGENS NOV. SPEC. 1)">(Payne <I> et al. </I> 1960).</ABBR></A>
    These early studies also revealed a broad range of tolerated pH conditions with an optimum at a pH of 7.5
   <A href="https://link.springer.com/content/pdf/10.1007/BF02538432.pdf"><ABBR title="SOME OBSERVATIONS ON THE PHYSIOLOGY OF PSEUDOMONAS NATRIEGENS"> (Payne <I> et al. </I> 1960).</ABBR></A>
    One feature of this newly discovered organism became immediately apparent: The incredible doubling rate, first determined to be around 9.8 minutes
   <A href="http://www.ncbi.nlm.nih.gov/pubmed/13888946"><ABBR title="Pseudomonas Natriegens, a Marine Bacterium with a Generation Time of Less than 10 Minutes.">(Eagon 1962).</ABBR></A>
    We found that doubling rates of around 7 minutes were possible (own Data).
</P><P>
  Only much later, studies found several possible reasons for this. Firstly, as most Vibrio species,
  <I>
     V. natriegens
  </I>
   genome is split and distributed on two chromosomes. This means that replication can start at two origins of replication (Ori), resulting in more in parallel replication.
</P><FIGURE style="width: 31%; float: right;"><FIGCAPTION><B>Figure 1:</B> EM photograph of V. natriegens 
  <A href="http://www.microbiologyresearch.org/docserver/fulltext/ijsem/28/2/ijs-28-2-315.pdf?expires1538916011id=idaccname=guestchecksum=EB71B4ED8A4561D45623F3A9FF44A733"><ABBR title="Recognition of Beneckea Natriegens (Payne  et al.; Baumann  et al.)  as a Member of the Genus Vibrio, as Previously Proposed by Webb and Payne">(Austin <I> et al. </I> 1978).</ABBR></A></FIGCAPTION></FIGURE><P>
  Additionally, it was shown that the gamma-proteobacterium has an increased <DFN data-info="ribosomal RNA"> rRNA </DFN> activity. This is because of a greater number of  <DFN data-info="ribosomal RNA"> rRNA </DFN>  operons which are additionally controlled by stronger promoters
   <A href="http://www.ncbi.nlm.nih.gov/pubmed/11844764"><ABBR title="RRNA Promoter Activity in the Fast-Growing Bacterium Vibrio Natriegens.">(Aiyar <I> et al. </I> 2002)</ABBR></A>
   when compared to
  <I>
     E. coli
  </I>
   . More <DFN data-info="ribosomal RNA"> rRNA </DFN> means more ribosomes since <DFN data-info="ribosomal RNA"> rRNA </DFN> synthesis has been shown to be the rate-limiting step in their assembly
   <A href="http://www.ncbi.nlm.nih.gov/pubmed/6793240"><ABBR title="Growth-Rate-Dependent Regulation of Ribosome Synthesis in E. Coli: Expression of the LacZ and GalK Genes Fused to Ribosomal Promoters.">(Miura <I> et al. </I> 1981).</ABBR></A>
   Estimates for the number of ribosomes in
  <I>
     V. natriegens
  </I>
   in the exponential phase suggest around 115,000 per cell, while <I>E. coli</I> is estimated to have between 70,000 and 90,000
   <A href="https://www.frontiersin.org/article/10.3389/fmicb.2018.01146/full"><ABBR title="Cell-Free Protein Synthesis From Fast-Growing Vibrio Natriegens">(Failmezger <I> et al. </I> 2018).</ABBR></A></P><P>
  This gives
  <I>
     V. natriegens
  </I>
   higher protein expression and lets it create biomass more quickly. Could this also be exploited for the production of high-value proteins? Most likely.
</P><P>
  It was soon discovered that
  <I>
     V. natriegens
  </I>
   would make a perfect example by which to effectively and harmlessly demonstrate the basic techniques of microbiology to students, taking full advantage of the immense doubling rate
   <A href="http://www.tandfonline.com/doi/abs/10.1080/00219266.1973.9653881"><ABBR title="Vibrio Natriegens: A Rapidly Growing Micro-Organism Ideally Suited for Class Experiments.">(Mullenger 1973;</ABBR></A><A href="http://www.tandfonline.com/doi/abs/10.1080/00219266.2000.9655749"><ABBR title="Using Vibrio Natriegetis for Studying Bacterial Population Growth, Artificial Selection, and the Effects of UV Radiation and Photo-Reactivation."> Delpech 2001).</ABBR></A>
    Experiments regarding population structure, UV stress, and simply bare growth can be observed in a much shorter time frame.
</P><I>
       V. natriegens
    </I>
     has a single stage lifecycle and does not form spores. The rod shaped bacterium does however possess a single polar flagella
     <A href="http://www.microbiologyresearch.org/docserver/fulltext/ijsem/28/2/ijs-28-2-315.pdf?expires1538916011id=idaccname=guestchecksum=EB71B4ED8A4561D45623F3A9FF44A733"><ABBR title="Recognition of Beneckea Natriegens (Payne  et al.; Baumann  et al.)  as a Member of the Genus Vibrio, as Previously Proposed by Webb and Payne">(Austin <I> et al. </I> 1978).</ABBR></A></DIV></DIV><DIV class="collapsible"><DIV class="btn_expand">Ecology</DIV><DIV class="content"><I> V. natriegens </I> has some amazing abilities. By understanding them as adaptations, it becomes quite apparent that
    <I>
       V. natriegens
    </I>
     is itself first and foremost a product of its environment. Therefore, getting a better understanding of its ecology helps explain some of these capabilities.
  

	<FIGURE style="width: 31%; float: left;"><FIGCAPTION><B>Figure 1:</B> Coastal salt marsh in the southeast USA, this is the habitat <I> V. natriegens </I> was first isolated from(Pendragon 1966).
		</FIGCAPTION></FIGURE><P>
  The rod-shaped bacterium occupies a special ecological nice in estuarine and salt marsh regions, where strongly fluctuating salt concentrations and nutrient availability create a challenging environment. In order to adapt to these challenges,
  <I>
     V. natriegens
  </I>
   can utilize a wide range of substrates to outgrow competitors, as well as remain in a low metabolic state for extended periods of time when the nutrient pool is depleted
   <A href="http://www.microbiologyresearch.org/docserver/fulltext/micro/116/2/mic-116-2-295.pdf?expires1538914706&amp;id=id&amp;accname=guest&amp;checksum=F87669F5F1195DDDD75B002444425E85"><ABBR title="Adenine Nucleotide Pools During Starvation of Beneckea Natviegens.">(Nazly 1980).</ABBR></A>
    Furthermore, it has the ability to quickly draw available phosphorous from its environs and store them in polyphosphate bodies
   <A href="http://www.nrcresearchpress.com/doi/10.1139/m87-101"><ABBR title="Growth, Elemental Composition, and Formation of Polyphosphate Bodies in Vibrio Natriegens Cultures Shifted from Phosphate-Limited to Phosphate-Pulsed Media.">(Nissen <I> et al. </I> 1987)</ABBR></A>
   for later utilization. Yet another way in which <I>
     V. natriegens
  </I> is adapted to its variable environment is the storage of carbon as polyhydroxyalkanoates (PHAs), biodegradable polyesters that are accumulated in intracellular granules that can be broken down under starvation conditions
   <A href="http://www.ncbi.nlm.nih.gov/pubmed/17416432"><ABBR title="Production of Poly-β-Hydroxybutyrate (PHB) by Vibrio Spp. Isolated from Marine Environment.">(Chien <I> et al. </I> 2007).</ABBR></A></P><P>
  From the well oxygenated upper layer of mud down to the anoxic layers, <I>
     V. natriegens
  </I> is able to grow in many surroundings. These extremes are challenges to which <I> V. natriegens </I> found solutions. At the surface of the mud, as well as in open waters, the bacteria are exposed to strong <DFN data-info="ultra violet"> UV </DFN> radiation. As a response to this, the <DFN data-info="desoxyribonucleic acid"> DNA </DFN> damage repair systems are highly active
   <A href="http://www.ncbi.nlm.nih.gov/pubmed/21046343"><ABBR title="Identification of Vibrio Natriegens UvrA and UvrB Genes and Analysis of Gene Regulation Using Transcriptional Reporter Plasmids">(Simons <I> et al. </I> 2010),</ABBR></A>
    with significantly elevated expression levels of some parts of the system compared to
  <I>
     E. coli
  </I>
   , thus increasing <DFN data-info="desoxyribonucleic acid"> DNA </DFN> sequence integrity. By switching to fermentation, growth can be maintained under anaerobic conditions, allowing the colonization of the lower strata in the mud.
</P><I>
      V. natriegens
  </I>
     was shown to be a vital part of the marine ecosystem because it has the rare ability to fix atmospheric nitrogen under anaerobic conditions, thereby enriching its habitat and playing a crucial role as a provider of essential, bioavailable nitrogen
   <A href="http://www.ncbi.nlm.nih.gov/pubmed/11607653"><ABBR title="N2 Fixation in Marine Heterotrophic Bacteria: Dynamics of Environmental and Molecular Regulation.">(Coyer <I> et al. </I> 1996)</ABBR></A>
     under anaerobic conditions.
  



	<FIGURE style="width: 31%; float: right;"><FIGCAPTION><B>Figure 2:</B> Diverse phages isolated from <I>V. natriegens</I> cultures (Zacherey <I>et al.</I> 1974)
		</FIGCAPTION></FIGURE><P>
  Unsurprisingly,
  <I>
     V. natriegens
  </I>
   has its own predators in its native habitat: Bacteriophages
   <A href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC380186/pdf/applmicro00040-0188.pdf"><ABBR title="Isolation of Bacteriophages of the Marine Bacterium Beneckea Natriegens from Coastal Salt Marshes">(Zachary 1974).</ABBR></A>
    Studies in the 70ies found a link between different phages and the prevailing environmental conditions
   <A href="http://www.nrcresearchpress.com/doi/10.1139/m78-053"><ABBR title="An Ecological Study of Bacteriophages of Vibrio Natriegens">(Zachary 1978)</ABBR></A>
   . This suggests the phages employ a divide and conquer strategy, specializing, thereby limiting competition while improving reproductive success. One phage may have adapted to be effective at lower temperatures, while others prefer higher temperatures. The same applies to other conditions like for instance salt concentration
   <A href="http://www.ncbi.nlm.nih.gov/pubmed/938035"><ABBR title="Physiology and Ecology of Bacteriophages of the Marine Bacterium Beneckea Natriegens: Salinity.">(Zachary 1976).</ABBR></A>
    Intriguingly, that opens the possibility of creating genetic parts derived from these phages, applicable to future work with
  <I>
     V. natriegens,
  </I>
   analogous to the T7 system in
  <I>
     E. coli.
  </I>
   On the other hand, one study could show, that the phage carrying the toxins associated with V. cholerae could not replicate in V. natriegens
   <A href="https://www.biorxiv.org/content/early/2016/06/12/058487"><ABBR title="Vibrio Natriegens, a New Genomic Powerhouse.">(Lee <I> et al. </I> 2016),</ABBR></A>
    further showing its harmlessness towards humans.
</P><P>
  In contrast to that, some strains of
  <I>
     Vibrio natriegens
  </I>
   seem to be predators themselves. Confirmed as being pathogenic in several marine crustacea, most notably in the swimming crab
  <I>
     Portunus trituberculatus,
  </I>
   in which
  <I>
     V. natriegens
  </I>
   can lead to mortalities up to 85%
  
     <A href="http://doi.wiley.com/10.1111/jwas.12305"><ABBR title="Isolation and Molecular Identification of Vibrio Natriegens from Diseased Portunus Trituberculatus in China">(Bi <I> et al. </I> 2016).</ABBR></A>
    This is especially problematic since
  <I>
     P. trituberculatus
  </I>
   is farmed commercially in aquaculture in south-east China. The mechanism of that infection is poorly studied and it is not clear whether all strains of <I>
     V. natriegens
  </I> are pathogenic to crustaceans, but it is well known that many Vibrio species cause opportunistic infections in crustacea.
</P><P>
  On a more positive note, a recent study could show that the sponge
  <I>
     Aplysilla rosea
  </I>
   develops a more diverse microbiome when exposed to
  <I>V. natriegens </I><A href="http://www.nature.com/articles/s41598-018-30295-y"><ABBR title="A Controlled Aquarium System and Approach to Study the Role of Sponge-Bacteria Interactions Using Aplysilla Rosea and Vibrio Natriegens.">(Mehbub <I> et al. </I> 2018).</ABBR></A>
    
  
    It can be speculated that this helps to protect the sponge from pathogenic microbes that then find it harder to colonize its surface.
</P></DIV></DIV><DIV class="collapsible"><DIV class="btn_expand">Metabolism </DIV><DIV class="content"><P>
  A diverse, well-balanced metabolism allows a bacterium to grow in many conditions, while simultaneously making it interesting for biotechnological applications.
  <I>
     V. natriegens
  </I>
   can utilize a wide range of substrates
   <A href="http://www.ncbi.nlm.nih.gov/pubmed/28887417"><ABBR title="High Substrate Uptake Rates Empower Vibrio Natriegens as Production Host for Industrial Biotechnology">(Hoffart <I> et al. </I> 2017),</ABBR></A>  as well as grow in the absence of oxygen while being faster than other bacteria under the same conditions.
</P><P>
  For many applications in which a modern laboratory work horse is expected to be useful, a high-resolution map of its metabolism is a prerequisite. One recent study generated just that
   <A href="http://www.ncbi.nlm.nih.gov/pubmed/29042298"><ABBR title="Metabolism of the Fast-Growing Bacterium Vibrio Natriegens Elucidated by 13C Metabolic Flux Analysis">(Long <I> et al. </I> 2017).</ABBR></A>
    By tracking 13C labeled carbon through the intricate network that constitutes
  <I>
     V. natriegens
  </I>
   metabolism, they were able to gain insight. Fore one, they found the it to be very similar to the well-studied
  <I>
     E. coli
  </I>
   metabolism. It has at the major catabolic and anabolic pathways, the same canonical amino acids and building blocks. Some notable differences between the two were the RNA content being 29% in
  <I>
     V. natriegens
  </I>
   compared to only 21% in
  <I>
     E. coli,
  </I>
    the presence of an enzyme for the decarboxylation of oxalate
   <A href="http://www.ncbi.nlm.nih.gov/pubmed/29042298"><ABBR title="Metabolism of the Fast-Growing Bacterium Vibrio Natriegens Elucidated by 13C Metabolic Flux Analysis.">
     (Long <I> et al </I> 2017).
  </ABBR></A></P><P>
  The specific uptake of many carbon sources per gram dry weight per time was shown to be significantly higher than in other comparable species of bacteria
   <A href="http://www.ncbi.nlm.nih.gov/pubmed/28887417"><ABBR title="High Substrate Uptake Rates Empower Vibrio Natriegens as Production Host for Industrial Biotechnology.">(Hoffart <I> et al. </I> 2017).</ABBR></A>
    Many commercially interesting compounds are already produced by unmodified V. natriegens. In the case of alanine production, only four deletions resulted in a strain that outperformed highly specialized
  <I>
     E. coli
  </I>
   and
  <I>
     C. glutamicum
  </I>
   production-strains by a factor of 9 to 13 times
   <A href="http://www.ncbi.nlm.nih.gov/pubmed/28887417"><ABBR title="High Substrate Uptake Rates Empower Vibrio Natriegens as Production Host for Industrial Biotechnology.">(Hoffart <I> et al. </I> 2017).</ABBR></A>
    Bioreactor based production on a large scale is expensive, therefore a strain generating similar or even higher yield per gram carbon source is highly desirable, improving temporal yield per fermentation unit.
</P><P>
  When growing anaerobically, acids are produced that lead to an acidification of the medium but when grown under strong aeration, the pH can, depending on the medium, rise. This is most likely due to formation of ammonia
   <A href="http://www.ncbi.nlm.nih.gov/pubmed/13888946"><ABBR title="Pseudomonas Natriegens, a Marine Bacterium with a Generation Time of Less than 10 Minutes.">(Eagon 1962).</ABBR></A></P><P>
  The dependence on Na+ ions for its metabolic activity conveys a form of natural biocontainment, inhibiting growth in accidentally released
  <I>
     V. natriegens</I><A href="http://www.ncbi.nlm.nih.gov/pubmed/4327612"><ABBR title="Influence of Na+ on Synthesis of Macromolecules by a Marine Bacterium.">(Webb and Payne 1971).</ABBR></A>
  
    The high salt content of the LBv2 media also presents possible contaminants with a barrier, reducing the likelihood of airborne microbes to take hold.
</P><FIGURE style="width: 50%; float: right;"><FIGCAPTION><B>Figure 2:</B> EM picture showing accumulated selenium nanobodies in <I>V. natriegens</I><A href="https://static.igem.org/mediawiki/2018/8/8c/T--Marburg--Vibrio-Phage.jpeg"><ABBR title="Speeding up Bioproduction of Selenium Nanoparticles by Using Vibrio Natriegens as Microbial Factory.">(Fern`ndez-Llamosas <I> et al. </I> 2017).</ABBR></A></FIGCAPTION></FIGURE><P>

  Recently, a study revealed that
  <I>
     V. natriegens,
  </I>
    surprisingly, is the most effective producer of selenium nanoparticles yet described
   <A href="https://static.igem.org/mediawiki/2018/8/8c/T--Marburg--Vibrio-Phage.jpeg"><ABBR title="Speeding up Bioproduction of Selenium Nanoparticles by Using Vibrio Natriegens as Microbial Factory.">(Fern`ndez-Llamosas <I> et al. </I> 2017).</ABBR></A>
    These particles with a diameter from 100-400<DFN data-info="nanometer, 10^-9m">nm</DFN> have applications in diverse fields, such as medicine, microelectronics and more. Producing the nanoparticles in living systems has benefits over other methods, for instance, the low energy consumption as well as a coat of proteins preventing agglomeration of the particles. Also,
  <I>
      V. natriegens
  </I>
   exceptionally high resistance to selenite suggests possible applications in bioremediation of contaminated water and soils.
</P><P><STRONG>
     Working with
    <I>
       V. natriegens
    </I></STRONG></P><P><DFN data-info="American Type Culture Collection, identical to DSM 759">ATCC</DFN> 14048 is the most commonly used strain of
  <I>
     V. natriegens.
  </I>
    Most results from earlier studies were generated from this strain. We recommend using <DFN data-info="American Type Culture Collection, identical to DSM 759">ATCC</DFN> 14048 in order to ensure that the findings will be applicable to your work and to adopt it as the standard strain. It is also the fastest strain known
   <A href="http://www.ncbi.nlm.nih.gov/pubmed/27571549"><ABBR title="Vibrio Natriegens as a Fast-Growing Host for Molecular Biology">(Weinstock <I> et al. </I> 2016)</ABBR></A>
   so you won´t miss out on its incredible doubling time.
</P><P>
  Stemming from subtropical regions,
  <I>
     V. natriegens
  </I>
   has not the same tolerance to low temperatures that we are used to from
  <I>
     E. coli,
  </I>
    but it is still much more resilient to cold than
  <I>
     B. subtilis.
  </I>
    At the root of that lies the low catalytic activity of
  <I>
     V. natriegens
  </I>
   native catalase at low temperatures, which is then unable to detoxify reactive oxygen species. To address that problem, scientists from <DFN data-info="Synthetic Genomic Solutions">SGI</DFN> introduced homologous catalases, thereby creating a strain with comparable cold tolerance. With this adaptation, storage on plates can now be extended beyond the four weeks at room temperature by placing the plates in the fridge
   <A href="http://www.ncbi.nlm.nih.gov/pubmed/27571549"><ABBR title="Vibrio Natriegens as a Fast-Growing Host for Molecular Biology">(Weinstock <I> et al. </I> 2016)</ABBR></A>
   In storage at -80°C in a standard 20% glycerol stock the cells stay viable almost indefinitely. And even when working with the wildtype, your cells won´t die overnight when placed at 4°C. Only after a week they will notably start decreasing in viability.
</P><P>
  When working with <I>V. natriegens </I><DFN data-info="American Type Culture Collection, identical to DSM 759">ATCC</DFN> 14048, no special precautions have to be taken since it has been shown that no known pathogenicity associated genes were present
   <A href="http://www.ncbi.nlm.nih.gov/pubmed/27571549"><ABBR title="Vibrio Natriegens as a Fast-Growing Host for Molecular Biology">(Weinstock <I> et al. </I> 2016)</ABBR></A>
    Also, there is continued record of research since 1958 with not a single documented incident of a human infection with
  <I>
     V. natriegens.
  </I></P></DIV></DIV><DIV class="collapsible"><DIV class="btn_expand"> Molecular Biology</DIV><DIV class="content">
    Accessing, altering and understanding the genetics of an organism has become commonplace since sequencing and methods for genetic engineering have become affordable and readily available. Making these tools available for V. natriegens is, therefore, paramount making it an attractive chassis.
<P>
  Lately, a new branch of microbiologists developed an interest in
  <I>
     V. natriegens.
  </I>
    Their main focus lies in establishing state of the art methods and bringing
  <I>
     V. natriegens
  </I>
   to its full potential. We see our project as part of this movement and wish to make
  <I>
     V. natriegens
  </I>
   easily accessible to researchers. One groundbreaking paper
   <A href="http://www.ncbi.nlm.nih.gov/pubmed/27571549"><ABBR title="Vibrio Natriegens as a Fast-Growing Host for Molecular Biology">(Weinstock <I> et al. </I> 2016)</ABBR></A>
   established several methods to make
  <I>
     V. natriegens
  </I>
   genetically accessible, as well as characterizing some central genetic parts. This paved the way for more research into applying its potential.
</P><P>
  If you want to introduce genetic material into
  <I>
     V. natriegens,
  </I>
    you have several tried and tested options to choose from. You could use electroporation, a quick and simple method and the one with the highest transformation efficiency. Electroporation can also be used on wildtype cells. Or you could use chemically competent cells, the transformation works very similar to
  <I>
     E. coli
  </I>
   heat shock transformation. But in order to achieve good efficiencies using chemically competent cells, a nuclease deletion strain is needed. And if you want to insert especially large constructs, you can also use conjugation with very good success (
    <A href="https://www.biorxiv.org/content/early/2016/06/12/058487"><ABBR title="Vibrio Natriegens, a New Genomic Powerhouse.">(Lee <I> et al. </I> 2016;</ABBR></A><A href="http://www.ncbi.nlm.nih.gov/pubmed/27571549"><ABBR title="Vibrio Natriegens as a Fast-Growing Host for Molecular Biology"> Weinstock <I> et al. </I> 2016).</ABBR></A></P><P>
  A fully sequenced genome is also a prerequisite for many studies. Currently, several sequences are available
   <A href="https://www.biorxiv.org/content/early/2016/06/12/058487"><ABBR title="Vibrio Natriegens, a New Genomic Powerhouse"> (Lee <I> et al. </I> 2016;</ABBR></A><A href="http://www.ncbi.nlm.nih.gov/pubmed/23969053"><ABBR title="Draft Genome Sequence of the Fast-Growing Bacterium Vibrio natriegens">Maida <I> et al. </I> 2013),</ABBR></A>
    one of them generated by us. The genome has a size of approximately 5.17 <DFN data-info="mega bases, 10^9 bp">Mb</DFN>. Automated annotation was performed, revealing more of the genomic structure, as well as enabling the reconstruction of the codon usage profile
   <A href="https://www.biorxiv.org/content/early/2016/06/12/058487"><ABBR title="Vibrio Natriegens, a New Genomic Powerhouse">(Lee <I> et al. </I> 2016).</ABBR></A></P><P>
  The genome of
  <I>
     V. natriegens
  </I>
   is, like in many species of the genus, split into two chromosomes. Chromosome one contains 3.24 <DFN data-info="mega bases, 10^9 bp">Mb</DFN>, chromosome two 1.92 <DFN data-info="mega bases, 10^9 bp">Mb</DFN>.
</P><P><STRONG>
     Plasmids, Promoter, resistance
  </STRONG></P><P>
  Usually, a big hurdle when changing your chassis organism is a very basic question: Will my constructs still work? Do I need different <DFN data-info="origen of replication">ori</DFN>? Do I have to redesign everything from scratch? Fortunately, the most commonly used origins of replication, ColE1, p15A and pMB1 are maintained just fine in
  <I>
     V. natriegens
  </I><A href="http://www.ncbi.nlm.nih.gov/pubmed/27571549"><ABBR title="Vibrio Natriegens as a Fast-Growing Host for Molecular Biology">(Weinstock <I> et al. </I> 2016)</ABBR></A>
    We could observe some differences in copy number, which we quantified. You can find the corresponding data on our Part collection page. To retrieve these plasmids from the cells, we have tested a wide variety of commercially available kits and found that all of them worked, returning plasmid of the same purity and yield as for
  <I>
     E. coli.
  </I>
    Promoter commonly used in
  <I>
     E. coli
  </I>
   also lead to expression in
  <I>
     V. natriegens.
  </I>
    Expression levels are slightly different, but we also quantified these and you can find the results on our toolbox page. The same goes for other basic genomic parts like terminators and <DFN data-info="RBS">ribosomal binding sites</DFN>.
</P><P>
  Commonly, when very high gene expression is needed, for instance in protein overproduction, the phage derived T7 system is used. The T7 phage infects <I>E. coli</I> but it was also successfully inserted into
  <I>
     V. natriegens
  </I>
   and shown to drive protein expression
   <A href="http://www.ncbi.nlm.nih.gov/pubmed/27571549"><ABBR title="Vibrio Natriegens as a Fast-Growing Host for Molecular Biology">(Weinstock <I> et al. </I> 2016)</ABBR></A>
    One way of generating even stronger, phage derived promoters could be using
  <I>
     V. natriegens
  </I>
   derived phage expression systems.
</P><P>
  Every organism has its unique set of resistances to certain antimicrobial molecules. In order to successfully use a new organism, awareness of its individual tolerances is required to use antibiotics successfully for selection. We could show that chloramphenicol, carbenicillin, tetracycline and kanamycin worked, but at different concentrations from
  <I>
     E. coli.
  </I>
    We measured the
  <I>
     V. natriegens
  </I>
   specific concentrations and made them available on our strain engineering page.
</P><P>
  Plasmids in absence of a selective pressure were only retained for a short time (Own Data). Curing of plasmids that have no longer a purpose is therefore quick and fairly effortless.
</P><P><STRONG>
     MuGENT - Recombineering
  </STRONG></P><P>
  Investigations into alternative genetic tools were also undertaken, most notably MuGENT and Recombineering.
</P><P>
  MuGENT takes advantage of
  <I>
     V. natriegens
  </I>
   natural competence by inducing the TfoX system. The TfoX system, which is known from many Vibrio species, enables uptake and genomic integration of <DFN data-info="deoxyribonucleic acid">DNA</DFN> parts via homologous recombination
   <A href="http://doi.wiley.com/10.1111/j.1462-2920.2010.02250.x"><ABBR title="Natural Transformation of Vibrio Fischeri Requires TfoX and TfoY">(Pollack-Berti 2010).</ABBR></A>
    In most cases, growth on chitin was identified as the trigger which activated the system. Regrettably, the trigger is not known for
  <I>
     V. natriegens.
  </I>
    Using this system, a group of researchers characterized this system in
  <I>
     V. natriegens,
  </I>
    making it usable by placing it under the control of an inducible promoter. Additionally, they demonstrated its potential by increasing the natural poly-hydroxybutyrate (PHB) production 100-fold
   <A href="http://www.ncbi.nlm.nih.gov/pubmed/28571309"><ABBR title="Multiplex Genome Editing by Natural Transformation (MuGENT) for Synthetic Biology in Vibrio Natriegens."> (Dalia <I> et al. </I> 2017).</ABBR></A></P><P>
  Recombineering takes advantage of the fact that short <DFN data-info="deoxyribonucleic acid">DNA</DFN> oligonucleotides, that can easily be delivered into cells by a plethora of ways, can be used to generate point mutations, knockouts and small modifications in the genome, as well as on plasmids. The homologous flanks required are relatively short, allowing for constructs under 100 <DFN data-info="base pairs">bp</DFN>. Having this system available in
  <I>
     V. natriegens
  </I>
   means that generating new strains and gene variants is easy and quickly implemented
   <A href="https://www.biorxiv.org/content/early/2017/04/26/130088"><ABBR title="Vibrio Natriegens, a New Genomic Powerhouse">(Lee <I> et al. </I> 2017).</ABBR></A></P><P><STRONG>
     Cell Free Protein-expression Systems (CFPS)
  </STRONG></P><P>
  Very recently, several groups have published their work about cell free protein expression systems (CFPS) in
  <I>
     V. natriegens
     <A href="https://www.frontiersin.org/article/10.3389/fmicb.2018.01146/full"><ABBR title="Cell-Free Protein Synthesis From Fast-Growing Vibrio Natriegens.">(Failmezger <I> et al. </I> 2018;</ABBR></A></I><A href="http://www.ncbi.nlm.nih.gov/pubmed/30107122"><ABBR title="Establishing a High-Yielding Cell-Free Protein Synthesis Platform Derived from Vibrio Natriegens.">Des Soye <I> et al. </I> 2018;</ABBR></A><A href="http://pubs.acs.org/doi/10.1021/acssynbio.8b00222"><ABBR title="Establishing a Cell-Free Vibrio Natriegens Expression System."> Wiegand <I> et al. </I> 2018).</ABBR></A>
    Cell free systems offer the advantage of direct access to many cell components as well as fine control over the conditions in the reaction
   <A href="http://www.ncbi.nlm.nih.gov/pubmed/22008973"><ABBR title="Cell-Free Protein Synthesis: Applications Come of Age."> (Carlson <I> et al. </I> 2012).</ABBR></A>
    While these are true for all <DFN data-info="Cell Free Protein-expression System">CFPSs</DFN>, the advantage of creating such a system from
  <I>
     V. natriegens
  </I>
   is the much higher ribosome count
   <A href="https://www.frontiersin.org/article/10.3389/fmicb.2018.01146/full"><ABBR title="Cell-Free Protein Synthesis From Fast-Growing Vibrio Natriegens.">(Failmezger <I> et al. </I> 2018).</ABBR></A></P><P>
  Although the system works well when expressing plasmids from
  <I>
     E. coli,
  </I>
    it could be shown that expression was much higher when plasmids prepared from
  <I>
     V. natriegens
  </I>
   were used. Possible explanations are differences in methylation patterns, resulting in degradation of the plasmid by the system
   <A href="https://www.frontiersin.org/article/10.3389/fmicb.2018.01146/full"><ABBR title="Cell-Free Protein Synthesis From Fast-Growing Vibrio Natriegens.">(Failmezger <I> et al. </I> 2018).</ABBR></A>
    All this could be done by using protocols very similar to well established methods for other bacteria.
</P><P><STRONG>
     Cre-
    <I>
       lox
    </I></STRONG></P><P>
  The Cre-
  <I>
     lox
  </I>
   system was also successfully implemented and shown to work consistently with 600bp homology flanks
   <A href="http://www.ncbi.nlm.nih.gov/pubmed/27571549"><ABBR title="Vibrio Natriegens as a Fast-Growing Host for Molecular Biology">(Weinstock <I> et al. </I> 2016)</ABBR></A>
    Recycling of selection markers, like antibiotic resistances, as well as the removal of incorporated genes becomes easy and reliable.
</P><P>
  Also interesting for future protein related applications is, that a putative secretion signal was identified
   <A href="http://www.ncbi.nlm.nih.gov/pubmed/27571549"><ABBR title="Vibrio Natriegens as a Fast-Growing Host for Molecular Biology">(Weinstock <I> et al. </I> 2016)</ABBR></A></P></DIV></DIV><P>
  Fun facts:
</P><P>
  The epithet <I> natriegens </I> was chosen after the realization that salt is needed for it to grow, <I>natrie</I> stemming from the Latin word natrium meaning sodium and the Latin verb <I>egere</I>, meaning to need.
</P></DIV></DIV><DIV class="collapsible"><DIV class="btn_expand"><DIV class="btn_e_title">Strain Engineering</DIV><DIV class="btn_e_desc">By creating strains suited for diverse applications, we accelerate your research!</DIV></DIV><DIV class="content"><DIV class="collapsible"><DIV class="btn_expand">Methods
	   
	</DIV><DIV class="content"><H3>Natural Competence</H3>
What happened with the genetic Information when cells lysis? The DNA will be released to the environment. But can it be reused there? Or is the hole information lost?  Under certain circumstances other cells can take up the free DNA. This ability of cells is called competence. A special kind is the natural competence. These are bacteria that have genes in their genome encoded for proteins involved in the DNA recognition and uptake and in the integration in the genome.<P>In 1928 natural competence was first observed by Frederick Griffith even if he did not understand the genetics behind 

        <A href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2167760/" target="blank"><ABBR title="Griffith F, The significance of pneumococcal types. Journal of Hygiene. 27, Nr. 2. (1928) S. 113–59.">(Griffith <I>et al.</I>1928)</ABBR></A>

        . He already recognized that information can be transferred from one bacterium to another and thus he set the starting point of a big era to understand the DNA exchange. Thanks many more great scientists researched in this area, we now know alot about natural competence and the ability of some bacteria to take up free DNA. However, as in almost every research, there are still many open questions.</P><P>
        One of the most famous Vibrio species is Vibrio cholerae. This bacterial species triggers the disease called cholera, which is well known. However, this bacterium is like many other vibrio species also interesting for many other genetic features. When you think about how he has achieved his pathogenic properties, it come into your mind that this bacterium is maybe capable to do extensive gene transfer. What we know is, that the most Vibrio species are natural competence, that means that these organism can take up linear DNA from the environment und this genetic material can used for nutrients, to repair the genome or to promote horizontal gene transfer when integrated it into their genome by homologous recombination 

        <A href="https://www.ncbi.nlm.nih.gov/pubmed/26185067"><ABBR title="Antonova ES, Hammer BK. Genetics of Natural Competence in Vibrio cholerae and other Vibrios. Microbiol Spectr. (2015);3(3).">(Antonova <I>et al.</I>2015)</ABBR></A>

        . There was identified multiple environmental cues sufficient to induce natural transformation. For the most Vibrio species the phenomenon of natural transformation is induced by Chitin 
        
        <A href="https://www.ncbi.nlm.nih.gov/pubmed/16357262"><ABBR title="Meibom KL, Blokesch M, Dolganov NA. Chitin induces natural competence in Vibrio cholerae. Science (2005) 310:1824–1827.">(Meibom <I>et al.</I>2005)</ABBR></A>

        . In the natural aquatic environment of for example V. cholerae there are many chitinous shells of crustacean zooplankton which induces the expression of the competence regulator Tfox 

        <A href="https://www.ncbi.nlm.nih.gov/pubmed/16357262"><ABBR title="Meibom KL, Blokesch M, Dolganov NA. Chitin induces natural competence in Vibrio cholerae. Science (2005) 310:1824–1827.">(Meibom <I>et al.</I>2005)</ABBR></A>.
        For natural transformation several steps are necessary: At first the DNA has to be transported into the cell. After the DNA is bound to the cell, the Type IV Pillus (T4P) picks it up and transported it through secretion pores in the outer membrane into the periplasm. From there the DNA is translocated across the membrane by using a conserved membrane pore into the cytoplasm. If the DNA is arrived in the cell, it can be integrated by homologous recombination 

        <A href="https://www.ncbi.nlm.nih.gov/pubmed/24488316"><ABBR title="Mell JC, Redfield RJ. Natural competence and the evolution of DNA uptake specificity. J Bacteriol. (2014);196(8):1471-83.">(Mell<I>et al.</I>2014)</ABBR></A>.
        This process is dependent on some cellular structures, including type IV Pilli or related pseudopilli and many competence-related proteins and various regulator systems. The regulator Tfox activated the Type VI secretion systems (T6SSs) and that allows bacteria to inject directly into prey cell membranes or cytoplasm. It is a nanomachine that have many functions, one is a part of the natural competence regulon. Although much process has been made in this area, the molecular details of natural competence is not understood completely 
        <A href="https://www.ncbi.nlm.nih.gov/pubmed/26442598"><ABBR title="Wu R., Zhao M., Li J., Gao H., Kan B., Liang W. Direct regulation of the natural competence regulator gene tfoX by cyclic AMP (cAMP) and cAMP receptor protein (CRP) in Vibrios. Sci Rep. (2015) 7;5:14921.">(Wu <I>et al.</I>2015)</ABBR></A>.</P>
        Is Vibrio natriegens - a member of the vibrio species - natural competent, too?
        At the moment it is not so much known about the functionality of this process in Vibrio natriegens. However, it was discovered that it is natural competent and the Tfox regulates this property. But, it is not clear how Tfox is induced. It has been shown that in <I>V. natriegens</I> it is not induced by chitin as is the case with most Vibrio species. It was shown that you can express Tfox on an IPTG-inducible plasmid and there you get a high rate of natural transformation. In 2017 the group of Ankur B. Dalia showed that it was possible to integrate a linear PCR product with an antibiotic cassette that replaces the gene encoding the DNA endonuclease Dns in <I>V. natriegens</I> by using natural transformation. Around 1-10% of the population had integrated the transforming DNA 

        <A href="https://www.ncbi.nlm.nih.gov/pubmed/28571309"><ABBR title="Dalia TN, Hayes CA, Stolyar S, Marx CJ, McKinlay JB, Dalia AB. Multiplex Genome Editing by Natural Transformation (MuGENT) for Synthetic Biology in Vibrio natriegens. ACS Synth Biol. (2017) 15;6(9):1650-1655..">(Dalia <I>et al.</I>2017)</ABBR></A>

        this is the highest rate of transformation in competent species 

        <A href="https://www.ncbi.nlm.nih.gov/pubmed/28571309"><ABBR title="Lorenz, M. G., and Wackernagel, W. Bacterial gene transfer by natural genetic transformation in the environment. Microbiol Rev. (1994) 58, 563−602.">(Lorenz <I>et al.</I>1994)</ABBR></A>

        . Thus it was shown that a natural transformation can be well used to perform a scarfree genome editing 

        <A href="https://www.ncbi.nlm.nih.gov/pubmed/28571309"><ABBR title="Dalia TN, Hayes CA, Stolyar S, Marx CJ, McKinlay JB, Dalia AB. Multiplex Genome Editing by Natural Transformation (MuGENT) for Synthetic Biology in Vibrio natriegens. ACS Synth Biol. (2017) 15;6(9):1650-1655..">(Dalia <I>et al.</I>2017)</ABBR></A>.
        A big advantage of the method is that it is possible to perform many genome changes together, where you only need one antibiotics cassette. Therefore cells are incubated with a selected and an unselected product. The selected product had to contain an antibiotics cassette and the unselected product is used for the scarless genome editing of interest at the desired locus. This was also shown with a remarkably efficient up to 80% in <I>V. natriegens</I><A href="https://www.ncbi.nlm.nih.gov/pubmed/28571309"><ABBR title="Dalia TN, Hayes CA, Stolyar S, Marx CJ, McKinlay JB, Dalia AB. Multiplex Genome Editing by Natural Transformation (MuGENT) for Synthetic Biology in Vibrio natriegens. ACS Synth Biol. (2017) 15;6(9):1650-1655..">(Dalia<I>et al.</I>2017)</ABBR></A>

        . MuGENT can be used to rapidly generate <I>V. natriegens</I> strains with several large, scarless genome edits without off- targeting effects at high efficiency.That can be used to prepare highly complex mutant population. Many methods are described for modifying the genome of bacteria, but the great advantage of MuGENT is that it is not limited to small chances. You can efficiently swap, insert or remove whole units. In the future natural transformation could be utilized in combination with genome engineering methods like MAGE and CAGE, which rely on electroporation for E.coli, that could allow more rapid genome designs, such as complete codon reassignments. 






		<H3>The Flp/<I>frt</I> system</H3>
		There are several methods available to manipulate bacterial chromosomes. As the strain engineering subgroup, we had to decide which kind of strategy to use. Since iGEM means a lot of work in a very short time, we decided to look up for a method that further accelerates our project. We could use common plasmid based methods that rely on selection for plasmid excision followed by its loss. For Gram-negative bacteria these strategy often relies on counter selectable levan-sucrase enzyme, encoded by <I>sacB</I><A href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC259042/" target="blank"><ABBR title="Donnenberg, M. S. and J. B. Kaper (1991). Construction of an eae deletion mutant of enteropathogenic Escherichia coli by using a positive-selection suicide vector. Infect Immun59(12): 4310-4317."> (Donnenberg &amp; Kaper 1991)</ABBR></A>

		. The excision can restore the genotype before insertion or result in an exchange between the chromosome and the plasmid-encoded copy of the modified gene. The disadvantage of this method is that it is time-consuming. First, you need to clone a plasmid and then horizontal gene transfer from a donor to <I><I>V. natriegens</I></I> would be necessary. Additionally, the excision of the plasmid can lead to the pre-insertion genotype.
        One of the big advantages of Vibrio species is their ability to take up free DNA from their environment by natural competence 

        <A href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3490321/" target="”blank”"><ABBR title="Blokesch, M. (2012). TransFLP--a method to genetically modify Vibrio cholerae based on natural transformation and FLP-recombination. J Vis Exp(68)"> (Blokesch et al. 2005)</ABBR></A>

        ,

        <A href="https://www.ncbi.nlm.nih.gov/pubmed/21966921” target=”blank”&gt;&lt;abbr title=”Pollack-Berti, A., M. S. Wollenberg and E. G. Ruby (2010). Natural transformation of Vibrio fischeri requires tfoX and tfoY. Environ Microbiol 12(8): 2302-2311.">(Wollenberg et al. 2010)</A>

        . Using this skill, plasmid independent genomic modification with linear DNA fragments can be performed in <I>V. natriegens</I> as well. To select for clones, which integrated those linear fragments at least one fragment containing an antibiotics resistance cassette is needed 

        <A href="https://www.ncbi.nlm.nih.gov/pubmed/28571309" target="”blank”"><ABBR title="Dalia, T. N., C. A. Hayes, S. Stolyar, C. J. Marx, J. B. McKinlay and A. B. Dalia (2017). Multiplex Genome Editing by Natural Transformation (MuGENT) for Synthetic Biology in Vibrio natriegens. ACS Synth Biol 6(9): 1650-1655.">(Hayes et al. 2017)</ABBR></A>
        .
        It would be a big disadvantage to create strains not compatible to some of your plasmids because of the antibiotics resistances incompatibility. The solution is, to design those antibiotics cassettes in a way they can be removed after integration.
        It was shown, that genomic modifications using recombinase-based methods like the Cre/Lox system are possible in <I>V. natriegens</I><A href="https://www.ncbi.nlm.nih.gov/pubmed/27571549" target="”blank”"><ABBR title="Weinstock, M. T., E. D. Hesek, C. M. Wilson and D. G. Gibson (2016). Vibrio natriegens as a fast-growing host for molecular biology. Nat Methods 13(10): 849-851.">(Hesek et al. 2016)</ABBR></A>

        . A similar method is the FLP/FRT system. FLP is a protein of the integrase family recombinases derived from the yeast <I>Saccharomyces cerevisiae</I><A href="https://www.ncbi.nlm.nih.gov/pubmed/3011407" target="blank"><ABBR title="Argos, P., A. Landy, K. Abremski, J. B. Egan, E. Haggard-Ljungquist, R. H. Hoess, M. L. Kahn, B. Kalionis, S. V. Narayana, L. S. Pierson, 3rd and et al. (1986). The integrase family of site-specific recombinases: regional similarities and global diversity. EMBO J 5(2): 433-440.">(Landy et al. 1986)</ABBR></A>

        , 

        <A href="https://www.ncbi.nlm.nih.gov/pubmed/3011407" target="blank"><ABBR title="Sadowski, P. D. (1995). The Flp recombinase of the 2-microns plasmid of Saccharomyces cerevisiae. Prog Nucleic Acid Res MolBiol 51:53-91.">(Sadowski 1995)</ABBR></A>)

        . The recombinases of this family are sharing conserved amino acids that are responsible for the mechanisms of strand cleavage and ligation by the integrase members 

        <A href="https://www.ncbi.nlm.nih.gov/pubmed/2211714" target="blank"><ABBR title="Evans, B. R., J. W. Chen, R. L. Parsons, T. K. Bauer, D. B. Teplow and M. Jayaram (1990). Identification of the active site tyrosine of Flp recombinase. Possible relevance of its location to the mechanism of recombination. J Biol Chem 265(30): 18504-18510.">(Chen et al. 1990)</ABBR></A>

        , 

        <A href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC359755/" target="blank"><ABBR title="Pan, G. and P. D. Sadowski (1992). Ligation activity of FLP recombinase. The strand ligation activity of a site-specific recombinase using an activated DNA substrate. J Biol Chem 267(18): 12397-12399.">(Pan &amp; Sadowski 1992)</ABBR></A>

        .
        The FLP recognition target contains an eight bp asymmetric core that is surrounded by two 13-bp symmetric elements (5'GAAGTTCCTATTCtctagaaaGtATAGGAACTTC3'). The recombinase attaches to the top or bottom strand next to the core region, cleaves the DNA and promotes efficient recombination between two FRT recombination sites 

        <A href="https://www.ncbi.nlm.nih.gov/pubmed/6286142" target="blank"><ABBR title="Broach, J. R., V. R. Guarascio and M. Jayaram (1982). Recombination within the yeast plasmid 2mu circle is site-specific. Cell 29(1):227-234.">(Guarascio et al. 1982)</ABBR></A>.
 
       <FIGURE style="display:block; float:center; width: 90%;"><FIGCAPTION><B>Figure 1:</B>Four different Flp mediated gene modifications are shown. On the top left side the process of insertion is pictured. On the Bottom left side the process of genomic excision is shown. On the right side at the top of the picture, the inversion is shown and at the bottom the process of translocation. All processes are depending on the orientation of the FRT sites, that are attached to the target gene region. 
</FIGCAPTION></FIGURE><P>
       	In general, Flp mediated recombination between these sites can lead to insertion, excision, inversion, and translocation of DNA, depending on the direction of the FRT sites (Figure 1). Using FRT sites that are oriented in the same direction, leads to Flp mediated deletion of the region in-between. 

       	<A href="https://www.ncbi.nlm.nih.gov/pubmed/3011407" target="blank"><ABBR title="Sadowski, P. D. (1995). The Flp recombinase of the 2-microns plasmid of Saccharomyces cerevisiae. Prog Nucleic Acid Res MolBiol 51:53-91.">(Sadowski 1995)</ABBR></A>.
 
        This system works in Vibrio species like <I>V. cholera</I> or <I>V. fischeri</I> for rapid genetic modifications 

        <A href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3490321/" target="blank"><ABBR title="Blokesch, M. (2012). " transflp--a="" method="" to="" genetically="" modify="" vibrio="" cholerae="" based="" on="" natural="" transformation="" and="" flp-recombination.="">(Blokesch 2012)</ABBR></A>

        .<B>In order to establish a new and rapid method for genomic modifications in <I>V. natriegens</I>, we will use the FLP/FRT system to create our strains.</B></P></DIV></DIV><DIV class="collapsible"><DIV class="btn_expand">Integration
	</DIV><DIV class="content"><H2>Abstract</H2><P>
	The construction of stable bacterial strains for heterologous gene or pathway expression is usually done by chromosomal integration. Especially for the industrial application this has high priority, because continuous selection for plasmids is financial unsustainable. However, transcription levels vary between different chromosome regions. We therefore measured the growth of 15 different <EM>Vibrio natriegens</EM> mutants with integrated chloramphenicol resistance in different integration sites. Furthermore, we will characterize the gene expression in the integration sites by using the lac operon. By measuring its β-galactosidase activity the transcription level at this locus can be determined.</P><H2>Introduction</H2><H3>Description Chromosomal Integration</H3><P>
Homologous recombination is a class of genetic recombination, in which a genetic exchange takes place between a pair of similar/ identical DNA sequences 
<A href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3490365/"><ABBR title="(Kanigowska, Paulina; Shen, Yue; Zheng, Yijing; Rosser, Susan and Cai, Yizhi:  Smart DNA Fabrication Using Sound Waves. Applying Acoustic Dispensing Technologies to Synthetic Biology, Journal of laboratory automation, (2016); 21(1), pp. 49–56">(Alberts et al., 2012) </ABBR></A>


. Plasmids and linear DNA fragments, which carry sequences homologous to chromosome of the host strain, integrate into the host genome by homologous recombination following transformation 

<A href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3490365/"><ABBR title="(Kanigowska, Paulina; Shen, Yue; Zheng, Yijing; Rosser, Susan and Cai, Yizhi:  Smart DNA Fabrication Using Sound Waves. Applying Acoustic Dispensing Technologies to Synthetic Biology, Journal of laboratory automation, (2016); 21(1), pp. 49–56">(Cassey et al., 1991) </ABBR></A>

(Cassey et al., 1991).</P><P>
	In order to achieve optimal flux throughput and omit unwanted intermediates, the expression of crucial genes in a metabolic pathway should lead to a protein stoichiometry that is as stable as possible 

<A href="https://onlinelibrary.wiley.com/doi/full/10.1002/yea.1705#bib19"><ABBR title="(Dongmei Bai Flagfeldt, Verena Siewers, Le Huang and Jens Nielsen: Characterization of chromosomal integration sites for heterologous gene expression in Saccharomyces cerevisiae, Journal of laboratory automation, (2009); 21(1), pp. 49–56">(Dongmei Bai Flagfeldt et al., 2009) </ABBR></A>

. This aspect is of special importance for industrial applications.</P><P>This can be achieved through chromosomal integration, as it rules out plasmid number fluctuations and the plasmid loss that sometimes occurs in bacterial strains.Another advantage of chromosomal integration is that it makes the use of antibiotics while cultivating obsolete, since there is no need to apply selective pressure to get mutants, saving money and optimizing the growth.</P><H3>Position effects</H3><P>Bryant and colleagues showed that the expression level of a gfp gene cassette varies ~300 fold, depending of its position on the <EM>E. coli</EM> chromosome. We assume that the potential integration sites, in <EM>Vibrio natriegens</EM>, also vary in their gene transcription. To rationally choose an optimal integration site rationally, potential integration sites must have their transcription levels characterized. The amount of transcription that will take place in an integration site is influenced by multiple factors.</P><P> </P><UL><LI>Common regulatory factors,</LI></UL><P>The local general enhancers and repressors, with for example recruit the RNA-polymerase to a nearby promotor or facilitate the formation of the open complex (Wu et al. 2017).</P><P> </P><UL><LI>DNA-methylation</LI></UL><P>It has been shown that the expression regulation of some genes is based on the DNA’s secondary structure caused by methylation. In <EM>E. coli</EM> GATC sites are methylated and can influence the transcription of nearby genes (Plasterk et al., 1983). We expect this to also be the case in <EM>Vibrio natriegens, </EM>so this must be considered when selecting an integration site.</P><P> </P><UL><LI>Superhelical density</LI></UL><P>Adjacent genes can cause transcription coupled DNA supercoiling, the strength of the supercoiling depends on the transcription rate and the length of the transcribed region 

<A href="http://www.pnas.org/content/84/20/7024"><ABBR title="(L F Liu and J C Wang: Supercoiling of the DNA template during transcription, Biochemistry, (1987); 84(20), pp. 7024-7027)">(Liu and Wang, 1987) </ABBR></A>

. It is also known that microorganisms possess negatively supercoiled domains. It has been shown that the strength of some promotors, can be affected by supercoiling 

<A href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4770239/"><ABBR title="(Sobetzko: Transcription-coupled DNA supercoiling dictates the
chromosomal arrangement of bacterial genes, Nucleic Acids Res., (2016); 44(4), pp. 1514–1524)">(Sobetzko, 2016) </ABBR></A>

. We expect integration sites that are close to highly transcribed regions to be affected by DNA-supercoiling and to exhibit unexpected transcription levels.</P><P> </P><UL><LI>Histone like proteins</LI></UL><P>One master regulator in bacterial cells are histone like proteins such as H-NS, they bind to the DNA and affect the nearby gene expression 

<A href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4770239/"><ABBR title="(Patrick Sobetzkoa, Andrew Traversb,c, and Georgi Muskhelishvilia: Gene order and chromosome dynamics coordinate spatiotemporal gene expression during the bacterial growth cycle, Proc Natl Acad Sci USA., (2012); 109(2), E.42-E50)">(Sobetzko, Travers and Muskhelishvili, 2012) </ABBR></A>

(Sobetzko, Travers and Muskhelishvili, 2012). The effect of histone like proteins on our integration sites gene expression can vary extremely, depending on if and what kind of histone like proteins bind nearby.</P><P> </P><UL><LI>Distance to the origin of replication</LI></UL><P>Through multiforked replication some chromosomal region have multiple copies. It has been show that chromosomal integration sites close to the ori have a higher transcription rate in <EM>E. coli </EM>most likely due to the increased copy number of the gene 

<A href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4770239/"><ABBR title="(Sousa C, de Lorenzo V and Cebolla A.: Modulation of gene expression through chromosomal positioning in Escherichia coli, Microbiology, (2012); 143(6), pp. 2071-2078)">(Sousa, De Lorenzo and Cebolla, 1997) </ABBR></A>

. We assume that this will even more so be the case in Vibrio natriegens, as it replicates even faster and therefore has to do even more multiforked replication.</P><P> </P><H2>Experiment execution</H2><H3>Constructing of the parts for the chromosomal integration</H3><P>
	For the chromosomal integration a construct was chosen, which includes the transcription unit for the chloramphenicol resistance and 1kb homology flanks on each side. The homology flanks were amplified by Q5 PCR from the place on the chromosome of <EM>V. natriegens</EM> we want to integrate at. The chloramphenicol resistance was amplified by Q5 PCR from the pYTK vector. After that the resistance fragment and a homology fragment for each side were fused by a Q5 fusion PCR. At the end we created 15 constructs for the integration at different places on the chromosome.</P><H3>Integration of the parts into the chromosome</H3><P>
	We introduced our fragments by natural transformation into <EM>Vibrio natriegens</EM>, because its high efficiency (Dalia et al., 2017). The integration in the genome appeared by homologous recombination. We verified a correct integration by colony PCR.</P><H3>Growth measurement</H3><P>
	The integration of the chloramphenicol may lead to growth deficits as it influences the transcription of neighbours gens. Therefor we conducted growth measurement were we always measured three cultures of the WT and three of the mutants with the inserted sequence. For every culture pre-culture was incubated at 37°C in a shaker until OD<SUB>600</SUB>= 0,4-0,6.  From this culture we inoculated the main-culture to OD<SUB>600</SUB>~0,02 and began to measure the OD<SUB>600</SUB> every 15 minutes.</P></DIV></DIV><DIV class="collapsible"><DIV class="btn_expand">Strains
	    </DIV><DIV class="content"><P>There are different <I>E. coli</I> strains for a variety of applications. In every lab diverse strains are stored in freezers for methods like cloning, protein expression and protein interaction. And all of them are being used. For our project Vibrigens, we decided to establish the world’s fastest three lab strains suited for specific needs based on our <I>V. natriegens</I> wildtype strain. <B>VibriClone</B>, <B>VibriExpress</B> and <B>VibriInteract</B>. <B>VibriClone</B> is engineered to be a chassis for accelerated cloning. <B>VibriExpress</B> is engineered to be a chassis to speed up heterologous protein expression. <B>VibriInteract</B> is engineered to be a chassis for accelerated protein interaction studies based on the Bacterial Two-Hybrid system.</P><DIV class="collapsible"><DIV class="btn_expand">
	    VibriClone 
	</DIV><DIV class="content"><DIV><P>
    Why do we actually use <I>E. coli</I> for cloning? What are the reasons why <I>E. coli</I> is the one and only workhorse for molecular cloning? A big milestone in molecular biology was the first cloning in the 70's with the organism <I>E. coli</I>. Since this eventful step, the cloning has been constantly improved and accelerated. In addition, the strains of <I>E. coli</I> used for the cloning were adapted as best as possible to the requirements.
    But what about the selected Chassi? Of course, <I>E. coli</I> is a good choice because it is genetically simple, has a fast growth rate and is now well studied. However, there are so many different microorganisms, maybe there is a species that is even better suited for cloning. We introduce to you the organism, it is Vibrio natriegens.  Imagine you have all the advantages of <I>E. coli</I> for cloning, but much faster and easier. Through the transition, it would be possible for us scientists to complete cloning in one day instead of three and as you all can imagine, that would speed up the entire science. The only thing missing so far to use <I>V. natriegens</I> as cloning host is the optimization and the characterization. Exactly this point is our project. We want to make <I>V. natriegens</I> accessible and useful for you so that you have a great advantage for your important work by the changeover. </P></DIV><H3>Exonucleases and Endonucleases</H3><FIGURE style="display:block; width: 100%;"><FIGCAPTION><B>Figure 1:</B>Exonucleases and endonucleases can degrade DNA. Exonucleases hydrolyzes the phosphodiester bonds in DNA from a free end. Endonucleases hydrolyzes the phosphodiester bonds in DNA from a free end.</FIGCAPTION></FIGURE>
    The wildtype<I> E. coli</I> strain has same nucleases. They are divided into two types: exonucleases and endonucleases. Exonucleases play an important role in cells in the repair of DNA, genetic recombination and mutation avoidance. The enzyme hydrolyzes the phosphodiester bonds in DNA from a free end 

    <A href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4238392/" target="blank"><ABBR title="Susan T. Lovett, The DNA exonucleases of Escherichia coli, (2003) 4(2)">(Lovett, 2003)</ABBR></A>

    .Endonucleases are enzymes that cleaves hydrolytically the internal phosphodiester bond of DNA and/or RNA and this leads to their degradation. Bacterial endonucleases are also called restriction enzymes. In contrast to Exonucleases multiple nucleotides arise 

    <A href="https://www.ncbi.nlm.nih.gov/pubmed/12483517" target="blank"><ABBR title="Nishino, T., Morikawa, T., Structure and function of nucleases in DNA repair: shape, grip and blade of the DNA scissors, Oncogene (2002) 21, 9022 – 9032">(Nishino<I>et al.</I>2002)</ABBR></A> (Figure 1).<P><I>Vibrio cholerae</I>, like <I>V. natriegens</I>, is able to pick up DNA from the environment. However, most of the DNA is degraded by the nucleases secreted by the microbial community. Some transformable material is mostly available and can be taken up by naturally competent cells. <I>V. cholerae</I> has two extracellular nucleases: xds and dns. It has been shown for <I>V. cholerae</I> that the transformation efficiency greatly increases when the two extracellular nucleases are knocked out. Thus, the DNA to be transformed into the organism is no longer degraded 

    <A href="https://www.ncbi.nlm.nih.gov/pubmed/18757542" target="blank"><ABBR title="”Blokesch," m.="">(Blokesch<I>et al.</I>2008)</ABBR></A>

    .
    In order to build a cloning strain of <I>V. natriegens</I>, it is also very important to knock out the nucleases. Otherwise, the DNAses would digest the DNA and it would not be possible to transform DNA. 

    </P><H3>RecA</H3><DIV><FIGURE style="width: 25%; float:left; padding: 0;"><FIGCAPTION><B>Figure 2:</B>Functionality of recA. 1) RecA bind single-stranded DNA to form a nucleoprotein filament. 2) RecA recognizes homologous regions between the donor strand and a recipient molecules. 3) There is an exchange between the two strands held by recA. 4) The homologous single-stranded DNA fragment was integrated.
</FIGCAPTION></FIGURE>

    The enzyme RecA is present in many bacteria. It is an ATPase that mediates homologous recombination. Homologous recombination is important for genetic diversity and genomic integrity 

    <A href="https://www.ncbi.nlm.nih.gov/pubmed/18497818" target="blank"><ABBR title="Chen, Z., Yang, H., Pavletich N. P., Mechanism of homologous recombination from the RecA–ssDNA/dsDNA structures, Nature (2008) 22;453(7194):489-4.">(Chen<I>et al.</I>2008)</ABBR></A>

    . The role of the allosterically regulated protein in this process is to detect the homologous sequence in a variety of heterologous sequences 

    <A href="https://www.ncbi.nlm.nih.gov/pubmed/21070965" target="blank"><ABBR title="Savir, Y.,Tlusty, T., RecA-mediated homology search as a nearly optimal signal detection system, Molecular Cell (2010) 40, 388–396.">(Savir<I>et al.</I>2010)</ABBR></A>

    . RecA - bound to ATP - can make a complex with single-stranded DNA. This helical filament can bind to double-stranded DNA, search homologies, and then catalyze the exchange of the homologous strand 

    <A href="https://www.ncbi.nlm.nih.gov/pubmed/18497818" target="blank"><ABBR title="Chen, Z., Yang, H., Pavletich N. P., Mechanism of homologous recombination from the RecA–ssDNA/dsDNA structures, Nature (2008) 22;453(7194):489-4.">(Chen<I>et al.</I>2008)</ABBR></A>

    . In addition, <I>recA</I> has another function. It has a co-protease function in the bacterial SOS response system being responsible for cleavage of LexA and the λ-repressor 

    <A href="https://www.ncbi.nlm.nih.gov/pubmed/6101204" target="blank"><ABBR title="Horii T.; Ogawa T.; Nakatani T.; Hase T.; Matsubara H. &amp; Ogawa H., Regulation of SOS functions: Purification of &lt;i&gt;E. coli&lt;/i&gt; LexA protein and determination of its specific site cleaved by the RecA protein, cell (1981)  Vol. 27, 515-522.">(Horii<I>et al.</I>1981)</ABBR></A>

    . Because you do not want to have a chromosome integration or homologous recombination in a Cloning strain, this gene is mutated in the <I>E. coli</I> strains. For our <B>VibriClone</B> we also want to have a greatly reduced single-stranded DNA-dependent ATPase activity for RecA 

    <A href="https://www.ncbi.nlm.nih.gov/pubmed/2967815" target="blank"><ABBR title="Bryan, FR., Construction of a recombinase-deficient mutant recA protein that retains single-stranded DNA-dependent ATPase activity, J Biol Chem. (1988); 263(18):8716-23.">(Bryan, 1988)</ABBR></A> (Figure 2).</DIV><H3>KatG</H3>
    The <I>katG</I> gene encodes an enzyme with a catalase and peroxidase activity. Peroxidases use hydrogen peroxide as a substrate. A special form is the Catalase, which is also KatG. Catalases are enzymes that catalyze the reaction of hydrogen peroxide to water and oxygen. The fact that they make hydrogen peroxide harmless has a protective effect on the cell. Many aerobic organisms including <I>E. coli</I> have a catalase because they contain flavoenzymes that contribute to peroxide formation 

    <ABBR title="”Fuchs," g.="">(Fuchs 2017)</ABBR>

    . Due to the catalase activity it is possible to store <I>E. coli</I> at low temperatures, which is very useful in the laboratory. That's what we want for our <B>VibriClone</B>, too. Because we will have the working with <B>VibriClone</B> as comfortable and practical as possible.
    
   <DIV style="display:block"><H3>The <I>lac</I> operon</H3><FIGURE style="display:block; margin:0 auto 0 auto; width:80%;"><FIGCAPTION><B>Figure 3:</B>Gene Cluster of the genes belong to the lactose operon (green). Next to these genes are genes encoding for a putative cyanate transporter, a mhp operon transcriptional activator and a propionate hydroxylase.</FIGCAPTION></FIGURE>
     The <I>lac</I> operon plays an important role in the transport and the degradation of lactose in bacteria and it is a section on the chromosome of <I>E. coli</I>. It contains the three genes <I>lacZ</I>, <I>lacA</I> and <I>lacY</I>, which are regulated together. They encode enzymes for the utilization of lactose. The lacI gene is no longer counted to the lactose operon. The Lactose repressor (LacI) has an central importance for the regulation of the expression of the lactose operon. This is an enzyme that occupies the cell membrane of the bacterial cell and is responsible for the transport of lactose into the cell 

     <A href="https://www.ncbi.nlm.nih.gov/pubmed/18426771" target="blank"><ABBR title="Santillán M, Mackey MC., Quantitative approaches to the study of bistability in the lac operon of Escherichia coli., J R Soc Interface. (2008) 6;5 Suppl 1:S29-39.">(Santillán<I>et al.</I>, 2008)</ABBR></A>

     . The ß-galactosidase (LacZ) convert lactose to glucose and galactose 

     <A href="https://www.ncbi.nlm.nih.gov/pubmed/21070965" target="blank"><ABBR title="Serebriiskii I.G., Golemis E.A., Uses of lacZ to study gene function: evaluation of β-galactosidase assays employed in the yeast two-hybrid system. Anal. Biochem. 2000;285:1–15.">(Serebriiskii<I>et al.</I>, 2000)</ABBR></A>

     . The glucose can then be metabolized normally by glycolysis. The exact function of the Thiogalactoside transacetylase (LacA) is unknown, but it may act as an detoxifying enzyme. Transcription of the lactose Operon is regulated by both the repressor and the cAMP-CAP complex. The lac operon is negatively regulated by a repressor and positively by an activator. In addition, the activity of the lac permease is controlled by the inducer exclusion mechanism.
     The lactose operon works the following way. When lactose is present in the surrounding medium and there is no energy source more favorable for the cell like glucose for cellular metabolism, the three genes of the lactose operon are expressed. In the following lactose is transported through the permease into the cell and there is decomposed by the enzyme ß-galactosidase into glucose, galactose and allolactose. The allolactose performs a backbond to bind to the lactose repressor and allows the transcription to proceed. Negative and positive regulation controls the degradation of the most efficient source of energy (Figure 3).

     <FIGURE style=""><FIGCAPTION><B>Figure 4:</B>Functionality of the blue-white screening. 1) If the gene is not inserted into the plasmid, the β-galactokinase can be assembled. She can then split X-Gal and the colonies turn blue. 2) The multiple cloning site of the used plasmid is integrated into the lacZ-alpha part. If the gene is inserted into the the plasmid, the β-galactokinase can´t be assembled. Because of this the colonies stay white.
</FIGCAPTION></FIGURE>
     If a strain has integrated the lactose operon, it is possible to perform a blue-white selection (Figure 4). The basic idea of the genetic engineering principle is to check whether the genetic modification was carried out correctly. The ß-galactosidase, the gene product of the LacZ genes, consists of two components: LacZ-alpha and LacZ-omega only if both components are functionally present, galactosidase can exert its catalytic effect 

     <A href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC432510/" target="blank"><ABBR title="Langley, KE., Villarejo, M.R., Fowler, A.V., Zamenhof, P.J., and Zabin, I, Molecular basis of beta-galactosidase alpha-complementation. Proc Natl Acad Sci U S A. (1975);72(4):1254-7.">(Langley<I>et al.</I>, 1975)</ABBR></A>

     . In the DH5𝛼 strain of <I>E. coli</I> the 𝛼 subunit is deleted. Now, a plasmid containing the alpha subunit is used. 

     <A href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC247395/" target="blank"><ABBR title="Zamenhof, P.J., Villarejo, M. Construction and Properties of &lt;i&gt;E. coli&lt;/i&gt; Strains Exhibiting α-Complementation of β-Galactosidase Fragments In Vivo. J Bacteriol. (1972); 110(1): 171–178.">(Zamenhof<I>et al.</I>, 1972)</ABBR></A>

     . By a successful transformation of the plasmid into the mutated strain, a funcional ß-galactosidase can be expressed. If an insertion into this vector occurs, the complementation can not work. After transformation, the cells are plated on selection medium. It contains IPTG, which makes the lac repressor inactive and allows expression of the <I>lacZ</I> gene. In addition, X-Gal is included, which is split by the ß-galactosidase in a blue product 

     <A href="https://www.ncbi.nlm.nih.gov/pubmed/22639266" target="blank"><ABBR title="Burn, S.F., Detection of β-galactosidase activity: X-gal staining. Methods Mol Biol. 2012;886:241-50.">(Burn 2012)</ABBR></A>

     . This means that the white colonies have integrated the plasmid with insert and the blue colonies without insert.
     This simple method makes it quick and easy to see which are the right colonies that you can pick. Our big goal is to make this method usable for the <B>VibriClone</B> as well. This would give you a huge time advantage through the rapid growth of <I>V. natriegens</I> and the rapid recognition of the result. 
     After many rounds of designing, and engineering <I>Vibrio natriegens</I>  to a suitable clone strain, we now have <B>VibriClone</B> in our hands, with all properties for a perfect cloning host, such as nuclease deficience, <I>recA</I> mutation, cold resistance and the ability for blue white screening.</DIV>
     We want to make our strain accessible to the whole iGEM community by providing the opportunity to request our <B>VibriClone</B> strain, to enable all future iGEM Teams to accelerate their projects with the next generation cloning chassis.

	</DIV></DIV><DIV class="collapsible"><DIV class="btn_expand">
	    VibriExpress 
	  </DIV><DIV class="content"><DIV><P>

		   Nowadays the production of high protein amounts is important for biotechnological and industrial uses. One of the widely used expression hosts is <I>E. coli</I> in combination with the T7 expression system. For the last century, the performance of <I>E. coli</I> was hugely satisfying, except of issues like the aggregation of inclusion bodies and the insolubility of some proteins

		   <A href="https://www.ncbi.nlm.nih.gov/pubmed/14514038" target="blank"><ABBR title="Villaverde, A. and M. M. Carrio (2003). Protein aggregation in recombinant bacteria: biological role of inclusion bodies. Biotechnol Lett 25(17): 1385-1395.">(Villaverde &amp; Carrio 2003)</ABBR></A>.

		    However, in the last years, scientific progress is getting faster and faster and it is time to question if there is an alternative to <I>E. coli</I>, a host that is keeping up with the time?
           Imagine you could start a day early in the morning performing a transformation with your construct. After several hours, you could inoculate colonies in liquid culture and induce the expression of your desired protein in an overnight culture. The second day you would be able to purify your protein.
           Using <I>V. natriegens</I> as a new host with its impressive doubling time and the ability to grow to a much higher cell density will arise new possibilities for molecular and synthetic biology. In order to do so, we will design the <B>VibriExpress</B> strain.
           </P><BLOCKQUOTE>“Creating a strain that is optimized for efficient production of high quality protein material is of high value for the drug discovery process.”</BLOCKQUOTE> this was the view of Dr. Serghei Glinca, the Chief Executive Officer of CrystalsFirst, a company that is specialized on protein-crystal based applications in drug discovery.

           <A href="https://2018.igem.org/Team:Marburg/Human_Practices">Click here to learn more about our Integrated Human Practices project.</A>

           The functionality of the T7 system in <I>V. natriegens</I> was already demonstrated and additionally it is pointed, that <I>V. natriegens</I> yields higher amounts of soluble protein than <I>E. coli</I><A href="https://www.ncbi.nlm.nih.gov/pubmed/27571549" target="blank"><ABBR title="Weinstock, M. T., E. D. Hesek, C. M. Wilson and D. G. Gibson (2016). Vibrio natriegens as fast-growing host for molecular biology. Nat Methods 13(10): 849-851."> (Weinstock, Hesek <I>et al.</I>,2016)</ABBR></A>.<B>But how is the commonly used T7 system working?</B></DIV><H3>The T7 system</H3>
           The system is based on the bacteriophage T7 RNA Polymerase, that shows a very high activity and synthesizes RNA at a rate several times that of <I>E. coli</I>. Besides, it is highly selective for its own promoter. Combined with the ability to inhibit selectively the host RNA polymerase with rifampicin

           <A href="https://www.ncbi.nlm.nih.gov/pubmed/4863911" target="blank"><ABBR title="Hartmann, G., Honikel, K. O., Knüsel, F. &amp; Nüesch, J. The specific inhibition of the DNA-directed RNA synthesis by rifamycin. Biochimica et biophysica acta 145, 843–844 (1967)."> (Hartmann <I>et al.</I>, 1967)</ABBR></A>

           , it allows the exclusive expression of genes under the control of a T7 RNA polymerase promoter

           <A href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC397196/" target="blank"><ABBR title="Tabor, S. &amp; Richardson, C. C. A bacteriophage T7 RNA polymerase/promoter system for controlled exclusive expression of specific genes. Proceedings of the National Academy of Sciences of the United States of America 82, 1074–1078 (1985).">(Tabor S., 1985)</ABBR></A>

           . Due to these advantages, the gene of the T7 RNA Polymerase was integrated into the <I>E. coli</I> BL21 genome and is controlled by a <I>lacUV5</I> promoter 

           <A href="https://www.ncbi.nlm.nih.gov/pubmed/3537305" target="blank"><ABBR title="Studier, F. W. &amp; Moffatt, B. A. Use of bacteriophage T7 RNA polymerase to direct selective high-level expression of cloned genes. Journal of molecular biology 189, 113–130 (1986).">(Moffatt et al, 1986)</ABBR></A>
           , 
           <A href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4395058/" target="blank"><ABBR title="Jeong, H., Kim, H. J. &amp; Lee, S. J. Complete Genome Sequence of Escherichia coli Strain BL21. Genome announcements 3 (2015).">(Jun Lee <I>et al.</I>, 2015)</ABBR></A>

           . The gene of the desired protein is introduced into the cell using a vector with a T7 promoter (e.g. pET vector, commercialized by Novagen). Incubating the cells with IPTG leads to the expression of the T7 RNA Polymerase. Finally, the Polymerase recognizes the T7 promoter and the expression of the desired protein is enforced (Figure 1)

           <A href=" https://www.ncbi.nlm.nih.gov/pubmed/10600732" target="blank"><ABBR title="Cheetham, G. M. &amp; Steitz, T. A. Structure of a transcribing T7 RNA polymerase initiation complex. Science (New York, N.Y.) 286, 2305–2309 (1999)." s="">(Steitz <I>et al.</I>, 1999)</ABBR></A>

           . To maximize the produced protein amount, proteases like Lon or OmpT can be knocked-out 

           <A href=" https://www.ncbi.nlm.nih.gov/pubmed/3277950" target="blank"><ABBR title="Grodberg, J. and J. J. Dunn (1988). ompT encodes the Escherichia coli outer membrane protease that cleaves T7 RNA polymerase during purification. J Bacteriol 170(3): 1245-1253."> (Dunn <I>et al.</I>, 1988)</ABBR></A>

           .<P><FIGURE style=""><FIGCAPTION><B>Figure 1:</B>The T7 expression system. Here the function of the T7 expression system is described. IPTG is inducing the expression of T7 RNA Polymerase that is encoded on the bacterial genome. The T7 RNA Polymerase is recognizing the T7 promoter on the plasmid and the expression of the desired protein begins.
</FIGCAPTION></FIGURE></P><H3>The Lon protease</H3>
            The Lon protease derives its name from the phenotype of <I>E. coli</I> lon gene mutants that form long undivided filaments upon UV irradiation 

            <A href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1210565/" target="blank"><ABBR title="Howard-Flanders, P., E. Simson and L. Theriot (1964). A Locus That Controls Filament Formation and Sensitivity to Radiation in Escherichia Coli K-12. Genetics 49: 237-246."> (Simson <I>et al.</I> 1964)</ABBR></A>

            . It is a component of the protein quality-control systems and degrades misfolded and unfolded proteins in an ATP dependent manner 

            <A href="https://www.ncbi.nlm.nih.gov/pubmed/6458037” target=”blank"><ABBR title="Chung, C. H. and A. L. Goldberg (1981). The product of the lon (capR) gene in Escherichia coli is the ATP-dependent protease, protease La. Proc Natl Acad Sci U S A 78(8): 4931-4935."> (Chung &amp; Goldberg 1981)</ABBR></A>

             and is also involved in the degradation of certain regulatory proteins like SulA, which is a component of the SOS system and an inhibitor of cell division 

             <A href=" https://www.ncbi.nlm.nih.gov/pmc/articles/PMC393376/" target="blank"><ABBR title="Mizusawa, S. and S. Gottesman (1983). Protein degradation in Escherichia coli: the lon gene controls the stability of sulA protein. Proc Natl Acad Sci U S A 80(2): 358-362."> (Mizusawa &amp; Gottesman 1983)</ABBR></A>

             , Huisman, 

             <A href="https://www.ncbi.nlm.nih.gov/pubmed/6087326" target="blank"><ABBR title="Huisman, O., R. D'Ari and S. Gottesman (1984). Cell-division control in Escherichia coli: specific induction of the SOS function SfiA protein is sufficient to block septation. Proc Natl Acad Sci U S A 81(14): 4490-4494."> (D'Ari et al. 1984)</ABBR></A>).
 
            <H3>The OmpT protease</H3>
            The other deleted protease OmpT is an aspartyl protease localizing at the outer membrane of <I>E. coli</I>. It is suggested that OmpT is involved in the pathogenicity of <I>E. coli</I>. More than 30 years ago is was shown that OmpT catalyzes the activation of plasminogen to plasmin 

            <A href="https://www.ncbi.nlm.nih.gov/pubmed/6453346" target="blank"><ABBR title="Leytus, S. P., L. K. Bowles, J. Konisky and W. F. Mangel (1981). Activation of plasminogen to plasmin by a protease associated with the outer membrane of Escherichia coli. Proc Natl Acad Sci U S A 78(3): 1485-1489."> (Bowles <I>et al.</I>, 1981)</ABBR></A>

            . This function seems to be physiologically relevant for the virulence of clinical <I>E. coli</I> isolates 

            <A href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC125623/" target="”blank”"><ABBR title="Lundrigan, M. D. and R. M. Webb (1992). Prevalence of ompT among Escherichia coli isolates of human origin. FEMS Microbiol Lett 76(1-2): 51-56."> (Lundrigan &amp; Webb 1992)</ABBR></A>

             and other pathogens like <I>Yersinia pestis</I><A href=" https://www.ncbi.nlm.nih.gov/pubmed/1439793" target="”blank”"><ABBR title="Sodeinde, O. A., Y. V. Subrahmanyam, K. Stark, T. Quan, Y. Bao and J. D. Goguen (1992). A surface protease and the invasive character of plague. Science 258(5084): 1004-1007."> (Subrahmanyam <I>et al.</I>, 1992)</ABBR></A>

             . Additionally the level of OmpT expression was shown to increase in response to the induction of recombinant protein overexpression 

             <A href="https://www.ncbi.nlm.nih.gov/pubmed/10937437" target="”blank”"><ABBR title="Gill, R. T., M. P. DeLisa, M. Shiloach, T. R. Holoman and W. E. Bentley (2000). OmpT expression and activity increase in response to recombinant chloramphenicol acetyltransferase overexpression and heat shock in &lt;i&gt;E. coli&lt;/i&gt;. J Mol Microbiol Biotechnol 2(3): 283-289."> (DeLisa <I>et al.</I>, 2000)</ABBR></A>

             . For T7 expression strains, the deletion of this protease is urgently required because it was also shown that OmpT is degrading T7 RNA Polymerase 

             <A href="https://www.ncbi.nlm.nih.gov/pubmed/3277950" target="”blank”"><ABBR title="Grodberg, J. and J. J. Dunn (1988). ompT encodes the Escherichia coli outer membrane protease that cleaves T7 RNA polymerase during purification. J Bacteriol 170(3): 1245-1253."> (Grodberg &amp; Dunn 1988)</ABBR></A>

             .
            According to NEB (New England Biolabs Ipswich, Massachusetts) strains that are deficient in both proteases Lon and OmpT are more amenable for the production of proteins<P>Our vision is to create a strain similar to <I>E. coli</I> BL21 and maximize the protein yield to an optimum by not only integrating the T7 system into <I>V. natriegens</I> but also deleting proteases to decrease protein degradation during the purification process.
            This way we will design an optimized strain for high-speed and high-level protein expression. Making VibriExpress accessible for everyone, we would like to accelerate your research and support you all, making the world a better place.</P></DIV></DIV><DIV class="collapsible"><DIV class="btn_expand">
	    VibriInteract</DIV><DIV class="content"><DIV><P>	
		Proteins are part of the building blocks of life and fulfill a variety of essential functions. But what could one protein alone achieve? Mostly proteins interact with partners or in complexes to fulfill their function. A general goal of science is to understand all biochemical processes that enable life as we know it. Thus, the understanding of protein interaction partners becomes more and more important.
        It is crucial to have methods investigating interactions between different proteins since most proteins do not act alone but in complexes with other proteins. This circumstance is of central interest since it can reveal a lot about the function of a protein and the processes that it is involved in.  Using those methods, one can study interactions of specific proteins but it is also possible to screen for potential interaction partners using a library of genes. The latter one was already used by the iGEM team Bielefeld-CeBiTec in 2016. In their project they established their own method to do interaction assays to screen for proteins with an innate affinity for a specific target

        (<A href="https://2016.igem.org/Team:Bielefeld-CeBiTec/Project/Selection/Bacterial_Two-Hybrid_System" target="blank">Click here to learn more about the Bacterial-Two-Hybrid assay developed by iGEM Bielefeld-CeBiTec</A>).

         This shows that interaction studies can not only be used in basic research but could also play an important role as an application in synthetic biology.</P><P>
         	Protein interaction studies can be done by several methods:
         	<UL><LI>Biochemical Methods 		
                </LI><LI>
                		e.g. Co-immunoprecipitation
                	</LI></UL><LI>
             	 Biophysical and Theoretical Methods
               </LI><LI>
               		e.g. Fluorescence Resonance Energy Transfer (FRET)
               	</LI><LI>
             	 Genetic Methods
               </LI><LI>
               		e.g. Two-Hybrid Systems
               	</LI></P><P>
         	A lot of these methods still need a lot of time to be carried out. This can be done faster! Therefore we created our third strain: <B>VibriInteract</B>. A strain which can be used for protein interaction studies.
         </P><P>
         	For Two-Hybrid Systems, the chassis organisms <I>Escherichia coli</I> and <I>Saccharomyces cerevisiae</I> are used in a Bacterial Two-Hybrid (B2H) and Yeast Two-Hybrid (Y2H) system, respectively. The Y2H assay was a novel genetic system at that time used to detect protein-protein interactions heterologously in yeast 

            (<A href=" https://www.ncbi.nlm.nih.gov/pubmed/2547163 " target="blank"><ABBR title=" Field S. &amp; O. Song, A novel genetic system to detect protein-protein interactions, Nature (1989) 340: 245-246.">Fields &amp; Song, 1989</ABBR></A>).

         	9 years later, a new system was proposed. Instead of S. cerivisiae, this system uses <I>E. coli</I> as chassis organisms 

            (<A href=" https://www.ncbi.nlm.nih.gov/pmc/articles/PMC20451/ " target="blank"><ABBR title=" Karimova, G., J. Pidoux, A. Ullmann &amp; D. Ladant,  A bacterial two-hybrid system based on a reconstituted signal transduction pathway. Proceedings of the National Academy of Sciences of the United States of America (1998) 95: 5752-5756.">Karimova <I>et al.</I>, 1998</ABBR></A>).

            That this genetic test is carried out in <I>E. coli</I> greatly facilitates the screening as well as the characterization of the interacting proteins.  Moreover, it accelerated this assay since <I>E. coli</I> has a much faster growth than <I>S. cerivisiae</I>. Although B2H assays are the easiest and fastest methods to test for interactions nowadays we think that they can be done faster. Using <I>E. coli</I> (which is already faster than using an Y2H system), the assay still needs 24 to 72 hours of incubation. Using <I>V. natriegens</I>, we can accelerate this assay to 8 to 24 hours of incubation.  Paired with one day cloning we can speed up this assay from 7-9 days if your work in <I>E. coli</I> to 3 days if you work in <I>V. natriegens</I>. By this, we create our very own Two-Hybrid Assay: <B>Vibrio Two-Hybrid (V2H)</B>.
         </P><TABLE><TBODY><TR><TH>An ordinary workflow of a Bacterial Two Hybrid Assay</TH><TH>An ordinary workflow of a Vibrio Two Hybrid Assay</TH></TR><TR><TD>3 days cloning (at best)
                2 - 4 days first transformation
                2 - 4 days spot assay
            </TD><TD>1 day of cloning (at best)
                1 day first transformation
                1 day spot assay
            </TD></TR><TR><TD>7-9 days in total</TD><TD>3 days in total</TD></TR></TBODY></TABLE></DIV><P>The Bacterial Two-Hybrid system is based on reconstitution of a signal transduction pathway relying on the adenylate cyclase (CyaA). CyaA is a key regulator in a lot of cells like e.g. <I>E. coli</I>. It catalyzes the conversion of adenosine triphosphate (ATP) to cyclic adenosine monophosphate (cAMP) (Figure 2 A)

    (<A href="https://link.springer.com/article/10.1007/BF00229806 " target="”blank”"><ABBR title="Krupinski, J., The adenylyl cyclase family. Molecular and cellular biochemistry (1991) 104: 73-79.">Krupinski, 1991</ABBR></A>).

    cAMP binds to the catalytic activator protein (CAP). This interaction effects a conformational change which leads to a higher binding affinity to DNA. The complex binds to the CAP site upstream of the promoter and can interact with the RNA polymerase 

     (<A href="https://www.ncbi.nlm.nih.gov/pubmed/8394684 " target="”blank”"><ABBR title="Kolb, A., S. Busby, H. Buc, S. Garges &amp; S. Adhya, Transcriptional regulation by cAMP and its receptor protein. Annual review of biochemistry (1993) 62: 749-795.">Kolb <I>et al.</I>, 1993</ABBR></A>).

    This interaction increases the affinity of the polymerase to the promoter. The positive regulation boosts the expression of the gene by a factor of approximately 50 compared to gene expression without the presence of cAMP (Figure 1). To conclude, the cAMP/CAP complex is a positive regulator of e.g. the <I>mal</I> operon and the <I>lac</I> operon.

    <FIGURE style=""><FIGCAPTION><B>Figure 1:</B> Schematic figure of how cAMP is involved in regulation of gene expression. Without cAMP, CAP can’t bind to its specific site and downstream genes are only expressed in low levels. With cAMP, CAP interacts with CAP and ,by this, can bind to the CAP site and interact with the RNA polymerase. Downstream genes are expressed in high levels.</FIGCAPTION></FIGURE></P><P>For B2H an <I>E. coli</I> strain lacking the cyA gene, which is encoding the catalytic domain of the adenylate cyclase and does not produce cAMP, is used. Consequently, cells can no longer utilize maltose or lactose. This phenotype can be rescued using the complementary T25 and T18 fragments of the catalytic domain of the adenylate cyclase toxin from <I>Bordetella pertussis</I>. Those fragments are cloned to genes of interest in plasmids called pKT25 and pUT18, respectively. Those fragments when splitted can no longer synthesize cAMP (Figure 2 B). If two hybrid proteins interact in this assay the two fragments of the catalytic domain come into close proximity and produce cAMP(Figure 2 C). The cAMP/CAP complex is formed again and regulates the mal or lac operon positively. Cells containing interacting constructs can utilize maltose and lactose again. Based on this, interaction of proteins can be screened using methods based on the lac operon or the maltose operon. For this, several different media are used:

    <OL><LI>
    		LBv2 agar plates supplemented with IPTG and Xgal used for Blue-White Screening if the lac operon is functional. You can read more about Blue-White Screening in the VibriClone part of our description.
    	</LI><LI>
    		MacConkey agar plates supplemented with maltose to test if the mal operon is functional. When cells utilize maltose, acids are formed and cells turn red because of the pH indicator neutral red.
    	</LI><LI>
        	
M63 minimal medium agar plates supplemented with maltose to test if the mal operon is functional by screening for growth of colonies.
        </LI></OL></P><P><FIGURE><FIGCAPTION><B>Figure 2:</B> Principle of a Two-Hybrid system based on reconsitution of the adenylate cyclase of <I>B. pertussis</I>. (A) The full-length catalytic domain can synthesize cAMP. (B) T25 and T18 fragments, when coexpressed as splitted peptides, are not able to interact and cAMP is not synthesized. (C) Both fragments, when fused to interacting proteins, get into close proximity and cAMP is synthesized.</FIGCAPTION></FIGURE></P></DIV></DIV></DIV></DIV><DIV class="collapsible"><DIV class="btn_expand">CRISPR/Cas9</DIV><DIV class="content">
		In the last few years, the clustered regularly interspaced short palindromic repeats (CRISPR)/<DFN data-info=" clustered regularly interspaced short palindromic repeats ">CRISPR
	</DFN>
	 associated protein (Cas) method became a popular application in genetic engineering. This method is based on an adaptive antiviral defense mechanism of prokaryotes
	
	<A href=" http://science.sciencemag.org/content/315/5819/1709 "><ABBR title=" CRISPR Provides Acquired Resistance Against Viruses in Prokaryotes ">
		 (Barrangou
		<EM>
			 et al.
		</EM>
		 2007).</ABBR></A> As a part of the bacterial immune system,
	<DFN data-info=" clustered regularly interspaced short palindromic repeats/clustered regularly interspaced short palindromic repeats associated protein ">
		 CRISPR/Cas
	</DFN>
	 provides protection against foreign DNA through the action of the RNA-guided DNA endonuclease
	<DFN data-info=" clustered regularly interspaced short palindromic repeats associated protein ">
		 Cas,
	</DFN>
	 which cleaves the entering viral DNA at a specific point
	<A href=" https://www.cell.com/action/showPdf?pii=S0092-8674%2813%2900211-0 "><ABBR title=" Repurposing CRISPR as an RNA-Guided Platform for Sequence-Specific Control of Gene Expression ">
			 (Qi
			<EM>
				 et al.
			</EM>
			 2013).</ABBR></A><P>
	 The first
	<DFN data-info=" clustered regularly interspaced short palindromic repeats ">
		 CRISPR-</DFN>loci were discovered in 1987 in the
	<EM><I>E. coli</I></EM>
	 strain K12 <A href=" https://www.ncbi.nlm.nih.gov/pmc/articles/PMC213968/pdf/jbacter00202-0107.pdf "><ABBR title=" Nucleotide Sequence of the iap Gene, Responsible for Alkaline Phosphatase Isozyme Conversion in Escherichia coli, and Identification of the Gene Product ">
		
			 (Ishino
			<EM>
				 et al.
			</EM>
			 1987)</ABBR></A>
	  and the first description of a
	<DFN data-info=" clustered regularly interspaced short palindromic repeats associated protein "><EM>
			 cas
		</EM></DFN>
	 gene and their homologies to DNA-related enzymes appeared in 2002
	<A href=" https://onlinelibrary.wiley.com/doi/epdf/10.1046/j.1365-2958.2002.02839.x "><ABBR title=" Identification of genes that are associated with DNA repeats in prokaryotes ">

			 (Jansen
			<EM>
				 et al.
			</EM>
			 2002).</ABBR></A>
	  Since then, more than 9000 research articles have been published about the different
	<DFN data-info=" clustered regularly interspaced short palindromic repeats/clustered regularly interspaced short palindromic repeats associated protein ">
		 CRISPR/Cas
	</DFN>
	 subtypes and their application in various
	<DFN data-info=" clustered regularly interspaced short palindromic repeats ">
		 CRISPR
	</DFN>
	 based technologies 
	<A href=" https https://www.nature.com/articles/s41467-018-04252-2.pdf "><ABBR title=" The CRISPR tool kit for genome editing and beyond ">(Adli 2018).</ABBR></A>
	  The potential applications are almost limitless, ranging from new methods in research to industrial applications as well as human gene therapy. With
	<DFN data-info=" clustered regularly interspaced short palindromic repeats/clustered regularly interspaced short palindromic repeats associated protein ">
		 CRISPR/Cas,
	</DFN>
	 it is possible to specifically switch off genes in a large number of prokaryotic and eukaryotic organisms, to integrate foreign DNA
	 into the genome of an organism or to use the
	
		 DNA-binding property for in vitro experiments
	
		<A href=" https://www.cell.com/action/showPdf?pii=S0092-8674%2814%2900604-7 "><ABBR title=" Development and Applications of CRISPR-Cas9 for Genome Engineering ">
			 (Hsu
			<EM>
				 et al.
			</EM>
			 2014).</ABBR></A></P><P><FIGURE style="width: 40%; float: right"><FIGCAPTION><B> Figure 1: CRISPR/Cas9.</B> The picture shows the Cas9 assembled with the sgRNA and DNA. The scaffold of the sgRNA is  illustrated in purple and the spacer in blue. The PAM is shown in red.</FIGCAPTION></FIGURE>
Due to its simplicity and modularity, the Type II
	<DFN data-info=" clustered regularly interspaced short palindromic repeats/clustered regularly interspaced short palindromic repeats associated protein ">
		 CRISPR/Cas9
	</DFN>
	 system from
	<EM>
		 Streptococcus pyogenes
	</EM>
	 is a commonly used tool for genetic engineering applications in a variety of organism
	<A href=" https://www.cell.com/action/showPdf?pii=S0092-8674%2814%2900604-7 "><ABBR title=" Development and Applications of CRISPR-Cas9 for Genome Engineering ">
			 (Hsu
			<EM>
				 et al.
			</EM>
			 2014,
		</ABBR></A><A href=" https://www.sciencedirect.com/science/article/pii/S1369527414000885?via%3Dihub "><ABBR title=" Harnessing CRISPR-Cas9 immunity for genetic engineering. ">
			 Charpentier &amp; Marraffini 2014,
		</ABBR></A><A href=" http://science.sciencemag.org/content/346/6213/1258096.full "><ABBR title=" Genome editing. The new frontier of genome engineering with CRISPR-Cas9 ">
			 Doudna &amp; Charpentier 2014).</ABBR></A>
	  For its RNA-guided silencing of foreign DNA it requires just one
	<DFN data-info=" clustered regularly interspaced short palindromic repeats associated protein ">
		 Cas9
	</DFN>
	 encoding gene and the action of the CRISPR RNA (crRNA) as well as the trans-activating
	<DFN data-info=" clustered regularly interspaced short palindromic repeats ribonucleic acid ">
		 crRNA
	</DFN>
	 (tracrRNA)
	<A href=" http://science.sciencemag.org/content/337/6096/ "><ABBR title=" A programmable dual-RNA-guided DNA endonuclease in adaptive bacterial immunity ">
			 (Jinek
			<EM>
				 et al.
			</EM>
			 2012,
		</ABBR></A><A href=" http://www.pnas.org/content/pnas/109/39/E2579.full.pdf "><ABBR title=" Cas9-crRNA ribonucleoprotein complex mediates specific DNA cleavage for adaptive immunity in bacteria
																																																									">
			 Gasiunas
			<EM>
				 et al.
			</EM>
			 2012).</ABBR></A>
	  During infection, the
	<DFN data-info=" clustered regularly interspaced short palindromic repeats ">
		 CRISPR-</DFN>locus will be transcribed into the pre-<DFN data-info=" clustered regularly interspaced short palindromic repeats ribonucleic acid ">crRNA
	</DFN>
	 containing the spacer region flanked by conservated repeats. The
	<DFN data-info=" trans-activating clustered regularly interspaced short palindromic repeats ribonucleic acid ">
		 tracrRNA
	</DFN>
	 binds to the pre-<DFN data-info=" clustered regularly interspaced short palindromic repeats ribonucleic acid ">crRNA
	</DFN>
	 due to 25 nucleotides which are complementary to the repeat regions of the pre-<DFN data-info=" clustered regularly interspaced short palindromic repeats ribonucleic acid ">crRNA
	</DFN><A href=" https://www.nature.com/articles/nature09886 "><ABBR title=" CRISPR RNA maturation by trans-encoded small RNA and host factor RNase III ">
			 (Deltcheva
			<EM>
				 et al.
			</EM>
			 2011).</ABBR></A>
	  After procession, the mature
	<DFN data-info=" clustered regularly interspaced short palindromic repeats ribonucleic acid: trans-activating clustered regularly interspaced short palindromic repeats ribonucleic acid ">
		 crRNA:tracrRNA
	</DFN>
	 duplex binds the
	<DFN data-info=" clustered regularly interspaced short palindromic repeats associated protein ">
		 Cas9
	</DFN>
	 with the
	<DFN data-info=" trans-activating clustered regularly interspaced short palindromic repeats ribonucleic acid ">
		 tracrRNA
	</DFN>
	 builded stem-loop constituting a
	<DFN data-info=" clustered regularly interspaced short palindromic repeats ">
		 CRISPR
	</DFN>
	 ribonucleoprotein (crRNP). The
	<DFN data-info=" clustered regularly interspaced short palindromic repeats ribonucleic acid ">
		 crRNA
	</DFN>
	 contained a 20 nucleotides large spacer region at the end of the N-terminus which is complementary to the invading DNA
	
	 (protospacer) therefore leading the
	<DFN data-info=" clustered regularly interspaced short palindromic repeats ribonucleoprotein ">
		 crRNP
	</DFN>
	 to the target. In addition, the presence of the Protospacer adjacent Motif (PAM, NGG in <I>S. pyogenes</I>) preceding the protospacer three nucleotides is necessary for the cleavage activity
	<A href=" https http://science.sciencemag.org/content/343/6176/1247997 "><ABBR title=" Structures of Cas9 Endonucleases Reveal RNA-Mediated Conformational Activation ">
			 (Jinek
			<EM>
				 et al.
			</EM>
			 2014,
		</ABBR></A><A href=" https://www.cell.com/action/showPdf?pii=S0092-8674%2814%2900156-1 "><ABBR title=" Crystal Structure of Cas9 in Complex with Guide RNA and Target DNA ">
			 Nishimasu
			<EM>
				 et al.
			</EM>
			 2014).</ABBR></A>
	  The
	<DFN data-info=" clustered regularly interspaced short palindromic repeats ribonucleic acid ">
		 crRNA
	</DFN>
	 and
	<DFN data-info=" trans-activating clustered regularly interspaced short palindromic repeats ribonucleic acid ">
		 tracrRNA
	</DFN>
	 can directly be linked by a linker loop into a single guided RNA (sgRNA) like we used in our project for a more simplified and modularized genetic engineering application
	<A href=" https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5033207/pdf/kgmc-06-04-1137690.pdf "><ABBR title=" A quick guide to CRISPR sgRNA design tools ">
			 (Brazelton
			<EM>
				 et al.
			</EM>
			 2016,
		</ABBR></A><A href=" http://science.sciencemag.org/content/337/6096/ "><ABBR title=" A programmable dual-RNA-guided DNA endonuclease in adaptive bacterial immunity ">
			 Jinek
			<EM>
				 et al.
			</EM>
			 2012).
	</ABBR></A></P></DIV></DIV></DIV></DIV><DIV class="collapsible"><DIV class="btn_expand"><DIV class="btn_e_title">Part Collection</DIV><DIV class="btn_e_desc">
The Marburg Collection is a powerful tool we created, tailored to <I>V. natriegens</I> to enable precise cloning at an unprecedented speed.
 </DIV></DIV><DIV class="content"><DIV class="collapsible"><DIV class="btn_expand">Genetic toolboxes, a review</DIV><DIV class="content"><P>
					Golden Gate Assembly is a revolutionary cloning method published 10 years ago (<A href="https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0003647"><ABBR title="Engler C, Kandzia R, Marillonnet S. 2008. A One Pot, One Step, Precision Cloning Method with High Throughput Capability. PLOS ONE 3: e3647.">Engler <I>et al.</I>, 2008</ABBR></A>). since its publication it became very popular giving rise to many assembly strategies based on this system. All systems published over the years offered improvements over previous publications. Nevertheless, although there are many systems available – often specialized for certain organisms or classes – the development by far is not at an end. We worked on a novel toolbox and developed an improved, highly flexible system focused on prokaryotes including a high number of parts characterized for the novel upcoming chassis <I>Vibrio natriegens</I>.
					
					The innovation given by Golden Gate assembly was the introduction of type IIS restriction enzymes used for DNA digestion. A current workflow at that time consisted of DNA restriction, gel purification and finally ligation of the single parts. For restriction different type II restriction enzymes were taken making it important to check for compatible buffers and getting a six base pair scar after ligation. By introducing type IIS restriction enzymes into this workflow the system was highly improved. This new method offered many advantages like I) an independent enzyme recognition site that is removed after restriction, II) restriction and ligation in one pot III) directional assembly by conscious design of the cleavage sequences IV) no buffer incompatibilities due to the usage of one enzyme cutting at several spots and still producing different cleavage sequences (<A href="https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0003647"><ABBR title="Engler C, Kandzia R, Marillonnet S. 2008. A One Pot, One Step, Precision Cloning Method with High Throughput Capability. PLOS ONE 3: e3647.">Engler <I>et al.</I>, 2008</ABBR></A>).
					</P><FIGCAPTION><B>Figure 1: Golden Gate assembly.</B>
							Golden Gate assembly introduced the usafe of type IIS restriction enzymes which cut the DNA outside their recognition sites. This way the restriction site is removed from the cutted fragment after the restriction and the resulting overhangs can be designed freely.
						</FIGCAPTION></DIV></DIV></DIV></DIV></ARTICLE>
					Based on this, many new systems arose offering the possibility of building complex constructs out of single genetic parts in a fast, simple and flexible way. Full libraries of standardized genetic parts were developed enabling the user to build full combinatorial libraries. The first system developed was the modular cloning system (MoClo) offering a new hierarchical cloning method of defined biological parts (<A href="https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0016765"><ABBR title="Weber E, Engler C, Gruetzner R, Werner S, Marillonnet S. 2011. A Modular Cloning System for Standardized Assembly of Multigene Constructs. PLOS ONE 6: e16765.">Weber <I>et al.</I>, 2011</ABBR></A>) and consequently setting a new standard over the years.
					Here we summarize some Golden Gate based systems in a chronological manner looking at the improvements over the years and compare their advantages and disadvantages.
				<DIV class="collapsible"><DIV class="btn_expand">Modular Cloning (MoClo) (<A href="https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0016765"><ABBR title="Weber E, Engler C, Gruetzner R, Werner S, Marillonnet S. 2011. A Modular Cloning System for Standardized Assembly of Multigene Constructs. PLOS ONE 6: e16765.">Weber <I>et al.</I>, 2011</ABBR></A>)</DIV><DIV class="content"><P>
							The modular cloning system was the first one proposing a standard for Golden Gate based assembly. This toolbox offers 5 types of modules designed mainly for eukaryotes. The modules are stored in level 0 acceptor plasmids derived from the pUC19 backbone with a spectinomycin resistance and a LacZα cassette as dropout for blue/white screening. Custom level 0 plasmids are assembled by flanking the sequences with BpiI recognition sites and setting a single restriction-ligation-reaction with the correspondent plasmid. Up to 5 level 0 modules are assembled in an acceptor plasmid with ampicillin resistance and a LacZα-dropout by restriction with BsaI to transcription units. 6 transcription units can be assembled using BpiI into level 2 multigene constructs containing a kanamycin resistance and a Cred-dropout. Alternatively Esp3I can be use to transfer the constructs to intermediary levels to reach higher levels for assembly of bigger constructs.
							This toolbox is best for constructs up to level 2 plasmids. Higher levels can be reached through intermediary levels but need two restriction enzymes for the assembly. Furthermore it is important to know in which level the current constructs stand to avoid messing up the acceptor plasmids for subsequent levels.
						</P></DIV></DIV><DIV class="collapsible"><DIV class="btn_expand">Golden Braid (<A href="https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0021622"><ABBR title="Sarrion-Perdigones A, Falconi EE, Zandalinas SI, Juárez P, Fernández-del-Carmen A, Granell A, Orzaez D. 2011. GoldenBraid: An Iterative Cloning System for Standardized Assembly of Reusable Genetic Modules. PLOS ONE 6: e21622.">Sarrion-Perdigones <I>et al.</I>, 2011</ABBR></A>)</DIV><DIV class="content"><P>
							The Golden Braid system was mainly developed with focus on plants. It picks up the idea of the MoClo system and improves it offering a looping system for reaching higher levels of assembly. Level 0 modules are stored in entry plasmids surrounded by BsaI restriction sites containing an ampicillin resistance and a LacZα dropout. This modules are assembled to transcription units in one of two level α entry plasmids with kanamycin resistance and a LacZα dropout. Here the transcription units are surrounded by BsmBI restriction sites. The transcription units can subsequently be transferred into level Ω entry plasmids with spectinomycin resistance and BsaI restriction sites surrounding the multigene constructs. Every level α plasmid is a precursor of a level Ω plasmid and vise versa. This way the system allows to loop over the plasmids building in theory infinite bigger constructs doubling their size on every round.
							This system makes the assembly of higher levels much simpler using only 1 restriction enzyme. Nevertheless the plasmids’ size is just doubled from level to level.
						</P></DIV></DIV><DIV class="collapsible"><DIV class="btn_expand">Golden Braid 2.0 (<A href="http://www.plantphysiol.org/content/162/3/1618"><ABBR title="Sarrion-Perdigones A, Vazquez-Vilar M, Palací J, Castelijns B, Forment J, Ziarsolo P, Blanca J, Granell A, Orzaez D. 2013. GoldenBraid 2.0: A Comprehensive DNA Assembly Framework for Plant Synthetic Biology. Plant Physiology 162: 1618–1631.">Sarrion-Perdigones <I>et al.</I>, 2013</ABBR></A>)</DIV><DIV class="content"><P>
							The Golden Braid 2.0 system is an improvement of its prior system. Some minor changes were introduced to make the system simpler and more versatile. The basic modules were defined and classified in 11 classes building optional subclasses e.g. protein tags. The level 0 modules are stored in a universal vector (pUPD) surrounding the sequences with BsaI and BtgZI recognition sites cutting on the same position. The current acceptor plasmids α and Ω have been supplemented by reverse plasmids allowing the inversion of single transcription units.  The fusion sites were simplified by merging unnecessary ones. ‘Superparts’ were introduced consisting of pre-assembled modules to reduce the number of fragments needed for the assembly.
							This changes do not create a completely new toolbox but complements the system with some handy facilities improving it notably. However the system is still limited to a duplication of the plasmids size from level to level.
						</P></DIV></DIV><DIV class="collapsible"><DIV class="btn_expand">MoClo – YTK (<A href="https://doi.org/10.1021/sb500366v"><ABBR title="Lee ME, DeLoache WC, Cervantes B, Dueber JE. 2015. A Highly Characterized Yeast Toolkit for Modular, Multipart Assembly. ACS Synthetic Biology 4: 975–986.">Lee <I>et al.</I>, 2015</ABBR></A>)</DIV><DIV class="content"><P>
							The yeas toolkit, also known as the Dueber toolbox, offers a Golden Gate based system adapted for yeast. The basic level 0 parts are classified in 8 types with optional subtypes. New basic parts are assembled into entry plasmids by restriction with BsmBI. For building level 1 cassettes at least 8 parts are assembled by a restriction-ligation step using BsaI. The innovation of this toolkit compared to the previously reviewed is the use of connector sequences for level 1 and higher assembly steps. This way plasmids for yeast can be built <I>de novo</I> without the need of a defined backbone. Furthermore they integrated a method for simple chromosomal integration by lenearization of the plasmids with NotI. On top of this the connectors can be used as homology sequences e.g. for ligation-independent cloning, Gibson assembly, ligase cycling reaction or yeast <I>in vivo</I> assembly. Although this toolkit focuses mainly on yeast it can be used for working with <I>E. coli</I> as well. Unfortunately this toolkit does not arrange to any standard becoming incompatible with other toolkits.
						</P></DIV></DIV><DIV class="collapsible"><DIV class="btn_expand">PhytoBrick (<A href="https://nph.onlinelibrary.wiley.com/doi/abs/10.1111/nph.13532"><ABBR title="Patron NJ, Orzaez D, Marillonnet S, Warzecha H, Matthewman C, Youles M, Raitskin O, Leveau A, Farré G, Rogers C, et al. 2015. Standards for plant synthetic biology: a common syntax for exchange of DNA parts. New Phytologist 208: 13–19.">Patron <I>et al.</I>, 2015</ABBR></A>)</DIV><DIV class="content"><P>
							The PhytoBrick system does not offer another genetic toolbox but a wide standard compatible with popular systems like GB 2.0 and MoClo aiming to create a standard focused primary on plant engineering efforts. The iGEM competition already accepted it as an standard and offers support for building parts designed for plants, yeast and bacteria. This system proposes 12 defined fusion sites applicable for the different genetic modules. The fusion sites are divided into three major classes for promoter parts, transcribed regions and terminator parts. This classes are divided into subclasses giving the flexibility to use optional modules like tags, promoter regulators and enhancer regions. The system also proposes two types of universal acceptor plasmids (UAPs) derived from the pSB1C3 plasmid where level 0 modules can be inserted by a single restriction-ligation step with BpiI or BsmBI respectively.
						</P></DIV></DIV><DIV class="collapsible"><DIV class="btn_expand">CIDAR MoClo (<A href="https://doi.org/10.1021/acssynbio.5b00124"><ABBR title="Iverson SV, Haddock TL, Beal J, Densmore DM. 2016. CIDAR MoClo: Improved MoClo Assembly Standard and New E. coli Part Library Enable Rapid Combinatorial Design for Synthetic and Traditional Biology. ACS Synthetic Biology 5: 99–103.">Iverson <I>et al.</I>, 2016</ABBR></A>)</DIV><DIV class="content"><P>
							The Cross-disciplinary Integration of Design Automation Research lab (CIDAR) MoClo Library represents the first toolbox adapted for bacteria focused on <I>E. coli</I>. It offers 95 genetic modules for bacteria and proposes a modified protocol reducing the reaction time of the assembly steps from 5 hours to just 90 minutes. The toolkit provides a series of level 0 plasmids derived from the iGEM pSB1A2 backbone. Four modules are assembled into a series of level 1 destination plasmids derived from pSB1K3 to transcription units using BsaI. In a next step this transcription units can be assembled to multigene modules by restriction with BbsI. Up to 4 transcription units can be assembled to a level 2 multigene construct.
						</P></DIV></DIV><DIV class="collapsible"><DIV class="btn_expand">EcoFlex (<A href="https://doi.org/10.1021/acssynbio.6b00031"><ABBR title="Moore SJ, Lai H-E, Kelwick RJR, Chee SM, Bell DJ, Polizzi KM, Freemont PS. 2016. EcoFlex: A Multifunctional MoClo Kit for E. coli Synthetic Biology. ACS Synthetic Biology 5: 1059–1069.">Moore <I>et al.</I>, 2016</ABBR></A>)</DIV><DIV class="content"><P>
							EcoFlex is a toolbox specialized on the use with <I>E. coli</I> that offers a wide variety of modules. It features a library of constitutive promoters, T7 expressions, RBS variations, synthetic terminators, protein purification tags and fluorescence proteins characterized <I>in vivo</I> and <I>in vitro</I>. Furthermore it is compatible with MoClo, Golden Braid 2.0 and PhytoBrick standard.
							One unique feature of this kit is the ‘entry vector’ for ORFs. The parts are flanked by NdeI and BamHI sites again flanked by BsaI restriction sites. The NdeI sites provide the fusion sites for Golden Gate assembly without addition of amino acids to the coded protein and in combination with BamHI additionally  providing compatible transfer to the commercial available pET vectors. If NdeI or BamHI are present in the coded protein NcoI and BglII can be used respectively for the assembly.
							The second unique feature of EcoFlex is the inclusion of a secondary module. Here level 2 transcription units (TUs) can be subcloned into a corresponding secondary module using compatible BpiI sites located upstream the primary MoClo BsmBI sites. This way a module of multiple TUs can be ligated into a secondary module containing further TUs. The introduction of this feature offers a simplified and faster method for combinatorial pathway optimization.
						</P></DIV></DIV><DIV class="collapsible"><DIV class="btn_expand">Loop assembly (<A href="https://www.biorxiv.org/content/early/2018/01/18/247593"><ABBR title="Pollak B, Cerda A, Delmans M, Álamos S, Moyano T, West A, Gutiérrez RA, Patron N, Federici F, Haseloff J. 2018. Loop Assembly: a simple and open system for recursive fabrication of DNA circuits. bioRxiv: 247593.">Pollak <I>et al.</I>, 2018</ABBR></A>)</DIV><DIV class="content"><P>
							Loop assembly provides a new toolbox designed for plant engineering purposes. It contains a series of 8 plasmids derived from pGreenII backbone divided in two sets to loop over several levels of assembly. Level 0 modules are assembled into odd receiver plasmids (pOdd) with a kanamycin resistance by restriction with BsaI. Four odd level transcription units are assembled into one even level receiver plasmid (pEven) with a spectinomycin resistance by restriction with SapI. The system can be elevated to higher levels by looping over odd and even receiver plasmids exponentially by the factor of 4. For the case that less than 4 modules are needed spacer plasmids are offered of 200bp random DNA free of BsaI or SapI recognition sites to close the gaps. The level 0 parts’ fusion sites are designed using the PhytoBrick standard previously reviewed and are therefore compatible with GB 2.0 and MoClo. A further improvement of this system is the use of unique nucleotide sequences (UNS) set between single transcription units. This UNS allow highly efficient PCR amplification of single functional units and allow the assembly into the plasmids by overlap assembly methods like Gibson assembly as well.
							Although this is an improved and very simple but useful system the usage of SapI delivers a 3 bp fusion site restricting the number of possible fusion sites to a lower number and simultaneously downgrading compatibility with previous systems.
						</P></DIV></DIV><DIV class="collapsible"><DIV class="btn_expand">Summary</DIV><DIV class="content"><DIV style="overflow-x:scroll"><TABLE style="width:100%;"><TBODY><TR><TH>Toolkit</TH><TH>MoClo</TH><TH>Golden Braid</TH><TH>Golden Braid 2.0</TH><TH>MoClo – YTK</TH><TH>PhytoBrick</TH><TH>CIDAR MoClo</TH><TH>EcoFlex</TH><TH>Loop Assembly</TH></TR><TR><TH>Year of publication</TH><TD>2011</TD><TD>2011</TD><TD>2013</TD><TD>2015</TD><TD>2015</TD><TD>2015</TD><TD>2016</TD><TD>2018</TD></TR><TR><TH>Restriction sites</TH><TD>BsaI, BpiI (Esp3 for intermediary levels)</TD><TD>BsaI, BsmBI</TD><TD>BsaI, BsmBI (BtgZI as BsaI alternative)</TD><TD>BsaI, BsmBI (NotI for chromosomal integration)</TD><TD>BsaI, BsmBI (BpiI for compatibility with MoClo)</TD><TD>BsaI, BbsI</TD><TD>BsaI, BsmBI (NdeI + BamHI for secondary modules)</TD><TD>BsaI, SapI</TD></TR><TR><TH>Dropouts for negative selection</TH><TD>LacZα, CRed</TD><TD>LacZα</TD><TD>LacZα</TD><TD>sfGFP (only lvl 0)</TD><TD>Depends on the system respectively</TD><TD>LacZα</TD><TD>LacZα, RFP</TD><TD>LacZα</TD></TR><TR><TH>Resistances</TH><TD>Spectinomycin → ampicillin → kanamycin</TD><TD>Ampicillin → kanamycin → spectinomycin</TD><TD>Ampicillin → kanamycin → spectinomycin</TD><TD>chloramphenicol → ampicillin → kanamycin</TD><TD>Chloramphenicol, ampicillin, kanamycin, spectinomycin</TD><TD>Ampicillin → kanamycin</TD><TD>Chloramphenicol → β-lactamase</TD><TD>Kanamycin → spectinomycin</TD></TR><TR><TH>Target organisms</TH><TD>Mainly eukaryotes</TD><TD>Plants</TD><TD>Plants</TD><TD>Yeast</TD><TD>Plants</TD><TD>Bacteria</TD><TD>E. coli</TD><TD>Plants</TD></TR><TR><TH>Special features</TH><TD>First Golden Gate Assembly standard</TD><TD>Theoretically infinite looping over two sets of plasmids and levels</TD><TD>Flexible direction inversion of single TUs during assembly steps, introduction of classes and subclasses</TD><TD>Connector-based de novo plasmid assembly for yeast, chromosomal integration, </TD><TD>Not a toolbox but a proposal for a standard widely compatible with popular systems, accepted standard for the iGEM competition</TD><TD>First system focused on bacteria, enhanced protocol for Golden Gate reaction in 90 minutes</TD><TD>Widely versatile toolbox, adapted for E. coli, designed for circuit design, combinatorial pathway optimization, recombinant protein purification</TD><TD>Simple usage, wide compatibility due to PhytoBrick standard, usage of UNS</TD></TR><TR><TH>Enhancement per level</TH><TD>6 TUs, but capped at level 3</TD><TD>2<SUP>n</SUP></TD><TD>2<SUP>n</SUP></TD><TD>Depends on the system respectively</TD><TD>4<SUP>n</SUP></TD><TD>Up to 5 TUs on level 2 and up to 20 on level 3</TD><TD>4<SUP>n</SUP></TD></TR><TR><TH>Number of parts</TH><TD>95</TD><TD>94</TD><TD>96</TD><TD>93</TD><TD>78</TD><TD>58</TD></TR></TBODY></TABLE></DIV></DIV></DIV></MAIN></DIV></DIV></DIV></DIV><DIV class="collapsible"><DIV class="btn_expand"><DIV class="btn_e_title">Metabolic Engineering</DIV><DIV class="btn_e_desc">We want to show the suitability of <I>V. natriegens</I> for Metabolic Engineering due to its genetic accessability, fast growth and high substrate-uptake rates.</DIV></DIV><DIV class="content"><DIV class="collapsible"><DIV class="btn_expand">Our vision of Accelerating Metabolic Engineering</DIV><DIV class="content"><H3>Our Goal</H3><P>
	Our goal is to accelerate metabolic engineering by establishing a workflow for fast pathway construction, product screening and pathway optimization. 
	Metabolic engineering is a broad field in synthetic biology, where existing pathways are divided into modular parts and then used and combined to build new pathways 

	<A href="https://doi.org/10.1021/sb300094q"><ABBR title="Stephanopoulos, G., 2012. Synthetic biology and metabolic engineering. ACS Synth. Biol. 1, 514–525. ">(Stephanopoulos <I>et al.</I>2012)</ABBR></A>

	. 
	One of the best examples for the impact of metabolic engineering on solving global problems is the engineered production pathway for artemisinic acid, the precursor for a drug against malaria 

	<A href="https://doi.org/10.1038/nature04640"><ABBR title="Ro, D.K., Paradise, E.M., Quellet, M., Fisher, K.J., Newman, K.L., Ndungu, J.M., Ho, K.A., Eachus, R.A., Ham, T.S., Kirby, J., Chang, M.C.Y., Withers, S.T., Shiba, Y., Sarpong, R., Keasling, J.D., 2006. Production of the antimalarial drug precursor artemisinic acid in engineered yeast. Nature 440, 940–943.">(Ro <I>et al.</I>2006)</ABBR></A>.

	In the old days the drug had to be isolated out of the plant <I>Artemisia annua</I>. 
	This was very inefficient because of the limitations of plant growth and low yields of artemisinic acid. 
	Therefore, the group of Jay Keasling engineered a metabolic pathway to produce the drug in yeast cells, which grow faster and produce a lot more artemisinic acid. 
	As in this example, metabolic engineering enables the production of drugs and other valuable products with an increased yield when compared to product isolation from natural sources. 
	Furthermore, the large variety of usable enzymes, pathways and chassis enables more flexibility in pathway design compared to chemical approaches. 
</P><H3>Why should we accelerate metabolic engineering?</H3><P>
	Although much progress in metabolic engineering was made in the last 20 years, there are still many drawbacks and limitations. 
	For instance, it is crucial to have a big set of genes as combinable parts to find the optimal pathway and they have to be predictable. 
	With many of them just poorly characterized, rational pathway design remains difficult. 
	Our understanding of the working and interactions of enzymes at the level of the metabolism is still rudimentary.  Consequently, metabolic engineering consists of much trial-and-error and researchers have to endure series of failures. 
	One strategy to face these issues is modeling of enzyme behavior but currently limitations in computing power prevent this from being used in high-throughput. Another strategy is to use chassis as workhorses to test  a lot of enzyme combinations and pathway variants. Here, organisms like <I>E. coli</I>, <I>S. cerevisiae</I> or <I>C. glutamicum</I> are often used but product formation is limited by growth rate and nutrient uptake 

	<A href="https://doi.org/10.1126/science.1139612"><ABBR title="Stephanopoulos, G., 2007. Challenges in engineering microbes for biofuels production. Science 315, 801–4. ">(Stephanopoulos <I>et al.</I>2007)</ABBR></A>.

	When a pathway is constructed and to be tested, one has to wait until the production strain has grown and then product formation is still limited by substrate uptake. Another big drawback is that screening for product often is expensive, time consuming and complicated. 
	Conventional methods like LC-MS need a lot of time and resources and it is hardly possible to use it in high-throughput, for instance for a whole pathway library.
	To overcome these limitations and accelerate metabolic engineering, we established <I>V. natriegens</I> as chassis for metabolic engineering. 
	Its doubling time of under 10 minutes makes it the perfect organism to quickly test many pathway variants or enzyme versions. 
	Additionally, it has a glucose uptake rate of 3.90 g g-1h-1 while <I>E. coli</I>, <I>S. cerevisiae</I> and <I>C. glutamicum</I> have much lower rates (1.90, 3.52 and 0.37 g g-1 h-1, respectively). 
	Coupled with the short doubling time, <I>V. natriegens</I> is a powerful chassis not only for research purposes but also for industrial usage, where high productivity is crucial.




</P><H3>How are we accelerating metabolic engineering?</H3><DIV style="display:inline-block;"><FIGURE style="float:right; width: 45%;"><FIGCAPTION><B>Figure 1: Metabolic Workflow established by iGEM Team Marburg 2018</B></FIGCAPTION></FIGURE>

	For optimal execution of metabolic engineering we established a workflow, basing on the <DFN data-info="Design-Build-Test-Learn">DBTL</DFN>-cycle. 
	We use our Marburg Collection to build whole pathway libraries, which can then be transformed into our producer strain VibriXpress.
	By using biosensors for product screening, we circumvent the expensive and time-intensive usage of <DFN data-info="Liquid Chromatography-Mass Spectrometry">LC-MS</DFN> and learn which pathway variants are the most promising. 
	These findings can then be fed into our pathway-design, thereby closing the DBTL-cycle. 
	To further optimize these pathways, we planned to use directed evolution for more adaptations to that route.
</DIV></DIV></DIV><DIV class="collapsible"><DIV class="btn_expand">Production of 3-hydroxypropionic acid</DIV><DIV class="content"><H3>3-Hydroxypropionic acid <DFN data-info="3-hydroxypropionic acid"> 3HPA </DFN></H3><P>
	As a proof of concept for our engineering workflow we chose the biological production of 3-Hydroxypropionic acid (<DFN data-info="3-hydroxypropionic acid"> 3HPA </DFN>). <DFN data-info="3-hydroxypropionic acid"> 3HPA </DFN> is a compound of high industrial value. 
	In 2004 the U.S. Department of Energy recommended <DFN data-info="3-hydroxypropionic acid"> 3HPA </DFN> as alternative to fossil oils for the chemical industry  

	<A href="LINK"><ABBR title="Werpy, T ; Petersen, G ; Aden, A ; Bozell, J ; Holladay, J ; White, J ; Manheim, Amy ; Eliot, D ; Lasure, L ; Jones, S, Top Value Added Chemicals From Biomass. Volume 1 - Results of Screening for Potential Candidates From Sugars and Synthesis Gas,DEPARTMENT OF ENERGY WASHINGTON DC">(Werpy <I>et al.</I>2004)</ABBR></A>. 

	<DFN data-info="3-hydroxypropionic acid"> 3HPA </DFN> as a platform chemical can be converted into many other compounds like acrylic acid and acrylamide, which both are also precursors for further compounds. 
	According to an estimate of 2014, acrylates alone have  an annual market value of USD12 billion, making its precursor <DFN data-info="3-hydroxypropionic acid"> 3HPA </DFN> an optimal target product for our metabolic engineering approach. 
	By finding the optimal pathway to produce <DFN data-info="3-hydroxypropionic acid"> 3HPA </DFN> we could show the efficiency of metabolic engineering in <I>V. natriegens</I> as well as getting a big step closer to an alternative to fossil oils by biological, renewable resources.

	<FIGURE style="float: top; width: 90%;"><FIGCAPTION><B>Figure 2:</B> Industrial applications of 3-hydroxypropionate.
 	</FIGCAPTION></FIGURE></P><H3>Our pathway</H3><P>
	In recent years many people concentrated on producing <DFN data-info="3-hydroxypropionic acid"> 3HPA </DFN> via many different metabolic pathways 

	<A href="https://doi.org/10.1007/s00253-013-4802-4"><ABBR title="Valdehuesa, K.N.G., Liu, H., Nisola, G.M., Chung, W.J., Lee, S.H., Park, S.J., 2013. Recent advances in the metabolic engineering of microorganisms for the production of 3-hydroxypropionic acid as C3 platform chemical. Appl. Microbiol. Biotechnol. 97, 3309–3321. ">(Valdehuesa <I>et al.</I>2013)</ABBR></A>
;

	 <A href="https://doi.org/10.2172/15008859"><ABBR title="Vidra, A., Németh, Á., 2017. Bio-based 3-hydroxypropionic Acid : A Review. period. Polytech. Chem. Eng. 1–11.
	Werpy, T., Petersen, G., 2004. Top Value Added Chemicals from Biomass Volume I — Results of Screening for Potential Candidates from Sugars and Synthesis Gas Top Value Added Chemicals From Biomass Volume I : Results of Screening for Potential Candidates. Other Inf. PBD 1 Aug 2004 Medium: ED; Size: 76 pp. pages. ">(Vidra <I>et al.</I>2017)</ABBR></A>. 

	Most of these pathways use glycerol or glucose as starting substrate, but there are also publications where acrylic acid, CO2 or uracil were converted to <DFN data-info="3-hydroxypropionic acid"> 3HPA </DFN><A href="https://doi.org/10.2172/15008859"><ABBR title="Vidra, A., Németh, Á., 2017. Bio-based 3-hydroxypropionic Acid : A Review. period. Polytech. Chem. Eng. 1–11.
	Werpy, T., Petersen, G., 2004. Top Value Added Chemicals from Biomass Volume I — Results of Screening for Potential Candidates from Sugars and Synthesis Gas Top Value Added Chemicals From Biomass Volume I : Results of Screening for Potential Candidates. Other Inf. PBD 1 Aug 2004 Medium: ED; Size: 76 pp. pages. ">(Vidra <I>et al.</I>2017)</ABBR></A>. 

	One promising route starts with glycerol which is dehydrated and oxidized. 
	For <I>E. coli</I>, a productivity of 6.6 mmol g-1 cdw h-1  has been shown 

	<A href="https://doi.org/10.1016/j.procbio.2008.04.027"><ABBR title="Raj, S.M., Rathnasingh, C., Jo, J.E., Park, S., 2008. Production of 3-hydroxypropionic acid from glycerol by a novel recombinant Escherichia coli BL21 strain. Process Biochem. 43, 1440–1446.">(Raj <I>et al.</I>2008)</ABBR></A>.


	However, this route depends on vitamin B12, which often can’t be taken up or produced by the chassis. 
	Many other routes start with glucose, which gets metabolized via propionate, lactate, β-alanine or malonyl-CoA 

	<A href="https://doi.org/10.2172/15008859"><ABBR title="Vidra, A., Németh, Á., 2017. Bio-based 3-hydroxypropionic Acid : A Review. period. Polytech. Chem. Eng. 1–11.
	Werpy, T., Petersen, G., 2004. Top Value Added Chemicals from Biomass Volume I — Results of Screening for Potential Candidates from Sugars and Synthesis Gas Top Value Added Chemicals From Biomass Volume I : Results of Screening for Potential Candidates. Other Inf. PBD 1 Aug 2004 Medium: ED; Size: 76 pp. pages. ">(Vidra <I>et al.</I>2017)</ABBR></A>. 

	We decided to use a pathway, based on the conversion of acetyl-CoA into malonyl-CoA and finally <DFN data-info="3-hydroxypropionic acid"> 3HPA </DFN>. According to 

	<A href="https://doi.org/10.1007/s00253-013-4802-4"><ABBR title="Valdehuesa, K.N.G., Liu, H., Nisola, G.M., Chung, W.J., Lee, S.H., Park, S.J., 2013. Recent advances in the metabolic engineering of microorganisms for the production of 3-hydroxypropionic acid as C3 platform chemical. Appl. Microbiol. Biotechnol. 97, 3309–3321. ">(Valdehuesa <I>et al.</I>2013)</ABBR></A>

	who evaluated many pyruvate-derived production pathways from a thermodynamic point of view, this is one of the most efficient routes for <DFN data-info="3-hydroxypropionic acid"> 3HPA </DFN> production. 
	This pathway was first established in <I>E. coli</I>, but it was also  tried in many other chassis like <I>S. cerevisiae and S. elongatus</I><A href="https://doi.org/10.1186/s12934-016-0451-5"><ABBR title="Kildegaard, K.R., Jensen, N.B., Schneider, K., Czarnotta, E., Özdemir, E., Klein, T., Maury, J., Ebert, B.E., Christensen, H.B., Chen, Y., Kim, I.K., Herrgård, M.J., Blank, L.M., Forster, J., Nielsen, J., Borodina, I., 2016. Engineering and systems-level analysis of Saccharomyces cerevisiae for production of 3-hydroxypropionic acid via malonyl-CoA reductase-dependent pathway. Microb. Cell Fact. 15. ">(Kildegaard <I>et al.</I>2016)</ABBR></A>; 

	<A href="https://doi.org/10.1080/07388551.2016.1272093
	"><ABBR title="Liu, C., Ding, Y., Xian, M., Liu, M., Liu, H., Ma, Q., Zhao, G., 2017. Malonyl-CoA pathway: a promising route for 3-hydroxypropionate biosynthesis. Crit. Rev. Biotechnol. 37, 933–941. ">(Liu <I>et al.</I>2017)</ABBR></A>; 

	<A href="https://doi.org/10.1016/j.jbiotec.2011.06.008"><ABBR title="Rathnasingh, C., Raj, S.M., Lee, Y., Catherine, C., Ashok, S., Park, S., 2012. Production of 3-hydroxypropionic acid via malonyl-CoA pathway using recombinant Escherichia coli strains. J. Biotechnol. 157, 633–640. ">(Rathnasingh <I>et al.</I>2012)</ABBR></A>.


	We chose this route for several reasons. 
	Firstly, it is based on glucose degradation and <I>V. natriegens</I> has an unbeaten glucose-uptake rate, enabling high glucose consumption. Secondly, vitamin B12 is not necessary, the only cofactors involved are NADPH and biotin, both of which occur naturally in the organism. 
	Thirdly, theoretically just one further enzyme is needed to complete the pathway. All enzymes necessary for the production of malonyl-CoA are already present in <I>V. natriegens</I> and we just have to integrate the last enzyme, malonyl-CoA reductase. 
	Nevertheless, overexpression of acetyl-CoA carboxylase increases production, so we took both enzymes into consideration. 

	<A href="https://doi.org/10.1080/07388551.2016.1272093
	"><ABBR title="Liu, C., Ding, Y., Xian, M., Liu, M., Liu, H., Ma, Q., Zhao, G., 2017. Malonyl-CoA pathway: a promising route for 3-hydroxypropionate biosynthesis. Crit. Rev. Biotechnol. 37, 933–941. ">(Liu <I>et al.</I>2017)</ABBR></A>

	proposed, that there are many ways to further optimize the pathway and direct the flow towards a high product titer. 
	Hence, implementing many variations of this pathway into <I>V. natriegens</I> and testing them in a fast manner promises a wealth  of knowledge about pathway optimization and general principles of metabolic engineering. 

	<FIGURE style="float: top; width: 90%;"><FIGCAPTION><B>Figure 2:</B> Pathway we have chosen for our metabolic engineering efforts.
 	</FIGCAPTION></FIGURE></P><DIV style="display:inline-block;">

	The first relevant enzyme in the pathway is the acetyl-CoA carboxylase (Acc) that catalyzes the carboxylation of acetyl-CoA. It is involved in fatty acid biosynthesis,consisting of four subunits in <I>E. coli</I>, a biotin-carboxylase (BC), a biotin-carboxyl-carrier-protein domain (BCCP) and two carboxyltransferase subunits (CT) 

	
	<A href="https://doi.org/10.1186/1754-6834-7-7"><ABBR title="Janßen, H.J., Steinbüchel, A., 2014. Fatty acid synthesis in Escherichia coli and its applications towards the production of fatty acid based biofuels. Biotechnol Biofuels 7, 7. ">(Jansen <I>et al.</I>2004)</ABBR></A>. 


	During catalysis, a biotin molecule is first linked to a lysine residue in the BCCP domain. 
	Via ATP-hydrolysis the BC domain carboxylates the biotin, which then induces a conformational change and brings the carboxy-group into close proximity of the CT domain. Finally, the CT domain transfers the carboxy-group to the acetyl-CoA, leading to the formation of malonyl-CoA 

	<A href="https://doi.org/10.1002/prot.21952"><ABBR title="Lee, C.-K., Cheong, H.-K., Ryu, K.-S., Lee, J. Il, Lee, W., Jeon, Y.H., Cheong, C., 2008. Biotinoyl domain of human acetyl-CoA carboxylase: Structural insights into the carboxyl transfer mechanism. Proteins Struct. Funct. Bioinforma. 72, 613–624. ">(Lee <I>et al.</I>2008)</ABBR></A>

	The subunits are organized in four genes in <I>E. coli</I><A href="https://doi.org/10.1016/j.jbiotec.2011.06.008"><ABBR title="Rathnasingh, C., Raj, S.M., Lee, Y., Catherine, C., Ashok, S., Park, S., 2012. Production of 3-hydroxypropionic acid via malonyl-CoA pathway using recombinant Escherichia coli strains. J. Biotechnol. 157, 633–640. ">(Rathnasingh <I>et al.</I>2012)</ABBR></A>

	, however, in other organisms like C. glutamicum the genes are located in two operons 

	<A href="https://doi.org/10.1128/JB.00254-07"><ABBR title="Gande, R., Dover, L.G., Krumbach, K., Besra, G.S., Sahm, H., Oikawa, T., Eggeling, L., 2007. The two carboxylases of corynebacterium glutamicum essential for fatty acid and mycolic acid synthesis. J. Bacteriol. 189, 5257–5264. ">(Gande <I>et al.</I>2007)</ABBR></A>.



	<FIGURE style="width: 50%; float: left;"><FIGCAPTION><B>Figure 3: Mechanism of ACC </B></FIGCAPTION></FIGURE>

	The second relevant enzyme in the pathway is the malonyl-CoA reductase (Mcr), which is involved in CO2 fixation by the 3-hydroxypropionate bicycle in thermophilic bacteria 

	<A href="https://doi.org/10.1074/jbc.M110802200"><ABBR title="Alber, B.E., Fuchs, G., 2002. Propionyl-coenzyme a synthase from Chloroflexus aurantiacus, a key enzyme of the 3-hydroxypropionate cycle for autotrophic CO2 fixation. J. Biol. Chem. 277, 12137–12143. ">(Alber <I>et al.</I>2002)</ABBR></A>.

	 It is a 132kDa large, bifunctional enzyme with alcohol dehydrogenase and aldehyde dehydrogenase function. The C-terminal domain catalyzes the NADPH-dependent reduction of malonyl-CoA into 3-oxopropanoic acid, while the N-terminal domain catalyzes consecutive reduction of 3-oxopropanoic acid to 3-hydroxypropionate 

	 <A href="https://doi.org/10.1371/journal.pone.0075554"><ABBR title="Liu, C., Wang, Q., Xian, M., Ding, Y., Zhao, G., 2013. Dissection of Malonyl-Coenzyme A Reductase of Chloroflexus aurantiacus Results in Enzyme Activity Improvement. PLoS One 8, 1–8.">(Liu <I>et al.</I>2013)</ABBR></A>.


</DIV></DIV></DIV><DIV class="collapsible"><DIV class="btn_expand">Sensor-based product screening</DIV><DIV class="content"><P>
	One major goal of synthetic biology is to make biology engineerable and predictable. Currently metabolic engineering does still involve a lot of trial and error. With the number of possibilities for pathway finetuning this scales very fast, becoming very difficult to tackle with traditional methods.
</P><P>
	Also, once we create such a library of metabolic pathway variants, possibly including thousands of strains, we have to find the most promising ones among these. A traditional approach would be to check the production of each strain in its own culture flask. Using for example <DFN data-info="High performancy liquid chromatography"> HPLC </DFN> or mass spectrometry.
</P><P>
	A more suitable, easy way to quickly screen for metabolite concentration in a semi quantitative way are biosensors. One of the easiest to use classes of biosensors are metabolite-responsive transcription factors 

	<A href="https://www.sciencedirect.com/science/article/pii/S1096717615000774#bib46"><ABBR title="Applications and Advances of Metabolite Biosensors for Metabolic Engineering.">(Liu et al. 2018).</ABBR></A> 

    They are usually very specific towards their distinct substrates. In vivo, most of these systems help their native organism to react to environmental clues by regulating expression of for instance catabolic enzymes or importers.
 In the past, these have ben co-opted by researchers to function as sensors or to control enzyme expression to regulate a pathway depending on the availability of a certain metabolite 

    <A href="http://www.ncbi.nlm.nih.gov/pubmed/26858408"><ABBR title="Genetically Encoded Sensors Enable Real-Time Observation of Metabolite Production.">(Rogers, Taylor, and Church 2016).</ABBR></A> 

    By using reporter genes like <DFN data-info="Green fluorescence protein"> GFP </DFN> or the <DFN data-info="luminescence operon"> LUX </DFN> operon, we have a measurable output that can be used to select for the best contenders. By running this in several iterations, nature's most powerful optimization tool, evolution, can be applied to further improve the pathway 

      <A href="https://www.sciencedirect.com/science/article/pii/S0167779916000354?via%3Dihub"><ABBR title="Synthetic Evolution of Metabolic Productivity Using Biosensors.">(Williams <I>	et al.	</I>	2016).</ABBR></A>

	 Applying fluorescence activated cell sorting (FACS) coupled with
	<I><I>V. natriegens</I></I>
	 incredible doubling time, thousands of generations can be screened for progressively better pathway variants.
</P><P>
	So, we decided to put our toolbox full of characterized parts to good use. The Marburg collection contains all the parts needed to find the ideal enzyme balance for highest production. Being able to build a large number of differentially assembled pathways is the answer to creating the most productive strain. Finding the right one in that complexity is another difficulty to overcome, one we solved by finding transcriptional regulators that specifically activate reporter gene expression relative to the concentration of key metabolites, 3-hydroxypropionate and Malonyl-CoA. This gives us a quantifiable output with which we are able to judge the different combinations. Also, utilizing the speed of
	<I><I>V. natriegens</I></I>
	, the time required for a high throughput screening can be reduced dramatically.
</P><P>
	We deemed the concentration of two different metabolites as being critical to our project: 3-Hydroxypropionate, the product itself, and malonyl-CoA, an important intermediate we also plan to overexpress.

</P><DIV style="display:inline-block"><FIGURE style="float: left; width: 45%;"><FIGCAPTION><B>Figure 1:</B>  Scheme of the working mechanism for the 3-HPA sensor
 	</FIGCAPTION></FIGURE>



	Using the information from the paper of 
	<A href="http://www.nature.com/articles/s41598-017-01850-w"><ABBR title="Characterisation of a 3-Hydroxypropionic Acid-Inducible System from Pseudomonas Putida for Orthogonal Gene Expression Control in Escherichia Coli and Cupriavidus Necator.">Hanko et al. (2017),</ABBR></A> 

    we settled for the
	<I>
		 hdpR
	</I>
	/
	<I>
		hdpH
	</I>
	 system coming from
	<I>
		Pseudomonas putida
	</I>
	 KT2440. It consists the constitutively expressed HdpR protein

	  <A href="http://parts.igem.org/Part:BBa_K2560304"><ABBR title="Link to the part in the iGEM registry">(BBa_K2560304).</ABBR></A>

    , which is able to bind to the P

	 
	<SUB>
		HdpR
	</SUB>
	and induce expression of a gene. In the native setting, that would be the
	<I>
		hdpH
	</I>
	 gene, encoding for a 3-hydroxypropionate dehydrogenase which enables
	<I>
		P. putida
	</I>
	 to grow on <DFN data-info="3-hydroxypropionic acid"> 3HPA </DFN>. We cloned our reporter gene in its position. This way we got a direct readout of the <DFN data-info="3-hydroxypropionic acid"> 3HPA </DFN>concentrations in the individual cells.
<P>
	Because our model showed that the low activity of
	
		<I>V. natriegens</I>
	
	 native <DFN data-info="acetyl-coA carboxylase"> Acc </DFN>, malonyl-CoA is likely to be a limiting factor in <DFN data-info="3-hydroxypropionic acid"> 3HPA </DFN> production. Having a direct indication to how much malonyl-CoA is available


<FIGURE style="float: right; width: 45%;"><FIGCAPTION><B>Figure 2:</B>  Scheme of the working mechanism for the malonyl-CoA sensor
 	</FIGCAPTION></FIGURE>

  and how our efforts to introduce heterologous Acc´s succeed in elevating the native levels would be quite helpful. This can be achieved in a quite similar way to the <DFN data-info="3-hydroxypropionic acid"> 3HPA </DFN> measurement. Our literature research quickly gave us several starting points. One of them, a system from
	<I>
		Bacillus subtilis
	</I>
	, in which the repressor FapR binds the 17bp motif
	<I>
		fapO,
	</I>
	 thereby sterically hindering polymerase binding to an adjacent promoter<A href="http://www.ncbi.nlm.nih.gov/pubmed/24377365"><ABBR title="Negative Feedback Regulation of Fatty Acid Production Based on a Malonyl-CoA Sensor-Actuator.">(Liu <I>et al.</I> 2015).</ABBR></A>  When the FapR protein binds malonyl-CoA, a conformational change is triggered which results in FapR dissociating from the
	<I>
		fapO
	</I>
	site. The now accessible promoter drives expression of the gene under its control, which we replaced with a reporter.
</P></DIV></DIV></DIV><DIV class="collapsible"><DIV class="btn_expand">Directed Evolution and Applications</DIV><DIV class="content"><H3>Directed evolution</H3>
	Although the Marburg Collection enables the construction of many pathway variations, there are still many more possibilities for direct metabolic fluxes towards <DFN data-info="3-hydroxypropionic acid"> 3HPA </DFN>. To enforce these flux improvements, we planned to use directed evolution coupled with FACS selection. 
	By implementing genes for the conversion of <DFN data-info="3-hydroxypropionic acid"> 3HPA </DFN> into succinate, we lead our product back to the central metabolism. By deleting all other genes responsible for succinate production, the cells are forced to use the <DFN data-info="3-hydroxypropionic acid"> 3HPA </DFN> bypass to get succinate, which is needed in the TCA cycle. 
	The big advantage of using <I>V. natriegens</I> for that purpose is that it’s short doubling time increases the number of cell divisions per time and thereby increases the number of mutations in a given time frame. 
	Coupled with our biosensors and fluorescence-activated cell sorting (FACS) we can easily and rapidly select for cells performing the most efficient pathways. 
	Additionally to mutation-guided evolution there are many genome modifications which could increase flow towards <DFN data-info="3-hydroxypropionic acid"> 3HPA </DFN>, for instance by downregulation of side reactions. In the <A href="https://2018.igem.org/Team:Marburg/Design"><ABBR title="Directed Evolution section of our metabolic engineering design">Directed Evolution section</ABBR></A> of our metabolic engineering design, a list of possible genome modifications is shown. 

	By using <DFN data-info="multiplex automated genome engineering"> MAGE </DFN>, it is possible to get a library of different genome modifications, which can then be selected via FACS for their efficiency to produce <DFN data-info="3-hydroxypropionic acid"> 3HPA </DFN> or succinate. <DFN data-info="multiplex automated genome engineering"> MAGE </DFN> stands for multiplex automated genome engineering and is a technique to create combinatorial genome modifications. 
	<DFN data-info="multiplex automated genome engineering"> MAGE </DFN> uses a set of oligos, which get transformed into a population of cells and then integrate into the chromosome via homologous recombination. 
	Different cells in the population will have different numbers and combinations of modifications leading to a genomic diversity. By applying <DFN data-info="multiplex automated genome engineering"> MAGE </DFN> in combination with FACS for our pathway we get a huge variety of possible pathway optimizations which can be sorted via FACS based on their efficiency.
<H3> Applications of our workflow</H3><P>
	By using <I>V. natriegens</I> as a chassis for metabolic engineering, it is possible to build, test and optimize pathways and enzymes at a high-throughput level. 
	The Marburg Collection enables the fast construction of pathway libraries, while the genetic accessibility of the strain makes genome modification and pathway integration simple. 
	The usage of our biosensors for screening of malonyl-CoA and <DFN data-info="3-hydroxypropionic acid"> 3HPA </DFN> accelerates identification of the best pathway variants and the short doubling time coupled with the high nutrient uptake rate empowers <I>V. natriegens</I> to be the perfect chassis for metabolic and enzyme engineering!
</P></DIV></DIV></DIV></DIV><DIV class="collapsible"><DIV class="btn_expand">Vibrary</DIV><DIV class="content">
  Aiyar, Sarah E, Tamas Gaal, and Richard L Gourse. 2002. RRNA Promoter Activity in the Fast-Growing Bacterium Vibrio Natriegens.
  <I>
     Journal of bacteriology
  </I>
   184(5): 1349-58. http://www.ncbi.nlm.nih.gov/pubmed/11844764 (August 23, 2018).

<P>
  Austin, Payne B, A Zachary, and R R Colwell. 2018. 28 International Association of Microbiological Societies
  <I>
     Recognition of Beneckea Natriegens (Payne et al.; Baumann  et al.) as a Member of the Genus Vibrio, as Previously Proposed by Webb and Payne
  </I>
   . www.microbFiologyresearch.org (August 23, 2018).
</P><P>
  Bi, Keran <I> et al. </I> 2016. Isolation and Molecular Identification of
  <I>
     Vibrio Natriegens
  </I>
   from Diseased
  <I>
     Portunus Trituberculatus
  </I>
   in China.
  <I>
     Journal of the World Aquaculture Society
  </I>
   47(6): 854-61. http://doi.wiley.com/10.1111/jwas.12305 (September 29, 2018).
</P><P>
  Carlson, Erik D, Rui Gan, C Eric Hodgman, and Michael C Jewett<DFN data-info="A fish is just a fish, and weather it is eaten depends much on the dish!">.</DFN> 2012. Cell-Free Protein Synthesis: Applications Come of Age.
  <I>
     Biotechnology advances
  </I>
   30(5): 1185–94. http://www.ncbi.nlm.nih.gov/pubmed/22008973 (October 1, 2018).
</P><P>
  CHIEN, C <I> et al. </I> 2007. “Production of Poly-β-Hydroxybutyrate (PHB) by Vibrio Spp. Isolated from Marine Environment.”<I>
     Journal of Biotechnology
  </I>
   132(3): 259–63. http://www.ncbi.nlm.nih.gov/pubmed/17416432 (September 22, 2018).
</P><P>
  Coyer, J A, A Cabello-Pasini, H Swift, and R S Alberte. 1996. “N2 Fixation in Marine Heterotrophic Bacteria: Dynamics of Environmental and Molecular Regulation.”<I>
     Proceedings of the National Academy of Sciences of the United States of America
  </I>
   93(8): 3575–80. http://www.ncbi.nlm.nih.gov/pubmed/11607653 (August 23, 2018).
</P><P>
  Dalia, Triana N. <I> et al. </I> 2017. “Multiplex Genome Editing by Natural Transformation (MuGENT) for Synthetic Biology in
  <I>
     Vibrio Natriegens
  </I>
   .”<I>
     ACS Synthetic Biology
  </I>
   6(9): 1650–55. http://www.ncbi.nlm.nih.gov/pubmed/28571309 (September 30, 2018).
</P><P>
  Delpech, Roger. 2001. “Using Vibrio Natriegetis for Studying Bacterial Population Growth, Artificial Selection, and the Effects of UV Radiation and Photo-Reactivation.”<I>
     Journal of Biological Education
  </I>
   35(2): 93–97. http://www.tandfonline.com/doi/abs/10.1080/00219266.2000.9655749 (August 23, 2018).
</P><P>
  EAGON, R G. 1962. “Pseudomonas Natriegens, a Marine Bacterium with a Generation Time of Less than 10 Minutes.”<I>
     Journal of bacteriology
  </I>
   83(4): 736–37. http://www.ncbi.nlm.nih.gov/pubmed/13888946 (August 23, 2018).
</P><P>
  Failmezger, Jurek, Steffen Scholz, Bastian Blombach, and Martin Siemann-Herzberg. 2018. “Cell-Free Protein Synthesis From Fast-Growing Vibrio Natriegens.”<I>
     Frontiers in Microbiology
  </I>
   9: 1146. https://www.frontiersin.org/article/10.3389/fmicb.2018.01146/full (September 30, 2018).
</P><P>
  Fernández-Llamosas, Helga <I> et al. </I> 2017. “Speeding up Bioproduction of Selenium Nanoparticles by Using Vibrio Natriegens as Microbial Factory.”<I>
     Scientific Reports
  </I>
   7(1): 16046. http://www.nature.com/articles/s41598-017-16252-1 (September 28, 2018).
</P><P>
  Hoffart, Eugenia <I> et al. </I> 2017. “High Substrate Uptake Rates Empower Vibrio Natriegens as Production Host for Industrial Biotechnology.”<I>
     Applied and environmental microbiology
  </I>
   : AEM.01614-17. http://www.ncbi.nlm.nih.gov/pubmed/28887417 (October 6, 2018).
</P><P>
  Lee, Henry H <I> et al. </I> 2016. “Vibrio Natriegens, a New Genomic Powerhouse.”<I>
     bioRxiv
  </I>
   : 058487. https://www.biorxiv.org/content/early/2016/06/12/058487 (September 30, 2018).
</P><P>
  Lee, Henry H, Nili Ostrov, Michaela A Gold, and George M Church. 2017. “Recombineering in Vibrio Natriegens.”<I>
     bioRxiv
  </I>
   : 130088. https://www.biorxiv.org/content/early/2017/04/26/130088 (September 30, 2018).
</P><P>
  Long, Christopher P., Jacqueline E. Gonzalez, Robert M. Cipolla, and Maciek R. Antoniewicz. 2017. “Metabolism of the Fast-Growing Bacterium Vibrio Natriegens Elucidated by 13C Metabolic Flux Analysis.”<I>
     Metabolic Engineering
  </I>
   44: 191–97. http://www.ncbi.nlm.nih.gov/pubmed/29042298 (October 5, 2018).
</P><P>
  Maida, I. <I> et al. </I> 2013. “Draft Genome Sequence of the Fast-Growing Bacterium Vibrio Natriegens Strain DSMZ 759.”<I>
     Genome Announcements
  </I>
   1(4). http://www.ncbi.nlm.nih.gov/pubmed/23969053 (August 23, 2018).
</P><P>
  Mehbub, Mohammad F. <I> et al. </I> 2018. “A Controlled Aquarium System and Approach to Study the Role of Sponge-Bacteria Interactions Using Aplysilla Rosea and Vibrio Natriegens.”<I>
     Scientific Reports
  </I>
   8(1): 11801. http://www.nature.com/articles/s41598-018-30295-y (September 29, 2018).
</P><P>
  Miura, A <I> et al. </I> 1981. “Growth-Rate-Dependent Regulation of Ribosome Synthesis in <I>E. coli</I>: Expression of the LacZ and GalK Genes Fused to Ribosomal Promoters.”<I>
     Cell
  </I>
   25(3): 773–82. http://www.ncbi.nlm.nih.gov/pubmed/6793240 (September 22, 2018).
</P><P>
  Mullenger, L., and Nijole R. Gill. 1973. “Vibrio Natriegens: A Rapidly Growing Micro-Organism Ideally Suited for Class Experiments.”<I>
     Journal of Biological Education
  </I>
   7(5): 33–39. http://www.tandfonline.com/doi/abs/10.1080/00219266.1973.9653881 (August 21, 2018).
</P><P>
  Nazly, E D. 1980. 116 Journal of General Microbiology
  <I>
     Adenine Nucleotide Pools During Starvation of Beneckea Natviegens
  </I>
   . www.microbiologyresearch.org (October 7, 2018).
</P><P>
  Nissen, Hilde, Mikal Heldal, and Svein Norland. 1987. “Growth, Elemental Composition, and Formation of Polyphosphate Bodies in
  <I>
     Vibrio Natriegens
  </I>
   Cultures Shifted from Phosphate-Limited to Phosphate-Pulsed Media.”<I>
     Canadian Journal of Microbiology
  </I>
   33(7): 583–88. http://www.nrcresearchpress.com/doi/10.1139/m87-101 (September 22, 2018).
</P><P>
  Payne, W J, R G Eagon, and A K Williams. 1960.
  <I>
     SOME OBSERVATIONS ON THE PHYSIOLOGY OF PSEUDOMONAS NATRIEGENS NOV. SPEC. 1)
  </I>
   . https://link.springer.com/content/pdf/10.1007/BF02538432.pdf (August 21, 2018).
</P><P>
  Payne, William J.
  <I>
     STUDIES ON BACTERIAL UTILIZATION OF URONIC ACIDS III. INDUCTION OF OXIDATIVE ENZYMES IN A MARINE ISOLATE’</I>
   . https://www.ncbi.nlm.nih.gov/pmc/articles/PMC290205/pdf/jbacter00506-0093.pdf (August 21, 2018).
</P><P>
  Pollack-Berti, Amber, Michael S. Wollenberg, and Edward G. Ruby. 2010. “Natural Transformation of Vibrio Fischeri Requires TfoX and TfoY.”<I>
     Environmental Microbiology
  </I>
   12(8): no-no. http://doi.wiley.com/10.1111/j.1462-2920.2010.02250.x (September 30, 2018).
</P><P>
  Simons, Keryn L., Susan M. Thomas, and Peter A. Anderson. 2010. “Identification of Vibrio Natriegens UvrA and UvrB Genes and Analysis of Gene Regulation Using Transcriptional Reporter Plasmids.”<I>
     The Journal of Microbiology
  </I>
   48(5): 644–56. http://www.ncbi.nlm.nih.gov/pubmed/21046343 (September 22, 2018).
</P><P>
  Des Soye, Benjamin J. <I> et al. </I> 2018. “Establishing a High-Yielding Cell-Free Protein Synthesis Platform Derived from
  <I>
     Vibrio Natriegens
  </I>
   .”<I>
     ACS Synthetic Biology
  </I>
   7(9): 2245–55. http://www.ncbi.nlm.nih.gov/pubmed/30107122 (September 30, 2018).
</P><P>
  Webb, C D, and W J Payne. 1971. “Influence of Na+ on Synthesis of Macromolecules by a Marine Bacterium.”<I>
     Applied microbiology
  </I>
   21(6): 1080–88. http://www.ncbi.nlm.nih.gov/pubmed/4327612 (August 23, 2018).
</P><P>
  Weinstock, Matthew T, Eric D Hesek, Christopher M Wilson, and Daniel G Gibson. 2016. “Vibrio Natriegens as a Fast-Growing Host for Molecular Biology.”<I>
     Nature Methods
  </I>
   13(10): 849–51. http://www.ncbi.nlm.nih.gov/pubmed/27571549 (September 29, 2018).
</P><P>
  Wiegand, Daniel J., Henry H. Lee, Nili Ostrov, and George M. Church. 2018. “Establishing a Cell-Free
  <I>
     Vibrio Natriegens
  </I>
   Expression System.”<I>
     ACS Synthetic Biology
  </I>
   : acssynbio.8b00222. http://pubs.acs.org/doi/10.1021/acssynbio.8b00222 (September 30, 2018).
</P><P>
  Zachary, A. 1976. “Physiology and Ecology of Bacteriophages of the Marine Bacterium Beneckea Natriegens: Salinity.”<I>
     Applied and environmental microbiology
  </I>
   31(3): 415–22. http://www.ncbi.nlm.nih.gov/pubmed/938035 (August 23, 2018).
</P><P>
  Zachary, Arthur. 1974. 27 APPLIED MICROBIOLOGY
  <I>
     NOTES Isolation of Bacteriophages of the Marine Bacterium Beneckea Natriegens from Coastal Salt Marshes’</I>
   . https://www.ncbi.nlm.nih.gov/pmc/articles/PMC380186/pdf/applmicro00040-0188.pdf (September 28, 2018).
</P><P>
  Zacheray, A. 1978. An Ecological Study of Bacteriophages of
  <I>
     Vibrio Natriegens
  </I>
   .
  <I>
     Canadian Journal of Microbiology
  </I>
   24(3): 321-24. http://www.nrcresearchpress.com/doi/10.1139/m78-053 (September 23, 2018).
</P><P>
	Brazelton, VA
	<I>
		et al.
	</I>
	 2016. A quick guide to CRISPR sgRNA design tools.
	<I>
		GM Crops Food
	</I>
	. 2015; 6(4): 266–276. https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5033207/pdf/kgmc-06-04-1137690.pdf
</P><P>
	Nishimasu, H., Ran, F. A., Hsu, P. D., Konermann, S., Shehata, S., Dohmae, N., Ishitani, R., Zhang, F., Nureki, O. (2014) Crystal Structure of Cas9 in Complex with Guide RNA and Target DNA,
	<I>
		Cell
	</I>
	,           156 (5), 935-949. https://www.cell.com/action/showPdf?pii=S0092-8674%2814%2900156-1
</P><P>
	Jinek, M., Jiang, F., Taylor, D. W., Sternberg, S. H., Kaya, E., Ma, E., Anders, C., Hauer, M., Zhou, K., Lin, S., Kaplan, M., Iavarone, A. T., Charpentier, E., Nogale, E., Doudna, J. A. (2014) Structures of Cas9 Endonucleases Reveal RNA-Mediated Conformational Activation,
	<I>
		Science
	</I>
	, 343 (6176), 1-28. http://science.sciencemag.org/content/343/6176/1247997
</P><P>
	Deltcheva, E., Chylinski, K., Sharma, C. M., Gonzales, K., Chao, Y., Pirzada, Z. A., Eckert, M. R., Vogel, J., Charpentier, E. (2011) CRISPR RNA maturation by
	<I>
		trans
	</I>
	-encoded small RNA and host factor RNase III,
	<I>
		Nature
	</I>
	, 471, 602-609. https://www.nature.com/articles/nature09886
</P><P> </P>
 
	
		Gasiunas, R. Barrangou, P. Horvath, V. Siksnys. 2012. Cas9-crRNA ribonucleoprotein complex mediates specific DNA cleavage for adaptive immunity in bacteria.
	

<P><I>
		Proc. Natl. Acad. Sci. USA
	</I>
	, 109 (2012), pp. E2579-E2586. http://www.pnas.org/content/pnas/109/39/E2579.full.pdf
</P>

	
		Jinek, K. Chylinski, I. Fonfara, M. Hauer, J.A. Doudna, E. Charpentier. 2012. A programmable dual-RNA-guided DNA endonuclease in adaptive bacterial immunity.
		<I>
			Science
		</I>
		, 337 (2012), pp. 816-821. http://science.sciencemag.org/content/337/6096/816
	

<P>
	Doudna, J. A. &amp; Charpentier, E. 2014.Genome editing. The new frontier of genome engineering with CRISPR-Cas9
	<I>
		.
	</I><I>
		Science
	</I>
	 346, 1258096 (2014). http://science.sciencemag.org/content/346/6213/1258096.full
</P><P>
	Charpentier, E. &amp; Marraffini, L. A. 2014. Harnessing CRISPR-Cas9 immunity for genetic engineering
	<I>
		.
	</I><I>
		Curr. Opin. Microbiol.
	</I>
	 19, 114–119 (2014). https://www.sciencedirect.com/science/article/pii/S1369527414000885?via%3Dihub
</P><P>
	Hsu, P. D., Lander, E. S., Zhang, F. (2014) Development and Applications of CRISPR-Cas9 for Genome Engineering,
	<I>
		Cell
	</I>
	, 157, 1262-1276. https://www.cell.com/action/showPdf?pii=S0092-8674%2814%2900604-7
</P><P>
	Mazhar Adli. 2018. The CRISPR tool kit for genome editing and beyond.
	<I>
		Nature Communications
	</I>
	volume 9, Article number: 1911 (2018) https://www.nature.com/articles/s41467-018-04252-2.pdf
</P><P>
	Jansen, R., van Embden, J. D. A., Gaastra, W., Schouls, L. M. (2002) Identification of genes that are associated with DNA repeats in prokaryotes,
	<I>
		Molecular Microbiology
	</I>
	, 43 (6), 1565-1575. https://onlinelibrary.wiley.com/doi/epdf/10.1046/j.1365-2958.2002.02839.x
</P><P>
	Ishino, Y., Shinagawa, H., Makino, K., Amemura, M., Nakata, A. (1987) Nucleotide Sequence of the
	<I>
		iap
	</I>
	 Gene, Responsible for Alkaline Phosphatase Isozyme Conversion in
	<I>
		Escherichia coli
	</I>
	, and Identification of the Gene Product,
	<I>
		Journal of Bacteriology
	</I>
	, 169 (12), 5429-5433. https://www.ncbi.nlm.nih.gov/pmc/articles/PMC213968/pdf/jbacter00202-0107.pdf
</P><P>
	Qi
	<I>
		et al.
	</I>
	 2013. Repurposing CRISPR as an RNA-Guided Platform for Sequence-Specific Control of Gene Expression.
	<I>
		Cell.
	</I>
	 2013 Feb 28; 152(5): 1173–1183. https://www.cell.com/action/showPdf?pii=S0092-8674%2813%2900211-0
</P><P>
	 Barrangou, R., Fremaux, C., Deveau, H., Richards, M., Boyaval, P., Moineau, S., Romero, D. A., Horvath, P. (2007) CRISPR Provides Acquired Resistance Against Viruses in Prokaryotes,
	<I>
		Science
	</I>
	, 315, 1709-1712. http://science.sciencemag.org/content/315/5819/1709
</P><P>
	Brazelton, VA
	<I>
		et al.
	</I>
	 2016. A quick guide to CRISPR sgRNA design tools.
	<I>
		GM Crops Food
	</I>
	. 2015; 6(4): 266–276. https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5033207/pdf/kgmc-06-04-1137690.pdf
</P><P>
	Nishimasu, H., Ran, F. A., Hsu, P. D., Konermann, S., Shehata, S., Dohmae, N., Ishitani, R., Zhang, F., Nureki, O. (2014) Crystal Structure of Cas9 in Complex with Guide RNA and Target DNA,
	<I>
		Cell
	</I>
	,           156 (5), 935-949. https://www.cell.com/action/showPdf?pii=S0092-8674%2814%2900156-1
</P><P>
	Jinek, M., Jiang, F., Taylor, D. W., Sternberg, S. H., Kaya, E., Ma, E., Anders, C., Hauer, M., Zhou, K., Lin, S., Kaplan, M., Iavarone, A. T., Charpentier, E., Nogale, E., Doudna, J. A. (2014) Structures of Cas9 Endonucleases Reveal RNA-Mediated Conformational Activation,
	<I>
		Science
	</I>
	, 343 (6176), 1-28. http://science.sciencemag.org/content/343/6176/1247997
</P><P>
	Deltcheva, E., Chylinski, K., Sharma, C. M., Gonzales, K., Chao, Y., Pirzada, Z. A., Eckert, M. R., Vogel, J., Charpentier, E. (2011) CRISPR RNA maturation by
	<I>
		trans
	</I>
	-encoded small RNA and host factor RNase III,
	<I>
		Nature
	</I>
	, 471, 602-609. https://www.nature.com/articles/nature09886
</P><P><I>
		Proc. Natl. Acad. Sci. USA
	</I>
	, 109 (2012), pp. E2579-E2586. http://www.pnas.org/content/pnas/109/39/E2579.full.pdf
</P>

	
		Jinek, K. Chylinski, I. Fonfara, M. Hauer, J.A. Doudna, E. Charpentier. 2012. A programmable dual-RNA-guided DNA endonuclease in adaptive bacterial immunity.
		<I>
			Science
		</I>
		, 337 (2012), pp. 816-821. http://science.sciencemag.org/content/337/6096/816
	
<P>
	Doudna, J. A. &amp; Charpentier, E. 2014.Genome editing. The new frontier of genome engineering with CRISPR-Cas9
	<I>
		.
	</I><I>
		Science
	</I>346, 1258096 (2014). http://science.sciencemag.org/content/346/6213/1258096.full
</P><P>
	Charpentier, E. &amp; Marraffini, L. A. 2014. Harnessing CRISPR-Cas9 immunity for genetic engineering
	<I>
		.
	</I><I>
		Curr. Opin. Microbiol.
	</I>
	 19, 114–119 (2014). https://www.sciencedirect.com/science/article/pii/S1369527414000885?via%3Dihub
</P><P>
	Hsu, P. D., Lander, E. S., Zhang, F. (2014) Development and Applications of CRISPR-Cas9 for Genome Engineering,
	<I>
		Cell
	</I>
	, 157, 1262-1276. https://www.cell.com/action/showPdf?pii=S0092-8674%2814%2900604-7
</P><P>
	Mazhar Adli. 2018. The CRISPR tool kit for genome editing and beyond.
	<I>
		Nature Communications
	</I>
	volume 9, Article number: 1911 (2018) https://www.nature.com/articles/s41467-018-04252-2.pdf
</P><P>
	Jansen, R., van Embden, J. D. A., Gaastra, W., Schouls, L. M. (2002) Identification of genes that are associated with DNA repeats in prokaryotes,
	<I>
		Molecular Microbiology
	</I>
	, 43 (6), 1565-1575. https://onlinelibrary.wiley.com/doi/epdf/10.1046/j.1365-2958.2002.02839.x
</P><P>
	Ishino, Y., Shinagawa, H., Makino, K., Amemura, M., Nakata, A. (1987) Nucleotide Sequence of the
	<I>
		iap
	</I>
	 Gene, Responsible for Alkaline Phosphatase Isozyme Conversion in
	<I>
		Escherichia coli
	</I>
	, and Identification of the Gene Product,
	<I>
		Journal of Bacteriology
	</I>
	, 169 (12), 5429-5433. https://www.ncbi.nlm.nih.gov/pmc/articles/PMC213968/pdf/jbacter00202-0107.pdf
</P><P>
	Qi
	<I>
		et al.
	</I>
	 2013. Repurposing CRISPR as an RNA-Guided Platform for Sequence-Specific Control of Gene Expression.
	<I>
		Cell.
	</I>
	 2013 Feb 28; 152(5): 1173–1183. https://www.cell.com/action/showPdf?pii=S0092-8674%2813%2900211-0
</P><P>
	 Barrangou, R., Fremaux, C., Deveau, H., Richards, M., Boyaval, P., Moineau, S., Romero, D. A., Horvath, P. (2007) CRISPR Provides Acquired Resistance Against Viruses in Prokaryotes,
	<I>
		Science
	</I>
	, 315, 1709-1712. http://science.sciencemag.org/content/315/5819/1709
</P><P>

Alber, B.E., Fuchs, G., 2002. Propionyl-coenzyme a synthase from Chloroflexus aurantiacus, a key enzyme of the 3-hydroxypropionate cycle for autotrophic CO2 fixation. J. Biol. Chem. 277, 12137–12143. 


Gande, R., Dover, L.G., Krumbach, K., Besra, G.S., Sahm, H., Oikawa, T., Eggeling, L., 2007. The two carboxylases of corynebacterium glutamicum essential for fatty acid and mycolic acid synthesis. J. Bacteriol. 189, 5257–5264. 


Hanko, E.K.R., Minton, N.P., Malys, N., 2017. Characterisation of a 3-hydroxypropionic acid-inducible system from Pseudomonas putida for orthogonal gene expression control in Escherichia coli and Cupriavidus necator. Sci. Rep. 7, 1724. 


Janßen, H.J., Steinbüchel, A., 2014. Fatty acid synthesis in Escherichia coli and its applications towards the production of fatty acid based biofuels. Biotechnol Biofuels 7, 7. 


Kildegaard, K.R., Jensen, N.B., Schneider, K., Czarnotta, E., Özdemir, E., Klein, T., Maury, J., Ebert, B.E., Christensen, H.B., Chen, Y., Kim, I.K., Herrgård, M.J., Blank, L.M., Forster, J., Nielsen, J., Borodina, I., 2016. Engineering and systems-level analysis of Saccharomyces cerevisiae for production of 3-hydroxypropionic acid via malonyl-CoA reductase-dependent pathway. Microb. Cell Fact. 15. 


Lee, C.-K., Cheong, H.-K., Ryu, K.-S., Lee, J. Il, Lee, W., Jeon, Y.H., Cheong, C., 2008. Biotinoyl domain of human acetyl-CoA carboxylase: Structural insights into the carboxyl transfer mechanism. Proteins Struct. Funct. Bioinforma. 72, 613–624. 


Liu, C., Ding, Y., Xian, M., Liu, M., Liu, H., Ma, Q., Zhao, G., 2017. Malonyl-CoA pathway: a promising route for 3-hydroxypropionate biosynthesis. Crit. Rev. Biotechnol. 37, 933–941. 


Liu, C., Wang, Q., Xian, M., Ding, Y., Zhao, G., 2013. Dissection of Malonyl-Coenzyme A Reductase of Chloroflexus aurantiacus Results in Enzyme Activity Improvement. PLoS One 8, 1–8. 


Raj, S.M., Rathnasingh, C., Jo, J.E., Park, S., 2008. Production of 3-hydroxypropionic acid from glycerol by a novel recombinant Escherichia coli BL21 strain. Process Biochem. 43, 1440–1446. 


Rathnasingh, C., Raj, S.M., Lee, Y., Catherine, C., Ashok, S., Park, S., 2012. Production of 3-hydroxypropionic acid via malonyl-CoA pathway using recombinant Escherichia coli strains. J. Biotechnol. 157, 633–640. 



Ro, D.K., Paradise, E.M., Quellet, M., Fisher, K.J., Newman, K.L., Ndungu, J.M., Ho, K.A., Eachus, R.A., Ham, T.S., Kirby, J., Chang, M.C.Y., Withers, S.T., Shiba, Y., Sarpong, R., Keasling, J.D., 2006. Production of the antimalarial drug precursor artemisinic acid in engineered yeast. Nature 440, 940–943. 


Stephanopoulos, G., 2012. Synthetic biology and metabolic engineering. ACS Synth. Biol. 1, 514–525. 


Stephanopoulos, G., 2007. Challenges in engineering microbes for biofuels production. Science 315, 801–4. 


Valdehuesa, K.N.G., Liu, H., Nisola, G.M., Chung, W.J., Lee, S.H., Park, S.J., 2013. Recent advances in the metabolic engineering of microorganisms for the production of 3-hydroxypropionic acid as C3 platform chemical. Appl. Microbiol. Biotechnol. 97, 3309–3321. 


Vidra, A., Németh, Á., 2017. Bio-based 3-hydroxypropionic Acid : A Review. period. Polytech. Chem. Eng. 1–11.
Werpy, T., Petersen, G., 2004. Top Value Added Chemicals from Biomass Volume I — Results of Screening for Potential Candidates from Sugars and Synthesis Gas Top Value Added Chemicals From Biomass Volume I : Results of Screening for Potential Candidates. Other Inf. PBD 1 Aug 2004 Medium: ED; Size: 76 pp. pages. 
Antonova ES, Hammer BK. Genetics of Natural Competence in Vibrio cholerae and other Vibrios. Microbiol Spectr. (2015);3(3). 

Argos, P., A. Landy, K. Abremski, J. B. Egan, E. Haggard-Ljungquist, R. H. Hoess, M. L. Kahn, B. Kalionis, S. V. Narayana, L. S. Pierson, 3rd and et al. (1986). &quot;The integrase family of site-specific recombinases: regional similarities and global diversity.&quot; EMBO J 5(2): 433-440.

Blokesch, M. (2012). &quot;TransFLP--a method to genetically modify Vibrio cholerae based on natural transformation and FLP-recombination.&quot; J Vis Exp(68).

Blokesch, M., Schoolnik, G. K., The Extracellular Nuclease Dns and Its Role in Natural Transformation of Vibrio cholerae, JOURNAL OF BACTERIOLOGY, (2008), 7232–7240.

Broach, J. R., V. R. Guarascio and M. Jayaram (1982). &quot;Recombination within the yeast plasmid 2mu circle is site-specific.&quot; Cell 29(1): 227-234.

Bryan, FR., Construction of a recombinase-deficient mutant recA protein that retains single-stranded DNA-dependent ATPase activity, J Biol Chem. (1988); 263(18):8716-23.
Burn, S.F., Detection of β-galactosidase activity: X-gal staining. Methods Mol Biol. 2012;886:241-50.

Chen, Z., Yang, H., Pavletich N. P., Mechanism of homologous recombination from the RecA–ssDNA/dsDNA structures, Nature (2008) 22;453(7194):489-4.
Cheetham, G. M. &amp; Steitz, T. A. Structure of a transcribing T7 RNA polymerase initiation complex. Science (New York, N.Y.) 286, 2305–2309 (1999).

Chung, C. H. and A. L. Goldberg (1981). &quot;The product of the lon (capR) gene in Escherichia coli is the ATP-dependent protease, protease La.&quot; Proc Natl Acad Sci U S A 78(8): 4931-4935.

Dalia, T. N., C. A. Hayes, S. Stolyar, C. J. Marx, J. B. McKinlay and A. B. Dalia (2017). &quot;Multiplex Genome Editing by Natural Transformation (MuGENT) for Synthetic Biology in Vibrio natriegens.&quot; ACS Synth Biol 6(9): 1650-1655.

Donnenberg, M. S. and J. B. Kaper (1991). &quot;Construction of an eae deletion mutant of enteropathogenic Escherichia coli by using a positive-selection suicide vector.&quot; Infect Immun 59(12): 4310-4317.

Evans, B. R., J. W. Chen, R. L. Parsons, T. K. Bauer, D. B. Teplow and M. Jayaram (1990). &quot;Identification of the active site tyrosine of Flp recombinase. Possible relevance of its location to the mechanism of recombination.&quot; J Biol Chem 265(30): 18504-18510.

Fuchs, G., Allgemeine Mikrobiologie, Thieme. (2017).
Fields, S. &amp; O. Song, (1989) A novel genetic system to detect protein-protein interactions. Nature 340: 245-246.https://www.ncbi.nlm.nih.gov/pubmed/2547163 

Karimova, G., J. Pidoux, A. Ullmann &amp; D. Ladant, (1998) A bacterial two-hybrid system based on a reconstituted signal transduction pathway. Proceedings of the National Academy of Sciences of the United States of America 95: 5752-5756.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC20451/

Kolb, A., S. Busby, H. Buc, S. Garges &amp; S. Adhya, (1993) Transcriptional regulation by cAMP and its receptor protein. Annual review of biochemistry 62: 749-795.
https://www.ncbi.nlm.nih.gov/pubmed/8394684

Krupinski, J., (1991) The adenylyl cyclase family. Molecular and cellular biochemistry 104: 73-79. https://link.springer.com/article/10.1007/BF00229806

Griffith F, The significance of pneumococcal types. Journal of Hygiene. 27, Nr. 2. (1928) S. 113–59.

Gill, R. T., M. P. DeLisa, M. Shiloach, T. R. Holoman and W. E. Bentley (2000). &quot;OmpT expression and activity increase in response to recombinant chloramphenicol acetyltransferase overexpression and heat shock in E. coli.&quot; J Mol Microbiol Biotechnol 2(3): 283-289.

Grodberg, J. and J. J. Dunn (1988). &quot;ompT encodes the Escherichia coli outer membrane protease that cleaves T7 RNA polymerase during purification.&quot; J Bacteriol 170(3): 1245-1253.

Horii T.; Ogawa T.; Nakatani T.; Hase T.; Matsubara H. &amp; Ogawa H., Regulation of SOS functions: Purification of E. coli LexA protein and determination of its specific site cleaved by the RecA protein, cell (1981)  Vol. 27, 515-522.
Hartmann, G., Honikel, K. O., Knüsel, F. &amp; Nüesch, J. The specific inhibition of the DNA-directed RNA synthesis by rifamycin. Biochimica et biophysica acta 145, 843–844 (1967).

Howard-Flanders, P., E. Simson and L. Theriot (1964). &quot;A Locus That Controls Filament Formation and Sensitivity to Radiation in Escherichia Coli K-12.&quot; Genetics 49: 237-246.

Huisman, O., R. D'Ari and S. Gottesman (1984). &quot;Cell-division control in Escherichia coli: specific induction of the SOS function SfiA protein is sufficient to block septation.&quot; Proc Natl Acad Sci U S A 81(14): 4490-4494.

Jeong, H., Kim, H. J. &amp; Lee, S. J. Complete Genome Sequence of Escherichia coli Strain BL21. Genome announcements 3 (2015).

Lorenz, M. G., and Wackernagel, W. Bacterial gene transfer by natural genetic transformation in the environment. Microbiol Rev. (1994) 58, 563−602.

Langley, KE., Villarejo, M.R., Fowler, A.V., Zamenhof, P.J., and Zabin, I, Molecular basis of beta-galactosidase alpha-complementation. Proc Natl Acad Sci U S A. (1975);72(4):1254-7.
Leytus, S. P., L. K. Bowles, J. Konisky and W. F. Mangel (1981). &quot;Activation of plasminogen to plasmin by a protease associated with the outer membrane of Escherichia coli.&quot; Proc Natl Acad Sci U S A 78(3): 1485-1489.

Lundrigan, M. D. and R. M. Webb (1992). &quot;Prevalence of ompT among Escherichia coli isolates of human origin.&quot; FEMS Microbiol Lett 76(1-2): 51-56.

Meibom, K. L., M. Blokesch, N. A. Dolganov, C. Y. Wu and G. K. Schoolnik (2005). &quot;Chitin induces natural competence in Vibrio cholerae.&quot; Science 310(5755): 1824-1827.

Mell JC, Redfield RJ. Natural competence and the evolution of DNA uptake specificity. J Bacteriol. (2014);196(8):1471-83.
Mizusawa, S. and S. Gottesman (1983). &quot;Protein degradation in Escherichia coli: the lon gene controls the stability of sulA protein.&quot; Proc Natl Acad Sci U S A 80(2): 358-362.

Nishino, T., Morikawa, T., Structure and function of nucleases in DNA repair: shape, grip and blade of the DNA scissors, Oncogene (2002) 21, 9022 – 9032.

Pan, G. and P. D. Sadowski (1992). &quot;Ligation activity of FLP recombinase. The strand ligation activity of a site-specific recombinase using an activated DNA substrate.&quot; J Biol Chem 267(18): 12397-12399.

Pollack-Berti, A., M. S. Wollenberg and E. G. Ruby (2010). &quot;Natural transformation of Vibrio fischeri requires tfoX and tfoY.&quot; Environ Microbiol 12(8): 2302-2311.

Sadowski, P. D. (1995). &quot;The Flp recombinase of the 2-microns plasmid of Saccharomyces cerevisiae.&quot; Prog Nucleic Acid Res Mol Biol 51: 53-91.

Susan T. Lovett, The DNA exonucleases of Escherichia coli, (2003) 4(2).
Savir, Y.,Tlusty, T., RecA-mediated homology search as a nearly optimal signal detection system, Molecular Cell (2010) 40, 388–396.
Santillán M, Mackey MC., Quantitative approaches to the study of bistability in the lac operon of Escherichia coli., J R Soc Interface. (2008) 6;5 Suppl 1:S29-39.
Serebriiskii I.G., Golemis E.A., Uses of lacZ to study gene function: evaluation of β-galactosidase assays employed in the yeast two-hybrid system. Anal. Biochem. 2000;285:1–15.
Studier, F. W. &amp; Moffatt, B. A. Use of bacteriophage T7 RNA polymerase to direct selective high-level expression of cloned genes. Journal of molecular biology 189, 113–130 (1986).

Sodeinde, O. A., Y. V. Subrahmanyam, K. Stark, T. Quan, Y. Bao and J. D. Goguen (1992). &quot;A surface protease and the invasive character of plague.&quot; Science 258(5084): 1004-1007.

Tabor, S. &amp; Richardson, C. C. A bacteriophage T7 RNA polymerase/promoter system for controlled exclusive expression of specific genes. Proceedings of the National Academy of Sciences of the United States of America 82, 1074–1078 (1985).

Villaverde, A. and M. M. Carrio (2003). &quot;Protein aggregation in recombinant bacteria: biological role of inclusion bodies.&quot; Biotechnol Lett 25(17): 1385-1395.

Wu R., Zhao M., Li J., Gao H., Kan B., Liang W. Direct regulation of the natural competence regulator gene tfoX by cyclic AMP (cAMP) and cAMP receptor protein (CRP) in Vibrios. Sci Rep. (2015) 7;5:14921.


Weinstock, M. T., E. D. Hesek, C. M. Wilson and D. G. Gibson (2016). &quot;Vibrio natriegens as a fast-growing host for molecular biology.&quot; Nat Methods 13(10): 849-851.

Zamenhof, P.J., Villarejo, M. Construction and Properties of E. coli Strains Exhibiting α-Complementation of β-Galactosidase Fragments In Vivo. J Bacteriol. (1972); 110(1): 171–178.

Antonova ES, Hammer BK. Genetics of Natural Competence in Vibrio cholerae and other Vibrios. Microbiol Spectr. (2015);3(3). 

Argos, P., A. Landy, K. Abremski, J. B. Egan, E. Haggard-Ljungquist, R. H. Hoess, M. L. Kahn, B. Kalionis, S. V. Narayana, L. S. Pierson, 3rd and et al. (1986). &quot;The integrase family of site-specific recombinases: regional similarities and global diversity.&quot; EMBO J 5(2): 433-440.

Blokesch, M. (2012). &quot;TransFLP--a method to genetically modify Vibrio cholerae based on natural transformation and FLP-recombination.&quot; J Vis Exp(68).

Blokesch, M., Schoolnik, G. K., The Extracellular Nuclease Dns and Its Role in Natural Transformation of Vibrio cholerae, JOURNAL OF BACTERIOLOGY, (2008), 7232–7240.

Broach, J. R., V. R. Guarascio and M. Jayaram (1982). &quot;Recombination within the yeast plasmid 2mu circle is site-specific.&quot; Cell 29(1): 227-234.

Bryan, FR., Construction of a recombinase-deficient mutant recA protein that retains single-stranded DNA-dependent ATPase activity, J Biol Chem. (1988); 263(18):8716-23.
Burn, S.F., Detection of β-galactosidase activity: X-gal staining. Methods Mol Biol. 2012;886:241-50.

Chen, Z., Yang, H., Pavletich N. P., Mechanism of homologous recombination from the RecA–ssDNA/dsDNA structures, Nature (2008) 22;453(7194):489-4.
Cheetham, G. M. &amp; Steitz, T. A. Structure of a transcribing T7 RNA polymerase initiation complex. Science (New York, N.Y.) 286, 2305–2309 (1999).

Chung, C. H. and A. L. Goldberg (1981). &quot;The product of the lon (capR) gene in Escherichia coli is the ATP-dependent protease, protease La.&quot; Proc Natl Acad Sci U S A 78(8): 4931-4935.

Dalia, T. N., C. A. Hayes, S. Stolyar, C. J. Marx, J. B. McKinlay and A. B. Dalia (2017). &quot;Multiplex Genome Editing by Natural Transformation (MuGENT) for Synthetic Biology in Vibrio natriegens.&quot; ACS Synth Biol 6(9): 1650-1655.

Donnenberg, M. S. and J. B. Kaper (1991). &quot;Construction of an eae deletion mutant of enteropathogenic Escherichia coli by using a positive-selection suicide vector.&quot; Infect Immun 59(12): 4310-4317.

Evans, B. R., J. W. Chen, R. L. Parsons, T. K. Bauer, D. B. Teplow and M. Jayaram (1990). &quot;Identification of the active site tyrosine of Flp recombinase. Possible relevance of its location to the mechanism of recombination.&quot; J Biol Chem 265(30): 18504-18510.

Fuchs, G., Allgemeine Mikrobiologie, Thieme. (2017).
Fields, S. &amp; O. Song, (1989) A novel genetic system to detect protein-protein interactions. Nature 340: 245-246.https://www.ncbi.nlm.nih.gov/pubmed/2547163 

Karimova, G., J. Pidoux, A. Ullmann &amp; D. Ladant, (1998) A bacterial two-hybrid system based on a reconstituted signal transduction pathway. Proceedings of the National Academy of Sciences of the United States of America 95: 5752-5756.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC20451/

Kolb, A., S. Busby, H. Buc, S. Garges &amp; S. Adhya, (1993) Transcriptional regulation by cAMP and its receptor protein. Annual review of biochemistry 62: 749-795.
https://www.ncbi.nlm.nih.gov/pubmed/8394684

Krupinski, J., (1991) The adenylyl cyclase family. Molecular and cellular biochemistry 104: 73-79. https://link.springer.com/article/10.1007/BF00229806

Griffith F, The significance of pneumococcal types. Journal of Hygiene. 27, Nr. 2. (1928) S. 113–59.

Gill, R. T., M. P. DeLisa, M. Shiloach, T. R. Holoman and W. E. Bentley (2000). &quot;OmpT expression and activity increase in response to recombinant chloramphenicol acetyltransferase overexpression and heat shock in E. coli.&quot; J Mol Microbiol Biotechnol 2(3): 283-289.

Grodberg, J. and J. J. Dunn (1988). &quot;ompT encodes the Escherichia coli outer membrane protease that cleaves T7 RNA polymerase during purification.&quot; J Bacteriol 170(3): 1245-1253.

Horii T.; Ogawa T.; Nakatani T.; Hase T.; Matsubara H. &amp; Ogawa H., Regulation of SOS functions: Purification of E. coli LexA protein and determination of its specific site cleaved by the RecA protein, cell (1981)  Vol. 27, 515-522.
Hartmann, G., Honikel, K. O., Knüsel, F. &amp; Nüesch, J. The specific inhibition of the DNA-directed RNA synthesis by rifamycin. Biochimica et biophysica acta 145, 843–844 (1967).

Howard-Flanders, P., E. Simson and L. Theriot (1964). &quot;A Locus That Controls Filament Formation and Sensitivity to Radiation in Escherichia Coli K-12.&quot; Genetics 49: 237-246.

Huisman, O., R. D'Ari and S. Gottesman (1984). &quot;Cell-division control in Escherichia coli: specific induction of the SOS function SfiA protein is sufficient to block septation.&quot; Proc Natl Acad Sci U S A 81(14): 4490-4494.

Jeong, H., Kim, H. J. &amp; Lee, S. J. Complete Genome Sequence of Escherichia coli Strain BL21. Genome announcements 3 (2015).

Lorenz, M. G., and Wackernagel, W. Bacterial gene transfer by natural genetic transformation in the environment. Microbiol Rev. (1994) 58, 563−602.

Langley, KE., Villarejo, M.R., Fowler, A.V., Zamenhof, P.J., and Zabin, I, Molecular basis of beta-galactosidase alpha-complementation. Proc Natl Acad Sci U S A. (1975);72(4):1254-7.
Leytus, S. P., L. K. Bowles, J. Konisky and W. F. Mangel (1981). &quot;Activation of plasminogen to plasmin by a protease associated with the outer membrane of Escherichia coli.&quot; Proc Natl Acad Sci U S A 78(3): 1485-1489.

Lundrigan, M. D. and R. M. Webb (1992). &quot;Prevalence of ompT among Escherichia coli isolates of human origin.&quot; FEMS Microbiol Lett 76(1-2): 51-56.

Meibom, K. L., M. Blokesch, N. A. Dolganov, C. Y. Wu and G. K. Schoolnik (2005). &quot;Chitin induces natural competence in Vibrio cholerae.&quot; Science 310(5755): 1824-1827.

Mell JC, Redfield RJ. Natural competence and the evolution of DNA uptake specificity. J Bacteriol. (2014);196(8):1471-83.
Mizusawa, S. and S. Gottesman (1983). &quot;Protein degradation in Escherichia coli: the lon gene controls the stability of sulA protein.&quot; Proc Natl Acad Sci U S A 80(2): 358-362.

Nishino, T., Morikawa, T., Structure and function of nucleases in DNA repair: shape, grip and blade of the DNA scissors, Oncogene (2002) 21, 9022 – 9032.

Pan, G. and P. D. Sadowski (1992). &quot;Ligation activity of FLP recombinase. The strand ligation activity of a site-specific recombinase using an activated DNA substrate.&quot; J Biol Chem 267(18): 12397-12399.

Pollack-Berti, A., M. S. Wollenberg and E. G. Ruby (2010). &quot;Natural transformation of Vibrio fischeri requires tfoX and tfoY.&quot; Environ Microbiol 12(8): 2302-2311.

Sadowski, P. D. (1995). &quot;The Flp recombinase of the 2-microns plasmid of Saccharomyces cerevisiae.&quot; Prog Nucleic Acid Res Mol Biol 51: 53-91.

Susan T. Lovett, The DNA exonucleases of Escherichia coli, (2003) 4(2).
Savir, Y.,Tlusty, T., RecA-mediated homology search as a nearly optimal signal detection system, Molecular Cell (2010) 40, 388–396.
Santillán M, Mackey MC., Quantitative approaches to the study of bistability in the lac operon of Escherichia coli., J R Soc Interface. (2008) 6;5 Suppl 1:S29-39.
Serebriiskii I.G., Golemis E.A., Uses of lacZ to study gene function: evaluation of β-galactosidase assays employed in the yeast two-hybrid system. Anal. Biochem. 2000;285:1–15.
Studier, F. W. &amp; Moffatt, B. A. Use of bacteriophage T7 RNA polymerase to direct selective high-level expression of cloned genes. Journal of molecular biology 189, 113–130 (1986).

Sodeinde, O. A., Y. V. Subrahmanyam, K. Stark, T. Quan, Y. Bao and J. D. Goguen (1992). &quot;A surface protease and the invasive character of plague.&quot; Science 258(5084): 1004-1007.

Tabor, S. &amp; Richardson, C. C. A bacteriophage T7 RNA polymerase/promoter system for controlled exclusive expression of specific genes. Proceedings of the National Academy of Sciences of the United States of America 82, 1074–1078 (1985).

Villaverde, A. and M. M. Carrio (2003). &quot;Protein aggregation in recombinant bacteria: biological role of inclusion bodies.&quot; Biotechnol Lett 25(17): 1385-1395.

Wu R., Zhao M., Li J., Gao H., Kan B., Liang W. Direct regulation of the natural competence regulator gene tfoX by cyclic AMP (cAMP) and cAMP receptor protein (CRP) in Vibrios. Sci Rep. (2015) 7;5:14921.


Weinstock, M. T., E. D. Hesek, C. M. Wilson and D. G. Gibson (2016). &quot;Vibrio natriegens as a fast-growing host for molecular biology.&quot; Nat Methods 13(10): 849-851.

Zamenhof, P.J., Villarejo, M. Construction and Properties of E. coli Strains Exhibiting α-Complementation of β-Galactosidase Fragments In Vivo. J Bacteriol. (1972); 110(1): 171–178.



							Engler C, Kandzia R, Marillonnet S. 2008. A One Pot, One Step, Precision Cloning Method with High Throughput Capability. PLOS ONE 3: e3647.
							https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0003647 (October 18, 2018)
							
							Iverson SV, Haddock TL, Beal J, Densmore DM. 2016. CIDAR MoClo: Improved MoClo Assembly Standard and New E. coli Part Library Enable Rapid Combinatorial Design for Synthetic and Traditional Biology. ACS Synthetic Biology 5: 99–103.
							https://doi.org/10.1021/acssynbio.5b00124 (October 18, 2018)
							
							Lee ME, DeLoache WC, Cervantes B, Dueber JE. 2015. A Highly Characterized Yeast Toolkit for Modular, Multipart Assembly. ACS Synthetic Biology 4: 975–986.
							https://doi.org/10.1021/sb500366v (October 18, 2018)
							
							Moore SJ, Lai H-E, Kelwick RJR, Chee SM, Bell DJ, Polizzi KM, Freemont PS. 2016. EcoFlex: A Multifunctional MoClo Kit for E. coli Synthetic Biology. ACS Synthetic Biology 5: 1059–1069.
							https://doi.org/10.1021/acssynbio.6b00031 (October 18, 2018)
							
							Patron NJ, Orzaez D, Marillonnet S, Warzecha H, Matthewman C, Youles M, Raitskin O, Leveau A, Farré G, Rogers C, et al. 2015. Standards for plant synthetic biology: a common syntax for exchange of DNA parts. New Phytologist 208: 13–19.
							https://nph.onlinelibrary.wiley.com/doi/abs/10.1111/nph.13532 (October 18, 2018)
							
							Pollak B, Cerda A, Delmans M, Álamos S, Moyano T, West A, Gutiérrez RA, Patron N, Federici F, Haseloff J. 2018. Loop Assembly: a simple and open system for recursive fabrication of DNA circuits. bioRxiv: 247593.
							https://www.biorxiv.org/content/early/2018/01/18/247593 (October 18, 2018)
							
							Sarrion-Perdigones A, Falconi EE, Zandalinas SI, Juárez P, Fernández-del-Carmen A, Granell A, Orzaez D. 2011. GoldenBraid: An Iterative Cloning System for Standardized Assembly of Reusable Genetic Modules. PLOS ONE 6: e21622.
							https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0021622 (October 18, 2018)
							
							Sarrion-Perdigones A, Vazquez-Vilar M, Palací J, Castelijns B, Forment J, Ziarsolo P, Blanca J, Granell A, Orzaez D. 2013. GoldenBraid 2.0: A Comprehensive DNA Assembly Framework for Plant Synthetic Biology. Plant Physiology 162: 1618–1631.
							http://www.plantphysiol.org/content/162/3/1618 (October 18, 2018)
							
							Weber E, Engler C, Gruetzner R, Werner S, Marillonnet S. 2011. A Modular Cloning System for Standardized Assembly of Multigene Constructs. PLOS ONE 6: e16765.
							https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0016765 (October 18, 2018)

</P></DIV></DIV><DIV style="position: absolute;left: 0;bottom: 0;color: #b1b1b1;margin: 0.5%;font-size: 0.8em;">B. Marchal</DIV></DIV></BODY></HTML>