Team:Marburg/Demonstrate
Team ▾
Team
Collaborations
Attributions
Project ▾
Description
Design
Experiments
Notebook
InterLab
Model
Results
Demonstrate
Measurement
Labautomation
Parts ▾
Parts Overview
Basic Parts
Composite Part
Part Collection
Improve
Safety
Human Practices ▾
Human Practices
Public Engagement
Awards ▾
Integrated Human Practices
Education and Public Engagement
Model
Measurement
Best New Basic Part
Best New Composite Part
Best Part Collection
Software
Judging Form
Demonstrate
The important achievement of Apollo was demonstrating
that humanity is not forever chained to this planet
and our visions go rather further than that and our opportunities are unlimited.
-- Neil Armstrong
Vibrio Basics
Realizing V. natriegens as a widely used host organism for synthetic biology requires well-funded knowledge about it! Realizing this, we prioritized fundamental research early on. We showed the unparalleled speed of V. natriegens replication, defined a range of optimal growth conditions, including pH and salt tolerance, and  the ease of its genetic accessibility. protocols
We managed to enable transformation demonstrated for with high electroporation efficiency and heat-shock transformation to drive synthetic biology research.
In combination with our Marburg-Collection, we accomplished cloning of simple plasmids, from transformation to miniprep, under 12 hours, and assembly and preparation of level 2 golden gate constructs in under three days!
Additionally, our team successfully implemented five fragment Gibson cloning as well as Aqua cloning and achieved high reliability at high performance.
We sequenced both chromosomes with Illumina sequencing, mapped them to existing genome maps and ran automated annotation tools to identify genetic features.
Working concentrations for most common antibiotics were elucidated and used throughout the project.
Applying several electron microscopic methods, we could, apart from generating nice pictures, highlight shape, form and volume of V. natriegens. Fortunately, we could observe several cell divisions in mid process.
Strain Engineering
We successfully demonstrated a genome engineering workflow in V. natriegens, using linear fragments and transforming them by natural competence induced by the regulator protein TfoX. Furthermore, we established the Flp/frt system in V. natriegens for excisions of selection markers that were integrated into the genome.
For VibriClone 1.0, we could successfully delete the nuclease dns and create linear
DNA
cassettes for further genomic modifications, to improve this strain and allow a highly efficient cloning. Moreover we designed VibriClone 2.0 with additional features, such as deletion of both nucleases, a recA1 mutation, cold resistance and the ability for blue white selection, which makes our strain more suitable as the next generation cloning chassis.
In case of VibriXpress we detected a protease with a high similarity to the Lon protease of E. coli that needs to be deleted for high protein yield and could design and create cassettes for the integration of the T7 polymerase, the deletion of lon and the deletion of dns. We were not able to cotransform these fragments into V. natriegens, but could show, that the integration of the dns deletion cassette was successful.
We created the strain VibriInteract carrying the deletion of cyaA. The excision of the selection marker, using the Flp/frt system and curing the plasmid for natural transformation was successful. By this, we provided a strain for fast protein interaction studies, could demonstrate its functionality by
V2H
assays and characteized its growth and cell morphology.
We successfully characterized 15 neutral chromosomal integration sites in Vibrio natriegens, eight of them on chromosome 1 and seven on chromosome 2. Furthermore, we verified the correct integration of 5 of them on both chromosomes, by sequencing and characterized their impact on the growth rate. Insertion of the two sequences (12 and 20) results in a deficit in growth and sequence 15 only affects the growth in a small impact. For the sequences 19 and 22 our results show that an integration has almost no effect on the growth. Therefore, they are very suitable for integration, and open up a broad range of application, such strain engineering or integrating enzymes in metabolic engineering.
Marburg Collection
We designed and constructed the Marburg Collection the most flexible golden-gate based toolbox for prokaryotes.
It contains 123 LVL0 parts including:   constitutive and inducible promoters,
RBS,
reporter and tools for genome engineering, terminators, oris, resistance cassettes and a set of self-designed connectors.
All parts were submitted to the registry to help future
iGEM
teams in achieving ambitious projects and, for increased convenience, we additionally enable download of plasmid maps of all parts in our wiki
To characterize our parts, we established a fast and convenient platereader workflow tailored to species-specific properties of V. natriegens and evaluated the optimal plasmidal context.
Little characterization has been done for genetic parts in V. natriegens, so we applied our own workflow to obtain the very first experimental data for promoter strength, dose dependency of inducible promoters,
RBS,
strength, terminator read-through and the insulating behavior of our novel connector parts.
We characterized our ori parts in V. natriegens by showing their impact on reporter expression and furthermore,
qPCR,
experiments revealed differences in plasmid copy numbers depending on reporter expression.
To additionally ease Golden-Gate cloning we developed the software tool Click ‘n’ Clone which provides a
GUI
in which a user can simply select the desired parts for building a plasmid. A detailed pipetting protocol for manual operation or a picking list that is compatible with lab automation is given as result.
Metabolic Engineering
By characterizing all involved enzymes, we laid the foundation to work with the
3-HPA
pathwayin Vibrio natriegens.  We tested several
Accs
from a range of organisms and demonstrated, their activity in V. natriegens.
Acc
from Synechococcus elongatus proved to be the most promising contender for maximum productivity.
Also, the
Mcr
was shown to be soluble and functional.  Combining
Mcr
and
Acc
in vitro, we could quantitatively detect our product
3-HPA
, and additionally, reliably differentiate it from its structural isomer lactate.
Drawing from the strength of our model, we rationally designed our pathway for optimal, directed metabolite flow.
Moreover, we used the Marburg Collection to create a library containing over 390 pathway variants. These harbor different combinations of promoter,
RBS,
and coding sequences for
Acc
,
Mcr
and BirA.
To enable rapid in vivo and in vitro product screening, we successfully cloned a
3-HPA
biosensor .
Benefiting from the flexibility of the Marburg Collection in combination with sensor based real time product screening, we created the possibility for directed evolution by designing a workflow utilizing
FACS
and
MAGE
.
Furthermore, we designed a bypass to re-feed our product into the central metabolism, thereby introducing evolutionary pressure to adopt and strengthen this route
To conclude, we considered each aspect of metabolic engineering and successfully accelerated the workflow of pathway optimization.
B. Marchal
