<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Marburg_Design skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Marburg/Design</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="header_wrapper"><DIV class="menu_wrapper" role="navigation" aria-label="Main Menu"><UL class="menu"><LI><A href="#">Team ▾</A></LI><LI><A href="/Team:Marburg/Team">Team</A></LI><LI><A href="/Team:Marburg/Collaborations">Collaborations</A></LI><LI><A href="/Team:Marburg/Attributions">Attributions</A></LI></UL><LI><A href="#">Project ▾</A></LI><LI><A href="/Team:Marburg/Description">Description</A></LI><LI><A href="/Team:Marburg/Design">Design</A></LI><LI><A href="/Team:Marburg/Experiments">Experiments</A></LI><LI><A href="/Team:Marburg/Notebook">Notebook</A></LI><LI><A href="/Team:Marburg/InterLab">InterLab</A></LI><LI><A href="/Team:Marburg/Model">Model</A></LI><LI><A href="/Team:Marburg/Results">Results</A></LI><LI><A href="/Team:Marburg/Demonstrate">Demonstrate</A></LI><LI><A href="/Team:Marburg/Measurement">Measurement</A></LI><LI><A href="/Team:Marburg/Labautomation">Labautomation</A></LI><LI><A href="#">Parts ▾</A></LI><LI><A href="/Team:Marburg/Parts">Parts Overview</A></LI><LI><A href="/Team:Marburg/Basic_Part">Basic Parts</A></LI><LI><A href="/Team:Marburg/Composite_Part">Composite Part</A></LI><LI><A href="/Team:Marburg/Part_Collection">Part Collection</A></LI><LI><A href="/Team:Marburg/Improve">Improve</A></LI><LI><A href="/Team:Marburg/Safety">Safety</A></LI><LI><A href="#">Human Practices ▾</A></LI><LI><A href="/Team:Marburg/Human_Practices">Human Practices</A></LI><LI><A href="/Team:Marburg/Public_Engagement">Public Engagement</A></LI><LI><A href="#">Awards ▾</A></LI><LI><A href="/Team:Marburg/Human_Practices">Integrated Human Practices</A></LI><LI><A href="/Team:Marburg/Public_Engagement">Education and Public Engagement</A></LI><LI><A href="/Team:Marburg/Model">Model</A></LI><LI><A href="/Team:Marburg/Measurement">Measurement</A></LI><LI><A href="/Team:Marburg/Basic_Part">Best New Basic Part</A></LI><LI><A href="/Team:Marburg/Composite_Part">Best New Composite Part</A></LI><LI><A href="/Team:Marburg/Part_Collection">Best Part Collection</A></LI><LI><A href="/Team:Marburg/Software">Software</A></LI><LI><A href="https://igem.org/2018_Judging_Form?id=2560">Judging Form</A></LI></DIV></DIV><MAIN><DIV class="titleWrapper"><DIV class="title">Design</DIV></DIV><ARTICLE><P><I>Careful planning puts you ahead in the long run; hurry and scurry puts you further behind</I><B>   -- King Solomon</B></P>


What does creating task specific lab strains, building a novel golden-gate based toolbox and envisioning a synthetic metabolic pathway have in common? 
All these approaches require intense theoretical groundwork. Many hours have to be spent with screening literature, summarizing the current state of knowledge and finally, developing own ideas. Constructs have to be designed and experiments have to be planned. 
All this preliminary work has to be completed before the lab work can be truly started. In addition to our initial theoretical work, we routinely iterated the steps: designing, building, testing and learning throughout the whole project. 
On this page, we want to provide you the theoretical groundwork that we did to design our projects. We want to demonstrate that we followed Synbio design principles and conceived our project based on the current state of knowledge.
<DIV class="collapsible"><DIV class="btn_expand"><DIV class="btn_e_title"> The vision of &quot;Cloning in one Day&quot; </DIV><DIV class="btn_e_desc"> We realized that the fast growth of V. natriegens is well suited to accelerate the most time consuming and least rewarding period of lab work. We approached to push the limits of growth speed to the absolute minimum by optimizing culture conditions. </DIV></DIV><DIV class="content"><DIV style="display:block; margin:0 auto;"><H1 style="font-size:250%; text-align:center">ONE DAY CLONING</H1></DIV><FIGURE style="width: 90%; display:block; margin:0 auto;"><FIGCAPTION><B> Figure 1:  Workflow for cloning in one day </B></FIGCAPTION></FIGURE>

Cloning is the foundation of most Synbio projects and consumes most of the time. However, cloning is at the same time the least rewarding phase of a project. In a publication, usually the assembly of constructs, the work that the scientists spent most time on, becomes a small paragraph in the material and methods section. 
It is our belief, that in future projects, the time spent on cloning should be reduced to a minimum. This will enable scientists to focus on designing constructs and performing experiments, rather than the laborious cloning steps.  
When we conceived our project, we immediately realized that<I> V. natriegens</I> could become a powerful cloning chassis. Its incredible growth rate should excessively reduce the time wasted by waiting for cultures to grow. We were caught by the vision of “Cloning in one day”. With a doubling time of seven minutes it should be possible to start with a transformation in the morning, picking colonies in the early afternoon and finally obtain purified plasmids in the evening. Such cloning cycles usually take three days when using<I> E. coli</I> as the cloning chassis and we attempt to perform the same steps in one day. 
When using our Marburg Collection, we should be able to create a multigene construct in three days by repeating “Cloning in one day” three times. 
We designed a LVL2 plasmid that contains a sfGFP and RFP transcription unit. This should facilitate easy detection of the intermediate plasmids thus omitting the need of sequencing.  <DIV style="display: inline-block;"><FIGURE style="width: 50%; float: right"><FIGCAPTION><B> Figure 1: Baffled falcons </B> Enhanced oxygen availability increases the growth rate even further</FIGCAPTION></FIGURE> 

During the process of cloning optimization we realized that the limits can be pushed even further with improved culture conditions. To enhance availability of oxygen during the liquid culture, we constructed self-made “baffled falcon”. We assume that 3 mL culture in such a modified 50 mL tube should provide the cells with the maximum possible oxygen, additionally increasing the growth. We payed attention to prewarm liquid media as well as the agar plates. 
As an outlook we want to suggest a future where cloning is done by robots instead of humans. We consider the combination of <I>V. natriegens</I> with colony picking and pipetting robots as a powerful method to incredibly accelerate scientific progress. Machines do not need after-work-hours and therefore the short growth periods of <I>V. natriegens</I> can be used to their fullest potential. 
</DIV></DIV></DIV><DIV class="collapsible"><DIV class="btn_expand"><DIV class="btn_e_title"> Design of the Marburg Collection </DIV><DIV class="btn_e_desc"> We created a novel golden-gate based cloning toolbox consisting of 123 parts. Here we describe the design of all relevant features of this toolbox. We provide instruction on how to use the connectors and the thought behind the selection of specific fusion sites. </DIV></DIV><DIV class="content"><DIV class="collapsible"><DIV class="btn_expand">Overview of the Marburg Collection</DIV><DIV class="content">

    A major goal of our project was to create a collection of characterized  parts for use in <I>V. natriegens </I>to provide other iGEM teams, as well as the whole scientific community a toolbox for the rational design of metabolic pathways, genetic circuits or any other DNA construct.
Our toolbox was initially conceived for being used in <I>V. natriegens</I>. However, while designing the Marburg Collection, we realized that a toolbox with maximum flexibility can easily be used in more than one bacterial species. Because alternative bacterial chassis, apart from <I>E. coli</I>, are gaining increasing importance 

<A href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5264504/"><ABBR title="Properties of alternative microbial hosts used in synthetic biology: towards the design of a modular chassis">(Kim <I>et al.</I> 2016.</ABBR></A>)

, a toolbox that is compatible with more than one bacterial species enables scientists to work with organisms that have the exact properties needed for specific applications. Our vision is to establish the Marburg Collection as the first broad-host range golden-gate-based cloning toolbox.<FIGURE style="width: 40%; float: right"><FIGCAPTION><B> Figure 1:  Hierarchical cloning is facilitated by subsequent Golden Gate reactions.</B>
Basic building blocks like promoters or terminators are stored in LVL0 plasmids. Parts from each category of our collection can be chosen to built LVL 1 plasmids harboring a single transcription unit. Up to five transcription units can be assembled into a LVL2 plasmid.  </FIGCAPTION></FIGURE>
We investigated existing bacterial toolboxes like EcoFlex 

<A href="https://pubs.acs.org/doi/abs/10.1021/acssynbio.6b00031"><ABBR title="EcoFlex: A Multifunctional MoClo Kit for E. coli Synthetic Biology">(Moore <I>et al.</I> 2016.)</ABBR></A>


and iGEMs PhytoBrick system, which is closely related to the GoldenBraid cloning <A href="https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0021622"><ABBR title="GoldenBraid: An Iterative Cloning System for Standardized Assembly of Reusable Genetic Modules">(Sarrion-Perdigones <I>et al.</I> 2011.)</ABBR></A>. All these toolboxes are based on predefined vectors that preset the origin of replication and the antibiotic resistance cassette thus preventing their use in organisms that are not compatible with either feature. We wanted to give scientists the freedom of choice to select oris and resistance cassettes that suit their organisms and applications best. This flexibility is possible because our toolbox does not rely on classical “backbones” but instead functions by the complete <I>de novo </I>assembly of at least eight basic genetic parts. 
All basic parts like promoters or resistance parts are stored in LVL0 plasmids. The assembly of a plasmid comprising a single transcription unit is done by assembling at least eight parts resulting in one LVL1 plasmid. One to five LVL1 plasmids can then be used for a subsequent round of assembly to obtain a multigene LVL2 plasmid which could already harbor a full synthetic metabolic pathway consisting of up to five enzymes. Our toolbox even allows more rounds of assembly, each combining up to five constructs of the previous level. This enormous cloning capacity results theoretically in an  infinite number of transcription units that can be assembled in a small amount of time.

<FIGURE style="width: 80%; display:block; margin:0 auto;"><FIGCAPTION><B> Figure 2: Hierarchical cloning is facilitated by subsequent Golden Gate reactions.</B> Basic building blocks like promoters or terminators are stored in LVL0 plasmids. Parts from each category of our collection can be chosen to built LVL1 plasmids harboring a single transcription unit. Up to five transcription units can be assembled into a LVL2 plasmid.  </FIGCAPTION></FIGURE>

An additional layer of flexibility in the construction of LVL2 plasmids is added to our toolbox by our 5’- and 3’- Connectors which flank the transcription units in LVL1 plasmids. These connectors provide the fusion sites for the assembly of LVL2 plasmids. By selecting the correct connectors, the user can define the order as well as the orientation of each transcription unit. The ability to assign an orientation to a transcription unit is supposed to reduce the influence of neighboring unit on each other caused by transcriptional read through and DNA supercoiling. Moreover, our connectors are designed to function as genetic insulators. They consist of 300 bp “neutral DNA” flanked by strong transcriptional terminators to separate each transcription unit in a LVL2 plasmid. We believe that this novel feature is an important step towards building genetic constructs in a rational and predictable manner.</DIV></DIV><DIV class="collapsible"><DIV class="btn_expand">Construction of novel part entry vectors</DIV><DIV class="content">
All LVL0 parts have to be stored in plasmids to allow for amplification and long term storage. To create new LVL0 parts, a <DFN data-info="polymerase chain reaction"> PCR </DFN> product or annealed oligos are cloned into a part entry vector. This vector harbours the resistance and ori that are required for selection and propagation. Furthermore, part entry vectors can be designed in a way that they contain a dropout. This dropout can be a transcription unit for a marker that generates a visible output. The first golden-gate-based toolbox MoClo <A href="https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0016765"><ABBR title="A Modular Cloning System for Standardized Assembly of Multigene Constructs">(Weber <I>et al.</I> 2011.)</ABBR></A> used a LacZ alpha transcription unit which can be used for blue white screening in many <I>E. coli</I>i&gt; cloning strains. This concept was also adapted by iGEMs PhytoBrick system.  During the cloning of LVL0 parts, this dropout is replaced by the desired part. When the cloning reaction is transformed into a suitable <I>E. coli</I> strain and the cells are plated on agar plates with supplemented <DFN data-info="Isopropyl-β-D-thiogalactopyranosid"> IPTG </DFN> and <DFN data-info="5-Brom-4-chlor-3-indoxyl-β-D-galactopyranosid"> X-Gal </DFN>. Colonies transformed with the religated entry plasmid appear blue while white colonies most probably contain the correctly assembled plasmid. The LVL0 part entry vector in iGEMs PhytoBrick system  has been designed as described and can be used for blue-white screening.<A href="http://parts.igem.org/Part:BBa_P10500">(BBa_P10500)</A>.
We appreciate the approach of using part entry plasmids with dropouts but, for two reasons, we think that LacZ is not an optimal reporter. First, blue white screening requires the two expensive chemicals <DFN data-info="Isopropyl-β-D-thiogalactopyranosid"> IPTG </DFN>  and <DFN data-info="5-Brom-4-chlor-3-indoxyl-β-D-galactopyranosid"> X-Gal </DFN> which have to be added to the agar plates. Second, blue-white screening is restricted to <I>E.coli </I>strains with an incomplete <I>lac</I>i&gt; operon that is complemented by the LacZ alpha fragment that is expressed from the plasmid <A href="https://www.ncbi.nlm.nih.gov/pubmed/1093175?dopt=Abstract"><ABBR title="Molecular basis of beta-galactosidase alpha-complementation.">(Langley <I>et al.</I> 1975.)</ABBR></A>. Consequently blue white screening is not compatible with a <I>V. natriegens </I>wild type strain <A href="https://2018.igem.org/Team:Marburg/Improve">(Link to Improve)</A>.


<FIGURE style="width: 40%; float: right"><FIGCAPTION><B> Figure 3: <A href="http://parts.igem.org/Part:BBa_K2560002">BBa_K2560002</A> was designed as a novel part entry vector </B> A  <DFN data-info="super folder green fluorescent protein"> sfGFP </DFN> dropout cassette is flanked by BsmBI recognition sites. During the cloning of LVL0 parts with BsmBI, this dropout cassette is replaced by the desired part </FIGCAPTION></FIGURE>

We overcame both drawbacks by replacing the LacZ alpha dropout of <A href="http://parts.igem.org/Part:BBa_P10500">(BBa_P10500)</A> by a <DFN data-info="red fluorescent protein"> RFP </DFN> and <DFN data-info="super folder green fluorescent protein"> sfGFP </DFN> dropout and thus created the two parts <A href="http://parts.igem.org/Part:BBa_K2560001">BBa_K2560001</A>and <A href="http://parts.igem.org/Part:BBa_K2560002">BBa_K2560002</A>, respectively. Like <A href="http://parts.igem.org/Part:BBa_P10500">(BBa_P10500)</A>, our two new dropout parts are located in a derivative of pSB1C3 which is extended with two BsaI recognition sites. The dropout parts are flanked by BsmBI recognition sites that are required for replacing the dropout by the desired LVL0 part. The resulting LVL0 parts are classified as PhytoBricks and are <DFN data-info="Request for Comments"> RFC </DFN>10 and <DFN data-info="Request for Comments"> RFC </DFN>5 compatible, assuming that the part itself does not contain prohibited recognition sites. The resulting plasmids can be used to clone and store all LVL0 parts. However, for resistance parts a different approach enables more convenient LVL1 cloning. In the LVL1 golden-gate-reaction, at least eight parts are combined in a single tube together with the required enzymes. All parts, except the resistance part, are stored in the previously described pSB1C3 derivate which confers a chloramphenicol resistance. Obviously, the plasmid that provides the new resistance cassette for the LVL1 plasmid (e.g. a Cas9 plasmid with kanamycin resistance), has to contain the same resistance cassette (kanamyicin in this example). Because cloning is never 100 % efficient, re-ligation of LVL0 plasmids is a common event, and in case of the LVL0 resistance part, results in false positive colonies which do not contain the desired LVL1 plasmid.
We developed a solution for this problem by creating the novel resistance entry vectors BBa_K2560005 and BBa_K2560006. These plasmids contain a <DFN data-info="red fluorescent protein"> RFP </DFN> and <DFN data-info="super folder green fluorescent protein"> sfGFP </DFN> dropout, respectively, and a chloramphenicol resistance cassette that is flanked by BsaI and BsmBI recognition sites. When a new LVL0 resistance part is cloned, the chloramphenicol resistance is replaced by the new antibiotic resistance marker resulting in a <DFN data-info="red fluorescent protein"> RFP </DFN> or <DFN data-info="super folder green fluorescent protein"> sfGFP </DFN> expressing plasmid with the respective resistance marker. When using these LVL0 parts for LVL1 cloning, the re-ligated resistance parts yield colonies with a visually detectable phenotype. As a result, correct plasmids can be easily identified, even for inefficient LVL1 clonings with &lt; 10 % efficiency.</DIV></DIV><DIV class="collapsible"><DIV class="btn_expand">Choice of fusion sites</DIV><DIV class="content">
Golden-gate based cloning relies upon the use of type IIs restriction endonucleases like BsaI, BsmBI or BpiI. In comparison to the commonly used type II restriction enzymes (e.g. EcoRI or PstI) they also recognize a specific DNA sequence but cleave outside of their recognition sequence <A href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC55916/"><ABBR title="Structure and function of type II restriction endonucleases">(Pingoud and Jeltsch, 2001)</ABBR></A>
. The golden-gate-cloning method is taking advantage of this property. A single enzyme can be used to create various single-stranded overhangs that match in a predefined order and finally lead to the correctly assembled plasmid <A href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC55916/"><ABBR title="Structure and function of type II restriction endonucleases">(Pingoud and Jeltsch, 2001)</ABBR></A>
.
When we started to design the Marburg Collection, we carefully investigated which fusion sites should be used. The fusion sites do not only set the order in which the single parts will be assembled but also affect the assembly efficiency and determine if a newly designed toolbox is compatible with already existing collections, so that parts can be shared easily. For us, the most important and decisive argument was that we wanted to be compatible with as many other toolboxes as possible. It is our strong belief that scientists all over the world should agree on one set of fusion sites to ensure complete interchangeability between different toolboxes. The toolboxes of MoClo <A href="https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0016765"><ABBR title="A Modular Cloning System for Standardized Assembly of Multigene Constructs">(Weber <I>et al.</I> 2011)</ABBR></A>, Loop Assembly <A href="https://www.biorxiv.org/content/early/2018/01/18/247593.full.pdf+html"><ABBR title="Loop Assembly: a simple and open system for recursive fabrication of DNA circuits">(Pollak <I>et al.</I> 2018)</ABBR></A>
 and the PhytoBrick system already use a common set of fusion sites. We decided to adapt these fusion sites for all parts that build the transcription unit (Promoter, <DFN data-info="ribosome binding site"> RBS </DFN>, <DFN data-info="coding sequence"> CDS </DFN>, Terminator, Tags). Fusion sites for parts that are novel to our system (Connectors, Oris, Resistance cassettes) had to be newly designed by us because these parts did not exist in the other toolboxes. 

We applied the following design principles to obtain optimal fusion sites. Firstly, the newly designed fusion sites must neither be identical to already existing fusion sites nor be palindromic to prevent assembly in a wrong order. Secondly, the fusion sites should not consist of bases that represent a portion of the recognition sequence of a restriction enzyme. If, for example, a fusion site with the sequence GGTC was used, and the sequence of the downstream part starts with TC, a BsaI recognition site would be reconstituted. So all fusion sites that would result in a partial recognition sequence of either BsaI, BsmBI or any of the enzymes that are used in the Biobrick cloning, are excluded. Lastly, the remaining candidates were sorted according to their GC content and the fusion sites with the highest GC content were chosen.
To make design of new parts as simple as possible, we created a collection of overhangs that can be copied from table 1 and pasted to the sequence specific part of a primer to create new LVL0 parts. These primers contain the cut sites for integration into the part entry vector as well as the predefined fusion sites that are required for correctly assembling LVL1 plasmids. In some cases these overhangs contain additional bases that will be discussed in the following chapter.
<DIV class="collapsible"><DIV class="btn_expand">View table with primer overhangs</DIV><DIV class="content"><P><FIGURE><TABLE style="width:100%"><TBODY><TR><TH>Part Category</TH><TH>Fwd Overhang</TH><TH>Rev Overhang</TH></TR><TR><TD>1 5’ Connector</TD><TD>AA<B><U>GGTCTC</U></B>GCTCG<B>AACA</B><U>CGTCTC</U>G<B><I>NNNN</I></B></TD><TD><B>GGAG</B>TGAGG<B><U>GAGACC</U></B>AA</TD></TR><TR><TD>2 Promoter</TD><TD>AA<U>CGTCTC</U>GCTCG<B>GGAG</B></TD><TD><B>TACTT</B>GAGG<U>GAGACG</U>AA</TD></TR><TR><TD>3 <DFN data-info="ribosome binding site"> RBS </DFN></TD><TD>AA<U>CGTCTC</U>GCTCG<B>TACT</B>AGAG</TD><TD>TAATC<B>AATGT</B>GAGG<U>GAGACG</U>AA</TD></TR><TR><TD>4 <DFN data-info="coding sequence"> CDS </DFN></TD><TD>AA<U>CGTCTC</U>GCTCG<B>AATG</B></TD><TD><B>GCTT</B>TGAGG<U>GAGACG</U>AA</TD></TR><TR><TD>5 Terminator</TD><TD>AA<U>CGTCTC</U>GCTCG<B>GCTT</B>AA</TD><TD><B>CGCT</B>TGAGG<U>GAGACG</U>AA</TD></TR><TR><TD>6 3’ Connector</TD><TD>AA<U><B>GGTCTC</B></U>GCTCG<B>CGCT</B></TD><TD><B><I>NNNN</I></B>G<U>GAGACG</U><B>AGCT</B>TGAGG<U><B>GAGACC</B></U>AA</TD></TR><TR><TD>7 Ori</TD><TD>AA<U>CGTCTC</U>GCTCG<B>AGCT</B></TD><TD><B>TGCT</B>TGAGG<U>GAGACG</U>AA</TD></TR><TR><TD>8 Resistance</TD><TD>AA<U>CGTCTC</U>GCTCGT<B>GCTT</B></TD><TD><B>AACA</B>TGAGG<U>GAGACG</U>AA</TD></TR><TR><TD>4x N-Tag</TD><TD>AA<U>CGTCTC</U>GCTCG<B>AATG</B></TD><TD>GG<B>GATG</B>TGAGG<U>GAGACG</U>AA</TD></TR><TR><TD>4yN-tagged <DFN data-info="coding sequence"> CDS </DFN></TD><TD>AA<U>CGTCTC</U>GCTCG<B>GATG</B></TD><TD><B>GCTT</B>TGAGG<U>GAGACG</U>AA</TD></TR><TR><TD>5a C-Tag</TD><TD>AA<U>CGTCTC</U>GCTCG<B>GCTT</B>TA</TD><TD>G<B>GGTA</B>TGAGG<U>GAGACG</U>AA</TD></TR><TR><TD>5b Terminator</TD><TD>AA<U>CGTCTC</U>GCTCGGGTAA</TD><TD><B>CGCT</B>TGAGG<U>GAGACG</U>AA</TD></TR></TBODY></TABLE><FIGCAPTION><B>Table 1:</B> Primer overhangs for creating new LVL0 parts. Overhangs have to be added to the 5’ end of the primer. Reverse overhangs have to be added as reverse complement. Underlined bases represent BsmBI recognition sites. Bolt bases indicate fusion sites for LVL1 assembly. Bold and underlined bases show BsaI recognition sites. N bases written in bold and italic show fusion sites for lvl1 assembly and have to be custom-designed. </FIGCAPTION></FIGURE></P></DIV></DIV></DIV></DIV><DIV class="collapsible"><DIV class="btn_expand">Additional bases between parts</DIV><DIV class="content">
The fusion sites of most golden-gate-based cloning methods create a four base pair scar that is referred to as the fusion site. The fusion sites are the feature that makes toolboxes compatible with each other. Between some parts, additional bases are required for different reasons. These bases were chosen carefully to achieve best performance of the respective part. Please note that these additional bases are not a strict requirement to use or being compatible with our toolbox but we recommend them for the design of additional parts.
The first additional bases were incorporated between the Promoter and <DFN data-info="ribosome binding site"> RBS </DFN> part. The fusion site is TACT and AGAG was added additionally. The sequence between promoter and <DFN data-info="ribosome binding site"> RBS </DFN> that results by using our suggested overhangs form the same scar that is created if the the parts were assembled with 3A Assembly <A href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2373286/"><ABBR title="Engineering BioBrick vectors from BioBrick parts">(Knight, 2003)</ABBR></A>. This means that the distance between promoter and <DFN data-info="ribosome binding site"> RBS </DFN> is not changed and therefore we do not expect negative effects in transcription or translation. Moreover, we hope that creating the same scar with a different method will make our experimental data more comparable to the data acquired with plasmids assembled with 3A Assembly in previous iGEM projects.
The next bases were integrated between <DFN data-info="ribosome binding site"> RBS </DFN> and <DFN data-info="coding sequence"> CDS </DFN> parts. The fusion site, which was adapted from the PhytoBrick system, AATG and TAATC was added upstream of it. Previous work has shown that the sequence between a <DFN data-info="ribosome binding site"> RBS </DFN> and the start codon dramatically affects the expression of the desired protein (Lentini<I> et al. </I>2013). A spacer length of six base pairs was shown to result in the strongest expression. When comparing different bases in a six bp spacer, the experimental data indicate significant differences. The 3A Assembly scar results in 50 % expression strength compared to the sequence TAATCT which was referred to as the reference (Lentini <I>et al. </I>2013). We chose to use the spacer sequence which is expected to result in highest expression as we think that a system that is designed to enable strongest expression can be easily adapted for low expression by using weak <DFN data-info="ribosome binding site"> RBS </DFN> or promoters while going to the opposite direction might be more difficult. Unfortunately, we could not adapt the exact “reference sequence” because the first A in the fusion site AATG is already part of the spacer. Eventually, we used the first five bases of the strongest spacer  (Lentini <I>et al. </I>2013) upstream of the fusion site. <FIGURE><FIGCAPTION><B> Figure 4: Additional bases and fusion sites ensure correct spacing and allow tags </B>
Between some parts, additional base pairs were integrated to ensure correct spacing and to maintain the triplet code. We expanded our toolbox by providing N- and C- terminal tags by creating novel fusions and splitting the <DFN data-info="coding sequence"> CDS </DFN> and terminator part, respectively.  </FIGCAPTION></FIGURE>


Close attention has to be paid to fusion sites that connect two sequences which are translated like the <DFN data-info="coding sequence"> CDS </DFN> part or N- and C- terminal Tags. A fusion site in our system consists of four bases which would result in disrupting the triplet code. To prevent mistranslated proteins, two additional bases have to be added to create a six bp scar that results in two translated amino acids. These can be seen as linkers between the joined <DFN data-info="coding sequence"> CDS </DFN> parts or tags. We decided to preferably use amino acids that are abundant in natural or synthetic flexible linkers like glycine and serine <A href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3726540/"><ABBR title="Fusion Protein Linkers: Property, Design and Functionality.">(Chen <I>et al.</I> 2012)</ABBR></A>
. Flexible linkers have been shown to improve the performance of epitope tags in Saccharomyces cerevisiae <A href="https://onlinelibrary.wiley.com/doi/pdf/10.1002/yea.1431"><ABBR title="A flexible protein linker improves the function of epitope-tagged proteins in Saccharomyces cerevisiae">(Sabourin <I>et al.</I> 2007)</ABBR></A>. Therefore we added the bases GG upstream of the fusion site between 4x (N-Tag) and 4y (<DFN data-info="coding sequence"> CDS </DFN>) which results in glycine and methionine (methionine is preset by the fusion site) and the bases TA downstream of the fusion site between 4 (<DFN data-info="coding sequence"> CDS </DFN>) and 5a (C-Tag) resulting in an  alanine and leucine linker. To allow for the optional use of C-terminal tags, a <DFN data-info="coding sequence"> CDS </DFN> part must not possess a stop codon. Therefore an additional linker has to be introduced between 4 (<DFN data-info="coding sequence"> CDS </DFN>) and 5 (Terminator) as well as between 5a (C-Tag) and 5b (Terminator) resulting in an alanine-STOP and glycine-STOP, respectively. </DIV></DIV><DIV class="collapsible"><DIV class="btn_expand">Design of Connectors</DIV><DIV class="content"><P> 
One key feature of the Marburg Collection are the connectors that provide our toolbox with the required flexibility. Our design is inspired by the “Dueber Toolbox”, a golden-gate-based cloning method designed for applications in Saccharomyces cerevisiae <A href="https://pubs.acs.org/doi/abs/10.1021/sb500366v"><ABBR title="A Highly Characterized Yeast Toolkit for Modular, Multipart Assembly">(Lee <I>et al.</I> 2015)</ABBR></A>
 . To our knowledge, the “Dueber Toolbox” is the first cloning system that performs assembly through all levels using restriction sites located on independent basic parts, termed connectors, instead of destination plasmids for level 1 and level 2 plasmids <A href="https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0016765"><ABBR title="A Modular Cloning System for Standardized Assembly of Multigene Constructs">(Weber <I>et al.</I> 2011)</ABBR></A>
. <FIGURE style="width: 50%; float: right"><FIGCAPTION><B> Figure 5: Default setup of connectors for building a LVL2 plasmid consisting of 5 transcription units</B> The connectors match in the following pattern: 5'Con(N) connects to 3'Con(N-1)   </FIGCAPTION></FIGURE>
Using connectors instead of destination plasmids enables the user to freely choose antibiotic resistance parts and oris. To achieve the flexibility to choose one out of four resistances and one out of three oris, like it is possible in our toolbox, twelve plasmids would be required for building LVL1 plasmids. For building LVL2 plasmids, LVL1 plasmids with different fusion sites are needed. In our system, this is also achieved through the connectors in contrast to the destination plasmids which are used in other cloning methods. 

In already existing toolboxes, the combinatorics that come with choosing oris and resistances would be multiplied by the number of required positional vectors which provide the fusion sites for LVL2 assembly. A toolbox with five possible positions, four resistances and three oris would require 60 LVL1 destination plasmids. To enable inversion of transcription units, a built-in feature in our toolbox, the previous number would be multiplied by two, resulting in 120 required LVL1 destination plasmids. 
Unsurprisingly, this theoretical toolbox would not be convenient to be built and used. 
The Marburg Collection presents a novel approach that achieves maximum flexibility with a minimum of required components. We provide two sets of connectors. The first set, the “short connectors” solely provide the fusion sites for subsequent assembly. The fusion sites are identical to the ones that are used in building LVL1 plasmids to avoid having to design a complete new set of fusion sites. This also enables using LVL0 ori and resistance parts in LVL2 plasmids. Inversion of individual transcription units was also achieved by designing fusion sites that match in reverse order. <FIGURE style="width: 96%; float: center"><FIGCAPTION><B> Figure 6: Inversion of transcription units is feasible with our connectors. </B> To achieve inversion, the respective transcription unit has to be built with connectors that match the neighboring transcription units. The function by following pattern: 3'Con(N)inv connects to 3'Con(N-1) and 5'Con(N)inv connects to 5'Con(N+1) </FIGCAPTION></FIGURE>

The 5’ and 3’ connectors can be chosen independently to build a LVL1 plasmid. This allows the user to combine any number of LVL1 plasmids between one and five into a LVL2 plasmid without end linkers that are required in other toolboxes (Andreou and Nakayama, 2018). An unusual example is shown in figure 7 to demonstrate the number of possibilities. The only logic that has to be followed for designing multigene constructs is to follow the formula <B>5'Con(N) connects to 3'Con(N-1)</B>. In the default set up <DFN data-info="transcription unit"> TU</DFN>(N) is built with 5'Con(N) and 3'Con(N) This will result in a LVL2 plasmid harboring five transcription units. 
However, any other combination that follows the previously given rule will result in a functionally assembled LVL2 plasmid.

<FIGURE style="width: 96%; float: center"><FIGCAPTION><B>Figure 7: Unusual examples are displayed to show the range of possibilities when using the Marburg Collection</B></FIGCAPTION></FIGURE></P><H3> Design of insulators </H3>
In addition to the “short connectors”, we designed a set of “long connectors” that function as genetic insulators. They fulfill the same basic functions as the short connectors, providing fusion sites for subsequent cloning. Additionally, they consist of 300 bp “neutral DNA” flanked by two strong transcriptional terminators.


The purpose of these long connectors is to reducing cross interaction between neighbouring transcription units, mainly transcriptional read through, a phenomenon that was described previously <A href="https://pubs.acs.org/doi/10.1021/sb5000115"><ABBR title="Preventing T7 RNA Polymerase Read-through Transcription—A Synthetic Termination Signal Capable of Improving Bioprocess Stability">(Mairhofer <I>et al.</I> 2015)</ABBR></A>. 
The design of “neutral DNA” started with generating 300 bp of random DNA which do not possess recognition sites of BsaI, BsmBI and any of the enzymes used in Biobrick Assembly, using a self-made Matlab script. These sequences were analyzed with the reverse mode of the R2oDNA designer <A href="https://www.ncbi.nlm.nih.gov/pubmed/24933158"><ABBR title="R2oDNA designer: computational design of biologically neutral synthetic DNA sequences.">(Casini <I>et al.</I> 2014)</ABBR></A>
. The R2oDNA designer can take a DNA sequence as input and quantifies the extent of secondary structures, repeats and forbidden sequence motifs, such as promoter or <DFN data-info="ribosome binding site"> RBS </DFN> motifs. Because the genome sequence of <I>V. natriegens </I>is not included in the tool, a BLAST search was performed manually with the sequences to check for homologies with the genome of <I>V. natriegens</I>. The top hit was considered as a fourth score to quantify the quality of a spacer sequence. At the end of this process, four scores were assigned to every sequence, each describing a different characteristic. We decided that spacer sequences with decent scores in each category are better suited as insulators than sequences with a superior score in one and inferior scores in other categories. <FIGURE style="width: 70%; display:block; margin:0 auto;"><FIGCAPTION><B> Figure 8: Design of our insulating connectors </B> 300 bp neutral DNA are flanked by two strong terminators. A BsmBI site for level 2 cloning is incorporated at the 5'end </FIGCAPTION></FIGURE>

We developed a Matlab script to find the “best” spacer sequences by increasing quantiles and picking spacers that fall into the lowest quantiles for all four categories. For example, the best sequence was found to be among the best 28 % of all tested sequences in terms of secondary structure, repeats, forbidden motifs and homology to the genome of <I>V. natriegens</I>. 
The selected sequences were flanked by synthetic transcriptional terminators that were developed in the lab of Christopher A. Voigt 


<A href="https://www.nature.com/articles/nmeth.2515"><ABBR title="Characterization of 582 natural and synthetic terminators and quantification of their design constraints.">(Chen<I>et al.</I> 2013.)</ABBR></A>. Most terminators were described to be unidirectional 
<A href="https://www.nature.com/articles/nmeth.2515"><ABBR title="Characterization of 582 natural and synthetic terminators and quantification of their design constraints.">(Chen<I>et al.</I> 2013.)</ABBR></A>


and therefore the orientation of the flanking terminators had to be considered. Our priority was to prevent transcription into the respective transcription unit from the upstream sequence. Therefore each spacer sequence was equipped with a strong terminator at the 3’ and 5’ end of the 5’ connector in forward and reverse orientation, respectively. 
We expect to achieve the best reduction of crosstalk between neighbouring transcription units by combining our insulators in combination with connectors that facilitate inversion of an individual transcription unit, thus providing a large step towards the reliable and predictable design of synthetic circuits.
</DIV></DIV><DIV class="collapsible"><DIV class="btn_expand">Selection of parts</DIV><DIV class="content">

In addition to the connectors that form the scaffold of our toolbox, we evaluated which parts should be integrated into the Marburg Collection. As a starting point, we focused on parts that are already commonly used in <I>E. coli</I> to demonstrate their compatibility with our new chassis <I>V. natriegens</I>. Therefore we integrated all 20 promoters of the Anderson Library, four <DFN data-info="ribosome binding site"> RBS </DFN> from the Community Collection and the most frequently used reporters and terminators. We also included two well established inducible promoters, three widely used oris and the four resistance parts covering the antibiotics used by iGEM. 
In addition to this standard set of parts, our toolbox provides N- and C-terminal fluorescence and epitope tags to facilitate fluorescence microscopy and protein purification and degradation tags that can be used to finetune protein levels. Lastly, we submitted all project related parts that can be used to perform genome engineering via CRISPR/Cas9 or natural competence and all enzymes that were used in our metabolic engineering project.   </DIV></DIV><DIV class="collapsible"><DIV class="btn_expand">Summary of the features of the Marburg Collection</DIV><DIV class="content">
The Marburg Collection is designed to be the most flexible golden-gate-based toolbox for prokaryotes. The high degree of flexibility of our toolbox is achieved by the de novo construction of plasmids instead of using entry vectors like most other toolboxes. A novel core feature are the connectors that provide the fusion sites for subsequent cloning steps and can be used to set the order and orientation of each individual transcription unit. A set of newly designed insulators, consisting of “neutral DNA”, and two transcriptional terminators, are designed to minimize crosstalk between neighbouring transcription units. 
</DIV></DIV></DIV></DIV><DIV class="collapsible"><DIV class="btn_expand"><DIV class="btn_e_title">Strain Engineering</DIV><DIV class="btn_e_desc">
			In Strain Engineering we modified <I>V. natriegens</I> to establish three different strains which were designed based on commonly used <I>E. coli</I> lab strains to accelerate daily lab work. Therefore, we designed a new workflow for fast genomic modificationsin <I>V. natriegens</I> to further speed up our project.
		</DIV></DIV><DIV class="content"><DIV class="collapsible"><DIV class="btn_expand">Methods</DIV><DIV class="content">
We designed a workflow for fast genomic modifications in <I>V. natriegens</I> using the Flp/FRT system for excision of antibiotics markers and combining this technique with the natural competence of V. natriegens for the uptake of linear DNA from their environment. Therefore, we utilized the Marburg Collection to assemble the desired constructs for introducing the Flp recombinase and TfoX into the <I>V. natriegens</I> cells.
<H3>Natural Competence</H3>In our project we wanted to build many different strains. For this we have to make many genomic changes of the wildtype of Vibrio natriegens. Mutations must be introduced in some genes, others must be deleted or inserted. In order to think about a good method that works in V. natriegens and with which we can make many modifications very quickly, we looked up into the literature. We found a great publication from the group of Ankur B. Dalia published 2017. They have shown that V. natriegens is natural competent and that it is possible to perform a natural transformation with this organism. This method is a good choice for many multiplex genome editing in a short time. They showed successful that it was possible to integrate a linear PCR product with a antibiotic cassette that replace the gene of interest. Besides, they have managed to perform a cotransformation. This fact was very interesting to us and we knew immediately that if we can reproduce it, then that's the method we want to use.
It was possible for us to get in contact with the group leader and asked him for his strain, he had used for the transformation. In this strain, Tfox is expressed under the control of an IPTG inducible promoter. It is not yet known how the natural Tfox is induced therefore, this method was resorted. 
After receiving the strain, we were able to start testing it to see if the method works in our hands. Before we start in the lab, several things have to be considered: Transformation is dependent on the length of homologous sequence. It was shown that the transformation efficiency is much higher with 3kb homologous regions than for example with 0.5kB but she is still high. In a cotransformation, however, it is important that the homologous regions are chosen to be slightly larger. There was a big difference between the efficiency of the transformation. For example, the efficiency is 6-fold lower with 1kb-flanks than with 3kb-flanks

<A href="https://www.ncbi.nlm.nih.gov/pubmed/28571309" target="blank"><ABBR title="Dalia TN, Hayes CA, Stolyar S, Marx CJ, McKinlay JB, Dalia AB. Multiplex Genome Editing by Natural Transformation (MuGENT) for Synthetic Biology in Vibrio natriegens. ACS Synth Biol. (2017) 15;6(9):1650-1655..">(Dalia<I>et al.</I>2017)</ABBR></A>
. Moreover, you have to remember that you have to make multiple iterative cycles to get the complexity of genome edits in the population 

<A href="https://www.ncbi.nlm.nih.gov/pubmed/24889608" target="blank"><ABBR title="Dalia, A. B., McDonough, E., and Camilli, A.. Multiplex genome editing by natural transformation. Proc. Natl. Acad. Sci. U. S. A. (2014) 111, 8937−8942.">(Dalia<I>et al.</I>2014)</ABBR></A>

. In each cycle it is important to change the antibiotic marker. An advantage is that the efficient in V. natriegens ist still high and you only need a low amount of the unselected product (25-50ng/108 CFU)  

<A href="https://www.ncbi.nlm.nih.gov/pubmed/28571309" target="blank"><ABBR title="Dalia TN, Hayes CA, Stolyar S, Marx CJ, McKinlay JB, Dalia AB. Multiplex Genome Editing by Natural Transformation (MuGENT) for Synthetic Biology in Vibrio natriegens. ACS Synth Biol. (2017) 15;6(9):1650-1655..">(Dalia<I>et al.</I>2017)</ABBR></A>.<FIGURE style="display:block; float: right; width: 30%;"><FIGCAPTION><B>Figure 1:</B>Schematic figure that shows how to build a cassette for integration. Amplify the upstream and downstream regions of the gene of interest and the antibiotic cassette with frt sites. These three Fragments will be fused.</FIGCAPTION></FIGURE>
To get in touch with the multiplex genome editing by natural transformation, we construct a linear DNA Fragment to transform. We have chosen an antibiotic resistance for selection, which has to be different to the resistance marker on the plasmid in the strain. Using a PCR we construct at first the chloramphenicol cassette with the frt sites and fuse this on the upstream and downstream regions of the gene we want to delete in order to integrate it by homologous recombination (Figure 1). In this case it was the dns. It was previously shown that cells with genetic modifications on this site did not developed unfavorable phenotype  

<A href="https://www.ncbi.nlm.nih.gov/pubmed/28571309"><ABBR title="Dalia TN, Hayes CA, Stolyar S, Marx CJ, McKinlay JB, Dalia AB. Multiplex Genome Editing by Natural Transformation (MuGENT) for Synthetic Biology in Vibrio natriegens. ACS Synth Biol. (2017) 15;6(9):1650-1655.">(Dalia<I>et al.</I>2017)</ABBR></A>. 

To use the strain after completion of the modifications it is important to get the plasmid out. At first we inoculate a culture without the antibiotic in order to lose the plasmid with the tfox gene. Every 12h 10μl were plated out and with 10µl a new culture is inoculate. By doing that we don´t get the plasmid out, so we have to think about another technique for getting rid of the plasmid. For that we looked up again into the literature. We could find, that Ankur B. Dalia wrote that the plasmid is very stable and you can use the strain without antibiotic. Then we contacted him again to pray for help. He told us that it was possible to get the plasmid out with another strategy. This makes us very relieved, because if the plasmid is still in the modified strain, which would be critical for using the strain for cloning, due to marker and Ori incompatibility.

With his proposal to grow a culture with the add of IPTG and than to screen for clones who are no longer carbenicillin resistance, we are able to get the pMMB-tfox Plasmid out of the strains.

After it was possible for us to use natural transformation we start to build our own plasmid. Therefore we use the Marburg Collection, to make this great part good and easy accessible for the scientific community. Therefore we build a level 0 and a level 1 Plasmid and integrated it into the wildtype of V. natriegens.


	<H3>The Flp/<I>frt</I> system</H3><P><FIGURE style="display:block; float: left; width: 30%;"><FIGCAPTION><B>Figure 2:</B>A Flp encoding plasmid. This plasmid is constructed using eight different parts. One of them is encoding the Flp recombinase additionally we used a constitutive promoter, a RBS, two connectors, a terminator and a backbone with a kanamycin resistance cassette.
</FIGCAPTION></FIGURE>
In order to establish the Flp/<I>frt</I> system in <I>V. natriegens</I> for fast genomic deletions two components are needed. First we need to introduce genomic FRT sites for the desired modifications and second a FLP recombinase needs to be introduced into the <I>V. natriegens</I> cells.
By attaching <I>frt</I> sites to the chloramphenicol resistance cassette that will be used for creating deletions, we will introduce the genomic FRT sites that are needed. Therefore, the resistance cassette from pYTK will be amplified using Primers with <I>frt</I> overhangs. Depending which kind of gene, we want to delete, the upstream and downstream homologous regions with a length of 500 to 3000 bp will also be amplified attaching <I>frt</I> overhangs to the upstream reverse and downstream forward primers. This way, upstream fragment, chloramphenicol resistance cassette and downstream fragment can be assembled to large linear fragments by performing fusion PCRs.
Using the natural transformation method we will transform these linear fragments to the <I>V. natriegens</I> cells. By homologous recombination, the fragments can integrate into the genome. This Insertion will be proved by colony PCR.
The second component for the Flp/<I>frt</I> system is introducing a functional FLP recombinase into the cells. We will involve the pBR-FLP plasmid that was used to establish the Flp/<I>frt</I> system in other Vibrio species like <I>V. cholerae</I><A href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3490321/"><ABBR title="Blokesch, M. (2012). TransFLP--a method to genetically modify Vibrio cholerae based on natural transformation and FLP-recombination. J Vis Exp(68).">(Blokesch 2012)</ABBR></A>

 into the Marburg toolbox  using primers with compatible overhangs. Assembling different parts of this toolbox to a plasmid with a full transcriptional unit, we will create a plasmid encoding the FLP recombinase. Because we already use all inducible promoters of the Marburg toolbox for inducing tfox or the T7 system , we decided to use a constitutive promoter for the <I>flp</I> recombinase.
The plasmid will be assembled using K2560007 as a constitutive promoter, K2560010 as an RBS, K2560011 and K2560012 as connectors, K2560271 as the coding sequence of FLP, K2560046 as an ori, K2560057 as backbone with a kanamycin resistance and K2560035 as a terminator.</P><P><FIGURE style="display:block; float: right; width: 30%;"><FIGCAPTION><B>Figure 3:</B>he process of Flp mediated excision. Flp is recognizing the <I>frt</I> sites that are in the same direction and is cutting the antibiotics cassette out, leaving a FRT scar behind.
</FIGCAPTION></FIGURE>
Introducing this plasmid into <I>V. natriegens</I> and expressing <I>flp</I>, the recombinase will recognize the integrated <I>frt</I> sites that are in the same orientation. By Flp mediated recombination, the resistance cassette will be cut out of the genome, creating the deletion of the desired gene.
Afterwards the strain will be cultivated without chloramphenicol and screened for chloramphenicol sensitive clones. The excision needs to be proved by colony PCR.
The next effort is to get rid of the Flp plasmid and its kanamycin resistance because we want our strains to be compatible with all your constructs.
Due to its high growth rate, <I>V. natriegens</I> loses plasmids, with some exceptions like pMMB-tfox 

<A href="”https://www.ncbi.nlm.nih.gov/pubmed/28571309”"><ABBR title="”Dalia," t.="" n.="">(Hayes <I>et al.</I>, 2017)</ABBR></A>

, very fast if there is no selection pressure by antibiotics in the medium. The Flp encoding plasmid will be selected out, cultivating the created strain without kanamycin and screening for kanamycin sensitive colonies on plate. A colony PCR, negative for Flp will prove the loss of the plasmid and the strain is ready for the usage.
</P></DIV></DIV><DIV class="collapsible"><DIV class="btn_expand">Integration</DIV><DIV class="content"><P>
To integrate a construct into the genome it is necessary to find suitable integration sites. Here for there are big differences and several factors to consider. The most basic aspect is that a non-coding sequence is necessary, so no possibly essential genes are disrupted. Secondly it is imported to consider that integration sites vary in their transcription levels for multiple reasons

<A href="https://www.sciencedirect.com/science/article/pii/S1386653214003060"><ABBR title="(Dean Bryant, Tiffany Onions, Rachel Raybould, Sadie Jones, Amanda Tristram, Samantha Hibbitts, Alison Fiander and Ned Powell:  Increased methylation of Human Papillomavirus type 16 DNA correlates with viral integration in Vulval Intraepithelial Neoplasia, (2014); 61(3), pp. 393-399">(Bryant et al., 2014) </ABBR></A>
.</P><H3>Choosing an Integration site</H3><P>
We searched manually for integration sites with low gene expression, which usually are between two terminators, because it was shown that there appears less readthrough. Another criterion was that there is more than 1000 bp non-coding space. Because we only could detect two places on the chromosome. To bypass screening through the genome manually, which is time consuming and less flexible if the search is repeated with different parameters, we created a Mathlab program script. With this we tried to find regions in the chromosomes of &lt;i&gt;V. natriegens&lt;/i&gt; with opposing transcription units and at least 500 bp between adjacent CDS. In order for the Mathlab program to work we imported &lt;i&gt;V. natriegens’&lt;/i&gt; two chromosomes, both as .gb and. fasta. The .gb file is used to get information about the location and orientation of all CDS in the genome. Using if-statements, we are able to identify regions that fulfill both required conditions: opposing CDS and an intergenic region larger than 500 bp. This resulted in 24 integration sites, of which 7 were not suitable because they had essential tRNA coding regions which were not detected by the Mathlab program.</P><DIV style="overflow-x: scroll;"><TABLE style="width: 100%"><TBODY><TR><TH>sequence</TH><TH>chromosome</TH><TH>position (1000 bp flanks)</TH><TH>distance</TH><TH>left CDS gene</TH><TH>left CDS function</TH><TH>right CDS gene</TH><TH>right CDS function</TH></TR><TR><TD>1</TD><TD>1</TD><TD>2734414 - 2736414</TD><TD>3502</TD><TD>mcp4</TD><TD>chemokine</TD><TD>rpfG3</TD><TD>Cyclic di-GMP phosphodiesterase response regulator </TD><TD>tRNAs + rRNAs</TD></TR><TR><TD>2</TD><TD>1</TD><TD>742810 - 744810</TD><TD>2906</TD><TD>ubiF</TD><TD>2-octaprenyl-3-methyl-6-methoxy-1,4-benzoquinol oxygenase</TD><TD>ychF</TD><TD>Ribosome-binding ATPase</TD><TD>tRNAs</TD></TR><TR><TD>3</TD><TD>1</TD><TD>3001118 - 3003118</TD><TD>1913</TD><TD>hypothetical</TD><TD>cadC2</TD><TD>Cadmium efflux system accessory protein</TD><TD>tRNAs</TD></TR><TR><TD>4</TD><TD>1</TD><TD>2927926 - 2929926</TD><TD>1872</TD><TD>hypothetical</TD><TD>tRNA epoxyqueuosine(34) reductase QueG</TD><TD>orn</TD><TD>Oligoribonuclease</TD><TD>tRNAs</TD></TR><TR><TD>5</TD><TD>1</TD><TD>940901 - 942901</TD><TD>973</TD><TD>hypothetical</TD><TD>mqo2</TD><TD>Probable malate:quinone oxidoreductase 2</TD><TD>tRNAs</TD></TR><TR><TD>6</TD><TD>1</TD><TD>3086570 - 3088570</TD><TD>908</TD><TD>hypothetical</TD><TD>fre</TD><TD>NAD(P)H-flavin reductase</TD><TD>tRNAs</TD></TR><TR><TD>7</TD><TD>1</TD><TD>1640886 - 1642886</TD><TD>856</TD><TD>luxQ3</TD><TD>Histidine kinase</TD><TD>hypothetical</TD></TR><TR><TD>8</TD><TD>1</TD><TD>84578 - 86578</TD><TD>756</TD><TD>polA</TD><TD>DNA polymerase I </TD><TD>engB</TD><TD>Probable GTP-binding protein EngB</TD></TR><TR><TD>9</TD><TD>1</TD><TD>1546674 - 1548674</TD><TD>729</TD><TD>hupR1 2</TD><TD>Hydrogenase transcriptional regulatory</TD><TD>pcpR 3</TD><TD>PCP degradation transcriptional activation protein</TD></TR><TR><TD>10</TD><TD>1</TD><TD>662952 - 664952</TD><TD>699</TD><TD>tadA</TD><TD>tRNA-specific adenosine deaminase</TD><TD>mltF 2</TD><TD>Membrane-bound lytic murein transglycosylase F</TD></TR><TR><TD>11</TD><TD>1</TD><TD>3103970 - 3105970</TD><TD>680</TD><TD>ggt</TD><TD>Glutathione hydrolase proenzyme</TD><TD>uvrD</TD><TD>DNA helicase II</TD></TR><TR><TD>12</TD><TD>1</TD><TD>2271574 - 2273574</TD><TD>668</TD><TD>putative ABC transporter binding protein</TD><TD>hypothetical</TD></TR><TR><TD>13</TD><TD>1</TD><TD>645351 - 647351</TD><TD>656</TD><TD>nreC</TD><TD>Oxygen regulatory protein (Staphylococcus)/ Ribonuclease 3(E. coli) </TD><TD>smpB</TD><TD>SsrA-binding protein</TD><TD>ssrA tmRNA</TD></TR><TR><TD>14</TD><TD>1</TD><TD>1457658 - 1459658</TD><TD>568</TD><TD>lamB2</TD><TD>Maltoporin 2 </TD><TD>panF</TD><TD>sodium/pantothenate symporter</TD></TR><TR><TD>15</TD><TD>1</TD><TD>2955169 - 2957169</TD><TD>552</TD><TD>tnpR</TD><TD>Transposon gamma-delta resolvase</TD><TD>hypothetical</TD></TR><TR><TD>16</TD><TD>2</TD><TD>192324 - 194324</TD><TD>823</TD><TD>hypothetical</TD><TD>pdeB 1</TD><TD>vinculin</TD><TD>tRNAs</TD></TR><TR><TD>17</TD><TD>2</TD><TD>907620 - 909620</TD><TD>679</TD><TD>hypothetical</TD><TD>glxR</TD><TD>2-hydroxy-3-oxopropionate reductase</TD></TR><TR><TD>18</TD><TD>2</TD><TD>1441789 - 1443789</TD><TD>659</TD><TD>hypothetical</TD><TD>putative phosphatase</TD></TR><TR><TD>19</TD><TD>2</TD><TD>1335027 - 1337027</TD><TD>568</TD><TD>hypothetical</TD><TD>hypothetical</TD></TR><TR><TD>20</TD><TD>2</TD><TD>1397103 - 1399103</TD><TD>560</TD><TD>rppH2</TD><TD>RNA pyrophosphohydrolase</TD><TD>hypothetical</TD></TR><TR><TD>21</TD><TD>2</TD><TD>415089 - 417089</TD><TD>558</TD><TD>hypothetical</TD><TD>hypothetical</TD></TR><TR><TD>22</TD><TD>2</TD><TD>410337 - 412337</TD><TD>550</TD><TD>pcpR 1</TD><TD>PCP degradation transcriptional activation protein</TD><TD>mlhB</TD><TD>Monoterpene epsilon-lactone hydrolase</TD></TR><TR><TD>23</TD><TD>2</TD><TD>59335  - 61335</TD><TD>513</TD><TD>rbsR</TD><TD>Ribose operon repressor</TD><TD>cueR</TD><TD>HTH-type transcriptional regulator</TD></TR><TR><TD>24</TD><TD>2</TD><TD>1544521 - 1546521</TD><TD>510</TD><TD>yusO</TD><TD>DNA binding transcription factor</TD><TD>yccF</TD><TD>Inner membrane protein YccF</TD></TR></TBODY></TABLE></DIV><P>
Finally, the middle between both neighboring CDS is calculated and a DNA sequence is drawn from the fasta files which includes 1000 bp both-sided and saved, representing the homology flanks. This sequence can either be used to design PCR primers for amplification of the homology flanks directly or for creating an alignment with the genome sequence to check for the local genomic context.</P><P>
	An integration can affect the genes next to it. Because of this it generally is unsuitable to integrate next to essential genes. This made it necessary to characterize the genes next to our integration sites. The sequence 8 is surrounded by polA, coding the DNA-polymerase I and the GTP binding protein, which is involved in the normal cell division and septation. His depletion leads to a reduction in growth rate and extensive filamentation with a block beyond the stage of segregation.</P><P>
	Behind the possible integration site 9 lies hupR1, which codes for a hydrogenase transcriptional regulatory protein, a member of the two-component regulatory system ZraS/ZraR and pcpR, coding for the PCP degradation transcriptional activation protein.</P><P>
	In front of sequence 10 is the tadA gene, which codes for the tRNA-specific adenosine deaminase. This protein is essential for cell viability. After sequence 10 lies mltF2, coding for the Membrane-bound lytic murein transglycosylase F, which degrades the murein glycan strands and sacculi.</P><P>
	Sequence 11 is surrounded by ggt, which codes for the Glutathione hydrolase proenzyme, involved in the <A href="https://www.ebi.ac.uk/QuickGO/term/GO:0006750">glutathione biosynthetic process</A> and the uvrD gene, encoding the DNA helicase II, involved in the DNA replication.</P><P>
	The sequence before sequence 14 is lamB2, coding for Maltoporin 2. This protein is involved in the ion transport.  After the sequence 14 comes panF, catalysing the sodium-dependent uptake of extracellular pantothenate.</P><P>
	The sequences on the chromosome 2 in front of the sequences 16, 17 and 18 code for hypothetical proteins of unknown function. Sequence 21 is even surrounded by two hypothetical proteins. The gene pdeB 1 after the sequence 16 encodes a DNA helicase II, like uvrD. Behind sequence 17 lies glxR, coding for 2-hydroxy-3-oxopropionate reductase. This protein is involved in the catabolic process of glycolate and glyoxylate and to the <A href="https://www.ebi.ac.uk/QuickGO/term/GO:0006974">cellular response to DNA damage</A> respectively. The sequence 18 is followed by a putative phosphatase. These enzymes use water to cleave a <A href="https://en.wikipedia.org/wiki/Phosphoric_acid">phosphoric acid</A><A href="https://en.wikipedia.org/wiki/Ester">monoester</A> into a <A href="https://en.wikipedia.org/wiki/Phosphate">phosphate ion</A> and an <A href="https://en.wikipedia.org/wiki/Alcohol">alcohol</A> and are therefore essential for many biological processes.</P><P>
	The penultimate sequence 23 has a rbs gene in front of it, coding for a ribose operon repressor. Behind the sequence lies cueR, an HTH-type transcriptional regulator. This protein <A href="https://www.ebi.ac.uk/QuickGO/term/GO:0045913">positively regulates the carbohydrate metabolic process</A>.</P><P>
	In front of the sequence 24 lies yusO, coding for an uncharacterized HTH-type transcriptional regulator. The sequence is followed by yccF, whichs inner membrane protein YccF is part of the plasma membrane.</P><H3>Characterizing different transcription levels of integration sits</H3><P>In order to characterize the different integration sits the transcription level at each has to be measured. Transcription levels can be determined by measuring β-galactosidase’s activity. We designed our experiment so that we first <A href="https://2018.igem.org/Team:Marburg/Experiments">transform</A> a high copy plasmid into<EM> Vibrio natriegens </EM>and then the lacZ α fragment into the genome trough <A href="https://2018.igem.org/Team:Marburg/Experiments">natural competence</A>.The lacZ α fragment is inserted into different chromosomal integration sites. The lacZ α fragment must complement the lacZ on the plasmid for there to be any β-galactosidase activity. By measuring the β-galactosidase’s activity it is possible to draw conclusions about the transcription level at the different integration sites.Because the lac Z α fragment should be the limiting factor for the enzyme complex, as only one copy of it is present in the cells from the genome and there are multiple copies of the rest of the <EM>lac</EM> operon stemming from the multiple copies of the plasmids in the cells. To quantify the amount of functional β-galactosidase, pre-experiments with a defined amount of β-galactosidase should be conducted, and the amount of processed ONPG at the end of the experiment measured. By comparing the data form the pre-experiments with the β-galactosidase activity of the different integration sites the transcription levels of the integration sites can be determined.</P>


As described previously, we tried to find regions in the chromosomes of <I>V. natriegens</I> with opposing transcription units and at least 500 bp between neighbouring CDS. 
To omit screening through the genome manually, which is time consuming and less flexible if the search is repeated with different parameters, we created a Matlab program to automate this process.<I>V. natriegens'</I> two chromosomes are imported both as .gb and .fasta. The .gb file is used to get information about the location and orientation of all CDS in the genome. Using if-statements, we are able to identify regions that fulfil both required conditions: opposing CDS and spacing larger than 500 bp. 

Finally, the middle between both neighbouring CDS is calculated and a DNA sequence is drawn from the fasta files which includes 1000 bp both-sided and saved, representing the homology flanks. This sequence can either directly be used for designing PCR primers for amplification of the homology flanks or used for creating an alignment with the genome sequence to check for the local genomic context. <DIV class="collapsible"><DIV class="btn_expand">Integration sites Mathlab</DIV><DIV class="content"><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: black;">clear </SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: #a020f0;">all</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: black;">clc</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: forestgreen;">%% Set desired properties for integration sites</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: black;">Required_distance = 500;</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: black;">Homology = 1000;</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: forestgreen;">%% Import and reshape Data</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: forestgreen;"> </SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: black;">Chr1 = genbankread(</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: #a020f0;">'Chr1.gb'</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: black;">);</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: black;">Chr2 = genbankread(</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: #a020f0;">'Chr2.gb'</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: black;">);</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: black;">Chr1_Fasta = fastaread(</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: #a020f0;">'Chr1.fasta'</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: black;">);</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: black;">Chr2_Fasta = fastaread(</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: #a020f0;">'Chr2.fasta'</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: black;">);</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: black;"> </SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: black;">PositionsChr1 = {Chr1.CDS.indices};</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: black;">PositionsChr1 = reshape(PositionsChr1,size(PositionsChr1,2),1);</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: black;"> </SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: black;">IndicesChr1 = zeros(size(PositionsChr1,1),6);</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: blue;">for</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: black;"> k = 1: size(PositionsChr1,1) </SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: black;">   IndicesChr1(k,1) = PositionsChr1{k,1}(1,1);</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: black;">   IndicesChr1(k,2) = PositionsChr1{k,1}(1,2);</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: blue;">end</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: blue;"> </SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: forestgreen;">%% Check for orientation</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: forestgreen;">% Insert 1 in column 3 if CDS is reversed</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: blue;">for</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: black;"> j = 1: size(IndicesChr1,1)</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: black;">   </SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: blue;">if</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: black;"> IndicesChr1(j,1) &gt; IndicesChr1(j,2)</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: black;">        IndicesChr1(j,3) = 1;</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: black;">   </SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: blue;">end</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: blue;">end</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: blue;"> </SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: forestgreen;">%% Check for distance from one CDS to next CDS</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: blue;">for</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: black;"> p = 1: size(IndicesChr1,1)-1</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: black;">    IndicesChr1(p,4) = min(IndicesChr1(p+1,1:2))-max(IndicesChr1(p,1:2));</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: blue;">end</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: blue;"> </SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: forestgreen;">%% Check for good candidates</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: black;">HitsChr1 = [];</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: blue;">for</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: black;"> q = 1: size(IndicesChr1,1)-1</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: black;">   </SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: blue;">if</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: black;"> IndicesChr1(q,3) == 0 &amp;&amp; IndicesChr1(q+1,3) == 1 &amp;&amp;</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: blue;">...</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: black;">       IndicesChr1(q,4) &gt;= Required_distance</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: black;">  </SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: black;">       IndicesChr1(q,5) = 1;</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: black;">       IndicesChr1(q,6) = round(max(IndicesChr1(q,1:2))+</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: blue;">...</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: black;">      </SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: black;">(IndicesChr1(q,4)/2));</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: black;">  </SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: black;">       HitsChr1 = [HitsChr1; IndicesChr1(q,:)];</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: black;">   </SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: blue;">end</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: blue;">end</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: black;">HitsChr1 = sortrows(HitsChr1,4,</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: #a020f0;">'descend'</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: black;">);</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: forestgreen;">%% Same for Chromosome 2</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: black;">PositionsChr2 = {Chr2.CDS.indices};</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: black;">PositionsChr2 = reshape(PositionsChr2,size(PositionsChr2,2),1);</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: black;"> </SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: black;">IndicesChr2 = zeros(size(PositionsChr2,1),5);</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: blue;">for</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: black;"> k = 1: size(PositionsChr2,1) </SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: black;">   IndicesChr2(k,1) = PositionsChr2{k,1}(1,1);</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: black;">   IndicesChr2(k,2) = PositionsChr2{k,1}(1,2);</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: blue;">end</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: blue;"> </SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: forestgreen;">%% Check for orientation</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: forestgreen;">% Insert 1 in column 3 if CDS is reversed</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: blue;">for</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: black;"> j = 1: size(IndicesChr2,1)</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: black;">   </SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: blue;">if</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: black;"> IndicesChr2(j,1) &gt; IndicesChr2(j,2)</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: black;">        IndicesChr2(j,3) = 1;</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: black;">   </SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: blue;">end</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: blue;">end</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: blue;"> </SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: forestgreen;">%% Check for distance from one CDS to next CDS</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: blue;">for</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: black;"> p = 1: size(IndicesChr2,1)-1</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: black;">    IndicesChr2(p,4) = min(IndicesChr2(p+1,1:2))-max(IndicesChr2(p,1:2));</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: blue;">end</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: blue;"> </SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: forestgreen;">%% Check for good candidates</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: black;">HitsChr2 = [];</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: blue;">for</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: black;"> q = 1: size(IndicesChr2,1)-1</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: black;">   </SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: blue;">if</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: black;"> IndicesChr2(q,3) == 0 &amp;&amp; IndicesChr2(q+1,3) == 1 &amp;&amp;</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: blue;">...</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: black;">      IndicesChr2(q,4) &gt;= Required_distance</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: black;">  </SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: black;">       IndicesChr2(q,5) = 1;      </SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: black;">       IndicesChr2(q,6) = round(max(IndicesChr2(q,1:2))+(</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: blue;">...</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: black;">      </SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: black;">IndicesChr2(q,4)/2));</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: black;">       HitsChr2 = [HitsChr2; IndicesChr2(q,:)];</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: black;">   </SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: blue;">end</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: blue;">end</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: black;">HitsChr2 = sortrows(HitsChr2,4,</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: #a020f0;">'descend'</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: black;">);</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: black;">Hits = sum(IndicesChr1(:,5)) + sum(IndicesChr2(:,5));</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: black;"> </SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: forestgreen;">%% Determine homologous regions and save as FASTA</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: forestgreen;"> </SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: blue;">for</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: black;"> f = 1 : size(HitsChr1,1)</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: black;">    Range = [HitsChr1(f,6)-Homology, HitsChr1(f,6)+Homology];</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: black;">    Sequence = Chr1_Fasta.Sequence(Range(1,1):Range(1,2)-1);</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: black;">    Fasta(f).Sequence = Sequence;</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: black;">    Fasta(f).Header = [</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: #a020f0;">'Sequence:'</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: black;">,num2str(f),</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: #a020f0;">'  Chromosome1'</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: black;">,</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: blue;">...</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: black;">   </SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: #a020f0;">'  Position:'</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: black;">, num2str(Range(1,1)), </SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: #a020f0;">'-'</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: black;">, num2str(Range(1,2))];</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: black;">  </SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: blue;">end</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: blue;"> </SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: blue;">for</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: black;"> h = 1 : size(HitsChr2,1)</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: black;">    Range = [HitsChr2(h,6)-Homology, HitsChr2(h,6)+Homology];</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: black;">    Sequence = Chr2_Fasta.Sequence(Range(1,1):Range(1,2)-1);</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: black;">    Fasta(f+h).Sequence = Sequence;</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: black;">    Fasta(f+h).Header = [</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: #a020f0;">'Sequence:'</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: black;">,num2str(f+h), </SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: #a020f0;">'  Chromosome2'</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: black;">,</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: blue;">...</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: black;">   </SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: #a020f0;">'Position:'</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: black;">, num2str(Range(1,1)), </SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: #a020f0;">'-'</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: black;">, num2str(Range(1,2))];</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: black;">  </SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: blue;">end</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: black;">Title = [</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: #a020f0;">'Integrationsites, min Distance ='</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: black;">, num2str(Required_distance),</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: blue;">...</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: #a020f0;">'  Homology='</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: black;">,num2str(Homology),</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: #a020f0;">'.txt'</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: black;">];</SPAN><SPAN style="font-size: 10.0pt; font-family: 'Courier New'; color: black;"> fastawrite(Title,Fasta)</SPAN></DIV></DIV><H2>Integration Sites</H2><TABLE style="width:100%"><TBODY><TR><TH><P><B>Sequence name: seq_12</B>
Chromosome number: 1
Distance to the ORI: 2271574 bp
Upstream neighbouring gene function: A putative ABC transporter binding protein, which has an ATPase activity and can bind ATP. The ATP-binding cassette transporter superfamily is the largest gene family. They are responsible for the transport across the lipid membrane, which is an essential function of all living organism.
Downstream neighbouring gene function: unknown (putative protein)</P></TH></TR><TR><TH><B>Sequence name: seq_15</B>
Chromosome number: 1
Distance to the ORI: 2955169 bp
Upstream neighbouring gene function: The transposon resolvase catalyses the resolution, a site-specific recombinase, of cointegrated replicon to yield the final transposition products.
Downstream neighbouring gene function: unknown (putative protein)</TH></TR><TR><TH><B>Sequence name: seq_19</B>
Chromosome number: 2
Distance to the ORI: 1335027 bp
Upstream neighbouring gene function: unknown (putative protein)
Downstream neighbouring gene function: unknown (putative protein)</TH></TR><TR><TH><B>Sequence name: seq_20</B>
Chromosome number: 2
Distance to the ORI: 1397103 bp
Upstream neighbouring gene function: The RNA pyrophosphohydrolase accelerates the degradation of transcripts by removing pyrophosphate from the 5énd of the triphosphorylated RNA. This leads to a more labile monophosphorylated state, which stimulates subsequent ribonuclease cleavage.
Downstream neighbouring gene function: unknown (putative protein)</TH></TR><TR><TH><B>Sequence name: seq_22</B>
Chromosome number: 2
Distance to the ORI: 410337 bp
Upstream neighbouring gene function: The PCP degradation transcriptional activation protein is responsible for the transcription regulation of the pcpA, pcpB and pcpE genes. These genes are essential for the pentachlorophenol (PCP) degradation. The organochloride PCP is used as a pesticide or disinfectant and only few bacteria can biodegradate it.
Downstream neighbouring gene function: The Monoterpene episolactone hydrolase is responsible for the degradation of monocyclic monoterpenes such as menthol and carvone. It catalyses the ring opening of lactones during the degradation of monocyclic monoterpenes.</TH></TR></TBODY></TABLE><A style="left:2%;" href="https://static.igem.org/mediawiki/2018/d/d9/T--Marburg--5_documents_from_Integration_sites.zip">DOWNLOAD FILE</A></DIV></DIV><DIV class="collapsible"><DIV class="btn_expand">Strains</DIV><DIV class="content"><P>There are different <I>E. coli</I> strains for a variety of applications. In every lab diverse strains are stored in freezers for methods like cloning, protein expression and protein interaction. And all of them are being used. For our project Vibrigens, we decided to establish the world’s fastest three lab strains suited for specific needs based on our <I>V. natriegens</I> wildtype strain. Here we describe how we designed our three strains VibriClone, VibriExpress and VibriInteract based on the genotype of  commonly used <I>E. coli</I> strains.</P><DIV class="collapsible"><DIV class="btn_expand">VibriClone</DIV><DIV class="content"><DIV><P>
 	To create a cloning strain that fulfills all the wishes of the scientific society, we have to think of a good strategy. A cloning strain is a kind of a tool in the laboratory day. We asked ourselves which features we needed urgently. The currently used chassis for cloning  is E. coli. We started to look in the genotype, to see what modifications are important. Then we think about which one we would like to take over. Our choice was to knock out the exonucleases so that it would be possible to transform DNA without degrading. In addition, we would like to insert a point mutation in recA, that the homologous recombination between plasmids and into the genome. Moreover, we also want to make it possible to store in the fridge. Therefore we insert the katG gene encodes an enzyme with an catalase and peroxidase activity into the genome. Furthermore, we also wanted to make the blue-and-white selection possible for our VibriClone, we can achieve that by inserting the lactose Operon in the genome.
</P></DIV><H3>Exonucleases</H3>
In the cloning strain of E. coli, the exonuclease EndA was knocked out. This is very important for such a strain, otherwise the exonucleases degrade the DNA to be transformed. EndA makes double-strand breaks in duplex DNA und is the most abundant DNase activity in E. coli  

<A href="https://www.ncbi.nlm.nih.gov/pubmed/4938550"><ABBR title="Dürwald, H. and Hoffmann-Berling, H., Endonuclease-I-deficient and ribonuclease I-deficient Escherichia coli mutants., J Mol Biol. (1968) 14;34(2):331-46.">(Dürwald<I>et al.</I>1968)</ABBR></A>. In the present of the exonuclease no colonies were formed during transformation <A href="https://www.ncbi.nlm.nih.gov/pubmed/20308163"><ABBR title="Kaneko, S., and Itaya, M., Integration of stable extracellular DNA released from Escherichia coli into the Bacillus subtilis genome vector by culture mix method., Nucleic Acids Res. (2010); 38(8):2551-7.">(Kaneko<I>et al.</I>2010)</ABBR></A>

. It was recognized that by only the complete loss of host endA there were no degradation after 30h at 39°C. In addition, it was also found out that it creates a structural stability to give transformants and that higher molecular weight genomic DNA bands were recovered from the supernatant and it leads to certain discrete bands in digestion  

<A href="https://www.ncbi.nlm.nih.gov/pubmed/23055537"><ABBR title="Itaya M, Kawata Y, Sato M, Tomita M, Nakahigashi K., A simple method to provide a shuttling plasmid for delivery to other host ascertained by prolonged stability of extracellular plasmid DNA released from Escherichia coli K12 endA mutant, deficient in major endonuclease., J Biochem. (2012);152(6):501-4.">(Itaya<I>et al.</I>2012)</ABBR></A>

.
V. cholerae has two extracellular nucleases: Xds and Dns. The Dns is homologous to EndA from E. coli. Both proteins have been shown to be secreted into the medium. These two would degrade the free DNA, which is the source of a natural transformation. Because of this the method would not work or work very poorly. To enable the uptake of intact DNA during the competency process, the functions would have to be extinguished. It was shown that when you delete the nucleases, the transformation process works well. In a mutant in which the dns was knocked out, transformation frequency is more than hundred times higher than WT and when xds was knocked out the transformation frequency was 2,5-fold increased. This result showed that it is good if both nucleases are knocked out, but that the dns is the most important one 

<A href="https://www.ncbi.nlm.nih.gov/pubmed/23055537"><ABBR title="Itaya M, Kawata Y, Sato M, Tomita M, Nakahigashi K., A simple method to provide a shuttling plasmid for delivery to other host ascertained by prolonged stability of extracellular plasmid DNA released from Escherichia coli K12 endA mutant, deficient in major endonuclease., J Biochem. (2012);152(6):501-4.">(Itaya<I>et al.</I>2012)</ABBR></A>

. In addition, you get through the deletion of the two nucleases a cleaner preparations of DNA.
We wanted our cloning strain to have all the benefits, so it was very important to us that the exonucleases be knocked out. Otherwise, transformation efficiency would be very low and you don´t get clear preparations of DNA and that would not be great for a cloning strain. We started by searching for homologous genes of endA of E. coli and dns and xds of V. cholera. We were able to find out by blast analysis that in Vibrio natriegens there are two exonucleases homologous to those of E. coli and V. cholera. They are called Dns and ExeM.

<FIGURE style="display: center; width: 80%;"><FIGCAPTION><B>Figure 1:</B>Alignment of dns with the exonucleases dns from V. cholerae and endA from E. coli. The alignment was created with the program ClustalX 2.0.10.
</FIGCAPTION></FIGURE>

In the alignment of the V. natriegens dns with the V. cholerae dns and E. coli endA you can conclude that the three proteins are highly conserved among the three strains. The intermediate region and the C-terminus are highly conserved in all three proteins, while the N-terminus is variable. This suggests that the parts that are highly conserved are important for the function of nucleases. It can be concluded that the dns of V. natriegens has a very similar function to the two exonucleases of the other two organisms.

<FIGURE style="display: center; width: 80%;"><FIGCAPTION><B>Figure 2:</B>Alignment of exeM with the exonucleases xds from V. cholerae. The alignment was created with the program ClustalX 2.0.10.
</FIGCAPTION></FIGURE>

The most parts of the two proteins ExeM from V. natriegens and Xds from V. cholerae are conserved, but both the N- and the C-Terminal vary a lot from each other. This suggests that the function of these two proteins has a similar function. Thus we can assume that it is very probable that the ExeM protein in V. natriegens behaves like the Xds from V. cholera. Because of this it is very important to delete this protein otherwise the DNA would be degraded. 

In 2016, Matthew T Weinstock showed that the quality of isolated plasmid DNA could be improved by deleting dns in V. natriegens 

<A href="https://www.ncbi.nlm.nih.gov/pubmed/27571549"><ABBR title="Weinstock MT, Hesek ED, Wilson CM, Gibson DG., Vibrio natriegens as a fast-growing host for molecular biology., J Nat Methods. (2016);13(10):849-51">(Weinstock<I>et al.</I>2016)</ABBR></A>

. In addition, the growth and viability of cells with a deletion of dns in rich medium should not be impaired  

<A href="https://www.ncbi.nlm.nih.gov/pubmed/28571309"><ABBR title="Dalia TN, Hayes CA, Stolyar S, Marx CJ, McKinlay JB, Dalia AB. Multiplex Genome Editing by Natural Transformation (MuGENT) for Synthetic Biology in Vibrio natriegens. ACS Synth Biol. (2017) 15;6(9):1650-1655..">(Dalia<I>et al.</I>2017)</ABBR></A>

. 

<P>After we made all these thoughts, we were able to start and plan the deletion of the exonucleases Dns and ExeM. We constructed cassettes to erase the genes. To first get to know the method and to check whether the natural transformation works as it has already been shown in our hands, we have started to knock out the dns. For this we constructed a cassette with a chloramphenicol resistance and the upstream and downstream region of the dns gene. To use the natural transformation it is enough to use 1000bp flanks but for the co transformation it is much better to use 3000bo flanks. By using Frt sequences between die single fragments, the antibiotics cassette can be excised from the flippase after integration. Because this knockout worked, we started planning for VibriClone. Our decision was made to perform the deletions with co-transformations. This was a big advantage because you only need one selected product. As selected product we chose the dns, due to the fact, it was shown that a deletion and insertion of an antibiotics cassette at this locus is not harmfulto the cells. This has not been shown for the other parts. The deletion of exeM is done by an unselected product. Therefore we create the upstream and downstream regions of this gene and take another gene that we would like to insert (katG) and build an unselected cassette by using fusion PCRs.
With this procedure, we are able to knock out the exonucleases and thus obtain a higher transformation efficiency.</P><P><FIGURE style="display: center; width: 40%; float: left;"><FIGCAPTION><B>Figure 3:</B>Sequence of recA. With a zoom in the region of the point mutation. 
At position 158  in recA of V. natriegens is a glycine. Through an alignment, we have found out that this is homologous to that in E. coli. To get the same kind of function loss, this glyin needs to be changed to an aspartic acid</FIGCAPTION></FIGURE></P><H3>RecA</H3>
The classic allosterically regulated enzyme RecA is involved in the regulation of the SOS System and the homologous recombination 

<A href="https://www.ncbi.nlm.nih.gov/pubmed/14693725"><ABBR title="McGrew DA, Knight KL. Molecular design and functional organization of the RecA protein. Crit Rev Biochem Mol Biol. (2003);38(5):385-432.">(McGrew<I>et al.</I>2003)</ABBR></A>

. If you think about what you need for a cloning strain, it quickly comes to your mind that homologous recombination in the genome and between plasmids is not only unnecessary but also undesirable.

In E. coli, a so-called recA1 mutation was performed. This mutation limits the recombination of an introduced plasmid with the genome and between plasmids thereby the stability of the plasmids are ensured. Another advantage is that the exogenous plasmid DNA is left unchanged by the modification of RecA. This allows the purification of this bacterial culture plasmid to allow high fidelity PCR amplification of the original plasmid sequence. 
The modification in this gene was a point mutation from G to A at position 720. Thereby the glycin at the place 160 chance to aspartic acid. We use this model to modify our strain VibriClone. 
At position 158  in recA of V. natriegens is a glycine. Through an alignment, we have found out that this is homologous to that in E. coli. To get the same kind of function loss, this glyin needs to be changed to an aspartic acid. 
For the modification of recA we build an unselected cassette from the gene with a point mutation. To achieve that two primers bind at the future mutation place, one forward and one reverse were created. In addition two more primers with a gap of 3000 bp to the other both were ordered. The results are two fragments which end at the point mutation. By a fusion PCR with only the outside primers, you get the hole fragment containing the recA with the desired mutation.
All in all with these mutation we make VibriClone fine for all of us to use for cloning without recombineering of an introduced plasmid with the genome and between two plasmids.<H3>KatG</H3>
Another feature, which would improve the applicability of V. natriegens as a cloning chassis, would be to engineer cold resistance, due to fact that we all are used to store our plates of E.coli in the fridge. It has been shown for other vibrio species that cold sensitivity is triggered by toxic peroxide present in media, which is normally removed  for example in E.coli by reactive oxygen species detoxification systems. Therefore our approach is to introduce such a system into our cloning strain, in order to transfer that tolerance, which enables keeping the working procedures from E.coli. It was shown, that catalase supplemented on agar plates could improve the viability under cold conditions. Nevertheless a more usable approach would be to include this tolerance within the strain itself. Several approaches exist to realize this goal, for instance increasing the glutathione pool within the cell or by introducing a plasmid  with an encoded catalase, such as KatG, KatE or the ahpCF operon.
We decided on introducing KatG into the genome, due to previous results, which were reported by Weinstock et al. They could show that KatG heterologous expressed from plasmid extends the time for fridge storage up to 3 weeks, which would suit for the needs for a cloning host. For integrating the katG into the genome, we combined introducing katG with the deletion of the second nuclease by utilizing 3000 bp homologous overhangs of the upstream and downstream flanking genomic region of exeM. These overhangs are fused to the katG transcription unit, that is amplified from the E.coli DH5𝛼 genome, by fusion PCR and subsequently transformed via natural transformation as a cotransformation.<H3>The <I>lac</I> operon</H3><FIGURE style="display: center; width: 30%; float:left;"><FIGCAPTION><B>Figure 4:</B>Alignment of lacI from <I>V. natriegens</I> with <I>lacI</I> from E. coli. Alignment of <I>lacZ</I> from <I>V. natriegens</I> with <I>lacZ</I> from <I>E. coli</I>. Alignment of <I>lacA</I> from <I>V. natriegens</I> with <I>lacA</I> from <I>E. coli</I>. The alignment was created with the program ClustalX 2.0.10.</FIGCAPTION></FIGURE>
The lac operon of E. coli consists of the three genes lacI, lacA and lacY. In the cloning strain, the fact that the β-galactokinase cleaves X-Gal into a blue dye became apparent. The lacZ gene encodes the β-galactokinase which consists of two parts: the lacZ-alpha and the lacZ-omega. It has been shown that if the alpha fragment was deleted, the omega fragment is not functional. By introduced the alpha fragment on a plasmid, alpha-complementation occurs.
We also wanted to make the blue-and-white selection possible for our VibriClone, as it is in the DH5𝛼 strain of E. coli. Therefore we checked at first which lac genes are present in the wildtype of V. natriegens. We found the lacA gene on chromosom 1 and the lacZ and lacI gene on chromosom 2.
As you can see, several parts are missing or different in all three proteins (Figure 4). What indicates that the lac genes are not functional in V. natriegens, which makes sense as lactose can not be found in its natural habitat and therefore genes for lactose metabolism are not needed and no selection pressure is present. That's the reason why we decided to amplify the lac Operon from DH5𝛼 that has no lacZ alpha subunit. 
	<FIGURE style="width: 80%;"><FIGCAPTION><B>Figure 5:</B>Alignment of <I>lacZ</I> with the alpha unit of <I>lacZ</I> from <I>E. coli</I>. The alignment was created with the program ClustalX 2.0.10.</FIGCAPTION></FIGURE><P>
Next we considered whether the wildtype lacZ from V. natriegens has a functional alpha subunit. If it exists, the 𝛼-complementation would not work anymore. To check this, we aligned the alpha subunit with the lacZ gene of V. natriegens (Figure 5). You can see that many parts of the subunit are missing, but a few sequences are conserved. To avoid that it can come to a complementation, we wanted to delete the part in our VibriClone. Since it appears that the entire lacZ may be nonfunctional, we decided to bring the amplified lactose operon on this place.
To achieve that we build a cassette consist of the upstream and downstream regions of lacZ and the lactose operon from DH5𝛼. The lactose operon of E. coli has a size of 6326bp. Since that is very large we decided to use smaller homologous flanks. It was shown that the natural co-transformation work well with 1000bp flanks, so we use this size. Otherwise the linear DNA Fragment has a size of ~12000bp. Thus, it is possible for us to introduce the lactose operon by cotransformation as an unselected product with the other described products.</P>
With this kind of genome modification, it is possible for the entire scientific community to immediately recognize a successful transformation with VibriClone. This would save not only through the fast growth rate but also through the method of cloning an incredible amount of working time.
<H3>Outlook</H3>
After developing the first cloning strain of Vibrio natriegens to fully replace E. coli as the most used cloning host in synthetic biology, we asked ourselves what would be the next step. VibriClone provides a starting point for further improvements for instance in case of special cloning application. When we created this strain we completed a full round of the design build test cycle and as we want to bring engineering into biology we started with the next cycle by designing VibriClone 2.0.
Therefore several different strains should be established in parallel to full fill the demand for more specialized applications. For E. coli several different strains exist, which are utilized for example for different sized constructs (TOP10), for allowing genome engineering tool, such as MAGE (mutS deficient), for in vivo mutagenesis, for toxic gene cloning or for preventing unwanted methylation.
In the beginning of our project we did next generation sequencing for our Vibrio natriegens strain and performed bioinformatic analysis and identified several other candidates for deletion and other genome engineering methods. One design feature for our 2.0 strain would include deleting methylase and restriction modification systems, which are present in our genome. In common cloning strains for E.Coli (DH5𝛼, Top10) restriction modification systems are deleted (Δ(mrr-hsdRMS-mcrBC) to enable cloning with DNA from different sources and to protect the DNA from endogenous restriction enzymes. That's why a desirable goal for the next generation of our cloning strain, would be to design a strain, similar to E.coli, which is deficient for all for those modification systems.<P>When we performed next generation sequencing in the beginning of our project for our wild type strain, we subsequently did a bioinformatic analysis and genome annotation. By utilizing these information we were able to identified several methylases and restriction modification systems.</P><TABLE align="center"><TBODY><TR><TD>Type I restriction - modification system subunit M</TD><TD>EC 2 . 1 . 1 . 72 </TD></TR><TR><TD>Type I restriction . modification system , specificity subunit S</TD><TD> EC 3. 1.21.3</TD></TR><TR><TD>Type I restriction modification system restriction subunit R</TD><TD>EC 3 . 1 . 21 . 3</TD></TR><TR><TD>Type III restriction - modification system methylation subunit</TD><TD>EC 2.1 . 1.72</TD></TR></TBODY></TABLE><P>We are sure, that more methylases and restriction modification systems can be found in the genome of Vibrio natriegens by utilizing and combining different bioinformatic tools such as the rebase database. 

Additionally we found several phage associated sequences and it has been shown that deleting phage related regions could make a bacteria more genetically accessible. In general a genome reduction could have advantages for a cloning host, as it has been shown that it could increase genome stability, reduce mutation rates and improved recombinational stability for cloning repetitive DNA sequences.

The most ambitious idea for our VibriClone 2.0 strain would be to free up chromosome II for utilization as a low copy plasmid in order to clone big DNA fragments or whole genomes and thereby establishing a synthetic chromosome for challenging cloning applications. In the future this could enable cloning of small of hole small genomes, such as the mycoplasma genome in order to make the genome engineering process easier and especially much faster, compared to common methods, where whole genome cloning heavily relies on yeast as a cloning chassis. Combining the fast growth of Vibrio natriegens with powerful genome engineering methods like Crispr Cas, MAGE, CAGE, the ability for natural transformation and MuGENT, would allow rapid genome prototyping and could open up complete new possibilities for building up synthetic genomes from scratch. This would enable engineering customizable microbes with complete new properties specifically tailored for the individual needs for a broad range of applications.</P></DIV></DIV><DIV class="collapsible"><DIV class="btn_expand">VibriExpress</DIV><DIV class="content"><DIV><P>
Sometimes it is really hard to part with old things and adopt to new ones. But as scientists, it is our responsibility to try out new and unexplored things in order to advance progress and improve on already known. To convince you all, to start working with a new chassis we will show that all commonly used techniques are transferable from <I>E. coli</I> to <I>V. natriegens</I>, without much effort. Since it is important to overexpress and purify proteins in heterologous systems, a fast growing protein expression strain is required.
Before starting to overexpress a protein, you need to make important decisions: “What kind of system would be the best?”, “Which chassis could be the most effective” or “Do I need a chassis at all, what about cell-free-expressions systems?”
There is not the one and only right way to overexpress a protein. For each protein, other aspects have to be considered in order to make a good decision. These decisions depend on different characteristics of the desired protein, like its solubility, if it is a bacterial protein or a eukaryotic one or if it is toxic or not.
In <I>E. coli</I>, several expression systems are established to cover a broad spectrum, but the most commonly used is the T7 expression system. To enable scientists, to overexpress their proteins much faster and purify a higher protein yield,  we decided to establish the T7 system in the world’s fastest growing organism <I>V. natriegens</I> and did further analysis to learn which genetic modifications are needed to use its full potential. As a model, we were guided by the <I>E. coli</I> strain BL21 (DE3). This <I>E. coli</I> strain is often used for heterologous protein overexpression. In our opinion, the most important features of BL21 (DE3) are the genomic integration of the T7 RNA Polymerase gene and the deletion of proteases like Lon and OmpT to improve the protein yield by decreasing the protein degradation 

<A href="https://www.ncbi.nlm.nih.gov/pubmed/6864790" target="blank"><ABBR title="Dunn, J. J. and F. W. Studier (1983). Complete nucleotide sequence of bacteriophage T7 DNA and the locations of T7 genetic elements. J Mol Biol 166(4):
477-535." j="" vis="" exp="">(Dunn &amp; Studier 1983)</ABBR></A>

, 

<A href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4395058/" target="blank"><ABBR title="Jeong, H., H. J. Kim and S. J. Lee (2015).Complete Genome Sequence of Escherichia coli Strain BL21. Genome Announc3(2).">(Kim <I>et al.</I> 2015)</ABBR></A>

.
We will try two different strategies to create a genomic T7 cassette integration (Figure 2). Because it was shown, that the deletion of dns is not harmful 

<A href="https://www.ncbi.nlm.nih.gov/pubmed/27571549" target="blank"><ABBR title="Weinstock, M. T., E. D. Hesek, C. M. Wilson and D. G. Gibson (2016). Vibrio natriegens as a fast-growing host for molecular biology. Nat Methods13(10): 849-851.">( Hesek <I>et al.</I> 2016)</ABBR></A>

, we decided to integrate the sequence of the T7 cassette at the endogenous locus of dns. In parallel, we will integrate the T7 cassette in one of our successfully characterized integration sites.
Looking at the genotype of <I>E. coli</I> BL21 we noticed that lacI localizes at the upstream site of the T7 RNA Polymerase gene 

<A href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4395058/" target="blank"><ABBR title="Jeong, H., H. J. Kim and S. J. Lee (2015). Complete Genome Sequence of Escherichia coli Strain BL21. Genome Announc3(2).">(Kim <I>et al.</I> 2015)</ABBR></A>

. LacI is the negative regulator of the lac operon 

<A href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC220206/" target="blank"><ABBR title="Gilbert, W. and B. Muller-Hill (1966). Isolation of the lac repressor. Proc Natl Acad Sci U S A 56(6): 1891-1898.">(Gilbert &amp; Muller-Hill 1966)</ABBR></A>

. In case of the T7 system, the <I>lacUV5</I> promoter controls the T7 RNA Polymerase expression 

<A href="https://www.ncbi.nlm.nih.gov/pubmed/3537305" target="blank"><ABBR title="Studier, F. W. and B. A. Moffatt (1986). Use of bacteriophage T7 RNA polymerase to direct selective high-level expression of cloned genes. J Mol Biol 189(1):
113-130">( Studier &amp; Moffatt 1986)</ABBR></A>

. By integrating the repressor LacI at the same time into the <I>V. natriegens</I> genome we will reduce the leakiness of the <I>lacUV5</I> promoter, as it was done in <I>E. coli</I>.
Additionally we checked if there are genes encoding proteins similar to Lon or OmpT, because these two were deleted in <I>E. coli</I> BL21 (DE3) to minimize the degradation of proteins 

<A href="https://www.ncbi.nlm.nih.gov/pubmed/6864790" target="blank"><ABBR title="Dunn, J. J. and F. W. Studier (1983). Complete nucleotide sequence of bacteriophage T7 DNA and the locations of T7 genetic elements. J Mol Biol 166(4):477-535." j="" vis="" exp="">(Dunn &amp; Studier 1983)</ABBR></A>

,<A href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4395058/"><ABBR title="Jeong, H., H. J. Kim and S. J. Lee (2015). Complete Genome Sequence of Escherichia coli Strain BL21. Genome Announc3(2).">(Kim <I>et al.</I> 2015)</ABBR></A>

. <A href="https://2018.igem.org/Team:Marburg/Description">Click here to read more about the proteases!</A> By BLAST analysis, we could find a Lon protease with a high degree of similarity to the Lon protease of <I>E. coli</I> in <I>V. natriegens</I> (Figure 1). We could not find any OmpT homologs neither by BLAST analysis nor by an <I>ompT</I> annotation in the genome of <I>V. natriegens</I>. Because of the high similarity of the Lon proteases to each other, it is likely that the Lon of <I>V. natriegens</I> fulfils a similar function like the Lon protease in <I>E. coli</I>. This convinced us to delete the lon of <I>V. natriegens</I>.

<FIGURE style="display: center; width: 80%;"><FIGCAPTION><B>Figure 1:</B>Alignment of Lon protease.This alignment is comparing the amino acid sequences of the lon protease of <I>E. coli</I> to the Lon protease of <I>V. natriegens</I>. This alignment was performed using ClustalX 2.0.10
</FIGCAPTION></FIGURE>


As already mentioned we will follow two different strategies to create the T7 integration. For the first strategy, we will fuse the T7 cassette. Amplified from the BL21 genome to a chloramphenicol resistance cassette and to 1000-bp up- and downstream homologous regions to <I>dns</I>. Since it is possible to create two or more genomic modifications, using only one selected product in natural co-transformation of linear fragments 

<A href="https://www.ncbi.nlm.nih.gov/pubmed/28571309" target="blank"><ABBR title="Dalia, T. N., C. A. Hayes, S. Stolyar, C. J. Marx, J. B. McKinlay and A. B. Dalia (2017). Multiplex Genome Editing by Natural Transformation (MuGENT) for Synthetic Biology in Vibrio natriegens. ACS Synth Biol6(9): 1650-1655.">(Hayes <I>et al.</I> 2017)</ABBR></A>

, we will use another fragment with homologous up- and downstream regions of <I>lon</I> but without a selection marker in between, to delete <I>lon</I>.
For the second strategy, we will integrate the T7 cassette to one of our characterized integration sites. For this reason, we will fuse the T7 cassette to 1000 bp long up- and downstream homologous regions to integration site 19 (int19). For the deletion of <I>lon</I>, we will use the same linear fragment as in strategy one. As a selected product with an antibiotics marker in between we will utilize the deletion cassette as for <I>Δdns</I> in VibriClone.

<FIGURE style="display: center; width: 80%;"><FIGCAPTION><B>Figure 2:</B> The two strategies of T7 integration. On the left side of the picture Strategy I is shown. Here we will integrate the T7 cassette into the endogenous locus of the nuclease <I>dns</I>. On the right side of the picture strategy II is shown. In this strategy we will integrate the T7 cassette at the integration site 19.
</FIGCAPTION></FIGURE>

As we will attach FRT sites to the antibiotic cassette by PCR, we will be able to remove the selection marker after integration of the selected product, using a plasmid that encodes a Flip recombinase.
After the genomic modifications are proved by PCRs, the T7 system needs to be tested and <B>VibriExpress</B> to be characterized. First of all a growth curve of this new strain has to be measured in comparison to <I>E. coli</I> BL21 to show you, that <I>V. antigens</I> is still faster than <I>E. coli</I>. Next, we will prove, that the cells are healthy after several genomic modifications by microscopic analysis. To show you, that changing from <I>E. coli</I> to <I>V. natriegens</I> will be easy and to prove, that tools like the T7 system are transferable without much effort, we will test a construct containing a T7 promoter in both organisms and compare the results to each other. We are convinced that the T7 system in VibriXpress will deliver a much higher amount of protein in a shorter time. This way we would like to speed up your scientific progress.
In 100 years of research with <I>E. coli</I>, BL21 is not the only used strain with a T7 expression system. Strains like Lemo21(DE3) (commercialized by NEB) are designed for fine-tuning of T7 expression especially for difficult targets. This fine-tuning is achieved due to the variation of the lysozyme (<I>lysY</I>) level, a gene encoding a variant of phage T7 lysozyme. The T7 Lysozyme is an inhibitor of T7 RNA Polymerase 

<A href="https://www.ncbi.nlm.nih.gov/pubmed/3568126"><ABBR title="Moffatt, B. A. and F. W. Studier (1987). T7 lysozyme inhibits transcription by T7 RNA polymerase. Cell 49(2): 221-227.">(Moffatt &amp; Studier 1987)</ABBR></A>

, 

<A href="https://www.ncbi.nlm.nih.gov/pubmed/9192997" target="blank"><ABBR title="Zhang, X. and F. W. Studier (1997). Mechanism of inhibition of bacteriophage T7 RNA polymerase by T7 lysozyme. J Mol Biol 269(1):10-27." cell="" 49="">(Zhang &amp; Studier 1997)</ABBR></A>

. By reducing the basal T7 RNA Polymerase activity toxic genes can be expressed in the same cell under control of a T7 promoter. Because of its ability to inhibit transcription the T7 lysozyme is often used for controlling expression in systems based on T7 RNA Polymerase 

<A href="https://www.ncbi.nlm.nih.gov/pubmed/3537305" target="blank"><ABBR title="Studier, F. W. and B. A. Moffatt (1986). Use of bacteriophage T7 RNA polymerase to direct selective high-level expression of cloned genes. J Mol Biol 189(1):113-130">( Studier &amp; Moffatt 1986)</ABBR></A>.</P><P>

In strains like Lemo21(DE3) the level of lysozyme can be optimized by inducing LysY with L-rhamnose to alleviate inclusion body formation or inhibitory effects from toxic proteins. In future, we will need to engineer this kind of specialized strains for troubleshooting as well in <I>V. natriegens</I>.
In addition, more research is necessary to study <I>V. natriegens</I> specific proteases, their function and their strength. Of course, we could not find an aspartyl protease similar to OmpT of <I>E. coli</I>, but this does not rule out that <I>V. natriegens</I> possesses other proteases that are crucial for protein degradation during overexpression. Finding highly active proteases and deleting them would further improve the amount of produced protein.
What about all the scientists that would like to use other expression systems than T7? As we mentioned before the T7 system is not always the first choice for protein overexpression. Other commonly used promoters like the synthetic <I>trc</I> promoter or T5 promoter are recognized from <I>E. coli</I> RNA Polymerases and do not require a genomic integration of a phage RNA polymerase 

<A href="https://www.ncbi.nlm.nih.gov/pubmed/2579077" target="blank"><ABBR title="Brosius, J., M. Erfle and J. Storella (1985). Spacing of the -10 and -35 regions in the tac promoter. Effect on its in vivo activity. J Biol Chem260(6): 3539-3541.">(Erfle <I>et al.</I> 1985)</ABBR></A>

, <A href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC214212/"><ABBR title="Gentz, R. and H. Bujard (1985). Promoters recognized by Escherichia coli RNA polymerase selected by function: highly efficient promoters from bacteriophage T5. J Bacteriol 164(1): 70-77.">(Gentz &amp; Bujard 1985)</ABBR></A>

. We could show, that these promoters are recognized from the <I>V. natriegens</I> RNA polymerase as well <A hre="https://2018.igem.org/Team:Marburg/Results">Click here to read more about the different promoters</A>, enabling you to use the potential of this impressing organism in a variety of tools and benefitting of its speed.
This result, together with all promising facts about <I>V. natriegens</I> like the higher solubility of some proteins that are difficult to purify and the simple transferability of <I>E. coli</I> tools to this organism, have strengthen our vision that <I>V. natriegens</I> is the perfect chassis to accelerate scientific progress in synthetic biology.


</P></DIV></DIV></DIV><DIV class="collapsible"><DIV class="btn_expand">VibriInteract</DIV><DIV class="content"><DIV><P>Protein interaction studies are known to be time-consuming. First, all the plasmids required need to be cloned and co-transformed into the interaction strain. Second, you have to cultivate those co-transformants and test them on inductive plates. Now imagine how much time you could save using a much faster growing organism, accelerating all steps required for protein interaction studies. And imagine this time, could be used in a meaningful way, to get new results and speed up scientific progress.</P><P>For the design of <B>VibriInteract</B> we had a look at the Genotype of the E. Coli strains used in Bacterial Two Hybrid Assays which are commercially available from Euromedex. It is also possible to use any other strain deficient of/from cya.</P><P><B>DHM1:</B><B>F-</B>, <B>cya-854</B>, recA1, endA1, gyrA96 (Nal r), thi1, <B>hsdR17</B>, spoT1, rfbD1, glnV44(AS).</P><P><B>BTH101:</B><B>F-</B>, <B>cya-99</B>, araD139, galE15, galK16, rpsL1 (Str r), <B>hsdR2</B>, mcrA1, mcrB1.</P><P>The genotypes of the  strains recommended and provided by Euromedex showed three common genetic modifications:</P><P><UL><LI><B>F-:</B> 
Both strains do not contain the F-plasmid which encodes the mechanism for conjugation in E. Coli 

<A href="https://www.ncbi.nlm.nih.gov/pubmed/6111288" target="blank"><ABBR title="Annu Rev Genet. 1980;14:41-76. The conjugation system of F-like plasmids. Willetts N, Skurray R.">(Willetts &amp; Skurray, 1980)</ABBR></A>

. <I>V. natriegens</I> has not been described to possess such a plasmid needed for conjugation. For this reason, we did not consider this part for <B>VibriInteract</B>.</LI><LI><B><I>hsdR17/hsdR2</I>:</B> 
HsdR is a restriction endonuclease found in E. coli of the EcoKI restriction-modification system. Endogenous sites get methylated by methylases. HsdR restricts foreign DNA by recognizing sites which or unmodified 

<A href="https://www.ncbi.nlm.nih.gov/pubmed/6255295" target="blank"><ABBR title="Molecular and General Genetics MGG November 1980, Volume 180, Issue 1, pp 35–46 | Cite asThe hsd (host specificity) genes of E. coli K12 Bela Sain Noreen E. Murray">(Sain &amp; Murray, 1980)</ABBR></A><ABBR title="Molecular and General Genetics MGG November 1980, Volume 180, Issue 1, pp 35–46 | Cite asThe hsd (host specificity) genes of E. coli K12 Bela Sain Noreen E. Murray">

. Point mutations in this gene lead to a nonfunctional protein. By this, non methylated  DNA is no longer degraded. This is important for strains that are designed to carry foreign DNA like plasmids e.g. cloning strains like VIBRICLONE. For <B>VibriInteract</B> this would be a useful alteration compared to the wildtype since our strain has to carry not only one but two plasmids at the same time. When searching for Type I restriction enzyme EcoKI R proteins we find several in V. natriegens. One of them, BA894_04835 shows a high degree of similarity when doing protein alignments using ClustalX 2.0.10. This could be used for later modifications in our strain but we did not choose it as our highest priority since it is not necessarily needed for functioning protein interaction studies based on B2H.

<FIGURE style="display: center; width: 80%;"><FIGCAPTION><B>Figure 1:</B> Alignment of the gene product of the putative <I>hsdR</I>  of <I>V. natriegens</I> with HsdR of <I>E. coli</I>. Alignment was performed with ClustalX 2.0.10.
</FIGCAPTION></FIGURE></ABBR></LI><LI><B><I>cya-854/cya-99</I>:</B> 
CyaA is a key regulator of the mal and the lac operon <A hre="https://2018.igem.org/Team:Marburg/Description">Click here to reread our description about CyaA</A>. Both strains recommended for a Bacterial Two-Hybrid assay in E. coli are deficient of this protein. We decided that this genetic modification our highest priority for the <B>VibriInteract</B> strain since we wanted to use complementation of CyaA as proof of protein interaction.  We identified  a homologous protein by doing a BLASTP search of CyaA from E. coli in V. natriegens, BA894_1512. By protein alignment using ClustalX 2.0.10 we see that the two proteins showed a high degree of  similarity. Therefore. we hypothesized that it serves the same or a similar function as the E. coli protein thus we decided to delete this gene using natural transformation for insertion of our deletion cassettes and the Flp recombinase to get rid of the antibiotic resistance cassette used for selection of insertion clones.  This deletion is the start of the development of a strain which can be used to check for the interaction of proteins.

<FIGURE style="display: center; width: 80%;"><FIGCAPTION><B>Figure 2:</B>Alignment of the gene product of the putative <I>cyaA</I>  of <I>V. natriegens</I> with cyaA of <I>E. coli</I>. Alignment was performed with ClustalX 2.0.10.
</FIGCAPTION></FIGURE></LI></UL></P><H3>Examinationof V. natriegens as chassis for protein interaction studies</H3>
Before we started to establish our strain, we had to verify if the wildtype strain of V. natriegens is suited to work in an interaction study assay based on the B2H system. For this purpose, we tested if our strain possesses a functional mal or lac operon by plating wildtype on diverse media which were:
<OL><LI>
		LBv2 agar plates supplemented with IPTG and Xgal used for Blue-White Screening if the lac operon is functional.
	</LI><LI>
        M63 minimal medium agar plates supplemented with maltose to test if the mal operon is functional.	
    </LI><LI>
    	MacConkey agar plates to test if the mal operon is functional.
    </LI></OL><H3>Characterization of VibriInteract</H3>
After obtaining our final strain we have to characterize it, because why would anyone use a new strain? With our characterization we want to show you how easy it is to use <B>VibriInteract</B>. Therefore, we designed experiments to demonstrate the benefits of V. natriegens and how little changes you have to undergo to use <B>VibriInteract</B> instead of the commonly used E. coli strain BTH101. First, we did growth curves, to verify that <B>VibriInteract</B> is faster than BTH101. Second, we did microscopic studies to verify that the cells of our strain still look healthy. Last and most important, we did a Protein Interaction study to verify that the pUT18 and pKT25 plasmids of the commercially sold B2H kit are working in our strain.
<H3>Outlook</H3>
For the future, we would like modify genes also modified in our cloning strain, which are <I>recA</I>, <I>katG</I>, <I>dns</I> and <I>exeM</I>. Those modification would help the foreign plasmids used for the B2H Assay to be retained in the cell. Read more about in the <B>VibriClone</B> part of our Strain Engineering subgroup. It could also be beneficial to eliminate the restriction endonuclease homologous to HsdR in E. coli. By this, foreign DNA is less likely to be degraded. Additionally, we would like to replace the non-functional lacZ gene with the lac operon of a wildtype E. coli strain like K12 MG1655. Hereby, we could use blue-white screening for our assays. This screening method is much faster than the selection on maltose minimal medium and would accelerate the assay even more. Another possible alteration of our strain could be the insertion of a reporter gene under control of an promoter which can be induced by the cAMP/CAP complex. This reporter gene could be an antibiotic resistance, a toxin or the luciferase. Using an antibiotic resistance one could use positive selection, since cells can grow on this specific antibiotic when proteins of interest are interacting. With a toxin under a cAMP/CAP dependent promoter one could use negative selection, since cells can no longer grow when proteins are interacting. The last possible reporter gene lux is giving an output signal through luminescence which could be measured. 

<P>We strongly believe, that the creation of <B>VibriInteract</B> is taking the community a step closer to establishing <I>V. natriegens</I> as the new standard chassis organism in synthetic biology.</P></DIV></DIV></DIV></DIV></DIV><DIV class="collapsible"><DIV class="btn_expand">CRISPR/Cas9</DIV><DIV class="content"><P>
Performing genome engineering in bacteria using CRISPR/Cas9  usually requires the presence of three components in a cell at the same time, namely: Cas9, gRNA and a donor template 
<A href="https://www.sciencedirect.com/science/article/pii/S0167701217301604"><ABBR title="A two-plasmid inducible CRISPR/Cas9 genome editing tool for &lt;i&gt;Clostridium acetobutylicum&lt;/i&gt;.">(Wasels <I>et al.</I> 2017).</ABBR></A> 
  Therefore we aimed to design a convenient system that allows easy adaption of CRISPR/Cas9 to target various sequences.<FIGURE style="width: 35%; float: right"><FIGCAPTION><B> Figure 1: LVL1-pTet-Cas9.</B> The LVL1 plasmid with p15A origin, pTET  promotor, Cas9 CDS and tretacyclin-resisstence is shown. </FIGCAPTION></FIGURE></P><P><B>Preparation of Cas9</B>
A Cas9 and dCas9 LVL0 part was created as a CDS part in our Marburg Collection. The cas9 sequence was PCR amplified from pCas9CR4 

<A href="https://www.nature.com/articles/srep15096.pdf"><ABBR title="The no-SCAR (Scarless Cas9 Assisted Recombineering) system for genome editing in &lt;i&gt;Escherichia coli&lt;/i&gt;">(Reisch and Prather 2017).</ABBR></A> 
To make this sequence compatible with our toolbox and the iGEM registry, we removed a BsmBI and EcoRI recognition site in the parts sequence creating the Cas9 LVL0 part <A href="http://parts.igem.org/Part:BBa_K2560047"><ABBR title="Link to the iGEM part registry">BBa_K2560047</ABBR></A>. For further experiments, a dCas9 part could prove useful and therefore we induced mutations the nuclease loop in the RuvC1 (D10A) &amp; HNH (H840A) subunits creating <A href="http://parts.igem.org/Part:BBa_K2560054"><ABBR title="Link to the iGEM part registry">BBa_K2560054</ABBR></A>.</P><P>
To establish CRISPR/Cas9 in <I>V. natriegens</I>, we conceived a Cas9 plasmid where the expression is tightly controlled by an inducible promoter. We expect this to help during cloning and transformation steps, as the toxicity of Cas9 expression is reduced in the absence of inducer. We chose pTet for this purpose. This decision is based on the characterization of this part that showed tight control and a 40 fold induction of this promoter when induced with ATc <A href="https://2018.igem.org/Team:Marburg/Results"> (Link to Results)</A> . Finally, we created LVL1-pTet-Cas9, a plasmid with the Cas9 CDS under control of pTet (figure 1). We chose to built this plasmid with p15A as the ori and a kanamycin resistance cassette. This enables cotransformation of this plasmid together with a second plasmid harboring the gRNA cassette.

<FIGURE style="width: 30%; float: left;"><FIGCAPTION><B> Figure 2: gRNA entry Vector.</B> The gRNA entry vector contained the scaffold of the gRNA and a sfGFP drop out for insertion of spacer sequences. </FIGCAPTION></FIGURE></P><P><B>Construction of a gRNA entry Vector</B>
To enable simple cloning of gRNAs, we constructed <A href="http://parts.igem.org/Part:BBa_K2560305"><ABBR title="Link to the iGEM part registry">BBa_K2560305</ABBR></A> an entry vector for gRNA spacer (figure 2). The spacer is defined as the 20 bp at the 5’ end of the gRNA that confers sequence specificity to the target sequence in the genome. This parts design is based on <A href=" http://parts.igem.org/Part:BBa_K2457002"><ABBR title="Link to the iGEM part registration">BBa_K2457002</ABBR></A> a part which harbors a gRNA targeting <I>LacZ</I> of <I>E. coli</I>. To achieve superior cloning flexibility we incorporated the approach of using a sfGFP dropout similar to the part entry vector of the Marburg Collection <A href="http://parts.igem.org/Part:BBa_K2560002"><ABBR title="Link to the iGEM part registration">BBa_K2560002</ABBR></A>. Analogous to the cloning of small LVL0 parts, new spacer sequences can easily be cloned by annealing of oligos and subsequent ligation into the gRNA entry vector using a golden-gate reaction. This drop out helps to distinguish correctly assembled plasmids from the religated entry plasmid. The succesfully cloned plasmid then contains a gRNA expression cassette as well as a pMB1 ori and the chloramphenicol resistance cassette.



</P><P>

The two plasmids LVL1-pTet-Cas9 and <A href="http://parts.igem.org/Part:BBa_K2560305"><ABBR title="Link to the iGEM part registry">BBa_K2560305</ABBR></A> after integration of a spacer are compatible and can be cotransformed. As a third component a donor template has to be added to the transformation. </P><P><FIGURE style="width: 35%; float: right"><FIGCAPTION><B> Figure 3: LVL2 plasmid.</B> Both the pTET-Cas9 LVL1 plasmid and the gRNA entry vector assembled in one LVL2 plasmid. </FIGCAPTION></FIGURE><B>Assembly of LVL2 plasmid</B>
We realized that cotransformation of three components into <I>V. natriegens</I> might be challenging and therefore, we designed a plasmid that is a combination of the previously designed LVL1-pTet-Cas9 and the gRNA entry vector <A href="http://parts.igem.org/Part:BBa_K2560305"><ABBR title="Link to the iGEM part registry">BBa_K2560305</ABBR></A>. Using our toolbox, we converted the LVL1 construct into a LVL2 plasmid to remove the BsmBI recognition sites that are present in the LVL1 plasmid. Subsequently, we integrated the gRNA transcription unit including the sfGFP dropout into the 5’ Connector via Gibson assembly. Finally, we created a plasmid that possesses a Cas9 CDS with pTet and the gRNA entry sequence to facilitate easy cloning (figure 3). </P><P>
In most bacterial CRISPR/Cas9 approaches, a two plasmid system is used separating Cas9 and gRNA on two different plasmids 
<A href="https://www.sciencedirect.com/science/article/pii/S0167701217301604"><ABBR title="A two-plasmid inducible CRISPR/Cas9 genome editing tool for &lt;i&gt;Clostridium acetobutylicum&lt;/i&gt;.">(Wasels <I> et al.</I> 2017).</ABBR></A>
 The reason is that during the process of gRNA cloning, a functional Cas9 transcription unit is present in the same cell with the correctly cloned gRNA, resulting in double strand breaks (DSB) leading to the death of the cells.</P><P>
Our approach to overcome this challenge is to use <I>E. coli</I> as cloning chassis for the construction of the Cas9-gRNA plasmid which is then transformed in <I>V. natriegens</I> together with a donor template to perform genome engineering. This is possible because the spacer sequence targeting the genome of <I>V. natriegens</I> should not match any sequence in the genome of <I>E. coli</I> and therefore we do not expect creation of DSB.</P><P><B>Designing of donor templates</B>
Different approaches for the design of donor templates are described in literature. The mutation can be introduced either as dsDNA in a linear form or integrated into a plasmid or as ssDNA <A href="https://aem.asm.org/content/aem/81/7/2506.full.pdf"><ABBR title="Multigene Editing in the &lt;i&gt;Escherichia coli&lt;/i&gt; Genome via the CRISPR-Cas9 System.">(Jiang <I>et al.</I> 2015).</ABBR></A></P><P>
We decided to use ssDNA because they can be easily ordered from companies offering synthesis. As a prove of concept we decided to tackle the dns gene on chromosome 1 of <I>V. natriegens</I>, as this mutation has already been shown not to be lethal and to significantly increase DNA yields of plasmid preparations 
<A href="https://www.biorxiv.org/content/biorxiv/early/2017/04/26/130088.full.pdf"><ABBR title="Recombineering in ​&lt;i&gt;Vibrio natriegens&lt;/i&gt;.">(Lee <I> et al.</I> 2017).</ABBR></A>
 We used two ssDNA oligos each targeting the leading or lagging strand. The oligos consists of the desired mutation flanked by 50 bp long homolougous sequences flanking the position of the introduced DSB. The last three nucleotides at both ends are connected via phosphorothioate-bonds providing the stabilization of the donor oligonucleotides and protection against digestion 
 <A href="https://www.nature.com/articles/srep15096.pdf"><ABBR title="The no-SCAR (Scarless Cas9 Assisted Recombineering) system for genome editing in &lt;i&gt;Escherichia coli&lt;/i&gt;">(Reisch and Prather 2017).</ABBR></A></P><P>
The donor template was designed to exchange four basepairs at the beginning of the dns gene with a three basepair stop-codon (figure 4). In this way, the knock-out is ensured through the premature translation termination as well as a frameshift in the sequence. 
The spacer sequences targeting the genomic sequence was designed using the CRISPR design tool of Geneious which utilizes the algorithm developed by 
<A href="https://www.nature.com/articles/nbt.3026.pdf"><ABBR title="Rational design of highly active sgRNAs for CRISPR-Cas9-mediated gene inactivation.">Doensch <I>et al.</I> (2014).</ABBR></A><FIGURE style="display:block; margin:0 auto 0 auto; width: 90%;"><FIGCAPTION><B> Figure 4: Alligment of the donor template and the <I>V. natriegens</I> genome.</B> The alligment shows that the the donor template differ in two nucleotides from the dns-CDS of  <I>V. natriegens</I>. The first nucelotide  leads to the conversion of an aminoacid to a stopcodon whereas the second, missing nucleotide leads to a frameshift.</FIGCAPTION></FIGURE></P></DIV></DIV></DIV></DIV><DIV class="collapsible"><DIV class="btn_expand"><DIV class="btn_e_title"> Metabolic Engineering</DIV><DIV class="btn_e_desc"> With Vibrigens we aim to speed up synthetic biology in all of its facettes. For that we designed a workflow for accelerated metabolic engineering in high-throughput. We use the Marburg Collection to construct whole pathway libraries, benefit from natural transformation to introduce it into our chassis, enable fast product screening with biosensors and implemented directed evolution for further pathway optimization. </DIV></DIV><DIV class="content"><DIV class="collapsible"><DIV class="btn_expand">Workflow for Metabolic Engineering and Design of Pathway and Enzymes</DIV><DIV class="content"><H3>Our workflow</H3><P>
	One of the central elements of synthetic biology is the Design-Build-Test-Learn cycle. 
	It illustrates the workflow of how systems like metabolic pathways, genetic circuits or anything else are constructed and evaluated. 
	The first step is to design the system basing on the existent knowledge about the system. 
	Then the system is built and its functionality is tested. The results of these tests describe the ‘Learning’ part of the cycle.  
	They are used to make an improved design. 
	Finally, a new round in the <DFN data-info="Design-Build-Test-Learn">DBTL</DFN> cycle can be made and the system gets improved in each round of the cycle.
	For our approach we decided to establish a workflow for metabolic engineering, which bases on the <DFN data-info="Design-Build-Test-Learn">DBTL</DFN> cycle and combines the existing principles of metabolic engineering with the advantages of the fast-growing organism <I>V. natriegens</I>.
	First of all, we use the Marburg Collection to construct a library of pathways with different combinations of promoters, ribosome binding sites, coding regions, enzyme ratios and plasmid copy numbers. 
	Next, we transform the plasmids for each pathway version together with the plasmid for the biosensor into our chassis. When the pathway is now performed, the cells produce a fluorescence or luminescence signal and its intensity correlates to the 3-hydroxypropionate (<DFN data-info="3-hydroxypropionic acid">3HPA-</DFN>) concentration. 
	By using <DFN data-info="Fluorescence activated cell sorting"><DFN data-info="Fluorescence activated cell sorting"> FACS </DFN></DFN> we can select for the most efficient pathway versions. 
	By using multiplex automated genome engineering (<DFN data-info="Multiplex automated genome engineering">MAGE</DFN>), refeeding <DFN data-info="3-hydroxypropionic acid">3HPA</DFN> into the central metabolism and repetition of <DFN data-info="Fluorescence activated cell sorting"> FACS </DFN> selection we have a cycle of directed evolution-guided pathway optimization. 
	The strains with the most promising pathway variants can then be isolated and their pathway composition elucidated. 
	Thus, we get information about designing principles for our pathway and can propose an improved design for a new round in the cycle of metabolic engineering.
	<FIGURE style="width: 50%; float: right;"><FIGCAPTION><B>
				Figure 1: Our implemented workflow of metabolic engineering
			</B>To accelerate metabolic engineering we implemented a workflow based on the <DFN data-info="Design-Build-Test-Learn">DBTL</DFN>-cycle. Starting by building a library of pathway variants, they all can be introduced into <I>V. natriegens</I> as test chassis. By using biosensors FACS-directed selection of promising variants is possible. Cultivation and repeating of FACS leads to directed evolution and finally the most efficient strains can be analyzes to investigate the genetic composition. These finding can be feeded into the Learning-part of the <DFN data-info="Design-Build-Test-Learn">DBTL</DFN>-cycle thereby closing the workflow. Repeating the workflow continiously increases pathway efficiency and enables high-throuput metabolic engineering.
		</FIGCAPTION></FIGURE></P><H3>
	Our pathway 
</H3><P>
	As mentioned in the description, there are many ways to produce <DFN data-info="3-hydroxypropionic acid">3HPA</DFN>. 
	We decided for a pathway, basing on the conversion of acetyl-CoA via malonyl-CoA to <DFN data-info="3-hydroxypropionic acid">3HPA</DFN>. 
	According to 

	<A href="https://doi.org/10.1007/s00253-013-4802-4"><ABBR title="Valdehuesa, K.N.G., Liu, H., Nisola, G.M., Chung, W.J., Lee, S.H., Park, S.J., 2013. Recent advances in the metabolic engineering of microorganisms for the production of 3-hydroxypropionic acid as C3 platform chemical. Appl. Microbiol. Biotechnol. 97, 3309–3321. ">(Valdehuesa <I>et al.</I> 2013)</ABBR></A> 

	who evaluated many pyruvate-derived production pathways from a thermodynamical point of view, this is one of the most efficient routes for <DFN data-info="3-hydroxypropionic acid">3HPA</DFN> production. 
	We decided for this route for several reasons. First it bases on glucose degradation and <I>V. natriegens</I> has an unbeaten glucose-uptake rate, enabling a high glucose consumption rate. Second there is no vitamin B12 necessary, the only cofactors which are involved are NADPH and biotin, both of them naturally occurring in the organism. 
	Third there is theoretically just one further enzyme needed to complete the pathway. 
	All enzymes necessary for the production of malonyl-CoA are still present in <I>V. natriegens</I> and we just have to integrate the last enzyme, malonyl-CoA reductase. 
	Nevertheless, overexpression of acetyl-CoA carboxylase increases productivity, so we took both enzymes into consideration. 

	<A href="https://doi.org/10.1186/s12934-016-0451-5"><ABBR title="Kildegaard, K.R., Jensen, N.B., Schneider, K., Czarnotta, E., Özdemir, E., Klein, T., Maury, J., Ebert, B.E., Christensen, H.B., Chen, Y., Kim, I.K., Herrgård, M.J., Blank, L.M., Forster, J., Nielsen, J., Borodina, I., 2016. Engineering and systems-level analysis of Saccharomyces cerevisiae for production of 3-hydroxypropionic acid via malonyl-CoA reductase-dependent pathway. Microb. Cell Fact. 15. ">(Kildegaard <I>et al.</I> 2016)</ABBR></A> 

	proposed, that there are many ways to further optimize the pathway and direct the flow to a high product titer. Hence, implementing many variations of this pathway into <I>V. natriegens</I> and testing them in a fast manner promises a high gain of knowledge about pathway optimization and general principles of metabolic engineering. 

	<FIGURE><FIGCAPTION><B>
				Figure 2: Pathway for the production of 3-hydroxypropionic acid
			</B>The pathway starts with pyruvate as product of glycolysis and other central metabolisms. Pyruvate is dehydrogenated by pyruvate dehydrogenase to acetyl-CoA. Acetyl-CoA gets carboxylated by <DFN data-info="Acetyl-CoA carboxylase">Acc</DFN> to form malonyl-CoA. Malonyl-CoA is finally reduced to 3-hydroxypropionic acid by Mcr. Acc and <DFN data-info="Malonyl-CoA reductase">Mcr</DFN> are the two enzyme, we planned to express heterologously. Acc = Acetyl-CoA carboxylase, Mcr = Malonyl-CoA reductase
		</FIGCAPTION></FIGURE></P><H3>

	Choice of enzymes
</H3><P>
	For the core pathway, we need to implement two enzymes, acetyl-CoA carboxylase (Acc) and malonyl-CoA reductase (Mcr).
	For Acc there are several candidates, which have still been tested for synthetic metabolisms 

	(<A href="https://doi.org/10.1186/s12934-016-0451-5"><ABBR title="Kildegaard, K.R., Jensen, N.B., Schneider, K., Czarnotta, E., Özdemir, E., Klein, T., Maury, J., Ebert, B.E., Christensen, H.B., Chen, Y., Kim, I.K., Herrgård, M.J., Blank, L.M., Forster, J., Nielsen, J., Borodina, I., 2016. Engineering and systems-level analysis of Saccharomyces cerevisiae for production of 3-hydroxypropionic acid via malonyl-CoA reductase-dependent pathway. Microb. Cell Fact. 15. ">(Kildegaard <I>et al.</I> 2016)</ABBR></A>;

	 <A href="https://doi.org/10.1371/journal.pone.0075554"><ABBR title="Liu, C., Wang, Q., Xian, M., Ding, Y., Zhao, G., 2013. Dissection of Malonyl-Coenzyme A Reductase of &lt;i&gt;Chloroflexus aurantiacus&lt;/i&gt; Results in Enzyme Activity Improvement. PLoS One 8, 1–8.">(Liu <I>et al.</I> 2013)</ABBR></A>;


	 <A href="https://doi.org/10.2172/15008859"><ABBR title="Vidra, A., Németh, Á., 2017. Bio-based 3-hydroxypropionic Acid : A Review. period. Polytech. Chem. Eng. 1–11.
	Werpy, T., Petersen, G., 2004. Top Value Added Chemicals from Biomass Volume I — Results of Screening for Potential Candidates from Sugars and Synthesis Gas Top Value Added Chemicals From Biomass Volume I : Results of Screening for Potential Candidates. Other Inf. PBD 1 Aug 2004 Medium: ED; Size: 76 pp. pages. ">(Vidra <I>et al.</I> 2017))</ABBR></A>. 


	Our plan was to first test enzymes from different organisms and then optimize expression profiles. 
	To benefit from the fast engineering workflow in <I>V. natriegens</I>, we decided to use enzymes from different organisms. 
	We planned to use the native <DFN data-info="Acetyl-CoA carboxylase">Acc</DFN> from <I>V. natriegens</I>, because it is still present in the cells and we prevent unnecessary burden by overexpressing a heterologous Acc. 
	The native Acc consists of four subunits together with an associated biotin ligase. 
	The genetic organization is the same like in <I>E. coli</I>, which is shown in figure 3. Nevertheless, further looks into literature revealed, that heterologous expression of pathway enzymes could be more promising because housekeeping Accs often are strictly regulated while foreign genes have different regulation motifs. 
	Hence, we also used the Acc from <I>E. coli</I> together with its associated biotin ligase (BirA). These enzymes have already been tested for <DFN data-info="3-hydroxypropionic acid">3HPA</DFN> production 

	<A href="https://doi.org/10.1371/journal.pone.0075554"><ABBR title="Liu, C., Wang, Q., Xian, M., Ding, Y., Zhao, G., 2013. Dissection of Malonyl-Coenzyme A Reductase of &lt;i&gt;Chloroflexus aurantiacus&lt;/i&gt; Results in Enzyme Activity Improvement. PLoS One 8, 1–8.">(Liu <I>et al.</I> 2013)</ABBR></A>.

	Additionaly, we decided to test the enzyme from Synechococcus elongatus. Since cyanobacteria require higher amounts of fatty acids to form thylakoid membranes, they might have more active enzymes. 
	The <I>S. elongatus</I> Acc consists of four subunits, just like those from <I>E. coli</I> and <I>V. natriegens</I>. For this enzyme it was also recommended to use it with a biotin ligase, because it needs a covalently bound biotin to be activ. 
	Unfortunately Accs consisting of four subunits form a very unstable complex and unbalanced expression levels of its subunits lead to gene repression 

	<A href="https://doi.org/10.1128/JB.01373-06
	"><ABBR title="Abdel-Hamid, A.M., Cronan, J.E., 2007. Coordinate expression of the acetyl coenzyme A carboxylase genes, &lt;i&gt;accB&lt;/i&gt; and &lt;i&gt;accC&lt;/i&gt;, is necessary for normal regulation of biotin synthesis in Escherichia coli. J. Bacteriol. 189, 369–376. ">(Abdel-Hamid <I>et al.</I> 2006)</ABBR></A>. 


	To circumvent this issue, we also chose the <DFN data-info="Acetyl-CoA carboxylase">Acc</DFN> from <I>C. glutamicum</I>. 
	In this organism, AccB and AccC are fused to one subunit, as well as AccA and AccD. Balancing of two subunits instead of four is much easier, therefore this enzyme is beneficial for us. Gande et al., 2007 still characterized this enzyme and confirmed its applicability for synthetic biology. Since <I>C. glutamicum</I> is a gram-positive bacterium, we decided to codon optimize the genes for a smoother translation and coexpressed the associated biotin ligase (Accession numbers NCgl0670, NCgl0678, NCgl0679).

</P><DIV style="display:inline-block"><FIGURE style="float: left; width: 50%"><FIGCAPTION><B>
				Figure 3: Genetic organisation of <I>acc</I> genes in <I>E. coli</I> and comparison to <DFN data-info="Acetyl-CoA carboxylase">Acc</DFN>s from other strains
			</B>In <I>E. coli</I> Acc is encoded by the genes <I>accA</I>, <I>accB</I>, <I>accC</I>, <I>accD</I> and the associated gene <I>birA</I>. In <I>V. natriegens</I> and <I>S. elongatus</I> there is the same genetic organization. Acc from <I>C. glutamicum</I> is encoded by only two genes, <I>accBC</I> and <I>accD</I>.
		</FIGCAPTION></FIGURE>


	The second enzyme in our pathway is the malonyl-CoA reductase (Mcr). 
	Since this enzyme only occurs in autotrophic organisms, performing the 3-hydroxypropionate bicycle, there aren’t many candidates for our enzyme choice. 
	In literature Mcr from <I>Chloroflexus aurantiacus</I> has mostly been used, because it is easy to purify, has a high activity and is still active at moderate temperatures 

	<A href="https://doi.org/10.1128/JB.184.9.2404-2410.2002
	"><ABBR title="Hügler, M., Menendez, C., Schägger, H., Fuchs, G., 2002. Malonyl-coenzyme A reductase from &lt;i&gt;Chloroflexus aurantiacus&lt;/i&gt;, a key enzyme of the 3-hydroxypropionate cycle for autotrophic CO(2) fixation. J. Bacteriol. 184, 2404–10. ">(Hügler <I>et al.</I> 2002)</ABBR></A>
;

	<A href="https://doi.org/10.1080/07388551.2016.1272093
	"><ABBR title="Liu, C., Ding, Y., Xian, M., Liu, M., Liu, H., Ma, Q., Zhao, G., 2017. Malonyl-CoA pathway: a promising route for 3-hydroxypropionate biosynthesis. Crit. Rev. Biotechnol. 37, 933–941. ">(Liu <I>et al.</I> 2017)</ABBR></A>. 

	We decided to use this enzyme and test it with different expression levels and <DFN data-info="Acetyl-CoA carboxylase">Acc</DFN>/Mcr ratios. However, recently it was shown, that a dissection of the enzyme strongly increases productivity 

	<A href="https://doi.org/10.1371/journal.pone.0075554"><ABBR title="Liu, C., Wang, Q., Xian, M., Ding, Y., Zhao, G., 2013. Dissection of Malonyl-Coenzyme A Reductase of &lt;i&gt;Chloroflexus aurantiacus&lt;/i&gt; Results in Enzyme Activity Improvement. PLoS One 8, 1–8.">(Liu <I>et al.</I> 2013)</ABBR></A>. 

	In general, <DFN data-info="Malonyl-CoA reductase">Mcr</DFN> has two functional domains, the C-terminal domain catalyzes the reduction of malonyl-CoA to 3-oxopropanoate and the N-terminal domain catalyzes the further reduction to <DFN data-info="3-hydroxypropionic acid">3HPA</DFN>. 
	In the mentioned publication there wasn’t an exact explanation for the increased productivity of the dissected enzymes. 
	Our hypothesis is, that the complete enzyme, which has its temperature optimum at 60°C, has a suboptimal protein folding at 37°C while the dissected enzymes are smaller and less bulky and therefore can fold easier. Deeper investigations of enzyme behavior additionally revealed that the C-terminal domain is much slower than the N-terminal domain and thereby the C-domain slows down the overall reaction 

	<A href="https://doi.org/10.1371/journal.pone.0075554"><ABBR title="Liu, C., Wang, Q., Xian, M., Ding, Y., Zhao, G., 2013. Dissection of Malonyl-Coenzyme A Reductase of &lt;i&gt;Chloroflexus aurantiacus&lt;/i&gt; Results in Enzyme Activity Improvement. PLoS One 8, 1–8.">(Liu <I>et al.</I> 2013)</ABBR></A>. 

	Therefore, we decided to use the complete enzyme of <DFN data-info="Malonyl-CoA reductase">Mcr</DFN> as well as the dissected enzymes McrC and McrN. 
	There is also another Mcr from <I>Sulfolobus tokodaii</I>, which only catalyzes the rate-limiting step from malonyl-CoA to 3-oxopropanoic acid. Since McrC has the same function, we planned to coexpress Mcr from <I>S. tokodaii</I> together with McrN from <I>C. aurantiacus</I> and test, if the activity increases.
	All in all, we planned to use the acetyl-CoA carboxylase from <I>E. coli</I>, <I>V. natriegens</I>, <I>C. glutamicum</I> and <I>S. elongatus</I>, the Mcr from <I>S. tokodaii</I> and the Mcr, McrC and McrN from <I>C. aurantiacus</I>. 
	First, we wanted to test enzyme activity and afterwards use the Marburg Collection to build combinations of enzymes and expression levels.

	<FIGURE style="width: 50%; float: right;"><FIGCAPTION><B>
				Figure 4: Composition of malonyl-CoA reductase and its splitted versions
			</B>It was published that splitting of <DFN data-info="Malonyl-CoA reductase">Mcr</DFN> leads to increased activity, therefore beside the whole enzyme we also want to test the splitted version McrN + McrC and a combination of McrN and Mcr from <I>S. tokodaii</I></FIGCAPTION></FIGURE></DIV></DIV></DIV><DIV class="collapsible"><DIV class="btn_expand">Rational Plasmid Design and Pathway Library</DIV><DIV class="content"><H3>
	Rational design of plasmids
</H3><P>
	Synthetic biology proceeded significantly in recent years and we have much knowledge about rational design of circuits and pathways. 
	Nevertheless, failures and unexpected outcomes slowdown scientific progress. 
	To tackle this issue, we decided to follow two approaches. On the one hand we planned to model our pathway and predict a – theoretically – optimal pathway, which we can then construct and test. 
	And on the other hand, we decided to build additionally to this particular pathway a whole library of many variations of this pathway with little differences. 
	In this way we create many producer strains in parallel and speed up the metabolic engineering workflow.
	Our rational approach bases on a Michaelis-Menton-based metabolic analysis. For more details see our

	<A href="https://2018.igem.org/Team:Marburg/Model"><ABBR title="Metabolic Model">metabolic model</ABBR></A>.

	Since there are no stochiometric flux data about <I>V. natriegens</I> yet, we used the data from <I>E. coli</I>, which has a very comparable metabolome to <I>V. natriegens</I> (citation needed). 
	For the model we only considered <I>acc</I> from <I>C. glutamicum</I> and <I>mcr</I> from <I>C. aurantiacus</I>, because they were from our point of view the most promising ones. 
	The model predicted, that <DFN data-info="Acetyl-CoA carboxylase">Acc</DFN> is the rate limiting enzyme in the pathway and has to be expressed ten times stronger than <DFN data-info="Malonyl-CoA reductase">Mcr</DFN>. 
	We used the promoter characterization data, obtained from our part collection team (For more details see the section about  

	<A href="https://2018.igem.org/Team:Marburg/Results">characterization of the promoters</A> of our Marburg Collection) and introduced them into the model. 
	The resulting prediction was, that we have to use the promoter <A href="http://parts.igem.org/Part:BBa_K2560014">J23106</A> for <I>acc</I> and <A href="http://parts.igem.org/Part:BBa_K2560023v">J2310</A> for <I>mcr</I>. Therefore, we designed the pathway plasmid according to this model. 
	We built the plasmid based on the Marburg Collection with 4 transcriptional units. For the ori we chose ColEI, which has a broad host range and a high copy number in <I>V. natriegens</I> and <I>E. coli</I> (<A href="https://2018.igem.org/Team:Marburg/Design"><ABBR title="link to ori characterization of the part collection"> link to ori characterization of the part collection</ABBR></A>)

	<A href="https://doi.org/10.1111/1574-6968.12241
	"><ABBR title="Jain, A., Srivastava, P., 2013. Broad host range plasmids. FEMS Microbiol. Lett. 348, 87–96. ">(Jain <I>et al.</I> 2013)</ABBR></A> 
	).

	We decided for that one, because <I>V. natriegens</I> shall be a chassis for testing pathway variants, which then maybe will be transferred into other strains, therefore a broad host range vector is reasonable. 
	For <I>accBC</I> and <I>accD</I> we chose the promoter J23106 and for <I>mcr</I> the promoter J23109 according to the model. For <I>birA</I> we also chose J23109 as promoter, because BirA is only relevant for activation of <DFN data-info="Acetyl-CoA carboxylase">Acc</DFN> and not for direct reaction catalysis. For ribosome binding sites and terminators, we used the same for each coding region, because otherwise we would have fluctuations which weren’t considered in the model. 
	An illustration of the plasmid is shown in figure 5.


	<FIGURE style="width: 90%;"><FIGCAPTION><B>
				Figure 5: Composition of our rationally designed pathway plasmid with the Marburg Collection
			</B>The scheme shows the composition of the level 2 plasmid, containing four transcriptional units for <I>accBC</I>, <I>accD</I>, <I>birA</I> and <I>mcr</I>. We chose the promoters according to the metabolic <A href="https://2018.igem.org/Team:Marburg/Model"><ABBR title="link to model section">link to model section</ABBR></A> to achieve optimized expression strengths.
		</FIGCAPTION></FIGURE></P><H3>
	Pathway library
</H3><P>
	For the second approach of pathway design, where we build a whole library, we also used the <A href="https://2018.igem.org/Team:Marburg/Description"><ABBR title="Marburg Collection">Marburg</ABBR></A> to combine different promoters, ribosome binding sites and coding regions. 
	In theory, the toolbox enables the plasmid assembly of 21780 different part combinations per transcriptional unit (22 promoters x 5 rbs<DFN data-info="ribosomal binding site">RBS</DFN> x 6 terminators x 3 oris x  11 resistance cassettes; connectors weren’t considered). 
	For a plasmid containing <I>accBC</I>, <I>accD</I>, <I>birA</I> and <I>mcr</I> that would make 6 x 1012 possible constructs, but assembly of so many versions is irrational. The better way is to select a few parts which are combined and then for the most promising versions one can think of more combinations. 
	Therefore, we focused on eight promoters and three ribosome binding sites with different strengths. Our idea was to build pathway plasmids containing the whole pathway, as well as plasmids containing only the acetyl-CoA carboxylase genes or only the <I>mcr</I> gene, which can then be cotransformed (Table 1). 
	To enable cotransformation, we used the oris ColEI and p15a, which are compatible to each other.
</P><FIGURE><FIGCAPTION><B>Table 1:</B> Plasmid categories for our pathway library</FIGCAPTION><TABLE><TBODY><TR><TH>Plasmid Category</TH><TH>Inserts</TH><TH>Description</TH></TR><TR><TD>I</TD><TD><I>mcr</I></TD><TD>These plasmids contain the whole <I>mcr</I> gene from <I>C. aurantiacus</I> and can be cotransformed with each ColEI plasmid, because they have a p15a ori.</TD></TR><TR><TD>II</TD><TD><I>mcrN  mcrC</I></TD><TD>These plasmids contain the dissected <I>mcr</I> fragments from <I>C. aurantiacus</I> and can be cotransformed with each ColEI plasmid, because they have a p15a ori.</TD></TR><TR><TD>III</TD><TD><I>mcrN  mcrSt</I></TD><TD>These plasmids contain the N-terminal<I> mcr</I> fragments from <I>C. aurantiacus</I> and the <I>mcr</I> from <I>S. tokodaii</I>. They can be cotransformed with each ColEI plasmid, because they have a p15a ori.</TD></TR><TR><TD>IV</TD><TD><I>accBC  accD  birA</I></TD><TD>These plasmids contain all required genes for the acetyl-CoA carboxylase from <I>C. glutamicum</I> and can be cotransformed with each p15a plasmid, because they have a ColEI ori.</TD></TR><TR><TD>V</TD><TD><I>accBC  accD  birA  mcr</I></TD><TD>These plasmids contain all genes for the complete pathway. They combine the <DFN data-info="Acetyl-CoA carboxylase">Acc</DFN> from <I>C. glutamicum</I> with <DFN data-info="Malonyl-CoA reductase">Mcr</DFN> from <I>C. aurantiacus</I>.</TD></TR><TR><TD>VI</TD><TD><I>accBC  accD  birA  mcrN  mcrC</I></TD><TD>These plasmids contain all genes for the complete pathway. They combine the Acc from <I>C. glutamicum</I> with the dissected Mcr from <I>C. aurantiacus</I>.</TD></TR><TR><TD>VII</TD><TD><I>accBC  accD  birA  mcrN  mcrSt</I></TD><TD>These plasmids contain all genes for the complete pathway. They combine the Acc from <I>C. glutamicum</I> with the N-terminal Mcr-domain from <I>C. aurantiacus</I> and Mcr from <I>S. tokodaii</I>.</TD></TR></TBODY></TABLE></FIGURE><P>
	The idea behind the plasmids with the whole pathway is, that the cells only have to maintain one particular plasmid. The strategy for the single-enzyme plasmids is that by cotransformation different combinations can be made to increase the number of possible pathway variants.  The different plasmid types can be seen in table 1, all in all we designed the construction of 1446 plasmids for our pathway library.
	For plasmids of category I (see table 1), we combined eight promoters with three ribosome binding sites, a kanamycin resistance gene and a p15a ori, giving us 24 constructs. For plasmids of category II and III we used the same promoters and ribosome binding sites for both transcriptional units, leading to 576 plasmids respectively. For plasmids of category IV we restricted our selection to three promoters and three ribosome binding sites. Since a misbalance of <I>accBC</I>- and <I>accD</I> expression levels leads to inclusion bodies and decreased activity we decided to always use the same promoters and <DFN data-info="ribosomal binding site">RBS</DFN> for both genes. In this category we get 81 plasmid variants. For categories V, VI and VII, containing the whole pathway, the combinatorial possibilities were very high, so we decided to further restrict the selection by always coupling strong promoters to strong <DFN data-info="ribosomal binding site">RBS</DFN>, medium promoter to medium <DFN data-info="ribosomal binding site">RBS</DFN> and weak promoter to weak <DFN data-info="ribosomal binding site">RBS</DFN>. Therefore, we got a total of 27 plasmids for category V and 81 plasmids for category VI and VII, respectively. 
	We also considered to integrate one or several genes, therefore we planned to search for integration sites and to characterize them
	( see <A href="https://2018.igem.org/Team:Marburg/Design"><ABBR title="link to ori characterization of the part collection"> link to ori characterization of the part collection</ABBR></A>). 





</P></DIV></DIV><DIV class="collapsible"><DIV class="btn_expand">Biosensors and Product Screening</DIV><DIV class="content"><H3>
	Our biosensors
</H3><P>
  The design of our biosensors was largely guided by the goal of having as little impact on the host metabolism, while simultaneously being simple and easy to implement. We chose to design all parts to be compatible with the Marburg Collection, <A href="http://parts.igem.org/Help:Standards/Assembly/RFC10"><ABBR title="Description of the iGEM RFC25 standard">RFC10, </ABBR></A><A href="http://parts.igem.org/Help:Standards/Assembly/RFC25"><ABBR title="Description of the iGEM RFC25 standard">RFC25, </ABBR></A>and<A href="http://parts.igem.org/Help:Standards/Assembly/RFC25"><ABBR title="Description of the iGEM RFC1000 standard"> RFC1000, </ABBR></A> to make it as easy as possible for us and future users to implement it and adapt it to new requirements.
  
  <FIGURE style="width: 83%; float: center"><FIGCAPTION><B> Figure 6: Scheme of the 3-HPA sensor </B> This part enables creation of new LVL0 parts by replacing the parts sequence with the desired insert. The sfGFP expression cassette is flanked by BsmBI recognition sites </FIGCAPTION></FIGURE></P><P>
  Annotation of the location for the regulatory sequences upstream of the HdpR coding sequence was insufficient for us to precisely remove the activator binding site or the constitutive promoter driving HdpR expression. So, we included an about 200bp long region upstream of the HdpR, which should be enough to include all regulatory motifs. Also included is the HdpR coding sequence with its own ribosomal binding site <DFN data-info="ribosomal binding site"> (RBS). </DFN> and promoter. This results in a part that includes the whole sensing function, but does not quite fit our scheme of clearly defined functional units. Since expression of any part cloned behind the regulatory region is directly controlled by it, and the part serves as both, a promoter and <DFN data-info="ribosomal binding site"> RBS. </DFN> It is basically an inducible promoter fused to an <DFN data-info="ribosomal binding site"> RBS. </DFN> Thus, we decided to give the part the overhangs of an <DFN data-info="ribosomal binding site"> RBS. </DFN> on the downstream and a promoter on the upstream end.
</P><P>
  In order to obtain the desired region from the
  <I>
    P. putida
  </I>
   KT2440 genome, we planned to amplify it via <DFN data-info="polymerase chain reaction"> PCR </DFN> and then isolate the fragment by gel purification of the corresponding band. Into the primer-design we factored in the overhangs we needed to make it Marburg Collection compatible. So, we gave them non-binding overhangs which automatically added the necessary BsmBI cut sites.
</P><FIGURE style="width: 83%; float: center"><FIGCAPTION><B> Figure 7: Scheme of the malonyl-CoA sensor </B> This sensor consists of two unique level 0 parts that combined with the lux operon as reporter constitute the biosensor </FIGCAPTION></FIGURE><P>
  For the malonyl-CoA sensor, our proceeding was very similar. Just that we knew the exact location of all functional parts, except of the location of the promoter, from the 2015 paper by Liu et al and from annotations in the published genome 
  <A href="http://www.ncbi.nlm.nih.gov/pubmed/24377365"><ABBR title="Negative Feedback Regulation of Fatty Acid Production Based on a Malonyl-CoA Sensor-Actuator.">(Liu <I>et al.</I> 2015).</ABBR></A> 
    Therefore, we could implement it fully into the Marburg Collection, by simply treating the FapR coding sequence as coding sequence. Not that straightforwardly was the design of the
  <I>
    fapO
  </I>
   regulator binding site. Since we did not know the promoter strength, and the mechanism of regulation was independent of the promoter sequence, we decided to implement a promoter from the Andersen library, J23100. By placing it right in front of the
  <I>
     fapO
  </I>
   site, we created a construct that, in theory, should give us a FapR controlled promoter. For the amplification of the individual parts from the
  <I>
    B. subtilis
  </I>
  168 genome, again, we used primers sporting non-binding regions, adding the necessary overhangs for our toolbox.
</P><P>
  For future projects, other sensors could also be employed. In a publication from 2016, Rogers 
  
    <I>et al.</I>
  
   proposed an alternative sensor for <DFN data-info="3-hydroxypropionate"> 3HPA</DFN> with a lower perception threshold
  
    <A href="http://www.ncbi.nlm.nih.gov/pubmed/26858408 (October 13, 2018)"><ABBR title="Genetically Encoded Sensors Enable Real-Time Observation of Metabolite Production Proceedings of the National Academy of Sciences">(Rogers<I>et al.</I>2016).</ABBR></A>  

    We choose not to implement it in our design because of its more complicated mechanism that requires the implementation of more genes. Intriguingly, it works by first converting <DFN data-info="3-hydroxypropionate"> 3HPA</DFN> to acrylate via a pathway of three heterologous enzymes. The acrylate is then sensed by a transcription factor, similarly to how our sensors described above work. They also report its successful implementation and application in increasing the <DFN data-info="3-hydroxypropionate"> 3HPA</DFN> titer up to 23-fold. It is conceivable that with a little more effort our system could yield comparable results. Off cause, we have the additional benefit of having a much shorter iteration interval.
</P><FIGCAPTION><B>Figure 8: Biosensors for malonyl-CoA and 3-hydroxypropionate</B>The malonyl-CoA Sensor FapR is a dimeric protein which binds to the fapO operator, thereby inhibiting expression of the reporter gene. When malonyl-CoA is bound, the regulator dissociates from the DNA and gene expression can occur. The Sensor for 3-hydroxypropionate works reversely. In the absence of 3HPA the 3HPA sensor HpdR binds to its operator, allowing the polymerase to read the HpdR promoter.
        </FIGCAPTION></DIV></DIV></DIV></DIV></ARTICLE><H3>Further product sceening</H3><P>
	The usage of biosensors for product screening is a fast and cheap method enabling high throughput analysis of pathway libraries. 
	However, to further characterize the pathway variants and implemented enzymes we planned to measure enzyme activities. 
	In that way we can quantitatively benchmark each enzyme and establish in vitro assays as a prototyping engineering approach.
	For the malonyl-CoA reductase, it is easy to measure reaction velocity, because the enzyme converts NADPH into NAD+ and both compounds have different absorbances at 345nm 

	<A href="https://doi.org/10.1016/j.cplett.2007.10.092
	"><ABBR title="De Ruyck, J., Famerée, M., Wouters, J., Perpète, E.A., Preat, J., Jacquemin, D., 2007. Towards the understanding of the absorption spectra of NAD(P)H/NAD(P)+ as a common indicator of dehydrogenase enzymatic activity. Chem. Phys. Lett. 450, 119–122. ">(De Ruyck <I>et al.</I> 2007)</ABBR></A>. 


	Based on that we designed an enzyme assay, where we use cell extract from <DFN data-info="Malonyl-CoA reductase">Mcr</DFN> producing cells or directly purified Mcr and mixed it with a defined amount of NADPH and MgCl2. 
	By measuring the absorbance, one can determine if NADPH concentration changes. 
	By adding malonyl-CoA to the reaction mix we then start the reaction and are able to observe a decrease in absorbance because the reaction consumes NADPH. 
	Here the starting concentrations of malonyl-CoA and NADPH have to be high enough to ensure, that nearly each Mcr molecule in the mixture is saturated and the reaction goes proportional to enzyme concentration. 
	By determining the slope of the linear decrease of absorbance it is possible to calculate the specific activity. 
	The specific activity is a value which characterizes the ability of enzymes, to convert a substrate into a product. 
	It is defined as the moles of product formed by an enzyme in a given amount of time under given conditions per milligram of total proteins, the unit is µmol min-1mg-1. 
	To calculate the specific activity, one needs the slope of the measured absorbance. This value defines the change of absorbance per time. 
	By using the Lambert-Beer law it is possible to calculate the amount of NADPH converted into NADP+ per time 

	
	<A href="https://en.wikipedia.org/wiki/Beer%E2%80%93Lambert_law"><ABBR title="H.F. Tabak, J.L. Murk, I.Braakman, H.J.Geuze, Peroxisomes start their life in the endoplasmic reticulum, Traffic 4 (2003) 512-518">(Lambert and Anding, 1760)</ABBR></A>.

	ε is 3400 M-1cm-1 and d is 1cm. By dividing with time and enzyme volume, we get the final formula to calculate the NADPH conversion per time.



	Since one conversion of malonyl-CoA corresponds to two conversions of NADPH, we can calculate, how much malonyl-CoA is consumed per time. This value is defined as enzyme activity with the unit µmol/min and depends on the enzyme amount. By taking the protein amount into consideration, the specific activity can be calculated:


	For <DFN data-info="Acetyl-CoA carboxylase">Acc</DFN> the enzyme assay must be redesigned because carboxylation of acetyl-CoA doesn’t consume NADPH or other photometrically measurable compounds. 
	Therefore, we decided to do a coupled enzyme assay, where we not only add Acc (or Acc containing cell extract) into the reaction mixture but also purified <DFN data-info="Malonyl-CoA reductase">Mcr</DFN> as indicator enzyme which metabolizes malonyl-CoA as product of Acc and thereby converts NADPH to NAD+. 
	It is crucial to add large amounts of Mcr into the mixture to ensure, that Acc is the rate-determining enzyme and not Mcr. For the Acc activity assay we planned to mix purified Acc or cell extract with ATP, MgCl2, NaHCO3, purified Mcr and NADPH. Our strategy was to start the reaction by adding Acetyl-CoA as precursor for all reaction steps and then measure absorbance loss just like in the Mcr assay. 
	The calculations for the specific activity are also the same. 
	Beside evaluating enzyme performances, we also wanted to specifically detect and quantify <DFN data-info="3-hydroxypropionic acid">3HPA</DFN> in our cells. 
	We planned to do that via HPLC and mass spectrometry. Establishing a HPLC/Mass spectrometry protocol is also essential for calibrating our biosensors. 
	The identification of malonyl-CoA as intermediate of our pathway is not so challenging because the CoA appendix lends it a larger mass of 853.12g/mol, which is specific for malonyl-CoA. 
	For <DFN data-info="3-hydroxypropionic acid">3HPA</DFN> the specific identification is more difficult because <I>V. natriegens</I> is able to fermentatively produce lactate, a structural isomer of <DFN data-info="3-hydroxypropionic acid">3HPA</DFN>. 
	To distinguish between <DFN data-info="3-hydroxypropionic acid">3HPA</DFN> and lactate we chose to use a triple quadrupole mass spectrometer. 
	In such a device two quadrupole mass analyzers are toggled in a row and separated by a third quadrupole, responsible for the collision-induced dissociation of ions. 
	Our strategy was to cultivate cells, performing the <DFN data-info="3-hydroxypropionic acid">3HPA</DFN> pathway or at least one of its reaction steps and then lysate the cells and measure lactate- as well as <DFN data-info="3-hydroxypropionic acid">3HPA</DFN>-concentrations in the cell extract and also in the medium, because there are no hints if <DFN data-info="3-hydroxypropionic acid">3HPA</DFN> stays in the cell or is released into the medium.




</P></MAIN></DIV></DIV><DIV class="collapsible"><DIV class="btn_expand">Directed Evolution and Outlook</DIV><DIV class="content"><H3>
	Directed evolution
</H3><P>
	Metabolic Engineering offers many possibilities to create suitable pathways but it consists of more than just building them. Pathway optimization is also crucial for efficient productivity. 
	In our workflow for metabolic engineering we want to implement directed evolution to drive metabolic fluxes towards <DFN data-info="3-hydroxypropionic acid">3HPA</DFN> production. 
	We elaborated two strategies to do that. Our first strategy was to implement genes for the conversion of <DFN data-info="3-hydroxypropionic acid">3HPA</DFN> into succinate. 
	In that way, we lead our product back into the central metabolism. 
	By deleting all other genes responsible for succinate production, the cells are forced to use the <DFN data-info="3-hydroxypropionic acid">3HPA</DFN> bypass to get succinate, which is needed in the TCA cycle. 
	The big advantage of using <I>V. natriegens</I> for that purpose is that its short doubling time increases the number of cell divisions per time and thereby increases the number of mutations in a given time frame. Our second strategy is to use <DFN data-info="Multiplex automated genome engineering">MAGE</DFN> together with our biosensors and fluorescence-activated cell sorting (<DFN data-info="Fluorescence activated cell sorting"> FACS </DFN>), to cause various genome modifications and rapidly select for cells with increased <DFN data-info="3-hydroxypropionic acid">3HPA</DFN> productivity (Figure 9). (See also <A href="https://2018.igem.org/Team:Marburg/Description"><ABBR title="Description of Metabolic Engineering"> link to ori characterization of the part collection</ABBR></A> for a description). 
	We looked into literature and found 16 possible genome modifications for <DFN data-info="3-hydroxypropionic acid">3HPA</DFN> production made by other groups. 
	Table 2 shows a list of all modifications. 
	Our plan was to order oligos for each of these modifications, leading to upregulation, downregulation, deletion or site-directed mutagenesis of the respective gene. 
	The aimed modification for each gene is shown in the table. For the next step we decided to use natural transformation for integration of these oligos into the strain. 
	Because each cell will take up different numbers and different combinations of oligos, we will get a population of genomic variations (<A href="https://2018.igem.org/Team:Marburg/Design"><ABBR title="link to method section of Strain Engineering">link to method section of Strain Engineering</ABBR></A>). 
	The genetically diverse population can then be put into <DFN data-info="Fluorescence activated cell sorting"> FACS </DFN> and we can select for cells with the strongest signal and thereby the best productivity. 
	Finally, the whole cycle of <DFN data-info="Multiplex automated genome engineering">MAGE</DFN> and <DFN data-info="Fluorescence activated cell sorting"> FACS </DFN> can be repeated, leading to continuously improved pathways. We believe that the combination of <DFN data-info="Multiplex automated genome engineering">MAGE</DFN>-<DFN data-info="Fluorescence activated cell sorting"> FACS </DFN> with the fast growth of <I>V. natriegens</I> will speed up the process of metabolic engineering and pathway optimization significantly!


	<FIGURE style="width: 50%; float: right;"><FIGCAPTION><B>
				Figure 9: Our workflow for directed evolution via MAGE-FACS
			</B>To use the fast growth of <I>V. natriegens</I> we designed a workflow for directed evolution. It combines MAGE with FACS and starts with the introduction of oligos for genome modifications into our strain. The usage of randomized oligos with variable bases increases the genetic diversity. By using FACS together with our biosensors we are able to rapidly select the most promising strains and can analyse their genetic composition. This finding can be re-feeded into the DBTL-cycle to start a new round in the workflow leading to a continously improved pathway.
		</FIGCAPTION></FIGURE></P><H3>
	Outlook
</H3><P>
	In our project we try to accelerate the process of metabolic engineering in all of its facets and thereby find a suitable pathway variant to achieve high <DFN data-info="3-hydroxypropionic acid">3HPA</DFN> productivity. 
	Nevertheless, there will still be many possibilities to further improve the production of <DFN data-info="3-hydroxypropionic acid">3HPA</DFN> in <I>V. natriegens</I>. Table 2 shows a list of 16 genome modifications, that were made in other organisms with the same or a similar pathway to direct the flow towards <DFN data-info="3-hydroxypropionic acid">3HPA</DFN>. 
	Some of these modifications target for instance NADPH homeostasis. 
	During glycolysis cells produce NADH while the production of <DFN data-info="3-hydroxypropionic acid">3HPA</DFN> depends on NADPH. 
	Therefore, a misbalance between NADH and NADPH could arise, but overexpression of <I>pntAB</I> (nicotinamide nucleotide transhydrogenase) or <I>gapA</I> (NADP+-dependent glycerol-3-phosphate dehydrogenase) has been shown to stabilize cofactor balance (See table for references). Another possibility to enrich the NADPH pool is to downregulate glycolysis while upregulating pentose phosphate pathway, which produces NADPH instead of NADH. 
	This can be achieved by deletion of NAD+-dependent <I>gapA</I> or deletion of <I>pfkA</I> (phosphofructokinase). Another critical point in the <DFN data-info="3-hydroxypropionic acid">3HPA</DFN> route are side reactions. 
	Pyruvate can be reduced to lactate or decarboxylated to acetate, what could be suppressed by deletion of <I>ldhA</I> (lactate dehydrogenase) or <I>pox</I> (pyruvate decarboxylase). 
	The flow from pyruvate to acetyl-CoA can be increased by overexpression of <I>pdh</I> (pyruvate dehydrogenase). 
	For acetyl-CoA there are also side reactions, for instance hydrolysis to acetate. This reaction can be suppressed by deletion of <I>pta</I> (phosphotransacetylase) and <I>ackA</I> (acetate kinase) or by overexpression of <I>ald6</I> (aldehyde dehydrogenase) and <I>acs</I> (acetyl-CoA synthase) for the reverse reaction to acetyl-CoA. 
	Malonyl-CoA can also go into side reactions for fatty acid biosynthesis instead of the reduction to <DFN data-info="3-hydroxypropionic acid">3HPA</DFN>. 
	However, downregulation of <I>fabAB</I> (fatty acid biosynthesis) or inducible inactivation of <I>fabAB</I> has been showed to increase <DFN data-info="3-hydroxypropionic acid">3HPA</DFN> titer. 
	Additionally it has also been shown that site-directed mutagenesis of <I>acc</I> and <I>mcr</I> genes increases product titer <A href="https://doi.org/10.1186/s12934-016-0451-5"><ABBR title="Kildegaard, K.R., Jensen, N.B., Schneider, K., Czarnotta, E., Özdemir, E., Klein, T., Maury, J., Ebert, B.E., Christensen, H.B., Chen, Y., Kim, I.K., Herrgård, M.J., Blank, L.M., Forster, J., Nielsen, J., Borodina, I., 2016. Engineering and systems-level analysis of Saccharomyces cerevisiae for production of 3-hydroxypropionic acid via malonyl-CoA reductase-dependent pathway. Microb. Cell Fact. 15. ">(Kildegaard <I>et al.</I> 2016)</ABBR></A>;

	<A href="https://doi.org/10.1016/j.ymben.2016.01.001
	"><ABBR title="Liu, C., Ding, Y., Zhang, R., Liu, H., Xian, M., Zhao, G., 2016. Functional balance between enzymes in malonyl-CoA pathway for 3-hydroxypropionate biosynthesis. Metab. Eng. 34, 104–111. ">(Liu <I>et. al.</I> 2016)</ABBR></A>.

	<FIGURE><FIGCAPTION><B>Table 2:</B> Genome modifications for further pathway optimization</FIGCAPTION></FIGURE></P><TABLE><TBODY><TR><TH>Gene</TH><TH>Organism</TH><TH>Function</TH><TH>Modification</TH><TH>Reference</TH></TR><TR><TD><B><I>gapA</I></B></TD><TD><I>S. cerevisiae</I></TD><TD>GAPDH (NADP-dependent)</TD><TD>Overexpression</TD><TD><A href="https://doi.org/10.1016/j.ymben.2014.01.005
		"><ABBR title="Chen, Y., Bao, J., Kim, I.-K., Siewers, V., Nielsen, J., 2014. Coupled incremental precursor and co-factor supply improves 3-hydroxypropionic acid production in Saccharomyces cerevisiae. Metab. Eng. 22, 104–109. ">(Chen <I>et. al.</I> 2014)</ABBR></A></TD></TR><TR><TD><B><I>mis1</I></B></TD><TD><I>S. cerevisia</I></TD><TD>malate synthase</TD><TD>Deletion</TD><TD><A href="https://doi.org/10.1016/j.ymben.2014.01.005
"><ABBR title="Chen, Y., Bao, J., Kim, I.-K., Siewers, V., Nielsen, J., 2014. Coupled incremental precursor and co-factor supply improves 3-hydroxypropionic acid production in Saccharomyces cerevisiae. Metab. Eng. 22, 104–109. ">(Chen <I>et. al.</I> 2014)</ABBR></A></TD></TR><TR><TD><B><I>pox</I></B></TD><TD><I>E. coli</I></TD><TD>pyruvate decarboxylase</TD><TD>Deletion</TD><TD><A href="https://doi.org/10.1021/bp049730r
			"><ABBR title="Dittrich, C.R., Vadali, R. V., Bennett, G.N., San, K.-Y., 2008. Redistribution of Metabolic Fluxes in the Central Aerobic Metabolic Pathway of &lt;i&gt;E. coli&lt;/i&gt; Mutant Strains with Deletion of the ackA-pta and poxB Pathways for the Synthesis of Isoamyl Acetate. Biotechnol. Prog. 21, 627–631.">(Dittrich <I>et. al.</I> 2008)</ABBR></A></TD></TR><TR><TD><B><I>acc1</I></B></TD><TD><I>S. cerevisiae</I></TD><TD>acetyl-Coa carboxylase</TD><TD>Site-directed mutagenesis</TD><TD><A href="https://doi.org/10.1186/s12934-016-0451-5"><ABBR title="Kildegaard, K.R., Jensen, N.B., Schneider, K., Czarnotta, E., Özdemir, E., Klein, T., Maury, J., Ebert, B.E., Christensen, H.B., Chen, Y., Kim, I.K., Herrgård, M.J., Blank, L.M., Forster, J., Nielsen, J., Borodina, I., 2016. Engineering and systems-level analysis of Saccharomyces cerevisiae for production of 3-hydroxypropionic acid via malonyl-CoA reductase-dependent pathway. Microb. Cell Fact. 15. ">(Kildegaard <I>et al.</I> 2016)</ABBR></A></TD></TR><TR><TD><B><I>ald6</I></B></TD><TD><I>S. cerevisiae</I></TD><TD>aldehyde dehydrogenase</TD><TD>Overexpression</TD><TD><A href="https://doi.org/10.1186/s12934-016-0451-5"><ABBR title="Kildegaard, K.R., Jensen, N.B., Schneider, K., Czarnotta, E., Özdemir, E., Klein, T., Maury, J., Ebert, B.E., Christensen, H.B., Chen, Y., Kim, I.K., Herrgård, M.J., Blank, L.M., Forster, J., Nielsen, J., Borodina, I., 2016. Engineering and systems-level analysis of Saccharomyces cerevisiae for production of 3-hydroxypropionic acid via malonyl-CoA reductase-dependent pathway. Microb. Cell Fact. 15. ">(Kildegaard <I>et al.</I> 2016)</ABBR></A></TD></TR><TR><TD><B><I>pdh</I></B></TD><TD><I>S. cerevisiae</I></TD><TD>pyruvate dehydrogenase</TD><TD>Overexpression</TD><TD><A href="https://doi.org/10.1186/s12934-016-0451-5"><ABBR title="Kildegaard, K.R., Jensen, N.B., Schneider, K., Czarnotta, E., Özdemir, E., Klein, T., Maury, J., Ebert, B.E., Christensen, H.B., Chen, Y., Kim, I.K., Herrgård, M.J., Blank, L.M., Forster, J., Nielsen, J., Borodina, I., 2016. Engineering and systems-level analysis of Saccharomyces cerevisiae for production of 3-hydroxypropionic acid via malonyl-CoA reductase-dependent pathway. Microb. Cell Fact. 15. ">(Kildegaard <I>et al.</I> 2016)</ABBR></A></TD></TR><TR><TD><B><I>acs</I></B></TD><TD><I>E. coli</I></TD><TD>acetyl-CoA synthase</TD><TD>Overexpression</TD><TD><A href="https://doi.org/10.1007/s00253-005-0230-4
			"><ABBR title="Lin, H., Castro, N.M., Bennett, G.N., San, K.-Y., 2006. Acetyl-CoA synthetase overexpression in Escherichia coli demonstrates more efficient acetate assimilation and lower acetate accumulation: a potential tool in metabolic engineering. Appl. Microbiol. Biotechnol. 71, 870–874. ">(Lin <I>et. al.</I> 2006)</ABBR></A></TD></TR><TR><TD><B><I>bauA</I></B></TD><TD>From <I>P. putida</I> into <I>E. coli</I></TD><TD>beta-ala-pyruvate aminotransferase</TD><TD>Complementation</TD><TD><A href="https://doi.org/10.1016/j.ymben.2016.01.001
"><ABBR title="Liu, C., Ding, Y., Zhang, R., Liu, H., Xian, M., Zhao, G., 2016. Functional balance between enzymes in malonyl-CoA pathway for 3-hydroxypropionate biosynthesis. Metab. Eng. 34, 104–111. ">(Liu <I>et. al.</I> 2016)</ABBR></A></TD></TR><TR><TD><B><I>panD</I></B></TD><TD><I>E. coli</I></TD><TD>aspartate-1-decarboxylase</TD><TD>Deletion</TD><TD><A href="https://doi.org/10.1016/j.ymben.2016.01.001
"><ABBR title="Liu, C., Ding, Y., Zhang, R., Liu, H., Xian, M., Zhao, G., 2016. Functional balance between enzymes in malonyl-CoA pathway for 3-hydroxypropionate biosynthesis. Metab. Eng. 34, 104–111. ">(Liu <I>et. al.</I> 2016)</ABBR></A></TD></TR><TR><TD><B><I>ackA</I></B></TD><TD><I>E. coli</I> BL21 (DE3)</TD><TD>acetate kinase</TD><TD>Deletion</TD><TD><A href="https://doi.org/10.1016/j.jbiotec.2011.06.008"><ABBR title="Rathnasingh, C., Raj, S.M., Lee, Y., Catherine, C., Ashok, S., Park, S., 2012. Production of 3-hydroxypropionic acid via malonyl-CoA pathway using recombinant Escherichia coli strains. J. Biotechnol. 157, 633–640. ">(Rathnasingh <I>et al.</I> 2012)</ABBR></A></TD></TR><TR><TD><B><I>ldhA</I></B></TD><TD><I>E. coli</I> BL21 (DE3)</TD><TD>lactate dehydrogenase</TD><TD>Deletion</TD><TD><A href="https://doi.org/10.1016/j.jbiotec.2011.06.008"><ABBR title="Rathnasingh, C., Raj, S.M., Lee, Y., Catherine, C., Ashok, S., Park, S., 2012. Production of 3-hydroxypropionic acid via malonyl-CoA pathway using recombinant Escherichia coli strains. J. Biotechnol. 157, 633–640. ">(Rathnasingh <I>et al.</I> 2012)</ABBR></A></TD></TR><TR><TD><B><I>pta</I></B></TD><TD><I>E. coli</I> BL21 (DE3)</TD><TD>phosphotransacetylase</TD><TD>Deletion</TD><TD><A href="https://doi.org/10.1016/j.jbiotec.2011.06.008"><ABBR title="Rathnasingh, C., Raj, S.M., Lee, Y., Catherine, C., Ashok, S., Park, S., 2012. Production of 3-hydroxypropionic acid via malonyl-CoA pathway using recombinant Escherichia coli strains. J. Biotechnol. 157, 633–640. ">(Rathnasingh <I>et al.</I> 2012)</ABBR></A></TD></TR><TR><TD><B><I>sucAB</I></B></TD><TD><I>E. coli</I> BL21 (DE3)</TD><TD>2-oxo-ketoglutarate dehydrogenase</TD><TD>Deletion</TD><TD><A href="https://doi.org/10.1016/j.jbiotec.2011.06.008"><ABBR title="Rathnasingh, C., Raj, S.M., Lee, Y., Catherine, C., Ashok, S., Park, S., 2012. Production of 3-hydroxypropionic acid via malonyl-CoA pathway using recombinant Escherichia coli strains. J. Biotechnol. 157, 633–640. ">(Rathnasingh <I>et al.</I> 2012)</ABBR></A></TD></TR><TR><TD><B><I>gapA</I></B></TD><TD><I>C. glutamicum</I></TD><TD>GADPH</TD><TD>Deletion</TD><TD><A href=" https://doi.org/10.1007/s00253-012-4314-7
			"><ABBR title="Siedler, S., Lindner, S.N., Bringer, S., Wendisch, V.F., Bott, M., 2013. Reductive whole-cell biotransformation with Corynebacterium glutamicum: improvement of NADPH generation from glucose by a cyclized pentose phosphate pathway using pfkA and gapA deletion mutants. Appl. Microbiol. Biotechnol. 97, 143–152.">(Siedler <I>et. al.</I> 2013)</ABBR></A></TD></TR><TR><TD><B><I>pfkA</I></B></TD><TD><I>C. glutamicum</I></TD><TD>phosphofructokinase</TD><TD>Deletion</TD><TD><A href=" https://doi.org/10.1007/s00253-012-4314-7
			"><ABBR title="Siedler, S., Lindner, S.N., Bringer, S., Wendisch, V.F., Bott, M., 2013. Reductive whole-cell biotransformation with Corynebacterium glutamicum: improvement of NADPH generation from glucose by a cyclized pentose phosphate pathway using pfkA and gapA deletion mutants. Appl. Microbiol. Biotechnol. 97, 143–152.">(Siedler <I>et. al.</I> 2013)</ABBR></A></TD></TR><TR><TD><B><I>fabAB</I></B></TD><TD><I>E. coli</I></TD><TD>fatty acid biosynthesis</TD><TD>Downregulation</TD><TD><A href="https://doi.org/10.1016/j.ymben.2015.03.018
			"><ABBR title="Yang, Y., Lin, Y., Li, L., Linhardt, R.J., Yan, Y., 2015. Regulating malonyl-CoA metabolism via synthetic antisense RNAs for enhanced biosynthesis of natural products. Metab. Eng. 29, 217–226. ">(Yang <I>et. al.</I> 2015)</ABBR></A></TD></TR><TR><TD><B><I>pntAB</I></B></TD><TD><I>E. coli</I> BL21 (DE3)</TD><TD>nicotinamide nucleotide tranhydrogenase</TD><TD>Overexpression</TD><TD><A href="https://doi.org/10.1016/j.jbiotec.2011.06.008"><ABBR title="Rathnasingh, C., Raj, S.M., Lee, Y., Catherine, C., Ashok, S., Park, S., 2012. Production of 3-hydroxypropionic acid via malonyl-CoA pathway using recombinant Escherichia coli strains. J. Biotechnol. 157, 633–640. ">(Rathnasingh <I>et al.</I> 2012)</ABBR></A></TD></TR><TR><TD><B><I>mcrC</I></B></TD><TD><I>E. coli</I> (from <I>C. aurantiacus</I>)</TD><TD>malonyl-CoA reductase</TD><TD>Site directed mutagenesis</TD><TD><A href="https://doi.org/10.1016/j.ymben.2016.01.001
			"><ABBR title="Liu, C., Ding, Y., Zhang, R., Liu, H., Xian, M., Zhao, G., 2016. Functional balance between enzymes in malonyl-CoA pathway for 3-hydroxypropionate biosynthesis. Metab. Eng. 34, 104–111. ">(Liu <I>et. al.</I> 2016)</ABBR></A></TD></TR></TBODY></TABLE><P>
	We chose the production of <DFN data-info="3-hydroxypropionic acid">3HPA</DFN> as application to show the functionality of the Marburg Collection and as proof-of-principle for our metabolic engineering workflow.
	However, our goal is not to produce high amounts of <DFN data-info="3-hydroxypropionic acid">3HPA</DFN>, but rather to show that <I>V. natriegens</I> can be used as test chassis for rapid testing of many pathway variations to find the best one. The next step would therefore be to evaluate, if the pathway with its genetic composition can be transmitted to other organisms to see if the route still works. 
	Besides that, there are many other compounds of medical and economical relevance, which still can’t be produced biotechnologically. 
	The production of the anticancer drug taxol involves for instance 19 enzymes and is therefore a very challenging pathway with lot of bottlenecks and tunable reactions (<A href="https://dx.doi.org/10.1007%2Fs11101-005-3748-2
			"><ABBR title="Croteau, R., Ketchum, R.E.B., Long, R.M., Kaspera, R., Wildung, M.R., 2006. Taxol biosynthesis and molecular genetics. Phytochem. Rev.">(Croteau <I>et. al.</I> 2006)</ABBR></A>). 
	We think it would be beneficial to use <I>V. natriegens</I> as chassis for testing this challenging pathway or parts of it.
	Its short doubling time and genetic accessibility will accelerate the process of enzyme engineering and finetune expression levels or gene combinations.
	Another crucial aspect, that has to be tackled is the universality of our workflow. 
	We involved biosensors for product screening but for many metabolites there are still no known sensor proteins. 
	There are three possibilities to circumvent this issue. 
	The first is to find new biosensors, what is complicated, because many compounds, especially drugs for medical purposes don’t occur in the nature, therefore it is unlikely that there are still proteins binding specifically to these compounds. 
	The second possibility would be to engineer proteins to become drug-specific sensors. The third possibility is to upscale <DFN data-info="Liquid Chromatography-Mass Spectrometry">LC-MS</DFN> techniques to enable larger sets of measurements and faster screening. 
	<DFN data-info="Liquid Chromatography-Mass Spectrometry">LC-MS</DFN> has an unbeaten accuracy but its bottleneck is the velocity. 
	The average time needed for one sample lies between 1 and 10 minutes, making measurements of large sets of samples impossible. 
	Furthermore <DFN data-info="Liquid Chromatography-Mass Spectrometry">LC-MS</DFN> requires high amounts of buffer. However, as mentioned in the Labautomation-section [soll ein Link sein], the company Labcyte developed a system to use acoustic waves for pipetting nanoliter droplets into microtiter plates in a very fast manner. 
	Now they invented an acoustic mass spectrometer, which combines acoustic nanodroplet formation with mass spectrometry. 
	The device is able to measure 3 samples per second without the need of a mobile phase. 
	Such a device could be used instead of biosensor-based flow cytometry to broader the applicability of our metabolic engineering workflow.

</P><H3>
	References
</H3>

Abdel-Hamid, A.M., Cronan, J.E., 2007. Coordinate expression of the acetyl coenzyme A carboxylase genes, <I>accB</I> and <I>accC</I>, is necessary for normal regulation of biotin synthesis in <I>Escherichia coli</I>. J. Bacteriol. 189, 369–376. 
https://doi.org/10.1128/JB.01373-06

Chen, Y., Bao, J., Kim, I.-K., Siewers, V., Nielsen, J., 2014. Coupled incremental precursor and co-factor supply improves 3-hydroxypropionic acid production in <I>Saccharomyces cerevisiae</I>. Metab. Eng. 22, 104–109. 
https://doi.org/10.1016/j.ymben.2014.01.005

Croteau, R., Ketchum, R.E.B., Long, R.M., Kaspera, R., Wildung, M.R., 2006. Taxol biosynthesis and molecular genetics. Phytochem. Rev. 
https://doi.org/10.1007/s11101-005-3748-2

De Ruyck, J., Famerée, M., Wouters, J., Perpète, E.A., Preat, J., Jacquemin, D., 2007. Towards the understanding of the absorption spectra of NAD(P)H/NAD(P)+ as a common indicator of dehydrogenase enzymatic activity. Chem. Phys. Lett. 450, 119–122. 
https://doi.org/10.1016/j.cplett.2007.10.092

Dittrich, C.R., Vadali, R. V., Bennett, G.N., San, K.-Y., 2008. Redistribution of Metabolic Fluxes in the Central Aerobic Metabolic Pathway of <I>E. coli</I> Mutant Strains with Deletion of the ackA-pta and poxB Pathways for the Synthesis of Isoamyl Acetate. Biotechnol. Prog. 21, 627–631. 
https://doi.org/10.1021/bp049730r

Gande, R., Dover, L.G., Krumbach, K., Besra, G.S., Sahm, H., Oikawa, T., Eggeling, L., 2007. The two carboxylases of <I>Corynebacterium glutamicum</I> essential for fatty acid and mycolic acid synthesis. J. Bacteriol. 189, 5257–5264. 
https://doi.org/10.1128/JB.00254-07

Hügler, M., Menendez, C., Schägger, H., Fuchs, G., 2002. Malonyl-coenzyme A reductase from <I>Chloroflexus aurantiacus</I>, a key enzyme of the 3-hydroxypropionate cycle for autotrophic CO(2) fixation. J. Bacteriol. 184, 2404–10. 
https://doi.org/10.1128/JB.184.9.2404-2410.2002

Jain, A., Srivastava, P., 2013. Broad host range plasmids. FEMS Microbiol. Lett. 348, 87–96. 
https://doi.org/10.1111/1574-6968.12241

Kildegaard, K.R., Jensen, N.B., Schneider, K., Czarnotta, E., Özdemir, E., Klein, T., Maury, J., Ebert, B.E., Christensen, H.B., Chen, Y., Kim, I.K., Herrgård, M.J., Blank, L.M., Forster, J., Nielsen, J., Borodina, I., 2016. Engineering and systems-level analysis of <I>Saccharomyces cerevisiae</I> for production of 3-hydroxypropionic acid via malonyl-CoA reductase-dependent pathway. Microb. Cell Fact. 15. 
https://doi.org/10.1186/s12934-016-0451-5

Lambert, J.H., Anding, E., 1760. Photometrie. Photometria, sive De mensura et gradibus luminis, colorum et umbrae (1760) : Lambert, Johann Heinrich, Internet Archive, Photometria, sive De mensura et gradibus luminis, colorum et umbrae. Eberhardt Klett, Augsburg.

Lin, H., Castro, N.M., Bennett, G.N., San, K.-Y., 2006. Acetyl-CoA synthetase overexpression in <I>Escherichia coli</I> demonstrates more efficient acetate assimilation and lower acetate accumulation: a potential tool in metabolic engineering. Appl. Microbiol. Biotechnol. 71, 870–874. 
https://doi.org/10.1007/s00253-005-0230-4

Liu, C., Ding, Y., Zhang, R., Liu, H., Xian, M., Zhao, G., 2016. Functional balance between enzymes in malonyl-CoA pathway for 3-hydroxypropionate biosynthesis. Metab. Eng. 34, 104–111. 
https://doi.org/10.1016/j.ymben.2016.01.001

Liu, C., Wang, Q., Xian, M., Ding, Y., Zhao, G., 2013. Dissection of Malonyl-Coenzyme A Reductase of <I>Chloroflexus aurantiacus</I> Results in Enzyme Activity Improvement. PLoS One 8, 1–8. 
https://doi.org/10.1371/journal.pone.0075554

  Liu, D., Xiao, Y., Evans, B.S., Zhang, F., 2015. Negative Feedback Regulation of Fatty Acid Production Based on a Malonyl-CoA Sensor-Actuator. ACS Synthetic Biology 4, 132-40
  http://www.ncbi.nlm.nih.gov/pubmed/24377365

Rathnasingh, C., Raj, S.M., Lee, Y., Catherine, C., Ashok, S., Park, S., 2012. Production of 3-hydroxypropionic acid via malonyl-CoA pathway using recombinant <I>Escherichia coli strains</I>. J. Biotechnol. 157, 633–640. 
https://doi.org/10.1016/j.jbiotec.2011.06.008

Rogers, J.K. &amp; Church, G.M., 2016. Genetically encoded sensors enable real-time observation of metabolite production. Proc. Natl. Acad. Sci. U.S.A. 113, 2388-93.
https://doi.org/10.1073/pnas.1600375113

Siedler, S., Lindner, S.N., Bringer, S., Wendisch, V.F., Bott, M., 2013. Reductive whole-cell biotransformation with <I>Corynebacterium glutamicum</I>: improvement of NADPH generation from glucose by a cyclized pentose phosphate pathway using pfkA and gapA deletion mutants. Appl. Microbiol. Biotechnol. 97, 143–152.
 https://doi.org/10.1007/s00253-012-4314-7

Valdehuesa, K.N.G., Liu, H., Nisola, G.M., Chung, W.J., Lee, S.H., Park, S.J., 2013. Recent advances in the metabolic engineering of microorganisms for the production of 3-hydroxypropionic acid as C3 platform chemical. Appl. Microbiol. Biotechnol. 97, 3309–3321. 
https://doi.org/10.1007/s00253-013-4802-4

Vidra, A., Németh, Á., 2017. Bio-based 3-hydroxypropionic Acid : A Review. period. Polytech. Chem. Eng. 1–11.
Yang, Y., Lin, Y., Li, L., Linhardt, R.J., Yan, Y., 2015. Regulating malonyl-CoA metabolism via synthetic antisense RNAs for enhanced biosynthesis of natural products. Metab. Eng. 29, 217–226. 
https://doi.org/10.1016/j.ymben.2015.03.018

</DIV></DIV></DIV><DIV class="collapsible"><DIV class="btn_expand"><DIV class="btn_e_title"> References </DIV><DIV class="btn_e_desc"> Here we present the literature which builds the foundation of our work.</DIV></DIV><DIV class="content">
		Andreou, Andreas I., und Naomi Nakayama. 2018. „Mobius Assembly: A Versatile Golden-Gate Framework towards Universal DNA Assembly“. PLOS ONE 13(1): e0189892.

Aiyar, Sarah E, Tamas Gaal, and Richard L Gourse. 2002. RRNA Promoter Activity in the Fast-Growing Bacterium Vibrio Natriegens. Journal of bacteriology 184(5): 1349-58. 

Casini, Arturo u. a. 2014. „R2oDNA Designer: Computational Design of Biologically Neutral Synthetic DNA Sequences“. ACS synthetic biology 3(8): 525–28.

Chen, Xiaoying, Jennica Zaro, und Wei-Chiang Shen. 2013. „Fusion Protein Linkers: Property, Design and Functionality“. Advanced drug delivery reviews 65(10): 1357–69.

Engler, Carola, Romy Kandzia, und Sylvestre Marillonnet. 2008. „A One Pot, One Step, Precision Cloning Method with High Throughput Capability“. PLOS ONE 3(11): e3647.

Kim, Juhyun u. a. 2016. „Properties of alternative microbial hosts used in synthetic biology: towards the design of a modular chassis“. Essays in Biochemistry 60(4): 303–13.

Langley, K. E. u. a. 1975. „Molecular Basis of Beta-Galactosidase Alpha-Complementation“. Proceedings of the National Academy of Sciences of the United States of America 72(4): 1254–57.

Moore, Simon J., Hung-En Lai, Richard J. R. Kelwick, Soo Mei Chee, David J. Bell, Karen Marie Polizzi, and Paul S. Freemon. 2016. „EcoFlex: A Multifunctional MoClo Kit for E. coli Synthetic Biology.”  ACS Synthetic Biology (ACS Publications) 5(10): 1059 – 1069. 

Pingoud, Alfred, und Albert Jeltsch. 2001. „Structure  and function of type II restriction endonucleases“. Nucleic Acids Research 29(18): 3705–27.

Pollak, Bernardo, Ariel Cerda, Mihails Delmans, Simón Álamos, Tomás Moyano, Anthony West, Rodrigo A Gutiérrez, Nicola Patron, Fernán Federici, Jim Haseloff. 2018.  „Loop Assembly: a simple and open system for recursive fabrication of DNA circuits”. bioRxiv. 

Mairhofer, Juergen, Alexander Wittwer, Monika Cserjan-Puschmann, und Gerald Striedner. 2015. „Preventing T7 RNA Polymerase Read-through Transcription—A Synthetic Termination Signal Capable of Improving Bioprocess Stability.” ACS Synthetic Biology (ACS Publications) 4(3): 265 – 273. 

Sabourin, Michelle, Creighton T. Tuzon, Timothy S. Fisher, und Virginia A. Zakian. 2007. „A Flexible Protein Linker Improves the Function of Epitope-Tagged Proteins InSaccharomyces Cerevisiae“. Yeast 24(1): 39–45.

Sarrion-Perdigones, Alejandro, Erica Elvira Falconi, Sara I. Zandalinas, Paloma Juárez, Asun Fernández-del-Carmen, Antonio Granell, Diego Orzaez. 2011. „GoldenBraid: An Iterative Cloning System for Standardized Assembly of Reusable Genetic Modules“. PLOS ONE 6(7): e21622.

Weber, Ernst, Carola Engler , Ramona Gruetzner, Stefan Werner, Sylvestre Marillonnet . „A Modular Cloning System for Standardized Assembly of Multigene Constructs“. 2011. PLOS ONE 6(2): e16765. 

A two-plasmid inducible CRISPR/Cas9 genome editing tool for Clostridium acetobutylicum. Journal of Microbiological Methods Volume 140, September 2017, Pages 5-11. https://www.sciencedirect.com/science/article/pii/S0167701217301604

Recombineering in ​<I>Vibrio natriegens</I>. bioRxiv preprint, Apr. 24, 2017. https://www.biorxiv.org/content/biorxiv/early/2017/04/26/130088.full.pdf

Multigene Editing in the Escherichia coli Genome via the CRISPR-Cas9 System. Applied and Environmental Microbiology, Apr. 2015 Volume 81 Number 7. https://aem.asm.org/content/aem/81/7/2506.full.pdf

The no-SCAR (Scarless Cas9 Assisted Recombineering) system for genome editing in <I>Escherichia coli</I>. Scientific Reports volume5, Article number: 15096 (2015). https://www.nature.com/articles/srep15096.pdf

Rational design of highly active sgRNAs for CRISPR-Cas9-mediated gene inactivation. Nat Biotechnol. 2014 Dec;32(12):1262-7. https://www.nature.com/articles/nbt.3026.pdf

Blokesch, M. (2012). &quot;TransFLP--a method to genetically modify Vibrio cholerae based on natural transformation and FLP-recombination.&quot; J Vis Exp(68).
Brosius, J., M. Erfle and J. Storella (1985). &quot;Spacing of the -10 and -35 regions in the tac promoter. Effect on its in vivo activity.&quot; J Biol Chem 260(6): 3539-3541.

Dunn, J. J. and F. W. Studier (1983). &quot;Complete nucleotide sequence of bacteriophage T7 DNA and the locations of T7 genetic elements.&quot; J Mol Biol 166(4): 477-535.

Dalia, T. N., C. A. Hayes, S. Stolyar, C. J. Marx, J. B. McKinlay and A. B. Dalia (2017). &quot;Multiplex Genome Editing by Natural Transformation (MuGENT) for Synthetic Biology in Vibrio natriegens.&quot; ACS Synth Biol 6(9): 1650-1655.

Dalia, A. B., McDonough, E., and Camilli, A.. Multiplex genome editing by natural transformation. Proc. Natl. Acad. Sci. U. S. A. (2014) 111, 8937−8942.

Dürwald, H. and Hoffmann-Berling, H., Endonuclease-I-deficient and ribonuclease I-deficient Escherichia coli mutants., J Mol Biol. (1968) 14;34(2):331-46.

Gentz, R. and H. Bujard (1985). &quot;Promoters recognized by Escherichia coli RNA polymerase selected by function: highly efficient promoters from bacteriophage T5.&quot; J Bacteriol 164(1): 70-77.
 
Gilbert, W. and B. Muller-Hill (1966). &quot;Isolation of the lac repressor.&quot; Proc Natl Acad Sci U S A 56(6): 1891-1898.

Jeong, H., H. J. Kim and S. J. Lee (2015). &quot;Complete Genome Sequence of Escherichia coli Strain BL21.&quot; Genome Announc 3(2).


Itaya M, Kawata Y, Sato M, Tomita M, Nakahigashi K., A simple method to provide a shuttling plasmid for delivery to other host ascertained by prolonged stability of extracellular plasmid DNA released from Escherichia coli K12 endA mutant, deficient in major endonuclease., J Biochem. (2012);152(6):501-4.

Issue Cite as The hsd (host specificity) genes of E. coli K12 Bela Sain Noreen E. Murray Molecular and General Genetics MGG November 1980, Volume 180, Issue 1, pp 35–46.

Kaneko, S., and Itaya, M., Integration of stable extracellular DNA released from Escherichia coli into the Bacillus subtilis genome vector by culture mix method., Nucleic Acids Res. (2010); 38(8):2551-7.

Moffatt, B. A. and F. W. Studier (1987). &quot;T7 lysozyme inhibits transcription by T7 RNA polymerase.&quot; Cell 49(2): 221-227.

Studier, F. W. and B. A. Moffatt (1986). &quot;Use of bacteriophage T7 RNA polymerase to direct selective high-level expression of cloned genes.&quot; J Mol Biol 189(1): 113-130.

Willetts N, Skurray R. The conjugation system of F-like plasmids. Annu Rev Genet. 1980;14:41-76.

Weinstock MT, Hesek ED, Wilson CM, Gibson DG., Vibrio natriegens as a fast-growing host for molecular biology., J Nat Methods. (2016);13(10):849-51”&gt;(Weinstock<I>et al.</I>2016)

Zhang, X. and F. W. Studier (1997). &quot;Mechanism of inhibition of bacteriophage T7 RNA polymerase by T7 lysozyme.&quot; J Mol Biol 269(1): 10-27.

A two-plasmid inducible CRISPR/Cas9 genome editing tool for Clostridium acetobutylicum. Journal of Microbiological Methods Volume 140, September 2017, Pages 5-11. https://www.sciencedirect.com/science/article/pii/S0167701217301604

Recombineering in ​<I>Vibrio natriegens</I>. bioRxiv preprint, Apr. 24, 2017. https://www.biorxiv.org/content/biorxiv/early/2017/04/26/130088.full.pdf

Multigene Editing in the Escherichia coli Genome via the CRISPR-Cas9 System. Applied and Environmental Microbiology, Apr. 2015 Volume 81 Number 7. https://aem.asm.org/content/aem/81/7/2506.full.pdf

The no-SCAR (Scarless Cas9 Assisted Recombineering) system for genome editing in <I>Escherichia coli</I>. Scientific Reports volume5, Article number: 15096 (2015). https://www.nature.com/articles/srep15096.pdf

Rational design of highly active sgRNAs for CRISPR-Cas9-mediated gene inactivation. Nat Biotechnol. 2014 Dec;32(12):1262-7. https://www.nature.com/articles/nbt.3026.pdf

Blokesch, M. (2012). &quot;TransFLP--a method to genetically modify Vibrio cholerae based on natural transformation and FLP-recombination.&quot; J Vis Exp(68).
Brosius, J., M. Erfle and J. Storella (1985). &quot;Spacing of the -10 and -35 regions in the tac promoter. Effect on its in vivo activity.&quot; J Biol Chem 260(6): 3539-3541.

Dunn, J. J. and F. W. Studier (1983). &quot;Complete nucleotide sequence of bacteriophage T7 DNA and the locations of T7 genetic elements.&quot; J Mol Biol 166(4): 477-535.

Dalia, T. N., C. A. Hayes, S. Stolyar, C. J. Marx, J. B. McKinlay and A. B. Dalia (2017). &quot;Multiplex Genome Editing by Natural Transformation (MuGENT) for Synthetic Biology in Vibrio natriegens.&quot; ACS Synth Biol 6(9): 1650-1655.

Dalia, A. B., McDonough, E., and Camilli, A.. Multiplex genome editing by natural transformation. Proc. Natl. Acad. Sci. U. S. A. (2014) 111, 8937−8942.

Dürwald, H. and Hoffmann-Berling, H., Endonuclease-I-deficient and ribonuclease I-deficient Escherichia coli mutants., J Mol Biol. (1968) 14;34(2):331-46.

Gentz, R. and H. Bujard (1985). &quot;Promoters recognized by Escherichia coli RNA polymerase selected by function: highly efficient promoters from bacteriophage T5.&quot; J Bacteriol 164(1): 70-77.
 
Gilbert, W. and B. Muller-Hill (1966). &quot;Isolation of the lac repressor.&quot; Proc Natl Acad Sci U S A 56(6): 1891-1898.

Jeong, H., H. J. Kim and S. J. Lee (2015). &quot;Complete Genome Sequence of Escherichia coli Strain BL21.&quot; Genome Announc 3(2).


Itaya M, Kawata Y, Sato M, Tomita M, Nakahigashi K., A simple method to provide a shuttling plasmid for delivery to other host ascertained by prolonged stability of extracellular plasmid DNA released from Escherichia coli K12 endA mutant, deficient in major endonuclease., J Biochem. (2012);152(6):501-4.

Issue Cite as The hsd (host specificity) genes of E. coli K12 Bela Sain Noreen E. Murray Molecular and General Genetics MGG November 1980, Volume 180, Issue 1, pp 35–46.

Kaneko, S., and Itaya, M., Integration of stable extracellular DNA released from Escherichia coli into the Bacillus subtilis genome vector by culture mix method., Nucleic Acids Res. (2010); 38(8):2551-7.

Moffatt, B. A. and F. W. Studier (1987). &quot;T7 lysozyme inhibits transcription by T7 RNA polymerase.&quot; Cell 49(2): 221-227.

Studier, F. W. and B. A. Moffatt (1986). &quot;Use of bacteriophage T7 RNA polymerase to direct selective high-level expression of cloned genes.&quot; J Mol Biol 189(1): 113-130.

Willetts N, Skurray R. The conjugation system of F-like plasmids. Annu Rev Genet. 1980;14:41-76.

Weinstock MT, Hesek ED, Wilson CM, Gibson DG., Vibrio natriegens as a fast-growing host for molecular biology., J Nat Methods. (2016);13(10):849-51”&gt;(Weinstock<I>et al.</I>2016)

Zhang, X. and F. W. Studier (1997). &quot;Mechanism of inhibition of bacteriophage T7 RNA polymerase by T7 lysozyme.&quot; J Mol Biol 269(1): 10-27.</DIV></DIV></DIV></DIV><FOOTER><DIV style="position: absolute;left: 0;bottom: 0;color: #b1b1b1;margin: 0.5%;font-size: 0.8em;">B. Marchal</DIV></FOOTER></BODY></HTML>